
# Native acridone synthases I and II from *Ruta graveolens* L. form homodimers

**Richard Lukačin**${}^{1,a}$, **Karin Springob**${}^{1,a}$, **Claus Urbanke**${}^{b}$, **Claudia Ernwein**${}^{a}$, **Gudrun Schröder**${}^{c}$, **Joachim Schröder**${}^{c}$, **Ulrich Matern**${}^{a,*}$

${}^{a}$Institut für Pharmazeutische Biologie, Universität Marburg, Deutschhausstrasse 17A, D-35037 Marburg, Germany  
${}^{b}$Medizinische Hochschule Hannover, Betriebseinheit Biophysikalisch-Biochemische Verfahren, Carl Neuberg Str. 1, 30623 Hannover, Germany  
${}^{c}$Biologisches Institut II, Albert-Ludwigs-Universität, Schänzlestrasse 1, D-79104 Freiburg, Germany  

Received 9 February 1999

---

### Abstract
Acridone synthase II cDNA was cloned from irradiated cell suspension cultures of *Ruta graveolens* L. and expressed in *Escherichia coli*. The translated polypeptide of $M_{r}$ 42681 revealed a high degree of similarity to heterologous chalcone and stilbene synthases (70–75%), and the sequence was 94% identical to that of acridone synthase I cloned previously from elicited *Ruta* cells. Highly active recombinant acridone synthases I and II were purified to apparent homogeneity by a four-step purification protocol, and the affinities to $N$-methylanthraniloyl-CoA and malonyl-CoA were determined. The molecular mass of acridone synthase II was estimated from size exclusion chromatography on a Fractogel EMDBioSEC (S) column at about 45 kDa, as compared to a mass of 44 ± 3 kDa found for the acridone synthase I on Superdex 75. Nevertheless, the sedimentation analysis by ultracentrifugation revealed molecular masses of 81 ± 4 kDa for both acridone synthases. It is proposed, therefore, that the acridone synthases of *Ruta graveolens* are typical homodimeric plant polyketide synthases.

© 1999 Federation of European Biochemical Societies.

#### Key words: Acridone synthase; Polyketide synthase; Subunit composition; Acridone alkaloid biosynthesis; *Ruta graveolens* L.

---

### 1. Introduction

Acridone synthase (ACS) catalyzes the condensation of $N$-methylanthraniloyl-CoA and three malonyl-CoAs to yield 1,3-dihydroxy-$N$-methylacridone (Fig. 1), a reaction pivotal to the biosynthesis of all acridone alkaloids [1]. ACS is thus classified as a member of the polyketide synthases which are involved in the biosynthesis of various secondary plant metabolites such as flavonoids, stilbenes, styrylpyrones, benzophenones or benzalacetone [2,3]. Acridone alkaloids are confined exclusively to the Rutaceae, and ACS had been initially purified from elicited cell suspension cultures of *Ruta graveolens* L. [4]. This enzyme showed an $M_{r}$ of 40,000 on SDS-PAGE, whereas a molecular mass of roughly 69 kDa was estimated for the native *Ruta* enzyme [4], which ambiguously left the option for a mono- or dimer-subunit composition.

---

### 2. Materials and methods

#### 2.1. Cell cultures and irradiation
Cell suspension cultures of the common rue (*Ruta graveolens* strain R-20) were grown as described previously [4,5], and the cells were induced by simultaneous irradiation with UV-B and white light in a standard light field consisting of two TL 40-W/12 Philips (UV-B), two TL 40-W/18 Philips (blue light) and two L 40-W/73 Osram (UV-A) lighting tubes (fluence rate = 11 W m${}^{-2}$).

#### 2.2. Poly(A)${}^{+}$ RNA
*Ruta* cells were harvested from cultures that had been irradiated for 3 or 6 h. The cells were deep-frozen immediately in liquid nitrogen and stored at $-70^{\circ}\mathrm{C}$ till use. Poly(A)${}^{+}$ RNA was isolated from the frozen cells as described elsewhere [6,7].

#### 2.3. Construction and screening of the cDNA library
Poly(A)${}^{+}$ RNA from the irradiated *Ruta* cells was used for cDNA synthesis [8], and the cDNA was extended with EcoRI and XhoI adaptors prior to ligation into the $\lambda$ ZAP vector (Stratagene, Heidelberg, Germany). The cDNA library of $2.4 \times 10^{5}$ recombinant phages was screened for polyketide synthases by plaque hybridization using a radiolabeled 1.4 kb EcoRI fragment from *Callistephus chinensis* chalcone synthase (EMBL accession number Z67988), yielding five positive clones in two further rounds of screening. One of these positive clones (pBK-CMV) was analyzed in detail.

#### 2.4. DNA sequencing
The cDNA inserts were subcloned into vector pTZ19R [9], and DNA sequencing was accomplished by the dideoxy nucleotide chain termination technique [10], using the universal and reverse sequencing primers and additional oligonucleotide primers synthesized as required.

#### 2.5. Modification of cDNA and construction of expression vector
The ACS II cDNA insert of the $\lambda$ ZAP plasmid pBK-CMV was restricted with BamHI and XbaI and ligated into the vector pTZ19R [8]. *E. coli* strain RZ1032 [11] was transformed with the resulting

---

*Corresponding author. Fax: (49) (6421) 28 6678.  
E-mail: matern@mailer.uni-marburg.de*

${}^{1}$These authors contributed equally to the work presented.

#### Abbreviations:
ACS, acridone synthase; CHS, chalcone synthase; ORF, open reading frame; IPTG, isopropyl-$\beta$-D-thiogalactoside; SDS-PAGE, polyacrylamide gel electrophoresis in the presence of sodium dodecylsulfate

0014-5793/99/$20.00 © 1999 Federation of European Biochemical Societies. All rights reserved.  
PII: S0014-5793(99)00355-5

construct, and single-stranded DNA was rescued by the in vivo excision protocol using the phage M13K07. The ssDNA was used as a template for the subsequent in vitro site-directed mutagenesis which was performed according to the procedure of Kunkel et al. [12]. The mutation was generated by using the oligonucleotide 5′-CAGG-GATTCCATGGCTCTACTCACAAATC-3′ mismatch primer for hybridization with the ACS II 5′-coding region in antisense orientation, which introduced a unique *NcoI* site at the translational start ATG codon. The mutation was verified by DNA sequencing [10], and the ACS II coding DNA was subsequently isolated by digestion with *NcoI* and *PstI* and subcloned into the expression vector pQE6 (ACS II-pQE6).

### 2.6. ACS expression and purification

*E. coli* strain M15 harboring the plasmid pRep4 was transformed with the ACS II-pQE6 construct. The expression of ACS II was induced by the addition of IPTG and the bacteria harvested as described [13]. The heterologous expression of ACS I was performed according to Junghanns et al. [5]. Crude bacterial extracts were prepared by ultrasonication and centrifugation (30,000 × g, 5°C, 10 min), and the clear supernatants were fractionated on a Fractogel EMD DEAE 650 (M) anionic exchanger (Merck, Darmstadt, Germany) followed by size exclusion chromatography on a Fractogel EMD BioSEC (S) column (Merck, Darmstadt), chromatography on hydroxyapatite (research sample; Bio-Rad, Munich, Germany), as described elsewhere [13], and a second fractionation on Fractogel EMD DEAE 650 (S). Each step of the purification was examined by SDS-PAGE [14] and activity assays of the individual enzyme fractions.

### 2.7. Size exclusion chromatography

The molecular weights of native ACS I and II were estimated by size exclusion chromatography on a Fractogel EMD BioSEC (S) column (600 × 16 mm; Merck, Darmstadt, Germany), equilibrated with 50 mM Tris-HCl, 200 mM NaCl, pH 7.5. The column was calibrated in the range of 18–300 kDa (protein calibration kit Combithek, Boehringer, Mannheim, Germany), and the void volume was determined with blue dextran 2000. Enzymes or reference proteins were applied in a small volume (200 μl), and individual runs were repeated at least three times.

### 2.8. Analytical ultracentrifugation

Analytical ultracentrifugation was performed in a Beckman XL-A analytical ultracentrifuge equipped with absorption scanner optics using the An-50 8-place rotor. For equilibrium measurements double sector charcoal filled epon centerpieces were used. 150 μl of the respective sample was overlaid on 50 μl artificial bottom (FLUORINERT FC-43, ABCR, Karlsruhe) and centrifuged at 15,000 rpm. Scans were taken at 30 min intervals for the first 10 h to measure the approach to equilibrium. After that scans were taken every 90 min. When the measured concentration profile remained unchanged for at least 12 h we assumed equilibrium to be attained and scans from these 12 h were averaged to yield the final equilibrium concentration profile.

For the approach to equilibrium data apparent molecular masses and sedimentation rate constants were evaluated by fitting Lamm's differential equation [15] describing the simultaneous sedimentation and diffusion of a single species to the measured data using the program package AKKUPROG [16]. The same program was used to determine apparent molecular masses by fitting the ideal equilibrium distribution for a single species [17] to the measured equilibrium concentration profile. The partial specific volumes of the proteins were calculated from the amino acid composition.

#### 2.9. General assays

The protein composition of samples was examined by SDS-PAGE [14], and protein amounts were quantified after precipitation with trichloroacetic acid and employing the Lowry procedure [18] with bovine serum albumin as a reference. The activity and the kinetic constants of the ACSs were determined as described previously [4,5], assaying at saturating concentrations of one or the other substrate.

## 3. Results

### 3.1. Molecular cloning and sequence analysis

A cDNA library from irradiated *Ruta graveolens* cells was established with the aim to isolate polyketide synthases related to ACS I, as reported previously from elicited *Ruta* cells [5], or to heterologous CHSs. Accordingly, the library was screened with a 1.4 *EcoRI* CHS cDNA fragment from *C. chinensis* (G. Forkmann, TU Freising-Weihenstephan, Germany) as a hybridization probe. The screening finally resulted in one clone encoding ACS I flanked in 3′ by an extended untranslated region, which might represent an allele of ACS I and was not studied further, and another clone of 1.37 kb spanning one long open reading frame (ORF) of 1173 bp, which revealed 90% sequence identity with the coding region of ACS I, flanked by 64 and 131 bp sequences at the 5′ and 3′ borders, respectively. The ORF encoded a polypeptide of 391 amino acids (Fig. 2). Alignment studies revealed a high degree of similarity of this polypeptide to the ACS I with 367 (93.9%) identical amino acid residues and 20 (5.1%) conservative exchanges including two Q/E transitions [19]. The translated polypeptide of the newly isolated cDNA clone thus differed significantly in only four amino acid residues (Met-12, Asn-107, Gln-119 and Glu-315) from the ACS I enzyme sequence (Fig. 2).

---

**Fig. 1. Schematic reaction catalyzed by the acridone synthase from *Ruta graveolens*. The conversion of the intermediate open-chain polyketide to N-methylacridone involves successive acylation, nucleophilic substitution and dehydration reactions.**

![Chemical Reaction Diagram](https://i.imgur.com/yourimageurl.png)

---

**FEBS 21823 29-3-99**

R. Lukačin et al./FEBS Letters 448 (1999) 135–140

|  | 10 | 20 | 30 | 40 | 50 | 60 |
| --- | --- | --- | --- | --- | --- | --- |
| ACSII | MESLKEMRKA QKSEGPAAIL AIGTATPDNV YIQADYPDYY FKITKSEHMT ELKDKFKTLC |  |  |  |  |  |
| ACS I | .M. | FM. | RM. | R. |  |  |

|  | 70 | 80 | 90 | 100 | 110 | 120 |
| --- | --- | --- | --- | --- | --- | --- |
| ACSII | EKSMIRKRHM CFSQEFLKAN PEVCKHMGKS LNARQDIAVV ETPRIGKEAA VKAIKEWGHP |  |  |  |  |  |
| ACS I | ED. | L.N. | Q. |  |  |  |

|  | 130 | 140 | 150 | 160 | 170 | 180 |
| --- | --- | --- | --- | --- | --- | --- |
| ACSII | KSSITHLIFC TSAGVDMPGA DYQLTRMLGL NPSVKRMMIY QQGCYAGGTV LRLAKDLAEN |  |  |  |  |  |
| ACS I | S. | I. | V. |  |  |  |

|  | 190 | 200 | 210 | 220 | 230 | 240 |
| --- | --- | --- | --- | --- | --- | --- |
| ACSII | NKGSRVLVVC SELTAPTFRG PSPDAVDSLV GQALFADGAA ALVVGADPDT SVERALYYIV |  |  |  |  |  |
| ACS I | S.I. | L. |  |  |  |  |

|  | 250 | 260 | 270 | 280 | 290 | 300 |
| --- | --- | --- | --- | --- | --- | --- |
| ACSII | SASQMLLPDS DGAIEGHIRE EGLTVHLKKD VPALFSANID TPLVEAFRPL GISDWNSIFW |  |  |  |  |  |
| ACS I | K. |  |  |  |  |  |

|  | 310 | 320 | 330 | 340 | 350 | 360 |
| --- | --- | --- | --- | --- | --- | --- |
| ACSII | IAHPGGPAIL DQIEVKLGLK EDKLRAKHKV MSEYGNMSSS CVLFVLDEMR NKSLQDGKST |  |  |  |  |  |
| ACS I | E. | SR. |  |  |  |  |

|  | 370 | 380 | 391 |
| --- | --- | --- | --- |
| ACSII | TGEGLDWGVL FGFGPGLTVE TVVLRSVPVE |  |  |
| ACS I | Q. | I. | I. |

Fig. 2. Alignment of the acridone synthase I and II polypeptides translated from *Ruta graveolens* cDNAs. Only those amino acid residues representing conservative exchanges or introducing major changes in the overall charge (bold printed) in ACS I as compared to ACS II are highlighted.

### 3.2. Functional classification

The coding region of the isolated cDNA was expressed in *E. coli* for functional characterization, and a prominent protein band of 42 kDa was observed upon SDS-PAGE examination of the extracts prepared from the IPTG-induced transformants. Moreover, these extracts contained considerable ACS activity (21 μkat/kg), whereas the extracts from the non-induced transformants lacked both the 42 kDa polypeptide and ACS activity. These results unequivocally confirmed that the cDNA isolated from irradiated *Ruta* cells encoded another ACS, designated as ACS II.

In parallel experiments, ACS I was also expressed in *E. coli* [5], and the recombinant polypeptides were purified by sequential ion exchange, size exclusion, and hydroxyapatite chromatographies followed by a final separation on Fractogel EMDDAE 650(S). This purification protocol yielded:

---

**Table 1: Kinetic comparison of purified ACS I and ACS II**

| Enzyme | ACS I | ACS II |
| --- | --- | --- |
| pH optimum | 7.5 | 7.0 |
| Optimal temperature (°C) | 32 | 40 |
| Specific activity (μkat/kg) | 159 | 236 |
| \(K_{\text{mmalonyl-CoA}}\) (μM) | 13 | 5 |
| \(K_{\text{mN-methylanthraniloyl-CoA}}\) (μM) | 62 | 77 |

*All determinations were made at least in triplicate. Both the pH optima were determined in various 0.1 M buffers [13]. ACS I activity in the range of pH 6.0–7.5 and pH 7.5–9.0 was assayed in sodium phosphate and Tris-HCl, respectively, and the activity at pH 7.5 found in Tris-HCl reached about 95% of that determined in phosphate. ACS II activity in the range of pH 6.0–7.5 was assayed in potassium phosphate buffer.*

---

FEBS 21823 29-3-99

to a lesser extent, on the ionic strength and pH of the solvent. These various parameters limit the reliability of the procedure. Therefore, the sedimentation rate constants of pure ACS I and ACS II were subjected to analytical ultracentrifugation in order to assess their partial specific volume. Fig. 4 shows the equilibrium concentration profiles for ACS I and ACS II which yielded molecular masses of 81 and 82 ± 4 kDa, respectively, corresponding to both proteins forming dimers. Table 2 shows the sedimentation rate constants evaluated from the approach to equilibrium measurements and the corresponding frictional ratios assuming a monomer and a dimer. Since the lowest possible frictional ratio is that of a perfect sphere (1.0) these data clearly confirm the dimeric structure of the proteins.

### 4. Discussion

The sequence homologies of ACS I and II with heterologous CHSs and STSs classify both ACSs as plant polyketide synthases [2,3]. Nevertheless, the mode of ACS action appears to differ slightly from that of the CHSs and STSs, since the acylation reaction forming the ring from three malonyl moieties must be followed by nucleophilic attack of the secondary amine on the neighboring ring-carbonyl and subsequent loss of water prior to aromatization (Fig. 1). This reaction mechanism as compared to that of CHSs or STSs must be the consequence of subtle changes in the primary sequence, since CHSs and STSs from various plants did not show ACS activity [5], and irrespective of the fact that the carboxy-terminal "CHS/STS signature motif" with the presumed binding site of the aromatic starter substrate are preserved [2,3].

Plant polyketide synthases are involved in the biosynthesis of a broad range of secondary metabolites of divergent physiological relevance, including the solar UV-B protectant flavonoids [21] and the stilbene phytoalexins [22]. Correspondingly, CHSs are commonly induced upon irradiation, whereas STSs are expressed on fungal elicitation. Acridones might serve more than one purpose, since they are capable of absorbing UV-B as well as visible light and possess a considerable antimicrobial potential [1,5]. Accordingly, ACS I was constitutively expressed in dark-grown *Ruta* cell cultures, but was transiently induced further upon the addition of fungal elicitor [23]. It remains to be established whether ACS II or related enzymes in *Ruta* respond to an appropriate light regime.

All the CHSs and STSs have been described as homodimers [2,3], although in case of CHS and STS a single active site in a dimer is sufficient for catalysis [20]. It was thus puzzling that the size exclusion chromatographies on Superdex 75 or Fractogel EMD BioSEC (S) matrices always assigned single polypeptides to the highly active, recombinant ACSs I and II.

---

**Table 2**  
Hydrodynamic properties of ACS I and II determined from approach to equilibrium experiments in the analytical ultracentrifuge

| Protein | ACS I | ACS II |
|---------|--------|---------|
| Partial specific volume (kg/m³) (from amino acid composition) | 7.40 × 10⁻⁴ | 7.42 × 10⁻⁴ |
| Theoretical molecular mass (kDa) (from amino acid sequence) | 42.7 | 42.6 |
| $S_{20,w}$ (10⁻¹³ s) | 5.56 ± 0.1 | 5.37 ± 0.1 |
| Apparent molecular mass (kDa) (from approach to equilibrium data) | 81 ± 4 | 82 ± 4 |
| Frictional ratio ($f/f_0$) (dimer) | 0.76 | 0.79 |
| Frictional ratio ($f/f_0$) (monomer) | 1.21 | 1.25 |

---

**Fig. 3. Separation of purified recombinant ACSs I and II (2.8 μg each; lanes 1 and 2) by SDS-PAGE using 12.5% separation and 5% stacking gels and stained with Coomassie brilliant blue R 250. Marker proteins in the range of 66–14.3 kDa (SDS-7 kit, Sigma) were used for comparison (left lane).**

Homogeneous ACSs I and II (Fig. 3), which preserved their high catalytic activities (Table 1). Kinetic examination revealed distinct pH and temperature optima for the ACS II, as compared to ACS I (Table 1), with about 60% and 85% of the maximal activity recorded at pH 7.0 and 8.0, respectively, for ACS I or 91% and 77% at pH 6.5 and 7.5, respectively, for ACS II; these minor discrepancies might be explained on the basis of the different overall charge due to the amino acid exchanges (Fig. 2).

#### 3.3. Subunit composition

The molecular masses of catalytically highly active ACS I and II were calculated from size exclusion chromatographies on a calibrated Fractogel EMD BioSEC (S) column, which is supposed to fractionate polypeptides to size in the range from 5 to 1000 kDa. Maximal activities were always recorded at an elution volume corresponding to masses of about 45 kDa, and these results comply with the molecular mass of 44 ± 3 kDa reported earlier for the *Ruta* ACS I [5]. The data thus suggested that both ACSs are active in the form of single polypeptides. However, the results are in sharp contrast to the homodimeric composition proposed for the closely related heterologous CHSs and stilbene synthases, which are built up from 388–400 amino acid polypeptide subunits [2,3], regardless of the fact that a single active site is sufficient for enzyme catalysis [20].

The elution characteristics of proteins on gel chromatography are supposed to depend primarily on the spatial protein structure as well as on the inertness of the gel matrix, but also,

R. Lukačin et al./FEBS Letters 448 (1999) 135–140

**Fig. 4. Analytical equilibrium centrifugation of ACS I and ACS II in 20 mM potassium phosphate pH 6.8 at 14,000 rpm. ■: measured concentration profile; solid line: theoretical concentration profile using a molecular mass of 72.6 kDa and 72.7 kDa for ACS I and ACS II, respectively (partial specific volumes are given in Table 2); π: difference between measured and theoretical concentration distribution (residuals, note the different abscissa scale).**

A careful examination by sedimentation analysis, however, proved the homodimeric composition of native ACSs I and II, corresponding to molecular masses of about 81 and 82 ± 4 kDa, respectively (Fig. 4, Table 2). Retrospectively, the results from size exclusion chromatography may be rationalized by an interaction of the enzyme with the gel matrix, causing the disaggregation of subunits. This assumption, however, is unlikely with reference to CHS and STS which do not reassociate to the active state once the subunits have been separated (J. Schröder, unpublished). Alternatively, the interaction of the ACS dimers with the gel matrix must be strong enough to cause the considerable retardation observed in the elution volume.

**Acknowledgements:** This work was supported by the Deutsche Forschungsgemeinschaft and Fonds der Chemischen Industrie. The authors wish to thank Dr. G. Buchholz for the propagation of cell cultures, Inga Gröning for her part in the protein purification and Dr. D. Kreusch for valuable discussions.

**References**

[1] Maier, W., Baumert, A., Schumann, B., Furukawa, H. and Gröger, D. (1993) Phytochemistry 32, 691–698.

[2] Schröder, J. (1997) Trends Plant Sci. 10, 373–378.

[3] Schröder, J. (1999) The chalcone-stilbene synthase-type family of condensing enzymes, in: Comprehensive Natural Products Chemistry, Vol. 1: Polyketides and Other Secondary Metabolites Including Fatty Acids and Their Derivatives (Sankawa, U., Ed.), Pergamon, Oxford.

[4] Baumert, A., Maier, W., Gröger, D. and Deutzmann, R. (1994) Z. Naturforsch. 49c, 26–32.

[5] Junghanns, K.T., Kneusel, R.E., Baumert, A., Maier, W., Gröger, D. and Matern, U. (1995) Plant Mol. Biol. 27, 681–692.

[6] Palmiter, R.D. (1974) Biochemistry 13, 3606–3615.

[7] Schröder, G., Brown, J.W.S. and Schröder, J. (1988) Eur. J. Biochem. 172, 161–169.

[8] Sambrook, J., Fritsch, E.F. and Maniatis, T. (1989) Molecular Cloning: A Laboratory Manual, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY.

[9] Zagursky, R.J. and Berman, M.L. (1984) Gene 27, 183–191.

[10] Sanger, F., Nicklen, S. and Coulson, A.R. (1977) Proc. Natl. Acad. Sci. USA 74, 5463–5467.

[11] Kunkel, T.A. (1985) Proc. Natl. Acad. Sci. USA 82, 488–492.

[12] Kunkel, T.A., Roberts, J.D. and Zakour, R.A. (1987) Methods Enzymol. 154, 367–382.

[13] Lukacin, R. and Britsch, L. (1997) Eur. J. Biochem. 249, 748–757.

[14] Laemmli, U.K. (1970) Nature 227, 680–685.

[15] Lamm, O. (1929) Ark. Mat. Astron. Fys. 21B, No. 2.

[16] Kindler, B. (1997) Doctoral Thesis, Universität Hannover.

[17] Yphantis, D.A. and Waugh, D.F. (1956) J. Phys. Chem. 60, 23.

[18] Sandermann, H. and Strominger, L. (1972) J. Biol. Chem. 247, 5123–5131.

[19] Britsch, L., Dedio, J., Saedler, H. and Forkmann, G. (1993) Eur. J. Biochem. 217, 745–754.

[20] Tropf, S., Kärcher, B., Schröder, G. and Schröder, J. (1995) J. Biol. Chem. 270, 7922–7928.

[21] Caldwell, M.M., Robberecht, R. and Flint, S.D. (1983) Plant Physiol. 58, 445–450.

[22] Hart, J.H. (1981) Annu. Rev. Phytopathol. 19, 437–458.

[23] Junghanns, K.T., Kneusel, R.E., Gröger, D. and Matern, U. (1998) Phytochemistry 49, 403–411.
