
Review

Metabolism and selected functions of sphingolipids in the yeast  
Saccharomyces cerevisiae  

Robert C. Dickson *, Robert L. Lester  

Department of Biochemistry and the Lucille P. Markey Cancer Center, University of Kentucky Medical Center, Lexington, KY 40536-0298, USA  

Received 11 February 1999; received in revised form 12 April 1999; accepted 26 April 1999  

---

### Abstract

Our knowledge of sphingolipid metabolism and function in *Saccharomyces cerevisiae* is growing rapidly. Here we discuss the current status of sphingolipid metabolism including recent evidence suggesting that exogenous sphingoid long-chain bases must first be phosphorylated and then dephosphorylated before incorporation into ceramide. Phenotypes of strains defective in sphingolipid metabolism are discussed because they provide hints about the undiscovered functions of sphingolipids and are one of the major reasons for studying this model eukaryote. The long-chain base phosphates, dihydrosphingosine-1-phosphate and phytosphingosine-1-phosphate, have been hypothesized to play roles in heat stress resistance, perhaps acting as signaling molecules. We evaluate the data supporting this hypothesis and suggest future experiments needed to verify it. Finally, we discuss recent clues that may help to reveal how sphingolipid synthesis and total cellular sphingolipid content are regulated. © 1999 Elsevier Science B.V. All rights reserved.

---

### Keywords: Phenotype; Heat shock; Long-chain base phosphate; Suppressor gene; Ceramide

---

### Contents

1. Introduction . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
R.C. Dickson, R.L. Lester / Biochimica et Biophysica Acta 1438 (1999) 305–321

3.9. LCB4 and LCB5 – Addition of phosphate to long-chain bases . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
R.C. Dickson, R.L. Lester / Biochimica et Biophysica Acta 1438 (1999) 305–321

![Diagram of sphingolipids](https://i.imgur.com/yourimage.png)

Fig. 1. Diagram of sphingolipids. These compounds, with the exception of the mammalian long-chain base sphingosine, are components of *S. cerevisiae* sphingolipid metabolism which is outlined in Fig. 2.

sphingosine or sphinganine (DHS) and phytosphingosine (PHS), having hydroxyls on both C-3 and C-4. For comparison (Fig. 1), PHS lacks the 4,5-double bond found in sphingosine, the primary mammalian long-chain base and the compound for which sphingolipids are named.

Steps in *S. cerevisiae* sphingolipid metabolism are outlined in Fig. 2; other reviews should be consulted for more detailed information [3–5, 20]. Sphingolipid synthesis begins with the condensation of palmitoyl-CoA and serine to yield 3-ketodihydrosphingosine (3-ketosphinganine), which is reduced to yield DHS. Under special circumstances, for example, for a 30 min period after yeast cells are transferred to 37°C from 25°C, stearoyl-CoA is used to make long-chain bases with 20 carbon atoms, in addition to those with 18 carbons [21, 22]. The function of C-20 long-chain bases is not clear, but they may play roles in heat stress as discussed below.

In *S. cerevisiae* DHS is rapidly amide-linked to a C-26 fatty acid to give N-acylsphinganine (dihydroceramide or ceramide-1, Fig. 2). Ceramide-1 is hydroxylated at C-4 by Sur2p/Syr2p to give ceramide-2 (also referred to as phytoceramide). Alternatively, DHS is hydroxylated at C-4 by Sur2p/Syr2p to give PHS [23, 24], which is acylated to give ceramide-2. It is not yet clear whether DHS or ceramide-1 is the preferred substrate for C-4 hydroxylation by the Sur2 enzyme. It is clear that PHS rapidly accumulates for a brief time following a 37°C heat shock, suggesting that under these conditions DHS can be hydroxylated before incorporation into ceramide [21, 22]. In mammals, DHS is acylated, usually with

a C-18 fatty acid, and the resulting dihydroceramide then serves as a substrate for formation of the 4,5-trans double bond found in the sphingosine component of ceramides [25, 26].

Wild type *S. cerevisiae* cells hydroxylate the fatty acid in ceramide-2 at C-2 producing ceramide-3. Further hydroxylation at C-3 of the fatty acid yields ceramide-4 (Fig. 2) [20]. The Scs7 protein hydroxylates C-2 and perhaps C-3 [23, 27, 28]. Differences in hydroxylation of the long-chain base and C-26 fatty acid yield four types of ceramides with ceramide-3 and -4 being the primary ones found in yeast sphingolipids.

*S. cerevisiae*, most other fungi, and plants make sphingolipids that contain myo-inositol [4]. The three myo-inositol-containing sphingolipids in *S. cerevisiae* (Fig. 2) are inositol-phosphoceramide (IPC), mannosyl-inositol-phosphoceramide (MIPC) and mannosyl-(inositol-P)₂-ceramide (MIP₂C). The structure of these compounds is incomplete. Available data indicate that the phosphorus in IPC is linked to the 1-OH of ceramide [29] and inositol (Lester et al.). NMR spectroscopy of the complete head group of M(IP)₂C (Shelling et al., unpublished) as well as other data (Lester et al., unpublished) suggest that the structure of M(IP)₂C is inositol-1-P(6)-mannose(α-1,2)-inositol-1-P-(1)-ceramide.

Addition of inositol phosphate is the only known modification to the C-1 OH of ceramides in *S. cerevisiae*. It is possible that other unidentified modifications are made, but they could only be present in small amounts (reviewed in [4]). In addition, some fungi and all plant species that have been examined

R.C. Dickson, R.L. Lester / Biochimica et Biophysica Acta 1438 (1999) 305–321

Malonyl-
CoA
Palmitoyl-CoA
LCB1
Serine
LCB2
3-ketodihydro sphingosine
SUR4
TSC10
FEN1
NADPH
Dihydrosphingosine (DHS)
ATP
LCB4, LCB5
H₂O
LHS-P
LHS-R
C₂₆-CoA
NADPH, O₂
SUR2
Phytosphingosine (PHS)
ATP
LCB4, LCB5
H₂O
LHS-R
PHS-P
Ceramide-1
(DHS, C₂₆FA)
NADPH, O₂
SUR2
C₂₆-CoA
Ceramide-2
(PHS, C₂₆FA)
NADPH,
O₂?
SCS7
Ethanolamine-P
+ C₁₆ aldehydes
DPL1
Ceramide-3
(PHS, OHC₂₆FA)
NADPH,
O₂?
Ceramide-4
(PHS, (OH)₂C₂₆FA)

Further metabolism of ceramide (CER)

Ceramide
Phosphatidyl-
inositol
Ins-P-Cer
GDP-
Mannose
Man-Ins-P-Cer
Phosphatidyl-
inositol
Man(Ins-P)₂-Cer
AUR1
(SUR1, CSG2)
IPT1
(IPC)
(MIPC)
(M(Ins-P)₂C)
↓
Cer-P-Ins-
glycan
anchored
proteins
Phospho-
Lipase(s) C
H₂O
H₂O
H₂O
Ins-P + Cer
Man-Ins-P + Cer
Man(Ins-P)₂ + Cer

Fig. 2. Overview of sphingolipid metabolism in *S. cerevisiae*. Metabolites are shown in bold lettering, genes are shown in italics and are the preferred designations listed in the Saccharomyces Genome Database (http://genome-www.stanford.edu/Saccharomyces). As indicated at the bottom of the figure, ceramide is used as the lipid anchor for some proteins (see [125] and references therein). The polar head groups on sphingolipids can be cleaved in a cell-free lysate by one or more unidentified phospholipase C-like enzymes [39]. Alternative names for the ceramide species are: ceramide-1, ceramide A; ceramide-2, ceramide B; ceramide-3, ceramide C; ceramide-4, ceramide D. The same nomenclature applies to complex sphingolipids that contain ceramide; thus, IPC-3 is IPC containing ceramide-3. The stereochemical configuration of the 2- and 3-OH groups on the fatty acid in ceramide-4 is unknown.

add one or more sugar residues to the C-1 of ceramide to form a second class of sphingolipids referred to as glycosylceramides [4]. Animals do not add inositol phosphate to ceramide, but instead they add phosphocholine to form sphingomyelin or glucose or galactose that can be further decorated with carbohydrates and sometimes with sulfates to form hundreds of complex sphingolipids [30].

Once it became clear that sphingolipid synthesis is essential for the viability of *S. cerevisiae* cells [31] and that IPC is not made by animals, it was logical to infer that inhibitors of IPC synthase (phosphatidyli-

BBAMCB 55455 2-6-99

R.C. Dickson, R.L. Lester / Biochimica et Biophysica Acta 1438 (1999) 305–321

nositol:ceramide phosphinositol transferase) might be effective anti-fungal drugs. A major impetus for using IPC synthase as a target for anti-fungal drugs was the discovery that aureobasidin A, a cyclic depsipeptide, is a potent inhibitor of IPC synthase activity and that the *AURI* gene encodes IPC synthase or is a subunit of the enzyme [32]. It is not clear at this time which type of ceramide (1–4, Fig. 2) is the preferred substrate for IPC synthase because the enzyme has not been purified and assayed in vitro with various ceramides (reviewed in [4]). Hence the pathway shown at the bottom of Fig. 2 indicates that IPC synthase can use any *S. cerevisiae* ceramide as a substrate.

It is assumed that the synthesis of sphingolipids begins in the endoplasmic reticulum and proceeds up to formation of ceramide or IPC. This assumption is based primarily upon work done in mammals [33, 34]. With three exceptions, there is no direct evidence in *S. cerevisiae*, contrary to what is stated in various publications and databases, that the enzymes necessary for ceramide synthesis are located in the endoplasmic reticulum. The three exceptions are Csg2p [35], Fen1(Elo2)p and Sur4(Elo3)p [36] which have been localized by light microscopy techniques to the endoplasmic reticulum. The function of Csg2p in sphingolipid synthesis is unclear (see below). Fen1p and Sur4p are necessary for elongation of C-22 fatty acids to C-26 [37].

The enzymes acting on ceramides to form IPC, MIPC or M(IP)₂C and the enzymes for metabolism of DHS-1-phosphate or PHS-1-phosphate (Fig. 2) have not been localized in *S. cerevisiae* to specific cellular compartments. The Scs7 and Sur2 proteins contain C-terminal sequences, KMKYE and VKKEK, respectively, that match a consensus sequence for retention in the endoplasmic reticulum [23, 27], but this location has not been verified experimentally. Synthesis of some species of IPC and their conversion to MIPC and MIP₂C is thought to occur in the Golgi apparatus based upon their intracellular accumulation in mutants blocked at specific steps in the protein secretory pathway [38]. These results need to be verified by localizing sphingolipid biosynthetic enzymes.

M(IP)₂C accounts for about 75% of the mass of total sphingolipids with the remainder being equally divided between IPC and MIPC [29]. We estimate that these three sphingolipids make up 7–8% of the mass of the plasma membrane since they constitute 30% of the plasma membrane phospholipids [1]. Because of their large mass and large negative charge they likely have general effects on processes that are dependent upon the plasma membrane. It is not known if they are distributed in one or both leaflets of the plasma membrane.

Breakdown of membrane sphingolipids is essential in mammals and occurs both as a part of normal cellular metabolism and in the generation of sphingolipid second messengers [10, 30]. In vivo there is no evidence for breakdown of IPC, MIPC or M(IP)₂C in *S. cerevisiae*. What has been detected is one or more phospholipase C-like activities in cell-free extracts that releases ceramide and the polar head groups from sphingolipids (Fig. 2, [39]). In addition, yeast extracts have been shown to contain an enzyme that releases ceramide from sphingomyelin [40]. The identity of these phospholipase activities awaits determination, as does their physiological function.

Finally, it was assumed that long-chain bases including 3-keto-DHS, DHS and PHS were taken up from the culture medium and used directly as substrates by the appropriate enzyme in the biosynthetic pathway shown in Fig. 2. Available evidence now indicates that this assumption is wrong and that at least DHS and PHS must first be phosphorylated and then dephosphorylated in order to be incorporated in sphingolipids. The first hint of this pathway came from studies of mutants that failed to incorporate \[^3\text{H}]\text{DHS}\) into sphingolipids. These studies uncovered a gene, termed *LCB3*, which was required for incorporation of exogenous \[^3\text{H}]\text{DHS}\) into sphingolipids [41]. Soon thereafter, *LCB3* was identified as a phosphatase that could dephosphorylate DHS-1-P [42] and PHS-1-P [43]. Proof of this pathway will require showing that utilization of exogenous long-chain bases requires phosphorylation and then dephosphorylation prior to incorporation into sphingolipids. The existence of the pathway is surprising and raises questions about the cellular location of these enzymes, their substrates, and their products.

3. Phenotypes of strains defective in sphingolipid metabolic genes

Genes have now been identified for most steps in *S. cerevisiae* sphingolipid metabolism (Fig. 2). A notable exception is ceramide synthesis for which no gene has been identified in any organism. It is possible that other genes encoding unidentified subunits of the enzymes shown in Fig. 2 may exist, since none of the enzymes have been purified to homogeneity and their subunit composition remains unknown. Ceramide synthase has been partially purified from bovine liver but awaits molecular characterization [44].

### 3.1. FEN1 (GNS1/ELO2) and SUR4 (APA1/ELO3/SRE1) – Fatty acid elongation

The primary in vivo activity of Fen1p is to elongate C-20:0 fatty acids to C-22:0 [37]. The enzyme can also convert C-22:0 fatty acids to C-24:0, but cannot convert C-24:0 to C-26:0. Sur4p is responsible for converting C-24:0 fatty acids to C-26:0; it also is able to make C-22:0 and C-24:0 fatty acids, but not as well as Fen1p. In *fen1* or *sur4* null mutant strains the level of sphingolipids is reduced and PHS accumulates, but these changes have not been well quantified by chemical analyses. Strains lacking either gene are viable but the double mutant is not [45].

FEN1 was first cloned by complementation of a *gns1* mutant strain [46]. The *gns1* mutant was isolated because it confers resistance to echinocandins. The drug-resistant mutant also had reduced β-glucan synthase activity which led the authors to suspect that Gns1p was a subunit of the glucan synthase. More likely, reduced glucan synthase activity and the slow growth phenotype of their strain were due to reduced synthesis of ceramide and sphingolipids, which are known to be necessary for transport of lipid-anchored proteins to the plasma membrane [47,48].

SUR4 mutants, in addition to FEN1 mutants, were identified in several ways, implying many roles or effects of sphingolipids in cells. SUR4 mutants (and SUR1-3) were first identified as suppressors that enhance survival of *rsv167* and *rsv161* mutant strains during nutrient starvation [49]. The *sur4* allele uncovered in these experiments produced interesting phenotypes by itself, but because the mutation is uncharacterized, the phenotypes must be interpreted with caution.

A second group found SUR4 mutants during attempts to identify genes that regulate the plasma membrane proton pump encoded by *PMA1* [50]. A collection of mutants resistant to hygromycin B was screened for ones unable to grow at low pH. The low pH-sensitive mutants defined seven complementation groups including *APA1* (affecting plasma membrane ATPase activity) [50]. Experiments demonstrated that an *apa1* null mutant was sensitive to low pH, had less Pma1 protein, and 3–5-fold less *PMA1* mRNA. The mRNAs for several other plasma membrane proteins were not affected by the *apa1* mutation, but the mRNAs for two glucose transporters, Hxt3 and Snf3, were reduced. How a block in synthesis of C-26 fatty acids reduces the level of *PMA1* mRNA remains a most intriguing question.

A third group isolated strains resistant to the immunosuppressant drug SR 31747, which inhibits Δ8-Δ7 sterol isomerase activity [51]. The resistant strains were mutated in *SUR4* or *FEN1*. Strains deleted for either gene grew slower and were more drug resistant than the parent on YPD medium. Strains deleted for *ERG2*, which encodes the Δ8-Δ7 sterol isomerase, cannot make ergosterol and do not grow under aerobic conditions. However, they do grow if either *SUR4* or *FEN1* is deleted. It is not clear how a block in synthesis of C-26 fatty acids compensates for a block in ergosterol synthesis. Perhaps cells require a balance between sphingolipids and sterols and this balance is restored in *erg2* mutants by blocking synthesis of C-26 fatty acid and making a new spectrum of sphingolipids, as is known to happen in a *sur4* mutant [37]. Complexes between cholesterol and sphingolipids are hypothesized to exist as microdomains or rafts in animal cell membranes [52] and maybe the balance between sphingolipids and cholesterol affects such microdomains. Microdomains have yet to be shown to exist in yeast.

Diploid cells defective in either *SUR4* or *FEN1* make few spores and the spores do not germinate. Thus, sporulation and germination require synthesis of C-26 fatty acids. It is not clear whether this requirement is for sphingolipids, for glycosylphosphatidylinositol anchors with a C-26 fatty acid [53], for an unidentified lipid with a C-26 fatty acid thought

to be in the nuclear membrane [54], or some combination of these possibilities.

SUR4 and FEN1-deleted strains, either haploid or diploid, are defective in selecting a budding site and select sites randomly [55]. This phenotype could be due to indirect effects on membrane-lipid composition or to a direct interaction between a protein involved in bud site selection and a lipid with a C-26 fatty acid.

Finally, mutations in SUR4 and FEN1 were isolated as suppressors of strains defective in SNC1 [36]. SNC1 encodes a vesicle-SNARE protein necessary for post-Golgi secretion and viability. These results imply that C-26 fatty acids, ceramides or sphingolipids are necessary for a normal rate of protein secretion as has been demonstrated using different experimental approaches [47, 48].

### 3.2. LCB1 and LCB2 (SCS1) – Serine palmitoyltransferase (SPT)

The first strains to be identified with defects in sphingolipid synthesis were mutated in LCB1 and LCB2, both of which are essential genes [31, 56]. The strains lack SPT activity [57], which catalyzes the first committed step in sphingolipid synthesis. Strains defective in either gene grow and make sphingolipids when fed a long-chain base. When the long-chain base is removed, DNA, protein, and glycerophospholipid synthesis continue, but viability declines [56].

The LCB1 [58] and LCB2 genes [59, 60] have been isolated. LCB2 is thought to encode a catalytic component of SPT because of amino acid similarity to enzymes that catalyze a similar chemical reaction [59, 60]. The Lcb1 protein homolog in Chinese hamster ovary cells is physically associated with SPT activity [61, 62] and may be a catalytic or regulatory component of the enzyme. Since SPT has not been purified from any organism, it is unclear if Lcb1p and Lcb2p are the only genes necessary for enzyme activity. Surprisingly there appear to be two LCB2 homologs in Caenorhabditis elegans and only one LCB1 homolog [63, 64]. The presence of two LCB2 homologs suggests that multicellular eukaryotes require specialized SPT enzymes. However, various types of DNA sequence data from other multicellular eukaryotes do not reveal multiple LCB2 homologs,

but this picture may change when complete genome sequences become available.

#### 3.3. TSC10 – 3-Ketodihydrosphingosine reductase

TSC10 has been shown to encode the enzyme, 3-ketodihydrosphingosine reductase, necessary for the second step in sphingolipid synthesis (Fig. 2) [65]. TSC10 is essential for viability, but tsc10-defective cells will grow if fed DHS or PHS. Like Lcb1p and Lcb2p, Tsc10p is a membrane-bound enzyme having a single predicted membrane-spanning domain.

#### 3.4. SUR2 – 4-Hydroxylation of DHS and ceramide-1 (dihydroceramide)

SUR2 is required for addition of a hydroxyl at C-4 of either DHS or ceramide-1 (Fig. 2). The gene is not essential for growth and a null mutant strain shows no defect in vegetative growth or stress resistance [23, 24]. It is hard to imagine, however, that C-4 hydroxylation is inconsequential, since nearly all sphingolipids in wild type *S. cerevisiae* cells are hydroxylated [20]. The hydroxyl could hydrogen bond with other lipids in the plane of the membrane or with membrane-embedded proteins. There are three ‘clues’ to the function of the C-4 hydroxyl group, but these could be difficult clues to follow because of complex phenotypes that may involve indirect effects of sphingolipids without C-4 hydroxylation.

The first clue came from studies of rvs161-defective strains. Such strains have multiple defects and, for instance, fail to endocytose [66], localize actin [45], survive starvation, utilize non-fermentable carbon sources, make a normal complement of glycerophospholipids [67], or fuse with other cells [68]. Death of rvs161-defective cells during starvation is rescued by mutations in SUR1 (CSGI), SUR2, SUR3 and SUR4 (ELO3) [49]. Since three of these genes, SUR1, SUR2, and SUR4, are involved in different, but non-essential, steps in sphingolipid synthesis, it may be that the Rvs161 protein normally requires hydroxylated sphingolipids for function. The gene corresponding to the SUR3 suppressor is unknown.

The second clue is that inactivating mutations in SUR2 relieve the growth inhibiting effect of a high concentration of calcium ions on strains defective in csg2 [23], a gene required for mannosylation of IPC

[60]. The function of the Csg2 protein is unknown. The third clue is that strains defective in SUR2 are resistant to the growth inhibition by syringomycin E. Possible explanations for this result have been presented [24].

### 3.5. SCS7 – Hydroxylation of the fatty acid component of ceramides

SCS7 is not an essential gene and the only known effect of deleting it in a wild type background is the synthesis of ceramides with a non-hydroxylated C-26 fatty acid [23, 27, 28]. The Scs7 protein, like the Sur2 protein, contains a cytochrome b₅ domain, predicted transmembrane domains, and a HX₂₋₃(XH) motif that is characteristic of membrane-bound fatty acid desaturases. It is not clear whether this enzyme or an unidentified enzyme adds the second hydroxyl group that is often found in the C-26 fatty acid of sphingolipids [20]. Mutations in SCS7, like those in SUR2, relieve the growth inhibiting effect of a high concentration of calcium ions on strains defective in csg2 [23, 27]. This phenotype might arise from altered interactions between other membrane lipids, membrane proteins, or both which would normally hydrogen bond with the hydroxyls on the C-26 fatty acid in sphingolipids.

### 3.6. AUR1 – IPC synthase

AUR1 is an essential gene, first identified because mutations in it give resistance to aureobasidin A [69, 70]. A drug-resistant AUR1 allele is commercially available as a selectable marker gene for *S. cerevisiae* transformation (PanVera, Madison, WI, USA). The Aur1 protein contains several predicted membrane-spanning domains and is most likely IPC synthase or a subunit of the enzyme. Aureobasidin A is a potent inhibitor of IPC synthase activity, but the mode of inhibition is unknown [32]. The Aur1 and Ipt1 proteins are related (the genes are paralogs), which is expected since the enzymes catalyze similar reactions in which inositol phosphate is transferred from phosphatidylinositol to ceramide or MIPC, respectively. Similarities between these proteins have been discussed [4].

### 3.7. SUR1(CSG1) and CSG2(CLS2) – Addition of mannose to IPC to form MIPC

The SUR1 and CSG2 genes are not essential for vegetative growth [71], indicating that *S. cerevisiae* cells can grow under laboratory conditions without making its two terminal sphingolipids, MIPC or M(IP)₂C (Fig. 2). Strains deleted for either gene are inhibited for growth by 100 mM calcium. This phenotype has been very helpful in identifying genes necessary for synthesis of IPC [60] (see below). The Sur1 protein is similar over a stretch of 93 amino acids [71] to the Och1 α-1,6-mannosyltransferase [72] and, thus, seems likely to be the enzyme responsible for adding mannose to IPC. As mentioned above, SUR1 was first identified by mutations that suppress various phenotypes of strains mutated in RVS161.

The Csg2 protein is predicted to have nine or ten transmembrane domains, to regulate the calcium level in a pool that is distinct from the vacuolar pool, and to have an EF-hand for calcium binding [35, 73]. It is thought to play a role in regulating the concentration of calcium ions in the endoplasmic reticulum [74] where the protein has been localized [35]. However, the actual function or functions of this protein are unknown.

### 3.8. IPT1 (SYR4) – Addition of inositol phosphate to MIPC to form M(IP)₂C

The IPT1 gene is necessary for synthesis of M(IP)₂C and most likely encodes the enzyme phosphatidylinositol:mannose-inositol-P-ceramide phosphoinositol transferase [75]. The IPT1 gene is not essential for vegetative growth and cells deleted for it show no severe phenotype. Mutant cells do grow slightly better in the presence of a high concentration of calcium than do wild type cells [75]. An uncharacterized mutant strain that fails to make M(IP)₂C [76] has increased resistance to the polyene antibiotic nystatin. Nystatin is thought to complex with sterols such as ergosterol and form pores in the plasma membrane [77]. Leber et al. [76] suggest that M(IP)₂C is also part of such complexes, a suggestion that is consistent with the notion that sphingolipids

and sterols exist as microdomains in some membranes [52].

### 3.9. LCB4 and LCB5 – Addition of phosphate to long-chain bases

The LCB4 and LCB5 genes encode lipid kinases that add phosphate to DHS and PHS to form DHS-1-P and PHS-1-P, respectively [78]. Deletion of either or both genes causes no change in vegetative growth under a variety of culture conditions. It is not clear why there are two similar genes, but most likely they perform specialized functions. Lcb4p represents more than 95% of total cellular kinase activity, with the remainder represented by Lcb5p [78].

### 3.10. LCB3 (LBP1/YSR2) and YSR3 (LBP2) – DHS-1-P and PHS-1-P phosphatases

LCB3 and YSR3 encode phosphatases capable of dephosphorylating phosphorylated long-chain bases [42, 43]. Their preferred substrate, DHS-1-P or PHS-1-P, has not been determined. Lcb3p and Ysr3p probably perform different physiological functions, although these have not been delineated. What is unique about them is that Lcb3p, but not Ysr3p, is necessary for incorporation of exogenous DHS or PHS into sphingolipids [41–43]. This difference is probably not of physiological importance since wild type yeast must make their own long-chain bases. More likely, this difference indicates that under some unidentified physiological conditions Lcb3p preferentially dephosphorylates one or both long-chain base phosphates. Another phenotypic difference is that an LCB3-deletion mutant, but not a YSR3-deletion mutant, is supersensitive to growth inhibition by australifungin, an inhibitor of ceramide synthase [43]. This difference suggests that in wild type cells the Lcb3p phosphatase plays a role in regulating the amount of DHS, PHS or both that is available for ceramide synthesis and glycerolipid synthesis [43]. However, neither the molar concentration of DHS-1-P and PHS-1-P nor the rate of flux through either compound has been measured. So it is possible that the observed phenotypes are due to indirect effects. The role of Lcb3p and Lbp2 in heat stress will be discussed below. Finally, each protein is predicted to have a glycosylphosphatidylinositol

cleavage/attachment site at its C-terminus [41], which, if proven to be true, could have important functional implications.

### 3.11. DPL1 – Long-chain base-1-phosphate lyase

The DPL1 gene encodes the lyase that cleaves long-chain base phosphates, DHS-1-P and PHS-1-P, to yield C-16 aldehydes and ethanolamine-P [79]. The ethanolamine-P is a substrate for synthesis of aminoglycerophospholipids [80], while the fate of the C-16 aldehydes is probably to enter the palmitate pool. The gene is not essential for vegetative growth, but a deletion mutant is more sensitive than the wild type to growth inhibition by D-erythro-sphingosine and by PHS [79]. The role of this gene in heat stress will be discussed below. The Dpl1 protein has a single predicted transmembrane domain but the cellular location of the protein has not been determined. Its location will be important in understanding the functions of long-chain base phosphates and the molecular means for mediating the functions.

## 4. Roles for sphingolipids in heat stress

The primary experimental data supporting the hypothesis that sphingolipids play roles in heat and other stress responses come from studies of mutant strains that can grow without making sphingolipids [81]. One strain of this class, 4R3, is able to grow because of a suppressor mutation in the SLC1 gene [82] which enables cells to make a set of novel glycerophospholipids that mimic sphingolipid structures [83] and probably some of their functions. The novel lipids are phosphatidylinositol (PI), mannosyl-PI, and inositol-P-(mannosyl-PI), all containing a C-26 fatty acid in the sn-2 position of the glycerol moiety. When making these novel lipids instead of sphingolipids, strain 4R3 cannot grow at 37°C, at pH 4, or in the presence of high osmotic pressure. When fed a long-chain base the strain makes normal sphingolipids instead of suppressor lipids and is able to grow under these stress conditions just like wild type *S. cerevisiae* cells [56]. In another suppressor strain, strain 7R4 [56], it has been demonstrated that reduced heat stress resistance is not due to the suppressor mutation [22]. The most straightforward interpre-

tatation of these results is that one or more sphingo-
lipids is necessary for a normal stress response. How-
ever, the possibility that one or more of the novel
lipids reduce stress resistance cannot be eliminated at
this time.

The role of sphingolipids in defending cells against
heat has been examined in greatest detail and we
briefly summarize previously reviewed data before
discussing new data. Switching wild type cells from
23–25°C to 37–39°C produces a 2–3-fold transient
increases in the concentration of C-18-DHS and
C-18-PHS and a more than 100-fold increase in
C-20-DHS and C-20-PHS, peaking between 5 and
10 min after the temperature shift [21, 22]. DHS-1-P
increases 5-fold and PHS-1-P increases 8-fold, with
both peaking after about 10 min and then decreasing
back to near the basal level [84]. The carbon chain
length of the DHS-1-P and PHS-1-P is not known,
but it may depend upon the long-chain base sub-
strates available after the heat stress begins. These
transient changes in concentration are consistent
with one or more of these compounds being a signal-
ing molecule. Changes in the concentration of the
intermediate 3-ketodihydrosphingosine have not
been examined.

Total ceramides increase about 3-fold 20 min after
a heat stress but remain elevated [21, 22, 39]. The in-
crease in ceramide, the only intermediate analyzed,
was not observed during osmotic or low pH stress
[39], indicating specificity for heat stress. It is not
clear from these results whether one or more species
of ceramide is acting as a signaling molecule or
whether the increased level of ceramides is an adjust-
ment necessary for growth at higher temperature.
These changes in concentration are thought to oc-
cur by increased de novo synthesis rather than by
increased breakdown of ceramide or complex sphin-
golipids. The data supporting this hypothesis are
strongest for ceramide where several types of exper-
imental evidence are available [39]. Because the com-
plex sphingolipids are so abundant in S. cerevisiae,
breakdown of a small percentage, such as 1%, would
be difficult to detect and prove experimentally. The
argument for de novo synthesis being responsible for
the transient increases in DHS and DHS-1-P is based
upon the fact that DHS is not found in complex
sphingolipids and, thus, cannot be derived from
them, unless by an unknown pathway. There is no

direct evidence to explain the origin of the transient
increase of PHS and PHS-1-P and one or both could
be derived from breakdown of ceramide or complex
sphingolipids.

Like all known cells, *S. cerevisiae* cells respond to
heat stress by inducing a set of proteins, the heat
shock protein [85]. Sphingolipids do not seem to
play a role in signaling synthesis of the heat shock
proteins because cells lacking sphingolipids induce
the major heat shock proteins as well as do cells
containing sphingolipids [21]. Sphingolipids could
play a role in signaling synthesis of the less abundant
heat shock proteins.

Another hallmark and essential element in heat
stress resistance in *S. cerevisiae* is the transient accu-
mulation of trehalose (α-D-glucopyranosyl-α-D-glu-
copyranoside, [86–89] and references therein). Sphin-
golipids are necessary for trehalose accumulation
because the 4R3 mutant strain fails to accumulate
trehalose following transfer from 25°C to 37°C
when cells lack sphingolipids [21]. Treating wild
type cells with DHS also causes trehalose to accumu-
late, suggesting that DHS or DHS-1-P is acting as a
signal to regulate trehalose accumulation. The mech-
anism for regulating trehalose accumulation is not
well understood [88] and it is not clear how DHS
or DHS-1-P might be mediating accumulation.

One possible downstream target for signaling by
DHS or DHS-1-P is induction of *TPS2* transcription.
This gene encodes a subunit of trehalose synthase
and its transcription is induced by heat stress [90–
92]. Treating cells with DHS also induces transcrip-
tion of *TPS2*, but the kinetics of induction are slower
(180 min to the peak) than for heat stress (peaks in
less than 30 min) [21]. Induction of *TPS2* transcrip-
tion by stresses is mediated by a promoter element
termed STRE (stress response element) [93–95]. The
STRE elements in the *TPS2* promoter are bound by
Msn2 and Msn4 transcription activator proteins
[96, 97] and it has been shown that *MSN2* and
*MSN4* are required for induction of *TPS2* expression
by DHS (Skrzypek et al., unpublished results). The
cellular location of Msn2p and Msn4p is regulated
by protein kinase A (PKA). PKA causes Msn2p and
Msn4p to translocate from the nucleus to the cyto-
plasm when cells are not heat stressed. During heat
stress the proteins translocate to the nucleus, bind
STRE elements and activate transcription [98]. Tak-

en together, these results suggest that DHS or DHS-1-P play a role in regulating PKA or some other step in the translocation process.

Identification of the genes for making and degrading DHS-1-P and PHS-1-P (Fig. 2) makes it possible to determine the physiological role of these sphingolipid metabolites by studying the behavior of mutant strains with altered levels of these compounds. None of the metabolic genes, *DPL1*, *LCB3*, *LCB4*, *LCB5*, *YSR3*, is essential, so that strains with null mutations can be studied.

The first such mutant study examined the role of the two LCB-1-P phosphatase genes, *LCB3* (*YSR2/LBP1*) and *YSR3* (*LBP2*) [43]. In the W303 strain background, a mutant lacking *LCB3* survived 2-fold better than the parent when cells were grown in defined medium and heat stressed at 50°C. A *ysr3* deletion mutant behaved like the parent and a *lcb3 ysr3* double mutant strain was slightly more (1.5 times) heat resistant than the parent. In complex medium (YPD) and with early stationary phase cells, the *lcb3* mutant was 5 times, the *ysr3* mutant was 40 times, and the double mutant was 50 times more resistant to killing at 45°C than was the parent. These results suggest that DHS-1-P, PHS-1-P or both are playing a role in heat stress resistance. However, more work needs to be done to verify this possibility. For example, the amount of DHS-1-P and PHS-1-P needs to be quantified at the same time as the cells are heat stressed in order to establish a correlation between the level of each long-chain base phosphate and resistance to heat stress. Also, the strains need to have the same nutrient requirements because differences could contribute to the heat stress resistance. Finally, it needs to be shown genetically that the heat resistance phenotype is due to the claimed mutant gene. Thus, the heat resistance phenotype must be shown to co-segregate with the mutated *lcb3* or *ysr3* gene.

A second study used a different strain background, JK9-3d [99]. Cells were grown to early stationary phase in YPD medium and heat stressed. *DPL1*-deleted cells were found to be 80-fold more resistant to killing than the parental cells after 30 min at 50°C. Since *DPL1*-deleted cells accumulate long-chain base phosphates [79], it appears that these compounds play a role in resistance to heat. Further data to support this conclusion were obtained using a mutant strain, derived from the *DPL1*-deleted strain,

that lacks long-chain base kinase activity. If long-chain base phosphates are playing a role in the increased heat resistance of the *DPL1*-deleted strain, then the kinase mutant should have decreased heat resistance. Indeed, the kinase-minus cells had reduced resistance to heat and behaved like the non-mutant parental cells. While these results support the hypothesis that DHS-1-P, PHS-1-P or both play a role in heat stress resistance, they should be interpreted with caution for several reasons. Most important is the observation that the double mutant strain (*dpl1* kinase−) had the same level of heat resistance as the non-mutant parental strain. If long-chain base phosphates were necessary for wild type cells to resist heat, the kinase-minus cells would be expected to have less resistance to heat than the wild type, not the same level of resistance. Thus, what these data suggest is that an abnormally high level of long-chain base phosphates induces a state of hyper-resistance to heat stress and that in wild type cells the long-chain base phosphates are not necessary for a normal level of heat stress resistance. Another problem with these experiments is that neither the type nor the level of long-chain base phosphates was quantified. In addition, the kinase-minus mutant is genetically uncharacterized and neither it nor the *dpl1* mutant have been shown by genetic analysis to give co-segregation of the heat stress phenotype and the mutant gene. Such an analysis would eliminate the possibility that the observed heat stress phenotype is due to an uncharacterized mutation.

Other studies in the JK9-3d strain background found that deletion of either *dpl1* or *lcb3* caused a 2–4-fold increase in resistance to heat in log phase cells [84]. The basal level of PHS-1-P was increased 24-fold in the *dpl1* mutant, but for unknown reasons DHS-1-P was not detectable. The *lcb3* mutant had increased amounts of both compounds. Thus, increased PHS-1-P correlates with increased heat resistance. In these same studies a *dpl1 lcb3* double mutant grew slowly, had a 10-fold increase in heat resistance, and a 500-fold increase in both PHS-1-P and DHS-1-P. Subsequent analysis of this strain revealed the presence of an unidentified suppressor gene that overcomes the growth inhibitory effect of deleting both *lcb3* and *dpl1* (Skrzypek et al., unpublished results). The increased heat resistance of this strain is most likely due to slow growth and any correlation

between heat resistance and long-chain base phosphates should be interpreted with caution. It is not known why deletion of both genes inhibits growth, but the large accumulation of long-chain base phosphates in the suppressor strain suggests that the accumulation inhibits one or more essential process in cells lacking the suppressor gene. The suppressor strain demonstrates for the first time that there is a flux through this pathway that diverts long-chain bases to glycerolipids.

Recently two genes, *LCB4* and *LCB5*, have been shown to be responsible for all measurable long-chain base kinase activity present in *S. cerevisiae* cells [78]. Strains lacking both genes contain no detectable DHS-1-P or PHS-1-P [78] (Skrzypek et al., unpublished results). Independent isolates of *lcb4 lcb5* double mutants, made in the JK9-3d strain background, were found to be only 2–3-fold less resistant to heat stress than wild type cells growing in log phase (Skrzypek et al., unpublished results). These results are consistent with the hypothesis that DHS-1-P and PHS-1-P are involved in resisting heat stress. However, the compounds seem to be playing minor roles in heat stress resistance, which may be due to redundancy with some other stress resistance mechanism, as often occurs in yeast.

### 5. Regulation of sphingolipid synthesis

The mechanisms for controlling the concentration of sphingolipids in *S. cerevisiae* cells are unknown. This lack of knowledge is in contrast to our extensive understanding of the mechanisms for regulating sterol [5,100] and glycerophospholipid levels [5,101–103]. For example, transcription of several genes for glycerophospholipid metabolism is regulated by a transcription activator complex composed of Ino2p and Ino4p [101]. A similar system for regulating transcription of sphingolipid metabolic genes has not been identified and there is no indication that one exists. IPC synthase activity has been shown to be highest in log phase cells and to decrease to a very low level in stationary phase cells [104] as is the case for many yeast enzymes and proteins [105].

Insight and clues into the mechanisms for regulating sphingolipid levels have come from studies of mutants that suppress the calcium sensitivity of a

strain defective in *csg1* (*sur1*) or *csg2* [60,65]. Additional information has come from analysis of gene expression by DNA microarray technology during the diauxic shift [106], and the cell cycle [107]. Analysis of transcript levels during sporulation by DNA microarray technology [108] has uncovered changes in the level of some sphingolipid metabolic genes, but the meaning of these changes is unclear, particularly since the sphingolipids in spores and cells undergoing sporulation are only partially characterized [109].

As implied by their gene acronyms, *csg* (calcium sensitive growth [73]) or *cls* (calcium sensitive [110]) mutants fail to grow in the presence of a high concentration (100 mM) of Ca²⁺. The basis for sensitivity is unknown, but is thought to have something to do with a calcium pool in the endoplasmic reticulum [73,74]. Suppressors of the calcium sensitive phenotype have been isolated in two ways by Beeler and Dunn. First, second-site suppressors of a *csg2* null mutant were isolated for growth in the presence of 100 mM Ca²⁺. These so-called SCS (suppressor of calcium-sensitive growth) mutants fell into seven complementation groups, *SCSI-SCS7* [60]. A key step in beginning to understand the function of *CSG2* was the identification of *LCB2* (*SCSI*) as one of the suppressor genes. Since *LCB2* was known to be necessary for serine palmitoyltransferase activity [59], its role as a suppressor of calcium sensitivity suggested that the *csg2* mutant strain had a defect in sphingolipid synthesis. Analysis of sphingolipids revealed that a *csg2* null mutant made one sphingolipid species, IPC-3 (IPC-C, it has ceramide composed of PHS and a C-26 fatty acid with an α-OH group). All seven SCS suppressor mutants were defective to varying degrees in sphingolipid synthesis, but all of them had gained the ability to make some of the other species of IPC and other sphingolipids (MIPC or M(IP)₂C) in addition to IPC-3 [60]. These results suggested that in the presence of high calcium, IPC-3 was toxic. The effect of suppressor mutations was to reduce the level of this toxic lipid either by reducing its rate of synthesis or by allowing it to be converted to other species of sphingolipids.

The second method for selecting suppressors of the calcium-sensitive phenotype of a *csg2*-null mutant was designed to identify mutations in sphingolipid metabolic genes, including ones essential for cell viability. In this method, suppressors of calcium sensi-

R.C. Dickson, R.L. Lester / Biochimica et Biophysica Acta 1438 (1999) 305–321

tivity were selected at 26°C and screened for ones unable to grow at 37°C. Fifty-nine temperature-sensitive suppressor mutants fell into 15 complementation groups, called TSC, for temperature-sensitive suppressors of csg2 mutants [65]. This approach appears to identify genes required for IPC synthesis since the two complementation groups with the most mutant strains correspond to LCB1 (TSC2) and LCB2 (TSCI). Since several of the 15 complementation groups contain only one mutant, there are likely to be additional, unidentified complementation groups.

The TSC suppressor alleles of LCB1 and LCB2 are predicted to reduce the rate of flux through the biosynthetic pathway and thereby reduce the rate of IPC synthesis. This prediction awaits experimental verification. Isolation of TSCI and TSC2 suppressor alleles suggests that wild type cells might control their rate of sphingolipid synthesis by modulating the amount or the activity of SPT. As noted below, transcription of LCB1 or LCB2 changes very little either during the cell cycle or during the diauxic shift, so this form of regulation does not seem to be important for these two genes. Assays with LCB1 or LCB2-LacZ reporter genes also indicate that exogenous PHS or N-acetyldihydrophosphingosine (C2-ceramide) does not regulate transcription of the genes by feedback inhibition (unpublished results). Changes in the amount of the Lcb1 and Lcb2 proteins or in SPT activity during the cell cycle have not been examined, and it remains to be determined if cells use SPT as a target for regulating synthesis of sphingolipids.

Temperature-sensitive mutations in LCB1 have also been identified in a search for strains defective in endocytosis. One such mutant strain containing the lcb1-100 (end8-1) allele is blocked in transport of Gas1p, a glycosylphosphatidylinositol-anchored protein, from the endoplasmic reticulum to the Golgi apparatus [111]. It is believed that Gas1p requires ceramide or a related sphingolipid for transport [47,48], so the effect of the lcb1-100 mutation is to reduce ceramide synthesis at the restrictive temperature [111].

TSC3 and TSC4 correspond to FAS2, which encodes the α subunit of the fatty acid synthase. Presumably these suppressors lower the steady-state level of IPC-3 by reducing the amount of palmitoyl-CoA enough so that the rate of IPC synthesis is decreased (Fig. 2). The TSC7 suppressors are in the SUR2 gene and presumably prevent or reduce hydroxylation of DHS. Such suppressor cells are predicted to preferentially make IPC, MIPC and M(IP)2C species with DHS rather than PHS based upon the known function of Sur2p [23,24]. The TSC10 suppressor mutation uncovered a new gene which was shown to encode 3-ketodihydrosphingosine reductase [65]. The way in which the suppressor mutation in this gene acts to overcome the toxic effect of IPC-3 is unknown, but the favored hypothesis is by reducing the rate of ceramide synthesis just as is predicted for the TSC mutations in FAS2, LCB1 and LCB2 ceramide. However, more indirect effects are possible and need to be ruled out.

Suppressors in four known genes are surprising and may offer new clues to regulation of sphingolipid synthesis in S. cerevisiae. The TSC8 and TSC9 suppressors are alleles of SIN3 and RPD3, respectively. Sin3p is a transcriptional regulator with both positive and negative effects on a wide range of genes [112] including ones involved in glycerophospholipid metabolism [113]. It is found in a large protein complex also containing Rpd3p [114], the first histone deacetylase to be identified in yeast [115]. The Sin3p and Rpd3 complex could be acting directly to regulate transcription of one or more sphingolipid biosynthetic genes and this hypothesis would explain why the TSC8 and TSC9 suppressor alleles were isolated. Alternatively, the effect of the suppressors could be indirect since SIN3 and RPD3 have pleiotropic actions. For example, the suppressors could perturb membrane-dependent enzymes including ones for sphingolipid synthesis by effecting glycerophospholipid synthesis.

The other two surprising TSC suppressor genes, TSC14 and TSC15, correspond to TOR2 and MSS4, respectively, both encoding essential phosphatidylinositol kinases. Tor2p is a vacuolar phosphatidylinositol-4 kinase [116] which regulates numerous cellular processes including G1 cell cycle arrest and nutrient sensing [117] and induction of autophagy [118]. Mss4p is a phosphatidylinositol-4-phosphate 5-kinase required for organization of the actin cytoskeleton [119,120]. It should be informative to determine how the TSC suppressor mutation affects each protein and how these in turn influence IPC-3 synthesis.

Two other suppressors, *TSC11 (YER093c)* and *TSC13 (YDL015c)*, encode proteins of unknown function. The Yer093 protein is related to a mammalian synaptic glycoprotein with little known about its function. Mutants defective in *TSC13* do not grow in the presence of 6 mM caffeine and have a reduced growth rate on rich medium [121]. Further studies of these genes could give important clues to mechanisms for regulating IPC-3 metabolism.

Examination of transcript levels during the diauxic shift by DNA microarray technology indicates very little change (less than 2-fold) in transcripts encoding most of the sphingolipid metabolic genes [106]. The exceptions are *SUR4* and *FEN1* (Fig. 2) whose transcript level decreases 4.3- and 3.1-fold, respectively, late in the shift, and *LCB2*, whose transcript decreases 2.5-fold. The significance, if any, of these changes is unknown.

Transcript levels during the cell cycle have also been examined by DNA microarray technology [107] and 19 genes involved in lipid and membrane synthesis were categorized as cell cycle-regulated. Five of these are specific to sphingolipid metabolism. Transcripts for *SUR1*, *SUR2* and *SUR4* reach a peak during the G₁ phase of the cell cycle as do transcripts for a very large number of cell cycle-regulated genes. Transcripts for *AUR1* and *LCB3* peak during the G₂ phase of the cell cycle. The first step in deciding if these changes have any functional significance will be to determine if the activity of these enzymes varies during the cell cycle.

### 6. Summary and future developments

The number of genes involved in sphingolipid metabolism has increased rapidly in the past 2 years and more are sure to be identified in the near future. These genes (Fig. 2) will continue to be basic elements in efforts to understand the cellular functions of sphingolipids. One surprising result from studies of these genes is that some null mutants have no observable phenotype under laboratory conditions. This result may mean that the missing sphingolipids or metabolites have a function that is redundant with some other metabolite or that the correct conditions for observing a phenotype have not been found. For example, sporulation, haploid invasive and diploid

filamentous growth have not been examined extensively using sphingolipid mutants nor has the full range of stress responses been tested. It may be that some sphingolipid modifications, such as hydroxylation or mannosylation, produce only small survival advantages and can only be uncovered by long-term growth experiments where mutant and non-mutant cells are co-cultivated [122].

Besides deleting, mutating or overexpressing genes, sphingolipid metabolism can be manipulated using chemical inhibitors (reviewed in [4, 123]) and these should be useful for a variety of experiments with wild type and mutant cells.

New techniques such as DNA microarray technology may be useful in helping to identify sphingolipid functions. For example, changes in transcript levels during the cell cycle, the diauxic shift, sporulation, and temperature stresses have been analyzed to identify clusters of genes whose transcript levels behave most similarly [124]. None of the sphingolipid metabolic genes fall into any of the clusters with an obvious function such as protein synthesis, glycolysis and protein degradation. However, *LCB3*, encoding one of the two long-chain base-1-P phosphatases, is clustered with a group of seven genes necessary for secretion, glycosylation, and lipid anchoring of proteins that pass through the protein secretory pathway. This result suggests that Lcb3p has something to do with protein secretion or a process dependent upon the secretory pathway.

The phenotypes of several mutant strains support the hypothesis that DHS-1-P and PHS-1-P play roles in heat stress resistance in both log and stationary phase cells. Future studies need to identify the processes regulated by long-chain base phosphates and their mechanism of regulation.

Other challenges are to purify and biochemically characterize the enzymes, determine their cellular location, determine if their activity is regulated and if so, to understand the regulatory mechanism(s). Finally, we need to understand how sphingolipid metabolism is integrated with other cellular metabolism.

### Acknowledgements

Our research has been supported by Grant GM43102 form the US Public Health Service.

greatly appreciate the effort of our laboratory colleagues, particularly Gerald Wells, Dr. Marek Nagiec and Dr. Marek Skrzypek, who have helped to advance our understanding of sphingolipids in *S. cerevisiae*.

References

[1] J.L. Patton, R.L. Lester, J. Bacteriol. 173 (1991) 3101–3108.
[2] P. Hechtberger, E. Zinser, R. Saf, K. Hummel, F. Paltauf, G. Daum, Eur. J. Biochem. 225 (1994) 641–649.
[3] R.C. Dickson, Annu. Rev. Biochem. 67 (1998) 27–48.
[4] R.C. Dickson, R.L. Lester, Biochim. Biophys. Acta 1426 (1999) 347–357.
[5] G. Daum, N.D. Lees, M. Bard, R. Dickson, Yeast 14 (1998) 1471–1510.
[6] P. Orlean, in: Yeast III, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1997, pp. 229–362.
[7] M.-Y. Kim, C. Linardic, L. Obied, Y.A. Hannun, J. Biol. Chem. 266 (1991) 484–489.
[8] S. Mathias, K.A. Dressler, R.N. Kolesnick, J. Biol. Chem. 88 (1991) 10009–100013.
[9] R.N. Kolesnick, M. Kronke, Annu. Rev. Physiol. 60 (1998) 643–665.
[10] L. Riboni, P. Viani, R. Bassi, A. Prinetti, G. Tettamanti, Prog. Lipid Res. 36 (1997) 153–195.
[11] Y.A. Hannun, Science 274 (1996) 1855–1859.
[12] J.M. Kyriakis, J. Avruch, J. Biol. Chem. 271 (1996) 24313–24316.
[13] L.R. Ballou, S.J.F. Laulederkind, E.F. Rosloniec, R. Raghow, Biochim. Biophys. Acta Lipids Lipid Metab. 1301 (1996) 273–287.
[14] E.A. Martinova, Biochemistry (Mosc.) 63 (1998) 102–110.
[15] S. Hakomori, Cancer Res. 56 (1996) 5309–5318.
[16] S. Spiegel, D. Foster, R.N. Kolesnick, Curr. Opin. Cell Biol. 8 (1996) 159–167.
[17] S. Spiegel, A.H. Merrill Jr., FASEB J. 10 (1996) 1388–1397.
[18] D.M. zu Heringdorf, C.J. van Koppen, K.H. Jakobs, FEBS Lett. 410 (1997) 34–38.
[19] T. Seufferlein, E. Rozengurt, J. Biol. Chem. 270 (1995) 24343–24351.
[20] R.L. Lester, R.C. Dickson, Adv. Lipid Res. 26 (1993) 253–272.
[21] R.C. Dickson, E.E. Nagiec, M. Skrzypek, P. Tillman, G.B. Wells, R.L. Lester, J. Biol. Chem. 272 (1997) 30196–30200.
[22] G.M. Jenkins, A. Richards, T. Wahl, C.G. Mao, L. Obeid, Y. Hannun, J. Biol. Chem. 272 (1997) 32566–32572.
[23] D. Haak, K. Gable, T. Beeler, T. Dunn, J. Biol. Chem. 252 (1997) 29704–29710.
[24] M.M. Grilley, S.D. Stock, R.C. Dickson, R.L. Lester, J.Y. Takemoto, J. Biol. Chem. 273 (1998) 11062–11068.
[25] A.H. Merrill Jr., E. Wang, J. Biol. Chem. 261 (1986) 3764–3769.
[26] J. Rother, G. van Echten, G. Schwarzmann, K. Sandhoff, Biochem. Biophys. Res. Commun. 189 (1992) 14–20.
[27] T.M. Dunn, D. Haak, E. Monaghan, T.J. Beeler, Yeast 14 (1998) 311–321.
[28] A.G. Mitchell, C.E. Martin, J. Biol. Chem. 272 (1997) 28281–28288.
[29] S.W. Smith, R.L. Lester, J. Biol. Chem. 249 (1974) 3395–3405.
[30] A.H. Merrill, C.C. Sweeley, in: D.E. Vance, J.E. Vance (Eds.), Biochemistry of Lipids, Lipoproteins and Membranes, Elsevier Science, New York, 1996, pp. 309–339.
[31] G.B. Wells, R.L. Lester, J. Biol. Chem. 258 (1983) 10200–10203.
[32] M.M. Nagiec, E.E. Nagiec, J.A. Baltisberger, G.B. Wells, R.L. Lester, R.C. Dickson, J. Biol. Chem. 272 (1997) 9809–9817.
[33] G. van Echten, K. Sandhoff, J. Biol. Chem. 268 (1993) 5341–5344.
[34] E.C. Mandon, I. Ehses, J. Rother, G. van Echten, K. Sandhoff, J. Biol. Chem. 267 (1992) 11144–11148.
[35] Y. Takita, Y. Ohya, Y. Anraku, Mol. Gen. Genet. 246 (1995) 269–281.
[36] D. David, S. Sundarababu, J.E. Gerst, J. Cell Biol. 143 (1998) 1167–1182.
[37] C.-S. Oh, D.A. Toke, S. Mandala, C.E. Martin, J. Biol. Chem. 272 (1997) 17376–17384.
[38] A. Puoti, C. Desponds, A. Conzelmann, J. Cell Biol. 113 (1991) 515–525.
[39] G.B. Wells, R.C. Dickson, R.L. Lester, J. Biol. Chem. 273 (1998) 7235–7243.
[40] K.M. Ella, C. Qi, J.W. Dolan, R.P. Thompson, K.E. Meier, Arch. Biochem. Biophys. 340 (1997) 101–110.
[41] L.X. Qie, M.M. Nagiec, J.A. Baltisberger, R.L. Lester, R.C. Dickson, J. Biol. Chem. 272 (1997) 16110–16117.
[42] C. Mao, M. Wadleigh, G.M. Jenkins, Y.A. Hannun, L.M. Obeid, J. Biol. Chem. 272 (1997) 28690–28694.
[43] S.M. Mandala, R. Thornton, Z. Tu, M.B. Kurtz, J. Nickels, J. Broach, R. Menzelev, S. Spiegel, Proc. Natl. Acad. Sci. USA 95 (1998) 150–155.
[44] H. Shimeno, S. Soeda, M. Sakamoto, T. Kouchi, T. Kowakame, T. Kihara, Lipids 33 (1998) 601–605.
[45] E. Revardel, M. Bonneau, P. Durrens, M. Aigle, Biochim. Biophys. Acta 1263 (1995) 261–265.
[46] M. el-Sherbeini, J.A. Clemas, J. Bacteriol. 177 (1995) 3227–3234.
[47] A. Horvath, C. Suetterlin, U. Manning-Krieg, N.R. Movva, H. Riezman, EMBO J. 13 (1994) 3687–3695.
[48] M. Skrzypek, R.L. Lester, R.C. Dickson, J. Bacteriol. 179 (1997) 1513–1520.
[49] L. Desfarges, P. Durrens, H. Juguelin, C. Cassagne, M. Bonneu, M. Aigle, Yeast 9 (1993) 267–277.
[50] M. Garcia-Arranz, A.M. Maldonado, M.J. Mazon, F. Portillo, J. Biol. Chem. 269 (1994) 18076–18082.
[51] S. Silve, P. Leplatois, A. Josse, P.H. Dupuy, C. Lanau, M.

Kaghad, C. Dhers, C. Picard, A. Rahier, M. Taton, G. Le Fur, D. Caput, P. Ferrara, G. Loison, Mol. Cell. Biol. 16 (1996) 2719–2727.

[52] T. Harder, P. Scheiffele, P. Verkade, K. Simons, J. Cell Biol. 141 (1998) 929–942.

[53] G. Sipos, F. Reggiori, C. Vionnet, A. Conzelmann, EMBO J. 16 (1997) 3494–3505.

[54] R. Schneiter, M. Hitomi, A.S. Ivessa, E.V. Fasch, S.D. Kohlwein, A.M. Tartakoff, Mol. Cell. Biol. 16 (1996) 7161–7172.

[55] J.E. Zahner, H.A. Harkins, J.R. Pringle, Mol. Cell. Biol. 16 (1996) 1857–1870.

[56] J.L. Patton, B. Srinivasan, R.C. Dickson, R.L. Lester, J. Bacteriol. 174 (1992) 7180–7184.

[57] W.J. Pinto, B. Srinivasan, S. Shepherd, A. Schmidt, R.C. Dickson, R.L. Lester, J. Bacteriol. 174 (1992) 2565–2574.

[58] R. Buede, C. Rinker-Schaffer, W.J. Pinto, R.L. Lester, R.C. Dickson, J. Bacteriol. 173 (1991) 4325–4332.

[59] M.M. Nagiec, J.A. Baltisberger, G.B. Wells, R.L. Lester, R.C. Dickson, Proc. Natl. Acad. Sci. USA 91 (1994) 7899–7902.

[60] C. Zhao, T. Beeler, T. Dunn, J. Biol. Chem. 269 (1994) 21480–21488.

[61] K. Hanada, T. Hara, M. Nishijima, O. Kuge, R.C. Dickson, M.M. Nagiec, J. Biol. Chem. 272 (1997) 32108–32114.

[62] K. Hanada, T. Hara, M. Fukasawa, A. Yamaji, M. Umeda, M. Nishijima, J. Biol. Chem. 273 (1998) 33787–33794.

[63] S.A. Chervitz, L. Aravind, G. Sherlock, C.A. Ball, E.V. Koonin, S.S. Dwight, M.A. Harris, K. Dolinski, S. Mohr, T. Smith, S. Weng, J.M. Cherry, D. Botstein, Science 282 (1998) 2022–2028.

[64] S.A. Chervitz, E.T. Hester, C.A. Ball, K. Dolinski, S.S. Dwight, M.A. Harris, G. Juvik, A. Malekian, S. Roberts, T. Roe, C. Scafe, M. Schroeder, G. Sherlock, S.W.Y. Zhu, J.M. Cherry, D. Botstein, Nucleic Acids Res. 27 (1999) 74–78.

[65] T. Beeler, D. Bacikova, K. Gable, L. Hopkins, C. Johnson, H. Slife, T. Dunn, J. Biol. Chem. 273 (1998) 30688–30694.

[66] A.L. Munn, B.J. Stevenson, M.I. Geli, H. Riezman, Mol. Biol. Cell 6 (1995) 1721–1742.

[67] M. Crouzet, M. Urdaci, L. Dulau, M. Aigle, Yeast 7 (1991) 727–743.

[68] V. Brizzio, A.E. Gammie, M.D. Rose, J. Cell Biol. 141 (1998) 567–584.

[69] S.A. Heidler, J.A. Radding, Antimicrob. Agents Chemother. 39 (1995) 2765–2769.

[70] T. Hashida-Okado, A. Ogawa, M. Endo, R. Yasumoto, K. Takesako, I. Kato, Mol. Gen. Genet. 251 (1996) 236–244.

[71] T.J. Beeler, D. Fu, J. Rivera, E. Monaghan, K. Gable, T.M. Dunn, Mol. Gen. Genet. 255 (1997) 570–579.

[72] K. Nakayama, T. Nagasu, Y. Shimma, J. Kuromitsu, Y. Jigami, EMBO J. 11 (1992) 2511–2519.

[73] T. Beeler, K. Gable, C. Zhao, T. Dunn, J. Biol. Chem. 269 (1994) 7279–7284.

[74] I. Tanida, Y. Takita, A. Hasegawa, Y. Ohya, Y. Anraku, FEBS Lett. 379 (1996) 38–42.

[75] R.C. Dickson, E.E. Nagiec, G.B. Wells, M.M. Nagiec, R.L. Lester, J. Biol. Chem. 272 (1997) 29620–29625.

[76] A. Leber, P. Fischer, R. Schneiter, S.D. Kohlwein, G. Daum, FEBS Lett. 411 (1997) 211–214.

[77] D. Kerridge, Adv. Microbiol. Physiol. 27 (1986) 1–72.

[78] M.M. Nagiec, M. Skrzypek, E.E. Nagiec, R.L. Lester, R.C. Dickson, J. Biol. Chem. 273 (1998) 19437–19442.

[79] D.J. Saba, F. Nara, A. Bielawska, S. Garrett, Y.A. Hannun, J. Biol. Chem. 272 (1997) 26087–26090.

[80] A.K. Percy, M.A. Carson, J.F. Moore, C.J. Waechter, Arch. Biochem. Biophys. 230 (1984) 69–81.

[81] R.C. Dickson, G.B. Wells, A. Schmidt, R.L. Lester, Mol. Cell. Biol. 10 (1990) 2176–2181.

[82] M.M. Nagiec, G.B. Wells, R.L. Lester, R.C. Dickson, J. Biol. Chem. 268 (1993) 22156–22163.

[83] R.L. Lester, G.B. Wells, G. Oxford, R.C. Dickson, J. Biol. Chem. 268 (1993) 845–856.

[84] M.S. Skrzypek, M.M. Nagiec, R.L. Lester, R.C. Dickson, J. Bacteriol. 181 (1999) 1134–1140.

[85] D.A. Parsell, S. Lindquist, in: R.I. Morimoto, A. Tissieres, C. Georgopoulos (Eds.), The Biology of Heat Shock Proteins and Molecular Chaperones, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1994, pp. 457–494.

[86] C. De Virgilio, T. Hottiger, J. Dominguez, T. Boller, A. Wiemken, Eur. J. Biochem. 219 (1994) 179–186.

[87] J.L. Parrou, M.A. Teste, J. Francois, Microbiology 143 (1997) 1891–1900.

[88] M.A. Singer, S. Lindquist, Trends Biotechnol. 16 (1998) 460–468.

[89] S. Nwaka, H. Holzer, Prog. Nucleic Acid Res. Mol. Biol. 58 (1998) 197–237.

[90] C. De Virgilio, N. Burckert, W. Bell, P. Jeno, T. Boller, A. Wiemken, Eur. J. Biochem. 212 (1993) 315–323.

[91] I.P. Sur, Z. Lobo, P.K. Maitra, Yeast 10 (1994) 199–209.

[92] N. Gounalaki, G. Thireos, EMBO J. 13 (1994) 4036–4041.

[93] G. Marchler, C. Schuller, G. Adam, H. Ruis, EMBO J. 12 (1993) 1997–2003.

[94] N. Kobayashi, K. McEntee, Mol. Cell. Biol. 13 (1993) 248–256.

[95] H. Ruis, C. Schuller, BioEssays 17 (1995) 959–965.

[96] M.T. Martinez-Pastor, G. Marchler, C. Schuller, A. Marchler-Bauer, H. Ruis, F. Estruch, EMBO J. 15 (1996) 2227–2235.

[97] A.P. Schmitt, K. McEntee, Proc. Natl. Acad. Sci. USA 93 (1996) 5777–5782.

[98] W. Gorner, E. Durchschlag, M.T. Martinez-Pastor, F. Estruch, G. Ammerer, B. Hamilton, H. Ruis, C. Schuller, Genes Dev. 12 (1998) 586–597.

[99] M.M. Lanterman, J.D. Saba, Biochem. J. 332 (1998) 525–531.

[100] L.W. Parks, S.J. Smith, J.H. Crowley, Lipids 30 (1995) 227–230.

[101] S.A. Henry, J.L. Patton-Vogt, Prog. Nucleic Acid Res. Mol. Biol. 61 (1998) 133–179.

[102] M.L. Greenberg, J.M. Lopes, Microbiol. Rev. 60 (1996) 1–20.

[103] G.M. Carman, G.M. Zeimetz, J. Biol. Chem. 271 (1996) 13293–13296.
[104] J. Ko, S. Cheah, A.S. Fischl, J. Bacteriol. 176 (1994) 5181–5183.
[105] M. Werner-Washburne, E.L. Braun, M.E. Crawford, V.M. Peck, Mol. Microbiol. 19 (1996) 1159–1166.
[106] J.L. DeRisi, V.R. Iyer, P.O. Brown, Science 278 (1997) 680–686.
[107] P.T. Spellman, G. Sherlock, M.Q. Zhang, V.R. Iyer, K. Anders, M.B. Eisen, P.O. Brown, D. Botstein, B. Futcher, Mol. Biol. Cell 9 (1998) 3273–3297.
[108] S. Chu, J. DeRisi, M. Eisen, J. Mulholland, D. Botstein, P.O. Brown, I. Herskowitz, Science 282 (1998) 699–705.
[109] R. Schroeder, M. Breitenbach, J. Bacteriol. 146 (1981) 775–783.
[110] Y. Ohya, Y. Ohsumi, Y. Anraku, J. Gen. Microbiol. 132 (1986) 979–988.
[111] C. Sutterlin, T.L. Doering, F. Schimmoeller, S. Schroeder, H. Riezman, J. Cell Sci. 110 (1997) 2703–2714.
[112] M. Vidal, R. Strich, R.E. Esposito, R.F. Gaber, Mol. Cell. Biol. 11 (1991) 6306–6316.
[113] K.A. Hudak, J.M. Lopes, S.A. Henry, Genetics 136 (1994) 475–483.

[114] M.M. Kasten, S. Dorland, D.J. Stillman, Mol. Cell. Biol. 17 (1997) 4852–4858.
[115] J. Taunton, C.A. Hassig, S.L. Schreiber, Science 272 (1996) 408–411.
[116] M.E. Cardenas, J. Heitman, EMBO J. 14 (1995) 5892–5907.
[117] M.N. Hall, Biochem. Soc. Trans. 24 (1996) 234–239.
[118] T. Noda, Y. Ohsumi, J. Biol. Chem. 273 (1998) 3963–3966.
[119] S. Desrivieres, F.T. Cooke, P.J. Parker, M.N. Hall, J. Biol. Chem. 273 (1998) 15787–15793.
[120] K. Homma, S. Terui, M. Minemura, H. Qadota, Y. Anraku, Y. Kanaho, Y. Ohya, J. Biol. Chem. 273 (1998) 15779–15786.
[121] V. Smith, K.N. Chou, D. Lashkari, D. Botstein, P.O. Brown, Science 274 (1996) 2069–2074.
[122] F. Baganz, A. Hayes, R. Farquhar, P.R. Butler, D.C.J. Gardner, S.G. Oliver, Yeast 14 (1998) 1417–1427.
[123] T. Kolter, K. Sandhoff, Chem. Soc. Rev. 25 (1996) 371–381.
[124] M.B. Eisen, P.T. Spellman, P.O. Brown, D. Botstein, Proc. Natl. Acad. Sci. USA 95 (1998) 14863–14868.
[125] F. Reggiori, A. Conzelmann, J. Biol. Chem. 273 (1998) 30550–30559.
