
JOURNAL OF BACTERIOLOGY, June 1999, p. 3816–3823

0021-9193/99/$04.00+0

Copyright © 1999, American Society for Microbiology. All Rights Reserved.

# Analysis of Quorum-Sensing-Dependent Control  
of Rhizosphere-Expressed (rhi) Genes in  
Rhizobium leguminosarum bv. viciae  

BELEN RODELAS,$^{1\ddagger}$ JAMES K. LITHGOW,$^{1}$ FLORENCE WISNIEWSKI-DYE,$^{1}$  
ANDREA HARDMAN,$^{2}$ ADAM WILKINSON,$^{1\sharp}$ ANASTASSIOS ECONOMOU,$^{1\S}$  
PAUL WILLIAMS,$^{2}$ AND J. ALLAN DOWNIE$^{1*}$  

John Innes Centre, Colney, Norwich NR4 7UH,$^{1}$ and School of Pharmaceutical Sciences,  
University of Nottingham, Nottingham, NG7 2RD,$^{2}$ United Kingdom  

Received 26 January 1999/Accepted 16 April 1999  

The *rhi* genes of *Rhizobium leguminosarum* biovar viciae are expressed in the rhizosphere and play a role in the interaction with legumes, such as the pea. Previously (K. M. Gray, J. P. Pearson, J. A. Downie, B. E. A. Boboye, and E. P. Greenberg, J. Bacteriol. 178:372–376, 1996) the *rhiABC* operon had been shown to be regulated by RhiR and to be induced by added *N*-(3-hydroxy-7-cis-tetradecenoyl)-*L*-homoserine lactone (*3OH,C*~14:1~-HSL). Mutagenesis of a cosmid carrying the *rhiABC* and *rhiR* gene region identified a gene (*rhiL*) that affects the level of *rhiA* expression. Mutation of *rhiL* slightly increased the number of nodules formed on the pea. The *rhiL* gene is (like *rhiA*) regulated by *rhiR* in a cell density-dependent manner. RhiL is similar to LuxI and other proteins involved in the synthesis of *N*-acyl-homoserine lactones (AHLs). Chemical analyses of spent culture supernatants demonstrated that RhiL produces *N*-(hexanoyl)-*L*-homoserine lactone (*C*~6~-HSL) and *N*-(octanoyl)-*L*-homoserine lactone (*C*~8~-HSL). Both of these AHLs induced *rhiA-lacZ* and *rhiL-lacZ* expression on plasmids introduced into an *Agrobacterium* strain that produces no AHLs, showing that *rhiL* is positively regulated by autoinduction. However, in this system no induction of *rhiA* or *rhiL* with 3OH,*C*~14:1~-HSL was observed. Analysis of the spent culture supernatant of the wild-type *R*. *leguminosarum* bv. viciae revealed that at least seven different AHLs are made. Mutation of *rhiL* decreased the amounts of *C*~6~-HSL and *C*~8~-HSL but did not block their formation, and in this background the *rhiL* mutation did not significantly affect the expression levels of the *rhiL* gene or *rhiABC* genes or the accumulation of RhiA protein. These observations suggest that there are additional loci involved in AHL production in *R*. *leguminosarum* bv. viciae and that they affect *rhiL* and *rhiABC* expression. We postulate that the previously observed induction of *rhiA* by 3OH,*C*~14:1~-HSL may be due to an indirect effect caused by induction of other AHL production loci.

One of the most abundant proteins made by strains of *Rhizobium leguminosarum* bv. viciae is the *rhiA* gene product, which was first observed as a heavily stained band following sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) of stationary-phase cultures (7). *rhiA* is in a three-gene operon (*rhiABC*) that is under the regulatory control of the *rhiR* gene, which encodes a LuxR-type regulator. The *rhiABC-rhiR* gene cluster is located between genes involved in nitrogen fixation (*nifHDK*) and nodulation (*nod*) on the symbiotic plasmid pRL1JI (5). DNA hybridizations have shown that the *rhi* genes are adjacent to nodulation genes in other strains of *R*. *leguminosarum* bv. viciae (9a, 18). Analysis of several strains of *R*. *leguminosarum* (biovars viciae, trifolii, and phaseoli), *Rhizobium meliloti*, and *Rhizobium* sp. strain NGR234, using antibody to RhiA, showed that RhiA seems to be specific to *R*. *leguminosarum* bv. viciae (6, 7), indicating that it may play some host-specific role in the interaction between this biovar and its symbiotic partners (pea, vetch, lentil, and *Lathyrus* spp.).

Initially, no effect on nodulation or symbiotic nitrogen fixation was observed for bacteria containing transposon insertions in *rhiA* (7). Subsequently, it was found that mutation of *rhiA* could affect nodulation in some mutant strains that were already impaired for nodulation due to deletion of some of the nodulation genes (5). In particular, in the absence of the *nodFEL* genes, mutations of *rhiA* could be seen to decrease the low level of nodulation even further.

Analysis of *rhiA-lacZ* and *rhiC-phoA* gene fusions revealed that the *rhiABC* genes are strongly induced during the transition from late exponential to early stationary growth phase (15). The *N*-acyl-homoserine lactone (AHL) termed *N*-(3-hydroxy-7-cis-tetradecenoyl)-*L*-homoserine lactone (referred to hereafter as 3OH,*C*~14:1~-HSL) was identified as both an inducer of the *rhiABC* genes and a potent inhibitor of the growth of some strains of *R*. *leguminosarum* bv. viciae (15). Indeed the compound previously known as “small bacteriocin” (17, 35) had been purified and shown to be 3OH,*C*~14:1~-HSL (30). It was suggested that this AHL induces stationary phase in *R*. *leguminosarum* bv. viciae since it is thought to induce gene expression that inhibits growth but does not kill the cells (15).

Many strains of the *R*. *leguminosarum* biovars viciae, trifolii, and phaseoli (15, 17, 35, 37) make small bacteriocin (and thus probably make 3OH,*C*~14:1~-HSL) as does *Rhizobium etli* (26), whereas *R*. *meliloti* and the closely related *Agrobacterium tumefaciens* do not (17, 37). In addition, *R*. *etli* makes at least six

* Corresponding author. Mailing address: John Innes Centre, Colney Lane, Norwich NR4 7UH, United Kingdom. Phone: 1603 452571. Fax: 1603 456844. E-mail: downie@bbsrc.ac.uk.
† Present address: Departamento de Microbiologia, Facultad de Farmacia, Universidad de Granada, 18071 Granada, Spain.
‡ Present address: MRC Laboratory of Molecular Biology, Cambridge CBQ 2H, United Kingdom.
§ Present address: IMBB-Forth and Department of Biology, University of Crete, P.O. Box 1527, 711 10 Iraklio-Crete, Greece.

other compounds that are probably AHLs (26). Mutation of one gene, *rhiI*, in *R. etli* abolished the production of some autoinducers by *R. etli*, but the production of 3OH,C<sub>14:1</sub>-HSL was unaffected (26).

Rhizobium strains often contain multiple plasmids, and it is possible that different plasmids encode the production of different AHLs. In this regard it is not yet known if the small bacteriocin locus is located on the chromosome or on a plasmid in those strains which make small bacteriocin. The locus encoding production of small bacteriocin had been shown not to be located on the symbiotic plasmid pRL1JI (17, 37); nevertheless, 3OH,C<sub>14:1</sub>-HSL induces *rhiABC* gene expression. The regulation of the *rhiABC* genes is also affected by genes other than *rhiR* present on the symbiotic plasmid pRL1JI. It was observed (5, 10) that flavonoid inducers of *nod* gene expression decreased (by about 50%) the level of expression of the *rhiABC* operon and that this required the *nod* gene regulator NodD (5, 10). Furthermore, using a plasmid carrying a *rhiA-lacZ* fusion, Gray et al. (15) observed that an ethyl acetate extract containing 3OH,C<sub>14:1</sub>-HSL had different effects on *rhiA* gene expression in strains containing or lacking pRL1JI.

We wish to understand the physiological role of the *rhi* gene region. Analysis of RhiA protein with antiserum revealed that *rhiA* is expressed by bacteria in the rhizosphere but not by nitrogen-fixing bacteria in nodules (7). Database searches revealed no protein sequences with strong similarity to RhiA, RhiB, or RhiC, although it has been shown that RhiC has an N-terminal signal sequence that targets it to the periplasm (5). In this work we have further characterized the *rhi* gene region, identifying an AHL production locus that is involved in the induction of *rhiABC* expression.

### MATERIALS AND METHODS

#### Microbiological techniques. Rhizobium strains were grown in TY medium (2). Antibiotics were added as appropriate to maintain selection for plasmids. Bacterial growth was monitored by measuring the optical density at 600 nm (OD<sub>600</sub>) by using an MSE Spectroplus spectrophotometer. β-Galactosidase activities were measured as described previously (27) by using a TiterTek Multiscan Plus spectrophotometer (EFLAB). For measurements of β-galactosidase throughout growth, bacteria from 24-h cultures were diluted 1 in 200 to a starting OD<sub>600</sub> of about 0.002. When added, AHLs were added at the start of growth to a final concentration of 0.1 μM. The AHLs *N*-hexanoyl-**L**-homoserine lactone (*C*<sub>6</sub>-HSL), *N*-(3-oxohexanoyl)-**L**-homoserine lactone (3O,*C*<sub>6</sub>-HSL), *N*-octanoyl-**L**-homoserine lactone (*C*<sub>8</sub>-HSL), *N*-(3-oxooctanoyl)-**L**-homoserine lactone (3O,*C*<sub>8</sub>-HSL), *N*-(3-oxobutanoyl)-**L**-homoserine lactone (3O,*C*<sub>4</sub>-HSL), and *N*-(3-hydroxy-tetradecenoyl)-**L**-homoserine lactone (3OH,*C*<sub>14:1</sub>-HSL) were synthesized (4) and kindly provided by Ram Chhabra, School of Pharmaceutical Sciences, University of Nottingham. Nodulation tests were done by using variety Frisson peas (*Pisum sativum* L.) as described previously (8, 23), with a minimum of 12 matched plants per test; two separate tests were carried out with similar results.

#### Bacterial strains and plasmids. *R. leguminosarum* sp. strain 8401 lacks a symbiotic plasmid, and all Rhizobium strains used are based on 8401 (Table 1). A34 is a derivative of 8401 carrying the symbiotic plasmid pRL1JI. Strains A160 and A161, which are isogenic with A34, were made by conjugating derivatives of pRL1JI carrying *rhiR1::Tn5* and *rhiA4::Tn5* (7) into 8401. The *rhiI15::Tn5* allele from pIJ7790 (see below) was recombined onto pRL1JI by marker exchange by using pPH1JI to select for recombinants (28). The derivative of pRL1JI carrying *rhiI15::Tn5* was conjugated into 8401 to form A721, which was confirmed to lack pPH1JI (which transfers at a relatively low frequency).

Mutagenesis of pIJ1089 with Tn5 and Tn3HoHo1 was done as described previously (9, 10). Derivatives of pIJ1089 were transferred to strain 8401 and screened for reduced RhiA production by SDS-PAGE analysis of proteins released from cells, following solubilization of proteins in SDS gel loading buffer, as described previously (10). Mutated plasmids were transferred to the *Escherichia coli* strain DH5α by transformation with DNA isolated from the Rhizobium strains. The location of Tn3HoHo1 within *rhiI* in pIJ1696 was determined by restriction enzyme mapping. The location of Tn5 within *rhiI* in pIJ7790 was determined by subcloning part of the Tn5 plus flanking DNA as a 4.9-kb EcoRI-BamHI fragment in pUC19 and sequencing the DNA by using a Tn5-specific primer.

Plasmid pIJ7794 was made by cloning the *rhiI* gene on a 2.5-kb HindIII-SmaI fragment subcloned from pIJ1089; the fragment was subcloned in pIJ1891, which

---

### TABLE 1. Bacterial strains and plasmids

| Strain or plasmid | Description                                                                                   | Source |
|-------------------|-----------------------------------------------------------------------------------------------|--------|
| **Strain**        |                                                                                               |        |
| *R. leguminosarum*|                                                                                               |        |
| 8401              | Strain lacking a symbiotic plasmid                                                            | 8      |
| A34               | Derivative of 8401 carrying pRL1JI (previously called 8401pRL1JI)                             | 9      |
| A160              | Derivative of A34 carrying *rhiR1::Tn5*                                                      | 5      |
| A161              | Derivative of A34 carrying *rhiA4::Tn5*                                                      | This work |
| A721              | Derivative of A34 carrying *rhiI15::Tn5*                                                     | This work |
| *A. tumefaciens*  |                                                                                               |        |
| C58.00            | Strain lacking the AT and Ti plasmids; does not make detectable AHLs                          | 36     |
| **Plasmids**      |                                                                                               |        |
| pRL1JI            | *R. leguminosarum* bv. *viciae* native symbiotic plasmid                                      | 19     |
| pIJ1089           | Cosmid carrying 30 kb of pRL1JI, including the *nifHD, rhiABCR,* and *nodOTNMLEFDABCII* genes | 9      |
| pIJ1242           | Derivative of pIJ1089 carrying *rhiR1::Tn5*                                                  | 7      |
| pIJ1243           | Derivative of pIJ1089 carrying *rhiA4::Tn5*                                                  | 7      |
| pIJ1642           | Derivative of pIJ1089 carrying *rhiA5::Tn3HoHo1 (rhiA-lacZ)*                                  | 10     |
| pIJ1696           | Derivative of pIJ1089 carrying *rhiI7::Tn3HoHo1 (rhiI-lacZ)*                                 | This work |
| pIJ1769           | *rhiA-lacZ* fusion                                                                            | 5      |
| pIJ1891           | Cloning vector derived from pLAFR3                                                           | 21     |
| pIJ7790           | Derivative of pIJ1089 carrying *rhiI15::Tn5*                                                 | This work |
| pIJ7794           | 2.5-kb HindIII-SmaI fragment carrying *rhiI* in pIJ1891                                       | This work |
| pIJ7982           | *rhiI-lacZ* fusion in pMP220                                                                  | This work |
| pMP220            | *lacZ* promoter probe vector                                                                 | 32     |

had been first cut with BamHI, filled in with Klenow fragment of DNA polymerase, and finally digested with HindIII. The *rhiI-lacZ* gene fusion on pIJ7982 was made by cloning a 0.7-kb fragment carrying part of *rhiI* plus the 0.3-kb upstream region into pMP220. The 0.7-kb fragment was subcloned from one of the plasmids, generated by ExoIII nuclease deletion, for DNA sequencing.

#### Molecular biology techniques. DNA cloning, ligations, transformation, restriction enzyme mapping, and DNA hybridization were done by standard methods (29). The DNA sequence of *rhiI* was determined on both strands by using an ordered series of ExoIII-generated deleted derivatives of the *rhiI* gene region that had been cloned as a blunt-ended HindIII-SmaI fragment in both orientations in the HincII site of pUC19. The sequencing reactions were carried out by using the Amersham Thermosequenase kit and an Applied Biosystems automated sequencer (ABI 377). Database searches of the protein sequence were done by using the BLAST and TFASTA (1) programs to find related sequences in the EMBL and SwissProt protein sequence databases.

#### Analysis of proteins. Rhizobium strains were grown to an OD<sub>600</sub> of 1.2 in 100 ml of TY medium, and the cells were harvested by centrifugation. The washed cells from 10 ml of culture were resuspended in 1 ml of 0.1 M Tris HCl (pH 8.0) and lysed by sonication (30-s sonication with an MSE Soniprep at full power, done six times). The protein extract was solubilized and separated by SDS-PAGE, and the gels were stained with Coomassie blue R250 or transferred to nitrocellulose and probed with RhiA antiserum as described previously (3, 7).

#### Assay of AHLs. Cultures were grown for 48 h in TY medium to an OD<sub>600</sub> of 1.0. The cells were removed by centrifugation, and the AHLs were extracted from culture supernatants as described previously (38). AHLs were analyzed by thin-layer chromatography (TLC) as described by Shaw et al. (31) but with use

of Chromobacterium violaceum CV026 as the AHL indicator organism (22). CV026 can be used as a biosensor for exogenous AHLs because it produces the purple pigment violacein in response to added AHLs. Culture supernatants and synthetic AHL standards (as 1-mg ml⁻¹ solutions in acetonitrile) were spotted (2 to 10 μl) onto aluminum-backed RP18 reverse-phase TLC plates (Merck) and dried in a stream of air. Samples were separated with 60% (vol/vol) methanol in water as the mobile phase. Once the solvent front had migrated to within 2 cm of the top of the chromatogram, the plate was removed from the chromatography tank, dried in air, and overlaid with a thin film of Luria-Bertani soft agar (0.7%, wt/vol) seeded with C. violaceum CV026. After overnight incubation at 30°C, AHLs were located by detection of purple spots against a white background.

Isolation, purification, and chemical characterization of AHLs. Spent supernatant (4 liters) from stationary-phase cultures was extracted with dichloromethane (supernatant/dichloromethane ratio, 7:3). Dichloromethane was removed by rotary evaporation, and the residue was redissolved in 1.0 ml of acetonitrile and applied to a C₈ reverse-phase semipreparative high-performance liquid chromatography (HPLC) column (Kromasil KR100-5C8 [250 by 8 mm] column; Hichrom, Reading, United Kingdom). Fractions were eluted with a linear gradient of acetonitrile in water (20 to 95%, vol/vol) over a 30-min period at a flow rate of 2 ml/min and monitored at 210 nm. Six fractions (F1 to F6), covering 5-min intervals, were collected and assayed for activity by using a variety of AHL reporter assays, including the C. violaceum CV026 detection system (22, 24) and the E. coli(pSB401) luxR plus luxI′::luxCDABE, and E. coli(pSB1075) lasR plus lasI′::luxCDABE luminescence detection systems (34, 39). No peaks of activity other than those corresponding with the four spots visualized with the TLC CV026 overlay assay described above were identified. Active fractions were rechromatographed by using an appropriate isocratic mobile phase of acetonitrile in water, and the fractions were assayed for AHL activity. Active subfractions were also reanalyzed on an analytical HPLC apparatus attached to a photodiode array system by using the same isocratic mobile phase (Waters 996 PDA system operating with a Millennium 2010 Chromatography manager; Watford, Hertfordshire, United Kingdom), and both retention time and spectral profiles were compared with those of a series of synthetic AHL standards. Following preparative HPLC, the final active subfractions were analyzed by HPLC-mass spectrometry (HPLC-MS) (Micromass Instruments, Manchester, United Kingdom) using an appropriate isocratic mobile phase. This technique couples the resolving power of C₈ reverse-phase HPLC directly with MS such that the mass of the molecular ion (M + H) and its major component fragments can be determined for a compound with a given retention time. Samples eluting from the HPLC column were ionized by positive-ion atmospheric-pressure chemical-ionization MS and were analyzed at two different cone voltages (18 and 28 eV). The spectra obtained were compared with those of the synthetic AHL standard subjected to the same HPLC-MS conditions.

---

**RESULTS**

**Identification of a novel locus producing AHLs.** Plasmid pIJ1089 carries about 30 kb of DNA from the symbiotic plasmid pRL1JI. Identified genes on pIJ1089 include those involved with nitrogen fixation (*nifHD*) and nodulation (*nodO*, *nodMNT*, *nodFEL*, *nodD*, and *nodABCII*) and the rhizosphere-expressed genes *rhiABC* and *rhiR*. This plasmid directs the production of high levels of the RhiA protein with an *M*ᵣ of 24,000 in strain 8401, which lacks a symbiotic plasmid (Fig. 1). pIJ1089 was mutagenized with Tn3HoHo1 or Tn5, the mutated derivatives were conjugated into strain 8401, and protein extracts of the transconjugants were analyzed by SDS-PAGE. Several mutants affected in production of the RhiA protein were identified, and DNA mapping or sequencing from the ends of the transposon confirmed that most of the mutations were in the structural and regulatory genes, *rhiA* and *rhiR*, respectively. However, with two of the mutated derivatives, pIJ1696 and pIJ7790 (carrying Tn3HoHo1 and Tn5, respectively), the sites of transposon insertions mapped to a new locus about 2 kb upstream of *rhiA*. As shown in Fig. 1A, strain 8401/pIJ1089 produces a prominent 24-kDa protein that is absent if *rhiA* is mutated and is very greatly reduced in intensity when *rhiR* is mutated (Fig. 1A, lanes 1, 2, and 4). With 8401/pIJ7790, the level of RhiA protein is significantly decreased (Fig. 1A, lane 3). Immunostaining performed by using anti-serum to RhiA (Fig. 1B) confirmed that mutation of the new locus in pIJ7790 significantly reduces the level of *rhiA* expression. Similar observations (data not shown) were made with pIJ1696 (carrying Tn3HoHo1 in the region about 2 kb up-

---

FIG. 1. Effects on RhiA production of mutating *rhiI*. Cell extracts were separated by SDS-PAGE and either stained with Coomassie blue R250 (A) or transferred to nitrocellulose and immunostained with RhiA antiserum followed by a secondary antibody (anti-rabbit immunoglobulin G) coupled to alkaline phosphatase (B). The order of lanes in panel A is the same as that in panel B; 20 μg (panel A) or 10 μg (panel B) of protein was loaded in each lane. Extracts were from the following: (lane 1) A31/pIJ1089 (wild type); (lane 2) A31/pIJ1243 (*rhiA4::Tn5*); (lane 3) A31/pIJ7790 (*rhiI5::Tn5*); (lane 4) A31/pIJ1242 (*rhiR1::Tn5*); (lane 5) A34 (wild type); (lane 6) A161 (*rhiA4::Tn5*); (lane 7) A721 (*rhiI5::Tn5*); and (lane 8) A160 (*rhiR1::Tn5*). The markers (lane M) have *M*ᵣ of 94,000, 67,000, 45,000, 30,000, and 20,000, as indicated.

stream of *rhiA*). We considered that the new locus, mutated in pIJ1696 and pIJ7790, may be involved in production of an AHL that influences expression of *rhiA*.

*A. tumefaciens* C58.00 makes no AHLs detectable by *C.* violaceum CV026 (Fig. 2) and so is a useful strain in testing for AHL production mediated by pIJ1089 and its derivatives (*E.* coli is not a good host for expression of *R.* leguminosarum genes). The culture supernatant of C58.00 carrying pIJ1089 was extracted with dichloromethane and separated by TLC. The chromatogram was overlaid with agar inoculated with the AHL-sensor strain *C.* violaceum CV026 to detect AHLs. As shown here (Fig. 2, lane b), introducing pIJ1089 into C58.00 results in the production of four components that activate pigment production by CV026. It should be noted that the intensity of staining does not reflect the relative amounts of individual AHLs made, since the sensor strain has different sensitivities for different AHLs (21). Two of the spots determined by pIJ1089 comigrate with the synthetic standards C₆-HSL and C₈-HSL. The crude extract was fractionated by HPLC by using a linear gradient of acetonitrile in water. When tested by the TLC CV026 overlay assay, fractions 3 and 4 were found to activate violacein production by CV026. Fractions 3 and 4 were further fractionated by using an isocratic acetonitrile-in-water mobile phase (percentages of acetonitrile in water, 25 and 35% [vol/vol], respectively). When active subfractions of fraction 3 were reanalyzed by analytical HPLC, the active compound was found to elute with the same retention time (17 min) and photodiode array absorption spectrum as synthetic C₆-HSL (data not shown). To confirm the identities

Vol. 181, 1999

QUORUM SENSING IN R. LEGUMINOSARUM BV. VICIAE 3819

![Diagram](https://i.imgur.com/yourimageurl.png)

FIG. 2. Identification of AHLs produced by Rhil. AHLs extracted from spent medium were separated by TLC and detected by using an overlay of agar containing C. violaceum CV026. Lanes a and b contain extracts of spent culture supernatants from A. tumefaciens C58.00 carrying either pIJ7790 (carrying the rhiI15::Tn5 mutation) (lane a) or pIJ1089 (lane b). Results similar to those seen in lane a (no AHLs were detected) were found with C58.00 carrying no introduced plasmid. Lane c contains the synthetic standards C₆-HSL and C₈-HSL. Lanes d, e, and f contain extracts of spent culture supernatants from the R. leguminosarum bv. viciae strains A34 (wild type), A160 (rhiR1::Tn5), and A721 (rhiI15::Tn5), respectively.

plJ7790 plJ1089

**Agrobacterium**

WT rhiR rhil

**Rhizobium**

of the AHLs, active fractions were subjected to HPLC-MS on an isocratic mobile phase (percentages of acetonitrile in water were 25 and 50% [vol/vol] for subfractions from fractions 3 and 4, respectively). Positive-ion atmospheric-pressure chemical-ionization MS for fraction 3 revealed the presence of a molecular ion [M + H] of 200, corresponding to the C₆-HSL, together with the characteristic fragmentation products at 102 and 99, which correspond to the homoserine lactone moiety and the C₆ acyl side chain [CH₃(CH₂)₄C≡O⁺], respectively. Similar analysis of the active subfraction of fraction 4 confirmed the presence of a molecular ion [M + H] of 228, corresponding to the C₈-HSL, and breakdown products at 102 and 127, which correspond to the homoserine lactone moiety and the C₈ acyl side chain [CH₃(CH₂)₆C≡O⁺], respectively. The two additional putative AHLs detected by the TLC CV026 overlay assay (Fig. 2) did not migrate with any of the known AHLs. Characterization of these compounds is under way.

The plasmid (pIJ7790) defective for RhiA production (Fig. 1) was also defective in the production of all four spots detected by C. violaceum CV026 following TLC (Fig. 2, lane a). A similar observation was made with pIJ1696 (data not shown). These results demonstrate that pIJ1089 carries a locus that directs the production of C₆-HSL and C₈-HSL (and possibly other AHLs) and that transposon insertions in a region upstream of rhiA abolish this phenotype.

Characterization of rhiI. A 2.5-kb fragment of DNA corresponding to the region mutated by Tn5 and Tn3HoHo1 was subcloned to make pIJ7794, which was confirmed to be involved in AHL production by using C. violaceum CV026 as an AHL sensor (data not shown). The DNA sequence of the region was determined, and the sites of the transposon insertions were determined by restriction enzyme mapping and DNA sequencing. Both insertions are located in a short open reading frame (Fig. 3), which encodes a 185-amino-acid protein with similarity to several other proteins (including LuxI) that are involved in AHL production. We called the gene rhiI since it is involved in AHL production and mutations of this gene in pIJ1089 decrease rhiA expression as judged by the

FIG. 3. Map of the rhi gene region. The open reading frames corresponding to the rhiI, rhiA, rhiB, rhiC, and rhiR genes are shown as arrows, and the locations of the rhiI7::Tn3HoHo1 and rhiI15::Tn5 insertions are indicated as open and closed triangles, respectively. The DNA cloned to form rhiI plasmid pIJ7794 and the rhiI-lacZ (pIJ7982) and rhiA-lacZ (pIJ1769) fusions is shown.

levels of RhiA protein detected by SDS-PAGE (Fig. 1; compare lanes 1 and 3).

Figure 4 shows an alignment of the predicted sequence of RhiI with the most closely related proteins identified in database searches. Although RhiI shows clear similarities with the LuxI family of proteins, it is evident that RhiI has only about 15 to 20% identity with LuxI (Vibrio fischeri), LasI (Pseudomonas aeruginosa), Rail (R. etli), Rhll (P. aeruginosa), and PhzI (Pseudomonas aureofaciens). Several other homologs were identified, but these had fewer similarities than those whose alignments are shown in Fig. 4. RhiI contains the highly conserved residues (12, 16, 25) found in multiple LuxI homologs (Arg-24, Phe-27, Trp-33, Glu-43, Asp-45, Asp-48, Arg-68, Glu-99, and Arg-102; numbered with reference to RhiI and marked in Fig. 4). In other RhiI homologs a Phe residue is normally found at position 82 (12), but in RhiI the conservatively substituted residue Tyr is found (Fig. 4).

rhiI is regulated by RhiR in a cell density-dependent manner. A rhiI-lacZ reporter plasmid (pIJ7982) was made by cloning a 0.7-kb fragment (containing 0.3 kb of DNA upstream of the predicted rhiI translation start site) into the lacZ fusion vector pMP220. The expression level of β-galactosidase was measured throughout growth in strain A34, in a derivative of it mutated in rhiR (A160), or in a strain lacking the symbiotic plasmid pRL1JI (8401). Parallel experiments were done with the rhiA-lacZ plasmid pIJ1769. In A34 there are very low levels of rhiI and rhiA expression early in growth, but the levels increase markedly in the late exponential phase and reach a plateau in stationary phase (Fig. 5). The expression of rhiI is rhiR dependent since very low levels of activity are seen with A160 (rhiR1::Tn5) or 8401 (which lacks pSym carrying rhiR). Therefore, rhiI (like rhiA) is regulated by RhiR in a cell density-dependent manner. We could identify no sequences upstream of rhiI or rhiA that showed strong similarity to the 20-bp region of dyad symmetry (lux box) found upstream of several genes regulated by LuxR-type proteins (13, 33). However, 45 bp upstream of the proposed translation start site of rhiI is a 22-bp region of dyad symmetry (Fig. 6). A similar sequence was found upstream of rhiA (Fig. 6), although the center of symmetry was slightly different. It remains to be demonstrated if these regions are involved in the binding of RhiR.

rhiI and rhiA are induced by C₆-HSL and C₈-HSL. The results presented above indicate that rhiI and rhiA are likely to be induced by AHLs. To analyze those AHLs that induce rhiI and rhiA, we made use of derivatives of pIJ1089 carrying lacZ

luxi  M T I M I K K S D F L A I P S E E Y K G I L S L R Y Q V F K Q R L E W D L V V E N N L . . . E S D E Y D N S N A E Y I Y 57
lasi  M I V Q I G R R E . J E F D K K L L G E M H K L R A Q V F K E R K G W D V S V I D E M . . . E I D G Y D A L S P Y Y M L 55
raii  . M L R I L T K D M L E T D R R A F D E M F R A R A A V F R B R L G W Q V D V R D Q W . . . E R D R Y D E A E D P . W Y 55
rhli  . M I E L L S E S L E G L S A A M I A E L G R Y R H Q V F I E K L G W D V V S T S R V R D Q E B D O E D H P Q T R Y I V 59
phzi  . . M H M E E H T L N Q M S D E L K L M L G R F R H E Q F V E K L G W R L P A H P S Q A G C E W D Q Y D T E H A R Y L L 58
rhii  . . M M K I C I G D S R A N A V I M E K I W R F R H O Q F V E R R G W Q A L R K S D R R . . E I D Q F D H D C A I H F S 56

luxi  A C D D T . . . E N V S G C W R L L P T T G D Y M L K S V F P E L L G Q Q S A P K D P N I V E L S R F A V G K N S S K I N 115
lasi  I Q E D T P E A Q V F G C W R I L D T T G P Y M L K N T F P E L L H G K E A P C S P H I W E L S R F A I . N S G Q K G S 114
raii  L V T Q Q P S G T L T G S L R L L P T T G A T M L K S E F R H F D Q P I D V D S P T T W E C T R F C L H P H A G D M K 115
rhli  A M G R Q . . . G C G C A R L L P T T D A Y L L K E V F A Y L C S E T P . P S D P S W E L S R F A A . . S A A D P 113
phzi  A F N R D R . . A V G C A R L I P T T F P N L L E G V F G H T C A G A P . P K H P A I W E M T R F T T . . R . . . E P 110
rhii  V L K N D . . . A V I G S S R L L P T V Q P H L L S S V P Q P M R G Q K W P R S H D W E W T R W A V . . A S G D E R 111

luxi  N S A S E I T M K L F E A I Y K H A V S Q G I T E Y V T V T S T A L E R F L K R I K V P C H R T G D K E I H V L G D T K 175
lasi  L G F S D C T L E A M R A L A R Y S L Q N D I Q T L V T V T T V G V E K M M I R A G L D V S R B G P H . . L K I G I E R 172
raii  Q S R A V A T . E L I S G L C D L A L D T G I E N I V G V Y D V A M V A V Y R R I G W R P T P L A R S R P E I G K L Y V 174
rhli  Q L A M K I F W . . . . S S L Q C A W Y L G A S S V V A V T T T A M E R Y F V R N G V I L Q R L G P P Q K V K G E T L V 169
phzi  Q L A M P L F W . . . . R S L K T A S L A G A D A I V G E V N S T M E R Y Y K I N G V H Y E R L G P V T H Q N E K I L 166
rhii  I D G V R V S R V L M I G L M E F C R V A G I T S L Z G E T H P K I N S L R A N G W Q T H I L S E P S I F E N E L V V 171

luxi  S V V L S M P I N E Q F K K A V L N . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
QUORUM SENSING IN R. LEGUMINOSARUM BV. VICIAE 3821

**FIG. 6.** Conserved DNA sequences found upstream of *rhiI* and *rhiA*. The sequences shown are found 45 and 146 bp upstream of the predicted translation start sites of *rhiI* and *rhiA*, respectively. Conserved residues (·) are marked, and the dyad symmetry is indicated with arrows.

$rhiI$ GAAATGAGTATTTCTACTCATTTTC  
$rhiA$ TCAGGGAGTAAATCTACTCATTTAC  

expression decreased *rhiA* expression by about 50% and that this decrease was *nodD* dependent (5, 10). We measured expression of *rhiI-lacZ* using pIJ7982 in the presence and absence of the *nod* gene inducer hesperetin, under conditions similar to those shown in Fig. 5A. After 42 h of growth, the level of β-galactosidase activity in the cells grown with hesperetin (6,800 ± 610 U) was about half that seen when hesperetin was not added (11,300 ± 840 U). Therefore, like that of *rhiA*, *rhiI* expression is decreased by inducers of *nod* gene expression. No significant effect of hesperetin on expression of the *rhiI-lacZ* fusion in the *nodD* mutant A57 was observed (data not shown), confirming that the hesperetin-induced reduction of *rhiI-lacZ* expression is *nodD* dependent.

We tested nodulation ability of the *rhiI* mutant (A721) relative to that of the control strain A34. The mutant formed normal nitrogen-fixing nodules on peas; the final number of nodules formed was slightly (but significantly) higher than that for the control, although the rate of nodule formation was similar to that for the control (Fig. 7).

**DISCUSSION**

The *rhiABC* operon is conserved in all *R. leguminosarum* bv. viciae strains tested and has not been found in other rhizobia. This, taken together with the location of the *rhi* gene cluster (between the *nod* and *nif* genes), indicates that these genes may play some kind of role in the interaction between *R. leguminosarum* bv. viciae and at least some of its host legumes.

It is now evident that the *rhiABC* genes are regulated by RhiR in response to AHLs made by RhiI. The *rhiI* gene is regulated in the same way, thereby forming a positive autoregulatory loop that results in high levels of expression from the *rhiI* and *rhiABC* promoters. This explains the very high levels of RhiA protein detected in late-stationary-phase cells, in which it is certainly one of the most abundant proteins (7). However, *rhiA* expression is not totally dependent on *rhiI* since

**TABLE 2. Effects of added AHLs on *rhiI-lacZ* and *rhiA-lacZ* expression**${}^{a}$

| AHL tested | Expression (Miller units) of: |
| --- | --- | --- |
|  | *rhiI-lacZ* | *rhiA-lacZ* |
| None (negative control) | 369.78 | 198.98 |
| $C_{6}$-HSL | 2,493.28${}^{b}$ | 1,079.54${}^{b}$ |
| $C_{8}$-HSL | 1,585.57${}^{b}$ | 546.43${}^{b}$ |
| 3O,$C_{6}$-HSL | 896.01${}^{b}$ | 196.35 |
| 3O,$C_{8}$-HSL | 1,191.00${}^{b}$ | 332.16${}^{b}$ |
| 3OH,$C_{4}$-HSL | 544.89${}^{b}$ | 154.39 |
| 3OH,$C_{14:1}$-HSL | 484.12 | 151.94 |
| LSD ($P < 0.05$) | 170.66 | 82.45 |

${}^{a}$ AHLs were tested in *Agrobacterium* sp. strain C58.00 carrying pIJ1642 or pIJ1696, using a final concentration of 0.1 μM each AHL.  
${}^{b}$ Values are significantly different from that for the negative control (Student’s $t$ test, $P < 0.05$).  
LSD, least significant difference.

and 500 kb in size (20). It remains to be determined if these plasmids harbor genes involved in AHL production.

It is not clear why *R. leguminosarum* bv. viciae should have the *rhiI-rhiR* regulatory system to induce expression of the *rhiABC* operon, although several lines of evidence relate this to some aspect of the interaction with leguminous plants. The observation that flavonoids which induce *nod* gene expression reduce *rhiI* expression suggests that the plant has the potential to influence the level of AHL production. However, the effect of flavonoids on *rhi* gene expression depends on the *R. leguminosarum* bv. viciae *nod* gene regulator NodD (10), indicating that the bacteria influence this decrease in *rhiI* and *rhiABC* expression. This is somewhat different from the observed inhibition of quorum-sensing regulated genes by halogenated furanones, which are thought to act as competitive inhibitors of AHL binding to LuxR-type regulators (14).

We do not yet know the biochemical role of the *rhiABC* gene products or why they should be regulated in a cell density-dependent manner. It is evident that in some way they influence the interaction with the plant since mutation of *rhiA* or *rhiR* significantly reduced nodulation in a strain lacking the *nodFEL* genes (5). Paradoxically, mutation of *rhiI* increased the final number of nodules formed. In *R. etli*, mutation of the *rail* gene, which is also involved in AHL production, resulted in increased levels of bean nodulation (26). Thus, for two separate *Rhizobium*-legume interactions there is independent evidence that production of (at least) some of the AHLs inhibits nodulation under the growth conditions tested.

there appears to be another AHL production locus which can form AHLs that stimulate the expression of both the *rhiABC* operon and *rhiI*. Since *rhiR* mutants show very little expression of *rhiI* or *rhiABC*, it is evident that although there are other loci for AHL production, any regulatory genes that might be associated with those loci are not able to induce *rhiABC* or *rhiI* expression. However, there could be an indirect effect on *rhiABC* expression since AHLs, made by products of genes other than *rhiI*, can stimulate RhiR to induce *rhiI* and *rhiABC* expression. Thus, there is likely to be a degree of cross talk between different AHL production loci.

Previously the *rhiA* promoter was observed to be induced when 3OH,C<sub>14:1</sub>-HSL was added to wild-type cells during early-exponential-phase growth (15). However, RhiR activates *rhiA* in response to C<sub>6</sub>-HSL and C<sub>8</sub>-HSL but not 3OH,C<sub>14:1</sub>-HSL. Therefore, the most-probable explanation for the previous results is that 3OH,C<sub>14:1</sub>-HSL induces the expression of other AHLs, which in turn activate RhiR-mediated *rhiABC* and *rhiI* expression. The fact that *rhiI* and *rhiR* mutants still produce many short-chain AHLs is good evidence that there is at least one other AHL production locus in *R. leguminosarum* bv. viciae; indeed, in other (unpublished) work, we have cloned four AHL production loci from strain A34. Two other *luxI*-like genes in rhizobia have been described. In *R. etli* the *rail* gene was sequenced and shown to be involved in the formation of several (chemically uncharacterized) AHLs (26). In *Rhizobium* sp. strain NGR234, a *traI* gene was identified in a symbiotic plasmid genome-sequencing project (11). Although Rhil described here is in the same family as these two proteins, it is not much more similar to RaiI or TraI proteins from rhizobia than to related proteins from several other bacteria (Fig. 4). *R. leguminosarum* bv. viciae strain A34 may have, in addition to *rhiI*, other AHL production genes homologous to *traI* and/or *rail*. Perhaps the diversity of AHL production systems in *Rhizobium* may be related to the fact that many strains harbor multiple large plasmids. Different plasmids may have different AHL production systems, and strain 8401 (lacking a symbiosis plasmid) contains two plasmids thought to be greater than 300

In the absence of significant protein sequence similarities to any other proteins of known function, it is difficult to predict the role of RhiA, RhiB, and RhiC. The RhiC protein appears to be located in the periplasm, possibly suggesting a role for the uptake of some metabolite, but our tests of growth of *rhiABC* mutants on various carbon sources have not identified any clear differences from the growth of the isogenic control strain (9a). It is evident that in *R. leguminosarum* bv. viciae quorum-sensing-based regulation is complex and may share similarities with the cascade of quorum-sensing-regulated genes in *P. aeruginosa* and *V. fischeri*. The reason for such complexity of regulation and for the apparent degree of redundancy of AHL production is not clear, and its elucidation will require characterization of the other AHL production loci in *R. leguminosarum* bv. viciae.

## ACKNOWLEDGMENTS

We thank S. R. Chhabra for the generous gift of chemically synthesized AHLs, Y. Dessaux for suggesting the use of and providing *Agrobacterium* strain C58.00, and A. Davies for skilled technical assistance.

This work was supported in part by the BBSRC, a fellowship from the Universidad de Granada (to B.R.), and a contract (B104-CT96-0181) from the EU (DGXII-SSMI).

## REFERENCES

1. Altschul, S. F., W. Gish, W. Miller, E. W. Myers, and D. J. Lipman. 1990. Basic local alignment search tool. J. Mol. Biol. **215**:403–410.
2. Beringer, J. E. 1974. R factor transfer in *Rhizobium leguminosarum*. J. Gen. Microbiol. **84**:188–198.
3. Bradley, D. J., E. A. Wood, A. P. Larkins, G. Galfrè, G. W. Butcher, and N. J. Brewin. 1988. Isolation of monoclonal antibodies reacting with peribacteroid membranes and other components of pea root nodules containing *Rhizobium leguminosarum*. Planta **173**:149–160.
4. Chhabra, S. R., P. Stead, N. J. Bainton, G. P. C. Salmond, G. S. A. B. Stewart, P. Williams, and B. W. Bycroft. 1993. Autoregulation of carbapenem biosynthesis in *Erwinia carotovora* ATCC 39048 by analogues of N-(3-oxohexanoyl)-L-homoserine lactone. J. Antibiot. **46**:441–454.
5. Cubo, T., A. Economou, G. Murphy, A. W. B. Johnston, and J. A. Downie. 1992. Molecular characterization and regulation of the rhizosphere-expressed genes *rhiABCR* that can influence nodulation by *Rhizobium legu-

minosarum biovar viciae. J. Bacteriol. 174:4026–4035.

6. Dibb, N. J. 1983. Plasmid-determined proteins in *Rhizobium leguminosarum*. Ph.D. thesis, University of East Anglia, Norwich, United Kingdom.

7. Dibb, N. J., J. A. Downie, and N. J. Brewin. 1984. Identification of a rhizosphere protein encoded by the symbiotic plasmid of *Rhizobium leguminosarum*. J. Bacteriol. 158:621–627.

8. Downie, J. A., G. Hombrecher, Q.-S. Ma, C. D. Knight, B. Wells, and A. W. B. Johnston. 1983. Cloned nodulation genes of *Rhizobium leguminosarum* determine host-range specificity. Mol. Gen. 190:359–365.

9. Downie, J. A., C. D. Knight, and A. W. B. Johnston. 1985. Identification of genes and gene products involved in nodulation of peas by *Rhizobium leguminosarum*. Mol. Gen. 198:255–262.

9a. Downie, J. A. Unpublished observations.

10. Economou, A., F. K. L. Hawkins, J. A. Downie, and A. W. B. Johnston. 1989. Transcription of *rhiA*, a gene on a *Rhizobium leguminosarum* bv. *viciae* Sym plasmid, requires *rhiR* and is repressed by flavonoids that induce *nod* genes. Mol. Microbiol. 3:87–93.

11. Freiberg, C., R. Fellay, A. Bairoch, W. J. Broughton, A. Rosenthal, and X. Perret. 1997. Molecular basis of symbiosis between *Rhizobium* and legumes. Nature 387:394–401.

12. Fuqua, C., S. C. Winans, and E. P. Greenberg. 1996. Census and consensus in bacterial ecosystems: the LuxR-LuxI family of quorum-sensing transcriptional regulators. Annu. Rev. Microbiol. 50:727–751.

13. Fuqua, W. C., S. C. Winans, and E. P. Greenberg. 1994. Quorum sensing in bacteria: the LuxR-LuxI family of cell density-responsive transcriptional regulators. J. Bacteriol. 176:269–275.

14. Givskov, M., R. de Nys, M. Manefield, L. Gram, R. Maximilien, L. Eberl, S. Molin, P. D. Steinberg, and S. Kjelleberg. 1996. Eukaryotic interference with homoserine lactone-mediated prokaryotic signalling. J. Bacteriol. 178:6618–6622.

15. Gray, K. M., J. P. Pearson, J. A. Downie, B. E. A. Boboye, and E. P. Greenberg. 1996. Cell-to-cell signaling in the symbiotic nitrogen-fixing bacterium *Rhizobium leguminosarum*: autoinduction of a stationary phase and rhizosphere-expressed genes. J. Bacteriol. 178:372–376.

16. Hanzelka, B. L., A. M. Stevens, M. R. Parsek, T. J. Crone, and E. P. Greenberg. 1997. Mutational analysis of the *Vibrio fischeri* LuxI polypeptide: critical regions of an autoinducer synthase. J. Bacteriol. 179:4882–4887.

17. Hirsch, P. R. 1979. Plasmid-determined bacteriocin production by *Rhizobium leguminosarum*. J. Gen. Microbiol. 113:219–228.

18. Hombrecher, G., R. Götz, N. J. Dibb, J. A. Downie, A. W. B. Johnston, and N. J. Brewin. 1984. Cloning and mutagenesis of nodulation genes from *Rhizobium leguminosarum* TOM, a strain with extended host range. Mol. Gen. 184:293–298.

19. Johnston, A. W. B., J. L. Beynon, A. V. Buchanan-Wollaston, S. M. Setchell, P. Hirsh, and J. E. Beringer. 1978. High frequency transfer of nodulating ability between strains and species of *Rhizobium*. Nature 276:634–636.

20. Lamb, J. W., G. Hombrecher, and A. W. B. Johnston. 1982. Plasmid-determined nodulation and nitrogen-fixation abilities in *Rhizobium phaseoli*. Mol. Gen. Genet. 186:449–452.

21. Mavridou, A., M. A. Barny, P. Poole, K. Plaskitt, A. E. Davies, A. W. B. Johnston, and J. A. Downie. 1995. *Rhizobium leguminosarum* nodulation gene (*nod*) expression is lowered by an allele-specific mutation in the dicarboxylate transport gene *dctB*. Microbiology 141:103–111.

22. McClean, K. H., M. K. Winson, L. Fish, A. Taylor, S. R. Chhabra, M. Camara, M. Daykin, S. Swift, B. W. Bycroft, G. S. A. B. Stewart, and P. Williams. 1997. Quorum sensing and *Chromobacterium violaceum*: exploitation of violacein production and inhibition for the detection of *N*-acylhomoserine lactones. Microbiology 143:3703–3711.

23. Miller, J. A. 1976. Experiments in molecular genetics, p. 352–355. Cold Spring Harbor Laboratory Press, Plainview, N.Y.

24. Milton, D. L., A. Hardman, M. Camara, S. R. Chhabra, B. W. Bycroft,

G. S. A. B. Stewart, and P. Williams. 1997. Quorum sensing in *Vibrio anguillarum*: characterization of the *vanI/vanR* locus and identification of the autoinducer *N*-(3-oxodecanoyl)-*L*-homoserine lactone. J. Bacteriol. 179:3004–3012.

25. Parsek, M. R., A. L. Schaefer, and E. P. Greenberg. 1997. Analysis of random and site-directed mutations in *rhll*, a *Pseudomonas aeruginosa* gene encoding an acylhomoserine lactone synthase. Mol. Microbiol. 26:301–310.

26. Rosemeyer, V., J. Michiels, C. Verreth, and J. Vanderleyden. 1998. *luxI*- and *luxR*-homologous genes of *Rhizobium etli* CNPAF512 contribute to synthesis of autoinducer molecules and nodulation of *Phaseolus vulgaris*. J. Bacteriol. 180:815–821.

27. Rossen, L., C. A. Shearman, A. W. B. Johnston, and J. A. Downie. 1985. The *nodD* gene of *Rhizobium leguminosarum* is autoregulatory and in the presence of plant exudate induces the *nodABC* genes. EMBO J. 4:3369–3373.

28. Ruvkun, G. B., and F. M. Ausubel. 1981. A general method for site-directed mutagenesis in prokaryotes. Nature 289:85–88.

29. Sambrook, J., E. F. Fritsch, and T. Maniatis. 1989. Molecular cloning: a laboratory manual, 2nd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.

30. Schripsema, J., K. E. E. de Rudder, T. B. van Vliet, P. P. Lankhorst, E. de Vroom, J. W. Kijne, and A. A. N. van Brussel. 1996. Bacteriocin *small* of *Rhizobium leguminosarum* belongs to the class of *N*-acyl-*L*-homoserine lactone molecules, known as autoinducers and as quorum sensing co-transcription factors. J. Bacteriol. 178:366–371.

31. Shaw, P. D., G. Ping, S. Daly, J. E. Cronan, Jr., K. Rinehart, and S. K. Farrand. 1997. Detecting and characterizing acyl-homoserine lactone signal molecules by thin layer chromatography. Proc. Natl. Acad. Sci. USA 94:6036–6041.

32. Spaink, H. P., R. J. H. Okker, C. A. Wijffelman, E. Pees, and B. J. J. Lugtenberg. 1987. Promoters in the nodulation region of the *Rhizobium leguminosarum* Sym plasmid pRLIJI. Plant Mol. Biol. 9:27–39.

33. Stevens, A. M., and E. P. Greenberg. 1997. Quorum sensing in *Vibrio fischeri*: essential elements for activation of the luminescence genes. J. Bacteriol. 179:557–562.

34. Throup, P. J., M. Camera, G. S. Briggs, M. K. Winson, S. R. Chhabra, B. W. Bycroft, P. Williams, and G. S. A. B. Stewart. 1995. Characterisation of the *yenI/yenR* locus from *Yersinia enterocolitica* mediating the synthesis of two *N*-acylhomoserine lactone signal molecules. Mol. Microbiol. 17:345–356.

35. van Brussel, A. A. N., S. A. J. Zaat, C. A. Wijffelman, E. Pees, and B. J. J. Lugtenberg. 1985. Bacteriocin *small* of fast-growing rhizobia is chloroform soluble and is not required for effective nodulation. J. Bacteriol. 162:1079–1082.

36. Vaudequin-Dransart, V., A. Petit, C. Poncet, C. Ponsonnet, X. Nesme, J. B. Jones, H. Bouzar, W. Scott Chilton, and Y. Dessaux. 1995. Novel Ti plasmids in *Agrobacterium* strains isolated from fig tree and chrysanthemum tumors and their opinelike molecules. Phytopathology 8:311–321.

37. Wijffelman, C. A., E. Pees, A. A. N. Van Brussel, and P. J. J. Hooykaas. 1983. Repression of small bacteriocin excretion in *Rhizobium leguminosarum* and *Rhizobium trifolii* by transmissible plasmids. Mol. Gen. Genet. 192:171–176.

38. Winson, M. K., M. Camara, A. Latifi, M. Foglino, S. R. Chhabra, M. Daykin, V. Chapon, B. W. Bycroft, G. P. C. Salmond, A. Lazdunski, G. S. A. B. Stewart, and P. Williams. 1995. Multiple quorum sensing modulons interactively regulate virulence and secondary metabolism in *Pseudomonas aeruginosa*: identification of the signal molecules *N*-butanoyl-*L*-homoserine lactone and *N*-hexanoyl-*L*-homoserine lactone. Proc. Natl. Acad. Sci. USA 92:9427–9431.

39. Winson, M. K., S. Swift, P. J. Hill, C. M. Sims, G. Griesmayr, B. W. Bycroft, P. Williams, and G. S. A. B. Stewart. 1998. Engineering the *luxCDABE* genes from *Photorhabdus luminescens* to provide a bioluminescent reporter for constitutive and promoter probe plasmids and mini-Tn5 constructs. FEMS Microbiol. Lett. 16:193–202.
