
Biosynthesis of polyketides (other than actinomycete macrolides)

Bernard J. Rawlings

Department of Chemistry, University of Leicester, University Road, Leicester, UK LE1 7RH.
E-mail: bjr2@le.ac.uk or Bernard.Rawlings@le.ac.uk

Received (in Cambridge) 20th January 1999
Covering: 1995, 1996, 1997 and early 1998
Previous review: 1998, 15, 275 and 1997, 14, 523

---

### Contents

1. **Introduction**
   - 1.1 Summary of recent highlights

2. **Gram negative bacteria**

3. **Gram positive bacteria: actinomycetes, Type II aromatic polyketides**
   - 3.1 Introduction
   - 3.2 Studies on the roles of PKS cluster components—hybrid clusters
     - 3.3 7,9,12-Octaketides
       - 3.3.1 Actinorhodin
         - 3.3.1.1 Regulation
           - 3.3.1.2 Malonyl transferase (MT)
           - 3.3.1.3 Acyl carrier protein
           - 3.3.1.4 3-Oxoacyl synthetase and chain length factor
           - 3.3.1.5 Ketoreductase KR1(9) (actIII)
           - 3.3.1.6 Cyclase CYC2/3 (actIV) and aromatase AROab (actVII)
           - 3.3.1.7 Post PKS reactions
       - 3.3.2 Other benzoisochromanequinones
     - 3.4 7,9,12-Nonaketides
       - 3.4.1 Tetracyclines
       - 3.4.2 Frenolicin B
     - 3.5 7,9,12-Decaketides
       - 3.5.1 Anthracyclines
         - 3.5.1.1 Aklavinone
         - 3.5.1.2 Daunorubicin and doxorubicin
         - 3.5.1.3 Nogalamycin
         - 3.5.1.4 Choice of starter unit
       - 3.5.2 Angucyclines
         - 3.5.2.1 Dehydrorabelomycins
         - 3.5.2.2 Tetrangomycins
     - 3.6 9,14-Decaketides: tetracenomycins
       - 3.6.1 Starter unit specificity
       - 3.6.2 Chain length
       - 3.6.3 First cyclisation
       - 3.6.4 Post PKS hybrids
     - 3.7 7,12-Decaketides: aureolic acids
     - 3.8 Other Type II aromatic polyketides
       - 3.8.1 5,7,10-Decaketides
       - 3.8.2 9,11,14-Dodecaketides

4. **Gram positive bacteria: Actinomycetes, Type I assembly**

5. **Protoctista**

6. **Plantae**
   - 6.1 6-Hydroxymellein
   - 6.2 Chalcones
   - 6.3 Stilbenes
   - 6.4 Other metabolites

7. **Fungi**
   - 7.1 6-Methylsalicyclic acid
   - 7.2 Aflatoxin
     - 7.2.1 Introduction
     - 7.2.2 Regulation
     - 7.2.3 Hexanoate synthetase
     - 7.2.4 Norsolorinic acid synthetase
     - 7.2.5 Averantin

---

### Biosynthetic Symbols Used Throughout This Series of Reviews

- Isolated methyl carbon from acetate (not now thought to be part of an intact acetate unit)
- Isolated carbonyl carbon from acetate (not now thought to be part of an intact acetate unit)
- Acetate unit, intact
- Intact acetate unit, including C-O bond from [1-¹³C, ¹⁸O] acetate
- Propionate unit, intact
- Intact propionate unit, including C-O bond from [1-¹³C, ¹⁸O] propionate
- Butanoate unit, intact
- 3-Methylbutanoate unit, intact
- Malonate unit, intact
- Succinate unit, intact
- Glycerol unit, intact
- Methyl carbon from [¹³CH₃] methionine via SAM
- From atmospheric dioxygen
- Hydride from NAD(P) H

---

**Fig. 1** Biosynthetic symbols used throughout this series of reviews.

Nat. Prod. Rep., 1999, 16, 425–484

of serine-36 to alanine. Many metabolites are abbreviated, with a capital followed by two or three lower case letters *e.g.* for tetracenomycin ‘Tcm’. Individual metabolites would be the abbreviation followed by a space *e.g.* Tcm F1, except for those remaining metabolites with fully capitalised abbreviations such as ‘SEK4’. Genes are lower case and italicised, *e.g.* *tcmN* and a series of *tcm* genes can be abbreviated to *e.g.* *tcmPQRS*. The protein product of that gene would be TcmN (no intervening space to distinguish from a metabolite) and a combination of proteins TcmPQRS. Many regions of a genome were first located by having a common phenotype in mutant studies, and labelled *e.g.* the ‘actI region’. Subsequently, several open reading frames may be located within this region, so the individual genes would be labelled *actI-ORF1; actI-ORF2 etc.* It is unfortunate that throughout the literature a very wide, confusing and at times bewildering array of conventions are employed. The author apologises for concentrating upon formation of the aglycones with less emphasis on the formation and attachment of sugars which is ironic as the biological importance of many polyketide aglycones often seems to be as a template for the rigid positioning of sugars in 3D, narrowly defining biological activity. For the reader’s convenience, the named author in the text is usually the author for correspondence, rather than always the first named author, and does not imply seniority or precedence.

Aspects of this rapidly moving area have been widely reviewed. In a seminal article, Hopwood has extensively surveyed the genetic contributions to understanding polyketide biosynthesis; ${}^{6}$ Khosla *et al.* have reviewed the chemistry and biology of fatty acids, polyketide and peptide biosynthesis; ${}^{7}$ Simpson has reviewed isotopic methods in secondary metabolic pathways; ${}^{8}$ Hutchinson *et al.* have comprehensively assessed gene manipulation of polyketide synthases; ${}^{9}$ Staunton and Wilkinson have extensively reviewed the biosynthesis of aliphatic polyketides; ${}^{10}$ Hershberger has produced an overview of the metabolic engineering of polyketide biosynthesis; ${}^{11}$ Cane has overviewed whether polyketide biosynthesis is molecular recognition or genetic programming; ${}^{12}$ Khosla and Tsoi have published an overview of combinatorial biosynthesis of ‘unnatural’ natural products; ${}^{13}$ Mann an overview of the rules for the manipulation of polyketides; ${}^{14}$ Simpson an overview of polyketide biosynthesis; ${}^{15}$ Rohr an overview of combinatorial biosynthesis; ${}^{16}$ Leadlay has overviewed combinatorial approaches to polyketide biosynthesis; ${}^{17}$ Verdine has overviewed the combinatorial chemistry of nature; ${}^{18}$ Khosla and Zawada have published an overview of the generation of polyketide libraries; ${}^{19}$ Omura and Nakagawa the biosynthesis of bioactive microbial metabolites and the production of hybrid compounds; ${}^{20}$ Kleinkauf and von Döhren have overviewed the link between peptide and polyketide biosynthesis; ${}^{21}$ Khosla *et al.* have overviewed evolutionarily guided enzyme design; ${}^{22}$ and Khosla and Fu have examined the antibiotic activity of polyketide products derived from combinatorial biosynthesis—the implications for directed evolution. ${}^{23}$

Abbreviations used in this series of fatty acid or polyketide biosynthesis reviews: A., Aspergillus; aa, amino acids; ACC, acetyl CoA carboxylase; ACP, acyl carrier protein; ACPS, holoacyl carrier protein synthase; Act, actinorhodin; ACV, α-aminoadipoylcysteinylvalinyl; Afl, aflatoxin; Akn, aklavinone; Aloe, aloesaponarin II; apo-, without prosthetic group; ARO, aromatase; AROab, benzoisochromanequinone type aromatase/didomain aromatase; AROc, Tcm type aromatase/monodomain aromatase; AS, acyl CoA synthetase; AT, acyl CoA ACP transacylase (acyl transferase); AcT, acetyl transferase; ATCC, American Type Culture Collection; B., Bacillus; BCCP, biotin carboxyl carrier protein; Bmt, (2S,3R,4R,6E)-2-amino-3-hydroxy-4-methylcloct-6-enoic acid; CHS, chalcone synthase; CH999, a strain of *S. coelicolor* A3(2) in which the Act genes have been removed; CI, chemical ionisation (mass spectros-copy); CLF, chain length factor or KSβ; CoA, coenzyme A;

CYC, cyclase; CYC1, first cyclase/ring one; Cyt, cytochrome; Da, dalton; Dau, daunorubicin (daunomycin) also Dnm and Dnr; ΔKR 5-oxoacyl ACP reductase; DMAC, 3,8-dihydroxy-1-methylenanthraquinone-2-carboxylic acid; Dnm, daunorubicin (daunomycin); Dnr, alternative abbrev. for daunorubicin (daunomycin); Dps, doxorubicin (adriamycin); Dox, alternative abbrev. for doxorubicin; DTX, a dinoflagellate toxin; E., Escherichia; Eco, *E. coli* proteins; Elm, elloramycin; ESMS, electrospray mass spectrometry; Ery, erythromycin; Fab, fatty acid biosynthesis proteins; Fren, frenolicin; Gra, granaticin; Gris, griseusin; HDDH, (3R)-hydroxydecanoyl ACP dehydratase; holo-, with prosthetic group *e.g.* holo-ACP; Jad, jadomycin; KAS, 3-oxoacyl ACP synthase (3-ketoacyl ACP synthase), also called KSα; KASIII, fatty acid KAS that catalyses the first condensation; kDa, kilodalton; KR, 3-oxoacyl ACP reductase (3-ketoacyl ACP reductase); KR1, ketoreductase in ring one, *M.*, *Mycobacterium*; KSα, ‘KAS’ (aromatic polyketides); KSβ, ‘CLF’ (aromatic polyketides); MCoA, malonyl CoA; MeT, methyl transferase *via* SAM; MMCoA, methylmalonyl CoA; MPT, malonyl/palmitoyl CoA ACP transacylase; MS, mass spectrometry; 6MSAS, 6-methylsalicyclic acid synthase; MT, malonyl CoA ACP transacylase; Mtm, mithramycin; NAC, *N*-acetyl cysteamine (*N*-acetyl-2-aminoethanethiol); NOE, nuclear Overhauser enhancement (NMR); Nsa, norsolorinic acid; ORF, open reading frame; OOX, oxidation of methylene to ketone in *e.g.* anthraquinone formation; Otc, oxytetracycline; OX, oxidation, *e.g.* ArH to ArOH; ppb, parts per billion (NMR isotope shift); PHA, polyhydroxyalkanoate; PHB, polyhydroxybutanoate; PK, polyketide; PKS, polyketide synthase; PMSF, phenylmethanesulfonyl fluoride (serine protease inhibitor); PrT, propionyl transferase; REG, regulatory protein/gene; RES, resistance protein/gene; *S.*, *Streptomyces*; SAM, *S*-adenosylmethionine; SDS, sodium dodecasulfate (gel electrophoresis); SEFT, spin echo Fourier transform NMR; *sno*, nogalamycin producing genes from *S. nogalater*; Stc, sterigmatocystin; STS, stilbene synthase; TAL, triacetic acid lactone; Tcm, tetracenomycin; TE, Acyl ACP thioesterase; Tü, strain culture collection; Urd, urdamycin; Vbs, versiconal B synthase; Ver, versicolorin; Whi, a grey spore pigment of unknown structure in *S. coelicolor*; X., *Xylaria.*

### 1.1 Summary of recent highlights†

Fifteen years ago, the biosynthesis of polyketides could have been summarised as a little understood black box ‘polyketide synthase’ assembling the ‘PKS product’ in a manner presumed to resemble fatty acid biosynthesis, followed by a series of discrete ‘post PKS’ or tailoring enzymes that mostly consisted of *O*-methylations, cytochrome P-450 oxidations and glycosylations, with the occasional rearrangement. The location of a Type II aromatic PKS gene cluster by Hopwood in 1984 ${}^{24}$ first led to elucidation of the putative function and role of individual cluster components, then the addition of heterologous additional components led to the production of modified ‘new’ metabolites, and there is now the widespread ‘mixing and matching’ of PKS biosynthetic genes from different metabolites by Khosla, Hutchinson, Hopwood and others. Whilst this gene mixing technology is only able to achieve the chemically rational, it has led to the production of many new metabolites with novel carbon skeleta, and some understanding of the role and function of the individual proteins involved. For example, whilst the CLF has a major role to play in determining chain length, the presence of other proteins can alter chain length, and presumably can play a part in determining chain length. Thus the function of many of these proteins is context dependent,

---

† References in main body of text.
depending upon the presence of other components and in reality, these proteins assemble into intimate complexes in which myriad protein–protein interactions are vital for proper function, and within each metabolite system, the components have evolved within that particular context to achieve maximum production of a single metabolite. Substitution of proteins from closely related systems usually leads to less efficient production and low level production of alternatives. The detailed molecular basis is still very poorly understood, but the first rays of light are being shed upon the molecular role of the Type II PK ACPs by Simpson, where enolate stabilising residues may have been identified. Simpson has found that these ACPs are quite capable of catalysing their own malonation, though it remains unclear whether they self-malonate *in vivo*, or whether the FAS MT performs this role. In addition, following conformational change, these ACPs can catalyse malonate decarboxylation, presumably as part of the overall condensation reaction. The full cluster responsible for the production of granaticin has just been published, which will enable overexpression of sugar assembly and regulatory export enzymes, and how they all interact with each other in multienzyme complexes, thus enabling a greater understanding of the ‘whole picture’, rather than just the PK assembly or individual proteins to be examined. One of the great challenges in the near future will be how all the proteins intimately interact and work together as a whole unit, rather than just understanding the properties of isolated overexpressed non-post-translationally modified proteins.

A large part of the daunorubicin cluster has now been characterised, though there remains uncertainty as to what part determines propionate as starter unit. The role of several components of the angucycline cluster has been examined in detail, but it remains unclear what determines the angular nature of the ring cyclisation. The monodomain aromatase TcmN is now better understood, though the molecular basis of the action of TcmJ remains unclear. It may ‘glue’ together other components in the correct orientation to maximise yield and minimise aberrant products. The aureolic acids have been considered to be assembled in a tetracenomycin like manner. However, a recent paper by Rohr, Salas *et al.* suggests a tetracycline like assembly.

The actinomycete macrolides will be discussed in a later review.

Marine polyketides such as okadaic acid and the amphidino-lides continue to present a challenge. Their labelling patterns do not conform to current understanding, and a novel pathway not yet well understood must be operating.

The gene for the fungal metabolite 6MSAS has been functionally expressed in a Streptomyces by removing the exons in the sequence, possibly providing an evolutionary horizontal link between the eukaryotic Type I process with the less understood prokaryotic Type I aromatic polyketide synthases. Whilst a few fungal PKS synthase genes have now been reported, the molecular basis for their assembly remains poorly understood compared with that for the prokaryotic systems. There has been an increased understanding of non-ribosomal peptide synthases in recent years, and an ever stronger connection between peptide synthases, polyketide synthases and FAS systems is becoming evident. There still remains a large scope for increased communication and cross fertilisation of ideas between the historically disparate scientific communities that study these three areas. There is a need for a collective term for metabolites derived from all these closely related systems. This author would like to suggest “The Polyacylides”.

2 Gram negative bacteria

The myxobacteria are ‘gliding’ Gram negative soil bacteria with a complex life cycle, multicellular cytodifferentiation, high G + C DNA base pair content, and producers of a wide range of unusual polyketide metabolites many of which have been studied by Höfle *et al.*²⁵ The soraphens are a family of antifungal polyketides produced by *Sorangium cellulosum* So ce26 that inhibit fungal acetyl CoA carboxylase and have promise in the control of plant fungal pathogens.²⁶ Soraphen A1 has an 18-membered macrocycle with a characteristic phenyl ring (benzoyl CoA starter unit) at the starter position.²⁷ Schupp *et al.* have exploited the high G + C DNA base pair content (*ca*. 70%) present in the myxobacteria in common with the Gram positive streptomycetes in which polyketide genetic studies are well advanced.²⁸a Using Southern hybridisation to a *Streptomyces violaceoruber* granaticin (*gral*) gene Schupp located and cloned a 40 kbp region of DNA, 32 kbp of which has a role in soraphen biosynthesis. Analysis suggests part of a single continuous ORF that continues beyond the cloned fragment, with homology to *eryAIII KR*, ACP, KAS, AT, DH, and second KR and ACP domains. The close similarity, not only in the DNA sequence, but also the order of the domains, found between the genes in two quite unrelated bacteria, unrelated except in their high G + C DNA content, very strongly indicates that horizontal transmission or exchange of these genes has occurred, which has allowed dispersal of the complex macrolide PKS genes found in the streptomycetes into other bacterial families with high G+C content. Soraphen A1 has a benzoyl CoA starter unit, with three acetate and three propionate extender units, but the vicinal hydroxy carbons at C-3,4 and C-11,12 were not labelled by acetate or propionate. In a recent paper, Hill, Harris and Siskos have incorporated two carbons from glycerol into the mid-chain regions C-3,4 and C-11,12 of soraphen A1 at *ca.* ten-fold enrichment levels, and show that it is the *pro-(S)*-hydroxymethyl carbon of glycerol that is lost.²⁸b One hydrogen of the *pro-(R)*-hydroxymethyl group is retained. Glycerol incorporation into C₂ units have been examined now in the *Sorangium*, Streptomycetes (fluoroacetate) and *Macrophaoma*, and in all three cases, it is the *pro-(S)* carbon that is metabolised (lost). Hill speculates that the *pro-(S)*-hydroxymethyl group can be oxidised and used in decarboxylative condensation with tri- and heptaketide intermediates. The retention of a single hydrogen from the *pro-(R)*-hydroxymethyl group at C-11 excludes oxidation to a CoA thioester or hydroxymalonate (unless the label has been redelivered by the cofactors involved), thus this group must remain as an (activated) alcohol, or oxidised to an aldehyde prior to incorporation.

Schummer *et al.* have isolated a boron containing macrodiolide tartrolon B2 from *Sorangium cellulosum* (Myxobacteria).²⁹ Its absolute configuration was established by X-ray crystallography to be related to that of the related compounds boromycin, aplasmomycin and borophycin that are found in Streptomycetes and cyanobacteria (Fig. 2).¹ Feeding experiments suggested a glycerol derived starter unit followed by nine acetates, with pendant methyl groups derived from methionine. However, the feeding experiments with

![Chemical Structures]

Nat. Prod. Rep., 1999, 16, 425–484 427
bond coupling could not be detected between C-19 and C-21, which would have been strong supportive evidence for intact glycerol incorporation, so the authors degraded the [1,3-<sup>13</sup>C<sub>2</sub>]glycerol labelled tartrolon B to methyl 4-oxopentanoate and showed by a greatly enhanced (9%) M+2 peak in the MS that the two labels from glycerol were indeed incorporated into the same molecule of tartrolon B.

Höfle *et al.* have examined the biosynthesis of the ripostatin A 3 from *Sorangium cellulosum* So ce377.<sup>30</sup> The hemiacetal at

![Chemical Structure](https://i.imgur.com/structure1.png)

Tartrolon B  
2  

Glycerol  

Tartrolon B 2  
(Myxobacteria)  

Boromycin  
(Streptomycetes)  

Aplasmomycin  
(Streptomycetes)  

Borophycin  
(Cyanobacteria)  

Fig. 2 Common stereochemical and biosynthetic relationships between myxobacterial, streptomyces and cyanobacterial metabolites (for full structures, see ref. 1).

[1,3-<sup>13</sup>C<sub>2</sub>]glycerol, whilst giving the expected label at C-19 and 21 (8–9%), also gave very high levels of incorporation at acetate methyl derived carbons (6–8%), but low levels at acetate carbonyl derived carbons (1–2%), suggesting high levels of conversion of the labelled glycerol into [2-<sup>13</sup>C]acetate. Two

Ripostatin A  
3  

= [2-<sup>13</sup>C]Phenylalanine  

Ratjadone  
4  

C-15 can exist as the corresponding open chain form. Whilst [2-<sup>13</sup>C]phenylalanine was readily incorporated, labelled phenylacetate was not, suggesting that the free phenylacetic acid was not an intermediate, and there was no enzyme available to convert it to the CoA thioester. It is interesting to speculate whether the oxidative decarboxylation of phenylpyruvate could be associated with or even part of the first ‘polyketide’ multienzyme complex, directly resulting in an enzyme linked phenylacetyl starter ready for propionate extension. The first extender unit was propionate, followed by eight acetates, with an acetate derived pendant methyl at C-9, and acetate derived pendant acetate at C-3, these groups being attached in a manner reminiscent of marine metabolites such as the okadaic acid DTX-4 (vide infra).

Höfle *et al.* have also examined the biosynthesis of ratjadone 4 produced by *Sorangium cellulosum* So ce360.<sup>31</sup> In contrast to the above examples, this metabolite is assembled from acetate and propionate in a rather more ‘regular’ pathway, and does not incorporate methionine in any pendent methyl group. Höfle suggests that the ‘non-acetate’ hydroxy group arises from the attack of a C-21 hydroxy group on a C-17/C-18 epoxide, forming the oxane ring.

3 Gram positive bacteria: actinomycetes, Type II aromatic polyketides

3.1 Introduction

Traditionally classified as octaketides, such as actinorhodin, nonaketides such as the tetracyclines, dekaketides such as the anthracyclines and angucyclines, and a few dodecaketides (smaller tetraketide mono-aromatic metabolites from bacteria are now thought to be assembled by a Type I process, see Section 4). Previously, bacterial and eukaryotic, Type I and Type II, aromatic polyketides have often been grouped all together, or classified on the basis of their final chemical structure, or number of acetate units, rather than the mechanism of biosynthetic assembly, the molecular and enzymatic basis of which is only now being elucidated. This review attempts to arrange these metabolites on the basis of their (presumed) biosynthesis (Scheme 1).

In 1984, the genes responsible for the biosynthesis of actinorhodin were located by Hopwood and found to be arranged in a cluster.²⁴ Since then, a number of gene clusters responsible for the biosynthesis of a diverse range of these polyketides have now been located and analysed, the PKS regions of several clusters mentioned in this Section are compared in Fig. 3. The gene clusters often also contain post PKS activities such as cytochrome P-450 oxidations or glycosylation, as well as transcriptional activator or regulation (REG) genes (often product repressed) and self-resistance (RES) genes (often metabolite exportation, or DNA/ribosome protection genes). The relative arrangements of the genes for each aromatic polyketide can be compared to that of other aromatic polyketides, and the role each protein has in the assembly process can be inferred by targeted gene deletion, or insertion of the gene into other gene clusters, and most usefully, the assembly of whole, or part gene clusters (containing genes from one, or more polyketides) in *E. coli* or non-producing strains of Streptomyces, with examination of the resulting polyketides. Overexpression in *E. coli* to obtain protein for characterisation often requires modification of the first few triplet codons to more ‘*E. coli* friendly’ codons containing lower G + C levels to get efficient transcription. This is useful for obtaining individual proteins for structural analysis (*e.g.* ACPs), but less useful to produce novel metabolites from expression of (part) gene clusters as the overall metabolism, supply of precursors, regulatory systems, and posttranslational modification of proteins are quite different in *E. coli* to those for the Streptomycetes.

A more biologically relevant and efficient streptomycete expression system has been developed based upon a recombinant strain of *S. coelicolor* with a defined chromosomal deletion of the entire actinorhodin gene cluster (CH999), or systems based upon strains of *S. lividans* or *S. glaucescens* without all or part of their own PKS genes. Various genes from any PK biosynthetic cluster can now be inserted on a plasmid, seemingly in any desired order and combination, transformed into the non-producing host, and the resulting *in vivo* PK product, if any, obtained and identified. It was quickly found that to produce any polyketide at all, a ‘minimal PKS’ system containing an ACP, KAS and CLF was always required. For example, insertion of plasmid pSEK24 containing genes for *S. coelicolor* Act KAS, CLF and ACP into *S. coelicolor* CH999 gave products SEK45 and SEK4b6 (Scheme 2), containing the same length of chain as the ‘parent’ metabolite, and with the ‘first six-membered ring’ already formed, with the correct regiochemistry, ‘CYC1(7/12)’. Thus the ‘minimal PKS’ components are able to promote assembly of the correct length of polyketo chain, catalyse and direct the regiochemistry of the first cyclisation, or at least deter other non-enzymatic cyclisations from occurring. In addition, other aberrant non-enzymatic cyclisations have often occurred, due to the high reactivity of the resulting pendant polyketo side chains, and the absence of the proteins that would have directed the folding to only allow cyclisation of subsequent rings in the ‘native’ manner. As other genes were added to the ‘minimal PKS’ expression system, correspondingly modified ‘more advanced’ polyketide products were obtained, allowing a presumed function or role of the added gene to be proposed. Early speculation was that each component would have a single immutable function or role independent of context, allowing new structures to be designed and assembled like a ‘Lego’ set by mixing and matching genes with a simple set of rules. Unfortunately, many components have more than one role, and some have ‘context dependent behaviour’, in which the roles change depending on the presence or absence of other components, and each component certainly has an effect upon the efficiency of other components. It is too easy to assume that if addition of gene X to a minimal PKS causes a much higher level of cyclisation between C-7 and C-12, that the role of protein X includes promoting first ring cyclisation. However, in reality, protein X may only be intimately binding to the minimal PKS, affecting its conformation. The minimal PKS would have evolved to work most selectively and efficiently only in the presence of (and in close intimate association with) its other cluster components, so we should not be too surprised if they are intricately interdependent, and function at their best only when in bed with their own partners. However, an impressive wide range of new structures have been formed by expressing heterologous clusters, and a series of rules or guidelines assembled to predict the product from a certain combination of genes.

Most bacterial ‘aromatic polyketides’ are reduced where the growing acyl chain ‘turns the corner’ and is folded. The molecular reason for this is little understood, but most are reduced at the keto group nine carbons in from the final position of the thioester carbonyl, by a KR1(9). The frequently encountered exceptions are the tetracenomycins and aureolic acids, where no such reduction takes place. As it is now becoming clear that this reduction has an important role in the activity and role of later proteins, this review will attempt to separate the ‘KR1 reduced’ from the ‘unreduced’ systems. Unfortunately, the tetracenomycins and ‘C-9 reduced’ dekaketides are frequently both referred to as anthracycline antibiotics, despite having quite different acetate-folding patterns. This review will avoid this, referring to ‘the tetracenomycins’ as a class, and only using the term anthracycline to refer to doxorubicin *etc.* To highlight the biosynthetic difference between the reduced, and non-reduced systems, this review will refer to non-‘C-9 reduced’ dekaketides such as the tetracenomycins as 9,14-dekaketides (CYC1 occurring at C-9/14), and the anthracyclines (and others) as 7,9,12-dekaketides [CYC1(7/12)KR1(9)]. A similar system was introduced by Hopwood in a recent review.⁶ As pointed out by Rohr,³² when the chain length, position of the first reduction relative to thioester, and regiochemistry of the first cyclisation are considered, there are only a few common basic folding types, as illustrated in Scheme 1, that encompass most bacterial Type II aromatic polyketides. A recent paper suggests that the aureolic acids are 7,12-dekaketides, rather than 9,14-dekaketides as previously thought, so they are now in a separate section (3.7). The labelling of aromatases and cyclases can also be rather confusing, and is historically derived from the concept of each protein having a single easily defined role. Initially, ActVII was thought to cyclise and aromatise the first ring, and was called ‘cyclase’, whilst ActIV was thought to promote aromatisation of the second ring, and was called the ‘aromatase’. However, in 1994, the realisation that the aromatisation of mutactin was non-enzymatic, and that ActVII can promote the aromatisation of the first ring whilst ActIV then goes on to cyclise the second, has led to the reverse labelling since then. To add further confusion, the tetracenomycin protein TcmN, often just labelled ARO, has a quite different spectrum of functions and roles to ActVII ARO, primarily aromatising the second ring and may promote

Minimal PKS corresponds to any ACP, the KAS1/AT (KSα), and CLF (KSβ)

OCT minimal PKS  
NON minimal PKS  
DEC minimal PKS  

![Chemical Structure 1](chemical_structure_1.png)  
KR1(9)  
![Chemical Structure 2](chemical_structure_2.png)  
CYC1(7/12)  

7,9,12-Polyketides  
![Chemical Structure 3](chemical_structure_3.png)  

Dehydration (ARO1)  
CYC2(5/14)  
![Chemical Structure 4](chemical_structure_4.png)  

CYC3(3/16)  
CYC4(1/18)  
MeT(6)  
![Chemical Structure 5](chemical_structure_5.png)  
Tetracyclines  
7,9,12-Nonaketide  

CYC3(3/16)  
OOX(6)  
CYC4(2/19)  
CYC3(4/17)  
CYC4(2/20)  
OOX(6)  
CYC3(O-3/15)  
![Chemical Structure 6](chemical_structure_6.png)  
Anthracyclines  
7,9,12-Decaketide  

Benzoisochromanquinones  
7,9,12-Octaketide  
![Chemical Structure 7](chemical_structure_7.png)  

Angucyclines  
7,9,12-Decaketide (Benz[a]anthraquinones)  
Oxidative cleavage  
![Chemical Structure 8](chemical_structure_8.png)  
Kinamycins  
7,9,12-Decaketide  

DEC minimal PKS  
CYC1(9/14)  
Spontaneous ARO1  
![Chemical Structure 9](chemical_structure_9.png)  
9,14-Decaketides  
![Chemical Structure 10](chemical_structure_10.png)  

Tetracenomycins  
9,14-Decaketides  
Oxidative cleavage  
![Chemical Structure 11](chemical_structure_11.png)  
Aureolic acids  
7,12-Decaketides  
![Chemical Structure 12](chemical_structure_12.png)  

CYC2(5/14)  
CYC3(3/16)  
CYC4(1/18)  
7,12-Decaketides  
![Chemical Structure 13](chemical_structure_13.png)  

Scheme 1 Biosynthetic relationships between bacterial Type II aromatic polyketides.

430 Nat. Prod. Rep., 1999, 16, 425–484

some first ring cyclase activity and the C-terminal half has O-
methyltransferase activity. The ActVII ‘ARO’ has two homologous halves or domains, N-terminal (a) and C-terminal (b), thought to catalyse each of the two dehydrations required, so this review will label benzoisochromanequinone ‘ActVII like’ AROs as ‘AROb’s. A series of other proteins with aromatase activity only contain a single such domain, most homologous with the N-terminal half (a) of the ActVII didomain aromatase. These ‘TcmN like’ AROs will be labelled in this review as ‘AROc’s. This ‘ARO’ labelling remains despite the function being dehydration, and both types having some cyclase directing/promoting activity, especially of ring one. ‘ActIV

like’ proteins, often just labelled CYC will be referred to in this review as CYC2/3, though the role in cyclisation of the third ring is questionable. This at least distinguishes them from the ‘action’ when referring to cyclisation of ring two as CYC2, and aromatisation of ring two as ARO2. The few metabolites in which aromatisation of the first ring does not occur at some point subsequent to CYC1 (mutactin 7, RM20b 27 and RM20c 28) are distinguished by an asterisk e.g. CYC1*(7/12).

There remains confusion over labelling the aromatases/cyclases, as some function separately, sometimes these functions are combined, and as both activities involve dehydrations (CYC being considered as aldol plus dehydration) a different all

---

**Fig. 3** Selected PK gene clusters for Type II aromatic polyketides.

Nat. Prod. Rep., 1999, 16, 425–484 431

![Image Description](image.png)

**Scheme 2** Biosynthesis of actinorhodin and related benzoisochromanquinones.

432 Nat. Prod. Rep., 1999, 16, 425–484

encompassing system of labelling, such as dehydratases with reference to carbon label or ring, may be preferred in the near future.

In the C-9 reduced systems, the minimal PKS controlled first cyclisation (aldol condensation) usually occurs between carbons 7 and 12 i.e. CYC1(7/12) to give SEK45 (in the Act minimal PKS, an alternative presumed non-enzymatic first cyclisation also occurs, CYC1(10/15) to give SEK4b6). This is then followed by an enzymatically triggered aromatisation or dehydration (ARO1) to form the first aromatic ring. In these KR1(9)CYC1(7/12) systems, the first cyclisation can be partially regiochemically directed and catalysed by just the minimal PKS. However, the trifunctional enzyme traditionally labelled ‘ARO’ (ActVII) and in this review referred to as ‘AROab’, is also thought to help promote and direct CYC1, aromatise the first ring ARO1 (acting as a dehydratase) as well as aromatise the second ring (but only after the action of a cyclase labelled CYC2/3). These processes, involving the minimal PKS, KR1, AROab and CYC2/3 will be referred to as the ‘early PKS’ system, with, OOX(6), CYC4 etc. to form the first readily isolable ‘wild type’ intermediates as ‘later PKS’, and subsequent steps (e.g. cytochrome P-450 oxidations, glycosyl transfers) as ‘post PKS’. The vast majority of metabolites in this section are KR1(9)CYC1(7/12), varying only in chain length (usually 16/18/20), starter unit (‘X’), and the later steps (Scheme 1). These labels have been abbreviated to e.g. 7,9,12-octaketides, and 7,9,12-decaketides, with non-reduced systems such as tetracenomycin labelled as 9,14-decaketides.

In the non-reduced systems, such as tetracenomycin, this first aromatisation is thought to occur spontaneously after ring closure without the need for an ARO enzyme activity, and again highlights the need to separate such systems from the corresponding reduced metabolites. The multifunctional ‘AROc’ protein responsible for CYC1/ARO1 (TcmN) affects the regiochemistry of the first cyclisation, and is thought to play a role in later cyclisations (CYC2 and 3), as well as having an O-MeT function. Any ‘role and function’ conclusions need to be treated with some caution as only a limited number of products may yet have been characterised from any one system, and many reactions can occur slowly without specific catalysis. If a rapid catalysed step is removed, other slow non-enzymatic reactions, not previously observed, can now appear dominant, and to be catalysed by the added protein.

Subsequent ‘later PKS’ enzymes, that convert the unstable ‘early PKS’ intermediate into an easily isolable product, seem to perform a wider variety of reactions than previously thought, possibly including oxidation to form quinoid systems, SAM methylation, and O-cyclisation to form pyran rings. [The second cyclase, that may be involved in the cyclase of both the second and third ring (CYC2/3) may be as expected performed by a discrete enzyme, but as many metabolites are quinoid, a double oxidation of the methylene to a ketone (OOX) may now occur either prior to CYC3 (triggering CYC3?) simultaneously or shortly afterwards]. Methyl transferases, isoprenyl transferases etc. may be an integral part of these gene clusters, and such reactions may be occurring before the full ring system is formed by an e.g. CYC4. Many of these final cyclases presumably attack a carbonyl with a carbonyl ketone oxygen or hydroxy, to give pryan rings. Such cyclisations by e.g. the oxygen of C-2 onto C-19 are classified in the tables (vide infra) as CYC4(O-2/19), and loss of C-1 as CO₂ (from thioester carbonyl carbon) as —CO2(1).

Combinations of genes from different organisms can be inserted into non-producing strains e.g. S. coelicolor CH999, to analyse their compatibility, and to analyse the effect of mixing e.g. octaketide genes with decaketide genes. Hopwood has recently published an excellent review on the genetic contribution to the understanding of polyketide synthases.⁶ Thus a whole new range of polyketides can now be assembled in vivo

using ‘Combinatorial Biosynthesis’, though it is always salutary to be reminded that molecular biology can only achieve the chemically rational! Many ‘unnatural’ combinations produce products in relatively low yield, and substantial extensive mutagenesis (or forced evolution) may be required to maximise the yields, as would happen in nature after such ‘horizontal gene movement’ to optimise the protein-protein interactions so that only a single product is formed. In addition, the function and role of many proteins appears to vary depending upon the presence of other proteins—‘context dependent behaviour’. However, based upon what has been shown in the last few years to be possible to achieve with various gene combinations, it is a real puzzle why nature makes so few different ‘core’ polyketide structures, or perhaps they are still ‘out there’ just waiting to be found.

### 3.2 Studies on role of PKS cluster components—hybrid clusters

A wide range of components of PKS clusters have now been inserted, in various combinations, into strains of *S. coelicolor* (CH999), *S. lividans* (e.g. TK24 or 1327) or *S. glaucescens* (e.g. WHM1077) in which the hosts PKS genes have been removed, and the resulting products analysed in order to help determine the role of each component. The clusters so far used include those from actinorhodin and grey spore pigment (Act and WhiE, both from *S. coelicolor* A3(2)), frenolicin (Fren, from *S. roseofulvus*), granaticin (Gra, from *S. violaceoruber*), griseusin (Gris, from *S. griseus*—stereostructure shown in some literature is inconsistent), tetracenomycin (Tcm, from *S. glaucescens*), daunorubicin (Dps, from *S. peucetius*) and jadomycin (Jad, from *S. venezuelae*). Cluster components are tabulated under:

- **KAS** KSα, includes an AT activity; e.g. ActI-ORF1, TcmK, DpsA, JadA;
- **CLF** KSβ; e.g. ActI-ORF2, TcmL, DpsB and JadB;
- **ACP** e.g. ActI-ORF3, TcmM and JadC;
- **KR1** KR in ring one; e.g. ActIII, DpsE and JadE. In this review, KR3(18) refers to reduction in the third ring on carbon 18 counted from thioester carbonyl;
- **AROab** e.g. ActVII, DpsF and JadD Aromatase, ‘reduced’ first ring aromatase ARO1 (and CYC1) and may promote aromatisation of second ring (ARO2) after CYC2 (catalysed by a CYC2/3); ActVII, benzoisochromanequinone first ring aromatase, functions to didehydrate the first ring after reduction by KR1, promotes CYC1(7/12) regiochemistry when added to min PKS. Also called ‘didomain aromatase’;
- **AROc** e.g. TcmN and WhiE. Second ring aromatase and second ring cyclase in ‘unreduced systems’, typified by N-terminal half of TcmN and all WhiE-ORFVI. Controls regiochemistry of unreduced systems CYC1 from (7/12) with min PKS only to (9/14). Designed for absence of KR and has little effect on KR1 reduced systems;
- **CYC2/3** Cyclase; e.g. ActIV. Promotes aldol condensation to form second ring. TcmN can catalyse CYC2 and CYC3 and thus can be referred to as CYC2/3 and placed in this column. Usually assumed to involve aldol between carbanion and a carbonyl, but perhaps also catalyses mechanistically similar alcohol attack or carbonyl oxygen attach to form pyrones, as in the third ring of actinorhodin.
- Other component used: TcmJ, unknown function, but can greatly enhance levels of production.

Thus the Act minimal PKS assembles SEK45 in which the correct chain length has been formed, the first ring has been correctly cyclised, and a non-enzymatic dehydration/aromatization of ring one has occurred (Scheme 2, Table 1). Later studies also located SEK4b6, of correct chain length, but aberrant (non-enzymic?) CYC1, showing that the min PKS does not have complete control over the regiochemistry of CYC1. Addition of AROab to min PKS has no effect, thus AROab does not recognise the unreduced (no KR1) octaketide product of the min

PKS. Addition of KR1 instead results in mutactin 7, in which the non-enzymatic aromatisation of ring one has not now occurred. Thus, whilst the minimal PKS can promote the correct CYC1, it does not aromatise the reduced first ring. Addition of both KR1 and AROab now gives SEK348 (and its dehydration product SEK34b9), in which the first ring has aromatised, affecting the regiochemistry of non-enzymatic CYC2. Addition of KR1, AROab and CYC2/3 gives DMAC10 (and its decarboxylated product aloesaponarin II, 'Aloe'11) in which the action of the CYC2/3 has allowed (caused?) the formation and aromatisation of an aberrant carbocyclic CYC3(2/15), different to that occurring in actinorhodin itself when the 'later PKS' genes are present. Also, an unexplained oxidation has occurred, OOX(6), that may be aerial non-enzymatic, or caused by unrelated oxidative enzymes in the microorganism, or may be an integral part of the PKS triggering or promoting subsequent regiospecific cyclisations to occur. However, it has been suggested that in the wild type, this oxidation is caused subsequently by various ORFs in the ActVA region. Addition of KR1 (actIII), AROab (actVII), CYC2/3 (actIV), actVI, actVA, and actVB is required to obtain actinorhodin 14.

The frenolicin-producing organism (S. roseofulvus) produces both the octaketide nanaomycin 18 and frenolicin 17, which may be an octaketide with a butyrate starter, or more likely, a nonaketide. A common PKS is thought to assemble both the octaketide and nonaketide precursors, the synthase having chain length variability. Fren minimal PKS (FrenKAS, FrenCLF and ActACP) produces only octaketide products (no equivalent nonaketide yet located, but could yet be present), whilst the addition of ActKR1 gives rise to a mixture of octaketides and nonaketides (Scheme 3, Table 2). Addition of TcmN to minimal Fren PKS changes both chain length (to only nonaketides) and regiochemistry of the first CYC1 to (9/14) to give PK819. This implies the intimate interaction of cluster components influencing each other's roles—'context-dependent behaviour'. Addition of GrisAROab to minimal FrenPKS and ActKR1 gives

SEK348 (the KR1 reduced analogue of SEK45) by presumably promoting first-ring aromatisation to the detriment of mutactin formation with its aberrant CYC2(6/15). However, ActCYC2/3 can direct and control the action of both FrenAROab and GrisAROab to give CYC2(5/14) products DMAC10 and SEK2620. DMAC and the aberrant RM1821 and RM18b22 are obtained from the use of ActKR1 and ActAROab with Fren minimal PKS. Thus, once again, the action of the later enzymes are affected by the exact nature of the minimal PKS, rather than just its products, strongly implying a close association and co-dependence of all these PKS components.

The Tcm'minPKS' produces only the correct chain length, but the predominant product SEK1523hasCYC1(7/12), as is found in most other 'minPKS' systems, but is aberrant for Tcm (Table 3). This suggests that the 'minPKS' is similar in its CYC1 tendency to other PKSs, and that it is the later components (TcmN) that redirects the CYC1 to occur (9/14) rather than (7/12). The addition of TcmJ has no effect upon product structure, but an enormous effect on yield. TcmN is multifunctional, promoting ARO2, the cyclisation of the second and perhaps the third ring, as well as possessing an O-methyltransferase activity. The unreduced chain does not need catalysis to dehydrate and undergo ARO1. Whilst TcmJKLMN in S.lividans gives TcmF2, in CH999it assembles RM8025 and RM80b26 which have aberrant third-ring cyclisations (Scheme 4).

The major product of the Tcm'minPKS'inCH999 is again the incorrectly folded CYC1(7/12) product SEK1523, with only a trace of the CYC1(9/14) product SEK15b24 (Table 4). Addition of TcmN entirely redirects the first cyclisation to CYC1(9/14) (Scheme 4) to give RM8025 and RM80b26. Aromatisation of the first unreduced ring appears spontaneous. CYC2 of the CYC1(7/12) pathway appears entirely non-enzymatic, leading to SEK15. However, CYC2 of the CYC1(9/14) pathway appears to occur at CYC2(7/16) whether or not TcmN is present. However, in the absence of TcmN and the

---

**Table 1** Act genes expressed in CH999

| KAS | CLF | ACP | KR1 | AROab | CYC2/3 | Product | Ref |
|-----|-----|-----|-----|-------|--------|---------|-----|
| Act | Act | Act |     |       |        | SEK4    | 33  |
|     |     |     |     |       |        | SEK4b   | 34,35 |
| Act | Act | Act | Act |       |        | SEK4    | 33  |
| Act | Act | Act | Act |       |        | Mutactin | 33  |
| Act | Act | Act | Act | Act   |        | SEK34/34b | 36  |
| Act | Act | Act | Act | Act   | Act    | DMAC/Aloe | 37  |

---

**Table 2** Frenolicin KAS and CLFinCH999

| KAS | CLF | ACP | KR1 | AROab or AROc | CYC2/3 | Product      | Ref |
|-----|-----|-----|-----|---------------|--------|--------------|-----|
| Fren | Fren | Act |     |               |        | SEK4         | 34  |
|      |      |     |     |               |        | SEK4b        |     |
| Fren | Fren | Act | Act |               |        | Mutactin (70%) | 33  |
|      |      |     |     |               |        | RM18 (10%)    | 34  |
|      |      |     |     |               |        | RM18b (20%)   |     |
| Fren | Fren | Act | TcmJ |              |        | SEK4         | 34  |
|      |      |     |     |               |        | SEK4b        |     |
| Fren | Fren | Act | TcmN and TcmJ |    |        | PK8          | 34  |
| Fren | Fren | Act | Act |              |        | DMAC/Aloe    | 38  |
|      |      |     |     |              |        | RM18         |     |
|      |      |     |     |              |        | RM18b        |     |
| Fren | Fren | Act | Act | Act          |        | DMAC/Aloe    | 38  |
|      |      |     |     |              |        | RM18         |     |
|      |      |     |     |              |        | RM18b        |     |
| Fren | Fren | Act | Act | Gris         |        | SEK34        | 39  |
| Fren | Fren | Act | Act | Gris         | Act    | DMAC         | 39  |
|      |      |     |     |              |        | SEK26        |     |
| Fren | Fren | Act | Act | Fren         | Act    | DMAC         | 34  |
|      |      |     |     |              |        | SEK26        | 39  |

---

434 Nat.Prod.Rep., 1999, 16, 425–484

Fren KAS  
Fren CLF  
Act ACP  
'Min Fren'  

![Chemical Structure 1]

and

![Chemical Structure 2]  
SEK4  
5  

![Chemical Structure 3]  
SEK4b  
6  

'Min Fren'  
TcmN  

![Chemical Structure 4]

![Chemical Structure 5]  
PK8  
19  

'Min Fren'  
Act KR  
Gris AROab  

![Chemical Structure 6]  
SEK34  
8  

'Min Fren'  
Act KR  

![Chemical Structure 7]  
Mutactin  
6  

'Min Fren', Act KR  
Fren AROab or Gris AROab  
Act CYC2/3  

![Chemical Structure 8]  
DMAC  
10  

and  

![Chemical Structure 9]  
SEK26  
20  

'Min Fren'  
Act KR  

![Chemical Structure 10]  

'Min Fren' Act KR  
or  
'Min Fren', Act KR  
Act AROab  

![Chemical Structure 11]  
RM18  
21  

'Min Fren', Act KR  
or  
'Min Fren', Act KR  
Act AROab  

![Chemical Structure 12]  
RM18b  
22  

Scheme 3

Nat. Prod. Rep., 1999, 16, 425–484  435

consequent rapid second ring aromatisation, aberrant CYC 3 and quinone formation seems to occur to give SEK15b; whilst with the rapid TcmN catalysed ARO 2 occurring, an alternative CYC 3 occurs resulting in RM80 25 and RM80b 26 (in S. lividans, TcmF2 34 is produced). ActACP can substitute for TcmACP. ActKR1(9) can reduce a decaketide precursor to

give RM20b (Scheme 5), however, TcmN will not recognise the KR1(9) reduced decaketide.

In the absence of other components, Fren minimal PKS was found to produce only octaketide products (Scheme 3), with no nonaketide detectable. As expected, Act min PKS produces octaketides, and Tcm min PKS decaketides (Table 5).

---

**Table 3** Tcm components in *S. lividans* or *glaucescens*

| KAS | CLF | ACP | TcmN | TcmJ | Product(s) | Ref |
| --- | --- | --- | --- | --- | --- | --- |
| Tcm | Tcm | Tcm |  |  | SEK15 (1 mg L⁻¹) | DEC:Ac/CYC1(7/12)CYC2(14/19)CYC3(O-1/5) |
|  |  |  |  |  | SEK15b (trace) | 41 |
|  |  |  | TcmN |  | TcmF2 (trace) | DEC:Ac/CYC1(9/14)CYC2(7/16)CYC3(6/19)CYC4(O-1/5) |
|  |  |  |  | TcmJ | SEK15 (5 mg L⁻¹) | 40 |
|  |  |  |  |  | SEK15b (<1 mg L⁻¹) | DEC:Ac/CYC1(7/12)CYC2(14/19)CYC3(O-1/5) |
|  |  |  | TcmN | TcmJ | TcmF2 (50 mg L⁻¹) | 42 |

---

**Tcm 'Min PKS'**
(major)
Folding regioselectively mainly at C-9
in absence of TcmN

**Tcm 'Min PKS'**
(minor)
+ TcmN
Folding regiospecificity determined by TcmN at C-11

![Chemical Structures](chemical_structures.png)

---

**Scheme 4** Control and direction of early steps in Tcm biosynthesis (in *S. coelicolor* CH999).

**436** Nat. Prod. Rep., 1999, 16, 425–484

Mixing Act and Tcm KAS/CLF gave little, if any, product (Table 6). However, in other contexts, they have been found compatible (vide infra).

Substitution of ACPs has no observable effect. Act ACP has also been replaced by Gra, Fren, Otc and Tcm ACPs, as well as the Saccharopolyspora erythraea FAS ACP and a hybrid Act/Tcm ACP (derived by gene fusion) to give functional polyketide synthases.⁴⁶ Act and Gra KAS and CLF can be interchanged without effect. Interchanging Fren and Tcm KAS and CLF gives no products, as found with Act KAS and Tcm CLF (Table 7). However, Tcm KAS and Act CLF gives octaketide products only, which suggested that the CLF is the primary chain length determining factor in this case (vide infra). Act KAS and Fren CLF give no products, but Fren KAS and Act CLF with other Act components gave octaketide products. Act KAS and Tcm CLF are also incompatible when expressed in S. lividans or glaucescens (Table 8), but Tcm KAS

and Act CLF results in a mixture of octaketide and dekaketide products, unlike the case observed in CH999 where only octaketide products were found (Table 7). The substitution of Act CLF for Tcm CLF in Tcm PKS affects chain length, but not regiochemistry of CYC1, causing production of some octaketide product UWM133, a normal product of ‘minActPKS’ and TcmJN (Scheme 6, Table 8).

Whilst Fren min PKS is only known to produce octaketides, Fren min PKS with Act KR1 produces both octaketides and nonaketides (Table 9). Thus the CLF cannot be exclusively responsible for the chain length, the presence of other components, and their possible effect on the CLF, have a role to play in determining chain length (Scheme 3). Act KR1 can reduce both octaketides, nonaketides and dekaketides, all at C-9 as counted from the thioester, and appears to prevent the nonenzymatic CYC1(10/15) octaketide and CYC1(9/14) dekaketide cyclisations occurring (Table 9). Presumably, the stabilis-

---

**Table 4** Tcm KAS/CLF in CH999 (Scheme 4)

| KAS | CLF | ACP | KR1 | TcmJN | Product(s)       | Ref |
|-----|-----|-----|-----|-------|------------------|-----|
| Tcm | Tcm | Tcm |     |       | SEK15 major      | 43  |
|     |     |     |     |       | SEK15b (trace)   |     |
| Tcm | Tcm | Act |     |       | SEK15 major      | 44  |
|     |     |     |     |       | SEK15b (trace)   |     |
| Tcm | Tcm | Tcm | Act |       | RM20b,c          | 45  |
|     |     |     |     |       | RM20             |     |
| Tcm | Tcm | Act | Act |       | RM20b            | 33  |
| Tcm | Tcm | Act | Act | TcmJN | RM20b            | 44  |
| Tcm | Tcm | Act |     | TcmJN | RM80/80b         | 44  |

---

Acetate + malonate  
Min PKS  
KR1  

![Chemical Structures](https://i.imgur.com/yourimageurl.png)

**Scheme 5**

Nat. Prod. Rep., 1999, 16, 425–484 437

ing and ‘directing’ presence of aKR1 binding site nearby for the delivery into of the newly created polyketide chain prevents the aberrant folding and cyclisation occurring. In the reduced decaketide, the non-aromatised products RM20bc can be isolated (Scheme 5).

The decaketide DpsKR1(9) will not reduce an octaketide precursor, but the JadKR1(9) will, efficiently replacing Act KR1(9) (Table 10). The use of DpsKR1(9) with Tcm‘min PKS’ gives a mixture of reduced and non-reduced products indicating low compatibility. JadCYC2/3 has no effect on an unreduced Tcm or Dps or Act unreduced octaketide min PKS product.

Act, Gris or FrenAROab has no effect on unreduced substrates, but may promote CYC1(7/12) regiochemistry in the Tcm system (Table 11). Addition of Act or GrisAROab to the reduced Act system promotes early aromatisation of the first ring, preventing mutactin 7 formation via aberrant CYC2(6/15). FrenAROab can substitute for ActAROab in the ActPKS, forming DMAC10.GrisAROab can aromatise the first ring from reduced Act, Fren and TcmPKS to form SEK348 (Scheme 3), DMAC and SEK4330. However, FrenAROab does not aromatise (ARO1) the reduced decaketide Tcm product, forming RM20b27 (Scheme 5), but does aromatise the reduced octaketide products from Act and Fren (Scheme 3) systems (Table 11).

ActAROab and CYC2/3 do not act upon a reduced decaketide precursor, whilst they act upon the corresponding octaketide system (Table 12). The experiment with Tcmmi n PKS, ActKR1 and ActAROab does not appear in the literature surveyed.

Addition of TcmN to octaketide and decaketide unreduced systems has a dramatic influence on the regiochemistry of the first cyclisation, changing it from the usual CYC1(7/12) to give exclusively the CYC1(9/14) products RM7732 and RM80/ RM80b25/26 respectively (Scheme 6, Table 13). The addition of TcmN also influences the chain length of the Fren minimal PKS from octaketide, to exclusively the nonaketide CYC1(9/ 14) product PK819 (Scheme 6). TcmN also appears to catalyse the cyclisation and aromatisation of the second ring. For example, ActminPKS with TcmN gives the CYC2(6/15) aromatised product RM7732 (ActPKS alone gives only non-enzymatic CYC2 compounds SEK45 and SEK4b6); Frenmin PKS with TcmN gives the aromatised CYC2(7/16) nonaketide product PK819; and Tcmmi nPKS with TcmN gives the CYC2(7/16) aromatised products RM8025andRM80b26(min TcmPKS alone gives SEK1523 with only non-enzymatic CYC2) (Scheme 6). It should be noted that in S.glaucescens minTcmPKS and TcmN produces the aromatised CYC3(5/18) product TcmF234, suggesting that TcmN can also promote CYC3 and ARO3. In all situations, TcmJ, added to TcmN, does not appear to have any effect upon product structure, but does greatly increase yields.

TcmN appears to have no effect whatsoever upon KR1(9) reduced backbones, implying that the reduced backbone is not bound by or recognised by the AROcCYC1 active site.

Table 5 Mixed minimal PKSinCH999

| KAS | CLF | ACP | Product | Ref |
| --- | --- | --- | --- | --- |
| Act | Act | Act | SEK4 | 33 |
|  |  |  | SEK4b | OCT:CYC1(7/12) |
|  |  |  |  | OCT:AcCYC1(10/15) |
| Fren | Fren | Act | SEK4 | 34 |
|  |  |  | SEK4b | OCT:Ac/CYC1(7/12) |
|  |  |  |  | OCT:AcCYC1(10/15) |
| Tcm | Tcm | Tcm | SEK15 | 33 |
|  |  |  |  | DEC:Ac/CYC1(7/12)CYC2 |
|  |  |  |  | (14/19)CYC3(O-1/5) |
| Tcm | Tcm | Act | SEK15 | 33 |
|  |  |  | (50%) | DEC:Ac/CYC1(7/12)CYC2 |
|  |  |  |  | (14/19)CYC3(O-1/5) |
|  |  |  | SEK15b | DEC:Ac/CYC1(9/14)CYC2 |
|  |  |  |  | (7/16)CYC3(6/19)CYC4(O-1/5) |

Table 6 Mixed minimal PKSinS.lividans/glaucescens

| KAS | CLF | ACP | Product | Ref |
| --- | --- | --- | --- | --- |
| Tcm | Tcm | Tcm | SEK15 | 40, |
|  |  |  |  | DEC:Ac/CYC1 |
|  |  |  |  | (7/12)CYC2(14/19)CYC3 |
|  |  |  |  | (O-1/5) |
|  |  |  | SEK15b | 41 |
|  |  |  | (trace) | DEC:Ac/CYC1 |
|  |  |  |  | (9/14)CYC2(7/16)CYC3 |
|  |  |  |  | (6/19)CYC4(O-1/5) |
| Act | Tcm | Tcm | No product | 42 |
| Tcm | Act | Tcm | Trace | 42 |
|  |  |  | decaketide |  |
| Act | Act | Tcm | SEK4 | 42 |
|  |  |  | SEK4b | OCT:Ac/CYC1(7/12) |
|  |  |  |  | OCT:Ac/CYC1(10/15) |

Table 7 Mixed KAS/CLF/ACP with other Act components in CH999

| KAS | CLF | ACP | KR1 | AROab | CYC2/3 | Product | Ref |
| --- | --- | --- | --- | --- | --- | --- | --- |
| Act | Act | Act | Act | Act | Act | DMAC/Aloe | 37 |
| Gra | Act | Act | Act | Act | Act | DMAC/Aloe | 37 |
| Act | Gra | Act | Act | Act | Act | DMAC/Aloe | 37 |
| Act | Act | Gra | Act | Act | Act | DMAC/Aloe | 37 |
| Fren | Act | Act | Act | Act | Act | DMAC/Aloe | 38 |
| Act | Fren | Act | Act | Act | Act | No Product | 38 |
| Act | Act | Fren | Act | Act | Act | DMAC/Aloe | 38 |
| Fren | Act | Act | Act | Act | Act | DMAC/Aloe | 38 |
|  |  |  |  |  |  | RM18 |  |
|  |  |  |  |  |  | RM18b |  |
| Fren | Act | Fren | Act | Act | Act | No Product | 38 |
| Act | Fren | Fren | Act | Act | Act | No Product | 38 |
| Fren | Fren | Fren | Act | Act | Act | DMAC/Aloe | 38 |
|  |  |  |  |  |  | RM18 |  |
|  |  |  |  |  |  | RM18b |  |
| Tcm | Act | Act | Act | Act | Act | DMAC/Aloe | 37 |
| Act | Tcm | Act | Act | Act | Act | No product | 37 |
| Act | Act | Tcm | Act | Act | Act | DMAC/Aloe | 37 |
| Tcm | Act | Act | Act | Act | Act | RM20 | 37 |
| Tcm | Act | Act | Act | Act | Act | DMAC/Aloe | 37 |
| Act | Tcm | Act | Act | Act | Act | No product | 37 |
| Tcm | Tcm | Act | Act | Act | Act | RM20 | 37 |
| Fren | Tcm | Act | Act | Act | Act | No product | 38 |
| Tcm | Fren | Act | Act | Act | Act | No product | 38 |

438 Nat.Prod.Rep., 1999, 16, 425–484

Table 8 Mixed KAS/CLF with other Tcm components in S. lividans or glaucescens

| KAS | CLF | ACP | AROc | TcmJ | Product | Ref |
|-----|-----|-----|------|------|---------|-----|
| Tcm | Tcm | Tcm |      |      | SEK 15 (1 mg L⁻¹) | 40 | DEC:Ac/CYC 1(7/12) CYC 2(14/19) CYC 3(O-1/5) |
|     |     |     |      |      | SEK15b (trace)    | 41 | DEC:Ac/CYC 1(9/14) CYC 2(7/16) CYC 3(6/19) CYC 4(O-1/5) |
| Act | Tcm | Tcm |      |      | No product        | 42 | |
| Act | Tcm | Tcm | TcmN |      | No product        | 42 | |
| Act | Tcm | Tcm |      | TcmJ | No product        | 42 | |
| Act | Tcm | Tcm | TcmN | TcmJ | No product        | 42 | |
| Act | Act | Tcm |      | TcmJ | SEK 4 (150 mg L⁻¹) | 42 | OCT:Ac/CYC 1(7/12) |
|     |     |     |      |      | SEK4b (25 mg L⁻¹)  |    | OCT:Ac/CYC 1(10/15) |
| Act | Act | Tcm | TcmN |      | UWM 1 (trace)     | 42 | OCT:Ac/CYC 1(9/14) CYC 2(6/15) CYC 3(O-1/5) |
| Tcm | Act | Tcm |      |      | Trace deca ketide | 42 | |
| Tcm | Act | Tcm | TcmN |      | Trace deca ketide | 42 | |
| Tcm | Act | Tcm | TcmN | TcmJ | TcmF 2            | 42 | DEC:Ac/CYC 1(9/14) CYC 2(7/16) CYC 3(5/18) |
|     |     |     |      |      | UWM 1             |    | OCT:Ac/CYC 1(9/14) CYC 2(6/15) CYC 3(O-1/5) |
| Tcm | Tcm | Tcm | TcmN | TcmJ | TcmF 2 (50 mg L⁻¹) | 42 | DEC:Ac/CYC 1(9/14) CYC 2(7/16) CYC 3(5/18) |
| Act | Act | Tcm | TcmN | TcmJ | UWM 1 (150 mg L⁻¹) | 42 | OCT:Ac/CYC 1(9/14) CYC 2(6/15) CYC 3(O-1/5) |

The WhiE AROc ARO 2 activity seems to substitute for TcmN on unreduced deca ketide precursors, regiochemically directing CYC 1 to 9/14, and aromatising the second ring. However, whilst TcmN has no effect on the reduced Act system, WhiE can aromatise the first ring in these systems (but not the second) to give EM 18 35 (Scheme 7, Table 13).

Table 14 firstly shows which KR1s act upon which min PKS components (in S. lividans). For example, the anthracycline (doxorubicin 50, Scheme 9) KR1 DpsF has no effect on the octaketide Act minimal PKS product, but the angucycline deca ketide Jadomycin B 69 (Scheme 11) JadKR1 does, and can substitute for ActKR1 to give mutactin 7. However, both Jad and DpsKR1 (like ActKR1) will reduce the Tcmm in PKS product to give RM20b,c (Scheme 5). However, a mixture of reduced (SEK 43 30 and UWM 4 31) and unreduced (SEK 15 23) products are obtained when Tcmm in PKS, DpsKR1 and JadD or DpsFAROab's are used, indicating that the DpsKR1 is not an efficient reducer when added to the Tcmm in PKS system. Addition of both DpsKR1 and TcmNAROc to Tcmm in PKS leads to competition between them for the Tcmm in PKS product, leading to a mixture of the non-aromatized reduced deca ketide RM20b 27, and the normal min TcmPKS plus TcmN product TcmF 2 34. The JadDAROab has no effect on unreduced precursors such as the Act, Dps or Tcmm in PKS product. The DpsFAROab has no effect upon a non-reduced octaketide, and when in combination with the inefficient Dps KR1 reducer of the Tcmm in PKS product, DpsF did not act upon 'unreduced' product SEK 15 23. Both the JadD and DpsF AROb will act upon ActKR1 reduced octaketide to give SEK 34 8.

The minDps system gives the same products as the Tcmm in PKS, and addition of TcmNJ to minDpsPKS gives TcmF 2 34 as usual, but less efficiently, as none is detected in the absence of the yield enhancer TcmJ (Table 14). Again, proteins from similar, but different sources are not perfectly matching. The JadKAS and CLF will substitute for TcmKAS and CLFinTcm F2 formation.

JadKAS/CLF/ACP/KR1/AROab with TcmJ gives the same products (SEK 43 30 and UWM 4 31) as the corresponding Dps genes, indicating that none of these proteins are responsible for the angular nature of the angucyclines (Scheme 5, Table 14). A separate cyclase, JadI may be involved in the cyclisation of the final ring CYC 4(2/19) and an as yet unlocated cyclase responsible for CYC 3(4/17) to give the angular fusion of ring three necessary to differentiate the angucyclines from the anthracyclines (vide infra 3.5.2.1).

3.3 7,9,12-Octaketides

Most bacterial octaketides biosynthetically assembled via KR1(9) CYC 1(7/12), with ketoreduction at C-9 (numbered from

thioester carbonyl), and first cyclisation (CYC 1) occurring between C-7 and C-12 [CYC 1(7/12)]. Many metabolites in this class mentioned in this review have the benzoisochromanequinone chemical structure, such as actinorhodin (Act) 14 from S. coelicolor A3(2), granaticin (Gra) 15 from S. violaceoruber Tu22, frenolicin B (Fren, probably a nonaketide) 17 nanaomicin 18 from S. roseofulvus, and griseusin B (Gris) 16 from S. griseus, all of which are CYC 2(5/14) CYC 3(O-3/15) OOX(6) structures (Scheme 2). The gene clusters are very similar, and can be compared (Fig. 1) between themselves, and with other types of Type II aromatic polyketide.

3.3.1 Actinorhodin. A dimeric benzoisochromanequinone red pigment produced by S. coelicolor A3(2) whose PK gene cluster was the first to be located by Hopwood et al. in 1984. 24 The 'early PKS' components are: KAS (gene actI-ORF1, also called KSα, a bifunctional protein with both KAS and AT active sites); CLF (gene actI-ORF2, homologous to KAS but lacks active site cysteine, also called KSβ); ACP (gene actI-ORF3); KR1(9) (actIII, reduces carbonyl at C-9); AROab (actVII, prior to 1994 confusingly referred to as the cyclase, acts to dehydrate the first and second ring resulting in aromatisation, may influence first ring CYC 1 regiochemistry), CYC 2/3 (actIV, promotes cyclisation of second and third rings, prior to 1994 sometimes referred to as the aromatase). These early PKS genes, on their own, will form DMAC 10 (and its decarboxylated derivative aloesaponarin II, 11 'Aloe') through aberrant cyclisation of the third ring CYC 3(2/15) instead of pyran formation CYC 3(O-3/15) which occurs upon addition of the later components actVI, actVA and actVB to give the benzoiso-chromanequinone structure. It is not clear what activity causes OOX(6) or whether this is non-enzymatic, forming the quinone structure. A summary of the biosynthesis is shown in Scheme 2. Post PKS enzymes include P-450 oxidations and formation of an Ar-Ar bond to assemble the final dimer. Mutations can result in the accumulation of (S)-DNPA 12 or dihydrokalafungin 13 (isolated as its shunt product kalafungin).

3.3.1.1 Regulation. Genetic control of polyketide biosynthesis has been reviewed. 50 Streptomyces coelicolor A3(2) produces actinorhodin 14, undecylprodigiosin, methylenomycin and a calcium dependent antibiotic, and for each of the first three, there is a cluster of genes involved in their biosynthesis. Genes responsible for regulating antibiotic production are often labelled as a transcriptional activator for the Act cluster. 51 However, so-called pleiotropic genes, outside the clusters, are also responsible for their regulation. For example, the biosynthesis of all four polyketide antibiotics is prevented by mutations in absA or absB. 52, 53

Nat. Prod. Rep., 1999, 16, 425–484 439

In *E. coli*, (p)ppGpp synthetases are regulated by *relA* or *spoT* genes, and are responsible for so-called stringent responses and affects the level of RNA transcription and the level of certain amino acid operons. Malpartida *et al.* have isolated a gene from *S. coelicolor* that is involved in regulating antibiotic production,

that encoded for an 847 residue protein resembling the (p)ppGpp synthetases.⁵⁴

Bld ('bald') mutants do not produce actinorhodin or undecylprodigiosin (though methylenomycin production is not affected), they do not sporulate, and it is thought that bld genes

(TcmN catalysed ARO2?)

Fren 'Min PKS'

No TcmN

Act 'Min PKS'

TcmN

Act 'Min PKS'

Act KR

SEK15

23

SEK 15b

24

Scheme 6

440 Nat. Prod. Rep., 1999, 16, 425–484

Table 9 Effect of ActKR1(9) (actIII) on various minimal PKS in CH 999

| KAS | CLF | ACP | KR1 | Product | Ref |
|-----|-----|-----|-----|---------|-----|
| Act | Act | Act |     | SEK4    | 33  | OCT:Ac/CYC1(7/12) |
|     |     |     |     | SEK4b   | 34,35 | OCT:Ac/CYC1(10/15) |
| Act | Act | Act | Act | Mutactin | 33  | OCT:Ac/KR1(9)CYC1*(7/12)CYC2(6/15)CYC3(O-1/5) |
| Fren | Fren | Act |     | SEK4    | 34  | OCT:Ac/CYC1(7/12) |
|     |     |     |     | SEK4b   |     | OCT:Ac/CYC1(10/15) |
| Fren | Fren | Act | Act | Mutactin(70%) | 33  | OCT:Ac/KR1(9)CYC1*(7/12)CYC2(6/15) |
|     |     |     |     | +RM18(10%) |     | NON:Ac/KR1(9)CYC1(7/12)CYC2(11/16)CYC3(O-5/13) |
|     |     |     |     | +RM18b(20%) |     | OCT:Ac/KR1(7)CYC1(5/10)CYC2(9/14)CYC3(O-3/11) |
| Tcm | Tcm | Act |     | SEK15(50%) | 33  | DEC:Ac/CYC1(7/12)CYC2(14/19)CYC3(O-1/5) |
|     |     |     |     | +SEK15b(50%) |     | DEC:Ac/CYC1(9/14)CYC2(7/16)CYC3(6/19)CYC4(O-1/5) |
| Tcm | Tcm | Act | Act | RM20(25%) | 33  | DEC:Ac/KR1(9)CYC1(7/12)CYC2(11/16)CYC3(O-5/13) |
|     |     |     |     | +RM20b,c(75%) |     | DEC:Ac/KR1(9)CYC1*(7/12)CYC2(O-11/15)CYC3(14/19) |

Table 10 Effect of various KR1s in S. lividans or glaucescens

| KAS | CLF | ACP | KR1 | CYC 2/3 | TcmJ | Product | Ref |
|-----|-----|-----|-----|---------|------|---------|-----|
| Act | Act | Tcm |     |         | TcmJ | SEK4(150mgL⁻¹) | 42  | OCT:Ac/CYC1(7/12) |
|     |     |     |     |         |      | SEK4b(25mgL⁻¹) |     | OCT:Ac/CYC1(10/15) |
| Act | Act | Tcm | Dps |         | TcmJ | SEK4    | 47  | OCT:Ac/CYC1(7/12) |
|     |     |     |     |         |      | SEK4b   |     | OCT:Ac/CYC1(10/15) |
| Act | Act | Tcm | Act |         | TcmJ | Mutactin | 33  | OCT:Ac/KR1(9)CYC1*(7/12)CYC2(6/15)CYC3(O-1/5) |
| Act | Act | Tcm | Jad |         | TcmJ | Mutactin | 47  | OCT:Ac/KR1(9)CYC1*(7/12)CYC26/15)CYC3(O-1/5) |
| Act | Act | Tcm | Dps | Dps     | TcmJ | SEK4    | 47  | OCT:Ac/CYC1(7/12) |
|     |     |     |     |         |      | SEK4b   |     | OCT:Ac/CYC1(10/15) |
| Act | Act | Tcm | Act | Dps     | TcmJ | SEK34   | 47  | OCT:Ac/KR1(9)CYC1(7/12)CYC2(O-11/15) |
| Act | Act | Tcm | Jad |         | TcmJ | SEK4    | 47  | OCT:Ac/CYC1(7/12) |
|     |     |     |     |         |      | SEK4b   |     | OCT:Ac/CYC1(10/15) |
| Act | Act | Tcm | Act | Jad     | TcmJ | SEK34   | 47  | OCT:Ac/KR1(9)CYC1(7/12)CYC2(O-11/15) |
| Tcm | Tcm | Tcm | Jad |         | TcmJ | SEK15   | 47  | DEC:Ac/CYC1(7/12)CYC2(14/19)CYC3(O-1/5) |
|     |     |     |     |         |      | SEK15b  |     | DEC:Ac/CYC1(9/14)CYC2(7/16)CYC3(6/19)CYC4(O-1/5) |
| Tcm | Tcm | Tcm | Dps | Jad     | TcmJ | SEK43   | 47  | DEC:Ac/KR1(9)CYC1(7/12)CYC2(14/20)CYC3(O-1/5) |
|     |     |     |     |         |      | UWM4    |     | DEC:Ac/KR1(9)CYC1(7/12)CYC2(13/18) |
|     |     |     |     |         |      | SEK15   |     | DEC:Ac/CYC1(7/12)CYC2(14/19)CYC3(O-1/5) |
| Dps | Dps | Tcm |     |         | TcmJ | SEK15   | 48  | DEC:Ac/CYC1(7/12)CYC2(14/19)CYC(O-1/5) |
|     |     |     |     |         |      | SEK15b  |     | DEC:Ac/CYC1(9/14)CYC2(7/16)CYC3(6/19)CYC4(O-1/5) |
| Dps | Dps | Tcm | Dps |         | TcmJ | RM20b,c | 47  | DEC:Ac/KR1(9)CYC1(7/12)CYC2(O-11/15)CYC3(14/19) |
| Dps | Dps | Tcm | Jad |         | TcmJ | SEK15   | 47  | DEC:Ac/CYC1(7/12)CYC2(14/19)CYC3(O-1/5) |
|     |     |     |     |         |      | SEK15b  |     | DEC:Ac/CYC1(9/14)CYC2(7/16)CYC3(6/19)CYC4(O-1/5) |
| Dps | Dps | Tcm | Dps | Jad     | TcmJ | SEK43   | 47  | DEC:Ac/KR1(9)CYC1(7/12)CYC2(14/20)CYC3(O-1/5) |
|     |     |     |     |         |      | UWM4    |     | DEC:Ac/KR1(9)CYC1(7/12)CYC2(13/18) |

are involved both in antibiotic production and aerial mycelium formation. 55 BldA codes for a leucyl tRNA that recognises the UUC codon. Due to the high G+C content in Streptomyces, this codon is normally extremely rare, and its occurrence and transcription could be a translational regulatory mechanism involved in turning on antibiotic production and sporulation. 56

S. lividans can carry the entire gene cluster for actinorhodin biosynthesis, but under non-stress conditions, does not express them. This may be due to a DNA binding 99aa residue protein that represses the expression of the Act genes. Expression of a gene (micX) from S. fradiae activates actinorhodin expression, 57 perhaps by encoding for an antisense RNA that acts at the translational level by complementary base pairing to the messenger RNA coding for ORF1. 58

3.3.1.2 Malonyl transferase (MT). Revill et al. have purified MT from S. coelicolor A3(2) and found that it labels ActPKS ACP with radiolabelled malonyl CoA. 59 No other MT activity could be found. Disruption was lethal, confirming its role in fatty acid metabolism. The gene was over 2.8Mbp from the act cluster, and is adjacent to and cotranscribed with a gene resembling a E.coli fabH(KASI), and was expressed in both vegetative and stationary phases. Revill suggests that the MT might be shared between FAS and PKS, a possible link with both FA and PK metabolism.

The S. coelicolor PKS gene cluster only contains one sequence (GHS, part of actI-ORF1 gene) that could have

acyltransferase activity, however it has been reported that mutation of this serine did not stop PK production. 60, 61 Mutation of the S.parvulus PK synthase at the putative KAS-AT GHS active site (S347L) prevented production of aloesaponarin II 11 in liquid culture, but on agar it still produced aloesaponarin II at significant levels. Whilst this was interpreted as meaning that this was not the transferase active site, it may be that the rest of the protein complex (ACP?) retains some self-transferase activity in the absence of the main AT activity (vide infra).

3.3.1.3 Acyl carrier protein. The essential role of the ACP in polyketide production has been shown as deletion causes total loss of activity, 62 and insertion of multiple copies increases production. 63 The ActACP(ActI-ORF3) can be substituted by other Type II aromatic PKS ACPs, or to some extent by the Saccharopolyspora erythraea FAS ACP. 46 FAS ACPs from S. glaucescens and E.coli can only feebly replace the TcmACP. 64 Schweizer et al. have overexpressed (10% total protein) a segment of the rat FAS (residues 2114 to 2202) corresponding to the extended (including some linker region) ACP domain (active site Ser-2151) in E.coli and obtained 1% in the holo-form. Co-expression with E.coli holo-ACP synthetase gene (acpS) increased holo-levels five-fold. When the rat FAS ACP was expressed in S.coelicolor A3(2) CH 999 in which all the PKS components except ActACP necessary to make aloesaponarin II 11, were present, the ACP was posttranslationally modified, and the yellow-brown pigment presumed to be 11 was

Nat.Prod.Rep., 1999, 16, 425–484 441

Table 11 Effect of benzoisochromanequinone AROab in CH 999

| KAS | CLF | ACP | KR1 | AROab | CYC 2/3 | Product | Ref |
|-----|-----|-----|-----|-------|---------|---------|-----|
| Act | Act | Act |     |       |         | SEK 4   | 33  |
|     |     |     |     |       |         | SEK4b   | 34  |
| Act | Act | Act |     |       |         | SEK 4   | 43  |
|     |     |     |     |       |         | SEK4b   |     |
| Act | Act | Act | Gris|       |         | SEK 4   | 39  |
| Act | Act | Act | Act |       |         | DMAC/Aloe | 37  |
| Act | Act | Act | Fren| Act   |         | SEK 4   | 39  |
| Act | Act | Act | Gris| Act   |         | SEK 4   | 39  |
| Act | Act | Act | Act |       |         | Mutactin | 33  |
| Act | Act | Act | Act |       |         | SEK 34   | 36  |
| Act | Act | Act | Act | Gris  |         | SEK 34   | 39  |
| Act | Act | Act | Act | Act   |         | DMAC     | 36  |
| Act | Act | Act | Act | Fren  | Act     | DMAC     | 39  |
| Fren| Fren| Act | Act |       |         | Mutactin (70%) | 33  |
|     |     |     |     |       |         | RM18(10%) |     |
|     |     |     |     |       |         | RM18b(20%) |     |
| Fren| Fren| Act | Act | Gris  |         | SEK 34   | 39  |
| Fren| Fren| Fren| Act | Act   |         | DMAC/Aloe | 38  |
|     |     |     |     |       |         | RM18(20%) |     |
|     |     |     |     |       |         | RM18b(30%) |     |
| Fren| Fren| Act | Act | Fren  | Act     | DMAC     | 39  |
| Fren| Fren| Act | Act | Gris  | Act     | DMAC     | 39  |
|     |     |     |     |       |         | SEK 26   |     |
| Tcm | Tcm | Act |     |       |         | SEK 15(50%) | 33  |
| Tcm | Tcm | Act |     |       |         | SEK 15b   | 43  |
| Tcm | Tcm | Act | Gris|       |         | SEK 15    | 39  |
| Tcm | Tcm | Act | Gris| Act   |         | SEK 15    | 39  |
| Tcm | Tcm | Act | Fren| Act   |         | SEK 15    | 39  |
| Tcm | Tcm | Act | Act | Fren  | Act     | RM20b     | 39  |
| Tcm | Tcm | Act | Act | Gris  | Act     | SEK 43    | 39  |

Table 12 Effect of Act CYC 2/3 in CH 999

| KAS | CLF | ACP | KR1 | AROab | CYC 2/3 | Product | Ref |
|-----|-----|-----|-----|-------|---------|---------|-----|
| Act | Act | Act | Act | Act   |         | SEK 34   | 36  |
| Act | Act | Act | Act | Act   | Act     | DMAC/Aloe | 37  |
| Tcm | Tcm | Act | Act | Act   |         | RM20(25%) | 33  |
|     |     |     |     |       |         | RM20b,c   |     |
| Tcm | Tcm | Tcm | Act | Act   | Act     | RM20(25%) | 37/45 |
|     |     |     |     |       |         | RM20b,c   |     |

observed by TLC at approx 1% levels of that seen for the corresponding proteins with Act ACP. 65 Expression with the rat FASACP$S2151A$ mutant gave no such pigment. Simpson et al. have overexpressed the actinorhodin (Act) (6 mg L$^{-1}$), granaticin (Gra) (3.5 mg L$^{-1}$), tetracenomycin (Tcm) (0.5 mg L$^{-1}$) and oxytetracycline (Otc) (0.2 mg L$^{-1}$) ACPs in E.coli. 66 Alteration of the first few codons in Act and Otc ACP genes to more ‘E.coli friendly’ triplet codons increased yields. However, ESMS showed that the resulting Act protein was at least 97% in the apo (non-phosphopantetheinylated) form and 3% holo-form, whilst Otc and Fren ACPs were entirely apo. The Gra ACP was approx. 66% apo and 34% holo. After a period of time, the Act and Otc apo-ACP slowly dimerised, presumably via the exposed Cys-17 residue. With a longer post ‘heat induction’ incubation period, the ratios of Gra, Otc and Fren holo-to apo-ACP were unchanged, however, after 6h, the level of Act holo-ACP was 40%, and after 12h, over 80% of the ACP was in the holo-form, though the overall yield had dropped five-fold. The four ACPs being overexpressed have around 25% sequence homology with E.coli FAS ACP, whilst Saccharo- polyspora erythraea FAS ACP, which is efficiently overex- pressed in E.coli in the holo form, has 40% homology, thus structural differences could at least contribute to the reduced level of posttranslational modification by the E.coli holo-ACP synthase. However, it is unclear why the Act ACP and Otc or

Fren ACPs are overexpressed so differently. Whilst E.coli FAS ACP runs slowly at twice $M_r$ the Otc ACP runs on SDS gel close to its $M_r$, and Act, Gra and Fren ACPs all ran fast (about half $M_r$) suggesting that even under these denaturing conditions, a compact structure is formed in these cases.

A major limiting factor in the posttranslational modification of overexpressed ACPs is the poor ability of the E.coli FAS ACPs to modify PKS ACPs, so Simpson et al. have recently coexpressed Act and Gris ACPs with ACPS ($acpS$) in E.coli and obtained high levels (10 mg L$^{-1}$) of PKS ACPs over 90% in the holo-form. 67.

Simpson et al. have extensively studied the Act ACP by NMR 68 and compare its structure to that for E.coli FAS ACP, which has 77 residues (active residue Ser-36), is highly helical (60%) with a four helix bundle (residues 3–15, 37–51, 56–63 and 65–75) and a hydrophobic core that is thought to chaperone to the growing fatty acyl chain. The authors have achieved near complete NMR assignment of the Act apo-ACP, which they find is largely $\alpha$-helix (49%), with four helices at residues 7–16, 42–53, 62–67 and 72–85, with a large loop between 17 and 41 that is part ill-defined (18–25) and part highly structured and presumably buried in the overall structure. The NMR data shows that there is a strong overall similarity of secondary structure between the FAS and PKS ACP, that is presumably reflected in the three-dimensional structure. A major difference

442 Nat. Prod. Rep., 1999, 16, 425–484

Table 13 Effect of AROc in CH 999

| KAS | CLF | ACP | KR1 | AROc | Product | Ref |
|-----|-----|-----|-----|------|---------|-----|
| Act | Act | Act |     |      | SEK 4   | 33  | OCT:Ac/CYC 1(7/12) |
|     |     |     |     |      | SEK4b   | 34/5 | OCT:Ac/CYC 1(10/15) |
| Act | Act | Act |     | TcmJ | SEK 4   | 44  | OCT:Ac/CYC 1(7/12) |
|     |     |     |     |      | SEK4b   |     | OCT:Ac/CYC 1(10/15) |
| Act | Act | Act |     | TcmN | RM77    | 44  | OCT:Ac/CYC 1(9/14) CYC 2(6/15) OOX(10) |
| Act | Act | Act |     | TcmN and TcmJ | RM77 | 44  | OCT:Ac/CYC 1(9/14) CYC 2(6/15) OOX(10) |
| Act | Act | Act | Act | WhiE | Mutactin | 49  | OCT:Ac/KR1(9) CYC 1*(7/12) CYC 2(6/15) CYC 3(O-1/5) |
|     |     |     |     |      | EM18    |     | OCT:Ac/KR1(9) CYC 1(7/12) CYC 2(6/17) CYC 3(O-1/5) |
| Fren | Fren | Act | Act | WhiE | Mutactin | 49  | OCT:Ac/KR1(9) CYC 1*(7/12) CYC 2(6/15) CYC 3(O-1/5) |
|     |     |     |     |      | EM18    |     | OCT:Ac/KR1(9) CYC 1(7/12) CYC 2(6/17) CYC 3(O-1/5) |
|     |     |     |     |      | RM18    |     | NON:Ac/KR1(9) CYC 1(7/12) CYC 2(11/16) CYC 3(O-5/13) |
| Tcm | Tcm | Tcm |     | WhiE | RM80/RM80b | 49  | DEC:Ac/CYC 1(9/14) CYC 2(7/16) CYC 3(O-15/19) |
| Act | Act | Act | Act |      | Mutactin | 33  | OCT:Ac/KR1(9) CYC 1*(7/12) CYC 2(6/15) CYC 3(O-1/5) |
| Act | Act | Act | Act | TcmJ | Mutactin | 44  | OCT:Ac/KR1(9) CYC 1*(7/12) CYC 2(6/15) CYC 3(O-1/5) |
| Act | Act | Act | Act | TcmN | Mutactin | 44  | OCT:Ac/KR1(9) CYC 1*(7/12) CYC 2(6/15) CYC 3(O-1/5) |
| Act | Act | Act | Act | TcmN and TcmJ | Mutactin | 44  | OCT:Ac/KR1(9) CYC 1*(7/12) CYC 2(6/15) CYC 3(O-1/5) |
| Fren | Fren | Act |     |      | SEK 4   | 34  | OCT:Ac/CYC 1(7/12) |
|     |     |     |     |      | SEK4b   |     | OCT:Ac/CYC 1(10/15) |
| Fren | Fren | Act |     | TcmJ | SEK 4   | 34  | OCT:Ac/CYC 1(7/12) |
|     |     |     |     |      | SEK4b   |     | OCT:Ac/CYC 1(10/15) |
| Fren | Fren | Act |     | TcmN | PK8    | 34  | NON:Ac/CYC 1(9/14) CYC 2(7/16) |
| Fren | Fren | Act |     | TcmN and TcmJ | PK8 | 34  | NON:Ac/CYC 1(9/14) CYC 2(7/16) |
| Tcm | Tcm | Tcm |     |      | SEK 15 (50%) | 33  | DEC:Ac/CYC 1(7/12) CYC 2(14/19) CYC 3(O-1/5) |
|     |     |     |     |      | SEK15b  |     | DEC:Ac/CYC 1(9/14) CYC 2(7/16) CYC 3(6/19) CYC 4(O-1/5) |
| Tcm | Tcm | Tcm |     | TcmJ | SEK 15  | 44  | DEC:Ac/CYC 1(7/12) CYC 2(14/19) CYC 3(O-1/5) |
|     |     |     |     |      | SEK15b  |     | DEC:Ac/CYC 1(9/14) CYC 2(7/16) CYC 3(6/19) CYC 4(O-1/5) |
| Tcm | Tcm | Tcm |     | TcmN | RM80/RM80b | 44  | DEC:Ac/CYC 1(9/14) CYC 2(7/16) CYC 3(O-15/19) |
| Tcm | Tcm | Tcm |     | TcmN and TcmJ | RM80/RM80b | 44  | DEC:Ac/CYC 1(9/14) CYC 2(7/16) CYC 3(O-15/19) |
| Tcm | Tcm | Tcm | Act |      | RM20b   | 33  | DEC:Ac/KR1(9) CYC 1*(7/12) CYC 2(O-11/15) CYC 3(14/19) |
| Tcm | Tcm | Tcm | Act | TcmJ | RM20b   | 44  | DEC:Ac/KR1(9) CYC 1*(7/12) CYC 2(O-11/15) CYC 3(14/19) |
| Tcm | Tcm | Tcm | Act | TcmN | RM20b   | 44  | DEC:Ac/KR1(9) CYC 1*(7/12) CYC 2(O-11/15) CYC 3(14/19) |
| Tcm | Tcm | Tcm | Act | TcmN and TcmJ | RM20b | 44  | DEC:Ac/KR1(9) CYC 1*(7/12) CYC 2(O-11/15) CYC 3(14/19) |

that would be expected between FAS and aromatic PKS ACPs would be the nature of the ‘chaperoning’ region, as the growing PKS acyl chain would be highly oxidised, and would need stabilisation presumably as a polyenolate by protonated residues such as arginines. There is a conserved arginine in Act (R 72), Otc(R 72), and Gris(R 73) ACP, all of which have diastereotopic protons on C-5 that are split in the NMR by over 0.5 ppm suggesting limited conformational freedom due to being located in the core of the protein, potentially able to stabilise the growing PK chain.

In a second paper, Simpson et al. use the distance and angle constraints from this NMR data to generate a series of three dimensional structures for the Act ACP. Unlike the E. coli FAS holo-ACP, the Act apo-ACP is not conformationally averaged between two or more structures, and whilst both E. coli and Act structures have a hydrophobic core, the Act ACP contains two buried hydrophilic residues, arginine and asparagine (R 72 and N 79) which are conserved with other PKS ACPs, but not FAS ACPs, and that may play a crucial role in chaperoning the growing polyketide chain. The R 72 side chain N-H protons gave weak NOEs to the L 31 methyl groups, and the N 79 terminal N-Hs gave short range NOEs to L 75 methyl groups. The guanidinium group of R 11 was solvent exposed, whilst its C-2 and C-3 protons gave long range NOEs to the buried F 30 (Phe) ring protons, implying that R 11 is part of a buried yet accessible groove. In unpublished observations, Simpson finds similar chemical shifts and diastereotopic splittings of C-5 proton chemical shifts for the R 73 and R 71 residues of Gris and Otc ACPs respectively, implying a conserved partially buried local structure and crucial role for this residue. Simpson speculates that the stabilisation of just two enolates out of eight ketones may be sufficient, leaving six carbonyl groups in three ‘pairs’, the first and second, fourth and fifth, and the seventh and eighth.

E. coli FAS uses a malonyl transferase (MT, malonyl CoA: holo-ACP transacylase, often abbreviated as MCAT in literature) to transfer the extender malonate unit from CoA and attach it to the ACP. However, no region corresponding to MT (though there is a putative ‘acyl transferase’ domain with GHSXG in the act1-ORF1KAS gene) has been located in the Type II PKS gene clusters, though MT proteins have been isolated from several Streptomyces spp., and the S. coelicolor fabD MT gene has been mapped to a putative FAS gene cluster (vide supra 3.3.1.2). It has been proposed that they catalyse the formation of both FAS and PKS malonyl-ACP. However, Simpson et al. have found that incubation of Act holo-ACP (the C17S mutant was used to avoid formation of a disulfide bond between C-17 and pantetheine thiol, but identical results obtained with wildtype, and similar but slower results obtained with Gris and Otc ACPs) with malonyl CoA in phosphate buffer at 30°C gave malonyl-ACP within 30 min as judged by ESMS. Incubation using either acetyl CoA or with apo-ACP gave no adduct formation. The overexpressed protein was over 99% pure, and to ensure that there had been no contamination by E. coli MT, PMSF (an effective inhibitor of E. coli MT) was added to the incubation mixture, without affecting the rate of reaction. The results show that the Act holo-ACP is capable of self-catalysing its own malonation by malonyl CoA onto its phosphopantetheine thiol, which the corresponding S. coelicolor FAS ACP was unable to achieve. Interestingly, partial digestion of malonyl-ACP with trypsin gave a phosphopante-theinylated fragment with acetate, not malonate attached, and partial denaturation with 4M urea gave protein that was now 90% acetylated, rather than malonated. This suggests that decarboxylation of ACP bound malonate is catalysed by a residue that only becomes accessible under these denaturing conditions. This raises the question of whether this mimics what happens to the ACP conformation during the approach to it of a second protein such as KAS. The catalysis of decarboxylation by the addition of a Cys deleted KAS mutant to malonated ACP would suggest that this conformational change and consequent decarboxylation is part of the driving force for C-C bond formation. The decarboxylation of malonyl CoA has been observed in a wide range of organisms to reduce local levels of

![Tcm 'min PKS' →](image1.png)

![CYC1(9/14) (ARO1)](image2.png)

![WhiE AROc (ARO2)](image3.png)

![spontaneous Phenolic -OH rapidly attacks C-19 carbonyl](image4.png)

![RM80 25 + RM80b 26](image5.png)

![Fren Min PKS Act KR1](image6.png)

![Act min PKS Act KR1](image7.png)

![WhiE AROc](image8.png)

![WhiE (ARO1)](image9.png)

---

**Scheme 7**

malonyl CoA (e.g. goose uropygial gland to ensure only methylmalonyl CoA available as FAS extender unit) and a gene cluster (*matABC*) from *Rhizobium trifolii* has recently been overexpressed in *E. coli* by An and Kim, ${}^{71b}$ the structure and mechanism of the MatA malonyl CoA decarboxylase protein may perhaps have some connection with PKS.

The Act C17S holo-ACP was also able to efficiently add methylmalonyl CoA, 3-oxobutanoyl CoA and the NAC thioester of 3-oxobutanoate, poorly add succinyl CoA, but did not add butyryl CoA. ${}^{71a}$ These observations are all consistent with the initial formation of a guanidino salt bridge to the carboxylate, prior to transthioesterification. Incubation with the NAC thioester of 3-oxobutanoate gave similar results to the corresponding CoA thioester, emphasising the important role of the acyl binding, and the low role played by the distal portion of the CoA fragment. In contrast, Staunton *et al.* had not observed addition after incubating *S. erythraea* FAS ACP with acyl-NAC thioesters. ${}^{72a}$ Simpson *et al.* found that addition of *S. coelicolor* MT initially slowed the C17S holo-ACP self-malonylation due to substrate competition, and after further incubation, did not result in any enhancement in the rate of ACP

**444 Nat. Prod. Rep., 1999, 16, 425–484**

Methylmalonyl, 3-oxobutanoyl and succinyl CoA were also unbound ‘free’ ACP, which after malonation, can then transfer
transferred, but not acetyl or butyryl CoA. When Simpson co- the malonate to ther complex. The remaining capability of MT
overexpresses KAS, CLF and ACP together, it was not possible to malonate ACP might be an evolutionary fossil left over from
to purify the KAS and CLF away from all traces of ACP. a time when it was needed, and since then the ACP, along with
Therefore, Simpson then co-overexpressed His tagged ActKAS KAS and CLF have evolved to more efficiently do the job
and CLF in CH999 (ca. 30 mg L⁻¹) and rapidly purified them themselves. Another option may be that the minimal PKS is
to obtain a homogenous protein extract containing both really a large oligomer of KAS/CLF/ACP, that can only
proteins, rigorously free of any MT or ACP. Addition of efficiently assemble into a productive self-malonating and PK
monomeric ActC17SholoACP and malonyl CoA to the forming ‘oligo-minimal PKS’ assemblage when a full equivalent of ACP is present. However, whilst the in vivo concentrations of malonyl CoA, ACP, KAS and CLF are unknown, the
extract, followed by extraction into ethyl acetate and RP-HPLC question of whether MT plays an in vivo role or not in
showed that SEK4 and SEK4b had been produced. At actinorhodin biosynthesis, or other polyketides, must remain
concentrations of ActACP above 20 μM, the rate of polyketide unresolved.
production was proportional to ACP concentration up to the 200
μM examined. However, below 20 μM ACP, polyketide
production rapidly dropped off to zero (at about 5 μM ACP)
with decreasing ACP concentration. At these low ACP
concentrations (0–20 μM) polyketide production was restored
to linear levels (but no more) by the addition of purified S.
coelicolor FASMT (1 μM). These observations are consistent
with the rate determining step in the formation of SEK45 and
SEK4b being the malonation of ACP. At high ACP
concentrations, relative to KAS and CLF concentration, self-
malonation is sufficiently rapid, and the KAS/CLF are saturated. At low ACP levels, self-malonation would be rate
limiting, unless FASMT is added. Simpson also speculates that
only free ACP self-malonates, and only when concentrations
are in excess of the KAS/CLF concentration is there any

3.3.1.4 3-Oxoacyl synthetase and chain length factor. The
ActI region codes for the ACP (ORF3) and for two very
homologous proteins that are thought to have arisen by gene
duplication and then divergence. The first (ActI-ORF1) contains an active site cysteine and is now considered as the KAS
or KSα, whilst the second (ActI-ORF2) does not contain
cysteine, but a serine that may act as a transferase. This second
protein has until recently been considered to determine the
length of the growing acyl chain and is referred to as the ‘chain
length factor’ or ‘CLF’. Mixed KAS/CLF hybrids suggest that
both these proteins have a role in determining chain length, and
other experiments show other ‘later’ early PKS proteins can

Table 14 Comparison of ARoCTcmN with AROabJadD and DpsF components in S. lividans/glaucescens

| KAS | CLF | ACP | KR1 | ArO | TcmJ | Product | Ref |
|-----|-----|-----|-----|-----|------|---------|-----|
| Act | Act | Tcm | Dps |     | TcmJ | SEK4    | 47  |
|     |     |     |     |     |      | SEK4b   | OCT:Ac/CYC1(7/12) |
|     |     |     |     |     |      |         | OCT:Ac/CYC1(10/15) |
| Act | Act | Tcm | Act |     | TcmJ | Mutactin | 33  |
|     |     |     |     |     |      |         | OCT:Ac/KR1(9)CYC1*(7/12)CYC2(6/15)CYC3(O-1/5) |
| Act | Act | Tcm | Jad |     | TcmJ | Mutactin | 47  |
|     |     |     |     |     |      |         | OCT:Ac/KR1(9)CYC1*(7/12)CYC2(6/15)CYC3(O-1/5) |
| Act | Act | Tcm |     | JadDAROab | TcmJ | SEK4    | 47  |
|     |     |     |     |     |      | SEK4b   | OCT:Ac/CYC1(7/12) |
|     |     |     |     |     |      |         | OCT:Ac/CYC1(10/15) |
| Act | Act | Tcm | Dps | JadDAROab | TcmJ | SEK4    | 47  |
|     |     |     |     |     |      | SEK4b   | OCT:Ac/CYC1(7/12) |
|     |     |     |     |     |      |         | OCT:Ac/CYC1(10/15) |
| Act | Act | Tcm | Dps | DpsFAROab | TcmJ | SEK4    | 47  |
|     |     |     |     |     |      | SEK4b   | OCT:Ac/CYC1(7/12) |
|     |     |     |     |     |      |         | OCT:Ac/CYC1(10/15) |
| Act | Act | Tcm | Act | DpsFAROab | TcmJ | SEK34   | 47  |
|     |     |     |     |     |      |         | OCT:Ac/KR1(9)CYC1(7/12)CYC2(O-11/15) |
| Act | Act | Tcm | Act | JadDAROab | TcmJ | SEK34   | 47  |
|     |     |     |     |     |      |         | OCT:Ac/KR1(9)CYC1(7/12)CYC2(O-11/15) |
| Tcm | Tcm |     |     |     | TcmJ | TcmF2   | 42  |
|     |     |     |     |     |      | SEK15   | DEC:Ac/CYC1(9/14)CYC2(7/16)CYC3(5/18) |
|     |     |     |     |     |      | SEK15b  | DEC:Ac/CYC1(7/12)CYC2(14/19)CYC3(O-1/5) |
| Tcm | Tcm | Tcm | Dps |     | TcmJ | RM20b,c | 47  |
|     |     |     |     |     |      |         | DEC:Ac/KR1(9)CYC1*(7/12)CYC2(O-11/15)CYC3(14/19) |
| Tcm | Tcm | Tcm | Jad |     | TcmJ | RM20b,c | 47  |
|     |     |     |     |     |      |         | DEC:Ac/KR1(9)CYC1*(7/12)CYC2(O-11/15)CYC3(14/19) |
| Tcm | Tcm | Tcm |     | TcmNAROc | TcmJ | TcmF2   | 42  |
|     |     |     |     |     |      |         | DEC:Ac/CYC1(9/14)CYC2(7/16)CYC3(5/18) |
| Tcm | Tcm |     | Dps | TcmNAROc | TcmJ | TcmF2   | 47  |
|     |     |     |     |     |      | RM20b   | DEC:Ac/KR1(9)CYC1*(7/12)CYC2(O-11/15)CYC3(14/19) |
| Tcm | Tcm | Tcm |     | JadDAROab | TcmJ | SEK15   | 47  |
|     |     |     |     |     |      | SEK15b  | DEC:Ac/CYC1(7/12)CYC2(14/19)CYC3(O-1/5) |
|     |     |     |     |     |      |         | DEC:Ac/CYC1(9/14)CYC2(7/16)CYC3(6/19)CYC4(O-1/5) |
| Tcm | Tcm |     | Dps | JadDAROab | TcmJ | SEK43   | 47  |
|     |     |     |     |     |      | UWM4    | DEC:Ac/KR1(9)CYC1(7/12)CYC2(14/20)CYC3(O-1/5) |
|     |     |     |     |     |      | SEK15   | DEC:Ac/CYC1(7/12)CYC2(14/19)CYC3(O-1/5) |
| Tcm | Tcm | Tcm | Dps | DpsFAROab | TcmJ | SEK43   | 47  |
|     |     |     |     |     |      | UWM4    | DEC:Ac/KR1(9)CYC1(7/12)CYC2(13/18) |
|     |     |     |     |     |      | SEK15   | DEC:Ac/CYC1(7/12)CYC2(14/19)CYC3(O-1/5) |
| Dps | Dps | Tcm |     |     | TcmJ | SEK15   | 48  |
|     |     |     |     |     |      | SEK15b  | DEC:Ac/CYC1(7/12)CYC2(14/19)CYC3(O-1/5) |
|     |     |     |     |     |      |         | DEC:Ac/CYC1(9/14)CYC2(7/16)CYC3(6/19)CYC4(O-1/5) |
| Dps | Dps | Tcm |     | TcmNAROc | TcmJ | No product | 48  |
| Dps | Dps | Tcm |     | TcmNAROc | TcmJ | TcmF2   | 48  |
|     |     |     |     |     |      |         | DEC:Ac/CYC1(9/14)CYC2(7/16)CYC3(5/18) |
| Dps | Dps | Tcm | Dps | TcmNAROc | TcmJ | RM20b,c | 47  |
|     |     |     |     |     |      |         | DEC:Ac/KR1(9)CYC1*(7/12)CYC2(O-11/15)CYC3(14/19) |
| Dps | Dps | Tcm |     | JadDAROab | TcmJ | SEK15   | 47  |
|     |     |     |     |     |      | SEK15b  | DEC:Ac/CYC1(7/12)CYC2(14/19)CYC3(O-1/5) |
|     |     |     |     |     |      |         | DEC:Ac/CYC1(9/14)CYC2(7/16)CYC3(6/19)CYC4(O-1/5) |
| Dps | Dps | Tcm | Dps | JadDAROab | TcmJ | SEK43   | 47  |
|     |     |     |     |     |      | UWM4    | DEC:Ac/KR1(9)CYC1(7/12)CYC2(14/20)CYC3(O-1/5) |
|     |     |     |     |     |      |         | DEC:Ac/KR1(9)CYC1(7/12)CYC2(13/18) |
| Dps | Dps | Tcm | Dps | DpsFAROab | TcmJ | SEK43   | 47  |
|     |     |     |     |     |      | UWM4    | DEC:Ac/KR1(9)CYC1(7/12)CYC2(13/18) |
| Jad | Jad | Tcm |     | TcmNAROc | TcmJ | TcmF2   | 47  |
|     |     |     |     |     |      |         | DEC:Ac/CYC1(9/14)CYC2(7/16)CYC3(5/18) |
| Jad | Jad | Jad |     | JadDAROab | TcmJ | SEK43   | 47  |
|     |     |     |     |     |      | UWM4    | DEC:Ac/KR1(9)CYC1(7/12)CYC2(14/20)CYC3(O-1/5) |
|     |     |     |     |     |      |         | DEC:Ac/KR1(9)CYC1(7/12)CYC2(13/18) |

a KAS: DpsA, JadA, TcmK. CLF: DpsB, JadB, TcmL. ACP: DpsG, JadC, TcmN. KR1: DpsE, JadE. TcmJ has unknown function, but increases yields.
also influence chain length, so it has been suggested that the term ‘CLF’ be replaced by KSβ. Early literature refers to it as KASII. However, it seems that little is understood at the molecular level how the ‘system’ assembles a defined length of polyketo chain. Some FAS systems (e.g. plants) use a Cys containing thioesterase as a chain length extension terminator, whilst bacterial and mammalian systems use Ser containing TEs. However, unlike the FAS systems, there would need to be some (arginine rich?) binding mechanism on the KAS, CLF or ACP to stabilise the reactive polyketo chain as an enolate and to prevent premature unwanted cyclisations. Perhaps the KAS assembles the chain on a ‘variable length binding pocket’ until a certain length is achieved, when it preferentially binds to the ‘CLF’ via the Ser and a ‘defined length binding pocket’, which is arranged so that a KR binds (most probably intimately bound all the time) to reduce at C-9 (if required), and a CLF aldolase activity promoting regiospecific first ring cyclisation.

Sherman *et al.* have functionally replaced *actI-ORF1* and *actI-ORF2* genes by the corresponding Gra genes. ${ }^{73}$ Sherman *et al.* then replaced various genes in the Act cluster, inserted the altered cluster into *S. lividans*, and looked for blue pigment (actinorhodin) production. The ‘KAS’ genes *otc1* and *whiE-ORFIII* (WhiE, spore pigment from *S. coelicolor* of unknown structure) were both capable of functionally replacing *actI-ORF1* (KAS), but not *actI-ORF2* (CLF). However, the ‘CLF’ genes *otc2* and *whiE-ORFIV* were both unable to replace either *actI-ORF1* or *actI-ORF2*. Thus the Act KAS can be replaced by a variety of Type II aromatic KASs, but the Act CLF can only by replaced by another benzoisochromanequinone CLF. Sherman also found that neither *E. coli fabB* KAS or rat FAS KAS domain could functionally replace either *actI-ORF1* (KAS) or *actI-ORF2* (CLF). ${ }^{74}$

Sherman *et al.* have investigated the role of the putative Act KAS active site Cys-169. The corresponding sequence in the *ActI-ORFII* CLF has a Gln. A vector containing the *actI*, -III, -VII and -IV genes from *S. coelicolor* produces aloesaponarin II 11. Replacement of the *actI-ORF1* with the C169Q (Gln) *actI-ORF1* mutant resulted in no detectable polyketide production. ${ }^{61}$

Khosla and Carreras isolated from *S. coelicolor* CH999 pSEK38 (plasmid SEK38 in strain CH999) a protein fraction containing 40% KAS/CLF that had the ability to form PK products SEK45 and SEK4b6 when supplied with radio- labelled malonyl CoA and PKS holo-ACP (acetyl CoA not required). ${ }^{75}$ However, further purification resulted in total loss of PK production. The other protein fraction that would restore this ability was purified and shown to be the FAS malonyl CoA:ACP transacylase, FabD MT. Khosla concludes that FabD MT is an essential component of PKS activity, and implies that it is essential for malonation of Fren holo-ACP. This is in contrast to the results of Simpson (*vide supra* 3.3.1.3) which show that the Act ACP is capable of self-malonation in the absence of an MT. Further purification of the KAS/CLF fraction gave a complex that behaved as an 168 kDa α₂β₂ heterotetramer by gel filtration. Gradient SDS-PAGE was able to separate the two proteins, only one of which was labelled by malonyl CoA which was larger and presumed to be the KAS (calc. $M_{r}$ 45016, for CLF 42523). The corresponding Tcm proteins, Tcm KAS (*tcmK*) and CLF (*tcmL*) behaved similarly, but appear as a heterodimer by gel filtration.

Khosla *et al.* have prepared cell-free extracts from *S. coelicolor* CH999 pSEK24 containing the genes for minimal Act PKS (KAS/AT, CLF and ACP) that upon addition of acetyl and malonyl CoA produces SEK45 and SEK4b6 (ca. 3 ng polyketide per mg total protein from 70% ammonium sulfate precipitation). ${ }^{76}$ The extract was also capable of producing both products from just malonyl CoA, by decarboxylation to obtain the primer unit. A cell-free extract of pSEK38 that in addition contains KR1, AROab and CYC2/3 activity, produced as major product DMAC 10, the major product of the *in vivo* Act PKS.

Using a plasmid (pZP4) additionally containing the *tcmO* (*O*-methyl transferase) gene, 8-methoxy DMAC was obtained.

3.3.1.5 Ketoreductase *KR1(9)* (*actIII*). Little appears to be understood about these proteins at the molecular level other than their sequences with cofactor binding domains and presumed function. Some KR1s appear to have a greater chain length specificity than others. *ActIII* will efficiently reduce the octaketide, nonaketide and dekaketide product of the respective minimal PKSs (Table 9). Jad *KR1* will reduce octaketide min PKS products, but Dps *KR1* will not (Table 14). Their overexpression, characterisation, and the effect of their addition to *in vitro* minimal PKSs is eagerly awaited. As these ketoreductases are no longer thought to reduce 3-oxoacyl groups, they need not be closely related to FAS KR, however their stereochemistry appears identical, *R*, as found in mutacin 7, opposite to that found in reduced fungal polyketides.

3.3.1.6 Cyclase *CYC2/3* (*actIV*) and aromatase *AROab* (*actVII*). Hopwood *et al.* have found that the Act AROab (in common with other benzoisochromanequinones AROabs) has an internally duplicated motif in which the N- and C-terminal halves are homologous ${ }^{77}$ with about 300 aa in total. ${ }^{78}$ Khosla then suggested that the N-terminal domains encode ‘first ring regiospecific cyclisation and second ring aromatases’ the latter only functioning after the second ring cyclase, and that the C-terminal half encodes the first ring aromatase, a reaction that in the absence of the *KR1(9)* would occur spontaneously. ${ }^{79}$ This assignment is based upon the observation that the N-terminal half has the greatest homology with *TcmN* whose role is now thought to include second ring cyclisation and aromatisation in addition to redirecting the regiospecificity of the the first ring cyclisation to (9/14), in addition to the *TcmD3 O*-methylation catalysed by the C-terminal half. The *TcmN* type (AROc) have about 150 aa in the AROc region. The WhiE aromatase (WhiE-ORFVI) has only around 150 aa, and despite being discrete, must be classed as AROc type, and would thus be expected to work best on unreduced systems, and its natural chain length is thought to be C-22 or C-24. Addition of WhiE-ORFVI to *Tcmmin* PKS gave RM80 25 and RM80b 26 (Scheme 7), the same products as obtained from *Tcmmin* PKS/*TcmN* (Scheme 4). However, WhiE-ORFVI had less effect upon reduced systems. Addition to *Actmin* PKS+*KR1* gave the non-ARO product mutactin 7, but also EM18 35 obtained presumably from first ring aromatisation. Addition to Fren Min PKS/*ActKR1* gave the nonaketide RM18 21.

A recent paper by Zawada and Khosla analyses the role of the various types of aromatase. ${ }^{78}$ The ActVII AROab type have been labelled ‘didomain aromatases’ or ‘didomain ARO/CYC’s, and the *TcmN* AROc type have also been labelled monodomain ARO/CYCs. A series of experiments are shown in Table 15 performed by fusing the N-terminal half of the ActVII gene, or the C-terminal half, and/or the Gris analogues, or the N-terminal portion of *TcmN*, with various minimal PKS systems, with the aromatase components being either fused, or expressed separately.

The didomain ActVII type, AROab, are thought to perform two dehydrations on the *KR1(9)CYC1(7/12)*. Expression of either the N-terminal or the C-terminal half alone failed to produce significant aromatase activity (resulting mainly in mutactin 7), suggesting that the two halves work ‘in concert’ with each other. The two separate half-proteins, when mixed, were active, giving SEK34 8. However, the presence of mutactin does indicate that whilst the two halves can partially reassociate to give partial aromatase activity, it is not as efficient as when they were originally covalently fused together. When added to *Tcmmin* PKS and *ActKR*, the two fused Gris halves resulted in some aromatised dekaketide, non-aromatised dec-

446 Nat. Prod. Rep., 1999, 16, 425–484aketide, and most interestingly, some aromatised nonaketide RZ 53 36 (Scheme 8). Unless this arises from retroaldol degradation of the decaketide RM 20 29, this shows that the AROab has affected overall chain length and the regiochemistry of KR1. The next set of experiments was with the insertion of fused halves. Addition of fused Act and Gris halves to Act min PKS and ActKR was functional to give SEK 34 8. However when attached to Tcmin PKS/ActKR1, whilst N-terminal Gris and C-terminal Act (or both halves Gris) was functional to give SEK 43 30, fusion of the N-terminal Act with either Act or the Gris C-terminal half was non-functional, though the chain length had been modified in both cases to give some nonaketide. There may be a difference between the two halves in chain length recognition. N-Terminal Act will only aromatise octaketides, whilst the corresponding Gris aromatises OCT/NON and DEC. There seems to be no discrimination in the C-terminal half, suggesting that chain length recognition resides in the N-terminal half. Thus it appears that part of the aromatase can influence chain length. The fusion of the N-terminal half of TcmN to either Act or Gris C-terminal half was then investigated. Fusion onto either Actmin PKS/KR or TcmMin PKS/ActKR gave non-functional hybrids, resulting in just min PKS+KR1 products.

### 3.3.1.7 Post-PKS reactions. The Act cluster contains six regions' not to be confused with individual Act genes or ORFs. Malpartida has sequenced and proposed functions for the six ORFs (ORFA, B, and 1–4) in the 'ActVI region' at the N-terminal end of the Act cluster.¹ ActVI mutants assemble aloesaponarin II 11. ActVI-ORF1 (KR3) resembles β-hydrox-yacyl CoA dehydrogenases, ORF2 and ORF4 resemble ERs,

suggesting that this region is responsible for the reductive steps leading to pyran ring formation.

The 'ActVB region' was thought to contain a single ORF (actVB) responsible for dimerisation as mutants lead to accumulation of kalafungin as a shunt product. Marsh et al. have overexpressed actVB (18.4 kDa) in E.coli.⁸⁰ The authors found that, instead of catalysing dimerisation as previously thought, ActVB is a flavin:NADH oxidoreductase, and is thus an essential auxiliary enzyme supplying reduced FMN to another enzyme in the pathway.

The 'ActVA region' is thought to contain ORFs coding for Act export genes and an oxidase responsible for quinoid formation and one for aryl hydroxylation.⁵¹ Marsh has examined individual ORFs in the ActVA region. ActVA-ORF6 has homology (39%) to the quinone forming dioxygen requiring monooxidase TcmH that converts TcmF1 to TcmD3. ActVA-ORF6 was overexpressed in E.coli (Mr 12 kDa) and was able to convert TcmF1 to TcmD3.⁸¹ The H26Q mutant was inactive, in agreement with earlier studies by Shen and Hutchinson that showed His modifying reagents inactivated TcmH.⁸²

A series of pigments related to Act has been isolated from S. coelicolor A3(2) by Bystrykh et al. and are thought to be late shunt metabolites due to mutations in the actII-ORF2, actII-ORF3 actVA-ORF1 and other genes.⁸³

### 3.3.2 Other benzisochromanequinones. Granaticin 15 is unusual in that a 2,6-dideoxyhexose moiety is unusually fused to a benzene ring via two ortho C-C linkages, not known in other natural products. Whilst the genes for the aglycone have been located⁸⁴a and heterologously expressed, ⁴⁶,⁷³ Floss et al. have now identified the gene responsible for glucose 4,6-dehy-

---

**Table 15 Effect of individual aromatase domains**

| Min PKS | KR | N-terminal | C-Terminal | Product (Ref 78) |
|---------|----|------------|------------|-------------------|
| **'Fused'** |  |  |  |  |
| Act     | Act | Act        | Mutactin   | OCT:Ac/KR1(9)CYC1*(7/12)CYC2(6/15)CYC3(O-1/5) |
| Act     | Act | Act        | Mutactin   | OCT:Ac/KR1(9)CYC1*(7/12)CYC2(6/15)CYC3(O-1/5) |
| Act     | Act | Act        | Mutactin   | OCT:Ac/KR1(9)CYC1*(7/12)CYC2(6/15)CYC3(O-1/5) |
|         |     |            | SEK 34     | OCT:Ac/KR1(9)CYC1(7/12)CYC2(O-11/15)          |
| Tcm     | Act | Act        | RM20       | DEC:Ac/KR1(9)CYC1(7/12)CYC2(11/16)CYC3(O-5/13) |
|         |     |            | RM20b,c    | DEC:Ac/KR1(9)CYC1*(7/12)CYC2(O-11/15)CYC3(14/19) |
| Tcm     | Act | Gris       | RM20       | DEC:Ac/KR1(9)CYC1(7/12)CYC2(11/16)CYC3(O-5/13) |
|         |     |            | RM20b,c    | DEC:Ac/KR1(9)CYC1*(7/12)CYC2(O-11/15)CYC3(14/19) |
|         |     |            | RZ53       | NON:Ac:KR1(7)CYC1(5/10)CYC2(9/14)CYC3(O-3/11)  |
| Tcm     | Act | Act        | RM20       | DEC:Ac/KR1(9)CYC1(7/12)CYC2(11/16)CYC3(O-5/13) |
|         |     |            | RM20b,c    | DEC:Ac/KR1(9)CYC1(7/12)CYC2(O-11/15)CYC3(14/19) |
| Tcm     | Act | Gris       | RM20       | DEC:Ac/KR1(9)CYC1(7/12)CYC2(11/16)CYC3(O-5/13) |
|         |     |            | RM20b,c    | DEC:Ac/KR1(9)CYC1*(7/12)CYC2(O-11/15)CYC3(14/19) |
| Act     | Act | Gris       | SEK 34     | OCT:Ac/KR1(9)CYC1(7/12)CYC2(O-11/15)           |
| Tcm     | Act | Gris       | RM20b      | DEC:Ac/KR1(9)CYC1*(7/12)CYC2(O-11/15)CYC3(14/19) |
|         |     |            | SEK 43     | DEC:Ac/KR1(9)CYC1(7/12)CYC2(14/20)CYC3(O-1/5)   |
|         |     |            | RZ53       | NON:Ac:KR1(7)CYC1(5/10)CYC2(9/14)CYC3(O-3/11)  |
| Act     | Act | Act+Gris   | SEK 34     | OCT:Ac/KR1(9)CYC1(7/12)CYC2(O-11/15)           |
| Act     | Act | Gris+Act   | SEK 34     | OCT:Ac/KR1(9)CYC1(7/12)CYC2(O-11/15)           |
| Tcm     | Act | Act+Gris   | RM20       | DEC:Ac/KR1(9)CYC1(7/12)CYC2(11/16)CYC3(O-5/13) |
|         |     |            | RM20b,c    | DEC:Ac/KR1(9)CYC1*(7/12)CYC2(O-11/15)CYC3(14/19) |
|         |     |            | RZ53       | NON:Ac:KR1(7)CYC1(5/10)CYC2(9/14)CYC3(O-3/11)  |
| Tcm     | Act | Gris+Act   | SEK 43     | DEC:Ac/KR1(9)CYC1(7/12)CYC2(14/20)CYC3(O-1/5)   |
| Tcm     | Act | Act+Act    | RM20       | DEC:Ac/KR1(9)CYC1(7/12)CYC2(11/16)CYC3(O-5/13) |
|         |     |            | RM20b,c    | DEC:Ac/KR1(9)CYC1*(7/12)CYC2(O-11/15)CYC3(14/19) |
|         |     |            | RZ53       | NON:Ac:KR1(7)CYC1(5/10)CYC2(9/14)CYC3(O-3/11)  |
| Tcm     | Act | Gris+Gris  | SEK 43     | DEC:Ac/KR1(9)CYC1(7/12)CYC2(14/20)CYC3(O-1/5)   |
| Act     | Act | Tcm+Act    | Mutactin   | OCT:Ac/KR1(9)CYC1*(7/12)CYC2(6/15)CYC3(O-1/5)   |
| Act     | Act | Tcm+Gris   | Mutactin   | OCT:Ac/KR1(9)CYC1*(7/12)CYC2(6/15)CYC3(O-1/5)   |
| Tcm     | Act | Tcm+Act    | RM20       | DEC:Ac/KR1(9)CYC1(7/12)CYC2(11/16)CYC3(O-5/13) |
|         |     |            | RM20b,c    | DEC:Ac/KR1(9)CYC1*(7/12)CYC2(O-11/15)CYC3(14/19) |
|         |     |            | RZ53       | NON:Ac:KR1(9)CYC1(5/10)CYC2(9/14)CYC3(O-3/11)  |
| Tcm     | Act | Tcm+Gris   | RM20       | DEC:Ac/KR1(9)CYC1(7/12)CYC2(11/16)CYC3(O-5/13) |
|         |     |            | RM20b,c    | DEC:Ac/KR1(9)CYC1*(7/12)CYC2(O-11/15)CYC3(14/19) |
|         |     |            | RZ53       | NON:Ac:KR1(9)CYC1(5/10)CYC2(9/14)CYC3(O-3/11)  |

---

**Nat. Prod. Rep., 1999, 16, 425–484** 447
dratase in *S. violaceoruber* Tü22 and found it linked to the PKS cluster. ${}^{84b}$

In a recent paper, Hopwood *et al.* have sequenced the entire granaticin cluster as two overlapping cosmids (39 250 bp, 37 ORFs) inserted into CH999 which resulted in the production of granaticin, granaticin B, dihydrogranaticin and dihydrogranaticin B. ${}^{85}$ Gra B is similar in structure to Gra, but with a deoxysugar rhodinose attached to one of the sugar hydroxy groups. Dihydro Gra does not have a hydroxy group available to form the lactone, as it has been fully reduced to a methylene, so the acid group is ‘free’. The cluster had 15 ORFs homologous with those in the Act cluster, nine potentially involved in sugar metabolism, and six are thought to be regulatory. The actual Gra cluster is thought to comprise just 33 of these 37 ORFs, compared with 22 thought to be in the Act cluster, which appears reasonable as at least nine are involved in the two deoxysugars needed to biosynthesise Gra B. The genes are labeled from N to C, gra9 to gra29, gra6,5,1,2,3,4, and gra30 to gra34. The first gene gra9 has homology with *actII-ORF4* a transcriptional activator; gra15 with *actII-ORF2* a transmembrane export protein; gra18 with *actVI-ORF3*, a cyclase/dehydratase; gra21 with *actVA-ORF5*, a hydroxylase; gra28 with *actVA-ORF3*, a tailoring protein of unknown function; gra33 with *actIV*, CYC2; gra34 with *actVB*, an FMN:NADH oxidoreductase. The sequence of these homologous genes does not exactly match that found in the Act cluster.

Hopwood *et al.* have cloned and characterised a 5.4 kbp segment of the griseusin 16 cluster containing KAS, CLF, ACP, KR and a CYC. ${}^{86}$ Griseusin may be an octaketide with a hexanoate starter, or most likely a dekaketide with an acetate starter. The stereostructure shown in the literature for griseusin B is not always consistent.

---

**Octaketide**

![Chemical Structure](chemical_structure_image)

---

**Nonaketide**

![Chemical Structure](chemical_structure_image)

---

**Dekaketide**

![Chemical Structure](chemical_structure_image)

---

**Scheme 8**

448 Nat. Prod. Rep., 1999, 16, 425–484

3.4 7,9,12-Nonaketides

### 3.4.1 Tetracyclines.
The sequence of the oxytetracycline synthase gene (otc) in *S. rimosus* has been published. A review by Hunter and Hill has appeared.

### 3.4.2 Frenolicin B.
The Fren cluster has been cloned and characterised. Fren minimal PKS (containing Act ACP) with Act KR1 has a relaxed chain length, producing a mixture of the  
KR1(9)CYC1(7/12) octaketide mutactin  
KR1(7)CYC1(5/12) octaketide RM18b  
KR1(9)CYC1(7/12) nonaketide RM18 21 (Scheme 3, Table 2). Khosla et al. found that the minimal PKS alone, produced only CYC1(7/12) octaketides SEK4 5 and SEK4b 6. Addition of TcmN to this minimal PKS produced a CYC1(9/14) folded nonaketide PK8 19, with no evidence of octaketide production. Thus the Act KR and the TcmN are affecting both the folding pattern and the chain length of the final product. Presumably the fren PKS has relaxed chain length requirements, forming nanaomycin 18 as its octaketide product, and frenolicin 17 as its nonaketide product, presumably with a different set of post-PKS enzymes working on the two different chain length PKS products.

Rhinehart et al. have isolated spectomycin A2 37 and its C-7 linked dimer spectomycin B1 from *S. spectabilis* and shown its nonaketide incorporation pattern. Whilst chemically related to aureolic acids, its biosynthesis would appear to be 7,9,12-nonaketide.

![Chemical Structure](https://i.imgur.com/structure.png)

**Spectomycin B1**  
(monomer is spectomycin A2 37)

3.5 7,9,12-Decaketides

Two main types, the ‘linear’ anthracyclines based upon a 7,8,9,10-tetrahydro-5,12-naphthacenequinone nucleus represented by doxorubicin 50 and daunorubicin 49, and the ‘non-linear’ angucyclines based upon the benzo[a]anthraquinone structure (often subsequently ring opened) that include tetrangulol 63, jadomycin B 69 and the kinamycins 73.

#### 3.5.1 Anthracyclines.
Hutchinson has reviewed the biosynthesis of the 7,9,12-anthracyclines and the 9,14-tetra-cenomycins, and Ebizuka and Fujii have reviewed anthracycline biosynthesis in *S. galilaeus*. These are all thought to be assembled via a common polyketide synthase product as intermediate, aklavinone 45, assembled from a propionate starter and nine malonate extender units, or close analogues with different starter units (Scheme 9). Ring four keto reduction and glycosylation gives aclacinomycin 44. Ring four keto reduction and monoglycosylation along with demethoxycarbonylation and side chain oxidation gives carminomycin 48, O-Met then gives daunorubicin 49, which is then side-chain oxidised to doxorubicin 50. Curiously, there is no aromatisation in the fourth ring, and all have reduction at C-17. This is the usual site of glycosylation, the agylcones are usually glycosylated with up to five sugar units at the C-2 or C-17 hydroxy groups, resulting in antimicrobial activity. The wide variation of structures is illustrated (Scheme 9): an acetate starter unit gives auramycinone 52, which after glycosylation gives the auramycins and nogalamycins. A propionate starter unit with variable oxidation levels can give β-rhodomycinone 53, ε-pyrromycinone 54, adriamycinone 55, and daunomycinone 56, whilst a butyrate starter unit results in sulfurmycinone 57. All the anthracyclines would be expected to have an similar gene clusters. The aklavinone and nogalamycin (*S. nogalater*) PKS clusters (Akn and Sno) are shown in Fig. 3, and those for daunorubicin and doxorubicin clusters in Fig. 4. Two main features of interest are firstly that the daunorubicin and doxorubicin clusters have an extra KAS and CLF like genes (*dpsCD*), not present in the aklavinone or nogalamycin clusters, that have until recently been considered involved in determining the starter unit to be propionate; and secondly, the Dps ACP (*dpsG*) is some distance from the KAS/CLF.

##### 3.5.1.1 Aklavinone.
The origin of the carbon and oxygen atoms of aklavinone 45 from *S. galilaeus* have been determined, and all oxygens except one quinone carbonyl are acetate or propionate derived. A series of mutants obtained from an aklavinone producing strain of *S. galilaeus* suggests that oxidation of ring two occurs before cyclisation of ring four, which appears to occur after methylation of the acid, which presumably facilitates anion ring formation for CYC4. Thus the order of intermediates would be aklanonic acid 41, aklanonic acid methyl ester 42, aklaviketone 43 then aklavinone 45.

The aklavinone PKS gene cluster (Akn) from *S. galilaeus* has KAS (*aknB*), CLF (*aknC*), ACP (*aknD*), with in the opposite direction, upstream from KAS, *aknX* (homologous to *tcmH*, an Ox gene), *aknA* [KR1(9)], and *aknE* (CYC) (Fig. 3). A mutant strain of *S. galilaeus* produces 2-hydroxyaklavinone 40, presumably by inactivation of the AknA KR1(9) gene. Insertion of *actIII* KR1(9) into that mutant strain gave aklavinone as usual. Addition of *actI* (KAS+CLF) to *S. galilaeus* gave the octaketide aloesaponarin II 11 as produced by minimal Act PKS plus ActIII KR. Addition of ActI (KAS+CLF) to the KR1(9) deficient mutant of *S. galilaeus* gave desoxyerythrolaccin (hydroxyaloesaponarin II), demonstrating the interchangeability of the Act and Akn KR1(9).

The steffimycins are thought to be 7,12-dekaketides (7,12-anthraquinones), with no KR1(9). Ylihonko et al. have added an insert from Sno containing ARO/KR1/O-Met/OX/Activator genes to *S. steffisburgensis* and found products now reduced at C-9, as well as further modified with oxidations and methylations.

Ylihonko, Salas et al. have examined the product of a KR1 deficient mutant strain of *S. galilaeus* that normally produces aclacinomycin 44 and obtained some 40, but that the main product, enterprisingly christened ‘compound 1’, was the result of CYC1(10/15). Thus, in this case, KR1(9) has an important role in dictating the regiochemistry of the first cyclisation from (10/15) to (7/12).

Overexpression of AknX in *E. coli* gave a protein trimer, with an essential tryptophan residue, capable of catalysing oxidations to form anthrones.

##### 3.5.1.2 Daunorubicin (daunomycin or rubidomycin) and doxorubicin (adriamycin).
A large part of the daunorubicin (Dau or Dnr) cluster has been characterised (Fig. 4). The PKS part of the cluster for Dau and doxorubicin 50 (Dps or Dox) have over 90% identity, and some gene labels have recently been standardised, the daunorubicin and doxorubicin PKS genes are all now referred to as ‘dps’ genes. The full clusters for daunorubicin (*dau/dnm/dps*) from *Streptomyces sp.* C5 and doxorubicin (*drr/dnr/dps/dnm*) from *S. peucitius* are illustrated in Fig. 4. The PKS gene cluster differs significantly from other Type II PKS clusters, in that the ACP is distant from the main KAS and CLF, and that there is an *E. coli* KASIII analogue, DauC or DpsC and adjacent AT (PrT?) that has been presumed to be involved in choosing the propionyl starter unit, and perhaps catalysing the first condensation. Insertion of the

Steffimycin
38
(Acetate starter unit)

Propionate
+ AknBCD
or DpsABCDG
9 x Malonate

S. steffisburgensis
(no KR1(9))

Mutants with no KR1(9)

COSEnz

AknA
or DpsE

COSEnz

HO
9
O
O
O
O
O
O
Me

HO
7
O
OH
12
O
OH
14
O
OH
16
O
OSugar

HO
13
O
OH
10
O
OH
8
O
OH
6
O
OH
1
O
OH
21
O
'Me

'Compound 1'
39

HO
7
O
OH
12
O
OH
14
O
OH
16
O
OH
Et
OH
CO2Me

2-Hydroxyaklavinone
40

HO
9
O
CO2H
O
O
O
O
O
O
Me

HO
7
O
OH
12
O
OH
14
O
OH
16
O
OH
CO2H
O
OH
O
OH
Me

12-Deoxyaklanonic acid
40a

Aklanonic acid
41

HO
7
O
OH
12
O
OH
14
O
OH
16
O
OH
CO2Me
O
OH
Me

Aklanonic acid, methyl ester
42

DauC
or DnrC

HO
7
O
OH
12
O
OH
14
O
OH
16
O
OH
CO2Me
O
OH
Me

DauD
or DnrD

HO
7
O
OH
12
O
OH
14
O
OH
16
O
OH
CO2Me
O
OH
Et
OH

Aklaviketone
43

DnrE

HO
7
O
OH
12
O
OH
14
O
OH
16
O
OH
CO2Me
O
OH
Et
OH

Aklavinone
45

Me
O
NMMe2

Me
O
HO
O
O

Aclacinomycin
(S. galilaeus)
44

HO
7
O
OH
12
O
OH
14
O
OH
16
O
OH
CO2Me
O
OH
Et
OH

ε-Rhodomycinone
46

DnrF

HO
7
O
OH
12
O
OH
14
O
OH
16
O
OH
CO2Me
O
OH
Me
OH

10-Methoxycarbonyl-13-deoxycarminomycin R = H, R' = H Carminomycin 48
(Rhodomycin D)
47

DnrK

R = H, R' = Me Daunorubicin (S. peucetius) 49
(Daunomycin)

DoxA

R = OH, R' = Me Doxorubicin (S. peucetius 50
var. caesius)

DnrH
DnrX

Baumycins
51

Variations on a theme

Acetate starter unit

R1 = Me, R2 = H, R3 = H Auramycinone (S. galilaeus) (glycosylation gives auramycins and nogalamycins from S. nogalater)
52

Propionate starter unit

R1 = Et, R2 = H, R3 = H Aklavinone (S. galilaeus) 45

R1 = Et, R2 = H, R3 = H, 2,4-dihydroxy, 2-Demethoxycarbonyl , β-Rhodomycinone (S. purpurascens) 53

R1 = Et, R2 = OH, R3 = H, epimeric at C-19 ε-Pyrromycinone (S. antibioticus) 54

R1 = COCH2OH, R2 = H, R3 = Me, 11-deoxy-2-demethoxycarbonyl Adriamycinone (S.peucetius) 55

R1 = COMe, R2 = H, R3 = Me, 4-hydroxy, 2-demethoxycarbonyl Daunomycinone (S.peucetius) 56

Butyrate starter unit:

R1 = CH2COMe, R2 = H, R3 = H Sulfurmycinone (S. galilaeus) 57

Scheme 9

450 Nat. Prod. Rep., 1999, 16, 425–484

Dps cluster except the gene for ACP ($dpsG$) into $S$. lividans  
TK24 did not produce aklanonic acid 41.  
Hutchinson *et al.* have shown that the putative ‘PrT’ DpsD  
was not essential for aklanonic acid production *i.e.* selection of  

a propionate starter.99 Strohl *et al.* then inserted $dpsG$, $daul$,  
$dpsABEF$ and $dauG$ into $S$. lividans TK24 and obtained efficient  
formation of aklanonic acid 41, showing that DpsC and DpsD  
are not required to specify a propionate starter.100 As Meurer  

| SUG | $dnmL$ Synthesis of TDP-glucose |
| --- | --- |
|   | $dnmM$ TDP-4-keto-6-deoxyglucose |
|   | (has frameshift error-inactive protein?) |
|   | $dnrN$ Regulatory |
|   | $dnrO$ Unknown function |
| OX | $dnrF$ Ox: Aklavinone to ε-rhodomycinone |
|   | $drrA$ Resistance |
| RES | $drrB$ Resistance |
|   | $drrC$ Resistance |
|   | $dnrX$ Conversion of doxorubicin into baumycins |
| PKS | $dpsY$ CYC 2/3? |
|   | $dnmZ$ Unknown function |
| SUG | $dnmU$ late step in daunosamine synthesis? |
|   | $dnmV$ Reduction of 4-ketone |
| $dnmJ$ or $dauJ$ | $dnmJ$ Introduction of C-3 amino group |
| $daul$ | REG | $dnrl$ Regulatory/transcriptional activator |
| $doxA$ | OX | $doxA$ or $dxrA$ Oxidation of daunorubicin to doxorubicin |
| ORFA | $dnrV$ |   |
| ORF1 | OX | $dnrU$ Oxidation |
| $dpsG$ or $dauA(g)$ | PKS | $dpsG$ ACP |
| $dauZ$ or ORF2 | PKS | $dpsH$ (formerly $dnr-ORF7$) Putative second/third ring cyclase |
| $dmnT$ or ORF3 | SUG | $dmnT$ Involved in C-2 deoxygenation? |
| $dauH$ | SUG-Tr | $dnrH$ Polyglycosylation to baumycins? |
| $dauE$ | KR | $dnrE$ (formerly $dnrH$) Aklaviketone to aklavinone reduction |
| $dpsF$ or $dauA(f)$ | PKS | $dpsF$ ARO/CYC 1(7/12) |
| $dpsE$ or $dauB$ | OOX | $dpsE$ KR1(9) |
| $dauG$ or $dauA(e)$ | $dnrG$ Anthraquinone formation |
| $dpsA$ or $dauA(a)$ | PKS | $dpsA$ KAS or KSα (+ AT?) |
| $dpsB$ or $dauA(b)$ | PKS | $dpsB$ CLF or KSβ |
| $dpsC$ or $dauA(c)$ | $dpsC$ KASIIL analogue |
| $dpsD$ or $dauA(d)$ | MeT | $dpsD$ AT, possible PrT? |
| $dauC$ | CYC | $dnrC$ Aklanonic acid MeT |
| $dauD$ | MeT | $dnrD$ Aklanonic acid methyl ester cyclase |
| $dauK$ | $dnrK$ Carminomycin MeT |
| $dauP$ | $-CO_2$ | $dnrP$ Unknown function, loss of $CO_2$Me? |
| SUG | $dnmQ$ or $dnrQ$ C-2 deoxygenation? |
|   | $dnmS$ or $dnrS$ TDP-L-Daunosamine glycosyltransferase |
| Insertion element |   |
| RES | $drrC$ Resistance |
| C-terminus |   |

Fig. 4 Gene map for daunorubicin/daunomycin cluster in $S$. sp. C5 and doxorubicin/adriamycin in $S$. peucetius. Gene labels for PKS region have been recently changed to $dpsABCDEF$ for consistency.
and Hutchinson showed that DpsA and DpsB alone did not specify propionate starter, Strohl suggests that the protein interactions between the KAS/CLF/CYC/KR when all together, specify the starter unit, making the prediction of various ‘mix and match’ experiments rather difficult! However, these observations could also be explained if the *S. lividans* possessed a fatty acid KASI which could substitute for DpsD. Strohl remarks that the sequence of DpsD is closer to an acetyl transferase, than to known propionyl transferases.

Strohl *et al.* have identified the genes responsible for the conversion of *O*-methyltransferase that converts aklanonic acid 41 into its methyl ester 42 (*dauC*), and then its cyclisation into aklaviketone 43 (*dauD*), and for the *O*-methylation of carminomycin 48 into daunorubicin 49 (*dauK*).<sup>98</sup> Hutchinson and Scotti have immobilised and stabilised carminomycin 4-*O*-methyl transferase (DnrK).<sup>101</sup> Hutchinson and Madduri examined a similar region of DNA, and suggest that as DnrP has homology with esterases, it is involved in saponification of the methyl ester prior to decarboxylation to carminomycin 49.<sup>102</sup>

Filippini *et al.* have expressed the 52 kDa 489 aa DnrF protein in *S. lividans* TK23 and *E. coli* and found similarities with FAD and NADPH dependent polycyclic aromatic hydroxylases, and that DnrF had high substrate specificity.<sup>103</sup> Strohl *et al.* have expressed the NADPH dependent 25 kDa aklaviketone reductase (DauE) from *S.* sp. C5 in *S. lividans*,<sup>104</sup> which was 97% homologous to DnrH sequenced by Grimm *et al.*<sup>99</sup> Strohl and Dickens have expressed and characterised the 46 kDa putative cytochrome P-450 enzyme DoxA from *S.* sp. strain C5 in *S. lividans* TK24 and shown that it converts daunomycin to doxorubicin, which requires hydroxylation of the terminal methyl group.<sup>105</sup>

Hutchinson *et al.* have found that *S. peucetius dnrH* mutants with a high copy plasmid containing *dnmT* had an 8.5 fold increase in doxorubicin production, whilst *dnmT* mutants accumulated ε-rhodomycinone 46.<sup>106</sup> As DnrH was homologous to glycosyltransferases, Hutchinson suggested that it may be involved in assembly of the polyglycosylated derivatives such as the baumycins 51. Hutchinson *et al.* have characterised the *dnmZUV*<sup>107</sup> and the *dnmQS* (*dnrQS*) genes<sup>108</sup> from *S. peucetius* that are involved in the biosynthesis and glycosyl-transfer of the amino sugar daunosamine, and lists the known functions of the *dnm* genes. Wild type *dnmM* contains a frameshift and thus produces inactive protein. Presumably, outside of the cluster there is an alternative TDP-glucose 4,6-dehydratase that provides the desired activity.<sup>109</sup>

Hutchinson *et al.* have overexpressed the DnrI protein from *S. peucetius* in *E. coli* and showed that it bound to the intergenic DNA regions between *dnrGdpsABCD* and *dpsEF* operons, and between the *dnrC* gene and the *dnrDKPdnmSQ* operon.<sup>110</sup> Insertional inactivation of *dnrI* in *S. peucetius* prevents transcription of *dmnLM*, *dnrXY*, *dnmZUV*, *dpsABCD*, *dnrDKPdnmQS*, *drrAB* and *drrC* operons or genes. DnrN may be required for the expression of *dnrI*.<sup>111</sup> The sequence of DnrI has similarity to the ActII-ORF4 transcriptional activator, and *dnrI* can complement an actII-ORF4 mutation.<sup>112</sup> Hutchinson and Furuya have overexpressed DnrI in both *E. coli* and *S. peucetius* and found that the protein specifically bound to the *dnrI* promotor region, which has a similar sequence to that favoured for binding by daunorubicin, which may explain product inhibition.<sup>113</sup> Hutchinson *et al.* found that expression of the 764 aa DrrC protein in *E. coli uvrA* strain conferred daunorubicin resistance to the normally very sensitive *E. coli* strain.<sup>114a</sup>

In a recent paper, Hutchinson *et al.* have further investigated the role of *S. peucetius dpsY* and *dnrX*.<sup>114b</sup> Disruption of *dpsY* led to the accumulation of UWM5, a homologue of SEK43 30 (Scheme 5) in which the methyl group is replaced by an ethyl group. Thus DpsY must be involved in the assembly of 12-deoxyaklanonic acid 40a (Scheme 9) and Hutchinson suggests it is involved in CYC2/3. Disruption of *dnrX* led to an

increase in production of doxorubicin 50, but prevented accumulation of any baumycins 51, and thus must be involved in a late step.

### 3.5.1.3 Nogalamycin

Nogalamycin (*S. nogalater*) is assembled from ten acetate units (acetate starter) and is epimeric to aklavinone 45 at C-9, with a methyl side chain. Ylihonko *et al.* have analysed 11 kbp of the nogalamycin gene cluster (Sno). Unlike the Dps cluster, the ACP is next to the KAS and CLF as normal.<sup>115</sup> The PKS gene cluster (Sno) was similar to that for aklavinone (Akn) (Fig. 3). When the Sno minimal PKS was inserted into mutant *S. galilaeus*, the major product was aklavinone, with smaller quantities being formed of auramycinone, which has an acetate starter but the same stereochemistry as aklavinone at C-9. Thus the nogalamycin minimal PKS, whilst it can determine the starter unit, can efficiently utilise the propionate acyl transferase from *S. galilaeus*.<sup>115,116</sup>

Ylihonko and Salas have inserted the Sno minimal PKS into the minimal PKS deficient aureolic acid mithramycin producer *S. argillaceus* and generated mithramycin 83 (Mtm) (Scheme 13).<sup>95</sup> As Mtm was considered a CYC1(9/14) metabolite at the time, this observation was used to demonstrate how the later PKS components such as KR1 can influence the regiochemistry of CYC1. However, since this paper, Salas *et al.* have shown that the aureolic acids are probably CYC1(7/12) compounds.<sup>117</sup>

#### 3.5.1.4 Choice of starter unit

Aklavinone derived anthraquinones use a propionate starter (Scheme 9), whilst nogalamycin uses acetate. The gene cluster (Sno) that produces nogalamycin from *S. nogalater* has a similar arrangement for the ‘minimal PKS’ as that for aklavinone with the ACP located just after the CLF.<sup>115</sup> However, the ACP in both the daunomycin cluster (Dau) from *S.* sp. C5 and doxorubicin cluster (Dps) from *S. peucetius* were located some distance from the KAS.<sup>97,99</sup> In addition, they both have what appears to be a second KAS downstream next to the CLF, followed by a putative propionyl transferase gene, neither of which have yet been located in the Akn or Sno clusters (some substrate starter unit flexibility is observed in the aklavinone producing strain of *S. galilaeus*, and *S. nogalater* uses an acetate starter).

Hutchinson and Meurer have inserted Dps and Tcm genes into non-producing strains of *S. lividans* and *S. glaucescens* to examine the role of the putative KAS and CLF proteins in determining the choice of starter and position of the first cyclisation.<sup>48</sup> Whilst insertion of *dpsABtcmMN* gave no detectable products, *tcmJdpsABtcmMN* gave Tcm F2 (Table 14). This confirmed the role of DpsAB as KAS and CLF as they were able to replace the *tcmKL* genes in the production of Tcm F2. However, in the presence of the TcmN, the minimal Dps proteins had no influence on the regiospecificity of the first cyclisation, as instead of occurring at CYC1(7/12) it occurs at (9/14) as in the tetracyclines. In addition, acetate was used as the starter unit, rather than propionate, implying that other proteins, such as DpsC and DpsD, may be responsible for selecting starter units other than acetate (*vide infra*). TcmJ, though unknown in function, was also found essential for producing reasonable yields from Tcm min PKS. Insertion of just *tcmJdpsABtcmM* (no *tcmN*) produced both SEK15 23 CYC1(9/14) and SEK15b 24 CYC1(7/12), consistent with the formation of two compounds by either the Act or the Tcm minimal PKSs, the CYC1 probably occurring non-enzymatically with just the minimal PKSs.

Niemi and Mäntsälä have identified six open reading frames (*rdmA-E*) in *S. pururascens* that when inserted into *S. galilaeus* produce novel anthracyclines.<sup>118</sup>

### 3.5.2 Angucyclines (benzo[a]anthraquinone decaketides)

The largest class of decaketides including urdamycin, kinamy-
452 Nat. Prod. Rep., 1999, 16, 425–484
cins, galtamycin, vineomycin, aquamycin, tetrangulols, jadomycons, and landanomycin. A major review by Rohr and Thiericke classified a large number of metabolites biosynthetically derived from DEC:KR1(9) CYC1(7/12) CYC2(5/14) CYC3(4/17) CYC4(2/19) as angucyclines or angucyclinones including some rearranged metabolites or ring contracted metabolites such as the kanamycins and jadomycins. ${}^{119}$ All are 7,9,12-decaketides, either with no further reduction (dehydro-rabelomycins, including kinamycin D 73 and jadomycin B 69); one extra reduction KR3(15) to give the tetrangulols 60–68, or with both KR2(13) and KR3(15) reduction to give land-anomycins 59 which were covered in the previous review ${}^{1}$ (Scheme 10). After CYC1, KR2(13) seems to prevent OOX(6), so these landanomycins do not have the anthraquinone structure found in the other angucyclines, and indeed the benzoisochromanquinones as well. The regiochemistry of CYC3(4/17) seems to dictate the angular ring structure, but work on the jadomycin synthase suggests that this regiochemistry is not dictated by the so-called cyclase CYC2/3, and must be directed by another cyclase (CYC4?) elsewhere in the cluster. Many angucyclines have extensive post PKS elaboration, oxidative cleavage gives the kinamycins (e.g. 73) and fridamycin E 60, whilst epoxidation of 66 and hydrolysis of 67 gives the vicinal diol system in PD116740 68 that on initial inspection would be incorrectly expected to arise from an acetate derived hydroxy and a P-450 oxidation. The putative landanomycin precursor 59 in Scheme 10 needs extensive oxidation and glycosidation to give landanomycin itself.

The timing of KR2(13) and KR3(15) is not yet clear. Most schemes show full reduction of the polyketo-chain prior to any cyclisations, based upon the precedent of fungal processive assembly, and KR1(9) occurring prior to CYC1(7/12). This would be DEC:Ac/KR1(9)KR2(13)KR3(15)CYC1(7/12). However, Scheme 10 shows the reductions occurring at the last possible moment, as this makes any pathway branching as late as possible, minimising the number of different steps that need to evolve. In addition, this is chemically more attractive, as it would be much easier to formally produce a carbanion for ring cyclisation between two carbonyls, rather than between a carbonyl and a hydroxy. It is attractive to suggest that KR3(15) dictates the unusual regiochemistry of CYC3(4/17), however the dehydrorabelomycin pathway does not involve a KR3(15).

### 3.5.2.1 Dehydrorabelomycins

Gould has recently reviewed the biosynthesis of the kinamycins ${}^{120}$ produced by *S. murayamaensis* whose structure has recently been revised to 5-azobenzo[b]fluorenes having been reported for many years as benzo[b]carbazole cyanamides. ${}^{1,121,122}$ Assembled via oxidative ring cleavage of dehydrorabelomycin 58, ${}^{1}$ Gould and coworkers have clarified the later stages in the pathway by synthesising kinobscurinone 70, stealthin C 71 and prekinamycin 72, isolated them from fermentation extracts, and by incorporating deuterated samples shown that kinobscurinone and stealthin C are intermediates in kinamycin D 73 biosynthesis (Scheme 11). ${}^{123-125}$ An isolate, previously thought to be prekinamycin, ${}^{124}$ has been shown not to be, though its structure remains unclear. The intermediacy of stealthin C 71 demonstrates that the two diazo nitrogens come from different amine sources. Gould et al. have cloned part of the cluster from *S. murayamaensis* into *S. lividans* and obtained the formation of kinobscurinone 70, stealthin C 71, and seongomycin 74 containing the N-acetylcysteine moiety. Although the role of this metabolite is unclear, it appears that the genes responsible for the addition of the side chain reside in the cluster. ${}^{126a}$ In a recent paper, Gould et al. describe the heterologous overexpression of one (*kin*) of the two clusters in *S. lividans ZX7* and isolated dehydrorabelomycin 58, kinobscurinone 70 and stealthin C 71. ${}^{127b}$ Thus this portion of the cluster is responsible

for the cleavage of the angucycline skeleton and the formation of the benzo[b]fluorene skeleton.

Roggo et al. have isolated balmoralmycin (a C-lipoglycosylated derivative of 58) from a *Streptomyces* sp. ${}^{127}$ They also isolated small quantities of SEK15 23 (major product from Tcmmin PKS) and SEK43 30 (major product from Tcmmin PKS and KR1) from the fermentation medium, suggesting that the KR1(9) occurs before subsequent cyclisations.

*Streptomyces venezuelae* ISP5230 normally produces chloramphenicol. However, replacing the glucose in the isoleucine containing medium by galactose, and raising the temperature to $37^{\circ}\mathrm{C}$, it produces a series of red pigments, one of which is jadomycin B 69. ${}^{128,129}$ Ayer also isolated the benzo[a]anthraquinone dehydrorabelomycin 58, and the benzo[a]phenanthridine of undetermined structure also found as a shunt metabolite by Gould et al. from UV mutants of kinamycin D producing microorganisms. ${}^{130}$ Vining et al. have cloned and characterised five ORFs corresponding to the KAS, CLF, ACP, KR1 and an AROab ${}^{131}$ thought to be those for JadB (Fig. 3) and found a close resemblance to those of the decaketide benziso-chromanequinone griseusin 16 with the KR1(9) placed between the ACP and the AROab. Vining et al. have identified two genes upstream of the PKS cluster ($jadR1$, $jadR2$) that resembled the repressor genes $tetC$, $tcmR$ and most closely the $E.\ coli\ envR$. Disruption of $jadR2$ led to mutants that produced Jad B 69 without the heat or ethanol stress treatment. ${}^{132}$

A recent paper by Meurer et al. examines the function of various Tcm, Dps and Jad genes in hybrid mixtures in *S. lividans* or *S. glaucescens* (Table 14 and related text). ${}^{47}$ Both the JadD or DpsF AROab functioned as ARO1/CYC1(7/12) on C-9 reduced decaketides as when added to DpsE KR1 and Tcm or Dps minimal PKS gave SEK43 30 and UWM 4 31 (Scheme 5). As expected, neither JadC nor DpsF played any role in promoting further cyclisations such as performed by TcmN AROc on unreduced systems. Omission of DpsE (KR1) resulted in a mixture of the CYC1(7/12) and CYC1(9/14) products SEK15 23 and SEK15b 24 as produced by just the Tcm or Dps minimal PKSs (Scheme 4). Thus the JadD AROab does not process C-9 unreduced decaketides. Any decaketide minimal PKS (Tcm, Dps or Jad) with any combination of Jad/Dps KR1 and AROab gave a mixture of SEK43 30 and UWM 4 31, both of which have a controlled KR1(9)CYC1(7/12) and ARO1 followed by non-enzymatic cyclisations. The JadD AROab appears to be similar to other KR1(9)CYC1(7/12)s in that it only affects the cyclisation and aromatisation of the first ring (and presumably ARO2 after CYC2), and has no role to play in the formation of subsequent angular ring cyclisation CYC3(14/17) that differentiates the angucyclines from the ‘linearly fused’ anthracyclines. However, in other contexts, it has been shown that DpsF is the only aromatase/cyclase required to form the tricyclic aklanonic acid. ${}^{100}$

The JadD or DpsF AROab failed to act upon either the Act KR1(9) reduced, or unreduced, octaketide produced by the Act Min PKS (Table 14), and thus has a chain length requirement. Addition of DpsE KR1 to Act min PKS, with or without Jad or Dps cyclases, failed to give any reduced products, suggesting that the decaketide Dps KR1 will not reduce an octaketide precursor, and thus has a minimum chain length requirement (Table 14). However, JadE KR1 was able to KR1(9) reduce both the minimal Tcm decaketides and the minimal Act octaketides. Thus decaketide KR1(9)s from different sources have different chain length requirements/specificity.

In the Jad gene cluster, there is a second putative cyclase JadI, that has sequence similarity with TcmI that converts tricyclic Tcm F2 34 to the tetracyclic Tcm F1 79 and thus both may be acting as CYC4. To determine whether JadI had any role in the non-linear shape of the angucyclines, Meurer constructed a cassette containing $tcmMjadAB$ (Jad minimal PKS) $tcmN$ (AROc) (that forms Tcm F2) and $jadI$ (CYC4?) (Table 14). The only product was the tricyclic Tcm F2 34, showing that Tcm F2

Nat. Prod. Rep., 1999, 16, 425–484 453

![Chemical Structures](attachment:chemical_structures.png)

**Scheme 10**

454 Nat. Prod. Rep., 1999, 16, 425–484

was not a substrate for JadI (CYC4), and that JadI did not influence the regiospecificity of the third ring cyclisation to that for the angucyclines. A cassette containing *jadABCDEI* (Jad minimal PKS, KR1, AROab and CYC4) produced SEK43 30 and UWM4 31, showing that JadI CYC4 did not act upon the singly aromatised intermediates SEK43/UWM4. Presumably other cyclases are responsible for CYC2 and the crucial CYC3(4/17) in the Jad KR1(9) reduced series, unlike the unreduced Tcm like compounds in which TcmN (AROc) seems also responsible for CYC2 and CYC3.³⁶

Gould *et al.* have investigated the biosynthesis of the 3-amino-4-hydroxybenzoic acid fragment of murayaanthraquinone 75 from *S. murayamaensis* and found it to be derived *via* the tricarboxylic acid cycle rather than the shikimic acid pathway as would be expected (Scheme 12).¹³³ Investigations with other metabolites show that the shikimate or aminoshikimate pathway was in use to some extent. However, Gould showed that labelled acetate, glutamate and succinate were specifically incorporated into a C-4 portion, and pyruvate was specifically incorporated into a C-3 portion, with some labelling occurring in the C-4 unit consistent with pyruvate carboxylase.¹³³ Gould suggests that other 3-amino-4-hydroxybenzoates might arise from this pathway, that have been assumed otherwise.

### 3.5.2.2 Tetrangomycins. Gould *et al.* have cloned and expressed the PD 116740 gene cluster from *Streptomyces* strain WP4669 and the tetrangulol cluster from *S. rimosus* (a different strain to the Otc producer) in *S. lividans* TK24.¹³⁴ Hybridisation experiments indicated that both *S. rimosus* and WP4669 each contain two clusters, possibly one for the known metabolite, and an unexpressed one possibly for a spore pigment. Expression of the *S. rimosus* cluster produced both tetrangulol 63 and tetrangomycin 61, as well as a new compound fridamycin E 60, resulting from oxidative cleavage of an angucyclinone. Expression of the WP4669 cluster produced tetrangulol 63, 19-hydrox-

---

**Scheme 12**

ytetrangulol 65, 8-*O*-methyltetrangulol 64, 19-hydroxy-8-*O*-methyltetrangulol 66, and PD 116740 68, suggesting the metabolic network shown in Scheme 10.

Decker and Haag have characterised six ORFs from *S. fradiae* Tu2717 that produces urdamycin A, including the KAS, CLF, ACP, putative KR1(9), CYC4 and a hydroxylase. The Urd KAS and CLF were able to complement Tcm KAS and CLF. Expression of the UrdE hydroxylase in *S. glaucescens*, that

---

**Scheme 11**

Nat. Prod. Rep., 1999, 16, 425–484 455

normally produces Tcm C, gave 6-hydroxy Tcm C as the main metabolite (40 mg L⁻¹) with some Tcm C (1 mg L⁻¹).<sup>135</sup>

### 3.6 9,14-Decaketides: tetracenomycins

The biosynthesis of tetracenomycin C 82 was extensively covered in the last review.<sup>1</sup> Hutchinson has reviewed the biosynthesis of tetracenomycin C (and daunorubicin).<sup>89</sup>

Rickards *et al.* showed that dioxygen labelled O-4, O-5 and O-12a of Tcm X [8-(O)-methyltetracenomycin C], but could not detect any label at C-4a.<sup>136</sup> Rohr *et al.* confirmed that all the other oxygen atoms in Tcm C were acetate derived, as expected, but could not detect any isotope shift in C-4a (Scheme 13).<sup>137a</sup> They then fermented *S. glaucescens* in 10% H₂¹⁸O (500 mL!) and obtained isotope shifts at 3-OMe, C-4a, 8-OMe, and 9-OMe (Scheme 13). Whilst three of these resonances are plausibly caused by solvent exchange during biosynthetic assembly, the C-4a upfield resonance, by far the largest, is not, and must biosynthetically originate from water. An epoxide intermediate between C-4a and C-12a is proposed, which is first opened by an enzyme thiol at C-4a, which is then hydrolysed by water.

The Tcm gene cluster from *S. glaucescens* has been sequenced and cloned, and various components expressed in *S. coelicolor* CH999 and in *S. glaucescens* in which *tcmGHIJKLMNO* genes are not expressed, to examine their roles.<sup>1</sup> A cell-free system containing the gene cluster can produce Tcm F2 34 from acetyl and malonyl CoA. Expression of just the minimal PKS (*tcmKLM*) assembles acetyl and nine malonyl CoAs to produce the decaketide SEK15 23, in which the CLF has incorrectly directed the first cyclisation to CYC1 (7/12) (Scheme 4, Table 3). This system is highly specific for acetyl CoA as starter, and that a FAS MT may be responsible for transferring malonate onto the ACP.<sup>64</sup> Addition of TcmJ has no obvious effect, but the addition of TcmN leads to formation of tricyclic Tcm F2 (Scheme 6), the first isolable product from the PKS. TcmN has been shown not to act upon C-9 reduced decaketides as Tcm min PKS/Act KR1 produce first ring non-aromatised RM20b 27 and RM20c 28 both in the presence or absence of TcmN (Scheme 5, Table 4).<sup>44</sup> Meurer *et al.* assembled an expression cassette containing *tcmJKLM* minimal PKS and both *tcmN* and *dpsE* (KR1) and obtained a mixture of Tcm F2 and RM20b/c, showing that TcmN and DpsE can compete for the same unreduced decaketide substrate (Table 14). Khosla *et al.*, and Hutchinson *et al.*, have produced two papers on the roles of Tcm PKS components.<sup>42,44</sup> The Tcm synthase has three critical decisions to make: choosing a primer, deciding how many malonate units to add, and where to fold/make the first ring. In a recent paper, Hutchinson *et al.* overexpressed the TcmK KAS and TcmL CLF and found by gel filtration that they purified as an αβ-heterodimer,<sup>137b</sup> as compared with the α₂β₂-heterotetramer found for the Act KAS/CLF.<sup>75</sup> Addition of the TcmM ACP, FabD malonyl transferase, malonyl CoA (to 150 μM) and

![Chemical Scheme](attachment:image.png)

**Scheme 13**

456 Nat. Prod. Rep., 1999, 16, 425–484
TcmN led to the formation of tricyclic Tcm F2. In an experiment in which MT was omitted from the assay, HPLC analysis found only a small peak (<5% previous peak) corresponding to Tcm F2, plus other peaks, but ‘no’detectable radioactivity (from [2-14] malonyl CoA) in any peak, suggesting the peaks were artifacts and that there was an absolute requirement for the MT. Acetyl CoA, the presumed starter unit, was not required to form Tcm F2 *in vitro*. Addition of acetyl CoA did not increase Tcm F2 levels. Addition of radiolabelled acetyl CoA with unlabelled malonyl CoA led to the formation of unlabelled Tcm F2. Presumably, the starter unit is obtained by malonyl-ACP decarboxylation and transfer of acetate to the KAS (decarboxylation possibly catalysed by conformational changes in the ACP caused by the approach and binding of the KAS). Addition of TcmJ increased the yield (*ca.* × 3), but it could not replace TcmN, reinforcing the concept of TcmJ being an important ‘architectural protein’ that enhances or glues together the other proteins in the most specific and efficient manner, without itself actually possessing any catalytic activity. The authors do show that the TcmM ACP can slowly self-malonate in the absence of any MT, but not at an adequate rate to account for the levels of Tcm F2 observed in the above *in vitro* assays, and the authors calculate that a maximum of 10% of malonate in Tcm F2 could be added by self-malonation. Addition of MT to ACP and malonyl CoA increased the rate of ACP malonation by about twenty-fold. These results should be compared and contrasted with the recent observations of Simpson *et al.*72b in which he shows that the Act ACP can efficiently self-malonate (Section 3.3.1.3).

### 3.6.1 Starter unit specificity.
The Tcm synthase has a high affinity for acetate as a starter unit, whilst daunorubicin synthase uses propionate. Replacement of the Tcm KAS and CLF genes with the corresponding daunorubicin genes (*DpsA* and *Dps*) [Table 14] still led to the exclusive use of acetate as starter, strongly suggesting that these minimal PKS proteins are not (in isolation) responsible for starter unit specificity.48 Genes reportedly thought to be responsible for this specificity are reported from *S. peucetius*99 and *Streptomyces* sp. strain C5,97 however, recent work implies that a interactive combination of Dps proteins are responsible for starter unit specificity, rather than individual proteins.100

### 3.6.2 Chain length.
Hutchinson *et al.* have combined Act and Tcm ACP, KAS and CLF genes to probe the role of CLF in determining chain length in *S. glaucescens* or *S. lividans*.42 *ActI-ORF1/ActI-ORF2/TcmM* (Act KAS, Act CLF, Tcm ACP) synthesises exclusively octaketides (Table 6). *TcmK/tcmL/tcmM* (Tcm KAS, CLF and ACP) synthesises exclusively decaketides (Table 3). McDaniel *et al.* have reported that hybrids of Tcm and Act KAS/AT and Tcm CLF are non-functional in *S. coelicolor* CH999 (Table 7), hybrids containing Tcm KAS and Act CLF produce only octaketides, suggesting that Tcm KAS has little influence on chain length.37 However, Hutchinson *et al.* found that in *S. lividans* or *S. glaucesens*, whilst a plasmid containing *actI-ORF1/tcmL/tcmM* (Act KAS, Tcm CLF, Tcm ACP) was dysfunctional, a plasmid containing *tcmK/ActI-ORF2/tcmM* (Tcm KAS, Act CLF, Tcm ACP) with *tcmJN* was able to produce both octa- (UWM 1 33) and deca-ketides (Tcm F2 34), strongly suggesting that CLF alone is not sufficient to control chain length, and that a combination of KAS and CLF controls it (Table 8, Scheme 6).42

### 3.6.3 First cyclisation.
Early work by the groups of Khosla and Hopwood1 indicated that the minimal PKS was crucial in determining the chain folding and resulting cyclisation of the first ring, as the products obtained from minimal Act PKS (without Act AROab, with or without KR) in *S. coelicolor* CH999 resulted from the same initial cyclisation as seen in actinorhodin. Cyclisation to the tetracenomycins requires folding at C-11 for cyclisation to the first ring CYC1(9/14). However, expression of *tcmKLM* produces mainly the CYC1(7/12) SEK15 23 with only a small quantity of CYC1(9/14) SEK15b 24, showing that the minimal Tcm PKS alone was not responsible for this initial folding/cyclisation in the desired manner. Addition of *tcmN*(AROc) or *tcmJN* gave the desired C-11 folded products, suggesting that the minimal PKS was responsible for chain length, and that AROc’s are responsible for subsequent cyclisations or are at least capable of completely redirecting KAS/CLF promoted cyclisations. The inclusion of TcmJ with TcmN appears to significantly increase yields, but not be responsible for structure. Perhaps TcmJ has a role in binding the other proteins together into a complex ensuring optimal interaction, increasing the overall yield, but not affecting the specificity of each protein.

To further probe the role of the TcmN as CYC1, Hutchinson *et al.* added TcmN to Act KAS/Act CLF/Tcm ACP (with TcmJ) in *S. lividans*. In the absence of TcmN, CYC1(7/12) gives SEK4 5, with lower levels of CYC(10/15) giving SEK4b 6 (Table 8, Scheme 6).42 Addition of TcmN results in CYC1(9/14) (as in Tcm production) to give UWM 1 33 but at much reduced levels unless TcmJ is present. The TcmN alone was therefore determining chain folding at C-11 (relative to thioester carbon) and regiospecificity of the first cyclisation CYC1(9/14) for both unreduced octaketide and decaketide chain lengths. It may be performing this role by promoting the desired cyclisation, or suppressing abnormal cyclisations. TcmN also has TcmD3 O-methyltransferase activity to convert TcmD3 into TcmB3. Hutchinson and Shen have purified TcmN from *S. lividans* and *E. coli*, and found that it eluted by gel filtration as a homodimer of *Mr* 103 kDa, suggesting that it may act *in vivo* as a homodimer.138 The protein was assayed using its TcmD3 O-methyltransferase activity. Addition of the purified TcmN protein to cell-free extracts of *S. lividans* with *tcmJKLM* inserted, resulted in exclusive production of Tcm F2 34 with all other aberrant cyclisations suppressed, with a ten-fold overall increase in polyketide production. Interestingly, the authors also observed that some Tcm F2 was very tightly bound to TcmJKLM, whilst aberrant products of TcmJKLM such as SEK15 23 were not bound, suggesting a binding pocket that might fold the linear decaketide and direct cyclisation. Addition of TcmN to TcmJKLM increased the amount of Tcm F2 that bound to the proteins by over ten-fold, highlighting the role this protein has in directing the reaction. Addition of TcmN to Tcm F2 already bound to TcmJKLM did not result in the release of any Tcm F2, confirming that the role of TcmN is not as a thioesterase, and it is not yet understood how the thioester bound product is released to be acted upon by the TcmI cyclase (CYC4?) that performs cyclisation and aromatisation of the fourth ring. Whilst it has been shown that TcmI can use free Tcm F2 as a substrate, perhaps it prefers TcmM (ACP) bound substrate, and joins in (displacing TcmN?) with all the other proteins? The above observations emphasise that, whilst not necessary for linear decaketide assembly, or low-level cyclisation to a variety of products including Tcm F2, TcmN, and the close intimate interaction between TcmJKLM and TcmN is vital to the efficient production and product channelling of the Tcm PKS complex.

Hutchinson and Meurer have shown that mutation of the TcmK KAS Cys (C173A or C173S) prevented PK formation. Mutation of the TcmK CLF active site Ser (S351A) still resulted in unaltered production of Tcm F2, but the H350L/S351A double mutant terminated PK production.60 The acyltransferase role of the CLF is uncertain, either the S351 is not an essential part of the AT activity, or there is no such activity, which might instead be performed by a component of the FAS system instead.

Khosla *et al.* have added the O-methyltransferase gene *tcmO* to the Act PKS gene cluster in *S. coelicolor* CH999 that

Nat. Prod. Rep., 1999, 16, 425–484   457
normally makes DMAC 10 and aloesaponarin II 11, and obtained aloesaponarin II and the new metabolite 3-O-methyl-DMAC, but no DMAC.¹³⁹ Thus this enzyme, which normally methylates Tcm B3, will methylate DMAC but not its decarboxylated analogue aloesaponarin II. 6-Methylsalicylic acid did not act as a substrate, indicating that both the carboxylate and the distal aromatic rings are required for recognition. A range of substrates were examined.

### 3.6.4 Post PKS hybrids. Rohr *et al.* discuss combining PKS with post-PKS genes from another antibiotic. They introduced the entire Tcm gene cluster into the urdamycin (Urd) producer *S. fradiae*, and obtained an array of novel Tcm, Urd and Elm metabolites including 6-hydroxy TcmC, 3,8-didemethyl TcmC and 8-β-D-olivosyl-8-demethyl TcmC.¹⁴⁰ Introduction of part of the elloramycin 81 (*elm*) cluster led to production of 8-demethyl TcmC as well as 3,8-didemethyl TcmC and 8-β-D-olivosyl-8-demethyl TcmC.

Elloramycin 81 has the same aglycone as Tcm but with methylation at O-13, and permethylated rhamnose attached to O-8. Hutchinson *et al.* have identified genes involved in the biosynthesis of elloramycin (Elm) from *S. olivaceus* Tu 2353.¹⁴¹ The *elm* cluster resembled the *tcm* cluster except for the locus of the *tcmP* gene, and most genes were able to complement each other. An Elm cosmid, 16F4, when introduced into *S. lividans* TK 24 produced 8-demethyl TcmC and Tcm B3, presumed intermediates in Elm biosynthesis. Rohr *et al.* have transferred the entire Tcm gene cluster from *S. glaucescens* and part of the the Elm gene cluster (as for cosmid 16F4) from *S. olivaceus* Tu 2353 into the urdamycin producer *S. fradiae* Tu 2717 and obtained three novel tetracenomycins: 6-hydroxytetracenomycin C (*tcm* cluster); 3,8-didemethyl-tetracenomycin C and 8-β-D-olivosyl-8-demethyltetracenomycin C from insertion of cosmid 16F4.¹⁴⁰

### 3.7 7,12-Decaketides: aureolic acids

The unusual acetate labelling pattern (Scheme 13) seen for aureolic acid antibiotics such as mithramycin 83 (also called aureolic acid, plicamycin, mithracin, and LA-7017) has raised suggestions that they may be assembled by more than a single polyketide chain.¹⁴²,¹⁴³ The PKS cluster for mithramycin has been cloned by Salas *et al.* from *S. argillaceus*.¹⁴⁴,¹⁴⁵ Insertional inactivation of the glycosyl transferase MtmD led to the accumulation of premithramycinone 77 which was assigned a structure that corresponded to a putative metabolite of Tcm F2. The mithramycin (Mtm) KAS and CLF genes from *S. argillaceus* and the Act ACP and Act KR1 have been inserted into CH999 by Khosla *et al.* and found to produce the DEC:Ac/KR1(9) CYC1(7/12) decaketides RM20b 27 and RM20c 28 (Scheme 5), confirming that the aureolic acids are assembled via a decaketide, followed by a rearrangement.¹⁴⁶ They suggested a folding pattern as found for the tetracenomycins, with mithramycin being obtained by the oxidative cleavage of Tcm F1.¹⁴⁶ Rohr *et al.* have inserted the mithramycin *mtmT*-ORFI min PKS genes (*mtmP*, *mtmK* and *mtmS*) and *mtmX* (unknown function), *mtmO-ORFI* (KR, not KR1), and part of *mtmOl* from *S. argillaceus* into *S. lividans* TK 21 and obtained SEK 15 23, the usual product of the unreduced decaketide minimal PKS (DEC:CYC1(7/12)), again confirming that aureolic acid antibiotics such as mithramycin are indeed decaketides.¹⁴⁷a However, when inserted into the Tcm producer *S. glaucescens* Tu49, which still contains the Tcm genes including Tcm minimal PKS and TcmN, a novel orange metabolite, Tcm M 80 was obtained in large quantities. Rohr and Salas suggested that the Mtm and Tcm biosynthetic pathways may branch at the closure of the fourth ring (Scheme 13). Ring closure using the cyclase TcmI would give Tcm F1, whilst in Mtm biosynthesis, an alternative aldol condensation catalysed by the cyclase

MtmX may occur to give an intermediate which could then be decarboxylated to TcmM, and in the presence of later Mtm genes, be further processed to Mtm itself. However, later work (*vide infra*) would suggest that in this system, Tcm F2 is exclusively assembled by TcmJKLMN, and that MtmX adventitiously converts it to Tcm M, which has no role in Mtm biosynthesis (Scheme 13).

In a paper which completely revises the biosynthesis of Mtm, Rohr, Salas *et al.* have recently reassigned the structure of premithramycinone and found a second product from disrupted mutants, demethylpremithramycinone 76.¹¹⁷ Assuming these compounds are biosynthetic intermediates to Mtm, this indicates a tetracycline like folding pattern rather than a tetracenomycin pathway to the aureolic acids, thus the aureolic acids are decaketide equivalents of tetracycline, without the KR1, DEC:Ac/CYC1(7/12)CYC2(5/14)CYC3(3/16)CYC4(1/18) and need a separate section of their own in this review (Scheme 13).

Rohr, Salas *et al.* have identified two genes involved in glycosylation to form Mtm, *mtmG-ORFI* and *mtmG-ORFII*.¹⁴⁷b The genes were independently inactivated, and both mutants produced the same four biosynthetic intermediates. Three intermediates were tetracyclic, containing one sugar moiety (premithramycin A1, A2 and A3), and one was tricyclic containing a trisaccharide chain (premithramycin A4), allowing the authors to propose a biosynthetic sequence, that involves several glycosyl transfers prior to the opening of ring four, contrary to that illustrated in Scheme 13. In consequence, the tertiary hydroxy group adjacent to ring cleavage site is derivatised as a trisaccharide during the cleavage.

In a recent paper, Rohr, Salas *et al.* identify *mtmO-ORFIV* as coding for the Baeyer-Villiger oxygenase responsible for cleavage of ring four.¹⁴⁷c Inactivation results in formation of premithramycin B that contains five deoxy-sugars as found in mithramycin 83, but retains the tetracyclic ring as found in premithramycinone 77 (Scheme 13). The *in vitro* addition of MtmO-ORFIV to premithramycin B converted it into a tricyclic compound. The authors suggest that MtmO-ORFIV may be useful in converting other tetracyclic compounds into clinically useful tricyclic antibiotics.

### 3.8 Other Type II aromatic polyketides

#### 3.8.1 5,7,10-Decaketides. Gould *et al.* have found that murayaquinone 85 (*S. murayamaensis*) is a rearranged decaketide with an unusual phenanthrene folding pattern (Scheme 14) followed by oxidative fission of the central ring and recyclisation, either by a pericyclic mechanism, or *via* a spiro compound somewhat reminiscent of that proposed for fredericamycin.¹⁴⁸a Further oxidation and decarboxylation gives the co-metabolite 84. Despite the reduced appearance of the starter end, butyrate, or its NAC thioester would not incorporate. Very unusually in actinomycete aromatic polyketides, the CYC2 has proceeded in reversed orientation to CYC1, *i.e.* DEC:Ac/KR1(7)CYC1(5/9)CYC2(4/13)KR3(15)CYC3(3/16).

#### 3.8.2 9,11,14-Dodecaketides: xanthones (lysolin), spore pigments and longer polyketides. The unknown grey spore pigment coded for by the Whi gene cluster, frequently discussed along with the benzoisochromanequinones, should be discussed in this section as it is now thought to be an undecaketide or dodecaketide. Khosla, Hopwood *et al.* have examined the role of WhiE-ORFVI, and found it to be around one half the size of Act ARO or TcmN, and to have properties similar to the aromatase N-terminal part of TcmN in that it can affect the CYC1 regiochemistry of non-reduced backbones, and promotes second ring aromatisation.⁴⁹ Thus WhiE can be classified as a AROc, without the other non-ARO/CYC activities such as O-MeT attached to the TcmN activity.

TW95b (8 mg L⁻¹) DODEC:Ac/CYC1(9/14)CYC2(7/16)CYC(O-1/5)CYC(18/23)OOX(10)

Thus, WhiE-ORFVI not only has a controlling influence on the chain length, but directs CYC1(9/14) regiochemistry. The addition of either whiE-ORFII or whiE-ORFVII to whiE min PKS gave only TW93a-d, so these must be later cyclases that only act on correctly CYC2 cyclised products, and only when they were all expressed. Addition of whiE-ORFVI to either construct resulted in TW95a and TW95b. When all three cyclases were added to the min PKS, TW95a and TW95b were absent, and several new, though as yet unidentified pigments were produced.

Pradimycin S 86 (Scheme 15) is assembled from twelve acyl units via a CYC1(9/14) but no KR1, as in Tcm. There may be a KR4(19) and KR5(22) during the assembly of 11-O-demethylpradinone II. Saitoh et al. have used frit-FAB LCMS on a time course study of production without any need of product isolation to identify the series of intermediates between 11-O-demethylpradinone II and pradimicin S. ${}^{149a}$

---

In a recent extensive paper, Moore et al. expressed PKS genes from the *S. coelicolor* (whiE), *S. halstedii* (sch) and *S. curacoi* (cur) spore pigment gene clusters in a strain of *S. coelicolor* (YU105) that has had *act* and *whiE* PKS clusters deleted. ${}^{148b}$ Expression of the minimal PKS's resulted in the formation of a mixture (ca. 12) of CYC1(7/12) novel polyketides including a heptaketide, undecaketide and two dodecaketides (2–10 mg L⁻¹ each):

- TW93a (4 mg L⁻¹) HEPT:Ac/CYC1(7/12)CYC(O-1/5)
- TW93b (3 mg L⁻¹) UNDEC:Ac/CYC1(7/12)CYC(O-1/5)CYC(O-13/17)CYC(16/21)
- TW93c (10 mg L⁻¹) DODEC:Ac/CYC1(7/12)CYC(O-1/5)CYC(14/23)CYC(17/22)OOX(18)
- TW93d (5 mg L⁻¹) DODEC:Ac/CYC1(7/12)CYC(O-1/5)CYC*(14/23)CYC(15/19)

Addition of the actIII KR1(9) gene to the minimal PKS's resulted in the formation of TW94a, identified as the known oracetophenone, and three novel reduced polyketides:

- TW94a (1.5 mg L⁻¹) PENT:Ac/CYC1(4/9)-CO2(1)
- TW94b (4.5 mg L⁻¹) UNDEC:Ac/CYC1(5/10)KR1(7)CYC(O-3/11)CYC(9/14)CYC(16/21)-CO2(1)
- TW94c (2.5 mg L⁻¹) UNDEC:Ac/CYC1(5/10)KR1(7)-CYC(O-3/11)CYC(9/14)CYC(16/12)CYC(O-17/14)-CO2(1)
- TW94d (5.5 mg L⁻¹) DODEC:Ac/CYC1(7/12)KR1(9)CYC(O-5/13)CYC(11/16)CYC(18/23)-CO2(1)

These TW94 tricyclic metabolites resemble polyketides from other min PKS + actIII systems such as RM1821, RM18b22 (Fren min + ActIII) (Scheme 3) and RM2029 (Tcm min + ActIII) (Scheme 5).

The role of the cyclase like whiE-ORFII, whiE-ORFVI and whiE-ORFVII genes were then analysed. The addition of whiE-ORFVI to whiE min PKS gave two dodecaketides, the first in very high yield:

- TW95a (50 mg L⁻¹) DODEC:Ac/CYC1(9/14)CYC2(7/16)CYC(O-1/5)CYC(18/23)

---

Lysolipin was discussed in the previous review, ${}^1$ and would be expected to be assembled via twelve acyl units with CYC1(9/14), but interestingly, would be expected to have a KR1(11), as well as a KR3(17) and a KR4(19).

Kakinuma et al. have determined the structure of the decaketide FD-594 87 produced by *S.* sp. TA-0256, and shown that it is assembled by fourteen acetate units, with loss of one carbon atom. ${}^{149b}$ If the molecule is a decaketide (*i.e.* not using a longer starter unit) then the biosynthesis would involve anthraquinone formation at C-14 and C-21, oxidation to a carboxy group and phenol at C-15, rotation of the bottom aromatic ring, followed by attack of the C-19 phenolic group to form the pyran ring.

---

### 4 Gram positive bacteria: Actinomycetes, Type I assembly

Small monocyclic aromatic tetraketides and pentaketides from bacteria may be assembled by a Type I process. Bechthold et al. have investigated the synthesis of avilamycin A 88, a member of the tetraketide orthosomycin antibiotics produced by *S. viridochromogenes* Tu57. ${}^{150}$ They failed to obtain clones using Type II hybridisation with Gra PKS cluster, suggesting that a

Nat. Prod. Rep., 1999, 16, 425–484 459

Recently a series of toxic water soluble okadaic acid esters have been isolated including dinophysistoxin-4 (DTX-4) 92 which contains three sulfates on the esterification fragment (Fig. 5).<sup>151</sup> In a series of papers, Wright *et al.* have investigated the biosynthesis of this fragment.<sup>152–154</sup> The two carbon backbones were both assembled from acetate, with terminal decarboxylation, a glycolate derived starter unit and an acetate methyl one carbon branch as found in the parent polyether structure. The parent polyether and ester fragment were both examined by feeding [2-<sup>13</sup>C, <sup>2</sup>H<sub>3</sub>]acetate. Only two deuteriums were located at the pendent methyls or methylenes, consistent with the proposed assembly involving attack by the acetate CH<sub>2</sub> anion or malonate at a chain carbonyl, followed by decarboxylation to give the pendent methyl.<sup>155</sup> The high level of deuterium observed (42%, slightly higher than backbone retention) suggests direct incorporation from the acetate pool. Approximately equal levels of CHD<sub>2</sub> and CH<sub>2</sub>D were observed. In the parent polyether, two deuteriums were retained at C-3 and C-18, showing that no dehydration had occurred at the preceding positions during the biosynthetic process. This was supported by retention of <sup>18</sup>O from labelled acetate at C-4, however, none was observed at C-19 presumably due to adventitious exchange. No significant deuterium label was observed at C-12, 24 or 26. Interestingly, the two carbons, one on either side of the internal ester functionality on the side chain of DTX-4 92 were found to be derived from the same acetate unit, having a J<sub>COC</sub> coupling of 2.6 Hz in the {<sup>1</sup>H} <sup>13</sup>C NMR spectrum, and <sup>18</sup>O from labelled acetate was retained in the carbonyl carbon, not the alcohol oxygen, showing a Baeyer–Villiger oxidation had occurred. Thus only two polyketide chains are required to assemble DTX-4, both starting with glycolate. The Baeyer–Villiger reaction could be mediated by a flavin monooxygenase.

Norte *et al.* have suggested that the carbon backbone of the parent polyether is biosynthesised in five fragments A–E, and that whilst fragments A, C, and E are assembled through a classical polyketide assembly (with attached pendent methyls derived from acetate methyl carbons), Norte suggests that fragment B is derived from succinate via the citric acid cycle, and fragment D could be from glutamate or valine.<sup>156</sup> However, Wright *et al.* observed fairly uniform <sup>13</sup>C labelling throughout the polyether carbon backbone, including the ‘isolated’ acetate chain methyl carbons.<sup>152–154</sup> If the sources of these carbons were differing citric acid cycle or amino acid derived fragments as suggested by Norte, then different levels of incorporation might be expected. Wright then speculates that the polyether chain isolated acetate methyls could actually arise from chain shortening of a single regular linear polyketide chain, with extrusion of several carboxyl carbons as illustrated in Fig. 6, rather than as previously proposed, a mixture of diacid and acetate building blocks. Wright suggests flavin monooxygenase mediated oxidation next to a carbonyl to give an α-diketone, followed by a Favorski type rearrangement, peroxide attack and opening of the cyclopropane ring with decarboxylation (Scheme 16). Similar mechanisms have been suggested in the biosynthesis of simaomicin<sup>157</sup> and lysolipin.<sup>158</sup> Presumably, similar processes could also explain the labelling patterns observed in larger ring polyethers such as brevetoxin. It is fun to speculate on a possible biosynthetic sequence based upon this ‘all acetate chain’ suggestion by Wright. A linear polyketide with glycolate starter is first subjected to five flavin mediated α-keto oxidations to give 93 (Fig. 6); this agrees with the lack of deuterium incorporation at C-12, 24 and 26 in DTX-4 92. Three carbonyl derived carbons, highlighted in boxes, can be extruded as carbon dioxide. The twice repeated formation of a carbanion at C-25 agrees with the lack of deuterium retention at this carbon, however these oxidations do result in a highly oxidised system. Cytochrome P-450 mediated epoxidation at C-16, 22 and 30, along with conversion of the carbonyl at C-25 to an epoxide, would give 94 (Fig. 7). A series of acetate anion or malonate anion attacks at carbonyl functions and epoxides

Type II process was not present. However, using a sugar dehydratase gene as a probe, they obtained a sequence containing four ORFs including aviM whose closest homology (37% identity) was with 6MSAS from *P. patulum*, with motifs for ACP, KAS, and AT. Insertion into *S. coelicolor* CH999 or *S. lividans* TK24 efficiently produced orsellinic acid. It is interesting to speculate whether these systems were obtained from the fungi via gene transfer, or evolved in the bacteria prior to eukaryotic evolution.

### 5 Protoctista

Members of the dinomastigota phylum (single celled marine plankton, also called dinoflagellates) are undoubtedly eukaryotic, yet the organisation of their nuclei and chromatin resembles the prokaryotes in many respects. The dinoflagellates *Prorocentrum lima, Dinophysis fortii* and *Dinophysis acuminata* (dinomastigota) produce a group of lipid soluble polyether phosphatase inhibitors, the dinophysistoxins and okadaic acid derivatives, which have been responsible for the red tide phenomenon known as diarrhetic shellfish poisoning. These toxins have an extremely unusual biosynthesis, as outlined in the previous review,<sup>1</sup> in which a glycolate starter unit is added to by acetate units, some units of which are cleaved to leave only the methyl derived carbon (reminiscent of larger ring polyethers found in other dinomastigota such as brevetoxin) to form dinophysistoxin 1 (DTX-1, (35R)-methylokadaic acid 89). Also, pendent methyl groups are acetate methyl derived. Aspects of this biosynthesis are clearly related to some polyketide biosynthetic pathways in prokaryotes. For example, the unusual starter unit glycine in the biosynthesis of myxovirescin A<sub>1</sub> by *Myxococcus virescens*, glycerol starter unit in aplasmomycin biosynthesised by *Streptomyces griseus* and the acetate methyl derived one carbon branches found in virginiamycin biosynthesised by *Streptomyces virginiae* and in oncorhyncolide biosynthesised by a marine bacterial isolate.<sup>1</sup>

460 Nat. Prod. Rep., 1999, 16, 425–484

would result in the observed labelling pattern (Fig. 5). However, the methylation at C-25 is not easily rationalised without the subsequent reduction, dehydration and epoxidation of the C-26 carbonyl group, allowing nucleophilic attack at C-25. Adjustment of certain oxidation levels would then give DTX-4 92. For example, hydride initiation at C-4 of the first ring closure, hydride reduction of the hemiacetal product at C-11, and the ketone at C-24 would be required in this mechanism. This overall mechanism agrees with the retention of ^{18}O at C-4 and 27, and the retention of two deuteriums at C-3 and C-18. Another interesting question is the linkage of these enzymes with the DTX-4 polyketide synthase. Due to the extremely high reactivity of the poly α-keto chain, it would be very tempting to associate the flavin oxidase, carbonyl extrusions and methylations with the growing polyketide chain, even as a part of the

multienzyme complex! A completely different alternative would involve the formation of a series of α,β-epoxyketones. Attack on the carbonyl by a hydroperoxide or certain enzyme nucleophiles could then promote a series of Tiffeneau-Demyanov type rearrangements to shorten the polyketide chain, followed by loss of the original carbonyl carbon by decarboxylation (Scheme 17). Whilst this involves chain shortening after P-450 oxidation, at least this route would not involve multiple flavanoid oxidations to form highly oxygenated polycarbonyl systems. Whatever the answer, linear acetate chain or diacid building blocks, this is an intriguing problem Nature has posed.

The amphidinolides are another set of metabolites produced by the dinoflagellates that pose intriguing biosynthetic questions still to be satisfactorily resolved. Produced by Amphidin-

![Chemical Structures](https://i.imgur.com/chemical_structures.png)

Fig. 5 The origin of the carbon, oxygen and hydrogen atoms in okadaic acids and dinophysistoxins and the five fragments of the polyether portion as proposed by Norte.

![Figure 6](https://i.imgur.com/figure_6.png)

concentrations suggests that electrostatic effects must play an important role in dimerisation (Fig. 9).<sup>164</sup> This is in complete contrast to animal Type I FAS where dissociation occurs at low salt concentration as dimerisation is due to hydrophobic interactions. Kurosaki found that the monomeric enzyme was able to bind both substrates, acetyl CoA and malonyl CoA, and convert them into the derailment product TAL, despite the presence of NADPH. This suggested that the monomer was capable of the condensation reaction, but not the δ-ketoreductase reaction.<sup>163</sup> In an animal FAS active site Stoops *et al.* showed that the monomer lacks the ability to perform condensation reactions, but can perform the ketoreduction, indicating that KR and ACP are from one polypeptide, and KS from the other.<sup>165</sup> In contrast, these observations by Kurosaki imply that in a 6-hydroxymellein synthase active site both KAS and ACP are on one polypeptide, and the ΔKR is on the other. The synthase lost the ability to bind acetyl CoA or malonyl CoA when the serine-OH blocking agent, 4-(2-aminoethyl)benzenesulfonyl fluoride was added, suggesting the presence of a transacylase domain, the first reported in a higher plant.<sup>166,167</sup> Selective thiol blocking suggested a common serine hydroxy for initial reversible binding to either acetyl CoA or malonyl CoA followed by selective transfer of bound malonyl to ACP-SH or bound acetyl to Cys-SH (Fig. 10). In contrast, in plant chalcone and stilbene synthases acyl CoAs are thought to directly transfer to Cys-SH without the intermediacy of a serine hydroxy group.<sup>168</sup>

### 6.2 Chalcones

Anthocyanins are widespread in plants, being responsible for petal colours, attractiveness of seeds and fruits to ‘dispersors’, acting as feeding deterrents and as protection from UV damage. The genetics and biochemistry of anthocyanin biosynthesis have been reviewed.<sup>169</sup> The first step is the extension of *p*-coumaryl CoA, (2E)-3-(4-hydroxyphenyl)prop-2-enoyl CoA 101 with three units of malonyl CoA by chalcone synthase (CHS) to give 4,2′,4′,6′-tetrahydroxychalcone 102 (Scheme 20). This is converted by chalcone isomerase into naringenin 103. Flavanone 3-hydroxylase then converts naringenin into dihydrokaempferol 104, which is the precursor of all the anthocyanins. Britsch *et al.* have characterised flavanone 3-hydroxylases from the carnation (*Dianthus caryophyllus*), china aster (*Callistephus chinensis*) and stock (*Matthiola incana*), and found strict conservation between fourteen residues including three His and one Asp supporting the involvement of iron II binding. The sequence suggested that these enzymes were 2-oxoglutarate dependent dioxygenases.<sup>170</sup>

Learn Jr. *et al.* have examined the evolution of thirteen chalcone synthases (CHS) in seven *Ipomoea* sp. (*Convolvulaceae*), and found them to be rapidly evolving. Many *Ipomoea* sp. contain more than one CHS due to gene duplication followed by divergence.<sup>171</sup> Helariutta *et al.* have examined a CHS like gene in *Gerbera hybrid*a (*Asteraceae*)<sup>172a</sup> and found that it had altered substrate specificity and a truncated catalytic profile, supporting the proposal of gene duplication followed by

**Scheme 16**

ium sp., which seems to have a symbiotic relationship with *Amphiscolops* sp. (marine flatworms), they possess an odd number of carbons in a macrolide ring, and sometimes possess vicinal pendent methyl groups, which again cannot be accounted for by the standard rules of polyketide biosynthesis. Kobayashi *et al.* has fed <sup>13</sup>C-labelled acetate and methionine to *Amphidinium* sp. and proposed that amphidinolide was assembled from large blocks as in Fig. 8. However, the similarities of the labelling pattern to that seen in DTX-4 would suggest that the alternative schemes involving chain shortening of a single polyketide chain (*vide supra*) might also apply to amphidinolide.<sup>159</sup>

Shimizu *et al.* have isolated linear polyketides 96 and 97 from an *Amphidinium* sp. that possesses unusual 1,4-diketone fragments and pendent methyls.<sup>160</sup> Shimizu suggest condensations involving dicarboxylic acids, or collapse of a polyepoxygenated polyene chain as shown in Scheme 18.

## 6 Plantae

### 6.1 6-Hydroxymellein

Carrot (*Daucus carota*: Umbelliferae) cells produce 6-methoxymellein in response to fungal infection or certain chemical elicitors using two inducible enzymes, 6-hydroxymellein synthase (a single multifunctional polypeptide chain with *M*<sub>r</sub> of 128 kDa) and 6-hydroxymellein *O*-methyltransferase, SAM dependent, recently partially purified and characterised by Kurosaki with *M*<sub>r</sub> 76 kDa (Scheme 19).<sup>161</sup> Kurosaki has shown that in the absence of NADPH, the synthase forms the derailment product TAL 98, suggesting that the triketide is reduced by a 5-oxoreductase (ΔKR).<sup>162</sup> Kinetic isotope studies have demonstrated that this reduction is the rate determining step in the synthase assembly of 6-hydroxymellein 99. Kurosaki observed that at high salt concentrations, gel filtration yielded monomeric synthase (*M*<sub>r</sub> 136 kDa) but with loss of activity.<sup>163</sup> Activity was restored by desalting, but the synthase now showed an apparent *M*<sub>r</sub> of 285 kDa, suggesting the active form was a homodimer. The dissociation to monomers at high salt

**Fig. 7**

![Scheme 17](image)

Amphidinolide  
95

![Scheme 18](image)

Acetyl CoA  
+  
Malonyl CoA  

Triacetic acid lactone (TAL)  
98  

No NADPH  

6-Hydroxymellein synthase  

6-Hydroxymellein  
99  

6-Methoxymellein  
100  

Scheme 19  

Fig. 8 Two possible pathways to assemble amphidinolide, the first from diacids as suggested by Kobayashi, and the second based upon a single acetate chain analogous to that outlined in Fig. 6.

divergence, in this case, a possible evolutionary reversal to a more primitive form closer to the fatty acid synthases that CHS are thought to have evolved from.

The chalcones from *Pinus strobus* (Eastern white pine) can be C-methylated in the dihydroxybenzene ring; the methyl

Fig. 9 Active dimeric form of 6-hydroxymellein synthase held together by electrostatic interactions.

group would be predicted to originate from chain extension with methylmalonyl CoA in the second condensation of the biosynthetic pathway. In a recent paper, Schröder *et al.* have described the overexpression in *E. coli* of two chalcone synthases (PStrCHS1 and PStrCHS2, 87% identical sequence)

Fig. 10 The use of a common serine for binding either acetyl or malonyl CoA on 6-hydroxymellein synthase.

![Diagram](https://i.imgur.com/yourimageurl.png)

Scheme 20

![Chemical Scheme](https://i.imgur.com/yourimageurl.png)

### 6.3 Stilbenes

Whilst chalcone synthases are ubiquitous in plants, stilbene synthases (STS) occur in a wide range but very limited number of quite unrelated plants, though stilbenes are widespread in gymnosperms. STS has been cloned from peanut (*Arachis hypogaea*), grape (*Vitis vinifera*) and Scots pine (*Pinus sylvestris*). The Eastern white pine contains two classes of stilbenes, pinosylvins 107 (biosynthetically derived from cinnamoyl CoA 105) and dihydropinosylvins 108 (biosynthetically derived from dihydrocinnamoyl CoA 106) (Scheme 21).

![Scheme 21](https://i.imgur.com/yourimageurl.png)

Raiber *et al.* have overexpressed in *E. coli* and characterised two closely related STSs (STS1 and STS2) from Eastern white pine (*Pinus strobus*), with only five amino acid residue differences, and it was expected that the two enzymes would differ in substrate preferences.¹⁷³ᵃ Whilst STS2 had a much higher activity (and a very different pH profile) than STS1, both enzymes preferred cinnamoyl CoA to dihydrocinnamoyl CoA as a substrate, thus both should be classed as pinosylvins synthases. Thus there may remain an additional STS that prefers dihydrocinnamoyl CoA as substrate, or the STS activity may be controlled by substrate availability. Raiber then constructed the STS2R313H mutant (STS2*) and found that its properties now closely resembled STS1. Apart from STS1, all other STSs contain Arg or Pro at position 313, and all CHSs contain Pro at the corresponding position. Raiber suggests that for maximal activity, lowering the pH of STS1 (and STS2*) will protonate the H₁₅-313, giving it a similar function to Arg at neutral pH, explaining the differing pH profile.

Stilbene synthases, in common with chalcone synthases, lack a phosphopantetheine arm, and a second cysteine residue is thought to functionally replace its thiol. Preisig-Muller *et al.* have examined several stilbene synthases, and the closely related bibenzyl synthases¹ by protein modification (e.g., N-ethylmaleimide) or site-directed mutagenesis and located the active cysteines and starter unit binding domains.¹⁷³ᵇ The studies showed that the covalent attachment of malonyl CoA takes place at a cysteine residue near the N-terminus of the 90 kDa homodimeric protein, and the exchange of radiolabel from radioactive hydrogen carbonate into malonyl CoA was independent of modifications on the starter unit binding area in the C-terminal half of the protein. Methyl malonyl CoA (but not succinyl CoA) functioned as a competitive inhibitor of the malonyl CoA:CO₂ exchange reaction, but was only 10% as effective.

There is a range of plant polyketide metabolites that are biosynthesised from hydroxycinnamoyl CoA as a starter unit followed by the addition of one, two or three malonate extender units. The very primitive plants belonging to *Psilotum* sp. (*Psilotum: psilophyta*. e.g., 'whisk fern') produce glycosylated arylpyrones such as the psilotons 109, which are assembled with only one malonate extender unit (Scheme 22). *Equisetum*

464 Nat. Prod. Rep., 1999, 16, 425–484

![Scheme 22](attachment:Scheme_22.png)

arvense (Equisetaceae: sphenophyta, 'horse tails') produce only styrylpyrones 110 (two malonate extender units) in their haploid gametophytes, with no evidence of other polyketides. However, the evolutionarily more advanced diploid sporophyte accumulates flavanoids (three malonate extender units via chalcones 111 or stilbenes 112). Some primitive angiosperm families, as well as many fungi, also produce styrylpyrones. Beckert *et al.* speculate that these enzymes in the order psilotin synthase, styrylpyrone synthase, chalcone synthase/stilbene synthase may represent an evolutionary trend in polyketide biosynthesis. They have now isolated styrylpyrone synthase ($M_r$ 56–77 kDa) from gametophytes of *E. arvense*, as a model system for examining this trend.

### 6.4 Other metabolites

Horper and Marner have examined the biosynthesis of primin 119, a quinone stored in the trichomes of *Primula obconica*, which is responsible for primrose dermatitis, and found that radioactive acetate, injected into the stems of cut leaves, was incorporated as expected. However, attempts to incorporate radiolabelled hexanoic acid failed. In order to demonstrate the biosynthetic sequence, a series of nor and homo analogues of 114–118 (Scheme 23) were synthesised, fed and all shown to be incorporated by MS, though the pathway via 116 was preferred. An alternative pathway via 115 was not ruled out.

Legumes in response to attack by pathogens can produce the isoflavonoid pterocarpan phytoalexins such as medicarpin 122. As the rate of biosynthesis correlates with the resistance of host plants, elucidation of the pathway has importance, as well as determining the factors affecting the product stereochemistry.

![Scheme 23](attachment:Scheme_23.png)

Most steps are now well established, but little has been known about the final step, the conversion of vestitone 120 to medicarpin 122 (Scheme 24). An elicited chick pea cell suspension culture converted vestitone to medicarpin. The activity ('pterocarpan synthase') has been partially purified and found to be NADPH dependent. However, all attempts to purify it from this, and other sources failed. Alfalfa accumulates (−)-(6aR)-11aR)-medicarpin. Paiva *et al.* have now located and purified two activities in *Medicago sativa* (alfalfa), both having similar $M_r$, the first being (3R)-vestitone reductase, and the second being 7,2′-dihydroxy-4′-methoxyisoflavanol dehydratase to give 122. Guo and Paiva then cloned and expressed catalytically active vestitone reductase (38 kDa) in *E. coli*. The protein contained an ADP βαβ-fold consensus sequence. Interestingly, during purification it was found that at low salt concentrations, vestitone reductase exhibited specific protein-protein interactions with the next and final enzyme in the pathway, 7,2′-dihydroxy-4′-methoxyisoflavanol dehydratase.

The 9-phenyl-(1H)-phenalen-1-ones are plant pigments produced mainly by the Haemodoraceae family. It has been postulated that these compounds are assembled from two ArC₃ units via a diarylheptanoid intermediate 123 (Scheme 25). Using ¹³C-labelled precursors, Hölscher and Schneider have obtained high levels of incorporation of two molecules of p-cinnamic acid, and one molecule of acetic acid into anigrorufone 124 from cultured roots of *Anigozanthos preissii*. Hölscher and Schneider then efficiently specifically incorporated [2-¹³C]-1-phenyl-7-(3,4-dihydroxyphenyl)hepta-1,3-dien-5-one 123, at far greater levels than labelled p-cinnamic acid. A subsequent (enzyme mediated?) Diels–Alder cyclisation and reduction/dehydration gives 124.

(3R)-Vestitone
120

p-Coumaroyl CoA
+ 3 x malonyl CoA

SAM

(3R)-Vestitone reductase
NADPH

'Pterocarpan synthase'

HO
OMe
7,2'-Dihydroxy-4'-methoxyisoflavanol dehydrata:
121

(-)-Medicarpin
122

Scheme 24

CH₃COOH
COOH
123
D-A

Anigorufone
124

Scheme 25

### 7 Fungi

#### 7.1 6-Methylsalicylic acid

The gene for 6MSAS from *Penicillium patulum* was sequenced by Schweizer *et al.* in 1990 and found to encode four 190,731 Da subunits. The gene for orsellinic acid synthase from *P. cyclopium* was reported by Jordan (now Shoolingin-Jordan) and Spencer in 1993 as a 130 kDa protein. The 6MSAS gene consists of two exons (29 and 1,745 bp) split by an intron (69 bp). In order to express 6MSAS in prokaryotic *Streptomyces coelicolor* CH999, Khosla, Hopwood, Schweizer and Bedford used a clone containing the large exon, and attached a synthetic analogue of the first exon that contained synonymous triplet codons with high G + C usage as found in the Streptomyces. The transformant produced 20 mg of 6MSA from 12 agar plates (ca. 300 mL medium), levels comparable to the expression of streptomycete genes in CH999. This implies that the protein had been efficiently phosphopantetheinylated, had folded in the usual manner, and had been post-translationally modified. Shoolingin-Jordan *et al.* have shown that 1,3-dibromopropane crosslinked the KAS thiol (Cys-204) and ACP phosphopantetheine thiol (on Ser-1733), this being inhibited by acetyl CoA or malonyl CoA. Crosslinking with 5,5'-dithiobis(2-nitrobenzoic acid) suggests the presence of two pairs of functional dimers in the 6MSAS tetramer, and the proximity of the KAS/ACP thiols from separate subunits, within each dimer. The authors relate this proximity of the KAS/ACP thiols from separate subunits with that found in vertebrate FAS. Shoolingin-Jordan and Child show the inactivation of 6MSAS by cerulenin, which they show covalently bonds to the KAS Cys-204.

#### 7.2 Aflatoxins

##### 7.2.1 Introduction

The aflatoxins are amongst the most toxic and carcinogenic metabolites produced by fungi, and whose complex biosynthesis has been most extensively studied since their isolation shortly after an outbreak of Turkey X disease in 1960 which led to the death of over 100,000 turkeys in the UK. Cytochrome P-450 epoxidation of aflatoxin B₁'s dihydrofuran double bond results in an ideal DNA interchelator with an epoxide perfectly placed to react with N-7 of guanine. In particular, a mutational hot spot has been located resulting in a mutation in the p53 protein that regulates the cell cycle, resulting in human cancer. A wide range of *Aspergillus* spp., most notably *A. nidulans*, produce sterigmatocystin (Stc), an intermediate in aflatoxin biosynthesis, whilst only *A. parasiticus*, *A. flavus* and *A. nomius* produce aflatoxins (Afl). The complexity of the structure has led to many conflicting biosynthetic proposals, and much research effort over the last few decades.

Minto and Townsend have recently published an excellent review on the enzymology and molecular biology of aflatoxin (Afl) biosynthesis. Woloshuk and Prieto have reviewed the genetic organisation, and Trail, Mahanti and Linz have reviewed the molecular biology. There is a very confusing array of different names for genes, the names of which being frequently altered. There is an urgent need for an 'aflatoxin pathway nomenclature committee'! The entire nucleotide sequence of the sterigmatocystin gene cluster in *A. nidulans* has been determined, and 25 genes located. The gene clusters in *A. parasiticus* and *A. flavus* have been compared. The clusters, and the sterigmatocystin producing cluster from *A. nidulans* are shown in Fig. 11. Clustering is less common in eukaryotic systems, and may be indicative of horizontal gene transfer from a prokaryote a very long time ago.

An overview of the biosynthesis as currently understood is given in Scheme 26. The investigation into aflatoxin biosynthesis parallels the progress in biosynthesis as a whole. In the late sixties, Büchi *et al.* established through meticulous chemical degradation studies that the location of radiolabels from acetate and methionine indicated the origin of twelve atoms in Afl B1 from a highly rearranged polyketide chain, and others from methionine. The arrangement of acetate units was in dispute until 1977 when Steyn *et al.* incorporated [1,2-¹³C₂]acetate. Meanwhile, during the seventies a whole series of mutants had been generated that accumulated norsolorinic acid, averatin, averufin, versicolorin A, and the observation that the addition of an insecticide, dichlorovos (dimethyl-2,2-dichlorovinyl phosphate) to wild type led to the accumulation of versiconal acetate (references contained in reference 195). The extreme toxicity of the aflatoxins had hindered research, but the availability of these mutants now allowed incorporation studies with these 'more

Sterigmatocystin producing gene cluster from A. nidulans

Aflatoxin producing cluster in A. parasiticus and A. flavus

Fig. 11

easily handled' metabolites. In 1980, in the first biosynthetic application of $^{18}$O isotope shifts, Vederas and Nakashima showed that all the oxygen atoms in averufin 129 were acetate derived, except for C-10, which was derived from dioxygen. ${ }^{196}$ The pattern of deuterium incorporation into sterigmatocystin 142 was examined by Sankawa et al. ${ }^{197}$ and into Afl B1 by Simpson et al. ${ }^{198}$ which allowed some delineation of the intricate rearrangement pathways. Simpson observed an acetate derived hydrogen at C-5 of Afl B1 (a carboxy carbon) suggesting an NIH shift during the conversion of Stc 142 into Afl B1 144. The metabolite averufanin 128 had earlier been isolated and radioactive averufanin was incorporated into averufin 129, but not averantin 126, suggesting that it was after averantin on the pathway or a shunt metabolite. ${ }^{199}$

In 1983 Townsend et al. showed the intact specific incorporation of [1-$^{13}$C]hexanoic acid at 3–4% levels into C-1' of averufin 129, which was much higher than the random incorporation (0.5%) of $^{13}$C into carboxy derived carbons of averufin due to fatty acid degradation of labelled hexanoate into acetate. ${ }^{195}$ Under parallel feeding conditions, [1-$^{13}$C]butyrate, [1-$^{13}$C]-5-oxohexanoate, and [1-$^{13}$C]-3-oxooctanoate gave no evidence of specific incorporation. ${ }^{200}$ Incorporation of 5-oxohexanoate would have allowed direct formation of the acetal in averufin 129, avoiding the need for several post-PKS oxidation steps at that carbon. 3-Oxoctanoate would be the first PKS product from malonate extension of hexanoate. The use of extended acyl units has only been rarely observed in fungal polyketides, but this successful specific incorporation of hexanoic acid led support to the proposal that hexanoate was indeed the starter unit in the biosynthesis of the aflatoxins. Chandler and Simpson tried to distinguish between this 'hexanoate starter plus octaketide' route, and earlier 'decaketide' routes that may be allowing hexanoate 'on board' as a triketide, by feeding diethyl [2-$^{13}$C]malonate. They observed essentially equal labelling at all nine 'malonate' sites, with lower levels at C-6', what would be the starter unit for either a FAS plus octaketide PKS or decaketide PKS. The equal intensity observed at all 'malonate' positions suggested that the malonates were inserted all at the same time period, not lending support to the hexanoate plus octaketide PKS proposals, unless the FAS and PKS activity/expression levels were linked. ${ }^{201}$ The stereochemistry of the enoyl reductase in the hexanoate portion of averufin was found to be the same as for fungal FAS systems, and opposite to known fungal PKS systems, reinforcing suggestions that the hexanoate portion was assembled by a FAS like synthetase, rather than PKS like. ${ }^{202}$

Brobst and Townsend have recently reexamined the above feeding experiments by Townsend using labelled NAC thioesters. ${ }^{203}$ As NAC thioesters, hexanoate now gave a remarkable 22% specific incorporation, and [1-$^{13}$C]-3-oxooctanoate was now specifically incorporated at ca. 5% enhancement, strongly supporting the malonate extension of hexanoate by an octaketide PKS. The NAC thioester of [2,3-$^{13}$C $_{2}$ ]butyrate gave no detectable C–C coupling in the side chain, indicating incorporation levels of less than 0.1%. The hexanoate could either be assembled by a FAS, or obtained by fatty acid degradation. The source of hexanoate was probed by feeding the NAC thioesters of [1-$^{13}$C,$^{18}$O $_{2}$ ]acetate and [1-$^{13}$C,$^{18}$O $_{2}$ ]hexanoate. In both examples, the oxygen attached to C-1' was heavily labelled with respect to the corresponding $^{13}$C standard resonance. The level of $^{13}$C incorporation from acetate into the hexanoate derived fragment was also high, and if the source of hexanoate had been intracellular hexanoyl CoA from fatty acid degradation, then lower levels of extracellular acetate would be expected to be incorporated due to the ready availability of hexanoyl CoA. The authors proposed that the hexanoyl unit was prepared by a specialised FAS in an as yet unknown thioester form, that is released for uptake by the PKS, or by a FAS attached to the PKS that will readily accept preformed hexanoate.

In 1996, Simpson et al. probed the substrate specificity of the 'norsolorinic acid synthase'. ${ }^{204}$ They fed the NAC thioesters of butyrate, pentanoate, hexanoate, heptanoate and octanoate, all labelled with deuterium at C-2, to A. parasiticus, and only obtained specific incorporation from pentanoate and hexanoate, at a similar level of incorporation to each other, as indicated by $^{2}$H NMR spectroscopy and MS of norsolorinic acid, tetramethyl ether. The authors then found that 6-fluorohexanoate, NAC thioester, was efficiently incorporated, as shown by $^{19}$F NMR spectroscopy and MS, but that 2-fluorohexanoate was not, possibly due to the formation of toxic 2-fluoroacetate, or that the substrate was a specific inhibitor of the norsolorinic acid synthase. McGuire and Townsend have shown that the ester oxygen of versiconal acetate 131 is derived from dioxygen, and that the oxygen label is retained in the furan ring of versicolorin. ${ }^{205}$

### 7.2.2 Regulation

Woloshuk et al. have characterised *aflR* (previously labelled as *afl2* in A. flavus and *apa2* in A. parasiticus) as coding for a 47 kDa DNA binding protein containing a GAL4 binuclear zinc binding CX $_{2}$CX $_{6}$CX $_{6}$CX $_{2}$CX $_{6}$C motif, which is thought to regulate expression of many parts of the cluster, ${ }^{206}$ and is closely conserved in both sequence and function between A. parasiticus and A. nidulans. ${ }^{207}$ Chang et al. found that AflR protein bound to a DNA base pair sequence upstream of *aflR*, and thus autoregulated its own production. ${ }^{208}$ Flaherty and Payne increased the expression of *aflR* in A. flavus ${ }^{209}$ and found

Acetyl CoA + Malonyl CoA

$ \downarrow $

EnzS [fas1A]

stcK [fas1B]

$ \downarrow $

abcA A. nidulans genes

[acbA] A. parasiticus genes

{abcA} A. flavus genes

stcA previously pksST

stcS previously verB

stcU previously verA

$ \downarrow $

Versiconal acetate 131

(versiconal hemiacetal acetate)

$ \downarrow $

Esterase stcl

Versiconal 133

(versiconal hemiacetal)

Cyclase/Versicolorin B synthase [vbs]

$ \downarrow $

(-)-Versicolorin B

135

stcS stcU ?

Baeyer Villiger Oxidation

Ketoreductase

Decarboxylation

$ \downarrow $

Dihydrodemethylsterigmatocystin

136

stcP

O-Methyltransferase I

SAM

$ \downarrow $

Dihydrosterigmatocystin

137

omtA

O-Methyltransferase II

SAM

$ \downarrow $

O-Methyldihydrosterigmatocystin

138

{ord1}

Dioxygenation

$ \downarrow $

Aflatoxin B₂

139

Aflatoxin B₁

144

$ \downarrow $

1'-Hydroxyversicolorone

130

stcB, L or W

$ \downarrow $

Versicolol acetate

132

?

Versicolol

134

Demethylsterigmatocystin

141

Sterigmatocystin

142

O-Methylsterigmatocystin

143

Norsolorinic acid

125

stcE [nor-1]

(-)-Averantin 126

Monoxygenase

stcF {avnA}

5'-Hydroxyaverantin

127

(1'S,5'S) and (1'S,5'R)

both converted to averufin

possibly via 5'-oxo

compound or via averufanin

stcG, N or V

Dehydrogenase

(+)-Averufin

129

stcB, L or W

$ \downarrow $

131

Scheme 26

Nat. Prod. Rep., 1999, 16, 425–484

increased levels of some biosynthetic genes and of Afl production, but only under conditions conducive to Afl production. For example, the overexpression did not overcome nitrate inhibition of Afl biosynthesis. and concluded that many other factors, such as medium dependent factors, are required for aflatoxin production.

### 7.2.3 Hexanoate synthetase.
Two research groups have recently located and examined the FAS (secondary metabolite FAS, sFAS) responsible for assembling the hexanoate starter unit for aflatoxin biosynthesis: Mahanti *et al.* have obtained double mutants called UVM8 (*fas-1A* and *nor-1*) by UV irradiation of the norsolorinic acid (Nsa) 125 accumulating mutant of *A. parasiticus*.²¹⁰ Sequence analysis showed homology with the yeast FAS β-subunit. Watanabe *et al.* examined the role of these mutants.²¹¹ Suspended cells incorporated (4%) labelled hexanoic acid into Nsa 125, but under identical conditions gave extremely high (22%) specific incorporation of the NAC thioester of hexanoate. They generated three mutants (Dis-1, Dis-2 and Dis-3) by insertional mutageneses at *fas-1* (FAS-β). Unexpectedly, all three mutants failed to support Nsa production when supplied with labelled hexanoic acid, and Dis-1 and Dis-2 could only support conversion of the NAC thioester to Nsa at 2% levels, whilst Dis-3, a double crossover mutant, failed to support any conversion at all. This led the authors to propose that the sFAS not only assembles hexanoate, but plays a crucial role in its transfer to the PKS, and only when physically associated with an active sFAS, can the PKS begin to assemble Nsa. The authors note that the PKS contains only a single ACP domain, whilst some other fungal PKSs have two, thus part of the sFAS may substitute for a PKS ACP.

Brown *et al.* have shown that *A. nidulans* contains two functional distinct FAS systems.²¹² Mutation of one system led to a requirement for long chain fatty acids, whilst mutation of the second (sFAS) led to unaltered growth except for the total absence of Stc 142 production, unless supplied with hexanoic acid. Brown locates both sets of genes, shows a much greater sequence homology between the primary FAS sequence and other fungal FAS sequences, than to the secondary metabolite FAS (sFAS) sequence.

### 7.2.4 Norsolorinic acid synthetase.
In 1995, Feng and Leonard reported that the *A. parasiticus* PKS gene (*pksL1*) coded for a 2109 residue protein with four domains (ACP, KAS, AT and TE), with no evidence of KR/DH/ER domains, and they generated four *pksL1* disruptants all incapable of Afl production.²¹³ There was a very high (61%) sequence similarity for the developmentally regulated WA protein from *A. nidulans* that is thought to be the PKS for conidial pigmentation, except that PksL1 lacks a second ACP domain.²¹⁴ In 1995 Bhatnagar *et al.* also located the PKS gene from *A. parasiticus*, and labelled it *pksA*, and reported three domains in the order KAS/AT/ACP.²¹⁵ Again in 1995, Yu and Leonard reported the cloning, sequencing and characterising of the PKS gene from *A. nidulans*, labelled *pksST*, coding for a 2181 aa residue polypeptide with KAS/AT/ACP/ACP/TE.²¹⁶ The significance of *A. nidulans* PKS having two duplicated tandem ACPs, and *A. parasiticus* is not yet resolved. As they are back-to-back, one may serve to transfer the starter from the sFAS to the PKS KAS, and the other as the malonate extender. In addition, many other fungal PKSs, such as 6-methylsalicylic acid synthase, do not have a TE. Interestingly, the serine region (GXSXG) of this *A. nidulans* TE has highest sequence similarity with the corresponding regions in ACV tripeptide synthetase, and FASs, both of which are similar to that found in serine esterases.

### 7.2.5 Averantin.
Disruption of the *nor-1* (previously *nar-1*) gene in *A. parasiticus* coding for a 29 kDa alcohol dehydrogenase led to accumulation of averantin 126²¹⁷ (Avn) and a

homologue has been located in *A. nidulans*. However, Nor-1 mutants still produce Afl B1, though at much reduced levels, suggesting other enzymes are present capable of this reduction. A second NADPH dependent protein, NorA, from *A. parasiticus* has shown norsolorinic acid dehydrogenase activity. Bhatnagar has characterised the *norA* gene as coding for a 43 kDa protein, and having 49% homology to an aryl alcohol dehydrogenase gene (*aad*) from *Phanerochaete chrysosporium*. Whilst *norA* is part of the gene cluster being just upstream of *verl*, disruption did not lead to the accumulation of a pigmented product or reduced Afl B1 production, its real role remains uncertain.²¹⁸

Cleveland *et al.* have sequenced the *A. flavus avnA* gene (between *verl* and *omtA*; formerly *ordl*, not to be confused with the *A. parasiticus ordl* gene which appears to convert O-methylsterigmatocystin 143 to Afl B1 144) as coding for 495 residues (mass 56 kDa) with cytochrome P450 motifs, and homology with *stcF* from *A. nidulans*.²¹⁹ Disruption led to accumulation of a yellow pigment thought to be averantin 126 (by TLC analysis), but retained the ability to convert added 5'-hydroxyaverantin 127 to Afl. Thus *avnA* and *stcF* code for a dioxygen and NADPH dependent hydroxylation of averantin, to give either (or both?) 5-hydroxyaverantin or averufanin 128.

Yabe *et al.* have proposed a metabolic grid involving the interconversion of the hemiacetal versiconal acetate 131 and versiconal 133 with their respective open chain alcohols versicol acetate 132 and versicol 134.²²⁰ They have recently isolated and purified two NADPH dependent 40 kDa reductases, designated versiconal hemiacetal acetate reductase I and II from the cytosol of *A. parasiticus* that converted either enantiomer of 131 to a similar mixture (mainly 2′R) of versicol acetate 132 enantiomers.²²¹ The reductases seem to function as a homooligomer of approximate mass 400 kDa in the cytosol. Kushimoto and Hseih have purified and characterised a 32 kDa serine esterase that converts versiconal hemiacetal acetate to versiconal.²²² The purification was complicated by the presence of three non-specific serine esterases also capable of this transformation.

### 7.2.6 Sterigmatocystin.
The dehydrative cyclisation of the hemiacetal, racemic versiconal 133 to (−)-versicolorin B (Ver B) 135 is catalysed by versiconal B synthase (Vbs) that has been isolated by Townsend *et al.* from *A. parasiticus* as a dimeric 78 kDa protein.²²³ Kinetic analysis suggested that the synthase selects one from the two rapidly equilibrating enantiomers of the substrate, cyclising only the desired enantiomer to enantiopure Ver B 133. The synthase gene has been located in the aflatoxin cluster.²²⁴ Attempts to overexpress Vbs in *E. coli* by Silva and Townsend led to inactive protein principally as inclusion body with low levels of soluble protein with molecular weight 7 kDa less than the native Vbs.²²⁵ As native synthase has three potential N-linked glycosylation sites (NXS or NXT) and treatment with N-glycosidase F reduced its molecular weight by 7 kDa, this inactivity was thought to be due to lack of glycosidation. Thus the protein was overexpressed in a recently developed ‘super secreting’ strain of yeast (*Saccharomyces cerevisiae*) that resulted in 100-fold increased levels (2–8 mg per L cell culture) of posttranslationally modified active protein that is continually excreted from the cell. This new methodology should now allow the overexpression and full characterisation of the other enzymes involved in Afl biosynthesis. The presence of the carbohydrate residues may be crucial for homodimer formation and activity, as the X-ray of a partially deglycosylated derivative showed sugar residues at the dimer interface.²²⁶

Versicolorin B (Ver B) 135 can now be converted into Afl B2 139 or acted upon by a desaturase, StcL (a cytochrome P-450 monooxygenase), that inserts the dihydrofuran double bond leading to the production of Afl B1 144.²²⁷ The high affinity of StcL for Ver B ensures that *A. nidulans* only accumulates Stc

Nat. Prod. Rep., 1999, 16, 425–484 469

142 not dihydro-Stc 137. A disruptive insertion into the *stcL* gene from *A. nidulans* (homologous to *A. parasiticus* and *A. flavus ordl*) led to the accumulation of dihydrosterigmatocystin 137,²²⁷ suggesting that StcL desaturates the terminal furan of Ver B 135 to give Ver A 140. Feeding the mutant (-)-versicolorin A led to the accumulation of both dihydrosterigmatocystin 137 and sterigmatocystin 142. The *stcL*, *stcU* double mutant led to the accumulation of only Ver B 135, indicating that StcU (a ketoreductase) acts on both versicolorin A 140 and Ver B 135.

Watanabe and Townsend have examined the oxidative ring cleavage of versicolorin A or B by analysing the incorporation of dioxygen into Afl B1,²²⁸ and found that ¹⁸O was incorporated (40%) at C-5 of demethylsterigmatocystin, in agreement with a possible Baeyer–Villiger-like process²²⁹ and that the C-1 oxygen of Afl B1 was derived from dioxygen (40% incorporation). The authors discuss various possible mechanisms for the conversion of sterigmatocystin into Afl B1, some of which was summarised in the last review.¹ Skory *et al.* have shown that *verI* (*A. parasiticus*) codes for the ketoreductase and has high homology with *stcU* and other NADPH requiring reductases.²³⁰ Keller *et al.* describe *stcS* (formerly *verB*) that is closely located to *stcU*, disruption of which leads to versicolorin accumulation, as a cytochrome P-450 involved in the conversion of versicolorin A 140 to demethylsterigmatocystin 141.²³¹

O-Methyltransferase I, which catalyses the conversion of demethylsterigmatocystin 141 to sterigmatocystin 142, and didehydrodemethylsterigmatocystin 136 to dihydrosterigmatocystin 137 has been purified by Yabe *et al.* to apparent homogeneity (mass: apparent 150 kDa, denatured 43 kDa) from *A. parasiticus*.²³²

The *omtA* genes that are thought to code for O-methyltransferase II from *A. parasiticus* and *A. flavus* have been compared. Each species only has a single copy of the gene adjacent to *verI*, and both genes have five exons interrupted by four short introns (50–60 bp) and show a 97% homology within coding region, and have identical SAM binding motifs.²³³ OrdI is thought to catalyse the final step, the conversion of O-methylsterigmatocystin 143 to Afl B1 144. Several steps are involved, including dioxygenation, decarboxylation and mono-oxygenation, presumed to be catalysed by a large oxidereductase, but no intermediates have ever been isolated, and the exact pathway is still unresolved. Prieto and Woloshuk have identified the *ordI* gene from *A. flavus* with six introns and seven exons that codes for 528 residues with predicted mass of 60 kDa.²³⁴ The sequence contains a conserved cysteine containing heme binding fragment only found in cytochrome P-450s, yet no significant (>40%) homology was found elsewhere in the sequence with any known Cyt P-450 (including others in the Afl pathway), thus this protein has been assigned as belonging to a new cytochrome P-450 family (CYP64). Heterologous expression in *Saccharomyces cerevisiae* resulted in the ability of the transformants to convert O-methylsterigmatocystin 143 to Afl B1 144. This may be due to coupling of the OrdI with a yeast hydroxylase (*e.g.* cinnamate 4-hydroxylase) known to be in the endoplasmic reticulum.

### 7.3 Aspyrone

The five metabolites, aspyrone, asperlactone, isoasperlactone (all from *Aspergillus melleus*), aspinonene (*A. ochraceus*)²³⁵ and avellaneol (*Hypocrea avellanea*)²³⁶ are all now thought biosynthetically related. Their unusual branched carbon skeletons has prompted extensive biosynthetic investigations as they have not been easy to rationalise in a polyketide manner. Early investigations using ¹³C-labelled acetates confirmed that all the carbons in aspyrone were indeed acetate derived, though extensive rearrangement had taken place. Holker and Simpson found that feeding [1,2-¹³C₂]acetate gave one bond couplings between carbons 3 + 4, 6 + 7 and 9 + 10, and also long range coupling (6 Hz) between 2 and 8, suggesting that those two

carbons were from a common acetate unit *via* an intramolecular Favorskii type rearrangement of a linear polyketide intermediate, followed by decarboxylation (Fig. 12).²³⁷ In 1984

![Fig. 12](https://i.imgur.com/1234567.png)

Staunton *et al.* showed that the pattern of acetate incorporation into asperlactone 146 was as for aspyrone 145, and found that

![Aspyrone and Asperlactone Structures](https://i.imgur.com/8901234.png)

upon feeding [2-¹³C, ²H₃]acetate, three deuteriums were retained at C-10, and two at C-7, ruling out previously suggested biosynthetic schemes involving aromatic co-metabolites such as mellein or hydroxymellein.²³⁸ Simpson, Staunton, Vederas *et al.* surprisingly found no incorporation from acetate into any oxygen atom in aspyrone or asperlactone. However, fermentation in an ¹⁸O₂ enriched atmosphere gave high incorporation at C-5, C-8/9 epoxide, with incorporation at both carbonyl and lactone oxygen at C-2 at half the intensity found at C-5, C-8 and C-9. Presumably, one oxygen at C-2 originates from the medium. Analogous results were obtained for asperlactone. The authors proposed a linear trienone intermediate 150, followed by epoxidation to give 151 and rearrangement to the aldehyde 152 (Scheme 27).²³⁹ Formation of a diepoxide 153 followed by intramolecular opening of one epoxide gives asperlactone 146 and aspyrone 145. However, the low quantities of labelled asperlactone 146 and isoasperlactone 147 obtained precluded analogous examination by ¹⁸O isotope shifted ¹³C NMR spectroscopy on these compounds. The stereochemistry observed in isoasperlactone 147 could be explained through a diastereotopic diepoxide intermediate 154, or *via* ring opening of aspyrone 145 followed by ring closure to the five-membered ring using the alternative hydroxy group. This would result in two different labelling intensities at the corresponding O atoms. In 1991, Staunton and Sutkowski used ¹⁷O NMR spectroscopy to resolve this issue. The ¹⁷O nucleus can be directly observed, but being quadrupolar, resonances are broad. However, due to low natural abundance (0.04%), the method is intrinsically very sensitive, and very low incorporations can be detected. Pulse breakthrough, which can lead to serious distortion of the baseline, was eliminated by setting the first few points of the FID to zero. As expected, feeding [¹⁷O]acetate gave ¹⁷O NMR spectra similar to the natural abundance spectra of asperlactone and isoasperlactone, showing, to the very low experimental limits, that no oxygens were derived from acetate. When the organism was grown in an atmosphere enriched with ¹⁷O₂, the resonance intensities for isoasperlactone 147 were consistent with the pathway *via* rearrangement of aspyrone 145, and not *via* the diastereomeric diepoxide 154,²⁴⁰ thus all three metabolites are biosynthesised *via* diepoxide 153a. An alternative linear PKS intermediate to the above trienone 150 is (4E,6E,8E)-3-hydroxydeca-4,6,8-tri-

PKS → HOOC—C—C=O — C=C—Me

'Trienone'
150

HOOC—C—C=O — C=C—Me
151

rearrangement
−CO₂

Me—C—CHO
152

[ ], −H₂O
NAD⁺/H₂O²⁺ * O₂

Me—C—CHO
154

* O
Me

* O
Me

153a

Me—C—O—Me
Asperlactone
146

Me—C—O—Me
Aspyrone
145

Observed labelling pattern
Not the observed labelling pattern

Me—C—O—Me
Isoasperlactone
147

Scheme 27

elimination was 'syn', the same as observed in FAS (Scheme 30).²⁴⁶

Me—C—O—Me
Isoasperlactone
147

Me—C—CH₂OH—O—Me
Aspinonene
148

CHO—C—O—H
Avellaneol
149

### 7.4 Cyclosporin A

Cyclosporin A is a cyclic undecapeptide produced by *Tolypocladium niveum* that has important clinical use as an immunosuppressant to prevent organ rejection after transplant surgery. The cyclic peptide is assembled via a non-ribosomal peptide synthetase rivalling in complexity the multienzyme macrolide polyketide synthetases.²⁴⁷–²⁴⁹ One of the eleven constituent amino acids is (2S,3R,4R,6E)-2-amino-3-hydroxy-4-methyl-oct-6-enoic acid 187 (Bmt) that was shown by Kobel et al. to be assembled via a polyketide pathway.²⁵⁰,²⁵¹ Offenzeller et al. have shown that (3R,4R,6E)-3-hydroxy-4-methyl-oct-6-enoyl thioester 186 is the product of the Bmt PKS which is released as the CoA thioester for subsequent oxidation and transamination to the amino acid Bmt 187.²⁵² Using partially purified Bmt synthase, Schneider-Scherzer et al. added various radiolabelled precursors along with SAM, malonyl CoA and NADPH, and examined the resulting extracts by HPLC (Scheme 31).²⁵³ (2E)-But-2-enoyl CoA 173 was incorporated into 186, confirming a processive pathway, however, butyryl CoA 176 was also incorporated into 186 just as efficiently, but also into the dihydroanalogue, (2S,3R,4R)-3-hydroxy-4-methyl-octanoyl 177, indicating some relaxed substrate specificity in the synthase. Hexanoyl CoA was converted into 3-hydroxy-octanoyl CoA, oct-3-enoyl CoA and octanoyl CoA, but not into any methylated derivative. As butyryl CoA but not hexanoyl CoA was converted into methylated products, this suggested that methylation (of 174 to give 180) was occurring at the β-position during the second PKS cycle prior to full reduction. When 3-oxo-hexanoyl CoA 189 (the saturated analogue of the expected biosynthetic intermediate, (4E)-3-oxo-hex-4-enoyl thioester 174, as that was found to be unstable under the incubation conditions) was added, several unmethylated products were obtained, but also 3-hydroxy-4-methyl-octanoyl CoA 177. When sorbyl CoA, (2E,4E)-hexa-2,4-dienoyl CoA 182, was added, no methylated product was obtained, again suggesting that (4E)-3-oxo-hex-4-enoyl thioester 174 was the substrate for methylation. In support, adding 3-oxo-hexanoyl CoA 189 to an incubation mixture with no added NADPH or malonyl CoA gave 2-methyl-3-oxo-hexanoyl CoA 192 in good yield (Scheme 32), whilst sorbyl CoA 182 was not methylated (*vide supra*). A potential

Compounds in square brackets not incorporated

Acetate + 4 x Malonate → HOOCCH(OH)CH=CHCH=CHCH=CHCHO (159)

HOOCCH(OH)CH=CHCH=CHCH=CHCHO (159) → HOOCCH(OH)CH=CHCH=CHCHO (Avellaneol 149)

Aspinonene (148) ← CHOCH2OH

Diepoxide (153b) ← CHO

Aspyrone (145) → COOH

Scheme 28

D3C D OH CH2 SNAC (165)
D3C OH CH2 SNAC (164)
D3C O SNAC (163)
HO D S CH2 SNAC (162)
D OH CH2 SNAC (161)
D3C O CH2 SNAC (160)

Specifically deuterated Aspyrone (145)

D3C D CH2 SNAC (166)
D OH CH2 SNAC (167)
HO D CH2 SNAC (168)
D3C D D CH2 SNAC (169)
D OH D D CH2 SNAC (170)
D OH D D CH2 SNAC (171)

Scheme 29

problem in methylation of the 3-oxo substrates is the competing very rapid enzymatic reduction of them by NADPH directly to the corresponding 3-hydroxy compounds before the methylation can take place. For example, incubation of 189 with NADPH and SAM (no malonyl CoA) gives 3-hydroxyhexanoyl thioester 190 (in the absence of methylated analogues) which was reduced to (2E)-hex-2-enoyl thioester 191 and hexanoate. In an ingenious modification of these incubation experiments, the authors added NADPH only after a 5 min preincubation period with SAM (no malonyl CoA), to give 3-hydroxy-2-methylhexanoyl CoA 193 which dehydrated to give 194 and 195. No saturated analog, 2-methylhexanoyl CoA 196 could be located. Incubation of 3-oxohexanoyl CoA 189 with SAM and malonyl CoA gave partial methylation, resulting in 3,5-dioxooctanoate 197 and the methylated analogue 3,5-dioxo-4-

Specifically labelled aspyrone (145)
Unlabelled aspyrone (145)

Scheme 30

Nat. Prod. Rep., 1999, 16, 425–484

Me  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=O}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce{=O}$  
SR  

$\ce{->}$ Me  
$\ce{=C}$  
$\ce{CH2}$  
$\ce
216 by cell-free extracts. (Scheme 33). 260–262 However, no intermediate at a lower oxidation level than 208 has been reported. This led Vederas *et al.* to reexamine the origin of the oxygen atoms in lovastatin using an improved strain of *A. terreus*. Fermentation with sodium [1-¹³C, ¹⁸O₂]acetate only gave a single isotope shift in the SEFT NMR at C-1′ as before, with only broadened resonances at C-11, C-13 and C-15, the other expected sites of incorporation.²⁶³ As the magnitude of ¹⁸O isotope shifts can be increased by derivatisation with electron withdrawing groups,²⁶⁴ the sample was reduced (LiAlH₄) and acetylated to give the tetraacetate 218. ¹³C NMR analysis now gave isotope shifts at C-11 (37 ppb), C-13 (36 ppb), C-15 (30 ppb), however considerable isotope exchange had evidently occurred. Feeding the NAC thioester of labelled acetate gave similarly high loss of ¹⁸O. Fermentation under ¹⁸O₂ gave a sample of lovastatin with only one detectable ¹⁸O isotope shift at C-8 of 34 ppb. However analysis by MS showed that fermentation of the old strain (ATCC 20542) had given (CI MS using isobutane) lovastatin 32% of which contained one ¹⁸O, 6% two, 4% three, 4% four and no five. The new, higher yielding strain (MF 4845) gave (CI MS using ammonia) ca. 25% singly labelled, 5% doubly, with no evidence of triple or higher incorporation, still strongly suggesting that a metabolic pathway must be present that is incorporating the labelled dioxygen into acetate derived oxygens (*vide infra*). The closely related (6-desmethyl) compound, compactin 215, is produced by *Penicillium aurantiogriseum* and *P. citrinum* whose biosynthesis from ¹³C-labelled acetate and methionine in *P. citrinum* has been studied by Endo *et al.* with the expected pattern.²⁵⁹ In a recent report, Vederas *et al.* examined the incorporation of deuterated or ¹⁸O labelled acetate and dioxygen into compactin and the putative intermediate ML-236C 211 from *P. aurantiogriseum*, and reexamined the incorporation of dioxygen into lovastatin from *A. terreus* MF4845.²⁶⁵ For ¹⁸O experiments chemical conversion to the tetraacetate 217

allowed ¹³C NMR analysis in combination with MS. The results from feeding ¹⁸O labelled acetate with compactin were as for mevinolin, except that there had been little loss of acetate derived ¹⁸O due to adventitious exchange. Vederas suggests that fatty acids in the fungus are being aerobically ω-oxidised, and then converted into labelled acetate, resulting in significant levels of dioxygen being incorporated at acetate positions (Scheme 34). Feeding experiments with deuterated acetate in compactin 215 and ML-236C 211 gave similar results to that found earlier for lovastatin 216, with consistent loss of all deuterium at C-3 and C-6 in compactin and equivalent sites on ML-236C. Whilst these labels can always be lost through adventitious exchange, there is no chemically rational expectation to lose labels at these two sites, and not others, with the current understanding of the biosynthetic sequence. Unexpected high levels of isotope shifted resonances at C-1 and C-2′ after feeding [2-¹³C, ²H₃]acetate are thought due to a γ-isotope shift (40 ppb) from the nearby triply labelled starter unit methyls due to multiple acetate ‘pulse’ labelling within individual molecules. These sites may indeed be directly labelled, but the multiple ‘γ’ incorporation obscures the spectral region.

There has been much speculation about possible biological Diels–Alder and other pericyclic reactions.²⁶⁶ The structures of the proposed PKS product in lovastatin biosynthesis 206 is very suggestive of a Diels–Alder cyclisation with an *endo* conformation and pseudoaxial methyl group of a hexaketide intermediate 203 during the assembly of the polyketide chain (Scheme 33). Vederas and Witter have prepared the NAC thioester of [2,11-¹³C₂]-(2E,6R,8E,10E)-6-methyldodeca-2,8,10-trienoic acid 204, but were unable to get intact incorporation into lovastatin due to rapid degradation.²⁶⁷ However, analogues of this compound underwent facile *in vitro* Diels–Alder cyclisation in both thermal or Lewis acid catalysis to give a mixture of *endo* and *exo* products both formed *via* chair like transition states.

Acetate  
Malonate  
SAM and NADPH  

$\downarrow$  

COSEnz  

Putative hexaketide intermediate on PKS  

R = H 202 During compactin biosynthesis (*P. aurantiogriseum*)  
R = Me 203 During lovastatin biosynthesis (*A. terreus*)  

• = ${}^{13}$C, R = Me 204  

Enzymatic Diels Alder?  

Putative intermediate on PKS after Diels  
Alder cyclisation of hexaketide intermediate  

R = H 205 R = Me 206  

Putative PKS product  

R = H 207 ML236L  
R = Me 208 (4a,5)-Dihydromonacolin L  

R = H 209  

R = Me 210 3α-Hydroxy-3,5-dihydromonacolin L  

R = H 211 ML-236C  
R = Me 212 Monacolin L  

R = H 213  
R = Me 214 Monacolin J  

R = H 215 Compactin  
R = Me 216 Lovastatin/Mevinolin  

R = H 217  
R = Me 218  

Scheme 33  

involved in melanin biosynthesis in *Alternaria alternata* has been reported.²⁷⁰  

Oikawa *et al.* have determined the origin of solanapyrone A  
221 (Scheme 35), a phytoxin produced by the pathogenic  
fungus *Alternaria solani*.²⁷¹ The retention of deuterium at C-5  
rules out many possible mechanisms to form the decalin ring,  
such as ring closure exploiting a carbonyl at C-6 (would result  
in a double bond between C-5 and C-10), or those involving  
aromatic intermediates followed by reduction, but are consistent  
with a Diels–Alder cyclisation with an exo transition state.  
Oikawa *et al.* isolated a diastereoisomer, solanapyrone D 222 as  
a minor component, which has the same stereochemistry at C-1  
and C-10, but opposite configuration at C-2 and C-5, which  
could arise from a similar transition state to that for solanapyrone A, but endo rather than exo.²⁷² Apparently, the active site  
holds the dienophile and methyl parts tightly, but allows either  
conformation of the diene. The two transition states are so  
similar that only a single active site may be required for both. In  
aqueous conditions (30 °C, 10 min), Oikawa found that 15% of  
a sample of prosolanapyrone III 220 is converted mainly into  
the endo product solanapyrone D 222, with only a trace of exo  
product solanapyrone A 221 (ratio endo:exo of 97:3), strongly  
suggesting an enzymatic Diels–Alder cyclisation in vivo. Under  
these in vitro conditions, no cycloaddition products were  
obtained from a sample of prosolanapyrone II 219. Oikawa *et al.*  
have developed a cell-free system from *Alternaria solani* that is  
able to catalyse the (25%) conversion of prosolanapyrone III  
220 to predominantly solanapyrone A 221 (53%) with high  
enantioselectivity, suggesting that the oxidation and Diels–  
Alder cyclisation may be enzymatically linked,²⁷³ along with  
solanapyrone D (47%).²⁷⁴ As denatured enzyme under the same  

7.6 Other metabolites  

*Colletotrichum lagenarium* is a plant pathogenic fungus that  
causes anthracnose of cucumber. The biosynthesis of melanin  
by the fungus is closely related to its penetration of the host  
plant. An albino mutant is unable to penetrate its host as melanin  
is required for the cell wall rigidity needed for formation of the  
‘penetration peg’. The *pks1* gene has been cloned,²⁶⁸ and when  
transformed into the albino mutant restored the melanin  
phenotype. Sequence analysis shows 2187 residues with KAS/  
AT/ACP domains and a very high homology throughout the  
entire sequence with wA from *Aspergillus nidulans* that is  
responsible for conidial pigmentation, but only active domain  
homology with 6MSAS from *P. patulum*.²⁶⁹ The gene cluster  

Acetate  
+  
Malonate  
+  
SAM  

$\ce{HOH2C}$
$\ce{OMe}$

$\ce{O}$
$\ce{O}$

$\ce{Me}$

Prosolanapyrone II 219

$\ce{OHCHO}$
$\ce{OMe}$

$\ce{O}$
$\ce{O}$

$\ce{Me}$

Prosolanapyrone III 220

$\ce{CHO}$

$\ce{D_3C}$

$\ce{HO?}$

$\ce{D}$

$\ce{D}$

$\ce{S}$

$\ce{D}$

$\ce{CHO}$

$\ce{OMe}$

Diels-Alder  
(exo)

$\ce{Me}$

$\ce{MeO}$

$\ce{CHO}$

220

Solanapyrone A 221

$\ce{CHO}$

$\ce{OMe}$

Diels-Alder  
(endo)

$\ce{Me}$

$\ce{MeO}$

$\ce{CHO}$

220

Solanapyrone D 222  
(Minor component)

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{O}$

$\ce{
Me CH₂CH₂CH₂CH₂OH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH OH COOH
notably against *Phytophthora infestans* (responsible for potato blight). Breinholt *et al.* have shown that it is assembled from three acetate units and two C₁-units from methionine, including C-8.²⁸⁹ The authors show that [2-¹³C]-1,3-dihydroxy-2,4-dimethylbenzene **240** was specifically incorporated to give 7-¹³C labelled xanthofusin. This is surprising, as it would be easier to rationalise labelling of the other SAM derived methyl group, C-8 (Fig. 13). The authors propose three possible biosynthetic routes to **239** all needing oxidative cleavage of the aromatic system: A triketide with methyl terminal methylation (R = H) (as observed for borophycin); a triketide with a malonate starter unit (R = COOH) (as observed for tetracyclines) and a tetraketide with total loss of the starter unit.

O'Hagan and Chesters have investigated the biosynthesis of piliformic acid **244** (Scheme 38) from *Poronia piliformis*, a slow growing dung fungus which produces optically active (as yet unknown configuration) piliformic acid, and *Xylaria mali*, which grows on rotting wood and produces racemic piliformic acid.²⁹⁰ Addition of [1,2-¹³C₂]acetate to *P. piliformis* resulted in extensive labelling of the C₈-unit, with only low level labelling of the C₃-unit, with C-1 and C-2 arising from an intact unit (Scheme 38). Deuterated acetate gave single single labelling (2.5%) at C-5 and C-7, with very low single labelling at C-11 (0.2%) which is consistent with acetate having been processed to oxaloacetate and decarboxylated to a C₃-unit. The stereochemistry of the acetate derived deuterium on the C₈-unit at C-5 was determined to be *pro-(R)*, the same as found in all fungal FAS systems so far examined, and opposite to known fungal PKS systems. This enoyl reductase stereochemistry suggests a FAS rather than a PKS origin for the C₈-unit, which could arise from β-oxidation of a longer chain, or by a specific short chain FAS. Labelled alanine, a metabolic precursor of pyruvate (2-oxopropanoate), was not incorporated (<0.1%) into the C₃-unit. [2,2,3,3-²H₄]Succinic acid resulted in the efficient incorporation of a single deuterium at the methyl of the ‘C₃-unit’, suggestive of processing via the citric acid cycle to [3-²H₁]oxaloacetate { [3-²H₁]-2-oxobutane-1,4-dioic acid }. O’Hagan and Chesters then specifically incorporated [1-¹³C]octanoate (5.5%) into piliformic acid, with low level (2%) background incorporation at ‘acetate carbonyl’ derived positions due to fatty acid oxidation. In order to probe the mechanism of attaching the C-3-unit to the octanoate, O’Hagan fed [²H₁₅]octanoate (to *P. piliformis* and *X. mali*) and the resulting piliformic acid was analysed by ²H NMR spectroscopy which showed deuterium incorporation at all the C₈-unit sites, including the double bond carbon C-4, ruling out β-oxidation to 3-oxooctanoate. This leaves as possible mechanisms a thiolase route or more likely an octanoyl CoA carboxylase mechanism via hexanylemalonyl CoA (octanoate has been incorporated intact into the polyene fungichromin, presumably via hexanylemalonyl CoA²⁹¹). Quite unexpectedly, deuterium was also observed at C-2, the central carbon of the attached C₃-unit, with an equal intensity to that found at C-4, consistently in piliformic acid from both organisms. Thus there is coherent transfer of the migrating deuterium which is not diluted by adventitious exchange processes (this rules out the formation of racemic piliformic acid in *X. mali* by racemisation as this would involve loss of some label). O’Hagan rationalises this by proposing a 1,3-hydrogen shift during double bond isomerisation (Scheme 39). O’Hagan speculates on how such a process can result in a racemic product as found in *X. mali*. For example, does one diastereotopic deuterium get transferred suprafacially to give one enantiomer, and the the other diastereotopic proton get transferred in equal quantities to give the opposite enantiomer, resulting in the racemate? Would this require two enzymes? If so they would need to operate equally efficiently. To demonstrate that this deuterium originates from C-3 and not C-2 of octanoate, O’Hagan and Chesters fed sodium [1-¹³C, 2-²H₁]octanoate. A high level of specific ¹³C incorporation at C-10 was observed, and some deuterium label via deuterated acetate, but there was no evidence for deuterium enrichment at C-2 of piliformic acid, which would therefore originate from C-3 of octanoate, not C-2. Support for the proposed mechanism comes from reports in the literature of fungal metabolites such as caperatic acid **245**.²⁹² O’Hagan *et al.* probed the origin of the octanoate unit by feeding [3-¹³C]decanoate.²⁹³ Overall incorporation into **244** was low, and indicative of degradation to acetate followed by incorporation, rather than degradation to octanoate and incorporation. Similarly, 8-fluorooctanoate was incorporated to give 9-fluoropiliformic acid, whilst 10-fluorodecanoate was not incorporated, strongly suggesting that the octanoate unit is not obtained by degradation of longer fatty acids, in direct contrast to that observed in the biosynthesis of fungichromin by the prokaryote *Streptomyces*

478 Nat. Prod. Rep., 1999, 16, 425–484

![Scheme 39](attachment:Scheme_39.png)

with retention of the amino nitrogen, all three carbons, both diastereotopic C-3 hydrogens, but some adventitious exchange of the C-2 hydrogen. As all four atoms attached to the α-carbon are retained, Harrison assigns the absolute configuration at C-5 of **246** to be S. ${}^{294b}$

Breinholt *et al.* have examined the biosynthesis of hypomycetin, a yellow crystalline metabolite produced by the mycoparasitic *Hypomyces aurantius*. ${}^{295}$ Hypomycetin closely resembles the tetracyclines produced by many *Streptomyces* spp., however Breinholt finds that a non-linear decaketide folding pathway CYC1(6/11)CYC2(4/13)CYC3(2/15)CYC4(1/18) is utilised in contrast with that found for the tetracyclines, consistent with the independent evolution of the two pathways due to the taxonomical distance involved. In this case, horizontal gene transfer does not appear to have occurred.

![Structures](attachment:Structures.png)

Harrison *et al.* have investigated the biosynthesis of the pyrrolidin-2-one containing pramanicin **246** and the related epoxyacid (2E)-4,5-epoxytetradecenoic acid **247** from a *Stagonospora* sp. and found the incorporation of eight acetates and one serine consistent with amidation of a linear octaketide and Claisen-like condensation onto the activated serine carboxylate. ${}^{294a}$ Harrison *et al.* then show by intact incorporation experiments that L-serine is enantiospecifically incorporated,

7.7 Lichen

A series of chlorinated emodins have been isolated from the foliose lichen *Nephroma laevigatum* (Nephromataceae) commonly found on shoreline rocks.²⁹⁶ Emodins are known to be assembled via the PK pathway in higher plants and fungi. Cohen and Towers have incorporated sodium [1-¹³C]acetate into 7-chloro-1-O-methylemodin 249 from the lichen and obtained enhancements of between 3.1% and 6% at C-1, 3, 4a, 6, 8, 9 and 10a; 1.9% enhancement at C-10, and 0.7% or less at other carbons. These results appear only consistent with the IUPAC numbering of anthraquinones (as shown) and not with the numbering system shown in the paper. The high level of incorporation at C-10, an 'acetate methyl' derived carbon, over half that found for many carbonyl derived carbons, and very much higher than other acetate methyl derived carbons, and if experimentally significant deserves comment and is not easily rationalised. The authors demonstrate efficient incorporation of Na³⁶Cl showing that the lichen itself was capable of chlorination, though no chloroperoxidase has yet been isolated from a lichen.

precursor to the 'methylated' melleins. However, feeding the ants [3-²H₃]propionic acid gave high levels of incorporation as shown by MS, demonstrating incorporation of one propionate in 251; two in 3-ethyl-8-hydroxyisochroman-1-one 252 and 8-hydroxy-3,5,7-trimethylisochroman-1-one 253; and three in 3,4-dihydro-5,7-dimethyl-3-ethyl-8-hydroxyisochroman-1-one 254. They then fed labelled propionate to *C. ligniperdus*, which uses the δ-lactones: 2,4-dimethylhexano-5-lactone 255, 2,4,6-trimethyloctano-5-lactone 256, and 2,4,6-trimethylno-nano-5-lactone 257 as its trail pheromones (and *C. herculeanus* which uses just 255 and 257), and found two, three and four propionate units incorporated respectively, pheromone 257 being exclusively biosynthesised from propionate. Whilst use of antibiotics did not reduce pheromone biosynthesis, biosynthesis by endosymbiotic microorganisms cannot be ruled out.

The marine bryozoan *Bugula neritina* (Bugula: ectoprocta, moss animals) produces the bryostatins such as bryostatin 1 258, which in combination with vincristine is in clinical development to treat cancer. Kerr et al. added radiolabelled precursors to crude enzyme preparations (with ATP, Mg²⁺, NADH and NADPH) and found incorporation of acetate, SAM, and glycerol, but no incorporation of propionate, butanoate, 2-methylpropionate or succinate.²⁹⁹

The Eastern Pacific dorid nudibranch ('sea slugs' *Triopha catalinae* (mollusca) produce triophamine 259, a diacylguanidine metabolite. Recently, Graziani and Anderson showed that each C₁₀ acyl unit is biosynthesised from five acetate units.³⁰⁰ Kubanek and Anderson have now injected aqueous solutions of [2,3-¹³C]ethylmalonate and [2,3-¹³C]butanoate directly into the dorsum, and then after seven days in sea water, extracted the

8 Animalia

Davies-Colman and Garson have recently reviewed marine polypropionates including their biosynthesis, a wide range of which are assembled by the mollusca.²⁹⁷ All those derived from the mollusca are assembled via propionate building blocks, rather than a 'C₂ + C₁' process, whilst protocist derived marine metabolites may use methionine or other building blocks.

Bestmann et al. have investigated the biosynthesis of mellein (8-hydroxy-3-methylisochroman-1-one) and δ-lactone trail pheromones in several species of ants (Formicinae).²⁹⁸ They proposed that mellein 250 was ingested by the ants, methylated to 3,7-dimethyl-8-hydroxyisochroman-1-one 251, and then stored as trail pheromones in their rectal bladders. *Camponotus rufipes*, *C. silvicola* (carpenter ants) and *Lasius niger* (black garden ant), which all use a combination of the 'methylated' melleins 251–254 as their trail pheromones, were fed labelled pentadeuteromellein [methyl-³H₂,3,4-²H₂]-3,4-dihydro-3-methylisochroman-1-one] for fourteen days. Subsequent analysis of their rectal bladders by GCMS showed uptake of this precursor into the bladder at high levels, but no detectable conversion into labelled methylated trail pheromones 251–254. In addition, feeding deuteriated methionine failed to label the 'methylated' melleins. Thus it was unlikely that mellein was a

![Chemical Structures](https://i.imgur.com/yourimage.png)

480 Nat. Prod. Rep., 1999, 16, 425–484

animals.³⁰¹ The butanoate was specifically labelled at the four expected positions to give expected doublets in the ¹³C NMR spectrum, but at low levels (0.17%), and ethylmalonate was not effectively incorporated. However, this does represent the first demonstration of butanoate incorporation in marine invertebrate polyketide biosynthesis.

## References

1. B. J. Rawlings, *Nat. Prod. Rep.*, 1998, **15**, 275; 1997, **14**, 523; 1997, **14**, 335.
2. D. O'Hagan, *Nat. Prod. Rep.*, 1995, **12**, 1; 1993, **10**, 593; 1992, **9**, 447.
3. T. J. Simpson, *Nat. Prod. Rep.*, 1991, **8**, 573, 1987, **4**, 339; 1985, **2**, 321; 1984, **1**, 28.
4. L. Margulis and K. V. Schwartz, *Five Kingdoms—An Illustrated Guide to the Phyla of Life on Earth*, W. H. Freeman, New York, 3rd edn., 1998.
5. T. D. Brock, M. T. Madigan, J. M. Martinko and J. Parker, *Biology of Microorganisms*, Prentice-Hall, London, 7th edn., 1994.
6. D. A. Hopwood, *Chem. Rev.*, 1997, **97**, 2465.
7. C. W. Carreras, R. Pieper and C. Khosla, *Top. Curr. Chem.*, 1997, **188**, 85.
8. T. J. Simpson, *Top. Curr. Chem.*, 1998, **195**, 1.
9. C. R. Hutchinson and I. Fujii, *Annu. Rev. Microbiol.*, 1995, **49**, 201.
10. J. Staunton and B. Wilkinson, *Top. Curr. Chem.*, 1998, **195**, 49.
11. C. L. Hershberger, *Curr. Opin. Biotechnol.*, 1996, **7**, 560.
12. D. E. Cane, *Science*, 1994, **263**, 338.
13. C. J. Tsoi and C. Khosla, *Chem. Biol.*, 1995, **2**, 355.
14. J. Mann, *Nature*, 1995, **375**, 533
15. T. J. Simpson, *Chem. Ind.*, 1995, 407.
16. J. Rohr, *Angew. Chem., Int. Ed. Engl.*, 1995, **34**, 881.
17. P. F. Leadlay, *Curr. Opin. Chem. Biol.*, 1997, **1**, 162.
18. G. L. Verdine, *Nature*, 1996, **384**, 11.
19. C. Khosla and R. J. X. Zawada, *Trends Biotechnol.*, 1996, **14**, 335.
20. A. Nakagawa and S. Ōmura, *J. Antibiot.*, 1996, **49**, 717.
21. H. Kleinkauf and H. von Döhren, *J. Antibiot.*, 1995, **48**, 563.
22. C. Khosla, R. Caren, C. M. Kao, R. McDaniel and S.-W. Wang, *Biotechnol. Bioeng.*, 1996, **52**, 122.
23. H. Fu and C. Khosla, *Mol. Diversity*, 1995, **1**, 121.
24. F. Malpartida and D. A. Hopwood, *Nature*, 1984, **309**, 462.
25. H. Reichenbach, K. Gerth, H. Irschik, B. Kunze and G. Höfle, *Trends Biotechnol.*, 1988, **6**, 115.
26. K. Gerth, N. Bedorf, H. Irschik, G. Höfle and H. Reichenbach, *J. Antibiot.*, 1994, **47**, 23.
27. N. Bedorf, D. Schonberg, K. Gerth, H. Reichenbach and G. Höfle, *Liebigs Ann.*, 1993, **9**, 1017.
28. (a) T. Schupp, C. Toupet, B. Cluzel, S. Neff, S. Hill, J. J. Beck and J. M. Ligon, *J. Bacteriol.*, 1995, **177**, 3673; (b) A. M. Hill, J. P. Harris and A. P. Siskos, *Chem. Commun.*, 1998, 2361.
29. D. Schummer, D. Schomberg, H. Irschik, H. Reichenbach and G. Höfle, *Liebigs Ann.*, 1996, 965.
30. H. Augustiniak, H. Irschik, H. Reichenbach and G. Höfle, *Liebigs Ann.*, 1996, 1657.
31. D. Schummer, K. Gerth, H. Reichenbach and G. Höfle, *Liebigs Ann.*, 1995, 685.
32. J. Rohr, *J. Org. Chem.*, 1992, **57**, 5217.
33. R. McDaniel, S. Ebert-Khosla, H. Fu, D. A. Hopwood and C. Khosla, *Proc. Natl. Acad. Sci. USA*, 1994, **91**, 11 542.
34. P. J. Kramer, R. J. X. Zawada, R. McDaniel, C. R. Hutchinson, D. A. Hopwood and C. Khosla, *J. Am. Chem. Soc.*, 1997, **119**, 635.
35. H. Fu, D. A. Hopwood and C. Khosla, *Chem. Biol.*, 1994, **1**, 205.
36. R. McDaniel, S. Ebert-Khosla, D. A. Hopwood and C. Khosla, *J. Am. Chem. Soc.*, 1994, **116**, 10 855.
37. R. McDaniel, S. Ebert-Khosla, D. A. Hopwood and C. Khosla, *Science*, 1993, **262**, 1546.
38. R. McDaniel, S. Ebert-Khosla, D. A. Hopwood and C. Khosla, *J. Am. Chem. Soc.*, 1993, **115**, 11 671.
39. R. McDaniel, S. Ebert-Khosla, D. A. Hopwood and C. Khosla, *Nature*, 1995, **375**, 549.
40. R. G. Summers, E. Wendt-Pienkowski, H. Motamedi and C. R. Hutchinson, *J. Bacteriol.*, 1993, **175**, 7571.
41. R. G. Summers, E. Wendt-Pienkowski, H. Motamedi and C. R. Hutchinson, *J. Bacteriol.*, 1992, **174**, 1810.
42. B. Shen, R. G. Summers, E. Wendt-Pienkowski and C. R. Hutchinson, *J. Am. Chem. Soc.*, 1995, **117**, 6811.
43. H. Fu, S. Ebert-Khosla, D. A. Hopwood and C. Khosla, *J. Am. Chem. Soc.*, 1994, **116**, 4166.
44. R. McDaniel, C. R. Hutchinson and C. Khosla, *J. Am. Chem. Soc.*, 1995, **117**, 6805.
45. H. Fu, R. McDaniel, D. A. Hopwood and C. Khosla, *Biochemistry*, 1994, **33**, 9321.
46. C. Khosla, R. McDaniel, S. Ebert-Khosla, R. Torres, D. H. Sherman, M. J. Bibb and D. A. Hopwood, *J. Bacteriol.*, 1993, **175**, 2197.
47. G. Meurer, M. Gerlitz, E. Wendt-Pienkowski, L. C. Vining, J. Rohr and C. R. Hutchinson, *Chem. Biol.*, 1997, **4**, 433.
48. G. Meurer and C. R. Hutchinson, *J. Am. Chem. Soc.*, 1995, **117**, 5899.
49. M. A. Alvarez, H. Fu, C. Khosla, D. A. Hopwood and J. E. Bailey, *Nature Biotechnol.*, 1996, **14**, 335.
50. C. R. Hutchinson, H. Decker, K. Madduri, S. L. Otten and L. Tang, *Antonie von Leeuwenhoek*, 1993, **64**, 165.
51. M. A. Fernández-Moreno, E. Martínez, J. L. Caballero, K. Ichinose, D. A. Hopwood and F. Malpartida, *J. Biol. Chem.*, 1994, **269**, 24 854.
52. T. Adamidis, P. Riggle and W. Champness, *J. Bacteriol.*, 1990, **172**, 2962.
53. W. Champness, P. Riggle and T. Adamidis, *J. Cell. Biochem.*, 1990, **14A**, 88.
54. O. H. Martínez-Costa, P. Arias, N. M. Romero, V. Parro, R. P. Mellado and F. Malpartida, *J. Biol. Chem.*, 1996, **271**, 10 627.
55. W. Champness, *J. Bacteriol.*, 1988, **170**, 1168.
56. B. K. Leskiw, E. J. Lawlor, A. J. Fernández and K. F. Chater, *Proc. Natl. Acad. Sci. USA*, 1991, **88**, 2461.
57. N. M. Romero, V. Parro, F. Malpartida and R. P. Mellado, *Nucleic Acids Res.*, 1992, **20**, 2767.
58. N. M. Romero and R. P. Mellado, *FEMS Microbiol. Lett.*, 1995, **127**, 79.
59. W. P. Revill, M. J. Bibb and D. A. Hopwood, *J. Bacteriol.*, 1995, **177**, 3946.
60. G. Meurer and C. R. Hutchinson, *J. Bacteriol.*, 1995, **177**, 477.
61. E.-S. Kim, K. D. Cramer, A. L. Shreve and D. H. Sherman, *J. Bacteriol.*, 1995, **177**, 1202.
62. C. Khosla, S. Ebert-Khosla and D. A. Hopwood, *Mol. Microbiol.*, 1992, **6**, 3237.
63. H. Decker, H. Motamedi and C. R. Hutchinson, *J. Bacteriol.*, 1993, **175**, 3876.
64. B. Shen and C. R. Hutchinson, *Science*, 1993, **262**, 1535.
65. S. Tropf, W. P. Revill, M. J. Bibb, D. A. Hopwood and M. Schweizer, *Chem. Biol.*, 1998, **5**, 135.
66. J. Crosby, D. H. Sherman, M. J. Bibb, W. P. Revill, D. A. Hopwood and T. J. Simpson, *Biochim. Biophys. Acta*, 1995, **1251**, 32.
67. R. J. Cox, T. S. Hitchman, K. J. Byrom, I. S. C. Findlow, J. A. Tanner, J. Crosby and T. J. Simpson, *FEBS Lett.*, 1997, **405**, 267.
68. M. P. Crump, J. Crosby, C. E. Dempsey, M. Murray, D. A. Hopwood and T. J. Simpson, *FEBS Lett.*, 1996, **391**, 302.
69. M. P. Crump, J. Crosby, C. E. Dempsey, J. A. Parkinson, M. Murray, D. A. Hopwood and T. J. Simpson, *Biochemistry*, 1997, **36**, 6000.
70. R. G. Summers, A. Ali, B. Shen, W. A. Wessel and C. R. Hutchinson, *Biochemistry*, 1995, **34**, 9389.
71. (a) T.J. Hitchman, J. Crosby, K.J. Byrom, R.J. Cox and T.J. Simpson, *Chem. Biol.*, 1998, **5**, 35; (b) J. H. An and Y. S. Kim, *Eur. J. Biochem.*, 1998, **257**, 395.
72. (a) A. M. Bridges, P. F. Leadlay, W. P. Revill and J. Staunton, *J. Chem. Soc., Chem. Commun.*, 1991, 776; (b) A.-L. Matharu, R. J. Cox, J. Crosby, K. J. Byrom and T. J. Simpson, *Chem. Biol.*, 1998, **5**, 699.
73. D. H. Sherman, E.-S. Kim, M. J. Bibb and D. A. Hopwood, *J. Bacteriol.*, 1992, **174**, 6184.
74. E.-S. Kim, D. A. Hopwood and D. H. Sherman, *J. Bacteriol.*, 1994, **176**, 1801.
75. C. W. Carreras and C. Khosla, *Biochemistry*, 1998, **37**, 2084.
76. C. W. Carreras, R. Pieper and C. Khosla, *J. Am. Chem. Soc.*, 1996, **118**, 5158.
77. M. J. Bibb, D. H. Sherman, S. Ōmura and D. A. Hopwood, *Gene*, 1994, **142**, 31.
78. R. J. X. Zawada and C. Khosla, *J. Biol. Chem.*, 1997, **272**, 16 184.

79 M. A. Fernández-Moreno, E. Martínez, J. L. Caballero, K. Ichinose, D. A. Hopwood and F. Malpartida, *J. Biol. Chem.*, 1994, **269**, 24854.

80 S. G. Kendrew, S. E. Harding, D. A. Hopwood and E. N. G. Marsh, *J. Biol. Chem.*, 1995, **270**, 17339.

81 S. G. Kendrew, D. A. Hopwood and E. N. G. Marsh, *J. Bacteriol.*, 1997, **179**, 4305.

82 B. Shen and C. R. Hutchinson, *Biochemistry*, 1993, **32**, 6656.

83 L. V. Bystrykh, M. A. Fernández-Moreno, J. K. Herrema, F. Malpartida, D. A. Hopwood and L. Dijkhuizen, *J. Bacteriol.*, 1996, **178**, 2238.

84 (a) D. H. Sherman, F. Malpartida, M. J. Bibb, H. M. Kieser and D. A. Hopwood, *EMBO*, 1989, **8**, 2717; (b) A. Bechthold, J. K. Sohng, T. M. Smith, X. Chu and H. G. Floss, *Mol. Gen. Genet.*, 1995, **248**, 610.

85 K. Ichinose, D. J. Bedford, D. Tornus, A. Bechthold, M. J. Bibb, W. P. Revill, H. G. Floss and D. A. Hopwood, *Chem. Biol.*, 1998, **5**, 647.

86 T.-W. Wu, M. J. Bibb, W. P. Revill and D. A. Hopwood, *J. Bacteriol.*, 1994, **176**, 2627.

87 (a) E.-S. Kim, M. J. Bibb, M. J. Butler, D. A. Hopwood and D. H. Sherman, *Gene*, 1994, **141**, 141; (b) I. S. Hunter and R. A. Hill, *Biotechnology of antibiotics*, ed. W. R. Strohl, 2nd edn., Marcel Dekker, New York, 1997, pp. 659–682.

88 A. L. Staley and K. L. Rhinehart, *J. Antibiot.*, 1994, **47**, 1425.

89 C. R. Hutchinson, *Chem. Rev.*, 1997, **97**, 2525.

90 I. Fujii and Y. Ebizuka, *Chem. Rev.*, 1997, **97**, 2511.

91 A. Yoshimoto, Y. Matsuzawa, T. Oki, T. Takeuchi and H. Umezawa, *J. Antibiot.*, 1981, **34**, 951.

92 K. Ylihonko, J. Hakala, J. Niemi, J. Lundell and P. Mäntsälä, *Microbiology.*, 1994, **140**, 1359.

93 P. L. Bartel, C.-B. Zhu, J. S. Lampel, D. C. Dosch, N. C. Connors, W. R. Strohl, J. M. Beale, Jr., and H. G. Floss, *J. Bacteriol.*, 1990, **172**, 4816.

94 T. Kunnari, J. Tuikkanen, A. Hautala, J. Hakala, K. Ylihonko and P. Mäntsälä, *J. Antibiot.*, 1997, **50**, 496.

95 J. Kantola, G. Blanco, A. Hautala, T. Kunnari, J. Hakala, C. Mendez, K. Ylihonko, P. Mäntsälä and J. Salas, *Chem. Biol.*, 1997, **4**, 751.

96 F. W. Studier, A. H. Rosenberg, J. J. Dunn and J. W. Dubendorff, *Methods Enzymol.*, 1990, **185**, 60.

97 J. Ye, M. L. Dickens, R. Plater, Y. Li, J. Lawrence and W. R. Strohl, *J. Bacteriol.*, 1994, **176**, 6270.

98 M. L. Dickens, J. Ye and W. R. Strohl, *J. Bacteriol.*, 1995, **177**, 536.

99 A. Grimm, K. Madduri, A. Ali and C. R. Hutchinson, *Gene*, 1994, **151**, 1.

100 V. B. Rajgarhia and W. R. Strohl, *J. Bacteriol.*, 1997, **179**, 2690.

101 C. Scotti and C. R. Hutchinson, *Biotechnol. Bioeng.*, 1995, **48**, 133.

102 K. Madduri and C. R. Hutchinson, *J. Bacteriol.*, 1995, **177**, 3879.

103 S. Fillipini, M. M. Solinas, U. Breme, M. B. Schlüter, D. Gabellini, G. Biamonti, A. L. Colombo and L. Garofano, *Microbiology*, 1995, **141**, 1007.

104 M. L. Dickens, J. Ye and W. R. Strohl, *J. Bacteriol.*, 1996, **178**, 3384.

105 M. L. Dickens and W. R. Strohl, *J. Bacteriol.*, 1996, **178**, 3389.

106 C. Scotti and C. R. Hutchinson, *J. Bacteriol.*, 1996, **178**, 7316.

107 S. L. Otten, M. A. Gallo, K. Madduri, X. Liu and C. R. Hutchinson, *J. Bacteriol.*, 1997, **179**, 4446.

108 S. L. Otten, X. Liu, J. Ferguson and C. R. Hutchinson, *J. Bacteriol.*, 1995, **177**, 6688.

109 M. W. Thompson, W. R. Strohl and H. G. Floss, *J. Gen. Microbiol.*, 1992, **138**, 779.

110 L. Tang, A. Grimm, Y.-X. Zhang and C. R. Hutchinson, *Mol. Microbiol.*, 1996, **22**, 801.

111 K. Madduri and C. R. Hutchinson, *J. Bacteriol.*, 1995, **177**, 1208.

112 K. J. Stutzman-Engwall, S. L. Otten and C. R. Hutchinson, *J. Bacteriol.*, 1992, **174**, 144.

113 K. Furuya and C. R. Hutchinson, *J. Bacteriol.*, 1996, **178**, 6310.

114 (a) N. Lomovskaya, S.-K. Hong, S.-U. Kim, L. Fonstein, K. Furuya and C. R. Hutchinson, *J. Bacteriol.*, 1996, **178**, 3238; (b) N. Lomovskaya, Y. Doi-Katayama, S. Fillipini, C. Nastro, L. Fonstein, M. Gallo, A. L. Colombo and C. R. Hutchinson, *J. Bacteriol.*, 1998, **180**, 2379.

115 K. Ylihonko, J. Tuikkanen, S. Jussila, L. Cong and P. Mäntsälä, *Mol. Gen. Genet.*, 1996, **251**, 113.

116 K. Ylihonko, J. Hakala, T. Kunnari and P. Mäntsälä, *Microbiol.*, 1996, **142**, 1665.

117 J. Rohr, U. Weißbach, C. Beninga, E. Künzel, K. Siems, K. U. Bindseil, F. Lombó, L. Prado, A. F. Braña, C. Méndez and J. A. Salas, *Chem. Commun.*, 1998, 437.

118 J. Niemi and P. Mäntsälä, *J. Bacteriol.*, 1995, **177**, 2942.

119 J. Rohr and R. Thiericke, *Nat. Prod. Rep.*, 1992, **9**, 103.

120 S. J. Gould, *Chem. Rev.*, 1997, **97**, 2499.

121 S. J. Gould, N. Tamayo, C. R. Melville, and M. C. Cone, *J. Am. Chem. Soc.*, 1994, **116**, 2207.

122 S. Mithani, G. Weeratunga, N. J. Taylor and G. I. Dmitrienko, *J. Am. Chem. Soc.*, 1994, **116**, 2209.

123 S. J. Gould and C. R. Melville, *Bioorg. Med. Chem. Lett.*, 1995, **5**, 51.

124 S. J. Gould, J. Chen, M. C. Cone, M. P. Gore, C. R. Melville and N. Tamayo, *J. Org. Chem.*, 1996, **61**, 5720.

125 S. J. Gould, C. R. Melville, M. C. Cone, J. Chen and J. R. Carney, *J. Org. Chem.*, 1997, **62**, 320.

126 J. R. Carney, S.-T. Hong and S. J. Gould, *Tetrahedron Lett.*, 1997, **38**, 3139.

127 (a) K. U. Bindseil, P. Hug, H. H. Peter, F. Petersen and B. E. Roggo, *J. Antibiot.*, 1995, **48**, 457; (b) S. J. Gould, S.-T. Hong and J. R. Carney, *J. Antibiot.*, 1998, **51**, 50.

128 S. W. Ayer, A. G. McInnes, P. Thibault, J. A. Walter, J. L. Doull, T. Parnell and L. C. Vining, *Tetrahedron Lett.*, 1991, **32**, 6301.

129 J. L. Douli, S. W. Ayer, A. K. Singh and P. Thibault, *J. Antibiot.*, 1993, **46**, 869.

130 M. C. Cone, A. M. Hassan, M. P. Gore, S. J. Gould, D. B. Borders and M. R. Alluri, *J. Org. Chem.*, 1994, **59**, 1923.

131 L. Han, K. Yang, E. Ramalingam, R. H. Mosher and L. C. Vining, *Microbiology*, 1994, **140**, 3379.

132 K. Yang, L. Han and L. C. Vining, *J. Bacteriol.*, 1995, **177**, 6111.

133 S. J. Gould, C. R. Melville and M. C. Cone, *J. Am. Chem. Soc.*, 1996, **118**, 9228.

134 S.-T. Hong, J. R. Carney and S. J. Gould, *J. Bacteriol.*, 1997, **179**, 470.

135 H. Decker and S. Haag, *J. Bacteriol.*, 1995, **177**, 6126.

136 M. G. Anderson, L.-Y. Khoo, R. W. Rickards, *J. Antibiot.*, 1989, **42**, 640.

137 (a) G. Udvarnoki, C. Wagner, R. Machinek and J. Rohr, *Angew. Chem., Int. Ed. Engl.*, 1995, **34**, 565; (b) W. Bao, E. Wendt-Pienkowski and C. R. Hutchinson, *Biochemistry*, 1998, **37**, 8132.

138 B. Shen and C. R. Hutchinson, *Proc. Natl. Acad. Sci. USA*, 1996, **93**, 6600.

139 H. Fu, M. A. Alvarez, C. Khosla and J. E. Bailey, *Biochemistry*, 1996, **35**, 6527.

140 H. Decker, S. Haag, G. Udvarnoki and J. Rohr, *Angew. Chem., Int. Ed. Engl.*, 1995, **34**, 1107.

141 H. Decker, J. Rohr, H. Motamedi, H. Zähner and C. R. Hutchinson, *Gene*, 1995, **166**, 121.

142 A. Montanari and J. P. Rosazza, *Tetrahedron Lett.*, 1988, **29**, 5513.

143 A. Montanari and J. P. N. Rosazza, *J. Antibiot.*, 1990, **43**, 883.

144 F. Lombó, K. Siems, A. F. Braña, C. Méndez, K. Bindseil and J. A. Salas, *J. Bacteriol.*, 1997, **179**, 3354.

145 F. Lombó, G. Blanco, E. Fernández, C. Méndez and J. A. Salas, *Gene*, 1996, **172**, 87.

146 G. Blanco, H. Fu, C. Méndez, C. Khosla and J. A. Salas, *Chem. Biol.*, 1996, **3**, 193.

147 (a) E. Künzel, S.-E. Wohlert, C. Beninga, S. Haag, H. Decker, C. R. Hutchinson, G. Blanco, C. Mendez, J. A. Salas, and J. Rohr, *Chem. Eur. J.*, 1997, **3**, 1675; (b) E. Fernández, U. Weißbach, C. S. Reillo, A. F. Braña, C. Méndez, J. Rohr and J. A. Salas, *J. Bacteriol.*, 1998, **180**, 4929; (c) L. Prado, E. Fernández, U. Weißbach, G. Blanco, L. M. Quirós, A. F. Braña, C. Méndez, J. Rohr and J. A. Salas, *Chem. Biol.*, 1999, **6**, 19.

148 (a) S. J. Gould, C. R. Melville and J. Chen, *Tetrahedron*, 1997, **53**, 4561; (b) T.-W. Wu, Y. Shen, R. McDaniel, H. G. Floss, C. Khosla, D. A. Hopwood and B. S. Moore, *J. Am. Chem. Soc.*, 1998, **120**, 7749.

149 (a) K. Saitoh, T. Furumai, T. Oki, F. Nishida, K.-I. Harada and M. Suzuki, *J. Antibiot.*, 1995, **48**, 162; (b) K. Kondo, T. Eguchi, K. Kakinuma, K. Mizoue and Y.-F. Qiao, *J. Antibiot.*, 1998, **51**, 288.

150 S. Gaisser, A. Trefzer, S. Stockert, A. Kirschning and A. Bechthold, *J. Bacteriol.*, 1997, **179**, 6271.

151 T. Hu, J. M. Curtis, J. A. Walter, J. L. McLachlan and J. L. C. Wright, *Tetrahedron Lett.*, 1995, **36**, 9273.

152 J. L. C. Wright, T. Hu, J. L. McLachlan, J. Needham and J. A. Walker, *J. Am. Chem. Soc.*, 1996, **118**, 8757.

153 J. Needham, T. Hu, J. L. MacLachan, J. A. Walter and J. L. C. Wright, *J. Chem. Soc., Chem. Commun.*, 1995, 1623.

154 T. Hu, J. M. Curtis, J. A. Walter and J. L. C. Wright, *J. Chem. Soc., Chem. Commun.*, 1995, 597.

155 J. Needham, R. J. Andersen and M. T. Kelly, *J. Chem. Soc.*, *Chem. Commun.*, 1992, 1367.

156 M. Norte, A. Padilla and J. Fernández, *Tetrahedron Lett.*, 1994, **35**, 1441.

157 G. T. Carter, J. J. Goodman, M. J. Torrey, D. B. Borders and S. J. Gould, *J. Org. Chem.*, 1989, **54**, 4321.

158 H. Bockholt, G. Udvarnoki, J. Rohr, U. Mocek, J. M. Beale and H. G. Floss, *J. Org. Chem.*, 1994, **59**, 2064.

159 J. Kobayashi, M. Takahashi and M. Ishibashi, *J. Chem. Soc., Chem. Commun.*, 1995, 1639.

160 I. Bauer, L. Maranda, K. A. Young, Y. Shimizu and S. Huang, *Tetrahedron Lett.*, 1995, **36**, 991.

161 F. Kurosaki, *Phytochemistry*, 1996, **41**, 1023.

162 F. Kurosaki, Y. Kizawa and A. Nishi, *Eur. J. Biochem.*, 1989, **185**, 85.

163 F. Kurosaki, *Archiv. Biochem. Biophys.*, 1995, **321**, 239.

164 F. Kurosaki, *Phytochemistry*, 1995, **39**, 515.

165 J. K. Stoops, P. Ross, M. J. Arslanian, K. C. Aune, S. J. Wakil and R. M. Oliver, *J. Biol. Chem.*, 1979, **254**, 7418.

166 F. Kurosaki, *FEBS Lett.*, 1996, **379**, 97.

167 F. Kurosaki, *Archiv. Biochem. Biophys.*, 1996, **328**, 213.

168 S. Tropf, B. Kächer, G. Schröder and J. Schröder, *J. Biol. Chem.*, 1995, **270**, 7922.

169 T. A. Holton and E. C. Cornish, *Plant Cell*, 1995, **7**, 1071.

170 L. Britsch, J. Dedio, H. Saedler and G. Forkmann, *Eur. J. Biochem.*, 1993, **217**, 745.

171 M. L. Durbin, G. H. Learn Jr., G. A. Huttlely and M. T. Clegg, *Proc. Natl. Acad. Sci.*, USA, 1995, **92**, 3338.

172 (a) Y. Helariutta, M. Kotilainen, P. Elomaa, N. Kalkkinen, K. Bremer, T. H. Teeri and V. A. Albert, *Proc. Natl. Acad. Sci.*, USA, 1996, **93**, 9033; (b) J. Schröder, S. Raiber, T. Berger, A. Schmidt, J. Schmidt, A. M. Soares-Sello, E. Bardshiri, D. Strack, T. J. Simpson, M. Veit and G. Schröder, *Biochemistry*, 1998, **37**, 8417.

173 (a) S. Raiber, G. Schröder and J. Schröder, *FEBS Lett.*, 1995, **361**, 299; (b) R. Preisig-Müller, R. Gehlert, F. Melchior, U. Steitz and H. Kindl, *Biochemistry*, 1997, **36**, 8349.

174 C. Beckert, C. Horn, J.-P. Schnitzler, A. Lehning, W. Heller and M. Veit, *Phytochemistry*, 1997, **44**, 275.

175 W. Horper and F.-J. Marner, *Phytochemistry.*, 1996, **41**, 451.

176 W. Bless and W. Barz, *FEBS Lett.*, 1988, **235**, 47.

177 L. Guo, R. A. Dixon and N. L. Paiva, *J. Biol. Chem.*, 1994, **269**, 22 372.

178 L. Guo and N. L. Paiva, *Archiv. Biochem. Biophys.*, 1995, **320**, 353.

179 R. Thomas, *Biochem. J.*, 1961, **78**, 807.

180 R. Thomas, *Pure Appl. Chem.*, 1979, **34**, 515.

181 D. Hölscher and B. Schneider, *Nat. Prod. Lett.*, 1995, **7**, 177.

182 D. Hölscher and B. Schneider, *J. Chem. Soc., Chem. Commun.*, 1995, 525.

183 J. Beck, S. Ripka, A. Siegner, E. Schiltz and E. Schweizer, *Eur. J. Biochem.*, 1990, **192**, 487.

184 P. M. Jordan and J. B. Spencer, *Biochem. Soc. Trans.*, 1993, **21**, 222.

185 D. J. Bedford, E. Schweizer, D. A. Hopwood and C. Khosla, *J. Bacteriol.*, 1995, **177**, 4544.

186 C. J. Child, J. B. Spencer, P. Bhogal and P. M. Shoolingin-Jordan, *Biochemistry*, 1996, **35**, 12 267.

187 C. J. Child and P. M. Shoolingin-Jordan, *Biochem. J.*, 1998, **330**, 933.

188 R. E. Minto and C. A. Townsend, *Chem. Rev.*, 1997, **97**, 2537.

189 C. P. Woloshuk and R. Prieto, *FEMS Microbiol. Lett.*, 1998, **160**, 169.

190 F. Trail, N. Mahanti and J. Linz, *Microbiol.*, 1995, **141**, 755.

191 D. W. Brown, J.-H. Yu, H. S. Kelkar, M. Fernandes, T. C. Nesbitt, N. P. Keller, T. H. Adams and T. J. Leonard, *Proc. Natl. Acad. Sci.*, USA, 1996, **93**, 1418.

192 J. Yu, P.-K. Chang, J. W. Cary, M. Wright, D. Bhatnagar, T. E. Cleveland, G. A. Payne and J. E. Linz, *Appl. Environ. Microbiol.*, 1995, **61**, 2365.

193 M. Biollaz, G. Büchi and G. Milne, *J. Am. Chem. Soc.*, 1970, **92**, 1035.

194 C. P. Gorst-Allman, K. G. R. Pachler, P. S. Steyn, P. L. Wessels and De B. Scott, *J. Chem. Soc. Perkin Trans. 1*, 1977, 2181.

195 C. A. Townsend and S. B. Christensen, *Tetrahedron*, 1983 **39**, 3575.

196 J. C. Vederas and T. T. Nakashima, *J. Chem. Soc., Chem. Commun.*, 1980, 183.

197 U. Sankawa, H. Shimada, T. Kobayashi, Y. Ebizuka, Y. Yamamoto, H. Noguchi and H. Seto, *Heterocycles*, 1982, **19**, 1053.

198 T. J. Simpson, A. E. de Jesus, P. S. Steyn and R. Vleggaar, *J. Chem. Soc., Chem. Commun.*, 1983, 338.

199 S. P. McCormick, D. Bhatnagar and L. S. Lee, *Appl. Environ. Microbiol.*, 1987, **53**, 14.

200 C. A. Townsend, S. B. Christensen and K. Trautwein, *J. Am. Chem. Soc.*, 1984, **106**, 3868.

201 I. M. Chandler and T. J. Simpson, *J. Chem. Soc., Chem. Commun.*, 1987, 17.

202 C. A. Townsend, S. W. Brobst, S. E. Ramer and J. C. Vederas, *J. Am. Chem. Soc.*, 1988, **110**, 318.

203 S. W. Brobst and C. A. Townsend, *Can. J. Chem.*, 1994, **72**, 200.

204 D. S. J. McKeown, C. McNicholas, T. J. Simpson and N. J. Willett, *Chem. Commun.*, 1996, 301.

205 S. M. McGuire and C. A. Townsend, *Bioorg. Med. Chem. Lett.*, 1993, **3**, 653.

206 C. P. Woloshuk, K. R. Foutz, J. F. Brewer, D. Bhatnagar, T. E. Cleveland and G. A. Payne, *Appl. Environ. Microbiol.*, 1994, **60**, 2408.

207 J.-H. Yu, R. A. E. Butchko, M. Fernandes, N. P. Keller, T. J. Leonard and T. H. Adams, *Curr. Genet.*, 1996, **29**, 549.

208 P.-K. Chang, K. C. Ehrlich, J. Yu, D. Bhatnagar and T. E. Cleveland, *Appl. Environ. Microbiol.*, 1995, **61**, 2372.

209 J. E. Flaherty and G. A. Payne, *Appl. Environ. Microbiol.*, 1997, **63**, 3995.

210 N. Mahanti, D. Bhatnagar, J. W. Cary, J. Joubran and J. E. Linz, *Appl. Environ. Microbiol.*, 1996, **62**, 191.

211 C. M. H. Watanabe, D. Wilson, J. Linz and C. A. Townsend, *Chem. Biol.*, 1996, **3**, 463.

212 D. W. Brown, T. H. Adams and N. P. Keller, *Proc. Natl. Acad. Sci.*, USA, 1996, **93**, 14 873.

213 G. H. Feng and T. J. Leonard, *J. Bacteriol.*, 1995, **177**, 6246.

214 M. E. Mayorga and W. E. Timberlake, *Mol. Gen. Genet.*, 1992, **235**, 205.

215 P.-K. Chang, J. W. Cary, J. Yu, D. Bhatnagar and T. E. Cleveland, *Mol. Gen. Genet.*, 1995, **248**, 270.

216 J.-H. Yu and T. J. Leonard, *J. Bacteriol.*, 1995, **177**, 4792.

217 F. Trail, P.-K. Chang, J. Cary and J. E. Linz, *Appl. Environ. Microbiol.*, 1994, **60**, 4078.

218 J. W. Cary, M. Wright, D. Bhatnagar, R. Lee and F. S. Chu, *Appl. Environ. Microbiol.*, 1996, **62**, 360.

219 J. Yu, P.-K. Chang, J. W. Cary, D. Bhatnagar and T. E. Cleveland, *Appl. Environ. Microbiol.*, 1997, **63**, 1349.

220 K. Yabe, Y. Ando and T. Hamasaki, *J. Gen. Microbiol.*, 1991, **137**, 2469.

221 K.-I. Matsushima, Y. Ando, T. Hamasaki and K. Yabe, *Appl. Environ. Microbiol.*, 1994, **60**, 2561.

222 K.-I. Kushimoto and D. P. H. Hseih, *Can. J. Microbiol.*, 1996, **42**, 804.

223 S. M. McGuire, J. C. Silva, E. G. Casillas and C. A. Townsend, *Biochemistry*, 1996, **35**, 11 470.

224 J. C. Silva, R. E. Minto, C. E. Barry III, K. A. Holland and C. A. Townsend, *J. Biol. Chem.*, 1996, **271**, 13 600.

225 J. C. Silva and C. A. Townsend, *J. Biol. Chem.*, 1996, **272**, 804.

226 H. J. Hecht, H. M. Kalisz, J. Hendle, R. D. Schmid and Schomberg, *J. Mol. Biol.*, 1993, **229**, 153.

227 H. S. Kelkar, T. W. Skloss, J. F. Haw, N. P. Keller and T. H. Adams, *J. Biol. Chem.*, 1997, **272**, 1589.

228 C. M. H. Watanabe and C. A. Townsend, *J. Org. Chem.*, 1996, **61**, 1990.

229 J. M. Schwab, W.-b. Li and L. P. Thomas, *J. Am. Chem. Soc.*, 1983, **105**, 4800.

230 C. D. Skory, P.-K. Chang, J. Cary and J. E. Linz, *Appl. Environ. Microbiol.*, 1992, **58**, 3527.

231 N. P. Keller, S. Segner, D. Bhatnagar and T. H. Adams, *Appl. Environ. Microbiol.*, 1995, **61**, 3628.

232 K. Yabe, K.-I. Matsushima, T. Koyama and T. Hamasaki, *Appl. Environ. Microbiol.*, 1998, **64**, 166.

233 J. Yu, P.-K. Chang, G. A. Payne, J. W. Cary, D. Bhatnagar and T. E. Cleveland, *Gene*, 1995, **163**, 121.

234 R. Prieto and C. P. Woloshuk, *Appl. Environ. Microbiol.*, 1997, **63**, 1661.

235 J. Fuchser, R. Thiericke and A. Zeeck, *J. Chem. Soc., Perkin Trans. 1*, 1995, 1663.

236 M. S. R. Nair, S. T. Carey and L. Ananthasbramanian, *J. Nat. Prod.*, 1982, **45**, 644.

237 J. S. E. Holker and T. J. Simpson, *J. Chem. Soc., Perkin Trans. 1*, 1981, 1397

238 R. G. Brereton, M. J. Garson and J. Staunton, *J. Chem. Soc., Perkin Trans. 1*, 1984, 1027

239 S. A. Ahmed, T. J. Simpson, J. Staunton, A. C. Sutkowski, L. A. Trimble and J. C. Vederas, *J. Chem. Soc., Chem. Commun.*, 1985, 1685.

240 J. Staunton and A. C. Sutkowski, *J. Chem. Soc., Chem. Commun.*, 1991, 1106.

241 J. Staunton and A. C. Sutkowski, *J. Chem. Soc., Chem. Commun.*, 1991, 1108.

242 J. Staunton and A. C. Sutkowski, *J. Chem. Soc., Chem. Commun.*, 1991, 1110.

243 A. Jacobs, J. Staunton and A. C. Sutkowski, *J. Chem. Soc., Chem. Commun.*, 1991, 1113.

244 A. M. Hill, A. Jacobs and J. Staunton, *J. Chem. Soc.*, Chem. Commun., 1995, 859.

245 A. M. Hill and J. Staunton, *J. Chem. Soc.*, Chem. Commun., 1995, 863.

246 A. Jacobs and J. Staunton, *J. Chem. Soc.*, Chem. Commun., 1995, 863.

247 G. Weber, K. Schörgendorfer, E. Schneider-Scherzer and E. Leitner, *Curr. Genet.*, 1994, **26**, 120.

248 A. Lawen and R. Zocher, *J. Biol. Chem.*, 1990, **265**, 11 355.

249 B. Schmidt, D. Reisner, A. Lawen and H. Kleinkauf, *FEBS Lett.*, 1992, **307**, 355.

250 H. Kobel, H. R. Loosli and R. Voges, *Experientia*, 1983, **39**, 873.

251 H. Senn, C. Weber, H. Kobel and R. Traber, *Eur. J. Biochem.*, 1991, **199**, 653.

252 M. Offenzeller, Z. Su, G. Santer, H. Moser, R. Traber, K. Memmert and E. Schneider-Scherzer, *J. Biol. Chem.*, 1993, **268**, 26 127.

253 M. Offenzeller, G. Santer, K. Totschnig, Z. Su, H. Moser, R. Traber and E. Schneider-Scherzer, *Biochemistry*, 1996, **35**, 8401.

254 A. W. Alberts, J. Chen, G. Kuron, V. Hunt, J. Huff, C. Hoffman, J. Rothrock, M. Lopez, H. Joshua, E. Harris, A. Patchett, R. Monaghan, S. Currie, E. Stapley, G. Albers-Schonberg, O. Hensens, J. Hirshfield, K. Hoogsteen, J. Liesch and J. Springer, *Proc. Natl. Acad. Sci.*, USA, 1980, **77**, 3957.

255 A. Endo and K. Hasumi, *Nat. Prod. Rep.*, 1993, **28**, 541.

256 M. D. Greenspan and J. B. Yudkovitz, *J. Bacteriol.*, 1985, **162**, 704.

257 J. K. Chan, R. N. Moore, T. T. Nakashima and J. C. Vederas, *J. Am. Chem. Soc.*, 1983, **105**, 3334.

258 R. N. Moore, G. Bigam, J. K. Chan, A. M. Hogg, T. T. Nakashima and J. C. Vederas, *J. Am. Chem. Soc.*, 1985, **107**, 3694.

259 A. Endo, Y. Negishi, T. Iwashita, K. Mizukawa and M. Hirama, *J. Antibiot.*, 1985, **38**, 444

260 D. Komagata, H. Shimada, S. Murakawa and A. Endo, *J. Antibiot.*, 1989, **42**, 407.

261 T. Nakamura, D. Komagata, S. Murakawa, K. Sakai and A. Endo, *J. Antibiot.*, 1990, **43**, 1597.

262 K. Kimura, D. Komagata, S. Murakawa and A. Endo, *J. Antibiot.*, 1990, **43**, 1621.

263 Y. Yoshizawa, D. J. Witter, Y. Liu and J. C. Vederas, *J. Am. Chem. Soc.*, 1994, **116**, 2693.

264 J. C. Vederas, *J. Am. Chem. Soc.*, 1980, **102**, 374.

265 K. Wagschal, Y. Yoshizawa, D. J. Witter, Y. Liu and J. C. Vederas, *J. Chem. Soc.*, Perkin Trans. 1, 1996, 2357.

266 S. Laschat, *Angew. Chem.*, Int. Ed. Engl., 1996, **35**, 289.

267 D. J. Witter and J. C. Vederas, *J. Org. Chem.*, 1996, **61**, 2613.

268 Y. Kubo, H. Nakamura, K. Kobayashi, T. Okuno and I. Furusawa, *Mol. Plant-Microbe Interact.*, 1991, **5**, 440.

269 Y. Takano, Y. Kubo, K. Shimizu, K. Mise, T. Okuno and I. Furusawa, *Mol. Gen. Genet.*, 1995, **249**, 162.

270 N. Kimura and T. Tsuge, *J. Bacteriol.*, 1993, **175**, 4427.

271 H. Oikawa, T. Yokota, T. Abe, A. Ichihara, S. Sakamura, Y. Yoshizawa and J. C. Vederas, *J. Chem. Soc.*, Chem. Commun., 1989, 1282.

272 H. Oikawa, T. Yokota, A. Ichihara and S. Sakamura, *J. Chem. Soc.*, Chem. Commun., 1989, 1284.

273 H. Oikawa, K. Katayama, Y. Suzuki and A. Ichihara, *J. Chem. Soc.*, Chem. Commun., 1995, 1321.

274 H. Oikawa, Y. Suzuki, A. Naya, K. Katayama and A. Ichihara, *J. Am. Chem. Soc.*, 1994, **116**, 3605.

275 Y. S. Tsantrizos, F. Zhou, P. Famili and X. Yang, *J. Org. Chem.*, 1995, **60**, 6922.

276 E. Saepudin and P. H. M. Harrison, *Can. J. Chem.*, **73**, 1.

277 H. Kumagai, H. Tomoda and S. Omura, *J. Antibiot.*, 1992, **45**, 563.

278 B. G. Turgeon, M. Kodama, G. Yang, M. S. Rose, S. W. Lu and O. C. Yoder, *Can. J. Bot.*, 1995, **73(Supp)**, S1071.

279 G. Yang, M. S. Rose, B. G. Turgeon and O. C. Yoder, *Plant Cell*, 1996, **8**, 2139.

280 M. Barinaga, *Science*, 1996, **272**, 1261.

281 A. K. Amegadzie, W. A. Ayer and L. Sigler, *Can. J. Chem.*, 1995, **73**, 2119.

282 A. I. Scott, H. Guilford and E. Lee, *J. Am. Chem. Soc.*, 1971, **93**, 3534.

283 R. Bentley, *J. Biol. Chem.*, 1963, **238**, 1895.

284 A. I. Scott and K. J. Weisner, *J. Chem. Soc.*, Chem. Commun., 1972, 1075.

285 M. C. O'Sullivan and J. M. Schwab, *Bioorg. Chem.*, 1995, **23**, 131.

286 K. Nitta, Y. Yamamoto, T. Inoue and T. Hyodo, *Chem. Pharm. Bull.*, 1966, **14**, 363.

287 J. Sonnenbichler, I. Sonnenbichler and D. Schwarz, *Phytochem.*, 1997, **44**, 267.

288 J. D. Bergstrom, C. Dufresne, G. F. Bills, M. Nallin-Omstead and K. Bryne, *Annu. Rev. Microbiol.*, 1995, **49**, 607.

289 J. Breinholt, S. Damtoft, L. B. Frederiksen, L. B. L. Hansen and S. R. Jensen, *Phytochem.*, 1995, **39**, 1359.

290 N. C. J. E. Chesters and D. O'Hagan, *J. Chem. Soc.*, Perkin Trans. 1, 1997, 827.

291 H. Noguchi, P. H. Harrison, K. Arai, T. Nakashima, L. A. Trimble and J. C. Vederas, *J. Am. Chem. Soc.*, 1988, **110**, 2938.

292 H. Thomas, *J. Pharm. Soc. Jpn.*, 1940, **60**, 318.

293 H. Culceth, J. Fuchser, S. J. Moss, J. Nieschalk and D. O'Hagan, *Tetrahedron Lett.*, 1998, **39**, 1949.

294 (a) P. H. M. Harrison, D. W. Hughes and R. W. Riddoch, *Chem. Commun.*, 1998, 273; (b) P. Duspasa, S. I. Jenkins, D. W. Hughes and P. H. M. Harrison, *Chem. Commun.*, 1998, 2643.

295 J. Breinholt, G. W. Jenson, A. Kjaer, C. E. Olsen and C. N. Rosendahl, *Acta Chem. Scand.*, 1997, **51**, 855.

296 P. A. Cohen and G. H. N. Towers, *Phytochem.*, 1996, **42**, 1325.

297 M. T. Davies-Colman and M. J. Garson, *Nat. Prod. Rep.*, 1998, **15**, 477.

298 H. J. Bestmann, E. Übler and B. Hölldobler, *Angew. Chem.*, Int. Ed. Engl., 1997, **36**, 395.

299 R. G. Kerr, J. Lawry and K. A. Gush, *Tetrahedron Lett.*, 1996, **37**, 8305.

300 E. I. Graziani and R. J. Andersen, *Chem. Commun.*, 1996, 2377.

301 J. Kubanek and R. J. Andersen, *Tetrahedron Lett.*, 1997, **38**, 6327.
