
JOURNAL OF BACTERIOLOGY, Sept. 1999, p. 5409–5413  
0021-9193/99/$04.00+0  
Copyright © 1999, American Society for Microbiology. All Rights Reserved.

**Two Different Types of Dehalogenases, LinA and LinB, Involved in γ-Hexachlorocyclohexane Degradation in *Sphingomonas paucimobilis* UT26 Are Localized in the Periplasmic Space without Molecular Processing**

YUJI NAGATA,* AKIKO FUTAMURA, KEISUKE MIYAUCHI, AND MASAMICHI TAKAGI  
Department of Biotechnology, The University of Tokyo, Bunkyo-ku, Tokyo 113-8657, Japan  

Received 8 March 1999/Accepted 11 June 1999

γ-Hexachlorocyclohexane (γ-HCH) is one of several highly chlorinated insecticides that cause serious environmental problems. The cellular proteins of a γ-HCH-degrading bacterium, *Sphingomonas paucimobilis* UT26, were fractionated into periplasmic, cytosolic, and membrane fractions after osmotic shock. Most of two different types of dehalogenase, LinA (γ-hexachlorocyclohexane dehydrochlorinase) and LinB (1,3,4,6-tetrachloro-1,4-cyclohexadiene halidohydrolase), that are involved in the early steps of γ-HCH degradation in UT26 was detected in the periplasmic fraction and had not undertaken molecular processing. Furthermore, immunoelectron microscopy clearly showed that LinA and LinB are periplasmic proteins. LinA and LinB both lack a typical signal sequence for export, so they may be secreted into the periplasmic space via a hitherto unknown mechanism.

---

γ-Hexachlorocyclohexane (γ-HCH; also called γ-BHC and lindane) is a highly halogenated organic insecticide that has been used worldwide. Due to its toxicity and long persistence in soil, most countries have prohibited the use of γ-HCH. However, many contaminated sites remain throughout the world. Moreover, some countries are presently using γ-HCH, mainly for economic reasons, so new sites are continually being contaminated (3, 14). *Sphingomonas* (formerly *Pseudomonas*) *paucimobilis* UT26 is a unique microorganism that utilizes γ-HCH as its sole source of carbon and energy under aerobic conditions (11). We have cloned four genes (*linA*, *linB*, *linC*, and *linD*), the products of which are involved sequentially in the degradation of γ-HCH in UT26 (12, 15, 17, 18) (Fig. 1). Three of them encode different types of dehalogenases: dehydrochlorinase, halidohydrolase, and reductive dehalogenase. Among them, *linA* encodes an enzyme that catalyzes a unique

![Chemical Reaction Diagram](attachment:chemical-reaction-diagram.png)

FIG. 1. Proposed degradation pathway of γ-HCH in *S. paucimobilis* UT26. Compounds: 1, γ-HCH (also called γ-BHC and lindane); 2, γ-pentachlorocyclohexene; 3, 1,3,4,6-tetrachloro-1,4-cyclohexadiene; 4, 1,2,4-trichlorobenzene; 5, 2,4,5-trichloro-2,5-cyclohexadiene-1-ol; 6, 2,5-dichlorophenol; 7, 2,5-dichloro-2,5-cyclohexadiene-1,4-diol; 8, 2,5-dichlorohydroquinone; 9, chlorohydroquinone; 10, hydroquinone.

* Corresponding author. Mailing address: Department of Biotechnology, The University of Tokyo, Yayoi 1-1-1, Bunkyo-ku, Tokyo 113-8657, Japan. Phone: 81-3-5841-5178. Fax: 81-3-5841-8015. E-mail: aynaga@hongo.ecc.u-tokyo.ac.jp.

TABLE 1. Cellular location of enzymic activities in *S. paucimobilis* UT26

| Cell fraction | Protein (mg) | Phosphatase (pH 6.0) $(10^3 \mathrm{U})$ | Glycerdehyde-3-phosphate dehydrogenase $(10^4 \mathrm{U})$ | Pyruvate dehydrogenase $(10^4 \mathrm{U})$ | LinA (dehydrochlorinase) (U) | LinB (halidohydrolase) (U) |
|---------------|--------------|-----------------------------------------|-------------------------------------------------------|------------------------------------------|--------------------------------|--------------------------------|
| Periplasmic   | $9.1(15.2)$  | $1.2(34)$                              | $0.040(1.3)$                                           | $0.24(15)$                               | $3.4(71)$                       | $0.59(83)$                     |
| Cytoplasmic   | $40.6(67.8)$ | $2.3(66)$                              | $3.1(98)$                                              | $1.3(82)$                                | $1.4(29)$                       | $0.12(17)$                     |
| Membrane      | $10.2(17)$   | ND$^a$                                 | $0.022(0.7)$                                           | $0.043(3)$                               | ND                             | ND                             |

$^a$ ND, not detectable.

dehydrochlorination reaction whose mechanism is still unknown. Dehalogenation is a key step in the degradation of halogenated compounds, so many enzymes that catalyze dehalogenation have been reported (7), and yet there is little or no information available about the distribution of dehalogenase in gram-negative bacteria.

Here we report that two different types of dehalogenase, LinA ($\gamma$-hexachlorocyclohexane dehydrochlorinase) (12, 16) and LinB (1,3,4,6-tetrachloro-1,4-cyclohexadiene halidohydrolase) (17, 20), that act on $\gamma$-HCH or its metabolite in *S. paucimobilis* are localized in the periplasmic space. As far as we know, this is the first report on the subcellular localization of dehalogenases that are involved in the degradation of halogenated xenobiotics in gram-negative bacteria.

MATERIALS AND METHODS

Subcellular fractionation of *S. paucimobilis*. UT26 was cultured in 200 ml of 1/3 Luria broth (3.3 g of Bacto tryptone, 1.7 g of yeast extract, and 5 g of sodium chloride per liter). Cells were harvested in the exponential growth phase. The periplasmic fraction was prepared by osmotic shock (21). After osmotic shock, cells were disrupted by sonication (Sonifier 250; Branson, Danbury, Conn.). After centrifugation at $12,000 \times g$ for 10 min, the pellet was discarded as debris. The supernatant was separated by centrifugation at $100,000 \times g$ for 1 h into cytoplasmic and membrane fractions. The protein concentration was determined by using the protein assay kit (Bio-Rad Laboratories, Richmond, Calif.), with bovine serum albumin as a standard.

Enzyme assays. Activities of glycerdehyde-3-phosphate dehydrogenase, pyruvate dehydrogenase, and phosphatase were measured as described by Duggleby and Dennis (6), Friedemann and Haugen (8) and Hayakawa et al. (9), and Dassa et al. (5), respectively. The activities of LinA and LinB were measured spectrophotometrically at 460 nm with mercuric thiocyanate and ferric ammonium sulfate by the method of Iwasaki et al. (13) as described in previous studies (16, 20). One unit of enzyme activity of LinA and LinB was defined as the amount of enzyme required for the release of $1 \mu \mathrm{mol}$ of chloride ion per min.

Western blot analysis. Antibodies were raised against LinA and LinB by using purified enzymes produced in *Escherichia coli* (16, 20). Rabbits were injected subcutaneously four times at weekly intervals with $0.5 \mathrm{mg}$ of purified LinA or LinB. The injections were given with $50 \%$ (vol/vol) Freund's complete adjuvant. Sera were collected after four injections. Samples were separated by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) and transferred to the nitrocellulose membrane Hybond C (Amersham). The ECL (enhanced chemiluminescence) Western blotting system (Amersham) was used for detection.

Immunogold-labeling electron microscopy. Cells were fixed in $2.5 \%$ glutaraldehyde in phosphate-buffered saline (PBS) for 3 h, dehydrated in graded ethanol and propylene oxide, and embedded in epoxy resins. Ultrathin sections were prepared by an ultramicrotome (LKB) and deposited on nickel grids. The grid was incubated in PBS containing $5 \%$ bovine serum albumin for 30 min and then in PBS containing the purified anti-LinA or anti-LinB immunoglobulin G for $30 \mathrm{~min}$ at room temperature. After pretreatment with $1 \% \mathrm{H}_2 \mathrm{O}_2$ for $3 \mathrm{~min}$, the grid was incubated in PBS containing $3 \%$ goat anti-rabbit immunoglobulin G conjugated to $10-\mathrm{nm}$-diameter gold particles (Amersham) for $30 \mathrm{~min}$ at room temperature. The sections were stained with $4 \%$ uranyl acetate and $0.4 \%$ lead citrate in $0.1 \mathrm{MNaOH}$. The preparation was examined with electron microscopes (JEM-1200EX and HU-800).

Determination of N-terminal amino acid sequences. The periplasmic proteins were fractionated by SDS-PAGE, transferred to a polyvinylidene difluoride-type membrane (Immobilon-PSQ; Millipore Corp.), and stained with Coomassie blue. The bands corresponding to LinA and LinB were excised and directly sequenced by automated Edman degradation with a model 477A protein sequencer equipped with a model 120A phenylthiohydantoin analyzer (Applied Biosystems).

VOL. 181, 1999 LOCALIZATION OF DEHALOGENASES FOR γ-HCH DEGRADATION 5411

FIG. 3. Immunogold-labeling electron microscopy. (a) Wild-type strain, UT26, with anti-LinA antibody. (b) UT26 with anti-LinB antibody. (c) The spontaneous *linA* deletion mutant, YO5, with anti-LinA antibody. (d) YO5 with anti-LinB antibody.

---

**RESULTS**

Subcellular fractionation of *S. paucimobilis* UT26. UT26, a nalidixic acid-resistant mutant of the first *S. paucimobilis* strain isolated, was cultured without an inducer, because the genes involved in the early steps of γ-HCH degradation are constitutively expressed in UT26 (19). Cells were harvested in the exponential growth phase, and the supernatant was saved and used as the extracellular fraction. The cellular proteins of UT26 were fractionated into periplasmic, cytosolic, and membrane fractions after osmotic shock. SDS-PAGE analysis showed a characteristic pattern of bands for each fraction (Fig. 2a). Table 1 shows a result typical of several independent experiments, because all of them showed the same tendency. The distributions of the cellular protein content across the periplasmic, cytoplasmic, and membrane fractions were 15.2, 67.8, and 17.0%, respectively (Table 1). Glyceraldehyde-3-phosphate dehydrogenase and pyruvate dehydrogenase, which are cytoplasmic enzymes, were mainly detected in the cytoplasmic fraction (98 and 82%, respectively) (Table 1), whereas only 34% of the phosphatase activity, which we expected to be

TABLE 2. Distribution of gold particles in immunogold-labeling electron microscopy

| Strain       | Antibody     | No. (%) of particles distributed among cells | Area used for counting (μm²) |
|--------------|--------------|--------------------------------------------|-----------------------------|
|              |              | Inside                                    | Periphery                   | Outside                    | Total                      |                           |
| UT26         | Anti-LinA    | 184 (24)                                  | 441 (58)                    | 152 (20)                   | 777 (100)                  | 61                          |
| UT26         | Anti-LinB    | 111 (17)                                  | 353 (53)                    | 203 (30)                   | 667 (100)                  | 54                          |
| YO 5 (∆linA) | Anti-LinA    | 16 (29)                                   | 5 (9)                       | 34 (62)                    | 55 (100)                   | 23                          |
| YO 5 (∆linA) | Anti-LinB    | 26 (12)                                   | 99 (46)                     | 88 (41)                    | 213 (100)                  | 32                          |

ACKNOWLEDGMENTS

We thank S. Yamashita of The University of Tokyo, Tokyo, Japan, for technical help with immunoelectron microscopy. We thank K. Ohgi and her colleagues of the Hoshi College of Pharmacy, Tokyo, Japan, for determining the N-terminal amino acid sequences.

This work was supported in part by a Grant-in-Aid for Scientific Research from the Ministry of Education, Science, Culture, and Sports of Japan. This work was performed with the facilities of the Biotechnology Research Center, The University of Tokyo.

REFERENCES

1. Ben-Bassat, A., K. Bauer, S.-Y. Chang, K. Myambo, A. Boosman, and S. Chang. 1987. Processing of the initiation methionine from proteins: properties of the Escherichia coli methionine aminopeptidase and its gene structure. J. Bacteriol. 169:751–757.
2. Beveridge, T. J. 1995. The periplasmic space and the periplasm in gram-positive and gram-negative bacteria. ASM News 61:125–130.
3. Blais, J. M., D. W. Schindler, D. C. G. Muir, L. E. Kimpe, D. B. Donald, and B. Rosenberg. 1998. Accumulation of persistent organochlorine compounds in mountains of western Canada. Nature 395:585–588.
4. Brurberg, M. B., V. G. H. Eijsink, A. J. Haandrikman, G. Venema, and I. F. Nes. 1995. Chitinase B from Serratia marcescens BJL200 is exported to the periplasm without processing. Microbiology 141:123–131.
5. Dassa, E., C. Tesu, and P.-L. Boquet. 1980. Identification of the acid phosphatase (optimum pH 2.5) of Escherichia coli. FEBS Lett. 113:275–278.
6. Duggleby, R. G., and D. T. Dennis. 1974. Nicotinamide adenine dinucleotide-specific glyceraldehyde 3-phosphate dehydrogenase from Pisum sativum. J. Biol. Chem. 249:162–166.
7. Fetzner, S., and F. Lingens. 1994. Bacterial dehalogenases: biochemistry, genetics, and biotechnological applications. Microbiol. Rev. 58:641–685.
8. Friedemann, T. E., and G. E. Haugen. 1943. Pyruvic acid 2. The determination of keto acids in blood and urine. J. Biol. Chem. 47:415–442.
9. Hayakawa, T., M. Hirashima, S. Ide, M. Hamada, K. Okabe, and M. Koike. 1966. Mammalian α-keto acid dehydrogenase complexes. J. Biol. Chem. 241:4694–4699.
10. Hirel, P., J. Schmitter, P. Dessen, G. Fayat, and S. Blanquet. 1989. Extent of N-terminal methionine excision from Escherichia coli proteins is governed by the side-chain length of the penultimate amino acid. Proc. Natl. Acad. Sci. USA 86:8247–8251.
11. Imai, R., Y. Nagata, K. Senoo, H. Wada, M. Fukuda, M. Takagi, and K. Yano. 1989. Dehydrochlorination of γ-hexachlorocyclohexane (γ-BHC) by γ-BHC-assimilating Pseudomonas paucimobilis. Agric. Biol. Chem. 53:2015–2017.
12. Imai, R., Y. Nagata, M. Fukuda, M. Takagi, and K. Yano. 1991. Molecular cloning of a Pseudomonas paucimobilis gene encoding a 17-kilodalton polypeptide that eliminates HCl molecules from γ-hexachlorocyclohexane. J. Bacteriol. 173:6811–6819.
13. Iwasaki, I., S. Utsumi, and T. Ozawa. 1952. New colorimetric determination of chloride using mercuric thiocyanate and ferric ion. Bull. Chem. Soc. Jpn. 25:226.
14. Iwata, H., S. Tanabe, N. Sakai, and R. Tatsukawa. 1993. Distribution of persistent organochlorines in the oceanic air and surface seawater and the role of ocean on their global transport and fate. Environ. Sci. Technol. 27:1080–1098.
15. Miyauchi, K., S.-K. Suh, Y. Nagata, and M. Takagi. 1998. Cloning and sequencing of a 2,5-dichlorohydroquinone reductive dehalogenase gene whose product is involved in degradation of γ-hexachlorocyclohexane by Sphingomonas paucimobilis. J. Bacteriol. 180:1354–1359.
16. Nagata, Y., T. Hatta, R. Imai, K. Kimbara, M. Fukuda, K. Yano, and M. Takagi. 1993. Purification and characterization of γ-hexachlorocyclohexane (γ-HCH) dehydrochlorinase (LinA) from Pseudomonas paucimobilis. Biosci. Biotechnol. Biochem. 57:1582–1583.
17. Nagata, Y., T. Nariya, R. Ohtomo, M. Fukuda, K. Yano, and M. Takagi. 1993. Cloning and sequencing of a dehalogenase gene encoding an enzyme with hydrolase activity involved in the degradation of γ-hexachlorocyclohexane in Pseudomonas paucimobilis. J. Bacteriol. 175:6403–6410.
18. Nagata, Y., R. Ohtomo, K. Miyauchi, M. Fukuda, K. Yano, and M. Takagi. 1994. Cloning and sequencing of a 2,5-dichloro-2,5-cyclohexadiene-1,4-diol dehydrogenase gene involved in the degradation of γ-hexachlorocyclohexane in Pseudomonas paucimobilis. J. Bacteriol. 176:3117–3125.
19. Nagata, Y., M. Fukuda, K. Miyauchi, and M. Takagi. 1996. Genes and enzymes for γ-hexachlorocyclohexane degradation in Sphingomonas paucimobilis UT26, p. 58–70. In T. Nakazawa, K. Furukawa, D. Haas, and S. Silver (ed.), Molecular biology of pseudomonads. American Society for Microbiology, Washington, D.C.
20. Nagata, Y., K. Miyauchi, J. Damborsky, K. Manova, A. Ansorgova, and M. Takagi. 1997. Purification and characterization of haloalkane dehalogenase of a new substrate class from a γ-hexachlorocyclohexane-degrading bacterium, Sphingomonas paucimobilis UT26. Appl. Environ. Microbiol. 63:3707–3710.
21. Nossal, N., and L. Heppel. 1966. The release of enzymes by osmotic shock from Escherichia coli in exponential phase. J. Biol. Chem. 241:3055–3062.
22. Oliver, D. B. 1987. Periplasm and protein secretion, p. 56–69. In F. C. Neidhardt, J. L. Ingraham, K. B. Low, B. Magasanik, M. Schaechter, and H. E. Umbarger (ed.), Escherichia coli and Salmonella typhimurium: cellular and molecular biology, vol. 1. American Society for Microbiology, Washington, D.C.
23. Oliver, D. B. 1996. Periplasm, p. 88–103. In F. C. Neidhardt, R. Curtiss III, J. L. Ingraham, E. C. C. Lin, K. B. Low, B. Magasanik, W. S. Reznikoff, M. Riley, M. Schaechter, and H. E. Umbarger (ed.), Escherichia coli and Salmonella: cellular and molecular biology, 2nd ed., vol. 1. American Society for Microbiology, Washington, D.C.
24. Pancholi, V., and V. A. Fischetti. 1992. A major surface protein on group A streptococci is a glyceraldehyde-3-phosphate-dehydrogenase with multiple binding activity. J. Exp. Med. 176:415–426.
25. Pancholi, V., and V. A. Fischetti. 1998. α-Enolase, a novel strong plasmin(ogen) binding protein on the surface of pathogenic streptococci. J. Biol. Chem. 273:14503–14515.
26. Salmond, G. P. C., and P. J. Reeves. 1993. Membrane traffic wardens and protein secretion in gram-negative bacteria. Trends Biochem. Sci. 18:7–12.
