
X-ray Crystal Structures of 70S Ribosome Functional Complexes

Jamie H. Cate,$^{1*†‡}$ Marat M. Yusupov,$^{1*‡}$ Gulnara Zh. Yusupova,$^{1}$ Thomas N. Earnest,$^{2}$ Harry F. Noller$^{1*}$

Structures of 70S ribosome complexes containing messenger RNA and transfer RNA (tRNA), or tRNA analogs, have been solved by x-ray crystallography at up to 7.8 angstrom resolution. Many details of the interactions between tRNA and the ribosome, and of the packing arrangements of ribosomal RNA (rRNA) helices in and between the ribosomal subunits, can be seen. Numerous contacts are made between the 30S subunit and the P-tRNA anticodon stem-loop; in contrast, the anticodon region of A-tRNA is much more exposed. A complex network of molecular interactions suggestive of a functional relay is centered around the long penultimate stem of 16S rRNA at the subunit interface, including interactions involving the "switch" helix and decoding site of 16S rRNA, and RNA bridges from the 50S subunit.

Translation of the RNA-encoded genetic message into the polypeptide chain of a protein links genotype to phenotype. It is carried out by the ribosome, an ancient ribonucleoprotein particle whose structural core and fundamental mechanism of action are conserved among all forms of life (1, 2). The smallest and best-studied examples are bacterial ribosomes, which have a molecular size of ~2.5 MD and are made up of a small (30S) and a large (50S) subunit. The 30S subunit is composed of 16S rRNA [~1500 nucleotides (nt)] and about 20 different proteins, whereas the large subunit contains 23S rRNA (~2900 nt), 5S rRNA (120 nt), and more than 30 different proteins. This degree of structural complexity is in keeping with that of its biological role.

The substrate of the ribosome is tRNA, which is commonly considered to bind to the ribosome at three different sites: A, P, and E (aminoacyl, peptidyl, and exit, respectively) (3, 4). Each tRNA binding site is partitioned between the two ribosomal subunits, resulting in as many as six different sites of interaction between tRNA and the ribosome. The anticodon ends of the tRNAs bind to the 30S subunit, which also carries messenger RNA (mRNA); the 3′-acceptor, or CCA ends of the tRNAs interact with the 50S subunit, which contains the catalytic site for peptide bond formation, peptidyl transferase (5). Thus, the complete *T. thermophilus* 70S ribosome, containing mRNA and tRNA or tRNA analogs, and the solution of their structures by x-ray crystallography at up to 7.8 Å resolution. Many specific features of the rRNA can now be identified, and in many instances, elements of protein structure are also recognizable. The interactions of tRNA with the ribosome in the A, P, and E sites are seen in the greatest detail so far obtained, providing new insights into the mechanism of translation.

### Structure Determination

Ribosome purification and crystallization were carried out as previously described, with some modifications (19). Ribosome complexes were formed with an mRNA fragment whose first three codons specifying Met, Phe, and Lys, were used to direct binding of tRNAs or tRNA anticodon stem-loops (ASLs) to the ribosome. Three different complexes yielded crystals used to determine the structures presented here. The first complex was designed to contain mRNA and a 6–base pair (bp) ASL fragment of tRNA<sup>Phe</sup> (ASL<sup>Phe</sup>) bound in the ribosomal P site (Table 1, data sets ASL1, ASL2, and λ1 to λ4). The second complex was formed with the mRNA, ASL<sup>Phe</sup>, plus deacylated tRNA<sup>Lys</sup> bound to the A site (Table 1, A site). The third complex was designed to contain the mRNA and a full-length tRNA<sub>f</sub><sup>Met</sup> bound to the P site (Table 1, P site).

Phases for the structure factor amplitudes were initially determined to 25 Å by molecular replacement with an electron density map derived from cryo-electron microscopy (EM) single-particle reconstructions of vacant *Escherichia coli* ribosomes (13). Briefly, an ensemble of scatterers with single-Gaussian form factors was fit to the EM-derived electron density positioned in a large triclinic unit cell (20). This pseudo-atom model was then used in molecular replacement searches. To confirm the molecular replacement solution, two ribosome complexes, one containing ASL<sup>Phe</sup> in the P site and the other containing ASL<sup>Phe</sup> in the P site and tRNA<sup>Lys</sup> in the A site, were constructed. A difference Fourier map derived from amplitudes measured from the two complexes shows density with the L shape of tRNA in a location predicted for A-site tRNA (Fig. 1A) (21). This result, as well as the absence of significant negative density in the difference map, illustrates the quality of the low-resolution phases.

To extend the resolution of the electron density map, phases derived from the molecular replacement solution were used in difference Fourier analyses to locate heavy-atom cluster compounds bound to the ribosome. The compound Ta<sub>6</sub>Br<sub>12</sub><sup>2+</sup> (22) provided phase information from a multiwavelength anomalous dispersion (MAD) experiment to 11.2 Å resolution (data not shown). This MAD experiment yielded phase information

---

<sup>1</sup>Center for Molecular Biology of RNA, Sinsheimer Laboratories, University of California, Santa Cruz, CA 95064, USA. <sup>2</sup>Macromolecular Crystallography Facility, Advanced Light Source, Physical Biosciences Division, Lawrence Berkeley National Laboratory, Berkeley, CA 94720, USA.

*To whom correspondence should be addressed. Email: cate@wi.mit.edu (J.H.C), marat@darwin.ucsc.edu (M.M.Y.), and harry@nuvolari.ucsc.edu (H.F.N.)

†Present address: Whitehead Institute for Biomedical Research and Massachusetts Institute of Technology, Cambridge, MA 02142, USA.

‡These authors contributed equally to this work.

of sufficient quality to locate the positions of many single heavy-atom sites in difference Fourier maps. For the structures described below, an iridium (III) hexamine derivative was used in a four-wavelength MAD experiment to phase the structure factors of the 70S ribosome-mRNA-ASL complex to 7.8 Å resolution (Table 1, λ1 to λ4). These phases were then improved by density modification procedures (20, 23).

As with the molecular replacement solution, the quality of the heavy-atom phasing was assessed by means of a difference map (Table 1, ASL2 and P site; ASL2 limits the resolution of the map to 8.8 Å). Comparison of diffraction data from ribosome complexes containing either intact tRNA<sub>ε</sub><sup>Met</sup> or an ASL in the ribosomal P site yielded positive density for tRNA minus its ASL at the predicted location of the ribosomal P site (Fig. 1B). Comparison of the observed difference map and an 8.8 Å electron density map calculated from the high-resolution crystal structure of tRNA<sup>Phe</sup> (24) shows striking similarity in detail, including clearly defined helical major and minor grooves and the single-stranded 3′ end, indicating the quality of the MAD phases (Fig. 1C).

### Overall Structure of the 70S Ribosome

The *T. thermophilus* ribosome has maximum dimensions of about 210 Å in each direction. Viewed from the back of the 30S subunit, general features of the ribosome observed in EM reconstructions are visible (Fig. 2A) (13, 25). The electron density of the platform and body of the 30S subunit merge near the middle of the subunit. As also seen in EM reconstructions, the head of the 30S subunit connects to the body through a “neck” near the platform and makes close contact to the shoulder of the body on the right side. A pronounced gap between the head and shoulder observed in some 30S subunit EM reconstructions (26, 27) suggests that this contact is stabilized by formation of 70S ribosomes. At least two additional contacts of lower electron density connect the head to the platform, one on the solvent side of the 30S subunit near the neck (marked b, Fig. 2A), and another from the side of the head to the tip of the platform (marked a, Fig. 2A). At the present resolution, the spur protruding from the lower right of the body of the 30S subunit (Fig. 2A) can clearly be identified as an RNA helix.

The interface cavity between the ribosomal subunits contains the binding sites for tRNA, as well as for elongation factors EF-Tu and EF-G (Fig. 2B) (15). The floor of the cavity is formed by many bridges joining the body and platform of the 30S subunit to the front of the 50S subunit. Above the cavity, additional bridges connect the head of the 30S subunit to the central protuberance and A-site finger of the 50S subunit (Fig. 2B). The three-pointed crown appearance of the

50S subunit first observed in classical EM studies (28) is readily apparent, both from the 50S solvent side view of the ribosome (Fig. 2C) and from the interface view of the isolated 50S subunit extracted from the 70S ribosome electron density map (Fig. 3A). In both the 9 Å 50S subunit (17) and the 7.8 Å 70S ribosome x-ray structures, many long rods of high electron density corresponding to helices of rRNA crisscross the 50S subunit. However, detailed comparison with the classical EM images and with more recent cryo-EM reconstructions as well as the 9 Å resolution x-ray structure (17, 29) reveals some significant differences. While the central protuberance has a similar appearance, both L1 and L7/L12 features vary among the different structures. In the EM images, the L7/L12 stalk is often more extensive and elongated, while this extra mass is absent in both the 50S and 70S x-ray maps. The (L7/L12)<sub>4</sub> tetramer is believed to function as a flexible element of the ribosome based on biochemical studies (30), NMR spectroscopy (31), and cryo-EM reconstructions (14, 32). Therefore, the apparent absence of L7/L12 in the x-ray structures may be due to local disorder. The L1 protuberance has a similar appearance in cryo-EM structures [for example, (33)] and the x-ray map of the 70S ribosome but is diminished in the x-ray structure of the isolated 50S subunit (17, 29).

#### Subunit Interface

The lower resolution bridges seen in EM reconstructions (34) can now be resolved as discrete contacts between individual molecular components of the two subunits (Fig. 3). Several lines of evidence have suggested that the subunit interface is RNA-rich (35–37). In the 70S ribosome electron density map, the bridges involve both RNA-RNA and RNA-protein interactions. Interestingly, in the central core of the interface (bridges B2a to B2c, B3, B5, and B7), all of the elements contributed by the small subunit appear to be composed mainly, if not exclusively, of RNA. Most if not all of their partners in the large subunit also appear to contain RNA elements (bridges B2a, B2c, B5, and B7), although

**Fig. 1. Controls for the quality of structure factor phasing.**

(A) Difference electron density map (A site – ASL1) with molecular replacement phases to 25 Å, contoured at 4σ (green) and −4σ (red). The difference density is superimposed on a 25 Å electron density map combining x-ray diffraction amplitudes and EM molecular replacement phases. (B) Stereo view of difference map (P site – ASL2) with MAD phasing to 8.8 Å resolution. The map, limited to 8.8 Å resolution by the native ASL data set (Table 1, data set ASL2), is contoured at 4σ (light blue) and −4σ (red). (C) Electron density map to 8.8 Å resolution calculated from the crystal structure of tRNA<sup>Phe</sup> (24). Because the ASL (gray) is common to both ribosome complexes, it is not seen in the difference map in (B).

some are more difficult to interpret with confidence at the present resolution. In contrast, the bridges at the periphery of the interface (Bla, B1b, B4, and B6) all seem to involve protein-RNA interactions. One of these interactions (B4) involves a bridge between the 715 helix in domain II of 23S RNA and protein S15 at the base of the small subunit platform (38). Another peripheral bridge, the A-site finger (Bla), is formed by a contact between an RNA helix from the top of the 50S subunit and a protein component from

the head of the 30S subunit. Bridge B1b also appears to involve interaction between an α-helical protein from the head of the 30S subunit and an RNA element in the central protuberance of the 50S subunit. Finally, at the bottom of the interface, a presumed protein from the 50S subunit reaches across to contact an RNA feature of the 30S subunit (bridge B6).

Four elements of 16S rRNA that contact the 50S subunit can be identified in the electron density map. Bridges B2b and B7 are part of the

platform of the small subunit and contain the 790 and 690 helices in 16S rRNA, respectively (Fig. 3B), whose positions are constrained by numerous biochemical and biophysical studies (39). Based on protection from hydroxyl radicals (36), both helices make minor-groove contacts to the 50S subunit, consistent with the electron density. The third element (B2c) can be assigned to the 900 loop in 16S rRNA, which originates at the center of the 30S subunit near the junction of the three major domains of 16S rRNA (39). The position and orientation of the 900 loop are further constrained by directed hydroxyl radical probing from Fe(II) tethered to protein S15, a component of the neighboring bridge B4 (38, 40) (Fig. 3). Protection of the 900 tetraloop from hydroxyl radicals by the 50S subunit (36) (Fig. 4A) supports the assignment of this feature to the 30S component of bridge B2c, which has the shape of an RNA tetraloop (41) in the electron density. On the 30S side, the tetraloop seems to contact the minor groove of the 790 helix in 16S rRNA, while on its tip, the tetraloop contacts an irregular feature of the 50S subunit. On the basis of its proximity to the bottom of protein S5 (42) and its protection by 30S subunits, this 50S feature may correspond to the noncanonical RNA helix around position 1700 of 23S rRNA (37).

The most striking feature of the subunit interface involves an ~100 Å RNA helix that runs vertically along the length of the 30S subunit body (Figs. 3B and 4B). This helix contacts the large subunit about once per helical turn along its minor groove (bridges B2a, B3, and B5), packing against the body of the small subunit on its opposite face (Fig. 4B). This pattern matches protections of the 16S rRNA penultimate stem from hydroxyl radicals when free 30S subunits associate with the large ribosomal subunit (36) (Fig. 4C). Bridge B2a connects the top of the penultimate stem to an RNA element of the 50S subunit (Fig. 3A). Results of 16S-23S rRNA cross-linking experiments (43) and directed hydroxyl radical probing studies from ASL “helical rulers” (44) suggest that this RNA element is the 30S-protected 1910 helix in domain IV of 23S rRNA (Fig. 3A). Finally, bridge B6 corresponds to density for presumed 50S subunit proteins that contact major-groove and noncanonical regions near the bottom of the penultimate stem, as noted above. The penultimate stem, therefore, contributes to about half of the interface contacts between the subunits.

The positioning of the penultimate stem in the electron density is exceptionally well constrained by numerous biochemical probing experiments. Its rotational orientation is fixed by close agreement between hydroxyl radical protection studies (36) and its observed contacts with the 50S subunit and packing against the body of the 30S subunit in the crystal structure (Fig. 4). Its orientation is

![Figure 2](https://www.science.org/cms/asset/0f8e2d0f-3f4a-4f8e-bf8e-2d0f3f4a4f8e/figure2.png)

**Fig. 2. Electron density of the 70S ribosome.** (A) Stereo view of the 7.8 Å electron density map from the solvent side of the 30S subunit (purple). The 30S subunit consists of the head (H) connected to the platform (P) and body (B). Additional features of the small subunit are the neck (N), spur (SP), shoulder (S), and contacts between the head and platform (a and b). The 50S subunit (gray) has the prominent features of the protein L1 stalk, central protuberance (CP), and L7/L12 region. (B) View of the ribosome rotated 90° about the vertical, with the L7/L12 region in the foreground. The A-site finger is marked with an asterisk. (C) View of the ribosome rotated 180° about the vertical axis compared with (A).

confirmed by directed probing experiments with hydroxyl radicals generated from tethered Fe(II); probes tethered to protein S15 in the base of the 30S platform (40) and to nucleotide 361 in the body of the small subunit (45) each target nucleotides in the penultimate stem facing their respective locations (Fig. 4). Nucleotides around the internal loop at positions 1434 to 1438 are protected from hydroxyl radicals by the binding to 16S rRNA of protein S20 (46), which has been localized to the bottom of the 30S subunit (47). In the electron density map, the bottom of the body of the 30S subunit contacts the penultimate stem at the predicted location of this internal loop (Fig. 4). Iridium hexamine is bound to the major groove of the penultimate stem at four sites predicted to contain noncanonical helical interactions, consistent with the RNA binding specificity previously observed for the chemically analogous osmium hexammine (48) (Fig. 4). Toward the top of the penultimate stem, phylogenetic covariation of nt 1417 and 1418 and nt 1482 and 1483 (49) and the reactivity patterns of their bases and riboses (36) suggest that they form tandem sheared A-G base pairs, consistent with the unusual shape of their electron density. Their predicted position places their minor groove surface in contact with bridge B3, explaining the protection of the N1 positions of A1418 and A1483, and the riboses of all four nucleotides, by 50S subunits (36). The bridge interactions may be similar to those seen in the J4/5 region of a group I intron (50) upon docking of its substrate duplex (51). A final test of the register of the penultimate stem is the fitting of solution NMR structures of a model RNA encompassing the “decoding site” region of 16S rRNA (nt 1404 to 1412, 1488 to 1497) (9, 52) to the electron density. Two conspicuous features of the NMR solution structure are an approximate 30° bend centered on the tRNA-protected nt 1492 and 1493 (53, 54), which results in a noticeably narrowed major groove; both features are observed at precisely the predicted positions of these nucleotides in the 70S ribosome crystal structure (Fig. 4). Electron density maps calculated from the NMR structures to 7.8 Å resolution contain a hole through the helix at the positions of A1492 and A1493; a single hole appears in the observed electron density of the penultimate stem at the predicted locations of these nucleotides. It is noteworthy that the x-ray map closely resembles the NMR structure of the paromomycin-RNA complex but not that of the drug-free RNA (9, 52). Finally, the electron density at the position of the 5′ strand (nt 1404) leads directly into the P site of the 30S subunit where C1400 has been photochemically cross-linked to the wobble base in the anticodon of tRNA (55), as described below.

Each of the interface contacts, taken individually, buries only a modest surface area from solvent. We estimate the total surface area buried to be close to 2100 Å²; the largest single area (B2a) covers about 600 Å² (56). By comparison, the P456 domain of the *T. thermophila* group I intron buries 2700 Å² of surface area (50, 57). Each of its three helical packing elements contributes about 900 Å². The COOH-terminal domain of ribosomal protein L11 excludes 1700 to 1900 Å² from solvent when it binds to 23S rRNA (11, 12, 57). Considering the size of the ribosome, the relatively small surface area involved in intersubunit contacts is consistent with the delicate balance between subunit association and dissociation observed in vitro (58).

### Positions of tRNA in the Ribosome

Three different tRNA-ribosome structures were used to determine the positions of tRNA bound to the ribosome. First, tRNA bound to the P site was positioned by combining the P-site difference map (Fig. 1B) with the ASL seen directly in the 7.8 Å electron density map (Fig. 5). The A-site tRNA was initially found in the 25 Å

![Figure 3](#fig3)

**Fig. 3. Interface between ribosomal subunits.** (A) Stereo view of large ribosomal subunit interface. Regions of contact between ribosomal subunits are surrounded by thick black lines. (B) Stereo view of small ribosomal subunit interface, marked as in (A). The bridges are the mirror image of those in (A). (C) The labels for the interface contacts are derived from bridges (B1 to B6) seen in EM reconstructions (34), with the addition of B7. Bridge B6 is shifted relative to its position in the EM studies (34).

A-site difference map described earlier (Fig. 1A). When the P-site difference map is contoured to reveal low-occupancy tRNA binding, additional density for A-site and E-site tRNA appears. Moreover, E-site tRNA, which was copurified with the ribosomes in a substoichiometric amount, is also seen in the ASL-containing complex as weaker electron density. Thus, the positions of each of the three tRNAs (or ASL) are confirmed independently in two different ribosome complexes (Fig. 5).

A model of three tRNAs bound to the ribosome is presented in Fig. 5C, in which calculated 7.8 Å maps of tRNA were positioned in the experimentally determined ribosome density. The tRNAs in the A and P sites are nearly parallel to one another, while the acceptor end of the E-site tRNA diverges toward the base of

**Fig. 4. RNA elements in the subunit interface.** (A) The 900 stem-loop (light blue) in 16S rRNA at the subunit interface (B2c). (B) Stereo view of the penultimate stem (light blue) of 16S rRNA packed between the body of the 30S subunit (purple) and the 50S subunit (gray). A possible protein contact from the 50S subunit to the penultimate stem is marked with an asterisk. (C) Biochemical constraints for positioning of the penultimate stem. (Left) Secondary structure of 16S rRNA. (Right) Minor-groove elements of the penultimate stem are protected from hydroxyl radicals in the 30S subunit (gray) or upon 50S subunit association (black) (36). Larger circles indicate stronger protection. Arrows mark positions cleaved in directed probing experiments from protein S15 (open) or 16S rRNA nucleotide 361 (filled). Asterisks denote the predicted locations of bound iridium hexamine ions. Numbering is for *E. coli* 16S rRNA. Data for nt 1402 to 1406, 1443 to 1462, 1469, and 1494 are not available. (D) Superposition of the biochemical constraints on a ribbon model approximating the helical register of the penultimate stem. Hydroxyl radical protections due to the 30S subunit are shown in light blue, those from the 50S subunit in gray. The positions of bound iridium hexamine ions (green spheres), protein S15 (40) and nt 361 (45) directed probing cleavages (red and yellow, respectively), and protections resulting from the binding of protein S20 to 16S rRNA (blue) (46) are also shown.

the L1 ridge (compare with Fig. 2A). The angle between the planes of the A- and P-site tRNAs is reduced compared with that inferred from EM reconstructions (15). Although the A-site tRNA seen in the higher-resolution P-site difference is bound in the presence of a noncognate codon (tRNAfMet with a UUU Phe codon in the A position), the low-resolution A-site difference map shows a cognate tRNA bound in a similar orientation, suggesting that the general position of the noncognate A-site tRNA is unlikely to be spurious. Given the experimentally determined position of the P tRNA 3′ end (Figs. 1B and 5), a parallel orientation for the A-site tRNA would still allow its aminoacyl end to reach the 3′ end of the P-site tRNA to enable the peptidyl transfer reaction. The 3′ CCA tRNA tail has been shown to take on different conformations depending on its structural context (59).

Transfer RNA–Ribosome Interactions

The single-stranded 3′ end of tRNAfMet appears clearly in the P-site difference map (Fig. 1B). The final base pair of its acceptor stem appears to be unpaired, as expected for bacterial initiator tRNA (60) (compare Fig. 1B with 1C). Nucleotide C74 in the universally conserved CCA end of tRNA has been shown to base pair with the invariant G2252 in 23S rRNA (61). This tRNA-rRNA base pair probably exists in the crystal because the single-stranded tail of the P-site tRNA is visible and is particularly strong in the predicted position of C74. However, the difference density does not seem large enough to

Details of the interactions between the ribosome, mRNA, and P-site ASL can be seen in the x-ray structure. The ASL component of tRNA<sup>phe</sup> and its P-site codon can be readily identified in the electron density (Fig. 5, E and F). In the P site, the ribosome grips the tRNA anticodon and mRNA with six fingers of electron density, three of which (a to c) form a molecular jig to position the anticodon stem. Finger (a) binds the tRNA backbone around positions 29 and 30, while finger (b) interacts with the tRNA across its minor groove near positions 31 and 41. On the opposite side of the stem, finger (c) contacts the tRNA backbone near positions 38 and 39. Finger (d) contacts the backbone of the anticodon loop near nucleotide 34, the wobble nucleotide, while finger (e) contacts the minor groove face of the mRNA at the first and second positions of the P-site codon. Fingers (d) and (e) grip the codon-anticodon duplex from opposite directions, as if to clamp the two strands of the triplet duplex together. Feature (f) has a very different appearance from that of fingers (a) to (e), merging with the density of the codon-anticodon helix near its end, indicating a possible stacking interaction with the wobble pair. The well-known photochemical cross-link between C1400 in 16S rRNA and the wobble base in the anticodon of P-site tRNA (55) suggests that feature (f) contains nucleotides from the 1400 region of 16S rRNA. In addition, the observed P-site ASL-dependent protection of the N7 position of G1401 from dimethyl sulfate (53, 54) could be due to interaction (f). The nearby location of the penultimate stem, which begins with nt 1404, is consistent with the predicted location of C1400 and G1401 in this feature. The extensive set of interactions observed in the electron density explains the minimal accessibility of the ASL backbone to hydroxyl radicals when in the small-subunit P site (62) and the ability of the ribosome to bind tRNA to the P site in the absence of mRNA (53, 64). The ribosomal components of fingers (a) to (e) have not yet been identified, but N1 positions of guanine residues identified in footprinting and modification-interference experiments (53, 54, 65) are potential candidates to make hydrogen bonds to phosphate oxygens or 2′-hydroxyl groups in the ASL and mRNA backbones.

These findings suggest that binding of tRNA by the 30S subunit has three consequences. First, the orientation of the anticodon end of peptidyl-tRNA is fixed by clamping its anticodon stem with fingers (a) to (c). Second, fingers (d) and (e) help stabilize codon-anticodon pairing, and feature (f) may provide further stability by extending the stacking of the triplet duplex (66). Third, feature (f) may also help to orient the mRNA for proper positioning of the A-site codon. Most importantly, all of these interactions appear to involve sequence-independent contacts to the ribosome, such that all tRNAs and

account for A76, suggesting that the very 3′ terminus of the deacylated tRNA<sub>f</sub><sup>Met</sup> is disordered. When the difference map is superimposed on the ribosome electron density, the 3′ end of the tRNA is pinched between two features of the 50S subunit which direct it toward an opening that leads to the presumed polypeptide exit tunnel (Fig. 5D). Although both parts of the pincer appear to be made of RNA, attempts to obtain a close fit of the solution structure of the 23S rRNA P loop (10), which contains G2252, to the electron density have so far been unsuccessful.

their respective codons can be bound to the 30S subunit P site in the same way.

In contrast to the P site, the A site in the 30S subunit is cavernous (Fig. 3B). The molecular surface of the tRNA-mRNA complex modeled in the A site appears to make only weak contacts with the ribosome density (Fig. 5G), at the back wall of the binding pocket, the P-site proximal side wall, and bridge B2a, which runs under the D stems of the A and P tRNAs. The openness of the A site is in keeping with its role in translation; whereas the P site grips the tRNA tightly and maintains the translational reading frame, the A site discriminates between cognate and noncognate tRNAs to exploit subtle differences in the free energy of codon-anticodon pairing (67–69). Interestingly, the maximal opening of the A site is oriented not in the direction of stably bound A-site tRNA, but toward the direction of L7/L12, matching the direction of incoming aminoacyl tRNA introduced by EF-Tu (14) (Fig. 5G).

In the absence of bound tRNA, the A-site codon is disordered in the electron density, suggesting that there are minimal, if any, A-site contacts between mRNA and the ribosome in the absence of A-site tRNA. It may be necessary to have some degree of flexibility in the position of the A-site codon to allow reorientation of the aminoacyl tRNA after it is released from EF-Tu, to maintain codon-anticodon pairing. On the basis of our modeling of A-site tRNA, codon-anticodon pairing would direct the 3′ end of the mRNA through the tunnel between the head and body, as predicted from EM reconstructions (13, 34) (compare Fig. 5G and Fig. 2A). The 16S rRNA bases most strongly implicated in A-site tRNA binding are the universally conserved nt A1492 and A1493 (I), whose N1 positions are protected by the binding of A-site tRNA (53, 54). However, their N1 positions are more than 15 Å from the mRNA-tRNA complex and point in the opposite direction toward the 50S subunit near bridge B2a, strongly suggesting that their protection by tRNA is indirect.

E-site tRNA contacts the ribosome in three locations. First, its 3′-CCA end is fused with the 50S density in the cleft between the L1 stalk and central protuberance (Fig. 5C), consistent with previous studies that have shown that the most important determinants of E-site binding reside in the deacylated A76 of tRNA (70, 71). A second interaction occurs between protein L1 and E-tRNA at the predicted position of its T loop. The anticodon loop of the E-site tRNA binds in the cleft between the platform and head of the 30S subunit, adjacent to that of P-site tRNA (Fig. 5, C and D). Codon-anticodon pairing in the E site would require movement of the anticodon end of E-tRNA by ~7 Å toward the P site. Whether E-site tRNA interacts with mRNA remains an open question, since the structures presented here involve noncognate E-tRNA.

**Table 1. X-ray diffraction data and heavy-atom phasing.** The ribosome crystals grew in space group I422 with cell dimensions of approximately \(a = b = 508\) Å and \(c = 803\) Å. For data collection, crystals were transferred into cryoprotectants containing 20 to 25% 2-methyl-2,4-pentanediol and flash-cooled in liquid propane. Crystals were held in a nitrogen gas stream at 105 K during measurements to minimize crystal decay in the x-ray beam. Data were measured with three different x-ray sources: beamline 5.0.2 at the Advanced Light Source (ALS) with an ADSC Quantum 4 CCD-based detector, beamline 9-1 at the Stanford Synchrotron Light Source with a MAR 345 image plate detector, and a Rigaku rotating anode with an R-Axis II detector at UCSC. Most of the data, including all of the MAD data, were collected at the ALS. All data were reduced to intensities and standard deviations with the programs Denzo and Scalepack (80). Further data analysis was performed with the CCP4 suite of programs, CNS, the mask and map manipulation programs MAMA and MAPMAN, and the graphics program O (20, 81). Heavy-atom refinement was carried out with the program MLPHARE (82), with slight modifications to the source code (83). For MAD phasing, the high-energy remote wavelength (\(\lambda 3\)) served as reference, with ASL2 added as an additional derivative.

| Data set | ASL1 | A site | ASL2 | P site |
|----------|------|--------|------|--------|
| High-resolution limit (Å) | 15.7 | 18. | 8.8 | 7.0 |
| \(R_{\text{sym}}\),* % (outer shell) | 4.3 (44.0) | 7.6 (32.7) | 8.4 (56.3) | 5.2 (38.0) |
| Mean \(I / \sigma(I)\) (outer shell) | 34.2 (3.2) | 27.2 (5.2) | 17.7 (2.6) | 27 (3.4) |
| Number of reflections |  |  |  |  |
| Unique | 8,515 | 4,034 | 41,520 | 81,043 |
| Observational redundancy | 4.4 | 4.8 | 5.9 | 3.9 |
| Completeness, % (outer shell) | 96.2 (97.7) | 76.3 (74.2) | 99.8 (99.1) | 97.4 (94.2) |

| MAD data set (iridium hexammine) | \(\lambda 1\) (1.1051 Å) | \(\lambda 2\) (1.1055 Å) | \(\lambda 3\) (1.0764 Å) | \(\lambda 4\) (1.1047 Å) |
|-----------------------------------|-------------------------|-------------------------|-------------------------|-------------------------|
| High-resolution limit (Å) | 7.5 | 7.5 | 7.5 | 7.5 |
| \(R_{\text{sym}}\),* % (at 7.8 Å) | 7.3 (39.5) | 7.2 (40.3) | 8.9 (46.3) | 7.9 (52.3) |
| Mean \(I / \sigma(I)\) (at 7.8 Å) | 16.1 (3.4) | 14.8 (3.0) | 15.8 (3.1) | 13.9 (2.4) |
| Number of reflections |  |  |  |  |
| Unique | 121,730 | 118,423 | 124,437 | 119,051 |
| Observational redundancy | 4.0 | 3.4 | 4.4 | 3.4 |
| Completeness, % (at 7.8 Å) | 95.9 (95.4) | 93.1 (91.5) | 97.7 (97.7) | 93.4 (92.2) |
| \(\chi^2\), unmerged† | 1.3 | 1.2 | 1.4 | 1.2 |
| \(\chi^2\), anomalous signal | 8.6 | 2.8 | 3.0 | 3.0 |
| \(\chi^2\), dispersive signal \(\lambda 1\) | – | 3.5 | 5.7 | 12.1* |
| \(\lambda 2\) | – | – | 4.1 | 7.1 |
| \(\lambda 3\) | – | – | – | 4.0 |
| Resolution (Å) Phasing ⟨FOM⟩‡ |  |  |  |  |
| 51.5 | 0.17 |  |  |  |
| 28.0 | 0.48 |  |  |  |
| 19.2 | 0.63 |  |  |  |
| 14.6 | 0.70 |  |  |  |
| 11.8 | 0.66 |  |  |  |
| 9.9 | 0.62 |  |  |  |
| 8.5 | 0.50 |  |  |  |
| 7.5 | 0.32 |  |  |  |
| Phasing ⟨FOM⟩‡ after density modification (at 7.8 Å): |  |  |  | 0.92 (0.80) |

*\(R_{\text{sym}} = \sum |I - \langle I \rangle| / \Sigma I\).
†\(\chi^2\) analyses taken from Scalepack (80). Data were originally scaled with \(\langle I+ \rangle\) and \(\langle I- \rangle\) kept separate (unmerged). Anomalous signal calculated after merging \(\langle I+ \rangle\) and \(\langle I- \rangle\). Dispersive signal calculated by comparing \(\langle I \rangle\) across wavelengths.
‡Mean figure of merit, or mean cosine of the phase error, given as a function of resolution.

Ribosome Structure and Movement

In contrast to early views of the ribosome as a passive surface upon which the process of translation occurs, there is increasing evidence that the ribosome is an RNA-based machine that itself carries out the fundamental steps of translation (7). Most striking is the demonstration that the ribosome is able to translocate tRNAs in the absence of elongation factors and GTP (6). Biochemical and biophysical studies provide evidence that the anticodon and acceptor ends of tRNA bind and move independently with respect to the small and large subunits (72). These observations and the juxtaposition of RNA elements implicated in tRNA binding and subunit association imply that these two apparently independent functions of the ribosome are somehow coupled. Direct evidence for a functional connection between tRNA binding and subunit association comes from the accessibility to chemical probes of “class III” sites in 16S rRNA, nucleotides that are protected by binding of either tRNA or 50S subunits (53, 73). Interestingly, these same nucleotides are also protected by the binding of antibiotics

that increase the error frequency of translation, thus linking tRNA binding, subunit association, and translational accuracy.

A framework for understanding the ways in which these three diverse functions are linked begins to emerge from the x-ray structure. Following the electron density from the predicted location of the 900 loop in 16S rRNA, the RNA strand containing the class III nt A909 is found to contact the penultimate stem at the predicted location of two additional class III nucleotides (A1413 and G1487; Figs. 6A and 4). These three residues are precisely the subset of class III sites specifically protected by the binding of A-site tRNA or streptomycin, as well as by 50S subunits (36, 53, 73). Thus, the protection of these three nucleotides can be explained by the reversible interaction of the 909 strand with the penultimate stem in a way that is stabilized by all three types of ligand. The x-ray structure illustrates how each of these ligands may exert such an effect. On the interface side, the 50S subunit packs against bridges B2a, B2c, and B3, which involve the penultimate stem and 900 loop. Both the large subunit and P-site tRNA contact the platform, which in turn packs against the 900 loop. Finally, the aminoglycoside antibiotics bind to the top of the penultimate stem beneath the A-site tRNA, and streptomycin footprints nucleotides in the 909 strand (9, 73).

Recent genetic studies (74) demonstrated the first example of rRNA movement directly tied to ribosome function, in which alternate pairing of a 3-bp helix in 16S rRNA correlates with translational accuracy. The “switch” helix, encompassing nt 885 to 890 and nt 910 to 912, neighbors class III nt A909, again pointing to the intimate relation between the three ribosome functions described above and further connecting them to the dynamics of ribosomal RNA. The switch helix is located at the center of the same network of RNA helical packing

interactions described above for the class III nucleotides (Fig. 6A). The electron density between the switch helix and the 900 loop (Fig. 6B) has the characteristic appearance of an S-turn (75, 76) as recently predicted from comparative sequence analysis (77). An S-turn for the 5′ strand of the switch region implies that the crystal structure contains the ram, or error-prone, conformation of the switch helix (nt 885 to 887, 910 to 912 pairing); nt 888 to 890, involved in the restrictive pairing mode, are part of the S-turn motif (77). The conformational switch must exert its influence on the accuracy of tRNA selection indirectly, because the switch helix is more than 15 Å from the closest approach of the A-site tRNA or codon. Instead, both the 5′ and 3′ strands of the switch helix make minor-groove contacts with the penultimate stem around positions 1489 and 1413, respectively. The tRNA-protected bases A1492 and A1493 in the penultimate stem, which are also out of contact range with both the A-site tRNA and mRNA, are very near the switch helix. The single position of close approach between the decoding region and A-site tRNA is the 16S rRNA backbone around positions 1494 and 1495. These nucleotides are within ~2 to 4 Å of the predicted position of the tRNA backbone near positions 37 and 38, directly adjacent to the A-site anticodon. It is noteworthy that cleavage of 16S rRNA between nt 1493 and 1494 by Colicin E3 causes ribosome inactivation (78).

These observations suggest that the ribosome may exert its influence on the binding of tRNA to the A site via a molecular relay mediated by the switch helix and its neighbors. Rearrangement from the restrictive to the ram conformational state, possibly triggered by a signal from the 50S subunit, would result in formation of the S-turn and the contacts between the switch helix and penultimate stem observed in the crystal structure. The increase in reactivity of A908 as a function of the error frequency of different mutant ribosomes (79) could be a manifestation of this change in RNA packing. Rearrangement of interactions between the switch helix and penultimate stem may modulate the conformation or orientation (or both) of the penultimate stem, influencing the position of the 1494-1495 strand. A further conformational change could then account for the observed tRNA-dependent protections of A1492 and A1493 and the enhanced reactivity of A892 (54), upon contact between the respective strands of the 16S rRNA decoding site and the tRNA anticodon loop. This relay mechanism would link events occurring in the 30S subunit decoding site with those taking place in the 50S subunit. Moreover, such a link could be bidirectional, providing a signal to the 50S subunit concerning the fit of codon-anticodon pairing in the 30S subunit; conversely, it could signal events in the 50S subunit, such as the release of EF-Tu or peptide bond formation, to modulate interactions between tRNA and the 30S subunit.

Note added in proof: After submission of this article, two papers describing the structures of the *T. thermophilus* 30S ribosomal subunit at 5.5 Å resolution (84) and the *Haloarcula marismortui* 50S ribosomal subunit at 5 Å resolution (85) were published. Although many of the details of the rRNA and ribosomal protein components are more clearly resolved in the subunit structures, some features seen in the 70S ribosome structure, such as protein L1 in the 50S subunit and part of the head of the 30S subunit, appear to be absent in the subunit maps, possibly because of local disorder that is not present in the 70S ribosome crystals. In addition, certain features of the 30S subunit, such as the orientation of the head and platform, differ between the isolated subunit and 70S ribosome.

Fig. 6. The switch helix in the 30S subunit. (A) Packing of the penultimate stem (green) between the 50S subunit (gray), the switch helix (light blue), and the remainder of the 30S subunit (purple), with A-site tRNA (red) and mRNA (dark purple) above. The approximate locations of bases A1492 and A1493 (a), class III nt 909, 1413, and 1487 (b), and the 1494 strand (c) are shown. (B) View of the packing interactions from the platform side of the A-site tRNA. A ribbon representation of the 900 stem-loop region of 16S rRNA containing the S-turn is shown in yellow, with the switch helix in orange.

References and Notes

1. C. R. Woese et al., Microbiol. Rev. 47, 621 (1983).
2. W. E. Hill et al., Eds., The Ribosome: Structure, Function, and Evolution (American Society for Microbiology, Washington, DC, 1990).
3. J. D. Watson, Bull. Soc. Chim. Biol. 46, 1399 (1964).
4. H. J. Rheinberger, H. Sternbach, K. H. Nierhaus, Proc. Natl. Acad. Sci. U.S.A. 78, 5310 (1981).
5. R. E. Monro, J. Mol. Biol. 26, 147 (1967).
6. S. Pestka, J. Biol. Chem. 244, 1533 (1969); L. P. Gavrilova and A. S. Spirin, Mol. Biol. (USSR) 6, 248 (1972).
7. R. Green and H. F. Noller, Annu. Rev. Biochem. 66, 679 (1997).
8. V. Ramakrishnan and S. W. White, Trends Biochem. Sci. 23, 208 (1998); P. B. Moore, Annu. Rev. Biophys. Biomol. Struct. 27, 35 (1998); S. V. Nikonov et al., Biol. Chem. 379, 795 (1988); A. A. Szewczak and P. B. Moore, J. Mol. Biol. 247, 81 (1995); A. Dallas and P. B. Moore, Structure 5, 1639 (1997); C. C. Correll, B. Freeborn, P. B. Moore, T. A. Steitz, Cell 91, 705 (1997).
9. D. Fourmy et al., Science 274, 1367 (1996).
10. E. V. Puglisi et al., Nature Struct. Biol. 4, 775 (1997).
11. G. L. Conn et al., Science 284, 1171 (1999).
12. B. T. Wimberly et al., Cell 97, 491 (1999).
13. J. Frank et al., Nature 376, 441 (1995).
14. H. Stark et al., ibid. 389, 403 (1997).
15. Reviewed in R. K. Agrawal and J. Frank, Curr. Opin. Struct. Biol. 9, 215 (1999).
16. A. Yonath et al., Biochem. Int. 1, 428 (1980); K. von Bohlen et al., J. Mol. Biol. 222, 11 (1991).
17. N. Ban et al., Cell 93, 1105 (1998).
18. S. D. Trakhanov et al., FEBS Lett. 220, 319 (1987); S. D. Trakhanov et al., J. Mol. Biol. 209, 327 (1989); H. A. Hansen et al., Biochim. Biophys. Acta 1050, 1 (1990); G. Yusupova et al., FEBS Lett. 290, 69 (1991); M. M. Yusupov et al., Biochimie 73, 887 (1991).
19. Ribosomes were prepared from *T. thermophilus* HB8 lysate as described [Z. Gogia, M. Yusupov, T. Spirina, Mol. Biol. (USSR) 20, 519 (1986)]. A 36-nt phage T4 gene 32 mRNA fragment with a modified Shine-Dalgarno sequence was chemically synthesized (Dharmacon). tRNA<sup>Met</sup> and tRNA<sup>Lys</sup> were isolated from *E. coli* (Subriden). An ASL of tRNA<sup>Phe</sup> (19 nucleotides, ASL<sup>Phe</sup>) was synthesized by in vitro transcription with T7 polymerase. The RNAs were purified on denaturing gels, eluted, and precipitated with ethanol before use. To form a P-site ribosome complex, the shortened gene 32 mRNA and ASL<sup>Phe</sup> or tRNA<sup>Met</sup> were incubated at 37°C for 30 min in a solution containing 20 mM MgCl<sub>2</sub>, 100 mM KCl, and 20 mM tris HCl (pH 7.4) before ribosome crystallization [M. M. Yusupov et al., Dokl. Akad. Nauk (USSR) 292, 1271 (1987); G. Zh. Yusupova and M. M. Yusupov, unpublished data]. All ligands were present in a 1.1- to 1.5-fold stoichiometric excess over the concentration of ribosomes to form the complexes before crystallization. Crystals of *T. thermophilus* 70S ribosome complexes were grown by the vapor diffusion method in hanging and sitting drops. Crystals grew to maximum dimensions of 0.5 mm by 0.5 mm by 0.25 mm. Heavy-atom derivatives were prepared by soaking crystals in solutions containing heavy-atom compound for 1 to 5 days. To obtain a ribosome complex with ASL in the P site and tRNA in the A site, crystals of the P-site complex (ribosome-mRNA-ASL<sup>Phe</sup>) were soaked in a solution containing tRNA<sup>Lys</sup> (2 μM, Sigma) for 48 hours at room temperature.
20. A. T. Brünger et al., Acta Crystallogr. D Biol. Crystallogr. 54, 905 (1998).
21. R. K. Agrawal et al., Science 271, 1000 (1996); H. Stark et al., Cell 88, 19 (1997); J. Wower et al., Biochim. Biophys. Acta 1050, 38 (1990); H. F. Noller et al., in (2), pp. 73–92.
22. G. Schneider and Y. Lindqvist, Acta Crystallogr. D Biol. Crystallogr. 50, 186 (1994).
23. J. P. Abrahams and A. G. W. Leslie, ibid. 54, 30 (1996).
24. B. Hingerty, R. S. Brown, A. Jack, J. Mol. Biol. 124, 523 (1978).
25. H. Stark et al., Structure 3, 815 (1995).
26. K. R. Lata et al., J. Mol. Biol. 262, 43 (1996).
27. I. S. Gabashvili et al., ibid. 286, 1285 (1999).
28. J. A. Lake, ibid. 105, 131 (1976); B. Kastner, M. Stöffler-Meilicke, G. Stöffler, Proc. Natl. Acad. Sci. U.S.A. 78, 6552 (1981); V. D. Vasiliev et al., J. Mol. Biol. 171, 561 (1983).

29. G. Harauz, E. Boekema, M. van Heel, Methods Enzymol. 164, 35 (1988); M. Radermacher, T. Wagenknecht, A. Verschoor, J. Frank, EMBO J. 6, 1107 (1987).
30. R. R. Traut et al., in Structure, Function, and Genetics of Ribosomes, B. Hardesty and G. Kramer, Eds. (Springer-Verlag, New York, 1986); A. T. Gudkov, FEBS Lett. 407, 253 (1997).
31. A. T. Gudkov et al., FEBS Lett. 138, 229 (1982); C. A. Cowgill et al., J. Biol. Chem. 259, 15257 (1984).
32. R. K. Agrawal et al., Nature Struct. Biol. 6, 643 (1999).
33. A. Malhotra et al., J. Mol. Biol. 280, 103 (1998).
34. J. Frank et al., Biochem. Cell Biol. 73, 757 (1995).
35. W. Kuhlbrandt and P. N. Unwin, J. Mol. Biol. 156, 431 (1982); M. M. Yusupov and A. S. Spirin, FEBS Lett. 197, 229 (1986); M. Santer and S. Shane, J. Bacteriol. 130, 900 (1977); N. M. Chapman and H. F. Noller, J. Mol. Biol. 109, 131 (1977); W. Herr and H. F. Noller, ibid. 130, 421 (1979); S. K. Vassilenko et al., ibid. 152, 699 (1981).
36. C. Merryman et al., J. Mol. Biol. 285, 97 (1999).
37. C. Merryman et al., ibid., p. 107.
38. G. M. Culver, J. H. Cate, G. Zh. Yusupova, M. M. Yusupov, H. F. Noller, Science 285, 2133 (1999).
39. H. F. Noller et al., Biochem. Cell Biol. 73, 997 (1995); T. R. Easterwood and S. C. Harvey, ibid., p. 751; D. L. Fink, R. O. Chen, H. F. Noller, R. B. Altman, RNA 2, 851 (1996); F. Mueller and R. Brimacombe, J. Mol. Biol. 271, 524 (1997).
40. G. M. Culver, unpublished data.
41. H. A. Heus and A. Pardi, Science 253, 191 (1991).
42. G. M. Culver, G. M. Heilek, H. F. Noller, J. Mol. Biol. 286, 355 (1999).
43. P. Mitchell, M. Osswald, R. Brimacombe, Biochemistry 31, 3004 (1992).
44. S. Joseph, B. Weiser, H. F. Noller, Science 278, 1093 (1997).
45. L. F. Newcomb and H. F. Noller, Biochemistry 38, 945 (1999).
46. T. Powers and H. F. Noller, RNA 1, 194 (1995).
47. G. Schwedler, R. Albrecht-Ehrlich, K. H. Rak, Eur. J. Biochem. 217, 361 (1993).
48. J. H. Cate and J. A. Doudna, Structure 4, 1221 (1997).
49. D. Gautheret, D. Konings, R. R. Gutell, J. Mol. Biol. 242, 1 (1994).
50. J. H. Cate et al., Science 273, 1678 (1996).
51. F. Michel and E. Westhof, J. Mol. Biol. 216, 585 (1990); A. M. Pyle, F. L. Murphy, T. R. Cech, Nature 358, 123 (1992); S. A. Strobel et al., Nature Struct. Biol. 5, 60 (1998).
52. D. Fourmy, S. Yoshizawa, J. D. Puglisi, J. Mol. Biol. 277, 333 (1998).
53. D. Moazed and H. F. Noller, Cell 47, 985 (1986).
54. ______, J. Mol. Biol. 211, 135 (1990).
55. J. Ofengand et al., Biochemistry 18, 4322 (1979); J. B. Prince et al., Proc. Natl. Acad. Sci. U.S.A. 79, 5450 (1982).
56. The surface area of the subunit interface was estimated by circumscribing the electron density at each bridge (lines shown in Fig. 3) and calculating the planar areas of these polygons.
57. M. Gerstein, Acta Crystallogr. A48, 271 (1992).
58. A. S. Spirin, FEBS Lett. 40, 38 (1974); A. Wishnia and A. S. Boussert, J. Mol. Biol. 116, 577 (1977).
59. M. A. Röüld et al., Science 246, 1135 (1989); P. Nissen et al., ibid. 270, 1464 (1995); M. Ruff et al., ibid. 252, 1682 (1991); R. Sankaranarayanan et al., Cell 97, 371 (1999).
60. D. Mangroo, X. Q. Wu, U. L. RajBhandary, Biochem. Cell Biol. 73, 1023 (1995).
61. R. R. Samaha, R. Green, H. F. Noller, Nature 377, 309 (1995).
62. A. Hüttenerhofer and H. F. Noller, Proc. Natl. Acad. Sci. U.S.A. 89, 7851 (1992).
63. S. J. d. Rose, P. T. Lowary, O. C. Uhlenbeck, J. Mol. Biol. 167, 103 (1983).
64. M. Nirenberg and P. Leder, Science 145, 1399 (1964).
65. U. von Ahsen and H. F. Noller, ibid. 267, 234 (1995).
66. H. F. Noller et al., in Structure, Function, and Genetics of Ribosomes, B. Hardesty and G. Kramer, Eds. (Springer-Verlag, New York, 1986), pp. 143–163.
67. H. Grosjean, D. G. Söll, D. M. Crothers, J. Mol. Biol. 103, 499 (1976).
68. J. J. Hopfield, Proc. Natl. Acad. Sci. U.S.A. 71, 4135 (1974).

69. C. G. Kurland, in Ribosomes: Structure, Function, and Genetics, G. Chambliss et al., Eds. (University Park Press, Baltimore, MD, 1980), pp. 597–614.
70. R. Lill, J. M. Robertson, W. Wintermeyer, EMBO J. 8, 3933 (1989).
71. S. V. Kirillov, E. M. Makarov, P. Semenkov Yu, FEBS Lett. 157, 91 (1983).
72. D. Moazed and H. F. Noller, Nature 342, 142 (1989); B. Hardesty, O. W. Odom, H. Y. Deng, in Structure, Function, and Genetics of Ribosomes, B. Hardesty and G. Kramer, Eds. (Springer-Verlag, New York, 1986), pp. 495–508; O. W. Odom, W. D. Picking, B. Hardesty, Biochemistry 29, 10734 (1990); C. Borowski, M. V. Rodnina, W. Wintermeyer, Proc. Natl. Acad. Sci. U.S.A. 93, 4202 (1996).
73. D. Moazed and H. F. Noller, Nature 327, 389 (1987).
74. J. S. Lodmell and A. E. Dahlberg, Science 277, 1262 (1997).
75. B. Wimberly, G. Varani, I. Tinoco Jr., Biochemistry 32, 1078 (1993).
76. C. C. Correll et al., Proc. Natl. Acad. Sci. U.S.A. 95, 13436 (1998).
77. N. B. Leontis and E. Westhof, J. Mol. Biol. 283, 571 (1998).
78. C. M. Bowman et al., Proc. Natl. Acad. Sci. U.S.A. 68, 964 (1971).
79. P. N. Allen and H. F. Noller, ibid., J. Mol. Biol. 457 (1989).
80. Z. Otwinowski et al. (Academic Press, San Diego, CA, 1997), vol. 276, pp. 307–326.
81. Collaborative Computing Project Number 4, Acta Crystallogr. D50, 760 (1994); T. A. Jones, S. Thirup, EMBO J. 5, 819 (1986); T. Jones et al., Acta Crystallogr. A47, 110 (1991); T. A. Jones, in Molecular Replacement, E. J. Dodson, S. Gover, W. Wolf, Eds. (Science and Engineering Research Council Daresbury Laboratory, Warrington, UK, 1991), pp. 91–105.
82. Z. Otwinowski, in Isomorphous Replacement and Anomalous Scattering, W. Wolf, P. R. Evans, A. G. W. Leslie, Eds. (Science and Engineering Research Council Daresbury Laboratory, Warrington, UK, 1991), pp. 80–86.
83. L. Grate and J. Cate, unpublished data.
84. W. M. Clemons Jr. et al., Nature 400, 833 (1999).
85. N. Ban et al., ibid., p. 841.
86. Electron density maps and molecular structure drawings were generated with RIBBONS [M. Carson, Methods Enzymol. 277, 493 (1997)]. We are grateful to A. S. Spirin for his support of early stages of efforts on the study and crystallization of *T. thermophilus* ribosomes. We thank J. Frank for making available a 25 Å cryo-EM electron density map; D. Moras and C. Wilson for their participation and encouragement in early stages of this work; S. Joseph for help with ASL synthesis; Y. Lindqvist and G. Schneider for the gift of Ta<sub>6</sub>Br<sub>12</sub>; H. Taube for providing osmium (III) hexamine; D. Sussman, J. Nix, and J. Diener for help with data collection; L. Grate for programming; H. Bellamy of SSRL for help with preliminary synchrotron experiments; L.-W. Hung, G. McDermott, K. Henderson, and C. Cork for help at the ALS; J. Murray for help with hardware configuration; J. Doudna, F. Michel, and W. Scott for helpful comments on the manuscript; and W. Scott and A. and H. Szöke for stimulating discussions. These studies were supported by NIH grants GM-17129 and GM-59140, and a grant from the Agouron Institute (to H.F.N.). J.H.C. was supported by a Damon Runyon–Walter Winchell postdoctoral fellowship. The Macromolecular Crystallography Facility at the Advanced Light Source is principally funded by the Office of Biological and Environmental Research of the Department of Energy, with additional funding from the National Institute of General Medical Sciences of the NIH, Lawrence Berkeley National Laboratory; and other participating research team members. The Stanford Synchrotron Radiation Laboratory (SSRL) is funded by the Department of Energy and the NIH, National Center for Research Resources, Biomedical Technology Program. Coordinates for the identified structures of the 70S ribosome have been deposited with the Protein Data Bank (accession number 486D).

24 August 1999; accepted 2 September 1999

http://www.jstor.org

LINKED CITATIONS
- Page 1 of 5 -

You have printed the following article:

X-Ray Crystal Structures of 70S Ribosome Functional Complexes
Jamie H. Cate; Marat M. Yusupov; Gulnara Zh. Yusupova; Thomas N. Earnest; Harry F. Noller
Science, New Series, Vol. 285, No. 5436. (Sep. 24, 1999), pp. 2095-2104.
Stable URL:
http://links.jstor.org/sici? sici=0036-8075%2819990924%293%3A285%3A5436%3C2095%3AXCSO7R%3E2.0.CO%3B2-Y

This article references the following linked citations:

References and Notes

4 Three tRNA Binding Sites on Escherichia coli Ribosomes
Hans-Jörg Rheinberger; Hans Sternbach; Knud H. Nierhaus
Proceedings of the National Academy of Sciences of the United States of America, Vol. 78, No. 9,
[Part 2: Biological Sciences]. (Sep., 1981), pp. 5310-5314.
Stable URL:
http://links.jstor.org/sici? sici=0027-8424%28198109%2978%3A9%3C5310%3ATTBSOE%3E2.0.CO%3B2-M

9 Structure of the A Site of Escherichia coli 16S Ribosomal RNA Complexed with an
Aminoglycoside Antibiotic
Dominique Fourmy; Michael I. Recht; Scott C. Blanchard; Joseph D. Puglisi
Science, New Series, Vol. 274, No. 5291. (Nov. 22, 1996), pp. 1367-1371.
Stable URL:
http://links.jstor.org/sici? sici=0036-8075%2819961122%293%3A274%3A5291%3C1367%3ASOTASO%3E2.0.CO%3B2-V

11 Crystal Structure of a Conserved Ribosomal Protein-RNA Complex
Graeme L. Conn; David E. Draper; Eaton E. Lattman; Apostolos G. Gittis
Science, New Series, Vol. 284, No. 5417. (May 14, 1999), pp. 1171-1174.
Stable URL:
http://links.jstor.org/sici? sici=0036-8075%2819990514%293%3A284%3A5417%3C1171%3ACSOACR%3E2.0.CO%3B2-U

21 Direct Visualization of A-, P-, and E-Site Transfer RNAs in the Escherichia coli Ribosome
Rajendra K. Agrawal; Pawel Penczek; Robert A. Grassucci; Yanhong Li; ArDean Leith; Knud H.
Nierhaus; Joachim Frank
Science, New Series, Vol. 271, No. 5251. (Feb. 16, 1996), pp. 1000-1002.
Stable URL:
http://links.jstor.org/sici? sici=0036-8075%2819960216%293%3A271%3A5251%3C1000%3ADVOAPA%3E2.0.CO%3B2-0

NOTE: The reference numbering from the original has been maintained in this citation list.

http://www.jstor.org

LINKED CITATIONS
- Page 2 of 5 -

28 Arrangement of the Subunits in the Ribosome of Escherichia coli: Demonstration by Immunoelectron Microscopy
Berthold Kastner; Marina Stoffler-Meilicke; Georg Stoffler
Proceedings of the National Academy of Sciences of the United States of America, Vol. 78, No. 11, [Part 2: Biological Sciences]. (Nov., 1981), pp. 6652-6656.
Stable URL:
http://links.jstor.org/sici? sici=0027-8424%28198111%2978%3A11%3C6652%3AAOTSIT%3E2.0.CO%3B2-U

38 Identification of an RNA-Protein Bridge Spanning the Ribosomal Subunit Interface
Gloria M. Culver; Jamie H. Cate; G. Zh. Yusupova; Marat M. Yusupov; Harry F. Noller
Science, New Series, Vol. 285, No. 5436. (Sep. 24, 1999), pp. 2133-2135.
Stable URL:
http://links.jstor.org/sici? sici=0036-8075%2819990924%293%3A285%3A5436%3C2133%3AIOARBS%3E2.0.CO%3B2-Y

41 Structural Features that Give Rise to the Unusual Stability of RNA Hairpins Containing GNRA Loops
Hans A. Heus; Arthur Pardi
Science, New Series, Vol. 253, No. 5016. (Jul. 12, 1991), pp. 191-194.
Stable URL:
http://links.jstor.org/sici? sici=0036-8075%2819910712%293%3A253%3A5016%3C191%3ASFTGRT%3E2.0.CO%3B2-O

44 Mapping the Inside of the Ribosome with an RNA Helical Ruler
Simpson Joseph; Bryn Weiser; Harry F. Noller
Science, New Series, Vol. 278, No. 5340. (Nov. 7, 1997), pp. 1093-1098.
Stable URL:
http://links.jstor.org/sici? sici=0036-8075%2819971107%293%3A278%3A5340%3C1093%3AMTIOTR%3E2.0.CO%3B2-H

50 Crystal Structure of a Group I Ribozyme Domain: Principles of RNA Packing
Jamie H. Cate; Anne R. Gooding; Elaine Podell; Kaihong Zhou; Barbara L. Golden; Craig E. Kundrot; Thomas R. Cech; Jennifer A. Doudna
Science, New Series, Vol. 273, No. 5282. (Sep. 20, 1996), pp. 1678-1685.
Stable URL:
http://links.jstor.org/sici? sici=0036-8075%2819960920%293%3A273%3A5282%3C1678%3ACSOAGI%3E2.0.CO%3B2-M

NOTE: The reference numbering from the original has been maintained in this citation list.

http://www.jstor.org

LINKED CITATIONS
- Page 3 of 5 -

55 Covalent Crosslinking of tRNA<sub>1</sub><sup>Val</sup> to 16S RNA at the Ribosomal P Site: Identification of Crosslinked Residues
Jeffrey B. Prince; Brian H. Taylor; David L. Thurlow; James Ofengand; Robert A. Zimmermann
Proceedings of the National Academy of Sciences of the United States of America, Vol. 79, No. 18, [Part 1: Biological Sciences]. (Sep. 15, 1982), pp. 5450-5454.
Stable URL:
http://links.jstor.org/sici? sici=0027-8424%2819820915%2979%3A18%3C5450%3AACOTT1%3E2.0.CO%3B2-8

59 Crystal Structure of the Ternary Complex of Phe-tRNA<sup>Phe</sup>, EF-Tu, and a GTP Analog
Poul Nissen; Morten Kjeldgaard; Søren Thirup; Galina Polekhina; Ludmila Reshetnikova; Brian F. C. Clark; Jens Nyborg
Science, New Series, Vol. 270, No. 5241. (Dec. 1, 1995), pp. 1464-1472.
Stable URL:
http://links.jstor.org/sici? sici=0036-8075%2819951201%29270%3A5241%3C1464%3ACSOTTC%3E2.0.CO%3B2-%23

59 Class II Aminoacyl Transfer RNA Synthetases: Crystal Structure of Yeast Aspartyl-tRNA Synthetase Complexed with tRNA<sup>Asp</sup>
M. Ruff; S. Krishnaswamy; M. Boeglin; A. Poterszman; A. Mitschler; A. Podjarny; B. Rees; J. C. Thierry; D. Moras
Science, New Series, Vol. 252, No. 5013. (Jun. 21, 1991), pp. 1682-1689.
Stable URL:
http://links.jstor.org/sici? sici=0036-8075%2819910621%29252%3A5013%3C1682%3ACIATRS%3E2.0.CO%3B2-W

62 Hydroxyl Radical Cleavage of tRNA in the Ribosomal P Site
Alexander Huttenhofer; Harry F. Noller
Proceedings of the National Academy of Sciences of the United States of America, Vol. 89, No. 17. (Sep. 1, 1992), pp. 7851-7855.
Stable URL:
http://links.jstor.org/sici? sici=0027-8424%2819920901%2989%3A17%3C7851%3AHRCOTI%3E2.0.CO%3B2-9

64 RNA Codewords and Protein Synthesis
Marshall Nirenberg; Philip Leder
Science, New Series, Vol. 145, No. 3639. (Sep. 25, 1964), pp. 1399-1407.
Stable URL:
http://links.jstor.org/sici? sici=0036-8075%2819640925%29145%3A3639%3C1399%3ARCAPS%3E2.0.CO%3B2-4

NOTE: The reference numbering from the original has been maintained in this citation list.

http://www.jstor.org

LINKED CITATIONS
- Page 4 of 5 -

65 Identification of Bases in 16S rRNA Essential for tRNA Binding at the 30S Ribosomal P Site
Uwe von Ahsen; Harry F. Noller
Science, New Series, Vol. 267, No. 5195. (Jan. 13, 1995), pp. 234-237.
Stable URL:
http://links.jstor.org/sici?sici=0036-8075%2819950113%293%3A267%3C5195%3C234%3CAIOBI1R%3E2.0.CO%3B2-V

68 Kinetic Proofreading: A New Mechanism for Reducing Errors in Biosynthetic Processes Requiring High Specificity
J. J. Hopfield
Proceedings of the National Academy of Sciences of the United States of America, Vol. 71, No. 10. (Oct., 1974), pp. 4135-4139.
Stable URL:
http://links.jstor.org/sici?sici=0027-8424%28197410%2971%3A10%3C4135%3CAKPANMF%3E2.0.CO%3B2-Y

72 Truncated Elongation Factor G Lacking the G Domain Promotes Translocation of the 3 <sup><font face=symbol>#</font></sup> End but not of the Anticodon Domain of Peptidyl-tRNA
Christian Borowski; Marina V. Rodnina; Wolfgang Wintermeyer
Proceedings of the National Academy of Sciences of the United States of America, Vol. 93, No. 9. (Apr. 30, 1996), pp. 4202-4206.
Stable URL:
http://links.jstor.org/sici?sici=0027-8424%2819960430%2993%3A9%3C4202%3CATEFGLT%3E2.0.CO%3B2-Q

74 A Conformational Switch in Escherichia coli 16S Ribosomal RNA During Decoding of Messenger RNA
J. Stephen Lodmell; Albert E. Dahlberg
Science, New Series, Vol. 277, No. 5330. (Aug. 29, 1997), pp. 1262-1267.
Stable URL:
http://links.jstor.org/sici?sici=0036-8075%2819970829%293%3A277%3C5330%3CAACSIEC%3E2.0.CO%3B2-Z

76 Crystal Structure of the Ribosomal RNA Domain Essential for Binding Elongation Factors
Carl C. Correll; Alexander Munishkin; Yuen-Ling Chan; Zhong Ren; Ira G. Wool; Thomas A. Steitz
Proceedings of the National Academy of Sciences of the United States of America, Vol. 95, No. 23. (Nov. 10, 1998), pp. 13436-13441.
Stable URL:
http://links.jstor.org/sici?sici=0027-8424%2819981110%2995%3A23%3C313436%3CACSOTRR%3E2.0.CO%3B2-W

NOTE: The reference numbering from the original has been maintained in this citation list.

http://www.jstor.org

LINKED CITATIONS  
- Page 5 of 5 -

78 Specific Inactivation of 16S Ribosomal RNA Induced by Colicin E3 in vivo  
C. M. Bowman; J. E. Dahlberg; T. Ikemura; J. Konisky; M. Nomura  
Proceedings of the National Academy of Sciences of the United States of America, Vol. 68, No. 5.  
(May, 1971), pp. 964-968.  

Stable URL:  
http://links.jstor.org/sici?sici=0027-8424%28197105%2968%3A5%3C964%3ASIO1RR%3E2.0.CO%3B2-6

NOTE: The reference numbering from the original has been maintained in this citation list.
