
Molecular Biology of the Cell  
Vol. 10, 3263–3277, October 1999  

# Phosphorylation Controls Timing of Cdc6p Destruction: A Biochemical Analysis  

Suzanne Elsasser, Yong Chi, Ping Yang, and Judith L. Campbell*  

Braun Laboratories, California Institute of Technology, Pasadena, California 91125  

Submitted March 5, 1999; Accepted August 2, 1999  
Monitoring Editor: Mark J. Solomon  

The replication initiation protein Cdc6p forms a tight complex with Cdc28p, specifically with forms of the kinase that are competent to promote replication initiation. We now show that potential sites of Cdc28 phosphorylation in Cdc6p are required for the regulated destruction of Cdc6p that has been shown to occur during the *Saccharomyces cerevisiae* cell cycle. Analysis of Cdc6p phosphorylation site mutants and of the requirement for Cdc28p in an in vitro ubiquitination system suggests that targeting of Cdc6p for degradation is more complex than previously proposed. First, phosphorylation of N-terminal sites targets Cdc6p for polyubiquitination probably, as expected, through promoting interaction with Cdc4p, an F box protein involved in substrate recognition by the Skpl-Cdc53-F-box protein (SCF) ubiquitin ligase. However, in addition, mutation of a single, C-terminal site stabilizes Cdc6p in G2 phase cells without affecting substrate recognition by SCF in vitro, demonstrating a second and novel requirement for specific phosphorylation in degradation of Cdc6p. SCF-Cdc4p- and N-terminal phosphorylation site-dependent ubiquitination appears to be mediated preferentially by Clbp/Cdc28p complexes rather than by Clnp/Cdc28ps, suggesting a way in which phosphorylation of Cdc6p might control the timing of its degradation at then end of G1 phase of the cell cycle. The stable *cdc6* mutants show no apparent replication defects in wild-type strains. However, stabilization through mutation of three N-terminal phosphorylation sites or of the single C-terminal phosphorylation site leads to dominant lethality when combined with certain mutations in the anaphase-promoting complex.

## INTRODUCTION

A hallmark of eukaryotic DNA replication is that replication is restricted to one phase of the cell cycle, S phase, and that this replication is mediated by the coordinated firing of a multitude of replication origins spread over many chromosomes. In *Saccharomyces cerevisiae*, replication is initiated from hundreds of origins, and initiation depends on the origin recognition complex (ORC), which can be detected at origins by means of in vivo footprinting and is present at origins throughout the cell cycle (Bell and Stillman, 1992; Bell *et al.*, 1993; Diffley and Cocker, 1992; Rowley *et al.*, 1995). At the end of mitosis the origin footprint expands, and genetic and molecular evidence suggest that the six-subunit ORC, Cdc6p, the six Mcms, and Cdc45p assemble at origins to form a prereplicative complex (preRC) at this time (Newlon, 1997). Origins are then said to be licensed for replication (Blow and Laskey, 1988). PreRCs are subsequently activated by two kinases, Clb5,6p/Cdc28p and Dbf4p/Cdc7p. Activation coincides with the return of the origin footprint to its unlicensed state.

To maintain genomic integrity, cells must coordinate DNA replication such that every origin of replication fires only once per cell cycle. In *S. cerevisiae*, the S phase cyclin/Cdc28ps, Clb5p/Cdc28p, and Clb6p/Cdc28p promote the initiation of DNA replication and also prevent aberrant reinitiation. The S phase cyclin/Cdc28 proteins appear well before the G1-S transition but are inhibited by the presence of Sic1p (Nasmyth, 1993, 1996; Schwob and Nasmyth, 1993; Schwob *et al.*, 1994; Surana *et al.*, 1993; Dahmann *et al.*, 1995; Dirick *et al.*, 1995). Destruction of Sic1p at the G1-S transition activates the S phase cyclin/Cdc28ps, and DNA replication follows closely upon activation, occurring earlier in the cell cycle if Sic1p is deleted and the S phase cyclin/Cdc28ps is prematurely activated and occurring later in the presence of a stabilized allele of Sic1p, which maintains the S phase cyclin/Cdc28ps in an inactive state (Schwob *et al.*, 1994; Verma *et al.*, 1997a). Additionally, in the absence of S phase cyclins, DNA replication is delayed until activation of the mitotic cyclins Clb1-Clb4. Thus, by all in vivo measures, initiation of DNA replication is defined by the activation of the Clbp/Cdc28p kinases. Phosphorylation of substrates may control origin firing in a number of ways: phosphorylation may cause the import or export of DNA replication proteins from the nucleus; phosphorylation may enhance or block the assembly of replication proteins into replication

* Corresponding author. E-mail address: jcampbel@cco.caltech.edu.
S. Elasser *et al.*

complexes; phosphorylation may trigger changes in conformation and activation or inactivation of activities of initiation proteins; and phosphorylation may target initiation proteins for degradation. To establish which mechanisms are at play requires identification of the substrates that are phosphorylated at initiation. Although one candidate, Cdc45p, has been suggested by recent experiments (Zou and Stillman, 1998), and ORC, Cdc6p, and the Mcms have consensus cyclin-dependent kinase (CDK) phosphorylation sites, verification of the critical substrates has proved difficult.

As just mentioned, firing of licensed preRCs depends on Clb/Cdc28 kinases, but formation of preRCs, by contrast, is inhibited in the presence of Clb/Cdc28 complexes (Dahmann *et al.*, 1995). Thus, once activated by Clb kinases, origins of replication cannot refire until passage through mitosis inhibits Clb kinases. The substrates that are phosphorylated by Clbp/Cdc28p and thus prevent spurious replication are also unknown.

We are interested in characterizing the substrates of the Clbp/Cdc28ps in the triggering of initiation and the prevention of reinitiation. We have previously shown that Cdc6p both associates with and is phosphorylated by Clbp/Cdc28p (Elsasser *et al.*, 1996). The results presented here lead to the strong inference that phosphorylation of N-terminal sequences in Cdc6p by Clbp/Cdc28p can target Cdc6p for recognition by Skp1-Cdc53-F-box protein (SCF)-Cdc4p and proteasomal degradation. It has been shown that S. *cerevisiae* arrested in S or G2 degrade bulk Cdc6p rapidly, but that Cdc6p is stable as cells progress through G1 (Piatti *et al.*, 1995; Drury *et al.*, 1997; Sanchez *et al.*, 1999). The cell cycle–dependent instability of Cdc6p could mean that Cdc6p is degraded in a Clbp/Cdc28–dependent manner. Cdc6p is considerably more stable in *cdc4/cdc34/cdc53* pathway mutants than in wild-type cells (Piatti *et al.*, 1995; Drury *et al.*, 1997; Sanchez *et al.*, 1999), suggesting a role for SCF-Cdc4p in Cdc6p turnover. Cdc6p is also stabilized by deleting the N terminus, a region involved in Cdc28 binding, phosphorylation by Cdc28, nuclear localization, restraint of aberrant mitosis, and Cdc4p binding (Elsasser *et al.*, 1996; Jong *et al.*, 1996; Drury *et al.*, 1997; Weinreich *et al.*, 1999). As expected from the stability in *cdc4* mutants, Cdc6p is ubiquitinated in vivo in a Cdc4-dependent manner (Sanchez *et al.*, 1999). Using an in vitro ubiquitination system combined with a collection of new *cdc6* alleles, we set out to explore the role of phosphorylation of Cdc6p in Cdc4p-dependent ubiquitination. We have correlated these findings with the effects of the mutations on Cdc6p stability in G2 phase. We confirm a role for the N terminus in targeting Cdc6p for degradation by the SCF-Cdc4p-mediated ubiquitination pathway and demonstrate a requirement for phosphorylation by either Clnp or Clbp kinases. However, the behavior of several of the *cdc6* mutants suggests that the regulation of Cdc6p turnover cannot be entirely accounted for by SCF-Cdc4p–mediated ubiquitination, as had been suggested to date. This may indicate either a direct or indirect role for the anaphase-promoting complex (APC) (Peters, 1999) as well as SCF in regulation of Cdc6p function.

Bacto-yeast extract, 2% Bacto-peptone, 2% raffinose), YPG (1% Bacto-yeast extract, 2% Bacto-peptone, 2% galactose), SD minus uracil (2% dextrose, 7.5% yeast nitrogen base, 0.5% casamino acids, 20 mg/l adenine sulfate, 20 mg/l tryptophan), SR minus uracil (2% raffinose, 7.5% yeast nitrogen base, 0.5% casamino acids, 20 mg/l adenine sulfate, 20 mg/l tryptophan), and SD minus leucine (2% dextrose, 7.5% yeast nitrogen base, 0.6 g/l CSM-his-leu-trp-ura [Bio101, Vista, CA], 20 mg/l histidine, 20 mg/l uracil).

### Strains

The strains used in this work include BJ2168 (*a prcl-407 prbl-1122 pep4-3 leu2 trpl ura3-52*), BJ5459 (*a ura3-52 trpl lys2-801 leu2Δ1 his3Δ200 pep4::HIS3 prblΔ1.6R can1 GAL*) (Yeast Genetic Stock Center, Berkeley, CA), RJD635 (*a ura3 leu2 trpl pep4::TRP1 cdc28::CDC28-HA::TRP1 barl::LEU2*) (Elsasser *et al.*, 1996), RJD885 (*a ura3 leu2 trpl cln1::URA3 cln2::LEU2 cln3::URA3 leu2::GAL-CLN3::LEU2 pep4::TRP1 cdc28::CDC28-HA::HIS3*) (Verma *et al.*, 1997b), RJD893 (*a cdc4 ura3 leu2 trpl cln1::URA3 cln2::LEU2 cln3::URA3 leu2::GAL-CLN3::LEU2 pep4::TRP1 cdc28::CDC28-HA::HIS3*), L40 (*a his3Δ200 trpl-901 leu2-3112 ade2 lys2-801 URA3::(lexA OP)*~g~*-LacZ LYS2::(lexA OP)~4~-HIS3*), and JLC1402 (*cdc16-26/cdc16-264 ura3/ura3 leu2/leu2 trpl/trpl*) (from K. Heichman, Myriad Genetics, Salt Lake City, UT).

### Plasmids

#### Preparation of Vectors with the GAL1,10 Promoter (All Constructs with GAL Promoter)

Yeast and Bluescript (Stratagene, La Jolla, CA) plasmids bearing the GAL promoter flanked by *SacI* and *BamHI* sites were prepared as follows. pBM150 (Johnston and Davis, 1984) was cut with *EcoRI*, and an oligonucleotide cassette composed of SE25 and SE26 (Table 1) was cloned in. This oligocassette has ends that anneal to the *EcoRI* ends but are designed such that the *EcoRI* sites are not regenerated, and the double-stranded region of the cassette contains a *SacI* site. The GAL promoter was excised with *SacI* and *BamHI* and cloned into the same sites in pBluescript SK(−), resulting in pELS42, and into the same sites in the yeast-integrating vector with a *URA3*-selectable marker, pRS306 (Sikorski and Hieter, 1989), resulting in pELS89.

#### Introduction of a Multiple myc Tag at the C Terminus of CDC6

A Bluescript-based vector containing the coding region for a multiple myc epitope tag was generated by ligation of a cassette composed of oligonucleotides SE53 and SE54 (Table 1) and cloning these ladders into the *SpeI* site of pBluescript SK(−). A construct containing seven head-to-tail myc tags, pELS47, was selected for preparation of poly-myc-tagged CDC6. A fragment encoding residues 392–513 of *CDC6* (from the *StyI* site to the last residue) fused to a multiple myc tag was generated by recombinant PCR using oligonucleotides SE61 through SE64 and templates pELS47 and pELS20 (Elsasser *et al.*, 1996). The resulting PCR product was cloned into the *SacI* and *SpeI* sites of pBluescript SK(−), generating pELS55. Sequencing of the PCR product revealed that two extra myc tags had been created during the course of the PCR reaction.

#### Preparation of Yeast Vectors Expressing Small N-terminal Deletions of CDC6 (Figure 1)

pRS315-based vectors (*ARS1, CEN4,* and *LEU2*) (Sikorski and Hieter, 1989) for the expression of *CDC6* alleles with small deletions in the N terminus were assembled from three fragments: the GAL promoter, N-terminal fragments (from residue 1 to 48) bearing various mutations, and a fragment (from residue 47 to 513) bearing the C terminus of *CDC6* fused the Myc9 epitope tag. Mutagenesis of the N terminus of *CDC6* was accomplished as follows. The region of *CDC6* encoding the first 48 residues of the protein was excised from pKTU6 (Elsasser *et al.*, 1996) with *BamHI* and *PstI* and cloned into the same sites in pBluescript SK(−), generating pELS12. (The sequence near the start of the open reading frame (ORF) is GGATCCCATATG, where the start codon is under-

### MATERIALS AND METHODS

#### Media

The following abbreviations are used for yeast media: YPD (1% Bacto-yeast extract, 2% Bacto-peptone, 2% dextrose), YPR (1%
Table 1. Oligonucleotides

| Oligonucleotide | Sequence |
|-----------------|----------|
| SE25           | AATTGGCCGAGCTCC |
| SE26           | AATTGGAGCTCGGCC |
| SE27           | GATCCGGGCATATGGGCGGCCCTGCA |
| SE28           | GGCCGCCCCATATGCCG |
| SE37           | GGCCAAAGCTTCAAATTGTACCCCACCTC |
| SE45           | GGATCCCCATATGTCATTTGACGATGCTCCAG |
| SE46           | CCAACTAAGCGTATCCCTTTGAAAAAGAAAAAAGTTGC |
| SE47           | CGATGCTCCAGCAGATGTTACACCAGAATC |
| SE48           | CGACCTTTGAAAAAGAAAAAAAAAGCTGCAGTTTGGCTC |
| SE53           | CTAGCGAACAAAAGTTGATTCTGAAGAAGAATTGC |
| SE54           | CTAGGCAAATCTTCTTCAGAAAATCAACTTTTGTTCG |
| SE61           | GCGAGCTCTGAACACTACATAGCC |
| SE62           | TGACTAGTGGGCCGCTCTAGAACT |
| SE63           | GAAACCTTTCCCTCACCCCCGGGGGTTCCACTAGC |
| SE64           | GCTAGTGGAAACCCCCGGGGGTGAAGGAAAAGGTTTC |
| SE72           | CCTCCACGACCTTTGGCAGCAGCAGCGTTGCAGTTCCACAGAT |
| SE74           | TATGTCAGCTATACCAATAACTCTGCA |
| SE75           | GAGTTATTGGTATAGCTGACA |
| SE84           | GCTCCAGCAGCGGCCTCCACGA |
| SE97           | GCTATACCAATAGCCCAACTAAGCGT |
| SE98           | ACACCAGAATCAGCGCCAGAAAAACTG |
| SE99           | TTGTCGCTCTCCGCGCCACGCTCGAAG |
| SE100          | CGGTTTCTGCTTGGCGCAAACAAGAGGA |
| SE101          | ACGCCAACTACTGCGCCGGTAAAGAAA |
| SE102          | GGCGGCCGCAATACGACTCACTATAGGGATGTCAGCTATACCAATA |
| SE103          | TTGAAACCTTTCCTCACTAGTCAGCTAATAGATTATT |
| SE108          | GGATCCTAATACGACTCACTATAGGAACAGACCACCATGGCAGCTATACCAATAACTCCAACT |
| SE128          | GGGGATCCGTATGTCAGCTATACCAATAACTCC |
| SE129          | GGGGATCCGTATGTCAGCTATACCAATAGCTC |
| SE130          | GGGGATCCGTATGTCATTTGACGATGCTCC |
| SE131          | GGGGATCCGTCTGCAGTTTGGCTCACAG |
| SE132          | GGGGGATCCTCACTAGTGAAAGGAAAGGTTCAAAATTGAAAT |
| SE133          | GGGGGTCGACTCACTAGTGAAAGGAAAGGTTCAAAATTGAAAT |
| SE134          | GGATCCTAATACGACTCACTATAGGAACAGACCACCATGGCAGCTATACCAATAACTCTGCAG |
| SE135          | GGATCCTAATACGACTCACTATAGGAACAGACCACCATGGCATTTGACGATGCTCCAGCAACG |
| SE136          | GGATCCTAATACGACTCACTATAGGAACAGACCACCATGGCAGCTATACCAATAGCGCCAACT |

lined.) pELS 12 was transformed into CJ 236 (dut ung), and single-stranded DNA was prepared from this strain. Deletions within the C-terminal 46 amino acids were prepared with the oligonucleotide primers indicated in Table 2 using the Muta-Gene mutagenesis kit (Bio-Rad, Hercules, CA) yielding pELS 32 (∆3–16), pELS 33 (∆13–26), pELS 34 (∆23–26), and pELS 52 (∆33–46). The GAL promoter was cloned in front of the mutated N termini by inserting the SacI-BamHI fragment derived from pELS 42 (above), generating plasmids pELS 64 (wild type), pELS 68 (∆3–16), pELS 65 (∆13–26), pELS 69 (∆23–26), and pELS 66 (∆33–46). A Bluescript-based construct bearing the CDC6 ORF from residues 47 to 513 fused in frame with the Myc9tag was generated by excising the natural C terminus with Styl and SpeI from pELS 51, a bacterial vector bearing the CDC6 ORF from residues 47 to 513 followed by 300 bases of the natural 3′ untranslated region (UTR), and cloning in the epitope-tagged C-terminus from pELS 55, resulting in pELS 60. pELS 60 was cut with PstI and HindIII, excising a fragment containing residues 47–513 of CDC6 fused to the Myc9tag, followed by the natural 3′ UTR and cloned into pRS 315, resulting in pELS 63. Fragments bearing the GAL promoter followed by the N-terminal amino acids (from pELS 64 through pELS 69, described above) were cloned into pELS 63, generating pELS 71 (wild type), pELS 72 (∆3–16), pELS 73 (∆13–26), pELS 74 (∆23–26), and pELS 75 (∆33–46).

Preparation of Vectors Containing the Entire CDC6 ORF and C-terminal Myc9 Epitope Tag. Bluescript-based constructs bearing the entire CDC6 ORF were generated by cloning the PstI-HindIII fragment from pELS 60 (containing the CDC6 ORF from residue 47 to 513 fused to the Myc9tag, followed by the natural 3′ UTR) into the

Table 2. Mutagenic oligonucleotides

| Sites mutated | Mutation | Oligonucleotide mutagenesis |
|---------------|----------|-----------------------------|
| ∆N terminus    | ∆8–48    |                              |
| ∆3–16         | ∆3–16    | SE45                        |
| ∆13–26        | ∆13–26   | SE46                        |
| ∆23–36        | ∆23–36   | SE47                        |
| ∆33–46        | ∆33–46   | SE48                        |
| NLS           | K29A,R30A,K31A,K32A | SE72                |
| A             | T7A      | SE97                        |
| B             | T23A     | SE84                        |
| C             | S43A     | SE98                        |
| D             | T135A    | SE99                        |
| E             | S354A    | SE100                       |
| F             | S372A    | SE101                       |

S. Elasser et al.

Table 3. Plasmids for in vitro ubiquitination and stability studies

| Sites mutated | Bluescript construct | GAL plasmid |
|---------------|----------------------|-------------|
| Wild type     | pELS 91              | pELS 147    |
| Δ8-46         | pELS 144             | pELS 148    |
| Δ3-16         | pELS 92              | pELS 149    |
| Δ13-26        | pELS 93              | pELS 150    |
| Δ23-36        | pELS 94              | pELS 151    |
| Δ33-46        | pELS 95              | pELS 152    |
| NLS           | pELS 105             | pELS 156    |
| A             | pELS 164             | pELS 176    |
| B             | pELS 193             | pELS 193    |
| C             | pELS 177             | pELS 177    |
| D             | pELS 165             | pELS 178    |
| E             | pELS 166             | pELS 179    |
| F             | pELS 167             | pELS 194    |
| A+B           | pELS 187             | pELS 198    |
| B+C           | pELS 162             | pELS 180    |
| A+C           | pELS 197             | pELS 199    |
| A-C           | pELS 188             | pELS 200    |
| E+F           | pELS 163             | pELS 181    |
| A-F           | pELS159X             | pELS 175    |

Table 4. Plasmids for two-hybrid assays

| 5' oligonucleotide | 3' oligonucleotide | Plasmid   |
|--------------------|--------------------|-----------|
| Wild type          | SE128              | pELS 211  |
| Δ8-48              | SE131              | pELS 212  |
| Δ3-16              | SE130              | pELS 213  |
| Δ13-26             | SE129              | pELS 226  |
| Δ23-36             | SE128              | pELS 214  |
| Δ33-46             | SE128              | pELS 215  |
| NLS                | SE128              | pELS 224  |
| A+B                | SE129              | pELS 217  |
| B+C                | SE128              | pELS 216  |
| A+C                | SE129              | pELS 225  |
| A-C                | SE129              | pELS 229  |
| E+F                | SE128              | pELS 222  |
| A-F                | SE129              | pELS 218  |

cloning into BTM 116. The assortment of oligonucleotides used at the 5' end was required to amplify mutants with engineered sequence differences at the N terminus. Inserts generated with SE132 were subcloned much more efficiently than inserts generated with SE133. The two oligonucleotides are exactly the same at the 3' end, which anneals to the last nine natural codons of CDC6. Following this, SE132 encodes a BamHI site, and SE133 encodes a SalI site. BTM 116 possesses a BamHI site followed directly by a SalI site in the polylinker between the LexA coding region and the transcriptional termination signals. In the cases in which SE132 was used to generate inserts, the BamHI site is maintained after the stop codons. In the cases in which SE133 was used, this site is removed.

Cdc28p Binding

Strain BJ2168 was transformed with pELS 71 through pELS 75. Two hundred-milliliter cultures were grown to midlog phase in SR-ura (2% raffinose, 0.5% casamino acids, 1.5% yeast nitrogen base, and 20 mg/l each adenine and tryptophan). Expression of wild-type and mutant CDC6 was induced by adding solid galactose to 2%. Cells were harvested after 4 h, washed, resuspended in high-salt buffer (Elsasser et al., 1996) at a concentration of 300 mg of wet cells/ml, and lysed by bead beating (Biospec, Bartlesville, OK; Mini Beadbeater, type BX-4), four times at 5000 rpm for 50 s each time, in the presence of 0.5 mm Zir/Silica beads (product number 11079105Z). The resulting extracts were diluted to 20 mg/ml, and small aliquots were removed and mixed with equal volumes of 4× SDS-PAGE loading buffer. Four milligrams of the remaining extract were mixed with 2 μg of 9E10 antibody and 40 μl of 100 μg/ml protein A-Sepharose slurry and incubated with mixing at 4°C for 90 min. Immunoprecipitated protein was recovered, and beads were washed thoroughly. Twenty microliters of 4× SDS-PAGE loading buffer were added to the immunoprecipitates.

Stability Assay

Degradation in Synchronized Cells. The following experiment was carried out at 30°C. Ten-milliliter cultures were grown overnight to early log phase (~2 × 10⁶ cells/ml) in YPR. Galactose was added at a final concentration of 2% to induce CDC6 transcription, and cells were incubated for 2 h. This induction protocol produced even expression levels of wild-type and mutant alleles of Cdc6. We observed that although overnight induction produced higher expression levels of stabilized Cdc6 alleles, the half-lives of various alleles were unaffected by the long induction, indicating that the destruction pathway is difficult to saturate. Nocodazole was then added to a final concentration of 15 μg/ml, and cells were incubated for an additional 3 h. CDC6 transcription was repressed by the

addition of glucose to 2%. Samples (0.75 ml) of cells were removed at various times after the addition of glucose and added to an equal volume of ice-cold 2× yeast stop mix (300 mM NaCl, 100 mM NaF, 2 mM NaN₃, 20 mM EDTA). Cells were recovered by two centrifugations and resuspended in 200 μl of lysis Buffer (50 mM NaOH, 2% SDS, 5% β-mercaptoethanol, 10% glycerol). After vortexing, samples were boiled for 5 min and titrated with 1N HCl to a pH of 7. Debris was removed by centrifugation at 14,000 rpm for 5 min. The concentration of the resulting extract was ~4 mg/ml as determined by A₂₈₀ and A₂₆₀ readings. One hundred fifty microliters of the clear supernatants were transferred to clean tubes and mixed with 1 μl of bromphenol blue. Ten microliters of cleared extract were electrophoresed on 7.5% SDS-PAGE gels and blotted to Hybond nitrocellulose membranes (Amersham, Arlington Heights, IL). Blots were stained with Ponceau-S to ensure that lanes were loaded evenly. Blots were incubated with a 1:1000 dilution of 9E10 ascites (Santa Cruz Biotechnology, Santa Cruz, CA) and a 1:2000 dilution of goat anti-mouse HRP conjugate (Bio-Rad) and developed with ECL reagents (Amersham).

**Degradation in Asynchronous Cells.** The procedure was the same as for synchronous cells, but cells were grown exponentially to a density of 1 × 10⁷ cells/ml before induction.

**In Vitro Transcription and Translation**

DNA templates for transcription of Myc9-tagged alleles of Cdc6p were prepared by PCR amplification of CDC6 sequences from Bluescript-based plasmids listed in Table 3. The 5′ primers were designed to introduce the T7 promoter upstream of the CDC6 coding region (SE108, SE134, SE135, and SE136), and the 3′ primer SE37 corresponds to a sequence from the CDC6 3′ UTR. These primers are listed in Tables 1 and 2. In all cases the 5′ primers anneal to the first 27 nucleotides of CDC6, which are the same as wild type in most cases but different for the alleles with a mutation at the first CDK consensus site (threonine 7 → alanine) and for the deletions Δ3–16 and Δ8–46. Amplification was carried out using 1 ng of plasmid DNA and 0.2 μM primers. DNA concentrations were determined by densitometry using an Alpha Innotech (San Leandro, CA) gel documentation system. A DNA template for the transcription of SIC1 was prepared as described (Verma *et al.*, 1997b). A DNA template for the transcription of untagged CDC6 was produced by amplifying the ORF from the plasmids listed in Table 3 with primers SE102 and SE103. PCR products were purified using the Qiagen (Hilden, Germany) PCR purification kit. Transcription was carried out in a volume of 100 μl by incubating 1 μg of DNA in the presence of 1 mM NTPs, 100 U of RNase inhibitor (Boehringer Mannheim, Indianapolis, IN; product 799025; 1 U inhibits 50% of 5 ng of RNase A), 2 U of T7 RNA polymerase, and accompanying buffer (Boehringer Mannheim) for 75 min at 37°C. Three microliters of RNA were used directly in 20 μl of wheat germ in vitro translation reactions (Promega, Madison, WI) as per the manufacturer’s instructions.

**Preparation of Fractionated Extracts for Ubiquitination**

Preparation of DEAE-fractionated cell extracts was carried out as previously described (Verma *et al.*, 1997b). Strain RJD885 was used for wild-type cell extract, and strain RJD893 was used for cdc4 extracts.

**Purification of Cdc34p**

Cdc34p was purified as previously described (Banerjee *et al.*, 1993). For the experiments in Figure 4 and Figure 5, Cdc34p purified to homogeneity was used. For the experiment in Figure 6, Cdc34p purified through the DEAE step (25% pure) was used.

---

**Expression of Proteins in Insect Cells**

Baculoviruses used for the expression of proteins in insect cells were as follows: vectors for the expression of GST-Cdc28-HAp, Cln2p, Clb5p, and Cdc4p were the same as those previously described (Feldman *et al.*, 1997). For the production of insect cell lysates containing Cln2p/Cdc28p and Clb5p/Cdc28p, SF9 insect cells were coinfected with the relevant viruses at a multiplicity of infection of 10. At 42 h after infection, cells were recovered by centrifugation and resuspended 0.1 vol in ice-cold insect cell lysis buffer (20 mM HEPES, pH 7.4, 100 mM NaCl, 0.1% Triton-X100, 5 mM MgCl₂, 5 mM EDTA, 2 mM DTT, 1 mM PMSF, 1 μg/ml leupeptin and pepstatin, 50 mM NaF, 50 mM β-glycerophosphate). Cells were briefly sonicated to ensure lysis. The lysate was cleared by centrifugation for 15 min at 13,000 × g, brought to a glycerol concentration of 10%, and frozen in aliquots in liquid nitrogen.

**Phosphorylation Assays**

Untagged CDC6 was amplified, and ³⁵S-labeled Cdc6p was produced by in vitro translation as described above. Two microliters of translation product were mixed with 4 μl of insect cell lysate containing GST-Cdc28-HAp and Clb5p, 2 μl of 10× reaction salts and protease inhibitors (50 mM magnesium acetate, 10 mM DTT, 5 mM PMSF, 100 μg/ml each leupeptin and pepstatin), 2 μl of an ATP-regenerating system (10 mM ATP, 350 mM creatine phosphate, 20 mM HEPES, pH 7.2, 10 mM magnesium acetate, 0.5 mg/ml creatine kinase), and 10 μl of distilled water. Half of the reaction was quenched immediately with SDS-PAGE loading buffer, and the other half of the reaction was incubated for 60 min at 25°C. The reaction was stopped by the addition of SDS-PAGE loading buffer.

**Ubiquitination Assays**

Ubiquitination reactions were carried out by combining 1 μl of in vitro translation mix containing substrate (Cdc6-Myc9p, ³⁵S-labeled Cdc6p, or ³⁵S-labeled Sic1p), 1 μl of insect cell lysate containing kinase, 4 μl of DEAE-fractionated yeast extract (as described above), 0.2 μg of Cdc34p (as described), 1 μl of ATP-regenerating system, 1 μl of 10× reaction salts and protease inhibitors (50 mM magnesium acetate, 10 mM DTT, 5 mM PMSF, and μg/ml each leupeptin and pepstatin), and 0.5 μl of 20 mg/ml bovine ubiquitin (Sigma, St. Louis, MO). Reactions were brought to a volume of 10 μl with 20 mM HEPES, pH 7.2, 100 mM potassium acetate. Reactions were mixed and incubated at 25°C for 1 h. Translation produced some high-molecular-weight products, which have been previously observed (Verma *et al.*, 1997b) and are thought to be protein–RNA adducts resulting from inefficient translation termination. Where indicated, translation products were treated with 20 μg/ml RNase A at 37°C for 15 min, which served to remove a small amount of high-molecular-weight background and improved the sensitivity of the ubiquitination assay.

**Two-Hybrid Assay**

Strains for the two-hybrid assay were created by transforming L40 with pairs of plasmids. All strains contained the pACT-CDC4 plasmid (bearing CDC4 fused to the activation domain of GAL4; Drury *et al.*, 1997) and one of the following: BTM116, which carries the LexA DNA binding domain, pBTM-CDC6 (Drury *et al.*, 1997), and the plasmids shown in Table 4. The two-hybrid assay was carried out essentially as described in the Clontech (Palo Alto, CA) manual: saturated cultures of cells were plated onto solid selective medium and incubated for 24 h at 30°C. Seventy-five-millimeter paper filters (VWR; product number 28321-055) were used to lift cells from the plates. Filters were frozen for 10 s in liquid nitrogen, colony side up, and then thawed at room temperature. Two clean filters were placed in a Petri dish saturated with Z buffer. Thawed filters were placed on top of filters saturated with Z buffer (100 mM sodium phosphate pH 7.0, 10 mM KCl, 10

Vol. 10, October 1999

S. Elasser *et al.*

mM MgSO₄, 40 mM β-mercaptoethanol, 0.3 mg/ml 5-bromo-4-chloro-3-indolyl-β-D-galactopyranoside), and incubated at 30°C until a blue color developed in the positive control (~75 min). Filters were then dried with a heat gun, which served to stop the reaction.

## RESULTS

### The N Terminus of Cdc6p Contains Signals Important for Regulated Destruction of Cdc6p

The interaction of Cdc6p with Cdc28p might serve to target the kinase to its substrates in the replication apparatus, to ensure efficient phosphorylation of Cdc6p, or both. To investigate the outcomes of the Cdc6p–Cdc28p interaction, we have carried out deletion mutagenesis within the Cdc28p binding region of Cdc6p and have changed the serines or threonines in the six putative CDK phosphorylation sites in Cdc6p, singly and in combination, to nonphosphorylatable alanines (Figure 1; Proteome [http://www.proteome.com/databases/YPD/YPDspreadsheet.html]; Holmes and Solomon, 1996). Genes containing these mutations were tagged at the C termini with nine copies of the Myc epitope. The CDC6-Myc9 fusion as well as all of the mutant *cdc6-Myc9* constructs complement a *cdc6* null.

Cdc6p, unlike most other DNA replication proteins, is unstable in vivo after the G1–S transition (Piatti *et al.*, 1995; Drury *et al.*, 1997). Because phosphorylation regulates destruction of other proteins at G1–S, for example Sic1p, we asked whether the *cdc6* mutations affected the degradation of Cdc6p. GAL-CDC6-Myc9 wild-type and mutant strains were grown in galactose, synchronized in G2 by the addition of nocodazole, and then shifted to glucose medium to repress expression of Cdc6p. The level of Cdc6p remaining during subsequent incubation in glucose medium was followed by protein blotting. This strategy has been shown to accurately reflect events that occur when Cdc6p is expressed from its own promoter (Drury *et al.*, 1997). Cdc6Δ8–46p, lacking the entire Cdc28p binding domain, was stabilized relative to wild type (Figures 1 and 2), in agreement with recent results of others (Drury *et al.*, 1997). In addition, in comparing four smaller deletions across the N-terminal 46 amino acids, we found that only deletion of amino acids 23–36 causes significant stabilization (Figure 2).

---

**Figure 1.** Schematic of Cdc6 protein. (A) Schematic of the N-terminal 48 amino acids of Cdc6 protein. The four large brackets represent the deletions used and are labeled to show the extent of the deletions. (B) Schematic of the entire Cdc6 protein. Black bars represent CDK sites A–F, and the amino acid numbers are given for consensus threonines and serines. Gray boxes indicate the ATP binding motifs.

These results indicate that Cdc6p must enter the nucleus to become susceptible to degradation, although they do not eliminate the possibility that Cdc6p must then subsequently be exported from the nucleus for interaction with the proteasome.

### Specific Phosphorylation Sites Are Required for Cdc6p Turnover In Vivo

We next determined the stability of Cdc6 proteins containing mutations at the six putative Cdc28 phosphorylation sites. Serines and threonines at T⁷TPK, T²³PPR, S⁴³PEK, T¹³⁴PR, S³⁵⁴PTR, and S³⁷²PVK (Figure 1) were changed to alanine, singly and in combination. These sites are hereafter referred to as sites A to F. The stability of the mutant proteins was analyzed in cells arrested in G2 with nocodazole as described in Figure 2. Mutation of the individual A, B, C, and E sites caused no change in stability compared with wild type (Figure 3A). Mutation of site D led to moderately increased half-life, and mutation of site F led to significant stabilization. The effect of changing C-terminal site F was interesting and unanticipated for two reasons. First, previous deletion analysis had suggested that the signals controlling degradation were localized in the N-terminal domain (Figure 2) (Drury *et al.*, 1997). Second, site F lies in a region that is dispensable for the tight binding to Clb/Cdc28p. The C-terminal portion of Cdc6p does appear to be phosphorylated by Clb5p/Cdc28p in vitro, because phosphorylation by Cdc28p causes reduced mobility in SDS gels only for mutants containing phosphorylatable E and F sites (Figure 3B). Interestingly, phosphorylation of sites A–C does not contribute to the detected electrophoretic mobility shift. Although we cannot be sure that the shift is due to phosphorylation, this is a likely interpretation. We have shown that a *cdc6A–F* mutant is efficiently localized (Luo, Elsasser, and Campbell, unpublished data).

### Mutation of the Nuclear Localization Signal Leads to Decreased Turnover of Cdc6p

Amino acids 23–36 contain a conserved nuclear localization signal (NLS), ²⁷PLKRKKKL³³, that has been shown to be sufficient for the localization of a PLKRKKL-β-galactosidase fusion protein to the yeast nucleus. Mutation of K29 renders the NLS nonfunctional in this context (Jong *et al.*, 1996). We investigated the role of the NLS on Cdc6p stability by changing amino acids ²⁹KRKK³² to AAAA and evaluated stability in the assay already described. We found that this allele is stabilized, and the effect is similar to that generated by deleting residues 23–36. This suggests that the nuclear localization signal is the key, nonredundant motif in this region contributing to the stability of Cdc6p. Using fusions to green fluorescent protein, we have verified that the KRKK-to-AAAA mutant protein is found throughout the cell, in contrast to wild-type GFP-Cdc6p, which is found only in the nucleus (Luo, Elsasser, and Campbell, unpublished data).

Time (min) 0 10 20 40 80 160

Wild type

Δ 8 - 46

Δ 3 - 16

Δ 13 - 26

Δ 23 - 36

Δ 33 - 46

NLS

Figure 2. Stability of Cdc6p N-terminal deletion mutants. Strain BJ5459 was transformed with integrating plasmids bearing wild-type or mutant GAL-CDC6-MYC9 (GAL plasmids described in Table 3), and transformants carrying the integrated gene were isolated. Cells were grown overnight in YPR to a cell density of \(2 \times 10^6\) cells/ml. Solid galactose was added to 2%, and cells were incubated for 2 h. Nocodazole was added to a final concentration of 15 μg/ml, and cells were incubated for an additional 3 h, at which point the population was arrested, as determined by morphology and flow cytometry (our unpublished observations). Transcription was repressed by removing cells from galactose-raffinose medium and resuspending in YPD, and samples were taken at the times indicated. To determine levels of Cdc6p, extracts were prepared as described in MATERIALS AND METHODS and 10 μg of protein were electrophoresed on a 10% gel, blotted to a polyvinylidene difluoride membrane (Millipore, Bedford, MA), and probed with anti-myc monoclonal antibody 9E10. The numbers on the left refer to the amino acids deleted. NLS refers to the KRKK to AAAA mutant described in the text.

Campbell, unpublished data), indicating that the stability of cdc6-F does not derive from failure to enter the nucleus.

We also tested the stability of alleles bearing multiple mutations in the putative Cdc28 phosphorylation sites in the Cdc6p N-terminal Cdc28p-binding domain (Elsasser *et al.*, 1996). The A–C triple mutant was strongly stabilized (Figure 3A), demonstrating the importance of phosphorylation, as opposed simply to binding of Cdc28p, in Cdc6p degradation. The A + C double mutant was as stable as the triple mutant, whereas sites B + C showed a modest increase in stability, and A + B showed no difference from wild type. Thus, phosphorylation of multiple sites in the N terminus, as well as in the C terminus, contributes to the regulation of Cdc6p stability. Although the stabilization of alleles mutated at the putative Cdc28 sites only suggests an involvement of the

Cdc28p phosphorylation of Cdc6p

Time (min)

Wild type

Mutated Cdc28 Consensus Sites

A

B

1 2 3 4 5 6
- + - + - + - + - + - +
(P)-Cdc6p
Cdc6p

Figure 3. Stability of Cdc6p point mutants. (A) Strain BJ5459 was transformed with integrating plasmids bearing wild-type or mutant GAL-CDC6-MYC9. Transformants carrying the integrated gene were grown overnight in YPR to a cell density of \(2 \times 10^6\) cells/ml. Solid galactose was added to 2%, and cells were incubated for 2 h. Nocodazole was added to a final concentration of 15 μg/ml, and cells were incubated for an additional 3 h, at which point the population was arrested, as determined by morphology and flow cytometry (our unpublished observations). Transcription was repressed as described, and samples were taken at the times indicated and processed as described in Figure 2. Extract was electrophoresed on 7.5% SDS-PAGE gels, blotted to Hybond nitrocellulose (Amersham), probed with monoclonal anti-myc antibody 9E10, and developed with ECL reagents. Mutations are described in the text. (B) Cdc6p is phosphorylated at the C-terminal CDK sites. Untagged CDC6 was amplified, and \({}^{35}\) S-labeled Cdc6p was produced by in vitro translation. Phosphorylation reactions were carried out as described in MATERIALS AND METHODS. Protein was electrophoresed on a 7.5% SDS gel (18 cm long), at 30 mA for 3 h, dried, and autoradiographed. For each pair, the reaction mix was quenched with SDS-PAGE loading buffer at time zero (-) or incubated for 1 h at \(30^\circ \mathrm{C}(+)\) and then mixed with loading buffer. The allele of Cdc6p in each pair or reactions was reaction 1, wild type; reaction 2, cdc6A–F; reaction 3, cdc6B + C; reaction 4, cdc6E + F; reaction 5, cdc6A–C; reaction 6, cdc6E + F plus T356A, T370A, T371A.

kinase, below we show a direct requirement for Cdc28p using an in vitro ubiquitination system.

S. Elasser *et al.*

Figure 4. In vitro ubiquitination of Cdc6p depends on Clb/Cdc28p and Cdc34p. Lanes 1–6, Clb5p/Cdc28p dependence of Cdc6p ubiquitination. Lanes 1–3, ubiquitination of Sic1p was carried out as previously described (Verma *et al.*, 1997c). Lanes 1–3, in vitro ubiquitination assays with ${}^{35}$S-labeled Sic1p, produced by in vitro translation. All reactions contained DEAE-fractionated extract of G1-arrested RJD885 cells, purified Cdc34p, an ATP-regenerating system, salts, and ubiquitin. Lane 1, no kinase added; lane 2, insect cell lysate containing Clnp2/Cdc28p was added; lane 3, insect cell lysate containing Clb5p/Cdc28p was added. Incubations were carried out for 1 h at $25^{\circ}\mathrm{C}$, treated with $20 \mu \mathrm{g} / \mathrm{ml}$ RNase A for 15 min at $37^{\circ}\mathrm{C}$ to remove RNA-protein adducts, and quenched with SDS-PAGE loading buffer. Reactions were electrophoresed, and the gel was dried and autoradiographed for 17 h. Bands corresponding to Sic1p, phosphorylated Sic1p, and ubiquitinated Sic1p are indicated and were identified based on previous descriptions (Feldman *et al.*, 1997). Lanes 4–6, ubiquitination of Cdc6p. Reactions are exactly as described for lanes 1–3, respectively, except that in vitro-translated Cdc6-Myc9p was not labeled with ${}^{35}$S. After electrophoresing, proteins were blotted to Hybond nitrocellulose and probed with monoclonal anti-Myc antibody 9E10. Westerns were developed with the Super Signal luminescent reagent (Pierce, Rockford, IL). Lanes 7 and 8, Cdc34p dependence of Cdc6p ubiquitination. All reactions contained DEAE-fractionated extract of G1 arrested RJD885 cells, insect cell lysate containing Clb5p/Cdc28p, an ATP-regenerating system, salts, and ubiquitin. Purified Cdc34p was added (lane 8) or omitted (lane 7). Samples were processed exactly as described for the samples shown in lanes 4–6.

---

**Roles of Phosphorylation in Degradation of Cdc6p**

Several different models might explain the apparent requirement for phosphorylation in the degradation of Cdc6p. Because mutations in the nuclear localization site and the phosphorylation sites each generate stable alleles of CDC6, mutation of the phosphorylation sites could cause a localization defect and consequently stability. Alternatively, phosphorylation of Cdc6p might allow recognition of Cdc6p by SCF-Cdc4p, ubiquitination, and targeting to the 26S proteasome (Drury *et al.*, 1997; Sanchez *et al.*, 1999). To distinguish between these models, we analyzed ubiquitination of Cdc6p in an in vitro reaction that has been shown to faithfully mimic in vivo events (Feldman *et al.*, 1997; Skowyra *et al.*, 1997; Verma *et al.*, 1997c). Ubiquitination of substrates in vitro generally requires the presence of ubiquitin, ubiquitin-activating enzyme (E1), ubiquitin-conjugating enzyme (E2), and ubiquitin ligase (E3), the last of which is typically substrate specific. Cdc34p, an E2, and SCF-Cdc4p, an E3 comprising Cdc4p, Cdc53p, Skp1p, and probably Rbx1p, target the Cdc28p inhibitor Sic1p for degradation at the G1-S phase transition (Schwob *et al.*, 1994; Bai *et al.*, 1996; Patton *et al.*, 1998; Kamura *et al.*, 1999; Skowyra *et al.*, 1999). This reaction has recently been recapitulated in vitro (Feldman *et al.*, 1997; Skowyra *et al.*, 1997; Verma *et al.*, 1997c; Skowyra *et al.*, 1999), and we have adapted this system to the study of Cdc6p. Because there is strong evidence that Cdc6p is degraded at the G1-S transition, we used fractionated extracts prepared from cells arrested in G1 by suppressing Cln expression (Deshaiers *et al.*, 1995). These extracts are devoid of all Cdc28 kinase activity, thus allowing us to test for a Cdc28p requirement in the ubiquitination of Cdc6p. In addition, the DEAE fraction recovered from these extracts contains SCF-Cdc4p (E3) but is devoid of Cdc34p (E2), allowing us to ask whether Cdc34p is required for Cdc6p ubiquitination (Verma *et al.*, 1997c; Skowyra *et al.*, 1999).

For our experiments, Sic1p was used as a positive control substrate for the ubiquitination reactions. Ubiquitination of Sic1p was first shown to require ubiquitin, the DEAE fraction containing Uba1p (E1) and SCF-Cdc4p (E3), Cdc34p (E2), and Cln2p/Cdc28p kinase (Verma *et al.*, 1997c). When in vitro-translated ${}^{35}$S-Sic1p and Cdc6-Myc9p are added to this reconstituted system, Cdc6-Myc9p, like Sic1p, is converted to more slowly migrating conjugates (Figure 4, lanes 2 and 6), indicating that Cdc6p is ubiquitinated in this reconstituted system. Although we have not strictly demonstrated that the modification of Cdc6p is due to ubiquitin, this is the most likely explanation of these results, because (1) modification of substrate by ubiquitin-like proteins has not been shown to lead to substrate degradation; and (2) no other modifications leading to such large shifts in mobility and dependent on Cdc34p and/or Cdc4p (see Figures 4 and 5) have been described.

Importantly, neither Sic1p nor Cdc6p is ubiquitinated in the absence of added Cdc28 kinase (Figure 4, lanes 1 and 4). However, there is a significant difference between the kinases required for the two different substrates. As reported previously, Sic1p ubiquitination is preferentially stimulated by Cln2p/Cdc28p (Figure 4, compare lanes 2 and 3) (Verma *et al.*, 1997c), even when sufficient Clb5/Cdc28 activity is present to generate a quantitative electrophoretic shift of Sic1p (Figure 4, lane 3; Feldman *et al.*, 1997). In contrast, Cdc6p ubiquitination is stimulated preferentially by Clb5p/Cdc28p rather than by Cln2p/Cdc28p (Figure 4, compare lanes 5 and 6). (We cannot resolve phosphorylated and unphosphorylated forms of Cdc6p tagged with the Myc9 epitope.) Although it is difficult to compare the specific activity of the two kinase preparations, the same amounts of kinase were added to both the Sic1p and Cdc6p reactions, and thus the reversal of efficiency represents an accurate qualitative measure of the relative efficiency of the respective kinases.

tive kinase on each substrate. We conclude that although Sic1p ubiquitination is preferentially mediated by Cln2/Cdc28p, Cdc6p ubiquitination is preferentially mediated by Clb5/Cdc28p. This difference in cyclin dependence is entirely consistent with the observations that Cdc6p is degraded during S phase when Clb5p/Cdc28p is active and is also consistent with our observation that Cdc6p interacts preferentially with Clb/Cdc28ps compared with Cln/Cdc28ps (Elsasser *et al.*, 1996). We do observe that Cln2p/Cdc28p can support ubiquitination to some degree in vitro, which was unexpected because Cdc6p is stable during most of G1. This suggests that high levels of Cln2p/Cdc28p may support in vivo degradation of Cdc6p. We propose that phosphorylation of Cdc6p, primarily by a cyclinB/Cdc28 kinase, although perhaps also in part by Cln/Cdc28 kinases, stimulates targeting of Cdc6p to the ubiquitination apparatus.

CDC4 and CDC34 Are Required for Phosphorylation-dependent Ubiquitination of Cdc6p In Vitro

Recent demonstration that Cdc6p is stabilized in a *cdc4sic1Δ* strain suggested that SCF-Cdc4p targets phosphorylated Cdc6p for ubiquitination rather than acting indirectly by degrading Sic1p and thus activating Clb/Cdc28 kinase. Our results confirm the direct targeting of Cdc6p by SCF-Cdc4p action. As shown in Figure 4, lanes 7 and 8, formation of high-molecular-weight conjugates of Cdc6p is dependent on Cdc34p. To compare Cdc6p ubiquitination in the presence and absence of Cdc4p, fractionated extracts were prepared from both wild-type and *cdc4* cells, and the fate of Cdc6p was followed by labeling with \[^{35}\]S in the in vitro translation reaction. As in Figure 4, the appearance of Cdc6 conjugates depends on addition of Cdc34p (compare lanes 4 and 5). Furthermore, Cdc6 conjugates were not obtained when using fractionated *cdc4* extracts (Figure 5, compare lanes 5 and 10), but if insect cell lysate containing recombinant Cdc4p was added, ubiquitination of Cdc6p was restored (Figure 5, compare lanes 10 and 12). Naturally, Cdc6p modification still required addition of Cdc34p (Figure 5, compare lanes 11 and 12). We conclude that the slowly migrating species are ubiquitinated Cdc6p, because their appearance depends on Cdc34p and Cdc4p. Although a requirement for Cdc34p and Cdc4p in this reaction could derive from a requirement to degrade Sic1p, we have determined that the Clb5/Cdc28 kinase added to these reactions is in considerable excess over Sic1p. We therefore propose that Cdc34p and Cdc4p are directly required for Cdc6p destruction and that Cdc4p is required for the recognition of Cdc6p as a substrate for ubiquitination. This model is consistent with the observations that Cdc6p and Cdc4p interact in a two-hybrid system (Drury *et al.*, 1997) and that Cdc6p is stable in G2 phase cells in a *cdc4sic1Δ* strain (Sanchez *et al.*, 1999).

Mapping of the Determinants in Cdc6p Required for Ubiquitination In Vitro

Having demonstrated that Cdc6p could be ubiquitinated in the Clb5p/Cdc28p-, Cdc34p-, and Cdc4p-dependent in vitro reaction, we used this assay to evaluate whether the stability of our *cdc6* mutants was anticorrelated with ability to be ubiquitinated in vitro. In each ubiquitination reaction,

Cdc28p phosphorylation of Cdc6p

Wild type | cdc4
--- | ---
- - + + | - - + + + +
- + - + + | - + - + - +
- + - - + | - + - - + +

Clb5p/Cdc28p
Cdc34p
Cdc4p

Ub-Cdc6p
(P)-Cdc6p
Cdc6p

1 2 3 4 5 6 7 8 9 10 11 12

Figure 5. In vitro ubiquitination of Cdc6p is dependent on Cdc4p. \[^{35}\]S-Labeled Cdc6p was prepared by in vitro translation as described in MATERIALS AND METHODS and used as a substrate for in vitro ubiquitination reactions. Lane 1 contains the input \[^{35}\]S-labeled Cdc6p without extract. Extract prepared from RJD885 cells (wild type for ubiquitination components) was used in the ubiquitination reactions in lanes 2–6. Extract from RJD893 cells (*cdc4* and isogenic to RJD885) was used in reactions in lanes 7–12. The complete ubiquitination reaction (lanes 5 and 10) included DEAE-fractionated extract of G1-arrested cells, purified Cdc34p, insect cell lysate containing Clb5p/Cdc28p kinase, an ATP-regenerating system, salts, and ubiquitin. The reactions in lanes 6, 11, and 12 were supplemented with recombinant Cdc4p purified from insect cells. Cdc34p and Clb5p/Cdc28p were omitted where indicated.

icipation of SCF-Cdc4p was controlled for by demonstrating dependence on Cdc34p (Figure 6). The Cdc6Δ8–46p, which is stable in vivo (Figure 2; Drury *et al.*, 1997), was not ubiquitinated, confirming that there are determinants in the N terminus required for in vitro ubiquitination. Deletion of amino acids 3–16 or 13–26 did not appear to reduce ubiquitination relative to wild type, and the mutants show rapid turnover (Figure 2). Deletion of amino acids 23–36, which deletes the NLS and one consensus CDK site and stabilizes Cdc6p, consistently reduced but did not abolish ubiquitination. However, the stabilized NLS (KRKK) mutant was efficiently ubiquitinated in vitro (Figure 6, lanes labeled NLS). This supports our proposition that the NLS allele is stable in vivo because of improper localization and argues against a role for any of the lysines in constituting a ubiquitin acceptor site. Surprisingly, we found that deletion of amino acids 33–46 significantly and reproducibly lowered but did not abolish ubiquitination. This was unanticipated because this mutant is as rapidly degraded as wild-type Cdc6p (Figure 2) and suggests that there may be at least one additional, Cdc4p independent turnover pathway (also see Figure 7).

We also tested the putative phosphorylation site mutants for ubiquitination. As shown in Figure 6, mutations in individual CDK sites did not abolish ubiquitination of Cdc6p. (Normal ubiquitination of the *cdc6-A* mutant was observed in other experiments, and the weak signal here derives from an inefficient in vitro translation reaction [our unpublished results].) Because the *cdc6-D* and *cdc6-F* mutants were stabi-

S. Elasser *et al.*

![Image Description](#)

Figure 6. In vitro ubiquitination of Cdc6p mutants. Wild-type and mutant Cdc6-Myc9p were produced by in vitro translation as described in MATERIALS AND METHODS, and ubiquitination reactions were carried out as described. For each pair of samples, the reaction in the right lane was a complete ubiquitination reaction (DEAE-fractionated extract of G1 arrested RJD885 cells, partially purified Cdc34p, insect cell lysate containing Clb5p/Cdc28p kinase, an ATP-regenerating system, salts, and ubiquitin). For the reactions run in the left lane, Cdc34p was omitted. After the ubiquitination reaction, samples were treated with RNase (as in Figure 4), mixed with SDS-PAGE loading buffer, and subjected to electrophoreses on 7.5% gels. Proteins were blotted to Hybond nitrocellulose and probed with 9E10 antibody, and the Western blots were developed with the Super Signal luminescent reagent (Pierce).

lized (Figure 3) and localize to the nucleus (Luo, Elsasser, and Campbell, unpublished data), it was surprising that they were ubiquitinated in vitro. In vivo, the mutant proteins may be protected from ubiquitination or protected from degradation at a step downstream of SCF-Cdc4p mediated ubiquitination, such as export from the nucleus. In addition, either of these sites may target Cdc6p for degradation by a different pathway (see Figure 9).

Although mutation of individual CDK sites did not affect in vitro ubiquitination, strong effects were seen in multiple mutants. Mutation of all three N-terminal CDK sites, A–C, which generated a highly stabilized allele, also abolished ubiquitination completely. The stabilized double mutants A + C and B + C were also significantly impaired for ubiquitination in vitro (Figures 6 and 3). Simultaneous mutation of the C-terminal sites E and F (Figure 6) did not reduce ubiquitination, although the latter mutation stabilized Cdc6p in vivo (Figure 3). In summary, there is a correlation among phosphorylation of N-terminal sites, in vitro ubiquitination, and rapid degradation in G2 phase. In contrast, although C-terminal CDK sites are important for in vivo degradation, phosphorylation of C-terminal sites is not important for ubiquitination by the SCF-Cdc4p in vitro system.

**Two-Hybrid Analysis of the Interaction of Cdc6p with Cdc4p**

Because phosphorylation of Sic1p is required for recognition by Cdc4p in SCF-Cdc4p (Feldman *et al.*, 1997; Skowyra *et al.*, 1997), we wanted to test whether the observed failure in

ubiquitination of CDC6 alleles bearing mutations in the Cdc28 consensus sites was a result of a failure in recognition of Cdc6p by Cdc4p. To evaluate the interaction of mutant Cdc6 proteins with Cdc4p, we used the two-hybrid system, in which the interaction of Cdc6p and Cdc4p was initially discovered (Drury *et al.*, 1997). In summary, only mutants that bind Cdc4p in the two hybrid assay are ubiquitinated in vitro (Figures 6 and 7). This suggests that the physical interaction of Cdc6p with Cdc4p is essential for modification by this pathway. However, interaction of Cdc6p with Cdc4p is not sufficient, because mutant A + C interacts with Cdc4p in the two-hybrid assay but fails in ubiquitination in vitro (Figure 7). In addition, in vivo degradation is not strictly dependent on a tight Cdc4p interaction, because mutant Δ33–46 interacts weakly with Cdc4p in the two-hybrid assay and is poorly ubiquitinated in vitro but is degraded as efficiently as wild type in vivo. The role of Cdc6 amino acids 33–46 in Cdc4p binding is not clear, but it is clear that serine 43 (CDK site C) is not required for the interaction (see Figure 7). The degradation of mutant Δ33–46 may be rationalized if Cdc4p-mediated ubiquitination is more robust in vivo than in vitro. Alternately, a Cdc4-independent targeting system may be at work in vivo.

The C-terminal sites again fall into a separate class from the N-terminal sites. Mutation of sites D and F does not affect Cdc4p binding (Figure 7; our unpublished data) or ubiquitination (Figure 6), yet the mutants D and F are stabilized (Figure 3). The ability of these mutants to bind Cdc4p strengthens the conclusion that phosphorylation of the C-

Cdc28p phosphorylation of Cdc6p

Cdc28p, and on the other, Cdc6p is a substrate of Cdc28p. These two modes of interaction may have different consequences in vivo. To investigate whether tight binding of Cdc6p to Cdc28p as well as phosphorylation of Cdc6p by Cdc28p is required for rapid degradation of Cdc6p, we have analyzed the interaction of Cdc6Δ8–46p, Cdc6Δ3–16p, Cdc6Δ13–26p, Cdc6Δ23–36p, and Cdc6Δ33–46p with Clbp/Cdc28p, using the Myc-tagged alleles described in Figure 1. Extracts of cells constitutively expressing the tagged mutant genes were immunoprecipitated with anti-myc 9E10 monoclonal antibody, and the immunoprecipitates were probed for the presence of Cdc28p by Western blotting. As shown in Figure 8, Cdc28p was present only in wild-type Cdc6-Myc9p immunoprecipitates. Failure of the deletion mutants to interact could be due either to altered structure of the deleted proteins or to deletion of signals important for interaction. We prefer the latter interpretation, because all mutants support growth in the absence of wild type Cdc6p (see below), showing that at least the portion of the protein required for replication is correctly folded. Thus, although phosphorylation by Cdc28p is required for degradation, tight binding of Cdc28p by Cdc6p is not required for targeting to the ubiquitination machinery.

## Another Degradation Pathway?

At least two results presented here do not fit the model that SCF-Cdc4p accounts for all Cdc6p turnover. First, cdc6Δ33–46p is unstable even when it is highly overproduced, but it does not bind Cdc4p. Second, mutants D and F are stable but are ubiquitinated in vitro and bind Cdc4p efficiently. We wished to use a simple genetic test for the involvement of degradation pathways other than the Cdc4 pathway in Cdc6p turnover. Constitutive expression of Cdc6p in *cdc4-1* cells, as well as constitutive expression of Sic1p in *cdc4* cells has been shown to be lethal, suggesting that the degradation pathway is saturated and that degradation of Cdc6p and Sic1p is blocked (Bai *et al.*, 1996; Sanchez *et al.*, 1999). We used a similar strategy to test whether the APC, the other known ubiquitin ligase that targets cell cycle proteins (Peters, 1999), is involved in Cdc6p degradation. In choosing an allele for study, we considered the report that *S. cerevisiae cdc16-264* mutants (but not all *cdc16* mutants) overreplicate. This controversial result suggests that the *cdc16-264* allele affects DNA replication (Heichman and Roberts, 1996, 1998; Pichler *et al.*, 1997). Unlike the case of *cdc4-1* mutants, overexpression of wild-type Cdc6p did not reduce the ability of the *cdc16* mutant to form colonies. However, Cdc28 consensus site mutants *cdc6A–C* and *cdc6A–F* mutants showed a dosage-dependent dominant lethal effect when introduced into the *cdc16* mutant. Analysis of the individual putative CDK site mutants revealed the surprising result that mutation of the C-terminal site F was sufficient to cause the dominant lethal phenotype, whereas none of the other single site mutants gave this effect (Figure 9). These results might suggest that enhanced levels of Cdc6 are sufficient to saturate the Cdc16 pathway. However, the stable Cdc6-Δ8–46 protein did not give a dominant negative effect (our unpublished observations). The stabilized alleles that appear to interact with the Cdc16 pathway belong to two classes of stabilized mutants, those that are ubiquitinated in vitro and those that are not, suggesting that interaction of Cdc6 with the Cdc4 pathway is not required to interact with the Cdc16

**Figure 7.** Two-hybrid interaction of Cdc4p with wild-type and mutant Cdc6p. Strains were prepared as described in MATERIALS AND METHODS. All strains carry a plasmid bearing a truncated allele of *CDC4* fused to the activation domain of GAL4 (Drury *et al.*, 1997). Strains also carry BTM116-based plasmids, which bear the lexA DNA binding domain fused to wild-type and mutant alleles of *CDC6*, as indicated in the schematic. β-Galactosidase reactions were carried out as described in MATERIALS AND METHODS.

terminal sites regulates a different part of the destruction pathway than phosphorylation of the N-terminal sequences, or that C-terminal phosphorylation of Cdc6p regulates a different degradation pathway entirely. In *Schizosaccharomyces pombe*, two different F box proteins similar to Cdc4p are critical for maintaining *cdc18+* at the proper physiological levels, but whether their interaction with *cdc18+* is regulated by differential phosphorylation has not been studied (Jallepalli *et al.*, 1997; Jallepalli *et al.*, 1998).

### Tight Binding of Cdc6p to Cdc28p Is Not Required for Rapid Degradation

The results of our earlier studies and those presented here suggest that there may be two modes of interaction of Cdc6p with Cdc28p: on the one hand, Cdc6p binds tightly to

S. Elasser *et al.*

**EXTRACTS**

![Extracts Image](https://i.imgur.com/extracts_image.png)

**IMMUNOPRECIPITATIONS**

![Immunoprecipitations Image](https://i.imgur.com/immunoprecipitations_image.png)

Figure 8. Interaction of mutant Cdc6p with Cdc28p. Strain BJ2168 was transformed with plasmids pELS71 (wild type), pELS72 (∆3–16), pELS73 (∆13–26), and pELS75 (∆33–46). Cells were grown overnight in synthetic medium prepared with raffinose and lacking leucine. Expression of Cdc6 proteins was induced for 4 h with 2% galactose. Extracts were prepared, and immunoprecipitations were carried out as described in MATERIALS AND METHODS. Top panel, 10 μg of protein was electrophoresed on a 10% gel, blotted to a polyvinylidene difluoride membrane (Millipore), and probed with anti-myc monoclonal antibody 9E10. The apparent molecular mass of Cdc6-Myc9p is 120 kDa. Bottom two panels, 9E10 immunoprecipitates were electrophoresed on a 12.5% gel and blotted to a polyvinylidene difluoride membrane. The top half of the membrane was probed with anti-myc monoclonal antibody 9E10 (middle panel), and the bottom half of the membrane (bottom panel) was probed with anti-Cdc28p polyclonal antibody RAA1. Protein blots were developed with ECL reagents (Amersham). The lane on the left labeled C is a control for the Cdc28p antibody reaction in which extract was run on the gel without previous immunoprecipitation.

pathway. These data may indicate that the APC may play a role in the regulation of Cdc6p levels and phosphorylation may govern the interaction. However, they would also be explained if Cdc6p participates in the same process as the APC and if overproduction of these Cdc6 mutants inhibited that process.

**DISCUSSION**

The rapid degradation of yeast Cdc6p at G1–S is conserved for homologues in other eukaryotes, although not in all,

suggesting that periodic degradation of Cdc6p is important for regulation of cell cycle progression (Drury *et al.*, 1997; Williams *et al.*, 1997; Hua and Newport, 1998; Saha *et al.*, 1998). In this work we have explored the role of phosphorylation of Cdc6p in its regulated destruction during the yeast cell cycle. By using an in vitro system prepared from cells arrested at the G1–S boundary, and thus avoiding multiple ambiguities introduced by evaluating the degradation of Cdc6p in vivo, we have demonstrated that Cdc6p is ubiquitinated and that Cdc34p and Cdc4p, which also participate in Cdc6p degradation in vivo, are directly involved in ubiquitination of Cdc6p. Integrity of the three N-terminal CDK phosphorylation sites appears to be required for Cdc4p binding and for ubiquitination, suggesting that phosphorylation is required for targeting Cdc6p to the ubiquitination machinery. We have also found that Clb/Cdc28ps are more effective at targeting Cdc6p for ubiquitination than are Clb/Cdc28ps, but that Clb/Cdc28p can target ubiquitination. Thus, at a point at which Clb/Cdc28 complexes are present at high concentration they could contribute to Cdc6p turnover.

Characterization of Sic1p ubiquitination in vitro generated a model of protein degradation in which phosphorylation of a substrate by a cell cycle kinase triggered ubiquitination of the substrate and consequent degradation by the proteasome (Bai *et al.*, 1996; Feldman *et al.*, 1997). Here we demonstrate that ubiquitination of Cdc6p is carried out in a similar manner, in that we have generated a class of mutants that are stabilized during G2 in vivo and defective in Cdc4-mediated ubiquitination in vitro. However, the regulation of Cdc6p turnover appears to be more complex than that of Sic1p. In addition to the class of mutants in which SCF-Cdc4p targeting defects and stabilization in G2 in vivo are correlated, we also discovered a second class of mutants in which SCF-Cdc4p targeting was intact and yet the proteins were stable (NLS, cdc6-D, and cdc6-F), as well as a third class of mutants in which SCF-Cdc4p targeting was defective but mutants were still unstable (cdc6∆33–46). The stability of the NLS mutant likely derives from a failure to concentrate in the nucleus. This suggests that in vivo at least one component of the destruction apparatus may be active only on Cdc6p once it is in the nucleus. This component could be merely the putative essential kinase and therefore does not rule out the possibility that the phosphorylated Cdc6 protein is normally exported to the cytoplasm for degradation by the proteasome.

Failure of nuclear localization does not account for the stability of the C-terminal cdc6-D and cdc6-F, however (Luo, Elsasser, and Campbell, unpublished results). The synergistic growth defect achieved by overproducing cdc6-F in a cdc16 mutant (Figure 9) that is defective in the APC-directed destruction apparatus may indicate that the APC also contributes to regulation of Cdc6p levels in the cell. Two pathways for Cdc6p degradation could be reconciled if there are two populations of Cdc6p (perhaps bound to chromatin and unbound) each sensitive to different destruction mechanisms; or two pathways might act on the same population but at different points in the cell cycle. A second possibility is that the APC might just be needed to degrade some other replication protein whose accumulation kills the cells in the presence of excess Cdc6p. A third alternative is that the cdc6-F mutant has a dominant negative effect on a step in

Molecular Biology of the Cell

Figure 9. Overexpression of cdc6-Fp in a cdc16 mutant that replicates its DNA is lethal. The respective phosphorylation site cdc6-A, -B, -C, -D, -E, or -F mutant genes were introduced into strain JLC1402. The transformants were streaked on YPD (left) or YPRG (right) plates to induce the cdc6 mutant proteins. Plates were incubated at 30°C for 4 d. In the course of carrying out these studies we discovered that JLC1402 is a diploid, cdc16-264/+ heterozygote and not a cdc16-264 haploid as originally thought. All experiments using this mutant were repeated in the appropriate haploid, and the same results were obtained.

![Image Description](#)

Glucose Galactose

mitosis that also requires full function of the APC, either the metaphase-to-anaphase transition or mitotic exit.

The third class of mutant fails to interact with SCF-Cdc4p but is nevertheless unstable: cdc6Δ33–48. The instability could be mediated by another degradation pathway that degrades Cdc6p that is not associated with Cdc28p, or the residual ubiquitination may be sufficient to target this mutant for degradation in vivo.

Several other proteins involved in G1-S control have been shown to be targeted for degradation by phosphorylation: Sic1p, Far1p, and the G1 cyclins (Willems et al., 1996; Henchoz et al., 1997; Verma et al., 1997a). For these G1 substrates of the 26S proteasome, phosphorylation at multiple sites is required for degradation, and this is also true for the S phase substrate Cdc6p. As with Sic1p, only a subset of the CDK sites on Cdc6p is required for ubiquitination, and no single site is sufficient. As observed for Sic1p and Far1p but not the G1 cyclins, phosphorylation of three N-terminal Cdc6p sites seems to be essential specifically for binding of Cdc4p, because there is a correlation between Cdc6p phosphorylation site mutants that cannot be phosphorylated and those that fail to bind Cdc4p. That is, SCF-Cdc4p targets N-terminally phosphorylated Cdc6p for destruction. The involvement of the same SCF complex in the destruction of Sic1p and Cdc6p suggests that the coordinate control of these processes was a design of evolution. However, there is a significant difference between the kinases involved in the targeting of Sic1p, Far1p, and Cdc6p for destruction. Sic1p and Far1p are preferentially targeted to SCF-Cdc4p by Cln2p/Cdc28p, whereas Cdc6p is preferentially targeted by Clb5p/Cdc28p, at least in vitro. The sequential activation of the Clns and Clbs might allow for staggering of the degradation of Sic1p and Cdc6p. Thus, Cdc6p would not be degraded (and the prereplicative complex thereby inactivated) until Sic1p is removed and the Clb/kinases are able to execute initiation

of replication. It is important to note that although Cdc6p is stable in elutriated G1 cells, and thus in the presence of active Cln2p/Cdc28p, Cdc6p is unstable in cells exposed to mating pheromone, which causes inactivation of the G1 cyclins and exit from the cell cycle (Drury et al., 1997; Jeoung et al., 1999). Destruction of Cdc6p in cells preparing to mate could be phosphorylation independent, or degradation may be mediated by the MAP kinases that are induced by the stimulation of the pheromone receptor.

Studies of the putative fission yeast Cdc6p homologue cdc18+ have shown that phosphorylation regulates cdc18+, and that high-molecular-weight conjugates of cdc18+ can form in proteasome mutants, but there is some uncertainty as to which ubiquitin ligase mediates this process in vivo, because both pop1+ (a Cdc4p homologue) and sud1+ have been implicated (Jallepalli et al., 1997, 1998; Kominami and Toda, 1997). Stabilizing cdc18+ was found to cause unscheduled initiation of DNA replication (Jallepalli et al., 1998). However, stabilizing Cdc6p does not cause the same phenomenon in *S. cerevisiae*, pointing up a substantial difference in the regulation. Therefore, degradation of Cdc6p cannot be the sole factor that restrains replication initiation to once per cell cycle. For such a key process as high-fidelity replication of the chromosome, it is not unexpected that there would be redundancy in the mechanism that prevents rereplication, and arguably the fact that the overproduction of the Cdc6p homologue in *S. pombe* causes overreplication is more surprising than the converse (Jallepalli and Kelly, 1996; Jallepalli et al., 1998). In *S. cerevisiae*, other candidates for proteins that may regulate rereplication are the Mcms, which have been demonstrated to be compartmentalized over the course of the cell cycle, and Dbf4p, which is unstable after the G1-S transition (Dixon, Yang, and Campbell, unpublished data). Assuming that rereplication is blocked by the destruction of multiple targets, more than one pathway may act on these

targets. If it proves to be true that DNA overreplicates in some *cdc16* mutants (Heichman and Roberts, 1996, 1998; Pichler *et al.*, 1997), and we think that the evidence weighs in favor of a modest rereplication phenotype for these particular APC mutants, one or more DNA replication proteins may be targeted for degradation by the APC. It has been reported that Cdc6p is not stabilized in nocodazole-arrested APC mutants (*cdc16* and *cdc23*), which would appear to rule out Cdc6p as an APC substrate (Drury *et al.*, 1997). However, it remains possible that Cdc6p in the preRC, present until the G1–S transition and absent in nocodazole arrest, might be susceptible to APC-mediated degradation. This model fits the results shown in Figure 9 in which a stabilized allele of Cdc6p in a *cdc16* mutant resulted in lethality (although other explanations for the result are possible). We expect to find one of three things to be true in such cells: first, they may suffer from unregulated initiation; second, initiation itself may be impaired, which would in turn suggest a role for both the APC and the phosphorylation of Cdc6p in initiation; and third, cells may be arrested in G2 or M as a consequence of failing to degrade Cdc6p and other DNA replication proteins (Bueno and Russell, 1992). Whatever the result, we expect that characterization of these *cdc6* mutants in different backgrounds will lead to a greater understanding of the role of both Cdc6p and the APC in the control of DNA replication in *S. cerevisiae*.

### ACKNOWLEDGMENTS

We thank Ray Deshaies and members of his laboratory for strains, discussions, and critical reading of the manuscript. The experiment in Figure 5 was carried out in the Deshaies laboratory.

### REFERENCES

Bai, C., Sen, P., Hofmann, K., Ma, L., Goebl, M., Harper, J.W., and Elledge, S.J. (1996). SKP1 connects cell cycle regulators to the ubiquitin proteolysis machinery through a novel motif, the F-box. Cell 86, 263–274.

Banerjee, A., Gregori, L., Xu, Y., and Chau, V. (1993). The bacterially expressed yeast CDC34 gene product can undergo autoubiquitination to form a multiubiquitin chain-linked protein. J. Biol. Chem. 268, 5668–5575.

Bell, S.P., Kobayashi, R., and Stillman, B. (1993). Yeast origin recognition complex functions in transcription silencing and DNA replication. Science 262, 1844–1849.

Bell, S.P., and Stillman, B. (1992). ATP-dependent recognition of eukaryotic origins of DNA replication by a multiprotein complex. Nature 357, 128–133.

Blow, J.J., and Laskey, R.A. (1988). A role for the nuclear envelope in controlling DNA replication within the cell cycle. Nature 332, 546–548.

Bueno, A., and Russell, P. (1992). Dual functions of CDC6—a yeast protein required for DNA replication also inhibits nuclear division. EMBO J. 11, 2167–2176.

Dahmann, C., Diffley, J.F.X., and Nasmyth, K.A. (1995). S phase-promoting cyclin-dependent kinases prevent rereplication by inhibiting the transition of replication origins to a prereplicative state. Curr. Biol. 5, 1257–1269.

Deshaies, R.J., Chau, V., and Kirschner, M. (1995). Ubiquitination of the G(1) cyclin Cln2p by a Ccd34p-dependent pathway. EMBO J. 14, 303–312.

Diffley, J.F.X., and Cocker, J.H. (1992). Protein-DNA interactions at a yeast replication origin. Nature 357, 169–172.

Dirick, L., Bohm, T., and Nasmyth, K. (1995). Roles and regulation of Cln-Cdc28 kinases at the start of the cell cycle of *Saccharomyces cerevisiae*. EMBO J. 14, 4803–4813.

Drury, L.S., Perkins, G., and Diffley, J.F.X. (1997). The Cdc4/34/53 pathway targets Cdc6p for proteolysis in budding yeast. EMBO J. 16, 5966–5976.

Elsasser, S., Lou, F., Wang, B., Campbell, J.L., and Jong, A. (1996). Interaction between yeast Cdc6 protein and B-type cyclin/Cdc28 kinases. Mol. Biol. Cell 7, 1723–1735.

Feldman, R.M.R., Correll, C.C., Kaplan, K.B., and Deshaies, R.J. (1997). A complex of Cdc4p, Skp1p, and Cdc53p/Cullin catalyzes ubiquitination of the phosphorylated Cdk inhibitor Sic1p. Cell 91, 221–230.

Heichman, K.A., and Roberts, J.M. (1996). The yeast *CDC16* and *CDC27* genes restrict DNA replication to once per cell cycle. Cell 85, 39–48.

Heichman, K.A., and Roberts, J.M. (1998). *CDC16* controls initiation at chromosome replication origins. Mol. Cell 1, 457–463.

Henchoz, S., Chi, Y., Catarin, B., Herskowitz, I., Deshaies, R.J., and Peter, M. (1997). Phosphorylation-dependent and ubiquitin-dependent degradation of the cyclin-dependent kinase inhibitor Far1p in budding yeast. Genes & Dev. 11, 3046–3060.

Holmes, J.K., and Solomon, M.J. (1996). A predictive scale for evaluating cyclin-dependent kinase substrates—a comparison of p34(CDC2) and p33(CDK2). J. Biol. Chem. 271, 25240–25246.

Hua, H., and Newport, J. (1998). Identification of a preinitiation step in DNA replication that is independent of origin recognition complex and cdc6, but dependent on cdk2. J. Cell Biol. 140, 271–281.

Jallepalli, P.V., Brown, G.W., Muzifalconi, M., Tien, D., and Kelly, T.J. (1997). Regulation of the replication initiator protein p65(cdc18) by Cdk phosphorylation. Genes & Dev. 11, 2767–2779.

Jallepalli, P.V., and Kelly, T.J. (1996). *RUM1* and *CDC18* link inhibition of cyclin-dependent kinase to the initiation of DNA replication in *Schizosaccharomyces pombe*. Genes & Dev. 10, 541–552.

Jallepalli, P.V., Tien, D., and Kelly, T.J. (1998). *sud1+* targets cyclin-dependent kinase-phosphorylated Cdc18 and Rum1 protein for degradation and stops unwanted diploidization in fission yeast. Proc. Natl. Acad. Sci. USA 95, 8159–8164.

Jeong, D.I., Oehlen, L.J., and Cross, F.R. (1999). Cln3-associated kinase activity in *Saccharomyces cerevisiae* is regulated by the mating factor pathway. Mol. Cell. Biol. 18, 433–441.

Johnston, M., and Davis, R. (1984). Sequences that regulate the divergent GAL1-GAL10 promoter in *Saccharomyces cerevisiae*. Mol. Cell. Biol. 4, 1440–1448.

Jong, A., Young, M., Chen, G.C., Zhang, S.Q., and Chan, C. (1996). Intracellular location of the *Saccharomyces cerevisiae CDC6* gene product. DNA Cell Biol. 15, 883–895.

Kamura, T., *et al.* (1999). Rbx1, a component of the VHL tumor suppressor complex and SCF ubiquitin ligase. Science 284, 657–661.

Kominami, K., and Toda, T. (1997). Fission yeast WD-repeat protein Pop1 regulates genome ploidy through ubiquitin-proteasome-mediated degradation of the CDK inhibitor Rum1 and the S-phase initiator Cdc18. Genes & Dev. 11, 1548–1560.

Nasmyth, K. (1993). Control of the yeast cell cycle by the Cdc28 protein kinase. Curr. Opin. Cell Biol. 5, 166–179.

Nasmyth, K. (1996). At the heart of the budding yeast cell cycle. Trends Genet. 12, 405–412.

Newlon, C.S. (1997). Putting it all together. Cell 91, 717–720.

Patton, E.E., Willems, A.R., and Sa, D. (1998). Cdc53 is a scaffold protein for multiple cdc34/skp1/F-box protein complexes. Genes & Dev. 12, 692–705.

Peters, J.M. (1999). Subunits and substrates of the anaphase-promoting complex. Exp. Cell Res. 248, 339–349.

Piatti, S., Lengauer, C., and Nasmyth, K. (1995). Cdc6 is an unstable protein whose *de novo* synthesis in G1 is important for the onset of S phase and for preventing a “reductional” anaphase in the budding yeast *Saccharomyces cerevisiae*. EMBO J. 14, 3788–3799.

Pichler, S., Piatti, S., and Nasmyth, K. (1997). Is the yeast anaphase promoting complex needed to prevent rereplication during G(2)-phase and M-phase. EMBO J. 16, 5988–5997.

Rowley, A., Cocker, J.H., Harwood, J., and Diffley, J.F.X. (1995). Initiation complex assembly at budding yeast replication origins begins with the recognition of a bipartite sequence by limiting amounts of the initiator, ORC. EMBO J. 14, 2631–2641.

Saha, P., Chen, J., Thome, K.C., Lawlis, S.J., Hou, Z.-H., Hendricks, M., Parvin, J.D., and Dutta, A. (1998). Human CDC6/Cdc18 associates with Orc1 and Cyclin-cdk and is selectively eliminated from the nucleus at the onset of S phase. Mol. Cell. Biol. 18, 2758–2767.

Sanchez, M., Calzada, A., and Bueno, A. (1999). The Cdc6 protein is ubiquitinated *in vivo* for proteolysis in *Saccharomyces cerevisiae*. J. Biol. Chem. 274, 9092–9097.

Schwob, E., Bohm, T., Medenhall, M.D., and Nasmyth, K. (1994). The B-type cyclin kinase inhibitor p40 (*SIC1*) controls the G1 to S transition in *Saccharomyces cerevisiae*. Cell 79, 233–244.

Schwob, E., and Nasmyth, K. (1993). CLB5 and CLB6, a new pair of B cyclins involved in S phase and mitotic spindle formation in *S. cerevisiae*. Genes & Dev. 7, 1160–1175.

Sikorski, R.S., and Hieter, P. (1989). A system of shuttle vectors and yeast host strains designed for efficient manipulation of DNA in *Saccharomyces cerevisiae*. Genetics 122, 19–27.

Skowyra, D., Craig, K.L., Tyers, M., Elledge, S.J., and Harper, J.W. (1997). F-box proteins are receptors that recruit phosphorylated substrates to the SCF ubiquitin ligase complex. Cell 91, 209–219.

Skowyra, D., Koepp, M.M., Kamura, T., Conrad, M.M., Conaway, R.C., Conaway, J.W., Elledge, S.J., and Harper, J.W. (1999). Reconstitution of G(1) cyclin ubiquitination with complexes containing SCFGrr1 and Rbx1. Science 284, 662–665.

Surana, U., Amon, A., Dowzer, C., McGrew, J., Byers, B., and Nasmyth, K. (1993). Destruction of the CDC28/CLB mitotic kinase is not required for the metaphase to anaphase transition in budding yeast. EMBO J. 12, 1969–1978.

Verma, R., Annan, R., Huddleston, M., Carr, S., Reynard, G., and Deshaies, R.J. (1997a). Phosphorylation of Sic1p by cyclin/Cdk required for its degradation and entry into S phase. Science 278, 455–460.

Verma, R., Chi, Y., and Deshaies, R.J. (1997b). Cell-free ubiquitination of cell-cycle regulators in budding yeast extracts. Methods Enzymol. 283, 366–376.

Verma, R., Feldman, R.M.R., and Deshaies, R.J. (1997c). Sic1 is ubiquitinated in-vitro by a pathway that requires Cdc4, Cdc34, and cyclin/Cdk activities. Mol. Biol. Cell 8, 1427–1437.

Weinreich, M., Liang, C., and Stillman, B. (1999). The Cdc6p nucleotide-binding motif is required for loading Mcm proteins onto chromatin. Proc. Natl. Acad. Sci. USA 96, 441–446.

Willems, A.R., Lanker, S., Patton, E.E., Craig, K.L., and Nason, T.F. (1996). Cdc53 targets phosphorylated G1 cyclins for degradation by the ubiquitin proteolytic pathway. Cell 86, 453–463.

Williams, R.S., Shohet, R.V., and Stillman, B. (1997). A human protein related to yeast Cdc6p. Proc. Natl. Acad. Sci. USA 94, 142–147.

Zou, L., and Stillman, B. (1998). Formation of a preinitiation complex by S phase Cdk-dependent loading of Cdc45p onto chromatin. Science 280, 593–596.
