
Microbiology (1999), 145, 2813–2820

**Genetic organization and characteristics of the 3-(3-hydroxyphenyl)propionic acid degradation pathway of Comamonas testosteroni TA441**

Hiroyuki Arai,$^{1\dagger}$ Takako Yamamoto,$^{1}$ Tohru Ohishi,$^{1}$ Takeshi Shimizu,$^{2}$ Tadashi Nakata$^{2}$ and Toshiaki Kudo$^{1}$

Author for correspondence: Toshiaki Kudo. Tel: +81 48 467 9544. Fax: +81 48 462 4672.  
e-mail: tkudo@postman.riken.go.jp

**Comamonas testosteroni TA441 degrades 3-(3-hydroxyphenyl)propionate (3HPP) via the meta pathway. A gene cluster required for degradation of 3HPP was cloned from strain TA441 and sequenced. The genes encoding six catabolic enzymes, a flavin-type hydroxylase (mhpA), extradiol dioxygenase (mhpB), 2-keto-4-pentenoate hydratase (mhpD), acetaldehyde dehydrogenase (acylating) (mhpF), 4-hydroxy-2-ketovalerate aldolase (mhpE) and the meta cleavage compound hydrolase (mhpC), were found in this cluster, encoded in this order. mhpD and mhpF were separated by two genes, orf4 and orf5, which were not necessary for growth on 3HPP. The gene mhpR, encoding a putative transcriptional activator of the IclR family, was located adjacent to mhpA in the opposite orientation. Disruption of the mhpB or mhpR genes affected growth on 3HPP or trans-3-hydroxycinnamate. The mhpB and mhpC gene products showed high specificity for 3-(2,3-dihydroxyphenyl)propionate (DHPP) and the meta cleavage compound produced from DHPP, respectively.**

Keywords: 3-(3-hydroxyphenyl)propionate, trans-3-hydroxycinnamate, meta pathway, biodegradation, Comamonas testosteroni

---

**INTRODUCTION**

Phenylalkanoic acids and their hydroxylated derivatives are ubiquitous in the environment as breakdown products of lignin or other plant-derived phenylpropanoids and flavonoids. Bacterial degradation of such compounds is important for recycling of carbon sources on earth; however, little is known about the genes required for degradation of phenylalkanoic acids. The genes for degradation of 3-(3-hydroxyphenyl)-propionate (3HPP) in Escherichia coli K-12 (Ferrández et al., 1997) and Rhodococcus globerulus PWD1 (Barnes et al., 1997), and the genes for degradation of 4-hydroxyphenylacetate in E. coli W (Prieto et al., 1996)

have been reported. Although the ability of pseudomonads and related bacteria to degrade aromatic compounds has been well studied, the existence of genes for degradation of natural phenylalkanoic acids in pseudomonads has not been reported. Comamonas [formerly Pseudomonas (Tamaoka et al., 1987)] testosteroni TA441 was isolated from a homogenate of the gut of the wood-feeding termite Reticulitermes speratus together with a biphenyl-degrading bacterium Rhodococcus erythropolis TA431 (Arai et al., 1998; Chung et al., 1994). Strain TA441 was found to display extradiol dioxygenase activity, as colonies on LB plates turned yellow after being sprayed with 2,3-dihydroxybiphenyl (DHBP), but this strain is unable to grow on biphenyl as sole carbon source (unpublished data). Because there is an abundance of ligninolytic compounds in the gut of wood-feeding termites, bacteria in the termite ecosystem are expected to display high activity in degrading phenylalkanoic acids. Actually, strain TA441 is capable of growth on 3HPP, 3-hydroxycinnamate (3HCI) or 3-hydroxyphenylacetate (3HPA). 3HPP was found to induce extradiol dioxygenase activity in strain TA441, indicating that 3HPP is degraded by the meta pathway. In this study, we cloned and sequenced a gene cluster

† Present address: Department of Biotechnology, University of Tokyo, Yayoi 1-1-1, Bunkyo-ku, Tokyo 113-8657, Japan.

Abbreviations: 3HCI, trans-3-hydroxycinnamate; 3HPA, 3-hydroxyphenylacetate; 3HPP, 3-(3-hydroxyphenyl)propionate; 3MC, 3-methylcatechol; 4MC, 4-methylcatechol; DHBP, 2,3-dihydroxybiphenyl; DHCI, trans-2, 3-dihydroxycinnamate; DHPP, 3-(2,3-dihydroxyphenyl)propionate; CFE, cell-free extract.

The DDBJ/EMBL/GenBank accession number for the sequence reported in this paper is AB024335.

that encodes the enzymes for the complete degradation pathway of 3HPP and 3HCI.

# METHODS

## Bacterial strains, plasmids and growth conditions.
The bacterial strains and plasmids used in this study are listed in Table 1. *C. testosteroni* strains were cultivated in LB medium or C-medium at 30 °C. C-medium was supplemented with 0·1% succinate, 3HPP, 3HCI or 3HPA as carbon source. Growth was monitored by measuring OD${}_{550}$. *E. coli* JM109, used as the host for gene manipulation (Sambrook et al., 1989), was cultivated in LB medium at 37 °C. The medium composition and concentration of antibiotics were described previously (Arai et al., 1998).

## Chemicals.
3HPP and 3-(2,3-dihydroxyphenyl)propionate (DHPP) were synthesized by the procedures described by Blakley & Simpson (1964) and Burlingame & Chapman (1983). Other chemicals were purchased from Tokyo Kasei or Sigma.

## Preparation of cell-free extract.
For preparation of cell-free extract (CFE), bacterial cells were cultured overnight. The cells were harvested by centrifugation, washed with 50 mM potassium phosphate buffer (pH 7·5), resuspended in the same buffer and disrupted by sonication. The cell debris was removed by centrifugation and the supernatant was used as CFE. Protein concentration was determined by the method of Bradford (1976) using the Bio-Rad protein assay kit, with bovine serum albumin as standard.

## Enzyme assays.
Extradiol dioxygenase activity was determined spectrophotometrically by monitoring the increase in the yellow ring-fission products upon addition of CFE to 50 mM potassium phosphate buffer (pH 7·5) containing 0·8 mM diol substrate at 30 °C. The molar extinction coefficient values ($\varepsilon$) for the ring-fission products were as follows (M${}^{-1}$ cm${}^{-1}$): DHBP, $\varepsilon_{434}=13200$; DHPP, $\varepsilon_{394}=19150$; catechol, $\varepsilon_{375}=36000$; 3-methylcatechol (3MC), $\varepsilon_{388}=15000$; 4-methylcatechol (4MC), $\varepsilon_{375}=13500$ (Barnes et al., 1997; Eltis et al., 1993).

The substrates for the assay of hydrolase activity were prepared enzymically from diol compounds using recombinant extradiol dioxygenases. pYT103, pYT40, pUC18-*xylE* and pUC19-*todE* express DHPP dioxygenase of strain TA441, DHBP dioxygenase of *R. erythropolis* TA421 (Maeda et al., 1995), catechol 2,3-dioxygenase of the TOL plasmid (Burlage et al., 1989) and methylcatechol dioxygenase of *Pseudomonas putida* F1 (Zylstra & Gibson, 1989), respectively. DHPP, DHBP, catechol or 3MC were converted to the ring-fission products in 50 mM potassium phosphate buffer (pH 7·5) using the CFE of *E. coli* JM109 harbouring the plasmid encoding the appropriate dioxygenase. After the ring-fission reaction was completed, the CFE for the hydrolase assay was added to the reaction mixture. The activity was measured spectrophotometrically by monitoring the disappearance of the yellow ring-fission compounds.

## Recombinant DNA techniques.
DNA manipulations were performed by standard methods (Sambrook et al., 1989) or as described previously (Arai et al., 1998). An ABI 377 DNA sequencer was used for sequence determination (Applied Biosystems). A BigDye terminator cycle sequence kit (Perkin-Elmer) was used for dideoxy chain-termination reactions. Synthetic oligonucleotides used as primers for sequence determination were purchased from Sawady Technology.

## Cloning of the *mhp* gene cluster.
For isolation of the extradiol dioxygenase gene (*mhpB*), total DNA of strain TA441 was digested with *SalI* and ligated with the *SalI* digest of pUC19.

---

## Table 1. Bacterial strains and plasmids

| Strain or plasmid | Relevant characteristics* | Source/reference |
|-------------------|---------------------------|------------------|
| *C. testosteroni* |                           |                  |
| TA441             | Wild-type (JCM 9806)      | Arai et al. (1998) |
| TTM101            | *mhpB* mutant of TA441; Km${}^{r}$ | This study |
| TTM107            | *orf4*, *orf5* mutant of TA441; Km${}^{r}$ | This study |
| TTM112            | *mhpR* mutant of TA441; Km${}^{r}$ | This study |
| *E. coli*         |                           |                  |
| JM109             | Host strain for DNA manipulation | Toyobo |
| Plasmid           |                           |                  |
| pUC18             | Cloning vector. Ap${}^{r}$ | Toyobo |
| pUC19             | Cloning vector. Ap${}^{r}$ | Toyobo |
| pYT10             | 4·3 kb *SalI* fragment from TA441 in pUC19 | This study |
| pYT11             | 5·5 kb *BglII* fragment from TA441 in pUC19 | This study |
| pYT12             | 6·0 kb *BglII* fragment from TA441 in pUC19 | This study |
| pYT101            | *mhpB*::Km${}^{r}$ in pYT10 | This study |
| pYT103            | *mhpB* in pUC19 | This study |
| pYT107            | (*orf4*, *orf5*)::Km${}^{r}$ in pUC19 | This study |
| pYT112            | *mhpR*::Km${}^{r}$ in pUC19 | This study |
| pYT121            | *mhpC* in pUC19 | This study |
| pTC40             | *bphC3* from *R. erythropolis* TA421 in pUC18 | Maeda et al. (1995) |
| pUC18-*xylE*      | *xylE* from pTS1045 in pUC18 | This study |
| pUC19-*todE*      | *todE* from *P. putida* F1 in pUC19 | This study |

*Ap${}^{r}$, ampicillin resistance; Km${}^{r}$, kanamycin resistance.

3HPP degradation genes of C. testosteroni

pYT11
pYT10
pYT12

mhpR mhpA mhpB mhpD orf4 orf5 mhpF mhpE mhpC

pYT103
pYT121
pYT101
pYT112
pYT107

Fig. 1. Physical map of the *mhp* genes from *C. testosteroni* TA441. Upper bars labelled pYT10, pYT11 and pYT12 indicate the fragments cloned from chromosomal DNA of strain TA441. Lower bars indicate pUC19-derived plasmids. pYT103 and pYT121 were used for expression of *mhpB* and *mhpC*, respectively, in *E. coli* JM109. pYT101, pYT107 and pYT112 were used for construction of mutant strains TTM101, TTM107 and TTM112, respectively, by homologous recombination.

coli JM109 was transformed with the ligation mixture and plated onto LB agar plates containing ampicillin (Ap) and IPTG. Colonies that expressed the extradiol dioxygenase were identified by spraying with a DHBP solution (20 mM in acetone). A 4·3 kb *SalI* fragment (pYT10) was obtained from the plasmid in a positive clone which turned yellow. A 5·5 kb *BglII* fragment (pYT11) and a 6·0 kb *BglII* fragment (pYT12), which hybridized with the upstream *SalI–SacI* fragment and the downstream *SacI–SalI* fragment of pYT10, respectively, were cloned from the total DNA of strain TA441 and identified by colony hybridization.

Construction of plasmids and mutant strains. The *mhpB* gene of pYT10 was disrupted by insertion of a blunt-ended 1·5 kb *HindIII–SalI* fragment of pSUP5011, which contained a kanamycin (Km) resistance gene (Simon, 1984), into the *EcoRV* site (Fig. 1). The resultant plasmid, pYT101, was introduced into strain TA441 by electroporation and the transformed cells were plated on agar plates containing Km. The Km-resistant colonies were transferred to plates containing carbenicillin (Cb). A Cb-sensitive and Km-resistant colony, whose *mhpB* gene was exchanged for the disrupted *mhpB* gene by double crossover reactions, was selected and designated as strain TTM101. *orf4* and *orf5* were disrupted in the same way by use of pYT107, which was constructed by replacement of the *BglII–Bpu1102I* fragment with the Km resistance gene (Fig. 1). The resultant mutant was designated strain TTM107. The *mhpR* mutant strain TTM112 was constructed by use of pYT112, whose *mhpR* gene was disrupted by insertion of the Km resistance gene at the *KpnI* site. Insertion of the Km resistance gene into the chromosome of the mutant strains was confirmed by PCR (data not shown).

pYT103 carries a PCR fragment containing the *mhpB* gene. The fragment was amplified with oligonucleotides YP4 (GGCACGGATCCTGCACCTCCCCACTGG) and YP5 (CGGCATGGAATTCATGAAAGGATGAGG) from a fresh colony of strain TA441. The amplified 1·0 kb fragment was digested with *BamHI* and *EcoRI*, and ligated to the respective site of pUC19, resulting in pYT103. pYT121, which was used for expression of the *mhpC* gene in *E. coli*, was constructed by subcloning a 1·5 kb *SalI–KpnI* fragment into pUC19 (Fig. 1). pUC18-*xylE* was constructed by ligation of a 1·8 kb *BamHI–XhoI* fragment of pTS1045 that carries the *xylE* gene (Inouye et al., 1986) with *BamHI/SalI*-digested pUC18. pUC19-*todE* was constructed by subcloning a *BamHI/EcoRI* digest of the 1·0 kb PCR fragment, which carries *todE* of *P. putida* F1 (Zylstra & Gibson, 1989), into pUC19. The fragment was amplified from a fresh colony of strain F1 with oligonucleotides TodE1 (GGCGCACGGATCCACAAGCACTT-CGG) and TodE2 (CATATGGGAATTCAGGTAGAAGC-AGGAC), which were designed to introduce *BamHI* and *EcoRI* sites, respectively.

RESULTS AND DISCUSSION

Induction of extradiol dioxygenase and hydrolase enzymic activities by 3HPP

3HPP is degraded to intermediates of the TCA cycle by the meta pathway in *E. coli* K-12 and in *R. globerulus* PWD1 (Barnes et al., 1997; Burlingame & Chapman, 1983; Ferrández et al., 1997) (Fig. 2). 3HPP is initially converted to DHPP by 3HPP hydroxylase. Extradiol dioxygenase catalyses the meta fission of DHPP. The ring-fission product is converted to succinate and 2-keto-4-pentenoate by a hydrolase. The three downstream reactions catalysed by hydratase, aldolase and acetaldehyde dehydrogenase (acylating) are the same as the reactions in the pathway for meta degradation of other aromatic compounds such as biphenyl, phenol and toluene (Lau et al., 1994; Hofer et al., 1994; Shingler et al., 1992). When strain TA441 was cultivated with 3HPP as carbon source, DHPP dioxygenase and hydrolase activities were detected (0·16 and 0·35 μmol min⁻¹ mg⁻¹, respectively), whereas these activities were not detected when the strain was cultured with succinate as carbon

H. ARAI and OTHERS

![Diagram](#)

Fig. 2. Putative catabolic pathway of 3HPP and 3HCl in *C. testosteroni* TA441. The metabolites are 3HPP (I), DHPP (II), 2-hydroxy-6-ketonadienedioate (III), succinate (IV), 3HCl (V), DHCl (VI), 2-hydroxy-6-ketonatrienedioate (VII), fumarate (VIII), 2-keto-4-pentenoate (IX), 4-hydroxy-2-ketovalerate (X), acetaldehyde (XI), pyruvate (XII) and acetyl CoA (XIII). Enzymes: MhpA, 3HPP hydroxylase; MhpB, DHPP 1,2-dioxygenase; MhpC, 2-hydroxy-6-ketonona-2,4-dienedioate hydrolase; MhpD, 2-keto-4-pentenoate hydratase; MhpE, 4-hydroxy-2-ketovalerate aldolase; MhpF, acetaldehyde dehydrogenase (acylating).

Table 2. Gene products of the *mhp* gene cluster of strain TA441 and homology with other proteins

| Gene    | No. of amino acids | Gene product                                      | Percentage identity with other gene products* |
|---------|--------------------|---------------------------------------------------|------------------------------------------------|
| *mhpA*  | 589                | 3HPP hydroxylase                                  | MhpA (60), HppA (48), PheA (26), TfdB (25)      |
| *mhpB*  | 321                | DHPP 1,2-dioxygenase                              | MhpB (44), HppB (43), MpcI (44), LigB (21), EdoD (45) |
| *mhpC*  | 295                | 2-Hydroxy-6-ketonona-2,4-dienedioate hydrolase   | MhpC (69), HppC (32), BphD (58), XylF (38), DmpD (35) |
| *mhpD*  | 263                | 2-Keto-4-pentenoate hydratase                     | MhpD (60), BphE (43), XylJ (44), DmpE (44)      |
| *mhpE*  | 343                | 4-Hydroxy-2-ketovalerate aldolase                 | MhpE (85), BphF (57), XylK (83), DmpG (84)      |
| *mhpF*  | 316                | Acetaldehyde dehydrogenase (acylating)            | MhpF (77), BphG (57), XylQ (77), DmpF (79)      |
| *mhpR*  | 261                | IclR-family regulator                             | MhpR (43), HppR (28), PobR (23), PcaU (22), PcaR (23) |
| *orf4*  | 325                | Unknown                                           | *orf5* (63), *tcb*-ORF3 (35), *clc*-ORF3 (36), ORF7 (35) |
| *orf5*  | 278                | Unknown                                           | *orf4* (63), *tcb*-ORF3 (35), *clc*-ORF3 (36), ORF7 (32) |

* Percentage identity at the amino acid level is given. Mhp, *E. coli* K-12 (Ferrández *et al.*, 1997); Hpp, *R. globerulus* PWD1 (Barnes *et al.*, 1997); PheA, *P. putida* EST1001 (Nurk *et al.*, 1991); TfdB, *R. eutropha* JMP134 (Perkins *et al.*, 1990); MpcI, *R. eutropha* JMP222 (Spence *et al.*, 1996); LigB, *Sphingomonas paucimobilis* SYK6 (Noda *et al.*, 1990); EdoD, *Rhodococcus sp.* I1 (Kulakov *et al.*, 1998); Bph, *Burkholderia sp.* LB400 (Erickson & Mondello, 1992; Hofer *et al.*, 1993); Xyl, TOL plasmid pWW0 (Harayama & Rekik, 1990); Dmp, *Pseudomonas sp.* CF600 (Shingler *et al.*, 1992); PobR and PcaU, *Acinetobacter calcoaceticus* ADP1 (DiMarco *et al.*, 1993; Kowalchuk *et al.*, 1994); PcaR, *P. putida* PRS1 (Harwood *et al.*, 1994); *tcb*-ORF3, *Pseudomonas sp.* P51 (van der Meer *et al.*, 1991); *clc*-ORF3, catabolic plasmid pAC27 (Frantz & Chakrabarty, 1987); ORF7, *Alcaligenes eutrophus* H16 (Valentin *et al.*, 1995).

source. These results indicated that 3HPP is degraded by the meta pathway in strain TA441, as in *E. coli* K-12 and *R. globerulus* PWD1.

### Cloning and sequencing of the 3HPP degradation gene cluster

A 4·3 kb *SalI* fragment encoding the extradiol dioxygenase was cloned from the total DNA of strain TA441 (pYT10, Fig. 1). Analysis of the nucleotide and deduced amino acid sequences revealed that the dioxygenase gene (*mhpB*) was accompanied by genes encoding proteins similar to the single-component hydroxylase (*mhpA*) and the 2-keto-4-pentenoate hydratase (*mhpD*). *BglII* fragments encoding the regions upstream (pYT11) and downstream (pYT12) of the *mhpB* gene were cloned using two *SalI-SacI* fragments of pYT10 as probes. A putative regulatory gene, *mhpR*, was located adjacent to *mhpA*, in the opposite orientation. The *mhpFEC* genes, expected to encode acetaldehyde dehydrogenase (acylating), 4-hydroxy-2-ketovalerate aldolase and 2-hydroxy-6-ketonona-2,4-dienedioate hydrolase, respectively, were located about 2 kb downstream of the *mhpD* gene. This gene arrangement is different from that of *E. coli* K-12 (*mhpABCDFE*) (Ferrández *et al.*, 1997). The *mhpC* gene of strain TA441 is moved to the end of the cluster and two ORFs (*orf4* and *orf5*) encoding proteins of unknown function are

MhpB–C. testosteroni TA441

HpcB–E. coli C  
HpaD–E. coli W  

MhpB–E. coli K-12  
Mpcl–R. eutropha JM222  
HppB–R. globerulus PWD1  
EdoD–Rhodococcus sp. I1  

LigB–S. paucimobilis SYK6  

Fig. 3. Molecular phylogenetic tree for type II dioxygenases. Multiple sequence alignment was done using CLUSTAL W. Tree topology and evolutionary distance estimations were done by the neighbour-joining method (PHYLIP 3.5). The numbers indicated at the nodes are bootstrap values calculated from 100 replications using the SEQBOOT, PROTDIST, NEIGHBOUR and CONSENSE programs of the PHYLIP 3.5 program package.

inserted between *mhpD* and *mhpF*. The arrangement is also different from that of the 3HPP catabolic genes (*hppACBKR*) of *R. globerulus* PWD1, in which the genes for the three downstream enzymes (hydratase, aldolase and acetaldehyde dehydrogenase) are not located within the cluster (Barnes *et al.*, 1997). The gene corresponding to *orfT* of *E. coli* K-12 and *hppK* of strain PWD1, encoding a putative transporter, was not found in the fragments cloned from strain TA441. It is not certain at present whether strain TA441 expresses a specific transporter for intake of 3HPP.

### Sequence comparisons

The deduced amino acid sequences of the *mhp* genes from strain TA441 were compared with the genes for other aromatic hydrocarbon degradation enzymes (Table 2). The *mhpA* gene encodes a protein consisting of 589 aa residues showing a high level of identity with the 3HPP hydroxylase of *E. coli* K-12 (MhpA) and that of *R. globerulus* PWD1 (HppA) (Barnes *et al.*, 1997; Ferrández *et al.*, 1997), suggesting that MhpA of strain TA441 is a 3HPP hydroxylase. MhpA also shows low similarity to single-component flavin-type hydroxylases, including phenol hydroxylase (PheA) from *P. putida* EST1001 (Nurk *et al.*, 1991) and 2,4-dichlorophenoxyacetate hydroxylase (TfdB) from *Ralstonia eutropha* JMP134 (Perkins *et al.*, 1990).

The *mhpB* gene encodes an extradiol dioxygenase consisting of 321 aa residues categorized as a type II (class III) dioxygenase (Eltis & Bolin, 1996). The phylogenetic tree of this type of dioxygenase is shown in Fig. 3. MhpB from strain TA441 is closely related to dioxygenases with high specificity for DHPP (MhpB from *E. coli*, HppB and Mpcl) (Barnes *et al.*, 1997; Ferrández *et al.*, 1997; Spence *et al.*, 1996) and is distantly related to a subunit of the protocatechuate 4,5-dioxygenase (LigB) from *Sphingomonas paucimobilis* (Noda *et al.*, 1990) and the 3,4-dihydroxyphenylacetate 2,3-dioxygenases (HpaD and HpcB) from *E. coli* strains (Prieto *et al.*, 1996; Roper & Cooper, 1990; Roper *et al.*, 1995). EdoD from *Rhodococcus* sp. I1 was most similar

to MhpB of strain TA441 (Table 2); however, the physiological role of the protein has not been identified (Kulakov *et al.*, 1998).

The *mhpC* gene encodes a protein consisting of 295 aa residues. The deduced sequence of the gene is homologous with the meta fission product hydrolases involved in degradation of aromatic compounds such as 3HPP (MhpC from *E. coli* and HppC), biphenyl (BphD), toluene (XylF) and phenol (DmpD). The *mhpD*, *mhpE* and *mhpF* genes encode proteins highly similar to hydratase, aldolase and acetaldehyde dehydrogenase (acylating), respectively. These three enzymic reactions are common to other meta degradation pathways for various aromatic compounds.

The *mhpR* gene encodes a protein consisting of 261 aa residues that belongs to the IclR family of transcriptional regulators. This family includes regulators for degradation of 4-hydroxybenzoate (PobR) and protocatechuate (PcaU and PcaR) (DiMarco *et al.*, 1993; Harwood *et al.*, 1994; Kowalchuk *et al.*, 1994). The product of the corresponding gene from *E. coli* K-12 was identified as a transcriptional activator that regulates transcription of the *mhp* genes (Ferrández *et al.*, 1997). Because expression of the enzymes for 3HPP degradation is responsive to 3HPP, it seems likely that MhpR acts as a regulator for the *mhp* genes in strain TA441, as in the case of *E. coli*.

The two ORFs, *orf4* and *orf5*, encode proteins consisting of 325 and 278 aa residues, respectively. The deduced sequences of *orf4* and *orf5* are highly similar to each other, with 63% of the amino acid residues being identical. The *orf4* gene product has a typical N-terminal signal sequence for membrane translocation, whereas the deduced sequence of *orf5* lacks the N-terminal signal, indicating that the *orf4* and *orf5* gene products are probably located in the periplasm and in the cytoplasm, respectively. Genes similar to *orf4* and *orf5* have been found in the gene cluster for the ortho degradation pathway of chlorocatechol in *Pseudomonas* sp. P51 (ORF3 of the *tcb* gene cluster) (van der Meer *et al.*, 1991) and in the catabolic plasmid pAC27 (ORF3 of

H. ARAI and OTHERS

Table 3. Substrate specificity of MhpB and MhpC

E. coli JM109 harbouring pYT103 or pYT121 was grown in LB medium containing 0·5 mM IPTG. Activities of the enzymes were measured in CFEs. Activities are expressed as the percentage of the activity when DHPP or meta fission product of DHPP was used as the substrate, which was defined as 100% (corresponding to 0·73 or 5·8 μmol min⁻¹ mg⁻¹, respectively). Values are means from three independent experiments.

| Substrate | Relative activity |
|-----------|-------------------|
|           | Dioxygenase       | Hydrolase* |
| DHPP      | 100               | 100        |
| DHBP      | 63                | 2·4        |
| Catechol  | 14                | <0·1       |
| 3MC       | 31                | 7·0        |
| 4MC       | 7·3               | ND         |

*Hydrolase activities were measured by monitoring the decrease of the ring-fission product of the diol substrate, which was prepared enzymically as described in Methods. ND, Not determined.

metabolism of hydroxyacids or ketoacids; however, the function of these genes has not been identified.

Substrate specificity of the mhpB and mhpC gene products

The mhpB and mhpC gene fragments were cloned in pUC19, resulting in pYT103 and pYT121, respectively (Fig. 1). CFEs of E. coli JM109 harbouring the plasmids were prepared from cells grown in LB medium containing 0·5 M IPTG. The extradiol dioxygenase activity in the CFE of JM109(pYT103) and the hydrolase activity in the CFE of JM109(pYT121) were measured with several substrates. The activities of DHPP dioxygenase and 2-hydroxy-6-ketonona-2,4-dienedioate hydrolase of strain JM109 harbouring pUC19 were not detected under the culture conditions used, indicating that the corresponding enzymes derived from the E. coli chromosome were not expressed (data not shown). Table 3 shows the substrate specificity of the recombinant enzymes. DHPP was the most preferable substrate for MhpB. MhpB also showed relatively high activity with DHBP as substrate, but the activity with catechol or methylcatechols was low. MhpC was highly specific for the meta cleavage compound produced from DHPP (2-hydroxy-6-ketonona-2,4-dienedioate) and showed very low activity with other substrates.

the clc gene cluster) (Frantz & Chakrabarty, 1987), and in the gene cluster for biosynthesis of poly(3-hydroxybutyrate-co-4-hydroxybutyrate) in Alcaligenes eutrophus H16 (ORF7) (Valentin et al., 1995). ORF3 of the tcb gene cluster and ORF7 in A. eutrophus H16 encode putative proteins with N-terminal signal sequences, whereas ORF3 in the clc gene cluster does not encode a protein with an N-terminal signal sequence. orf4-like genes were also found in the gene clusters for the metabolism of tartrate and urea (McMillan et al., 1998; Salomone et al., 1996). Most of the gene clusters that contain an orf4-like gene are associated with the

The mhp gene cluster is involved in utilization of 3HPP and 3HCI

The mutant strains TTM101 (mhpB⁻), TTM107 (orf4-orf5⁻) and TTM112 (mhpR⁻) were constructed by insertion of the Km resistance gene by homologous recombination. The growth curves of the strains on 3HPP, 3HCI or 3HPA are shown in Fig. 4. Strains TA441 and TTM101 showed identical growth curves when 3HPA was used as a carbon source. They also

![Figure 4](#fig4)

Fig. 4. Growth of C. testosteroni strains TA441 (a), TTM101 (b), TTM107 (c) and TTM112 (d). The strains were grown in 100 ml C-medium supplemented with 0·1% 3HPP (○), 3HCI (□) or 3HPA (△).

grew equally well on succinate or 4-hydroxybenzoate (data not shown). On the contrary, the growth of strain TTM101 was severely affected when 3HPP or 3HCI was used as carbon source (Fig. 4b). These results clearly indicate that the *mhp* gene cluster in strain TA441 is involved in utilization of 3HPP and 3HCI. The culture medium of strain TTM101 grown on 3HPP or 3HCI turned brown, which caused a gradual increase in optical density. Strain TTM101 could produce DHPP and 2,3-dihydroxycinnamic acid (DHCI) from 3HPP and 3HCI, respectively, because the gene for 3HPP hydroxylase (*mhpA*) is intact in this strain. The brownish compounds were probably spontaneously oxidized products derived from these catechols. A yellow *meta* cleavage product accumulated when strain TA441 was grown on 3HCI, indicating that the hydrolase (MhpC) has relatively low specificity for the *meta* cleavage compound produced from DHCI compared to that produced from DHPP.

Elimination of a region containing *orf4* and *orf5* had no effect on the utilization of 3HPP or 3HCI (Fig. 4c), therefore the role of these ORFs is still unclear. Strain TTM112 showed poor growth on 3HPP or 3HCI (Fig. 4d). This observation suggests that the putative transcriptional regulator encoded by the *mhpR* gene is an activator. Accumulation of the brownish compound, which was found with strain TTM101, was not observed, suggesting that the expression of 3HPP hydroxylase (MhpA) may be affected also by the *mhpR* mutation. The *mhpR* and *mhpA* genes are transcribed divergently. Probably, MhpR acts on a putative operator located in the *mhpR-mhpA* intergenic region.

We have reported another gene cluster (*aphKLMNO-PQB*) encoding a set of enzymes, including a hydroxylase and an extradiol dioxygenase, which is required for adaptive growth of strain TA441 on phenol (Arai *et al.*, 1998). The phenol hydroxylase encoded by the *aph-KLMNOP* genes belongs to a family of multicomponent-type monoxygenases that differ from the flavin-type MhpA. The *aphB* gene encodes catechol 2,3-dioxygenase which has low specificity for substituted catechols. Möbus *et al.* (1997) reported that a protein (TIP1), whose N-terminal sequence was similar to that of the DHBP dioxygenase, was expressed when *C.* testosteronei was grown with testosterone as a carbon source. Comparing the N-terminal sequences, neither that of MhpB nor that of AphB was identical to that of TIP1. The extradiol dioxygenase activity was found to be induced when strain TA441 was grown on steroids such as testosterone or bile acids as a carbon source. The *aphB⁻mhpB⁻* double mutant of strain TA441 could also grow on steroids and expressed dioxygenase activity (unpublished data). These results indicate that strain TA441 has at least one more *meta* cleavage enzyme involved in steroid utilization.

### ACKNOWLEDGEMENTS

We thank Y. Ichikawa (Biodesign DNA Sequence Facility, RIKEN) for determination of the nucleotide sequence. This work was partially supported by Special Coordination Funds for Promoting Science and Technology from the Science and Technology Agency of the Japanese Government and by a grant for the Eco Molecular Sciences Research Program from RIKEN.

### REFERENCES

Arai, H., Akahira, S., Ohishi, T., Maeda, M. & Kudo, T. (1998). Adaptation of *Comamonas testosteronei* TA441 to utilize phenol: organization and regulation of the genes involved in phenol degradation. *Microbiology* 144, 2895–2903.

Barnes, M. R., Duetz, W. A. & Williams, P. A. (1997). A 3-(3-hydroxyphenyl)propionic acid catabolic pathway in *Rhodococcus globerulus* PWD1: cloning and characterization of the *hpp* operon. *J Bacteriol* 179, 6145–6153.

Blakley, E. R. & Simpson, F. J. (1964). The microbial metabolism of cinnamic acid. *Can J Microbiol* 10, 175–185.

Bradford, M. M. (1976). A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. *Anal Biochem* 72, 248–254.

Burlage, R. S., Hooper, S. W. & Sayler, G. S. (1989). The TOL (pWW0) catabolic plasmid. *Appl Environ Microbiol* 55, 1323–1328.

Burlingame, R. & Chapman, P. J. (1983). Catabolism of phenylpropionic acid and its 3-hydroxy derivative by *Escherichia coli*. *J Bacteriol* 155, 113–121.

Chung, S.-Y., Maeda, M., Song, E., Horikoshi, K. & Kudo, T. (1994). A gram-positive polychlorinated biphenyl-degrading bacterium, *Rhodococcus erythropolis* strain TA421, isolated from a termite ecosystem. *Biosci Biotechnol Biochem* 58, 2111–2113.

DiMarco, A. A., Averhoff, B. & Ornston, L. N. (1993). Identification of the transcriptional activator *pobR* and characterization of its role in the expression of *pobA*, the structural gene for *p*-hydroxybenzoate hydroxylase in *Acinetobacter calcoaceticus*. *J Bacteriol* 175, 4499–4506.

Eltis, L. D. & Bolin, J. T. (1996). Evolutionary relationships among extradiol dioxygenases. *J Bacteriol* 178, 5930–5937.

Eltis, L. D., Hofmann, B., Hecht, H.-J., Lünsdorf, H. & Timmis, K. N. (1993). Purification and crystallization of 2,3-dihydroxybiphenyl 1,2-dioxygenase. *J Biol Chem* 268, 2727–2732.

Erickson, B. D. & Mondello, F. J. (1992). Nucleotide sequencing and transcriptional mapping of genes encoding biphenyl dioxygenase, a multicomponent PCB-degrading enzyme in *Pseudomonas* strain LB400. *J Bacteriol* 174, 2903–2912.

Ferrández, A., García, J. L. & Díaz, E. (1997). Genetic characterization and expression in heterologous hosts of the 3-(3-hydroxyphenyl)propionate catabolic pathway of *Escherichia coli* K-12. *J Bacteriol* 179, 2573–2581.

Frantz, B. & Chakrabarty, A. M. (1987). Organization and nucleotide sequence determination of a gene cluster involved in 3-chlorocatechol degradation. *Proc Natl Acad Sci USA* 84, 4460–4464.

Harayama, S. & Rekik, M. (1990). The *meta* cleavage operon of TOL degradative plasmid pWW0 comprises 13 genes. *Mol Gen Genet* 221, 113–120.

Harwood, C. S., Nichols, N. N., Kim, M. K., Ditty, J. L. & Parales, R. E. (1994). Identification of the *pcaRKF* gene cluster from *Pseudomonas putida*: involvement in chemotaxis, biodegradation, and transport of 4-hydroxybenzoate. *J Bacteriol* 176, 6479–6488.

Hofer, B., Backhaus, S. & Timmis, K. N. (1994). The biphenyl/polychlorinated biphenyl-degradation locus (*bph*) of *Pseudo-

H. ARAI and OTHERS

monas sp. LB400 encodes four additional metabolic enzymes. Gene 144, 9–16.

Hofer, B., Eltis, L. D., Dowling, D. N. & Timmis, K. N. (1993). Genetic analysis of a Pseudomonas locus encoding a pathway for biphenyl/polychlorinated biphenyl degradation. Gene 130, 47–55.

Inouye, S., Asai, Y., Nakazawa, A. & Nakazawa, T. (1986). Nucleotide sequence of a DNA segment promoting transcription in *Pseudomonas putida*. J Bacteriol 166, 739–745.

Kowalchuk, G. A., Hartnett, G. B., Benson, A., Houghton, J. E., Ngai, K.-L. & Ornston, L. N. (1994). Contrasting patterns of evolutionary divergence within the *Acinetobacter calcoaceticus pca* operon. Gene 146, 23–30.

Kulakov, L. A., Delcroix, V. A., Larkin, M. J., Ksenzenko, V. N. & Kulakova, A. N. (1998). Cloning of new *Rhodococcus* extradiol dioxygenase genes and study of their distribution in different *Rhodococcus* strains. Microbiology 144, 955–963.

Lau, P. C. K., Bergeron, H., Labbé, D., Wang, Y., Brousseau, R. & Gibson, D. T. (1994). Sequence and expression of the *todGIH* genes involved in the last three steps of toluene degradation by *Pseudomonas putida* F1. Gene 146, 7–13.

McMillan, D. J., Mau, M. & Walker, M. J. (1998). Characterisation of the urease gene cluster in *Bordetella bronchiseptica*. Gene 208, 243–251.

Maeda, M., Chung, S. Y., Song, E. & Kudo, T. (1995). Multiple genes encoding 2,3-dihydroxybiphenyl 1,2-dioxygenase in the gram-positive polychlorinated biphenyl-degrading bacterium *Rhodococcus erythropolis* TA421, isolated from a termite ecosystem. Appl Environ Microbiol 61, 549–555.

van der Meer, J. R., Eggen, R. I., Zehnder, A. J. & de Vos, W. M. (1991). Sequence analysis of the *Pseudomonas* sp. strain P51 *tcb* gene cluster, which encodes metabolism of chlorinated catechols: evidence for specialization of catechol 1,2-dioxygenases for chlorinated substrates. J Bacteriol 173, 2425–2434.

Möbus, E., Jahn, M., Schmid, R., Jahn, D. & Maser, E. (1997). Testosterone-regulated expression of enzymes involved in steroid and aromatic hydrocarbon catabolism in *Comamonas testosteroni*. J Bacteriol 179, 5951–5955.

Noda, Y., Nishikawa, S., Shiozuka, K. & 7 other authors (1990). Molecular cloning of the protocatechuate 4,5-dioxygenase genes of *Pseudomonas paucimobilis*. J Bacteriol 172, 2704–2709.

Nurk, A., Kasak, L. & Kivisaar, M. (1991). Sequence of the gene (*pheA*) encoding phenol monooxygenase from *Pseudomonas* sp. EST1001: expression in *Escherichia coli* and *Pseudomonas putida*. Gene 102, 13–18.

Perkins, E. J., Gordon, M. P., Caceres, O. & Lurquin, P. F. (1990). Organization and sequence analysis of the 2,4-dichlorophenol hydroxylase and dichlorocatechol oxidative operons of plasmid pJP4. J Bacteriol 172, 2351–2359.


Prieto, M. A., Díaz, E. & García, J. L. (1996). Molecular characterization of the 4-hydroxyphenylacetate catabolic pathway of *Escherichia coli* W: engineering a mobile aromatic degradative cluster. J Bacteriol 178, 111–120.

Roper, D. I. & Cooper, R. A. (1990). Subcloning and nucleotide sequence of the 3,4-dihydroxyphenylacetate (homoprotocatechuate) 2,3-dioxygenase gene from *Escherichia coli* C. FEBS Lett 275, 53–57.

Roper, D. I., Stringfellow, J. M. & Cooper, R. A. (1995). Sequence of the *hpcC* and *hpcG* genes of the meta-fission homoprotocatechuic acid pathway of *Escherichia coli* C: nearly 40% amino-acid identity with the analogous enzymes of the catechol pathway. Gene 156, 47–51.

Salomone, J.-Y., Crouzet, P., De Ruffray, P. & Otten, L. (1996). Characterization and distribution of tartrate utilization genes in the grapevine pathogen *Agrobacterium vitis*. Mol Plant Microbe Interact 9, 401–408.

Sambrook, J., Fritsch, E. F. & Maniatis, T. (1989). *Molecular Cloning: a Laboratory Manual*, 2nd edn. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory.

Shingler, V., Powloski, J. & Marklund, U. (1992). Nucleotide sequence and functional analysis of the complete phenol/3,4-dimethylphenol catabolic pathway of *Pseudomonas* sp. strain CF600. J Bacteriol 174, 711–724.

Simon, R. (1984). High frequency mobilization of gram-negative bacterial replicons by the in vitro constructed Tn5-Mob transposon. Mol Gen Genet 196, 413–420.

Spence, E. L., Kawamukai, M., Sanvoisin, J., Braven, H. & Bugg, T. D. (1996). Catechol dioxygenases from *Escherichia coli* (MhpB) and *Alcaligenes eutrophus* (MpcI): sequence analysis and biochemical properties of a third family of extradiol dioxygenases. J Bacteriol 178, 5249–5256.

Tamaoka, J., Ha, D.-M. & Komagata, K. (1987). Reclassification of *Pseudomonas acidovorans* den Dooren de Jong 1926 and *Pseudomonas testosteronei* Marcus and Talalay 1956 as *Comamonas acidovorans* comb. nov. and *Comamonas testosteronei* comb. nov., with an emended description of the genus *Comamonas*. Int J Syst Bacteriol 37, 52–59.

Valentin, H. E., Zwingmann, G., Schonebaum, A. & Steinbuchel, A. (1995). Metabolic pathway for biosynthesis of poly(3-hydroxybutyrate-co-4-hydroxybutyrate) from 4-hydroxybutyrate by *Alcaligenes eutrophus*. Eur J Biochem 227, 43–60.

Zylstra, G. J. & Gibson, D. T. (1989). Toluene degradation by *Pseudomonas putida* F1. Nucleotide sequence of the *todC1C2-BADE* genes and their expression in *Escherichia coli*. J Biol Chem 264, 14940–14946.

Received 22 March 1999; revised 19 May 1999; accepted 28 May 1999.
