
ENZYMOLOGY:

Purification, Cloning, and Heterologous Expression of a Catalytically Efficient Flavonol 3-O-Galactosyltransferase Expressed in the Male Gametophyte of *Petunia hybrida*

Keith D. Miller, Virginie Guyon, Jeremy N. S. Evans, Wendy A. Shuttleworth and Loverine P. Taylor

*J. Biol. Chem.* 1999, 274:34011-34019.  
doi: 10.1074/jbc.274.48.34011

---

Access the most updated version of this article at [http://www.jbc.org/content/274/48/34011](http://www.jbc.org/content/274/48/34011)

Find articles, minireviews, Reflections and Classics on similar topics on the JBC Affinity Sites.

Alerts:
- When this article is cited
- When a correction for this article is posted

Click here to choose from all of JBC's e-mail alerts

This article cites 41 references, 18 of which can be accessed free at [http://www.jbc.org/content/274/48/34011.full.html#ref-list-1](http://www.jbc.org/content/274/48/34011.full.html#ref-list-1)

THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 1999 by The American Society for Biochemistry and Molecular Biology, Inc.
Vol. 274, No. 48, Issue of November 26, pp. 34011–34019, 1999
Printed in U.S.A.

# Purification, Cloning, and Heterologous Expression of a Catalytically Efficient Flavonol 3-O-Galactosyltransferase Expressed in the Male Gametophyte of *Petunia hybrida*

(Received for publication, July 12, 1999, and in revised form, September 15, 1999)

Keith D. Miller‡§, Virginie Guyon¶, Jeremy N. S. Evans∥, Wendy A. Shuttleworth∥, and Loverine P. Taylor‡¶**

From the ‡Program in Plant Physiology, Washington State University, Pullman, Washington 99164-6340, the ∥Department of Biochemistry and Biophysics, Washington State University, Pullman, Washington 99164-4660, and the ¶Department of Genetics and Cell Biology, Washington State University, Pullman, Washington 99164-4234

Flavonols are plant-specific molecules that are required for pollen germination in maize and petunia. They exist *in planta* as both the aglycone and glycosyl conjugates. We identified a flavonol 3-O-galactosyltransferase (F3GalTase) that is expressed exclusively in the male gametophyte and controls the formation of a pollen-specific class of glycosylated flavonols. Thus an essential step to understanding flavonol-induced germination is the characterization of F3GalTase. Amino acid sequences of three peptide fragments of F3GalTase purified from petunia pollen were used to isolate a full-length cDNA clone. RNA gel blot analysis and enzyme assays confirmed that F3GalTase expression is restricted to pollen. Heterologous expression of the F3GalTase cDNA in *Escherichia coli* yielded active recombinant enzyme (rF3GalTase) which had the identical substrate specificity as the native enzyme. Unlike the relatively nonspecific substrate usage of flavonoid glycosyltransferases from sporophytic tissues, F3GalTase uses only UDP-galactose and flavonols to catalyze the formation of flavonol 3-O-galactosides. Kinetic analysis showed that the $k_{cat}/K_m$ values of rF3GalTase, using kaempferol and quercetin as substrates, approaches that of a catalytically perfect enzyme. rF3GalTase catalyzes the reverse reaction, generation of flavonols from UDP and flavonol 3-O-galactosides, almost as efficiently as the forward reaction. The biochemical characteristics of F3GalTase are discussed in the context of a role in flavonol-induced pollen germination.

metylation to the three rings (1) (Fig. 1). Flavonoids perform a variety of functions including protection from ultraviolet light (2), signaling between leguminous roots and nitrogen-fixing soil bacteria (3), and the visible anthocyanin pigments in flowers attract pollinators (4). More recently, an essential role in plant reproduction was established for the flavonol class (Fig. 1) (5). In flowering plants when pollen germinates on a receptive stigma it extrudes a tube which rapidly grows by apical extension to transport the sperm cells to the embryo sac for fertilization. Chalcone synthase mutants in maize and petunia produced viable pollen which lacked flavonols and was unable to germinate or produce a functional pollen tube. However, the mutant pollen was conditionally male fertile (CMF) (6); it could be biochemically complemented by adding flavonols such as kaempferol or quercetin to the stigma at pollination or to a suspension of the mutant pollen in germination medium (5, 7). The latter procedure was the basis of an *in vitro* bioassay that was used to demonstrate that the germination response was rapid (tube outgrowth was observed within 5 min), sensitive (0.4 $\mu$M kaempferol elicits the maximum response), and specific for flavonols (5, 8).

Flavonols are synthesized in the tapetal cells of the anther wall, released into the anther locule and taken up by the developing pollen grain where they are glycosylated. Wild-type petunia pollen accumulates 2 flavonol glycosides, the 3-$O$-(2′)-$O$-$\beta$-$D$-glucopyranosyl)-$\beta$-$D$-galactopyranoside conjugates of kaempferol and quercetin (7, 9). The 1 → 2 interglycosidic linkage is common in pollen (10) but rare in sporophytic tissues such as petunia corollas which accumulate an entirely different flavonol conjugate, flavonol rhamnosyl 1 → 6 glucoside (11).

When a suspension of CMF pollen is treated with kaempferol or quercetin to induce germination, the added flavonals are rapidly converted to glycosyl conjugates identical to those that accumulate in wild-type pollen (7, 12). Xu *et al.* (12) detected radiolabeled kaempferide 3-$O$-galactoside and kaempferide 3-$O$-glucosylgalactoside within 1 min of adding [${}^{14}$C]kaempferide, a flavonol that induces pollen germination as efficiently as kaempferol, to a suspension of CMF pollen. This study demonstrated that formation of the two flavonol glycosides preceded visible tube outgrowth and ruled out a structural role in cell wall formation for flavonols or their catabolites.

* This work was supported in part by National Science Foundation Grant IBN-9405361 and by a generous grant from Pioneer Hi-Bred Intl. (to L. P. T.). The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
* The nucleotide sequence(s) reported in this paper has been submitted to the GenBank™/EBI Data Bank with accession number(s) AF165148.
* § Supported in part by Washington State University Plant Biochemistry Research and Training Center grant.
* ** To whom correspondence should be addressed: School of Molecular Biosciences, Washington State University, Pullman, WA 99164-4234. Tel.: 509-335-3612; Fax: 509-335-1907; E-mail: ltaylor@wsu.edu.

1 The abbreviations used are: CMF, conditionally male fertile; F3GalTase, flavonol 3-$O$-galactosyltransferase; PCR, polymerase chain reaction; HPLC, high performance liquid chromatography; DTT, dithiothreitol; PAGE, polyacrylamide gel electrophoresis; MES, 2-($N$-morpholino)ethanesulfonic acid; RACE, rapid amplification of cDNA ends; Ni-NTA, nickel nitrilotriacetic acid; IPTG, isopropyl-$\beta$-$D$-thiogalactopyranoside; bp, base pair(s); kbp, kilobase pair(s).

This paper is available online at http://www.jbc.org

Petunia Pollen-expressed F3GalTase Activity

![Chemical Structure](https://i.imgur.com/chemical_structure.png)

FIG. 1. Flavonol glycoside biosynthesis in petunia pollen. The biosynthesis of flavonol glycosides in petunia pollen is catalyzed by two glycosyltransferase activities. F3GalTase forms the flavonol 3-O-galactoside (II) from the flavonol (I) and UDP-galactose. F2"GT forms the flavonol 3-O-glucosylgalactoside (III) from II and UDP-glucose. The flavonol (I) shown is kaempferol.

The two enzyme activities responsible for formation of the pollen-unique class of flavonol glycosides were identified as a F3GalTase, which used UDP-galactose as a substrate to convert flavonols to the flavonol 3-O-galactosides, and a flavonol 3-O-galactoside-2"-O-glucosyltransferase (F2"GT) which transfers glucose from UDP-glucose to the 2" position of galactose to form the flavonol glucosylgalactoside (Fig. 1). Both glycosyltransferase activities were simultaneously extracted from pollen and catalyzed the formation of the flavonol diglycosides *in vitro* (7). This study also demonstrated that kaempferol, quercetin, and UDP-galactose were the only endogenous substrates of F3GalTase, and that the flavonol 3-O-galactoside is the only endogenous substrate that the F2"GT activity glucosylates (e.g., it does not glucosylate the flavonol). Importantly, formation of the flavonol 3-O-galactoside was a prerequisite to formation of the flavonol diglycoside (7).

Glycosylation is one of the final steps in flavonoid biosynthesis (1, 13) and converts a chemically reactive molecule into a less reactive form. Presumably, the enhanced solubility of flavonoid glycosides aids their intracellular transport and sequestration (14, 15). Although the cellular location of flavonol glycosides is unknown, another class of flavonoids, the anthocyanidin glucosides are targeted to the vacuole following conjugation with glutathione (16). Glycosyl conjugation might also function to provide a reservoir of active molecules. The activity of plant growth regulators and other signal molecules is proposed to be regulated by interconversion between the active aglycone and the inactive glycosyl conjugate (15). The alternative roles of glycosylation are not mutually exclusive and in the case of flavonols there is evidence for both. High levels of flavonols are cytotoxic (8) and flavonols induce germination but the glucosylgalactosides accumulate in pollen (5, 8, 17, 18).

We want to establish the mechanism by which flavonols induce pollen to germinate and the role of glycosyl conjugate formation in this process. F3GalTase activity was deemed to be a critical component of flavonol-mediated pollen germination because: 1) F3GalTase is the sole activity in petunia pollen that metabolizes flavonols; 2) formation of flavonol glucosylgalactosides in petunia pollen is strictly dependent on F3GalTase activity; and 3) F3GalTase activity, through its forward and reverse reactions, has the potential to modulate the abundance of the active molecule. We used a directed approach, exploiting amino acid sequences in the native protein, to isolate a F3GalTase cDNA clone. Heterologous expression of the F3GalTase cDNA in Escherichia coli resulted in obtaining soluble, active F3GalTase which was used for biochemical characterization. The resulting information was used to construct a model of how F3GalTase activity might function during flavonol-induced pollen germination.

### EXPERIMENTAL PROCEDURES

Plant Material—Pollen from anthers of stage 2–9 flowers (17) of the flavonoid-deficient line of Petunia hybrida, termed CMF, previously characterized by Taylor and Jorgensen (6) was used as a source of both native F3GalTase and total RNA for PCR and Northern blot analyses.

All manipulations following excision of the protein from the gel were performed at the Protein and Nucleic Acid Facility at Stanford University Medical Center.

$cDNA$ Library Construction—cDNA was synthesized using the SuperScript Lambda System (Life Technologies, Inc.) from 2 $\mu g$ of poly(A)$^{+}$ RNA isolated from stage 6 CMF pollen. Size-fractionated cDNA (20 ng) was ligated into 0.5 $\mu g$ of $NotI/SalI$-digested ZipLox DNA (Life Technologies, Inc.) and the resulting constructs were packaged in vitro using the Gigapack II System (Stratagene). The library was amplified by plating on $E$. coli host strain Y1090 and contained $2.3 \times 10^{9}$ phage ml${}^{-1}$.

Synthesis and Cloning of F3GalTase RACE PCR Products—To obtain a nucleotide probe for screening the petunia pollen cDNA library, a RACE PCR protocol (21) was performed using total RNA isolated from stage 6 pollen and the following primers in the 5′ to 3′ direction: primer 1, CTGAGAGAACTAGTCTCGAGCTCTAGAACAAAGC(T)$_{17}$; 2, CTGAGAGAACTAGTCTCGAG; 3, TCTCGAGCTCTAGAACAAAGC; 4, AAYTTYITYAAYGTNGGNCC; 5, GTNGCNACNCCNCCNCC; 6, GGICCIACRTTIARRAARTTRTTRAA; 7, CAAATCACAGGCACACCAGC; 8, AAC TTGTACTTGTGGTGCCC; 9, GCTAGTCGTGCATATGTCCAATTACC ATG; 10, GGGTCAGCTCGAGTTAATTGCAAGAGTG.

First strand cDNA synthesis was performed in 1× reaction buffer (50 mM Tris-HCl, pH 8.3, 75 mM KCl, 3 mM MgCl$_{2}$, and 10 mM DTT) containing 5 $\mu g$ of total RNA, 10 units of RNasin, 25 pmol of primer 1, 0.5 mM dNTPs, and 200 units of Superscript II RNase H$^{-}$ reverse transcriptase (Life Technologies, Inc.) in 20 $\mu l$ final volume. The reaction was incubated for 50 min at 45 °C and terminated by heating at 70 °C for 15 min. After cooling on ice, 2 units of $E$. coli RNase H was added and the mixture was incubated for an additional 20 min at 37 °C. The RNase H-treated cDNA was diluted into 0.5 ml of TE buffer and stored at −80 °C. A 3′-RACE PCR product was generated using 3 $\mu l$ of the cDNA as template, 25 pmol of primers 2 and 4, and 2.5 units of Taq DNA polymerase (Life Technologies, Inc.) in 50 $\mu l$ of reaction volume. The addition of the other components were as described by the manufacturer. The cycling parameters were: 2 min hot start at 94 °C, 5 min at 45 °C, 20 min at 72 °C, one cycle; 60 s at 94 °C, 60 s at 45 °C, 1.25 min at 72 °C, 35 cycles; final extension for 10 min at 72 °C. One $\mu l$ of a 1:1000 dilution (in TE) of the reaction was used as template in a nested PCR experiment using 25 pmol of primers 3 and 5 and other reaction components as described previously. Cycling conditions were: 2 min hot start at 94 °C; 60 s at 94 °C, 30 s at 45 °C, 1 min at 72 °C, 30 cycles; final extension for 10 min at 72 °C. The most abundant amplicon at about 600 bp was ligated into pCR2.1 (Invitrogen) and transformed into $E$. coli TOP10F competent cells according to the manufacturer’s instructions. The deduced peptide sequences from three of the five plasmids subjected to nucleotide sequencing contained all of peptide 3 and residues 14–20 of peptide 2 (Fig. 3). The deduced peptide sequence of the plasmid, designated pF3GalTase3′RACE, showed a high level of identity with flavonoid glucosyltransferase from cassava (22), grape (23), and maize (24, 25).

A 5′-RACE PCR product was obtained using 5 $\mu g$ of total RNA from stage 6 pollen as described previously for the 3′-RACE protocol except that 10 pmol of primer 7 was used. Following RNase H treatment, the mixture was passed through Sephadex G-25 resin (Amersham Pharmacia Biotech), evaporated to 10 $\mu l$, and a poly(A) tail was added to the 5′ end of the cDNA using terminal deoxynucleotidyl transferase (Life Technologies, Inc.) according to the manufacturer’s instructions. The reaction mixture was diluted into 250 $\mu l$ of TE buffer and used as the 5′-RACE cDNA pool. PCR was performed on 3 $\mu l$ of this cDNA using 10 pmol of primer 1 and 25 pmol of primers 2 and 8. Other components and the cycling parameters were as described previously for the nested 3′-RACE PCR experiment. A nested PCR experiment was conducted on 3 $\mu l$ of a 1:100 dilution (in TE buffer) of the first 5′-RACE PCR experiment using the previously described components except that 25 pmol of primers 3 and 6 were used. The cycling conditions were: 2 min hot start at 94 °C; 30 s at 94 °C, 30 s at 58 °C, 60 s at 72 °C, every two cycles the annealing temperature was decreased by 2 °C until 48 °C was reached; 30 s at 94 °C, 30 s at 48 °C, 60 s at 72 °C, 15 cycles; final extension for 10 min at 72 °C. The most abundant amplicon (∼700 bp) was cloned as described previously and the deduced amino acid sequence of the resulting 725-bp clone, designated as pF3GalTase5′RACE, contained residues 1–9 of peptide 2 and all residues of peptide 1 (Fig. 3).

Isolation of F3GalTase cDNA—Approximately $1.0 \times 10^{5}$ plaque forming units of the previously described stage 6 petunia pollen cDNA library were plated onto $E$. coli Y1090 host cells, the DNA transferred to nylon filters, and hybridized with the 5′-RACE PCR product labeled with ${ }^{[32]}$PdCTP. Positive plaques were purified by three rounds of plating and hybridization. In vivo excision of the pZIP plasmid from the phage in the positive plaques was unsuccessful so $\lambda$ phage DNA was isolated and digested with $NotI/SalII$. The resulting restriction fragment (about 1.5 kbp) was gel purified and cloned into the pBluescript (SK+) phagemid (Stratagene) for sequence analysis. The deduced amino acid sequence from the resulting 1.431-kbp clone (pF3GalTasel) was determined to contain the sequences in peptides 1–3 (Fig. 3) as well as the nucleotide sequences comprising both the 3′ and 5′-RACE PCR products.

Tissue-specific and Developmental Expression of F3GalTase mRNA—Total RNA was extracted from anthers or pollen of stages 2, 4, 6, 7, 8, and 9 flowers as well as leaf, corolla, stigma/style, and seedling (shoot + root) tissues and 10–15 $\mu g$ was subjected to RNA gel blot analysis. Hybridization and washing conditions are described in Guyon et al.$^{2}$ To quantify the expression of F3GalTase mRNA, the autoradiographs were scanned and analyzed by the NIH Image 1.62 software program. The mRNA signals were normalized to the rRNA signals and mRNA/rRNA ratios were calculated for each sample.

F3GalTase cDNA Expression in $E$. coli—The pF3GalTasel plasmid was used as a template to generate a PCR product suitable for expression in $E$. coli. A 100-$\mu l$ reaction used 5 units of native Pfu DNA polymerase (Stratagene), 10 ng of pF3GalTasel plasmid, and 50 pmol of primers 9 and 10 at an annealing temperature of 55 °C. The other reaction components and the cycling parameters were performed according to the manufacturer’s instructions for a 2.0-kbp template. The resulting amplicon was gel-purified, digested with $NdeI/XhoI$, ligated into pET28a (Novagen), and used to transform $E$. coli BL21(DE3) cells (Stratagene). For expression, the resulting plasmid (pETF3GalTase) in BL21(DE3) cells was grown at 37 °C in LB medium + 30 $\mu g$ ml${}^{-1}$ kanamycin to an OD${}_{600}=0.4-0.6$. The culture was adjusted to 22 °C and rF3GalTase expression was induced by the addition of IPTG to a final concentration of 5.0 $\mu M$. The culture was incubated for at least 16 h at 22 °C (300 rpm) prior to harvesting the cells for rF3GalTase purification. Cells were harvested by centrifugation, resuspended in 0.1 M sodium phosphate buffer, pH 8.0, 0.5 M KCl, 1% (v/v) Triton X-100, 1 mM phenylmethylsulfonyl fluoride, 0.01 M imidazole, 2.0 mg ml${}^{-1}$ lysozyme, and frozen in liquid nitrogen. The cells were thawed and after 5 min at room temperature, the lysate was adjusted to 5 mM MgCl${}_{2}$ and DNase was added to a final concentration of 0.1 mg ml${}^{-1}$. The lysate was incubated for an additional 10 min and the cell debris was pelleted by centrifugation at 12,000 × $g$ for 5 min (4 °C). The supernatant was decanted and Ni-NTA-agarose resin (Qiagen) was added (0.01 ml of the 50% slurry, as supplied by the manufacturer, per ml of lysate) and mixed with the lysate for 1 h at room temperature. The resin was pelleted by brief centrifugation, washed two times, and eluted with 0.2 M imidazole according to the manufacturer’s instructions except that 1% Triton X-100 was included in the wash and elution buffers. The N-terminal 6× His-tag was then removed using thrombin at 10 units mg${}^{-1}$ rF3GalTase, in elution buffer, at 4 °C for 16 h. To terminate the reaction, phenylmethylsulfonyl fluoride was added to 1 mM. rF3GalTase was diluted 10-fold in wash buffer, excluding the imidazole, and incubated with Ni-NTA-agarose resin as described previously for 1 h at room temperature. The resin was pelleted by centrifugation and rF3GalTase in the supernatant was quantified using the BCA protein assay (Pierce).

F3GalTase Enzyme Assay—rF3GalTase activity in the forward direction was measured in 50 mM HEPES-OH, pH 7.5, 75 mM KCl, 50 $\mu$M MnCl${}_{2}$, 1 mM DTT, 5% glycerol, 5 mM UDP-galactose, and 2.5 ng of purified rF3GalTase in a 50-$\mu$l final volume. The mixture was incubated at 43 °C for 3 min and the reaction was initiated by adding various concentrations of kaempferol or quercetin in dimethyl sulfoxide (final dimethyl sulfoxide concentration, 1%), and vortexed briefly to mix. After 3 min, the reaction was terminated with 2 volumes of 100% methanol and the assay mixture centrifuged at 12,000 × $g$ for 5 min. The amount of flavonol 3-$O$-galactoside in 50 $\mu l$ of the supernatant was determined by HPLC. Quantification was performed by integrating the peak area and using the first order regression equation derived from calibration curves of kaempferol 3-$O$-galactoside and quercetin 3-$O$-galactoside standards. One unit of rF3GalTase activity is defined as that amount of enzyme which catalyzes the formation of 1 $\mu$mol of flavonol 3-$O$-galactoside in 1 min using the described conditions. rF3GalTase activity in the reverse direction was measured as described previously except that 50 mM MES-Tris buffer, pH 6.0, and 5 mM UDP were used. The reaction was initiated by adding various concentrations of flavonol 3-$O$-galactoside (in dimethyl sulfoxide). The calibration curves for quantification of flavonols were based on kaempferol and

$^{2}$ V. Guyon, J. Astwood, and L. P. Taylor, submitted for publication.

Petunia Pollen-expressed F3GalTase Activity

M<sub>r</sub> (kDa)
- 97.4
- 66.2
- 45
- 31
- 21.5

1 2 3 4 5

FIG. 2. SDS-PAGE of proteins in fractions from the native F3GalTase purification protocol. Lane 1, crude homogenate; lane 2, High-Q preparative anion exchange chromatography eluate; lane 3, Mono-Q 16/10 anion exchange chromatography eluate; lane 4, Mono-Q HR 5/5 anion exchange chromatography eluate; lane 5, UDP-diphosphohexanolamine affinity chromatography eluate. Proteins were size fractionated through 12% (w/v) acrylamide and visualized by silver staining. The migration of standard proteins of known mass are shown on the right-hand side of the gel.

quercetin standards. One unit of rF3GalTase activity in the reverse direction is defined as that amount of enzyme which catalyzes the formation of 1 μmol of kaempferol or quercetin in 1 min using the described conditions. Anthocyanidins were assayed as potential substrates according to the method previously described (26).

Temperature/pH Optimum and Native Molecular Mass of rF3GalTase—The activity of rF3GalTase in the forward direction was evaluated at temperatures from 35 to 60°C using the previously described assay conditions except that the indicated temperatures were used. rF3GalTase activity in both the forward and reverse directions were evaluated from pH 4.5–9.0 using two buffer systems: MES-Pi (pH 4.5–6.5) and Tris-MES, pH 6.5–9.0. All other components in the assay mixture and reaction conditions were as described previously. Non-denaturing PAGE was used to calculate the native mass of purified rF3GalTase according to the method described by Hedrick and Smith (27). MW-ND-500 kit (Sigma) native standard proteins were used to construct the standard curve as described previously (Sigma Technical Bulletin MKR-137).

Steady-state Kinetic Analyses of rF3GalTase—The initial velocity versus substrate concentration curves for the rF3GalTase reaction in the forward direction were determined using a fixed UDP-galactose concentration of 5 mM and varying the concentration of flavonol from 1 to 25 μM, or using a fixed flavonol concentration of 25 μM and varying the concentration of UDP-galactose from 0.1 to 5 mM. For the rF3GalTase reaction in the reverse direction, a fixed UDP concentration of 5 mM was used while varying the concentration of flavonol 3-O-galactoside from 1 to 25 μM, or using a fixed flavonol 3-O-galactoside concentration of 25 μM and varying the concentration of UDP from 0.1 to 5 mM. Enzyme assays were performed as described previously. \(K_m\) and \(V_{\text{max}}\) values were estimated after linear transformation of the initial velocity versus substrate concentration data points using Hanes-Woolf plots. \(K_{\text{cat}}\) values were calculated using the predicted rF3GalTase \(M_r\) of 48,907 g mol<sup>-1</sup>.

RESULTS

Purification and Sequencing of Native F3GalTase—Previous flavonoid galactosyltransferase purification procedures (28, 29) used a combination of ammonium sulfate fractionation, chromatography on anion exchange, size exclusion, hydrophobic interaction, hydroxyapatite, and Mono-P matrices. All of these, except the hydroxyapatite, were evaluated for F3GalTase purification and, with the exception of anion exchange chromatography, were unsatisfactory. It was determined that F3GalTase precipitated at its pI value of 5.1, thus chromatography through a pH gradient was not a feasible purification step.

The purification of native F3GalTase from stage 6 CMF pollen was accomplished using preparative and analytical anion exchange chromatography matrices. A silver-stained SDS-PAGE analysis of the fractions from the purification are shown in Fig. 2. Affinity chromatography on UDP diphosphohexanol-

---

TTGTCCAAAATGTCGAATTACCATGTTGCTGTTCTAGCATTTCCTTTTGCAACACATGCTGGCCTTTTA 69
L G L V Q R L A N A L P N V T F T F F N T S K 43
P1
S N S S L F T T P H D N N I K P F N I S D G V 66
TCAAATTCTTCATTATTCACTACTCCTCATGACAACAAACATTTAACCGTTTAATATCTCCGATGGCGTC 207
P E G Y V V G K G I E A L I G L F F K S A K 89
CCGGAGGGTTACGTGGTCGGAAGGAGGTATTGAAGCGCTTATAGGGTTGTTCTTTAAGTCTGCTAAA 276
E N I Q N A M A A A V E E S G K K I T C V M A 112
GAGAATATTCAAATGCATGGCAGCTGCTGGGAGAATCGGGAAGAAGATTACTTGTGTTATGGCA 345
D A F M W F S G E I A E E L S V G W I P L W T 135
GATGCATTTATGTGGTTTCTCGTGAGATGCTGAGGAATTGAGCGTTGGTTGGATCCCTTTAAGGACT 414
S A A G S L S V H V Y T D L I R E N V E A Q G 158
TCTGCTGCTGGGATCACTCTCTGTTCACTGTTTACACTGATCTCATTAGAGAAAAATGTTGAGGCTCAAGGA 483
I A G R E D E I L T F I P G F A E L R L G S L 181
ATTGCTGGACGTGAAAGATGAAATTCTAAACATTCCAGGATTTGCCGAATTACGGCTAGGTAGTTTG 552
P S G V V S G D L E S P F S V M L H K M G K T 204
CCTAGTGGAGTTGTCTCTGGGAGACTTGGAAATCACCTTTCCGTAATGCTTCACAAAATGGGCAAAACA 621
I G K A T A L P V N S F E E L D P P I V E D L 227
ATAGGAAAAGCCACTGCCTTGCAGTAAACTCCTTTGAAGAACTAGACCCTCAATTGTTGAAGATCTC 690
P2
K S K F N N F L N V G P F N L T T P P P S A N 250
AAGTCCAAATCCAATAACTT CCTCAATGTGGTCCC TTTAACTTAACTACCCCACCACCTCAGCAAAC 759
P3
I T D E Y G C I A W L D K Q E P G S V A Y I G 273
ATTACAGATGAATATGGGTGCATTCATGGCTAGACAAACAAAGACCAAGGCTCAGTGGCTTACATAGGA 828
F G T V A T P P P N E L K A M A E A L E E S K 296
TTTGGCACGGTGGCTACACCACCACTAATGAGCTAAAAGCTATGGGCTGAAGCACTTGAAGAAAGTAAA 897
T P F L W S L K D L F K S F F P E G F L E R T 319
ACTCCTTTTCTTTGGTCATTAAAGACCTTTTCAATTTTTTCCAGAAGTTTTTTGGAAGAACT 966
S E Y G K I V S W A P Q V V L S H G S V G V 342
AGTGAGTATGGTAAAATTGTGTCATGGGCACCACAAGTACAAGTTCTTAGTCATGGTTCAGTTGGTGT 1035
F I N H C G W N S V L E S I A A G V P V I C R 365
TTTATAATCATGTGGATGGAACTCTGTTTTGGAGAGTATAGCTGCTGGTGCCCTGTGATTGTAGG 1104
P F F G D H Q L N A W M V E K V W K I G V K I 388
CCATTTTTTTGGAGATCATCAATTGAATGCTTGGATGGTTGAGAAAATATGGAAAATTGGAGT GAAAATT 1173
E G G V F T K D G T M L A L D L V L S K D K R 411
GAAGGAGGAGTTTTTACTAAAAGATGGAAACAAATGCTTGCACTTGACTGGTTTATCAAAAGACAAAAGG 1242
N T E L K Q Q I G M Y K E L A L N A V G P S G 434
AATACAGAATTGAAACAAACAAATTGGGATGTAAAAAGACTTGCTCAAATGCTGTTGGACCAAGTGGG 1311
S S A E N F K K L V D I I T S C N Stop
AGCTCTGCTGAAAATTTCAAGAAATTTGTTGATATTATCACCTCTTGCAATTTAAATTTTACTTGAA 1380
GAGATTCAATGATTTTACTATCTGTTGAAATAATAATCCACTTTAATTT 1431

FIG. 3. Nucleotide and deduced amino acid sequence of the petunia pollen F3GalTase cDNA. The nucleotide sequence of the NotI/SalI insert of the pF3GalTasel plasmid is shown as well as the deduced amino acid sequence of the 1.353-kbp open reading frame. Sequenced peptides 1, 2, and 3 obtained from endopeptidase-treated native F3GalTase are underlined and the initiation codon and polyadenylation signal are in bold.

amine-agarose resulted in a near homogenous F3GalTase preparation which was sufficient for determining that the size of the F3GalTase peptide was about 45 kDa. However, the yield of active protein from the column was unacceptable for further biochemical characterization, so preparative SDS-PAGE was used to obtain sufficient material for protein sequencing. The 45-kDa protein in the Mono HR 5/5 eluate was gel purified and subjected to proteolytic digestion with Lys-C endopeptidase followed by N-terminal sequencing of the HPLC-purified peptide fragments. The sequence of three peptides, P1, P2, and P3 are shown in Fig. 3.

Synthesis and Cloning of F3GalTase RACE PCR Products and Isolation of the Full-length F3GalTase cDNA—The sequence of a 3′-RACE product, generated using degenerate oligonucleotide primers to the native F3GalTase protein sequence, showed a significant level of positional identity with glucosyltransferases from several plant species and mammalian tissues. The sequences responsible for this identity are a highly conserved region in the C terminus of glucosyltransferases from numerous eukaryotic and prokaryotic sources (22, 26, 30). To increase the likelihood of recovering a F3GalTase clone, we synthesized a 5′-RACE product with the expectation it would more likely contain gene-specific sequences. The deduced amino acid sequence of the resulting 725-bp PCR product

Petunia Pollen-expressed F3GalTase Activity

P. hybrid a 1 MSN---------YHVAVLAFPFATHAGLLLGLVQRLLANAL---PNVTFTFFNTSKSNSSLFTT--------PH
V. vinifera 1 TTN---------PHVAVLAFPFSSTHAAPLLAVVRRLLAAAA---PAVFSFFSTSQSNASIFHDSM------HTM
V. mungo    1 MGNSE------EKKHVAVSFPFGSHEPTPLLNLVLKLTNAA---PNLQFSFIGTEHBNKSLLISKP------HI
H. vulgare  1 MAP--------PPPHIAVVAAFPSHHAALFESFARALAAAA--PAGTLSFLTTADNAAQLRKAG-------AL
Z. mays     1 MAPADGESSPPFHVAVVAAFPSHHAALLSIARALAAAAAPS GATLSFLSTASSLAQLRKASSASAGHGL
R. norvegicus 1 MKS---------YTPYFMLLLWS---AVGIARAAKIIVP---PIMFEHLYIFKTLASALHER--------GH

P. hybrid a 54 DNNIKPFNISDGVPPEGYVGKGGI----EALIGLFFKSAKEN-TQNAMAAAVEESG-KKITCVMDAFMW
V. vinifera 57 QCNIKSYDISDGVPPEGYVFA-GRP---QEDIELFTRAAPES-FRQGMVMVAETG-RPWSCLVADAFIW
V. mungo    60 PDTIKFYSISDGVPEGHVPG-GHP---VERVNLFQASPQN-LQKGIDMAVAHTK-ERVTCSVISDAFVA
H. vulgare  58 PGNLRFVEVPDGVPGETSCLSPB---RRMDLFMAAEAGGVRVGLEAASAGGARWSCVGDADFVW
Z. mays     71 PGNLRFVEVPDGAPAAEESVPVP----RQMQLFMEAAEAGGVKAWLEAARRAAGGARWTCVVDADFVW
R. norvegicus 51 HTVLLLSEGRDIAPSNHYSLQRYEGIFNSTTSDAFLQSKMRN-IFSGRLTAVELVDILDHYTKNCDMMVG

P. hybrid a 118 FSGEIAEELS--VGWIPFWTS AAG-SLSVHVY-TDLIRENVEAQGIAQR-EDFILTFIPGFAEL---RLG
V. vinifera 120 FAADMAAEMG--VAWLPFWTAGPN-SLSTHVY-IDEREKIGVSGIQQR-EDELLNFIPGMSKV---RFR
V. mungo    123 PSLTVARQLN--VPWVPVWPLSC-SLSAHFY-TELIRQTCNSA--AG---DTPLDFVPGLSKM---RVE
H. vulgare  123 -TADAASAAG--APWVAWTTAAC-ALLAHLR-TDALRRDVGDQA-ASR-ADELLVAHAGLGGY---RVR
Z. mays     135 PAADAAASAG--APWVEWTTAAC-ALLAHIR-TDALREDVGDQA-ANR-VDEPLISHGLASY---RVR
R. norvegicus 120 NQALIQGLKKEKFDLLLVDPNDMCGFVIAHLLGVKYAVFSTGLWYPAEVGAPAPLAYDEFNSLLTD RMN

P. hybrid a 180 SLP-----SGVVSGD---LESPFSVMLHKMGKTIGK-ATALPVNSPEELDPP-IVEDLKSKFN--NFLNVG
V. vinifera 182 DLQ-----EGIVFGN---LNSLFSRMLHRMGQVLPK-ATAVFINSPEELDDS-LTNDLKSKLK--TYLNIG
V. mungo    181 DLP-----EDVIQGAGE-EETLFSKTLASLGSVLPQ-AEAVVVNFFEEELDPPLLVNDMKSKFK--YYLYVG
H. vulgare  183 DLP-----DGVVSGD---FNYVISLLVHRQARLPKAATAVALNTEPGFLDPEDLIAALAELP--NCLPLG
Z. mays     196 DLP-----DGVVSGD---FNYVINLLVHRMQCLPRSAAAVALNTEPGFLDPEDVTAAALEILP--NCVPFG
R. norvegicus 190 FLERMKNTGVYLISRIGVSFLVPKYERIMQYNLLPAKSMYDLVHGSSLWMLCDVALEFPRPTLPNVV

P. hybrid a 239 PFNLTTPPPSA--N---ITDEYGCIAWLDKQEPGSVAYIGFGT-VATPPPNELKAMRAALEESKTPFLWS
V. vinifera 241 PFNLITPPP---V---VPNTTGCLQWLKERKPTSVYISFGT-VTTPPPEAVVALEASRVPIWS
V. mungo    243 FLTLSLPLPPLPPS---DTDETGCLSWLDKQKGGSVVYVSFGT-VTTPPHEIVAVAELASGFPFLWS
H. vulgare  244 PYHLLFGAEPDTADTNEAPADPHGCLAWLDRRPARSVAYVSFGT-NATARPDELQELAAGLEASGAPFLWS
Z. mays     257 PYHLLL-AEDDADT-AAAPADPHGCLAWLGRQPARGVAYVSFGT-VACPRDELRELAAGLEASGAPFLWS
R. norvegicus 260 YVGGILTKP----A---SPLPEDLQRWVSGAQEHGFVLVSFGAGVKYLSEDIANKLAGRLPQKVIWR

P. hybrid a 303 LKDLFKSFFEBGFLERTSEY---GKIVSWAPQVQVLSHGSVGVFINHCGWNSVLESIAAGVPVICRFFF GD
V. vinifera 303 LRDKARVHLPEGFLEKTRGY---GMVVWPAPQAEVLAEVGAFTHCGWNSLWESVAGGVPLICRFFF GD
V. mungo    309 LKEHLKGVLPNGFLERTSER---GKVVGWAPQTQVLGHGSVGVFTHCGCNSVFEMSNGVPMICRFFF GD
H. vulgare  313 LR-GVVAAAPRGFLERAP----GLVVWPAPQVGVLRHAAVGAFVTHAGWASVMEGVSSGVPMACRFFF GD
Z. mays     324 LREDSWTLLEPGFLDRAGTGSLVVWPAPQVAVLRHPSVGAFVTHAGWASVLEGVSSGVPMACRFFF GD
R. norvegicus 323 FS-----GTKPKNLGNNSK-----LIEWLPQNDLLGHSNIRAFLSHGLNSIFETMYHGPVVGIPLFGD

P. hybrid a 371 HQLNAWMVEKVWKIGVKIEGGVFTKDGTMLLDLVLSDKKRNTELKQQIGMYKELALNMVG-PGSSS---
V. vinifera 371 QRLNGRMVEDVLEIGVRIEGGVFTKGLMSCFDQILSQEK-GKKLRENLRALERETADRAVG-PKGSS---
V. mungo    377 HGLTGRMVDEVWEIGVRVEGGVFTKDGLLSKLRLILVEEE-GNLMMKNAVKVKTVDLAG-AQGKA---
H. vulgare  378 QTMNARSVASVWGFGTAFDG--PMTRGAVANAVATLLRGED-GERRMAKAQELQAMVGKAFE-PDGGC---
Z. mays     394 QRMNARSVAHWGFGAAFEC-AMTSAGVAAAVEELLRGEF-GARMRARAKVQALVAEAFG-PGGEC---
R. norvegicus 383 HYDTMTRVQ-AKGMGILLEWN TVTEGELYDALVKVINNPS---YRQRAQKLEITHKDQPCHPVNR TTYW

P. hybrid a 437 ----------AENFKKLVDIITSCN---
V. vinifera 436 ----------TENFITLVDLVSKPKDV-
V. mungo    442 ----------AQDFNTLVELVSRS----
H. vulgare  442 ----------RKNFDEFVEIVCRV-----
Z. mays     458 ----------RKNFDRFVEIVCRA-----
R. norvegicus 448 IDYILRHDGARHLRSAVHQISFCQYFL


(pF3GalTase5′RACE) contained the sequence of peptide 1 and residues 1–9 of peptide 2 (Fig. 3). When pF3GalTase5′RACE was used as a hybridization probe to screen the pollen cDNA library, a 1.431-kbp cDNA clone (pF3GalTase1, Fig. 3) containing a single open reading frame of 1.353 kbp was isolated. The translated peptide of the pF3GalTase1 cDNA clone is 451 residues with a predicted molecular mass of 48.9 kDa and contains all of the sequences of peptides 1–3 (Fig. 3). Residues 1–30 potentially comprise a cleavable signal sequence (PSORT program) reducing the mass of the peptide to 45.7 kDa, which corresponds to the observed mass of native F3GalTase (Fig. 2).

The deduced peptide sequence of pF3GalTasel showed significant identity with several plant glycosyltransferases and an alignment with representatives of two dicot and two monocot flavonoid glycosyltransferases as well as a mammalian ceramide galactosyltransferase are shown in Fig. 4. The highest identity was with a grape flavonoid glucosyltransferase (53%) (26) and a mung bean flavonoid galactosyltransferase (49%) (29). Lower but significant identity was observed with the flavonoid glucosyltransferases from maize (37%) (24, 25) and barley (36%) (31). Although F3GalTase has significant identity with other flavonol glycosyltransferases, it contains putative regulatory motifs that are not present in any of the four plant sequences in Fig. 4. The predicted motifs were determined using the MOTIF program and include a single amidation site, two cAMP/cGMP-dependent protein kinase phosphorylation sites, six casein kinase II phosphorylation sites, four protein kinase C phosphorylation sites, eight N-myristoylation sites, and six N-glycosylation sites. It is unknown if native F3GalTase is modified at any of these sites.

The high level of sequence identity in the region spanning residues 328 to 370 in the petunia F3GalTase sequence has been noted in a number of glycosyltransferases from bacteria, plants, and animals. Based solely on the similar function of these proteins, part or all of the region has been proposed as a UDP-binding site (30) or a plant secondary product glucosyltransferase gene signature sequence (22, 26). The conservation of these sequences across kingdoms is highlighted by the fact that even though the overall positional identity with a rat brain ceramide galactosyltransferase is 19%, within this 43-amino acid sequence it rises to 44% (32, 33). The predicted amino acid sequences of 11 glycosyltransferases, all of which are predicted to conjugate small organic molecules such as flavonoids, auxins, and zeaxanthin, were aligned using GCG PILEUP with a Blosum 62 matrix (data not shown). From residues 328 to 370 the overall identity for the 10 plant and one bacterial protein was greater than 50% and included contiguous residues ((WA/CPQ) and (HC/AGWS)) with 90% positional identity. These residues are obvious targets for site-directed mutagenesis to determine their potential role in enzyme activity.

Tissue-specific and Developmental Expression of F3GalTase mRNA—RNA gel blot analysis was used to determine the tissue-specific and developmental expression of F3GalTase mRNA. Fig. 5 shows that F3GalTase mRNA accumulation was restricted to pollen; transcripts were not detected, even after prolonged exposure of the autoradiograph, in any of the sporophytic tissues tested (seedling, leaf, stigma/style, and corolla). The lack of F3GalTase mRNA in sporophytic tissues was corroborated by an inability to detect F3GalTase enzyme activity in total soluble protein extracts from these tissues (data not shown). Thus, F3GalTase is expressed specifically in the male gametophyte where it modifies a molecule required for functional pollen.

A developmental profile of F3GalTase mRNA accumulation showed that transcripts were present throughout pollen development with the highest steady state levels in stage 4 of V26,

Petunia Pollen-expressed F3GalTase Activity

![Image Description](image1)

FIG. 5. RNA gel blot analyses of developmental and tissue-specific F3GalTase mRNA expression. Total RNA from V26 (10 μg) and CMF (15 μg) pollen at various developmental stages and from the tissues indicated were size fractionated, transferred to nylon membranes, and hybridized with pF3GalTase1 (F3GalTase) or ribosomal DNA (rRNA) probes. The autoradiographs were scanned and the hybridization signals were quantified and normalized as described under “Experimental Procedures.” The normalized value of the mRNA/rRNA ratio is displayed under each sample (m.a.g., minutes after germination; dehisced is mature pollen released from anther).

and, in stage 6 of CMF, pollen. In wild-type (V26) pollen during the progamic phase of development, i.e. after the pollen is released from the anther and during germination, F3GalTase mRNA levels decreased to about 6% of the levels in stage 4 pollen. This pattern of F3GalTase expression parallels the accumulation of flavonol diglycosides which peak at stage 9 and decrease with maturity in V26 anthers (stage 10–12) (17).

**F3GalTase cDNA Expression in E. coli and Substrate Usage of the Recombinant Enzyme—**The pF3GalTase1 plasmid was used as a template to construct, via PCR, an insert for expression in E. coli. rF3GalTase was expressed in a soluble active form by growth for 16 h at 22 °C in the presence of 5.0 μM IPTG (Fig. 6). At temperatures above 22 °C or at IPTG concentrations above 5.0 μM, rF3GalTase expression resulted in the formation of inclusion bodies and no detectable rF3GalTase activity in the soluble fraction. In the absence of the pETF3GalTase plasmid, rF3GalTase activity in crude lysates from BL21(DE3) cells was not detectable.

The recombinant enzyme retained the same substrate specificity as native F3GalTase (7, 18), accepting only UDP-galactose and flavonols as substrates. In addition to the endogenous flavonols kaempferol and quercetin, rF3GalTase also conjugated galangin, iso-rhamnetin, kaempferide, rhamnetin, morin, myricetin, and fisetin (Table I). Importantly, all of these flavonols induce pollen germination. Other flavonoids including the flavone apigenin, the flavanone naringenin, the dihydroflavonol dihydroquercetin, and the anthocyanidins pelargonidin, delphinidin, and cyanidin were tested and none were used as substrates by rF3GalTase. Likewise, a sample of biologically active small molecules (listed under “Experimental Procedures”), including several plant growth regulators, were not used as substrates by rF3GalTase.

Substrate usage was determined for the reverse reaction catalyzed by F3GalTase and was shown to be as specific as the forward reaction. rF3galTase assayed in the reverse direction accepted only kaempferol 3-O- and quercetin 3-O-galactosides as substrates and not the respective flavonol 3-O-glucosides, similar to that reported for native F3GalTase (18).

**Temperature/pH Optimum and Native Molecular Mass of rF3GalTase—**rF3GalTase activity was determined at temperatures from 35 to 60 °C and maximum activity was observed at 50 °C (Fig. 7). However, incubation of the enzyme at 50 °C for 15 min prior to initiating the reaction resulted in a 26% reduction of rF3GalTase activity compared with rF3GalTase assayed without preincubation (data not shown). Preincubation at

---

FIG. 6. SDS-PAGE of proteins in fractions from rF3GalTase expression in E. coli and of purified rF3GalTase. Lane 1, preinduction control; lane 2, total soluble protein from BL21(DE3) cells containing the pF3GalTase2 plasmid following treatment with 5.0 μM IPTG for 16 h at 22 °C; lane 3, rF3GalTase purified using Ni-NTA and anion exchange chromatography resins followed by treatment with thrombin to remove the His-tag; lane 4, protein size standards. 5 μg of protein (10 μg in lane 3) was size fractionated through 12% acrylamide and then stained with Coomassie Brilliant Blue to visualize the separated proteins.

TABLE I

Relative rF3GalTase activity with different substrates

The activity of rF3GalTase using 100 μM kaempferol as a substrate was set as 100%. Enzyme assays were conducted as described in the text except that each of the indicated substrates were used at 100 μM.

| Substrate               | Activity     |
|-------------------------|--------------|
| Kaempferol              | 100          |
| Myricetin               | 23.4 (±0.9)<sup>a</sup> |
| Fisetin                 | 23.3 (±1.4)  |
| Galangin                | 15.1 (±0.7)  |
| Iso-rhamnetin           | 13.0 (±1.7)  |
| Rhamnetin               | 6.9 (±0.4)   |
| Kaempferide             | ND<sup>b</sup> |
| Morin                   | ND           |
| Apigenin                | 0<sup>c</sup> |
| Naringenin              | 0            |
| Dihydroquercetin        | 0            |
| Pelargonidin            | 0            |
| Delphinidin             | 0            |
| Cyanidin                | 0            |
| Kaempferol 3-O-glucoside| 0<sup>d</sup> |
| Quercetin 3-O-glucoside | 0            |
| NAD<sup>+</sup>         | 0            |
| Ferrulic acid           | 0            |
| Sinapic acid            | 0            |
| Naphthylphthamic acid   | 0            |
| Gibberillic acid        | 0<sup>e</sup> |
| Indole acetic acid      | 0            |
| Abscisic acid           | 0            |
| Jasmonic acid           | 0            |

<sup>a</sup> Values in parentheses are the standard deviations of the mean of two separate experiments.

<sup>b</sup> ND, rF3GalTase activity was not detected using the standard amount of enzyme. Increasing the concentration of the enzyme 5-fold (kaempferide) to 100-fold (morin) resulted in rF3GalTase activity comparable to that obtained using kaempferol as a substrate in this experiment.

<sup>c</sup> All of the compounds which did not function in rF3GalTase assays (i.e. those with a value of 0%) were also assayed with 5-fold higher levels of enzyme. In all cases, the same results were obtained.

<sup>d</sup> Kaempferol and quercetin 3-O-glucosides were assayed for rF3GalTase activity in the reverse direction as described under “Experimental Procedures.”

<sup>e</sup> Gibberellic acid, indole acetic acid, abscisic acid, and jasmonic acid were assayed for rF3GalTase activity at both 43 °C and at room temperature with the same results.

43 °C for the same amount of time resulted in only a 5% reduction of activity so to avoid potential artifacts caused by thermal inactivation of the enzyme, rF3GalTase assays were conducted at 43 °C.

The activity of rF3GalTase in both the forward and reverse

Petunia Pollen-expressed F3GalTase Activity

FIG. 7. Temperature optimum of rF3GalTase activity. rF3GalTase activity in the forward direction was measured at temperatures from 35 to 60 °C as described under “Experimental Procedures.” The data points shown are the average of two replications.

FIG. 8. pH optimum of rF3GalTase activity. rF3GalTase activity was measured in both the forward and reverse directions between pH 4.5 and 9.5 as described under “Experimental Procedures.” The data points shown are the average of two replications.

directions was assayed from pH 4.5 to 9.5 (Fig. 8). Maximum rF3GalTase activity was measured in the forward direction at pH 7.5–8.0, whereas maximum activity in the reverse direction was measured at pH 6.0. The optimum pH of 7.5 for rF3GalTase in the forward direction agrees with the pH optimum of flavonoid glycosyltransferases from several other plant species and tissues (28, 34–36).

Purified rF3GalTase was analyzed in the absence of substrates by native PAGE (Fig. 9) and was chromatographed through Sephacryl S-200 size exclusion matrices (data not shown) to determine the native molecular mass and subunit composition. From these experiments, native rF3GalTase was calculated to be about 50 kDa, which is close to the predicted mass of 48,907 Da which was derived from the amino acid sequence. Thus, we conclude that rF3GalTase exists in its native conformation as a monomer.

Steady-state Kinetic Analyses of rF3GalTase—Purified rF3GalTase was used to determine the $K_m$, $V_{\text{max}}$, $k_{\text{cat}}$, and $k_{\text{cat}}/K_m$ values of kaempferol, quercetin, and UDP-galactose in the forward direction at pH 7.5 and kaempferol 3-O-galactoside, quercetin 3-O-galactoside, and UDP in the reverse direction at pH 6.0 (Table II). Using a saturating concentration of UDP-galactose, the $K_m$ for kaempferol was calculated as 1.1 μM with a $V_{\text{max}}$ of 75 units mg⁻¹, whereas for quercetin the $K_m$ value was 0.96 μM and the $V_{\text{max}}$ was 73.4 units mg⁻¹. Using a saturating concentration of flavonol, the $K_m$ of UDP-galactose was determined to be 426 μM with a $V_{\text{max}}$ of 75.1 units mg⁻¹ when kaempferol was the co-substrate and a $K_m$ value of 387 μM with a $V_{\text{max}}$ of 70.5 units mg⁻¹ when quercetin was the co-substrate. The $K_m$ values of kaempferol 3-O-galactoside, quercetin 3-O-galactoside, UDP with kaempferol 3-O-galactoside as the co-substrate, and UDP with quercetin 3-O-galactoside as the co-substrate were 8.47, 2.65, 66, and 102 μM, respectively, and the $V_{\text{max}}$ values were 73.5, 101, 62.5, and 110 units mg⁻¹, respectively. The $K_m$ values of kaempferol and UDP-galactose are similar to the values reported for a flavonol GalTase from mung bean leaf tissue (28).

The calculation of $k_{\text{cat}}$ and $k_{\text{cat}}/K_m$ values determined the turnover number and the catalytic efficiency, respectively, of rF3GalTase with each of the substrates tested. The values reported in Table II indicate that rF3GalTase is highly efficient: with the flavonol substrates, the catalytic efficiency is only 1.6–1.8-fold less than the lower end of a diffusion-controlled limit ($10^8$-$10^{12}$ M⁻¹ s⁻¹) of a chemical reaction rate in solution. The $k_{\text{cat}}/K_m$ values of rF3GalTase using kaempferol and quercetin in the forward direction, and quercetin 3-O-galactoside in the reverse reaction, is comparable to carbonic anhydrase when carbon dioxide is the substrate ($8.3 \times 10^7$ M⁻¹ s⁻¹) (37), or fumarase when malate is the substrate ($3.6 \times 10^7$ M⁻¹ s⁻¹) (38). Carbonic anhydrase and fumarase are enzymes considered to have attained virtual catalytic perfection. Because rF3GalTase is a reversible enzyme, its activity is reduced

FIG. 9. Nondenaturing PAGE analysis to estimate the native molecular mass and subunit composition of rF3GalTase. Purified rF3GalTase and standard proteins were run in nondenaturing PAGE gels of different acrylamide concentrations. Their migration ($R_F$ value) was plotted against acrylamide concentration and a second-order regression equation was used to fit the data points and obtain a slope value for each protein. The –slope value of each of the standard proteins were plotted against their known molecular mass (27) to obtain the calibration curve shown here. The standard proteins used were: 1, α-lactalbumin, 14.2 kDa; 2, carbonic anhydrase, 29 kDa; 3, chicken albumin, 45 kDa; 4, bovine serum albumin (monomer), 66 kDa; 5, bovine serum albumin (dimer), 132 kDa.

TABLE II

Steady-state kinetic parameters of rF3GalTase

Enzyme assays of pure rF3GalTase were conducted as described in the text. $K_m$ and $V_{\text{max}}$ values were estimated following linear transformation of the initial velocity versus substrate concentration data points. The $V_{\text{max}}$ values are the mean of at least two separate experiments.

| Substrate         | $V_{\text{max}}$ | $K_m$       | $k_{\text{cat}}$ | $k_{\text{cat}}/K_m$ |
|-------------------|------------------|--------------|------------------|----------------------|
| Kaempferol        | 75               | $1.1 \times 10^{-6}$ | 61.1              | $5.6 \times 10^7$   |
| Quercetin         | 73.4             | $9.6 \times 10^{-7}$ | 59.8              | $6.23 \times 10^7$  |
| UDP-Gal(K)        | 75.1             | $4.26 \times 10^{-4}$ | 61.2              | $1.44 \times 10^5$  |
| UDP-Gal(Q)        | 70.5             | $3.87 \times 10^{-4}$ | 57.4              | $1.48 \times 10^5$  |
| K-3-Gal           | 73.5             | $8.47 \times 10^{-6}$ | 59.9              | $7.07 \times 10^6$  |
| Q-3-Gal           | 101              | $2.65 \times 10^{-6}$ | 82.3              | $3.1 \times 10^7$   |
| UDP(K)            | 62.5             | $6.6 \times 10^{-5}$ | 50.9              | $7.7 \times 10^5$   |
| UDP(Q)            | 110              | $1.02 \times 10^{-4}$ | 89.6              | $8.8 \times 10^5$   |



by the presence of the reaction products in either direction. So that maximum catalytic rates can be measured throughout the linear portion of the assay, the reaction must be assayed, in the forward or reverse directions, with an excess of substrates. If the rF3GalTase reaction could be coupled with another enzyme which uses, for example, either UDP or kaempferol 3-O-galactoside as substrates at rates equal to or greater than the rF3GalTase catalyzed production of these compounds, then the \( k_{cat} / K_m \) of the rF3GalTase catalyzed reaction might be increased to the level where it achieves catalytic perfection.

DISCUSSION

The flavonols that induce pollen germination are rapidly metabolized by F3GalTase, a gametophytically expressed enzyme which catalyzes the reversible addition of kaempferol and quercetin to UDP-galactose to form the respective flavonol 3-O-galactosides. We exploited amino acid sequences in the native protein to isolate a full-length F3GalTase cDNA from a petunia pollen cDNA library. RNA gel blot analysis and enzyme assays of various tissues showed that F3GalTase is expressed exclusively in pollen. Conclusive proof that the cDNA encoded F3GalTase was provided by heterologous expression in *E. coli* and demonstration that the recombinant enzyme retained the substrate specificity of the native enzyme in both the forward and reverse reactions.

The availability of active rF3GalTase in a pure form allowed us to determine substrate usage and steady-state kinetic parameters not possible with a partially purified enzyme. The ability to unambiguously determine these characteristics helped to clarify the role of F3GalTase during pollen germination. We found that F3GalTase uses only UDP-galactose and flavonols to catalyze the formation of flavonol 3-O-galactosides in pollen. The only other flavonol galactosyltransferase described to date is active in sporophytic tissue; the highest abundance of the mRNA, as well as enzyme activity, was detected in the leaves of mung bean (28, 29). In addition, this enzyme is relatively nonspecific because it accepts four classes of flavonoids and two nucleotide sugars as substrates.

The total exclusion of other flavonoids as substrates, including the closely related anthocyanidins, makes F3GalTase unique among flavonoid glycosyltransferases, all of which show some activity toward more than one class of flavonoid (26, 28, 29, 39–41). The paradigm of a single activity with broad substrate usage has its historic roots in the UDP glucose:flavonoid glucosyltransferase activity encoded by the maize *Bronze-1* gene. Genetic and biochemical analysis of various maize tissues, including pollen, demonstrated that the UDP glucose:flavonoid glucosyltransferase encoded by the *Bz1* gene was able to conjugate both flavonol and anthocyanidin substrates (39–41). *Bz1* was the first flavonoid glucosyltransferase cloned and served as a hybridization probe to isolate virtually every flavonoid glycosyltransferase gene described to date. However, very few of these enzymes have been purified or tested for substrate usage so their identity rests solely on similarity with the *Bz1* gene product. The view that flavonoid glycosylation results from the ubiquitous expression of a single activity with broad substrate acceptance is most certainly simplistic. The characteristics of F3GalTase indicate that conjugate formation is a more complex process, with highly specific activities expressed in a tissue-specific and/or developmentally controlled manner.

Understanding the role of flavonol glycoside formation in pollen germination requires that we reconcile the finding that flavonols, which is the only form that induces pollen germination *in vitro*, are not detected in developing wild-type pollen which accumulates only the diglycoside. The inability of flavonol glycosides to be internalized precludes a direct test of their

germination-inducing properties *in vitro*. The finding that F3GalTase is relatively efficient in catalyzing both the forward and reverse transglycosylation reactions, added to our present understanding, allowed us to construct the following model of flavonol metabolism during pollen germination. As pollen develops within the anther, expression of F3GalTase located in the pollen wall facilitates the rapid uptake and conjugation of flavonols from the locular fluid. The inability to detect the galactoside in developing pollen requires that the F2"GT activity is highly efficient or a mechanism exists to channel the intermediate.

If the high levels of flavonol glucosylgalactosides that accumulate in wild-type pollen function as a latent pool of the active molecule, then a mechanism is required to re-generate the flavonol when the pollen is ready to germinate on the stigma. Although we have shown that F3GalTase can form flavonols from flavonol 3-O-galactosides, numerous attempts to detect a reverse F2"GT activity in pollen extracts have been unsuccessful. This, coupled with the fact that catabolism of metabolites rarely proceeds via a reversal of the biosynthetic pathway through which they formed, makes it unlikely that formation of the flavonol proceeds via a simple reversal of the pathway in Fig. 1.

The two most common mechanisms to remove a glycosyl moiety are hydrolysis by glycosidases or transglycosylation with UDP as the acceptor substrate. We propose that both activities are used in pollen and have preliminary evidence that intact pollen can catalyze the reverse F3GalTase reaction *in vitro*. When CMF pollen was incubated with kaempferol 3-O-galactoside or quercetin 3-O-galactoside and increasing concentrations of UDP, kaempferol or quercetin was detected in the surrounding germination medium in amounts that increased as UDP levels increased (18). Coincident with the increased production of kaempferol or quercetin in the pollen suspension, the percentage of germinated grains increased in a UDP concentration-dependent manner. When the same experiment was repeated with the corresponding flavonol 3-O-glucosides and UDP, flavonols were not detected and the pollen did not germinate (18).

Pollen is released from the anther in a metabolically quiescent state induced by the loss of water that occurs just before dehiscence. Upon contact with the stigma exudate, the rapid rehydration of the pollen is accompanied by a dramatic increase in metabolic activity (42). It has been previously demonstrated that numerous plant tissues, including pollen, contain glucosidases (43–48) which remove glucose molecules from various glucosides, including flavonoids. Glucosidase activity in hydrated pollen could catabolize the flavonol glucosylgalactosides to the respective flavonol galactosides. F3GalTase activity in the reverse direction could then generate flavonols from the flavonol 3-O-galactosides, thus producing the molecule required for pollen to germinate. In further support of this model, we have recently detected quercetin in mature V26 pollen hydrated in germination medium for 30 min. Alternatively, these data are not inconsistent with a model that involves the activation of a flavonol-mediated signaling pathway early in pollen development when flavonols accumulate in the anther. In this scenario, a second, non-flavonol signal would be stored until activated by hydration of the pollen on the stigma.

REFERENCES

1. Heller, W., and Forkmann, G. (1994) in *The Flavonoids: Advances in Research Since 1986* (Harborne, J. B., ed) pp. 499–535, Chapman and Hall Ltd., London
2. Li, J., Ou-Lee, T.-M., Raba, R., Amundson, R. G., and Last, R. L. (1993) *Plant*

3 K. D. Miller and L. P. Taylor, manuscript in preparation.
4 L. P. Taylor, unpublished observations.

3. Burn, J., Rossen, L., and Johnston, A. W. B. (1987) *Genes Dev.* **1**, 456–464
4. Harborne, J. B. (1980) in *Secondary Plant Products* (Bell, E. A., and Charlwood, B. V., eds) pp. 329–395, Springer-Verlag, Berlin
5. Mo, Y., Nagel, C., and Taylor, L. P. (1992) *Proc. Natl. Acad. Sci. U. S. A.* **89**, 7213–7217
6. Taylor, L. P., and Jorgensen, R. (1992) *J. Hered.* **83**, 11–17
7. Vogt, T., and Taylor, L. P. (1995) *Plant Physiol.* **108**, 903–911
8. Vogt, T., Wollenweber, E., and Taylor, L. P. (1995) *Phytochemistry* **38**, 589–592
9. Zerback, R., Bokel, M., Geiger, H., and Hess, D. (1989) *Phytochemistry* **28**, 897–899
10. Markham, K. R., and Campos, M. (1996) *Phytochemistry* **43**, 763–767
11. Wiering, H., and de Vlaming, P. (1984) in *Petunia* (Sink, K. C., ed) pp. 49–68, Springer-Verlag, Berlin
12. Xu, P., Vogt, T., and Taylor, L. P. (1997) *Planta* **202**, 257–265
13. Harborne, J. B., and Williams, C. A. (1988) in *The Flavonoids, Advances in Research Since 1980* (Harborne, J. B., ed) pp. 303–328, Chapman and Hall, London
14. Bandurski, R. S., Cohen, J. D., Slovin, J., and Reinecke, D. M. (1995) in *Plant Hormones: Physiology, Biochemistry, and Molecular Biology* (Davies, P. J., ed) pp. 39–65, Kluwer Academic Publishers Group, Dordrecht, Netherlands
15. Sembdner, G., Atzorn, R., and Schneider, G. (1994) *Plant Mol. Biol.* **26**, 1459–1481
16. Marrs, K. A., Alfenito, M. R., Lloyd, A. M., and Walbot, V. (1995) *Nature* **375**, 397–400
17. Pollak, P. E., Vogt, T., Mo, Y., and Taylor, L. P. (1993) *Plant Physiol.* **102**, 925–932
18. Taylor, L. P., Strenge, D., and Miller, K. D. (1998) in *Flavonoids in the Living System* (Manthey, J. A., and Buslig, B. S., eds) pp. 35–44, Plenum Press, New York
19. Napoli, C., Lemieux, C., and Jorgensen, R. (1990) *Plant Cell* **2**, 279–289
20. Shuttleworth, W. A., Hough, C. D., Bertrand, K. P., and Evans, J. N. S. (1992) *Protein Eng.* **5**, 461–466
21. Frohman, M. A., Dush, M. K., and Martin, G. R. (1988) *Proc. Natl. Acad. Sci. U. S. A.* **85**, 8998–9002
22. Hughes, J., and Hughes, M. A. (1994) *DNA Sequence* **5**, 41–49
23. Sparvoli, F., Martin, C., Scienza, A., Gavazzi, G., and Tonelli, C. (1994) *Plant Mol. Biol.* **24**, 743–755

24. Furtek, D., Schiefelbein, J. W., Johnston, F., and Nelson, O. E. (1988) *Plant Mol. Biol.* **11**, 473–481
25. Ralston, E. J., English, J. J., and Dooner, H. K. (1988) *Genetics* **119**, 185–197
26. Ford, C. M., Boss, P. K., and Høj, P. B. (1998) *J. Biol. Chem.* **273**, 9224–9233
27. Hedrick, J. L., and Smith, A. J. (1968) *Arch. Biochem. Biophys.* **126**, 155–164
28. Ishikura, N., and Mato, M. (1993) *Plant Cell Physiol.* **34**, 329–335
29. Mato, M., Ozeki, Y., Itoh, Y., Higeta, D., Yoshitama, K., Teramoto, S., Aida, R., Ishikura, N., and Shibata, M. (1998) *Plant Cell Physiol.* **39**, 1145–1155
30. Hundle, B. S., O'Brien, D. A., Alberti, M., Beyer, P., and Hearst, J. E. (1992) *Proc. Natl. Acad. Sci. U. S. A.* **89**, 9321–9325
31. Wise, R. P., Rohde, W., and Salamini, F. (1990) *Plant Mol. Biol.* **14**, 277–279
32. Schulte, S., and Stoffel, W. (1993) *Proc. Natl. Acad. Sci. U. S. A.* **90**, 10265–10269
33. Stahl, N., Jurevics, H., Morell, P., Suzuki, K., and Popko, B. (1994) *J. Neurosci. Res.* **38**, 234–242
34. Ishikura, N., Yang, Z., and Teramoto, S. (1993) *Z. Naturforsch. Sect. Biosci. C* **48**, 563–569
35. Ishikura, N., and Yang, Z.-Q. (1994) *Phytochemistry* **36**, 1139–1145
36. Cheng, G. W., Malencik, D. A., and Breen, P. J. (1994) *Phytochemistry* **35**, 1435–1439
37. Kernohan, J. C. (1964) *Biochim. Biophys. Acta* **81**, 346
38. Teipel, J. W., Hass, G. M., and Hill, R. L. (1968) *J. Biol. Chem.* **243**, 5684–5694
39. Dooner, H. K. (1979) *Phytochemistry* **18**, 749–751
40. Dooner, H. K., and Nelson, O. E. (1977) *Biochem. Genet.* **15**, 509–519
41. Larson, R. L., and Coe, E. H., Jr. (1977) *Biochem. Genet.* **15**, 153–156
42. Taylor, L. P., and Hepler, P. K. (1997) *Ann. Rev. Plant Physiol. Plant Mol. Biol.* **48**, 461–491
43. Hosel, W., (1981) in *The Biochemistry of Plants* (Conn, E. E., ed) pp. 725–753, Academic Press, New York
44. Hosel, W., and Conn, E. E. (1982) *Trends Biochem. Sci.* **7**, 219–221
45. Sharma, S., Singh, M. B., and Malik, C. P. (1981) *Plant Cell Physiol.* **22**, 927–931
46. Feldwisch, J., Vente, A., Zettl, R., Bako, L., Campos, N., and Palme, K. (1994) *Biochem. J.* **302**, 15–21
47. Kalinowska, M., and Wojciechowski, Z. A. (1978) *Phytochemistry* **17**, 1533–1537
48. Gussin, A. E. S., McCormack, J. H., Waung, L. Y.-L., and Gluckin, D. S. (1969) *Plant Physiol.* **44**, 1163–1168
