
Molecular Microbiology (1999) 34(2), 341–349

# Gene $vanXY_{C}$ encodes D,D -dipeptidase (VanX) and D,D-carboxypeptidase (VanY) activities in vancomycin-resistant *Enterococcus gallinarum* BM4174

**Peter E. Reynolds,**${}^{1\ast}$ **Cesar A. Arias**${}^{1}$ **and Patrice Courvalin**${}^{2}$

${}^{1}$Department of Biochemistry, University of Cambridge, Cambridge CB2 1QW, UK.

${}^{2}$Unité des Agents Antibactériens, Institut Pasteur, 28 rue du Dr Roux, 75724 Paris, Cedex 15, France.

## Summary

VanX and VanY have strict D,D-dipeptidase and D,D-carboxypeptidase activity, respectively, that eliminates production of peptidoglycan precursors ending in D-alanyl-D-alanine (D-Ala-D-Ala) in glycopeptide-resistant enterococci in which the C-terminal D-Ala residue has been replaced by D-lactate. *Enterococcus gallinarum* BM4174 synthesizes peptidoglycan precursors ending in D-Ala-D-serine (D-Ala-D-Ser) essential for VanC-type vancomycin resistance. Insertional inactivation of the *vanC-1* gene encoding the ligase that catalyses synthesis of D-Ala-D-Ser has a polar effect on both D,D-dipeptidase and D,D-carboxypeptidase activities. The open reading frame downstream from *vanC-1* encoded a soluble protein designated VanXY${}_{C}$ ($M_{r}$ 22 318), which had both of these activities. It had 39% identity and 74% similarity to VanY in an overlap of 158 amino acids, and contained consensus sequences for binding zinc, stabilizing the binding of substrate and catalysing hydrolysis that are present in both VanX- and VanY-type enzymes. It had very low dipeptidase activity against D-Ala-D-Ser, unlike VanX, and no activity against UDP-MurNAc-pentapeptide[D-Ser], unlike VanY. The introduction of plasmid pAT708(*vanC-1,XY${}_{C}$) or pAT717(*vanXY${}_{C}$) into vancomycin-susceptible *Enterococcus faecalis* JH2-2 conferred low-level vancomycin resistance only when D-Ser was present in the growth medium. The peptidoglycan precursor profiles of *E*. *faecalis* JH2-2 and JH2-2(pAT708) and JH2-2(pAT717) indicated that the function of VanXY${}_{C}$ was hydrolysis of D-Ala-D-Ala and removal of D-Ala from UDP-MurNAc-

Received 26 March, 1999; revised 24 July, 1999; accepted 29 July, 1999. *For correspondence. E-mail per@mole.bio.cam.ac.uk; Tel. (+44) 1223 333644; Fax (+44) 1223 333345.*

pentapeptide[D-Ala]. VanC-1 and VanXY${}_{C}$ were essential, but not sufficient, for vancomycin resistance.

## Introduction

Acquired resistance to glycopeptides in enterococci is achieved by (i) synthesis of abnormal peptidoglycan precursors terminating in acyl-D-alanyl-D-lactate (D-Ala-D-Lac) in strains with VanA, VanB and VanD phenotypes (Arthur et al., 1996a; Perichon et al., 1997), and (ii) elimination of the normal host D-Ala-terminating precursors as a result of the activity of a VanX D,D-dipeptidase (Reynolds et al., 1994a; Wu et al., 1995) and a VanY D,D-carboxypeptidase (Wright et al., 1992; Arthur et al., 1994). The level of resistance to vancomycin and teicoplanin is dependent on the ratio of UDP-MurNAc-L-Ala-D-Glu-L-Lys-D-Ala-D-Ala (pentapeptide[D-Ala]) to UDP-MurNAc-L-Ala-D-Glu-L-Lys-D-Ala-D-Lac (pentadepsipeptide) (Arthur et al., 1996b). In VanA-type strains in which the resistance genes are on multicopy plasmids, VanX has high activity and VanY is accessory to resistance (Arthur et al., 1994). In VanB-type strains, the activity of VanX${}_{B}$ is much lower, and VanY${}_{B}$ has an important role in metabolizing pentapeptide to UDP-MurNAc-L-Ala-D-Glu-L-Lys-D-Ala (tetrapeptide) (Reynolds, 1998). VanC-type *Enterococcus gallinarum*, *Enterococcus casseliflavus* and *Enterococcus flavescens* are intrinsically resistant to low levels of vancomycin but not to teicoplanin, and the resistance genes are chromosomal. The basis for this type of resistance is a change in the terminal residue of peptidoglycan precursors from D-Ala to D-serine (D-Ser) (Billot-Klein et al., 1994a; Reynolds et al., 1994b). Acyl-D-Ala-D-Ser has a lower affinity than acyl-D-Ala-D-Ala for vancomycin (Billot-Klein et al., 1994b). *E*. *gallinarum* BM4174 contains at least two dipeptide ligases, Ddl, which synthesizes D-Ala-D-Ala, and VanC-1 (Dutka-Malen et al., 1992), recently characterized as a D-Ala:D-Ser ligase in *E*. *casseliflavus* (Park et al., 1997). *E*. *gallinarum* BM4174 contains D,D-dipeptidase (hydrolysis of D-Ala-D-Ala) and D,D-carboxypeptidase (removal of D-Ala from compounds containing acyl-D-Ala-D-Ala) activities that limit production of the peptidoglycan precursor pentapeptide[D-Ala] in the vancomycin-susceptible pathway of peptidoglycan synthesis (Reynolds et al., 1994b). Insertional inactivation of the *vanC-1* gene results

in loss of vancomycin resistance (Dutka-Malen *et al.*, 1992) and has a polar effect on both D,D-dipeptidase and D,D-carboxypeptidase production (Reynolds *et al.*, 1994b), suggesting that the genes are clustered. In VanA- and VanB-type strains, the VanX (VanX<sub>B</sub>) D,D-dipeptidase is located in the cytoplasm, whereas VanY (VanY<sub>B</sub>) is encoded by another gene and is mainly membrane bound (Arthur *et al.*, 1992). In *E*. gallinarum BM4174, both activities are present predominantly in the cytoplasm (Reynolds *et al.*, 1994b). In this paper, we describe a cytoplasmic protein from strain BM4174, encoded by the *vanXY<sub>C</sub>* gene located immediately downstream from *vanC-1*, that has both D,D-dipeptidase and D,D-carboxypeptidase activities.

**Results**

*Identification, cloning and sequencing of the *vanXY<sub>C</sub>* gene encoding D,D-dipeptidase and D,D-carboxypeptidase activities*

Insertional inactivation of the *vanC-1* gene in *E*. gallinarum BM4174 had a polar effect on expression of D,D-dipeptidase and D,D-carboxypeptidase activities (Reynolds *et al.*, 1994b), suggesting that the gene(s) encoding these enzymes were downstream from *vanC-1*. Plasmids pAT704 (*ΔvanC-1, XY<sub>C</sub>, ΔT*), pAT705 (*vanC-1, XY<sub>C</sub>, ΔT*) and pAT706 (*ΔvanXY<sub>C</sub>, ΔT*) (Fig. 1), containing restriction fragments of BM4174 total DNA, which hybridized to a probe internal to *vanC-1*, ligated into vectors pUC1813 (for pAT704) or pAT113 (for pAT705 and 706), were tested for D,D-dipeptidase activity in *Escherichia coli* JM83 (Table 1). Expression was under the control of the *lac* promoter in all constructs. No activity was detected in the first 2 h after induction with 0.2, 0.5 or 1.0 mM isopropylthiogalactoside (IPTG), but expression of D,D-dipeptidase activity occurred after 2–3 h in *E. coli* harbouring pAT704 (*ΔvanC-1, XY<sub>C</sub>, ΔT*) and pAT705 (*vanC-1, XY<sub>C</sub>, ΔT*). The use of different constructs established which part of the DNA encoded D,D-dipeptidase activity but, as different vectors were used, the amounts of D,D-peptidase cannot be compared. Nearly all the activity was present in the cytoplasmic fraction after sonication and removal of cell membrane fragments. No D,D-dipeptidase activity was present in extracts of *E. coli* harbouring pAT706, which had a *PstI–EcoRI* fragment in which the *PstI* site was ≈ 100 bases downstream from the 3′ end of *vanC-1* (Fig. 1). Sequencing of the insert in pAT704 downstream from *vanC-1* revealed an open reading frame (ORF) in which the start codon overlapped with the last base of *vanC-1* and that contained a putative

![Diagram](attachment:diagram.png)

**Fig. 1.** Plasmids constructed. pAT705 (*vanC-1, XY<sub>C</sub>, ΔT*) in pAT113 (Trieu-Cuot *et al.*, 1991) and pAT704 (*ΔvanC-1, XY<sub>C</sub>, ΔT*) in pUC1813 contained fragments of *E*. gallinarum BM4174 chromosomal DNA obtained by partial digestion with *HindIII* and *KpnI*, or *EcoRI* respectively, which hybridized with a *vanC-1* probe. Plasmid pAT706 (*ΔvanXY<sub>C</sub>, ΔT*) was obtained by digestion of pAT704 DNA with *PstI* and *EcoRI* followed by cloning in pAT113. Plasmid pAT707 (*vanXY<sub>C</sub>* in pUC18) was obtained by cloning the PCR product containing the ORF encoding the putative D,D-dipeptidase/D,D-carboxypeptidase. Similarly, a PCR product containing the *vanC-1* and *vanXY<sub>C</sub>* genes was cloned in the shuttle vector pAT392 to produce plasmid pAT708 (*vanC-1, XY<sub>C</sub>*). Primers containing *SacI* and *XbaI* restriction sites were used for the production of plasmids pAT707 and pAT708. Restriction sites: E, *EcoRI*; H, *HindIII*; K, *KpnI*; P, *PstI*. Only the relevant sites are shown.

Table 1. D,D-dipeptidase activities in cytoplasmic extracts of *E. coli* containing various plasmids.

| Plasmid                          | Original vector | D,D-dipeptidase (X)<sup>a,b</sup> (nmol min<sup>-1</sup> mg<sup>-1</sup> protein) | D,D-carboxypeptidase(Y)<sup>a,c</sup> (nmol min<sup>-1</sup> mg<sup>-1</sup> protein) | Ratio X/Y |
|----------------------------------|-----------------|---------------------------------------------------------------|------------------------------------------------------------------------|-----------|
| pAT705 (*vanC-1, XY<sub>C</sub>, ΔT*) | pAT113         | 16                                                              | 6                                                                      | 2.7       |
| pAT704 (*ΔvanC-1, XY<sub>C</sub>, ΔT*) | pUC1813        | 135                                                             | 55                                                                     | 2.5       |
| pAT706 (*ΔvanXY<sub>C</sub>, ΔT*) | pAT113         | 0                                                               | 0                                                                      | –         |
| pAT707 (*vanXY<sub>C</sub>)*      | pUC18          | 152                                                             | 62                                                                     | 2.5       |
| pAT708 (*vanC-1, XY<sub>C</sub>)* | pAT392         | 28                                                              | 11                                                                     | 2.6       |

a. Activities were measured in lysed cells (membranes + cytoplasm).  
b. The substrate was D-Ala-D-Ala (3 mM).  
c. The substrate was pentapeptide[D-Ala] (3 mM).

ribosomal binding site (5′AGGAGGN<sub>7</sub>ATG) that displayed complementarity to the 3′ extremity of *E. faecalis* 16S ribosomal RNA (3′OH-UCCUCC). The ORF, designated *vanXY<sub>C</sub>*, encoded a putative protein of 190 amino acids. The beginning of a second ORF, immediately downstream from and overlapping *vanXY<sub>C</sub>*, was also revealed. This second gene encodes the membrane-bound serine racemase VanT (Arias et al., 1999). The *vanXY<sub>C</sub>* gene, including the ribosome binding site, was cloned into pUC18 under the control of the *lac* promoter to generate pAT707(*vanXY<sub>C</sub>*) (Fig. 1). Transformants expressed D,D-dipeptidase activity when induced with 0.2 mM IPTG. As with plasmids pAT704 and pAT705 containing this gene, no induction of enzymic activity occurred in the first 2 h.

Determination of D,D-carboxypeptidase (VanY-type) activity indicated that this enzyme was also present in the cytoplasm of *E. coli* harbouring pAT704, pAT705 and pAT707 that expressed D,D-dipeptidase activity (Table 1), but was absent from the cytoplasm of *E. coli* harbouring pUC18 that served as a control for pAT707. The ratio of D,D-peptidase to D,D-carboxypeptidase activities was constant in all the constructs containing a complete *vanXY<sub>C</sub>* gene (Table 1). We concluded that the polypeptide of 190 amino acids encoded by *vanXY<sub>C</sub>* catalysed both D,D-dipeptidase and D,D-carboxypeptidase activities.

The hydrophobicity profile of the protein (not shown) was consistent with its cytoplasmic location: no obvious regions of hydrophobicity were present that might indicate a transmembrane anchor as found in VanY and VanY<sub>B</sub>. Comparison with VanX, VanX<sub>B</sub>, VanY and VanY<sub>B</sub> indicated that VanXY<sub>C</sub> had a higher degree of identity and similarity to a VanY-type enzyme (39% identity and 74% similarity to VanY over a 158-amino-acid sequence) than to VanX-type enzymes. The consensus sequences present in a number of D,D-peptidases and D,D-carboxypeptidases (McCafferty et al., 1997; Lessard et al., 1998; Lessard and Walsh, 1999) were conserved in VanXY<sub>C</sub> (Fig. 2).

Effect of the presence of the *vanC-1* and *vanXY<sub>C</sub>* genes on expression of resistance and synthesis of peptidoglycan precursors in *E. faecalis* JH2-2

Fragments containing the *vanC-1* and *vanXY<sub>C</sub>* genes or *vanXY<sub>C</sub>* were amplified by polymerase chain reaction (PCR), cloned, under the control of the *P<sub>2</sub>* promoter in the shuttle vector pAT392, into *E. coli*, and the resulting plasmids pAT708 (*vanC-1, XY<sub>C</sub>*) and pAT717 (*vanXY<sub>C</sub>*) introduced into *E. faecalis* JH2-2 by electrotransformation. D,D-dipeptidase activity was detected in cytoplasmic extracts of both *E. coli* and *E. faecalis* transformants, whereas no activity was present in extracts of the same hosts containing pAT392. Introduction of pAT708 or pAT717 into *E. faecalis* raised the vancomycin minimum inhibitory concentration (MIC) from 2 to 8 μg ml<sup>-1</sup>, but only when D-Ser was included in the growth medium (Table 3); the addition of L-Ser did not alter the vancomycin MIC. Surprisingly, the vancomycin MIC of *E. faecalis* JH2-2 was raised twofold when D-Ser was present.

The nature of potential peptidoglycan synthetic activity in *E. faecalis* JH2-2, *E. faecalis* JH2-2/pAT708 (*vanC-1, XY<sub>C</sub>*) and JH2-2/pAT717 (*vanXY<sub>C</sub>*) was assessed by growing the bacteria in the presence or absence of L- and D-Ser, and inhibiting one of the membrane-bound steps of peptidoglycan synthesis with ramoplanin (Sommer and Reynolds, 1990) for a short time. Under these circumstances, soluble peptidoglycan precursors continue to be

Substrate specificity of VanXY<sub>C</sub>

The activity of VanXY<sub>C</sub> in the cytoplasmic fraction of *E. coli* JM83/pAT707 (*vanXY<sub>C</sub>*) was tested against a range

VanX and homologues

|  | 1 | 2 | 3 | 4 | 5 | 1 | 3 | 5 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| VanX<sub>B</sub> | DAKYAT........GYGLLLWDGYRPKSAVDCF L........SQSSHSRGSTIDLT L........MGSNFD FMD....GFQS------------YRF EWWHYKL |
| VanX<sub>A</sub> | DAKYAT........GYGLLLWDGYRPKRAVNCFM........SKSSH SRGSAIDLT L........MGSRFDFMD....GFEA------------YSLEWWHYVL |
| StoVanX | DAKYAT........GFGLLLWDGYRPQRAVDCF L........ARSGHSRGSTVDLT L........MGGDHDLMD....GFK A------------YEYEWWHYTL |
| EcoVanX | ELKYAC........GLQLVIYDAYRPQQAAQAMLW........VGS NHSRGT AIDLT L........MGAGFDEMH....GFVG------------ISSEWWHFEL |
| MtuVanX | DLRYAT........GQVLVFWDCYRPHDVQVRMF........YAHSHEA GRSVDVTF .......MGTD FDDFS....GLTV------------YSGEWWHFDG |
| SynVanX | GVKSLK........SWEILVFDAYRPIAVQQFMV........TPPPHSTGA ALDITL .......MGGEIDE LS....GFLR------------HPGEWWHFSQ |

VanXY<sub>C</sub>

|  | 2 | 3 | 2 | 4 | 5 | 5 | 3 | 5 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
|  | KDIRLV DGYRTEKEQ RRLW........PGCSEHQIGLAIDVGL..................GFILRYPEDKQEITGISYEPWHFRYVGL |
| 53 |  | 91 |  |  | 136 |  |  |  |

VanY and homologues

|  | 3 | 2 | 4 | 5 | 3 | 5 |
| --- | --- | --- | --- | --- | --- | --- |
| VanY<sub>B</sub> | G-VYPIVASGYRTTEKQQEIM........PGTSEHQLGLAVDIN..................GFIRRYPPDKTEITGVSN EPWHYRVGI |
| VanY<sub>A</sub> | GVS HFIINS GYRDFDEQSVLY........AGYSEHNSGLSLDVGS..................GFILRYPEDKT ELTGIQYEPWHIRYVGL |
| SmuVanY | D-SSEHLISGYRSVAYQEELY........PGSSEHQTGLAIDMST..................GFVLRFPEGKKDATGIDY EDWHYRVGV |
| BsTVanY1 | G-YELAAVSGYRSYDRQKVIF........PGESEHQTGLAMDISS..................GFIIRYPKNKEDITKYEYEP WHLRYVGK |
| TpaVanY | G-VALSVGSAYRSFAYQKKLF........EGTSQHQLGTVVDFGS..................GWLSLFPPGYEQVTGYV WEPWHFRYGV |
| BbuVanY | G-IEIKIKSAYRTQEQKFLF........PGHSQHHMGT AIDFIN..................GFSVSYPKGYETDTGYKAEPWHYL YIGP |
| SynVanY1 | G-VRLTIISGFRSIASQDALF........PGHSEHHTGYALDIGD..................GFELSFP PN--NSQGVSYEPWHWRYVAS |
| SynVanY2 | G-ISLVPISAFRTVTEQEQLF........PGYSEHHTGYAVDIGD..................SFELSFP PD--NPQGIAYEPWHWRYVG D |

Fig. 2. Sequence alignment of VanXY<sub>C</sub> with VanX and VanY homologues in the regions containing active site residues. The numbers under the VanXY<sub>C</sub> sequence refer to the positions of the amino acids. The amino acids depicted in bold have been identified in the active site of VanX (McCafferty et al., 1997; Lessard and Walsh, 1999) and are also present in VanY-type enzymes. The function of these residues (numbers above the sequences) is as follows: 1, Tyr and Asp bind to the α-NH<sub>3</sub><sup>+</sup> of D-Ala-D-Ala and are only present in VanX and homologues; 2, Asp in VanX-type proteins or Gln in VanY-type proteins orient the Arg transition state residue; 3, Glu is essential for catalysis, and Arg acts as a transition state residue; 4, Ser binds to the C-terminal carboxylate of the substrate; 5, two His residues and Asp act as Zn<sup>2+</sup> ligands. The sequences are of D,D-dipeptidases (VanX<sub>A</sub>, VanX<sub>B</sub>), D,D-carboxypeptidases (VanY<sub>A</sub>, VanY<sub>B</sub>), or VanX or VanY homologues from: Sto, Streptomyces toyocaensis (a glycopeptide producer); Eco, *E. coli*; Mtu, *Mycobacterium tuberculosis*; Syn, *Synechocystis* sp. PCC6803; Smu, *Streptococcus mutans*; Bst, *Bacillus subtilis*; Tpa, *Treponema pallidum*; Bbu, *Borrelia burgdorferi*. Two VanY homologues are present in *Synechocystis* sp. PCC6803. The figure is based on Lessard and Walsh (1999) in which the sequence alignment was obtained using CLUSTAL W (Higgins et al., 1996).

synthesized, but are not utilized and accumulate in the cytoplasm. *E. faecalis* JH2-2 synthesized pentapeptide[D-Ala] exclusively when no amino acid supplement was present in the growth medium, but additionally synthesized the pentapeptide[D-Ser] when D-Ser (50 mM) was present (Table 4). This effect was observed at concentrations of D-Ser greater than 10 mM. When *E. faecalis* JH2-2/

pAT708 (vanC-1, XY<sub>C</sub>) and JH2-2/pAT717(vanXY<sub>C</sub>) were incubated with ramoplanin, the peptidoglycan precursors synthesized were mainly tetrapeptide, thus confirming the D,D-carboxypeptidase activity of VanXY<sub>C</sub>, and a small amount of pentapeptide[D-Ala]. This distribution was unaffected by L-Ser, but growth in the presence of D-Ser resulted in the production of pentapeptide[D-Ser]

Table 2. Substrate specificity of VanXY<sub>C</sub>.

| Substrate<sup>a</sup> | Enzyme activity<sup>b</sup> (nmol min<sup>-1</sup> mg<sup>-1</sup> protein) | Relative activity (%) |
| --- | --- | --- |
| D-Ala-D-Ala | 512 | [100] |
| D-Ala-D-Ser | 15 | 3 |
| D-Ala-Gly | 10 | 2 |
| D-Ala-L-Ser | 3 | 1 |
| D-Ala-D-Ala-NH<sub>2</sub> | 0 | 0 |
| Ac-D-Ala-D-Ala | 34 | 6 |
| D-Ala-D-Ala-D-Ala | 33 | 6 |
| Ac<sub>2</sub>-L-Lys-D-Ala-D-Ala | 403 | 79 |
| UDP-MurNAc-L-Ala-γ-D-Glu-L-Lys-D-Ala-D-Ala | 197 | 38 |
| UDP-MurNAc-L-Ala-γ-D-Glu-L-Lys-D-Ala-D-Ser | 0 | 0 |
| UDP-MurNAc-L-Ala-γ-D-Glu-L-Lys-D-Ala-D-lactate* | 9 | 2 |

a. Substrates were present at 10 mM with the exception of * which was present at 2.1 mM.

b. The enzymic activities were not inhibited by penicillin G (250 μg ml<sup>-1</sup>).

Vancomycin resistance in enterococci 345

| Strain | Medium supplement (50mM) | MIC ($\mu$g ml$^{-1}$) |
|--------|--------------------------|-----------------------|
| E. faecalis JH2-2 | None | 2 |
|  | L-serine | 2 |
|  | D-serine | 4 |
| E. faecalis JH2-2/pAT708 (vanC-1, XY$_{c}$) | None | 2 |
|  | L-serine | 2 |
|  | D-serine | 8 |
| E. faecalis JH2-2/pAT709 (vanXY$_{c}$) | None | 2 |
|  | L-serine | 2 |
|  | D-serine | 8 |
| E. gallinarum BM4174 | None | 16 |
| E. gallinarum BM4175$^{a}$ | None | 2 |

a. The vanC-1 gene had been insertionally inactivated (Dutka-Malen *et al.*, 1992).

and tetrapeptide, together with a small amount of pentapeptide[D-Ala] (Table 4). The ratio of pentapeptide[D-Ser] to pentapeptide[D-Ala] was approximately one in E. faecalis JH2-2 and six in E. faecalis JH2-2/pAT708 (vanC-1, XY$_{c}$) when both strains were grown in the presence of 50 mM D-Ser. The amount of tetrapeptide present in cell extracts before the addition of ramoplanin was hardly detectable. This indicates that it can be used as a precursor of peptidoglycan, and that it is utilized as fast as it is produced when peptidoglycan synthesis is not inhibited.

**Discussion**

Two enzymes are required in VanA- and VanB-type enterococci to eliminate precursors in the vancomycin-susceptible pathway of peptidoglycan synthesis (Reynolds, 1998). VanX hydrolyses D-Ala-D-Ala, but has no activity against D-Ala-D-Lac (Reynolds *et al.*, 1994a). VanY hydrolyses D-Ala from peptidoglycan precursors ending in acyl-D-Ala-D-Ala, and consequently prevents molecules to which glycopeptides would bind from reaching the outer surface of the cytoplasmic membrane (Arthur *et al.*, 1998).

VanY is membrane bound and this location determines its effectiveness in the glycopeptide resistance mechanism (Arthur *et al.*, 1998). Its active site is in the cytoplasm, as indicated by the accumulation of tetrapeptide when peptidoglycan synthesis is inhibited by ramoplanin (Arthur *et al.*, 1998). It has been proposed that the primary role of VanY is the removal of D-Ala, not from pentapeptide[Ala], but from lipid-linked peptidoglycan precursors on the inside surface of the membrane, and that the membrane location of the enzyme is important (Arthur *et al.*, 1998). The combination of the removal of D-Ala-D-Ala, its replacement by D-Ala-D-Lac and prevention of precursors containing acyl-D-Ala-D-Ala from appearing on the outside surface of the membrane is required for glycopeptide resistance in VanA and VanB-type strains (Arthur *et al.*, 1996a).

In E. gallinarum BM4174, D,D-dipeptidase and D,D-carboxypeptidase activities are present on a single polypeptide that is shorter than VanX, VanX$_{B}$, VanY or VanY$_{B}$. Comparison of sequences indicates that VanXY$_{c}$ is more closely related to VanY than to VanX (Fig. 2). There is 39% identity of a 158-amino-acid sequence close to the C-terminus with part of VanY, but it lacks the membrane-spanning segment, present in both VanY

**Table 4. Analysis of peptidoglycan precursors in enterococci.**

| Strain | Medium supplement (50mM) | Peptidoglycan precursors (%) $^{a}$ |
|--------|--------------------------|-------------------------------------|
|  |  | UDP-MurNAc-pentapeptide[D-Ala] | UDP-MurNAc-pentapeptide[D-Ser] | UDP-MurNAc-tetrapeptide |
| E. faecalis JH2-2 | None (or L-Ser) | 100 | 0 | 0 |
|  | D-Ser | 54 | 46 | 0 |
| E. faecalis JH2-2/pAT708 | None | 14 | 0 | 86 |
| (vanC-1, XY$_{c}$) | L-Ser | 16 | 0 | 84 |
|  | D-Ser | 7 | 43 | 50 |
| E. faecalis JH2-2/pAT709 | None | 3 | 0 | 97 |
| (vanXY$_{c}$) | L-Ser | 3 | 0 | 97 |
|  | D-Ser | 4 | 56 | 40 |
| E. gallinarum BM4174 | None | 0 | 24 | 76 |
| E. gallinarum BM4175$^{b}$ | None | 100$^{c}$ | 0 | 0 |

a. The relative proportions of precursors were determined from the integrated peak areas after separation by HPLC, following a 15 min incubation of the bacteria with ramoplanin.
b. The vanC-1 gene had been insertionally inactivated (Dutka-Malen *et al.*, 1992).
c. Some UDP-MurNAc-hexapeptide with a D-Asp residue attached to the $\epsilon$-NH$_{2}$ group of lysine was also present.

© 1999 Blackwell Science Ltd, *Molecular Microbiology*, **34**, 341–349

and VanY<sub>B</sub>, that anchors them to the membrane. The greater similarity of VanXY<sub>C</sub> to VanY than to VanX is not surprising. VanX hydrolyses D,D-dipeptides unsubstituted at either the N- or C-terminus and hydrolyses D-Ala-D-Ser and D-Ala-Gly, though at a lower rate in comparison with D-Ala-D-Ala (Reynolds *et al.*, 1994a). The role of VanX (VanX<sub>B</sub>) is to eliminate D-Ala-D-Ala without hydrolysing the depsipeptide D-Ala-D-Lac, which is incorporated into peptidoglycan precursors instead of D-Ala-D-Ala and for which glycopeptides have a very low affinity (Bugg *et al.*, 1991). The strategy employed by enterococci of the VanC type to eliminate vancomycin-susceptible precursors is different. These organisms synthesize D-Ala-D-Ser to replace D-Ala-D-Ala, and a D,D-dipeptidase would have to hydrolyse D-Ala-D-Ala but leave D-Ala-D-Ser essentially intact. As VanX hydrolyses D-Ala-D-Ser at approximately 50% of the rate of D-Ala-D-Ala (Reynolds *et al.*, 1994a), it may have proved difficult for VanXY<sub>C</sub> to evolve from a VanX-type structure (Bussiere *et al.*, 1998) in order to achieve this altered specificity. However, both VanY- and VanX-type enzymes have consensus sequences involved in catalysis and in binding the substrate and Zn<sup>2+</sup> (McCafferty *et al.*, 1997; Lessard and Walsh, 1999), and these residues are present in VanXY<sub>C</sub>. It has recently been shown that an amino acid involved in stabilizing a conserved arginine, which interacts with a carbonyl group in both D-Ala-D-Ala and acyl-D-Ala-D-Ala, is either glutamine in a VanY-type D,D-carboxypeptidase or aspartate in a D,D-dipeptidase (Lessard and Walsh, 1999). Interestingly, VanXY<sub>C</sub> contains both of these amino acids. Furthermore, it lacks two important amino acids, tyrosine and aspartate, involved in the interaction of VanX with the free amino-terminus of D-Ala-D-Ala (Lessard and Walsh, 1999). Crystallographic studies indicate that VanX has restricted access for substrates to the active site: D,D-dipeptides can enter, but larger compounds containing acyl-D-Ala-D-Ala are excluded (Bussiere *et al.*, 1998). Presumably, the construction of an active site in an enzyme that would catalyse hydrolysis of D-Ala-D-Ala but not D-Ala-D-Ser, and removal of D-Ala but not of D-Ser from acyl-D-Ala-D-Ala or acyl-D-Ala-D-Ser respectively, was more easily achieved based on a VanY-type than on a VanX-type architecture.

The *vanXY<sub>C</sub>* gene was located immediately downstream from *vanC-1* (Fig. 1), in the position occupied by *vanX* (*vanX<sub>B</sub>*) relative to the *vanA* (*vanB*) ligase genes (Arthur *et al.*, 1996a). VanX has high activity in VanA-type strains because the genes are borne on multicopy plasmids and minute amounts of pentapeptide[D-Ala] are synthesized (Arthur *et al.*, 1996b). In VanB-type strains, the genes are part of a transposon in the chromosome, and VanX<sub>B</sub> activity is lower than that of VanX; consequently VanY has an important role in reducing the amount of pentapeptide (Arthur *et al.*, 1996b; Reynolds, 1998). In

E. gallinarum BM4174, the relative importance of the D,D-dipeptidase and D,D-carboxypeptidase activities of VanXY<sub>C</sub> is indicated by the amounts of peptidoglycan precursors (Table 4). The ratio of pentapeptide[D-Ala] to pentapeptide[D-Ser] is dependent on the availability of D-serine (Arias *et al.*, 1999) and the relative activities of D-Ala:D-Ala and D-Ala:D-Ser ligases. The large amount of tetrapeptide present in the cytoplasm of ramoplanin-inhibited cultures, ≈50% of the total peptidoglycan precursor synthetic capacity, suggests that the D,D-peptidase activity of VanXY<sub>C</sub> is insufficient to eliminate D-Ala-D-Ala, and therefore pentapeptide[D-Ala] continues to be synthesized. This is then degraded to tetrapeptide by the D,D-carboxypeptidase activity of VanXY<sub>C</sub>. Although tetrapeptide can function as a precursor, it cannot act as a donor but only as an acceptor in transpeptidation reactions (Ghuysen *et al.*, 1979). Consequently, the lack of activity of VanXY<sub>C</sub> against pentapeptide[D-Ser] is important in ensuring that a molecule with potential donor activity in cross-linking reactions is incorporated into polymerized peptidoglycan.

The ability of E. faecalis JH2-2 to synthesize pentapeptide[D-Ser] (Table 4) suggests that the D-Ala:D-Ala ligase has relatively low specificity with respect to the incorporation of a D-amino acid at the C-terminal position. Alternatively, the host bacterium may contain a second ligase, as is the case with E. coli (Zawadzke *et al.*, 1991), that is able to utilize D-Ser. Both the increase in vancomycin MIC (Table 3) and synthesis of the serine-containing precursor (Table 4) occurred at concentrations of D-Ser greater than 10 mM. VanC-1 and VanXY<sub>C</sub> are essential but not sufficient for resistance to vancomycin in E. gallinarum BM4174. Additionally, D-Ser is necessary for synthesis and incorporation of D-Ala-D-Ser into peptidoglycan precursors. Synthesis of D-Ser is accomplished by the serine racemase activity of VanT (Arias *et al.*, 1999). Co-synthesis of pentapeptide[D-Ala] and pentapeptide[D-Ser] in a 1:1 ratio (Table 4) confers a low degree of resistance to vancomycin (Table 3), but the presence of VanXY<sub>C</sub> converts most of the pentapeptide[D-Ala] to the tetrapeptide derivative, thus raising the ratio of the pentapeptides in favour of the one terminating in D-Ser (Table 4) and increasing the MIC of vancomycin (Table 3). The cytoplasmic location of VanXY<sub>C</sub> results in high activity against pentapeptide[D-Ala], and the specificity ensures that some D-Ala-D-Ala is destroyed and pentapeptide[D-Ala] is reduced to low levels, without any deleterious effect on peptidoglycan precursors containing D-Ser in the terminal position.

### Experimental procedures

#### Strains and growth conditions

E. faecalis JH2-2 (Jacob and Hobbs, 1974), E. gallinarum

BM4174 (constitutively resistant to vancomycin; Dutka-Malen *et al.*, 1990), *E. gallinarum* BM4175, in which the *vanC-1* gene had been insertionaly inactivated (Dutka-Malen *et al.*, 1992) and *E. coli* JM83 (Yanisch-Perron *et al.*, 1985) were grown in brain heart/0.5% yeast extract (BHY) broth or on BHY agar. *E. coli* XL-1 Blue was grown in Luria–Bertani (LB) broth or on LB agar.

*E. coli* strains harbouring various plasmids were selected as follows: ampicillin (100 µg ml⁻¹) was added for pUC derivatives, erythromycin (150 µg ml⁻¹) for plasmids based on pAT113 (Trieu-Cuot *et al.*, 1991) and gentamicin (8 µg ml⁻¹) for plasmids based on pAT392 (Arthur *et al.*, 1994). *E. faecalis* JH2-2 containing derivatives of pAT392 were selected on gentamicin (150 µg ml⁻¹). The recombinant plasmids used are depicted in Fig. 1. Overnight cultures were diluted to A₆₀₀ = 0.1 and incubated until they were in the exponential phase of growth. The MICs of vancomycin for the *E. faecalis* strains were determined using twofold dilutions of the antibiotic in BHY broth; the inoculum was 10⁶ cells ml⁻¹ and tubes were incubated for 24 h at 37°C.

### DNA manipulation and sequencing

The isolation and purification of plasmid DNA, digestion with restriction endonucleases, amplification employing the polymerase chain reaction (PCR) using *Pfu* DNA polymerase (Stratagene), ligation with T4 DNA ligase (Pharmacia) and transformation of *E. coli* strains with recombinant plasmid DNA were carried out using standard methods (Sambrook *et al.*, 1989). Electroporation of pAT708 and pAT717 into *E. faecalis* JH2-2 was carried out as has been described (Cruz-Rodz and Gilmore, 1990). DNA sequencing of pAT704 downstream from *vanC-1* and pAT707 on both strands was carried out with dye-labelled terminators using a 373-A automated DNA sequencer (Perkin-Elmer). The sequence of *vanXY*ₐ has been deposited in GenBank (accession number AF 132548).

The *vanXY*ₐ gene, including the ribosome binding site, was amplified by PCR using DNA of plasmid pAT704 as a template, *Pfu* polymerase and deoxyribonucleotides PR1 (5′-TTCAGAGCTTTGATTGCACCTGGCAGAG) and PR2 (5′-TTGCTCTAGATCATGCGAACTGCCCTCA) containing, respectively, *SacI* and *XbaI* restriction sites (underlined) as primers. The amplification product was digested with *SacI* and *XbaI*, purified by gel electrophoresis, and cloned in pUC18 (Vieira and Messing, 1982) digested with the same enzymes to obtain plasmid pAT707.

A fragment containing *vanC-1* and *vanXY*ₐ was amplified by PCR using chromosomal DNA of *E. gallinarum* BM4174 as a template, *Pfu* polymerase and deoxyribonucleotides PR3 (5′-CTCGGAGCTCAAAGAAAAAGGAAAAGGAAG) containing a *SacI* site (underlined) and the RBS of *vanC-1* (italicized) and PR2 as primers. The product was purified, digested with *SacI* and *XbaI*, and ligated with pAT392 cut with the same enzymes to obtain plasmid pAT708. The PCR product obtained with primers PR1 and PR2 was digested with *SacI* and *XbaI* and cloned into pAT392 digested with the same enzymes to obtain plasmid pAT717. DNA from plasmids pAT708 and pAT717 was extracted, purified and introduced into *E. faecalis* JH2-2 by electroporation.

---

Vancomycin resistance in enterococci 347

#### Growth of E. coli strains, induction of D,D-peptidase activity and cell breakage

Overnight cultures (50 ml) in BHY supplemented with the appropriate antibiotic were diluted with fresh medium and incubated with shaking at 37°C until A₆₀₀ had reached 0.6. IPTG (0.5 mM) was added, incubation continued and samples (20 ml) were taken at 1 h intervals. Bacteria were harvested, washed, resuspended in 1 ml of 100 mM PO₄ buffer (pH 7.2), broken by sonication, and the membrane and cytoplasmic fractions separated by centrifugation at 100 000 g for 45 min. The membrane fraction was resuspended in 200 µl of 100 mM PO₄ buffer (pH 7.2). In preliminary experiments to determine which strains expressed D,D-peptidase activity, the total cell lysate was used.

Membrane and cytoplasmic preparations of *E. gallinarum* BM4174 and *E. faecalis* JH2-2 derivatives were prepared by osmotic lysis resulting from digestion of the peptidoglycan with lysozyme and M1 muramidase (Arias *et al.*, 1999). The protein content was assayed by the method of Bradford (1976) using bovine serum albumin as a standard.

#### Assay of D,D-dipeptidase and D,D-carboxypeptidase activities

In a final volume of 30 µl, the assay mixture contained, 100 mM PO₄ buffer (pH 7.2) (*E. coli* preparations) or 100 mM Tris-HCl (pH 7.2) (enterococcal preparations), 10 mM D-Ala-D-N (where N is another amino acid, VanX activity) or substrates ending in acyl-D-Ala-D-N (VanY activity), and 10 µl of a suitable dilution of the cytoplasmic or membrane fraction so that less than 10% of the substrate was converted to product during the incubation (30 min at 37°C for VanX activity; 40 min at 37°C for VanY activity). The concentration of D-Ala or D-Ser released was measured using the D-amino acid oxidase assay with appropriate standards (Messer and Reynolds, 1992). A more sensitive assay, using 2,2′-azinodi(3-ethylbenzthiazoline sulphonate) instead of o-dianisidine as the chromogenic reagent, was adopted when the activity was low (Granier *et al.*, 1994).

The activity of VanXYₐ against the nucleotide-linked pentapeptides was also assayed using high-performance liquid chromatography (HPLC). After incubation to release D-Ala, D-Ser or D-Lac, the volume of the incubation mixture was increased to 50 µl with water, and proteins were precipitated by the addition of 0.25 mg of sulphosalicylic acid. Denatured proteins were removed by centrifugation at 18 000 g for 10 min, and the supernatant diluted to give a nucleotide concentration of 100 µM. The tetrapeptide and residual substrate were separated by HPLC as described by Arthur *et al.* (1998). The elution times were: tetrapeptide, 15 min; pentapeptide[D-Ser], 17 min; pentapeptide[D-Ala], 24 min; and pentadepsipeptide[D-lac], 38 min.

#### Extraction and separation of peptidoglycan precursors

Soluble peptidoglycan precursors were extracted and assayed as has been described (Arthur *et al.*, 1996a). In brief, enterococci growing exponentially in BHY containing various supplements were incubated with ramoplanin for

© 1999 Blackwell Science Ltd, *Molecular Microbiology*, **34**, 341–349

15 min to accumulate wall precursors. The bacteria were extracted with trichloroacetic acid, the extract neutralized with NaHCO₃, desalted on G10 Sephadex and the precursors separated by HPLC.

### Acknowledgements

Part of this work was carried out during the tenure of a Senior CIBA Fellowship to P.E.R. held at the Institut Pasteur. C.A.A. was funded by COLCIENCIAS (Instituto para el Desarrollo de la Ciencia y Tecnología, ‘Francisco José de Caldas’) and the Overseas Research Scheme Award of the Universities of the United Kingdom. We thank S. Dutka-Malen for construction of plasmids pAT704, pAT705 and pAT706, M. Arthur for helpful discussions and C. Hill and J. Lester, Cambridge Centre for Molecular Recognition, for synthesis of oligonucleotides and automated DNA sequencing respectively.

### References

Arias, C.A., Martin-Martinez, M., Blundell, T.L., Arthur, M., Courvalin, P., and Reynolds, P.E. (1999) Characterisation and modelling of VanT, a novel, membrane-bound, serine racemase from vancomycin-resistant *Enterococcus gallinarum* BM4174. *Mol Microbiol* **31**: 1653–1664.

Arthur, M., Molinas, C., and Courvalin, P. (1992) Sequence of the *vanY* gene required for production of a vancomycin-inducible D,D-carboxypeptidase in *Enterococcus faecium* BM4147. *Gene* **120**: 111–114.

Arthur, M., Depardieu, F., Snaith, H.A., Reynolds, P.E., and Courvalin, P. (1994) Contribution of VanY D,D-carboxypeptidase to glycopeptide resistance in *Enterococcus faecalis* by hydrolysis of peptidoglycan precursors. *Antimicrob Agents Chemother* **38**: 1899–1903.

Arthur, M., Reynolds, P.E., and Courvalin, P. (1996a) Glycopeptide resistance in enterococci. *Trends Microbiol* **4**: 401–407.

Arthur, M., Depardieu, F., Reynolds, P.E., and Courvalin, P. (1996b) Quantitative analysis of the metabolism of soluble peptidoglycan precursors of glycopeptide-resistant enterococci. *Mol Microbiol* **21**: 33–44.

Arthur, M., Depardieu, F., Cabanié, L., Reynolds, P., and Courvalin, P. (1998) Requirement of the VanY and VanX D,D-peptidases for glycopeptide resistance in enterococci. *Mol Microbiol* **31**: 819–830.

Billot-Klein, D., Gutmann, L., Sable, S., Guinet, E., and van Heijenoort, J. (1994a) Modification of peptidoglycan precursors is a common feature of the low-level vancomycin-resistant VANB-type *Enterococcus* D366 and of the naturally glycopeptide-resistant species *Lactobacillus casei*, *Pediococcus pentosaceus*, *Leuconostoc mesenteroides* and *Enterococcus gallinarum*. *J Bacteriol* **176**: 2398–2405.

Billot-Klein, D., Blanot, D., Gutmann, L., and van Heijenoort, J. (1994b) Association constants for the binding of vancomycin and teicoplanin to N-acetyl-D-alanyl-D-alanine and N-acetyl-D-alanyl-D-serine. *Biochem J* **304**: 1021–1022.

Bradford, M.M. (1976) A rapid and sensitive method for rapid quantification of microgram quantities of protein utilising a principle of protein-dye binding. *Anal Biochem* **72**: 248–254.

Bugg, T.D.H., Wright, G.D., Dutka-Malen, S., Arthur, M., Courvalin, P., and Walsh, C.T. (1991) Molecular basis for vancomycin resistance in *Enterococcus faecium* BM4147: biosynthesis of a depsipeptide peptidoglycan precursor by vancomycin resistance proteins VanH and VanA. *Biochemistry* **30**: 10408–10415.

Bussiere, D.E., Pratt, S.D., Katz, L., Severin, J.M., Holzman, T., and Park, C. (1998) The structure of VanX reveals a novel amino-dipeptidase involved in mediating transposon-based vancomycin resistance. *Mol Cell* **2**: 75–84.

Cruz-Rodz, A.L., and Gilmore, M.S. (1990) High efficiency introduction of plasmid DNA into glycine-treated *Enterococcus faecalis* by electroporation. *Mol Gen Genet* **224**: 152–154.

Dutka-Malen, S., Leclercq, R., Coutant, V., Duval, J., and Courvalin, P. (1990) Phenotypic and genotypic heterogeneity of glycopeptide resistance determinants in gram-positive bacteria. *Antimicrob Agents Chemother* **34**: 1875–1879.

Dutka-Malen, S., Molinas, C., Arthur, M., and Courvalin, P. (1992) Sequence of the *vanC* gene of *Enterococcus gallinarum* BM4174 encoding a D-alanine:D-alanine ligase-related protein necessary for vancomycin resistance. *Gene* **112**: 53–58.

Ghuysen, J.-M., Frère, J.-M., Leyh-Bouille, M., Coyette, J., Dusart, J., and Nguyen-Distèche, M. (1979) Use of model enzymes in the determination of the mode of action of penicillin and Δ3-cephalosporins. *Ann Rev Biochem* **48**: 73–101.

Granier, B., Jamin, M., Adam, M., Galleni, M., Lakaye, B., Zorzi, W., *et al.* (1994) Serine-type D-Ala-D-Ala peptidases and penicillin-binding proteins. *Methods Enzymol* **244**: 249–266.

Higgins, D.G., Thompson, J.D., and Gibson, T.J. (1996) Using CLUSTAL for multiple sequence alignments. *Methods Enzymol* **266**: 383–402.

Jacob, A.E., and Hobbs, S.J. (1974) Conjugal transfer of plasmid-borne multiple antibiotic resistance in *Streptococcus faecalis* var. zymogenes. *J Bacteriol* **117**: 360–372.

Lessard, I.A.D., and Walsh, C.T. (1999) Mutational analysis of active site residues of the enterococcal D-Ala-D-Ala dipeptidase VanX and comparison with *Escherichia coli* D-Ala:D-Ala ligase and identification of potential active site residues for the D-Ala-D-Ala carboxypeptidase VanY. *Chem Biol* **6**: 177–187.

Lessard, I.A.D., Pratt, S.D., McCafferty, D.G., Bussiere, D.E., Hutchins, C., Wanner, B.L., *et al.* (1998) Homologs of the vancomycin resistance D-Ala-D-Ala dipeptidase VanX in *Streptomyces toyocaensis*, *Escherichia coli* and *Synechocystis*: attributes of catalytic efficiency, stereoselectivity and regulation with implications for function. *Chem Biol* **5**: 489–504.

McCafferty, D.G., Lessard, I.A., and Walsh, C.T. (1997) Mutational analysis of potential zinc-binding residues in the active site of the enterococcal D-Ala-D-Ala dipeptidase VanX. *Biochemistry* **36**: 10498–10505.

Messer, J., and Reynolds, P.E. (1992) Modified peptidoglycan precursors produced by glycopeptide-resistant enterococci. *FEMS Microbiol Lett* **94**: 195–200.

Park, I.S., Chung-Hung, L., and Walsh, C.T. (1997) Bacterial resistance to vancomycin: overproduction, purification and

characterisation of VanC-2 from Enterococcus casseliflavus as a D-Ala:D-Ser ligase. Proc Natl Acad Sci USA 94: 10040–10044.

Perichon, B., Reynolds, P., and Courvalin, P. (1997) VanD-type glycopeptide-resistant Enterococcus faecium BM4339. Antimicrob Agents Chemother 41: 2016–2018.

Reynolds, P.E. (1998) Control of peptidoglycan synthesis in vancomycin-resistant enterococci: D,D-peptidases and D,D-carboxypeptidases. Cell Mol Life Sci 54: 325–331.

Reynolds, P.E., Depardieu, F., Dutka-Malen, S., Arthur, M., and Courvalin, P. (1994a) Glycopeptide resistance mediated by enterococcal transposon Tn1546 requires production of VanX for hydrolysis of D-alanyl-D-alanine. Mol Microbiol 13: 1065–1070.

Reynolds, P.E., Snaith, H.A., Maguire, A.J., Dutka-Malen, S., and Courvalin, P. (1994b) Analysis of peptidoglycan precursors in vancomycin-resistant Enterococcus gallinarum BM4174. Biochem J 301: 5–8.

Sambrook, J., Fritsch, E.F., and Maniatis, T. (1989) Molecular Cloning: A Laboratory Manual, 2nd edn. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.

Sommer, E.A., and Reynolds, P.E. (1990) Inhibition of peptidoglycan biosynthesis by ramoplanin. Antimicrob Agents Chemother 34: 413–419.

Trieu-Cuot, P., Carlier, C., Poyart-Salmeron, C., and Courvalin, P. (1991) An integrative vector exploiting the

transposition properties of Tn1545 for insertional mutagenesis and cloning of genes from Gram-positive bacteria. Gene 106: 21–27.

Vieira, J., and Messing, J. (1982) The pUC plasmids and M13mp7-derived system for insertion mutagenesis and sequencing with synthetic universal primers. Gene 19: 259–268.

Wright, G.D., Molinas, C., Arthur, M., Courvalin, P., and Walsh, C.T. (1992) Characterisation of VanY, a D,D-carboxypeptidase from vancomycin-resistant Enterococcus faecium BM4147. Antimicrob Agents Chemother 36: 1514–1518.

Wu, Z., Wright, G.D., and Walsh, C.T. (1995) Overexpression, purification and characterisation of VanX, a D,D-dipeptidase which is essential for vancomycin resistance in Enterococcus faecium BM4147. Biochemistry 34: 2455–2463.

Yanisch-Perron, C., Vieira, J., and Messing, J. (1985) Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. Gene 33: 103–119.

Zawadzke, L.E., Bugg, T.D.H., and Walsh, C.T. (1991) Existence of two D-alanine:D-alanine ligases in Escherichia coli: cloning and sequencing of the ddlA gene and purification and characterisation of the DdlA and DdlB enzymes. Biochemistry 30: 1673–1682.
