
JOURNAL OF BACTERIOLOGY, Dec. 1999, p. 7154–7160
0021-9193/99/$04.00+0
Copyright © 1999, American Society for Microbiology. All Rights Reserved.

# Bacillus subtilis *yckG* and *yckF* Encode Two Key Enzymes of the Ribulose Monophosphate Pathway Used by Methylotrophs, and *yckH* Is Required for Their Expression

**HISASHI YASUEDA**, **YOSHIO KAWAHARA**, AND **SHIN-ICHI SUGIMOTO**

*Fermentation and Biotechnology Laboratories, Ajinomoto Co., Inc., Kawasaki-ku, Kawasaki-shi, 210-0801, Japan*

Received 14 June 1999/Accepted 14 September 1999

The ribulose monophosphate (RuMP) pathway is one of the metabolic pathways for the synthesis of compounds containing carbon-carbon bonds from one-carbon units and is found in many methane- and methanol-utilizing bacteria, which are known as methylotrophs. The characteristic enzymes of this pathway are 3-hexulose-6-phosphate synthase (HPS) and 6-phospho-3-hexuloisomerase (PHI), neither of which was thought to exist outside methylotrophs. However, the presumed *yckG* gene product (YckG) of *Bacillus subtilis* shows a primary structure similar to that of methylotroph HPS (F. Kunst et al., Nature 390:249–256, 1997). We have also investigated the sequence similarity between the *yckF* gene product (YckF) and methylotroph PHI (Y. Sakai, R. Mitsui, Y. Katayama, H. Yanase, and N. Kato, FEMS Microbiol. Lett. 176:125–130, 1999) and found that the *yckG* and *yckF* genes of *B.* subtilis express enzymatic activities of HPS and PHI, respectively. Both of these activities were concomitantly induced in *B.* subtilis by formaldehyde, with induction showing dependence on the *yckH* gene, but were not induced by methanol, formate, or methylamine. Disruption of either gene caused moderate sensitivity to formaldehyde, suggesting that these enzymes may act as a detoxification system for formaldehyde in *B.* subtilis. In conclusion, we found an active *yckG* (for HPS)-*yckF* (for PHI) gene structure (now named *hxlA-hxlB*) in a nonmethylotroph, *B.* subtilis, which inherently preserves the RuMP pathway.

---

The ribulose monophosphate (RuMP) pathway is involved in formaldehyde fixation in many methylotrophs (11, 15, 17, 22). Typically, the RuMP pathway has two characteristic enzymes, 3-hexulose-6-phosphate synthase (HPS) and 6-phospho-3-hexuloisomerase (PHI), and it also shares some enzymes with the pentose phosphate pathway and the glycolytic or Entner-Doudoroff pathway. PHI catalyzes isomerization between fructose 6-phosphate and hexulose 6-phosphate, which is synthesized from ribulose 5-phosphate (Ru5P) and formaldehyde by HPS. Although HPS and PHI have been detected in many methylotrophs (3, 12, 18–20, 23, 32, 34), the genes encoding these enzymes remain largely unknown. The *hps* gene, encoding HPS, was first cloned from an obligate methylotroph, *Methylomonas aminofaciens* 77a (44), and cloning of the gene (*rmpB*) encoding PHI from *M.* *aminofaciens* 77a (35) and a facultative methylotroph, *Mycobacterium gastri* MB19 (28), was recently reported. Neither enzyme was thought to exist outside methylotrophs, but sequence analysis of the *Bacillus subtilis* genome (24, 41) has suggested that *yckG* may be a homolog of the gene encoding HPS (44) in *M.* *aminofaciens* 77a. The *yckG* gene is located between *yckF* and *yckH*, at about 32° on the *B.* subtilis genome map (24) (Fig. 1a). Here, we present data showing that a nonmethylotroph, *B.* subtilis, contains genes (*yckG* and *yckF*) encoding two key enzymes of the RuMP pathway (HPS and PHI) and that the activities of both enzymes can be induced by formaldehyde with dependence on the *yckH* gene. These findings indicate the presence of characteristic enzymes from the RuMP pathway in a nonmethylotroph. The significance of detection of these enzymes in *B.*

---

**MATERIALS AND METHODS**

Bacterial strains, oligonucleotide primers, and plasmids. The wild-type strain *B.* subtilis 168 (1) was purchased from the American Type Culture Collection (Rockville, Md.), and *Escherichia coli* JM109 (45) was obtained from Takara Shuzo Co. (Kyoto, Japan). *B.* subtilis YD101 (*yckG*-deficient mutant) and YD102 (*yckF*-deficient mutant) were constructed, as described below, by inserting the spectinomycin resistance (*spc*) DNA fragment from pDG1726 (13) (*Bacillus* Genetic Stock Center, Columbus, Ohio) into the individual genes of the chromosome of strain 168. Strains YD111 (*yckH::spc*) and YD112 (*yckH::spc*) were also constructed from strain 168 in the same way (Fig. 1a). Strain YD121 (*yckH yckH⁺*) contained both the intact *yckH* gene and the defective one (Fig. 1b).

The potential open reading frame (ORF) of the *yckG* or *yckF* gene was cloned by PCR amplification (29) using the oligonucleotide primer pair BsYck-G1 (5′-GAGTATCGATAAAATGGAAATTACAGCTTGCATTAGACCTCGT-3′; the translation start codon is underlined, and the restriction enzyme site [ClaI] for ligation to the vector is italicized) plus BsYck-G6 (5′-AATTGTGGATCCCC ATTGAGAATTTCCGCTACGTATTCAGTGCG-3′; the restriction enzyme site [BamHI] for ligation to the vector is italicized) or BsYck-F1 (5′-AAGCATCG ATAAAATGAAAAACGACTGAATACGTAGCGGAA-3′) plus BsYck-F2 (5′- ATCTTGGATCCGGTTGTGTGATGTITATTCAAGGTTTGGCG-3′). The region containing the *yckG* and *yckF* genes was amplified by PCR using the primer pair BsYck-G1 plus BsYck-F2. PCR was performed with Takara LA Taq DNA polymerase (Takara Shuzo) for 28 cycles with purified *B.* subtilis genomic DNA. The PCR products were purified with a QIAquick gel extraction kit (Qiagen). The fragments thus obtained were ligated behind the *E.* coli *trp* promoter to mediate overexpression in *E.* coli (46). Each amplified DNA fragment was digested with restriction enzymes ClaI and BamHI, and the fragments were introduced into the vector to construct the expression vectors pT-Bsb-yckG6, pT-Bsb-yckF1, and pT-Bsb-yckGF1 for the overexpression of *yckG*, *yckF*, and *yckGF*, respectively. The fragment containing *yckH* was amplified by PCR using primers BsYck-H1 (5′-CAATGTTAACGTCAGGCTTTTGCTGGATC ACTTCTGGCA-3′; the restriction enzyme site for cloning of the amplified DNA fragment is italicized) and BsYck-H2 (5′-GTGTACCGAATTCGTTTTTGTGGCAT CCGTTAAAGGGTA-3′). After digestion with *HpaI* and *EcoRI*, the *yckH* fragment was cloned into pACYC184 (Nippon Gene, Tokyo, Japan) treated with *Bst1107I* and *EcoRI* to construct pAY-Bsb-yckH.

For the purpose of gene disruption, the *spc* fragment was introduced into the cloned *yckG* and *yckF* genes to construct pT-Bsb-yckG-Sp5 and pT-Bsb-yckF Sp6, respectively. The *spc* gene was also cloned into the *PvuII* site in the *yckH* gene to construct two plasmids, pAY-Bsb-yckH-SpA and pAY-Bsb-yckH-SpB, in

* Corresponding author. Mailing address: Fermentation and Biotechnology Laboratories, Ajinomoto Co., Inc., 1-1, Suzuki-cho, Kawasaki-ku, Kawasaki-shi, 210-0801, Japan. Phone: 81-44-245-5972. Fax: 81-44-222-0129. E-mail: blr_yasueda@te2.ajinomoto.co.jp or Hisashi Yasueda @Ajinomoto.com..

VOL. 181, 1999 ONE-CARBON COMPOUND METABOLISM IN B. SUBTILIS 7155

![Diagram](#)

of 50 mM potassium phosphate (pH 7.6), 5 mM MgCl₂, 5 mM ribose-5-phosphate (Ri5P) (Nacalai Tesque, Inc., Kyoto, Japan), 2.5 mM NADP (Sigma, St. Louis, Mo.), 10 U of phosphoribose isomerase (PRI) (Sigma), 10 U of glucose-6-phosphate dehydrogenase (Boehringer Mannheim), 10 U of phosphoglucose isomerase (PGI) (Boehringer Mannheim), 50 U of recombinant PHI purified from *E. coli* overproducing *M. aminofaciens* 77a PHI (35), and the test sample of cell extract. The mixture was preincubated for 3 min at 37°C to achieve equilibrium of Ru5P and Ri5P (catalyzed by PRI), and then the reaction was started by adding formaldehyde at a final concentration of 5 mM. In the PHI assay, recombinant *M. aminofaciens* 77a HPS (35) was used instead of recombinant PHI, and both were eliminated from the reaction mixture when the serial reaction of HPS and PHI enzymes was assayed. Activity was assessed by monitoring the change in optical density (OD) at 340 nm with NADP as a cofactor, and correction was made for formaldehyde-independent activity. The protein concentration was determined by the method of Bradford (8) with a Bio-Rad protein assay kit (Bio-Rad Laboratories, Richmond, Calif.) and bovine serum albumin as the standard.

Assay of other enzymes. NAD-dependent methanol dehydrogenase (MDH) and glutathione-dependent formaldehyde dehydrogenase (GS-FDH) were assayed by using the *B. subtilis* cell extract treated with formaldehyde. MDH activity was assayed as described by Arfman et al. (2); GS-FDH activity was assayed as described by Attwood (6).

Effect of *yckG* and *yckF* mutations on cell growth in the presence of formaldehyde or NaCl. *B. subtilis* 168 (wild type) and two isogenic mutant strains (YD101 and YD102) were cultured at 30°C with shaking in LB medium containing spectinomycin when necessary. Overnight cultures were inoculated into warmed fresh LB medium, or into medium containing formaldehyde or NaCl, and cell growth and lysis profiles were automatically recorded at an OD of 660 nm with a Bio-photorecorder (Advantec, Tokyo, Japan).

¹³C NMR analysis of the products obtained with [¹³C]formaldehyde as the substrate. A reaction mixture containing 50 mM potassium phosphate (pH 7.6), 5 mM MgCl₂, 5 mM Ri5P, 10 U of PRI, and 10 U of PGI was prepared. After preincubation at 37°C for 10 min to produce Ru5P from Ri5P, cell extract containing the *yckG* gene product (YckG) and *yckF* gene product (YckF) was added to the mixture. The serial YckG and YckF reaction was started by addition of [¹³C]formaldehyde as the substrate. After incubation at 37°C for 20 min, EDTA was added to the mixture to chelate Mg²⁺, which is required for HPS activity. The total activity in the mixture was assayed with ¹³C nuclear magnetic resonance (NMR) spectroscopy by monitoring the decrease of the resonance signal for the ¹³C atom of formaldehyde and the appearance of signals for the C atoms of fructose 6-phosphate and glucose 6-phosphate, which was yielded by addition of PGI. In the negative control sample, EDTA was added to the mixture immediately after addition of the cell extract.

NMR spectra were obtained at 100 MHz with a Fourier transform (FT) NMR spectrometer (JNM-A400; JEOL, Tokyo, Japan), using 0.6-ml samples containing 15% D₂O in 5-mm-diameter tubes to provide a lock signal. A total of 256 transients were acquired by using 32k data points. Gated proton decoupling was used, and 45° pulses were applied, with a 2.5-s relaxation delay between pulses to give a repetition time of 3.9 s. During acquisition, the samples were kept at 25°C. [¹³C]formaldehyde (99% ¹³C atom) was purchased from Cambridge Isotope Laboratories (Andover, Mass.).

RNA analysis. *B. subtilis* cells were grown to mid-log phase in LB medium at 37°C. After the addition of formaldehyde at final concentration of 0.5 mM, culture was continued for 45 min. Then RNA was extracted by using an ISOGEN kit (Nippongene, Toyama, Japan) and treated with RNase-free DNase (Takara Shuzo). About 1 μg of RNA was mixed with the YGF-R2 oligonucleotide DNA (5′-TTCAGGCCGGTTGTGTGATGC-3′), and cDNA was synthesized by using a reverse transcription (RT) kit (RNA LA PCR kit; Takara Shuzo). YGF-R2 corresponded to the minus strand of DNA just behind the *yckF* ORF. Then a 20-μl aliquot of the cDNA mixture was combined with DNA primers YGF-F1 (5′-ATGGAATTACAGCTTGCACTTAGACC-3′) and YGF-R1 (5′-TGTGATGCTATTCAAGGTTTGC-3′), LA Taq DNA polymerase, and PCR buffer (Takara Shuzo) in a volume of 100 μl. RT-PCR was carried out according to the manufacturer’s instructions. YGF-F1 and YGF-R1 corresponded to the plus strand of DNA encoding the N-terminal region of YckG and to the minus strand of DNA encoding the C-terminal region of YckF, respectively. As a negative control, mixture without RT was used for PCR amplification.

Computer analysis. Nucleotide and amino acid sequences were analyzed with the Genetyx-Mac computer program (Software Development Co., Tokyo, Japan). Alignment of deduced amino acid sequences was performed by comparison with entries in the SWISS-PROT database (release 35.0).

**RESULTS**

Overproduction and enzyme activity of the *yckG* and *yckF* gene products. To identify the orthologs of the gene encoding PHI, FASTA searches (30, 31) of the microorganism protein sequence database were performed, using the amino acid sequence of the *rmpB* gene product (PHI) (35) of *M. aminofaciens* 77a as the query sequence. As a result, the primary

YASUEDA ET AL.

YckF: 1' MKTTEYVAEILNELHNSAAYISNEEADQLADHILSSHQIFTAGAGRSGLM  
PHI: 1" MNKYQELVVSKLTNVINNTAEGYDDKILSLVDAAGRTFIGGAGRSLLV  
YckF: 51' AKSFAMRLMHMGFNAHIVGEILTPPLAEGDLVIIGSGSGETKSLIHTAAK  
PHI: 49" SRFFAMRLVHAGYQVSMVGEVVTPSIQAGDLFIVISGSGSTETLMPLVKK  
YckF: 101' AKSLHGIVAALTINPESSIGKQADLIIRMPGSPKDQSNGSYKTIQPMGSL  
PHI: 99" AKSQGAKIIVISMKAAQSPMAELADLVVPVGGNDDANAFDKT--HGMPMGTI  
YckF: 151' FEQTLLLFYDAVILKLMEEKKGLDSETMFTHANLE  
PHI: 147" FELSTLWFLEATIAKLVDQKGLTEEGMRAIHANLE  

FIG. 2. Alignment of amino acid sequences of YckF of *B. subtilis* (24) and PHI of *M. aminofaciens* 77a (35). Identical amino acids and conservative replacements are indicated by asterisks and dots, respectively. Amino acids are numbered from the N terminus of each protein.

sequence of the *yckF* gene product (YckF) in *B. subtilis* showed the highest structural similarity with that of PHI (Fig. 2). The *yckF* gene is situated five nucleotides downstream of the *yckG* gene in the *B. subtilis* genome (Fig. 1a), and the *yckG* gene product (YckG) is known to show structural similarity with HPS (24). Thus, both putative genes involved in one-carbon compound metabolism were situated side-by-side in the *B. subtilis* genome. Because *B. subtilis* cannot utilize methanol as its sole carbon source (i.e., it is a nonmethylothrop), the detection of such genes in this microorganism was interesting with respect to function and evolution.

We first expressed the *yckG*- and/or *yckF* ORF in *E. coli* to evaluate whether the translated product of each ORF displayed HPS or PHI activity. After each corresponding DNA fragment with the postulated ORF was amplified by PCR, the fragments were inserted into the expression plasmid and overproduction was stimulated (Fig. 3a). As a result, the *yckG* gene yielded a dense protein band with the expected molecular mass of 23 kDa on SDS-PAGE, and the *yckF* gene yielded a protein with a molecular mass of 19 kDa. Furthermore, overexpression of a DNA fragment containing the *yckG* and *yckF* genes in the intact gene structure led to concomitant overproduction of both ORF products (Fig. 3a, lane 3), showing that termination of transcription did not occur between the *yckG* and *yckF* genes in *E. coli*.

The HPS and PHI activities of cell extracts were examined by the enzyme assay method. *E. coli* overexpressing *yckG* or *yckF* showed HPS or PHI activity, respectively (Table 1). To confirm that formaldehyde was incorporated into pentose phosphate, we used the ${}^{13}$C NMR technique with [${}^{13}$C]formaldehyde. Preincubation of the reaction mixture containing Ri5P, PRI and other chemicals yielded Ru5P, which is an acceptor of formaldehyde in the reaction with HPS. Addition of a cell extract containing YckG and YckF to the mixture caused the appearance of the corresponding resonance signals for [${}^{13}$C]fructose-6-phosphate and [${}^{13}$C]glucose-6-phosphate, along with a decrease of the [${}^{13}$C]formaldehyde signal (Fig. 3b). This showed that the serial reaction catalyzed by both gene products could progress, because YckG produced hexulose-6-phosphate from formaldehyde and Ru5P, while YckF and PGI then created fructose-6-phosphate and glucose-6-phosphate, respectively. Since the two enzymes were previously thought to be unique to methylotrophs (11, 15), it was surprising that

*B. subtilis* contained nucleotide sequences that could express these enzyme activities.

Expression of *yckG* and *yckF* genes by *B. subtilis*. To investigate whether the *yckG* and *yckF* genes intrinsically expressed any enzyme activity in *B. subtilis*, we assessed extracts of cells grown under various culture conditions. The extract of wild-type cells cultured in LB medium showed negligible HPS or PHI activity, but both activities were detected after culture in the presence of formaldehyde (Table 2). When various concentrations of formaldehyde were tested, significant HPS and PHI activity was found in the cell extract at formaldehyde concentrations of around 0.5 mM. At a concentration of over 1 mM, however, both activities almost disappeared.

Formaldehyde is produced from methanol by methanol dehydrogenase and is also oxidized to formate by formaldehyde dehydrogenase in some methylotrophs (11). Therefore, we examined whether the synthesis of HPS and PHI could be induced by methanol or formate, but neither enzyme activity was detected after cells were grown in medium containing methanol or formate (Table 2). On the other hand, since formaldehyde is also synthesized by oxidation of methylamine in several methylotroph species (26), we looked for HPS or PHI activity in extracts of *B. subtilis* grown in medium containing methylamine hydrochloride (0.4 or 1.2% [wt/vol], final concentration) but did not observe the induction of these enzymes. Therefore, the nonmethylotroph *B. subtilis* possessed two key enzymes of the RuMP pathway, and the expression of both enzymes was induced by formaldehyde.

Analysis of *yckG* and/or *yckF* disruption mutants. We next examined whether the *yckG* or *yckF* gene encoded the observed formaldehyde-inducible HPS or PHI activity. We constructed a *yckG* mutant (strain YD101) and a *yckF* mutant (strain YD102) by insertional mutagenesis (Fig. 1a) and analyzed the formaldehyde-inducible HPS or PHI activity of each strain. After induction with formaldehyde, we detected no significant enzyme activity in the YD101 strain and only HPS activity in the YD102 strain (Table 2). Thus, we identified YckG and YckF as the only formaldehyde-inducible HPS and PHI enzymes, respectively, and our results also suggested that a polar effect occurred between the *yckG* and *yckF* genes. To confirm the transcription unit of the two genes, total RNA was isolated from wild-type *B. subtilis* cells exposed to formaldehyde and analyzed by RT-PCR. The presence of mRNA species (of at least 1.2 kb) spanning the *yckG* ORF to the *yckF* ORF (data not shown) was revealed, suggesting that the two genes (*yckG* and *yckF*) were expressed as a polycistronic mRNA from an operon.

We also investigated the effect of formaldehyde on the phenotype of these mutants. In LB medium, the wild-type strain and the two mutant strains all showed typical growth and lysis curves (Fig. 4a). However, after addition of formaldehyde to the medium, the mutants showed a prolonged lag phase relative to the growth profile of the wild-type strain, which was dependent on the concentration of formaldehyde added (Fig. 4). These results indicated that the *yckG-yckF* system might endow *B. subtilis* with the ability to detoxify formaldehyde produced via endogenous metabolism or environmental changes.

It appeared that the cells were under stress when exposed to formaldehyde, since growth was arrested for a few hours in its presence (Fig. 4c and d). To examine whether the increase of HPS and PHI activities after exposure to formaldehyde was due to a general stress response rather than to formaldehyde itself, we assessed the levels of these enzymes in cells grown under heat (increase from 30 to $48^{\circ} \mathrm{C}$ ), acid (decrease from pH 6.8 to 5.2), and salt (addition of 5% NaCl) stress. We also examined the growth profiles of wild-type and mutant cells

a SM 1 2 3 4 5 SM

yckG
yckF

b

Upper profile
FA

lower profile
G6-P
F6-P

FIG. 3. Analysis of the products of yckG and yckF. (a) The yckG and/or yckF gene was overproduced in E. coli and analyzed by SDS-PAGE. Cells harboring the expression plasmid were cultured in M9 medium supplemented with Casamino Acids, thiamine, and metal ions. Indoleacrylic acid was added to induce the expression of each gene. Aliquots were taken from cells harboring pT-Bsb-yckG (lane 2), pT-Bsb-yckGF (lane 3), pT-Bsb-yckF (lane 4), and the control plasmid (lane 5). The single band in lane 1 indicates the position of purified HPS of M. aminofaciens 77a, and the positions of YckG and YckF are

TABLE 1. HPS and PHI activity in E. coli bearing an expression plasmid

| Strain$^a$ | Sp act (nmol of NADP reduced/min/ mg of protein)$^b$ in cell extracts |
|-------------|---------------------------------------------------------------|
|             | HPS                          | PHI                           | HPS + PHI$^c$ |
| $E. coli$ JM109/pT-Bsb-yckG6 | 3,160                        | <1                            | ND$^d$       |
| $E. coli$ JM109/pT-Bsb-yckF1 | <1                           | 25,900                       | ND           |
| $E. coli$ JM109/pT-Bsb-yckGF1 | ND                           | ND                            | 3,970        |
| $E. coli$ JM109/pT-Bs-$\Delta$Nco$^e$ | <1                           | <1                            | ND           |

$a$ Cells were grown in modified M9 Casamino Acids medium with indoleacrylic acid.

$b$ Mean value from three experiments.

$c$ Total activity for serial reaction of HPS and PHI.

$d$ ND, not determined.

$e$ Negative control.

under salt stress. Enzyme activities were never detected except after exposure to formaldehyde, but comparison of the growth profiles of cells exposed to salt stress (5% NaCl) revealed a slight delay in the growth of both mutants relative to the wild-type strain. This observation suggests that the yckG or yckF gene may be involved also in the salt stress response, although HPS or PHI activity was below the level of detection, as described above.

Effect of the yckH gene on expression of the yckG and yckF genes. Upstream of the yckG-yckF genes, the yckH gene is an ORF transcribed in the reverse direction from the operon (Fig. 1a). This gene arrangement led us to speculate that yckH might be a regulator of yckG-yckF expression, by analogy with the L-arabinose (ara) operon structure (araC and araBAD) (39) in $E. coli$. It has been reported that yckH encodes a protein of 120 amino acids (41), but its function remains unclear. Although the overall amino acid sequence does not share any significant homology with DNA-binding proteins, a helix-turn-helix motif that is characteristic of nucleic acid-binding proteins (16) was indicated at the carboxyl terminus by our prediction of the secondary structure (data not shown). To examine the effect of the yckH gene on expression of yckG-yckF, the yckH locus was inactivated by insertion of an spc gene. We constructed two mutant strains, YD111 and YD112, in which the directions of the spc gene were the same as and opposite, respectively, that of to the yckH gene (Fig. 1a). Although read-through of transcription of the spc gene could proceed into the yckG-yckF region of the chromosome in YD112, neither of the yckH-disrupted mutants (strains YD111 and YD112) could express detectable HPS and PHI activity when exposed to formaldehyde (Table 2). We thought that in these mutants, the promoter element for the yckG-yckF genes might have been destroyed by spc gene insertion, since the location of the promoter region was not identified. Therefore, we constructed a new strain, YD121 ($yckHyckH^+$) (Fig. 1b), in which the DNA region just adjacent to the yckG-yckF locus remained disrupted as in

shown by arrows. The positions of migration of molecular mass standards (SM) are indicated on both sides of the gel (from top to bottom, 150, 100, 75, 50, 35, 25, and 15 kDa). (b) Formation of glucose-6-phosphate and fructose-6-phosphate by the condensation of $^{13}$C-labeled formaldehyde and Ru5P and the subsequent isomerization of sugar-phosphates was analyzed by $^{13}$C NMR spectroscopy as described in the text. The upper profile exhibits the peak of formaldehyde (FA) in the test mixture, in which the reaction was immediately stopped after addition of cell extract containing YckG and YckF; the lower profile displays the carbon peaks in the reaction mixture after incubation for 20 min at 37°C. The C-1 ($\alpha,\beta$ anomers) positions of glucose-6-phosphate (G6-P) and fructose-6-phosphate (F6-P) are indicated.

TABLE 2. HPS and PHI activities in *B. subtilis* 168 and deletion mutants under different growth conditions

| Strain           | Growth conditions$^a$ | Sp act (nmol of NADPH produced/ min/mg of protein)$^b$ in cell extracts |
|------------------|-----------------------|---------------------------------------------------------------|
|                  |                       | HPS                          | PHI                          | HPS + PHI                     |
| 168 wild type    | LB                    | <1                           | <1                           | ND$^c$                        |
|                  | LB + 0.5 mM FA        | 690                          | ND                           | 310                           |
|                  | LB + 1.0 mM FA        | 95                           | ND                           | <1                            |
|                  | LB + 2.0 mM FA        | <1                           | ND                           | <1                            |
|                  | LB + 2% (vol/vol)     | <1                           | ND                           | <1                            |
|                  | MeOH                  |                              |                              |                               |
|                  | LB + 0.5 mM FM        | <1                           | ND                           | <1                            |
| YD101 (*yckG*)   | LB + 0.5 mM FA        | <1                           | <1                           | ND                            |
| YD102 (*yckF*)   | LB + 0.5 mM FA        | 290                          | <1                           | ND                            |
| YD111 (*yckH*)   | LB + 0.5 mM FA        | <1                           | <1                           | <1                            |
| YD112 (*yckH*)   | LB + 0.5 mM FA        | <1                           | <1                           | <1                            |
| YD121 (*yckH yckH*<sup>+</sup>) | LB + 0.5 mM FA        | 1,050                         | ND                           | 200                           |

$a$ FA, formaldehyde; MeOH, methanol; FM, formate.

$b$ Mean value from three experiments.

$c$ ND, not determined.

Vol. 181, 1999 ONE-CARBON COMPOUND METABOLISM IN B. SUBTILIS 7159

![Graphs](https://i.imgur.com/yourimage.png)

FIG. 4. Effect of formaldehyde on the growth of *B. subtilis*. Overnight cultures of wild-type strain 168 (——), YD101 (*yckG*) (•••••), and YD102 (*yckF*) (---) in LB medium were inoculated into prewarmed LB medium containing formaldehyde at concentrations of 0.0 (a), 0.3 (b), 0.6 (c), and 0.9 (d) mM. Growth and lysis of cells at 37°C were monitored automatically at an OD of 660 nm.

ences in the *B. subtilis* strains used. Either way, the existence of two key enzymes was substantially established by our research.

The organization of divergent transcription between the two transcription terminators in the *B. subtilis* genome suggested that *yckH* may be a regulator gene (24). The putative gene product of *yckH* is estimated to be a polypeptide composed of 120 amino acids (41). However, our disruption experiment indicated that the *yckH* gene is required for the expression of HPS and PHI activities, and it appears that *yckH* (now named *hxlR*) may positively regulate *yckG-yckF* gene expression through DNA binding either directly or indirectly after stimulation by formaldehyde. We also suggest that *yckG* and *yckF* (now named *hxlA* and *hxlB*, respectively) are organized into an operon structure, based on the following data: (i) the existence of mRNA species covering the *hxlA* and *hxlB* ORFs, (ii) the polar effect between *hxlA* and *hxlB* genes, (iii) the simultaneous expression of the two genes in response to an inducer, (iv) the lack of *hxlB* expression concomitant with abolition of *hxlA* expression in an *hxlR* (*yckH*)-deficient mutant, and (v) the gene organization of *hxlA-hxlB* followed by a typical transcriptional terminator. Recently, it was shown that *M. gastri* MB19 contains a *phi* (*rmpB*)-*hps* (*rmpA*) operon (28), although the

* hps (*rmpA*) and *phi* (*rmpB*) genes of *M. aminofaciens* 77a are separated by an insertion element (35). We found that the *hps* (*hxlA*)-*phi* (*hxlB*) gene organization was conserved as a set of both genes even in a nonmethylothrop. Curiously, the genes of *B. subtilis* and *M. gastri* were arranged in reverse order. Therefore, more detailed bioinformatic analysis of the conserved features and differences in gene organization among three types of microbes (facultative methylotrophs, obligate methylotrophs, and nonmethylothrop) may provide insight into the evolution of the RuMP pathway.

### ACKNOWLEDGMENTS

We gratefully acknowledge N. Kato, Y. Sakai, and R. Mitsui (Kyoto University) for the generous gift of the enzyme samples (HPS and PHI from *M. aminofaciens* 77a) and for helpful discussion; E. Suzuki and N. Ootu for NMR analysis; and K. Sato, M. Ooba, and K. Kobayashi for technical support.

### REFERENCES

1. Anagnostopoulos, C., and J. Spizizen. 1961. Requirement for transformation in *Bacillus subtilis*. J. Bacteriol. 81:741–746.
2. Arfman, N., E. M. Watling, W. Clement, R. J. van Oosterwijk, G. E. de Vries,

W. Harder, M. M. Attwood, and L. Dijkhuizen. 1989. Methanol metabolism in thermotolerant methylotrophic *Bacillus* strains involving a novel catabolic NAD-dependent methanol dehydrogenase as a key enzyme. Arch. Microbiol. **152**:280–288.

3. Arfman, N., L. Bystrykh, N. I. Govorukhina, and L. Dijkhuizen. 1990. 3-Hexulose-6-phosphate synthase from thermotolerant methylotrophic *Bacillus* C1. Methods Enzymol. **188**:391–397.

4. Arfman, N., K. J. de Vries, H. R. Moezelaar, M. M. Attwood, G. K. Robinson, M. van Geel, and L. Dijkhuizen. 1992. Environmental regulation of alcohol metabolism in thermotolerant methylotrophic *Bacillus* strains. Arch. Microbiol. **157**:272–278.

5. Attwood, M. M., and J. R. Quayle. 1984. Formaldehyde as a central intermediary metabolite of methylotrophic metabolism, p. 315–323. *In* R. L. Crawford and R. S. Hanson (ed.), Proceedings of the 4th International Symposium on Microbial Growth on C₁ Compounds. American Society for Microbiology, Washington, D.C.

6. Attwood, M. M. 1990. Formaldehyde dehydrogenases from methylotrophs. Methods Enzymol. **188**:314–327.

7. Barns, S. M., C. F. Delwiche, J. D. Palmer, and N. R. Pace. 1996. Perspectives on archaeal diversity, thermophily and monophyly from environmental rRNA sequences. Proc. Natl. Acad. Sci. USA **93**:9188–9193.

8. Bradford, M. M. 1976. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal. Biochem. **72**:248–251.

9. De Vries, G. E., N. Arfman, P. Terpstra, and L. Dijkhuizen. 1992. Cloning, expression, and sequence analysis of the *Bacillus methanolicus* C1 methanol dehydrogenase gene. J. Bacteriol. **174**:5346–5353.

10. De Wulf, P. 1998. Presence of the ribulose monophosphate pathway in *Bacillus subtilis*. Microbiology **144**:596–597. (Comment.)

11. Dijkhuizen, L., P. R. Levering, and G. E. de Vries. 1992. The physiology and biochemistry of aerobic methanol-utilizing Gram-negative and Gram-positive bacteria, p. 149–181. *In* J. C. Murrell and H. Dalton (ed.), Methane and methanol utilizers. Plenum Press, New York, N.Y.

12. Ferenci, T., T. Strøm, and J. R. Quayle. 1974. Purification and properties of 3-hexulose phosphate synthase and phospho-3-hexuloisomerase from *Methylococcus capsulatus*. Biochem. J. **144**:477–486.

13. Guérout-Fleury, A.-M., K. Shazand, N. Frandsen, and P. Stragier. 1995. Antibiotic-resistance cassettes for *Bacillus subtilis*. Gene **167**:335–336.

14. Gutheil, W. G., B. Holmquist, and B. L. Valleé. 1992. Purification, characterization, and partial sequence of the glutathione-dependent formaldehyde dehydrogenase from *Escherichia coli*: a class III alcohol dehydrogenase. Biochemistry **31**:475–481.

15. Hanson, R. S., and T. E. Hanson. 1996. Methanotrophic bacteria. Microbiol. Rev. **60**:439–471.

16. Honikoff, S., G. W. Haughn, J. M. Calvo, and J. C. Wallace. 1988. A large family of bacterial activator proteins. Proc. Natl. Acad. Sci. USA **85**:6602–6606.

17. Johnson, P. A., and J. R. Quayle. 1965. Microbial growth on C₁ compounds. Synthesis of cell constituents by methane- and methanol-grown *Pseudomonas methanica*. Biochem. J. **95**:859–867.

18. Kato, N., H. Ohashi, T. Hori, Y. Tani, and K. Ogata. 1977. Properties of 3-hexulose phosphate synthase and phospho-3-hexuloisomerase of a methanol-utilizing bacterium, 77a. Agric. Biol. Chem. **41**:1133–1140.

19. Kato, N., H. Ohashi, Y. Tani, and K. Ogata. 1978. 3-Hexulosephosphate synthase from *Methylomonas aminoafaciens* 77a: purification, properties and kinetics. Biochim. Biophys. Acta **523**:236–244.

20. Kato, N., N. Miyamoto, M. Shimao, and C. Sakazawa. 1988. 3-Hexulose phosphate synthase from a new facultative methylotroph, *Mycobacterium gastri* MB19. Agric. Biol. Chem. **52**:2659–2661.

21. Kelly, D. P., G. Malin, and A. P. Wood. 1993. Microbial transformations and biogeochemical cycling of one-carbon substrates containing sulphur, nitrogen or halogens, p. 47–64. *In* J. C. Murrell and D. P. Kelly (ed.), Microbial growth on C₁ compounds. Intercept Ltd., Andover, Mass.

22. Kemp, M. B., and J. R. Quayle. 1967. Microbial growth on C₁ compounds. Uptake of (^{14}C)formaldehyde and (^{14}C)formate by methane-grown *Pseudomonas methanica* and determination of the hexose labelling pattern after brief incubation with (^{14}C)methanol. Biochem. J. **102**:94–102.

23. Kemp, M. B. 1974. Hexulose phosphate synthase from *Methylococcus capsulatus* makes d-arabino-3-hexulose phosphate. Biochem. J. **139**:129–134.

24. Kunst, F., N. Ogasawara, I. Moszer, et al. 1997. The complete genome sequence of the Gram-positive bacterium *Bacillus subtilis*. Nature (London) **390**:249–256.

25. Laemmli, U. K. 1970. Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature (London) **227**:680–685.

26. Large, P., and J. Green. 1984. Oxidation of mono-, di-, and trimethylamine by methanotrophic yeasts: properties of the microsomal and peroxisomal enzymes involved and comparison with bacterial enzyme systems, p. 155–164. *In* R. L. Crawford and R. S. Hanson (ed.), Microbial growth on C₁ compounds. American Society for Microbiology, Washington, D.C.

27. Levering, P. R., L. M. Croes, and L. Dijkhuizen. 1985. Regulation of methyamine and formaldehyde metabolism in *Arthrobacter* P1. Formaldehyde is the inducing signal for the synthesis of the RuMP cycle enzyme hexulose phosphate synthase. Arch. Microbiol. **144**:272–278.

28. Mitsui, R. 1998. Organization and regulation of the genes involved in the ribulose monophosphate pathway in methylotrophic bacteria. Ph.D. thesis. Kyoto University, Kyoto, Japan.

29. Mullis, K. B., and F. A. Faloona. 1987. Specific synthesis of DNA *in vitro* via a polymerase catalyzed chain reaction. Methods Enzymol. **155**:335–350.

30. Pearson, W. R., and D. J. Lipman. 1988. Improved tools for biological sequence comparison. Proc. Natl. Acad. Sci. USA **85**:2444–2448.

31. Person, W. R. 1990. Rapid and sensitive sequence comparison with FASTP and FASTA. Methods Enzymol. **183**:63–98.

32. Quayle, J. R. 1982. 3-Hexulose-6-phosphate synthase from *Methylomonas (Methylococcus)* *capsulatus*. Methods Enzymol. **90**:314–319.

33. Reizer, J., A. Reizer, and M. H. Saier, Jr. 1997. Is the ribulose monophosphate pathway widely distributed in bacteria? Microbiology **143**:2519–2520. (Comment.)

34. Sahm, H., H. Schütte, and M.-R. Kula. 1982. 3-Hexulose-phosphate synthase from *Methylomonas* M15. Methods Enzymol. **90**:319–323.

35. Sakai, Y., R. Mitsui, Y. Katayama, H. Yanase, and N. Kato. 1999. Organization of the genes involved in the ribulose monophosphate pathway in an obligate methylotrophic bacterium, *Methylomonas aminoafaciens* 77a. FEMS Microbiol. Lett. **176**:125–130.

36. Sambrook, J., E. F. Fritsch, and T. Maniatis. 1989. Molecular cloning: a laboratory manual, 2nd ed. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.

37. Sauer, U., V. Hatzimanikatis, H.-P. Hohmann, M. Mannenberg, A. P. G. M. van Loon, and J. E. Bailey. 1996. Physiology and metabolic fluxes of wild-type and riboflavin-producing *Bacillus subtilis*. Appl. Environ. Microbiol. **62**:3687–3696.

38. Sauer, U., V. Hatzimanikatis, J. E. Bailey, M. Hochuli, T. Szyperski, and K. Wüthrich. 1997. Metabolic fluxes in riboflavin-producing *Bacillus subtilis*. Nature Biotechnol. **15**:448–452.

39. Schleif, R. 1992. Regulation of the l-arabinose catabolic operon *araBAD*, p. 643–665. *In* S. L. McKnight and K. R. Yamamoto (ed.), Transcriptional regulation, vol. 2. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.

40. Speer, B. S., L. Chistoserdova, and M. E. Lidstrom. 1994. Sequence of the gene for a NAD(P)-dependent formaldehyde dehydrogenase (class III alcohol dehydrogenase) from a marine methanotroph *Methylobacter marinus* A45. FEMS Microbiol. Lett. **121**:349–356.

41. SubtiList. 1997. Data release R14.2. *In* I. Mozer and A. Danchin (ed.), http://www.pasteur.fr/Bio/SubtiList.html.[Online.] Institut Pasteur, Paris, France.

42. Van Spanning, R. J. M., C. W. Wansell, T. De Boer, M. J. Hazelaar, H. Anazawa, N. Harms, L. F. Oltmann, and A. H. Stouthamer. 1991. Isolation and characterization of the *moxJ, moxG, moxI,* and *moxR* genes of *Paracoccus denitrificans*: inactivation of *moxJ, moxG,* and *moxR* and the resultant effect on methylotrophic growth. J. Bacteriol. **173**:6948–6961.

43. Woese, C. R. 1987. Bacterial evolution. Microbiol. Rev. **51**:221–271.

44. Yanase, H., K. Ikeyama, R. Mitsui, S. Ra, K. Kita, Y. Sakai, and N. Kato. 1996. Cloning and sequence analysis of the gene encoding 3-hexulose-6-phosphate synthase from the methylotrophic bacterium, *Methylomonas aminoafaciens* 77a, and its expression in *Escherichia coli*. FEMS Microbiol. Lett. **135**:201–205.

45. Yanisch-Perron, C., J. Vieira, and J. Messing. 1985. Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. Gene **33**:103–119.

46. Yasueda, H., K. Nakanishi, Y. Kumazawa, K. Nagase, M. Motoki, and H. Matsui. 1995. Tissue-type transglutaminase from red sea bream (*Pagrus major*): sequence analysis of the cDNA and functional expression in *Escherichia coli*. Eur. J. Biochem. **232**:411–419.
