
Cell Death and Differentiation (1999) 6, 1028–1042

© 1999 Stockton Press All rights reserved 13509047/99 $15.00

http://www.stockton-press.co.uk/cdd

Review

# Caspase structure, proteolytic substrates, and function during apoptotic cell death

**DW Nicholson**,¹

¹ Merck Frosst Centre for Therapeutic Research, Merck Frosst Canada & Co., PO Box 1005, Pointe Claire – Dorval, Quebec, Canada, H9R 4P8

* Corresponding author: DW Nicholson Merck Frosst Centre for Therapeutic Research, Merck Frosst Canada & Co., PO Box 1005, Pointe Claire – Dorval, Quebec, Canada, H9R 4P8. Fax: +1 514 428-4900; E-mail: donald_nicholson@merck.com

Received 10.9.99; accepted 21.9.99;

Edited by G Melino

## Abstract

Caspases play an essential role during apoptotic cell death. These enzymes define a new class of cysteine proteases and comprise a multi-gene family with more than a dozen distinct mammalian family members. The discrete and highly limited subset of cellular polypeptides that are cleaved by these proteases is sufficient to account for the majority of cellular and morphological events that occur during cell death. In some cases, caspases also play a contributory role in escalating the propensity for apoptosis, and in doing so may exacerbate disease pathogenesis.

**Keywords:** apoptosis; caspase; protease; neurodegeneration

**Abbreviations:** ICE, interleukin-1β converting enzyme; caspase, cysteinyI aspartate-specific proteinase; CED, product of cell-death-abnormal gene

## Proteases in apoptosis (the ICE age)

ICE (interleukin-1β converting enzyme; caspase-1) is the prototypical caspase and was initially identified as the protease responsible for the proteolytic maturation of proIL-1β to its pro-inflammatory, biologically active form.¹,² When originally discovered, ICE defined a new class of cysteinyI proteases that was distinguishable from other cysteine protease families based on general structural organization and the absolute requirement for aspartic acid in the P₁ position of the scissile bond. A key role for ICE in inflammation had been largely established, which was quickly secured by inhibitor studies and by the phenotype of ICE-deficient mice, but other potential biological functions were not evident. At approximately the same time, a genetic pathway for cell death was being defined in the nematode *C. elegans*.³ One of the genes in this genetic pathway, *ced-3*, encoded a protein that was essential for all 131 programmed cell deaths that occurred during hermaphrodite development. When *ced-3* was cloned and sequenced, it was found to be a *C. elegans* homologue of mammalian ICE.⁴,⁵ This and other evidence strongly implicated ICE (or related family members, as it turned out) in a similar mammalian cell death pathway. Importantly, this collective information also demonstrated an essential role for specific proteolysis in apoptotic cell death, which led to the identification of proteolytic ‘victims’ of the caspases and shed light on the biochemical events that occurred as a consequence of their cleavage. The importance of *C. elegans* genetics in defining the mammalian cell death pathway is underscored by the fact that the individual cell death components, their molecular ordering and cellular functions have been largely conserved throughout evolution⁶ (Figure 1). In addition to caspases, other proteases also contribute to the apoptotic cell death pathway. The serine protease granzyme B, for example, has been well established as a caspase activator during CTL-mediated killing, and owing to its P₁ Asp bias can function as a caspase surrogate when caspases themselves are inoperative.⁷⁻⁹ The calpains have also been implicated in apoptosis,¹⁰ although their precise role remains to be determined.

### The mammalian caspase gene family and functional sub-families

The caspase gene family¹¹ thus far contains at least 14 mammalian members, of which 11 human enzymes are known (Figure 2). A phylogenetic analysis indicates that the gene family is composed of two major sub-families which are related to either ICE (caspase-1) or to the mammalian counterparts of CED-3. Further subdivisions can be made depending on whether the proenzymes harbour short prodomains (caspases-3, -6, -7) or long prodomains (the remainder). Alternatively, these proteases can be subdivided on the basis of their substrate specificities which has been defined using a positional scanning combinatorial substrate library.¹²,¹³ Using the latter technique, the proteases fall into only three specificity subgroups (Figure 3). As expected from earlier studies, the major specificity determinant is the S₄ subsite (most of the enzymes are promiscuous at P₂ and P₃, although they prefer His or Ile in P₂ and prefer Glu in P₃). Group I caspases (1, 4, 5, 13) are tolerant of liberal substitutions in P₄ but prefer bulky hydrophobic amino acids such as Tyr or Trp. This preference is consistent with their role in cytokine processing but does not support a substantial role in apoptosis since none of the polypeptides that are cleaved during apoptosis contain hydrophobic residues in P₄. The group II caspases (2, 3, 7) are substantially more stringent in S₄, requiring a P₄ Asp. This specificity and stringency is nearly indistinguishable from that of *C. elegans* CED-3. The preferred cleavage motif (DExD) for group II caspases appears in many proteins that are cleaved during cell death, consistent with group II caspases being the major effectors of cell death. Group III caspases (6, 8, 9, 10), on the other hand, prefer branched chain aliphatic amino acids in P₄; residues

that are found at the maturation site of most group II and group III caspases. This specificity is consistent with the group III enzymes being upstream activators of the group II effector caspases (Figure 4). This molecular ordering of group III and group II caspases has been upheld in several cases and is best exemplified by caspase-8-mediated activation of caspase-3 and -7 in the CD95 (Fas, APO-1) system,^{14,15} and caspase-9-mediated activation of caspase-3 in the

APAF-1/cytochrome c pathway^{16} (both of these examples have been substantiated in caspase-8 or -9 knock-out mice as well).^{17–20} One possible exception to this general ordering based on substrate specificity, which remains unresolved, is whether caspase-6 (a group III caspase) plays an effector role (e.g. lamin proteolysis)^{21} instead of or in addition to a putative activation role. Another exception may be caspase-2 which appears to be a self-activating effector caspase.^{12,22,23}

**Regulation** | **Execution** | **Engulfment** | **Nuclear**
--- | --- | --- | ---
C. elegans | EGL-1 | CED-9 | CED-4 | CED-3 | CED-1 | CED-6 | CED-7 | nuc1
Mammalian | Bid/Bad (BH3 subfamily) | Bcl-2/ Bcl-XL (Bcl-2 subfamily) | APAF-1 | Caspases | ABC1 (CED-7) | DOCK180 (CED-5) | CAD

**Figure 1** C. elegans cell death genetic pathway and mammalian counterparts

**ICE Subfamily**

| caspase |
| --- |
| cysteinyI aspartate-specific proteinase |
| cysteinyI aspartate-specific proteinase |

| caspase-13 (ERICE) |
| --- |
| caspase-5 (ICE_rel-III, TY) |
| caspase-4 (ICE_rel-II, TX, ICH-2) |
| caspase-1 (ICE) |

**Inflammation Group I**

**CED-3 Subfamily**

| caspase-7 (Mch3, ICE-LAP3, CMH-1) |
| --- |
| caspase-3 (CPP32, apopain, Yama) |
| caspase-6 (Mch2) |

| caspase-8 (MACH, FLICE, Mch5) |
| --- |
| caspase-10 (Mch4) |
| caspase-2 (ICH-1) |
| caspase-9 (ICE-LAP6, Mch6) |

**Apoptosis Group II Group III**

**Figure 2** The human caspase gene family. Caspases segregate into two major phylogenetic sub-families (ICE, CED-3). Based on their proteolytic specificities (see Figure 3), caspases further divide into three groups: group I enzymes (blue) mediate cytokine maturation whereas the apoptotic caspases are either group II (red) effectors of cell death or group III (green) upstream activators. Most caspases have long prodomains (>10 kDa) except for caspases-3, -6 and -7 (box) which have short peptidic prodomains (<30 aa). With the exception of caspase-13, the human chromosomal location for all of the caspases has been determined. At least two gene clusters have been identified, consistent with some caspases arising from tandem gene duplication. These include the caspases-1, -4, -5 gene cluster on 11q22.2-q22.3 and the caspases-8, -10, cFLIP/Usurpin gene cluster on 2q33-q34.^{71,72} (cFLIP/Usurpin (aka CASH, Casper, CLARP, FLAME-1, I-FLICE, MRIT) is homologous to caspases -5, -8 and -10, except that substrate binding and catalytic determinants are absent, making it a dominant-negative death repressor.) The human counterparts of murine caspases-11, -12 and -14 have not yet been identified (although murine caspase-12 may be equivalent to human caspase-5)

Caspace structure and role during apoptotic cell death
DW Nicholson

General caspase structural features
Caspases are synthesized as catalytically-dormant tripartite proenzymes (Figure 5). Both the large and the small subunits, which together make up the active form of the enzyme, are liberated from the proenzyme by cleavage at Asp(P₁)-X(P₁') bonds. The presence of Asp at the maturation cleavage sites is consistent with the ability of caspases to auto-activate or to

**Caspase Substrate**


P₄    P₃    P₂    P₁


Group I Hydrophobic Caspases-1, 4, 5, 13  
Group II Asp Glu Xxx Asp Caspases-2, 3, 7, CED-3  
Group III Aliphatic Caspases-6, 8, 9, 10, (Granzyme B)

Figure 3 Caspase proteolytic specificity. The caspases recognize a core tetrapeptide motif corresponding to the four residues N-terminal to the scissile bond (P₄-P₃-P₂-P₁). A positional-scanning combinatorial substrate library, comprised of all 8000 possible P₁ Asp tetrapeptides, segregates the caspases into three specificity groups with the indicated sub-site preferences.¹²,¹³ Caspase-2 also appears to have a P₅ requirement.⁷³

**Group III**
Caspase-6 | Caspase-8 | Caspase-9 | Caspase-10
---|---|---|---
| | | |
| | | |
| | | |

**Group II**
Caspase-2 | Caspase-3 | Caspase-7
---|---|---

Apoptosis

Figure 4 Molecular ordering of caspases. Exceptions to this general ordering may also exist. Caspase-2, for example, may be self activating. Caspase-6 may function as an effector protease. Amplification circuits also exist. The caspase complement varies considerably between different cell types, directly bearing on the pathways available and how they function within different cellular environments.


be activated by other caspases as part of an amplification cascade. Components of the proteolytic device, including the active site Cys and His residues, are harboured within the large subunit whereas residues which form the S₁ subsite that tethers the carboxylate side chain of the essential P₁ Asp are derived from both the large and small subunits (Figure 6). Similarly, both the large and the small subunit contribute residues to form the substrate binding cleft (S₄-S₁), although the major determinants for substrate specificity (e.g. S₄) are contained within the small subunit. Prodomain structures vary considerably between different caspase family members ranging from small peptides with unknown (if any) function (e.g. caspases-3, -6, -7) to large domains that are involved in

---

**Figure 5 Caspase proenzyme organization**


N- ──────────────────────── 32-53 kDa ──────────────────────── C-
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
    │                       │                        │
formed the basis for inhibitor and synthetic substrate design. As described above, these proteases have an absolute requirement for Asp in P₁, are promiscuous in P₂, prefer Glu in P₃, but have varying preferences in P₄ that enable their assignment to one of three generic subgroups (I, II, III). Despite these apparently simple requirements, however, caspases are extremely stringent, indicating that three-dimensional context and the appropriate surface presentation are key factors in determining whether the presence of an appropriate motif also makes it eligible for caspase proteolysis within a polypeptide. The molecular basis of high affinity substrate binding and the specificity determinants at S₄ have been demonstrated for caspases-1 and -3 and can be inferred, as a consequence, for the other caspases. The active site groove is well defined and is extended along the surface of the enzyme (Figure 8). The carboxylate side chain

of the P₁ Asp fits into a highly restrictive ‘socket’ and is tethered by hydrogen bond interactions with three residues that are conserved in all caspases (Arg¹⁷⁹, Gln²⁸³, Arg³⁴¹) (Figure 9). The tight physical dimensions of S₁ accounts for the lack of tolerance for anything other than Asp in this position. The P₂ and P₃ binding sites (S₂ and S₃) are reasonably distinctive, although tolerant of broad substitutions. The peptide backbone of the bound substrate (or peptidyl inhibitor) forms main-chain hydrogen bonds with Ser³³⁹ (conserved in most caspases) and Arg³⁴¹ (conserved in all caspases) as it extends along the binding cleft through these sites. The P₄ binding site (S₄), which is the key determinant of substrate specificity, varies markedly between the different caspase family members. This is best exemplified in a comparison of caspase-1 versus caspase-3 where the S₄ subsites vary radically in both geometry and chemical nature (see Figure 8). In the case of caspase-1, S₄ is a large shallow depression on the protease surface that readily accommodates bulky hydrophobic residues, such as the preferred Tyr or Trp. This site accommodates other residues as well; hence the relative promiscuity of this enzyme. Caspase-3, on the other hand, has a well defined and narrow S₄ pocket that envelops the side chain of the P₄ Asp. The intricate network of polar interactions and the physical geometry of the site accounts for the strong Asp preference and the enzymes overall specificity profile. The physical shape of the pocket is conferred in part by Trp³⁴⁸ (conserved in all groups II and III but not group I caspases) and by a small subunit-derived surface loop that makes an irregular reverse turn over the active site and contributes to the formation of S₄. Overall, this information affords a molecular understanding of the features which dictate the specificity of these enzymes and their relative promiscuity or stringency. Following substrate binding, catalysis employs a typical cysteine protease mechanism involving a catalytic dyad that is composed of Cys²⁸⁵ and His²³⁷, plus an ‘oxyanion hole’ involving Gly²³⁸ and Cys²⁸⁵ (all of which are conserved in all caspases). Interestingly, inhibitors bind in an unexpected non-transition state configuration with the oxyanion of the thiohemiacetal being stabilized by the active site His²³⁷.

### Caspase inhibition

The sufficiency of a P₄-P₁ tetrapeptide for caspase recognition and high-affinity binding has been the platform for most of the currently available caspase inhibitors (as well as fluorogenic and colorimetric substrates). Suitable electrophiles that can interact reversibly with the active site Cys²⁸⁵ include aldehydes, nitriles and ketones. Because of the anticipated stability of ketones *in vivo*, this class of inhibitor is well suited for further development. Irreversible caspase inhibitors, which form covalent adducts with the active site Cys²⁸⁵, are of the general structure [tetrapeptide]-CO-CH₂-X, including ketones where X is —Cl or —F (chloro- or fluoromethylketones), -N₂ (diazomethylketones) or —OCOR ((acyloxy)methylketones). Inhibitors of the latter class ((acyloxy)methylketones) are the most promising of the irreversible inhibitors owing to their very high potency against caspase enzymes and low intrinsic reactivity with other biological nucleophiles. The major challenge in developing

**Figure 8** Topology of caspase-1 (A) and caspase-3 (B) active sites. The solvent-accessible surface is shown in green. Bound inhibitors are shown in yellow (nitrogens are coloured blue, oxygens are red). Caspase-1 (A) is shown with Ac-WEHD-CHO¹³ whereas caspase-3 (B) is shown with Ac-DEVD-CHO²⁶ (aldehyde inhibitor versions of the optimal substrate for each enzyme). The major subsites (S₄, S₃, S₂, which bind their respective P₄, P₃, P₂ residues) are indicated. The P₁ Asp penetrates into the plane of the figure and is not visible. Note the major difference in S₄ subsite which is a large, open depression in caspase-1 versus a smaller, tighter pocket in caspase-3

Caspase structure and role during apoptotic cell death
DW Nicholson

Asp-X QACXG Asp-X

prodomain large subunit small subunit

| CASPASE-1 | 176 | IPRRTAEVITGMTMLLQNTSSDSTTEEAFFHRPEKTSFTFLFMTGIRE-------------------------GCKKHSEQVPDILQLNAFNMLNTK |
| CASPASE-2 | 199 | LEFRSGDVHSTTVTLRKLDDHHLLCDQQEQKQONFQOLPARVRTSCIVALLSNGVEG-------------------ASKQVDG------------------KLLQLQEYFQLSDNA |
| CASPASE-3 | 61 | MTSSSTDVAANRETRNKKERNKNDEEETVELMRDVSKEDHSKRSEFVCLEEG------------------------IFNG--------------------PVDLKKTNFERGD |
| CASPASE-4 | 149 | LPPRGADFVITGMKELELGRDXSDVEENPRDMESALRAFTRPERKSSDSTPLMLMGILE------------------GQC---------------------VHDEKKPDVLLYDTFQIINNR |
| CASPASE-5 | 190 | LPARNAHYDIVGMKRLLOQPXTVVDEKNPRDMESVARAFWARPENKSSDSTPLMLMGILE-----------------GTC--------------------AHKKKDPDVLLYDTFQIINNR |
| CASPASE-6 | 61 | LPERRRTCARDNTRRSSDFEKCFNKAEELLIKHEVST-VSADACFCVCMFEGEGN----------------------HYA--------------------KIBIQTLTGLSKGD |
| CASPASE-7 | 84 | MGVRNGTDKDAEAFKCESRSFDVIMYNDCSACKQDLKKASE-EDTNAACFCACILEEN--------------------VYKDG------------------VTPIKDLTAHRGD |
| CASPASE-8 | 257 | IRDRNTHLAGATTBEEHFELKPHECVEQIYEISKIYQL-MDSNMCFICCIDKG------------------------ISQ--------------------EPPIYELTSQTGL |
| CASPASE-9 | 177 | LRTTRSNICEKRRRESSHFMEVKGDIPKKVLAGLEBLR-QDHGALCCVVMSHGCAQASHLQFGAVDGG-----------CPVSVEKVNIINGT |
| CASPASE-10 | 254 | LKDROTHKAEILSHVQWETHTLHHNVKVEEMVHQKQCNPAADGQCFVFCITHORFG--------------------AVSSBE-----------------ALIPIREMSSHITAL |
| CASPASE-13 | 149 | MPPRNAAALDILGMKQLLEGALMTVEEEKIRARDMESVWKFWAREEHKSSSTFLPMSHGILD---------------GOCMHSEEEPDVLPYDTFRTHNNR |

| CASPASE-1 | NPSUKDKIVMKDSPGV-VWFKDSVGVSGNLSLPTTEEFEDDATIKSAHIKPIAFCSTHDNSWRHPTMVYDGRLEIH 356 |
| CASPASE-2 | NPSTQNKRMFRBCRDTRGVDDQQDGKNHAGSPGCBEESDAGKEKLPKMRLTRSDMICGYACLKTAAAMRNTKRFYEAIAQV 376 |
| CASPASE-3 | RGRSLTGSLRSTTELDC-GIETSG-VDDMAC--------------------------HNIEVRADELYAYSTATAYYRNSKDCAM 222 |
| CASPASE-4 | NLSDLKDKIVVCRGANRGE-LWVRDSPASLEVASQSSENLE------------------EDAVYKTHVIKFIAFCSTSHNSWRDSTMIRTQMITC 329 |
| CASPASE-5 | NLSDLKDKIVVCEHKGE-LWVRDSPASLAVISQSSENLE-------------------ADSVCKIHEKDEIAFCSTSHNSWRDTRTIRITEITC 370 |
| CASPASE-6 | KSHSVGKRIITACRGNQHDVPVIPLDVVDNQTEKLDTNITEVD----------------AASVYTLAQABLMCYSVAEYYYHERETVNAMYQDLCEM 235 |
| CASPASE-7 | RKTKTLELKLPCRELDD-GIQADCGPINDTDANPR-----------------------YKIPVEADELFAYSTVYYERSPGRCWEVQAICSII 248 |
| CASPASE-8 | KGPSLAGRVVVFQIDNYQK-GIPVETDSEEQPYLEMDLS-------------------SPQTRYIDRAERLLGMAATVNNCSYYRNPAECMYQSWQS 428 |
| CASPASE-9 | SIPSUGGELFLQGBEQKDHGEFASTSPDEDESPGSNPEPDAATPFQEGLRTFDQLDAISSLTTPSUIFVSYTFPFRDPKSQYVETRDDI 370 |
| CASPASE-10 | QPRDAEBRLXFFCOEEIQP-SVSIEDALNFEQAPTSLO--------------------SSIMAALLGLATVQYFHVEEYMYQSECNH 422 |
| CASPASE-13 | NLSDLKDKIVVCRGANRGE-LWVRDSPPALADSFSQSSENLE----------------EDAVYKTHVIKFIAFCSTSHNSWRDIKKGLTRMITC 329 |

Figure 9 Conservation of residues critical for substrate binding and catalysis. The area shown bridges most of the large subunit and part of the small subunit (as indicated in upper panel by grey frame). The catalytic dyad (blue), residues which participate in stabilization of the carboxylate side chain of the P₁ Asp (red) and residues that contribute to the ‘oxyanion hole’ (green) are indicated. Known or predicted maturation sites between the large and small subunits are highlighted in yellow. Caspase-1, for example, undergoes two cleavage events to liberate a linker peptide that separates the large and small subunits. In caspase-3, however, the large subunit is contiguous with the small subunit. Caspase-9 can be activated by either autolytic cleavage (left site) or by a caspase-3-mediated amplification event (right site). Numbering is based on caspase-1 residue positions.

inhibitors that are suitable for the current state of apoptosis research or for therapeutic usage is replacement of the tetrapeptide with non-peptide moieties. Peptide-based inhibitors have severely limited utility in cell-based and *in vivo* models owing to their very poor membrane permeability and other substantial disadvantages including poor metabolic stability. Despite these obstacles, interesting experiments have been performed *in vivo* with the esterified non-selective tripeptide-fluoromethylketone, Z-VAD(OMe)-CH₂F. The non-esterified counterpart of this inhibitor is modestly potent against most caspases, with the exception of caspase-2, although drawbacks of this compound include its lack of chemical stability (*t*₁/₂ of free acid < 50 min)²⁸ and electrophilic promiscuity which enables it to attack other biological nucleophiles, including cathepsins.²⁹ Several macromolecular caspase inhibitors have also been identified, including baculovirus p35³⁰⁻³² (a broad-spectrum caspase inhibitor), the cowpox serpin CrmA (selective for group I and III caspases, but not group II enzymes),²⁸ and members of the IAP superfamily (which appear to be largely selective for group II caspases).

### Caspase activation mechanisms

At least three distinct pathways for caspase activation exist in mammalian cells: (1) recruitment-activation, (2) trans-activation, and (3) autoactivation. In the first case, two examples of caspase activation following recruitment of multiple homologous proenzymes to a common site have been demonstrated. Ligation of the CD95 (Fas, APO-1) receptor, for example, recruits procaspase-8 to an oligomeric activation complex using the adapter protein FADD/MORT1.¹⁴,¹⁵ This mechanism appears to be common to other ‘death domain’-containing receptors (e.g. TNF-R1) and probably to caspase-10 as well. Similarly, oligomerization of procaspase-9 is mediated by APAF-1 following its release from Bcl-Xₗ and a cytochrome c-dependent conformational change.¹⁶ In this

case, recruitment is mediated by homophilic CARD-domain interactions. Other recruitment-activation mechanisms appear to exist (e.g. RAIDD-mediated activation of caspase-2²²,²³ and CARDIAK-mediated activation of caspase-1),³⁷ but are less well characterized. Following recruitment of multiple caspase proenzymes to a common oligomerization site, the low level of endogenous catalytic activity that the proenzymes harbour is sufficient to initiate full catalytic activation by proteolysis of the Asp-X site at the junction between the large and small subunits.³⁸⁻⁴¹ Trans-activation of one caspase by another is a second well established mechanism for caspase proenzyme maturation and activation. In general, upstream group III activator caspases (e.g. caspases-8 or -9 once they have undergone recruitment-activation) cleave and activate downstream group II effector caspases (e.g. caspases-3 or -7) by proteolysis of the Asp-X site between the large and small subunits. CTL-derived granzyme B can also mediate this event and thereby ‘trick’ the target cell into launching its endogenous apoptotic response.⁷⁻⁹ These trans-activation events are assisted by the activity of Hsp60, suggesting that the vulnerability of group II effector caspases to activation by upstream group III caspases is modulated by Hsp-driven folding and/or unfolding of the proenzyme.⁴² Finally, caspases can, in principle, undergo autocatalytic activation, although definitive proof for a non-recruitment type of autoactivation has not been established. One indication that such a mechanism may exist comes from the observation that RGD peptides can directly stimulate the autoactivation of procaspase-3.⁴³

### Substrates for caspases during apoptosis

During apoptotic cell death, only a fraction of the cellular proteome is cleaved by caspases. Current estimates based on comparative 2-D gel analysis between healthy and apoptotic cells place the number at fewer than 200 polypeptides.⁴⁴ Thus far, about 70 of these caspase

Caspase structure and role during apoptotic cell death

DW Nicholson

**Figure 10 Substrates cleaved by caspases during apoptosis.**

| **Site** | **Cleaved Protein** |
|----------|---------------------|
| DEVD     | PARP                |
| DEVD     | DNA-PKcs            |
| DVLD     | Rad51               |
| DELD     | Acinus              |
| DETD     | DFF45/ICAD site I   |
| DAVD     | DFF45/ICAD site II  |
| DEVD     | DNA-RFC140          |
| DEAD     | Rb                  |
| DVPD     | HDM2/MDM2           |
| DHVD     | p21CIP1/WAF1        |
| DSLD     | NuMA                |
| DYPD     | ATM                 |
| DGPD     | U1-70K snRNP        |
| DXXD     | hnRNP-C1/C2         |
| DEPD     | SREBP               |
| DRGD     | IkB-α               |
| DELD     | D4-GDI              |
| DELD     | cPLA2               |
| DMQD     | PKC δ              |
| DEVD     | PKC θ               |
| DGVD     | PKC ζ               |
| DTVD     | MEKK-1              |
| DEMD     | Mst1                |
| DITD     | PRK2                |
| DEQD     | PP2A                |
| DQTD     | FAK                 |
| DETD     | αII-Fodrin          |
| DEVD     | βII-Fodrin          |
| DQTQ     | Gelsolin            |
| DALD     | Cytokeratin-18      |
| DXXD     | LAP2                |
| DITD     | Nup153              |
| DESD     | Rabaptin-5          |
| DNID     | APC                 |
| DEED     | Hsp90               |
| DQPD     | UbqCE NEDD4         |
| DAGD     | V                   |
| DSYD     | Presenilin 2        |
| DXXD     | Huntingtin          |
| DEDD     | SBMA-AR             |
| DSLD     | Atrophin-1          |

| **Site** | **Cleaved Protein** |
|----------|---------------------|
| MELD     | STAT1               |
| NSPD     | Sp-1                |
| SELD     | SRP p72             |
| VFTD     | NF-κB               |
| YVPD     | PITSLRE Kinase      |
| SHVD     | PAK-2               |
| EERD     | p59FYN              |
| PAPD     | CaMK-IV             |
| AAVD     | p28 Bap31           |
| ELPD     | Actin               |
| SRVD     | Gas2                |
| VEID     | Lamin A             |
| VEVD     | Lamin B             |
| HLAD     | Bcl-X<sub>L</sub>   |
| LQTD     | BID                 |
| VEVD     | β-APP               |
| SSTD     | proIL-16            |
| XXXD     | pro-Caspases        |

| **Site** | **Cleaved Protein** |
|----------|---------------------|
| XXXD     | MCM3                |
| XXXD     | p27KIP1             |
| XXXD     | Wee1                |
| XXXD     | CDC27               |
| XXXD     | SAF-A/hnRNP-U       |
| XXXD     | hnRNP-A1            |
| XXXD     | RasGAP              |
| XXXD     | Raf1                |
| XXXD     | Akt1                |
| XXXD     | Cbl                  |
| XXXD     | Cbl-b               |
| XXXD     | PKN                  |
| XXXD     | β-, γ-Catenin       |
| XXXD     | Kinectin            |
| XXXD     | Calpastatin         |
| XXXD     | Ataxin-3            |
| XXXD     | AMPA Receptors      |

'victims' have been identified (Figure 10) and in most of these cases the cellular rationale for proteolysis during cell death can be reasonably predicted. For example, one of the hallmark events of apoptotic cell death is genomic disassembly and breakdown into oligonucleosomal fragments. Caspases disable normal DNA repair processes, in order to prevent counterproductive events from occurring simultaneously, by inactivating at least two key proteins involved in the homeostatic maintenance of genomic integrity; PARP and DNA-PK. At the same time, an apoptosis-dedicated endonuclease (CAD) is activated by caspase-mediated crippling of its cognate inhibitor (ICAD/DFF45). Together these cleavages contribute to the changes in the genomic DNA that accompany apoptotic cell suicide. Comparable scenarios can be envisioned for most of the polypeptides that are cleaved by caspases during cell death and the biological consequences that are associated with the apoptotic phenotype. The cumulative effects of these cleavage events are to (a) disable homeostatic and repair processes, (b) halt cell cycle progression, (c) inactivate inhibitors of apoptosis, (d) mediate structural disassembly and morphological changes, and (e) mark the dying cell for engulfment and disposal. In order to mediate these events, caspases can modify the function of their target polypeptides in one of four generic ways. For example, they can either inactivate the normal biochemical function of their substrates (e.g. ICAD, PARP, DNA-PK<sub>cS</sub>) or activate them by removal of regulatory domains (e.g. cPLA<sub>2</sub>, PKCs, SREBP). Alternatively, caspases can alter or invert the function of their target proteins (e.g. BID, which converts from mildly to strongly apoptotic following caspase proteolysis; Bcl<sub>2</sub> and Bcl-X<sub>L</sub>, which switch from anti- to pro-apoptotic following cleavage). Finally, the structural components of the cytoskeleton and nuclear scaffold require disassembly during apoptosis and caspases play a key proteolytic role in these steps as well (e.g. lamins, fodrin, gelsolin). Caspase cleavage normally occurs at a single, discrete site within the target polypeptide, although examples of multiplicity, redundancy and nesting also exist (Figure 11).

---

factor Sp1),<sup>163,164</sup> SRP p72 (72 kDa protein of signal recognition particle),<sup>165</sup> NF-κB (nuclear factor-κB transcription factor),<sup>166,167</sup> PITSLRE kinase (p34<sup>CDC2</sup>-related protein kinases; note that additional cleavage sites have been identified in addition to this major site (TEGD/Y and DDRD/S)),<sup>168–170</sup> PAK2 (aka PAK65, γ-PAK, PAKI; p21-activated protein kinase),<sup>171–173</sup> p59<sup>FYN</sup> (src-like tyrosine kinase p59<sup>FYN</sup>),<sup>174</sup> CaMK-IV (Ca/calmodulin-dependent protein kinase IV),<sup>175</sup> p28 Bap31 (BCR-associated protein, 28 kDa Bcl2-interacting protein),<sup>176,177</sup> actin,<sup>178–183</sup> Gas2 (growth arrest-specific gene product 2),<sup>21,185–189</sup> Bcl-X<sub>L</sub> (long version of Bcl-2-related gene product X),<sup>190,191</sup> BID (BH3 interacting domain death agonist),<sup>192–194</sup> β-APP (amyloid-β precursor protein),<sup>47–49,195,196</sup> proIL-16 (pro interleukin-16),<sup>197</sup> pro-caspases,<sup>12</sup> MCM3 (minichromosome maintenance protein 3, nuclear replication factor),<sup>198</sup> p27<sup>KIP1</sup> (27 kDa cyclin dependent kinase inhibitor),<sup>102</sup> Wee1 (Wee1 kinase, inhibitor of cdc2 and cdk2),<sup>199</sup> CDC27 (CDC27 component of anaphase-promoting complex),<sup>199</sup> SAF-A/hnRNP-U (scaffold attachment factor A, heteronuclear ribonucleoprotein U),<sup>200</sup> hnRNP-A1 (heteronuclear ribonucleoprotein A1),<sup>44</sup> RasGAP (Ras GTPase-activating protein),<sup>201</sup> Raf1 (Ras activated/associated factor 1),<sup>201</sup> Akt1 (aka protein kinase B),<sup>201</sup> Cbl (Cbl protooncogene product, negative regulator of receptor tyrosine kinase signaling),<sup>201</sup> PKN (protein kinase N),<sup>202</sup> catenin (aka plakoglobin),<sup>203–207</sup> kinectin,<sup>208</sup> calpastatin (calpain inhibitor),<sup>209,210</sup> ataxin-3 (gene product defective in spinocerebellar ataxia type 3 (SCA3)),<sup>160</sup> AMPA receptors<sup>211</sup>

Pathogenic exacerbation by caspase-mediated cleavage of disease-associated substrates

Apoptotic cell death is dependent on caspase activity and inappropriate apoptosis contributes to or accounts for several disease pathogeneses. In some cases, however, caspases also appear to play a role in aberrant processing events that culminate in an increased propensity or vulnerability to cell death. Recent examples include the role of caspases in polyglutamine-repeat disorders and Alzheimer's disease (Figure 12). Huntington's disease, for example, is a progressive neurodegenerative disorder in which the molecular defect is attributable to an expanded polyglutamine stretch in the amino terminus of the Huntingtin protein. Caspase-3 (a group II effector caspase) liberates the amino-terminal fragment, containing the polyglutamine expansion, by cleavage within a cluster of DXXD sites. These truncated fragments then appear to aggregate within the cell and recruit the proenzyme of caspase-8 (a group III activator caspase) to the polyglutamine aggregates. The ensuing recruitment-activation of caspase-8 presumably initiates an apoptotic cascade similar to the CD95 (Fas, APO-1) pathway, including further activation of caspase-3. This cycle may begin at a low level that is below the threshold necessary for cell death; however, with the cumulative cycle of polyglutamine fragment generation and caspase activation, a point may be reached in vulnerable neurons where this threshold is breached and the cells die prematurely. In Alzheimer's disease, caspase-3 adulterates the normal processing of the amyloid-β precursor protein (APP) by removal of the carboxy-terminal cytosolic domain. The resulting truncated APP, now deprived of key re-internalization signals, appears to be shunted to a degradative pathway that results in the generation of the cytotoxic amyloid-β peptide (Aβ) as one of the peptide derivatives of the full length APP polypeptide. Although the mechanism by which Aβ mediates its cytotoxicity is not understood, it leads to an increased propensity for apoptosis, including caspase-3 activation. Increased caspase-3 activity (again below the apoptotic threshold) may result in accelerated formation of Aβ and further neuronal stress, eventually resulting in exacerbation of the cycle to a point which leads to cell death and neurodegeneration. In both of these examples, a vicious cycle appears to exist, although how the cycle is initiated or emerges is unclear. Vulnerability of the target polypeptides to caspases may arise as a consequence of genetic mutations (e.g. polyglutamine expansion in the case of Huntington's; APP or presenilin mutations in the case of Alzheimer's). Alternatively, the seed of caspase activity may arise as a consequence of other trauma or from the endogenous low-level activity of the caspase proenzymes themselves.

Prospects for caspase-directed therapeutics

Inappropriate apoptosis clearly underlies the etiology of several human diseases. The control of caspases, as a key and central component of the biochemical pathway that mediates apoptotic cell death, is an attractive first step in modulating this process. Caspase activation for the treatment of disorders where insufficient apoptosis occurs (e.g. cancer)

Single cleavage site (e.g. Gelsolin) (731aa)
![Diagram](attachment:diagram_1.png)

Nested multiple sites (e.g. PARP) (1014aa)
![Diagram](attachment:diagram_2.png)

Redundant clustered sites (e.g. Huntingtin) (3144aa)
![Diagram](attachment:diagram_3.png)

Distal multiple sites (e.g. DFF45/ICAD) (331aa)
![Diagram](attachment:diagram_4.png)

Figure 11 Caspase cleavage strategies. The majority of substrates that are proteolyzed by caspases during apoptosis are cleaved at a single site, although some polypeptide substrates contain multiple sites that are either nested, redundant within a short stretch, or spread out across the molecule.

represents a substantial challenge. 'Trojan horse' gene therapy approaches may be viable, such as that described for HIV infection in which a TAT-caspase-3 construct containing a HIV-protease recognition motif selectively induces apoptosis in HIV-infected cells only. Alternatively, the molecular constraints on caspase activation is poorly

A

Q(>35)

(5x) DXXD

Active Caspase-3

p17

Dormant Caspase-3 Proenzyme

pro p17

Vicious Cycle

Active Caspase-8

Dormant Caspase-8 Proenzyme

DED p12

DED p12

B

Amyloid Precursor Protein

AB

Casapse-3 Cleavage

AB Peptide Formation

Dormant Caspase-3 Proenzyme

pro p17

p17

ACCELERATED APOPTOSIS OF VULNERABLE NEURONS

Figure 12 Potential involvement of caspases in 'vicious cycles' leading to pathogenic exacerbation in Huntington's disease (A) and Alzheimer's disease (B)

understood and my harbour opportunities for selective caspase activation once they are resolved. On the other hand, caspase inhibition for the treatment of disorders where excessive apoptosis occurs (e.g. neurodegeneration) appears to be more amenable to therapeutic intervention with classical small-molecule inhibitors. Preliminary experiments in animal models using non-selective caspase inhibitors such as Z-VAD(OMe)-CH₂F, for example, have shown *in vivo* efficacy in ischemic and hypoxic brain injury, traumatic and excitotoxic brain damage, neuronal transplantation, acute bacterial meningitis as well as in cardiac and kidney ischemia/ reperfusion injury and models of acute liver failure.⁵⁵⁻⁶⁵ In addition, transgenic mice expressing dominant-negative caspase-1 show resistance to CNS injury in models of ALS, focal ischemia, excitotoxic injury and Huntington's disease.⁶⁶⁻⁷⁰ (It is not yet clear whether this protection is afforded by the attenuation of a secondary inflammatory response, or whether the dominant-negative caspase-1 affects other caspase family members as well.) The clinical utility of caspase inhibitors will depend on several key issues that require further resolution. For example, will cells that are saved from apoptotic death remain functional and survive without perpetual caspase inhibition? Will the delayed- administration ‘window of opportunity’ be sufficient for practical use in a clinical setting? Can highly selective caspase-3 inhibitors be used for chronic administration without unacceptable adverse events occurring? Early work in all of these areas is highly encouraging.

### References

1. Thornberry NA, Bull HG, Calaycay JR, Chapman KT, Howard AD, Kostura MJ, Miller DK, Molineaux SM, Weidner JR and Aunins J *et al.* (1992) A novel heterodimeric cysteine protease is required for interleukin-1 beta processing in monocytes. Nature 356: 768–774
2. Cerretti DP, Kozlosky CJ, Mosley B, Nelson N, Van Ness K, Greenstreet TA, March CJ, Kronheim SR, Druck T and Cannizzaro LA *et al.* (1992) Molecular cloning of the interleukin-1 beta converting enzyme. Science 256: 97–100
3. Ellis RE, Yuan JY and Horvitz HR (1991) Mechanisms and functions of cell death. Annu. Rev. Cell. Biol. 7: 663–698
4. Xue D, Shaham S and Horvitz HR (1996) The Caenorhabditis elegans cell-death protein CED-3 is a cysteine protease with substrate specificities similar to those of the human CPP32 protease. Genes. Dev. 10: 1073–1083
5. Yuan J, Shaham S, Ledoux S, Ellis HM and Horvitz HR (1993) The C. elegans cell death gene ced-3 encodes a protein similar to mammalian interleukin-1 beta-converting enzyme. Cell 75: 641–652
6. Horvitz HR (1999) Genetic control of programmed cell death in the nematode Caenorhabditis elegans. Cancer Res. 59: 1701s–1706s
7. Darmon AJ, Nicholson DW and Bleackley RC (1995) Activation of the apoptotic protease CPP32 by cytotoxic T-cell-derived granzyme B. Nature 377: 446–448
8. Martin SJ, Amarante-Mendes GP, Shi L, Chuang TH, Casiano CA, O’Brien GA, Fitzgerald P, Tan EM, Bokoch GM, Greenberg AH and Green DR (1996) The cytotoxic cell protease granzyme B initiates apoptosis in a cell-free system by proteolytic processing and activation of the ICE/CED-3 family protease, CPP32, via a novel two-step mechanism. EMBO J. 15: 2407–2416
9. Andrade F, Roy S, Nicholson D, Thornberry N, Rosen A and Casciola-Rosen L (1998) Granzyme B directly and efficiently cleaves several downstream caspase substrates: implications for CTL-induced apoptosis. Immunity 8: 451–460
10. Squier MK, Sehnert AJ, Sellins KS, Malkinson AM, Takano E and Cohen JJ (1999) Calpain and calpastatin regulate neutrophil apoptosis. J. Cell. Physiol. 178: 311–319

11. Alnemri ES, Livingston DJ, Nicholson DW, Salvesen G, Thornberry NA, Wong WW and Yuan J (1996) Human ICE/CED-3 protease nomenclature [letter]. Cell 87: 171
12. Thornberry NA, Rano TA, Peterson EP, Raspet DM, Timkey T, Garcia-Calvo M, Houtzager VM, Nordstrom PA, Roy S, Vaillancourt JP, Chapman KT and Nicholson DW (1997) A combinatorial approach defines specificities of members of the caspase family and granzyme B. Functional relationships established for key mediators of apoptosis. J. Biol. Chem. 272: 17907–17911
13. Rano TA, Timkey T, Peterson EP, Rotonda J, Nicholson DW, Becker JW, Chapman KT and Thornberry NA (1997) A combinatorial approach for determining protease specificities: application to interleukin-1beta converting enzyme (ICE). Chem. Biol. 4: 149–155
14. Muzio M, Chinnaiyan AM, Kischkel FC, O’Rourke K, Shevchenko A, Ni J, Scaffidi C, Bretz JD, Zhang M, Gentz R, Mann M, Krammer PH, Peter ME and Dixit VM (1996) FLICE, a novel FADD-homologous ICE/CED-3-like protease, is recruited to the CD95 (Fas/APO-1) death-inducing signaling complex. Cell 85: 817–827
15. Boldin MP, Goncharov TM, Goltsev YV and Wallach D (1996) Involvement of MACH, a novel MORT1/FADD-interacting protease, in Fas/APO-1- and TNF receptor-induced cell death. Cell 85: 803–815
16. Li P, Nijhawan D, Budihardjo I, Srinivasula SM, Ahmad M, Alnemri ES and Wang X (1997) Cytochrome c and dATP-dependent formation of Apaf-1/caspase-9 complex initiates an apoptotic protease cascade. Cell 91: 479–489
17. Varfolomeev EE, Schuchmann M, Luria V, Chiannilkulchai N, Beckmann JS, Mett IL, Rebrikov D, Brodianski VM, Kemper OC, Kollet O, Lapidot T, Soffer D, Sobe T, Avraham KB, Goncharov T, Holtmann H, Lonai P and Wallach D (1998) Targeted disruption of the mouse Caspase 8 gene ablates cell death induction by the TNF receptors, Fas/Apo1, and DR3 and is lethal prenatally. Immunity 9: 267–276
18. Juo P, Kuo CJ, Yuan J and Blenis J (1998) Essential requirement for caspase-8/FLICE in the initiation of the Fas-induced apoptotic cascade. Curr. Biol. 8: 1001–1008
19. Hakem R, Hakem A, Duncan GS, Henderson JT, Woo M, Soengas MS, Elia A, de la Pompa JL, Kagi D, Khoo W, Potter J, Yoshida R, Kaufman SA, Lowe SW, Penninger JM and Mak TW (1998) Differential requirement for caspase 9 in apoptotic pathways in vivo. Cell 94: 339–352
20. Kuida K, Haydar TF, Kuan CY, Gu Y, Taya C, Karasuyama H, Su MS, Rakic P and Flavell RA (1998) Reduced apoptosis and cytochrome c-mediated caspase activation in mice lacking caspase 9. Cell 94: 325–337
21. Lazebnik YA, Takahashi A, Moir RD, Goldman RD, Poirier GG, Kaufmann SH and Earnshaw WC (1995) Studies of the lamin proteinase reveal multiple parallel biochemical pathways during apoptotic execution. Proc. Natl. Acad. Sci. USA 92: 9042–9046
22. Duan H and Dixit VM (1997) RAIDD is a new ‘death’ adaptor molecule. Nature 385: 86–89
23. Ahmad M, Srinivasula SM, Wang L, Talanian RV, Litwack G, Fernandes-Alnemri T and Alnemri ES (1997) CRADD, a novel human apoptotic adaptor molecule for caspase-2, and FasL/tumor necrosis factor receptor-interacting protein RIP. Cancer Res. 57: 615–619
24. Wilson KP, Black JA, Thomson JA, Kim EE, Griffith JP, Navia MA, Murcko MA, Chambers SP, Aldape RA, Raybuck SA *et al.* (1994) Structure and mechanism of interleukin-1 beta converting enzyme [see comments]. Nature 370: 270–275
25. Walker NP, Talanian RV, Brady KD, Dang LC, Bump NJ, Ferenz CR, Franklin S, Ghayur T, Hackett MC, Hammill LD *et al.* (1994) Crystal structure of the cysteine protease interleukin-1 beta-converting enzyme: a (p20/p10) 2 homodimer. Cell 78: 343–352
26. Rotonda J, Nicholson DW, Fazil KM, Gallant M, Gareau Y, Labelle M, Peterson EP, Raspet DM, Ruel R, Vaillancourt JP, Thornberry NA and Becker JW (1996) The three-dimensional structure of apopain/CPP32, a key mediator of apoptosis. Nat. Struct. Biol. 3: 619–625
27. Mittl PR, Di Marco S, Krebs JF, Bai X, Karanewsky DS, Priestle JP, Tomaselli KJ and Grutter MG (1997) Structure of recombinant human CPP32 in complex with the tetrapeptide acetyl-Asp-Val-Ala-Asp fluoromethyl ketone. J. Biol. Chem. 272: 6539–6547
28. Garcia-Calvo M, Peterson EP, Leiting B, Ruel R, Nicholson DW and Thornberry NA (1998) Inhibition of human caspases by peptide-based and macromolecular inhibitors. J. Biol. Chem. 273, 32608–32613
29. Schotte P, Declercq W, Van Huffel S, Vandenberghe P and Beyaert R (1999) Non-specific effects of methyl ketone peptide inhibitors of caspases. FEBS Lett. 442: 117–121

30. Bump NJ, Hackett M, Hugunin M, Seshagiri S, Brady K, Chen P, Ferenz C, Franklin S, Ghayur T, Li P *et al.* (1995) Inhibition of ICE family proteases by baculovirus antiapoptotic protein p35. Science 269: 1885–1888

31. Xue D and Horvitz HR (1995) Inhibition of the Caenorhabditis elegans cell-death protease CED-3 by a CED-3 cleavage site in baculovirus p35 protein. Nature 377: 248–251

32. Fisher AJ, Cruz W, Zoog SJ, Schneider CL and Friesen PD (1999) Crystal structure of baculovirus P35: role of a novel reactive site loop in apoptotic caspase inhibition. EMBO J. 18: 2031–2039

33. Tamm I, Wang Y, Sausville E, Scudiero DA, Vigna N, Oltersdorf T and Reed JC (1998) IAP-family protein survivin inhibits caspase activity and apoptosis induced by Fas (CD95), Bax, caspases, and anticancer drugs. Cancer Res. 58: 5315–5320

34. Deveraux QL, Roy N, Stennicke HR, Van Arsdale T, Zhou Q, Srinivasula SM, Alnemri ES, Salvesen GS and Reed JC (1998) IAPs block apoptotic events induced by caspase-8 and cytochrome c by direct inhibition of distinct caspases. EMBO J. 17: 2215–2223

35. Roy N, Deveraux QL, Takahashi R, Salvesen GS and Reed JC (1997) The c-IAP-1 and c-IAP-2 proteins are direct inhibitors of specific caspases. EMBO J. 16: 6914–6925

36. Deveraux QL, Takahashi R, Salvesen GS and Reed JC (1997) X-linked IAP is a direct inhibitor of cell-death proteases. Nature 388: 300–304

37. Thome M, Hofmann K, Burns K, Martinon F, Bodmer JL, Mattmann C and Tschopp J (1998) Identification of CARDIAK, a RIP-like kinase that associates with caspase-1. Curr. Biol. 8: 885–888

38. Yang X, Chang HY and Baltimore D (1998) Autoproteolytic activation of procaspases by oligomerization. Mol. Cell. 1: 319–325

39. Srinivasula SM, Ahmad M, Fernandes-Alnemri T and Alnemri ES (1998) Autoactivation of procaspase-9 by Apaf-1-mediated oligomerization. Mol. Cell. 1: 949–957

40. Muzio M, Stockwell BR, Stennicke HR, Salvesen GS and Dixit VM (1998) An induced proximity model for caspase-8 activation. J. Biol. Chem. 273: 2926–2930

41. Martin DA, Siegel RM, Zheng L and Lenardo MJ (1998) Membrane oligomerization and cleavage activates the caspase-8 (FLICE/MACHalpha1) death signal. J. Biol. Chem. 273: 4345–4349

42. Xanthoudakis S, Roy S, Rasper D, Hennessey T, Aubin Y, Cassady R, Tawa P, Ruel R, Rosen A and Nicholson DW (1999) Hsp60 accelerates the maturation of pro-caspase-3 by upstream activator proteases during apoptosis. EMBO J. 18: 2049–2056

43. Buckley CD, Pilling D, Henriquez NV, Parsonage G, Threlfall K, Scheel-Toellner D, Simmons DL, Akbar AN, Lord JM and Salmon M (1999) RGD peptides induce apoptosis by direct caspase-3 activation [see comments]. Nature 397: 534–539

44. Brockstedt E, Rickers A, Kostka S, Laubersheimer A, Dorken B, Wittmann-Liebold B, Bommert K and Otto A (1998) Identification of apoptosis-associated proteins in a human Burkitt lymphoma cell line. Cleavage of heterogeneous nuclear ribonucleoprotein A1 by caspase 3 [published erratum appears in J Biol Chem 1998 Dec 11;273(50):33884]. J. Biol. Chem. 273: 28057–28064

45. Goldberg YP, Nicholson DW, Rasper DM, Kalchman MA, Koide HB, Graham RK, Bromm M, Kazemi-Esfarjani P, Thornberry NA, Vaillancourt JP and Hayden MR (1996) Cleavage of huntingtin by apopain, a proapoptotic cysteine protease, is modulated by the polyglutamine tract [see comments]. Nat. Genet. 13: 442–449

46. Sanchez I, Xu CJ, Juo P, Kakizaka A, Blenis J and Yuan J (1999) Caspase-8 is required for cell death induced by expanded polyglutamine repeats [see comments]. Neuron 22: 623–633

47. Gervais FG, Xu D, Robertson GS, Vaillancourt JP, Zhu Y, Huang J, LeBlanc A, Smith D, Rigby M, Shearman MS, Clarke EE, Zheng H, Van Der Ploeg LH, Ruffolo SC, Thornberry NA, Xanthoudakis S, Zamboni RJ, Roy S and Nicholson DW (1999) Involvement of caspases in proteolytic cleavage of Alzheimer’s amyloid-beta precursor protein and amyloidogenic A beta peptide formation. Cell 97: 395–406

48. Weidemann A, Paliga KU, Dr Reinhard FB, Schuckert O, Evin G and Masters CL (1999) Proteolytic processing of the Alzheimer’s disease amyloid precursor protein within its cytoplasmic domain by caspase-like proteases. J. Biol. Chem. 274, 5823–5829

49. Barnes NY, Li L, Yoshikawa K, Schwartz LM, Oppenheim RW and Milligan CE (1998) Increased production of amyloid precursor protein provides a substrate for caspase-3 in dying motoneurons. J. Neurosci. 18: 5869–5880

50. Hetts SW (1998) To die or not to die: an overview of apoptosis and its role in disease [see comments]. JAMA 279: 300–307

51. Thompson CB (1995) Apoptosis in the pathogenesis and treatment of disease. Science 267: 1456–1462

52. Uren AG and Vaux DL (1996) Molecular and clinical aspects of apoptosis. Pharmacol. Ther. 72: 37–50

53. Barr PJ and Tomei LD (1994) Apoptosis and its role in human disease. Biotechnology (NY) 12: 487–493

54. Vocero-Akbani AM, Heyden NV, Lissy NA, Ratner L and Dowdy SF (1999) Killing HIV-infected cells by transduction with an HIV protease-activated caspase-3 protein [see comments]. Nat. Med. 5: 29–33

55. Loddick SA, MacKenzie A and Rothwell NJ (1996) An ICE inhibitor, z-VAD-DCB attenuates ischaemic brain damage in the rat. Neuroreport 7: 1465–1468

56. Yakovlev AG, Knoblauch SM, Fan L, Fox GB, Goodnight R and Faden AI (1997) Activation of CPP32-like caspases contributes to neuronal apoptosis and neurological dysfunction after traumatic brain injury. J. Neurosci. 17: 7415–7424

57. Cheng Y, Deshmukh M, D’Costa A, Demaro JA, Gidday JM, Shah A, Sun Y, Jacquin MF, Johnson EM and Holtzman DM (1998) Caspase inhibitor affords neuroprotection with delayed administration in a rat model of neonatal hypoxic-ischemic brain injury [see comments]. J. Clin. Invest. 101: 1992–1999

58. Rodriguez I, Matsuura K, Ody C, Nagata S and Vassalli P (1996) Systemic injection of a tripeptide inhibits the intracellular activation of CPP32-like proteases in vivo and fully protects mice against Fas-mediated fulminant liver destruction and death. J. Exp. Med. 184: 2067–2072

59. Fink K, Zhu J, Namura S, Shimizu-Sasamata M, Endres M, Ma J, Dalkara T, Yuan J and Moskowitz MA (1998) Prolonged therapeutic window for ischemic brain damage caused by delayed caspase activation. J. Cereb. Blood Flow Metab. 18: 1071–1076

60. Endres M, Namura S, Shimizu-Sasamata M, Waeber C, Zhang L, Gomez-Isla T, Hyman BT and Moskowitz MA (1998) Attenuation of delayed neuronal death after mild focal ischemia in mice by inhibition of the caspase family. J. Cereb. Blood Flow Metab. 18: 238–247

61. Schierle GS, Hansson O, Leist M, Nicotera P, Widner H and Brundin P (1999) Caspase inhibition reduces apoptosis and increases survival of nigral transplants. Nat. Med. 5: 97–100

62. Braun JS, Novak R, Herzog KH, Bodner SM, Cleveland JL and Tuomanen EI (1999) Neuroprotection by a caspase inhibitor in acute bacterial meningitis. Nat. Med. 5: 298–302

63. Holly TA, Drincic A, Byun Y, Nakamura S, Harris K, Klocke FJ and Cryns VL (1999) Caspase inhibition reduces myocyte cell death induced by myocardial ischemia and reperfusion in vivo. J. Mol. Cell. Cardiol. 31: 1709–1715

64. Black SC, Huang JQ, Rezaiefar P, Radinovic S, Eberhart A, Nicholson DW and Rodger IW (1998) Co-localization of the cysteine protease caspase-3 with apoptotic myocytes after in vivo myocardial ischemia and reperfusion in the rat. J. Mol. Cell Cardiol. 30: 733–742

65. Daemen MA, van’t Veer C, Denecker G, Heemskerk VH, Wolfs TG, Clauss M, Vandenberghe P and Buurman WA (1999) Inhibition of apoptosis induced by ischemia-reperfusion prevents inflammation. J. Clin. Invest. 104: 541–549

66. Ona VO, Li M, Vonsattel JP, Andrews LJ, Khan SQ, Chung WM, Frey AS, Menon AS, Li XJ, Stieg PE, Yuan J, Penney JB, Young AB, Cha JH and Friedlander RM (1999) Inhibition of caspase-1 slows disease progression in a mouse model of Huntington’s disease [see comments]. Nature 399: 263–267

67. Friedlander RM, Brown RH, Gagliardi V, Wang J and Yuan J (1997) Inhibition of ICE slows ALS in mice [letter] [published erratum appears in Nature 1998 Apr 9;392(6676):560]. Nature 388: 31

68. Hara H, Fink K, Endres M, Friedlander RM, Gagliardi V, Yuan J and Moskowitz MA (1997) Attenuation of transient focal cerebral ischemic injury in transgenic mice expressing a mutant ICE inhibitory protein. J. Cereb. Blood Flow Metab. 17: 370–375

69. Friedlander RM, Gagliardi V, Hara H, Fink KB, Li W, MacDonald G, Fishman MC, Greenberg AH, Moskowitz MA and Yuan J (1997) Expression of a dominant negative mutant of interleukin-1 beta converting enzyme in transgenic mice prevents neuronal cell death induced by trophic factor withdrawal and ischemic brain injury. J. Exp. Med. 185: 933–940

70. Klevenyi P, Andreassen O, Ferrante RJ, Schleicher Jr JR, Friedlander RM and Beal MF (1999) Transgenic mice expressing a dominant negative mutant interleukin-1beta converting enzyme show resistance to MPTP neurotoxicity. Neuroreport 10: 635–638

71. Rasper DM, Vaillancourt JP, Hadano S, Houtzager VM, Seiden I, Keen SL, Tawa P, Xanthoudakis S, Nasir J, Martindale D, Koop BF, Peterson EP, Thornberry NA, Huang J, MacPherson DP, Black SC, Hornung F, Lenardo MJ, Hayden MR, Roy S and Nicholson DW (1998) Cell death attenuation by 'Usurpin', a mammalian DED-caspase homologue that precludes caspase-8 recruitment and activation by the CD-95 (Fas, APO-1) receptor complex. Cell Death Differ. 5: 271–288

72. Nasir J, Theilmann JL, Vaillancourt JP, Munday NA, Ali A, Scherer S, Beatty B, Nicholson DW and Hayden MR (1997) Interleukin-1beta-converting enzyme (ICE) and related cell death genes ICErel-II and ICErel-III map to the same PAC clone at band 11q22.2–22.3. Mamm. Genome 8: 611–613

73. Talanian RV, Quinlan C, Trautz S, Hackett MC, Mankovich JA, Banach D, Ghayur T, Brady KD and Wong WW (1997) Substrate specificities of caspase family proteases. J. Biol. Chem. 272: 9677–9682

74. Lazebnik YA, Kaufmann SH, Desnoyers S, Poirier GG and Earnshaw WC (1994) Cleavage of poly(ADP-ribose) polymerase by a proteinase with properties like ICE. Nature 371: 346–347

75. Kaufmann SH, Desnoyers S, Ottaviano Y, Davidson NE and Poirier GG (1993) Specific proteolytic cleavage of poly(ADP-ribose) polymerase: an early marker of chemotherapy-induced apoptosis. Cancer Res. 53: 3976–3985

76. McConnell KR, Dynan WS and Hardin JA (1997) The DNA-dependent protein kinase catalytic subunit (p460) is cleaved during Fas-mediated apoptosis in Jurkat cells. J. Immunol. 158: 2083–2089

77. Le Romancer M, Cosulich SC, Jackson SP and Clarke PR (1996) Cleavage and inactivation of DNA-dependent protein kinase catalytic subunit during apoptosis in Xenopus egg extracts. J. Cell. Sci. 109: 3121–3127

78. Han Z, Malik N, Carter T, Reeves WH, Wyche JH and Hendrickson EA (1996) DNA-dependent protein kinase is a target for a CPP32-like apoptotic protease. J. Biol. Chem. 271: 25035–25040

79. Teraoka H, Yumoto Y, Watanabe F, Tsukada K, Suwa A, Enari M and Nagata S (1996) CPP32/Yama/appain cleaves the catalytic component of DNA-dependent protein kinase in the holoenzyme. FEBS Lett. 393: 1–6

80. Song Q, Lees-Miller SP, Kumar S, Zhang Z, Chan DW, Smith GC, Jackson SP, Alnemri ES, Litwack G, Khanna KK and Lavin MF (1996) DNA-dependent protein kinase catalytic subunit: a target for an ICE-like protease in apoptosis. EMBO J. 15: 3238–3246

81. Casciola-Rosen L, Nicholson DW, Chong T, Rowan KR, Thornberry NA, Miller DK and Rosen A (1996) Apopain/CPP32 cleaves proteins that are essential for cellular repair: a fundamental principle of apoptotic death [see comments]. J. Exp. Med. 183: 1957–1964

82. Casciola-Rosen LA, Anhalt GJ and Rosen A (1995) DNA-dependent protein kinase is one of a subset of autoantigens specifically cleaved early during apoptosis. J. Exp. Med. 182: 1625–1634

83. Huang Y, Nakada S, Ishiko T, Utsugisawa T, Datta R, Kharbanda S, Yoshida K, Talanian RV, Weichselbaum R, Kufe D and Yuan ZM (1999) Role for caspase-mediated cleavage of Rad51 in induction of apoptosis by DNA damage. Mol. Cell. Biol. 19: 2986–2997

84. Flygare J, Armstrong RC, Wennborg A, Orsan S and Hellgren D (1998) Proteolytic cleavage of HsRad51 during apoptosis. FEBS Lett. 427: 247–251

85. Sahara S, Aoto M, Eguchi Y, Imamoto N, Yoneda Y. & Tsujimoto Y (1999) Acinus is a caspase-3-activated protein required for apoptotic chromatin condensation [In process Citation]. Nature 401: 168–173

86. Tang D and Kidd VJ (1998) Cleavage of DFF-45/ICAD by multiple caspases is essential for its function during apoptosis. J. Biol. Chem. 273: 28549–28552

87. Liu X, Zou H, Slaughter C and Wang X (1997) DFF, a heterodimeric protein that functions downstream of caspase-3 to trigger DNA fragmentation during apoptosis. Cell 89: 175–184

88. Sakahira H, Enari M and Nagata S (1998) Cleavage of CAD inhibitor in CAD activation and DNA degradation during apoptosis [see comments]. Nature 391: 96–99

89. Enari M, Sakahira H, Yokoyama H, Okawa K, Iwamatsu A and Nagata S (1998) A caspase-activated DNase that degrades DNA during apoptosis, and its inhibitor ICAD [see comments] [published erratum appears in Nature 1998 May 28;393(6683):396]. Nature 391: 43–50

90. Rheaume E, Cohen LY, Uhlmann F, Lazure C, Alam A, Hurwitz J, Sekaly RP and Denis F (1997) The large subunit of replication factor C is a substrate for caspase-3 in vitro and is cleaved by a caspase-3-like protease during Fas-mediated apoptosis. EMBO J. 16: 6346–6354

91. Ubeda M and Habener JF (1997) The large subunit of the DNA replication complex C (DSEB/RF-C140) cleaved and inactivated by caspase-3 (CPP32/YAMA) during Fas-induced apoptosis. J. Biol. Chem. 272: 19562–19568

92. Song Q, Lu H, Zhang N, Luckow B, Shah G, Poirier G and Lavin M (1997) Specific cleavage of the large subunit of replication factor C in apoptosis is mediated by CPP32-like protease. Biochem. Biophys. Res. Commun. 233: 343–348

93. Browne SJ, MacFarlane M, Cohen GM and Paraskeva C (1998) The adenomatous polyposis coli protein and retinoblastoma protein are cleaved early in apoptosis and are potential substrates for caspases. Cell Death Differ. 5: 206–213

94. Chen WD, Otterson GA, Lipkowitz S, Khleif SN, Coxon AB and Kaye FJ (1997) Apoptosis is associated with cleavage of a 5 kDa fragment from RB which mimics dephosphorylation and modulates E2F binding. Oncogene 14: 1243–1248

95. Janicke RU, Walker PA, Lin XY and Porter AG (1996) Specific cleavage of the retinoblastoma protein by an ICE-like protease in apoptosis. EMBO J. 15: 6969–6978

96. An B and Dou QP (1996) Cleavage of retinoblastoma protein during apoptosis: an interleukin 1 beta-converting enzyme-like protease as candidate. Cancer Res. 56: 438–442

97. Tan X, Martin SJ, Green DR and Wang JYJ (1997) Degradation of retinoblastoma protein in tumor necrosis factor- and CD95-induced cell death. J. Biol. Chem. 272: 9613–9616

98. Chen L, Marechal V, Moreau J, Levine AJ and Chen J (1997) Proteolytic cleavage of the mdm2 oncoprotein during apoptosis. J. Biol. Chem. 272: 22966–22973

99. Erhardt P, Tomaselli KJ and Cooper GM (1997) Identification of the MDM2 oncoprotein as a substrate for CPP32-like apoptotic proteases. J. Biol. Chem. 272: 15049–15052

100. Zhang Y, Fujita N and Tsuruo T (1999) Caspase-mediated cleavage of p21Waf1/Cip1 converts cancer cells from growth arrest to undergoing apoptosis. Oncogene 18: 1131–1138

101. Gervais JL, Seth P and Zhang H (1998) Cleavage of CDK inhibitor p21(Cip1/Waf1) by caspases is an early event during DNA damage-induced apoptosis. J. Biol. Chem. 273: 19207–19212

102. Levkau B, Koyama H, Raines EW, Clurman BE, Herren B, Orth K, Roberts JM and Ross R (1998) Cleavage of p21Cip1/Waf1 and p27Kip1 mediates apoptosis in endothelial cells through activation of Cdk2: role of a caspase cascade. Mol. Cell. 1: 553–563

103. Park JA, Kim KW, Kim SI and Lee SK (1998) Caspase 3 specifically cleaves p21WAF1/CIP1 in the earlier stage of apoptosis in SK-HEP-1 human hepatoma cells. Eur. J. Biochem. 257: 242–248

104. Gueth-Hallonet C, Weber K and Osborn M (1997) Cleavage of the nuclear matrix protein NuMA during apoptosis. Exp. Cell. Res. 233: 21–24

105. Casiano CA, Martin SJ, Green DR and Tan EM (1996) Selective cleavage of nuclear autoantigens during CD95 (Fas/APO-1)-mediated T cell apoptosis. J. Exp. Med. 184: 765–770

106. Hsu HL and Yeh NH (1996) Dynamic changes of NuMA during the cell cycle and possible appearance of a truncated form of NuMA during apoptosis. J. Cell. Sci. 109: 277–288

107. Smith GC, di Fagagna F, Lakin ND and Jackson SP (1999) Cleavage and inactivation of ATM during apoptosis [In Process Citation]. Mol. Cell. Biol. 19: 6076–6084

108. Tewari M, Beidler DR and Dixit VM (1995) CrmA-inhibitable cleavage of the 70-kDa protein component of the U1 small nuclear ribonucleoprotein during Fas- and tumor necrosis factor-induced apoptosis. J. Biol. Chem. 270: 18738–18741

109. Casciola-Rosen LA, Miller DK, Anhalt GJ and Rosen A (1994) Specific cleavage of the 70-kDa protein component of the U1 small nuclear ribonucleoprotein is a characteristic biochemical feature of apoptotic cell death. J. Biol. Chem. 269: 30757–30760

110. Waterhouse N, Kumar S, Song Q, Strike P, Sparrow L, Dreyfuss G, Alnemri ES, Litwack G, Lavin M and Watters D (1996) Heteronuclear ribonucleoproteins C1 and C2, components of the spliceosome, are specific targets of interleukin 1beta-converting enzyme-like proteases in apoptosis. J. Biol. Chem. 271: 29335–29341

111. Wang X, Zelenski NG, Yang J, Sakai J, Brown MS and Goldstein JL (1996) Cleavage of sterol regulatory element binding proteins (SREBPs) by CPP 32 during apoptosis. EMBO J. 15: 1012–1020
112. Wang X, Pai JT, Wiedenfeld EA, Medina JC, Slaughter CA, Goldstein JL and Brown MS (1995) Purification of an interleukin-1 beta converting enzyme-related cysteine protease that cleaves sterol regulatory element-binding proteins between the leucine zipper and transmembrane domains. J. Biol. Chem. 270: 18044–18050
113. Barrett M, Xue D, Horvitz HR and Gilmore TD (1997) Phosphorylation of IkappaB-alpha inhibits its cleavage by caspase CPP 32 in vitro. J. Biol. Chem. 272: 29419–29422
114. Krieser RJ and Eastman A (1999) Cleavage and nuclear translocation of the caspase 3 substrate rho GDP-dissociation inhibitor, D4-GDI, during apoptosis [In Process Citation]. Cell Death Differ. 6: 412–419
115. Na S, Chuang TH, Cunningham A, Turi TG, Hanke JH, Bokoch GM and Danley DE (1996) D4-GDI, a substrate of CPP 32, is proteolyzed during Fas-induced apoptosis. J. Biol. Chem. 271: 11209–11213
116. Adam-Klages S, Schwandner R, Luschen S, Ussat S, Kreder D and Kronke M (1998) Caspase-mediated inhibition of human cytosolic phospholipase A2 during apoptosis. J. Immunol. 161: 5687–5694
117. Atsumi G, Tajima M, Hadano A, Nakatani Y, Murakami Mand Kudo I (1998) Fas-induced arachidonic acid release is mediated by Ca2+-independent phospholipase A2 but not cytosolic phospholipase A2, which undergoes proteolytic inactivation. J. Biol. Chem. 273: 13870–13877
118. Wissing D, Mouritzen H, Egeblad M, Poirier GG and Jaattela M (1997) Involvement of caspase-dependent activation of cytosolic phospholipase A2 in tumor necrosis factor-induced apoptosis. Proc. Natl. Acad. Sci. USA 94: 5073–5077
119. Khwaja A and Tatton L (1999) Caspase-mediated proteolysis and activation of protein kinase Cdelta plays a central role in neutrophil apoptosis. Blood 94: 291–301
120. Frutos S, Moscat J and Diaz-Meco MT (1999) Cleavage of zetaPKC but not lambda/iotaPKC by caspase-3 during UV-induced apoptosis. J. Biol. Chem. 274: 10765–10770
121. Denning MF, Wang Y, Nickoloff BJ and Wrone-Smith T (1998) Protein kinase Cdelta is activated by caspase-dependent proteolysis during ultraviolet radiation-induced apoptosis of human keratinocytes. J. Biol. Chem. 273: 29995–30002
122. Mizuno K, Noda K, Araki T, Imaoka T, Kobayashi Y, Akita Y, Shimonaka M, Kishi S and Ohno S (1997) The proteolytic cleavage of protein kinase C isoforms, which generates kinase and regulatory fragments, correlates with Fas-mediated and 12-O-tetradecanoyl-phorbol-13-acetate-induced apoptosis. Eur. J. Biochem. 250: 7–18
123. Datta R, Kojima H, Yoshida K and Kufe D (1997) Caspase-3-mediated cleavage of protein kinase C theta in induction of apoptosis. J. Biol. Chem. 272: 20317–20320
124. Ghayur T, Hugunin M, Talanian RV, Ratnofsky S, Quinlan C, Emoto Y, Pandey P, Datta R, Huang Y, Kharbanda S, Allen H, Kamen R, Wong W and Kufe D (1996) Proteolytic activation of protein kinase C delta by an ICE/CED 3-like protease induces characteristics of apoptosis. J. Exp. Med. 184: 2399–2404
125. Datta R, Banach D, Kojima H, Talanian RV, Alnemri ES, Wong WW and Kufe DW (1996) Activation of the CPP 32 protease in apoptosis induced by 1-beta-D-arabinofuranosylcytosine and other DNA-damaging agents. Blood 88: 1936–1943
126. Emoto Y, Manome Y, Meinhardt G, Kisaki H, Kharbanda S, Robertson M, Ghayur T, Wong WW, Kamen R, Weichselbaum R et al. (1995) Proteolytic activation of protein kinase C delta by an ICE-like protease in apoptotic cells. EMBO J. 14: 6148–6156
127. Widmann C, Gerwins P, Johnson NL, Jarpe MB and Johnson GL (1998) MEK kinase 1, a substrate for DEVD-directed caspases, is involved in genotoxin-induced apoptosis. Mol. Cell. Biol. 18: 2416–2429
128. Cardone MH, Salvesen GS, Widmann C, Johnson G and Frisch SM (1997) The regulation of anoikis: MEKK-1 activation requires cleavage by caspases. Cell 90: 315–323
129. Kakeya H, Onose R and Osada H (1998) Caspase-mediated activation of a 36-kDa myelin basic protein kinase during anticancer drug-induced apoptosis. Cancer Res. 58: 4888–4894
130. Lee KK, Murakawa M, Nishida E, Tsubuki S, Kawashima S, Sakamaki K and Yonehara S (1998) Proteolytic activation of MST/Krs, STE 20-related protein kinase, by caspase during apoptosis. Oncogene 16: 3029–3037
131. Graves JD, Gotoh Y, Draves KE, Ambrose D, Han DK, Wright M, Chernoff J, Clark EA and Krebs EG (1998) Caspase-mediated activation and induction of apoptosis by the mammalian Ste20-like kinase Mst1. EMBO J. 17: 2224–2234
132. Cryns VL, Byun Y, Rana A, Mellor H, Lustig KD, Ghanem L, Parker PJ, Kirschner MW and Yuan J (1997) Specific proteolysis of the kinase protein kinase C-related kinase 2 by caspase-3 during apoptosis. Identification by a novel, small pool expression cloning strategy. J. Biol. Chem. 272: 29449–29453
133. Santoro MF, Annand RR, Robertson MM, Peng YW, Brady MJ, Mankovich JA, Hackett MC, Ghayur T, Walter G, Wong WW and Giegel DA (1998) Regulation of protein phosphatase 2A activity by caspase-3 during apoptosis. J. Biol. Chem. 273: 13119–13128
134. Gervais FG, Thornberry NA, Ruffolo SC, Nicholson DW and Roy S (1998) Caspases cleave focal adhesion kinase during apoptosis to generate a FRNK-like polypeptide. J. Biol. Chem. 273: 17102–17108
135. Levkau B, Herren B, Koyama H, Ross R and Raines EW (1998) Caspase-mediated cleavage offocal adhesionkinasepp125FAKanddisassembly offocal adhesions in human endothelial cell apoptosis. J. Exp. Med. 187: 579–586
136. Wen LP, Fahrni, JA, Troie S, Guan JL, Orth K and Rosen GD (1997) Cleavage of focal adhesion kinase by caspases during apoptosis. J. Biol. Chem. 272: 26056–26061
137. Crouch DH, Fincham VJ and Frame MC (1996) Targeted proteolysis of the focal adhesion kinase pp125FAK during c-MYC-induced apoptosis is suppressed by integrin signalling. Oncogene 12: 2689–2696
138. Cryns VL, Bergeron L, Zhu H, Li Hand Yuan J (1996) Specific cleavage of alpha-fodrin during Fas- and tumor necrosis factor-induced apoptosis is mediated by an interleukin-1 beta-converting enzyme/Ced-3 protease distinct from the poly(ADP-ribose) polymerase protease. J. Biol. Chem. 271: 31277–31282
139. Vanags DM, Porn-Ares MI, Coppola S, Burgess DH and Orrenius S (1996) Protease involvement in fodrin cleavage and phosphatidylserine exposure in apoptosis. J. Biol. Chem. 271: 31075–31085
140. Martin SJ, O'Brien GA, Nishioka WK, McGahon AJ, Mahboubi A, Saito TC and Green DR (1995) Proteolysis of fodrin (non-erythroid spectrin) during apoptosis. J. Biol. Chem. 270: 6425–6428
141. Wang KK, Posmantur R, Nath R, McGinnis K, Whitton M, Talanian RV, Glantz SB and Morrow JS (1998) Simultaneous degradation of alphall- and betall-spectrin by caspase 3 (CPP 32) in apoptotic cells. J. Biol. Chem. 273: 22490–22497
142. Nath R, Raser KJ, Stafford D, Hajimohammadreza I, Posner A, Allen H, Talanian RV, Yuen P, Gilbertsen RB and Wang KK (1996) Non-erythroid alpha-spectrin breakdown by calpain and interleukin 1 beta-converting-enzyme-like protease(s) in apoptotic cells: contributory roles of both protease families in neuronal apoptosis. Biochem. J. 319: 683–690
143. Kamada S, Kusano H, Fujita H, Ohtsu M, Koya RC, Kuzumaki N and Tsujimoto Y (1998) A cloning method for caspase substrates that uses the yeast two-hybrid system: cloning of the antiapoptotic gene gelsolin. Proc. Natl. Acad. Sci. USA 95: 8532–8537
144. Kothakota S, Azuma T, Reinhard C, Klippel A, Tang J, Chu K, McGarry TJ, Kirschner MW, Koths K, Kwiatkowski DJ and Williams LT (1997) Caspase-3-generated fragment of gelsolin: effector of morphological change in apoptosis. Science 278: 294–298
145. Leers MP, W KI Bjorklund V, Bergman T, Tribbick G, Persson B, Bjorklund P, Ramaekers FC, Bjorklund B, Nap Jr MH and Schutte B (1999) Immunocytochemical detection and mapping of a cytokeratin 18 neo-epitope exposed during early apoptosis. J. Pathol. 187: 567–572
146. Ku NO, Liao J and Omary MB (1997) Apoptosis generates stable fragments of human type I keratins. J. Biol. Chem. 272: 33197–33203
147. Caulin C, Salvesen GS and Oshima RG (1997) Caspase cleavage of keratin 18 and reorganization of intermediate filaments during epithelial cell apoptosis. J. Cell. Biol. 138: 1379–1394
148. Prasad S, Soldatenkov VA, Srinivasarao G and Dritschilo A (1998) Identification of keratins 18, 19 and heat-shock protein 90 beta as candidate substrates of proteolysis during ionizing radiation-induced apoptosis of estrogen-receptor negative breast tumor cells. Int. J. Oncol. 13: 757–764
149. Buendia B, Santa-Maria A and Courvalin JC (1999) Caspase-dependent proteolysis of integral and peripheral proteins of nuclear membranes and nuclear pore complex proteins during apoptosis [In Process Citation]. J. Cell. Sci. 112: 1743–1753
150. Cosulich SC, Horiuchi H, Zerial M, Clarke PR and Woodman PG (1997) Cleavage of rabaptin-5 blocks endosome fusion during apoptosis. EMBO J. 16: 6182–6191

151. Webb SJ, Nicholson D, Bubb VJ and Wyllie AH (1999) Caspase-mediated cleavage of APC results in an amino-terminal fragment with an intact armadillo repeat domain. FASEB J. 13: 339–346
152. Harvey KF, Harvey NL, Michael JM, Parasivam G, Waterhouse N, Alnemri ES, Watters D and Kumar S (1998) Caspase-mediated cleavage of the ubiquitin-protein ligase Nedd4 during apoptosis. J. Biol. Chem. 273: 13524–13530
153. Grandgirard D, Studer E, Monney L, Belser T, Fellay I, Borner C and Michel MR (1998) Alphaviruses induce apoptosis in Bcl-2-overexpressing cells: evidence for a caspase-mediated, proteolytic inactivation of Bcl-2. EMBO J. 17: 1268–1278
154. Kirsch DG, Doseff A, Chau BN, Lim DS, de Souza-Pinto NC, Hansford R, Kastan MB, Lazebnik YA and Hardwick JM (1999) Caspase-3-dependent cleavage of Bcl-2 promotes release of cytochrome c. J. Biol. Chem. 274: 21155–21161
155. Cheng EH, Kirsch DG, Clem RJ, Ravi R, Kastan MB, Bedi A, Ueno K and Hardwick JM (1997) Conversion of Bcl-2 to a Bax-like death effector by caspases. Science 278: 1966–1968
156. Grunberg J, Walter J, Loetscher H, Deutschle U, Jacobsen H and Haass C (1998) Alzheimer's disease associated presenilin-1 holoprotein and its 18-20 kDa C-terminal fragment are death substrates for proteases of the caspase family. Biochemistry 37: 2263–2270
157. Vito P, Ghayur T and D'Adamio L (1997) Generation of anti-apoptotic presenilin-2 polypeptides by alternative transcription, proteolysis, and caspase-3 cleavage. J. Biol. Chem. 272: 28315–28320
158. Loetscher H, Deutschle U, Brockhaus M, Reinhardt D, Nelboeck P, Mous J, Grunberg J, Haass C and Jacobsen H (1997) Presenilins are processed by caspase-type proteases. J. Biol. Chem. 272: 20655–20659
159. Kim TW, Pettingell WH, Jung YK, Kovacs DM and Tanzi RE (1997) Alternative cleavage of Alzheimer-associated presenilins during apoptosis by a caspase-3 family protease. Science 277: 373–376
160. Wellington CL, Ellerby LM, Hackam AS, Margolis RL, Trifiro MA, Singaraja R, McCutcheon K, Salvesen GS, Propp SS, Bromm M, Rowland KJ, Zhang T, Rasper D, Roy S, Thornberry N, Pinsky L, Kakizuka A, Ross CA, Nicholson DW, Bredesen DE and Hayden MR (1998) Caspase cleavage of gene products associated with triplet expansion disorders generates truncated fragments containing the polyglutamine tract. J. Biol. Chem. 273: 9158–9167
161. Ellerby LM, Hackam AS, Propp SS, Ellerby HM, Rabizadeh S, Cashman NR, Trifiro MA, Pinsky L, Wellington CL, Salvesen GS, Hayden MR and Bredesen DE (1999) Kennedy's disease: caspase cleavage of the androgen receptor is a crucial event in cytotoxicity. J. Neurochem. 72: 185–195
162. King P and Goodbourn S (1998) STAT1 is inactivated by a caspase. J. Biol. Chem. 273: 8699–8704
163. Rickers A, Peters N, Badock V, Beyaert R, Vandenberghe P, Dorken B and Bommert K (1999) Cleavage of transcription factor SP1 by caspases during anti-IgM-induced B-cell apoptosis. Eur. J. Biochem. 261: 269–274
164. Piedrafita FJ and Pfahl M (1997) Retinoid-induced apoptosis and Sp1 cleavage occur independently of transcription and require caspase activation. Mol. Cell. Biol. 17: 6348–6358
165. Utz PJ, Hottelet M, Le TM, Kim SJ, Geiger ME, van Venrooij WJ and Anderson P (1998) The 72-kDa component of signal recognition particle is cleaved during apoptosis. J. Biol. Chem. 273: 35362–35370
166. Ravi R, Bedi A and Fuchs EJ (1998) CD95 (Fas)-induced caspase-mediated proteolysis of NF-kappaB. Cancer Res. 58: 882–886
167. Levkau B, Scatena M, Giachelli CM, Ross R and Raines EW (1999) Apoptosis overrides survival signals through a caspase-mediated dominant-negative NF-kappaB loop. Nat. Cell. Biol. 1: 227–233
168. Tang D, Gururajan R and Kidd VJ (1998) Phosphorylation of PITSLRE p110 isoforms accompanies their processing by caspases during Fas-mediated cell death. J. Biol. Chem. 273: 16601–16607
169. Beyaert R, Kidd VJ, Cornelis S, Van de Craen M, Denecker G, Lahti JM, Gururajan R, Vandenberghe P and Fiers W (1997) Cleavage of PITSLRE kinases by ICE/CASP-1 and CPP32/CASP-3 during apoptosis induced by tumor necrosis factor. J. Biol. Chem. 272: 11694–11697
170. Lahti JM, Xiang J, Heath LS, Campana D and Kidd VJ (1995) PITSLRE protein kinase activity is associated with apoptosis. Mol. Cell. Biol. 15: 1–11
171. Walter BN, Huang Z, Jakobi R, Tuazon PT, Alnemri ES, Litwack G and Traugh JA (1998) Cleavage and activation of p21-activated protein kinase gamma-PAK by CPP32 (caspase 3). Effects of autophosphorylation on activity. J. Biol. Chem. 273: 28733–28739

172. Lee N, MacDonald H, Reinhard C, Halenbeck R, Roulston A, Shi T and Williams LT (1997) Activation of hPAK65 by caspase cleavage induces some of the morphological and biochemical changes of apoptosis. Proc. Natl. Acad. Sci. USA 94: 13642–13647
173. Rudel T and Bokoch GM (1997) Membrane and morphological changes in apoptotic cells regulated by caspase-mediated activation of PAK2. Science 276: 1571–1574
174. Ricci JE, Maulon L, Luciano F, Guerin S, Livolsi A, Mari B, Breittmayer JP, Peyron JF and Auburger P (1999) Cleavage and relocation of the tyrosine kinase P59FYN during Fas-mediated apoptosis in T lymphocytes. Oncogene 18: 3963–3969
175. McGinnis KM, Whittom MM, Gnegy ME and Wang KK (1998) Calcium/calmodulin-dependent protein kinase IV is cleaved by caspase-3 and calpain in SH-SY5Y human neuroblastoma cells undergoing apoptosis. J. Biol. Chem. 273: 19993–20000
176. Granville DJ, Carthy CM, Jiang H, Shore GC, McManus BM and Hunt DW (1998) Rapid cytochrome c release, activation of caspases 3, 6, 7 and 8 followed by Bap31 cleavage in HeLa cells treated with photodynamic therapy. FEBS Lett. 437: 5–10
177. Ng FW and Shore GC (1998) Bcl-XL cooperatively associates with the Bap31 complex in the endoplasmic reticulum, dependent on procaspase-8 and Ced-4 adaptor. J. Biol. Chem. 273: 3140–3143
178. Brown SB, Bailey K and Savill J (1997) Actin is cleaved during constitutive apoptosis. Biochem. J. 323: 233–237
179. Mashima T, Naito M, Noguchi K, Miller DK, Nicholson DW and Tsuruo T (1997) Actin cleavage by CPP-32/apopain during the development of apoptosis. Oncogene 14: 1007–1012
180. Song Q, Wei T, Lees-Miller S, Alnemri E, Watters D and Lavin MF (1997) Resistance of actin to cleavage during apoptosis. Proc. Natl. Acad. Sci. USA 94: 157–162
181. Chen Z, Naito M, Mashima T and Tsuruo T (1996) Activation of actin-cleavable interleukin 1beta-converting enzyme (ICE) family protease CPP-32 during chemotherapeutic agent-induced apoptosis in ovarian carcinoma cells. Cancer Res. 56: 5224–5229
182. Kayalar C, Ord T, Testa MP, Zhong LT and Bredesen DE (1996) Cleavage of actin by interleukin 1 beta-converting enzyme to reverse DNase I inhibition. Proc. Natl. Acad. Sci. USA 93: 2234–2238
183. Mashima T, Naito M, Fujita N, Noguchi K and Tsuruo T (1995) Identification of actin as a substrate of ICE and an ICE-like protease and involvement of an ICE-like protease but not ICE in VP-16-induced U937 apoptosis. Biochem. Biophys. Res. Commun. 217: 1185–1192
184. Brancolini C, Benedetti M and Schneider C (1995) Microfilament reorganization during apoptosis: the role of Gas2, a possible substrate for ICE-like proteases. EMBO J. 14: 5179–5190
185. Zhivotovsky B, Gahm A and Orrenius S (1997) Two different proteases are involved in the proteolysis of lamin during apoptosis. Biochem. Biophys. Res. Commun. 233: 96–101
186. Rao L, Perez D and White E (1996) Lamin proteolysis facilitates nuclear events during apoptosis. J. Cell. Biol. 135: 1441–1455
187. Takahashi A, Alnemri ES, Lazebnik YA, Fernandes-Alnemri T, Litwack G, Moir RD, Goldman RD, Poirier GG, Kaufmann SH and Earnshaw WC (1996) Cleavage of lamin A by Mch2 alpha but not CPP32: multiple interleukin 1 beta-converting enzyme-related proteases with distinct substrate recognition properties are active in apoptosis. Proc. Natl. Acad. Sci. USA 93: 8395–8400
188. Orth K, Chinnaiyan AM, Garg M, Froelich CJ and Dixit VM (1996) The CED-3/ICE-like protease Mch2 is activated during apoptosis and cleaves the death substrate lamin A. J. Biol. Chem. 271: 16443–16446
189. Shimizu T, Cao CX, Shao RG and Pommier Y (1998) Lamin B phosphorylation by protein kinase alpha and proteolysis during apoptosis in human leukemia HL60 cells. J. Biol. Chem. 273: 8669–8674
190. Fujita N, Nagahashi A, Nagashima K, Rokudai S and Tsuruo T (1998) Acceleration of apoptotic cell death after the cleavage of Bcl-XL protein by caspase-3-like proteases. Oncogene 17: 1295–1304
191. Clem RJ, Cheng EH, Karp CL, Kirsch DG, Ueno K, Takahashi A, Kastan MB, Griffin DE, Earnshaw WC, Veliuona MA and Hardwick JM (1998) Modulation of cell death by Bcl-XL through caspase interaction. Proc. Natl. Acad. Sci. USA 95: 554–559

192. Gross A, Yin XM, Wang K, Wei MC, Jockel J, Milliman C, Erdjument-Bromage H, Tempst P and Korsmeyer SJ (1999) Caspase cleaved BID targets mitochondria and is required for cytochrome c release, while BCL-XL prevents this release but not tumor necrosis factor-R1/Fas death. J. Biol. Chem. 274: 1156–1163

193. Li H, Zhu H, Xu CJ and Yuan J (1998) Cleavage of BID by caspase 8 mediates the mitochondrial damage in the Fas pathway of apoptosis. Cell 94: 491–501

194. Luo X, Budihardjo I, Zou H, Slaughter C and Wang X (1998) Bid, a Bcl2 interacting protein, mediates cytochrome c release from mitochondria in response to activation of cell surface death receptors. Cell 94: 481–490

195. LeBlanc A, Liu H, Goodyer C, Bergeron C and Hammond J (1999) Caspase-6 role in apoptosis of human neurons, amyloidogenesis, and Alzheimer's disease [In Process Citation]. J. Biol. Chem. 274: 23426–23436

196. Pellegrini L, Passer BJ, Tabaton M, Ganjei JK and D'Adamio L (1999) Alternative, non-secretase processing of Alzheimer's beta-amyloid precursor protein during apoptosis by caspase-6 and -8. J. Biol. Chem. 274: 21011–21016

197. Zhang Y, Center DM, Wu DM, Cruikshank WW, Yuan J, Andrews DW and Kornfeld H (1998) Processing and activation of pro-interleukin-16 by caspase-3. J. Biol. Chem. 273: 1144–1149

198. Schwab BL, Leist M, Knippers R and Nicotera P (1998) Selective proteolysis of the nuclear replication factor MCM3 in apoptosis. Exp. Cell. Res. 238: 415–421

199. Zhou BB, Li H, Yuan J and Kirschner MW (1998) Caspase-dependent activation of cyclin-dependent kinases during Fas-induced apoptosis in Jurkat cells. Proc. Natl. Acad. Sci. USA 95: 6785–6790

200. Gohring F, Schwab BL, Nicotera P, Leist M and Fackelmayer FO (1997) The novel SAR-binding domain of scaffold attachment factor A (SAF-A) is a target in apoptotic nuclear breakdown. EMBO J. 16: 7361–7371

201. Widmann C, Gibson S and Johnson GL (1998) Caspase-dependent cleavage of signaling proteins during apoptosis. A turn-off mechanism for anti-apoptotic signals. J. Biol. Chem. 273: 7141–7147

202. Takahashi M, Mukai H, Toshimori M, Miyamoto M and Ono Y (1998) Proteolytic activation of PKN by caspase-3 or related protease during apoptosis. Proc. Natl. Acad. Sci. USA 95: 11566–11571

203. Brancolini C, Sgorbissa A and Schneider C (1998) Proteolytic processing of the adherens junctions components beta-catenin and gamma-catenin/plakoglobin during apoptosis. Cell Death Differ. 5: 1042–1050

204. Tesco G, Kim TW, Diehlmann A, Beyreuther K and Tanzi RE (1998) Abrogation of the presenilin 1/beta-catenin interaction and preservation of the heterodimeric presenilin 1 complex following caspase activation. J. Biol. Chem. 273: 33909–33914

205. Schmeiser K, Hammond EM, Roberts S and Grand RJ (1998) Specific cleavage of gamma catenin by caspases during apoptosis. FEBS Lett. 433: 51–57

206. Herren B, Levkau B, Raines EW and Ross R (1998) Cleavage of beta-catenin and plakoglobin and shedding of VE-cadherin during endothelial apoptosis: evidence for a role for caspases and metalloproteinases. Mol. Biol. Cell. 9: 1589–1601

207. Brancolini C, Lazarevic D, Rodriguez J and Schneider C (1997) Dismantling cell-cell contacts during apoptosis is coupled to a caspase-dependent proteolytic cleavage of beta-catenin. J. Cell. Biol. 139: 759–771

208. Machleidt T, Geller P, Schwandner R, Scherer G and Kronke M (1998) Caspase 7-induced cleavage of kinectin in apoptotic cells. FEBS Lett. 436: 51–54

209. Porn-Ares MI, Samali A and Orrenius S (1998) Cleavage of the calpain inhibitor, calpastatin, during apoptosis. Cell Death Differ. 5: 1028–1033

210. Wang KK, Posmantur R, Nadimpalli R, Nath R, Mohan P, Nixon RA, Talanian RV, Keegan M, Herzog L and Allen H (1998) Caspase-mediated fragmentation of calpain inhibitor protein calpastatin during apoptosis. Arch. Biochem. Biophys. 356: 187–196

211. Chan SL, Griffin WS and Mattson MP (1999) Evidence for caspase-mediated cleavage of AMPA receptor subunits in neuronal apoptosis and Alzheimer's disease. J. Neurosci. Res. 57: 315–323
