
FEMS Microbiology Letters 181 (1999) 253–260

**Cyanobacterial ycf27 gene products regulate energy transfer from phycobilisomes to photosystems I and II**

Mark K. Ashby, Conrad W. Mullineaux*

Department of Biology, University College London, Darwin Building, Gower Street, London WC1E 6BT, UK

Received 5 July 1999; received in revised form 7 October 1999; accepted 15 October 1999

---

### Abstract

Two open reading frames (slr0115 and slr0947) in the genome of the cyanobacterium *Synechocystis* sp. PCC 6803 are shown to be involved in the regulation of the coupling of phycobilisomes to photosynthetic reaction centres. Homologues of these genes, called *ycf27*, have been found in a range of phycobilin-containing organisms. The slr0115 and slr0947 gene products are OmpR-type DNA-binding response regulator proteins. Deletion of slr0115 results in increased efficiency of energy transfer from phycobilisomes to photosystem II relative to photosystem I. Reduction of the copy number of slr0947 has the opposite phenotypic effect. We have given the slr0115 and slr0947 genes the designations *rpaA* and *rpaB* respectively. © 1999 Federation of European Microbiological Societies. Published by Elsevier Science B.V. All rights reserved.

---

### Keywords: Cyanobacterium; Photosynthesis; Regulation; Two-component; Phycobilisome; Photosystem I/II

---

### 1. Introduction

Cyanobacteria are prokaryotic organisms that perform oxygenic photosynthesis in a manner remarkably similar to green plants. Phycobilisomes serve as the major light-harvesting antenna for cyanobacteria, transferring energy to the photosystem (PS) II and PSI reaction centres [1]. Efficient energy transfer from phycobilisomes to PSI can be observed in a PSII-free mutant of *Synechocystis* 6803, suggesting that phycobilisomes can interact directly with PSI as well as with PSII [2, 3]. *Synechocystis* 6803 is known to respond to changes in the quality of incident light by altering the ratio of PSI to PSII [4] and by a process known as state transition that regulates the interaction of phycobilisomes with reaction centres [3, 5]. These mechanisms are believed to ensure that electron transport through PSI and PSII is balanced in different light regimes.

We have been using a strategy based on the screening of random mutants with a fluorescence video imaging system to identify genes that are involved in the regulation of light harvesting in *Synechocystis* 6803 [6]. Here we report the identification of two DNA-binding response regulators implicated in controlling the interaction of phycobilisomes and reaction centres.

---

* Corresponding author. Tel.: +44 (171) 504 2326; Fax: +44 (171) 380 7096; E-mail: c.mullineaux@ucl.ac.uk

Abbreviations: PSI/II, photosystem I/II; RpaA/B, regulator of phycobilisome attachment A/B; PC, phycocyanin; APC, allophycocyanin

0378-1097/99/$20.00 © 1999 Federation of European Microbiological Societies. Published by Elsevier Science B.V. All rights reserved.

PII: S0378-1097(99)00547-9

2. Materials and methods

Synecchocystis 6803 was grown in BG11 medium [7] at 30°C under white light. Light intensity was 12 μE m⁻² s⁻¹ for liquid cultures and 5–10 μE m⁻² s⁻¹ for agar plates. Media for 6803 and 68115 were supplemented with 10 mM sodium bicarbonate and media for PsbD1CD2⁻, PS2115, PS947 and 68947 were supplemented with 5 mM glucose. Liquid cultures were grown aerobically in a shaking incubator without any gas bubbling. Kanamycin was added to media when required at 50 μg ml⁻¹ and chloramphenicol at 25 μg ml⁻¹. Strains and plasmids used are listed in Table 1. Genomic DNA was prepared using the Nucleon Phytopure Plant DNA extraction kit (Scotlab Bioscience). The complete sequence of the Synecchocystis 6803 genome was searched and analysed at the CyanoBase site (http://www.kazusa.or.jp/cyano/). Synecchocystis 6803 plasmid transformations were carried out as described by Chauvat et al. [8].

Routine DNA manipulations were performed as in Sambrook et al. [9] using commercially available reagents. TBLASTN searches were performed at the National Centre for Biotechnology web site (http://www.ncbi.nlm.gov/).

Random insertional mutant libraries were made according to the methods described by Chauvat et al.[8]. Kanamycin-resistant colonies were screened using a fluorescence video imaging system [6].

Cell absorption spectra, fluorescence emission spectra and pigment content were measured as described in Emlyn-Jones et al. [9]. 77K fluorescence spectra were recorded for samples at a chlorophyll concentration of 5 μM.

Intact phycobilisomes were isolated according to the method of Glazer [10].

3. Results

3.1. Construction and genetic characterisation of mutants

A library of random insertional mutants in a PSII-free strain of Synecchocystis 6803 (PsbD1CD2⁻) was screened using a fluorescence video imaging system [6]. A mutant was isolated that had increased fluo-

Fig. 1. Diagrams of the slr0115 (A) and slr0947 (B) loci, showing the positions of the coding regions on the genome (CyanoBase), restriction enzyme sites used for Southern blot analysis and insertional inactivation, and the position and orientation of the oligonucleotides used for PCR amplification of the genes. The antibiotic resistance genes used for insertional inactivation and the names of the mutants generated are shown beside them. C: Southern blot of NcoI-digested genomic DNA probed with the slr0947 probe. Lanes A, 6803; B, PsbD1CD2⁻; C, 68947; D, PS2947.

rescence from the phycobilisome core components, suggesting that the efficiency of energy transfer from phycobilisomes to PSI was decreased. Southern analysis (data not shown) was used to identify an MaeI fragment containing the kanamycin cassette and flanking genomic sequence. This was cloned by marker rescue in pGEM5Zf and sequenced to deter-

mine the insertion site. The open reading frame slr0115 was found to have been disrupted by an insertion of the kanamycin gene at the SalI site at position 2996550 (Fig. 1). This plasmid (pSal110-MaeI) was transformed back into Synechocystis 6803 and PsbD1CD2− to generate the mutants 68115 and PS2115 respectively.

The translation of slr0115 (rpaA) generates a hypothetical protein of 241 amino acids with sequence similarity to a number of putative bacterial and plastid OmpR-type regulators (Fig. 2). These include in particular the ycf27 homologues in the plastid genomes of the protist Cyanophora paradoxa [11], the red alga Porphyra purpurea [12] and the cryptophyte alga Guillardia theta [13]. There is also strong homology with another Synechocystis 6803 open reading frame, slr0947. This prompted us to construct slr0947 insertion mutants, 68947 and PS2947 (see Table 1). The structures of the slr0115 and slr0947 genes and the position of the resistance cassette insertion sites are shown in Fig. 1.

The correct insertion and segregation of the kanamycin cassette in the genome of 68115 and PS2115 was confirmed by Southern analysis of MaeI-digested genomic DNA (data not shown). For 68947 and PS2947, Southern blots of SacII/XbaI-digested DNA show fragments hybridising to the probe at

3.4 kb and 2.2 kb (Fig. 1C). A 2.2-kb band is predicted from the wild-type sequence, and is also observed in the wild-type (Fig. 1C). The 3.4-kb band is as predicted for the insertion of a 1.2-kb cassette into the SmaI site in the gene. The presence of both bands in the mutants shows that the slr0947 gene has been correctly disrupted but the mutants have not been able to segregate, even after streaking from single colonies more than a dozen times on media containing glucose and kanamycin. This suggests that the complete loss of slr0947 is lethal under the growth conditions tested.

### 3.2. Whole cell 77K fluorescence emission spectra

Low-temperature fluorescence emission spectra of intact cells are shown in Fig. 3. The excitation wavelength of 600 nm is absorbed mainly by phycobilisome components. Fig. 3a shows spectra for the PSII-deficient mutants, grown in the presence of glucose. The peak at 650–660 nm comes from phyco-cyanin and allophycocyanin, the peak at about 680 nm is from the long-wavelength terminal emitters of the phycobilisome core and the peak at 725 nm is from PSI. The 680-nm peak is somewhat higher than in wild-type cells due to the presence of some energetically decoupled phycobilisomes [2].

---

Table 1  
Bacterial strains and plasmids used

| Name          | Description                                                                                                      | References and source |
|---------------|------------------------------------------------------------------------------------------------------------------|------------------------|
| Strains       |                                                                                                                  |                        |
| Synechocystis 6803 | Glucose tolerant ‘wild-type’ strain                                                                           | [21]                   |
| PsbD1CD2−     | Photosystem 2 free mutant of 6803. PsbD1C locus inactivated by a chloramphenicol resistance gene. PsbD2 locus inactivated by streptomycin resistance gene. Glucose requiring. | [22]                   |
| 68947         | slr0947 locus inactivated by insertion of kanr (pUC-4K) into SmaI site at position 2013433 in 6803 background. Usually grown in the presence of glucose. | This work             |
| PS2947        | As above in PsbD1CD2− background.                                                                                | This work             |
| 68115         | slr0115 locus inactivated by insertion of kanr (pBSSK) into SalI site at position 2996551 in 6803 background.           | This work             |
| PS2115        | As above in PsbD1CD2− background.                                                                                | This work             |
| Plasmids      |                                                                                                                  |                        |
| pBSSK         | Kanr from Tn903 (pPR691) cloned in EcoRI/PstI sites of pBluescript SK+.                                           | This work and [23]    |
| pGEM5Zf       | Multipurpose cloning vector with polylinker.                                                                      | Promega                |
| pSal110MaeI   | 1.7kb MaeI fragment plasmid rescued (Kanr) from PS2115 and cloned into NdeI site pGEM5Zf.                            | This work             |
| pBS947        | 1430-bp PCR product of slr0947 KpnI/SacI ends cloned in pBSSK+.                                                    | This work             |
| pBS947ΔK      | HincII Kanr (pUC-4K) cloned in SmaI site pBS947.                                                                  | This work             |

FEMSLE 9124 24-11-99

10 20 30 40 50 60 70 80 90
RpaA MPR- -------- ILII DDDPAISDLV SINLE MAGYD VQQAVDGIKG QALAVQLQPD LIMLDMLPK VDGFTVCQRL RRDER TADIP
RpaB ------- IVV DDEASIRRIL ETRLSMIGYE VVTAADGEEA IATFHESDPD LVVLDMMPK LDGYGVCQEL RKES---DIP
P.pur ycf 27 M-ENQ------- KEKILVV DDEASIRRIL ETRLTIIGYE VITASNGEEA LIIFRKEYPS LVVLDMMPK LDGYGVCQEL RKES---DVP
C.par ycf 27 MEI-------- N--KKKILVV DDEASIRRIL ETRLSMIGYE VVTAADGEEA LTIFQLEHPN LVVLDMMPK LDGYGVCQEL RKES---DIP
G.the ycf 27 MIKNRCLDID LETKEKLVIV DDEASIRKIL ETRLSMIGYF VVTASDGEAA LNLFHQEMPD VVILDIMMPK LDGYGVCQEI RKDS---DVP
P.aer OmpR MQ--------- KEKILVI DDEASIRRIL ETRLSIIGYD VISAADGEEA LSIFKREHPN LVVLDMLMPK LDGYGVCQEL RKES---DVP
B.sub PhoP MNKKILVVDD MN--KKILVV DDEESIVTLL QYNLERSGYD VITASDGEAA LKKAETEKPD LIVLDVMLPK LDGIEVCKQL RQQKLMF--P
T.mar drrA MA--------- KKKILVV DDDPAILELV GYNLSKEGYE VLKAYDGEAA LKIANDEDVD MFIVDIMLPG IDGFELVRKI RSMEKYKNTP
E.coli OmpR MQENY------ KILVV DDDMRLRALL ERYLTEQGFQ VRSVANAEQM DRLLTRESFH LMVLDMLPG EDGLSICRRL RSQS--NPMP

D

100 110 120 130 140 150 160 170 180
RpaA VMLMTALGQI QDKIQGFDSG ADDYLTKPFD VEEMLARVRA LLRRTDRIPO AA--KHSEIL NQ---GPLTL VPERFEAIWF GKSIKLTHLE
RpaB IIMLTALGDV ADRTGLELG ADDYVVKPFS PKELEARIRS VLRRVDKNGM PG-IPSSGVL QI---ATIRI DTNKROVYKG DERIRL TGME
P.pur ycf 27 IIMLTALGEV CDRTGLEIG ADDYVVKPFS PKELEARIRS VLRRADKITT NLGVPNSGI- --ISIGFLKI DTNKROVYKN NERVLTGME
C.par ycf 27 IIMLTALGDV ADRTGLELG ADDYVVKPFS PKELEARIRS LLRRTENPTL STTSTNGENL QI---GFLKI DINKRQVFKN GERIRL TGME
G.the ycf 27 IIMLTALGDV ADRTGLELG ADDYVVKPFS PKEEARIRA VLRRTNKAAF SSHLTTSGII NF---NFLTI DLNKROIYKD NERIRL TGME
P.aer OmpR IIMLTALSDV SDRITGLELG ADDYIVKPFS PKELEARIRS VLRRVDKASS NNNLPNSGII NI---GFLKI DVNKHQVYKN NERVLTGME
B.sub PhoP IMLTAKDEE FDKVGLLEG ADDYMTKPFS PREVNARVKA ILRARSEIAAP SSEMKNDEME GQIVIGDLKI LPDHYEAYFK ESQLELTPKE
T.mar drrA VIFLSAKGEE FDKVGLLEG ADDYITKPFS VRELLARVKA IFRRLSTATQ SKE----ERP KKIIAKDLEI DVEKYEVKVR GKKVNLTPLE
E.coli OmpR IIMVTAKGEE VDRIVGLEIG ADDYIPKPFN PRELLARIRA VLRRQANELP GAPSQEEAV- --IAFGKFKL NLGTREM FRE DEPMPLTSGE
LXKXE

190 200 210 220 230 240 250 260
RpaA FELLHCLLQR HGQTVSPSDI LREVWGY--E PDDDIETIRV HIRHLRTKLE PNPRRPRFIK TVYGAGYCLE LSTEEGGGSPT
RpaB FSLELELLVSR SGEPFSRSEI LQE VWGYTPE RHVDTRVVDV HISRLRAKLE EDPSNPELIL TARGETGYLFQ RIL-EP-GEEP
P.pur ycf 27 FSLELELLVSK AGEPFSRASI LQE VWGYTPE RHVDTRVVDV HISRLRAKLE DDPSNPDLIL TARGETGYLFQ RIIE-MNKL--
C.par ycf 27 FSLELELLISK MGE PFSRAQI LQE VWGYTPE RHIDTRVVDV HISRLRSKLE ENPSNPDLIL TARGETGYLFQ -------NTSN
G.the ycf 27 FSLELELLISR SGQPFSRADI LQE VWGYTPE RHVDTRVVDV HISRLRAKLE TDPSNPDLIL TARGETGYLFQ RIT-DQHVLKT
P.aer OmpR FSLELELLISK AGQPFSRATI LQE VWGYTAE RQVDTRVVDV HISRLRAKLE DDPSNP-LIL TARGETGYLFQ RL----NDSIV
B.sub PhoP FELLLYLGRH KGRVLTRDLL LSAVWNY--D FAGDTRIVDV HISHLRDKIE NNTKKPIYIK TIRGLGYKLE EPKMNE------
T.mar drrA FELLRFLAEN EGKVFSRDVL LDKLWGY--D YYGDTRTVDV HIRRLRTKIE EDPSNPKYII TVRGKGYKFR DPGKED------
E.coli OmpR FAVLKALVSH PREPLSRDKL MNLARG--RE YSAMERSIDV QISRLRRMVE EDPAHPRYIQ TVWGLGYVFV ------PDGSKA
xxxL LR TxxGxGY

Fig.2. Alignment of selected OmpR-type bacterial regulator protein sequences. Aspartate residue 65 (phosphorylation site and motifs associated with DNA binding are highlighted. Accession numbers: P. purpurea ycf 27, U 38804; C. paradoxa ycf 27, U 30821; G.theta ycf 27, AF 041468; P. aeruginum ompR, X 62579; B. subtilis phoP, X 67676; T. maritima drrA, U 67196; E. coli ompR, P 03025.

However, most phycobilisomes transfer energy efficiently to PSI, and hence the terminal emitter fluorescence is much lower than in isolated phycobilisomes [2]. As compared to the parent strain, PS 2115 shows increased fluorescence from the phycobilosome terminal emitters, whereas PS 2947 shows decreased fluorescence from the phycobilosome terminal emitters. This suggests that the deletion of slr 0115 reduces the efficiency of energy transfer to PSI, whereas the reduction in the copy number of

slr 0947 increases the efficiency of energy transfer to PSI.
Fig.3b shows spectra of wild-type Synechocystis 6803 cells grown in the absence of glucose and adapted to state 1 or state 2 before freezing. The spectra show the peaks seen in the PSII-free mutants (Fig.3a) with an additional peak at about 695 nm from PSII. PSII also contributes to the peak at 680-685 nm. In state 1, most of the energy absorbed by phycobilisomes is transferred to PSII. In state 2, the

Table 2
Pigment content of Synechocystis strains used in this publication

| Strain       | 6803      | 68115     | 6803 on glucose | 68947 on glucose | PsbD1CD 2- | PS2115    | PS2947    |
|--------------|-----------|-----------|-----------------|------------------|------------|-----------|-----------|
| Chl/cell     | 1.6×10⁷   | 1.3×10⁷   | 1.2×10⁷         | 1.2×10⁷          | 1.4×10⁷    | 1.4×10⁷   | 1.4×10⁷   |
| PCB/cell     | 6.5×10⁶   | 4.5×10⁶   | 7.1×10⁶         | 5.3×10⁶          | 7.3×10⁶    | 6.2×10⁶   | 6.7×10⁶   |
| PCB/Pbsom    | 201       | 190       | 131             | 190              | 235        | 151       | 158       |
| Pbsom/cell   | 32400     | 29600     | 30700           | 27800            | 31100      | 41300     | 42300     |
| PSI/cell     | 82500     | 87400     | 89700           | 126100           | 74800      | 103200    | 113000    |
| PSII/cell    | 15700±4400| 10700±2000| 13100±723       | 9600±542         | X          | X         | X         |
| PSII/PSI     | 0.19±0.05 | 0.12±0.02 | 0.15±0.08       | 0.076±0.004      | X          | X         | X         |

PCB, phycocyanobilin; Pbsom, phycobilisome.

M.K. Ashby, C.W. Mullineaux / FEMS Microbiology Letters 181 (1999) 253–260

a
![Image](image_a.png)

b
![Image](image_b.png)

c
![Image](image_c.png)

d
![Image](image_d.png)

e
![Image](image_e.png)

f
![Image](image_f.png)

Wavelength (nm)

Fig. 3. 77K fluorescence emission spectra of whole cells. Excitation is at 600 nm for a–e, and 435 nm for f. a: PSII-deficient mutants, dark-adapted. b: 6803. c: 68115. d: 6803 on glucose. e: 68947 on glucose. f: 6803 and 68947 grown on glucose and dark-adapted. Spectra are normalised at 650 nm or 725 nm.

proportion of energy transferred to PSII is lower, resulting in a lower PSII emission relative to PSI and the phycobilisomes [3, 5]. Spectra for 68115 are shown in Fig. 3c. This mutant performs state transitions. However, 68115 shows higher fluorescence from PSII relative to PSI and the phycobilisomes, as compared to the wild-type. The changes in the spectra are particularly striking because the PSII/

PSI ratio is lower in 68115 (see Table 2 and discussion below). This indicates that the efficiency of energy transfer from phycobilisomes to PSII is increased in 68115.

The partially segregated 68947 mutant could only be grown in liquid culture in the presence of glucose. We therefore recorded spectra of wild-type Synechocystis 6803 grown in the presence of glucose for com-

parison. These are shown in Fig. 3d. Growth in the presence of glucose prevents state transitions: wild-type cells always show quite a strong PSII fluorescence emission. By contrast, 68947 shows virtually no 695-nm PSII fluorescence emission with 600-nm excitation (Fig. 3e). This loss of the 695-nm PSII peak is not due to a direct effect on the PSII core complex, because the 695-nm peak can be clearly seen with 435-nm excitation which is absorbed by the chlorophylls of the reaction centre complexes (Fig. 3f). Therefore energy transfer from phycobilisomes to PSII must be disrupted in the 68947 mutant.

77K fluorescence emission spectra of isolated phycobilisomes show that the phycobilisomes of all the mutants are functionally intact, with a large single emission peak at about 680 nm (data not shown).

### 3.3. Pigment content of *Synechocystis* cells

Table 2 shows the cell content of photosynthetic complexes in wild-type and mutant cells. The most notable changes occur in the 68115 and 68947 mutants. As judged from atrazine-binding assays and spectrophotometric measurements of P~700~ concentration, 68115 and 68947 have a significantly lower PSII/PSI ratio as compared to wild-type cells grown under the same conditions. The cell content of phycobilisomes is somewhat increased in PS2115 and PS2947.

### 3.4. Sequence similarity of *ycf27* deduced proteins

Alignments of the deduced amino acid sequences of slr0947, slr0115, *ycf27* and four other OmpR-type bacterial regulator proteins are shown in Fig. 2. Some key motifs are highlighted. These include an aspartate residue [14] which is phosphorylated by a histidine kinase, and the LxxxExxxL, LR and TxxGxGY DNA binding motifs characteristic of OmpR-type proteins [15–17]. slr0115 and slr0947 encode proteins that contain all the structural elements of typical, functional OmpR-type regulators. We have sequenced partial clones (347–627 bp) of slr0947 and slr0115 homologues from the cyanobacteria *Calothrix* PCC 7601, *Dactylococcus salina*, *Spirulina platensis* and *Synechococcus* PCC 7002 (GenBank database, accession numbers AF135386–

92). Comparison of the deduced amino acid sequences from either slr0115 or slr0947 shows matches of greater than 85% (data not shown).

## 4. Discussion

The two *ycf27* genes of *Synechocystis* 6803 (slr0115 and slr0947) encode putative OmpR-type transcriptional regulators. Such regulators are invariably involved in two-component sensory transduction pathways [15]. Our results show that the products of the slr0115 and slr0947 genes influence the distribution of excitation energy absorbed by the phycobilisomes. An slr0115-deficient mutant, 68115, shows increased energy transfer from phycobilisomes to PSII relative to PSI, as compared to the wild-type (Fig. 3). Mutants in which slr0947 is interrupted will not fully segregate, suggesting that the complete inactivation of this gene is lethal. However, the reduction in the copy number of slr0947 has a remarkably strong phenotypic effect, suggesting that a delicately balanced regulatory mechanism is being affected. The reduction of the copy number of slr0947 has the opposite phenotypic effect from the deletion of slr0115. The 68947 mutant shows strongly decreased energy transfer from phycobilisomes to PSII relative to PSI (Fig. 3). Comparable effects can be seen when the genes are interrupted in a PSII-free background: 68115 shows decreased energy transfer from phycobilisomes to PSI and 68947 shows increased energy transfer from phycobilisomes to PSI (Fig. 3a).

Mutation of the *ycf27* genes also leads to changes in the PSII/PSI ratio (Table 2). In 68115 this may be a secondary consequence of the energy transfer changes, since the PSII/PSI ratio is controlled by redox-regulated feedback mechanisms [18]. Increased efficiency of energy transfer from phycobilisomes to PSII relative to PSI would lead to lead to reduction of the intersystem electron carriers. This triggers a feedback control mechanism that enhances the expression of PSI components relative to PSII [18]. We have seen comparable effects in mutants deficient in certain phycobilisome core components [19]. The inability of the 68947 mutant to segregate fully, and its inability to grow in liquid culture in the absence of glucose, suggest that the slr0947 gene product may be required for another critical process in the cell.
The lower cell content of PSII in 68947 (Table 2) could indicate some destabilisation of the PSII core complex.

We propose that the *ycf27* gene products are involved in a regulatory mechanism that influences the distribution of phycobilisome-absorbed light energy between the two kinds of reaction centres. Since the *ycf27* gene products are transcriptional regulators, this mechanism must involve the regulation of gene expression. The mechanism is distinct from state transitions, a short-term mechanism that adjusts the function of the light-harvesting apparatus on a timescale of seconds to minutes [3,5]. We have recently identified a gene required for state transitions in *Synechocystis* [6]. The *ycf27* gene products could be involved in a longer-term counterpart of state transitions.

We propose the designations *rpaA* for *slr0115* and *rpaB* for *slr0947* (*regulator of phycobilisome association*). The gene(s) whose expression is controlled by the RpaA and RpaB proteins remain to be identified, but it is probable that RpaA and RpaB modulate the expression of factors required to couple the phycobilisomes to PSI or PSII. Phycobilisomes isolated from *rpaA* and *rpaB* mutants are intact and show normal fluorescence emission spectra, suggesting that the phycobilisome core composition is unchanged.

Phylogenetic and gene loss analysis of chloroplast plastid sequences have shown that *ycf27* was lost from the plastid genome at two branch points in the evolution of the chloroplast [20]. In both cases, the loss of *ycf27* correlates with the loss of phycobilisomes, which is consistent with a role for *ycf27* in regulating phycobilisome function.

Two-component sensory transduction systems are a universal method which bacteria use to respond to their environment [14], so it is no surprise to find this system involved in the long-term adaptation of light harvesting. The mode of control of the activity of RpaA/B (presumably via histidine kinases) and their site of action remain to be elucidated.

### Acknowledgements

DNA cycle sequencing was performed by Laura Winskill in the Department of Biology, University College London and by the Advanced Biotechnology Centre, Imperial College School of Medicine, Charing Cross Campus. We thank Wim Vermaas (Arizona State University) for the gift of the PsbD1CD2 mutant, Jean Houmard (Ecole Normale Supérieure) for the gift of *Calothrix* PCC 7601 DNA and Peter Horton (University of Sheffield) for the gift of ${}^{14}$C-labelled atrazine. M.K.A. is supported by a BBSRC Research Grant. This work was also supported by a Nuffield Foundation Award to Newly Appointed Science Lecturers and an award from the UCL Graduate School.

### References

[1] Mullineaux, C.W. (1992) Excitation energy transfer from phycobilisomes to photosystem I in a cyanobacterium. Biochim. Biophys. Acta 1100, 285–292.

[2] Mullineaux, C.W. (1994) Excitation energy transfer from phycobilisomes to photosystem I in a cyanobacterial mutant lacking photosystem II. Biochim. Biophys. Acta 1184, 71–77.

[3] van Thor, J.J., Mullineaux, C.W., Matthijs, H.C.P. and Hellingwerf, K.J. (1998) Light-harvesting and state transitions in cyanobacteria. Bot. Acta 111, 430–443.

[4] Fujita, Y. (1997) A study on the dynamic features of photosystem stoichiometry: Accomplishments and problems for future studies. Photosynth. Res. 53, 83–93.

[5] Allen, J.F. (1992) Protein phosphorylation in regulation of photosynthesis. Biochim. Biophys. Acta 1098, 275–335.

[6] Emlyn-Jones, D., Ashby, M.K. and Mullineaux, C.W. (1998) A gene required for the regulation of photosynthetic light-harvesting in the cyanobacterium *Synechocystis* 6803. Mol. Microbiol. 33, 1050–1058.

[7] Castenholz, R.W. (1988) Culturing methods for cyanobacteria. Methods Enzymol. 167, 68–93.

[8] Chauvat, F., Rouet, P., Bottin, H. and Boussac, A. (1989) Mutagenesis by random cloning of an *Escherichia coli* kanamycin resistance gene into the genome of the cyanobacterium *Synechocystis* PCC 6803: Selection of mutants defective in photosynthesis. Mol. Gen. Genet. 216, 51–59.

[9] Sambrook, J., Fritsch, E.F. and Maniatis, T. (1989) Molecular Cloning: A Laboratory Manual, 2nd edn. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY.

[10] Glazer, A.N. (1988) Phycobilisomes. Methods Enzymol. 167, 304–312.

[11] Stirewalt, V.L., Michalowski, C.B., Loffelhardt, W., Bohnert, H.J. and Bryant, D.A. (1995) Nucleotide sequence of the cyanelle genome from *Cyanophora paradoxa*. Plant Mol. Biol. Rep. 13, 327–332.

[12] Reith, M. and Munholland, J. (1995) Complete nucleotide sequence of the *Porphyra purpurea* chloroplast genome. Plant Mol. Biol. Rep. 13, 333–335.

[13] Douglas, S.E. and Penny, S.L. (1999) The plastid genome of
the cryptophyte alga, *Guillardia theta*: Complete synteny groups confirm its common ancestry with red algae. J. Mol. Evol. 48, 236–244.

[14] Parkinson, J.S. and Kofoid, E.C. (1992) Communication modules in bacterial signalling proteins. Annu. Rev. Genet. 26, 71–112.

[15] Hakenbeck, R. and Stock, J.B. (1996) Analysis of two-component signal transduction systems involved in transcriptional regulation. Methods Enzymol. 273, 281–300.

[16] Mizuno, T. and Tanaka, I. (1997) Structure of the DNA-binding domain of the OmpR family of response regulators. Mol. Microbiol. 24, 665–667.

[17] Martinez-Hackert, E. and Stock, A.M. (1997) Structural relationships in the OmpR family of winged-helix transcription factors. J. Mol. Biol. 269, 301–312.

[18] Fujita, Y., Murakami, A., Ohki, K. and Hagiwara, N. (1988) Regulation of photosystem composition in cyanobacterial photosynthetic system. Plant Cell Physiol. 29, 557–564.

[19] Ashby, M.K. and Mullineaux, C.W. (1999) The role of ApcD and ApcF in energy transfer from phycobilisomes to PSI and PSII in a cyanobacterium. Photosynth. Res. (in press).

[20] Martin, W., Stoebe, B., Goremykin, V., Hansmann, S., Hasegawa, M. and Kowallik, K. (1998) Gene transfer to the nucleus and the evolution of chloroplasts. Nature 393, 162–165.

[21] Williams, J.G.K. (1988) Construction of specific mutations in photosystem II photosynthetic reaction center by genetic engineering methods in *Synechocystis* 6803. Methods Enzymol. 167, 766–778.

[22] Vermaas, W., Charite, J. and Eggers, B. (1989) System for site-directed mutagenesis in the *PsbDI/C* operon of *Synechocystis* sp. PCC 6803. In: Current Research in Photosynthesis (Baltcheffsky, M., Ed.), Vol. 1, pp. 231–238. Kluwer Academic, Dordrecht.

[23] Jiang, X.-M., Brahmbhatt, H.N., Quigley, N.B. and Reeves, P.R. (1987) A low copy number cosmid. Plasmid 18, 170–172.
