
Microbiology (1999), 145, 3255–3264

The oxygenase component of the  
2-aminobenzenesulfonate dioxygenase  
system from Alcaligenes sp. strain O-1  

Jörg Mampel, Jürgen Ruff, Frank Junker† and Alasdair M. Cook  

Author for correspondence: Alasdair M. Cook. Tel: +49 7531 88 4247. Fax: +49 7531 88 2966.  
e-mail: alasdair.cook@uni-konstanz.de  

Department of Biology,  
The University, D-78457  
Konstanz, Germany  

Growth of Alcaligenes sp. strain O-1 with 2-aminobenzenesulfonate (ABS; orthanilate) as sole source of carbon and energy requires expression of the soluble, multicomponent 2-aminobenzenesulfonate 2,3-dioxygenase system (deaminating) (ABSDOS) which is plasmid-encoded. ABSDOS was separated by anion-exchange chromatography to yield a flavin-dependent reductase component and an iron-dependent oxygenase component. The oxygenase component was purified to about 98% homogeneity and an $\alpha_{2}\beta_{2}$ subunit structure was deduced from the molecular masses of 134, 45 and 16 kDa for the native complex, and the $\alpha$ and $\beta$ subunits, respectively. Analysis of the amount of acid labile sulfur and total iron, and the UV spectrum of the purified oxygenase component indicated one [2Fe–2S] Rieske centre per $\alpha$ subunit. The inhibition of activity by the iron-specific chelator o-phenanthroline indicated the presence of an additional iron-binding site. Recovery of active protein required strictly anoxic conditions during all purification steps. The FAD-containing reductase could not be purified. ABSDOS oxygenated nine sulfonated compounds; no oxygen uptake was detected with carboxylated aromatic compounds or with aliphatic sulfonated compounds. $K_{\mathrm{m}}$ values of 29, 18 and 108 μM and $V_{\max }$ values of 140, 110 and 72 pkat for ABS, benzenesulfonate and 4-toluenesulfonate, respectively, were observed. The N-terminal amino acid sequences of the $\alpha$- and $\beta$-subunits of the oxygenase component allowed PCR primers to be deduced and the DNA sequence of the $\alpha$-subunit was thereafter determined. Both redox centres were detected in the deduced amino acid sequence. Sequence data and biochemical properties of the enzyme system indicate a novel member of the class IB ring-hydroxylating dioxygenases.

Keywords: Alcaligenes sp. strain O-1, characterization of oxygenase, 2-aminobenzenesulfonate dioxygenase system, oxygenase sequence, oxygenation and deamination

INTRODUCTION

The complete mineralization of sulfonated aromatic compounds, which, with one known exception (Laskin & Lechevalier, 1984), are all xenobiotics, is restricted to aerobic micro-organisms, which use oxygenases to activate those inert compounds (Cook *et al.*, 1998). Most organisms studied to date utilize only one or two sulfonated compounds (Cook *et al.*, 1998); Alcaligenes sp. strain O-1 can utilize three of them, benzenesulfonate (BS), 4-toluenesulfonate (TS) and 2-aminobenzene-sulfonate (ABS; orthanilate) (Thurnheer *et al.*, 1986). Metabolism of these three compounds involves two independently induced pathways involving either a BS/TS dioxygenase system or an ABS dioxygenase system (ABSDOS) (Junker *et al.*, 1994b). The enzymes

† Present address: Estación Experimental del Zaidín, Consejo Superior de Investigaciones Científicas, E-18008 Granada, Spain.  
Abbreviations: ABS, 2-aminobenzenesulfonate; ABSDOS, 2-ABS dioxygenase system; BS, benzenesulfonate; 3SCDO, 3-sulfocatechol dioxygenase; TS, 4-toluenesulfonate.  
The GenBank accession number for the sequence reported in this paper is AF109074.

J. MAMPEL and OTHERS

ABSDOS                                                                                     3SCDO

![Diagram](https://i.imgur.com/yourimageurl.png)

Fig. 1. Initial steps in the degradation of ABS by Alcaligenes sp. strain O-1. The overall reactions of ABSDOS and 3SCDO are each considered to include spontaneous step(s). Each conversion is stoichiometric (Junker et al., 1994a, b).

for the degradation of ABS are encoded on plasmid pSAH, one of the two known plasmids in this organism (Jahnke et al., 1990, 1993).

The degradation of ABS in strain O-1 requires an undefined transport system (Thurnheer et al., 1990), ABSDOS and 3-sulfocatechol dioxygenase (3SCDO) (Fig. 1) (Junker et al., 1994b). The unstable intermediate in the reaction catalysed by ABSDOS (Fig. 1) indicates that the overall reaction consists of a specific oxygenation followed by a spontaneous loss of the amino group with concomitant regeneration of the aromatic ring. This meta ring cleavage is also considered to involve an unstable intermediate (Fig. 1), which is subject to spontaneous hydrolysis and rearrangement to yield the standard intermediate in the meta pathway, (Z,E)-2-hydroxymuconate (Fig. 1) (Junker et al., 1994a; Whitman et al., 1991). ABSDOS can desulfonate BS and TS quantitatively (Junker et al., 1994a; Thurnheer et al., 1990), as well as deaminate ABS (Fig. 1), so we wished to find out more about this enzyme.

We now confirm that ABSDOS is a two-component ring-hydroxylation system, whose oxygenase component has been characterized and sequenced, and the enzyme has been attributed to a novel branch of the class IB oxygenases (cf. Butler & Mason, 1997).

### METHODS

#### Materials. Chemicals were of the highest purity available commercially and they were purchased from Fluka, Roth, Merck, Serva or Sigma. Sulfonated aromatic compounds were from Tokyo Chemical Industries except naphthalene-1-sulfonate (Fluka). 3-Sulfocatechol was derived from previous work and its purity was determined enzymically and photometrically ($\varepsilon_{286} = 5100 \, \text{M}^{-1} \, \text{cm}^{-1}$; Junker et al., 1994a) to be 35%. Commercial columns for enzyme separations, except the 6 ml Resource Q column (Pharmacia), were described elsewhere (Junker et al., 1994b; Locher et al., 1991; Schläfli et al., 1994).

#### Organisms, growth, harvesting of cells and preparation of cell-free extracts. Alcaligenes sp. strain O-1 (DSM 6325) was isolated and identified as described elsewhere (Busse & Auling, 1992; Busse et al., 1989; Thurnheer et al., 1986). Caution: the German Central Committee for Biological Safety, ZKBS, has declared this organism pathogenic, despite the evidence from three countries, including Germany, that no problem ever arose in research, advanced or freshman teaching labs. Containment level L2 was enforced in this work. Strain O-1 harbours plasmid pSAH, and a pSAH-cured strain, Cur30 (DSM 6326), is available (Jahnke et al., 1990) which we also used.

Strain O-1 was grown at $30^\circ \text{C}$ in a 12:5 l fermenter with a 9 l working volume (Biostat V, B. Braun) in $6 \, \text{mM}$ ABS-salts medium described previously (Thurnheer et al., 1986), harvested at an $\text{OD}_{500}$ of 0.8–0.9 under conditions of optimal enzyme yield (see Results) in a Pallikon cassette filtration system (Millipore), washed in $50 \, \text{mM}$ potassium phosphate buffer, pH 7.5, by centrifugation and stored frozen. Cell-free extracts free of nucleic acids (Junker et al., 1994b) could be stored for several weeks at $-20^\circ \text{C}$ without significant loss of activity.

#### Enzyme assays. Enzyme assays for ABSDOS involving oxygen uptake in an oxygen electrode have been described elsewhere (Junker et al., 1994b). The putative reductase component (AbsB) of ABSDOS was assayed as a cytochrome c reductase (Locher et al., 1991). ABSDOS assays with separated components contained 2 vols DEAE reductase fraction to 1 vol. oxygenase fraction.

#### Purification of ABSDOS under an air atmosphere. After separation of ABSDOS components, oxygenase AbsA was sensitive to oxygen, so all solutions were sparged with oxygen-free nitrogen prior to use and protected by a blanket of nitrogen. Oxygenase- and reductase-containing fractions tended to precipitate after freezing and thawing, so we added glycerol to a final concentration of 20% (v/v) to fractions before freezing.

FPLC was done at room temperature with Pharmacia apparatus. The aerobic protocol was a three-step chromatographic procedure. After every step, fractions containing significant activity were combined, concentrated and desalted by membrane filtration (10 kDa exclusion limit, Diaflo, Amicon) in a stirring cell (model 8050, Amicon).

Step 1. The DEAE column was used as described by Junker et al. (1994b); it yielded active proteins.

Step 2. A commercial anion exchange column (Mono Q, HR 10/10) was routinely equilibrated with $20 \, \text{mM}$ Tris-sulfate, pH 7.5, $0.1 \, \text{mM}$ DTT (buffer A) at $4 \, \text{ml min}^{-1}$. Pooled samples of AbsA or AbsB from step 1 were loaded on to the column and $4 \, \text{ml}$ fractions were collected. After washing with $60 \, \text{ml}$ buffer A, a gradient to 20% buffer B ($20 \, \text{mM}$ Tris-sulfate, pH

7·5, 1M Na₂SO₄, 0·1 mM DTT) was applied over the next 200 ml and then ramped to 100% at 320 ml total elution volume. AbsA-containing fractions were routinely recognized by their characteristic red colour and their identity was confirmed in ABSDOS assays with saturating amounts of crude component AbsB from step 1. AbsB was tentatively identified as a cytochrome c reductase and its identity was confirmed in the coupled oxygenase assay. ABSDOS components eluted between 64 and 72 mM Na₂SO₄ for AbsA and between 72 and 76 mM Na₂SO₄ for cytochrome c reductase.

Step 3. Hydrophobic interaction chromatography was done with a phenyl Superose column, which was equilibrated at a flow rate of 0·15 ml min⁻¹ with 20 mM Tris-sulfate, pH 7·5, 1·7 M (NH₄)₂SO₄, 0·1 mM DTT (buffer C). Combined fractions of AbsA were brought to 1·7 M (NH₄)₂SO₄ by addition of 3 M (NH₄)₂SO₄. Fractions of 0·75 ml were collected. After 6 ml buffer C was added to remove unbound protein from the column, a linear gradient to 0 M (NH₄)₂SO₄ over 15 ml was applied by ramping buffer A. Buffer A was rinsed through the column for another 5 ml to clear the column of remaining proteins. AbsA eluted between 200 and 0 mM (NH₄)₂SO₄ but showed no ABSDOS activity. Cytochrome c reductase did not bind to the column.

Purification of ABSDOS under strictly anoxic conditions. Losses of activity caused by exposure to air, and to freezing and thawing, led us to modify the protocol to complete the procedure in an anoxic glove box (N₂/H₂, 90:10, by vol.) in 1 d without freezing steps.

Step 1a. The Mono Q column was equilibrated with buffer D (50 mM Tris/HCl, pH 7·5, 0·1 mM DTT) at a flow rate of 3 ml min⁻¹. Cell-free extract was applied to the column and after rinsing unbound protein from the column (7·5 ml of buffer), proteins were eluted by starting a linear gradient to 60% buffer E (1 M Tris/HCl, pH 7·5, 0·1 mM DTT) (at 52·5 ml eluate). Residual proteins were removed by increasing buffer E to 100%. Fractions of 1·5 ml were collected. We observed the same elution profile as with DEAE Sepharose. The oxygenase component eluted at 140–180 mM Tris/HCl, cytochrome c reductase at 240–295 mM.

Step 2a. Hydrophobic interaction chromatography of concentrated AbsA from step 1a was done as in step 3 (above).

Step 2b. The AbsB fractions from step 1a were combined, concentrated and subjected to chromatography on a Resource Q column equilibrated with buffer F (20 mM MES buffer, pH 6·0, 0·1 mM DTT) at a flow rate of 2 ml min⁻¹ and 1 ml samples were collected. Buffer F was then pumped for 5 min and proteins were eluted by ramping buffer G (20 mM MES, pH 6·0, 1 M Na₂SO₄, 0·1 mM DTT) to 20% over 7·5 min. Buffer G was brought to 100%, at 40 ml total elution volume, to remove all proteins from the column. Most proteins eluted in one region (300–350 mM Na₂SO₄).

Analytical methods. Absorbance and OD₅₀₀ were measured in a Uvikon 922 spectrophotometer (Kontron). Reverse-phase HPLC has been described previously (Laue et al., 1996; Locher et al., 1991; Thurnheer et al., 1986). SDS-PAGE was done as described by Laemmli (1970) and Locher et al. (1991) to monitor protein purification and to estimate molecular masses under protein-denaturing conditions. Values for the molecular mass of native proteins were assayed by gel filtration chromatography on a Superose 12 column by using 50 mM Tris/HCl buffer, pH 7·5, containing 150 mM NaCl (Locher et al., 1991) at a flow rate of 0·4 ml min⁻¹. Flavin cofactors were extracted from protein by boiling samples for 5 min. The precipitated protein was removed by centrifugation and the supernatant was analysed by HPLC. The iron content of purified AbsA (step 3) was determined in triplicate by atomic absorption spectroscopy (model 3030-B, Perkin-Elmer); acid-washed glassware was used throughout. Samples were adjusted to 100–200 µg Fe ml⁻¹ with iron-free 50 mM Tris/HCl buffer, pH 7·5. Samples of protein-free buffer were used as negative controls. Inorganic sulfur was extracted from purified AbsA (step 3) by zinc acetate treatment and determined by the formation of methylene blue (Beinert, 1983). Ferredoxin from *Spinacia oleracea* (Fluka) was used as a reference for efficiency of extraction. N-terminal amino acid sequences of blotted proteins (step 3) were determined after Edman degradation, as indicated previously (Schläfli et al., 1994). Protein from whole cells was assayed in a Lowry-type assay (Kennedy & Fewson, 1968); protein in cell-free extracts and purifications was assayed by the method of Bradford (1976).

Amplification, nucleotide sequencing and sequence analysis of the *absAα* gene. Degenerate primer pairs for PCR amplification were deduced from the N-terminal amino acid sequences of AbsAα and AbsAβ; the successful pair was: Aα, 5′-GARTTYYTIAARCCICARAAYGT-3′ and Aβ, 5′-SWRTCRAAYTGYTCYTGRITTIARIARRTC-3′. PCR was done with total DNA from strain O-1 and from mutant Cur30 (as a negative control) using the Expand Long Template PCR System (Boehringer Mannheim). The nucleotide sequence of the 1·3 kb PCR product was determined by cycle sequencing and primer walking using the ABI PRISM BigDye Terminator Cycle Sequencing Ready Reaction Kit and an ABI 377 DNA sequencer (GATC GmbH). Sequence analysis was done using the DNASTAR Lasergene program package. The NCBI BLAST programs were used to search for similarities to the obtained sequences (Altschul et al., 1997). Multiple sequence alignments were carried out using the CLUSTAL method with a pairwise progress of MEGALIGN from the DNASTAR Lasergene program package.

Preparation of total DNA and Southern blot hybridization. Total DNA from 100 ml of stationary-phase culture of 6 mM ABS-grown strain O-1 or 10 mM-succinate-grown mutant Cur30 was prepared by the cetyltrimethylammonium bromide precipitation method (Ausubel et al., 1987). Southern blot hybridization was done against total DNA blotted on a nylon membrane (Hybond-N; Amersham). The gene probe, an 801 bp DNA fragment, was generated by PCR using the following primer pair: 5′-GGTGAGCTCACCGGAGCACC-3′ (second quarter of *absAα*) and 5′-CCCACTCTCGATA GAACGTTCTC-3′ (*absAα*, 3′ end). The probe was labelled with digoxigenin, hybridized at 68 °C and detected by luminescence according to the manufacturer’s protocols (DIG DNA labelling Kit, DIG DNA Luminescent Detection Kit; Boehringer Mannheim).

---

**RESULTS**

Specific activity of ABSDOS as a function of the growth phase

Strain O-1 grew exponentially in ABS-salts medium in the fermenter (Fig. 2). Growth was concomitant with substrate utilization and product formation (ammonia and sulfur oxyanions; data not shown), and the growth rate (*μ* = 0·09 h⁻¹) and growth yield [6·5 g protein (mol C)⁻¹] resembled those in the initial report [*μ* = 0·1 h⁻¹ and 5·6 g protein (mol C)⁻¹ in shake flasks; Thurnheer et al., 1986]. We could detect no ABSDOS activity in whole

J. MAMPEL and OTHERS

![Graph](https://i.imgur.com/1234567.png)

**Fig. 2. Growth of Alcaligenes sp. strain O-1 in 6 mM ABS-salts medium and the ABS-dependent uptake of oxygen by whole cells.** The culture was inoculated with exponentially growing cells. Samples were taken at intervals to assay turbidity to determine the concentrations of protein, substrate and products, and to measure the oxygen uptake of whole cells. The samples (1–10 ml) for oxygen uptake were centrifuged (10000 g, 1 min, about 20 °C) and the supernatant fluid was discarded; the cells were resuspended immediately in 0·5 ml of 50 mM Tris/HCl buffer, pH 7·5, and washed (10000 g, 1 min, about 20 °C). The cells were resuspended in fresh Tris/HCl buffer and examined immediately for ABS-dependent oxygen uptake. •, Growth (OD<sub>500</sub>); ▲, specific oxygen uptake rate.

Nine compounds were determined to be substrates for oxygenation by cell-free extracts of ABS-grown cells, because substrate disappearance and oxygen uptake were observed; there was no reaction with extracts from succinate-grown cells. The compounds were ABS, BS, TS, 2-nitrobenzenesulfonate, 3- and 4-aminobenzene sulfonates, 4-chloro- and 4-hydroxybenzenesulfonates and pyridine-3-sulfonate. *K*<sub>m</sub> values (29, 18 and 108 μM) and *V*<sub>max</sub> values (140, 118 and 72 pkat) were measured for ABS, BS and TS, respectively, based on Lineweaver-Burke plots; the affinities for the other six compounds were lower than that for TS and the activity was low. Naphthalene-1-sulfonate was not attacked but caused oxygen uptake. Many other compounds (21 in total), such as the carboxy analogues of the substrates, other naphthalenesulfonates and aliphatic sulfonates, caused no oxygen uptake. When examined stoichiometrically, each substrate caused consumption of about 2 mol O<sub>2</sub> mol<sup>−1</sup>, which suggested 1 mol for the attack by ABSDOS and 1 mol for ring cleavage. Ring cleavage after oxygenation of at least BS, TS (as previously observed by Thurnheer *et al.*, 1990) and 4-chlorobenzenesulfonate was confirmed by observing a yellow ring-cleavage product; the ring cleavage product of 3-sulfocatechol (2-hydroxymuconate, Fig. 1) is colourless (Junker *et al.*, 1994a).

---

### Purification and physical properties of the oxygenase component (AbsAαAβ) of ABSDOS

The DEAE-Sepharose anion exchanger column separates ABSDOS into two fractions, which were identified as the red/brown-coloured oxygenase component A (AbsA) and the yellow-coloured reductase fraction B (AbsB) (Junker *et al.*, 1994b). AbsA was purified to near homogeneity (Fig. 3) in two additional chromatographic steps, a second anion exchanger (Mono Q) and a hydrophobic interaction column (see Methods). This preparation was catalytically inactive, but was reasonably easily available and was confirmed to be the correct protein (see below). Some 10% of the original protein was present in the purified protein, so we presume that AbsA comprises about 10% of the soluble cell protein, in agreement with SDS-PAGE gels (Fig. 3).

Under denaturing conditions, AbsA was seen to comprise two major bands, which we presumed to be subunits, termed α (45 ± 0·7 kDa) and β (16 ± 1 kDa); we attributed two minor bands to impurities (Fig. 3) rather than degradation products, which were detected at 38 kDa if samples were not worked up immediately. The native protein was examined by gel filtration chromatography and data from the single symmetrical peak (not shown), interpolated in a standard curve, indicated a molecular mass of 134 ± 12 kDa. Given the unique microprotein sequences for each subunit (see below), we presume the native enzyme to have an α<sub>2</sub>β<sub>2</sub> structure.

The N-terminal amino acid sequence of each subunit of the oxygenase component was determined. Each yielded a unique sequence. The first 13 aa in AbsAα were

---

### ABSDOS in cell-free extracts

The specific activity of ABSDOS in crude extract was observed to be a linear function of the protein concentration (Thurnheer *et al.*, 1990), but, after showing that two fractions were necessary for activity (Junker *et al.*, 1994b), we have now found that the addition of separated reductase to the crude extract gives a specific activity [about 0·24 mkat (kg protein)<sup>−1</sup>] of the oxygenase component independent of protein concentration. The specific activity of the reductase fraction [4·3 kat (kg protein)<sup>−1</sup> with cytochrome c] was independent of protein concentration.

The oxygenase component had a sharp pH optimum at pH 7·5, whereas the reductase had a broad optimum from pH 7·0 to 9·0; the temperature optima were 35 and 40 °C, respectively. Cell extracts on ice suffered a 50% loss in activity of ABSDOS under air in 24 h, which was attributed to the oxygenase component, because the reductase was stable in the cell-free extract. The loss was reduced to 15% under an atmosphere of nitrogen. Active proteins could be purified, if at all, only under anoxic conditions.

Orthanilate dioxygenase system (deaminating)

Fig. 3. SDS-PAGE of fractions of proteins from Alcaligenes sp. strain O-1 during purification of AbsA. Proteins (12–15 μg per lane) were stained with Coomassie blue in 12% gels. Lanes: 1–4, aerobic purification (1, crude extract; 2, DEAE fractions; 3, Mono Q fractions; 4, hydrophobic interaction fractions); 6–7, anaerobic purification (6, MonoQ fractions; 7, hydrophobic interaction fractions); 5 and 8, standard protein markers.

genase component of the benzoate dioxygenase system [maxima in the oxidized form at 464 and 560 nm (shoulder), and in the reduced state at 518 nm] (Yamaguchi & Fujisawa, 1982). We thus had preliminary evidence for a Rieske [2Fe–2S] centre. The iron content of AbsA was 1·9 mol Fe per α-subunit and the acid-labile sulfur, corrected for extraction efficiency, was also 1·9 mol per α-subunit; these values are the means of two independent experiments. These data support the presence of a Rieske centre.

### AbsAαAβ with catalytic activity

A rapid, two-step purification under anoxic conditions yielded preparations of about 90 % purity (Fig. 3) which retained some activity when coupled with the reductase fraction (Table 1). This confirmed that the inactive material examined above represented the oxygenase component.

All compounds which were substrates in crude extract (see above) caused oxygen uptake in these preparations. Naphthalene-1-sulfonate, which caused uncoupling of oxygen consumption and oxygenation in the crude extract, showed the same behaviour with the purified oxygenase. Uncoupling in oxygenases has been known for some time (Hayashi, 1974) and has been observed in several multicomponent systems since 1973 (e.g. Bernhardt et al., 1973; Bünz & Cook, 1993; Schläfli et al., 1994).

The purified enzyme was active without added iron. This activity suffered an 82 % inhibition if the enzyme was preincubated in the presence of 2 mM o-phenanthroline, an iron-specific chelator, but the inhibition was partly relieved on the addition of 2·5 mM FeCl₃. We interpret these data as indicating the presence of a binding site for mononuclear iron, the site of oxygen activation (Butler & Mason, 1997). The iron is presumably only loosely bound because it is not seen in the total iron analysis; the mononuclear iron in many purified multicomponent oxygenases is known to be weakly bound and can be readily replaced by other cations for spectral studies (e.g. Bertini et al., 1996).

Fig. 4. Absorption spectra of the oxygenase component of ABSDOS. Purified AbsA (2·3 mg ml⁻¹) in (A) oxic 20 mM Tris-sulfate buffer, pH 7·5, and (B) after reduction with 43 μmol ml⁻¹ sodium dithionite in the presence of 0·42 mg ml⁻¹ of reductase and 0·28 mM NADH, all of which were necessary.

determined as Ser-Arg-Ser-Ala-Ala-Glu-Phe-Leu-Lys-Pro-Gln-Asn-Val-; the first 28 aa in AbsAβ were Met-Asp-Thr-Val-Ser-Ile-Ala-Glu-Phe-Leu-Trp-Thr-Asn-Ala-Asp-Leu-Leu-Asn-Gln-Glu-Gln-Phe-Asp-Ser-Gly-Leu-Glu-Gln-.

The UV-visible spectrum of AbsA (Fig. 4) gave absorption maxima which resembled those of the oxy-

J. MAMPEL and OTHERS

Table 1. Separation of the oxygenase component, AbsA, of ABSDOS under strictly anoxic conditions

| Purification step | Oxygenase AbsA |
|------------------|----------------|
|                  | Total          | Total         | Specific       | Yield   | Purification    |
|                  | activity       | protein       | activity       | (%)     | (-fold)         |
|                  | (nkat)         | (mg)          | [mkat(kg      |         |                 |
|                  |                |               | protein)]     |         |                 |
| Cell-free extract | 120            | 600           | 0·2            | 100     | 1·0             |
| Mono Q           | 30             | 20            | 1·5            | 25      | 7·0             |
| HIC              | 2              | 5             | 0·4            | 2       | 2·2             |

Table 2. Separation of the reductase component, AbsB, of ABSDOS under strictly anoxic conditions

| Purification step | Reductase AbsB |
|------------------|----------------|
|                  | Total          | Total         | Specific       | Yield   | Purification    |
|                  | activity       | protein       | activity       | (%)     | (-fold)         |
|                  | (mkat)         | (mg)          | [kat(kg       |         |                 |
|                  |                |               | protein)]     |         |                 |
| Cell-free extract | 5·3            | 880           | 6              | 100     | 1               |
| Mono Q           | 0·7            | 22            | 32             | 13      | 5               |
| Resource Q       | 0·3            | 4·2           | 71             | 6       | 11              |

Properties of the reductase fraction

The protein(s) responsible for the reductase activity could be separated easily (Table 2), but with high loss of activity in fractions which contained many proteins. None of these could be attributed to the reductase, whose activity was lost on further separation. We never observed any small proteins (≤12kDa) in SDS-PAGE gels of these fractions (cf. Butler & Mason, 1997), so we presumed that we were dealing with a single-component reductase which we termed AbsB. The only cytochrome c reductase activity in the extract co-eluted with AbsB (measured in the coupled oxygenase test) and this activity was tentatively considered to represent AbsB. All active fractions of AbsB were yellow, more so in the anoxic separations (Table 2). We extracted this yellow colour, which co-eluted from an HPLC column with FAD, and whose UV-visible spectrum was identical with that of FAD. The FAD was released from the protein by boiling, so it was presumably non-covalently bound to the enzyme, and we tended to separate flavin from protein in some columns, e.g. Mono Q.

We obtained a PCR fragment of about 1·3kb, which corresponded to the molecular mass of the AbsAα subunit of about 45kDa (Fig. 3) and which indicated contiguous absAαAβ genes. The DNA sequence of the PCR fragment was determined for both strands by cycle sequencing. The sequence showed the third base of the codon for the 13th known N-terminal amino acid of AbsAα to be a cytosine, contiguous with the primer sequence, and cycle sequencing gave a specific sequence for a region initially represented by the degenerate primer. Thus the newly determined, specific sequence started within the N-terminal primer sequence and continued through the absAα stop codon to the first 14 codons of absAβ, the first of which was located 15bp downstream of absAα. We thus had an overlap with the N-terminal amino acid sequences of both AbsAα and AbsAβ. The sequence of 393aa of AbsAα is thus known, though an N-terminal methionine or methionyl peptide has presumably been cleaved off. The calculated molecular mass (45·5kDa) of the known sequence corresponded with the observed value (Fig. 3).

We could identify two motifs in the deduced amino acid sequence, a Rieske [2Fe–2S] centre (Fig. 5a; cf. Fig. 4) and a mononuclear Fe(II)-binding site (Fig. 5b).

Sequence of the absAα gene and identification of conserved motifs

Degenerate primer pairs for PCR were deduced to give an amplified DNA fragment comprising the absAα gene and some bases of the 5′end of the absAβ gene, assuming the latter gene is located not far downstream of absAα.

Localization of the absAα gene

We generated a gene probe for absAα by PCR: it comprised a fragment of 801bp representing 75% of the gene towards the 3′end. Samples of total DNA from

(a)

| AbsAα | (79–113) | TFVNACSHRG IQIVRRPRGN AKTMECIFHR WNYDS |
| --- | --- | --- |
| AntA | (88–122) | AMVNACEHRG ATLTRVAKGN QSVFTCPFHA WCYKS |
| XylX | (85–121) | AFVNACSHRG ATLCRFRSGN KATHTCSFHG WTFSN |
| BenA | (90–124) | AMINACSHRG AQLLGHKRGN KTTYTCPFHG WTFNN |
| CbdA | (93–127) | AVINACSHRG AELCRRKQGN RSTFTCQFHG WTFSN |

(b)

| AbsAα | (194–218) | TNWKN WQETNLDLYHEFM-HF ANRKT |
| --- | --- | --- |
| AntA | (206–231) | GNWKL QNENGLDGYHVSTVHY NYVST |
| XylX | (207–232) | GNWKV QVENGADGYHVSTVHW NYAAT |
| BenA | (210–235) | GNWKL TAENGADGYHVASVHW NYAAT |
| CbdA | (213–238) | GNWKL QIENGADGYHGVSVHW NYVAT |

(c)

![Diagram](attachment:phylogenetic_tree.png)

**Fig. 5.** Alignments of conserved Rieske [2Fe–2S] (a) and mononuclear Fe(II) (b) binding domains of aromatic oxygenase components (α-subunit) of enzymes in EC 1.14.12.-, and an unbalanced dendrogram (c) of 16 α-subunits. (a, b) Regions of AbsAα, XylX, BenA and CbdA were aligned using the CLUSTAL method of the DNASTAR program MEGALIGN. Identical amino acids are shown in bold letters, the iron-binding amino acids are marked with an asterisk and a dash indicates a gap introduced into the alignment. (c) Complete amino acid sequences were aligned prior to drawing the phylogenetic tree. The lengths of the branches indicate the relative phylogenetic distances between the amino acid sequences, so related oxygenases are clustered. Each (sub)cluster bears the appropriate nomenclature from Batie et al. (1992). AbsA, ABS dioxygenase (AF109074); AntA, anthranilate dioxygenase (AF071556; Bundy et al., 1998); BenA, benzoate dioxygenase (P07769; Neidle et al., 1991); BnzA, benzene dioxygenase (M17904; Irie et al., 1987); BphA1, biphenyl dioxygenase (M83673; Taira et al., 1992); CbaA, chlorobenzoate dioxygenase (U00692; Nakatsu et al., 1995); CbdA, 2-halobenzoate dioxygenase (X79076; Haak et al., 1995); DntAc, 2,4-dinitrotoluene dioxygenase (U62430; Suen et al., 1996); NahAcPp, naphthalene dioxygenase Fe–S large subunit (M83949; Simon et al., 1993); NdoB, naphthalene dioxygenase (M23914; Kurkela et al., 1988); Pht3, phthalate dioxygenase (D13229; Nomura et al., 1992); PobA, phenoxybenzoate dioxygenase (X78823; Dehmel et al., 1995); TodC1, toluene dioxygenase (J04996; Zylstra & Gibson, 1989); TsaM, 4-tolylsulfonate monooxygenase (U32622; Junker et al., 1997); VanA, vanillate demethylase (Y14759; Venturi et al., 1998); XylX, benzoate dioxygenase (PIR A41659; Harayama et al., 1991). All sequences are from GenBank, unless otherwise indicated.

Alcaligenes sp. strain O-1 and from mutant Cur30, which lacks plasmid pSAH and the ABS-mineralization phenotype, were transferred by a slot-blot device in increasing amounts in parallel rows to a nylon membrane. Southern hybridization with the DNA probe (not shown) showed a strong signal for DNA from strain O-1 even at the lowest concentration of genomic DNA, but no signal for Cur30 DNA. This confirms the earlier observation that ABSDOS is plasmid-encoded (Jahnke *et al.*, 1990).

---

### DISCUSSION

We have confirmed our earlier observations (Junker *et al.*, 1994b; Thurnheer *et al.*, 1986) that ABSDOS is an inducible enzyme system whose specific activity in growing cells is about 0·63 mkat (kg protein)^−1^ [μ = 0·09 h^−1^, Y = 6·5 g protein (mol C)^−1^]. The specific activity we assayed in crude extracts [0·24 mkat (kg protein)^−1^], is some 38% of the value required by growing cells and the separated oxygenase has the catalytic properties attributed to the enzyme, so we are convinced that we have worked on the correct proteins. The gene (*absA*) encoding one of these proteins has been confirmed by Southern blot analyses to be present in wild-type Alcaligenes sp. strain O-1 but absent in the cured strain Cur30. This corresponds to published genetic data (Jahnke *et al.*, 1990), so our new data support established work and allow us to expand on earlier data.

We consider ABSDOS to be a two-component oxygenase system, comprising reductase AbsB, which we failed to purify, and oxygenase AbsAαAβ, which we have characterized biochemically together with the sequence of the *absAα* gene. We attribute this enzyme system to class IB in the classification of Batie *et al.* (1992). Our data for the reductase show (i) a single protein, because specific activity is independent of the amount of protein in the assay, in which cytochrome c is reduced with high activity, which implies (ii) that a plant-type ferredoxin module is present in the protein (cf. Locher *et al.*, 1991), and (iii) the only flavin chromophore which co-elutes with the cytochrome c reduction activity is FAD. These properties are representative of class IB reductases (Batie *et al.*, 1992). The oxygenase component with its α₂β₂ structure, a Rieske [2Fe–2S] centre (Fig. 4, Fig. 5a) and a mononuclear-iron-binding site (Fig. 5b), in that order in the α-subunit, is also representative of class IB (Batie *et al.*, 1992). ABSDOS thus belongs to EC 1.14.12.- (e.g. Butler & Mason, 1997) with the trivial name 2-aminobenzene-sulfonate 2,3-dioxygenase system and the systematic name 2-aminobenzenesulfonate,NADH:oxygen oxidoreductase (2,3-hydroxylating, ammonia-forming). The system can be further defined as follows: iron-sulfur flavoprotein (FAD) reductase; no independent ferredoxin; heteromultimeric iron-sulfur oxygenase.

The sequence of *absA* indicates that this enzyme is not a typical class IB enzyme. This is initially visible in the structure of the mononuclear-iron-binding site (Fig. 5b), where the two iron-binding histidines are separated by only three amino acids, whereas the proteins used in alignments have four, as do most others (Butler & Mason, 1997), including the crystallized naphthalene

J. MAMPEL and OTHERS

dioxygenase oxygenase (Kauppi *et al.*, 1998). However, mononuclear-iron-binding sites with three amino acids in this position can be found in the databases, e.g. ORF H1 in *Sphingomonas* sp. strain RW1 (accession no. AJ223220; Armengaud *et al.*, 1998) and *msmA* encoding methanesulfonate monooxygenase (large subunit) in *Methylomonas methylovora* (accession no. AF-091716; de Marco *et al.*, 1999), but only AF091716 encodes a protein of known function (de Marco *et al.*, 1999). This unusual mononuclear-iron-binding site, together with the divergence round the [2Fe–2S] centre (Fig. 5a), led us to compare AbsAα with other oxygenase α-subunit proteins (Fig. 5c). The well-represented classes (IA, IB, IIB, III), suggested by Batie *et al.* (1992) for the reductase component(s), are readily visible (there are very few characterized class IIA systems). However, although there is a clear division between class IA (FMN-reductases, two-component) and the other three classes, there is as much divergence within class IA, between the monooxygenases (e.g. TsaM) and the dioxygenases (e.g. PobA), as there is in the other groups between class IB (FAD reductases, two-component) and the combined classes IIB and III (three-component systems). We have attributed AbsA to class IB and find a larger divergence between the AbsA branch and the AntA branch of this group than between classes IIB (four redox centres in system) and III (five redox centres). de Marco *et al.* (1999) point out that inclusion of their sequence (*msmA* from *M. methylovora*) in dendrograms causes significant changes in groupings and we chose to remain within the enzymes of aromatic metabolism for our comparisons (Fig. 5c).

Attribution of ABSDOS to class IB can also be deduced from the non-catalytic β-subunit. The N-terminal amino acid sequence of AbsAβ has highest levels of homology to class IB systems, e.g. 50% with BenB (not shown). The biochemical properties of ABSDOS and the sequence data for *absAαAβ* all fit class IB, but this need not be the case, as illustrated in the oxodihydroquinoline monooxygenase system, where the oxygenase is a class IA enzyme while the reductase is class IB (Rosche *et al.*, 1995).

AbsA differs from the longer known members of class IB in having an apparent α₂β₂ structure, whereas the others have an α₃β₃ structure (e.g. Yamaguchi & Fujisawa, 1982). The weight to put on this deduction is limited, because the older data are based on cross-linking experiments, whereas ours are based on gel filtration, which is known to deviate from the theoretical when the protein concerned is not spherical (e.g. le Maire *et al.*, 1996). Indeed, naphthalene dioxygenase oxygenase was long considered to have an α₂β₂ structure and was shown to be α₃β₃ only when the crystal structure became available (Kauppi *et al.*, 1998); the oxygenase was found not to be spherical but mushroom-shaped.

We presume the apparently unusual structure of the mononuclear-iron-binding site (Fig. 5b) to be integral with its function in dioxygenation. The consequence of dioxygenation, however, can be very different, depend-

ing on the substrate. 2-Aminobenzenesulfonate is deaminated, whereas benzenesulfonate is desulfonated (Junker *et al.*, 1994a, b). The difference presumably depends on the amino acids in the active site and it is odd that ABSDOS is a better BSDOS (*K*ₘ = 18 μM, *V*ₘₐₓ = 118 pkat) than it is an ABSDOS (*K*ₘ = 29 μM, *V*ₘₐₓ = 140 pkat). The regulation of enzyme induction is what gives the bacterium its specificity (Junker *et al.*, 1994b). While there would be no apparent problem for the organism to degrade catechol via the 3-sulfocatechol meta cleavage (Fig. 1), the benzenesulfonate dioxygenase system has to be prevented from misrouting ABS by desulfonating it to the unstable aminocatechol. One wonders why the organism has developed such a fine regulatory system, which we are exploring, instead of routing BS down the ABSDOS pathway. This unusual enzyme seems to be worthy of crystallizing, but its tendency to clump and precipitate, as for example when being prepared for electron paramagnetic resonance (EPR) studies (J. Mampel, unpublished data) and its instability in Konstanz (compared with its stability in another lab; cf. Thurnheer *et al.*, 1990) make it unsuitable at present.

## ACKNOWLEDGEMENTS

G. Auling, University of Hannover, kindly made strain Cur30 available to us. The research was supported by the University of Konstanz. F.J. held a studentship from the Schweizerische Nationalfonds.

## REFERENCES

Altschul, S. F., Madden, T. L., Schäffer, A. A., Zhang, J., Zhang, Z., Miller, W. & Lipman, D. J. (1997). Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. *Nucleic Acids Res* 25, 3389–3402.

Armengaud, J., Happe, B. & Timmis, K. N. (1998). Genetic analysis of dioxin dioxygenase of *Sphingomonas* sp. strain RW1: catabolic genes dispersed on the genome. *J Bacteriol* 180, 3954–3966.

Ausubel, F. M., Brent, R., Kingston, R. E., Moore, D. D., Seidman, J. G., Smith, J. A. & Struhl, K. (1987). *Current Protocols in Molecular Biology*. New York: Wiley.

Batie, C. J., Ballou, D. P. & Correll, C. C. (1992). Phthalate dioxygenase reductase and related flavin-iron-sulfur containing electron transferases. In *Chemistry and Biochemistry of Flavoenzymes*, pp. 543–556. Edited by F. Müller. Boca Raton, FL: CRC Press.

Beinert, H. (1983). Semi-micro methods for analysis of labile sulfide and of labile sulfide plus sulfane sulfur in unusually stable iron-sulfur proteins. *Anal Biochem* 131, 373–378.

Bernhardt, F.-N., Erdin, N., Staudinger, H. & Ullrich, V. (1973). Interactions of substrates with a purified 4-methoxybenzoate monooxygenase system (O-demethylating) from *Pseudomonas putida*. *Eur J Biochem* 35, 126–134.

Bertini, I., Cremonini, M. A., Ferretti, S., Lozzi, I., Luchinat, C. & Viezzoli, M. A. (1996). Arene hydroxylases: metalloenzymes catalysing dioxygenation of aromatic compounds. *Coord Chem Rev* 151, 145–160.

Bradford, M. M. (1976). A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. *Anal Biochem* 72, 248–254.

Bundy, B. M., Campbell, A. L. & Neidle, E. (1998). Similarities between antABC-encoded anthranilate dioxygenase and the benABC-encoded benzoate dioxygenase of Acinetobacter sp. strain ADP1. *J Bacteriol* 180, 4466–4474.

Bünz, P. & Cook, A. M. (1993). Dibenzofuran 4,4a-dioxygenase from Sphingomonas sp. strain RW1: angular dioxygenation by a three-component enzyme system. *J Bacteriol* 175, 6467–6475.

Busse, H.-J. & Auling, G. (1992). The genera Alcaligenes and ‘Achromobacter’. In *The Prokaryotes*, 2nd edn, pp. 2544–2555. Edited by A. Balows, H. G. Trüper, M. Dworkin, W. Harder & K.-H. Schleifer. New York: Springer.

Busse, H.-J., El-Banna, T. & Auling, G. (1989). Evaluation of different approaches for identification of xenobiotic-degrading pseudomonads. *Appl Environ Microbiol* 55, 1578–1583.

Butler, C. S. & Mason, J. R. (1997). Structure–function analysis of the bacterial aromatic ring-hydroxylating dioxygenases. *Adv Microb Physiol* 38, 47–84.

Cook, A. M., Laue, H. & Junker, F. (1998). Microbial desulfonation. *FEMS Microbiol Rev* 22, 399–419.

Dehmel, U., Engesser, K.-H., Timmis, K. N. & Dwyer, D. F. (1995). Cloning, nucleotide sequence, and expression of the gene encoding a novel dioxygenase involved in metabolism of carboxy-diphenyl ethers in Pseudomonas pseudoalcaligenes POB310. *Arch Microbiol* 163, 35–41.

Haak, B., Fetzner, S. & Lingens, F. (1995). Cloning, nucleotide sequence, and expression of the plasmid-encoded genes for the two-component 2-halobenzoate 1,2-dioxygenase from Pseudomonas cepacia 2CBS. *J Bacteriol* 177, 667–675.

Harayama, S., Rekik, M., Bairoch, A., Neidle, E. L. & Ornston, L. N. (1991). Potential DNA slippage structures acquired during evolutionary divergence of Acinetobacter calcoaceticus chromosomal benABC and Pseudomonas putida TOL pWW0 plasmid xylXYZ genes encoding benzoate dioxygenases. *J Bacteriol* 173, 7540–7548.

Hayaishi, O. (1974). *Molecular Mechanisms of Oxygen Activation*. New York: Academic Press.

Irie, S., Doi, S., Yorifuji, T., Takagi, M. & Yano, K. (1987). Nucleotide sequencing and characterization of the genes encoding benzene oxidation enzymes of Pseudomonas putida. *J Bacteriol* 169, 5174–5179.

Jahnke, M., El-Banna, T., Klintworth, R. & Auling, G. (1990). Mineralization of orthanilic acid is a plasmid-associated trait in Alcaligenes sp. O-1. *J Gen Microbiol* 136, 2241–2249.

Jahnke, M., Lehmann, F., Schoebel, A. & Auling, G. (1993). Transposition of the TOL catabolic genes (Tn4651) into the degradative plasmid pSAH of Alcaligenes sp. O-1 ensures simultaneous mineralization of sulpho- and methyl-substituted aromatics. *J Gen Microbiol* 139, 1959–1966.

Junker, F., Field, J. A., Bangerter, F., Ramsteiner, K., Kohler, H.-P., Joannou, C. L., Mason, J. R., Leisinger, T. & Cook, A. M. (1994a). Oxygenation and spontaneous deamination of 2-aminobenzene-sulphonic acid in Alcaligenes sp. strain O-1 with subsequent meta ring cleavage and spontaneous desulphonation to 2-hydroxy-muconic acid. *Biochem J* 300, 429–436.

Junker, F., Leisinger, T. & Cook, A. M. (1994b). 3-Sulphocatechol 2,3-dioxygenase and other dioxygenases (EC 1.13.11.2 and EC 1.14.12.-) in the degradative pathways of 2-aminobenzene-sulphonic, benzenesulphonic and 4-toluenesulphonic acids in Alcaligenes sp. strain O-1. *Microbiology* 140, 1713–1722.

Junker, F., Kiewitz, R. & Cook, A. M. (1997). Characterization of the p-toluenesulfonate operon tsaMBCD and tsaR in Comamonas testosteronei T-2. *J Bacteriol* 179, 919–927.

Kauppi, B., Lee, K., Carredano, F., Parales, R. E., Gibson, D. T., Eklund, H. & Ramaswamy, S. (1998). Structure of an aromatic-ring-hydroxylating dioxygenase – naphthalene 1,2-dioxygenase. *Structure* 6, 571–586.

Kennedy, S. I. T. & Fewson, C. A. (1968). Enzymes of the mandelate pathway in bacterium NCIB 8250. *Biochem J* 107, 497–506.

Kurkela, S., Lehvaslaiho, H., Palva, E. T. & Teeri, T. H. (1988). Cloning, nucleotide sequence and characterization of genes encoding naphthalene dioxygenase of Pseudomonas putida strain NCIB9816. *Gene* 73, 355–362.

Laemmlli, U. K. (1970). Cleavage of structural proteins during the assembly of the head of bacteriophage T4. *Nature* 227, 680–685.

Laskin, A. I. & Lechevalier, H. A. (1984). *CRC Handbook of Microbiology*. Boca Raton, FL: CRC Press.

Laue, H., Field, J. A. & Cook, A. M. (1996). Bacterial desulfonation of the ethanesulfonate metabolite of the chloroacetanilide herbicide metazachlor. *Environ Sci Technol* 30, 1129–1132.

Locher, H. H., Leisinger, T. & Cook, A. M. (1991). 4-Sulpho-benzoate 3,4-dioxygenase: purification and properties of a desulphonative two-component enzyme system from Comamonas testosteroni T-2. *Biochem J* 274, 833–842.

le Maire, M., Ghasi, A. & Moller, J. V. (1996). Gel chromatography as an analytical tool for characterization of size and molecular mass of proteins. *ACS Symp Ser* 635, 36–51.

de Marco, P., Moradas-Ferreira, P., Higgins, T. P., McDonald, I., Kenna, E. M. & Murrell, J. C. (1999). Molecular analysis of a novel methanesulfonic acid monooxygenase from the methylotroph Methylosulfonomonas methyllovora. *J Bacteriol* 181, 2244–2251.

Nakatsu, C. H., Straus, N. A. & Wyndham, R. C. (1995). The nucleotide sequence of the Tn5271 3-chlorobenzoate 3,4-dioxygenase genes (cbaAB) unites the class IA oxygenases in a single lineage. *Microbiology* 141, 485–495.

Neidle, E. L., Hartnett, C., Ornston, L. N., Bairoch, A., Rekik, M. & Harayama, S. (1991). Nucleotide sequences of the Acinetobacter calcoaceticus benABC genes for benzoate 1,2-dioxygenase reveal evolutionary relationships among multicomponent oxygenases. *J Bacteriol* 173, 5385–5395.

Nomura, Y., Nakagawa, M., Ogawa, N., Harashima, S. & Oshima, Y. (1992). Genes in PHT encoding the initial degradation pathway of phthalate in Pseudomonas putida. *J Ferment Bioeng* 74, 333–344.

Rosche, B., Tshisuaka, B., Fetzner, S. & Lingens, F. (1995). 2-Oxo-1,2-dihydroquinoline-8-monooxygenase, a two-component enzyme system from Pseudomonas putida 86. *J Biol Chem* 270, 17836–17842.

Schläfli, H. R., Weiss, M. A., Leisinger, T. & Cook, A. M. (1994). Terephthalate 1,2-dioxygenase system from Comamonas testosteroni T-2: purification and some properties of the oxygenase component. *J Bacteriol* 176, 6644–6652.

Simon, M. J., Osslund, T. D., Saunders, R. & 7 other authors (1993). Sequences of genes encoding naphthalene dioxygenase in Pseudomonas putida strains G7 and NCIB 9816-4. *Gene* 127, 31–37.

Suen, W. C., Haigler, B. E. & Spain, J. C. (1996). 2,4-Dinitrotoluene dioxygenase from Burkholderia sp. strain DNT: similarity to naphthalene dioxygenase. *J Bacteriol* 178, 4926–4934.

Taira, K., Hirose, J., Hayashida, S. & Furukawa, K. (1992). Analysis of bph operon from the polychlorinated biphenyl-degrading strain of Pseudomonas pseudoalcaligenes KF707. *J Biol Chem* 267, 4844–4853.

J. MAMPEL and OTHERS

Thurnheer, T., Köhler, T., Cook, A. M. & Leisinger, T. (1986). Orthanilic acid and analogues as carbon sources for bacteria: growth physiology and enzymic desulphonation. *J Gen Microbiol* 132, 1215–1220.

Thurnheer, T., Zürrer, D., Höglinger, O., Leisinger, T. & Cook, A. M. (1990). Initial steps in the degradation of benzene sulfonic acid, 4-toluene sulfonic acid and orthanilic acid in Alcaligenes sp. strain O-1. *Biodegradation* 1, 55–64.

Venturi, V., Zennaro, F., Degrassi, G., Okeke, B. C. & Bruschi, C. V. (1998). Genetics of ferulic acid bioconversion to protocatechuic acid in plant-growth-promoting *Pseudomonas putida* WCS358. *Microbiology* 144, 965–973.

Whitman, C. P., Aird, B. A., Gillespie, W. R. & Stolowich, N. J. (1991). Chemical and enzymatic ketonization of 2-hydroxymuconate, a conjugated enol. *J Am Chem Soc* 113, 3154–3162.

Yamaguchi, M. & Fujisawa, H. (1982). Subunit structure of oxygenase component in benzoate-1,2-dioxygenase system from *Pseudomonas arvilla* C-1. *J Biol Chem* 257, 12497–12502.

Zylstra, G. J. & Gibson, D. T. (1989). Toluene degradation by *Pseudomonas putida* F1: nucleotide sequence of the *todC1C2-BADE* genes and their expression in *Escherichia coli*. *J Biol Chem* 264, 14940–14946.

Received 18 May 1999; revised 14 July 1999; accepted 27 July 1999.
