
Biochemical and Biophysical Research Communications 266, 322–325 (1999)

Article ID bbrc.1999.1805, available online at http://www.idealibrary.com on IDEAL®

---

**Characterization of 2,6-Dichloro-p-hydroquinone 1,2-Dioxygenase (PcpA) of Sphingomonas chlorophenolica ATCC 39723**

Luying Xun,$^{1}$ Jan Bohuslavek, and Mian Cai

Department of Microbiology, Washington State University, Pullman, Washington 99164-4233

Received October 10, 1999

---

because 2,6-DiCH is accumulated when pcpA is disrupted by homologous recombination (1). When this enzyme was initially purified, 6-chlorohydroxyquinol was identified as the enzyme product by HPLC analysis on the basis of retention time and peak spectrum (6). Thus, the enzyme was named as “DiCH chlorohydrolase” converting 2,6-DiCH to 6-chlorohydroxyquinol (6). Recently, another group repeated the purification, found that the enzyme required O₂ for activity, and reported this enzyme as a dioxygenase (18). Because they did not detect 6-chlorohydroxyquinol nor 2-chloromaleylacetate in the reaction mixture, they proposed that the dioxygenase broke the ring between the 3- and 4-positions to yield 2,4-dichloro-3-hydroxycis,cis-muconic semialdehyde which was a very labile compound (18). In order to clearly illustrate the PCP degradation pathway, we overexpressed pcpA gene in E. coli and used the overproduced PcpA to detect the end product of this enzyme as 2-chloromaleylacetate.

---

Pentachlorophenol (PCP) is a general biocide and a major environmental pollutant. The initial steps of PCP degradation by Sphingomonas chlorophenolica ATCC 39723 have been studied and characterized. Two enzymes are responsible for converting PCP to 2,6-dichloro-p-hydroquinone (2,6-DiCH) which is a common metabolic intermediate of the biodegradation of polychlorinated phenols. 2,6-DiCH is degraded by PcpA from strain ATCC 39723, but the reaction end product has been misidentified as 6-chlorohydroxyquinol and has been elusive to detection. We report here the overproduction of PcpA in Escherichia coli and the demonstration of quantitative conversion of 2,6-DiCH to 2-chloromaleylacetate with the coconsumption of one equivalent O₂ and release of one equivalent Cl⁻ by purified PcpA. On the basis of the reaction stoichiometry, the enzyme is proposed to be 2,6-DiCH 1,2-dioxygenase. © 1999 Academic Press

---

Pentachlorophenol (PCP), a highly toxic pesticide, has been widely used in disinfectants and preservatives (10). U.S. Environmental Protection Agency regulates PCP as a priority pollutant (14). Several aerobic microorganisms capable of degrading PCP have been isolated (3). Sphingomonas chlorophenolica ATCC 39723 is one of the organisms able to degrade PCP (8, 12) and it oxidizes PCP to tetrachloro-p-hydroquinone (TeCH) by PCP 4-monooxygenase (20). The second enzyme involved in PCP degradation is TeCH reductive dehalogenase that catalyzes sequential dehalogenation of TeCH to 2,3,6-trichloro-p-hydroquinone and then to 2,6-dichloro-p-hydroquinone (2,6-DiCH) with reduced glutathione as the reductant (21), and the enzyme has been characterized as a glutathione transferase (7, 9). The next enzyme involved in PCP degradation uses 2,6-DiCH as its substrate (6) and is encoded by pcpA

$^{1}$ To whom correspondence should be addressed. Fax: (509) 335-1907. E-mail: xun@mail.wsu.edu.

---

MATERIALS AND METHODS

Bacterial strains. S. chlorophenolica ATCC 39723 was provided by R. L. Crawford (University of Idaho) and was cultured in a mineral medium (12). Escherichia coli strain NovaBlue was used for pET-30 LIC cloning and strain BL21 (DE3) was used for protein expression (Novagen, Madison, WI).

Gene expression. To obtain an N-terminal fusion protein of PcpA, PCR primers were designed to clone pcpA into pET-30 LIC vector according to Novagen’s recommendations (Novagen). The first 12 bases of the forward primer (PcpaF: 5′-GAC-GAC-AAG-ATG-GAA-ACG-AAC-CAT-ATC-ACC-AGT-C-3′) were the LIC overhang and followed by the first 25 bases (at base position 194 to 219) of the pcpA gene (GenBank: M55159), in which the starting codon was converted from GTG to ATG. The reverse primer (PcpaR: 5′-GAG-GAG-AAG-CCC-GGT-GCC-TTC-GCC-AAC-AAG-ATC-3′) was located at position 1652 to 1635 with the 15 bases of LIC overhang at the 5′ end. The gene was amplified with the two primers and genomic DNA of strain ATCC 39723 by 30 cycles of PCR (94°C for 30 sec, 60°C for 30 sec, and 72°C for 60 sec). Only a single 1.5 kilobase product was obtained and cloned into pET-30 LIC by the ligation independent cloning method of Novagen to obtain plasmid pMM. pMM was transformed into E. coli strain NovaBlue for plasmid identification and recovery, and then the recovered plasmid DNA was transformed

0006-291X/99 $30.00

Copyright © 1999 by Academic Press  
All rights of reproduction in any form reserved.

Vol. 266, No. 2, 1999

BIOCHEMICAL AND BIOPHYSICAL RESEARCH COMMUNICATIONS

into *E. coli* strain BL21 (DE3) for protein production upon induction by IPTG (Novagen).

For the construction of nonfusion PcpA, one new forward primer and one reverse primer were designed. The forward primer (CF1: 5′-GAC-GAC-GAC-CAT-ATG-GAA-ACG-AAC-3′) contained the modified start codon ATG of *pcpA* on pMM and an introduced *NdeI* site (noted by the underlined sequence in the primer). The reverse primer (CR1: 5′-CCA-AGG-GGT-TAT-GCT-AGT-TAT-TGC-3′) was part of the T7 terminator on the pET-30 LIC vector. The gene was amplified with primers CF1 and CR1 by using pMM plasmid DNA as template for 30 PCR cycles (94°C for 30 sec, 60°C for 30 sec, and 72°C for 60 sec). The PCR product was digested by *NdeI* and *BamHI* and ligated into pET-30 LIC vector previously digested by *NdeI* and *BamHI*. The resulting plasmid pCR1 was transformed into strain NovaBlue for plasmid recovery and then transformed into strain BL21 (DE3) for gene expression.

**PcpA purification.** Strain BL21 (DE3) containing pCR1 in 200 ml of LB medium was grown to an optical density of 0.4 at 600 nm at 37°C. IPTG was added to 1 mM and the culture was incubated at room temperature (24°C) for 4 hours to produce active PcpA. The cells were harvested by centrifugation and washed once with 25 ml of 20 mM potassium phosphate (KPi) buffer (pH 7) containing 1 mM EDTA. PcpA was purified as previously reported (6) but with fewer steps for the overproduced PcpA. Briefly, the washed cells were lysed by passing through a French pressure cell (Model FA-030; Aminco, Urbana, IL) once at 18,000 psi. Nucleic acids were precipitated by protamine sulfate. Then the sample was directly loaded onto a 5-ml EconoQ column (Bio-Rad, Hercules, CA) and eluted off the column with a 50 ml linear gradient of 100 to 400 mM NaCl in 20 mM KPi (pH 7) with 1 mM dithiothreitol. The main peak around 250 mM NaCl was collected, concentrated and injected onto a Superdex 75 (gel filtration) column (Pharmacia, Piscataway, NJ) eluted with 20 mM KPi buffer (pH 7) with 150 mM NaCl and 1 mM dithiothreitol to obtain PcpA to apparent homogeneity. Protein concentrations were determined with a protein dye reagent (Bio-Rad, Hercules, CA) with bovine serum albumin as a standard.

**Enzyme assays.** The assay for 2,6-DiCH degradation was the same as reported before (8) and ascorbic acid was used to reduce dichloro-*p*-benzoquinone to 2,6-DiCH. A new UV/vis spectral assay was developed to monitor the decrease of 2,6-DiCH and the increase of 2-chloromaleylacetate. Two hundred μM dichloro-*p*-benzoquinone in 20 mM KPi buffer (pH 6.3 to 7.8) was reduced to 2,6-DiCH by 1.2 mM NaBH₄ (100 mM stock freshly prepared in distilled water). Enzyme and Fe²⁺ were added to initiate the reaction. 2-Chloromaleylacetate production was measured by following the absorption increase at 253 nm. All the reaction volume for the spectral assay was 1 ml. One unit of activity was defined as the production of 1 nmol of 2-chloromaleylacetate (*ε*₂₅₃ = 9.6 mM⁻¹ cm⁻¹) (16) per min at room temperature in 20 mM KPi buffer (pH 6.7).

**Oxygen consumption.** Oxygen consumption was determined in a closed reaction vessel (0.6-ml total volume) fitted with a Clark-type oxygen electrode (Instech, Plymouth Meeting, PA). The electrode was calibrated with a chemical method by using N-methylphynazonium methosulfate and NADH to quantitatively consume O₂ (11). Experiments were carried out with 100 μM 2,6-DiCH generated by NaBH₄ reduction of dichloro-*p*-benzoquinone in 20 mM KPi buffer (pH 7.0). Fifteen μg of PcpA and 25 nmol FeSO₄ in volumes less than 5 μl were added to initiate the reaction.

**Liquid chromatography (LC)-mass spectrometry (MS) analysis.** For LC-MS analysis, the reaction was performed in 40 mM NH₄HCO₃ buffer (pH 7.8). The sample (100 μl) was injected onto a Nova Pak C-18 column (3.9 × 150 nm) equilibrated with distilled H₂O by an HPLC system equipped with an electrospray mass detector (a Finnigan LCQ system, Finnigan, Bremen, Germany) operated in a negative-ion mode. The sample was eluted with distilled H₂O at flow rate of 0.5 ml per min. The typical electrospray source parameters were at capillary voltage of 41 V, source voltage of 3,500 V, and

FIG. 1. Spectra of 200 nmol of DiCH oxidation by 11 μg PcpA in 1 ml of 20 mM KPi buffer, pH 6.7. The spectra were taken at reaction time of 0, 2, 4, 6, and 8 min.

capillary temperature of 200°C. The end product was detected by the mass detector at 2.44 min.

**Chloride assay.** Chloride was determined by measuring the turbidity of AgCl as described (18). A standard curve was generated with 25 to 200 μM NaCl in the assay mixture and it was linear (*R*² = 0.99).

**RESULTS**

**Overexpression of *pcpA*.** Strain BL21 (DE3) with pMM produced N-terminal fusion PcpA, while strain BL21 (DE3) with plasmid pCR1 overproduced nonfusion PcpA. Both proteins were active in degrading 2,6-DiCH only when the host cells grew at room temperature. When the host cells grew at 37°C, PcpA was overproduced but was insoluble and inactive. The overproduced PcpA from pCR1 was easily purified by an anion exchange column and a gel filtration column. Seventeen milligrams of PcpA was purified from 74 mg of cell extracts with a recovery of about 50%.

**The reaction end-product determination.** The purified PcpA actively degraded 2,6-DiCH as analyzed by the HPLC method. The reaction required Fe²⁺ and reaction conditions were similar as reported for “DiCH chlorohydrolase” (6). However, the end product could not be detected by the HPLC method. In order to detect the end product, a spectral assay was developed. NaBH₄ instead of ascorbic acid was used to reduce dichloro-*p*-benzoquinone to 2,6-DiCH because ascorbic acid interferes with the absorption spectra of 2,6-DiCH and the reaction end product in the UV range. 2,6-DiCH (200 μM) was completely converted to an end product with a peak maximum at 253 nm within 8 min by 11 μg of PcpA (Fig. 1). The spectrum of the end product is the same as that of 2-chloromaleylacetate (16), suggesting the end product is 2-chloromaleylacetate. HPLC-MS was used to confirm the identity of the end product. The end product lost its peak at 253 nm under acidic conditions. In order to follow this compound by HPLC, distilled H₂O was used to run HPLC. The end product did not bind to the C-18 column and came off the column in the void volume. The MS spectrum of the end product identified it as [2-chloromaleylactic acid-H]⁻ with peaks at *m/z* 191.1

Vol. 266, No. 2, 1999

BIOCHEMICAL AND BIOPHYSICAL RESEARCH COMMUNICATIONS

![Figure](https://i.imgur.com/relative_abundance.png)

FIG. 2. Mass spectrum of the end product. Reaction mixture was analyzed by HPLC coupled with a mass detector. The main peak at 2.44 min gave this spectrum.

(with $^{35}$Cl) and 193.1 (with $^{37}$Cl) and its first fragment under ionization conditions at $m/z$ 147.2 (loss of –COOH from 191.1) and 149.2 (loss of –COOH from 193.1) (Fig. 2). The masses and relative intensities are characteristic of a molecule containing one chlorine atom.

**Stoichiometry.** For 60 nmol of 2,6-DiCH consumed, 56 ± 2 nmol (standard deviation of 3 samples) of O₂ was consumed. The reaction did not generate any H₂O₂ because catalase added at the end of the run did not release any O₂. For 200 nmol of 2,6-DiCH consumed, 208 ± 15 nmol (standard deviation of 8 samples) of 2-chloromaleylacetate was produced and 198 ± 7 nmol (standard deviation of 3 samples) of Cl⁻ was released. On the basis of the reaction stoichiometry, the reaction catalyzed by PcpA was proposed in Fig. 3. The direct end product of

![Chemical Reaction Diagram](https://i.imgur.com/reaction_diagram.png)

FIG. 3. Proposed conversion of 2,6-DiCH to 2-chloromaleylacetate.

the enzymatic reaction is likely to be an intermediate which is an acid chloride (or acyl chloride). Acid chlorides are unstable in aqueous solutions and are rapidly hydrolyzed to the corresponding acid (2).

**Properties of PcpA.** Using the spectral assay, the optimal pH for PcpA in 40 mM KPi buffer was 6.7, and the enzyme was active from pH 6.3 to 8.0. The optimal ionic strength was at 20 mM KPi buffer (pH 6.7), and the enzyme was active from 5 to over 100 mM KPi. All other activities were measured at room temperature using the spectral assay. PcpA required Fe²⁺ for activity. Without Fe²⁺, PcpA did not have any activity. The $K_{\text{m}}$ of PcpA for Fe²⁺ was 4.6 μM. The $K_{\text{m}}$ for 2,6-DiCH was 13.5 μM, $V_{\text{max}}$ was 3,492 nmol min⁻¹ mg of PcpA⁻¹, and $k_{\text{cat}}$ was 2.1 s⁻¹ at room temperature in 20 mM KPi Buffer (pH 6.7). The activity of PcpA was rather specific. PcpA used only 2,6-DiCH as its substrate and did not oxidize 2,5-DiCH, 6-chlorohydroxyquinol, hydroxyquinol, 2-chloro-p-hydroquinone, catechol, and 4-fluorocatechol.

DISCUSSION

PcpA was first identified about 10 years ago (19). PcpA is shown to be involved in PCP degradation as it is induced by PCP and its release by EDTA from strain

ATCC 39723 instantaneously stops PCP degradation. When *pcpA* gene was disrupted by homologous recombination, 2,6-DiCH was accumulated (1), suggesting that *pcpA* encodes the 2,6-DiCH-degrading enzyme. Direct evidence to pinpoint that *pcpA* encodes the 2,6-DiCH-degrading enzyme has recently been reported by Xu *et al.* (18) and is presented in this report. PcpA is shown to be a ring-cleavage dioxygenase (Fig. 3). All the known ring-cleavage dioxygenases are generally classified into three groups: intradiol dioxygenases that break aromatic rings between two hydroxyl groups, extradiol dioxygenases that break aromatic rings adjacent to two hydroxyl groups, and gentisate 1,2-dioxygenases that cleave aromatic rings between a hydroxyl group and a carboxylic group (4). Since PcpA cleaves the aromatic ring between a hydroxyl group and a chlorine group and it does not share any significant sequence similarity with other reported dioxygenases, it is difficult for us to assign PcpA to any known classes of dioxygenases.

Initially, the end product of PcpA or "DiCH chlorohydrolase" has been misidentified by HPLC as 6-chlorohydroxyquinol (6). Xu *et al.* (18) have shown the enzyme is an oxygenase by using more rigorous anaerobic conditions. PcpA purified from *E. coli* was about 190 times more active (*k*~cat~ of 2.1 s⁻¹) than the original enzyme (*k*~cat~ of 0.011 s⁻¹) isolated from *S. chlorophenolica* ATCC 39723. The difference may be partly due to the assay conditions and partly due to the extensive inactivation of the original enzyme during purification as we have noticed that ammonium sulfate and dialysis tubing could inactivate the enzyme (data not shown). EDTA-released PcpA (19) from *S. chlorophenolica* ATCC 39723 was shown to oxidize 2,6-DiCH to 2-chloromaleylacetate (data not shown).

2,6-DiCH is a common metabolic intermediate of PCP, tetrachlorophenols, 2,4,6-trichlorophenol, and 2,6-dichlorophenol (15, 17, 20). PcpA is the only reported enzyme that can degrade 2,6-DiCH. It only takes three enzymes, PcpB, PcpC, and PcpA to convert PCP to 2-chloromaleylacetate. 2-Chloromaleylacetate is an intermediate during the degradation of several chloroaromatic compounds, and it is converted to 3-oxoadipate by maleylacetate reductase (5, 13). Many aromatic compounds enter TCA cycle via 3-oxoadipate.


ACKNOWLEDGMENTS

This research was supported by NSF Grant MCB-9722970. We thank Dr. Ron L. Crawford of University of Idaho for suggesting to using NaBH₄ to reduce dichloro-*p*-benzoquinone to 2,6-DiCH and Steven Halls for LC-MS analysis.


REFERENCES

1. Chanama, S., and Crawford, R. L. (1997) Appl. Environ. Microbiol. **63**, 4833–4838.
2. Fessenden, R.J., and Fessenden, J.S. (1986) Organic Chemistry. Brooks/Cole Publishing Company, Monterey, CA.
3. Haggblom, M. M. (1992) FEMS Microbiol. Rev. **103**, 29–72.
4. Harayama, S., Kok, M., and Neidle, E. L. (1992) Annu. Rev. Microbiol. **46**, 565–601.
5. Kaschabek, S. R., and Reineke, W. (1995) J. Bacteriol. **177**, 320–325.
6. Lee, J.-Y., and Xun, L. (1997) J. Bacteriol. **179**, 1521–1524.
7. McCarthy, D. L., Navarrete, S., Willett, W. S., Babbitt, P. C., and Copley, S. D. (1996) Biochemistry **35**, 14634–14642.
8. Nohynek, L. J., Suhonen, E. L., Urmiaho-Lassila, E. L., Hantula, J., and Salkinoja-Salonen, M. (1995) Syst. Appl. Microbiol. **18**, 527–538.
9. Orser, C. S., Dutton, J., Lange, C., Jablonski, P., Xun, L., and Hargis, M. (1993) J. Bacteriol. **175**, 2640–2644.
10. Rao, K. R. (1978) Environmental Science Research, Vol. 12. Plenum, New York.
11. Robinson, J., and Cooper, J. M. (1970) Anal. Biochem. **33**, 390–399.
12. Saber, D. L., and Crawford, R. L. (1985) Appl. Environ. Microbiol. **50**, 1512–1518.
13. Seibert, V., Stadler-Fritzsche, K., and Schlömann, M. (1993) J. Bacteriol. **175**, 6745–6754.
14. Sittig, M. (1981) Handbook of Toxic and Hazardous Chemicals, Noyes Publications, Park Ridge, NJ.
15. Takizawa, N., Yokoyama, H., Yanagihara, K., Hatta, T., and Kiyohara, H. (1995) J. Ferment. Bioeng. **80**, 318–326.
16. Tiedje, J. M., Duxbury, J. M., Alexander, M., and Dawson, J. E. (1969) J. Agr. Food Chem. **17**, 1021–1026.
17. Wieser, M., Wagner, B., Eberspacher, J., and J., Lingens, F. (1997) J. Bacteriol. **179**, 202–208.
18. Xu, L., Resing, K., Lawson, S. L., Babbitt, P. C., and Copley, S. D. (1999) Biochemistry **38**, 7659–7669.
19. Xun, L., and Orser, C. S. (1991) J. Bacteriol. **173**, 2920–2927.
20. Xun, L., Topp, E., and Orser, C. S. (1992) J. Bacteriol. **174**, 2898–2902.
21. Xun, L., Topp, E., and Orser, C. S. (1992) J. Bacteriol. **174**, 8003–8007.
