
Microbiology (2000), 146, 139–146

The mycarose-biosynthetic genes of  
Streptomyces fradiae, producer of tylosin  

Neil Bate, Andrew R. Butler, Ian P. Smith and Eric Cundliffe  

Author for correspondence: Eric Cundliffe. Tel: +44 116 252 3451. Fax: +44 116 252 3369.  
e-mail: ec13@le.ac.uk  

Department of  
Biochemistry, University  
of Leicester, Leicester  
LE1 7RH, UK  

The *tylCK* region of the *Streptomyces fradiae* genome was sequenced, revealing an incomplete set of five *tylC* genes encoding all-but-one of the enzymes involved in the biosynthesis of mycarose. The latter is a 6-deoxyhexose sugar required during production of the macrolide antibiotic, tylosin. The missing mycarose-biosynthetic gene, *tylCVI*, was found about 50 kb distant from its functional partners, on the other side of the *tylG* (polyketide synthase) gene complex. Mutational analysis, involving targeted gene transplacement, was employed to confirm the functions of specific genes, including *tylCVI*. Particularly interesting was the similarity between the tylosin-biosynthetic mycarosyltransferase enzyme, TylCV, and proteins of the macrolide glycosyltransferase (MGT) family that inactivate macrolides via glycosylation of attached sugar residues and are involved in resistance and/or antibiotic efflux. The arrangement of genes within the ‘mycarose cluster’ would allow their expression as two short operons with divergent, and perhaps co-regulated, promoters. Whether displacement of *tylCVI* relative to the other *tylC* genes provides additional regulatory opportunities remains to be established.

Keywords: tylosin production, *Streptomyces fradiae*, mycarose biosynthesis, deoxyhexose metabolism

INTRODUCTION

Tylosin, a macrolide antibiotic produced by *Streptomyces fradiae*, comprises a polyketide lactone, protylonolide (also known as tylactone), to which are attached three 6-deoxyhexose sugars (Fig. 1). The biosynthetic route to tylosin was determined by analysis of compounds accumulated by blocked mutants of *St.* *fradiae*, together with co-synthesis and bioconversion studies (Baltz & Seno, 1981; Ōmura *et al.*, 1982a, b; Baltz *et al.*, 1983). Thus, tylactone is first substituted with D-mycaminose followed, after ring hydroxylation at C-20 and C-23, by 6-deoxy-D-allose and then L-mycarose, in a preferred but not obligatory order. The last two steps of tylosin biosynthesis involve conversion of the deoxy-allose moiety to D-mycinose via bis-O-methylation. Complementation of blocked mutants using cloned DNA fragments (Beckmann *et al.*, 1989; Fishman *et al.*, 1987), allowed 13 genetic loci (*tylA–M*) to be mapped within the *St.* *fradiae* genome (Fig. 2). Most of the mutants displayed the TylG phenotype and were defective in polyketide biosynthesis. The TylA phenotype was initially characterized as inability to produce or add any of the three tylosin sugars, but complementation analysis later divided such mutants into two groups (*tylA* and *tylL*) with the respective loci located on either side of *tylG*. Similarly, mycarose-minus mutants (originally designated *tylC*) were subdivided into *tylC* and

![Structure of tylosin](https://i.imgur.com/yourimageurl.png)

Abbreviation: MGT, macrolide glycosyltransferase.

The GenBank accession number for the sequence determined in this work is AF147704.

N. BATE and OTHERS

![Diagram](#)

Fig. 2. Tylosin-biosynthetic structural gene cluster of St. fradiae (not drawn to scale). The box represents the entire cluster (~85 kb) showing 13 *tyl* loci flanked by the resistance genes *tlrB* and *tlrC*. All of the structural genes required for tylosin production probably lie between *tlrB* and *tlrC*. Arrows represent genes of the *tyl* cluster. Those analysed here are shown in grey and their locations within the cluster are indicated on the box. The *tylG* locus represents 5 mega genes (~41 kb) encoding the polyketide synthase complex. Immediately upstream of *tylG* are 12 genes including *tlrC* (*orf1*, 1a and 2–11), which occupy about 15 kb. On the downstream side, *tylG* is separated from *tlrB* by 25 ORFs that occupy about 29 kb. Beginning immediately downstream of *tylG*, these are numbered 1*–25* to distinguish them from ORFs on the upstream side of *tylG*. Only the first 11 downstream genes, proximal to *tylG*, are shown here.

---

**tylK** strains, while others defective in deoxyallose metabolism were sorted into *tylD* and *tylJ* strains. Polyketide ring hydroxylation at C-20 and C-23 was associated with the *tylI* and *tylH* loci, respectively, and bis-O-methylation of the deoxyallose moiety was shown to involve sequential action of the TylE and TylF proteins (Baltz & Seno, 1981). The various *tyl* loci are flanked by the resistance genes *tlrB* and *tlrC*, which lie about 85 kb apart in the St. fradiae genome (Baltz & Seno, 1988; Beckmann *et al.*, 1989).

Sequencing of the *tyl* gene cluster, defined for present purposes as the region of the St. fradiae genome flanked by *tlrB* and *tlrC*, began at Lilly Research Laboratories, Indianapolis, USA, in the late 1980s and has been completed in this laboratory. The sequence (41 kb) of *tylG*, encoding the polyketide synthase that produces tylactone, has been released but not formally published (GenBank accession number U78289) and the genes (*tylB*, *tylMI*, *tylMII* and *tylMIII*) encoding mycaminose biosynthesis have also been sequenced (Merson-Davies & Cundliffe 1994; Gandecha *et al.*, 1997). More recently, a contiguous DNA ‘cassette’ encoding the full complement of genes involved in mycinose biosynthesis, together with hydroxylation of the polyketide ring at C-23 to allow attachment of the sugar, has been analysed (Bate & Cundliffe, 1999). In the present work, the genes involved in mycarose biosynthesis were sequenced and functions were assigned to specific gene products by a combination of database comparisons, targeted gene disruptions and complementation of specifically disrupted strains.

---

**METHODS**

**Bacterial strains, growth conditions and genetic manipulation.** St. fradiae T59235 (wild-type, also known as C373.1) and its derivatives were maintained and propagated at 37°C on AS-1 agar (Wilson & Cundliffe, 1998) or at 30°C in tryptic soy broth (Difco). Plasmids were manipulated in Escherichia coli using standard protocols (Sambrook *et al.*, 1989). DNA was introduced into St. fradiae via conjugal transfer from E. coli as described elsewhere (Fish & Cundliffe, 1997). The vector used for targeted gene disruption/transplacement was pOJ260 (Bierman *et al.*, 1992); this plasmid, which carries an apramycin resistance marker, does not replicate in Streptomyces spp. Cloned fragments of the *tyl* gene cluster were inserted into the vector pLST9828 and tested for their ability to complement specific *tyl* gene disruptions. This vector, which uses the strong constitutive promoter, *ermEp*+, for expression of the inserted genes, integrates into the chromosomal *φC31 attB* site.

**DNA sequencing and computer analysis.** The sequence of *orf6* was obtained by combining that given under accession numbers U08223 (Merson-Davies & Cundliffe, 1994) and AF145042 (Bate *et al.*, 1999). The St. fradiae *tyl* DNA sequenced here was obtained from pMOMT4 (Beckmann *et al.*, 1989). Fragments were subcloned in pIJ2925 (Janssen & Bibb, 1993) and both strands of the DNA were sequenced independently in overlapping fashion by a combination of nested deletion analysis and primer walking. Sequencing was done on an ABI 377 automated DNA sequencer using Taq FS polymerase with dye terminator chemistry (Perkin Elmer). DNA sequences together with the corresponding chromatograms were imported into Seq Ed version 1.0.3 and aligned using AUTO ASSEMBLER. Sequences were analysed using the University of Wisconsin GCG software programmes. ORFs were identified using CODONPREFERENCE, BLASTX and 6 frame

translation with DNA STRIDER. Deduced products were analysed using BLASTP.

Targeted gene disruption through gene transplacement. The DNA fragment used to disrupt orf7* was generated by PCR using pMOMT4 as template (Beckmann et al., 1989). The primers used to create the orf7*-disruption fragment were 5'-ACTTGAAATTCATGACGACGGGTCCGGGACG-3' and 5'-GGCGAAGCTTGGCCGGAGGTCAggCCGGGG-3'. These contained, respectively, EcoRI and HindIII restriction sites (underlined) that were used to ligate the product (1513 bp with a unique central SalI site) into pIJ2925. The hygromycin resistance cassette, Ωhyg, together with its flanking transcriptional terminators (Blondelet-Rouault et al., 1997) was then inserted at the SalI site, thereby interrupting the orf7* coding sequence 640 bp downstream of the earliest possible translational start.

A 2272 bp KpnI fragment containing orf6 together with flanking DNA was excised from pSET552 (Beckmann et al., 1989) and inserted into pIJ2925. The hygromycin resistance cassette was inserted at the unique SexAI site, 584 bp downstream from the start of orf6 and 784 bp upstream of the translational stop.

Disrupted ORFs plus flanking DNA were ligated into the BamHI site of pOJ260 and introduced into St. fradiae. Transconjugants that had undergone gene transplacement were resistant to hygromycin and sensitive to apramycin.

Complementation of disrupted strains. Three different DNA fragments were reintroduced into the orf7*-disrupted strain. These were: (i) a 2167 bp fragment containing orf7* plus orf6* selected from a nested deletion library; (ii) an 825 bp fragment containing orf6* together with flanking DNA (227 bp upstream and 4 bp downstream) also selected from a nested deletion library; and (iii) the 1513 bp PCR product containing orf7* plus flanking DNA (118 bp downstream and 9 bp upstream). The orf6-disrupted strain was complemented using a 1630 bp NcoI-SmaI fragment generated from pSET552. This contained the whole of orf6 together with flanking DNA (150 bp upstream and 112 bp downstream). DNA fragments for complementation analysis were ligated into pLST9828 downstream of ermEp*.

Antibiotic-production fermentation and metabolite analysis. Fermentation of St. fradiae strains and the analysis of products by HPLC was done as described elsewhere (Butler et al., 1999).

**RESULTS AND DISCUSSION**

DNA sequence analysis

The sequence of tylCK DNA (5908 bp, GenBank accession number AF147704) contains five complete ORFs, each of which displays the biased codon usage typical of actinomycete genes, plus flanking DNA. Following the numbering scheme introduced previously (Merson-Davies & Cundliffe, 1994; Gandecha et al., 1997) and subsequently extended (Bate & Cundliffe, 1999; Bate et al., 1999), the 'new' ORFs are designated orf6*-10* (Fig. 2). At the left-hand end in the orientation of Fig. 2, the sequence presented here extends 50 bp into the co-directional orf11* and overlaps by 100 bp the sequence given under accession number AF145049 (Bate et al., 1999). At the right-hand end, the sequence terminates 50 bp inside the resistance determinant tlrD (orf5*) and overlaps by 209 bp the sequence deposited under accession number X97721 (Gandecha & Cundliffe, 1996). Database comparisons and mutational analysis (below) revealed that orf6*-10* comprise an incomplete set of mycarose-biosynthetic genes (lacking one functional member). The 'missing' mycarose-biosynthetic gene (orf6) was unexpectedly encountered over 50 kb distant from the tylCK region on the other side of the tylG cluster (accession number AF145042; Bate et al., 1999) and its function has been confirmed here via targeted disruption analysis. Since the mycarose-minus phenotype was originally termed 'TylC' (Baltz & Seno, 1981), the mycarose-biosynthetic genes of St. fradiae have been assigned 'tylC' designations. To facilitate comparison with the mycarose-biosynthetic (eryB) genes of Saccharopolyspora erythraea, the numbering system already in use for the eryB genes has been adopted here (Table 1).

**The biosynthetic route to mycarose**

The biosynthetic pathway leading to mycarose is not known with certainty (Fig. 3). Epimerization at C-5 must occur at some stage since mycarose is an L-sugar derived ultimately from D-glucose. A priori, this could be achieved via 5-epimerization or 3,5-epimerization and might occur before or after 2-deoxygenation. Were epimerization to precede 2-deoxygenation, the stereo-specificity at C-3 would be lost in the dehydratase step and would ultimately be determined by the 2,3-enoyl-reductase (here, TylCII). Consistent with such a model, an eryBII mutant of Sa. erythraea produced a derivative of erythromycin C (Salah-Bey et al., 1998) containing a neutral sugar ostensibly derived, via 4-ketoreduction, from that shown in Fig. 3 (compound 2). Alternatively, 2-deoxygenation might occur first (Summers et al., 1997), in which case the stereochemistry at C-3 following the enoylreductase reaction would dictate the mode of

**Table 1. Mycarose-biosynthetic genes of St. fradiae and Sa. erythraea**

| ORF | St. fradiae gene name | Sa. erythraea gene name | Gene product |
|-----|-----------------------|-------------------------|--------------|
| orf6* | tylCVII              | eryBVII                 | 5-(or 3,5) Epimerase |
| orf6 | tylCVI               | eryBVI                  | 2,3-Dehydratase |
| orf10* | tylCII               | eryBII                  | 2,3-Enoylreductase |
| orf8* | tylCIII              | eryBIll                 | 3-C-Methyltransferase |
| orf9* | tylCIV               | eryBIV                  | 4-Ketoreductase |
| orf7* | tylCV                | eryBV                   | Glycosyltransferase |

characterized as the product. From our data, we cannot discriminate between these alternative models.

### Assignment of ORFs

**orf6* (tylCVII).** The deduced product of *orf6\** is a protein of 197 amino acid residues maximum (*M~r~* 21400) that shows convincing end-to-end similarities to the products of numerous putative NDP-4-keto-6-deoxyglucose 3,5-epimerases including (with 52% and 49% sequence identity, respectively) the products of *gra-orf25* from *Streptomyces violaceoruber* Tü22 (Ichinose et al., 1998) and *lanZ1* from *Streptomyces cyanogenus* (Westrich et al., 1999). In particular, the Orf6\* protein shares 44% sequence identity with EryBVII (Gaisser et al., 1997; Summers et al., 1997). In *Sa. erythraea*, *eryBVII* encodes a mycarose-biosynthetic enzyme and similar genes in other organisms are also associated with the production of L-sugars, including L-rhamnose (*rfbC* gene in *Shigella*; Macpherson et al., 1994), L-ascarylose (*ascE* in *Yersinia*; Thorson et al., 1994) and L-dihydrostreptose (*strM* in *Streptomyces griseus*; Pissowatzky et al., 1991). Although direct experimental evidence is lacking, StrM (Pissowatzky et al., 1991) and EryBVII (Gaisser et al., 1997; Summers et al., 1997) are believed to be NDP-4-keto-6-deoxyglucose 3,5- (or 5-) epimerases and a similar role is proposed here for the *orf6\** product. Proof that Orf6\* is involved in the biosynthesis of mycarose was obtained by fermentation analysis of a strain in which *orf6\** was specifically disrupted. The resultant strain produced desmycosin (demycarosyl-tylosin) and complementation with *orf6\** restored normal levels of tylosin production (data not shown). We concluded that Orf6\* (now designated TylCVII) is the 3,5- (or 5-) epimerase involved in mycarose biosynthesis and noted significant similarity (41% sequence identity) between it and TylJ, the putative 3-epimerase involved in the formation of 6-deoxyallose (Bate & Cundliffe, 1999).

**orf7* (tylCV).** In database searches, the deduced product of *orf7\** (461 amino acids; *M~r~* 50100) displayed greater than 30% sequence identity to various macrolide glycosyltransferases (MGTs), such as OleI and OleD from *St.* *antibioticus*, producer of oleandomycin (Quirós et al., 1998; Hernandez et al., 1993), GimA from *Streptomyces ambofaciens*, producer of spiramycin (Gourmelen et al., 1998) and Mgt from *Streptomyces lividans* (Jenkins & Cundliffe, 1991). Such enzymes inactivate their substrates and are involved in antibiotic efflux and/or resistance. Less close matches (20–25% sequence identity) were also seen with glycosyltransferases involved in antibiotic biosynthesis such as OleG2 (Olano et al., 1998), EryCIII and EryBV (Gaisser et al., 1997; Summers et al., 1997; Salah-Bey et al., 1998). Nevertheless, when the role of *orf7\** was addressed by targeted gene disruption it proved to be a mycarose-biosynthetic gene and not an *mgt*. Thus, the *orf7\**-disrupted strain accumulated desmycosin during fermentation (Fig. 4a) and tylosin production was restored following complementation with cloned DNA, but only when *orf6\** was added together with *orf7\** (Fig. 4b), in this case on a single DNA fragment. Reintroduction of

subsequent epimerization. Support for the latter route, at least in the biosynthesis of other 2,6-dideoxyhexoses, came from recent *in vitro* studies (Draeger et al., 1999) involving pairs of purified enzymes that catalyse 2-deoxygenation of dTDP-4-keto-6-deoxy-D-glucose (Fig. 3, compound 1) during the synthesis of L-rhodinose in *Streptomyces violaceoruber* Tü22, and of L-oleandrose in *Streptomyces antibioticus* Tü99. In either case, dTDP-4-keto-2,6-dideoxy-D-glucose (Fig. 3, compound 3) was

Mycarose-biosynthetic genes of St. fradiae

Fig. 4. Fermentation products from orf7*-disrupted St. fradiae. HPLC analysis of material produced by (a) orf7*-disrupted strain, (b) orf7*-disrupted strain complemented with orf7* and 6*, (c) orf7*-disrupted strain complemented with orf7*, and (d) orf7*-disrupted strain complemented with orf6*. The arrows indicate where tylosin would be expected to elute.

either gene alone did not restore tylosin production (Fig. 4c, d) although unidentified material, which was clearly separable from both tylosin and desmycosin, accumulated together with the latter when complementation was attempted with orf7* alone (Fig. 4c). These data established that orf7*, like orf6*, must be a mycarose-biosynthetic gene and that normally the two genes are probably co-transcribed. Disruption of orf7* using the Qhyg cassette (which is flanked by transcriptional terminators) evidently caused a downstream polar effect on the expression of orf6*.

We concluded that the orf7* product (designated TylCV) is the mycarosyl transferase involved in tylosin biosynthesis and we noted its closer resemblance to macrolide glycosyltransferases than to enzymes involved in antibiotic production. Unlike EryBV, which attaches mycarose directly to the polyketide during erythromycin production, TylCV acts on a glycosylated macrolide and forms a disaccharidic moiety. In this sense, TylCV more closely resembles MGT enzymes, although the latter modify their substrates at the 2'-OH rather than the 4'-OH that is targeted by TylCV.

orf8* (tylCIII). The deduced product of orf8* (418 amino acids maximum; Mr 46600) displays extremely close end-to-end similarity (with 70% sequence identity) to that of eryorf3 (Haydock et al., 1991), now renamed eryBIII (Gaisser et al., 1998), and is much more closely related to that protein than to any other in the database. Moreover, Orf8* and EryBIII each possess multiple sequence motifs characteristic of S-adenosylmethionine-dependent methyltransferases (Kagan & Clarke, 1994). Since a strain of Sa. erythraea with a point mutation in eryBIII accumulated 3"-C-desmethyl-erythromycin A during fermentation (Gaisser et al., 1998), the unequivocal conclusion was that eryBIII encodes a 3-C-methyltransferase activity required during mycarose biosynthesis. An equivalent role is proposed for the orf8* product, designated TylCIII.

orf9* (tylCIV). The deduced product of orf9* (maximum size, 347 amino acids; Mr 36200) displays 49% sequence identity to that of eryBIV (Gaisser et al., 1997; Summers et al., 1997). The only other significant end-to-end sequence match (46% identity) was to an uncharacterized protein, PCZA 361.4, from vancomycin-producing Amycolatopsis orientalis (van Wageningen et al., 1998). Mutants of Sa. erythraea, altered in eryBIV, accumulated modified erythromycins in minor amounts and mass spectrometric analysis (Salah-Bey et al., 1998) revealed that the neutral sugar (mycarose) appeared to lack two hydrogen atoms. From these data, it was

N. BATE and OTHERS

![Figure](https://i.imgur.com/1234567.png)

Fig. 5. Fermentation products from St. fradiae wild-type and the orf6-disrupted strain. HPLC analysis of material produced by (a) wild-type, (b) orf6-disrupted strain, and (c) orf6-disrupted strain complemented with orf6.

concluded that eryBIV encodes 4-ketoreductase activity involved in mycarose metabolism. An equivalent role is proposed here for the orf9* product, TylCIV. Consistent with this proposal, TylCIV and EryBIV are distantly related to UDP-glucose 4-epimerases and to various oxidoreductases that manipulate sugars at the 4-position, including NDP-glucose 4,6 dehydratases such as TylAII (formerly TylA2; Merson-Davies & Cundliffe, 1994) and StrE (Pissowatzki et al., 1991). Moreover, TylCIV contains a candidate NADPH-binding sequence (GRGVIAVTAG) with a downstream arginine cluster (RDLLDVR) positioned suitably to form a binding site for the 2'-phosphate of NADPH (N. S. Scrutton, personal communication).

**orf10* (tylCI).** The deduced product of orf10* (329 amino acids; Mr 36100) is very similar (74% sequence identity) to EryBII from Sa. erythraea (Summers et al., 1997; Salah-Bey et al., 1998). Less close matches (41% and 30% sequence identity, respectively) were found with MocA from Agrobacterium tumefaciens (Kim & Farrand, 1996), which is 'weakly related' to glucose-6-phosphate dehydrogenase, and with an authentic aryl alcohol dehydrogenase from the white rot fungus, Phanerochaete chrysosporium (Reiser et al., 1994). Despite the absence of any obvious NAD(P)H-binding motif in the amino acid sequence, these data suggest that Orf10* might be one of the two reductases involved in mycarose biosynthesis. The function of EryBII was revealed during incisive analysis of Sa. erythraea mutant BII92 (Salah-Bey et al., 1998), a specifically deleted strain that produced a 2",3"-unsaturated derivative of 3"-C-desmethyl-erythromycin C. Those authors suggested that eryBII encodes 2,3-enoyl reductase activity involved in mycarose biosynthesis and a similar role is proposed here for the orf10* product, designated TylCII.

**orf6 (tylCVI).** The 'missing' gene of the mycarose-biosynthetic pathway (tylCVI) is not present in the tylCK region of the St. fradiae genome. Somewhat surprisingly, it is located about 50 kb distant from orf6*-10*, on the opposite side of the tylG gene cluster adjacent to the regulatory gene tylR (Bate et al., 1999). The maximum length of the orf6 product is 455 amino acid residues, which would give a 50 kDa protein. The involvement of orf6 in mycarose metabolism was first suspected when the upstream gene (orf5) was specifically disrupted and then complemented with intact orf5. The resultant strain produced desmycosin (demycarosyl-tylosin) rather than tylosin, suggesting that disruption of orf5 had caused a downstream polar effect on the expression of a mycarose-biosynthetic gene (Butler et al., 1999). Validation of that interpretation involved targeted disruption of orf6 via gene transplantation (confirmed by Southern analysis) followed by HPLC screening of fermentation products. This revealed a major peak of UV-absorbing material that co-eluted with desmycosin and was clearly resolved from tylosin (Fig. 5b). The yield of desmycosin approximated that of tylosin seen in control fermentations with the wild-type strain (Fig. 5a). More detailed analysis, involving HPLC-MS, revealed that the single peak in Fig. 5(b) represented a mixture of desmycosin plus 20-dihydrodesmycosin, together with two minor components consistent with N-demethyl-desmycosin and N-demethyl-20-dihydrodesmycosin (D. E. Kiehl & H. A. Kirst, personal communication). Macrolide reductase activity, which converts tylosin to relomycin (20-dihydrotylosin) and also acts on the corresponding aldehyde moieties of other macrolides, was purified from tylosin-producing strains of St. fradiae and was also detected in other Streptomyces spp. (such as St. lividans) that do not produce macrolides (Huang et al., 1993). Therefore, the presence of 20-dihydrodesmycosin in the orf6-disrupted strain is readily rationalized, although the detection of N-demethyl compounds, albeit at low levels, was unexpected.
When $orf6$, together with $ermEp^*$, was integrated into the $\phi C31 attB$ site of the genome, tylosin production was restored in the $orf6$-disrupted strain and HPLC revealed two main fermentation products (Fig. 5c). One of these co-eluted with desmycosin whereas the major product was inseparable from tylosin. These data clearly established a role for $orf6$ in mycarose biosynthesis despite its isolated location with respect to its functional partners.

The $orf6$ product, TylCVI, displays end-to-end similarity (with 51% sequence identity) to the deduced product of $eryBVI$ from *Sa.* erythraea (*Gaisser et al.*, 1997; *Summers et al.*, 1997) and to the products of $dnmT$ from *Streptomyces peucetius* (*Scotti & Hutchinson*, 1996) and $orf3$ from *Streptomyces sp.* C5 (*Dickens et al.*, 1996), producers of doxorubicin and daunomycin, respectively. Arguing by elimination, it was suggested that EryBVI is the 2,3-dehydratase required for deoxygenation at C-2 during mycarose biosynthesis in the erythromycin producer (*Gaisser et al.*, 1997; *Summers et al.*, 1997) and a similar role is invoked for TylCVI during tylosin production in *St.* fradiae.

### Concluding remarks

It will be interesting to study regulation of the $tylC$ genes during tylosin production. The physical organization of $orf6^*–10^*$ would permit possible co-regulation from a pair of divergent (perhaps equivalent) promoters located in the non-coding gap between $orf7^*$ and $orf8^*$. Although the minimum size of this gap is 212 bp, we suspect that the TylCV coding sequence is considerably shorter than $orf7^*$ and that the intergenic region is accordingly greater than 400 bp. Gaps of this size are rare in the $tyl$ cluster. Since there are no spaces between $orf8^*, 9^*$ and $10^*$, and since $orf7^*$ and $6^*$ are separated by only 43 bp, this group of genes might constitute two short operons. But what, if anything, is gained by having one mycarose gene displaced so far from the others? And is it significant that the mycaminose-biosynthetic genes are similarly split, with three co-directional $tylM$ genes ($orf1^*–3^*$) located near the ‘mycarose-gene cluster’ whereas $tylB$ lies on the other side of $tylG$, near $tylCVI$ (*Merson-Davies & Cundliffe*, 1994).

### ACKNOWLEDGEMENTS

This work was supported by an MRC research studentship awarded to I.P.S., by project grant 91/T08195 from the BBSRC, and by Eli Lilly and Co., Indianapolis, USA. We thank Douglas E. Kiehl and Herbert A. Kirst for mass spectrometric analysis of fermentation products and Nigel S. Scrutton for evaluation of protein sequence motifs.

### REFERENCES

Baltz, R. H. & Seno, E. T. (1981). Properties of *Streptomyces fradiae* mutants blocked in biosynthesis of the macrolide antibiotic tylosin. Antimicrob Agents Chemother 20, 214–225.

Baltz, R. H. & Seno, E. T. (1988). Genetics of *Streptomyces fradiae* and tylosin biosynthesis. Annu Rev Microbiol 42, 547–574.

Baltz, R. H., Seno, E. T., Stonesifer, J. & Wild, G. M. (1983). Biosynthesis of the macrolide antibiotic tylosin. A preferred pathway from tylactone to tylosin. J Antibiot 36, 131–141.

Bate, N. & Cundliffe, E. (1999). The mycinose-biosynthetic genes of *Streptomyces fradiae*, producer of tylosin. J Ind Microbiol Biotechnol 23, 118–122.

Bate, N., Butler, A. R., Gandecha, A. R. & Cundliffe, E. (1999). Multiple regulatory genes in the tylosin biosynthetic cluster of *Streptomyces fradiae*. Chem Biol 6, 617–624.

Beckmann, R. J., Cox, K. & Seno, E. T. (1989). A cluster of tylosin biosynthetic genes is interrupted by a structurally unstable segment containing four repeated sequences. In *Genetics and Molecular Biology of Industrial Microorganisms*, pp. 176–186. Edited by C. L. Hershberger, S. W. Queener & P. L. Skatrud. Washington, DC: American Society for Microbiology.

Bierman, M., Logan, R., O’Brien, K., Seno, E. T., Rao, R. N. & Schoner, B. E. (1992). Plasmid cloning vectors for the conjugal transfer of DNA from *Escherichia coli* to *Streptomyces* spp. Gene 116, 43–49.

Blondelet-Rouault, M.-H., Weiser, J., Lebrihi, A., Branny, P. & Pernodet, J.-L. (1997). Antibiotic resistance cassettes derived from the $\Omega$ interposon for use in *E. coli* and *Streptomyces*. Gene 190, 315–317.

Butler, A. R., Bate, N. & Cundliffe, E. (1999). Impact of thioesterase activity on tylosin biosynthesis in *Streptomyces fradiae*. Chem Biol 6, 287–292.

Dickens, M. L., Ye, J. & Strohl, W. R. (1996). Cloning, sequencing, and analysis of aklaviketone reductase from *Streptomyces sp.* strain C5. J Bacteriol 178, 3384–3388.

Draeger, G., Park, S.-H. & Floss, H. G. (1999). Mechanism of the 2-deoxygenation step in the biosynthesis of the deoxyhexose moieties of the antibiotics granaticin and oleandomycin. J Am Chem Soc 121, 2611–2612.

Fish, S. A. & Cundliffe, E. (1997). Stimulation of polyketide metabolism in *Streptomyces fradiae* by tylosin and its glycosylated precursors. Microbiology 143, 3871–3876.

Fishman, S. E., Cox, K., Larson, J. L., Reynolds, P. A., Seno, E. T., Yeh, W.-K., Van Frank, R. & Hershberger, C. L. (1987). Cloning genes for the biosynthesis of a macrolide antibiotic. Proc Natl Acad Sci USA 84, 8248–8252.

Gaisser, S., Böhm, G. A., Cortés, J. & Leadlay, P. F. (1997). Analysis of seven genes from the $eryA1$-$eryK$ region of the erythromycin biosynthetic gene cluster in *Saccharopolyspora erythraea*. Mol Gen Genet 256, 239–251.

Gaisser, S., Böhm, G. A., Doumith, M., Raynal, M.-C., Dhillon, N., Cortés, J. & Leadlay, P. F. (1998). Analysis of $eryBI$, $eryBIII$ and $eryBVII$ from the erythromycin biosynthetic gene cluster in *Saccharopolyspora erythraea*. Mol Gen Genet 258, 78–88.

Gandecha, A. R. & Cundliffe, E. (1996). Molecular analysis of $tlrD$, an MLS resistance determinant from the tylosin producer, *Streptomyces fradiae*. Gene 180, 173–176.

Gandecha, A. R., Large, S. L. & Cundliffe, E. (1997). Analysis of four tylosin biosynthetic genes from the $tylLM$ region of the *Streptomyces fradiae* genome. Gene 184, 197–203.

Gourmelen, A., Blondelet-Rouault, M.-H. & Pernodet, J.-L. (1998). Characterization of a glycosyl transferase inactivating macro-lides, encoded by $gimA$ from *Streptomyces ambofaciens*. Anti-microb Agents Chemother 42, 2612–2619.

Haydock, S. F., Dowson, J. A., Dhillon, N., Roberts, G. A., Cortés, J. & Leadlay, P. F. (1991). Cloning and sequence analysis of genes
N. BATE and OTHERS

involved in erythromycin biosynthesis in *Saccharopolyspora erythraea*, sequence similarities between EryG and a family of S-adenosylmethionine-dependent methyltransferases. Mol Gen Genet 230, 120–128.

Hernandez, C., Olano, C., Méndez, C. & Salas, J. A. (1993). Characterization of a *Streptomyces antibioticus* gene cluster encoding a glycosyltransferase involved in oleandomycin inactivation. Gene 134, 139–140.

Huang, S.-L., Hassell, T. C. & Yeh, W.-K. (1993). Purification and properties of NADPH-dependent tylosin reductase from *Streptomyces fradiae*. J Biol Chem 268, 18987–18993.

Ichinose, K., Bedford, D. J., Tornus, D., Bechtold, A., Bibb, M. J., Revill, W. P., Floss, H. G. & Hopwood, D. A. (1998). The granaticin biosynthetic gene cluster of *Streptomyces violaceoruber* Tu22: sequence analysis and expression in a heterologous host. Chem Biol 5, 647–659.

Janssen, G. R. & Bibb, M. J. (1993). Derivatives of pUC18 that have *BglII* sites flanking a modified multiple cloning site and that retain the ability to identify recombinant clones by visual screening of *Escherichia coli* colonies. Gene 124, 133–134.

Jenkins, G. & Cundliffe, E. (1991). Cloning and characterization of two genes from *Streptomyces lividans* that confer inducible resistance to lincomycin and macrolide antibiotics. Gene 108, 55–62.

Kagan, R. M. & Clarke, S. (1994). Widespread occurrence of three sequence motifs in diverse S-adenosylmethionine-dependent methyltransferases suggests a common structure for these enzymes. Arch Biochem Biophys 310, 417–427.

Kim, K.-S. & Farrand, S. K. (1996). Ti plasmid-encoded genes responsible for catabolism of the crown gall opine mannopine by *Agrobacterium tumefaciens* are homologs of the T-region genes responsible for synthesis of this opine by the plant tumor. J Bacteriol 178, 3275–3284.

Macpherson, D. F., Manning, P. A. & Morona, R. (1994). Characterization of the dTDP-rhamnose biosynthetic genes encoded in the *rfb* locus of *Shigella flexneri*. Mol Microbiol 11, 281–292.

Merson-Davies, L. A. & Cundliffe, E. (1994). Analysis of five tylosin biosynthetic genes from the *tyllBA* region of the *Streptomyces fradiae* genome. Mol Microbiol 13, 349–355.

Olano, C., Rodriguez, A. M., Michel, J.-M., Méndez, C., Raynal, M.-C. & Salas, J. A. (1998). Analysis of a *Streptomyces antibioticus* chromosomal region involved in oleandomycin biosynthesis, which encodes two glycosyltransferases responsible for glycosylation of the macrolactone ring. Mol Gen Genet 259, 299–308.

Omura, S., Sadakane, N. & Matsubara, H. (1982a). Bioconversion and biosynthesis of 16-membered macrolide antibiotics. XXII. Biosynthesis of tylosin after protylonolide formation. Chem Pharm Bull 30, 223–229.

Omura, S., Tanaka, H. & Tsukui, M. (1982b). Biosynthesis of tylosin: oxidations of 5-O-mycaminosylprotylonolide at C-20 and C-23 with a cell-free extract from *Streptomyces fradiae*. Biochem Biophys Res Commun 107, 554–560.

Pissowatzki, K., Mansouri, K. & Piepersberg, W. (1991). Genetics of streptomycin production in *Streptomyces griseus*: molecular structure and putative function of genes *strELMB2N*. Mol Gen Genet 231, 113–123.

Quirós, L. M., Aguirrezabalaga, I., Olano, C., Méndez, C. & Salas, J. A. (1998). Two glycosyltransferases and a glycosidase are involved in oleandomycin modification during its biosynthesis by *Streptomyces antibioticus*. Mol Microbiol 28, 1177–1185.

Reiser, J., Muheim, A., Hardegger, M., Frank, G. & Fiechter, A. (1994). Aryl-alcohol dehydrogenase from the white rot fungus *Phanerochaete chrysosporium*. J Biol Chem 269, 28152–28159.

Salah-Bey, K., Doumith, M., Michel, J.-M., Haydock, S., Cortés, J., Leadlay, P. F. & Raynal, M.-C. (1998). Targeted gene inactivation for the elucidation of deoxysugar biosynthesis in the erythromycin producer *Saccharopolyspora erythraea*. Mol Gen Genet 257, 542–553.

Sambrook, J., Fritsch, E. F. & Maniatis, T. (1989). *Molecular Cloning: a Laboratory Manual*, 2nd edn. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory.

Scotti, C. & Hutchinson, C. R. (1996). Enhanced production by manipulation of the *Streptomyces peucetius dnrH* and *dnmT* genes involved in doxorubicin (adriamycin) biosynthesis. J Bacteriol 178, 7316–7321.

Summers, R. G., Donadio, S., Staver, M. J., Wendt-Pienkowski, E., Hutchinson, C. R. & Katz, L. (1997). Sequencing and mutagenesis of genes from the erythromycin biosynthetic gene cluster of *Saccharopolyspora erythraea* that are involved in L-mycarose and D-desosamine production. Microbiology 143, 3251–3262.

Thorson, J. S., Lo, S. F., Ploux, O., He, X. & Liu, H. W. (1994). Studies of the biosynthesis of 3,6-dideoxyhexoses: molecular cloning and characterization of the *asc* (ascarylose) region from *Yersinia pseudotuberculosis* serogroup VA. J Bacteriol 176, 5483–5493.

van Wageningen, A. M. A., Kirkpatrick, P. N., Williams, D. H., Harris, B. R., Kershaw, J. K., Lennard, N. J., Jones, M., Jones, S. J. M. & Solenberg, P. J. (1998). Sequencing and analysis of genes involved in the biosynthesis of a vancomycin group antibiotic. Chem Biol 5, 155–162.

Westrich, L., Domann, S., Faust, B., Bedford, D., Hopwood, D. A. & Bechtold, A. (1999). Cloning and characterization of a gene cluster from *Streptomyces cyanogenus* S136 probably involved in landomycin biosynthesis. FEMS Microbiol Lett 170, 381–387.

Wilson, V. T. W. & Cundliffe, E. (1998). Characterization and targeted disruption of a glycosyltransferase gene in the tylosin producer, *Streptomyces fradiae*. Gene 214, 95–100.

Received 18 June 1999; revised 29 September 1999; accepted 11 October 1999.
