
Microbiology (2000), 146, 155–163

**Elucidation of anthracyclinone biosynthesis by stepwise cloning of genes for anthracyclines from three different *Streptomyces* spp.**

Jaana Kantola,$^{1}$ Tero Kunnari,$^{2}$ Anne Hautala,$^{2}$ Juha Hakala,$^{2}$ Kristiina Ylihonko$^{1,2}$ and Pekka Mäntsälä$^{1}$

Author for correspondence: Kristiina Ylihonko. Tel: +358 2 3336879. Fax: +358 2 3336860.  
e-mail: kristiina.ylihonko@finabo.abo.fi

---

**1 Department of Biochemistry, University of Turku, Vatselantie 2, FIN-20014 Turku, Finland**

**2 Galilaeus Oy, PO BOX 113, FIN-20781 Kaarina, Finland**

The anthracycline skeleton is biosynthesized by aromatic (type II) polyketide synthases. Furthermore, three post-polyketide steps are needed to form the basic aglycone of anthracyclines. Auramycinone was produced in *Streptomyces lividans* by introducing nine structural genes from three different anthracycline-producing *Streptomyces* species. The genes used to construct the auramycinone biosynthesis cluster were derived from nogalamycin-, daunomycin- and aclacinomycin-producing *Streptomyces* strains. The biosynthetic stages were divided into polyketide and post-polyketide steps on the assumption that the first stable intermediate would be nogalonic acid, named analogously to aklanonic acid, the precursor of several anthracyclines. Single genes were cloned in the expression construct in the order determined by the proposed biosynthetic pathway. This facilitated investigation of the products formed in the heterologous host after addition of each separate gene to the construct. The results thus elucidate the biosynthesis steps, products and the genes responsible for the reactions needed to build up an anthracyclinone.

---

Keywords: anthracyclines, auramycinone, biosynthesis, heterologous expression, *Streptomyces*

---

### INTRODUCTION

Anthracyclines are aromatic polyketide antibiotics produced by *Streptomyces* species and consist of an aglycone skeleton and sugar moieties (Fig. 1). Diversity arises from structural differences in the aglycone and from a wide array of attached sugar residues. The significant antitumour activity of anthracyclines makes them commercially interesting and indeed anthracyclines have been successfully used in cancer chemotherapy for decades. Anthracyclines are divided into several subgroups, of which the daunomycins are the best known, since all of the clinically used anthracyclines except aclacinomycin A belong to the daunomycin group.

Auramycins are related in structure to aclacinomycin (Fujiwara *et al.*, 1982) and are produced by a mutant strain OBB-111, derived from *Streptomyces galilaeus* ATCC 31533, which accumulates aclacinomycins. The only difference between auramycins and aclacinomycins is the alkyl group at position 9 (Fig. 1). Probably, biosynthesis proceeds in the same manner as in aclacinomycins though the starter unit in biosynthesis is different.

The organization of the polyketide synthase (PKS) gene cluster of *S. galilaeus* has been reported recently (Fujii & Ebizuka, 1997) and the molecular genetics of daunomycin biosynthesis has been intensively studied by Hutchinson, Strohl and their coworkers (e.g. Madduri & Hutchinson, 1995; Rajgarhia & Strohl, 1997). Related work on other aromatic polyketides has added to our knowledge of biosynthetic reactions in the anthracycline pathway (e.g. Hutchinson & Fujii, 1995; McDaniel *et al.*, 1995; Hopwood, 1997; Zawada & Khosla, 1997).

We report here a step-by-step introduction of single genes for anthracyclinone biosynthesis into a plasmid construct in the order predicated by the proposed biosynthetic pathway (Ylihonko *et al.*, 1996b). The compounds accumulated in a heterologous host were

---

Abbreviations: AAME, aklanonic acid methyl ester; minPKS, minimal PKS; PKS, polyketide synthase.  
The GenBank accession number for *acmA* reported in this paper is AF043550.

J. KANTOLA and OTHERS

(a)

![Chemical Structure](chemical_structure_a.png)

Nogalamycin

(b)

![Chemical Structure](chemical_structure_b.png)

R = CH₂CH₃ Aclacinomycin A  
R = CH₃ Auramycin A

(c)

![Chemical Structure](chemical_structure_c.png)

Daunomycin

(d)

![Chemical Structure](chemical_structure_d.png)

8-Demethoxy steffimycin

(e)

![Chemical Structure](chemical_structure_e.png)

Actinorhodin

**Fig. 1.** The structures of (a) nogalamycin, (b) aclacinomycin A and auramycin A, (c) daunomycin, (d) 8-demethoxy steffimycin and (e) actinorhodin.

analysed after each step. The choice of *Streptomyces lividans* as the host for expression cloning was based on evidence that it does not itself accumulate anthracycline metabolites in the culture conditions used for fermentation.

### METHODS

#### Bacterial strains and plasmids. Manipulations of *Streptomyces* DNA were carried out in *Escherichia coli* XL2-Blue (*recA1 endA1 gyrA96 thi-1 hsdR17 supE44 relA1 lac [F' proAB lacI<sup>q</sup>Z ΔM15 Tn10(Tet<sup>r</sup>) Amy Cam<sup>r</sup>]*)(Stratagene). *Streptomyces* strains used were *Streptomyces nogalater* ATCC 27451 (Bhuyan & Dietz, 1965), *Streptomyces steffisburgensis* ATCC 27466 (Dietz, 1967), *S*. *galilaeus* ATCC 31615 (Oki et al., 1975), the blocked *S*. *galilaeus* mutants H036, H039 and H061 (Ylihonko et al., 1994), *Streptomyces peucetius* ATCC 27952 (Arcamone et al., 1969) and *S*. *lividans* TK24 (Hopwood et al., 1985). The plasmids used are listed in Table 1.

#### Culture conditions and cloning procedures. DNA propagated in *E. coli* was ligated into pIJ486 derivatives and introduced by protoplast transformation into *S*. *lividans* TK24. DNA isolation and manipulation were carried out by standard procedures (Sambrook et al., 1989; Hopwood et al., 1985). *Streptomyces* were transformed by standard methods (Hopwood et al., 1985), with minor modifications (Ylihonko et al., 1996a).

Strains were cultivated in 60 ml E1 medium for 5–6 d (28 °C, 330 r.p.m.) to produce polyketides. Products were isolated by extracting with chloroform/methanol (3:1, v/v) for TLC and HPLC analysis.

#### Expression constructs. Plasmids were made in pSY21 carrying the nogalamycin minimal PKS (minPKS) genes or in pSY15 expressing the genes for the nogalamycin chromophore (Ylihonko et al., 1996b). MinPKS and an activator from the *S*. *nogalater* gene cluster were cloned in the *Streptomyces* vector pIJ486 as a 5·5 kb *SacI-BglII* fragment to give pSY21 (Fig. 2). A 12 kb *BglII* fragment from the *sno* cluster was cloned in pIJ486 to give pSY15 (Fig. 2). According to Table 1, suitable restriction sites were used for cloning. The restriction sites were made blunt-ended by treatment with Klenow polymerase to make pSY21a, pSY15a and pSY15b. Furthermore, the inserts introduced into the vector to generate pSY21b, pSY21c and pSY21d were cloned through the *E. coli* vector pSL1180 to add convenient restriction sites. The gene for aklanonic acid methyl ester cyclase was subcloned from pAcmA, derived from the *S*. *galilaeus* gene cluster and the gene encoding aklaviketone reductase was subcloned from pDx2 carrying a 2·2 kb DNA fragment of the daunomycin gene cluster cloned from *S*. *peucetius*. All plasmids were introduced into *S*. *lividans* strain TK24. Constructs are listed in Table 1.

#### Isolation and purification of products. Strains were grown in a 10 l fermenter (E1 medium, 28 °C, 500 r.p.m.) for 6 d.

Stepwise elucidation of auramycinone biosynthesis

Table 1. Plasmids used in this study

sno, acm and dau indicate that the gene was cloned from S. nogalater, S. galilaeus and S. peucetius, respectively. ARO, aromatase; KR, polyketide ketoreductase; OXY, 12-mono-oxygenase; MET, nogalonic acid 14-methyltransferase.

| Plasmid       | Relevant characteristics                                                                 | Reference or source                  |
|---------------|-----------------------------------------------------------------------------------------|--------------------------------------|
| pSL1180       | E.coli cloning vector                                                                  | Brosius (1989)                      |
| pUC19        | E.coli cloning vector                                                                  | Yanisch-Perron et al. (1985)         |
| pIJ486       | Streptomyces cloning vector                                                            | Ward et al. (1986)                   |
| pSY21(10-12)* | Expression vector for sno minimal PKS                                                   | Ylihonko et al. (1996b)              |
| pSY15(1-12)*  | Causes production of the nogalamycin chromophore                                       | Ylihonko et al. (1996a)              |
| pAcmA        | acmA (AAME cyclase) in pUC19                                                           | This work                            |
| pDx2         | dauE (aklavinone reductase) in pUC19                                                    | This work                            |
| pSY21a(3-6)* | snoD (KR) added to pSY21                                                                | This work                            |
| pSY21b(2-5)* | snoD (KR) and snoE (ARO) added to pSY21                                                 | This work                            |
| pSY21c(7-9)* | snoB (OXY) added to pSY21b                                                             | This work                            |
| pSY21d(4-8)* | snoB (OXY) and snoC (MET) added to pSY21b                                               | This work                            |
| pSY15a       | acmA (AAME cyclase) + added to pSY15                                                    | This work                            |
| pSY15b       | dauE (aklavinone reductase) + added to pSY15                                            | This work                            |

* Restriction sites used for cloning are indicated by the numbers in parentheses. The restriction sites correspond to Fig.2a.
+ Restriction sites used for cloning are indicated by asterisks in Fig.2b, c.

Compounds were extracted separately from the mycelium and supernatant with dichloromethane/methanol (3:1, v/v) at pH 3.0. Solvents were removed under vacuum and the viscous residue was chromatographed through a polyamide 11 column using water/methanol from 1:9 to 0:10 (v/v) as eluent. Individual fractions were further purified using a preparative reversed-phase (C-18) column with acetonitrile/1% acetic acid in water (1:1, v/v) as mobile phase. Evaporation of the solvent under vacuum gave pure products.

Instrumental analysis. 1H and 13C NMR spectra using CDCl₃ or DMSO-d₆ as solvent were recorded on a JEOL JNM-GX 400 spectrometer. NMR analysis included NON (1D proton spectra), BMC (1D carbon spectra), NOE (nuclear Overhauser effect), DEPT (distortionless enhancement by polarization transfer) and HMBC (heteronuclear multiple-bond connectivities) techniques. Spectra were internally referenced to tetramethylsilane. MS was performed on a Varian VG707E spectrometer. Metabolites were detected by HPLC (LaChrom, Merck Hitachi, pump L-7100, detector L-7400 and integrator D-7500) using a LiChroCART RP-18 column.

also a precursor of daunomycin, whereas the aglycone moiety of nogalamycin differs from aklavinone in the nature of its side chain and its configuration at C-9 (see Fig. 1 for differences).

A sno cluster containing the nogalamycin biosynthetic region was discovered by sequential use of the actI and acm probes (Ylihonko et al., 1996a). The sequence is listed in GenBank under the accession number (Z48262)/AJ224512. It was used as the source of seven structural genes encoding activities needed for poly-ketide biosynthesis and of the gene responsible for esterification of the aklanonic acid analogue, nogalonic acid (Fig. 3). A fragment containing the gene required for each biosynthetic step was subcloned from the sno cluster using convenient restriction sites shown in Table 1 and Fig. 2(a).

The actI probe (Malpartida et al., 1987) was used for hybridization to locate and clone the acm cluster from S. galilaeus. A 3kb BamHI fragment of the acm cluster ('acm probe') containing the gene acmA encodes AAME cyclase responsible for closing the last ring of aklavinone (A-ring, Fig. 1a) in aclacinomycin biosynthesis. acmA was similar (83% similarity and 70% identity at the amino acid level) to dauD encoding AAME cyclase for daunomycin biosynthesis (Dickens et al., 1995). The gene was inserted into pSY15 from the 3kb BamHI fragment using restriction sites Eco47III and BamHI (Fig. 2b). The DNA sequence of acmA was deposited in GenBank under the accession number AF043550.

RESULTS

Cloning of anthracycline biosynthesis clusters

To clone single genes from different anthracycline pathways, gene clusters from S. nogalater (sno), S. galilaeus (acm) and S. peucetius (dau) (Fig. 2) were used. These strains produce nogalamycin, aclacinomycins and daunomycin, respectively (see Fig. 1 for structures). Aklavinone, the aglycone moiety of aclacinomycins, is

J. KANTOLA and OTHERS

(a)

1 kb


| BglII | MluI | SphI | BspMII | SpII | SacI | BglII |
|-------|------|------|--------|------|------|-------|
| 1     | 2    | 4    | 5      | 8    | 10   | 12    |


NotI
3



snoE
ARO



snoD
KR



NotI SphI
6 7



snoC snoB
MET OXY ACTIVATOR



Eco47III
9



HindIII
11



sno1
KSI



sno2
KSII



sno3
ACP


pSY15

pSY21

(b)


BamHI
SphI Smal Smal Eco47III* BamHI*



acmA
CYC


pAcmA

(c)


BamHI AvrII* SalI SacI* BamHI



dauE


pDx2

Fig. 2. The fragments of anthracycline biosynthesis clusters used in this study. (a) *S. nogalater* genomic DNA (*sno*) carrying part of the nogalamycin biosynthetic cluster. (b) *S. galilaeus* genomic DNA (*acm*) carrying part of aclacinomycin biosynthetic cluster. (c) *S. peucetius* genomic DNA (*dau*) carrying part of daunomycin biosynthetic cluster. The ORFs are shown by arrows. DNA fragment inserts in pSY15, pSY21, pAcmA and pDx2 are indicated. The numbering of restriction sites used for cloning *sno* ORFs corresponds to that in Table 1. The restriction sites used for cloning *acmA* and *dauE* are marked with asterisks. Abbreviations of ORF functions: ARO, aromatase; KR, polyketide ketoreductase; MET, nogalonic acid 14-methyltransferase; OXY, 12-mono-oxygenase; KSI, ketosynthase I; KSII, ketosynthase II; ACP, acyl carrier protein; CYC, alkanonic acid methyl ester cyclase. *dauE* is aklaviketone reductase.

---

Adau cluster covering about 60 kb contiguous DNA for daunomycin biosynthesis was located and cloned by hybridizing to the *acm* probe. A 2.2 kb DNA fragment (pDx2) of the *dau* cluster complemented H036, a *S. galilaeus* mutant accumulating aklaviketone. Sequencing and sequence analysis of pDx2 revealed one complete ORF named *dauE*, because the nucleotide sequence was 98% identical to the gene (*dauE*) previously characterized from *Streptomyces sp.* strain C5 (Dickens et al., 1996) encoding aklaviketone reductase. A SacI-AvrII fragment of pDx2 (Fig. 2c) was used to introduce the C-7 ketoreductase gene into the final construct, pSY15b (Fig. 4).

Products generated by pSY21-and pSY15-based clones

Plasmids pSY21 and pSY15 were used as expression vectors into which the DNA fragments containing single specific genes were sequentially introduced. Plasmid pSY21 is the expression construct for nogalamycin minPKS and pSY15 contains the genes needed to form the nogalamycin chromophore (Ylihonko et al., 1996b).

The route for preparation of constructs that finally resulted in pSY21d and pSY15b (Fig. 4) is shown in Table 1. The products from separate expression constructs were first studied in *S. lividans* TK24 and, whenever possible, the structures of the compounds accumulated were determined. Spectral data for structural elucidation of the compounds IVC, VA-VD, VI and VII are shown in Table 2.

Plasmids pSY21, pSY21a and pSY21b each caused the production of a confusing mixture of dozens of unstable compounds in *S. lividans*. No structural analysis of these compounds was carried out, since the relative amounts differed in separate fermentation batches. However, using the *S. galilaeus* blocked mutants and *S. steffisburgensis* as the hosts, the functionality of the various constructs was analysed.

Plasmid pSY21 was introduced into the *S. galilaeus* mutant H039, which produces aklavinone with attached rhodinose residues (Ylihonko et al., 1994). When *S. nogalater* minPKS in pSY21 was expressed in H039, the hydrolysed products obtained were aklavinone and auramycinone, whereas aklavinone was obtained after

Stepwise elucidation of auramycinone biosynthesis

10 Acetates

I $\downarrow$ min PKS (snoA + snoI–3) in pSY21

$\begin{aligned} & \text{[ } \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \text{O} \\ & \quad \text{C} \\ & \quad \
J. KANTOLA and OTHERS

![Diagram](#)

synthesis (Ylihonko *et al.*, 1996b). When the hydrolysed products of H061/pSY21a were analysed, the aglycones obtained were aklavinone and auramycinone, as expected. H061 accumulates a 2-OH compound exhibiting a different folding pattern from anthra- cyclines (Kantola *et al.*, 1997).

*S. steffisburgensis* producing steffimycin (see Fig. 1 for the structure of 8-demethoxy steffimycin) was employed to analyse the effect of co-expression of aromatase with ketoreductase in pSY21b. We recently characterized the products of *S. steffisburgensis* (Kunnari *et al.*, 1997) and found that the main product of *S. steffisburgensis* (ATCC 27466) was 8-demethoxy steffimycin in the conditions used for fermentation. Steffimycinone differs from nogalamycin aglycone in position 2, the site for polyketide ketoreductase action (Figs 1 and 3) in nogalamycin biosynthesis. The absence of this step in the steffimycin pathway results in a methoxy group at C-2 in steffimycin. Expression of *snoD* and *snoE* in *S. steffisburgensis* caused the production of 2,8-demethoxy steffimycins (Kunnari *et al.*, 1997) and pSY21b generated the same products in this strain, confirming the functionality of the construct.

Plasmid pSY21c was the first construct that caused the production of stable compounds in TK24 and in the same proportion in separate fermentation batches. Structural analysis of these products revealed the aklanonic acid analogue nogalonic acid (IVA, Fig. 3) supporting the function of *snoB* as an oxygenase and suggesting that *snoB* is probably the last component in the PKS complex, as a stable compound was formed. Nevertheless, the major compound obtained (IVC) was a hybrid, perhaps caused by the concomitant action of cloned nogalamycin genes and the host's endogenous actinorhodin biosynthesis genes. Furthermore, a com- pound IVB of 18 carbons was also accumulated. IVB was suggested to be a biosynthetic product derived from nine acetates, presumably derived from the flexibility of minPKS in chain elongation of a growing polyketide chain. This was, however, a minor product. We have also found that some *S. galilaeus* mutants accumulating intermediates in aklavinone biosynthesis produce anthracycline metabolites derived from shorter poly- ketide chains than expected if the typical ten building blocks were used.

Plasmid pSY21d contains the same genes for bio- synthesis of the aglycone as pSY15, and the product profiles of TK24/pSY21d and TK24/pSY15 were identical, as expected. The accumulation of nogalonic acid methyl ester (VA) demonstrated that *snoC* is responsible for methylation of a carboxy group at C-10. The structures of the compounds produced by TK24/pSY15 are shown in Fig. 3. All the compounds, including the intermediate IVA, the shunt product IVB and the hybrid product IVC, were converted to the methylated forms by the action of SnoC, resulting in VA, VB and VC, respectively. Furthermore, the reduced compound VD was accumulated in TK24/pSY21d as in TK24/pSY15. Because the product profiles caused by pSY15 and

hydrolysis of H039 products. Aklavinone and auramycinone differ only in the ethyl or methyl substituent at C-9, respectively, due to use of a different starter unit in building the polyketide chain (Ylihonko *et al.*, 1996b). Plasmid pSY21a was introduced into the polyketide- ketoreductase-deficient *S. galilaeus* mutant H061 for complementation. H061 was previously complemented to produce aklavinone glycosides by *snoD* encoding polyketide ketoreductase (KR) for nogalamycin bio-

Stepwise elucidation of auramycinone biosynthesis

Table 2. Spectral data of the compounds VII, VI, VA, VD, VB, IVC and VC shown in Fig. 3

(a) ${}^{1}$H NMR (400 MHz) chemical shifts. The spectra were measured in CDCl₃ except VB, which was measured in DMSO/CDCl₃. All spectra were internally referenced to tetramethyl silane.

| Site | VII | VI | VA | VD | VB | IVC | VC |
|------|-----|----|----|----|----|-----|----|
| 1     | 7·69, 1H, dd, 7·5, 1·3 | 7·75, 1H, dd, 7·5, 1·2 | 7·85, 1H, dd, 7·5, 1·1 | 7·80, 1H, dd, 7·6, 1·2 | 7·85, 1H, dd, 8·4, 1·2 | 7·18, 1H, dd, 8·4, 1·0 | 7·19, 1H, dd, 7·8, 1·0 |
| 2     | 7·62, 1H, dd, 8·3, 7·5 | 7·65, 1H, dd, 8·4, 7·5 | 7·72, 1H, dd, 8·4, 7·6 | 7·70, 1H, dd, 8·4, 7·6 | 7·73, 1H, dd, 8·5, 7·4 | 7·38, 1H, dd, 8·4, 7·8 | 7·45, 1H, dd, 7·8, 7·3 |
| 3     | 7·23, 1H, dd, 8·3, 1·3 | 7·30, 1H, 8·4, 1·2 | 7·33, 1H, dd, 8·4, 1·1 | 7·31, 1H, dd, 8·4, 1·2 | 7·34, 1H, dd, 8·5, 1·2 | 6·82, 1H, dd, 7·8, 1·0 | 7·03, 1H, dd, 7·3, 1·0 |
| 4-OH 11·78, 1H, s | 12·59, 1H, s | 11·94, 1H, s | 11·85, 1H, s | 11·95, 1H, s | 11·16, 1H, s | — | — |
| 6-OH 12·55, 1H, s | 13·90, 1H, s | 12·58, 1H, s | 12·52, 1H, s | — | — | — | — |
| 6a    | — | — | — | — | 5·90, 1H, s | — | 5·75, 1H, s |
| 7     | 5·33, 1H, dd, 5·1, 1·8 | — | — | — | — | — | — |
| 8A    | 2·58, 1H, dd, 14·9, 5·1 | 3·45, 1H, d, 17·6 | 5·94, 1H, s, (enol) | 3·25, 1H, dd, 17·7, 2·7 | 2·96, 3H, s | 5·44, 1H, s | 5·38, 1H, s |
| 8B    | 2·22, 1H, ddd, 14·9, 1·8, | 2·83, 1H, dd, 17·6, 1·7 | 4·18, 2H, s, (keto) | 3·11, 1H, dd, 17·7, 9·0 | — | — | — |
| 1·5   | — | — | — | 4·42, 1H, ddd, 9·0, 6·3, | — | — | — |
|       |       |       |       | 2·7 |       |       |       |
| 9     | — | — | 15·46, 1H, brs (enol) | — | — | — | — |
| 9-OH 4·28, 1H, s | — | — | — | — | — | — | — |
| 10A   | 4·04, 1H, d, 1·4 | 4·18, 1H, d, 1·7 | 3·86, 2H, s | 3·86, 1H, s | 3·88, 2H, s | 3·18, 1H, dd, 16·4, 3·1 | 3·20, 1H, dd, 16·0, 3·0 |
| 10B   | — | — | — | — | — | 3·00, 1H, dd, 16·4, 10·6 | 3·03, 1H, dd, 16·0, 10·2 |
| 10a   | — | — | — | — | — | 4·69, 1H, cm | 4·73, 1H, cm |
| 11A   | 7·55, 1H, s | 7·71, 1H, s | 7·77, 1H, s | 7·68, 1H, s | 7·71, 1H, s | 2·83, 1H, dd, 16·1, 7·8 | 2·91, 1H, dd, 15·8, 7·2 |
| 11B   | — | — | — | — | — | 2·74, 1H, dd, 16·1, 7·8 | 2·75, 1H, dd, 15·8, 5·8 |
| 12    | — | — | — | — | — | 7·32, 1H, s | 7·28, 1H, s |
| 13    | 1·41, 3H, s | 1·53, 3H, s | 2·20, 3H, s, (enol) 2·32, | 1·29, 3H, d, 6·3 | — | 2·03, 3H, s | 2·15, 3H, s |
|       |       |       | 3H, s, (keto) |       |       |       |       |
| 15    | 3·70, 3H, s | 3·76, 3H, s | 3·72, 3H, s | 3·73, 3H, s | — | — | 3·77, 3H, s |

(b) ${}^{13}$C NMR (100 MHz) chemical shifts measured in CDCl₃ and internally referenced to tetramethyl silane.

| Site | VII | VA | VD | IVC | VC |
|------|-----|----|----|-----|----|
| 1     | 120·2 | 122·3 | 120·4 | 118·1 | 118·3 |
| 2     | 137·4 | 137·5 | 137·7 | 130·1 | 130·9 |
| 3     | 124·8 | 125·0 | 125·1 | 112·5 | 113·7 |
| 4     | 162·4 | 162·6 | 162·7 | 154·3 | 157·6 |
| 4a    | 115·5 | 115·6 | 115·6 | 111·5 | 112·6 |
| 5     | 192·3 | 184·4 | 192·6 | 149·9 | 155·5 |
| 5a    | 114·4 | 115·1 | 115·4 | 106·8 | 106·9 |
| 6     | 161·1 | 160·1 | 160·1 | 157·4 | 157·6 |
| 6a    | 133·2 | 132·2 | 135·2 | 96·8 | 97·6 |
| 7     | 62·4 | 192·5 | 205·7 | 161·7 | 162·2 |
| 8     | 37·0 | 103·8 | 52·6 | 96·6 | 97·1 |
| 9     | 69·9 | 191·3 | 64·3 | 192·8 | 194·3 |
| 10    | 57·8 | 39·3 | 38·8 | 39·2 | 39·6 |
| 10a   | 142·3 | 142·7 | 141·9 | 74·2 | 74·0 |
| 11    | 121·1 | 120·3 | 120·4 | 31·2 | 32·2 |
| 11a   | 132·6 | 133·2 | 133·2 | 125·7 | 124·7 |
| 12    | 180·9 | 180·9 | 180·8 | 121·7 | 122·1 |
| 12a   | 132·4 | 133·6 | 133·7 | 136·5 | 137·1 |
| 13    | 27·5 | 24·9 | 22·8 | 30·0 | 30·4 |
| 14    | 171·3 | 170·9 | 170·5 | 170·6 | 170·0 |
| 15    | 52·5 | 52·3 | 52·6 | — | 52·2 |

(c) MS data for compounds VII, VB, IVC and VC. EIMS, electron impact mass spectroscopy.

| Compound | EIMS (m/e) (relative intensity) |
|----------|---------------------------------|
| VII      | 398 (M⁺, 100), 380 (20), 362 (100), 331 (32), 321 (47) |
| VB       | 354 (M⁺, 100), 307 (60), 294 (88), 251 (8) |
| IVC      | 352 (M⁺, 100), 337 (90), 293 (70), 251 (20) |
| VC       | 366 (M⁺, 100), 351 (75), 293 (52), 251 (13) |

pSY21d were identical, pSY15 was chosen as a vector for preparation of the last two constructs to allow the usage of suitable restriction sites for cloning.

Plasmid pSY15a, expressing acmA in addition to genes covering steps 1–5 (Fig. 3), caused the accumulation of auraviketone (VI) in TK24. Finally, the addition of the SacI-AvrII fragment carrying dauE, aC-7 ketoreductase from S. peucetius, to pSY15a generated pSY15b,

resulting in the production of auramycinone in S. lividans TK24. Auramycinone made up about 30% of the detectable metabolites.

DISCUSSION

According to the proposed biosynthetic pathway (Ylihonko et al., 1996b), we speculated that a specific cyclase is needed to close each aromatic ring and 11

structural genes are required for an anthracyclinone pathway. The work presented here suggests, however, that nine structural genes are needed to form an anthracyclinone in S. lividans.

The host chosen for this study was S. lividans TK24 even though Streptomyces coelicolor CH999 has been frequently used in the investigation of aromatic polyketide metabolites (e.g. Zawada & Khosla, 1997; Fu et al., 1996). CH999 has been genetically engineered to lack all the genes needed for actinorhodin biosynthesis (McDaniel et al., 1993). Unfortunately, our attempts to produce metabolites with pSY15- or pSY21-based CH999 clones were not successful, since aromatic polyketides were not accumulated in the liquid cultures. Plasmids pSY21c- and pSY15-based constructs caused the formation of 20-carbon hybrid products in TK24 (Fig. 3, IVC, VC), whereas metabolites of 16 carbons (actinorhodin is built up from eight acetate units; Fig. 1) were not recovered. SnoA is an activator that presumably promotes expression of other polyketide genes in addition to nogalamycin biosynthetic genes. An expression plasmid comparable to pSY21c but containing the ermE promoter (Bibb et al., 1985) in pIJ486 and not containing snoA, decreased the production of metabolites though hybrid products were not obtained in the culture extract. However, the yield was too low for structural analysis of the compounds (J. Kantola, unpublished results).

The production of stable compounds when oxygenase was present suggests that this enzyme is the last PKS component in forming the product, which is then released from the enzyme complex. The hybrid products obtained (Fig. 3, IVC, VC) sharing structural features of actinorhodin (Fig. 1) and nogalamycin were not oxygenated, suggesting that the reaction possibly directed at the actVI locus (Fernandez-Moreno et al., 1994; Ichinose et al., 1999) is competitive with that directed by the oxygenase. The factor that determines condensation of the second and third rings was not, however, clarified. The presence of oxygenase (or a corresponding enzyme) with the PKS may result in the correct orientation of a polyketide chain to allow condensation of two aromatic rings (B and C), suggesting that specific cyclases are not essential. This hypothesis, that no separate gene products are needed for the cyclizations, is consistent with the minimal construct being able to cause the production of aklanonic acid for daunomycin biosynthesis (Rajgarhia & Strohl, 1997; Gerlitz et al., 1997). On the other hand, dpsY, recently identified in the daunomycin cluster, is probably essential for closing of the aromatic rings (Lomovskaya et al., 1998), which is why we cannot rule out the possibility that cyclization activities were derived from TK24. Also DpsH has been mentioned to be involved in closing the second and the third rings (Gerlitz et al., 1997), but recently published results by Lomovskaya et al. (1999), have suggested that DpsH is involved in daunosamine biosynthesis or its attachment to rhodomycinone. Since spontaneous cyclization reactions result in only minor amounts of products (generally less than 1%), the fact that aura-

mycinone was a major product (30% of the whole extract) suggests that an enzymatic reaction was involved.

# ACKNOWLEDGEMENTS

This work was supported by the Academy of Finland, the National Technology Agency, Finland (TEKES) and the European Union (BIOTECH program BIO4-CT96-0068).

# REFERENCES

Aramone, F., Cassinelli, G., Fantini, G., Grein, A., Orezzi, P., Pol, C. & Spalla, C. (1969). Adriamycin, 14-hydroxydaunomycin, a new antitumour antibiotic from *S. peucetius* var. *caesius*. *Biotechnol Bioeng* 11, 1101–1110.

Bhuyan, B.K. & Dietz, A. (1965). Fermentation, taxonomic, and biological studies of nogalamycin. *J Antimicrob Chemother* 5, 836–844.

Bibb, M.J., Janssen, G.R. & Ward, J.M. (1985). Cloning and analysis of the promoter region of the erythromycin resistance gene (*ermE*) of *Streptomyces erythraeus*. *Gene* 38, 215–226.

Brosius, J. (1989). Laboratory methods: superpolylinkers in cloning and expression vectors. *DNA* 8, 759–777.

Dickens, M.L., Ye, J. & Strohl, W.R. (1995). Analysis of clustered genes encoding both early and late steps in daunomycin biosynthesis by *Streptomyces sp.* strain C5. *J Bacteriol* 177, 536–543.

Dickens, M.L., Ye, J. & Strohl, W.R. (1996). Cloning, sequencing and analysis of aklaviketone reductase from *Streptomyces sp.* strain C5. *J Bacteriol* 178, 3384–3388.

Dietz, A. (1967). *Streptomyces steffisburgensis* sp.n. *J Bacteriol* 94, 2022–2026.

Fernandez-Moreno, M.A., Martinez, E., Caballero, J.L., Ichinose, K., Hopwood, D.A. & Malpartida, F. (1994). DNA sequence and functions of the *actVI* region of the actinorhodin biosynthetic gene cluster of *Streptomyces coelicolor* A3(2). *J Biol Chem* 269, 24854–24863.

Fu, H., Alvarez, M.A., Khosla, C. & Bailey, J.E. (1996). Engineered biosynthesis of novel polyketides: regiospecific methylation of an unnatural substrate by the *tcmO* O-methyltransferase. *Biochemistry* 35, 6527–6532.

Fujii, I. & Ebizuka, Y. (1997). Anthracycline biosynthesis in *Streptomyces galilaeus*. *Chem Rev* 97, 2511–2523.

Fujiwara, A., Hoshino, T., Tazoe, M. & Fujiwara, M. (1982). New anthracycline antibiotics, auramycins and sulfurmycins. I. Isolation and characterization of auramycins A and B, and sulfurmycins A and B. *J Antibiot* 35, 164–175.

Gerlitz, M., Meurer, G., Wendt-Pienkowski, E., Madduri, K. & Hutchinson, C.R. (1997). The effect of the daunorubicin *dpsH* gene on the choice of starter unit and cyclization pattern reveals that Type II polyketide synthases can be unfaithful yet intriguing. *J Am Chem Soc* 119, 7392–7393.

Hopwood, D.A. (1997). Genetic contributions to understanding polyketide synthases. *Chem Rev* 97, 2465–2497.

Hopwood, D.A., Bibb, M.J., Chater, K.F. & 7 other authors (1985). *Genetic Manipulation of Streptomyces: a Laboratory Manual*. Norwich: The John Innes Foundation.

Hutchinson, C.R. & Fujii, I. (1995). Polyketide synthase gene manipulation: a structure-function approach in engineering novel antibiotics. *Annu Rev Microbiol* 49, 201–238.

Ichinose, K., Surti, C., Taguchi, T., Malpartida, F., Booker-Milburn, K.I., Stephenson, G.R., Ebizuka, Y. & Hopwood, D.A. (1999).

Proof that the actVI genetic region of Streptomyces coelicolor A3(2) is involved in stereospecific pyran ring formation in the biosynthesis of actinorhodin. Bioorg Med Chem Lett 9, 395–400.

Kantola, J., Blanco, G., Hautala, A., Kunnari, T., Hakala, J., Mendez, C., Ylihonko, K., Mäntsälä, P. & Salas, J. (1997). Folding of the polyketide chain is not dictated by minimal polyketide synthase in the biosynthesis of mithramycin and anthracycline. Chem Biol 4, 751–755.

Kunnari, T., Tuikkanen, J., Hautala, A., Hakala, J., Ylihonko, K. & Mäntsälä, P. (1997). Isolation and characterization of 8-demethoxy steffimycins and generation of 2,8-demethoxy steffimycins in Streptomyces steffisburgensis by the nogalamycin biosynthesis genes. J Antibiot 50, 496–501.

Lomovskaya, N., Doi-Katayama, Y., Filippini, S., Nastro, C., Fonstein, L., Gallo, M., Colombo, A. L. & Hutchinson, C. R. (1998). The Streptomyces peucetius dpsY and dnrX genes govern early and late steps of daunorubicin and doxorubicin biosynthesis. J Bacteriol 180, 2379–2386.

Lomovskaya, N., Otten, S. L., Doi-Katayama, Y. & 8 other authors (1999). Doxorubicin overproduction in Streptomyces peucetius: cloning and characterization of the dnrU ketoreductase and dnrV genes and the doxA cytochrome P-450 hydroxylase gene. J Bacteriol 181, 305–318.

McDaniel, R., Ebert-Khosla, S., Hopwood, D. A. & Khosla, C. (1993). Engineered biosynthesis of novel polyketides. Science 262, 1546–1550.

McDaniel, R., Ebert-Khosla, S., Hopwood, D. A. & Khosla, C. (1995). Rational design of aromatic polyketide natural products by recombinant assembly of enzymatic subunits. Nature 375, 549–554.

Madduri, K. & Hutchinson, C. R. (1995). Functional characterization and transcriptional analysis of a gene cluster governing early and late steps in daunorubicin biosynthesis in Streptomyces peucetius. J Bacteriol 177, 3879–3884.

Malpartida, F., Hallam, S. E., Kieser, H. M. & 9 other authors (1987). Homology between Streptomyces genes coding for synthesis of different polyketides and its use to clone antibiotic biosynthetic genes. Nature 325, 818–821.

Oki, T., Matsuzawa, Y., Yoshimoto, A. & 10 other authors (1975). New antitumour antibiotics, aclacinomycins A and B. J Antibiot 28, 830–834.

Rajgarhia, V. B. & Strohl, W. R. (1997). Minimal Streptomyces sp. strain C5 daunorubicin polyketide biosynthesis genes required for aklanonic acid biosynthesis. J Bacteriol 179, 2690–2696.

Sambrook, J., Fritsch, E. F. & Maniatis, T. (1989). Molecular Cloning: a Laboratory Manual, 2nd edn. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory.

Ward, J. M., Janssen, G. R., Kieser, T., Bibb, M. J., Buttner, M. J. & Bibb, M. J. (1986). Construction and characterization of a series of multicopy promoter-probe plasmid vectors for Streptomyces using the aminoglycoside phosphotransferase from Tn5 as indicator. Mol Gen Genet 203, 468–478.

Yanisch-Perron, C., Vieira, J. & Messing, J. (1985). Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. Gene 33, 103–119.

Ylihonko, K., Hakala, J., Niemi, J., Lundell, J. & Mäntsälä, P. (1994). Isolation and characterization of aclacinomycin A-non-producing Streptomyces galilaeus (ATCC 31615) mutants. Microbiology 140, 1359–1365.

Ylihonko, K., Hakala, J., Kunnari, T. & Mäntsälä, P. (1996a). Production of hybrid anthracycline antibiotics by heterologous expression of Streptomyces nogalater nogalamycin biosynthesis genes. Microbiology 142, 1965–1972.

Ylihonko, K., Tuikkanen, J., Jussila, S., Cong, L. & Mäntsälä, P. (1996b). A gene cluster involved in nogalamycin biosynthesis from Streptomyces nogalater: sequence analysis and complementation of early-block mutations in the anthracycline pathway. Mol Gen Genet 251, 113–120.

Zawada, R. & Khosla, C. (1997). Domain analysis of the molecular recognition features of aromatic polyketide synthase subunits. J Biol Chem 272, 16184–16188.

Received 21 June 1999; revised 10 September 1999; accepted 24 September 1999.
