
ENZYME CATALYSIS AND
REGULATION:
5-Hydroxyconiferyl Aldehyde Modulates
Enzymatic Methylation for Syringyl
Monolignol Formation, a New View of
Monolignol Biosynthesis in Angiosperms

Laigeng Li, Jacqueline L. Popko, Toshiaki
Umezawa and Vincent L. Chiang
J. Biol. Chem. 2000, 275:6537-6545.
doi: 10.1074/jbc.275.9.6537

Access the most updated version of this article at [http://www.jbc.org/content/275/9/6537](http://www.jbc.org/content/275/9/6537)

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/affinity-sites).

Alerts:
- When this article is cited
- When a correction for this article is posted

Click here to choose from all of JBC's e-mail alerts

This article cites 47 references, 13 of which can be accessed free at [http://www.jbc.org/content/275/9/6537.full.html#ref-list-1](http://www.jbc.org/content/275/9/6537.full.html#ref-list-1)

THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2000 by The American Society for Biochemistry and Molecular Biology, Inc.

**5-Hydroxyconiferyl Aldehyde Modulates Enzymatic Methylation for Syringyl Monolignol Formation, a New View of Monolignol Biosynthesis**

(Received for publication, October 21, 1999, and in revised form, December 9, 1999)

Laigeng Li, Jacqueline L. Popko, Toshiaki Umezawa, and Vincent L. Chiang‡

From the Plant Biotechnology Research Center, School of Forestry and Wood Products, Michigan Technological University, Houghton, Michigan 49931

S-Adenosyl-L-methionine-dependent caffeate O-methyltransferase (COMT, EC 2.1.1.6) has traditionally been thought to catalyze the methylation of caffeate and 5-hydroxyferulate for the biosynthesis of syringyl monolignol, a lignin constituent of angiosperm wood that enables efficient lignin degradation for cellulose production. However, recent recognition that coniferyl aldehyde prevents 5-hydroxyferulate biosynthesis in lignifying tissue, and that the hydroxylated form of coniferyl aldehyde, 5-hydroxyconiferyl aldehyde, is an alternative COMT substrate, demands a re-evaluation of the role of COMT during monolignol biosynthesis. Based on recombinant aspen (*Populus tremuloides*) COMT enzyme kinetics coupled with mass spectrometry analysis, this study establishes for the first time that COMT is in fact a 5-hydroxyconiferyl aldehyde O-methyltransferase (AldOMT), and that 5-hydroxyconiferyl aldehyde is both the preferred AldOMT substrate and an inhibitor of caffeate and 5-hydroxyferulate methylation, as measured by \(K_m\) and \(K_i\) values. 5-Hydroxyconiferyl aldehyde also inhibited the caffeate and 5-hydroxyferulate methylation activities of xylem proteins from various angiosperm tree species. The evidence that syringyl monolignol biosynthesis is independent of caffeate and 5-hydroxyferulate methylation supports our previous discovery that coniferyl aldehyde prevents ferulate 5-hydroxylation and at the same time ensures a coniferyl aldehyde 5-hydroxylase (CAld5H)-mediated biosynthesis of 5-hydroxyconiferyl aldehyde. Together, our results provide conclusive evidence for the presence of a CAld5H/AldOMT-catalyzed coniferyl aldehyde 5-hydroxylation/methylation pathway that directs syringyl monolignol biosynthesis in angiosperms.

considerably less energy and chemical cost than gymnosperm lignin, which consists almost entirely of guaiacyl moieties (4). As a result, there has been long standing incentive to understand the biosynthesis of syringyl monolignol in angiosperm trees in order to genetically engineer more syringyl lignin in trees to facilitate wood processing (5–8).

For more than four decades, our thinking of syringyl monolignol biosynthesis in angiosperms has followed the doctrine that S-adenosyl-L-methionine-dependent caffeate O-methyltransferase (COMT)¹-catalyzed methylation of caffeate and 5-hydroxyferulate constitutes a central pathway for the formation of syringyl monolignol via sinapate (Fig. 1A) (Refs. 9–19, reviewed in Refs. 20–24). This methylation pathway has, however, been challenged, as the involvement in this pathway (Fig. 1A) of the ferulate 5-hydroxylation step (12–14, 25) was recently overturned (26). Based on mass spectrometry characterization of the biochemical function of sweetgum (*Liquidambar styraciflua*) plant proteins as well as recombinant proteins, Osakabe *et al.* (26) showed that coniferyl aldehyde 5-hydroxylase (CAld5H)-mediated coniferyl aldehyde 5-hydroxylation followed by COMT-catalyzed methylation of 5-hydroxyconiferyl aldehyde would direct syringyl monolignol biosynthesis via sinapyl aldehyde (Fig. 1B). The evidence also led to the conclusion that in lignifying tissues there is no ferulate 5-hydroxylase *per se*. When CAld5H substrates coniferyl aldehyde (\(K_m = 2.77 \, \mu M\)) and ferulate (\(K_m = 286 \, \mu M\)) were present together, a situation that is likely to exist *in vivo*, coniferyl aldehyde completely inhibited ferulate 5-hydroxylation in a noncompetitive manner (\(K_i = 0.59 \, \mu M\)) (26). This suggests that 5-hydroxyferulate is not a lignin pathway intermediate and would thus obviate methylation of 5-hydroxyferulate to sinapate as a pathway *in vivo*. This contrasts with the well documented *in vitro* evidence that COMT methylates 5-hydroxyferulate for syringyl monolignol biosynthesis (20–25). In order to resolve this apparent inconsistency, we re-evaluated angiosperm COMT by characterizing the catalytic properties of a recombinant aspen (*Populus tremuloides*) COMT and of lignifying xylem proteins from a variety of angiosperm tree species.

Regardless of its importance to tree growth, lignin is problematic to postharvest, cellulose-based wood processing for fiber, chemical, and energy production, because it must be degraded from cellulose at great expense. Certain structural constituents of lignin, such as the guaiacyl moiety, promote monomer cross-linkages that increase lignin resistance to degradation (1–3). In angiosperms, lignin is composed of a mixture of guaiacyl and syringyl monolignols, and can be degraded at

---

* This work was supported in part by U.S. Department of Agriculture Grant 95-37103-2061 and the U.S. Department of Agriculture McIntire-Stennis Forestry Research Program. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
‡ To whom correspondence should be addressed: School of Forestry and Wood Products, Michigan Technological University, Houghton, MI 49931. Tel.: 906-487-2959; Fax: 906-487-2915; E-mail: vchiang@mtu.edu.

This paper is available online at http://www.jbc.org

---

EXPERIMENTAL PROCEDURES

Plant Materials—Differentiating stem xylem was collected during the growing season from basswood (*Tilia americana*), black ash (*Fraxinus americana*), hophorn beam (*Ostrya virginiana*), red maple (*Acer rubrum*), sugar maple (*Acer saccharum*), white birch (*Betula papyrif*

---

¹ The abbreviations used are: COMT, caffeate O-methyltransferase; CAld5H, coniferyl aldehyde 5-hydroxylase; HPLC-UV/MS, high pressure liquid chromatography-UV/mass spectrometry; SIM, selective ion monitoring; CA, caffeate; CAld, coniferyl aldehyde; AldOMT, 5-hydroxyconiferyl aldehyde O-methyltransferase; 4CL, 4-coumarate-CoA ligase; CCoA3H, 4-coumaroyl-CoA 3-hydroxylase; CCoAOMT, caffeoyl-CoA O-methyltransferase; PAL, phenylalanine ammonia lyase; Rt, retention time.

6538

**Methylation Reactions in Monolignol Biosynthesis**

stirred at 0 °C for 6 min followed by dilution with ethyl acetate and washing successively with water and saturated NaCl solution. The organic layer was then dried over Na₂SO₄ and evaporated to give 5-hydroxyconiferyl aldehyde (33.9 mg, 100%). The structure of 5-hydroxyconiferyl aldehyde was confirmed by ¹H and ¹³C NMR, CH-COSY (CH-correlation spectroscopy), COLOC (correlation spectroscopy via long-range coupling), HMBC (heteronuclear multiple bond connectivity), and MS. NMR spectra were recorded with a JNM-LA400MK FT-NMR System (JEOL). Electron impact mass spectrometry (70 eV) was recorded with a JMS-DX303HF Mass Spectrometer equipped with a JMA-DA5000 Mass Data System (JEOL). 5-Hydroxyconiferyl aldehyde: ¹H NMR δ (acetone-d₆, carbon numbers are shown in Fig. 1B), 3.88 (3H, s, OCH₃), 6.60 (1H, dd, *J* = 15.6, *J* = 7.8, C₈H), 6.88 (1H, d, *J* = 1.7, C₆H), 6.95 (1H, d, *J* = 1.7, C₂H), 7.50 (1H, d, *J* = 15.6, C₇H), 9.61 (1H, d, *J* = 7.8, C₉H); ¹³C NMR δ (acetone-d₆), 56.6 (OCH₃), 104.7 (C₂), 111.2 (C₆), 126.5 (C₁), 127.2 (C₈), 138.3 (C₄), 146.5 (C₅), 149.3 (C₃), 154.4 (C₇), 193.8 (C₉); MS *m/z* (%), 194 (M⁺, 100), 177 (10.8), 166 (24.4), 151 (53.9), 133 (8.0), 123 (21.1), 105 (7.0). All other chemicals used were obtained from Sigma/Aldrich.

Expression of Aspen COMT cDNA in Escherichia coli and Purification of COMT Recombinant Protein—Polymerase chain reaction was employed to introduce the *SphI* site at the 5′-end and *HindIII* site at the 3′-end of the coding sequence of the aspen COMT (6) using a sense primer (5′-ATCCGCATGCCGCTGGGTTCACAGGTTGAA-3′) and an antisense primer (5′-GGAAGCTTAGGGCCTTCTTGCGGGAATTC-3′). The polymerase chain reaction product was first cloned into a pCR2.1 vector (Invitrogen), from which the *SphI* and *HindIII* fragment containing the COMT coding region was then cloned into a pQE-32 (Qiagen Inc.) expression vector to fuse a His-tag at the N-terminal of COMT cDNA sequence. After sequencing to confirm the accuracy of the aspen COMT-pQE construct, the construct was transferred into *E. coli* M15-pREP₄ strain. The induction and expression of recombinant aspen COMT were conducted according to Li *et al.* (32). The M15-pREP₄ cell strain containing pQE-32 vector without a COMT cDNA insert was used as the control. After harvesting by centrifugation (2000 × g for 10 min), the cell pellet was processed to affinity purify the COMT protein using the HisBind Resin affinity purification system (Novagen) according to the manufacturer’s protocol. The purification was monitored by SDS-polyacrylamide gel electrophoresis and Western blot analysis, as described (32). After dialysis, the purified recombinant aspen COMT protein (200 μg/ml) was stored in 50 mM Tris-HCl buffer (pH 7.5) containing 2 mM β-mercaptoethanol and 30% glycerol at −20 °C until use for enzyme activity assay.

Enzyme Reaction Kinetics of Recombinant Aspen COMT Protein—Our established COMT enzymatic activity assay (27, 33) was modified to allow a direct quantification of the reaction products by HPLC-UV/Mass Spectrometry (MS) analysis for the enzyme kinetic studies. The reaction volume was 300 μl containing 50 mM Tris-HCl (pH 7.5), 2 mM MgCl₂, 5 mM β-mercaptoethanol, 100 μM S-adenosyl-L-methionine, 1 μg of purified recombinant COMT protein (~24 pmol) (boiled protein was used as the control), and varying concentrations of substrates (caffeate, 10–800 μM; 5-hydroxyferulate, 5–800 μM; 5-hydroxyconiferyl aldehyde, 0.25–50 μM). For inhibition kinetics, two sets of 5-hydroxyconiferyl aldehyde concentrations, (0.25, 0.5, 0.75, and 1 μM) and (0.75, 1, 2, and 4 μM), were used to assay the 5-hydroxyconiferyl aldehyde-induced inhibition of COMT-mediated methylation of 5-hydroxyferulate and caffeate, respectively. 5-Hydroxyferulate (10, 50, 100, and 200 μM) was used as the inhibitor for COMT-catalyzed reactions with 5-hydroxyconiferyl aldehyde and caffeate, respectively. Two sets of caffeate concentrations (25, 50, 100, and 150 μM and 10, 50, 100, and 200 μM) were tested for their inhibition effect on COMT-mediated methylation of 5-hydroxyferulate and 5-hydroxyconiferyl aldehyde, respectively. The reaction mixture was incubated at 30 °C for 5 min and the reaction was terminated by adding 10 μl of 12 N HCl, and after adding 500 ng of o-coumarate as the internal standard, the reaction mixture was analyzed directly by HPLC UV/MS.

Methylation and 5-Hydroxylation Activities of Plant Proteins from Stem Secondary Developing Xylem—Developing xylem tissue (10 g) from each tree species was ground in liquid nitrogen and homogenized at 4 °C for 2 min in 25 ml of extraction buffer (0.1 M sodium phosphate, pH 7.5) containing 0.25 M sucrose, 1 mM EDTA, 1% polyvinylpyrrolidone-40, 20 mM ascorbic acid, 5 mM β-mercaptoethanol, 1 mM phenylmethylsulfonyl fluoride, and 1 μg/ml leupeptin. The homogenate was filtered and centrifuged at 10,000 × g for 10 min at 4 °C, and the supernatant was further centrifuged at 100,000 × g for 90 min at 4 °C. The supernatant was then used for the COMT enzyme assay following the protocols described above, except the concentration of each substrate was fixed at 100 μM and the crude protein quantity at 15 μg

Methylation Reactions in Monolignol Biosynthesis

The pellet was resuspended in 1 ml of phosphate buffer (50 mM, pH 7.5) containing 0.1 mM EDTA, 30% glycerol, and 1 μg/ml leupeptin, and used immediately for the 5-hydroxylation activity assay according to Osakabe *et al.* (26), and the reaction mixture was analyzed directly by HPLC UV/MS. Protein concentrations were determined by using the Bradford dye reagent (Bio-Rad) with bovine serum albumin as the standard.

**HPLC UV/MS Analysis of Products from Enzymatic Reaction with Recombinant and Plant Proteins—** Without solvent extraction, the enzymatic reaction mixture (50 μl) was directly injected onto a Supelcosil ABZ-Plus column (25 cm × 2.1 mm × 5 μm, Supelco) maintained at 40 °C with a flow rate of 0.25 ml/min and a gradient program of 20% acetonitrile in 10 mM formic acid (pH 2.5) for 12 min and then 20–100% acetonitrile from 12 to 16 min, and held at 100% acetonitrile for 5 min. The column was equilibrated in 20% acetonitrile for 5 min before each injection. The LC system was a Hewlett-Packard LC/MSD 1100 with a diode array detector and atmospheric pressure electrospray ionization (ESI) mass spectral detection. Sinapyl aldehyde: UV (measured in 20% acetonitrile in 10 mM formic acid, pH 2.5) λ_max I 244 nm, λ_max II 344 nm, λ_min 275 nm; MS (70 V) m/z (%), 207 (100), 192 (69.0), 177 (11); retention time (Rt) 10.64 min/UV, 10.79 min/MS (the flow path separation between UV detection and MS detection was 0.15 min). Sinapate: UV, λ_max I 238 nm, λ_max II 322 nm, λ_min 264 nm; MS (70 V) m/z (%) 224 (12), 223 (100), 208 (36.0); Rt 9.45 min/UV, 9.59 min/MS. Ferulate: UV λ_max I 236 nm, λ_max II 322 nm, λ_min 262 nm; MS (70 V) m/z (%) 193 (100), 178 (43), 149 (13), 134 (13); Rt 10.80 min/UV, 10.95 min/MS. Based on these properties of the authentic compounds, the detection and identification of the reaction products by the diode array detector was at their respective λ_max (344 nm for sinapyl aldehyde, 322 nm for sinapate and ferulate) and MS detection was in selective ion monitoring (SIM) negative ion mode (m/z 207.1 for sinapyl aldehyde, 223.1 for sinapate, 193.1 for ferulate, and 163.1 for o-coumarate) with a fragment voltage of 70 at which [M – H]⁻ is usually the base peak in the MS spectrum. The identity of the product was further confirmed by comparing the ion fragmentation patterns of the product and authentic standard in MS scanning mode at 70 and 150 V. The product quantity was then obtained based on the response ratio of the authentic compound/internal standard using the UV signal.

**Measurement of Kinetic Constants for Recombinant Aspen COMT Protein—** $K_m$ and $V_{max}$ values were determined from Lineweaver-Burk plots, and $k_{cat}$ by dividing $V_{max}$ by the enzyme concentration, based on two to three independent assays. $K_i$ was derived from the slope of a plot of the apparent $K_m$ for the substrate versus the corresponding inhibitor concentration at which the apparent $K_m$ was obtained.

**RESULTS**

Traditionally, functional characterization of lignin pathway O-methyltransferase has not given consideration to the purity and structural identity of enzymatic reaction products in question. In this study an approach to directly quantify the structurally confirmed products was adapted. Non-radioactive S-adenosyl-L-methionine was used as the methyl donor and the reaction mixture was directly subjected to HPLC separation without solvent extraction to avoid variability in extraction efficiency for different organic compounds. The product structural identity was corroborated by diode array UV and MS signature comparison to authentic compounds. A major advantage of the current approach is that it allows unequivocal quantification of reaction products derived from the simultaneous reaction with an array of substrates. This enabled characterization of the protein-multisubstrate interactions likely to be typical of lignifying tissues where monolignol biosynthetic pathway intermediates, such as coniferyl aldehyde derivatives, are present together (26).

**Structural Confirmation of COMT Enzymatic Reaction Products Using E. coli-expressed and Purified Recombinant Aspen COMT—** 5-Hydroxyconiferyl aldehyde, 5-hydroxyferulate (both were chemically synthesized, see “Experimental Procedures”), and caffeate, were incubated individually with purified recombinant aspen COMT from *E. coli*. Diode array detector (at 344 nm, the λ_max of sinapyl aldehyde) analysis of reaction products from 5-hydroxyconiferyl aldehyde (denoted as Sample in green in Fig. 2A) revealed a single peak with a retention time (Rt) of 10.64 min which co-eluted with authentic sinapyl aldehyde. No such peak could be detected in control (boiled protein) reaction (purple). To confirm the authenticity of this sinapyl aldehyde peak, the control and sample HPLC eluants were analyzed by MS, and the mass spectral data was acquired in the SIM mode targeting the [M – H]⁻ fragment ion of m/z 207 from sinapyl aldehyde. The total ion chromatogram of the sample eluent (red) showed a major peak at 10.79 min, which was absent in the control (black) and corresponds to the sinapyl aldehyde peak at 10.64 min (green) detected by diode array detector. Furthermore, the relative intensities of the fragmented ions (Fig. 2A, inset) derived from this 10.79-min MS peak were identical to those from authentic sinapyl aldehyde, confirming that the aldehyde precursor can be methylated during monolignol biosynthesis and that COMT can catalyze this reaction. Using the same analytical system, the COMT-catalyzed methylation of 5-hydroxyferulate (Fig. 2B) and caffeate (Fig. 2C) for the formation of sinapate and ferulate, respectively, were also confirmed. We then conducted the kinetics of COMT-catalyzed reactions.

**Kinetic Analysis of COMT-catalyzed Methylation of 5-Hydroxyconiferyl Aldehyde, 5-Hydroxyferulate, and Caffeate—** Lineweaver-Burk analysis indicated that *E. coli*-expressed COMT recombinant protein exhibited a similar turnover rate with each of the three different phenolic substrates used (Table I). However, COMT showed the highest affinity for 5-hydroxyconiferyl aldehyde with a $V_{max}/K_m$ value 5 and 31 times those for 5-hydroxyferulate and caffeate, respectively, overturning the conventional claim that 5-hydroxyferulate is the best COMT substrate (16–19, 28, 34). The kinetic data further suggest that COMT is more specific for 5-methylation than for 3-methylation, and that, in the presence of all three potential substrates, COMT-catalyzed methylation of 5-hydroxyconiferyl aldehyde would be the dominant reaction leading to syringyl monolignol biosynthesis. In light of the discovery of Osakabe *et al.* (26) that noncompetitive inhibition among substrates modulates lignin pathway CAld5H activity, we tested whether the COMT function may also be regulated by its substrates. A mixture of equal molar caffeate, 5-hydroxyferulate and 5-hydroxyconiferyl aldehyde was incubated with recombinant COMT. This mixed substrate reaction resulted in a complete inhibition of caffeate and 5-hydroxyferulate methylation, but the conversion of 5-hydroxyconiferyl aldehyde into sinapyl aldehyde was conserved (Fig. 2D). Thus, while CAld5H function is modulated by coniferyl aldehyde (26), COMT activity, likewise, is regulated by its substrate 5-hydroxyconiferyl aldehyde.

Our interpretation of these results is that COMT associated with monolignol biosynthesis is a 5-hydroxyconiferyl aldehyde O-methyltransferase (AldOMT) with some affinity for caffeate and 5-hydroxyferulate. Therefore, we refer to this OMT enzyme as AldOMT, and propose that those plant OMT genes encoding EC 2.1.1.6 (35) also encode a 5-hydroxyconiferyl aldehyde O-methyltransferase. To understand how 5-hydroxyconiferyl aldehyde may inhibit AldOMT function with 5-hydroxyferulate and caffeate *in vivo*, we studied AldOMT enzyme inhibition kinetics.

**AldOMT Enzyme Inhibition Kinetics—** Kinetic analysis showed that 5-hydroxyconiferyl aldehyde competitively inhibited AldOMT-catalyzed methylation of 5-hydroxyferulate (Fig. 3A) with a $K_i$ of 0.26 μM (Fig. 3A, inset). On the other hand, 5-hydroxyferulate was also a competitive inhibitor for the methylation of 5-hydroxyconiferyl aldehyde (Fig. 3B), but with a $K_i$ (Fig. 3B, inset) over 400 times that for 5-hydroxyconiferyl aldehyde. Furthermore, the $K_i$ value for 5-hydroxyferulate as an inhibitor of AldOMT-catalyzed methylation of 5-hydroxyconiferyl aldehyde is over 7-fold higher than the $K_m$ (Table I) for

6540 Methylation Reactions in Monolignol Biosynthesis

FIG. 2. HPLC separation of products from enzymatic reactions with recombinant COMT and plant xylem proteins. Reaction conditions and product separation and identification are described under “Experimental and Procedures.” A, HPLC-UV (344 nm) and HPLC-MS (SIM, 70 V; $m/z$ 207.1) chromatograms of control (boiled recombinant protein: purple, UV; black, MS) and sample (purified recombinant COMT protein: 24 pmol, green, UV; red, MS) reaction mixtures with 5-hydroxyconiferyl aldehyde (50 $\mu$M) as the substrate (Rt 6.83 min/UV). The internal standard (I.S.) was o-coumarate (Rt 19.00 min/UV). Inset A, negative ion electrospray (NI-ES) mass spectrum (scanning mode at 70 V) of the reaction product sinapyl aldehyde (Rt 10.79 min/MS, 10.64 min/UV) is identical to that of the authentic standard. B, HPLC-UV (322 nm) and HPLC-MS (SIM, 70 V; $m/z$ 223.1) chromatograms of control (boiled recombinant protein: purple, UV; black, MS) and sample (purified recombinant COMT protein: 24 pmol, green; UV, red, MS) reaction mixtures with 5-hydroxyferulate (50 $\mu$M) as the substrate (Rt 6.03 min/UV). The internal standard (I.S.) was o-coumarate (Rt 19.00 min/UV) and the only product was identified as sinapate based on its identical retention time (9.45 min/UV) and UV spectral property to those of the authentic standard. Inset B, NI-ES mass spectrum (scanning mode at 70 V) of the reaction product (Rt 9.59 min/MS) is also identical to that of the authentic sinapate. C, HPLC-UV (322 nm) and HPLC-MS (SIM, 70 V; $m/z$ 193.1) chromatograms of control (boiled recombinant protein: purple, UV; black, MS) and sample (purified recombinant COMT protein: 24 pmol, green,

Methylation Reactions in Monolignol Biosynthesis

TABLE I  
Kinetic parameters of methylation of various phenolic substrates by purified recombinant aspen COMT protein  
Values were mean ± S.D. ($n = $ two independent assays).

| Substrate                  | $K_{m}$     | $V_{\text{max}}$    | $k_{\text{cat}}$   |
|----------------------------|--------------|----------------------|--------------------|
|                            | $\mu M$      | $\mu M \, min^{-1}$  | $min^{-1}$         |
| 5-Hydroxyconiferyl aldehyde | $2.6 \pm 0.3$ | $2.0 \pm 0.3$        | $24.9 \pm 3.5$     |
| 5-Hydroxyferulate          | $15.0 \pm 0.6$ | $2.2 \pm 0.3$        | $28.1 \pm 4.2$     |
| Caffeate                   | $75.1 \pm 3.3$ | $1.85 \pm 0.1$       | $23.1 \pm 1.1$     |

5-hydroxyferulate as an AldOMT substrate. These kinetic data clearly indicate that methylation of 5-hydroxyferulate *in vivo* is unlikely to take place in the presence of 5-hydroxyconiferyl aldehyde, whereas 5-hydroxyconiferyl aldehyde methylation would not be affected by 5-hydroxyferulate. This identifies for the first time a substrate-sensitive methylation mechanism involved in syringyl monolignol biosynthesis in that the presence of 5-hydroxyconiferyl aldehyde would eliminate the production of sinapate from 5-hydroxyferulate *in vivo*. Similarly, 5-hydroxyconiferyl aldehyde was a competitive inhibitor (Fig. 4A) of AldOMT-catalyzed methylation of caffeate with a $K_{i}$ of $2.1 \, \mu M$ (Fig. 4A, inset), whereas caffeate competitively inhibited methylation of 5-hydroxyconiferyl aldehyde (Fig. 4B) with a considerably higher $K_{i}$ of $121.7 \, \mu M$ (Fig. 4B, inset). These inhibition kinetic data together with the 30-fold higher $V_{\text{max}}/K_{m}$ value for 5-hydroxyconiferyl aldehyde than for caffeate (Table I) suggest that AldOMT-mediated methylation of 5-hydroxyconiferyl aldehyde is unlikely to be affected by caffeate, whereas caffeate methylation would be blocked in the presence of 5-hydroxyconiferyl aldehyde. However, it could be argued that the *in vivo* concentrations of these competing acid substrates may be presented at levels high enough to relieve their inhibited reactions. Using HPLC/MS, we then analyzed the quantities (micrograms/g fresh tissue) of these phenolic acids in methanol extracts from xylem cells. The concentration of caffeate was estimated to be about $0.003 \, \mu M$ and no ferulate could be detected, arguing strongly against that the *in vivo* concentrations of these acids would be sufficient to overcome their inhibited enzymatic reactions. This and the kinetic properties illustrated above and previously (26) further attest to the idea that, in the presence of coniferyl aldehyde derivatives, the acid pathway is not a viable option. In the absence of 5-hydroxyconiferyl aldehyde, however, Lineweaver-Burk analysis further indicated that the methylation of caffeate and 5-hydroxyferulate would not be effectively inhibited by 5-hydroxyferulate ($K_{i} = 58.4 \, \mu M$, Fig. 5A) and caffeate ($K_{i} = 75 \, \mu M$, Fig. 5B), respectively. These kinetic data, in light of our early finding that ferulate 5-hydroxylation can occur in the absence of coniferyl aldehyde, lead to the conclusion that ferulate 5-hydroxylation/methylation may take place when CAld5H/AldOMT-mediated coniferyl aldehyde 5-hydroxylation/methylation is absent. This suggests that CAld5H/AldOMT modulation may be a part of a regulatory mechanism associated with cell wall development involving both lignification and the biosynthesis of other phenolics which utilize ferulate 5-hydroxylation/methylation pathway in angiosperms. To test whether these inhibitory effects on recombinant protein are also true for native plant proteins, methyl-

UV; red, MS) reaction mixtures with caffeate ($50 \, \mu M$) as the substrate (Rt $6.68 \, \text{min}/\text{UV}$). The internal standard (I.S.) was $o$-coumarate (Rt $19.00 \, \text{min}/\text{UV}$) and the only product was identified as ferulate based on its identical retention time ($9.45 \, \text{min}/\text{UV}$) and UV spectral property to those of the authentic standard. Inset C, NI-ES mass spectrum (scanning mode at $70 \, V$) of the reaction product (Rt $10.95 \, \text{min}/\text{MS}$, $10.80 \, \text{min}/\text{UV}$) is also identical to that of the authentic ferulate. D, HPLC-MS (SIM, $70 \, V$; $m/z$ $193.1$, $179.1$, and $209.1$ ($0.00–8.00 \, \text{min}$); $m/z$ $193.1$, $207.1$, and $223.1$ ($8.00–15.00 \, \text{min}$) $m/z$ $163.1$ ($15.00–21.00 \, \text{min}$)) extracted ion chromatograms of control (boiled aspen xylem protein extracts: black) and sample (aspen xylem protein extracts, $15 \, \mu g$ of total protein) reaction mixtures with 5-hydroxyferulate ($50 \, \mu M$, brown, $m/z$ $209.1$, Rt $6.18 \, \text{min}/\text{MS}$), caffeate ($50 \, \mu M$, red, $m/z$ $179.1$, Rt $6.83 \, \text{min}/\text{MS}$), and 5-hydroxyconiferyl aldehyde ($50 \, \mu M$, green, $m/z$ $193.1$; Rt $6.98 \, \text{min}/\text{MS}$) as substrates. The internal standard (I.S.) was $o$-coumarate ($m/z$ $163.1$, Rt $19.15 \, \text{min}/\text{MS}$). The only reaction product detected was sinapyl aldehyde (blue, $m/z$ $207.1$, Rt $10.79 \, \text{min}/\text{MS}$) based on its identical retention time and UV and MS spectral properties to those of the authentic standard.

FIG. 3. Kinetic analysis of the inhibition of 5-hydroxyferulate and 5-hydroxyconiferyl aldehyde methylation by 5-hydroxyconiferyl aldehyde and 5-hydroxyferulate, respectively. Kinetic experiments are described under “Experimental and Procedures.” A, Lineweaver-Burk plot of purified recombinant aspen COMT-catalyzed methylation of 5-hydroxyferulate ($10$ to $800 \, \mu M$) in the presence of 5-hydroxyconiferyl aldehyde at different concentrations (as shown). Inset A, a plot of the apparent $K_{m}$ for the substrate 5-hydroxyferulate versus the corresponding inhibitor 5-hydroxyconiferyl aldehyde concentration at which it was obtained. Slope $= K_{m}/K_{i}$. B, Lineweaver-Burk plot of purified recombinant aspen COMT-catalyzed methylation of 5-hydroxyconiferyl aldehyde ($0.25$ to $50 \, \mu M$) in the presence of 5-hydroxyferulate at different concentrations (as shown). Inset B, a plot of the apparent $K_{m}$ for the substrate 5-hydroxyconiferyl aldehyde versus inhibitor (5-hydroxyferulate) concentration, as described in inset A.

ation reactions mediated by proteins from lignifying stem secondary xylem were studied.

**AldOMT Enzymatic Activities of Plant Xylem Proteins—Ten angiosperm tree species were randomly selected to test whether the 5-hydroxyconiferyl aldehyde-modulated AldOMT function observed for recombinant aspen AldOMT is common to angiosperm species. Soluble protein extracts isolated from secondary developing xylem of these plants were found to exhibit methylation activities with caffeate, 5-hydroxyferulate, and 5-hydroxyconiferyl aldehyde when these substrates were present individually (Table II). When these substrates were mixed

6542

**Methylation Reactions in Monolignol Biosynthesis**

![Graph A](#)
- **A**
  - $1/v$ (min/µM)
  - [5-hydroxyconiferyl aldehyde] (µM)
  - $K_i = 2.1 \, \mu M$
  - 4.0 µM
  - 2.0 µM
  - 1.0 µM
  - 0.75 µM
  - 5-hydroxyconiferyl aldehyde, 0 µM
  - $1/[Caffeate], (1/\mu M)$

![Graph B](#)
- **B**
  - $1/v$ (min/µM)
  - [Caffeate], (µM)
  - $K_i = 121.7 \, \mu M$
  - 200 µM
  - 100 µM
  - 50 µM
  - 10 µM
  - Caffeate, 0 µM
  - $1/[5\text{-hydroxyconiferyl aldehyde}], (1/\mu M)$

![Graph A Inset](#)
- **A**
  - $1/v$ (min/µM)
  - [5-hydroxyferulate] (µM)
  - $K_i = 58.4 \, \mu M$
  - 200 µM
  - 100 µM
  - 50 µM
  - 10 µM
  - 5-hydroxyferulate, 0 µM
  - $1/[Caffeate], (1/\mu M)$

![Graph B Inset](#)
- **B**
  - $1/v$ (min/µM)
  - [Caffeate] (µM)
  - $K_i = 75 \, \mu M$
  - 150 µM
  - 100 µM
  - 50 µM
  - 25 µM
  - Caffeate, 0 µM
  - $1/[5\text{-hydroxyferulate}], (1/\mu M)$

**FIG. 4.** Kinetic analysis of the inhibition of caffeate and 5-hydroxyconiferyl aldehyde methylation by 5-hydroxyconiferyl aldehyde and caffeate, respectively. Kinetic experiments are described under “Experimental Procedures.” A, Lineweaver-Burk plot of purified recombinant aspen COMT-catalyzed methylation of caffeate (10 to 800 µM) in the presence of 5-hydroxyconiferyl aldehyde at different concentrations (as shown). Inset A, a plot of the apparent $K_m$ for the substrate caffeate versus inhibitor (5-hydroxyconiferyl aldehyde) concentration, as described above. B, Lineweaver-Burk plot of purified recombinant aspen COMT-catalyzed methylation of 5-hydroxyconiferyl aldehyde (0.25–50 µM) in the presence of caffeate at different concentrations (as shown). Inset B, a plot of the apparent $K_m$ for the substrate 5-hydroxyconiferyl aldehyde versus inhibitor (caffeate) concentration, as described above.

and incubated with plant proteins, enzymatic methylation activities with caffeate and 5-hydroxyferulate were either abolished or drastically reduced, while methylation of 5-hydroxyconiferyl aldehyde became the dominant reaction (Table II). Thus, the biochemical characterization of recombinant as well as extracted plant proteins from various species provide unambiguous evidence that 5-hydroxyconiferyl aldehyde inhibits the enzymatic formation of ferulate and sinapate from caffeate and 5-hydroxyferulate, respectively, arguing against the involvement of ferulate and sinapate in syringyl monolignol biosynthesis in angiosperms. The irrelevance of ferulate to syringyl monolignol biosynthesis is in line with our previous recombinant CAld5H kinetic data showing that coniferyl aldehyde blocks ferulate 5-hydroxylation (26). The present evidence that sinapate is not involved in syringyl monolignol biosynthesis is consistent with the blocked biosynthesis of 5-hydroxyferulate (26) and with reports that proteins from actively lignifying tissues of various angiosperms do not utilize sinapate as a substrate for CoA ligation (36–38). To strengthen our present data it was necessary to test whether coniferyl aldehyde-induced inhibition of CAld5H activity with ferulate is also true for the plant CAld5H enzyme.

**FIG. 5.** Kinetic analysis of the inhibition of caffeate and 5-hydroxyferulate methylation by 5-hydroxyferulate and caffeate, respectively. Kinetic experiments are described under “Experimental Procedures.” A, Lineweaver-Burk plot of purified recombinant aspen COMT-catalyzed methylation of caffeate (10 to 800 µM) in the presence of 5-hydroxyferulate at different concentrations (as shown). Inset A, a plot of the apparent $K_m$ for the substrate caffeate versus inhibitor (5-hydroxyferulate) concentration, as described above. B, Lineweaver-Burk plot of purified recombinant aspen COMT-catalyzed methylation of 5-hydroxyferulate (10–800 µM) in the presence of caffeate at different concentrations (as shown). Inset B, a plot of the apparent $K_m$ for the substrate 5-hydroxyferulate versus inhibitor (caffeate) concentration, as described above.

**CAld5H Enzymatic Activities of Plant Xylem Proteins—Microsomal proteins isolated from secondary developing xylem tissue of the same 10 species used above were characterized for their 5-hydroxylation activities with ferulate and coniferyl aldehyde. Consistent with recombinant CAld5H function (26), xylem microsomal proteins from each plant exhibited a significantly higher rate of 5-hydroxylation for coniferyl aldehyde than for ferulate (Table III). However, in reactions of plant microsomal proteins with mixed substrates of ferulate and coniferyl aldehyde, enzymatic 5-hydroxylation of ferulate was completely abolished, whereas coniferyl aldehyde 5-hydroxylation activity was not compromised (Table III), in conformity with recombinant CAld5H function (26) and validating the idea that coniferyl aldehyde 5-hydroxylation/methylation is a general mechanism in angiosperms directing syringyl lignin biosynthesis.**

**DISCUSSION**

The present study demonstrates that competition among structurally similar pathway intermediates as substrates may modulate enzyme activities to guide the phenolic metabolism for monolignol formation. Specifically, here we show for the first time that substrate-modulated enzymatic methylation directs syringyl monolignol biosynthesis. 5-Hydroxyconiferyl al-

Methylation Reactions in Monolignol Biosynthesis

**TABLE II**

Effect of 5-hydroxyconiferyl aldehyde on the O-methyltransferase activity of developing xylem proteins from various angiosperm tree species

The following abbreviations are used: CA, caffeate; FA, ferulate; 5OHFA, 5-hydroxyferulate; SA, sinapate; 5OHCAld, 5-hydroxyconiferyl aldehyde; SAld, sinapyl aldehyde.

| Species                          | CA to FA       | 5OHFA to SA      | 5OHCAld to SAlld     | CA+5OHFA+5OHCAld to |
|----------------------------------|----------------|------------------|----------------------|---------------------|
|                                  | nmol/min/mg proteins |                  |                      |                     |
| Aspen (P. tremuloides)           | 12.5           | 15.6             | 12.8                 | FA                  |
| Acacia (A. mangium × auriculiformis) | 12.4           | 27.9             | 37.8                 | SA                  |
| Basswood (T. americana)          | 11.2           | 25.6             | 39.4                 | SAld                |
| Black ash (F. americana)         | 11.0           | 18.9             | 46.8                 |                     |
| Hophor beam (O. virginiana)      | 8.9            | 17.9             | 20.8                 |                     |
| Red maple (A. rubrum)            | 16.1           | 24.7             | 13.0                 |                     |
| Sugar maple (A. saccharum)       | 15.2           | 21.2             | 45.7                 |                     |
| Sweetgum (L. styraciflua)        | 7.7            | 12.7             | 16.2                 |                     |
| White birch (B. papyrifera)      | 9.6            | 19.4             | 7.6                  |                     |
| Yellow birch (B. alleghaniensis) | 11.9           | 25.0             | 20.8                 |                     |

**nmol/min/mg proteins**
| 2.2 | 0.1 | 13.7 |
| 0.7 | 0   | 19.0 |
| 0.6 | 0.7 | 23.2 |
| 0.8 | 0   | 15.3 |
| 0.6 | 0   | 27.9 |
| 0.4 | 0   | 15.1 |
| 0.5 | 0.2 | 17.7 |
| 2.2 | 0.3 | 19.2 |
| 0.4 | 0   | 11.1 |
| 0.5 | 0.2 | 22.1 |

**TABLE III**

Effect of coniferyl aldehyde on the 5-hydroxylase activity of developing xylem microsomal proteins from various angiosperm tree species

The following abbreviations are used: CALd, coniferyl aldehyd. Other abbreviations are the same as in Table II.

| Species                          | Substrate specificity | FA to 5OHFA | CALd to 5OHCAld | FA + CALd to 5OHFA 5OHCAld |
|----------------------------------|-----------------------|-------------|-----------------|---------------------------|
|                                  | pmol/min/mg proteins  |             |                 |                           |
| Aspen (P. tremuloides)           |                       | 15.4        | 948             | 970                       |
| Acacia (A. mangium × auriculiformis) |                       | 1.4         | 87              | 84                        |
| Basswood (T. americana)          |                       | 0.8         | 326             | 352                       |
| Black ash (F. americana)         |                       | 5.5         | 403             | 393                       |
| Hophor beam (O. virginiana)      |                       | 2.9         | 1014            | 970                       |
| Red maple (A. rubrum)            |                       | 12.2        | 342             | 386                       |
| Sugar maple (A. saccharum)       |                       | 0.6         | 232             | 274                       |
| Sweetgum (L. styraciflua)        |                       | 10.3        | 279             | 297                       |
| White birch (B. papyrifera)      |                       | 14.6        | 1261            | 1418                      |
| Yellow birch (B. alleghaniensis) |                       | 4.4         | 466             | 481                       |

(43), Zinnia (44), and Lithospermum erythrorhizon (45), although these activities may actually be polyphenol oxidases (45–47). Thus, the conversion of 4-coumarate 3 to caffeoyl-CoA 6 via 4-coumaroyl-CoA 4 represents one lignin precursor pathway. Alternatively, 4-coumarate 3 can be enzymatically 3-hydroxylated into caffeate 5 (48, 49), mediated by 4-coumarate 3-hydroxylase as shown by Kojima and Takeuchi (49), and in turn caffeate 5 can be activated by 4CL into caffeoyl-CoA 6 (38, 50, 51) for further metabolism into downstream intermediates, representing another possible biosynthetic pathway for monolignol biosynthesis. It needs, however, to be stressed that neither CCoA3H nor 4-coumarate 3-hydroxylase have been cloned and their activities have not been unambiguously demonstrated in actively lignifying tissues of any plant species (23). 4CL substrate specificity for 4-coumarate 3 and caffeate 5 also must be clarified in consideration of possible substrate-modulated enzyme activities, as reported here and in our previous studies (26, 52). Prior to the corroboration of definitive 3-hydroxylation and CoA ligation steps involved in lignin biosynthesis, the current available data seem to support the operation of either or both of these alternative pathways for monolignol biosynthesis.

Monolignol biosynthesis from caffeate 5 to ferulate 13 is unlikely in the presence of 5-hydroxyconiferyl aldehyde 9 (Fig. 2D and Table II), obviating also the CoA ligation of ferulate 13 to feruloyl-CoA 7 as a path to monolignols *in vivo*. Instead, feruloyl-CoA 7 can be formed from caffeoyl-CoA 6 by the action of caffeoyl-CoA 3-O-methyltransferase (CCoAOMT) (44, 53–55), as shown in Fig. 6. Although with an insignificant catalytic activity as compared with 3-methylation, CCoAOMT was also shown to mediate the *in vitro* 5-methylation of 5-hydroxyferuloyl-CoA 16 (33, 44), proposed to result from CoA ligation of 5-hydroxyferulate 14 (22, 36) or 5-hydroxylation of feruloyl-CoA 7 (44). However, CoA ligation of 5-hydroxyferulate 14 is not likely in view of our previous and current studies demonstrating that 5-hydroxyferulate 14 is not a pathway intermediate for monolignol biosynthesis (26). To date there is also no evidence to support a feruloyl-CoA 7 5-hydroxylation pathway. Indeed, our previous enzymatic characterization of aspen and sweetgum plant proteins did not result in detectable 5-hydroxylation activity with feruloyl-CoA 7 (26). Together these results refute a pathway from feruloyl-CoA 16 to sinapoyl-CoA 17 via 5-hydroxyferuloyl-CoA 16 for the biosynthesis of syringyl monolignol (Fig. 6). Furthermore, using recombinant CCoAOMT protein, we did not detect its activity with 5-hydroxyconiferyl alcohol 18, suggesting that OMTs in lignifying xylem tissue do not utilize

6544 Methylation Reactions in Monolignol Biosynthesis

![Diagram](attachment:diagram.png)

FIG. 6. Biosynthetic pathway from phenylalanine 1 to coniferyl alcohol 11 and sinapyl alcohol 12 for the formation of guaiacyl-syringyl lignin in angiosperms. $P A L$, phenylalanine ammonia-lyase; $C 4 H$, cinnamate 4-hydroxylase; $C 3 H$, 4-coumarate-3-hydroxylase; 4CL, 4-coumarate-CoA ligase; CCoAOMT, caffeoyl-CoA O-methyltransferase; CCR, cinnamoyl-CoA reductase; CAD, cinnamoyl alcohol dehydrogenase.

5-hydroxyconiferyl alcohol effectively and thus arguing against a pathway from coniferyl alcohol 11 to sinapyl alcohol 12 via 5-hydroxyconiferyl alcohol 18, as proposed by Fukushima et al. (56) and Chen et al. (57). However, the suggestion by Chen et al. (57) that oxidation of coniferyl alcohol 11 to coniferyl aldehyde 8 might lead to the biosynthesis of sinapyl alcohol 12 through sinapyl aldehyde 10 is consistent with our demonstration that coniferyl aldehyde 5-hydroxylation followed by AldOMT-mediated methylation direct syringyl monolignol biosynthesis in angiosperms.

Taken together, these results provide evidence that ring substitution-specific methylation directs monolignol biosynthesis in angiosperms. CCoAOMT mediates specifically the ring-3 methylation of guaiacyl intermediate caffeoyl-CoA 6 for the biosynthesis of common substrates for both guaiacyl and syringyl monolignols, and AldOMT, catalyzing distinctly the ring-5 methylation of 5-hydroxyconiferyl aldehyde 9, is specific for syringyl monolignol biosynthesis. This is in line with the finding that suppression of CCoAOMT activity in transgenic tobacco reduced the biosynthesis of both guaiacyl and syringyl moieties, and that down-regulation of AldOMT (or previously COMT) preferentially attenuated syringyl monolignol biosynthesis (58). The specific function of CCoAOMT in catalyzing ring-3 methylation of guaiacyl derivative caffeoyl-CoA 6 is also consistent with its role as the major lignin pathway methylating enzyme in loblolly pine (33), a gymnosperm containing primary guaiacyl lignin. The commonality of a guaiacyl moiety-specific methylation function for CCoAOMTs in both angiosperm and gymnosperm species is in agreement with the highly conserved amino acid sequences of CCoAOMTs in these species (35). The amino acid sequences of AldOMTs (or previously COMT), however, differ significantly from those of CCoAOMTs (35), suggesting that AldOMT's specific function in the biosynthesis of syringyl monolignol was evolved after the divergence of angiosperms from gymnosperms, and that CCoAOMTs function remained in angiosperms after the divergence. Thus, in angiosperms, two distinct hydroxylation/methylation pathways regulate the biosynthesis of guaiacyl and syringyl mono-

lignols. CCoA3H (or 4-coumarate 3-hydroxylase)/CCoAOMT, in conjunction with cinnamate 4-hydroxylase, 4CL, cinnamoyl-CoA reductase, and cinnamoyl alcohol dehydrogenase, are associated with the biosynthesis of guaiacyl monolignol 11, whereas CAld5H/AldOMT divert the guaiacyl pathway at coniferyl aldehyde 8 into the biosynthesis of syringyl monolignol 12 (Fig. 6).

Our results demonstrate conclusively that lignin is not the sink for ferulate 13, 5-hydroxyferulate 14, and sinapate 15. Instead, these hydroxycinnamates are the substrates of glucosyltransferases for the formation of their glucosyl esters, and become the acyl donors for esterification to the cell walls, where they can contribute to cross-linkages that increase cell wall strength (59, 60). When lignin biosynthesis is interrupted, the diminished levels of coniferyl and 5-hydroxyconiferyl aldehydes may, for example, be a part of a signaling mechanism triggering a diversion of lignin precursors into hydroxycinnamates for cell wall cross-linkages during a defense response. This is consistent with our recent discovery that lignin reduction in transgenic aspen trees did not alter the stem cell wall morphology but was accompanied by a drastic increase in wall-bound hydroxycinnamate esters (8). For normal cell wall development, CAld5H/AldOMT-mediated coniferyl aldehyde 5-hydroxylation and methylation divert precursor flow away from the ferulate pathway (Fig. 6) to streamline the lignification process. Taken together, our results provide clarification to previous ambiguous perceptions concerning certain aspects of monolignol biosynthesis in angiosperms and a biological insight into a defense response involving a CAld5H/AldOMT-modulated deposition of both lignin and other cell wall cross-linkages that may be crucial to growth and development in angiosperm trees.

Acknowledgments—We are grateful to Drs. Scott Harding and Chung-Jui Tsai for critically reading the manuscript. We thank Priit Pechter for collecting the xylem tissue.

**REFERENCES**

1. Sarkanen, K. V. (1971) in *Lignins: Occurrence, Formation, Structure and Reaction* (Sarkanen, K. V., and Ludwig, C. H., eds) pp. 639–694, Wiley-Interscience, New York
2. Chang, H. M., and Sarkanen, K. V. (1973) *Tech. Assoc. Pulp Pap. Ind.* J. **56**, 132–136
3. Chiang, V. L., and Funaoka, M. (1990) *Holzforschung* **44**, 309–313
4. Freudenberg, K. (1965) *Science* **148**, 595–600
5. Trotter, P. C. (1990) *Tech. Assoc. Pulp Pap. Ind.* J. **73**, 198–204
6. Bugos, R. C., Chiang, V. L., and Campbell, W. H (1991) *Plant Mol. Biol.* **17**, 203–1215
7. Boudet, A. M., Lapierre, C., and Grima-Pettenati, J. (1995) *New Phytol.* **129**, 203–226
8. Hu, W.-J., Harding, S. A., Lung, J., Popko, J. L., Ralph, J., Stokke, D. D., Tsai, C.-J., and Chiang, V. L. (1999) *Nature Biotech.* **17**, 808–812
9. Byerrum, R. U., Flokstra, J. H., Dewey, L. J., and Ball, C. D. (1954) *J. Biol. Chem.* **210**, 633–643
10. Brown, S. A., and Neish, A. C. (1955) *Can. J. Biochem. Physiol.* **33**, 948–956
11. Brown, S. A., and Neish, A. C. (1956) *Can. J. Biochem. Physiol.* **34**, 769–775
12. Higuchi, T., and Brown, S. A. (1963) *Can. J. Biochem. Physiol.* **41**, 65–76
13. Higuchi, T., and Brown, S. A. (1963) *Can. J. Biochem. Physiol.* **41**, 613–620
14. Higuchi, T., and Brown, S. A. (1963) *Can. J. Biochem. Physiol.* **41**, 621–627
15. Higuchi, T., Shimada, M., and Ohashi, H. (1967) *Agri. Biol. Chem.* **31**, 1459–1464
16. Shimada, M., and Higuchi, T. (1972) *Phytochemistry* **11**, 2247–2252
17. Shimada, M., Fushiki, H., and Higuchi, T. (1973) *Mokuzai Gakkuishi* **19**, 13–21

18. Kuroda, H., Shimada, M., and Higuchi, T. (1975) *Phytochemistry* **14**, 1759–1763
19. Kuroda, H., Shimada, M., and Higuchi, T. (1981) *Phytochemistry* **20**, 2635–2639
20. Sarkanen, K. V. (1971) in *Lignins: Occurrence, Formation, Structure and Reaction* (Sarkanen, K. V., and Ludwig, C. H., eds) pp. 95–155, Wiley-Interscience, New York
21. Grisebach, H. (1981) in *The Biochemistry of Plants* (Conn, E. E., ed) pp. 457–478, Academic Press, New York
22. Higuchi, T. (1985) in *Biosynthesis and Biodegradation of Wood Components* (Higuchi, T., ed) pp. 141–160, Academic Press, New York
23. Higuchi, T. (1997) in *Biochemistry and Molecular Biology of Wood* (Timell, T. E., ed) pp. 131–233, Springer, New York
24. Whetten, R. W., MacKay, J. J., and Sederoff, R. R. (1998) *Annu. Rev. Plant Physiol. Mol. Biol.* **49**, 585–609
25. Grand, C. (1984) *FEBS Lett.* **169**, 7–11
26. Osakabe, K., Tsao, C. C., Li, L., Popko, J. L., Umezawa, T., Carraway, D. T., Smeltzer, R. H., Joshi, C. P., and Chiang, V. L. (1999) *Proc. Natl. Acad. Sci. U. S. A.* **96**, 8955–8960
27. Tsai, C. J., Popko, J. L., Mielke, M. R., Hu, W. J., Podila, G. K., and Chiang, V. L. (1998) *Plant Physiol.* **117**, 101–112
28. Bugos, R. C., Chiang, V. L., and Campbell, W. H. (1992) *Phytochemistry* **31**, 1495–1498
29. Stockigt, J., and Zenk, M. H. (1975) *Z. Naturforsch.* **30**, 352–358
30. Banerjee, S. K., Manolopoulos, M., and Pepper, J. M. (1962) *Can. J. Chem.* **40**, 2175–2177
31. Struble, R. E. (1963) *Org. Syntheses Coll.* **4**, 417–419
32. Li, L., Popko, J. L., Zhang, X.-H., Osakabe, K., Tsai, C. J., Joshi, C. P., and Chiang, V. L. (1997) *Proc. Natl. Acad. Sci. U. S. A.* **94**, 5461–5466
33. Li, L., Osakabe, Y., Joshi, C. P., and Chiang, V. L. (1999) *Plant Mol. Biol.* **40**, 555–565
34. Meng, H., and Campbell, W. H. (1996) *Arch. Biochem. Biophys.* **330**, 329–341
35. Josh, C. P., and Chiang, V. L. (1998) *Plant Mol. Biol.* **37**, 663–674
36. Kutsuki, H., Shimada, M., and Higuchi, T. (1982) *Phytochem.* **21**, 267–271
37. Gross, G. G., Mansell, R. L., and Zenk, M. H. (1975) *Biochem. Physiol. Pflanz.* **168**, 41–51
38. Hu, W. J., Kawaoka, A., Tsai, C. J., Lung, J., Osakabe, K., Ebinuma, H., and Chiang, V. L. (1998) *Proc. Natl. Acad. Sci. U. S. A.* **95**, 5407–5412
39. Potts, J. R., Weklych, R., and Conn, E. E. (1974) *J. Biol. Chem.* **249**, 5019–5026
40. Urban, P., Werck-Reichhart, D., Teutsch, H., Durst, F., Regnier, S., Kazmaier, M., and Pompon, D. (1994) *Eur. J. Biochem.* **222**, 843–850
41. Durst, N., and O'Keefe, D. P. (1995) in *Plant Cytochromes P-450: An Overview in Drug Metabolism and Drug Interactions* (Durst, F., and O'Keefe, D. P., eds) pp. 171–183, Freund Publishing House, London
42. Kamsteeg, J., van Brederode, J., Verschuren, P. M., and van Nigtevecht, G. (1981) *Z. Pflanzenphysiol.* **102**, 435–442
43. Kneusel, R. E., Matern, U., and Nicolary, K. (1989) *Arch. Biochem. Biophys.* **269**, 455–462
44. Ye, Z-H., Kneusel, R. E., Matern, U., and Varner, J. E. (1994) *Plant Cell* **6**, 1427–1439
45. Wang, Z. X., Li, S. M., Loscher, R., and Heide, L. (1997) *Arch. Biochem. Biophys.* **347**, 249–255
46. Tanaka, M., and Kojima, M. (1991) *Arch. Biochem. Biophys.* **284**, 151–157
47. Matern, U., Grimmig, B., and Kneusel, R. E. (1995) *Can. J. Bot.* **73**, Suppl. 1, S511–S517
48. Vaughan, P. F. T., Eason, R., Paton, J. Y., and Ritchie, G. A. (1975) *Phytochemistry* **14**, 2383–2386
49. Kojima, M., and Takeuchi, W. (1989) *J. Biochem. (Tokyo)* **105**, 265–270
50. Lozoya, E., Hoffmann, H., Douglas, C., Schulz, W., Scheel, D., and Hahlbrock, K. (1988) *Eur. J. Biochem.* **176**, 661–667
51. Lee, D., and Douglas, C. J. (1996) *Plant Physiol.* **112**, 193–205
52. Zhang, X.-H., and Chiang, V. L. (1997) *Plant Physiol.* **113**, 65–74
53. Kühnl, T., Koch, U., Heller, W., and Wellmann, E. (1989) *Plant Sci.* **60**, 21–25
54. Pakusch, A. E., Kneusel, R. E., and Matern, U. (1989) *Arch. Biochem. Biophys.* **271**, 488–494
55. Ye, Z.-H., and Varner, J. E. (1995) *Plant Physiol.* **108**, 459–467
56. Fukushima, K., Matsui, N., Taguchi, S., and Yasuda, S. (1997) *Proceedings of the 9th International Symposium on Wood and Pulp Chemistry, Montreal, June 9–12*, pp. 28–34, Tech. Assoc. Pulp Pap. Ind. J., Atlanta
57. Chen, F., Yasuda, S., and Fukushima, K. (1999) *Planta* **207**, 597–603
58. Zong, R., Morrison, W. H., Negrel, J., and Ye, Z-H. (1998) *Plant Cell* **10**, 2033–2045
59. Mock, H. P., and Strack, D. (1993) *Phytochemistry* **32**, 575–579
60. Tan, K. S., Hoson, T., Masuda, Y., and Kamisaka, S. (1992) *J. Plant Physiol.* **140**, 460–465
