
Epstein–Barr Virus Latency: LMP2,  
A Regulator or Means for Epstein–  
Barr Virus Persistence?

Richard Longnecker  
Northwestern University Medical School, Chicago, IL 60611  

I. Introduction  
II. EBV Latency  
III. Latent Membrane Protein 2  
    A. The LMP2 Genes  
    B. Tyrosine Motifs  
    C. Serine, Threonine, and Proline Motifs  
IV. B-Cell Signal Transduction and LMP2A  
V. LMP2A Site-Specific Mutant LCLs  
    A. Tyrosine Point Mutant LCLs: Y112  
    B. Tyrosine Point Mutant LCLs: Y74 and Y85  
    C. Deletion Mutant and Tyrosine Point Mutant LCLs: Y60, Y64, and Y101  
VI. Model of LMP2A and LMP2B Function *in Vitro*  
VII. *In Vivo* Models of LMP2A Function  
VIII. LMP2 Function in Epithelial Cells  
IX. Discussion  
References  

Like other herpesviruses, Epstein–Barr virus (EBV) persists in its host through an ability to establish a latent infection that periodically reactivates, producing infectious virus that infects naïve hosts. Disease syndromes in humans caused by EBV reflect the cell types that EBV infects, being primarily of lymphoid or epithelial origin. The most notable lymphoid disease, infectious mononucleosis, is a self-limiting lymphoproliferative disease that occurs in normal adolescents on primary infection. Children are normally able to resolve primary EBV infection with few or no symptoms. By the age of 25 most individuals are EBV seropositive. EBV is associated with a variety of hematopoietic cancers such as African Burkitt’s lymphoma, Hodgkin’s, and adult T-cell leukemia. EBV-associated lymphoproliferative disease occurs in individuals with congenital or acquired cellular immune deficiencies. The two notable epithelial diseases associated with EBV infection are nasopharyngeal cancer, a malignancy endemic to southern China, and oral hairy leukoplakia, an epithelial hyperplasia of the lingual squamous epithelium in AIDS patients. Latent membrane protein 2 (LMP2) is expressed both in normal EBV latency and EBV-associated pathologies. LMP2 may regulate reactivation from latency by interfering with normal B-cell signal transduction processes and in doing so may also provide a survival signal that could be important for viral persistence. Current knowledge about the function of LMP2 is described, defining a new class of regulators of herpesvirus latency. © 2000 Academic Press.

Advances in CANCER RESEARCH  
0065-230X/00 $35.00  
Copyright © 2000 by Academic Press.  
All rights of reproduction in any form reserved

I. INTRODUCTION

Epstein-Barr virus (EBV) is one of eight human herpesviruses that routinely establishes latent infections in human hosts following initial infection (Roizman, 1993). The eight human herpesviruses are organized into three families (α, β, γ) depending on biological characteristics and evolutionary relatedness. The three human α-herpesviruses, herpes simplex virus 1 (HSV-1), HSV-2, and varicella zoster virus (VZV), are characterized by their rapid reproductive cycle and capacity to establish latent infections in sensory ganglia. The three human β-herpesviruses, human cytomegalovirus (HCMV), human herpesvirus 6 (HHV6), and HHV7, typically have a longer lytic reproductive cycle and may have the ability to remain latent in more than one cell type. The two human γ-herpesviruses, EBV and KSHV (or HHV8), are distinguished by their latent infection of transformed lymphocyte cell lines in culture and their link with human proliferative disorders (Chang et al., 1994; Longnecker, 1998; Rickinson and Kieff, 1996).

Considerable interest has focused on EBV since its discovery and its link with Burkitt’s lymphoma in the early 1960s (D. Burkitt, 1962; D. P. Burkitt, 1983; Epstein et al., 1964). Along with HHV8, EBV is the only herpesvirus with a known role in human malignancies. EBV infection is linked with endemic Burkitt’s lymphoma and nasopharyngeal carcinoma (NPC) (Rickinson and Kieff, 1996). The virus has also been recognized as an important pathogen in individuals lacking cellular immunity either from genetic defects, immune suppression for organ transplantation, or loss of immune function due to HIV infection (Liebowitz, 1998; Longnecker, 1998; Rickinson and Kieff, 1996). In immunosuppressed patients, EBV causes a variety of proliferative disorders including oral hairy leukoplakia in AIDS patients, immunoblastic lymphomas, and an unusual tumor of muscle origin in children with AIDS or those who are under immune suppression after liver transplantation (Liebowitz, 1998; Longnecker, 1998; Rickinson and Kieff, 1996). EBV may also be a factor in a variety of other human malignancies including some T-cell lymphomas, gastric carcinomas, and Hodgkin’s disease (Liebowitz, 1998; Longnecker, 1998; Rickinson and Kieff, 1996).

Infection with EBV usually occurs early in childhood, resulting in an asymptomatic infection. If primary infection occurs later in life, proliferation of B lymphocytes infected with EBV and the resulting immune response can result in infectious mononucleosis. After primary infection, most individuals will harbor the virus for the remainder of their life, and carriers develop cellular immunity against a variety of both lytic and latency associated proteins (Liebowitz, 1998; Longnecker, 1998; Rickinson and Kieff, 1996). Periodically, virus is shed from latently infected individuals by the induction of lytic replication in B lymphocytes. The true site of latent infection has not been determined, but the virus likely resides in B lymphocytes. Recent studies have

shown that EBV can be detected in circulating peripheral blood lymphocytes in carriers of EBV latent infections by PCR both for viral DNA and viral mRNA (Chen et al., 1995; Decker et al., 1996; Khan et al., 1996; Miyashita et al., 1995; Qu and Rowe, 1992; Tierney et al., 1994).

The virus can also be isolated by culturing peripheral lymphocytes (Yao et al., 1985). It has not been determined if this is the true site of latency. Other potential sites of EBV latency include bone marrow, lymph nodes, or other lymphoid organs. Further evidence of the hematopoietic site of EBV latency comes from engraftment of bone marrow cells, which can result in the loss of the resident virus or the appearance of a new virus strain from donor lymphocytes (Gratama et al., 1988, 1989). Lytic replication is presumed to occur when EBV-infected B lymphocytes traffic through oral epithelium. This resulting infectious virus provides a source for infection of other individuals. There is considerable disagreement about whether lytic replication occurs in epithelial cells and if this replication is important for the transmission. Recent studies using samples from patients with acute infectious mononucleosis indicate no detectable lytic replication in oral epithelial cells despite abundant lytic replication in lymphocytes which have trafficked to the epithelium (Karajannis et al., 1997). Studies from AIDS patients with the unusual epithelial hyperplasia of the tongue indicate that the virus can gain access to epithelial cells and undergo lytic replication (Liebowitz, 1998; Longnecker, 1998; Rickinson and Kieff, 1996). Whether this is a pathological consequence from the underlying immune suppression found in AIDS patients needs to be resolved. EBV is also found in NPC, an epithelial malignancy found predominantly in Chinese populations (Liebowitz, 1998; Rickinson and Kieff, 1996).

## II. EBV LATENCY

B lymphocytes are readily infected with EBV *in vitro* and provide a tissue culture model of EBV growth transformation and latency. Latently infected B lymphocytes grown *in vitro* are called lymphoblastoid cell lines (LCLs) and express nine proteins and two small RNAs. The proteins are expressed either in the nucleus (EBNA, Epstein-Barr nuclear antigen) or plasma membrane (LMP, latent membrane protein). The EBNA proteins are responsible for maintaining the viral episome and regulating viral gene expression (Kieff, 1996; Longnecker, 1998). LMP1, a tumor necrosis factor family receptor mimic, is an inducer of cell proliferation (Kieff, 1996; Longnecker, 1998). The LMP2 genes are the topic of this article.

The genes expressed in EBV immortalized B-cell cells grown in tissue culture are not strictly conserved in any other human herpesvirus, but in the hu-
man γ-herpesvirus KSHV there are hints of functional homologs. In the case of LMP2, two proteins have been identified that share structural as well as sequence homology with the EBV LMP2 proteins. The KSHV K1 contains a functional immunoreceptor activation motif (ITAM), but it appears to be expressed only in lytic infection and has a single transmembrane domain (Lagunoff et al., 1999; Lee et al., 1998; Russo et al., 1996). Also in KSHV, K15 or LAMP is a gene composed of multiply spliced mRNAs expressed during latent infection. Predicted sequence from the group of transcripts indicates a maximum of 12 transmembrane domains with predicted products of fewer transmembrane domains as well (Glenn et al., 1999; Poole et al., 1999). The predicted proteins have a short amino terminus with a larger carboxyl terminus containing motifs reminiscent of both LMP1 and LMP2A of EBV (Glenn et al., 1999; Poole et al., 1999). In addition, homologs to many of the EBV latent genes expressed during in vitro latent infection have been identified in the EBV-related herpesviruses that infect monkeys (Franken et al., 1995, 1996; Ling et al., 1993; Rivailler et al., 1999; Yates et al., 1996).

Of the nine viral proteins expressed in latent infection, only EBNA1, LMP1, and LMP2A are consistently detected in NPC tumor biopsies, and other EBV-related proliferative disorders and only LMP2A and EBNA1 have been detected by polymerase chain reaction (PCR) analysis from B lymphocytes purified from healthy individuals harboring EBV latent infections (Chen et al., 1995; Miyashita et al., 1995; Qu and Rowe, 1992; Tierney et al., 1994). The detection of EBNA1 is not surprising since it is known to be required for EBV genome persistence in infected cells (Yates et al., 1985). Because of the uniform detection of LMP2A messages in vivo, it is likely to be important for viral replication, persistence, and EBV-related diseases in the human host. Because the full complement of viral genes expressed in vitro is not found in normal EBV latent infection in human hosts, the use of in vitro tissue culture models of EBV latent infection should be regarded with caution. This review concentrates on our analysis of LMP2A and LMP2B function in our models of in vitro and in vivo latency.

### III. LATENT MEMBRANE PROTEIN 2

#### A. The LMP2 Genes

The two LMP2 genes (LMP2A and LMP2B) are transcribed into two mRNAs across the circularized viral genome from two spatially distinct promoter elements (Laux et al., 1988, 1989; Sample et al., 1989). The LMP2A start site is approximately 5 kb from the bidirectional promoter responsible for the expression of LMP1 and LMP2B. The two transcripts are mul-
tiply spliced, and the resultant mRNAs differ only in their first exons. The LMP2A first exon contains an initiator of translation that results in a 119-hydrophilic-amino-acid amino terminus. The LMP2B first exon lacks a methionine codon, and translation of LMP2B initiates at a methionine near at the beginning of the second exon, which is common to both mRNAs. This latter methionine codon immediately precedes the first of 12 transmembrane domains, each of which traverses the B-cell plasma membrane, and a 27-amino-acid cytoplasmic carboxyl-terminal tail both of which are common to LMP2A and LMP2B (Fig. 1).

LMP2A and LMP2B localize to numerous small patches in the plasma membrane of latently infected B lymphocytes (Longnecker *et al.*, 1991; Longnecker and Kieff, 1990). LMP2A patches colocalize with LMP1 (Longnecker *et al.*, 1991; Longnecker and Kieff, 1990) and LMP2A associates with LMP1 in immunoprecipitates (Longnecker *et al.*, 1991). However, LMP2A and LMP1 each independently have the ability to aggregate in patches in plasma membrane of B lymphoblasts (Liebowitz *et al.*, 1986; Longnecker *et al.*, 1991). Truncation of LMP2A after the fifth or seventh transmembrane domain results in diffuse plasma membrane localization, confirming the importance of the multiple membrane domains in the fully aggregated phenotype (Liebowitz *et al.*, 1986; Longnecker *et al.*, 1991, 1993a). Most of the phosphotyrosine reactivity in latently infected B lymphocytes is associated

![Diagram](attachment:diagram.png)

Fig. 1 Predicted structure of LMP2A and LMP2B. The 12 transmembrane domains of LMP2A and LMP2B are shown traversing the plasma membrane. The large amino terminal and small cytoplasmic domain, both found within the cytoplasm, are also indicated. The SH2–phosphotyrosine interactions of LMP2A with the Lyn and Syk PTK at specific LMP2A tyrosine residues are indicated. An arrow indicates the LMP2B initiator methionine.
with LMP2A patches (Longnecker *et al.*, 1991). LMP2A is tyrosine phosphorylated in transfected B lymphoma cells and in EBV-transformed B lymphocytes (Burkhardt *et al.*, 1992; Longnecker *et al.*, 1991). Both LMP2A and LMP2B have been shown to be dispensable for EBV-mediated transformation of primary B lymphocytes (Kim and Yates, 1993; Longnecker *et al.*, 1992, 1993a,b; Speck *et al.*, 1999).

## B. Tyrosine Motifs

The LMP2A 119-amino-acid amino-terminal domain includes eight tyrosine residues, some of which are phosphorylated (Longnecker *et al.*, 1991). Each of these phosphorylated tyrosine residues provides a potential site for binding with cellular proteins containing Src homology 2 (SH2) domains (Fig. 1 and Table I). SH2 domains are noncatalytic domains conserved among cytoplasmic signaling proteins that bind tyrosine phosphorylated proteins (Pawson and Scott, 1997). Comparison of the LMP2A tyrosine motifs with tyrosine motifs known or predicted to bind SH2 domains contained within specific cellular proteins reveals homology to many cellular proteins involved in signal transduction (Table I) (Songyang *et al.*, 1993, 1994). Different SH2 domains have differing preferences for hydrophobic or hydrophilic residues immediately carboxyl prime to the tyrosine residue (Songyang *et al.*, 1993, 1994). Comparison of the eight tyrosine residues and their surrounding motifs reveals homology to several motifs that predict optimal binding to identified proteins involved in signal transduction (Table I). The Y31 motif (YPSA) has homology with group III members defined by Y-hydrophobic-X-hydrophobic (Songyang *et al.*, 1993, 1994). This phosphotyrosine motif optimally binds the SH2 domain of proteins such as the 85-kDa subunit adapter protein of PI3 kinase, PLCγ2, and Shc. Y60 (YEDP) contains the general motif Y-hydrophilic-hydrophilic-I/P or the group IB optimal motif. Of the group IB proteins, Y60 has the best homology with Crk (YDHP) and Nck (YDEP) motifs. LMP2A tyrosine residues, Y74 and Y85, form an immunoreceptor tyrosine-based activation motif (ITAM) (Cambier, 1995; Reth, 1989). ITAMs consist of paired tyrosine and leucine residues and have been shown to play a central role in signal transduction of the B-cell receptor (BCR) and the T-cell receptor (TCR) (Benschop and Cambier, 1999; Cambier *et al.*, 1994; Weiss and Littman, 1994). Interesting, Y85 bears some resemblance to an immunoreceptor tyrosine-based inhibitory motif (ITIM). It has been demonstrated that the phosphatase SHP-1 binds to the ITIM within the cytoplasmic domain of the Fc receptor. This interaction is important in the negative regulation of the BCR (Benschop and Cambier, 1999; Cambier *et al.*, 1994; Weiss and Littman, 1994). The consensus ITIM binding motif for SHP-1 is T/S-X-X-Y-X-X-L, which is very similar to
Table I. LMP2A Tyrosine and Serine Motifs

| LMP2A tyrosine | Tyrosine motif | SH2 domain site homology$^a$ | Conservation of tyrosine motif |
|-----------------|----------------|-------------------------------|----------------------------------|
| Y23             | YDGG           | None?                        | No                               |
| Y31             | YPSA           | Shc, PI3-K, PLCγ2            | Yes                              |
| Y60             | YEDP           | Abl, Crk, Nck                | Yes                              |
| Y64             | YWGN           | None?                        | No$^d$                           |
| Y74             | YQPL           | Syk                          | Yes                              |
| Y85             | YLGL           | Syk                          | Yes                              |
| Y101            | YSPR           | Csk                          | Yes                              |
| Y112            | YEEA           | Lyn, Src family              | Yes                              |

| EBV clinical$^b$ | $h$. papio$^c$ |
|-------------------|----------------|
| —                 | YPPS           |
|                   | YDAP           |
|                   | YGGS           |
|                   | YATL           |
|                   | YAGL           |
|                   | YSPR           |
|                   | YEEP           |

Conservation of serine motif

| LMP2A serine | Serine motif   | Ser/Thr kinase site homology$^e$ | EBV clinical$^b$ | $h$. papio$^c$ |
|--------------|----------------|-----------------------------------|-------------------|----------------|
| S15          | AGPPSPGGD      | MAPK                            | Yes               | GGPRSHGGP      |
| S52          | EERESNEEP      | Casein kinase II                | Yes               | —              |
| S102         | PPPYSPRDD      | MAPK, PKC                       | Yes               | PPPYSPRQG      |

$a$ Phosphopeptide sequences (one-letter code) within the LMP2A amino-terminal domain that are predicted to bind specific SH2 domains or are recognized by specific serine-threonine kinases. Proteins with an SH2 domain that are predicted to bind to LMP2A from Songyang et al. (1993, 1994).

$b$ LMP2A sequences from EBV clinical samples were previously reported (Berger et al., 1999; Busson et al., 1995).

$c$ LMP2A sequences from $H$. papio were previously reported (Franken et al., 1995).

$d$ Only 43% (Busson et al., 1995) or 17% (Berger et al., 1999) of the LMP2A clinical sequences contained Y64.

$e$ Recognition motifs for specific serine-threonine kinases as reported by Songyang et al. (1996).

of the isolates whereas Y64 was contained in only 43% of the sequenced isolates. A more recent study, which analyzed the LMP2A sequence in patients with EBV-associated reactive and malignant lymphoproliferations as well as normal healthy EBV carriers, found that Y74 and Y85 was strictly conserved in 76 cases (Berger et al., 1999). In 54 cases, Y60 and Y64 were analyzed. Y64 was changed to asparagine in 45 cases, whereas Y60 was strictly conserved. There was no correlation with the changes in the LMP2A amino acid sequence with any disease state. Seven of the eight LMP2A tyrosine residues are homologous and conserved between LMP2A and the amino acid sequence of LMP2A homologs in nonhuman γ-herpesviruses that infect primates (Fig. 2 and Table I) (Franken et al., 1995). Only tyrosine 23 is not found in the nonhuman LMP2A-like protein sequence (Fig. 2 and Table I).


     ①
1          ⑮         23        31
EBV MGSLEMVPMGAGPPSPGGDPDGYDGGNNSQYPSA
HVP MVSLEMQPLRVPGGPRSHGGPDGDEEDSNPYPPS

     ②               ③
                     60
SGSSGNTPTPPNDEERESNEEPPPPYEDP-----
: : : : : : . . . . : : : :
SFGES---------WDRD-GDPVPPDYDAPSHRPP

64       74       85
-YWNGNGDRHSDYQPLGTQDQSLYLGLQHDGNDGL
: : : : : : . . : : : . : : : : :
SYGGSGG-----YATLGQQEPSLYAGLGQNDDGGG

     ④      ⑤
           102
101       112       120
PPPPYSRDDSSQHIYEEAGRGS
: : : : : : : : : : : : : :
PPPPYSPRQ-GSEHVYEEPRDARM
                                 114


Fig. 2 Sequence conservation of the LMP2A genes from EBV and *H. papio*. EBV serines (circle) or tyrosines (box) are indicated. Proline pairs, which may bind SH3 domains, are indicated by brackets. Y60 and Y101 may form PY motifs and be sites of binding with proteins containing WW domains. Sequence data are adapted from Franken et al. (1995).

Epstein-Barr Virus Latency

C. Serine, Threonine, and Proline Motifs

In addition to the LMP2A phosphotyrosine interactions with cellular proteins, the amino-terminal domain of LMP2A also includes multiple serine and threonine residues, as well as proline-rich regions, many of which are conserved in related γ-herpesviruses (Fig. 2). LMP2A is serine and threonine phosphorylated (Longnecker *et al.*, 1991), so at least a subset of serine residues and threonine residues in the LMP2A amino-terminal domain provides potential sites recognized by serine and threonine kinases. Two LMP2A serine residues, S15 and S102, are sites of *in vitro* mitogen-activated protein kinase (MAPK) phosphorylation of a purified LMP2A fusion protein (Panousis and Rowe, 1997). MAPK bound both the LMP2A fusion protein and LMP2A in wild-type LCLs, presumably at either or both of these serine residues (Panousis and Rowe, 1997). The functional significance of the MAPK interaction with LMP2A is not known.

LMP2A also contains proline-rich regions that are conserved in both the LMP2A sequence from the different EBV clinical isolates and *Herpesvirus papio* (Fig. 2). These proline-rich regions may recruit Src homology 3 (SH3) domain-containing proteins (Pawson and Gish, 1992) or proteins with WW domains (Sudol, 1996). SH3 domains are noncatalytic conserved domains that are found in a number of cellular kinases and adapter proteins that recognize proline-rich regions of proteins and are essential in many signal transduction pathways linking cellular receptors to downstream signaling events. WW domains consist of paired tryptophan residues separated by 20–22 amino acids and have been shown to specifically bind to the motif XPPXY (Sudol, 1996), of which LMP2A has two (Fig. 2) and recently been shown to bind to ubiquitin ligases (Ikeda *et al.*, 2000). WW domains are contained in a diverse group of proteins also implicated in the regulation of cellular growth.

IV. B-CELL SIGNAL TRANSDUCTION AND LMP2A

In primary B lymphocytes and EBV- B-cell lines, activation of the BCR leads to an intricate signal cascade. Initially, there is recruitment and activation of the Src family protein tyrosine kinases (PTKs) followed by the recruitment and activation of the Syk PTK. The B-cell linker protein (BLNK or SLP-65) then interfaces the Syk PTK with PLCγ, the Vav guanine nucleotide exchange factor, and the Grb2 and Nck adapter proteins (Fu *et al.*, 1998; Ishiai *et al.*, 1999; Wienands *et al.*, 1998). These initial interactions lead to the mobilization of calcium and activation of the Ras pathway and protein kinase C (PKC), ultimately leading to the expression of BCR signal-specific genes (Fig. 3) (Benschop and Cambier, 1999; Cambier *et al.*, 1994; Weiss and Littman, 1994).

Fig. 3 B-cell signal transduction. Ligation of the BCR results in the activation of the Lyn and Syk PTKs, resulting in the tyrosine phosphorylation, recruitment, and activation of various cell proteins important for BCR-mediated signal transduction. These initial interactions and activation result in the activation of PKC, the Ras pathway, and the release of intracellular calcium, which results in the activation of nuclear gene transcription.

The first indication that LMP2A might be involved in BCR signal transduction was transfection studies utilizing the EBV-B-cell line BJAB. In BJAB cells expressing LMP2A, calcium mobilization was blocked following BCR cross-linking (Miller *et al.*, 1993). Further studies using LCLs infected with wild-type EBV indicated that normal B-cell signal transduction was also blocked as measured by induction of tyrosine phosphorylation, calcium mobilization, and activation of nuclear gene transcription following BCR cross-linking (Miller *et al.*, 1994a,b, 1995). In these wild-type infected LCLs, the amino-terminal domain of LMP2A was tyrosine phosphorylated and was associated with the Src family PTKs and the Syk PTK (Burkhardt *et al.*, 1992; Miller *et al.*, 1995). BCR cross-linking failed to activate Lyn, Syk, PI3-kinase, PLCγ2, Vav, MAPK, and Shc in these same LCLs (Miller *et al.*, 1995). Syk, PI3-kinase, PLCγ2, and Vav were constitutively tyrosine phosphorylated, and the tyrosine phosphorylation remained stable after BCR cross-linking (Miller *et al.*, 1995). In contrast, cross-linking the BCR on LCLs with null mutations in LMP2A resulted in normal BCR signal transduction as measured by induction of tyrosine phosphorylation, calcium mobilization, and switch to lytic replication (Miller *et al.*, 1994b, 1995). These results in-

dicated that LMP2A is sufficient for the block in B-cell signal transduction observed in wild-type EBV-infected LCLs (Fig. 4).

# V. LMP2A SITE-SPECIFIC MUTANT LCLs

To further understand the LMP2A-mediated block on BCR signal transduction, a number of site-specific mutations changing tyrosines into phenylalanines (Y to F) and three deletion mutations have been engineered into the LMP2A gene and then incorporated into the viral genome to generate LMP2A recombinant LCLs (Fruehling *et al.*, 1996, 1998; Fruehling and Longnecker, 1997; Swart *et al.*, 1999). These mutations are described next. Site-specific mutations at the proline-rich regions, serine, and threonine residues are currently under investigation. These studies will be vital in identifying...

![Diagram](attachment:diagram.png)

**Fig. 4** B-cell signal transduction in LMP2A expressing cells. In EBV latently infected B cells, LMP2A is expressed and the multiple hydrophobic domains of LMP2A mediate aggregation in the plasma membrane, where the amino-terminal domains of LMP2A resemble cross-linked receptor tails and become tyrosine phosphorylated. The Src family PTKs and the Syk PTK bind. Other SH2-containing proteins may also bind. This complex then blocks signal transduction through the BCR, preventing activation of lytic replication following BCR activation. This LMP2A-mediated block can be bypassed by chemical inducers of gene transcription, such as phorbol esters (PMA) or calcium ionophores (A23187). Activation to EBV lytic replication may be mediated by an as yet unidentified pathway such as an interleukin cytokine pathway (IL-x and IL-xR) that is not blocked by LMP2A.

identifying regions of LMP2A that may be important for LMP2A function and provide insight into the involvement of yet identified proteins in LMP2A regulation of EBV latency.

# A. Tyrosine Point Mutant LCLs: Y112

Initial studies using transient transfection of LMP2A cDNA mutants into BJAB cells identified tyrosine 112 (Y112) as the site of Lyn binding to LMP2A (Fruehling *et al.*, 1998). Unlike wild-type EBV-infected LCLs, LCLs with the LMP2AY112F mutation behaved as LMP2A null mutations. They were not blocked in BCR signal transduction, calcium mobilization, induction of tyrosine phosphorylation, or activation of lytic replication (Fruehling *et al.*, 1998). Interestingly, LMP2A in the Y112F mutant LCLs was not tyrosine phosphorylated before or after BCR cross-linking (Fig. 5) (Fruehling *et al.*, 1998). In addition, two cellular proteins of 60 and 57 kDa were constitutively phosphorylated in LMP2AY112F mutant infected LCLs. These proteins may represent an important early step required for LMP2A func-

![Figure 5](attachment://image.png)

**Fig. 5** LMP2A phosphorylation in EBV+LMP2AY112F LCLs. The BCR was untreated (-) or activated for the indicated time (1 or 5 min) in wild-type or mutant infected LCLs. The cells were then lysed and LMP2A was immunoprecipitated and separated by duplicate SDS-PAGE. Separated proteins were transferred to immobilon. Parallel immunoblots were probed with an anti-LMP2A antibodies (A) or anti-phosphotyrosine antibodies (B). (A) LMP2A expression in two EBV+LMP2A+ LCLs (A, lanes 1–6) and three EBV+LMP2AY112F point mutant LCLs (A, lanes 10–18) was similar before (-) and after (1 or 5 min) BCR activation. No LMP2A was detected in the EBV+LMP2A- LCL included as a negative control (A, lanes 7–9). (B) LMP2A tyrosine phosphorylation before and after BCR activation. In two EBV+LMP2A+ LCLs, LMP2A was constitutively phosphorylated, which did not change with BCR activation (B, lanes 1–6). LMP2A was not phosphorylated in the three EBV+LMP2AY112F point mutant LCLs even after BCR activation (C, lanes 10–18). (Reprinted with permission from Fruehling *et al.*, 1998)

tion. These EBV recombinant LCLs infected with the LMP2AY112F mutation have defined tyrosine Y112 as essential for LMP2A function and the binding of the Src family PTKs to LMP2A.

# B. Tyrosine Point Mutant LCLs: Y74 AND Y85

To further test the importance of the LMP2A ITAM in LMP2 function, LCLs were derived containing either a tyrosine to phenylalanine point mutation at LMP2A tyrosine 74 (Y74F) or tyrosine 85 (Y85F). In LMP2AY74F and LMP2AY85F LCLs, B-cell signal transduction was similar to LMP2A null LCLs as assessed by mobilization of intracellular calcium, activation of tyrosine phosphorylation, and the induction of lytic viral replication after BCR cross-linking (Fruehling and Longnecker, 1997). In addition, Syk was co-immunoprecipitated with LMP2A in wild-type LCLs, but not in either the LMP2AY74F or LMP2AY85F LCLs despite LMP2A being tyrosine phosphorylated (Fig. 6) (Fruehling and Longnecker, 1997). Finally, like LMP2A null LCLs, Syk was not phosphorylated in the absence of BCR cross-linking, but rapidly became phosphorylated with a concomitant increase in kinase activity following BCR cross-linking, unlike wild-type LCLs in which Syk phosphorylation and activity remained unchanged after BCR cross-linking (Fruehling and Longnecker, 1997). These data demonstrated that each of the two tyrosine residues of the LMP2A ITAM was essential for the LMP2A-mediated block of BCR signal transduction and binding of the Syk PTK to LMP2A.

# C. Deletion Mutant and Tyrosine Point Mutant LCLs: Y60, Y64, and Y101

In addition to the Y74, Y85, and Y112 tyrosine point mutations, the five remaining tyrosines have been mutated by either point or deletion mutation. Analysis of EBV-transformed LCLs infected with a tyrosine to phenylalanine mutation at tyrosine 60, 64, or 101 indicated that the expression, subcellular localization, or phosphorylation of LMP2A was unaffected by these mutations (Swart *et al.*, 1999). In addition, normal BCR signal transduction was similarly blocked when compared to wild-type EBV-infected LCLs as measured by calcium mobilization, tyrosine phosphorylation, and the induction of BZLF1. Although somewhat surprising, it is apparent that tyrosines 60, 64, and 101 of LMP2A are not required for LMP2A function *in vitro*. Previously, it was hypothesized that specific SH2 domain containing proteins may bind LMP2A on phosphorylation of the LMP2A tyrosines (Longnecker and Miller, 1996). The potential Y60 SH2 binding motif has the best ho-

Fig. 6 LMP2A co-immunoprecipitates with the Syk PTK in EBV+LMP2A+ LCLs but not in LMP2A ITAM mutant LCLs. (A) Expression of LMP2A is detected in an EBV+LMP2A+ LCL (A, lane 1), an EBV+LMP2AY74 LCL (A, lane 3), and an EBV+LMP2AY85 LCL (A, lane 4), whereas no LMP2A was detected in an EBV+LMP2A- LCL (A, lane 2). (B) Equivalent levels of the Syk PTK are immunoprecipitated from all four of the LCLs: EBV+LMP2A+ LCL (B, lane 1), EBV+LMP2A- LCL (B, lane 2), EBV+LMP2AY74 LCL (B, lane 3), and EBV+LMP2AY85 LCL (B, lane 4). (C) LMP2A co-immunoprecipitates with Syk in an EBV+LMP2A+ LCL (C, lane 1), but not in an EBV+LMP2A- LCL (C, lane 2), an EBV+LMP2AY74 LCL (C, lane 3), and an EBV+LMP2AY85 LCL (C, lane 4). Unstimulated LCLs were lysed and immunoprecipitated in parallel with anti-LMP2A or anti-Syk antibodies. Following immunoprecipitation, proteins were separated by SDS–6% PAGE, and transferred to an immobilon. LCL clone numbers are indicated above each lane and protein standards are indicated in kilodaltons. (Reprinted with permission from Fruehling and Longnecker, 1997.)

mology for phosphotyrosines, which bind the SH2 domain of the adapter proteins Crk or Nck (Table I). These adapter molecules once recruited to LMP2A complexes might recruit additional proteins important in B-cell signal transduction and LMP2A function in B lymphocytes. The tyrosine 101 motif contains an arginine at the +3 position that is characteristic of the Csk SH2 preferred motif (Songyang et al., 1994). Csk downregulates the activity of Src family PTKs when they are recruited to the BCR (Howell and Cooper, 1994). The interaction of LMP2A with Csk would provide a convenient model for the LMP2A-mediated affect on Lyn activity seen in wild-type infected LCLs.

Previous studies had indicated a potential important functional role for tyrosines 60 and 64 of LMP2A. Deletion of amino acids 21–64 but not amino acids 21–36 resulted in an LMP2A protein with only partial function in

EBV-transformed LCLs grown *in vitro* (Fruehling *et al.*, 1996). In particular, there was little constitutive phosphorylation of cellular proteins in the 21–64 deletion mutant when compared to wild-type infected cells in the absence of BCR activation. In addition, LMP2A was only phosphorylated following BCR activation in contrast to wild-type infected LCLs in which LMP2A is highly tyrosine phosphorylated. Finally, calcium mobilization was blocked as in wild-type infected LCLs, whereas lytic viral replication was readily induced following BCR activation similar to LMP2A null infected LCLs. Because mutation of tyrosines 60 and 64 resulted in none of the phenotypes associated with the 21–64 deletion mutant, it is apparent that other LMP2A sequences within the deletion must be considered. Two proline-rich sequences were deleted, one a potential binding site for SH3 domains, and the other a PY motif that may be important for the binding of proteins containing a WW domain.

In summary, tyrosines 23, 31, 60, 64, and 101 are nonessential for LMP2A function *in vitro* (Fruehling *et al.*, 1996; Swart *et al.*, 1999). The importance of tyrosines 74, 85, and 112 in binding cellular PTKs was previously shown to be essential for LMP2A *in vitro* function. Although tyrosines 23, 31, 60, 64, and 101 are not necessary for the LMP2A-mediated block on BCR activation of protein tyrosine phosphorylation, calcium mobilization, and lytic replication, these tyrosines may be important in other LMP2A functions related to the persistence of EBV in the human host. A recent *in vivo* study, described later, using LMP2A transgenic mice indicates that LMP2A provides both B-cell development and survival signals (Caldwell *et al.*, 1998). Studies using a similar transgenic approach using these same mutations may reveal that these tyrosines are required for promoting B-cell development and/or survival *in vivo*.

VI. MODEL OF LMP2A AND LMP2B FUNCTION *IN VITRO*

From our studies of the LMP2A mutations in LCLs, we propose the model of LMP2A and LMP2B function depicted in Fig. 7. LMP2A is expressed and forms complexes by interaction of the multiple hydrophobic membrane-spanning domains in the plasma membrane of EBV-infected B cells. By forming constitutive membrane complexes, LMP2A is able to mimic an activated BCR receptor. Initially, the Src family PTKs, such as Lyn, are recruited to the LMP2A complexes possibly through the interaction of the Lyn SH3 domain with a LMP2A proline-rich region (Fig. 7A). Alternatively, Lyn may be recruited to the DQSL sequence and is similar to the DCSM sequence, which has been shown to be important for recruitment of Lyn to the BCR associated pro-

Fig. 7 Model for LMP2A function. (A) The Src family PTK Lyn is recruited to LMP2A possibly by an interaction of the Lyn SH3 domain with a LMP2A proline-rich region or interaction of the Lyn unique region with the LMP2A DQSL sequence. WW domain-containing proteins may also be recruited to a nonphosphorylated PPPPY motif. (B) LMP2A is phosphorylated at Y112, by the Lyn PTK. Phosphorylation of the unidentified cellular proteins, p57 and p60, is induced in both wild-type EBV+LMP2A- and EBV+LMP2AY112F LCLs prior to BCR cross-linking. (C) Once Y112 is phosphorylated, the Lyn SH2 domain binds. Subsequent to the binding of Lyn, the remaining LMP2A tyrosines are phosphorylated including the LMP2A ITAM. Binding of Lyn to Y112 and the subsequent phosphorylation of LMP2A is blocked in the Y112F mutant. (D) The Syk PTK, other SH2 containing proteins, and MAPK bind to phosphorylated LMP2A. Once bound to LMP2A their activities are reduced and they are no longer able to participate in BCR signal transduction. Binding of Syk is blocked in LMP2A ITAM (Y74F and Y85F) mutants, but not LMP2A phosphorylation.

tein Igα (Pleiman *et al.*, 1994). Upon recruitment of Lyn to LMP2A, a 60-kDa protein and 57-kDa protein become phosphorylated (Fig. 7B). This is followed by the phosphorylation of LMP2A and the binding of Lyn via its SH2 domain to Y112 of LMP2A (Fig. 7C). Once bound, Lyn phosphorylates the remaining LMP2A tyrosines. In the Y112F mutation, the initial phosphorylation of
LMP2A at Y112 is blocked, thereby preventing interaction of the Lyn SH2 domain with Y112 and subsequent phosphorylation of LMP2A.

Following phosphorylation of LMP2A, other SH2 containing proteins are recruited to LMP2A complexes. Specifically, the Syk PTK binds to the phosphorylated LMP2A ITAM (Fig. 7D). Binding of Syk to LMP2A is blocked when either of the tyrosines within the LMP2A ITAM is mutated, although LMP2A is tyrosine phosphorylated in both ITAM mutants (Fruehling and Longnecker, 1997). Other unidentified SH2 domain-containing proteins may also be recruited to LMP2A phosphotyrosines, such as PI(3)-kinase or candidate proteins shown in Table I. LMP2A associates with at least eight unidentified proteins (Burkhardt *et al.*, 1992). If activated, PI(3)-kinase may prevent apoptosis of EBV infected cells by activation of the serine-threonine kinase Akt resulting in the phosphorylation of BAD, which is an important regulator of apoptosis (Datta *et al.*, 1997; del Peso *et al.*, 1997; Downward, 1997). Other potential targets of PI(3)-kinase and the Akt pathway include Caspase 9 (Cardone *et al.*, 1998), or the transcription factor forkhead (Brunet *et al.*, 1999; Kops *et al.*, 1999). The function of other proteins bound to LMP2A may also be altered. Proteins such as MAPK, WW domain-containing proteins, or the SH2 domain-containing tyrosine phosphatase 1 (SHP-1) may also be recruited to LMP2A patches (Figs. 7A and 7D). SHP-1, when recruited to the BCR complex by the Fc receptor, results in down-modulation of BCR signal transduction (D'Ambrosio *et al.*, 1995). Ongoing studies are focused on determining the intriguing mechanism of LMP2A regulation of BCR signal transduction.

## VII. IN VIVO MODELS OF LMP2A FUNCTION

Because EBV is host specific, animal models need to be developed to understand LMP2A and LMP2B function *in vivo*. Recent studies using transgenic mice have proven informative for dissecting a function for LMP2A *in vivo* (Caldwell *et al.*, 1998). Transgenic mice were constructed that express the LMP2A in B lineage cells (Caldwell *et al.*, 1998). LMP2A expression resulted in the bypass of normal B-lymphocyte developmental checkpoints, allowing immunoglobulin negative cells to colonize peripheral lymphoid organs, indicating that LMP2A possesses a constitutive signaling activity (Caldwell *et al.*, 1998). B lymphocytes from mice bearing targeted deletions of recombinase activating gene 1 (RAG-1), RAG-2, or the immunoglobulin heavy chain are unable to make functional immunoglobulin and as a result have a complete absence of mature B cells in peripheral lymphoid organs (Kitamura *et al.*, 1991; Mombaerts *et al.*, 1992; Shinkai *et al.*, 1992). In animals lacking peripheral immunoglobulin gene expression by means of inducible Cre-loxp-mediated gene targeting, receptorless B lymphocytes areeliminated by apoptosis (Lam et al., 1997). These experiments illustrate the necessity for competent BCR signaling in all stages of B-cell development, from early B-cell ontogeny to peripheral B-cell maintenance. Surprisingly, B cells lacking BCR expression are prominent in the bone marrow and periphery of the LMP2A transgenic mice. Although these B cells exhibit many of the hallmarks of appropriately developing B lymphocytes; including downmodulation of CD43 expression, IL7 responsiveness, and colonization of splenic environments, they are unable to rearrange functional immunoglobulin genes and subsequently do not express a BCR.

As well as providing a developmental signal that bypasses the requirement for immunoglobulin gene expression in bone marrow, LMP2A provides a survival signal to progenitor and peripheral B cells as evidenced by the increase in CD19+BCR cells in spleens of LMP2A transgenic mice. Most striking is that the LMP2A transgene allows receptorless B cells to mature into CD43− cells in the bone marrow that are able to colonize the spleen in a RAG-1−/− background. LMP2A transgenic animals were bred to RAG-1−/− mice and subsequent TgE+RAG-1−/− animals were identified by PCR analysis of genomic tail DNA. Flow cytometric analysis of splenic cells from TgE+RAG-1−/− animals identified significant numbers of CD19+ cells, a population absent in spleens of nontransgenic RAG-1−/− animals (Fig. 8A). Similarly, bone marrow of TgE+RAG-1−/− animals exhibited a dramatic increase in CD19+ cells when compared to the nontransgenic RAG-1−/− littermates (Fig. 8B). Examination of CD43 expression in these bone marrow samples revealed that unlike nontransgenic RAG-1−/− littermates, TgE+RAG-1−/− bone marrow contained a large percentage of CD19+ cells that did not express CD43 (Fig. 8C).

## VIII. LMP2 FUNCTION IN EPITHELIAL CELLS

LMP2 transcripts are readily detected in NPC biopsies, but little is known about the role of the LMP2 genes in NPC and other epithelial proliferative disorders. LMP2A has been shown to coprecipitate with PTKs and was phosphorylated in vitro kinase assays from epithelial cell lines expressing LMP2A (Scholle et al., 1999). The use of LMP2A deletion mutants indicated that multiple LMP2A tyrosines were important for phosphorylation of LMP2A (Scholle et al., 1999). The phosphorylation of LMP2A was induced on adhesion to extracellular matrix proteins and the PTK Csk was shown to be important for LMP2A phosphorylation (Scholle et al., 1999). The phosphorylation of LMP2A in epithelial cells is particularly tantalizing because it provides evidence that LMP2A may alter normal epithelial biology in a fashion similar to the alteration of normal B-cell function by LMP2A. Ep-
ithelial cells are also rich in proteins that regulate signal transduction. The generation of transgenic models with targeted LMP2A expression to epithelial cells may provide answers.

IX. DISCUSSION

From the *in vitro* studies and initial studies with LMP2A transgenic mice, several roles for LMP2A function *in vivo* can be surmised. The LMP2A transgenic studies suggest LMP2A may resemble a BCR thus providing an inappropriate developmental and survival signal to B cells in latently infected human hosts. This observation may have importance in EBV-associated

![RAG-1<sup>-/-</sup> TgE<sup>+</sup> RAG-1<sup>-/-</sup>](image.png)

Fig. 8 Analysis of B lymphocytes in TgE x RAG-1<sup>-/-</sup> animals. Single cell suspension of bone marrow and spleen cells from nontransgenic RAG-1<sup>-/-</sup> and TgE+RAG-1<sup>-/-</sup> littermates were stained with the indicated antibodies and analyzed by flow cytometry. Plots show representative data from two independent experiments.

malignancies. Although Reed-Sternberg cells in Hodgkin’s disease have been characterized as germinal center derived B lineage cells, some of these cells contain somatic mutations resulting in the absence of immunoglobulin expression, thereby indicating that these cells do not require BCR signaling for survival (Braeuninger *et al.*, 1997; Kanzler *et al.*, 1996). By mimicking signals derived from the BCR, LMP2A may provide a signal that would maintain these cells in the absence of a competent BCR. This may be an important first step in the development of HD.

The role of LMP2A in EBV latency has been the subject of much conjecture as the message for LMP2A is readily detected in immune competent hosts with EBV latent infections (Miyashita *et al.*, 1995; Qu and Rowe, 1992; Tierney *et al.*, 1994). Our studies discussed earlier indicate LMP2A blocks normal BCR signal transduction in EBV latently infected LCLs grown in culture. This block in BCR signal transduction prevents switch from latent to lytic replication following BCR activation (Miller *et al.*, 1994b, 1995). These *in vitro* observations suggest that the *in vivo* role of LMP2A in latent EBV infection was to prevent activation of lytic EBV replication by BCR-mediated signal transduction (Longnecker and Miller, 1996; Miller *et al.*, 1994b, 1995). This LMP2A function would be important in preventing lytic replication in latently infected lymphocytes as they circulate in the peripheral blood, bone marrow, or lymphatic tissues where they might encounter antigens, super antigens, or other ligands that could engage B-cell receptors and activate EBV lytic replication (Fig. 9).

---

**Fig. 9** EBV infection and latency in the human host. Primary infection: Infectious virus is spread by saliva where it encounters a circulating B lymphocyte trafficking in the oral epithelium. EBV can infect oral epithelial cells, which can result in NPC and OHL. Once the B cell is infected, it becomes activated and is driven to proliferate by expression of all of the latency associated gene products (EBNAs, LMP1, and LMP2s). This activated phenotype is similar to immunoblastic lymphomas seen in patients with immune suppression. The infected cells may then traffic to other lymphoid organs where progenitor B cells may become infected. These may be important for the development of EBV-related malignancies, such as Hodgkin’s or Burkitt’s or latency. Upon the activation of a vigorous cytotoxic T-cell response, the proliferating infected B lymphocytes are killed. Latent infection: EBV is found in about 1 in $10^7$ or $10^9$ peripheral B cells depending on the human donor (Khan *et al.*, 1996; Yao *et al.*, 1985). This level remains relatively stable throughout the lifetime of the given human host. There is limited viral gene expression of LMP2 and EBNA1 (Chen *et al.*, 1995; Decker *et al.*, 1996; Khan *et al.*, 1996; Miyashita *et al.*, 1995; Qu and Rowe, 1992; Tierney *et al.*, 1994; Yao *et al.*, 1985). EBV may also gain access to the bone marrow where it may infect a B lineage stem cell. Reactivation: Virus is spread to uninfected individuals when a latently infected cell traffics to the oral epithelium where it is induced to undergo lytic replication by an as yet undefined mechanism. LMP2A may have roles in each of the phases of the establishment and persistence of EBV latent infections. In the proliferation stage, LMP2A may provide a growth-stimulating signal, whereas in the latent stage, it may provide a signal that is important for the survival of the latently infected cell. In the reactivation stage, LMP2A may be important in blocking the activation of lytic infection, allowing it to occur only when an EBV-infected B cell traffics to the oral epithelium.

LMP2A Function

| Stage               | Cell Type                     | Expression Profile                          | Outcome           |
|---------------------|------------------------------|------------------------------------------|--------------------|
| Primary Infection   | Naive B Cell                 | ACTIVATED B CELL                         | Proliferation      |
|                     |                              | EBNA1                                    |                    |
|                     |                              | LMP1                                     |                    |
|                     |                              | LMP2                                     |                    |
|                     |                              | Immunoblastic                            |                    |
| Latent Infection    | TONSIL                      | PERIPHERAL B CELL                       | Survival          |
|                     |                              | EBNA1                                    |                    |
|                     |                              | LMP2                                     |                    |
|                     |                              | Burkitt's                                |                    |
|                     |                              | Hodgkin's                                |                    |
| Reactivation        | BONE MARROW                 |                                          | Inhibition        |
|                     |                              | LMP2                                     |                    |

More recent results from our transgenic model of EBV latency suggest another potential function for LMP2A *in vivo*. This function of LMP2A would be to provide a constitutive survival signal in mature B cells. This LMP2A derived signal would obviate the necessity for BCR-induced survival signals in EBV latently infected peripheral B cells, allowing persistence of virally infected B cells in the infected host. The importance of signal transduction through the BCR for B-cell survival has been demonstrated in murine conditional knockouts of the BCR receptor (Lam *et al.*, 1997) and in *in vitro* studies showing that activation of the BCR prevents apoptosis (Rathmell *et al.*, 1996; Rothstein *et al.*, 1995). Finally, LMP2A may be important in maintaining EBV-infected lymphocytes in an inactivated state, thereby preventing killing by cytotoxic T cells specific for LMP2A.

Studies to determine the site of EBV latency in immune competent human hosts have shown that in peripheral blood EBV resides in memory B lymphocytes which are CD19+, CD23−, B7−, sIg+, and sIgD− (Babcock *et al.*, 1998; Lam *et al.*, 1994; Miyashita *et al.*, 1995, 1997) and express EBNA1 and LMP2A (Babcock *et al.*, 1998; Kahn *et al.*, 1996; Miyashita *et al.*, 1995, 1997; Qu and Rowe, 1992; Tierney *et al.*, 1994). Other potential sites of EBV latency may include bone marrow, lymph nodes, or other lymphoid organs because EBV can be isolated from virtually any lymphoid tissue (Rickinson and Kieff, 1996). Our recent results with our transgenic model of EBV latency (Caldwell *et al.*, 1998) and the observation that bone marrow cells can harbor EBV (Gratama *et al.*, 1988, 1989), have led us to speculate that bone marrow may serve as the site of EBV latency. In this model of EBV latency, progenitor B cells may be infected by EBV when circulating B cells containing the virus traffic to bone marrow. Progeny of latently infected bone marrow cells could generate the relatively stable number of EBV-infected lymphocytes observed in peripheral blood of latently infected individuals (Lam *et al.*, 1991, 1994; Lewin *et al.*, 1987; Miyashita *et al.*, 1995; Yao *et al.*, 1985, 1991). Theoretically, once latently infected bone marrow cells enter the periphery, constitutive signaling from LMP2A could provide a survival signal that would maintain these cells in the absence of a competent BCR or the requirement for BCR signal transduction.

Whatever the site of EBV latency in the human host, peripheral B cells harboring the virus may undergo lytic replication when trafficking to the oral epithelium, thereby allowing shedding of virus to naive hosts. In infected hosts, latent infection enables the virus to escape immune destruction, while periodic reactivation transfers virus back to epithelial tissue where the virus replicates. However, the signals *in vivo* that may induce lytic replication are unknown. But it is likely that under specific conditions, such as the interaction of an EBV-infected lymphocyte with epithelial-produced cytokines or with epithelial surfaces, lytic replication may be induced. Both the *in vitro* and *in vivo* studies discussed will allow for the determination of the validity.

ty of the current ideas of LMP2 function in EBV latent infection and elucidate the mechanisms underlying the persistence of EBV in latently infected humans. The understanding of the unique pathologies associated with EBV latent infections may allow the development of novel therapeutics to treat EBV infections and its associated diseases.

# ACKNOWLEDGMENTS

R.L. is supported by Public Health Service grants CA62234 and CA73507 from the National Cancer Institute and DE13127 from the National Institute of Dental and Craniofacial Research. R.L. is a Scholar of the Leukemia Society of America.

# REFERENCES

Babcock, G. J., Decker, L. L., Volk, M., and Thorley-Lawson, D. A. (1998). *Immunity* 9, 395–404.

Benschop, R. J., and Cambier, J. C. (1999). *Curr. Opin. Immunol.* 11(2), 143–151.

Berger, C., Rothenberger, S., Bachmann, E., McQuain, C., Nadal, D., and Knecht, H. (1999). *Int. J. Cancer* 81(3), 371–375.

Braeuninger, A., Kuppers, R., Strickler, J. G., Wacker, H. H., Rajewsky, K., and Hansmann, M. L. (1997). *Proc. Nat. Acad. Sci. U.S.A.* 94(17), 9337–9342.

Brunet, A., Bonni, A., Zigmond, M. J., Lin, M. Z., Juo, P., Hu, L. S., Anderson, M. J., Arden, K. C., Blenis, J., and Greenberg, M. E. (1999). *Cell (Cambridge, Mass.)* 96(6), 857–868.

Burkhardt, A. L., Bolen, J. B., Kieff, E., and Longnecker, R. (1992). *J. Virol.* 66(8), 5161–5167.

Burkitt, D. (1962). *Nature (London)* 194, 232–234.

Burkitt, D. P. (1983). *Cancer (Philadelphia)* 51(10), 1777–1786.

Busson, P., Edwards, R. H., Tursz, T., and Raab Traub, N. (1995). *J. Gen. Virol.* 76, 139–145.

Caldwell, R. G., Wilson, J. B., Anderson, S. J., and Longnecker, R. (1998). *Immunity* 9, 405–411.

Cambier, J. C. (1995). *Immunol. Today* 16, 110.

Cambier, J. C., Pleiman, C. M., and Clark, M. R. (1994). *Annu. Rev. Immunol.* 12(457), 457–486.

Cardone, M. H., Roy, N., Stennicke, H. R., Salvesen, G. S., Franke, T. F., Stanbridge, E., Frisch, S., and Reed, J. C. (1998). *Science* 282 1318–1321.

Chang, Y., Cesarmen, E., Pessin, M. S., Lee, F., Culpepper, J., Knowles, D. M., and Moore, P. S. (1994). *Science* 266 1865–1869.

Chen, F., Zou, J. Z., di Renzo, L., Winberg, G., Hu, L. F., Klein, E., Klein, G., and Ernberg, I. (1995). *J. Virol.* 69(6), 3752–3758.

D'Ambrosio, D., Hippen, K. L., Minshoff, S. A., Mellman, I., Pani, G., Siminovitch, K. A., and Cambier, J. C. (1995). *Science* 268, 293–297.

Datta, S. R., Dudek, H., Tao, X., Masters, S., Fu, H., Gotoh, Y., and Greenberg, M. E. (1997). *Cell (Cambridge, Mass.)* 91(2), 231–241.

Decker, L. L., Klaman, L. D., and Thorley Lawson, D. A. (1996). *J. Virol.* 70(5), 3286–3289.

del Peso, L., Gonzalez-Garcia, M., Page, C., Herrera, R., and Nunez, G. (1997). *Science* 278, 687–689.

Downward, J. (1997). *Curr. Opin. Cell Biol.* 10, 262–267.

Epstein, M. A., Achong, B. G., and Barr, Y. M. (1964). *Lancet* 1, 702–703.

Franken, M., Annis, B., Ali, A. N., and Wang, F. (1995). *J. Virol* 69(12), 8011–8019.

Franken, M., Devergne, O., Rosenzweig, M., Annis, B., Kieff, E., and Wang, F. (1996). *J. Virol.* 70, 7819–7826.

Fruehling, S., and Longnecker, R. (1997). *Virology* 235, 241–251.

Fruehling, S., Lee, S. K., Herrold, R., Frech, B., Laux, G., Kremmer, E., Grasser, F. A., and Longnecker, R. (1996). *J. Virol* 70(9), 6216–6226.

Fruehling, S., Swart, R., Dolwick, K. M., Kremmer, E., and Longnecker, R. (1998). *J. Virol* 72, 7796–7806.

Fu, C., Turck, C. W., Kurosaki, T., and Chan, A. C. (1998). *Immunity* 9(1), 93–103.

Glenn, M., Rainbow, L., Aurade, F., Davison, A., and Schulz, T. F. (1999). *J. Virol.* 73, 6953–6963.

Gratama, J. W., Oosterveer, M. A., Zwaan, F. E., Lepoutre, J., Klein, G., and Ernberg, I. (1988). *Proc. Natl. Acad. Sci. U.S.A.* 85(22), 8693–8696.

Gratama, J. W., Oosterveer, M. A., Lepoutre, J., van Rood, J. J., Zwaan, F. E., Vossen, J. M., Kapsenberg, J. G., Klein, G., and Ernberg, I. (1989). *Transplant. Proc.* 21(1, Pt. 3), 3097–3098.

Howell, B. W., and Cooper, J. A. (1994). *Mol. Cell. Biol.* 14(8), 5402–5411.

Ikeda, M., Ikeda, A., Longan, L. C., and Longnecker, R. (2000). *Virology*, accepted for publication.

Ishiai, M. Kurosaki, M., Pappu, R., Okawa, K., Ronko, I., Fu, C., Shibata, M., Iwamatsu, A., Chan, A. C., and Kurosaki, T. (1999). *Immunity* 10(1), 117–125.

Kanzler, H., Kuppers, R., Hansmann, M. L., and Rajewsky, K. (1996). *J. Exp. Med.* 184(4), 1495–1505.

Karajannis, M. A., Hummel, M., Anagnostopoulos, I., and Stein, H. (1997). *Blood* 89(8), 2856–2862.

Khan, G., Miyashita, E. M., Yang, B., Babcock, G. J., and Thorley-Lawson, D. A. (1996). *Immunity* 5(2), 173–179.

Kieff, E. (1996). In “Fundamental Virology” (B. N. Fields, D. M. Knipe, and P. M. Howley, eds.), pp. 1109–1162. Lippincott-Raven, Philadelphia.

Kim, O. J., and Yates, J. L. (1993). *J. Virol.* 67(12), 7634–7640.

Kitamura, D., Roes, J., Kuhn, R., and Rajewsky, K. (1991). *Nature (London)* 350, 423–426.

Kops, G. J., de Ruiter, N. D., De Vries-Smits, A. M., Powell, D. R., Bos, J. L., and Burgering, B. M. (1999). *Nature (London)* 398, 630–634.

Lagunoff, M., Majeti, R., Weiss, A., and Ganem, D. (1999). *Proc. Nat. Acad. Sci. U.S.A.* 96(10), 5704–5709.

Lam, K., Kuhn, R., and Rajewsky, K. (1997). *Cell (Cambridge, Mass.)* 90, 1073–1083.

Lam, K. M., Syed, N., Whittle, H., and Crawford, D. H. (1991). *Lancet* 337, 876–878.

Lam, K. M., Whittle, H., Grzywacz, M., and Crawford,, D. H. (1994). *Immunology* 82(3), 383–388.

Laux, G., Perricaudet, M., and Farrell, P. J. (1988). *EMBO J.* 7(3), 769–774.

Laux, G., Economou, A., and Farrell, P. J. (1989). *J. Gen. Virol.* 70(Pt. 11), 3079–3084.

Lee, H., Guo, J., Li, M., Choi, J. K., DeMaria, M., Rosenzweig, M., and Jung, J. U. (1998). *Mol. Cell. Biol.* 18(9), 5219–5228.

Lewin, N., Aman, P., Masucci, M. G., Klein, E., Klein, G., Oberg, B., Strander, H., Henle, W., and Henle, G. (1987). *Int. J. Cancer* 39(4), 472–476.

Liebowitz, D. (1998). In “Humor Tumor Viruses” (D. McCaance, ed.), pp. 173–198. Am. Soc. Microbiol.

Liebowitz, D., Wang, D., and Kieff, E. (1986). *J. Virol.* 58(1), 233–237.

Ling, P. D., Ryon, J. J., and Hayward, S. D. (1993). *J. Virol.* 67(6), 2990–3003.

Longnecker, R. (1998). In “Humor Tumor Viruses” (D. McCaance, ed.), pp. 133–172. Am. Soc. Microbiol.

Longnecker, R., and Kieff, E. (1990). *J. Virol.* 64(5), 2319–2326.

Longnecker, R., and Miller, C. L. (1996). *Trends Microbiol.* 4, 38–42.

Longnecker, R., Druker, B., Roberts, T. M., and Kieff, E. (1991). *J. Virol.* 65(7), 3681–3692.

Longnecker, R., Miller, C. L., Miao, X.-Q., Marchini, A., and Kieff, E. (1992). *J. Virol.* 66, 6461–6469.

Longnecker, R., Miller, C. L., Miao, X. Q., Tomkinson, B., and Kieff, E. (1993a). *J. Virol.* 67(4), 2006–2013.

Longnecker, R., Miller, C. L., Tomkinson, B., Miao, X. Q., and Kieff, E. (1993b). *J. Virol.* 67(8), 5068–5074.

Miller, C. L., Longnecker, R., and Kieff, E. (1993). *J. Virol.* 67, 3087–3094.

Miller, C. L., Lee, J. H., Kieff, E., Burkhardt, A. L., Bolen, J. B., and Longnecker, R. (1994a) *Infect. Agents Dis.* 3, 128–136.

Miller, C. L., Lee, J. H., Kieff, E., and Longnecker, R. (1994b). *Proc. Natl. Acad. Sci. U.S.A.* 91(2), 772–776.

Miller, C. L., Burkhardt, A. L., Lee, J. H., Stealey, B., Longnecker, R., Bolen, J. B., and Kieff, E. (1995). *Immunity* 2, 155–166.

Miyashita, E. M., Yang, B., Lam, K. M., Crawford, D. H., and Thorley Lawson, D. A. (1995). *Cell (Cambridge, Mass.)* 80(4), 593–601.

Miyashita, E. M. Yang, B., Babcock, G. J., and Thorley-Lawson, D. A. (1997). *J. Virol.* 71(7), 4882–4891.

Mombaerts, P., Iacomini, J., Johnson, R. S., Herrup, K., Tonegawa, S., and Papaioannou, V. E. (1992). *Cell (Cambridge, Mass.)* 68, 869–877.

Panousis, C. G., and Rowe, D. T. (1997). *J. Virol.* 71, 4752–4760.

Pawson, T., and Gish, G. D. (1992). *Cell (Cambridge, Mass.)* 71(3), 359–362.

Pawson, T., and Scott, J. D. (1997). *Science* 278, 2075–2080.

Pleiman, C. M., Abrams, C., Gauen, L. T., Bedzyk, W., Jongstra, J., Shaw, A. S., and Cambier, J. C. (1994). *Proc. Natl. Acad. Sci. U.S.A.* 91(10), 4268–4272.

Poole, L. J., Zong, J.-C., Ciufó, D. M., Alcendor, D. J., Cannon, J. S., Ambinder, R., Orenstein, J. M., Reitz, M. S., and Hayward, G. S. (1999). *J. Virol.* 73, 6646–6660.

Qu, L., and Rowe, D. (1992). *J. Virol.* 66(6), 3715–3724.

Rathmell, J. C., Townsend, S. E., Xu, J. C., Flavell, R. A., and Goodnow, C. C. (1996). *Cell (Cambridge, Mass.)* 87(2), 319–329.

Reth, M. (1989). *Nature (London)* 338 383–384.

Rickinson, A. B., and Kieff, E. (1996). In “Fields Virology” (B. N. Fields, D. M. Knipe, and P. M. Howley, eds.), pp. 2397–2446. Lippincott-Raven, Philadelphia.

Rivailler, P., Quink, C., and Wang, F. (1999). *J. Virol.* (in press).

Roizman, B. (1993). In “The Human Herpesviruses” (B. Roizman, R. J. Whitley, and C. Lopez, eds.), pp. 1–9. Raven Press, New York.

Rothstein, T. L., Wang, J. K., Panka, D. J., Foote, L. C., Wang, Z., Stanger, B., Cui, H., Ju, S. T., and Marshak-Rothstein, A. (1995). *Nature (London)* 374, 163–165.

Russo, J. J., Bohenzky, R. A., Chien, M.-C., Chen, J., Yan, M., Maddalena, D., Parry, J. P., Peruzzi, D., Edelman, I. S., Chang, Y., and Moore, P. S. (1996). *Proc. Natl. Acad. Sci. U.S.A.* 93, 14862–14867.

Sample, J., Liebowitz, D., and Kieff, E. (1989). *J. Virol.* 63(2), 933–937.

Scholle, F., Longnecker, R., and Raab-Traub, N. (1999). *J. Virol.* 73(6), 4767–4775.

Shinkai, Y., Rathbun, G., Lam, K., Oltz, E. M., Stewart, V., Mendelsohn, M., Charon, J., Data, M., Young, F., Stall, A. M., and Alt, F. W. (1992). *Cell (Cambridge, Mass.)* 68, 855–867.

Songyang, Z., Shoelson, S. E., Chaudhuri, M., Gish, G., Pawson, T., Hasegawa, W. G., King, F., Roberts, T., Ratnofsky, S., Lechleider, R. J., Neel, B. G., Birge, R. B., Fajardo, J. E., Chou,

M. M., Hanafusa, H., Schaffhausen, B., and Cantley, L. C. (1993). *Cell* (Cambridge, Mass.) 72(5), 767–778.

Songyang, Z., Shoelson, S. E., McGlade, J., Olivier, P., Pawson, T., Bustelo, X. R., Barbacid, M., Sabe, H., Hanafusa, H., Yi, T., Ren, R., Baltimore, D., Ratnofsky, S., Feldman, R. A., and Cantley, L. C. (1994). *Mol. Cell. Biol.* 14(4), 2777–2785.

Songyang, Z., Lu, K. P., Kwon, Y. T., Tsai, L. H., Filhol, O. Cochet, C., Brickey, D. A., Soderling, T. R., Bartleson, C., Graves, D. J., DeMaggio, A. J., Hoekstra, M. F., Blenis, J., Hunter, T., and Cantley, L. C. (1996). *Mol. Cell. Biol.* 16(11), 6486–6493.

Speck, P., Kline, K. A., Cheresh, P., and Longnecker, R. (1999). *J. Gen. Virol.* 80, 2193–2199.

Sudol, M. (1996). *Prog. Biophys. Mol. Biol.* 65(1-2), 113–132.

Swart, R., Fruehling, S., and Longnecker, R. (1999). *Virology* 263, 485–495.

Tierney, R. J., Steven, N., Young, L. S., and Rickinson, A. B. (1994). *J. Virol.* 68(11), 7374–7385.

Weiss, A., and Littman, D. R. (1994). *Cell* (Cambridge, Mass.) 76(2), 263–274.

Wienands, J., Schweikert, J., Wollscheid, B., Jumaa, H., Nielsen, P. J., and Reth, M. (1998). *J. Exp. Med.* 188(4), 791–795.

Yao, Q. Y., Rickinson, A. B., and Epstein, M. A. (1985). *Int. J. Cancer* 35, 35–42.

Yao, Q. Y., Czarnecka, H., and Rickinson, A. B. (1991). *Int. J. Cancer* 48(2), 253–257.

Yates, J. L., Warren, N., and Sugden, B. (1985). *Nature* (London) 313, 812–815.

Yates, J. L., Camiolo, S. M., Ali, S., and Ying, A. (1996). *Virology* 222(1), 1–13.
