
Biosci. Biotechnol. Biochem., 64 (6), 1203–1209, 2000

**Transcription of emrKY is Regulated by the EvgA-EvgS Two-Component System in Escherichia coli K-12**

Akinori KATO, Hiroe OHNISHI, Kaneyoshi YAMAMOTO, Eiji FURUTA, Hiroyuki TANABE, and Ryutaro UTSUMI†

Department of Agricultural Chemistry, Kinki University, 3327-204 Nakamachi, Nara 631-8505, Japan

Received December 6, 1999; Accepted February 16, 2000

Spontaneous mutations have been isolated in *Escherichia coli* that result in the constitutive expression of an *emrKY* promoter. These mutations were found to be single-nucleotide substitutions within the linker region of the sensor protein EvgS, which is part of a two-component regulatory system along with EvgA. In the linker mutants (*evgS1* and *evgS4*), *emrKY* expression became constitutive and MIC against sodium deoxycholate was 20 mg/ml, eight-fold higher than in the wild type. Furthermore, the start site of transcription from the promoter of *emrKY* was identified; EvgA was shown to bind at the –52 to –84 region by the footprinting experiment.

Key words: drug efflux; two-component system; EvgA-EvgS; EmrK-EmrY

To survive, bacteria must adapt to their rapidly changing environment, and two-component regulatory systems are the key to this process. More than 250 two-component pairs have been discovered, and these systems sense and respond to a variety of environmental changes.¹)

One of the two-component systems characterized in *Escherichia coli*²) is EvgA/EvgS which is highly similar in sequence to the BvgA/BvgS system of *Bordetella pertussis*.³⁻⁵) EvgS (the sensor) contains a C-terminal “transmitter” domain and EvgA (the regulator) carries an N-terminal “receiver” domain.³) The predicted amino acid sequence of EvgS contains an N-terminal region that is localized in the periplasm and a cytoplasmic region that can be divided into four domains: linker, transmitter, receiver, and output (Hpt).⁵) The periplasmic domain of EvgS is involved in signal recognition and transduces the signals into the transcriptional regulation network via a cascade of phosphorylation.⁶,⁷) The EvgS protein is an autophosphorylating kinase that responds to environmental stimuli and modulates EvgA through phosphorylation.⁶) EvgA, a transcriptional control factor which binds the inverted repeat sequence found upstream of *emrKY* and *evgA* promoters, has been reported.⁷) The genes *emrK* and *emrY* (DDBJ, accession number D78168) which abut *evgA* constitute an operon⁸) and show high similarity to *emrA* and *emrB*, which code for the multidrug-efflux pump.⁹) The response of the EvgA/EvgS system to environmental signals is likely to be an important aspect of drug resistance.⁸,¹⁰) However, the function of the Evg system in nonpathogenic bacteria *E. coli* K-12 has not yet been well recognized. In this paper, we describe the isolation and characterization of a class of mutations that affect sensory transduction by the EvgA/EvgS system.

### Materials and Methods

#### Bacterial strains, plasmids, growth conditions, and P1 transduction.
*E. coli* strains and plasmids used in this study are listed in Table 1. They were usually grown at 37°C in LB medium.¹¹) Generalized P1 transduction was done by the standard protocol of Miller.¹¹)

#### Construction of single-copy *emrK′*-lacZ operon fusion.
The strain KMY1 was constructed as described by Simon et al.¹²) The fragment containing the possible *emrKY* promoter (GenBank entry AE000325, positions 3193 to 2652) was prepared by PCR with pSK001³) and the primers: forward, 5′-TAGAATTCTCCCTTTTGCAATGAAGC-3′, reverse 5′-CGGGATCCCGTACTAATCATGTCTTC-3′. After the amplified fragment was treated with EcoRI and BamHI, it was ligated with pRS551¹²) cut with the same enzymes to obtain pKMY1. After pKMY1 was recombined with phage λRS45,¹²) MK12 was lysogenized to isolate KMY1 on plates containing kanamycin (30 μg/ml).

---

† To whom correspondence should be addressed. Tel: 81-742-43-1511 or 7273 (ext 3309); Fax: 81-742-43-1445; E-mail: utsumi@nara.kin-dai.ac.jp

Abbreviations: Km<sup>r</sup>, kanamycin-resistant; Cm<sup>r</sup>, chloramphenicol-resistant; Ap<sup>s</sup>, ampicillin-sensitive; *cat*, chloramphenicol acetyltransferase gene

Table 1. Bacterial Strains and Plasmids Used in This Study

| Strains and plasmids | Description                                                                                   | Source or reference                          |
|----------------------|-----------------------------------------------------------------------------------------------|----------------------------------------------|
| Strains             |                                                                                               |                                              |
| JC7623              | thr-1 leu-6 thi-1 lacY1 galK2 ara-14 xyl-5 mtl-1 proA2 his4 argE3 str-31 tsx-33 supE344 recB21 recC22 abcB15 | Winans                                      |
| CA64                | JC7623 evgA::cat                                                                              | This work                                    |
| CS7                 | JC7623 evgS::cat                                                                              | This work                                    |
| RS3338              | fadL71::Tn10 fadR                                                                            | National Institute of Genetics, Japan        |
| CAG18484            | F⁻ zej-223::Tn10                                                                             | 22                                           |
| CAG18467            | F⁻ zfb-1::Tn10                                                                               | 22                                           |
| CAG18468            | F⁻ nupC510::Tn10                                                                             | 22                                           |
| MC4100              | F⁻ Δ(argF-lac)U169 araD139 rpsL150 ptsF25 fibB5301 rbsR deoC relA1                                 | 23                                           |
| MC1197              | MC4100 evgS::cat                                                                              | MC4100 × P1 (CS7) → Cmʳ                       |
| MR4100              | MC4100 fadL71::Tn10                                                                          | MC4100 × P1 (RS3338) → Tcʳ                    |
| MR2001              | MC4100 evgS1 fadL71::Tn10                                                                    | MC1197 × P1 (KMR2001) → Tcʳ, Cmˢ              |
| MR2004              | MC4100 evgS4 fadL71::Tn10                                                                    | MC1197 × P1 (KMR2004) → Tcʳ, Cmˢ              |
| MR204               | MC4100 evgA::cat fadL71::Tn10                                                                | MR4100 × P1 (CA64) → Cmʳ                      |
| MR1197              | MC4100 evgS::cat fadL71::Tn10                                                                | MR4100 × P1 (CS7) → Cmʳ                       |
| MC1061              | F⁻ araD139 Δ(ara-leu)7696 ΔlacY74 galU galK strA mcrA mcrB                                       | 23                                           |
| MK12                | MC4100 Δ(ara-leu)7696                                                                        | MC4100 × P1 (MC1061) → Araʳ(24), Leu⁻         |
| KMY1                | MK12 λRS45[Φ(emrK′-lacZ)]                                                                     | This work                                    |
| KMY204              | KMY1 evgA::cat                                                                               | KMY1 × P1 (CA64) → Cmʳ                        |
| KMY1197             | KMY1 evgS::cat                                                                               | KMY1 × P1 (CS7) → Cmʳ                         |
| KMY2001             | KMY1 evgS1                                                                                   | This work                                    |
| KMY2004             | KMY1 evgS4                                                                                   | This work                                    |
| KMR2001             | KMY2001 fadL71::Tn10                                                                         | KMY2001 × P1 (RS3338) → Tcʳ                   |
| KMR2004             | KMY2004 fadL71::Tn10                                                                         | KMY2004 × P1 (RS3338) → Tcʳ                   |
| Plasmids            |                                                                                               |                                              |
| pHSG399             | Cmʳ                                                                                          | 14                                           |
| pUC19               | Apʳ                                                                                          | 25                                           |
| pSK001              | pUC19, HindIII (5.6-kb) fragment containing evgAS                                               | 3                                            |
| pSK00CA1            | pSK001, HhaI fragment (1.1-kb) containing cat from pHSG399 is inserted into HpaI site           | This work                                    |
| pSK00CS1            | pSK001, HhaI fragment (1.1-kb) containing cat from pHSG399 is inserted into NruI site           | This work                                    |
| pMRK19P             | pUC19, EcoRI-BamHI (0.5-kb) fragment containing emrKY promoter from pSK001ᵃ                     | This work                                    |
| pBAD18              | Apʳ, a vector containing the arabinose P_BAD promoter                                          | 26                                           |
| pMV191              | pBAD18, Tcʳ, EcoRI-AbaI fragment containing tetR gene of pBR322 was blunt ended then inserted into the ScaI site of pBAD18 | Volkert                                     |
| pAS191              | pMV191, EcoRI-XbaI (4.3-kb) fragment containing evgAS and SD sequence from pSK001ᵇ               | This work                                    |
| pRS551              | Apʳ, Kmʳ, lacZ operon fusion vector                                                           | Simons                                       |
| pKMY1               | pRS551, Φ(emrK′-lacZ)                                                                        | This work                                    |

Tcʳ: tetracycline (12.5 μg/ml), Cmʳ: chloramphenicol (20 μg/ml), Araʳ: L-arabinose (1%).

ᵃ The possible emrKY promoter (GenBank entry AE000325, position 3217 to 2603 in Fig. 3) was amplified by PCR with pSK001 and the primers (forward, 5′-GGGGATCCATAATTATTGCGTTTCAT-3′; reverse, 5′-TCGAATTCCATTGACCAATAGGCAT-3′). It was treated with EcoRI and BamHI and inserted into the same site of pUC19.

ᵇ After the evgAS-coding region and the SD sequence (GenBank entry AE00325, positions 3158–7432) was amplified by PCR with pSK001 and the primers (forward, 5′-CTGAATTCCCTACAGGGAGAAAGGGA-3′; reverse, 5′-CATCTAGACATTGTGGGAGCCGCT-3′), it was inserted into EcoRI-XbaI sites of pMV191.

---

### Construction of the evgA and evgS null mutants.
The evgA and evgS null mutants were constructed by the method of Winans et al.¹³⁾ To prepare the Cmʳ cassette, pHSG399¹⁴⁾ was treated with HhaI, and a 1.1-kb fragment was recovered from an agarose gel and blunted with Pfu polymerase (Stratagene). It was inserted into the HpaI site and NruI site of the pSK001³⁾ to obtain pSK00CA1 and pSK00CS1 where evgA and evgS are disrupted, respectively. pSK00CA1 and pSK00CS1 were digested with BamHI and used to transform the strain JC7623.¹³⁾ Cmʳ Apˢ transformants were selected to isolate CA64 (evgA::cat) and CS7 (evgS::cat). These disruptions were confirmed by Southern hybridization (data not shown).

### Isolation of the constitutive mutants of emrKY expression.
To isolate linker mutants of EvgS, the spontaneous mutants that result in the constitutive expression of emrKY promoter were isolated as follows: An overnight culture of KMY1 grown in LB medium was appropriately diluted and plated onto a Minimal A medium¹¹⁾ with leucine and 0.2% lactose instead of sodium citrate and incubated at 37°C.

### DNA sequencing of linker mutation of evgS.
DNA sequencing of the evgAS coding region was done using PCR fragments amplified from chromosomal DNAs of strains KMY2001 and KMY2004. The PCR was done in two steps. First, PCR was done to obtain a 5.5-kb fragment containing the full coding region

of *evgAS* using the primers (forward, TCGAATTCCATTGACCAATAGGCAT; reverse, AGTGCGCGTTGGTACATCTC) and the chromosomal DNA. This product was diluted 1000-fold and used as template for the second PCR. In the second reactions, the 5.5-kb fragment was divided into the first half 2.8-kb and the second half 2.7-kb fragments. While the first half was amplified with the primers (forward, TCGAATTCCATTGACCAATAGGCAT; reverse, ACGAACTGAGCGTAA), the second half was amplified with the primers (forward, GGATGGGATATAATA; reverse, AAGCGAGAAATATGT). These products were purified by Ultrafree C3HK (Millipore). The complete *evgS* sequence was analyzed by primer walking using a Thermo sequence II dye terminator cycle sequencing kit (Amersham Pharmacia).

**Drug resistance.** The MICs of several drugs were measured as previously described.^[16)]

**DNase I Footprinting.** The primers A (5′-CATATCAAACACAGTGCA TGTGT-3′, 10 pmol) and B (5′-GAATTCGACAAGTATACCAT-3′, 10 pmol) were labeled with [γ-³²P] ATP (5000 Ci/mmol; Amersham) by T4 polynucleotide kinase (Toyobo). The labeled DNA fragments (probes A and B) encompassing the *emrKY* promoter region (−195 to +48) were prepared by PCR as follows: the labeled primer A (coding strand) or B (non-coding strand) was used with pMRK19P and Pfu DNA polymerase. They were recovered from polyacrylamide gels, precipitated with ethanol, and dissolved with H₂O. The purified EvgA⁶ and 40,000 cpm of labeled DNA probe (8 fmol) were mixed in 25 μl of binding buffer (10 mM Tris-HCl [pH 7.4], 3 mM MgCl₂, 5 mM CaCl₂, 1 mM Na₂EDTA, 50 mM KCl, 5% glycerol, 0.05% Nonidet P-40, and 3.2 ng/μl poly[dI-dC]) and incubated for 10 min at 37°C. The nucleolytic reactions were initiated by the addition of 5 ng DNasel (Takara). After incubation for 30 seconds at 25°C, 25 μl of phenol-chloroform was added and the aqueous phase was precipitated with ethanol. The precipitate was dissolved in sequencing loading buffer and electrophoresed in 6% polyacrylamide-urea sequencing gel. Maxam-Gilbart A + G reactions were also done and electrophoresed on the same gel. After drying, the gel was exposed onto an IP film (Fuji Bas 1000 Mac) overnight.

**Nucleotide sequence accession numbers.** The nucleotide sequences of *evgS1* and *evgS4* have been submitted to the GenBank and assigned no. AF201840 and AF201841, respectively.

**Results and Discussion**

**Characterization of *emrKY* expression**

In order to characterize the *emrKY* expression, we constructed disrupted mutants of *evgA* and *evgS* and investigated the expression of *emrKY* in these mutants. On a MacConkey plate, KMY1, which has the *emrKY* promoter fused to the *lac* operon, remained white, and the β-galactosidase activity was 37 units, and in KMY204 (*evgA::cat*) and KMY1197 (*evgS::cat*), no effect was observed (Fig. 1, lanes 1–3). However, in KMY1 containing pAS191, β-galactosidase production greatly increased by the presence of 0.2% arabinose (Fig. 1, lane 5). These results suggest that the *emrKY* promoter can be expressed when EvgA and EvgS were overproduced from the *araBAD* promoter.

**Genetic analysis of linker mutants of *EvgS***

The spontaneous mutant strains that result in the constitutive expression of *emrKY* promoter were isolated as described in Materials and Methods. Seventeen of twenty spontaneous *lac⁺* strains of KMY1 formed red colonies on the MacConkey plate and were expected to have constitutive expression from the *emrKY* promoter. We selected the most strongly constitutive strains and termed them KMY2001 and KMY2004. As predicted in KMY2001 and KMY2004, *lacZ* expression became constitutive (Fig. 1, lanes 8–9).

To map these mutation loci, P1 transduction experiments were done. Consequently the mutations in KMY2001 and KMY2004 were located within the *evgAS* operon region (Table 2). Furthermore, sequencing of the *evgAS* region of KMY2001 and KMY2004 found a single nucleotide substitution of

![Figure 1](#fig1)

**Fig. 1.** Expression of the *emrKY* Promoter by Overproduction of EvgA and EvgS.  
*E. coli* strains KMY1 (lane 1), KMY204 (lane 2), KMY1197 (lane 3), KMY1 containing pAS191 (lanes 4, 5) or pMV191 (lanes 6, 7), KMY2001 (lane 8), and KMY2004 (lane 9) were grown overnight in LB medium in the presence (lanes 5 and 7) or absence (lanes 1–4, 6, 8, 9) of 0.2% arabinose to assay β-galactosidase activity.^[11)] Data are the mean of duplicate values with standard deviation within 10%.

Table 2. Transduction Mapping of evgS1 and evgS4

| Donor | Chromosomal location of Tet<sup>r</sup> and Cm<sup>r</sup> (min) | Recipient |
| --- | --- | --- |
|  |  | KMY2001 (evgS1) | KMY2004 (evgS4) |
| CAG18484 | zej-223::Tn10 (49.50) | 0.6 (1/177) | 0.0 (0/281) |
| RS3338 | fadL71::Tn10 (50.50) | 47.4 (175/369) | 38.7 (174/450) |
| CAG18467 | zfb-1::Tn10 (51.00) | 80.5 (153/190) | 74.0 (148/200) |
| CA64 | evgA::cat (51.00) | 100.0 (571/571) | 100.0 (746/746) |
| CS7 | evgS::cat (51.00) | 94.4 (34/36) | 100.0 (18/18) |
| CAG18468 | nupC510::Tn10 (51.75) | 31.7 (40/126) | 25.7 (18/70) |

<sup>a</sup> The first selection was done for tetracycline resistance or chloramphenicol resistance, followed by scoring *lac*<sup>-</sup> colonies on MacConkey plates.

the linker region (evgS1: Phe<sup>577</sup>, TTC→Ser, TCC) and (evgS4: Glu<sup>701</sup>, GAA→Gly, GGA) of EvgS, respectively (Fig. 2). We analyzed other constitutive mutants, and found twelve mutations also localized at the *evgAS* locus while others were not identified (data not shown).

Previously the linker mutations of BvgS (Fig. 2) were also reported to result in output signals in the absence of overt stimuli.<sup>17-19</sup> Parkinson *et al.* proposed that communication between the periplasmic receptor and cytoplasmic signaling domains occurs by direct propagation of conformational changes through the linker.<sup>20</sup> The constitutive mutations, *evgS1* and *evgS4*, would then lock the sensor protein, EvgS, into an active conformation to induce *emrKY* expression, regardless of the state of the periplasmic region or the presence of modulating signals.

Analysis of the *emrKY* cis-acting regulatory region

The transcriptional start site of the *emrKY* operon was identified by S1 nuclease assay. No transcript from the *emrKY* operon was detected using strains MR4100, MR204, and MR1197 (Fig. 3, lanes 1–3). However, a transcript which started 218-bp upstream of the ATG start codon of *emrK* was confirmed in both of the constitutive mutants, MR2001 and MR2004 (Fig. 3, lanes 4 and 5). The possible EvgA binding region,<sup>7</sup> which was observed previously by the gel retardation assay, was found to be –65 to –91 region containing the inverted repeat (Fig. 3). To obtain a more detailed understanding of how EvgA interacts with the *emrKY* regulatory region, we examined the interaction between EvgA and the DNA on both the coding and non-coding strands using DNase I footprinting technique. As shown in Fig. 4, bases in the region between –52 to –84 were protected on both strands.

Drug resistance and *emrKY* expression

The effects of *evgS1* and *evgS4* on drug-resistance were investigated. The MIC of sodium deoxycholate in KMY2001 and KMY2004 was 20 mg/ml, eight-

Fig. 2. Mapping of the Linker Mutations of EvgS.

The linker mutations of KMY2001 (*evgS1*) and KMY2004 (*evgS4*) were identified directly by genomic sequencing with the PCR method.<sup>27</sup> The alignment of the amino acid sequence of the linker region<sup>17</sup> is shown. The phosphorylated amino acids are shown as H729, H721, D1023, D1009, H1172, and H1137.<sup>5,6</sup> Asterisks and dots mean ‘identical’ and ‘similar’, respectively. Boldface type indicates amino acid substitution in KMY2001 and KMY2004. Bvg-constitutive mutations previously isolated<sup>17-19</sup> are also shown under the BvgS sequence. TM: transmembrane region.

fold higher than in KMY1, KMY204, and KMY1197 (Table 3). As shown in Fig. 1 and Fig. 3, transcription from the possible *emrKY* promoter was much induced in KMY2001 and KMY2004, but not in KMY1, KMY204, and KMY1197. More sodium deoxycholate may have effluxed when *emrKY* expression was constitutively synthesized in the locked

The EvgA-EvgS Two-component System

Table 3. Drug Resistance

| Strain     | Genotype    | Sodium cholate | Sodium deoxycholate | Tetracycline |
|------------|-------------|----------------|---------------------|--------------|
| KMY1       | wild        | 40,000         | 2,500               | 0.6          |
| KMY2001    | evgS1       | 40,000         | 20,000              | 0.6          |
| KMY2004    | evgS4       | 40,000         | 20,000              | 0.6          |
| KMY204     | evgA::cat   | 40,000         | 2,500               | 0.6          |
| KMY1197    | evgS::cat   | 40,000         | 2,500               | 0.6          |

Fig. 3. Identification of the emrKY Promoter.

(a) S1 nuclease assay. 430-bp EcoRI-DdeI fragment of pMRK19P was used as Probe DNA. *E. coli* cells were grown in LB overnight. The overnight culture (0.2 ml) of *E. coli* strains MR4100 (lane 1), MR204 (lane 2) MR1197 (lane 3), MR2001 (lane 4), and MR2004 (lane 5) was diluted 100-fold into 20 ml of LB and grown to mid-log phase ($OD_{600}$, 0.8) to prepare total RNA. S1 nuclease assay was done as previously described. Electrophoresis was done with a 4% acrylamide sequencing gel. Lanes A + G represent the Maxam-Gilbert sequence reaction. PemrKY points to the corresponding protected transcript. Transcriptional start site is marked with asterisks. (b) DNA sequence (coding strand) around the promoter of emrKY (GenBank entry AE000325, positions 2989 to 2878). The inverted region is indicated by arrows facing each other.

Fig. 4. DNase I Footprinting Analysis of the cis-acting Regulatory Region.

The coding (a, probe A) or non-coding (b, probe B) strand containing the region that extends from base-pairs –191 to +48 upstream of emrKY operon was labeled and used for DNase I footprinting analysis as described in Materials and Methods. Lanes A + G represent the Maxam-Gilbert sequence reaction. Lanes 2, 3, and 4 contain 0.2, 0.6, 1.8 $\mu$M of EvgA, respectively.

mutants of EvgS. Bile salts containing sodium deoxycholate appeared to be effluxed by the AcrAB and EmrAB system, but unknown efflux systems by the proton motive force also play an important role.²¹⁾ Our results suggest that the EmrKY system can efflux sodium deoxycholate when EvgA/EvgS signaling is initiated.

In this study, we used a selective scheme to isolate mutations (*evgS1* and *evgS4*) in *E. coli* that specifically affect sensory transduction. The *evgS1* and *evgS4* mutations map to a linker region of EvgS (Fig. 2). They were single-nucleotide substitution within *evgS* that lead to amino acid replacement, causing the constitutive expression from the promoter of the *emrKY* operon. These results, therefore, provide direct evidence that the EvgA/EvgS two-component system is involved in the *emrKY* expression.

### Acknowledgments

We thank M. Volkert for the gift of plasmid pMV191 and R. W. Simon for providing pRS551 and λRS45.

### References

1) Hsing, W., Russo, F. D., Bernd, K. K., and Silhavy, T., Mutations that alter the kinase and phosphatase activities of the two-component sensor EnvZ. *J. Bacteriol.*, 180, 4538–4546 (1998).

2) Egger, L. A., Park, H., and Inouye, M., Signal transduction via the histidyl-aspartyl phospho-relay. *Genes to Cells*, 2, 167–184 (1997).

3) Utumi, R., Katayama, S., Taniguchi, M., Horie, T., Ikeda, M., Igaki, S., Nakagawa, H., Miwa, A., Tanabe, H., and Noda, M., Newly identified genes involved in the signal transduction of *Escherichia coli* K-12. *Gene*, 140, 73–77 (1994).

4) Uhl, M. A. and Miller, J. F., *Bordetella pertussis* BvgAS virulence control system. In “Two-component signal transduction”, eds. Hoch, J. A. and Silhavy, T. J., ASM Press, Washington, D.C., pp. 333–349 (1995).

5) Peraud, A. L., Kimmel, B., Weiss, V., and Gross, R., Specificity of the BvgAS and EvgAS phosphorelay is mediated by the C-terminal Hpt domains of the sensor proteins. *Mol. Microbiol.*, 27, 875–887 (1998).

6) Utumi, R., Kawamoto, K., Yamazaki, K., Taniguchi, M., Yoshioka, S., and Tanabe, H., Characterization of the signal transduction via EvgS and EvgA in *Escherichia coli*. *J. Gen. Appl. Microbiol.*, 42, 155–162 (1996).

7) Tanabe, H., Yamasaki, K., Kato, A., Yoshioka, S., and Utumi, R., Identification of the promoter region and the transcriptional regulatory sequence of the *evgAS* operon of *Escherichia coli*. *Biosci. Biotechnol. Biochem.*, 62, 286–290 (1998).

8) Tanabe, H., Yamasaki, K., Furue, M., Yamasaki, K., Katoh, A., Yamamoto, M., Yoshioka, S., Tagami, H., Aiba, H., and Utumi, R., Growth

phase-dependent transcription of *emrKY*, a homolog of multidrug efflux *emrAB* genes of *Escherichia coli*, is induced by tetracycline. *J. Gen. Appl. Microbiol.*, 43, 257–263 (1997).

9) Lomovskaya, O. and Lewis, K., *emr*, an *Escherichia coli* locus for multidrug resistance. *Proc. Natl. Acad. Sci. USA*, 89, 8938–8940 (1992).

10) Utumi, R., Yamasaki, K., Furue, M., Katoh, A., Kawamoto, K., Yoshioka, S., and Tanabe, H., Induction mechanism of drug resistances by the two-component system, EvgS and EvgA. *Nucleic Acids Symposium Series No. 35*, 309–310 (1996).

11) Miller, J. H., Experiments in molecular genetics. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y. (1972).

12) Simon, R. W., Hausman, F., and Kleckner, N., Improved single and multicopy *lac*-based cloning vectors for protein and operon fusions. *Gene*, 53, 85–96 (1987).

13) Winans, S. C., Elledge, S. J., Krueger, J. H., and Walker, G. C., Site-directed insertion and deletion mutagenesis with cloned fragment in *Escherichia coli*. *J. Bacteriol.*, 161, 1219–1221 (1985).

14) Takeshita, S., Sato, M., Toba, M., Masahashi, W., and Hashinmoto-Gotoh, T., High-copy-number and low-copy-number plasmid vectors for *lacZ* alpha-complementation and chloramphenicol or kanamycin-resistance selection. *Gene*, 61, 63–74 (1987).

15) Kato, A., Tanabe, H., and Utumi, R., Molecular characterization of the PhoP-PhoQ two-component system in *Escherichia coli* K-12: Identification of extracellular Mg²⁺-responsive promoters. *J. Bacteriol.*, 181, 5516–5520 (1999).

16) Okusu, H., Ma, D., and Nikaido, H., *acrAB* efflux pump plays a major role in the antibiotic resistance phenotype of *Escherichia coli* multiple-antibiotic-resistance (*mar*) mutants. *J. Bacteriol.*, 178, 306–308 (1996).

17) Miller, J. F., Johnson, S. A., Black, W. J., Beattle, D. T., Mekalanos, J. J., and Falkow, S., Constitutive sensory transduction mutations in the *Bordetella pertussis bvgS* gene. *J. Bacteriol.*, 174, 970–979 (1992).

18) Manetti, R., Arico, B., Rappuoli, R., and Scarato, V., Mutations in the linker region of BvgS abolish response to environmental signals for the regulation of the virulence factors in *Bordetella pertussis*. *Gene*, 150, 123–127 (1994).

19) Goyard, S., Bellalor, J., Mreaux, H., and Ullmann, A., *bvgS* gene that confers altered expression of the *fhaB* gene in *Escherichia coli*. *J. Bacteriol.*, 176, 5163–5166 (1994).

20) Ames, P. and Perkinson, J. S., Transmembran signaling by bacterial chemoreceptors: *E. coli* transducer with locked signal output. *Cell*, 55, 817–826 (1998).

21) Thanassi, D., Cheng, L. W., and Nikaido, H., Active efflux of bile salts by *Escherichia coli*. *J. Bacteriol.*, 179, 2512–2518 (1997).

22) Singer, M., Backer, T. A., Schmizler, G., Deischel, S. M., Goel, M., Dove, W., Jaacks, J. J., Grossman, A. D., Erickson, J. W., and Gross, C. A., A collection of strains containing genetically linked alternating antibiotic resistance elements for genetic mapping

of *Escherichia coli*. Microbiol. Rev., 53, 1–24 (1989).

23) Casadaban, M. J., Transposition and fusion of *lac* genes to selected promoters in *Escherichia coli* using bacteriophage lambda and Mu. *J. Mol. Biol.*, 104, 541–555 (1976).

24) Bremer, E., Silhavy, T. J., Weisman, J. M., and Weistrock, G. M., λplacMu: a transposable derivative of bacteriophage Lambda for creating *lacZ* protein fusions in a single step. *J. Bacteriol.*, 158, 1084–1093 (1984).

25) Yanisch-Perron, C., Vieira, J., and Messing, J., Im-

proved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. *Gene*, 33, 103–119 (1985).

26) Guzman, L-M., Belin, D., Carson, M. J., and Beckwith, J., Tight regulation, modulation, and high-level expression by vectors containing the arabinose P<sub>BAD</sub> promoter. *J. Bacteriol.*, 177, 4121–4230 (1995).

27) Russo, F., Slauch, J. M., and Silhavy, T. J., Mutations that affect separate functions of ompR the phosphorylated regulator of porin transcription in *Escherichia coli*. *J. Mol. Biol.*, 231, 261–273 (1993).
