
Pharmacology & Therapeutics 87 (2000) 27–49

Associate editor: E. Lolis

# The structure and mechanism of bacterial type I signal peptidases
## A novel antibiotic target

Mark Paetzel${}^{a}$, Ross E. Dalbey${}^{b}$, Natalie C.J. Strynadka${}^{a,*}$

${}^{a}$Department of Biochemistry and Molecular Biology, Faculty of Medicine, University of British Columbia,  
2146 Health Sciences Mall, Vancouver, BC, Canada V6T 1Z3  

${}^{b}$Department of Chemistry, The Ohio State University, Columbus, OH 43210, USA

---

### Abstract

Type I signal peptidases are essential membrane-bound serine proteases that function to cleave the amino-terminal signal peptide extension from proteins that are translocated across biological membranes. The bacterial signal peptidases are unique serine proteases that utilize a Ser/Lys catalytic dyad mechanism in place of the classical Ser/His/Asp catalytic triad mechanism. They represent a potential novel antibiotic target at the bacterial membrane surface. This review will discuss the bacterial signal peptidases that have been characterized to date, as well as putative signal peptidase sequences that have been recognized via bacterial genome sequencing. We review the investigations into the mechanism of *Escherichia coli* and *Bacillus subtilis* signal peptidase, and discuss the results in light of the recent crystal structure of the *E. coli* signal peptidase in complex with a β-lactam-type inhibitor. The proposed conserved structural features of Type I signal peptidases give additional insight into the mechanism of this unique enzyme. © 2000 Elsevier Science Inc. All rights reserved.

### Keywords:
Signal peptidase; Leader peptidase; Signal peptide; Protein translocation; Serine protease; Antibiotic target

### Abbreviations:
DFP, diisopropyl fluorophosphate; ER, endoplasmic reticulum; FRET, fluorescence resonance energy transfer; Imp, mitochondrial inner membrane peptidase (the mitochondrial signal peptidases, e.g., Imp1 and Imp2); MBP, maltose-binding protein; PMSF, phenylmethylsulphonyl fluoride; SDS-PAGE, sodium dodecyl sulphate-polyacrylamide gel electrophoresis; SPC, signal peptidase complex.

---

### Contents

1. Introduction ..... 28
2. Type I signal peptidases ..... 29
   2.1. Type I signal peptidases from gram-negative bacteria ..... 29
   2.2. Type I signal peptidases from gram-positive bacteria ..... 32
   2.3. Type I signal peptidases from higher order species ..... 33
   2.4. Regions of sequence similarity and identity (boxes B–E) ..... 33
3. Mechanistic studies of Type I signal peptidases ..... 33
   3.1. The signal peptide ..... 33
   3.2. Type I signal peptidase assays ..... 35
   3.3. Site-directed mutagenesis and site-directed chemical modification studies ..... 36
   3.4. Inhibitors of bacterial Type I signal peptidases ..... 36
4. The three-dimensional structure of Type I signal peptidase ..... 38
   4.1. X-ray crystal structure of a soluble, catalytically active fragment of *Escherichia coli* Type I signal peptidase ..... 38
   4.2. The protein fold ..... 38
   4.3. The membrane association surface ..... 38
   4.4. The tertiary structure of boxes B–E ..... 41
   4.5. The catalytic residues ..... 41
   4.6. The substrate-binding sites and substrate specificity ..... 42
   4.7. Predicted structural similarities and differences in Type I signal peptidases ..... 43
   4.8. A proposed mechanism for bacterial Type I signal peptidase ..... 44

---

* Corresponding author. Tel.: 604-822-8032 (laboratory), 604-822-0789 (office); fax: 604-822-5227.  
E-mail address: natalie@byron.biochem.ubc.ca (N.C.J. Strynadka).

0163-7258/00/$ – see front matter © 2000 Elsevier Science Inc. All rights reserved.  
PII: S0163-7258(00)00064-4

M. Paetzel et al. / Pharmacology & Therapeutics 87 (2000) 27–49

5. Conclusion . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
Type I signal peptidases (von Heijne, 1990). These differences support the choice of bacterial Type I signal peptidase as an attractive target for the development of novel antibiotic therapies.

It should be noted that there are three different types of bacterial signal peptidases. In addition to the Type I signal peptidase, there is a Type II signal peptidase that cleaves the signal peptides from lipid modified proteins and a Type III signal peptidase that specializes in the cleavage of the prepillin proteins. There is no sequence similarity among the three types of signal peptidases. Unlike the Type I signal peptidase, both Type II and Type III signal peptidases are not thought to be essential for bacterial viability. For an extensive review of the individual types of signal peptidases, see the comprehensive book by von Heijne (1994).

Type I signal peptidases (EC 3.4.21.89) are integral membrane serine endopeptidases. They belong to the serine protease family S26 (Rawlings & Barrett, 1994). Based on tertiary structure and conserved sequence motifs (Figs. 2 and 3), Type I signal peptidases have been classified into the evolutionary clan of serine proteases SF, which utilize a Ser/Lys catalytic dyad mechanism as opposed to the more common Ser/His/Asp catalytic triad mechanism (Table 1). To date, the majority of the work toward the understanding of the mechanism of bacterial Type I signal peptidases has been performed using the gram-negative *Escherichia coli* signal peptidase and the gram-positive *Bacillus subtilis* SipS. With the rapid increase in the number of putative signal peptidases being identified via the bacterial genome projects and the recent elucidation of the first crystal structure of a Type I signal peptidase (Paetzel et al., 1998), we are now more fully armed to investigate the potential of Type I signal peptidase as a universal and novel antibiotic target at the bacterial membrane surface.

expression is put under the control of the araB promoter (Dalbey & Wickner, 1985), or the natural promoter is partially deleted (Date, 1983), the limited expression of signal peptidase is associated with limited cell growth and division. The temperature-sensitive *E. coli* strain IT41, which has a mutation in the lepB gene, shows an accumulation of preproteins and a reduced growth rate when grown at the nonpermissive temperature of 42°C (Inada et al., 1989).

Typical of many gram-negative species, *E. coli* signal peptidase (323 amino acids, 35,988 Da) has two predicted amino-terminal transmembrane segments (residues 4–28 and 58–76), a small cytoplasmic domain (residues 29–57), and a large carboxy-terminal catalytic domain (residues 77–323) (Fig. 3). The *E. coli* signal peptidase membrane topology has been investigated both by proteolysis (Wolfe et al., 1983a; Moore & Miura, 1987) and by gene-fusion studies (San Millan et al., 1989). A model for the *E. coli* transmembrane domain has been proposed based on disulfide cross-linking studies (Whitley et al., 1993). This study suggests that aliphatic amino acids primarily form the interface between the two transmembrane segments and that the two helices pack against each other in a left-handed supercoil. Electrospray-mass spectrometry analysis has shown that the amino-terminus is blocked by a modification consistent with acetylation (Kuo et al., 1993).

The function of the small 28 amino acid cytoplasmic loop that links the two amino-terminal transmembrane segments currently is unknown. Early work with the *E. coli* Type I signal peptidase was made difficult by an autocatalytic degradation that occurred during the purification of the protein. It was shown by amino-terminal sequencing to occur between residues 40 and 41, which are predicted to lie in the small cytoplasmic domain of the enzyme (Talarico et al., 1991; Kuo et al., 1993) (Fig. 3). The protein is now most efficiently purified by replacing the residues in this region with a series of histidine residues and then utilizing nickel-affinity chromatography (Paetzel et al., 1997).

Deletion studies on the *E. coli* signal peptidase have shown that the first transmembrane segment and the cytoplasmic domain are not directly involved in catalysis (Bilgin et al., 1990). Kuo and co-workers (1993) were able to construct and purify a soluble, catalytically active fragment of *E. coli* signal peptidase that lacked both transmembrane segments and the small cytoplasmic segment. This construct, referred to as Δ2–75, was further characterized biochemically and kinetically (Tschantz et al., 1995) and then crystallized (Paetzel et al., 1995). It was found using both preprotein and peptide substrates that Δ2–75 required detergent or phospholipid for optimal activity, even though it lacks its transmembrane segments (Tschantz et al., 1995). It has been shown that Δ2–75 signal peptidase will bind to the inner and outer membranes of *E. coli*, as well as to vesicles composed of purified inner membrane lipids (van Klompenburg et al., 1998). Lipid surface tension experiments are consistent with the Δ2–75 penetrating into the lipid in a phosphatidylethanolamine-dependent fashion (van Klompen-

| #a.a. | MW     | pl   | TM                                      | Box B          | Box C         | Box D           | Box E            |
|-------|--------|------|-----------------------------------------|----------------|----------------|------------------|-------------------|
| Gram Negative                                                                                                                       |
| *Escherichia coli* | 323 | 35,973 | 7.65 | 4-28, 58-76 | SGSMMPTL | RGDIVVFK | DYIKRAVGLPGD | GDNRDNSADSR |
| *Salmonella typhimurium* | 324 | 35,778 | 6.01 | 4-28, 59-77 | SGSMMPTL | RGDIVVFK | DYIKRAVGLPGD | GDNRDNSADSR |
| *Pseudomonas fluorescens* | 284 | 31,903 | 6.85 | 7-28, 58-76 | SGSMKPTL | RGDMVVFR | NYIKRVVGGLPGD | GDNRDNSNDSR |
| *Haemophilus influenzae* | 349 | 39,734 | 5.88 | 4-23, 31-52, 84-105 | SGSMESTL | RGDIVVK | DYIKRIVGKGGD | GDHRDHSDDSR |
| *Helicobacter pylori* | 290 | 33,710 | 8.36 | 10-34 | SRSMVGTI | RGEVVVFI | YYVKRNFAIGGD | GDNRDNSSDSR |
| *Bradyrhizobium japonicum SipF* | 254 | 28,831 | 6.91 | 16-35 | SGSMKATL | RGDIVVFR | DYIKRVIGLPGD | GDNRDNSTDSR |
| *Bradyrhizobium japonicum SipS* | 259 | 27,614 | 9.84 | ND | SGSMEPTL | QGDVVVFR | AWVKRVVGGLPGD | GDNRDNSADSR |
| *Borrelia burgdorferi 1* | 211 | 25,043 | 9.89 | 19-43, 57-81, 112-132, 187-207 | SNEMLOPTI | MNDIVLYE | YKVSRIAAVQGD | NDNLSVLNDSR |
| *Borrelia burgdorferi 2* | 326 | 38,481 | 8.89 | 5-23, 39-58 | SGSMENTL | ESDIIIFE | FLVKRGAFADGK | GDNRDNSHDGR |
| *Borrelia burgdorferi 3* | 168 | 19,837 | 9.64 | 11-34 | GESMTPAI | QKYLLLWK | IAIKKIFAPIGE | GENKQISLDSR |
| *Treponema pallidum 1* | 235 | 26,880 | 9.56 | 16-37 | ADSMQPTL | RGDLVLAT | GLFKRAMNAVLG | CDNRIVSSDSR |
| *Treponema pallidum 2* | 512 | 57,619 | 9.22 | 2-23, 53-77 | SESMVPSF | RGDIVVFS | PLVKRIVALPGE | GLDTGQEGPSL |
| *Thermotoga maritima* | 306 | 36,082 | 6.48 | 13-34 | TGSMIPTI | IGEIVVFW | KYVKRLVGKGGD | GDNTKESLDCR |
| *Phormidium laminosum* | 203 | 22,485 | 5.64 | ND | SESMLPTL | RGDIIIVFH | AFIKRVIGLPGE | GDNRRNNSYDSH |
| *Azotobacter vinelandii* | 284 | 32,039 | 6.54 | 7-28, 59-77 | SGSMKPTL | RGDMVVFR | NYIKRVVGGLPGD | GDNRDNSNDSR |
| *Rickettsia prowazekii* | 264 | 30,870 | 8.73 | 16-34 | TGSMKATI | RGDIVVFR | RYIKRLIGLPGD | GDNDRRSNDSR |
| *Rhodobacter capsulatus* | 265 | 28,878 | 7.66 | 15-36 | SGSMKDTL | RGDVVVFR | DFIKRLIGLPGD | GDNRDNSEDSR |
| Gram Positive                                                                                                                        |
| *Bacillus subtilis SipS* | 184 | 21,047 | 9.35 | 15-39 | GDSMYPTL | RGDIVVLN | HYVKRIIGLPGD | GDNRRNSMDSR |
| *Bacillus subtilis SipT* | 193 | 21,854 | 9.46 | 16-37 | GSSMYPTL | RGDIVIIN | HYVKRLIGKPGE | GDNRLNSMDSR |
| *Bacillus subtilis SipP (pTA 1015)* | 186 | 21,251 | 9.41 | 11-30 | GKSMDPNL | RGDIIIILN | HYVKRLIGLPGD | GDNRQESMDSR |
| *Bacillus subtilis SipP (pTA 1040)* | 185 | 21,568 | 9.34 | 15-34 | GESMKPTL | RGDIVVLN | HYVKRLIGLPGD | GDNRQNSMDSR |
| *Bacillus subtilis SipU* | 187 | 21,182 | 9.43 | 16-36 | GSSMAPTL | RGDIIIVIH | SFVKRLIGLPGD | GDNRLNSLDSR |
| *Bacillus subtilis SipV* | 168 | 18,956 | 9.71 | 5-26 | GVSMNPTF | RFDIVLFK | VLIKRVIIGLPGE | GDNRIYSFDSR |
| *Bacillus subtilis SipW* | 190 | 20,678 | 5.59 | 3-24, 144-165 | SGSMEEPF | KGDIVTFM | AVTHRIVDITKQ | GDPGVMVYAFAV |
| *Bacillus amylo liquefaciens SipS 1* | 185 | 21,104 | 9.63 | 12-31 | GESMEPTL | RGQIVVLN | HYVKRIIGLPGD | GDNRRNSMDSR |
| *Bacillus amylo liquefaciens SipS 2* | 193 | 21,860 | 9.51 | 16-37 | GSSMYPTL | RGDIVIIN | HYVKRLIGKPGE | GDNRLNSMDSR |
| *Bacillus licheniformis* | 186 | 21,145 | 8.63 | 16-35 | GTSMDPNL | RGDIVIID | HYVKRLIGLPGD | GDNQRSMDSR |
| *Bacillus caldolyticus* | 182 | 21,263 | 9.84 | 12-34 | GKSMMPTL | RFDIIIVFH | DYVKRVIGLPGD | GDNRLSSWDSR |
| *Staphylococcus aureus SpsA* | 174 | 20,146 | 9.83 | 7-28 | NNDSMPTL | NGDIITYR | IYTSRIIAKPGQ | NDHDNNQHDSR |
| *Staphylococcus aureus SpsB* | 191 | 21,691 | 9.02 | 8-27 | GESMDPTL | KGNVVVFR | DYVKRVIGVPGD | GDNREVSKDSR |
| *Staphylococcus carnosus sipA* | 174 | 19,931 | 9.72 | 7-31 | NNEMSPTL | NGDVVMYK | TYFGRVIGLPQQ | NDNRANQSSTR |
| *Staphylococcus carnosus sipB* | 189 | 21,431 | 9.33 | 8-27 | GDSMYPTL | KGNVIVFH | DYVKRVIGMPGD | GDNREVSKDSR |
| *Streptococcus pneumoniae Spi* | 204 | 23,497 | 5.84 | 11-29 | GHSMDPNL | RFDIVVAH | DIVKRVIGMPGD | GDDRLVSSDSR |
| *Streptomyces coelicolor gi* | 148 | 16,069 | 11.67 | ND | GPSMVPTL | PGDVVVLR | LVVKRAAERRGA | GDNAFAGGDEST |
| *Streptococcus mitis* |  |  |  | ND, (partial sequence) | GHSMDPNL | RFDIVVAH | DIVKRVIGMPGD | GDDRLVSSDSR |
| *Streptomyces lividans SipX* | 320 | 34,548 | 9.25 | 57-81 | SGSMEQTI | RGDVVVFR | DLIKRVVGVGGD | GDHRSNSADSR |
| *Streptomyces lividans SipZ* | 258 | 26,536 | 9.73 | 31-52, 219-240 | TSSMTPTI | RGDVVVFK | PMVKRVVAVGGD | GDERRNSVDST |
| *Streptomyces lividans SipW* | 259 | 27,662 | 4.72 | 49-68 | SGSMERGL | RGDIVVFD | DYIKRVVGVGGD | GDHRSDSSDSR |
| *Streptomyces lividans SipY* | 291 | 31,470 | 5.52 | 41-62, 259-280 | SSSMENTL | RGEVVVFH | DLIKRVIGVAGD | GDHRQNSRDSR |
| *Streptomyces coelicolor 1* | 258 | 26,550 | 9.73 | 31-52, 219-240 | TSSMTPTI | RGDVVVFK | PMVKRVVAVGGD | GDERRNSVDST |
| *Streptomyces coelicolor 2* | 336 | 35,865 | 5.28 | 86-107, 304-325 | SSSMENTL | RGEVVVFH | DLIKRVIGVAGD | GDHRQNSRDSR |
| *Streptomyces coelicolor 3* | 446 | 49,018 | 11.11 | 57-81, 291-310 | SGSMEQTI | RGDVVVFR | DLIKRVVGVGGD | GDHRSNSADSR |
| *Streptomyces coelicolor 4* | 259 | 27,662 | 4.72 | 49-68 | SGSMERGL | RGDIVVFD | DYIKRVVGVGGD | GDHRSDSSDSR |
| *Aquifex aeolicus* | 256 | 29,676 | 6.02 | 8-28 | SASMEPTL | RGDMIVFK | DFIKRIIARGGD | GDNRDNSQDSR |
| *Chlamydia trachomatis* | 628 | 71,504 | 8.41 | 81-99, 602-620 | TGSMRPTI | RGELVVFT | RYIKRCMGKPGD | GDNCPMSSADSR |
| *Chlamydia pneumoniae* | 636 | 72,507 | 9.17 | 80-99, 600-619 | TGSMRPTI | RGGLVVFT | RYIKRCMGRPQD | GDNYPMSSADSR |
| *Mycobacterium leprae* | 289 | 31,086 | 5.67 | 56-75 | SESMEPTL | PGDIVFK | DLVKRVIAVGGQ | GDNRIHSADSR |
| *Mycobacterium tuberculosis* | 294 | 31,880 | 5.63 | 68-87 | SESMEPTL | PGDIVFR | DLVKRVIAVGGQ | GDNRTHSADSR |
| *Synechocystis sp PCC 6803 1* | 196 | 22,230 | 5.67 | 16-35 | SDMLPTL | VGDIIVFH | AFIKRVIALPGQ | GDNRRNNSNDSH |
| *Synechocystis sp PCC 6803 2* | 218 | 24,733 | 5.41 | ND | SSSMEPTL | RGEIVVFN | AFIKRIIGLPGD | GDNRRNNSYDSH |
| Mitochondrial                                                                                                                         |
| *Saccharomyces cerevisiae IMP 1* | 190 | 21,433 | 7.01 | 2-26 | GESMLPTL | MGDCIVAL | RICKRVTGMPGD | GDNLSHSLDSR |
| *Saccharomyces cerevisiae IMP 2* | 177 | 19,930 | 10.01 | 14-35 | GTSMQPTL | RDDIIILFK | VYCKRVKGLPFD | GDNYFHSIDSN |
| *Schizosaccharomyces pombe IMP 2* | 157 | 17,269 | 8.63 | 2-21 | GPSMMPTL | VGDVVVSA | HVCKRIIGMPGD | GDNIAHSLDSR |
| Thylakoid                                                                                                                            |
| *Arabidopsis thaliana 1* | 340 | 37,853 | 6.03 | ND | STSMYPTL | VSDIVIFK | VFIKRIVASEGD | GDNRNKSFDSH |
| *Arabidopsis thaliana 2* | 250 | 27,909 | 4.85 | ND | STSMYPTL | VSDIVIFK | VFIKRIVASEGD | GDNRNKSFDSH |
| *Arabidopsis thaliana 3* | 313 | 34,515 | 7.1 | 16-40 | GPSMIPTL | RGDIVVIR | TPIKRVVGVVEGD | GDYTHNSRDSR |
| Archaea                                                                                                                              |
| *Pyroccus horikoshii* | 154 | 17,184 | 6.42 | 8-29 | SGSMEPVF | VGDIVYR | PIHRVRGIKYI | GDHNVPDIYY |
| *Methanococcus jannaschii* | 203 | 23,724 | 6.51 | 2-23, 174-195 | SDSMYPIM | VGDIVVYK | PVIRVIDKVEF | GDNNPIHDEPL |
| *Methanobacterium thermoautotrophicum* | 144 | 16,164 | 4.95 | 10-31 | SGSMEPVF | RGDIIVYD | PVIRVIGVETD | GDNNQDPDPAP |
| *Pyroccus abyssi 1* | 327 | 36,503 | 9.3 | 6-25, 135-153, 169-189, 291-310 | SDSMTPTI | VGDIIVFN | WTVHRVYAITES | GDNNVATDQQD |
| *Pyroccus abyssi 2* | 155 | 17,110 | 9.23 | 8-29 | SGSMRPVF | VGDIVYK | PIHRVRGIKQV | GDNNPVPDLYE |
| *Archaeoglobus fulgidus 1* | 151 | 16,534 | 6.59 | 5-26, 120-141 | GTSMLEL | TGDLLILIF | LITHRVVEKTSE | GDNLPREDEPV |
| *Archaeoglobus fulgidus 2* | 189 | 20,618 | 5.58 | 14-38 | SGSMEHL | YGDIVYK | PIHRARIAAYVHK | GDNVRTNQLPD |
| *Archaeoglobus fulgidus 3* | 290 | 32,667 | 9.1 | 3-22, 129-148, 180-198, 248-272 | SSSMEPLM | LGDVVAFK | LITHRVVEIGDG | GDAVEDVDPDFD |
| ER                                                                                                                                   |
| *Human SPC 18* | 167 | 19,131 | 9.58 | 6-30, 139-163 | SGSMEPAF | VGEIVVFR | PIVHRVLKIHEK | GDNNAVDDRGL |
| *Dog SPC 18* | 179 | 20,625 | 9.48 | 18-42 | SGSMEPAF | VGEIVVFR | PIVHRVLKIHEK | GDNNAVDDRGL |
| *Dog SPC 21* | 191 | 21,600 | 9.22 | 4-22, 34-55, 164-188 | SGSMEPAF | AGEIVVFK | PIVHRVIKVHEK | GDNEVDDRGL |
| *Rat SPC 18* | 179 | 20,599 | 9.15 | 18-42, 151-175 | SGSMEPAF | VGEIVVFR | PIVHRVLKIHEK | GDNNAVDDRGL |
| *Saccharomyces cerevisiae Sec11* | 167 | 18,762 | 9.19 | 10-34 | SGSMEPAF | VGDIVVYE | PIVHRVLRQHNN | GDNNAGNDISL |

M. Paetzel et al. / Pharmacology & Therapeutics 87 (2000) 27–49

The gram-negative  
Escherichia coli Type I  
signal peptidase  

The gram-positive  
Bacillus subtilis Type I  
signal peptidase SipS  

Outside  
Cytoplasmic  
membrane  
Inside  

Fig. 3. The membrane topology of the gram-negative *E. coli* Type I signal peptidase and the gram-positive *Bacillus subtilis* Type I signal peptidase SipS. The majority of gram-negative signal peptidases have multiple transmembrane segments. Most gram-positive signal peptidases are smaller and have a single transmembrane segment. The periplasmic region of all gram-negative signal peptidases contains a β-sheet Domain I and II. Domain I contains all of the conserved boxes of sequence and the catalytic and binding-site residues. Domain II and the extended β-ribbon are smaller in the gram-positive signal peptidases. The orientation of the catalytic domain relative to the plan of the lipid bilayer is proposed based on the location of the catalytic residues, as well as the large exposed hydrophobic surface (Paetzel et al., 1998). The gram-positive *Bacillus subtilis* SipS structure was modeled based on the *E. coli* signal peptidase crystal structure (Paetzel et al., 1998).

burg et al., 1998). It should be noted that all in vitro studies to date are consistent with bacterial Type I signal peptidases acting as monomeric enzymes and requiring no additional proteins or cofactors for catalytic activity.

Many other Type I signal peptidases from gram-negative species have been cloned, expressed in *E. coli*, and shown to have signal peptidase activity by genetically complementing the temperature-sensitive *E. coli* strain IT41. This has become a standard method for confirming that putative Type I signal peptidase genes impart Type I signal peptidase activity when expressed. The *E. coli* strain IT41 contains a mutation in the gene for Type I signal peptidase (*lepB*), which gives a temperature-sensitive phenotype (Inada et al., 1989). The strain shows normal growth at 32°C, but slow growth at 42°C. Transformation of this strain with a plasmid containing a Type I signal peptidase complements the mutant *lepB* gene, allowing growth at the elevated temperature. The nature of the mutation in the *lepB* gene has been shown to be a nonsense mutation at codon 39 (Cregg et al., 1996). Apparently, at the permissive temperature, there is read-through that is not allowed at the elevated temperature. The studies that have utilized this method for demonstrating Type I signal peptidase activity in gram-negative bacteria include those for the signal peptidase from *Pseudomonas fluorescens* (Black et al., 1992), the thermophilic cyanobacterium *Phormidium laminosum* (Packer et al., 1995), *Azotobacter vinelandii* (Jock et al., 1997), and *Bradyrhizobium japonicum* SipS/SipF (Bairl & Muller,

1998). In a similar experiment, the signal peptidase from *Salmonella typhimurium* was cloned, expressed, and was demonstrated to display Type I signal peptidase activity by using a second temperature-sensitive *E. coli* mutant where the expression of signal peptidase is repressed at 28°C (N4156::pGD28) (van Dijl et al., 1990). The Type I signal peptidase from *Rhodobacter capsulatus* has also been cloned and expressed (Klug et al., 1997). The inability to culture an *R. capsulatus* strain with a disrupted *lepB* gene is consistent with Type I signal peptidase being essential for the viability of the *R. capsulatus* cell (Klug et al., 1997).

In gram-negative bacteria, such as *E. coli*, *Pseudomonas fluorescens*, *Salmonella typhimurium*, *R. capsulatus*, and *Haemophilus influenzae*, there is only one identifiable Type I signal peptidase. Recently, it has been shown that *Bradyrhizobium japonicum* (Muller et al., 1995; Bairl & Muller, 1998) has two functional and distinct Type I signal peptidases (SipS and SipF). Mutant phenotype analysis suggests that the SipS and SipF exhibit different substrate specificity (Bairl & Muller, 1998). This would be similar to the situation with the mitochondrial inner membrane protease (Imp1 and Imp2) from yeast mitochondria (Nunnari et al., 1993). From an analysis of newly emerging bacterial genome sequences, it appears that there are likely several examples of multiple Type I signal peptidase genes in a single gram-negative species. For example, *Borrelia burgdorferi* has three putative Type I signal peptidases and *Treponema pallidum* and a *Synechocystis* sp. (PCC 6803) have two each (Fig. 2).

M. Paetzel et al. / Pharmacology & Therapeutics 87 (2000) 27–49

Table 1  
Evolutionary clans of serine proteases  

| Clan | Catalytic residues | Example |
|------|---------------------|---------|
| SA   | Ser/His/Asp        | Chymotrypsin (PDB: 2cga) |
| SB   | Ser/His/Asp        | Subtilisin (PDB: 2st1) |
| SC   | Ser/His/Asp        | Carboxypeptidase C (PDB: lyc) |
| SE   | Ser/Lys            | D-Ala-D-Ala transpeptidase (PDB: 3pte) |
| SF   | Ser/Lys            | Type I signal peptidase (PDB: 1b12) |
| SH   | Ser/His/His        | Cytomegalovirus protease (PDB: 1cmv) |

two plasmid encoded [SipP pTA1015 and SipP pTA1040]  
(Meijer et al., 1995; Tjalsma et al., 1997, 1998). The Bacillus subtilis signal peptidases are all approximately the same size, ranging from 18,956 to 21,854 Da. They all have one proposed transmembrane segment, except for SipW, which has a second predicted transmembrane segment at its carboxy-terminus (Tjalsma et al., 1998). SipW is more like the eukaryotic endoplasmic reticulum (ER) or archaeal bacterial signal peptidase in that it has a histidine residue replacing the conserved lysine general-base (see Section 2.3). This makes *Bacillus subtilis* the only known organism to contain in the same membrane signal peptidases with lysine and histidine general bases (Tjalsma et al., 1998). Using temperature-sensitive SipS variants, van Dijl and co-workers (Tjalsma et al., 1997) have been able to show that only strains of *Bacillus subtilis* lacking both SipS and SipT are nonviable. They proposed that the functionally redundant signal peptidases may provide a growth advantage for this gram-positive eubacterium, which secretes large amounts of protein. Two different Type I signal peptidases have been cloned and sequenced from the related species *Bacillus amyloliquefaciens* (Hoang & Hofemeister, 1995).

Two of the most notoriously antibiotic-resistant pathogenic “superbugs” are the gram-positive species *Streptococcus pneumoniae* and *Staphylococcus aureus*. Considerable resources worldwide are being expended to develop new families of antibiotics against these clinical pathogens (Neu, 1992). The Type I signal peptidase from *Streptococcus pneumoniae* (spi) has been cloned and overexpressed in *E. coli*. It was shown to be essential for the viability of *Streptococcus pneumoniae*. Introduction of the spi gene into the previously described temperature-sensitive *E. coli* strain (IT41) allowed normal growth rates at elevated temperatures, indicating that spi has signal peptidase I activity (Zhang et al., 1997). The Type I signal peptidase from *Staphylococcus aureus* (spsB) has also been cloned, expressed in *E. coli*, and shown to cleave *E. coli* preproteins in vivo (Cregg et al., 1996). Use of a temperature-sensitive plasmid has revealed that spsB is essential and contains the only Type I signal peptidase activity in *Staphylococcus aureus* (Cregg et al., 1996). *Staphylococcus aureus* contains a second open reading frame (spsA), just 15 nucleotides upstream of spsB, that has 62% similarity and 31% identity to spsB. Interestingly, sequence alignments indicate that the spsA gene encodes a protein that does not contain the essential active-site Ser and Lys (Cregg et al., 1996). Newly sequenced bacterial genomes indicate that there are likely more examples of these proteins with high sequence similarity to the Type I signal peptidases, except for the catalytic residues. For example, *Borrelia burgdorferi*, the Lyme disease spirochete, and *Staphylococcus carnosus* (sipA) contain such genes (Fig. 2). It is not clear what function these proteins play in the bacterial cell.

Recently, four genes encoding distinct Type I signal peptidases have been cloned from *Streptomyces lividans* TK21 (Schacht et al., 1998; Parro & Mellado, 1998; Parro et al.,

2.2. Type I signal peptidases from gram-positive bacteria  

As in the gram-negative species, Type I signal peptidases from gram-positive bacteria appear to be monomeric enzymes containing an amino-terminal transmembrane anchor and a carboxy-terminal catalytic domain. However, typically, the gram-positives have only a single transmembrane segment and a short amino-terminal cytoplasmic tail. The catalytic domain is also significantly smaller in size compared with the gram-negative species (Fig. 2).

In gram-positive bacteria, the occurrence of multiple Type I signal peptidases within a single species is very common. By far, the most thoroughly characterized gram-positive Type I signal peptidases are those from *Bacillus subtilis*. *Bacillus subtilis* contains seven Type I signal peptidases (five chromosomal [SipS, SipT, SipU, SipV, and SipW] and

1999). All four genes (sipW, sipX, sipY, and sipZ) are clustered such that three of them constitute an operon and the fourth is the first gene of another operon. Three of the four (sipX, sipY, and sipZ) have been reported to contain a putative carboxy-terminal transmembrane segment in addition to the amino-terminal transmembrane segment. Using the program HMMTOP (Tusnady & Simon, 1998), we detected a putative carboxy-terminal transmembrane segment for only SipY and SipZ (Fig. 2). Preliminary studies suggest that all four genes may be essential for cell viability (Parro et al., 1999). Only when the four genes are expressed together in *E. coli* IT41 are they able to compensate for the temperature-sensitive *E. coli* signal peptidase mutation (Parro et al., 1999). *Streptomyces lividans* Type I signal peptidases will be unique amongst bacterial Type I signal peptidases if further experimental analysis confirms that all four of the sip genes are required for Type I signal peptidase activity.

### 2.3. Type I signal peptidases from higher order species

The Type I signal peptidases from the eukaryotic ER belong to the serine protease family S27 and possess a conserved histidine in place of the proposed lysine general base found in the bacterial Type I signal peptidases of the family S26 (Rawlings & Barrett, 1994). Unlike the “stand-alone” bacterial Type I signal peptidases, the active sites of the eukaryotic Type I signal peptidases are sequestered inside the lumen of the ER, and exist as a hetero-oligomeric membrane protein complex known as the signal peptidase complex (SPC). For example, in the yeast *Saccharomyces cerevisiae*, the SPC is composed of four proteins (Sec11, SPC1, SPC2, and SPC3) (YaDeau & Blobel, 1989). Both Sec11 and SPC3 are required for optimal signal peptidase activity, but only Sec11 contains the conserved Ser/His catalytic residues (Fang et al., 1996, 1997; Meyer & Hartmann, 1997). It has been shown that SPC1 and SPC2 are nonessential for Type I signal peptidase activity (Mullins et al., 1996). The mammalian SPC consists of five polypeptide chains (SPC18, SPC21, SPC12, SPC25, and SPC22/23) (Evans et al., 1986; Greenburg & Blobel, 1994; Kalies & Hartman, 1996; Lively et al., 1994). The proteins SPC18 and SPC21 are homologs to the yeast Sec11 protein and have active-site residues. The function of the proteins SPC 12, 25, and 22/23 as of yet are unknown. The proteins SPC18, 21, and 22/23 are single transmembrane proteins that are exposed to the luminal side of the ER (Shelness et al., 1993), whereas the SPC12 and SPC25 span the membrane twice, with the majority of the protein facing the cytosol rather than the ER lumen, suggesting that they may play some role other than catalytic (Kalies & Hartmann, 1996). In contrast, based on sequence similarity and substrate specificity, the mitochondrial inner membrane peptidases (Imp1 and Imp2; Behrens et al., 1991; Nunnari et al., 1993; Schneider et al., 1994) and the thylakoidal-processing peptidase (Halpin et al., 1989; Chaal et al., 1998) appear to be more related to the bacterial

Type I signal peptidases. This is consistent with the endosymbiont hypothesis, which states that the mitochondria and chloroplast organelles evolved from bacteria (Cavalier-Smith, 1987).

#### 2.4. Regions of sequence similarity and identity (boxes B–E)

Using standard multiple-sequence alignment algorithms, it is possible to identify five regions of high-sequence similarity and identity in Type I signal peptidases from bacteria to human. These regions of sequence are referred to as boxes A–E (Dalbey et al., 1997). Box A corresponds to the transmembrane segment, and boxes B–E all reside in the carboxy-terminal catalytic domain. Fig. 2 shows the alignment of the conserved regions of sequence (boxes B–E) for the Type I signal peptidase sequences and putative signal peptidase sequences presently available. Box B (residues 88–95, *E. coli* numbering) contains the nucleophilic Ser90 and a conserved Met91. Box C contains residues 127–134, and box D (residues 142–153) contains the proposed general-base Lys145 and a conserved Arg146. Box E (residues 272–282) contains the highly conserved Gly272, Asp273, and Asn274, as well as Asp280 and Arg282. The precise structural details of the conserved boxes will be discussed in Section 4.4 in light of the recently solved crystal structure of the *E. coli* Type I signal peptidase. It has revealed that boxes B–E all lie near the signal peptidase active site and that they contribute to what is predicted to be a conserved catalytic Type I signal peptidase protein fold.

### 3. Mechanistic studies of Type I signal peptidases

#### 3.1. The signal peptide

Although signal peptides do not show a great deal of sequence homology, they do contain three recognizable domains. They have a positively charged amino-terminus (n-region, 1–5 residues in length), a central hydrophobic domain (h-region, 7–15 residues in length), and a neutral, but polar, C-terminal domain (c-region, 3–7 residues in length) (Fig. 4). The n-region and h-region are required for efficient translocation, whereas the c-region specifies the signal peptide cleavage site (von Heijne, 1990). The signal peptides from gram-positive bacteria are significantly longer than those from other organisms, and they have a much longer h-region (von Heijne & Abrahmsen, 1989). The average eukaryotic signal peptide is 22.6 amino acids in length, the average gram-negative signal peptide is 25.1 amino acids in length, and the average gram-positive signal peptide contains 32.0 amino acids (Nielsen et al., 1997a, 1997b). The gram-positive signal peptides also in general have an amino-terminus containing more positively charged lysine and arginine residues (Edman et al., 1999). The eukaryotic signal peptides have fewer lysine residues in their n-region, and have a shorter h-region dominated by the presence of leucine residues (Nielsen et al., 1997a, 1997b). Statistical analysis of the amino acid sequences surrounding signal peptide cleav-

M. Paetzel et al. / Pharmacology & Therapeutics 87 (2000) 27–49

| n-region | h-region | c-region | Mature protein |
| --- | --- | --- | --- |
| Positively charged<br>1-5 residues | Hydrophobic<br>$\alpha$-helical<br>7-15 residues | Extended<br>$\beta$-conformation (-1 to -4)<br>3-7 residues |  |

Cleavage site

-6<br>
(P6)<br>
P/G

-3<br>
(P3)<br>
Ala

-1<br>
(P1)<br>
Ala

Fig. 4. The features of a typical bacterial signal peptide. Signal peptides have a positively charged amino-terminus (n-region), a hydrophobic central region (h-region), and a neutral, but polar, carboxy-terminus (c-region). The boundary between the h-region and the c-region is usually marked by a helix-breaking residue (Pro or Gly) at the $-6$ (P6) position relative to the cleavage site. The cleavage recognition sequence consists of small residues at the $-1$ (P1) and $-3$ (P3) positions relative to the cleavage site. By far, the most common residue at these positions is Ala.

age sites has led to the so-called ($-3$, $-1$) rule that states that the residues at the $-3$ and $-1$ positions relative to the cleavage site must be small and neutral for cleavage to occur (von Heijne, 1983, 1985; Perlman & Halvorson, 1983). Note that the $-3$ and $-1$ residues are synonymous with the P3 and P1 residues in the Schechter and Berger (1967) nomenclature. The residues at the $-3$ and $-1$ positions are most commonly Ala; therefore, the signal peptide cleavage-site specificity is sometimes called the (Ala-X-Ala) rule. The eukaryotic signal peptides show a less stringent requirement for Ala at the $-3$ and $-1$ positions, accepting Gly and Ser almost as frequently as Ala at the $-1$ position and Val, Ser, and Thr almost as frequently as Ala at the $-3$ position (Nielsen et al., 1997a, 1997b).

In addition to these statistical analyses, the substrate specificity or preference of Type I signal peptidase has been studied extensively in vitro and in vivo (Watts et al., 1983; Koshland et al., 1982; Dierstein & Wickner, 1986; Folz et al., 1988; Nothwehr & Gordon, 1989, 1990; Fikes et al., 1990; Kuhn & Wickner, 1985; Shen et al., 1991; Ryan & Edwards, 1995; Pratap & Dikshit, 1998; Karamyshev et al., 1998; Wrede et al., 1998). Using synthetic peptides based on the signal peptide cleavage site of pre-maltose-binding protein (MBP), Dev and co-workers (1990) found that the minimal segment of preprotein that can be cleaved by *E. coli* Type I signal peptidase is the $-3$ to $+2$ region ($-3$-Ala-Leu-Ala-/Lys-Ile-$+2$). Using site-directed mutagenesis and both in vitro and in vivo Type I signal peptidases, Shen et al. (1991) investigated the limits of sequence variation tolerated for the processing of the M13 procoat protein by the *E. coli* Type I signal peptidase. They found that almost any residue is allowed at the $+1$, $-2$, $-4$, and $-5$ positions, whereas only the residues Ala, Ser, Gly, or Pro at the $-1$ position and Ser, Gly, Thr, Val, or Leu at the $-3$ position allowed processing of the M13 procoat protein. A recent mutagenesis study using the *E. coli* alkaline phosphatase preprotein revealed that large amino acids at the $-2$ position and middle-sized residues at the $-5$ position allowed for the most efficient processing by the *E. coli* signal peptidase (Karamyshev et al., 1998). These results are consistent with the statistical analysis of signal peptide cleavage-site sequences (Nielsen et al., 1997a, 1997b). The $-6$

position of the signal peptide is most often occupied by a Pro, Gly, or Ser residue, and it has been suggested that this position defines the transition from the h-region to the c-region (von Heijne, 1990). Shen et al. (1991) found that the M13 procoat will only be processed efficiently if there is a proline at the $-6$ position. Conversely, a proline residue at the $+1$ position relative to the cleavage site in pre-MBP prevents cleavage of the signal peptide; this construct was also shown to be an effective inhibitor of *E. coli* Type I signal peptidase (Barkocy-Gallagher & Bassford, 1992). It has been shown that a $+1$ proline has a similar effect in both Sec-dependent and Sec-independent preproteins (Nilsson & von Heijne, 1992). Prolines at the $+2$ or $+3$ position relative to the cleavage site have also been shown to negatively affect the efficiency of signal peptide cleavage (Hegner et al., 1992).

A number of studies have focused on the structure, orientation, and interactions of signal peptides in lipid bilayers or membrane mimetic environments (Cornell et al., 1989; Bruch et al., 1989; Jones et al., 1990; McKnight et al., 1991a, 1991b; Wang et al., 1993; Rizo et al., 1993; Jones & Gierasch, 1994; Bechinger et al., 1996; Keller et al., 1996; Voglino et al., 1998, 1999). Most studies to date are consistent with the central h-region adopting an $\alpha$-helical conformation when in a lipid or hydrophobic environment (McKnight et al., 1989; Chupin et al., 1995; Voglino et al., 1998; Batenburg et al., 1988; Rizo et al., 1993; Wang et al., 1993). The boundary between the h-region and the c-region ($-6$ to $-4$), which often contains proline or glycine residues, has been suggested to have a $\beta$-turn structure (Rosenblatt et al., 1980; Perlman & Halvorson, 1983). The recent conformational, statistical, and mutational analysis by Karamyshev and co-workers (1998) is consistent with the signal peptide having an extended $\beta$-conformation in the $-5$ to $-1$ region, while bound to the signal peptidase-binding pocket. In addition, NMR (Killian et al., 1990) and ESR (Sankaram et al., 1994) experiments provide evidence for signal peptides inducing nonbilayer lipid structure (de Vrije et al., 1990). This may be a possible explanation for the processing of signal peptides with rather short h-regions. To date, there is no direct experimental evidence to confirm whether the bacterial signal peptide is associated with lipid

and/or the proteins of the secretion machinery (translocase) during the cleavage event. It is also presently unclear whether bacterial signal peptidases cleave the signal peptide during or after translocation.

There are a number of programs available for the identification of signal peptides and the prediction of their cleavage sites. The weighted matrix method of identifying signal peptides and predicting the location of the signal peptidase cleavage sites is commonly used (von Heijne, 1986). Recently, a neural network approach to identifying signal peptides and predicting their cleavage sites has also been developed (Nielsen et al., 1997a, 1997b). The network ensembles were trained on separate data sets from gram-positive, gram-negative, and eukaryotic organisms. This program (SignalP) shows a significant improvement in performance over the weighted matrix method and is publicly available (http://www.cbs.dtu.dk/services/SignalP/). The SignalP program has been further improved by including a hidden Markov model, making it possible to discriminate between cleaved signal peptides and uncleaved signal anchors that mediate the targeting to and translocation of membrane proteins across the cytoplasmic membrane (Nielsen et al., 1999).

### 3.2. Type I signal peptidase assays

Development of Type I signal peptidase assays was instrumental in the initial purification and characterization of signal peptidase activity (Chang et al., 1978; Zwizinski & Wickner, 1980; Jackson, 1983). Some of these initial assays, as well as the more recently developed signal peptidase assays, are listed in Table 2. It has been observed that the in vivo assay can be a more sensitive monitor of Type I signal peptidase activity than the available in vitro assays (Sung & Dalbey, 1992), and caution should be used when interpreting the in vitro results.

Although a number of synthetic peptide assays have been developed that were helpful in determining the Type I signal peptidase substrate specificity (Caulfield et al., 1988, 1989; Dev et al., 1990), they tend to be rather poor substrates in comparison with full-length preprotein substrates. The Type I signal peptidase substrate that gives the best catalytic constants to date is the pro-OmpA nuclease A (see Tables 2 and 3 in Chatterjee et al., 1995). It is a hybrid protein of the *Staphylococcus aureus* nuclease A attached to the signal peptide of the *E. coli* outer membrane protein A (OmpA). Using this substrate, Suciu et al. (1997) have estimated the activation energy of *E. coli* signal peptidase to be 10.4 ± 0.6 kcal/mol, which indicates that it is catalytically as efficient as the traditional Ser/His/Asp serine proteases. Also using the pro-OmpA nuclease A substrate, Paetzel and co-workers (1997) measured the pH-rate profile of *E. coli* signal peptidase, revealing a maximum efficiency at pH 9.0 and apparent pKa values for titratable groups at ~8.7 and 9.3. Using the pro-OmpA nuclease A substrate at pH 9.0 and in the presence of the detergent Triton X-100, the purified wild-type *E. coli* signal peptidase gave the catalytic constants \( k_{\text{cat}} = 120.0 \pm 10.7 \), \( K_{\text{m}} = 10.9 \pm 2.8 \, \text{mM} \), \( k_{\text{cat}} / K_{\text{m}} = 1.1 (\pm 0.2) \times 10^7 \, \text{sec}^{-1} \text{M}^{-1} \). Although an excellent substrate, this assay requires sodium dodecyl sulfate-poly-

**Table 2**

Type I signal peptidase assays

| Assay | Analysis method | Reference |
| --- | --- | --- |
| In vivo | Pro-OmpA expressed in the temperature-sensitive *E. coli* strain IT41 | SDS-PAGE/immunoprecipitation/fluorography | Bilgin et al., 1990; Sung and Dalbey, 1992; Tschantz et al., 1993 |
| Complementation of temperature-sensitive *E. coli* strain IT41 | Cell growth | Inada et al., 1989; Black et al., 1992; Black, 1993; Kim et al., 1995b |
| Pre(A13i)-β-lactamase plate assay | β-Lactamase activity | Van Dijl et al., 1992; Tjalsma et al., 1998; Meijer et al., 1995 |
| Radiolabeled hybrid precursor pre(A13I)-β-lactamase specified by plasmid pGDL48. [Note: *Bacillus subtilis* signal peptidases will cleave pre(A13i)-β-lactamase, but *E. coli* signal peptidases will not] | SDS-PAGE/immunoprecipitation/fluorography | Tjalsma et al., 1997 |
| In vitro | Radiolabeled preproteins synthesized by in vitro transcription translation | SDS-PAGE/immunoprecipitation/fluorography | Zwizinski and Wickner, 1980; Zwizinski et al., 1981; Sung and Dalbey, 1992; Tschantz et al., 1993; Vehmaanpera et al., 1993 |
| Synthetic peptide | 125I label | Caulfield et al., 1988, 1989 |
| HPLC-UV detector | Dev et al., 1990; Kim et al., 1995a,b; Kuo et al., 1993, 1994 |
| Continuous spectrophotometric or spectrofluorometric | Kuo et al., 1994 |
| FRET-continuous spectrofluorometric | Zhong and Benkovic, 1998 |
| Pro-OmpA nuclease A fusion protein | SDS-PAGE/densitometry | Chatterjee et al., 1995; Suciu et al., 1997; Paetzel et al., 1997 |

acrylamide gel electrophoresis (SDS-PAGE) densitometry for its kinetic analysis (Table 2), and thus is not practical for the high-throughput screening of inhibitor libraries that is now commonly used for the elucidation of new lead compounds.

In an attempt to design a signal peptidase substrate suitable for high-throughput inhibitor screening, a direct continuous signal peptidase assay based on fluorescence resonance energy transfer (FRET) has been developed by Zhong and Benkovic (1998). The peptide substrate (YNO₂-F-S-A-S-A-L-A-K-I-K-Abz) incorporates the signal peptidase cleavage site of the MBP signal peptide. The fluorescence of the anthraniloyl group (Abz) is quenched by the 3-nitrotyrosine until the peptide is cleaved by signal peptidase between the residues A and K. The \( k_{\text{cat}}/K_{\text{m}} \) for *E. coli* signal peptidase measured with this substrate was \( 71.1 \, \mathrm{M}^{-1} \mathrm{sec}^{-1} \), four orders of magnitude lower than that achieved with the pro-OmpA nuclease A substrate (Table 3). Introduction of the hydrophobic h-region of the signal peptide into the Benkovic substrate may provide a more sensitive continuous assay that is required for high-throughput inhibitor screening. The recently acquired knowledge of the three-dimensional structure of Type I signal peptidases will also likely aid in the structure-based design of chromophoric or fluorogenic substrates.

### 3.3. Site-directed mutagenesis and site-directed chemical modification studies

The catalytic mechanism of bacterial signal peptidases has been studied by site-directed mutagenesis in the *E. coli* signal peptidase and the *Bacillus subtilis* SipS. Deletion studies showed that the catalytic activity of signal peptidase resides in the large carboxy-terminal periplasmic domain (Bilgin et al., 1990; Kuo et al., 1993; Tschantz et al., 1995). Sung and Dalbey (1992) have used site-directed mutagenesis to classify *E. coli* signal peptidase into a protease class, and found neither cysteine nor histidine residues to be essential for cleaving the bacteriophage M13 procoat protein. They found Ser90 to be critical for activity. Later work revealed that Lys145 is the only basic residue that is essential for *E. coli* signal peptidase activity, and suggested that it is involved in a Ser/Lys dyad mechanism (Tschantz et al., 1993; Black, 1993). Site-directed mutagenesis and chemical modification studies are consistent with Trp300 being important for efficient processing of the signal peptide (Kim et al., 1995a, 1995b). In parallel, extensive site-directed mutagenesis studies of the gram-positive signal peptidase *Bacillus subtilis* SipS has been performed by van Dijl and co-workers (1995). They found that the residues Ser43, Lys83, and Asp153 (the equivalent residues to *E. coli* Ser90, Lys145, and Asp280) were all essential for SipS activity.

To further explore the role of the proposed active-site residue Ser90 of *E. coli* Type I signal peptidase, Tschantz and co-workers (1993) constructed an active thiol signal peptidase by mutating the Ser90 to Cys. This variant was inhibited by the cysteine-specific reagent N-ethylmaleimide, whereas wild-type signal peptidase was not affected. Paetzel and co-workers (1997) combined site-directed mutagenesis and chemical modification to introduce unnatural amino acids at the 145 position. They changed Lys145 to Cys, which produced an inactive enzyme, and then showed that the active enzyme could be regenerated by reacting the Cys with 2-bromoethylamine-HBr to produce the lysine analog (γ-thia-lysine) at the 145 position. Modification of the Cys145 with (2-bromoethyl)trimethylammonium-HBr to form a positively charged nonitratable side chain at the 145 position failed to restore activity to the inactive Lys145Cys mutant. These observations are consistent with the essential Lys145 playing a role as the active-site general base.

### 3.4. Inhibitors of bacterial Type I signal peptidases

It has been observed in many laboratories that bacterial Type I signal peptidases are not inhibited by the standard protease inhibitors (Zwizinski et al., 1981; Talarico et al., 1991; Black et al., 1992; Kuo et al., 1993; Kim et al., 1995a). For example, Kim et al. (1995a) have found that *E. coli* signal peptidase is not inhibited by diisopropyl fluorophosphate (DFP), phenylmethylsulphonyl fluoride (PMSF), soybean trypsin inhibitor, APMSF, elastal, benzamidine, leupeptin, tosyl-lysine chloromethyl ketone, *n*-tosylphenylalanine chloromethyl ketone, chymostatin, *p*-chloromercuribenzoate, EDTA, *O*-phenanthroline, phosphoramidon, and bestatin. They also found that *E. coli* signal peptidase was not labeled by [³H]DFP, even at very high concentrations (20 mM), and that the activity of *E. coli* signal peptidase is adversely affected by high concentration of sodium chloride.

---

**Table 3**

Kinetic constants for *Escherichia coli* signal peptidase measured with various substrates

| Substrate               | Method of analysis | Reaction conditions | \( k_{\text{cat}} \) (\( \mathrm{sec}^{-1} \)) | \( K_{\text{m}} \) (\( \mathrm{M}^{-1} \)) | \( k_{\text{cat}}/K_{\text{m}} \) (\( \mathrm{M}^{-1} \mathrm{sec}^{-1} \)) | Reference           |
|-------------------------|--------------------|---------------------|---------------------------------------------|------------------------------------------|----------------------------------------------------|----------------------|
| Peptide (12-mer, MBP)   | HPLC               | a                   | \( 1.25 \times 10^{-4} \)                   | \( 1.4 \times 10^{-3} \)                 | \( 8.9 \times 10^{-2} \)                          | Dev et al., 1990     |
| Peptide (9-mer, MBP)    | HPLC               | a                   | \( 3.2 \times 10^{-2} \)                   | \( 8 \times 10^{-4} \)                   | 40                                               | Dev et al., 1990     |
| ProOmpA-nuclease A      | SDS-PAGE           | b                   | 8.73                                       | \( 1.65 \times 10^{-5} \)                 | \( 5.29 \times 10^{5} \)                          | Chatterjee et al., 1995 |
| Peptide (11-mer, MBP)   | FRET-continuous    | d                   | —                                          | —                                        | 71                                               | Zhong and Benkovic, 1998 |

* a Sodium phosphate (pH 7.7, 25 mM), 1% Triton X-100, 37°C.
* b Tris-HCl (50 mM), 1% Triton X-100, pH 8.0, 37°C.
* c ProOmpA-nuclease A is a fusion protein made up of the signal peptide of outer membrane protein A from *E. coli* and the nuclease A from *Staphylococcus aureus*.
* d Triethanolamine-HCl (50 mM), 1 mM EDTA, 1% Triton X-100, pH 8.0, 37°C.

(>160 mM), magnesium chloride (>1 mM), and dinitrophenol (Zwizinski et al., 1981).

*Escherichia coli* signal peptidase can be inhibited in a competitive fashion by signal peptides. The 23 amino acid residue signal peptide from M13 procoat protein (MKKSLVLKASVAVATLVPMLSFA) was found to inhibit the in vitro processing of the procoat protein and pre-MBP by *E. coli* signal peptidase (Wickner et al., 1987). As mentioned in Section 3.1, a synthetic pre-MBP signal peptide with a proline at the +1 (P1′) position has been shown to inhibit *E. coli* signal peptidase activity in vivo (Barkocy-Gallagher & Bassford, 1992). The first report of an effective, nonpeptide, inhibitor of Type I signal peptidase was by Kuo and colleagues in 1994. They reported that β-lactam compounds could inhibit *E. coli* signal peptidase in a pH- and time-dependent manner. β-Lactam-type compounds have also been shown to be effective irreversible inhibitors of a number of serine proteases and hydrolases, such as elastase (Wilmouth et al., 1999; Taylor et al., 1999), phospholipase A₂ (Tew et al., 1998), and β-lactamase (Strynadka et al., 1992). In all these cases, the β-lactam bond mimics the peptide bond of the natural substrate. SmithKline Beecham Pharmaceuticals (Harlow, Essex, UK) has studied extensively the potential of β-lactam (or penem)-type inhibitors against bacterial Type I signal peptidase (Allsop et al., 1995, 1996, 1997; Perry et al., 1995; Black & Bruton, 1998). The most effective penem compounds are the 5S stereoisomers (Fig. 5a), which are capable of inhibiting both

![Figure 5](#fig-5)

Fig. 5. (a) The 5R and 5S stereoisomers of the β-lactam type (penem) ring system. The arrows show the direction of the order of priority at the C5 chiral center. The bacterial signal peptidases bind the 5S isomers, whereas the β-lactamases and penicillin-binding proteins bind the 5R isomers. (b) Structure of the β-lactam (penem) inhibitor allyl(5S,6S)-6-[(R)-acetoxyethyl]-penem-3-carboxylate (Allsop et al., 1997; Black & Bruton, 1998).

the gram-negative *E. coli* and the gram-positive *Staphylococcus aureus* signal peptidases. The 5S stereoisomer is opposite stereochemically to that of the 5R β-lactams that are required for β-lactamases and penicillin-binding proteins (Fig. 5a). The compound allyl (5S,6S)-6-[(R)-acetoxyethyl]-penem-3-carboxylate (Fig. 5b) has an IC₅₀ of less than 1 μM. A rationale for the face of nucleophilic attack by the *E. coli* signal peptidase Ser90 Oγ previously was proposed based on the stereochemistry of the penem inhibitors (Allsop et al., 1997). The X-ray structure of the *E. coli* signal peptidase–5S,6S penem complex confirmed the *si*-face attack of the nucleophile (Paetzel et al., 1998) (see Section 4.5). The penem inhibitors have also been reported to inhibit signal peptidases from cyanobacterium and the chloroplast thylakoids in vitro (Barbrook et al., 1996).

### 4. The three-dimensional structure of Type I signal peptidase

#### 4.1. X-ray crystal structure of a soluble, catalytically active fragment of Escherichia coli Type I signal peptidase

Crystallization of membrane proteins is a challenging, but increasingly feasible, endeavor (Song & Gouaux, 1997; Ostermeier & Michel, 1997). The recent plethora of integral membrane protein structures (Xia et al., 1997; Iwata et al., 1995; Ferguson et al., 1998; Buchanan et al., 1999; Doyle et al., 1998; Chang et al., 1998; Olson et al., 1999; Dutzler et al., 1999) has occurred in parallel with advances in molecular biology, an increase in the number of commercially available surfactants needed to mimic the membrane milieu, and the development of novel crystallographic phasing strategies, including multiple anomalous diffraction, where structures can be solved from the data collected using a single crystal (Hendrickson & Orgata, 1997). However, structure determination of membrane proteins still remains a relatively daunting pursuit. In particular, there have been no reported successes in crystallizing proteins such as native signal peptidase, which contains a single transmembrane anchor domain.

In an effort to obtain Type I signal peptidase structural information, a soluble form of *E. coli* signal peptidase was constructed, expressed, purified, characterized (Kuo et al., 1993; Tschantz et al., 1995), and crystallized (Paetzel et al., 1995). The construct lacks the residues 2–75, which correspond to the 2 transmembrane segments, and is referred to as Δ2–75. The protein has a molecular weight of 27,952 Da, as measured by electrospray ionization mass spectrometry (Kuo et al., 1993), and a measured isoelectric point of 5.6 (Tschantz et al., 1995). The presence of detergent was essential for optimal activity and crystallization, specifically the addition of Triton X-100 at approximately the critical micelle concentration (Paetzel et al., 1995). Native data to 2.3-Å resolution have been collected. However, addition of an inhibitor greatly enhanced the order of the signal peptidase crystals. The structure of Δ2–75 in complex with a β-lactam type inhibitor (allyl (5S,6S)-6-[(R)-acetoxyethyl]-

penem-3-carboxylate) (Fig. 5) was solved to high resolution (1.95 Å) by multiple isomorphous replacement (Paetzel et al., 1998).

#### 4.2. The protein fold

The overall dimension of *E. coli* signal peptidase Δ2–75 is ~60 × 40 × 70 Å, with the smallest dimension running perpendicular to the view seen in Fig. 6a. *Escherichia coli* Type I signal peptidase has a primarily β-sheet protein fold consisting of two anti-parallel β-sheet domains. Domain I, which we have termed the “catalytic core,” contains the conserved regions of sequence (Boxes B–E), all of which reside near or are part of the active site (see Figs. 6 and 7). The structure has four extended loops or hairpins that are disordered (residues 108–124, 170–176, 198–202, and 304–313).

Despite very limited sequence identity (van Dijl et al., 1995), Domain I shares significant structural similarities with the UmuD′ protease, the proteolytic domain of a self-cleaving repressor protein involved in the SOS DNA-repair response in *E. coli* (Paetzel & Strynadka, 1999) (Fig. 6c). It has also been proposed to be a Ser/Lys protease (Takehiko et al., 1988; Peat et al., 1996). The large β-sheet Domain II (residues 154–264) and the long extended β-ribbon (residues 107–122) observed in the *E. coli* signal peptidase structure are insertions within Domain I (see Figs. 6 and 7). Domain II contains a disulfide bond between Cys170 and Cys176 that previously was detected by biochemical studies (Whitley & von Heijne, 1993).

#### 4.3. The membrane association surface

The conserved Domain I also contains a large exposed hydrophobic surface that runs along the β-sheet made up from β-strands 1, 2, and 17. Exposure of significant areas of hydrophobic surface is rarely observed in soluble proteins, and it has been proposed that in signal peptidase, this unusual surface is likely to be involved in membrane association (Paetzel et al., 1998). β-Strand 17 contains the previously mentioned Trp300, which was shown to be essential for optimal activity in *E. coli* signal peptidase (Kim et al., 1995a, 1995b), even though the structure shows that it is localized more than 20Å from the enzyme active site (Paetzel et al., 1998). Aromatic amino acids are thought to play an important role in protein/membrane interfaces (Landolt-Marticorena et al., 1993), and presumably Trp300 facilitates the insertion or association of *E. coli* signal peptidase into the membrane. A tryptophan residue was found to be essential for the interfacial catalysis of phospholipase A₂ at the membrane surface (Gelb et al., 1999). Sequence alignments indicate that several conserved aromatic or hydrophobic residues exist in the proposed membrane-association domain in both gram-positive and gram-negative bacterial Type I signal peptidases (Fig. 7). Interestingly, *Bradyrhizobium japonicum* SipF contains a very high content of tryptophan residues near its carboxy-terminus (Bairl & Muller,

M. Paetzel et al. / Pharmacology & Therapeutics 87 (2000) 27–49

Fig. 6. (a) A stereo view of the overall protein fold of the *E. coli* type I signal peptidase catalytic domain (∆2–75). Domain I is in green and red. The red regions correspond to the conserved boxes (B–E) of sequence in the type I signal peptidases (see Fig. 7). Domain II and the extended β-ribbon are in gray. The β-strands are numbered in the order of appearance in the sequence (see Fig. 7). Note that two β-strands (β10 and β11), a small α-helix (α1), and small 3<sub>10</sub> helices (3<sub>10</sub> 1–3) are not shown for clarity of the figure. The figure was prepared with the program MOLSCRIPT (Kraulis, 1991). (b) A schematic diagram of the protein fold in the conserved Domain I of Type I signal peptidases. Domain II and the extended β-ribbon, as seen in the gram-negative Type I signal peptidases, are insertions within Domain I. (c) A stereo view of the C<sub>α</sub> backbone of the *E. coli* UmuD′ protease (dark blue) superimposed onto the C<sub>α</sub> backbone tracing of the catalytic domain of *E. coli* Type I signal peptidase (light blue). The side chains of the Ser/Lys dyad in each protease are shown in ball-and-stick. This figure was prepared with the program MOLSCRIPT (Kraulis, 1991).

M. Paetzel et al. / Pharmacology & Therapeutics 87 (2000) 27–49

$\beta 1$

$\begin{array}{cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc
M. Paetzel et al. / Pharmacology & Therapeutics 87 (2000) 27–49

### 4.4. The tertiary structure of boxes B–E

The structural motifs within which the four conserved boxes reside can be summarized as follows. Box B (residues 88–95) contains the proposed nucleophilic Ser90 that resides on a well-ordered loop between the first 2 β-strands of the periplasmic catalytic portion of *E. coli* Type I signal peptidase. Box B also contains a conserved serine at position 88 in the gram-negative species. There is a conserved Met at position 91, which allows for the placement of its side-chain sulfur adjacent to the nucleophilic serine. This is a common feature observed in the active site of several Ser proteases and β-lactamases. Boxes C (residues 127–134) and D (residues 142–153) contain two antiparallel hydrogen bonded β-strands (β5 and β6; see Figs. 6 and 7). β-Strand 6 contains the proposed general base Lys145 and has main-chain atoms aligned perfectly to make β-strand-type hydrogen bonds with the signal peptide substrate, as seen in other serine proteases (Perona & Craik, 1995). Box E (residues 272–282) contains a small 3<sub>10</sub> helix (residues 275–278) and part of a small α-helix (residues 280–282). The strict conservation of Gly272 is explained by the fact that it sits directly adjacent to the Nζ of Lys145 (Fig. 8). Any other residue at this site would necessarily position a side chain into this region that would clash with the essential Lys145 side chain. The conserved Ser278, which hydrogen bonds to the Lys145, most likely helps position the Lys145 Nζ for its general base role with Ser90. The highly conserved Asp280 and Arg282, which are positioned more than 5 Å from the catalytic residues (Ser90 or Lys145), form a strong salt-bridge (Fig. 8), and likely contribute to the structural stabilization of the active site rather than playing a direct role in catalysis.

(Paetzel et al., 1998) was facilitated by the addition of a covalently bound inhibitor. The electron density at the active site of the *E. coli* signal peptidase inhibitor complex revealed a covalent bond between the Ser90 Oγ and the carbonyl (C7) of the inhibitor (Fig. 8). This provided the first direct evidence that the Ser90 Oγ acts as the acylating nucleophile in the signal peptidase hydrolysis reaction. The signal peptidase inhibitor complex shows that Type I signal peptidases indeed are unique serine proteases, in that they attack the scissile bond of the substrate from the *si*-face of the amide bond. All known Ser/His/Asp serine proteases attack the scissile bond from the *re*-face (James, 1994) (see Fig. 9). The observation likely explains the preference for the 5*S* versus the 5*R* stereochemistry of the penem inhibitor, and supports the earlier proposals of Allsop and co-workers (1997).

The Nζ of Lys145 forms a strong hydrogen bond (2.9 Å) to the Ser90 Oγ, and is the only titratable group in the vicinity of the active-site nucleophile (Fig. 8). The Lys145 Nζ position is fixed relative to Ser90 Oγ by hydrogen bonds to the conserved Ser278 Oγ (2.9 Å) and the carbonyl oxygen (O10) of the inhibitor side chain (2.9 Å) (Fig. 8). Therefore, Lys145 Nζ is correctly positioned to act as the general base in both the acylation step and the deacylation step of catalysis. The side chain of Lys145 is completely buried in the inhibitor complex (Fig. 8), where it makes van der Waals contacts with the side-chain atoms of Tyr143, Phe133, Met270 and the main-chain atoms of Met270, Met271, Gly272, and Ala279, all of which are contained within the catalytic core (Domain I). The hydrophobic environment surrounding the Lys145 ε-amino group is likely essential for lowering its p*K*<sub>a</sub>, such that it can reside in the deprotonated state required for its role as the general base (Paetzel et al., 1997; Paetzel & Dalbey, 1997).

The main-chain amide of Ser90 forms a strong hydrogen bond to the carbonyl of the acylated inhibitor. This suggests that the Ser90 amide may contribute to the formation of the

---

### 4.5. The catalytic residues

As mentioned in Section 4.1, the ability to obtain high-resolution structural data for the *E. coli* signal peptidase

---

**Fig. 8.** A stereo view of the active-site of the *E. coli* Type I signal peptidase. Shown in thick black lines, the β-lactam-type inhibitor allyl (5S,6S)-6-[(R)-acetoxyethyl]-penem-3-carboxylate is covalently bound to the Oγ of Ser90. The carbonyl oxygen (O8) of the cleaved β-lactam (the bond between C7 and N4 has been broken) is sitting in the oxyanion hole formed by the main-chain NH of Ser90 (S90). The methyl group (C16) of the inhibitor sits in the S1 substrate-binding site. The Lys145 Nζ is hydrogen bonded to the Ser90 Oγ, Ser278 Oγ, and atom O10 of the penem inhibitor.

M. Paetzel et al. / Pharmacology & Therapeutics 87 (2000) 27–49

**si-face attack**
Ser/Lys proteases  
e.g. signal peptidase  

**re-face attack**
Ser/His/Asp proteases  
e.g. chymotrypsin  

Fig. 9. The Ser Oγ nucleophile of bacterial Type I signal peptidases attacks the scissile bond of its substrate from the *si*-face rather than the *re*-face, as is seen with all known Ser/His/Asp catalytic triad serine proteases such as chymotrypsin.

oxyanion hole (Fig. 8). A second contributor to the oxyanion hole is not obvious, but it is proposed that the Ser88 Oγ potentially could contribute to the oxyanion hole in the case of the natural preprotein substrate by a simple rotation of its χ₁ angle. The rotation is prevented in the β-lactam inhibitor complex structure by unfavorable van der Waals contacts between the Ser88 Oγ and the S1 and C15 atoms of the inhibitor (Fig. 8).

### 4.6. The substrate-binding sites and substrate specificity

The 5S,6S penem inhibitor that was co-crystallized with *E. coli* Type I signal peptidase has a methyl group (C16) that was found to be essential for effective inhibition (Allsop et al., 1995, 1996, 1997; Black & Bruton, 1998) (see Fig. 5). It was proposed that the C16 methyl group likely mimics the −1 (P1) alanine side chain of the signal peptide. The crystal structure of the complex reveals that the C16 methyl group sits in a small hydrophobic cleft, which can be recognized as the S1-binding pocket of the *E. coli* signal peptidase. The residues contributing atoms to the S1 pocket are primarily hydrophobic and include Ile86, Pro87, Ser88, Ser90, Met91, Leu95, Ile144, Tyr143, and Lys145 (see Figs. 8 and 10). The mapping of surface clefts on the signal peptidase structure has been performed using the algorithm CAST (Liang et al., 1998). This analysis indicates that the largest surface cleft corresponds to the proposed S1-binding site. The second most prominent cleft (the S3-binding site) was also easily recognized, both visually and by using the automated algorithm. It is also primarily hydrophobic in nature and sits on the other side of the S1 cleft relative to the active-site nucleophilic Ser90.

Modeling of a tetra-poly-alanine peptide into the signal peptidase active site also supported the identification of the S3 substrate-binding pocket. The position of the inhibitor methyl group (C16) in the S1 substrate-binding site, as well as the inhibitor carbonyl group (C7, O8), in the oxyanion hole was used as a guide. An extended β-strand conformation of the modeled peptide was needed in order to provide

a favorable fit and β-sheet-type hydrogen bonding with the conserved β-strand containing Lys145 (β-strand 6; see Figs. 6a and 10) (Paetzel et al., 1998). These modeling studies supported earlier work that suggested that the carboxy-terminal 5-6 residues of the signal peptide would adopt a β-strand type conformation (Izard & Kendall, 1994) (Fig. 10). The β-sheet type of interaction at the binding site is common amongst Ser proteases (Perona & Craik, 1995). In the model, the side chain of the P3 (−3) Ala points into a shallow hydrophobic depression formed by Phe84, Ile86, Ile101, Val132, Ile144, and the Cβ of Asp142 (the proposed S3 substrate specificity site; Fig. 10). Although Ala is the most common residue at the P3 (−3) site of signal peptides, larger aliphatic residues, such as Val, Leu, and Ile, can also occur at this position (Nielsen et al., 1997a, 1997b). The hydrophobic depression for the S3 site is more shallow and broad as compared with the S1 site. This may help to accommodate the larger residues at the P3 site of the signal peptide substrate.

In the penem-signal peptidase structure, the thiazolidine ring of the penem inhibitor (Fig. 8) makes a stacking interaction with the aromatic side chain of Tyr143. Modeling studies show that the side chains of the residues at P2 and P4 point out of the active site towards the solvent (Fig. 10), which is consistent with the observed signal peptide sequence variability at these positions (von Heijne, 1985) (Fig. 10). In the model, the side chain of Tyr143 would be

adjacent to the P2 position in the signal peptide (Paetzel et al., 1998). Van der Waals interactions between the substrate and the enzyme at this position may be beneficial to the binding, given that the recent mutagenesis studies of the *E. coli* alkaline phosphatase signal peptide have shown that variants with large residues at the P2 position are processed more efficiently (Karamyshev et al., 1998). Interestingly, most bacterial signal peptides have an aromatic residue at the 143 position (*E. coli* numbering), yet the eukaryotic signal peptidases have an Ile at this position (Figs. 2 and 7). The analysis of temperature-sensitive mutants identified in *Bacillus subtilis* SipS lends experimental support for the importance of the Tyr143 side chain. The mutants Leu74Ala and Tyr81Ala (the equivalent to Phe133 and Tyr143 in *E. coli* signal peptidase) are structurally stable, but catalytically impaired, at 48°C (Bolhuis et al., 1999). Collectively, this model explains the Ala-X-Ala (or −3, −1 rule) cleavage-site specificity of signal peptidases.

### 4.7. Predicted structural similarities and differences in Type I signal peptidases

A structure-based multiple sequence alignment of representative Type I signal peptidases reveal that Domain I is conserved throughout evolution (Fig. 7). A stereo view of Domain I can be seen in Fig. 6a (Domain I is in green and red), and a diagram of the protein fold of Domain I is seen in Fig. 6b. In general, the smaller size of the gram-positive Type I signal peptidases relative to the gram-negative Type I signal peptidases is a result of a smaller Domain II (154–264) and lack of an extended β-ribbon (residues 107–122). The eukaryotic ER and archaeal bacterial Type I signal peptidases are completely missing Domain II. The signal peptidase from yeast, Sec11, has been shown to require a second component, SPC3, for catalytic activity. The SPC3 has some sequence similarity to the Domain II of *E. coli* signal peptidase (VanValkenburgh et al., 1999) and, therefore,

**Fig. 11.** The proposed mechanism of *E. coli* Type I signal peptidase. To date, all data are consistent with bacterial signal peptidases utilizing a Ser/Lys dyad mechanism. (a) The Nζ of the lysine general base (Lys145) abstracts the proton from the Oγ of the nucleophilic serine (Ser90). The activated Ser90 Oγ attacks the *si*-face of the scissile bond of the preprotein forming the oxyanion tetrahedral intermediate I. (b) The oxyanion tetrahedral intermediate I is stabilized by the oxyanion hole formed from the main-chain amide of Ser90 and the side-chain hydroxyl of Ser88. (c) The acyl-enzyme intermediate is attacked by a deacylating water made nucleophilic by its association with the Lys145 Nζ general base. (d) The tetrahedral intermediate II then collapses, releasing the cleaved signal peptide.

SPC3 may provide the same function as Domain II. The functional roles played by Domain II and the extended β-ribbon of gram-negative Type I signal peptidases are unknown. Given the position of the β-ribbon in the *E. coli* signal peptidase, it may be speculated that it could play a role in membrane association.

It has been proposed that the *E. coli* signal peptidase Ser88 Oγ contributes to the oxyanion hole (Paetzel et al., 1998). From the multiple sequence alignment, it appears that the 88 position is most often occupied by a Ser, Thr, or Gly residue (Figs. 2 and 7). The gram-negative bacteria, archaea, and ER signal peptidases usually contain a Ser or Thr, whereas the gram-positives tend to have a Gly at this position. The gram-positive Type I signal peptidases may utilize the main chain NH of Gly as a contributor to the oxyanion transition state stabilization.

### 4.8. A proposed mechanism for bacterial Type I signal peptidase

Based on the available biochemical data, the structure of the *E. coli* Type I signal peptidase inhibitor complex, and modeling of peptide substrates, we can propose a catalytic mechanism for the cleavage of signal peptides by *E. coli* Type I signal peptidase (Fig. 11). The signal peptide binds to the active site of the signal peptidase with the P1 (−1) to P4 (−4) residues of the signal peptide in an extended β-conformation, such that their main-chain hydrogen bonds with the conserved box D β-strand, which contains the Lys145 general base (β6; Fig. 10). The P1 (−1) and P3 (−3) side chains sit in the hydrophobic S1 and S3 substrate-binding clefts (Fig. 10). The Lys145 Nζ would act as the general base to abstract the proton from the Ser90 Oγ that performs a nucleophilic attack on the *si*-face of the scissile bond (Figs. 9 and 11a). The oxyanion tetrahedral intermediate I is then formed with electrostatic stabilization by hydrogen bonds to the oxyanion hole (the Ser90 main-chain amide and the Ser88 side-chain hydroxyl) (Fig. 11b). The ammonium group from the side chain of Lys145 would then donate a proton to the leaving group amide, the new amino-terminus of the now mature exported protein (Fig. 11b), forming the acyl-enzyme intermediate (Fig. 11c). Lys145 would then again act as the general base, this time activating the deacylating water that forms the tetrahedral intermediate II (Fig. 11d). The Lys145 side-chain ammonium group then donates a proton to the Ser90 Oγ, and the cleaved signal peptide dissociates, regenerating the signal peptidase active site. Multiple sequence alignments (Fig. 7) suggest that the residues involved in such a mechanism are conserved. It appears from the structure of the *E. coli* signal peptidase inhibitor complex that the reason the trapped acyl-enzyme was so stable (>60 days in the crystal) was that the deacylating water was completely excluded from the active site by the bound inhibitor.

Other enzymes with a potential Ser nucleophile and a Lys general base include the *E. coli* LexA repressor (Little et al., 1994), the *E. coli* UmuD protease (Peat et al., 1996), the *E. coli* Tsp protease (Keiler & Sauer, 1995), the class A and C β-lactamases (Strynadka et al., 1992; Oefner et al., 1990), the Streptomyces D-Ala-D-Ala peptidases (Kelly et al., 1989), the *Pseudomonas aeruginosa* TraF (Haase & Lanka, 1997), and the rat fatty acid hydrolase (Patricelli & Cravatt, 1999) (Table 4).

Enzymes other than proteases and hydrolases that appear to have a lysine general base are listed in Table 5. For a review of the evidence for the Ser/Lys dyad mechanism, see Paetzel and Dalbey (1997).

## 5. Conclusion

Bacterial Type I signal peptidase is a unique serine protease that represents a novel antibiotic target accessible at the bacterial membrane surface. As a result of the many new primary sequences available from the bacterial genome sequencing projects (http://www.tigr.org/), the recent three-dimensional structural information from *E. coli* signal peptidase, and the many site-directed mutagenesis experiments, we are now finally beginning to understand this unique class of serine protease.

With the first structure of a Type I signal peptidase now available, modeling of other bacterial Type I signal peptidases, as well as docking of potential inhibitors and the development of more effective peptide substrates for high-throughput screening, is now possible.

---

**Table 4**

**Enzymes with a potential serine nucleophile and lysine general base**

| Enzyme                          | Residues     | Mutational evidence                                      | Structural evidence          |
|----------------------------------|---------------|----------------------------------------------------------|------------------------------|
| *E. coli* Type I signal peptidase | S90, K145     | Sung and Dalbey, 1992; Tschantz et al., 1993; Black, 1993 | Paetzel et al., 1998        |
| *E. coli* repressor LexA         | S119, K156    | Slilaty and Little, 1987                                 |                              |
| *E. coli* UmuD                   | S60, K97      | Takehiko et al., 1988                                    | Peat et al., 1996            |
| *E. coli* Tsp protease           | S430, K455    | Keiler and Sauer, 1995                                   |                              |
| *E. coli* β-lactamase            | S70, K73      | Gibson et al., 1990                                      | Strynadka et al., 1992       |
| Streptomyces D-Ala-D-Ala peptidase | S62, K65     |                                                          | Kelly et al., 1989           |
| *Pseudomonas asparaginase*       | T100, K173    | Swain et al., 1993; Miller et al., 1993                  | Lubkowski et al., 1994       |
| *Pseudomonas aeruginosa* (RP4) TraF | S37, K89   | Haase and Lanka, 1997                                    |                              |
| Rat fatty acid amide hydrolase   | S241, K142    | Patricelli and Cravatt, 1999                             |                              |

Table 5  
Enzymes with a potential lysine general base  

| Enzyme                                      | Residue | Mutational evidence                  | Structural evidence                          |
|---------------------------------------------|---------|---------------------------------------|--------------------------------------------|
| *Pseudomonas putida* mandelate racemase     | K166    | Landro et al., 1994                 | Landro et al., 1994                        |
| Yeast enolase                               | K345    | Poyner et al., 1996                 | Wedekind et al., 1994                      |
| *E. coli* aspartate aminotransferase        | K258    | Planas and Kirsch, 1991             | Kirsch et al., 1984                        |
| *Bacillus stearothermophilus* leucine dehydrogenase | K80   | Sekimoto et al., 1993               | Baker et al., 1992 (related enzyme)        |
| 6-Phosphogluconate dehydrogenase            | K183    | Zhang et al., 1999                  |                                            |

Many fundamental questions remain to be answered regarding the function of bacterial Type I signal peptidase. Does signal peptidase cleave off the signal peptide from preproteins during or after translocation? And in what milieu does the cleavage occur? In the bulk solvent of the periplasm, in the head groups of the lipid bilayer, in the acyl chains of the lipid bilayer, or in the proteinaceous pore of the translocase? Is interfacial catalysis involved? What residues within the signal peptidase catalytic domain associate with the lipid bilayer? What roles, if any, do Domain II and the extended β-ribbon of the gram-negative signal peptidases play in catalysis? Why do some bacterial species have multiple Type I signal peptidases?

### Acknowledgments

This work was supported by the Medical Research Council of Canada, the Canadian Bacterial Diseases Network of Excellence, and the Burroughs Wellcome Foundation grants to N.C.J.S. R.E.D. is supported by NIH grant GM 48805. M.P. is funded by a Medical Research Council of Canada post-doctoral fellowship and N.C.J.S. by a Medical Research Council of Canada Scholarship.

### References

Allsop, A. E., Brooks, G., Bruton, G., Coulton, S., Edwards, P. D., Hatton, I. K., Mclean, A. C., Person, N. D., Smale, T. C., & Southgate, R. (1995). Penem inhibitors of bacterial signal peptidase. *Bioorg Med Chem Lett* 5, 443–448.

Allsop, A., Brooks, G., Edwards, P. D., Kaura, A. C., & Southgate, R. (1996). Inhibitors of bacterial signal peptidase: a series of 6-(substituted oxyethyl)penems. *J Antibiot* 49, 921–928.

Allsop, A. E., Ashby, M. J., Brooks, G., Bruton, G., Coulton, S., Edwards, P. D., Elsmere, S. A., Hatton, I. K., Kaura, A. C., McLean, S. D., Pearson, M. J., Pearson, N. D., Perry, C. R., Smale, T. C., & Southgate, R. (1997). Inhibition of protein export in bacteria: the signaling of a new role for β-lactams. In P. H. Bentley & P. J. O’Hanlon (Eds.), *Proceedings of the 2nd International Symposium on Recent Advances in Chemistry of Anti-infective Agents*, Churchill College, Cambridge, England, July 7–10, 1996 (pp. 61–72). Cambridge: The Royal Society of Chemistry.

Bairl, A., & Muller, P. (1998). A second gene for type I signal peptidase in *Bradyrhizobium japonicum*, sipF, is located near genes involved in RNA processing and cell division. *Mol Gen Genet* 260, 346–356.

Baker, P. J., Britton, K. L., Engel, P. C., Farrant, G. W., Lilley, K. S., Rice, D. W., & Stillman, T. J. (1992). Subunit assembly and active site location in the structure of glutamate dehydrogenase. *Proteins* 12, 75–86.

Barbrook, A. C., Packer, C. L., & Howe, C. J. (1996). Inhibition by penem of processing peptidases from cyanobacteria and chloroplast thylakoids. *FEBS Lett* 398, 198–200.

Barkocy-Gallagher, G. A., & Bassford, P. J. (1992). Synthesis of precursor maltose-binding protein with proline in the +1 position of the cleavage site interferes with the activity of *Escherichia coli* signal peptidase I. *J Biol Chem* 267, 1231–1238.

Batenburg, A. M., Brasseur, R., Russychaert, J.-M., van Scharenburg, G. J., Slotboom, A. J., Demel, R. A., & de Kruijff, B. (1988). Characterization of the interfacial behavior and structure of the signal sequence of *Escherichia coli* outer membrane pore protein PhoE. *J Biol Chem* 263, 4202–4207.

Bechinger, B., Gierasch, L. M., Montal, M., Zasloff, M., & Opella, S. J. (1996). Orientation of helical peptides in membrane bilayers by solid state NMR spectroscopy. *Solid State Nucl Magn Reson* 7, 185–191.

Behrens, M., Michaelis, G., & Pratje, E. (1991). Mitochondrial inner membrane protease I of *Saccharomyces cerevisiae* shows sequence similarity to the *Escherichia coli* leader peptidase. *Mol Gen Genet* 228, 167–176.

Bilgin, N., Lee, J. I., Zhu, H. Y., Dalbey, R. E., & von Heijne, G. (1990). Mapping of catalytically important domains in *Escherichia coli* leader peptidase. *EMBO J* 9, 2717–2722.

Black, M. T. (1993). Evidence that the catalytic activity of prokaryote leader peptidase depends upon the operation of a serine-lysine catalytic dyad. *J Bacteriol* 175, 4957–4961.

Black, M., & Bruton, G. (1998). Inhibitors of bacterial signal peptidase. *Curr Pharm Design* 4, 133–154.

Black, M. T., Munn, J. G. R., & Allsop, A. E. (1992). On the catalytic mechanism of prokaryotic leader peptidase. *Biochem J* 282, 539–543.

Bolhuis, A., Tjalsma, H., Stephenson, K., Harwood, C. R., Venema, G., Bron, S., & van Dijl, J. M. (1999). Different mechanisms for thermal inactivation of *Bacillus subtilis* signal peptidase mutants. *J Biol Chem* 274, 15865–15868.

Bruch, M. D., McKnight, C. J., & Gierasch, L. M. (1989). Helix formation and stability in a signal sequence. *Biochemistry* 28, 8554–8561.

Buchanan, S. K., Smith, B. S., Venkatramani, L., Xia, D., Esser, L., Palnitkar, M., Chakraborty, R., van der Helm, D., & Deisenhofer, J. (1999). Crystal structure of the outer membrane active transporter FepA from *Escherichia coli*. *Nature Struct Biol* 6, 56–63.

Caulfield, M. P., Duong, L. T., O’Brien, R., Majzoub, J. A., & Rosenblatt, M. (1988). A chemically synthesized radiolabeled signal peptide: design, preparation, and biological evaluation of an iodinated analog of preproparathyroid hormone. *Mol Endocrinol* 2, 452–458.

Caulfield, M. P., Duong, L. T., Baker, R. K., Rosenblatt, M., & Lively, M. O. (1989). Synthetic substrate for eukaryotic signal peptidase. Cleavage of a synthetic peptide analog of the precursor region of preproparathyroid hormone. *J Biol Chem* 264, 15813–15817.

Cavalier-Smith, T. (1987). The origin of eukaryotic and archaebacteria cells. *Ann NY Acad Sci* 503, 17–54.

Chaal, B. K., Mould, R. M., Barbrook, A. C., Gray, J. C., & Howe, C. J. (1998). Characterization of a cDNA encoding the thylakoidal processing peptidase from *Arabidopsis thaliana*: implications for the origin and catalytic mechanism of the enzyme. *J Biol Chem* 273, 689–692.

Chang, G., Spencer, R. H., Lee, A. T., Barclay, M. T., & Rees, D. C. (1998). Structure of the MscL homolog from *Mycobacterium tuberculosis*: a gated mechanosensitive ion channel. *Science* 282, 2220–2226.

Chang, N. C., Blobel, G., & Model, P. (1978). Detection of prokaryotic signal peptide in an *Escherichia coli* membrane fraction: endoproteolytic cleavage of nascent f1 pre-coat protein. *Proc Natl Acad Sci USA* 75, 361–365.

Chatterjee, S., Suciu, D., Dalbey, R., Kahn, P. C., & Inouye, M. (1995). Determination of Km and kcat for signal peptidase I using a full length secretory precursor, pro-OmpA-nuclease A. *J Mol Biol* 245, 311–314.

Chupin, V., Killian, J. A., Breg, J., de Jongh, H. H. J., Boelens, R., Kaptein, R., & de Kruijff, B. (1995). PhoE signal peptide inserts into micelles as a dynamic helix-break-helix structure, which is modulated by the environment. A two-dimensional 1H NMR study. *Biochemistry* 34, 11617–11624.

Cornell, D. G., Dluhy, R. A., Briggs, M. S., McKnight, C. J., & Gierasch, L. M. (1989). Conformations and orientations of a signal peptide interacting with phospholipid monolayers. *Biochemistry* 28, 2789–2797.

Cregg, K. M., Wilding, E. I., & Black, M. T. (1996). Molecular cloning and expression of the spsB gene encoding an essential type I signal peptidase from *Staphylococcus aureus*. *J Bacteriol* 178, 5712–5718.

Dalbey, R. E., & Wickner, W. (1985). Leader peptidase catalyzes the release of exported proteins from the outer surface of the *Escherichia coli* plasma membrane. *J Biol Chem* 260, 15925–15931.

Dalbey, R. E., Lively, M. O., Bron, S., & van Dijl, J. M. (1997). The chemistry and enzymology of the type I signal peptidases. *Protein Sci* 6, 1129–1138.

Date, T. (1983). Demonstration by a novel genetic technique that leader peptidase is an essential enzyme in *Escherichia coli*. *J Bacteriol* 154, 76–83.

Date, T., & Wickner, W. (1981). Isolation of the *Escherichia coli* leader peptidase gene and effects of leader peptidase overproduction *in vivo*. *Proc Natl Acad Sci USA* 78, 6106–6110.

Dev, I. K., Ray, P. H., & Novak, P. (1990). Minimum substrate sequence for signal peptidase I of *Escherichia coli*. *J Biol Chem* 265, 20069–20072.

de Vrije, G. J., Batenburg, A. M., Killian, J. A., & de Kruijff, B. (1990). Lipid involvement in protein translocation in *Escherichia coli*. *Mol Microbiol* 4, 143–150.

Dierstein, R., & Wickner, W. (1986). Requirement for substrate recognition by bacterial leader peptidase. *EMBO J* 5, 427–431.

Doyle, D. A., Morais Cabral, J., Pfuetzner, R. A., Kuo, A., Gulbis, J. M., Cohen, S. L., Chait, B. T., & MacKinnon, R. (1998). The structure of the potassium channel: molecular basis of K+ conduction and selectivity. *Science* 280, 69–77.

Dutzler, R., Rummel, G., Alberti, S., Hernandez-Alles, S., Phale, P., Rosenbusch, J., Benedi, V., & Schirmer, T. (1999). Crystal structure and functional characterization of OmpK36, the osmoporin of *Klebsiella pneumoniae*. *Structure* 7, 425–434.

Economou, A. (1999). Following the leader: bacterial protein export through the Sec pathway. *Trends Microbiol* 7, 315–320.

Edman, M., Jarhede, T., Sjostrom, M., & Wieslander, A. (1999). Different sequence patterns in signal peptides in signal peptidases from mycoplasmas, other gram-positive bacteria, and *Escherichia coli*: multivariate data analysis. *Proteins Struct Funct Genet* 35, 195–205.

Evans, E. A., Gilmore, R., & Blobel, G. (1986). Purification of microsomal signal peptidase as a complex. *Proc Natl Acad Sci USA* 83, 581–585.

Fang, H., Panzner, S., Mullins, C., Hartmann, E., & Green, N. (1996). The homologue of mammalian SPC12 is important for efficient signal peptidase activity in *Saccharomyces cerevisiae*. *J Biol Chem* 271, 16460–16465.

Fang, H., Mullins, C., & Green, N. (1997). In addition to Sec11, a newly identified gene, SPC3, is essential for signal peptidase activity in the yeast endoplasmic reticulum. *J Biol Chem* 272, 13152–13158.

Ferguson, A. D., Hofmann, E., Coulton, J. W., Diederichs, K., & Welte, W. (1998). Siderophore-mediated iron transport: crystal structure of FhuA with bound lipopolysaccharide. *Science* 282, 2215–2220.

Fikes, J. D., & Bassford, P. J., Jr. (1987). Export of unprocessed precursor maltose-binding protein to the periplasm of *Escherichia coli* cells. *J Bacteriol* 169, 2352–2359.

Fikes, J. D., Barkocy-Gallagher, G. A., Klapper, D. G., & Bassford, P. J., Jr. (1990). Maturation of *Escherichia coli* maltose-binding protein by signal peptidase I *in vivo*: sequence requirement for efficient processing and demonstration of an alternate cleavage site. *J Biol Chem* 265, 3417–3423.

Folz, R. J., Nothwehr, S. F., & Gordon, J. I. (1988). Substrate specificity of eukaryotic signal peptidase. Site-saturation mutagenesis at position –1 regulates cleavage between multiple sites in human pre (delta pro) apolipoprotein A-II. *J Biol Chem* 263, 2070–2078.

Gelb, M. H., Cho, W., & Wilton, D. C. (1999). Interfacial binding of secreted phospholipases A₂: more than electrostatics and a major role for tryptophan. *Curr Opin Struct Biol* 9, 428–432.

Gibson, R. M., Christensen, H., & Waley, S. G. (1990). Site-directed mutagenesis of β-lactamase I. Single and double mutants of Glu-166 and Lys-73. *Biochem J* 272, 613–619.

Greenburg, G., & Blobel, G. (1994). cDNA-derived primary structure of the 25-kDa subunit of canine microsomal signal peptidase complex. *J Biol Chem* 269, 25354–25358.

Haase, J., & Lanka, E. (1997). A specific protease encoded by the conjugative DNA transfer system of IncP and Ti plasmids is essential for pilus synthesis. *J Bacteriol* 179, 5728–5735.

Halpin, C., Elderfield, P. D., James, H. E., Zimmermann, R., Byran, D., & Robinson, C. (1989). The reaction specificities of the thylakoidal processing peptidase and the *Escherichia coli* leader peptidase are identical. *EMBO J* 8, 3917–3921.

Hegner, M., von Kieckebusch-Gluck, A., Falchetto, R., James, P., Semenza, G., & Mantei, N. (1992). Single amino acid substitutions can convert the uncleaved signal-anchor of sucrase-isomaltose to a cleaved signal sequence. *J Biol Chem* 267, 16928–16933.

Hendrickson, W. A., & Orgata, C. M. (1997). Phase determination from multiwavelength anomalous diffraction measurements. *Methods Enzymol* 276, 494–523.

Hoang, V., & Hofemeister, J. (1995). *Bacillus amyloliquefaciens* possesses a second type I SPase with extensive sequence similarity to other *Bacillus* SPases. *Biochim Biophys Acta* 1269, 64–68.

Hutchinson, E. G., & Thornton, J. M. (1996). PROMOTIF—a program to identify and analyze structural motifs in proteins. *Protein Sci* 5, 212–220.

Inada, T., Court, D. L., Ito, K., & Nakamura, Y. (1989). Conditional lethal amber mutations in the leader peptidase gene of *Escherichia coli*. *J Bacteriol* 171, 585–587.

Iwata, S., Ostermeier, C., Ludwig, B., & Michel, H. (1995). Structure at 2.8 Å resolution of cytochrome c oxidase from *Paracoccus denitrificans*. *Nature* 376, 660–669.

Izard, J. W., & Kendall, D. A. (1994). Signal peptides: exquisitely designed transport promoters. *Mol Microbiol* 13, 765–773.

Jackson, R. C. (1983). Quantitative assay for signal peptidase. *Methods Enzymol* 96, 784–794.

James, M. N. G. (1994) Convergence of active-centre geometries among the proteolytic enzymes. In J. S. Bond & A. J. Barrett (Eds.), *Proteolysis and Protein Turnover* (pp. 1–8). Brookfield: Portland Press.

Jock, C. A., Pulakat, L., Lee, S., & Gavini, N. (1997). Nucleotide sequence and genetic complementation analysis of lep from *Azotobacter vinelandii*. *Biochem Biophys Res Commun* 239, 393–400.

Jones, J. D., & Gierasch, L. M. (1994). Effect of charged residue substitutions on the membrane-interactive properties of signal sequences of the *Escherichia coli* LamB protein. *Biophys J* 67, 1534–1544.

Jones, J. D., McKnight, C. J., & Gierasch, L. M. (1990). Biophysical studies of signal peptides: implications for signal sequence functions and the involvement of lipid in protein export. *J Bioenerg Biomembr* 22, 213–232.

Kalies, K., & Hartmann, E. (1996). Membrane topology of the 12- and the 25-kDa subunits of the mammalian signal peptidase complex. *J Biol Chem* 271, 3925–3929.

Karamyshev, A. L., Karamysheva, Z. N., Kajava, A. V., Ksenzenko, V. N., & Nesmeyanova, M. A. (1998). Processing of *Escherichia coli* alkaline phosphatase: role of the primary structure of the signal peptide cleavage region. *J Mol Biol* 277, 859–870.

Keiler, K. C., & Sauer, R. T. (1995). Identification of the active site residues of the Tsp protease. *J Biol Chem* 270, 28864–28868.

Keller, R. C. A., ten Berge, D., Nouwen, N., Snel, M. M. E., Tommassen, J., Marsh, D., & de Kruijff, B. (1996). Mode of insertion of the signal sequence of a bacterial precursor protein into phospholipid bilayers as revealed by cysteine-based site-directed spectroscopy. *Biochemistry* 35, 3063–3071.

Kelly, J. A., Knox, J. R., Zhao, H., Frere, J., & Ghuysen, J. (1989). Crystallographic mapping of β-lactams bound to a d-alanyl-d-alanine peptidase target enzyme. *J Mol Biol* 209, 281–295.

Killian, J. A., de Jong, A. M. P., Bijvelt, J., Verkleij, A. J., & de Kruijff, B. (1990). Induction of non-bilayer lipid structures by functional signal peptides. *EMBO J* 9, 815–819.

Kim, Y. T., Muramatsu, T., & Takahashi, K. (1995a). Leader peptidase from *Escherichia coli*: overexpression, characterization, and inactivation by modification of tryptophan residues 300 and 310 with *N*-bromosuccinimide. *J Biochem (Japan)* 117, 535–544.

Kim, Y. T., Muramatsu, T., & Takahashi, K. (1995b). Identification of Trp300 as an important residue for *Escherichia coli* leader peptidase activity. *Eur J Biochem* 234, 358–362.

Kirsch J. F., Eichele, G., Ford, G. C., Vincent, M., & Jansonius, J. N. (1984). Mechanism of action of aspartate aminotransferase proposed on the basis of its spatial structure. *J Mol Biol* 174, 497–525.

Klug, G., Jager, A., Heck, C., & Rauhut, R. (1997). Identification, sequence analysis, and expression of the lepB gene for a leader peptidase in *Rhodobacter capsulatus*. *Mol Gen Genet* 253, 666–673.

Koshland, D., Sauer, R. T., & Botstein, D. (1982). Diverse effects of mutations in the signal sequence on the secretion of beta-lactamase in *Salmonella typhimurium*. *Cell* 30, 903–914.

Kraulis, P. G. (1991). *Molscript*: a program to produce both detailed and schematic plots of protein structures. *J Appl Crystallogr* 24, 946–950.

Kuhn, A., & Wickner, W. (1985). Conserved residues of the leader peptide are essential for cleavage by leader peptidase. *J Biol Chem* 260, 15914–15918.

Kuo, D. W., Cahn, H. K., Wilson, C. J., Griffin, P. R., Williams, H., & Knight, W. B. (1993). *Escherichia coli* leader peptidase: production of an active form lacking a requirement for detergent and development of peptide substrates. *Arch Biochem Biophys* 303, 274–280.

Kuo, D., Weidner, J., Griffin, P., Shah, S. K., & Knight, W. B. (1994). Determination of the kinetic parameters of *Escherichia coli* leader peptidase activity using a continuous assay: pH dependence and time-dependent inhibition by β-lactams are consistent with a novel serine protease mechanism. *Biochemistry* 33, 8347–8354.

Landolt-Marticorena, C., Williams, K. A., Deber, C. M., & Reithmeier, R. A. (1993). Non-random distribution of amino acids in the transmembrane segments of human type I single span membrane proteins. *J Mol Biol* 229, 602–608.

Landro, J. A., Gerlt, J. A., & Kozarich, J. W. (1994). The role of lysine 166 in the mechanism of mandelate racemase from *Pseudomonas putida*: mechanistic and crystallographic evidence for stereospecific alkylation by (*R*)-α-phenylglycidate. *Biochemistry* 33, 635–643.

Liang, J., Edelsbrunner, H., & Woodward, C. (1998). Anatomy of protein pockets and cavities: measurement of binding site geometry and implications for ligand design. *Protein Sci* 7, 1884–1897.

Little, J. W., Baek, K., Roland, K. L., Smith, M. H., Lin, L., & Slilaty, S. N. (1994). Cleavage of LexA repressor. *Methods Enzymol* 244, 266–284.

Lively, M. O., Newsome, A. L., & Nusier, M. (1994). Eukaryotic microsomal signal peptidases. *Methods Enzymol* 244, 301–314.

Lubkowski, J., Wlodawer, A., Ammon, H. L., Copeland, T. D., & Swain, A. L. (1994). Structural characterization of *Pseudomonas* 7A glutaminase-asparaginase. *Biochemistry* 33, 10257–10265.

McKnight, C. J., Briggs, M. S., & Gierasch, L. M. (1989). Functional and nonfunctional LamB signal sequences can be distinguished by their biophysical properties. *J Biol Chem* 264, 17293–17297.

McKnight, C. J., Rafalski, M., & Gierasch, L. M. (1991a). Fluorescence analysis of tryptophan-containing variants of the LamB signal sequence upon insertion into a lipid bilayer. *Biochemistry* 30, 6241–6246.

McKnight, C. J., Stradley, S. J., Jones, J. D., & Gierasch, L. M. (1991b). Conformational and membrane-binding properties of a signal sequence are largely unaltered by its adjacent mature region. *Proc Natl Acad Sci USA* 88, 5799–5803.

Meijer, W. J. J., de Jong, A., Bea, G., Wisman, A., Tjalsma, H., Venema, G., Bron, S., & van Dijl, J. M. (1995). The endogenous *Bacillus subtilis* (natto) plasmids pTA1015 and pTA1040 contain signal peptidase-encoding genes: identification of a new structural module on cryptic plasmids. *Mol Microbiol* 17, 621–631.

Meyer, H. A., & Hartmann, E. (1997). The yeast SPC22/23 homolog SPC3p is essential for signal peptidase activity. *J Biol Chem* 272, 13159–13164.

Miller, M., Roa, J. K. M., Wlodawer, A., & Gribskov, M. R. (1993). A left-handed crossover involved in amidohydrolase catalysis. Crystal structure of *Erwira chrysanthemi* L-asparaginase with bound L-aspartate. *FEBS Lett* 328, 275–279.

Moore, K. E., & Miura, S. (1987). A small hydrophobic domain anchors leader peptidase to the cytoplasmic membrane of *Escherichia coli*. *J Biol Chem* 262, 8806–8813.

Muller, P., Ahrens, K., Keller, T., & Klaucke, A. (1995). A TnphoA insertion within the *Bradyrhizobium japonicum* sipS gene, homologous to prokaryotic signal peptidases, results in extensive changes in the expression of PBM-specific nodulins of infected soybean (*Glycine max*) cells. *Mol Microbiol* 18, 831–840.

Mullins, C., Meyer, H.-A., Hartmann, E., Green, N., & Fang, H. (1996). Structurally related SPC1p and SPC2p of yeast signal peptidase complex are functionally distinct. *J Biol Chem* 271, 29094–29099.

Neu, H. C. (1992). The crisis in antibiotic resistance. *Science* 257, 1064–1073.

Nielsen, H., Engelbrecht, J., Brunak, S., & von Heijne, G. (1997a). Identification of prokaryotic and eukaryotic signal peptides and prediction of their cleavage sites. *Protein Eng* 10, 1–6.

Nielsen, H., Engelbrecht, J., Brunak, S., & von Heijne, G. (1997b). A neural network method for identification of prokaryotic and eukaryotic signal peptides and prediction of their cleavage sites. *Int J Neural Syst* 8, 581–599.

Nielsen, H., Brunak S., & von Heijne, G. (1999). Machine learning approaches to the prediction of signal peptides and other protein sorting signals. *Protein Eng* 12, 3–9.

Nilsson, I. M., & von Heijne, G. (1992). A signal peptide with a proline next to the cleavage site inhibits leader peptidase when present in a secreted protein. *FEBS Lett* 299, 243–246.

Nothwehr, S. F., & Gordon, J. I. (1989). Eukaryotic signal peptide structure/function relationships. Identification of conformational features which influence the site and efficiency of co-translational proteolytic processing by site-directed mutagenesis of human pre(delta pro)apolipoprotein A-II. *J Biol Chem* 264, 3979–3987.

Nothwehr, S. F., & Gordon, J. I. (1990). Structural features in the NH₂-terminal region of a model eukaryotic signal peptide influence the site of its cleavage by signal peptidase. *J Biol Chem* 265, 17202–17208.

Nunnari, J., Fax, T. D., & Walter, P. (1993). A mitochondrial protease with two catalytic subunits with nonoverlapping specificities. *Science* 262, 1997–2004.

Oefner, C., D’Arcy, A., Daly, J. J., Gubernator, K., Charnas, R. L., Heinze, I., Hubscherlen, C., & Winkler, F. K. (1990). Refined crystal structure of beta-lactamase from *Citrobacter freundii* indicates a mechanism for beta-lactam hydrolysis. *Nature* 343, 284–288.

Olson, R., Nariya, H., Yokota, K., Kamio, Y., & Gouaux, E. (1999). Crystal structure of staphylococcal LukF delineates conformational changes accompanying formation of a transmembrane channel. *Nature Struct Biol* 6, 134–140.

Ostermeier, C., & Michel, H. (1997). Crystallization of membrane proteins. *Curr Opin Struct Biol* 7, 697–701.

Packer, J. C. L., Andre, D., & Howe, C. J. (1995). Cloning and sequence analysis of a signal peptidase 1 from the thermophilic cyanobacterium *Phormidium laminosum*. *Plant Mol Biol* 27, 199–204.

Paetzel, M., & Dalbey, R. E. (1997). Catalytic hydroxyl/amine dyads with serine proteases. *Trends Biochem Sci* 22, 28–31.

Paetzel, M., & Strynadka, N. J. C. (1999). Common protein architecture and binding sites in proteases utilizing a Ser/Lys dyad mechanism. *Protein Sci* 8, 2533–2536.

Paetzel, M., Chernia, M., Strynadka, N., Tschantz, W., Cao, G., Dalbey, R. E., & James, M. N. G. (1995). Crystallization of a soluble, catalytically active form of *Escherichia coli* leader peptidase. *Proteins Struct Funct Genet* 23, 122–125.

Paetzel, M., Strynadka, N. C. J., Tschantz, W. R., Casareno, R., Bullinger, P. R., & Dalbey, R. E. (1997). Use of site-directed chemical modification to study an essential lysine in *Escherichia coli* leader peptidase. *J Biol Chem* 272, 9994–10003.

Paetzel, M., Dalbey, R. E., & Strynadka, N. C. J. (1998). Crystal structure of a bacterial signal peptidase in complex with a β-lactam inhibitor. *Nature* 396, 186–190.

Parro, V., & Mellado, R. P. (1998). A new signal peptidase gene from *Streptomyces lividans* TK21. *DNA Seq* 9, 71–77.

Parro, V., Schacht, S., Anne, J., & Mellado, R. P. (1999). Four genes encoding different type I signal peptidases are organized in a cluster in *Streptomyces lividans* TK21. *Microbiology* 145, 2255–2263.

Patricelli, M. P., & Cravatt, B. F. (1999). Fatty acid amide hydrolase competitively degrades bioactive amides and esters through a nonconventional catalytic mechanism. *Biochemistry* 38, 14125–14130.

Peat, T. S., Frank, E. G., McDonald, J. P., Levine, A. S., Woodgate, R., & Hendrickson, W. A. (1996). Structure of the UmuD′ protein and its regulation in response to DNA damage. *Nature* 380, 727–730.

Perlman, D., & Halvorson, H. O. (1983). A putative signal peptidase recognition site and sequence in eukaryotic and prokaryotic signal peptides. *J Mol Biol* 167, 391–409.

Perona, J. J., & Craik, C. S. (1995). Structural basis of substrate specificity in the serine proteases. *Protein Sci* 4, 337–360.

Perry, C. R., Ashby, M. J., & Elsmere, S. A. (1995). Penems as research tools to investigate the activity of *E. coli* leader peptidase. *Biochem Soc Trans* 23, 548S.

Planas, A., & Kirsch, J. F. (1991). Reengineering the catalytic lysine of aspartate aminotransferase by chemical elaboration of a genetically introduced cysteine. *Biochemistry* 30, 8268–8276.

Poyner, R. R., Laughlin, L. T., Sowa, G. A., & Reed, G. H. (1996). Toward identification of acid/base catalysts in the active site of enolase: comparison of the properties of K345A, E168Q, and E211Q variants. *Biochemistry* 35, 1692–1699.

Pratap, J., & Dikshit, K. L. (1998). Effect of signal peptide changes on the extracellular processing of streptokinase from *Escherichia coli*: requirement for secondary structure at the cleavage junction. *Mol Gen Genet* 258, 326–333.

Rawlings, N. D., & Barrett, A. J. (1994). Families of serine proteases. *Methods Enzymol* 244, 19–61.

Rizo, J., Blanco, F. J., Kobe, B., Bruch, M. D., & Gierasch, L. M. (1993). Conformational behavior of *Escherichia coli* OmpA signal peptides in membrane mimetic environments. *Biochemistry* 32, 4881–4894.

Rosenblatt, M., Beaudette, N. V., & Fasman, G. D. (1980). Conformational studies of the synthetic precursor-specific region of preproparathyroid hormone. *Proc Natl Acad Sci USA* 77, 3983–3987.

Ryan, P., & Edwards, C. O. (1995). Systematic introduction of proline in a eukaryotic signal sequence suggests asymmetry within the hydrophobic core. *J Biol Chem* 270, 27876–27879.

Sankaram, M. B., Marsh, D., Gierasch, L. M., & Thompson, T. E. (1994). Reorganization of lipid domain structure in membranes by a transmembrane peptide: an ESR spin label study on the effect of the *Escherichia coli* outer membrane protein A signal peptide on the fluid lipid domain connectivity in binary mixtures of dimyristoyl phosphatidylcholine and distearoyl phosphatidylcholine. *Biophys J* 66, 1959–1968.

San Millan, J. L., Boyd, D., Dalbey, R., Wickner, W., & Beckwith, J. (1989). Use of phoA fusions to study the topology of the *Escherichia coli* inner membrane protein leader peptidase. *J Bacteriol* 171, 5536–5541.

Schacht, S., van Mellaert, L., Lammertyn, H., Tjalsma, H., van Dijl, J. M., Bron, S., & Anne, J. (1998). The sip (sli) gene of *Streptomyces lividans*

TK24 specifies an unusual signal peptidase with a putative C-terminal transmembrane anchor. *DNA Seq* 9, 79–88.

Schechter, I., & Berger, A. (1967). On the size of the active site in proteases. I. Papain. *Biochem Biophys Res Commun* 27, 157–162.

Schneider, A., Oppiger, W., & Jeno, P. (1994). Purified inner membrane protease I of yeast mitochondria is a heterodimer. *J Biol Chem* 269, 8635–8638.

Sekimoto, T., Matsuyama, T., Fukui, T., & Tanizawa, K. (1993). Evidence for lysine 80 as general base catalyst of leucine dehydrogenase. *J Biol Chem* 268, 27039–27045.

Shelness, G. S., Lin, L., & Nicchitta, C. V. (1993). Membrane topology and biogenesis of eukaryotic signal peptidase. *J Biol Chem* 268, 5201–5208.

Shen, L. M., Lee, J. I., Cheng, S., Jutte, H., Kuhn, A., & Dalbey, R. E. (1991). Use of site-directed mutagenesis to define the limits of sequence variation tolerated for processing of the M13 procoat protein by the *Escherichia coli* leader peptidase. *Biochemistry* 30, 11775–11781.

Slilaty, S. N., & Little, J. W. (1987). Lysine-156 and serine-119 are required for LexA repressor cleavage: a possible mechanism. *Proc Natl Acad Sci USA* 84, 3987–3991.

Song, L., & Gouaux, J. E. (1997). Membrane protein crystallization: application of sparse matrices to the α-hemolysin heptamer. *Methods Enzymol* 276, 60–74.

Strynadka, N. C. J., Adachi, H., Jensen, S. E., Johns, K., Sielecki, A., Betzel, C., & James, M. N. G. (1992). Molecular structure of the acyl enzyme intermediate in β-lactamase at 1.7 Å resolution. *Nature* 359, 393–400.

Suciu, D., Chatterjee, S., & Inouye, M. (1997). Catalytic efficiency of signal peptidase 1 of *Escherichia coli* is comparable to that of members of the serine protease family. *Protein Eng* 10, 1057–1060.

Sung, M., & Dalbey, R. E. (1992). Identification of potential active-site residues in the *Escherichia coli* leader peptidase. *J Biol Chem* 267, 13154–13159.

Swain, A. L., Jaskolski, M., Housset, D., Rao, J. K. M., & Wlodawer, A. (1993). Crystal structure of *Escherichia coli* L-asparaginase, an enzyme used in cancer therapy. *Proc Natl Acad Sci USA* 90, 1474–1478.

Takehiko, N., Battista, J. R., Dodson, L. A., & Walker, G. C. (1988). RecA-mediated cleavage activates UmuD for mutagenesis: mechanistic relationship between transcriptional derepression and posttranslational activation. *Proc Natl Acad Sci USA* 85, 1816–1820.

Talarico, T. L., Dev, I. K., Bassford, P. J., & Ray, P. H. (1991). Intermolecular degradation of signal peptidase I in vitro. *Biochem Biophys Res Commun* 181, 650–656.

Taylor, P., Anderson, V., Dowden, J., Flitsch, S. L., Turner, N. J., Loughran, K., & Walkinshaw, M. D. (1999). Novel mechanism of inhibition of elastase by β-lactams is defined by two inhibitor crystal complex. *J Biol Chem* 274, 24901–24905.

Tew, D. G., Boyd, H. F., Ashman, S., Theobald, C., & Leach, C. A. (1998). Mechanism of inhibition of LDL phospholipase A2 by monocyclic beta-lactams. Burst kinetics and the effect of stereochemistry. *Biochemistry* 37, 10087–10093.

Tjalsma, H., Noback, M. A., Bron, B., Venema, G., Yamane, K., & van Dijl, J. M. (1997). *Bacillus subtilis* contains four closely related type I signal peptidases with overlapping substrate specificity. *J Biol Chem* 272, 25983–25992.

Tjalsma, H., Bolhuis, A., van Rossmalen, M. L., Wieger, T., Schumann, W., Broekhuizen, C. P., Quax, W. J., Verema, G., Bron S., & van Dijl, J. M. (1998). Functional analysis of the secretory precursor processing machinery of *Bacillus subtilis*: identification of a eubacterial homolog of archael and eukaryotic signal peptidase. *Genes Dev* 12, 2318–2331.

Tschantz, W. R., & Dalbey, R. E. (1994). Bacterial leader peptidase 1. *Methods Enzymol* 244, 285–301.

Tschantz, W. R., Sung, M., Delgado-Partin, V. M., & Dalbey, R. E. (1993). A serine and a lysine residue implicated in the catalytic mechanism of *Escherichia coli* leader peptidase. *J Biol Chem* 268, 27349–27354.

Tschantz, W. R., Paetzel, M., Cao, G., Suciu, D., Inouye, M., & Dalbey, R. E. (1995). Characterization of a soluble, catalytically active form of Es-

cherichia coli leader peptidase: requirement of detergent or phospholipid for optimal activity. *Biochemistry* 34, 3935–3941.

Tusnady, G. E., & Simon, I. (1998). Principles governing amino acid composition of integral membrane proteins: applications to topology prediction. *J Mol Biol* 283, 489–506.

van Dijl, J. M., van den Bergh, R., Reversma, T., Smith, H., Bron, S., & Venema, G. (1990). Molecular cloning of the *Salmonella typhimurium* lep gene in *Escherichia coli*. *Mol Gen Genet* 223, 233–240.

van Dijl, J. M., de Jong, A., Vehmaanpera, J., Venema, G., & Bron, S. (1992). Signal peptidase 1 of *Bacillus subtilis*: patterns of conserved amino acids in prokaryotic and eukaryotic type 1 signal peptidases. *EMBO J* 11, 2819–2828.

van Dijl, J. M., de Jong, A., Venema, G., & Bron, S. (1995). Identification of the potential active site of the signal peptidase SipS of *Bacillus subtilis*: structural and functional similarities with LexA-like proteases. *J Biol Chem* 270, 3611–3618.

van Klompenburg, W., Whitley, P., Demel, R. A., von Heijne, G., & de Kruijff, B. (1995). A quantitative assay to determine the amount of signal peptidase I in *E. coli* and the orientation of membrane vesicles. *Mol Membr Biol* 12, 349–353.

van Klompenburg, W., Paetzel, M., de Jong, J. M., Dalbey, R. E., Demel, R. A., von Heijne, G., & de Kruijff, B. (1998). Phosphatidylethanolamine mediated insertion of the catalytic domain of leader peptidase in membranes. *FEBS Lett* 431, 75–79.

VanValkenburg, C., Chen, X., Mullins, C., Fang, H., & Green, N. (1999). The catalytic mechanism of endoplasmic reticulum signal peptidase appears to be distinct from most eubacterial signal peptidases. *J Biol Chem* 274, 11519–11525.

Vehmaanpera, J., Gorner, A., Venema, G., Bron, S., & van Dijl, J. M. (1993). In vitro assay for the *Bacillus subtilis* signal peptidase SipS: systems for efficient in vitro transcription-translation and processing of precursors of secreted proteins. *FEMS Microbiol Lett* 114, 207–214.

Voglino, L., McIntosh, T. J., & Simon, S. A. (1998). Modulation of the binding of signal peptides to lipid bilayers by dipoles near the hydrocarbon-water interface. *Biochemistry* 37, 12241–12252.

Voglino, L., Simon, S. A., & McIntosh, T. J. (1999). Orientation of LamB signal peptides in bilayers: influence of lipid probes on peptide binding and interpretation of fluorescence quenching data. *Biochemistry* 38, 7509–7516.

von Heijne, G. (1983). Patterns of amino acids near signal-sequence cleavage sites. *Eur J Biochem* 133, 17–21.

von Heijne, G. (1985). Signal sequences. The limits of variation. *J Mol Biol* 184, 99–105.

von Heijne, G. (1986). A new method for predicting signal sequence cleavage sites. *Nucleic Acids Res* 14, 4683–4690.

von Heijne, G. (1990). The signal peptide. *J Membr Biol* 115, 195–201.

von Heijne, G. (1994). *Signal Peptidases*. Molecular Biology Intelligence Unit. Austin: R.G. Landes Company.

von Heijne, G., & Abrahmsen, L. (1989). Species-specific variation in signal peptide design. Implications for protein secretion in foreign hosts. *FEBS Lett* 244, 439–446.

Wang, Z., Jones, J. D., Rizo, J., & Gierash, L. M. (1993). Membrane-bound conformations of a signal peptide: a transferred nuclear Overhauser effect analysis. *Biochemistry* 32, 13991–13999.

Watts, C., Wickner, W., & Zimmermann, R. (1983). M13 procoat and a pre-immunoglobulin share processing specificity but use different membrane receptor mechanisms. *Proc Natl Acad Sci USA* 80, 2809–2813.

Wedekind, J. E., Poyner, R. R., Reed, G. H., & Rayment, I. (1994). Chelation of serine 39 to Mg²⁺ latches a gate at the active site of enolase: structure of the Bis(Mg²⁺) complex of yeast enolase and the intermediate analog phosphonacetohydroxamate at 2.1 Å resolution. *Biochemistry* 33, 9333–9342.

Whitley, P., & von Heijne, G. (1993). The DsbA-DsbB system affects the formation of disulfide bonds in periplasmic but not in intramembraneous protein domains. *FEBS Lett* 332, 49–51.

Whitley, P., Nilsson, L., & von Heijne, G. (1993). Three-dimensional model for the membrane domain of *Escherichia coli* leader peptidase based on disulfide mapping. *Biochemistry* 32, 8534–8539.

Wickner, W., Moore, K., Dibb, N., Geissert, D., & Rice, M. (1987). Inhibition of purified *Escherichia coli* leader peptidase by the leader (signal) peptide of bacteriophage M13 procoat. *J Bacteriol* 169, 3821–3822.

Wilmouth, R. C., Kassamally, S., Westwood, N. J., Sheppard, R. J., Claridge, T. D. W., Aplin, R. T., Wright, P. A., Pritchard, G. J., & Schofield, C. J. (1999). Mechanistic insights into the inhibition of serine proteases by monocyclic lactams. *Biochemistry* 38, 7989–7998.

Wolfe, P. B., Silver, P., & Wickner, W. (1982). The isolation of homogeneous leader peptidase from strain of *Escherichia coli* which overproduces the enzyme. *J Biol Chem* 257, 7898–7902.

Wolfe, P. B., Wickner, W., & Goodman, J. M. (1983a). Sequence of the leader peptidase gene of *Escherichia coli* and the orientation of leader peptidase in the bacterial envelope. *J Biol Chem* 258, 12073–12080.

Wolfe, P. B., Zwizinski, C., & Wickner, W. (1983b). Purification and characterization of leader peptidase from *Escherichia coli*. *Methods Enzymol* 97, 40–57.

Wrede, P., Landt, O., Klages, S., Fatami, A., Hahn, U., & Schneider, G. (1998). Peptide design aided by neural networks: biological activity of artificial signal peptidase I cleavage sites. *Biochemistry* 11, 3588–3593.

Xia, D., Yu, C.-A., Kim, H., Xia, J.-Z., Kachurin, A. M., Zhang, L., Yu, L., & Deisenhofer, J. (1997). Crystal structure of the cytochrome bc1 complex from bovine heart mitochondria. *Science* 277, 60–66.

YaDeau, J. T., & Blobel, G. (1989). Solubilization and characterization of yeast signal peptidase. *J Biol Chem* 264, 2928–2934.

Zhang, L., Chooback, L., & Cook, P. F. (1999). Lysine 183 is the general base in the 6-phosphogluconate dehydrogenase-catalyzed reaction. *Biochemistry* 38, 11231–11238.

Zhang, Y.-B., Greenberg, B., & Lacks, S. A. (1997). Analysis of a *Streptococcus pneumoniae* gene encoding signal peptidase I and overproduction of the enzyme. *Gene* 194, 249–255.

Zhong, W., & Benkovic, S. J. (1998). Development of an internally quenched fluorescent substrate for *Escherichia coli* leader peptidase. *Anal Biochem* 255, 66–73.

Zwizinski, C., & Wickner, W. (1980). Purification and characterization of leader (signal) peptidase from *Escherichia coli*. *J Biol Chem* 255, 7973–7977.

Zwizinski, C., Date, T., & Wickner, W. (1981). Leader peptidase is found in both the inner and outer membranes of *Escherichia coli*. *J Biol Chem* 256, 3593–3597.
