
Molecular Microbiology (2000) 37(4), 898–912

# The CitST two-component system regulates the expression of the Mg-citrate transporter in *Bacillus subtilis*

**Hiroki Yamamoto, Masayoshi Murata and Junichi Sekiguchi*  
Department of Applied Biology, Faculty of Textile Science and Technology, Shinshu University, 3-15-1 Tokida, Ueda-shi, Nagano 386-8567, Japan.**

## Summary

*citS* and *citT* genes encoding a new two-component system were identified in the 71° region between the *pel* and *citM* loci on the *Bacillus subtilis* chromosome. *citS*- and *citT*-deficient strains were unable to grow on minimal plates including citrate as a sole carbon source. In addition, a strain deficient in *citM*, which encodes the secondary transporter of the Mg-citrate complex, exhibited the same phenotype on this medium. Northern blot analysis revealed that *citM* was polycistronically transcribed with the downstream *yfIN* gene, and that CitS and CitT were necessary for transcription of the *citM–yfIN* operon. Upon addition of 2 mM citrate to DSM, this operon was strongly induced after the middle of the exponential growth phase in the wild type, but not in the *citST* double null mutant. Moreover, the transcription of this operon was completely repressed in the presence of 1% glucose. We found a sequence exhibiting homology to a catabolite-responsive element (*cre*) in the *citM* promoter region. Glucose repression was lost in *ccpA* and *citM–cre* mutants. From the result of a *citM–promoter* deletion experiment, putative CitT target sequences were found to be located around two regions, from –62 to –74 and from –149 to –189, relative to the *citM* start point. Furthermore, DNase I footprinting assays revealed that these two CitT target regions extended maximally from –36 to –84 and from –168 to –194. From these findings, we concluded that the expression of *citM* is positively regulated by the CitST system and negatively regulated by CcpA.

---

## Introduction

The two-component system is one of the most important devices for signal transduction in microorganisms. Most of these systems consist of a sensor kinase and a response regulator. The sensor kinase senses the environmental or intracellular signals and transmits them to the response regulator through phosphoryl group transfer. The response regulator often binds to the promoter regions of target genes and regulates their transcription through an activation or a repression (Stock et al., 1995).

From the results of the genome sequencing project, it was determined that 36 sensor kinases and 32 response regulators are encoded on the *Bacillus subtilis* chromosome (Kunst et al., 1997; Fabret et al., 1999). They seem to play many important roles in the responses to various environmental conditions. Among them, the *citS–citT* genes (originally defined as *yflR–yflQ*) are located in the 71° region between the *pel* and *citM* loci on the *B.* subtilis chromosome (Kunst et al., 1997) (Fig. 1A). Homology searches of CitS and CitT revealed high similarity to sensor protein CitA (28.3% identity over 540 residues) and response regulator CitB (33.5% identity over 218 residues) of *Klebsiella pneumoniae* (Bott et al., 1995) respectively. The CitA–CitB two-component system of *K.* pneumoniae has been well studied (Bott et al., 1995; Meyer et al., 1997; Kaspar et al., 1999). Null mutations in *citA* and *citB* cause inability to grow anaerobically with citrate as the sole carbon source (Bott et al., 1995). This system senses environmental citrate and regulates the expression of two operons, *citCDEFG* and *citS–oadGAB*. Moreover, CitB and its carboxyl-terminal domain bind to AT-rich target sequences in the promoter regions of these operons (Meyer et al., 1997), and the periplasmic domain of CitA recognizes the dianionic form H-citrate²⁻ but not the Mg-citrate complex (Kaspar et al., 1999).

In Gram-negative bacteria, several transport systems involving C₄-dicarboxylates, a group of tricarboxylic acid (TCA) cycle intermediates, are regulated a sensor kinase and a response regulator, such as *dctB* and *dctD* in *Rhizobium leguminosarum* (Reid and Poole, 1998) and *Rhizobium meliloti* (Jording et al., 1992), *dctS* and *dctR* in *Rhodobacter capsulatus* (Hamblin et al., 1993) and *dcuS* and *dcuR* in *Escherichia coli* (Zientz et al., 1998; Golby et al., 1999). In a Gram-positive bacterium, the *B.* subtilis

Accepted 8 June, 2000. *For correspondence. E-mail jsekigu@gipc.shinshu-u.ac.jp; Tel. (+81) 268 21 5344; Fax (+81) 268 21 5331.*

© 2000 Blackwell Science Ltd

Regulation of Mg-citrate transporter in Bacillus subtilis 899

Fig. 1. Feature map around the citST loci and nucleotide sequence of the citM promoter region.

A. Seven deduced open reading frames (ORFs) and their transcriptional and translational directions are indicated by arrows. The thick bars indicate the regions cloned into plasmid pMUTIN2. The shaded arrows indicate the directions and the regions interrupted by the insertion of a kanamycin or tetracycline resistance gene to construct citS–citT double null mutants, CITSTdd1Km and CITSTdd2Tc respectively. The deduced rho-independent terminators are denoted as follows: T1, ΔG = −22.5 kcal mol⁻¹; T2, ΔG = −24.4 kcal mol⁻¹; T3, ΔG = −25.4 kcal mol⁻¹; T4, ΔG = −20.1 kcal mol⁻¹; and T5, DG = −23.2 kcal mol⁻¹.

B. Nucleotide sequences around the citM promoter region. The nucleotide numbers are the distances with respect to the citM transcription start site (+1), and the region from −228 to +132 corresponds to that from 17750 to 17390 in GSDB, DDBJ, EMBL and NCBI, accession number D86417. The transcriptional start points and directions are shown by hooked arrows (PcitM). The N-terminal 21-amino-acid sequence of CitM is presented in a single-letter code. A potential ribosome binding site (RBS) is denoted by an open box, and a putative T4 terminator is denoted by converging arrowheads. Putative citM promoter –10 and –35 sequences and citM–cre are overlined and underlined respectively. The consensus cre sequence is shown under citM–cre, and matched nucleotides are denoted by vertical bars. The black arrowhead indicates the protected site. The region exhibiting high affinity to CitT is boxed in black, and that exhibiting lower affinity in grey. The striped bars show putative consensus sequences, (A/T)(A/T)CAAA tandem repeats.

dctBSRP (originally defined as ydbEFGH) genes participating in C₄-dicarboxylate transport have been reported by Asai et al. (2000). They revealed that the dctP and dctB genes encode a C₄-dicarboxylate transporter and a solute-binding protein, respectively, and that the dctSR two-component genes are necessary for the activation of dctP expression. This dct system is responsible for the utilization of fumarate or succinate, but not malate.

In B. subtilis, the citM gene, which encodes the secondary transporter of the Mg-citrate complex, is located approximately 1.3 kb downstream of citT (Kunst et al., 1997) (Fig. 1A). citM and its homologue, yxiQ, have been cloned and expressed in E. coli (Boorsma et al., 1996). [Boorsma et al. (1996) termed the homologue citH, whereas Jin and Sonenshein (1994a) have already reported that citH encodes malate dehydrogenase. To avoid confusion, we therefore call it yxiQ in this report.] CitM exhibits 60% sequence identity with YxiQ, and both proteins have 12 predicted transmembrane segments. Boorsma et al. (1996) have reported a remarkable difference between these two transporters. The substrates of CitM and YxiQ are the Mg-citrate complex and the free citrate anion respectively.

We report here that the CitST system is essential for growth under aerobic conditions if citrate is the sole carbon source. Moreover, we report the role of CitS–CitT, which acts as a positive regulator for the transcription of the citM–yflN operon.

Results

CitS and CitT are essential for cell growth on citrate-minimal plates

To determine whether or not CitS and CitT sense environmental citrate, the growth of the citS (sensor

900 H. Yamamoto, M. Murata and J. Sekiguchi

A

![Graph](https://i.imgur.com/1234567.png)

B

![Diagram](https://i.imgur.com/8901234.png)

C

![Image](https://i.imgur.com/5678901.png)

D

![Image](https://i.imgur.com/2345678.png)

Fig. 2. Effect of the $citS-citT$ mutation on transcription of the $citM-yflN$ operon in mSMM.

A. The growth rates of the wild-type strain and CITSTdd2Tc are shown by open squares and filled circles respectively. I to VI are the sampling times (9.5, 11, 12.5, 15, 16 and 18 h respectively) for total RNA preparations.

B. A transcription map of the $citM-yflN$ operon.

C and D. Northern blot analysis was performed with $citM-$ and $yflN$-specific RNA probes respectively. Lanes I to VI correspond to the total RNA samples from cells at phases I to VI in (A) respectively. Each lane contained 10 μg of total RNA. The rRNA positions are indicated by thick bars (23S, 2.93 kb 23S rRNA; 16S, 1.55 kb 16S rRNA).

kinase) or $citT$ (response regulator) mutant on a minimal plate containing citrate as a sole carbon source was monitored (data not shown). Both CITSD (CitS⁻) and CITTD (CitT⁻) were unable to grow under these conditions, which strongly suggested that CitS and CitT sensed environmental citrate and regulated gene expression participating in citrate uptake and/or utilization. In addition, CITSD was unable to grow on this plate containing 1 mM IPTG, suggesting that the Cit⁻ phenotype of CITSD does not result from a polar effect, and that the intact pair of CitS and CitT is necessary for growth under these conditions. CITSTdd2Tc and CITSTdd1Km, double null mutants as to $citS$ and $citT$, showed the same phenotype. However, the growth of SDCST, which requires IPTG induction for $citST$ transcription, was restored on a citrate-minimal plate with 1 mM IPTG, but not without IPTG. Interestingly, CITMD carrying a mutation in $citM$, which is located two genes downstream of $citT$ and encodes the Mg-citrate-specific secondary transporter (Boorsma et al., 1996), was also unable to grow on this plate. These results indicate that the CitST system may regulate the

expression of $citM$ at the transcriptional level. In addition, CITBD carrying a mutation in $citB$, which encodes aconitase, could not grow on this medium, suggesting that citrate is transported by CitM and then catalysed by CitB.

$citM$ gene expression is positively regulated by the CitST system

After the wild-type strain entered glucose-starved conditions in mSMM medium, it began to use citrate, and its multiplication ability was restored (Fig. 2A, phases IV to VI). In contrast, CITSTdd2Tc showed poor restoration of continuous growth (Fig. 2A). To examine $citM$ transcription in this period, total RNAs from both the wild-type and CITSTdd2Tc strains were prepared, and then Northern blot analysis was performed with a $citM$-specific RNA probe (Fig. 2C). A 2.4 kb signal was detected in lanes III to VI, and the maximum transcriptional activity was observed in lane V. This signal seems to correspond to a polycistronic transcript of both $citM$ and its downstream

© 2000 Blackwell Science Ltd, *Molecular Microbiology*, **37**, 898–912

Regulation of Mg-citrate transporter in Bacillus subtilis 901

A
DSM
Wild-type
1 2 3 4 5 6
DSM+2mM Citrate (DSMC)
Wild-type CITSTdd2Tc
1 2 3 4 5 6 1 2 3 4 5 6
2.4 kb
23S
16S

B
Wild-type
CCneo (∆ccpA)
DSMC DSMC+
1% glucose DSMC DSMC+
1% glucose
1 2 3 4 5 1 2 3 4 5 1 2 3 4 5 1 2 3 4 5
2.4 kb
23S
16S

Fig. 3. Effects on *citM* transcription in DSM of the addition of 2 mM citrate (DSMC) and/or 1% glucose. Northern blot analysis was performed with the *citM*-specific probe. Each lane contained 10 μg of total RNAs from cells at *t*~−2~ (lane 1), *t*~−1~ (lane 2), *t*~−0.5~ (lane 3), *t*~0~ (lane 4), *t*~0.5~ (lane 5) and *t*~1~ (lane 6). The rRNA positions are indicated by thick bars (23S, 2.93 kb 23S rRNA; 16S, 1.55 kb 16S rRNA).

A. Transcription of the *citM–yflN* operon in DSM and induction on the addition of 2 mM citrate (DSMC).

B. Effect on *citM* transcription of the addition of 1% glucose. The wild-type and *ccpA* mutant strains were cultured in DSMC with or without 1% glucose.

*yflN*, whose function is unknown. To validate these data, Northern blot analysis was performed with a *yflN*-specific probe. The result indicated that the *yflN*-specific probe also hybridized with the 2.4 kb transcript at phases III to VI (Fig. 2D). Therefore, we concluded that the *citM* and *yflN* loci are transcribed polycistrionally under these conditions (Fig. 2B). Moreover, we confirmed that the β-galactosidase activities of the CITMd (*citM–lacZ*) and YFLNd (*yflN–lacZ*) strains in mSMM medium were consistent with the results of Northern blot analysis (data not shown). On the other hand, no signal corresponding to the co-transcript of *citM* and *yflN* was detected for CITSTdd2Tc (Fig. 2C and D), which strongly suggests that the transcription of this operon is positively regulated by the CitST system.

In Difco sporulation medium (DSM), *citM* was weakly co-transcribed with *yflN* at 0.5 h after the onset of sporulation (*t*~0.5~) in the wild-type strain (Fig. 3A). Moreover, in DSMC medium containing 2 mM trisodium citrate, this operon was strongly transcribed after the middle of the exponential growth phase (Fig. 3A). In contrast, this

induction was completely lost in CITSTdd2Tc (Fig. 3A), suggesting that the CitST system is necessary for the transcription of the *citM–yflN* operon.

### Determination of the transcriptional start points of the *citM–yflN* operon

From the results of Northern blot analysis with the *citM-* and *yflN*-specific probes (Fig. 2C and D), a transcriptional start point seemed to exist in the region upstream of *citM*. To determine the start point, primer extension was performed with a radiolabelled CM-EX2 primer. Two signals (*P*~*citM*~) were detected with total RNAs from wild-type cells at *t*~0.5~ in DSM (Fig. 4). In addition, the transcription of *citM* in mSMM medium started from the same position. Therefore, we determined that the *citM* transcriptional start points are the C (+1 in Fig. 1B) and A nucleotides at positions 72 and 71 bp upstream of the 5′ end of the *citM* gene. On the contrary, no transcriptional signal from *P*~*citM*~ was detected for the *citST* null mutant (Fig. 4), which is consistent with the results of Northern

© 2000 Blackwell Science Ltd, Molecular Microbiology, 37, 898–912

902 H. Yamamoto, M. Murata and J. Sekiguchi

blot analysis. However, the –35 (TTTACT) and –10 (CATACG) regions, at a spacing of 15 bp, do not seem to match any of the promoter consensus sequences (Haldenwang, 1995) (Fig. 1B).

Transcription of the citM–yfIN operon is regulated by CcpA and the citM–cre sequence

From the results of Northern blot analysis of mSMM, citM transcription seemed to be repressed in the presence of glucose. To confirm this, the effect of the addition of 1% glucose to DSMC on citM transcription was examined. As shown in Fig. 3B, the citM–yfIN transcript was completely repressed in the presence of 1% glucose and, thus, we presumed that this glucose repression might result from catabolite repression by means of CcpA (Hueck et al., 1994; Fujita et al., 1995; Hueck and Hillen, 1995; Henkin, 1996). Indeed, this glucose repression was lost in a ccpA null mutant (Fig. 3B). Therefore, we searched for a cis-acting element around the citM promoter region. As a

Fig. 4. Determination of the transcriptional start points of the citM– yfIN operon by primer extension analysis. Total RNAs (40 µg) from B. subtilis 168 (W) and CITSTdd2Tc (ST) in DSM at t₀.₅ and 20 µg from those in mSMM (corresponding to the samples at phase V in Fig. 2A) were hybridized with the CM-EX2 primer. Sequencing reactions (G, A, T and C) were performed using the same primer. The positions of the products are indicated by arrows on the right of the sequence.

Fig. 5. Identification of the CcpA binding site in the citM promoter.
A. Construction of the citM–cre mutant strain, CMUDΔF4CR, as to the amyE locus on the B. subtilis chromosome. The native and mutated citM promoter fragments were fused to the lacZ reporter gene at the translational level.
B and C. β-Galactosidase activities of the wild type and mutant. CMUDΔF4 and CMUDΔF4CR were cultured in DSMC (B) and DSMC containing 1% glucose (C). Open squares and filled circles indicate the β-galactosidase activities of CMUDΔF4 and CMUDΔF4CR respectively.

Regulation of Mg-citrate transporter in Bacillus subtilis 903

|   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |  
904 H. Yamamoto, M. Murata and J. Sekiguchi

CitT CitT-P
0 625 1250 2500 5000 0 625 1250 2500 5000 (nM CitT)

A
free DNA →
complex
complex

B
free DNA →
complex

C
free DNA →

Fig. 7. Gel shift assays showing the binding of h-CitT to the *citM* promoter regions. Three labelled DNA fragments were used for the binding reaction. The F4 (A), F5 (B) and F1D (C) fragments correspond to the regions from −228 to +76, from −98 to +76 and from −62 to +76 relative to the *citM* start point respectively. Before the addition of these fragments, h-CitT was preincubated with (CitT-P) or without acetylphosphate (CitT) as described in Experimental procedures. The retarded species are denoted as complex.

in vitro before the addition of radioactive DNA fragments. Three end-labelled DNA fragments (F1D, +76 to −62; F5, +76 to −98; and F4, +76 to −228 relative to the *citM* start point) were used as radioactive probes. The results in Fig. 7C indicate that the F1D fragment was unable to bind with either unphosphorylated or phosphorylated h-CitT. In contrast, with increasing h-CitT concentrations, the F5 and F4 fragments exhibited decreased mobility in both the presence and the absence of acetylphosphate (Fig. 7A and B). This suggests that both unphosphorylated h-CitT and phosphorylated h-CitT bind to the region from −62 to −98 bp relative to the *citM* start point. However, less protein was needed to form a complex with the *citM* promoter when h-CitT was phosphorylated. This result indicates that phosphorylated h-CitT binds to the *citM* promoter more efficiently. These data were consistent with the results of the *citM* promoter deletion experiment described in the previous section (Fig. 6). When we used the F4 fragment, complex formation was higher than that of the F5 fragment (Fig. 7A and B), suggesting that an additional binding site may exist in a region further upstream of the region between −62 and −98 bp. This idea is supported by the results of the *citM* promoter deletion experiment, as the β-galactosidase activity of CMUDΔF7 was fivefold higher than that of CMUDΔF1 (Fig. 6).

DNase I footprinting assays

To identify CitT binding sites in the *citM* promoter, DNase I footprinting was performed on both strands of the *citM* promoter. As shown in Fig. 8, two protected regions were observed on both strands. One was a strongly protected region from −46 to −84 for the non-coding strand and from −36 to −83 for the coding strand relative to the *citM* start point (Fig. 1B). The other was a weakly protected region from −169 to −194 for the non-coding strand and from −168 to −194 for the coding strand (Fig. 1B). However, both unphosphorylated h-CitT and phosphorylated h-CitT were able to bind to these two regions, but the

Noncoding strand
CitT CitT-P
1 2 3 4 5 6 7 A 1 2 3 4 5 6 7

Coding strand
CitT CitT-P
1 2 3 4 5 6 7 A 1 2 3 4 5 6 7

-46
-84
-169
-194
-168
-83
-36

Fig. 8. DNase I footprinting assays of the *citM* promoter region. The 304 bp EcoRI–SalI fragment (for the non-coding strand) derived from pUCΔF4 and the 224 bp PstI–EcoRI fragment (for the coding strand) from pGCMFP1 were used as DNA probes. Before the addition of these fragments, h-CitT was incubated with and without acetylphosphate (CitT-P and CitT respectively) as described in Experimental procedures. The concentrations of the proteins in lanes 1–7 were 0, 156, 313, 625, 1250, 2500 and 5000 nM respectively. The black arrowhead indicates the protected site. The maximum protected regions are shown by thick bars. The numbers are the distances with respect to the *citM* transcription start site. Lanes G+A contained sequencing ladders.

© 2000 Blackwell Science Ltd, Molecular Microbiology, 37, 898–912

binding ability of phosphorylated h-CitT was greater than that of unphosphorylated h-CitT (Fig. 8). These data are consistent with the results of both the *citM* promoter deletion experiment and the gel shift assays described above, and strongly suggest that phosphorylated CitT binds efficiently to these two regions in the *citM* promoter.

### Discussion

The *citS* and *citT* genes, identified as part of the *B. subtilis* genome project, have been proposed to encode a new two-component system (Kunst et al., 1997). Based on the classification of kinases (as to the sequence around the phosphorylated histidine) and response regulators (as to the sequence of the C-terminal domains), CitS and CitT have been classified into class IV and others-A, respectively, together with DctSR (YdbFG), YufLM and YcbAB (Fabret et al., 1999). This classification may provide us with helpful information for analysing the functions of other unknown systems in *B. subtilis*. Kaspar et al. (1999) have also reported that CitS and CitT are classified in the CitA and CitB subfamilies of *K. pneumoniae* respectively. In the CitAB subfamilies, the other five two-component systems, CitAB and DcuSR of *E. coli*, DctSR and YufLM of *B. subtilis* and CitAB of *Streptomyces coelicolor*, are also included (Kaspar et al., 1999). CitS sensor kinase (542 amino acids, $M_{r} = 59\,859$) has two transmembrane segments in its N-terminal region, which indicates that CitS may be a membrane-anchored protein and that the periplasmic domain between two transmembrane segments may recognize the environmental citrate or Mg-citrate complex. Indeed, the periplasmic domain of CitA of *K. pneumoniae* functions as a highly specific citrate receptor and binds citrate, but not the Mg-citrate complex (Kaspar et al., 1999). In addition, a putative autophosphorylation histidine residue (His-339) is conserved in CitS as well as the other sensor kinases in the CitA subfamily. Moreover, in CitT response regulator (226 amino acids, $M_{r} = 25392$), a presumed phosphorylation site (Asp-54) in the N-terminal region and a putative helix-turn-helix motif in the C-terminal region are highly conserved (Kaspar et al., 1999).

When the *citS*- and *citT*-deficient strains were cultured in DSM, we did not observe any significant differences in the phenotypes compared with the wild-type strain, i.e. the rates of growth, sporulation and germination (data not shown). However, these single mutants and the double mutant with null mutations in both genes were unable to grow on a minimal medium including citrate as a sole carbon source. In addition, the *citM*-deficient strain showed the same phenotype, suggesting the possibility that another citrate transporter gene, such as *yxiQ*, may not function under these conditions. Boorsma et al. (1996) have reported that citrate uptake by YxiQ is inhibited in

Regulation of Mg-citrate transporter in Bacillus subtilis 905

the presence of $\mathrm{Mg}^{2+}$. Thus, its expression might also be repressed in the presence of a large amount of $\mathrm{Mg}^{2+}$.

After the wild-type strain entered glucose-starved conditions in mSMM medium, it began to use citrate, and its multiplication ability was restored, but this was not seen for the *citST* null mutant (Fig. 2A). Moreover, in DSMC medium containing trisodium citrate, the *citM–yflN* operon was strongly induced after the middle of the exponential growth phase (Fig. 3A). In contrast, this induction was completely lost in the *citST* null mutant (Fig. 3A), which suggested that the CitST system was necessary for the transcription of this operon. However, the result of primer extension analysis indicated that –35 and –10 regions of the *citM* promoter were not very similar to the $\sigma^{\mathrm{A}}$ consensus sequences (Haldenwang, 1995). The other sigma factor mutants ($\sigma^{\mathrm{D}}, \sigma^{\mathrm{L}}$ and $\sigma^{\mathrm{H}}$) did not affect *citM* transcription (data not shown). Therefore, we presume that the *citM* promoter may be recognized by $\sigma^{\mathrm{A}}$ or ECF sigma factors. Further experiments are needed to determine which sigma factor directs *citM* transcription.

The *citM–yflN* operon was completely repressed in the presence of glucose. We found a *cre*-like DNA sequence, TGTAAGCGGATTTCA, located just upstream of the *citM* ribosome binding site (Fig. 1B). Finally, we defined that the *citM–cre* sequence functions as an operator for the *citM–yflN* operon, and that CcpA probably binds to the operator in the presence of glucose, and thereby represses the transcription of this operon. Asai et al. (2000) have reported that the expression of *dctP*, which encodes a $C_{4}$-dicarboxylate transport protein and is activated by the DctSR two-component system, was repressed in the presence of glucose, and that this glucose repression was lost in a *ccpA* mutant. Moreover, they found a *cre*-like sequence in the –10 region of the *dctP* promoter, which suggests that both citrate uptake and $C_{4}$-dicarboxylate uptake systems are controlled through a very similar mechanism in *B. subtilis*.

Most of the Krebs cycle enzymes are expressed at the end of the exponential growth phase, e.g. aconitase (*citB*) (Dingman et al., 1987), isocitrate dehydrogenase (*citC*) (Jin and Sonenshein, 1994a,b), fumarase (*citG*) (Miles and Guest, 1985; Price et al., 1989; Tatti et al., 1989) and malate dehydrogenase (*citH*) and citrate synthases (*citZ*) (Jin and Sonenshein, 1994a). The TCA cycle is necessary for sporulation, at least when sporulation is induced by nutrient depletion (Ireton et al., 1995; Craig et al., 1997; Jin et al., 1997; Alén and Sonenshein, 1999; Matsuno et al., 1999). The period of *citM* expression seems to correspond to that of TCA cycle activation. When glucose in the culture medium has been consumed, cells may begin to use environmental citrate as a carbon source, and citrate transported via CitM may be catalysed by CitB and metabolized through the TCA cycle. This hypothesis led us to presume that *citB* might be one of the regulon

© 2000 Blackwell Science Ltd, *Molecular Microbiology*, **37**, 898–912

genes of the CitST system. We examined its effect on *citB* transcription, but we did not detect any significant differences between the wild-type strain and the *citST* mutant (data not shown), suggesting that *citB* transcription is not dependent on this system.

A homology search with YflN revealed some amino acid sequence similarity to YybB and YqgX of *B. subtilis*, whose functions are unknown (Kunst et al., 1997). CitM is a hydrophobic protein consisting of 12 membrane-spanning domains (Boorsma et al., 1996), but YflN has neither this feature nor any obvious sequence motifs. As mentioned above, *yflN* is co-transcribed with *citM*, but a *yflN*-deficient strain was able to grow normally on the minimal medium containing citrate as a sole carbon source, suggesting that YflN is not required for citrate uptake or utilization.

The results of the promoter deletion experiment indicated that putative CitT target sites were present around, at least, two regions from –62 to –74 and from –149 to –189 bp (Fig. 6). This was supported by the gel shift assay data (Fig. 7). Complex formation of the F4 fragment (including two target regions: –62 to –74 and –149 to –189) was more efficient than that of the F5 fragment (including a target region: –62 to –74). In addition, two kinds of complex were observed in the case of the F4 fragment, which suggests that some higher order structure of the DNA–CitT complex may be involved in the activation. The DNA-binding ability of phosphorylated CitT was approximately threefold higher than that of unphosphorylated CitT (Fig. 7). We presume that this low affinity mainly results from using acetylphosphate as a phosphoryl group donor. Therefore, only a small portion of CitT might be phosphorylated by acetylphosphate treatment.

DNase I footprinting assays revealed two protected regions extending maximally from –36 to –84 and from –168 to –194 bp (Figs 8 and 1B). The –36 to –84 region was strongly protected from DNase I degradation. In particular, as a feature of this region, a very high A+T content of 82% was observed, in contrast to the average A+T ratio of 56.5% for the whole *B. subtilis* genome (Kunst et al., 1997). The –168 to –194 region was only protected at higher CitT concentrations. These results suggest that these regions are the target sites of both unphosphorylated and phosphorylated CitT. Similar results with two target regions have been reported on the *phoD* (Eder et al., 1999) and *srfA* (Nakano and Zuber, 1993; Roggiani and Dubnau, 1993) promoters in *B. subtilis*. Moreover, we found tandemly repeated hexanucleotide sequences, (A/T)(A/T)CAAA, in these two regions (Fig. 1B). In the *B. subtilis* Pho regulon genes, which are regulated by the PhoP–PhoR two-component system, it has been reported that tandemly repeated hexanucleotide sequences are conserved in their promoter

regions and are essential for the Pho response (Liu and Hulett, 1997; 1998; Birkey et al., 1998; Liu et al., 1998a,b; Qi and Hulett, 1998a,b; Eder et al., 1999). Moreover, in the *B. subtilis* DctSR system, Asai et al. (2000) have reported that two tandem repeat sequences in the *dctP* promoter region are important for the induction of *dctP* transcription. Therefore, we presume that the repeated sequences in the *citM* promoter may be consensus sequences recognized by CitT. This hypothesis still remains to be resolved.

In this study, we have demonstrated that the CitST system is essential for the response to the environmental Mg-citrate complex, and that *citM* transcription was positively regulated by this system and negatively regulated by CcpA. Based on these findings and the knowledge of other two-component systems (Stock et al., 1995; Fabret et al., 1999; Kaspar et al., 1999), a conceivable summary of the regulatory mechanism for the *citM* operon by the CitST system is presented in Fig. 9. The environmental Mg-citrate complex is recognized by the N-terminal periplasmic domain of CitS, and its C-terminal kinase domain is autophosphorylated, then the phosphoryl group is transferred to the N-terminal receiver domain of CitT. The phosphorylated CitT specifically binds to two regions in the *citM* promoter to activate the transcription of the *citM–yflN* operon. However, in the presence of glucose, CcpA binds to the *citM–cre* sequence to repress the transcription of the *citM* operon. We are currently attempting further identification of the regulon genes controlled by the CitST system.

### Experimental procedures

#### Bacterial strains and culture conditions

The bacterial strains used in this study are listed in Table 1. The bacterial strains were cultured in LB medium (5 g l⁻¹ yeast extract, 10 g l⁻¹ polypeptone and 5 g l⁻¹ NaCl, pH 7.2) at 37°C. If necessary, ampicillin, erythromycin, tetracycline or kanamycin was added to a final concentration of 50, 0.3, 10 or 10 μg ml⁻¹ respectively. *B. subtilis* strains were grown in Difco sporulation medium (DSM) (Schaeffer et al., 1965), Spizizen’s minimal medium (SMM) (Anagnostopoulos and Spizizen, 1961) or modified SMM (mSMM). SMM and mSMM were made by adding glucose to a final concentration (w/v) of 0.5% and 0.05%, respectively, to Spizizen’s minimal salts (SMS; 2 g l⁻¹ (NH₄)₂SO₄, 14 g l⁻¹ K₂HPO₄, 6 g l⁻¹ KH₂PO₄, 1 g l⁻¹ trisodium citrate·2H₂O and 0.2 g l⁻¹ MgSO₄·7H₂O). When *B. subtilis* strains were grown in minimal medium, L-tryptophan was added to a final concentration of 50 μg ml⁻¹.

#### Plasmid and mutant construction

The plasmids and primers used in this study are listed in Tables 2 and 3 respectively. Derivatives of pMUTIN2 (Vagner et al., 1998) and pGEM-3Zf(+) were used to

© 2000 Blackwell Science Ltd, *Molecular Microbiology*, **37**, 898–912

Regulation of Mg-citrate transporter in Bacillus subtilis 907

Fig. 9. Regulatory mechanism for *citM* expression resulting from the CitST system. Proteins are indicated by ovals. The black and grey bars indicate the CitT target sites in the *citM* promoter region. The nucleotide numbers are the distances with respect to the *citM* start point (*P*<sub>citM</sub>, +1). The *cre* and ribosome binding sites are indicated by striped and open boxes respectively.

Table 1. Bacterial strains used in this study.

| Strain           | Relevant genotype                                                                 | Source or reference         |
|------------------|-----------------------------------------------------------------------------------|------------------------------|
| Bacillus subtilis |                                                                                   |                              |
| 168              | *trpC2*                                                                           | S. D. Ehrlich                |
| CITSD            | *trpC2 citS::pMCITS*                                                             | This study                  |
| CITTD            | *trpC2 citT::pMCITT*                                                             | This study                  |
| CITMD            | *trpC2 citM::pMCITM*                                                             | This study                  |
| YFLNd            | *trpC2 yflN::pMYFLN*                                                             | This study                  |
| CITBd            | *trpC2 citB::pMCITB*                                                             | This study                  |
| CITSTdd1Km       | *trpC2 citST::kan*                                                               | This study                  |
| CITSTdd2Tc       | *trpC2 citST::tet*                                                               | This study                  |
| SDCST            | *trpC2 citS::pMSDCST (Pspac–citST)*                                              | This study                  |
| 327SD1           | *purB his-1 smo-1 sigD::tet*                                                     | Rashid *et al.* (1995)      |
| 168SD1           | *trpC2 sigD::tet*                                                                | 327SD1 → 168                 |
| SIGLd            | *trpC2 sigL::pMSIGL*                                                            | This study                  |
| SIGHd            | *trpC2 sigH::pMSIGH*                                                            | This study                  |
| 1A1              | *trpC2 ccpA::neo*                                                                | Y. Fujita                    |
| CCneo            | *trpC2 ccpA::neo*                                                                | 1A1 → 168                    |
| AMLZ             | *trpC2 amyE::(cat lacZ)*                                                         | Yamamoto *et al.* (1999)     |
| CMUDΔF2          | *trpC2 amyE:: (PcitM (−39 to +76)<sup>a</sup> citM′−′lacZ (cat)*                 | pDHΔF2 → 168                 |
| CMUDΔF1D         | *trpC2 amyE:: (PcitM (−62 to +76)<sup>a</sup> citM′−′lacZ (cat)*                 | pDHΔF1D → 168                |
| CMUDΔF6          | *trpC2 amyE:: (PcitM (−74 to +76)<sup>a</sup> citM′−′lacZ (cat)*                 | pDHΔF6 → 168                 |
| CMUDΔF1Sau       | *trpC2 amyE:: (PcitM (−86 to +76)<sup>a</sup> citM′−′lacZ (cat)*                 | pDHΔF1Sau → 168              |
| CMUDΔF5          | *trpC2 amyE:: (PcitM (−98 to +76)<sup>a</sup> citM′−′lacZ (cat)*                 | pDHΔF5 → 168                 |
| CMUDΔF1S         | *trpC2 amyE:: (PcitM (−110 to +76)<sup>a</sup> citM′−′lacZ (cat)*                | pDHΔF1S → 168                |
| CMUDΔF1          | *trpC2 amyE:: (PcitM (−149 to +76)<sup>a</sup> citM′−′lacZ (cat)*                | pDHΔF1 → 168                 |
| CMUDΔF7          | *trpC2 amyE:: (PcitM (−189 to +76)<sup>a</sup> citM′−′lacZ (cat)*                | pDHΔF7 → 168                 |
| CMUDΔF4          | *trpC2 amyE:: (PcitM (−228 to +76)<sup>a</sup> citM′−′lacZ (cat)*                | pDHΔF4 → 168                 |
| STΔF4            | *trpC2 amyE:: (PcitM (−228 to +76)<sup>a</sup> citM′−′lacZ (cat) citST::tet*      | pDHΔF4 → CITSTdd2Tc          |
| CMUDΔF4CR        | *trpC2 amyE:: (PcitM (−228 to +76 region carrying a CG to AT dinucleotide change at +47 and +48 relative to the citM start point)<sup>a</sup> citM′−′lacZ (cat)* | pDHΔF4CR → 168               |
| Escherichia coli |                                                                                   |                              |
| JM109            | *recA1 supE44 endA1 hsdR17 gyrA96 relA1 thi-1 Δ(lac-proAB)/F' [traD36 proAB<sup>+</sup> lacI<sup>q</sup> lacZΔM15]* | Takara                      |
| C600             | *supE44 hsdR17 thi-1 thr-1 leuB6 lacY1 tonA21*                                   | Laboratory stock            |
| M15              | *Na<sup>S</sup> Str<sup>S</sup> Rif<sup>S</sup> lac ara gal mtl F<sup>−</sup> recA<sup>+</sup> uvr<sup>+</sup>* | Qiagen                      |

a. Nucleotide numbers correspond to the *citM* promoter regions relative to the *citM* start point.

© 2000 Blackwell Science Ltd, *Molecular Microbiology*, **37**, 898–912

Table 2. Plasmids used in this study.

| Plasmid          | Relevant genotype                                                                 | Source or reference         |
|------------------|----------------------------------------------------------------------------------|----------------------------|
| pMUTIN2          | erm bla lacZ lacI Pspac                                                          | Vagner et al. (1998)       |
| pMUTIN4          | erm bla lacZ lacI Pspac                                                          | Vagner et al. (1998)       |
| pGEM-3Zf(+)      | bla lacZ                                                                         | Promega                    |
| pUC119           | bla lacZ                                                                         | Takara                     |
| pDG782           | bla kan                                                                          | BGSC                       |
| pDG1513          | bla tet                                                                          | BGSC                       |
| pQE30            | bla                                                                              | Qiagen                     |
| pREP4            | lacI kan                                                                         | Qiagen                     |
| pUCCT            | bla citT                                                                         | This study                 |
| pQECT            | bla citT                                                                         | This study                 |
| pMCITS           | pMUTIN2::ΔcitS                                                                   | This study                 |
| pMCITT           | pMUTIN2::ΔcitT                                                                   | This study                 |
| pMCITM           | pMUTIN2::ΔcitM                                                                   | This study                 |
| pMYFLN           | pMUTIN2::ΔyflN                                                                   | This study                 |
| pMCITB           | pMUTIN2::ΔcitB                                                                   | This study                 |
| pMSDCST          | pMUTIN4::ΔcitS (containing the citS SD sequence)                                | This study                 |
| pMSIGL           | pMUTIN2::ΔsigL                                                                   | This study                 |
| pMSIGH           | pMUTIN2::ΔsigH                                                                   | This study                 |
| pGCITS           | pGEM-3Zf(+):ΔcitS                                                                | This study                 |
| pGCITT           | pGEM-3Zf(+):ΔcitT                                                                | This study                 |
| pGCITM           | pGEM-3Zf(+):ΔcitM                                                                | This study                 |
| pGYFLN           | pGEM-3Zf(+):ΔyflN                                                                | This study                 |
| pGCITB           | pGEM-3Zf(+):ΔcitB                                                                | This study                 |
| pGSIGL           | pGEM-3Zf(+):ΔsigL                                                                | This study                 |
| pGSIGH           | pGEM-3Zf(+):ΔsigH                                                                | This study                 |
| pGCMFP1          | pGEM-3Zf(+) with a 224 bp citM promoter region                                  | This study                 |
| pCSKm25          | bla kan citS fragment in pDG782                                                  | This study                 |
| pCSTKm1          | bla citST::kan                                                                   | This study                 |
| pCSTTc10         | bla citST::tet                                                                   | This study                 |
| pUCAF6           | pUC119 with a 150 bp citM promoter region                                        | This study                 |
| pUCAF5           | pUC119 with a 174 bp citM promoter region                                        | This study                 |
| pUCAF4           | pUC119 with a 304 bp citM promoter region                                        | This study                 |
| pUCAF4CR         | pUC119 with a 304 bp citM promoter region (carrying a CG to AT dinucleotide change at +47 and +48 relative to the citM start point) | This study                 |
| pDHAFBLZ         | bla amyE::(lacZ cat)                                                             | Yamamoto et al. (1999)     |
| pDHAΔF2          | pDHAFBLZ with a 115 bp citM promoter region fused to lacZ                        | This study                 |
| pDHAΔF1D         | pDHAFBLZ with a 138 bp citM promoter region fused to lacZ                        | This study                 |
| pDHAΔF6          | pDHAFBLZ with a 150 bp citM promoter region fused to lacZ                        | This study                 |
| pDHAΔF1Sau       | pDHAFBLZ with a 162 bp citM promoter region fused to lacZ                        | This study                 |
| pDHAΔF5          | pDHAFBLZ with a 174 bp citM promoter region fused to lacZ                        | This study                 |
| pDHAΔF1S         | pDHAFBLZ with a 186 bp citM promoter region fused to lacZ                        | This study                 |
| pDHAΔF1          | pDHAFBLZ with a 225 bp citM promoter region fused to lacZ                        | This study                 |
| pDHAΔF7          | pDHAFBLZ with a 265 bp citM promoter region fused to lacZ                        | This study                 |
| pDHAΔF4          | pDHAFBLZ with a 304 bp citM promoter region fused to lacZ                        | This study                 |
| pDHAΔF4CR        | pDHAFBLZ with a 304 bp citM promoter region fused to lacZ (carrying a CG to AT dinucleotide change at +47 and +48 relative to the citM start point) | This study                 |

a. BGSC, Bacillus Genetic Stock Center, Ohio State University, USA.

construct mutants of *B. subtilis* and to prepare gene-specific RNA probes respectively. For the construction of gene-disrupted mutants, pMUTIN2 derivatives containing an internal region of each gene were used. For example, a citS-internal DNA fragment amplified with primers CS-4H and CS-5B was digested with HindIII and BamHI. Then, the digested fragment was cloned into the HindIII and BamHI sites of pMUTIN2 to obtain pMCITS. In the same way, to obtain plasmids pMCITT, pMCITM, pMYFLN, pMCITB, pMSIGL and pMSIGH, internal fragments of citT, citM, yflN, citB, sigL and sigH were amplified with primers CT-1H and CT-2B, CM-1H and CM-2B, LN-1H and LN-2B, CIB-1H and CIB-2B, SL-HF and SL-BR, and SH-HF and SH-BR respectively (Fig. 1 and Table 3). To prepare a digoxigenin (DIG)-labelled RNA probe, HindIII- and BamHI-digested polymerase chain reaction (PCR) fragments were cloned

into pGEM-3Zf(+) to generate plasmids pGCITS, pGCITT, pGCITM, pGYFLN, pGCITB, pGSIGL and pGSIGH. Conventional transformation of *B. subtilis* was performed according to the procedure of Anagnostopoulos and Spizizen (1961). CITSD, CITTd, CITMd, YFLNd, CITBd, SIGLd and SIGHd were constructed by means of Campbell-type integration with plasmids prepared from *E. coli* C600 cells harbouring pMCITS, pMCITT, pMCITM, pMYFLN, pMCITB, pMSIGL and pMSIGH respectively. Proper integration of the plasmids was confirmed by Southern hybridization. To construct *B. subtilis* 168SD1 and CCneo, *B. subtilis* 168 was transformed with chromosomal DNAs prepared from 327SD1 and 1A1 respectively.

For the construction of an IPTG-inducible P<sub>spac</sub>-citST strain, SDCST, a 436 bp fragment was amplified by PCR with the CS-3H and CS-5B primers, followed by HindIII and

Table 3. Primers used in this study.

| Primer    | Sequence (5′ → 3′) or source                                                                 | Location<sup>a</sup> | Restriction site |
|-----------|------------------------------------------------------------------------------------------|---------------------|------------------|
| CS-3H     | GCCGAAAGCTTCTAGGTAGAAAGGACGG                                                             | 20898               | HindIII          |
| CS-4H     | GCCGAAAGCTTTCAATTGGTGTCGTGACC                                                            | 20800               | HindIII          |
| CS-5B     | CGCGGATCCACTCACTCTTCGTCCATCC                                                             | 20497               | BamHI            |
| CT-1H     | GCCGAAAGCTTCGCAAAGAGACATTGGC                                                             | 19168               | HindIII          |
| CT-2B     | CGCGGATCCGCTGTCTTACACCGTTC                                                              | 18867               | BamHI            |
| cS1StuF   | GCGAGGCCCTTTCAGCATACACAGGGAG                                                            | 20769               | Stul             |
| cS2BR     | GCCGGATCCGATCACTTGGCTGTATC                                                               | 20444               | BamHI            |
| cT3PF     | GCCGCCCTGCAGGCGAGGTGCTGCTTCAG                                                           | 18939               | PstI             |
| cT4BR     | GCCGGATCCTTGACACAAGGTACTCGG                                                             | 18703               | BamHI            |
| CM-1H     | GCCGAAAGCTTCCGTGTTGATCCAATGG                                                            | 17202               | HindIII          |
| CM-2B     | CGCGGATCCCCACCATACAAAGCATC                                                              | 16897               | BamHI            |
| LN-1H     | GCCGAAAGCTTGACTTCAGTCAGAAGCG                                                            | 16072               | HindIII          |
| LN-2B     | CGCGGATCCCCTTTACCTGTTACATAAGG                                                           | 15774               | BamHI            |
| CIB-1H    | GCCGAAAGCTTAAAGACGTTTCCAAGCG                                                            |                     | HindIII          |
| CIB-2B    | GCGCGGATCCGAGATCTACTCGC                                                                |                     | BamHI            |
| SL-HF     | GCCGAAAGCTTCTGCTTGGCTATCATTCG                                                          |                     | HindIII          |
| SL-BR     | GCGCGGATCATATCGAGTGACTGCTG                                                             |                     | BamHI            |
| SH-HF     | GCGCAAGCTTAACAAAAGACAGTTTTGCC                                                          |                     | HindIII          |
| SH-BR     | GCGCGGATCCAATATCCTCTCTGTCCG                                                            |                     | BamHI            |
| CM-EX2    | AGCGCAAACACAATCGG                                                                      | 17359               |                  |
| CMUD-creF | GAGAAAAAAATGTAAGATGATTCATTTAAGGGG                                                      | 17459               |                  |
| CMUD-creR | CCCCCTTAATGAATCATCTTACATTTTTTCTC                                                       | 17492               |                  |
| CMUD-RS   | GCCGGTCGACTAACACATCCATTCCCC                                                            | 17462               | Sall             |
| CMUD-F1   | GCCGGAATTCCTCAAAGCATCATGACAGG                                                          | 17654               | EcoRI            |
| CMUD-F2   | GCCGGAATTCCCAATTATTACTTGGCGG                                                           | 17543               | EcoRI            |
| CMUD-F4   | GCCGGAATTCCTAAACGAACAGGACTGGG                                                          | 17734               | EcoRI            |
| CMUD-F5   | GCCGGAATTCATGCCGAAAATTGATCAAAA                                                         | 17600               | EcoRI            |
| CMUD-F6   | GCCGGAATTCCAAACGGTTTTAAAAATTAA                                                          | 17575               | EcoRI            |
| CMUD-F7   | GCCGGAATTCGGGAAACGAAACGGTTTT                                                           | 17693               | EcoRI            |
| CMUD-F8   | GCCGGAATTCAGAAGAGCAATCTGCTTTT                                                          | 17673               | EcoRI            |
| CMFP-PF1  | GCCGCTGCAGTAAACGAAACAGGACTGGG                                                          | 17734               | PstI             |
| CMFP-ER1  | GCGCGAATTCGTATGGGAAAACCAACC                                                            | 17543               | EcoRI            |
| CTP-BF    | GCGCGGATCCATTCACATCGCGATTGCG                                                           | 19269               | BamHI            |
| CTP-HR    | GCCGAAAGCTTGGATTATAGATTTTTTCATATC                                                      | 18609               | HindIII          |
| -21M13    | Universal primer (Takara)                                                              |                     |                  |
| M13RV     | Universal primer (Takara)                                                              |                     |                  |

a. The 3′ end positions of primers corresponding to the nucleotide numbers in GSDB, DDBJ, EMBL and NCBI, accession number D86417, are indicated.

Additional sequences and restriction sites that do not correspond to the sequences near the *citST* loci are italicized and underlined respectively.

BamHI digestion. Then, the DNA fragment containing the Shine-Dalgarno (SD) sequence and the 5′ flanking region of *citS* was cloned into the corresponding sites of pMUTIN4 to generate pMSDCST. *B. subtilis* 168 was transformed with this plasmid to obtain SDCST, in which *citST* transcription is under the control of the *P<sub>spac</sub>* promoter at the *citST* loci on the *B. subtilis* chromosome.

To construct a double null mutant as to *citS* and *citT*, PCR was performed to amplify a 0.36 kb region in the *citS* gene with primers cS1StuF and cS2BR. The resulting fragment was digested with Stul and BamHI, and then cloned into the Stul and BglII sites of pDG782 to generate pCSKm25. Then, a 0.27 kb fragment in the *citT* gene amplified with primers cT3PF and cT4BR was ligated to the PstI and BamHI sites of pCSKm25 to obtain pCSTKm1. The 2.1 kb tetracycline resistance (*Tc*) gene, which was obtained by digesting pDG1513 with Clal, was cloned into the Clal site of pCSTKm1 to generate pCSTTc10. The direction of the *Tc* gene was confirmed by double digestion with PstI and Scal. *B. subtilis* 168 was transformed with linearized pCSTKm1 and pCSTTc10 to obtain CITSTdd1Km and CITSTdd2Tc respectively. Proper construction by means of a double cross-over recombination event was confirmed by PCR.

For overproduction and purification of response regulator CitT in *E. coli*, expression plasmid pQECT was constructed. A 700 bp PCR fragment obtained with primers CTP-BF and CTP-HR was digested with BamHI and HindIII, followed by ligation into the corresponding sites of pUC119. The DNA solution was used for the transformation of *E. coli* JM109 cells, and then the resulting plasmid, pUCCT, was sequenced and digested with BamHI and HindIII. A 0.7 kb DNA fragment was ligated to the BamHI and HindIII sites of a histidine-tagged plasmid, pQE30, and then used for the transformation of *E. coli* M15 (pREP4). Consequently, the synthesis of the histidine-tagged CitT (h-CitT) was controlled by IPTG induction.

To construct translational fusion strains to the gradually deleted *citM*-promoter and the *lacZ* reporter gene at the *amyE* locus on the *B. subtilis* chromosome, pDHAFBLZ was used (Yamamoto et al., 1999). The six PCR products obtained with a reverse primer, CMUD-RS, and forward primers, CMUD-F2, CMUD-F6, CMUD-F1, CMUD-F7 and CMUD-F4, were digested with EcoRI and SalI. Then, the fragments were cloned into the EcoRI-SalI sites of pDHAFBLZ to generate pDHΔF2, pDHΔF6, pDHΔF5, pDHΔF1, pDHΔF7 and pDHΔF4 respectively. A 138 bp

© 2000 Blackwell Science Ltd, *Molecular Microbiology*, 37, 898–912

Dral–SalI fragment derived from pDHΔF6 and a 186 bp Smal–SalI fragment from pDHΔF1 were cloned into the Smal–SalI sites of pDHAFBLZ to obtain pDHΔF1D and pDHΔF1S, respectively, and a 162 bp Sau3AI–SalI fragment from pDHΔF5 was cloned into the BamHI–SalI sites to generate pDHΔF1Sau. Finally, these pDHAFBLZ derivatives were linearized and transformed by means of a double cross-over event at the amyE locus to obtain B. subtilis AMLZ, CMUDΔF2, CMUDΔF1D, CMUDΔF6, CMUDΔF1Sau, CMUDΔF5, CMUDΔF1S, CMUDΔF1, CMUDΔF7, CMUDΔF4 and STΔF4.

For the gel shift and DNase I footprinting assays, EcoRI–SalI fragments derived from pDHΔF6, pDHΔF5 and pDHΔF4 were subcloned into the corresponding sites of pUC119 to obtain pUCΔF6, pUCΔF5 and pUCΔF4 respectively. To prepare a coding strand probe for the DNase I footprinting assays, a 224 bp fragment in the citM promoter region was amplified with CMFP-PF1 and CMFP-ER1 as primers and B. subtilis chromosomal DNA as a template. The amplified fragment was digested with PstI and EcoRI, and then cloned into the corresponding sites of pGEM-3Zf(+) to generate pGCMFP1.

### Site-directed mutagenesis

To introduce mutations into the citM–cre sequence, site-directed mutagenesis was performed with a QuickChange site-directed mutagenesis kit (Stratagene) according to the manufacturer’s instructions. PCR was performed with CMUD-creF and CMUD-creR as primers and pUCΔF4 as a template to generate pUCΔF4CR. Then, pUCΔF4CR, with a GC to AT dinucleotide change at the centre of the citM–cre sequence, was sequenced and digested with EcoRI and SalI. The mutated citM promoter region was subcloned into pDHAFBLZ to generate pDHΔF4CR. Then, pDHΔF4CR was linearized and transformed by means of a double cross-over event at the amyE locus to obtain B. subtilis CMUDΔF4CR.

### Overexpression and purification of CitT

E. coli M15 (pREP4, pQECT) was cultured in 200 ml of LB medium containing ampicillin (100 μg ml⁻¹), kanamycin (25 μg ml⁻¹) and glucose (1%) to a cell density (OD₆₀₀) of approximately 0.8 at 37°C. Then, 1 mM IPTG was added to the culture to the final concentration, followed by further incubation for 1 h. The culture was centrifuged, and the pellet was suspended in 25 ml of a 10 mM imidazole–NPB solution (10 mM imidazole and 1 M NaCl in 20 mM sodium phosphate buffer, pH 7.4). After ultrasonication, the suspension was centrifuged, and the supernatant was filtered through a 0.45 μm membrane filter (Nalgene), followed by application to a HiTrap chelating column (1 ml; Pharmacia). The column was washed with 20 ml of the 10 mM imidazole–NPB solution, and then the h-CitT protein was eluted with the NPB solution containing a stepwise gradient of imidazole, from 60 to 300 mM.

### Northern blot and primer extension analyses

RNA preparation and probe labelling were performed as described previously (Yamamoto et al., 1999). Northern blot analysis of RNAs fractionated by electrophoresis in agarose–formaldehyde gels was performed as described by Sambrook et al. (1989). Probe labelling was performed with a DIG RNA labelling kit (Boehringer Mannheim) according to the manufacturer’s instructions with some minor modifications. Briefly, the internal regions cloned into pGEM-3Zf(+) derivatives were amplified by PCR with –21M13 and M13RV as primers. The amplified fragments were digested with HindIII and then used as templates for in vitro run-off transcription with T7 RNA polymerase. Hybridization and detection were performed with a DIG luminescent detection kit (Boehringer Mannheim) according to the manufacturer’s instructions.

Primer extension analysis was performed as described previously (Kuroda and Sekiguchi, 1993) with an end-labelled CM-EX2 primer.

### Gel shift assays

A 138 bp Dral–SalI fragment derived from pUCΔF6, a 174 bp EcoRI–SalI fragment from pUCΔF5 and a 304 bp EcoRI–SalI fragment from pUCΔF4 were dephosphorylated with calf intestine alkaline phosphatase (Takara) and then purified from 2% agarose gel with an UltraClean GelSpin DNA purification kit (MO BIO). Then, these fragments were labelled with [γ-³²P]-ATP using T4 polynucleotide kinase (Nippon Gene). The in vitro phosphorylation reaction of h-CitT was performed with acetylphosphate. The binding reaction and native PAGE were carried out as described by Meyer et al. (1997), except for the concentration of h-CitT. A dilution series of h-CitT (3.125–25 μM) in buffer B (50 mM Tris-HCl, pH 7.5, 50 mM KCl, 10 mM MgCl₂, 0.1 mM EDTA and 10% glycerol) was incubated for 1 h at 37°C in a final volume of 20 μl with either 50 mM acetylphosphate or buffer B. The binding mixtures contained 2 μg of poly-(dl–dC) as the competitor DNA, 1 nM probe and 4 μl of different h-CitT preparations in buffer B (total volume 20 μl). Therefore, the final concentrations of h-CitT ranged from 0.625 to 5 μM. The mixtures were incubated for 20 min at room temperature. After the binding reaction, 2 μl of a 0.1% xylene cyanol solution was added, and the samples were loaded onto a 10% native polyacrylamide gel.

### β-Galactosidase assay

Cells were precultured on DSM agar plates without antibiotics at 30°C for 10 h. They were then suspended in 4 ml of liquid DSM, followed by inoculation into appropriate media at a cell density of 0.01 OD₆₀₀. For culture in minimal medium, B. subtilis strains were precultured in 5 ml of SMM at 37°C for 12 h with vigorous shaking, and then cells were inoculated into 50 ml of fresh mSMM at a cell density of 0.01 OD₆₀₀. After shaking at 37°C, samples were withdrawn at various intervals to assay β-galactosidase activity. Measurement and calculation of β-galactosidase activity (expressed as units mg⁻¹ protein or OD₆₀₀) were carried out as described by Shimotsu and Henner (1986). One unit of β-galactosidase activity was defined as the amount of enzyme necessary to release 1 nmol of 2-nitrophenol from ONPG in 1 min at 28°C.

DNase I footprinting assays

As coding and non-coding strand probes of the *citM* promoter region, a 224 bp *PstI–EcoRI* fragment derived from pGCMFP1 and a 304 bp *EcoRI–SalI* fragment from pUCΔF4 were purified from a 2% agarose gel with an UltraClean GelSpin DNA purification kit (MO BIO). These fragments were labelled with [α-³²P]-dATP using the Klenow fragment of DNA polymerase I (Takara). Unincorporated nucleotides were removed using Microcon 100 (Amicon). h-CitT was phosphorylated in buffer C [40 mM Tris-HCl, pH 7.2, 6 mM MgCl₂, 1 mM dithiothreitol (DTT), 50 mM KCl, 0.5 mM CaCl₂ and 10% (v/v) glycerol] by incubation at 37°C for 1 h with acetylphosphate at a final concentration of 50 mM. Then, the reaction mixture was changed to buffer C using Microcon 10 (Amicon) to remove acetylphosphate. Constant amounts of the probe (2 nM) and 3 μg of salmon sperm DNA as the competitor DNA were incubated with different amounts of unphosphorylated and phosphorylated h-CitT (0.156–5 μM final concentration) at 20°C for 30 min in a total volume of 95 μl. Subsequently, 5 μl of a 20× salt solution (190 mM CaCl₂ and 1 M KCl) and 1 μl of a DNase I solution (0.1 units μl⁻¹; Takara) were added to the reaction mixture, followed by incubation for 1 min at 20°C. DNase I digestion was terminated by adding 100 μl of an ice-cold 2× stop solution (50 mM EDTA, 0.8 M sodium acetate and 0.02 μg μl⁻¹ salmon sperm DNA) to the reaction solution, and then the mixture was subjected to phenol–chloroform extraction and ethanol precipitation. The pellet was redissolved in loading buffer (0.01% bromophenol blue, 0.01% xylene cyanol and 1 mM EDTA in 90% formamide) and then separated on a denaturing 6% polyacrylamide gel. The G+A sequencing ladder used as size markers was prepared according to Belikov and Wieslander (1995).

Acknowledgements

We would like to thank Dr D. Ehlrich and Dr Y. Fujita for kindly providing the pMUTIN derivatives and the *ccpA* null mutant respectively. This research was supported by a grant, JSPS-RFTF96L00105, from the Japan Society for the Promotion of Science, and a Grant-in-Aid for Scientific Research on Priority Areas from the Ministry of Education, Science, Sports, and Culture, Japan.

References

Alén, C., and Sonenshein, A.L. (1999) *Bacillus subtilis* aconitase is an RNA-binding protein. *Proc Natl Acad Sci USA* **96**: 10412–10417.

Anagnostopoulos, C., and Spizizen, J. (1961) Requirements for transformation in *Bacillus subtilis*. *J Bacteriol* **81**: 741–746.

Asai, K., Baik, S.-H., Kasahara, Y., Moriya, S., and Ogasawara, N. (2000) Regulation of the transport system for C₄-dicarboxylic acids in *Bacillus subtilis*. *Microbiology* **146**: 263–271.

Belikov, S., and Wieslander, L. (1995) Express protocol for generating G+A sequencing ladders. *Nucleic Acids Res* **23**: 310.

Birkey, S.M., Liu, W., Zhang, X., Duggan, M.F., and Hulett,

F.M. (1998) Pho signal transduction network reveals direct transcriptional regulation of one two-component system by another two-component regulator: *Bacillus subtilis* PhoP directly regulates production of ResD. *Mol Microbiol* **30**: 943–953.

Boorsma, A., van der Rest, M.E., Lolkema, J.S., and Konings, W.N. (1996) Secondary transporters for citrate and the Mg²⁺-citrate complex in *Bacillus subtilis* are homologous proteins. *J Bacteriol* **178**: 6216–6222.

Bott, M., Meyer, M., and Dimroth, P. (1995) Regulation of anaerobic citrate metabolism in *Klebsiella pneumoniae*. *Mol Microbiol* **18**: 533–546.

Craig, J.E., Ford, M.J., Blaydon, D.C., and Sonenshein, A.L. (1997) A null mutation in the *Bacillus subtilis* aconitase gene causes a block in Spo0A-phosphate-dependent gene expression. *J Bacteriol* **179**: 7351–7359.

Dingman, D.W., Rosenkrantz, M.S., and Sonenshein, A.L. (1987) Relationship between aconitase gene expression and sporulation in *Bacillus subtilis*. *J Bacteriol* **169**: 3068–3075.

Eder, S., Liu, W., and Hulett, F.M. (1999) Mutational analysis of the *phoD* promoter in *Bacillus subtilis*: Implication for PhoP binding and promoter activation of Pho regulon promoters. *J Bacteriol* **181**: 2017–2025.

Fabret, C., Feher, V.A., and Hoch, J.A. (1999) Two-component signal transduction in *Bacillus subtilis*: How one organism sees its world. *J Bacteriol* **181**: 1975–1983.

Fujita, Y., Miwa, Y., Galinier, A., and Deutscher, J. (1995) Specific recognition of the *Bacillus subtilis gnt* cis-acting catabolite-responsive element by a protein complex formed between CcpA and seryl-phosphorylated HPr. *Mol Microbiol* **17**: 953–960.

Golby, P., Davies, S., Kelly, D.J., Guest, J.R., and Andrews, S.C. (1999) Identification and characterization of a two-component sensor-kinase and response-regulator system (DcuS–DcuR) controlling gene expression in response to C₄-dicarboxylates in *Escherichia coli*. *J Bacteriol* **181**: 1238–1248.

Haldenwang, W.G. (1995) The sigma factors of *Bacillus subtilis*. *Microbiol Rev* **59**: 1–30.

Hamblin, M.J., Shaw, J.G., and Kelly, D.J. (1993) Sequence analysis and interposon mutagenesis of a sensor-kinase (DctS) and response-regulator (DctR) controlling synthesis of the high-affinity C₄-dicarboxylate transport system in *Rhodobacter capsulatus*. *Mol Gen Genet* **237**: 215–224.

Henkin, T.M. (1996) The role of CcpA transcriptional regulator in carbon metabolism in *Bacillus subtilis*. *FEMS Microbiol Lett* **135**: 9–15.

Hueck, C.J., and Hillen, W. (1995) Catabolite repression in *Bacillus subtilis*: a global regulatory mechanism for the Gram-positive bacteria? *Mol Microbiol* **15**: 395–401.

Hueck, C.J., Hillen, W., and Saier, M.H., Jr (1994) Analysis of a cis-active sequence mediating catabolite repression in Gram-positive bacteria. *Res Microbiol* **145**: 503–518.

Ireton, K., Jin, S., Grossman, A.D., and Sonenshein, A.L. (1995) Krebs cycle function is required for activation of the Spo0A transcription factor in *Bacillus subtilis*. *Proc Natl Acad Sci USA* **92**: 2845–2849.

Jin, S., and Sonenshein, A.L. (1994a) Identification of two distinct *Bacillus subtilis* citrate synthase genes. *J Bacteriol* **176**: 4669–4679.

Jin, S., and Sonenshein, A.L. (1994b) Transcriptional regulation of *Bacillus subtilis* citrate synthase genes. *J Bacteriol* 176: 4680–4690.

Jin, S., Levin, P.A., Matsuno, K., Grossman, A.D., and Sonenshein, A.L. (1997) Deletion of the *Bacillus subtilis* isocitrate dehydrogenase gene causes a block at stage I of sporulation. *J Bacteriol* 179: 4725–4732.

Jording, D., Sharma, P.K., Schmidt, R., Engelke, T., Uhde, C., and Pühler, A. (1992) Regulatory aspects of the C₄-dicarboxylate transport in *Rhizobium meliloti* – transcriptional activation and dependence on effective symbiosis. *J Plant Physiol* 141: 18–27.

Kaspar, S., Perozzo, R., Reinelt, S., Meyer, M., Pfister, K., Scapozza, L., *et al.* (1999) The periplasmic domain of the histidine autokinase CitA functions as a highly specific citrate receptor. *Mol Microbiol* 33: 858–872.

Kunst, F., Ogasawara, N., Moszer, I., Albertini, A.M., Alloni, G., Azevedo, V., *et al.* (1997) The complete genome sequence of the Gram-positive bacterium *Bacillus subtilis*. *Nature* 390: 249–256.

Kuroda, A., and Sekiguchi, J. (1993) High-level transcription of the major *Bacillus subtilis* autolysin operon depends on expression of the sigma D gene and is affected by a *sin* (*flaD*) mutation. *J Bacteriol* 175: 795–801.

Liu, W., and Hulett, F.M. (1997) *Bacillus subtilis* PhoP binds to the *phoB* tandem promoter exclusively within the phosphate starvation-inducible promoter. *J Bacteriol* 179: 6302–6310.

Liu, W., and Hulett, F.M. (1998) Comparison of PhoP binding to the *tuaA* promoter with PhoP binding to other Pho-regulon promoters establishes a *Bacillus subtilis* Pho core binding site. *Microbiology* 144: 1443–1450.

Liu, W., Eder, S., and Hulett, F.M. (1998a) Analysis of *Bacillus subtilis* *tagAB* and *tagDEF* expression during phosphate starvation identifies a repressor role for PhoP. *J Bacteriol* 180: 753–758.

Liu, W., Qi, Y., and Hulett, F.M. (1998b) Sites internal to the coding regions of *phoA* and *pstS* bind PhoP and are required for full promoter activity. *Mol Microbiol* 28: 119–130.

Matsuno, K., Blais, T., Serio, A.W., Conway, T., Henkin, T.M., and Sonenshein, A.L. (1999) Metabolic imbalance and sporulation in an isocitrate dehydrogenase mutant of *Bacillus subtilis*. *J Bacteriol* 181: 3382–3391.

Meyer, M., Dimroth, P., and Bott, M. (1997) In vitro binding of the response regulator CitB and of its carboxy-terminal domain to A+T-rich DNA target sequences in the control region of the divergent *citC* and *citS* operons of *Klebsiella pneumoniae*. *J Mol Biol* 269: 719–731.

Miles, J.S., and Guest, J.R. (1985) Complete nucleotide sequence of the fumarase gene (*citG*) of *Bacillus subtilis* 168. *Nucleic Acids Res* 11: 131–140.

Nakano, M.M., and Zuber, P. (1993) Mutational analysis of

the regulatory region of the *srfA* operon in *Bacillus subtilis*. *J Bacteriol* 175: 3188–3191.

Price, V.A., Feavers, I.M., and Moir, A. (1989) Role of sigma H in expression of the fumarase gene (*citG*) in vegetative cells of *Bacillus subtilis* 168. *J Bacteriol* 171: 5933–5939.

Qi, Y., and Hulett, F.M. (1998a) PhoP-P and RNA polymerase sigmaA holoenzyme are sufficient for transcription of Pho regulon promoters in *Bacillus subtilis*: PhoP-P activator sites within the coding region stimulate transcription in vitro. *Mol Microbiol* 28: 1187–1197.

Qi, Y., and Hulett, F.M. (1998b) Role of Pho-P in transcriptional regulation of genes involved in cell wall anionic polymer biosynthesis in *Bacillus subtilis*. *J Bacteriol* 180: 4007–4010.

Rashid, M.H., Sato, N., and Sekiguchi, J. (1995) Analysis of the minor autolysins of *Bacillus subtilis* during vegetative growth by zymography. *FEMS Microbiol Lett* 132: 131–137.

Reid, C.J., and Poole, P.S. (1998) Roles of DctA and DctB in signal detection by the dicarboxylic acid transport system of *Rhizobium leguminosarum*. *J Bacteriol* 180: 2660–2669.

Roggiani, M., and Dubnau, D. (1993) ComA, a phosphorylated response regulator protein of *Bacillus subtilis*, binds to the promoter region of *srfA*. *J Bacteriol* 175: 3182–3187.

Sambrook, J., Fritsch, E.F., and Maniatis, T. (1989) *Molecular Cloning. A Laboratory Manual*, 2nd edn. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.

Schaeffer, P., Millet, J., and Aubert, J.P. (1965) Catabolic repression of bacterial sporulation. *Proc Natl Acad Sci USA* 54: 704–711.

Shimotsu, H., and Henner, D.J. (1986) Modulation in *Bacillus subtilis* levansucrase gene expression by sucrose, and regulation of the steady-state mRNA level by *sacU* and *sacQ* genes. *J Bacteriol* 168: 380–388.

Stock, J.B., Surette, M.G., Levit, M., and Park, P. (1995) Two-component signal transduction systems: structure-function relationships and mechanisms of catalysis. In *Two-Component Signal Transduction*. Hoch, J.A., and Silhavy, T.J. (eds). Washington, DC: American Society for Microbiology Press, pp. 24–51.

Tatti, K.M., Carter, H.L., III, Moir, A., and Moran, C.P., Jr (1989) Sigma H-directed transcription of *citG* in *Bacillus subtilis*. *J Bacteriol* 171: 5928–5932.

Vagner, V., Dervyn, E., and Ehrlich, S.D. (1998) A vector for systematic gene inactivation in *Bacillus subtilis*. *Microbiology* 144: 3097–3104.

Yamamoto, H., Mori, M., and Sekiguchi, J. (1999) Transcription of genes near the *sspE* locus of the *Bacillus subtilis* genome. *Microbiology* 145: 2171–2180.

Zientz, E., Bongaerts, J., and Unden, G. (1998) Fumarate regulation of gene expression in *Escherichia coli* by the DcuSR (*dcuSR* genes) two-component regulatory system. *J Bacteriol* 180: 5421–5425.
