
Molecular Microbiology (2000) 37(5), 1208–1219

# ResD signal transduction regulator of aerobic respiration in *Bacillus subtilis*: ctaA promoter regulation

Xiaohui Zhang and F. Marion Hulett*

Laboratory for Molecular Biology, Department of Biological Sciences University of Illinois at Chicago, 900 S. Ashland Avenue (M/C 567), Chicago, IL 60607, USA.

## Summary

A two-component signal transduction system composed of a sensor kinase, ResE, and a response regulator, ResD, encoded by *resD* and *resE* genes of the *res* operon (*resABCDE*), has a regulatory role in both aerobic and anaerobic respiration. In terms of aerobic respiration, *resD* functions upstream of *ctaA*, a gene required for haem A biogenesis and hence for the synthesis of haem A-containing cytochrome terminal oxidases. Although ResD is probably a transcription factor, there was no direct evidence that ResD protein, either phosphorylated or unphosphorylated, interacts directly with regulatory regions of ResD-controlled genes. Here, we report the overexpression and purification of ResD and ResE and their role in gene activation. ResD can be phosphorylated by ResE *in vitro* and is a monomer in solution in either the phosphorylated or unphosphorylated state. The binding activity of ResD to the *ctaA* promoter was examined by gel shift assays and DNase I footprinting assays. DNase I footprinting showed both unphosphorylated and phosphorylated ResD binding to the *ctaA* promoter and showed that there are three binding sites (A1, A2, and A3), two (A1 and A2) upstream of the –35 promoter region and one (A3) downstream of the –10 of the promoter. The role of each site in *ctaA* promoter activity and ResD binding was characterized using deletion analysis, followed by the DNase I footprinting and *in vivo* transcription assays of promoter–*lacZ* fusions. Our results showed that the concentration of ResD required to bind at each site is different and that ResD binding at the A1 site is independent of the other two ResD binding sites, but that the concentration of ResD~P required to protect site A2 is reduced when site A3 is present. *In vivo* transcription assays from promoter–*lacZ* fusion constructs showed that DNA containing ResD-binding site A2 was essential for promoter activity and that promoter constructs containing both binding sites A2 and A3 were sufficient for full promoter activity.

## Introduction

ResD and ResE were identified in the *Bacillus* genome sequencing project (Sorokin *et al.*, 1993) as genes encoding a response regulator and sensor histidine kinase of prokaryotic two-component signal transduction systems. They share similarity to the OmpR family of regulators. Within *Bacillus subtilis* regulators, they belong to a small subclass (based on properties of the sensor kinase) that also includes PhoP and PhoR, which regulate the phosphate deficiency response (Pho response), and YycF and YycG, which are essential two-component regulators of unknown function (Fabret and Hoch, 1998; Fabret *et al.*, 1999).

ResD and ResE are global regulators of genes involved in aerobic and anaerobic respiration. Originally, they were shown to be required for transcription of the *resABCDE* operon (*resABC* encode proteins similar to those involved in cytochrome *c* biogenesis), of *ctaA* (responsible for conversion of haem O to haem A) and of *petCBD* (encoding subunits of the cytochrome *bf* complex) and to be essential for anaerobic growth of *B. subtilis* in the presence of nitrate (Sun *et al.*, 1996). ResD and ResE are also required for *ctaB* transcription (responsible for haem B conversion to haem O) and for full induction of the Pho response via a mechanism which involves a positive feedback loop, resulting in the amplification of both Res and Pho regulon induction during phosphate deprivation Birkey *et al.*, 1998). Genes involved in nitrate respiration which require ResD and ResE include *fnr*, *hmp* and *nasDEF* (nitrite reductase) (Nakano *et al.*, 1996, 1998; Nakano and Hulett, 1997).

Although ResD has been shown to be required for transcription of the above genes *in vivo*, it has yet to be shown that ResD acts directly. We chose to examine the role of ResD in the activation of the *ctaA* promoter. The *ctaA* gene encodes a membrane-bound protein that catalyses the conversion of haem O to haem A (Svensson *et al.*, 1993, 1996; Svensson and Hederstedt, 1994) and, as such, is essential for production of aa3 and caa3 terminal oxidases.

Accepted 26 June, 2000. *For correspondence. E-mail hulett@uic.edu; Tel. (+1) 312 996 5460; Fax (+1) 312 413 2691.*

© 2000 Blackwell Science Ltd

Results

Overexpression and purification of ResD and ResE

ResD was overexpressed and purified for investigation of its functions *in vitro*. We used IMPACT (Intein-Mediated Purification with an Affinity Chitin-binding Tag, New England Biolabs) as an affinity protein purification system and obtained purified recombinant ResD protein from Escherichia coli within 24 h by a single chromatographic step. The ResD protein was successfully expressed in E. coli grown at 30°C, and the soluble protein obtained from the supernatant fraction of a cell lysate after centrifugation at 120 000 g for 1 h was applied to a chitin affinity column. After the column was washed to remove non-binding protein, the fusion protein was induced to undergo an intein-mediated self-cleavage on the column by overnight incubation at 4°C in the presence of dithiothreitol (DTT). The ResD protein was released while the intein chitin-binding domain remained bound to the column. The fractions of the eluted ResD exhibited greater than 98% homogeneity, as determined by SDS-PAGE with Coomassie brilliant blue staining (Fig. 1A, lanes 6 and 7).

Purified ResE was necessary for phosphorylation of ResD. As the insoluble nature of the complete ResE protein complicated the *in vitro* studies, we constructed an N-terminal truncated ResE, *ResE*, in a glutathione-S-transferase (GST) fusion vector. This construct provides a simplified purification of the soluble form of the protein. Having its N-terminus 226 amino acids deleted, *ResE* retains the majority of its extended cytoplasmic domain and the complete conserved catalytic domain. The GST-*ResE* protein was successfully overexpressed in E. coli, and it accounted for 35% of the total cellular protein (Fig. 1B). Approximately 70% of the GST-*ResE* protein expressed at 30°C was in the soluble fraction after centrifugation at 120 000 g for 1 h. The soluble fusion protein was applied to a glutathione agarose column and the column was washed to remove unbound protein. The glutathione agarose beads with a bound fusion protein were mixed with 10 U of thrombin. After thrombin cleavage, 95% of the truncated *ResE* protein was collected in the eluted fraction from the beads (Fig. 1B, lanes 6 and 7).

Phosphorylation of ResD by *ResE*

To determine whether the N-terminal truncated form of ResE protein functions as an autokinase, the purified *ResE* was mixed with [γ-³²P]-ATP. *ResE* showed autophosphorylation (Fig. 1C, lane 1). To determine whether ResD could be phosphorylated by *ResE*, *ResE* was incubated with ResD in the presence of [γ-³²P]-ATP. Figure 1C, lane 3, shows that ResD could be

Fig. 1. A–C. Purification of ResD and ResE for *in vitro* phosphorylation studies. The fractions from each step of each purification were analysed by SDS-PAGE, and the gel was stained with Coomassie brilliant blue. The amount of protein loaded in each lane is given below in parentheses.

A. Purification of intein–ResD from *E. coli*. Arrows mark the calculated molecular weight for the ResD–intein–CBD fusion protein and the position of the purified ResD protein. Lanes: 1, prestained protein standards (Bio-Rad); 2, sonication lysate from uninduced cells (30 μg protein); 3, sonication lysate from induced cells (35 μg protein); 4, clarified crude extract from induced cells (35 μg protein); 5, column flow-through (35 μg protein); 6 and 7, fractions of eluted by cleavage buffer with DTT (2 and 5 μg protein respectively).

B. Purification of overexpressed GST–*ResE* from *E. coli*. Arrows mark the calculated molecular weight for the GST–ResE fusion protein and the position of the purified ResE. Lanes: 1, prestained protein standards (Bio-Rad); 2, sonication lysate from uninduced cells (5 μg protein); 3, sonication lysate from induced cells (10 μg protein); 4, pellet fraction (10 μg protein); 5, clarified crude extract from induced cells (35 μg protein); 6 and 7, elution collected after thrombin cleavage on glutathione agarose beads (2.8 and 7 μg protein respectively).

C. Phosphorylation of ResD by *ResE* in the presence of ATP. ResD (9.6 μM) and/or *ResE* (5.9 μM) were incubated with 1 μCi of [γ-³²P]-ATP at room temperature for 15 min. The samples were analysed by SDS–PAGE. The gel was dried and exposed to an X-ray film. Lanes: 1, ResE incubated with [γ-³²P]-ATP; 2, ResD incubated with [γ-³²P]-ATP; 3, ResD incubated with ResE and [γ-³²P]-ATP.

phosphorylated by *ResE*. Unlike *ResE*, ResD was not phosphorylated by ATP alone (Fig. 1C, lane 2). This result indicates that ResE is a kinase capable of ResD phosphorylation.

1210 X. Zhang and F. M. Hulett

**Binding of the ctaA promoter by ResD and ResD~P**

We have shown that the ctaA promoter is subject in vivo to regulation by ResD, the response regulator of the ResD–ResE two-component system (Sun et al., 1996). To determine whether ResD binds to the ctaA promoter in vitro, ResD and ResD~P were incubated with the ctaA promoter for gel shift assays. With increasing ResD concentrations, we observed that ctaA promoter mobility decreased in the presence or absence of ATP (all reactions contained *ResE*), suggesting that both unphosphorylated ResD and phosphorylated ResD bind to the promoter (Fig. 2A and B). A slight shift was apparent with either ResD or ResD~P at 74 nM protein and became more pronounced at increased ResD concentrations. The gel shift pattern was complex, suggesting that additional ResD molecules bound the ctaA promoter at higher protein concentrations and that several DNA-protein complexes could be formed.

**ResD or ResD~P binds to three sites within the ctaA promoter**

The binding specificities of ResD and ResD~P to the ctaA

**Fig. 2.** A and B. Gel shift assay of the ctaA promoter by (A) ResD or (B) ResD~P. ResD (74 nM, 148 nM, 296 nM, 593 nM, 1.18 μM, 2.37 μM and 4.74 μM) was incubated with (A) or without (B) *ResE* in the presence of ATP. The 406 bp ctaAB end-labelled promoter probe was added. After the binding reaction, the samples were loaded onto a 5% native polyacrylamide gel to separate the free DNA from the DNA–protein complex. The amount of ResD protein added to the reaction is indicated above each lane.

**The ResD is a monomer, unphosphorylated or phosphorylated**

Certain response regulators are known to oligomerize upon phosphorylation. We used gel filtration to determine whether ResD oligomerizes upon phosphorylation. Purified ResD was considered unphosphorylated as the conditions for overexpression of the protein do not favour the formation of ResD~P. ResD and ResD phosphorylated by GST-*ResE* were individually applied to a Superdex 75 gel filtration column. Unphosphorylated ResD was eluted at a relative molecular mass of 27 000, a value in good agreement with that calculated from the deduced amino acid sequence of *resD*. The elution pattern of phosphorylated ResD from the column showed that the protein peak for the unphosphorylated ResD and the major radioactivity peak for ResD~P were coincident at the same elution volume, indicating that ResD~P, like unphosphorylated ResD, is a monomer in solution.

**Coding**  
F ResD FGF ResD~pF  

**Non-coding**  
F ResD FGF ResD~pF  

**Fig. 3.** DNase I footprinting assays of the ctaA promoter bound by ResD and ResD~P. Various amounts of ResD incubated with *ResE* in the presence or absence of ATP were mixed with the 406 bp ctaAB promoter probe, and DNase I footprinting experiments were performed on both the coding strand and non-coding strands (see Experimental procedures). The concentrations of ResD or ResD~P used in each reaction are, from left to right, 0 nM, 9 nM, 18 nM, 37 nM, 74 nM, 148 nM, 296 nM, 594 nM and 0 nM. Increasing concentrations of ResD or ResD~P are indicated by a wedge. F, ResD-free lane; G, G sequencing reaction. The bold lines represent the ResD and ResD~P binding regions. The hypersensitive site is indicated by an arrowhead.

© 2000 Blackwell Science Ltd, Molecular Microbiology, **37**, 1208–1219

ResD phosphorylation, oligomerization and DNA binding 1211

Table 1. ResD or ResD~P dissociation constants for sites A1, A2 and A3.

| Site | $K_{\mathrm{D}} \pm$ s.e. |
|------|-------------------------|
|      | Unphosphorylated        | Phosphorylated       |
| A1   | $12.6 \pm 3.3$          | $8.9 \pm 1.4$        |
| A2   | $230 \pm 40.1$          | $200 \pm 32.3$       |
| A3   | $180 \pm 29.9$          | $47 \pm 7.6$         |

equilibrium dissociation constant $K_{\mathrm{D}}$ (Table 1). The ResD binding data are plotted (Fig. 4) as per cent base protection vs. concentration of ResD or ResD~P ([P]). Exact sequences of the DNA protected by ResD on each DNA strand are shown in Fig. 5A.

ResD binding at any one of the three ctaA promoter binding sites is independent of the other sites; binding site two is essential for promoter activity whereas binding site three is required for full promoter activity.

Five deletion constructions of the ctaA promoter were made (Fig. 5B) to assess the role of each ResD binding site on (i) ResD binding at another site and (ii) promoter activation. Using JH642 genomic DNA as a template, primers (Fig. 5A and Experimental procedures) were used to amplify DNA fragments which contained only a single ResD binding site or two of the three binding sites in the ctaA promoter (Fig. 5B). Each of the five amplified fragments were cloned into pCR2.1(Invitrogen). Each deleted promoter was used in footprinting studies and each was cloned into pDH32 constructing a promoter-lacZ fusion (Table 2) which was integrated in single copy into the chromosome of JH642 (or the isogenic resD mutant strain MH5260; Sun et al., 1996) at the amyE locus via homologous recombination (Table 2). Figure 6 shows a footprint for ResD~P binding at any one of the three sites alone similar to that observed at that site in the complete promoter (Fig. 3), albeit the protection by Res~P at a concentration of 296 nM was not as extensive for A2 or A3 as in the complete promoter (Fig. 3). The promoter activity from the lacZ fusion constructs containing either only A2 (bp -152 to +11) in B. subtilis MH5915 or only A3 (bp -50 to +72) in B. subtilis MH5916 was compared with the promoter activity

A1
![Graph](https://example.com/graph1.png)

A2
![Graph](https://example.com/graph2.png)

A3
![Graph](https://example.com/graph3.png)

ResD Concentration (nM)

Fig. 4. ResD and ResD~P protection at binding sites A1, A2 and A3 differ in the concentration of protein required for binding site protection and in the discrimination between ResD and ResD~P. Per cent protection vs. nM protein for ResD (open symbols) and ResD~P (closed symbols).

© 2000 Blackwell Science Ltd, Molecular Microbiology, 37, 1208–1219

1212 X. Zhang and F. M. Hulett

A


TGAGTTAGCCATAACGTAACCTCCCTTACAAACCTAACGCCAGGAGAATAAAATAACCTGCAAC
ACCTCAATCGGTATTGCATTTGGAGGGAATGTTTGGATTCGCGTCCTCTTATTTTTATTGGACGTTG
ctab
TCTATCATACAAGATAACATCTTGTAAAAAAAAACATCCTATTTAAATATTTAAACGACAAAACAGGAAT
AGATAGTATGTTCTATTTGTAGAACATTTTTTTGTAGATAATTTATAATTTGCTGTTTGTCCTTA
-209 -179
TTATTTGTGAACTTTTTATGAATTTTACCAAAGATGAAAGGCAATCTCCAAATCGGCATTGCGTTT
AATAAACACTTGAAAAATACTTTAAAATGGTTTCTACTTCCCTTAGAGTTTAGCCGTAACGCAA
FMH385 -108
ACCTTATTTTCTATCAGACCCGECATGAATAATCAAGTGTACATTTCCAGAAGCCGTCATTCTAT
TGGAAATAAAGATAGTCTGGGCCGTACTTATTTAGTTACATGTA AAAAGGTCTTCGGCAGTAAAGATA
FMH370 FMH383
TATATTTTTGTGAACAAAAAGGCTCTGGGAATTGCCACAAAAACACATATTCGGTTACACTTGGAAAC
-55
ATATAAAAAACACTTGTTTTCCGAGACCCCTTAACGGTTTTTTGTGTATAAGGAATGTGAACCTTG
-2 +1 ctaA +43
GTATATAAAATTGCAGCAGTATGTTAAGAAGGTGAATATTTGTATGAAATAAAGCATTTAAAGCTCT
CATATATTTTAAACGTCGTCATACAATTCTTCCACTTATAACATATACTTATTTTGTAAATTTTCGAGA
FMH384 FMH372
CGGTGTTCTGACGACATTTTGTCATGCTAATTTGTTTTAATCGGGGGTGCCCCCTCGTTACAAAAAACAG
GCCACAAAGACTGCTGTAACAGTACGATTAACAAAAATTAGCCCCCACGGGAGCAAATGTTTTTGTG
FMH255
GTTCCGGCCAAGGATGC
CAAGGCCGGTTTCCTACG


B


Plasmids/strains
-209 A1 -179 -108 A2 -55 -35 -10 -2 A3 +43
-335 pES68/pXH34/MH5909
-335 pXH41/pXH42/MH5919
-335 pXH28/pXH31/MH5906
-152 pXH38/pXH40/MH5916
-152 pXH37/pXH39/MH5915
pXH43/pXH44/MH5921


Fig. 5. ResD binding sites on the ctaAB intercistronic promoter sequence and diagrams of various promoter subclones.

A. The intercistronic ctaAB promoter sequence and 5' coding sequence of for CtaA showing the ResD and ResD~P binding sites. The coding and non-coding sequences of the fragment are shown. The binding sites for both ResD and ResD~P are represented by bold solid lines below (coding) the sequence. The transcriptional start site is shown in bold and is labelled. Primers used for amplification of the complete ctaAB promoter and the various ctaAB promoter deletions are shown by arrows (pXH43, FMH385 and FMH372; pXH28, FMH254 and FMH370; pXH37, FMH385 and FMH384; pXH38, FMH383 and FMH372; pXH41, FMH254 and FMH384; pES68, FMH254 and FMH255).

B. Diagrams of the various ctaAB promoter-amplified fragments, the pCR2.1 plasmids containing each fragment and the name of the strain containing a lacZ-promoter fusion construct of each fragment. The ResD binding sites are marked as black boxes. The 5' and 3' bp of each promoter fragment are labelled relative to the transcription start site +1.

Table 2. Bacterial strains and plasmids.

| B. subtilis strains | Genotype or antibiotic resistance | Source or reference |
|---------------------|----------------------------------|----------------------|
| JH642               | pheA1 trpC2                      | J. A. Hoch           |
| MH5906              | pheA1 trpC2 amyE::pXH31 Cm'      | This study           |
| MH5909              | pheA1 trpC2 amyE::pXH34 Cm'      | This study           |
| MH5915              | pheA1 trpC2 amyE::pXH39 Cm'      | This study           |
| MH5916              | pheA1 trpC2 amyE::pXH40 Cm'      | This study           |
| MH5919              | pheA1 trpC2 amyE::pXH42 Cm'      | This study           |
| MH5921              | pheA1 trpC2 amyE::pXH44 Cm'      | This study           |

| Plasmids | Description                                                                 | Source or reference |
|----------|-----------------------------------------------------------------------------|----------------------|
| pCR2.1   | Vector for cloning PCR products (Amp' Kan')                                 | Invitrogen           |
| pDH32    | Vector for construction of promoter-lacZ fusion (Amp' Cm')                  | Shimotsu (1986)      |
| pES68    | Amp' Kan' (4.3 kb) ctaA (A1 + A2 + A3) promoter (406 bp) in pCR2.1            | This study           |
| pXH6     | Amp' Kan' (4.6 kb) resD gene (720 bp) in pCR2.1                             | This study           |
| pXH22    | Amp' (8.0 kb) resD gene (720 bp) in pTYB₂                                   | This study           |
| pXH1     | Amp' Kan' (5.0 kb) resE gene (1.14 kb) in pCR2.1                            | This Study           |
| pXH3     | Amp' (6.1 kb) resE gene (1.14 kb) in pGEX-2T                                | This study           |
| pXH28    | Amp' Kan' (4.2 kb) ctaA promoter (A1) (227 bp) in pCR2.1                     | This study           |
| pXH37    | Amp' Kan' (4.0 kb) ctaA promoter (A2) (163 bp) in pCR2.1                     | This study           |
| pXH38    | Amp' Kan' (3.9 kb) ctaA promoter (A3) (122 bp) in pCR2.1                     | This study           |
| pXH41    | Amp' Kan' (4.2 kb) ctaA promoter (A1 + A2) (346 bp) in pCR2.1                | This study           |
| pXH43    | Amp' Kan' (4.1 kb) ctaA promoter (A2 + A3) (224 bp) in pCR2.1                | This study           |
| pXH34    | Amp' Cm' (10.3 kb) ctaA-lacZ fusion (A1 + A2 + A3) (406) in pDH32             | This study           |
| pXH31    | Amp' Cm' (10.1 kb) ctaA-lacZ fusion (A1) in pDH32                           | This study           |
| pXH39    | Amp' Cm' (10.0 kb) ctaA-lacZ fusion (A2) in pDH32                           | This study           |
| pXH40    | Amp' Cm' (10.0 kb) ctaA-lacZ fusion (A3) in pDH32                           | This study           |
| pXH42    | Amp' Cm' (10.2 kb) ctaA-lacZ fusion (A1 + A2) in pDH32                       | This study           |
| pXH44    | Amp' Cm' (10.1 kb) ctaA-lacZ fusion (A2 + A3) in pDH32                       | This study           |

from the complete ctaA promoter (bp -335 to +72) in MH5909. Strains containing all promoter fusions were grown in 2× SSG containing 0.5% glucose and β-galactosidase-specific activity was assayed (Fig. 7). Strain MH5915 containing the promoter fusion with only ResD binding site A2 showed less than 50% of the activity expressed by the full-length promoter whereas the strain

containing only binding site A3 in the promoter-lacZ fusion was not functional, although that promoter fusion retained both the -10 and -35 regions (Fig. 5A and B). To determine whether adjacent DNA containing either the A1 (5') or A3 (3') binding site might enhance promoter expression from the promoter fusion containing only ResD binding site A2, β-galactosidase expression from MH5919

Fig. 6. ResD or ResD~P protection of promoter fragments containing a single ResD binding site. The probe for binding site A1 was the insert in pXH28, for A2 the insert in pXH37 and for A3 the insert in pXH38. Only the coding strand is labelled. The concentrations of ResD used in each reaction are, from left to right, 0 nM, 37 nM, 74 nM, 296 nM, 1.18 μM, 2.37 μM and 0 nM. The amounts of *ResE and ATP in each reaction were the same as those used for footprinting both the coding and non-coding strands of the wild-type ctaA promoter shown in Fig. 3. The labels are also the same as those in Fig. 3.

1214 X. Zhang and F. M. Hulett

![Graph Image](image_url)

Time(hours)

Fig. 7. Expression of the *ctaA* promoter is dependent on ResD binding sites A2 and A3. Parental, *resD* mutant strains containing the various *ctaA-lacZ* promoter fusions were grown in 2× SSG, as previously described (Sun et al., 1996). Cell growth (*OD*<sub>540</sub>) and β-galactosidase activity were analysed during the 11 h growth period. Growth (filled symbols) and β-galactosidase (open symbols) from strains (●) MH5909 (bp –335 to +72; A1 + A2 + A3); (■), MH5915 (bp –152 to +11; A2); (▲) MH5916 (bp –50 to +72; A3); (◆) MH5921 (bp –152 to +72; A2 + A3) and (▽) MH5919 (bp –335 to +11; A1 + A2).

and MH5921 was assayed (Fig. 7). Expression from the promoter fusion with sites A1 and A2 (MH5919) was not appreciably different from the fusion containing only A2 (MH5915) whereas expression from the promoter fusion with sites A2 and A3 (MH5921) was similar to the full-length promoter (MH5909), suggesting that site A3 was required for full promoter expression. The *in vivo* expression data from the five promoter fusion strains were consistent during three independent experiments. No expression was obtained from any of these five promoter fusions in isogenic strains that carried a null mutation in *resD* (data not shown).

The ResD protected region on DNA fragments containing binding site A2 with either site A1 or site A3 showed no change in the size of the protected DNA areas at any site (Fig. 8). The only change in the pattern noted was the appearance of a small unprotected region apparent within ResD binding site A1, perhaps indicating some conformational change in this area when site 3 was deleted from the whole promoter (Fig. 8). However, the enhanced affinity of ResD for site A2 by the presence of site A3 on the same DNA fragment (Fig. 8) correlates with the enhanced promoter activity from the promoter-containing

pXH41 (1+2)
Coding
F ResD FGF ResD~P F

pXH43 (2+3)
Coding
F ResD FGF ResD-PF

-209
-179
-108
-55
-2
+43

Fig. 8. ResD and ResD~P DNase I footprinting protection of *ctaA* promoter fragments containing two of the three ResD binding sites, A1 + A2 or A2 + A3. The *ctaA* promoter inserts in pXH41 and pXH43 were used as probes. Only the coding strand was labelled. Labels and the amounts of ResD, *ResE* and ATP in each reaction were the same as those used for footprinting assays of *ctaA* promoter fragments containing a single ResD binding site (Fig. 6).

sites A2 and A3 compared with the *lacZ* promoter fusion containing only site A2 (Fig. 7).

Discussion

*ctaA* is one of several genes involved in aerobic or anaerobic respiration which require ResD for expression. To determine whether the activation of these genes by ResD involves direct promoter binding and to explore the mechanism by which ResD regulates gene transcription, we used biochemical analysis to characterize the ResD protein with respect to phosphorylation, oligomerization and DNA-binding activity. The ResD used in this study is identical to the wild-type protein except for an additional glycine residue at the C-terminal. The ResE protein used in these studies is a soluble protein as the sequences encoding the N-terminal hydrophobic domains and a portion of the C2 domain were deleted (236 amino acids of the 589 total amino acids) from the *resE-GST* fusion construction.

ResD binding site A2 (bp –108 to –55) and A3 (bp –2 to +43) are required for full *ctaA* promoter activity

The ResD target DNA used in these studies included the *ctaA* promoter and the divergent promoter for *ctaB*. We

© 2000 Blackwell Science Ltd, Molecular Microbiology, **37**, 1208–1219
had previously shown that *in vivo* transcription of *ctaA* required ResD (Sun *et al.*, 1996). Recently, it has been shown that the *ctaB* promoter requires ResD for post-exponential transcription (Liu and Taber, 1998). Three binding sites for ResD were located upstream of *ctaA*. Deletion of the most 5′ site did not influence the expression of *ctaA in vivo*, suggesting that site A1 may be involved in regulation of *ctaB*. When DNA containing site A1 and A2 was deleted, there was no *ctaA in vivo* expression, although the promoter fusion contained the −35 and −10 regions as well as binding site A3. This phenotype, which matched that of a *resD* null mutant strain exactly, indicates that DNA containing the ResD binding site A2 is essential for *ctaA* expression *in vivo*. Further, we have shown that DNA between A1 and A2 has little importance for *ctaA* promoter activity as promoter fusion constructs starting at the 5′ end of the A2 site (bp −108 to +72) retained *in vivo* promoter activity similar to that shown in Fig. 7 for the complete promoter fusion in MH5909 (X. Zhang and F. M. Hulett, unpublished). 3′ deletions of the complete promoter, removing DNA 3′ of +11 and thus part of site A3 (bp −2 to +43), caused a 2.5- to threefold reduction in *ctaA in vivo* expression under the conditions tested. Thus, the A3 region is necessary for full promoter activity.

All promoter fusions showed induction at approximately the same post-exponential stage of growth followed by a decline in promoter activity after 4–5 h, except in the fusion that contained only the A2 binding site (strain MH5915; Fig. 7) in which expression failed to decline. This observation was reproducible but is not understood. The decrease in *ctaA* promoter activity correlates with a decrease in *resA* transcription (Sun *et al.*, 1996), suggesting that decreasing ResD–ResE protein concentrations may account for a *ctaA* transcription turn off. However, this does not explain the prolonged transcription of the A2 promoter fusion.

### A consensus recognition sequence for ResD within the ResD binding sites is proposed

Certain members of the OmpR family of response regulators, including Pho regulon regulators PhoP of *B. subtilis* (Liu and Hulett, 1998; Eder *et al.*, 1999) or PhoB of *E. coli* (Kasahara *et al.*, 1991; Makino *et al.*, 1991; Kim *et al.*, 1993), bind highly conserved DNA sequences TT(A/T/C)ACA N4–5 TT(A/T/C)ACA and TGTCATA(A/ T)A(T/A)TGTCAT(C/T), respectively, such that analysis of binding sites from multiple promoters predict a reliable binding consensus sequence (Makino *et al.*, 1996; Eder *et al.*, 1999). The binding of PhoB to the bases of the predicted sequence (TGTCAT) has been demonstrated by nuclear magnetic resonance (NMR) structural analysis of the PhoB/DNA complex (Okamura *et al.*, 2000). In contrast to PhoP or PhoB, other OmpR class regulators, such as OmpR of *E. coli*, show only weak similarity between the DNA sequences bound by the protein, a fact which led to proposals of different consensus sequence alignments (Harlocker *et al.*, 1995; Huang and Igo, 1996). In the case of OmpR, DNA affinity cleavage analysis was required to construct a DNA/OmpR interaction model. These data provided evidence that repeated 6 bp sequences, spaced 4 bp apart, each bind one OmpR molecule (Harrison-McMonagle *et al.*, 1999). Both PhoB and OmpR bind to the target DNA in tandem. Considering the results from PhoP, PhoB and OmpR DNA-binding studies collectively, we might expect to find a consensus of 5 or 6 bp repeats separated by 4 or 5 bases within the ResD binding sequences. Reviewing the three ResD binding sites in the *ctaA* promoter and those determined elsewhere (Nakano *et al.*, 2000) for anaerobically induced genes requiring ResD, a 9 bp sequence, TTTGTGAAT, is observed in all promoters. This sequence is repeated in the *ctaA3* binding site (6/9 and 7/9 matches respectively), the *ctaA1* binding site (8/9 matches each), the *nasD* binding site (6/9 matches each) and the *hmp* site (6/9 and 7/9 respectively). Half-sites (a single 9 bp consensus) are found in the *ctaA2* ResD binding site (8/9), the *fnr* binding site (8/9 matches on the complementary strand) and in the *resA* binding site (6/9 matches) (X. Zhang and F. M. Hulett, unpublished). Preliminary evidence has suggested that certain base pairs within this sequence are important for *fnr* promoter activity. The relevance of this putative ResD binding consensus sequence to the ResD activation of these various promoters remains to be tested.

### ResD is a monomer and binds target DNA independent of phosphorylation

Activation of response regulators is generally believed to be indirectly correlated with phosphorylation of the receiver domain relieving a mutual functional inhibition of the N- and C-terminal domains, allowing oligomerization and/or binding of target DNA by the C-terminal domain (Fiedler and Weiss, 1995). Indeed the crystal structure of NarL supports this idea as the N-terminal regulatory domain and the C-terminal DNA binding domain are positioned such that DNA binding to the recognition helix would be precluded (Baikalo *et al.*, 1996, 1998). Although the structure for only the DNA-binding domain of the OmpR has been solved, it was determined that the N-terminus of OmpR could not be positioned with respect to the C-terminus in a manner similar to those domains in NarL (Martinez-Hackert and Stock, 1997). Still, it was concluded based on biochemical characterization of OmpR that the unphosphorylated N-terminal domain interferes with DNA binding, but that the surfaces responsible for inhibition are unknown. For PhoB of © 2000 Blackwell Science Ltd, *Molecular Microbiology*, **37**, 1208–1219
1216 X. Zhang and F. M. Hulett

E. coli, another OmpR family response regulator, it has been shown that deletion of the C-terminal domain mimics phosphorylation activation of the N-terminus such that the N-terminal domain dimerizes, a dimer for which the structure and the interaction surfaces are known (Sola et al., 1999). Studies reported here show that ResD is a monomer in solution and is able to bind target DNA either in the phosphorylated or unphosphorylated state, albeit the phosphorylated form usually binds at somewhat lower protein concentrations. Our data suggest that ResD is a member of a third class of response regulators in which the N-terminus and the C-terminus function independently. Similar results were shown for a second B. subtilis OmpR family response regulator which has significant identity in protein sequence with ResD, PhoP; except in the case of PhoP, the dimeric state was independent of phosphorylation and phosphorylation did not result in further oligomerization nor was it essential for PhoP binding to target DNA (Liu and Hulett, 1997). Other response regulators which both bind DNA without phosphorylation and do not change oligomeric state upon phosphorylation include E. coli UhpA (Dahl et al., 1997) and Bordetella pertussis BvgA (Boucher et al., 1997). For this class of proteins, phosphorylation may function only to change the conformation of the protein, thereby effecting interaction with target DNA and/or the RNA polymerase and resulting in activation of transcription.

### Experimental procedures

#### Bacterial strains and plasmids

E. coli DH5α was used as the host for plasmid constructions. E. coli BL21 (DE3) (Novagen) served as the host for overexpressing the ResD and ResE protein. The resD gene was amplified from B. subtilis JH642 chromosomal DNA by PCR using primers FMH238 (5′-CAT⁴⁴²⁶ATGGACCAAAAC-GAACGAAAC⁴⁴⁴⁵-3′) at the 5′ end of the gene and FMH356 (5′-TTCCCGGG⁵¹⁴⁶TTCAGCGCCGACCTCAA⁵¹³²-3′) at the 3′ end of the gene (the superscript numbers indicate the base pair number in GenBank accession no. L09228). The PCR product was cloned into pCR2.1 (Invitrogen) to construct pXH6. The resD gene was released from pXH6 by Ndel and Smal digestion and was cloned into the Ndel and Smal sites of pTYB2 (New England) downstream of the T₇ promoter, yielding pXH22. Thus, resD was inserted into a multiple cloning site of pTYB2 to create a gene fusion between the 3′ bp of resD and the 5′ bp of the gene encoding the intein. The DNA encoding a small 5 kDa chitin binding domain from Bacillus circulans has been added to the 3′ end of the intein gene for affinity purification of the three-part fusion. Expression of the fusion construct is controlled by an IPTG-inducible T₇ promoter.

The resE gene was amplified from B. subtilis JH642 chromosomal DNA by PCR using primers FMH268 (5′-GGATCC⁵⁸²⁶GGTGAACTGGCGACC⁵⁸⁴¹-3′) at the 5′ end of the gene and FMH284 (5′-TTGAATTC⁶⁹⁷¹AGTCAGCGGGGTG TTTC⁶⁹⁵⁴-3′) at the 3′ end of the gene. The PCR product was cloned into pCR2.1 (Invitrogen) to construct pXH1. The PCR product was digested with BamHI and EcoRI and ligated to pGEX-2T (Pharmacia Biotech) and digested with the same enzymes, yielding pXH3. All clones generated were sequenced to determine PCR accuracy.

Plasmid pES68, which contains the intercistronic ctaAB sequence plus part of the 5′ coding sequence for CtaA, was constructed by amplifying a 406 bp fragment, using JH642 DNA as template and primers FMH254 (5′-TAGCCA-TAACGTAACCTCC³⁸⁵-3′) and FMH255 (5′-ACAAATGTC GTCAGAACACC²⁸-3). The amplified product was cloned into pCR2.1 and sequenced (the superscript numbers indicate the base pair number in GenBank accession no. X54140). Plasmids containing deletions of the ctaA promoter were made using the same method and the primers whose sequences are identified in Fig. 5A. We constructed pXH43 (primers FMH853 and FMH372), pXH28 (primers FMH254 and FMH370), pXH37 (primers FMH385 and FMH384), pXH38 (primers FMH383 and FMH372) and pXH41 (primers FMH254 and FMH384). Primers FMH372, FMH384 and FMH370 each contained at BamHI site added at the 5′ end, GGATCC, which is not homologous to adjacent DNA in the ctaA promoter and is not indicated in Fig. 5A.

ctaA–lacZ transcriptional promoter fusions were constructed in pDH32, which has unique EcoRI and BamHI sites upstream of the promoterless lacZ to orientate correctly the promoter DNA fragment. ctaA promoter fragments in pCR2.1 which were derived from PCR amplification with the coding region proximal primer containing a BamHI site were digested with EcoRI and BamHI to release the promoter fragment which was ligated into pDH32 digested with the same enzymes. Thus, the EcoRI–BamHI promoter fragment from pXH41 cloned into pDH32 yielded pXH42, from pXH28 cloned into pDH32 yielded pXH31, from pXH38 cloned into pDH32 yielded pXH40, from pXH37 cloned into pDH32 yielded pXH39, and from pXH43 cloned into pDH32 yielded pXH44. The complete ctaA promoter in pES68 was liberated by EcoRI digestion and cloned into the EcoRI site in pDH32 to construct pXH34. The promoter orientation in pXH34 was determined by restriction digestion. Strains containing single copy promoter–lacZ fusions were constructed by transforming JH642 with linearized plasmid such that the promoter fusion was integrated into the chromosome by double crossover homologous recombination at the amyE locus with the amyE front and amyE back on the pDH32 plasmid. Transformants were selected on chloramphenicol and were screened for the amyE minus phenotype. Transformation of JH642 with pXH34 produced strain MH5909 (A1,A2,A3), with pXH42 produced MH5919 (A1,A2), with pXH31 produced MH5906 (A1), with pXH40 produced MH5916 (A3), with pXH39 produced MH5915 (A2) and with pXH44 produced MH5921 (A2,A3).

### Overexpression and purification of ResD and ResE

#### Protein expression. E. coli BL21 (DE3) cells transformed with pXH22 or pXH3 were incubated overnight at 30°C in Luria–Bertani (LB) medium containing penicillin (150 μg ml⁻¹) and were then inoculated into 2 l of the same medium at a ratio of 1:100. The cells were grown at 30°C until the optical density

at 600 nm (OD${}_{600}$) of the culture reached 0.6. Then, 1 mM IPTG was added to the culture, and growth was continued for another 3 h. Cells were harvested by centrifugation at 4°C and washed with cell lysis and column buffer (20 mM sodium HEPES, 500 mM NaCl, 0.1 mM EDTA, 0.1% Triton X-100, pH 8) for ResD. The ResE-containing cells were washed with P buffer (50 mM HEPES, 50 mM KCl, 5 mM MgCl${}_{2}$, pH 8.0). The cell pellet was stored at $-70^{\circ} \mathrm{C}$.

**Cell lysis.** The cell pellets were suspended on ice in 40 ml of their respective buffer, column buffer or P buffer, containing 1 mM phenylmethylsulphonyl fluoride (PMSF) and were immediately subjected to sonication. After a 5 min sonication on ice at an output of 100 W, the cell lysate was centrifuged at 120 000 g for 1 h at 4°C. The supernatant fraction was filtered through a 0.45 $\mu$m membrane.

**ResD purification.** The lysed cell supernatant fraction was applied to a 1.5 ml Chitin Bead affinity column which was attached to a Waters 650E FPLC (fast protein liquid chromatography) system. The column was washed with column buffer until the OD${}_{280}$ of the eluate was less than 0.02, then the column was washed with three volumes of cleavage buffer (with DTT) (20 mM HEPES, 50 mM NaCl, 0.1 mM EDTA, 30 mM DTT, pH 8). The column flow was stopped and the column was left at 4°C overnight. The ResD protein was eluted using cleavage buffer (without DTT).

**ResE purification.** The supernatant fraction was applied to a 2 ml glutathione–agarose (Pharmacia Biotech) affinity column equilibrated with P buffer. The column was washed with P buffer until the OD${}_{280}$ of the eluate was less than 0.02. The bound GST–ResE fusion protein was cleaved using P buffer containing 20% (w/v) glycerine and 20 U of thrombin (Sigma) and was shaken at 4°C overnight; the eluted ResE was collected in the flow-through by centrifugation. The protein was aliquoted and stored at $-70^{\circ} \mathrm{C}$.

**Phosphorylation of ResD by ResE**

Phosphorylation assays were carried out at room temperature in the presence of 5 $\mu$Ci of [$\gamma$-${}^{32}$P]-ATP (specific activity 6000 Ci mmol${}^{-1}$; Amersham Biotech) in P-buffer for 20 min. The reactions were initiated by the addition of [$\gamma$-${}^{32}$P]-ATP into a 20 $\mu$l reaction mixture, terminated by the addition of an equal volume of 6× sodium dodecyl sulphate (SDS) sample buffer, and subjected to SDS–PAGE on 10% polyacrylamide gels (Laemmli, 1970). Labelled proteins were detected by autoradiography of dried gels with X-ray film (Fuji Medical X-Ray Film).

**Determination of oligomeric state of ResD and ResD~P by gel filtration**

For these studies, ResD was phosphorylated by GST–ResE bound to glutathione beads and separated from the GST–ResE by a procedure described previously for phosphorylation of PhoP by GST–PhoR (Liu and Hulett, 1997). ResD or ResD~P were individually loaded onto an HR10/30

Superdex 75 column (Pharmacia), pre-equilibrated with 50 mM tris-HCl (pH 7.5), 100 mM KCl and 4 mM DTT, and then the protein was eluted at 0.5 ml min${}^{-1}$ and collected in 400 $\mu$l fractions. The protein molecular weight standards (MW-GF-200 kit; Sigma) were applied to the column under the same conditions, and the elution volume and molecular weight of each standard were used to generate a standard curve to determine the relative molecular mass of the ResD protein.

**Gel shift assays**

To make probes of the ctaAB promoter for gel shift assays, we used pES68, which contains the ctaAB intercistronic region and the 5′ coding region of ctaA. pES68 was digested with HindIII and end-labelled with Klenow fragment in the presence of [$\alpha$-${}^{32}$P]-dATP. The inserts were released by Xhol digestion. The probes were isolated as described by Manzara et al. (1991). The DNA probe was further purified using Elutip-d column (Schleicher and Schuell), as instructed by the manufacturer. In each reaction, ResE (1.6 $\mu$M) was incubated with 74 nM, 148 nM, 296 nM, 593 nM, 1.18 $\mu$M, 2.37 $\mu$M and 4.74 $\mu$M of ResD in the presence of 5 mM ATP at room temperature for 15 min in binding buffer [50 mM piperazine-$N,N'$-bis(2-ethane-sulphonic acid) (PIPES; pH 6.1), 50 mM NaCl, 1 mM EDTA, 10% glycerol, 4 mM DTT, 4 mM MgCl${}_{2}$]. The probe (10 000 c.p.m.) was added to the reaction, and the incubation was continued for another 30 min. The samples were loaded onto a 5% native polyacrylamide gel made in 1× Tris-borate-EDTA (Sambrook, 1989). The gel was run at 4°C for 2 h, dried by vacuum and exposed to X-ray film.

**DNase I footprinting of the ctaAB promoter**

The coding strand of each promoter fragment was labelled as described above as the complete ctaA promoter cloned in pCR2.1 (pES68), and the deleted ctaA promoters cloned in pCR2.1 (plasmids pXH43, pXH28, pXH37, pXH38 and pXH41) are in same orientation. To label the non-coding strand, pES68 was digested by Xhol and was labelled with Klenow fragment in the presence of [$\alpha$-${}^{32}$P]-dATP and [$\alpha$-${}^{32}$P]-dCTP. The plasmid was then digested by HindIII to release the insert. Purification of each probe was carried out as described above. The DNA-binding reaction described above was used but with 20 000 c.p.m. of probe. The amounts of ResD used in the reactions with the complete promoter probes from pES68 varied (0, 9 nM, 18 nM, 37 nM, 74 nM, 148 nM, 296 nM and 594 nM). Reactions containing deleted ctaA promoter probes contained ResD concentrations of 0, 37 nM, 74 nM, 296 nM, 1.18 $\mu$M or 2.37 $\mu$M. After the 30 min incubation at room temperature, 3 $\mu$l of DNase I (0.9 U) in 5 mM MgCl${}_{2}$ and 5 mM CaCl${}_{2}$ was added to each reaction mixture, and digestion was conducted for 40 s for the ResD-containing samples and 20 s for ResD-free samples. The reactions were stopped, and the samples were prepared as described by Manzara et al. (1991). The samples were run on a 6% polyacrylamide gel containing 7 M urea in 1× Tris-borate-EDTA for 2 h at 65 mA. The gel was then dried and exposed to an X-ray film with an intensifying screen at $-70^{\circ} \mathrm{C}$.

1218 X. Zhang and F. M. Hulett

Analysis of DNase I footprint data for calculation of the ResD and ResD~P dissociation constant at each binding site

The protocol of Brenowitz *et al.* (1996) for quantification of protein-binding equilibria was followed. The autoradiogram (data shown in Fig. 3) was scanned using a HP Scanjet 3C scanner with a transparency adapter and the ResD or ResD~P protection quantified using Bio-Rad QUANTITY ONE 4.1.0 graphics software. All data were fitted using the nonlinear least squares fitting routine in the program TABLE-CURVE 2D (SPSS Scientific).

### Growth conditions and β-galactosidase activity

The cells were grown in modified Schaeffer’s sporulation medium (SSM) (Leighton and Doi, 1971) and supplemented with 0.5% glucose (2× SSG). A single colony of strains containing *ctaA-lacZ* promoter fusions was used to inoculate 5 ml LB starter cultures which were grown for several hours and used to inoculate 200 ml 2× SSG and incubated overnight. The overnight cultures were used to inoculate 400 ml SSG to approximately 0.2 OD<sub>540</sub>. Growth was measured at OD<sub>540</sub> and β-galactosidase specific activity was assayed every hour for 11 h. β-Galactosidase activity was detected using the method of Ferrari *et al.* (1986). One unit was defined as 0.33 μM o-nitrophenol produced min<sup>-1</sup> at 37°C. The specific activity was calculated as activity per milligram of protein.

### Acknowledgements

This work was supported by NIH grant GM33471 to F.M.H. We thank Andrew Mesecar for assistance with *K*<sub>D</sub> determinations. We thank Wei Liu and Elena Sarkova for the gel filtration data.

### References

Baikalov, I., Schroder, I., Kaczor-Grzeskowiak, M., Grzeskowiak, K., Gunsalus, R.P., and Dickerson, R.E. (1996) Structure of the Escherichia coli response regulator NarL. *Biochemistry* 35: 11053–11061.

Baikalov, I., Schroder, I., Kaczor-Grzeskowiak, M., Cascio, D., Gunsalus, R.P., and Dickerson, R.E. (1998) NarL dimerization? Suggestive evidence from a new crystal form. *Biochemistry* 37: 3665–3676.

Birkey, S.M., Liu, W., Zhang, X., Duggan, M.F., and Hulett, F.M. (1998) Pho signal transduction network reveals direct transcriptional regulation of one two-component system by another two-component regulator: Bacillus subtilis PhoP directly regulates production of ResD. *Mol Microbiol* 30: 943–953.

Boucher, P.E., Murakami, K., Ishihama, A., and Stibitz, S. (1997) Nature of DNA binding and RNA polymerase interaction of the Bordetella pertussis BvgA transcriptional activator at the *fha* promoter. *J Bacteriol* 179: 1755–1763.

Brenowitz, M., Senear, D., and Kinston, R. (eds). (1996)

DNase I footprint analysis of protein-DNA binding. New York: John Wiley and Sons.

Dahl, J., Wei, B., and Kadner, R. (1997) Protein phosphorylation affects binding of the Escherichia coli transcription activator UhpA to the *uhpT* promoter. *J Biol Chem* 2: 1910–1919.

Eder, S., Liu, W., and Hulett, F.M. (1999) Mutational analysis of the *phoD* promoter in *Bacillus subtilis*: implications for PhoP binding and promoter activation of Pho regulon promoters. *J Bacteriol* 181: 2017–2025.

Fabret, C., and Hoch, J.A. (1998) A two-component signal transduction system essential for growth of *Bacillus subtilis*: implications for anti-infective therapy. *J Bacteriol* 180: 6375–6383.

Fabret, C., Feher, V.A., and Hoch, J.A. (1999) Two-component signal transduction in *Bacillus subtilis*: how one organism sees its world. *J Bacteriol* 181: 1975–1983.

Ferrari, E., Howard, S., and Hoch, J.A. (1986) Effect of stage 0 sporulation mutants on subtilisin expression. *J Bacteriol* 170: 173–179.

Fiedler, U., and Weiss, V. (1995) A common switch in activation of the response regulators NtrC and PhoB: phosphorylation induces dimerization of the receiver modules. *EMBO J* 14: 3696–3705.

Harlocker, S.L., Bergstrom, L., and Inouye, M. (1995) Tandem binding of six OmpR proteins to the *ompF* upstream regulatory sequence of *Escherichia coli*. *J Biol Chem* 270: 26849–26856.

Harrison-McMonagle, P., Denissova, N., Martinez-Hackert, E., Ebright, R.H., and Stock, A.M. (1999) Orientation of OmpR monomers within an OmpR: DNA complex determined by DNA affinity cleaving. *J Mol Biol* 285: 555–566.

Huang, K.J., and Igo, M.M. (1996) Identification of the bases in the *ompF* regulatory region, which interact with the transcription factor OmpR. *J Mol Biol* 262: 615–628.

Kasahara, M., Makino, K., Amemura, M., Nakata, A., and Shinagawa, H. (1991) Dual regulation of the *ugp* operon by phosphate and carbon starvation at two interspaced promoters. *J Bacteriol* 173: 549–558.

Kim, S.K., Makino, K., Amemura, M., Shinagawa, H., and Nakata, A. (1993) Molecular analysis of the *phoH* gene, belonging to the phosphate regulon in *Escherichia coli*. *J Bacteriol* 175: 1316–1324.

Laemmli, U.K. (1970) Cleavage of structural proteins during the assembly of the head of bacteriophage T4. *Nature* 227: 680–685.

Leighton, T.J., and Doi, R.H. (1971) The stability of messenger ribonucleic acid during sporulation in *Bacillus subtilis*. *J Biol Chem* 246: 3189–3195.

Liu, W., and Hulett, F.M. (1997) *Bacillus subtilis* PhoP binds to the *phoB* tandem promoter exclusively within the phosphate starvation-inducible promoter. *J Bacteriol* 179: 6302–6310.

Liu, W., and Hulett, F.M. (1998) Comparison of PhoP binding to the *tuaA* promoter with PhoP binding to other Pho-regulon promoters establishes a *Bacillus subtilis* Pho core binding site. *Microbiology* 144: 1443–1450.

Liu, X., and Taber, H. (1998) Catabolite regulation of the *Bacillus subtilis ctaBCDEF* gene cluster. *J Bacteriol* 180: 6154–6163.

Makino, K., Kim, S.K., Shinagawa, H., Amemura, M., and
© 2000 Blackwell Science Ltd, *Molecular Microbiology*, **37**, 1208–1219

Nakata, A. (1991) Molecular analysis of the cryptic and functional phn operons for phosphonate use in *Escherichia coli* K-12. *J Bacteriol* 173: 2665–2612.

Makino, K., Amemura, M., Kawamoto, T., Kimura, S., Shinagawa, H., Nakata, A., and Suzuki, M. (1996) DNA binding of PhoB and its interaction with RNA polymerase. *J Mol Biol* 259: 15–26.

Manzara, T., Carrasco, P., and Gruissem, W. (1991) Developmental and organ-specific changes in promoter DNA–protein interactions in the tomato *rbcs* gene family. *Plant Cell* 3: 1305–1316.

Martinez-Hackert, E., and Stock, A.M. (1997) The DNA-binding domain of OmpR: crystal structures of a winged helix transcription factor. *Structure* 5: 109–124.

Nakano, M.M., and Hulett, F.M. (1997) Adaptation of *Bacillus subtilis* to oxygen limitation. *FEMS Microbiol Lett* 157: 1–7.

Nakano, M.M., Zuber, P., Glaser, P., Danchin, A., and Hulett, F.M. (1996) Two-component regulatory proteins ResD–ResE are required for transcriptional activation of *fnr* upon oxygen limitation in *Bacillus subtilis*. *J Bacteriol* 178: 3796–3802.

Nakano, M.M., Hoffmann, T., Zhu, Y., and Jahn, D. (1998) Nitrogen and oxygen regulation of *Bacillus subtilis nasDEF* encoding NADH-dependent nitrite reductase by TnrA and ResDE. *J Bacteriol* 180: 5344–5350.

Nakano, M.M., Zhu, Y., LaCelle, M., Zhang, X., and Hulett, F.M. (2000) Interaction of ResD with regulatory regions of anaerobically induced genes in *Bacillus subtilis*. *Mol Microbiol* 37: 1198–1207.

Okamura, H., Hanaoka, S., Nagadoi, A., Makino, K., and Nishimura, Y. (2000) Structural comparison of the PhoB

and OmpR DNA-binding/transactivation domains and the arrangement of PhoB Molecules on the phosphate box. *J Mol Biol* 295: 1225–1236.

Sambrook, J., Fritsch, E.F., and Maniatis, T. (1989). *Molecular Cloning*. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.

Shimotsu, H., and Henner, D.J. (1986) Construction of a single-copy integration vector and its use in analysis of regulation of the *trp* operon of *Bacillus subtilis*. *Gene* 43: 85–94.

Sola, M., Gomis-Ruth, F.X., Serrano, L., Gonzalez, A., and Coll, M. (1999) Three-dimensional crystal structure of the transcription factor PhoB receiver domain. *J Mol Biol* 285: 675–687.

Sorokin, A., Zumstein, E., Azevedo, V., Ehrlich, S.D., and Serror, P. (1993) The organization of the *Bacillus subtilis* 168 chromosome region between the *spoVA* and *serA* genetic loci, based on sequence data. *Mol Microbiol* 10: 385–395.

Sun, G., Sharkova, E., Chesnut, R., Birkey, S., Duggan, M.F., Sorokin, A., *et al.* (1996) Regulators of aerobic and anaerobic respiration in *Bacillus subtilis*. *J Bacteriol* 178: 1374–1385.

Svensson, B., and Hederstedt, L. (1994) *Bacillus subtilis* CtaA is a heme-containing membrane protein involved in heme A biosynthesis. *J Bacteriol* 176: 6663–6671.

Svensson, B., Lubben, M., and Hederstedt, L. (1993) *Bacillus subtilis* CtaA and CtaB function in haem A biosynthesis. *Mol Microbiol* 10: 193–201.

Svensson, B., Andersson, K.K., and Hederstedt, L. (1996) Low-spin heme A in the heme A biosynthetic protein CtaA from *Bacillus subtilis*. *Eur J Biochem* 238: 287–295.
