
Molecular Microbiology (2000) 37(4), 752–762

# Biosynthesis of the anti-parasitic agent megalomicin: transformation of erythromycin to megalomicin in Saccharopolyspora erythraea

Yanina Volchegursky, Zhihao Hu, Leonard Katz and Robert McDaniel*

Kosan Biosciences, 3832 Bay Center Place, Hayward, CA 94545, USA.

## Summary

Megalomicin is a therapeutically diverse compound which possesses antiparasitic, antiviral and antibacterial properties. It is produced by *Micromonospora megalomicea* and differs from the well-known macrolide antibiotic erythromycin by the addition of a unique deoxyamino sugar, megosamine, to the C-6 hydroxyl. We have cloned and sequenced a 48 kb segment of the megalomicin (*meg*) biosynthetic gene cluster which contains the modular polyketide synthase (PKS) and the complete pathway for megosamine biosynthesis. The similarities and distinctions between the related megalomicin and erythromycin gene clusters are discussed. Heterologous expression of the megalomicin PKS in *Streptomyces lividans* led to production of 6-deoxyerythronolide B, the same macrolactone intermediate for erythromycin. A 12 kb fragment harbouring the putative megosamine pathway was expressed in *Saccharopolyspora erythraea*, resulting in the conversion of erythromycin to megalomicin. Considering the extensive knowledge surrounding the genetic engineering of the erythromycin PKS and the familiarity with genetic manipulation and fermentation of *S. erythraea*, the ability to produce megalomicin in this strain should allow the engineering of novel megalomicin analogues with potentially improved therapeutic activities.

## Introduction

The megalomicins are 6-O-glycosides of erythromycin C with acetyl or propionyl groups esterified to the 3‴ or 4‴ hydroxyls of the mycarose sugar (Fig. 1). They were discovered in 1969 as antibacterial agents produced by *Micromonospora megalomicea* sp. n. (Weinstein et al., 1969). The deoxyamino sugar at C-6 was named ‘megosamine’ (Nakagawa and Omura, 1984) and is identical in structure to L-rhodosamine (or N-dimethylaldanosamine). Although the initial structural assignment of megalomicin was in error, a thorough reassessment of NMR data coupled with an X-ray crystal structure of a megalomicin A derivative (Nakagawa and Omura, 1984) established the current structures.

Therapeutic interest in megalomicin stems from its several different observed biological activities. As antibacterials, they act like erythromycin, which inhibits protein synthesis through selective binding to the bacterial 50S ribosomal RNA. The potency, spectrum of activity and toxicity of the megalomicins is very similar to that of erythromycin A (Weinstein et al., 1969). Megalomicin also affects protein trafficking in eukaryotic cells (Bonay et al., 1996, 1997). Although the mechanism of this action is not entirely clear, it appears to involve inhibition of vesicular transport between the medial and trans Golgi, resulting in undersialylation of proteins. They also strongly inhibit the ATP-dependent acidification of lysosomes *in vivo* (Bonay et al., 1997) and cause an anomalous glycosylation of viral proteins, which may be responsible for their antiviral activity against herpes (Alarcon et al., 1984, 1988). Strikingly, the megalomicins are also potent antiparasitic agents, effective against *Plasmodium falciparum*, *Trypanosoma* sp. and *Leishmania donovani* (Bonay et al., 1998). As erythromycin does not have antiparasitic activity, the antiparasitic action of megalomicin is most probably related to the presence of the deoxyamino sugar at C-6. In animal studies, megalomicin was shown to have exceedingly low toxicity, raising the prospects that derivatives could be developed into potent drugs for treatment of malarial disease.

The aglycone backbone of both megalomicin and erythromycin is the complex polyketide 6-deoxyerythronolide B (6-dEB), produced from the successive condensations of a propionyl-CoA starter unit and 6 methylmalonyl-CoA extender units (Fig. 2). Complex polyketides are assembled by modular polyketide synthases (PKSs), which are composed of multifunctional polypeptides that contain the activities (as enzymatic domains) for the condensation and subsequent reductions required to produce the polyketide chain (Katz, 1997; Cane et al., 1998). In modular PKSs, each module

Accepted 9 June, 2000. *For correspondence. E-mail mcdaniel@kosan.com; Tel. (+1) 510 732 8400; Fax (+1) 510 732 8401.*

© 2000 Blackwell Science Ltd

Engineered biosynthesis of megalomicin 753

![Structures of megalomics and erythromycin A](image1.png)

Fig. 1. Structures of megalomics and erythromycin A.

contains the required catalytic domains for a single round of condensation and reduction steps; 6-dEB synthesis requires six modules (Donadio *et al.*, 1991). During assembly, the nascent polyketide is tethered to the PKS through thioester linkages. The prototypical modular PKS is 6-deoxyerythronolide B synthase (DEBS) (Cortés *et al.*, 1990; Donadio *et al.*, 1991), encoded by the *eryA* genes from *Saccharopolyspora erythraea*.

Making derivatives of megalomicin through genetic engineering would require either the ability to manipulate genetically the producing host *M. megalomicea* or the outcloning of the megalomicin biosynthetic genes in a more genetically tractable host. The proposed biosynthetic pathway of megalomicin and its relationship to that of erythromycin is shown in Fig. 2. Both pathways are identical through erythromycin C, the penultimate intermediate of erythromycin A and megalomicin A. We believed therefore that the genes for synthesis of the polyketide, L-mycarose and D-desosamine (Gaisser *et al.*, 1997; Summers *et al.*, 1997) would be similar in the corresponding hosts and that we could use the erythromycin (*ery*) genes as probes to locate the megalomicin (*meg*) biosynthesis cluster. Furthermore, we believed that the *meg* cluster would contain an additional set of genes for synthesis of the unique deoxysugar L-megosamine and that expression of these genes in *S. erythraea* would result in the production of megalomicin.

We describe the characterization of a major portion of the megalomicin biosynthesis gene cluster containing the megalomicin PKS, genes for the complete megosamine pathway and portions of the mycarose and desosamine pathways. Transfer of the megosamine genes to *S. erythraea* resulted in production of megalomics. In contrast to *M. megalomicea*, *S. erythraea* is a well-characterized

![Erythromycin and proposed megalomicin biosynthetic pathways](image2.png)

Fig. 2. Erythromycin and proposed megalomicin biosynthetic pathways.

© 2000 Blackwell Science Ltd, *Molecular Microbiology*, **37**, 752–762

754 Y. Volchegursky et al.

Fig. 3. Organization of the megalomicin (meg) and erythromycin (ery) gene clusters. A series of four overlapping cosmids was obtained which contain the meg gene cluster. The portions of the two gene clusters depicted here share a similar organization except for the megD/Y region. The similar domain organization within the polyketide synthases is also shown. KS, ketosynthase; AT, acyltransferase; KR, ketoreductase; DH, dehydratase; ER, enoylreductase; ACP, acyl carrier protein; TE, thioesterase.

host with established genetic engineering tools and extensive fermentation development. Thus, the conversion of S. erythraea into a megalomicin producer enables the prospect of generating new antiparasitic compounds by genetic engineering.

**Results**

**Nucleotide sequence of the meg gene cluster**

A series of four overlapping inserts containing the meg cluster (Fig. 3) was isolated from a cosmid library prepared from total genomic DNA of M. megalomicea and covers > 100 kb of the genome. A contiguous 48 kb segment which encodes the megalomicin PKS and several deoxysugar biosynthetic genes was sequenced and analysed. The segment contains 17 complete open reading frames (ORFs) as well as an incomplete ORF at each end, organized as shown in Fig. 3. These sequence data have been submitted to the DDBJ/EMBL/GenBank databases under the accession number AF263245.

PKS genes. The ORFs megAl, megAll and megAllI encode the polyketide synthase responsible for synthesis of 6-dEB. The enzyme complex meg DEBS is highly similar to ery DEBS, with each of the three predicted polypeptides sharing an average of 83% overall similarity

with its ery PKS counterpart. Both PKSs are composed of six modules (two modules per polypeptide) and each module is organized in an identical manner (Fig. 3). A dendrogram analysis (Schwecke et al., 1995) using 70 acyltranferase (AT) domains revealed that the six meg extender AT domains cluster with AT domains that incorporate methylmalonyl CoA (not shown). The loading module of meg DEBS also lacks a KSQ domain which is utilized by most macrolide PKSs for decarboxylation of the starter unit to initiate polyketide synthesis (Kuhstoss et al., 1996; Kakavas et al., 1997; Xue et al., 1998; Bisang et al., 1999), implying that priming begins with a propionate unit. In addition, a conserved Gly to Pro substitution in the NADPH-binding region of the ketoreductase (KR) domain of module 3 is observed in meg DEBS, which has been proposed to account for its inactivity in ery DEBS (Donadio et al., 1991).

Deoxysugar genes. BLAST (Altschul et al., 1990) analysis of the genes flanking the PKS indicated that 12 complete ORFs and one partial ORF appear to encode functions required for synthesis of one of the three megalomicin deoxysugars. Tentative assignment of each ORF to a specific deoxysugar pathway was made based on comparison to the ery genes and other related genes involved in deoxysugar biosynthesis (Table 1). Three

© 2000 Blackwell Science Ltd, Molecular Microbiology, 37, 752–762

Table 1. Deduced functions of genes identified in the megalomicin gene cluster.

| Gene      | Closest match (polypeptide)${}^{a}$                                                                 | % Sim${}^{a}$ | Proposed pathway    | Proposed function               | Reference                          |
|-----------|-----------------------------------------------------------------------------------------------|----------------|----------------------|----------------------------------|------------------------------------|
| megT      | EryBVI                                                                                         |                | Mycarose/megosamine  | 2,3-Dehydratase                 | Summers et al. (1997); Gaisser et al. (1997) |
| megDVI    | EryCII                                                                                         | 63             | Megosamine          | 3,4-Isomerase                   | Summers et al. (1997)              |
| megDI     | EryCIII                                                                                        | 79             | Megosamine          | Glycosyltransferase             | Summers et al. 1997)               |
| megY      | AcyA (Streptomyces thermotolerans)                                                              | 52             | Mycarose O-acyl-transferase | Arisawa et al. (1994)            |
| megDII    | EryCI                                                                                          | 58             | Megosamine          | Aminotransferase                | Dhillon et al. (1989); Summers et al. (1997) |
| megDIII   | DesVI (Streptomyces venezuelae)                                                                 | 61             | Megosamine          | Dimethyltransferase             | Xue et al. (1998)                  |
| megDIV    | DmnU (S. peucetius)                                                                            | 65             | Megosamine          | 3,5-Epimerase                   | Olano et al. (1999)                |
| megDV     | Dehydrogenase (Amycolatopsis orientalis)                                                       | 61             | Megosamine          | 4-Ketoreductase                 | Summers et al. (1997); van Wageningen et al. (1998) |
| megDVII   | EryBII                                                                                         | 73             | Megosamine          | 2,3-Reductase                   | Summers et al. (1997)              |
| megBV     | EryBV                                                                                          | 86             | Mycarose            | Glycosyltransferase             | Summers et al. (1997; Gaisser et al. (1997) |
| megBIV    | EryBIV                                                                                         | 80             | Mycarose            | 4-Ketoreductase                 | Summers et al. (1997); Gaisser et al. (1997) |
| megAI     | EryAl                                                                                          | 81             | 6-dEB               | Polyketide synthase             | Donadio and Katz (1992)            |
| megAll    | EryAll                                                                                         | 85             | 6-dEB               | Polyketide synthase             | Donadio and Katz (1992)            |
| megAIII   | EryAllI                                                                                        | 83             | 6-dEB               | Polyketide synthase             | Donadio and Katz (1992)            |
| megCII    | EryCII                                                                                         | 82             | Desosamine          | 3,4-Isomerase                   | Summers et al. (1997)              |
| megCIII   | EryCIII                                                                                        | 89             | Desosamine          | Glycosyltransferase             | Summers et al. (1997)              |
| megBIll   | EryBII                                                                                         | 87             | Mycarose            | 2,3-Reductase                   | Summers et al. (1997)              |
| megH      | EryH                                                                                            | 84             |                    | Thioesterase                    | Haydock et al. (1991)              |
| megF      | EryF                                                                                            |                |                    | C-6 Hydroxylase                 | Weber et al. (1991)                |

a. Determined by BLASTX analysis using default parameters.

756 Y. Volchegursky et al.

![Chemical Structures](chemical_structures.png)

Fig. 4. Proposed biosynthetic pathways for synthesis of the deoxysugars involved in erythromycin and megalomicin synthesis. Similar routes to desosamine and mycarose have been previously proposed in erythromycin biosynthesis (Gaisser *et al.*, 1997; Summers *et al.*, 1997) and for desosamine in methymycin/picromycin biosynthesis (Zhao *et al.*, 1998).

ORFs, *megBV*, *megCIII* and *megDI*, encode glycosyltransferases, apparently one for attachment of each deoxysugar to the macrolide. *MegBV* was most similar to *EryBV*, the erythromycin mycarosyltransferase, and hence was assigned to the mycarose pathway in the *meg* cluster. The closest match for both of the remaining glycosyltransferases was *EryCIII*, the desosaminyltransferase in erythromycin biosynthesis. Given the higher degree of similarity between *EryCIII* and *MegCIII* (Table 1), *MegCIII* was designated the desosaminyltransferase, leaving *MegDI* as the proposed megosaminyltransferase. In similar fashion, assignments were made accordingly for: *MegCII* and *MegDVI*, two putative 3,4-isomerases similar to *EryCII*; *MegBII* and *MegDVII*, 2,3-reductases homologous to *EryBII*; *MegBIV* and *MegDV*, putative 4-ketoreductases similar to *EryBIV* (Table 1). The remaining ORFs involved in deoxysugar biosynthesis, *megT*, *megDII*, *megDIII* and *megDIV*, each encode a putative 2,3-dehydratase, aminotransferase, dimethyltransferase and 3,5-epimerase respectively (Table 1). As both the megosamine and desosamine pathways require an aminotransferase and a dimethyltransferase, and as mycarose and megosamine each require a 2,3-dehydratase and a 3,5-epimerase, assignments of these four genes to a specific pathway could not be made on the basis of sequence comparison alone. However, the last three are implicated in megosamine biosynthesis by experiments described below.

Other genes. Two additional complete ORFs, designated megY and megH, and an incomplete ORF, designated megF, were also identified in the cluster. MegH and MegF share high degrees of similarity with EryH and EryF. EryH and homologues in other macrolide gene clusters are thioesterase-like proteins with unknown function in polyketide gene clusters (Haydock *et al.*, 1991; Xue *et al.*, 1998; Butler *et al.*, 1999; Tang *et al.*, 1999). EryF encodes the erythronolide B C-6 hydroxylase (Fig. 2) (Weber *et al.*, 1991; Andersen and Hutchinson, 1992). MegY does not have an ery counterpart, but appears to belong to a (small) family of O-acyltransferases that transfer short acyl chains to macrolides. Two classes exist: AcyA and MdmB transfer acetyl or propionyl groups to the C-3 hydroxyls on 16-membered macrolide rings (Hara and Hutchinson, 1992; Arisawa *et al.*, 1994); CarE and Mpt transfer isovalerate or propionate to the mycarosyl moiety of carbomycin and midecamycin respectively (Epp *et al.*, 1989; Arisawa *et al.*, 1993; Gu *et al.*, 1996). The structures of various megalomicins suggest that MegY belongs to the latter class and is the acyltransferase which converts megalomicin A to megalomicins B, C1 or C2 (verified experimentally below).

Heterologous expression of the meg PKS genes

The wild-type and genetically modified versions of the ery DEBS have been used extensively in heterologous Streptomyces hosts for enzyme studies and the production of novel polyketide compounds. Given the similarities between the ery and meg DEBSs, we wished to compare their production characteristics in a commonly used Streptomyces host strain. The three megA ORFs were cloned into the expression plasmid pKAO127/kan' (Ziermann and Betlach, 1999) in place of the eryA ORFs. Both plasmids, pKAO127/kan' encoding ery DEBS and pKOS108-06 encoding meg DEBS, were introduced in Streptomyces lividans K4-114 and the production of 6-dEB was determined in shake-flask fermentations. The production profiles were similar in both cases and the maximum titre of 6-dEB was between 30 and 40 mg l⁻¹. In addition, both PKSs produced small amounts (~5%) of 8,8a-deoxyoleandrolide, which results from the priming of the PKS with acetate instead of propionate (Kao *et al.*, 1994a). This observation indicates that the loading AT domains of the PKSs display similar relaxed specificities towards starter units.

Conversion of erythromycin to megalomicin in Saccharopolyspora erythraea

An examination of the meg cluster revealed that the putative megosamine biosynthetic genes are clustered directly upstream of the PKS genes. If the hypothesis that these genes are sufficient for biosynthesis and attachment of megosamine to an erythromycin intermediate is correct, then functional expression of these genes in a strain which produces erythromycin, such as S. erythraea, should result in production of megalomicin. A 12 kb DNA fragment carrying all the genes between the leftmost Xhol site and the EcoRI site (Fig. 3) was integrated in the chromosome of S. erythraea using the site-specific integrating vector pSET152 (Bierman *et al.*, 1992). It was surmised that the left and right ends of this fragment would contain necessary promoter regions for transcription of the convergent set of genes in M. megalomicea and that they would be likely to operate in S. erythraea.

Fermentation broth from S. erythraea/KOS97-42 containing the integrated meg genes was compared with fermentation broth from the parent S. erythraea strain grown under the same conditions. Liquid chromatography/mass spectrometry (LC/MS) analysis indicated that the engineered strain produced erythromycin A (~50 mg l⁻¹) and at least four new compounds not present in the unmodified parent S. erythraea. Authentic megalomicin standards obtained from M. megalomicea were used to confirm that these compounds corresponded to megalomicins A, B, C1 and C2. The two most abundant congeners, megalomicins B and C1, were present at about 5–10 mg l⁻¹ each. Megalomicin A and C2 were present in trace amounts (<1 mg l⁻¹). The presence of these compounds demonstrates that the predicted megosamine biosynthetic and glycosyltransferase genes are contained within the cloned meg fragment. The presence of the megalomicins B, C1 and C2 also provides evidence for the function of the O-acyl transferase MegY, which is present in the integrated meg fragment.

Discussion

The homologies observed among modular PKSs enabled the use of ery PKS genes to clone the meg biosynthetic gene cluster from M. megalomicea. The close similarities between the megalomicin and erythromycin biosynthetic pathways are also reflected in the overall organization of their genes and in the high degree of homology of the corresponding individual gene-encoded polypeptides. Because both PKSs synthesize the same polyketide, sequence comparisons between the two may be useful for determining the importance of particular amino acids when engineering the synthases. A more detailed examination reveals that the highest degree of homology exists within the catalytic domains of the PKS and the lowest in the regions between domains, modules or peptides. This could suggest that catalytic domains are important for recognition of polyketide intermediates rather than the interdomain, intermodule and interpolypeptide

758 Y. Volchegursky et al.

regions, which may be more important for protein–protein interactions (Gokhale *et al.*, 1999).

As seen in Fig. 3, the ~40 kb segments of the two clusters from the ery/megBV on the left to the ery/megF genes retain a nearly identical organizational arrangement. The notable differences in this region are eryG and IS1136, which are absent from the segment of the meg cluster that was analysed. The eryG gene encodes an S-adenosylmethionine (SAM)-dependent mycarosyl methyltransferase that converts erythromycin C to erythromycin A (Fig. 2) (Weber *et al.*, 1990; Haydock *et al.*, 1991). The mycarose moiety is modified by esterification (MegY) in megalomicin biosynthesis (Fig. 2) and therefore the absence of an eryG homologue would be expected in the meg cluster. The IS1136 element located between eryAI and eryAII (Donadio and Staver, 1993) is not known to play a role in erythromycin biosynthesis and its origin in the ery cluster has not been determined.

Upstream of the common meg/eryBIV and BV genes, the gene clusters diverge. The ~6 kb segment between eryBV and eryK, the left border of the ery gene cluster (Pereda *et al.*, 1997), contains the remaining genes required for mycarose (eryBVI and BVII) and desosamine biosynthesis (eryCIV, CV and CVI) and the C-12 hydroxylase (eryK) (Stassi *et al.*, 1993). In contrast, the region upstream of megBV encodes a set of genes (megDI–DVII and megY) which can account for all the activities unique to megalomicin biosynthesis (Fig. 3). Because introduction of this meg DNA segment into *S.* erythraea results in production of megalomicins, it is clear that these genes encode the functions for TDP-megosamine biosynthesis and transfer to its putative substrate erythromycin C and to acylate megalomicin A (Fig. 2). The remaining unsequenced region upstream of megDVI should therefore encode genes only for mycarose and desosamine biosynthesis. Furthermore, if the organization resembles that of the left arm of the ery cluster, it is quite possible that a megosamine biosynthesis ‘island’ was formed via an insertion of the megD and megY genes into an existing erythromycin or common ancestral gene cluster.

Olano *et al.* (1999) have recently described a pathway for biosynthesis of TDP-L-daunosamine (Fig. 4), a deoxy-sugar component of the anti-tumour compounds daunorubicin and doxorubicin produced by *Streptomyces peucetius*. Their pathway proposes four steps from the intermediate TDP-4-keto-6-deoxyglucose (Fig. 4) controlled by the gene cluster dnmJQTVUZ, although the functions for dnmQ and dnmZ could not be identified and the precise order of reactions in the pathway could not be determined. The genes dnmT, dnmU, dnmJ and dnmV each have proposed counterparts in the meg cluster: megT, megDIV, megDII and megDV respectively.

It is possible to describe a pathway to convert TDP-2,6-dideoxy-3,4-diketo-D-hexose (or its enol tautomer), the last intermediate common to the mycarose and megosamine pathways, to TDP-megosamine through the sequence of 5-epimerization, 4-ketoreduction, 3-amination and 3-N-dimethylation using the genes megDIV, megDV, megDII and megDIII (Fig. 4). This uses the same functions proposed for biosynthesis of TDP-daunosamine by Olano *et al.* (1999) but in a different sequential order. However, it does not account for the megDVI and megDVII genes as their activities are not required for this route. A parallel pathway which uses these genes is also shown in Fig. 4. In this alternative route, 2,3-reduction and 3,4-tautomerization are performed by the megDVII and megDVI gene products respectively. A unified single pathway that uses both 4-ketoreduction (megDV) and 2,3-reduction (megDVII) could not be determined. Because the entire gene set from megDVI to megDVII was introduced in *S.* erythraea to produce TDP-megosamine, it is not possible to determine which, if either, of the two alternative pathways is operative. This could be addressed through systematic gene disruption and complementation.

The 48 kb segment sequenced also contains genes required for synthesis of TDP-L-mycarose and TDP-D-desosamine (Fig. 4). The megCII gene encodes a putative 3,4-isomerase which catalyses the presumed first step in the committed TDP-desosamine pathway. The start codon of megCII overlaps the stop codon of megAIII in exactly the same manner as their erythromycin counterparts eryCII and eryAIII overlap (Summers *et al.*, 1997), suggesting that these genes are translationally coupled in both systems. The high degree of similarity between MegCII and EryCII suggests that the pathway to desosamine in the megalomicin- and erythromycin-producing organisms is most probably the same. Similarly, the finding that megBII and megBIV, encoding a 2,3-reductase and 4-ketoreductase, contain close homologues in the mycarose pathway for erythromycin also suggests that TDP-L-mycarose synthesis in the two host organisms is the same.

Of interest are the two genes that encode putative 2,3-reductases megBII and megDVII. Because MegBII most closely resembles EryBII, a known mycarose biosynthetic enzyme (Weber *et al.*, 1990), and because megBII resides in the same location of the meg cluster as its counterpart in the ery cluster, we were confident in assigning megBII to the mycarose pathway and megDVII to the megosamine pathway. Furthermore, the lower degree of similarity between MegDVII and either EryBII or MegBII (Table 1) provided a basis for assigning the opposite L- and D-isomeric substrates to each of the enzymes (Fig. 4). Finally, megT, which encodes a putative 2,3-dehydratase, is also related to a gene in the ery mycarose pathway eryBVI. In *S.* erythraea, the proposed intermediate generated by EryBVI represents

© 2000 Blackwell Science Ltd, *Molecular Microbiology*, **37**, 752–762
the first committed step in the biosynthesis of mycarose (Fig. 4). However, the proposed pathways in Fig. 4 suggest that this may be an intermediate common to both mycarose and megosamine biosynthesis in *M.* *megalomicea*. Therefore, *megT* is named after the designation of the equivalent gene in the daunosamine pathway, *dnmT* (Olano et al., 1999)

The host–vector system utilized here for expression of *meg* DEBS has been used previously for the heterologous expression of modular PKS genes for erythromycin (Kao et al., 1994b; Ziermann and Betlach, 1999), picromycin (Tang et al., 1999) and oleandomycin (Shah et al., 2000) as well as for the generation of novel polyketide backbones in which domains have been removed, added or exchanged in various combinations (McDaniel et al., 1999). Recently, hybrid polyketides have been generated through the co-expression of subunits from different PKS systems (Tang et al., 2000). How the subunits communicate with each other is not yet fully understood (Gokhale et al., 1999). The high degree of similarity between the *meg* and *ery* PKSs suggests that they could be interchanged and may provide additional insight into how PKS subunits recognize and communicate with each other.

Expression of the *megDVI–megDVII* segment in *S.* *erythraea* and the corresponding production of megalomicins in this host established the likely order of sugar attachment in megalomicin synthesis. Furthermore, it has provided a means to produce megalomicin in a more genetically friendly host organism, leading to the possibility of creating megalomicin analogues by manipulating the erythromycin PKS. We and colleagues have recently described the construction of over 60 6-dEB analogues produced by combinatorial biosynthesis using the *ery* DEBS (McDaniel et al., 1999; Xue et al., 1999). The titres of megalomicin could also be significantly increased above the 5 mg l⁻¹ obtained from *M.* *megalomicea* by introducing the genes into an industrially optimized strain of *S.* *erythraea*, many of which can produce as much as 10 g l⁻¹ of erythromycin.

### Experimental procedures

#### Bacterial strains, media and growth conditions

Routine DNA manipulations were performed in *Escherichia coli* XL1 Blue or *E.* *coli* XL1 Blue MR (Stratagene) using standard culture conditions (Sambrook et al., 1989). *M.* *megalomicea* subs. *nigra* NRRL3275 was obtained from the ATCC collection and cultured according to recommended protocols. For isolation of genomic DNA, *M.* *megalomicea* was grown in tryptone soya broth (TSB) (Hopwood et al., 1985) at 30°C. *S.* *lividans* K4-114 (Ziermann and Betlach, 1999), which carries a deletion of the actinorhodin biosynthetic gene cluster, was used as the host for expression of the *meg* DEBS genes. *S.* *lividans* strains were maintained on

R5 agar at 30°C and were grown in liquid yeast extract–malt extract (YEME) for preparation of protoplasts (Hopwood et al., 1985). *S.* *erythraea* NRRL2338 was used for expression of the megosamine genes. *S.* *erythraea* strains were maintained on R5 agar at 34°C and grown in liquid TSB for preparation of protoplasts.

#### Manipulation of DNA and organisms

Manipulation and transformation of DNA in *E.* *coli* was performed according to standard procedures (Sambrook et al., 1989) or to suppliers’ protocols. Protoplasts of *S.* *lividans* and *S.* *erythraea* were generated for transformation by plasmid DNA using the standard procedure (Hopwood et al., 1985). *S.* *lividans* transformants were selected on R5 using 2 ml of a 0.5 mg ml⁻¹ thiostrepton overlay. *S.* *erythraea* transformants were selected on R5 using 1.5 ml of a 0.6 mg ml⁻¹ apramycin overlay.

#### Isolation of the *meg* gene cluster

A cosmid library was prepared in SuperCos (Stratagene) from *M.* *megalomicea* total DNA partially digested with Sau3AI and introduced into *E.* *coli* using a Gigapack III XL (Stratagene) in vitro packaging kit. ³²P-labelled DNA probes encompassing the KS2 domain from *ery* DEBS, or a mixture of segments encompassing modules 1 and 2 from *ery* DEBS, were used separately to screen the cosmid library by colony hybridization. Several colonies which hybridized with the probes were further analysed by sequencing the ends of their cosmid inserts using T3 and T7 primers. BLAST (Altschul et al., 1990) analysis of the sequences revealed several colonies with DNA sequences highly homologous to genes from the *ery* cluster. Together with restriction analysis, this led to the isolation of two overlapping cosmids, pKOS079-93A and pKOS079-93D which covered ~45 kb of the *meg* cluster. A 400 bp PCR fragment was generated from the left end of pKOS079-93D and used to reprobe the cosmid library. Likewise, a 200 bp PCR fragment generated from the right end of pKOS079-93A was used to reprobe the cosmid library. Analysis of hybridizing colonies, as described above, resulted in identification of two additional cosmids pKOS079-138B and pKOS79-124B, which overlap the previous two cosmids. BLAST analysis of the far left and right end sequences of these cosmids indicated no homology to any known genes related to polyketide biosynthesis, and therefore indicates that the set of four cosmids spans the entire megalomicin biosynthetic gene cluster.

#### DNA sequencing and analysis

PCR-based double-stranded DNA sequencing was performed on a Beckman CEQ 2000 capillary sequencer using reagents and protocols provided by the manufacturer. A shotgun library of the entire cosmid pKOS079-93D insert was made as follows: DNA was first digested with *DraI* to eliminate the vector fragment, then partially digested with Sau3AI. After agarose electrophoresis, bands between 1 and 3 kb were excised from the gel and ligated with *BamHI*-digested pUC19. Another shotgun library was generated from
a 12 kb Xhol–EcoRI fragment subcloned from cosmid pKOS079-93A to extend the sequence to the *megF* gene. A 4 kb *BglII*–Xhol fragment from cosmid pKOS079-138B was sequenced by primer walking to extend the sequencing to the *megT* gene. Sequence was assembled using the SEQUENCHER (Gene Codes) software package and analysed with MacVector (Oxford Molecular Group) and the NCBI BLAST server (http://www.ncbi.nlm.nih.gov/blast/).

### Plasmids

Plasmid pKOS108-6 is a modified version of pKAO127′kan′ (Ziermann and Betlach, 1999, 2000), in which the eryAl–III genes between the *PadI* and EcoRI sites have been replaced with the *megAl–III* genes. This was carried out by first substituting a synthetic nucleotide DNA duplex (5′-TAAGAA TTCGGAGATCTGGCCTCAGCTCTAGAC, complementary oligo-5′-AATTGTCTAGAGCTGAGGCCAGATCTCCGAATTC TTAAT) between the *PadI* and EcoRI sites of the pKAO127′kan′ vector fragment. The 22 kb EcoRI–*BglII* fragment from cosmid pKOS079-93D containing the *megAl–II* genes was inserted into EcoRI and *BglII* sites of the resulting plasmid to generate pKOS024-84. A 12 kb *BglII*–*BbvCI* fragment containing the *megAlll* and part of the *megClI* gene was subcloned from pKOS079-93A and excised as a *BglII*–*Xbal* fragment and ligated into the corresponding sites of pKOS024-84 to yield the final expression plasmid pKOS108-06.

The megosamine integrating vector pKOS97-42 was constructed as follows: a subclone was generated containing the 4 kb Xhol–Scal fragment from pKOS79-138B together with the 1.7 kb Scal–*PstI* fragment from pKOS79-93D in Litmus 28 (Stratagene). The entire 5.7 kb fragment was then excised as a Spel–*PstI* fragment and combined with the 6.3 kb *PstI*–EcoRI fragment from KOS79-93D and EcoRI– *Xbal*-digested pSET152 (Bierman *et al.*, 1992) to construct plasmid pKOS97-42.

### Production and analysis of secondary metabolites

Fermentation for production of polyketide, LC/MS analysis and quantification of 6-dEB for *S. lividans* K4-114/pKOS108-6 and *S. lividans* K4-114/pKAO127′kan′ were essentially as previously described (Xue *et al.*, 1999). *S. erythraea* NRRL2338 and *S. erythraea*/pKOS97-42 were grown for 6 days in F1 medium (Brünker *et al.*, 1998). Samples of broth were clarified in a microcentrifuge (5 min, 13 000 r.p.m.). For LC/MS preparation, isopropanol was added to the supernatant (1:2 ratio) and centrifuged again. Samples were run on a C-18 reversed phase column (Inertsil ODS3, Metachem) using a 5-mM ammonium acetate (aqueous) acetonitrile–methanol (4:1) gradient (0–15%, 3 min; 15–60%, 10 min; 1 ml min⁻¹ flow). Erythromycins and megalomicins were detected by electrospray mass spectrometry and quantity was determined by evaporative light scattering detection (ELSD). A purified extract from *M. megalomicea* containing megalomicin A, B, C1 and C2 was used for the standard reference (provided by G. Ashley, Kosan Biosciences). The LC retention time and mass spectra of erythromycin and the four megalomicins were identical to those from the standards.

### Acknowledgements

We thank Alivelu Basava, Nick Corpuz and Mary Betlach for assistance with DNA sequencing, John Carney for assistance with LC/MS analysis and Gary Ashley for providing purified megalomicin samples.

### References

Alarcon, B., Lacal, J.C., Fernandez-Souza, J.M., and Carrasco, L. (1984) Screening for new compounds with antiherpes activity. *Antiviral Res* **4**: 231–243.

Alarcon, B., Gonzalez, E., and Carrasco, L. (1988) Megalomycin C, a macrolide antibiotic that blocks protein glycosylation and shows antiviral activity. *FEBS Lett* **231**: 207–211.

Altschul, S.F., Gish, W., Miller, W., Myers, E.W., and Lipman, D.J. (1990) Basic local alignment search tool. *J Mol Biol* **215**: 403–410.

Andersen, J.F., and Hutchinson, C.R. (1992) Characterization of *Saccharopolyspora erythraea* cytochrome P-450 genes and enzymes, including 6-deoxyerythronolide B hydroxylase. *J Bacteriol* **174**: 725–735.

Arisawa, A., Kawamura, N., Tsunekawa, H., Okamura, K., Tone, H., and Okamoto, R. (1993) Cloning and nucleotide sequences of two genes involved in the 4″-O-acylation of macrolide antibiotics from *Streptomyces thermotolerans*. *Biosci Biotechnol Biochem* **57**: 2020–2025.

Arisawa, A., Kawamura, N., Takeda, K., Tsunekawa, H., Okamura, K., and Okamoto, R. (1994) Cloning of the macrolide antibiotic biosynthesis gene *acyA*, which encodes 3-O-acyltransferase, from *Streptomyces thermotolerans* and its use for direct fermentative production of a hybrid macrolide antibiotic. *Appl Environ Microbiol* **60**: 2657–2660.

Bierman, M., Logan, R., O’Brien, K., Seno, E.T., Nagaraja, R., and Schoner, B.E. (1992) Plasmid cloning vectors for the conjugal transfer of DNA from *Escherichia coli* to *Streptomyces* spp. *Gene* **116**: 43–49.

Bisang, C., Long, P.F., Cortés, J., Westcott, J., Crosby, J., Matharu, A.-L., *et al.* (1999) A chain initiation factor common to both modular and aromatic polyketide synthases. *Nature* **401**: 502–505.

Bonay, P., Munro, S., Fresno, M., and Alarcon, B. (1996) Intra-Golgi transport inhibition by megalomicin. *J Biol Chem* **271**: 3719–3726.

Bonay, P., Fresno, M., and Alarcon, B. (1997) Megalomicin disrupts lysosomal functions. *J Cell Sci* **110**: 1839–1849.

Bonay, P., Duran-Chica, I., Fresno, M., Alarcon, B., and Alcina, A. (1998) Antiparasitic effects of the intra-Golgi transport inhibitor megalomicin. *Antimicrob Agents Chemother* **42**: 2668–2673.

Brünker, P., Minas, W., Kallio, P.T., and Bailey, J.E. (1998) Genetic engineering of an industrial strain of *Saccharopolyspora erythraea* for stable expression of the Vitreoscilla haemoglobin gene (*vhb*). *Microbiology* **144**: 2441–2448.

Butler, A.R., Bate, N., and Cundliffe, E. (1999) Impact of thioesterase activity on tylosin biosynthesis in *Streptomyces fradiae*. *Chem Biol* **6**: 287–292.

Cane, D.E., Walsh, C.T., and Khosla, C. (1998) Harnessing

© 2000 Blackwell Science Ltd, *Molecular Microbiology*, **37**, 752–762

the biosynthetic code: combinations, permutations, and mutations. Science 282: 63–68.

Cortés, J., Haydock, S.F., Roberts, G.A., Bevitt, D.J., and Leadlay, P.F. (1990) An unusually large multifunctional polypeptide in the erythromycin-producing polyketide synthase of Saccharopolyspora erythraea. Nature 348: 176–178.

Dhillon, N., Hale, R.S., Cortes, J., and Leadlay, P.F. (1989) Molecular characterization of a gene from Saccharopolyspora erythraea (Streptomyces erythraeus) which is involved in erythromycin biosynthesis. Mol Microbiol 3: 1405–1414.

Donadio, S., and Katz, L. (1992) Organization of the enzymatic domains in the multifunctional polyketide synthase involved in erythromycin formation in Saccharopolyspora erythraea. Gene 111: 51–60.

Donadio, S., and Staver, M.J. (1993) IS1136, an insertion element in the erythromycin gene cluster of Saccharopolyspora erythraea. Gene 126: 147–151.

Donadio, S., Staver, M.J., McAlpine, J.B., Swanson, S.J., and Katz, L. (1991) Modular organization of genes required for complex polyketide biosynthesis. Science 252: 675–679.

Epp, J.K., Huber, M., Turner, J.R., Goodson, T., and Schoner, B.E. (1989) Production of a hybrid macrolide antibiotic in Streptomyces ambofaciens and Streptomyces lividans by introduction of a cloned carbomycin biosynthetic gene from Streptomyces thermotolerans. Gene 85: 293–301.

Gaisser, S., Böhm, G.A., Cortés, J., and Leadlay, P.F. (1997) Analysis of seven genes from the eryAI–eryK region of the erythromycin biosynthetic gene cluster in Saccharopolyspora erythraea. Mol Gen Genet 256: 239–251.

Gokhale, R.S., Tsuji, S.Y., Cane, D.E., and Khosla, C. (1999) Dissecting and exploiting intermodular communication in polyketide synthases. Science 284: 482–485.

Gu, H., Wang, Y., Xu, X., Wei, X., and Fong, M. (1996) Increase of the expression of midecamycin 4′-hydroxyl propionyltransferase gene (*mpt*) by a promoter-like fragment from the midecamycin producing strain. Chin J Biotechnol 12: 147–152.

Hara, O., and Hutchinson, C.R. (1992) A macrolide 3-O-acyltransferase gene from the midecamycin-producing species Streptomyces mycarofaciens. J Bacteriol 174: 5141–5144.

Haydock, S.F., Dowson, J.A., Dhillon, N., Roberts, G.A., Cortes, J., and Leadlay, P.F. (1991) Cloning and sequence analysis of the genes involved in erythromycin biosynthesis in Saccharopolyspora erythraea; sequence similarities between eryG and a family of S-adenosylmethionine-dependent methyltransferases. Mol Gen Genet 230: 120–128.

Hopwood, D.A., Bibb, M.J., Chater, K.F., Kieser, T., Bruton, C.J., Kieser, H.M., *et al.* (1985) Genetic Manipulation of Streptomyces: A Laboratory Manual. Norwich, UK: The John Innes Foundation.

Kakavas, S.J., Katz, L., and Stassi, D. (1997) Identification and characterization of the niddamycin polyketide synthase genes from Streptomyces caelestis. J Bacteriol 179: 7515–7522.

Kao, C.M., Luo, G., Katz, L., Cane, D.E., and Khosla, C. (1994a) Engineered biosynthesis of a triketide lactone from

an incomplete modular polyketide synthase. J Am Chem Soc 116: 11612–11613.

Kao, C.M., Katz, L., and Khosla, C. (1994b) Engineered biosynthesis of a complete macrolactone in a heterologous host. Science 265: 509–512.

Katz, L. (1997) Manipulation of modular polyketide synthases. Chem Rev 97: 2557–2576.

Kuhstoss, S., Huber, M., Turner, J.R., Paschal, J.W., and Rao, R.N. (1996) Production of a novel polyketide through the construction of a hybrid polyketide synthase. Gene 183: 231–236.

McDaniel, R., Thamchaipenet, A., Gustafsson, C., Fu, H., Betlach, M., Betlach, M., *et al.* (1999) Multiple genetic modifications of the erythromycin gene cluster to produce a library of novel ‘unnatural’ natural products. Proc Natl Acad Sci USA 96: 1846–1851.

Nakagawa, A., and Omura, S. (1984) Structure and stereochemistry of macrolides. In Macrolide Antibiotics. Omura, S. (ed.). New York: Academic Press, pp. 37–84.

Olano, C., Lomovskaya, N., Fonstein, L., Roll, J.T., and Hutchinson, C.R. (1999) A two-plasmid system for the glycosylation of polyketide antibiotics: bioconversion of ε-rhodomycinone to rhodomycin D. Chem Biol 6: 845–855.

Pereda, A., Summers, R., and Katz, L. (1997) Nucleotide sequence of the *ermE* distal flank of the biosynthesis cluster in Saccharopolyspora erythraea. Gene 193: 65–71.

Sambrook, J., Fritsch, E.F., and Maniatis, T. (1989). Molecular Cloning: a Laboratory Manual. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.

Schwecke, T., Aparicio, J.F., Molnar, I., König, A., Khaw, L.E., Haydock, S.F., *et al.* (1995) The biosynthetic gene cluster for the polyketide immunosuppressant rapamycin. Proc Natl Acad Sci USA 92: 7839–7843.

Shah, S., Xue, Q., Tang, L., Carney, J.R., Betlach, M., and McDaniel, R. (2000) Cloning, characterization and heterologous expression of a polyketide synthase and P-450 oxidase involved in the biosynthesis of the antibiotic oleandomycin. J Antibiotics 53: 502–508.

Stassi, D., Donadio, S., Staver, M.J., and Katz, L. (1993) Identification of a Saccharopolyspora erythraea gene required for the final hydroxylation step in erythromycin biosynthesis. J Bacteriol 175: 182–189.

Summers, R.G., Donadio, S., Staver, M.J., Wendt-Pienkowski, E., Hutchinson, C.R., and Katz, L. (1997) Sequencing and mutagenesis of genes from the erythromycin biosynthetic gene cluster of Saccharopolyspora erythraea that are involved in L-mycarose and D-desosamine production. Microbiology 143: 3251–3262.

Tang, L., Fu, H., Betlach, M.C., and McDaniel, R. (1999) Elucidating the mechanism of chain termination switching in the picromycin/methymycin polyketide synthase. Chem Biol 6: 553–558.

Tang, L., Fu, H., and McDaniel, R. (2000) Formation of functional heterologous complexes using subunits from the picromycin, erythromycin, and oleandomycin polyketide synthases. Chem Biol 7: 77–84.

van Wageningen, A., Kirkpatrick, P., Williams, D., Harris, B., Kershaw, J., Lennard, N., *et al.* (1998) Sequencing and analysis of genes involved in the biosynthesis of a vancomycin group antibiotic. Chem Biol 3: 155–162.

Weber, J.M., Leung, J.O., Maine, G.T., Potenz, R.H., Paulus,

T.J., and DeWitt, J.P. (1990) Organization of a cluster of erythromycin genes in *Saccharopolyspora erythraea*. *J Bacteriol* **172**: 2372–2383.

Weber, J.M., Leung, J.O., Swanson, S.J., Idler, K.B., and McAlpine, J.B. (1991) An erythromycin derivative produced by targeted gene disruption in *Saccharopolyspora erythraea*. *Science* **252**: 114–117.

Weinstein, M.J., Wagman, G.H., Marquez, J.A., Testa, R.T., Oden, E., and Waitz, J.A. (1969) Megalomicin, a new macrolide antibiotic complex produced by *Micromonospora*. *J Antibiot* **22**: 253–258.

Xue, Q., Ashley, G., Hutchinson, C.R., and Santi, D.V. (1999) A multi-plasmid approach to preparing large libraries of polyketides. *Proc Natl Acad Sci USA* **96**: 11740–11745.

Xue, Y., Zhao, L., Liu, H.-W., and Sherman, D.H. (1998) A gene cluster for the macrolide antibiotic biosynthesis in *Streptomyces venezuelae*: architecture of metabolic diversity. *Proc Natl Acad Sci USA* **95**: 12111–12116.

Zhao, L., Sherman, D.H., and Liu, H.-W. (1998) Biosynthesis of desosamine: construction of a new methymycin/neomethymycin analogue by deletion of a desosamine biosynthetic gene. *J Am Chem Soc* **120**: 10256–10257.

Ziermann, R., and Betlach, M.C. (1999) Recombinant polyketide synthesis in *Streptomyces*: Engineering of improved host strains. *Biotechniques* **26**: 106–110.

Ziermann, R., and Betlach, M. (2000) A two-vector system for the production of recombinant polyketides in *Streptomyces*. *J Ind Microbiol Biotech* **24**: 46–50.
