
MEMBRANE TRANSPORT STRUCTURE
FUNCTION AND BIOGENESIS:
TatC Is a Specificity Determinant for
Protein Secretion via the Twin-arginine
Translocation Pathway

Jan D. H. Jongbloed, Ulrike Martin, Haike
Antelmann, Michael Hecker, Harold Tjalsma,
Gerard Venema, Sierd Bron, Jan Maarten van
Dijl and Jörg Müller
J. Biol. Chem. 2000, 275:41350-41357.
doi: 10.1074/jbc.M004887200 originally published online September 27, 2000

Access the most updated version of this article at doi: 10.1074/jbc.M004887200

Find articles, minireviews, Reflections and Classics on similar topics on the JBC Affinity Sites.

Alerts:
- When this article is cited
- When a correction for this article is posted

Click here to choose from all of JBC's e-mail alerts

This article cites 64 references, 33 of which can be accessed free at
http://www.jbc.org/content/275/52/41350.full.html#ref-list-1

THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2000 by The American Society for Biochemistry and Molecular Biology, Inc.

**TatC Is a Specificity Determinant for Protein Secretion via the Twin-arginine Translocation Pathway**

Received for publication, June 6, 2000, and in revised form, September 27, 2000  
Published, JBC Papers in Press, September 27, 2000, DOI 10.1074/jbc.M004887200

Jan D. H. Jongbloed,$^{a,b,c,d}$ Ulrike Martin,$^{b,e,f}$ Haike Antelmann,$^{g,h}$ Michael Hecker,$^{d,g,h}$ Harold Tjalsma,$^{a,i}$ Gerard Venema,$^{a}$ Sierd Bron,$^{a,d}$ Jan Maarten van Dijl,$^{d,j,k}$ and Jörg Müller$^{e,f}$

From the $^{a}$Department of Genetics, Groningen Biomolecular Sciences and Biotechnology Institute, Kerklaan 30, 9751 NN Haren, The Netherlands, the $^{c}$Institute of Molecular Biology, Jena University, Winzerlaer Strasse 10, D-07745 Jena, Germany, the $^{g}$Institut für Mikrobiologie und Molekularbiologie, Ernst-Moritz-Arndt-Universität Greifswald, F.-L.-Jahn-Strasse 15, D-17487 Greifswald, Germany, and the $^{j}$Department of Pharmaceutical Biology, University of Groningen, A. Deusinglaan 1, 9713 AV Groningen, The Netherlands

The recent discovery of a ubiquitous translocation pathway, specifically required for proteins with a twin-arginine motif in their signal peptide, has focused interest on its membrane-bound components, one of which is known as TatC. Unlike most organisms of which the genome has been sequenced completely, the Gram-positive eubacterium *Bacillus subtilis* contains two tatC-like genes denoted *tatCd* and *tatCy*. The corresponding TatCd and TatCy proteins have the potential to be involved in the translocation of 27 proteins with putative twin-arginine signal peptides of which ~6–14 are likely to be secreted into the growth medium. Using a proteomic approach, we show that PhoD of *B. subtilis*, a phosphodiesterase belonging to a novel protein family of which all known members are synthesized with typical twin-arginine signal peptides, is secreted via the twin-arginine translocation pathway. Strikingly, TatCd is of major importance for the secretion of PhoD, whereas TatCy is not required for this process. Thus, TatC appears to be a specificity determinant for protein secretion via the Tat pathway. Based on our observations, we hypothesize that the TatC-determined pathway specificity is based on specific interactions between TatC-like proteins and other pathway components, such as TatA, of which three paralogues are present in *B. subtilis*.

The Gram-positive eubacterium *Bacillus subtilis* is known to secrete a great variety of proteins into the growth medium (1). Together with components of the protein secretion pathways of *B. subtilis*, these native secreted proteins form the so-called secretome (2). The sequencing of the *B. subtilis* genome (3) has allowed a first analysis of the secretome based on the computer-assisted prediction of signal peptides. The results of this analysis indicated that ~300 of the 4107 identified genes specify putative exported proteins with an amino-terminal signal peptide. We previously predicted that 114 of these are lipoproteins, which are retained in the cytoplasmic membrane (4). A closer examination of the remaining putative secreted proteins suggests that these can be divided into various subgroups on the basis of particular motifs in their signal peptides.¹ Notably, the signal peptides belonging to one of these subgroups (see Table I) contain a so-called “twin-arginine” motif with the RRXφφ (where φ is a hydrophobic residue) consensus sequence (5–7). Signal peptides with this consensus motif (RR signal peptides) have been implicated in the transport of proteins via a novel Sec-independent pathway that seems to be conserved in eubacteria and organelles, such as chloroplasts and mitochondria (8, 9). The bacterial equivalent of this protein export system has been termed the Tat pathway (for twin-arginine translocation) (10, 11).

Even though the Tat pathway was first identified in thylakoids (called the ΔpH pathway in chloroplasts) (12–14), the equivalent pathway of *Escherichia coli* is presently best characterized. In this organism, four genes are known to encode proteins involved in the Tat export pathway. Three of these genes form an operon (*tatABC*), whereas the *tatE* gene is monocistronic (see Refs. 9–11). The TatABCE proteins are membrane-bound and believed to function in the Tat protein translocase in the plasma membrane. The protein encoded by the fourth gene in the *tatABC* operon, denoted *tatD*, was recently shown not to be required for Tat-dependent protein export (15). Inactivation of *tatB* (also described as *mttA*) (16–18) or *tatC* resulted in a total block in the export of proteins bearing RR signal peptides (11). In contrast, the products of the *tatA* and *tatE* genes, which are paralogues of the *tatB*-encoded protein (19), were shown to have overlapping functions in protein export via the Tat pathway (10, 18). The precise roles of TatABCE in the recognition of RR signal peptides and the protein translocation process are presently unknown. Notably, a functional Tat pathway is required for anaerobic growth of *E. coli*, which is due to the fact that various proteins required for anaerobic growth are exported via this pathway (see Refs. 5 and 19).

Genes specifying homologues of the *E. coli* TatABE proteins were identified in most bacteria, including *B. subtilis* (20) (see Fig. 1A). In contrast to *E. coli*, which contains only one *tatC* gene, two genes specifying TatC homologues were identified in

---

* The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
$b$ These authors contributed equally to this work.
$c$ Supported by Grant 805-33.605 from the Stichting Levenswetenschappen.
$d$ Supported by Quality of Life and Management of Living Resources Grants QLK3-CT-1999-00415 and QLK3-CT-1999-00917 from the European Union.
$f$ Supported by the Deutsche Forschungsgemeinschaft.
$h$ Supported by grants from the Deutsche Forschungsgemeinschaft; the Bundesministerium für Bildung, Wissenschaft, Forschung, und Technologie; and the Fonds der Chemischen Industrie.
$i$ Supported by Genencor International (Leiden, The Netherlands).
$k$ To whom correspondence should be addressed. Tel.: 31-50-3633079; Fax: 31-50-3636908; E-mail: j.m.van.dijl@farm.rug.nl.

---

¹ Jan D. H. Jongbloed, Ulrike Martin, Haike Antelmann, Michael Hecker, Harold Tjalsma, Gerard Venema, Sierd Bron, Jan Maarten van Dijl, and Jörg Müller, unpublished observations.

Tat Pathway Specificity

**B. subtilis** (see Fig. 1B). Interestingly, each of the two *tatC* genes of *B. subtilis* was preceded by a *tatABE*-like gene. These observations are consistent with the identification of genes for (putative) exported proteins with RR signal peptides in *B. subtilis*. Strikingly, however, the WapA and WprA proteins, which are synthesized with potential RR signal peptides, were recently shown to be secreted in an Ffh- and SecA-dependent manner (21). As the transport of proteins via the Tat pathway of *E. coli* was shown to be independent of SecA and largely independent of Ffh (22), this observation raised the question whether a functional Tat pathway exists in *B. subtilis*. On the contrary, the observation that *B. subtilis* contains two paralogous *tatC* genes, each with an upstream *tatA* gene, might even suggest that two parallel routes for twin-arginine translocation exist in this organism. This idea was, to some extent, also suggested by the observation that one set of *tatAC* genes of *B. subtilis* was preceded by the *phoD* gene, which specifies a secreted phosphodiesterase (*i.e.* PhoD) (23) with an RR signal peptide (see Table I) and which is expressed only under conditions of phosphate starvation (23). In the present study, we show that the latter *tatC* gene, denoted *tatCd*, is expressed only under conditions of phosphate starvation. Moreover, it seems to be specifically required for the secretion of PhoD, a process that was almost completely blocked when the *tatCd* gene was disrupted, but not when the *tatCy* gene was disrupted. These observations show that the TatCd protein of *B. subtilis* is a specificity determinant for Tat-dependent protein secretion.

---

### EXPERIMENTAL PROCEDURES

#### Plasmids, Bacterial Strains, and Media—Table II lists the plasmids and bacterial strains used. TY medium (Tryptone/yeast extract) contained Bacto-Tryptone (1%), Bacto-yeast extract (0.5%), and NaCl (1%). Minimal medium was prepared as described (24). Schaeffer's sporulation medium was prepared as described (25). High phosphate and low phosphate (LPDM)² defined media were prepared as described (26). To test anaerobic growth, S7 medium was prepared as described (27, 28) and supplemented with NaNO₃ (0.2%) and glycerol (2%). When required, media for *E. coli* were supplemented with ampicillin (100 μg/ml), erythromycin (100 μg/ml), kanamycin (40 μg/ml), or spectinomycin (100 μg/ml); media for *B. subtilis* were supplemented with erythromycin (1 μg/ml), kanamycin (10 μg/ml), spectinomycin (100 μg/ml), and/or isopropyl-β-D-thiogalactopyranoside (IPTG; 100 μM).

#### DNA Techniques—Procedures for DNA purification, restriction, ligation, agarose gel electrophoresis, and transformation of *E. coli* were carried out as described (29). Enzymes were from Roche Molecular Biochemicals. *B. subtilis* was transformed as described (30). PCR was carried out with the Pwo DNA polymerase (New England Biolabs Inc.) as described (31).

To construct *B. subtilis ΔtatCd*, the 5′-region of the *tatCd* gene was amplified by PCR with primers JJ14bT (5′-CCC AAG CTT ATG AAA GGG AGG GCT TTT TTG AAT GG-3′, containing a HindIII site) and JJ15bT (5′-GCG GAT CCA AAG CTG AGC ACG GG-3′, containing a BamHI site). The amplified fragment was cleaved with HindIII and BamHI and cloned in the corresponding sites of pMutin2 (32), resulting in pMICd1. *B. subtilis ΔtatCd* was obtained by a Campbell-type integration (single crossover) of pMICd1 into the *tatCd* region of the chromosome.

To construct *B. subtilis ΔtatCy*, the 5′-region of the *tatCy* gene was amplified by PCR with primers JJ03iJ (5′-CCC AAG CTT AAA AAG AAA GAA GAT CAG TAA GTT AGG ATG-3′, containing a HindIII site) and JJ04iJ (5′-GCG GAT CCA AGT CCT GAG AAA TCC G-3′, containing a BamHI site). The amplified fragment was cleaved with HindIII and BamHI and cloned in the corresponding sites of pMutin2, resulting in pMICy1. *B. subtilis ΔtatCy* was obtained by a Campbell-type integration (single crossover) of pMICy1 into the *tatCy* region of the chromosome.

To construct *B. subtilis ΔtatCd*, the *tatCd* gene was amplified by PCR with primers JJ33Cdd (5′-GGA ATT CGT GGG ACG GCT ACC-3′, containing an EcoRI site and 5′-sequences of *tatCd*) and JJ34Cdd

---

### TABLE I  
**Predicted twin-arginine signal peptides of *B. subtilis***

Putative twin-arginine signal peptides were identified in two ways. First, the presence of the consensus sequence RRXφφ (where φ is a hydrophobic residue), immediately in front of an amino-terminal hydrophobic region as predicted with the TopPred2 algorithm (35, 36), was determined. For this purpose, the first 60 residues of all annotated proteins of *B. subtilis* in the SubtiList Database were used. Second, within the group of twin-arginine membrane-sorting signals, cleavable signal peptides were identified with the SignalP algorithm (62, 63). Conserved residues of the twin-arginine consensus sequence (RRXφφ) are indicated in boldface. In addition, positively charged residues that could function as a so-called Sec avoidance signal (55) are indicated in boldface italics. The hydrophobic H-domains are shaded. In signal peptides with a predicted signal peptidase I cleavage site, residues from positions –3 to –1 relative to the signal peptidase I cleavage site are underlined. Notably, some of these proteins contain one or more putative transmembrane segments elsewhere in the protein (indicated by TM) or are putative lipoproteins. Residues forming a so-called lipobox for signal peptidase II cleavage are enlarged.

| protein | signal peptide |
|---------|----------------|
| AlbB    | MSPAQRRLILSPIFVIGAVVYFVKSDYLFTLIPDAIALIF |
| AmyXᵀᴹ  | MVSIRRSFEAYVDDMNIIITVLIPAEQXEIM            |
| AppBᵀᴹ  | MAAYIIRRTLMSIPILLGITILSFVIMKAAPG           |
| LipA    | MKFVKRRRIIALVTILMLSVTSLPALQPSAKAAEH        |
| OppBᵀᴹ  | MLKYIGRRIVYMITTLFVIVTVTFFLMQAAPG           |
| PbpX    | MTSPTRRRTAKRRRKLNRGKILFGLLAWVCITIWNALHR     |
| PhoD    | MAYDSRFDEWVQKLKEESFQNNTFDRRKFIQGACKIAGLSGLTIAQSFGAEV |
| QcrA    | MGGKHDISSRQFLNYTLIGVGGFMAASMLMPMVRA         |
| SpoIIIJᵀᴹ | MLLKKRIGLLLLSMGVFMLLAGCSSV                |
| TlpAᵀᴹ   | MKKTLLTTIRRSSIARRLIISFLILIVPITALSVSAYQS    |
| WapA    | MKKRKRNFKFIAAPFLVALMISLVPADVLAKST          |
| WprA    | MKRRKFSSVAAVLFALIFSFLFSGTKAAAAGA           |
| YceAᵀᴹ   | MEMFDLEFMRAFLAGGMIAVMAPILGVYLVLRQ          |
| YdeJ    | MKKRRKICYCNTALLLIMILLAGCTDS                 |
| YdhF    | MRRILSILVFAIMLAGCSSN                        |
| YdhK    | MSAGKSYRKKMKQRRNMKISKYALGILMLSLVFVL SACGNNN |
| YesMᵀᴹ   | MKKRVAGWYRMKIKDKLFPVFLSLIMAVSFLFVYSGVQYAPHV |
| YesW    | MRRSCLMIRRRKRMFTAVTLLLVLLVMGSTVCPVKAEGA     |
| YfkNᵀᴹ   | MRIQKRRTHVENILRILLPPIMILSLILPTPIHAEES      |
| YkpC    | MLRDLGRRVAIAAILSGLIILGGMSISLANMP            |
| YkuE    | MKKMSRRQFLKGMFALAAGALTAGGGYGARYL            |
| YmaC    | MRRFLLNVILVLAIVLFLEYVHYHSLEPE               |
| YmzC    | MFSESAELRRIRIALWWIAVFLLFGACGN               |
| YubFᵀᴹ   | MQKYRRNTVAFTVLAYFTFFAGVFLFSIGLYNADNL       |
| YuiC    | MMLNMIRRLMTCLFLAAGTTFLSVSGIEAKDL            |
| YvhJ    | MAERVRRVRKKKKSKRRKILKREMLLFALALLVVVGLGGYKLY |
| YwbN    | MSDEQKKPEQIHRRDILKWGAMAGAAVAIGASGLGLAPLVTAAKP |

(5′-CGG GAT CCA TCA TGG GAA GCG-3′, containing a *Bam*HI site and 3′-sequences of *tatCd*). Next, the PCR-amplified fragment was cleaved with *Eco*RI and *Bam*HI and ligated into the corresponding sites of pUC21, resulting in pJCd1. Plasmid pJCd2 was obtained by replacing an internal *Bcl*I-*Acc*I fragment of the *tatCd* gene in pJCd1 with a pDG792-derived kanamycin resistance marker, flanked by *Bam*HI and *Cla*I restriction sites. Finally, *B. subtilis ΔtatCd* was obtained by a double crossover recombination event between the disrupted *tatCd* gene of pJCd2 and the chromosomal *tatCd* gene.

To construct *B. subtilis ΔtatCy*, the *tatCy* gene was amplified by PCR with primers JJ29Cyd (5′-GGG GTA CCG GAA AAC GCT TGA TCA GG-3′, containing a *Kpn*I site and 5′-sequences of *tatCy*) and JJ30Cyd (5′-CGG GAT CCT TTG GGC GAT AGC C-3′, containing a *Bam*HI site and 3′-sequences of *tatCy*). Next, the PCR-amplified fragment was cleaved with *Kpn*I and *Bam*HI and ligated into the Asp718 and *Bam*HI sites of pUC21, resulting in pJCy1. Plasmid pJCy2 was obtained by ligating a pDG1726-derived spectinomycin resistance marker, flanked by *Pst*I restriction sites, into the unique *Pst*I site of the *tatCy* gene in pJCy1. Finally, *B. subtilis ΔtatCy* was obtained by a double crossover recombination event between the disrupted *tatCy* gene of pJCy2 and the chromosomal *tatCy* gene.

*tatCd-tatCy* double mutants were constructed by transforming the ΔtatCy mutant with chromosomal DNA of the ΔtatCd or ItatCd mutant strain. Correct integration of plasmids or resistance markers into the chromosome of *B. subtilis* was verified by Southern blotting. The BLAST algorithm (33) was used for protein comparisons in the GenBankᵀᴹ/EBI Data Bank. Protein sequence alignments were carried out with the ClustalW program (34) using the Blosum matrices or Version 6.7 of the PCGene Analysis Program (IntelliGenetics Inc.). Putative transmembrane segments and their membrane topologies were predicted with the TopPred2 algorithm (35, 36).

#### Competence and Sporulation—Competence for DNA binding and uptake was determined by transformation with plasmid or chromosomal DNA (37). The efficiency of sporulation was determined by overnight growth in Schaeffer’s sporulation medium, killing of cells with 0.1

---

² The abbreviations used are: LPDM, low phosphate defined medium; IPTG, isopropyl-β-D-thiogalactopyranoside; PCR, polymerase chain reaction.

volume of chloroform, and subsequent plating.

**Enzyme Activity Assays—** The assay and calculation of β-galactosidase units (expressed as units/A₆₀₀) were carried out as described (38). Overnight cultures were diluted 100-fold in fresh medium, and samples were taken at hourly intervals for A₆₀₀ readings and β-galactosidase activity determinations. Induction of the phosphate starvation response was monitored by alkaline phosphatase activity determinations as described (39).

**Western Blot Analysis and Immunodetection—** To detect PhoB and PhoD, *B. subtilis* cells were separated from the growth medium by centrifugation (14,000 rpm, 2 min, room temperature). Proteins in the growth medium were concentrated 20-fold upon precipitation with trichloroacetic acid, and samples for SDS-polyacrylamide gel electrophoresis were prepared as described previously (40). After separation by SDS-polyacrylamide gel electrophoresis, proteins were transferred to a nitrocellulose membrane (Schleicher & Schüll) as described (41). PhoB and PhoD were visualized with specific antibodies (42) and alkaline phosphatase-conjugated goat anti-rabbit antibodies (Sigma) according to the manufacturer's instructions.

**Two-dimensional Gel Electrophoresis of Secreted Proteins—** *B. subtilis* strains were grown at 37°C under vigorous agitation in 1 liter of a synthetic medium (43, 44) containing 0.16 mM KH₂PO₄ to induce a phosphate starvation response. After 1 h of post-exponential growth, cells were separated from the growth medium by centrifugation. The secreted proteins in the growth medium were precipitated overnight with ice-cold 10% trichloroacetic acid and collected by centrifugation (40,000 × g, 2 h, 4°C). The pellet was washed three times with 96% ethanol; dried; and resuspended in 400 μl of rehydration solution containing 2 M thiourea, 8 M urea, 1% Nonidet P-40, 20 mM dithiothreitol, and 0.5% Pharmalyte (pH 3–10). Cells were disrupted by sonication as described (45), and cellular proteins were resuspended in rehydration solution as described above. Samples of secreted or cellular proteins in rehydration solution were used for the re-swelling of immobilized pH gradient strips (pH 3–10; Amersham Pharmacia Biotech). Next, protein separation in the immobilized pH gradient strips (first dimension electrophoresis) was performed as recommended by the manufacturer. Electrophoresis in the second dimension was performed as described (46). The resulting two-dimensional gels were stained with silver nitrate (47) or Coomassie Brilliant Blue R-250.

**Protein Identification—** In-gel tryptic digestion of proteins, separated by two-dimensional gel electrophoresis, was performed using a peptide-collecting device (48). For this purpose, 0.5 μl of peptide solution was mixed with an equal volume of a saturated α-cyano-4-hydroxycinnamic acid solution in 50% acetonitrile and 0.1% trifluoroacetic acid. The resulting mixture was applied to the sample template of a matrix-assisted laser desorption/ionization mass spectrometer (Voyager DE-STR, PerSeptive Biosystems). Peptide mass fingerprints were analyzed using MS-Fit software.

---

### RESULTS

#### Identification of tat Genes of B. subtilis—
To investigate whether *B. subtilis* contains a potential Tat pathway, a search for homologues of *E. coli* Tat proteins was performed using the complete sequence of the *B. subtilis* genome (3). First, sequence comparisons revealed that *B. subtilis* contains three paralogous genes (*i.e.*, *yczB*, *ydiI*, and *ynzA*) that specify proteins with sequence similarity to the three paralogous *E. coli* TatA, TatB, and TatE proteins. Specifically, the YdiI protein (57 residues), which was renamed TatAy, showed the highest degree of sequence similarity to the *E. coli* TatA protein (58% identical residues and conservative replacements); the YczB protein (70 residues), which was renamed TatAd, showed the highest degree of sequence similarity to the *E. coli* TatB protein (54% identical residues and conservative replacements); and the YnzA protein (62 residues), which was renamed TatAc, showed the highest degree of sequence similarity to the *E. coli* TatB protein (53% identical residues and conservative replacements). All three *B. subtilis* proteins were renamed TatA to avoid possible misinterpretations with respect to their respective functions, which are presently unknown. Like TatA, TatB, and TatE of *E. coli*, the three TatA proteins of *B. subtilis* appear to have one amino-terminal membrane-spanning domain (Fig. 1A), and the carboxyl-terminal parts of these proteins are predicted to face the cytoplasm. Even though TatAc, TatAd, and

---

**FIG. 1.** Tat components of *B. subtilis* and *E. coli*. The amino acid sequences of Tat components of *B. subtilis* and *E. coli* as deduced from the SubtiList and Colibri Databases were used for comparisons. Identical amino acids (*) or conservative replacements (.) are marked. Putative transmembrane segments (*shaded*) were predicted with the TopPred2 algorithm (35, 36). A, comparison of TatAc (YnzA), TatAd (YczB), and TatAy (YdiI) of *B. subtilis* (*Bsu*) with TatA, TatB, and TatE of *E. coli* (*Eco*); B, comparison of TatCd (YcbT) and TatCy (YdiJ) of *B. subtilis* with TatC of *E. coli*.

TatAy of *B. subtilis* show significant similarity to TatA, TatB, and TatE of *E. coli* when the amino acid sequences of these proteins are compared pairwise, only a limited number of residues are conserved in all six amino acid sequences (17% identical residues and conservative replacements) (Fig. 1A).

Second, in contrast to *E. coli*, which contains a unique *tatC* gene (11), *B. subtilis* was shown to contain two paralogous *tatC*-like genes (*i.e.*, *ycbT* and *ydiJ*). The YcbT protein (245 residues), which was renamed TatCd, and the YdiJ protein (254 residues), which was renamed TatCy, showed significant similarity to the *E. coli* TatC protein (57% identical residues and conservative replacements in the three aligned sequences) (Fig. 1B). Like TatC of *E. coli*, TatCd and TatCy of *B. subtilis* have six potential transmembrane segments (Fig. 1B), and the amino termini of these proteins are predicted to face the cytoplasm (data not shown).

In contrast to *E. coli*, in which the *tatA*, *tatB*, and *tatC* genes form one operon and the *tatE* gene is monocistronic (10), the *tat* genes of *B. subtilis* are located at three distinct chromosomal regions. Two of these regions contain adjacent *tatA* and *tatC* genes, with the *tatAd* and *tatAy* genes being located immediately upstream of the *tatCd* and *tatCy* genes, respectively (Fig. 2). Strikingly, the *tatAd* and *tatCd* genes, which map at 24.4° on the *B. subtilis* chromosome, are located immediately downstream of the *phoD* gene, specifying a secreted protein with a putative RR signal peptide (Table I). Furthermore, the *tatAy* and *tatCy* genes are located at 55.3° on the *B. subtilis* chromosome, within a cluster of genes with unknown function (Fig. 2), and the *tatAc* gene is located at 162.7° on the *B. subtilis* chromosome.

Tat Pathway Specificity

A $B$. subtilis

![Diagram](#)

B $E$. coli

FIG. 2. tatAC regions of $B$. subtilis and $E$. coli. A, chromosomal organization of the $B$. subtilis tatAd-tatCd and tatAy-tatCy regions (adapted from the SubtiList Database). Note that the tatAd and tatCd genes are located downstream of the phoD gene. B, chromosomal organization of the $E$. coli tatABCD region (adapted from the Colibri Database).

mosome (data not shown), immediately downstream of the cotC gene, specifying a spore coat protein (49).

Taken together, these observations strongly suggest that $B$. subtilis has a Tat pathway for the translocation of proteins with RR signal peptides across the cytoplasmic membrane. Furthermore, the observation that the tatAd and tatCd genes are located downstream of the phoD gene, which is a member of the pho regulon (23), suggests that the tatAd and tatCd genes might be exclusively expressed under conditions of phosphate starvation.

TatC-dependent Secretion of the PhoD Protein—To investigate whether an active Tat pathway exists in $B$. subtilis, various single and double tatC mutants were constructed. For this purpose, the tatCd gene was either disrupted with a kanamycin resistance marker or placed under the control of the IPTG-dependent P$_{spac}$ promoter of plasmid pMutin2, resulting in the $B$. subtilis strains $\Delta$tatCd and ItatCd, respectively (Fig. 3, A and B). Similarly, the tatCy gene was either disrupted with a spectinomycin resistance marker or placed under the control of the IPTG-dependent P$_{spac}$ promoter of plasmid pMutin2, resulting in the $B$. subtilis strains $\Delta$tatCy and ItatCy, respectively (Fig. 3, A and C). tatCd-tatCy double mutants were constructed by transforming the $\Delta$tatCy mutant with chromosomal DNA of the $\Delta$tatCd or ItatCd mutant strain.

The fact that tatCd-tatCy double mutants could be obtained shows that TatC function is not essential for viability of $B$. subtilis, at least not when cells are grown aerobically in TY or minimal medium at 37 °C or anaerobically in S7 medium supplemented with NaNO₃ (0.2%) and glycerol (2%) at 37 °C (data not shown). Furthermore, the $\Delta$tatCd-$\Delta$tatCy double mutation did not inhibit the development of competence for DNA binding and uptake, sporulation, and the subsequent spore germination (data not shown), showing that these primitive developmental processes do not require TatC function.

The effects of single and double tatC mutations on protein secretion via the Tat pathway were studied using PhoD as a native reporter protein. For this purpose, tatC mutant strains were grown under conditions of phosphate starvation in LPDM. As shown by Western blotting, the secretion of PhoD was strongly reduced in the $\Delta$tatCd mutant strain and the $\Delta$tatCd-$\Delta$tatCy double mutant, whereas it was not affected or even improved in the $\Delta$tatCy mutant strain (Fig. 4A). In contrast, the secretion of the alkaline phosphatase PhoB, which is dependent of the major (Sec) pathway for protein secretion (50), was not affected in the tatC mutants of $B$. subtilis (Fig. 4B). Notably, in some experiments, very low amounts of PhoD were detectable in the growth medium of $B$. subtilis $\Delta$tatCd (data not shown), but never in that of the $\Delta$tatCd-$\Delta$tatCy and ItatCd-$\Delta$tatCy double mutants (Fig. 4, A and C). As exemplified with the $B$. subtilis ItatCd-$\Delta$tatCy double mutant strain, the cells of

A $B$. subtilis 168

![Diagram](#)

B $B$. subtilis ItatCd

![Diagram](#)

C $B$. subtilis ItatCy

![Diagram](#)

FIG. 3. Construction of tatC mutant strains of $B$. subtilis. A, schematic presentation of the construction of $B$. subtilis $\Delta$tatCd and $\Delta$tatCy. The chromosomal tatCd gene was disrupted with a kanamycin resistance marker ($Km^{r}$) by homologous recombination. For this purpose, $B$. subtilis 168 was transformed with plasmid pJCd2, which cannot replicate in $B$. subtilis and contains a mutant copy of the tatCd gene with an internal BclI-AccI fragment replaced by a kanamycin resistance marker. The chromosomal tatCy gene was disrupted with a spectinomycin resistance marker ($Sp^{r}$) by homologous recombination. For this purpose, $B$. subtilis 168 was transformed with plasmid pJCy2, which cannot replicate in $B$. subtilis and contains a mutant copy of the tatCy gene with a spectinomycin resistance marker in the PstI site. Only restriction sites relevant for the construction are shown: tatCd', 5'-end of the tatCd gene; 'tatCd, 3'-end of the tatCd gene; tatCy', 5'-end of the tatCy gene; 'tatCy, 3'-end of the tatCy gene. B, schematic presentation of the tatCd region of $B$. subtilis ItatCd. By a Campbell-type integration of the pMutin2 derivative pMICd1 into the $B$. subtilis 168 chromosome, the tatCd gene was placed under the control of the IPTG-dependent P$_{spac}$ promoter, which can be repressed by the product of the lacI gene. Simultaneously, the spoVG-lacZ reporter gene of pMutin2 was placed under the transcriptional control of the tatCd promoter region. PCR-amplified regions are indicated by black bars. ori pBR322, replication functions of pBR322; $Ap^{r}$, ampicillin resistance marker; $Em^{r}$, erythromycin resistance marker; tatCd', 3'-truncated tatCd gene; $T_{1}T_{2}$, transcriptional terminators on pMutin2. C, schematic presentation of the tatCy region of $B$. subtilis ItatCy. By a Campbell-type integration of the pMutin2 derivative pMICy1 into the $B$. subtilis 168 chromosome, the tatCy gene was placed under the control of the IPTG-dependent P$_{spac}$ promoter. Simultaneously, the spoVG-lacZ reporter gene of pMutin2 was placed under the transcriptional control of the tatCy promoter region. tatCy', 3'-truncated tatCy gene.

all tatC mutant strains contained similar amounts of pre-PhoD, which were comparable to those in the parental strain 168 (Fig. 4C) (data not shown). Finally, two-dimensional gel electrophoresis of proteins in the medium of phosphate-starved cells of $B$. subtilis $\Delta$tatCd-$\Delta$tatCy or the parental strain 168 showed that PhoD is the only protein of which the secretion is detectably affected by the tatC double mutation under conditions of phosphate starvation (Fig. 5). As expected, the secretion of proteins lacking an RR signal peptide (such as the glycerophosphoryl-diester phosphodiesterase GlpQ; the pectate lyase Pel; the alkaline phosphatases PhoA and PhoB; the phosphate-binding protein PstS; the minor extracellular serine protease Vpr; and the protein with unknown function, YncM) was not significantly affected by the tatC double mutation. Surprisingly, however, the secretion of YdhF, a phosphate starvation-inducible protein of unknown function (44), and the 2',3'-cyclic

168 strains (Fig. 5) (data not shown), despite the presence of an RR motif in the WprA signal peptide (Table I). Consistent with the above observations, no differences in the cellular proteomes of *B. subtilis* ΔtatCd-ΔtatCy and the parental strain 168 could be detected by two-dimensional gel electrophoresis (data not shown). In summary, these results show that an active Tat pathway exists in *B. subtilis* and that TatCd has a critical role in the secretion of PhoD.

**Expression of tatCd and tatCy Genes—** To study the expression of the tatCd and tatCy genes, the transcriptional tatCd-lacZ and tatCy-lacZ gene fusions, present in *B. subtilis* ItatCd and ItatCy, respectively, were used. As expected, upon a medium shift from high to low phosphate defined medium to induce a phosphate starvation response, tatCd transcription could be observed in *B. subtilis* ItatCd. In this strain, relatively low but constant levels of β-galactosidase production were reached within a period of 4 h after the change to LPDM, whereas no β-galactosidase production was detectable in the parental strain 168 (no lacZ gene fusion present) (Table II). In contrast, when cells of *B. subtilis* ItatCd were grown in minimal, Schaeffer’s sporulation, or TY medium, none of which induces a phosphate starvation response, no transcription of the tatCd gene was detectable; under these conditions, the β-galactosidase levels in cells of *B. subtilis* ItatCd were similar to those of the parental strain 168. Completely different results were obtained with *B. subtilis* ItatCy: the tatCy gene was transcribed in all growth media tested, and notably, the transcription of tatCy in LPDM was much higher than that of the tatCd gene (Table III). In contrast to the tatCd gene, the highest levels of tatCy transcription were observed in minimal and TY media, whereas the lowest levels of tatCy transcription were observed in Schaeffer’s sporulation medium (Table III). In conclusion, these findings show that tatCd is transcribed only under conditions of phosphate starvation, in contrast to tatCy, which is transcribed under all conditions tested.

**DISCUSSION**

In this study, we demonstrate for the first time that a functional Tat pathway, required for secretion of the PhoD protein, exists in the Gram-positive eubacterium *B. subtilis*. The TatCd protein, specified by one of the two tatC genes of *B. subtilis*, plays a critical role in this secretion pathway. In contrast, the TatCy protein appears to be of minor importance for PhoD secretion. Even though no particular function for TatCy was identified, our results show that the corresponding gene is transcribed under conditions of phosphate starvation when TatCd fulfills its critical role in PhoD secretion. Furthermore, as inferred from the fact that low levels of PhoD secretion by *B.* subtilis ΔtatCd (but never by tatCd-tatCy double mutants) were observed in some experiments, TatCy seems to be actively involved in RR preprotein translocation. Notably, these observations imply that TatC is a specificity determinant for protein secretion via the Tat pathway. In fact, our observation that the secretion of PhoD was increased in the absence of TatCy suggests that abortive interactions between pre-PhoD and TatCy or TatCy-containing translocases can occur. Nevertheless, alternative, more indirect explanations for this observation can presently not be excluded. Interestingly, the positive effect of the tatCy mutation on PhoD secretion is reminiscent of the effect that was observed when certain genes (*i.e.* sipS and/or sipU) for paralogous type I signal peptidases of *B.* subtilis were disrupted. This resulted in significantly improved rates of processing of the α-amylase AmyQ precursor by the remaining type I signal peptidases (*i.e.* SipT, SipV, and/or SipW) (24, 30, 51). Taken together, these observations suggest that, in general, the presence of two or more paralogous secretion machinery components in *B.* subtilis may result in as yet undefined abor-

**FIG. 4.** TatCd is required for secretion of PhoD. *B.* subtilis 168 (parental strain), ΔtatCd, ΔtatCy, or ΔtatCd-ΔtatCy cells were grown under conditions of phosphate starvation in LPDM. To study the secretion of PhoD (A) or PhoB (B), *B.* subtilis cells were separated from the growth medium by centrifugation. Secreted PhoD and PhoB in the growth medium were visualized by SDS-polyacrylamide gel electrophoresis and Western blotting using PhoD- and PhoB-specific antibodies, respectively. In C, cells of *B.* subtilis 168 and ItatCd-ΔtatCy were grown under conditions of phosphate starvation in LPDM. Next, cells and growth medium were separated by centrifugation, and PhoD was visualized by SDS-polyacrylamide gel electrophoresis and Western blotting using PhoD-specific antibodies.

**FIG. 5.** Two-dimensional gel electrophoretic analysis of the TatC-dependent secretion of PhoD. *B.* subtilis 168 (left panel) or ΔtatCd-ΔtatCy (right panel) cells were grown under conditions of phosphate starvation in LPDM. Secreted proteins were analyzed by two-dimensional gel electrophoresis as described under “Experimental Procedures.” The names of proteins identified by mass spectrometry are indicated.

nucleotide 2′-phosphodiesterase YfkN,³ which are both synthesized with potential RR signal peptides (Table I), was also not affected by the disruption of tatCd and tatCy (Fig. 5). Similarly, comparable WprA-derived protein spots could be demonstrated in the medium fractions of the *B.* subtilis ΔtatCd-ΔtatCy and

³ Please note that, accidentally, the YfkN spot was previously designated XkdE (43).

Tat Pathway Specificity

TABLE II  
Plasmids and strains  

| Plasmids | Relevant properties | Ref. |
|----------|---------------------|------|
| pUC21    | Cloning vector, 3.2 kb, $^{a}$ Ap$^{r}$ | 64   |
| pJCd1    | pUC21 derivative carrying the tatCd gene, 5.4 kb, Ap$^{r}$ | This work |
| pJCd2    | pUC21 derivative for the disruption of tatCd, 6.3 kb, Ap$^{r}$, Km$^{r}$ | This work |
| pJCy1    | pUC21 derivative carrying the tatCy gene, 5.3 kb, Ap$^{r}$ | This work |
| pJCy2    | pUC21 derivative for the disruption of tatCy, 6.5 kb, Ap$^{r}$, Sp$^{r}$ | This work |
| pMutin2  | pBR322-based integration vector for B. subtilis containing a multiple cloning site downstream of the P$_{spac}$ promoter and a promoter-less lacZ gene preceded by the RBS of the spoVG gene, 8.6 kb, Ap$^{r}$, Em$^{r}$ | 32   |
| pMICd1   | pMutin2 derivative carrying the 5'-part of the B. subtilis tatCd gene | This work |
| pMICy1   | pMutin2 derivative carrying the 5'-part of the B. subtilis tatCy gene | This work |
| pDG792   | Contains a Km resistance cassette, 4.0 kb, Ap$^{r}$, Km$^{r}$ | 65   |
| pDG1726  | Contains an Sp resistance cassette, 3.9 kb, Ap$^{r}$, Sp$^{r}$ | 65   |

| Strains       | Relevant properties                                                                                   | Ref. |
|---------------|-------------------------------------------------------------------------------------------------------|------|
| E. coli       | F$^{-}$, araD139, $\Delta$(ara-leu)7696, $\Delta$(lac)X74, galU, galK, hsdR2, mcrA, mcrB1, rspL            | 66   |
| MC1061        | trpC2                                                                                                 |      |
| B. subtilis 168 | trpC2                                                                                                 | 3    |
| $\Delta$tatCd | trpC2, tatCd, Km$^{r}$                                                                                | This work |
| $\Delta$tatCy | trpC2, tatCy, Sp$^{r}$                                                                                | This work |
| ItatCd        | trpC2, P$_{spac}$-tatCd, tatCd-lacZ, Em$^{r}$                                                         | This work |
| ItatCy        | trpC2, P$_{spac}$-tatCy, tatCy-lacZ, Em$^{r}$                                                         | This work |
| $\Delta$tatCd-$\Delta$tatCy | trpC2, tatCd, Km$^{r}$, tatCy, Sp$^{r}$                                                  | This work |
| ItatCd-$\Delta$tatCy | trpC2, P$_{spac}$-tatCd, tatCd-lacZ, Em$^{r}$, tatCy, Sp$^{r}$                                   | This work |

$^{a}$ kb, kilobases; Ap$^{r}$, ampicillin resistance marker; Km$^{r}$, kanamycin resistance marker; Sp$^{r}$, spectinomycin resistance marker; Em$^{r}$, erythromycin resistance marker; RBS, ribosomal binding site.

---

TABLE III  
$\beta$-Galactosidase activity  

To investigate the transcription of the tatCd and tatCy genes, cells of B. subtilis ItatCd (tatCd-lacZ), ItatCy (tatCy-lacZ), or the parental strain 168 (no lacZ gene fusion) were grown for 10 h in LPDM, minimal medium (MM), Schaeffer's sporulation medium (SSM), or TY medium after dilution from an overnight culture. Samples for $\beta$-galactosidase activity determinations were taken at hourly intervals, starting 4 h after dilution from the overnight culture. As the $\beta$-galactosidase activities showed little variation during the entire period of sampling, average values were determined. The numbers represent average values from three different experiments (units/A$_{600}$). Note that high phosphate defined medium was used for the overnight culture of cells grown in LPDM, whereas overnight cultures of cells grown in minimal, Schaeffer's sporulation, or TY medium were prepared with the respective media.

| Strain     | LPDM    | MM      | SSM     | TY       |
|------------|---------|---------|---------|----------|
| 168        | 0       | 0.1 ± 0.1 | 0.3 ± 0.2 | 0.6 ± 0.2 |
| ItatCd     | 1.1 ± 0.7 | 0.1 ± 0.1 | 0.3 ± 0.2 | 0.5 ± 0.2 |
| ItatCy     | 6.1 ± 2.5 | 10.0 ± 3.6 | 4.0 ± 2.0 | 13.2 ± 5.5 |

the H-region is an important determinant that allows the cell to discriminate between Sec-type and RR signal peptides. Notably, the predicted RR motifs of WapA, WprA, YdhF, and YfkN are also different from previously described RR signal peptides because they contain either a His residue at position +2 relative to the twin arginines or a Lys or Ser residue at position +3 (Table I). In fact, hydrophilic residues are completely absent from positions +2 and +3 relative to the twin arginines of known RR signal peptides (5, 6, 10, 14, 18, 22). If low overall hydrophobicity and the presence of hydrophobic residues at positions +2 and +3 are used as criteria for the prediction of RR signal peptides, the total number of predicted B. subtilis signal peptides of this type can be reduced from 27 to 11. Of these 11 preproteins, four contain additional transmembrane segments, and one lacks a signal peptidase cleavage site. Thus, based on these more stringent criteria, one would predict that merely six proteins of B. subtilis (i.e., AlbB, LipA, PhoD, YkuE, YuiC, and YwbN) are secreted into the growth medium via the Tat pathway. This would explain why the secretion of only one protein, PhoD, was detectably affected in B. subtilis $\Delta$tatCd-$\Delta$tatCy under conditions of phosphate starvation. In this respect, it is important to note that TatC-dependent secretion of some other proteins with (predicted) RR signal peptides may have remained unnoticed in the present study because they are expressed at very low levels under conditions of phosphate starvation. Furthermore, it is conceivable that other TatC-dependent proteins were missed in the two-dimensional gel electrophoretic analysis due to their poor separation in the first dimension.

Interestingly, the YdhF protein was also predicted to be a lipoprotein (Table I) (4). The fact that YdhF was found in the growth medium suggests either that this prediction was wrong or that YdhF is released into the growth medium via a secondary processing event that follows cleavage by the lipoprotein-specific (type II) signal peptidase (52). Such secondary processing events have been described previously for other Bacillus lipoproteins (see Ref. 4). In fact, the latter possibility most likely explains why the phosphate-binding protein PstS, which is a typical lipoprotein (previously known as YqgG) (4, 53), was found in the growth medium. As expected for lipoproteins, significant amounts of PstS were also present in a cell-associated form.

Tat Pathway Specificity

TABLE IV  
**Twin-arginine signal peptides of PhoD and PhoD-like proteins**

Homologues of *B. subtilis* (Bsu) PhoD were identified by amino acid sequence similarity searches in the GenBank™/EBI Data Bank using the BLAST algorithm. SP1 (Sco), gene SCC75A.32c of *Streptomyces coelicolor* (accession number CAB61732); SP2 (Sco), gene SCF43A.18 of *S. coelicolor* (accession number CAB48905); SP3 (Sco), gene SC4G6.37 of *S. coelicolor* (accession number CAB51460); SP4, *phoD* gene of *Streptomyces tendae* (accession number CAB62565). Conserved residues of the twin-arginine consensus sequence are indicated in boldface. The hydrophobic H-region is shaded. Signal peptidase I recognition sequences predicted with the SignalP algorithm (62, 63) are underlined.

| protein | signal peptide |
|---------|----------------|
| PhoD    | MAYDSRFD EWVQKLKEESFQNNTFDRRKFIQGAGKIAGLSLGLTIAQS V GAFEV |
| (Bsu)   |                |
| SP1     | MTPANHQAPTSAPSPAPSQSSHAPELRAAARSLGRRRFLTVTGAAAA LAFAVNLP AAGTASAAEL |
| (Sco)   |                |
| SP2     | MAPTGRPSALAEHAFSPHDAVLGAAARHLGRRRFLTVTAAAAALAFST NLPARGAVA APE |
| (Sco)   |                |
| SP3     | MTSRHRASENSRTPSRTTVVKAAAAGAVLAAPLAAALPAGAADAAAPA |
| (Sco)   |                |
| SP4     | MTPAARPSQHAPELRAAARHLGRRRFLTVTGAAAA LAFAVNLP AAGTAAAAEL |
| (Ste)   |                |

of Berks et al. (19), in which, on the basis of theoretical considerations, it was proposed that the TatABE proteins form a protein-conducting channel, whereas the TatC protein acts as an RR signal peptide receptor. Alternatively, it is still conceivable that certain proteins with RR signal peptides are recognized by TatA-like proteins, provided that a specific TatC-like partner protein is present. A third possibility would be that specific TatA- and TatC-like partner proteins are jointly involved in substrate recognition. The facts that neither TatAc nor TatAd of *B. subtilis* was able to complement *tatA*, *tatB*, or *tatE* mutations in *E. coli* and that TatCd of *B. subtilis* was unable to complement the *E. coli tatC* mutation¹ suggest that the TatC-determined pathway specificity, as described in the present study, is based on specific interactions between TatA- and TatC-like proteins. If so, this implies that *B. subtilis* contains two parallel routes for twin-arginine translocation, one of which involves the TatCd protein. As shown in the present study, the TatCd-dependent translocation appears to be activated specifically under conditions of phosphate starvation, perhaps with the sole purpose of translocating PhoD. Similar to the situation in *B. subtilis*, parallel routes for twin-arginine translocation may be present in other organisms, such as *Archaeoglobus fulgidus*, which was shown to contain two paralogous *tatC*-like genes (19, 61). In our ongoing research on protein secretion in *B. subtilis*, we are trying to challenge this hypothesis.

Acknowledgments—We thank Dr. A. J. M. Driessen for providing sera against PhoB and Drs. A. Bolhuis, C. Robinson, and M. L. van Roosmalen and other members of the ExporteRRs Consortium for useful discussions.

REFERENCES

1. Simonen, M., and Palva, I. (1993) *Microbiol. Rev.* **57**, 109–137
2. Tjalsma, H., Bolhuis, A., Jongbloed, J. D. H., Bron, S., and van Dijl, J. M. (2000) *Microbiol. Mol. Biol. Rev.* **64**, 515–547
3. Kunst, F., Ogasawara, N., Moszer, I., Albertini, A. M., Alloni, G., Azevedo, V., Bertero, M. G., Bessieres, P., Bolotin, A., Borchert, S., *et al.* (1997) *Nature* **390**, 249–256
4. Tjalsma, H., Kontinen, V. P., Prágai, Z., Wu, H., Meima, R., Venema, G., Bron, S., Sarvas, M., and van Dijl, J. M. (1999) *J. Biol. Chem.* **274**, 1698–1707
5. Berks, B. C. (1996) *Mol. Microbiol.* **22**, 393–404
6. Brink, S., Bogsch, E. G., Edwards, W. R., Hynds, P. J., and Robinson, C. (1998) *FEBS Lett.* **434**, 425–430
7. Cristóbal, S., de Gier, J. W., Nielsen, H., and von Heijne, G. (1999) *EMBO J.* **18**, 2982–2990
8. Settles, A. M., and Martiensen, R. (1998) *Trends Cell Biol.* **8**, 494–501
9. Dalbey, R. E., and Robinson, C. (1999) *Trends Biochem. Sci.* **24**, 17–22
10. Sargent, F., Bogsch, E. G., Stanley, N. R., Wexler, M., Robinson, C., Berks, B. C., and Palmer, T. (1998) *EMBO J.* **17**, 3640–3650
11. Bogsch, E. G., Sargent, F., Stanley, N. R., Berks, B. C., Robinson, C., and Palmer, T. (1998) *J. Biol. Chem.* **273**, 18003–18006
12. Mould, R. M., and Robinson, C. (1991) *J. Biol. Chem.* **266**, 12189–12193
13. Robinson, C., Cai, D., Hulford, A., Brock, I. W., Michl, D., Hazell, L., Schmidt I., Herrmann R. G., and Klösgen, R. B. (1994) *EMBO J.* **13**, 279–285
14. Chaddock, A. M., Mant, A., Karnauchov, I., Brink, S., Herrmann, R. G., Klösgen, R. B., and Robinson, C. (1995) *EMBO J.* **14**, 2715–2722
15. Wexler, M., Sargent, F., Bogsch, E. G., Stanley, N. R., Robinson, C., Berks, B. C., and Palmer, T. (2000) *J. Biol. Chem.* **275**, 16717–16722
16. Weiner, J. H., Bilous, P. T., Shaw, G. M., Lubitz, S. P., Frost, L., Thomas, G. H., Cole, J. A., and Turner, R. J. (1998) *Cell* **93**, 93–101
17. Chanal, A., Santini, C., and Wu, L. (1998) *Mol. Microbiol.* **30**, 674–676
18. Sargent, F., Stanley, N. R., Berks, B. C., and Palmer, T. (1999) *J. Biol. Chem.* **274**, 36073–36082
19. Berks, B. C., Sargent, F., and Palmer, T. (2000) *Mol. Microbiol.* **5**, 260–274
20. Settles, A. M., Yonetani, A., Baron, A., Bush, D. R., Cline, K., and Martiensen, R. (1997) *Science* **278**, 1467–1470
21. Hirose, I., Sano, K., Shioda, I., Kumano, M., Nakamura, K., and Yamane, K. (2000) *Microbiology (Read.)* **146**, 65–75
22. Santini, C. L., Ize, B., Chanal, A., Müller, M., Giordano, G., and Wu, L. F. (1998) *EMBO J.* **17**, 101–112
23. Eder, S., Shi, L., Jensen, K., Yamane, K., and Hulett, F. M. (1996) *Microbiology (Read.)* **142**, 2041–2047
24. Tjalsma, H., Bolhuis, A., van Roosmalen, M. L., Wieger, T., Schumann, W., Broekhuizen, C. P., Quax, W. J., Venema, G., Bron, S., and van Dijl, J. M. (1998) *Genes Dev.* **12**, 2318–2331
25. Schaeffer, P. Millet, J., and Aubert, J. (1965) *Proc. Natl. Acad. Sci. U. S. A.* **271**, 5463–5467
26. Müller, J. P., An, Z., Merad, T., Hancock, I. C., and Harwood, C. R. (1997) *Microbiology (Read.)* **143**, 947–956
27. van Dijl, J. M., de Jong, A., Smith, H., Bron, S., and Venema, G. (1991) *J. Gen. Microbiol.* **137**, 2073–2083

28. van Dijl, J. M., de Jong, A., Smith, H., Bron, S., and Venema, G. (1991) *Mol. Gen. Genet.* 227, 40–48
29. Sambrook, J., Fritsch, E. F., and Maniatis, T. (1989) *Molecular Cloning: A Laboratory Manual*, 2nd Ed., Cold Spring Harbor Laboratory, Cold Spring Harbor, NY
30. Tjalsma, H., Noback, M. A., Bron, S., Venema, G., Yamane, K., and van Dijl, J. M. (1997) *J. Biol. Chem.* 272, 25983–25992
31. van Dijl, J. M., de Jong, A., Venema, G., and Bron, S. (1995) *J. Biol. Chem.* 270, 3611–3618
32. Vagner, V., Dervyn, E., and Ehrlich, S. D. (1998) *Microbiology (Read.)* 144, 3097–3104
33. Altschul, S. F., Madden, T. L., Schaffer, A. A., Zhang, J., Zhang, Z., Miller, W., and Lipman, D. J. (1997) *Nucleic Acids Res.* 25, 3389–3402
34. Thompson, J. D., Higgins, D. G., and Gibson, T. J. (1994) *Nucleic Acids Res.* 22, 4673–4680
35. Sipos, L., and von Heijne, G. (1993) *Eur. J. Biochem.* 213, 1333–1340
36. Cserzo, M., Wallin, E., Simon, I., von Heijne, G., and Elofsson, A. (1997) *Protein Eng.* 10, 673–676
37. Bron, S., and Venema, G. (1972) *Mutat. Res.* 15, 1–10
38. Miller, J. H. (1982) *Experiments in Molecular Biology*, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY
39. Hulett, F. M., Bookstein, C., and Jensen, K. (1990) *J. Bacteriol.* 172, 735–740
40. Laemmli, U. K. (1970) *Nature* 227, 680–685
41. Towbin, H., Staehelin, T., and Gordon, J. (1979) *Proc. Natl. Acad. Sci. U. S. A.* 76, 4350–4354
42. Müller, J. P., and Wagner, M. (1999) *FEMS Microbiol. Lett.* 180, 287–296
43. Antelmann, H., Engelmann, S., Schmid, R., Sorokin, A., Lapidus, A., and Hecker, M. (1997) *J. Bacteriol.* 179, 7251–7256
44. Antelmann, H., Scharf, C., and Hecker, M. (2000) *J. Bacteriol.* 182, 4478–4490
45. Eymann, C., Mach, H., Harwood, C. R., and Hecker, M. (1996) *Microbiology (Read.)* 142, 3163–3170
46. Bernhardt, J., Völker, U., Völker, A., Antelmann, H., Schmid, R., Mach, H., and Hecker, M. (1997) *Microbiology (Read.)* 143, 999–1017
47. Blum, H., Beier, H., and Gross, H. J. (1987) *Electrophoresis* 8, 93–99
48. Otto, A., Thiede, B., Müller, E. C., Scheler, C., Wittmann-Liebold, B., and Jungblut, P. (1996) *Electrophoresis* 17, 1643–1650
49. Donovan, W., Zheng, L. B., Sandman, K., and Losick, R. (1987) *J. Mol. Biol.* 196, 1–10
50. Swaving, J., van Wely, K. H., and Driessen, A. J. (1999) *J. Bacteriol.* 181, 7021–7027
51. Bolhuis, A., Sorokin, A., Azevedo, V., Ehrlich, S. D., Braun, P. G., de Jong, A., Venema, G., Bron, S., and van Dijl, J. M. (1996) *Mol. Microbiol.* 22, 605–618
52. Prágai, Z., Tjalsma, H., Bolhuis, A., van Dijl, J. M., Venema, G., and Bron, S. (1997) *Microbiology (Read.)* 143, 1327–1333
53. Qi, Y., and Hulett, F. M. (1998) *J. Bacteriol.* 180, 4007–4010
54. Rodrigue, A., Chanal, A., Beck, K., Müller, M., and Wu, L. F. (1999) *J. Biol. Chem.* 274, 13223–13228
55. Bogsch, E., Brink, S., and Robinson, C. (1997) *EMBO J.* 16, 3851–3859
56. Hynds, P. J., Robinson, D., and Robinson, C. (1998) *J. Biol. Chem.* 273, 34868–34874
57. Yu, J., Hederstedt, L., and Piggot, P. J. (1995) *J. Bacteriol.* 177, 6751–6760
58. Stephenson, K., and Harwood, C. R. (1998) *Appl. Environ. Microbiol.* 64, 2875–2881
59. Bolhuis, A., Tjalsma, H., Stephenson, K., Harwood, C. R., Venema, G., Bron, S., and van Dijl, J. M. (1999) *J. Biol. Chem.* 274, 15865–15868
60. Bolhuis, A., Tjalsma, H., Smith, H. E., de Jong, A., Meima, R., Venema, G., Bron, S., and van Dijl, J. M. (1999) *Appl. Environ. Microbiol.* 65, 2934–2941
61. Klenc, H. P., Clayton, R. A., Tomb, J. F., White, O., Nelson, K. E., Ketchum, K. A., Dodson, R. J., Gwinn, M., Hickey, E. K., Peterson, J. D., Richardson, D. L., et al. (1997) *Nature* 390, 364–370
62. Nielsen, H., Engelbrecht, J., Brunak, S., and von Heijne, G. (1997) *Protein Eng.* 10, 1–6
63. Nielsen, H., Brunak, S., and von Heijne, G. (1999) *Protein Eng.* 12, 3–9
64. Vieira, J., and Messing, J. (1990) *Gene (Amst.)* 100, 189–194
65. Guerout-Fleury, A. M., Shazand, K., Frandsen, N., and Stragier, P. (1995) *Gene (Amst.)* 167, 335–336
66. Wertman, K. F., Wyman, A. R., and Botstein, D. (1986) *Cell* 49, 253–262
