
For the determination of pKa values, AhpC C46A and C165A (20 μM) were treated with iodoacetamide (200 μM) in 100 mM potassium phosphate, 1 mM EDTA at pH values from 4 to 10 for 45 min at room temperature. DTNB (5,5′-dithiobis(2-nitrobenzoic acid); 200 μM) was added, and sulphydryl content determined by absorbance at a wavelength of 412 nm.

Stopped-flow experiments were performed on an OLIS RSM-16 instrument. One syringe contained 5 μM peroxynitrite in 3 mM NaOH, and the other contained 100 mM potassium phosphate pH 7.0, 100 μM DTPA (KPi buffer). Equal volumes were injected into a stopped-flow cell, and the pH of the reaction (6.75) was measured at the outlet. 400 scans were collected for each run, and \( k_{\text{obs}} \) was obtained by fitting experimental data to a single exponential decay function.

TBH reductase activity was measured in the presence of NADH (200 μM), AhpF (500 nM) and AhpC (1 μM), and expressed relative to untreated AhpC. Sulphydryl content was measured with DTNB. Tyrosine nitration was measured by absorbance at 430 nm.

Received 6 June; accepted 10 July 2000.

1. Pfeiffer, S. *et al.* Metabolic fate of peroxynitrite in aqueous solution. Reaction with nitric oxide and pH-dependent decomposition to nitrite and oxygen in a 2:1 stoichiometry. *J. Biol. Chem.* **272**, 3465–3470 (1997).
2. Coddington, J. W., Hurst, J. K. & Lymar, S. V. Hydroxyl radical formation during peroxynitrous acid decomposition. *J. Am. Chem. Soc.* **121**, 2438–2443 (1999).
3. Merényi, G., Lind, J., Goldstein, S. & Czapski, G. Peroxynitrous acid homolyses into 'OH and 'NO₂ radicals. *Chem. Res. Toxicol.* **11**, 712–713 (1998).
4. Niles, J. C., Burney, S., Singh, S. P., Wishnok, J. S. & Tannenbaum, S. R. Peroxynitrite reaction products of 3′,5′-di-O-acetyl-8-oxo-7,8-dihydro-2′-deoxyguanosine. *Proc. Natl Acad. Sci. USA* **96**, 11729–11734 (1999).
5. Wink, D. A. *et al.* Nitric oxide protects against cellular damage and cytotoxicity from reactive oxygen species. *Proc. Natl Acad. Sci. USA* **90**, 9813–9817 (1993).
6. Jin, D.-Y. & Jeang, K.-T. in *Antioxidation and Redox Regulation of Genes* (eds Sen, C. K., Sies, H. & Baeuerle, P. A.) 381–407 (Academic, New York, 2000).
7. Storz, G. *et al.* An alkyl hydroperoxide reductase induced by oxidative stress in *Salmonella typhimurium* and *Escherichia coli*: genetic characterization and cloning of *ahp*. *J. Bacteriol.* **171**, 2049–2055 (1989).
8. Chen, L., Xie, Q. & Nathan, C. Alkyl hydroperoxide reductase subunit C (AhpC) protects bacterial and human cells against reactive nitrogen intermediates. *Mol. Cell* **1**, 795–805 (1998).
9. Nathan, C. & Shiloh, M. U. Reactive oxygen and nitrogen intermediates in the relationship between mammalian hosts and microbial pathogens. *Proc. Natl Acad. Sci. USA* **97**, 8844–8848 (2000).
10. Wilson, T., de Lisle, G. W., Marcinkeviciene, J. A., Blanchard, J. S. & Collins, D. M. Antisense RNA to *ahpC*, an oxidative stress defence gene involved in isoniazid resistance, indicates that AhpC of *Mycobacterium bovis* has virulence properties. *Microbiology* **144**, 2687–2695 (1998).
11. Szabo, C., Zingarelli, B., O'Connor, M. & Salzman, A. L. DNA strand breakage, activation of poly (ADP-ribose) synthetase, and cellular energy depletion are involved in the cytotoxicity of macrophages and smooth muscle cells exposed to peroxynitrite. *Proc. Natl Acad. Sci. USA* **93**, 1753–1758 (1996).
12. Ellis, H. R. & Poole, L. B. Roles for the two cysteine residues of AhpC in catalysis of peroxide reduction by alkyl hydroperoxide reductase from *Salmonella typhimurium*. *Biochemistry* **36**, 13349–13356 (1997).
13. Ellis, H. R. & Poole, L. B. Novel application of 7-chloro-4-nitrobenzo-2-oxa-1,3-diazole to identify cysteine sulfenic acid in the AhpC component of alkyl hydroperoxide reductase. *Biochemistry* **36**, 15013–15018 (1997).
14. Claiborne, A. *et al.* Protein-sulfenic acids: diverse roles for an unlikely player in enzyme catalysis and redox regulation. *Biochemistry* **38**, 15407–15416 (1999).
15. Briviba, K., Kissner, R., Koppenol, W. H. & Sies, H. Kinetic study of the reaction of glutathione peroxidase with peroxynitrite. *Chem. Res. Toxicol.* **11**, 1398–1401 (1998).
16. Pearce, L. L., Pitt, B. R. & Peterson, J. The peroxynitrite reductase activity of cytochrome c oxidase involves a two-electron redox reaction at the heme a (3)-Cu(B) site. *J. Biol. Chem.* **274**, 35763–35767 (1999).
17. Sherman, D. R. *et al.* Compensatory *ahpC* gene expression in isoniazid-resistant *Mycobacterium tuberculosis*. *Science* **272**, 1641–1643 (1996).
18. Yu, K. *et al.* Toxicity of nitrogen oxides and related oxidants on mycobacteria: *M. tuberculosis* is resistant to peroxynitrite anion. *Tuber. Lung Dis.* **79**, 191–198 (1999).
19. Choi, H.-J., Kang, S. W., Yang, C.-H., Rhee, S. G. & Ryu, S.-E. Crystal structure of a novel human peroxidase enzyme at 2.0 Å resolution. *Nature Struct. Biol.* **5**, 400–406 (1998).
20. Becker, K., Savvides, S. N., Keese, M., Schirmer, R. H. & Karplus, P. A. Enzyme inactivation through sulphydryl oxidation by physiologic NO-carriers. *Nature Struct. Biol.* **5**, 267–271 (1998).
21. Radi, R., Beckman, J. S., Bush, K. M. & Freeman, B. A. Peroxynitrite oxidation of sulphydryls. The cytotoxic potential of superoxide and nitric oxide. *J. Biol. Chem.* **266**, 4244–4250 (1991).
22. Beckman, J. S. & Koppenol, W. H. Nitric oxide, superoxide, and peroxynitrite: the good, the bad, and ugly. *Am. J. Physiol.* **271**, C1424–C1437 (1996).
23. Go, Y. M. *et al.* Evidence for peroxynitrite as a signaling molecule in flow-dependent activation of c-Jun NH(2)-terminal kinase. *Am. J. Physiol.* **277**, H1647–H1653 (1999).
24. Kamisaki, Y. *et al.* An activity in rat tissues that modifies nitrotyrosine-containing proteins. *Proc. Natl Acad. Sci. USA* **95**, 11584–11589 (1998).
25. Denu, J. M. & Tanner, K. G. Specific and reversible inactivation of protein tyrosine phosphatases by hydrogen peroxide: evidence for a sulfenic acid intermediate and implications for redox regulation. *Biochemistry* **37**, 5633–5642 (1998).
26. Koppenol, W. H., Kissner, R. & Beckman, J. S. Synthesis of peroxynitrite: to go with the flow or on solid grounds? *Methods Enzymol.* **269**, 296–302 (1996).
27. Uppu, R. M. & Pryor, W. A. Biphasic synthesis of high concentrations of peroxynitrite using water-insoluble alkyl nitrite and hydrogen peroxide. *Methods Enzymol.* **269**, 322–329 (1996).
28. Poole, L. B. & Ellis, H. R. Flavin-dependent alkyl hydroperoxide reductase from *Salmonella typhimurium*. 1. Purification and enzymatic activities of overexpressed AhpF and AhpC proteins. *Biochemistry* **35**, 56–64 (1996).
29. O'Toole, P. W., Logan, S. M., Kostrzynska, M., Wadstrom, T. & Trust, T. J. Isolation and biochemical and molecular analyses of a species-specific protein antigen from the gastric pathogen *Helicobacter pylori*. *J. Bacteriol.* **173**, 505–513 (1991).

Acknowledgements

We thank L. Chen for generating *ahpC* mutations, G. St John for the *H. pylori* clone, H. Erdjument-Bromage and P. Tempst for protein sequencing and T. Sakmar for access to his stopped-flow spectrophotometer. This work was supported by a Norman and Rosita Winston fellowship (R.B.) and by an NIH grant (C.N.).

Correspondence and requests for materials should be addressed to C.N. (e-mail: cnathan@med.cornell.edu).

---

**Peptide cyclization catalysed by the thioesterase domain of tyrosidine synthetase**

John W. Trauger*, Rahul M. Kohli*, Henning D. Mootz†, Mohamed A. Marahiel† & Christopher T. Walsh*

* Department of Biological Chemistry and Molecular Pharmacology, Harvard Medical School, 240 Longwood Avenue, Boston, Massachusetts 02115, USA  
† Biochemie/Fachbereich Chemie, Philipps-Universität Marburg, Hans-Meerwein-Strasse, 35032 Marburg, Germany  

In the biosynthesis of many macrocyclic natural products by multidomain megasynthases, a carboxy-terminal thioesterase (TE) domain is involved in cyclization and product release¹,²; however, it has not been determined whether TE domains can catalyse macrocyclization (and elongation in the case of symmetric cyclic peptides) independently of upstream domains. The inability to decouple the TE cyclization step from earlier chain assembly steps has precluded determination of TE substrate specificity, which is important for the engineered biosynthesis of new compounds¹. Here we report that the excised TE domain from tyrosidine synthetase efficiently catalyses cyclization of a decapeptide-thioester to form the antibiotic tyrosidine A, and can catalyse pentapeptide-thioester dimerization followed by cyclization to form the antibiotic gramicidin S. By systematically varying the decapeptide-thioester substrate and comparing cyclization rates, we also show that only two residues (one near each end of the decapeptide) are critical for cyclization. This specificity profile indicates that the tyrosidine synthetase TE, and by analogy many other TE domains, will be able to cyclize and release a broad range of new substrates and products produced by engineered enzymatic assembly lines.

An enormous range of medicinally important polyketide and peptide natural products assembled by modular polyketide synthases (PKSs), non-ribosomal peptide synthetases (NRPSs) and mixed PKS/NRPS systems have macrocyclic structures, including the antibiotics erythromycin (PKS) and daptomycin (NRPS), the immunosuppressants cyclosporin (NRPS) and rapamycin (PKS/NRPS), and the antitumour agent epothilone (PKS/NRPS). PKSs and NRPSs are large, multifunctional proteins that are organized into sets of functional domains termed modules¹,². The order of modules corresponds directly to the sequence of monomers in the product. Synthetic intermediates are covalently tethered by thioester linkages to a carrier protein domain in each module. The thiol tether on each carrier domain is phosphopantetheine, which is attached to a conserved serine residue in the carrier protein in a post-translational priming reaction catalysed by a phosphopantetheinyltransferase³. Chain initiation involves loading a specific monomer onto the thiol tether of each carrier protein. Subsequent chain elongation steps involve transfer of the growing chain from an upstream carrier protein to the adjacent downstream carrier-protein-bound monomer. Release of the full-length chain

letters to nature

from the most C-terminal carrier domain is almost always catalysed by a C-terminal TE domain of relative molecular mass 28,000–35,000 (Mr 28K–35K)¹. Thioesterase domains of PKS and NRPS use a catalytic triad consisting of a serine residue (or cysteine in some cases), histidine and an acidic residue⁴⁵. Thioester cleavage involves sequential acylation and deacylation of the active-site serine⁵⁻⁷. In the acylation step, the full-length chain is transferred from the thiol tether of the upstream carrier protein to the serine residue. Deacylation of the resulting acyl-O-TE intermediate occurs either by intramolecular cyclization to form macrolactones or macrolactams or by hydrolysis.

It was not clear at the outset whether an isolated TE domain would retain cyclization activity; when the deoxyerythronolide B synthase TE domain was overproduced and purified it had hydrolysis activity, but did not cyclize synthetic acyl-thioester substrates⁷⁸. The absence of cyclization in these cases might reflect that either the simplified substrate analogues lacked functional groups required for substrate recognition or the TE cannot independently catalyse cyclization. With regard to TE specificity, engineered PKS and NRPS systems in which domains have been added, substituted, or deleted can produce new polyketides⁹⁻¹⁴ and peptides¹⁵⁻¹⁷. For example, engineered variants of deoxyerythronolide B synthase, which normally produces a 14-membered macrolactone, produce of a variety of cyclic products ranging in size from 6- to 16-membered rings⁹⁻¹³; however, the product yield of an engineered synthase is often much lower than that of its natural counterpart¹². This limitation may reflect the specificity of individual domains within a synthase. For example, analysis of a peptide-bond-forming C domain from an NRPS revealed that this domain has low selectivity for the upstream residue, but high selectivity for the downstream residue¹⁸. As a synthase engineered to assemble a new polyketide or peptide chain will be useful only if the TE can efficiently release the product by cyclization or hydrolysis, characterizing TE specificity is a critical step toward robust engineered synthases.

We wanted to determine whether the TE domain from the tyrocidine NRPS (Fig. 1a), which catalyses assembly of the cyclic decapeptide antibiotic tyrocidine A¹⁹,²⁰, can independently catalyse peptide cyclization. We replaced full-length TycC (Mr 724K) with overexpressed and purified TycC TE domain (Mr 28K) and replaced

![Figure 2](#fig2)

**Figure 2** Cyclization activity and substrate specificity of the TycC TE domain. **a**, HPLC analysis of reactions that initially contained 2 μM TLP, 50 nM TycC TE and 25 mM MOPS (pH 7.0, 24 °C). **b**, Kinetic parameters for peptide-SNAC cyclization catalysed by the TE domain of TycC.

![Figure 1](#fig1)

**Figure 1** Role of the thioesterase (TE) domain in biosynthesis of the cyclic decapeptide antibiotic tyrocidine A. **a**, The tyrocidine non-ribosomal peptide synthetase from *B. brevis*. Synthetase subunits TycA, TycB and TycC are represented by a series of boxes. Each box represents a functional domain: A, adenylation (catalyses amino-acid activation); PCP, peptidyl carrier protein; C, condensation (catalyses peptide-bond formation); E, epimerization. The TE domain is shaded. Thiol (SH) and hydroxyl (OH) groups represent phosphopantetheine and the TE active-site serine residue, respectively. **b**, Left, proposed mechanism of TE-domain catalysed macrocyclization and product release. Decapeptide intermediates are shown with the N-terminal residue D-Phe¹ highlighted in red. Right, Experimental system for probing the mechanism and specificity of TE-domain catalysed cyclization.
(D-Phe-Pro-Phe-D-Phe-Asn-Gln-Tyr-Val-Orn-Leu-SNAC (TLP); where Orn is ornithine) was incubated with purified TycC TE, efficient cyclization to tyrocidine A occurred as well as a minor flux of hydrolysis to the decapeptide (ratio of cyclization:hydrolysis, 6:1) (Fig. 2a). The cyclic product was identified as tyrocidine A by its co-elution with authentic tyrocidine A during high performance liquid chromatography (HPLC) and by mass spectrometry. Kinetic analysis of the cyclization reaction established a \( k_{\text{cat}} \) of 59 turnovers per minute and a \( K_{M} \) of 3 μM. No hydrolysis or cyclization was detectable under the reaction conditions in the absence of enzyme. Thus, the isolated TE domain retains a robust capacity to recognize the tyrocidine decapeptide chain and to catalyse efficiently cyclization to a 30-membered macrocycle. To our knowledge, the small flux to hydrolysis is not observed with the natural synthetase, and may reflect the absence of upstream domains. We measured background hydrolysis and cyclization rates for TLP, which revealed that the TE domain of TycC accelerates hydrolysis and cyclization by factors of \( 1 \times 10^6 \) and \( 2 \times 10^7 \), respectively.

To explore the substrate specificity of TycC TE, we synthesized a series of mutant peptide-SNAC substrates that differ from the wild-type tyrocidine A sequence by a single amino-acid substitution. Specifically, we changed the amino-terminal residue D-Phe1 to either L-Phe or D-Ala, D-Phe4 to D-Ala, Orn9 to Glu (Orn was not changed to Ala because the resulting cyclic molecule would be uncharged and have poor water solubility), and each of the other seven residues to Ala. We determined kinetic parameters for the cyclization of each of the mutant substrates (Fig. 2b). Notably, mutation of the N-terminal residue D-Phe1 to either L-Phe (DF1F) or D-Ala (DF1DA) abolishes cyclization activity, indicating that recognition of both the stereochemistry and side chain of this residue is essential for cyclization. Hydrolysis of DF1F and DF1DA is observed, with kinetic parameters similar to the wild-type substrate TLP, indicating that these mutations affect the cyclization step and not peptide-O-TE formation. Recognition of Orn9 is also critical for cyclization: when changed to Glu (O9E), cyclization still occurs, but with \( k_{\text{cat}} \) decreased 100-fold and \( K_M \) increased 20-fold. Changing Orn to Glu affects cyclization and hydrolysis equally, indicating that the mutation affects the peptide-O-TE formation step. Mutants in which each of the remaining eight residues are changed to alanine (without changing L- or D- configuration) have relatively little effect on cyclization kinetics: \( k_{\text{cat}} \) values for all of these substrates are within a factor of 2 of the wild-type substrate TLP except for L10A (fourfold reduction in \( k_{\text{cat}} \)), and all of the \( K_M \) values are within a factor of 2 of TLP except Y7A (fivefold increase in \( K_M \)) and V8A (threefold increase in \( K_M \)). These results suggest that Tyr7, Val8 and Leu10 contribute to substrate recognition, although their contribution is much less than that of D-Phe1 and Orn9.

The observed specificity of the TE domain of TycC suggests that

various substrates that retain the key ‘recognition residues’ will be cyclized. To test this prediction, we synthesized 9-residue (D-Phe-Pro-Phe-Asn-Gln-Tyr-Val-Orn-Leu-SNAC; TLP9) and 11-residue (D-Phe-Pro-Phe-D-Phe-Asn-Ala-Gln-Tyr-Val-Orn-Leu-SNAC; TLP11) substrates in which one residue near the centre of the TLP sequence is either deleted or inserted. Both 9- and 11-membered substrates are cyclized by the TE domain of TycC (Fig. 2b). The 15-fold reduction in \( k_{\text{cat}} \) for TLP9 may result from strain in the cyclic conformation. These results show that TycC TE can catalyse formation of cyclic peptides with various ring sizes.

We next investigated whether the TycC TE could catalyse assembly of the cyclic decapeptide antibiotic gramicidin S. The amino-acid sequence of gramicidin S is a pentapeptide repeat, and the gramicidin S NRPS contains only five amino-acid activation modules (Fig. 3a), suggesting that the gramicidin TE domain (GrsB TE) catalyses dimerization of two pentapeptides and cyclization of the resulting decapeptide²¹,²². A proposed mechanism for this process is shown in Fig. 3b. Comparison of the sequence of the tyrocidine decapeptide precursor with that of the gramicidin S pentapeptide precursor (D-Phe-Pro-Val-Orn-Leu) reveals that both peptides have the same two N-terminal residues (D-Phe-Pro) and the same three C-terminal residues (Val-Orn-Leu).

![Figure 4](#fig4)

**Figure 4** The tyrocidine synthetase thioesterase domain (TycC TE) catalyses dimerization of the pentapeptide-SNAC GLP5 and cyclization of the resulting decapeptide-SNAC to form gramicidin S. **a**, HPLC analysis of a reaction that initially contained 200 μM GLP5, 100 nM TycC TE and 25 mM MOPS (pH 7.0, 24 °C, 1 min reaction time). **b**, Kinetic parameters for GLP5 dimerization and hydrolysis and GLP10 cyclization catalysed by the TE domain of TycC.

![Figure 3](#fig3)

**Figure 3** Role of the TE domain in biosynthesis of the cyclic decapeptide antibiotic gramicidin S. **a**, Gramicidin S synthetase from *B. brevis*. **b**, Proposed mechanism of pentapeptide dimerization and decapeptide cyclization catalysed by the TE domain of GrsB: (1) a pentapeptide is built up by the synthetase and transferred to the TE active-site serine; (2) a second pentapeptide is built up; (3) the N-terminal amine of the pentapeptide-S-PCP reacts with the peptide-O-TE to form a decapeptide-S-PCP intermediate; and (4) the PCP-tethered decapeptide is transferred to the TE serine and cyclized.
letters to nature

As our results with TycC TE indicated that these common N- and C-terminal sequences are sufficient for substrate recognition, we reasoned that the TycC TE should be able to dimerize the gramicidin S pentapeptide-SNAC (D-Phe-Pro-Val-Orn-Leu-SNAC; GLP5) to form a decapeptide-SNAC that would then cyclize to gramicidin S. On incubation of GLP5 with TycC TE, we observed efficient chain dimerization and subsequent cyclization, as well as hydrolysis to the pentapeptide (Fig. 4). The identities of the products were confirmed by HPLC co-elution with authentic standards and by mass spectrometry. This result shows that the isolated TycC TE domain can catalyse peptide elongation and reveals the mechanism for pentapeptide dimerization and cyclization by gramicidin S synthetase, as well as the likely mechanism for assembly of other symmetric cyclic peptides such as the antitumour antibiotic thiocoraline²³. This finding complements our previous demonstration that the TE-containing protein EntF from enterobactin synthetase can catalyse two elongation cycles followed by cyclization to form enterobactin, a 12-membered trilactone⁶. Finally, the fact that the gramicidin S precursor decapeptide-SNAC (D-Phe-Pro-Val-Orn-Leu-D-Phe-Pro-Val-Orn-Leu-SNAC; GLP10), which differs from the tyrocidine A precursor TLP by the substitution of five central residues, cyclizes at a rate similar to TLP (fivefold lower $k_{cat}$ and comparable $K_M$ for GLP10 compared with TLP) further shows that the TycC TE can cyclize various substrates provided that the necessary ‘recognition residues’ near each end of the substrate are present. We note that ligation²⁴ and cyclization of synthetic acyl-thioesters may be a generally useful application of excised TE domains from PKS and NRPS systems.

### Methods

#### Peptide-SNAC synthesis
Peptides were prepared by automated solid-phase synthesis (0.3 mmol scale, diisopropylcarbodiimide/hydroxybenzotriazole (DICPDI/HOBt) activation) on 2-chlorotrityl resin derivatized with the appropriate C-terminal amino acid using Fmoc-protected monomers (side-chain protecting groups used were: trityl for Asn and Gln, $t$-butyl for Tyr, and Boc for Orn) except for the N-terminal monomer, which was Boc-protected. The peptide was cleaved from the resin using 1:1:3 acetic acid:trifluoroethanol:dichloromethane (DCM) (3 h, 24 °C), then precipitated with $n$-hexane and the solvent removed by rotary evaporation. The protected peptide (1 eq.) was dissolved in tetrahydrofuran (THF) or dimethylformamide (DMF). A solution of DCC (1.2 eq.) and HOBt (1.2 eq.) in THF (or DMF) and $N$-acetylcysteamine (2.5 eq.) were added, and the reaction stirred for 35 min at 24°C. Potassium carbonate (0.6 eq.) was then added, and the reaction stirred for 3 h at 24°C, filtered and concentrated. The protected peptide-SNAC was treated with 16:3:1 trifluoroacetic acid (TFA):DCM:$N$-acetylcysteamine (3 h, 24°C) and precipitated with ether. Reverse-phase ($C_{18}$) HPLC purification (20 to 50% acetonitrile in 0.1% TFA/water over 30 min) afforded the peptide-SNAC TFA salt (10–25% yield from the protected peptide) in more than 95% purity (by analytical HPLC) as a white solid. The identities of the all peptide-SNACs were verified by MALDI-TOF mass spectrometry.

#### Protein expression
TycC TE DNA was amplified from *Bacillus brevis* (ATCC 8185) chromosomal DNA using the primers 5′-ATAAGATCTCAT AAGCGCTTTGAGAGCAG-3′ and 5′-ATAGGATCCTTTCAGGATGAACAGTTCTTG-3′ and Vent DNA Polymerase (New England Biolabs). The fragment was digested with BamHI and BglII and cloned into BglII-digested pQE60 (Qiagen), which adds the sequence GSRSHHHHHH to the C terminus of the expressed protein. Expression of the resulting plasmid pQE60-TE(TycC) in *Escherichia coli* M15[pREP4] (Qiagen) followed by Ni-NTA affinity chromatography yielded TycC TE (25 mg l⁻¹; > 95% pure by SDS–PAGE).

#### Assays
We carried out reactions in 25 mM MOPS, pH 7.0, in a total volume of 400 μl. Reactions were initiated by addition of TycC TE and quenched at various time points by the addition of 25 μl 1.7% TFA/water, flash frozen in liquid nitrogen and stored at –80°C (for O9E, reactions were quenched by the addition of sodium phosphate, pH 5.3 to 100 mM). We then thawed the reactions, added 85 μl acetonitrile, and analysed them by analytical HPLC with monitoring at 220 nm (20–80% acetonitrile in 0.1% TFA/water (or in 25 mM potassium phosphate, pH 5.3, for O9E) over 35 min, Vydac protein and peptide $C_{18}$ column). Initial rates were calculated using 1-min time points. Peptide-SNAC and reaction product concentrations were determined for all Tyr-containing peptides based on the estimated extinction coefficient ε (280 nm) = 1,280 M⁻¹ cm⁻¹, which agrees with the experimentally determined ε (280 nm) of TLP. For peptide-SNACs not containing Tyr, ε (220 nm) was determined experimentally, and concentrations of corresponding cyclic products determined by assuming equal ε (220 nm) values for the peptide-SNAC and cyclic product.

---

### Product characterization
All cyclic products were characterized by MALDI-TOF mass spectrometry. Cyclic products enzymatically synthesized from TLP, P2A, L10A, TLP9, TLP11 and GLP10 were further characterized by ESI-ion trap mass spectrometry. Enzymatically synthesized (from TLP) and authentic tyrocidine A gave identical fragment ions, including four internal fragment ions (observed both with and without loss of NH₃ from Asn or Gln) that contain the Leu10-D-Phe1 dipeptide segment formed by head-to-tail cyclization, and at least two of the corresponding fragment ions were identified for cyclic peptides from P2A, L10A, TLP9 and TLP11, confirming that these products result from head-to-tail cyclization. For example, an Orn9 to Tyr7 ion was observed for each cycle (TLP cycle, M+H for OLFPFFNQY: calculated 1157.6, observed 1157.6; P2A cycle, M+H for OLFPFFNQY: calculated 1131.6, observed 1131.5; L10A cycle, M+H for OAFPFFNQY: calculated 1115.5, observed 1115.5; TLP9 cycle, M+H for OLFPFFNQY: calculated 1009.5, observed 1009.3; TLP11 cycle, M+H for OLFPFFNAQY calculated 1228.6, observed 1228.6; the underline highlights the LF bond formed by head to tail condensation). Similarly, the GLP10 cycle and authentic gramicidin S gave the same fragmentation pattern, and one ion confirming head-to-tail cyclization was detected in both samples (GLP10 cycle, M+H for LFPVOLFPV: calculated 914.6, observed 914.5).

Received 23 May; accepted 3 July 2000.

1. Cane, D. E., Walsh, C. T. & Khosla, C. Harnessing the biosynthetic code: combinations, permutations and mutations. *Science* **282**, 63–68 (1998).
2. Marahiel, M. A., Stachelhaus, T. & Mootz, H. D. Modular peptide synthetases involved in nonribosomal peptide synthesis. *Chem. Rev.* **97**, 2651–2674 (1997).
3. Lambalot, R. H. *et al.* A new enzyme superfamily—the phosphopantetheinyl-transferases. *Chem. Biol.* **3**, 923–936 (1996).
4. Lawson, D. M. *et al.* Structure of a myristoyl-ACP-specific thioesterase from *Vibrio harveyi*. *Biochemistry* **33**, 9382–9388 (1994).
5. Li, J., Szittner, R., Derewenda, Z. S. & Meighen, E. A. Conversion of serine-114 to cysteine-114 and the role of the active site nucleophile in acyl transfer by myristoyl-ACP thioesterase from *Vibrio harveyi*. *Biochemistry* **35**, 9967–9973 (1996).
6. Shaw-Reid, C. A. *et al.* Assembly line enzymology by multimodular nonribosomal peptide synthetases: the thioesterase domain of *E. coli* EntF catalyses both elongation and cyclolation. *Chem. Biol.* **6**, 385–400 (1999).
7. Aggarwal, R., Caffrey, P., Leadlay, P. F., Smith, C. J. & Staunton, J. The thioesterase domain of the erythromycin-producing polyketide synthase: mechanistic studies *in vitro* to investigate its mode of action and substrate specificity. *J. Chem. Soc. Chem. Comm.* **15**, 1519–1520 (1995).
8. Gokhale, R. S., Hunziker, D., Cane, D. E. & Khosla, C. Mechanism and specificity of the terminal thioesterase domain from the erythromycin polyketide synthase. *Chem. Biol.* **6**, 117–125 (1998).
9. Cortes, J. *et al.* Repositioning of a domain in a modular polyketide synthase to promote specific chain cleavage. *Science* **268**, 1487–1489 (1995).
10. Kao, C. M. *et al.* Gain of function mutagenesis of the erythromycin polyketide synthase. 2. Engineered biosynthesis of an eight-membered ring tetraketide lactone. *J. Am. Chem. Soc.* **119**, 11339–11340 (1997).
11. Jacobsen, J. R., Hutchinson, C. R., Cane, D. E. & Khosla, C. Precursor-directed biosynthesis of erythromycin analogs by an engineered polyketide synthase. *Science* **277**, 367–369 (1997).
12. McDaniel, R. *et al.* Multiple genetic modifications of the erythromycin polyketide synthase to produce a library of novel “unnatural” natural products. *Proc. Natl Acad. Sci. USA* **96**, 1846–1851 (1999).
13. Tang, L., Fu, H. & McDaniel, R. Formation of functional heterologous complexes using subunits from the picromycin, erythromycin and oleandomycin polyketide synthases. *Chem. Biol.* **7**, 77–84 (2000).
14. Xue, Y. & Sherman, D. H. Alternative modular polyketide synthase expression controls macrolactone structure. *Nature* **403**, 571–575 (2000).
15. Stachelhaus, T., Schneider, A. & Marahiel, M. A. Rational design of peptide antibiotics by targeted replacement of bacterial and fungal domains. *Science* **269**, 69–72 (1995).
16. de Ferra, F. Rodriguez, F., Tortora, O., Tosi, C. & Grandi, G. Engineering of peptide synthetases. *J. Biol. Chem.* **272**, 25304–25309 (1997).
17. Mootz, H. D., Schwarzer, D. & Marahiel, M. A. Construction of hybrid peptide synthetases by module and domain fusions. *Proc. Natl Acad. Sci. USA* **97**, 5848–5853 (2000).
18. Belshaw, P. J., Walsh, C. T. & Stachelhaus, T. Aminoacyl-CoAs as probes of condensation domain selectivity in nonribosomal peptide synthesis. *Science* **284**, 486–489 (1999).
19. Mootz, H. D. & Marahiel, M. A. The tyrocidine biosynthesis operon of *Bacillus brevis*: complete nucleotide sequence and biochemical characterization of functional internal adenylation domains. *J. Bacteriol.* **179**, 6843–6850 (1997).
20. Roskoski, R., Kleinkauf, H., Gevers, W. & Lipmann, F. Isolation of enzyme-bound peptide intermediates in tyrocidine biosynthesis. *Biochemistry* **9**, 4846–4851 (1970).
21. Hori, K. *et al.* Molecular cloning and nucleotide sequence of the gramicidin S synthetase 1 gene. *J. Biochem. (Tokyo)* **106**, 639–645 (1989).
22. Turgay, K., Krause, M. & Marahiel, M. A. Four homologous domains in the primary structure of GrsB are related to domains in a superfamily of adenylate-forming enzymes. *Mol. Microbiol.* **6**, 529–546 (1992).
23. Boger, D. L. & Ichikawa, S. Total syntheses of thiocoraline and BE-22179: establishment of relative and absolute stereochemistry. *J. Am. Chem. Soc.* **122**, 2956–2957 (2000).
24. Jackson, D. Y. *et al.* A designed peptide ligase for total synthesis of ribonuclease A with unnatural catalytic residues. *Science* **266**, 243–247 (1994).

### Acknowledgements
This work was supported by grants from the NIH (to C.T.W.), the Deutsche Forschungsgemeinschaft and the Fonds der Chemischen Industrie (to M.A.M.), a NIH postdoctoral fellowship (to J.W.T.) and a PhD fellowship of the Stiftung Stipendien-Fonds des Verbandes der Chemischen Industrie (to H.D.M.).

Correspondence and requests for materials should be addressed to C.T.W. (e-mail: walsh@walsh.med.harvard.edu).
