
THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2001 by The American Society for Biochemistry and Molecular Biology, Inc.

Flavonoid 6-Hydroxylase from Soybean (Glycine max L.),  
a Novel Plant P-450 Monooxygenase*

Received for publication, July 14, 2000, and in revised form, September 26, 2000  
Published, JBC Papers in Press, Moo6277200, DOI 10.1074/jbc.M006277200

Akinwunmi Olumide Latunde-Dada‡§, Francisco Cabello-Hurtado§¶, Nikola Czittrich‡,  
Luc Didierjean¶, Christel Schopfer‡, Norbert Hertkorn∥, Danièle Werck-Reichhart¶,  
and Jürgen Ebel‡**

From the ‡Botanisches Institut der Universität, Menzinger Strasse 67, D-80638 München, Germany,  
the §Department of Stress Response, Institute of Plant Molecular Biology, Centre National de la Recherche  
Scientifique FRE 2161, 28 rue Goethe, F-67083 Strasbourg, France, and the ∥Institut für Ökologische Chemie,  
GSF-Forschungszentrum für Umwelt und Gesundheit, Ingolstädter Landstrasse 1, D-85764 Neuherberg, Germany

Cytochrome P-450-dependent hydroxylases are typical enzymes for the modification of basic flavonoid skeletons. We show in this study that CYP71D9 cDNA, previously isolated from elicitor-induced soybean (Glycine max L.) cells, codes for a protein with a novel hydroxylase activity. When heterologously expressed in yeast, this protein bound various flavonoids with high affinity (1.6 to 52 μM) and showed typical type I absorption spectra. These flavonoids were hydroxylated at position 6 of both resorcinol- and phloroglucinol-based A-rings. Flavonoid 6-hydroxylase (CYP71D9) catalyzed the conversion of flavanones more efficiently than flavones. Isoflavones were hardly hydroxylated. As soybean produces isoflavonoid constituents possessing 6,7-dihydroxy substitution patterns on ring A, the biosynthetic relationship of flavonoid 6-hydroxylase to isoflavonoid biosynthesis was investigated. Recombinant 2-hydroxyisoflavanone synthase (CYP93C1v2) efficiently used 6,7,4′-trihydroxyflavanone as substrate. For its structural identification, the chemically labile reaction product was converted to 6,7,4′-trihydroxyisoflavone by acid treatment. The structures of the final reaction products for both enzymes were confirmed by NMR and mass spectrometry. Our results strongly support the conclusion that, in soybean, the 6-hydroxylation of the A-ring occurs before the 1,2-aryl migration of the flavonoid B-ring during isoflavanone formation. This is the first identification of a flavonoid 6-hydroxylase cDNA from any plant species.

---

Flavonoids are a diverse group of natural products that serve important roles in plants during growth, during development, and in defense against microorganisms and pests (1, 2). These compounds are synthesized from phenylpropanoid- and acetate-derived precursors through central pathways furnishing basic C<sub>6</sub>-C<sub>3</sub>-C<sub>6</sub> flavonoid skeletons and, in addition, through a variety of reactions leading to a range of modified aglycones and subsequently to their glycosylated derivatives within each flavonoid class. Many of the enzymes of flavonoid biosynthesis have been extensively studied (3), and recent molecular biological approaches have complemented biochemical methods in elucidating the mechanism and regulation of flavonoid biosynthesis (4). Typical enzymes belonging to the complex branch pathways for the elaboration of flavonoid skeletons are cytochrome P-450-dependent hydroxylases (3), such as flavonoid 3′-hydroxylase (5), flavonoid 3′,5′-hydroxylase (6), isoflavone 2′-hydroxylase (7), flavanone 2-hydroxylase (8), flavone synthase II (9, 10), and 2-hydroxyisoflavanone synthase (2HIS)<sup>1</sup> (11–13). Whereas flavonoid 3′-hydroxylase and flavonoid 3′,5′-hydroxylase are responsible for the formation of the 3′,5′-hydroxylation pattern of the flavonoid B-ring, hydroxylation of the isoflavone B-ring at the 2′ position (isoflavone 2′-hydroxylase) is one of the key reactions leading to pterocarpan structures. The formation of flavones and isoflavones from flavanones is catalyzed by several evolutionarily related P-450s, either in a single concerted reaction leading directly to the flavone double bond (flavone synthase II) or in a two-step process, the first being a monooxygenation of the C-ring, which yields a 2-hydroxyflavanone (flavanone 2-hydroxylase) or a 2-hydroxyisoflavanone (2HIS) intermediate. An alternative route for the conversion of flavanones to flavones involves, instead, a 2-oxoglutarate-dependent dioxygenase (14). The A-ring hydroxyl group in positions 5 and/or 7 is formed during the synthesis of the flavonoid skeleton catalyzed by chalcone synthase, a member of plant polyketide synthases (15). Additional hydroxyl groups in the A-ring of some flavonoid classes are found in positions 6 and 8. Enzymes involved in the hydroxylation of the A-ring at these positions have, however, not yet been described.

Isoflavone and pterocarpan derivatives play important roles in plant-microbe interactions as phytoalexins and nodulation factors and as phytoestrogens. In contrast to the constitutive production of isoflavonoids in plants, pterocarpans, such as glyceollin from soybean (Glycine max L.), are inducible and accumulate in pathogen-infected or elicitor-treated plant tissues (16). In an attempt to investigate transcriptionally regulated cytochromes P-450 activated by biotic stress in soybean, the technique of differential display of mRNA was recently employed (17). Eight full-length cDNA clones were subsequently isolated that represented elicitor-activated cytochromes P-450. One of these, CYP73A11 cDNA, encoded cin-

* This work was supported by Deutsche Forschungsgemeinschaft Grant SFB 369 and by funds from the Fonds der Chemischen Industrie (to J. E.), the Spanish Ministerio de Agricultura, Pesca y Alimentacion (to F. C.-H.), and the Alexander von Humboldt Foundation (to A. O. L.-D.). The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
§ These authors contributed equally to this work.
** To whom correspondence should be addressed. E-mail: j.ebel@botanik.biologie.uni-muenchen.de.

---

<sup>1</sup> The abbreviations used are: 2HIS, 2-hydroxyisoflavanone synthase; HPLC, high performance liquid chromatography; RP, reverse-phase; F6H, flavonoid 6-hydroxylase; Tricine, N-[2-hydroxy-1,1-bis(hydroxymethyl)ethyl]glycine.

namate 4-hydroxylase, and a second one, CYP93A1 cDNA, coded for 3,9-dihydroxypterocarpan 6a-hydroxylase (17, 18).

We now present the functional identification of CYP71D9 whose cDNA was also previously isolated from elicitor-induced soybean cells by using the differential display method (17). By employing heterologous expression the CYP71D9 cDNA was demonstrated to encode a protein capable of catalyzing the hydroxylation of ring A of flavonoid substrates. Combined studies with recombinant 2-hydroxyisoflavanone synthase (2HIS; CYP93C1v2) indicated that A-ring hydroxylation occurs before the 1,2-aryl shift of the flavonoid B-ring during isoflavanone formation.

### EXPERIMENTAL PROCEDURES

#### Chemicals—
Daidzein (7,4′-dihydroxyisoflavone), eriodictyol (5,7,3′4′-tetrahydroxyflavanone), genistein (5,7,4′-trihydroxyisoflavone), kaempferol (3,5,7,4′-tetrahydroxyflavone), luteolin (5,7,3′,4′-tetrahydroxyflavone), naringenin (5,7,4′-trihydroxyflavanone), and quercetin (3,5,7,3′,4′-pentahydroxyflavone) were purchased from Roth (Karlsruhe, Germany); factor 2 (6,7,4′-trihydroxyisoflavone) and liquiritigenin (7,4′-dihydroxyflavanone) were from Extrasynthèse (Genay, France). Apigenin (5,7,4′-trihydroxyflavone), biochanin A (5,7-dihydroxy-4′-methoxyisoflavone), dihydroquercetin (3,5,7,3′,4′-penta-hydroxyflavanone), and isoliquiritigenin (2′,6′,4-trihydroxychalcone) were from Sigma. Dihydrobiochanin A (5,7-dihydroxy-4′-methoxyisoflavone), dihydrokaempferol (3,5,7,4′-tetrahydroxyflavanone), formononetin (7-hydroxy-4′-methoxyisoflavone), 7-methoxy-4′-hydroxyflavanone, 7,4′-dimethoxyflavanone, and medicarpin (3-hydroxy-9-methoxypterocarpan) were from our laboratory collection.

#### Cell Culture and Extraction—
Soybean (*G. max* L. cv. Harosoy 63) cell suspension cultures were propagated in the dark as described earlier (19). For elicitation experiments, 6-day-old cultures were transferred into fresh medium 12 h prior to treatment with a β-glucan elicitor fraction (20) from *Phytophthora sojae* (200 μg/ml glucose equivalents) for 3–30 h. Cells were harvested by filtration, frozen in liquid nitrogen, and stored at −80 °C. Microsomal fractions were isolated by a modified version of the protocol described by Diesperger *et al.* (21). Frozen cells were homogenized in a mortar with a pestle and suspended in 0.2 M Tris-HCl, pH 7.5, 15% sucrose, 30 mM MgCl₂, 1 mM phenylmethylsulfonyl fluoride, and 1 mM dithiothreitol, in the presence of Dowex 1 × 2. After filtration through a nylon mesh and centrifugation at 12,000 × g for 20 min, the microsomal fraction was collected from the supernatant by centrifugation for 30 min at 50,000 × g, resuspended in 0.1 M KH₂PO₄/K₂HPO₄, pH 7.4, containing 30% glycerol, frozen in liquid nitrogen, and stored at −80 °C.

#### Candidate cDNA Expression in Yeast—
The *Saccharomyces cerevisiae* strain W303-1B, designated W(N), and its derivatives W(R) and WAT11 (22, 23), as well as the expression vector pYeDP60 (24), were provided by Rhône-Poulenc Agro (Lyon, France) and D. Pompon (Gif-sur-Yvette, France). The yeast strains had previously been engineered to either overexpress the NADPH-cytochrome P-450 reductase from yeast (W(R)) or the *Arabidopsis thaliana* isoform ATR1 (WAT11) upon galactose induction. The yeast strain WVS1 was engineered in the same way to overexpress the *Vicia sativa* P-450 reductase VS1 (accession number Z26250); VS1 was inserted into the integrative plasmid pYeDP110 (25) after polymerase chain reaction amplification for addition of *BamHI* and *SacI* restriction sites just 5′ and 3′ of the coding sequence using primers 5′-CGGGATCCCATGACTTCCTCTAATCCG (5′-end) and 5′-CGGGAGCTCTCACCAAACATCTCTTAGG (3′-end) and then integrated into W(N) with the *GAL10-CYC1* promoter at the locus of the endogenous reductase by homologous recombination.

The coding regions of the soybean P-450s were amplified using the primers specified earlier (26) and inserted into pYeDP60 according to Urban *et al.* (27). CYP93C1v2 cDNA (AF135484; a gift from R. A. Dixon and M. Gijzen, Samuel Roberts Noble Foundation, Ardmore, OK) (11) was amplified with the polymerase chain reaction primers 5′-atatagt-gatccATGTTCGCTTGAACCTTGCCAC (5′-end) and 5′-tatataggtaacTAAT-TAAGAAAGGAGTTTAG (3′-end) to generate *BamHI* and *KpnI* restriction sites just 5′ and 3′ of the coding sequence, before insertion into pYeDP60. Polymerase chain reaction was performed as described earlier (17), and the resulting plasmids were confirmed for identity by restriction and sequence analyses of the CYP-coding regions. Yeast strains W(R) (for CYP93C1v2) or WAT11 and WVS1 (for the other soybean P-450s) were transformed, and microsomal fractions were isolated as described earlier (17, 27). CYP expression was induced using the high density procedure (25), and cultures of 1–5 × 10⁸ cells/ml were used for microsome preparation.

#### Assay of Flavonoid 6-Hydroxylase and 2-Hydroxyisoflavanone Synthase Activities—
The standard assay for flavonoid 6-hydroxylase (F6H; CYP71D9, accession number: Y10490) contained in a total volume of 100 μl of 50 mM Tricine/KOH, pH 7.9, 0.5 mM reduced glutathione, 60 μg of microsomal protein from yeast, and 100 μM substrate (dissolved in either Me₂SO or 2-methoxy-ethanol). After equilibration for 2 min at 18 °C, the reaction was started with the addition of 20 μl of a NADPH-regenerating system (comprising 100 μM NADPH, 160 μM glucose 6-phosphate, and 0.04 unit of glucose 6-phosphate dehydrogenase in the reaction mixture) and terminated by the addition of 100 μl of 3% acetic acid in ethyl acetate. The products were extracted once with this solvent and twice thereafter with ethyl acetate from the reaction mixture. The organic phase was pooled and evaporated, and the residue was dissolved in 200 μl of a mixture of 40% methanol, 60% water, 0.2% acetic acid (v/v) and analyzed by reverse-phase high performance liquid chromatography (RP-HPLC) (LiChrosorb RP-18, 4 × 250 mm; flow rate, 1 ml min⁻¹; linear gradient from 35 to 65% methanol in 16 min; eluent 1). Compounds were detected at 290 nm and, when (*R*,*S*)-liquiritigenin (7,4′-dihydroxyflavanone) was used as substrate, the retention times were 10 min for the reaction product and 15 min for liquiritigenin. Amounts of products were calculated using molar extinction coefficients (ε₂₉₀ nm, MeOH = 6500 M⁻¹ cm⁻¹ for liquiritigenin; ε = 15500 M⁻¹ cm⁻¹ for naringenin; ε = 16800 M⁻¹ cm⁻¹ for eriodictyol) or relative to the conversion of liquiritigenin (all other substrates tested). The pH optimum was determined as described for the standard assay using 50 mM Tricine/KOH buffer, pH 7–9.

The assay for 2HIS contained in a total volume of 100 μl of 50 mM Tricine/KOH, pH 8.6 (28), 0.5 mM reduced glutathione, 60 μg of microsomal protein from yeast, and 100 μM liquiritigenin or 6,7,4′-trihydroxyflavanone. The reaction was run for 1 h at 15 °C. Products were extracted as described for the F6H assay and were analyzed by RP-HPLC using eluents comprising 1% acetic acid in a 50:50 mixture of methanol and acetonitrile (solvent A) and 1% acetic acid in H₂O (solvent B). Compounds were eluted at a flow rate of 1 ml min⁻¹, in a linear gradient of solvent A from 25 to 70% (eluents 2), within 18 min. Eluates were monitored at 290 nm.

For characterization of the reaction product from the combined catalytic action of F6H and 2HIS, the incubation with (*R*,*S*)-liquiritigenin was carried out at a larger scale (240 times that described above for the F6H and 2HIS standard assays). The F6H reaction was initiated by incubation at 18 °C, pH 7.9, for 1 h. This was followed by the addition of 2HIS, adjustment of the pH to 8.6 and a switch to 15 °C for an extra hour. The reaction was terminated, and the products were extracted as described above. To achieve a positive identification of the tetrahydroxylated isoflavanone product, the ethyl acetate residue was subjected to acid treatment by stirring in 500 μl of 10% HCl (v/v) in methanol for 1 h at room temperature. The mixture was extracted thrice with ethyl acetate, and, upon evaporation, the pooled organic phase was dissolved in 300 μl of methanol and analyzed by RP-HPLC as described above for the standard 2HIS assay. Fractions of the eluate containing the isoflavone derivative (*R*ₜ = ~16 min) were collected, concentrated, reapplied on the column to ascertain purity, reduced to dryness, and subjected to UV and NMR spectroscopy.

#### Spectrophotometric Measurements—
Spectrophotometric measurements of total P-450 content and evaluation of substrate binding were performed according to Omura and Sato (29) and Schalk *et al.* (30), respectively. Substrate-binding spectra were recorded using double cuvettes. *K*ₛ, ΔAₘₐₓ, and the corresponding S.D. values were calculated from the ΔA₃₉₀₋₄₂₀ nm for eight to ten ligand concentrations using the nonlinear regression program DNRPEASY. Flavonoids were dissolved in Me₂SO.

#### Mass Spectrometry—
Products of the F6H-catalyzed reaction were analyzed by mass spectrometry to determine the site of hydroxylation of the flavonoid substrates. For gas chromatography-mass spectrometry analysis, the samples were converted to trimethylsilyl ether derivatives with a mixture of bis-(trimethylsilyl)trifluoroacetamide containing 1% trimethylchlorosilane and pyridine (1:1 v/v) for 2 h at room temperature. The electronic impact analysis of 1-μl samples was performed on a Trio 2000 Micromass Quadrupole apparatus fitted with a J and W Scientific DB 5 MS (5% phenyl, 95% methyl) column (30 m × 0.320 mm inner diameter, 0.1 μm film) using He TPH 55 at 60 kPa as carrier gas. Initial column temperature was 120 °C, held for 2 min, and ramped to 250 °C at 15 °C min⁻¹ and then from 250 °C to 280 °C at 2 °C min⁻¹.

#### NMR Spectroscopy—
Nuclear magnetic resonance spectra of reaction products were acquired with a Bruker DMX 500 NMR spectrometer using a 5-mm inverse geometry probehead (90°(¹H) = 9.3 μs; 90°(¹³C) = 9.8 μs) in acetone-*d*₆ (2.04, 29.8 ppm) at 303 K. A phase-sensitive

Flavonoid 6-Hydroxylase, CYP71D9, from Soybean

the CYPs active sites. Type I binding spectra, indicative of a displacement of solvent in the vicinity of heme (33, 34), were recorded upon addition of naringenin to CYP71D9 and CYP82A2, dihydrokaempferol to CYP71D8 and CYP71D9, and eriodictyol to CYP71A9, CYP71D8, CYP71D9, and CYP82A2. Largest amplitude spectra were obtained upon binding of eriodictyol to CYP71D8 and CYP71D9 (Table II). As an example, spectra for CYP71D9 are shown in Fig. 1. No interaction of formononetin, genistein, or daidzein with any of the CYPs was detected.

The preliminary screening thus indicated that some of the soybean CYPs were binding flavonoids in their active site. For some of them, in particular CYP71D8 and CYP71D9, displacement of solvent was effective enough so that a positioning suitable for an oxidative attack was likely to be achieved. To test such a possibility, recombinant yeast microsomes were incubated with NADPH and liquiritigenin, eriodictyol, naringenin, or dihydrokaempferol. Formation of polar metabolites was observed with all four flavonoids but only upon incubation with CYP71D9.

Characterization of the Flavonoid 6-Hydroxylase—To further characterize the metabolite formed by CYP71D9, microsomes of recombinant WAT11 yeast were incubated with liquiritigenin and NADPH, and the ethyl acetate extract of the reaction mixture was analyzed by RP-HPLC (Fig. 2). Following RP-HPLC, the reaction product was identified by three criteria: retention time during HPLC, mass spectrometry, and NMR spectroscopy.

As shown in Fig. 2, the product P formed from liquiritigenin (S) had a smaller retention time (6 min) than the substrate (10 min), required NADPH for its formation, and was not formed when yeast cells were transformed with the empty vector. Similar results were recorded with naringenin as a substrate. The higher polarity of the products when compared with the substrates was fully supported by mass spectrometry of their trimethylsilyl derivatives. The metabolites of liquiritigenin and naringenin exhibited molecular ion peaks at *m/z* 488.2 and 576.2, respectively. The retro-Diels-Alder fragment peaks were found at *m/z* 296.1 (A-ring) and *m/z* 192.1 (B-ring) for the product formed from liquiritigenin, and at *m/z* 384.2 (A-ring) and at *m/z* 192.1 (B-ring) for that formed from naringenin. These results indicated that recombinant CYP71D9 protein catalyzed the monooxygenation of ring A of both flavanones (Fig. 3). Fragments hydroxylated on ring A were also observed upon gas chromatography-mass spectrometry analysis of the metabolites of eriodictyol and dihydrokaempferol.

To elucidate the position of hydroxylation of the A-ring, ^1^H NMR spectra were recorded in acetone-*d*~6~ of the product formed from liquiritigenin (Table III). The spectra clearly showed that the proton signal of H-6 was absent in the reaction product, whereas H-5 and H-8 formed singlets. All other signals were very similar to those observed for liquiritigenin. Taken together, the chemical characterization identified 6,7,4′-trihydroxyflavanone as the product formed from liquiritigenin in the reaction catalyzed by the recombinant CYP71D9 protein (Fig. 3). We conclude that the enzyme encoded by CYP71D9 is a F6H.

Catalytic Properties of Flavonoid 6-Hydroxylase—The binding constants (*K*~a~) for the interaction of different flavonoid compounds with CYP71D9 were determined with microsomes of recombinant WAT11 (Table IV). The results indicated that CYP71D9 exhibited highest affinity (1.6 μM) for the flavanone eriodictyol. Other flavanones, naringenin and liquiritigenin, as well as the dihydroflavonol dihydrokaempferol, were bound to CYP71D9 with affinities ranging from 9 to 52 μM. Methylation of the 7-hydroxyl position of the A-ring decreased the affinity

TABLE I  
Comparison of expression of the soybean CYPs in yeast strains overexpressing V. sativa (WVS1) or A. thaliana ATR1 (WAT11) P450 reductase  

P450 content in microsomal fractions prepared from yeast cells grown for 16h in the presence of galactose was determined from CO-reduced versus reduced difference spectra.

| cDNA      | P450     |
|-----------|----------|
|           | pmol mg⁻¹ protein |
| WVS1      | WAT11    |
| CYP71A9   | 14.1     | 44.9     |
| CYP71D8   | 24.7     | 13.1     |
| CYP71D9   | 72.4     | 127.8    |
| CYP82A2   | 35.9     | 106.5    |
| CYP82A3   | 0        | 16.5     |
| CYP82A4   | 0        | 0        |
| CYP93A3   | 21.3     | 56.7     |

(echo-antiecho selection) and sensitivity enhanced ^1^H, ^13^C heteronuclear single quantum coherence NMR spectrum of 6,7,4′-trihydroxyisoflavone was acquired using Bruker standard software (^1^J(CH) = 165 Hz; acquisition time, 203 ms; spectral width, 5040 Hz (F2), 106 F1 (^13^C) increments with a final resolution of 83 Hz; ^13^C GARP decoupling: 70 μs, gradient; pulse, 1 ms; recovery, 450 μs). The pK values of 6,7,4′-trihydroxyisoflavone were calculated with the ACD/Labs (Pegnitz, Germany) p*K*~a~ DataBase, Version 4.5.

RESULTS

Yeast Expression and Functional Screening of Soybean CYPs—Earlier investigations (17) identified eight cDNA clones representing cytochromes P-450 whose expression in soybean cell cultures was activated by elicitor treatment concomitantly with the production of glyceollins, the pterocarpanoid phytoalexins of soybean: CYP73A11, CYP82A2, CYP82A3, CYP82A4, CYP93A1, CYP93A3, CYP71D8, and CYP71D9. Activated expression of another P-450 (CYP71A9) isolated in the same mRNA differential display screening was refuted when tested by Northern blot analysis. Previous analyses of the catalytic properties of recombinant proteins expressed in yeast disclosed the function of two of the clones. CYP73A11 cDNA encoded a cinnamate 4-hydroxylase (17), whereas CYP93A1 cDNA coded for 3,9-dihydroxypterocarpan 6a-hydroxylase (18). The former cytochrome P-450 thus represented a well-studied enzyme of general phenylpropanoid metabolism whose action gives rise to the hydroxyl group in position 4′ of the flavonoid B-ring. More significantly, the latter cytochrome P-450 catalyzes the stereoselective and regioselective hydroxylation of position 6a of 3,9-dihydroxypterocarpan to give (S)-3,6a,9-trihydroxypterocarpan (glycinol), a biosynthetic intermediate of the glyceollins (18). These earlier studies thus demonstrated that within the isolated group of CYP clones at least two cDNAs were related to phenylpropanoid and isoflavonoid pathways.

In an attempt to identify other candidate P-450 cDNAs involved in flavonoid biosynthesis, all the other coding sequences were also expressed in yeast. Because we previously showed that the level of P-450 expression in yeast might be strongly dependent on the coexpressed P-450 reductase (31, 32), a yeast strain overexpressing a P-450 reductase (accession number Z26250), isolated from the legume V. sativa, was constructed. The expression of the different P-450s in this strain WVS1 was compared with that in the strain WAT11, overexpressing the A. thaliana reductase ATR1 (Table I). No expression of CYP82A4 was obtained in either strain. For all other CYPs, except CYP71D8, twice as much expression was obtained in WAT11 compared with WVS1. The ratio was reversed for CYP71D8, which appeared to be more stable in the presence of the reductase from V. sativa. Microsomal fractions isolated from the most favorable yeast strains were then systematically screened with a variety of flavonoid compounds for specific binding into

Flavonoid 6-Hydroxylase, CYP71D9, from Soybean

TABLE II  
Screening for flavonoid binding to soybean CYPs in recombinant yeast microsomes  

The microsomal fraction was prepared from yeast cells grown for 16 h in the presence of galactose. Difference spectra were recorded in 0.1 M Tris-HCl, pH 7.5, containing 30% glycerol and 1 mM EDTA after adding 100 μM of each of the compounds to the oxidized microsomes in the sample cuvette and an equal volume of solvent to the reference microsomes.

| Compounds tested | CYP93A3 | CYP71A9 | CYP71D8 | CYP71D9 | CYP82A2 | CYP82A3 |
|------------------|---------|---------|---------|---------|---------|---------|
| Naringenin       | 0       | 0       | 0       | 13      | 7       | 0       |
| Dihydrokaempferol | 0       | 0       | 8       | 22      | 0       | 0       |
| Eriodictyol      | 0       | 28      | 56      | 56      | 8       | 0       |
| Dihydroxypterocarpan | 18     | 0       | 0       | 0       | 1.4     | 0       |
| Formononetin     | 0       | 0       | 0       | 0       | 0.8     | 0       |
| Genistein        | 0       | 0       | 0       | 0       | 0       | 0       |
| Daidzein         | 0       | 0       | 0       | 0       | 0       | 0       |

FIG. 1. Carbon monoxide and substrate-binding spectra recorded with recombinant CYP71D9 in WAT11 yeast microsomes. The microsomal fraction was prepared from yeast cells grown for 16 h in the presence of galactose, resuspended, and diluted to 1 mg ml⁻¹ protein in 0.1 M Tris-HCl, pH 7.5, containing 30% glycerol and 1 mM EDTA. A, difference spectrum of CO-treated reduced microsomes versus reduced microsomes. B, spectra recorded upon addition of 200 μM of each of the indicated flavonoid compounds to the sample cuvette containing reduced microsomes, an equal volume of solvent being added to the reference reduced microsomes. Base lines were recorded before addition of CO or substrate. DHkaempferol, dihydrokaempferol.

by more than 1 order of magnitude. The highest maximal amplitudes of ΔA₃₉₀₋₄₂₀ₙₘ, at saturating ligand concentration, were obtained for eriodictyol and 7-methoxy-4′-hydroxyflavanone followed by liquiritigenin. The high light absorption of flavones above 350 nm prevented determination of the binding constants for these compounds.

Product formation with recombinant CYP71D9 in WAT11 was proportional to time for about 10 min at a temperature of 20 °C or below. At 30 or 25 °C, activity was higher, but the enzyme was highly unstable. The optimal pH for the reaction was 7.9. A significant increase in activity and stability of the enzyme or amount of product of the reaction was achieved by inclusion of reduced glutathione at 0.5 mM in the reaction mixture. When assayed under optimal conditions, the apparent Kₘ value of recombinant F6H for liquiritigenin was found to be about 7 μM. The Kcat at 18 °C was around 50 min⁻¹. Complete conversion of racemic liquiritigenin was obtained, indicating that the enzyme did not discriminate the 2S and 2R configurations of the flavanone. A variety of compounds were tested as possible substrates for the recombinant F6H at a substrate concentration of 100 μM under standard assay conditions. Product analysis was performed by RP-HPLC. As reported in Table

FIG. 2. HPLC analysis of (R,S)-liquiritigenin conversion in the F6H assay. A, incubation of liquiritigenin and NADPH with a microsomal fraction from yeast cells (strain WAT11) expressing CYP71D9. B, assay as in A but omitting NADPH. C, incubation with a microsomal fraction from control yeast transformed with the vector pYeDP 60 without insert. Ethyl acetate extracts of reaction mixtures were separated on a LiChrosorb RP-18 column using eluent 1. S, substrate; P, product.

V, recombinant F6H catalyzed the hydroxylation of different classes of flavonoids. Most efficient hydroxylation was observed with flavanones (liquiritigenin, eriodictyol, and naringenin), and the dihydroflavonol, dihydrokaempferol. Except for 7-methoxy-4′-hydroxyflavanone, which occupied a large volume of the active site but was not properly positioned for oxygenation, efficiency of metabolism correlated well with the amplitudes of

Flavonoid 6-Hydroxylase, CYP71D9, from Soybean

min, was the predominant product with liquiritigenin (S) as substrate. When 6,7,4′-trihydroxyflavanone (S) was incubated with the microsomal fraction containing recombinant 2HIS two new products with retention times of 5.4 (P1) and 15.1 min (P2), respectively, were formed (Fig. 4C). The retention time during RP-HPLC indicated that the main product P1 was more polar than the substrate, similar to the situation with liquiritigenin as substrate (Fig. 4, B and C).

For product characterization, the suspected 2,6,7,4′-tetrahydroxyisoflavanone (Fig. 3) was produced from (R,S)-liquiritigenin on a larger scale by the combined catalytic action of recombinant F6H and 2HIS. Upon separation of the ethyl acetate extract of the reaction mixture by HPLC (Fig. 5), it could be seen that liquiritigenin was almost completely converted. As shown in the HPLC chromatogram of the reaction products, the amount of the fraction at \( R_t \) of 5.4 min, containing the putative 2,6,7,4′-tetrahydroxyisoflavanone (P2) and the fraction at \( R_t \) 14 min (6, 7, 4′-trihydroxyflavanone, P1) each accounted for ~50%. Products in P2 appeared to be rather unstable and were sensitive to both alkaline and acid treatments. When the ethyl acetate extract was treated with HCl, the relative amount of the product(s) P2 greatly decreased, and the product P3 eluting at \( R_t \sim 16 \) min was formed (Fig. 5). The UV spectrum of the latter compound (\( \lambda_{\text{max}} = 212, 230, 257, \) and 323 nm in methanol) was identical to that of authentic 6,7,4′-trihydroxyisoflavone and consistent with values reported earlier (39). Furthermore, the NMR spectrum of the compound proved that it was 6,7,4′-trihydroxyisoflavone (Fig. 3 and Table III) also known as factor 2 (39). Owing to strong electronic interactions within the 6,7,4′-trihydroxyisoflavone system, the pK values of the individual phenolic protons cover a wide range (pK (6), 11.96 ± 0.15; pK (7), 7.39 ± 0.40; and pK(4′), 9.76 ± 0.15). Traces of acids might have accumulated during purification and strongly affected the proton chemical shift of individual positions in arbitrary directions, and deviations up to 0.3 ppm were observed without altering the splitting by J coupling. The \({}^{13}\) C chemical shifts of all cross-peaks within a heteronuclear single quantum coherence NMR spectrum measured (40) before high vacuum treatment perfectly agreed with those of the reference standard compound (Table III). After high vacuum treatment (5 h, \( 10^{-3} \) mbar), also all of the proton chemical shifts agreed within 0.02 ppm with those of the reference compound.

A major product of the sequential catalytic actions of recombinant F6H and 2HIS thus most likely represented 2,6,7,4′-tetrahydroxyisoflavanone, which upon acid treatment could be dehydrated to yield the corresponding isoflavone. The experiments outlined in Figs. 4 and 5 therefore clearly demonstrated that recombinant 2HIS efficiently utilized as substrate a flavanone carrying a hydroxyl group in position 6 of the A-ring.

DISCUSSION

Natural products with hydroxyl substitution at position 6 of the A-ring are reported for several flavonoid classes including the isoflavonoids (37). Hydroxylation of the A-ring at this position requires an enzyme-catalyzed reaction subsequent to chalcone skeleton formation in the reaction mediated by chalcone synthase. A flavonoid 6-hydroxylase, of necessity, should occur in soybean because the 6,7,4′-trihydroxylated substitution pattern, as found in glycitein and afrormosin (7-hydroxy-6, 4′-dimethoxyisoflavone), has been reported for isoflavonoid constituents of this plant (35, 36, 41). In the current study, we identified the catalytic function of the CYP71D9 protein, whose full-length cDNA had been cloned earlier from elicited soybean cell cultures (17), to be a flavonoid 6-hydroxylase. To our knowledge, this represents the first demonstration of a cytochrome P-450-dependent monooxygenase from plants capable of mediating A-ring hydroxylation of flavonoid substrates. Product

Flavonoid 6-Hydroxylase, CYP71D9, from Soybean

**TABLE III**

${}^{1} \mathrm{H}$ and ${}^{13} \mathrm{C}$ NMR spectra of liquiritigenin, $6,7,4^{\prime}$-trihydroxyflavanone, and $6,7,4^{\prime}$-trihydroxyisoflavone measured in acetone-$d_{6}$

| Proton | 7,4'-Dihydroxyflavanone (Liquiritigenin) | $6,7,4^{\prime}$-Trihydroxyflavanone | $6,7,4^{\prime}$-Trihydroxyisoflavone |
| --- | --- | --- | --- |
|  | $\delta$ | $\delta$ | $\delta$ |
|  | $ppm$ | $ppm$ | $ppm$ |
|  | $Hz$ | $Hz$ | $Hz$ |
| H-2 | $5.44(d d, 1)$ | $5.36(d d, 1)$ | $8.08(s, 1)$ |
| H-3ax | $3.03(d d, 1)$ | $2.96(d d, 1)$ |  |
| H-3eq | $2.67(d d, 1)$ | $2.62(d d, 1)$ |  |
| H-5 | $7.72(d, 1)$ | $7.23(s, 1)$ | $7.53(s, 1)$ |
| H-6 | $6.56(d d, 1)$ |  |  |
| H-8 | $6.41(d, 1)$ | $6.43(s, 1)$ | $6.91(s, 1)$ |
| H-2'/6' | $7.39(d, 2)$ | $7.37(d, 2)$ | $7.45(d, 2)$ |
| H-3'/5' | $6.89(d, 2)$ | $6.87(d, 2)$ | $6.86(d, 2)$ |

$\delta^{1} \mathrm{H}^{a}$ | $\delta^{13} \mathrm{C}^{a}$ |
| --- | --- |
| $2.9,13.0$ | $8.06$ | $152.71$ |
| $13.0,16.7$ | $13.1,16.7$ |  |
| $2.9,16.7$ | $2.9,16.7$ |  |
| $8.7$ |  | $7.51$ | $110.39$ |
| $2.3,8.7$ |  |  |  |
| $2.3$ | $8.6$ | $6.88$ | $105.11$ |
|  | $8.3$ | $8.7$ | $7.46$ | $131.90$ |
|  | $8.7$ | $8.5$ | $6.86$ | $116.90$ |

$a$ Chemical shift values taken from the ${}^{1} \mathrm{H},{ }^{13} \mathrm{C}$ heteronuclear single quantum coherence NMR spectrum before vacuum treatment.

**TABLE IV**

Binding constants and maximal binding amplitudes measured upon binding of different flavonoids to recombinant CYP71D9

$K_{s}$ values were calculated from the shift in the Soret absorbance measured when increasing concentrations of ligand were added to oxidized recombinant yeast microsomes. The maximal amplitude of the $\Delta A_{390-420 \mathrm{~nm}}$ at saturating substrate concentrations shows how efficient is the displacement of solvent interacting with heme resulting from the binding of each flavonoid. High values usually indicate favourable positioning for oxidative attack. Kinetic data were fitted using the nonlinear regression program DNRPEASY derived by Duggleby (53) from DNRP 53.

| Ligand | $K_{s}$ | $\Delta A_{\max }$ |
| --- | --- | --- |
|  | $\mu M$ | $\Delta A_{390-420 n m}$ |
|  |  | $m M^{-1} P 450$ |
| Eriodictyol | $1.60 \pm 0.02$ | $77.40 \pm 0.4$ |
| Naringenin | $9.10 \pm 0.4$ | $21.80 \pm 0.2$ |
| Dihydrokaempferol | $37.10 \pm 0.7$ | $42.20 \pm 0.2$ |
| Liquiritigenin | $51.60 \pm 5.0$ | $64.60 \pm 3.0$ |
| 7-Methoxy-4'-hydroxyflavanone | $647.10 \pm 53.6$ | $73.30 \pm 5.2$ |

**TABLE V**

Relative rates of conversion of different flavonoid substrates in the reaction catalyzed by F6H

The ethyl acetate extracts of the reaction mixtures were analyzed by RP-HPLC. NO, no conversion observed.

| Substrate | Relative V (%) |
| --- | --- |
| Flavanones |  |
| Liquiritigenin${ }^{a}$ | 100 |
| Naringenin${ }^{a}$ | 30 |
| Eriodictyol${ }^{a}$ | 81 |
| 7-Methoxy-4'-hydroxyflavanone | NO |
| 7,4'-Dimethoxy-flavanone | NO |
| Chalcones |  |
| Isoliquiritigenin | NO |
| Flavones |  |
| Apigenin${ }^{b}$ | 18 |
| Luteolin${ }^{b}$ | 19 |
| (Dihydro)flavonols |  |
| Dihydrokaempferol${ }^{b}$ | 36 |
| Kaempferol${ }^{b}$ | 2 |
| Dihydroquercetin | NO |
| Dihydroisoflavones |  |
| Dihydrobiochanin A${ }^{b}$ | 6 |
| Isoflavones |  |
| Daidzein | NO |
| Genistein | NO |
| Biochanin A${ }^{b}$ | 1 |

$a$ Amount of products calculated based on molar extinction coefficients (liquiritigenin, naringenin, and eriodictyol).

$b$ Amount of products calculated as values relative to the conversion of liquiritigenin (100% represents $122 \mu \mathrm{kat} \mathrm{kg}^{-1}$).

**FIG. 4.** HPLC analysis of liquiritigenin and $6,7,4^{\prime}$-trihydroxyflavanone conversion in the 2HIS assay. $A$, reference compounds. $R 1$, $6,7,4^{\prime}$-trihydroxyflavanone; $R 2$, $6,7,4^{\prime}$-trihydroxyisoflavone; $R 3$, liquiritigenin ($7,4^{\prime}$-dihydroxyflavanone); $R 4$, daidzein ($7,4^{\prime}$-dihydroxyisoflavone). $B$, incubation of $(R, S)$-liquiritigenin and NADPH with a microsomal fraction from yeast W(R) expressing CYP93C1v2. $C$, assay as in $B$ using $6,7,4^{\prime}$-trihydroxyflavanone as substrate. Ethyl acetate extracts of reaction mixtures were separated by RP-HPLC using eluent 2. No products were formed in $B$ or $C$ in the absence of NADPH or with microsomes from the yeast strain transformed with empty vector. $R$, reference compound; $S$, substrate; $P$, product.

characterization by mass spectrometry and NMR spectroscopy conclusively showed that the enzyme catalyzed a regiospecific monooxygenation at position 6 of the substrates. Whereas other CYP proteins using flavonoids as substrates have been assigned to the families CYP75 (flavonoid 3',5'-hydroxylase,

Flavonoid 6-Hydroxylase, CYP71D9, from Soybean

![Figure](https://i.imgur.com/yourimageurl.png)

FIG. 5. HPLC analysis of products formed from (R,S)-liquiritigenin by the combined catalytic action of recombinant F6H and 2HIS. Liquiritigenin and NADPH were first incubated with a microsomal fraction from yeast WAT11 expressing CYP71D9 before addition of microsomes from W(R) expressing CYP93C1v2. Reaction mixtures were extracted with ethyl acetate, and the extracted compounds were analyzed by RP-HPLC using eluent 2 either directly (solid line) or after treatment with 10% HCl in methanol (dashed line). P, product.

member of enzymes in the CYP71 family being capable of accepting flavonoids as substrates. The four other members of the CYP71D subfamily whose enzyme functions have been identified so far are CYP71D12 from Madagascar periwinkle (*Catharanthus roseus*) (42) and CYP71D13, CYP71D15, and CYP71D18 isolated from different species of *Mentha* (43). Upon functional expression in heterologous hosts, CYP71D12 protein was demonstrated to be tabersonine 16-hydroxylase, an enzyme involved in indole alkaloid biosynthesis, whereas the monooxygenases from mint were shown to catalyze the regiospecific hydroxylations in position 6 or 3 of (-)-limonene leading, respectively, to the synthesis of carvone or menthol, which are responsible for the characteristic flavors of spearmint and peppermint.

In the current study, in a quest for clues to the physiological functions of the orphan soybean CYPs (17, 18), we screened for potential substrates using spectrophotometric detection of ligand binding. Although this method uses relatively large amounts of recombinant material, it is much faster than assays of metabolism and provided information about the type of compounds (e.g., flavonoids or isoflavonoids) that are able to bind to each active site. In the case of CYP71D9, affinity and positioning of the ligands could be predicted from saturation curves and correlated well with data from metabolism. However, some of the other P-450s, equally induced upon elicitation, were able to bind flavonoids in their active site but did not metabolize them. For example, CYP71D8 was observed to bind eriodictyol with an efficiency comparable with that of CYP71D9 but showed no sign of metabolism of the ligand. This raises the interesting possibility that flavonoids may act as regulators of the activity of P-450 enzymes involved in other metabolic pathways. Numerous examples of inhibition of mammalian or insect P-450s by dietary flavonoids have been reported with impact on the metabolism of drugs (44), the activation of procarcinogens (45), or the biosynthesis of steroid hormones (46, 47). In the case of plants, the binding of nonsubstrate flavonoids to P-450 enzymes could have an impact on specific physiological functions and result in a regulatory cross-talk between independent pathways. It is significant that CYP71D8 and CYP71D9 do not

FIG. 6. Phylogenetic relationships in the CYP71D sub-family of P-450 enzymes. Soybean P-450s are indicated on a gray background, and enzymes of known functions are shown in relief. CYP71D6 and CYP71D7 are P-450s from *Solanum chacoense* (U48434 and U48435), CYP71D8, CYP71D9, and CYP71D10 are from *G. max* (Y10493, Y10490 and AF022459), CYP71D11 is from *Lotus japonicus* (AF000403), CYP71D12 is the tabersonine 16-hydroxylase from *C. roseus* (AJ238612), CYP71D13 and CYP71D15 are (-)-limonene-3-hydroxylases from *Mentha piperita* (AF124816 and AF124817), CYP71D18 is (-)-limonene-6-hydroxylase from *Mentha spicata* (AF124815), and CYP76D16 is from *Nicotiana tabacum* (AF166821). The scale bar represents amino acid substitutions per site.

share more than 47% identity, thus arguing for quite divergent functions of the two enzymes. P-450s from the subfamily CYP71D isolated so far have been shown to be involved in diverse pathways, ranging from indole alkaloid to monoterpene biosynthesis. Our report in this paper adds a third biosynthetic pathway to the list. A phylogenetic analysis of the CYP71D sub-family (Fig. 6) indicates that this functional diversity is related to a relatively large evolutionary distance between its members, which are likely to participate in still further biosynthetic routes.

Combined data obtained in the current study from binding and metabolism provide valuable information on the substrate specificity of F6H. The recombinant hydroxylase appears capable of converting a variety of flavonoid substrates containing a resorcinol- and phloroglucinol-based A-ring but with a marked preference for flavanones over flavones (Table V). The hydroxylation status of rings A and B may have a critical influence on the anchoring and orientation of the compounds in the active site (Tables IV and V). Double hydroxylation of the B-ring increases both the affinity for F6H and the efficiency of metabolism. Double hydroxylation of the A-ring also increases affinity but decreases metabolism. Binding data suggest that a hydroxyl at C5 increases the constraints on the bound substrate and the distance from the 6-position to the ferryl-oxo species that carries on the oxidative attack. Affinity, as judged by \(K_s\), is drastically decreased, and metabolism is completely abolished upon methylation of the 7-hydroxyl group on ring A. The 7-hydroxyl group is thus essential both for anchoring the substrate and maintaining the 6-position at the proper distance to the active center. Furthermore, both the oxidation status and the substitution pattern of ring C also appear to have an impact on the binding and rate of conversion of flavonoids by F6H (Tables IV and V). In all, a number of flavanones and the dihydroflavonol, dihydrokaempferol, are hydroxylated at high efficiency, and flavones are converted to a lesser extent, whereas the flavonol, kaempferol, and (dihydro)isoflavones are barely converted. It is thus likely that the *in vivo* function of F6H is to introduce a 6-hydroxyl group at the flavanone level. This assumption is strongly supported by our

finding that 2HIS accepts 6-hydroxyflavanones, e.g. 6,7,4′-tri-
hydroxyflavanone, as substrates. Hydroxyl groups at position 6
of the A-ring are also known for other flavonoid classes includ-
ing flavones, flavonols, and anthocyanins (37). It will be inter-
esting to characterize the biosynthetic relationship of F6H to
the flavonoid branch pathways giving rise to these compounds.
F6H cDNA was isolated initially from elicitor-treated soy-
bean cell cultures (17). Subsequently it was shown that elicitor
treatment caused an increase in the level of F6H mRNA exhib-
iting a time course similar to that observed for several other
elicitor-responsive mRNAs encoding enzymes of flavonoid bio-
synthesis pathways (17). It remains to be analyzed whether
among the inducibly formed flavonoid compounds 6,7-dihy-
droxylated derivatives might occur. This substitution pattern
is represented in 6,7,4′-trihydroxyisoflavone (factor 2), a puta-
tive precursor of the constitutively formed isoflavonoid gly-
citein (35, 36). The formation of polyhydroxylated isoflavones,
including 6,7,4′-trihydroxyisoflavone, from the soybean isofla-
vones daidzein and glycitein has been observed during tempe
(fermented soybean) production and has been attributed to the
metabolic capacity of bacteria including *Micrococcus* and Ar-
throbacter* species by performing hydroxylation and demethy-
lation reactions at ring A (39, 48). In the present study, 6,7,4′-
trihydroxyisoflavone was obtained from enzymatically formed
2,6,7,4′-tetrahydoxyisoflavanone by acid treatment. *In vivo*, 
this dehydration is catalyzed by a dehydratase (49). Polyhy-
droxylated isoflavones have been known to exhibit anti-oxi-
dant, anti-inflammatory, anti-allergic, and anti-carcinogenic
activities and thus have an impact on mammalian biology (50).
For the antioxidant activity of trihydroxyisoflavone, the 6,7-
ortho-dihydroxy group seems to be essential (51, 52). In addi-
tion to their impact on mammalian biology, it would be inter-
esting to learn more about the possible roles of these
compounds in the interactions of plants with their
environment.

Acknowledgments—The kind provision of yeast strain W(R) by
Rhône-Poulenc Agro (Lyon, France), of strain WAT11 by Dr. D. Pompon
(Gif-sur-Yvette, France), of CYP93C1v2 cDNA by Drs. R. A. Dixon and
M. Gijzen (Ardmore, OK), of the *V. sativa* P-450 reductase cDNA VS1 by
Drs. I. Benveniste and F. Durst (Department of Enzymology, CNRS-
IBMP, Strasbourg, France) and the kind gift of \[^{14}C\]liquiritigenin by
Dr. K. Stich (Vienna, Austria) are gratefully acknowledged. The advice
of Drs. A. Mithöfer and E. G. Cosio is gratefully acknowledged.

**REFERENCES**

1. Harborne, J. B., and Grayer, R. J. (1994) in *The Flavonoids: Advances in*
   Research since 1986 (Harborne, J. B., ed) pp. 589–618, Chapman and Hall,
   London
2. Dixon, R. A., and Steele, C. L. (1999) *Trends Plant Sci.* 4, 394–400
3. Heller, W., and Forkmann, G., (1994) in *The Flavonoids: Advances in Research*
   since 1986 (Harborne, J. B., ed) pp. 499–535, Chapman and Hall, London
4. Forkmann, G. (1994) in *The Flavonoids: Advances in Research since 1986*
   (Harborne, J. B., ed) pp. 537–564, Chapman and Hall, London
5. Brugliera, F., Barri-Rewell, G., Holton, T. A., and Mason J. G. (1999) *Plant J.*
   19, 441–451
6. Holton, T. A., Brugliera, F., Lester, D. R., Tanaka, Y., Hyland, C. D., Menting,
   J. G. T., Lu C.-Y., Farcy, E., Stevenson, T. W., and Cornish, E. C. (1993)
   *Nature* 366, 276–279
7. Akashi, T., Aoki, T., and Ayabe, S. (1998) *Biochem. Biophys. Res. Commun.*
   251, 67–70
8. Akashi, T., Aoki, T., and Ayabe, S. (1998) *FEBS Lett.* 431, 287–290
9. Martens, S., and Forkmann, G. (1999) *Plant J.* 20, 611–618
10. Akashi, T., Fukuchi-Mizutani, M., Aoki, T., Ueyama, Y., Yonekura-
    Sakakibara, K., Tanaka, Y., Kusumi, T., and Ayabe, S. (1999) *Plant Cell*
    *Physiol.* 40, 1182–1186
11. Steele, C. L., Gijzen, M., Qutob, D., and Dixon, R. A. (1999) *Arch. Biochem.*
    *Biophys.* 367, 146–150
12. Akashi, T., Aoki, T., and Ayabe, S. (1999) *Plant Physiol.* 121, 821–828
13. Jung, W., Yu, O., Lau, S.-M. C., O’Keefe, D. P. O., Odell, J., Fader, G., and
    McGonigle, B. (2000) *Nat. Biotechnol.* 18, 208–212
14. Britsch, L. (1990) *Arch. Biochem. Biophys.* 282, 152–160
15. Schröder, J. (1997) *Trends Plant Sci.* 2, 373–378
16. Ebel, J., and Grisebach, H. (1988) *Trends Biochem. Sci.* 13, 23–27
17. Schopfer, C. R., and Ebel, J. (1998) *Mol. Gen. Genet.* 258, 315–322
18. Schopfer, C. R., Kochs, G., Lottspeich, F., and Ebel, J. (1998) *FEBS Lett.* 432,
    182–186
19. Ebel, J., Ayers, A., and Albersheim, P. (1976) *Plant Physiol.* 57, 775–779
20. Schmidt, W. E., and Ebel, J. (1987) *Proc. Natl. Acad. Sci. U. S. A.* 84,
    4117–4121
21. Diesperger, H., Müller, C. R., and Sandermann, H., Jr. (1974) *FEBS Lett.* 43,
    155–158
22. Truan, G., Cullin, C., Reisdorf, P., Urban, P., and Pompon, D. (1993) *Gene*
    *(Amst.)* 125, 49–55
23. Urban, P., Mignotte, C., Kazmaier, M., Delorme, F., and Pompon, D. (1997)
    *J. Biol. Chem.* 272, 19176–19186
24. Urban, P., Cullin, C., and Pompon, D. (1990) *Biochimie (Paris)* 72, 463–472
25. Pompon, D., Louerat, B., Bronnie, A., and Urban, P. (1996) *Methods Enzymol.*
    272, 51–64
26. Schopfer, C. (1998) *Isolation of elicitor inducible CYPs of soybean (G. max L.)*
    *and identification and functional expression of cDNAs encoding cinnamate*
    *4-hydroxylase and dihydroxypterocarpan 6a-hydroxylase*. Doctoral thesis,
    University of Munich, Germany
27. Urban, P., Werck-Reichhart, D., Teutsch, H. G., Durst, F., Regnier, S.,
    Kazmaier, M., and Pompon, D. (1994) *Eur. J. Biochem.* 222, 843–852
28. Kochs, G., and Grisebach, H. (1986) *Eur. J. Biochem.* 155, 311–318
29. Omura, T., and Sato, R. (1964) *J. Biol. Chem.* 239, 2370–2378
30. Schalk, M., Batard, Y., Seyer, A., Nedelkina, S., Durst, F., and Werck-
    Reichhart, D. (1997) *Biochemistry* 36, 15253–15261
31. Cabello-Hurtado, F., Batard, Y., Salaün, J. P., Durst, F., Pinot, F., and Werck-
    Reichhart, D. (1998) *J. Biol. Chem.* 273, 7260–7267
32. Robineau, T., Batard, Y., Nedelkina, S., Cabello-Hurtado, F., Le Ret, M.,
    Sorokine, O., Didierjean, L., and Werck-Reichhart, D. (1998) *Plant Physiol.*
    118, 1049–1056
33. Raag, R., and Poulos, T. L. (1989) *Biochemistry* 28, 917–922
34. Loida, P. J., and Sligar, S. G. (1993) *Biochemistry* 32, 11530–11538
35. Naim, M., Gestetner, B., Zilkah, S., Birk, Y., and Bondi, A. (1974) *J. Agr. Food*
    *Chem.* 22, 806–810
36. Kudou, S., Fleury, Y., Welter, D., Magnolato, D., Uchida, T., Kitamura, K., and
    Okubo, K. (1991) *Agric. Biol. Chem.* 55, 2227–2233
37. Harborne, J. B., ed. (1994) *The Flavonoids: Advances in Research since 1986*
    (Harborne, J. B., ed) pp. 1–676, Chapman and Hall, London
38. Hashim, M. F., Hakamatsuka, T., Ebizuka, Y., and Sankawa, U. (1990) *FEBS*
    *Lett.* 271, 219–222
39. Klus, K., and Barz, W. (1995) *Arch. Microbiol.* 164, 428–434
40. Agrawal, P. K. (1989) in *Carbon-13 NMR of Flavonoids* (Agrawal, P. K., ed) pp.
    96–116 and 192–207, Elsevier, Amsterdam
41. Caballero, P., Smith, C. M., Fronczek, F. R., and Fischer, N. H. (1986) *J. Nat.*
    *Prod.* 49, 1126–1129
42. Schröder, G., Unterbusch, E., Kaltenbach, M., Schmidt, J., Strack, D.,
    De Luca, V., and Schröder, J. (1999) *FEBS Lett.* 458, 97–102
43. Lupien, S., Karp, F., Wildung, M., and Croteau, R. (1999) *Arch. Biochem.*
    *Biophys.* 368, 181–192
44. Li, Y., Wang, E., Patten, C. J., Chen, L., and Yang, C. S. (1994) *Drug Metab.*
    *Dispos.* 22, 566–571
45. Lee, H., Wang, H.-W., Su, H.-H., and Hao, N. J. (1994) *Mutagenesis* 9, 101–106
46. Mitchell, M. J., Keogh, D. P., Crooks, J. R., and Smith, S. L. (1993) *Insect*
    *Biochem. Biol.* 23, 65–71
47. Le Bail, J. C., Laroche, T., Marre-Fournier, F., and Habrioux, G. (1998) *Canc.*
    *Lett.* 133, 101–106
48. Klus, K., and Barz, W. (1998) *Phytochemistry* 47, 1045–1048
49. Hakamatsuka, T., Mori, K., Ishida, S., Ebizuka, Y., and Sankawa, U. (1998)
    *Phytochemistry* 49, 497–505
50. Middleton, E. Jr., and Kandaswami, C. (1994) in *The Flavonoids: Advances in*
    *Research since 1986* (Harborne, J. B., ed) pp. 619–652, Chapman and Hall,
    London
51. György, P., Murata, K., and Ikehata, H. (1964) *Nature* 203, 870–872
52. Jha, H. C., Bockemühl, S., and Egge, H. (1990) in *Second Asian Symposium on*
    *Non-Salted Soybean Fermentation* (Hermana, Mien M., and Karyadi, D., eds)
    pp. 4–14, Ministry of Health, Jakarta, Indonesia
53. Duggleby, R. G. (1984) *Comput. Biol. Med.* 14, 447–455

METABOLISM AND BIOENERGETICS:
Flavonoid 6-Hydroxylase from Soybean (Glycine maxL.), a Novel Plant P-450 Monooxygenase

Akinwunmi Olumide Latunde-Dada, Francisco Cabello-Hurtado, Nikola Czittrich, Luc Didierjean, Christel Schopfer, Norbert Hertkorn, Danièle Werck-Reichhart and Jürgen Ebel
J. Biol. Chem. 2001, 276:1688-1695.
doi: 10.1074/jbc.M006277200 originally published online October 10, 2000

Access the most updated version of this article at doi: 10.1074/jbc.M006277200

Find articles, minireviews, Reflections and Classics on similar topics on the JBC Affinity Sites.

Alerts:
- When this article is cited
- When a correction for this article is posted

Click here to choose from all of JBC's e-mail alerts

This article cites 45 references, 10 of which can be accessed free at http://www.jbc.org/content/276/3/1688.full.html#ref-list-1
