
Autophagy as a Regulated Pathway of Cellular Degradation
Daniel J. Klionsky and Scott D. Emr
Science 290, 1717 (2000);
DOI: 10.1126/science.290.5497.1717

This copy is for your personal, non-commercial use only.

If you wish to distribute this article to others, you can order high-quality copies for your colleagues, clients, or customers by [clicking here](https://www.science.org/action/showCitFormats?doi=10.1126/science.290.5497.1717).

Permission to republish or repurpose articles or portions of articles can be obtained by following the guidelines [here](https://www.science.org/about/rights-and-permissions).

The following resources related to this article are available online at www.sciencemag.org (this information is current as of October 8, 2012):

Updated information and services, including high-resolution figures, can be found in the online version of this article at:
[http://www.sciencemag.org/content/290/5497/1717.full.html](http://www.sciencemag.org/content/290/5497/1717.full.html)

This article cites 81 articles, 43 of which can be accessed free:
[http://www.sciencemag.org/content/290/5497/1717.full.html#ref-list-1](http://www.sciencemag.org/content/290/5497/1717.full.html#ref-list-1)

This article has been cited by 742 article(s) on the ISI Web of Science

This article has been cited by 100 articles hosted by HighWire Press; see:
[http://www.sciencemag.org/content/290/5497/1717.full.html#related-urls](http://www.sciencemag.org/content/290/5497/1717.full.html#related-urls)

This article appears in the following subject collections:
Cell Biology
[http://www.sciencemag.org/cgi/collection/cell_biol](http://www.sciencemag.org/cgi/collection/cell_biol)

Science (print ISSN 0036-8075; online ISSN 1095-9203) is published weekly, except the last week in December, by the American Association for the Advancement of Science, 1200 New York Avenue NW, Washington, DC 20005. Copyright 2000 by the American Association for the Advancement of Science; all rights reserved. The title *Science* is a registered trademark of AAAS.

SCIENCE'S COMPASS • REVIEW

**REVIEW: CELL BIOLOGY**

# Autophagy as a Regulated Pathway of Cellular Degradation

Daniel J. Klionsky${}^{1*}$ and Scott D. Emr${}^{2}$

---

Macroautophagy is a dynamic process involving the rearrangement of subcellular membranes to sequester cytoplasm and organelles for delivery to the lysosome or vacuole where the sequestered cargo is degraded and recycled. This process takes place in all eukaryotic cells. It is highly regulated through the action of various kinases, phosphatases, and guanosine triphosphatases (GTPases). The core protein machinery that is necessary to drive formation and consumption of intermediates in the macroautophagy pathway includes a ubiquitin-like protein conjugation system and a protein complex that directs membrane docking and fusion at the lysosome or vacuole. Macroautophagy plays an important role in developmental processes, human disease, and cellular response to nutrient deprivation.

Normal cell growth and development requires a well-controlled balance between protein synthesis and organelle biogenesis versus protein degradation and organelle turnover. The major pathways for degradation of cellular constituents are autophagy and cytosolic turnover by the proteasome. These degradative pathways are particularly important during development and under certain environmental stress conditions. For example, cellular death and resorption are critical during processes that involve extensive cellular remodeling such as insect metamorphosis, postpartum luteal cell regression, differentiation, and aging (1, 2), as well as in preventing various disease states including cardiomyopathy and some types of cancer. In some cases, degradation and turnover of the entire cell occurs as part of programmed cell death. At other times, turnover occurs on a subcellular scale; under starvation conditions, cells need to scavenge nonessential proteins and organelles and to recycle the components for reuse in the cytosol.

In eukaryotic cells, the lysosome or vacuole is a major degradative organelle. This compartment contains a range of hydrolases that are able to degrade essentially any subcellular constituent (proteins, lipids, nucleic acids, and carbohydrates). In addition, regulated turnover of organelles is confined to the lysosome. Cyttoplasmic components are degraded within the lysosome by microautophagy, chaperone-mediated autophagy, and macroautophagy (3–5). In mammalian cells, microautophagy has not been well characterized, and chaperone-mediated autophagy is a secondary response that temporally follows macroautophagy. In this review, we will limit our discussion to macroautophagy, the major inducible pathway for general turnover of cytoplasmic components. The process of macroautophagy is seen in all nucleate cell types that have been analyzed. The morphology of the process is essentially the same in yeast, plants, and animal cells. In addition, the recent identification of genes encoding components of the autophagic machinery in several organisms indicates that the autophagic process is highly conserved.

## Morphology, Biochemistry, and Genetics

Macroautophagy is a dynamic process in which subcellular membranes undergo dramatic morphological changes [for reviews, see (5, 6)]. In this process, portions of cytoplasm are sequestered within double-membrane vesicles known as autophagic vacuoles in mammalian cells or autophagosomes in yeast (Fig. 1). For simplicity, we will use the term autophagosome. Macroautophagy has been extensively characterized morphologically in both yeast and mammalian systems. Genetic screens have been used in yeast to identify the genes that are involved in the autophagic process (Table 1). Screens for yeast mutants that were starvation-sensitive or defective in the degradation of specific cytosolic proteins (7, 8) produced *apg* and *aut* mutants, which overlap with mutants in the cytoplasm to vacuole targeting (Cvt) pathway, that were isolated because of defects in proteolytic processing of the resident vacuolar hydrolase aminopeptidase I (9–11). The overlap between the *apg*, *aut*, and *cvt* mutants was surprising, because the respective pathways, autophagy and Cvt, operate under distinct conditions. Autophagy is primarily used for degradation and is induced under starvation conditions. In contrast, the Cvt pathway is biosynthetic and operates under nutrient-rich conditions. However, biochemical and morphological studies show that in both pathways, the basic mechanism of cytoplasm-to-vacuole transport involves sequestration by a cytosolic double-membrane vesicle (12, 13). The major distinction between the two pathways appears to be the regulation of the size of the autophagosome. The process of macroautophagy can be broken down into at least four discrete steps: induction, formation of the autophagosome, autophagosome docking and fusion with the lysosome or vacuole, and autophagic body breakdown. The molecular details of these steps have begun to be elucidated.

### Induction of macroautophagy

Autophagic degradation is both developmentally and nutritionally regulated. Nonspecific autophagy is inhibited under nutrient-rich conditions and is induced by starvation. In mammalian cells, phosphorylation of ribosomal protein S6 strongly correlates with inhibition of macroautophagy (14). The activity of p70S6 kinase is regulated by mTor kinase (15, 16). Inhibition of phosphorylation resulting from inactivation of mTor by treatment with rapamycin induces autophagy even under nutrient-rich conditions; however, the details of the regulatory mechanism are not understood. As in mammalian cells, autophagy can be induced in yeast by rapamycin-dependent inhibition of Tor2 (17). In yeast, Tor2 phosphorylates Tap42 causing it to interact with protein phosphatase 2A (PP2A), resulting in a decrease in PP2A activity (18). Inhibition of Tor2 results in activation of PP2A and the induction of autophagy. In mammalian cells, inhibition of PP2A by okadaic acid has a strong inhibitory effect on autophagy (19). Thus, changes in localization and/or activity of PP2A may be one mechanism that controls autophagy, although other phosphatases are also involved in its regulation (20).

Although Tor2 regulates the expression of many genes, the majority of *APG* genes are constitutively expressed. In part, this is owing to the overlap between autophagy and the biosynthetic Cvt pathway. Accordingly, it appears that Tor2 may regulate the transition from the Cvt pathway to autophagy during the response

---

${}^{1}$Department of Biology, University of Michigan, 830 North University, Ann Arbor, MI 48109–1048, USA.  
${}^{2}$Division of Cellular and Molecular Medicine, University of California, San Diego, School of Medicine, and Howard Hughes Medical Institute, La Jolla, CA 92093–0668, USA.  

*To whom correspondence should be addressed. E-mail: klionsky@umich.edu*

www.sciencemag.org SCIENCE VOL 290 1 DECEMBER 2000 1717

to starvation. One of the downstream effectors of the yeast Tor2 kinase may be the Apg13 protein (Fig. 2). Apg13 is part of a dynamic protein complex, which includes the serine and threonine kinase Apg1 (20, 21). Under nutrient-rich conditions, Apg13 is highly phosphorylated and maintains a weak interaction with the Apg1 kinase. Upon inhibition of the Tor2 kinase, and presumably activation of a phosphatase, Apg13 is partially dephosphorylated and associates more tightly with Apg1, stimulating its kinase activity (Fig. 2). Thus, the Apg1 kinase appears to act downstream of Apg13; however, its target or targets have not been identified. The Apg1 kinase interacts with at least two other proteins, Apg17 and Cvt9. These two proteins have specific functions in either the autophagic or Cvt pathways, respectively (20). This finding fits well with the predicted role of the Apg13-Apg1 complex in controlling transition between biosynthetic and degradative pathways. The identification of *APG1* homologs in *Caenorhabditis elegans* (22) and mice (23) suggests that the Apg1 kinase is required for autophagy in higher eukaryotes.

Formation of the autophagosome. After induction of autophagy, a double-membrane vesicle begins to form in the cytosol, resulting in the sequestration of cytoplasmic components (Fig. 1). The origin of the sequestering membrane is not known, but for mammalian cells, it is generally thought to be the endoplasmic reticulum (24). However, an alternative compartment termed the phagophore may represent the donor membrane (25). Sequestration is highly regulated and is under the control of GTPases, phosphatidylinositol kinases, and various phosphatases. Increasing the levels of the class III phosphoinositide 3-kinase product phosphatidylinositol 3-phosphate in human colon cancer cells stimulates macroautophagy (26). In contrast, increases in class I phosphoinositide 3-kinase products, phosphatidylinositol 3,4-bisphosphate and phosphatidylinositol 3,4,5-trisphosphate, inhibit autophagy (26). In addition, the guanosine diphosphate (GDP)-bound form of the heterotrimeric G protein subunit Ga<sub>i3</sub> is an activating factor for autophagy, whereas the GTP-bound form is inhibitory (27). Regulation by protein phosphatases and kinases is complex and not yet resolved (28).

Molecular genetic studies in yeast have identified some of the components required for autophagosome formation. The sequestration process involves a protein conjugation system (29) (Fig. 2). Apg12 is covalently attached to Apg5 through a COOH-terminal glycine and an internal lysine residue, respectively. This process requires the action of the Apg7 protein, which is homologous to the E1 family of ubiquitin-activating enzymes (30, 31), and Apg10, which functions as a protein-conjugating enzyme (32). A third protein, Apg16, binds to the conjugated Apg5 and dimerizes to link a pair of Apg12-Apg5 conjugates (33). This conjugation event is required for formation or completion of the sequestering vesicle (34). Human homologs of Apg5 and Apg12 have been identified and undergo a similar covalent linkage, indicating that this conjugation system is conserved in mammalian cells (35).

The level of most proteins involved in autophagy does not change during a shift from nutrient-rich to starvation conditions. In contrast, the *AUT7* gene is highly up-regulated under starvation conditions or after rapamycin treatment (36, 37) (Fig. 2). Aut7 binds the forming autophagosome and becomes sequestered inside the vesicle along with the cytoplasmic cargo. The localization and upregulation of Aut7 suggests that it functions at least in part as a structural component during autophagosome formation.

Autophagosome docking and fusion. In mammalian cells, delivery to and fusion of the completed autophagosome with the lysosome (Fig. 1) depends on microtubules and maintenance of proper acidification. In yeast, components of the SNARE machinery, including the vacuolar syntaxin homolog Vam3p (38) and the SNAP-25 homolog Vam7p (39), are required for autophagosome docking and/or fusion with the vacuole (Table 1). In addition, the regulatory Rab GTPase, Ypt7 (30), and the class C Vps protein complex (13, 40–42) also function at this step to ensure the accurate and efficient fusion of the autophagosome with the vacuole.

Autophagic body breakdown. Although the lysosome or vacuole is the terminal destination for endocytic and certain biosynthetic trafficking pathways, little is known about what happens to membranes after their delivery to this organelle. After fusion of the autophagosome with the vacuole, the outer membrane of the autophagosome is incorporated into the limiting membrane of the vacuole (Fig. 1). It is not known how this large patch of new membrane avoids degradation by the lipases and other hydrolases present within this organelle. The vacuole membrane itself contains a variety of glycoproteins that are presumed to play a role in protecting this membrane from degradation. Fusion causes the release of the single-membrane bound inner vesicle of the autophagosome, the autophagic body (Fig. 1), into the vacuole lumen. Because of the degradative nature of the lysosome or vacuole, the autophagic body is broken down. Efficient degradation is dependent on proteinase B, lumenal acidification, and the Cvt17 protein, a candidate lipase that may degrade the autophagic body (13, 43–45).

Specific Autophagic Pathways

Biosynthetic transport and organelle turnover. Autophagy is a ubiquitous process that occurs in plant, animal, and fungal cells. Although autophagy is generally considered to be nonselective, the protein components required for autophagic uptake are also used in specific transport events. For example, as noted above, autophagy in yeast overlaps with the biosynthetic Cvt pathway that transports aminopeptidase I (API) to the vacuole. API is synthesized as a precursor in the cytosol and rapidly oligomerizes into a dodecamer (46). The dodecameric precursor API assembles into a larger complex composed of multiple dodecamers, termed a Cvt complex. The Cvt complex is then enwrapped by a double-membrane that forms a cytosolic vesicle. This Cvt vesicle is smaller than an autophagosome and appears to exclude cytosol

![Figure 1](#fig1)

Fig. 1. Schematic model of macroautophagy in yeast. A signal transduction event regulated by the Tor kinase leads to the following: (1) The induction of autophagy. (2) Membrane from an unknown source sequesters cytosol and/or organelles (a mitochondrion is depicted) resulting in the formation of a double-membrane vesicle (300 to 900 nm) termed an autophagosome. (3) On completion, the autophagosome docks with the lysosome or vacuole. Fusion of the autophagosome outer membrane with the vacuole releases the inner vesicle into the vacuole lumen. The inner vesicle is termed an autophagic body. (4) Breakdown within the vacuole allows recycling of the degraded autophagic body and its hydrolyzed cargo (amino acids, fatty acids, sugars, and nucleotides). The morphology of macroautophagy in mammalian cells is similar to that shown; however, in mammalian cells autophagy can be induced by environmental cues other than starvation.

(12, 13). After fusion with the vacuole, the inner vesicle is degraded in the lumen, allowing release and maturation of precursor API.

In addition to the biosynthetic transport of precursor API, regulated autophagy functions in the selective sorting and turnover of certain cytoplasmic organelles. When methylotrophic yeasts or mammalian cells are grown on methanol or fatty acids, respectively, peroxisomes proliferate to allow maximal use of the available nutrients. Adaptation to preferred carbon sources such as glucose or ethanol result in the specific degradation of peroxisomes through an autophagic mechanism termed pexophagy (5). In some yeasts this degradation has been shown to occur by both micropexophagy, occurring at the vacuole surface, and macropexophagy, where the initial sequestration step takes place away from the vacuole (47). The proteins required for macroautophagy are also required for pexophagy (48, 49).

In general, maintenance of organelles is energetically costly. Organelles like the peroxisome are specifically degraded when they are no longer needed. In mitochondria, certain types of stress can lead to a loss of integrity of the inner and outer membranes resulting in depolarization, a process termed the "mitochondrial permeability transition" (50). It has been proposed that conversion of the mitochondrial permeability transition pore to an open state leads to mitochondrial degradation through macroautophagy and subsequently through apoptosis and necrosis (51). Macroautophagy then can protect cells from oxidative damage that could result from a loss of mitochondrial integrity without triggering apoptosis. However, lysosomal degradation of mitochondria can lead to clinical problems in patients lacking one or more lysosomal hydrolases. The lysosomal accumulation of subunit 9 of the mitochondrial ATPase due to inefficient degradation is associated with neuronal ceroid lipofuscinosis (Batten disease), a neurodegenerative disorder in children and young adults (52). In yeast, mitochondria have been detected within autophagosomes (43), providing additional evidence for mitochondrial degradation by macroautophagy.

Autophagy in development and disease. Macroautophagy plays a critical role in the homeostatic process of recycling proteins and organelles. However, autophagy has also been linked to developmental and pathological conditions. For example, intersegmental muscles are required in the larval and pupal stages of the moth life cycle and during emergence of the adult moth. These muscle cells are not needed in the adult and undergo cell death shortly after emergence. In this case, the characteristics of cell death are largely distinct from those associated with apoptosis and are proposed to occur through autophagy (53). Programmed cell death in developing neurons clearly involves apoptosis (54). However, cellular death in neurons may also involve autophagy (55, 56). Similarly,

the loss of Rohon-Beard neurons from the spinal cord of *Xenopus* tadpoles requires the activity of lysosomal hydrolases and is thought to occur through autophagy-related cell death (57).

There are many lines of evidence that connect macroautophagy to human disease. For example, elevated levels of autophagy are associated with neurodegenerative diseases such as Parkinson's (58). The mechanism by which neuronal degeneration occurs is not well understood and clearly requires further study to allow the development of effective treatments. In contrast to the situation with Parkinson's disease, lowered levels of autophagy are problematic in other diseases. A direct link to autophagy and heart disease was discovered by the finding that a deficiency of LAMP-2, a transmembrane protein in the late endosome and lysosome, is associated with the cardiomyopathic Danon's disease (59), and LAMP-2-deficient mice are defective in the maturation of autophagosomes (60). Similarly, reduced levels of autophagy are seen in some forms of cancer (61). A key role for autophagy in controlling the unregulated cell growth that is associated with tumor development has been suggested through the study of a tumor suppressor gene, *beclin 1* (62). Beclin 1 is the human homolog of the yeast autophagy gene *APG6*. Beclin 1 interacts with the antiapoptotic protein Bcl-2, which prevents the Bax-dependent release of mitochondrial cytochrome c. Decreased levels of the Beclin 1 protein have been correlated with the development or progression of breast tumors. In multicellular organisms, autophagy has also been linked with programmed cell death. Autophagy is specifically associated with type II (nonapoptotic) programmed cell death (53, 63). However, there is an indication that early stages of autophagy may play a role in type I (apoptotic) programmed cell

A Regulation of autophagy

![Diagram A](#)

B Autophagosome formation

![Diagram B](#)

C Size regulation of the autophagosome

![Diagram C](#)

Fig. 2. Molecular genetics of macroautophagy in yeast. (A) The Tor kinase exerts a negative regulatory effect on autophagy when cells are growing under nutrient-rich conditions. When starvation occurs, the Tor kinase is inactivated, and the negative regulation is relieved resulting in induction of autophagy. Most of the proteins required for autophagy are constitutively expressed and are used for biosynthetic import through the cytoplasm to vacuole targeting pathway under these conditions. The downstream effectors of Tor are likely to include phosphatases and kinases that modulate the phosphorylation state of Apg13. An inductive signal such as carbon or nitrogen starvation inactivates Tor and results in partial dephosphorylation of Apg13. This form of Apg13 associates more tightly with the Apg1 kinase and stimulates its activity. The function of Apg1 kinase is required for autophagosome formation. (B) The Apg7 (E1-like) and Apg10 proteins form thioester intermediates through a COOH-terminal glycine of Apg12. Apg12 is ultimately conjugated to Apg5 through an internal lysine residue in Apg5 in a process that is similar to ubiquitination. Apg16 binds the conjugated Apg5 protein noncovalently and dimerizes to form a complex that is required for formation and completion of the autophagosome. (C) Under nutrient-rich conditions, the Tor kinase negatively regulates the expression of the *AUT7* gene resulting in basal levels of Aut7 synthesis. Under these conditions the Cvt pathway is operative and 150-nm Cvt vesicles are formed. Inhibition of Tor by transduction of an environmental signal or after treatment with rapamycin allows the activation of a presumed transcriptional activator protein that increases expression of *AUT7*. The resulting increase in Aut7 levels allows an expansion in the size of the autophagosome from 150 nm to a range of 300 to 900 nm.

death (64). Along these lines, a human apoptosis-specific protein was shown to be homologous to the APG5 gene product (65). Inhibition of autophagy may be therapeutic in certain neurodegenerative diseases, although its selective activation in specific cells could provide an avenue for cancer treatment.

An interesting connection between apoptosis and autophagy also has been seen in the coordinated regulation of Akt (protein kinase B) and p70S6 kinase. The activity of p70S6 kinase is controlled through mTor but also through the action of the phosphoinositide-dependent protein kinase-1, PDK1 (66, 67). PDK1 is a multifunctional effector that regulates various kinases (68, 69). Phosphorylation of p70S6 kinase by mTor, or presumably PDK1, prevents autophagy. Class I phosphatidylinositol 3-kinase (PI 3-kinase) activity allows the membrane recruitment of Akt through its pleckstrin homology domain. Phosphorylation by PDK1 activates Akt and inhibits apoptosis (70). Thus, class I PI 3-kinase products may block both apoptosis and autophagy through PDK1. Along these lines,

some of the same signals that trigger apoptosis may induce autophagy (55). Once it is activated, autophagy may be able to cause cell death even in the presence of apoptosis inhibitors. These findings support the idea that autophagy represents a second mechanism for programmed cell death. In fact, autophagy-related programmed cell death may have evolved before apoptosis (53).

### Conclusions

The capacity for limited turnover of cytoplasmic contents by autophagy or for the programmed death of specific cells plays an important role in development. However, the capacity to degrade cells and cellular contents necessitates strict regulatory safeguards to prevent indiscriminate destruction of essential components. Accordingly, macroautophagy is regulated through the action of kinases (e.g., Tor and Apg1), phosphatases (e.g., PP2A), and GTPases (Go<sub>i3</sub> and Ypt7) that dictate the conditions under which it operates and the targets of the sequestration process.

Autophagy involves dynamic rearrangements of cellular membranes. This is not surprising as this process can result in the engulfment of entire organelles. Some aspects of autophagy are similar to those of other subcellular trafficking events, for example, the targeting, docking, and fusion of the autophagosome requires components such as Rabs and SNARES that are used in both the secretory and endocytic transport systems. However, other components of the autophagy system are specific (see Table 1). The origin of the sequestering membrane is not known, nor is it understood how this membrane can form an enwrapping vesicle in the cytosol. Formation of the autophagosome is topologically distinct from that of a transit vesicle that buds off a preexisting organelle and is likely to require a large number of gene products (Table 1), including those of a novel ubiquitin-like protein conjugation system (Fig. 2). The autophagic machinery is used for a range of processes that are carried out under different conditions and that display varying specificities.

The autophagic machinery is highly conserved in organisms as diverse as plants, animals, and yeast. Recent studies have demonstrated a role for autophagy in both promoting and preventing human diseases. In the last few years, tremendous advances have been made in identifying the molecular components required for macroautophagy through genetic studies in yeast. Combining these analyses with the pharmacological and biochemical information that is available from work in mammalian systems has begun to provide a detailed understanding of the mechanisms that underlie each stage of the autophagic process. Furthermore, workers in both plant and animal systems are exploiting the genomic data now available to identify homologs of the yeast APG genes for directed studies in these multicellular systems. Such studies should allow researchers to address tissue- and development-specific roles for autophagic degradation. Many questions still remain, including the following: What is the mechanism by which cells sense the starvation signal required to induce autophagy? What is the origin of the autophagosomal membrane? How does the autophagic machinery drive the deformation of membranes? How are specific substrates recognized as cargo for sequestration? However, with the progress that has been made in the last few years, it is likely that these and other important problems in the field of autophagy will be solved in the near future.

---

**Table 1. Genes required for the autophagy and Cvt pathways.**

| Gene       | Mammalian homolog | Characteristics                                                                 | References     |
|------------|--------------------|----------------------------------------------------------------------------------|----------------|
| **Induction of autophagy and formation of autophagosome**                                                                                                      |
| Kinase signaling system                                                                                                                                     |
| Tor2       | mTor               | Rapamycin-sensitive protein kinase                                             | (15)           |
| APG1       | ULK1               | SER or THR protein kinase, in complex with Apg13, Apg17, Cvt9                      | (23, 44)       |
| APG6       | BECN1              | 63-kD peripheral membrane protein, binds Apg14                                   | (62, 71)       |
| APG9       |                    | Integral membrane protein                                                      | (72)           |
| APG13      |                    | Associates with and activates Apg1 kinase                                       | (20)           |
| APG14      |                    | Peripheral membrane protein, interacts with Apg6                                | (71)           |
| APG17      |                    | Interacts with Apg1 kinase, specific for autophagy                              | (20)           |
| CVT9       |                    | Interacts with Apg1 kinase, specific for Cvt pathway                            | (20)           |
| VAC8       |                    | Armadillo repeat protein, interacts with Apg13                                  | (21, 73)       |
| **APG protein conjugation system**                                                                                                                           |
| APG5       | hAPG5              | 34-kD protein that forms conjugate with Apg12                                   | (29)           |
| APG7       | HsGSA7             | E1-like ubiquitin activating enzyme                                             | (30, 31)       |
| APG10      |                    | Protein-conjugating enzyme                                                     | (32)           |
| APG12      | hAPG12             | Ubiquitin type modifier                                                        | (29)           |
| APG16      |                    | Coiled-coil protein that binds Apg5                                            | (33)           |
| **Size regulation of the autophagosome**                                                                                                                     |
| AUT2       |                    | Cysteine protease processes Aut7                                               | (74)           |
| AUT7       | GATE-16            | Up-regulated by starvation, regulates autophagosome size                         | (36, 37, 75–77)|
| MAP LC3    |                    |                                                                              |                |
| **Docking and fusion**                                                                                                                                      |
| VAM3       | Syntaxin 7         | Vacuolar tSNARE                                                               | (38, 78, 79)   |
| VAM7       |                    | SNAP-25 family member, forms part of SNARE complex                             | (39)           |
| VPS genes  |                    | Class C Vps complex composed of Vps11, 16, 18, 33, 39, and 41                     | (41, 42)       |
| YPT7       | Rab7               | Rab GTPase                                                                    | (80)           |
| **Breakdown**                                                                                                                                                |
| CVT17      |                    | Lipase homolog                                                                | (13, 45)       |
| PRB1       |                    | Vacuolar proteinase                                                           | (43)           |
| VMA genes  | V-ATPase           | Vacuolar-type ATPase acidifies lumen                                           | (81, 82)       |

---

**References and Notes**

1. G. E. Mortimore, G. Miotto, R. Venerando, M. Kadowaki, *Subcell. Biochem.* **27**, 93 (1996).
2. S. Vittorini et al., *J. Gerontol. A Biol. Sci. Med. Sci.* **54**, B318 (1999).
3. W. A. Dunn Jr., *Trends Cell Biol.* **4**, 139 (1994).
4. A. M. Cuervo, J. F. Dice, *J. Mol. Med.* **76**, 6 (1998).
5. J. Kim, D. J. Klionsky, *Annu. Rev. Biochem.* **69**, 303 (2000).
6. D. J. Klionsky, Y. Ohsumi, *Annu. Rev. Cell Dev. Biol.* **15**, 1 (1999).

SCIENCE'S COMPASS

7. M. Thumm et al., FEBS Lett. 349, 275 (1994).
8. M. Tsukada, Y. Ohsumi, FEBS Lett. 333, 169 (1993).
9. T. M. Harding, K. A. Morano, S. V. Scott, D. J. Klionsky, J. Cell Biol. 131, 591 (1995).
10. T. M. Harding, A. Hefner-Gravink, M. Thumm, D. J. Klionsky, J. Biol. Chem. 271, 17621 (1996).
11. S. V. Scott et al., Proc. Natl. Acad. Sci. U.S.A. 93, 12304 (1996).
12. M. Baba, M. Osumi, S. V. Scott, D. J. Klionsky, Y. Ohsumi, J. Cell Biol. 139, 1687 (1997).
13. S. V. Scott, M. Baba, Y. Ohsumi, D. J. Klionsky, J. Cell Biol. 138, 37 (1997).
14. E. F. Blommaart, J. J. Luiken, P. J. Blommaart, G. M. van Woerkom, A. J. Meijer, J. Biol. Chem. 270, 2320 (1995).
15. G. Thomas, M. N. Hall, Curr. Opin. Cell Biol. 9, 782 (1997).
16. E. J. Brown et al., Nature 377, 441 (1995).
17. T. Noda, Y. Ohsumi, J. Biol. Chem. 273, 3963 (1998).
18. T. Beck, M. N. Hall, Nature 402, 689 (1999).
19. I. Holen, P. B. Gordon, P. O. Seglen, Biochem. J. 284, 633 (1992).
20. Y. Kamada et al., J. Cell Biol. 150, 1507 (2000).
21. S. Scott et al., J. Biol. Chem. 275, 25840 (2000).
22. K. Ogura et al., Genes Dev. 8, 2389 (1994).
23. J. Yan et al., Biochem. Biophys. Res. Commun. 246, 222 (1998).
24. W. A. Dunn Jr., J. Cell Biol. 110, 1923 (1990).
25. P. E. Strømhaug, T. O. Berg, M. Fengsrud, P. O. Seglen, Biochem. J. 335, 217 (1998).
26. A. Petiot, E. Ogier-Denis, E. F. Blommaart, A. J. Meijer, P. Codogno, J. Biol. Chem. 275, 992 (2000).
27. E. Ogier-Denis, J.-J. Houri, C. Bauvy, P. Codogno, J. Biol. Chem. 271, 28593 (1996).
28. P. Codogno, E. Ogier-Denis, J.-J. Houri, Cell. Signal. 9, 125 (1997).
29. N. Mizushima et al., Nature 395, 395 (1998).
30. J. Kim, V. M. Dalton, K. P. Eggerton, S. V. Scott, D. J. Klionsky, Mol. Biol. Cell 10, 1337 (1999).
31. I. Tanida et al., Mol. Biol. Cell 10, 1367 (1999).
32. T. Shintani et al., EMBO J. 18, 5234 (1999).
33. N. Mizushima, T. Noda, Y. Ohsumi, EMBO J. 18, 3888 (1999).
34. M. D. George et al., Mol. Biol. Cell 11, 969 (2000).
35. N. Mizushima, H. Sugita, T. Yoshimori, Y. Ohsumi, J. Biol. Chem. 273, 33889 (1998).
36. W. P. Huang, S. V. Scott, J. Kim, D. J. Klionsky, J. Biol. Chem. 275, 5845 (2000).
37. T. Kirisako et al., J. Cell Biol. 147, 435 (1999).
38. T. Darsow, S. E. Rieder, S. D. Emr, J. Cell Biol. 138, 517 (1997).
39. T. K. Sato, T. Darsow, S. D. Emr, Mol. Cell. Biol. 18, 5308 (1998).
40. S. E. Rieder, S. D. Emr, Mol. Biol. Cell 8, 2307 (1997).
41. T. K. Sato, P. Rehling, M. R. Peterson, S. D. Emr, Mol. Cell 6, 661 (2000).
42. D. F. Seals, G. Eitzen, N. Margolis, W. T. Wickner, A. Price, Proc. Natl. Acad. Sci. U.S.A. 97, 9402 (2000).
43. K. Takeshige, M. Baba, S. Tsuboi, T. Noda, Y. Ohsumi, J. Cell Biol. 119, 301 (1992).
44. A. Matsuura, M. Tsukada, Y. Wada, Y. Ohsumi, Gene 192, 245 (1997).
45. S. A. Teter, D. J. Klionsky, unpublished data.
46. J. Kim, S. V. Scott, M. Oda, D. J. Klionsky, J. Cell Biol. 137, 609 (1997).
47. D. L. Tuttle, W. A. Dunn Jr., J. Cell Sci. 108, 25 (1995).
48. W. Yuan, P. E. Stromhaug, W. A. Dunn Jr., Mol. Biol. Cell 10, 1353 (1999).
49. M. U. Hutchins, M. Veenhuis, D. J. Klionsky, J. Cell Sci. 112, 4079 (1999).
50. M. Crompton, Biochem. J. 341, 233 (1999).
51. J. J. Lemasters et al., Biochim. Biophys. Acta 1366, 177 (1998).
52. G. Dawson, S. Cho, J. Neurosci. Res. 60, 133 (2000).
53. L. M. Schwartz, S. W. Smith, M. E. Jones, B. A. Osborne, Proc. Natl. Acad. Sci. U.S.A. 90, 980 (1993).
54. H. Dudek et al., Science 275, 661 (1997).
55. L. Xue, G. C. Fletcher, A. M. Tolokovsky, Mol. Cell. Neurosci. 14, 180 (1999).
56. K. Isahara et al., Neuroscience 91, 233 (1999).
57. J. E. Lamborghini, J. Comp. Neurol. 264, 47 (1987).
58. P. Anglade et al., Histol. Histopathol. 12, 25 (1997).
59. I. Nishino et al., Nature 406, 906 (2000).
60. Y. Tanaka et al., Nature 406, 902 (2000).
61. R. Schulte-Hermann et al., Toxicol. Pathol. 25, 89 (1997).
62. X. H. Liang et al., Nature 402, 672 (1999).
63. W. Bursch et al., J. Cell Sci. 113, 1189 (2000).
64. L. Jia et al., Br. J. Haematol. 98, 673 (1997).
65. E. M. Hammond et al., FEBS Lett. 425, 391 (1998).
66. D. R. Alessi, M. T. Kozlowski, Q. P. Weng, N. Morrice, J. Avruch, Curr. Biol. 8, 69 (1998).
67. N. Pullen et al., Science 279, 707 (1998).
68. B. Vanhaesebroeck, D. R. Alessi, Biochem. J. 346, 561 (2000).
69. P. J. Coffer, J. Jin, J. R. Woodgett, Biochem. J. 335, 1 (1998).
70. B. A. Hemmings, Science 275, 628 (1997).
71. S. Kametaka, T. Okano, M. Ohsumi, Y. Ohsumi, J. Biol. Chem. 273, 22284 (1998).
72. T. Noda et al., J. Cell Biol. 148, 465 (2000).
73. Y. X. Wang, N. L. Catlett, L. S. Weisman, J. Cell Biol. 140, 1063 (1998).
74. T. Kirisako et al., J. Cell Biol. 151, 263 (2000).
75. Y. Sagiv, A. Legesse-Miller, A. Porat, Z. Elazar, EMBO J. 19, 1494 (2000).
76. T. Lang et al., EMBO J. 17, 3597 (1998).
77. Y. Kabeya et al., EMBO J. 19, 5720 (2000).
78. Y. Wada, N. Nakamura, Y. Ohsumi, A. Hirata, J. Cell Sci. 110, 1299 (1997).
79. B. M. Mullock et al., Mol. Biol. Cell 11, 3137 (2000).
80. A. Haas, D. Scheglmann, T. Lazar, D. Gallwitz, W. Wickner, EMBO J. 14, 5258 (1995).
81. L. A. Graham, B. Powell, T. H. Stevens, J. Exp. Biol. 203, 61 (2000).
82. N. Nakamura, A. Matsuura, Y. Wada, Y. Ohsumi, J. Biochem. 121, 338 (1997).
83. The authors would like to thank J. Kim, T. Sato, A. Wurmser, and S. Scott for helpful comments. D.J.K. is supported by Public Health Service grant GM53396 from NIH and S.D.E. is supported by NIH grant CA58689 and is an Investigator with the Howard Hughes Medical Institute.

---

REVIEW: CELL BIOLOGY

How Cells Handle Cholesterol

Kai Simons${}^{1*}$ and Elina Ikonen${}^{2}$

Cholesterol plays an indispensable role in regulating the properties of cell membranes in mammalian cells. Recent advances suggest that cholesterol exerts many of its actions mainly by maintaining sphingolipid rafts in a functional state. How rafts contribute to cholesterol metabolism and transport in the cell is still an open issue. It has long been known that cellular cholesterol levels are precisely controlled by biosynthesis, efflux from cells, and influx of lipoprotein cholesterol into cells. The regulation of cholesterol homeostasis is now receiving a new focus, and this changed perspective may throw light on diseases caused by cholesterol excess, the prime example being atherosclerosis.

S terols are ubiquitous components of cell membranes in eukaryotes. Mammalian cells produce their own cholesterol and receive cholesterol by uptake from lipopro-

${}^{1}$Max Planck Institute for Molecular Cell Biology and Genetics, Pfotenhauerstrasse, 01307 Dresden, Germany. ${}^{2}$Department of Biochemistry, National Public Health Institute, Mannerheimintie 166, 00300 Helsinki, Finland.

*To whom correspondence should be addressed. E-mail: kai.simons@embl-heidelberg.de

teins. Cells also continuously lose cholesterol to the outside circulation. Regulation of synthesis, influx and efflux keeps cellular cholesterol levels precisely controlled. But why cholesterol homeostasis is under such stringent control has not been obvious from our understanding of cholesterol function. Cholesterol was simply thought to rigidify the fluid membrane, thus reducing passive permeability and increasing the mechanical durability of the lipid bilayer. Cholesterol was also postulated to facilitate post-Golgi protein sorting by regulating the thickness of the lipid bilayer ($I$). These functions have now been expanded by the key role that cholesterol plays in lipid raft assembly and function. Lipid rafts function to segregate and concentrate membrane proteins (2). These dynamic assemblies are involved in sorting and distributing lipids and proteins to the cell surface (3), where they play an important role in signal transduction and in generating cell surface polarity (4). For these functions, it is crucial that not only the total cellular level of cholesterol but also its distribution between membranes and within a given membrane are tightly regulated. Here, we will review the movements of cholesterol within the cell and the regulation of cellular cholesterol from this new perspective.

What Are Lipid Rafts?

Lipid rafts are constituted of cholesterol and sphingolipids (sphingomyelin and glycosphingolipids) in the exoplasmic leaflet of the bilayer [for reviews on rafts, see (2, 3, 5)].
