
# An herbivore elicitor activates the gene for indole emission in maize

Monika Frey$^{*†}$, Cornelia Stettner$^{*}$, Paul W. Paré$^{‡}$, Eric A. Schmelz$^{§}$, James H. Tumlinson$^{§}$, and Alfons Gierl$^{*}$

$^{*}$Lehrstuhl für Genetik, Technische Universität München, Lichtenbergstrasse 4, 85747 Garching, Germany; $^{†}$Texas Tech University, Department of Chemistry and Biochemistry, 1 Chemistry Road, Lubbock, TX 79409; and $^{§}$United States Department of Agriculture, Agricultural Research Service, Center for Medical Agricultural and Veterinary Entomology, 1700 SW 23rd Drive, Gainesville, FL 32608

Contributed by James H. Tumlinson III, October 20, 2000

Maize and a variety of other plant species release volatile compounds in response to herbivore attack that serve as chemical cues to signal natural enemies of the feeding herbivore. $N$-(17-hydroxy-linolenoyl)-L-glutamine is an elicitor component that has been isolated and chemically characterized from the regurgitant of the herbivore-pest beet armyworm. This fatty acid derivative, referred to as volicitin, triggers the synthesis and release of volatile components, including terpenoids and indole in maize. Here we report on a previously unidentified enzyme, indole-3-glycerol phosphate lyase (IGL), that catalyzes the formation of free indole and is selectively activated by volicitin. IGL’s enzymatic properties are similar to BX1, a maize enzyme that serves as the entry point to the secondary defense metabolites DIBOA and DIMBOA. Gene sequence analysis indicates that $Igl$ and $Bx1$ are evolutionarily related to the tryptophan synthase alpha subunit.

Materials and Methods

Plant Source. Maize seeds, *Zea mays*, planted in potting soil were maintained in growth chambers illuminated with metal halide and high-pressure sodium lamps on a 14-h light/10-h dark photoperiod. Incubator temperature fluctuated between 20°C during the dark cycle and 30°C when the lights were on. The maize inbred-line CI31A was the source of genomic clones. Maize line B73 was the source for the Stratagene maize cDNA library (whole plant, 7 leaves stage, 1.5 million primary clones) used for the isolation of the cDNA clones. CI31A and $bx1bx1$ mutant lines (12) were used for the induction experiments. For all experiments, seedlings were harvested 11 days after planting.

Heterologous Expression and IGL Enzyme Assay. The IGL protein was expressed in *Escherichia coli* by inserting the cDNA into a modified pET3a vector (13). The enzyme was purified by a six-nucleotide oligomeric COOH-terminal histidine tag (Qiagen Ni-NTA purification system) and was used for fluorometric enzyme assays (14) as described by Frey *et al.* (10).

Volicitin Incubations. Maize seedlings were cut off above the root and placed in shell vials containing 500 $\mu$l of 50 mM sodium phosphate buffer adjusted to pH 8; seedlings were treated with buffer alone or with buffer containing 300 pmol synthetic volicitin. For RNA analysis, plantlets were incubated for 0, 1, 2, 4, and 8 h, and 3 plantlets were combined for each time point. When the seedlings were dissected, material from nine seedlings was combined. All tissue was frozen in liquid nitrogen and stored at $-70^{\circ} \mathrm{C}$. For volatile collections, seedlings were incubated first for 0, 1.5, 3, 4.5, 6, or 10 h.

Collection and Analysis of Volatiles. Treated seedlings or freshly cut plantlets were placed into individual volatile-collection chambers, and volatiles were collected for 1.5 h as described (9). All treatments were run in triplicate. Volatiles were analyzed by capillary GC (Hewlett-Packard 6890) on a 15-m × 0.25-mm (i.d.) fused silica column with a 0.1-mm-thick bonded methyl-silicone mode for 0.3 min with an injector temperature of $230^{\circ} \mathrm{C}$

Abbreviations: GAP C, glycerol-phosphate dehydrogenase, cytosolic form; IGP, indole-3-glycerol phosphate; TSA, tryptophan synthase alpha subunit; TSB, tryptophan synthase beta subunit; IGL, IGP lyase.

Data deposition: The sequences reported in this paper have been deposited in the GenBank database (accession nos. AF271383 and AF271384).

$^{*}$To whom reprint requests should be addressed at: Lehrstuhl für Genetik, Technische Universität München, Lichtenbergstrasse 4, 85747 Garching, Germany. E-mail: Monika.Frey@bio.tum.de.

The publication costs of this article were defrayed in part by page charge payment. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. §1734 solely to indicate this fact.

Article published online before print: Proc. Natl. Acad. Sci. USA, 10.1073/pnas.260499897. Article and publication date are at www.pnas.org/cgi/doi/10.1073/pnas.260499897

and detector temperature of 250°C. The column was held at 40°C for 0.5 min, increased 12°C·min⁻¹ to 180°C, and then increased 40°C·min⁻¹ to 220°C and held for 2 min. Helium was used as a carrier gas at a linear flow velocity of 35 cm·sec⁻¹. Plant volatile components were identified by comparison of GC retention times with those of authentic standards. Compounds were quantified via flame-ionization detection by using the peak area of nonyl acetate added as an internal standard. Select samples were also analyzed by mass spectroscopy (HP 5973) interfaced to the GC and operated in the electron impact mode. Oven conditions were the same as listed above except that a DB-5MS column was used.

**Transcript Analysis.** RNA was isolated as depicted by Logemann *et al.* (15), and poly(A)⁺ RNA was purified with OligoTex beads (Qiagen, Germany) as directed by the supplier. For Northern blot analysis, the poly(A)⁺ RNA was separated, transferred to Hybond N membrane (Amersham Pharmacia), and hybridized with ³²P-labeled DNA probes as described by Frey *et al.* (16). All membranes were hybridized by using the *Igl* and glycerol-phosphate dehydrogenase, cytosolic form (*GAP C*) cDNAs (17). Poly(A)⁺ RNA equivalent to 1.5 plantlets was applied per lane. To normalize the differences in loading, all values of the *GAP C* signals were divided by the value of the highest *GAP C* hybridization signal. Every *Igl* signal value was then divided by the so-calculated fraction of the respective *GAP C* signal. The Storm 860 PhosphorImager (Amersham Pharmacia) and the program IMAGE QUANT were used for quantification. One hybridization unit is defined as 10³ IMAGE QUANT volumes. For the determination of transcript quantities by the real-time PCR experiment shown in Fig. 2, gene-specific primers were designed: *Igl*, 5′-ATGGCCTCCGCGATCAAGGCTGCATC-3′ and 5′-CT-GAGAGTGAGAGCACACGAGTTCC-3′; *Bx1*, 5′-ATG-GCTTTCGCGCCCCAAAACGTCCCTC-3′ and 5′-CGTG-GACCCCCGCCCTTTTCATCTCG-3′; *TSA*, 5′-CCACA-AAGGCAGCGCTCGGAGGTG-3′ and 5′-GCCTCGCTCCT-CAGCAACGTGCTCT-3′; *GAP C*, 5′-GCTAGCTGCACCA-CAAACTGCCT-3′ and 5′-TAGCCCCACTCGTTGTGTCG-TACCA-3′. The experiment was performed on the LightCycler instrument with the FastStart DNA Master SYBR Green I kit (Roche Molecular Biochemicals). Two independent incubation experiments were analyzed in triplicate. Linearized plasmids containing the different cDNAs were quantified by agarose gel electrophoresis and ethidium-bromide staining, and used as quantification standards. Single-stranded template cDNA was synthesized by using 0.5 μg of poly(A)⁺ RNA and the TaqMan Reverse Transcription Reagents kit (Perkin–Elmer) by priming with random hexamers as directed by the supplier. After quantification, the final PCR products were checked for homogeneity and absence of intron sequences by agarose gel electrophoresis.

**Molecular Biology Methods.** Cloning was performed as described by Frey *et al.* (16). The Stratagen maize cDNA library (10⁶ plaque forming units) was screened, and four cDNA clones were identified for *Igl* and five were identified for *TSAlke*. cDNA and genomic sequences were cloned into the pBluescript KS(+) vector (Stratagene). Sequencing was done by primer walking on an Applied Biosystems PRISM 377 DNA sequencer with oligomeric primers purchased from Eurogentec (Brussels) and GIBCO/BRL. The 5′ end of the transcripts was isolated with the 5′-3′ rapid amplification of cDNA ends kit (Roche Molecular Biochemicals). DMSO at a final concentration of 5% (vol/vol) was included in the first strand cDNA transcription mix.

**Mapping.** Data were generated by Southern blot analysis with the internal genomic 1.2-kb *PstI* fragment of *Igl* as a probe. A unique restriction fragment length polymorphism was revealed for the CM37xT232 recombinant inbred-mapping population (18).

---

Fig. 1. Genes related to TSA in maize. (A) Gene structure of *Igl*, *Bx1*, and *TSAlke*. Exons are represented by filled boxes, and corresponding exons are indicated by dashed lines. A putative TSA gene (*TSAlke*) is located downstream of *Igl* (1,615 bp separate the first polyadenylation site of *Igl* from the transcription start site of *TSAlke*). The first exons of IGL, BX1, and TSAlke are only 45% identical with respect to each other. Homology with bacterial TSA enzymes starts with the amino acids encoded by exon 2 of the three genes. The exon-intron structure of BX1 and IGL is not absolutely conserved. IGL and BX1 have 82% identity in the amino acids encoded by exons 2–6. Comparison to *TSAlke* reveals a 66% identity for IGL and 67% identical amino acids for BX1 in this fraction of the protein. Exons 2 and 3 of *TSAlke* comprise the second exon of *Bx1*. The genomic sequence of *TSAlke* was not fully determined. (B) Phylogenetic tree. The relationship is calculated for the conserved region of the proteins that begins with exon 2 for the plant enzymes. The program PILEUP of the GCG package, version 8.1 for UNIX, was used for the calculation.

**Results**

**Isolation of the *Igl* Gene and Enzymatic Properties.** To probe for genes that are involved in the synthesis of indole, a pivotal intermediate for both primary and secondary metabolism, the *BX1*-cDNA probe, was used for the screening of a genomic library (16). A genomic clone that encoded a protein with homology to TSAs was isolated (Fig. 1B). Because of a close homology to *Bx1*, the IGL of the DIMBOA biosynthetic pathway, the gene was termed *Igl*.

Several cDNA clones of *Igl* were identified by screening a whole plant cDNA library with the 1.2-kb internal *PstI* fragment of the genomic clone. Because the cDNA clones proved to be incomplete at the 5′ end, a 5′-rapid amplification of cDNA ends experiment was performed to get the full-size sequence of the gene. The reading frame of *Igl* encodes a 339-aa peptide and is interrupted by five intervening sequences as revealed by comparison with the cDNA clones (Fig. 1A). The exon-intron structure of *Igl* and *Bx1* is not fully conserved. Whereas the amino acids encoded by exon 2 to the C terminus display 82% identity to BX1 and 63% to TSA of *Arabidopsis thaliana*, the amino acids encoded by the first exon are <45% identical

Table 1. Comparison of catalytic properties

|           | $E. coli$ | Zea mays |
|-----------|-----------|----------|
|           | $\alpha$  | $\alpha 2 \beta 2$ | BX1 | IGL |
| $K_{m}^{IGP}$ | $0.5 \mathrm{mM}$ | $0.03 \mathrm{mM}$ | $0.013 \mathrm{mM}$ | $0.1 \mathrm{mM}$ |
| $K_{\text {cat }}$ | $0.002 \mathrm{s}^{-1}$ | $0.2 \mathrm{s}^{-1}$ | $2.8 \mathrm{s}^{-1}$ | $2.3 \mathrm{s}^{-1}$ |
| $K_{\text {cat }} / K_{m}^{IGP}$ | $0.004 \mathrm{mM}^{-1} \cdot \mathrm{s}^{-1}$ | $7.4 \mathrm{mM}^{-1} \cdot \mathrm{s}^{-1}$ | $215 \mathrm{mM}^{-1} \cdot \mathrm{s}^{-1}$ | $23 \mathrm{mM}^{-1} \cdot \mathrm{s}^{-1}$ |

between these protein species. In the case of $Bx1$ (11) and $TSA$ of $A. thaliana$ (19), exon 1 encodes a transit peptide.

A third gene with homology to $TSA$ was isolated from maize as a full-size cDNA clone and as a partial genomic clone (Fig. 1). A $5^{\prime}$ rapid amplification of cDNA ends experiment was performed to verify the $5^{\prime}$-end. The product of this gene is more related to $TSA$ from $A. thaliana$ ($70\%$ identity) than to $Igl$ and $Bx1$ ($65\%$ and $64\%$ identity, respectively; Fig. 1B). This product represents a candidate for the maize $TSA$ and was termed provisionally $TSAlike$. Sequence analyses of the genomic $Igl$ clone revealed that $Igl$ and $TSAlike$ are separated by $1.6 \mathrm{~kb}$ (Fig. 1). The genes map to the long arm of chromosome one ($1 \mathrm{L} 260$, id. tum11).

Phylogenetic analysis of the maize and $A. thaliana$ TSA homologous proteins defines two branches: one is comprised of the $A. thaliana$ TSAs and the $TSAlike$ sequence from maize, and the second consists of BX1 and IGL (Fig. 1B). Association with the tryptophan synthase beta subunit (TSB) has been shown to be basic for enzymatic activity of TSA in bacteria and in plants (19–21). In contrast, BX1 activity is independent of TSB (10). To address the question of whether the intimate structural similarity of IGL and BX1 reflects the same enzymatic mechanism, i.e., the conversion of IGP to indole, $Igl$ was expressed in $E. coli$, purified, and used for the determination of the steady-state kinetic constants as described for BX1 (10). The Michaelis constant $K_{m}^{IGP}$ of $0.1 \mathrm{mM}$ is $\approx 8$-fold higher compared with BX1, but the catalytic rate constant $k_{\text {cat }}$ of $2.3 \mathrm{s}^{-1}$ is similar for both enzymes. The $k_{\text {cat }} / K_{m}^{IGP}$ of $23 \mathrm{mM}^{-1} \cdot \mathrm{s}^{-1}$ proves that IGL is catalyzing the formation of indole, and is $\approx 3$ times more efficient in this reaction as bacterial TSA in its active form in the $\alpha 2 \beta 2$ complex (Table 1; ref. 22). Hence, two maize enzymes, BX1 and IGL, are capable of producing indole efficiently.

$Igl$ Transcript Level Is Elevated by Volicitin. Real-time PCR with gene-specific primer pairs was used to reveal a possible causal relationship between the biosynthesis of volatile indole induced by volicitin and expression of the homologous maize genes (Fig. 2). Whereas the steady-state transcript level of $Bx1$ is not influenced by volicitin treatment of the seedlings, for $Igl$ an $\approx 8$-fold increase compared with the buffer control was observed. This finding indicates that $Igl$ is a putative target gene in the signaling cascade that starts with the perception of volicitin and leads to the synthesis and emission of a defined blend of volatiles. The amount of $Igl$ transcript at $2 \mathrm{~h}$ after volicitin treatment exceeds the level of $GAP C$ and represents a major transcript under this condition. $TSAlike$ is not influenced by volicitin treatment and is expressed at a lower level compared with $Bx1$ and $Igl$ (Fig. 2).

Emission of Volatile Indole Is Independent of a Functional $Bx1$ Gene. To examine how the enzymes BX1 and IGL catalyze free indole emission in maize, a mutant line defective in the $Bx1$ gene product was examined. The $bx1 bx1$ line carries a deletion of the $Bx1$ gene that removes $\approx 0.5 \mathrm{~kb}$ of the $5^{\prime}$ part of the mRNA (10), allowing the hybridization signals derived from $Igl$ and $bx1$ to be distinguished unambiguously by the size difference. Elevated levels of indole were detected for volicitin treatment in the mutant line within $1.5 \mathrm{~h}$, and the maximum response was

![Figure 2](#fig2)

Fig. 2. Quantification of mRNA levels in the maize inbred-line CI31A. Incubation in buffer or volicitin solution was for $2 \mathrm{~h}$. The standard deviation is indicated by the bar.

observed at the $3-\mathrm{h}$ collection, with indole emissions of $326 \pm 77 \mathrm{ng} \cdot \mathrm{h}^{-1}$ compared with $13 \pm 4$ and $6 \pm 7$ for buffer and untreated controls, respectively (Fig. 3B).

Volicitin Induction of $Igl$ Expression Is Fast and Transient. The kinetics of $Igl$ induction were investigated by Northern blot analysis with the $bx1 bx1$ mutant maize line. $GAP C$, a gene not influenced by volicitin treatment (Fig. 2), was used to normalize for loading differences on the gels. The low level of $Igl$ transcript in untreated plants was increased by wounding (buffer control), and more dramatically by wounding in combination with volicitin treatment (Fig. 3A). The increase induced by cutting and incubation in buffer was $10$-fold, and the maximum value was reached $1 \mathrm{~h}$ after treatment and dropped slowly to resume the noninduced level after $8 \mathrm{~h}$. In contrast, a $50$- to $60$-fold increase was achieved with volicitin incubation. The principal augmentation of $Igl$ transcript was found within the first hour of treatment, but there was still an increase during the next hour. The decrease within the next $6 \mathrm{~h}$ was more pronounced as compared with that of the buffer treatment. Thus, the rise of $Igl$ transcript levels induced by volicitin is clearly transient. The emission of indole showed a similar transient release that peaked about $2 \mathrm{~h}$ after the maximum level of $Igl$ transcript was reached (Fig. 3). The effect of wounding on indole emission is relatively small. This response could indicate that a certain threshold of $Igl$ induction has to be exceeded for notable indole production to occur.

$Igl$ Is Expressed Highest in Upper Parts of the Plantlet. To determine whether $Igl$ transcript responds differently in distinct plant parts to volicitin, plants were dissected into three fractions after incubation (Fig. 4). For both incubation times ($1$ and $2 \mathrm{~h}$), the highest level of $Igl$ transcript was detected in the upper part of the seedling. The buffer-control seedlings displayed a lower level of $Igl$ transcript for all parts, and there was no pronounced variation between the different fractions.

Discussion

An Enzymatic Mechanism to Produce Indole. A crucial step in the evolution of secondary metabolic pathways is the establishment of the branch reactions that channel metabolites out of primary metabolism. In maize, the first step in the synthesis of the benzoxazinoid DIMBOA (Fig. 5) is the conversion of IGP to indole. This reaction is catalyzed by the BX1 enzyme (10) and defines the branch point from tryptophan biosynthesis. We have

Fig. 3. Analysis of *Igl* transcript levels and indole emission in the *bx1bx1* mutant line. The seedlings were harvested at the indicated time points after incubation in buffer or buffer with volicitin. Uncut seedlings were taken as a control. The standard deviation is given by the vertical bars. (A) RNA quantification. The hybridization signals of the *Igl* and GAPC probe were quantified with the PhosphorImager. GAPC values were taken to normalize the loading of the lanes. Five individual experiments were made for each time point except for 8 h, at which time only two repetitions were made. An example of a Northern blot is given (Inset). The *Igl* signal is shown in the upper part, and hybridization of GAPC is shown in the lower part. (B) Determination of indole emission. Indole produced within 1.5 h was collected at the indicated time points and quantified as described in Materials and Methods.

characterized here a second maize enzyme, IGL, with similar properties. Both enzymes act efficiently *in vitro* as IGP lyases (Table 1). The sequence similarity suggests that IGL and BX1 originated from TSA, the enzyme responsible for the penultimate reaction in tryptophan biosynthesis (Fig. 5). Tryptophan synthase is typically a tetrameric heterosubunit complex that is formed by two TSA–TSB complexes that are linked via TSB (20). These complexes were originally described in bacteria; however, an analogous heterosubunit tryptophan synthase complex exists in plants (19, 23). Indole is usually not released from the tryptophan synthase complex but is directly converted to tryptophan by TSB. TSA activity depends almost completely on complex formation.

For both IGL and BX1, changes relative to the TSA-consensus sequence are observed and located in two domains. In IGL and BX1, an unusual tyrosine substitutes a conserved aliphatic residue (position 58 of *Salmonella typhimurium* TSA). Two conserved glutamine residues (position 134, 135 of *S.* *typhimurium* TSA) are substituted by valine and alanine in the case of BX1, and by glycine and asparagine in the case of IGL. A 4-aa deletion (BX1) and an addition of 2 aa (IGL) are found in close proximity to these substitutions; hence the architecture of this part of the protein might be altered. Both modified regions are part of the interaction domains of TSA and TSB based on the crystal structure of the *S.* *typhimurium* tryptophan synthase complex (24, 25). Whether these alterations are significant for the establishment of the different enzymatic properties of IGL and BX1 versus TSA remains to be shown. No deviation from the TSA consensus is found for TSAlike.

The synthesis of several other secondary metabolites in plants, such as the indole glucosinolates, anthranilate-derived alkaloids, and tryptamine derivatives (19, 26, 27), could depend on indole as an intermediate. IGP was proposed as a branch point from the tryptophan pathway for the synthesis of the indolic phytoalexin camalexin (3-thiazol-2′-yl-indole) in *A.* *thaliana* (28, 29). It would not be surprising if BX1 and IGL would represent the preferred route for indole production.

PRIMARY METABOLISM

Indole-3-glycerol
phosphate

(TSA-TSB)₂
IGL
BX1

![Chemical Structure 1](chemical_structure_1.png)
Tryptophan

![Chemical Structure 2](chemical_structure_2.png)
Volatile
Indole

![Chemical Structure 3](chemical_structure_3.png)

DIMBOA

Fig. 5. Branching of primary and secondary metabolic pathways in maize. (TSA-TSB)₂: tryptophan synthase complex; BX2–7: enzymes necessary for the conversion of indole into DIMBOA. IGL and BX1 are TSA-homologous proteins that catalyze indole formation. TSA is active only in a complex with TSB. Indole formed transiently in this complex is immediately converted to tryptophan.

Function of *Igl* in Maize. Indole has been identified as one of the blend of volatile chemicals emitted from maize in response to herbivore damage (2). The production and release of indole in plants has been shown to be an active process in which *de novo* synthesis is triggered in response to insect feeding (30). Because volicitin (9) has been identified as an elicitor molecule that can mimic larvae feeding in that it triggers indole emissions, induction of an indole-biosynthetic pathway that responds directly or indirectly to volicitin application is predicted. The data presented here suggest that the *Igl* gene product is a key player in this process: IGL is an efficient indole-3-glycerol phosphate lyase *in vitro* (see above). *Igl* transcript accumulation (Fig. 3) closely resembles the temporary fashion of indole emission induced by volicitin application. *Igl* transcript abundance is ahead of the rise of indole production by about 2 h; this delay could result from translation of *Igl* mRNA into a functional enzyme. A similar time interval was recognized for the sesquiterpene cyclase *stcl* transcript and emission of a naphthalene-based sesquiterpene [see accompanying paper (ref. 31)]. Neither *Bxl* nor *TSAlke* transcript levels are influenced by volicitin, and, hence, neither are expected to contribute directly to volicitin-induced volatile indole production.

A further correlation of *Igl* induction and volatile emission is found in the requirement of a certain developmental stage (two to three expanded leaves) for volatile emission and *Igl* induction as response to herbivory and volicitin treatment. Application in earlier stages of plant development (4 days) results in stimulation of neither *Igl* transcription nor indole formation (data not shown). The same developmental stage is necessary for the induction of *stcl* [see accompanying paper (ref. 31)]. The results implicate that *Igl* is a target gene in the volicitin signaling cascade; however, ultimate proof will require an *Igl* mutant.

The cell types that are responsible for volatile production remain to be determined. Whether there is substrate channeling and indole produced by IGL is used exclusively as a volatile, or whether it is available for other biosynthetic pathways, for example DIMBOA formation, can be addressed in future experiments.

Elevated *Igl* Transcript Level with Elicitor Application. *Igl* mRNA is hardly detectable in maize plantlets (Figs. 3 and 4). However, cutting and incubation in volicitin solution leads to a drastic increase of the *Igl* transcript steady-state level (Figs. 3 and 4), with peak amounts even exceeding the transcript levels of the housekeeping gene GAP C. Accumulation is achieved within 1 to 2 h after treatment with volicitin, and, although application is at the base of the plantlet, the top part displays the highest *Igl* transcript level by volicitin treatment (Fig. 4). This response demonstrates a relatively rapid movement of the signal. The induction kinetics are comparable to the examples of induced transcription of certain mitogen-activated protein kinases by the fungal elicitor Avr9 and wounding; here, maximal transcript accumulation is reached 1 to 2 h after elicitation of cell cultures or plants (32). For *stcl* in maize [see accompanying paper (ref. 31)], induction by insect regurgitant or volicitin reaches maximum transcript levels at 4 h. Peak levels at 8 h or 12 h after caterpillar chewing are reported for the protease inhibitor II and 3-hydroxy-3-methylglutaryl-CoA reductase genes of potato (33). Interestingly, in these cases, induction is delayed for wounding compared with wounding, in conjunction with inducer application. In contrast, for *Igl*, both kinetics are comparable, but the maximum transcript levels are different (Fig. 3).

Evolution of Secondary Metabolic Pathways. Duplication of genes of the primary metabolism and subsequent modification of gene

function may be a driving force for the evolution of secondary metabolic pathways. *Bxl* and *Igl* seem to be derived from *TSA* by gene-duplication events and were modified during evolution to obtain their distinct function in secondary metabolism. Interestingly, both enzymes efficiently catalyze indole formation independently of a TSB-like subunit. Gene duplication is indicated by the fact that *Igl* and *TSAlike* are separated by only 1.6 kb in the maize genome. Another example for the recruitment of a gene of the primary metabolism is represented by homospermidine synthase from *Senecio vernalis*. This enzyme is derived from deoxyhypusine synthase, an enzyme required for activation of translation initiation factor 5A (34).

The coexistence of IGL and BX1 demonstrates that modified genes can catalyze the same reaction directed toward different biochemical routes. Such parallel secondary metabolic pathways, at least in maize, seem to augment the chemical-defense capacity of the plant. *Bxl* is expressed constitutively in the young seedlings and catalyzes the accumulation of the indole-derived antifeedant components DIBOA and DIMBOA. In contrast, *Igl* is induced later in plant development in response to herbivore damage, is able to catalyze the synthesis of free indole that is released from the leaves, and is part of the blend of odors that attracts beneficial insects. An inducible indole-producing function is also required for cotton (35) and several other plants (36) that respond to different chemical signals with emission of volatile indole. Because the efficiency of indole formation depends on the amount of available substrate (IGP) delivered by the primary metabolism, regulatory interactions with the primary pathway as demonstrated for camalexin biosynthesis (37) are also anticipated for volatile indole production.

We thank B. Burr for processing the mapping data, H. Alborn for providing volicitin, and Verena Goretzki for sequence analysis of *TSAlike*. The excellent technical assistance of Regina Hüttl is acknowledged. This work was supported by the Deutsche Forschungsgemeinschaft (SFB 369) and by Fonds der Chemischen Industrie.

1. Dicke, M., Sabelis, M. W., Takabayashi, J., Bruin, J. & Posthumus, M. A. (1990) *J. Chem. Ecol.* **16**, 3091–3118.
2. Turlings, T. C. J., Tumlinson, J. H. & Lewis, W. J. (1990) *Science* **250**, 1251–1253.
3. Takabayashi, J., Dicke, M. & Posthumus, M. A. (1994) *J. Chem. Ecol.* **20**, 1329–1354.
4. Takabayashi, J., Dicke, M. & Posthumus, M. A. (1991) *Phytochemistry* **30**, 1459–1462.
5. De Moraes, C. M., Lewis, W. J., Paré, P. W., Alborn, H. T. & Tumlinson, J. H. (1998) *Nature (London)* **393**, 570–573.
6. Turlings, T. C., Tumlinson, J. H., Heath, J. H., Proveau, A. T. & Doolittle, R. E. (1991) *J. Chem. Ecol.* **17**, 2235–2251.
7. Turlings, T. C., McCall, P. J., Alborn, H. T. & Tumlinson, J. H. (1993) *J. Chem. Ecol.* **19**, 411–425.
8. Mattiacci, L., Dicke, M. & Posthumus, M. A. (1995) *Proc. Natl. Acad. Sci. USA* **92**, 2036–2040.
9. Alborn, H. T., Turlings, T. C., Jones, T. H., Stenhagen, G., Loughrin, J. H. & Tumlinson, J. H. (1997) *Science* **276**, 945–949.
10. Frey, M., Chomet, P., Glawischnig, E., Stettner, C., Grun, S., Winklmair, A., Eisenreich, W., Bacher, A., Meeley, R. B., Briggs, S. P., *et al.* (1997) *Science* **277**, 696–699.
11. Sicker, D., Frey, M., Schulz, M. & Gierl, A. (2000) *Int. Rev. Cytol.* **198**, 319–346.
12. Hamilton, R. H. (1964) *Weeds* **12**, 27–30.
13. Rosenberg, A. H., Lade, B. N., Chui, D. S., Lin, S. W., Dunn, J. J. & Studier, F. W. (1987) *Gene* **56**, 125–135.
14. Creighton, T. E. (1970) *Eur. J. Biochem.* **13**, 1–10.
15. Logemann, J., Schell, J. & Willmitzer, L. (1987) *Anal. Biochem.* **163**, 16–20.
16. Frey, M., Kliem, R., Saedler, H. & Gierl, A. (1995) *Mol. Gen. Genet.* **246**, 100–109.
17. Brinkmann, H., Martinez, P., Quigley, F., Martin, W. & Cerff, R. (1987) *J. Mol. Evol.* **26**, 320–328.
18. Burr, B. & Burr, F. (1991) *Trends Genet.* **7**, 55–60.
19. Radwanski, E. R., Zhao, J. & Last, R. L. (1995) *Mol. Gen. Genet.* **248**, 657–667.
20. Creighton, T. E. & Yanofsky, C. (1966) *J. Biol. Chem.* **241**, 980–990.
21. Radwanski, E. R., Barczak, A. J. & Last, R. L. (1996) *Mol. Gen. Genet.* **253**, 353–361.
22. Weischet, W. O. & Kirschner, K. (1976) *Eur. J. Biochem.* **65**, 365–373.
23. Nagao, R. T. & Moore, T. C. (1972) *Arch. Biochem. Biophys.* **149**, 402–413.
24. Hyde, C. C., Ahmed, S. A., Padlan, E. A., Miles, E. W. & Davies, D. R. (1988) *J. Biol. Chem.* **263**, 17857–17871.
25. Weyand, M. & Schlichting, I. (1999) *Biochemistry* **38**, 16469–16480.
26. Kutchan, T. M. (1995) *Plant Cell* **7**, 1059–1070.
27. Bak, S., Nielsen, H. L. & Halkier, B. A. (1998) *Plant Mol. Biol.* **38**, 725–734.
28. Tsuji, J., Zook, M., Somerville, S. C., Last, R. L. & Hammerschmidt, R. (1993) *Plant Physiol. Mol. Plant Pathol.* **43**, 221–229.
29. Zook, M. (1998) *Plant Physiol.* **118**, 1389–1393.
30. Paré, P. W. & Tumlinson, J. H. (1997) *Nature (London)* **385**, 30–31.
31. Shen, B., Zheng, Z. & Dooner, H. K. (2000) *Proc. Natl. Acad. Sci. USA*, 10.1073/pnas.240284097. http://www.pnas.org/cgi/doi/10.1073/pnas.240284097
32. Romeis, T., Piedras, P., Zhang, S., Klessig, D. F., Hirt, H. & Jones, J. D. (1999) *Plant Cell* **11**, 273–287.
33. Korth, K. L. & Dixon, R. A. (1997) *Plant Physiol.* **115**, 1299–1305.
34. Ober, D. & Hartmann, T. (1999) *Proc. Natl. Acad. Sci. USA* **96**, 14777–14782.
35. Loughrin, J. H., Manukian, A., Heath, J. H., Turlings, T. C. & Tumlinson, J. H. (1994) *Proc. Natl. Acad. Sci. USA* **91**, 11836–11840.
36. Boland, W., Hopke, J., Donath, J., Nüsse, J. & Bublitz, F. (1995) *Angew. Chem. Int. Ed. Engl.* **34**, 1600–1602.
37. Zhao, J., Williams, C. C. & Last, R. L. (1998) *Plant Cell* **10**, 359–370.
