
14. Wall, K. A., Pierce, J. D. & Elbein, A. D. Inhibitors of glycoprotein processing alter T-cell proliferative responses to antigen and to interleukin 2. *Proc. Natl Acad. Sci. USA* **85**, 5644–5648 (1988).

15. Cummings, R. D., Trowbridge, I. S. & Kornfeld, S. A mouse lymphoma cell line resistant to the leucoagglutinating lectin from *Phaseolus vulgaris* is deficient in UDP-GlcNAcα-D-mannoside β1,6 N-acetylglucosaminyltransferase. *J. Biol. Chem.* **257**, 13421–13427 (1982).

16. Granovsky, M. *et al.* Suppression of tumor growth and metastasis in Mgat5-deficient mice. *Nature Med.* **6**, 306–312 (2000).

17. Lafaille, J. J., Nagashima, K., Katsuki, M. & Tonegawa, S. High incidence of spontaneous autoimmune encephalomyelitis in immunodeficient anti-myelin basic protein T cell receptor transgenic mice. *Cell* **78**, 399–408 (1994).

18. Downward, J., Graves, J. D., Warne, P. H., Rayter, S. & Cantrell, D. A. Stimulation of p21<sup>ras</sup> upon T-cell activation. *Nature* **346**, 719–723 (1990).

19. Trevillyan, J. M., Lu, Y. L., Atluru, D., Phillips, C. A. & Bjorndahl, J. M. Differential inhibition of T cell receptor signal transduction and early activation events by a selective inhibitor of protein-tyrosine kinase. *J. Immunol.* **145**, 3223–3230 (1990).

20. Wang, J. *et al.* Atomic structure of an αβ T cell receptor (TCR) heterodimer in complex with an anti-TCR Fab fragment derived from a mitogenic antibody. *EMBO J.* **17**, 10–26 (1998).

21. Hubbard, S. C., Kranz, D. M., Longmore, G. D., Sitkovsky, M. V. & Eisen, H. N. Glycosylation of the T-cell antigen-specific receptor and its potential role in lectin-mediated cytotoxicity. *Proc. Natl Acad. Sci. USA* **83**, 1852–1856 (1986).

22. Reich, Z. *et al.* Ligand-specific oligomerization of T-cell receptor molecules. *Nature* **387**, 617–620 (1997).

23. Reif, K. & Cantrell, D. A. Networking Rho family GTPases in lymphocytes. *Immunity* **8**, 395–401, (1998).

24. Rudd, P. M. *et al.* Roles for glycosylation of cell surface receptors involved in cellular immune recognition. *J. Mol. Biol.* **293**, 351–366 (1999).

25. Pace, K. E., Lee, C., Stewart, P. L. & Baum, L. G. Restricted receptor segregation into membrane microdomains occurs on human T cells during apoptosis induced by galectin-1. *J. Immunol.* **163**, 3801–3811 (1999).

26. Chung, C. D., Patel, V. P., Moran, M., Lewis, L. A. & Carrie Miceli, M. Galectin-1 induces partial TCR zeta-chain phosphorylation and antagonizes processive TCR signal transduction. *J. Immunol.* **165**, 3722–3729 (2000).

27. Offner, H. *et al.* Recombinant human beta-galactoside binding lectin suppresses clinical and histological signs of experimental autoimmune encephalomyelitis. *J. Neuroimmunol.* **28**, 177–184 (1990).

28. Barkal, N. & Leibler, S. Robustness in simple biochemical networks. *Nature* **387**, 913–917 (1997).

29. Oliveira-dos-Santos, A. J. *et al.* CD28 costimulation is crucial for the development of spontaneous autoimmune encephalomyelitis. *J. Immunol.* **162**, 4490–4495 (1999).

30. Moloney, D. J. *et al.* Fringe is a glycosyltransferase that modifies Notch. *Nature* **406**, 369–375 (2000).

### Acknowledgements

We thank S. Kulkarni and J. Tsang for technical assistance. This research was supported by grants from NCI of Canada, the Mizutani Foundation, the National Science and Engineering Research Council of Canada, and GlycoDesign, Toronto. Correspondence and requests for materials should be addressed to J.W.D. (e-mail: dennis@mshri.on.ca).

---

### Crystal structure of photosystem II from *Synechococcus elongatus* at 3.8 Å resolution

**Athina Zouni**, Horst-Tobias Witt, Jan Kern, Petra Fromme, Norbert Krauß, Wolfram Saenger, Peter Orth

*Max-Volmer-Institut für Biophysikalische Chemie und Biochemie,*  
Technische Universität Berlin, Straße des 17. Juni 135, D-10623, Berlin, Germany  

*Institut für Chemie, Kristallographie, Freie Universität Berlin,* Takustrasse 6, D-14195 Berlin, Germany

---

Oxygenic photosynthesis is the principal energy converter on earth. It is driven by photosystems I and II, two large protein-cofactor complexes located in the thylakoid membrane and acting in series. In photosystem II, water is oxidized; this event provides the overall process with the necessary electrons and protons, and the atmosphere with oxygen. To date, structural information on the architecture of the complex has been provided by electron microscopy of intact, active photosystem II at 15–30 Å resolution<sup>1</sup>, and by electron crystallography on two-dimensional crystals of D1-D2-CP47 photosystem II fragments without water oxidizing activity at 8 Å resolution<sup>2</sup>. Here we describe the X-ray structure of photosystem II on the basis of crystals fully active in water oxidation<sup>3</sup>. The structure shows how protein subunits and cofactors are spatially organized. The larger subunits are assigned and the locations and orientations of the cofactors are defined. We also provide new information on the position, size and shape of the manganese cluster, which catalyzes water oxidation.

Conversion of light to chemical energy at photosystem II (PSII) is associated with charge separation across the thylakoid membrane (for review see ref. 4). It is initiated by ejection of an electron from the excited primary donor P680, a chlorophyll *a* located towards the luminal side of the membrane at the heart of the PSII reaction centre that is formed by protein subunits D1 and D2. When the cationic radical P680<sup>+</sup> is formed, the electron moves by means of a phaeophytin to the electron stabilizing acceptor Q<sub>A</sub>, a plastoquinone that is tightly bound at the stromal side of subunit D2. After each of four successive charge separating steps that are light induced, P680<sup>+</sup> abstracts one electron from a manganese cluster (generally assumed to contain four manganese ions) by means of the redox-active tyrosine residue Tyr<sub>Z</sub>. In turn, the four positive charges accumulated in the manganese cluster oxidize two water molecules, coupled with the release of one O<sub>2</sub> and four H<sup>+</sup>. In the first two charge separations, Q<sub>A</sub> doubly reduces one mobile molecule Q<sub>B</sub> docked to the binding site B on D1. After uptake of two protons, Q<sub>B</sub>H<sub>2</sub> is released into the plastoquinone pool that is embedded in the membrane, and replaced by a new Q<sub>B</sub> from the pool for another round of reduction and release.

We isolated PSII from the thermophilic cyanobacterium *Synechococcus elongatus* in the form of homodimers as shown by electron microscopy (E. J. Boekema, unpublished observations). With these preparations, three-dimensional crystals were grown<sup>5</sup> that are suitable for X-ray analysis (see Methods). According to SDS-polyacrylamide gel electrophoresis and mass-spectrometry (MALDI-TOF) (data not shown), this PSII is composed of at least 17 subunits<sup>6</sup> of which 14 are located within the photosynthetic membrane: the reaction centre proteins D1 (PsbA) and D2 (PsbD); the chlorophyll-containing inner-antenna subunits CP43 (PsbC) and CP47 (PsbB); α- and β-subunits of cytochrome b-559 (PsbE and PsbF); and the smaller subunits PsbH, PsbI, PsbJ, PsbK, PsbL, PsbM, PsbN and PsbX. The membrane-extrinsic cytochrome c-550

![Figure 1](#fig1)

**Figure 1** Electron densities of PSII after density modification and their interpretation.  
*a*, Surface of PSII homodimer drawn from the averaging mask generated during density modification. View direction along the membrane plane; the position of local-C2 rotation axis, is shown. *b*, Cytochrome (Cyt) b-559 heterodimer with electron densities contoured at 1.2σ (r.m.s. deviation above the mean electron density) for protein and haem group, and at 4.0σ for Fe<sup>2+</sup>. The termini of the α-helices are labelled N, C for the α-subunit, and N′, C′ for the β-subunit. *c*, Head groups of P680 chlorophyll (Chl) aP<sub>D1</sub> and P<sub>D2</sub> with view perpendicular to their planes. Mg<sup>2+</sup> are depicted as red spheres. *d*, Head group of phaeophytin Pheo<sub>D1</sub>.


(PsbV), the protein with a relative molecular mass of 12,000 (M<sub>r</sub> 12K) (PsbU) and the manganese-stabilizing 33K protein (PsbO) are located at the luminal side of PSII (ref. 6).

In the crystal structure, PSII occurs as a homodimer with the longest dimensions of the membrane integral part 190 Å × 100 Å (Fig. 1a). This part is 40 Å thick and extends from the stromal side of the membrane by no more than 10 Å. The luminal side of each monomer has prominent protrusions up to 55 Å above the membrane. The two monomers in the dimer are related by a local-C2 rotation axis orientated perpendicular to the membrane plane.

In the electron density (see Methods), we used four prominent maxima as landmarks for positioning of the individual subunits; the three smaller maxima were assigned to iron and one more extended to the manganese cluster (Fig. 2a, b, arrows). Secondary structure elements, such as α-helices and β-sheets as well as cofactors, have been modelled into the electron density map using C<sub>α</sub> traces and porphyrin molecules. We used porphyrin molecules because the orientation in space of the tetrapyrrole planes of the chlorophyll *a* molecules, pheophytins and haems is well defined in the electron density map, whereas the orientations within the planes are arbitrary at this level of resolution. Representative electron densities are shown in Fig. 1b–d.

The membrane integral part of the PSII monomer defines the membrane plane, which is parallel to the paper plane in Fig. 2a. It is characterized by a field of 36 transmembrane α-helices. Twenty-two of these, assigned to D1, D2, CP43 and CP47, are arranged with a local pseudo-two-fold rotation symmetry. The corresponding symmetry axis, named pseudo-C2 axis, is orientated parallel to the local-C2 axis and passes through the centre of the field of α-helices and through the non-haem iron located at the stromal side of the membrane (Fig. 3a).

Two groups of five transmembrane α-helices each are arranged in two semicircles interlocked in a handshake motif and related by the pseudo-C2 axis. They have been assigned to α-helices A–E of D1 and D2, respectively, with α-helices C and D connected in each of the subunits by a long α-helix CD on the luminal side; the short α-helix DE located on the stromal side connects α-helices D and E, as proposed for PSII (refs 7, 8). Subunits D1 and D2 are clearly distinguished from each other by the position of the manganese cluster, which is coordinated by D1 (ref. 6). This assignment is supported by the location of the Q<sub>A</sub> binding site at D2 (refs 6–8) (see below).

The arrangement of α-helices of D1 and D2 resembles that of subunits L and M in the purple bacterial reaction centre (PbRC)<sup>9</sup> and the five carboxy-terminal helices of PsaA and PsaB in photosystem I (PSI) (ref. 10). This supports the hypothesis of the evolution of all photosynthetic reaction centres from one common ancestor<sup>2,10</sup>. Crosslinking studies have shown that the homologous inner-antenna subunits CP43 and CP47 flank both sides of the complex D1/D2/Cyt *b*-559 (ref. 11). As D1 and CP43 are crosslinked by acceptor-side photoinhibition of PSII (ref. 12), CP43 and CP47 could be assigned. Both consist of six transmembrane α-helices arranged as a trimer of dimers related by the pseudo-C2 axis and coordinating the antenna chlorophyll *a* (see Fig. 2a). They are of similar structure to the six amino-terminal transmembrane α-helices of PsaA and PsaB in PSI (refs 2, 10). The arrangement of transmembrane helices as well as the assignment of the subunits D1, D2 and CP47 corresponds to the structural model provided by electron cryo-crystallography at 8 Å resolution of a PSII fragment<sup>2,13</sup>.

One cytochrome *b*-559 has been identified in the electron density by virtue of its haeme Fe<sup>2+</sup>; it is found in the same position as proposed in ref. 13. The two subunits α and β, each forming a single transmembrane α-helix, can be distinguished because the C-terminus of the longer α-subunit extends far into the lumen (Fig. 1b). We confirmed the presence of only one cytochrome *b*-559 per reaction centre by spectroscopic analysis of fresh PSII

**Figure 2** Structure of PSII with assignment of protein subunits and cofactors. **a**, Arrangement of transmembrane α-helices and cofactors in PSII. One monomer of the dimer is shown completely, with part of the second monomer related by the local-C2 axis (filled ellipse on the dotted interface). Chl *a* head groups and haems are indicated by black wire drawings. The view direction is from the luminal side, perpendicular to the membrane plane. The α-helices of D1, D2 and Cyt *b*-559 are labelled. D1/D2 are highlighted by an ellipse and antennae, and CP43 and CP47 by circles. Seven unassigned α-helices are shown in grey. The four prominent landmarks (three irons and the manganese (Mn) cluster) are indicated by arrows. **b**, Side view of PSII monomer looking down the long axis of the D1/D2 subunits from the right side in Fig. 2a, at slightly tilted membrane plane and rotated 180° so that the luminal side is bottom. PsbO (33K protein) is shown as a β-sheet structure (green), and Cyt *c*-550 as a helical model (grey).

preparations and of redissolved PSII crystals (data not shown). The controversial issue of one or two cytochrome b-559 per reaction centre present in PSII is, however, still unresolved, as this may depend on the preparation and/or organism⁶.

Crosslinking studies indicate that PsbI and PsbX are close to D2 and cytochrome b-559 (refs 14, 15); our electron density shows two transmembrane α-helices in this location (Fig. 2a). They could be distinguished with reference to the two-dimensional crystal structure of the D1-D2-CP47 PSII fragment²³, as it contains PsbI (and no PsbX). One unassigned α-helix² is in a position close to CP47 and D2, and was consequently assigned as PsbI (see Fig. 2a).

PsbH, PsbK and PsbL have been implicated in the stabilization of the PSII dimer¹⁶,¹⁷. They were tentatively assigned to a three-helix bundle close to the local-C2 axis (Fig. 2a). In the present model, one of these three transmembrane helices is only about 13 Å apart from Qₐ and could correspond to PsbH, which is supposed to influence the electron transport between Qₐ and Qᵦ (ref. 18). Seven additional transmembrane α-helices have been found in the electron density map (Fig. 2a, b, grey α-helices) but could not be assigned.

At the luminal side of the electron density, two of the three extrinsic subunits (cytochrome c-550, 33K and 12K proteins) could be located unambiguously (Fig. 2b). The position of cytochrome c-550 is indicated by its haem iron, surrounded by three tubular structures. These helped to model part of the sequentially homologous horse heart cytochrome c (Protein Data Bank accession number 1HRC into the electron density. A roughly 35 Å long, cylindrical arrangement characterized by β-strands with uncertain connectivities has been assigned to the 33K subunit (PsbO), in agreement with Fourier transform infrared data¹⁹. The modelled cylindrical structure of PsbO is tilted against the membrane by 45° and corresponds to about half of the molecular mass of this subunit. The expected extrinsic 12K subunit could not yet be located because of the high amount of non-assigned secondary structure elements belonging either to loop regions of membrane intrinsic subunits (especially the long luminal loops of CP43 and CP47) or to the remaining parts of subunit PsbO or PsbV.

The internal-antenna subunits CP43 and CP47 (Fig. 2a) contain 12 and 14 chlorophyll *a*, respectively, located in the open space between the three helix dimers, similar to that found for CP47 (ref. 2). The number of chlorophyll *a* molecules found here (26) is close to the lower limit of the range (25–50)²⁰. The chlorophyll *a* molecules in each subunit occupy two layers close to the stromal and the luminal sides of the membrane, respectively (data not shown), which is consistent with the observation that most of the conserved histidines (12 in CP43 and 8 in CP47) are also located towards the stromal and luminal ends of the respective transmembrane helices⁶. Within the two antenna systems the centre-to-centre distances between pairs of nearest chlorophylls are in the range 8.5–13.5 Å.

The cofactors of the electron transfer chain form two branches organized symmetrically along the pseudo-C2 axis (Fig. 3a, b). They were assigned to the D1 and D2 proteins by analogy with the arrangement of α-helices in L/M subunits of the PbRC. Towards the luminal side, two chlorophyll *a* molecules P<sub>D₁</sub> and P<sub>D₂</sub> (see Fig. 1c for electron density) are observed with a Mg–Mg distance of 10 Å (roughly 11 Å was assessed in ref. 2). Figure 3b shows that the head groups are parallel (with 5 Å interplanar distance) and arranged perpendicular to the membrane plane. They possibly represent P680. Owing to the large separation of these two chlorophyll molecules, excitonic coupling is weak and they can be regarded as monomeric chlorophyll molecules, suggesting that the unpaired electron in P680⁺ is located on one of them.

Towards the stromal side, two spectroscopically unidentified chlorophyll *a* molecules (Chl<sub>D₁</sub> and Chl<sub>D₂</sub>) are located at 9.8 and 10.0 Å, respectively, apart from P<sub>D₁</sub> and P<sub>D₂</sub>. Their planes are tilted roughly 30° against the membrane plane, which is analogous to the accessory bacteriochlorophylls in PbRC. They are followed by two

pheophytin molecules Pheo<sub>D₁</sub> and Pheo<sub>D₂</sub> (Fig. 3a). The site of tightly bound Qₐ is at 12.0 Å from Pheo<sub>D₁</sub> and occupied by a plastoquinone, the centre of its ring being at a distance of 10.5 Å from the non-haem iron. The distances between P680⁺ and Qₐ, determined by electron paramagnetic resonance²¹ (27.4 ± 0.3 Å), are in agreement with the structural model (P<sub>D₂</sub>–Qₐ: 28 ± 1 Å; P<sub>D₁</sub>–Qₐ: 26 ± 1 Å). The putative binding site for the mobile Qᵦ is unoccupied, indicating that this plastoquinone probably dissociated during preparation. P680⁺ is probably located on P<sub>D₁</sub>, which is close to Tyr<sub>Z</sub>, as this acts as an immediate electron donor to the cationic radical⁴.

Two extra chlorophyll *a* molecules were assigned to the spectroscopically identified species Chlz<sub>D₁</sub> and Chlz<sub>D₂</sub>, shown to be coordinated to His 118 in D1 and His 117 in D2 (refs 22, 23). They are 30.2 and 30.4 Å apart from P<sub>D₁</sub> and P<sub>D₂</sub>, respectively. Owing to the large separation of CP43 and CP47 from the central cofactors P<sub>D₁</sub> and P<sub>D₂</sub> of P680, a possible route for the exciton

![Figure 3](#fig3)

**Figure 3** Arrangement of cofactors of the electron transfer chain located in subunits D1 and D2. **a**, View direction along the membrane plane. Full lines indicate centre-to-centre distances (Å) between the cofactors (uncertain to about ± 1 Å, as estimated visually). The pseudo-C2 axis is shown by the vertical dotted line; it runs through the non-haem iron Fe and is parallel to the local-C2 axis. The asterisk indicates the putative Q<sub>B</sub> binding site. **b**, View onto membrane plane, full lines defined as in **a**, dotted lines refer to distances between centres of cofactors and the pseudo-C2 axis. As this axis is not crystallographic, distances are approximate (~). Tyrosines Tyr<sub>Z</sub> and Tyr<sub>D</sub> are omitted because Tyr<sub>Z</sub> is below the manganese cluster. Chl<sup>43,47</sup> are the nearest Chl *a* of CP43 and CP47.

letters to nature

Table 1 Crystallographic statistics

| Data collection | Native | Cd | Hg I | Hg II | Hg III | Au | Pt | Native |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Wavelength (Å) | 0.934† | 0.934† | 0.9072‡ | 0.8439§ | 0.9072‡ | 0.8439§ | 1.894∥ |  |
| Resolution (Å) | 20–4.2 | 20–3.8 | 20–4.7 | 20–5.8 | 20–5.7 | 20–4.2 | 20–4.8 |  |
| Unique reflections | 63,639 | 84,964 | 45,429 | 30,210 | 21,643 | 22,458 | 52,611 | 48,420 |
| Redundancy | 3.3 | 3.2 | 3.0 | 1.9 | 2.6 | 2.8 | 3.0 | 2.3 |
| Completeness (%) ¶ | 98.1 (98.5) | 95.4 (84.5) | 95.5 (87.5) | 55.0 (52.1) | 80.3 (82.7) | 86.7 (83.9) | 81.1 (81.0) | 85.0 (70.7) |
| Completeness of Friedel pairs (%) ¶ | – | – | 76.6 (65.8) | 59.8 (53.6) | 28.0 (18.7) | – | – | 55.9 (44.8) |
| ⟨I/σ⟩ | 12.8 (3.5)# | 17.1 (3.4) | 14.8 (3.2) | 17.5 (2.5) | 11.3 (3.5) | 12.5 (5.3) | 13.3 (3.8) | 10.5 (5.0) |
| Rmerge (%) ¶ | 6.8 (28.8) | 6.8 (41.8) | 4.1 (28.8) | 5.4 (30.0) | 4.5 (42.1) | 6.2 (26.3) | 5.0 (35.6) | 9.2 (20.0) |

Phasing statistics

|  | Number of heavy atom sites | Rcullis (acentric) | Phasing power# | Acentric/centric |
| --- | --- | --- | --- | --- |
|  | – | 0.92 (0.96) | 0.78/0.49 | – |
|  | 5 | 0.85 (0.85) | 1.07/0.73 | (0.72/0.52)¶ |
|  | 13 | 0.83 (0.91) | 1.21/0.91 | (1.3/0.76) |
|  | 16 | 0.70 (0.83) | 1.12/0.75 | (1.02/0.62) |
|  | 11 | – | – | (1.08/0.68) |
|  | 2 | 0.95 (0.99) | 0.62/0.44 | (0.70/0.51) |
|  | 6 | 0.96 (0.99) | 0.49/0.51 | (0.42/0.44) |
|  | – | – | – | – |

*Heavy atoms derivatives: Cd, cadmium sulphate; Hg I, ethylmercuriphosphate; Hg II, chloromercuriacetate; Hg III, p-chloromercuriphenylsulphonic acid; Au, potassium dicyanoaurate (I); Pt, potassium tetracyanoplatinate(II).

†ID14-EH1 ESRF.

‡X11 DESY.

§BW7B DESY.

∥ID13 ESRF.

¶The numbers in parentheses indicate the values in the highest resolution shell (taken from SCALEPACK²⁶).

#Phasing power was calculated between 20 and 4.2 Å.

transfer from the antenna systems to the primary donor P680 may include the combinations ChlzD₁/ChlD₁ and ChlzD₂/ChlD₂. As the nearest chlorophyll *a* of each antenna system in CP43 and CP47 is positioned about 21 Å apart from the respective pheophytin of the electron transfer chain (Fig. 3b), pheophytins may also be involved in exciton transfer.

The pseudo-C2 symmetry of the cofactor arrangement is broken by cytochrome *b*-559 (Figs 1b and 2a), its haem-iron being 27.0 Å apart from ChlzD₂, and about 8 Å apart from the stromal side. For cytochrome *b*-559, different functions have been discussed⁶, including photoprotection by charge recombination between Qₚ and P680⁺ (ref. 24). Such a putative pathway could include the haem of cytochrome *b*-559, ChlzD₂ and ChlD₂ (Fig. 3a, b).

The electron transfer between P680⁺ and the manganese cluster (see below) is bridged by the redox-active Tyr161 of D1 (TyrZ). The latter could be located through a protrusion of the electron density in the last turn at the luminal side of helix C in D1. The distance from the manganese cluster is 7.0 Å (Fig. 3a). Tyr161 of D2 (TyrD) is found by a corresponding electron density at helix C in D2. This position is related to that of TyrZ by the pseudo-C2 axis.

The centre-to-centre distance between the P680 chlorophyll molecules and the manganese cluster is 18.5 Å (to PD₁) and 25.1 Å (to PD₂), respectively. The cluster is located on the luminal side (close to the membrane plane) and about 15 Å off the pseudo-C2 axis (Fig. 3b). The electron density of the manganese cluster is bulged in three directions in the form of a ‘Y’ (Fig. 4a, b).

The dimensions of the electron density contoured at the 5-σ level are 6.8 Å × 4.9 Å × 3.3 Å; the long axis (Fig. 4c) is roughly parallel to the CD helix of D1 and tilted roughly 23° against the membrane plane (Fig. 4d). In the three bulges of the electron density, three manganese ions were positioned to form the corners of an isosceles triangle, and a fourth manganese ion was placed near the centre of the triangle. The inter-atomic distances are about 3 Å, which is close to values derived from spectroscopic data²⁵. To verify that the electron density of this cluster is due to the contribution of manganese, anomalous diffraction data were collected with X-ray wavelength close to the manganese edge (1.894 Å). The global maximum of an electron density calculated with these data (not shown) fits well to the model shown in Fig. 4. In addition to manganese ions, Ca²⁺ ions are important for water oxidation; however, these could not be located at the present resolution.

The first three-dimensional structure of a water-oxidizing PSII complex described here shows the spatial distribution of most of the protein subunits and cofactors involved in excitation energy transfer and electron transport. It provides direct information regarding the size, shape and location of the manganese cluster responsible for water oxidation. The present study is an important advance and will stimulate specific functional investigations to obtain new insights into the mechanism of water oxidation by PSII.

**Methods**

Fully active PSII complex from the thermophilic cyanobacterium *S. elongatus* without the phycobilisome antenna was purified and crystallized as described⁵. X-ray diffraction data were collected under cryogenic conditions (100 K) at ESRF beamlines ID2B, ID13, ID14-EH1, ID14-EH2 (EMBL-outstation Grenoble, France), DESY beamlines X11, BW7B (EMBL-outstation Hamburg, Germany), BW6 (MPG-GBF, Hamburg), and Elettra beamline 5.2 (Trieste, Italy). The data were processed using DENZO²⁶ and the CCP4 program suite²⁷ to show a unit cell of dimensions *a* = 130, *b* = 227, *c* = 308 Å and space group *P*2₁2₁2₁. We determined phases to 4.2 Å resolution using multiple isomorphous replacement and anomalous scattering (MIRAS) within the program MLPHARE²⁷. Phase extension from 4.2 to 3.8 Å using the Cd data set was achieved by two-fold non-crystallographic averaging over the local-C2 symmetry, and solvent flattening within the program DM²⁷. We used X-ray diffraction data taken with a wavelength close to the manganese edge (see Table 1) to calculate an anomalous difference Fourier map using MIRAS phases. All electron density map visualization and interpretation was done using the program O²⁸. Figures were generated using BOBSCRIPT²⁹ and Raster3D³⁰. The current model of dimeric PSII contains 2,478 Cα-positions, and two side chains of TyrZ, TyrD, 64 porphyrin molecules assigned to 64 chlorophyll *a* molecules, 4 pheophytins, 4 haems and 2 benzene molecules assigned to 2 plastoquinones, 8 manganese ions and 2 non-haem irons. With these 4,342 atoms, which represent about 10% of the total of about 45,000 atoms for the complete structural model, the crystallographic *R*-factor is 0.59.

Figure 4 Location and orientation of manganese cluster. **a**, Close-up view of the reaction centre, with the electron density of the manganese cluster contoured at 5σ. The view is from the luminal side onto the membrane plane, as in Fig. 2a. **b**, Enlarged view of the electron density of the manganese cluster; **c**, 90° rotated around the horizontal axis (view along the membrane with the luminal side on top). **d**, Orientation of the short and long axes of **c**. The latter is tilted 23° against the luminal side of the membrane plane (hatched line).

Received 21 July; accepted 8 December 2000.

1. Nield, J. *et al.* Three-dimensional structure of *Chlamydomonas reinhardtii* and *Synechococcus elongatus* photosystem II complexes allow for comparison of their oxygen-evolving complex organisation. *J. Biol. Chem.* **275**, 27940–27946 (2000).

2. Rhee, K.-H., Morris, E. P., Barber, J. & Kühlbrandt, W. Three-dimensional structure of photosystem II reaction centre at 8 Å resolution. *Nature* **396**, 283–286 (1998).

3. Zouni, A., Jordan, R., Scholder, E., Fromme, P. & Witt, H. T. First photosystem II crystals capable of water oxidation. *Biochim. Biophys. Acta* **1457**, 103–105 (2000).

4. Witt, H. T. Primary reactions of oxygenic photosynthesis. *Ber. BunsenGes. Phys. Chem.* **100**, 1923–1942 (1996).

5. Zouni, A. *et al.* in *Photosynthesis: Mechanisms and Effects* (ed. Garab, G.) 925–928 (Kluwer Academic, Dordrecht, 1998).

6. Barry, B. A., Boerner, R. J. & de Paula, J. C. in *The Molecular Biology of Cyanobacteria* (ed. Bryant, D. A.) 217–257 (Kluwer Academic, Dordrecht, 1994).

7. Svensson, B. *et al.* A model for the photosystem II reaction center core including the structure of the primary donor P680. *Biochemistry* **35**, 14486–14502 (1996).

8. Xiong, J., Subramaniam, S. & Govindjee. A knowledge-based three dimensional model of the photosystem II reaction center of *Chlamydomonas reinhardtii*. *Photosynth. Res.* **56**, 229–254 (1998).

9. Michel, H. & Deisenhofer, J. Relevance of the photosynthetic reaction center from purple bacteria to the structure of photosystem II. *Biochemistry* **27**, 1–7 (1988).

10. Schubert, W.-D. *et al.* A common ancestor for oxygenic and anoxygenic photosynthetic systems: a comparison based on the structural model of photosystem I. *J. Mol. Biol.* **280**, 297–314 (1998).

11. Harrer, R., Bassi, R., Testi, M. G. & Schäfer, C. Nearest-neighbor analysis of a photosystem II complex from *Marchantia polymorpha* L. (liverwort), which contains reaction center and antenna proteins. *Eur. J. Biochem.* **255**, 196–205 (1998).

12. Ishikawa, Y. *et al.* Turnover of the aggregates and cross-linked products of the D1 protein generated by acceptor-side photoinhibition of photosystem II. *Biochim. Biophys. Acta* **1413**, 147–158 (1999).

13. Rhee, K.-H. *Three-dimensional Structure of Photosystem II Reaction Center by Electron Cryo-microscopy*. Thesis, Univ. Heidelberg (1998).

14. Tomo, T., Enami, I. & Satoh, K. Orientation and nearest neighbor analysis of psbI gene product in the photosystem II reaction center complex using bifunctional cross-linkers. *FEBs Lett.* **323**, 15–18 (1993).

15. Shi, L. X., Kim, S. J., Marchant, A., Robinson, C. & Schröder, W. P. Characterisation of the PsbX protein from photosystem II and light regulation of its gene expression in higher plants. *Plant. Mol. Biol.* **40**, 737–744 (1999).

16. Summer, E. J., Schmid, V. H., Bruns, B. U. & Schmidt, G. W. Requirement for the H phosphoprotein in photosystem II of *Chlamydomonas reinhardtii*. *Plant. Physiol.* **113**, 1359–1368 (1997).

17. Zheleva, D., Sharma, J., Panico, M., Morris, H. R. & Barber, J. Isolation and characterization of monomeric and dimeric CP47-reaction center photosystem II complexes. *J. Biol. Chem.* **273**, 16122–16127 (1998).

18. Mayers, S. R. *et al.* Further characterization of the psbH locus of *Synechocystis* sp. PCC 6803: inactivation of psbH impairs $Q_A$ to $Q_B$ electron transport in photosystem 2. *Biochemistry* **32**, 1454–1465 (1993).

19. Ahmed, A., Tajmir-Riahi, H. A. & Carpentier, R. A quantitative secondary structure analysis of the 33 kDa extrinsic polypeptide of photosystem II by FTIR spectroscopy. *FEBs Lett.* **363**, 65–68 (1995).

20. Ghanotakis, D. F., Tsiotis, G. & Bricker T. M. in *Concepts in Photobiology: Photosynthesis and Photomorphogenesis* (eds Singhal, G. S., Renger, G., Sopory, S. K., Irrgang, K.-D. & Govindjee) 264–291 (Narosa, New Delhi, 1999).

21. Zech, S. G. *et al.* Pulsed EPR measurement of the distance between P680**−** and $Q_A^-$ in photosystem II. *FEBS Lett.* **414**, 454–456 (1997).

22. Schelvis, J. P. M., van Noort, P. I., Aartsma, T. J. & van Gorkom, H. J. Energy transfer, charge separation and pigment arrangement in the reaction center of Photosystem II. *Biochim. Biophys. Acta* **1184**, 242–250 (1994).

23. Ruffle, S., Hutchison, R. & Sayre, R. T. in *Photosynthesis: Mechanisms and Effects* (ed. Garab, G.) 1013–1016 (Kluwer Academic, Dordrecht, 1998).

24. Buser, C. A., Diner, B. A. & Brudvig, G. W. Photooxidation of cytochrome b559 in oxygen-evolving photosystem II. *Biochemistry* **31**, 11449–11459 (1992).

25. Yachandra, V. K., Sauer, K. & Klein, M. P. Manganese cluster in photosynthesis: where plants oxidize water to dioxygen. *Chem. Rev.* **96**, 2927–2950 (1996).

26. Otwinowski, Z. & Minor, W. Processing of X-ray diffraction data collected in oscillation mode. *Methods Enzymol.* **276**, 307–326 (1996).

27. CCP4 Collaborative computational project number 4. The CCP4 suite: programmes for protein crystallography. *Acta Cryst. D* **50**, 760–763 (1994).

28. Jones, T. A., Zou, J. Y., Cowan, S. W. & Kjeldgaard, M. Improved methods for binding protein models in electron density maps and the location of errors in these models. *Acta Cryst. A* **47**, 110–119 (1991).

29. Esnouf, R. M. An extensively modified version of MolScript that includes greatly enhanced coloring capabilities. *J. Mol. Graphics* **15**, 132–134 (1997).

30. Merritt, E. A. & Bacon, D. J. Raster3D—photorealistic molecular graphics. *Methods Enzymol.* **277**, 505–524 (1997).

### Acknowledgements

We thank C. Lüneberg, H. Schmidt and D. DiFiore for technical assistance, and W.-D. Schubert, R. Bittl, E. Schollder, K. Irrgang and P. Jordan for discussions. Beamline assistance at DESY (Hamburg), ESRF (Grenoble) and Elettra (Trieste), and help of M. Burghammer with data collection at the manganese edge is gratefully acknowledged. We thank Egbert J. Boekema for providing us with electron micrographs of PSII. This work was supported by Deutsche Forschungsgemeinschaft, Sonderforschungsbereiche 312 and 498, BMBF (W.S.), and Fonds der Chemischen Industrie (W.S., H.-T.W., N.K. and P.F.). Correspondence and requests for materials should be addressed to W.S. (e-mail: saenger@chemie.fu-berlin.de) or H.-T.W. (e-mail: witt@phosis1.chem.tu-berlin.de). Atomic coordinates are deposited in the Protein DataBank under accession number 1FE1.


---

### corrections

#### Language trees support the express-train sequence of Austronesian expansion

Russell D. Gray & Fiona M. Jordan

*Nature* **405**, 1052–1055 (2000).

There was an error in the geographical character-state mapping in this paper. The authors inadvertently reported the values corresponding to mapping these characters onto the tree in an unordered manner. Correctly ordering the character-state changes according to the ‘express-train’ model does not change the main conclusion of the paper: the express-train model fits much better than would be expected owing to chance. The correct fit is 18 steps, and the fit of the randomly assigned character states now ranges from 95 to 122 (mean 108.4, s.d. 5.1).


---

#### Warm-coding deficits and aberrant inflammatory pain in mice lacking P2X₃

Veronika Souslova, Paolo Cesare, Yanning Ding, Armen N. Akopian, Louise Stanfa, Rie Suzuki, Katherine Carpenter, Daniela Nebenius-Oosthuizen, Andrew J. H. Smith, Emma J. Kidd & John N. Wood

*Nature* **407**, 1015–1017 (2000)

The address of the author Emma J. Kidd was cited incorrectly. Her affiliation should have been given as the Glaxo Institute of Applied Pharmacology, Department of Pharmacology. University of Cambridge, Tennis Court Road, Cambridge CB2 1QJ, UK, which was where her work was carried out. The Welsh School of Pharmacy, Cardiff University, Cardiff CF1 3XF, UK, which was cited as her affiliation in this paper, is her present address.


---

### erratum

#### The protein–protein interaction map of Helicobacter pylori

Jean-Christophe Rain, Luc Selig, Hilde De Reuse, Véronique Battaglia, Céline Reverdy, Stéphane Simon, Gerlinde Lenzen, Fabien Petel, Jérôme Wojcik, Vincent Schächter, Y. Chemama, Agnès Labigne & Pierre Legrain

*Nature* **409**, 211–215 (2001).

The list of interactions between *Helicobacter pylori* proteins that are described and analysed in this paper are available as Supplementary Information on Nature’s World-Wide Web site (http://www.nature.com) or as paper copy from the London editorial office of *Nature*. Proteins are named according to the nomenclature of The Institute for Genomic Research microbial database.
