
The Streptomyces venezuelae pikAV gene contains a transcription unit essential for expression of enzymes involved in glycosylation of narbonolide and 10-deoxymethynolide

Shuo Chen${}^{a}$, Jeffrey B. Roberts${}^{a}$, Yongquan Xue${}^{b}$, David H. Sherman${}^{b}$, Kevin A. Reynolds${}^{a,*}$

${}^{a}$Department of Medicinal Chemistry and Institute for Structural Biology and Drug Discovery, Virginia Commonwealth University, 800 East Leigh Street, Richmond, VA 23219, USA

${}^{b}$Department of Microbiology and Biological Process Technology Institute, University of Minnesota, Box 196, 1460 Mayo Memorial Building, Minneapolis, MN 55455, USA

Received 14 July 2000; received in revised form 3 November 2000; accepted 16 November 2000

Received by A.M. Campbell

---

### Abstract

In *Streptomyces venezuelae*, four polyketide synthase (PKS) polypeptides encoded by *pikAI-pikAIV* are used to generate 10 and 12-membered macrocyclic structures, narbonolide and 10-deoxymethynolide. Sequence analysis suggests these genes are translationally coupled with downstream genes, *pikAV* (encoding a type II thioesterase), *desVIII-desVI* (encoding enzymes responsible for production of the final glycosylated products pikromycin, narbomycin, methymycin and neomethymycin) and *desR* (a resistance gene). Type II thioesterases have been suggested to have an editing function in polyketide biosynthesis and deletion of the corresponding genes often leads to decreased levels of polyketide production. Surprisingly an in-frame deletion of 687 bp of the 843 bp *pikAV* ORF led to a strain SC1022 that produced normal yields of polyketide products, but only in the aglycone form. Plasmid-based expression of the *desVIII-VI* and *desR* in the SC1022 strain completely restored production of glycosylated products, despite the absence of a functional *pikAV* gene product. Under these conditions the PikAV TEII therefore does not play an important role in polyketide biosynthesis, and its function remains an enigma. These observations also demonstrate that the region of *pikAV* DNA deleted in strain SC1022 contains a transcription unit essential for expression of the *des* genes. A sequence alignment of PikAV with members of the highly conserved type II thioesterases revealed a short divergent region at the carboxy terminus, suggesting a region of *pikAV* that might contain such a transcription unit. DNA containing this region of *pikAV* was shown to be able to increase plasmid-based expression of both crotonyl CoA reductase gene (*ccr*) and the erythromycin resistance gene (*ermE*) in *S*. *venezuelae*. © 2001 Elsevier Science B.V. All rights reserved.

Keywords: Thioesterases; Polyketide; Glycosylation; Pikromycin; Venezuela

---

### 1. Introduction

The pikromycin biosynthetic gene cluster (*pik*) of *Streptomyces venezuelae* ATCC 15432 contains a total of five genetic loci (Fig. 1A). The first four open reading frames of the polyketide synthase (PKS) locus (*pikA*) encode four large multifunctional proteins, PikAI-PikAIV. These proteins are typical of a modular type I PKS and contain one loading domain module and six extension modules

Abbreviations: CCR, crotonyl CoA reductase; *ermE*, erythromycin resistance gene; NRPS, non-ribosomal polypeptide synthase; PGM, media for production pikromycin; PKS, polyketide synthase; SCM, media for production of methymycin; SGCP, media for production of both methymycin and pikromycin; TEII, type II thioesterase

* Corresponding author. Tel.: +1-804-828-5679; fax: +1-804-827-3664. E-mail address: kareynol@hsc.vcu.edu (K.A. Reynolds).

required for biosynthesis of the 14-membered macrolide product narbonolide (Fig. 2) (Xue et al., 1998). Under certain growth conditions, the use of an alternative start codon in *pikAIV* results in expression of a N-terminally truncated module unable to catalyze the final extension step, leading to the production of a 12-membered macrolide product 10-deoxymethyonolide (Xue and Sherman, 2000). The *pikB* locus contains a set of genes encoding DesI-VIII, responsible for biosynthesis of desoamine, and a glycosyl transferase. These genes along with the *pikC* locus, which encodes a cytochrome P450 hydroxylase, are responsible for generating the final hydroxylated and glycosylated polyketide products (pikromycin, narbomycin, methymycin and neomethymycin) (Fig. 2).

The *pikA* and part of *pikB* loci contains a >40 kB region with nine genes (*pikAI-desR*) transcribed in the same direction.

A.
$pikR$ $pikA$ $pikB$ $pikC\&D$
0 10 20 30 40 50 (kb)
$pikR2$ $pikR1$ $pikA1$ $pikAII$ $pikAIII$ $pikAV$ $pikC$ $pikD$

B.
$pikAV$ $desVIII$ $desVII$ $desVI$ $desR$ $desV$ $desIV$ $desIII$ $desII$ $desI$

C. SC1022 chromosome
$pikAV$ $desVIII$ $desVII$ $desVI$ $desR$

D. pLZ4
(vector) Partial $pikAV$ $pikAV$ $desVIII$ $desVII$ $desVI$ $desR$ $desV$ $desIV$ $desIII$ $desII$ $desI$ $pikC$ $pikD$

E. pSC67
$PermE^*$ $desVIII$ $desVII$ $desVI$ $desR$

F. pSC64
$BgII$ $3'$ $pikAV$ $180bp$ $ermE$

G. pSC65
$ermE$

H. pSC66
$BamHI$ $BgII$ $pikAV$ $ccr$

Fig. 1. Organization of pikromycin biosynthetic gene cluster in the wild type (A,B) and SC1022 mutant (C). Insert DNA for the major plasmids used in this study is shown (D–H). X represents a destroyed restriction site created by fusing together the $BamHI$ and $BgII$ sites of $pikAV$. pSC61 contains the five genes $pikAV-desR$ without $PermE^*$. pSC71 is the same as pSC66 with $pikAV$ removed.

tion (Fig. 1). The genes either have overlapping stop and start codons or are separated by short junctures (60–150 bp), suggestive of translational coupling. Thus a single polycistronic mRNA may direct the biosynthesis of all of these nine proteins. Such a phenomenon is observed frequently among bacterial operons (Yanofsky, 1992) and has often been proposed for bacterial polyketide biosynthetic gene clusters

(Motamedi and Shafiee, 1998; Staunton and Wilkinson, 1997). Within this putative operon is a small gene $pikAV$, which has overlapping stop and start codons with both $pikAIV$ and $desVIII$. This gene encodes an independent type II thioesterase (TEII). As only the thioesterase domain at the C-terminus of PikAIV is required for catalyzing cyclization of polyketide intermediates into 10-deoxymethylno-

Fig. 2. Structures of 10-deoxymethynolide (A), narbonolide (B), methymycin (C, R₁ = OH and R₂ = H), neomethymycin (C, R₁ = H and R₂ = OH), pikromycin (D, R₃ = OH) and narbomycin (R₃ = H)

lide and narbonolide, the role of TEII is unclear. Similar TEII genes have been found in other type I PKS as well as non-ribosomal peptide synthase (NRPS) enzyme complexes. Inactivation of these genes typically leads to significant decreases in antibiotic yields (Butler et al., 1999; Schneider and Marahiel, 1998) indicating they have some role in the polyketide assembly process. One suggestion is an editing function, releasing abnormal substrates affixed to either modular PKS, or non-ribosomal peptide synthetase enzymes (Butler et al., 1999; Schneider and Marahiel, 1998; Reynolds, 1998). In the case of polyketide synthases it has been further suggested based on model studies that the TEII might remove short-chain acyl groups from the PKS. Such groups may form by premature decarboxylation of enzyme-bound malonate and methylmalonate groups, the carbon chain extension substrates, before they can react with a growing polyketide chain (Heathcote et al., 1999).

In this study we have generated a S. venezuelae mutant (SC1022) with an in-frame deletion of 687 kb of pikAV. Using three different media this strain is able to produce normal levels of polyketide products, but only as aglycones. Plasmid-based expression of the DesVIII–VI and DesR restored pikromycin, narbomycin, methymycin and

neomethymycin production to at least normal levels. Thus under these conditions the complete PikAV is not required for maximal antibiotic production. The pikAV region deleted in the SC1022 strain, on the other hand, clearly contains a transcription element essential for expression of DesVIII-DesR. This element was shown in plasmid systems in S. venezuelae to be able to increase expression of two different reporter genes. These observations indicate that despite the corresponding genetic organization, the nine proteins PikAI-DesR are unlikely to be generated from a single polycistronic mRNA under the control of one promoter.

2. Material and methods

2.1. Bacterial strains, growth conditions, genetic manipulation and sequence analysis

Wild type S. venezuelae ATCC 15439 and its mutants were maintained and propagated at 30°C on sporulation agar plates (per liter): yeast extract 1 g; beef extract 1 g; tryptose 2 g; glucose 10 g; FeSO₄ trace; agar 15 g. For antibiotic production the strains were grown in liquid
culture using SGGP, SCM or PGM media as previously described (Chen et al., 2000; Xue and Sherman, 2000). Plasmids were manipulated in E. coli using standard protocols. Genetic manipulation of the S. venezuelae chromosome was carried out by targeted gene replacement as described previously (Xue et al, 1998). The low-copy number plasmids used in this study are derivatives of pDHS 702 (Xue and Sherman, 2000), which has a SCP2* origin of replication, and two antibiotic resistance markers, thiostrepton and ampicillin. The plasmid pSE34, a derivative of pWHM3 with the ermE* promoter cloned into the EcoRI and BamHI sites, was used to generate high-copy plasmids (Liu and Reynolds, 1999). Target genes introduced into these plasmids and their derivatives were under the control of the pikAI promoter (Xue and Sherman, 2000), ermE* promoter (Liu and Reynolds, 1999), or the desVIII transcription unit. ClusterW alignment was performed with MacVector 6.0 using all default parameters except that open gap penalty was set to 1. All strains and plasmids are summarized in Table 1.

### 2.2. Construction of pikAV in-frame deletion mutant SC1022

A pikAV in-frame deletion mutant was constructed via targeted gene replacement on strain AX912, in which a kanamycin resistance gene has replaced the DNA region encoding the TEI domain and the majority of TEII (Xue and Sherman, 2000). A BglII-XbaI 1kb fragment was PCR amplified with primers (5' → 3'): forward GGCACGAGATCTGCAAC-GACATCTCCGA (BglII) and reverse GGGTCTA-GAGCTGCACCGGCGGGTCGTAGCGGA (XbaI) using a cosmid template pLZ4 (Xue et al, 1998). This fragment covers 118bp of the 3'-DNA sequence of pikAV and approximately 880bp of downstream sequence (desVIII). A natural 2.4kb EcoRI-BamHI fragment covering DNA sequence

---

### 5' region of pikAV was cloned from cosmid pLZ51 (Xue et al, 1998). These two fragments were ligated and subsequently cloned as an EcoRI-XbaI fragment into pKC1139 to generate pSC1022. The 3.4kb insert of pSC1022 contains DNA sequence covering the region pikAIV-desVIII, with a 687 bp BamHI-BglII region of pikAV deleted. This plasmid and the AX912 strain were then used with standard protocols to generate a new strain SC1022 with a complete pikAIV and a shortened pikAV and the kanamycin resistance gene marker was replaced (Fig.1C).

### 2.3. Plasmids and cosmids carrying desVIII-desR

A 6kb BglII-PstI natural fragment containing desVIII, desVII, desR was isolated from pLZ51 (Xue et al, 1998) and cloned first into pSP73 (Promega). The insert was then re-isolated as a BglII-HindIII fragment and cloned into the corresponding sites of pSC24 to give pSC67 (Fig.1E). The pSC24 is a derivative of plasmid pSE34 with a BglII site engineered between XbaI and HindIII of the multiple cloning sites located downstream of the ermE* promoter. Cosmid pLZ4 (Fig.1D) and plasmid pSC67 were used to transform SC1022. To express the desVIII-desR genes on a low-copy plasmid, the natural 6kb BglII-PstI fragment was cloned into BglII-NsiI-digested pDHS702 (Xue and Sherman, 2000) to create pSC60. Subsequently, pSC60 was digested with NdeI and EcoRI and the larger fragment isolated and religated after end-filling with Klenow enzyme and dNTP to create pSC61. In this construct, PpikAI promoter was excised, leaving the desVIII-desR genes downstream of a complete pikAV, the same arrangement as on chromosome.

### 2.4. Plasmids carrying the erythromycin resistance gene (ermE)

Plasmid pDHS702E is a derivative of pDHS702 (Xue and

---

### Table 1  
**Streptomyces venezuelae strains and plasmids used in this study**

| Strain/Plasmid | Description                                                                                   | Source/Reference         |
|----------------|-----------------------------------------------------------------------------------------------|--------------------------|
| SC1022         | BamHI-BglII region of pikAV deleted                                                            | This work                |
| AX912          | pikAV and 3' region of pikAIV replaced by kan' (kanamycin resistance gene)                       | (Xue and Sherman, 2000)   |
| BL3001         | desVI replaced by kan'                                                                        | Zhao et al, 1998)        |
| pLZ4           | Cosmid harboring the region pikAIV-pikD                                                       | (Xue et al, 1998)        |
| pDHS702        | Low copy plasmid based on SCP2* origin                                                        | (Xue and Sherman, 2000)   |
| pDHS702E       | pDHS702 derivative with ermE gene under the control of PpikAI                                   | (Xue, unpublished)        |
| pSC64          | pDHS702E derivative with PpikAI and oriT replaced with 180bp 3' region of pikAV                  | This work                |
| pSC65          | pDHS702E derivative with PpikAI and oriT replaced with NdeI-BamHI fragment containing            | This work                |
|                | the same truncated pikAV as found in SC1022                                                    |                          |
| pSC61          | pDHS702E in which PpikAI, ermE and oriT are replaced with the pikAIV-desR five genes             | This work                |
| pSE34          | High copy number pWHM3 derivative with the ermE* promotor (PermE*)                               | Pfizer Inc.              |
| pHl18          | pSE34 derivative with ccr placed downstream of ermE*                                           | (Liu and Reynolds, 1999)  |
| pSC66          | pHl18 derivative with ermE* replaced with pikAV                                               | This work                |
| pSC71          | pHl18 derivative without the ermE*                                                            | This work                |
| pSC24          | pSE34 derivative with a BglII site downstream of ermE*                                         | This work                |
| pSC67          | pSC24 derivative with desVIII, desVII, desVI downstream of ermE*                               | This work                |
| pSC68          | pSC67 without ermE*                                                                           | This work                |Sherman, 2000), in which the pikAV is replaced with a BamHI-PstI fragment containing the erythromycin resistance gene ermE (Y. Xue, D. Sherman, unpublished data). Using pLZ4 as a template a 180bp NdeI-BamHI fragment covering the entire 3′ region of desVIII (36477–36660 nt) was PCR amplified and used to replace the corresponding fragment (containing oriT and PpikAI) in pDHS702E to generate pSC64 (Fig. 1F). Primers were forward 5′-CCAGCGGGCCCATATGCCTCCGGGCGTA (NdeI), reverse 5′-TCACCGTGGGATCCTCCATCTCTTCGCC (BamHI). The 54 bp NdeI-BglII fragment on pSC64 was replaced with a 1 kb NdeI-BamHI fragment covering gene of TEI domain and 38 bp 5′- of pikAV to generate pSC65. The region upstream of ermE in this plasmid and strain SC1022 thus contained the same shortened pikAIV.

TEII (the full length TEII is predicted to be 281 amino acids).

The parent strain AX912 used in the generation of SC1022, does not produce any detectable levels of pikromycin, narbomycin, methymycin and neomethymycin, or the corresponding aglycones. These aglycones however, were produced in five separate colonies of SC1022 generated from AX912 (Fig. 3, lane 4). Additional mutants (SC1015 and SC1016) containing both pikAIV mutations (in the region encoding the ACP domain) and in-frame deletions of pikAV, have also been shown to produce exclusively aglycone products (Chen et al., 2000).

The production level of 10-deoxymethynolide (12 mg/l) and narbonolide (52 mg/l) in SC1022 was at least equivalent to the level of aglycone production in an S. venezuelae BL3001, a desVI mutant (Zhao et al., 1998). The majority of the pikAV gene encoding TEII is deleted in SC1022 but not BL3001 and demonstrates that under the conditions of these experiments, this TEII does not appear to play a significant role in producing aglycone polyketide products.

Genetic analysis of other type I PKS and non-ribosomal peptide synthetase (NRPS) enzyme complex systems, such as those involved in tylosin, rifamycin, erythromycin, surfactin, lichenysin, and yersiniabactin, have demonstrated the presence of monofunctional thioesterase II-enzymes (TEII) (de Farra et al., 1997; Schneider and Marahiel,

2.5. Plasmids carrying the ccr gene

Plasmid pHl18, a derivative of pSE34 with ccr under the control of the ermE* promoter was reported previously (Liu and Reynolds, 1999). The entire 0.843 kb pikAV ORF was amplified as an EcoRI-NdeI fragment using pLZ4 as template and the following primers: forward 5′-GGCTCTA-GATCGAATTCATGAGGGGGGCGGG (EcoRI), reverse 5′-AGGGCCCCCGTCATATGGTGCGGTACC (NdeI). This fragment was used to replace the corresponding region (containing the ermE* promoter) of pHl18 to generate pSC66 (Fig. 1G). A second plasmid pSC71 was generated from pHl18 by removing the EcoRI-NdeI region containing the ermE* promoter, end-filling with Klenow enzyme and religating. This plasmid contains the ccr gene without either the ermE* promoter or the desVIII* transcription unit.

2.6. Antibiotic-production fermentation, polyketide products analysis and assays

Fermentation of S. venezuelae strains and analysis of products by TLC and HPLC was done as described elsewhere (Chen et al., 2000). CCR enzyme assays were performed as previously described (Liu and Reynolds, 1999).

3. Results and discussion

3.1. Construction and characterization of the pikAV in-frame deletion mutant

Targeted gene replacement was used to generate an in-frame deletion of pikAV in S. venezuelae between the two internal restriction sites (BamHI and BglII). The resulting strain SC1022 like the wild type strain has a complete pikAIV and thus encodes the last module of the polyketide synthase along with a TE domain. The shortened downstream pikAV in SC1022 has overlapping stop and start codons with pikAIV and desVIII and encodes a protein consisting of only the first 13 and last 39 amino acids of

Fig. 3. TLC analyses of organic extracts from S. venezuelae fermentations. Fermentation media: Methymycin producing media SCM (lanes 1–2); pikromycin/methymycin producing media SGGP (lanes 3–7). Samples: SC1022/pSC67 (lane 1); wild type (lane 2); SC1022/pSC67 (lane 3); SC1022 (lane 4); SC1022/pLZ4 (lane 5); SC1022/pSC68 (lane 6); SC1022/pSC61 (lane 7).1998; Butler et al., 1999; Konz et al., 1999). These TEII proteins all have significant predicted amino acid sequence similarities to the acyl-CoA thioesterases used in various fatty acid biosynthetic processes, and have been proposed to perform a housekeeping function by liberating mischarged PKS or NRPS modules (Reynolds, 1998; Butler et al., 1999). For the case of a PKS enzyme complex, this role might be to remove acetate and propionate groups from extension modules (Heathcote et al., 1999; Doi-Katayama et al., 2000). These groups, which may have been formed as a result of premature decarboxylation of enzyme-bound 2-methylmalonate and malonate (prior to reacting the acylthioester polyketide intermediate bound to the KS domain), would block further extension of the carbon chain by the PKS. Such a function has been supported by model studies with a purified recombinant Streptomyces fradiae TEII (Heathcote et al., 1999) and by the observation that deletion of the TEII genes in tylosin producer S. fradiae (tylOor previous tyl orf5) (Butler et al., 1999) and the rifamycin producer Amycolatopsis mediterranei S699A (rifR) resulted in 85% and 30–60% decreases in polyketide production, respectively (Doi-Katayama et al., 2000).

We have previously reported that a S. venezuelae mutant AX905, in which the pikAV gene had been insertionally inactivated with a kanamycin resistance gene, produces less than 5% of the polyketide products than the wild type strain (Xue et al., 1998). Under some circumstances this strain can also produce normal levels of polyketide products (data not shown). In the current study, an in-frame deletion of the pikAV gene encoding a TEII does not under the conditions of our experiment have any measurable effect on polyketide production. A similar puzzling scenario has been observed in the surfactin NRPS biosynthetic gene cluster of Bacillus subtilis, where deletion of the srfA-TE gene encoding a type II thioesterase has been reported to result both in 85% (Schneider and Marahiel, 1998), and 0% decreases (Cosmina et al., 1993) in surfactin production. In the latter case, it has been suggested that the presence of another enzyme in the cell substitute for the loss of SrfA-TE protein (Cosmina et al., 1993). The exact functions of TEII proteins associated with PKS and NRPS biosynthetic processes thus remains to be determined, as do the specific conditions under which this function becomes a rate limiting step in the biosynthetic process.

(Xue and Sherman, 2000) (data not shown), indicating that a cis element involved in the glycosylation process was missing. Analysis of the in-frame fused pikAV region generated in SC1022 did not indicate the presence of any messenger RNA secondary structure that might block translation. Thus the exclusive production of aglycone products in SC1022 is most likely the result of the deletion of a region of pikAIV containing a transcription unit involved in expression of genes in the pikB locus. Support for this hypothesis was provided by demonstrating that introduction into SC1022 of a cosmid pLZ4 containing the complete pikAV and the entire pikB locus but not the PpikAI promoter, resulted in a complete restoration of formation of glycosylated products (Fig. 3, lane 5). A similar restoration of pikromycin, narbomycin, methymycin and neomethymycin production was accomplished when the pSC67 (Fig. 1E), containing just the desVIII-VI and desR genes under control of the ermE* promoter, was introduced into SC1022 (Fig. 3, lane 1 and 3). Expression of these specific genes in the pikB locus is, therefore, dependent upon the transcriptional unit within pikAV.

A sequence alignment of the various TEII enzymes involved in various PKS and NRPS processes has previously shown the presence of a highly conserved Gly-x-Ser-x-Gly putative active site motif in the 80–90 amino acid region (Schneider and Marahiel, 1998). Approximately 130–140 amino acids to the C-terminal from this motif is a highly preserved His residue within a Gly-x-His motif. Mutagenesis studies have demonstrated that both the serine and histidine residues are important for catalytic activity of type II thioesterases (Pazirandeh et al., 1991). An alignment of the PikAV, RifR, and TylO, type II thioesterases showed the presence of both of these motifs along with strong sequence similarity (almost 70% identity in at least two of the sequences) in the first 245 amino acids (Fig. 4). At the C-terminus, however, the sequence similarity is significantly lower and in the case of PikAV there is an additional region of 17 amino acids that have no clear sequence similarity at either TylO or RifR. The in-frame deletion of pikAV in SC1022 resulted in loss of DNA encoding the first four amino acids of this region as well as the majority of the conserved region of PikAV (Fig. 4). These observations suggest that the region of pikAV encoding the first four amino acids of this divergent region of PikAV might contain part of a transcription unit responsible for expression of components of the pikB locus. A comparison of this region with sequences of apparent streptomycetes promoters (Strohl, 1992) indicated a potential promoter region centered around this BglII site.

### 3.2. Cosmid complementation of SC1022 and TEII sequence comparison

The finding that SC1022, SC1015 and SC1016, grown in three different media SCM, PGM and SGGP, produce only aglycone products was surprising because these mutants contain only an in-frame deletion of pikAV and no alterations to the pikB locus (involved in biosynthesis and subsequent transfer of desosamine). As expected formation of glycosylated products was not restored by providing a complete pikAV expression system in trans, (pDHS702)

### 3.3. Verification of the activity of the desVIII* transcription unit

Two different SCP2* origin low-copy plasmids containing the promoterless erythromycin resistance gene ermE were constructed. In pSC64 a 180 bp DNA sequence of the 3'
ents

PikAV V T D RP L N V DS G L W I RRF H PAP NS - A V RL VCL PHAG
RifR M H RP - - EA E K WLR RF E R AP DA - RA RL VCL PHAG
TylO R P . . . . WLR RF P AP . . A H RL VCF PHAG

B

PikAV GSASY FFR FS E ELHPS - VEALSVQYPRQRRAEP
RifR GSASF FFP LA K ALAPA - VEV LAVQYPGRQDRREHP
TylO GSARA RAY R PFA L ELAAAAPA VE . LAVQYPGRQDRRK EP

PikAV CLESV EEELAEHVVA AATEP WWQE - GR- LAF FGHSL
RifR PVDSI GGLTN RLLL EV LRP FG D - - - RPLALFGHSM
TylO FAR TL EEELAERVL P E LRL D APD G V PVALFGHSM

PikAV GASVAF ETARIL EQ RHGV RP EGL YVSGRRAPS LAP
RifR GAIIG YELALRM PE AGL PAP VH LF ASGRRAPS RYR
TylO GAVVAYETARLL HR SG A PRP GL . SGRRAPS R

PikAV DRLVHQLD DR AF LAEIRRL S GTDE RFL QDD ELLRL
RifR D D VR GAS DER LV AELRKL G GSD AAM LADPELLAM
TylO TE TA HL LG DRE LL AEIRRL Q GTD PGA LAD EELL RM

PikAV VLPA LRS DY K A ETYLHR PS AKLT CPVMALAGDRD
RifR VLPAIRSDY R A VETYRH E PGR RVDCPVTVTGDHD
TylO VLPAIR GDY A AVGRYRH V PG R . CPVTVTFTGD D

PikAV PKAPLN E VAE WRR HTSGPFCL R A Y SGGHFYLNDQW
RifR PRVSVEAR AWEEHTTGP AD LRVL PGGHFFLV DQ -
TylO PNVTLP EAE AWREL TTGA FA LRVF PGGHFYLNDQ . LNDQ

PikAV HEICNDISDHLLVTRG AP DAR VVQPPTEKLAGALK
RifR - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
lapping or closely spaced stop and start codons and no
obvious termination sequence. Such an analysis would
suggest that expression of the *pikAI-pikR* genes are driven
from a single polycistronic mRNA (Fig. 1A). It is now clear
that an additional transcriptional unit, most likely a promo-
ter, located within *pikAV*, is essential for expression of the
last four genes of this putative operon.

A similar observation has been made in the erythromycin
biosynthetic gene cluster where sequence analysis would
suggest that *eryG* is the sixth gene of a six gene biosynthetic
operon (Weber et al., 1989). Experimental evidence,
however, has clearly demonstrated that the preceding gene
*eryBII* (required for biosynthesis of the desosamine moiety
of erythromycin) contains a promoter essential for expres-

![image](https://i.imgur.com/1234567.png)

Fig. 5. Growth of *S. venezuelae* strains after 2 days on solid sporulation agar
medium in the presence of erythromycin (400 μg/ul). A. AX912/pSC64, B.
AX912/pSC65, C. AX912/pSC61, D. wild type, E. AX912/pDHS702E.

stream and that a region within the *BamHI-BglII* region is
essential for this activity.

The *Streptomyces collinus* crotonyl-CoA reductase gene
(*ccr*) was used as a reporter in order to quantitatively
compare the effectiveness of the *desVIII** transcription
unit and *PermE**, a strong constitutive promoter. The
CCR enzyme catalyzes the conversion of NADPH-depen-
dent reduction of crotonyl-CoA to butyryl-CoA and can be
readily followed spectrophotometrically (Wallace et al.,
1995). No significant CCR activity could be detected in
cell extracts generated from 12–64 h fermentations of *S.*
*venezuelae* SC1022. When a high-copy plasmid pSC71
containing a promoterless *ccr* was introduced into this
strain, low levels of CCR activity could be observed
throughout the fermentation process (Fig. 6A). The level
of CCR activity was approximately one-fold higher with
SC1022 strains carrying a related plasmid pSC66 (Fig.
6A), in which the *ccr* was placed after *pikAV* (Fig. 1H). In
contrast to this observation, almost a 100-fold increase in
CCR activity was observed for cell-extracts of SC1022
transformed with pHL18, which has *ccr* expression under
control of the *ermE*** (Fig. 6B). The *ermE*** promoter works
relatively effectively in *S.* *venezuelae* and is significantly
more effective at enhancing expression of downstream
genes than the *desVIII** transcription unit.

Genes involved in most aspects of secondary metabolic
processes in bacteria and fungi are often clustered (Martin
and Liras, 1989; Minto and Townsend, 1997). These genes
tend to form operons, ‘the genetic unit of coordinated
expression’(Jacob and Monod, 1961), which is believed to
provide the advantage of coordinated regulation and
production of functionally related elements. Convention-
ally, a series of genes transcribed in one direction are
often described as a biosynthetic operon if they have over-
![image](https://i.imgur.com/8901234.png)
Fig. 6. CCR activity in cell extracts of *S.* *venezuelae* SC1022/pSC71
(A, -♦-), SC1022/pSC66 (A, -■-), SC1022/pSE34 (B, -▲-) and SC1022/
pHL18 (B, -●-) grown in a 64 h fermentation. No detectable levels of CCR
activity could be observed for the SC1022 strain alone. Activities were
measured (Liu and Reynolds, 1999) in triplicate from separate fermenta-
tions of three different colonies of each transformant.

expression of the gene located 30 nt downstream, *eryG* (encoding a hydroxylase). The reason for the presence of this promoter and the putative *desVIII* promoter is not clear. In the latter case a *desVIII* promoter might allow independent control of genes in the *pikA* and the *pikB* locus. Another possible explanation for the existence of this promoter is that expression of genes from the 3′-end of a long mRNA transcript that starts from the *PpikAI* promoter might be inefficient, possibly due to degradation. Ribosome re-initiation varies in efficiency according to the distance between adjacent stop and start codons as well as the strength of the binding signal, the Shine-Dalgarno site (Richardson, 1991; Stanssens et al., 1986). It has been suggested that a lower frequency of ribosome initiation may result in mRNA termination or degradation due to poor ribosome coverage (von Gabain et al., 1983; Emory and Belasco, 1990). Interestingly, analysis of the DNA sequence 11 nucleotides before the *desVIII* start codon does reveal a putative Shine-Dalgarno sequence region (GGCAGAA) predicted to have a low binding strength (∆G > −5 kcal/mol) in the 3′ region of the 16S rRNA of *S. lividans*. A similar analysis predicts significantly higher binding affinities (∆G ≤ −10 kcal/mol) for Shine-Dalgarno sequences regions of the preceding ORFs (GGAGACGGAG for *pikAI*) in the *pikAI* locus (Strohl, 1992). It has previously been suggested that expression of the proximal and distal genes in the cobalamin biosynthetic gene cluster of *Salmonella typhimurium* is established by a combination of varying degrees of translational coupling in concert with weak internal, constitutive promoters (Roth et al., 1993). A similar scenario appears to be the case for pikromycin biosynthesis, and possibly other secondary metabolic biosynthetic gene clusters.

### 3.4. Overexpression of the DesVIII does not enhance pikromycin, narbomycin, methymycin and neomethymycin production

In certain microorganisms, the presence of plasmids often negatively impacts polyketide production (Thoma et al., 1991). This phenomenon is not observed in *S. venezuelae* where the total amounts and relative ratios of glycosylated products (pikromycin, narbomycin, methymycin and neomethymycin) and the corresponding aglycone products (narbonolide/10-deoxymethylenolide) using SC1022/pLZ4, SC1022/pSC67, and SC1022/pSC61 were indistinguishable from that observed with the wild type strain. The level of expression of DesVIII-DesR in these systems is likely to vary dramatically with the greatest levels predicted with use of the high copy plasmid pSC67 and the strong constitutive *ermE* promoter. Significantly lower levels of expression are predicted for SC1022 carrying pSC61 (a low copy plasmid contain the same *des* genes under the control of the *desVIII* transcription unit), or the wild type strain. Under the conditions of all of these experiments, the products were predominantly glycosylated, with the same low levels of aglycone products (Fig. 3). In the wild type strain the

---

expression of the DesVII glycosyltransferase and the desosamine biosynthetic enzymes DesVIII and DesVI, from the *desVIII* transcription unit is clearly sufficient and not the rate limiting step in the formation of pikromycin, narbomycin, methymycin and neomethymycin.

## 4. Conclusions

1. An *S. venezuelae* SC1022 mutant with an in-frame deletion of *pikAV* encoding a type II thioesterase does not significantly impact the overall yields of polyketide products. The role of TEII and the conditions under which it plays a significant role in maintaining polyketide production remains to be determined.
2. The *pikAV* gene contains a transcription unit whose presence is essential for expression of the glycosyl transferase (DesVIII) and desosamine biosynthetic enzymes (DesVI and DesVII) involved in converting 10-deoxymethylenolide and narbonolide to pikromycin, narbomycin, methymycin and neomethymycin. In the wild type strain, the levels of these enzymes are not the limiting factor(s) contributing to the presence of low levels of aglycone products.
3. Pikromycin, narbomycin, methymycin and neomethymycin production in *S. venezuelae* is not negatively impacted by the presence of various plasmids. Plasmid-based combinatorial biosynthetic approaches for generating novel polyketides can therefore be pursued.

## Acknowledgements

The authors thank L. Zhao and H.-w. Liu (University of Minnesota Department of Chemistry) for providing pLZ4, and the *S. venezuelae* BL3001 strain. We thank Beom Seok Kim for reading the manuscript. This work was supported by a grant from the NIH (GM48562) to DHS and KAR.

## References

Butler, A.R., Bate, N., Cundliffe, E., 1999. Impact of thioesterase activity on tylosin biosynthesis in *Streptomyces fradiae*. Chem. Biol. 6, 287–292.

Chen, S., Xue, Y., Sherman, D.H., Reynolds, K.A., 2000. Mechanisms of molecular recognition in the pikromycin polyketide synthase Chem. Biol. (in press).

Cosmina, P., Rodriguez, F., de Ferra, F., Grandi, G., Perego, M., Venema, G., van Sinderen, D., 1993. Sequence and analysis of the genetic locus responsible for surfactin synthesis in *Bacillus subtilis*. Mol. Microbiol. 8, 821–831.

de Farra, F., Rodriguez, F., Tortora, O., Tosi, C., Grandi, G., 1997. Engineering of peptide synthetases: key role of the thioesterase-like domain for efficient production of recombinant peptides. J. Biol. Chem. 272, 25304–25309.

Doi-Katayama, Y., Yoon, Y.J., Choi, C.-Y., Yu, T-W., Floss, H.G., Hutchinson, C.R., 2000. Thioesterase and the premature termination of poly-

ketide chain elongation in rifamycin B biosynthesis by *Amycolatopsis mediterranei* S699. J. Antibiot. 53, 484–495.

Emory, S.A., Belasco, J.G., 1990. The ompA 5′ untranslated RNA segment functions in *Escherichia coli* as a growth-rate-regulated mRNA stabilizer whose activity is unrelated to translational efficiency. J. Bacteriol. 172, 4472–4481.

Heathcote, M.L., Staunton, J., Leadlay, P.F., 1999. Substrate specificity and putative roles of thioesterases in polyketide biosynthesis. Abstract of Eleventh International Symposium on the Biology of the Actinomycetes, Sissi, Crete, October, 1999, p. 187.

Jacob, F., Monod, J., 1961. Genetic regulatory mechanisms in the synthesis of proteins. J. Mol. Biol. 3, 256–318.

Konz, D., Doekel, S., Marahiel, M.A., 1999. Molecular and biochemical characterization of the protein template controlling biosynthesis of the lipopeptide lichenysin. J. Bacteriol. 181, 133–140.

Liu, H., Reynolds, K.A., 1999. Role of crotonyl coenzyme A reductase in determining the ratio of polyketides monensin A and monensin B produced by *Streptomyces cinnamonensis*. J. Bacteriol. 181, 6806–6813.

Martin, M.F., Liras, P., 1989. Organization and expression of genes involved in the biosynthesis of antibiotics and other secondary metabolites. Ann. Rev. Micro. 43, 173–206.

Minto, R.E., Townsend, C.A., 1997. Enzymology and molecular biology of aflatoxin biosynthesis. Chem. Rev. 97, 2537–2555.

Motamedi, H., Shafiee, A., 1998. The biosynthetic gene cluster for the macrolactone ring of the immunosuppressant FK506. Eur. J. Biochem. 256, 528–534.

Pazirandeh, M., Chirala, S.S., Wakil, S.J., 1991. Site-directed mutagenesis of the recombinant thioesterase domain of chicken fatty acid synthase expressed in *Escherichia coli*. J. Biol. Chem. 266, 20946–20952.

Richardson, J.P., 1991. Preventing the synthesis of unused transcripts by Rho factor. Cell 64, 1047–1049.

Reynolds, K.A., 1998. Combinatorial biosynthesis: lesson learned from nature. Proc. Natl. Acad. Sci. USA 95, 12744–12746.

Roth, J.R., Lawrence, J.G., Rubenfield, M., Kieffer-Higgins, S., Church, G.M., 1993. Characterization of the cobalamin (vitamin B12) biosynthetic genes of *Salmonella typhimurium*. J. Bacteriol. 175, 3303–3316.

Schneider, A., Marahiel, M.A., 1998. Genetic evidence for a role of thioesterase domains, integrated in or associated with peptide synthetases, in

non-ribosomal peptide biosynthesis in *Bacillus subtilis*. Arch. Microbiol. 169, 404–410.

Stanssens, P., Remaut, E., Fiers, W., 1986. Inefficient translation initiation causes premature transcription termination in the *lacZ* gene. Cell 44, 711–718.

Staunton, J., Wilkinson, B., 1997. Biosynthesis of erythromycin and rapamycin. Chem. Rev. 97, 2611–2629.

Strohl, W., 1992. Compilation and analysis of DNA sequences associated with apparent streptomycete promoters. Nucleic Acids Res. 20, 961–974.

Thoma, D.I., Jonathan, H.C., Baumberg, S., Jones, C.A., Rudd, B.A.M., 1991. Plasmid effects on secondary metabolite production by a streptomycete synthesizing an anthelmintic macrolide. J. Gen. Microbiol. 137, 2331–2337.

von Gabain, A., Belasco, J.G., Schottel, J.L., Chang, A.C., Cohen, S.N., 1983. Decay of mRNA in *Escherichia coli*: investigation of the fate of specific segments of transcripts. Proc. Natl. Acad. Sci. USA 80, 653–657.

Wallace, K.K., Bao, Z., Dai, H., McArthur, H.A.I., Reynolds, K.A., 1995. Purification of crotonyl CoA reductase from *Streptomyces collinus* and cloning, sequencing, expression and expression of the corresponding gene in *Escherichia coli*. Eur. J. Biochem. 233, 954–962.

Weber, J.M., Schoner, B., Losick, R., 1989. Identification of a gene required for the terminal step in erythromycin A biosynthesis in *Saccharopolyspora erythraea*. Gene 75, 235–241.

Xue, Y., Sherman, D.H., 2000. Alternative modular polyketide synthase expression controls macrolactone structure. Nature 403, 571–574.

Xue, Y., Zhao, L., Liu, H.-w, Sherman, D.H., 1998. A gene cluster for macrolide antibiotic biosynthesis in *Streptomyces venezuelae*: architecture of metabolic diversity. Proc. Natl. Acad. Sci. USA 95, 12111–12116.

Yanofsky, C., 1992. Transcriptional regulation: elegance in design and discovery. In: Mcknight, S.L., Yamamoto, K.R. (Eds.), Transcription Regulation, Vol. 1. Cold Spring Harbor Laboratory Press, Plainview, New York, pp. 3–23.

Zhao, L., Sherman, D.H., Liu, H.-w, 1998. Biosynthesis of desosamine: construction of a new methymycin/neomethymycin analogue by deletion of a desosamine biosynthetic gene. J. Am. Chem. Soc. 120, 10256–10257.
