
doi:10.1006/jmbi.2001.4451 available online at http://www.idealibrary.com on IDEAL J. Mol. Biol. (2001) 307, 93–105

**JMB**

The hydH/G Genes from *Escherichia coli* Code for a Zinc and Lead Responsive Two-component Regulatory System

Susanne Leonhartsberger${}^{1}$, Andrea Huber${}^{1}$, Friedrich Lottspeich${}^{2}$ and August Böck${}^{1*}$

${}^{1}$Lehrstuhl für Mikrobiologie der Universität München Maria-Ward-Str. 1a D-80638 Munich, Germany

${}^{2}$Max-Planck-Institut für Biochemie, Abteilung Proteinchemie, Am Klopferspitz D-82152 Martinsried Germany

The hydH/G genes from *Escherichia coli* code for a two-component regulatory system that has been implicated in the regulation of hydrogenase 3 formation. In a detailed study of the function of HydH/G employing $hycA'$-$'lacZ$ reporter gene fusions, it was shown that HydH/G indeed led to a stimulation of activation of the $hycA$ promoter responsible for hydrogenase 3 synthesis but only when $hydG$ is overexpressed from a plasmid in a strain lacking FhlA. Since the stimulation was not observed with an $fdhF'$-$'lacZ$ fusion, and since it was independent from a functional $hydH$ gene product, it must be considered as unspecific cross-talk. An extensive search for the actual physiological signal of HydH/G showed that the system responds to high concentrations of zinc or lead in the medium. Expression of $zraP$, a gene inversely oriented to $hydH/G$ whose product seems to be involved in acquisition of tolerance to high $Zn^{2+}$ concentrations, is stimulated by high $Zn^{2+}$ and $Pb^{2+}$ concentrations and this stimulation requires both HydH and HydG. Purified HydG in the presence of phosphoryl donors binds to a region within the $zraP-hydHG$ intergenic region that is characterised by two inverted repeats separated by a 14 bp spacer. Putative $-12/-24 \sigma^{54}$-dependent promoter motifs are present upstream of both the $zraP$ and the $hydHG$ transcriptional units; in accordance, transcription of $zraP$ is strictly dependent on the presence of a functional $rpoN$ gene. The expression of $hydH/G$ is autoregulated: high $Zn^{2+}$ and $Pb^{2+}$ concentrations lead to a significant increase of the HydG protein content which took place only in a $hydH^{+}$ genetic background. Since HydH binds to membranes tightly, it is assumed that the HydH/G system senses high periplasmic $Zn^{2+}$ and $Pb^{2+}$ concentrations and contributes to metal tolerance by activating the expression of $zraP$. The redesignation of $hydH/G$ as $zraS/R$ is suggested.

© 2001 Academic Press

Keywords: $hydG$; hydrogenase; zinc tolerance; $zraP$; two-component system

---

**Introduction**

Escherichia coli has the capacity to synthesize four different hydrogenases, designated hydrogenase 1, 2, 3 and 4. Hydrogenase 1 is encoded by the genes of the $hya$ operon (Menon et al., 1990) and appears to be involved in the respiratory recycling of fermentatively produced hydrogen (Sawers & Boxer, 1986; Francis et al., 1990). The physiological role of hydrogenase 2, encoded by the $hyb$ operon genes, resides in H₂ oxidation coupled to fumarate reduction (Menon et al., 1994). Hydrogenase 3 (gene designation $hyc$) (Böhm et al., 1990) is part of the formate hydrogen lyase complex and is involved in fermentative hydrogen production (Peck & Gest, 1957). Hydrogenase 4 ($hyf$ genes) has been characterized until now only at the genetic level (Andrews et al., 1997). Apart from the genes organized in the structural gene transcriptional units, there is a set of genes (designated $hyp$)

Present address: A. Huber, GPC Biotech AG, Fraunhoferstrasse 20, 82152 Martinsried, Germany E-mail address of corresponding author: august.boeck@lrz.uni-muenchen.de

© 2001 Academic Press

whose products are involved in the maturation of the active enzymes (Lutz *et al.*, 1991).

In connection with their physiological role, the syntheses of the hydrogenases 1, 2 and 3 are differentially regulated (for a review, see Sawers, 1994). Formation of hydrogenases 1 and 2 is induced under conditions of anaerobic respiration where expression of the *hya* genes is subject to control by the *arcAB* and *appY* regulatory genes (Brondsted & Atlung, 1994; Richard *et al.*, 1999) and that of hydrogenase 2 by *fnr* and *arcAB* (Richard *et al.*, 1999). The genes coding for the components of hydrogenase 3 are part of the formate regulon (Rossmann *et al.*, 1991): expression is activated by the transcriptional activator FhlA (Sankar *et al.*, 1988; Schleensog & Böck, 1990) and requires formate as inducer and an upstream activatory sequence (UAS) plus a –12/–24 promoter recognized by RNA polymerase·σ<sup>54</sup> holoenzyme (Birkmann *et al.*, 1987; Lutz *et al.*, 1990). In contrast, the *hya* and *hyb* transcriptional units are transcribed from σ<sup>70</sup>-dependent promoters.

In contrast to the more or less well-defined functions of the structural and regulatory genes for the four hydrogenase isoenzymes, there are two genes, *hydH* and *hydG*, whose products have been postulated to play a role in the regulation of hydrogenase 3 (Stoker *et al.*, 1989). They are members of the family of the two-component regulatory systems; however, neither the physiological signal nor the molecular mechanism of their action has been studied. During a project in cloning the *fhlA* gene from *Klebsiella oxytoca* we discovered that HydG from this organism strongly activates expression of the genes of the formate regulon from *E. coli* (Leonhartsberger *et al.*, 2000). Here, we show that HydG has the interesting property of unspecifically activating transcription from –12/–24 promoters in the absence of defined UAS motifs. Its described action on the formation of the hydrogenase 3 thus is fortuitous. An extensive search for the physiological signal revealed that the HydH/G system is responsive to high Zn<sup>2+</sup> concentrations in the medium and that it regulates the expression of the *zraP* gene, which codes for a periplasmic Zn<sup>2+</sup>-binding protein (Noll *et al.*, 1998).

### Results

#### Cloning and sequencing analysis of *hydH* and *hydG* from *K. oxytoca* M5a1 and *E. coli*

In our project on the analysis of the mechanism of the action of FhlA of *E. coli*, we had initiated the cloning and expression of the *fhlA* genes from several other enteric bacteria, including *K. oxytoca*. For this purpose, a gene bank of chromosomal DNA from this organism (Fiedler *et al.*, 1996) was used to transform *E. coli* SA22 cells (∆*fhlA*), which carry a *hycA′-′lacZ* fusion in the λ attachment site. Transformant colonies were screened for the expression of the reporter gene system on MacConkey-lactose plates. From 70,000 clones, six *lac<sup>+</sup>* clones were retrieved; sequence analysis showed that two of them carried *fhlA* from *K. oxytoca*, one the *lacZ* from this organism and two others the complete *hydG* plus part of the *hydH* gene. Plasmid pSKOG (2.7 kb insert starting at base 519 of *hydH* and reaching 400 bp downstream of *hydG*) was chosen for further studies. Using primers derived from the *E. coli* *hydH* sequence and from the sequence stretch carried by pSKOG, the *hydH* sequence from *K. oxytoca* was completed and the 5′ sequence of *hydH* was determined (GenBank accession number AF305914).

The derived amino acid sequence of HydH from *K. oxytoca* contains 462 residues and shares 64% identity with *E. coli* HydH. Analysis of the hydrophathy profile (Kyte & Doolittle, 1982) indicates the existence of at least two transmembrane domains (results not shown). The derived HydG sequence is 443 residues in size and is 83% identical with the *E. coli* sequence (in all cases we refer to the sequences determined by Blattner *et al.*, 1997, which show various differences from that reported by Stoker *et al.*, 1989). As already reported, HydH and HydG, amongst all the two-component systems known, exhibit the closest similarity to NtrB and NtrC (Stoker *et al.*, 1989). Since expression of *K. oxytoca hydG* led to the apparent activation of the *hycA′-′lacZ* reporter gene fusion, which is normally under the control of FhlA, the sequences of the two regulatory proteins were aligned (data not shown). It is evident that statistically significant similarity covers only the central domain.

---

**The activation of the *hycA′-′lacZ* expression by HydG is the consequence of unspecific cross-talk**

Because the *hydH/G* gene products had been addressed as specifically regulating the formation of hydrogenase 3 from *E. coli* (Stoker *et al.*, 1989) and because of the high activation capacity of the *K. oxytoca* HydG, we have analysed the regulatory effect in more detail, since the possibility existed that HydH/G constituted a regulatory circuit additional to that exerted by FhlA. For this purpose, the *hydH* and *hydG* genes from *E. coli* were cloned and mutants of *E. coli* were constructed that possess in-frame deletions in their *hydH* or *hydG* genes (see Materials and Methods). First, the effect of HydG on the expression of the chromosomally located *hycA′-′lacZ* reporter gene fusion in a ∆*fhlA* genetic background was analysed. It is evident that the activation by overexpression of the heterologous *hydG* gene is more pronounced than that of the *E. coli* gene (Figure 1). Surprisingly, the activity of HydG was dependent on anoxic conditions and expression was independent of the presence of a functional *hydH* gene.

Next it was studied whether mutational inactivation of the *hydG* or *hydH* genes influenced FhlA-dependent expression of the *hycA′-′lacZ* reporter gene fusion (Figure 2). In the *fhlA<sup>+</sup>* genetic back-

hydH/G Code for a Two-component System

Figure 1. Unspecific activation of a $hycA^{\prime}$-$lacZ$ fusion by HydG. SA22 ($\Delta fhlA$, $hycA^{\prime}$-$lacZ$) or SA22$\Delta$H ($\Delta fhlA$, $\Delta hydH$, $hycA^{\prime}$-$lacZ$) cells harbouring the different plasmids (pSU2719: vector control, pSECg: $hydG_{Ec}$, pSKOG: $hydG_{Ko}$) were grown in TGYEP medium at pH 6.5 under oxic or anoxic conditions. Values are averages of two experiments assayed in triplicate.

Figure 2. Activation of a $hycA^{\prime}$-$lacZ$ reporter gene fusion in $hydG$ and $hydH$ deletion strains. Cells were grown in TGYEP medium at pH 6.5 under oxic or anoxic conditions in the presence or absence of 30 mM sodium formate. Values are averages of three experiments assayed in triplicate.

HydH/G are involved in the zinc-dependent expression of the $zraP$ gene

Since regulatory genes are frequently located in the proximity of their target genes, it was tested whether $zraP$, a gene separated by a 237 bp intergenic region from $hydH$, might be under the control of HydH/G. ZraP is a periplasmic protein from $E$. coli involved in the tolerance to high $Zn^{2+}$ concentrations (Noll et al., 1998). A $zraP^{\prime}$-$lacZ$ fusion was constructed and integrated into the $\lambda$ attachment site of MC4100 and of derivatives thereof carrying deletions in $hydH$ and $hydG$. They were used to analyse $Zn^{2+}$-dependent expression of the fusion (Table 1).

Intriguingly, $Zn^{2+}$ concentrations above 0.2 mM led to an increase of the expression of $zraP^{\prime}$-$lacZ$ and this stimulation in the presence of the metal did not occur when either $hydH$ or $hydG$ carried a

Table 1. Activation of the expression of a $zraP^{\prime}$-$lacZ$ fusion by the HydH/G two-component system in response to $ZnCl_2$

| [ZnCl$_2$] (mM) | 0          | 0.05        | 0.2         | 0.5         | 1           |
|------------------|------------|-------------|-------------|-------------|-------------|
| MCzra/pSU2719    | 4 ± 2      | 6 ± 1       | 17 ± 1      | 80 ± 9      | 1052 ± 493  |
| MC$\Delta$Gzra/pSU2719 | 4$^a$   | 7$^a$      | 8$^a$       | 6$^a$       | 8$^a$       |
| MC$\Delta$Hzra/pSU2719 | 8$^a$   | 15$^a$     | 11$^a$      | 16$^a$      | 6$^a$       |
| MC$\Delta$Gzra/pSECg | 778 ± 172 | 781 ± 133 | 1095 ± 381 | 1392 ± 216 | 3812 ± 457 |

Cells were grown in LB containing 0.4% glucose and different concentrations of $ZnCl_2$ under oxic conditions. Values are $\beta$-galactosidase activities given in Miller units as averages of three experiments assayed in triplicate unless indicated otherwise.

$^a$ Results of an individual experiment.

Table 2. Induction of the expression of the zraP'-lacZ fusion is specific for zinc and lead

| -     | CuCl₂ | NiCl₂ | CoCl₂ | MnCl₂ | FeCl₂ | MgCl₂ | HgCl₂ | CdCl₂ | PbCl₂ | ZnCl₂ |
|-------|-------|-------|-------|-------|-------|-------|-------|-------|-------|-------|
| 5 ± 3 | 2 ± 1 | 9 ± 7 | 1 ± 1 | 7 ± 5 | 10 ± 4| 6 ± 1 | 11 ± 6| 56 ± 17| 732 ± 245 | 1700 ± 522 |

MCzra (zraP'-lacZ) was grown in LB containing 0.4% glucose and different metals under oxic conditions. NiCl₂, CoCl₂, MnCl₂, FeCl₂, MgCl₂, PbCl₂ and ZnCl₂ were added at 1 mM, CuCl₂ and CdCl₂ at 0.5 mM and HgCl₂ at 1 μM. Values are β-galactosidase activities given in Miller units as averages of at least two experiments assayed in triplicate.

deletion. Transformation of MCΔGzra with pSECg increased the extent of expression in the presence of zinc and caused transcriptional activation in the absence of the effector. The results listed in Table 2 indicate that besides zinc, lead stimulated zraP'-lacZ expression, albeit at a somewhat lower level. The expression observed in the presence of Cd²⁺ was distinctly above the basal level but only about 3% of that from cells grown in the presence of Zn²⁺. Unlike the unspecific action of HydG on the hycA'-lacZ expression, the expression level of zraP was the same under oxic and anoxic conditions (data not shown).

Since the results supported the conjecture that HydH/G is a metal-responsive, two-component regulatory system, we have studied the cellular location of HydH as putative sensor protein and we have purified HydG to study its characteristics.

Cellular localisation of HydH and purification of HydG

Since the hydropathy profile had predicted two transmembrane segments both for the K. oxytoca and the E. coli HydH, the cellular localisation of the E. coli HydH was determined. HydH was selectively labelled with [³⁵S]methionine employing the phage T7 promoter-polymerase system. Extracts of cells were fractionated by differential centrifugation and the distribution of HydH in the different fractions was assessed by SDS-PAGE and autoradiography (Figure 3). The results show that HydH displays the sedimentation behaviour of a typical membrane protein; since 1 M KCl did not solubilize the protein from the P₁₀₀ fraction it appears to be an integral membrane protein. The minor amounts present in the S₁₀₀ fractions may be carry-over or minute membrane particles not sedimenting under the conditions employed.

Protein HydG of E. coli was purified after overproduction in BL21(DE3)/pHYDG, as detailed under Material and Methods. Figure 4 shows the course of purification as visualised by SDS-PAGE and Coomassie blue staining. From a total of 14 g of cells (wet weight) about 22 mg of purified HydG was recovered. N-terminal sequencing proved the authenticity and showed that the N-terminal methionine residue was still present. In gel-filtration experiments, HydG eluted in a position indicative of an apparent molecular mass of 56 kDa, which is close to the value (48.4 kDa) deduced from the gene sequence (data not shown). It suggests that the purified HydG is a monomeric protein.

![Figure 3](https://i.imgur.com/yourimage.png)

Figure 3. Cellular localization of HydH. An amount (15 µl) of the suspension of the cell lysate and the differential centrifugation fractions (S, supernatant; and P, pellet of the centrifugation at 30,000 g, 100,000 g and 100,000 g after KCl treatment) were applied onto an SDS/(10%) polyacrylamide gel. Radioactivity was detected by autoradiography with a phosphorscreen (Molecular Dynamics).

hydH/G Code for a Two-component System

To delineate the DNA fragment to which HydG binds in more detail, DNase I footprinting experiments were carried out (Figure 6). The protected area covers around 55 bp of the zraP-hydH intergenic region located about in the middle of the two genes, reaching from –150 to –95 bp relative to the hydH start codon. It contains the complete inverted repeat motif, including both copies of the 16-meric sequence plus the spacer.

### zraP expression requires σ<sup>54</sup>

The results described so far have shown that HydG, most probably in the phosphorylated form, binds to an upstream activating sequence situated in the middle of the hydH/G and zraP intergenic sequence. This finding and the fact that hydH/G share close sequence similarity with ntrB/C prompted the attempt to determine whether expression of these genes requires σ<sup>54</sup>. To this end, a strain was constructed carrying a mutant rpoN gene and the zraP′-'lacZ fusion in the λ attachment site on the chromosome (strain BN450zra). Growth of BN450zra in LB medium containing 0.4 % (w/v) glucose under aerobic conditions and at 37°C led to an expression level of zraP′-'lacZ equivalent to 10 (±4) β-galactosidase Miller units in the absence of zinc and of 11 (±5) units in the presence of 1 mM zinc. This result strongly supports the conjecture that the expression of the zraP′-'lacZ gene fusion is rpoN-dependent.

A scrutiny of the sequences of the hydH and zraP upstream regions showed that putative –12/–24 promoter motifs can be found in front of both genes. Sequence comparison of the zraP-hydH intergenic region in E. coli, K. oxytoca, Salmonella typhi (sequence data was produced by the S. typhi Sequencing Group at the Sanger Centre and can be obtained from http://www.sanger.ac.uk/Projects/S_typhi/) and Salmonella typhimurium (Genbank accession number AF170176) revealed a very high degree of conservation of both the HydG binding site and the putative –12/–24 promoters (Figure 5(b)), which corroborates the notion that these motifs are indeed the in vivo functional promoters.

#### Figure 4. Purification of HydG. Samples of the different purification steps were applied to an SDS/(10%) polyacrylamide gel and stained with Coomassie blue. Lane 1, crude extract; lane 2, supernatant of the 30,000 g centrifugation; lane 3, supernatant of the 100,000 g centrifugation; lane 4, 0-20% ammonium sulfate fraction; lane 5, purified HydG after Q-Sepharose chromatography.

#### DNA-binding properties of HydG

The binding of the purified HydG protein to the intergenic region separating the hydH and zraP genes was tested by gel-retardation assays. Figure 5(a) shows that HydG binds to a DNA fragment covering basepairs –264 to –66 from the ATG start codon of hydH. Binding is increased about 2.5-fold when HydG is incubated with carbamoyl phosphate and acetyl phosphate before the onset of the mobility shift assay. No binding took place to a DNA fragment extending from –264 to –187 bp upstream of the start codon of hydH and very weak binding was observed with a fragment reaching from –207 to –124 bp, whereas a fragment extending from –207 to –66 was bound as tightly as the complete region (data not shown).

Under the same conditions DNA fragments containing the hycA upstream region (base-pairs –215 to –2 or base-pairs –450 to –152 from the start codon of hycA) were not bound by HydG again providing evidence that the activation of the expression of the hycA′-'lacZ fusion is indeed a matter of cross-talk (data not shown).

A comparison of these results with sequence motifs present in the hydH-zraP intergenic region showed that the DNA fragment providing strong binding of HydG contained two strictly conserved inverted repeats, which are separated by a 14 bp spacer. It is not present in the DNA fragment that does not bind to HydG and only one complete repeat is present in the weakly binding fragment.

#### Autoregulation of the expression of the hydH/G transcription

The results described thus far have demonstrated that the expression of zraP is stimulated by high zinc concentrations in the medium and that it depends on the presence of intact hydH and hydG genes. In order to analyse whether stimulation of zraP is mediated directly by the HydG and HydH proteins, it was determined whether presence of the metal has any influence on their formation. The results presented in Figure 7(a) show that growth in the presence of zinc led to greatly increased cellular levels of HydG protein. This increase required the activity of an intact hydH gene, since a strain with a deletion in hydH displayed HydG levels

hydH/G Code for a Two-component System

a

![Image](image.png)

-50 100 200 500 50 100 200 500  
+ phosphoryl donors - phosphoryl donors  

molar excess  
of HydG  

b

zraP -264  
E. coli  
S. typhi  
S. typhimurium  
K. oxytoca  

-207 σ<sup>54</sup> promoter -187  

-150  
E. coli  
S. typhi  
S. typhimurium  
K. oxytoca  

-124  
HydG  
binding  
site  

-95  
E. coli  
S. typhi  
S. typhimurium  
K. oxytoca  

-66  

-1  
σ<sup>54</sup> promoter  

hydH  

Figure 5. Binding of HydG to the hydH upstream region. (a) The DNA fragment spanning the region from –264 to –66 from the first codon of hydH was incubated with increasing amounts of purified HydG. Where indicated, acetyl phosphate and carbamoyl phosphate were added to the buffer. The arrow indicates the bound DNA. (b) Alignment of the zraP-hydH intergenic region of E. coli, K. oxytoca, S. typhi (sequence data was produced by the S. typhi Sequencing Group at the Sanger Centre and can be obtained from http://www.sanger.ac.uk/Projects/S_typhi/) and S. typhimurium (Genbank accession number AF170176). The beginnings of the hydH and zraP genes are indicated, as well as the putative –24/–12 promoters and the HydG binding site in the E. coli region as determined by DNase I footprinting. Bases are counted concerning the E. coli sequence in relation to the hydH start codon.

hydH/G Code for a Two-component System

a                                                                 b
bases                                                              bases
-91
-99
-101
-109
-114
-121
-124
-127
-128
-130
-132
-145
protected region
-154

-151
-144
-142
-134
-129
-125
-122
-119,-118
-115
-113
-111
-95
protected region
-86

G
HydG

G
HydG

Figure 6. DNase I footprinting of HydG bound to the zraP-hydH intergenic region. (a) Results obtained for the non-coding strand, (b) results from the coding strand. (a) and (b) The left lane shows the reaction without HydG, the following three lanes contain increasing amounts of HydG (100, 500 and 2000-fold molar excess in relation to the DNA). The right lane shows the G track of the Maxam - Gilbert sequence. Bases are numbered in relation to the hydH start codon as depicted in Figure 5(b).

(Figure 7(b)). The results clearly show that from the metal ions tested only zinc and lead significantly affected the level of HydG protein synthesised.

### Discussion

The results presented in this study conclusively show that the HydH/G two-component regulatory system of *E. coli*, which previously was conceived as being implicated in the regulation of the formation of hydrogenase 3 (Stoker et al., 1989) in fact is not a physiological regulator of hydrogenase formation but rather is responsible for the regulation of the expression of zraP (nomenclature adopted from that used by Noll et al. (1998), formally yjal) in the presence of high Zn²⁺ and Pb²⁺ concentrations. ZraP has been found to be involved in the acquisition of tolerance against high Zn²⁺ concentrations (Noll et al., 1998) since the level of toler-

Figure 7. Autoregulation of $hydH/G$ expression. Total lysates of cells grown in LB medium containing 0.4% (w/v) glucose (a) in the presence or absence of 1 mM ZnCl₂ or (b) in the presence of different metal ions were separated in an SDS/(10%) polyacrylamide gel and subjected to immunoblotting analysis with anti-HydG antibodies. X denotes an unspecifically reacting protein of *E. coli*. (b) MC4100 was grown in the presence of the following metals: lane 1, 1 mM MgCl₂; lane 2, 1 μM HgCl₂; lanes 3 and 5, no metal added; lanes 4 and 6, 1 mM ZnCl₂; lane 7, 0.5 mM CdCl₂; lane 8, 1 mM PbCl₂; lane 9, 1 mM NiCl₂; lane 10, 1 mM MnCl₂; lane 11, 0.5 mM CuCl₂; lane 11, 1 mM CoCl₂; lane 13, 1 mM FeCl₂.

HydH is assumed to activate HydG by phosphorylation, since binding of purified HydG to the $zraP-hydH$ intergenic region is stimulated in the presence of phosphorylating compounds. Also, Zn²⁺-dependent stimulation of the expression of both the $hydH/G$ and the $zraP$ transcriptional units requires the presence of a functional $hydH$ gene. Support for our assumption that HydH is the metal sensor comes also from the results that both the autoregulation of $hydH/G$ and the expression of $zraP$ show the same metal specificity.

Since HydH co-separates with the membranes in cell fractionation experiments, it is speculated that it functions as sensor of some periplasmic signal, possibly the external metal concentration. Although HydH possesses no classical zinc-binding motif, there are 13 histidine, one cysteine and various other residues that could be involved in metal binding. For instance, such a function could be assigned to the sequence H₁₅₉GMHNMPRC₁₆₇, which, according to the hydrophobicity plots, is located between the two transmembrane regions and probably periplasmically oriented. One has to stress, however, that the assumption that HydH senses the metal is plausible but not yet proven; alternatively, the metal could be bound to ZraP which by protein-protein interaction might transfer the signal to HydH to initiate the phosphorylation cascade. Direct metal-binding experiments involving purified HydH or mutational inactivation of putative metal-binding motifs of HydH are required to solve this issue.

Our result, that the expression of $zraP$ is under control of the HydH/G system, seems to be at first in contrast with the report that this gene is activated by a MerR-like transcriptional activator (PMTR), which shares significant homology with ZntR (Noll et al., 1998). However, the gene for PMTR was from a heterologous source (*Proteus mirabilis*) and it was not shown that it directly affects transcription of $zraP$. Moreover, transcription activators of the MerR family normally act at σ⁷⁰-dependent promoters (Brocklehurst et al., 1999; Kulkarni & Summers, 1999) and for $zraP$ we have shown that expression is strictly σ⁵⁴-dependent. Also, no ZntR-dependent promoter like that in front of $zntA$ could be identified in the 5′ region of $zraP$. For a clear discrimination of the effects of PMTR and the results reported here, the expression analysis has to be conducted with mutants in $hydH/G$ and employing ZntR and the other *E. coli* homolog of PMTR, CueR (YbbI) (Outten et al., 2000).

The involvement of a two-component regulatory system in the acquisition of metal tolerance, as could be the case for HydH/G, is unusual and has to be discussed against the background of zinc tolerance of *E. coli* in general. A cation-specific P-type ATPase has been identified that preferentially exports Zn²⁺ and, to a lesser degree, Pb²⁺ and Cd²⁺ (Rensing et al., 1997; Sharma et al., 2000). It is encoded by the $zntA$ gene; mutational inactivation of $zntA$ increases the sensitivity to Zn²⁺ (Beard et al., 1997). The expression of $zntA$ is regulated by ZntR (Brocklehurst et al., 1999). ZntR responds to Zn²⁺, Cd²⁺ and Pb²⁺. Among these, Cd²⁺ is the most effective inducer (Binet & Poole, 2000) in contrast to the HydH/G system, which responds preferentially to Zn²⁺ and only to a lesser degree to Pb²⁺. Both HydH/G and ZntR, however, share

the same metal concentration-dependence in their regulatory properties. The Zn²⁺-dependent activation of *zntA* and *zraP*, respectively, initiates at about 0.2 mM Zn²⁺ and is maximal at a concentration of 1 mM. Concerning the probable co-existence of two systems involved in regulation of Zn²⁺ tolerance in *E. coli*, recent studies on the copper-resistance of *E. coli* showed a similar phenomenon. While CopA, the principal copper efflux ATPase, is regulated by the MerR homologue CueR, there exists also a two-component system (CusS/R) that activates transcription of its adjacent genes coding for a proton/cation antiporter complex (*Munson et al.*, 2000). CusS/R may sense the copper level in the periplasm, while CueR probably is a cytoplasmic regulator (*Outten et al.*, 2000). In the same manner, HydH/G could be responsible for periplasmic zinc sensing, in contrast to the cytoplasmic ZntR. It would be interesting to see whether both regulatory circuits overlap. Other interesting issues concern the questions of which genes are regulated by HydH/G and ZntR in addition to those reported until now.

The experiments described here show conclusively that the HydH/G system regulates expression of *zraP*. The assumption that it is involved in regulation of Zn²⁺ tolerance is based only on the report that ZraP has been implicated in acquisition of metal tolerance (*Noll et al.*, 1998). Until now we found no evidence for the involvement of HydH/G in zinc resistance, neither in growth kinetics nor in the determination of the minimal inhibitory concentration of Zn²⁺. However, preliminary experiments show an effect of HydG on the growth of cells after a shift from Zn²⁺-free medium to a medium containing 1 mM Zn²⁺, since HydG-containing cells seem to have longer lag phases following the shift (data not shown). Other functions of ZraP, like protection of periplasmic proteins against mis-incorporation of Zn²⁺ or that of a Zn²⁺ donor protein in the periplasm, cannot be ruled out yet. So, the physiological function of HydH/G still remains to be elucidated.

Last, but not least, the problem of the functional mis-assignment of HydH/G as regulators of the synthesis of hydrogenase 3 needs to be discussed. HydG activates the expression of a *hycA′-lacZ* reporter gene fusion but only when it is encoded on a plasmid, i.e. when it is overproduced. The activation is independent of HydH but is stimulated by anaerobiosis. When FhlA is present, overproduced HydG is without any effect. Additional arguments that the HydG action on hydrogenase 3 formation is unspecific comes from the observations that an *fdhF′-lacZ* fusion is not activated at all by HydG (our unpublished results), that HydG from *K. oxytoca* is more than an order of magnitude more effective in activating expression of *hycA′-lacZ* as compared to the homologous protein, that HydG does not bind to the *hycA* upstream region and that the hydrogenase activity is not decreased in Δ*hydG* cells. Our results stress

the problems that arise when functions of gene products are delineated by complementation *in trans* by orthologs encoded on a plasmid or from some heterologous source.

As a consequence of our results, *hydH/G* should be redesignated; we propose *zraS* for *hydH* and *zraR* for *hydG*, to relate them to the gene whose expression they regulate, namely *zraP*.

# Material and Methods

## Bacterial strains, plasmids and growth conditions

The strains of *E. coli* and the plasmids used in this work are listed in Table 3. For determination of β-galactosidase activity and hydrogenase activity, bacteria were cultivated in LB medium (*Miller*, 1992) or in the buffered rich medium (TGYEP) described by *Begg et al.* (1977), which was supplemented with 1 μM sodium molybdate, 1 μM sodium selenite and 5 μM nickel chloride. Where indicated, sodium formate was added at 30 mM. The techniques developed by *Balch & Wolfe* (1976) were employed for the anaerobic growth of organisms. When required for the maintenance of plasmids, the media were supplemented with kanamycin sulfate (50 μg/ml), chloramphenicol (30 μg/ml), tetracycline (20 μg/ml) or ampicillin (100 μg/ml).

## Recombinant DNA techniques

Standard recombinant DNA techniques were adopted from *Sambrook et al.* (1989). Plasmids were introduced into *E. coli* by electroporation (*Fiedler & Wirth*, 1988) or by the transformation procedure developed by *Chung et al.* (1989). PCR products were sequenced using the Dye Terminator Cycle Sequencing Kit with AmpliTaq® DNA Polymerase FS from PE Applied Biosystems (Warrington, Great Britain). Data were collected with an ABI Prism 377 sequencer.

## Plasmid and strain constructions

Plasmid pSKOG contains a 2.7 kb DNA fragment from *K. oxytoca* M5a1 that codes for a 3′ fragment of *hydH* and the complete *hydG* gene of this organism. It was isolated by transferring a gene bank of *K. oxytoca* DNA restricted with Sau3A and cloned into BamHI-linearized vector pSU2719 (*Fiedler et al.*, 1996) into cells of *E. coli* SA22; screening was on MacConkey-lactose plates (Gibco) enriched with 30 mM sodium formate.

Plasmid pHYDHG is a pT7-7 derivative that carries the complete *hydHG* operon from *E. coli*. The fragment containing the operon was amplified from *E. coli* MC4100 chromosomal DNA using the primers HydH-N (5′-CAAAGAAGCATATGCGTTTTATGCAAC-3′) and HydG-C (5′-CGAGAATTCCCTAACGGCGACAGTTTTGC-3′). The fragment was trimmed with endonucleases NdeI and EcoRI and ligated into vector pT7-7 linearized with the same enzymes.

Plasmid pSECG carries a *hydH* gene fragment starting from base 568 plus the complete *hydG* gene. It was obtained by digestion of pHYDHG with the endonucleases BstXI and EcoRI followed by removal of the protruding ends with Klenow enzyme and ligation into SmaI linearized vector pSU2719.

Plasmids pHYDH and pHYDG contain the *E. coli* *hydH* or *hydG* gene, respectively, under control of the

Table 3. *E. coli* strains and plasmids used in this study

| Strain or plasmid | Genotype | Source or reference |
|-------------------|----------|---------------------|
| MC4100            | $F^{-} araD139\Delta(argF-lac) U196 ptsF25 deoC1 relA1 flbB5301 rpsL150 \lambda^{-}$ | Casadaban & Cohen (1979) |
| MC4100$\Delta$G   | Like MC4100, $\Delta hydG$, zja::Tn10, Tc${}^{R}$ | This work |
| MC4100$\Delta$H   | Like MC4100, $\Delta hydH$, zja::Tn10, Tc${}^{R}$ | This work |
| MC4102            | Like MC4100, $hycA^{-}-'lacZ$ | Hopper *et al.* (1994) |
| MC4102$\Delta$G   | Like MC4102, $\Delta hydG$, zja::Tn10, Tc${}^{R}$ | This work |
| MC4102$\Delta$H   | Like MC4102, $\Delta hydH$, zja::Tn10, Tc${}^{R}$ | This work |
| SA22              | Like MC4100, $\Delta fhlA hycA^{-}-'lacZ$ | Hopper (1995) |
| SA22$\Delta$H     | Like SA22, $\Delta hydH$, zja::Tn10, Tc${}^{R}$ | This work |
| BL21(DE3)         | $F^{-} hsdS gal \lambda DE3$ | Studier & Moffatt (1986) |
| EM66 (SM6009)     | aceA1 zja::Tn10, Tc${}^{R}$ | Maloy *et al.* (1980) |
| K38               | HfrC phoA4 pit-10 tonA22 ompF627 relA1 ($\lambda$) | Lyons & Zinder (1972) |
| MCzra             | Like MC4100, $zraP^{-}-'lacZ$ | This work |
| MC$\Delta$Gzra    | Like MC4100$\Delta$G, $zraP^{-}-'lacZ$, zja::Tn10, Tc${}^{R}$ | This work |
| MC$\Delta$Hzra    | Like MC4100$\Delta$H, $zraP^{-}-'lacZ$, zja::Tn10, Tc${}^{R}$ | This work |
| BN450             | Like MC4100, $ntrA208::Tn10$ | Birkmann *et al.* (1987) |
| BN450zra          | Like BN450, $zraP^{-}-'lacZ$, zja::Tn10, Tc${}^{R}$ | This work |

| plasmids |  |  |
|----------|---|---|
| pSU2719  | Cm${}^{R}$, lacZ${}_{1}$ | Martinez *et al.* (1988) |
| pSKOG    | pSU2719, $'hydH-hydG(K. oxytoca)$ | This work |
| pSECQ    | pSU2719, $'hydH-hydG(E. coli)$ | This work |
| pT7-7    | Ap${}^{R}$, T7$\Phi$10 | Tabor & Richardson (1985) |
| pGP1-2   | Km${}^{R}$, T7 gene 1 (RNA polymerase) | Tabor & Richardson (1985) |
| pHYDHG   | pT7-7 ($hydHG$) | This work |
| pHYDG    | pT7-7 ($hydG$) | This work |
| pHYDH    | pT7-7 ($hydH$) | This work |
| pHYD$\Delta$G | pHYDG $hydG(\Delta 359$ to 985) | This work |
| pHYD$\Delta$H | pHYDH $hydH(\Delta 568$ to 979) | This work |
| pMAK700  | Cm${}^{R}$ | Hamilton *et al.* (1989) |
| pMAK$\Delta$G | pMAK700 $hydG(\Delta 359$ to 985) | This work |
| pMAK$\Delta$H | pMAK700 $hydH(\Delta 568$ to 979) | This work |
| pRS552   | Km${}^{R}$, Ap${}^{R}$, $'lacZ lacY^{+} lacA^{+}$ | Simons *et al.* (1987) |
| pUC19    | Ap${}^{R}$ | Yanisch-Perron *et al.* (1985) |
| pUHupA   | pUC19 (226 bp $zraP-hydH$ intergenic region in orientation A) | This work |
| pUHupB   | pUC19 (226 bp $zraP-hydH$ intergenic region in orientation B) | This work |

phage T7 promoter. pHYDH was obtained by amplification of genomic DNA from *E. coli* MC4100 using the primers HydH-N (see above) and HydH-C (5'-TTAT-GAATTCTCATCCTTGTGGGGTCC-3'), trimming of the ends with restriction endonucleases NdeI and EcoRI and ligation into vector pT7-7 treated with the same enzymes. The strategy for the cloning of pHYDG was essentially the same. $hydG$ was amplified using the primers HydG-N (5'-GGACCCACACATATGACGCAC-GATAAT-3') and HydG-C (see above).

For the construction of strains possessing in-frame deletions in $hydG$ and $hydH$, derivatives of pHYDG and pHYDH were first generated. Nucleotides 359 to 985 of $hydG$ were removed from plasmid pHYDG by cleavage with Eco47III followed by insertion of four 5'-GGGATCCC-3' linkers and subsequent religation, yielding plasmid p$\Delta$HYDG. For the generation of the $\Delta hydH$ mutant, pHYDH was cleaved with PvuII and BstXI, and the nucleotides 568 to 979 of $hydH$ were removed. After treating with Klenow enzyme, four copies of the linker 5'-CGGAATTCG-3' were inserted and the plasmid was religated, yielding p$\Delta$HYDH. The XbaI fragments of p$\Delta$HYDG and p$\Delta$HYDH, respectively, were inserted into the NheI site of pMAK700, giving rise to pMAK$\Delta$G and pMAK$\Delta$H. These in-frame deletions were integrated into the genome of EM66 using the method of Hamilton *et al.* (1989) and transferred to various MC4100 derivative strains using P1 transduction (Miller, 1992).

$zraP^{-}-'lacZ$ gene fusions were constructed as follows: A DNA fragment containing the 200 bp untranslated 5' region and the first six codons of $zraP$ was amplified using PCR with genomic DNA of MC4100 as a template and the primers laczra1 (5'-CGGGATCCATTTCG TGTCCGTTTCATG-3') and laczra2 (5'-GCAGAATT CATGCCAT CTTTTATC-3'). After treatment with BamHI and EcoRI it was ligated into pRS552. The gene fusions were then transduced into various strains according to the method of Simons *et al.* (1987).

Plasmids pUHupA and pUHupB, which were used for the DNase I footprinting assay, are pUC19 derivatives each containing a 213 bp fragment from the $hydH$ upstream region (-19 to -232 relative to the start codon of $hydH$). The respective DNA fragment was amplified from genomic DNA of MC4100 using PCR and the primer pair FootA (5'-GGTGGATCCTCCAGTGGTT GTCTTAC-3') and FootB (5'-CCTGCGGATCCCTTTC TGACAGCAG-3'). The resulting 226 bp product was purified, treated with BamHI and cloned in two orientations into the BamHI site of pUC19.

### Determination of enzyme activities

For $\beta$-galactosidase assays, cells were grown to an absorbance at 600 nm of between 0.4 and 0.6. The enzyme activity was determined as described by Miller (1992).

HydH/G Code for a Two-component System

Hydrogenase activity was assayed by H₂-dependent benzylviologen reduction according to the procedure of Ballantine & Boxer (1985) using the 30,000 g supernatant of cellular extracts from cells that were grown anaerobically in the presence of 15 mM formate.

### Subcellular localisation of HydH

HydH was selectively labelled with L-[³⁵S]-methionine using strain K38 and the plasmids pGP1-2 and pHYDH of the T7 promoter-RNA polymerase system (Tabor & Richardson, 1985). The procedure described by Hrynewicz et al. (1990) was followed throughout.

The labelled cells were taken up together with unlabelled carrier cells in 600 μl of buffer (50 mM Tris-HCl (pH 7.5 at 4°C), 1 mM EDTA, 1 mM DTT) to which PMSF had been added at 1 mM. The cells were broken by sonication.

The broken cell suspension was centrifuged consecutively: twice at 3000 g for ten minutes each; once at 30,000 g for 15 minutes; and finally at 100,000 g for 120 minutes. The pellet fraction of the 100,000 g centrifugation was suspended in the buffer given above and divided into two portions. One of them was brought to 1 M KCl, stirred for one hour and afterwards centrifuged again at 100,000 g for 120 minutes. All fractions were subjected to SDS-PAGE and subsequent autoradiography.

### Overproduction and purification of HydG

BL21(DE3)/pHYDG were grown in LB broth at 28°C. When the culture reached an absorbance at 600 nm of 0.6, production of HydG was induced by adding IPTG to a concentration of 0.1 mM. After a further two hours, the cells were harvested by centrifugation and washed in cold buffer A (50 mM Tris-HCl (pH 7.5), 1 mM EDTA, 1 mM DTT). The pellet was frozen in liquid nitrogen and stored at –80°C.

Purification was conducted at 4°C. The protein concentration was assayed during purification according to the method of Whitaker & Granum (1980). The HydG distribution of the fractions was analyzed by SDS-PAGE (Laemmli, 1970).

After suspending the cells in a twofold volume of buffer A containing 1 mM PMSF, they were broken by three consecutive passages through a French pressure cell at 133 MPa. The extract was clarified by centrifugation at 30,000 g for 20 minutes. PMSF was again added to the supernatant to a concentration of 1 mM and sodium chloride to a concentration of 200 mM, and the suspension was stirred for 30 minutes on ice followed by centrifugation at 100,000 g for 90 minutes to remove the membranes. After adjusting the protein concentration of the supernatant to 15 mg/ml with buffer A, ammonium sulfate solution in buffer A was added to 20% saturation. Precipitated proteins were sedimented by centrifugation at 20,000 g for 15 minutes, resolubilized in buffer A and dialysed overnight against buffer A. The sample was subjected to anion-exchange chromatography by applying it to a Q Sepharose column of 8 ml bed volume (Pharmacia Biotech, Freiburg, Germany). HydG was eluted in a 100 ml linear gradient from buffer A to buffer B (buffer A containing 400 mM NaCl) at a concentration of 200 mM NaCl. Appropriate fractions were pooled and dialysed against buffer C (100 mM potassium phosphate, (pH 7.5), 1 mM EDTA, 1 mM DTT, 50 % (w/v) glycerol) and stored at –20°C.

### Determination of the N-terminal sequence of HydG

The N-terminal sequence was analysed using an Applied Biosystems 477A pulsed liquid-phase sequencer (Weiterstadt) with a model 120A phenylthiohydantoin analyser.

### DNA mobility-shift assay

DNA fragments containing different parts of the hydH upstream region were amplified by PCR using MC4100 genomic DNA as template and appropriate primers. After purification of the fragments they were radioactively labelled with [γ-³²]P ATP by polynucleotide kinase. DNA (0.01 pmol) was incubated with 0.5, 1, 2 or 5 μg of purified HydG protein in a buffer containing 100 mM potassium phosphate (pH 7.5), 0.5 mM EDTA, 1 mM DTT, 5 mM magnesium chloride, 1 mM ATP, 4 mM spermidine and 10 % (w/v) glycerol. Unless indicated otherwise, acetyl phosphate and carbamoyl phosphate were each present at a concentration of 50 mM. The reaction was separated in a 5 % (w/v) polyacrylamide gel in TBE buffer (Sambrook et al., 1989) pH 7.6 and analyzed using phosphoimaging (Molecular Dynamics, Sunnyvale, CA).

### DNase I footprinting assay

pUHupA (10 pmol) and pUHupB (10 pmol), respectively, were cleaved with EcoRI and PvulI and the resulting 434 bp fragment was purified. The DNA was labelled at only one 3′ end by the following fill-in reaction: 5 pmol of DNA was incubated at 25°C with 5 μl of Klenow enzyme buffer, 5 μl of [α-³²]P dATP (50 μCi, 0.33 μM), 1 μl of 0.1 mM dTTP, dCTP and dGTP each, and 6 units of Klenow enzyme in a volume of 50 μl. After 25 minutes, an excess of non-radioactive dNTPs was added and after a further five minutes the labelled DNA was separated from the free nucleotides.

For the footprinting assay, 0.05 pmol of DNA was incubated with different amounts of purified HydG in a buffer containing 50 mM Tris-HCl (pH 7.5), 50 mM KCl, 1 mM DTT, 0.5 mM EDTA, 5 mM MgCl₂, 1 mM ATP, 4 mM spermidine, 50 μg/ml salmon sperm DNA, 10 mM acetyl phosphate and 10 mM carbamoyl phosphate in a volume of 20 μl. After 30 minutes of incubation at 30°C, five units of DNase I (grade 1, Boehringer, Mannheim) were added and the reaction was allowed to proceed for one minute. It was stopped by the addition of EDTA to a final concentration of 25 mM and tRNA to a concentration of 22.2 μg/ml. The volume was adjusted to 180 μl and the reaction mixture was transferred onto ice. After precipitation with LiCl and ethanol, the sample was resuspended in 20 μl of formamide loading dye. The G sequencing reaction according to Maxam & Gilbert (1977) was performed essentially as described (Ausubel et al., 1997). The fragments of the footprinting and of the sequencing reaction were separated in a 7 M urea/6 % (w/v) polyacrylamide gel. The gel was dried and analyzed by phosphoimaging (Molecular Dynamics).

### Immunoblot analysis

Chicken antibodies directed against HydG protein were produced commercially by Davids Biotechnology (Regensburg, Germany); they were treated with a crude extract of MC4100ΔG cells and used in a 1:2000 dilution

for immunoblotting. Peroxidase-conjugated AffiniPure rabbit anti-chicken IgY* from Dianova (Hamburg, Germany) were used as secondary antibodies with the ECL system of Roche Molecular Biochemicals (Mannheim, Germany).

### Computer-based analysis and graphical presentation

Immunoblots and autoradiographs were digitalized using an UMAX UC840 MaxVision scanner and Adobe Photoshop 5 software.

---

### Acknowledgements

We are grateful to Dr K. Hantke for helpful discussions. This work was supported by grants from the Deutsche Forschungsgemeinschaft and the Fonds der Chemischen Industrie.

### References

Andrews, S. C., Berks, B. C., McClay, J., Ambler, A., Quail, M. A., Golby, P. & Guest, J. R. (1997). A 12-cistron *Escherichia coli* operon (*hyf*) encoding a putative proton-translocating formate hydrogenase system. *Microbiology*, 143, 3633-3647.

Ausubel, F. M., Brent, R., Kingston, R. E., Moore, D. D., Seidmann, J. G., Smith, J. A. & Struhl, K. (1997). *Current Protocols in Molecular Biology*, J. Wiley & Sons, Inc, New York.

Ballantine, S. P. & Boxer, D. H. (1985). Nickel-containing hydrogenase isoenzymes from anaerobically grown *Escherichia coli* K-12. *J. Bacteriol.* 163, 454-459.

Balch, W. E. & Wolfe, R. S. (1976). New approach to the cultivation of methanogenic bacteria: 2-mercaptoethanesulfonic acid (HS-CoM)-dependent growth of *Methanobacterium ruminantium* in a pressurized atmosphere. *Appl. Environ. Microbiol.* 32, 781-791.

Beard, S. J., Hashim, R., Membrillo-Hernandez, J., Hughes, M. N. & Poole, R. K. (1997). Zinc(II) tolerance in *Escherichia coli* K-12: evidence that the *zntA* gene (o732) encodes a cation transport ATPase. *Mol. Microbiol.* 25, 883-891.

Begg, Y. A., Whyte, J. A. & Haddock, B. Y. (1977). The identification of mutants of *Escherichia coli* deficient in formate dehydrogenase and nitrate reductase activities using dye indicator plates. *FEMS Microbiol. Letters*, 2, 47-50.

Binet, M. R. B. & Poole, R. K. (2000). Cd(II), Pb(II) and Zn(II) ions regulate expression of the metal-transporting P-type ATPase ZntA in *Escherichia coli*. *FEBS Letters*, 473, 67-70.

Birkmann, A., Sawers, R. G. & Böck, A. (1987). Involvement of the *ntrA* gene product in the anaerobic metabolism of *Escherichia coli*. *Mol. Gen. Genet.* 210, 535-542.

Blattner, F. R., Plunkett, G., III, Bloch, C. A., Perna, N. T., Burland, V., Riley, M., Collado-Vides, J., Glasner, J. D., Rode, C. K., Mayhew, G. F., Gregor, J., Davis, N. W., Kirkpatrick, H. A., Goeden, M. A., Rose, D. J., Mau, B. & Shao, Y. (1997). The complete genome sequence of *Escherichia coli* K-12. *Science*, 277, 1453-1474.

Böhm, R., Sauter, M. & Böck, A. (1990). Nucleotide sequence and expression of an operon in *Escherichia coli* coding for formate hydrogenlyase components. *Mol. Microbiol.* 4, 231-243.

Brocklehurst, K. R., Hobman, J. L., Lawley, B., Blank, L., Marshall, S. J., Brown, N. L. & Morby, A. P. (1999). ZntR is a Zn(II)-responsive MerR-like transcriptional regulator of *zntA* in *Escherichia coli*. *Mol. Microbiol.* 31, 893-902.

Brondsted, L. & Atlung, T. (1994). Anaerobic regulation of the hydrogenase 1 (*hya*) operon of *Escherichia coli*. *J. Bacteriol.* 176, 5423-5428.

Casadaban, M. J. & Cohen, S. N. (1979). Lactose genes fused to exogenous promoters in one step using a Muc-lac bacteriophage: *in vivo* probe for transcriptional control sequences. *Proc. Natl Acad. Sci. USA*, 76, 4530-4533.

Chung, C. T., Niemela, S. L. & Miller, R. H. (1989). One-step preparation of competent *Escherichia coli*: transformation and storage of bacterial cells in the same solution. *Proc. Natl Acad. Sci. USA*, 86, 2172-2175.

Fiedler, G., Pajatsch, M. & Böck, A. (1996). Genetics of a novel starch utilisation pathway present in *Klebsiella oxytoca*. *J. Mol. Biol.* 256, 279-291.

Fiedler, S. & Wirth, R. (1988). Transformation of bacteria with plasmid DNA by electroporation. *Anal. Biochem.* 170, 38-44.

Francis, K., Patel, P., Wendt, J. C. & Shanmugam, K. T. (1990). Purification and characterization of two forms of hydrogenase isoenzyme 1 from *Escherichia coli*. *J. Bacteriol.* 172, 5750-5757.

Hamilton, C. M., Aldea, M., Washburn, B. K., Babitzke, P. & Kushner, S. R. (1989). New method for generating deletions and gene replacements in *Escherichia coli*. *J. Bacteriol.* 171, 4617-4622.

Hopper, S. (1995). Das Formiat-Regulon von *Escherichia coli*: Untersuchungen zur FhlA-abhängigen Regulation. Dissertation, Ludwig-Maximilians-Universität München.

Hopper, S., Babst, M., Schlensoq, V., Fischer, H. M., Hennecke, H. & Böck, A. (1994). Regulated expression *in vitro* of genes coding for formate hydrogenlyase components of *Escherichia coli*. *J. Biol. Chem.*, 269, 19597-19604.

Hryniewicz, M., Sirko, A., Palucha, A., Böck, A. & Hulanicka, D. (1990). Sulfate and thiosulfate transport in *Escherichia coli* K-12: identification of a gene encoding a novel protein involved in thiosulfate binding. *J. Bacteriol.* 172, 3358-3366.

Kulkarni, R. D. & Summers, A. O. (1999). MerR cross-links to the alpha, beta, and sigma 70 subunits of RNA polymerase in the preinitiation complex at the *merTPCAD* promoter. *Biochemistry*, 38, 3362-3368.

Kyte, J. & Doolittle, R. F. (1982). A simple method for displaying the hydrophobic character of a protein. *J. Mol. Biol.* 157, 105-132.

Laemmli, U. K. (1970). Cleavage of structural proteins during the assembly of the head of bacteriophage T4. *Nature*, 227, 680-685.

Leonhartsberger, S., Ehrenreich, A. & Böck, A. (2000). Analysis of the domain structure and the DNA binding site of the transcriptional activator FhlA. *Eur. J. Biochem.* 267, 3672-3684.

Lutz, S., Böhm, A., Beier, A. & Böck, A. (1990). Characterisation of divergent NtrA-dependent promoters in the anaerobically expressed gene cluster coding for hydrogenase 3 components of *Escherichia coli*. *Mol. Microbiol.* 4, 13-20.

Lutz, S., Jacobi, A., Schlensoq, V., Böhm, R., Sawers, G. & Böck, A. (1991). Molecular characterization of an operon (*hyp*) necessary for the activity of the three

hydrogenase isoenzymes in *Escherichia coli*. Mol. Microbiol. 5, 123-135.

Lyons, L. B. & Zinder, N. D. (1972). The genetic map of the filamentous bacteriophage f1. Virology, 49, 45-60.

Maloy, S. R., Bohlander, M. & Nunn, W. D. (1980). Elevated levels of glyoxylate shunt enzymes in *Escherichia coli* strains constitutive for fatty acid degradation. J. Bacteriol. 143, 720-725.

Martinez, E., Bartolome, B. & de la Cruz, F. (1988). pACYC184-derived cloning vectors containing the multiple cloning site and lacZ alpha reporter gene of pUC8/9 and pUC18/19 plasmids. Gene (Amst.), 68, 159-162.

Maxam, A. M. & Gilbert, W. (1977). A new method for sequencing DNA. Proc. Natl Acad. Sci. USA, 74, 560-564.

Menon, N. K., Robbins, J., Peck, H. D. J., Chatelus, C. Y., Choi, E. S. & Przybyla, A. E. (1990). Cloning and sequencing of a putative *Escherichia coli* [NiFe] hydrogenase-1 operon containing six open reading frames. J. Bacteriol. 172, 1969-1977.

Menon, N. K., Chatelus, C. Y., Dervartanian, M., Wendt, J. C., Shanmugam, K. T., Peck, H. D. J. & Przybyla, A. E. (1994). Cloning, sequencing, and mutational analysis of the *hyb* operon encoding *Escherichia coli* hydrogenase 2. J. Bacteriol. 176, 4416-4423.

Miller, J. H. (1992). *A Short Course in Bacterial Genetics*, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, New York.

Munson, G. P., Lam, D., Outten, F. W. & O'Halloran, T. V. (2000). Identification of a copper-responsive two-component system on the chromosome of *Escherichia coli* k-12. J. Bacteriol. 182, 5864-5871.

Noll, M., Petrukhin, K. & Lutsenko, S. (1998). Identification of a novel transcription regulator from Proteus mirabilis, PMTR, revealed a possible role of YJAI protein in balancing zinc in *Escherichia coli*. J. Biol. Chem. 273, 21393-21401.

Outten, F. W., Outten, C. E., Hale, J. & O'Halloran, T. V. (2000). Transcriptional activation of an *E. coli* copper efflux regulon by the chromosomal MerR homologue, CueR. J. Biol. Chem, 275, 31024-31029.

Peck, H. D. & Gest, H. (1957). Formic dehydrogenase and the hydrogenase enzyme complex in coliform bacteria. J. Bacteriol. 73, 706-721.

Rensing, C., Mitra, B. & Rosen, B. P. (1997). The *zntA* gene of *Escherichia coli* encodes a Zn(II)-translocating P-type ATPase. Proc. Natl Acad. Sci. USA, 94, 14326-14331.

Richard, D. J., Sawers, G., Sargent, F., McWalter, L. & Boxer, D. H. (1999). Transcriptional regulation in response to oxygen and nitrate of the operons

encoding the [NiFe] hydrogenases 1 and 2 of *Escherichia coli*. Microbiology, 145, 2903-2912.

Rossmann, R., Sawers, G. & Böck, A. (1991). Mechanism of regulation of the formate-hydrogenlyase pathway by oxygen, nitrate, and pH: definition of the formate regulon. Mol. Microbiol. 5, 2807-2814.

Sambrook, J., Fritsch, E. F. & Maniatis, T. (1989). *Molecular Cloning: A Laboratory Manual*, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY.

Sankar, P., Lee, J. H. & Shanmugam, K. T. (1988). Gene-product relationships of *fhlA* and *fdv* genes of *Escherichia coli*. J. Bacteriol. 170, 5440-5445.

Sawers, G. (1994). The hydrogenases and formate dehydrogenases of *Escherichia coli*. Antonie Van Leeuwenhoek, 66, 57-88.

Sawers, R. G. & Boxer, D. H. (1986). Purification and properties of membrane-bound hydrogenase isoenzyme 1 from anaerobically grown *Escherichia coli* K12. Eur. J. Biochem. 156, 265-275.

Schlensog, V. & Böck, A. (1990). Identification and sequence analysis of the gene encoding the transcriptional activator of the formate hydrogenlyase system of *Escherichia coli*. Mol. Microbiol. 4, 1319-1327.

Sharma, R., Rensing, C., Rosen, B. P. & Mitra, B. (2000). The ATP hydrolytic activity of purified ZntA, a Pb(II)/Cd(II)/Zn(II)-translocating ATPase from *Escherichia coli*. J. Biol. Chem. 275, 3873-3878.

Simons, R. W., Houman, F. & Kleckner, N. (1987). Improved single and multicopy lac-based cloning vectors for protein and operon fusions. Gene, 53, 85-96.

Stoker, K., Reijnders, W. N., Oltmann, L. F. & Stouthamer, A. H. (1989). Initial cloning and sequencing of *hydHG*, an operon homologous to *ntrBC* and regulating the labile hydrogenase activity in *Escherichia coli* K-12. J. Bacteriol. 171, 4448-4456.

Studier, F. W. & Moffatt, B. A. (1986). Use of bacteriophage T7 RNA polymerase to direct selective high-level expression of cloned genes. J. Mol. Biol. 189, 113-130.

Tabor, S. & Richardson, C. C. (1985). A bacteriophage T7 RNA polymerase/promoter system for controlled exclusive expression of specific genes. Proc. Natl Acad. Sci. USA, 82, 1074-1078.

Yanisch-Perron, C., Vieira, J. & Messing, J. (1985). Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. Gene, 33, 103-109.

Whitaker, J. R. & Granum, P. E. (1980). An absolute method for protein determination based on difference in absorbance at 235 and 280 nm. Anal. Biochem. 109, 156-159.


Edited by G. Von Heijne

(Received 2 October 2000; received in revised form 8 January 2001; accepted 8 January 2001)
