
the adol mutant exhibits a longer period than that of wild-type Arabidopsis seedlings, whereas under red light cotyledon movement and stem elongation are arrhythmic. Both yeast two-hybrid and in vitro binding studies show that there is a physical interaction between ADO1 and the photoreceptors CRY1 and phyB. We propose that ADO1 is an important component of the Arabidopsis circadian system.

In plants, the circadian clock controls daily changes of photosynthetic activities, leaf movement, cell growth and gene expression^{3,4}. Several molecular components have been described for plant circadian systems^{5-9}, including the blue light photoreceptor cryptochrome and the red light photoreceptor phytochrome, which mediate rhythm entrainment in Arabidopsis^{10}. Cryptochrome also functions in both fly and mammalian circadian clocks^{11}—unexpectedly, in the latter, cryptochrome is described as having the properties of an integral component of the clock^{12-14}.

ADO1 is identical to ZTL^{15} and LKP1 (ref. 16), and is a member of a three-gene family together with ADO2 (LKP2)^{15} and ADO3 (FKF1)^{17} (see Supplementary Information). The PAS domain of ADO1 is similar to both that of white collar 1 (WC-1), an essential component of the Neurospora clock^{18,19}, and to the Arabidopsis proteins phototropin (NPH1)^{20} and NPL1 (ref. 21). These domains mediate protein–protein interactions and are often associated with signalling pathways that monitor changes in light, redox potential, oxygen, small ligands and the overall energy level of the cell^{22}. In the case of phototropin, this PAS/LOV domain binds FMN^{20}. In addition to a PAS domain, the three ADO proteins also contain an F-box domain, which may link target proteins to a ubiquitination complex (SCF complex)^{23}, and six kelch repeats, which are possibly implicated in protein–protein interactions^{24} (Fig. 1; and Supplementary Information).

To define the function of ADO1, we pursued a reverse genetic approach. By using the polymerase chain reaction (PCR) to screen pools of T-DNA (transferred DNA) insertion lines, we identified an insertion in the ADO1 gene within the region encoding the kelch motifs. The T-DNA insertion adds six new codons before an in-frame stop codon occurs (Fig. 1a), which predicts a truncated protein. However, we did not detect expression of ADO1 mRNA in the mutant by northern blot analysis (Fig. 1b); therefore, adol may be a null, or severely hypomorphic allele.

To examine whether ADO1, like the WC-1 of Neurospora^{18}, was affected in clock-regulated processes, we determined by northern blot analysis whether the temporal expression pattern of the endogenous CCR2 gene was altered in the adol mutant background. CCR2 is an Arabidopsis gene that encodes an RNA-binding protein and exhibits circadian expression^{25}. RNA was prepared at 2-h intervals from 3-week-old wild-type and adol mutant plants, which had been entrained to an 8-h photoperiod

---

### An Arabidopsis circadian clock component interacts with both CRY1 and phyB

#### Jose A. Jarillo*, Juan Capel*†‡, Ru-Hang Tang*†, Hong-Quan Yang*, Jose M. Alonso*$§$, Joseph R. Ecker*$§$ & Anthony R. Cashmore*

* Plant Science Institute, Department of Biology, University of Pennsylvania, Philadelphia, Pennsylvania 19104-6018, USA  
‡ Departamento de Biología Aplicada, Universidad de Almería, 04120 Almería, Spain  
† These authors contributed equally to this work  

Most organisms, from cyanobacteria to mammals, use circadian clocks to coordinate their activities with the natural 24-h light/dark cycle. The clock proteins of Drosophila and mammals exhibit striking homology but do not show similarity with clock proteins found so far from either cyanobacteria or Neurospora^{1,2}. Each of these organisms uses a transcriptionally regulated negative feedback loop in which the messenger RNA levels of the clock components cycle over a 24-h period. Proteins containing PAS domains are invariably found in at least one component of the characterized eukaryotic clocks^{1}. Here we describe ADAGIO1 (ADO1), a gene of Arabidopsis thaliana that encodes a protein containing a PAS domain. We found that a loss-of-function adol mutant is altered in both gene expression and cotyledon movement in circadian rhythmicity. Under constant white or blue light,

$§$ Present address: Plant Biology Laboratory, The Salk Institute for Biological Studies, La Jolla, California 92037, USA.

---

**Figure 1** ADO1 gene structure and T-DNA insertion mutant. **a**, Exon structure of ADO1 gene, site of T-DNA insertion and protein domains. Filled boxes represent the two exons, open box represents the intron. The longest cDNA isolated for ADO1 was ~2,300 bp, encoding a protein of 609 amino acids. The site of insertion of the T-DNA in the adol mutant is indicated, as are the conserved motifs in the ADO1 protein. Arrowhead designates C82, the codon split by the intron in the ADO1 genomic sequence. **b**, Northern blot hybridization with an ADO1 probe against RNA prepared from wild-type and adol mutant seedlings. The band in the wild-type lane corresponds to a size of 2.3 kb. No hybridization signal was detected for the mutant.

letters to nature

Col
![Image Description](#)
ado1
![Image Description](#)

Figure 2 The periodicity of CCR2 gene expression is lengthened in the ado1 mutant. Top, hybridization signals for CCR2 mRNA in the wild-type (Col) and ado1 mutant plants. Middle, open and filled boxes indicate day and night, respectively; grey boxes represent subjective night. Bottom, the amount of CCR2 mRNA (relative to ribosomal RNA) is plotted for the wild-type Col and ado1 plants.

and then removed to constant white light for a further 48 h. The period of cyclical CCR2 gene expression was lengthened in the ado1 mutant (Fig. 2). We observed that the period length of CCR2 mRNA expression was roughly 24 h for the wild type, consistent with previous reports²⁵. In contrast, the period length of the ado1 mutant was extended by about 3 h (Fig. 2). We also examined CAB gene expression and observed a similar change in periodicity for the ado1 mutant (data not shown).

We examined a second, essentially unrelated, circadian process. The circadian clock controls both hypocotyl elongation and cotyledon movement in Arabidopsis seedlings²⁶, with circadian dysfunction disrupting these processes⁵,⁶,²⁶,²⁷. Analysis of ado1 mutant seedlings showed that the periodicity of cotyledon movement was lengthened by roughly 6 h in white light (Fig. 3a). Because both blue and red light also serve to entrain the circadian clock in Arabidopsis¹⁰, we similarly defined the periodicity of ado1 mutant seedlings under these light conditions. We observed that the periodicity of cotyledon movements was lengthened in the ado1 seedlings under blue light, as under white light (Fig. 3b). In marked contrast, cotyledon movement and growth of the ado1 seedlings under red light was arrhythmic (Fig. 3c).

Because ado1 was affected in several outputs from the circadian clock, we concluded that ADO1 has a role, either in the clock itself or in processing light signalling to the clock. Both cryptochrome and phytochrome photoreceptors have been shown to function in entraining Arabidopsis circadian rhythms in response to light¹⁰, so we used the yeast two-hybrid system to define potential interactions between ADO1 and these photoreceptors. A fusion of ADO1 to the GAL4 DNA-binding domain (GDB) interacted with the carboxy terminus of PHYB fused to the GAL4 DNA activation domain (GAD), as shown by the production of significant levels of β-galactosidase activity (Fig. 4a, b). Similarly, a strong interaction was observed between ADO1 and the C terminus of CRY1. No interactions were observed between any of these (ADO1, PHYB or CRY1) and unrelated control proteins. We extended these studies by in vitro binding experiments. Again, the GAD–ADO1 fusion protein was found to bind to the C terminus of both CRY1 and PHYB (Fig. 4c). Significantly, no binding of GAD–ADO1 was observed with the related PAS-domain-containing protein NPH1 (Fig. 4c).

Our original analysis of ADO1 was prompted by the striking similarity of its PAS domain to that of the white collar protein WC-1, a component of the Neurospora circadian clock¹⁸,¹⁹. Our observations, together with those indicating that mutations at the ADO1/ZTL locus affect the periodicity of two essentially unrelated processes—CCR2 and CAB gene expression, as well as cotyledon movement¹⁵—argue strongly that ADO1 does not simply affect output but either affects on input to the clock or is an integral component of the circadian oscillator itself. Given the similarity of

Figure 3 Cotyledon tip movement in the ado1 mutant. Cotyledon movement was recorded for wild-type (Col) and mutant (ado1) seedlings grown under a 12-hour photoperiod and then transferred to constant white (a), blue (b) or red (c) light.

the ADO1 PAS domain to the corresponding flavin-binding PAS/LOV domains of phototropin and WC-1, it might be argued that ADO1 is a photoreceptor, as is thought to be the case for these other two PAS-domain proteins²⁰,²⁸. An alternative viewpoint is that ADO1 is an integral component of the Arabidopsis circadian clock, receiving input signals for photo-entrainment from both phytochrome and cryptochrome.

Our observation that ADO1 physically interacts with both phytochrome and cryptochrome photoreceptors of Arabidopsis is consistent with both of these interpretations. Similarly, our finding that stem elongation and cotyledon movement for the ado1 mutant

Figure 4 AD01 interacts with both cryptochrome and phytochrome. **a**, In the yeast two-hybrid system, there is strong interaction between AD01 and CRY1 (I), and between PHYB and AD01 (II). A weak interaction was observed for the positive control proteins SNF1 and SNF4 (III). No interactions were observed for PHYB and pACT vector (IV), pAS vector and AD01 (V), pAS vector and CRY1 (VI), PHYB and SNF4 (VII), SNF1 and AD01 (VIII), or SNF1 and CRY1 (IX). **b**, Quantification of the amount of β-galactosidase produced in **a** for each

are arrhythmic under red light does not distinguish whether AD01 has a role in input or whether it is an integral component of the clock. The long-period rhythmicity that we observed for *ado1* under blue and white light might reflect a redundant activity of *ADO2* and/or *ADO3*. A better understanding of the roles of these genes in the *Arabidopsis* clock should be obtained by examining mutants affected in these additional members of the *ADAGIO* family. Finally, irrespective of the precise function of these genes, our findings indicate that at least some components of the circadian clock in *Arabidopsis* are more similar to those in *Neurospora* than they are to those of flies and mammals.

### Methods

#### AD01 sequence determination
The AD01 genomic sequence was amplified by PCR with a 5′ primer complementary to a 23-base-pair (bp) sequence in the 5′ untranslated region of the AD01 complementary DNA, 130 bp upstream of the ATG codon, and with a 3′ primer complementary to a 22-bp sequence immediately downstream of the TGA codon. The amplified PCR products were sequenced using an ABI 373 sequencer. Flanking genomic sequence outside the cDNA sequence was subsequently obtained from P1 clone MSF19 (GenBank accession number AB016891).

#### Isolation of *ado1*, a T-DNA insertion mutant
The *ado1* mutant was identified by PCR screening of 30,000 T-DNA insertion lines (J.M.A. and J.R.E., unpublished), using oligonucleotides specific for the *AD01* gene and the T-DNA. We showed by DNA sequencing that the insertion disrupted the gene after the codon for Glu 440, adding six amino-acid codons before an in-frame stop codon occurred.

#### Detection of mRNA by northern hybridization
Plants were grown for 3 weeks at 22°C under an 8-h photoperiod and then moved for 48 h to continuous white light of 120 μmol m⁻² s⁻¹. Every 2 h, the aerial parts of at least 5 plants were pooled for RNA extractions. Total RNA (10 μg) was electrophoresed on agarose gels and transferred to Hybond N⁺ (Amersham). We analysed CCR2 gene expression with a probe prepared from expressed sequence tag 151C20T7. After northern hybridization, nylon membranes were exposed to a Bio-Rad imaging plate at room temperature. The image was visualized using a Bio-Rad FX phosphorimager. Expression levels were normalized against the signal obtained by hybridizing the same blot with an *Arabidopsis* 26S ribosomal DNA probe. The normalized values were then expressed as a proportion of the highest value obtained, and graphed. Experiments were repeated twice.

#### Cotyledon movement
Seeds were surface-sterilized and sown in two rows, one of mutant seed and one of wild-type, on MS medium containing 2% sucrose. The seeds were incubated in Petri dishes in a growth chamber under 12 h (80 μmol m⁻² s⁻¹) cool white fluorescent light/12 h dark at 22°C. After 5 d, the Petri dishes were placed before a Logitech Quickcam video camera. Cotyledon movement was recorded under continuous white, blue or red light of 10 μmol m⁻² s⁻¹ at 22°C. We recorded the position in pixels of one cotyledon tip per seedling for 96 h. Periods of cotyledon movements were estimated as described²⁹. The

plasmid combination relative to the number of yeast cells. **c**, *In vitro* transcription/translation was used to generate polypeptides for binding studies. The autoradiograph of the SDS–PAGE gel shows the binding of *in vitro* transcription/translation products to the GAD–AD01 fusion protein immobilized to protein A/agarose beads using antibody against GAD. Both the CRY1 C terminus and the PHYB C terminus bind to AD01, but no binding is observed for the related PAS-domain-containing NPH1 protein.

periodicity of cotyledon movement under continuous white light (and blue light) for the mutant was about 6 h longer than that observed for wild-type. By contrast, the periodicity of CCR2 gene expression for the *ado1* mutant under constant white light was about 3 h longer than that of wild type. These differences in period extension may reflect changes in either the age of the plants and/or the entrainment conditions for the two experiments.

#### Yeast two-hybrid assay
We used the matchmaker yeast two-hybrid system (Clontech) for protein interaction studies. Plasmid pAS2-1 contains the yeast GBD cloning vector, whereas pACT2 contains the GAD cloning vector. To eliminate false positives, yeast cells were transformed with the bait and prey, mated, and positive colonies were selected on SD/–LEU/–TRP/–HIS/AT medium. A non-lethal β-galactosidase plate assay for yeast was achieved on agar plates by adding X-gal directly to the medium (buffered to pH 7.0), colonies were allowed to grow for 3 d, flooded with chloroform for 5 min, and then incubated at 30°C for 24 h. Quantification was determined using a colorimetric assay with chlorophenol red β-D-galactopyranoside as the substrate according to the manufacturer’s protocol. We determined the averages and standard errors for β-galactosidase activities on three colonies, and repeated this three times.

#### *In vitro* protein binding studies
DNA templates encoding the *Arabidopsis* phyB C-terminal domain (645–1172), the CRY1 C-terminal domain (490–681), the full-length NPH1 and the full-length AD01 protein fused to GAD (GAD–AD01) were cloned into the pBluescript SK(−) vector. These templates were transcribed with T7 polymerase and translated in the presence of [³⁵S]Met using the T’n’T-coupled *in vitro* transcription/translation reaction mix (Promega). GAD–AD01 was attached to 20 μl of protein A/agarose beads by adding 1 μg monoclonal antibody against GAD (Santa Cruz Biotechnology). The bead-bound protein was incubated for 45 min at 4°C in 100 μl PBS binding buffer (pH 7.4) containing complete protease inhibitor, 0.1% NP-40, 0.1% bovine serum albumin (BSA) and 1 mM phenyl-methyl-sulphonyl fluoride (PMSF). The protein was washed twice with 1.0 ml PBS binding buffer without BSA and PMSF. For binding studies, agarose beads carrying ~3.5 fmol GAD–AD01 were incubated with candidate binding proteins in a 60-μl reaction on a rotating wheel at 4°C for 1 h. After centrifugation, the beads were pelleted and washed three times with the above ice-cold washing buffer. The washed pellet was resuspended in 2 × SDS sample buffer, and the proteins were analysed by SDS–PAGE and autoradiography.

Received 14 April 2000; accepted 2 January 2001.

1. Dunlap, J. C. Molecular basis for circadian clocks. *Cell* **96**, 271–290 (1999).
2. Lakin-Thomas, P. L. Circadian rhythms: new functions for old clock genes? *Trends Genet.* **16**, 135–142 (2000).
3. Millar, A. Biological clocks in *Arabidopsis thaliana*. *New Phytol.* **141**, 175–197 (1999).
4. Somers, D. The physiology and molecular bases of the plant circadian clock. *Plant Physiol.* **121**, 9–19 (1999).
5. Wang, Z. & Tobin, E. Constitutive expression of the circadian clock associated 1 (*CCA1*) gene disrupts circadian rhythms and suppresses its own expression. *Cell* **93**, 1207–1217 (1998).
6. Schaffer, R. *et al.* The late elongated hypocotyl mutation of *Arabidopsis* disrupts circadian rhythms and the photoperiodic control of flowering. *Cell* **93**, 1219–1229 (1998).
7. Fowler, S. *et al.* GIGANTEA: a circadian clock-controlled gene that regulates photoperiodic flowering in *Arabidopsis* and encodes a protein with several possible membrane-spanning domains. *EMBO J.* **18**, 4679–4688 (1999).
8. Sugano, S., Andronis, C., Ong, M. S., Green, R. M. & Tobin, E. M. The protein kinase CK2 is involved in regulation of circadian rhythms in *Arabidopsis*. *Proc. Natl Acad. Sci. USA* **96**, 12362–12366 (1999).

letters to nature

9. Park, D. H. *et al.* Control of circadian rhythms and photoperiodic flowering by the *Arabidopsis* GIGANTEA gene. *Science* 1579–1582 (1999).

10. Somers, D., Devlin, P. & Kay, S. Phytochromes and cryptochromes in the entrainment of the *Arabidopsis* circadian clock. *Science* 282, 488–490 (1998).

11. Cashmore, A. R., Jarillo, J. A., Wu, Y. J. & Liu, D. Cryptochromes: blue light receptors for plants and animals. *Science* 284, 760–765 (1999).

12. Okamura, H. *et al.* Photic induction of *mPer1* and *mPer2* in cry-deficient mice lacking a biological clock. *Science* 286, 2531–2534 (1999).

13. Griffin, E. A., Staknis, D. & Weitz, C. J. Light-independent role of CRY1 and CRY2 in the mammalian circadian clock. *Science* 286, 768–771 (1999).

14. Kume, K. *et al.* mCRY1 and mCRY2 are essential components of the negative limb of the circadian clock feedback loop. *Cell* 98, 193–205 (1999).

15. Somers, D. E., Schultz, T. F., Milnamow, M. & Kay, S. A. ZEITLUPE encodes a novel clock-associated PAS protein from *Arabidopsis*. *Cell* 101, 319–329 (2000).

16. Kiyosue, T. & Wada, M. LKP1 (LOV kelch protein 1): a factor involved in the regulation of flowering time in *Arabidopsis*. *Plant J.* 23, 807–815 (2000).

17. Nelson, D. C., Lasswell, J., Rogg, L. E., Cohen, M. A. & Bartel, B. *FKF1*, a clock-controlled gene that regulates the transition to flowering in *Arabidopsis*. *Cell* 101, 331–340 (2000).

18. Ballario, P. *et al.* White collar-1, a central regulator of a blue light responses in *Neurospora*, is a zinc finger protein. *EMBO J.* 15, 1650–1657 (1996).

19. Crosthwaite, S. K., Dunlap, J. C. & Loros, J. J. *Neurospora wc-1* and *wc-2*: transcription, photoresponses, and the origins of circadian rhythmicity. *Science* 276, 763–769 (1997).

20. Christie, J. M. *et al.* *Arabidopsis NPH1*: a flavoprotein with the properties of a photoreceptor for phototropism. *Science* 282, 1698–1701 (1998).

21. Jarillo, J. A., Ahmad, M. & Cashmore, A. R. *NPL1* (Accession number AF053941): a second member of the NPH serine/threonine kinase family of *Arabidopsis*. *Plant Physiol.* 117, 719 (1998).

22. Taylor, B. L. & Zhulin, I. B. PAS domains: internal sensors of oxygen, redox potential, and light. *Microbiol. Mol. Biol. Rev.* 63, 479–506 (1999).

23. Patton, E. E., Willems, A. R. & Tyers, M. Combinatorial control in ubiquitin-dependent proteolysis: don’t Skp the F-box hypothesis. *Trends Genet.* 14, 236–243 (1998).

24. Adams, J., Kelso, R. & Cooley, L. The kelch repeat superfamily of proteins: propellers of cell function. *Trends Cell Biol.* 10, 17–24 (2000).

25. Carpenter, C. D., Kreps, J. A. & Simon, A. E. Genes encoding glycine-rich *Arabidopsis thaliana* proteins with RNA-binding motifs are influenced by cold treatment and an endogenous circadian rhythm. *Plant Physiol.* 104, 1015–25 (1994).

26. Dowson-Day, M. & Millar, A. Circadian dysfunction causes aberrant hypocotyl elongation patterns in *Arabidopsis*. *Plant J.* 17, 63–71 (1999).

27. Hicks, K. A. *et al.* Conditional circadian dysfunction of the *Arabidopsis early-flowering 3* mutant. *Science* 274, 790–792 (1996).

28. Ballario, P. & Macino, G. White collar proteins: PASSing the light signal in *Neurospora crassa*. *Trends Microbiol.* 5, 458–462 (1997).

29. Ruf, T. The Lomb–Scargle periodogram in biological rhythm research: analysis of incomplete and unequally spaced time-series. *Biol. Rhythm Res.* 30, 178–201 (1999).

Supplementary information is available on Nature’s World-Wide Web site (http://www.nature.com) or as paper copy from the London editorial office of *Nature*.

**Acknowledgements**

We thank N. Bonini for comments on the manuscript. This work was supported by grants from the NIH and the DOE to A.R.C., from NSF to J.R.E., and a NATO fellowship awarded to J.C.

Correspondence and requests for materials should be addressed to A.R.C. (e-mail: cashmore@sas.upenn.edu).

tathione (GSNO), and show that it is the glutathione-dependent formaldehyde dehydrogenase. Although the enzyme is highly specific for GSNO, it controls intracellular levels of both GSNO and S-nitrosylated proteins. Such ‘GSNO reductase’ activity is widely distributed in mammals. Deleting the reductase gene in yeast and mice abolishes the GSNO-consuming activity, and increases the cellular quantity of both GSNO and protein SNO. Furthermore, mutant yeast cells show increased susceptibility to a nitrosative challenge, whereas their resistance to oxidative stress is unimpaired. We conclude that GSNO reductase is evolutionarily conserved from bacteria to humans, is critical for SNO homeostasis, and protects against nitrosative stress.

The current perspective on the breakdown of SNOs *in vivo* is shaped by the facile decomposition of these compounds *in vitro* by ascorbate, thiols and copper ions<sup>6–8</sup>. Although a number of enzymes can decompose SNOs *in vitro*, including the thioredoxin system<sup>9</sup>, glutathione peroxidase<sup>10</sup>, γ-glutamyl transpeptidase<sup>11</sup>, xanthine oxidase<sup>12</sup> and glutathione-dependent formaldehyde dehydrogenase (GS-FDH)<sup>13</sup>, none has been shown to regulate levels of any endogenous SNO or to be involved in any NO or SNO-mediated response. The activity of GS-FDH has, however, been shown to be particularly robust<sup>13</sup>.

We previously observed that *E. coli* rapidly degrades SNOs, and concluded that intracellular GSNO was being metabolized<sup>3</sup>. We attributed part of the GSNO-metabolizing activity to ‘SNO lyases’, which cleave the SNOs homolytically to NO (ref. 14). Only a fraction of the added SNO, however, could be recovered as nitrite/nitrate, suggesting that there are other reductive routes for SNO metabolism<sup>3,14</sup>. In support of this, we detected a GSNO-consuming activity in *E. coli* lysates that depended on NADH (nicotinamide adenine dinucleotide) but not O<sub>2</sub> (Fig. 1a). We purified this activity to homogeneity and identified it as glutathione-dependent formaldehyde dehydrogenase (GS-FDH) by amino-terminal sequencing (see Methods).

The catalytic rate constant/Michaelis constant (*k*<sub>cat</sub>/*K*<sub>M</sub>) ratios for GSNO were near diffusion-limited rates (Fig. 1b). The enzyme was highly specific for GSNO: no activity was seen for S-nitrosocysteine and S-nitrosohomocysteine, whereas the SNO derivatives of the glutathione (GSH) metabolites cysteinyl-glycine and γ-glutamyl-cysteine were degraded at ~1% of the rate of GSNO. Ammonia (NH<sub>3</sub>) and glutathione disulphide (GSSG) were main products of the *E. coli* enzyme, and the yields depended on GSH, consistent with the following scheme (see also Methods, ‘Products of *E. coli* GSNO reductase’):

GSNO + NADH + H<sup>+</sup> → GSNHOH + NAD<sup>+</sup>

GSNHOH + NADH + H<sup>+</sup> → GSNH<sub>2</sub> + NAD<sup>+</sup> + H<sub>2</sub>O

GSNH<sub>2</sub> + GSH → GSSG + NH<sub>3</sub>

We previously developed a mouse macrophage (RAW 264.7) model of nitrosative stress, characterized by the accumulation of SNOs to toxic levels after activation by interferon-γ (IFN-γ) and lipopolysaccharide (LPS), and ultimately apoptotic cell death<sup>15</sup>. As a first step to understand the fate of SNOs, cytosolic constituents were separated by size. ‘High-mass’ S-nitrosothiols, with a larger relative molecular mass (*M*<sub>r</sub>) than 5,000 (5K), were present in substantial amounts in the cell lysate (Fig. 2a), whereas ‘low-mass’ SNOs (<5K) were not detectable (Fig. 2a) (limit of sensitivity 1 pmol GSNO<sup>4</sup>). As GSNO (~300 pmol per 10<sup>6</sup> cells) formed readily in the extracellular medium of IFN-γ/LPS-treated RAW 264.7 cells incubated with GSH<sup>16</sup>, and GSH is the predominant source of intracellular thiol, we reasoned that GSNO was either rapidly exported or metabolized. Rates of SNO accumulation in the medium of RAW 264.7 cells were very slow and at the limits of detection (ref. 16; and data not shown), whereas GSNO was metabolized quickly when it was incubated with extracts from either resting or cytokine-activated

---

### A metabolic enzyme for S-nitrosothiol conserved from bacteria to humans

Limin Liu<sup>*†</sup>, Alfred Hausladen<sup>‡</sup>, Ming Zeng<sup>†</sup>, Loretta Que<sup>†</sup>, Joseph Heitman<sup>*‡§</sup> & Jonathan S. Stamler<sup>*†¶</sup>

* Howard Hughes Medical Institute, † Department of Medicine, Pulmonary and Cardiology Divisions, and ¶ Department of Biochemistry, Duke University Medical Center, Durham, North Carolina 27710, USA  
‡ Department of Genetics, Duke University Medical Center, Durham, North Carolina 27710, USA  

Considerable evidence indicates that NO biology involves a family of NO-related molecules and that S-nitrosothiols (SNOs) are central to signal transduction and host defence<sup>1–5</sup>. It is unknown, however, how cells switch off the signals or protect themselves from the SNOs produced for defence purposes. Here we have purified a single activity from *Escherichia coli*, *Saccharomyces cerevisiae* and mouse macrophages that metabolizes S-nitroso glu-
