
THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2001 by The American Society for Biochemistry and Molecular Biology, Inc.

AP-1 and Cbfa/Runt Physically Interact and Regulate Parathyroid Hormone-dependent MMP13 Expression in Osteoblasts through a New Osteoblast-specific Element 2/AP-1 Composite Element*

Received for publication, November 26, 2000, and in revised form, February 16, 2001
Published, JBC Papers in Press, March 26, 2001, DOI 10.1074/jbc.M010601200

Jochen Hess, Dominique Porte‡, Christine Munz, and Peter Angel§

From Deutsches Krebsforschungszentrum Heidelberg, Division of Signal Transduction and Growth Control (B0800), Im Neuenheimer Feld 280, D-69120 Heidelberg, Germany

The expression of MMP13 (collagenase-3), a member of the matrix metalloproteinase family, is increased *in vivo* as well as in cultured osteosarcoma cell lines by parathyroid hormone (PTH), a major regulator of calcium homeostasis. Binding sites for AP-1 and Cbfa/Runt transcription factors in close proximity have been identified as *cis*-acting elements in the murine and rat *mmp13* promoter required for PTH-induced expression. The cooperative function of these factors in response to PTH in osteoblastic cells suggests a direct interaction between AP-1 and Cbfa/Runt transcription factors. Here, we demonstrate interaction between c-Jun and c-Fos with Cbfa/Runt proteins. This interaction depends on the leucine zipper of c-Jun or c-Fos and the Runt domain of Cbfa/Runt proteins, respectively. Moreover, c-Fos interacts with the C-terminal part of Cbfa1 and Cbfa2, sharing a conserved transcriptional repression domain. In addition to the distal osteoblast-specific element 2 (OSE2) element in the murine and rat *mmp13* promoter, we identified a new proximal OSE2 site overlapping with the TRE motif. Both interaction of Cbfa/Runt proteins with AP-1 and the presence of a functional proximal OSE2 site are required for enhanced transcriptional activity of the *mmp13* promoter in transient transfected fibroblasts and in PTH-treated osteosarcoma cells.

Interstitial collagenase (collagenase-3, MMP13) is a member of the large family of matrix metalloproteinases that play a decisive role in the degradation of components of the extracellular matrix, particularly the collagens. It degrades collagen type II most efficiently but also collagens type I, III, and X, which are the major components of cartilage and bone (1). MMP13 is expressed in hypertrophic chondrocytes and osteoblasts during embryogenesis and in the adult bone and is thought to be involved in endochondral ossification and bone remodeling (2–4). The level of MMP13 expression depends on the exposure to a variety of systemic and local factors including hormones and cytokines present in the bone microenvironment (5–9). High constitutive levels of MMP13 were observed in

osteosarcomas and chondrosarcomas, suggesting a critical role in the formation of bone tumors (2, 10).

Parathyroid hormone (PTH)¹ has been shown to stimulate MMP13 expression *in vivo* as well as *in vitro* cultures of primary bone cells and osteoblast-derived osteosarcoma cell lines (8, 9, 11). PTH is secreted by the parathyroid gland and is a major regulator of calcium homeostasis. Intracellular signaling initiated by ligand-activated PTH/PTHrP receptor present on the plasma membrane of osteoblasts (12) is mediated predominantly by activation of adenylate cyclase, resulting in enhanced concentration of cAMP and activation of the protein kinase A (PKA) (13, 14).

We and others have identified the minimal PTH-responsive region within the *mmp13* promoter (8, 9). This region contains a conserved AP-1 binding site and an osteoblast-specific element 2 (OSE2), which is recognized by members of the Cbfa/Runt family of transcription factors. Both elements act cooperatively and are absolutely required for PTH-dependent promoter activation. In agreement with the crucial role of AP-1 and Cbfa/Runt transcription factors in osteosarcoma cell lines, expression of MMP13 is reduced in mice lacking *c-fos* and is completely absent in *cbfa1⁻/⁻* embryos (9, 15).²

While DNA binding activity of AP-1 is increased upon PTH treatment through enhanced *de novo* synthesis of c-Fos and c-Jun proteins (16, 17), a high basal level of Cbfa/Runt binding activity was detected independently of PTH treatment, suggesting different mechanisms for transcriptional activation (9). Since no changes in Cbfa/Runt protein or RNA levels were detectable after PTH treatment, posttranslational modifications of Cbfa1 in the signaling pathway for PTH-induced *mmp13* promoter activation was suggested. Indeed, Cbfa1 was shown to be phosphorylated *in vitro* by PKA, a major target of the PTH/PTHrP receptor pathway. Additionally, PTH stimulated the transactivation capacity of Cbfa1 through a PKA site within the C-terminal transactivation domain (18).³ Cbfa1-dependent transcription was also induced by mitogen-activated protein kinase pathways that have been shown to be stimulated by PTH via a cAMP-mediated pathway independent of Ras (19, 20).

The close proximity of the AP-1 and OSE2 sites and their cooperative function in response to PTH in osteoblastic cells suggest a physical interaction between AP-1 and Cbfa/Runt

* This work was supported by European Community Marie Curie Research Training Grant ERB4001GT962949 and grants from the Training and Mobility of Researchers (ERB-FMRX-CT96-0044) and Biomed-2 (BMH4-CT98-3505) programs of the European Community. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
‡ Present address: Laboratoire Arago (UMR 7628 du CNRS), BP44, 66651 Banyuls sur Mer, Cedex, France.
§ To whom correspondence should be addressed. Tel.: 49-6221-42-4570; Fax: 49-6221-42-4554; E-mail: p.angel@dkfz.de.

This paper is available on line at http://www.jbc.org

1 The abbreviations used are: PTH, parathyroid hormone; PKA, protein kinase A; OSE2, osteoblast-specific element 2; PAGE, polyacrylamide gel electrophoresis; GST, glutathione S-transferase; EMSA, electrophoretic mobility shift assay; TPA, 12-O-tetradecanoylphorbol-13-acetate; HA, hemagglutinin; TLE, transducin-like enhancer of split; HA, hemagglutinin.
² J. Tuckermann, S. Mundlos, and P. Angel, unpublished data.
³ D. Porte and P. Angel, unpublished data.

transcription factors. Previous reports have already demonstrated that Cbfa/Runt proteins are promoter organizers that cooperate and interact with neighboring factors and recruit transcriptional co-activators or co-repressors to regulate expression of tissue-specific genes. Specifically, functional interaction between Cbfa/Runt proteins and transcriptional regulators (e.g. ALY (21), Ear-2 (22), p300 (23), Groucho/TLE/R-esp and HES-1 (24–26), Smads (27, 28), YAP (29), Ets-1 (30), C/EBPs (32–34), and PU.1 (35)) have been described.

Here, we demonstrate for the first time a direct interaction between AP-1 and Cbfa/Runt transcription factors in vitro as well as in cultured cells. The leucine zipper of c-Jun or c-Fos and the Runt domain of Cbfa/Runt proteins were sufficient for interaction. Additionally, c-Fos was able to interact with a C-terminal part of Cbfa/Runt proteins sharing a conserved repression domain. Furthermore, we identified a new proximal OSE2 site overlapping with the TRE element in the murine and rat *mmp13* promoter. Transient transfection assays in fibroblasts and PTH-treated osteosarcoma cells demonstrated that interaction between Cbfa/Runt proteins and AP-1 and the presence of a functional proximal OSE2 site are required for synergistic activity on the *mmp13* promoter. These data provide evidence for a direct interaction between AP-1 and Cbfa/Runt transcription factors in the process of PTH-induced expression of MMP13 and for a new functional composite OSE2/TRE element.

EcoRI fragment of Cbfa1 excised from pBSK-Cbfa1 into pcDNA3.1(B) (Invitrogen) linearized with BamHI/EcoRI. pcDNA3.1-Cbfa1ΔC was generated by excision of the HindIII fragment from the pcDNA3.1-Cbfa1 and introduced into pcDNA3.1(B) (Invitrogen) linearized by HindIII.

The luciferase reporter plasmids mColl-luc(−66/+29) and mColl-luc(−663/+29) were generated by replacement of the CAT-SV40poly(A) sequence of the mColl-CAT vector (9) by the luciferase-SV40poly(A) sequence derived from the pGL3 basic vector (Promega) with XhoI/BamHI restriction. mColl-luc(0) was obtained after HindIII restriction of the mColl-luc(−66/+29) construct followed by a religation that resulted in the deletion of the promoter region containing the OSE2/TRE site. mColl-luc(−66/+29)sdm and mColl-luc(−663/+29)sdm plasmids were generated using the QuickChange™ site-directed mutagenesis kit (Stratagene). The following primers were used: TRE/sdm-A (5′-CACACCCAAAGTACTGACTCATCACTATC-3′) and TRE/sdm-B (5′-GATAGTGATGAGTCAGTACTTTGGGGGTGTG-3′). pRSV-lacZ plasmid was kindly provided by T. Wirth.

**Purification of Recombinant Proteins and Analysis of Protein-Protein Interactions—**Recombinant proteins were expressed in BL21pLysS bacteria (Stratagene) cultured in 300 ml of TY medium (10 g of tryptone/peptone, 10 g of yeast extract, 5 g of NaCl, 1 g of casaminoacids, 1 liter of H₂O, and 100 μg/ml ampicillin) in the presence of 0.1 mM isopropyl-1-thio-β-D-galactopyranoside (Sigma), and lysis was performed using the PP100 buffer (25 mM Tris/HCl, pH 7.8; 5 mM MgCl₂, 100 mM KCl, 10% glycerol, 0.1% Nonidet P-40, 2 mM dithiothreitol, 1 mM phenylmethylsulfonyl fluoride, 10 μg/ml leupeptin, 10 μg/ml aprotinin, 10 μg/μl pepstatin). 250 μl of glutathione-agarose beads (Sigma) were mixed with the cell lysate and incubated for 60 min at 4 °C with gentle rotation, and subsequently the beads were washed three or four times with 1 ml of PP100 buffer. For electrophoretic mobility shift assay (EMSA) experiments, the recombinant proteins were eluted from the beads using an elution buffer (50 mM Tris/HCl, pH 7.8, 0.1% Nonidet P-40, 10% glycerol, 1 mM dithiothreitol, 1 mM phenylmethylsulfonyl fluoride, 5 mM glutathione). The yield of the purification and the relative amount of recombinant proteins were estimated by SDS-PAGE and Coomassie staining.

For GST pull-down assays, equal amounts of purified recombinant GST fusion proteins were mixed with glutathione-agarose beads (Sigma) and in vitro translated c-Jun or c-Fos proteins (TNT Coupled Reticulocyte Lysate Systems; Promega). Alternatively, cell lysates prepared with PP100 buffer from PTH-treated or untreated UMR106 cells or transiently transfected HEK293 cells were used. After incubation (1 h at 4 °C), the beads were washed three or four times with 1 ml of PP100 buffer, and the coprecipitated proteins were separated by SDS-PAGE using a 15% polyacrylamide gel. GST pull-down assays with in vitro translated proteins were analyzed by autoradiography. GST pull-down assays with cell lysates were analyzed by Western blot.

For co-immunoprecipitation, the cell lysates prepared with PP100 buffer were first precleared by incubation (30 min, 4 °C) with 10 mg of protein A-Sepharose CL-4B beads (Amersham Pharmacia Biotech) and then were mixed with 10 mg of Protein A-Sepharose CL-4B beads and 2 μg of monoclonal anti-Myc antibody (9E10; Roche Molecular Biochemicals). After 2 h of incubation at 4 °C with gentle rotation, the protein A-coupled Sepharose beads were washed three times with 1 ml of PP100 buffer, and co-precipitated proteins were separated by SDS-PAGE using a 15% polyacrylamide gel and analyzed by Western blot.

Western blots were performed as described (40) using the following antibodies: anti-HA (sc-805; Santa Cruz Biotechnology, Inc., Santa Cruz, CA), anti-GST (27-4577-01; Amersham Pharmacia Biotech), anti-Myc (9E10; Roche Molecular Biochemicals), anti-Cbfa1 (sc-8566; Santa Cruz Biotechnology), anti-c-Jun (J31920; Transduction Laboratories), and anti-c-Fos (sc-52; Santa Cruz Biotechnology).

**EMSA—**Nuclear extracts from UMR106 cells were prepared according to the protocol described (41), and protein concentration was determined using the protein bioassay solution (Bio-Rad). EMSA were performed with 2–4 μg of protein of nuclear extracts and with 30,000–50,000 cpm of a ³²P-radiolabeled probe as described (41, 42). The oligonucleotides for the TRE-Coll probe were described by Porte et al. (9), and the sequences of the other probes were as follows: mutOSE2/mutTRE-Coll-A, 5′-AGCTAAAAGTGACGACTCATCACTAT-3′; mutOSE2/mutTRE-Coll-B, 5′-AGCTATAGTGATGAGTCGTCATT-3′; mutOSE2/TRE-Coll-A, 5′-AGCTAAAAGTACTGACTCATCACTAT-3′; mutOSE2/TRE-Coll-B, 5′-AGCTATAGTGATGAGTCAGTACTTT-3′; OSE2/mutTRE-Coll-A, 5′-AGCTAAAAGTGGCGCATCACTAT-3′; OSE2/mutTRE-Coll-B, 5′-AGCTATAGTGATGCGCCACCACCTTT-3′.

The oligonucleotides for the Oct probe were kindly provided by T. Wirth, and sequences were previously described (68).

**Cell Culture and Transfections—**Medium, growth conditions, and

EXPERIMENTAL PROCEDURES

Expression and Reporter Plasmids—pGST-Cbfa1 fusion protein resulted from insertion of the BamHI/EcoRI fragment of Cbfa1 (58) into the pGex1 vector (Amersham Pharmacia Biotech) opened with BamHI/EcoRI. pGST-Cbfa1ΔRN1 resulted from deletion of the BamHI/SmaI fragment from GST-Cbfa1 and religation of blunt ends after fill-in. pGST-Cbfa1ΔRN2 was obtained by deletion of the BamHI/ApaI fragment from pGST-Cbfa1 followed by fill-in and religation. pGST-Cbfa1ΔC was cloned upon the deletion of the HindIII/EcoRI fragment from pGST-Cbfa1 followed by fill-in and religation. pGST-Cbfa1ΔRC was derived upon deletion of the NcoI/EcoRI fragment from pGST-Cbfa1 followed by fill-in and religation. For the construction of pGST-Cbfa1ΔNC or pGST-Cbfa1ΔN, the NcoI/HindIII fragment or the NcoI/EcoRI fragment was isolated from pGST-Cbfa1 and introduced in pBluescript SK(+) (Stratagene) linearized with SmaI/HindIII or SmaI/EcoRI (giving rise to pBl-Cbfa1ΔNC or pBl-Cbfa1ΔN). The BamHI/HindIII fragment of pBl-Cbfa1ΔNC or the BamHI/EcoRI fragment of pBl-Cbfa1ΔN was introduced into pGex1 (Amersham Pharmacia Biotech) linearized with BamHI/HindIII or BamHI/EcoRI. pGST-Cbfa1ΔRN3 resulted from a deletion of the SmaI/EcoRI sequence from the pGST-Cbfa1ΔRN2 vector. The XhoI/EcoRI fragment of Cbfa2 excised from the pBluescript clone was introduced into pGex1 (Amersham Pharmacia Biotech) linearized with SmaI and EcoRI to generate pGST-Cbfa2. The pGST-Cbfa2ΔC mutant was cloned by removal of the SmaI/EcoRI fragment, which was used to built the pGST-Cbfa2ΔRN vector by insertion of this fragment into pGex3 (Amersham Pharmacia Biotech) linearized with SmaI/EcoRI. All constructs have been sequenced to confirm that the Cbfa/Runt fragments are in frame with the GST sequence, and the constructs are schematically shown in Fig. 1a.

pBAT-Cbfa1 resulted from the ligation between the SalI/EcoRV fragment of Cbfa1 excised from pBSK-Cbfa1 into the pBAT-c-Jun vector (36), where the c-Jun coding sequence was removed by SalI/EcoRV restriction. Plasmids for in vitro translation of full-length c-Jun and the mutants Δ6–194, Δ6–223, Δ146–221, and Δ194–223 were described previously (37, 38). For construction of pBAT-c-Jun basic D, a HindIII/PstI fragment was excised from pBAT-c-JunΔ6–194 and a PstI/XbaI fragment was excised from pBAT-c-Jun m1 (36). Both fragments were inserted into the pBAT vector linearized with the restriction enzymes HindIII and XbaI. The plasmid for in vitro translation of full-length c-Fos was described previously (36), and the plasmids for c-Fos-298, c-Fos-218, and c-Fos-171 were kindly provided by R. Müller (69).

Expression vectors for c-Fos-HA and c-Jun-HA in which the tag is in frame in the C-terminal end of the c-Fos and c-Jun genes were kindly provided by D. Bohmann. pCEV is the expression vector for the PKA catalytic subunit and is described in Uhler and McKnight (39). The Myc-tagged Cbfa1 construct (pcDNA3.1-Cbfa1) resulted from the insertion of the XbaI/

Functional Interaction of Fos/Jun and Cbfa/Runt Proteins

FIG. 1. Interaction between purified recombinant Cbfa/Runt proteins and in vitro translated c-Jun or c-Fos. $a$, schematic representation of GST-Cbfa/Runt constructs used for pull-down analysis. QA, glutamine- and alanine-rich region; Runt, Runt domain; AD, activation domain; ID, inhibitory domain. $b$, pull-down experiments were performed using full-length GST-Cbfa1 or deletion mutants and in vitro translated c-Jun and c-Fos labeled with ${}^{35}$S-methionine. The input amount of in vitro translated c-Fos or c-Jun proteins is shown in lane 1. $c$, pull-down experiments were performed with GST-Cbfa1 or GST-Cbfa1$\Delta$C and in vitro translated c-Jun and c-Fos labeled with ${}^{35}$S-methionine. Lanes 1 and 2 show in vitro translated c-Jun and c-Fos proteins. $d$, pull-down experiment with full-length GST-Cbfa2 or deletion mutants and in vitro translated c-Jun and c-Fos labeled with ${}^{35}$S-methionine. The quality and the quantity of recombinant GST fusion proteins used for these GST pull-down assays was estimated by Coomassie staining, and the amount of GST proteins used in the experiment was calculated to be equal (data not shown).

treatment of cells with PTH or TPA have been described previously (9). Human HEK293 embryonic kidney cells and mouse F9 embryonic carcinoma cells were transiently transfected according to the calcium phosphate method as described by Angel et al. (43). Rat UMR106 osteosarcoma cells were transfected by electroporation using the 0.4-cm cuvettes and setting the Gene-pulser (Bio-Rad) to 0.25 kV and 960 microfarads. Transfected cells were split on two 94-mm dishes and cultured overnight. For PTH stimulation, after a 24-h culture, transfected cells were treated for 8 h with PTH.

Measurement of luciferase and $\beta$-galactosidase activity was performed as described (40) using a TD-20/20 Luminometer (Turner Designs). pRSV-lacZ was co-transfected in all experiments, and $\beta$-galactosidase activity was used to normalize for different transfection efficiencies in the individual experiments. A minimum of three independent transfections were performed and S.D. values were calculated.

**RESULTS**

Cbfa/Runt Family Members Directly Interact in Vitro with c-Jun and c-Fos—To address the question of whether AP-1 and Cbfa/Runt transcription factors could directly interact with each other, we performed in vitro pull-down assays using purified GST-Cbfa1 fusion proteins and in vitro translated ${}^{35}$S-labeled c-Jun or c-Fos proteins (Fig. 1$b$, lane 1). Association with c-Jun and c-Fos was observed using GST-Cbfa1 but not using GST alone (Fig. 1$b$, lanes 2 and 3). Comparison of the input amount of in vitro translated c-Jun or c-Fos with the amount of the proteins pulled down by GST-Cbfa1 suggests a higher affinity of Cbfa1 for c-Jun as for c-Fos. Posttranslational modifications (e.g., phosphorylation) of c-Jun or c-Fos occurred in the rabbit reticulocyte lysate (38) and were visible as additional shifted bands in the SDS-PAGE but did not influence the interaction with GST-Cbfa1. Vice versa, in vitro translated ${}^{35}$S-labeled Cbfa1 was pulled down by recombinant GST-c-Jun or GST-c-Fos. In agreement with the initial data, the binding activity between Cbfa1 and c-Jun was stronger than that between Cbfa1 and c-Fos (data not shown).

Next we determined the domains of Cbfa1 responsible for the interaction with c-Jun or c-Fos. Therefore, Cbfa1 deletion mutants fused to GST were generated (Fig. 1$a$), and purified proteins were incubated with in vitro translated, ${}^{35}$S-labeled c-Jun or c-Fos proteins in GST pull-down assays. c-Jun could associate with GST-Cbfa1 deletion mutants lacking either the C- or N-terminal region or even both, but not with mutants missing the Runt domain of Cbfa1 (Fig. 1$b$). Similarly, c-Fos could interact with GST-Cbfa1 deletion mutants lacking either the C or N terminus or both (Fig. 1$b$). These data demonstrate that the Runt domain is sufficient for interaction with c-Jun or c-Fos. However, additional regions within the Cbfa1 protein were found to be involved in the association; c-Fos could be pulled down by GST-Cbfa1$\Delta$RN1 and GST-Cbfa1$\Delta$RN2, the mutants lacking the Runt domain but containing the C terminus of Cbfa1 (Fig. 1$b$, lanes 6 and 9). The interaction between Cbfa1 and c-Fos was abolished when, in addition to the Runt domain, the extreme C-terminal end of Cbfa1 was deleted (Fig. 1$b$, lane 10). Furthermore, deletion of the N terminus of Cbfa1 resulted in decreased binding of c-Jun but enhanced binding of c-Fos, suggesting that binding affinity for either c-Jun or c-Fos could be specifically regulated by motifs in this region.

To investigate if c-Jun and c-Fos are able to interact simultaneously with Cbfa1, we used either full-length GST-Cbfa1 or GST-Cbfa1$\Delta$C fusion proteins together with in vitro translated c-Jun and/or c-Fos labeled with ${}^{35}$S-methionine for pull-down assays (Fig. 1$c$). Consistent with the previous results, both c-Jun and c-Fos associate with GST-Cbfa1 as well as GST-Cbfa1$\Delta$C (Fig. 1$c$). Although c-Jun and c-Fos interact with the Runt domain, no difference between the pull-down assays with c-Jun and c-Fos alone or with the combination of the two proteins was detectable, even if the GST-Cbfa1$\Delta$C mutant was used, which lacks the additional binding site for c-Fos. Consequently, we suggest that c-Jun and c-Fos could associate with Cbfa1 as a monomer or as a Jun-Fos heterodimer and possibly as a Jun homodimer.

Cbfa2 is another Cbfa/Runt family member that has been postulated to bind to the upstream OSE2 site on the mmp13 promoter (9). The members of the Cbfa/Runt family are highly

20032 Functional Interaction of Fos/Jun and Cbfa/Runt Proteins

a

| I | δ | II | III | bZIP |
| --- | --- | --- | --- | --- |
|  |  |  |  | c-Jun |
|  |  |  |  | c-JunΔ6-194 |
|  |  |  |  | c-JunΔ6-223 |
|  |  |  |  | c-JunΔ146-221 |
|  |  |  |  | c-JunΔ194-223 |
|  |  |  |  | c-Jun-basicD |

b

| in vitro translation | GST pulldown |
| --- | --- |
| + - - - - | + + - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Functional Interaction of Fos/Jun and Cbfa/Runt Proteins

FIG. 3. Interaction of Cbfa/Runt proteins and c-Jun expressed in mammalian cells. $a$, pull-down experiments were performed with GST-Cbfa/Runt proteins and crude extracts prepared from HEK293 cells transiently transfected with 5 $\mu$g of expression vectors encoding c-Jun-HA or c-Fos-HA. The transfected cells were treated or not treated with 100 ng/ml TPA for 2 h or co-transfected with 5 $\mu$g of an expression vector encoding the catalytic subunit of PKA. Expression of exogenous c-Jun-HA or c-Fos-HA and co-precipitation of these proteins was analyzed by Western blot using an anti-HA antibody. Equal amounts of GST-Cbfa1 and GST-Cbfa2 used in pull-down assays was also tested by Western blot with an anti-GST antibody. $b$, co-immunoprecipitation of Cbfa1-Myc and c-Jun-HA expressed in F9 cells. F9 cells were transiently transfected with 10 $\mu$g of a Cbfa1-Myc expression vector and 5 $\mu$g of a c-Jun-HA expression vector. Expression of the proteins was analyzed by Western blot with anti-Myc or anti-HA antibodies (*, nonspecific signal). Immunoprecipitation (IP) was performed with 100–200 $\mu$g of crude extract and 2 $\mu$g of the anti-Myc antibody, and co-precipitation of c-Jun-HA was analyzed by Western blot with the anti-HA antibody. $c$, pull-down experiments were performed with purified GST-Cbfa/Runt proteins and 200–300 $\mu$g of crude extracts from UMR106 cells. UMR106 cells were left untreated or were treated with $10^{-8}$ M rat PTH fragment (residues 1–34) or 100 ng/ml TPA for 2 h. Expression of endogenous c-Jun in crude cell extracts or c-Jun interacting with GST-Cbfa1 or GST-Cbfa2 was analyzed by Western blot using an anti-c-Jun antibody. Equal amounts of GST-Cbfa/Runt proteins used were confirmed by Western blot using an anti-GST antibody. $d$, pull-down experiment with GST-c-Jun$\Delta$6–223 and 200–300 $\mu$g of cell extracts from untreated or PTH-treated UMR106 cells. The expression of endogenous Cbfa1 and of the co-precipitated protein was analyzed by Western blot using an anti-Cbfa1 antibody.

To demonstrate that interaction occurs in cultured cells as well, we transiently transfected F9 embryonal carcinoma cells with expression vectors for Myc-tagged Cbfa1 (Cbfa1-Myc) and HA-tagged c-Jun (c-Jun-HA). Expression of tagged proteins was confirmed by Western blot using a monoclonal anti-Myc antibody or a polyclonal anti-HA antibody (Fig. 3$b$). Interaction was investigated by co-immunoprecipitation experiments with the anti-Myc antibody followed by a Western blot with the anti-HA antibody. Co-precipitation of c-Jun-HA was only detectable in cells co-transfected with Cbfa1-Myc, confirming interaction between these two proteins in cultured cells.

Transcription from the $mmp13$ promoter can be induced by TPA or PTH treatment. TPA activates PKC and the mitogen-activated protein kinase pathway and enhances phosphorylation of c-Jun, whereas PTH predominantly activates the PKA signaling pathway. Although c-Jun is not phosphorylated by PKA, both the OSE2 and the TRE elements in the promoter region are necessary for PTH-dependent activity of MMP13 transcription (8, 9). To investigate if posttranslational modifications of c-Jun downstream of TPA or PTH signaling pathways would modify the interaction between c-Jun and Cbfa/Runt proteins, GST pull-down assays were performed. To yield eventually modified c-Jun, HEK293 cells expressing c-Jun-HA were either treated with TPA or co-transfected with an expression vector for PKA, leading to overexpression of the catalytic subunit, which can not be titrated by the lower amount of endogenous regulatory subunits (39). Neither TPA treatment triggering phosphorylation of c-Jun nor overexpression of the catalytic PKA subunit revealed an alteration in the amount of c-Jun-HA pulled down by GST-Cbfa1 or GST-Cbfa2 (Fig. 3$a$, lanes 4 and 9). To use a more *in vivo* relevant system to study the role of TPA or PTH on the Cbfa1/c-Jun interaction, we used the UMR106 rat osteosarcoma cell line. Treatment of these cells by TPA or PTH revealed an induced expression of endogenous c-Jun and c-Fos (Fig. 3$c$, lanes 1–3, and data not shown). In crude extracts from untreated UMR106 cells or from cells treated by TPA or PTH, c-Jun could be pulled down by GST-Cbfa1 and GST-Cbfa2 but not by the GST control (Fig. 3$c$, lanes 4, 7, and 10). TPA treatment of the cells specifically increased the amount of associated c-Jun that most likely could be due to enhanced synthesis of the protein (Fig. 3$c$). Applying similar conditions, we could see only very weak binding of c-Fos to GST-Cbfa2 appearing after a long exposure of the protein gels (data not shown). Endogenous Cbfa1 from UMR106 cell lysates could be pulled down by GST-c-Jun$\Delta$6–223, a c-Jun GST fusion protein lacking its transcriptional activation domain, but not by GST alone, showing that this protein-protein interaction was affected neither by the c-Jun transcriptional activation domain nor by posttranslational modifications on Cbfa1 induced upon PTH treatment (Fig. 3$d$).

A New OSE2 Element Overlaps the TRE Motif and Affects AP-1 Binding—Recently, we identified an OSE2 element in close proximity to the TRE site in the mouse and rat $mmp13$ promoter (9). Upon close analysis, we could identify an additional GTGGGT sequence partially overlapping with the TRE site (Fig. 4$a$). We will refer to this sequence as the OSE2/TRE motif. Both recombinant GST-Cbfa1 and GST-Cbfa2 were able to bind to the OSE2/TRE-Coll probe (Fig. 4, $b$ and $c$, lanes 1–3, 7, and 8). The difference in complex formation observed between GST-Cbfa1 and GST-Cbfa2 is most likely due to slight differences in protein amounts, which were calculated by Coomassie staining of the purified proteins after SDS-PAGE. Alternatively, we cannot exclude the possibility that under these conditions (e.g. in the absence of Cbf$\beta$) GST-Cbfa2 exhibits a

20034

**Functional Interaction of Fos/Jun and Cbfa/Runx Proteins**

pre-treatment with anti-c-Jun or anti-c-Fos antibodies (Fig. 5a, upper panel). Anti-Cbfa1#70 specifically supershifted Cbfal associated with DNA (Fig. 5a, middle panel) but did not interfere with binding of the ubiquitous transcription factor Oct1 to the consensus octamer binding site (Fig. 5a, lower panel). These data demonstrate complex formation of Cbfal and AP-1 members at the proximal OSE2/TRE motif, which additionally might be stabilized in vivo by Cbfa/Runx-binding at the distal OSE2 element within the mmp13 promoter.

To investigate whether the newly identified OSE2 element contributes to the complex formation at the OSE2/TRE motif, we performed a bandshift assay with nuclear extracts from PTH-treated UMR106 cells and OSE2/TRE-Col1 or mutOSE2/TRE-Col1 probes. We observed a reduced complex formation if the OSE2 site was mutated (Fig. 5b, lanes 3 and 4). The reduced bandshift activity was most likely due to interference with binding of Cbfal at the mutated OSE2 element. However, we could not exclude the possibility that the introduced mutation affects AP-1 binding at the TRE element independent of Cbfal function. Therefore, we performed a bandshift analysis with in vitro translated c-Jun and c-Fos in the absence of Cbfa/Runx proteins. No major differences in AP-1 binding activity to the OSE2/TRE-Col1 or the mutOSE2/TRE-Col1 probes was detected (Fig. 5c, lanes 2 and 4), suggesting that in the absence of Cbfa/Runx proteins, the heterodimer of c-Jun/c-Fos bound to both oligonucleotides with similar affinity. This observation excluded the possibility that the sequence flanking the TRE site at the mmp13 promoter is required for AP-1 binding in the absence of Cbfa/Runx proteins. Therefore, we propose that Cbfa/Runx proteins bound to the overlapping OSE2 site enhance the stabilization of AP-1-DNA complex formation by direct protein-protein interaction.

Interaction between Cbfal and AP-1 Increases Transcriptional Activation by AP-1—To assess the functional relevance of Cbfal binding to AP-1 and the OSE2/TRE motif, we analyzed the transcriptional activity of Cbfal on a luciferase reporter gene driven by a mmp13 promoter fragment (mCol1-luc(−66/+29)) containing the OSE2/TRE motif. HEK293 cells were transiently transfected with an expression vector encoding Cbfal together with mCol1-luc(−66/+29). Expression levels of endogenous c-Jun and c-Fos were not influenced by the presence of the Cbfal-Myc protein (Fig. 6b), showing that the effects on luciferase activity were due to Cbfal expression and not due to changes in c-Jun or c-Fos levels. Increasing amounts of exogenous Cbfal resulted in a concomitant increase of mmp13 promoter transcriptional activity (data not shown). A 3-fold stimulation of transcriptional activity was measured by transfection of 10 μg of Cbfal expression vector together with mCol1-luc(−66/+29). In contrast, the control luciferase vector (mCol1-luc(0)) was not activated by Cbfal (Fig. 6, c and d). Upon mutation of the OSE2 site in the −66/+29 promoter (Fig. 6c), the inducibility is reduced to 30–50% (Fig. 6c). Similar results were obtained upon transient transfection of GM637 human fibroblasts (data not shown). Therefore, we conclude that Cbfal can induce transcription of the −66/+29 mmp13 promoter fragment through the newly identified OSE2 site. Nevertheless, when the OSE2 site is mutated, Cbfal can still interact with AP-1 and trigger partial stimulation of the promoter.

Transfection of HEK293 cells with an expression vector for a Cbfal mutant lacking the C-terminal region (CbfalΔC) stimulated luciferase expression to a similar extent as the full-length Cbfal (Fig. 6d). This observation suggests that the C-terminal region of Cbfal containing activation and repression domains is not involved in transcriptional activation of the −66/+29

FIG. 4. Identification of a new OSE2 site overlapping with the conserved TRE motif in the proximal mmp13 promoter. Shown are schematic representations of the OSE2/TRE motif in the proximal mmp13 promoter region of the mouse and rat genes (a) and of the mutations within the OSE2/TRE motif used for EMSA experiments (b). c, EMSA experiments were performed with equal amounts of purified GST or GST-Cbfa/Runx fusion proteins and 32P-labeled wild type or mutant OSE2/TRE-Col1 probes.

lower binding affinity to the OSE2/TRE element compared with GST-Cbfal. Mutations within the putative OSE2 site (mutOSE2/TRE-Col1 and mutOSE2/mutTRE-Col1) interfered with binding of GST-Cbfa/Runx to the OSE2/TRE motif (Fig. 4c, lanes 4–6, 9, and 10). Upon mutation of the TRE site (OSE2/mutTRE), the GST-Cbfa/Runx proteins were still able to bind the OSE2 site and to shift the probe (Fig. 4c, lanes 11 and 12). As a control, the recombinant GST-Cbfa/Runx proteins were preincubated with an anti-GST antibody, or unlabeled OSE2-Col1 probe containing the distal OSE2 element of the mmp13 promoter was added in excess. Both resulted in impaired binding of GST-Cbfa/Runx proteins to this OSE2/TRE site (data not shown).

Since Cbfa/Runx proteins can bind c-Jun and c-Fos in vitro as well as in cultured cells, we wanted to analyze whether this association in combination with the presence of the overlapping OSE2 motif could modify AP-1 binding on the OSE2/TRE site. To investigate whether Cbfa/Runx proteins are associated with the AP-1 complex on the OSE2/TRE motif, we preincubated PTH-treated UMR106 cell nuclear extracts with an anti-Cbfal polyclonal antibody (anti-Cbfal#70) or added a nonlabeled OSE2-Col1 probe containing the conserved OSE2 motif of the distal mmp13 promoter. The addition of the competitor OSE2-Col1 probe did not result in a significant change in AP-1-DNA complex formation (data not shown), suggesting that Cbfa/Runx proteins are able to simultaneously interact with the OSE2 motif as well as AP-1 members bound to DNA. In contrast, pretreatment with anti-Cbfal#70 resulted in decreased complex formation at the OSE2/TRE-Col1 probe, similar to

Functional Interaction of Fos/Jun and Cbfa/Runt Proteins 20035

FIG. 5. Binding of the Cbfa/Runt proteins to the AP-1 complex and the newly identified OSE2 site in the OSE2/TRE motif is required for efficient AP-1-DNA complex formation at the TRE site. EMSA experiments were performed with 2 μg of nuclear extracts from PTH-treated UMR106 cells, which were preincubated with anti-c-Jun, anti-c-Fos, or anti-Cbfal antibodies (a), 2 μg of nuclear extracts from untreated (-PTH) or PTH-treated (+PTH) UMR106 cells (b), or in vitro translated c-Jun and c-Fos proteins together with 32P-labeled wild type (OSE2/TRE-Coll) or mutant (mutOSE2/TRE-Coll) probes (c). To confirm the specificity of the antibodies used in a, supershift experiments with a 32P-labeled OSE2-Coll probe spanning the distal OSE2 element of the mmp13 promoter or a 32P-labeled Oct probe containing a consensus octamer binding site were done.

promoter. Although we cannot exclude the involvement of the remaining N-terminal region of Cbfal, it is more feasible that the interaction between Cbfal and AP-1 is responsible for enhanced transcription due to stabilization of the complex formation on DNA.

The New OSE2 Site Is Involved in PTH-induced Transcriptional Activation of the mmp13 Promoter—The TRE motif and a distal OSE2 site located 80 base pairs upstream have been previously shown to be essential for PTH-dependent induction of MMP13 expression (8, 9). To study the involvement of the newly identified OSE2 site of the OSE2/TRE motif in the PTH response, we transiently transfected UMR106 cells with control mColl-luc(0), wild-type mColl-luc(-663/+29), or mutant mColl-luc(-663/+29)sdm reporter constructs (Fig. 7a) and measured reporter gene expression in untreated and PTH-treated cells. In agreement with the transient transfections in fibroblasts, mutation of the OSE2 site in the OSE2/TRE motif did not influence basal transcriptional activity (Fig. 7b). PTH treatment of UMR106 cells transfected with mColl-luc(-663/+29) resulted in a 2-fold induced luciferase activity compared with the basal level (Fig. 7b). In contrast, PTH-induced luciferase expression was significantly reduced if the OSE2 site in the OSE2/TRE motif was mutated, confirming that this site is crucial for full activation of the mmp13 promoter after PTH treatment.

DISCUSSION

In the present work, we show for the first time that Cbfa/Runt proteins interact directly with c-Jun and c-Fos. This interaction is mediated by the Runt domain and the leucine zipper, respectively. Moreover, the leucine zipper of c-Fos can also interact with a region in the C terminus of Cbfal and Cbfa2. Furthermore, we identified a new OSE2 site overlapping the TRE site in the proximal promoter region of the murine and rat mmp13 gene (OSE2/TRE motif). The newly identified OSE2 site is involved in (i) stabilization of AP-1 DNA binding on the TRE site, (ii) enhancement of AP-1 transcriptional activity in the presence of Cbfal, and (iii) PTH-dependent induction of the mmp13 promoter activity in UMR106 osteoblast-like cells.

Interaction between Cbfa/Runt and c-Jun or c-Fos proteins has been shown in vitro by GST pull-down experiments. Furthermore, we could demonstrate in vivo interaction between Cbfal and c-Jun expressed in F9 cells by co-immunoprecipitation. GST pull-down experiments revealed no difference between monomers and dimers of Jun/Fos proteins in their efficiency to interact with Cbfal. Therefore, we assume that only one component of the AP-1 dimer might interact with the Cbfa/Runt protein. In light of the higher binding affinity between Cbfa/Runt proteins and c-Jun, one could imagine that c-Jun might be a better candidate for the interaction with Cbfa/Runt proteins. We speculate that the residues in the leucine zipper involved in the dimerization between c-Jun and c-Fos on one hand and in the interaction with Cbfa/Runt proteins on the other hand would not be identical, because one would expect a competition in interaction between c-Jun and c-Fos with Cbfa/Runt.

Analysis of deletion mutants of Cbfal or Cbfa2 and c-Jun or c-Fos revealed direct interaction between the Runt domain and the leucine zipper, respectively. It has been demonstrated that the Runt domain is specifically involved in the interaction with the co-factor Cbfβ (44) and other transcription factors, like Ets-1 (30), PU.1 (35), and C/EBP proteins (31, 34, 35). For example, the Runt domain of Cbfa2 interacts with PU.1 and C/EBPα in the synergistic activation of the human macrophage colony-stimulating factor receptor promoter in myeloid cell lines (31, 35). Moreover, synergistic transcriptional activity due

20036

**Functional Interaction of Fos/Jun and Cbfa/Runx Proteins**

a

- luciferase - mColl-luc(0)
- OSE2 TRE - luciferase - mColl-luc(-66/+29)
- OSE2 TRE - luciferase - mColl-luc(-66/+29)-sdm

b

- 10 μg
- Cbfal
- anti-myc ← Cbfal-myc
- anti-cJun ← cMyc
- cJun
- anti-cFos cFos
- 1 2

c

![Graph](https://i.imgur.com/placeholder.png)

d

![Graph](https://i.imgur.com/placeholder.png)

Cbfa1
mColl-luc(0)
mColl-luc(-66/+29)
mColl-luc(-66/+29)sdm

Cbfal
mColl-luc(-66/+29)
CbfalΔC
mColl-luc(-66/+29)

FIG. 6. Cbfal enhances AP-1-mediated transactivation in HEK293 cells. $a$, schematic representation of the luciferase reporter plasmids regulated by the $-66/+29$ $mmp13$ promoter fragment. mColl-luc(0) represents a control vector lacking the $mmp13$ promoter fragment. mColl-luc($-66/+29$) contains the $-66/+29$ promoter fragment of the murine $mmp13$ gene with the proximal OSE2/TRE element in front of the luciferase reporter gene. mColl-luc($-66/+29$)sdm was generated by mutating the OSE2 site in the mColl-luc($-66/+29$) vector. $b$, HEK293 cells were transiently transfected with 10 μg of an expression vector for Cbfal-Myc. Expression of exogenous Cbfal-Myc or endogenous c-Jun and c-Fos proteins were detected by Western blot analysis. Similar amounts of endogenous c-Myc protein detected with the anti-Myc antibody indicate similar quality and quantity of loaded protein extracts. $c$, Cbfal was co-transfected with 5 μg of the indicated luciferase vectors. Relative luciferase activity obtained with mColl-luc(0) reporter construct alone was brought to the value of 1. $d$, 10 μg of Cbfal or CbfalΔC expression vectors were co-transfected with 5 μg of mColl-luc($-66/+29$). Relative luciferase activity obtained with mColl-luc($-66/+29$) reporter construct alone was brought to a value of 1. For $c$ and $d$, a minimum of three independent transfections were performed, and S.D. values were calculated.

to direct interaction between the Runt domain of Cbfal and C/EBPδ has been shown in osteoblasts (34).

Interestingly, superposition of the AML1/Cbfβ coordinates (Protein Data Bank code 1e50) onto the NFAT-AP-1-DNA complex (Protein Data Bank code 1a02) suggests that the Jun/Fos interaction site on the Runt domain overlaps with that of Cbfβ (44–46). Thus, it will be interesting to investigate whether Cbfβ and c-Jun or c-Fos can bind simultaneously or in a competitive fashion to the Runt domain. Since there exists evidence that Cbfβ is constitutively present in the Cbfal complex bound on the distal OSE2 sites independently of PTH treatment (8), we propose that Cbfβ, Cbfa/Runt, and AP-1 proteins can interact simultaneously. This idea was further supported by the observation that we were able to co-precipitate *in vitro* translated c-Jun, c-Fos, and Cbfβ using GST-Cbfal. Interestingly, the affinity between Cbfal and the AP-1 members appeared to be enhanced in the presence of the co-factor (data not shown).

In addition to the requirement of the leucine zipper and the Runt domain for interaction, we could demonstrate an interaction between the leucine zipper of c-Fos and the C-terminal region of Cbfa/Runt proteins (amino acids 467–514 of Cbfal). Interestingly, the region of Cbfal containing residues 443–516 is highly homologous to a domain in the C-terminus of Cbfa2 responsible for transcriptional repression function (47, 48). It has been suggested that the repression activity is due to the recruitment of co-repressors of the transducin-like enhancer of split (TLE) family, which also interact with the same C-terminal region of Cbfal and Cbfa2 (24–26). Therefore, it is tempting to speculate that c-Fos could interfere with the interaction between Cbfa/Runt and TLE proteins, thereby reducing the inhibitory effect exerted by the TLEs on the transcriptional activity of Cbfa/Runt proteins. A similar model has been postulated for the competitive binding to Cbfal C-terminal end between TLE and HES-1 (hairy/enhancer of split-1), which can potentiate Cbfal-mediated transactivation (26). However, a possible competitive binding between c-Fos and TLE proteins to the C terminus of Cbfa/Runt transcription factors remains to be demonstrated.

The interaction between Cbfa/Runt and AP-1 transcription factors suggests that AP-1 transcriptional activity is potentiated by Cbfa/Runt proteins. In addition, a number of observations suggest that DNA-binding of Cbfal to the newly identified OSE2 site in the proximal region of the murine $mmp13$ promoter would contribute to enhanced transcriptional activity of AP-1. (i) nuclear extract from PTH-treated UMR106 cells showed reduced DNA binding activity of AP-1 on the OSE2/TRE motif when the OSE2 site was mutated. However, c-Jun and c-Fos translated *in vitro* and used in excess for EMSA experiments bound to the TRE site in this OSE2/TRE element with the same efficiency regardless of the OSE2 sequence. (ii) Basal activity of the proximal $mmp13$ promoter ($-66/+29$), harboring the OSE2/TRE element, was identical in fibroblasts

4 A. Warren, personal communication.

Functional Interaction of Fos/Jun and Cbfa/Runt Proteins

FIG. 7. The new OSE2 site is involved in PTH-dependent activation of the mmp13 promoter in UMR106 cells. $a$, schematic representation of luciferase reporter constructs. mColl-luc(-663/+29) contains the -663/+29 promoter fragment of the murine *mmp13* gene with the distal OSE2 sites and the proximal OSE2/TRE element in front of the luciferase reporter gene. mColl-luc(-663/+29)-sdm was generated by mutating the proximal OSE2 site in the OSE2/TRE element of the mColl-luc(-663/+29) vector. $b$, 5 $\mu$g of the indicated luciferase reporter constructs were transiently transfected in UMR106 cells. 24 h after transfection, cells were treated with $10^{-8}$ M PTH for 2 h. Relative luciferase activity obtained with mColl-luc(-663/+29) reporter construct or mColl-luc(-663/+29)-sdm reporter construct in the absence of PTH treatment was brought to the value of 1. The inset graph shows similar basal activity of mColl-luc(-663/+29) and mColl-luc(-663/+29)-sdm in transiently transfected UMR106 cells in the absence of PTH.

regardless of the presence or absence of a functional OSE2 site. Nevertheless, transactivation of this promoter by co-transfection of a Cbfa1 expression vector was 30–50% higher when the OSE2 site was intact. (iii) Mutation of the newly identified OSE2 site inhibits PTH-dependent induction of the *mmp13* promoter by more than 50% in transiently transfected UMR106 cells. Therefore, PTH-dependent induction of the *mmp13* promoter does not only require the previously characterized distal OSE2 and proximal TRE sites (8, 9) but seems to require also the binding of Cbfa1 to the proximal OSE2/TRE site. It has been shown that the level of Cbfa1 protein in osteoblasts is not affected by PTH treatment and that the distal OSE2 sites are constitutively bound by Cbfa/Runt proteins (8, 9, 18). These observations do not exclude the possibility of a synergistic binding of Cbfa/Runt and AP-1 transcription factors at the proximal OSE2/TRE element. Numerous examples have been reported describing direct interaction and cooperation of AP-1 with other transcription factors, like Ets (49–51), MyoD, (52), Smads (53, 54), Rb (55), and NF-$\kappa$B (56). Thus, a model could be proposed, in which Cbfa/Runt and AP-1 transcription factors synergistically bind to the proximal OSE2/TRE motif. Protein-protein interaction as well as DNA binding of Cbfa/Runt proteins to the OSE2/TRE motif could be responsible for the observed synergism. In addition to PTH-dependent synergy between the OSE2 and TRE sites at the composite OSE2/TRE element in the proximal promoter region, the distal OSE2 sites and the TRE site in the OSE2/TRE element may also contribute to functional synergism. At present, both possibilities seem to be important for PTH-dependent expression of MMP13 in osteoblasts, since mutation of the proximal OSE2 site did not result in the complete loss of PTH-induced transcriptional activity.

Cbfa1 has an essential role in bone development, since it is required for osteoblast differentiation and is mutated in patients with cleidocranial dysplasia (57–60), whereas Cbfa2 is required for hematopoiesis and is a frequent target of chromosomal translocations in acute leukemias (61). Cbfa/Runt binding sites have been characterized in the promoter of different genes that are specifically expressed in bone development, such as MMP13, osteocalcin, osteopontin, osteoprotegerin, type I collagen, bone sialoprotein, and Cbfa1 (9, 57, 62–65), or in the promoter of genes involved in hematopoiesis, such as the granulocyte-macrophage colony-stimulating factor, macrophage colony-stimulating factor receptor, myeloperoxidase, neutrophil elastase, and genes of the T-cell receptor subunits (reviewed in Refs. 66 and 67).

Our work demonstrates that Cbfa/Runt proteins promote transcriptional activity of AP-1 on the promoter of the murine *mmp13* gene. In the future, it will be interesting to address the question of whether synergistic activation mediated by AP-1 and Cbfa/Runt transcription factors also contributes to tissue-specific expression of other genes involved in bone development, hematopoiesis, and pathological processes.

Acknowledgments—We are grateful to D. Sandmaier for excellent technical assistance. We thank R. Müller for plasmids encoding c-Fos deletion mutants, D. Bohmann for c-Fos-HA and c-Jun-HA expression vectors, and W. Schmid for the PKA expression vector. We thank S. Andrecht for help with the figures and M. Schorpp-Kistner, B. Hartenstein, and A. Warren for critical reading of the manuscript.

REFERENCES

1. Knauper, V., Will, H., Lopez-Otin, C., Smith, B., Atkinson, S. J., Stanton, H., Hembry, R. M., and Murphy, G. (1996) *J. Biol. Chem.* **271**, 17124–17131
2. Gack, S., Vallon, R., Schmidt, J., Grigoriadis, A., Tuckermann, J., Schenkel, J., Weiher, H., Wagner, E. F., and Angel, P. (1995) *Cell Growth Differ.* **6**, 759–767
3. Stahle-Backdahl, M., Sandstedt, B., Bruce, K., Lindahl, A., Jimenez, M. G., Vega, J. A., and Lopez-Otin, C. (1997) *Lab. Invest.* **76**, 717–728
4. Johansson, N., Saarialho-Kere, U., Airola, K., Herva, R., Nissinen, L., Westermarck, J., Vuorio, E., Heino, J., and Kahari, V. M. (1997) *Dev. Dyn.* **208**, 387–397

5. Varghese, S., Ramsby, M. L., Jeffrey, J. J., and Canalis, E. (1995) *Endocrinology* **136**, 2156–2162

6. Canalis, E., Rydziel, S., Delany, A. M., Varghese, S., and Jeffrey, J. J. (1995) *Endocrinology* **136**, 1348–1354

7. Franchimont, N., Rydziel, S., Delany, A. M., and Canalis, E. (1997) *J. Biol. Chem.* **272**, 12144–12150

8. Selvamurugan, N., Chou, W. Y., Pearman, A. T., Pulumati, M. R., and Partridge, N. C. (1998) *J. Biol. Chem.* **273**, 10647–10657

9. Porte, D., Tuckermann, J., Becker, M., Baumann, B., Teurich, S., Higgins, T., Owen, M. J., Schorpp-Kistner, M., and Angel, P. (1999) *Oncogene* **18**, 667–678

10. Grigoriadis, A. E., Wang, Z. Q., and Wagner, E. F. (1995) *Trends Genet.* **11**, 436–441

11. Winchester, S. K., Selvamurugan, N., D'Alonzo, R. C., and Partridge, N. C. (2000) *J. Biol. Chem.* **275**, 23310–23318

12. Juppner, H., Abou-Samra, A. B., Freeman, M., Kong, X. F., Schipani, E., Richards, J., Kolakowski, L. F., Jr., Hock, J., Potts, J. T., Jr., Kronenberg, H. M., and Segre, G. V. (1991) *Science* **254**, 1024–1026

13. Evans, D. B., Hipskind, R. A., and Bilbe, G. (1996) *J. Bone Miner Res.* **11**, 1066–1074

14. Yang, R., and Gerstenfeld, L. C. (1996) *J. Biol. Chem.* **271**, 29839–29846

15. Jimenez, M. J., Balbin, M., Lopez, J. M., Alvarez, J., Komori, T., and Lopez-Otin, C. (1999) *Mol. Cell. Biol.* **19**, 4431–4442

16. Clohisy, J. C., Scott, D. K., Brakenhoff, K. D., Quinn, C. O., and Partridge, N. C. (1992) *Mol. Endocrinol.* **6**, 1834–1842

17. Rajakumar, R. A., and Quinn, C. O. (1996) *Mol. Endocrinol.* **10**, 867–878

18. Selvamurugan, N., Pulumati, M. R., Tyson, D. R., and Partridge, N. C. (2000) *J. Biol. Chem.* **275**, 5037–5042

19. Verheijen, M. H., and Defize, L. H. (1997) *J. Biol. Chem.* **272**, 3423–3429

20. Xiao, G., Jiang, D., Thomas, P., Benson, M. D., Guan, K., Karsenty, G., and Franceschi, R. T. (2000) *J. Biol. Chem.* **275**, 4453–4459

21. Bruhn, L., Munnerlyn, A., and Grossschedl, R. (1997) *Genes Dev.* **11**, 640–653

22. Ahn, M. Y., Huang, G., Bae, S. C., Wee, H. J., Kim, W. Y., and Ito, Y. (1998) *Proc. Natl. Acad. Sci. U. S. A.* **95**, 1812–1817

23. Kitabayashi, I., Yokoyama, A., Shimizu, K., and Ohki, M. (1998) *EMBO J.* **17**, 2994–3004

24. Levanon, D., Goldstein, R. E., Bernstein, Y., Tang, H., Goldenberg, D., Stifani, S., Paroush, Z., and Groner, Y. (1998) *Proc. Natl. Acad. Sci. U. S. A.* **95**, 11590–11595

25. Javed, A., Guo, B., Hiebert, S., Choi, J. Y., Green, J., Zhao, S. C., Osborne, M. A., Stifani, S., Stein, J. L., Lian, J. B., van Wijnen, A. J., and Stein, G. S. (2000) *J. Cell Sci.* **113**, 2221–2231

26. McLarren, K. W., Lo, R., Grbavec, D., Thirunavukkarasu, K., Karsenty, G., and Stifani, S. (2000) *J. Biol. Chem.* **275**, 530–538

27. Hanai, J., Chen, L. F., Kanno, T., Ohtani-Fujita, N., Kim, W. Y., Guo, W. H., Imamura, T., Ishidou, Y., Fukuchi, M., Shi, M. J., Stavnezer, J., Kawabata, M., Miyazono, K., and Ito, Y. (1999) *J. Biol. Chem.* **274**, 31577–31582

28. Pardali, E., Xie, X. Q., Tsapogas, P., Itoh, S., Arvanitidis, K., Heldin, C. H., ten Dijke, P., Grundstrom, T., and Sideras, P. (2000) *J. Biol. Chem.* **275**, 3552–3560

29. Yagi, R., Chen, L. F., Shigesada, K., Murakami, Y., and Ito, Y. (1999) *EMBO J.* **18**, 2551–2562

30. Kim, W. Y., Sieweke, M., Ogawa, E., Wee, H. J., Englmeier, U., Graf, T., and Ito, Y. (1999) *EMBO J.* **18**, 1609–1620

31. Zhang, D. E., Hetherington, C. J., Meyers, S., Rhoades, K. L., Larson, C. J., Chen, H. M., Hiebert, S. W., and Tenen, D. G. (1996) *Mol. Cell. Biol.* **16**, 1231–1240

32. Zhang, D. E., Hohaus, S., Voso, M. T., Chen, H. M., Smith, L. T., Hetherington, C. J., and Tenen, D. G. (1996) *Curr. Top. Microbiol. Immunol.* **211**, 137–147

33. Westendorf, J. J., Yamamoto, C. M., Lenny, N., Downing, J. R., Selsted, M. E., and Hiebert, S. W. (1998) *Mol. Cell. Biol.* **18**, 322–333

34. McCarthy, T. L., Ji, C., Chen, Y., Kim, K. K., Imagawa, M., Ito, Y., and Centrella, M. (2000) *J. Biol. Chem.* **275**, 21746–21753

35. Petrovick, M. S., Hiebert, S. W., Friedman, A. D., Hetherington, C. J., Tenen, D. G., and Zhang, D. E. (1998) *Mol. Cell. Biol.* **18**, 3915–3925

36. van Dam, H., Huguier, S., Kooistra, K., Baguet, J., Vial, E., van der Eb, A. J., Herrlich, P., Angel, P., and Castellazzi, M. (1998) *Genes Dev.* **12**, 1227–1239

37. Angel, P., Smeal, T., Meek, J., and Karin, M. (1989) *New Biol.* **1**, 35–43

38. Bannister, A. J., Oehler, T., Wilhelm, D., Angel, P., and Kouzarides, T. (1995) *Oncogene* **11**, 2509–2514

39. Uhler, M. D., and McKnight, G. S. (1987) *J. Biol. Chem.* **262**, 15202–15207

40. Pfisterer, P., Annweiler, A., Ullmer, C., Corcoran, L. M., and Wirth, T. (1994) *EMBO J.* **13**, 1655–1663

41. Schreiber, M., Baumann, B., Cotten, M., Angel, P., and Wagner, E. F. (1995) *EMBO J.* **14**, 5338–5349

42. Schorpp, M., Mattei, M. G., Herr, I., Gack, S., Schaper, J., and Angel, P. (1995) *Biochem. J.* **308**, 211–217

43. Angel, P., Allegretto, E. A., Okino, S. T., Hattori, K., Boyle, W. J., Hunter, T., and Karin, M. (1988) *Nature* **332**, 166–171

44. Warren, A. J., Bravo, J., Williams, R. L., and Rabbitts, T. H. (2000) *EMBO J.* **19**, 3004–3015

45. Glover, J. N., and Harrison, S. C. (1995) *Nature* **373**, 257–261

46. Chen, L., Glover, J. N., Hogan, P. G., Rao, A., and Harrison, S. C. (1998) *Nature* **392**, 42–48

47. Thirunavukkarasu, K., Mahajan, M., McLarren, K. W., Stifani, S., and Karsenty, G. (1998) *Mol. Cell. Biol.* **18**, 4197–4208

48. Kanno, T., Kanno, Y., Chen, L. F., Ogawa, E., Kim, W. Y., and Ito, Y. (1998) *Mol. Cell. Biol.* **18**, 2444–2454

49. Wasylyk, B., Wasylyk, C., Flores, P., Begue, A., Leprince, D., and Stehelin, D. (1990) *Nature* **346**, 191–193

50. Bassuk, A. G., and Leiden, J. M. (1995) *Immunity* **3**, 223–237

51. Benbow, U., and Brinckerhoff, C. E. (1997) *Matrix Biol.* **15**, 519–526

52. Bengal, E., Ransone, L., Scharfmann, R., Dwarki, V. J., Tapscott, S. J., Weintraub, H., and Verma, I. M. (1992) *Cell* **68**, 507–519

53. Zhang, Y., Feng, X. H., and Derynck, R. (1998) *Nature* **394**, 909–913

54. Dennler, S., Prunier, C., Ferrand, N., Gauthier, J. M., and Atfi, A. (2000) *J. Biol. Chem.* **275**, 28858–28865

55. Nishitani, J., Nishinaka, T., Cheng, C. H., Rong, W., Yokoyama, K. K., and Chiu, R. (1999) *J. Biol. Chem.* **274**, 5454–5461

56. Yang, X., Chen, Y., and Gabuzda, D. (1999) *J. Biol. Chem.* **274**, 27981–27988

57. Ducy, P., Zhang, R., Geoffroy, V., Ridall, A. L., and Karsenty, G. (1997) *Cell* **89**, 747–754

58. Komori, T., Yagi, H., Nomura, S., Yamaguchi, A., Sasaki, K., Deguchi, K., Shimizu, Y., Bronson, R. T., Gao, Y. H., Inada, M., Sato, M., Okamoto, R., Kitamura, Y., Yoshiki, S., and Kishimoto, T. (1997) *Cell* **89**, 755–764

59. Otto, F., Thornell, A. P., Crompton, T., Denzel, A., Gilmour, K. C., Rosewell, I. R., Stamp, G. W., Beddington, R. S., Mundlos, S., Olsen, B. R., Selby, P. B., and Owen, M. J. (1997) *Cell* **89**, 765–771

60. Mundlos, S., Otto, F., Mundlos, C., Mulliken, J. B., Aylsworth, A. S., Albright, S., Lindhout, D., Cole, W. G., Henn, W., Knoll, J. H., Owen, M. J., Mertelsmann, R., Zabel, B. U., and Olsen, B. R. (1997) *Cell* **89**, 773–779

61. Lutterbach, B., and Hiebert, S. W. (2000) *Gene (Amst.)* **245**, 223–235

62. Geoffroy, V., Ducy, P., and Karsenty, G. (1995) *J. Biol. Chem.* **270**, 30973–30979

63. Frendo, J. L., Xiao, G., Fuchs, S., Franceschi, R. T., Karsenty, G., and Ducy, P. (1998) *J. Biol. Chem.* **273**, 30509–30516

64. Ducy, P., Starbuck, M., Priemel, M., Shen, J., Pinero, G., Geoffroy, V., Amling, M., and Karsenty, G. (1999) *Genes Dev.* **13**, 1025–1036

65. Thirunavukkarasu, K., Halladay, D. L., Miles, R. R., Yang, X., Galvin, R. J., Chandrasekhar, S., Martin, T. J., and Onyia, J. E. (2000) *J. Biol. Chem.* **275**, 25163–25172

66. Lenny, N., Westendorf, J. J., and Hiebert, S. W. (1997) *Mol. Biol. Rep.* **24**, 157–168

67. Speck, N. A., and Terryl, S. (1995) *Crit. Rev. Eukaryotic Gene Expression* **5**, 337–364

68. Annweiler, A., Zwilling, S., and Wirth, T. (1994) *Nucleic Acids Res.* **22**, 4250–4258

69. Schuermann, M., Neuberg, M., Hunter, J. B., Jenuwein, T., Ryseck, R.-P., Bravo R., and Müller, R. (1989) *Cell* **56**, 507–516

MECHANISMS OF SIGNAL
TRANSDUCTION:
AP-1 and Cbfa/Runt Physically Interact
and Regulate Parathyroid
Hormone-dependent MMP13 Expression in
Osteoblasts through a New
Osteoblast-specific Element 2/AP-1
Composite Element

Jochen Hess, Dominique Porte, Christine
Munz and Peter Angel
J. Biol. Chem. 2001, 276:20029-20038.
doi:10.1074/jbc.M010601200 originally published online March 26, 2001

Access the most updated version of this article at doi: 10.1074/jbc.M010601200

Find articles, minireviews, Reflections and Classics on similar topics on the JBC Affinity Sites.

Alerts:
- When this article is cited
- When a correction for this article is posted

Click here to choose from all of JBC's e-mail alerts

This article cites 69 references, 39 of which can be accessed free at
http://www.jbc.org/content/276/23/20029.full.html#ref-list-1
