
The Plant Cell, Vol. 13, 965–978, April 2001, www.plantcell.org © 2001 American Society of Plant Physiologists

# Functional Genomic Analysis of the HY2 Family of Ferredoxin-Dependent Bilin Reductases from Oxygenic Photosynthetic Organisms

**Nicole Frankenberg,**$^{a}$ **Keiko Mukougawa,**$^{b}$ **Takayuki Kohchi,**$^{b}$ and **J. Clark Lagarias**$^{a,1}$

$a$ Section of Molecular and Cellular Biology, University of California at Davis, One Shields Avenue, Davis, California 95616  
$b$ Graduate School of Biological Sciences, Nara Institute of Science and Technology, 8916-5 Takayama, Ikoma, Nara 630-0101, Japan

Phytobilins are linear tetrapyrrole precursors of the light-harvesting prosthetic groups of the phytochrome photoreceptors of plants and the phycobiliprotein photosynthetic antennae of cyanobacteria, red algae, and cryptomonads. Previous biochemical studies have established that phytobilins are synthesized from heme via the intermediacy of biliverdin IX$\alpha$ (BV), which is reduced subsequently by ferredoxin-dependent bilin reductases with different double-bond specificities. By exploiting the sequence of phytochromobilin synthase (HY2) of Arabidopsis, an enzyme that catalyzes the ferredoxin-dependent conversion of BV to the phytochrome chromophore precursor phytochromobilin, genes encoding putative bilin reductases were identified in the genomes of various cyanobacteria, oxyphotobacteria, and plants. Phylogenetic analyses resolved four classes of HY2-related genes, one of which encodes red chlorophyll catabolite reductases, which are bilin reductases involved in chlorophyll catabolism in plants. To test the catalytic activities of these putative enzymes, representative HY2-related genes from each class were amplified by the polymerase chain reaction and expressed in *Escherichia coli*. Using a coupled apophytochrome assembly assay and HPLC analysis, we examined the ability of the recombinant proteins to catalyze the ferredoxin-dependent reduction of BV to phytobilins. These investigations defined three new classes of bilin reductases with distinct substrate/product specificities that are involved in the biosynthesis of the phycobiliprotein chromophore precursors phycoerythrobilin and phycocyanobilin. Implications of these results are discussed with regard to the pathways of phytobilin biosynthesis and their evolution.

## INTRODUCTION

Phytobilins are linear tetrapyrrole molecules synthesized by plants, algae, and cyanobacteria that function as the direct precursors of the chromophores of the light-harvesting phycobiliproteins and of the photoreceptor phytochrome (Beale, 1993; Hughes and Lampter, 1999). The pathways of phytobilin biosynthesis have been elucidated by biochemical fractionation of plant and algal extracts, by overcoming a blocked step with exogenous putative intermediates, and by analysis of linear tetrapyrrole-deficient mutants (Beale and Cornejo, 1991a, 1991b, 1991c; Terry et al., 1993). These studies indicate that the biosynthesis of phytobilins shares common intermediates with heme and chlorophyll biosynthetic pathways to the level of protoporphyrin IX, at which point the latter two pathways diverge by metalation with iron or magnesium (Beale, 1993). Phytobilins are derived from heme, which is converted to biliverdin IX$\alpha$ (BV), the first committed intermediate in their biosynthesis. In red algae, cyanobacteria, and plants, this interconversion is accomplished by ferredoxin-dependent heme oxygenases that are related in sequence to the mammalian heme oxygenase (Cornejo et al., 1998; Davis et al., 1999; Muramoto et al., 1999). Although they catalyze the same reaction, mammalian heme oxygenases use an NADPH-dependent cytochrome P450 reductase to generate reducing power for heme catabolism (Maines, 1988).

The metabolic fate of BV differs in mammals, cyanobacteria, and plants, with BV being metabolized by different reductases with unique double-bond specificities (Figure 1). Mammalian biliverdin IX$\alpha$ reductase (BVR), an NAD(P)H-dependent enzyme that catalyzes the two-electron reduction of BV at the C10 methine bridge to produce bilirubin IX$\alpha$ (BR), was the first of these enzymes to be discovered (Maines and Trakshel, 1993). A similar enzyme, encoded by the gene *bvdR*, was identified recently in cyanobacteria (Schluchter and Glazer, 1997). Cyanobacteria and red algae also possess novel ferredoxin-dependent bilin reductases for the synthesis of the linear tetrapyrrole precursors of their

$^{1}$ To whom correspondence should be addressed. E-mail jclagarias@ucdavis.edu; fax 530-752-3085.

phycobiliprotein light-harvesting antennae complexes (Beale and Cornejo, 1991a, 1991b, 1991c; Cornejo et al., 1998). Primarily on the basis of studies with the red alga *Cyanidium caldarium*, these investigators proposed that the biosynthesis of the two major phycobiliprotein chromophore precursors, phycoerythrobilin (PEB) and phycocyanobilin (PCB), requires two ferredoxin-dependent bilin reductases and several double-bond isomerases. The first bilin reductase catalyzes the two-electron reduction of BV at the C15 methine bridge to produce the BR isomer 15,16-dihydrobiliverdin (DHBV), whereas the second bilin reductase catalyzes the conversion of 15,16-DHBV to 3Z-PEB, a formal two-electron reduction of the C2 and C3¹ diene system. In *C. caldarium*, an additional enzyme mediates the isomerization of 3Z-PEB to 3Z-PCB, both of which appear to be isomerized to their corresponding 3E isomers before assembly with the nascent phycobiliprotein apoproteins (Beale and Cornejo, 1991a, 1991b, 1991c). More recent studies lend support for a similar pathway of PCB and PEB synthesis in cyanobacteria (Cornejo and Beale, 1997). In contrast with mammals and phycobiliprotein-containing organisms, plants and green algae reduce BV to 3Z-PΦB by the ferredox-in-dependent enzyme PΦB synthase, which targets the 2,3,3¹,3²-diene system for reduction (Terry et al., 1995; Wu et al., 1997). In plants, 3Z-PΦB is isomerized to its 3E isomer, which appears to be the immediate precursor of the phytochrome chromophore (Terry et al., 1995). The green alga *Mesotae- nium caldariorum* possesses a second bilin reductase activity that catalyzes the reduction of the 18-vinyl group of PΦB to produce 3Z-PCB (Wu et al., 1997). These investigations also revealed that 3E-PCB is the natural phytochrome chromophore precursor in this organism.

Despite the extensive biochemical analysis of the phytobilin biosynthetic pathways in plants, algae, and cyanobacteria, the low levels of bilin reductase expression have hindered efforts to clone these enzymes. Using a genetic approach, Kohchi and colleagues (2001) recently cloned the HY2 locus of Arabidopsis, which encodes the enzyme PΦB synthase. The studies reported here were undertaken to identify HY2-related genes in the protein and nucleic acid databases. Using cloning, expression, and biochemical characterization, our investigations revealed three new classes of ferredoxin-dependent bilin reductases with either unique substrate or product specificities.

formed to identify putative bilin reductases in the nonredundant National Center for Biotechnology Information database, in CyanoBase (Nakamura et al., 2000), and in the Joint Genome Institute Microbial Genome database (http://spider.jgi-psf.org/JGI_microbial/html). These searches identified 15 putative proteins from various photosynthetic bacteria and two known proteins from plants. Figure 2 shows a multiple sequence alignment of this family of proteins using CLUSTAL W (Higgins et al., 1996), hand adjustment with MEME (Bailey and Elkan, 1995), and highlighting with GENEDOC (http://www.psc.edu/biomed/genedoc). This alignment revealed regions of strong similarity interspersed with highly diverged regions, with an average pairwise similarity score of 25%. No sequence similarity of these proteins was observed with mammalian biliverdin reductases.

On the basis of the biochemical data presented here, we name these HY2-related cyanobacterial loci after their roles in the biosynthesis of PCB (i.e., *pcyA*) and PEB (i.e., *pebA* and *pebB*). One of these proteins, the product of locus *slr0116* (i.e., *pcyA*) in the genome of the cyanobacterium *Synechocystis* sp. PCC6803, appears to be part of an operon with a putative response regulator located 62 bp upstream (Ashby and Mullineaux, 1999). Interestingly, this response regulator belongs to the OmpR subfamily for which a mutation (*ycf27*) was shown to cause a reduced energy transfer from the phycobilisomes to photosystem I (Ashby and Mullineaux, 1999). *pcyA*-related open reading frames (orfs) also were found in the oxyphotobacterium *Prochlorococcus* sp. MED4 (CCMP1378), which is also known as *Prochlorococcus marinus* MED4, in the marine cyanobacterium *Synechococcus* sp. WH8102, and in the nitrogen-fixing, heterocyst-forming filamentous cyanobacteria *Anabaena* sp. PCC7120 and *Nostoc punctiforme*. Among the other identified HY2-related genes are two orfs, *orf236* and *orf257*, from the marine cyanobacterium *Synechococcus* sp. WH8020 that lie adjacent to each other within the major phycobiliprotein gene cluster (Wilbanks and Glazer, 1993a, 1993b). These orfs, which encode the proteins Ycp2_SYNPY and Ycp3_SYNPY, appear to be part of a three-gene operon containing an upstream orf of unknown function, *orf200*. A similar operon was identified in *Synechococcus* sp. WH8102. The genomes of *N. punctiforme* and *Prochlorococcus*, both the MED4 and SS120 (CCMP1375) subspecies, also contain similar operons. In contrast to the *N. punctiforme* and *Anabaena* operons, an upstream orf in the *Prochlorococcus* operons exhibits a striking similarity to the ferredoxin-dependent heme oxygenase gene *HY1* (Davis et al., 1999; Muramoto et al., 1999) and its homologs in the cyanobacterium *Synechocystis* sp. PCC6803 (Cornejo et al., 1998). On the basis of their roles in PEB biosynthesis shown in this study, we name these orfs *pebA* and *pebB*.

PSI-BLAST iterations also identified a weak relatedness of HY2 to the red chlorophyll catabolite reductase (RCCR) from barley and Arabidopsis. RCCR is involved in chlorophyll catabolism and catalyzes the ferredoxin-dependent reduction of the linear tetrapyrrole, red chlorophyll catabolite

RESULTS

The HY2-Related Gene Family in Cyanobacteria, Oxyphotobacteria, and Plants

A recently published paper describes the cloning of the HY2 gene of Arabidopsis (Kohchi et al., 2001). Using the deduced protein sequence of HY2, TBLASTN, BLASTP, and PSI-BLAST searches (Altschul et al., 1990, 1997) were performed.
(RCC), to yield the primary fluorescent chlorophyll catabolite (Wüthrich et al., 2000). These investigators showed that RCCR was incapable of reducing BV to either bilirubin or PΦB (Wüthrich et al., 2000). Interestingly, the sequence similarity between RCCR and the other HY2-related proteins is so weak that TBLASTN searches using the two RCCR sequences failed to identify HY2 or other HY2-related proteins present in the publicly available databases (Wüthrich et al., 2000). This divergence undoubtedly reflects the unusual substrate specificity of the RCCR for bilins derived from chlorophyll catabolism.

Phylogenetic analysis of the HY2-related family of proteins was performed using a heuristic parsimony search with a modified PAM250 weighting matrix and the program PAUP* version 4.0 (see Methods). A single tree obtained with this analysis (Figure 3) revealed four clades of HY2-

related proteins with strong bootstrap support: PcyA, PebA, PebB, and RCCR. We noted that HY2 lies within in the PebB clade.

### Recombinant HY2-Related Proteins Are Bilin Reductases

The HY2-related cyanobacterial orfs were amplified by polymerase chain reaction and cloned into the Escherichia coli expression vector pGEX-6P-1, which is similar to the vector described for mHY2 (Kohchi et al., 2001). With this vector, the proteins were expressed as glutathione S-transferase (GST) fusions, which enabled their purification by affinity chromatography. The GST tag was removed via site-specific protease digestion, which resulted in an additional five

![Chemical Structures](chemical_structures.png)

**Figure 1. Pathway of the Biosynthesis of Bilin Pigments.**

The mammalian bile pigment bilirubin and the linear tetrapyrrole precursors of the phytochrome and phycobiliprotein chromophores of plants, algae, and cyanobacteria share the common intermediate biliverdin IXα. HY2, phytochromobilin synthase or 3Z-phytochromobilin:ferredoxin oxidoreductase; PcyA, 3Z-phycocyanobilin:ferredoxin oxidoreductase; PebA, 15,16-dihydroliverdin:ferredoxin oxidoreductase; PebB, 3Z-phycoerythrobilin:ferredoxin oxidoreductase; BVR/BvdR, biliverdin IXα:NAD(P)H oxidoreductase. The dashed arrow with a question mark indicates a second type of putative 3Z-phycocyanobilin:ferredoxin oxidoreductase. The dashed arrow indicates a putative phycoerythrobilin–phycocyanobilin isomerase (Beale and Cornejo, 1991b).
PcyA_ANASP : ---MSLTSIPSLREQQHPDLRQLADCIIEEVWHQHLDLSFYHDPDEL : 43
PcyA_NOSPU : ---MSFTSMPSLREQQHPDLIRQLADCHEAAWHQHLDLSPYHLPEDEL : 43
PcyA_SYNY3 : ---MAVTDLSLTNSSLMPTLNPMIQQLALATAASWQS-LPLKPYLPEDL : 46
PcyA_SYN81 : ---MQSPPSSESSSTVAPLIPSLAETIRGAWIGLPBELKPPLDADSDF : 42
PcyA_PROME : ---LNLLSKSLTKTKLIDPLILTLLOQNIVKQRSKLNDLNCIEVDPKL : 44
Peba_SYNPY : ---MFDSFLNECHSDITKRGGSBPLPLEGL : 27
Peba_SYN81 : ---MFDPFLEEDQTGIQA-RGGISVEVPAGL : 27
Peba_PROMA : ---MNKLMLQDLHNNEKKRIIS-HGGKPIEVENGM : 31
Peba_PROME : ---MFESLKNFKTNLE--DLDGKELEISK : 25
Peba_NOSPU : MLNSQSPLRNVALFLINETCMIAITYFHARVNKSCSMYKPFLEFEKELEFQRFDLQSRVIPPGLE : 64
PebB_SYNPY : ---MTNQRFKSTDPVNEGWSWQPFLDEAIKRLEG-LNEPYPVPDRF : 44
PebB_SYN81 : ---MSIDLRASSLDPVQIPGWRWQPFLEDEASAALKP-FNPSYPPIAETF : 45
PebB_PROMA : ---MIIKRDNSLSKIDLRDWWTTPFFNDVDKLCSV-FEIEPYPVSHDF : 44
PebB_PROME : ---MLIQNNTIFYSQERWAKFIKFLISQLDN-YHCVEHKIASDF : 40
PebB_NOSPU : ---MNSERSDVTLYQPFLDYAIAYMRSRLDEBYPIPTGF : 37
HY2_ARATH : MALSMEEFGSIGSCFKAPN-PPVLISASPNIKINFTRRRKKRFLRRVSASYKEFAESAEEETRKR-IVLEPSHLQEKY : 77
RCCR_ARATH : MAMIFCNTLYSSSSPSYLSPITSKPSRFSKNLRPRAQFQSMEDHDDHLRRKFMEFPYVSPTRKQLMVVDLMSTVENRLQS--QLLPCNLPPDV : 90
RCCR_HORVU : --- : -


PcyA_ANASP : GYVEGRLEGEK----LTIDENRCYQTQPFRRMHLEELAKVGNMEDILHCVMFPRPEYDLPMFGCDLVGGR-GQISAATADLSPIVHL---DRTL : 126
PcyA_NOSPU : GYVEGRLEGEK----LTIDENRCYQTQPFRRMHLELANIGNMEDILHCVMFPRPQYNLPMFGCDLVGGR-GQISAATADLSPIQL---DRTL : 126
PcyA_SYNY3 : GYVEGRLEGEK----LVIDENRCYQTQPFRRMHLEAKVGKGEDIILHCVMFPEPLYGLPLFGCDIVAGP-GGVSAAITADLSPITQS---DRLQ : 129
PcyA_SYN81 : SSIEGQLEGDD----LLIRNELLCCRGVRKIHELARLGRLGQILHCWFEPDRFDLPIFGADIVAGP-AGVSAAILVDLSPIV-------SGTL : 123
PcyA_PROME : SNIISNEEGKE----LYIDENEFYKAKGFRKLHIEVAEFSKSCLKILHCVFFPDPKYDIPIFGMDLVKVN-ELVSAAILVDLSPISSK---NQNL : 127
Peba_SYNPY : EECRSSKSS------SVIQSWLWDVPGFRWRVTRLDAQDSLQVFNSVAYPDYNYDHPLMGVDLLWFGARQKLVAVLDFQPLVQ---DKDY : 109
Peba_SYN81 : EHNQSQKGS------STIQSWLWQVPGFRWRVTRLDAQDSLQVLNSVAYPFDLDHPLMGVDLLWFGARQKLVAVLDFQPLVQ---DKDY : 109
Peba_PROMA : SERFSHKQD------TVIKSWLWDVPGFRWRVTRMDAGDKLQVLNSVAYPAYTNDKPILGIDILWFGLKRLVAVLDFQPLVQ---BERY : 113
Peba_PROME : EFKEHHNKD------SKYIKNWIFESQQYRKWRITKLDGGDKLQVFNTVAYPNFKSEFPLLGADILWFGTSQKLLAIDFYQPLIQ---BKKY : 109
Peba_NOSPU : EFKVSDRGR------NPATIRSWCYQSQELRKIRTYIDAGESAQIFNSWWYPSHNYDLPLGIDFLSFG-KVKNLIVLDFQPLFQ---DEDY : 147
PebB_SYNPY : LQREDTQGSKSK-SIPVTTATWACKTEKFRVRAACVSAGSAAASVLNFVINPKSTYGLPFGCDLVTFP--AGHLLALLDQPAIK--TEDVH : 131
PebB_SYN81 : LQKEGSTGSKAK-PVPVTATTAWACSTDKLQVRCACVEAMAASVLNFVINPSCRFDLPFEGADLVTLT--NGHLLALLDQPDK--ADPDH : 132
PebB_PROMA : LSKESETGSRRN-PVHVTTLTWAANKFEKIQVRlacIKGESWSVFNLLIHPLNDYDLPFQADEVTLP--NGHLLALLDQPALK--LDNIH : 131
PebB_PROME : SYKESSYGSKKS-KKNINLFTWGATHQRINPARAVCINSPNYSVLNFLIIPKTSYNIPLFGVDFVSLP--TSHLLVLDFOPSLK--VENQF : 127
PebB_NOSPU : ESNSAVVGKGKN-QEEVTTTSYAFQTAKLROIRAHHVQGENSLQVLNFVIFPHLNYDLPEFGADLVTLT--GGHLLALLDMQPLFR--DDSAY : 124
HY2_ARATH : SSMTPLDG------KTELQMLAFKSSKIRLLRSAIEN-ETMQVFDFAFMEPEYDTPTFCANFFTST--NVNIVVLDLNPLHQLTDQTDY : 159
RCCR_ARATH : RNFNNPNGSAEAASLHIRSGDKSSPIDFVIGSWIHCKIPTGVSENITSISGFLNSSTKAPNFVVELLIQSS-SKSLVLILDLPHRKDLVLNPDY : 181
RCCR_HORVU : ----MDFMLQSSLHCKVPN-G-AIDIISLFINLNASTDAPHFIMEFFIQGS-PTSMVLLDLLPRKDALAHPEY : 66


PcyA_ANASP : PESYNSAATSINT---LNFSQPRELPSWGNI---FSDCITVVRP---------SSPEEEAAMFLGRVREFLQVHCQGAIAA--SPVSAEQK : 199
PcyA_NOSPU : PESYTTALAQOLPV---LNFSQPRELPEWGNI---FSDCITVVRP---------GSPEEEAAMFLSRVREFLDIHCMQAIAAS--HPVSVEQV : 199
PcyA_SYNY3 : PAAYQKSLAELGQ---PEFEQQRLELPPWGEI---FSEYCLFIRP---------SNVTEEERFVQRVVDFLQIHCHQSIVA--EPLSEAQT : 202
PcyA_SYN81 : PSGIETALAGTPS---PAFRQVRDLPGWGTI---FSPHVCFIRP---------DGAEEEVLFRSRVEEVLTILRTAVLQTACEPATAAST : 198
PcyA_PROME : --KYDHLSSHIDK---SVFKSKREIPIWGNI---FSKNVFEASL---------KNESEKNAFCKIVDNFLSVLIQLSQST--SPDSDYEI : 198
Peba_SYNPY : LDRYFSGIKELNQRFPDLNGEETMRSFDPNQY---FSSWLLECRG---------GAEQADLSLPKAFSAFLKAYWDLHDNA--KSTPSTIP : 186
Peba_SYN81 : LDRHFDGILKDLNARFPDLNGEETMRSFDPNQY---FSSWLLECRG---------GSEEAADRSLPKAFSAFLKAYWGLHDEA--SKEPSSIS : 186
Peba_PROMA : FCRYYKDQIQILKNRFVDFNSQTKMIYDSNKY---FSPWVLLYNG---------SFDDLQCSLAKILDEFILHAYWQVDNNN--SREYIKII : 190
Peba_PROME : LQKYCSSSLDFIKNQYSVFDNHKMKNIYDSKKY---FSPWMICRG---------NKLNLDRDLNNIFCSFVSNYLTINKLH--QNNQFLD : 185
Peba_NOSPU : QNKYIAPLKYLHNKYPDLAQNLEMKFYDANQY---FSKYLFAKT---------DAETVSTRVFEAQDYLNLYWQLADA--QALHDPED : 224
PebB_SYNPY : TTHVWDRLLIPFERWRDQLPYGGPIPEEAQPF---FSPGFLWTRPLPGE----EGDELIQSIVRPAFNDYLDLYLELAASA--ERVTDERS : 213
PebB_SYN81 : TQPVWERIMPPLFERWQAELPDG GPIPEEAQPY---FSPAFLWTRIPLGE----EGDELIERVERPAFIDYLQLYLNLVAEA--EPVSDDRA : 214
PebB_PROMA : TENVWPRLLIPLHDHWQSLLLPSGGEIPKEAEPY---FSPGFLWSRLPLSK---ESDNIISEILRPAFGEYLSLYIELLHIA--KPLKKERA : 213
PebB_PROME : NSELLEQTIKILKKSCHSLSLPAEKMSEDVAKF---FSPGLIWSDLAKHQ---DSDNLIENQLYDSFKEYLNLYLKTLFES--EEVGHGLQ : 209
PebB_NOSPU : QAQYTEPILLPJFHAHQQHLSWGGDFPEBAQPF---FSPAFLWTRP--------QETAVVETQVFAAFKDYLKAYLDFVEQA--EAVTDSQN : 202
HY2_ARATH : QDKYYYNKIMSYYHKYAETFPWGGKLTGESIKF---FSPLMWTRF---------SSSKEKHKALFSAFLEYQAWLEMTIQV--REEMEPSH : 237
RCCR_ARATH : LKEYYQDTALDSHRQSLLKLPEVNPYVPSLFRVSASFPTASMLKIDAEE--EDKLEEILRDRHVSAAKEVLEWLERCVKKEEEKIVVGEEB : 271
RCCR_HORVU : IEKYYEDTEVDKQRKIIEQLPQARPYLPSLFRVSASFPTAVEFTIDCGKGGE GTLEIVHGHLASVVKGILOILDTCASD---ASEMEEG : 155


PcyA_ANASP : QQIAGQHNVCSDKQQQNDKTRRVLEKAFGVDWABENYMTTVLFDLPE : 245
PcyA_NOSPU : TQNLAGQHNYCTKQQQNDKTRRVLEKAFGPVWAENYMTTVLFDLPT : 245
PcyA_SYNY3 : LEHRQGQIHYCQQQQKNDKTRRVLEKAFGEAWAERYMSQVLFDVIQ : 248
PcyA_SYN81 : IRRYEGQLSYCLQQKRNDKTRRVLEKAFDASWADRYIEELLDDPLPG : 247
PcyA_PROME : IEERINYKNYCVQQMKNEKTSLLLLKYFDKVWDEYIKKVLEDF : 243
Peba_SYNPY : PEEWKNLQDKYOIYSAERDPAHGLFTSHFGKESURLELFPASSSHK : 236
Peba_SYN81 : PGDVERLONAYDVYSAERDPAHGLFTSHFGKESURLELFPASQPA : 235
Peba_PROMA : PSKWEQLHINYDIYSAERDPAHGLFKSYFGQTWADQFVREFLPHSHLTAD : 241
Peba_PROME : LEQIKNNQIDYDKYSAEKDPADKLFKTFGETWTEFINNFLTLNNHNPBK : 236
Peba_NOSPU : IQRIVKAQKDYDQYSADRDPASGLFSSYFGHEARLHELEDAVPVASASKR : 280
PebB_SYNPY : EVLLQ-GQRKYTDRAEKDPARGMLTRFHGSEWTEAYIHTVLFDL : 257
PebB_SYN81 : ELLLS-GOKRYTAYRAEKDPARGMLTRFGSEWTEYIHGVLFDELEAA : 262
PebB_PROMA : LKILE-GOKAYINYRSTKDPARAMLCRFYGKEWTEDYIHKVLFNI : 257
PebB_PROME : QELIN-GQNDYLNRRDNDPARMLSLSLFGKDFTESLINKVLFSTNKVL : 257
PebB_NOSPU : LVAIKQAOQLRYLRYAEKDPARGMKFRFGAEWTEEYIHGFLEDLERKLT VVK : 255
HY2_ARATH : VRANCEACHKYLTWRAQKDPGHGILLKRLVGGEAKAEELLRDFLENGVD ELGTKTFIDYFPEYQTEDGTVSDKRSIIGKSYETRPWDLTGQFIG : 329
RCCR_ARATH : ERMELERRDKSFRRKSIEDDLDLQFPRMFGEEVSSRVWHAIKEAFGVL : 319
RCCR_HORVU : ERETMVKDRRTVRSKSIEDLTANLPRMFGPDVSGRIIAEIRKAFGVQEG : 205

Figure 2. Multiple Sequence Alignment of the Identified Bilin Reductases.

All identified sequences were aligned using the programs CLUSTAL W and MEME. Conserved residues in 90 or 70% of the aligned sequences are depicted in the consensus sequence with uppercase or lowercase letters, respectively. Sequence similarity groups, labeled 1 (D = E), 2 (R = K), 3 (F = Y = W), and 4 (L = I = V = M), shown in the consensus sequence reflect conservation in >90% of the sequences. Dark shading with white letters, gray shading with white letters, and gray shading with black letters reflect 90, 70, and 50% sequence conservation, respectively. SYNY3, *Synechocystis* sp PCC6803; SYNPY, *Synechococcus* sp WH8020; SYN81, *Synechococcus* sp WH8102; PROMA, *Prochlorococcus* sp SS120; PROME, *Prochlorococcus* sp MED4; NOSPU, *Nostoc punctiforme*; ANASP, *Anabaena* sp PCC7120; ARATH, *Arabidopsis thaliana*; and HORVU, *Hordeum vulgare*. Database accession numbers are GB: AF339056 for PcyA_ANASP (*CyanoBase* contig 362), GB: AF339057 for PcyA_NOSPU (JGI contig 632), PIR: S76709 for PcyA_SYNY3, PcyA_SYN81 is on JGI contig 51, GB: AF352050 for PcyA_PROME (JGI contig 26), SW: Q02189 for PebA_SYNPY, PIR: S31075 (fragment)/ JGI contig 72 for PebA_SYN81, EMB: CAB95700.1 for PebA_PROMA, PebA_PROME is on JGI contig 26, GB: AF352049 for PebA_NOSPU (JGI contig 622), SW: Q02190 for PebB_SYNPY, PebB_SYN81 is on JGI contig 72, EMB: CAB95701.1 for PebB_PROMA, PebB_PROME is on JGI contig 26, GB: AF339058 for PebB_NOSPU (JGI contig 622), DDBJ: AB045112 for HY2_ARATH, EMB: CAB77705.1 for RCCR_HORVU, EMB: CAB16763.1 for RCCR_ARATH. Asterisks indicate every tenth amino acid; dashes indicate gaps; numbers above the line indicate amino acid sequence numbering starting with number one.

to eight N-terminal amino acids due to the cloning strategy. Figure 4 shows SDS-PAGE results of purified recombinant protein representatives of the PcyA, PebA, PebB, and HY2 subfamilies. One liter of bacterial culture yielded between 1 and 10 mg of soluble recombinant protein depending on which protein was expressed. The deduced molecular masses of the recombinant processed proteins, confirmed by SDS-PAGE (Figure 4), are as follows: Anabaena PcyA, 28.7 kD; Synechocystis PcyA, 28.9 kD; Synechococcus sp WH8020 PebA, 28 kD; Synechococcus sp WH8020 PebB, 30.3 kD; and Arabidopsis mHY2, 33.4 kD.

To determine whether the recombinant HY2-related proteins possess bilin reductase activity, we used a coupled holophytochrome assembly assay to analyze crude protein extracts from *E. coli* expressing these proteins for their ability to convert BV to phytoobilins under standard PB synthase assay conditions (M.T. McDowell and J.C. Lagarias, manuscript submitted for publication). Figure 5A shows that crude bacterial lysates containing GST fusions of mHY2 and PcyA_SYNY3 (PcyA_ANASP; not shown) all exhibited BV reductase activities, yielding phytoobilin products that could combine with the cyanobacterial phytochrome Cph1 apoprotein (apoCph1) to yield phytochrome difference spectra. The bilin metabolites incubated with apoCph1 resulted in different maxima and minima, suggesting that the various proteins reduced BV to distinct products. Both PcyA-containing extracts produced a BV metabolite(s) that gave spectra identical to those of the PCB adduct of apoCph1, with difference maxima at 655 nm and minima at 705 nm (Yeh et al., 1997). Figure 5A shows that both difference peaks of the mHY2 metabolites were markedly red shifted, with maxima at 670 and 730 nm, which is characteristic of the PB adduct of apoCph1 (Yeh et al., 1997; Kohchi et al., 2001). Identical results were obtained using the purified recombinant HY2 and PcyA proteins (data not shown). Similarly, *E. coli* extracts lacking HY2 or PcyA proteins failed to metabolize BV to bilin products that could functionally assemble with apoCph1 (data not shown).

In contrast to the results for PcyA and HY2, no phytochrome difference spectrum was observed when the BV metabolites from reactions containing PebA_SYNPY, PebB_SYNPY, or a 1:1 mixture of the two Synechococcus-derived proteins were incubated with apoCph1. To determine whether fusion to GST is responsible for inhibiting the enzyme activity of these proteins, GST was removed by protease digestion and the full-length proteins were purified (Figure 4). Neither the purified proteins nor the 1:1 mixture of PebA and PebB were able to convert BV to a bilin product(s) that yielded a photoactive adduct with apoCph1 (Figure 5A). The observation that coinubcation of a 1:1 ratio of PebA and PebB with BV elicited a color change of the assay mixture from bluish-green to pink suggested that these proteins converted BV to bilins unable to form a photoconvertible holophytochrome. It is noteworthy that this pronounced color change was not observed when either PebA or PebB was assayed separately. This strongly implied that the PebA/PebB mixture could convert BV to PEB, the precursor of the phycobiliprotein C-phycoerythrin. To test this hypothesis, BV-derived bilin metabolites from PebA, PebB, and PebA/PebB were incubated with apoCph1, and the mixtures were analyzed spectrofluorometrically for the production of the fluorescent PEB-apoCph1 “phytofluor” adducts (Murphy and Lagarias, 1997). Only the PebA/PebB product mixture yielded a highly fluorescent compound, whose excitation and emission spectra were consistent with the formation of a phytofluor (Figure 5B). This result suggested that PebA and PebB were both required for the conversion of BV to PEB.

HPLC Reveals Distinct Substrate/Product Specificity for Each Member of the HY2 Family

HPLC analysis was performed to identify the bilin metabolites of the HY2 family members using a chromatographic system that is able to separate 3E and 3Z isomers of PB.

observed (data not shown). HPLC coelution experiments showed the purple and pink pigments to be the 3E and 3Z isomers of PEB, respectively (data not shown). Both pigments are chemically stable in the HPLC mobile phase, eluting as single peaks after purification and re-injection. Moreover, both HPLC-purified pigments form phytofluors upon incubation with apoCph1, indicating that these are configurational isomers of PEB. HPLC-purified 15,16-DHBV from the PebA-mediated reduction of BV, however, was unable to form a fluorescent adduct with apoCph1 (data not shown).

Biochemical studies of ferredoxin-dependent bilin reductases from algae and plants indicated that the 3Z isomers of PEB, PCB, and PΦB were the primary metabolites of these enzymes, with the formation of the 3E isomer requiring distinct bilin isomerase(s) (Beale and Cornejo, 1991a, 1991b, 1991c; Cornejo and Beale, 1997; M.T. McDowell and J.C. Lagarias, manuscript submitted). Our results show that both bilin isomers are produced with recombinant HY2, PcyA, and PebA/PebB proteins. We believe that the production of the 3E isomers occurred because of the presence of glutathione in the assay mixture and because of heating in the Speed-Vac concentrator. In this regard, glutathione-mediated 3Z to 3E isomerization of phycobilins has been reported for bilin reductases from *C. caldarium* (Beale and Cornejo, 1991c). Preliminary experiments performed with GST fusion proteins that did not come in contact with reduced glutathione or with proteins that were eluted from the affinity column by protease digestion greatly increased the relative amount of 3Z isomers produced (data not shown). Heating that occurred during concentration also contributed to the formation of the 3E isomers. If the drying time was reduced, only 3Z isomers were detected (data not shown). Therefore, we conclude that in all cases the 3Z isomers are the primary reaction product of these reductases and that the production of the 3E isomers occurs by non-enzyme-mediated side reactions caused by heat and reduced glutathione.

The HY2 Family of Bilin Reductases Are Ferredoxin Dependent

All of the reductive interconversions of BV and 15,16-DHBV presented here were dependent on reduced ferredoxin, which necessitated the inclusion of ferredoxin:NADP⁺ oxidoreductase and an NADPH-regenerating system in the assay mixture. Indeed, none of the reduced bilin metabolites were detectable via HPLC when either ferredoxin or the NADPH-regenerating system was omitted from the assay mixture (data not shown). These results are in agreement with the ferredoxin dependence of the bilin reductases from plants and algae (Beale, 1993; M.T. McDowell and J.C. Lagarias, manuscript submitted). Thus, this family of proteins constitutes a new class of bilin:ferredoxin oxidoreductases (EC 1.3.7.n).

HY2-Related Bilin Reductases 971

DISCUSSION

Using a combination of protein-based pattern searches of genomic databases, phylogenetic analysis, and biochemical characterization, these investigations establish that the HY2 family of ferredoxin-dependent bilin reductases can be subdivided into five classes: PcyA, PebA, PebB, HY2, and RCCR families (Figure 3). This classification system is supported by the distinct substrate preference and double-bond regiospecificity of representative members of each bilin reductase subfamily. PcyA, PebA, and HY2 all recognize BV as a substrate, yet each yields different bilin products. PebB and RCCR possess unique bilin substrates (i.e., 15,16-DHBV and RCC, respectively), and neither metabolizes BV (Wüthrich et al., 2000; this study). Biochemical analyses of representatives of the three new classes of bilin reductases identified here, PcyA, PebA, and PebB, document their involvement in the biosynthesis of the phycobiliprotein chromophore precursors PCB and PEB.

The PcyA Family of Ferredoxin-Dependent BV Reductases Plays a Key Role in PCB Biosynthesis

In this investigation, we have documented that the *pcyA* genes of the cyanobacteria *Synechocystis* sp. PCC6803, *Anabaena* sp. PCC7120, and *N. punctiforme* (data not shown) encode bilin reductases that catalyze the four-electron reduction of BV to 3Z-PCB. PCB is the precursor of the chromophores of the phycobiliproteins phycocyanin and allophycocyanin, which are abundant in all three cyanobacteria. PcyA enzymes are atypical bilin reductases because all others catalyze two-electron reductions. Formally, these enzymes catalyze two-electron reductions of both the A and D rings of BV; however, we have not detected the production of semireduced intermediates such as PΦB and 18¹,18²-DHBV. Thus, it appears that the partially reduced intermediates are tightly bound to the enzyme. The direct conversion of BV to PCB in these cyanobacteria is in contrast to the proposed pathways of PCB biosynthesis in the red alga *C. caldarium*, which involves the intermediary of PEB (Beale, 1993), and in the green alga *M. caldariorum*, in which 3Z-PΦB is an isolable intermediate (Wu et al., 1997). *pcyA*-related genes also are present in the oxyphotobacterium *Prochlorococcus* sp. MED4, an unanticipated observation in view of the lack of phycobiliproteins in this organism. Phylogenetic analyses place this oxyphotobacterial protein in the PcyA clade of PCB:ferredoxin oxidoreductases. During the review process of this manuscript, we were able to clone the *Prochlorococcus* sp. MED4 *pcyA* gene and express it as an N-terminal GST fusion. We determined that recombinant PcyA_PROME was able to reduce BV to PCB in our standard phytochrome-based assay (data not shown). It therefore possesses the same enzymatic activity as all other studied PcyA enzymes.

peb Operons Encode Bilin Reductases Involved in  
PEB Biosynthesis  

We have observed that the *pebA* and *pebB* genes of the cyanobacteria *Synechococcus* sp WH8020 and *N. punctiforme* encode bilin reductases that catalyze the conversions of BV to 15,16-DHBV and 15,16-DHBV to 3Z-PEB, respectively (Figure 1). PebA therefore is a 15,16-DHBV:ferredoxin oxidoreductase, whereas PebB is a 3Z-PEB:ferredoxin oxidoreductase. Both activities are consistent with the pathway of PEB biosynthesis in the red alga *C. caldarium* (Beale, 1993). The two *peb* genes also are found in the same operon in both phycoerythrin-producing cyanobacteria, and their close association with the major phycobiliprotein gene clusters supports their role in phycobilin biosynthesis (Wilbanks and Glazer, 1993b). We hypothesize that PebA and PebB function as a dual enzyme complex, in view of the synergistic metabolism of BV observed when the two enzymes are coincubated. A *peb* operon is not present in the genome of the cyanobacterium *Synechocystis* sp PCC6803, an organism that lacks phycoerythrin. This strongly suggests that PCB is synthesized in this cyanobacterium via the PcyA-dependent pathway, as opposed to the PEB pathway found in *C. caldarium* (Beale, 1993). In this regard, biochemical analyses of crude extracts from *Synechocystis* sp PCC6803 provide no evidence for the production of PEB (Cornejo and Beale, 1997).

The MED4 and SS120 subspecies of the oxyphotobacteria *Prochlorococcus* also possess *peb* operons very similar to those of *Synechococcus* sp WH8020 and WH8102, except that the former possess upstream genes related to heme oxygenase. This strongly suggests that both oxyphotobacterial subspecies can synthesize PEB. In this regard, genes encoding the α and β subunits of a novel phycoerythrin have been identified in the SS120 subspecies of *Prochlorococcus* (Hess et al., 1996, 1999). It also has been shown that this unusual phycoerythrin plays a role in light harvesting in this ecotype (Lokstein et al., 1999), which is adapted for photoautotrophic growth at great ocean depths where light is limited. This observation is consistent with the lack of

Sep-Pak C<sub>18</sub> reversed-phase column and analyzed by HPLC on a Phenomenex Ultracarb 5-μm ODS20 4.6 mm × 250 mm column with a 4.6 mm × 30 mm guard column. The HPLC solvent was acetonitrile:20 mM formic acid (50:50, v/v), and the effluent was monitored at 560 nm for the first 11.5 min and at 380 nm for the remaining time. STDS, mixture of different bilin standards; HY2, metabolites obtained by mHY2; PcyA, metabolites obtained by PcyA_SYNY3; PebA, metabolites obtained by PebA_SYNPY; PebB, metabolites obtained by PebB_SYNPY; PebA + PebB, metabolites obtained by a 1:1 mixture of PebA_SYNPY and PebB_SYNPY. Symbols are used for better visualization of peaks. Single symbols indicate the 3E-isomer (except 15,16-DHBV and BV) and double symbols indicate the 3Z-isomer, respectively.

**Figure 6.** HPLC Analysis of the BV Metabolites of PebA, PebB, PcyA, and HY2 Bilin Reductases.

Forty micrograms of purified protein was incubated at 28°C under green safelight in a total assay volume of 5 mL. The assay system contained an NADPH-regenerating system, spinach ferredoxin-NADP<sup>+</sup> reductase, spinach ferredoxin, and BSA. The reaction was started by adding 5 μM BV and was stopped by placing the mixture on ice. Bilins were extracted from the incubation mixture using a
phycoerythrin genes in the high light–adapted MED4 eco-
type. Although the enzymatic activities of *Prochlorococcus*
PebA and PebB have not been determined experimentally,
our phylogenetic reconstructions suggest that these pro-
teins may be functional orthologs of the *Synechococcus*
and *Nostoc* enzymes. Further analysis of the bilin biosyn-
thetic pathways in *Prochlorococcus* and marine cyanobac-
teria such as *Synechococcus* sp WH8020 will be interesting,
because the shorter wavelength–absorbing phycourobilin
chromophores are major constituents of their phycoeryth-
rins (Ong and Glazer, 1991; Hess et al., 1996). Although we
have identified PCB and PEB biosynthetic enzymes in both
organisms, it remains to be determined whether either these
or other enzymes play a role in phycourobilin biosynthesis.

### Phycobilin Isomerases: Are They Necessary?

PcyA, HY2, and PebB mediate bilin reductions that yield the
3Z isomer of their respective products. Because numerous
studies have established that the more thermodynamically
stable 3E isomers are substrates for assembly of the phyco-
biliprotein and phytochrome holoproteins, it has been pro-
posed that there are unique 3Z/3E isomerases that mediate
this interconversion (Beale, 1993; Terry et al., 1993). It
should be noted that the 3Z isomer of PΦB has been shown
to be a substrate for apophytochrome (Terry et al., 1995);
however, these investigators suggested that isomerization
to the 3E isomer is necessary to yield the correct stereo-
chemistry of the holophytochrome chromophore. Such an
isomerase activity has been identified in extracts of the red
alga *C. caldarium*; however, this reaction also can be medi-
ated by reduced glutathione (Beale and Cornejo, 1991b).
For this reason, the need for a 3Z/3E isomerase has been
questioned. All of the *hy* mutant loci have now been cloned
from Arabidopsis, and none of these genes appear to en-
code a bilin isomerase. Thus, the isomerization of 3Z-PΦB
may occur chemically or may be catalyzed by a genetically
redundant family of bilin isomerases.
X-ray crystallographic analyses of phycobiliproteins have
revealed that the stereochemistries of the thioether linkages
to distinct cysteine residues are not all the same (Schirmer
et al., 1987; Schmidt et al., 1987). Therefore, we propose
that the different stereochemistries arise from the use of the
3Z and 3E isomers of the phycobilin precursor as substrates
for assembly to distinct cysteinyl moieties. Beale and
Cornejo (1991b) have identified a bilin isomerase that cata-
lyzes the conversion of 3Z-PEB to 3Z-PCB in *C. caldarium*,
which appears to be the sole pathway for PCB biosynthesis
in this organism. More recently, a lyase/isomerase from the
cyanobacterium *Mastigocladus laminosus* was described
that is involved in both the isomerization of PCB to phycovi-
olobilin and its covalent attachment to apo-α-phycoerythro-
cyanin (Zhao et al., 2000). On the basis of these results and
the diversity of bilin isomers found in phycobiliproteins from
marine cyanobacteria, cryptomonads, and oxyphotobacte-

ria (Ong and Glazer, 1991; Hess et al., 1996; Wedemayer et al.,
1996), it is likely that numerous bilin isomerases are present
in these oxygen-evolving photosynthetic organisms.

#### Molecular Evolution of the HY2 Family of
Bilin Reductases

A single phylogenetic tree that is well supported with boot-
strap replicates was obtained for the HY2 family (Figure 3).
This tree delineates four clades of bilin reductases, which is
in good agreement with the enzymes’ double-bond specific-
ity for reduction. HY2 appears most closely related to the
PebB clade of enzymes that catalyze a reduction of 15,16-
DHBV to PEB. We predict that phytochromobilin synthases,
because of their exquisite BV substrate specificity, will form
a distinct clade when HY2 orthologs from other plant species
are identified. The relatedness of HY2 and PebA enzymes is
reasonable because both families mediate reduction of the
vinyl pyrrole A ring to form the ethylidene moiety. We specu-
late that these two classes arose from a common ancestor
that used BV as a substrate. This notion is based on the ob-
servation that the PebA family of bilin reductases also rec-
ognizes BV as a substrate.
Unlike HY2 and PebB, members of the PebA family target
the 15,16 double bond of BV for reduction. To evolve the
PebA and PebB/HY2 subfamilies, we envisage a duplication
of a ferredoxin-dependent BV reductase gene and subse-
quent divergence in a marine cyanobacterium growing in
a light-limited environment. Such an environment would
provide the selection pressure favoring evolution of the bio-
synthetic pathway for PEB, whose incorporation into phyco-
erythrin extends the light-harvesting wavelength range of
their phycobilisomes. Depending on the rooting of the HY2
family tree, the comparative branch lengths of the PebA and
PebB/HY2 families on the phylogenetic tree suggest that
the A ring reductases are more ancient, with the 15,16 re-
ductases evolving more recently. On the basis of these in-
ferences, we speculate that a cyanobacterial progenitor of
plant chloroplasts possessed a bilin reductase with an A
ring reductase regiospecificity. The progenitor of present
day cyanobacteria likely would have possessed the ability to
synthesize PCB, an essential component of their allophyco-
cyanin-containing phycobilisome core. Thus, the common
pebA/pebB ancestor might have resembled present-day
pcyA genes, which encode atypical BV reductases that cat-
alyze the four-electron reduction of BV to PCB. To date,
pcyA genes appear to be present in all cyanobacteria,
whereas a peb operon is lacking in the phycoerythrin-defi-
cient cyanobacterium *Synechocystis* sp PCC6803.
The role of the *pebA*, *pebB*, and *pcyA* genes in *Prochloro-
coccus* sp MED4 remains a mystery. Members of this genus
are distinguished by the presence of integral membrane an-
tennae complexes that contain divinyl chlorophyll *a*₂ and *b*₂
and by the lack of phycobilisomes (Partensky et al., 1999).
Functional phycoerythrins have been detected only for the

SS120 subspecies. As such, these organisms have been thought by some to be descendants of the class of prokaryotic photosynthetic organisms whose endosymbiosis led to higher plant chloroplasts. Phylogenetic analyses using 16S rRNA indicate that this probably is not the case, because *Prochlorococcus* species appear more similar to marine *Synechococcus* species than to chloroplasts (Urbach et al., 1998). These analyses also suggest that *Prochlorococcus* evolved more recently from a phycobilisome-containing ancestor that resembled a marine *Synechococcus* species. The need for *pebA*, *pebB*, and *pcyA* genes for phycobilin biosynthesis in this ancestor is self-evident, and such genes may not yet have been lost from *Prochlorococcus* species. It is conceivable that these BV reductases are required to make bilin chromophore precursors of light receptors, such as the phytochromes (Hughes and Lampter, 1999). Although phytochrome-like genes are abundant in some cyanobacterial genomes, none are present in the genome of *Prochlorococcus* sp MED4 (data not shown). Alternatively, BV reductases may be needed to drive heme oxygenase, whose role in iron metabolism is well documented (Poss and Tonegawa, 1997a, 1997b; Richaud and Zabulon, 1997; Schmitt, 1997).

In addition to the bilin reductases involved in phytobilin biosynthesis, a separate class exists of bilin reductases that are involved in chlorophyll degradation (Hörtensteiner et al., 2000; Wüthrich et al., 2000). The pathway of chlorophyll degradation that occurs during plant senescence is similar to the heme degradation pathway (Matile and Hörtensteiner, 1999). After dephytylation and magnesium removal, the chlorophyll macrocycle ring is opened by a monooxygenase that has yet to be cloned (Hörtensteiner et al., 1998). This is followed by a ferredoxin-dependent reduction of the bilin product catalyzed by the RCCR (Hörtensteiner et al., 2000; Wüthrich et al., 2000). RCCRs are the most diverged members of the ferredoxin-dependent bilin reductase family. Indeed, these enzymes have markedly different substrate specificities. It is notable that RCCRs catalyze a reduction very similar to that mediated by the PebA family (i.e., a 15,16 double-bond reduction). The structural determinants that are responsible for RCCR’s unique substrate specificity and double-bond regiospecificity will be interesting to discover. Presumably, chlorophyll catabolism would be important for chlorophyll-containing prokaryotes; however, to date, RCCR genes are not readily identifiable in the genomes of any photosynthetic prokaryotes. It is possible that RCCR genes were lost, or alternately, that they evolved more recently from an HY2-like gene in the chloroplast endosymbiont progenitor, because they are found in cryptogams and plants (Hörtensteiner et al., 2000).

mechanistically quite different from the NAD(P)H-dependent BVR/BvrD family of BV reductases. Preliminary analyses to date have failed to identify a metal or flavin cofactor in any of the recombinant enzymes reported here, suggesting that electrons are transferred directly to the bilin moiety, possibly via reduction of an amino acid residue within the enzyme. Although this finding suggests the presence of bilin radical intermediates, additional experiments are needed to assess this hypothesis. The oxygen sensitivity of RCCR supports the hypothesis that bilin radicals, which react with molecular oxygen, are produced during RCC catalysis (Wüthrich et al., 2000).

Examination of highly conserved residues in the entire HY2 family and those within each of the five classes of bilin reductases provides information regarding residues important to the protein structure, ferredoxin interaction site, and substrate/product specificity. In this regard, only a small number of residues are conserved in the entire HY2 family of enzymes. These include hydrophobic residues at positions 137, 157, 158, 256, and 314, Pro-151, Phe-221, Ser-222, and Asp-171 (Figure 2). The notable lack of conserved basic residues suggests that the propionyl moieties of the bilin substrates do not form salt linkages with the enzymes. The conserved hydrophobic residues proline and phenylalanine are likely to be involved in overall protein structure (i.e., folding). Alternately, they may form hydrophobic interactions with conserved regions of the various bilin substrates. The loss-of-function *hy2-1* and *hy2-104* alleles of phytochromobilin synthase from Arabidopsis support the critical role of Pro-151 in HY2’s structure. The conserved serine and aspartate residues likely play catalytic roles, such as hydrogen bonding with the substrate and/or substrate protonation to make the bound bilin a better electron acceptor.

Despite the wide divergence of the HY2 family, we believe that these conserved residues indicate that the active sites of all members of this class are similar. We speculate that the distinct double-bond reduction specificities of the BV reductases (i.e., PcyA, PebA, HY2), the 15,16-DHBV reductases (i.e., PebB), and the RCCR families reflect the positioning of the respective substrates within the catalytic pocket. Because the A/B and C/D rings of BV are very similar but not identical, it is conceivable that the substrate binding sites of the PebA and HY2 enzymes are tailored to position BV in opposite orientations, favoring electron transfer to the bilin C/D ring or A ring, respectively. If this is true, then the PebB class might tether its 15,16-DHBV substrate in an orientation similar to that of the HY2 family, whereas RCC might be bound to RCCR in a manner similar to that in which BV is bound to PebA. Future studies will address the unique substrate/product specificity using domain swapping, site-directed mutagenesis, synthetic biliverdin substrates, and x-ray crystallography.

### Mechanistic Implications

Ferredoxin-dependent bilin reductase catalyzes two- and four-electron reductions of linear tetrapyrroles. Because ferredoxin is a one-electron carrier, these enzymes are

### Biotechnological Implications

The availability of genes for bilin reductases that mediate the biosynthesis of PΦB, PCB, and PEB provides us with

useful tools for numerous biotechnological applications. The ability to engineer the biosynthesis of PEB in any BV-producing organism is now feasible via the introduction of one or two genes. In this way, phytofluors potentially can be produced in any ferredoxin-containing organism. Coexpression of bilin reductase genes with apophytochromes should enable us to produce holophytochromes in bacteria and yeast. This will facilitate not only three-dimensional structural analysis of phytochrome but also the reconstruction of phytochrome signaling in a nonplant system in which we can exploit the power of molecular genetic analyses. This approach has proven invaluable for the structure–function analysis of the steroid hormone receptor family. By introducing the pcyA gene into wild-type and chromophore-deficient mutant plants, we also should be able to change the wavelength specificity of phytochrome, which may favorably alter plant growth and development in the field environment. Introduction of the pebA and pebB genes into plants potentially will shunt the conversion of BV to PEB, yielding photomorphogenetically challenged plants with fluorescent phytochromes. This would be especially useful for the analysis of the temporal and spatial patterns of phytochrome expression in plants.

### METHODS

#### Reagents

All chemicals, including glutathione agarose, were purchased from Sigma (St. Louis, MO) and were American Chemical Society grade or better. Restriction enzymes and Taq polymerase were from Gibco BRL (Cleveland, OH). HPLC-grade acetone and 80% formic acid were purchased from Fisher Scientific (Pittsburgh, PA). The expression vector pGEX-6P-1 and PreScission protease were obtained from Amersham Pharmacia Biotech (Piscataway, NJ). Centricon-10 concentrator devices were purchased from Amicon (Beverly, MA).

#### Bioinformatics

Protein and nucleic acid database searches were performed using programs at publicly available World Wide Web sites. Preliminary sequence data were obtained from the Department of Energy Joint Genome Institute (http://spider.jgi-psf.org/JGI_microbial/html/). Multiple sequence alignments were performed using the programs CLUSTAL W (Higgins et al., 1996), GENEDOC (http://www.psc.edu/biomed/genedoc), and MEME (Bailey and Elkan, 1995) to guide hand alignments. Phylogenetic analysis of the HY2-related family of proteins based on the alignment shown in Figure 2 was conducted using a heuristic parsimony search with a modified PAM250 weighting matrix (Dayhoff et al., 1978) using the program PAUP* version 4.0 (Swofford, 1993). Because there are negative values in the PAM250 matrix, the most negative penalty was set equal to zero, and all other values were increased correspondingly. Scores for transitions to and from gaps were not defined in the original matrix; they were set equal to the most costly transition (25) defined in the matrix. These modifi-

cations were made by R.K. Kuzoff (personal communication). Characters 1 to 65 and 323 to 368 in the alignment were excluded from our analysis because they correspond to N- and C-terminal extensions not common to all members of the HY2 family (i.e., plastid transit peptide found on HY2 and red chlorophyll catabolite reductase [RCCR], C-terminal extension found only on HY2). For *Hordeum vulgare* RCCR, missing characters 65 to 116 were replaced with question marks, which were weighted as zero. A rescaled consistency index was used for character weighting.

#### Construction of Expression Vectors

HY2-related genes from *Synechocystis* sp PCC6803, *Synechococcus* sp WH8020, and *Anabaena* sp PCC7120 were amplified from chromosomal DNA via polymerase chain reaction using the following primers, which contained the indicated and underlined restriction sites: *Synechocystis* pcyA, BamHIfwd: 5′-AAGGATCCATGGGCCGTCACTGATTTAAG-3′, Sallrev: 5′-ACGCGTCGACTATTATTTGGATAACATCAAATAAGAC-3′; *Synechococcus* pebA, EcoRIfwd: 5′-GGAATTCATCTTTGATTCATTTCTCAATG-3′, NotIrev: 5′-ATAGTTAGCGGCATCTTTGTGAGAGGAGGAGGC-3′; *Synechococcus* pebB, EcoRIfwd: 5′-GGAATTCATCACAAATCAAAAGATTCAAAAGC-3′, NotIrev: 5′-ATAGTTAGCGGCCGCTTATAGATCAAAAAGCACAGTGTTGG-3′; and *Anabaena* pcyA, EcoRIfwd: 5′-GGAATTCATCTCACTTACTTCCATCCCCTC-3′, NotIrev: 5′-ATAGTTAGCGGCCGCTTATTCTGGGAGATCAAATAAC-3′. The polymerase chain reaction products were then cut with the indicated enzymes and inserted into similarly restricted pGEX-6P-1. The integrity of the plasmid constructs was verified by complete DNA sequence determination of the insert (Davis Sequencing, Davis, CA). All of the constructs place the HY2-related gene downstream of and in frame with the glutathione S-transferase (GST) gene of *Schistosoma japonicum* under the control of a Ptac promoter. A recognition sequence for PreScission protease is located upstream of the cloned gene. Proteolytic cleavage yields the native protein with a small N-terminal extension. In all cases, the original initiation methionine was changed to an isoleucine.

#### Expression and Purification

Expression and purification were performed according to instructions supplied by the manufacturer (Amersham Pharmacia Biotech) and as described before (Kohchi et al., 2001). Between 1 and 10 mg of purified protein was obtained per liter of bacterial culture.

#### Protein Determination

Protein concentration was determined by the Bradford method with BSA as a standard (Bradford, 1976) or by measuring the absorbance at 280 nm and using the calculated ε<sub>280 nm</sub> for each individual protein (Gill and von Hippel, 1989).

#### Standard Bilin Reductase Activity Assay

Assays for bilin reductase activity were performed as described previously for PΦB synthase (Kohchi et al., 2001) and are based on the PhD dissertation work of Michael T. McDowell (M.T. McDowell and J.C. Lagarias, manuscript submitted).

Direct HPLC Analysis

Bilin reductase assay mixtures were loaded onto a Waters (Milford, MA) C<sub>18</sub> Sep-Pak Light preconditioned as follows: 3-mL wash with acetonitrile to wet the Sep-Pak, 3-mL wash with MilliQ water, and 3-mL wash with 50 mM 4-methylmorpholine/glacial acetic acid, pH 7.7. After the sample was loaded onto the Sep-Pak, it was washed with 3 mL of 4-methylmorpholine/glacial acetic acid, pH 7.7, followed by 3 mL of 0.1% (v/v) trifluoroacetic acid. The bilin metabolites were then eluted from the Sep-Pak with 2 mL of 100% acetonitrile. The eluate was dried using a Speed-Vac lyophilizer (Savant Instruments Inc., Farmingdale, NY), and the dried samples were analyzed by HPLC. Samples were first dissolved in 10 μL of DMSO and then diluted with 200 μL of the HPLC mobile phase (50:50 v/v acetone:20 mM formic acid). After the samples were dissolved, they were centrifuged briefly to collect the sample, passed through a 0.45-μm polytetrafluoroethylene syringe filter, and chromatographed using a Varian (Palo Alto, CA) 5000 liquid chromatograph. The HPLC column used for all of the analyses was a Phenomenex (Torrance, CA) Ultra-carb 5-μm ODS20 4.6 × 250-mm analytical column with a 4.6 × 30-mm guard column of the same material. The mobile phase used with this column was acetone:20 mM formic acid (50:50, v/v). The flow rate was 0.8 mL/min. The eluate was monitored at 560 nm for the first 11.5 min and at 380 nm for the remaining time using a Varian UV100 flow-through absorbance detector. Peak areas were quantitated using a Hewlett-Packard (Palo Alto, CA) model 3365 Chemstation II.

Coupled Spectrophotometric and Spectrofluorometric Analysis

An aliquot of ~20 μg of crude recombinant Cph1 apoprotein (Yeh and Lagarias, 1998) was added to 1 mL of bilin reductase assay mixture under green safelight. Mixtures were incubated for 30 min at room temperature to permit phytobilin binding. Phytochrome difference spectra were obtained as described previously (Terry and Lagarias, 1991). A spectrofluorometric assay was used to detect the formation of intensely fluorescent phycocerythrobilin (PEB) adducts of Cph1 (Murphy and Lagarias, 1997). Emission spectra were obtained with an excitation wavelength of 545 nm using an SLM Aminco Bowman AB2 spectrofluorometer (Spectronic Instruments Inc., Rochester, NY).

ACKNOWLEDGMENTS

We thank Dr. Alex Glazer for generously providing us with Synechococcus sp WH8020 cells, Dr. Wendy Schluchter for Anabaena sp PCC7120 genomic DNA, Elsie Campbell for *N. punctiforme* cells, and Drs. Jane Lamerdin and E. Teresa Janecki for *Prochlorococcus* sp MED4 DNA. We thank Dr. Gabrielle Rocap for helpful information regarding *Prochlorococcus* nomenclature and Dr. Bob Kuzoff for help with phylogenetic analysis. Preliminary sequence data were obtained from the DOE Joint Genome Institute (JGI) at http://spider.jgi-psf.org/JGI_microbial/html/. Thanks also are due to Beronda Montgomery and Jennifer Santos for critically reading the manuscript. This work was supported in part by the U.S. Department of Agriculture National Research Initiative Competitive Research Grants program (Grant No. AMD-9801768 to J.C.L.), the Research for the Future Program (Grant No. JSPS-RFTF 00L01605 to T.K.), a Grant-in-Aid for Scientific Research on Priority Areas from the Ministry of Education, Sciences, Sports, and Culture of Japan to T.K., and a research fellowship from the Deutsche Forschungsgemeinschaft to N.F. (FR 1487/1-1).

Received September 25, 2000; accepted January 27, 2001.

REFERENCES

Altschul, S.F., Gish, W., Miller, W., Myers, E.W., and Lipman, D.J. (1990). Basic local alignment search tool. J. Mol. Biol. **215**, 403–410.

Altschul, S.F., Madden, T.L., Schaffer, A.A., Zhang, J.H., Zhang, Z., Miller, W., and Lipman, D.J. (1997). Gapped BLAST and PSI-BLAST: A new generation of protein database search programs. Nucleic Acids Res. **25**, 3389–3402.

Ashby, M.K., and Mullineaux, C.W. (1999). Cyanobacterial *ycf27* gene products regulate energy transfer from phycobilisomes to photosystems I and II. FEMS Microbiol. Lett. **181**, 253–260.

Bailey, T.L., and Elkan, C. (1995). The value of prior knowledge in discovering motifs with MEME. In Proceedings of the Third International Conference on Intelligent Systems for Molecular Biology (Menlo Park, CA: American Association of Artificial Intelligence Press), pp. 21–29.

Beale, S.I. (1993). Biosynthesis of phycobilins. Chem. Rev. **93**, 785–802.

Beale, S.I., and Cornejo, J. (1991a). Biosynthesis of phycobilins: Ferredoxin-mediated reduction of biliverdin catalyzed by extracts of *Cyanidium caldarium*. J. Biol. Chem. **266**, 22328–22332.

Beale, S.I., and Cornejo, J. (1991b). Biosynthesis of phycobilins: 3(Z)-Phycoerythrobilin and 3(Z)-phycocyanobilin are intermediates in the formation of 3(E)-phycocyanobilin from biliverdin IXa. J. Biol. Chem. **266**, 22333–22340.

Beale, S.I., and Cornejo, J. (1991c). Biosynthesis of phycobilins: 15,16-Dihydrobiliverdin IXa is a partially reduced intermediate in the formation of phycobilins from biliverdin IXa. J. Biol. Chem. **266**, 22341–22345.

Bradford, M.M. (1976). A rapid and sensitive method for the quantitation of microgram quantities of protein using the principle of protein-dye binding. Anal. Biochem. **72**, 248–254.

Cornejo, J., and Beale, S.I. (1997). Phycobilin biosynthetic reactions in extracts of cyanobacteria. Photosynth. Res. **51**, 223–230.

Cornejo, J., Willows, R.D., and Beale, S.I. (1998). Phytoobilin biosynthesis: Cloning and expression of a gene encoding soluble ferredoxin-dependent heme oxygenase from *Synechocystis* sp. PCC 6803. Plant J. **15**, 99–107.

Davis, S.J., Kurepa, J., and Vierstra, R. (1999). The *Arabidopsis thaliana HY1* locus, required for phytochrome-chromophore biosynthesis, encodes a protein related to heme oxygenases. Proc. Natl. Acad. Sci. USA **96**, 6541–6546.

Dayhoff, M.O., Schwartz, R.M., and Orcutt, B.C. (1978). A model of evolutionary change in proteins. In Atlas of Protein Sequences and Structure, M.O. Dayhoff, ed (Washington, DC: National Biomedical Research Foundation), pp. 345–352.

Gill, S.C., and von Hippel, P.H. (1989). Calculation of protein

extinction coefficients from amino acid sequence data. Anal. Biochem. 182, 319–326.

Hess, W.R., Partensky, F., Van der Staay, G.W.M., Garcia-Fernandez, J.M., Börner, T., and Vallot, D. (1996). Coexistence of phycoerythrin and a chlorophyll a/b antenna in a marine prokaryote. Proc. Natl. Acad. Sci. USA 93, 11126–11130.

Hess, W.R., Steglich, C., Lichtle, C., and Partensky, F. (1999). Phycoerythrins of the oxyphotobacterium *Prochlorococcus marinus* are associated to the thylakoid membrane and are encoded by a single large gene cluster. Plant Mol. Biol. 40, 507–521.

Higgins, D.G., Thompson, J.D., and Gibson, T.J. (1996). Using CLUSTAL for multiple sequence alignments. Methods Enzymol. 266, 383–402.

Hörtensteiner, S., Wüthrich, K.L., Matile, P., Ongania, K.H., and Kräutler, B. (1998). The key step in chlorophyll breakdown in higher plants: Cleavage of pheophorbide alpha macrocycle by a monooxygenase. J. Biol. Chem. 273, 15335–15339.

Hörtensteiner, S., Rodoni, S., Schellenberg, M., Vicentini, F., Nandi, O.I., Qui, Y.L., and Matile, P. (2000). Evolution of chlorophyll degradation: The significance of RCC reductase. Plant Biol. 2, 63–67.

Hughes, J., and Lamparter, T. (1999). Prokaryotes and phytochrome: The connection to chromophores and signaling. Plant Physiol. 121, 1059–1068.

Kohchi, T., Mukougawa, K., Frankenberg, N., Masuda, M., Yokota, A., and Lagarias, J.C. (2001). The Arabidopsis *HY2* gene encodes phytochromobilin synthase, a ferredoxin-dependent biliverdin reductase. Plant Cell 13, 425–436.

Lokstein, H., Steglich, C., and Hess, W.R. (1999). Light-harvesting antenna function of phycoerythrin in *Prochlorococcus marinus*. Biochim. Biophys. Acta 1410, 97–98.

Maines, M.D. (1988). Heme oxygenase: function, multiplicity, regulatory mechanisms, and clinical applications. FASEB J. 2, 2557–2568.

Maines, M.D., and Trakshel, G.M. (1993). Purification and characterization of human biliverdin reductase. Arch. Biochem. Biophys. 300, 320–326.

Matile, P., and Hörtensteiner, S. (1999). Chlorophyll degradation. Annu. Rev. Plant Physiol. Plant Mol. Biol. 50, 67–95.

Muramoto, T., Kohchi, T., Yokota, A., Hwang, I., and Goodman, H.M. (1999). The Arabidopsis photomorphogenic mutant *hy1* is deficient in phytochrome chromophore biosynthesis as a result of a mutation in a plastid heme oxygenase. Plant Cell 11, 335–347.

Murphy, J.T., and Lagarias, J.C. (1997). The phytofluors: A new class of fluorescent protein probes. Curr. Biol. 7, 870–876.

Nakamura, Y., Kaneko, T., and Tabata, S. (2000). CyanoBase, the genome database for *Synechocystis* sp. strain PCC6803: Status for the year 2000. Nucleic Acids Res. 28, 72.

Ong, L., and Glazer, A. (1991). Phycoerythrins of marine unicellular cyanobacteria. I. Bilin types and locations and energy transfer pathways in *Synechococcus* spp. phycoerythrins. J. Biol. Chem. 266, 9515–9527.

Partensky, F., Hess, W.R., and Vallot, D. (1999). *Prochlorococcus*, a marine photosynthetic prokaryote of global significance. Microbiol. Mol. Biol. Rev. 63, 106–127.

Poss, K.D., and Tonegawa, S. (1997a). Heme oxygenase 1 is required for mammalian iron reutilization. Proc. Natl. Acad. Sci. USA 94, 10919–10924.

Poss, K.D., and Tonegawa, S. (1997b). Reduced stress defense in heme oxygenase 1-deficient cells. Proc. Natl. Acad. Sci. USA 94, 10925–10930.

Richaud, C., and Zabulon, G. (1997). The heme oxygenase gene (*pbsA*) in the red alga *Rhodella violacea* is discontinuous and transcriptionally activated during iron limitation. Proc. Natl. Acad. Sci. USA 94, 11736–11741.

Schirmer, T., Bode, W., and Huber, R. (1987). Refined three-dimensional structures of two cyanobacterial C-phycocyanins at 2.1- and 2.5-angstrom resolution: A common principle of phyco-bilin-protein interaction. J. Mol. Biol. 196, 677–695.

Schluchter, W.M., and Glazer, A.N. (1997). Characterization of cyanobacterial biliverdin reductase: Conversion of biliverdin to bilirubin is important for normal phycobiliprotein biosynthesis. J. Biol. Chem. 272, 13562–13569.

Schmidt, G., Siebzehnrübl, S., Fischer, R., Rüdiger, W., Scheer, H., Schirmer, T., Bode, W., and Huber, R. (1987). ZZE-Configuration of chromophore beta-153 in C-phycocyanin from *Mastigocladus laminosus*. Z. Naturforsch. 42C, 845–848.

Schmitt, M.P. (1997). Utilization of host iron sources by *Corynebacterium diphtheriae*: Identification of a gene whose product is homologous to eukaryotic heme oxygenases and is required for acquisition of iron from heme and hemoglobin. J. Bacteriol. 179, 838–845.

Swofford, D.L. (1993). PAUP: A computer program for phylogenetic inference using maximum parsimony. J. Gen. Physiol. 102, 9A.

Terry, M.J., and Lagarias, J.C. (1991). Holophytochrome assembly: Coupled assay for phytochromobilin synthase *in organello*. J. Biol. Chem. 266, 22215–22221.

Terry, M.J., Wahleithner, J.A., and Lagarias, J.C. (1993). Biosynthesis of the plant photoreceptor phytochrome. Arch. Biochem. Biophys. 306, 1–15.

Terry, M.J., McDowell, M.T., and Lagarias, J.C. (1995). (3Z)- and (3E)-phytochromobilin are intermediates in the biosynthesis of the phytochrome chromophore. J. Biol. Chem. 270, 11111–11118.

Urbach, E., Scanlan, D.J., Distel, D.L., Waterbury, J.B., and Chisholm, S.W. (1998). Rapid diversification of marine picophytoplankton with dissimilar light-harvesting structures inferred from sequences of *Prochlorococcus* and *Synechococcus* (cyanobacteria). J. Mol. Evol. 46, 188–201.

Wedemeyer, G.J., Kidd, D.G., and Glazer, A.N. (1996). Cryptomonad biliproteins: Bilin types and locations. Photosynth. Res. 48, 163–170.

Wilbanks, S.M., and Glazer, A.N. (1993a). Rod structure of a phycoerythrin-II-containing phycobilisome. 1. Organization and sequence of the gene cluster encoding the major phycobiliprotein rod components in the genome of marine *Synechococcus* sp.WH8020. J. Biol. Chem. 268, 1226–1235.

Wilbanks, S.M., and Glazer, A.N. (1993b). Rod structure of a phycoerythrin-II–containing phycobilisome. 2. Complete sequence and bilin attachment site of a phycoerythrin gamma-subunit. J. Biol. Chem. 268, 1236–1241.

Wu, S.-H., McDowell, M.T., and Lagarias, J.C. (1997). Phycocyanobilin is the natural chromophore precursor of phytochrome from the green alga *Mesotaenium caldariorum*. J. Biol. Chem. 272, 25700–25705.

Wüthrich, K.L., Bovet, L., Hunziker, P.E., Donnison, I.S., and Hörtensteiner, S. (2000). Molecular cloning, functional expression and characterisation of RCC reductase involved in chlorophyll catabolism. Plant J. 21, 189–198.

Yeh, K.-C., and Lagarias, J.C. (1998). Eukaryotic phytochromes: Light-regulated serine/threonine protein kinases with histidine kinase ancestry. Proc. Natl. Acad. Sci. USA 95, 13976–13981.

Yeh, K.-C., Wu, S.-H., Murphy, J.T., and Lagarias, J.C. (1997). Cyanobacterial phytochrome two-component light sensory system. Science 277, 1505–1508.

Zhao, K.H., Deng, M.G., Zheng, M., Zhou, M., Parbel, A., Storf, M., Meyer, M., Strohmann, B., and Scheer, H. (2000). Novel activity of a phycobiliprotein lyase: Both the attachment of phycocyanobilin and the isomerization to phycovirolobilin are catalyzed by the proteins PecE and PecF encoded by the phycoerythrocyanin operon. FEBS Lett. 469, 9–13.

Functional Genomic Analysis of the HY2 Family of Ferredoxin-Dependent Bilin Reductases from Oxygenic Photosynthetic Organisms

Nicole Frankenberg, Keiko Mukougawa, Takayuki Kohchi and J. Clark Lagarias

Plant Cell 2001;13;965-978

DOI 10.1105/tpc.13.4.965

This information is current as of June 5, 2012

| References | This article cites 46 articles, 26 of which can be accessed free at: http://www.plantcell.org/content/13/4/965.full.html#ref-list-1 |
| Permissions | https://www.copyright.com/ccc/openurl.do?sid=pd_hw1532298X&issn=1532298X&WT.mc_id=pd_hw1532298X |
| eTOCs | Sign up for eTOCs at: http://www.plantcell.org/cgi/alerts/ctmain |
| CiteTrack Alerts | Sign up for CiteTrack Alerts at: http://www.plantcell.org/cgi/alerts/ctmain |
| Subscription Information | Subscription Information for The Plant Cell and Plant Physiology is available at: http://www.aspb.org/publications/subscriptions.cfm |

© American Society of Plant Biologists

ADVANCING THE SCIENCE OF PLANT BIOLOGY
