
# Cloning and Expression of the Benzoate Dioxygenase Genes from *Rhodococcus sp.* Strain 19070

**Sandra Haddad, D. Matthew Eby, and Ellen L. Neidle**

*Appl. Environ. Microbiol.* 2001, 67(6):2507. DOI: 10.1128/AEM.67.6.2507-2514.2001.

---

## Updated information and services can be found at:
[http://aem.asm.org/content/67/6/2507](http://aem.asm.org/content/67/6/2507)

### These include:

#### REFERENCES
This article cites 46 articles, 28 of which can be accessed free at: [http://aem.asm.org/content/67/6/2507#ref-list-1](http://aem.asm.org/content/67/6/2507#ref-list-1)

#### CONTENT ALERTS
Receive: RSS Feeds, eTOCs, free email alerts (when new articles cite this article), [more »](more%20%E2%80%BA)

---

## Information about commercial reprint orders:
[http://journals.asm.org/site/misc/reprints.xhtml](http://journals.asm.org/site/misc/reprints.xhtml)

To subscribe to another ASM Journal go to:
[http://journals.asm.org/site/subscriptions/](http://journals.asm.org/site/subscriptions/)

---

[Journals.ASM.org](https://journals.asm.org)

APPLIED AND ENVIRONMENTAL MICROBIOLOGY, June 2001, p. 2507–2514  
0099-2240/01/$04.00+0 DOI: 10.1128/AEM.67.6.2507–2514.2001  
Copyright © 2001, American Society for Microbiology. All Rights Reserved.

**Cloning and Expression of the Benzoate Dioxygenase Genes from Rhodococcus sp. Strain 19070**

SANDRA HADDAD,* D. MATTHEW EBY, AND ELLEN L. NEIDLE  

*Department of Microbiology, University of Georgia, Athens, Georgia 30602*

Received 7 September 2000/Accepted 12 March 2001

The *bopXYZ* genes from the gram-positive bacterium *Rhodococcus* sp. strain 19070 encode a broad-substrate-specific benzoate dioxygenase. Expression of the BopXY terminal oxygenase enabled *Escherichia coli* to convert benzoate or anthranilate (2-aminobenzoate) to a nonaromatic *cis*-diol or catechol, respectively. This expression system also rapidly transformed *m*-toluate (3-methylbenzoate) to an unidentified product. In contrast, 2-chlorobenzoate was not a good substrate. The BopXYZ dioxygenase was homologous to the chromosomally encoded benzoate dioxygenase (BenABC) and the plasmid-encoded toluate dioxygenase (XylXYZ) of gram-negative acinetobacters and pseudomonads. Pulsed-field gel electrophoresis failed to identify any plasmid in *Rhodococcus* sp. strain 19070. Catechol 1,2- and 2,3-dioxygenase activity indicated that strain 19070 possesses both *meta-* and *ortho*-cleavage degradative pathways, which are associated in pseudomonads with the *xyl* and *ben* genes, respectively. Open reading frames downstream of *bopXYZ*, designated *bopL* and *bopK*, resembled genes encoding *cis*-diol dehydrogenases and benzoate transporters, respectively. The *bop* genes were in the same order as the chromosomal *ben* genes of *P. putida* PRS2000. The deduced sequences of BopXY were 50 to 60% identical to the corresponding proteins of benzoate and toluate dioxygenases. The reductase components of these latter dioxygenases, BenC and XylZ, are 201 residues shorter than the deduced BopZ sequence. As predicted from the sequence, expression of BopZ in *E. coli* yielded an approximately 60-kDa protein whose presence corresponded to increased cytochrome *c* reductase activity. While the N-terminal region of BopZ was approximately 50% identical in sequence to the entire BenC or XylZ reductases, the C terminus was unlike other known protein sequences.

For many years, investigations of the prokaryotic degradation of aromatic compounds focused almost exclusively on the metabolism of gram-negative bacteria. Nevertheless, some gram-positive bacteria, such as the actinomycetes, can mineralize a wide array of hydrocarbons, including *n*-alkanes and aromatic compounds (1, 7, 8, 18, 29, 45–47). For example, in a study of 34 strains of actinomycetes representing nine genera, *Rhodococcus* sp. strain 19070 was found to use at least six different long-chain alkanes as the sole source of carbon and energy (17, 27). In addition, each of the following aromatic compounds supported the growth of this strain: toluene, *m*-xylene, *p*-xylene, *o*-xylene, trimethylbenzene, benzyl alcohol, and benzoate. Relatively little is known about the biochemistry or genetics of the catabolic routes for these compounds in *Rhodococcus*.

To exploit the microbial potential for bioremediation and environmental detoxification and to understand more fully how catabolic pathways have evolved in diverse microorganisms, more information is needed about aromatic compound degradation by actinomycetes. By analogy with catabolic pathways of gram-negative bacteria, some of the aromatic compounds listed above that serve as carbon sources for *Rhodococcus* sp. strain 19070 might be converted to catechol or a substituted catechol (22, 23). Aromatic ring cleavage could then be mediated by either an intradiol or extradiol ring-cleavage dioxygenase. Both intradiol and extradiol catechol dioxygenases have been studied from several *Rhodococcus* strains (12, 26, 41).

The well-characterized degradation of toluene, xylene, benzyl alcohol, and related compounds occurs via a plasmid-encoded pathway in gram-negative *Pseudomonas* species. The TOL plasmid pWW0 of *Pseudomonas putida* mt-2 encodes broad-substrate-specific enzymes that convert the initial growth substrate to either benzoate or a substituted benzoate (51). Dihydroxylation of the benzoate ring by the *xylXYZ*-encoded dioxygenase then yields a nonaromatic *cis*-diol which is converted to catechol by a *xylL*-encoded dehydrogenase. The TOL plasmid also encodes enzymes for the extradiol cleavage of catechol and a “meta-cleavage” pathway that feeds catabolites into the tricarboxylic acid cycle (19, 22).

*Pseudomonas* strains carrying TOL plasmids also have a chromosomally encoded pathway for the degradation of benzoate but not substituted benzoates. The chromosomal *benABC* genes, which are evolutionarily related to *xylXYZ*, encode a relatively narrow substrate-specific aromatic ring hydroxylating benzoate dioxygenase (20, 34). The product of the BenABC-catalyzed reaction is converted to catechol by the *benD*-encoded dehydrogenase (32). Catechol is then cleaved by an intradiol ring-cleavage dioxygenase, and catabolism proceeds via the β-ketoadipate pathway (23).

In this study, the *xylXYZ* genes of the TOL plasmid pWW0 were used as hybridization probes to isolate homologs from *Rhodococcus* sp. strain 19070. The *xylXY* counterparts from *Rhodococcus* 19070 were expressed in *Escherichia coli* and enabled rapid transformations of benzoate, anthranilate, and *m*-toluate. In addition, adjacent genes were isolated and found to be homologous to genes involved in benzoate degradation.

---

*Corresponding author. Mailing address: Department of Microbiology, University of Georgia, Athens, GA 30602-2605. Phone: (706) 542-2852. Fax: (706) 542-2674. E-mail: shaddad@arches.uga.edu.*

There were sequence similarities to both the *xyl* and *ben* genes of pseudomonads. The apparent broad substrate range of the *Rhodococcus* enzyme was reminiscent of the *xylXYZ*-encoded toluate dioxygenase, yet the gene order matched that of the chromosomal *ben* genes of *P. putida*. Since it was not clear which designation was most appropriate for the *Rhodococcus* genes, they were given the distinct name *bop* (benzoate oxidation participation). The characterization of these genes enabled the construction of phylogenetic trees to evaluate the evolution of enzymes involved in benzoate degradation by bacteria.

---

### MATERIALS AND METHODS

#### Bacterial strains, plasmids, and culture conditions.
*Rhodococcus* sp. strain ATCC 19070 was grown at 37°C in Luria-Bertani broth or M9 minimal medium (39). Carbon sources were provided at 4 mM. Toluene, as a carbon source, was placed in the arm of a sidearm flask. Growth conditions for *Acinetobacter* sp. strain ADP1, also known as strain BD413 (25), have been described (42). *E. coli* strains XL-1 Blue (Stratagene), DH5α (Life Technologies), and MC1061 (4) were used as hosts. DNA from TOL plasmid pWW0 was isolated from *P. putida* mt-2 (48). Cloning vectors pEMBL8b (9), pUC19, M13mp18, and M13mp19 (50) were used. The *bopZ* gene from *Rhodococcus* sp. strain 19070 genomic DNA was PCR amplified and ligated into pUC19 to form plasmid pSAM3. Plasmid pSAM4 is a derivative of pSAM3, created by deleting a 1-kb *PstI* restriction fragment within the *bopZ* gene. Plasmid pIB1354 carries the *benABC* genes of *Acinetobacter* sp. strain ADP1 (34).

#### Isolation of the *bop* genes, DNA sequence, and analysis.
Total DNA from *Rhodococcus* sp. strain 19070 (24) was digested with *EcoRI* and used to generate a partial genomic library (39). Fragments ranging from 0.5 to 6.0 kb were extracted with phenol from low-melting-temperature agarose and were ligated to pUC19. Following the transformation of *E. coli*, plasmids from 156 transformants were purified and digested with *EcoRI*. Individual *Rhodococcus* DNA inserts were isolated, and samples were fixed on nitrocellulose membranes for hybridization. The 8.2-kb *SacI-D* restriction fragment of the *P. putida* mt-2 TOL pWW0 plasmid, containing *xylXYZLTEGFJ* (14), was radioactively labeled. It hybridized to the 2.3-kb *Rhodococcus* insert from plasmid pSAM1.

This 2.3-kb *EcoRI* fragment, labeled as a probe, was used in standard colony hybridizations (39) to identify adjacent genomic DNA. *Rhodococcus* DNA digested with *BglII* and in the 5- to 12-kb size range was ligated to pUC19 and transformed into *E. coli* MC1061. Approximately 2,000 transformants were screened, and four colonies hybridized to the probe. Three contained an identical 7.4-kb *BglII* fragment of *Rhodococcus* DNA, and one contained a 9.0-kb insert. Southern hybridization indicated that these two *BglII* fragments contained genomic DNA flanking each end of the 2.3-kb *EcoRI* fragment on pSAM1. A 0.9-kb *BamHI-EcoRI* fragment immediately upstream of the *bopY* gene was inserted into pSAM1. This generated plasmid pSAM2 with the complete *bopXY* and partial *bopZ* genes, as was confirmed by nucleotide sequencing.

Some single-stranded DNA, prepared from M13mp19 and M13mp18 clones (39), was sequenced by the Sanger method (40) with the T7 Sequenase 7-deaza-dGTP DNA sequencing kit (Promega). Some DNA was sequenced at the Molecular Genetics and Instrumentation Facility at the University of Georgia. Computer-assisted analysis was done with AssemblyLign and MacVector software (6.5 ed; Oxford Molecular, Ltd.) Homology searches (BLAST) were carried out at the network server of the National Center for Biotechnology Information. Amino acid sequences were aligned using the Pileup program of the Genetics Computer Group package (10). Phylogenetic trees were generated with the Fitch-Margoliash or neighbor-joining method from a distance matrix created by PROTDIST of the PHYLIP program package (13).

#### Southern hybridization and DNA labeling.
DNA fragments were labeled by nick translation with [α-<sup>32</sup>P]dATP (39) or by random-primed labeling with digoxigenin (DIG DNA labeling kit; Roche). Hybridizations were done at 42°C for 12 to 16 h in 3× SSC (1× SSC buffer is 0.15 M NaCl plus 0.015 M sodium citrate; pH 7.4) with 50% formamide (39). Wash conditions after hybridization of the *xyl* probes to *Rhodococcus* DNA consisted of two 5-min washes with 2× SSC plus 0.1% sodium dodecyl sulfate (SDS) at room temperature, three washes for 45 min in 0.1× SSC plus 0.1% SDS at 55°C, and four rinses with 0.1× SSC. Higher-stringency conditions after hybridization of *Rhodococcus* probes to *Rhodococcus* target DNA used 70°C rather than 55°C washes.

#### Analysis of metabolites.
*E. coli* (pSAM2) were grown overnight in 5 ml of M9 medium with glucose (10 mM) and ampicillin (50 mg/ml). Wild-type *Acinetobacter* ADP1 was grown overnight in ADP1 minimal medium with 10 mM succinate. On the addition of 1 mM benzoate, anthranilate, *m*-toluate, or *o*-chlorobenzoate to the cultures, a 1-ml culture sample was collected. Isopropyl β-D-thiogalactopyranoside (IPTG) was then added to each *E. coli* culture to a final concentration of 1 mM to induce expression from the *lac* promoter of recombinant plasmids. Samples of the culture medium (1 ml) were collected at various time points and filtered through 0.22-μm (pore-size) nylon filters (MSI) to remove whole cells. Metabolites were detected by high-performance liquid chromatography (HPLC). The samples were analyzed with a Bio-Rad 2800 HPLC system with an AS-100 HPLC sampler. The samples were resolved on a reversed-phase Bio-Sil QDS-5S HPLC column (BioRad; 250 by 4 mm). Elution at a rate of 0.8 ml/min was carried out with 30% acetonitrile containing 1% acetic acid. The eluant was monitored by UV detection at 230 nm. Under these conditions, the retention times were as follows: benzoate (13.3 min), 2-hydro-1,2-dihydroxybenzoate (*cis*-diol) (3.9 min), anthranilate (8.8 min), catechol (6.9 min), *m*-toluate (23.3 min), and *o*-chlorobenzoate (17.1 min). Authentic chemical samples were analyzed to assure proper peak identification. A sample of the benzoate *cis*-diol was kindly provided by A. Reiner.

#### Expression of *bop* genes in *E. coli*.
Single colonies of *E. coli* with plasmids were used to inoculate 2 ml of medium (Luria-Bertani broth with 150 μg of ampicillin per ml). Cultures at an optical density at 600 nm (OD<sub>600</sub>) of 0.1 were then used to inoculate 50 ml of medium. To these cultures (at an OD<sub>600</sub> of 0.3 to 0.4), IPTG was added to a final concentration of 1 mM. Five hours later, cells were harvested by centrifugation. Cell pellets were stored frozen at −20°C.

#### Electrophoresis.
Proteins were analyzed by SDS–8% polyacrylamide gel electrophoresis (PAGE) with Coomassie blue staining (39). Approximately 20 μg of total protein from each sample was loaded per well. DNA for pulsed-field gel electrophoresis (PFGE) was prepared as described elsewhere (15, 43). A CHEF DRIII system (Bio-Rad) was used to run 1% agarose gels at 4°C with a 150-V/cm electric field for 38 h. Pulse times increased from 50 to 120 s, and the field angle was 120°. For hybridization, DNA was transferred to a nylon membrane (TurboBlotter; Schleicher & Schuell).

#### Enzyme assays.
Cells were sonicated, and cell extracts were prepared (42). Protein concentrations were determined with bovine serum albumin as the standard (2). Catechol 1,2- or 2,3-dioxygenase activity was assayed spectrophotometrically by monitoring the increase in *cis,cis*-muconate concentration at A<sub>260</sub> or the increase in 2-hydroxymuconic semialdehyde at A<sub>375</sub>, respectively (31, 35, 38). Treatment with 40 mM hydrogen peroxide inactivated catechol 2,3-dioxygenase (31). Heating at 55°C for 10 min inactivated catechol 1,2-dioxygenase (30). The NADH-dependent reduction of cytochrome *c* was detected by an increase in A<sub>550</sub> (49). The specific activities were averages of at least two independent repetitions done in duplicate. Standard deviations were less than 20% of the reported value.

#### Nucleotide sequence accession number.
The DNA sequence of *bopXYZLK* has been submitted to GenBank under accession no. AF279141.

---

### RESULTS AND DISCUSSION

#### Identification of the *Rhodococcus* sp. 19070 *bopXYZ* genes and gene products.
*Rhodococcus* sp. strain 19070 can grow on benzoate and substituted benzoates, as well as other aromatics such as toluene and xylenes that might be metabolized via benzoate or methylated benzoates. This ability suggested that strain 19070 would possess at least one aromatic ring-hydroxylating dioxygenase similar to the toluate dioxygenase encoded by the *P. putida xylXYZ* genes. Consistent with this possibility, the *xyl* genes were found to hybridize to a 2.3-kb *EcoRI* restriction fragment of *Rhodococcus* DNA on plasmid pSAM1. DNA sequence analysis revealed a complete open reading frame (ORF) on pSAM1 with significant similarity to *xylY* and two incomplete ORFs which resembled *xylX* and *xylZ*. Labeled DNA from pSAM1 was then used to isolate the adjacent regions of the *Rhodococcus* genome. Sequence determination of a 5.8-kb region, encompassing the *EcoRI* fragment of pSAM1, revealed three complete ORFs, designated *bopXYZ*, that seemed likely to encode a dioxygenase.

The deduced sequences of BopXY were approximately 60% identical to XylXY and BenAB, which are the alpha and beta

#### Figure 1: SDS-PAGE of proteins from plasmid-containing *E. coli* strain DH5α.
- Plasmid pSAM3 encodes the entire `bopZ` gene.
- pSAM4 has a deletion in its `bopZ` allele.
- The BopZ protein and the sizes of protein standards (in kilodaltons) are indicated.

![Figure 1](#fig1)

---

#### Subunits of the terminal oxygenase components of two-component benzoate dioxygenases:
- Sequence identity was also significant, i.e., approximately 40 to 55%, between BopXY and AntAB or CbdAB.
- The latter proteins comprise the alpha and beta subunits of the terminal oxygenase of a two-component anthranilate (2-aminobenzoate) dioxygenase or a chlorobenzoate dioxygenase (3, 16).

In each case, the second component of the two-component dioxygenase is a reductase that transfers electrons from NADH to the terminal oxygenase.
- The XylZ, BenC, and AntC reductases are all similar in size, i.e., ca. 337 amino acid residues (39 kDa), and all share a common evolutionary ancestor (11).
- The `bopZ` ORF was considerably longer than all of its counterparts, and it was predicted to encode a 60-kDa protein with an additional 201 amino acid residues beyond that corresponding to the C terminus of XylZ.
- The first 337 residues of the deduced BopZ sequence were approximately 50% identical to the entire XylZ and BenC sequences.
- In contrast, homology searches using the BopZ C-terminal region did not identify any significant similarity to sequences in current databases.

To investigate the Bop protein sizes, cell extracts of *E. coli* cultures carrying recombinant plasmids were analyzed by SDS-PAGE.
- DNA sequencing predicted that plasmid pSAM2, with complete *Rhodococcus bopXY* genes, carried only part of the `bopZ` gene.
- This portion of `bopZ`, however, would have been sufficient to encode a reductase that was the size of `xylZ` or `benC`.
- Consistent with the DNA sequence analysis, *E. coli* cells carrying pSAM2 produced high levels of proteins of the expected sizes for BopX (56 kDa) and BopY (20 kDa), but not BopZ (60 kDa) (data not shown).
- Moreover, in the size range expected for a XylZ-type of reductase (39 kDa), there were no notable differences in the protein profiles of cells that did or did not carry pSAM2 (data not shown).

To enable BopZ synthesis, a plasmid (pSAM3) with the entire coding region was constructed.
- A second plasmid, pSAM4, was derived from pSAM3 by deleting the central portion of `bopZ`.
- *E. coli* cells carrying pSAM3, but not pSAM4, had high levels of an approximately 65-kDa protein (Fig. 1).
- Additional controls with different plasmids and plasmid-free cells confirmed that the 65-kDa protein was correlated with the presence of an intact `bopZ` gene (data not shown).
- Therefore, although the apparent size was slightly larger than predicted, the protein was inferred to be BopZ.
- The amount of BopZ relative to other cellular proteins was variable, however.
- It appeared that expression or stability of this protein in *E. coli* was sensitive to slight variations in experimental conditions.

#### Function of `bopXY` genes expressed in *E. coli*:
To test whether the BopXY proteins expressed from pSAM2 conferred any aromatic ring hydroxylating abilities to *E. coli*, HPLC methods were used to monitor metabolite transformations.
- *E. coli* does not have known genes or enzyme activities that correspond to those of benzoate dioxygenases.
- *E. coli* cultures that contained pSAM2, no plasmid, or a plasmid vector with no heterologous DNA were provided with 1 mM amounts of either benzoate, *m*-toluate, anthranilate, or *o*-chlorobenzoate as a substrate for hydroxylation.
- After incubation for variable amounts of time ranging from 1 to 18 h, whole cells were removed, and the amount of aromatic acid remaining in the medium was assessed.
- Only in *E. coli* cultures containing pSAM2 did the concentration of substrates decrease over time.

*E. coli* expressing `bopXY` quantitatively converted 1 mM benzoate to the cis-diol, 2-hydro-1,2-dihydroxybenzoate, in approximately 5 h (Fig. 2). This cis-diol is the product of benzoate.

---

#### Figure 2: Metabolites in the culture medium of DH5α(pSAM2) provided with benzoate, *m*-toluate, *o*-chlorobenzoate, or anthranilate (1 mM each).
- (A) Conversion of benzoate (■) to cis-diol (□).
- (B) Consumption of *m*-toluate (◆) or *o*-chlorobenzoate (▲).
- (C) Conversion of anthranilate (●) to catechol (○).
- Compounds were identified by HPLC analysis.

![Figure 2](#fig2)

---

### Final Markdown Output:


### Figure 1: SDS-PAGE of proteins from plasmid-containing *E. coli* strain DH5α.
- Plasmid pSAM3 encodes the entire `bopZ` gene.
- pSAM4 has a deletion in its `bopZ` allele.
- The BopZ protein and the sizes of protein standards (in kilodaltons) are indicated.

![Figure 1](#fig1)

#### Subunits of the terminal oxygenase components of two-component benzoate dioxygenases:
- Sequence identity was also significant, i.e., approximately 40 to 55%, between BopXY and AntAB or CbdAB.
- The latter proteins comprise the alpha and beta subunits of the terminal oxygenase of a two-component anthranilate (2-aminobenzoate) dioxygenase or a chlorobenzoate dioxygenase (3, 16).

In each case, the second component of the two-component dioxygenase is a reductase that transfers electrons from NADH to the terminal oxygenase.
- The XylZ, BenC, and AntC reductases are all similar in size, i.e., ca. 337 amino acid residues (39 kDa), and all share a common evolutionary ancestor (11).
- The `bopZ` ORF was considerably longer than all of its counterparts, and it was predicted to encode a 60-kDa protein with an additional 201 amino acid residues beyond that corresponding to the C terminus of XylZ.
- The first 337 residues of the deduced BopZ sequence were approximately 50% identical to the entire XylZ and BenC sequences.
- In contrast, homology searches using the BopZ C-terminal region did not identify any significant similarity to sequences in current databases.

To investigate the Bop protein sizes, cell extracts of *E. coli* cultures carrying recombinant plasmids were analyzed by SDS-PAGE.
- DNA sequencing predicted that plasmid pSAM2, with complete *Rhodococcus bopXY* genes, carried only part of the `bopZ` gene.
- This portion of `bopZ`, however, would have been sufficient to encode a reductase that was the size of `xylZ` or `benC`.
- Consistent with the DNA sequence analysis, *E. coli* cells carrying pSAM2 produced high levels of proteins of the expected sizes for BopX (56 kDa) and BopY (20 kDa), but not BopZ (60 kDa) (data not shown).
- Moreover, in the size range expected for a XylZ-type of reductase (39 kDa), there were no notable differences in the protein profiles of cells that did or did not carry pSAM2 (data not shown).

To enable BopZ synthesis, a plasmid (pSAM3) with the entire coding region was constructed.
- A second plasmid, pSAM4, was derived from pSAM3 by deleting the central portion of `bopZ`.
- *E. coli* cells carrying pSAM3, but not pSAM4, had high levels of an approximately 65-kDa protein (Fig. 1).
- Additional controls with different plasmids and plasmid-free cells confirmed that the 65-kDa protein was correlated with the presence of an intact `bopZ` gene (data not shown).
- Therefore, although the apparent size was slightly larger than predicted, the protein was inferred to be BopZ.
- The amount of BopZ relative to other cellular proteins was variable, however.
- It appeared that expression or stability of this protein in *E. coli* was sensitive to slight variations in experimental conditions.

#### Function of `bopXY` genes expressed in *E. coli*:
To test whether the BopXY proteins expressed from pSAM2 conferred any aromatic ring hydroxylating abilities to *E. coli*, HPLC methods were used to monitor metabolite transformations.
- *E. coli* does not have known genes or enzyme activities that correspond to those of benzoate dioxygenases.
- *E. coli* cultures that contained pSAM2, no plasmid, or a plasmid vector with no heterologous DNA were provided with 1 mM amounts of either benzoate, *m*-toluate, anthranilate, or *o*-chlorobenzoate as a substrate for hydroxylation.
- After incubation for variable amounts of time ranging from 1 to 18 h, whole cells were removed, and the amount of aromatic acid remaining in the medium was assessed.
- Only in *E. coli* cultures containing pSAM2 did the concentration of substrates decrease over time.

*E. coli* expressing `bopXY` quantitatively converted 1 mM benzoate to the cis-diol, 2-hydro-1,2-dihydroxybenzoate, in approximately 5 h (Fig. 2). This cis-diol is the product of benzoate.

---

### Figure 2: Metabolites in the culture medium of DH5α(pSAM2) provided with benzoate, *m*-toluate, *o*-chlorobenzoate, or anthranilate (1 mM each).
- (A) Conversion of benzoate (■) to cis-diol (□).
- (B) Consumption of *m*-toluate (◆) or *o*-chlorobenzoate (▲).
- (C) Conversion of anthranilate (●) to catechol (○).
- Compounds were identified by HPLC analysis.

![Figure 2](#fig2)

#### Figure 3: Organization of the `bop`, `ben`, and `xyl` Genes

**Arrows indicate the direction of transcription. Gene products are known or are predicted to be:**
- The α and β subunits of a terminal dioxygenase (solid black and horizontal lines, respectively),
- The reductase components of a two-component dioxygenase (dots),
- cis-diol dehydrogenases (vertical lines),
- Membrane proteins involved in aromatic compound transport (white),
- A transcriptional regulator (diagonal lines).

---

### Text Content

#### Dihydroxylation by Either Benzoate 1,2- or Toluate 1,2-Dioxygenases

- *E. coli* (pIB1354) with the `benABC` genes of *Acinetobacter* sp. strain ADP1 were also able to carry out this transformation in a similar amount of time (34).
- Whole cells of *Acinetobacter* sp. strain ADP1 were able to remove benzoate from the medium, but the cis-diol did not accumulate, presumably because this bacterium is able to mineralize benzoate completely (data not shown).
- When *m*-toluate was used as a substrate, *E. coli* cultures expressing the `bopXY` genes were able to remove the substrate at a rate similar to that of benzoate (Fig. 2). The disappearance of the *m*-toluate peak on the HPLC chromatographs corresponded over time to the broadening of a different peak with a retention time of 4.4 min.
- The transformation product was not identified, because this compound was not clearly resolved under the experimental conditions that were used.

#### Expression of `bopXY` in *E. coli*

- Expression of `bopXY` in *E. coli* enabled the removal of anthranilate, albeit at a rate slightly slower than for benzoate or *m*-toluate (Fig. 2).
- Under the same conditions, *E. coli* expressing the *Acinetobacter* `benABC` genes did not transform anthranilate to catechol (data not shown).
- Most likely, the dihydroxylation of anthranilate by BopXY produces a cis-diol that spontaneously deaminates and decarboxylates to yield catechol.
- The dihydroxylation of benzoate or toluate produces a more stable cis-diol that requires the action of a dehydrogenase to form catechol (3).
- With *o*-chlorobenzoate as substrate, the `bopXY` genes enabled a small amount of the substrate (0.2 mM) to be removed from the medium during a relatively long 18-h incubation period.

#### Function of `bopZ` Expressed in *E. coli*

- To confirm that the `bopZ` gene of pSAM2 was incomplete, cytochrome *c* reductase activity was measured.
- Previously characterized reductases can transfer electrons from NADH to cytochrome *c* in the absence of their terminal oxygenases (49).
- This activity was measured in extracts of plasmid-free *E. coli* and *E. coli* carrying pSAM2, pUC19, or pSAM4 (with a deletion in the `bopZ` allele).
- The specific activities of all of these samples were comparable (≤20 ± 5 nmol/min per mg of protein).
- Thus, the truncated `bopZ` on pSAM2 was not sufficient to encode a functional NADH-dependent cytochrome *c* reductase in *E. coli*.
- In contrast, either the entire `bopZ` gene (pSAM3) or `benC` (pIB1354) increased the reductase activity in *E. coli* (100 ± 11 or 600 ± 76 nmol/min per mg of protein, respectively).
- The unusually large BopZ, therefore, appeared to have an activity similar to that of the smaller XylZ and BenC proteins.

#### Organization of the `bop` Gene Cluster and Putative Functions of Adjacent Genes

- Sequence analyses of genes in the vicinity of `bopXYZ` were consistent with the likelihood that they are all involved in benzoate catabolism (Fig. 3).
- The deduced amino acid sequence of the ORF immediately downstream of `bopXYZ`, designated `bopL`, was approximately 60% identical to that of XylL or BenD.
- The latter proteins are cis-diol dehydrogenases that use as their substrates the products of XylXYZ- or BenABC-catalyzed reactions, respectively.
- By analogy, BopL may be a dehydrogenase that converts the products of BopXYZ-catalyzed reactions to catechol or substituted catechols.
- The ORF immediately downstream of `bopL` was designated `bopK` based on its sequence similarity to `benK`.

---

### Notes

- The figure shows the organization of `bop`, `ben`, and `xyl` genes from various bacterial strains, indicating the direction of transcription and the putative functions of the gene products.
- The text discusses the roles of these genes in benzoate and toluate degradation, focusing on the function of `bopZ` and the organization of the `bop` gene cluster. 

#### Figure 4: Catechol Dioxygenase Activity of Rhodococcus sp. Strain 19070

**Figure Description:**
- **Panel A:** Catechol 1,2-dioxygenase activity was assayed spectrophotometrically at 260 nm with the ring cleavage product inferred to be cis,cis-muconate.
- **Panel B:** Catechol 2,3-dioxygenase activity was assayed spectrophotometrically at 375 nm with the ring cleavage product inferred to be 2-hydroxymuconic semialdehyde.
- **White bars:** Indicate the activity in samples that were heat-treated to inactivate catechol 1,2-dioxygenase.
- **Cross-hatched bars:** Indicate activity in samples that were treated with hydrogen peroxide to inactivate catechol 2,3-dioxygenase.

**Substrate Specificity:**
- **Benzoate, Toluene, m-Toluate, Glucose:** Used as the sole carbon source for growth.

---

#### Figure 5: PFGE of Genomic DNA from Rhodococcus sp. Strain 19070

**Panel A:**
- DNA was either uncut (lane 1) or digested with SspI (lane 2) or XbaI (lane 3).
- The sizes of DNA markers are indicated adjacent to the corresponding DNA (lambda ladder from New England Biolabs) in lane M.

**Panel B:**
- Results of Southern hybridization of gel in panel A with a `bopXY` probe.

---

### Text Content

#### Catabolic Pathways for Benzoate Degradation in Rhodococcus sp. Strain 19070

One difference between the *xyl*- and *ben*-encoded catabolic pathways in pseudomonads is that catechols formed from *xyl*-encoded enzymes are cleaved by an extradiol-cleaving catechol dioxygenase (encoded by *xylE*). Catabolites are then channeled through the meta-cleavage pathway (22). In contrast, catechol formed by the *ben*-encoded enzymes is cleaved by an intradiol-cleaving catechol dioxygenase (encoded by *catA*). Catabolites are channeled through the ortho-cleavage pathway, also known as the β-ketoadipate pathway (23).

The *xylE* gene of the TOL plasmid pWW0 is downstream of *xylXYZL* and is coexpressed with them (44). The *catA* genes of *P. aeruginosa* PAO1 and *Acinetobacter* sp. strain ADP1 are located near the *benABCD* genes (33, 52). In *Rhodococcus* sp. strain 19070, partial DNA sequencing in the regions near the *bop* genes did not reveal the presence of a gene encoding either an intradiol or extradiol catechol dioxygenase.

The activities of catechol 1,2-dioxygenase (intradiol) and catechol 2,3-dioxygenase (extradiol) can be distinguished by the distinct absorbance patterns of their ring cleavage products. Furthermore, in previous studies, H₂O₂ has been shown to inactivate catechol 2,3-dioxygenase, whereas heat treatment inactivates catechol 1,2-dioxygenase (31). The activities of both enzymes were measured in cell extracts of *Rhodococcus* sp. strain 19070 grown on benzoate, toluene, m-toluate, or glucose as the sole carbon source. For each carbon source, the activity patterns in cell extracts were compared. As shown in Fig. 4, catechol 1,2-dioxygenase was induced by growth on benzoate, whereas catechol 2,3-dioxygenase was induced by growth on toluene or m-toluate. Similar results are observed for pseudomonads, in which the *xylXYZ*-encoded dioxygenase participates in toluene catabolism and the *benABC*-encoded dioxygenase participates in benzoate catabolism. Further work is needed to establish under which growth conditions the *bop* genes of *Rhodococcus* sp. strain 19070 are expressed. Nevertheless, the presence of additional dioxygenase genes cannot be precluded.

---

### Additional Notes

- **Acinetobacter sp. Strain ADP1:** BenK is a benzoate transporter that is encoded by a gene upstream of the operon containing the *benABCD* genes (Fig. 3) (5). A recently identified *benK* homolog in *P. putida* PRS2000 lies immediately downstream of the *benABCD* genes (6) in the same arrangement as the *bop* genes (Fig. 3). No *benK* homolog, however, has been identified near the *xylXYZL* genes.
  
- **Similarity of *bop* and *ben* Clusters:** The similarity of the *bop* and *ben* clusters raised the possibility that *Rhodococcus* sp. strain 19070 might have a second genetic region, one corresponding more closely to the *xyl* genes, that encodes a benzoate dioxygenase distinct from *BopXYZ*. To test this possibility, a variety of restriction enzymes, including BglII, EcoRI, PstI, SacI, SalI, and SphI, were used to digest *Rhodococcus* DNA for Southern hybridizations. A labeled *bopXY* probe, with the same conditions that allow the *Pseudomonas xyl* genes to hybridize to the *bop* genes, did not detect DNA fragments other than those of the sizes expected for the *bop* genes (data not shown). Nevertheless, the presence of additional dioxygenase genes cannot be precluded.

---

### Final Markdown Output


# Figure 4: Catechol Dioxygenase Activity of Rhodococcus sp. Strain 19070

**Figure Description:**
- **Panel A:** Catechol 1,2-dioxygenase activity was assayed spectrophotometrically at 260 nm with the ring cleavage product inferred to be cis,cis-muconate.
- **Panel B:** Catechol 2,3-dioxygenase activity was assayed spectrophotometrically at 375 nm with the ring cleavage product inferred to be 2-hydroxymuconic semialdehyde.
- **White bars:** Indicate the activity in samples that were heat-treated to inactivate catechol 1,2-dioxygenase.
- **Cross-hatched bars:** Indicate activity in samples that were treated with hydrogen peroxide to inactivate catechol 2,3-dioxygenase.

**Substrate Specificity:**
- **Benzoate, Toluene, m-Toluate, Glucose:** Used as the sole carbon source for growth.

---

# Figure 5: PFGE of Genomic DNA from Rhodococcus sp. Strain 19070

**Panel A:**
- DNA was either uncut (lane 1) or digested with SspI (lane 2) or XbaI (lane 3).
- The sizes of DNA markers are indicated adjacent to the corresponding DNA (lambda ladder from New England Biolabs) in lane M.

**Panel B:**
- Results of Southern hybridization of gel in panel A with a `bopXY` probe.

---

## Catabolic Pathways for Benzoate Degradation in Rhodococcus sp. Strain 19070

One difference between the *xyl*- and *ben*-encoded catabolic pathways in pseudomonads is that catechols formed from *xyl*-encoded enzymes are cleaved by an extradiol-cleaving catechol dioxygenase (encoded by *xylE*). Catabolites are then channeled through the meta-cleavage pathway (22). In contrast, catechol formed by the *ben*-encoded enzymes is cleaved by an intradiol-cleaving catechol dioxygenase (encoded by *catA*). Catabolites are channeled through the ortho-cleavage pathway, also known as the β-ketoadipate pathway (23).

The *xylE* gene of the TOL plasmid pWW0 is downstream of *xylXYZL* and is coexpressed with them (44). The *catA* genes of *P. aeruginosa* PAO1 and *Acinetobacter* sp. strain ADP1 are located near the *benABCD* genes (33, 52). In *Rhodococcus* sp. strain 19070, partial DNA sequencing in the regions near the *bop* genes did not reveal the presence of a gene encoding either an intradiol or extradiol catechol dioxygenase.

The activities of catechol 1,2-dioxygenase (intradiol) and catechol 2,3-dioxygenase (extradiol) can be distinguished by the distinct absorbance patterns of their ring cleavage products. Furthermore, in previous studies, H₂O₂ has been shown to inactivate catechol 2,3-dioxygenase, whereas heat treatment inactivates catechol 1,2-dioxygenase (31). The activities of both enzymes were measured in cell extracts of *Rhodococcus* sp. strain 19070 grown on benzoate, toluene, m-toluate, or glucose as the sole carbon source. For each carbon source, the activity patterns in cell extracts were compared. As shown in Fig. 4, catechol 1,2-dioxygenase was induced by growth on benzoate, whereas catechol 2,3-dioxygenase was induced by growth on toluene or m-toluate. Similar results are observed for pseudomonads, in which the *xylXYZ*-encoded dioxygenase participates in toluene catabolism and the *benABC*-encoded dioxygenase participates in benzoate catabolism. Further work is needed to establish under which growth conditions the *bop* genes of *Rhodococcus* sp. strain 19070 are expressed. Nevertheless, the presence of additional dioxygenase genes cannot be precluded.

---

### Additional Notes

- **Acinetobacter sp. Strain ADP1:** BenK is a benzoate transporter that is encoded by a gene upstream of the operon containing the *benABCD* genes (Fig. 3) (5). A recently identified *benK* homolog in *P. putida* PRS2000 lies immediately downstream of the *benABCD* genes (6) in the same arrangement as the *bop* genes (Fig. 3). No *benK* homolog, however, has been identified near the *xylXYZL* genes.
  
- **Similarity of *bop* and *ben* Clusters:** The similarity of the *bop* and *ben* clusters raised the possibility that *Rhodococcus* sp. strain 19070 might have a second genetic region, one corresponding more closely to the *xyl* genes, that encodes a benzoate dioxygenase distinct from *BopXYZ*. To test this possibility, a variety of restriction enzymes, including BglII, EcoRI, PstI, SacI, SalI, and SphI, were used to digest *Rhodococcus* DNA for Southern hybridizations. A labeled *bopXY* probe, with the same conditions that allow the *Pseudomonas xyl* genes to hybridize to the *bop* genes, did not detect DNA fragments other than those of the sizes expected for the *bop* genes (data not shown). Nevertheless, the presence of additional dioxygenase genes cannot be precluded.

---

**FIG. 6. Phylogenetic trees based on comparisons with homologs of the Rhodococcus sp. strain 19070 Bop proteins. (A) BopX was aligned with alpha components of oxygenase subunits. (B) BopY was aligned with the corresponding beta components of oxygenases from Panel A. (C) BopZ was aligned with available reductase components associated with oxygenases from panel A and other reductases or putative reductases. (D) The putative BopL dehydrogenase was aligned with dehydrogenases and putative dehydrogenases associated with the proteins displayed in trees A to C, as well as ORF4 (Ro). (E) BopK, a putative transport protein, was aligned with proteins that may be involved in transporting organic compounds. Gene clusters are associated with metabolism and/or transport of the indicated compounds: BopXYZLK, Rhodococcus sp. ATCC 19070, benzoate-toluene (AF279141), BenABCDK(Pp), P. putida strain PRS200, benzoate (AF218267), BenABCDK(Ac), Acinetobacter sp. strain ADP1, benzoate (AF009224), XylXYZ(pDK1), P. putida sp. plasmid pDK1, toluate (AF134348), XylXYZL(TOL), P. putida sp. TOL plasmid, benzoate (M64747), CbdABC, Burkholderia cepacia, halobenzoate (X79076), AntABC, Acinetobacter sp. strain ADP1, anthranilate (AF071556), TftAB, B. cepacia, 2,4,5-trichlorophenoxycetic acid (U11420), AtdAB, Acinetobacter sp. plasmid pYA1, aniline (D86080), TdnA1B1, P. putida strain UCC22 (pTDN1) F1, aniline (D85415), CmtAbAcB, P. putida p-cymene (U24215), NidAB, Rhodococcus sp. strain I24, indene (AF121905), NarAaAbB, Rhodococcus sp. strain NCIMB12038, naphthalene (AF082663), BphA1A2B, Rhodococcus sp. strain RHA1, biphenyl (D32142), NahAcAdAa, P. putida strain G7, naphthalene (M83949), BpdC1C2B, Rhodococcus sp. strain M5, biphenyl-chlorobiphenyl (U27591), TodC1C2D, P. putida sp. strain F1, toluene (J04996), ORF6, A. calcoaceticus strain NCIB8250, phenol (Z36909), XylA, P. putida sp. TOL plasmid, xylene (M37480), PheA6, P. putida sp. strain P35X (NCIB9869), phenol (X79063), TbmF, Pseudomonas sp. strain JS150, toluene-benzene (L40033), PahAB, P. aeruginosa strain PaK1, naphthalene (D84146), ORF4(Ro), Rhodococcus opacus sp. strain 1CP putative short-chain dehydrogenase (AF030176), PcaK(Ac), Acinetobacter sp. strain ADP1, protocatechuate transporter (L05770), ORF4(Sg), Streptomyces griseus, putative tyrosine transporter (AB022095), PcaK(Pp)**

---

**Nevertheless, this induction of catechol 1,2-and 2,3-dioxygenases indicates the presence of both ortho- and meta-cleavage pathways.**

**PFGE analysis of Rhodococcus sp. strain 19070. In P. putida mt-2, xylXYZL are on the TOL plasmid pWW0, while benABCD are on the chromosome. PFGE analysis of undigested genomic DNA of Rhodococcus sp. strain 19070 demonstrated that no DNA was visible below 700 kb (Fig. 5A, lane 1) or hybridized to a labeled bopXY probe (Fig. 5B, lane 1). Thus, it appeared that the bop genes are either chromosomal or on a very large plasmid. The bopXY probe hybridized to distinct fragments of circa 640 or 190 kb in DNA digested with SspI or XbaI, respectively (Fig. 5B, lanes 2 and 3). These hybridization patterns and the clear separation of molecular weight standards in the range of 50 to 730 kb suggest that the presence of the bop genes on a small plasmid could have been detected.**

**Phylogeny of Rhodococcus BopXYZLK. The Bop protein sequences were aligned with those found by database searches to be most similar. Phylogenetic trees constructed from these alignments indicated that each of the BopXYZ proteins, from a gram-positive bacterium, was closely related to the corresponding component of the benzoate (Ben) and toluate (Xyl) dioxygenases from gram-negative bacteria (Fig. 6A, B, and C). Moreover, the close relationships among BopL, XylL, and BenD (Fig. 6D) and between BopK and BenK (Fig. 6E) support a role for the Bop proteins in benzoate degradation.**

**The presence of a benK-like gene within the bop gene cluster might indicate that BopXYZ is more similar to BenABC than XylXYZ. However, BopXYZ in E. coli, unlike BenABC of ADP1, was able to hydroxylate anthranilate. The rate of anthranilate hydroxylation was reduced relative to benzoate or m-toluene as a substrate, and the phylogenetic relationship to AntABC was more distant than to the Ben or Xyl proteins (Fig. 6A, B, and C). Therefore, while the primary role for BopXYZ is most likely not anthranilate catabolism, the substrate specificity of BopXYZ appears to be broader than that of BenABC. Further work may clarify whether BopXYZL converts benzoate to catechol for degradation via an ortho-cleavage pathway or whether BopXYZL converts toluates to methyl-catechols for degradation via a meta-cleavage pathway. An intriguing possibility, which remains to be investigated, is that it could carry out both functions.**

**The most unusual feature of the BopXYZ dioxygenase was the large size of BopZ. Whereas approximately two-thirds of the BopZ protein was closely related to the entire BenC and XylZ reductases (Fig. 6C), the C-terminal region of the protein did not resemble any known sequences. The ability of the BopXY terminal oxygenase to function without BopZ in E. coli is consistent with oxygenase components of aromatic-ring-hydroxylating dioxygenases having relaxed requirements for specific reductases.**

**P. putida sp. strain PRS2000, protocatechuate transporter (U10895), HppK, Rhodococcus globerulus sp. strain PWD1, putative 3-hydroxyphenyl propionate transporter (U89712), FcbT, Arthrobacter sp. strain TM1, 4-chlorobenzoate transporter (AF042490), MucK, Acinetobacter sp. strain ADP1, cis,cis-muconate transporter (U87258), VanK, Acinetobacter sp. strain ADP1, vanillate transporter (AF009672). Accession numbers are indicated parenthetically. Circles represent branch points that occur with a frequency of 85 to 100%, respectively, as calculated by bootstrap analysis using 100 replicates.**

---

### ACKNOWLEDGMENTS

We thank A. Reams for assistance with PFGE analysis. S. Haddad also gratefully acknowledges the support of J. W. Bennett and W. A. Toscano, Jr.

This work was supported by the National Science Foundation grant MCB-9808784 (to E.L.N.) and postdoctoral research fellowship in microbial biology DBI-0074398 (to S.H.).

### REFERENCES

1. Andreoni, V., S. Bernasconi, M. Colombo, J. B. van Beilen, and L. Cavalca. 2000. Detection of genes for alkane and naphthalene catabolism in *Rhodococcus* sp. strain 1BN. Environ. Microbiol. **2**:572–577.
2. Bradford, M. M. 1976. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal Biochem. **72**:248–254.
3. Bundy, B. M., A. L. Campbell, and E. L. Neidle. 1998. Similarities of the *antABC*-encoded anthranilate dioxygenase and the *benABC*-encoded benzoate dioxygenase of *Acinetobacter* sp. strain ADP1. J. Bacteriol. **180**:4466–4474.
4. Casadaban, M. J., J. Chou, and S. N. Cohen. 1980. In vitro gene fusions that join an enzymatically active beta-galactosidase segment to amino-terminal fragments of exogenous proteins: *Escherichia coli* plasmid vectors for the detection and cloning of translational initiation signals. J. Bacteriol. **143**:971–980.
5. Collier, L. S., N. N. Nichols, and E. L. Neidle. 1997. *benK* Encodes a hydrophobic permease-like protein involved in benzoate degradation by *Acinetobacter* sp. strain ADP1. J. Bacteriol. **179**:5943–5946.
6. Cowles, C. E., N. N. Nichols, and C. S. Harwood. 2000. BenR, a XylS homologue, regulates three different pathways of aromatic acid degradation in *Pseudomonas putida*. J. Bacteriol. **182**:6339–6346.
7. De Schrijver, A., and R. De Mot. 1999. Degradation of pesticides by actinomycetes. Crit. Rev. Microbiol. **25**:85–119.
8. Deeb, R. A., and L. Alvarez-Cohen. 1999. Temperature effects and substrate interactions during the aerobic biotransformation of BTEX mixtures by toluene-enriched consortia and *Rhodococcus rhodochrous*. Biotechnol. Bioeng. **62**:526–536.
9. Dente, L., G. Cesarini, and R. Cortese. 1983. pEMBL: a new family of single-stranded plasmids. Nucleic Acids Res. **11**:1645–1655.
10. Devereux, J., P. Haeberli, and O. Smithies. 1984. A comprehensive set of sequence analysis programs for the VAX. Nucleic Acids Res. **12**(Pt 1):387–395.
11. Eby, D. M., Z. M. Beharry, E. D. Coulter, D. M. Kurtz, and E. L. Neidle. 2001. Characterization and evolution of anthranilate 1,2-dioxygenase from *Acinetobacter* sp. strain ADP1. J. Bacteriol. **183**:109–118.
12. Eulberg, D., L. A. Golovleva, and M. Schlömann. 1997. Characterization of catechol catabolic genes from *Rhodococcus erythropolis* 1CP. J. Bacteriol. **179**:370–381.
13. Felsenstein, J. 1989. PHYLIP-Phylogeny inference package (version 3.2). Cladistics **5**:164–166.
14. Franklin, F. C. H., M. Bagdasarian, M. M. Bagdasarian, and K. N. Timmis. 1981. Molecular and functional analysis of the TOL plasmid pWW0 from *Pseudomonas putida* and cloning of genes for the entire regulated aromatic ring meta cleavage pathway. Proc. Natl. Acad. Sci. USA **78**:7458–7462.
15. Gralton, E. M., A. L. Campbell, and E. L. Neidle. 1997. Directed introduction of DNA cleavage sites to produce a high-resolution genetic and physical map of the *Acinetobacter* sp. strain ADP1 (BD413UE) chromosome. Microbiology **143**:1345–1357.
16. Haak, B., S. Fetzner, and F. Lingens. 1995. Cloning, nucleotide sequence, and expression of the plasmid-encoded genes for the two-component 2-halo-benzoate 1,2-dioxygenase from *Pseudomonas cepacia* 2CBS. J. Bacteriol. **177**:667–675.
17. Haddad, S. 1998. Ph.D. thesis. Tulane University, New Orleans, La.
18. Hanne, L. F., L. L. Kirk, S. M. Appel, A. D. Narayan, and K. K. Bains. 1993. Degradation and induction specificity in actinomycetes that degrade *p*-nitrophenol. Appl. Environ. Microbiol. **59**:3505–3508.
19. Harayama, S., P. R. Lerhbach, and K. N. Timmis. 1984. Transposon mutagenesis analysis of *meta*-cleavage pathway operon genes of the TOL plasmid of *Pseudomonas putida* mt-2. J. Bacteriol. **160**:251–255.
20. Harayama, S., M. Rekik, A. Bairoch, E. L. Neidle, and L. N. Ornston. 1991. Potential DNA slippage structures acquired during evolutionary divergence of *Acinetobacter calcoaceticus* chromosomal *benABC* and *Pseudomonas putida* TOL pWW0 plasmid *xylXYZ*, genes encoding benzoate dioxygenases. J. Bacteriol. **173**:7540–7548.
21. Harayama, S., M. Rekik, and K. N. Timmis. 1986. Genetic analysis of a relaxed substrate specificity aromatic ring dioxygenase, toluate 1,2-dioxygenase, encoded by TOL plasmid pWW0 of *Pseudomonas putida*. Mol. Gen. Genet. **202**:226–234.
22. Harayama, S., and K. N. Timmis. 1989. Catabolism of aromatic hydrocarbons by *Pseudomonas*, p. 151–174. In D. A. Hopwood and K. Chater (ed.), Genetics of bacterial diversity. Academic Press, Inc., New York, N.Y.
23. Harwood, C. S., and R. E. Parales. 1996. The B-ketoadipate pathway and the biology of self-identity. Annu. Rev. Microbiol. **50**:553–590.
24. Hopwood, D. A., M. J. Bibb, K. F. Chater, T. Kiesser, C. J. Bruton, H. M. Kiesser, D. J. Lydiate, C. P. Smith, J. M. Ward, and H. Schrempf. 1985. Genetic manipulation of *Streptomyces*—a laboratory manual. The John Innes Foundation, Norwich, United Kingdom.
25. Juni, E., and A. Janik. 1969. Transformation of *Acinetobacter calcoaceticus* (*Bacterium anitratum*). J. Bacteriol. **98**:281–288.
26. Kulakov, L. A., V. A. Delcroix, M. J. Larkin, V. N. Ksenzenko, and A. N. Kulakova. 1998. Cloning of new *Rhodococcus* extradiol dioxygenase genes and study of their distribution in different *Rhodococcus* strains. Microbiology **144**:955–963.
27. Lofgren, J., S. Haddad, and K. Kendall. 1995. Metabolism of alkanes by *Rhodococcus erythropolis*, p. 252–263. In D. W. Tedder and F. G. Pohland (ed.), Emerging technologies in hazardous waste management, vol. V. American Chemical Society, Washington, D.C.
28. Martin, V. J. J., and W. W. Mohn. 1999. A novel aromatic-ring-hydroxylating dioxygenase from the diterpenoid-degrading bacterium *Pseudomonas abietaniphila* BKME-9. J. Bacteriol. **181**:2675–2682.
29. McCarthy, A. J., and S. T. Williams. 1992. Actinomycetes as agents of biodegradation in the environment—a review. Gene **115**:189–192.
30. Murray, K., and P. A. Williams. 1974. Role of catechol and the methylcatechols as inducers of aromatic metabolism in *Pseudomonas putida*. J. Bacteriol. **117**:1153–1157.
31. Nakazawa, T., and T. Yokota. 1973. Benzoate metabolism in *Pseudomonas putida* (*arvilla*) mt-2: demonstration of two benzoate pathways. J. Bacteriol. **115**:262–267.
32. Neidle, E., C. Harnett, L. N. Ornston, A. Bairoch, M. Rekik, and S. Harayama. 1992. *cis*-Diol dehydrogenases encoded by the TOL pWW0 plasmid *xylL* gene and the *Acinetobacter calcoaceticus* chromosomal *benD* gene are members of the short-chain alcohol dehydrogenase superfamily. Eur. J. Biochem. **204**:113–120.
33. Neidle, E. L., C. Hartnett, S. Bonitz, and L. N. Ornston. 1988. DNA sequence of the *Acinetobacter calcoaceticus* catechol 1,2-dioxygenase I structural gene *catA*: evidence for evolutionary divergence of intradiol dioxygenases by acquisition of DNA sequence repetitions. J. Bacteriol. **170**:4874–4880.
34. Neidle, E. L., M. K. Shapiro, and L. N. Ornston. 1987. Cloning and expression in *Escherichia coli* of *Acinetobacter calcoaceticus* genes for benzoate degradation. J. Bacteriol. **169**:5496–5503.
35. Ngai, K.-L., E. L. Neidle, and L. N. Ornston. 1990. Catechol and chlorocatechol 1,2-dioxygenase. Methods Enzymol. **188**:122–126.
36. Reiner, A. M. 1971. Metabolism of benzoic acid by bacteria: 3,5-cyclohexadiene-1,2-diol-1-carboxylic acid is an intermediate in the formation of catechol. J. Bacteriol. **108**:89–94.
37. Reiner, A. M., and G. D. Hegeman. 1971. Metabolism of benzoic acid by bacteria. Accumulation of (−)-3,5-cyclohexadiene-1,2-diol-1-carboxylic acid by a mutant strain of *Alcaligenes eutrophus*. Biochemistry **10**:2530–2536.
38. Sala-Trepat, J. M., and W. C. Evans. 1971. The *meta* cleavage pathway of catechol by *Azobacter* species. Eur. J. Biochem. **20**:400–413.
39. Sambrook, J., E. F. Fritsch, and T. Maniatis. 1989. Molecular cloning: a laboratory manual, 2nd ed. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.
40. Sanger, F., S. Nicklen, and A. R. Coulson. 1977. DNA sequencing with chain-terminating inhibitors. Proc. Nat. Acad. Sci. USA **74**:5463–5467.
41. Schreiner, A., K. Fuchs, F. Lottspeich, H. Poth, and F. Lingens. 1991. Degradation of 2-methylaniline in *Rhodococcus rhodochrous*: cloning and expression of two clustered catechol 2,3-dioxygenase genes from strain CTM. J. Gen. Microbiol. **137**:2041–2048.
42. Shanley, M. S., E. L. Neidle, R. E. Parales, and L. N. Ornston. 1986. Cloning and Expression of *Acinetobacter calcoaceticus catBCDE* Genes in *Pseudomonas putida* and *Escherichia coli*. J. Bacteriol. **165**:557–563.
43. Smith, C. L., and C. R. Cantor. 1987. Purification, specific fragmentation and separation of large DNA molecules. Methods Enzymol. **155**:449–467.
44. Spooner, R. A., M. Bagdasarian, and F. C. H. Franklin. 1987. Activation of the *xylDLEGF* promoter of the TOL toluene-xylene degradation pathway by overproduction of the *xylS* regulatory gene product. J. Bacteriol. **169**:3581–3586.
45. Uz, I., Y. P. Duan, and A. Ogram. 2000. Characterization of the naphthalene-degrading bacterium, *Rhodococcus opacus* M213. FEMS Microbiol. Lett. **185**:231–238.
46. Vanderberg, L. A., R. Krieger-Grumbine, and M. N. Taylor. 2000. Evidence for diverse oxidations in the catabolism of toluene by *Rhodococcus rhodo*-

chrous strain OFS. Appl. Microbiol. Biotechnol. **53**:447–452.

47. Warhurst, A. M., and C. A. Fewson. 1994. Biotransformations catalyzed by the genus *Rhodococcus*. Crit. Rev. Biotech. **14**:29–73.

48. Williams, P. A., and K. Murray. 1974. Metabolism of benzoate and the methylbenzoates by *Pseudomonas putida (arvilla)* mt-2: evidence for the existence of a TOL plasmid. J. Bacteriol. **120**:416–423.

49. Yamaguchi, M., and H. Fujisawa. 1978. Characterization of NADH-cytochrome *c* reductase, a component of benzoate 1,2-dioxygenase system from *Pseudomonas arvilla* C-1. J. Biol. Chem. **253**:8848–8853.

50. Yanisch-Perron, C. J., J. Vieira, and J. Messing. 1985. Improved M13 cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. Gene **33**:103–119.

51. Zeyer, J., P. R. Lehrbach, and K. N. Timmis. 1985. Use of cloned genes of *Pseudomonas* TOL plasmid to effect biotransformation of benzoates to cis-dihydrodiols and catechols by *Escherichia coli* cells. Appl. Environ. Microbiol. **50**:1409–1413.

52. Zhang, C., M. Huang, and B. W. Holloway. 1993. Mapping of the *ben*, *ant* and *cat* genes of *Pseudomonas aeruginosa* and evolutionary relationship of the *ben* region of *P. aeruginosa* and *P. putida*. FEMS Microbiol. Lett. **108**:303–310.
