
Three-dimensional structure of cyanobacterial photosystem I at 2.5 Å resolution

Patrick Jordan*, Petra Fromme†, Horst Tobias Witt†, Olaf Klukas*, Wolfram Saenger* & Norbert Krauß*

*Institut für Chemie/Kristallographie, Freie Universität Berlin, D-14195 Berlin, Takustraße 6, Germany  
†Max Volmer Laboratorium für Biophysikalische Chemie, Institut für Chemie, Fakultät 2; Technische Universität Berlin, D-10623 Berlin, Straße des 17. Juni 135, Germany  

Life on Earth depends on photosynthesis, the conversion of light energy from the Sun to chemical energy. In plants, green algae and cyanobacteria, this process is driven by the cooperation of two large protein–cofactor complexes, photosystems I and II, which are located in the thylakoid photosynthetic membranes. The crystal structure of photosystem I from the thermophilic cyanobacterium *Synechococcus elongatus* described here provides a picture at atomic detail of 12 protein subunits and 127 cofactors comprising 96 chlorophylls, 2 phylloquinones, 3 Fe₄S₄ clusters, 22 carotenoids, 4 lipids, a putative Ca²⁺ ion and 201 water molecules. The structural information on the proteins and cofactors and their interactions provides a basis for understanding how the high efficiency of photosystem I in light capturing and electron transfer is achieved.

All higher organisms on earth receive their energy directly or indirectly from oxygenic photosynthesis performed by plants, green algae and cyanobacteria, the last two representing an important part of the marine food web. Photosynthesis converts solar energy to chemical energy by means of two large protein complexes located in the thylakoid membranes: photosystems I (PSI) and II (PSII). They use the energy of absorbed photons to translocate electrons across the membrane along a chain of cofactors, leading to a transmembrane difference in the electrical potential and in H⁺ concentration, which drives ATP synthesis and reduction of NADP⁺ to NADPH. In the subsequent dark reactions, NADPH and ATP are used to convert CO₂ to carbohydrates.

PSI and PSII belong to the superfamily of photosynthetic reaction centres, which are divided into two classes according to their terminal electron acceptors: Fe₄S₄ clusters (type I) or quinone (type II; for review see ref. 1). For purple bacterial reaction centres and PSII, both belonging to the type II reaction centres, three-dimensional structures are known at high and medium resolution, respectively²³. For type I reaction centres, a structural model of cyanobacterial PSI at medium resolution (4 Å) is available⁴⁵ but elucidation of the mechanism of light capture and electron transfer in PSI has been impeded by lack of more detailed structural information.

Isolated cyanobacterial PSI exists as a trimer⁶ (relative molecular mass Mr 3×356,000); this form is also present *in vivo*. One monomer consists of at least 11 different protein subunits coordinating more than 100 cofactors. PSI captures light energy by a large internal antenna system and guides it to the core of the reaction centre with high efficiency. After primary charge separation initiated by excitation of the chlorophyll dimer P700, the electron passes along the electron transfer chain (ETC) consisting of the spectroscopically identified cofactors A₀ (Chla), A₁ (phylloquinone) and the Fe₄S₄ clusters Fₓ, Fₐ and Fₔ (for review see ref. 7). At the stromal (cytoplasmic) side, the electron is donated by Fₔ to ferredoxin (or flavodoxin) and hence transferred to NADP⁺ reductase. The reaction cycle is completed by re-reduction of P700⁺• by cytochrome c₆ (or plastocyanin) at the inner (lumenal) side of the membrane. The electron carried by cytochrome c₆ is provided by PSII by way of a pool of plastoquinones and the cytochrome b₆/f complex.

To understand the role of the protein subunits in binding of the cofactors, and the interactions between the cofactors, we have determined the X-ray crystal structure of PSI from the thermophilic cyanobacterium S. *elongatus* and describe here the atomic model.

### Structure determination
The trimeric PSI, isolated from *S. elongatus*, was crystallized as described elsewhere⁸. Detailed information on X-ray data collection, evaluation and refinement at 2.5 Å resolution are provided in Table 1 and as Supplementary Information.

In the electron density map, the 11 biochemically characterized and sequenced protein subunits⁹ were identified. Ten of these were interpreted with the known amino-acid sequences, whereas the peripheral subunit PsaK could only be modelled as polyalanine, because the electron density becomes less well defined with increasing distance from the centre of the PSI trimer. A twelfth polypeptide chain containing one transmembrane α-helix could be fitted to well defined electron density. Its sequence is homologous to the controversial subunit PsaX identified in PSI from the thermophilic cyanobacteria *Synechococcus vulcanus* and *Anabaena variabilis*¹⁰,¹¹.

For the 96 chlorophylls, the positions and orientations of the chlorophyll head groups were determined, leading to the assignment of the orientation of the Qₓ and Qᵧ transition dipolar moments. For 48 of them the ring substituents, including the complete phytyl side chains, could be modelled into the electron density map, whereas for the remaining Chla molecules the phytyl chain could be modelled only partially. In addition, the three Fe₄S₄ clusters, two phylloquinones and for the first time 22 carotenoids (16 in all-trans, 5 in different cis configurations and one modelled as an incomplete β-carotene molecule), 4 lipids and a putative Ca²⁺ ion were identified. The total of 127 cofactors located in one PSI monomer contributes around 30% of the molecular mass of PSI.

### Overall architecture
Trimeric PSI resembles a clover leaf (Fig. 1a). Its threefold rotation axis coincides with the crystallographic C₃ axis (space group P6₃), which is perpendicular to the membrane plane (Fig. 1b). At the

‡Present address: Institut für Biochemie, Universitätsklinikum Charité, Medizinische Fakultät der Humboldt Universität zu Berlin, Monbijoustraße 2, D-10117 Berlin, Germany.

‘trimerization domain’ close to the C₃ axis, PsaL forms most of the contacts between the monomers.

PSI contains nine protein subunits featuring transmembrane α-helices (PsaA, PsaB, PsaF, PsaJ, PsaK, PsaL, PsaM and PsaX) and three stromal subunits (PsaC, PsaD and PsaE). The large (Mᵣ 83K) subunits PsaA and PsaB are related by a pseudo-C₂ axis located at the centre of the PSI monomer and oriented parallel to the C₃ axis. The organic cofactors of the ETC are arranged in two branches along the pseudo-C₂ axis, and most of the antenna Chla molecules, the carotenoids and the lipids are bound to PsaA/B. The other membrane-intrinsic subunits are peripheral to the PsaA/PsaB core and contribute to the coordination of antenna cofactors (Fig. 1a). The structure of PSI suggests a docking site for cytochrome c₆ or plastocyanin (donating electrons to PSI) at the luminal side close to

![Figure 1](image.png)

**Figure 1** Structural model of PS I trimer at 2.5 Å resolution. **a**, View along the membrane normal from the stromal side. For clarity, stromal subunits have been omitted. Different structural elements are shown in each of the three monomers (I, II and III). I, arrangement of the transmembrane α-helices (cylinders). Subunits are labelled. The transmembrane α-helices of PsaA (blue) and PsaB (red) are named A-a to A-k (B-a to B-k) from the N to the C terminus (capital letters omitted). All loop regions of PsaA and PsaB are named according to the transmembrane helices which they connect. For all other subunits the α-helices and β-sheets are numbered in alphabetical order from the N to the C terminus. Six helices in extra-membranous loop regions are drawn as spirals. II, membrane-intrinsic subunits. In addition to the transmembrane α-helices of the stromal and luminal loop regions are shown in ribbon representation. III, complete set of cofactors shown with the transmembrane α-helices (the side chains of the antenna Chla molecules have been omitted). ETC: quinones and chlorophylls in blue, iron and sulphur atoms of the three Fe₄S₄ clusters as orange and yellow spheres, respectively. Antenna system: chlorophylls in yellow, carotenoids in black, lipids in turquoise. **b**, Side view of the arrangement of all proteins in one monomer of PSI (colours as in **a**), including the stromal subunits PsaC (pink), PsaD (turquoise), PsaE (green) and the Fe₄S₄ clusters. View direction indicated by arrow at monomer II in **a**. The vertical line (right) shows the crystallographic C₃ axis. **c**, View as in **a** showing stromal subunits PsaC, PsaD and PsaE. They cover some of the loop regions and helices of PsaA and PsaB (light grey). Dashed ellipse: putative docking site of ferredoxin, covering loops of PsaA.

Table 1 Data collection, phasing and refinement statistics

| Dataset | Native (1) | Native (2) | EMTS | PIP |
|---------|------------|------------|------|-----|
| Wavelength (Å) | 0.99 | 1.44 | 0.99 | 0.99 |
| Resolution (Å) | 30–2.5 | 30–2.9 | 50–3.0 | 50–3.0 |
| $R_{\text{merge}}$* | 0.064 (0.247) | 0.079 (0.207) | 0.076 (0.224) | 0.065 (0.254) |
| Completeness (%) | 97.1 (88.9) | 90.4 (76.3) | 98.8 (95.6) | 93.3 (68.3) |
| $\langle I / \sigma(I) \rangle$ | 10.6 (3.4) | 10.8 (4.6) | 20.5 (8.9) | 21.1 (4.2) |

### Phasing statistics
| No. of heavy atom sites | – | – | 4 | 8 |
| $R_{\text{culis}}$ (centric)t | – | – | 0.86 | 0.81 |
| Phasing power (iso/ano)t | – | –0.51 | 1.23/1.03 | 1.62/1.07 |
| Resolution (Å) | – | 30–2.5 | 30–3.0 | 30–3.0 |
| FOM‡ | – | 0.57 | – | – |

### Refinement statistics
| Reflections (working set) | 233,377 | – | Number of non-hydrogen atoms | – |
| Reflections (test set) | 4,743 | – | Protein | 17,404 |
| $R_{\text{work}} / R_{\text{free}}$ (%) | 19.8/21.7 | – | Cofactors | 6,602 |
| R.m.s.d. bond lengths (Å) | 0.0126 | – | Water | 201 |
| R.m.s.d. bond angles (°) | 1.475 | – | Metal | 1 |
| Coordinate error (Å)‡ | – | – | – | – |
| Luzzati | 0.32 | – | – | – |
| SigmaA | 0.36 | – | – | – |

Numbers in parentheses correspond to values in the highest resolution shell.

* $R_{\text{merge}} = \Sigma_h \Sigma_i |I_i(h)| - \langle I(h) \rangle / \Sigma_h \Sigma_i |I_i(h)|$, where $I_i(h)$ are individual intensities of any reflection and $\langle I(h) \rangle$ is the mean intensity.
† FOM (figure of merit), $R_{\text{culis}}$ (centric) and phasing power determined by the program SHARP (see Supplementary Information).
‡ From Luzzati plot and SigmaA analyses, as determined with CNS (see Supplementary Information).

The pseudo-$C_2$ axis and for ferredoxin or flavodoxin (accepting electrons from PSI) at the stromal side. The binding pocket for the latter is formed by subunits PsaC, PsaD and PsaE (Fig. 1c).

### Polypeptide subunits
Subunits PsaA and PsaB share similarities in protein sequence and structure. They contain 11 transmembrane helices each that are divided into an amino-terminal domain (six α-helices: A/B-a to A/B-f) and a carboxy-terminal domain (five α-helices: A/B-g to A/B-k); for nomenclature see Fig. 1. The latter form two interlocked semicircles enclosing the ETC cofactors. This core structure is separated from the N-terminal α-helices A/Ba-d and the transmembrane α-helices of the smaller PSI subunits by an elliptically distorted cylindrical region bridged by α-helices A/B-e and f and harbouring a large number of the antenna Chla molecules and carotenoids (see Fig. 1a and 'The core antenna system' section).

The 'flat but rugged' lumenal surface of PSI is mainly formed by loop regions connecting transmembrane α-helices of subunits PsaA and PsaB. The amino-acid sequences of the loops contain segments not conserved between PsaA and PsaB, breaking the pseudo-$C_2$ symmetry. The loops, forming several short α-helices and β-sheets, shield the cofactors from the aqueous phase. A hollow close to the pseudo-$C_2$ axis has been suggested to be the binding site for the electron carriers cytochrome c₆ or plastocyanin¹². The base of the hollow is formed by α-helices A/B-ij(2) of loops A/B-ij in which two conserved Trp residues A655 and B631 are located. Their indole groups are stacked at van der Waals distance and related by the pseudo-$C_2$ axis. As they point into the putative binding site, they may be engaged in interaction with and/or electron transfer from cytochrome c₆ or plastocyanin to the oxidized primary electron donor P700⁺. This view is supported by mutagenesis studies on cyanobacterial PSI where substitution of some of the amino acids of A/B-ij(2) including Trp affects electron transfer to P700¹³.

Besides the lumenal loops of PsaA/B, subunit PsaF contributes prominent structural features to this surface of PSI with two hydrophilic α-helices F-c and F-d at the N terminus of transmembrane helix F-f (Fig. 1a, b). As the shortest distance between their helix axes and the pseudo-$C_2$ axis is 27 Å, direct interaction with cytochrome c₆ or plastocyanin is unlikely, consistent with the observation that deletion of PsaF does not influence the kinetics of electron transfer in cyanobacteria¹⁴. By contrast, an insertion and an extension in the N-terminal domain in PsaF of higher plants (see ref. 15 and references therein) are involved in plastocyanin docking.

The loops at the stromal surface of PsaA/B partly constitute the binding interface to subunits PsaC, PsaD and PsaE. These subunits are close together and arranged as a crescent, with PsaD closest to the C₃ axis, PsaC in the middle and PsaE farthest out (Fig. 1b, c). They surround an indentation on PsaA that was proposed as the binding site for ferredoxin¹²,¹⁶.

PsaC harbours the two Fe₄S₄ clusters Fₐ and Fₔ. It exhibits pseudo-twofold symmetry similar to bacterial 2Fe₄S₄ ferredoxins¹⁷, but contains an insertion of 10 amino acids in the loop connecting the iron-sulphur cluster binding motifs and extensions of the N and C termini by 2 and 14 amino acids, respectively (Fig. 2). As the insertion extrudes as a large loop, it may be engaged in docking of ferredoxin or flavodoxin. The long C terminus of PsaC (green in Fig. 2) interacts with PsaA/B/D and appears to be important for the proper assembly of PsaC into the PSI complex.

![Figure 2](https://via.placeholder.com/500x500)

**Figure 2** Cofactors of the electron transfer chain (ETC) and of PsaC. View parallel to the membrane plane. The pairs of chlorophylls of the ETC are arranged in two branches A and B. They are labelled eC, followed by the letter A or B indicating whether PsaA or PsaB, respectively, coordinates Mg²⁺, and by numbers 1 to 3 starting from the lumenal side. Phylloquinones are Qₖ-A and Qₖ-B. The Fe₄S₄ clusters are labelled Fₓ, Fₐ and Fₔ according to their spectroscopic terms. The centre-to-centre distances between the cofactors (black lines) are given in Å. In PsaC, parts analogous to 2[Fe₄S₄] bacterial ferredoxins¹⁷ are pink, an insertion and extensions at C and N termini are green.

Table 2 Coordination of the antenna Chla

Central ligands of the antenna Chla

| His | Asn | Gln | Asp | Glu | Tyr | Backbone | Phospho-diester | Water | Unknown ligand | Total |
|-----|-----|-----|-----|-----|-----|----------|-----------------|-------|----------------|-------|
| All subunits | 65 | 3 | 1 | 2 | 1 | 1 | 1 | 15 | 1 | 90 |
| PsaA | 32 | 2 | 1 | 2 | 1 | 1 | 1 | 5 |  | 40 |
| PsaB | 29 | 1 | 1 | 1 | 1 |  |  | 7 |  | 39 |
| PsaJ | 1 |  |  | 1 |  |  |  | 1 |  | 3 |
| PsaK | (1)* |  |  |  |  |  | 1 |  |  | 2 |
| PsaL | 1 |  |  | 1 |  |  |  | 1 |  | 3 |
| PsaM |  |  |  |  |  |  |  | (1)* |  | 1 |
| PsaX |  | 1 |  |  |  |  |  |  |  | 1 |
| PG |  |  |  |  |  |  |  |  | 1 | 1 |

Sectioning of antenna Chla coordinated by PsA/PsaB into peripheral and central antenna domains†

| N-terminal sequence region | C-terminal sequence region |
|----------------------------|----------------------------|
| transm. α-helices | Loops | transm. α-helices | loops |
| a | b | e | f | N-ter‡ | ab | g | h | k | fg | jk |
| central domain | PsaA | 3 | 1 | 1 | 3 | 2 | 2 | 3 | 3 | 1 | 1 | 2 |
|  | PsaB | 3 | – | 1 | 3 | 1 | 3 | 3 | 3 | 1 | 1 | 2 |
|  |  | b | c | d | e | cd | de |  |  | gh |  |  |
| peripheral domain | PsaA | 1 | 4 | 3 | 2 | 1 | 5 | – | – | – | 2 | – |
|  | PsaB | 1 | 4 | 3 | 2 | – | 5 | – | – | – | 3 | – |

* Numbers in parentheses correspond to suspect ligands.
† Secondary structural elements are labelled according to the scheme presented in legend to Fig. 1.
‡ Loops N-terminal of transmembrane α-helices A/B-a.

PsaD forms an antiparallel, four-stranded β-sheet, in which the loop connecting the third and fourth strands contains an α-helix, followed by a two-stranded β-sheet. The loop segment reaching from His D95 to the C terminus is attached by numerous hydrogen bonds to the stromally exposed sides of PsaC and E, and seems to help in the positioning of these subunits, as suggested by the importance of PsaD for electron transfer from Fx to FA/FB.

The structure of PsaE consists of a five-stranded antiparallel β-barrel similar to that found in NMR studies on PsaE from Synechococcus PCC 7002 and Nostoc sp.¹⁸,¹⁹. The long loop connecting β-strands c and d seen here, however, is in a conformation different from that in solution, probably because it interacts with PsaC and the stromal sides of PsaA and PsaB.

Together with PsaA and PsaB, the small subunits featuring transmembrane α-helices bind and stabilize the cofactors of the core antenna system. PsaJ, PsaK, PsaL, PsaM and PsaX coordinate the Mg²⁺ ions of antenna Chla molecules directly or through water molecules (Table 2). Subunits PsaF, PsaI, PsaJ, PsaL, PsaM and to a lesser extent PsaK are in hydrophobic contact with the carotenoids. In the trimerization domain, PsaL contributes to most contacts between monomers, supported by PsaI, PsaB and PsaA. According to the electron density map, the C terminus of PsaL is six amino-acid residues longer than the published sequence⁹ but agrees with the complementary DNA sequence if a frame shift is accounted for. A high spherical electron density (at 5σ (Fo–Fc)) located at the interface between adjacent monomers and surrounded by Asp side chains of PsaL and PsaA may belong to a metal ion that could be important for trimer formation. It might be a Ca²⁺ as found in trimeric but not in monomeric PSI from Synechocystis PCC 6803 (P. Chitinis, personal communication).

eC-B2 and eC-A2.

Charge separation is initiated at the primary electron donor P700 formed by the chlorophyll pair eC-A1/eC-B1. The chlorin planes of these chlorophylls are parallel at 3.6 Å interplanar distance and oriented perpendicular to the membrane plane. Rings I and II overlap partially with the Mg²⁺ ions separated by 6.3 Å (Fig. 3a). The structure of P700 differs from the ‘special pair’ formed by bacteriochlorophylls in PbRC, where the Mg²⁺ separation is larger (7.6 Å)²⁰ than in P700 and π–π interaction can be assumed to be stronger because of the perfect overlap of rings I²¹.

In contrast to the homodimeric special pair in PbRC, P700 is a heterodimer, eC-B1 being Chla (Mg²⁺ coordinated to His B660) and eC-A1 being chlorophyll a′ (Chla′), the C13² epimer of Chla (Mg²⁺ coordinated to His A680). This is consistent with chlorophyll extraction experiments suggesting that P700 contains one or two Chla′ molecules²². The asymmetry extends to the binding pockets for the two chlorophylls. Whereas the side chains of transmembrane α-helices A-i and A-k and a water molecule form hydrogen bonds to Chla′ eC-A1 (Fig. 3a, b), no hydrogen bonds are found to Chla eC-B1. In analogy to ref. 23, the electron spin density in P700• should be localized to a larger extent on the non-hydrogen-bonded Chla (eC-B1), in agreement with mutagenesis²⁴ and electron nuclear double resonance (ENDOR) studies²⁵, showing that ≈80% of the spin density in P700• is probably located on the chlorophyll bound to PsaB.

The chlorophylls of the second pair of Chla molecules next to P700, eC-A1/eC-B2, are not engaged in any hydrogen bonds. Their Mg²⁺ ions are coordinated by water molecules hydrogen-bonded to side chains of Asn A604 and Asn B591, respectively (Fig. 4a).

One or both of the Chla molecules of the third pair of chlorophylls eC-A3/eC-B3 probably represents the electron acceptor A₀. The Chla molecules are coordinated to sulphur atoms of Met A688 and Met B668 (distance Mg²⁺–S = 2.6 Å), respectively (Fig. 4a). This is remarkable, as interactions between the hard acid Mg²⁺ and the soft base S should be weak according to the concept of hard and soft acids and bases. To our knowledge, this coordination of Mg²⁺…S is unique in proteins. It remains to be elucidated whether the Mg²⁺…S coordination could contribute to the unusually low redox potential (approximately −105 mV) of electron acceptor A₀ (ref. 7). Short hydrogen bonds are donated by the hydroxyl groups of tyrosines A696 and B676 (dO–O = 2.7 Å) to the keto oxygens of rings V of

The electron transfer chain

This functionally most important part of PSI is formed by six chlorophylls, two phylloquinones and three Fe₄S₄ clusters (Fig. 2). The chlorophylls and phylloquinones are arranged in two branches A and B as pairs of molecules related by the pseudo-C₂ axis and coordinated to PsaA and PsaB. The A branch features chlorophylls eC-A1, eC-B2, eC-A3 and phylloquinone QK-A, and the B branch features chlorophylls eC-B1, eC-A2, eC-B3 and phylloquinone QK-B (for nomenclature see Fig. 2). The cofactors forming one branch are not bound exclusively to one subunit of PSI, the crossovers being
eC-A3 and eC-B3, respectively. As the chlorin planes of eC-B2 and eC-A2 are roughly parallel with and only ~3.8 Å from those of eC-A3 and eC-B3, respectively, they may be directly engaged in electron transfer from P700 to A₀ and possibly affect the spectroscopic and redox properties of the third Chla pair.

All the amino acids involved in Mg²⁺ coordination and hydrogen bonding to the second and third Chla pairs of the ETC are strictly conserved between PsaA and PsaB from cyanobacteria to higher plants, suggesting that these protein–chlorophyll interactions are essential for fine-tuning the redox potentials of the cofactors. As the edge-to-edge distances between adjacent chlorophylls of the ETC are very short (~4 Å), we expect that each participates in electron transfer.

One or both of the phylloquinones Qₖ-A and Qₖ-B might correspond to the electron acceptor A₁. Their quinone planes are π-stacked at 3.0–3.5 Å with indole rings of conserved Trp A697 and Trp B677 on the stromal surface α-helices A/B-jk(1), respectively. This agrees with modelling based on electron paramagnetic resonance (EPR) data²⁶ and quinone reconstitution studies²⁷. Only the

![Diagram](attachment:diagram.png)

Figure 3 Local environment of P700. **a**, eC-A1 (Chla′) and eC-B1 (Chla), with surrounding transmembrane α-helices (grey). View perpendicular to the chlorin planes. Rings I and II overlap partially. All possible hydrogen bonds between eC-A1, a water (red sphere) and amino acids of PsaA (turquoise) as dashed lines with distances (Å) lettered: a, Thr A7430γ ⋯ eC-A10-13¹ = 3.0; b, Thr A7430γ ⋯ water = 2.7; c, water ⋯ eC-A1 methoxyO-13⁴ = 3.2; d, water ⋯ Gly A7390 = 3.3; e, Tyr A6030η ⋯ water = 2.7; f, Ser A6070γ ⋯ water = 2.9; g, Tyr A7350η ⋯ eC-A1 O-17³ = 2.8. These residues are strictly conserved between subunits PsaA of many organisms, but none of them is conserved between PsaA and PsaB. The corresponding amino-acid side chains in PsaB (pink) cannot form hydrogen bonds to eC-B1. His A680 and His B660 coordinating eC-A1 and eC-B1, respectively, are also shown. **b**, As in **a** but view along the pseudo-C₂ axis.

carbonyl oxygen atom in ortho position to the phytyl chain accepts hydrogen bonds from backbone NH of Leu A722 (Qₖ-A) and Leu B706 (Qₖ-B) (Fig. 4c, d). It remains to be seen whether the lack of a hydrogen bond to the para-carbonyl oxygen contributes to the low redox potential (approximately –820 mV) of the phylloquinone in PSI, whereas that of the quinone Qₐ in PbRC and in PSII is around –130 mV.

A controversial issue in PSI is whether one or both branches of the ETC are active (for review see refs 7, 28). Spectroscopic data on P700⁺• and the radical pair P700⁺•/A₁⁻• suggest an asymmetry of electron transfer along the ETC in PSI. P700 is the only site where a significant structural difference between the two branches is obvious. The spin-carrying half of P700⁺• is probably Chla eC-B1, yielding two possible radical pairs: eC-B1⁺•/Qₖ-B⁻• and eC-B1⁺•/Qₖ-A⁻•, with centre-to-centre distances of 24.4 ± 0.3 Å and 26.0 ± 0.3 Å, respectively. As the latter value agrees better with the distance of 25.4 ± 0.3 Å between P700⁺• and A₁⁻• determined by EPR spectroscopy at 80 K²⁹, this phylloquinone might be Qₖ-A, as also suggested by mutagenesis studies³⁰. According to kinetic investigations, electron transfer in PSI involves both branches of the ETC with different rate constants³¹,³² of 35 × 10⁶ s⁻¹ and 4.4 × 10⁶ s⁻¹ for the electron transfer steps from each phylloquinone to Fₓ. Mutation of Trp A697 in the Qₖ-A binding site decreases the slower rate, indicating that the A branch is the slower one³¹,³². Surprisingly, the edge-to-edge distances (which according to the Moser–Dutton approximation³³ determine the optimal electron transfer rates) are identical (R = 6.8 Å) between Qₖ-A/Qₖ-B and Fₓ, and would correspond to optimal rates³³ of 8.3 × 10¹⁰ s⁻¹ for the corresponding electron transfer steps in both branches, orders of magnitude faster than the values for the experimental rate but in agreement with an estimation of 9 × 10¹⁰ s⁻¹ derived from the temperature dependence of the slower experimental rate³⁴. The different transfer rates may be caused by the most obvious elements of asymmetry in the vicinity of the two binding pockets: Trp B673 (not conserved in PsaA) located between Qₖ-B and Fₓ, the negatively charged phospholipid (I) at 15 Å from Qₖ-A, contrasting to the uncharged galactolipid (II) at the same distance from Qₖ-B and differences in the number and arrangement of water molecules located between Qₖ-A, Qₖ-B and Fₓ. Other possible explanations might relate to minor structural differences not detectable at the present resolution.

The pseudo-C₂ axis passes right through the [4Fe4S] cube of Fₓ (ref. 5), which is a rare example of an inter-polypeptide iron sulphur cluster. Fₓ is coordinated to both PsaA and PsaB in strictly conserved loop segments A/B-hi containing two cysteines each (Fig. 4b), as proposed from the amino-acid sequences³⁵. A possible geometrical distortion of Fₓ associated with the unusually low redox potential of around –0.7 mV (ref. 36) is not obvious at the present resolution. The assignment of the two Fe₄S₄ clusters in PsaC to Fₐ and Fₔ is now clear, as the cysteine residues of PsaC coordinating Fₐ and Fₔ are known from mutational studies³⁷ and are well defined in the structure. The arrangement of the clusters with Fₐ being closer to Fₓ than Fₔ suggests a sequence Fₓ → Fₐ → Fₔ in electron transfer, in agreement with spectroscopic data (ref. 38 and refs therein).

## The core antenna system

In contrast to PbRC, which collects light energy through separate membrane-intrinsic light-harvesting complexes, PSI features a core antenna system formed by 90 Chla molecules and 22 carotenoids (Figs 1 and 5). Seventy-nine of the antenna Chla molecules are bound to PsaA and PsaB (Table 2), but the small subunits PsaJ, PsaK, PsaL, PsaM, PsaX and lipid (III), a phosphatidylglycerol, also coordinate to Mg²⁺ of 11 Chla molecules either directly or indirectly through water molecules (Table 2). Except for aC-A40 and aC-B39, the 43 innermost Chla molecules of the antenna fill the volume provided by the elliptically distorted cylindrical region centred on the pseudo-C₂ axis and are separated from the chlorophylls of the
articles

ETC by not less than ~18 Å (Fig. 5). The antenna is extended by 2 sets of 18 peripheral Chla molecules bound to PsaA and PsaB, forming layer-like structures at the stromal and lumenal sides (Fig. 5b and Table 2). The arrangement of the PsaA/B-bound antenna Chla molecules follows the pseudo-C2 symmetry in a loose sense. Almost all of the Chla molecules are at Mg²⁺–Mg²⁺ distances between 7 and 16 Å (Fig. 5c), a range supporting fast excitation energy transfer of the Förster type³⁹. Although most of the Chla molecules are coordinated to His imidazoles, some are coordinated to oxygen atoms of Gln, Asp, Glu and Tyr side chains, to peptides or to water oxygens (Table 2).

Two of the Chlas (aC-A40 and aC-B39) are special as they structurally and perhaps functionally connect eC-A3 and eC-B3 of the ETC to the antenna (Fig. 5c). The centre-to-centre distances are 12.8 Å between aC-A40 and eC-A3 and 10.9 Å between aC-B39 and eC-B3. Otherwise, the ETC and antenna are well separated and isolated from each other. It is not yet clear whether transfer of energy proceeds through these two Chlas from the antenna to P700. If this were the case, eC-A2, eC-A3, eC-B2 and eC-B3 could be engaged both in excitation energy and in electron transfer.

The 'red' chlorophylls

In *S. elongatus* PSI, 9 to 11 Chla molecules per monomer absorb light at longer wavelengths (bathochromically or ‘red-shifted’) than P700 (ref. 40), possibly facilitating light energy capture under extreme environmental conditions or focussing excitation energy to the core of the reaction centre (for review see ref. 41). Among several causes of the red shift⁴¹, excitonic coupling of Chla can be most safely deduced from the present structure. Putative candidates for strongly excitonically coupled Chla molecules are one trimer and three dimers. In the Chla trimer at the lumenal side of the membrane and close to PsaX, aC-B31, aC-B32 and aC-B33 are stacked like a staircase (Fig. 5), with interplanar separations within 3.5–3.7 Å and lateral centre-to-centre shifts of 8.3 Å. Of these Chlas, only aC-B31 is directly coordinated to His B470, the Mg²⁺ of the other two being coordinated to water. The pseudo-translational symmetry of this trimer implies that the transition dipole moments of the chlorins are roughly parallel, reminiscent of crystal structures of chlorophyll derivatives⁴², with absorption maxima at 740 nm. This extreme red shift compared to the solution state is thought to derive from strong excitonic coupling supported by π–π interactions within the stacks. The Chla trimer is singular in PSI, possibly because a counterpart in PsaA related by the pseudo-C2 symmetry would interfere with the monomer–monomer contact in the PSI trimer and the hydrophobic segment stabilizing the Chla trimer in PsaB is deleted in PsaA.


![Figure 4](#fig4)

**Figure 4** Local environment of membrane-intrinsic cofactors of the electron transfer chain. **a**, Chlorophylls of the A branch of the ETC, with axial ligands and hydrogen bonds involving chlorophylls eC-B2 and eC-A3. In the B branch, eC-A2 and eC-B3 show similar interactions, as these amino acids are conserved between PsaA and PsaB. **b**, Loops A-hi of PsaA (turquoise) and B-hi of PsaB (pink) surround the inter-polypeptide iron sulphur cluster Fₓ, coordinated by cysteines A578, A587 and B565, B574. Pseudo-C₂ axis as grey vertical line. **c**, Phylloquinone Qₖ-A (yellow) in its binding pocket. Qₖ-B bound similarly by PsaB. Main chain of PsaA as ribbon and a carotenoid (car) in grey. Trp A697 π–π stacks with Qₖ-A. Only one hydrogen bond, Leu A722 NH to carbonyl-oxygen 4 of Qₖ-A, is formed. A carotenoid (car) is located in the vicinity of Qₖ-A. **d**, View of the quinone binding site normal to the quinone plane.

In the three putative red-shifted dimers, the chlorin planes are nearly parallel, π-stacked at 3.5 Å interplanar distance and laterally shifted with Mg²⁺–Mg²⁺ separation of 7.6–8.9 Å. This suggests π–π interactions in addition to the calculated strong excitonic coupling due to interaction of the Qy transition dipoles.

As the Chla pair aC-A32/aC-B7 (Fig. 5) is located at the lumenal side of the ‘trimerization domain’ and forms hydrophobic contacts to the C-terminal region of PsaL of the adjacent monomer, it could have a role in excitation energy transfer between the monomers. These Chla molecules may be identical to chlorophylls absorbing at 719 nm, which are lost when the trimer of S. elongatus is split into monomers⁴⁰.

The two other Chla dimers aC-A38/aC-A39 and aC-B37/aC-B38 are coordinated to PsaA and PsaB at the stromal side of the membrane. They obey the pseudo-C₂ symmetry, with one dimer located close to PsaF and the other close to PsaL. Each of the two Chla pairs is close (centre-to-centre distance 16 Å) to one of the ‘connecting’ Chla molecules, aC-A38/aC-A39 to aC-A40 and aC-B37/aC-B38. Theoretical and mutagenesis studies should be used to show whether these dimers play a role in fast excitation energy transfer to the ETC through the two connecting chlorophylls.

### The carotenoids

The 22 carotenoids identified in the electron density were modelled as β-carotene, which is dominant in PSI⁴³,⁴⁴. Of the carotenoids that were modelled as complete β-carotenes, 16 adopt the all-trans configuration and five contain one or two cis bonds, of which two were modelled as 9-cis and one each as 9,9′-cis, as 9,13′-cis and as 13-cis carotenoids, respectively. The numbers and configurations of these isomers agree well with results from high-performance liquid chromatography analysis on PSI from S. vulcanus⁴⁵. The carotenoids are deeply inserted in the membrane, with only few closer to the lumenal or stromal surface. Their arrangement may be subdivided into six clusters (Fig. 5a). Clusters (1), (2), (3) and (4) contain 10 carotenoids and are bound to PsaA and PsaB by hydrophobic interactions, with (1) forming a few additional contacts to PsaK. Clusters (5) and (6), each consisting of six carotenoids, are in contact with PsaA and PsaB; (5) is also bound to PsaF and PsaJ and (6) to PsaI, PsaL and PsaM. The carotenoids in (1) and (3) are roughly related by the pseudo-C₂ axis to those in (2) and (4), respectively, but the pseudosymmetry is less obvious for (5) and (6).

The carotenoids might serve two major functions in photosynthesis (for review see ref. 46), namely light harvesting in the 450–570-nm range where absorption by Chla is negligible, and photoprotection by quenching excited Chla triplet states by a charge transfer mechanism, preventing formation of toxic singlet oxygen. The 22 carotenoids are in van der Waals contact (<3.6 Å) to 60 Chla headgroups, thereby facilitating efficient energy transfer from excited carotenoids to the Chla molecules as well as quenching of Chla triplets. Several of the carotenoids show extended π–π stacking with chlorophylls, which appears ideal for fine-tuning the absorption properties of the antenna.

### Lipids

Four lipids are found associated with PsaA/PsaB. Three are phospholipids (I, III, IV) and one is a galactolipid (II). These types of lipid were also identified biochemically (D. Linke, personal communication) and immunologically in PSI to be phosphatidylglycerol and monogalactosyldiglyceride⁴⁴,⁴⁷. The head groups of the four lipids are well defined in the electron density map. They are located at the stromal side of the membrane plane and form hydrogen bonds with PsaA/B; the phosphodiester group of lipid (III) binds the antenna Chla aC-PL1. The two fatty acid chains of each lipid are anchored between transmembrane α-helices of PsaA/B and extend towards the middle of the membrane. Palmitic acid chains of 16 C atoms and stearic acid chains of 18 C atoms were modelled into the electron density map for the innermost lipids (I) and (II), respectively, whereas only shorter chains could be modelled for lipids (III) and (IV), probably owing to a higher degree of disorder. The lipids are related by the pseudo-C₂ symmetry: galactolipid (II)

Figure 5 Spatial organization of the cofactors of the ETC and the antenna system in one monomer of PSI. **a**, View from the stromal side onto the membrane plane. The C₃ axis is indicated (triangle). Ring substituents of all chlorophylls omitted for clarity, putative nine excitonically coupled ‘red’ chlorophylls in red, chlorophylls coordinated by PsaA/PsaB in yellow, those bound to peripheral subunits in the colour of the coordinating subunits (Fig. 1), chlorophylls of the ETC in blue. Asterisks denote ‘connecting chlorophylls’ (see text). The 22 carotenoids are arranged in clusters 1 to 6. Four lipids (turquoise) labelled with roman numerals. **b**, Side view of PSI antenna rotated 90° about the horizontal axis with respect to **a**. Two vertical lines indicate separation of the antenna Chlas coordinated by PsaA/B into a central part and two peripheral parts. **c**, Schematic view of the chlorophyll positions represented by their central Mg²⁺ ions; view direction as in **a**. All Mg²⁺–Mg²⁺ distances <16 Å, thin lines. Chlas belonging to the ETC in blue, those of the antenna in grey and red. Some of the chlorophylls referred to in the text are named by a capital letter indicating the subunits involved in axial liganding and numbered from N to C; PL1 is the single Chla bound to a phospholipid. The prefix aC- is omitted for clarity.
and phospholipid (I) are close to the palisade of five transmembrane α-helices each in PsaB and PsaA (A/B-g to A/B-k), respectively. Of the other two lipids, one (III; bound to PsaA) is close to the monomer–monomer interface and the other (IV; bound to PsaB) is in contact with PsaX. The locations of lipids (I) and (II) close to the core of PSI and the binding of an antenna Chla by lipid (III) indicate that the four lipids are integral to and functionally important in the PSI complex and are not mere preparation artefacts.

### Comparison of the antenna systems in PSI and PSII

The distribution of the peripheral antenna Chla molecules bound to PsaA and PsaB in PSI strongly resembles that of the membrane integral antenna proteins CP43 and CP47 of PSII. Their amino-acid sequences are similar to those of the N-terminal domains of PsaA and PsaB<sup>48</sup> and fold into comparable inner-membrane structures consisting of six transmembrane α-helices each<sup>3</sup>.

In both types of structures the Chla molecules are arranged in two quasi-two-dimensional layers. The shortest distances between Chlas of adjacent layers in PSI are ≥18 Å, whereas the inner-layer distances are less than 16 Å (Fig. 5c). As the centre-to-centre distance between Chla molecules determines, among other factors, the inter-molecule excitation transfer rates, it is more likely that an exciton is transferred within a single layer than that it is exchanged between Chlas in two different layers. It can be speculated that this might favour more efficient exciton transfer from these peripheral antennae to the reaction centre core rather than a comparable situation where the same number of cofactors would be evenly distributed over the whole depth of the membrane.

The PSII reaction centre domain formed by subunits D1 and D2 has a structure similar to the C-terminal domains of PsaA and PsaB and probably binds only two Chla molecules in addition to the cofactors of the ETC<sup>3</sup>. In contrast to PSII, the C-terminal domains of PsaA and PsaB of PSI cannot be considered as pure reaction centre domains<sup>48</sup>, as they coordinate 12 (PsaA) and 13 (PsaB) Chlas of the antenna (Table 2). Moreover, the whole central part of the PSI core antenna, which contains 43 Chlas, is strikingly different from the corresponding region in PSII.

As each of the Chla molecules in the central PsaA/PsaB antenna of PSI is located at relatively short distances from the ETC, the excitation energy may be transferred with comparable efficiency to the cofactors of the ETC, in accordance with existing models<sup>49</sup>. It will be of interest to clarify whether the almost complete lack of such a central antenna in PSII is associated with its lower efficiency, as compared to PSI, in transferring excitation energy to the reaction centre core<sup>50</sup>.

This first structural model of PSI at atomic detail provides many new insights into the working of the photosynthetic system both of cyanobacteria and of plants. Whereas uncertainties remain as to the form adopted by and hence functioning of the peripheral PsaK, we have been able to resolve PsaX in the structure, proving it to be an intrinsic component of the photosystem. A number of novel modes of chlorophyll binding have also been revealed. The structure will help to answer current questions central to PSI chemistry, such as the role of the carotenoid cofactors and whether the PsaA/PsaB pseudosymmetry is reflected at the functional level. The structure provides a sound basis for functional analyses of exciton transfer and electron transport in plant and cyanobacterial photosystems, information that should prove valuable in a wealth of other contexts involving similar atomic interactions.

Received 26 February; accepted 2 May 2001.

1. Nitschke, W. & Rutherford, A. W. Photosynthetic reaction centres: variations on a common structural theme? *Trends Biochem. Sci.* **16**, 241–245 (1991).
2. Lancaster, C. R. D., Bibikova, M. V., Sabatino, P., Oesterhelt, D. & Michel, H. Structural basis of the drastically increased initial electron transfer rate in the reaction center from a *Rhodopseudomonas viridis* mutant described at 2.00-Å resolution. *J. Biol. Chem.* **275**, 39364–39368 (2000).
3. Zouni, A. *et al*. Crystal structure of photosystem II from *Synechococcus elongatus* at 3.8 Å resolution.
4. Krauss, N. *et al*. Photosystem I at 4 Å resolution represents the first structural model of a joint photosynthetic reaction centre and core antenna system. *Nature Struct. Biol.* **3**, 965–973 (1996).
5. Klukas, O. *et al*. Localization of two phylloquinones, Q<sub>K</sub> and Q<sub>K</sub>', in an improved electron density map of photosystem I at 4 Å resolution. *J. Biol. Chem.* **274**, 7361–7367 (1999).
6. Boekema, E. J. *et al*. Evidence for a trimeric organization of the photosystem I complex from the thermophilic cyanobacterium *Synechococcus* sp. *FEBS Lett.* **217**, 283–286 (1987).
7. Brettel, K. Electron transfer and redox-cofactors in photosystem I. *Biochim. Biophys. Acta* **1318**, 322–373 (1997).
8. Fromme, P. & Witt, H. T. Improved isolation and crystallization of photosystem I for structural analysis. *Biochim. Biophys. Acta* **1365**, 175–184 (1998).
9. Mühlenhoff, U., Haehnel, W., Witt, H. T. & Herrmann, R. G. Genes encoding eleven subunits of photosystem I from the thermophilic cyanobacterium *Synechococcus* sp. *Gene* **127**, 71–78 (1993).
10. Koike, K., Ikeuchi, M., Hiyama, T. & Inoue, Y. Identification of photosystem I components from the cyanobacterium *Synechococcus vulcanus* by N-terminal sequencing. *FEBS Lett.* **253**, 257–263 (1989).
11. Ikeuchi, M., Nyhus, K. J., Inoue, Y. & Pakrasi, H. B. Identities of four low-molecular-mass subunits of the photosystem I complex from *Anabaena variabilis* ATCC 29413. Evidence for the presence of the *psaI* gene product in a cyanobacterial complex. *FEBS Lett.* **287**, 5–9 (1991).
12. Fromme, P., Schubert, W.-D. & Krauss, N. Structure of photosystem I: Suggestions on the docking sites for plastocyanin, ferredoxin and the coordination of P700. *Biochim. Biophys. Acta* **1187**, 99–105 (1994).
13. Sun, J. *et al*. Oxidizing side of the cyanobacterial photosystem I. Evidence for interaction between the electron donor proteins and a lumenal surface helix of the PsaB subunit. *J. Biol. Chem.* **274**, 19048–19054 (1999).
14. Xu, Q., Yu, L., Chitnis, V. P. & Chitnis, P. R. Function and organization of photosystem I in a cyanobacterial mutant strain that lacks PsaF and PsaJ subunits. *J. Biol. Chem.* **269**, 3205–3211 (1994).
15. Hippler, M., Drepper, F., Rochaix, J. D. & Mühlenhoff, U. Insertion of the N-terminal part of PsaF from *Chlamydomonas reinhardtii* into photosystem I from *Synechococcus elongatus* enables efficient binding of algal plastocyanin and cytochrome c<sub>6</sub>. *J. Biol. Chem.* **274**, 4180–4188 (1999).
16. Mühlenhoff, U. *et al*. Characterization of a redox-active cross-linked complex between cyanobacterial photosystem I and its physiological acceptor flavodoxin. *EMBO J.* **15**, 488–497 (1996).
17. Adman, E. T., Sieker, L. C. & Jensen, L. H. The structure of a bacterial ferredoxin. *J. Biol. Chem.* **248**, 3987–3996 (1973).
18. Falzone, C. J., Kao, Y. H., Zhao, J., Bryant, D. A. & Lecomte, J. T. Three-dimensional solution structure of PsaE from the cyanobacterium *Synechococcus* sp. strain PCC 7002, a photosystem I protein that shows structural homology with SH3 domains. *Biochemistry* **33**, 6052–6062 (1994).
19. Mayer, K. L., Shen, G., Bryant, D. A., Lecomte, J. T. & Falzone, C. J. The solution structure of photosystem I accessory protein E from the cyanobacterium *Nostoc* sp. strain PCC 8009. *Biochemistry* **38**, 13736–13746 (1999).
20. Deisenhofer, J., Epp, O., Sinning, I. & Michel, H. Crystallographic refinement at 2.3 Å resolution and refined model of the photosynthetic reaction centre from *Rhodopseudomonas viridis*. *J. Mol. Biol.* **246**, 429–457 (1995).
21. Scheer, H. *Chlorophylls* (CRC Press, Boca Raton, Florida, 1991).
22. Watanabe, T., Kobayashi, M., Hongu, A., Nakazato, M. & Hiyama, T. Evidence that a chlorophyll a' dimer constitutes the photochemical reaction centre 1 (P700) in photosynthetic apparatus. *FEBS Lett.* **235**, 252–256 (1985).
23. Artz, K. *et al*. Relationship between the oxidation potential and electron spin density of the primary electron donor in reaction centers from *Rhodobacter sphaeroides*. *Proc. Natl Acad. Sci. USA* **94**, 13582–13587 (1997).
24. Webber, A. N. *et al*. Site-directed mutations affecting the spectroscopic characteristics and midpoint potential of the primary donor in photosystem I. *Biochemistry* **35**, 12857–12863 (1996).
25. Käss, H., Fromme, P., Witt, H. T. & Lubitz, W. Orientation and electronic structure of the primary donor radical cation P700<sup>+</sup>• in Photosystem I: a single crystals EPR and ENDOR study. *J. Phys. Chem. B* **105**, 1225–1239 (2001).
26. Kamlowski, A. *et al*. The quinone acceptor A<sub>1</sub> in photosystem I: Binding site, and comparison to Q<sub>A</sub> in purple bacteria reaction centers. *J. Phys. Chem. B* **102**, 8278–8287 (1998).
27. Iwaki, M. & Itoh, S. Structure of the phylloquinone-binding (Q<sub>A</sub>) site in green plant photosystem I reaction centers: the affinity of quinones and quinoid compounds for the Q<sub>A</sub> site. *Biochemistry* **30**, 5347–5352 (1991).
28. Golbeck, J. H. in *Advances in Photosynthesis: The Molecular Biology of Cyanobacteria* (ed. Bryant, D. A.) 319–360 (Kluwer Academic, Dordrecht, Netherlands, 1994).
29. Bittl, R., Zech, S. G., Fromme, P., Witt, H. T. & Lubitz, W. Pulsed EPR structure analysis of photosystem I single crystals: localization of the phylloquinone acceptor. *Biochemistry* **36**, 12001–12004 (1997).
30. Yang, F. *et al*. Deletion of the PsaF polypeptide modifies the environment of the redox active phylloquinone (A<sub>1</sub>). Evidence for unidirectionality of electron transfer in Photosystem I. *J. Phys. Chem. B* **102**, 8288–8299 (1998).
31. Joliot, P. & Joliot, A. In vivo analysis of the electron transfer within photosystem I: are the two phylloquinones involved? *Biochemistry* **38**, 11130–11136 (1999).
32. Guergova-Kuras, M., Boudreaux, B., Joliot, A., Joliot, P. & Redding, K. Evidence for two active branches for electron transfer in Photosystem I. *Proc. Natl Acad. Sci. USA* **98**, 4437–4442 (2001).
33. Moser, C. C. & Dutton, P. L. Engineering protein structure for electron transfer function in photosynthetic reaction centers. *Biochim. Biophys. Acta* **1101**, 171–176 (1992).
34. Schlodder, E., Falkenberg, K., Gergeleit, M. & Brettel, K. Temperature dependence of forward and reverse electron transfer from A1-, the reduced secondary electron acceptor in photosystem I. *Biochemistry* **37**, 9466–9476 (1998).
35. Fish, L., Kück, U. & Bogorad, L. Two partially homologous adjacent light-inducible maize chloroplast genes encoding polypeptides of the P700 chlorophyll a protein complex of photosystem I. *J. Biol. Chem.* **260**, 1413–1421 (1985).
36. Chamorowsky, S. K. & Cammack, R. Direct determination of the midpoint potential of the acceptor X in chloroplast photosystem I by electrochemical reduction and ESR spectroscopy. *Photobiophys. Photobiol.* **4**, 195–200 (1982).
37. Zhao, J., Li, N., Warren, P. V., Golbeck, J. H. & Bryant, D. A. Site-directed conversion of a cysteine to aspartate leads to the assembly of a [3Fe-4S] cluster in PsaC of photosystem I. The photoreduction of F<sub>A</sub> is independent of F<sub>B</sub>. *Biochemistry* **31**, 5093–5099 (1992).
38. Golbeck, J. H. A comparative analysis of the spin state distribution of in vitro and in vivo mutants of PsaC. *Photosynth. Res.* **61**, 107–144 (1999).

39. Förster, T. Zwischenmolekulare Energiewanderung und Fluoreszenz. *Ann. Phys. (Leipzig)* **2**, 55–75 (1948).

40. Pålsson, L. O. *et al.* Energy transfer and charge separation in photosystem I: P700 oxidation upon selective excitation of the long-wavelength antenna chlorophylls of *Synechococcus elongatus*. *Biophys. J.* **74**, 2611–2622 (1998).

41. Fleming, G. R. & van Grondelle, R. Femtosecond spectroscopy of photosynthetic light-harvesting systems. *Curr. Opin. Struct. Biol.* **7**, 738–748 (1997).

42. Kratky, C. & Dunitz, J. D. Ordered aggregation states of chlorophyll and same of its derivatives. *J. Mol. Biol.* **113**, 431–442 (1977).

43. Coufal, J., Hladik, J. & Sofrova, D. The carotenoid content of photosystem I-pigment-protein complexes of the Cyanobacterium *Synechococcus elongatus*. *Photosynthetica* **23**, 603–616 (1989).

44. Makewicz, A., Radunz, A. & Schmidt, G. H. Comparative immunological detection of lipids and carotenoids on peptides of photosystem I from higher plants and cyanobacteria. *Z. Naturforsch.* **51c**, 319–328 (1996).

45. Ashikawa, I., Miyata, A., Koike, H., Inoue, Y. & Koyama, Y. Light-induced structural change of β-carotene in thylakoid membranes. *Biochemistry* **25**, 6154–6160 (1986).

46. Cogdell, R. J. Carotenoids in photosynthesis. *Pure Applied Chem.* **57**, 723–728 (1985).

47. Makewicz, A., Radunz, A. & Schmidt, G. H. in *Plant Lipid Metabolism* (eds Kader, J.-C. & Mazliak, P.) 156–160 (Kluwer Academic, Dordrecht, Netherlands, 1995).

48. Schubert, W. D. *et al.* A common ancestor for oxygenic and anoxygenic photosynthetic systems: a comparison based on the structural model of photosystem I. *J. Mol. Biol.* **280**, 297–314 (1998).

49. Valkunas, L., Liulia, V., Dekker, J. P. & van Grondelle, R. Description of energy migration and

trapping in photosystem I by a model with two distance scaling parameters. *Photosyn. Res.* **43**, 149–154 (1995).

50. Karapetyan, N. V., Holzwarth, A. R. & Rögner, M. The photosystem I trimer of cyanobacteria: molecular organization, excitation dynamics and physiological significance. *FEBS Lett.* **460**, 395–400 (1999).

Supplementary information is available on Nature’s World-Wide Web site (http://www.nature.com) or as paper copy from the London editorial office of *Nature*.

### Acknowledgements

These studies are financially supported by Deutsche Forschungsgemeinschaft, by Bundesministerium für Bildung und Forschung and by Fonds der Chemischen Industrie. We thank D. DiFiore and H. Schmidt for technical assistance; W. Schroeder for supporting the analytical characterisation of the PSI preparations; B. Rasmussen and J. Lescar for long-term support during data collection at beamline ID2B at ESRF in Grenoble; and J. Hughes for reading and improving the manuscript. We particularly thank D. A. Bryant for discussions regarding PsaX and acknowledge R. Bittl, K. Brettel, M. Byrdin, P. Chitnis, J. Golbeck, B. Loll, W. Lubitz, E. Schlodder and D. Stehlik for discussions.

Correspondence and requests for materials should be addressed to P.F. (e-mail: fromme@phosis1.chem.tu-berlin.de) or N.K. (e-mail: nkrauss@chemie.fu-berlin.de). The X-ray crystallographic coordinates will be deposited at Brookhaven data bank under accession number 1JB0.
