
Journal of Industrial Microbiology & Biotechnology (1999) 23, 380–390  
© 1999 Society for Industrial Microbiology 1367-5435/99/$15.00  

http://www.stockton-press.co.uk/jim  

# Complete analysis of genes and enzymes for γ-hexachlorocyclohexane degradation in *Sphingomonas paucimobilis* UT26

Y Nagata, K Miyauchi and M Takagi  

**Department of Biotechnology, The University of Tokyo, 1-1-1 Yayoi, Bunkyo-ku, Tokyo 113–8657, Japan**

γ-Hexachlorocyclohexane (γ-HCH; also called BHC or lindane) is one of the highly chlorinated pesticides which can cause serious environmental problems. *Sphingomonas paucimobilis* UT26 degrades γ-HCH under aerobic conditions. The unique degradation pathway of γ-HCH in UT26 is revealed. In the upstream pathway, γ-HCH is transformed to 2,5-dichlorohydroquinone (2,5-DCHQ) by two different dehalogenases (LinA and LinB) and one dehydrogenase (LinC) which are expressed constitutively. In the downstream pathway, 2,5-DCHQ is reductively dehalogenated, and then ring-cleaved by enzymes (LinD and LinE, respectively) whose expressions are regulated. We have cloned and sequenced five structural genes (*linA*, *linB*, *linC*, *linD*, and *linE*) directly involved in this degradation pathway. The *linD* and *linE* genes form an operon, and its expression is positively regulated by the LysR-type transcriptional regulator (LinR). The genes *linA*, *linB*, and *linC* are constitutively expressed, and are present separately from each other in the UT26 genome. Cell fractionation analysis, Western blotting, and immuno electron microscopy revealed that LinA and LinB are localized in the periplasmic space of UT26.

**Keywords:** *Sphingomonas*; γ-hexachlorocyclohexane; biodegradation; dehalogenase

## Introduction

γ-Hexachlorocyclohexane (γ-HCH; also called BHC or lindane) is a halogenated organic insecticide which has been used worldwide. Because of its toxicity and persistence in soil, many countries have prohibited the use of γ-HCH. However, many contaminated sites still remain throughout the world. Moreover, some countries are presently using γ-HCH for economic reasons, and thus new sites are continuously being contaminated by γ-HCH and its stereoisomers (a technical mixture of HCH consists of α, β, γ, and δ isomers) [4,20].

γ-HCH is degraded rapidly under anaerobic conditions, but it is considered extremely persistent under aerobic conditions. *Sphingomonas* (formerly *Pseudomonas*) *paucimobilis* SS86 was isolated from an experimental field to which γ-HCH had been applied once a year for 12 years [68]. We isolated a mutant of *S.* *paucimobilis* SS86, named UT26, that has nalidixic acid resistance as a genetic marker [15]. Strains SS86 and UT26 utilize γ-HCH as a sole source of carbon and energy under aerobic conditions.

The degradation pathways of γ-HCH are of interest because of its structural features: a cyclohexane skeleton and a highly chlorinated state. General information for degradation of HCH by microorganisms was reviewed in another article [23]. In this article, we focus on the enzymes and genes involved in the degradation pathway of γ-HCH in *S.* *paucimobilis* UT26. Some parts of this article were previously reviewed [47].

---

## Degradation pathway of γ-HCH in UT26

The degradation pathway of γ-HCH in strain UT26 is shown in Figure 1. Because γ-HCH has six chlorine atoms per molecule, dechlorination is a very significant step in its degradation. In fact, we have shown that three different types of dechlorination reactions are sequentially involved in the degradation of γ-HCH by *S.* *paucimobilis* UT26. The first reaction is dehydrochlorination of γ-HCH to 1,3,4,6-tetrachloro-1,4-cyclohexadiene (1,4-TCDN) via γ-pentachlorocyclohexene (γ-PCCH) [16]. The second reaction is hydrolytic dechlorination of 1,4-TCDN to 2,5-dichloro-2,5-cyclohexadiene-1,4-diol (2,5-DDOL) via 2,4,5-trichloro-2,5-cyclohexadiene-1-ol (2,4,5-DNOL) [43]. The third reaction is reductive dechlorination of 2,5-dichlorohydroquinone (2,5-DCHQ), which is produced from 2,5-DDOL by a dehydrogenase (LinC) [45], to hydroquinone (HQ) via chlorohydroquinone (CHQ) [37].

Because 2,5-DCHQ is mineralized by strain UT26 [41], the pathway via 2,5-DCHQ is considered to be the assimilation pathway of γ-HCH in UT26. Recently, we reported the degradation pathway of 2,5-DCHQ in strain UT26 (K Miyauchi, Y Nagata and M Takagi, unpublished data). HQ, which is produced from 2,5-DCHQ by LinD is ring-cleaved by a novel type of dioxygenase (LinE) to γ-hydroxymuconic semialdehyde (γ-HMSA). However, the pathway from CHQ to HQ seems not to be the major pathway in UT26, since CHQ is a better substrate for LinE than for LinD. As a result, most of the CHQ is directly ring-cleaved by LinE to acylchloride, although we have directly detected only maleylacetate and β-ketoadipate. β-ketoadipate is expected to be easily utilized in strain UT26.

Through this series of reactions, two dead-end products, 1,2,4-trichlorobenzene (1,2,4-TCB) and 2,5-dichlorophenol

Genes and enzymes for γ-HCH degradation  
Y Nagata et al  

![Chemical Reaction Diagram](attachment:chemical-reaction-diagram.png)  

Figure 1 Proposed degradation pathway of γ-HCH in *Sphingomonas paucimobilis* UT26. Compounds: 1, γ-hexachlorocyclohexane (γ-HCH, γ-BHC, lindane); 2, γ-pentachlorocyclohexene; 3, 1,3,4,6-tetrachloro-1,4-cyclohexadiene; 4, 1,2,4-trichlorobenzene; 5, 2,4,5-trichloro-2,5-cyclohexadiene-1-ol; 6, 2,5-dichlorophenol; 7, 2,5-dichloro-2,5-cyclohexadiene-1,4-diol; 8, 2,5-dichlorohydroquinone; 9, chlorohydroquinone; 10, hydroquinone; 11, acylchloride; 12, γ-hydroxymuconic semialdehyde (γ-HMSA); 13, maleylacetate; 14, β-ketoadipate.

(2,5-DCP), are produced from the unstable intermediates 1,4-TCDN and 2,4,5-DNOL, respectively [40–42]. We have not yet detected the degradation activity of γ-HMSA by strain UT26. γ-HMSA may also be a dead-end product.

In strain UT26, the activities of LinA, LinB, and LinC are constitutively expressed [47], while those of LinD and LinE are inducibly expressed [46]. The degradation pathway of γ-HCH to 2,5-DCHQ was named the upstream pathway, and that of 2,5-DCHQ was named the downstream pathway (Figure 1).

### Genes involved in γ-HCH degradation

The broad-host-range cosmid vector pKS13 [28] was used for the construction of a strain UT26 gene library in *Escherichia coli* HB101. Each clone of the *E. coli* library was mobilized into *Pseudomonas putida* PpY101 by triparental mating [28]. The resultant library in *P. putida* was screened for γ-HCH and its degradation metabolites by gas chromatography with an electron capture detector.

We have cloned five structural genes and one regulatory gene involved in degradation of γ-HCH in strain UT26 (Table 1). The *linA* gene encodes γ-HCH dehydrochlorinase (LinA), which is responsible for the conversion of γ-HCH to 1,4-TCDN via γ-PCCH [16,17]. The *linB* gene encodes 1,4-TCDN halidohydrolase (LinB), which is responsible for the conversion of 1,4-TCDN to 2,5-DDOL [43]. The *linC* gene encodes 2,5-DDOL dehydrogenase (LinC), which is responsible for the conversion of 2,5-DDOL to 2,5-DCHQ [45]. The *linD* gene encodes 2,5-DCHQ reductive dehalogenase (LinD), which is responsible for conversion of 2,5-DCHQ to HQ [37]. Recently, we cloned and sequenced a new gene (*linE*), which encodes the ring-cleavage oxygenase (LinE) which is responsible for conversion of CHQ to acylchloride and of HQ to γ-HMSA (K Miyayuchi, Y Nagata and M Takagi, unpublished data). The *linA*, *linB*, and *linC* genes are constitutively expressed [47], while the *linD* and *linE* genes are inducible in strain UT26 ([37]; K Miyayuchi, Y Nagata and M Takagi, unpublished data). The *linD* and *linE* genes form an operon with other open reading frames (Figure 2) whose functions are still unknown. Expression of *linD* and *linE* is regulated by the LysR-type transcriptional regulator (LinR) whose gene is located in the upstream region of the *linE* gene (Figure 2).

Four plasmids were detected in the CsCl-purified preparation from strain UT26 [16]. None of six genes for γ-HCH degradation hybridized with these four plasmids, suggesting that all of these genes are located on chromosomal DNA. However, the possibility of locating them on an unrecoverable plasmid, such as a megaplasmid or linear plasmid, cannot be excluded. The spontaneous mutants lacking

Genes and enzymes for γ-HCH degradation  
Y Nagata et al  

Table 1 Six genes involved in the γ-HCH degradation in Sphingomonas paucimobilis UT 26  

| Gene | Nucleotide (bp) | Amino acid residue | Molecular mass (kD) | G+C content (%) | Function | Expression in UT26 | Reference |
|------|------------------|--------------------|---------------------|-----------------|----------|---------------------|-----------|
| linA | 468              | 156                | 17.3                | 53.9            | Dehydrochlorinase | Constitutive | [16]      |
| linB | 888              | 296                | 33.1                | 62.5            | Halidohydrolase   | Constitutive | [43]      |
| linC | 750              | 250                | 25.6                | 64.3            | Dehydrogenase     | Constitutive | [45]      |
| linD | 1038             | 346                | 38.4                | 61.0            | Reductive dechlorinase | Inducible | [37]      |
| linE | 963              | 321                | 36.0                | 60.1            | Ring-cleavage oxygenase | Inducible | Miyachi, Nagata and Takagi, unpublished data |
| linR | 909              | 303                | 33.6                | 61.3            | LysR-type transcriptional regulator for linD and linE | ?         |               |

![Diagram](attachment:gene_organization_diagram.png)

Figure 2 Gene organization of the operon containing *linD*, *linE*, and their flanking regions. Size and direction of open reading frames are shown by arrows.

| Name    | Number of nucleotide (bp) | Number of amino acid | Molecular weight (kDa) | G + C content (%) | Function / Homologue (acc. no<sup>a</sup>) (identity / similarity) |
|---------|----------------------------|-----------------------|------------------------|-------------------|---------------------------------------------------------------|
| *linD*  | 1038                       | 346                   | 38.4                  | 60.8              | 2,5-dichlorohydroquinone reductive dehalogenase                 |
| *linE*  | 963                        | 321                   | 36.0                  | 60.1              | (chloro) hydroquinone 1,2-dioxygenase                          |
| *linR*  | 909                        | 303                   | 33.6                  | 61.3              | LysR-type transcriptional regulator                             |
| *orf2*  | 798                        | 266                   | 27.6                  | 62.0              | unknown / β-ketoadipate enol-lactone hydrolase (AF009224-17) (22% / 42%) |
| *orf3*  | 714                        | 238                   | 25.5                  | 61.9              | unknown / carboxyl esterase (D90904-70) (21% / 37%)             |
| *orf4*  | 639                        | 213                   | 22.9                  | 60.9              | unknown / 5-chloro-1,2,4-THB dechlorinase (U19883-5) (13% / 29%) |
| *orf5*  | 963                        | 321                   | 35.1                  | 59.8              | unknown / vanillate demethylase reductase subunit (M22077-2) (53% / 70%) |
| *orf6*  | (816)<sup>b</sup>          | (272)<sup>b</sup>      | —                     | (61.0)<sup>b</sup> | unknown / AraC/XylS family transcriptional regulator (L02356-2) (19% / 34%)<sup>b</sup> |
| *orf7*  | 915                        | 305                   | 34.3                  | 57.9              | unknown / vanillate demethylase oxygenase subunit (Y11521-1) (28% / 46%) |
| *orf8*  | 2544                       | 848                   | 91.1                  | 59.5              | unknown / pesticin receptor (Z35106-1) (20% / 35%)             |

<sup>a</sup> Accession number of DAD database.  
<sup>b</sup> Note that the open reading frame does not terminate within the sequenced region.

the whole *linA* or *linC* gene are easily isolated, indicating that these two genes are located on the relatively unstable DNA region.

The G+C content of the *linA* gene is considerably lower than that of the other five genes (*linB*, *linC*, *linD*, *linE* and *linR*) (Table 1) and the genomic DNA of the *S*. *paucimobilis* strains (65%) [57], suggesting that the *linA* gene may have originated from another organism with a lower G+C content. As described above, *linD* and *linE* form an operon. The *linA*, *linB*, and *linC* appear to locate separately from each other in the strain UT26 genome; at least these three genes are not organized as an operon. Considering the fact that genes involved in catabolic pathways usually form operons in prokaryotic cells [73], the genes involved in the upstream pathway of γ-HCH degradation in strain UT26 seem to be an exception. Similar cases were reported with genes for 2,4-dinitrotoluene degradation by *Pseudomonas sp.* DNT [70] and for pentachlorophenol degradation by *Sphingomonas chlorophenolica* strain ATCC 39723

[55, 56]. In these cases (including that of strain UT26), it is suggested that genes which take part in other function(s) may be involved. Recently, it was revealed that the genes in *Sphingomonas* necessary for degradation of one type of aromatic compound are distributed into multiple operons that also possess genes for the degradation of other aromatic compounds [1, 61, 82]. It is proposed that members of the genus *Sphingomonas* have a less well evolved and regulated but more dynamic genetic organization than organisms such as *Pseudomonas* species [1].

In the following sections, properties of the genes and enzymes which are involved in γ-HCH degradation by strain UT26 are discussed in detail.

### Enzymes for the upstream pathway

#### γ-HCH dehydrochlorinase (LinA)
A database search failed to find any significantly homologous sequences to the *linA* gene. Southern blot analyses

using the *linA* gene as a probe against total DNAs of several related strains revealed no homologous sequence [16]. The origin of the *linA* gene is of great interest, but is still unknown. Recently, genes homologous to *linA* were cloned independently by two groups in France and India. Thomas *et al* [71] isolated a *linA*-like gene by the PCR technique from a newly isolated γ-HCH-degrading bacterium. The gene they isolated was identical to the *linA* gene that we cloned. However, the gene that Lal *et al* cloned from *Sphingomonas paucimobilis*, which is the same strain reported by Sahu *et al* [64], is partly different from ours (R Lal, personal communication). The deduced amino acid sequences of our LinA and Lal’s exhibit 90.4% identity. Most of the differences are found at the C-terminal region. The evolutionary relationship between these genes remains to be elucidated.

The *linA* gene was highly expressed in recombinant *E. coli* cells, and the gene product (LinA) was purified to homogeneity [44]. In addition to γ-HCH and γ-PCCH, α- and δ-isomers of HCH and α-PCCH were also dehydrochlorinated by LinA; however, β-HCH was not [44]. These results are consistent with those obtained using resting cells of strain UT26 [40]. It is suggested that dehydrochlorination by LinA occurs stereoselectively at a trans and diaxial pair of hydrogen and chlorine [40]. The other chlorinated compounds tested were not dehydrochlorinated by the purified enzyme [44], indicating that the substrate specificity of LinA is narrow.

Dehydrochlorinase is an enzyme which eliminates HCl from the substrate molecule, leading to the formation of a double bond [10, 22]. The properties of three dehydrochlorinases (including LinA) have so far been reported. A eukaryotic dehydrochlorinase isolated from *Musca domestica* catalyzes the monodehydrochlorination of 1,1,1-trichloro-2,2-bis (*p*-chloro-phenyl) ethane (DDT) [7, 34]. 3-Chloro-D-alanine dehydrochlorinase was isolated from *P. putida* [39]. LinA is, however, very different from these two other dehydrochlorinases. DDT dehydrochlorinase and 3-chloro-D-alanine dehydrochlorinase require glutathione (GSH) and pyridoxal 5′-phosphate (PLP), respectively, for their activities, while LinA does not require any cofactors. Purified LinA did not show glutathione S-transferase (GST) or DDT dehydrochlorinase activity in the presence of glutathione. Thus, it seems most likely that LinA is not a GST-type enzyme. LinA is thought to be a unique dehydrochlorinase, and its mechanism of dehydrochlorination is of great interest. Recently Murzin proposed that LinA is a member of a novel structural superfamily of enzymes, and predicted a 3D structure (AG Murzin, personal communication). This superfamily [38] contains four proteins, scytalone dehydratase, steroid Δ-isomerase, nuclear transport factor-2, and the β-subunit of naphthalene dioxygenase, known structures with different enzymatic activities. We are now trying to elucidate the reaction mechanism of LinA according to his model.

The LinA activity is expressed in strain UT26 grown with no inducer [15]. A sequence (5′-CAGAC-GAAGCTAAATAT-3′) partially homologous to the consensus sequence of the constitutive promoters in *P. putida* (5′-AA-AAATG-TAAATAT-3′) [18] was located between 112 and 127 bp upstream of the initiation codon of the *linA*

---

Genes and enzymes for γ-HCH degradation  
Y Nagata *et al*

gene. Northern blotting analysis of total RNA of strain UT26 using the *linA* gene as a probe revealed one major band of around 690 nucleotides (nt) (Y Nagata, R Imai and M Takagi, unpublished data). The size of the mRNA estimated from the positions of the putative promoter and the terminator sequences was approximately 630 nt [16], and was similar to that of the observed transcript. Primer extension analysis revealed that the adenine at –97 and the thymine at –93 were the initiation sites of the transcription (Y Nagata, R Imai and M Takagi, unpublished data). These results strongly suggest that the sequence homologous to the consensus sequence of the constitutive promoters in *P. putida* is functional for constitutive expression of the *linA* gene in *S. paucimobilis* UT26, and that the *linA* gene is transcribed in a mono-cistronic manner. Furthermore, the G+C contents of the regions upstream and downstream of the putative *linA* promoter and terminator, respectively, are higher than those of the *linA* gene and its flanking regions (Figure 3). It is speculated that the *linA* gene including its promoter and terminator was introduced into the genome of strain UT26 by transfer from another organism with a low G+C content during evolution of this organism in soil.

### 1,4-TCDN halidohydrolase (*LinB*)

The deduced amino acid sequence of LinB shows significant similarity to three types of α/β-hydrolase fold enzymes [54], haloalkane dehalogenase (DhlA) from *Xanthobacter autotrophicus* GJ10 [21], haloacetate dehalogenase (DehH1) from *Moraxella* sp B [25], and serine hydrolases represented by 2-hydroxymuconic semialdehyde hydrolase (DmpD) from *Pseudomonas* sp CF600 [51]. The identities between these enzymes and LinB were 29.3%, 22.6%, and 20.5%, respectively. Renilla-luciferin 2-monooxygenase (LUCI) from *Renilla reniformis* [35] showed the highest identity to LinB (41.2%), but both seem not to be monooxygenases. Conversely, LUCI is considered to be a member of the α/β-hydrolase family, judging from the deduced amino acid sequence. LinB also showed significant similarity with part of the epoxide hydrolases from eukaryotic cells [2, 12, 30]. A novel human serine hydrolase, which showed similarity with bacterial serine hydrolase (BphD, DmpD, TodF, and XylF) has been reported [59]. Evolutionary and functional relationships of these bacterial and eukaryotic hydrolases require further study.

The deduced amino acid sequence of LinB showed the highest level of similarity to haloalkane dehalogenase (DhlA) from *X. autotrophicus* GJ10 [21], suggesting that LinB belongs to the family of haloalkane dehalogenase enzymes catalyzing dehalogenation by a hydrolytic mechanism [43]. We have shown that LinB has broader specificity than Dh1A. For example, 1-chlorodecane and 2-chlorobutane, which are poor substrates for DhlA [27], were good substrates for resting recombinant *E. coli* cells overproducing LinB [43]. Furthermore, the *linB* gene from *S. paucimobilis* UT26 was highly expressed in *E. coli*, and the LinB was purified to homogeneity and characterized [48]. Principal component analysis of substrate activities of various haloalkane dehalogenases suggested that LinB probably constitutes a new substrate specificity class within this group of enzymes [48]. It would be useful to analyze the reaction mechanism of LinB in order to better understand

Genes and enzymes for γ-HCH degradation  
Y Nagata et al

**Figure 3** Restriction map and G+C content of the *linA* gene and its flanking regions. (a) Restriction map of the 5-kb HindIII fragment containing the *linA* gene. The location and direction of *linX*, ORFUP, and *linA* are indicated by the arrows under the restriction map. P<sub>ORFUP</sub> and P<sub>*linA*</sub> indicate the putative promoter sequences (Y Nagata, R Imai and M Takagi, unpublished data). T at the end of the *linA* gene indicates the proposed terminator nucleotide [16]. (b) G+C content distribution diagram of *linA* and its flanking regions in *S. paucimobilis* UT26. The G+C content distribution was visualized using GENETYX-MAC software (version 8.0; Software Development Co., Tokyo, Japan). Average span was 50. The G+C content of the *linA* gene is obviously lower than that of its neighboring regions.

stand the structure-function relationships within halidohydrolases. As described above, LinB is believed to belong to the family of α/β-hydrolases which employ a catalytic triad, i.e., nucleophile-histidine-acid, during the catalytic reaction [54]. The position of the catalytic triad within the sequence of LinB was probed by site-directed mutagenesis. The catalytic triad residues of the haloalkane dehalogenase LinB are proposed to be D108, H272, and E132 [14]. For further analysis, we determined the three-dimensional structure of LinB. The purified LinB was crystallized using the hanging-drop vapor-diffusion method [69]. The crystals diffract to at least 1.60 Å using synchrotron X-ray under cryogenic (100 K) conditions [69]. Recently, the structure of LinB was solved (J Marek, J Vevodov, I Smatanov, LA Svensson, J Newman, Y Nagata, J Damborsky, and M Takagi, unpublished data) by molecular replacement with dehalogenase (DhaA) from *Rhodococcus* [32].

### 2,5-DDOL dehydrogenase (LinC)

The deduced amino acid sequence of LinC showed significant similarity to members of the short-chain alcohol dehydrogenase superfamily [24, 50, 58]. It is thought that there are two highly conserved regions of these enzymes [50, 58]. The first conserved region is located at the N-terminus. The predicted structures of several dehydrogenases indicate that this region consists of alternating regions of β-sheet and α-helix (β-α-β) and that this region has been proposed to be part of the NAD⁺ binding site. The second homologous region can be seen around positions 150 and 154 of the consensus sequence [50]. Two amino acids, tyrosine at position 150 and lysine at position 154, are highly conserved among the members of the short-chain alcohol dehydrogenase family. The tyrosine residue is conserved in all of them, while the lysine residue is not. Ensor and Tai demonstrated that substitution of the tyrosine by alanine abolished the 15-hydroxyprostaglandin dehydrogenase activity [9]. In the deduced amino acid sequence of LinC, the amino acid residues mentioned above are highly conserved, suggesting that LinC has the activity of NAD⁺-dependent dehydrogenation.

Members of the short-chain alcohol dehydrogenase family are distributed from bacteria to humans. The question then arises concerning the substrate specificities of these enzymes. In preliminary studies, we tested whether another short-chain dehydrogenase enzyme has LinC activity (= 2,5-DDOL dehydrogenase activity). Recombinant *E. coli* cells expressing BphB (biphenyl-cis-diol dehydrogenase from *Pseudomonas* sp. KKS102; [11]), or BnzB (cis-benzene glycol dehydrogenase from *Pseudomonas putida* BE-81; [19]), which is also known as TodD (cis-toluene dihydrol diol dehydrogenase from *Pseudomonas putida* F1; [81]) did not show the activity.

We showed that there is a gene in the vicinity of the *linA* gene, named *linX*, encoding a protein which has LinC (2,5-dichloro-2,5-cyclohexadiene-1,4-diol dehydrogenase) activity (Figure 3) [45]. The deduced amino acid sequence

of LinX shows 33.1% identity with that of LinC, and LinX is also a member of the short-chain alcohol dehydrogenase family. The region between *linX* and *linA* was homologous to the *linX* gene at the nucleotide level (69%). An open reading frame of 294 nucleotides was found in this region, and was named ORFUP. ORFUP encodes a 10.1-kDa poly-peptide consisting of 98 amino acid residues. The deduced amino acid sequence of ORFUP showed 70.7% identity with that of LinX for the 58 N-terminal amino acid residues. It is interesting that there is a sequence near the *linA* gene which has the potential to convert one of the intermediate metabolites of γ-HCH. It is possible that strain UT26 is in the process of forming a kind of operon for γ-HCH degradation. This sequence, however, is thought to be non-functional, judging from the following two results (Y Nagata, R Imai and M Takagi, unpublished data). First, the ORFUP is much shorter than LinX (Figure 3). Second, *E. coli* harboring the plasmid overexpressing this region showed no LinC activity and produced a short protein similar in size to the estimated size of the protein product of ORFUP. Only a few point mutations would be necessary to make this sequence functional as a gene for dehydrogenase.

### Enzymes for the downstream pathway

#### 2,5-DCHQ reductive dehalogenase (*LinD*)
The deduced amino acid sequence of LinD shows some similarity to class theta glutathione S-transferases (GSTs). GSTs are categorized into four classes, alpha, beta, pi, and theta, and all known bacterial GSTs are placed in class theta [75]. Unlike other classes of GSTs, there is little information on class theta GSTs concerning their three-dimensional structures and residues necessary for activity except for some cases [60, 62, 74, 76]. PcpC of *Sphingomonas chlorophenolica* ATCC 39723, which is responsible for the conversion of tetrachlorohydroquinone to 2,6-DCHQ [56], is the enzyme most similar to LinD.

Recombinant *E. coli* cells overproducing LinD convert 2,5-DCHQ to HQ via CHQ, although the conversion rate from CHQ to HQ is much lower than that from 2,5-DCHQ to CHQ [37]. The LinD activity of the cell-free crude extract of *E. coli* overproducing LinD rose in the presence of glutathione [37]. Northern blot analysis revealed that the expression of the *linD* gene is induced by 2,5-DCHQ [37].

#### (Chloro) hydroquinone 1,2-dioxygenase (*LinE*)
The *linE* gene, whose product (LinE) is responsible for CHQ degradation, is located 3 kb upstream from the *linD* gene (Figure 2). The deduced amino acid sequence of LinE showed significant similarity to PcpA (51% identity and 72% similarity), which is involved in pentachlorophenol degradation in *S. chlorophenolica* ATCC 39723, although its function is still unknown ([77]; note that PcpA appears as 2,6-dichloro-p-hydroxyquinone chlorohydrolase in the database; see below), and to some ORFs in *Bacillus subtilis* (YkcA, YodE, and YdfO) whose function is also unknown. In addition to these open reading frames, LinE showed a very low level of similarity to meta-cleavage dioxygenases. The alignment between LinE and meta-cleavage dioxygenases is shown in Figure 4. All three amino acids for Fe²⁺ binding [8], which were revealed by

---

Genes and enzymes for γ-HCH degradation  
Y Nagata et al  

the crystallographic analyses of BphC (2,3-dihydroxy biphenyl 1,2-dioxygenases) from *Pseudomonas sp* KKS102 [67] and *Burkholderia cepacia* LB400 [13], are conserved in LinE, suggesting that LinE is a member of this type of dioxygenase.

CHQ and HQ were converted to maleylacetate and γ-HMSA (a slightly yellow compound [46]), respectively by *E. coli* overproducing LinE (K Miyauchi, Y Nagata and M Takagi, unpublished data), indicating that LinE has ring-cleavage dioxygenase activity for CHQ and HQ. Acylchloride, which is proposed to be a direct metabolite of CHQ, is easily converted to maleylacetate in the presence of water. Because the activity of LinE on CHQ seems to be much stronger than that of LinD, it is most likely that CHQ is mainly degraded by LinE and that the degradation pathway via HQ is a minor pathway in strain UT26.

As far as we know, LinE is the first enzyme reported which prefers HQ as its substrate over catechol, which is one of major substrates for meta-cleavage dioxygenases. The crude extract of *E. coli* overproducing LinE consumed oxygen when CHQ or HQ, but not catechol, was added (K Miyauchi, Y Nagata and M Takagi, unpublished data). In contrast to LinE, *E. coli* overproducing a typical ring-cleavage dioxygenase, XylE (catechol 2,3-dioxygenase) [29], did not show activity toward CHQ or HQ. These results show that LinE is a novel type of meta-cleavage dioxygenase, which cleaves the aromatic ring with two hydroxy groups in the para-position. Although PcpA, which shows the highest level of similarity to LinE, appears as 2,6-dichloro-p-hydroxyquinone chlorohydrolase in the database (accession number M55159), there is no direct evidence about it. Conversely, we have some direct evidence that PcpA has ring-cleavage dioxygenase activity toward HQ, CHQ, and 2,6-DCHQ (Y Ohtsubo, K Miyauchi, K Kanda, T Hatta, H Kiyohara, T Senda, Y Nagata, Y Mitsui, and M Takagi, unpublished data).

#### A regulatory factor for the *linD* and *linE* gene expression (*LinR*)
The deduced amino acid sequence of LinR shows similarity to LysR-type transcriptional regulators (LTTRs) [66]. Some LTTRs are involved in the degradation pathways of aromatic compounds such as catechol [63], naphthalene [78], and chlorocatechol [33, 36]. The palindromic TN₁₁A sequence is known as a recognition sequence for LTTRs. We found the palindromic sequence immediately upstream of *linE* (K Miyauchi, Y Nagata and M Takagi, unpublished data). The fragment containing the sequence was ligated with the reporter gene, *lacZ*, and was inserted into the plasmid expressing LinR under the control of a *lac* promoter. When the resultant plasmid was introduced into *E. coli*, the LacZ activity rose in the presence of 2,5-DCHQ or CHQ in the medium (K Miyauchi, Y Nagata and M Takagi, unpublished data). Furthermore, Northern blot analysis for total RNAs of strains UT26 and UT102, a mutant of *linR* [46], revealed that the expression of *linD* and *linE* was induced in the presence of 2,5-DCHQ, CHQ or HQ in strain UT26, but not in strain UT102 (K Miyauchi, Y Nagata and M Takagi, unpublished data). These results indicate that LinR is a positive transcriptional regulator for expression of *linD* and *linE*.

Genes and enzymes for γ-HCH degradation
Y Nagata et al

| 1 | 10 | 20 | 30 | 40 | 51 | 60 | 70 |
| --- | --- | --- | --- | --- | --- | --- | --- |
| LinE_UT26 | MMQLPERVEGLHHITVAT--GSAQGDVDLLVKTLGQRLVKKTMFYDGARPVYHLYFGNELGEPGT---LYTTFPVRQAGYTGK |
| BphC_KKS102 | MSIERLGYLGFAV--KDVPAWDHFLTKSVGMAAGSAG-DAALYRADQRAWRAVQPGEL--DDLAYAGLEVDDAAALE |
| CatE_PSHV3 | MALTVIRPGYVQLRV--LDLDEAIHYRDRIGLNFVNREG-DRAFFQAFDEFDRHSIILREADQAGMDVMGFKVAKDADLD |
| Cdo2_MT15 | MGVMRIGHVNMRV--MDIELAVHHYEMVLGMKKTLEDKDGNVYLKCWDEWKFSLLILTPSDRAGLSHVAYKVERDSLD |
| Cdo_RRCTM | MSNITSDSNIVEVSVPVHNLHHVELLT--PKPNESLDFTRVLGLHETHREG-QSVYLRGSGEWGQYSTILTESPTAGLGHMGWQVAEPEHVQ |
| TbuE_PSP | MGVLRIGMRPVVAGSFGQHHLRQA PRFDLGLQLVEV---GILLDLVGVEVEAGLIRRREDERILVPLVPALEVDVAGV |
| TODE_PSF1 | MSIQRLGYLGFEV--ADVRSWRTFATTRLGMMEASASE-TEATFRIDSRAWRLSVSRGPA--DDYLFAGFEVDSEQGLQ |

| 80 | 90 | 100 | 110 | 120 | 130 | 140 | 150 | 160 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| LinE_UT26 | RGAGQISAVSYNAPVGTLSWWQEHLIKRAVTVSEVERFGQKYLSEHPDCGVGFIEIIEQDTDGQFEPWDSPYVPKEVALRGFHSTAT-LN-- |
| BphC_KKS102 | RMADKLRQAGVAFTRGDEALMQQRKV---MGLLCLQDPFGLPLEIYYGPAEIF----HEPFLPSAPVSG-FVTG---DQGIGHFVRCV---- |
| CatE_PSHV3 | HFTERLLDIGHVVDVIPAGEDPGVG----RKIRFNTPTQHVFELYAEMALSAT---GPAVKNPDVWVVEPRGMR---ATRFDHCALNG---- |
| Cdo2_MT15 | LLKQRIESYGFTDMILPAGEILTMG----RIVRFTIPSGHELRLYAEKECVGT---EVGSRNPDWPDNIRGAG---VKWLDHIALVCELNPE |
| Cdo_RRCTM | GWGRRLKDAGVEHRFEAGGSSFAQG----DTVSEGPYGHKMELFYDERFVP---EDRSKLLSQVKFPTPGIG---ARRLDHLNITA---- |
| TbuE_PSP | VLHRHLHAEHVFVVPHGGGHVHHLQ---AGMLQFNLPSGHEMRLYAMKEVVGT---EVGSRNPDWPDNLKGAG---VHWLDHALLMCELNPE |
| TODE_PSF1 | EVKESLQAHGVTVKVEGGELIAKRGV----LGLISCTDPFGNRVEIYYGATELF----ERPFASPTGVSG-FQTG---DQGLGHYVLSV---- |

| 170 | 180 | 190 | 200 | 210 | 220 | 230 | 240 | 250 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| LinE_UT26 | --RNEEMDSFMRNAWNLKPQGRDGNYQRY-AFGNGGAAKVLDVYIDEDERP GTWAL-GEGQVHHA AFEVADLDVQAALKFDVEGLG-YTDFS D |
| BphC_KKS102 | ---PDTAKAMAFYTEVLGFVLSDIIDIQMGPETSVPAHFLHCNGRHHTIALAAFP I--PKRIHHFMLQANTIDDVGYAFDR LDAAG-RITSLL |
| CatE_PSHV3 | ---VDIASSAKIFVDALDFSVAEELVDETS--GARLGIFLSCSNKAHDVAFLGYPE--DGKIHHTSFNLESWHDVGHAADIISRYDISLDIGP |
| Cdo2_MT15 | AGINHVADNVKFFMGCLDFYLSEQLVGPD-ASTQAVAFLFRATKPHDIAFLPGP---SAGVHHISFFLDSWHDVLKAADVMAKNKVIDLAP |
| Cdo_RRCTM | ---KDVDARNWYSDVLGFKLREALRTPD---GDLGAWMSVSSQVHDMAIMRDGMGEEGRLHHIAYWLETPETLLRAVDTFVEEGTPIDVGP |
| TbuE_PSP | AGVNTVADNTRFMQEVLGFFLTEQVVVGPD-GCVQAAARLARSTTPHDIAFVGGP---R SGLHHIAFFLDSWHDVLKAADVMAKN QTKIDVAP |
| TODE_PSF1 | ---ADVDAALAFYTKALGFQLADVIDWTIGDGLSVTLYFLYCNGRHHSFAFAKLPG--SKRLHHFMLQANGMDDVGLAYDKFDAER-AVVM SL |

| 260 | 270 | 280 | 290 | 300 | 310 |
| --- | --- | --- | --- | --- | --- |
| LinE_UT26 | RKHR-GYFESIYVRTPGG-VLFE ASVTLGFTHD-ESPEKL GSEVKVAPQLEGVKDELLRT-MNDPIVI- : S. paucimobilis UT26 |
| BphC_KKS102 | GRHTNDQTLSFYADTPSPMIEVE FGWGPRTVDS-------SWTVARHSRTAMWGHKSVR-GQR--------: P. sp. strain KKS102 |
| CatE_PSHV3 | TRHGITRQTIYFFDPSG-NRNE T FSG-GYIYY-PDNPQLRWQAENAGK-AIFYYEKAL- NDRFMTVNT : P. sp. strain HV3 |
| Cdo2_MT15 | TRHGITRGETVYFFDPSG-NRNE TFAGLGYLAQ-PDRPVNTWTEDSLWR-GILFHSGEP-YP AFTEVYT : P. putida MT-15 |
| Cdo_RRCTM | GKHGLTQAFYVYVFE PGG-NRVE LF-TGGYPIYGPDWD PVIWEGENMDR-AIVWYGGQL-PDTFFNQAT : R. rhodochrous CTM |
| TbuE_PSP | TRHGITRQTIYFFDPSG-NRNE TFAGLGYLAQ-PDRPVTTWSEDKLWT-GIFYHTGDTLVPSFTDVYT : P. picketti |
| TODE_PSF1 | GRHTNDHMISFYGATPSG-FAVE YGWAREVTR-------HWSV VRYDRISIWGHKFQA-PA--------: P. putida F1 |

Figure 4 Alignment between LinE and meta-cleavage dioxygenases. The residues involved in Fe²⁺ binding in BphC of *Pseudomonas* sp KKS102 are marked with a sharp. The alignment was generated by using the ClustalX sequence alignment program. Abbreviations: BphC__KKS102, 2,3-dihydroxybi-phenyl 1,2-dioxygenase from *Pseudomonas* sp strain KKS102 [28]; CatE__PSHV3, catechol 2,3-dioxygenase from *Sphingomonas* sp strain HV3 [79]; Cdo2__MT15, catechol 2,3-dioxygenase from *P. putida* MT-15 [26]; Cdo__RRCTM, catechol 2,3-dioxygenase from *Rhodococcus rhodochrous* [6]; TbuE__PSP, catechol 2,3-dioxygenase from *Ralstonia pickettii* [31]; TODE__PSF, 3-methylcatechol 2,3-dioxygenase from *P. putida* F1 [81].

Subcellular localization of enzymes for γ-HCH degradation

In order to explore the localization of enzymes for γ-HCH degradation in strain UT26, the cellular proteins were separated into periplasmic, cytosolic and membrane fractions after osmotic shock [49]. Most of the LinA and LinB activities (71% and 83%, respectively) were detected in the periplasmic fraction. For further analysis, antibodies were raised against LinA and LinB using purified enzymes produced in recombinant *E. coli* cells [49]. Western blot analysis of each cell fraction revealed that most of LinA and LinB was present in the periplasmic fraction. LinA and LinB were not detected in the extracellular fraction, indicating that LinA and LinB are not secreted extracellularly [49]. The periplasmic localization of LinA and LinB was confirmed by immunoelectron microscopy [49] (Figure 5). LinA and LinB were almost exclusively detected in the periphery of the cells.

The periplasmic space lies between the inner and outer membranes of Gram-negative bacteria, and is the location of many functions [3, 52, 53]. For example, some proteins residing in the periplasmic space have important functions in detection and processing of essential nutrients and their transport into the cell. Enzymes that detoxify antibiotics, such as β-lactamase, also exist in the periplasmic space [53]. Our findings have added another to the list of reactions that occur in the periplasmic space: degradation of halogenated xenobiotics. Elimination of halogens from halogenated xenobiotic molecules is a key step in their degradation because the carbon-halogen bond is relatively stable [10]. These compounds may enter the periplasm through non-specific porins in the outer membrane. Since dehalogenases degrade complex halogenated molecules into simpler ones for utilization and possibly for detoxification, the localization of dehalogenases in the periplasmic space seems reasonable.

In general, the signal peptide of periplasmic proteins functions in the translocation process [52]. We were surprised to find that these dehalogenases are not subject to N-terminal processing during translocation to the periplasmic space [49]. We suggest that dehalogenases of *S. paucimobilis* UT 26 are exported by a secretion mechanism that differs from the signal peptide-based secretion mechanism that is common in prokaryotes. Two mechanisms have been reported by which proteins that lack a typical N-terminal signal peptide and that are not processed during translocation are secreted [65]. However, in both cases the proteins cross both cell membranes whereas the dehalogenases of *S. paucimobilis* UT 26 are simply exported to the periplasm and are not secreted into the culture medium. A similar finding was made in a study of chitinase produced by *Serratia marcescens* [5]. A novel mechanism for protein

Genes and enzymes for γ-HCH degradation  
Y Nagata et al  

Figure 5 Immunogold-labeling electron microscopy. See Ref [49] for details. (a) Strain UT26 with anti-LinA antibody. (b) Strain UT26 with anti- LinB antibody.

![Diagram](attachment:diagram.png)

Figure 6 Proposed mechanism of adaptive acquisition of assimilation ability for γ-HCH in *S. paucimobilis* UT26. See text for details.

accumulation in the periplasmic space may be one of several protein translocation pathways that operate in Gram-negative bacteria.

### Conclusions and prospects

We have demonstrated a unique degradation pathway of γ-HCH in *S. paucimobilis* UT26 and have cloned genes involved in this pathway. It seems that the pathway from γ-HCH to 2,5-DCHQ (we named it the upstream pathway) is especially specific for γ-HCH degradation in strain UT26. The 2,5-DCHQ degradation pathway (we named it the downstream pathway) seems to be relatively widely distributed in nature. There might be other aerobic γ-HCH degradation pathways which are not yet identified. For example, *Burkholderia cepacia* AC1100 degrades 2,4,5-trichlorophenoxyacetic acid (2,4,5-T) via 2,5-DCHQ. However, the 2,5-DCHQ degradation pathway in AC1100 is different from that in strain UT26 [80]. Comparison of the γ-HCH degradation pathway in strain UT26 with such other pathways in other organisms will be of interest.

We propose a model for adaptive acquisition of the γ-HCH assimilation ability in *S. paucimobilis* UT26 (Figure 6). For the first attack on γ-HCH, strain UT26 seems to use an enzyme coded by a gene that may have originated from another organism (*linA*). Then, strain UT26 degrades the intermediate metabolites, which seem to be relatively common substrates, by developing variants of endogenous genes (*linB* and *linC*). Finally, for the common substrate (2,5-DCHQ), strain UT26 uses established genes which are expressed inducibly.

Our current understanding can be applied for bioremediation of contaminated environments. Strain UT26 grows slowly when γ-HCH is supplied as sole source of carbon.

and energy. Because the genes for γ-HCH degradation which we cloned could be overproduced in other organisms, it may be possible to create strains which degrade γ-HCH more effectively. The introduction of genes for γ-HCH degradation into plants is one possible strategy. In preliminary studies, we introduced the *linA* gene with a CaMV 35S promoter into tobacco by using *Agrobacterium tumefaciens*, and succeeded in expressing LinA activity in the transgenic plant.

The γ-HCH degradation pathway in strain UT 26, however, is inefficient because two dead-end products, 1,2,4-TCB and 2,5-DCP are produced (Figure 1). Enhancement of LinB activity in strain UT 26 or introduction of the *linA* gene into a strain which can assimilate 1,2,4-TCB, such as *Pseudomonas* sp. strain P51 [72], are strategies for creating strains which degrade γ-HCH more effectively than strain UT 26 does.

### Acknowledgements

We thank M. Fukuda, Nagaoka University of Technology, for helpful discussions. This work was supported in part by a Grant-in-Aid for Scientific Research from the Ministry of Education, Science, Sports and Culture of Japan. This work was performed using the facilities of the Biotechnology Research Center, The University of Tokyo.

### References

1. Armengaud J, B Happe and KN Timmis. 1998. Genetic analysis of dioxin dioxygenase of *Sphingomonas* sp. strain RW1: catabolic genes dispersed on the genome. *J. Bacteriol.* **180**:3954–3966.
2. Beetham JK, T Tian and BD Hammock. 1993. cDNA cloning and expression of a soluble epoxide hydrolase from human liver. *Arch. Biochem. Biophys.* **305**:197–201.
3. Beveridge TJ. 1995. The periplasmic space and the periplasm in gram-positive and gram-negative bacteria. *ASM News* **61**:125–130.
4. Blais JM, DW Schindler, DCG Muir, LE Kimpe, DB Donald and B Rosenberg. 1998. Accumulation of persistent organochlorine compounds in mountains of western Canada. *Nature* **395**:585–588.
5. Brurberg MB, VGH Eijsink, AJ Haandrikman, G Venema and IF Nes. 1995. Chitinase B from *Serratia marcescens* BJL200 is exported to the periplasm without processing. *Microbiology* **141**:123–131.
6. Candidus S, KH van Pee and F Lingens. 1994. R. rhodochrous plasmid pTC1 sequence of *cdo* gene for catechol 2,3-dioxygenase. *Microbiology* **140**:321–330.
7. Clark AG and NA Shanaan. 1984. Evidence that DDT-dehydrochlorinase from the house fly is a glutathione S-transferase. *Pesticide Biochem. Physiol.* **22**:249–261.
8. Eltis LD and JT Bolin. 1996. Evolutionary relationships among extradiol dioxygenases. *J. Bacteriol.* **178**:5930–5937.
9. Ensor CM and H-HTai. 1991. Site-directed mutagenesis of the conserved tyrosine 151 of human placental NAD+-dependent 15-hydroxyprostaglandin dehydrogenase yields a catalytically inactive enzyme. *Biochem. Biophys. Res. Comm.* **176**:840–845.
10. Fetzner S and F Lingens. 1994. Bacterial dehalogenases: biochemistry, genetics, and biotechnological applications. *Microbiol. Rev.* **58**:641–685.
11. Fukuda M, Y Yasukochi, Y Kikuchi, Y Nagata, K Kimbara, H Horiu-chi, M Takagi and K Yano. 1994. Identification of the *bphA* and *bphB* genes of *Pseudomonas* sp. strain KKS102 involved in degradation of biphenyl and polychlorinated biphenyls. *Biochem. Biophys. Res. Commun.* **202**:850–856.
12. Grant DF, DH Storms and BD Hammock. 1993. Molecular cloning and expression of Murine liver soluble epoxide hydrolase. *J. Biol. Chem.* **268**:17628–17633.
13. Han S, LD Eltis, KN Timmis, SW Muchmore and JT Bolin. 1995. Crystal structure of the biphenyl-cleaving extradiol dioxygenase from a PCB-degrading pseudomonad. *Science* **270**:976–980.

14. Hynkova K, Y Nagata, M Takagi and J Damborsky. 1999. Identification of the catalytic triad in the haloalkane dehalogenase from *Sphingomonas paucimobilis* UT26. *FEBS Lett.* **446**:177–181.
15. Imai R, Y Nagata, K Senoo, H Wada, M Fukuda, M Takagi and K Yano. 1989. Dehydrochlorination of γ-hexachlorocyclohexane (γ-BHC) by γ-BHC-assimilating *Pseudomonas paucimobilis*. *Agric. Biol. Chem.* **53**:2015–2017.
16. Imai R, Y Nagata, M Fukuda, M Takagi and K Yano. 1991. Molecular cloning of a *Pseudomonas paucimobilis* gene encoding a 17-kilodalton polypeptide that eliminates HCl molecules from γ-hexachlorocyclohexane. *J. Bacteriol.* **173**:6811–6819.
17. Imai R, Y Nagata, M Fukuda, K Yano and M Takagi. 1992. Isolation and characterization of a dehydrochlorinase gene for the degradation of γ-hexachlorocyclohexane in *Pseudomonas paucimobilis*. In: *Pseudomonas*: Molecular Biology and Biotechnology (Galli E, S Silver, and B Witholt, eds), pp 292–300, American Society for Microbiology, Washington, DC.
18. Inouye S, Y Asai, A Nakazawa and T Nakazawa. 1986. Nucleotide sequence of a DNA segment promoting transcription in *Pseudomonas putida*. *J. Bacteriol.* **166**:739–745.
19. Irie S, S Doi, T Yorifuji, M Takagi and K Yano. 1987. Nucleotide sequencing and characterization of the genes encoding benzene oxidation enzymes of *Pseudomonas putida*. *J. Bacteriol.* **169**:5174–5179.
20. Iwata H, S Tanabe, N Sakai and R Tatsukawa. 1993. Distribution of persistent organochlorines in the oceanic air and surface seawater and the role of ocean on their global transport and fate. *Environ. Sci. Technol.* **27**:1080–1098.
21. Janssen DB, F Pries, J van der Ploeg, B Kazemier, P Terpstra and B Witholt. 1989. Cloning of 1,2-dichloroethane degradation genes of *Xanthobacter autotrophicus* GJ10 and expression and sequencing of the *dhlA* gene. *J. Bacteriol.* **171**:6791–6799.
22. Janssen DB, F Pries and JR van der Ploeg. 1994. Genetics and biochemistry of dehalogenating enzymes. *Annu. Rev. Microbiol.* **48**:163–191.
23. Johri AK, MDua, DTuteja, RSaxena, DMSaxena and RLal. 1996. Genetic manipulations of microorganisms for the degradation of hexachlorocyclohexane. *FEMS Microbiol. Rev.* **19**:69–84.
24. Jörnvall H, MPersson and JJeffery. 1981. Alcohol and polyol dehydrogenase are both divided into two protein types, and structural properties cross-relate the different enzyme activities within each type. *Proc. Natl. Acad. Sci. USA* **78**:4226–4230.
25. Kawasaki H, K Tsuda, IMatsushita and KTonomura. 1992. Lack of homology between two haloacetate dehalogenase genes encoded on a plasmid from *Moraxella* sp. strain B. *J. Gen. Microbiol.* **138**:1317–1323.
26. Keil H, MR Lebens and PA Williams. 1985. TOL plasmid pWW15 contains two nonhomologous, independently regulated catechol 2,3-oxygenase genes. *J. Bacteriol.* **163**:248–255.
27. Keuning S, DB Janssen and B Witholt. 1985. Purification and characterization of hydrolytic haloalkane dehalogenase from *Xanthobacter autotrophicus* GJ10. *J. Bacteriol.* **163**:635–639.
28. Kimbara K, THashimoto, MFukuda, TKoana, MTakagi, MOishi and KYano. 1989. Cloning of two tandem genes involved in degradation of 2,3-dihydroxybiphenyl to benzoic acid in the polychlorinated biphenyl-degrading soil bacterium *Pseudomonas* sp. strain KKS102. *J. Bacteriol.* **171**:2740–2747.
29. Kitayama A, TAchioku, TYanagawa, KKanou, MKikuchi, HUeda, ESuzuki, HNishimura, TNagamune and YKawakami. 1996. Cloning and characterization of extradiol aromatic ring-cleavage dioxygenases of *Pseudomonas aeruginosa* JI104. *J. Ferment. Bioeng.* **82**:217–223.
30. Knehr M, H Thomas, M Arand, TGebel, HD Zeller and FOesch. 1993. Isolation and characterization of a cDNA encoding rat liver cytosolic epoxide hydrolase and its functional expression in *Escherichia coli*. *J. Biol. Chem.* **268**:17623–17627.
31. Kukor JJ and RH Olsen. 1996. Catechol 2,3-dioxygenases functional in oxygen-limited (hypoxic) environments. *Appl. Environ. Microbiol.* **62**:1728–1740.
32. Kulakova AN, MJ Larkin and LA Kulakov. 1997. The plasmid-located haloalkane dehalogenase gene from *Rhodococcus rhodochrous* NCIMB 13064. *Microbiology* **143**:109–115.
33. Leveau JHJ and JR van der Meer. 1996. The *tfdR* gene product can successfully take over the role of the insertion element-inactivated *TfdT* protein as a transcriptional activator of the *tfdCDEF* gene cluster, which encodes chlorocatechol degradation in *Ralstonia eutropha*. *JMP* **134** (pJP4). *J. Bacteriol.* **178**:6824–6832.

34 Lipke H and CW Kearns. 1959. DDT dehydrochlorinase. J Biol Chem 234: 2123–2132.
35 Lorenz WW, RO McCann, M Longiaru and MJ Cormier. 1991. Isolation and expression of a cDNA encoding Renilla reniformis luciferase. Proc Natl Acad Sci USA 88: 4438–4442.
36 McFall SM, SA Chugani and AM Chakrabarty. 1998. Transcriptional activation of the catechol and chlorocatechol operons: variations on a theme. Gene 223: 257–267.
37 Miyayuchi K, S-K Suh, Y Nagata and M Takagi. 1998. Cloning and sequencing of a 2,5-dichlorohydroquinone reductive dehalogenase gene whose product is involved in degradation of γ-hexachlorocyclohexane by Sphingomonas paucimobilis. J Bacteriol 180: 1354–1359.
38 Murzin AG. 1998. How far divergent evolution goes in proteins. Curr Opin Struct Biol 8: 380–387.
39 Nagasawa T, H Ohkishi, B Kawasaki, H Yamano, H Hosono, Y Tani and H Yamada. 1982. 3-chloro-β-alanine chloride-lyase (deaminating) of Pseudomonas putida CR 1–1. J Biol Chem 257: 13749–13756.
40 Nagasawa S, R Kikuchi, Y Nagata, M Takagi and M Matsuo. 1993. Stereochemical analysis of γ-HCH degradation by Pseudomonas paucimobilis UT26. Chemosphere 26: 1187–1201.
41 Nagasawa S, R Kikuchi, Y Nagata, M Takagi and M Matsuo. 1993. Aerobic mineralization of γ-HCH by Pseudomonas paucimobilis UT26. Chemosphere 26: 1719–1728.
42 Nagasawa S, R Kikuchi and M Matsuo. 1993. Indirect identification of an unstable intermediate in γ-HCH degradation by Pseudomonas paucimobilis UT26. Chemosphere 26: 2279–2288.
43 Nagata Y, T Nariya, R Ohtomo, M Fukuda, K Yano and M Takagi. 1993. Cloning and sequencing of a dehalogenase gene encoding an enzyme with hydrolase activity involved in the degradation of γ-hexachlorocyclohexane (γ-HCH) in Pseudomonas paucimobilis. J Bacteriol 175: 6403–6410.
44 Nagata Y, T Hatta, R Imai, K Kimbara, M Fukuda, K Yano and M Takagi. 1993. Purification and characterization of γ-hexachlorocyclohexane (γ-HCH) dehydrochlorinase (LinA) from Pseudomonas paucimobilis. Biosci Biotech Biochem 57: 1582–1583.
45 Nagata Y, R Ohtomo, K Miyayuchi, M Fukuda, K Yano and M Takagi. 1994. Cloning and sequencing of a 2,5-dichloro-2,5-cyclohexadiene-1,4-diol dehydrogenase gene involved in the degradation of γ-hexachlorocyclohexane in Pseudomonas paucimobilis. J Bacteriol 176: 3117–3125.
46 Nagata Y, K Miyayuchi, S-K Suh, A Futamura and M Takagi. 1996. Isolation and characterization of Tn5-induced mutants of Sphingomonas paucimobilis defective in 2,5-dichlorohydroquinone degradation. Biosci Biotech Biochem 60: 689–691.
47 Nagata Y, M Fukuda, K Miyayuchi and M Takagi. 1996. Genes and enzymes for γ-hexachlorocyclohexane degradation in Sphingomonas paucimobilis UT26. In: Molecular Biology of Pseudomonads (Nakazawa T, K Furukawa, D Haas and S Silver, eds), pp 58–70, American Society for Microbiology, Washington, DC.
48 Nagata Y, K Miyayuchi, J Damborsky, K Manova, A Ansorgova and M Takagi. 1997. Purification and characterization of haloalkane dehalogenase of a new substrate class from a γ-hexachlorocyclohexane-degrading bacterium, Sphingomonas paucimobilis UT26. Appl Environ Microbiol 63: 3707–3710.
49 Nagata Y, A Futamura, K Miyayuchi and M Takagi. 1999. Two different types of dehalogenase, LinA and LinB, which are involved in the γ-HCH degradation in Sphingomonas paucimobilis UT26, are localized in periplasmic space without molecular processing. J Bacteriol 181: 5409–5413.
50 Neidle E, C Hartnett, LN Ornston, A Bairoch, M Rekik and SHarayama. 1992. Cis-diol dehydrogenase encoded by the TOL pWW0 plasmid xylL gene and the Acinetobacter calcoaceticus chromosomal benD gene are members of the short-chain alcohol dehydrogenase superfamily. Eur J Biochem 204: 113–120.
51 Nordlund I and V Shingler. 1990. Nucleotide sequences of the meta-cleavage pathway enzymes 2-hydroxymuconic semialdehyde dehydrogenase and 2-hydroxymuconic semialdehyde hydrolase from Pseudomonas CF600. Biochim Biophys Acta 1049: 227–230.
52 Oliver DB. 1987. Periplasm and protein secretion. In: Escherichia coli and Salmonella typhimurium: Cellular and Molecular Biology (Neidhardt FC et al, eds), pp 56–69, American Society for Microbiology, Washington, DC.
53 Oliver DB. 1996. Periplasm. In: Escherichia coli and Salmonella

Genes and enzymes for γ-HCH degradation
Y Nagata et al

typhimurium: Cellular and Molecular Biology, 2nd edn (Neidhardt FC et al, eds), pp 88–103, American Society for Microbiology, Washington, DC.
54 Ollis DL, E Cheah, MCygler, BDijkstra, FFrolow, SMFranken, MHarel, SJRemington, ISilman, JSchrag, JLSussman, KHGVerschueren and AGoldman. 1992. The α/β hydrolase fold. Protein Engin 5: 197–211.
55 Orser CS, CCLange, LXun, TCZahrt and BJSchneider. 1993. Cloning, sequencing analysis, and expression of the Flavobacterium pentachlorophenol-4-monooxygenase gene in Escherichia coli. J Bacteriol 175: 411–416.
56 Orser CS, JDutton, CCLange, PJablonski, LXun and MHargis. 1993. Characterization of a Flavobacterium glutathione S-transferase gene involved in reductive dechlorination. J Bacteriol 175: 2640–2644.
57 Palleroni NJ. 1984. Genus I Pseudomonas. In: Bergey's Manual of Systematic Bacteriology, vol 1 (Krieg NR and JG Holt, eds), p 198, The Williams & Wilkins Co, Baltimore.
58 Persson B, MKrook and HJörnvall. 1991. Characteristics of short-chain alcohol dehydrogenases and related enzymes. Eur J Biochem 200: 537–543.
59 Puente XS and CLópez-Otin. 1995. Cloning and expression analysis of a novel human serine hydrolase with sequence similarity to prokaryotic enzymes involved in the degradation of aromatic compounds. J Biol Chem 270: 12926–12932.
60 Reinemer P, LPrade, PHof, TNeuefeind, RHuber, RZettl, KPalme, JSchell, IKoelln, HDBartunik and BBieseler. 1996. Three-dimensional structure of glutathione S-transferase from Arabidopsis thaliana at 2.2 resolution: structural characterization of herbicide-conjugating plant glutathione S-transferases and a novel active site architecture. J Mol Biol 255: 289–309.
61 Romine MF, LCStillwell, KK Wong, SJThurston, ECSisk, CSensen, TGaasterland, JKFredrickson and JDSaffer. 1999. Complete sequence of a 184-kilobase catabolic plasmid from Sphingomonas aromaticivorans F199. J Bacteriol 181: 1585–1602.
62 Rossjohn J, GPolekhina, SCFeil, NAllocati, MMasulli, CDe Illio and MWParker. 1998. A mixed disulfide bond in bacterial glutathione transferase: functional and evolutionary implications. Structure 6: 721–734.
63 Rothmel RK, TL Aldrich, JE Houghton, WMCoco, LNOrnston and AMChakrabarty. 1990. Nucleotide sequencing and characterization of Pseudomonas putida catR: a positive regulator of the catBC operon is a member of the LysR family. J Bacteriol 172: 922–931.
64 Sahu SK, KKPatnaik, MSharmila and NSethunathan. 1990. Degradation of alpha-, beta-, and gamma-hexachlorocyclohexane by a soil bacterium under aerobic conditions. Appl Environ Microbiol 56: 3620–3622.
65 Salmond GPC and PJReeves. 1993. Membrane traffic wardens and protein secretion in gram-negative bacteria. TIBS 18: 7–12.
66 Schell MA. 1993. Molecular biology of the LysR family of transcriptional regulators. Annu Rev Microbiol 47: 597–626.
67 Senda T, KSugiyama, HNarita, TYamamoto, KKimbara, MFukuda, MSato, KYano and YMitsui. 1996. Three-dimensional structures of free form and two substrate complexes of an extradiol ring-cleavage type dioxygenase, the BphC enzyme from Pseudomonas sp strain KKS102. J Mol Biol 255: 735–752.
68 Senoo K and HWada. 1989. Isolation and identification of an aerobic γ-HCH-decomposing bacterium from soil. Soil Sci Plant Nutr 35: 79–87.
69 Smatanov I, YNagata, LASvensson, MTakagi and JMarek. 1999. Crystallization and preliminary X-ray diffraction analysis of haloalkane dehalogenase LinB from Sphingomonas paucimobilis UT26. Acta Crystallographica Section D, Biological Crystallography D55: 1231–1233.
70 Suen WC and JCSpain. 1993. Cloning and characterization of Pseudomonas sp strain DNT genes for 2,4-dinitrotoluene degradation. J Bacteriol 175: 1831–1837.
71 Thomas J-C, FBerger, MJacquier, DBernillon, FBaud-Grasset, NTruffaut, PNormand, TMVogel and PSimonet. 1996. Isolation and characterization of a novel γ-hexachlorocyclohexane-degrading bacterium. J Bacteriol 178: 6049–6055.
72 van der Meer JR, ARWvan Neerven, EJde Vries, WMde Vos and AJBZehnder. 1991. Cloning and characterization of plasmid-encoded genes for the degradation of 1,2-dichloro-, 1,4-dichloro, and 1,2,4-trichlorobenzene of Pseudomonas sp strain P51. J Bacteriol 173: 6–15.

Genes and enzymes for γ-HCH degradation
Y Nagata et al

73 van der Meer JR, WM de Vos, S Harayama and AJB Zehnder. 1992. Molecular mechanisms of genetic adaptation to xenobiotic compounds. Microbiol Rev 56: 677–694.

74 Vuilleumier S and T Leisinger. 1996. Protein engineering studies of dichloromethane dehalogenase/glutathione S-transferase from *Methyl- ophilus* sp strain DM11. Ser12 but not Tyr6 is required for enzyme activity. Eur J Biochem 239: 410–417.

75 Vuilleumier S. 1997. Bacterial glutathione S-transferases: what are they good for? J Bacteriol 179: 1431–41.

76 Wilce MCJ, PG Board, SC Feil and MW Parker. 1995. Crystal structure of a theta-class glutathione transferase. EMBO J 14: 2133–2143.

77 Xun L and CS Orser. 1991. Purification of a *Flavobacterium* pentachlorophenol-induced periplasmic protein (PcpA) and nucleotide sequence of the corresponding gene (*pcpA*). J Bacteriol 173: 2920–2926.

78 You IS, D Ghosal and IC Gunsalus. 1988. Nucleotide sequence of

plasmid NAH7 gene *nahR* and DNA binding of the *nahR* product. J Bacteriol 170: 5409–5415.

79 Yrjala K, L Paulin, S Kilpi and M Romantschuk. 1994. Cloning of *cmpE*, a plasmid-borne catechol 2,3-dioxygenase-encoding gene from the aromatic- and chloroaromatic-degrading *Pseudomonas* sp HV3. Gene 138: 119–121.

80 Zaborina O, DL Daubaras, A Zago, L Xun, K Sido, T Klem, D Nikolic and AM Chakrabarty. 1998. Novel pathway for conversion of chlorohydroxyquinol to maleylacetate in *Burkholderia cepacia* AC1100. J Bacteriol 180: 4667–4675.

81 Zylstra GJ and DT Gibson. 1989. Toluene degradation by *Pseudomonas putida* F1. J Biol Chem 264: 14940–14946.

82 Zylstra GJ and E Kim. 1997. Aromatic hydrocarbon degradation by *Sphingomonas yanoikuyae* B1. J Ind Microbiol Biotechnol 19: 408–414.
