
Chemistry & Biology 8 (2001) 681–700

**Research Paper**

**Insights about the biosynthesis of the avermectin deoxysugar L-oleandrose through heterologous expression of Streptomyces avermitilis deoxysugar genes in Streptomyces lividans**

Sven-Eric Wohlert ${}^{a}$, Natalia Lomovskaya ${}^{a}$, Kerry Kulowski ${}^{a}$, Leonid Fonstein ${}^{a}$, James L. Occi ${}^{b}$, Keith M. Gewain ${}^{b}$, Douglas J. MacNeil ${}^{b}$, C. Richard Hutchinson ${}^{a,c,*}$

${}^{a}$ School of Pharmacy, University of Wisconsin, 425 N. Charter St., Madison, WI 53706, USA  
${}^{b}$ Department of Metabolic Disorders, Merck Research Labs., Rahway, NJ 07065, USA  
${}^{c}$ Department of Bacteriology, University of Wisconsin, 425 N. Charter St., Madison, WI 53706, USA  

Received 24 January 2001; revisions requested 9 March 2001; revisions received 20 April 2001; accepted 27 April 2001  
First published online 7 June 2001

---

### Abstract

#### Background:
The avermectins, produced by *Streptomyces avermitilis*, are potent anthelmintic agents with a polyketide-derived macrolide skeleton linked to a disaccharide composed of two α-linked L-oleandrose units. Eight contiguous genes, *avrBCDEFGHI* (also called *aveBI-BVIII*), are located within the avermectin-producing gene cluster and have previously been mapped to the biosynthesis and attachment of thymidinediphospho-oleandrose to the avermectin aglycone. This gene cassette provides a convenient way to study the biosynthesis of 2,6-dideoxysugars, namely that of L-oleandrose, and to explore ways to alter the biosynthesis and structures of the avermectins by combinatorial biosynthesis.

#### Results:
A *Streptomyces lividans* strain harboring a single plasmid with the *avrBCDEFGHI* genes in which *avrBEDC* and *avrIHGF* were expressed under control of the *actI* and *actIII* promoters, respectively, correctly glycosylated exogenous avermectin Ala aglycone with identical oleandrose units to yield avermectin Ala. Modified versions of this minimal gene set produced novel mono- and disaccharide avermectins. The results provide further insight into the biosynthesis of L-oleandrose.

#### Conclusions:
The plasmid-based reconstruction of the *avr* deoxysugar genes for expression in a heterologous system combined with biotransformation has led to new information about the mechanism of 2,6-deoxysugar biosynthesis. The structures of the di-demethyldeoxysugar avermectins accumulated indicate that in the oleandrose pathway the stereochemistry at C-3 is ultimately determined by the 3-O-methyltransferase and not by the 3-ketoreductase or a possible 3,5-epimerase. The AvrF protein is therefore a 5-epimerase and not a 3,5-epimerase. The ability of the AvrB (mono-)glycosyltransferase to accommodate different deoxysugar intermediates is evident from the structures of the novel avermectins produced.

© 2001 Elsevier Science Ltd. All rights reserved.

Keywords: *avr* Gene; Biotransformation; Deoxysugar biosynthesis; Glycosyltransferase; Methyltransferase; Epimerase; Avermectin; Mycarose; Oleandrose; Combinatorial biosynthesis

---

### 1. Introduction

Microorganisms, particularly Actinomycetes, produce complex natural products that are often decorated with highly modified deoxyhexoses. Some of the most important antibiotic and antitumor agents, such as the erythromycins, vancomycin, and doxorubicin, lack substantial activity without the attached deoxysugars. In contrast, the series of anthelmintic, 16-membered macrolides produced by *Streptomyces hygroscopicus* (milbemycins) and *Streptomyces thermoactaensis* (nemadectins) (Fig. 1) are potent anthelmintic agents whose structures are closely related to the avermectins (*Streptomyces avermitilis*), but do not have sugars attached. Removing the oleandrose disaccharide attached at the C-13 position of avermectin macrolide causes a considerable decrease in activity even though it is not strictly required for activity [1–4]. Recognition of the

* Correspondence: C. Richard Hutchinson. Present address: Kosan Biosciences, Inc., 3832 Bay Center Place, Hayward, CA 94545, USA; E-mail: hutchinson@kosan.com

1074-5521/01/$ – see front matter © 2001 Elsevier Science Ltd. All rights reserved.  
PII: S1074-5521(01)00043-6

the plasmid or be replaced with homologs from other bacteria to survey the possibility of creating analogs of the avermectins or other glycosidic metabolites by combining steps from different deoxysugar biosyntheses. Although it is not known whether avermectins with modified glycosides will have improved properties, the ability to make manifold changes in the deoxysugar portion of these and other macrolide antibiotics would certainly further the quest for new drugs.

To accomplish this goal, the first step was to outline the role for each of the *avr* genes in L-oleandrose biosynthesis, following the suggestions made by the Merck group [1,12] and researchers at the Kitasato Institute [2,13,14]. The biosynthetic pathway of L-oleandrose has a close analogy to that of the structurally related L-mycarose, as found in macrolides like the erythromycins [15–18] and the tylosins [19,20] (Table 1), while L-oleandrose is also found in oleandomycin [21]. As illustrated in Fig. 2, the first two steps in the oleandrose and mycarose pathways involve conversion of glucose-1-phosphate to its TDP form by a thymidylyl transferase (*avrD/aveBII*) and then to TDP-4-keto-6-deoxyglucose by a dehydratase (*avrC/aveBIII*). These two steps are common to the formation of all 6-deoxyhexoses in bacteria [5–8], even though the corresponding genes do not always lie in the same cluster as the rest of the deoxysugar biosynthesis genes for a given secondary metabolite, as in the case of erythromycin. For oleandrose and mycarose, C-2 deoxygenation takes place next—as shown by Floss et al. [22] and Liu et al. [20]—involving *avrG/aveBVI* (encoding a C-2,3-dehydratase) and *avrI/aveBVIII* (encoding a C-3-ketoreductase) or the homologous *eryBVI* and *eryBII* genes.

The proposed biosynthetic pathway for TDP-L-oleandrose then modifies the TDP-2,6-dideoxy-4-ketoglucose by the AvrF/AveBV C-(3)5-epimerase, AvrH/AveBVII C-3-O-methyltransferase, and AvrE/AveBIV C-4-ketoreductase. For TDP-L-mycarose, the *eryBVII* and *eryBIV* genes control the epimerization and ketoreduction steps [15–18], whereas *eryBIII* is believed to encode the C-3-C-methyltransferase [18]. The order in which each of these three reactions occurs is unknown though, as mentioned above, the 2-deoxygenation step follows 4,6-dehydration [20,22].

---

**Fig. 1. Structures of different antibiotic and antitumor agents.**

importance of deoxysugars for the bioactivity of antibiotics and antitumor agents has caused a growing interest in their biosynthesis over the last years [5–8].

Genetic analysis of *S. avermitilis* has revealed eight genes, *avrBCDEFGHI*, that by mutations [1,9–11] and DNA sequence analysis [12,13] have been assigned to the biosynthesis of thymidinediphospho (TDP)-oleandrose and its transfer to avermectin aglycones. Ikeda et al. [13] have recently named these genes as *aveBI–BVIII*, respectively. This cluster of contiguous genes provides a convenient system for studying the biosynthesis of neutral 2,6-dideoxysugars by biotransformation studies in a heterologous host. The cluster can be restructured on self-replicating plasmids and expressed with artificial promoters to investigate the role of the individual genes in oleandrose biosynthesis and glycosylation of the avermectin aglycones. One or more of the genes can be removed from

---

**Table 1**

Functional assignments for genes associated with the biosynthesis of oleandrose in *S. avermitilis* and mycarose in *S. erythraea* and *S. fradiae*

| Gene<sup>a</sup>          | Function<sup>b</sup>                                      | Amino acid sequence comparisons (% similarity/% identity) |
|---------------------------|----------------------------------------------------------|---------------------------------------------------------|
| *avrB/aveBI*              | glycosyltransferase                                      | EryBV (45/21), TylCV (57/39)                             |
| *avrC/aveBII*             | TDP-glucose 4,6-dehydratase                              | EryGdh (83/69)                                           |
| *avrD/aveBIII*            | glucose-1-phosphate-thymidylyltransferase                 | none in erythromycin genes                                |
| *avrE/aveBIV*             | TDP-4-ketohexulose reductase                             | EryBIV (66/50), TylCIV(51/30), TylID (50/30)             |
| *avrF/aveBV*              | TDP-4-keto-6-deoxyglucose 3-epimerase                    | EryBVII (71/55), TylJ (63/46), TylCVII (58/43)           |
| *avrG/aveBVI*             | TDP-4-keto-6-deoxyglucose 2,3-dehydratase                | EryBVI (66/47)                                           |
| *avrH/aveBVII*            | TDP-6-deoxy-L-hexose 3-O-methyltransferase                | EryBIII (51/19), TylE (51/25), TylF (46/20)              |
| *avrI/aveBVIII*           | TDP-4-keto-6-deoxy-L-hexose 3-ketoreductase              | EryBII (73/61), TylCII (73/60)                           |

<sup>a</sup> The genes are listed in the order they occur in the avermectin gene cluster.

<sup>b</sup> As proposed in the references cited in the text.

Research Paper Biosynthesis of the avermectin deoxysugar L-oleandrose S.-E. Wohlert et al. 683

![Diagram](attachment:diagram.png)

Fig. 2. Proposed pathway for the biosynthesis of L-oleandrose and L-mycarose.

We therefore favor the order of reactions as shown in Fig. 2. The same order has also been proposed by Salas and coworkers in their study of the oleandrose biosynthesis and glycosyltransferase genes in the oleandomycin producer [23]. Raynal et al. [18] have proposed a different sequence of reactions for mycarose. They propose that TDP-4-keto-6-deoxyglucose undergoes epimerization by EryBII, followed by 2,3-dehydration (EryBVI) and reduction (EryBIII), C-methyl transfer (EryBIII) and finally reduction of the 4-keto group by EryBIV to give TDP-L-mycarose. Although the TDP group may not be the activating nucleotide in all such pathways [5,6], Schulman and coworkers have unambiguously established, by isolation and bioconversion experiments, that avermectin biosynthesis utilizes TDP-L-oleandrose [24]. Its attachment to the avermectin aglycone is catalyzed by the glycosyltransferase AvrB/AveBI while the genes eryBV and tylCV govern the corresponding steps involving TDP-L-mycarose in the erythromycin [15–18] and tylosin [20] pathways. With this information as a guide, we set out to establish how avr genes could be used to convert avermectin aglycones to both known and novel avermectin disaccharides.

## 2. Results

### 2.1. Restoration of avermectin production in a S. avermitilis avrCDEFGHI deletion strain

The proposed roles of the avrBCDEFGHI genes in avermectin biosynthesis have rested so far mainly on phenotypes of S. avermitilis mutants that produce avermectin aglycones only [4,11,12,25,26] and on the relationship of the deduced sequences of the gene products to those of known deoxysugar biosynthesis enzymes [13] (Table 1). The function of AvrE as a TDP-4-keto-2,6-dideoxyhexose reductase has been independently established through the synthesis of 4′-epidaunorubicin and epirubicin [27].

Our first approach involved creating a single plasmid (pWHM482; Fig. 3B) holding the eight genes – avrBCDEFGHI – and introducing it by transformation into S. avermitilis GG1657, a strain lacking all the genes of L-oleandrose biosynthesis except avrB [9]. The native arrangement of the eight genes [13] suggested that promoters lie between avrD and avrE, downstream of avrI and upstream of avrB. Since the latter promoter could be located upstream of the ORF1 gene adjacent to avrB [13] and to ensure expression of avrB, a fragment containing the 3′-end of ORF1 and the genes avrBCDEFGHI was excised from pVE1190 [25]. AvrB was then orientated downstream of the strong, constitutive ermE* promoter [28] by cloning it in pWHM1251 [27], a high copy number plasmid derived from the Escherichia coli/Streptomyces shuttle vector pWHM3 [29]. In the resulting pWHM482 plasmid avrB should be expressed under the control of ermE*p and the seven remaining avr genes under that of their native promoters, assuming the cloned fragment indeed contains the avrI promoter. (Expression of the avermectin biosynthesis genes in S. avermitilis is believed to be controlled, all or in part, by the pathway-specific avrR gene located at one end of the gene cluster and encoding a putative DNA-binding protein [13], since deletions or transposon insertions into this region abolish avermectin biosynthesis and bioconversion of avermectin aglycones [9,11,13,25].)

S. avermitilis GG1657/pWHM482 was grown on solid growth media and the metabolite profile of the strain, analyzed by thin-layer chromatography and high performance liquid chromatography (HPLC) after extraction of

Table 2  
Plasmids used in this work  

| Plasmid       | Description                                                                                          |
|---------------|------------------------------------------------------------------------------------------------------|
| pVE1190       | 11 kb PstI fragment containing avrBCDEFGHI cloned into pVE1053                                           |
| pWHM465       | PacI-HindIII fragment containing the act promoter region cloned from pRM5 into pNEB193                   |
| pWHM466       | FseI-PacI PCR fragment from pRM5 cloned into pWHM465                                                  |
| pWHM467       | HindIII-EcoRI fragment containing act promoter region cloned from pWHM466 into pRM5                      |
| pWHM468       | polylinker LEH3 cloned into pNEB193 between EcoRI and HindIII sites                                     |
| pWHM469       | 2.1 kb BstEII-KpnI fragment containing avrC and avrD cloned from pVE1190 into pWHM468                     |
| pWHM470       | polylinker LEB4 cloned into pWHM1251 between EcoRI and BamHI sites                                      |
| pWHM471       | 1.1 kb AvrII-PstI fragment containing avrE cloned from pWHM470 into pWHM469                              |
| pWHM472       | 1.3 kb XbaI-AvrII PCR fragment containing avrB cloned into pWHM468                                       |
| pWHM473       | 1.0 kb SstI-PmlI fragment cloned from pVE1190 into pWHM472 to replace central portion of avrB amplicon    |
| pWHM474       | 1.3 kb XbaI-AvrII fragment containing avrB cloned from pWHM473 into pWHM471                              |
| pWHM476       | 1.4 kb XhoI-NcoI and 2.8 kb NcoI-NotI fragments from pVE1190, containing avrF, avrG, avrH, and avrI were cloned into pANT841 |
| pWHM477       | 4.2 kb BglII-SpeI fragment containing avrF, avrG, avrH, and avrI cloned from pWHM476 into pWHM468          |
| pWHM479       | XbaI site was eliminated from pWHM474                                                                |
| pWHM480       | 4.5 kb PacI-EcoRI fragment containing avrB, avrE, avrD, and avrC cloned from pWHM479 into pWHM467          |
| pWHM481       | 4.2 kb XbaI-NsiI fragment containing avrI, avrH, avrG, and avrF cloned from pWHM477 into pWHM480           |
| pWHM482       | 8.7 kb NsiI-HindIII fragment containing part of ORF1 and avrBCDEFGHI cloned from pVE1190 into pWHM1250 at the PstI-HindIII sites |
| pWHM485       | polylinker LSN5 cloned into pGEM-7zf(+) between SphI and NsiI sites                                      |
| pWHM488       | 1.25 kb SpeI-NotI fragment, containing eryBIII cloned from pFL401 into pANT841 between NheI and NotI sites |
| pWHM2100      | 2.1 kb XhoI-MscI fragment cloned from pWHM477 into pSE380                                              |
| pWHM2101      | 2.1 kb HindIII-PstI fragment cloned from pWHM2100 into pWHM468                                          |
| pWHM2102      | pWHM2101 with in-frame deletion in avrH gene between Bpu11021 and StuI sites                             |
| pWHM2103      | 3.1 kb SstI-MscI fragment cloned from pWHM477 into pSE380                                               |
| pWHM2104      | 1.4 kb MscI-XhoI fragment cloned from pWHM2102 into pWHM2103                                            |
| pWHM2105      | 2.4 kb MscI-SacI fragment cloned from pWHM2104 into pWHM477                                             |
| pWHM2106      | 3.5 kb XbaI-NsiI fragment containing the avrFGI and avrH with an in-frame deletion cloned from pWHM210 into pWHM480, containing avrBEDC |
| pWHM2107      | XbaI site was eliminated from pWHM471                                                                 |
| pWHM2108      | 3.25 kb PacI-EcoRI fragment cloned from pWHM2107 into pWHM467                                           |
| pWHM2109      | 4.2 kb XbaI-NsiI fragment containing avrIHGF cloned from pWHM477 into pWHM2108, containing avrEDC          |
| pWHM2110      | In-frame deletion in avrH and avrI between NcoI and MscI sites in pWHM477                                |
| pWHM2111      | 3.2 kb XbaI-NsiI fragment containing avrFG and avrHI with in-frame deletion cloned from pWHM2110 into pWHM480 containing avrBEDC |
| pWHM2129      | In-frame deletion in avrF in pWHM477 between Eco47III and SstI sites                                    |
| pWHM2130      | 3.8 kb XbaI-NsiI fragment containing avrF with in-frame deletion and avrIHG cloned from pWHM2129 into pWHM480, containing avrBEDC |
| pWHM2133      | 3.25 kb PmlI-NsiI fragment, containing avrHGF and avrI with in-frame deletion cloned from pWHM477 into Litmus28 |
| pWHM2134      | 3.25 kb XbaI-NsiI fragment cloned from pWHM2133 into pWHM480, containing avrBEDC                          |
| pWHM2135      | 3.25 kb XbaI-NsiI fragment cloned from pWHM2133 into pWHM2114, containing avrBDC                          |
| pWHM2118      | 1.25 kb HindIII-EcoRI fragment, containing eryBIII cloned from pWHM488 into pWHM485                      |
| pWHM2119      | 1.25 kb PacI-NsiI fragment, containing eryBIII cloned from pWHM2118 into pKOS025-129                      |
| pWHM486       | 1.55 kb SpeI-PstI fragment, containing eryBV gene cloned into pANT841                                    |
| pWHM2120      | 1.55 kb BglII-PstI fragment, containing eryBV cloned from pWHM486 into Litmus28                          |
| pWHM2121      | 1.55 kb XbaI-NsiI fragment, containing eryBV cloned from pWHM2120 into pWHM485                            |
| pWHM2122      | 1.55 kb PacI-NsiI fragment, containing eryBV cloned from pWHM2121 into pKOS025-129                        |

the plates, proved to be identical to that of the wild-type, avermectin-producing strain S. avermitilis MA6593.

2.2. Bioconversion of avermectin Ala aglycone to known avermectins, by recombinant Streptomyces lividans strains hosting plasmid-borne avrBCDEFGHI genes

To avoid working with the rather difficult to transform S. avermitilis, our second approach used S. lividans as a host for the deoxysugar genes. Avermectin Ala aglycone was exogenously provided within the growth media. The eight avr genes were rearranged in two gene cassettes on one plasmid and put under control of the actI and actIII promoters, respectively. To expedite the cloning work, pRM5 [30] was modified as described in Section 5 to create pWHM467, having new cloning sites introduced downstream of each of the act promoters and the actIII and actI-VI genes of pRM5 deleted. The genes avrE and avrB (pWHM480) so that one promoter could control expression of all four of these genes. Genes avrIHGF were cloned into pWHM468 (Table 2) having suitable cloning sites to create pWHM477. Next, avrBEDC from pWHM480 and avrIHGF from pWHM477 were inserted into pWHM467

Research Paper Biosynthesis of the avermectin deoxysugar L-oleandrose S.-E. Wohlert et al. 685

A
![Diagram A](#)
native promoter

B
pWHM482
![Diagram B](#)
native promoter

C
pWHM481
PactIII PactI
![Diagram C1](#)

pWHM2106
![Diagram C2](#)

pWHM2111
![Diagram C3](#)

pWHM2109
![Diagram C4](#)

pWHM2130
![Diagram C5](#)

pWHM2134
![Diagram C6](#)

pWHM2135
![Diagram C7](#)

glycosyltransferase
TDP-4-ketoglucose-
reductase
TDP-4-keto-6-deoxy-
glucose-2,3-dehydratase
TDP-glucose-
4,6-dehydratase
TDP-6-deoxy-hexose-
3-ketoreductase
glucose1-phosphate-
thymidylyltransferase
TDP-4-keto-6-deoxy-
glucose-3-epimerase
TDP-6-deoxy-L-hexose-
5-O-methyltransferase
in-frame deletion
1 kb

Fig. 3. Constructs for expression of the avr L-oleandrose deoxysugar genes. (A) Native arrangement of the avrBCDEFGHI genes. (B) and (C) Structure of plasmid-borne constructs with modified versions of avr L-oleandrose genes used for biotransformation. Filled wedges oriented in the direction of expression indicate the genes, open brackets indicate the absence of the gene in the construct, and open circles indicate the promoter regions.

downstream of the promoters actI and actIII, respectively,
to create pWHM481 (Fig. 3C), which was transformed into S. lividans K4–114 [31].

To verify that avrB is essential for formation and transfer of L-oleandrose to the aglycone in avermectin biosynthesis, we created plasmid pWHM2109 (Fig. 3C), hosting genes avrEDCIHGF but missing avrB. The three genes

avrEDC, which had already been reoriented all in the same direction, were cloned from plasmid pWHM2107 (Table 2) into pWHM467 before adding the avrIHGF cassette in a second step and transforming the resulting plasmid pWHM2109 into S. lividans K4–114.

Bioconversion experiments with the S. lividans K4–114/pWHM481 and S. lividans K4–114/pWHM2109 strains

CHBIOL 108 5-7-01 Cyaan Magenta Geel Zwart

686 Chemistry & Biology 8/7 (2001) 681–700

![Chemical Structures](chemical_structures.png)

Fig. 4. Novel avermectin Ala derivatives from this work.

were carried out in solid growth media, to which purified avermectin Ala aglycone was added, and the extracts of the strains were analyzed by thin-layer chromatography and HPLC. Fermentation of S. lividans K4–114/pWHM481 resulted in the production of avermectin Ala while S. lividans K4–114/pWHM2109 did not show conversion of the aglycone. These results, in combination with those gained from the S. avermitilis GG1657/pWHM482 strain, confirm that the avrBCDEFGHI genes could restore production of TDP-L-oleandrose and support the belief that avrB is the glycosyltransferase responsible for attaching the deoxysugar to the avermectin aglycones.

### 2.3. Bioconversion experiments to identify the functions of individual avrBCDEFGHI genes

Our next goal was to take a closer look at the proposed functions of individual genes involved in the biosynthesis of L-oleandrose. For this we wanted to expand our earlier approach using the L-oleandrose genes in two cassettes on a single plasmid by leaving out, deleting, adding or replacing single or multiple genes. By in-frame deletion we eliminated genes from the avrIHGF cassette while for manipulation of the avrBEDC genes we used cassettes hosting only the genes of interest. The plasmids were transformed

![Bioconversion Pathway](bioconversion_pathway.png)

Fig. 5. (A) Hypothesis for substitution of the function of the AvrI 3-ketoreductase by the 4-ketoreductase AvrE and keto-enol tautomerism. (B) A hypothesis for how the AvrH methyltransferase controls the stereochemistry at C-3.

Research Paper Biosynthesis of the avermectin deoxysugar L-oleandrose S.-E. Wohlert et al. 687

Table 3  
Functional assignments for genes associated with the biosynthesis of oleandrose in *S. avermitilis* and mycarose in *S. erythraea*

| Plasmid and genes | $m/z$ | Possible structures with $m/z$ equals [(M+Na)+]; all modifications within the sugar moieties |
|-------------------|-------|--------------------------------------------------------------------------------------------|
| pWHM482a, avrBCDEFGHI | 909.4 | avermectin Ala (HPLC confirmed) |
|                   | 895.5 | demethyl-aver mectin Ala |
|                   | 881.5 | di-demethyl-aver mectin Ala |
| pWHM481b, avrBEDCIIGF | 909.5 | avermectin Ala (HPLC confirmed) |
|                   | 895.5 | demethyl-aver mectin Ala |
|                   | 881.5 | di-demethyl-aver mectin Ala |
| pWHM2109b, avrBEDCIHGF, ΔavrB | - | - |
| pWHM2106b, avrBEDCIGF, ΔavrH | 881.5 | avermectins Ala1 and Ala2 and at least one more di-demethyl-aver mectin Ala |
| pWHM2111b, avrBEDCGF, ΔavrHI | 751.5 | desoleandro se-demethyl-aver mectin Ala |
|                   | 881.6 | avermectins Ala1 and Ala2 (this work) and at least one more di-demethyl-aver mectin Ala |
| pWHM2134b, avrBEDCHGF, ΔavrI | 751.6 | desoleandro se-demethyl-aver mectin Ala |
|                   | 909.6 | avermectin Ala (HPLC confirmed) |
|                   | 895.6 | demethyl-aver mectin Ala |
|                   | 893.8 | demethyl-4-keto-aver mectin Ala |
|                   | 879.6 | demethyl-(demethyl-4-keto)-aver mectin Ala |
| pWHM2130b, avrBEDCIHG, ΔavrF | 751.6 | avermectin Ala3 |
| pWHM2135b, avrBDCHGF, ΔavrEΔavrI | 909.6 | avermectin Ala (HPLC confirmed) |
|                   | 891.6 | demethyl-di-4-keto-aver mectin Ala |
|                   | 877.6 | di-demethyl-di-4-keto-aver mectin Ala |
|                   | 763.6 | desoleandro se-4-keto-aver mectin Ala |
|                   | 749.6 | desoleandro se-(demethyl-4-keto)-aver mectin Ala |
| pWHM2106+pWHM2119b, avrBEDCIGF, avrΔH+eryBIII | 895.6 | di-demethyl-3′/3″-C-methyl-aver mectin Ala |
|                   | 881.6 | di-demethyl-aver mectin Ala |
|                   | 879.6 | demethyl-(demethyl-4-keto)-aver mectin Ala |
|                   | 751.6 | desoleandro se-demethyl-aver mectin Ala |

- no bioconversion observed and therefore also no mass data.
a Carried in *S. avermitilis* GG1657.
b Carried in *S. lividans* K4-114.

into *S. lividans* K4-114 and once again grown on agly-cone-enriched solid media to trap any produced deoxysugars in avermectin Ala glycosides.

In a first experiment we investigated AvrH, supposed to be a 3-O-methyltransferase. Fermentation of *S. lividans* K4-114/pWHM2106 (ΔavrH; Fig. 3C), which contains only the genes avrIGF and avrBEDC, in the presence of avermectin Ala aglycone led to a number of products with UV-absorption pattern typical for avermectins. Analytical and semi-preparative HPLC followed by mass spectrometry showed that no avermectin Ala was produced but indicated the formation of avermectin Ala derivatives having demethylated mono- and disaccharide moieties (Table 3). With a conversion rate of less than 5% we concentrated on isolating only the main products and were able to characterize the two novel disaccharide avermectins Ala1 and Ala2 (Fig. 4) by mass spectrometry as well as one-and two-dimensional (1D and 2D) proton and carbon nuclear magnetic resonance (NMR) spectroscopy. A third avermectin derivative, having the same mass as the avermectins Ala1 and Ala2, was also isolated but not characterized. The carbon spectra of both compounds showed 47 signals. Chemical shifts and couplings observed in the H, H-COSY, HSQC and HMBC spectra confirmed the structure of the aglycone to be that of avermectin Ala,

accounting for 35 carbon signals. For each compound two anomeric proton signals were identified and using the 2D-NMR spectra a total of six carbon signals could be assigned for each of the sugars. Small coupling constants (3.4, 0.8 and 4.1, 1.0 Hz, respectively) confirmed that both were sugars of the L-series and α-glycosidically linked. Deoxygenated methylene groups were assigned for position 2 of each sugar and CH groups for positions 3, 4 and 5, followed by a 6-methyl group. The coupling patterns and constants for the proton signals at C-2, C-3 and C-4 facilitated the assignment of the configuration for each deoxysugar and compound as shown in Fig. 4. The patterns of the 4-H were particularly helpful; two large axial-axial couplings (around 9 Hz) indicated two axial protons in close proximity (3-H and 5-H) while a large axial-axial and a smaller equatorial-axial one indicated axial (5-H) and equatorial (3-H) neighboring protons. 1D-TOCSY and 1D-NOESY NMR verified the configuration of 3-H of the deoxysugars. The interconnectivities between sugar-aglycone and sugar-sugar were determined for avermectin Ala1 by couplings observed in the HMBC and 2D-NOESY spectrum. For avermectin Ala2 the sugar-sugar connection was concluded from the 2D-NOESY spectrum while the interglycosidic linkage followed from the chemical shifts. The structures of the two compounds

showed that the hydroxyl groups in position 3 of the deoxysugars in both cases had not been methylated, confirming that *avrH* encodes a 3-*O*-methyltransferase. Avermectin Ala1 has L-olivose (demethylated L-oleandrose) attached to the aglycone, followed by a 2,6-dideoxy-L-ribo-hexopyranose (L-digitoxose) that structurally differs from L-olivose in its inverted configuration of the C-3 stereocenter. In avermectin Ala2 this sugar is directly attached to the aglycone, with L-olivose completing the disaccharide moiety (Fig. 4).

For a second and third experiment we created plasmids hosting the genes *avrBEDCHGF* (pWHM2134; Δ*avrI*) and *avrBEDCGF* (pWHM2111; Δ*avrHI*), respectively. pWHM2134 is missing the *avrI* 3-ketoreductase gene supposed to be involved in the 2-deoxygenation of the deoxysugar, while pWHM2111 has the genes encoding the 3-ketoreductase and the 3-*O*-methyltransferase disrupted (Fig. 3C). HPLC analysis and mass spectrometry of the metabolites produced by *S. lividans* K4–114/pWHM2134 showed that avermectin Ala was still being made as well as avermectin Ala derivatives with mono- or di-demethylated disaccharides (Table 3). *S. lividans* K4–114/pWHM2111 (Δ*avrHI*) gave the same overall metabolic profile as *S. lividans* K4–114/pWHM2106 (Δ*avrH*), producing avermectins Ala1 and Ala2. These results, which mirror those observed for the L-mycarose pathway in *Saccharopolyspora erythraea* [18], let us conclude that the function of AvrI as a 3-ketoreductase in the 2-deoxygenation steps of the deoxysugar biosynthesis can be either replaced by a similar ketoreductase from *S. lividans* or be substituted by the 4-ketoreductase AvrE after keto-enol tautomerism (Fig. 5A). In the proposed pathway the 3-keto group caused by the 2-deoxygenation would be reduced by the AvrI 3-ketoreductase, followed by epimerisation, methyl transfer and finally the reduction of the 4-keto group by AvrE. However, the 3-keto group could, by keto-enol tautomerism, be moved into the 4-position. Reduction of the 4-keto group by AvrE would now make the AvrI 3-ketoreductase redundant.

To take a closer look at the epimerization step that is responsible for switching from the D- into the L-series of sugars in this pathway, we constructed pWHM2130, having the proposed epimerase gene *avrF* removed by in-frame deletion (Fig. 3C). HPLC analysis of the fermentation extracts of *S. lividans* K4–114/pWHM2130 showed only one compound with a typical avermectin UV-absorption spectrum. Mass spectrometry data indicated an avermectin with a demethylated monosaccharide moiety and by 1D- and 2D-NMR the compound was characterized to be avermectin Ala3, with D-olivose as the deoxysugar (Fig. 4). Thirty-five of the 41 carbon signals observed could again be assigned to the avermectin Ala aglycone. The deoxysugar protons 3, 4 and 5 were all observed to be in axial configuration, while the larger axial–axial and smaller equatorial–axial coupling pattern for the single anomeric proton (10.0 and 1.7 Hz), caused by the neighboring methylene group, proved this sugar to be D-olivose, β-glycosidically linked to the aglycone. The only alternative structure for a deoxysugar having the same characteristic signals and patterns would have been L-olivose, also β-glycosidically connected. However, this would raise the question why L-olivose should have been linked β-glycosidically here when L-sugars in nature are normally found to be linked α-glycosidically. Avermectins Ala1 and Ala2 (Fig. 4) have L-olivose α-glycosidically connected to the aglycone. Having D-olivose attached to the aglycone proves that AvrF is indeed the epimerase of the L-oleandrose biosynthetic pathway, but acts as a 5- and not a 3,5-epimerase, as explained further below. The attachment of D-olivose instead of L-oleandrose to the aglycone also shows that the AvrB glycosyltransferase is relatively flexible towards its deoxysugar substrates, as has already been observed for a number of glycosyltransferases from other pathways [32–38]. In our case, this flexibility finds a limit in the inability to interconnect two D-olivose molecules, which confirms the report by Schulman et al. [24] that two monosaccharides and not a disaccharide are transferred consecutively to the avermectin aglycone.

An interesting aspect of avermectin Ala3 is the equatorial R-configuration of the C-4′ stereocenter of D-olivose. In the native biosynthetic pathway AvrE, the 4-ketoreductase, is thought to catalyze the last step before attaching the sugar to the aglycone, which creates the equatorial S-configuration at position 4 as in avermectin Ala (Fig. 1). This can be explained in two ways: (i) AvrE is rather flexible, having its 4-keto sugar substrate located in such a way that the resulting 4-hydroxyl group is always found in an equatorial configuration. (ii) AvrE is not flexible and a 4-ketoreductase from *S. lividans* substitutes for AvrE, reducing the 4-keto group into the configuration found.

Although we do not have conclusive support for either hypothesis we favor explanation (ii), since the AvrB glycosyltransferase seems to tolerate both equatorial and axial hydroxyl groups in position 3 (see avermectins Ala1 and Ala2). We found only avermectin Ala3 but no avermectin glycosylated with D-oliose (β-D-oliose differs from D-olivose and D-oleandrose by having the 4-hydroxyl group in the axial instead of equatorial position). This may indicate that the flexibility of AvrB is limited to only the three positions in L-deoxysugars. However, in another experiment where we removed both *avrI* and *avrE* from our eight-member gene set (plasmid pWHM2135; Fig. 3C), HPLC and mass spectrometry analysis of fermentation extracts from *S. lividans* K4–114/pWHM2135 indicated the production of several derivatives with methylated and demethylated mono- and disaccharides attached to the avermectin Ala aglycone (Table 5). These compounds were not further characterized due to low conversion rates. We also observed production of avermectin Ala in low yield. These results show that the functions of AvrE and AvrI can be substituted by

similar enzymes from *S. lividans*, as already observed for a related pathway [18].

Further information about how the stereochemistry at C-3 in the L-oleandrose pathway is controlled and the function of the AvrF epimerase comes from some of our earlier experiments above. Both the ΔavrH and ΔavrHI mutant strains produced avermectins having sugar moieties with a 3-R- or 3-S-configuration, respectively. The AvrE 4-ketoreductase and the AvrF epimerase were still present in both cases. Tight control by either AvrI or AvrF over the C-3 stereochemistry should have resulted in deoxysugars with 3-S-configuration only and not a mixture of C-3 isomers as observed. This implies that the AvrH methyltransferase only methylates an equatorial hydroxyl group at C-3. In the normal pathway any deoxysugar intermediates with initially axial C-3 hydroxyl groups would not be methylated until keto-enol tautomerism causes isomerisation of the hydroxyl group (Fig. 5B). This idea also does not contradict the fact that avermectin Ala3 is the only product of *S. lividans* K4–114/pWHM2130 (ΔavrF) without traces of a second (or more) products having the corresponding D-sugar with an axial instead of an equatorial hydroxyl group at C-3. Our data show that the AvrB glycosyltransferase has a limited ability to handle D-deoxysugars and might not be able to attach two D-sugars to the aglycone.

We conclude from these results that, regardless of the actual sequence of individual steps in the deoxysugar pathway, the configuration of C-3 is ultimately determined by the AvrH methyltransferase and not by the AvrF epimerase nor the AvrI 3-ketoreductase. The results also show that AvrF is a 5-epimerase and not a 3,5-epimerase.

### 2.4. Bioconversion of avermectin Ala aglycone to novel avermectins by combining genes from different deoxysugar biosynthetic pathways in *S. lividans*

The above results show that different deoxysugars can be attached to the avermectin Ala aglycone by modifying the biosynthesis of L-oleandrose. We next wanted to extend this approach by combining different deoxysugar pathways with one another. We decided to combine the avr genes with eryB genes from the closely related mycarose pathway of erythromycin biosynthesis in *S. erythraea*. The proposed 3-C-methyltransferase encoded by eryBIll was chosen for our initial experiments.

Co-transformation of *S. lividans* K4–114 with plasmids pWHM2106 (avrBEDCIGF, ΔavrH) and pWHM2119 (eryBIll; Table 2) could potentially lead to an avermectin Ala derivative having one or two L-mycarose molecules attached. A bioconversion experiment with the resulting strain gave only very low conversion rates (around 0.1%) to what appeared to be glycosylated avermectin compounds. We did not isolate sufficient amounts of any of these metabolites for NMR analysis, but mass spectrometry data indicated the production of an avermectin derivative with two 2,6-dideoxyhexoses, both bearing additional methyl groups. Since we knew from our earlier experiment that the gene for the 3-O-methyltransferase, avrH, had been successfully disrupted in plasmid pWHM2106, any additional methyl groups had to result from a 3-C-methyltransferase by EryBIll. Therefore, the result is consistent with the successful combination of avr and eryBIll genes.

## 3. Discussion

We have examined the possibilities of using single and dual plasmid systems for expression of deoxysugar biosynthesis genes in a heterologous host to both verify the function of genes that had been assigned on the basis of sequence analysis and to produce novel avermectin glycosides. The 2,6-dideoxysugar L-oleandrose was chosen for our experiments as the individual steps in its biosynthesis are predictable and the rearrangement of the eight genes involved in its biosynthesis into two gene cassettes offered a convenient way to construct various expression plasmids. The inherent risk is that changes in gene expression caused by reorganization of some of the avrBCDEF-GHI genes into artificial operons, or effects of the heterologous host on enzyme activity, would result in low product yields in the bioconversion experiments. Nonetheless, the results obtained prove that the approach, which builds upon our recent study of the formation anthracycline glycosides by the same method [39], is overall a useful way to produce both known and novel glycosides.

We were able to both restore avermectin production in a *S. avermitilis* mutant missing all genes required for the biosynthesis of L-oleandrose and produce avermectin Ala in *S. lividans* using exogenously supplied aglycone. Modifications of the plasmid-borne avr genes also enabled us to verify and decode the function of several individual steps and enzymes in the biosynthesis of L-oleandrose, and led to the production of three novel avermectins, including the first avermectin with a D-deoxysugar attached to its aglycone. (Although rare, there are at least two other reports where glycosyltransferases recognize both D- and L-deoxysugars [34,38].) Our results confirm that AvrH is the 3-O-methyltransferase of the L-oleandrose biosynthetic pathway and ultimately determines the configuration of the C-3 stereocenter. AvrF was shown to be not a 3,5-but a 5-epimerase only. On the basis that AvrE and AvrI are indeed (3- and 4-)ketoreductases, respectively, as predicted by gene sequence homologies, we further conclude that similar enzymes from *S. lividans* can substitute for these two enzymes.

The results also suggest a degree of both substrate flexibility and inflexibility for the AvrB glycosyltransferase. AvrB seems to attach and interconnect deoxysugars that differ from its natural substrate by missing the 3-O-methyl group, having inverted configuration at the C-3 stereocenter.

Table 4  
<sup>13</sup>C NMR data of the novel avermectins Ala1, Ala2, Ala3 (relative to TMS, * in methanol-d<sub>4</sub>, # in chloroform-d<sub>1</sub>, 125.9 MHz, δ, multiplicity (DEPT))

| Ala1* | Ala2* | Ala2# | Ala3* |
|-------|-------|-------|-------|
| 1     | 173.0, s | 173.0, s | 173.0, s | 173.8, s |
| 2     | 46.8, d | 46.8, d | 45.6, d | 46.9, d |
| 3     | 121.3, d | 121.3, d | 118.3, d | 121.2, d |
| 4     | 134.5, s | 134.8, s | 136.0, s | 135.2, s |
| 4a    | 19.9, q | 19.9, q | 19.9, q | 19.9, q |
| 5     | 78.3, d | 78.3, d | 77.9, d | 78.2, d |
| 5a    | 57.8, q | 57.7, q | 57.8, q | 55.7, q |
| 6     | 79.4, d | 79.4, d | 78.0, d | 79.4, d |
| 7     | 83.0, s | 82.0, s | 80.5, s | 81.1, s |
| 8     | 141.0, s | 141.0, s | 140.5, s | 141.2, s |
| 8a    | 68.7, t | 68.7, t | 68.0, t | 68.7, t |
| 9     | 121.8, d | 121.8, d | 119.5, d | 121.8, d |
| 10    | 126.5, d | 126.4, d | 125.5, d | 126.2, d |
| 11    | 138.5, d | 138.4, d | 136.4, d | 139.0, d |
| 12    | 41.0, d | 41.0, d | 39.5, d | 40.9, d |
| 12a   | 20.9, q | 20.6, q | 20.3, q | 20.0, q |
| 13    | 83.9, d | 83.5, d | 82.0, d | 84.0, d |
| 14    | 136.5, s | 136.6, s | 134.5, s | 137.2, s |
| 14a   | 15.2, q | 15.1, q | 15.1, q | 15.0, q |
| 15    | 119.8, d | 119.6, d | 119.4, d | 119.2, d |
| 16    | 35.0, t | 35.1, t | 34.3, t | 35.1, t |
| 17    | 70.2, d | 70.0, d | 68.0, d | 70.1, d |
| 18    | 37.6, t | 37.6, t | 36.6, t | 37.5, t |
| 19    | 70.0, d | 69.8, d | 68.0, d | 69.9, d |
| 20    | 42.0, t | 42.0, t | 40.5, t | 42.0, t |
| 21    | 97.3, s | 97.3, s | 95.8, s | 97.3, s |
| 22    | 136.5, d | 136.5, d | 136.3, d | 136.6, d |
| 23    | 129.3, d | 129.3, d | 127.8, d | 129.2, d |
| 24    | 31.8, d | 31.8, d | 30.6, d | 31.8, d |
| 24a   | 16.8, q | 16.8, q | 16.4, q | 16.8, q |
| 25    | 75.9, d | 75.9, d | 74.8, d | 75.9, d |
| 26    | 36.5, d | 36.5, d | 35.3, d | 36.6, d |
| 26a   | 13.5, q | 13.5, q | 13.0, q | 13.5, q |
| 27    | 28.6, t | 28.6, t | 27.5, t | 28.5, t |
| 28    | 12.6, q | 12.5, q | 12.1, q | 12.5, q |
| 1'    | 96.2, d | 95.9, d | 95.0, d | 99.1, d |
| 2'    | 39.5, t | 36.1, t | 35.2, t | 40.8, t |
| 3'    | 70.3, d | 63.6, d | 62.7, d | 72.5, d |
| 4'    | 84.3, d | 76.6, d | 75.1, d | 78.7, d |
| 5'    | 68.7, d | 64.3, d | 62.7, d | 73.2, d |
| 6'    | 18.7, q | 18.1, q | 18.2, q | 17.2, q |
| 1"    | 99.4, d | 94.3, d | 92.4, d | — |
| 2"    | 36.2, t | 38.9, t | 37.7, t | — |
| 3"    | 68.2, d | 69.7, d | 69.3, d | — |
| 4"    | 74.1, d | 78.8, d | 78.4, d | — |
| 5"    | 65.9, d | 69.8, d | 68.0, d | — |
| 6"    | 18.1, q | 18.6, q | 17.6, q | — |

Table 5  
¹H NMR data of the novel avermectins Ala1, Ala2, Ala3 (relative to TMS, *in methanol-d₄, # in chloroform-d₁, 500.6 MHz, δ)

| Ala1* | Ala2* | Ala2# | Ala3* |
|-------|-------|-------|-------|
| 2     | 3.22, m | 3.22, m | 3.31, m | 3.21, m |
| 3     | 5.47, m | 5.47, m | 5.38, m | 5.47, m |
| 4a    | 1.79, s | 1.78, s | 1.81, s | 1.78, s |
| 5     | 3.95ᵃ, m | 3.93ᵃ, m | 3.98, m | 3.89ᵃ, m |
| 5a    | 3.46, s | 3.46, s | 3.50, s | 3.46, s |
| 6     | 3.95ᵃ, m | 3.93ᵃ, m | 4.03, m | 3.89ᵃ, m |
| 8a    | 4.61, dd, 14.3, 2.2 | 4.62, dd, 14.4, 2.3 | 4.62, dd, 14.6, 2.1 | 4.60, dd, 14.5, 2.1 |
|       | 4.65, dd, 14.3, 2.2 | 4.64, dd, 14.4, 2.3 | 4.64, dd, 14.6, 2.1 | 4.64, dd, 14.5, 2.1 |
| 9     | 5.82, ddd, 11.3, 2.2, 2.2 | 5.82, ddd, 11.1, 2.3, 2.3 | 5.81, ddd, 11.0, 2.1, 2.1 | 5.80, ddd, 11.2, 2.1, 2.1 |
| 10    | 5.89, dd, 14.8, 11.3 | 5.89, dd, 14.9, 11.1 | 5.76, dd, 14.1, 11.0 | 5.86, dd, 14.5, 11.2 |
| 11    | 5.72, dd, 14.8, 9.8 | 5.79, dd, 14.9, 9.8 | 5.62, dd, 14.1, 9.8 | 5.65, dd, 14.5, 9.9 |
| 12    | 2.65, ddq, 9.8, 6.9, 2.5 | 2.65, ddq, 9.8, 6.9, 2.9 | 2.65, ddq, 9.8, 6.9, 2.9 | 2.60, ddq, 9.9, 6.8, 3.1 |
| 12a   | 1.16, d, 6.9 | 1.20, d, 6.9 | 1.20, d, 6.9 | 1.12, d, 6.8 |
| 13    | 3.99, sbr | 4.05, sbr | 4.05, sbr | 4.09, sbr |
| 14a   | 1.54, s | 1.54, s | 1.51, s | 1.53, s |
| 15    | 5.15, dd, 8.0, 4.9 | 5.22, dd, 9.0, 4.7 | 5.05, dd, 10.4, 4.7 | 5.24, dd, 10.7, 4.4 |
| 16    | 2.27, m | 2.28, m | 2.29, m | 2.26, m |
| 17    | 3.89, m | 3.89, m | 3.86, m | 3.89, m |
| 18e   | 1.93, ddd, 14.3, 4.2, 4.2 | 1.96, m | 1.74, m | 1.92, ddd, 12.7, 4.5, 4.5 |
| 18a   | 0.85, m | 0.84, m | 0.88, m | 0.86, m |
| 19    | 5.05, dddd, 11.3, 11.3, 4.2, 4.2 | 5.05, dddd, 11.2, 11.2, 4.6, 4.6 | 5.36, m | 5.04, dddd, 11.8, 11.8, 4.5, 4.5 |
| 20e   | 2.20, ddd, 12.0, 4.2, 1.1 | 2.17, m | 2.03, ddd, 12.0, 4.7, 1.6 | 2.19, ddd, 11.8, 4.5, 1.6 |
| 20a   | 1.36, m | 1.35, m | 1.48, m | 1.35, m |
| 22    | 5.75, dd, 10.0, 1.4 | 5.74, dd, 9.9, 1.7 | 5.77, dd, 10.0, 1.6 | 5.75, dd, 10.0, 1.6 |
| 23    | 5.54, dd, 10.0, 2.6 | 5.54, dd, 9.9, 2.6 | 5.55, dd, 10.0, 2.7 | 5.54, dd, 10.0, 2.5 |
| 24    | 2.29, m | 2.29, m | 2.28, m | 2.29, m |
| 24a   | 0.92, d, 7.2 | 0.92, d, 7.4 | 0.91, d, 6.9 | 0.92, d, 7.2 |
| 25    | 3.48, m | 3.48, m | 3.47, m | 3.46, m |
| 26    | 1.63, m | 1.63, m | 1.60, m | 1.65, m |
| 26a   | 0.94, d, 7.0 | 0.94, d, 6.9 | 0.93, d, 7.3 | 0.94, d, 6.7 |
| 27    | 1.48–1.54, m | 1.47–1.54, m | 1.48–1.54, m | 1.48–1.58, m |
| 28    | 0.98, t, 7.1 | 0.98, t, 7.4 | 0.95, t, 7.1 | 0.99, t, 7.3 |
| 1′    | 4.75, dd, 3.4, 0.8 | 4.75, dd, 3.5, 0.9 | 4.83, dd, 3.4, 0.8 | 4.38, dd, 10.0, 1.7 |
| 2′e   | 2.08, ddd, 12.9, 5.0, 0.8 | 2.15, m | 2.14, ddd, 14.3, 3.2, 0.8 | 2.14, ddd, 12.1, 5.2, 1.7 |
| 2′a   | 1.70, m | 1.98, ddd, 14.7, 3.5, 3.5 | 1.92, ddd, 14.3, 3.4, 3.2 | 1.60, m |
| 3′    | 4.02, m | 4.25, m | 4.27, ddd, 3.2, 3.2, 2.9 | 3.48, m |
| 4′    | 3.12, dd, 9.0, 9.0 | 3.30ᵇ | 3.29, dd, 9.0, 2.9 | 2.91, dd, 9.0, 9.0 |
| 5′    | 3.89, dq, 9.0, 6.2 | 4.29, dq, 9.1, 6.2 | 4.19, dq, 9.0, 6.4 | 3.14, dq, 9.0, 6.2 |
| 6′    | 1.28, d, 6.2 | 1.23, d, 6.2 | 1.27, d, 6.4 | 1.26, d, 6.2 |
| 1″    | 5.28, dd, 4.1, 1.0 | 5.10, dd, 4.0, 0.9 | 5.18, dd, 3.7, 0.8 | – |
| 2″e   | 2.21, m | 2.18, ddd, 14.2, 11.2, 4.0 | 2.23, ddd, 13.1, 5.0, 0.8 | – |
| 2″a   | 1.93, m | 1.62, ddd, 14.2, 5.1, 0.9 | 1.71, m | – |
| 3″    | 3.94, m | 3.79, ddd, 11.2, 9.1, 5.1 | 3.95, m | – |
| 4″    | 3.19, dd, 9.2, 3.2 | 2.95, dd, 9.1, 9.1 | 3.19, dd, 9.2, 9.2 | – |
| 5″    | 4.08, dq, 9.2, 6.2 | 3.65, dq, 9.1, 6.2 | 3.67, dd, 9.2, 6.4 | – |
| 6″    | 1.23, d, 6.2 | 1.24, d, 6.2 | 1.30, d, 6.4 | – |

a Assignment interchangeable.  
b Only indirectly observed in 2D experiments due to overlapping with solvent.

(FMC Bioproducts, Rockland, ME, USA). Sequencing reactions using reverse and forward M13 primers were performed with an ABI Prism Big Dye terminator cycle sequencing kit at the University of Wisconsin Biotechnology Center. To introduce unique FseI, NsiI, and XbaI restriction sites downstream of the *actIII* promoter in pRM 5, two oligonucleotides, PM1 and PO2, were designed for replacement of the original plasmid DNA. The sequence of the original and new DNA is as follows. Original DNA: 5′-TCCTGCGTGGCCATGTGTTCCcctccCTGCCTCG-TGGTCC-3′; PM1: 5′-TCCTGGCCGGCCATGCA TTCcCcCTG TCTAGAGGTCC-3′. Changes are shown in bold italic with the *actIII* ribosome-binding site (rbs) in lowercase. The second oligonucleotide, PO2, was distal to the PacI site in pRM 5 downstream of the left promoter to allow PacI digestion of the PCR product. The sequence of this primer was identical to that of pRM5: 5′-CCCAGTCACGACGTTGTAAAACGACGGGCC-3′.

A new polylinker, LEH 3, for pNEB 193 was created as a synthetic oligonucleotide containing rbs and the following internal sites: BstEII, KpnI, PstI, AvrII, XbaI, PacI, BglII, SpeI, NsiI. Top strand: 5′-AATTCCGTAACC GGTACCCTGCAGCCTA-GGTCTAGAGGAGGATTAATAAGATCTACTAGTATG-CATA-3′; bottom strand: 3′-GCCATTGGCCATGGGAC-GTCGGATCCAGATCTCCTCCTAATTAATTTCTAGATGA-TCATACGTATTCGA-5′. The LEH 3 polylinker was used to replace the original polylinker of pNEB 193 by cloning into the EcoRI and HindIII sites to create pWHM 468. Similarly, a synthetic DNA linker, LEB 4, containing rbs and the internal sites XbaI and AvrII were cloned between the EcoRI and BamHI sites of the polylinker of pWHM 1251 to create pWHM 472; top strand: 5′-AATTCTCTAGACCTAGGGGAGGAG-3′; bottom strand: 3′-GAGATCTGGATCCCCCTCCTCCTAG-5′. A new polylinker, LSN 5, for pGEM-7zf(+) was created as a synthetic oligonucleotide containing rbs and the following internal sites: PacI, HindIII, MscI, EcoRI, XbaI, BglII, and PstI. Top strand: 5′-CTTAATTAAGGAGGAAAGCTTTGGCCAGAATTCTCT-AGAAGATCTCTGCAGATGCA-3′; bottom strand: 3′-GT-ACGAATTAATTCCTCCTTTCGAAACCGGTCTTAAGAGA-TCTTCTAGAGACGTCT-5′. The LSN 5 polylinker was used to replace the original polylinker of pGEM-7zf(+) by cloning into SphI and NsiI sites to create pWHM 485.

To create plasmid pWHM472a 1.3kb PCR amplicon for *avrB* with XbaI-AvrII sites was made (Easy Start Kit, Molecular Bio-Products, Inc.). The following two primers were used: PO6: 5′-TTTTCTAGACAGTGAGAGATGTCAGATCA-3′ and PO7: 5′-TTTCCTAGGAACCCTGTGGGAGCCACTCA-3′ to introduce the needed XbaI and AvrII sites. Amplification was made using the Easy Start Kit. The reaction mixture was incubated at 94°C for 2min and then 30 cycles of denaturing (94°C, 1min), annealing (55°C, 1.5min) and elongation (70°C, 2min) followed by a cycle of elongation (70°C, 10min).

### 5.4. Modification of vector pRM 5 for in vivo expression of avermectin L-oleandrose genes

The shuttle vector pRM 5 has been used for expressing polyketide synthase genes in *Streptomyces coelicolor* strains that lack the

---

Table 6  
HMBC couplings of avermectin Ala1 in methanol-d4 (δ, relative to TMS)

| Position | ¹H   | ¹³C  | Position |
|----------|------|------|----------|
| 4a       | 1.79 | 78.3 | 5/6      |
|          |      | 121.3| 3        |
|          |      | 134.5| 4        |
| 5a       | 3.46 | 78.3 | 5/6      |
| 5/6      | 3.95 | 46.8 | 2        |
| 12a      | 1.16 | 41.0 | 12       |
|          |      | 83.9 | 13       |
|          |      | 138.5| 11       |
| 14a      | 1.54 | 83.9 | 13       |
|          |      | 119.8| 15       |
|          |      | 136.4| 14       |
| 18a      | 0.85 | 70.2 | 17       |
| 24a      | 0.92 | 31.8 | 24       |
|          |      | 75.9 | 25       |
|          |      | 136.5| 22       |
| 26a      | 0.94 | 28.6 | 27       |
|          |      | 36.5 | 26       |
|          |      | 75.9 | 25       |
| 28       | 0.98 | 36.5 | 26       |
|          |      | 28.6 | 27       |
| 4′       | 3.12 | 68.7 | 5′       |
|          |      | 99.4 | 1′       |
| 6′       | 1.28 | 84.3 | 4′       |
|          |      | 68.7 | 5′       |
| 2′a      | 1.93 | 68.2 | 3′       |
| 4′       | 3.19 | 65.9 | 5′       |
| 6′       | 1.23 | 74.1 | 4′       |
|          |      | 65.9 | 5′       |

entire *act* biosynthetic cluster [30]. To make this vector more useful for cloning and expression of *Streptomyces* genes, the following modifications were done as described below: (a) part of *actIII* was deleted to ensure that the ActIII protein would not be expressed, (b) the *act* genes downstream of the *actI* promoter were deleted, and (c) new cloning sites were introduced downstream of both the *actI* and *actIII* promoters. The HindIII-PacI fragment containing the *actII/actIII* promoter region was cloned into pNEB 193 to facilitate manipulation and to provide an EcoRI site next to the PacI site (pWHM 465, Table 2). A PCR fragment was amplified using primers PM1 and PO2 (above) to introduce unique FseI, NsiI, and XbaI sites downstream of the *actIII* promoter. The new FseI site allowed the deletion of about one-third of *actIII* by cutting a single internal FseI site followed by re-ligation. The PCR product was digested with FseI and PacI and cloned into pWHM 465 to yield pWHM 466 (Table 2). The HindIII-EcoRI fragment from pWHM 466 containing the *actII*-ORF4 activator gene and *act* promoter region was used to replace the *act* genes in pRM 5 to give pWHM 467. Amplification was made as described above.

### 5.5. Construction of plasmids for expression of the *avrBCDEFGHI* genes

Plasmid pWHM 482 was constructed using the *Streptomyces* expression vector pWHM 1250 and the native L-oleandrose bio-

Table 7  
H, H-COSY couplings of avermectin A1a1 in methanol-d4 (δ, relative to TMS)

| Position | ¹H | ¹H | Position |
|----------|----|----|----------|
| 2        | 3.22 | 1.79 | 4a       |
|          |     | 3.95 | 5/6      |
|          |     | 5.47 | 3        |
| 3        | 5.47 | 1.79 | 4a       |
|          |     | 3.22 | 2        |
|          |     | 3.95 | 5/6      |
| 4a       | 1.79 | 3.22 | 2        |
|          |     | 5.47 | 3        |
| 5/6      | 3.95 | 5.47 | 3        |
| 6        | 3.95 | 3.22 | 2        |
| 8a       | 4.61, 4.65 | 5.82 | 9        |
| 9        | 5.82 | 4.61, 4.65 | 8a       |
| 10       | 5.89 | 5.72 | 11       |
| 11       | 5.72 | 2.65 | 12       |
|          |     | 5.89 | 10       |
| 12       | 2.65 | 5.72 | 11       |
|          |     | 1.16 | 12a      |
| 12a      | 1.16 | 2.65 | 12       |
| 14a      | 1.54 | 5.15 | 15       |
| 15       | 5.15 | 1.54 | 14a      |
|          |     | 2.27 | 16       |
| 16       | 2.27 | 3.89 | 17       |
|          |     | 5.15 | 15       |
| 17       | 3.89 | 0.85 | 18a      |
|          |     | 2.27 | 16       |
| 18e      | 1.93 | 0.85 | 18a      |
|          |     | 2.20 | 20e      |
|          |     | 5.05 | 19       |
| 18a      | 0.85 | 1.93 | 18e      |
|          |     | 3.89 | 17       |
|          |     | 5.05 | 19       |
| 19       | 5.05 | 0.85 | 18a      |
|          |     | 1.36 | 20a      |
|          |     | 1.93 | 18e      |
|          |     | 2.20 | 20e      |
| 20e      | 2.20 | 1.36 | 20a      |
|          |     | 1.93 | 18e      |
|          |     | 5.05 | 19       |
| 20a      | 1.36 | 2.20 | 20e      |
|          |     | 5.05 | 19       |
| 22       | 5.75 | 2.29 | 24       |
|          |     | 5.54 | 23       |
| 23       | 5.54 | 2.29 | 24       |
|          |     | 5.75 | 22       |
| 24       | 2.29 | 1.63 | 26       |
|          |     | 0.92 | 24a      |
|          |     | 3.48 | 25       |
|          |     | 5.54 | 23       |
|          |     | 5.75 | 22       |
| 24a      | 0.92 | 2.29 | 24       |
| 25       | 3.48 | 2.29 | 24       |
| 26       | 1.63 | 0.94 | 26a      |
|          |     | 2.29 | 24       |
| 26a      | 0.94 | 1.63 | 26       |
| 27       | 1.48-1.54 | 0.98 | 28       |
| 28       | 0.98 | 1.48-1.54 | 27       |
| 1'       | 4.75 | 1.70 | 2'a      |
| 2'e      | 2.08 | 1.70 | 2'a      |
|          |     | 4.02 | 3'       |
| 2'a      | 1.70 | 2.08 | 2'e      |
|          |     | 4.02 | 3'       |
|          |     | 4.75 | 1'        |

Table 7 (continued)

| Position | ¹H | ¹H | Position |
|----------|----|----|----------|
| 3'       | 4.02 | 1.70 | 2'a      |
|          |     | 2.08 | 2'e      |
|          |     | 3.12 | 4'       |
| 4'       | 3.12 | 3.89 | 5'       |
|          |     | 4.02 | 3'       |
| 5'       | 3.89 | 1.28 | 6'       |
|          |     | 3.12 | 4'       |
| 6'       | 1.28 | 3.89 | 5'       |
| 1"       | 5.28 | 1.93 | 2"a      |
| 2"e      | 2.21 | 1.93 | 2"a      |
|          |     | 5.28 | 1"        |
| 2"a      | 1.93 | 2.21 | 2"e      |
|          |     | 5.28 | 1"        |
| 3"       | 3.94 | 3.19 | 4"       |
| 4"       | 3.19 | 1.23 | 6"       |
|          |     | 3.94 | 3"        |
|          |     | 4.08 | 5"        |
| 5"       | 4.08 | 1.23 | 6"       |
|          |     | 3.19 | 4"        |
| 6"       | 1.23 | 3.19 | 4"       |
|          |     | 4.08 | 5"        |

synthetic gene cluster avrBDCEFGHI from S. avermitilis. The cluster, which contained a C-terminal portion of ORF1, was sub-cloned as an 8.7 kb NsiI-HindIII fragment from pVE1190 behind the ermE* promoter of pWHM1250.

A plasmid, pWHM481, was constructed in which the eight avr genes were divided into two groups: genes avrB, avrE, avrD and avrC in one, avrI, avrH, avrG and avrF in the other (Fig.3C). The two sets of genes are each oriented in opposite directions and expressed under control of the actI and actIII promoters. The avrB and avrE genes were reoriented from the native arrangement into the same direction as avrC and avrD as follows: the poly-linker LEH3 (see above) was synthesized and cloned into pNEB193 to create pWHM468 (Table 2), then avrD and avrC were cloned from pVE1190asa 2.1 kb fragment into pWHM468 between the KpnI and BstEII sites to create pWHM469 (Table 2).

The polylinker LEB4 containing arbs and novel XbaI and AvrII restriction sites was introduced upstream of the start codon of avrE between the EcoRI and BamHI sites of pWHM1251 [27] to create pWHM470.The AvrII-PstI fragment containing the entire avrE gene was excised from pWHM470 and cloned into pWHM469 to create pWHM471 (Table 2). The avrB gene was amplified from pVE1190byPCR with the primers PO6 and PO7 (above) as a 1.3 kb XbaI-AvrII fragment and cloned into pWHM468 to create pWHM472.The central portion of this fragment between SstI and PmlI sites in pWHM472 was replaced with a non-amplified fragment from pVE1190 to produce pWHM473 (Table 2). The remaining ends of the avrB gene were sequenced to verify authenticity.The XbaI-AvrII fragment containing avrB was cloned from pWHM473 into pWHM471 to create pWHM474 (Table 2). The XbaI site was eliminated from pWHM474 to create pWHM479.The genes avrB, avrC, avrD and avrE, all oriented in the same direction, were cloned as a PacI-EcoRI fragment from pWHM479 into pWHM467 under control

Table 8  
2D-NOESY couplings of avermectin A1a1 in methanol-d4 (δ, relative to TMS)

| Position | ¹H   | ¹H   | Position |
|----------|------|------|----------|
| 2        | 3.22 | 5.47 | 3        |
| 3        | 5.47 | 1.79 | 4a       |
|          |      | 3.22 | 2        |
| 4a       | 1.79 | 5.47 | 3        |
|          |      | 3.95 | 5/6      |
| 5/6      | 3.95 | 1.79 | 4a       |
| 9        | 5.82 | 5.72 | 11       |
| 11       | 5.72 | 5.82 | 9        |
| 12       | 2.65 | 1.54 | 14a      |
|          |      | 3.99 | 13       |
| 12a      | 1.16 | 3.99 | 13       |
| 13       | 3.99 | 1.16 | 12a      |
|          |      | 1.54 | 14a      |
|          |      | 2.65 | 12       |
| 14a      | 1.54 | 2.27 | 16       |
|          |      | 2.65 | 12       |
|          |      | 3.99 | 13       |
| 16       | 2.27 | 1.54 | 14a      |
| 18e      | 1.93 | 0.85 | 18a      |
| 18a      | 0.85 | 1.93 | 18e      |
| 20e      | 2.20 | 1.36 | 20a      |
| 20a      | 1.36 | 2.20 | 20e      |
| 22       | 5.75 | 0.92 | 24a      |
|          |      | 2.29 | 24       |
|          |      | 5.54 | 23       |
| 23       | 5.54 | 5.75 | 22       |
| 24       | 2.29 | 0.92 | 24a      |
|          |      | 0.94 | 26a      |
|          |      | 5.75 | 22       |
| 24a      | 0.92 | 2.29 | 24       |
|          |      | 5.75 | 22       |
| 26       | 1.63 | 0.94 | 26a      |
| 26a      | 0.94 | 1.63 | 26       |
|          |      | 2.29 | 24       |
| 2'e      | 2.08 | 4.02 | 3'       |
| 3'       | 4.02 | 2.08 | 2'e      |
| 4'       | 3.12 | 5.28 | 1"       |
|          |      | 1.28 | 6'       |
| 5'       | 3.89 | 1.28 | 6'       |
| 6'       | 1.28 | 5.28 | 1"       |
|          |      | 3.12 | 4'       |
|          |      | 3.89 | 5'       |
| 1"       | 5.28 | 1.28 | 6'       |
|          |      | 1.93 | 2"a      |
|          |      | 3.12 | 4'       |
| 2'e      | 2.21 | 1.93 | 2"a      |
| 2"a      | 1.93 | 2.21 | 2"e      |
|          |      | 3.19 | 4"       |
|          |      | 3.94 | 3"       |
|          |      | 5.28 | 1"       |
| 3"       | 3.94 | 1.93 | 2"a      |
|          |      | 3.19 | 4"       |
| 4"       | 3.19 | 1.23 | 6"       |
|          |      | 1.93 | 2"a      |
|          |      | 3.94 | 3"       |
|          |      | 4.08 | 5"       |
| 5"       | 4.08 | 1.23 | 6"       |
|          |      | 3.19 | 4"       |
| 6"       | 1.23 | 3.19 | 4"       |
|          |      | 4.08 | 5"       |

cloned into pWHM 480, containing the avrBEDC genes, to create
pWHM 2134 (Table 2; Fig.3C).

A plasmid with an in-frame deletion in the avrI gene and without the avrE gene was constructed as follows. A 3.25kb XbaI-
NsiI fragment from pWHM 2133, containing genes avrHGF and
the ΔavrI gene, was cloned into pWHM 2114, already containing
genes avrBDC, to create pWHM 2135 (Table 2; Fig.3C).

To introduce additional genes (eryBIII, and eryBV) a two-plasmid system was used. pWHM 467 contained the avr genes and
thiostrepton resistance gene while pKOS 025-129 carried the additional genes and hygromycin resistance marker. A derivative of
pKOS 025-129, containing the gene eryBIII, was constructed by
cloning a 1.25kb SpeI-NotI fragment, containing eryBIII, from
pFL401intopANT 841 between the sites NheI and NotI to create
pWHM 488.A 1.25kb HindIII-EcoRI fragment, containing ery-
BIII from pWHM 488, was next cloned into pWHM 485 to give
pWHM 2118.A 1.25kb PacI-NsiI fragment from pWHM 2118
was cloned into pKOS 025-129 under control of the actI promoter, creating pWHM 2119.

5.6. Bioconversion of avermectin aglycones and product analysis

Avermectin aglycones were produced by shaken-flask cultures
of the S. avermitilis Δ1066 strain which has a 16kb deletion of all
the TDP-oleandrose biosynthesis genes except avrB [25], or
S. avermitilis MA6057a, a mutant mapped to a deletion of the
avrF gene, using previously described seed [47] and liquid production media [48].

Avermectin Ala aglycone was purified and characterized in the
following way. The crude extract residues obtained by methanol
and ethyl acetate extraction of recovered mycelia were fraction-
ated over silica gel using a flash funnel method: 350mg of crude
extract residue coated on 5g of silica gel in a 4cm diameter, 4cm

Table 9
HMBC couplings of avermectin A1a2 in methanol-d4 (δ, relative to
TMS)

| Position | ¹H | ¹³C | Position |
|----------|----|-----|----------|
| 4a       | 1.78 | 121.3 | 3        |
|          |     | 134.8 | 4        |
| 5a       | 3.46 | 78.3  | 5/6      |
| 12a      | 1.20 | 41.0  | 12       |
|          |     | 83.5  | 13       |
| 14a      | 1.54 | 138.4 | 11       |
|          |     | 83.5  | 13       |
|          |     | 119.6 | 15       |
|          |     | 136.6 | 14       |
| 24a      | 0.92 | 31.8  | 24       |
|          |     | 75.9  | 25       |
| 26a      | 0.94 | 28.6  | 27       |
|          |     | 36.5  | 26       |
| 28       | 0.98 | 75.9  | 25       |
|          |     | 28.6  | 27       |
|          |     | 36.5  | 26       |
| 6'       | 1.23 | 64.3  | 5'       |
|          |     | 76.6  | 4'       |
| 6"       | 1.24 | 69.8  | 5"       |
|          |     | 78.8  | 4"       |

Table 10
H, H-COSY couplings of avermectin A1a2 in methanol-d4 (δ, relative to
TMS)

| Position | ¹H   | ¹H   | Position |
|----------|------|------|----------|
| 2        | 3.22 | 1.78 | 4a       |
| 3        | 5.47 | 1.78 | 4a       |
| 4a       | 1.78 | 3.22 | 2        |
|          |      | 5.47 | 3        |
| 10       | 5.89 | 5.79 | 11       |
| 11       | 5.79 | 2.65 | 12       |
|          |      | 5.89 | 10       |
| 12       | 2.65 | 1.20 | 12a      |
|          |      | 5.79 | 11       |
| 12a      | 1.20 | 2.65 | 12       |
| 15       | 5.22 | 2.28 | 16       |
| 16       | 2.28 | 3.89 | 17       |
|          |      | 5.22 | 15       |
| 17       | 3.89 | 0.84 | 18a      |
|          |      | 2.28 | 16       |
| 18e      | 1.96 | 0.84 | 18a      |
| 18a      | 0.84 | 1.96 | 18e      |
|          |      | 3.89 | 17       |
|          |      | 5.05 | 19       |
| 19       | 5.05 | 0.84 | 18a      |
|          |      | 1.35 | 20a      |
| 20e      | 2.17 | 1.35 | 20a      |
| 20a      | 1.35 | 2.17 | 20e      |
|          |      | 5.05 | 19       |
| 22       | 5.74 | 5.54 | 23       |
| 23       | 5.54 | 5.74 | 22       |
| 24       | 2.29 | 0.92 | 24a      |
|          |      | 3.48 | 25       |
| 24a      | 0.92 | 2.29 | 24       |
|          |      | 3.48 | 25       |
| 25       | 3.48 | 0.92 | 24a      |
|          |      | 2.29 | 24       |
| 26       | 1.63 | 0.94 | 26a      |
| 26a      | 0.94 | 1.63 | 26       |
| 27       | 1.47-1.54 | 0.98 | 28       |
| 28       | 0.98 | 1.47-1.54 | 27       |
| 1'       | 4.75 | 1.98 | 2'e      |
|          |      | 2.15 | 2'e      |
| 2'e      | 2.15 | 1.98 | 2'a      |
|          |      | 4.25 | 3'       |
|          |      | 4.75 | 1'       |
| 2'a      | 1.98 | 2.15 | 2'e      |
|          |      | 4.25 | 2'e      |
| 3'       | 4.25 | 2.15 | 6'       |
| 5'       | 4.29 | 1.23 | 6'       |
| 6'       | 1.23 | 4.29 | 5'       |
| 1"       | 5.10 | 1.62 | 2"a      |
| 2"e      | 2.18 | 1.62 | 2"a      |
| 2"a      | 1.62 | 2.18 | 2"e      |
|          |      | 3.79 | 3"        |
|          |      | 5.10 | 1"        |
| 3"       | 3.79 | 1.62 | 2"a      |
|          |      | 2.95 | 4"        |
| 4"       | 2.95 | 3.65 | 5"        |
|          |      | 3.79 | 3"        |
| 5"       | 3.65 | 1.24 | 6"        |
|          |      | 2.95 | 4"        |
| 6"       | 1.24 | 3.65 | 5"        |

high sintered glass funnel was eluted with three successive meth-
ylene chloride washes, followed by elution with a 5% step gra-
dient of ethyl acetate in methylene chloride and fractions of 20ml
were collected. The appropriate fractions from three such purifi-
cations were combined to give one sample of enriched avermectin
Ala aglycone, which was further purified by semi-preparative
HPLC using solvent gradient system 1 (see below). Alternatively
avermectin Ala aglycone was isolated directly out of the crude
extract residue in two purification steps by semi-preparative
HPLC (solvent gradient system 2). The resulting purified material
was analyzed by ESI-LRMS and ESI-HRMS as well as by HPLC
comparison with authentic avermectin aglycone obtained through
acid hydrolysis of a concentrated crude extract of avermectins
obtained from fermentation of S. avermitilis ATCC 31272 (kindly
provided by R.W. Fedechko and K. Stutzmann-Engwall at Chas.
Pfizer and Co.).

For bioconversion experiments using S. lividans K4-114 har-
boring the pWHM 481, pWHM 2106, pWHM 2109, pWHM 2111,

Table 11
2D-NOESY couplings of avermectin A1a2 in methanol-d4 (δ, relative to
TMS)

| Position | ¹H | ¹H | Position |
|----------|----|----|----------|
| 3        | 5.47 | 1.78 | 4a       |
| 4a       | 1.78 | 5.47 | 3        |
| 5/6      | 3.93 | 3.46 | 5a       |
| 5a       | 3.46 | 3.93 | 5        |
| 10       | 5.89 | 5.79 | 11       |
| 11       | 5.79 | 5.89 | 10       |
| 12       | 2.65 | 1.54 | 14a      |
| 12a      | 1.20 | 4.05 | 13       |
| 13       | 4.05 | 1.20 | 12a      |
| 14a      | 1.54 | 2.65 | 12       |
| 18e      | 1.96 | 0.84 | 18a      |
| 18a      | 0.84 | 1.96 | 18e      |
| 20e      | 2.17 | 1.35 | 20a      |
| 20a      | 1.35 | 2.17 | 20e      |
| 22       | 5.74 | 0.92 | 24a      |
|          | 5.54 |     | 23       |
| 23       | 5.54 | 5.74 | 22       |
| 24       | 2.29 | 0.92 | 24a      |
| 24a      | 0.92 | 2.29 | 24       |
|          | 3.48 |     | 25       |
|          | 5.74 |     | 22       |
| 25       | 3.48 | 0.92 | 24a      |
| 27       | 1.47–1.54 | 0.98 | 28       |
| 28       | 0.98 | 1.47–1.54 | 27       |
| 1′       | 4.75 | 1.98 | 2′a      |
|          | 2.15 |     | 2′e      |
| 2′e      | 2.15 | 4.75 | 1′       |
|          | 1.98 |     | 2′a      |
| 2′a      | 1.98 | 2.15 | 2′e      |
|          | 4.75 |     | 1′       |
| 5′       | 4.29 | 1.23 | 6′       |
|          | 5.10 |     | 1″       |
| 6′       | 1.23 | 4.29 | 5′       |
| 1″       | 5.10 | 4.29 | 5′       |
| 2″e      | 2.18 | 1.62 | 2″a      |
| 2″a      | 1.62 | 2.18 | 2″e      |
| 5″       | 3.65 | 1.24 | 6″       |
| 6″       | 1.24 | 3.65 | 5″       |

Table 12
HMBC couplings of avermectin A1a2 in chloroform-d1 (δ, relative to
TMS)

| Position | ¹H | ¹³C | Position |
|----------|----|-----|----------|
| 4a       | 1.81 | 118.3 | 3        |
|          |     | 136.0 | 4        |
| 5/6      | 3.98 | 78.0 | 5/6      |
| 5a       | 3.50 | 77.9 | 5/6      |
| 12a      | 1.20 | 39.5 | 12       |
|          |     | 82.0 | 13       |
|          |     | 136.4 | 11       |
| 14a      | 1.51 | 82.0 | 13       |
|          |     | 119.4 | 15       |
|          |     | 134.5 | 14       |
| 18a      | 0.88 | 68.0 | 17       |
| 24a      | 0.91 | 30.6 | 24       |
|          |     | 74.8 | 25       |
|          |     | 136.3 | 22       |
| 26a      | 0.93 | 27.5 | 27       |
|          |     | 35.3 | 26       |
| 28       | 0.95 | 27.5 | 27       |
|          |     | 35.3 | 26       |
| 6′       | 1.27 | 62.7 | 3′       |
|          |     | 75.1 | 4′       |
| 6″       | 1.30 | 68.0 | 5″       |
|          |     | 78.4 | 4″       |

pWHM 2130, pWHM 2134, or pWHM 2135 plasmids (Fig.3C), or
both pWHM 2106 and pWHM 2119 or both pWHM 2109 and
pWHM 2122, transformants were grown on solid R2YE without
sucrose with exogenous Ala avermectin aglycone (10 μg/ml).
Antibiotics were added to the media as follows: for transform-
ants carrying pWHM 481, pWHM 2106, pWHM 2109,
pWHM 2111, pWHM 2130, pWHM 2134 and pWHM 2135, thio-
strepton at 30 μg/ml; for transformants carrying pWHM 2106 and
pWHM 2119 or pWHM 2109 and pWHM 2122, thiostrepton at 30
μg/ml and hygromycin at 200 μg/ml. After 5 days of incubation,
avermectin metabolites were extracted from chopped solid R2YE
using equal volumes of ethyl acetate, concentrated by evapora-
tion under a stream of nitrogen to give an oily residue, which was
resuspended in methanol.

Our initial approach was to grow three agar plates of each
strain. The methanol extracts of these were first analyzed by
HPLC using the analytical system (see below) and those metab-
olites that showed the typical avermectin UV-absorption spec-
trum were isolated by semi-preparative HPLC (systems 1 and
3) before further being analyzed by electrospray mass spectrom-
etry. The strains that were investigated in the above described
manner are shown with positive ion ESI-MS m/z data for metab-
olites isolated from the respective strains and what possible prod-
uct this (these) mass(es) might reflect in Table 3.

5.7. Isolation and characterization of avermectins A1a1, A1a2 and
A1a3

Under the conditions described above S. lividans K4-114/
pWHM 2106, S. lividans K4-114/pWHM 2111 and S. lividans
K4-114/pWHM 2130 strains were grown on 3 l, 1.5 l and 1.5 l
of solid media (8.5 cm diameter plates, with each plate holding

Table 13  
H, H-COSY couplings of avermectin A1a2 in chloroform-d₁ (δ, relative  
to TMS)  

| Position | ¹H | ¹H | Position |
|----------|----|----|----------|
| 2        | 3.31 | 1.81 | 4a       |
|          |     | 3.98 | 5/6      |
|          |     | 5.38 | 3        |
| 3        | 5.38 | 1.81 | 4a       |
|          |     | 3.31 | 2        |
|          |     | 3.98 | 5/6      |
| 4a       | 1.81 | 3.31 | 2        |
|          |     | 5.38 | 3        |
| 5/6      | 3.98 | 3.31 | 2        |
|          |     | 5.38 | 3        |
| 8a       | 4.62, 4.64 | 5.81 | 9        |
| 9        | 5.81 | 4.62, 4.64 | 8a       |
| 11       | 5.62 | 2.65 | 12       |
| 12       | 2.65 | 1.20 | 12a      |
|          |     | 5.62 | 11       |
|          |     | 4.05 | 13       |
| 12a      | 1.20 | 2.65 | 12       |
| 13       | 4.05 | 2.65 | 12       |
|          |     | 2.29 | 16       |
|          |     | 5.05 | 15       |
| 15       | 5.05 | 2.29 | 16       |
|          |     | 4.05 | 13       |
| 16       | 2.29 | 3.86 | 17       |
|          |     | 4.05 | 13       |
|          |     | 5.05 | 15       |
| 17       | 3.86 | 2.29 | 16       |
|          |     | 0.88 | 18a      |
| 18e      | 1.74 | 0.88 | 18a      |
| 18a      | 0.88 | 1.74 | 18e      |
|          |     | 3.86 | 17       |
|          |     | 5.36 | 19       |
| 19       | 5.36 | 1.48 | 20a      |
|          |     | 2.03 | 20e      |
|          |     | 0.88 | 18a      |
| 20e      | 2.03 | 1.48 | 20a      |
|          |     | 5.36 | 19       |
| 20a      | 1.48 | 2.03 | 20e      |
|          |     | 5.36 | 19       |
| 22       | 5.78 | 5.55 | 23       |
| 23       | 5.55 | 5.77 | 22       |
| 24       | 2.28 | 0.91 | 24a      |
|          |     | 3.47 | 25       |
| 24a      | 0.91 | 2.28 | 24       |
| 25       | 3.47 | 2.28 | 24       |
| 26       | 1.60 | 0.93 | 26a      |
| 26a      | 0.93 | 1.48–1.54 | 27       |
|          |     | 1.60 | 26       |
| 27       | 1.48–1.54 | 0.95 | 28       |
|          |     | 0.93 | 26a      |
| 28       | 0.95 | 1.48–1.54 | 27       |
| 1′       | 4.83 | 1.92 | 2′a      |
| 2′e      | 2.14 | 1.92 | 2′a      |
|          |     | 4.27 | 3′       |
| 2′a      | 1.92 | 2.14 | 2′e      |
|          |     | 4.27 | 3′       |
|          |     | 4.83 | 1′       |
| 3′       | 4.27 | 1.92 | 2′a      |
|          |     | 2.14 | 2′e      |
| 4′       | 3.29 | 4.19 | 5′       |
| 5′       | 4.19 | 1.27 | 6′       |
|          |     | 3.29 | 4′       |

---

Table 13 (continued)

| Position | ¹H | ¹H | Position |
|----------|----|----|----------|
| 1″       | 5.18 | 1.71 | 2″a      |
| 2″e      | 2.23 | 1.71 | 2″a      |
|          |     | 3.95 | 3″       |
| 2″a      | 1.71 | 2.23 | 2″e      |
|          |     | 3.95 | 3″       |
|          |     | 5.18 | 1″       |
| 3″       | 3.95 | 1.71 | 2″a      |
|          |     | 2.23 | 2″e      |
|          |     | 3.19 | 4″       |
| 4″       | 3.19 | 3.67 | 5″       |
|          |     | 3.95 | 3″       |
| 5″       | 3.67 | 1.30 | 6″       |
|          |     | 3.19 | 4″       |
| 6″       | 1.30 | 3.67 | 5″       |

---

Table 14  
2D-NOESY couplings of avermectin A1a2 in chloroform-d₁ (δ, relative  
to TMS)  

| Position | ¹H | ¹H | Position |
|----------|----|----|----------|
| 3        | 5.38 | 1.81 | 4a       |
| 4a       | 1.81 | 5.38 | 3        |
| 12       | 2.65 | 1.20 | 12a      |
| 12a      | 1.20 | 2.65 | 12       |
| 13       | 4.05 | 2.65 | 12       |
| 14a      | 1.51 | 2.65 | 12       |
| 15       | 5.05 | 2.29 | 16       |
| 16       | 2.29 | 5.05 | 15       |
| 19       | 5.36 | 2.03 | 20e      |
| 20e      | 2.03 | 1.48 | 20a      |
| 20a      | 1.48 | 5.36 | 19       |
| 22       | 5.78 | 5.55 | 23       |
| 23       | 5.55 | 5.77 | 22       |
| 24       | 2.28 | 0.91 | 24a      |
| 24a      | 0.91 | 2.28 | 24       |
| 25       | 3.47 | 2.28 | 24       |
| 26       | 1.60 | 0.93 | 26a      |
| 26a      | 0.93 | 1.48–1.54 | 27       |
|          |     | 1.60 | 26       |
| 27       | 1.48–1.54 | 0.95 | 28       |
|          |     | 0.93 | 26a      |
| 28       | 0.95 | 1.48–1.54 | 27       |
| 1′       | 4.83 | 1.92 | 2′a      |
| 2′e      | 2.14 | 1.92 | 2′a      |
|          |     | 4.27 | 3′       |
| 2′a      | 1.92 | 2.14 | 2′e      |
|          |     | 4.27 | 3′       |
|          |     | 4.83 | 1′       |
| 3′       | 4.27 | 1.92 | 2′a      |
|          |     | 2.14 | 2′e      |
| 4′       | 3.29 | 4.19 | 5′       |
| 5′       | 4.19 | 1.27 | 6′       |
|          |     | 3.29 | 4′       |

---

| Position | ¹H | ¹H | Position |
|----------|----|----|----------|
| 1′       | 4.83 | 1.92 | 2′a      |
| 2′e      | 2.14 | 1.92 | 2′a      |
|          |     | 4.27 | 3′       |
| 2′a      | 1.92 | 2.14 | 2′e      |
|          |     | 4.27 | 3′       |
|          |     | 4.83 | 1′       |
| 3′       | 4.27 | 1.92 | 2′a      |
|          |     | 2.14 | 2′e      |
| 4′       | 3.29 | 4.19 | 5′       |
| 5′       | 4.19 | 1.27 | 6′       |
|          |     | 3.29 | 4′       |


about 25ml material), respectively. The plates were chopped and extracted four times with ethyl acetate and three times with methanol, before evaporating the combined extracts of each strain to dryness. The avermectins Ala2 and Ala3 were separated out of the crude extracts from avermectin Ala1 by semi-preparative HPLC (system 4, see below). By using the same semi-preparative HPLC treatment a second time, purified avermectin Ala1 was obtained. Avermectin Ala2 was separated from a third compound by preparative thin-layer chromatography (silica gel 60/ F254TLC plates, E. Merck, Darmstadt, Germany; ethyl acetate: methylene chloride:acetonitrile 50:100:8) and extracted from the scraped off silica gel by ethyl acetate and methanol, before purifying the compounds on Sephadex LH 20 (Pharmacia; solvent methanol). The third compound also showed the typical avermectin UV-spectrum and the mass determined to be the same as that of avermectins Ala1 and Ala2. Avermectin Ala3 was purified out of its crude extract by two semi-preparative HPLC steps using system 2. We received 0.5 mg/l of avermectin Ala1, 0.4 mg/l avermectin Ala2, and 0.3 mg/l for the third avermectin derivative from both S.lividans K4-114/pWHM 2106 (avrBED- CIGF, ΔavrH) and S.lividans K4-114/pWHM 2111 (avrBEDCGF, ΔavrHI) as well as 1.1 mg/l avermectin Ala3 from S.lividans K4- 114/pWHM 2130 (avrBEDCIHG, ΔavrF).

5.8. Avermectin Ala1

M = 859.06 (C47H70O14); ESI-MS pos. mode m/z: 881.5 [(M+Na)+]; MP: 171°C; IR(KBr): ν = 3487, 2994, 2935, 2866, 1381, 1368, 1365, 1157, 1148, 1125, 1065, 1055, 1030, 991, 975, 956, 803, 615, 470, 419, 393 cm⁻¹; UV(methanol) max nm:

Table 15
HMBC couplings of avermectin Ala3 in methanol-d₄ (δ, relative to TMS)

| Position | ¹H   | ¹³C  | Position |
|----------|------|------|----------|
| 4a       | 1.78 | 121.2| 3        |
|          |      | 135.2| 4        |
| 5a       | 3.46 | 78.2 | 5/6      |
| 12a      | 1.12 | 84.0 | 13       |
|          |      | 139.0| 11       |
| 14a      | 1.53 | 84.0 | 13       |
|          |      | 119.2| 15       |
|          |      | 137.2| 14       |
| 18a      | 0.86 | 69.9 | 19       |
|          |      | 70.1 | 17       |
| 20a      | 1.35 | 69.9 | 19       |
|          |      | 97.3 | 21       |
| 22       | 5.75 | 97.3 | 21       |
| 24a      | 0.92 | 31.8 | 24       |
|          |      | 75.9 | 25       |
| 26a      | 0.94 | 136.6| 22       |
|          |      | 28.5 | 27       |
|          |      | 36.6 | 26       |
|          |      | 75.9 | 25       |
| 28       | 0.99 | 28.5 | 27       |
|          |      | 36.6 | 26       |
| 4'       | 2.91 | 72.5 | 3'       |
| 6'       | 1.26 | 73.2 | 5'       |
|          |      | 78.7 | 4'       |

Table 16
H,H-COSY couplings of avermectin Ala3 in methanol-d₄ (δ, relative to TMS)

| Position | ¹H     | ¹H     | Position |
|----------|--------|--------|----------|
| 2        | 3.21   | 1.78   | 4a       |
| 3        | 5.47   | 1.78   | 4a       |
| 4a       | 1.78   | 3.21   | 2        |
|          |        | 5.47   | 3        |
| 8a       | 4.60-4.64 | 5.80   | 9        |
| 9        | 5.80   | 4.60-4.64 | 8a       |
| 10       | 5.86   | 5.65   | 11       |
| 11       | 5.65   | 2.60   | 12       |
|          |        | 5.86   | 10       |
| 12       | 2.60   | 1.12   | 12a      |
|          |        | 5.65   | 11       |
|          |        | 4.09   | 13       |
| 12a      | 1.12   | 2.60   | 12       |
| 13       | 4.09   | 2.60   | 12       |
| 14a      | 1.53   | 5.24   | 15       |
| 15       | 5.24   | 1.53   | 14a      |
|          |        | 2.26   | 16       |
| 16       | 2.26   | 5.24   | 15       |
|          |        | 3.89   | 17       |
| 17       | 3.89   | 2.26   | 16       |
| 18e      | 1.92   | 0.86   | 18a      |
| 18a      | 0.86   | 1.92   | 18e      |
| 19       | 5.04   | 1.35   | 20a      |
|          |        | 2.19   | 20e      |
| 20e      | 2.19   | 1.60   | 20a      |
|          |        | 5.04   | 19       |
| 20a      | 1.35   | 2.19   | 20e      |
|          |        | 5.04   | 19       |
| 22       | 5.75   | 5.54   | 23       |
| 23       | 5.54   | 5.75   | 22       |
| 24       | 2.29   | 0.92   | 24a      |
|          |        | 3.46   | 25       |
| 24a      | 0.92   | 2.29   | 24       |
| 25       | 3.46   | 2.29   | 24       |
| 26       | 1.65   | 0.94   | 26a      |
| 26a      | 0.94   | 1.65   | 26       |
| 27       | 1.48-1.58 | 0.99   | 28       |
| 28       | 0.99   | 1.48-1.58 | 27       |
| 1'       | 4.38   | 1.60   | 2'a      |
| 2'e      | 2.14   | 1.60   | 2'a      |
| 3'       | 3.48   | 3'      |
| 2'a      | 1.60   | 2.14   | 2'e      |
| 4.38     | 1'     |
| 3'       | 3.48   | 2.14   | 2'e      |
| 4'       | 2.91   | 3'      |
| 4'       | 3.48   | 3'      |
| 5'       | 2.91   | 4'      |
| 19       | 3.14   | 5'      |
| 17       | 1.26   | 6'      |
| 19       | 2.91   | 4'      |
| 21       | 3.14   | 5'      |
| 21       | 1.26   | 5'      |
| 24       |        |        |

236.0, 243.5, 252.7; Rf: 0.17 (ethyl acetate:methylene chloride: acetonitrile 50:100:8; Rf avermectin Ala aglycone 0.67); Rf 17.49 min (analytical HPLC system; Rf avermectin Ala aglycone 12.95 min); NMR data see Tables 4-8.

5.9. Avermectin Ala2

M = 859.06 (C47H70O14); ESI-MS pos. mode m/z: 881.5

[(M+Na)⁺]; mp: 169°C; IR (KBr): ν = 3480, 2985, 2930, 2860, 1376, 1364, 1360, 1150, 1141, 1119, 1060, 1046, 1022, 985, 967, 950, 794, 607, 467, 415, 393 cm⁻¹; UV (methanol) max nm: 236.0, 243.5, 252.7; \( R_f \): 0.15 (ethyl acetate:methylene chloride:acetonitrile 50:100:8); \( R_f \) avermectin Ala aglycone 0.67); \( R_f \) 18.49 min (analytical HPLC system; \( R_f \) avermectin Ala aglycone 12.95 min); NMR data see Tables 4, 5, 9–14.

### 5.10. Avermectin Alα3

\( M = 728.92 \) (\( C_{41}H_{60}O_{11} \)); ESI-MS pos. mode \( m/z \): 751.6 [(M+Na)⁺]; mp: 140°C; IR (KBr): ν = 3420, 2981, 2921, 2839, 1722, 1625, 1370, 1355, 1328, 1154, 1146, 1115, 1062, 990, 902, 795, 586, 468, 417, 391 cm⁻¹; UV (methanol) max nm: 236.0, 243.5, 252.7; \( R_f \): 0.27 (ethyl acetate:methylene chloride:acetonitrile 50:100:8); \( R_f \) avermectin Ala aglycone 0.67); \( R_f \) 14.24 min (analytical HPLC system; \( R_f \) avermectin Ala aglycone 12.95 min); NMR data see Tables 4, 5, 15 and 16.

### 5.11. HPLC

A Waters HPLC system (controlled by Millennium³² software, v. 3.5) with two pumps (models 510 and 515), pump control module, and photodiode array detector (model 996) was used. For analytical purposes we used a Nova-Pak C18 60A 4 μm (150 × 3.9 mm) column with 1.0 ml/min flow speed and the following solvent system: solvent A = water; solvent B = methanol; flow rate of 1.0 ml/min, ambient temperature, 30% A:70% B to 10% A:90% B (linear gradient, curve 6, 0–20 min); 10% A:90% B to 100% B (concave gradient, curve 9, 20–25 min); 100% B to 30% A:70% B (curve 9, 25–30 min). For semi-preparative fractioning several different solvent gradients were developed, using water as solvent A, methanol as solvent B and a flow rate of 2 ml/min: System 1: 30% A:70% B to 100% B (curve 6, 0–40 min); 100% B to 30% A:70% B (curve 9, 40–50 min). System 2: 30% A:70% B to 40% A:60% B (curve 6, 0–60 min); 40% A:60% B to 30% A:70% B (curve 6, 60–62 min). System 3: 30% A:70% B to 10% A:90% B (curve 6, 0–40 min); 10% A:90% B to 100% B (curve 6, 40–45 min); 100% B to 30% A:70% B (curve 6, 45–50 min). System 4: 30% A:70% B to 12.5% A:87.5% B (curve 6, 0–110 min); 12.5% A:87.5% B to 100% B (curve 6, 110–115 min); 100% B to 30% A:70% B (curve 6, 115–120 min). For both analytical and semi-preparative work Sentry Guard Columns Nova-Pak C18 60A 4 μm (20 × 3.9 mm) were used in addition.

### 5.12. Additional instruments

The \({ }^1\)H and \({ }^{13}\)C NMR spectra were recorded in methanol and chloroform at 500 and 125.71 MHz, respectively. The different structures were elucidated by using 1D spectra and 2D homo- and heteronuclear correlation experiments (\({ }^1\)H, \({ }^{13}\)C, DEPT, H,H-COSY, HMQC, HSQC, HMBC, 2D-NOESY, 1D-NOESY, 1D-TOCSY) recorded on a Varian Unity-500. Electrospray ionization mass spectrometry (ESI-MS, University of Wisconsin Biotechnology Center) and matrix-assisted laser desorption/ionization mass spectrometry (HR-MALDI-FT-MS, Mass Consortium,

San Diego, CA, USA) was used to determine the mass. IR-spectra were recorded on a Perkin-Elmer Infrared Spectrometer 599 B. A Unimelt Thomas Hoover Capillary Melting Point Apparatus was used to determine melting points and PCR amplification was performed in a thermal cycler Perkin-Elmer Cetus model 480.

#### Acknowledgements

We thank Jon T. Roll for construction of the modified pRM5 vector, pWHM467, and Charles G. Fry for excellent help using the NMR instrumentation. This work was supported in part by a Grant from the National Institutes of Health (GM 31925). The NMR spectra obtained in the Chemistry and Biochemistry Departments at the University were determined on instruments supported by Grants from the NSF (CHE-8813550 and CHE-9629688) and NIH (S10RR04981-01).

#### References

[1] D.J. MacNeil, Avermectins, in: L.C. Vining, C. Stuttard (Eds.), Genetics and Biochemistry of Antibiotic Production, Butterworth-Heinemann, Boston, MA, 1995, pp. 421–442.

[2] H. Ikeda, S. Omura, Avermectin biosynthesis, Chem. Rev. 97 (1997) 2591–2609.

[3] H.G. Davies, R.H. Green, Avermectins and milbemycins, Nat. Prod. Rep. 3 (1986) 87–121.

[4] Q.A. McKellar, H.A. Benchaoui, Avermectins and milbemycins, J. Vet. Pharmacol. Ther. 19 (1996) 331–351.

[5] A. Kirschning, A.F.-W. Bechthold, J. Rohr, Chemical and biochemical aspects of deoxysugars and deoxysugar oligosaccharides, Top. Curr. Chem. 188 (1997) 1–84.

[6] D.A. Johnson, H.-w. Liu, Mechanisms and pathways from recent deoxysugar biosynthesis research, Curr. Opin. Chem. Biol. 2 (1998) 642–649.

[7] T.M. Hallis, H.-w. Liu, Learning nature's strategies for making deoxy sugars: pathways, mechanisms, and combinatorial applications, Acc. Chem. Res. 32 (1999) 579–598.

[8] A. Trefzer, J.A. Salas, A. Bechthold, Genes and enzymes involved in deoxysugar biosynthesis in bacteria, Nat. Prod. Rep. 16 (1999) 283–299.

[9] D.J. MacNeil, K.M. Gewain, C.L. Ruby, G. Dezeny, P.H. Gibbons, T. MacNeil, Analysis of *Streptomyces avermitilis* genes required for avermectin biosynthesis utilizing a novel integration vector, Gene 111 (1992) 61–68.

[10] M.D. Schulman, D. Valentino, O. Hensens, D. Zink, M. Nallin, L. Kaplan, D.A. Ostlind, Demethylavermectins. Biosynthesis, isolation and characterization, J. Antibiot. 38 (1985) 1494–1498.

[11] H. Ikeda, H. Kotaki, S. Omura, Genetic studies of avermectin biosynthesis in *Streptomyces avermitilis*, J. Bacteriol. 169 (1987) 5615–5621.

[12] J.L. Occi, K.M. Gerwain, C.L. Ruby, D.J. MacNeil, Cloning and sequencing a nine kb region from *Streptomyces avermitilis* essential for glycosylation of avermectin, Abstracts of the annual meeting of the society for industrial microbiology, P49, 1991, p. 93, Philadelphia, PA.

[13] H. Ikeda, T. Nonomiya, M. Usami, T. Ohta, S. Omura, Organization of the biosynthetic gene cluster for the polyketide anthelmintic macrolide avermectin in *Streptomyces avermitilis*, Proc. Natl. Acad. Sci. USA 96 (1999) 9509–9514.

[14] H. Ikeda, S. Omura, Control of avermectin biosynthesis in *Strepto*

myces avermitilis for the selective production of a useful component, J. Antibiot. 48 (1995) 549–562.

[15] R.G. Summers, S. Donadio, M. Staver, E. Wendt-Pienkowski, C.R. Hutchinson, L. Katz, Sequencing and mutagenesis of genes from the erythromycin biosynthetic gene cluster of *Saccharopolyspora erythraea* that are involved in L-mycarose and D-desosamine production, Microbiology 143 (1997) 3251–3262.

[16] S. Gaisser, G.A. Böhm, J. Cortés, P.F. Leadlay, Analysis of seven genes from the *eryAI-eryK* region of the erythromycin biosynthetic gene cluster in *Saccharopolyspora erythraea*, Mol. Gen. Genet. 256 (1997) 239–251.

[17] S. Gaisser, G.A. Böhm, M. Boumith, M.C. Raynal, N. Dhillon, J. Cortes, P.F. Leadlay, Analysis of *eryBI, eryBIII* and *eryBVII* from the erythromycin biosynthetic gene cluster in *Saccharopolyspora erythraea*, Mol. Gen. Genet. 258 (1998) 78–88.

[18] K. Salah-Bey, M. Dourmith, J.-M. Michel, S. Haydock, J. Cortes, P.F. Leadlay, M.C. Raynal, Targeted gene inactivation for the elucidation of deoxysugar biosynthesis in the erythromycin producer *Saccharopolyspora erythraea*, Mol. Gen. Genet. 257 (1998) 542–553.

[19] H. Chen, Z. Guo, H.-w. Liu, Expression, purification, and characterization of TylM1, an N,N-dimethyltransferase involved in the biosynthesis of mycaminose, J. Am. Chem. Soc. 120 (1998) 9951–9952.

[20] H. Chen, G. Agnihotri, Z. Guo, N.L.S. Que, X.H. Chen, H.-w. Liu, Biosynthesis of mycarose: Isolation and characterization of enzymes involved in the C-2 deoxygenation, J. Am. Chem. Soc. 121 (1999) 8124–8125.

[21] C. Olano, A.M. Rodriguez, J.-M. Michel, C. Mendez, M.-C. Raynal, J.A. Salas, Analysis of a *Streptomyces antibioticus* chromosomal region involved in oleandomycin biosynthesis, which encodes two glycosyltransferases responsible for glycosylation of the macrolactone ring, Mol. Gen. Genet. 259 (1998) 299–308.

[22] G. Draeger, S.H. Park, H.G. Floss, Mechanism of the 2-deoxygenation step in the biosynthesis of the deoxyhexose moieties of the antibiotics granaticin and oleandomycin, J. Am. Chem. Soc. 121 (1999) 2611–2612.

[23] I. Aguirrez-Balaga, C. Olano, N. Allende, L. Rodriguez, A. Brana, C. Mendez, J. Salas, Identification and expression of genes involved in biosynthesis of L-oleandrose and its intermediate L-olivose in the oleandomycin producer *Streptomyces antibioticus*, Antimicrob. Agents Chemother. 44 (2000) 1266–1275.

[24] M.D. Schulman, S.L. Acton, D.L. Valentino, B.H. Arison, Purification and identification of dTDP-oleandrose, the precursor of the oleandrose units of the avermectins, J. Biol. Chem. 265 (1990) 16965–16970.

[25] T. MacNeil, K.M. Gewain, D. MacNeil, Deletion analysis of the avermectin biosynthetic genes of *Streptomyces avermitilis* by gene cluster displacement, J. Bacteriol. 175 (1993) 2552–2563.

[26] M.D. Schulman, D. Valentino, S. Streicher, C. Ruby, *Streptomyces avermitilis* mutants defective in methylation of avermectins, Antimicrob. Agents Chemother. 31 (1987) 744–747.

[27] K. Madduri, J. Kennedy, G. Rivola, A. Inventi-Solari, S. Filippini, G. Zanuso, A.L. Colombo, K.M. Gewain, J.L. Occi, D.J. MacNeil, C.R. Hutchinson, Production of the antitumor drug epirubicin (4′-epidoxorubicin) and its precursor by a genetically engineered strain of *Streptomyces peucetius*, Nat. Biotechnol. 16 (1998) 69–74.

[28] M.J. Bibb, J. White, J.M. Ward, G.R. Janssen, The mRNA for the 23S rRNA methylase encoded by the *ermE* gene of *Saccharopolyspora erythraea* is translated in the absence of a conventional ribosome-binding site, Mol. Microbiol. 14 (1994) 533–545.

[29] J. Vara, M. Lewandowska-Skarbek, Y.G. Wang, S. Donadio, C.R. Hutchinson, Cloning of genes governing the deoxysugar portion of the erythromycin biosynthetic pathway in *Saccharopolyspora erythraea* (*Streptomyces erythreus*), J. Bacteriol. 171 (1989) 5872–5881.

[30] R. McDaniel, S. Ebert-Khosla, D.A. Hopwood, C. Khosla, Engineered biosynthesis of novel polyketides, Science 262 (1993) 1546–1550.

[31] R. Ziermann, M.C. Betlach, Recombinant polyketide synthesis in

Streptomyces*: engineering of improved host strains, BioTechniques 26 (1999) 106–110.

[32] H. Decker, S. Haag, G. Udvarnoki, J. Rohr, Novel genetically engineered tetracenomycins, Angew. Chem. Int. Ed. Engl. 34 (1995) 1107–1110.

[33] L. Zhao, D.H. Sherman, H.-w. Liu, Biosynthesis of desosamine: construction of a new methymycin/neomethymycin analogue by deletion of a desosamine biosynthetic gene, J. Am. Chem. Soc. 120 (1998) 10256–10257.

[34] S.-E. Wohlert, G. Blanco, F. Lombó, E. Fernandez, A.F. Brana, S. Reich, G. Udvarnoki, C. Mendez, H. Decker, J. Frevert, J.A. Salas, J. Rohr, Novel hybrid tetracenomycins through combinatorial biosynthesis using a glycosyltransferase encoded by the *elm* genes in cosmid 16F4 and which shows a broad sugar substrate specificity, J. Am. Chem. Soc. 120 (1998) 10596–10601.

[35] S.A. Borisova, L. Zhao, D.H. Sherman, H.-w. Liu, Biosynthesis of desosamine: construction of a new macrolide carrying a genetically designed sugar moiety, Org. Lett. 1 (1999) 133–136.

[36] L. Zhao, J. Ahlert, Y. Xue, J.S. Thorson, D.H. Sherman, H.-w. Liu, Engineering a methymycin/pikromycin-calicheamycin hybrid: construction of two new macrolides carrying a designed sugar moiety, J. Am. Chem. Soc. 121 (1999) 9881–9882.

[37] E. Künzel, B. Faust, C. Oelkers, U. Weissbach, D.W. Bearden, G. Weitnauer, L. Westrich, A. Bechthold, J. Rohr, Inactivation of the *urdGT2* gene, Which encodes a glycosyltransferase responsible for the C-glycosyltransfer of activated D-olivose, leads to formation of the novel urdamycins I, J, and K, J. Am. Chem. Soc. 121 (1999) 11058–11062.

[38] H. Yamase, L. Zhao, H.-w. Liu, Engineering a hybrid sugar biosynthesis pathway: Production of L-rhamnose and its implication on dihydrostreptose biosynthesis, J. Am. Chem. Soc. 122 (2000) 12397–12398.

[39] C. Olano, N. Lomovskaya, L. Fonstein, J.T. Roll, C.R. Hutchinson, A two-plasmid system for the glycosylation of polyketide antibiotics: bioconversion of ε-rhodomycinone to rhodomycin D, Chem. Biol. 6 (1999) 845–855.

[40] R. McDaniel, A. Thamchaipenet, C. Gustafsson, H. Fu, M. Betlach, M. Betlach, G. Ashley, Multiple genetic modifications of the erythromycin polyketide synthase to produce a library of novel ‘unnatural’ natural products, Proc. Natl. Acad. Sci. USA 96 (1999) 1846–1851.

[41] Q. Xue, G. Ashley, C.R. Hutchinson, D.V. Santi, A multi-plasmid approach to preparing large libraries of polyketides, Proc. Natl. Acad. Sci. USA 96 (1999) 11740–11745.

[42] P.J. Solenberg, P. Matsushima, D.R. Stack, S.C. Wilkie, R.C. Thompson, R.H. Baltz, Production of hybrid glycopeptide antibiotics in vitro and in *Streptomyces toyocaensis*, Chem. Biol. 4 (1997) 195–202.

[43] P.A. Patten, R.J. Howard, W.P.C. Stemmer, Applications of DNA shuffling to pharmaceuticals and vaccines, Curr. Opin. Biotechnol. 8 (1997) 724–733.

[44] J. Sambrook, E.F. Fritsch, T. Maniatis, Molecular Cloning: A Laboratory Manual, 2nd edn., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1989.

[45] K.M. Gerwain, J.L. Occi, F. Foer, D.J. MacNeil, Vectors for generating nested deletions and facilitating subcloning G+C-rich DNA between *Escherichia coli* and *Streptomyces* sp., Gene 119 (1992) 149–150.

[46] D.A. Hopwood, M.J. Bibb, C.P. Smith, J.M. Ward, H. Schrempf, Genetic Manipulation of *Streptomyces*: A Laboratory Manual, The John Innes Foundation, Norwich, 1985.

[47] R.W. Burg, B.M. Miller, E.E. Baker, J. Birnbaum, S.A. Currie, R. Hartman, Y.L. Kong, R.L. Monaghan, G. Olson, I. Putter, J.B. Tunac, H. Wallick, E.O. Stapley, R. Oiwa, S. Omura, Avermectins, new family of potent anthelmintic agents: producing organism and fermentation, Antimicrob. Agents Chemother. 15 (1979) 361–367.

[48] T.S. Chen, B.H. Arison, V.P. Gullo, E.S. Inamine, Further studies on the biosynthesis of avermectins, J. Ind. Microbiol. 4 (1989) 231–237.
