
insight review articles

Plant pathogens and integrated defence responses to infection

J effery L. Dangl* & J onathan D. G. J ones†

*Department of Biology and Curriculum in Genetics, Coker Hall 108, CB#3280, University of North Carolina at Chapel Hill, Chapel Hill, North Carolina 27599-3280, USA (e-mail: dangl@email.unc.edu)  
†The Sainsbury Laboratory, John Innes Centre, Colney, Norwich NR4 7UH, UK (e-mail: jonathan.jones@bbsrc.ac.uk)

Plants cannot move to escape environmental challenges. Biotic stresses result from a battery of potential pathogens: fungi, bacteria, nematodes and insects intercept the photosynthate produced by plants, and viruses use replication machinery at the host's expense. Plants, in turn, have evolved sophisticated mechanisms to perceive such attacks, and to translate that perception into an adaptive response. Here, we review the current knowledge of recognition-dependent disease resistance in plants. We include a few crucial concepts to compare and contrast plant innate immunity with that more commonly associated with animals. There are appreciable differences, but also surprising parallels.

Most plants are resistant to most plant pathogens. Passive protection against pathogens that are not specialized to attack a specific host is provided by waxy cuticular ‘skin’ layers and preformed anti-microbial compounds. Plant pathogens can be broadly divided into those that kill the host and feed on the contents (necrotrophs) and those that require a living host to complete their life cycle (biotrophs). Microbial necrotrophy is often accompanied by production of toxins. Viruses are quintessential biotrophs, although infection can lead eventually to host cell death. Bacteria and fungi can adopt either lifestyle. Many insects cause damage by chewing. They induce a wound response that includes the production of protease inhibitors and other anti-feedants such as alkaloids. Additionally, wound responses include release of volatiles which attract insects that feed on, or deposit eggs into, the larvae of the herbivorous insect. By contrast, sap-feeding insects and nematodes can adopt more intimate and sophisticated modes of biotrophic parasitism, imposing developmental responses on the plant cells, leading to the appearance of galls, root knots or cysts. The plant innate immune response is highly polymorphic in its capacity to recognize and respond to biotrophs, and we focus here on this aspect of plant defence.

Resistance in hosts and avirulence in pathogens

Plant–pathogen interactions, particularly those involving biotrophic parasites, are governed by specific interactions between pathogen *avr* (avirulence) gene loci and alleles of the corresponding plant disease resistance (*R*) locus. When corresponding *R* and *avr* genes are present in both host and pathogen, the result is disease resistance. If either is inactive or absent, disease results. The simplest model that accounts for this genetic interaction requires that *R* products recognize *avr*-dependent signals and trigger the chain of signal-transduction events that culminates in activation of defence mechanisms and an arrest of pathogen growth. *R* genes specify a polymorphic component of a particular recognition event. Specific *R*-mediated innate immunity is superimposed onto one or more basal defence pathways. Basal defences inhibit pathogen spread after successful infection and onset of disease. The existence of basal

defence is inferred from the identification of mutants that are more susceptible to a virulent pathogen than are their parents (detailed below). Genetic overlap between specific and basal resistance responses suggests that one function of *R*-mediated signalling is to more rapidly and effectively activate defence mechanisms that are shared by both pathways²⁻⁴.

A significant effort by several laboratories in the past 5–10 years has resulted in the identification of many *R* genes from model and crop species⁵⁻⁷. Functional *R* genes isolated so far encode resistance to bacterial, viral, fungal, oomycete and even nematode and insect pathogens with very different lifestyles, outside or inside the plant cell. Despite this wide range of pathogen taxa and their presumed pathogenicity effector molecules, *R* genes encode only five classes of proteins (Fig. 1).

The largest class of *R* genes encodes a ‘nucleotide-binding site plus leucine-rich repeat’ (NB-LRR) class of proteins (Fig. 1). These function, so far, exclusively as *R* genes and they are highly evolved (see below) for that function. Although computer analyses do not predict localization, at least one NB-LRR protein is associated with the plasma membrane⁸. Their most striking structural feature is a variable number of carboxy-terminal LRRs. LRR domains are found in diverse proteins and function as sites of protein–protein interaction, peptide–ligand binding and protein–carbohydrate interaction⁹,¹⁰. In addition, each *R* protein contains a conserved nucleotide-binding (NB) site, which in other proteins is critical for ATP or GTP binding¹¹. But it is not clear how or which of these nucleotides is bound. The nucleotide-binding site is part of a larger domain that includes additional homology between *R* proteins and some eukaryotic cell death effectors such as Apaf-1 and Ced4 (Fig. 2). This enlarged region is termed the NB-ARC or Ap-ATPase domain¹²,¹³. By analogy with Apaf-1 function, activation of *R* proteins may involve Avr-dependent release of the Ap-ATPase domain from inhibition by the C-terminal LRRs, followed by multimerization of a complex that recruits additional proteins to the amino-terminal domain for further signalling events. The NB-LRR class can be subdivided based on deduced N-terminal structural features: many have a domain with homology to the intracellular signalling domain of the *Drosophila* Toll and mammalian interleukin (IL)-1 receptors (TIR-NB-LRR),

Figure 1 Representation of the location and structure of the five main classes of plant disease resistance proteins. Xa21 and Cf-X proteins carry transmembrane domains and extracellular LRRs. The recently cloned RPW8 gene product carries a putative signal anchor at the N terminus. The Pto gene encodes a cytoplasmic Ser/Thr kinase, but may be membrane associated through its N-terminal myristoylation site. The largest class of R proteins, the NB-LRR class, are presumably cytoplasmic (although they could be membrane associated) and carry distinct N-terminal domains.

![Diagram](https://i.imgur.com/yourimageurl.png)

NB-LRRs

whereas others contain putative coiled-coil domains (CC-NB-LRR). The CC-NB-LRR class probably comprises multiple subfamilies, varying in size and in the location of the coiled-coil domain.

Comparative sequence analyses demonstrated that $R$ specificity resides largely in the LRRs, which are under diversifying selection to increase amino-acid variability in residues thought to be solvent exposed ${ }^{14-18}$. Construction of domain chimaeras has supported these findings for both NB-LRR and extracellular LRR classes of $R$ proteins ${ }^{19-22}$. Recent evidence indicates that in the $L$ class of flax rust resistance genes, diversifying selection also acts on residues in the TIR domain, and that these residues are apparently co-evolving with the corresponding LRR domain to provide specificity ${ }^{23}$. Mechanisms for the evolution of new specificities include unequal recombination and gene conversion, as well as accumulation of amino-acid codon exchanges in members of anciently duplicated gene families.

### R-gene diversity

The complete *Arabidopsis* sequence permits a comprehensive analysis of the diversity of NB-LRR $R$-gene sequences in one plant ${ }^{24}$. Annotation revealed ~150 sequences with homology to the NB-LRR class of $R$ genes. $R$ homologues are unevenly distributed between chromosomes, with 49 on chromosome 1, 2 on chromosome 2, 16 on chromosome 3, 28 on chromosome 4, and 55 on chromosome 5. Not all of these seem intact. Despite the fact that many previously isolated $R$ genes seem to reside in local multigene families, there are 46 singleton *Arabidopsis* $R$-gene homologues, 25 doublets, 7 loci with three copies, and individual loci with four, five, seven, eight and nine NB-LRR-encoding genes. In recent months, the $RPP7$ family has been defined as an additional cluster of ~14 copies on chromosome 1 (A. Cuzick and E. Holub, personal communication). A continuously updated annotation of *Arabidopsis* $R$ genes by B. Meyers and colleagues can be found at http://pgfsun.ucdavis.edu/niblrrs/At_RGenes/. There are more TIR-NB-LRR genes (~60%) than CC-NB-LRR genes (~40%). Both inverted and direct repeats of *Arabidopsis* $R$ genes exist, at a ratio of about 3:2. The largest clusters are at the $RPP5$ locus, which carries the $RPP4$ gene and seven other $RPP5$ homologues, and at the complex $RPP7$ locus on chromosome 1.

Thus, *Arabidopsis* has ~100 $R$ loci distributed over all the chromosomes. This seems a surprisingly small number of genes to mediate recognition of all possible pathogen-encoded ligands. Several models could explain this. Perhaps many $R$ proteins actually perceive the presence of more than one Avr protein, whether that Avr protein comes from pathogens of similar or different lifestyles. 'Dual recognition' has been demonstrated in a few cases. For example, $RPM1$ recognizes two non-homologous avr genes ${ }^{25,26}$, the tomato $Mi$

NATURE | VOL 411 | 14 JUNE 2001 | www.nature.com
© 2001 Macmillan Magazines Ltd

insight review articles

(reviewed in refs 49, 50). There are probably ~15–20 TLR genes in the human genome, and perhaps twice that number of TIR domain-containing proteins (D. Golenbock, personal communication). TLRs recognize a limited, but highly conserved and common, set of pathogen-encoded structures that may represent signatures for a given pathogen class. Recent reports of combinatorial functions of TLR proteins suggest a modest expansion of the germline repertoire<sup>51</sup>. However, it seems that the overall recognition potential is limited, albeit to important and non-mutable ligands such as lipopolysaccharide (LPS).

In addition, there is a class of intracellular NB-LRR proteins that also have a role in animal innate immunity. Surprisingly, these are structurally analogous to the NB-LRR class of plant R proteins. These mammalian Nod proteins contain N-terminal caspase-activating recruitment domains (CARDs) and NB-ARC/Ap-ATPase domains, like Apaf-1. But the Nod proteins also carry C-terminal LRRs, like plant R proteins, instead of WD-40 repeats, like Apaf-1. Intriguingly, Nod1 (also called CARD4) confers recognition of bacterial LPS and subsequent NF-κB activation in a TLR4-independent manner<sup>52</sup>. Although Nod1 is broadly expressed in many cell types, Nod2 is expressed primarily in monocytes, a key cell type that binds and can engulf bacteria in the animal innate immune system<sup>53</sup>. NF-κB activation following Nod2 stimulation occurs probably through a direct interaction with the RICK Ser/Thr protein kinase, mediated by the Nod2 CARD domains<sup>53</sup>. Most important, mutations in *Nod2* have recently been implicated in Crohn’s disease<sup>54,55</sup>, an inflammatory bowel disorder that phenotypically resembles an autoimmune disease. The LRR domains of both Nod1 and Nod2 are required for function. The existence of ~30 *Nod* genes with similar NB-LRR structures, but possessing varied N-terminal domains, suggests mammals may carry a system of intracellular receptors that, like plant R proteins, determines recognition of intracellular ligands. There is currently no evidence that polymorphism in either the TLR or Nod proteins further contributes to diversity. It could be that subsequent to the evolution of the adaptive immune system, there was no evolutionary pressure for expansion of TLRs in animals as they were already adapted to recognize key, non-mutable pathogen-encoded ligands.

**Complex evolution**

If several proteins in the recognition and response pathway are functionally polymorphic, then the optimal set of proteins will need to evolve together. The existence of cytoplasmic and transmembrane classes of R protein indicates that some are specialized to detect secreted ligands or surface components from the pathogen, and some are dedicated to recognize ligands that appear inside the cell (see below). The discovery of multiple types of both intracellular and transmembrane R proteins suggests that polymorphism may exist not only in recognition but also in several elements in response pathways. In plant breeding, this polymorphism may be uncovered experimentally in simple pairwise comparisons of resistant and susceptible inbred host lines. Yet it is probable that selective pressure is not acting on only a single pathogen recognition element, assuming that the LRRs are either directly or indirectly responsible for ligand contact. Selection could act to diversify and then fine-tune the output of the response. NB-LRR proteins probably work in complexes. Thus, it may be that genetic buffering<sup>56</sup>, whereby evolutionary experiments in polymorphism are protected by redundancy, and by flexible, yet robust signalling processes, facilitates phenotypic variation. This, in turn, allows a flexible evolutionary space in which to diversify several elements of the system.

Because most proteins work in complexes, it is perhaps unsurprising that co-evolution of the components is required for optimization. This strategy is most easily detected in outbred populations, and can be revealed as quantitative trait differences among inbred species. Hamilton *et al.*<sup>57</sup> proposed that the main selective advantage leading to the retention of sexual reproduction and

**LRR receptor polymorphism in animal innate immunity**

Animal innate immune systems also use LRR receptors, of the extracellular variety, called Toll-like receptors or TLRs (named after the first member identified; Fig. 3). The mammalian and *Drosophila* innate immune receptors couple to internal cell-death signals, kinase cascades and effector arms that are transcriptionally activated

outcrossing is that polymorphism at loci contributing to parasite recognition will restrict loss of fitness due to disease. According to this model, if a host population is extremely heterogeneous in its recognition capacity, then most isolates of the parasite will not be able to grow on most hosts. In the absence of outcrossing, such polymorphism would be more likely to be lost, unless it is maintained by selection (Fig. 4). Furthermore, if sexual recombination between parasites leads to exchange of dominant avirulence genes, then most progeny of most parasites will not be able to find a host. There is still debate about whether such frequency-dependent selection, in which rare resistance (recognition) specificities are less likely to be overcome by the parasite, is the main explanation for the enormous diversity of human haplotypes at major histocompatibility complex (MHC) loci. An alternative model proposes that this diversity can be explained by overdominance (heterozygote advantage), through which heterozygotes have twice the recognition capacity and resistance of any homozygote. Many plant species, including *Arabidopsis*, reproduce by self-fertilization, and overdominance cannot explain the extreme polymorphism of *R* loci compared with other loci in such species. This inference is of general significance, because it implies that frequency-dependent selection could be part of the explanation for MHC polymorphism in animals, and is perhaps even sufficient to explain it. The restriction of parasite success in plant varietal mixtures is consistent with this overall concept, and the approach deserves further exploration as a strategy to provide more durable resistance in crop varieties (Fig. 4).

NB-LRR proteins are probably intracellular and are likely to be receptors for an *avr*-encoded ligand, or to function in a protein complex that is the functional receptor. Although a cytoplasmic location of an R protein is unsurprising for those conferring resistance to viral pathogens, the existence of intracellular NB-LRR R proteins active against microbial pathogens implies that the ligands from bacterial and fungal pathogens are also intracellular. Plant and animal bacterial pathogens, like *P. syringae*, use a type III delivery system to traffic proteins into host cells (reviewed in refs 58, 59). Avr-R recognition for several bacterial systems (reviewed in ref. 60) occurs inside the plant cell following expression of *avr* genes using plant transcriptional control signals. Curiously, expression of bacterial Avr proteins in

disease-susceptible plants can lead to delayed, weak cytotoxic effects, suggesting that Avr proteins may have additional targets inside disease-susceptible plant cells<sup>61-63</sup>. Based on analogy to mammalian pathosystems, it is inferred that type III effectors from phytopathogenic bacteria are translocated into the host cell, although direct demonstrations of this are rare<sup>64</sup>. Despite these recent advances, little is known about how the subcellular localization, and site of action, of type III effector proteins influences initiation of disease, or resistance, in hosts of the appropriate genotype<sup>65</sup>.

Many fungal pathogens form intimate membrane contacts with host cells at the surface of specialized feeding structures called haustoria, which could facilitate the traffic of disease effectors into the host. It will be of great significance to isolate *avr* genes from haustorial pathogens, such as rusts, powdery mildews and downy mildews. Several secreted Avr proteins from *Cladosporium fulvum* have been defined that trigger resistance mediated by transmembrane R proteins of the Cf-X family<sup>66-69</sup>. However, they do not seem to interact directly with their corresponding Cf-X proteins. Indeed, Avr9 peptide binds with 70-picomolar affinity to a plasma membrane protein that is present even in tomato lines that lack Cf-9 (ref. 70). The rice blast (*Magnaporthe grisea*) *avrPita* gene has been isolated; it encodes a putative secreted metalloprotease. The corresponding R protein (Pita) is of the CC-NB-LRR class (although the LRRs are rather degenerate), and AvrPita and Pita interact directly in yeast two-hybrid and *in vitro* experiments<sup>71</sup>. Although *M. grisea* is not a haustorial pathogen, this suggests that fungal and downy mildew Avr proteins might be secreted proteins that are internalized by or translocated into the plant cell and recognized intracellularly.

### R proteins as guards of cellular machinery

The 'guard hypothesis' provides an intriguing conceptual framework for the action of disease effectors and the R-protein complex. It was put forward in an attempt to rationalize why Pto protein kinase requires the NB-LRR protein Prf to activate defence upon recognition of AvrPto<sup>72</sup>. According to this model, Pto is a general component of host defence, perhaps in a pathway for response to nonspecific elicitors of phytopathogenic bacteria. The function of AvrPto for *P. syringae* is to target Pto and suppress this nonspecific

**Figure 3** Comparison of R proteins with proteins involved in animal innate immunity. Toll in *Drosophila*, and TLRs in mammals, are activated by extracellular ligands. They carry extracellular LRRs (such as Xa21 and Cf-9), but also carry a cytoplasmic TIR domain. Upon activation, the TIR domain recruits death domain-containing adaptor proteins, and the protein kinases PELLE or IRAK. PELLE and IRAK are the animal proteins that are most homologous to Pto. Pto requires the NB-LRR protein Prf to function.

insight review articles

Avr proteins. Each scenario is consistent with the conceptual framework that R proteins monitor whether a cellular protein is under attack from a pathogen effector protein.

There are several predictions from the guard hypothesis. First, R proteins may interact constitutively with their 'guardees'. Such interactors will be targets of virulence factors required for successful infection, and/or components of defence signalling<sup>74</sup>. These possibilities are not mutually exclusive if the nominal target of virulence has also evolved a role in the stability of an R-containing complex whose structural integrity is required for resistance. For example, the viral coat protein of turnip crinklevirus is the Avr protein detected by HRT in this case. It interacts in yeast with an *Arabidopsis* NAC protein, probably a transcriptional regulator. Mutants in the coat protein that do not interact with this NAC protein are still virulent, but lose the ability to trigger HRT-mediated resistance<sup>75</sup>. Thus, this particular NAC protein is required for HRT function. It could also be required for basal defence and be a target for the coat protein, if the latter can also act as a virulence factor. Second, the complex of the effector with its target might be present in both susceptible and resistant host cells, except that in the latter the R protein will also be part of the complex. This suggests that a mutation in an R-protein partner might result in a loss of resistance conferred by that R protein, perhaps explaining the *pbs1* mutant phenotype. Third, if a limited number of host protein complexes are targets of the effector set from a given pathogen, then one host protein complex might be a target for multiple pathogen effector molecules. This suggests that multiple R proteins could physically associate with the same host protein complex and hence with each other. This notion may explain the functional interference of two structurally related R proteins<sup>76,77</sup>, and the counter-intuitive finding that a particular type III effector could be co-immunoprecipitated with the 'wrong' R protein<sup>78</sup>. Fourth, a corollary of these predictions is that the number of host cellular targets of type III effectors may in fact be limited, and potentially targeted by several effectors. This is supported by at least two examples of a single R protein having the ability to recognize two different effectors<sup>26,27</sup>. In addition, the *Arabidopsis* PBS1 gene is also a Ser/Thr kinase and could be the RPS5 'guardee'. Further support is provided by the fact that 46 of the *Arabidopsis* NB-LRR genes are single copy. This implies that they are ancient and provide effective resistance. Their effectiveness could be due to strict structural constraints on the corresponding type III effectors that are in turn targeted to a limited set of cellular targets. For the Cf-X class of R protein, Cf-9 could 'guard' the Avr9-binding site that is present even in stocks that lack the Cf-9 gene<sup>79</sup>.

The guard hypothesis is by no means proven, but it does provide a step beyond the previous notion that R proteins are simply direct receptors for Avr proteins. This elicitor/receptor model may still be true for some systems, but for many others, the lack of direct R/Avr interaction is sufficiently convincing that it can be excluded.

Signal transduction and the effector arm

Genetic screens, almost exclusively in *Arabidopsis*, have defined loci required for R-gene action. Such loci are likely to encode proteins that function either as guardees (described above) or to mediate the series of biochemical events outlined below<sup>2,80</sup>. Note that a protein required for assembly of a cellular component targeted by a type III effector and guarded by an R protein might have multiple functions. Several mutants were identified by loss of a particular R function, and then tested for loss of additional R functions. Some of the resulting mutations are R specific, as they eliminate one specificity, whereas others define common steps in signal-transduction pathways required for the action of several R genes. This is a critical experimental step, as it is often easier to measure subtle effects on R-gene functions that are different than the one used in the original screening. These screens are inefficient; typically ~90% of the mutants are *r* alleles<sup>81</sup>. These results suggest that most mutations in the other genes required for the R signal pathway in question might be lethal, or that these steps are encoded by genes with overlapping or redundant functions.

Loci required for basal defence have also been defined²⁸⁰. Mutants in these loci express enhanced disease susceptibility (eds) phenotypes and support more growth of virulent pathogens than the wild-type plant. Some of them also are required for the function of one or more R genes, and some are also required for pathogen-nonspecific systemic acquired resistance (SAR). Their requirement during R-dependent signalling proves that basal and specific defence systems can overlap.

Several genes required for multiple R functions are known. The ndr1 and eds1 mutants were defined in screens for loss of race-specific resistance to strains of the bacterium *P. syringae* or the oomycete *Peronospora parasitica*⁸²,⁸³. EDS1 and NDR1 are each required for the function of different NB-LRR R genes⁸⁴. The R genes suppressed by the ndr1 mutation are not affected by eds1 mutants, and vice versa. eds1 suppresses TIR-NB-LRR R genes, whereas ndr1 suppresses a subset of CC-NB-LRR resistance proteins. Although these observations suggest a model in which EDS1 and NDR1 mediate distinct R gene-dependent signalling pathways, there are several examples of CC-NB-LRR R genes which function independently of both EDS1 and NDR1 (refs 84, 85). RAR1 is required for the function of some, but not all, barley *Mla* resistance genes⁸⁶. Differences in the amino-acid sequence between proteins encoded by RAR1-dependent and RAR1-independent *Mla* alleles are only around 5% (refs 87, 88). Thus, signalling proteins can discriminate between highly related R proteins. RAR1 is a novel zinc-coordinating protein, which, in higher metazoans, is co-linear with a protein domain homologous to the yeast SGT1 protein. SGT1 regulates delivery of targets to the SCF protein degradation machine⁸⁹. Thus, upon activation, RAR1 may target a negative regulator of disease resistance and the hypersensitive response for degradation. Alternatively, the RAR1-containing SCF complex is a critical host target for a variety of powdery mildew effectors, and is therefore guarded by the products of several *Mla* alleles.

![](image.png)

Figure 5 The guard hypothesis for R-protein function. **a**, A cellular complex of proteins (blue), which includes both the 'guardee' molecule (red) and an NB-LRR protein (grey, shaded from the N terminus through NB and LRR domains), is a target for a bacterial type III effector of disease (orange). **b**, Binding of the type III effector to its targets results in disassociation and activation of the NB-LRR protein and thus disease resistance. **c**, Alternatively, the NB-LRR protein may not be part of the target complex until after type III effector binding. **d**, Recruitment to the type III effector/target complex would then activate the NB-LRR protein.

The earliest events following R engagement are calcium influx, alkalinization of the extracellular space, protein kinase activation, production of reactive oxygen intermediates (ROIs) and nitric oxide, and transcriptional reprogramming. Studies using both nonspecific and race-specific elicitors have documented the chain of events that most rapidly ensue upon pathogen perception by plants. Plant cell cultures are more amenable to reproducible biochemical and pharmacological study than are whole plants. Elicitation of parsley cells with the PEP13 peptide derived from a non-race-specific elicitor from *Phytophthora megasperma*, elicitation of *Arabidopsis* cells with a conserved peptide derived from bacterial flagellin, and tobacco cells expressing the tomato *Cf-9* gene elicited by the Avr9 peptide system have revealed essentially similar processes⁹⁰⁻⁹². Changes in ion fluxes, including calcium influx, occur within minutes of elicitation. Subsequently, ROIs (including H₂O₂ and/or O₂⁻) are produced and mitogen-activated protein kinase and other protein kinase pathways are activated⁹³,⁹⁴. The ROIs may be involved in pathogen elimination, subsequent signalling of downstream effector functions, or (most likely) both. Studies on whole plants using bacterial strains recognized by NB-LRR R genes have revealed similar processes⁹⁵. In addition, nitric oxide (NO) has been shown to accumulate through an as yet unidentified biosynthetic pathway⁹⁶⁻⁹⁸.

Within 15 minutes, a set of new transcripts can be identified, comprising ~1% of total messenger RNA; these encode additional signalling molecules including protein kinases and transcription factors⁹⁹. The protein kinases are upstream or independent of the oxidative burst, and presumably result in the activation of latent transcription factors required for defence gene activation¹⁰⁰. NO and ROI could also contribute to rapid transcriptional activation of a battery of 'defence genes' in and surrounding the infected cell. Functions of these defence genes include biosynthesis of salicylic acid, induction of ethylene biosynthesis, cell-wall strengthening, lignification, production of various antimicrobial compounds, and a form of rapid cell death termed the 'hypersensitive response' (reviewed in ref. 101, and see review in this issue by Lam, Kato and Lawton, pages 848–853). It is, however, still unclear which of these events are causal mediators of R-gene action, and which are not. In addition to local resistance to infection, this set of events can also lead to establishment of SAR¹⁰².

Transcriptional reprogramming establishes the 'effector' arm of the plant innate immune system. The defence response is clearly controlled by interacting signalling pathways. For example, in tobacco and *Arabidopsis*, enzymatic blocking of salicylic acid accumulation¹⁰³, or mutation of the *EDS16/SID2* which blocks salicylic acid biosynthesis¹⁰⁴, seriously impairs basal defence locally and induction of SAR in distal tissues. A key to understanding systemic signalling was the identification and isolation of the *Arabidopsis NPR1/NIM1* gene¹⁰⁵,¹⁰⁶. This gene transduces a salicylic acid-dependent signal to distal tissues, functions in a local basal-defence pathway, and is required for the action of a small number of tested R genes, but is not generally required for R-gene function. Ethylene signalling is also important in basal defence and can be required combinatorially with *NPR1/NIM1* for function of at least the *Arabidopsis* CC-NB-LRR gene *RPS2* (ref. 107). Jasmonic acid mediates wound responses and responses to necrotrophic fungal pathogens. Interestingly, the jasmonic acid signal pathway can act antagonistically to the salicylic acid pathway, allowing the response to be marshalled in a focused manner¹⁰⁸,¹⁰⁹. Large-scale studies of gene expression profiling are beginning to dissect this transcriptional output. The impact of co-regulatory circuits is beginning to be appreciated, with the finding that the cis-element bound by the plant-specific WRKY transcription factors is the common element in a set of SAR co-regulated genes¹¹⁰.

**The road at 'Rest and Be Thankful'**

It is traditional in summing up to joyfully celebrate the past decade's substantial achievements by those on whose shoulders we now stand, while soberly and seriously looking ahead at the new horizons that have come into view. There is a parking area and scenic view on a

insight review articles

small backroad in western Scotland between Loch Lomond and the sea where the tourist is bid to ‘Rest and be Thankful’. Looking backwards, down the tortuous route climbed to this point, the traveller breathes relief. However, a look forward, down the road yet to run, reveals more of the same twists and drops. The field retains many enduring challenges and mysteries. Molecular geneticists need to grapple with identifying the *avr* genes of fungal pathogens that potentially traffic disease effectors into the host via haustoria. The completion of the *Arabidopsis* genome sequence, and the imminent completion of the genomes of several model bacterial plant pathogens (*Xylella fastidiosa* and *X. citri*, *Ralstonia solanacearum* and *P. syringae* DC3000) provide a rich new field for bioinformaticists and cell biologists to investigate gene function. It is hoped that in the next few years the genome sequences of rice blast (*M. grisea*), powdery mildew (*Blumeria graminis*) and other fungal pathogens will become available to the public sector. The availability of expression arrays and new proteomics tools will enable a complete transcriptional and post-transcriptional description of the defence response, at least in model plants. Such descriptive work is an essential prelude to further investigations of mechanisms. Despite the 7 years that have elapsed since the isolation of the first *R* genes, there is a great deal still to learn about how *R* proteins function to confer Avr recognition. The challenge of determining how *R* proteins work requires some change in focus towards biochemistry and cell biology. And despite some plausible interpretations, there is still a great need to do more field experiments to study how *R* genes work in natural populations, and to test approaches using genetic polymorphism to provide more durable disease resistance in crops.

1. Flor, H. H. Current status of the gene-for-gene concept. *Annu. Rev. Phytopathol.* **9**, 275–296 (1971).
2. Glazebrook, J., Rogers, E. E. & Ausubel, F. M. Use of Arabidopsis for genetic dissection of plant defense responses. *Annu. Rev. Genet.* **31**, 547–569 (1997).
3. Yang, Y., Shah, J. & Klessig, D. F. Signal perception and transduction in plant defense responses. *Genes Dev.* **11**, 1621–1639 (1997).
4. McDowell, J. M. & Dangl, J. L. Signal transduction in the plant innate immune response. *Trends Biochem. Sci.* **25**, 79–82 (2000).
5. Bent, A. Function meets structure in the study of plant disease resistance genes. *Plant Cell* **8**, 1757–1771 (1996).
6. Ellis, J., Dodds, P. & Pryor, T. Structure, function, and evolution of plant disease resistance genes. *Curr. Opin. Plant Biol.* **3**, 278–284 (2000).
7. Jones, J. D. G. Putting knowledge of plant disease resistance genes to work. *Curr. Opin. Plant Biol.* (in the press).
8. Boyes, D. C., Nam, J. & Dangl, J. L. The *Arabidopsis thaliana RPM1* disease resistance gene product is a peripheral plasma membrane protein that is degraded coincident with the hypersensitive response. *Proc. Natl Acad. Sci. USA* **95**, 15849–15854 (1998).
9. Jones, D. A. & Jones, J. D. G. The roles of leucine-rich repeats in plant defences. *Adv. Bot. Res. Adv. Plant Pathol.* **24**, 90–167 (1996).
10. Kajava, A. V. Structural diversity of leucine-rich repeat proteins. *J. Mol. Biol.* **277**, 519–527 (1998).
11. Saraste, M., Sibbald, P. R. & Wittinghofer, A. The P-loop—a common motif in ATP- and GTP-binding proteins. *Trends Biotechnol.* **15**, 430–435 (1990).
12. Aravind, L., Dixit, V. M. & Koonin, E. V. The domains of death: evolution of the apoptosis machinery. *Trends Biochem.* **24**, 47–53 (1999).
13. van der Biezen, E. A. & Jones, J. D. G. Homologies between plant resistance gene products and regulators of cell death in animals. *Current Biol.* **8**, R226–R227 (1998).
14. Parniske, M. *et al.* Novel disease resistance specificities result from sequence exchange between tandemly repeated genes at the *Cf-4/9* locus of tomato. *Cell* **91**, 821–832 (1997).
15. McDowell, J. M. *et al.* Intragenic recombination and diversifying selection contribute to the evolution of Downy Mildew resistance at the *RPP8* locus of Arabidopsis. *Plant Cell* **10**, 1861–1874 (1998).
16. Botella, M. A. *et al.* Three genes of the Arabidopsis *RPP1* complex resistance locus recognize distinct *Peronospora parasitica* avirulence determinants. *Plant Cell* **10**, 1847–1860 (1998).
17. Meyers, B. C. *et al.* The major resistance gene cluster in lettuce is highly duplicated and spans several megabases. *Plant Cell* **10**, 1817–1832 (1998).
18. Michelmore, R. W. & Meyers, B. C. Clusters of resistance genes in plants evolve by divergent selection and a birth-and-death process. *Genome Res.* **8**, 1113–1130 (1998).
19. Wang, G.-L. *et al.* *Xa21D* encodes a receptor-like molecule with a leucine-rich repeat domain that determines race-specific recognition and is subject to adaptive evolution. *Plant Cell* **10**, 765–779 (1998).
20. Ellis, J. G., Lawrence, G. J., Luck, J. E. & Dodds, P. N. Identification of regions in alleles of the flax rust resistance gene *L* that determine differences in gene-for-gene specificity. *Plant Cell* **11**, 495–506 (1999).
21. Wulff, B. B. H., Thomas, C. M., Smoker, M., Grant, M. & Jones, J. D. G. Domain swapping and gene shuffling identify specificity determinants required for induction of an Avr-dependent hypersensitive response by the tomato *Cf-4* and *Cf-9* proteins. *Plant Cell* **13**, 255–272 (2001).
22. Van der Hoorn, R. A. L., Roth, R. & De Wit, P. J. G. Identification of distinct specificity determinants in resistance protein *Cf-4* allows construction of a *Cf-9* mutant that confers recognition of avirulence protein AVR4. *Plant Cell* **13**, 273–285 (2001).
23. Luck, J. E., Lawrence, G. J., Dodds, P. N., Shepherd, K. W. & Ellis, J. G. Regions outside of the leucine-rich repeats of Flax Rust resistance proteins play a role in specificity determination. *Plant Cell* **12**, 1367–1377 (2000).

24. Initiative, T. A. G. Analysis of the genome of the flowering plant *Arabidopsis thaliana*. *Nature* **408**, 796–815 (2000).
25. Bisgrove, S. R., Simonich, M. T., Smith, N. M., Sattler, N. M. & Innes, R. W. A disease resistance gene in Arabidopsis with specificity for two different pathogen avirulence genes. *Plant Cell* **6**, 927–933 (1994).
26. Grant, M. R. *et al.* Structure of the *Arabidopsis RPM1* gene enabling dual specificity disease resistance. *Science* **269**, 843–846 (1995).
27. Rossi, M. *et al.* The nematode resistance gene *Mi* of tomato confers resistance against the potato aphid. *Proc. Natl Acad. Sci. USA* **95**, 9570–9754 (1998).
28. Cooley, M. B., Pathirana, S., Wu, H.-J., Kachroo, P. & Klessig, D. F. Members of the Arabidopsis *HRT/RPP8* family of resistance genes confer resistance to both viral and oomycete pathogens. *Plant Cell* **12**, 663–676 (2000).
29. van der Vossen, E. A. *et al.* Homologues of a single resistance-gene cluster in potato confer resistance to distinct pathogens: a virus and a nematode. *Plant J.* **23**, 567–576 (2000).
30. Stahl, E. A., Dwyer, G., Mauricio, R., Kreitman, M. & Bergelson, J. Dynamics of disease resistance polymorphism at the *RPM1* locus of Arabidopsis. *Nature* **400**, 667–671 (1999).
31. Riely, B. K. & Martin, G. B. Ancient origin of pathogen recognition specificity conferred by the tomato disease resistance gene *Pto*. *Proc. Natl Acad. Sci. USA* **98**, 2059–2064 (2001).
32. Dangl, J. L. in *Bacterial Pathogenesis of Plants and Animals: Molecular and Cellular Mechanisms* (ed. Dangl, J. L.) 99–118 (Springer, Heidelberg, 1994).
33. Medzhitov, R. *et al.* MyD88 is an adaptor protein in the hToll/IL-1 receptor family signaling pathways. *Mol. Cell* **2**, 253–258 (1998).
34. Dinesh-Kumar, S. P. & Baker, B. J. Alternatively spliced N resistance gene transcripts: their possible role in tobacco mosaic virus resistance. *Proc. Natl Acad. Sci. USA* **97**, 1908–1913 (2000).
35. Eulgem, T., Rushton, P. J., Robatzek, S. & Somssich, I. E. The WRKY superfamily of plant transcription factors. *Trends Plant Sci.* **5**, 199–206 (2000).
36. Grant, M. R. *et al.* Independent deletions of a pathogen-resistance gene in *Brassica* and *Arabidopsis*. *Proc. Natl Acad. Sci. USA* **95**, 15843–15848 (1998).
37. Xiao, S. *et al.* Broad-spectrum mildew resistance in *Arabidopsis thaliana* mediated by *RPW8*. *Science* **291**, 118–120 (2001).
38. Noël, L. *et al.* Pronounced intraspecific haplotype divergence at the *RPP8* complex disease resistance locus of Arabidopsis. *Plant Cell* **11**, 2099–2111 (1999).
39. Scofield, S. R. *et al.* Molecular basis of gene-for-gene specificity in bacterial speck disease of tomato. *Science* **274**, 2063–2065 (1996).
40. Tang, X. *et al.* Physical interaction of *avrPto* and the *Pto* kinase defines a recognition event involved in plant disease resistance. *Science* **274**, 2060–2063 (1996).
41. Salmerón, J. M. *et al.* Tomato *Prf* is a member of the leucine-rich repeat class of plant disease resistance genes and lies embedded within the *Pto* kinase gene cluster. *Cell* **86**, 123–133 (1996).
42. Song, W.-Y. *et al.* A receptor kinase-like protein encoded by the rice disease resistance gene, *Xa21*. *Science* **270**, 1804–1806 (1995).
43. He, Z. *et al.* Perception of Brassinosteroids by the extracellular domain of the receptor kinase BRI1. *Science* **288**, 2360–2363 (2000).
44. Wang, Z.-Y., Seto, H., Fujioka, S., Yoshida, S. & Chory, J. BRI1 is a critical component of a plasma-membrane receptor for plant steroids. *Nature* **410**, 380–383 (2001).
45. Jones, D. A., Thomas, C. M., Hammond-Kosack, K. E., Balint-Kurti, P. J. & Jones, J. D. G. Isolation of the tomato *Cf-9* gene for resistance to *Cladosporium fulvum* by transposon tagging. *Science* **266**, 789–793 (1994).
46. Trotochaud, A. E., Hao, T., Wu, G., Yang, Z. & Clark, S. E. The *CLAVATA1* receptor-like kinase requires *CLAVATA3* for its assembly into a signaling complex that includes KAPP and a Rho-related protein. *Plant Cell* **11**, 393–405 (1999).
47. Gómez-Gómez, L. & Boller, T. FLS2: an LRR receptor like kinase involved in the perception of the bacterial elicitor flagellin in Arabidopsis. *Mol. Cell* **5**, 1003–1011 (2000).
48. Swiderski, M. R. & Innes, R. W. The Arabidopsis *PBS1* resistance gene encodes a member of a novel protein kinase subfamily. *Plant J.* **26**, 101–112 (2001).
49. Aderem, A. & Ulevitch, R. J. Toll-like receptors in the induction of the innate immune response. *Nature* **406**, 782–787 (2000).
50. Medzhitov, R. & Janeway, C. Jr The Toll receptor family and microbial recognition. *Trends Microbiol.* **8**, 452–456 (2000).
51. Ozinsky, A. *et al.* The repertoire for pattern recognition of pathogens by the innate immune system is defined by cooperation between Toll-like receptors. *Proc. Natl Acad. Sci. USA* **97**, 13766–13771 (2000).
52. Inohara, N., Ogura, Y., Chen, F. & Núñez, G. Human *Nod1* confers responsiveness to bacterial lipopolysaccharides. *J. Biol. Chem.* **276**, 2551–2554 (2001).
53. Ogura, Y. *et al.* *Nod2*, a *Nod1/Apa1* family member that is restricted to monocytes and activates NF-κB. *J. Biol. Chem.* **276**, 4812–4818 (2001).
54. Hugot, J.-P. *et al.* Association of *NOD2* leucine-rich repeat variants with susceptibility to Crohn’s disease. *Nature* **411**, 599–603 (2001).
55. Ogura, Y. *et al.* A frameshift in *Nod2* associated with susceptibility to Crohn’s disease. *Nature* **411**, 603–606 (2001).
56. Hartman, J. L.IV, Garvik, B. & Hartwell, L. Principles for the buffering of genetic variation. *Science* **291**, 1001–1004 (2001).
57. Hamilton, W. D., Axelrod, R. & Tanese, R. Sexual reproduction as an adaptation to resist parasites. *Proc. Natl Acad. Sci. USA* **87**, 3566–3573 (1990).
58. Gálan, J. E. & Collmer, A. Type III secretion machines: bacterial devices for protein delivery into host cells. *Science* **284**, 1322–1328 (1999).
59. Hueck, C. J. Type III protein secretion systems in bacterial pathogens of animals and plants. *Microbiol. Mol. Biol. Rev.* **62**, 379–433 (1998).
60. Kjemtrup, S., Nimchun, Z. & Dangl, J. L. Effector proteins of phytopathogenic bacteria: bifunctional signals in virulence and host recognition. *Curr. Opin. Microbiol.* **3**, 73–78 (2000).
61. Gopalan, S. *et al.* Expression of the *Pseudomonas syringae* avirulence protein *AvrB* in plant cells alleviates its dependence on the hypersensitive response and pathogenicity (Hrp) secretion system in eliciting genotype-specific hypersensitive cell death. *Plant Cell* **8**, 1095–1105 (1996).
62. McNellis, T. W. *et al.* Glucocorticoid-inducible expression of a bacterial avirulence gene in transgenic *Arabidopsis thaliana* induces hypersensitive cell death. *Plant J.* **14**, 247–258 (1998).
63. Nimchun, Z. *et al.* Eukaryotic fatty acylation drives plasma membrane targeting and enhances function of several Type II effector proteins from *Pseudomonas syringae*. *Cell* **101**, 353–363 (2000).
64. Mudgett, M. & Staskawicz, B. Characterization of the *Pseudomonas syringae pv. tomato AvrRpt2*

protein: demonstration of secretion and processing during bacterial pathogenesis. *Mol. Microbiol.* **32**, 927–941 (1999).

65. Nimchun, Z., Rohmer, L., Chang, J. H. & Dangl, J. L. Knowing the dancer from the dance: R gene products and their interactions with other proteins from host and pathogen. *Curr. Opin. Plant Biol.* (in the press).

66. Laugé, R., Joosten, M. H. A., Van den Ackerveken, G. F. J. M., Van den Broek, H. W. J. & De Wit, P. G. M. The in-planta produced extracellular proteins ECP1 and ECP2 of *Cladosporium fulvum* are virulence factors. *Mol. Plant-Microbe Interact.* **10**, 725–734 (1997).

67. Laugé, R. *et al.* Successful search for a resistance gene in tomato targeted against a virulence factor of a fungal pathogen. *Proc. Natl Acad. Sci. USA* **95**, 9014–9018 (1998).

68. Laugé, R., Dmitriev, A. P., Joosten, M. H. A. J. & De Wit, P. G. J. M. Additional resistance gene(s) against *Cladosporium fulvum* present on the Cf-9 introgression segment are associated with strong PR protein accumulation. *Mol. Plant-Microbe Interact.* **11**, 301–308 (1998).

69. Laugé, R., Goodwin, P. H., de Wit, P. J. & Joosten, M. H. Specific HR-associated recognition of secreted proteins from *Cladosporium fulvum* occurs in both host and non-host plants. *Plant J.* **23**, 735–745 (2000).

70. Kooman-Gersman, M., Honée, G., Bonnema, G. & De Wit, P. J. G. M. A high-affinity binding site for the AVR9 peptide elicitor of *Cladosporium fulvum* is present on plasma membranes of tomato and other Solanaceous plants. *Plant Cell* **8**, 929–938 (1996).

71. Bryan, G. T. *et al.* A single amino acid difference distinguishes resistant and susceptible alleles of the rice resistance gene *Pi-ta*. *Plant Cell* **12**, 2033–2045 (2000).

72. van der Biezen, E. A. & Jones, J. D. G. Plant disease resistance proteins and the “gene-for-gene” concept. *Trends Biochem. Sci.* **23**, 454–456 (1998).

73. Oldroyd, G. E. D. & Staskawicz, B. J. Genetically engineered broad-spectrum disease resistance in tomato. *Proc. Natl Acad. Sci. USA* **95**, 10300–10305 (1998).

74. Chen, Z., Kloek, A. P., Boch, J., Katagiri, F. & Kunkel, B. N. The *pseudomonas syringae avrRpt2* gene product promotes pathogenicity from inside the plant cell. *Mol. Plant Pathol.* **13**, 1312–1321 (2000).

75. Ren, T., Qu, F. & Morris, T. J. *HRT* gene function requires interaction between a NAC protein and viral capsid protein to confer resistance to Turnip Crinkle Virus. *Plant Cell* **12**, 1917–1925 (2000).

76. Ritter, C. & Dangl, J. L. Interference between two specific pathogen recognition events mediated by distinct plant disease resistance genes. *Plant Cell* **8**, 251–257 (1996).

77. Reuber, T. L. *et al.* Correlation of defense gene induction defects with powdery mildew susceptibility in *Arabidopsis* enhanced disease susceptibility mutants. *Plant J.* **16**, 473–485 (1998).

78. Leister, R. T. & Katagiri, F. A resistance gene product of the nucleotide binding site-leucine rich repeats class can form a complex with bacterial avirulence proteins *in vitro*. *Plant J.* **22**, 345–354 (2000).

79. Dixon, M. S., Golstein, C., Thomas, C. M., van Der Biezen, E. A. & Jones, J. D. Genetic complexity of pathogen perception by plants: the example of Rcr3, a tomato gene required specifically by Cf-2. *Proc. Natl Acad. Sci. USA* **97**, 8807–8814 (2000).

80. Glazebrook, J. Genes controlling expression of defense responses in Arabidopsis—2001 status. *Curr. Opin. Plant Biol.* (in the press).

81. Jørgensen, J. H. Genetic analysis of barley mutants with modifications of the powdery mildew resistance gene *Mla12*. *Genome* **30**, 129–132 (1988).

82. Century, K. S., Holub, E. B. & Staskawicz, B. J. *NDR1*, a locus of *Arabidopsis thaliana* that is required for disease resistance to both a bacterial and a fungal pathogen. *Proc. Natl Acad. Sci. USA* **92**, 6597–6601 (1995).

83. Parker, J. E. *et al.* Characterization of *eds1*, a mutation in *Arabidopsis* suppressing resistance to *Peronospora parasitica* specified by several different *RPP* genes. *Plant Cell* **8**, 2033–2046 (1996).

84. Aarts, N. *et al.* Different requirements for *EDS1* and *NDR1* by disease resistance genes define at least two R gene mediated signalling pathways in *Arabidopsis*. *Proc. Natl Acad. Sci. USA* **95**, 10306–10311 (1998).

85. McDowell, J. M. *et al.* Downy mildew (*Peronospora parasitica*) resistance genes in *Arabidopsis* vary in functional requirements for *NDR1*, *EDS1*, *NPR1*, and Salicylic Acid accumulation. *Plant J.* **22**, 523–530 (2000).

86. Freialdenhoven, A. *et al.* *Nar-1* and *Nar-2*, two loci required for *Mla-12*-specified race resistance to powdery mildew in barley. *Plant Cell* **6**, 983–994 (1994).

87. Zhou, F. *et al.* Cell-autonomous expression of barley *Mla1* confers race-specific resistance to the powdery mildew fungus via a *Rar1* independent signaling pathway. *Plant Cell* **13**, 337–350 (2001).

88. Halterman, D., Zhou, F., Wei, F., Wise, R. P. & Schulze-Lefert, P. The *Mla6* coiled-coil, NBS-LRR protein functions in barley and wheat to confer resistance specificity to *Blumeria graminis f. sp. hordei*. *Plant J.* (in the press).

89. Kitigawa, K., Skowyra, D., Elledge, S. J., Harper, J. W. & Hieter, P. *SGT1* encodes an essential component of the yeast kinetochore assembly pathway and a novel subunit of the SCF ubiquitin complex. *Mol. Cell* **4**, 21–33 (1999).

90. Jabs, T., Colling, C., Tschöpe, M., Hahlbrock, K. & Scheel, D. Elicitor-stimulated ion fluxes and reactive oxygen species from the oxidative burst signal defense gene activation and phytoalexin synthesis in parsley. *Proc. Natl Acad. Sci. USA* **94**, 4800–4805 (1997).

91. Felix, G., Duran, J. D., Volko, S. & Boller, T. Plants have a sensitive perception system for the most conserved domain of bacterial flagellin. *Plant J.* **18**, 265–276 (1999).

92. Piedras, P., Hammond-Kosack, K. E., Harrison, K. & Jones, J. D. G. Rapid, Cf-9 and Avr9 dependent, production of active oxygen species in tobacco suspension cultures. *Mol. Plant-Microbe Interact.* **11**, 1155–1166 (1998).

93. Ligterink, W., Kroj, T., zur Nieden, U., Hirt, H. & Scheel, D. Receptor-mediated activation of a MAP kinase in pathogen defense of plants. *Science* **276**, 2054–2057 (1997).

94. Romeis, T. *et al.* Rapid, Avr9- and Cf-9-dependent activation of MAP kinases in tobacco cell cultures and leaves: convergence of resistance gene, elicitor, wound and salicylate responses. *Plant Cell* **11**, 273–287 (1999).

95. Grant, M. *et al.* The *RPM1* plant disease resistance gene facilitates a rapid and sustained increase in cytosolic calcium that is necessary for the oxidative burst and hypersensitive cell death. *Plant J.* **23**, 441–450 (2000).

96. Durner, J., Wendehenne, D. & Klessig, D. F. Defense gene induction in tobacco by nitric oxide, cyclic GMP and cyclic ADP ribose. *Proc. Natl Acad. Sci. USA* **95**, 10328–10333 (1998).

97. Delledonne, M., Xia, Y., Dixon, R. A. & Lamb, C. J. Nitric oxide functions as a signal in plant disease resistance. *Nature* **394**, 585–588 (1998).

98. Delledonne, M., Zeier, J., Marocco, A. & Lamb, C. J. Signal interactions between nitric oxide and reactive oxygen intermediates in the plant hypersensitive disease resistance response. *Proc. Natl Acad. Sci. USA* (in the press).

99. Durrant, W. E., Rowland, O., Piedras, P., Hammond-Kossak, K. E. & Jones, J. D. G. cDNA-AFLP reveals a striking overlap in the race-specific resistance and wound response expression profiles. *Plant Cell* **12**, 963–977 (2000).

100. Dröge-Laser, W. *et al.* Rapid stimulation of a soybean protein serine-threonine kinase which phosphorylates a novel bZIP DNA-binding protein, G/HBF-1, during the induction of early transcription-dependent defenses. *EMBO J.* **16**, 726–738 (1997).

101. Scheel, D. Resistance response physiology and signal transduction. *Curr. Opin. Plant Biol.* **1**, 305–310 (1998).

102. Delaney, T. P. Genetic dissection of acquired resistance to disease. *Plant Physiol.* **113**, 5–12 (1997).

103. Gaffney, T. *et al.* Requirement for salicylic acid for the induction of systemic acquired resistance. *Science* **261**, 754–756 (1993).

104. Nawrath, C. & Métraux, J.-P. Salicylic acid induction-deficient mutants of *Arabidopsis* express PR-2 and PR-5 and accumulate high levels of camalexin after pathogen attack. *Plant Cell* **11**, 1393–1404 (1999).

105. Cao, H., Glazebrook, J., Clark, J. D., Volko, S. & Dong, X. The *Arabidopsis NPR1* gene that controls systemic acquired resistance encodes a novel protein containing ankyrin repeats. *Cell* **88**, 57–64 (1997).

106. Ryals, J. *et al.* The *Arabidopsis NIM1* protein shows homology to the mammalian transcription factor inhibitor IκB. *Plant Cell* **9**, 425–439 (1997).

107. Clarke, J. D., Volko, S. M., Ledford, H., Ausubel, F. M. & Dong, X. Roles of salicylic acid, jasmonic acid, and ethylene in *cpr*-induced resistance in *Arabidopsis*. *Plant Cell* **12**, 2175–2190 (2000).

108. Reymond, P. & Farmer, E. E. Jasmonate and salicylate as global signals for defense gene expression. *Curr. Opin. Plant Biol.* **1**, 404–411 (1998).

109. Reymond, P., Weber, H., Damond, M. & Farmer, E. E. Differential gene expression in response to mechanical wounding and insect feeding in *Arabidopsis*. *Plant Cell* **12**, 707–719 (2000).

110. Maleck, K. *et al.* The transcriptome of *Arabidopsis* during systemic acquired resistance. *Nature Genet.* **26**, 403–410 (2000).

**Acknowledgements**

Work in the Dangl laboratory is supported by grants from the NSF, NIH, DOE, USDA-NRI and Syngenta. The Jones laboratory is supported by the Gatsby Charitable Trust and the BBSRC. We thank P. Epple, T. Eulgem, J. McDowell, S. Peck, J. Rathjen and B. Staskawicz for critical reading of this manuscript, and G. Nuñez and D. Golenbock for useful suggestions.
