
Type I polyketide biosynthesis in bacteria (Part B)

Bernard J. Rawlings

Department of Chemistry, University of Leicester, University Road, Leicester, UK LE1 7RH.
E-mail: bjr2@le.ac.uk or Bernard.Rawlings@le.ac.uk

Received (in Cambridge, UK) 4th January 2001
First published as an Advance Article on the web 23rd May 2001

Covering 1995 to mid-2000. Previous review: 1999, 16, 425

1 Introduction and list of abbreviations used
1.1 Summary of recent highlights
2 Oleandomycin (Ole)
2.1 Synthase (OLES)
2.2 Post-PKS (tailoring)
2.3 Autoresistance
3 Tylosin (Tyl)
3.1 Acyl building units
3.2 Synthase (TYLS)
3.3 Post-PKS (tailoring)
3.4 Deoxysugar biosynthesis and attachment
3.4.1 Mycarose
3.4.2 6-Deoxyallose/D-mycinose
3.4.3 D-Mycaminose
3.5 Resistance
3.6 Regulation
4 Magnamycins and related 16-membered macrolides
4.1 Spiramycin (Srm) and platenomycin
4.2 Niddamycin (Nid)
4.3 Pikromycin (picromycin), narbomycin, methymycin and neomethymycin
4.3.1 Synthase (PIKS)
4.3.2 Post-PKS (tailoring)
4.3.3 Deoxysugar biosynthesis and attachment
4.4 Mycinamicin
4.5 Albocycline
5 Polyethers
5.1 Starter units: branched chain amino acid metabolism
6 Avermectins and related compounds
6.1.1 Starter units
6.1.2 Synthase (AVES)
6.1.3 Post-PKS (tailoring)
7 Polyenes and large ring macrolides
7.1 Nystatin (Nys)
7.1.1 Deoxysugar
7.2 Amphotericin B
7.3 Pimaricin (Pim)
8 Miscellaneous actinomycete Type I polyketides
8.1 Pironetin (Streptomyces)
8.2 Linearmycin (Streptomyces)
8.3 Aflatatins (Streptomyces)
8.4 Pulvomycin (Streptoverticillium)
8.5 Verucopeptin (Actinomadura)
8.6 Azamacrolides (macrolactams)
8.6.1 Vicenistatin
9 Ansamycins
9.1 3-Amino-5-hydroxybenzoic acid (AHBA) —the 'aminoshikimate' pathway
9.2 Rifamycin
9.2.1 Starter unit
9.2.2 Synthase (RIFS)

9.2.3 Post-PKS (tailoring)
9.3 Naphthalenoid and naphthoquinoid ansamycins:
naphthomycins, actamycin and ansatrienin
9.3.1 Naphthomycin
9.3.2 Ansatrienin (mycotrienin)
9.3.3 Mitomycins
10 Manumycins
10.1 3-Amino-4-hydroxybenzoic acid (3, 4-AHBA)
10.2 Manumycin A and asukamycin
11 Rapamycin and related compounds
11.1 Cyclohexanecarboxylic acid (CHC) and 3, 4-dihydroxycyclohexanecarboxylic acid (DHCHC)
11.2 Rapamycin (Rap)
11.2.1 Synthase (RAPS)
11.2.2 Post-PKS (tailoring)
11.3 FK506
11.3.1 Synthase (FKBS)
11.3.2 Post-PKS (tailoring)
11.4 Kendomycin—a carbamacrolide (chalcone synthase starter unit?)
12 Decalin and cyclopropyl containing polyketides
12.1 Decalins and related compounds—assembly using a Diels–Alder reaction
12.1.1 Nargenicin and nodusmicin
12.1.2 Coloradocin
12.1.3 Cochleamycin
12.2 Cyclizidine—a cyclopropyl polyketide
13 Myxobacteria
13.1 Epothilones
13.2 Soraphens
13.3 Chrysophysarin A
13.4 TA (myxovirescen)
13.5 Myxalamid
13.6 Myxotheiazol
14 Pseudomonas
14.1 Pyoluteorin (Plt)
14.2 Coronatine (Cor)
14.3 2,4-Diacetylphloroglucinol
15 Mycobacteria
16 References

1 Introduction and list of abbreviations used
This is the second part (Part B) of a review that summarises the literature for Type I polyketide biosynthesis in bacteria from the end of 1994 through to mid-2000, and should be read in conjunction with Part A (erythromycin biosynthesis) published in the previous issue of *Natural Product Reports*¹ and with other recent reviews on polyacylides (fatty acids, polyketides and non-ribosomal polypeptides) in *Natural Product Reports*,²⁻⁵ and elsewhere.

The full introduction to this review appears in part A, which concentrates on erythromycin. Topics covered include the

DOI: 10.1039/b100191o
Nat. Prod. Rep., 2001, 18, 231–281 231

This journal is © The Royal Society of Chemistry 2001

- Isolated methyl carbon from acetate  
  (not now thought to be part of an intact acetate unit)  

- Isolated carbonyl carbon from acetate  
  (not now thought to be part of an intact acetate unit)  

- Acetate unit, intact  

- Intact acetate unit, including C-O bond  
  from [1-¹³C,¹⁸O]acetate  

- Propionate unit, intact  

- Intact propionate unit, including C-O bond  
  from [1-¹³C,¹⁸O]propionate  

- Intact unit derived from glycolate or glycerol or  
  hydroxymalonate or methoxymalonate  

- Butanoate unit, intact  

- 3-Methylbutanoate unit, intact  

- Malonate unit, intact  

- Succinate unit, intact  

- Glycerol unit, intact  

- Methyl carbon from  
  [¹³CH₃]methionine via SAM  

- From atmospheric dioxygen  

- Hydride from NAD(P)H  

Fig. 1 Biosynthetic symbols used throughout this series of reviews.

Identification of the gene cluster and identification of the catalytic domains, overexpression of the DEBS proteins in *E. coli* and in *S. coelicolor*, characterisation and mechanistic studies on the role of the individual domains and linker regions, and post PKS modifications.

A consistent set of biosynthetic symbols used in these reviews is illustrated in Fig. 1, but unfortunately a confusing variety of conventions is still used in the literature. For a further discussion of these issues, see the Introduction to Part A. The carbon numbering system varies from system to system; the most systematic and frequently used appears to be taking the carbonyl terminus as '1', counting to the acyl chain terminus e.g. in Ery '15'. Any methyl appendages that are part of the parent structure are then labelled in ascending order starting with those closest to the carbonyl carbon as e.g. in Ery A 2 '16', '17' and so on. A natural ethyl appendage would be e.g. 19 and 20 as found in tylactone 185. Sugar appendages are, starting with those closest to the carbonyl carbon, 1', 1" and so on. A disaccharide would be e.g. 1' and 1" with the next monosaccharide on the aglycone ring being 1'", as for mycaminose, mycarose and mycinose respectively in tylosin 191.

The reference sources for the review are given in Part A and also include a recent review on the biosynthesis of aromatic polyketides, isoprenoids, alkaloids. Compound numbering is discussed in Part A.

Abbreviations used in this series of fatty acid or polyketide biosynthesis reviews:  
$A.$, *Aspergillus*; aa, amino acids; ACC, acetyl CoA carboxylase; ACP, acyl carrier protein; ACP0, acyl carrier protein of a loading module; ACPS, holo-acyl carrier protein synthase; Act, Actinorhodin; AD, adenylation domain; AHBA, 3-amino-5-hydroxybenzoic acid; 3,4-AHBA, 3-amino-4-hydroxybenzoic acid; Ans, ansatrienin (mycotriienin); ANSA, series of polyketides containing an aromatic-like portion with a polyketide loop or 'handle'; ANSS, ansatrienin synthase; apo-, without prosthetic group; AS, acyl CoA synthetase; AT, acyl CoA acetyltransferase (acyltransferase); AT0, acetyltransferase of the starter unit/module; AT2°, acetyltransferase of module 2 with active site serine mutated; AT(A), acetyl/malonyl- transferase; AT(P), propionyl/methylmalonyltransferase; ATCC, American Type Culture Collection; Ave, avermectin; AVES avermectin aglycone synthase; $B.$, *Bacillus*; bp, base pair (in DNA); Ccr, (2E)-butenoyl (crotonyl) CoA reductase; Cfa, coronafacic acid; CHC, cyclohexylcarbonyl; CHS, chalcone synthase; CH999, A strain of *S. coelicolor* A3(2) in which the Act genes have been removed; CI, chemical ionisation (mass spectroscopy); CIF, chain initiating factor (previously called CLF or KSβ); CL, carboxylic acid ligase; CLF, ‘chain length factor’ or KSβ the; CoA, Coenzyme A; Cor, coronatine; COSY, correlation spectroscopy (NMR); Cyt, cytochrome; Da, Dalton or RMM; DAHP, 3-deoxy-D-arabino-heptulosonic acid 7-phosphate; 6-dEB, 6-deoxyerythronolide B; DEBS, 6-deoxyerythronolide B synthase; Des, D-desosamine; DHCHC, (1R,3R,4R)-3,4-dihydroxycyclohexylcarbonyl; DHS, dehydroshikimic acid; DHQ, 3-dehydroquinic acid; DQF, double quantum filtered (NMR); dTDP, deoxythymidyl, same as ‘TDP’; DTX, a dinoflagellate toxin; $E.$, *Escherichia*; Eco, $E.$ *coli* proteins; E4P, erythrose-4-phosphate Epo, epothilone; EPOS, epothilone synthase; ESMS, electrospray mass spectrometry; Ery, erythromycin; Ery B, erythromycin B; EtMCoA, ethylmalonyl CoA; Fab, fatty acid biosynthesis proteins; Fkb, FK505; FKBS, FKB506 synthase; HC, heterocyclisation domain in NRPS; HDDH, (3R)-hydroxydecanoyl ACP dehydratase; holo-, with prosthetic group e.g. holo-ACP; Hyg, from the second PKS cluster in *S. hygroscopicus*; KAS, 3-oxoacyl ACP synthase (3-ketoacyl ACP synthase), also called KS or KSa; KAS1°, KAS1 with active site cysteine mutated; KASIII, fatty acid KAS that catalyses the first FAS condensation; kbp, kilobase pair; kDa, kiloDalton; KR, 3-oxoacyl ACP reductase (3-ketoacyl ACP reductase); KR1, ketoreductase in aromatic ring one or Mod1; $M.$, *Mycobacterium*; KSa, ‘KAS’ (aromatic polyketides); KSβ, ‘CLF’ (aromatic polyketides); KAS(Q), a KAS with active site cysteine replaced by glutamine; KAS(S), a KAS with active site cysteine replaced by a serine; MALDI-TOF, Matrix-assisted laser desorption/ionisation time-of-flight mass spectrometry; MCoA, malonyl CoA; Meg, megalomicin; MeT, methyltransferase via SAM; MGT, macrolide glycosyltransferase; MLS, magnamycin-leucomycin-spiramycin resistance; MMCoA, methylmalonyl CoA; Mod, macrolide megasynthase module; MPT, malonyl/palmitoyl CoA ACP transacylase; MS, mass spectrometry; 6MSAS, 6-methyl salicyclic acid synthase; MT, malonyl CoA ACP transacylase; NAC, N-acetyl cysteamine ($N$-acetyl 2-aminoethanethiol); Nap, naphthomycin; Nbm, narbomycin; Nid, niddamycin; NOE, nuclear Overhauser enhancement (NMR); NRPS, nonribosomal peptide synthase; Nsa, norsolorinic acid; Nuc nucleophile/nucleophilic; Nys, nystatin; NYSS, nystatin synthase; Ole, oleandomycin; OLES, (8,8a or) 8,18-deoxyoleandrolide synthase; ORF, open reading frame; PAL, phenylalanine ammonia lyase; PCP, peptidyl carrier protein in NRPS; ppb, parts per billion (NMR isotope shift); P450 (or P-450), cytochrome P₄₅₀ monooxygenase; PIE, pipecolic acid inserting enzyme; Pik, pikromycin (picromycin); Pim, pimaricin; PIMS, pimaricinolide synthase; PK, polyketide PKS, polyketide synthase; PLP, pyridoxal phosphate; Plt, pyoluteorin; PLTS, pyoluteorin synthase; PMP, pyridoxamine phosphate; PMSF, phenylmethylsulfonyl fluoride (serine protease inhibitor); PNP, 4-nitrophenol; PrT, propionyl transferase; Rap, rapamycin; RAPS, rapamycin synthase; REG, regulatory protein/gene; RES, resistance protein/gene; Rif, rifamycin; RIFS, rifamycin synthase; $S.$, *Streptomyces*; Sac., *Saccharopolyspora*; SAM, S-adenosyl methionine; SARP, Streptomyces antibody regulatory protein; SDS, sodium dodecasulfate (gel electrophoresis); SEFT, spin echo fourier transform NMR; SMAT, secondary metabolite aminotransferase gene; Sno, nogalamycin producing genes from *S. nogalater*; Sor, soraphen; SORS, soraphen synthase; Srm, spiromycin; SRMS, spiromycin (platenolid I) synthase; Stc, sterigmatocystin; STS, stilbene synthase; TAL, triacetic acid lactone;

TCA, tricarboxylic acid cycle (citric acid cycle); TDP, always shown as the identical 'dTDP'; TE, acyl ACP thioesterase; Tu, strain culture collection; Tyl, tylosin; TYLS, tylactone synthase; Vdh, valine dehydrogenase.

Abbreviations used in PKS Tables: ✓, active module present; A, AT(A); P, AT(P); B, butanoate/ethylmalonate loading AT; G, glycine loading AT; Y, KAS with tyrosine replacing cysteine in active site; CL carboxyligase; ■ inactive module with apparently intact sequence; □ inactive module with dysfunctional sequence. In gene tables, an arrow indicates direction of transcription, and a black dot indicates an origin of transcription.

### 1.1 Summary of recent highlights (references in main body of text)

The period of this review has seen a flurry of sequencing clusters in prokaryotes responsible for polyketide biosynthesis, mostly from the Streptomycetes, Nocardioforms (e.g., Saccharopolyspora), Myxobacteria, and the Pseudomonadaceae. These clusters usually contain genes encoding proteins responsible for building block assembly, the megasynthase, deoxysugar and aminosugar synthesis and attachment, cytochrome P-450 oxidative enzymes, regulatory proteins, export proteins, and systems for protecting the host organism from its own antibiotic-autoresistance. It now seems that given time and resources, the DNA cluster for any prokaryotic polyketide from these three families (and hopefully soon the mycobacteria) can be located, the DNA cloned, relocated, and analyzed through sequence alignment with known systems and overexpression in a heterologous host.

There has been rapid progress in understanding the biosynthesis and attachment of deoxysugars, and the opportunities available for substituting sugars to alter biological activity. In particular, the mechanism of C-2 deoxygenation has recently been clarified. The genes for the intricate network of regulatory proteins and the factors that affect polyketide production and autoresistance are now better understood. Tylosin in particular appears to have several different mechanisms of autoresistance and regulation.

The syntheses for the polyenes nystatin and pimaricin are now sequenced, and for these very large rings there remains a module for each cycle, however repetitive.

The aminoshikimate pathway is now well established, and the genes required to assemble AHBA (3-amino-5-hydroxybenzoic acid) appear within the polyketide cluster. The product of the first module in the rifamycin synthase (RifA) appears to be oxidized by two P-450's before transfer to the next module (RifB). However, in contrast, the origin of the 3,4-AHBA (3-amino-4-hydroxybenzoic acid) moiety in manumycin and asukamycin is the citric acid cycle.

The metabolites based upon a cyclohexylcarbonyl starter unit, such as rapamycin and FB506, appear to have an ER on the loading domain (to reduce cyclohexenylcarbonyl CoA to cyclohexanycarbonyl CoA?) and an ATP-dependent carboxylic acid ligase to load the starter, rather than an acyltransferase. Instead of a TE, these systems have a pipecolate inserting enzyme (PIE) that not only forms the amide bond but also catalyzes the macrolactonization.

Two groups have recently published the synthase involved in epothilone assembly in a Myxobacterium, and found a loading module, followed by a non-ribosomal peptide synthase module, followed by normal polyketide modules. Interestingly, the additional SAM-derived methyl unit is added by the synthase. The sequence suggests that SAM binds to a SAM binding domain. The superfluous enoylreductase (no alcohol to dehydrate due to absence of active KR) still has an active histidine base that removes a C-2 proton as normal. The resulting enolate then reacts with the bound SAM. The biosynthesis of coronatine by *Pseudomonas syringae* involves both a Type I and Type II synthase. Compounds 1–176 are found in Part A of this review in the previous issue.

## 2 Oleandomycin (Ole)

Oleandomycin 183 is a 14-membered mesolide (macrolide) with two deoxysugars L-oleandrose 176 and D-desosamine 166 attached to an aglycone 8,18-deoxyoleandrolide (8,8a-deoxyoleandrolide) 182 which is identical to 6-dEB 1 except for an acetate starter unit (Scheme 1). Hydroxylation at C-8 and epoxide formation occur after transfer of both sugars onto the

![Chemical Structures](chemical_structures.png)

**Scheme 1**

Nat. Prod. Rep., 2001, 18, 231–281

Table 1 Domain organisation of OLES in S. antibioticus

| OLES | 0 | 1 | 2 | 3 | 4 | 5 | 6 |
|------|---|---|---|---|---|---|---|
| KAS  | Q | ✓ | ✓ |   |   | ✓ | ✓ |
| AT   | A | P | P |   |   | P | P |
| DH   |   |   |   | ✓ | ✓ |   |   |
| ER   |   |   |   | ✓ | ✓ |   |   |
| KR   |   | ✓ | ✓ |   |   | ✓ | ✓ |
| ACP  | ✓ | ✓ | ✓ |   |   | ✓ | TE1 |

---

Table 2 Gene cluster for oleandomycin biosynthesis in S. antibioticus (oleD not in cluster)

| Gene    | Size               | Putative function                                                                 |
|---------|--------------------|------------------------------------------------------------------------------------|
| oleW    | 327aa/35834Da      | 3-Oxoreductase (Oleandrose) 177 to 179                                             |
| oleV    | 474aa/53149Da      | 2,3-Dehydratase (Oleandrose) 160 to 177                                            |
| oleL    | 204aa/22248Da      | dTDP-4-oxo-6-deoxyhexose 5-epimerase (EryB7) 179 to 171                             |
| oleS    | 356aa/38109Da      | dTDP glucose synthase (from glucose phosphate) 158 to 159                          |
| oleE    | 335aa/36664Da      | dTDP-glucose 4,6-dehydratase 159 to 160                                            |
| oleU    | 295aa/32098Da      | 4-Oxoreductase (Oleandrose) (EryB4) 171 to 175                                      |
| oleN1   | 393aa/42673Da      | 3,4-Dehydration (Desosamine) (EryC4) 161 to 164 (part of)                           |
| oleT    | 485aa/53774Da      | 3,4-Reductases (EryC5) 161 to 164 (part of)                                         |
| oleI    | 424aa/42384Da      | Glycosyltransferase involved in autoresistance                                       |
| oleN2   | 369aa/39580Da      | 3-Aminotransferases in desosamine biosynthesis (EryC1) 164 to 165                    |
| oleR    | 768aa/80681Da      | Analogous to EryBl, extracellular glycosidase?                                       |
| oleA1   | 4151aa             | OLES-A1 or OleA1 (Mod0+Mod1+Mod2)                                                   |
| oleA2   | 3817aa             | OLES-A2 (ORFA) or OleA2 (Mod3+Mod4)                                                 |
| oleA3   | 3520aa/368564Da    | OLES-A3 (PKS-ORF3orORFB) or OleA3 (Mod5+Mod6)                                       |
| oleP1   | 401aa/42141Da      | 4,6-Dideoxy-4-oxo-dTDP-D-glucose 3,4-isomerase 160 to 161                           |
| oleG1   | 426aa/46976Da      | Desosaminyltransferase                                                             |
| oleG2   | 426aa/46006Da      | Oleandrosyltransferase                                                             |
| oleM1   | Uncertain start codon | N,N-Dimethyltransferase in desosamine biosynthesis 165 to 166                   |
| oleY    | 486aa/42484Da      | O-MeT 175 to 176?                                                                  |
| oleP    | 1224bp             | P-450 epoxidation                                                                  |
| oleB    | 570aa/61747Da      | ABC transporter                                                                    |

---

aglycone.' Oleandomycin nomenclature conventionally labels C-18 as '8a', but this conflicts with nomenclature in other related systems, so for consistency many compounds are labelled differently to that found in the literature.

### 2.1 Synthase (OLES)

Salas et al. have located and sequenced a section of DNA coding for the ACP of the second multifunctional enzyme complex (OleA2) and the whole of the third complex (ORFB or OleA3). The GC content is abnormally low (67.4%) for a Streptomyces apart from the final 800bp coding for the TE which is normal (92%), suggesting it was a late addition to the gene. In particular, in ORFB many triplet codons end in T (20%), and include many codons very rarely found in Streptomycetes including two TTA codons which are normally reserved as regulatory codons, and are the first examples of these codons in Streptomyces polyketide synthase assembly genes (see pimaricin). The sequence of amino acids coded for by these genes is similar to that for the corresponding Ery proteins; however the protein sequence similarity is much closer than the coding sequences, suggesting convergent evolution. Other differences between Ole and Ery synthase proteins are that the linker regions are about 200aa longer, and that the two modules in OleA3 have a much higher degree of identity (81%) with each other than that with the corresponding Ery gene. It may be that S. antibioticus has only recently acquired these genes (where from?), and that they have not had enough time to diversify or evolve to GC rich sequences. The full domain organisation is shown to be as expected and is illustrated in Table 1, and contains a KAS(Q) in the loading domain, in contrast to DEBS.' However, in this paper there is no indication whether or not there is a dysfunctional KR3 as found in DEBS 2. McDaniel et al. have recently sequenced OleA1 and

the remaining part of OleA2 and published the full domain organisation and produced 8,18-deoxyoleandrolide by heterologously expressing OLES in S. lividans. The aa sequences between DEBS and OLES showed only 45% identity, with even less homology in the loading domain, agreeing with previous suggestions of convergent evolution.

### 2.2 Post-PKS (tailoring)

The gene cluster is shown in Table 2. Salas et al. have over-expressed OleP in E.coli, which is homologous to EryF and EryK P-450 enzymes, which is thought responsible for epoxidation of C-8/C-18 very late in the Ole biosynthetic pathway. McDaniel et al. have co-overexpressed OleP and DEBS in S. lividans and obtained 8,19-dihydroxy-6-deoxyerythronolide B184a (labelled 8,8a in the literature), indicating that OleP is involved in the epoxidation pathway (the '8a' nomenclature is used in oleandrolide 182, but is unconventional in Ery nomenclature where '8a' is carbon '19'). The mechanism of epoxidation is still unclear; it could involve the exomethylene double bond. The observed diol could be a hydrolysed product of epoxide formation. However, it is interesting to speculate that this epoxidation pathway could involve initial dihydroxylation, followed by conversion of the primary hydroxy to a leaving group (phosphate?) followed by nucleophilic ring closure (some antibiotics such as streptomycin are believed to be 'inactivated' by phosphorylation as part of an autoresistance mechanism, see Section 2.3). Thus, in the DEBS+OleP system, the enzyme involved in transforming the primary alcohol into a leaving group is absent, resulting in no epoxidation, only diol184a formation.

Salas et al. have sequenced 6.4kb from the oleandomycin producer S. antibioticus (Table 2) and reported that oleP1 encodes for a P-450 like protein; oleM1 that encodes a methyl-

transferase possibly dimethylating the desosamine amino group; oleG1 and oleG2 encoding for two glycosyltransferases attaching the two sugars to the macrolide ring; and oleY of unknown function, 12 but since assigned as an O-MeT. Salas showed that targeted gene deletion of oleG1 led to accumulation of the aglycone, 8,18-deoxyoleandrolide 182, leading the authors to conclude that oleG1 encoded the oleandrosyltransferase. Presumably, during earlier targeted gene disruption work, polar effects of oleG1 deletion led to inactivation of oleG2 (see below).

Attempts to individually overexpress the glycosyltransferases OleG1 and OleG2 were unsuccessful. A plasmid was constructed which contained both oleG1 and oleG2, along with proximal parts of the operon; insertion into eryC3 or eryB5 mutants of Sac. erythraea restored Ery production, showing that the plasmid encoded both amino sugar and neutral sugar transferase activity. However, when a similar construct containing a disrupted transferase activity was introduced, eryC3 mutants were only complemented by the oleG1 containing construct, and eryB5 complemented by the oleG2 containing plasmid. Thus oleG1 encodes the desosaminyltransferase and oleG2 the oleandrosyltransferase. 13 The product from oleG2 complementation was found to be 3-L-rhamnosyl-6-deoxyerythromycin B, showing that the oleandrosyltransferase has a good affinity for dTDP-L-rhamnose, and that the glycosylation is efficiently competing with the EryF C-6 hydr-oxylase for the common substrate, 6-dEB. Additionally, the EryK C-12 hydroxylase cannot recognise the 3-O-rhamnosyl containing substrate. The origin of the L-rhamnose is not clear, but it may be being generated as a cell wall component of Sac. erythraea. The use of a glycosyltransferase with relaxed substrate specificity, coupled with alternative availability of sugars, may be a practical strategy to the production of novel mesolides.

OleP1 was initially reported by Salas et al. as being 'P-450-like'. 12 Salas et al. then report that oleP1 has a high encoded sequence to similarity to EryC2 and that it was able to complement eryC2 mutants, thus it should be the 4,6-dideoxy-4-oxo-dTDP-D-glucose 2,3-isomerase involved in D-desosamine biosynthesis. However, in this paper oleP1 is also labelled in a figure as orf10 (and oleGl as orf11) and in a later paper 'orf10' and 'orf11' are shown by Floss to encode the 2,3-dehydratase and enoyl reductase. Floss et al. have examined the activity of 'Orf10' (OleP1?) and Orf11' (OleG1?) derived from S. anti-bioticus Tui99 and show that they are the dTDP-4-oxo-4,6-dideoxyglucose 2,3-dehydratase and enoyl reductase analogous to EryB6 and EryB2, which would be involved in L-oleandrose 176 formation (Scheme 2) converting 160 to 180. In the absence of the Orf11 oxoreductase, maltol 178 was formed. 14,15

In a recent paper, Salas et al. report sequencing 8 kbp encoding deoxysugar biosynthesis genes, 16 and assign them functions in the biosynthesis of desosamine and oleandrose. The reported order and functions involve a 3,5-epimerase (OleL) to convert 180 into 175 (dotted arrow), as reported for similar systems in other papers. However, in Scheme 2 OleL has been assigned as the 5-epimerase to convert 179 into 171 (see Scheme 4la in Part A). Salas et al. constructed two plasmids, one containing oleSEVWYU, and the other oleSEVWU (lacking the gene for O-MeT) and inserted them into S. albus that already had the gene coding for dTDP-L-oleandrosyl-transferase OleG2 inserted. The former, when incubated with erythronolide, produced 3-L-oleandrosylerythronolide B (3'-O-methyl184b), and the latter 3-L-olivosylerythronolide B184b. Thus these enzymes assembled dTDP-L-oleandrose, and OleG2 transferred either dTDP-L-olivose or dTDP-L-oleandrose. These are the first plasmids capable of assembling dTDP-sugars, and could potentially be added to a wide variety of aglycones, producing a wide variety of novel glycosylated products.

This area is rapidly moving, with conclusions based upon sequence data being rapidly challenged by new results, and

highlighting the necessity for common conventions and nomenclature to avoid confusing or misleading other workers, and emphasises the need for many of these enzymes to be purified and characterised.

### 2.3 Autoresistance

The ribosomes in S. antibioticus have been found to be sensitive to Ole throughout its production, thus they are not being methylated by an ErmE analogue as in Ery production. 17

Several ATP binding cassette (ABC) transporters have been located in some mesolide (macrolide) producers such as tylosin and oleandomycin (OleB and OleC), which use energy from ATP to ensure rapid export of the mesolide. 18–21 OleB encodes an ABC transporter protein that contains two ATP binding domains but no hydrophobic transmembrane region (unlike most ABC transporters), and has been overexpressed in E. coli. 21 Presumably OleB interacts with a membrane protein that is part of the export system, perhaps OleC. Salas et al. have overexpressed the nucleotide binding portion of OleB (OleB') and characterised its ATP binding properties, and found an interaction between the ABC transporter and oleandomycin in the absence of any membrane component. 22 These ABC transporters may actually be transporting modified glycosylated antibiotic (see below).

There is a ‘second’ mechanism of autoresistance in S. antibioticus, through antibiotic modification, export, then removal of the modification by extracellular enzymes. Whilst some antibiotics such as streptomycin are believed to be phosphorylated, exported and then reactivated by phosphatases, a high proportion of Streptomyces strains have been found to possess macrolide glycosyltransferase (MGT) activity. 23 Salas et al. have purified and characterised a glycosyltransferase (57 kDa) OleD that attaches glucose to the 2"-hydroxy group (for consistency numbering as shown in 183 not 2' as used in the literature) of the desosamine moiety on oleandomycin to give 184, but oleD was not located near the PKS gene cluster. 24–27 Salas et al. have recently overexpressed and purified OleD in S. lividans. 28 The kinetics with another macrolide, lankamycin, were studied, and shown consistent with a compulsory order mechanism, to give glycosylated lankamycin. The authors suggest that OleD glycosylates a wide range of mesolides, whilst OleI, a second glycosyltransferase recently reported and sequenced by Salas and found in the cluster, is a specific oleandomycin glycosyl-transferase.

Salas et al. have recently reported sequencing 5.2 kbp of the gene cluster encoding OleI, a second glycosyltransferase; OleN2 an aminotransferase; and OleR a glycosidase. 29 OleI and the previously located OleD were expressed in S. lividans and partially purified, and both found to glycosylate oleandomycin. OleI was far more substrate specific, with negligible activity with carbomycin, tylosin and erythromycin substrates, whilst OleD was more active with carbomycin as a substrate than oleandomycin. Perhaps OleI is a specific autoresistance mechanism, whilst OleD is for resistance to mesolides produced by competing micro-organisms? Glycosidated oleandomycin had a much smaller inhibitory effect on protein synthesis than oleandomycin. Cell-free extracts containing expressed OleR showed glycosidase activity, and OleR was found in the culture supernatant, suggesting that it is exported from the cell, as previously found. 26 The high homology with EryBl, formerly assumed to have a role in the biosynthesis of mycarose, although recent inactivation did not affect Ery production, 30 suggests that EryBl may have had a mesolide glycosylation role. No Ery glycosyltransferase has yet been found, so EryBl might be an evolutionary fossil from a time when Sac. erythraea had not yet evolved its current ErmE ribosome methylation auto-resistance mechanism, and instead used EryBl in conjugation with an Ery glucosidase enzyme.

D-glucose-1-phosphate
158

α-D-glucose 1-phosphate thymidytransferase
(dTDP-D-glucose synthetase)
Des3 TylA1 AveB3 OleS

dTDP-D-glucose
159

dTDP-D-glucose 4,6-dehydratase
Des4 AveB2 TylA2 NysD3 OleE

4,6-dideoxy-4-oxo-dTDP-D-glucose

4,6-dideoxy-4-oxo-dTDP-D-glucose 3,4-isomerase
TylM3 EryC2 Des8 OleP1

3,6-dideoxy-3-oxo-dTDP-D-glucose
161

PLP
TylB
3-aminotransferase

Me
HO
O-dTDP
162

TylM1
SAM

N,N-dimethyltransferase
OleM1 EryC6 Des6 SAM

dTDP-D-mycaminose
163

dTDP-D-desosamine
166

tylosin

erythromycin
methymycin
pikromycin
neomethymycin
oleandomycin

dTDP-L-megosamine

maltol
178

transaminase (MegD2)
N,N-dimethyltransferase (MegD3)

177c

(4-oxo reductase)
177b

MegD5 or MegD7 and MegD6

Absence of enoylreductase
or NADPH

MegD4 (5-epimerase)

177a

2,3-dehydratase (Zn²⁺)
TylX3 or TylC6 EryB6 AveB6 MegTOleV

O-dTDP

3-epimerase
TylJ

3,4-dehydratase
EryC1 (or EryC4?) Des1 OleN1

Des2
EryC5 (3,4-reductase) OleT

3-aminotransferase
Des5 EryC4 (or EryC1?) OleN2

dTDP-6-deoxyallose
168

TylN
Attach to aglycone
then bis O-methylate
TylE and TylF

dTDP-D-mycaminose
163

dTDP-D-desosamine
166

tylosin

erythromycin
methymycin
pikromycin
neomethymycin
oleandomycin

dTDP-L-olivose
181

3,5-epimerase ?
EryB7

170

C-2 deoxygenase
TylD
4-oxoreductase

dTDP-6-deoxyallose
168

TylN
Attach to aglycone
then bis O-methylate
TylE and TylF

dTDP-D-mycaminose
163

dTDP-D-desosamine
166

tylosin

erythromycin
methymycin
pikromycin
neomethymycin
oleandomycin

dTDP-L-mycarose
173

Attach to aglycone EryB5
then methylate EryG

L-cladinose
174

erythromycin

dTDP-L-olivose
175

SAM
AveB7 OleY

dTDP-L-oleandrose
176

OleG2

avermectins
oleandomycin

Scheme 2 Possible biosynthetic route to deoxysugars and aminosugars used in macrolides (Des refers to desosamine biosynthesis genes in S. venezuelae; Meg refers to megosamine 177c biosynthesis in Micromonospora megalomicaea—see Scheme 4la (Part A). Mycosamine pathway not included.

Salas summarises the overall scheme as the final product of the oleandomycin biosynthetic pathway is really the inactive glycosylated product, which is secreted out of the cell by the ATP driven ABC transporters OleB (and perhaps OleC) where it is then reactivated by the extracellular glycosidase OleR.

Table 3 Domain organisation of tylactone synthase (TYLS) in *S. fradiae* (Q = KAS(Q), KAS with Q replacing active site S; B = AT(B), a butanoate or ethylmalonate loading transferase)

| $S$. fradiae | $tylG1$ | $tylG2$ | $tylG3$ | $tylG4$ | $tylG5 \Rightarrow$ tylactone (protylonolide) |
|--------------|---------|---------|---------|---------|---------------------------------------------|
| TYLS         |         |         |         |         |                                             |
| KAS          | $Q$     | $\checkmark$ | $\checkmark$ |         |                                     |
|              | $P$     | $P$     | $P$     |         |                                     |
| AT           |         | $\checkmark$ | $A$     |         |                                     |
| DH           |         | $\checkmark$ | $\checkmark$ |         |                                     |
| ER           |         | $\checkmark$ | $B$     |         |                                     |
| KR           |         | $\checkmark$ | $\checkmark$ |         |                                     |
| ACP          | $\checkmark$ | $\checkmark$ | $\checkmark$ |         |                                     |

![Chemical Structures](https://i.imgur.com/chemical_structures.png)

Scheme 3 Biosynthesis of tylosin.

In a recent paper, Salas *et al.* report characterising the product of adding UDP-$\alpha$-D-glucose, oleandomycin and OleI.³¹ The glucosylated oleandomycin 184 was purified, examined by NMR and shown to possess a $\beta$-glycosidic linkage indicating that inversion at the anomic centre had occurred. A detailed presentation of data, kinetics and possible mechanisms is given.

### 3 Tylosin (Tyl)

Tylosin 191 (*S*. fradiae) is a commercially important antibiotic as a veterinary drug and animal growth promoter. The Tyl gene cluster is 85 kbp long, with 43 genes including $tylGI$ to $tylG5$ responsible for assembling from two acetate, five propionate and one butanoate units the 16-membered aglycone tylactone

#### 3.1 Acyl building units

L-Valine 202 can be converted into either methylmalonate 207 or ethylmalonate 208 and used as a source of acyl building blocks, as illustrated in Scheme 4. Disruption of the valine dehydrogenase gene $vdh$ disrupts tylosin and spiramycin production, which can be restored by the addition of propionate and butanoate to the medium; however disruption of $vdh$ in acetate derived polyketide producers such as *S*. coelicolor has no effect on e.g. actinorhodin production, implying that valine dehydrogenase has an important role in the supply of

Nat. Prod. Rep., 2001, 18, 231–281

pyruvate

$\alpha$-ketoisovaleric acid

Me  
Me  CO₂H  
NH₂  

L-leucine  
192  

Me  
Me  CO₂H  
O  

4-methyl-2-oxopentanoic acid  
193  

Me  
Me  COSCoA  

3-methylbutanoic acid  
194  

Me  
Me  COSCoA  

3-methylbut-2-enoic acid  
195  

acetyl CoA  
acetoacetyl CoA  

Me  
Me  CO₂H  
NH₂  

L-isoleucine  
198  

Me  
Me  CO₂H  
O  

3-methyl-2-oxopentanoic acid  
199  

Me  
Me  COSCoA  

2-methylbutanoyl CoA  
200  

Me  
Me  COSCoA  

2-methylbut-2-enoyl CoA  
(tiglic acid)  
201  

acetyl CoA  
propionyl CoA  
methylmalonyl CoA  

OH  
CO₂H  
NH₂  

L-threonine  
196  

transaminase  
(PLP dependent)  

Me  
CO₂H  
O  

2-oxobutanoate  
197  
($\alpha$-ketobutyrate)  

acetohydroxy acid synthase  
acetohydroxy acid isomerase  
dihydroxyacid dehydratase  
transamination  

Me  
Me  CO₂H  
NH₂  

L-valine  
202  

Me  
Me  CO₂H  
O  

3-methyl-2-oxobutanoic acid  
203  

Me  
Me  COSCoA  

2-methylpropanoyl CoA  
204  

acyl CoA  
dehydrogenase  

Me  
H₂C=  
COSCoA  

2-methylprop-2-enoyl CoA  
(methacrylyl CoA)  
206  

HO₂C  
Me  COSCoA  

methylmalonyl CoA  
207  

vitamin B₁₂  

Me  
COSCoA  

butanoyl CoA  
205  

ethylmalonyl CoA  
208  

Scheme 4 Conversion of branched chain amino acids into starter units.

precursors such as propionate and butanoate, rather than polyketide synthesis as a whole. The original valine dehydrogenase (Vdh1) reported from *S. fradiae*³² acted as twelve 18 kDa subunits, quite unlike the Vdh's reported from other PK producers. Bêhal *et al.* could not detect this original Vdh1, and have isolated a second Vdh (Vdh2) from *S. fradiae* that occurs as a dimer, each subunit being 41 kDa,³³ and suggest that expression of Vdh1 may be unstable. Hutchinson *et al.* have cloned and sequenced the gene for the Vdh2 from *S. coelicolor*.³⁴

### 3.2 Synthase (TYLS)

The synthase sequence has been placed in databases by Dehoff *et al.*, but the domain organisation (sadly) has not been published. The five synthase genes occupy 41 kbp, and are presumably similar to those for platenolide (*vide infra*) and the domain organisation is illustrated in Table 3. The final synthase protein (TylG5) ends with an integral TE, and further downstream there is an additional TE (Orf5), TE2, with $M_r$ 32 150 Da, that shows homology with Ery Orf5 and TE’s from duck uropygial glands and rat mammary glands, both of which produce short chain fatty acids. Leadlay first suggested that such discrete TE’s might act as an editing mechanism, removing aberrant growing polyketides from ‘stalled’ synthase proteins, freeing them up for fresh acyl chain synthesis. These small discrete TE’s would be like egrets removing ticks from the backs of elephants. They would forage around the surface of the multienzyme-complex hunting for any acyl chains to hydrolyse, or perhaps probe into the interior of the complex to find acyl chains. They need not be specific; any protein lumbered with an aberrant acyl chain it cannot process would have such a long lifetime that it would be more likely hydrolysed by an unspecific TE than a rapidly processed ‘true’ acyl chain, a few of which no doubt are also hydrolysed as well. Whilst there are many advantages for a system to have so many parts so intimately linked
together, these multienzyme complexes performing e.g. 70 chemical transformations also suffer, as the probability of one thing going ‘wrong’ relative to a single discrete enzyme must be ca. seventy-fold larger. There would be such a large ‘investment’ by the organism in synthesising any one synthase complex, that it would need a ‘repair’ mechanism. It could not afford to lose the entire complex every time a ‘mistake’ was made in just one of the ca. 70 steps involved.

The role of the discrete TE2 has been examined by Cundliffe et al.³⁵ Disruption of *tylO* (*orf5*) prevented any tylosin production, reduced overall ‘tylosin like polyketide’ production by over 85%, accumulating two minor products, one unidentified, and the second identified as demycarosyltylosin, formed due to a polar effect the disruption has had on a neighboring gene *tylC6* (*orf6*, also called *tylX3*) involved in mycarose biosynthesis. Addition of tylactone to the *tylO* disrupted fermentation led to the efficient production of demycarosyltylosin, indicating that any effect had been on tylactone formation. Complementation with *tylO* dramatically raised production levels of demycarosyltylosin, but did not lead to any tylosin production, indicating that *tylC6* was cotranscribed with *tylO*. Replacement of *tylO* by the integral TE (part of *tylG5*) did not raise production. *NbmB* is a discrete TE2 gene located immediately downstream of *nbmA*, the synthase genes for the 14-membered macrolide narbomycin production in *S. narbonensis*, and adjacent to the integral TE. Replacement of *tylO* by *nbmB* resulted in restoration of demycarosyltylosin production to levels around 40% of that seen following complementation by *tylO*. Thus a TE2 associated with the formation of a 14-membered ring macrolide could moderately efficiently function in the formation of a 16-membered mesolide. Studies by Schneider and Marahiel involving discrete TE’s in nonribosomal peptide synthases suggest that they may also have an editing function.³⁶ Heathcote et al. have reported³⁷a that the TylO TE2 when overexpressed in *E. coli* and purified, showed greatest hydrolytic activity to the 4-nitrophenyl and NAC thioesters of propionate over acetate, butanoate and other thioesters, indicating that the real role of TE2 might be the removal of enzyme linked propionate resulting from decarboxylation of methylmalonate occurring before reaction with KAS bound growing acyl chain.

In a recent paper Staunton et al. investigate the *in vitro* behaviour of TylO TE2 towards NAC and PNP esters to obtain a substrate specificity profile.³⁷b The overexpressed (His₆-tagged in *E. coli*) protein (10 mg/g wet cells) was monomeric, unlike TE1 that behaves as a dimer. NAC thioesters were examined as close structural analogues of the natural substrates (release of thiolate determined using Ellman’s reagent), and 4-nitrophenol (PNP) esters to explore the structure specificity of the rate determining break up of the acyl-enzyme intermediate. The propionate NAC thioester was hydrolysed faster than butanoate > acetate > pentanoate > Tyl 3-*epi*-diketide (2R,3S)-2-methyl-3-hydroxypentanoate NAC thioester. The *K*~cat~/*K*~M~ for propionate was 14-fold greater than that for the ‘incorrect’ diketide, suggesting that propionate was the substrate, rather than incorrectly processed acyl chains ‘stuck’ on the synthase. The relative rates for PNP esters was propionate > butanoate > pentanoate > acetate > ‘incorrect’ triketide. Many experiments have now shown that early reports of aberrant KAS catalysed non-condensative decarboxylation of methylmalonate in the DEBS1/TE system were due to impurities in the enzyme system or propionate contamination in methylmalonate. However, cell-free extracts of DEBS3 can catalyse the decarboxylation of MMCoA to generate a triketide lactone, so the occurrence of aberrant decarboxylation may occur, and TE2 would as its primary role rapidly hydrolyse propionate thioesters, regenerating active synthase, with only as a secondary role, editing out long lived aberrant acyl chains. There are four cysteine residues on TE2, that the authors show are readily derivatised by Ellman’s reagent. It is intriguing

whether these cysteines play a role in causing TE2’s to bind tightly to specific surface thiols on the synthase (rather than randomly foraging over the whole surface), guarding each active site. Addition of TE2’s to purified protein may lead to co-purification.

### 3.3 Post-PKS (tailoring)

Early studies³⁸ indicate that the putative main pathway from tylactone 185 to tylosin 191 is: addition of dTDP-D-myca-minose 163 by TylM2 to C-5 hydroxy group to give 5-O-β-D-mycaminosyltylactone 186, followed by oxidations at the C-20 methyl to the aldehyde by TylI and at the C-23 methyl to the alcohol by TylH1 to give 5-O-β-D-mycaminosyltylonolide. This is followed by addition of dTDP-6-deoxyallose 168 by TylN to give demethylactenocin 188, and addition of dTDP-L-mycarose 173 to the mycaminose C-4 hydroxy group by TylC5 to give 2″-O-demethylmacrocin 189, and then the 6-deoxyallose moiety is firstly 2″-O-methylated by TylE to give macrocin 190 and 3″-O-methylated by TylF to give the D-mycinose 169 moiety as part of tylosin 191 (Scheme 3). However, this order is not obligatory.³⁹

### 3.4 Deoxysugar biosynthesis and attachment

Cundliffe et al. have sequenced the *tylIBA* region of the genome⁴⁰ and found five ORFs; *tyl2* encodes a P-450; *tylA1* encodes dTDP-D-glucose synthase, *tylA2* the dTDP-D-glucose 4,6-dehydratase and TylB some form of glycosyltransferase, which has subsequently been assigned as the 3-amino-transferase.

#### 3.4.1 Mycarose

Cundliffe et al. have sequenced the *tylCK* region encoding five *tylC* genes involved in dTDP-L-mycarose 173 biosynthesis;⁴¹ the missing sixth *tylC6* gene was located 50 kbp away, but within the cluster. This TylC6 may be identical to the TlyX3 reported by Floss. In addition, the labelling of genes, or their order, appears inconsistent. Liu et al. label the 3-oxoreductase/enoylreductase as TlyC1, which Cundliffe et al. had previously labelled as TylC2 (Scheme 2). Liu et al. have purified TylX3 (RMM 55 kDa) with 0.8 eq. of Zn²⁺, and ‘TylC2’ (TylC1 in their paper) (36 kDa).¹⁵ On incubation of substrate with TylX3, maltol 178 was obtained. Incubation of the substrate, TylX3, ‘TylC2’ and NADPH gave 179. Incubation in D₂O showed that the 2-OH was replaced with a solvent hydrogen with retention of configuration, and that the NADPH hydride is transferred to C-3 (Scheme 5). TylX3, previously labelled TylC6, shows homology with EryB6, and may represent a small family of zinc dependent dehydratases for C-2 deoxygenation.

#### 3.4.2 6-Deoxyallose/D-mycinose

Bate and Cundliffe have recently reviewed the mycinose 169 biosynthetic genes in *S. fradiae*.⁴² Cundliffe and Wilson have identified and characterised TylN as the 6-deoxyallosyl-transferase.⁴³ Barredo et al. have sequenced 12.9 kbp of the cluster and located eleven ORFs, ten at one end of the cluster.⁴⁴ Three, *tylD*, *tylJ* and *tylN* are proposed as part of the dTDP-6-deoxyallose 168 biosynthetic pathway (Scheme 2). TylE had a glycine rich motif (VLE/DxGxGxG) that may bind SAM. *TylD* mutants are blocked only in mycinose biosynthesis. *TylH* mutants are unable to oxidise the C-23 methyl position. The third component of this P-450 system, the ferredoxin reductase, could not be located. TylE methylates the 2″-OH of the 6-deoxyallose, whilst TylF, the final step, and a major bottleneck in tylosin production, methylates the 3″-hydroxy group (Scheme 3). TylP has homology with acyl CoA oxidases found in eukaryotic peroxisomes. The authors speculate that it may return butanoyl CoA to crotonyl CoA, the reverse of *ccr*, presumably to correct any imbalance. However, Cundliffe et al.

Nat. Prod. Rep., 2001, 18, 231–281   239
Liu *et al.* have overexpressed in *E. coli*, purified and characterised TylB which is thought to transaminate the 3-oxo moiety of 3,6-dideoxy-3-oxo-dTDP-D-glucose 161 as a homodimer devoid of PLP coenzyme. Due to the instability of its putative substrate, the authors examined the reverse reaction. Incubation of TylB with PLP, α-oxoglutarate (to recycle PMP to PLP) and the 3-amino sugar 162 gave the corresponding 3-oxo-sugar 161. PMP was found to efficiently substitute for PLP. The immediate precursor of the natural substrate, the 4-oxo-sugar 160, was not a substrate. Recovered enzyme contained PLP or PMP; the presence of substrates may be necessary for coenzyme reconstitution.

3-O-Acyl-4″-O-(3-methylbutanoyl)tylosin (an isovaleryl ester on the C-4 of the mycarose moiety) is widely used to prevent the growth of tylosin resistant bacteria by veterinarians. Arisawa *et al.* have inserted a macrolide 3-O-acyltransferase (*acyA*) (along with its regulatory gene) from the carbomycin producer *S. thermotolerans* into *S. fradiae* and obtained high levels of the hybrid antibiotic 3-O-acetyltylosin.

### 3.5 Resistance

Four fragments of DNA from *S. fradiae* have been shown to confer tylosin resistance (and to varying extents, resistance to other mesolides), and are labelled *tlrA–D*.

Gandecha and Cundliffe sequenced 2.2 kb of the *tylLM* region. They found that it encompassed *tlrD*, an *ermE*-like 238 aa (35 kDa) A-2058 ribosome monomethylating macrolide (MLS) resistance gene that contains the rare leucine coding TTA triplet codon, and the SAM binding motif GxGxG, which methylates 23S rRNA at position 2058 to give N⁶-monomethyladenosine. The inducible protein TlrA dimethylates the same amine to give N⁶,N⁶-dimethyl analogue, increasing the level of resistance to tylosin. TlrC appears to be an ATP-binding protein involved in tylosin efflux. Wilson and Cundliffe have sequenced *tlrB* which confers inducible resistance to mesolides and expressed it in *E. coli* as a MalE fusion protein, and found that in the presence of SAM it conferred resistance to mesolides. However, unlike the other methylases, TlrB is thought to methylate tylosin itself, not DNA, though its site of methylation and mode of action is not yet understood.

### 3.6 Regulation

Cundliffe *et al.* have found that *tylR* was homologous with the *acyB2* regulatory gene found in the carbomycin producer *S. thermotolerans*. Disruption prevented mesolide formation. Addition of biosynthetic intermediates gave no bioconversion to later intermediates—indicating that disruption of *tylR* shuts down most, if not all, aspects of tylosin biosynthesis, and was a ‘positive regulator’ that controlled multiple biosynthetic promoters and other regulators. The authors also found that TylP was homologous to A-factor binding proteins in contrast to the finding of Barredo *et al.* (see above) and that TylQ was possibly the target protein for TylP, though as there is some sequence similarity in protein sequences for these different proteins, there remains some uncertainty. It may be that Barredo has assigned as *tylP* the gene Cundliffe assigns as ‘*aco*’. Orf16* encodes a P-450, though its role is unclear. Orf12* shows no known homology, and as the start of *tylT* remains uncertain, these assignments need clarification.

TylT and tylS are both pathway specific activators (Streptomyces antibiotic regulatory proteins, SARP), both containing a TTA codon.

## 4 Magnamycins and related 16-membered macrolides

The carbomycins (magnamycins), leucomycins, maridomycins, midecamycins, platenomycins (e.g. platenomycin A₁ 215), spiramycins (e.g. spiramycin I 214) and turimycins all have a

Table 4 Gene cluster for tylosin biosynthesis in S. fradiae

| S. fradiae gene | Size | Putative function |
|-----------------|------|------------------|
| tlrB            | 280aa/30.4 kDa | Resistance: unknown mechanism |
| tylN            | 422aa/46.6 kDa | dTDP-6-deoxyallose transferase 187 to 188 |
| tylE            | 395aa/43.2 kDa | Demethylmacrocin 2"-O-methyltransferase 189 to 190 |
| tylD            | 336aa/36.0 kDa | 4,6-Dideoxy-4-oxo-dTDP-allose 4-oxoreductase 167 to 168 |
| tylH2           | 81aa/8.2 kDa   | [3Fe-4S] ferredoxin for tylH1 |
| tylH1           | 420aa/45.5 kDa | P-450 oxidation of C-23 methyl group 186 to 187. |
| tylF            | 255aa/28.6 kDa | Macrocin 3"-O-methyltransferase 190 to 191 |
| tylJ            | 205aa/22.8 kDa | 4,6-Dideoxy-4-oxo-dTDP-D-glucose 3-epimerase 160 to 167 |
| aco             | 641aa         | Acyl CoA oxidase-formation of short chain fatty acid? |
| tylP            | 658aa/71.5 or 226aa/24.8 kDa | Butanoyl CoA to (2E)-butenoyl (crotonyl) CoA oxidation or A-factor binding protein |
| orf16*          | max 430 aa    | P-450 |
| tylQ            | 224aa/24.7 or 223aa/23.1 kDa | REG; A-factor receptor or tylP binding protein |
| tylS            | 227aa or 293aa | SARP |
| tylT            |               | SARP |
| orf12*          | max 212 aa    | ? |
| orf11*          | max 425 aa    | NTP binding protein? regulated proteolysis? |
| tylC2           | 36 kDa        | 2,3-Enoylreductase of 177 to give 179 (Also called TylC1) |
| tylC4           |               | 4-Deoxy-4-oxo-dTDP-L-mycarose 4-oxoreductase |
| tylC3           |               | 3-C-Methyltransferase converting 171 to 172 |
| tylC5           |               | Demethylactenocin 4'-O-dTDP-L-mycarose transferase |
| tylC7           |               | 5-Epimerase converting 179 into 171 |
| tlrD            |               | ErmE Type I type resistance (23S RNA methylase) |
| ccr             | 423aa/47100Da | (2E)-Butenoyl CoA reductase (crotonyl CoA reductase) |
| tylM1           | 254aa/27530Da | 3-Amino-3,6-dideoxy-dTDP-D-glucose N,N-dimethyltransferase 162 to 163 |
| tylM2           | 452aa/48640Da | Tylactone 5-O-dTDP-D-mycaminosyltransferase 185 to 186 |
| tylM3           | 423aa/44150Da | Unknown |
| tylG5           |               | TYLS:Mod7/TE |
| tylG4           |               | TYLS:Mod6 |
| tylG3           |               | TYLS:Mod4/Mod5 |
| tylG2           |               | TYLS:Mod3 |
| tylGl           |               | TYLS:Loading domain/Mod1/Mod2 |
| tylI            | 417aa/54911Da | P-450 hydroxylation of macrolide ring at C-20 methyl group 186 to 187 |
| orfla           |               | ? |
| tylB            | 388aa/48161Da | 3,6-Dideoxy-3-oxo-dTDP-D-glucose 3-aminotransferase (PLP dependent) |
| tylA1           | 295aa/37459(?) | dTDP-D-glucose synthase 158 into 159 |
| tylA2           | 333aa/42171Da | dTDP-D-glucose 4,6-dehydratase 159 into 160 |
| tylO            | 252aa/32157Da | TE2. Discrete thioesterase/editing out 'mistakes' from TYLS |
| tylC6           | 55 kDa        | 4,6-Dideoxy-4-oxo-dTDP-D-glucose 2,3-dehydratase. Also called TylX3 |
| tylR            |               | Regulatory gene |
| metK            |               | SAM synthase |
| orf9            |               | ? |
| metF            |               | 5,10-Methylenetetrahydrofolate reductase |
| tlrC            |               | Resistance-ATP binding protein |

similar 16-membered lactone ring, based upon platenolide I 209  
(C-9 oxo group, also called platenolide A) or platenolide II 212  
(C-9 hydroxy group), in which a glycolate (or methoxy-  
malonate) acyl unit is used as the sixth extender unit, and with  
D-mycaminose 163 and L-mycarose 173 sugar moieties on the  
C-5 hydroxy group, and in the case of spiramycins (e.g.  
spiramycin I 214) with the 2,3,4,6-tetradeoxy 4-(dimethyl-  
amino) sugar forosamine on the C-9 hydroxy group; these have  
been collectively referred to as the magnamycins.  

### 4.1 Spiramycin (Srm) and platenomycin  
The aglycone platenolide I 209 is produced both by *S. ambo-  
faciens*, in which it is tailored to the spiramycins, and by *S. pla-  
tensis*, in which it is tailored to the platenomycins (Scheme 6).  
Decaris et al. located a section of the genome of *S. ambofaciens*  
involved in spiramycin biosynthesis, with a large ORF (*orfPS*)  
homologous to *eryA* and RAPS, and sequenced nearly 4 kbp. Whilst the  
overall structure of the platenolide I aglycone is very similar to  
tylactone (protylonolide), the selection of building blocks is  
quite different, being AAAAAPB (glycolate?) A compared to  
PPPAPBPA. The literature labels *srmG-orf1* etc. have been  
altered to *srmGI* etc. for consistency with other systems.  
The loading unit contains a KAS(Q). KAS(Q) is homolo-  
gous to KAS, in which the active site cysteine is replaced  

by glutamine, and occurs in tylosin, carbomycin, spiromycin  
synthases, and monensin (but not DEBS, AVES or RAPS) and  
is similar to the so-called 'CLF' or 'KSβ' or 'CIF' in aromatic  
polyketide synthesis. The SRMS KAS(Q)/AT0(A)/ACP0 was  
replaced by TYLS KAS(Q) + AT0(P) + ACP0 from *S. fradiae*,  
along with blocking of sugar attachment, to yield 16-methyl-  
platenolide II 211 in 4-fold increased yields, showing that all the  
later synthase enzymes show disinterest in the modified starter  
unit.  

A glycosyltransferase *GimA* that can glycosylate forocidin, a  
precursor of spiramycin I 214, and play a role in autoresistance  
has been isolated from *S. ambofaciens* and characterised: *gimA*  
is downstream of *srmA*, a gene that encodes a ribosome  
methylating resistance protein.  

### 4.2 Niddamycin (Nid)  
Niddamycin 210 (*S. caelestis* and *S. djakartensis*) is a 16-  
membered macrolide assembled from acetate, propionate,  
butanoate and perhaps glycolate via hydroxymalonate, forming  
the aglycone referred to as platenolide A, but identical to  
platenolide I 209, followed by tailoring (Scheme 6). Omura  
et al. have shown that one unit of a related macrolide, leuco-  
mycin A₃ 213, is derived from glycerol, perhaps via hydroxy-  
malonate or methoxymalonate. Stassi et al. of Abbott  
laboratories have sequenced (40012 bp) five large ORFs  
corresponding to the seven modules required, with a smaller  


Table 5 Domain organisation of SRMS in S. ambofaciens (G refers to the putative glycolate/hydroxymalonate or methoxymalonate transferase)

| SRMS | $S. \text{ambofaciens: } srmG1$ | $srmG2$ | $srmG3$ | $srmG4$ | $srmG5$ platenolide 1 |
|------|---------------------------------|---------|---------|---------|-----------------------|
| KAS   | $\begin{array}{c|c|c} 0 & 1 & 2 \\ \hline Q & \checkmark & \checkmark \\ A & A & A \\ \end{array}$ | $\begin{array}{c} 3 \\ \hline \checkmark \\ A \\ \end{array}$ | $\begin{array}{c|c} 4 & 5 \\ \hline \checkmark & \checkmark \\ P & B \\ \end{array}$ | $\begin{array}{c} 6 \\ \hline G \\ \end{array}$ | $\begin{array}{c} 7 \\ \hline A \\ \end{array}$ |
| AT    | $\begin{array}{c|c|c} \hline \end{array}$ | $\begin{array}{c} \hline \end{array}$ | $\begin{array}{c|c} \hline \end{array}$ | $\begin{array}{c} \hline \end{array}$ | $\begin{array}{c} \hline TE1 \\ \end{array}$ |
| DH    | $\begin{array}{c|c|c} \hline \end{array}$ | $\begin{array}{c} \hline \end{array}$ | $\begin{array}{c|c} \hline \end{array}$ | $\begin{array}{c} \hline \end{array}$ | $\begin{array}{c} \hline \end{array}$ |
| ER    | $\begin{array}{c|c|c} \hline \end{array}$ | $\begin{array}{c} \hline \end{array}$ | $\begin{array}{c|c} \hline \end{array}$ | $\begin{array}{c} \hline \end{array}$ | $\begin{array}{c} \hline \end{array}$ |
| KR    | $\begin{array}{c|c|c} \hline \end{array}$ | $\begin{array}{c} \hline \end{array}$ | $\begin{array}{c|c} \hline \end{array}$ | $\begin{array}{c} \hline \end{array}$ | $\begin{array}{c} \hline \end{array}$ |
| ACP   | $\begin{array}{c|c|c} \hline \end{array}$ | $\begin{array}{c} \hline \end{array}$ | $\begin{array}{c|c} \hline \end{array}$ | $\begin{array}{c} \hline \end{array}$ | $\begin{array}{c} \hline \end{array}$ |

---

**Scheme 6**

- **acetate**
- **propionate**
- **butanoate**
- **glycolate etc**

$\downarrow$

**Tylosin loading domains**

$\downarrow$

[H]

![Chemical Structures](chemical_structures.png)

---

**ORF6 (132 aa) corresponding to a 'TylI like' P-450 hydroxylase and ORF7 (153 aa) a 'TylMI like' N-MeT.** The arrangement of the NIDS modules appears homologous to the TYLS as shown below, with an inactive KR4 (DxTxxPxL instead of GxGxxGxANADPH binding motif), and a GxSxG motif and GxH motif at the end of NidA5 identifying the TE1. The AT domains show far more sequence variability than the highly conserved KAS1-7 domains. AT0, 1, 2, 3 and 7 all have the malonate loading sequence. AT4, 5 and 6 all show sequence corresponding to propionate/methylmalonate loading trans- ferases, however AT5 presumably loads ethylmalonate, and though unproved, AT6 presumably loads some form of hydroxymalonate, again showing that all 'non-malonate' types of acyl transferase may have very similar sequences to each other. All ACPs contained GFDSL pantotheine attaching site motif.

---

### 4.3 Pikromycin (picromycin), narbomycin, methymycin and neomethymycin

S. venezuelae ATCC 15439 in some media (e.g. 'PGM') produces the 14-membered mesolides (macrolides) narbomycin 231 and pikromycin 232, whilst in other media it produces the 12-membered mesolides methymycin 219 (first isolated in 1954) and neomethymycin 218 (Scheme 7). The factors which determine whether 12- or 14-membered mesolides are formed have given rise to much speculation, firstly that different TE's acted upon either Mod5 or Mod6, or whether Mod6 is always expressed at the same levels as the other modules, or whether TE1 is equally accessible by either ACP5 or ACP6, or whether Mod6 can be 'skipped' and the growing acyl chain delivered directly from ACP5 to ACP6 without being extended and reduced.

Table 6 Domain organisation of NIDS in S. caelestis

| S. caelestis | nidA1 | nidA2 | nidA3 | nidA4 | nidA5 | platenolide 1 |
|--------------|-------|-------|-------|-------|-------|---------------|
| NIDS         |       |       |       |       |       |               |
| KAS          | Q     | ✓     | ✓     |       |       |               |
| AT           | A     | A     | A     |       |       |               |
| DH           |       | ✓     |       |       |       |               |
| ER           |       | ✓     | ✓     |       |       |               |
| KR           |       | ✓     | ✓     |       |       |               |
| ACP          | ✓     | ✓     | ✓     |       |       |               |

Table 7 Partial gene cluster for niddamycin biosynthesis in S. caelestis

| S. caelestis | Size        | Putative function                                                                 |
|--------------|-------------|-----------------------------------------------------------------------------------|
| ORF6         | 132 aa      | Cytochrome P-450 hydroxylase (N-terminal portion)                                 |
| ORF1/nidA1   | 458 kbp     | NIDS: loading domain/Mod1/Mod2                                                   |
| ORF2/nidA2   | 194 kbp     | NIDS Mod 3                                                                      |
| ORF3/nidA3   | 379 kbp     | NIDS Mod 4/Mod 5                                                                |
| ORF4/nidA4   | 167 kbp     | NIDS Mod 6                                                                      |
| ORF5/nidA5   | 198 kbp     | NIDS Mod 7                                                                      |
| ORF7         | 157 aa      | N-Methyltransferase (N-terminal portion)                                         |

![Chemical Structures and Reactions](https://i.imgur.com/chemical_structures.png)

Scheme 7

Cane *et al.* have determined the acetate and propionate incorporation into methymycin, and showed the intact incorporation of labelled NAC thioesters of the diketide and triketide processive intermediates.⁵⁸

### 4.3.1 Synthase (PIKS)

To determine whether TE2 was responsible for premature product release resulting in the formation of 12-membered mesolides, Sherman *et al.* deleted TE2 (*pikA*5) and obtained a twenty-fold reduction in levels of production of the 12-membered products as expected, but also a twenty-fold reduction in the production of 14-membered products.⁵⁹ Thus TE2 acts on both the product of Mod6 PikA4 as well as Mod5 PikA3, and in its absence, products are not efficiently released. An intriguing question is that whilst other clusters contain discrete TE2's, they do not appear to play a role in generating more than one macrolide product—however, they are more distant on the gene from the synthase. Another possibility is that *pikA*4 is expressed at lower levels than *pikA*3, resulting in ca. 50% stalling at the end of Mod5, with release of the hexaketide as an 'aberrant' protein by TE2 as proposed for the role of Tyl TE2. Reynolds has provided commentary on this work and its connection with related work.⁶⁰ The most consistent explanation.

Nat. Prod. Rep., 2001, 18, 231–281

Table 8 Domain organisation of PIKS in S. venezuelae (Inactive KAS 6 results in formation of 10-deoxymethynolide)

| S. venezuelae pikA1 | pikA2 | pikA3 | pikA4 | pikA5 ⇒ narbonolide |
|---------------------|-------|-------|-------|----------------------|
| PIKS                |       |       |       |                      |
| KAS                 | Q     | ✓     | ✓     |                      |
| AT                  | P     | P     | P     |                      |
| DH                  |       | A     | ✓     | ✓                    |
| ER                  |       | ✓     | ✓     | P                    |
| KR                  |       | ✓     | ✓     | ■                    |
| ACP                 | ✓     | ✓     | ✓     | ✓                    |

---

ation is that TE1 releases both the hexaketide and heptaketide products, and that TE2, as in other systems, removes aberrant intermediates, and has no role in determining levels of either product, only in enhancing overall production.

In only the second report of the heterologous expression of a modular PKS cluster, McDanielt al. functionally expressed the PIKS cluster in S. lividans, and efficiently produced a ca. 1:1 mixture of narbonolide and 10-deoxymethynolide (5–10 mg L⁻¹). Co-expressing additional TE2 produced 4–7 times more narbonolide and 10-deoxymethynolide, reinforcing the idea that the presence or absence of TE2 does not influence the ratio of 12-to 14-membered mesolide production, only the total polyketide production, and that this TE2 may act, like other TE2s, as an editing mechanism. The authors then replaced the DEBS TE1 in DEBS with the PIKS TE1 (with inactivation of EryKR6) and only obtained 3,6-dideoxy-3-oxoerythronolide B, and obtained no evidence of 12-membered mesolide formation.

Another possibility is that an ancestral PIKS did not contain PikA3 (Mod5), so TE1 is ‘designed’ to hydrolyse hexaketides. Recent evolutionary insertion of PikA3 into the synthase results in inefficient heptaketide hydrolyses as the TE1 has not yet had time to fully evolve.

Xue and Sherman appear to have now found a plausible explanation for the bifunctionality of the TE: when expressed in S. venezuelae grown in a 'PGM' medium, Xue and Sherman found that the synthase produced the 14-membered aglycone narbonolide as expected. The authors placed a His tag onto PikA4, and then examined Western blots under conditions that produce the 14-membered mesolides. The protein was of the expected size, 110 kDa, whilst under conditions that produced 12-membered mesolides, the protein appeared as only 85 kDa. As the TE1 is required for efficient production of either ring size, the authors concluded that there must be an N-terminal truncation of the PikA4 protein occurring, i.e., at the start of the KAS6 domain. Analysis of the sequence revealed an alternative translational start site 600 nucleotides further upstream of the originally used start site, containing a putative ribosome binding site GGAGGGG followed seven nucleotides later by two start codons GTGxxxGTG. Translation using this start site would give a truncated PikA4 of 88 kDa, with only half of the KAS6 domain. Conversion of both GTGs to GTC abolished production of 12-membered mesolides. The authors then removed further portions of the PikA4 protein, but found that for efficient production all the domains, especially the TE1, were necessary. The authors propose that a small movement of the TE1 domain allows it to fill the unoccupied space caused by KAS6 truncation, thus interacting directly with ACP5. To distinguish between this model, and KAS mediated ACP(−1) to ACP direct transfer (module skipping), it would be interesting to construct synthases with a mutated KAS6 cysteine active residue. In this system, there would be no scope for such realignment of domains, yet KAS mediated ACP to ACP transfer could still take place, resulting in the skipping of Mod6 and 12-membered macrolide formation.

Tang, Fu and McDanielf KOSAN biosciences inserted the erythromycin, oleandomycin and pikromycin PK synthases into S. lividans to form 6-dEB1, 8, 18-deoxyoleandolide 182 and

narbonolide 230, respectively (Scheme 8). They then assembled heterologous mixtures of the three synthases, and obtained levels of new products similar to that obtained from the parent synthases in the S. lividans: DEBS3 and OleA3 differ from 'PikA3+PikA4' in having a KR6. Replacement of the module PikA3 (Mod5) and PikA4 (Mod6) in PIKS by DEBS3 or OleA3 and expression in S. lividans gave 3-deoxo-(3S)-hydroxynarbonolide 66 in levels similar to that obtained for production of narbonolide 230 in this system, and over 100-fold greater than that obtained previously. Thus the DEBS3 and OleA3 subunits have fully complemented the PikA3/PikA4 module units. In the PIKS substituted with DEBS3 (PikA1+PikA2+DEBS3); replacing EryAT6(P) by RapAT2(A) or by RapDH4/KR4 gave only the expected products 235 and 10-demethyl-2,3:10,11-dianhydro-6-dEB 236 respectively, and deletion of EryKR5 gave the expected 5-oxo analogue 237. However, replacement of EryKR5 by either RapDH4/KR4 or RapDH2/ER2/KR1 failed to give more than trace amounts of the expected products 10-demethyl-4,5:10,11-dianhydro-6-dEB 238 and 10,11-anhydro-10-demethyl-5-deoxy-6-dEB 239 respectively. Presumably the Rap domains are not recognising the acyl chain intermediate, and are not being utilised, as was also observed when these domains were similarly inserted into DEBS system. In previous work, attempts to produce 10-nor or 10,11-anhydromesolides from the DEBS system gave very poor yields; however, using the PIKS system has extended the range of mesolides that can be efficiently produced through combinatorial biosynthesis.

Tang, Fu and McDanielf then extended this work by fusing parts of Ery and Pik Modules 5+6. Replacing Mod6 of DEBS3 with PikA4 gave no product, even though the correct interlinker regions had been maintained, and an analogous fusion had worked before. Replacement of DEBS3 by EryKAS5/EryAT5(P)/PikKR5/PikACP5 and PikA4 gave the expected mesolide 3-deoxy-3-oxo-6-dEB 81at 3mg L⁻¹ (Scheme 8). However, replacement with the two modules fused together gave a much lower yield. This low yield highlights the need for further understanding of inter-linker regions and domain/module fusions to achieve efficient results from future combinatorial biosynthesis.

### 4.3.2 Post-PKS (tailoring)

Sherman et al. have analysed the single biosynthetic gene cluster in S. venezuelae that is responsible for the production both of the 12-membered ring mesolides methymycin 219 and neomethymycin 218 from the same precursor 10-deoxymethynolide 216; and the 14-membered ring mesolides narbomycin 231 and pikromycin 232 both derived from the same precursor narbonolide 230. This 60 kbp ‘pik’ cluster (sometimes ‘pic’ is used) contains 18 ORFs, including four large ORFs pikA1-pikA4 that encode the synthase containing six modules. The cluster also contains an adjacent second thioesterase (TE2) pikA5 immediately downstream of the synthase, P-450 hydroxylase (pikC, also referred to as picK!) homologous to eryF and eryK, and a cluster of nine genes: seven genes for desosamine biosynthesis (des1-6 and des8); adTDP-D-desosaminyltrans-

![DEBS1](image1.png)
![DEBS2](image2.png)
![DEBS3](image3.png)

6-dEB

1

![OLEA1](image4.png)
![OleA2](image5.png)
![OleA3](image6.png)

8,18-deoxyoleandrolide

[AT0(A)]

182

![PikA1](image7.png)
![PikA2](image8.png)
![PikA3](image9.png)
![PikA4](image10.png)

narbonolide

[AT2(A), +DH2, no KR6]

230

---

![PikA1](image11.png)
![PikA2](image12.png)
![DEBS3](image13.png)
or
![PikA1](image14.png)
![PikA2](image15.png)
![OleA3](image16.png)

66

10,11-anhydro-10-demethyl-6-dEB
or 3-deoxo-(3S)-hydroxynarbonolide

---

![PikA1](image17.png)
![PikA2](image18.png)
![DEBS3](image19.png)
with
RapAT2(A)
replacing
EryAT6(P)

235

10,11-anhydro-2,10-didemethyl-6-dEB
or 2-demethyl-3-deoxo-3-hydroxynarbonolide

---

![PikA1](image20.png)
![PikA2](image21.png)
![DEBS3](image22.png)
with
RapDH4/KR4
replacing
EryKR6

236

10-demethyl-2,3:10,11-dianhydro-6-dEB

---

![PikA1](image23.png)
![PikA2](image24.png)
![DEBS3](image25.png)
with
EryKR5
disrupted

237

10,11-anhydro-10-demethyl-5-deoxy-5-oxo-6-dEB

---

![PikA1](image26.png)
![PikA2](image27.png)
![DEBS3](image28.png)
mainly 7+
with
RapDH4/KR4
replacing
EryKR5

238

10-demethyl-4,5:10,11-dianhydro-6-dEB

---

![PikA1](image29.png)
![PikA2](image30.png)
![DEBS3](image31.png)
mainly 233+
with
RapDH1/ER1/KR1
replacing
EryKR5
(trace amount)

239

10,11-anhydro-10-demethyl-5-deoxy-6-dEB

---

![DEBS1](image32.png)
![DEBS2](image33.png)
Ery Mod 5
fused to
PikA4

No product

OR

![DEBS1](image34.png)
![DEBS2](image35.png)
EryKAS5/AT5
fused to
PikKR5/ACP5
KAS6/AT6/
ACP6/TE

[LOW YIELD]

![DEBS1](image36.png)
![DEBS2](image37.png)
EryKAS5/AT5
fused to
PikKR5/ACP5

PikA4

[3 mg/L⁻¹]

81

3-deoxy-3-oxo-6-dEB

---

**Scheme 8**

ferase (des7); and a large ORF desR that is involved in mesolide self-protection by inactivation/reactivation of the antibiotic.⁶⁷ This 'des' cluster; was followed by a regulatory gene pikD hom-

ologous to Rap orfH. There is no SAM synthase in the cluster; presumably a SAM synthase located further downstream amongst genes for primary metabolism assists in N,N-dimethyl-

Table 9 Gene cluster for pikromycin biosynthesis in S. venezuelae

| S. venezuelae | Size | Putative function |
|---------------|------|-------------------|
| gene          |      |                   |
| pikR2         | 322aa | rRNA methyltransferase (MLS resistance) |
| pikR1         | 336aa | rRNA methyltransferase (MLS resistance) |
| pikA1         | 4613aa | Mod 0/1/2 |
| pikA2         | 3739aa | Mod 3/4 |
| pikA3         | 1562aa | Mod 5 |
| pikA4         | 1346aa | Mod 6 |
| pikA5         | 281aa | TE2 |
| des8          | 402aa | 4,6-Dideoxy-4-oxo-dTDP-D-glucose 3,4-isomerase |
| des7          | 426aa | Glycosyltransferase |
| des6          | 237aa | 3,4,6-Trideoxy-3-amino-dTDP-D-glucose N,N-dimethyltransferase: 165 to 166 |
| desR          | 809aa | β-Glucosidase (as part of resistance mechanism) |
| des5          | 379aa | 3,4,6-Trideoxy-3-oxo-dTDP-D-glucose 3-aminotransferase: 164 to 165 |
| des4          | 337aa | dTDP-D-glucose 4,6-dehydratase: 159 to 160 |
| des3          | 292aa | dTDP-D-glucose synthetase: 158 to 159 |
| des2          | 485aa | Part of: 3,6-dideoxy-3-oxo-dTDP-D-glucose 3,4-reductase: 161 to 164 |
| des1          | 415aa | Part of: 3,6-dideoxy-3-oxo-dTDP-D-glucose 3,4-reductase: 161 to 164 |
| pikC          | 416aa | P-450 hydroxylase. Also labelled picK |
| pikD          | 928aa | Regulator? |

The entire desosamine biosynthetic cluster has been cloned from *S. venezuelae*.² The eight ORFs appear in a cluster along with *desR* encoding a mesolide β-glucosidase. However, it has proved difficult to express and purify these proteins in soluble form. *Des1* and *Des5* show homology with coenzyme B₁₂-dependent catalysts, and are assigned as the 3,4-dehydratase involved in C-4 deoxygenation; and the 3-aminotransferase responsible for C-3 transamination (Scheme 2). 3,4-Dehydratases in the 3,6-dideoxysugar pathway uses PMP (pyridoxamine 5′-phosphate) as a cofactor with a presumed [Fe₄S₄] cluster (see Scheme 40 in Part A¹).³ However, it is not clear from their sequences whether Pik Des1 and Des5 are PMP or PLP dependent—as in TylB.⁴ Liu et al. deleted the *des5* gene abolishing production of methymycin and neomethymycin, obtaining good yields of 10-deoxy-methynolide 216 (120 mg L⁻¹) and two new mesolides 244 (5 mg L⁻¹) and 245 (8 mg L⁻¹) containing a 4,6-dideoxyglucose moiety (Scheme 9).² The absence of an amino functionality in this sugar is in agreement with *Des5* being the 3-aminotransferase (and *Des1* the 3,4-dehydratase), and as the sugar is 4-deoxy, suggests that C-4 deoxygenation precedes transamination at C-3. Replacement of the *des5* gene with *tylB*, which catalyses the transamination of the 3-oxo-4-hydroxy analogue 161 during the biosynthesis of dTDP-D-mycaminose 163, did not lead to the formation of D-desosamine containing products, again suggesting that in desosamine biosynthesis C-4 deoxygenation precedes C-3 transamination, as TylB is not transaminating the available C-4 deoxy-3-oxo-precursor 164 (Scheme 2).

N,N-Dimethylation increases the basicity of the amino-sugar's nitrogen, increasing the ease of protonation, which may be crucial in binding affinity towards the negatively charged ribosomal RNA target. Liu et al. have expressed and purified *Des6*, and found *in vitro* that it efficiently N,N-dimethylated synthetic 165 to 166, and with short incubation times, some mono-methylated product was isolated.⁷⁴

In the D-desosamine pathway, 4-oxo-sugar 160 is converted into the 3-oxo sugar 161 by the 3,4-isomerase *Des8*. A protein from *Micromonospora echinospora* spp. *calichensis*, *CalH*, is thought to transaminate the 4-oxo sugar 160 to give 240. Liu et al. have inserted *calH* into a *des8* mutant and obtained two new mesolides containing the ‘unnatural’ N-acetylated 4-amino sugar (Scheme 9).⁷⁵ This shows that *CalH* is indeed the 4,6-dideoxy-4-oxo-dTDP-D-glucose 4-aminotransferase of the calicheamicin pathway, and that the Pik *Des7* glycosyl transferase can transfer 4-amino-4,6-dideoxy-dTDP-D-glucose 240 or its N-acetyl derivative, to give either the 16-membered mesolide 242 or the 14-membered mesolide 243.

---

ation to form desosamine. As there is only a single set of deoxysugar genes, and only a single hydroxylase, both presumably function on 12-membered and 14-membered precursors. Addition of P-450 monooxygenase inhibitors leads to the accumulation of 10-deoxymethynolide 216.⁶⁸ In a separate report, Cane and Graziani investigated the P-450.⁶⁹ After attempting to detect enzymatic hydroxylation of 10-deoxymethynolide 216, they instead chemically synthesised labelled YC-17217 (from labelled 216), and isolated the protein that, along with added NADPH, ferredoxin and ferredoxin-NADP⁺ reductase, gave both methymycin 219 (C-10 hydroxylation: *K*ₘ 95 μM) and neomethymycin 218 (C-12 hydroxylation: *K*ₘ 255 μM). The author suggests that *in vivo* this difference in *K*ₘ may indicate the presence of two different P-450 monooxygenases—however the gene cluster has since been found to contain only one P-450 gene *pikC*, unless a second is located outside the cluster. McDaniel et al. of KOSAN Biosciences have also investigated this P-450PikC, which they refer to as PicK. The P-450 gene *pikC* (*picK*) shows homology with the gene for EryDC-12 hydroxylase *eryK*. McDaniel et al. cloned, overexpressed in *E. coli* and characterised PikC as the narbomycin 231C-12 hydroxylase P-450 from *S. venezuelae* (Scheme 7).⁷⁰ Whilst the *K*ₐₜ value for PikC (1.4 s⁻¹) was similar to that for EryK or EryF, the *K*ₘ value (120 μM; *v*ₘₐₓ 0.34 μMs⁻¹) was much higher. EryK and EryF have both been shown to have very high substrate specificity, and thus may not be very appropriate for combinatorial biosynthesis; however, the much lower binding affinity of *pikC* for its substrate promises future applicability in hydroxylating unnatural substrates. Cane et al. (vide supra) have shown that a cell-free extract presumably containing only PikC can hydroxylate YC-17217 at either position, to give the 12-membered mesolides methymycin 219 and neomethymycin 218.⁶⁹

### 4.3.3 Deoxysugar biosynthesis and attachment

Liu et al. deleted the *des6* gene that encodes N,N-dimethylation, the final step in the biosynthesis of dTDP-D-desosamine 166 before attachment onto 10-deoxymethynolide 216 by the glycosyltransferase *Des7*,⁷¹ and obtained the aglycone 10-deoxymethynolide (100 mg L⁻¹) and the de-(N,N-dimethyl)-N-acetyl analogues of neomethymycin 246 and methymycin 247 (<10 mg L⁻¹) (Scheme 9). Presumably the *Des7* glycosyltransferase is capable of recognising (presumably much less efficiently) the 3-amino dTDP-hexose 165. It remains unclear whether the N-acetylation occurs prior to glycosyl transfer, or whether it occurs afterwards in an unspecific manner or perhaps as a self-preservation/autoresistance mechanism.

![Chemical Structures](chemical_structures.png)

Scheme 9 Mutant studies to investigate the desosamine biosynthetic pathway.

### 4.4 Mycinamycin

Inouye *et al.* have described the construction of a *Micro*-*monospora-E. coli* shuttle cosmid vector used to clone and analyse the *mycG* P-450 (397 aa) involved in the hydroxylation and/or epoxidation of intermediates such as protomycinolide IV 248 in the biosynthesis of mycinamycin II 249 in *Micro*-*monospora griseorubida* (Scheme 10).²⁷⁶

### 4.5 Albocycline

Albocycline 252 (*Streptomyces* sp. strain Lu7285) and cineromycin B 251 are unusual 14-membered mesolides (macrolides) in that they lack any sugar rings, yet they retain biological activity, which is normally absent in mesolide aglycones. Zeeck and Taddei have reported the incorporation of labelled acetate and propionate into albocycline (Scheme 11).⁷⁷ They observe the formation of 2,3-dihydroalbocycline 255 late in the fermentation process after the production of albocycline, and suggest that albocycline 252 is reduced to 2,3-dihydroalbocycline, rather than the alternative route through a synthase with a partially active ER6, which for some reason is more active late in the fermentation (dotted arrow). The mesolide is assembled using a very unusual alternating sequence APAPAPAPA. The C-5–C-6 double bond needs isomerisation at some time during the sequence, and it is tempting to speculate that the C-7 hydroxy group was initially a ketone, facilitating the isomerisation, and then is subsequently reduced by a discrete 7-oxoreductase.

### 5 Polyethers

#### 5.1 Starter units: branched chain amino acid metabolism

The starter unit of many polyethers can be derived from a branched chain amino acid such as valine, leucine or isoleucine (Scheme 4). The first steps are conversion of the amino acid into the 2-oxoacid by a dehydrogenase, followed by oxidative decarboxylation to the nor-acid, for example valine 202 is converted into 2-methylpropanoyl (isobutyryl) CoA 204. This has two main fates: the first is a vitamin B₁₂ dependent 2-methylpropanoyl CoA mutase catalysed rearrangement to butanoyl CoA 205;³ the second, in a process analogous to fatty acid oxidation, involves oxidation by an acyl CoA dehydrogenase to 2-methylprop-2-enyl CoA 206, followed by further oxidation to methylmalonyl CoA 207.

Robinson *et al.* have cloned and sequenced the L-valine dehydrogenase gene *vdh* from the monensin A producer *S. cinnamonensis*, and found it to encode for an enzyme of *M*ₐ 37 718 Da containing 358 aa including 6 cysteines.⁷⁸ Whilst *vdh* disruption eliminated Vdh activity, it did not disrupt monensin production even when L-valine was the sole nitrogen source probably due to an alternative pathway involving a PLP dependent valine transaminase (Scheme 4).

The publication of the sequence and domain organisation of a polyether synthase and overall cluster, including the location of P-450 genes relative to synthase components, is eagerly awaited.

6 Avermectins and related compounds<sup>79</sup>

Avermectins, produced by *S. avermitilis*, lack significant antibacterial or antifungal activity, but act selectively on nematode and insect GABA-gated chloride channels in neurons. 'Ivermectin', a C-22/C-23 hydrogenated semi-synthetic derivative of avermectin B1 has been in commercial use as an agricultural antiparasitic (mainly antinematodal) agent since 1985, and is now being investigated as a treatment of the tropical disease human onchocerciasis, also called river blindness, caused by a parasitic worm transmitted by black fly. A mutant (*bkd*) of *S. avermitilis* in which normal starter unit production is blocked, when supplied with cyclohexane-

carboxylic acid, produces the commercially important Doramectin.

Ikeda and Omura have extensively reviewed avermectin biosynthesis.<sup>80,81</sup> The avermectin 'A's have a C-5 methoxy group, whilst the 'B' series has a C-5 hydroxy group. The '1' series has a C-22–C-23 double bond, whilst the '2' series has a saturated C-23 hydroxy group. The 'a' series uses a 2-methylpropanoyl starter unit, whilst the 'b' series uses a 2-methylbutanoyl starter unit. The structural formula in Fig. 1 of the recent review<sup>80</sup> corrects the table in the earlier review<sup>81</sup> The gene cluster has now been analysed; the precise biosynthetic sequence and many aspects of its biosynthesis remain to be elucidated. Early labelling studies showed that the oxygens at C-1, 5, 7, 13, 17, 19 and 23 (for '2' components) are acyl unit derived, but that the benzofuran oxygen was not, and is presumed to be dioxygen derived. The L-oleandrose units were labelled by glucose, and both the oleandrose O-methyls and the C-5 hydroxy methyl are SAM derived.<sup>82</sup>

### 6.1.1 Starter units

Hutchinson *et al.* have cloned, sequenced and expressed in both *E. coli* and *S. coelicolor* an acyl CoA dehydrogenase *acdH* (RMM 41 737 Da, 386 aa) from *S. avermitilis*.<sup>83</sup> Both AcdH flavoproteins were able to oxidise butanoyl, pentanoyl, 2-methylpropanoyl, 3-methylbutanoyl, and cyclohexanecarbonyl CoA to the corresponding ‘2-enes’. Disruption of the *acdH* in *S. avermitilis* removed the ability to grow on valine, leucine or isoleucine as sole carbon sources. Feeding studies with labelled [2,3,3-<sup>13</sup>C<sub>3</sub>]-2-methylpropanoic acid led to dramatically reduced label in ‘propionate’ derived positions, but did lead to label in ‘acetate’ positions, showing that in the mutant conversion of branched chain amino acids to methylmalonate had been affected (Scheme 4).

Riccardi *et al.* have cloned the acetohydroxy acid synthase (*ilvBN*) and the acetohydroxy acid isomeroreductase (*ilvC*) from *S. avermitilis*.<sup>84</sup> These enzymes are involved in the formation of isoleucine and valine from threonine.

Reynolds *et al.* have located five (*ansJKLchcAansM*) of the nine genes responsible for the biosynthesis from shikimic acid of cyclohexanecarbonyl CoA (CHC-CoA) in *S. collinus* and transferred them on a plasmid into *S. avermitilis*, which now produced fatty acids containing 50% with a cyclohexylcarbonyl starter unit, and produced doramectin in the absence of exogenous cyclohexanecarboxylic acid.<sup>85</sup> Other enzymes

**Scheme 10 Biosynthesis of mycinamycin II in *Micromonospora griseorubida*.**

**Scheme 11**

Table 10 Domain organisation of AVES in S. avermitilis (N refers to an acyltransferase that accepts a wide range of substrates, ½ to partial activity)

| S. avermitilis | aveA1 | aveA2 | aveA3 | aveA4 |
| --- | --- | --- | --- | --- |
| AVES | 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 |
| KAS |  | √ | √ | √ | √ | √ | √ | √ | √ | √ | √ | √ | √ |
| AT | N | P | A | A | A | A | P | P | A | P | A | P | A |
| DH |  |  | ½ |  |  |  | √ |  | √ |  |  |  |  |
| ER |  |  |  |  |  |  |  |  |  |  |  |  |  |
| KR |  | √ | √ | √ | √ | √ | √ | √ | √ | √ | √ | √ | TE |
| ACP |  | √ | √ |  |  |  |  |  |  |  |  |  |  |

---

Table 11 Gene cluster for avermectin biosynthesis in S. avermitilis (sugar biosynthesis genes adapted to fit hypothesis in Section 2.16 in Part A)

| S. avermitilis | Size | Putative function |
| --- | --- | --- |
| gene |  |  |
| ? |  | ABC transport |
| ? |  | Membrane protein |
| aveR | → | 949aa | DNA binding positive regulator. Homology with rapH |
| aveF | ← | 302aa | C-5-oxoreductase |
| aveD | ← | 283aa | C-5O-MeT |
| aveA1 | → | 3953aa | AVES 1 (Mod 0/1/2) |
| aveA2 | → | 6239aa | AVES 2 (Mod 3/4/5/6) |
| aveC | → | 347aa | ? |
| aveE | ← | 456aa | P-450 |
| aveA3 | ← | 5532aa | AVES 3 (Mod 7/8/9) |
| aveA4 | ← | 4881aa | AVES 4 (Mod 10/11/12/TE) |
| orf1 | → | 238aa | Reductase? |
| aveB1 | → | 412aa | dTDP-L-oleandrosyltransferase |
| aveB2 | ← | 355aa | dTDP-D-glucose 4, 6-dehydratase: 159 to 160 (Scheme 2) |
| aveB3 | ← | 299aa | dTDP-D-glucose synthetase: 158 to 159 |
| aveB4 | → | 343aa | 4-Oxo-4-deoxy-dTDP-L-olivose 4-oxoreductase: 171 (or 180) to 175 |
| aveB5 | ← | 225aa | 179 5-epimerase (or 180 3, 5-epimerase) |
| aveB6 | ← | 469aa | 4, 6-Dideoxy-4-oxo-dTDP-D-glucose 2, 3-dehydratase: 160 to 177 |
| aveB7 | ← | 257aa | dTDP-L-olivose 3-O-methyltransferase: 175 to 176 (Scheme 2) |
| aveB8 | ← | 347aa | 174 3-oxoreductase: 174to175a (or 175b) |

---

present in S. avermitilis are presumably substituting for the four gene products not transferred on the plasmid, or the transferred enzymes can perform several steps on the pathway.

### 6.1.2 Synthase (AVES)

Early genetic studies summarised in the review by Omura suggested two large synthase genes labelled aveA I and aveA II. However, caution is advised when using the domain map in this early review as the genes themselves, their domains, and their labels have now been considerably revised by Omura et al. who have recently analysed the 82 kbp biosynthetic gene cluster containing 18 ORFs. Use of a valine derived 2-methylpropanoyl CoA starter gives the 'a' series, and use of isoleucine derived 2-methylbutanoyl CoA as starter gives the 'b' series, which is then extended using seven acetate and five propionates. The sequence of steps may be to give initially 256 or 257, with acetal formation to 258 and 259, further chain extension, six-membered carbocycle formation (without dehydration/aromatisation) and lactonisation giving the 6, 8a-seco-6, 8a-deoxy-5-oxoavermetin aglycone 261 (Scheme 12). Partial activity of DH2 results in a mixture of C-22-C-23 double bonds (A series) and C-23 hydroxy compounds (B series) e.g. avermectin B2a 262. The genes encoding the twelve modules are arranged in two pairs of genes, aveA1/aveA2 and aveA3/aveA4, which are convergently transcribed and are labelled AVES 1-4 (6, 8a-seco-6, 8a-deoxy-5-oxoavermetin synthase) and are now labelled in the order they are used in the acyl chain assembly. The loading domain was similar to that for DEBS, with only an AT0/ACP0, with homology between the AT0 sequences and the absence of KAS(Q), implying that they are both monocarboxylic acid loading. The KR10 and DH7 are presumed inactive, though the sequence of KR10 appears fully consistent with activity. DH2 contains one mismatched amino

The authors present an extensive phylogenetic analysis of acyltransferases from actinomycete Type I PKSs. The malonyl-transferases were clustered, along with the malonate loading NIDSAT0. The methylmalonyl ATs were clustered, along with methylmalonyl loading AT0sPIKSAT0 and TYLSAT0. One sub-branch contained the ethylmalonyl loading TYLSAT5 and NIDSAT5, and the putative hydroxymalonate loading NIDSAT6, and another, more separated/distant sub-branch, regional branch, contained the monocarboxylate loading DEBSAT0 and AVESAT0, both of whose loading domains lack any form of decarboxylative KAS(Q)-like domain. There are some inconsistencies between the modules in the 1997 review and the latest paper; Table 11 reflects the latest paper. For example, the diagram in the review has a non-functional KR domain in Mod 3, whilst the recent paper indicates that Mod3 may lack a KR domain.

There are two ORFs, aveC and aveE, sandwiched between the two pairs of PKS ORFs. AveC may be translationally coupled to aveA2; and aveE (encoding for aP-450) coupled to aveA4 and aveA3. Disruption of aveC leads to enhanced production of the hydrated avermectin '2' series. The aveE gene is involved in post-translational furan ring formation, probably by P-450 oxidation of the allylic C-8a methyl group, followed by radical activation of C-6 by the same P-450 and radical combination to form the benzofuran.

### 6.1.3 Post-PKS (tailoring)

Avermectin B's have a free C-5 hydroxy group, whilst the more effective anthelmintic avermectin A's have a C-5 methoxy

Nat. Prod. Rep., 2001, 18, 231–281

Starter unit  
(2-methylbutanoate-'b' series  
or 2-methylpropionate-'a'-series)  

then PA-AAAP  
with partially active DH2  

active DH2  

EnzS  
O  
OH  
('1'-series)  
OH  
Me/Et  

256  

' skipped '  
DH2  

EnzS  
O  
OH  
('2'-series)  
OH  
Me/Et  

257  

'b'-series  
'a'-series  

Enz-S  
O  
Me  
OH  
258  

acetal formation  

Enz-S  
O  
Me  
OH  
259  

HO  
Me  
Me/Et  

HO  
Me  
Me/Et  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me  

HO  
Me  
Me
Table 12 Domain organisation of NYSS in S. noursei (two blocks separated by seven genes)

| S. noursei nysA | nysB | nysC | ...nysD1, D2, D3, EFGH... |
|-----------------|------|------|--------------------------|
| NYSS            |      |      |                          |
| KAS             | S    | ✓    | ✓                        |
| AT              | A    | P    | A                        |
| DH              |      | □    | A                        |
| ER              |      |      | ✓                        |
| KR              |      | ✓    | ✓                        |
| ACP             | ✓    | ✓    | ✓                        |

| S. noursei NYSS | nysI | nysJ | nysK |
|-----------------|------|------|------|
| KAS             | ✓    | ✓    | ✓    |
| AT              | A    | A    | A    |
| DH              | ✓    | □    | □    |
| ER              |      |      |      |
| KR              | ✓    | ✓    | ✓    |
| ACP             | ✓    | ✓    | TEI  |

7 Polyenes and large ring macrolides

7.1 Nystatin (Nys)

Nystatin (e.g., nystatin A1 265) is a 38-membered macrolactone polyene with one deoxysugar, D-mycosamine, produced by S. noursei that has been used for many years as a clinical antifungal agent in treatment of mycoses. The synthase NYSS assembles sixteen acetate and three propionate units into the putative aglycone 264 which would then be oxidised and glycosylated to give 265 (Scheme 13). The mycosamine moiety may be synthesised from a different pathway to the other sugars illustrated in Scheme 2, and this may be why preliminary attempts to locate these clusters using sugar biosynthesis related probes from other macrolide systems have failed.

some unsuspected role, such as acting as a base, or assisting with a decarboxylative process for the C and Q deficient KAS0 during the loading of malonate.

NysB (3192 aa) contains two modules; the sequence is consistent with Mod1/2, and both ATs are propionate specific, and both modules contain inactive DH domains lacking an active site motif due to large deletions.

NysC (11096 aa) contains six acetate loading modules with a 49% identity to the six-module RAPS2 protein, strongly suggesting an evolutionary link via a common ancestor. NysC is the largest bacterial protein to date.

NysI (9477 aa) has a propionate loading AT domain on module 11 consistent with elongation steps 9–14. DH10, 11, 12, 13, and 14 all have large deletions including the active site motif. KR13 also lacks part of the active site motif, resulting in the formation of the hemiacetal ring, presumably crucial for conformational organisation of the long acyl chain, and for recognition and further chain elongation by NysJ.

In NysJ (5435 aa), DH16 has a large active site deletion, and in DH17 the active site His (Hx₃Gx₄P) is replaced by a Tyr.

NysK (2066 aa)⁹¹ contains an apparently intact but superfluous DH18, and contains a terminal TE1. A second discrete TE2 (NysE), whose gene is present downstream of nysC, shows high homology to other discrete TE2s, but rather less homology to known modular TE1s.

7.1.1. Deoxysugar

Several genes found in the cluster are involved in biosynthesis of the D-mycosamine moiety, which is the 2-epimer of dTDP-D-mycaminose 163. NysD3 shows far closer homology to GDP-D-mannose 4,6-dehydratases than to glucose 4,6-dehydratases. NysD2 shows strong homology to perosamine synthetases which 4-transaminate 4,6-dideoxy-4-oxo-GDP-D-mannose into 4-amino-4,6-dideoxy-GDP-D-mannose. Presumably NysD2 acts as the corresponding 3-transaminase on the corresponding 3-oxo analogue. Thus a putative route to mycosamine could be in parallel to that for dTDP-D-mycaminose shown in Scheme 2, but starting with the 2-epimer of dTDP-D-glucose, dTDP-D-mannose (obtained from fructose-6-phosphate) with NysD3 being analogous to TylA2, and NysD2 analogous to TylB. Other necessary genes are presumably outside of the cluster or recruited from primary metabolism.

A second cluster of over twelve genes was located using hybridisation with deoxysugar biosynthetic genes by Zotchev et al. from S. noursei that are consistent with macrolide production.⁹² However, deletion experiments showed that this cluster was not responsible for nystatin production.

Table 13 Partial gene cluster for nystatin biosynthesis in S. noursei

| Gene    | Size      | Putative function                                                                 |
|---------|-----------|-----------------------------------------------------------------------------------|
| `nysA`  | 1366aa    | NYSS (Loading domain)                                                            |
| `nysB`  | 3192aa    | NYSS Mod 1/2                                                                     |
| `nysC`  | 11096aa   | NYSS Mod 3-8                                                                     |
| `nysD1` | 506aa     | Aglycone O-19dTDP-D-mycaminosyltransferase                                        |
| `nysD2` | 352aa     | 3,6-Dideoxy-3-oxo-dT(or G) DP-D-mannose 3-aminotransferase                         |
| `nysD3` | 344aa     | dT(or G) DP-D-mannose-4,6-dehydratase                                             |
| `nysE`  | 251aa     | TE2                                                                              |
| `nysF`  | 254aa     | Apo-ACP 4'-phosphopantetheine transferase (ACPS)                                  |
| `nysG`  | 605aa     | ABC transporter [Export]                                                         |
| `nysH`  | 584aa     | ABC transporter [Export]                                                         |
| `nysI`  | 9477aa    | NYSS Mod 9-14                                                                    |
| `nysJ`  | 5435aa    | NYSS Mod 15-17                                                                   |
| `nysK`  | 2066aa    | NYSS Mod 18/TE1                                                                  |
| `nysL`  | 394aa     | P-450 [Hydroxylation at C-10 or C-16MeOx]                                         |
| `nysM`  | 64aa      | Ferredoxin [part of one or both P-450 systems]                                    |
| `nysN`  | 398aa     | P-450 [C-16MeOx or Hydroxylation at C-10]                                          |
| `nysR1` | 966aa     | Transcriptional activator/regulation                                              |
| `nysR2` | 953aa     | Transcriptional activator/regulation                                              |
| `nysR3` | 927aa     | Transcriptional activator/regulation                                              |
| `ORF4`  | 210aa     | Transcriptional activator/?                                                       |
| `ORF3`  | 253aa     | Transcriptional repressor/?                                                       |
| `ORF2`  | 354aa     | Transcriptional activator/?                                                       |

### 7.2 Amphotericin B

Amphotericin B 267 (S. nodosus) shows potent antifungal activity and has been in clinical use for over forty years. Though often the treatment of choice, its toxicity and low solubility have restricted its use to serious systemic or tropical infections. It is thought to complex with sterols present in eukaryotic membranes forming transmembrane pores resulting in uncontrolled loss of small molecules and ions from the cell, though it may instead act as an ion/small molecule transmembrane shuttle.

Presumably sixteen acetate units and three propionate units are assembled by a synthase complex similar to NYSS into the aglycone 266, and then C-8 and C-41 are oxidised by P-450s and dTDP-mycosamine is attached to the C-19 hydroxy group (Scheme 14). Presumably the co-metabolite amphotericin A (28,29-dihydroamphotericin B) is due to a defective or partially active ER5, despite a report of a separate cyanide inhibited enzyme activity. For many years, researchers have been only able to incorporate very low levels of labelled acetate into the metabolite. McNamara et al. grew S. nodosus on a fructose based broth and obtained high levels of acetate and propionate incorporation in the expected positions. No detectable acetate or propionate was incorporated into the mycosamine moiety, which is presumably derived from fructose. Significant levels of isolated ¹³C were observed in propionate 'methyl' carbons after feeding dilabelled acetate, indicating that the acetate was incorporated into the citric acid cycle, metabolised to succinate, and then to methylmalonate. However, label in C-3 of propionate was not incorporated into acetate positions, suggesting that it was all carboxylated to MMCoA, and not incorporated via the citric acid cycle. Examination by electrospray mass spectrometry revealed that acetate pulse feeding resulted in 50% of the molecules being unlabelled, and the other half being multiply (2–6 acetate units) labelled. Due to its low solubility, coupled with available high field spectrometers being routinely set-up for only proton irradiation or detection, Norwood used double quantum filtered heteronuclear single quantum coherence (DQF-HSBC) NMR spectroscopy to identify intact acetate units from [1,2-¹³C₂]acetate feeding experiments. Excitation of a proton, with transfer of magnetisation onto its linked ¹³C and then onto its partner ¹³C (DQF ensuring only resonances involving this transfer observed) occur before transfer back to the partner's proton for detection. Excitation and observation of protons leads to a significant increase in sensitivity compared with INADEQUATE or ¹³C–¹³C COSY

---

### Scheme 14

[Chemical structure diagram]

---

experiments, and the resulting spectrum is easily interpreted as resonances for the two protons and two carbons involved in the H–¹³C–¹³C–H four spin system form a rectangle.

Caffrey et al. have cloned the amphotericin B cluster from S. nodosus, disruption of which abolished amphotericin B production. The sequencing is nearly complete, and shows considerable homology to the nystatin synthase, with a similar overall organisation of genes.

### 7.3 Pimaricin (Pim)

Pimaricin 269 (Pimafucin®) is a 26-membered tetraene macrolactone that for many years has been added to cheese and cured meats to prevent mould contamination, and also shows antiviral properties and stimulates immune response. Martin et al. have partially sequenced 110 kbp from S. natalensis (Table 14). Pimaricin is presumably assembled via the aglycone pimaricinolide 268 as illustrated in Scheme 15. At one end of this cluster is `pimS0` that appears to encode a discrete loading domain PimS0 and convergently transcribed a ‘four-module’ synthase gene `pimS1` with a P-450 at the end of the cluster, `pimD`. `PimS0` and `pimS1` were fully sequenced, and

Table 14 Domain organisation of PIMS in S. natalensis

| PIMS | S. natalensis pimS0 (PIMS0) | pimS1 (PIMS1) | pimS2(PIMS2) | pimS3(PIMS3) | pimS4(PIMS4) |
|------|-----------------------------|---------------|--------------|--------------|--------------|
|      | -1                          | 0             | 1            | 2            | 3            | 4            | 5            | 6            | 7            | 8            | 9            | 10           | 11           | 12           |
| KAS  |                             |               | ✓            | ✓            | ✓            | ✓            |              |              |              |              |              |              |              |              |
| AT   | CL                          | A             | A            | A            | A            | A            | A            | A            | P            | A            | A            | A            | A            | A            |
| DH   |                             |               | ✓            | ✓            | ✓            | ✓            |              |              |              |              |              |              | ✓            | ✓            |
| ER   |                             |               |              |              |              |              |              |              |              |              |              |              |              | TE1          |
| KR   |                             |               | ✓            | ✓            | ✓            | ✓            | ✓            | ✓            | ✓            | ✓            | ✓            | ✓            | ✓            | ✓            |
| ACP  | ✓                           | ✓             |              |              |              |              |              |              |              |              |              |              |              |              |

---

Table 15 Partial gene cluster for pimaricin biosynthesis in S. natalensis

| Gene    | Size     | Putative function                                      |
|---------|----------|---------------------------------------------------------|
| orfX    | ?        | Unknown                                                |
| pimK    | 458aa    | Glycosyl transferase                                    |
| pimS4   | 2024aa   | PIMSMOD12incTE1                                        |
| pimS3   | 1808aa   | PIMSMOD11                                              |
| pimS2   | 9507aa   | PIMSMOD5toMod10                                         |
| pimI    | 255aa    | TE2                                                    |
| pimJ    | 343aa    | dTDP-mannose 4,6-dehydratase                            |
| pimA    | 602aa    | ABC transporter                                         |
| pimB    | 626aa    | ABC transporter                                         |
| pimE    | 549aa    | Homology with cholesterol oxidase                        |
| pimC    | 352aa    | Aminotransferase or regulatory                          |
| pimG    | 398aa    | P450                                                   |
| pimF    | 63aa     | Ferredoxin                                              |
| pimS0   | 1847aa   | Loading module; Mod(-1) and Mod0                         |
| pimS1   | 6797aa   | PIMSMOD1toMod4                                          |
| pimD    | 397aa    | P450                                                   |
| pimH    | 432aa    | Efflux pump                                             |

---

synthase genes being separated by other genes, other than tailoring genes (*i.e.* regulatory/export), interestingly recently found in the nystatin cluster as well (*vide supra*). *PimS0* encodes 1847 aa (193407 Da) and appears to be a discrete loading module downstream and convergently transcribed with *PimS1*. The N-terminal region is an ATP-dependent carboxylic acid ligase, CL(−1), similar to that found in RAPS1, followed by an ACP(−1), KAS0(S), AT0(A), then ends with a second ACP, ACP0. This may indicate massive deletions to obtain the loading domain, the first (−1) ACP having lost all of its previous domains. Deletion of PIMS0 terminated pimaricin production. *PimS1* encodes 6797 aa (710250 Da) contains modules consistent with the first four elongation steps, as all contain a KR, whilst Mod2+3+4 all contain a Hx₃Gx₄PDH domain.

In a recent paper, Aparicio *et al.* reported sequencing 85 kbp containing the full PKS module organization and other genes (Tables 14 and 15).⁹⁷ The KR9 has PxSx₂Gx₃A instead of the usual NAD(P)H binding GxGx₂Gx₃A motif that all the other KRs have, and is presumed inactive. The normal KAS domains have 60–90% sequence identity, and over 40% with other KASs, notably Rap, Ole and Ery. Several ACPs deviate from LGxDS; ACP0 has MGINS, ACP1 has QGFDS, ACP6 has AGFDS and ACP7 VGFDS. The presence of DH11 implies the formation of a double bond that is subsequently epoxidised. The codon usage in *pimS2* and *pimS0*, *pimS1* or *pimS4* suggest that *pimS2* has a different evolutionary origin or source and was not derived by duplication as for the other *pimS* genes. There is a rare TTA codon in *pimS1* that could be involved in regulation of expression.

The synthase genes are interrupted by pimaricinolide modifying genes, including two P450s, a small acidic ferredoxin, and a protein with homology to cholesterol oxidase of unclear function. *PimJ* shows homology to mannose 4,6-dehydratase from *Pseudomonas aeruginosa*, and is presumably involved in mycosamine (3-amino-3,6-dideoxymannose) biosynthesis.

---

8 Miscellaneous actinomycete Type I polyketides

8.1 Pironetin (Streptomyces)

Kobayashi *et al.* have investigated the incorporation of acetate, propionate, butanoate and SAM into pironetin 270 from *Streptomyces* sp. NK10958.⁹⁸

8.2 Linearmycin (Streptomyces)

Sakuda *et al.* have investigated the biosynthesis of linearmycin A271 and B from *Streptomyces* sp. and suggest that it is assembled from one 4-aminobutanoate (GABA) unit, 24 acetate and four propionate units.⁹⁹ The 4-aminobutanoate starter unit, though (poorly) labelled by acetate, presumably originates from ornithine.

8.3 Aflatrins (Streptomyces)

Sakuda *et al.* have examined the biosynthesis of the aflatrins (*e.g.* 272) from *Streptomyces* sp. MR1142 which are inhibitors

Nat. Prod. Rep., 2001, 18, 231–281

of aflatoxin production by *Aspergillus parasiticus*. Sakuda proposes that five of the ‘units’ are derived from glycolate. The incorporation levels of \[^{13}C\]glycolate at C-7, 27, 29, 33 and 35 were 1.3–1.4%, compared with 0.8–1.2% at many other sites. In addition, labelling from \[^{13}C\]acetate was significantly lower at these five positions (0.8–1.0%) than at the predicted acetate positions (*ca.* 3%). The *N*-methyl carbon was not labelled by acetate, propionate or glycolate, and as the corresponding *N*-demethyl metabolite can be isolated, is presumably SAM derived. The aflastatins are very unusual in having two 1,2,3,4-tetraol moieties, and it appears that they are derived from pairs of glycolate.

### 8.4 Pulvomycin (Streptoverticillium)

Pulvomycin (labilomycin) 273 was first isolated from *Streptoverticillium netropsis* in 1957; however, the reported structure was revised in 1985. It acts by inhibiting prokaryote protein biosynthesis, but in an unusual manner by blocking formation of the ternary complex between Tu (elongation factor), GTP and the amino acyl tRNA. Priestly and Gröger show that the backbone is derived entirely from acetate, with observable \[^{18}O\) isotope shifts at C-1, 5, 23, 25 and 33, and with no enrichment from feeding labelled propionate. The three pendant methyls attached to C-2 acetate positions (at C-4, 22 and 24) are all derived from methionine, reminiscent of virginiamycin and oncorhyncolide; the pendant methyls attached to the ‘carbonyl carbon’ of backbone acetates at C-7 and C-11 are both derived from the methyl carbon of acetate, presumably in a manner resembling similar pendant methyls found in some protocist metabolites such as the dinophysistotoxins.

---

### 8.5 Verucopeptin (Actinomadura)

Verucopeptin 274 is an antitumour metabolite isolated from *Actinomadura verrucospora* that consists of a polyketide attached to a 19-membered cyclodepsipeptide that includes an unusual piperazic acid and 3-hydroxyleucine unit. Labelling studies by Tsantrizos *et al.* showed the presence of five propionate units (2.1–3.6%) in the expected positions, but they could only detect two (3.6–3.9%) of the three expected acetate units. As the ‘incorporation’ at the ‘third’ acetate position C-26/C-27 was reported as equal to natural abundance (1.0%) and deuterated glycolate was also not incorporated, the experimental data strongly suggest a different alternative (methoxymalonte?). The biosynthetic pattern suggests a modular Type I synthase and the authors speculate that it may be in direct interaction with the peptide thiotemplate synthetase.

---

### 8.6 Azamacrolides (macrolactams)

#### 8.6.1 Vicenistatin

Kakinuma *et al.* have fed labelled acetate, propionate, glutamate 276 and glucose to the *Streptomyces sp.* that produces vicenistatin.

by Floss.¹⁰⁵ The ‘ansa’ antibiotics, or ‘ansamycins’ are characterised by a benzenic or naphthalenic quinoid based chromophore with an ‘ansa’ bridge (ansa = having a handle) linking one end of the chromophore with the other end by an amide linkage. The chromophore is derived from a *mC*₇*N* unit derived from 3-amino-5-hydroxybenzoic acid (AHBA) produced by the ‘aminoshikimate’ pathway; any remaining part of the chromophore, and the ‘ansa’ bridge are polyketide derived, with the starter unit amino group closing the macro-lactam ring. A semisynthetic rifamycin, rifampicin is one of the most effective clinical treatments of mycobacterial infection. Rifamycins also block the RNA-dependent DNA polymerases of retroviruses.

### 9.1 3-Amino-5-hydroxybenzoic acid (AHBA)—the ‘aminoshikimate’ pathway

Many feeding experiments have shown that AHBA 273a (Scheme 17) is the starter unit for both the benzenic (*e.g.* ansamitocin or ansatrienin A) and naphthalenic (*e.g.* rifamycin B) ansa antibiotics and some other polyketides such as mitomycin C.¹⁰⁶⁻¹⁰⁹ The naphthalenic portion is assembled from AHBA through a Michael reaction of the first part of the growing acyl chain on the P-450 oxidised AHBA, but it is not clear when this happens. Initial proposals that AHBA was derived from shikimate, its immediate precursors or its metabolites, could not be supported by any incorporation experiments. Indeed, *¹³C*-labelled shikimic acid was readily incorporated into the cyclohexanycarbonyl moiety of ansatrienin, but not into the AHBA portion, showing that lack of incorporation was not due to membrane impermeability.¹¹⁰ In 1980 Horneman *et al.* suggested that 3-deoxy-D-arabino-heptulosonic acid 7-phosphate (DAHP) 266a was converted into amino-DAHP 270a,¹¹¹ whilst Rhinehart suggested transamination of a 5-dehydroshikimate.¹¹² Floss *et al.* studied the pathway using cell-free extracts of the rifamycin B producer *Amycolatopsis mediterranei* and the ansatrienin producer *S. collinus* Tül1892.¹¹³,¹¹⁴ Some experiments were performed as ‘inverse isotope dilution’ assay, in which *¹³C*-labelled product was added at the end of the incubation, to quantify unlabelled AHBA formation from an unlabelled added substrate by GCMS. Addition of amino-DAHP 270a, amino-DHQ 271a and amino-DHS 272a (*ca.* 95%) all gave high (>40%) conversion into AHBA 273a, and low incorporation into PEP 264a or E4P 265a (6%), whilst addition of DAHP 266a (with or without glutamine) DHQ 267a, DHS 268a or quinic acid gave no significant formation of AHBA 273a (<1%). A small amount of aminoshikimate was converted into AHBA (1%) presumably due to some activity of a shikimate dehydrogenase. These results disproved earlier proposals of Horneman¹¹¹ and Gygax¹¹⁵ that DAHP 266a was a precursor of AHBA 273a. Thus Floss proposed a sequence parallel to that of the shikimate pathway, in which the nitrogen is introduced right at the beginning (Scheme 17). Reaction of the aldehyde of E4P 265a with an ammonia equivalent to give the corresponding imine, followed by an aldol reaction analogous to that forming DAHP, would give amino-DAHP 270a. An early report by Jiao *et al.*¹¹⁶ indicated that glutamine amide N was the source of this nitrogen. However, Floss was unable to incorporate *¹⁵N* from glutamine or ammonium chloride (<3%) into AHBA.¹¹⁴

The aromatisation of amino-DHS 272a to AHBA 273a in rifamycin biosynthesis is catalysed by a pyridoxal phosphate dependent enzyme RifK, deletion of *rifK* terminates antibiotic production (Scheme 18). Floss *et al.* cloned AHBA synthase from *Amycolatopsis mediterranei* and overexpressed it in *E. coli* as a dimer with one pyridoxal phosphate per subunit, which, after forming a Schiff’s base with the amino group 274a, catalyses an α,β-dehydration and stereospecific 1,4-enolation of the intermediate 275a with loss of the pro*S* hydrogen.¹¹⁷ The RifK sequence has highest homology with PLP dependent

Nat. Prod. Rep., 2001, 18, 231–281 255

PEP
Phosphoenol pyruvate
264a

E4P
Erythrose-4-phosphate
265a

3-deoxy-D-arabino-heptulosonic acid 7-phosphate
DAHP
266a

4-amino-3,4-dideoxy-D-arabino-heptulosonic acid 7-phosphate
Amino-DAHP
270a

HO
COOH
O
OH
OH
OH
OH
OH
OH
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH₂
NH
AHBA  
273a  

RifA  
2 x MeMalonate  
1 x malonate  

PKS-S  
274b  

RifS, RifT  
unknown cyclase  

P-450  

PKS-S  
275b  

P-450  

PKS-S  
276a  

276a  
(redrawn)  

S-PKS  

Me  

O  

O  

NH₂  

O  

OH  

Me  

277a  

S-PKS  

RifF  

HO  

Me  

Me  

Me  

Me  

HO  

HO  

HO  

HO  

Me  

O  

NH₂  

O  

O  

HO  

OH  

OH  

Me  

278a  

R = H proansamycin X  

carbon lost via  
decarboxylation  

P-450 (R = H to OH)  
and oxidative aromatisation  

HO  

Me  

Me  

Me  

HO  

HO  

HO  

HO  

CH₂R  

Me  

O  

NH  

O  

O  

HO  

R'  

Me  

-H₂O  
(aromatization)  

R = -OH, R' = H  
protorifamycin I  
279  

P-450 (R' = H to -OH)  

R = -OH, R' = -OH rifamycin W  
280  

36  

Me  

O  

35  

O  

MeO  

27  

C₂H₅  

25  

Me  

34  

OH  

Me  

21  

OH  

Me  

31  

HO  

Me  

29  

CO₂H  

O  

12  

Me  

11  

O  

10  

O  

1  

OH  

14  

Me  

8  

OH  

2  

NH  

15  

O  

30  

Me  

17  

rifamycin B  
281  

oxygen derived from AHBA  
carboxylate group  

Scheme 19  

duplicated ten times. The interdomain linker regions between  
DH and KR domains (except Mod2) are all very well con-  
served, again suggesting a crucial role for this linker region pre-  
sumably in protein–protein recognition. As found for DEBS2,  
DEBS3, RAPS2 and RAPS3, there are N-terminal extensions  
of around 30–34 aa in front of the first KAS domain, and there  
is also a long C-terminal tail on all RIFS except RifE, whose  
terminal sequence overlaps the predicted translation site of the  
cyclase gene rifF. There are DH domains in all modules except  
Mod2 and Mod3. DH1 and DH5 have large deletions in the  
active site motif, DH8 has large deletions at either terminus.  
Dehydration is required in modules 4, 9 and 10. However, whilst  
the active site motif Hx₃Gx₄P of DH4 (and that of the  
superfluous DH6) conforms with all DH’s so far found in  
modular PKS, the active site motif in DH9 is Hx₃Gx₄A and in  
DH10 is Hx₃Dx₄P (and that of the superfluous DH7). The  
overall sequence similarity in the active site motif has little  
correlation with active function; for example DH4 (used), DH6  
(superfluous), DH7 (superfluous) and DH9 (used) all have  
HxxGGVV as the first part of the motif, whilst DH8 (flanking  
deletions) has HxxEDLV and DH10 (used) has HxxRDV,  
i.e. two acidic and basic residues have appeared in the motif,  
presumably for a functional purpose. Curiously, the overall  
sequence of DH9 and DH10, both functionally required, dis-  
plays considerable differences from other PKS functional DH’s.  
The KR domain in Mod3 lacks an NADPH binding motif. The  
RIFS ACP sequences as a family have signature sequences that  
distinguish them from the ACP’s of DEBS or RAPS.  

A gene immediately downstream of the modular cluster, rifF  
is thought to code for an enzyme that catalyses the intra-  
molecular cyclisation of the alicyclic growing acyl chain 277  
through reaction of the AHBA amino group with the terminal  
thioester to form the amide bond in proansamycin X 278. RifF  
contains homology to arylamine N-acetyltransferases, agreeing  
with the proposed role as a rifamycin amide synthase. Schupp  
et al. and Yu et al. have independently introduced a small  
deletion into rifF, and obtained a series of acyclic intermediates  
282–289 (Scheme 20), presumably due to the resulting traffic  
jam of intermediates attached to the synthase when release of  
the final product is no longer promoted by an active RifF.¹²³,¹²⁴  
All intermediates produced by modules 3–10 were isolated  
(some as lactones or dehydrated) and characterised, these may  
be being released by the discrete RifR ‘TE2’ roaming all over  
the ‘elephantine’ PKS like an egret ‘picking off’ partly formed  
intermediates, stuck on the synthase by the log jam at the end of  
the production line. Some intermediates isolated by Yu et al.  
contained the corresponding naphthoquinone structures, lack-  
ing an 8-hydroxy group. Curiously, the final intermediate, 289,  
has a different reported double bond stereochemistry at the  
carboxy end to that observed in proansamycin X 278. In a  
recent paper, Hutchinson et al. observed a 30–60% decrease in  
both rifamycin biosynthesis in the release of these partly  
assembled intermediates when the RifR TE2 was disrupted in  
the RifF disruptant.¹²⁵ As the effect of deleting RifR was simi-  
lar on rifamycin production and part-formed intermediates,  
they concluded that TE2 was not assisting in the premature  

Nat. Prod. Rep., 2001, 18, 231–281 257

Table 16 Gene cluster for rifamycin biosynthesis in *Amycolatopsis mediterranei* (with analogous genes for AHBA biosynthesis in the naphthomycin cluster in *S. collinus*) (arrows show direction of gene transcription)

| Gene | Size | Putative function | Naphthomycin |
|------|------|-------------------|--------------|
| orf0 | →    | 397aa             | P-450        |
| rifA | →    | 4735aa            | RIFS Mod 0/Mod 1/Mod 2/Mod 3 |
| rifB | →    | 5060aa            | RIFS Mod 4/Mod 5/Mod 6 |
| rifC | →    | 1763aa            | RIFS Mod 7   |
| rifD | →    | 1728aa            | RIFS Mod 8   |
| rifE | →    | 3413aa            | RIFS Mod 9/Mod 10 |
| rifF | →    | 260aa             | Amide synthase/cyclase |
| orf1 | →    | 62aa              | orf1 unknown |
|      |      |                   | [napA1 naphthomycin ring closure/RifF] |
|      |      |                   | [napB naphthomycin ring closure] |
| rifG | →    | 336aa             | Aminodehydroquinate synthase |
| rifH | →    | 441aa             | Amino-DAHP synthase |
| rifI | →    | 263aa             | Aminoquinate dehydrogenase |
| rifK | →    | 388aa             | AHBA synthase |
| rifL | →    | 360aa             | Oxidoreductase |
| rifM | →    | 232aa             | Phosphatase   |
| rifN | →    | 303aa             | Kinase       |
| rifO | ?    | 261aa             | REG          |
| orf2 | →    | 310aa             | Esterase     |
| rifP | ?    | 525aa             | Efflux/export |
| rifQ | ?    | 242aa             | Transcriptional repressor |
| orf3 | ←    | 166aa             | Unknown      |
| orf4 | ←    | 403aa             | P-450        |
| orf5 | ←    | 421aa             | P-450        |
| orf6 | ←    | 435aa             | dNTP-hexose dehydratase |
| orf7 | →    | 381aa             | dNTP-hexose glucosyl transferase |
| orf8 | →    | 206aa             | dNTP-hexose 3,5-epimerase |
| orf9 | ←    | 432aa             | Aminotransferase |
| orf10 | ←   | 330aa             | Oxidoreductase |
| orf11 | →   |                   | [H]          |
|      |      |                   | Not yet fully sequenced |
| rifR | ←    | 252aa             | TE2         |
| orf13 | ←   | 500aa             | P-450       |
| orf14 | →   | 272aa             | MeT         |
| orf15 | →   | 590aa             | Transketolase |
| orf16 | ←   | 351aa             | P-450       |
| rifJ | →    | 163aa             | Aminodehydroquinate dehydratase |

Table 17 Domain organisation of RIFS in *Amycolatopsis mediterranei* (CL is carboxy ligase)

| Amycolatopsis mediterranei rifA | rifS | rifT | rifB | rifC | rifD | rifE |
|---------------------------------|------|------|------|------|------|------|
| RIFS                            |      |      |      |      |      |      |
| KAS                             | 0    | 1    | 2    | 3    | 4    | 5    | 6    | 7    | 8    | 9    | 10   |
| AT                              | P450 | P450 | √    | √    | √    | √    | √    | √    | √    | √    | √    | RifF |
| DH                              | CL   | P    | A    | P    | P    | P    | P    | P    | P    | A    | P    |
| ER                              |      |      |      |      |      |      |      |      |      |      |      |
| KR                              |      |      |      |      |      |      |      |      |      |      |      |
| ACP                             | √    | √    | √    | √    | √    | √    | √    | √    | √    | √    | √    |

release of partly formed intermediates. This would concur with the observations of Heathcote et al. that TE2's main role could be removal of propionate groups after premature decarboxylation of methyl malonate (see Section 3.2). 37

9.2.3 Post-PKS (tailoring)

The conversion of the putative PKS product proansamycin X278 into rifamycin B281 involves many steps, some not clarified. Downstream of the modular cluster are seven genes for AHBA synthesis, then a cluster of largely unexplored regulatory and tailoring genes, with an eighth AHBA synthesis gene. Orf6-orf10 encode enzymes for the biosynthesis of an amino sugar that could be involved in forming an as yet unknown rifamycin glycoside, perhaps for deactivation and export purposes.

The conversion of proansamycin X278 has been thought to proceed via dehydration to the aromatic ring in protorifamycin I279, followed by oxidation to rifamycin W280, though there is some controversy over this sequence with Rickards suggesting tautomerisation followed by oxidation. The corresponding synthase domain has an inactive DH1, suggesting that the C-8 hydroxy group should originate from AHBA, rather than subsequent oxidation. Yu et al. fed 85% 18O-labelled AHBA to a rifI mutant unable to make AHBA, obtaining highly (82%) labelled rifamycin B, indicating that that hydroxy group is derived from AHBA carboxy group, and that many of the intermediates isolated by them that contained a naphthoquinone structure without an 8-hydroxy group were shunt metabolites. 12 Thus, Yu proposed that the pathway, shown in Scheme 19 as a dotted arrow, in which dehydration to the benzene ring to protorifamycin I279 is followed by P-450 oxidation, was not the main pathway. The final steps presumably involve double bond oxidation/cleavage, loss of one carbon by decarboxylation, and acetal formation at C-12 by one phenolic hydroxy and a hydroxymethyl.

The *Amycolatopsis* produce not only rifamycin, but also vancomycin and dethycin, all extremely promising drugs, yet

the genetic tools for *Amycolatopsis* sp. are relatively underdeveloped, preventing as much progress occurring as in other better understood systems. Some attempts to disrupt PKS genes to determine their function have been unsuccessful due to low transformation and recombination frequencies. Khosla *et al.* have recently developed a specific deletion mutant of *A. mediterranei* as a suitable host for gene expression, in which either all ten PKS modules, or just the last six, have been deleted. The latter, after reintroduction of *rifA*, produces the expected tetraketide. Khosla *et al.* fed 3-hydroxybenzoic acid and 3,5-dihydroxybenzoic acid to mutants unable to synthesise AHBA and obtained efficient production of the corresponding analogues of tetraketide **282**, indicating that early modules showed relaxed specificity towards the starter unit but Rif5 P450 or later modules did not.

### 9.3 Naphthalenoid and naphthoquinoid ansamycins: naphthomycins, actamycin and ansatrienin

#### 9.3.1 Naphthomycin

Floss *et al.* and Rickards *et al.* have shown that naphthomycin A **293** is biosynthesised in *Streptomyces collinus* from seven acetate and six propionate units as expected, and that label from ${}^{13}$C$_3$-glycerol was incorporated as expected via PEP **264** and E4P **265** into the starter unit AHBA **273** (Scheme 21).

Using incubation under ${}^{18}$O$_2$, Hooper and Rickards have shown using ESMS that the oxygen at C-30 in actamycin **290** originates from dioxygen (20% labelling), and is introduced late on in the biosynthetic pathway. A second carbon showed an ${}^{18}$O-isotope shift; gradient HMBC NMR showed that this second dioxygen derived oxygen was on carbonyl C-28 (not C-31) as that carbonyl showed three bond correlation with both the solitary aromatic proton and to the lactam NH. These two oxygen atoms are derived from separate dioxygen molecules.

The related naphthomycins H **292** and A **293** contain chlorine at C-30; and naphthomycin J **294** and I **295** contain the alkylthio substituent -SCH$_2$CHNHAcCO$_2$H. Prefunctionalised chloro/alkylthio AHBA is not incorporated into these compounds, the functionality presumably being added late in the biosynthetic process. To show this, Hooper and Rickards added NAC thioesters of deuterated 2-methyl-3-hydroxy-3-(3-amino-5-hydroxyphenyl)propionate to *S.* sp. E/784, but could detect no significant levels of labelled chlorinated metabolite, yet obtained normal levels of unlabelled hydroxylated metabolites actamycin **290** and naphthomycin D **291**. The fermentation broth also yielded two new metabolites **296** and **297** in which the thiol group of *N*-acetylcysteamine has added to C-30, to give compounds analogous to the natural thioansamycins naphthomycin F and G. The authors suggest that all these thionaphthomycins are obtained through a non-enzymatic addition-elimination reaction on the corresponding chlorometabolites (Scheme 22).

#### 9.3.2 Ansatrienin (mycotrienin)

Naphthomycins and ansatrienin A **299** are both produced by the same bacterium, *S.* *collinus* Tü 1892, and both have AHBA **273** as a starter unit. Reynolds *et al.* have sequenced 20 kbp of the ansatrienin producing gene cluster (Ans), and 13 kbp of the naphthomycin cluster (Nap), both containing their own set of genes to synthesise AHBA, adjacent to the acyl CoA ligase domain at one end of the polyketide synthase responsible for loading the AHBA as starter unit. The two AHBA synthesis ing clusters were compared with each other, and to that found in *Amycolatopsis mediterranei* the producer of rifamycin. The organisation of the AHBA synthase genes in the naphthomycin cluster (Nap) was very similar to that found for rifamycin as illustrated in Table 16. In contrast, the organisation of the AHBA synthase genes in the ansatrienin producing cluster was quite different (Table 18). The discrete Nap ACP2 has homology with Type I ACP domains, not with the discrete Type II discrete ACPs. The role of the NapA2 and ACP2 remains

Nat. Prod. Rep., 2001, 18, 231–281

290 R = OH, R' = H: actamycin  
291 R = OH, R' = Me, naphthomycin D  
292 R = Cl, R' = H: naphthomycin H  
293 R = Cl, R' = Me, naphthomycin A  
294 R = -SCH₂CH(NHAc)CO₂H, R' = H: naphthomycin J  
295 R = -SCH₂CH(NHAc)CO₂H, R' = Me, naphthomycin I  
296 R = -SCH₂CH₂NHAc, R' = H  
297 R = -SCH₂CH₂NHAc, R' = Me  

formed by addition of NAC thioesters  

Scheme 21  

The biosynthesis of CHC is illustrated in Scheme 24. The genes involved in CHC biosynthesis *ansJKLM* and *chcA* are clustered. The gene responsible for the isomerase ChcB (*chcB*) 320 to 321 (Scheme 24) is not thought to be located within the cluster. The five genes have been transferred into *S. avermitilis* enabling production of CHC-CoA 300,⁸⁵ indicating that ChcA performs three enoyl reductions, and another enzyme elsewhere in *S. avermitilis* can perform the Δ²,Δ¹-enoyl CoA isomerisation normally performed by ChcB. In a recent paper, Reynolds *et al.* report the isolation and purification of ChcB from *S. collinus*,¹³² and its overexpression in *E. coli* as a 28 kDa protein belonging to the hydratase/isomerase superfamily. The neighboring DNA did not contain polyketide synthase-like sequence. An insertional deletant was unable to produce ω-cyclohexanyl fatty acids, but was able to grow on (9Z)-octadecenoate, indicating that ChcB was involved in CHC production, but was not involved in fatty acid degradation. The ChcB showed wide substrate activity, with a significant (10-fold *K*ₐₜ/*K*ₘ) preference for acyclic enoyl CoA substrates such as (3E)-hexenyl CoA, with a very high rate (*K*ₐₜ 13 min⁻¹). They also found ChcB activity in all Streptomycetes examined: *S. cinnamonensis, lividans* and *hygroscopicus*. This, coupled with the substrate preference for acyclic enoyl CoAs suggests that (S)-cyclohex-2-enecarbonyl CoA 320 is not the prime physiological substrate. This is a rare example of a gene required for polyketide synthesis that is not contained within the biosynthetic cluster.

### 9.3.3 Mitomycins

The mitomycins (e.g. mitomycin C 302) were isolated over 40 years ago and have been used in anticancer therapy since the 1960’s due to the presence of an aziridine unit that after bioactivation by a reductive enzyme (e.g. a cytochrome c reductase) can efficiently alkylate and cross-link double stranded DNA, inhibiting its synthesis in a rapidly dividing tumour. They are still important in combination cancer chemotherapy. Mitomycin preferentially alkylates G+C so effectively that cell-death occurs from a single cross-linking event. Whilst not strictly assembled via a polyketide assembly, it uses an AHBA 273 starter unit, so is briefly mentioned here. The mitomycins are assembled from AHBA, D-glucosamine 304, L-methionine and arginine or citrulline derived carbamoyl phosphate 303 (Scheme 25). Sherman *et al.* have sequenced 55 kbp from *S. lavendulae* containing 47 genes.¹³³,¹³⁴ Curiously, adjacent to

unclear. NapA1 shows homology with RifF, the arylamine acetyltransferase-like ring closing amide forming enzyme in rifamycin biosynthesis, and contains a Cys that may form a thioester with the completed polyketide chain, prior to attack by the aromatic amine group of 298. AnsC has homology with the pipecolate incorporating enzymes (PIE) in Rap, and may form an amide bond between cyclohexanycarbonyl CoA thioester 300 and D-alanine to form 301. NapB is homologous with non-heme oxygenases/halogenases.

The attachment of D-alanine and CHC CoA 300 to the backbone of ansatrienin occurs late in the biosynthetic sequence as products without these groups have been isolated.¹³⁰,¹³¹ Whilst AnsC forms the amide linkage between the alanine and CHC, the mechanism of attachment of the D-alaninyl moiety to a backbone hydroxy group remains unclear (Scheme 23).

Nat. Prod. Rep., 2001, 18, 231–281

Table 18 Partial gene cluster for ansatrienin biosynthesis in S. collinus Tü 1892

| Gene    | Putative function                                                                 |
|---------|-----------------------------------------------------------------------------------|
| $ansA$  | ANSS (ansatrienin polyketide synthase)                                             |
| $ansB$  | Dehydroquinate synthase [AHBA]                                                    |
| $ansC$  | Peptide synthase [CHC D-alanine ligation]                                          |
| $ansD$  | Unknown                                                                          |
| $ansE$  | Aminodehydroquinate dehydratase [AHBA]                                            |
| $ansF$  | AHBA synthase [AHBA]                                                              |
| $ansG$  | Oxidoreductase [AHBA]                                                             |
| $ansH$  | Phosphatase [AHBA]                                                                |
| $ansI$  | DAHP synthase [AHBA]                                                              |
| $ansJ$  | Shikimate (or -3-phosphate) to (3R,4R)-3,4-dihydroxycyclohexa-1,5-dienecarboxylic acid |
| $ansK$  | Ligase for 4,5-dihydroxycyclohexa-1,5-dienecarboxylic acid to CoA thioester 311 to 315 |
| $ansL$  | Acyl CoA dehydrogenase 316 to 317                                                  |
| $chcA$  | 1-Cyclohexenylcarbonyl CoA reductase and other ERs 315 to 316; 318 to 319 and 321 to 300 |
| $ansM$  | NADH dependent flavin oxidoreductase 317 to 318                                    |

---

**Scheme 23**

![Chemical Structure Diagram](https://i.imgur.com/chemical_structure_1.png)

---

**Scheme 24 Biosynthesis of DHCHC and CHC (ans and chc genes from S. collinus). (See also Scheme 30.)**

![Chemical Synthesis Pathway Diagram](https://i.imgur.com/synthesis_pathway_1.png)

Nat. Prod. Rep., 2001, 18, 231–281

Glucose → PEP  
Citric acid cycle → oxosuccinate  

Scheme 25 Biosynthesis of mitomycin C.

PEP 264 and a citric acid derived four-carbon unit (oxo-succinate?) via moieties based upon 309 and 310 (Scheme 26), instead of PEP 264 combining with erythrose-4-phosphate via the aminoshikimate pathway as for AHBA (Scheme 17). All three carbons of glycerol, and all four carbons of succinic acid were incorporated intact (Scheme 27), and labelled glutamate presumably incorporated via 2-oxoglutarate and decarboxylation to succinyl CoA.

### 10 Manumycins

The manumycin family (e.g., manumycin A 305, Scheme 27) has been recently extensively reviewed by Sattler, Thiericke and Zeeck. The family is grouped by having a 3,4-AHBA 307 moiety that biosynthesised (Scheme 26) in a quite unrelated manner to the shikimate or aminoshikimate pathway. This 3,4-AHBA moiety acts as the equivalent of an amino acid NRPS like extender unit part way through what is otherwise a polyketide chain with an unusual termination unit.

#### 10.1 3-Amino-4-hydroxybenzoic acid (3,4-AHBA)

3-Amino-4-hydroxybenzoic acid 307 is a component of manumycin A 305 and asukamycin 306, the aromatic polyketide murayaanthraquinone, and 4-hydroxy-3-nitroso-benzamide. Gould et al. have recently shown that in S. murayamaensis, 3,4-AHBA is assembled from glucose via

Scheme 27 Biosynthesis of manumycin A and asukamycin.

Starter unit

![Chemical Structure 1](chemical_structure_1.png)

PIE

RAPS3-ACP14-S

PKS

Putative PKS product

Post-PKS enzymes

rapamycin

Starter unit

(4R,5R)-4,5-dihydroxycyclohex-1-enecarboxylic acid, CoA thioester

(thioester of 313)

or

(1R,3R,4R)-3,4-dihydroxycyclohexanecarboxylic acid CoA thioester

(thioester of 314)

Termination unit

deaminative cyclisation

lysine

pipecolyl adenosylmonophosphate

Scheme 28 Biosynthesis of rapamycin.

more modules with three more acyl units, with a C₅N amine used as a termination functionality in place of a TE. Two monooxygenases are either required afterwards, or conceivably during megasynthase assembly (Scheme 27). A precursor directed intermediate has been isolated upon feeding 3-amino-benzoic acid.¹³⁶ᵇ

### 11 Rapamycin and related compounds

This section covers the three potent immunosuppressants rapamycin 324, FK506 328 and FK520 (also called ascomycin and immunomycin) that prevent T-cell proliferation by binding to immunophilins, being particularly effective at preventing graft rejection after organ transplantation. These potent immunosuppressants act at very much lower concentrations than cyclosporin. They are all structurally and biosynthetically closely related, having a (1R,3R,4R)-3,4-dihydroxycyclohex-1-enyl (or -hexanyl)-carbonyl starter unit; followed by PKS extension (14 extender units (7 × C₂ and 7 × C₃) for Rap, or ten (4 × C₂, 5 × C₃, 1 × C₅) for FK506), followed by a lysine derived pipecolate (piperidine-2-carboxylate) termination unit (as its adenosylmonophosphate), without glycosylation, but some O-methylation and P-450 oxidation occurring. Rapamycin is assembled in *Streptomyces hygroscopicus* from a (1R,3R,-4R)-3,4-dihydroxycyclohexanecarbonyl CoA (DHHC CoA) starter unit followed by seven acetate and seven propionate extender units, and terminate with a pipecolic acid unit to form a large macrocyclic ring (Scheme 28). FK506 328 is assembled in *S. tsukubaensis* and FK520 (immunomycin/ascomycin) is assembled in *S. hygroscopicus* var. *ascomyceticus*, both from a DHHC CoA starter unit, ten extender units, terminating with a pipecolic acid moiety. After ring assembly to give the putative PKS product 327, oxidations and O-methylations of the main chain along with oxidation and dehydration of the C-21 side chain propyl group give the product FK506 328 (Scheme 29).

#### 11.1 Cyclohexanecarboxylic acid (CHC) and 3,4-dihydroxy-cyclohexanecarboxylic acid (DHCHC)

Whilst the cyclohexanecarboxy unit is relatively rare in nature, several natural products from a range of micro-organisms use CHC based starter units: the ω-hydroxy-fatty acids produced

Starter unit
HO
HO
AMP

Termination PIE
COSCoA

SU-PPAXPPAAAPA-PIE
10 extension rounds

FKBS

Starter unit
HO
HO

O-Met

termination
unit
[O] / -H₂O
P-450/O-MeT
P-450/O-MeT

327
Putative PKS product

Tailoring
5 x P-450
1 x DH
3 x O-MeT

MeO
Me
Me
HO
Me
OMe
OMe
FK506
328

Scheme 29 Biosynthesis of FK506.

by the thermophile *Alicyclobacillus acidocaldarius* (formerly *Bacillus acidocaldarius*) and the mesophile *Curtobacterium pusillum*,² ansatrienin (mycotrienin) produced by *S. collinus* and *S. rishiriensis*, whilst the immunosuppressants rapamycin, FK506 and FK520 produced by strains of *S. hygroscopicus* use 3,4-dihydroxycyclohexanecarboxylate 314 (DHCHC) based starter units.

Demain *et al.* showed in 1993 that labelled shikimic acid 269 was incorporated into the 3,4-dihydroxycyclohexanycarbonyl 314 starter unit of rapamycin.¹³⁸ Floss *et al.*,¹³⁹⁻¹⁴¹ Reynolds *et al.*¹⁴²⁻¹⁴⁴ and O'Hagan *et al.*¹⁴⁵ have elucidated the pathway in detail using labelled precursors and blocked mutants and the stereochemistry of each step B (Scheme 24 and 30). Reynolds reports that *anti* 1,4-conjugate elimination of water from 269 with loss of the 6-*pro-(R)* hydrogen produces (3*R*,4*R*)-3,4-dihydroxycyclohexa-1,5-dienecarboxylic acid 311 (still as its free acid).¹⁴⁴ This process appears identical in both the preparation of DHCHC 314 in *S. hygroscopicus* and in the biosynthesis of CHC 300 by *S. collinus* and *Alicyclobacillus acidocaldarius* as reported by Handa and Floss.¹⁴¹ The two pathways now diverge: the sequence of intermediates to DHCHC 314 are thought to be as free acids, whilst the sequence of intermediates to CHC 300 appear to be coenzyme A thioesters.

A *syn* 1-Si enoyl reductase reduction of the Δ¹-double bond in 311 gives (1*S*,4*R*,5*R*)-4,5-dihydroxycyclohex-2-enecarboxylic acid 312, followed by a suprafacial Δ²,Δ¹-double bond isomer-

COSCoA
H
OH
317

*H*
anti, 3 re
*H*
*H*
OH
318

COSCoA
HS
H*
OH
319

-H₅OH
anti
H*
H
320

COSCoA
321

ER D₂O
NADPH* (pro-S)
anti, 1 si
D
H*
300
CHC

Scheme 30 Stereochemistry of steps in CHC biosynthesis. (See also Scheme 24.)

ation gives (4*R*,5*R*)-4,5-dihydroxycyclohex-1-enecarboxylic acid 313, with *anti*, 1-Re enoyl reduction giving the product DHCHC, still as the free acid.

The CHC pathway, in contrast, firstly is thought to convert the free acid 311 into its CoA thioester 315. *anti* 1-Si enoyl reduction to 316 followed by *syn* 1,4-dehydration gives the (5*R*)-5-hydroxycyclohexa-1,3-dienylcarbonyl CoA thioester 317. This is reduced overall in *anti* 3-Re stereochemistry to 318, though it may proceed either via direct reduction of the Δ³-double bond, or through a two-step process involving 1,4-conjugate reduction to the 5-hydroxy-2-ene followed by isomerisation to the 5-hydroxy-1-ene product. Further ER to 319, *anti*-dehydration to 320, suprafacial isomerisation to 321 is followed by the third *anti* 1-Si ER gives CHC 300 as illustrated in Schemes 24 and 30.

Reynolds *et al.*¹⁴² have shown that in the final step, enoyl reduction of 321 to 300 uses the 4-*pro-(S)* hydrogen of NADPH with addition of hydride to the 2-Si face, and *anti* addition of a solvent proton, and have cloned and overexpressed in *E. coli* this ER gene (*chcA*) from *S. collinus* coding for a 280 aa (29.7 kDa) protein¹⁴⁶ that showed homology with short chain alcohol dehydrogenases. The overexpressed protein was found capable of catalysing all three enoyl reductases in this pathway that all occur with the same stereochemistry.¹³⁹,¹⁴⁰ O'Hagan *et al.* has investigated the effect of substituting fluorine at C-3 on the rate of the ER, and found a five-fold increase in *V*ₘₐₓ* presumably due to the electronegativity of the fluorine lowering the energy barrier for approach of the hydride to C-2. In a recent paper, Reynolds *et al.* have transferred five genes (*ansJKLchc-AansM*) from *S. collinus* to *S. avermitilis* and produced Doramectin, the avermectin with a CHC starter unit.⁸⁵ The isomerisation catalysed by ChcB (not located in ansatrienin biosynthetic cluster) is presumably being catalysed by another protein.

### 11.2 Rapamycin (Rap)

In 1995, Leadlay *et al.* reported cloning and sequencing 107 kbp from *S. hygroscopicus* that contained three PKS genes, a gene coding for PIE, and 24 additional ORFs that may be involved in Rap biosynthesis, regulation or transport (Fig. 2).¹⁴⁷ Stassi *et al.* have reported isolating a second gene PKS cluster (codenamed 'Hyg') from the rapamycin producing strain of *S. hygroscopicus*.¹⁴⁸ There are four large ORF's, each encoding a single PKS module, the first also containing a ligase domain with high homology to that of the RAPS1 ligase CL, suggesting that there may be a shikimate derived starter unit. The Hyg AT's were AT(P)1, AT(A)2, AT(P)3, AT(P)4. DH and KR domains were identified in all four modules (not necessarily

(a)

![Diagram of Gene Cluster](#)

(b)

0 kbp    20 kbp    40 kbp    60 kbp    80 kbp    100 kbp   120 kbp
(Pipecolate inserting enzyme)
Mod 14+13+12+11        Mod 0+1+2+3+4         Mod 5+6+7+8+9+10
rapQONMLKJIHGFE        rapC                  rapP                 rapA                  rapB                  rapZYXWVRSUT
13 ORFS                (RAPS3)               (PIE)                (RAPS1)               (RAPS2)               9 ORFS
C                      N                      N                     C
6,260 aa               1,541 aa              8,563                 10,222 aa
660 kDa                170 kDa                900 kDa               1.07 MDa

(c)
Loading domain
RAPS1: CL0 - ER0 - KAS1 - AT(P)1 - DH1 - ER1 - KR1 - KAS2 - AT(A)2 - DH2 - KR2 - KAS3 - AT(P)3 - DH3 - ER3 - KR3 - KAS4 - AT(P)4 - DH4 - KR4 -
ACP1                       ACP2                       ACP3                       ACP4
(inactive?)
RAPS2: KAS5 - AT(P)5 - DH5 - KR5 - KAS6 - AT(A)6 - DH6 - KR6 - KAS7 - AT(P)7 - DH7 - ER7 - KR7 - KAS8 - AT(A)8 - DH8 - KR8 - KAS9 - AT(A)9 - DH9 - KR9 - KAS10 - AT(P)10 - KR10 -
ACP5                       ACP6                       ACP7                       ACP8                       ACP9                       DH10                       ACP10
(inactive?)
RAPS3: KAS11 - AT(A)11 - DH11 - KR11 - KAS12 - AT(A)12 - DH12 - KR12 - KAS13 - AT(P)13 - DH13 - ER13 - KR13 - KAS14 - AT(A)14 -
ACP11                      ACP12                      ACP13                      ACP14
(inactive?)

Module 14, only core domains

Fig. 2 Gene cluster for rapamycin biosynthesis in *S. hygroscopicus*. a) entire cluster; b) expansion of polyketide synthase portion including the pipecolic acid inserting gene; c) expansion of RAPS1, RAPS2 and RAPS3.

Table 19 Domain organisation of RAPS from S. hygroscopicus

| S. hygroscopicus | raps1 | raps2 | raps3(rapC) | rapP |
|------------------|-------|-------|-------------|------|
| RAPS            | 0     | 1     | 2           | 3    | 4   | 5   | 6   | 7   | 8   | 9   | 10  | 11  | 12  | 13  | 14  |      |
| KAS             |       | ✓     | ✓           | ✓    | ✓   | ✓   | ✓   | ✓   | ✓   | ✓   | ✓   | ✓   | ✓   | ✓   | ✓   | PIE |
| AT              | CL    | P     | A           | P    | P   | A   | P   | P   | A   | A   | P   | A   | A   | P   | P   |      |
| DH              |       | ✓     | ☐           | ■    | ✓   | ☐   | ■   | ✓   | ✓   | ✓   | ✓   | ☐   | ☐   | ✓   |      |      |
| ER              |       | ✓     | ■           | ■    | ■   | ■   | ■   | ✓   | ■   | ■   | ■   | ■   | ■   | ✓   |      |      |
| KR              |       | ✓     | ✓           | ■    | ✓   | ✓   | ✓   | ✓   | ✓   | ✓   | ✓   | ✓   | ✓   | ✓   | ✓   |      |
| ACP             |       | ✓     | ✓           | ✓    | ✓   | ✓   | ✓   | ✓   | ✓   | ✓   | ✓   | ✓   | ✓   | ✓   | ✓   |      |

active), but only one ER, in Mod 3. There was no ER, ER0 or ER1, in Mod 1. There was a Hyg TE at the end of Mod 4, with no PIE. Flanking on either side was a series of small 'tailoring' ORF's, some of which had recognisable homologies.

### 11.2.1 Synthase (RAPS)

In 1995, Leadlay *et al.* found three colossal multifunctional polypeptides (and an intervening pipecolic acid inserting enzyme 'PIE') containing all fourteen modules required to assemble the putative PKS product with over seventy catalytic activities (Fig. 2). ${}^{147,149}$ RAPS 2 contained six modules with a predicted $M_{r}$ of 1.07 MDa, whilst RAPS 1 and RAPS 3 contained 'just' four modules, with predicted molecular weights of 900 and 660 kDa respectively (Table 19). In module three, there is a superfluous ER module that may be inactive, or providing 'temporary' reduction necessary for conformational folding etc., that is subsequently reoxidised by post-PKS enzymes. They identified an ER on the loading domain (ER0), which may serve to reduce the 3,4-dihydroxycyclohex-1-enylcarbonyl 325 starter unit to the corresponding cyclohexanyl moiety 326; prior to the ER0 is an ATP dependent carboxylic acid ligase (CL) that may form the acyl adenylate of the starter unit. ${}^{150a}$ It was initially thought that this ligase could directly transfer the acyl adenylate to the thiol of KAS1, in contrast to the usual transthioesterification via an ACP0. However, whilst not highlighted in some of the original papers, a probable ACP0 sequence has now been located and is indicated in Table 19. ${}^{150b}$ Translationally coupled to $rapC$ (RAPS 3) is $rapP$ that has homology with the PIE enzyme from FK 506; there was no conventional TE. The order of the domains is analogous to that found in DEBS and other PKSs. The sequence of each AT showed unambiguous homology with corresponding acetate or propionate incorporating acyltransferases. The 'core' KAS, AT and ACP domains have an extremely high mutual sequence homology, not reinforcing proposals that these domains are primarily involved in recognising and discriminating between growing acyl chains in macrolide PKSs. Except for the final module (Mod 14), which contains only core domains, all modules contain DH and KR domains. The KAS domains of Mod 1 to Mod 13 have extremely high sequence identity (85–90%). Most DH have Hx$_{3}$Gx$_{4}$P as found in DEBSDH 4. Four DH's; DH2, DH5, DH11 and DH12, all have a characteristic five aa deletion in their active site and are presumed inactive. The sequence of DH3 and DH6 appears intact, yet these domains would be expected to be inactive. ER is only present in AT(P) containing modules, all four ERs contain GGVGMAATQIA as their NADPH binding site (required motif Gx$_{2}$Gx$_{2}$Ax$_{3}$A). There is no requirement for a reduction by ER3, but its sequence provides no indication of inactivity. ER0 aligns well with other RapER and other PKSERs, and is believed to be involved in shikimate reduction to provide the starter unit, possibly converting a cyclohex-1-enylcarbonyl moiety to a cyclohexanycarbonyl moiety. Mod 3 and Mod 6 contain apparently superfluous KR's whose sequence provides no evidence of inactivity. Presumably DH3, DH6 and ER3 are also non-functional because their corresponding carbonyl has not been reduced. As found in DEBS Mod 4 (the only DEBS Mod containing a DH), there is a long interdomain region

between DH and the adjacent ER or KR, whose sequence retains homology, and is somewhat resistant to proteolysis, indicating that this region is more than just an uninvolved 'floppy' interdomain loop, and has an as yet unknown important role. Interestingly, the only exception (again) is module fourteen which only contains core domains, whose KAS has significant differences in sequence to KAS1–13, and does not have this interdomain region. Analysis of the N-terminal end of RAPS 2 and RAPS 3 indicates a 32aa region that does not appear to be part of the adjacent KAS (in common with N-terminal ends of DEBS 2, DEBS 3 and OLES 3) and is thought to form amphipathic helices that may be involved in initiating the protein's dimerisation process, or in forming interpeptide interactions, as discussed in Section 2.15 (Part A of this review).

Pipecolic acid inserting enzyme (PIE) was first located and purified as an active dimer by Nielson *et al.* in 1991 from the immunomycin producing strain (var. *ascomyceticus*) of $S.$ hygroscopicus that catalysed both the transfer of the acyl chain to form the new peptide bond, and to catalyse subsequent macrolactonisation, eliminating the need for a separate thioesterase (Scheme 28). ${}^{151}$ Leadlay *et al.* overexpressed PIE from the rapamycin producing strain of $S.$ hygroscopicus in both $E.$ coli and $S.$ coelicolor. ${}^{152}$ The DNA sequence of $rapP$ has conserved sequence motifs with non-ribosomal peptide synthases, including amino acid activation, ATP binding and a core sequence of LGGHS (Ser-1033) for attachment of the phosphopantetheine moiety, a peptide transfer motif (HHX$_{3}$-DGXS) for epimerisation and peptide bond formation, but no motifs similar to that found for TE's, as is found for some NRPS's (Scheme 31). However, the C-terminal 470aa's are very similar to the N-terminal 470aa's, which include three motifs including the peptide transfer motif, and it is thought that it is this second copy involved in macrolactonisation presumably via use of the incoming hydroxy as nucleophile in place of an amine function, as would be the case in peptide bond formation. In contrast to that observed for the ascomycin PIE, the overexpressed Rap PIE was found to purify as a monomer, with little phosphopantetheinylation when expressed in $E.$ coli, and around 75% modified when expressed in $S.$ coelicolor.

---

**Scheme 31 Loading of pipecolate onto RAPS.**

![Chemical Structure](attachment:chemical_structure.png)

Table 20 Domain organisation for FKBS from S. tsukubaensis

| FKBS | $f k b B$ | $f k b C$ | $f k b A$ |
|------|----------|----------|----------|
|      | 0        | 1        | 2        | 3        | 4        | 5        | 6        | 7        | 8        | 9        | 10       |
| KAS  |          | √        | √        | √        | √        |          |          | √        | √        | √        | √        |
| AT   | CL       | P        | P        | A        | N        | P        | P        | A        | A        | P        | A        |
| DH   |          | √        | ■        | □        | ■        | √        | √        | √        | □        | √        |          |
| ER   |          |          |          |          |          | √        | √        | √        | √        | √        |          |
| KR   |          | √        | √        | √        | √        | √        | √        | √        | √        | √        | √        |
| ACP  |          | √        | √        | √        | √        | √        | √        |          |          |          | PIE      |

In summary, these enormous proteins possess a considerable number of superfluous domains. The ‘superfluous’ peptide sequence may still be required for folding/spatial orientation (or other as yet unknown function) of the multifunctional protein, or it may not have yet been evolutionarily ‘lost’ (more ‘time/ evolutionary pressure’ required). The extraordinarily high sequence similarity between the first thirteen KAS domains may indicate that gene duplication to give thirteen domains only occurred relatively recently, and many superfluous domains are yet to be lost, *i.e.* that this is an ‘immature’ system with lots of evolving to do.

### 11.2.2 Post-PKS (tailoring) ${ }^{149}$

Leadlay *et al.* report that there are two P-450 proteins (RapJ: 386aa and RapN: 404aa) presumed to catalyse hydroxylation of C-9 and C-27 of the putative PKS product 323 (Scheme 28), whose sequences contain an oxygen binding site (LLxAG) and heme binding site (GxGxHxCxGxxLxR) with heme binding cysteine. ${ }^{147,149}$ In most biosynthetic gene clusters, the other electron transport components of P-450 oxidation are located elsewhere on the genome, or recruited from other metabolic activity. However, immediately downstream, and translationally coupled to rapN, is rapO coding for a 78aa acidic protein homologous with ferredoxins containing [3Fe-4S] clusters. Presumably the NADH:Fd oxidoreductase is located outside the biosynthetic cluster. In addition, RapK (341aa) has a sequence homologous with pteridine binding proteins, and in particular a pteridine dependent cysteine dioxygenase that forms sulfeno-alanine.

Three genes (rapI, rapM and rapQ) have motifs which show homology to parts of other SAM dependent methylases, but it is not yet known which catalyses which methylation.

RapT (264aa), immediately to one side of the PKS, shows homology with short chain alcohol dehydrogenases, with a NADP binding site (Gx,GxGx,Ax,A) and with Tyr and Lys catalytic residues, and may be involved with synthesis of the DHCHC moiety.

Most of the other flanking genes (Fig. 2) are thought to be involved in regulation or export, however their uncertain and unproven role in Rap biosynthesis leads to some of them usually being labelled e.g. 'orfG' instead of 'rapG'.

Whilst the PKS genes have a high (90%) G+C content in the third codon position, the flanking genes are lower (80%), almost entirely due to the use of T instead of C in 'wobble' positions (where using T instead of C does not change the coded aa). This phenomenon has been observed in the oleandomycin cluster, but does not occur in the Ery cluster.

---

### 11.3 FK506

Tanaka *et al.* reported the structure of FK506 328 from S. tsukubaensis in 1987. ${ }^{153}$ Whilst the starter unit, the final four PK extension units and termination unit are the same, FK506 PKS (FKBS) needs four less modules than RAPS, one of which loads a very unusual propylmalonate unit, lacks the triene section, and KR2 has the opposite stereochemistry, and the resultant hydroxy group is the macrolactonisation alcohol (Scheme 29).

---

### 11.3.1 Synthase (FKBS)

Motamedi *et al.* reported sequencing a 40 kbp segment of DNA from the FK506 producer S. spp. MA 6548. ${ }^{154,155}$ FkbP encodes 1442aa, 154792 kDa, homologous to parts of RapP PIE. FkbA encodes 6420aa, 630660 Da, with 19 domains. Disruption prevents FK506 biosynthesis. FkbAT(A) 7, AT(A) 8, AT(P) 9, AT(A) 10 correlates with last four modules, Mod 7–10. FkbB encodes 790129 kDa, 7540aa, 21 domains in five modules including the loading module containing the 3, 4-dihydroxycyclohexenyl CoA synthetase, followed by an ER0 (or dihydroxycyclohex-1-enyl CoA reductase), as for RAPS. The authors noted a 68% homology of their FkbACP0 with the analogous region in the RAPS sequence. The apparently intact DH2 is superfluous, and DH3 is inactive due to a deletion. The superfluous KR4 has a 75aa deletion including the nucleotide binding motif, and is thus presumed inactive. The superfluous DH4 may be functionally active, but would lack an alcohol substrate. The AT's correlate with their expected substrate (A) or (P); however the pentanoate (propyl-malonate) loading AT4 carries a propionate specific motif. Presumably other parts of the AT4 sequence, coupled with substrate availability, determine its longer chain specificity. FkbC has 374438Da, 3592aa.

---

### 11.3.2 Post-PKS (tailoring)

FkbL shows homology to RapL lysine deaminase and is presumably involved in ring cyclisation to form pipecolate; and FkbO, D and M are homologues of RapK, J, and I respectively. In contrast to RAPS, and other known PKS clusters, there is a small oxidative gene fkbO in the genome wedged in between the pipecolate inserting gene and fkbB, which is homologous to the pteridine dependent dioxygenase RapK, believed to oxidise the RapC-9 hydroxy group to the corresponding α-dicarbonyl, which would also be required in the FK506 pathway (Scheme 28). The C-21 propyl side chain in 327, which is derived from saturated propylmalonate, needs oxidising and dehydration to form the propen-2-y1 side chain in 328. Shafiee *et al.* have characterised the C-31 O-MeT FkbM and the C-9 hydroxylase. ${ }^{155b,155c}$

---

### 11.4 Kendomycin—a carbomacrolide (chalcone synthase starter unit?)

Bode and Zeeck have investigated the biosynthesis of the kendomycin 338 from S. violaceoruber using ${ }^{13}$C-labelled acetate propionate and methionine. ${ }^{156,157}$ Kendomycin is unusual in that the large ring comprises only carbons, necessitating an unusual termination step. The ansa bridge was labelled by acetate and propionate as expected (Scheme 32), but the chromophore showed two patterns of labelling from dilabelled acetate indicating a symmetrical intermediate, and the methyl was labelled only by methionine. They initially proposed either a 3, 5-dihydroxybenzoic acid thioester starter unit 335 with much later modification, or a 4-methyl-2, 3, 5, 6-tetrahydroxy-benzoyl CoA 336 starter unit, possibly assembled from a chalcone synthase (CHS). The ‘starter unit’ 336 would then be extended by eight polyketide Type I modules as usual. The authors propose that the pyran ring is formed by reductive

Nat. Prod. Rep., 2001, 18, 231–281

![image description](attachment:chemical-reaction-diagram.png)

**Scheme 32 Biosynthesis of kendomycin.**

cyclisation between the carbonyls on C-5 and C-9, or between a hydroxy and a carbonyl carbon. Presumably, if the C-9 carbonyl oxygen attacks the C-5 carbon, then the resulting (protonated) hemiacetal can lose water assisted by the neighboring enol (as well as the other ether-like oxygen), followed by hydride attack in an elimination-addition reaction. The authors propose a new mode of 'termination'; an aldol condensation with decarboxylation and dehydration, which gives the observed labelling pattern and a carbocyclic ring.

In a recent paper, Bode and Zeeck found that malonate only labelled acetate (not propionate) positions, even though previous results showed that acetate was incorporated into the methylmalonate positions. Direct incorporation of acetate into MMCoA can occur via a thiolase mediated formation of 3-oxobutanoyl CoA, reduction to butanoyl CoA, and either oxidation to succinyl CoA followed by methylmalonyl CoA mutase rearrangement, or first 2-methylpropanoyl CoA mutase mediated rearrangement to 2-methylpropanoyl CoA and oxidation to methylmalonyl CoA (Scheme 33). Either route gives the labelling pattern seen in 339, along with some 340 from an exchange of CoA thioesters through free succinic diacid. Incorporation of acetate via citric acid cycle would give the labelling pattern seen in 341. Feeding of [1,4-<sup>13</sup>C<sub>2</sub>]succinate gave the same labelling pattern as previously seen for feeding [1-<sup>13</sup>C]propionate. Feeding [2,4-<sup>13</sup>C]butanoate gave an identical labelling pattern to that obtained from feeding [2-<sup>13</sup>C]acetate, so all the dilabelled butanoate had been oxidised back to acetyl CoA (rather than converted to butanoyl CoA). The authors perform a detailed analysis of the labelling (from dilabelled acetate) to estimate the contribution from each of the three isotopomers 339, 340 and 341, as 4:3:2, so the overall contribution of butanoyl CoA pathway (339 and 340) derived methylmalonyl CoA being 3.5 times as great as that derived from the citric acid pathway that gives 341.

The authors add the P-450 inhibitor, ancymidol, and see a drastic reduction in the production of kendomycin 338, but no other compounds produced. This suggested that the P-450 oxidation might occur early in the pathway, even on 335, preventing formation of 2,3,5,6-tetrahydroxy-4-methylbenzoyl

**268 Nat. Prod. Rep., 2001, 18, 231–281**

335 might be derived from 331 rather than the chalcone synthase pathway via 333 and 334 (Scheme 32).

Feeding benzoic acid or 3,5-dihydroxybenzoic acid led to similar levels of kendomycin production. Feeding of a variety of NAC thioesters all led to the production of 20-(2-acetyl-aminosulfanyl)kendomycin, and addition of 2-sulfanylpypyridine gave 20-(pyridin-2-yl)sulfanylkendomycin, presumably in a Michael like addition of the thiol to C-20. The facile reaction with thiols normally present in the cell raises the question of why this is not happening *in vivo*.

### 12 Decalin and cyclopropyl containing polyketides

#### 12.1 Decalins and related compounds—assembly using a Diels–Alder reaction

A series of polyketides are characterised by a *cis*-fused bicyclo[4.4.0]dec-2-ene moiety which is thought to be assembled through a Diels–Alder reaction, and are found in both fungi (e.g. lovastatin²) and bacteria. Oikawa *et al.* have demonstrated that an enzymatic Diels–Alder reaction is involved in the biosynthesis of the fungal polyketide metabolite solanopyrone,¹⁵⁸,¹⁵⁹ and Vederas and Reeves have investigated the role of a putative Diels–Alderase in lovastatin biosynthesis.²,¹⁶⁰

##### 12.1.1 Nargenicin and nodusmicin

Nargenicin 342 was first isolated from *Nocardia argentinensis*¹⁶¹ and is the pyrrole-2-carboxylate ester of nodusmicin 343 previously isolated from *Saccharopolyspora hirsuta*,¹⁶² which also produces 18-deoxynargenicin 344. Cane and Yang have shown that nargenicin 342 is assembled from five acetate and four propionate building blocks in the expected manner,¹⁶³ as Snyder and Rhinehart have shown for nodusmicin (Scheme 34).¹⁶⁴ Cane observed incorporation of acetate and [1-¹³C]propionate into the pyrrole-2-carboxylate moiety, but at lower levels (2%) than in the nargenicin nucleus (5–7%) and was unable to detect label in the pyrrole from [2-¹³C]propionate. Cane suggests that the pyrrole-2-carboxylate moiety is formed by the conversion of propionate to succinate, then to 2-oxoglutarate and then via dihydroproline. Cane and Yang have also demonstrated the acetate, propionate or dioxygen origin of each oxygen atom in nargenicin.¹⁶³,¹⁶⁵ These observations ruled out earlier theories involving epoxy-olefin cyclisations, and pointed to the involvement of an intramolecular Diels–Alder cyclisation of a triene intermediate such as 350. Cane *et al.* then demonstrated intact incorporation of labelled diketides 345 and 346,¹⁶⁶ triketide 347, tetraketide 348,¹⁶⁷ and most recently, (2R,3S,4E,6E,8R,9R)-3,9-dihydroxy-2,6,8-trimethylundeca-4,6-dienoic acid (pentaketide) 349¹⁶⁸ intermediates as NAC thioesters into the nargenicin nucleus, each resulting in specific incorporation of label. The latter feeding experiment included the fatty acid oxidation inhibitors pent-4-yonic acid and 4-thiaoctadecanoic acid,¹⁶⁹,¹⁷⁰ as well as the carrier 2,6-*O*-dimethylcyclodextrin,¹⁷¹,¹⁷² and represented the successful intact incorporation of the most advanced and complex polyketide chain elongation intermediate to that time, and as the incorporated intermediates possessed the conjugated (*E,E*)-diene required for the intramolecular Diels–Alder reaction, was strongly suggestive of this mechanism. Polyketide extension to 350 followed by an intramolecular Diels–Alder reaction and macrolactonisation would give 351, which after slight oxidative modification would give either nargenicin 342 or nodusmicin 343.

Roush *et al.* have examined the *in vitro* Diels–Alder reaction in their studies on nargenicin synthesis. The protected open chain ‘precursor’ 354 underwent Diels–Alder cyclisation to give a mixture of unwanted stereoisomer 356 (56% yield) and desired *cis*-fused stereoisomer 355 (27%) (Scheme 35). However, if that precursor was lactonised to 352 before the Diels–Alder cyclisation, then only the desired stereoisomer 353, in 86% yield, was obtained. This is indicative that macrolactonisation may occur prior to the Diels–Alder reaction during the biosynthesis, and thus is not part of the ‘synthase’ as a growing acyl chain stabilisation reaction.¹⁷³

##### 12.1.2 Coloradocin

McAlpine *et al.* have shown that labelled acetate propionate and succinate are incorporated into coloradocin 357 in *Actino-planes coloradoensis*.¹⁷⁴ The ‘nargenicin-like’ portion of the molecule is assembled from acetate and propionate as expected, however, propionate did not label the remaining eleven carbon unit, and acetate only indirectly. Succinate efficiently labelled C-26, 27 and 29, at far higher levels than the methylmalonate derived carbons in the ‘nargenicin-like’ portion. However, no significant labelling of C-29 to C-32 was observed from labelled succinate. Attempts to incorporate labelled ¹³C-labelled oxaloacetate or glutamate into C-29 to C-32 failed. The authors suggest that a hyperactive tricarboxylic acid cycle contributes to the eleven carbon atom portion of the molecule, with succinate for C-26 to C-28 and as yet undefined precursors for C-29 to C-32.

![Image Description](image.png)

**Scheme 34 Biosynthesis of nargenicin.**

**12.1.3 Cochleamycin**

Shindo *et al.* have investigated the biosynthesis of the cochleamycins 360–363 (Scheme 36), which are antitumour antibiotics produced by *Streptomyces* sp. DT136, and found them assembled from eight acetate and one propionate unit with no incorporation into the 2-methylpropionate moiety—assumed to be derived from L-valine. The authors propose formation of the lactone, and oxidation of the starter methyl to an aldehyde 358, followed by a Diels–Alder cyclisation to 359, followed by an aldol to form the oxabicyclo[6.2.2]-dodecenedione moiety. Reduction of the C-4–C-5 double bond (or Michael hydride attack at C-5), followed by attack of a carbanion on C-4 to a leaving group on C-16, would give the cochleamycin B series 362 and 363.

**12.2 Cyclizidine—a cyclopropyl polyketide**

Leeper *et al.* have investigated the biosynthesis of the cyclopropyl containing indolizidine alkaloid cyclizidine 364 by

**Nat. Prod. Rep., 2001, 18, 231–281**

Coloradocin 357

Unknown origin 357

Scheme 36 Biosynthesis of the cochleamycins.

Streptomyces sp. NCIB 11649 using labelled acetate (Scheme 37). This is the only example so far of a monosubstituted cyclopropyl ring not of terpene origin. Labelled acetate and propionate were incorporated in the expected positions as illustrated, with all three carbon atoms of the cyclopropyl ring originating from a single propionate unit, though unusually all the double bonds are intraunit rather than interunit as is normal. The C-2 hydroxy group was acetate labelled, ruling out ring closure onto a dioxygen derived C-1–C-2 epoxide. Feeding [1-¹³C, 3-²H₂]propionate showed three deuterium atoms on both methyl groups, and two on one of the (C-16, the pro-S) cyclopropylmethyls. However, whilst ¹³C-labelled propionate was efficiently incorporated into the cyclopropyl ring, CD₃-labelled propionate was much less efficiently incorporated (Scheme 37). This could only occur due to an isotope effect in derivatisation of the propionate prior to incorporation. Feeding [2-²H₂]-propionate led to stereospecific incorporation of the pro-R hydrogen of the C-15 pro-(R) methylene unit, though much (80%) label was lost. Feeding (2S)-MeCHDCOO⁻, but not (2R), led to similar labelling. This means that the single deuterium in (2S)-[2-²H₁]propionate undergoes an overall inversion of configuration during the incorporation process. To explain the high loss of label, Leeper proposes that rapid and reversible carboxylation of propionate gives rise to stereospecific loss of the pro-(R) hydrogen (deuterium), with partial loss of the remaining pro-(S) deuterium through some reversible epimerisation (methylmalonyl CoA epimerase). To account for the inversion of configuration, Leeper suggests dehydrogenation of propionyl CoA with loss of the pro-(R) hydrogen to 365, followed by addition of HX (the new H adding onto the pro-(S) position) to give 366, where X is a leaving group that could participate in both cyclopropyl and indolizidine ring formation (Scheme 38). Nucleophilic attack by the nitrogen in 367, with double bond migrations and cyclopropyl ring closure of 367, gives a precursor of 364. Propionyl CoA dehydrogenases have been reported in the interconversion of propionate and lactate, and related enzymes are required in fatty acid oxidation.

13 Myxobacteria

The myxobacteria (order: Myxobacterales) are aerobic unicellular Gram-negative rod-like soil bacteria that are frequently found in soil, dead plant material or in animal dung, often

Scheme 37 Biosynthesis of cyclizidine.

Scheme 38 Formation of cyclopropyl ring in cyclizidine.

utilise cellulose and its degradation products, and are mobile through an ability to glide, have complex social interactions, multicellular cytodifferentiation, and produce elaborate, often brightly coloured, fruiting bodies and myxospores. They are one group (δ) of the proteobacteria (purple bacteria), and genera include the *Myxococcus*, *Polyangium*, *Stigmatella*, and *Sorangium*. Like the actinomycetes, they have a high G+C DNA base pair content.

### 13.1 Epothilones<sup>181,182</sup>

The epothilones (epothilone A 370 and epothilone B 371 Scheme 39) stabilise microtubule arrays by binding to the paclitaxel-binding site on β-tubulin, inhibiting microtubule depolymerisation. Höfle *et al.* first isolated epothilones in 1987,<sup>182</sup> but their enormous cytotoxic paclitaxel (Taxol®) like potential was not realised until the mid-nineties. Two papers from different groups reporting the cluster appeared at virtually the same time in early 2000: unfortunately, the two groups have labelled the genes and modules in two different ways. For example, the Novartis group label the NRPS as EPOSB, whilst the KOSAN group label it EpoB, and module numbers do not correspond; the KOSAN laboratories label the NRPS as the first module, so all others are 'one higher' than in the listed Novartis labelling system.

---

**Scheme 39** Biosynthesis of epothilones. The origin of the R = Me methyl carbon (Propionate or SAM) is not yet reported.

![Chemical Structure](https://i.imgur.com/structure1.png)

---

**Scheme 40** Formation of thiazole starter unit.

![Chemical Reaction Scheme](https://i.imgur.com/reaction_scheme.png)

in the final metabolites. Julien *et al.* report a DH in the first module of EpoD (KOSAN Mod3/Novartis Mod2) not listed in the Novartis paper, and not needed in the final putative structure, so listed in the table as inactive. In contrast, both the papers do not report a 'DH3' needed to insert a double bond between C-12–13—presumably a post-PKS or non-enzymatic dehydration occurs subsequently. Perhaps the module sequence does not correspond to function sequence and Mod2/Mod3 are reversed?

The EPOSA KAS contains the two characteristic histidines of KAS domains, but the KAS active site cysteine has been replaced by a tyrosine, shown as KAS(Y), rather than the usual glutamine residue as found in other KAS(Q)'s, or the serine containing KAS, KAS(S) found in pimaricin synthase, and it is not known whether either KAS(S) or KAS(Y) can still catalyse the decarboxylation of malonyl ACP. The ER0 may not be functional because a highly conserved arginine in the GGVGxAAxQxAR motif is replaced by a glutamine Q. A possible alternative role is in the oxidation of the thiazoline ring to the thiazole ring (Scheme 40). EPOSP contains a modified condensation domain found in other thiazole or oxazole containing metabolites, and is labelled the HC or heterocyclisation domain. Adenylate binding domains are specific for one amino acid adenylate. A predictive model of AD substrate specificity has been proposed,<sup>185</sup> the amino acid sequence codes specifying for ten different substrate amino acid adenylates has now been solved<sup>186</sup> and EPOSP ‘AD’ domain is predicted to bind cysteine as expected. In addition, EPOSP contains a domain not reported in other NRPS sequences that has homology with some oxidoreductases, and presumably oxidises the initially formed thiazoline ring to the thiazole (Scheme 40). Thus a condensation reaction between the EPOSP aminoacyl-S-PCP and the EPOSA acetyl-S-ACP, followed by heterocyclisation domain catalysed attack on the acetyl carbonyl by the free cysteine thiol with dehydration followed by oxidation (OX) results in the (2-methylthiazol-4-yl)carbonyl-S-PCP moiety for further acyl chain extension by the eight PKS modules. In the bleomycin synthase, the oxidation to form a thiazole is thought to be carried out by a discrete oxidase elsewhere in the cluster.<sup>187</sup> A further possibility is that the otherwise superfluous ER0 is able to catalyse this oxidation (rather than the ‘normal’ enoyl reduction).

Molnár *et al.* at Novartis have sequenced 68 750 bp DNA from the gliding myxobacterium *Sorangium cellulosum* So ce90 that produces the epothilones and found a cluster of 22 ORFs including nine modules of polyketide synthase and one non-ribosomal peptide synthase.<sup>183</sup> One PKS module (EPOSA) and the NRPS are thought to assemble the 2-methylthiazole-4-carboxylic acid thioester starting unit from an acetate and cysteine through condensation, cyclodehydration and dehydrogenation (Scheme 40), which is then chain extended by the eight collinearly arranged PKS modules to 368 or 369, then oxidised by a P-450 to 370 or 371, and then transported out of the cell. No studies with labelled precursors have yet been reported, and the nature of the synthase product is uncertain. EPOSA has 1421 aa; EPOSB has 1410 aa; EPOSC has 7257 aa; EPOSD has 3798 aa; EPOSE has 2439 aa; EPOSF has 419 aa and the remaining Orfs are all small discrete proteins.

Julien *et al.* of the KOSAN laboratories sequenced 56 kbp of the cluster and expressed the genes in CH999 to produce epothilone A 370 and B 371, but the authors do not report in what yield.<sup>184</sup> EpoK (EPOSF) was overexpressed in *E. coli* and shown to epoxidise a double bond at C-12–13 into the epoxide

272 Nat. Prod. Rep., 2001, 18, 231–281

Table 21 Domain organisation for epothilone biosynthesis in Sorangium cellulosum So ce 90 (HC is heterocyclisation domain, Y is KAS(Y), PCP is peptidyl carrier protein)

| KOSAN | EpoA | EpoB | EpoC | EpoD | EpoE | EpoF | EpoK |
|-------|------|------|------|------|------|------|------|
| NOVARTIS EPOSA | EPOSP | EPOSB | EPOSC | EPOSD | EPOSE | EPOSF | Novartis Mod# |
| EPOS | NRPS | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 |
| KAS | Y□ | HC | √ | √ | √ | √ | √ | √ | √ |
| AT | A | ADcys | P | A | A/P | A | P | P | P | A |
| DH | Ox | √ | √ | √ | √ | MeT | □ |
| Misc. | | | | | | | | | √ | TEI |
| ER | | | | | | | | | P450 |
| KR | | | | | | | | | |
| ACP | | | | | | | | | |

Table 22 Domain organisation of SORS in Sorangium cellulosum

| Sorangium cellulosum | sorA | sorB |
|-----------------------|------|------|
| SORS | -1 | 0 | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 |
| KAS | √ | √ | √ | √ | √ | √ | √ | √ | √ | √ |
| AT | Bz | A | A | N | A | P | P | N | P |
| DH | √ | √ | √ | √ | √ | √ | √ | √ | □ |
| ER | √ | √ | √ | √ | √ | √ | √ | √ | □ |
| KR | √ | √ | √ | √ | √ | √ | √ | √ | √ | TE1 |
| ACP | | | | | | | | | | |

EPOS (B to E) KAS domains are well conserved between themselves and with other KAS, though KAS1 that catalyses a condensation with the unusual thiazole starter unit shows the greatest divergence. AT 0, 2, 3, 4 and 8 are predicted from their sequence to be malonate loading, whilst AT 1, 5, 6 and 7 are expected to load methylmalonate. Thus module three is expected to load a malonate, necessitating a post-PKS methylation at C-12, unless that AT was promiscuous as indicated in Table 21. KR7 is predicted to be inactive as the product has a carbonyl at C-5, and a signature motif sequence has an arginine replacing a serine that is conserved in all the other polyketide KRs so far sequenced.

Mod1, 4, 5, 7 and 8 all contain an DH domain, all with the active site motif LxxHx₃Gx₄P. Hutchinson et al. have suggested that the presence of a phenylalanine (F) just before the proline (P) renders the DH domain inactive. However, the presumed functionally active DH1 and 5 (and DH8) all contain this 'F'. Epothilones contain a hydroxy group at C-3; unless the PKS product is a double bond subsequently rehydrated, DH8 should be presumed inactive. There is no reported DH3, so the authors suggest a PKS product with a hydroxy group at C-13. Unless a 'DH3' can be located (or Mod2/Mod3 order switched), this hydroxy has to be dehydrated by a discrete dehydratase to give the corresponding double bond ready for P-450 epoxidation. Epothilone C and D have a C-12-C-13 double bond.

ER8 is presumed inactive due to substitutions in the NAD(P)H binding site motif. A putative TE1 domain with a GxSxD motif has been identified at the carboxy terminus of EPOSE.

Module 7 contains a SAM binding domain that has homology with the MeT's recently located in other metabolites. The authors propose that this MeT domain methylates at C-4 to produce the gem-dimethyl moiety. An interesting possibility is that KR7 is inactive, leaving a 2-methyl-3-oxoacyl chain; the 'active' DH7 has no 3-hydroxy moiety to dehydrate, but its basic residue still removes the only C-2 proton, as normal. The resulting enolate then nucleophilically attacks the SAM bound in the MeT7 SAM binding site, resulting in the gem-dimethyl functionality observed (Scheme 41).

EPOS F codes for a 420 aa P-450 protein that may be involved in the epoxidation of the C-12-C-13 double bond. Some of the remaining ORFs show homology to transport/export proteins but many ORFs in the cluster show no homology with databank sequences.

**Scheme 41 Use of DH base in SAM methylation.**

13.2 Soraphens

Soraphen A 372, produced by Sorangium cellulosum So ce 26, is a potent inhibitor of fungal acetyl CoA carboxylase, with potential to control plant fungal pathogens, that is assembled from a phenylalanine derived benzoyl CoA starter unit, three acetate, three propionate and two glycerol derived hydroxy-acetate (glycolate) units—presumably incorporated as hydroxymalonyl CoAs, after the action of a 'glycerol oxidase' to convert glycerol to hydroxymalonyl CoA, but this has not yet been located. Hill et al. have shown that the pro-(S) hydroxymethyl moiety of glycerol is lost during its incorporation. The co-metabolite soraphen B is partially O-methylated, and soraphen C is non-O-methylated, so presumably these C-2 units are either incorporated as hydroxyacetate units and subsequently methylated; or the corresponding AT (glycolate)s, are promiscuous, loading either hydroxymalonyl CoA or methoxymalonyl CoA.

Schupp et al. published the first sequence information of a myxobacteriales derived PKS, then subsequently published in a patent the domains found in soraphen synthase SORS.

extended by three acetate units, then one leucine, terminating with one more acetate unit that is decarboxylated. Presumably the starter unit (urocanoyl CoA) is formed from histidine by a 'histidine ammonia lyase' (urocanase—an enzyme involved in histidine metabolism) analogous to the 'phenylalanine ammonia lyase (PAL) used to convert phenylalanine into cinnamoyl CoA.

### 13.4 TA (myxovirescin)

The myxomycete *Myxococcus xanthus*, when under nutrient stress, can produce an antibiotic TA ${}^{194}$ that inhibits bacterial cell-wall synthesis. TA may be identical to, or a stereoisomer of the previously reported myxovirescin, 374. ${}^{195,196}$ Whilst labelling studies are yet to be reported, eleven polyketide extender units would be expected to be added to a $N$-(2-hydroxypentanoyl)-glycyl starter unit. An attractive alternative to that discussed in the literature would be an unusual 2-hydroxypentanoate starter followed by one addition of glycine via a NRPS module, followed by eleven acyl units. The C-17 methoxymethyl appendage and C-13 ethyl appendage are problematic. These appendages are attached to 'carbonyl carbons' and are presumably added post-PKS by an as yet unknown process, possibly similar to that observed for methyl appendages in some marine polyketides (acetate methyl carbon nucleophilically attacks carbonyl carbon, followed by decarboxylation). The C-2 and C-4 methyls may be propionate derived or SAM derived. The C-19–C-20 may be acetate or glycol/glycerol derived, though they do report the presence of a gene for P-450 hydroxylation (one may instead be required for pentanoate to 2-hydroxypentanoate). Rosenberg et al. mapped the genes involved in TA production ${}^{197}$ within 36 kbp and have recently described part of the synthase. ${}^{198a}$ Cloning and sequencing of 7.2 kbp fragment (7178 bp, G + C = 67%) shows that it encodes a single ORF (2392 aa) labelled Tal with two regions; the first 1020 aa shows similarity to peptide syntheses, and the second region shows homology to polyketide syntheses with a KAS1/KR1/ACP1/KAS2, with good sequence homology to soraphen and Ery proteins. Thus a peptide synthase and polyketide synthase may be residing on the same ORF.

---

modules, and is shown as ACP0-KAS0-AT0-AT1-KR1-ACP1 etc. This is a very unusual order, ACP0 (surely ACP(−1)?) before AT0, with the lack of KAS1. Perhaps the KAS located within the loading domain, labelled KAS0, is or was in effect 'KAS1'. SorB is ca. 26 451 bp in length and its coding sequence just overlaps that for sorA, suggesting that they are part of the same transcriptional unit (along with sorR), which contains five synthase modules, terminating in protein homologous to EryA TE.

Immediately upstream of sorA is sorR, 378 bp coding for a discrete protein homologous to the reductase domains found in EryA (involved in benzoyl CoA biosynthesis?). Immediately downstream of sorB is sorM, 1044 bp in length coding for a protein homologous to the O-MeT found in the assembly of rapamycin (RapQ?).

### 13.3 Chrysophysarin A

The myxomycete, *Physarum polycephalum* is an intensely yellow-coloured slime mould that moves and sporulates in response to light, and produces physarochrom A, physarorubinic acids, polycephalins and the yellow pigment chrysophysarin A 373 which was extracted and purified in the dark. Eisenbarth and Steffan have elucidated the structure of chrysophysarin A, and shown that it is assembled from L-leucine, acetate and histidine. ${}^{193}$ A histidine derived starter unit is

---

Nat. Prod. Rep., 2001, 18, 231–281

13.5 Myxalamid

In a recent paper, Müller et al. report the sequencing of 50 kbp cloning and analysis of the gene cluster for the biosynthesis of myxalamid in *Stigmatella aurantiaca* Sga 15. The myxalamids are effective inhibitors of electron transport (NADH:ubiquinone oxidoreductase) in the respiratory chain.

The order of domains in the starter modules is unusual. The 2-methylpropanoate loading AT, AT0, usually located first in the sequence, is after ACP and KAS0, presumably the three-dimensional arrangement of the domains is altered to accommodate this. The result is two AT domains adjacent to each other. The AT(A) 4 in MxaC does not display a malonate-specific consensus sequence. There are large spacer domains of more than 300 aa between AT and KR domains in each module, and due to their 20% homology with ER domains have been inserted into the table as inactive ER (Mod1 even has a LxLxSGVGxLxL homologous to ER active site LxHx₃-GGVGxAx₃A). MxaE encodes a superfluous DH2, whilst MxaC Mod5 lacks a necessary DH5—perhaps dehydration occurs non-enzymatically or post PKS. The most unusual feature is Mod7 which most definitely occurs on two different proteins MxaB1 and MxaB2. The two proteins must interact, because MxaB1 is missing 'its' ACP. In addition, the relatively hydrophobic amino-terminal region at the end of MxaB1 resembles the interpolyptide regions recently described in DEBS. The final protein MxaA is a single module NRPS. An alanine activating domain (ADala) adenylates alanine which is condensed onto the PK chain by the condensing domain C. The terminal domain is, not a TE, but a NAD(P) H-dependent reductase domain (homologous to yeast Lys2) that may reduce the peptide carrier protein (PCP) linked thioester to the free aldehyde, which would require further reduction to the product alcohol.

13.6 Myxothiazol

Müller et al. have sequenced the gene cluster responsible for the biosynthesis of myxothiazol in *Stigmatella aurantiaca* from 3-methylbutanoate, three acetates, two propionates and two cysteines.¹⁹⁸c

The authors found *mtaA* adjacent to two developmental genes *fbfA* and *fbfB*, that was homologous to *E. coli* ACP synthetase.

Table 23 Gene cluster for myxalamid biosynthesis in *Stigmatella aurantiaca* Sga 15

| Gene   | size     | Putative function                     |
|--------|----------|---------------------------------------|
| Orf4   | ←        | Transcriptional regulator            |
| Orf3   | →        | Acetyltransferase                    |
| Mxaf   | ← 2368aa | Mod(−1)/Mod0/Mod1                    |
| Mxae   | ← 1862aa | Mod2                                 |
| Mxad   | ← 1840aa | Mod3                                 |
| Mxac   | ← 3291aa | Mod4/Mod5/Mod6                       |
| Mxab1  | ← 939aa  | Mod7a                                |
| Mxab2  | ← 1233aa | Mod7b                                |
| Mxaa   | ← 1515aa | Mod8                                 |
| Orf2   | →        | Histidine ammonia lyase              |
| Orf1   | ←        | Aspartokinase                        |

Table 24 Domain organization of myxalamid synthase in *Stigmatella aurantiaca*

| MXAS | Mxaf | Mxae | Mxad | Mxac | Mxab1 Mxab2 | Mxaa (NRPS) |
|------|------|------|------|------|--------------|--------------|
|      | -1   | 0    | 1    | 2    | 3            | 4            | 5            | 6            | 7a           | 7b           | 8             |
| KAS  | ✓    | Bu¹  | P    | ✓    | P            | ✓            | A            | ✓            | ✓            | ✓            | C             |
| AT   |      |      | ✓    |      | ✓            | ✓            | A            | ✓            | ✓            | ✓            | ADala         |
| DH   |      |      |      |      |      |      | ✓            | ✓            | ✓            | ✓            | PCP          |
| ER   |      |      |      |      |      |      | ✓            | ✓            | ✓            | ✓            | [H]          |
| KR   |      |      |      |      |      |      | ✓            | ✓            | ✓            | ✓            |              |
| ACP  | ✓    |      | ✓    | ✓    | ✓            | ✓            | ✓            | ✓            | ✓            | ✓            |              |


The chemical structures and additional text in the right column are not included in the markdown output as requested.
Table 25 Gene cluster for myothiazole biosynthesis in *Stigmatella aurantiaca* DW 4/3-1

| Gene   | Putative function                                                                 |
|--------|----------------------------------------------------------------------------------|
| MtaA   | Phosphopantetheine transferase, homologous to AcpS (ACP synthase in *E. coli*)      |
| MtaB   | Mod0/Mod1/Mod2                                                                   |
| MtaC   | NRPS                                                                             |
| MtaD   | NRPS+PKS                                                                         |
| MtaE   |                                                                                  |
| MtaF   |                                                                                  |
| mtaG   |                                                                                  |

terminal amides are found in many peptide hormones, formed from oxidative cleavage of glycine extended precursors, but no homology could be detected between the monooxidation domain of MtaG and these enzymes.

### 14 Pseudomonas

The Pseudomonas are also heterotrophic proteobacteria, the fluorescent Pseudomonas belonging to the γ-grouping.

#### 14.1 Pyoluteorin (Plt)

Van Pee and Ligons have recently reviewed the biosynthesis of pyrrolnitrin<sup>199</sup> and other phenylpyrrole derivatives (including pyoluteorin) by bacteria.<sup>200</sup>

Pyoluteorin 381, produced by the soil bacterium *Pseudomonas fluorescens* Pf-5 is a proline derived chlorinated pyrrole containing antifungal aromatic polyketide metabolite which inhibits the growth of the phytopathogenic oomycete fungus *Pythium ultimum*. The biosynthetic gene cluster contains two genes that have characteristics of both fungal and bacterial Type I synthases (Table 27) but does not contain a loading domain, and AT2 has little sequence homology with Type I ATs apart from the actual active site; the most significant homology being with EryA.<sup>201,202</sup>

PltB is 7.4 kbp and encodes 2458 aa. Module 1 (1013 aa) contains only the core domains KAS1/AT1/ACP1. Module 2 (1424 aa) contains in addition a partial KR2 domain lacking the NADPH binding region. AT2 has several anomalies in its sequence not found in other polyketide ATs; a normal AT serine active site has PxGHSGx3A, but in AT2, the proline has been replaced by an alanine, and the histidine by a phenylalanine; also there have been two sequence insertions, and some other well-conserved residues are missing. This has to pose the question as to whether an AT2(A) is necessary or functional in this system (the ACP may be self-loading), and related Type I systems assembling acetate derived aromatic like polyketides. The ACP's do not conform either: most PKS ACP's have an LGxDS sequence motif, all three Plt ACPs have the G and S, but in ACP1 the motif is LGxSS, in ACP3 it is MGxSS and in ACP3 it is YGxSS, with limited flanking similarities. Again this raises the question of the functionality, and despite appearances whether or not the assembly is fully Type I, or whether some proteins could be reused?

PltC is 5.3 kbp and encodes 1774 or 1786 aa, depending on actual starting site, has a functional KR3, and a non-functional

Table 26 Domain organization of myothiazole synthase in *Stigmatella aurantiaca* DW 4/3-1

| MTAS | MtaB       | MtaC        | MtaD        | MtaE        | MtaF        | MtaG        |
|------|------------|-------------|-------------|-------------|-------------|-------------|
|      | -1         | 0           | 1           | 2           | 3           | 4           | 5           | 6           | 7           | 8           |
| KAS  |            |             |             |             |             |             |             |             |             |             |
| AT   | √          | N           | A           | P           | HC          | ADcys       | HC          | ADcys       | P           | C           |
| DH   |            | √           | √           |             |             | √           |             | √           | O-MeT       | AD          |
| Misc |            |             |             |             |             |             |             |             |             | Ox          |
| ER   |            | □           |             |             |             |             |             |             | √           | AD          |
| KR   |            | √           | √           |             |             |             |             |             | √           | PCP         |
| ACP  | √          | √           | √           |             |             |             |             |             | √           | TE1         |

Table 27 Domain organisation of PLTS in *Pseudomonas fluorescens* Pf-5

| PLTS | PltB     | PltC    |
|------|----------|---------|
| KAS  | √        | √       |
| AT   | A        | A       |
| DH   |          | □       |
| ER   |          |         |
| KR   | □        | √       |
| ACP  | √        | √       |

putative DH3 domain. As dehydration results in aromaticity, this may occur non-enzymatically before or after release from the synthase, or be so facile that it only needs catalysis by an otherwise dysfunctional looking DH protein. There was no evidence of an integrated TE, but there may be a downstream discrete TE (see PltG below). There are long interdomain linkers between KAS2 and AT2, and between DH3 and KR3. A gene has been identified within the cluster that may encode a peptide synthetase, which may aid in transferring the starter unit onto the PKS. The lack of a loading domain, and the unusual (dysfunctional?) AT2 sequence in PLTS could be explained if ACP1 and ACP2 domains are self-loading, as has been observed for some Type II ACPs.<sup>203</sup> Interestingly, loading domains have not yet been identified in fungal Type I systems such as 6MSAS from *Penicillium patulum*.

Are these systems more advanced, having evolved to cope without a loading domain, or are they more ancient, not yet having evolved one?

Loper et al. have located ten genes in a cluster, eight of which could be assigned by sequence.<sup>204</sup> Most genes were close on the genome except pltL and pltR with a 480 bp gap. PltA, PltD and PltM all have sequence similarity to known halogenating enzymes, but PltD (unlike PltA and PltM) lacks the GxGxx-(G/A)x3(G/A)x6GNADH binding motif and is presumably non-functional. PltE has homology to the acyl CoA dehydrogenase involved in the biosynthesis of proline derived undecylprodigiosin. PltF shows homology to adenylation domains, so the starter unit may be activated as an adenylate rather than CoA thioester. PltG contains GxSxG and GxH motifs separated by 131 aa and shows homology to rat TE2 and CmaT (see below). PltL shows some homology to ACPs, but with poor homology to the active site motif. However, PltL may be involved in transferring the adenylate of proline onto KAS1. Thus the *de facto* starter unit could actually be the ACP thioester of proline. PltR shows homology to the LysR transcriptional regulators and contains an E bright box motif. There are no proteins in this cluster for cyclisation and aromatisation of the resorcinol ring—these may be outside the currently defined region, or occur concurrently with thioester hydrolysis etc. The authors added L-[1-<sup>13</sup>C] proline and showed that it was specifically (6% enrichment) incorporated into pyoluteorin's dichloropyrrole ring, but were unable to incorporate later putative intermediates (Scheme 42). PltE may act as a 2,3-dehydrogenase, converting activated proline 376 into 2,3-dehydropoline; the subsequent aromatisation to 377 may

Table 28 Gene cluster for pyoluteorin biosynthesis in *Pseudomonas fluorescens* Pf-5

| Gene    | Size (bp) | Putative function                                      |
|---------|-----------|--------------------------------------------------------|
| pltM    | 1506      | Pyrrole chlorination                                   |
| pltR    | 1029      | Pathway specific regulator                             |
| pltL    | 264       | ?                                                      |
| pltA    | 1347      | Pyrrole chlorination                                   |
| pltB    | 7374      | PKS (resorcinol)                                       |
| pltC    | 5322      | PKS (resorcinol)                                       |
| pltD    | 1632      | Pyrrole chlorination (non-functional?)                  |
| pltE    | 1140      | Pyrrole synthesis (flavin/acyl-CoA dehydrogenase)      |
| pltF    | 1494      | Precursor activation (acyl CoA synthetase)             |
| pltG    | 780       | Thioesterase-termination of resorcinol biosynthesis   |

occur spontaneously. The relative order of pyrrole formation, chlorination, and resorcinol formation is unknown, or more than one of the possible pathways illustrated in Scheme 42 may occur.

---

**Scheme 42 Biosynthesis of pyoluteorin.**

![Chemical Structure](https://i.imgur.com/chemical_structure.png)

---

Table 29 Domain organisation of CORS in *Pseudomonas syringae*

| Pseudomonas syringae | cfa6 | cfa7 |
|-----------------------|------|------|
| CORS                  | 0    | 1    | 2    |
| KAS                   | ✓    | ✓    | ✓    |
| AT                    | ✓    | N    | N    |
| Base                  | ✓    | ✓    | ✓    |
| DH                    | ✓    | ✓    | ✓    |
| ER                    | ✓    | ✓    | ✓    |
| KR                    | ✓    | ✓    | ✓    |
| ACP                   | ✓    | ✓    | TE1  |

soluble active protein and show that it can act as a thioesterase.²⁰⁷

A possible biosynthetic scheme is illustrated in which the discrete enzymes are used to convert 2-oxopentanedioic acid (α-oxoglutarate) 382 via 383 and 384 into the starter unit 5-oxocyclopent-1-ene-1-carboxylic acid CoA thioester or enzyme bound thioester 385 (Scheme 43). This starter unit is either transferred as a CoA thioester, or ACP bound thioester, onto Cfa6ACP1. Cfa6 is then used to add on and fully reduce a butanoate to give 386. The free acid of this intermediate, 2-(1-oxocyclopent-2-en-2-ylmethyl)butanoic acid, has been isolated from the fermentation broth.²⁰⁸ Cfa7 then performs a two carbon extension to 387 with Michael addition to form the six-membered ring 388, followed by KR and DH to 390. The catalyst for the Michael reaction is not clear, but is presumably contained within Cfa7. The PKS product CFA 390 is then combined with the cyclopropyl amino acid CMA by Cfl or Cfa5 to form coronatine.

### 14.3 2,4-Diacetylphloroglucinol

Several strains of fluorescent *Pseudomonas* spp. produce 2,4-diacetylphloroglucinol 394, which has a broad spectrum of antibiotic activity, in particular against root and seedling pathogens of plants.

Bangera and Thomashow have sequenced the 6.5 kbp of DNA from *Pseudomonas fluorescens* Q2-87 responsible for its production, and found six ORFs conforming with standard *Pseudomonas* codon usage (Table 31).²⁰⁹ *PhlA* was far more homologous to *E. coli* KASI than to KASIIIs from other organisms. However, it lacks the active site cysteine residue (in common with Type II CLFs), and in the phosphopantetheine binding motif threonine replaces serine. *PhlC* contains an acetyl CoA binding site with active site cysteine. A second region may be the malonyl CoA binding site, and the reaction to form 3-oxobutanoyl CoA may take place on PhlC, to form the ‘starter unit’. This sequence of this second region, GHAS-GCDG may be an evolutionary connection between thiolases and condensing enzymes, with the conserved histidine. PhlABC is also thought responsible for aromatic ring acylation, converting 393 to 394.

PhlD shows high homology with the active site and family signature sequences for higher plant chalcone and stilbene synthases, and to three unknown bacterial chalcone synthases.

Table 30 Gene cluster for coronafacic acid biosynthesis in *Pseudomonas syringae*

| Pseudomonas syringae | Size | Putative function |
|----------------------|------|-------------------|
| $cfl$               | $\Rightarrow$ | ACP (discrete Type II) |
| $ORF1$              | $\Rightarrow$ | DH (discrete Type II) |
| $ORF2$              | $\Rightarrow$ | KAS (discrete Type II) |
| $ORF3$              | $\Rightarrow$ | ? |
| $ORF4$              | $\Rightarrow$ | CL |
| $ORF5$              | $\Rightarrow$ | Loading domain/Mod1 |
| $cfa6$              | $\Rightarrow$ | 8.0 kbp, 284 kDa |
| $cfa7$              | $\Rightarrow$ | 6.2 kbp, 221 kDa |
| $cfa8$              | $\Rightarrow$ | Crotonyl CoA reductase |
| $cfa9$              | $\Rightarrow$ | TE |
| $corR$              | $\cdot\Rightarrow$ | Regulation |
| $corS$              | $\cdot\Rightarrow$ | Regulation (protein kinase-environmental sensor) |
| $corP$              | $\Leftarrow\cdot$ | Regulation |
| $cmaA$              | $\cdot\Rightarrow$ | Shows homology with PltG TE2 |
| $cmaB$              | $\Rightarrow$ |  |
| $cmaT$              | $\Rightarrow$ |  |
| $cmaU$              | $\cdot\Rightarrow$ |  |

---

**Scheme 43**

![Chemical Reaction Scheme](https://i.imgur.com/chemical_scheme_43.png)

---

Table 31 Gene cluster for diacetylphloroglucinol biosynthesis in *Pseudomonas fluorescens* Q2-87

| Gene   | Size           | Putative function                                      |
|--------|----------------|---------------------------------------------------------|
| $phlE$ | 423 aa/45.2 kDa | Efflux                                                 |
| $phlD$ | 349 aa/38.4 kDa | Chalcone/stilbene synthases                            |
| $phlB$ | 146 aa/16.6 kDa | Thiolase, formation of 3-oxo-                          |
|        |                | butanoyl CoA?                                          |
| $phlC$ | 398 aa/42.5 kDa | $E. coli$ KASI (3-oxoacyl ACP synthase III)            |
| $phlA$ | 360 aa/37.9 kDa | Regulatory or repressor                                |
| $phlF$ | 202 aa/23.0 kDa |  |

---

**Scheme 44**

![Chemical Reaction Scheme](https://i.imgur.com/chemical_scheme_44.png)

---

responsible for pigment production recently. The chalcone synthases are characterised by consisting of a homodimer (42 kDa) that catalyses three condensation reactions and the aromatisation/cyclisation reaction, using only malonate extender units, with the absence of an acyl carrier protein domain, i.e., the extender units and intermediates remain as CoA thioesters. Whilst plant CHSs use a range of starter units, usually 4-coumaroyl CoA, in this bacterial example, the starter unit may be 3-oxobutanoyl CoA (Scheme 44). A 372 aa protein RppA from *S. griseus* shows homology with chalcone synthases and catalyses polyketide synthesis using malonyl CoA (not an aromatic starter) as starter and after four condensations produces 1,3,6,8-tetrahydroxynaphthalene; disruption of RppA abolishes hyphal melanin production.

This may be the bacterial forerunner of higher plant CHSs, or the gene cluster may have evolved in higher plants and then moved horizontally to the Pseudomonas. However, the closest homologues of PhlA, B and C are found in archaeobacterial systems. The plant CHSs use phenylpropanoid derived starter units, unavailable in these bacterial systems. After horizontal transfer of *phlD*, it may have recruited genes for 3-oxobutanoyl synthesis, to use as its source of primer.

### 15 Mycobacteria

The genome for the Gram positive *Mycobacterium tuberculosis* reveals a large number of Type I PKS genes, yet no macrolides have yet been identified.²¹² Mycolactone A 395 and B 396, which appear to be in equilibrium with each other, have recently been isolated from *Mycobacterium ulcerans*, and are thought to be the virulence factor associated with Buruli ulcers caused by *M. ulcerans*, which are progressive necrotising ulcers for which there is no medical treatment. This is the first identification of a macrolide produced by a human pathogen, and the first by a Mycobacterium.²¹³,²¹⁴ Their structures suggest two Type I polyketide chains. It remains to be seen whether similar compounds are involved in TB and leprosy, providing a new therapeutic target.

![Chemical Structures](https://i.imgur.com/chemical_structures.png)

**mycolactone A 395**

**mycolactone B 396** is the (4′E)-isomer

**Putative biosynthesis of mycolactone 395**

### 16 References

1. B. J. Rawlings, *Nat. Prod. Rep.*, 2001, **18**, 190.
2. B. J. Rawlings, *Nat. Prod. Rep.*, 1999, **16**, 425.
3. B. J. Rawlings, *Nat. Prod. Rep.*, 1998, **15**, 275.
4. B. J. Rawlings, *Nat. Prod. Rep.*, 1997, **14**, 335.
5. B. J. Rawlings, *Nat. Prod. Rep.*, 1997, **14**, 523.
6. *Biosynthesis: Aromatic polyketides, isoprenoids, alkaloids, Topics in Current Chemistry*, vol. 209, ed. F. J. Leeper and J. C. Vederas, Springer-Verlag, Berlin, 2000.
7. R. Spagnoli, L. Cappelletti and L. Toscano, *J. Antibiot.*, 1983, **36**, 365.
8. D. G. Swan, A. M. Rodriguez, C. Vilches, C. Méndez and J. A. Salas, *Mol. Gen. Genet.*, 1994, **242**, 358.
9. L. Tang, H. Fu and R. McDaniel, *Chem. Biol.*, 2000, **7**, 77.
10. S. Shah, Q. Xue, L. Tang, J. R. Carney, M. Betlach and R. McDaniel, *J. Antibiot.*, 2000, **53**, 502.
11. A. M. Rodriguez, C. Olano, C. Méndez, C. R. Hutchinson and J. A. Salas, *FEMS Microbiol. Lett.*, 1995, **127**, 117.
12. C. Olano, A. M. Rodriguez, J.-M. Michel, C. Méndez, M.-C. Raynal and J. A. Salas, *Mol. Gen. Genet.*, 1998, **259**, 299.
13. M. Doumith, R. Legrand, C. Lang, J. A. Salas and M.-C. Raynal, *Mol. Microbiol.*, 1999, **34**, 1039.
14. G. Draeger, S.-H. Park and H. G. Floss, *J. Am. Chem. Soc.*, 1999, **121**, 2611.
15. H. Chen, G. Agnihotri, Z. Guo, N. L. S. Quo, X. H. Chen and H.-w. Liu, *J. Am. Chem. Soc.*, 1999, **121**, 8124.
16. I. Aguirrezabalaga, C. Olano, N. Allende, L. Rodriguez, A. F. Braña, C. Mendez and J. A. Salas, *Antimicrob. Agents Chemother.*, 2000, **44**, 1266.
17. J. F. Fierro, C. Hardisson and J. A. Salas, *J. Gen. Microbiol.*, 1983, **133**, 1931.
18. P. R. Rostek, P. A. Reynolds and C. L. Hersberger, *Gene*, 1991, **102**, 27.
19. A. M. Rodriguez, C. Olano, C. Vilches, C. Mendez and J. A. Salas, *Mol. Microbiol.*, 1993, **8**, 571.
20. C. Méndez and J. A. Salas, *FEMS Microbiol. Lett.*, 1998, **158**, 1.
21. C. Olano, A. M. Rodriguez, C. Méndez and J. A. Salas, *Mol. Microbiol.*, 1995, **16**, 333.
22. A. Buche, C. Méndez and J. A. Salas, *Biochem. J.*, 1997, **321**, 139.
23. J. Sasaki, K. Mizoue, S. Morimoto and S. Omura, *J. Antibiot.*, 1996, **49**, 1110.
24. L. M. Quirós and J. A. Salas, *J. Biol. Chem.*, 1995, **270**, 18234.
25. C. Hernández, C. Olano, C. Méndez and J. A. Salas, *Gene*, 1993, **134**, 139.
26. L. M. Quirós, C. Hernández and J. A. Salas, *Eur. J. Biochem.*, 1994, **222**, 129.
27. C. Vilches, C. Hernandez, C. Mendez and J. A. Salas, *J. Bacteriol.*, 1992, **174**, 161.
28. L. M. Quiros, R. J. Carbajo, A. F. Braña and J. A. Salas, *J. Biol. Chem.*, 2000, **275**, 11713.
29. L. M. Quirós, I. Aguirrezabalaga, C. Olano, C. Mendez and J. A. Salas, *Mol. Microbiol.*, 1998, **28**, 1177.
30. S. Gaisser, G. A. Bohm, M. Doumith, M.-C. Raynal, N. Dhillon, J. Cortes and P. F. Leadlay, *Mol. Gen. Genet.*, 1998, **258**, 78.
31. L. M. Quiros, R. J. Carbajo and J. A. Salas, *FEBS Lett.*, 2000, **476**, 186.
32. A. Vancura, I. Vancurova, J. Marsalek, K. Melzoch, G. Basrova and V. Kristan, *J. Gen. Microbiol.*, 1988, **134**, 3213.
33. L. T. Nguyen, K. T. Nguyen, J. Spižek and V. Běhal, *Microbiology*, 1995, **141**, 1139.
34. L. Tang and C. R. Hutchinson, *J. Bacteriol.*, 1993, **175**, 4176.
35. A. R. Butler, N. Bate and E. Cundliffe, *Chem. Biol.*, 1999, **6**, 287.
36. A. Schneider and M. A. Marahiel, *Arch. Microbiol.*, 1998, **169**, 404.
37. (a) M. L. Heathcote, J. Staunton and P. F. Leadlay, *Abstracts of the Eleventh International Symposium of the Biology of Actinomycetes*, Sissi, Crete, 1999, p. 187; (b) M. L. Heathcote, J. Staunton and P. F. Leadlay, *Chem. Biol.*, 2001, **8**, 207.
38. S. Omura, *Macrolide antibiotics: Chemistry, Biology and Practice*, Academic Press, Orlando, 1984.
39. S. A. Fish and E. Cundliffe, *Microbiology*, 1997, **143**, 3871.
40. L. A. Merson-Davies and E. Cundliffe, *Mol. Microbiol.*, 1994, **13**, 349.
41. N. Bate, A. R. Butler, I. P. Smith and E. Cundliffe, *Microbiology*, 2000, **146**, 139.
42. N. Bate and E. Cundliffe, *J. Ind. Microbiol. Biotechnol.*, 1999, **23**, 118.
43. V. T. W. Wilson and E. Cundliffe, *Gene*, 1998, **214**, 95.
44. R. Fouces, E. Mellado, B. Diez and J. L. Barredo, *Microbiology*, 1999, **145**, 855.
45. N. Bate, A. R. Butler, A. R. Gandecha and E. Cundliffe, *Chem. Biol.*, 1999, **6**, 617.
46. A. R. Gandecha, S. L. Large and E. Cundliffe, *Gene*, 1997, **184**, 197.
47. S. Gaisser, J. Reather, G. Wirtz, L. Kellenberger, J. Staunton and P. F. Leadlay, *Mol. Microbiol.*, 2000, **36**, 391.
48. H. Chen, S.-M. Yeung, N. L. S. Que, T. Muller, R. S. Schmidt and H.-w. Liu, *J. Am. Chem. Soc.*, 1999, **121**, 7166.
49. A. Arisawa, N. Kawamura, K. Takeda, H. Tsunekawa, K. Okamura and R. Okamoto, *Appl. Environ. Microbiol.*, 1994, **60**, 2657.
50. A. R. Gandecha and E. Cundliffe, *Gene*, 1996, **180**, 173.
51. V. T. W. Wilson and E. Cundliffe, *J. Antibiot.*, 1999, **52**, 288.
52. B. Aigle, D. Schneider, C. Morilhat, D. Vandewiele, A. Dary, A.-C. Holl, J.-M. Simonet and B. Decaris, *Microbiology*, 1996, **142**, 2815.
53. S. Kuhstoss, M. Huber, J. R. Turner, J. W. Paschal and R. N. Rao, *Gene*, 1996, **183**, 231.
54. A. Gourmelen, M.-H. Blondelet-Roualt and J.-L. Pernodet, *Antimicrob. Agents Chemother.*, 1998, **42**, 2612.
55. S. Omura, K. Tsuzuki, A. Nakagawa and G. Lukacs, *J. Antibiot.*, 1983, **36**, 611.
56. S. J. Kakavas, L. Katz and D. Stassi, *J. Bacteriol.*, 1997, **179**, 7515.

57 T. Hori, I. Maezawa, N. Nagahama and M. Suzuki, Chem. Commun., 1971, 304.

58 D. E. Cane, R. H. Lambalot, P. C. Prabhakaran and W. C. Ott, J. Am. Chem. Soc., 1993, 115, 522.

59 Y. Xue, L. Zhao, H.-w. Liu and D. H. Sherman, Proc. Natl. Acad. Sci. USA, 1998, 95, 12 111.

60 K. A. Reynolds, Proc. Natl. Acad. Sci. USA, 1998, 95, 12 744.

61 L. Tang, H. Fu, M. C. Betlach and R. McDaniel, Chem. Biol., 1999, 6, 553.

62 Y. Xue and D. H. Sherman, Nature, 2000, 403, 571.

63 R. McDaniel, A. Thamchaipenet, C. Gustafsson, H. Fu, M. Betlach, M. Betlach and G. Ashley, Proc. Natl. Acad. Sci. USA, 1999, 96, 1846.

64 Q. Xue, G. Ashley, C. R. Hutchinson and D. V. Santi, Proc. Natl. Acad. Sci. USA, 1999, 96, 11 740.

65 R. S. Gokhale, S. Y. Tsuji, D. E. Cane and C. Khosla, Science, 1999, 284, 482.

66 Y. Xue, L. Zhao, H.-w. Liu and D. H. Sherman, Proc. Natl. Acad. Sci. USA, 1998, 95, 12 111.

67 L. Zhao, D. H. Sherman and H.-w. Liu, J. Am. Chem. Soc., 1998, 120, 9374.

68 R. H. Lambalot and D. E. Cane, J. Antibiot., 1982, 45, 1981.

69 D. E. Cane and E. I. Graziani, J. Am. Chem. Soc., 1998, 120, 2682.

70 M. C. Betlach, J. T. Kealey, M. C. Betlach, G. W. Ashley and R. McDaniel, Biochemistry, 1998, 37, 14 937.

71 L. Zhao, D. H. Sherman and H.-w. Liu, J. Am. Chem. Soc., 1998, 120, 10 256.

72 L. Zhao, N. L. S. Que, Y. Xue, D. H. Sherman and H.-w. Liu, J. Am. Chem. Soc., 1998, 120, 12 159.

73 H.-w. Liu and J. S. Thorsen, Annu. Rev. Microbiol., 1994, 48, 223.

74 C.-w. Chang, L. Zhao, H. Yamase and H.-w. Liu, Angew. Chem., Int. Ed., 2000, 39, 2160.

75 L. Zhao, J. Ahlert, Y. Xue, J. S. Thorson, D. H. Sherman and H.-w. Liu, J. Am. Chem. Soc., 1999, 121, 9881.

76 M. Inouye, Y. Takada, N. Muto, T. Beppu and S. Horinouchi, Mol. Gen. Genet., 1994, 245, 456.

77 A. Taddei and A. Zeeck, J. Antibiot., 1997, 50, 526.

78 A. Leiser, A. Birch and J. A. Robinson, Gene, 1996, 177, 217.

79 H. G. Davies and R. H. Green, Nat. Prod. Rep., 1986, 3, 87.

80 H. Ikeda and S. Omura, Chem. Rev., 1997, 97, 2591.

81 H. Ikeda and S. Omura, J. Antibiot., 1995, 48, 549.

82 D. E. Cane, T.-C. Liang, L. Kaplan, M. K. Nallin, M. D. Schulman, O. D. Hensens, A. W. Douglas and G. Albers-Schonberg, J. Am. Chem. Soc., 1983, 105, 4110.

83 Y.-X. Zhang, C. D. Denoya, D. D. Skinner, R. W. Fedechko, H. A. I. McArthur, M. R. Morgenstern, R. A. Davies, S. Lobo, K. A. Reynolds and C. R. Hutchinson, Microbiology, 1999, 145, 2323.

84 E. D. Rossi, R. Leva, L. Gusberti, P. L. Manachini and G. Riccardi, Gene, 1995, 166, 127.

85 T. A. Cropp, D. J. Wilson and K. A. Reynolds, Nature: Biotechnol., 2000, 18, 980.

86 H. Ikeda, T. Nonomiya, M. Usami, T. Ohta and S. Omura, Proc. Natl. Acad. Sci. USA, 1999, 96, 9509.

87 H. Ikeda, L.-R. Wang, T. Ohta, J. Inokoshi and S. Omura, Gene, 1998, 206, 175.

88 H. Ikeda, C.-H. Pang and H. Endo, J. Antibiot., 1995, 48, 532.

89 C.-H. Pang, K. Matsuzaki, H. Ikeda, H. Tanaka and S. Omura, J. Antibiot., 1995, 48, 59.

90 T. Brautaset, O. N. Sekurova, H. Sletta, T. E. Ellingsen, A. R. Strøm, S. Valla and S. B. Zotchev, Chem. Biol., 2000, 7, 395.

91 T. Brautaset, O. N. Sekurova, H. Sletta, T. E. Ellingsen, A. R. Strøm, S. Valla and S. B. Zotchev, Chem. Biol., 2000, 7, R176.

92 S. Zotchev, K. Haugan, O. Sekurova, H. Sletta, T. E. Ellingsen and S. Valla, Microbiology, 2000, 146, 611.

93 P. R. James and B. J. Rawlings, Bioorg. Med. Chem. Lett., 1996, 6, 505.

94 Y. T. Liu, PhD Thesis, Rutgers University, NJ, 1981.

95 (a) C. M. McNamara, S. Box, J. M. Crawforth, B. S. Hickman, T. J. Norwood and B. J. Rawlings, J. Chem. Soc., Perkin Trans. 1, 1998, 83; (b) P. Caffrey, personal communication.

96 J. F. Aparicio, A. J. Colina, E. Ceballos and J. F. Martín, J. Biol. Chem., 1999, 274, 10 133.

97 J. F. Aparicio, R. Fouces, M. V. Mendes, N. Olivera and J. F. Martín, Chem. Biol., 2000, 7, 895.

98 S. Kobayashi, K. Tsuchiya, M. Nishide, T. Nishikiori, T. Nakagawa and N. Shimada, J. Antibiot., 1995, 48, 893.

99 S. Sakuda, U. Guce-Bigol, M. Itoh, T. Nishimura and Y. Yamada, J. Chem. Soc., Perkin Trans. 1, 1996, 2315.

100 M. Ono, S. Sakuda, H. Ikeda, K. Furihata, J. Nakayama, A. Suzuki and A. Isogai, J. Antibiot., 1998, 51, 1019.

101 N. D. Priestley and S. Gröger, J. Org. Chem., 1995, 60, 4951.

102 B. S. Moore, Nat. Prod. Rep., 1999, 16, 653.

103 Y. S. Tsantrizos, J. Shen and L. A. Trimble, Tetrahedron Lett., 1997, 38, 7033.

104 M. Otsuka, T. Eguchi, K. Shindo and K. Kakinuma, Tetrahedron Lett., 1998, 39, 3185.

105 H. G. Floss, Nat. Prod. Rep., 1997, 14, 433.

106 J. P. Lee, S.-W. Tsao, C.-j. Chang, X.-G. He and H. G. Floss, Can. J. Chem., 1994, 72, 182.

107 J. J. Kirby, I. A. McDonald and R. W. Rickards, Chem. Commun., 1980, 768.

108 M. G. Anderson, D. Monypenny, R. W. Rickards and J. M. Rothchild, Chem. Commun., 1989, 311.

109 M. G. Anderson, J. J. Kirby, R. W. Rickards and J. M. Rothchild, Chem. Commun., 1980, 1277.

110 R. Casati, J. M. Beale and H. G. Floss, J. Am. Chem. Soc., 1987, 109, 8102.

111 U. Horneman, J. H. Eggert and D. P. Honor, Chem. Commun., 1980, 11.

112 K. L. Rhinehart, M. Potgieter and D. A. Wright, J. Am. Chem. Soc., 1982, 104, 2649.

113 C.-G. Kim, A. Kirschning, P. Bergon, Y. Ahn, J. J. Wang, M. Shibuya and H. G. Floss, J. Am. Chem. Soc., 1992, 114, 4941.

114 C.-G. Kim, A. Kirschning, P. Bergon, P. Zhou, E. Su, B. Sauerbrei, S. Ning, Y. Ahn, M. Breuer, E. Leistner and H. Floss, J. Am. Chem. Soc., 1996, 118, 7486.

115 D. Gygax, O. Ghisalba, H. Treichler and J. Nuesch, J. Antibiot., 1990, 43, 324.

116 R. Jiao, C. Liu, Z. Jin, X. Zhang, L. Ni and Z. Lu, Sinica. Ser. B (Engl. Edn.), 1984, 27, 380.

117 C.-G. Kim, T.-W. Yu, C. B. Fryhle, S. Handa and H. G. Floss, J. Biol. Chem., 1998, 273, 6030.

118 J. C. Eads, M. Beeby, G. Scapin, T.-W. Yu and H. G. Floss, Biochemistry, 1999, 38, 9840.

119 H. G. Floss and T.-W. Yu, Curr. Opin. Chem. Biol., 1999, 3, 592.

120 T. Schupp, C. Toupet, N. Engel and S. Goff, FEMS Microbiol. Lett, 1998, 159, 201.

121 P. R. August, L. Tang, Y. J. Yoon, S. Ning, R. Müller, T.-W. Yu, M. Taylor, D. Hoffman, C.-G. Kim, X. Zhang, C. R. Hutchinson and H. G. Floss, Chem. Biol., 1998, 5, 69.

122 L. Tang, Y. J. Yoon, C.-Y. Choi and C. R. Hutchinson, Gene, 1998, 216, 255.

123 A. Stratmann, C. Toupet, W. Schilling, R. Traber, L. Oberer and T. Schupp, Microbiology, 1999, 145, 3365.

124 T.-W. Yu, Y. Shen, Y. Doi-Katayama, L. Tang, C. Park, B. S. Moore, C. R. Hutchinson and H. G. Floss, Proc. Natl. Acad. Sci. USA, 1999, 96, 9051.

125 Y. Doi-Katayama, Y. J. Yoon, C.-Y. Choi, T.-W. Yu, H. G. Floss and C. R. Hutchinson, J. Antibiot., 2000, 53, 484.

126 (a) Z. Hu, D. Hunziker, C. R. Hutchinson and C. Khosla, Microbiology, 1999, 145, 2335; (b) D. Hunziker, T. W. Yu, C. R. Hutchinson, H. G. Floss and C. Khosla, J. Am. Chem. Soc., 1998, 120, 1092.

127 A. M. Hooper and R. W. Rickards, J. Antibiot., 1998, 51, 845.

128 A. M. Hooper and R. W. Rickards, J. Antibiot., 1998, 51, 958.

129 S. Chen, D. von Bamberg, V. Hale, M. Breuer, B. Hardt, R. Müller, H. G. Floss, K. A. Reynolds and E. Leistner, Eur. J. Biochem., 1999, 261, 98.

130 M. Sugita, Y. Natori, N. Sueda, K. Furihata, H. Seto and N. Otake, J. Antibiot., 1982, 35, 1474.

131 M. Damberg, P. Russ and A. Zeeck, Tetrahedron Lett., 1982, 23, 59.

132 S. M. Patton, T. A. Cropp and K. A. Reynolds, Biochemistry, 2000, 39, 7595.

133 Y. Mao, M. Varoglu and D. H. Sherman, Chem. Biol., 1999, 6, 251.

134 Y. Mao, M. Varoglu and D. H. Sherman, J. Bacteriol., 1999, 181, 2199.

135 I. Sattler, R. Thiericke and A. Zeeck, Nat. Prod. Rep., 1998, 15, 221.

136 (a) S. J. Gould, C. R. Melville and M. C. Cone, J. Am. Chem. Soc., 1996, 118, 9228; (b) H. Cho, I. Sattler, J. M. Beale, A. Zeeck and H. G. Floss, J. Org. Chem., 1993, 58, 7925.

137 Y. Hu, C. R. Melville, S. J. Gould and H. G. Floss, J. Am. Chem. Soc., 1997, 119, 4301.

138 N. L. Paiva, M. F. Roberts and A. L. Demain, J. Ind. Microbiol., 1993, 12, 423.

139 B. S. Moore, K. Poralla and H. G. Floss, J. Am. Chem. Soc., 1993, 115, 5267.

140 B. S. Moore, H. Cho, R. Casati, E. Kennedy, K. A. Reynolds, U. Mocek, J. M. Beale and H. G. Floss, *J. Am. Chem. Soc.*, 1993, **115**, 5254.

141 S. Handa and H. G. Floss, *Chem. Commun.*, 1997, 153.

142 K. A. Reynolds, K. M. Fox, Z.-m. Yuan and Y. Lam, *J. Am. Chem. Soc.*, 1991, **113**, 4339.

143 K. K. Wallace, K. A. Reynolds, K. Koch, H. A. I. McArthur, M. S. Brown, R. G. Wax and B. S. Moore, *J. Am. Chem. Soc.*, 1994, **116**, 11600.

144 K. A. Reynolds, K. K. Wallace, S. Handa, M. S. Brown, H. A. I. McArthur and H. G. Floss, *J. Antibiot.*, 1997, **50**, 701.

145 C. F. Bridge, D. O'Hagan, K. A. Reynolds and K. K. Wallace, *Chem. Commun.*, 1995, 2329.

146 P. Wang, C. D. Denoya, M. R. Morgernstern, D. D. Skinner, K. K. Wallace, R. Digate, S. Patton, N. Banavali, G. Schuler, M. K. Speedie and K. A. Reynolds, *J. Bacteriol.*, 1996, **178**, 6873.

147 T. Schwecke, J. F. Aparicio, I. Molnár, A. König, L. E. Khaw, S. F. Haydock, M. Oliynyk, P. Caffrey, J. Cortés, J. B. Lester, G. A. Böhm, J. Staunton and P. F. Leadlay, *Proc. Natl. Acad. Sci. USA*, 1995, **92**, 7839.

148 X. Ruan, D. Stassi, S. A. Lax and L. Katz, *Gene*, 1997, **203**, 1.

149 (a) I. Molnár, J. F. Aparicio, S. F. Haydock, L. E. Khaw, T. Schwecke, A. König, J. Staunton and P. F. Leadlay, *Gene*, 1996, **169**, 1; (b) J. F. Aparicio, I. Molnár, T. Schwecke, A. König, S. F. Haydock, L. E. Khaw, J. Staunton and P. F. Leadlay, *Gene*, 1996, **169**, 9.

150 (a) P. A. S. Lowden, G. A. Böhm, J. Staunton and P. F. Leadlay, *Angew. Chem., Int. Ed. Engl.*, 1996, **35**, 2249; (b) P. F. Leadlay, personal communication.

151 J. B. Nielson, M. J. Hsu, K. M. Bryne and L. Kaplan, *Biochemistry*, 1991, **30**, 5789.

152 A. König, T. Schwecke, I. Molnár, G. A. Böhm, P. A. S. Lowden, J. Staunton and P. F. Leadlay, *Eur. J. Biochem.*, 1997, **247**, 526.

153 H. Tanaka, A. Kuroda, H. Marusawa, H. Hatanaka, T. Kino, T. Goto and M. Hashimoto, *J. Am. Chem. Soc.*, 1987, **109**, 5031.

154 H. Motamedi, S.-J. Cai, A. Shafiee and K. O. Elliston, *Eur. J. Biochem.*, 1997, **244**, 74.

155 (a) H. Motamedi and A. Shafiee, *Eur. J. Biochem.*, 1998, **256**, 528; (b) A. Shafiee, H. Motamedi and T. Chen, *Eur. J. Biochem.*, 1994, **225**, 755; (c) H. Motamedi, A. Shafiee, S.-J. Cai, S. L. Streicher, B. H. Arison and R. R. Miller, *J. Bacteriol.*, 1996, **178**, 5243.

156 H. B. Bode and A. Zeeck, *J. Chem. Soc., Perkin Trans. 1*, 2000, 323.

157 H. B. Bode and A. Zeeck, *J. Chem. Soc. Perkin Trans. 1*, 2000, 2665.

158 H. Oikawa, Y. Suzuki, A. Naya, K. Katayama and A. Ichihara, *J. Am. Chem. Soc.*, 1994, **116**, 3605.

159 H. Oikawa, K. Katayama, Y. Suzuki and A. Ichihara, *Chem. Commun.*, 1995, 1321.

160 L. Hendrickson, C. R. Davis, C. Roach, D. K. Nguyen, T. Aldrich, P. C. McAda and C. D. Reeves, *Chem. Biol.*, 1999, **6**, 429.

161 W. D. Celmer, G. N. Chmurny, C. E. Moppett and R. S. Ware, *J. Am. Chem. Soc.*, 1980, **102**, 4203.

162 H. A. Whaley, C. G. Chidester and S. A. Mizzak, *Tetrahedron Lett.*, 1980, **21**, 3659.

163 D. E. Cane and C.-C. Yang, *J. Am. Chem. Soc.*, 1984, **106**, 784.

164 W. C. Snyder and K. L. R. Jr, *J. Am. Chem. Soc.*, 1984, **106**, 787.

165 D. E. Cane and C.-C. Yang, *J. Antibiot.*, 1985, **38**, 423.

166 D. E. Cane and W. R. Ott, *J. Am. Chem. Soc.*, 1988, **110**, 4840.

167 D. E. Cane, W. Tan and W. R. Ott, *J. Am. Chem. Soc.*, 1993, **115**, 527.

168 D. E. Cane and G. Luo, *J. Am. Chem. Soc.*, 1995, **117**, 6633.

169 Z. Li, M. Martin and J. C. Vederas, *J. Am. Chem. Soc.*, 1992, **114**, 1531.

170 Y. Yoshizawa, Z. Li, P. B. Reese and J. C. Vederas, *J. Am. Chem. Soc.*, 1990, **112**, 3212.

171 H. Patzelt, V. Gaudet and J. A. Robinson, *J. Antibiot.*, 1992, **45**, 1806.

172 H. Patzelt and J. A. Robinson, *Chem. Commun.*, 1993, 1258.

173 W. R. Roush, K. Koyama, M. L. Curtin and K. J. Moriarty, *J. Am. Chem. Soc.*, 1996, **118**, 7502.

174 J. B. McAlpine, L. A. Mitscher, M. Jackson, R. R. Rasmussen, D. V. Velde and E. Veliz, *Tetrahedron*, 1996, **52**, 10327.

175 K. Shindo, M. Sakakibara and H. Kawai, *J. Antibiot.*, 1996, **49**, 249.

176 K. Shindo, M. Matsuoka and H. Kawai, *J. Antibiot.*, 1996, **49**, 241.

177 K. Shindo, H. Iijima and H. Kawai, *J. Antibiot.*, 1996, **49**, 244.

178 F. J. Leeper, P. Padmanabhan, G. W. Kirby and G. N. Sheldrake, *Chem. Commun.*, 1987, 505.

179 F. J. Leeper, S. E. Shaw and P. Satish (née Padmanabhan), *Can. J. Chem.*, 1994, **72**, 131.

180 R. D. Kuchta and R. H. Abeles, *J. Biol. Chem.*, 1985, **260**, 13181.

181 K. C. Nicolaou, F. Roschangar and D. Vourloumis, *Angew. Chem., Int. Ed.*, 1998, **37**, 2014.

182 K. Gerth, N. Bedorf, G. Höfle, H. Irschik and H. Reichenbach, *J. Antibiot.*, 1996, **49**, 560.

183 I. Molnár, T. Schupp, M. Ono, R. E. Zirkle, M. Milnamow, B. Nowak-Thompson, N. Engel, C. Toupet, A. Stratmann, D. D. Cyr, J. Gorlach, J. M. Mayo, A. Hu, S. Goff, J. Schmid and J. M. Ligon, *Chem. Biol.*, 2000, **7**, 97.

184 L. Tang, S. Shah, L. Chung, J. Carney, L. Katz, C. Khosla and B. Julien, *Science*, 2000, **287**, 640.

185 G. L. Challis, J. Ravel and C. A. Townsend, *Chem. Biol.*, 2000, **7**, 211.

186 T. Stachelhaus, H. D. Mootz and M. A. Marahiel, *Chem. Biol.*, 1999, **6**, 493.

187 B. Shen, L. Du, C. Sanchez, M. Chen and D. J. Edwards, *Bioorg. Chem.*, 1999, **27**, 155.

188 K. Gerth, H. Bedorf, G. Irschik, G. Hofle and H. Reichenbach, *J. Antibiot.*, 1994, **47**, 23.

189 N. Bedorf, D. Schonburg, H. Gerth, H. Riechenbach and G. Hofle, *Liebigs Ann. Chem.*, 1993, **9**, 1017.

190 A. M. Hill, J. P. Harris and A. P. Siskos, *Chem. Commun.*, 1998, 2361.

191 T. Schupp, C. Toupet, B. Cluzel, S. Neff, S. Hill, J. J. Beck and J. M. Ligon, *J. Bacteriol.*, 1995, **177**, 3673.

192 J. M. Ligon, T. Schupp, J. J. Beck, D. S. Hill, S. Neff and J. A. Ryals, in *Genes for the biosynthesis of soraphen*, USA, 1998. Novartis, New York, US Pat., 5716849, 1998.

193 S. Eisenbarth and B. Steffan, *Tetrahedron*, 2000, **56**, 363.

194 E. Rosenberg, S. Fytlovitch, S. Carmeli and Y. Kashman, *J. Antibiot.*, 1982, **35**, 788.

195 W. Trowitzch, V. Wray, K. Gerth and G. Hof, *Chem. Commun.*, 1982, 1340.

196 K. Gerth, H. Irschik, H. Riechenbach and W. Trowitzsch, *J. Antibiot.*, 1982, **35**, 1454.

197 M. Varon, N. Fuchs, M. Monosov, S. Tolchinsky and E. Rosenberg, *Antimicrob. Agents Chemother.*, 1992, **36**, 2316.

198 (a) Y. Paitan, G. Alon, E. Orr, E. Z. Ron and E. Rosenberg, *J. Mol. Biol.*, 1999, **286**, 465; (b) B. Silakowski, G. Nordsiek, B. Kunze, H. Blöcker and R. Müller, *Chem. Biol.*, 2001, **8**, 59; (c) B. Silakowski, H. U. Schairer, H. Ehret, B. Kunze, S. Weinig, G. Nordsiek, P. Brandt, H. Blöcker, G. Höfle, S. Beyer and R. Müller, *J. Biol. Chem.*, 1999, **274**, 37391.

199 S. Kirner, P. E. Hammer, D. S. Hill, A. Altmann, L. J. Fischer, L. J. Weislo, M. Lanahan, K.-H. van Pee and J. M. Ligon, *J. Bacteriol.*, 1998, **180**, 1939.

200 K.-H. van Pee and J. M. Ligon, *Nat. Prod. Rep.*, 2000, **17**, 157.

201 B. Nowak-Thompson, S. J. Gould and J. E. Loper, *Gene*, 1997, **204**, 17.

202 J. Kraus and J. Loper, *Appl. Environ. Microbiol.*, 1995, **61**, 849.

203 A.-L. Matharu, R. J. Cox, J. Crosby, K. J. Byrom and T. J. Simpson, *Chem. Biol.*, 1998, **5**, 699.

204 B. Nowak-Thompson, N. Chaney, J. S. Wing, S. J. Gould and J. E. Loper, *J. Bacteriol.*, 1999, **181**, 2166.

205 V. Rangaswamy, R. Mitchell, M. Ullrich and C. L. Bender, *J. Bacteriol.*, 1998, **180**, 3330.

206 V. Rangaswamy, S. Jiralerspong, R. Parry and C. L. Bender, *Proc. Natl. Acad. Sci.*, USA, 1998, **95**, 15469.

207 J. Patel, J. C. Hoyt and R. J. Parry, *Tetrahedron*, 1998, **54**, 15927.

208 R. E. Mitchell, H. Young and M. J. Liddell, *Tetrahedron Lett.*, 1995, **36**, 3237.

209 M. G. Bangera and L. S. Thomashow, *J. Bacteriol.*, 1999, **181**, 3155.

210 K. Ueda, K.-M. Kim, T. Beppu and S. Horinouchi, *J. Antibiot.*, 1995, **48**, 638.

211 N. Funa, I. Fujii, M. Shibuya, Y. Masaaki and S. Horinouchi, *Nature*, 1999, **400**, 897.

212 S. T. Cole, R. Brosch, J. Parkhill, T. Garnier, C. Churcher, D. Harris, S. V. Gordon, K. Eiglmeier, S. Gas, C. E. Barry III, F. Tekaia, K. Badcock, D. Basham, D. Brown, T. Chillingworth, R. Connor, R. Davies, K. Devlin, T. Feltwell, S. Gentles, N. Hamlin, S. Holroyd, T. Hornsby, K. Jagels, A. Krogh, J. McLean, S. Moule, L. Murphy, K. Oliver, J. Osborne, M. A. Quail, M.-A. Rajandream, J. Rogers, S. Rutter, K. Seeger, J. Skelton, R. Squares, S. Squares, J. E. Sulston, K. Taylor, S. Whitehead and B. G. Barrell, *Nature (London)*, 1998, **393**, 537.

213 G. Gunawardana, D. Chatterjee, K. M. George, P. Brennan, D. Whittern and P. L. C. Small, *J. Am. Chem. Soc.*, 1999, **121**, 6092.

214 J. Rohr, *Angew. Chem., Int. Ed.*, 2000, **39**, 2847.
