
Glycobiology vol. 11 no. 8 pp. 655–661, 2001

# Expression and identification of the RfbE protein from Vibrio cholerae O1 and its use for the enzymatic synthesis of GDP-D-perosamine

**Christoph Albermann and Wolfgang Piepersberg**¹  
Chemische Mikrobiologie, Bergische Universität GH Wuppertal,  
Gauss-Str. 20, D-42097, Germany  

Received on January 15, 2001; revised on March 26, 2001; accepted on March 28, 2001  

The 4-amino-6-deoxy-monosaccharide D-perosamine is an important element in the glycosylation of interesting cell products, such as antibiotics and lipopolysaccharides (LPS) of Gram-positive and Gram-negative bacteria. The biosynthetic pathway of the precursor molecule, GDP-D-perosamine, in *Vibrio cholerae* O1 starts with an isomerisation of fructose-6-phosphate catalyzed by the bifunctional enzyme phosphomannose isomerase-guanosine diphosphomannose pyrophosphorylase (RfbA; E.C. 2.7.7.22) creating the intermediate mannose-6-phosphate, which is subsequently converted by the phosphomanno-mutase (RfbB; E.C. 5.4.2.8) and further by RfbA to GDP-D-mannose, to GDP-4-keto-6-deoxymannose by a 4,6-dehydratase (RfbD; E.C. 4.2.1.47) and finally to GDP-D-perosamine by an aminotransferase (RfbE; E.C. not yet classified).

We cloned the *rfbD* and the *rfbE* genes of *V. cholerae* O1 in *Escherichia coli* expression vectors. Both biosynthetic enzymes were overproduced in *E. coli* BL21 (DE3) and their activities were analyzed. The enzymatic conversion from GDP-D-mannose to GDP-D-perosamine was optimized and the final product, GDP-D-perosamine, was purified and identified by nuclear magnetic resonance, mass spectrometry, and chromatography. The catalytically active form of the GDP-4-keto-6-deoxy-D-mannose-4-aminotransferase seems to be a tetramer of 170 kDa. The His-tag RfbE fusion protein has a \(K_{\text{m}}\) of 0.06 mM and a \(V_{\text{max}}\) value of 38 nkat/mg protein for the substrate GDP-4-keto-6-deoxy-D-mannose. The \(K_{\text{m}}\) and \(V_{\text{max}}\) values for the cosubstrate L-glutamate were 0.1 mM and 42 nkat/mg protein, respectively. The intention of this work is to establish a basis for both the *in vitro* production of GDP-D-perosamine and for an *in vivo* perosaminylation system in a suitable bacterial host, preferably *E. coli*.

Key words: GDP-α-D-perosamine/6-deoxyhexose/enzymatic synthesis/GDP-4-keto-6-deoxy-D-mannose 4-aminotransferase

---

**Introduction**

The 6-deoxyhexose D-perosamine is an important and characteristic element in glycoconjugates of bacteria. In Gram-positive bacteria, for example, the D-perosamine was first detected in the antibiotic perimycin of *Streptomyces coelicolor* var. *aminophilus* (Lee and Schaffner, 1966, 1969). In Gram-negative bacteria it is also found as a main constituent of lipopolysaccharides (LPS) in *Brucella* spp., *Escherichia coli* O157, and *Vibrio cholerae* O1 (Wu and Mackenzie, 1987; Bilge *et al.*, 1996; Villeneuve *et al.*, 2000).

The O-antigen of *V. cholerae* O1, the causative agent of cholera, is a homopolymer of the dideoxy amino sugar perosamine, which is substituted via the amino group with a 4-carbon aliphatic acid, 3-deoxy-L-glycero-tetronic acid. This structure is repeated on the average 18 times and is joined to a lipid A molecule by a linker containing 6-, 7-, and 8-carbon sugars (core unit) (Hisatsune *et al.*, 1993). The introduction of the perosamine monosaccharidic unit into the glycosylated endproducts is dependent on nucleotide activation as a GDP-hexose and subsequent modification. The *rfb* region of *V. cholerae* O1 encodes the gene cluster that is responsible for the perosamine biosynthesis, the 3-deoxy-L-glycero-tetronate biosynthesis, the O-antigen transport, and of the O-antigen modification (Hisatsune *et al.*, 1996; Stroeher *et al.*, 1998). The perosamine biosynthesis cluster consists of four genes that are predicted to encode the enzymes for the conversion of fructose-6-phosphate to the activated sugar metabolite GDP-D-perosamine in five steps (Stroeher *et al.*, 1995). The first step is the conversion of fructose-6-phosphate to mannose-6-phosphate catalyzed by the phosphomannose isomerase-guanosine diphosphomannose pyrophosphorylase (RfbA; E.C. 2.7.7.22); subsequently, the mannose-6-phosphate is converted to mannose-1-phosphate by the phosphomanno-mutase (RfbB; E.C. 5.4.2.8). The third step, the nucleotide activation, is catalyzed once again by the bifunctional phosphomannose isomerase-guanosine diphosphomannose pyrophosphorylase (RfbA) to GDP-D-mannose. The two further biosynthetic steps characteristic for the formation GDP-D-perosamine basically follow that of other 6-deoxyhexoses, which are made via the dTDP- or CDP-activated sugars. It starts with a dehydration reaction catalyzed by an NAD⁺-dependent NDP-hexose 4,6-dehydratase, which forms NDP-6-deoxy-D-4-hexulose (Piepersberg, 1994; Liu and Thorson, 1994; Piepersberg and Distler, 1997). By further reactions this intermediate can be modified by a large variety of following steps, for example, the 4-carbonyl group can be enantioselectively reduced, transaminated, or fully reduced. In other cases the NDP-6-deoxy-D-4-hexuloses are converted first by an 3,5-epimerase reaction to NDP-6-deoxy-L-4-hexulose. In the case of GDP-D-perosamine, it is supposed that the activated sugar metabolite GDP-D-mannose is converted by the GDP-D-mannose 4,6-dehydratase (RfbD; E.C. 4.2.1.47) to GDP-4-keto-6-deoxy-D-mannose

---

¹ To whom correspondence should be addressed

© 2001 Oxford University Press

C. Albermann and W. Piepersberg

followed by a transamination catalyzed by the perosamine synthetase (RfbE; E.C. not yet classified) (Figure 1).

To produce novel glycoconjugates or oligosaccharides by the use of glycosyltransferases, it is necessary to synthesize the required sugar donors in a large scale. Other nucleotide-activated sugars, for instance, GDP-D-mannose, GDP-L-fucose, or dTDP-L-rhamnose, are already available fully on the basis of enzymatic reactions (Elling *et al.*, 1996; Pastuszak *et al.*, 1998; Elling and Büttler, 1999; Albermann *et al.*, 2000) with the respective enzymes overproduced in a suitable expression system such as *E. coli*. In this article we describe the cloning and the overproduction of the RfbE (GDP-D-perosamine synthetase) from *V. cholerae* O1. This work presents the evidence that the final product of the enzymatic reaction catalyzed by RfbE-protein is GDP-α-D-perosamine.

### Results

#### Cloning and expression of GDP-D-mannose 4,6-dehydratase (RfbD) and GDP-D-perosamine synthetase (RfbE)

The two open reading frames *rfbD* and *rfbE* of the LPS biosynthesis cluster of *V. cholerae* strain O1 (Stroeher *et al.*, 1992; Manning *et al.*, 1995) are predicted to encode a GDP-D-mannose 4,6-dehydratase and GDP-D-perosamine synthetase. The RfbD protein has high similarity with other 4,6-dehydratases from *E. coli*, *Salmonella enterica*, *Yersinia enterocolitica*, or *Helicobacter pylori*. The RfbE protein possesses a pyridoxal-phosphate-binding domain, which is supposed to be a group VI (secondary metabolic) aminotransferase and is in the cluster of orthologous proteins COG0399 (Tatusov *et al.*, 2001) (Figure 2). Both genes were amplified and modified by polymerase chain reaction (PCR), respectively, to introduce each an N-terminal *NdeI*, overlapping the ATG start codon, and a *BamHI* restriction site downstream from the stop codon. These were used to clone the respective PCR products into the expression vector pET11a and were verified by sequence analysis. Additionally the genes were cloned into the pET16b expression vector to obtain their products as N-terminal His-tag fusion proteins. The plasmids pCAW 13.1 (*rfbD*), pCAW 13.2 (His-tag *rfbD*), pCAW 14.1 (*rfbE*), and pCAW 14.2 (His-tag *rfbE*) constructed this way were transformed into *E. coli* BL21 (DE3) pLysS, where their expression under induced conditions resulted in the overproduction of large amounts of RfbD and RfbE proteins. The apparent molecular masses in analyses by sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) agreed with the calculated masses of the proteins (RfbD 42.0 kDa; His-tag RfbD 44.5 kDa; RfbE 40.9 kDa; His-tag RfbE 43.4 kDa). Crude ribosome-free extracts from the recombinant *E. coli* strains were preliminarily tested for the activities of GDP-D-mannose 4,6-dehydratase and of GDP-4-keto-6-deoxy-D-mannose 4-aminotransferase (GDP-D-perosamine synthetase). The recombinant native RfbD protein and the His-tag fusion protein variant of RfbD did not show any activity. Therefore the recombinant GDP-D-mannose 4,6-dehydratase (Gmd; plasmid pCAW21.1) of *E. coli* K12 was used.

| V.chole. RfbE | M--IPVYE--PSLDGNERKYLNDCIDSGWSSRGKY-IDRFETEFAEBFLK |
| B.melit. PerA | M-DIPVYS--PYLCGNFKKYVNECLDTGWISSRGEF-ISRFEDAFAYVD    |
| S.lincol LmbS  | MSDYIPFAA-PCFDTAEEEAVLRVVRSGWVSTGAEA--QSFEEEFAYIG   |
| S.griseus StrS | MSSFQELPRWPQLTDDDIEAAVAALRSNRLVGQGNSTVEEFEAALAAGQQ  |
|                | **\***\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\used; it had previously already been shown to catalyze the synthesis of GDP-4-keto-6-deoxy-D-mannose (Sturla *et al.*, 1997; Albermann *et al.*, 2000). This finding was surprising, because the amino acid sequence of the RfbD protein (*V. cholerae* O1) and the Gmd protein (*E. coli* K12) have 79% identity and 90% homology. In contrast, both the native RfbE protein and its His-tag fusion variant were catalytically active in the conversion of GDP-4-keto-6-deoxy-D-mannose to GDP-D-perosamine.

**Purification and properties of the His-tag fusion protein RfbE**

After harvesting of *E. coli* BL21 (DE3) pLysS cells containing the plasmid pCAW14.2, a crude extract was prepared and chromatographed on a Ni-NTA-agarose column, which bound the His-tag RfbE fusion protein tightly. The elution of the recombinant protein from the affinity column was accomplished by increasing the imidazole concentration up to 250 mM. The imidazole was removed from the protein fraction by gel filtration. The eluted fractions each contained one major band in an SDS-PAGE analysis (Figure 3) consistent with the calculated molecular masses of His-tag Gmd (44.5 kDa) and of His-tag RfbE (43.4 kDa), respectively.

A gel permeation analysis was used to investigate the molecular mass of the His-tag RfbE fusion protein under native conditions. The apparent mass of 170 kDa, measured by comparison of the elution volume of the His-tag RfbE peak with those of known molecular weight standards, suggested that the native protein is a tetramer.

RfbE protein from *V. cholerae*

**Determination of enzyme specificity and specific activity**

The conversion of GDP-α-D-mannose to GDP-4-keto-6-deoxy-amnnose and futher to GDP-D-perosamine was monitored by high-performance liquid chromatography (HPLC) (Figure 4). In coupled reaction assays, with recombinant Gmd and RfbE or His-tag RfbE, the complete conversion of GDP-α-D-mannose to GDP-D-perosamine could be obtained. As an amino-donor out of the tested alternatives the amino acid L-glutamate was used as substrate of the RfbE protein. From the other tested aminodonors (L-alanine, L-arginine, L-aspartate, L-glycine, L-glutamine, L-leucine, L-lysine, L-ornithine, L-phenyl-alanine, L-serine) only L-glutamine showed activity, for which the conversion rate was only 10% of the activity compared with L-glutamate. It was also shown that the addition of the cosubstrate pyridoxal-5-phosphate was essential for catalysis of the aminotransfer reaction by RfbE. The specific activities of the crude extract of *E. coli* BL21(DE3) pLysS pCAW 14.2 and of the purified His-tag fusion protein under the given reaction conditions were 8 and 44 nkat/mg protein, respectively. This indicated approximately a 6.5-fold enrichment of the purified protein. In the negative controls, that is, crude extracts from *E. coli* BL21 (DE3) pLysS with either vector plasmid, pET11a, or pET16b, no activity of either GDP-D-mannose 4,6-dehydratase or GDP-D-perosamine synthetase could be detected under the same conditions.

For the determination of the kinetic properties the concentration of GDP-4-keto-6-deoxy-D-mannose was measured by the UV absorption in alkaline medium at 320 nm. The \(K_{\text{m}}\) and \(V_{\text{max}}\) values for the substrate GDP-4-keto-6-deoxy-D-mannose were 0.06 mM and 38 nkat/mg protein, respectively. The \(K_{\text{m}}\) and \(V_{\text{max}}\) values for the cosubstrate L-glutamate were 0.1 mM and 42 nkat/mg protein, respectively. The reaction catalyzed by the His-tag RfbE protein had its pH optimum at 8.0, and the optimum concentration of the divalent cation \(\mathrm{Mg}^{2+}\) was 20 mM.

**Preparative synthesis and isolation of GDP-α-D-perosamine**

To identify the product of the RfbE reaction, we used the overproduced enzyme GDP-D-mannose 4,6-dehydratase (Gmd) and the purified GDP-D-perosamine synthetase (His-tag RfbE) for a preparative conversion of GDP-α-D-mannose. No product inhibition of the GDP-D-mannose 4,6-dehydratase by

Fig. 3. Purification of the His-tag RfbE protein by Ni-NTA-agarose. The SDS–PAGE analysis of the following protein fractions is shown in the lanes: (1) Low molecular weight marker; (2) S30 of *E. coli* BL21 pLysS pCAW14.2 (His tag RfbE); (3) wash step 1 with 20 mM imidazole; (4) wash step 2 with 50 mM imidazole; (5) His-tag RfbE after purification on Ni-NTA-agarose and gel filtration; (6) elution step with 250 mM imidazole; (7) S30 of *E. coli* BL21 pLysS pCAW14.1 (native RfbE).

Fig. 4. HPLC analysis of the enzymatic reactions catalyzed by Gmd and RfbE. (A) Enzyme assay after 1 min of incubation, GDP-α-D-perosamine (7 min), GDP-α-D-mannose (24 min), GDP-4-keto-6-deoxy-D-mannose (37 min). (B) Enzyme assay after complete conversion, GDP-α-D-perosamine (7 min).
C. Albermann and W. Piepersberg

GDP-D-perosamine could be observed. Such a one-pot method was applied to the complete synthesis of GDP-D-perosamine. The synthesis was started from GDP-α-D-mannose and the formation of the product GDP-D-perosamine and the consumption of the precursors, GDP-α-D-mannose and GDP-4-keto-6-deoxy-mannose, was monitored by HPLC and of L-glutamate by thin-layer chromatography (TLC). After removal of proteins and separation of the side product, α-ketoglutarate, by ion exchange chromatography and size exclusion chromatography the final product GDP-D-perosamine was obtained in 70% overall yield. The result of ¹H-nuclear magnetic resonance (NMR); ¹H¹H-COSY; ³¹P-NMR spectroscopy (Table I), of the mass spectrometry (MS) (GDP-α-D-perosamine sodium salt [(M+Na)+] 611.13 amu) and the TLC analysis of the D-perosamine component proved the structure of the product to be in accord with GDP-α-D-perosamine.

**Discussion**

The intention of this work was to identify the function of the RfbE protein by an overexpression of the *rfbE* gene and an *in vitro* production of GDP-α-D-perosamine. As yet, the specific function of a GDP-D-perosamine-synthetase has not been directly proven by *in vitro* assays. Rather it could been shown by mutational studies that the encoded proteins are involved in the biosynthesis of the LPS antigens (Hisatsune

**Table I.** ¹H-, ³¹P-NMR, chemical shifts, and coupling for GDP-D-perosamine

| Residue          | ¹H/³¹P     | Chemical shift (p.p.m.) | Coupling constants (Hz) |
|------------------|------------|-------------------------|--------------------------|
| α-D-Perosamine   | H-1"       | 5.41                    | ³J₁₂" 1.8               |
|                  |            |                         | ³JₚH₁" 7.1              |
|                  | H-2"       | 3.97                    | ³J₂₃" 3.5               |
|                  |            |                         | ³J₁₂" 1.8               |
|                  | H-3"       | 3.94                    | ³J₃₄" 10.1              |
|                  |            |                         | ³J₂₃" 3.5               |
|                  | H-4"       | 3.01                    | ³J₃₄" 10.1              |
|                  |            |                         | ³J₄₅" 9.8               |
|                  | H-5"       | 4.08                    | ³J₅₆" 6.2               |
|                  |            |                         | ³J₄₅" 9,8               |
|                  | H-6"(CH₃)  | 1.21                    | ³J₅₆" 6.2               |
| Ribose           | H-5'a,b    | 4.20                    |                          |
|                  | H-4'       | 4.31                    |                          |
|                  | H-3'       | 4.45                    | ³J₂₃' 3.7               |
|                  | H-2'       | 4.70                    | ³J₁₂' 6.1               |
|                  |            |                         | ³J₂₃' 3.7               |
|                  | H-1'       | 5.90                    | ³J₁₂' 6.1               |
| Base             | H-1'       | 8.10                    |                          |
| Phosphate        | P-1        | -10.1                   | ³Jₚₚ 19.6               |
|                  | P-2        | -12.6                   | ³Jₚₚ 19.6               |
|                  |            |                         | ³JₚH₁" 7.1              |

et al., 1996; Godfroid et al., 1998; Cloeckaert et al., 2000). The RfbE protein shows homology to pyridoxal-phosphate-binding proteins (COG0399; Tatusov et al., 2001), which in part catalyze the aminogroup transfer onto deoxysugars, like NysDII (3-amino-3,6-dideoxy-D-mannose, mycosamine) in *Streptomyces noursei* (Brautaset et al., 2000). Therefore, as the required tools we herein evaluate the overexpressed D-perosamine biosynthetic enzymes RfbD and RfbE from *V. cholerae* O1, as well as the purification of GDP-D-perosamine-synthetase RfbE to homogeneity and its characterization. Surprisingly, the native and the His-tag RfbD protein were enzymatically inactive, even though they could be over-expressed in *E. coli* BL21 pLysS as soluble proteins. This is even though the amino acid sequence of the RfbD protein (*V. cholerae*) and the Gmd protein (*E. coli*) are 79% identical, indicating a high degree of structural congruence in the 3D structure. However, also in case of the Gmd protein of *E. coli* it is known that the enzyme is unstable (Sturla et al., 1997) and that the N-terminal His-tag fusion variant of the Gmd protein is also inactive (Albermann et al., 2000). Nevertheless, the recombinant Gmd protein from *E. coli* K12 catalyzed the formation of GDP-D-mannose to GDP-4-keto-6-deoxymannose (Sullivan et al., 1998; Albermann et al., 2000). The GDP-4-keto-6-deoxymannose, which can be synthesized this way, is used as the substrate of the GDP-4-keto-6-deoxy-D-mannose 4-amino-transferase (perosamine synthetase; RfbE). The recombinant RfbE proteins, the native and the His-tag fusion proteins, respectively, converted GDP-4-keto-6-deoxymannose in presence of L-glutamate to a new product. The NMR and MS analyses and the results of the chromatography supported that this product of the Gmd and RfbE catalyzed reactions is GDP-α-D-perosamine. The *in vitro* preparation could be carried out efficiently in a one-step (“one-pot”) reaction system. In contrast to GDP-β-L-fucose (Sturla et al., 1997), the GDP-α-D-perosamine did not show any effect of product inhibition on the activity of the GDP-D-mannose 4,6-dehydratase. Whether the GDP-α-D-perosamine exerts feedback inhibition of other biosynthetic enzymes, for instance, RfbA, RfbB, and RfbD, of *V. cholerae* O1 will be the subject of further investigations.

In the present work, we have established a preparative enzymatic route to the synthesis of GDP-α-D-perosamine by use of the two recombinant enzymes, Gmd and RfbE, which can easily be used now for the production of GDP-α-D-perosamine, at gram scale.

**Materials and methods**

**Biochemicals**

All protein standards, antibiotics, isopropyl-β-thiogalactoside (IPTG), and nucleotides were obtained from Sigma (Deisenhofen, Germany); GDP-α-D-mannose was obtained from Calbiochem (Schwalbach, Germany).

**Bacterial strains, growth conditions, and media**

The bacterial strains used in this study were *E. coli* DH5α and *E. coli* BL21 (DE3) pLysS. Strains of *E. coli* were grown at 37°C in Luria-Bertani broth (LB) (Miller, 1972); for solid media agar-agar in a concentration of 18 g/L was added. Antibiotics were used at the following final concentrations: ampicillin 100 μg/ml, chloramphenicol 25 μg/ml. For induc-

tion of the lac operator IPTG was used at a final concentration of 24 μg/ml.

**Cloning of rfbD and rfbE**

The DNA-sequence of the rfb-gene cluster for lipopolysaccharide biosynthesis in V. cholerae O1 has been described by others (Manning et al., 1995). The genes rfbD and rfbE were amplified by PCR with Vent-polymerase (New England BioLabs, Schwalbach/Taunus, Germany). The following primers, used to amplify the genes rfbD and rfbE from the chromosomal DNA of V. cholerae O1 were designed on the basis of the published sequences and were obtained from Interactiva (Ulm, Germany): for the rfbD gene 5′-GGATATTTACATATGA-ATAAAAAAAGTTG-3′ (forward direction); 5′-CAGGAATC-ATTAAAAGATCTCACTCTAC-3′ (reverse direction); for rfbE 5′-GTGAGGTCCTCATATGATTCCTGTAT-3′ (forward direction); 5′-GGAGGTAAGGGATCCCAAACACTACTA-3′ (reverse direction). The products of the PCR amplification were further treated with NdeI and BglII restriction enzymes (Gibco BRL, Eggenstein, Germany) in the case of the rfbD fragment and with NdeI and BamHI for the rfbE PCR product, and were ligated into the vectors pET11a and pET16b (Novagen, Madison, WI), which were hydrolyzed with NdeI and BamHI, respectively (Sambrook et al., 1989). After ligation with T4-DNA-ligase (BioLabs, Schwalbach/Taunus), the recombinant plasmids pET11a/rfbD (pCAW 13.1), pET11a/rfbE (pCAW 14.1), pET16b/rfbD (pCAW 13.2), and pET16b/rfbE (pCAW 14.2) were obtained and transformed into competent cells of E. coli DH5α (Hanahan, 1983). The respective cloning of the gmd gene from E. coli K12 in plasmid pCAW21.1 had been published previously (Albermann et al., 2000).

**DNA sequencing**

The insert structure of each recombinant derivative was verified by restriction analysis and DNA sequencing. The sequence of the amplified DNA was determined by the dideoxynucleotide chain-termination method (Sanger et al., 1977) using the Thermo Sequenase fluorescent labeled primer cycle sequencing kit with 7-deaza-dGTP (Amersham, Braunschweig, Germany) and an A.L.F.-Express DNA sequencer (Pharmacia, Freiburg, Germany). The sequences of both strands each were determined from the DNA of double-stranded plasmids prepared by the QIAprep spin miniprep kit (Qiagen, Hilden, Germany).

**Overproduction of GDP-D-mannose 4,6-dehydratase (RfbD, Gmd) and GDP-4-keto-6-deoxy-D-mannose-4-aminotransferase (RfbE)**

The recombinant plasmids (pCAW 13.1, pCAW 13.2, pCAW 14.1, pCAW 14.2, pCAW 21.1) were retransformed into competent E. coli BL21 (DE3) pLysS (Noreagen). The E. coli BL21 (DE3) pLysS cells with the recombinant plasmids were grown in LB media to an optical density of 0.6 at 540 nm. Then the cells were induced by IPTG for 90 min. Subsequently, the cells were harvested by centrifugation, washed twice in ice-cold 50 mM Tris-HCl buffer, pH 8, and suspended in extraction buffer (50 mM Tris-HCl buffer, pH 8, 150 mM NaCl, 20 mM MgCl₂, 5 mM β-mercaptoethanol, 5 mM pyridoxal-5-phosphate). After disruption by using a 5-ml French press cell (Aminco, Silver Spring, MD) the crude extract was clarified by centrifugation at 30,000 × g for 30 min (S30).

---

**Purification of RfbE-His-tag fusion protein**

The RfbE His-tag fusion-protein was purified as soluble protein by affinity chromatography, using Ni-NTA-agarose (Qiagen) (Le Grice and Grueninge-Leitch, 1990; Schmitt et al., 1993). The crude extract (containing His-tag RfbE) was loaded on a Ni-NTA-agarose-column (15 mm × 60 mm) which was equilibrated with 50 mM Tris-HCl buffer, pH 8, with 300 mM NaCl, 10 mM MgCl₂, and 5 mM β-mercaptoethanol. The loaded column was washed with 50 mM Tris-HCl buffer, pH 8, containing 300 mM NaCl, 10 mM MgCl₂, 5 mM β-mercaptoethanol, and 20 mM imidazole. After the elution with 50 mM Tris-HCl buffer, pH 8, with 300 mM NaCl, 10 mM MgCl₂, 5 mM β-mercaptoethanol, 250 mM imidazole, the protein-containing fractions were analysed by SDS-PAGE (Laemmli, 1970) to check their purity and molecular masses. The final purification step was carried out by gel filtration with a Sepharyl S-200HR column (2.5 × 90 cm) (Pharmacia). The column was equilibrated with 50 mM Tris-HCl buffer, pH 8, with 10 mM MgCl₂, 5 mM β-mercaptoethanol, and 300 mM NaCl.

**Characterization of recombinant His-tag RfbE protein**

The size determination of the native His-tag RfbE was performed by size exclusion chromatography on an HPLC system (Pharmacia) with a Superose 12 gel filtration column (Pharmacia). A 50 mM Tris-HCl buffer, pH 8, containing 150 mM NaCl was used as mobile phase, at a flow rate 0.5 ml/min. For the calibration curve the following proteins were applied: ovalbumin 43 kDa; bovine serum albumin 67 kDa; alcohol dehydrogenase of yeast 150 kDa; β-amylase of potato 200 kDa; apoferritin from horse spleen 443 kDa; bovine thyroglobulin 669 kDa.

**HPLC**

GMP, GDP, GTP, GDP-α-D-mannose, GDP-4-keto-6-deoxy-D-mannose and GDP-α-D-perosamine were separated by HPLC and detected by UV photometry (Beckman, Munich, Germany) at 260 nm. As stationary phase a reversed phase column, Eurospher ODS18, 5 µm, 250 × 4.6 mm (Knauer, Berlin, Germany) was used. As mobile phase a phosphate buffer (30 mM potassium phosphate, pH 6.0; 5 mM tetrabutyl-ammonium hydrogensulfate, 2% acetonitrile) and acetonitrile was used (Payne and Ames, 1982).

**TLC**

To analyze the sugar components, the nucleotide moieties were cleaved from the sugar by incubating in 1 M HCl for 10 min at 100°C. The resulting mixture and the standards of 6-deoxy-L-mannose, 6-deoxy-L-galactose, D-glucose, 2-amino-2-deoxy-D-glucose, 2-amino-2-deoxy-D-galactose, 2-amino-2-deoxy-D-mannose, and D-mannose were chromatographed on TLC sheets, Silica gel F₂₅₄ (Merck, Darmstadt, Germany), using a solvent system consisting of acetic acid:chloroform:water (7:6:1). All the sugar components were localized by naphthoresorcinol staining (0.2 g naphthoresorcinol in 96 ml ethanol and 4 ml concentrated sulfuric acid), aminosugars by Morgan-Elson reagent (Chaplin and Kennedy, 1986). L-glutamate was chromatographed by the same mobile and stationary phase (see above) and detected by staining with ninhydrine (0.3 g ninhydrine in 100 ml butanol).

C. Albermann and W. Piepersberg

### Enzyme assays

For the measurement of the activity of the GDP-D-mannose 4,6-dehydratase (Gmd, RfbD) each a spectroscopic (Okazaki *et al.*, 1962; Kornfeld and Ginsburg, 1966) and chromatographic test system was used to estimate the increase of GDP-4-keto-6-deoxy-D-mannose. The standard assay contained 50 mM Tris–HCl buffer, pH 8, 10 mM MgCl₂, 4 mM GDP-D-mannose, 50 μM NADP⁺, and different amounts of GDP-D-mannose 4,6-dehydratase containing crude extract in a final volume of 100 μl. Protein concentrations were determined according to Bradford (1976). The reactions were performed at 37°C and measured at different times between 0 and 30 min. For the photometric analysis, the reactions were stopped by adding 950 μl 100 mM NaOH to 50 μl of the reaction mixtures; the solutions were further incubated for 20 min at 37°C. After this, the absorption was measured at 320 nm (ε₃₂₀ₙₘ = 2200 L mol⁻¹ cm⁻¹). For the HPLC analysis the samples were boiled for 1 min, and the proteins were removed by centrifugation. The enzyme activity of the GDP-4-keto-6-deoxy-D-mannose 4-aminotransferase RfbE was determined by measuring the decrease of GDP-4-keto-6-deoxy-D-mannose (see above). GDP-4-keto-6-deoxy-D-mannose was obtained by a previously published procedure (Albermann *et al.*, 2000). The standard assay contained 50 mM Tris–HCl buffer, pH 8, 2 mM GDP-4-keto-6-deoxy-D-mannose, 20 mM MgCl₂, 50 μM pyridoxal-5-phosphate, and different amounts of RfbE or His-tag RfbE and 4 mM sodium-L-glutamate. Other tested aminodonor cosubstrates (L-alanine, L-arginine, L-aspartate, L-glycine, L-glutamine, L-leucine, L-lysine, L-ornithine, L-phenylalanine, L-serine) were also added at 4 mM final concentration. The reaction was started by the addition of different concentrations of GDP-4-keto-6-deoxy-D-mannose (final volume 50 μl). Also, the increase of GDP-α-D-perosamine was observed by HPLC analysis.

### Enzymatic synthesis of GDP-α-D-perosamine

The preparative synthesis of GDP-α-D-perosamine was carried out in a coupled enzymatic reaction by recombinant Gmd from *E. coli* K12 (plasmid pCAW21.1; Albermann *et al.*, 2000) and the purified His-tag RfbE protein. The assay contained 33 μmol (20 mg) GDP-D-mannose, sodium-L-glutamate 38 μmol (7.1 mg), 50 mM Tris–HCl buffer, pH 7.5, 20 mM MgCl₂, 50 μM pyridoxal phosphate, 50 μM NADP⁺, crude extract of *E. coli* BL21 pLysS pCAW21.1 and His-tag RfbE, with a GDP-D-mannose 4,6-dehydratase and GDP-4-keto-6-deoxy-D-mannose 4-aminotransferase activity each of 25 nkat in a final volume of 3 ml. This mixture was incubated for 60 min at 37°C. The proteins were removed by boiling for 1 min and subsequent centrifugation at 10,000 × g for 30 min. The time course of the reaction to GDP-α-D-perosamine was monitored by assaying small samples with HPLC analysis.

### Isolation of GDP-α-D-perosamine

The separation of GDP-α-D-perosamine from the side products α-ketoglutarate and L-glutamate was done by an anion-exchange chromatography. This was performed with a Dowex 1 × 8 resin (mesh 200–400; formate form) (Serva, Heidelberg, Germany), elution with a linear gradient of ammonium-formate solution from 0.05 M up to 1 M in 2 h at a flow rate of 5 ml/min. GDP-α-D-perosamine-containing fractions were pooled and evaporated to a volume of 8 ml under reduced pressure (20 mbar) at 20–25°C. For the desalting of the preparation a gel filtration on a Sephadex G-10 column (SR 25/100; Pharmacia), with a total volume of 420 ml, was used and eluted at a flow rate of 1 ml/min. The GDP-α-D-perosamine-containing fractions were detected by UV absorption, pooled, and applied to a membrane anion exchanger Q15 (Sartorius, Göttingen, Germany) by which the GDP-α-D-perosamine preparation was equilibrated against 150 mM NaCl to yield the sodium form. After another volume reduction by evaporation to a volume of about 10 ml and a further gel filtration (Sephadex G-10 column), the sodium GDP-α-D-perosamine was lyophilized (Cryograph LCD-1, Christ, Osterode, Germany) (yield 14 mg).

The enzymatically synthesized GDP-α-D-perosamine was analyzed by NMR spectroscopy: ¹H-NMR (400 MHz, D₂O) δ (p.p.m.) 1.20 (d, H-6″ ³J₅″₆″ = 6.2 Hz); 3.01 (dd, H-4″ ³J₃″₄″ = 10.1 Hz, ³J₄″₅″ = 9.8 Hz); 3.94 (dd, H-3″ ³J₃″₄″ = 10.1 Hz, ³J₂″₃″ = 3.5 Hz); 3.97 (dd, H-2″ ³J₂″₃″ = 3.5 Hz, ³J₁″₂″ = 1.8 Hz); 4.08 (m, H-5″ ³J₅″₆″ = 6.2 Hz, ³J₄″₅″ = 9.8 Hz); 5.41 (H-1″ ³J₁″₂″ = 1.8 Hz, ³J_PH₁″ = 7.1 Hz); 4.20 (2H-5′); 4.31 (m, H-4′); 4.45 (dd, H-3′ ³J₂′₃′ = 3.7 Hz); 4.70 (dd, H-2′ ³J₁′₂′ = 6.1 Hz, ³J₂′₃′ = 3.7 Hz); 5.90 (dd, H-1′ ³J₁′₂′ = 6.1 Hz); 8.10 (s, H-1). ³¹P-NMR (160 MHz, D₂O) δ (p.p.m.) –10.1 (P-1 ²J_PP = 19.6 Hz); –12.6 (P-2 ²J_PP = 19.6 Hz, ³J_P-2H-1″ = 7.1 Hz).

### Acknowledgments

We thank S. Kuberski for expert technical assistance. We are grateful to J. Hacker for providing DNA of *V. cholerae* O1 and F. Wedekind for MS analysis. This work was supported by a grant from Roche Diagnostics, Penzberg (Germany).

### Abbreviations

HPLC, high performance liquid chromatography; IPTG, isopropyl-β-thiogalactoside; LB, Luria-Bertani; LPS, lipopolysaccharide; MS, mass spectrometry; NMR, nuclear magnetic resonance; PCR, polymerase chain reaction; SDS–PAGE, sodium dodecyl sulfate–polyacrylamide gel electrophoresis; TLC, thin-layer chromatography.

### References

Albermann, C., Distler, J., and Piepersberg, W. (2000) Preparative synthesis of GDP-beta-L-fucose by recombinant enzymes from enterobacterial sources. *Glycobiology*, **10**, 875–881.

Bilge, S.S., Vary, J.C., Dowell, S.F., and Tarr, P.I. (1996) Role of the *Escherichia coli* O157:H7 O side chain in adherence and analysis of an rfb locus. *Infect. Immun.*, **64**, 4795–4801.

Bradford, M.M. (1976) A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. *Anal. Biochem.*, **72**, 248–254.

Brautaset, T., Sekurova, O.N., Sletta, H., Ellingsen, T.E., Strm, A.R., Valla, S., and Zotchev, S.B. (2000) Biosynthesis of the polyene antifungal antibiotic nystatin in *Streptomyces noursei* ATCC 11455: analysis of the gene cluster and deduction of the biosynthetic pathway. *Chem. Biol.*, **7**, 395–403.

Chaplin, M.F., and Kennedy, J.F. (1986) *Carbohydrate analysis: a practical approach*. IRL Press, Oxford.

Cloeckaert, A., Grayon, M., Verger, J.M., Letesson, J.J., and Groidfroid, F. (2000) Conservation of seven genes involved in the biosynthesis of the lipopolysaccharide O-side chain in *Brucella spp.* *Res. Microbiol.*, **151**, 209–216.

Elling, L., and Büttler, T. (1999) Enzymatic synthesis of nucleotide sugars. Glycoconj. J., 16, 147–159.

Elling, L., Ritter, J.E., and Verseck, S. (1996) Expression, purification and characterization of recombinant phosphomannomutase and GDP-alpha-D-mannose pyrophosphorylase from *Salmonella enterica*, group B, for the synthesis of GDP-alpha-D-mannose from D-mannose. Glycobiology, 6, 591–597.

Godfroid, F., Taminiau, B., Danese, I., Denoel, P., Tibor, A., Weynants, V., Cloeckaert, A., Godfroid, J., and Letesson, J.J. (1998) Identification of the perosamine synthetase gene of *Brucella melitensis* 16M and involvement of lipopolysaccharide O side chain in *Brucella* survival in mice and in macrophages. Infect. Immun., 66, 5485–5493.

Hanahan, D. (1983) Studies on transformation of *Escherichia coli* plasmids. J. Mol. Biol., 127, 100–104.

Hisatsune, K., Kondo, S., Isshiki, Y., Iguchi, T., and Haishima, Y. (1993) Occurrence of 2-O-methyl-N-(3-deoxy-L-glycero-tetronyl)-D-perosamine (4-amino-4, 6-dideoxy-D-manno-pyranose) in lipopolysaccharide from Ogawa but not from Inaba O forms of O1 *Vibrio cholerae*. Biochem. Biophys. Res. Commun., 190, 302–307.

Hisatsune, K., Kondo, S., Iguchi, T., Ito, T., and Hiramatsu, K. (1996) Lipopolysaccharides of *Escherichia coli* K12 strains that express cloned genes for the Ogawa and Inaba antigens of *Vibrio cholerae* O1: identification of O-antigenic factors. Microbiol. Immunol., 40, 621–626.

Kornfeld, R.H., and Ginsburg, V. (1966) Control of synthesis of guanosine 5′-diphosphate D-mannose and guanosine 5′-diphosphate L-fucose in bacteria. Biochim. Biophys. Acta, 117, 79–87.

Laemmli, U.K. (1970) Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature, 277, 680–685.

Le Grice, S.F., and Grueninger-Leitch, F. (1990) Rapid purification of homodimer and heterodimer HIV-1 reverse transcriptase by metal chelate affinity chromatography. Eur. J. Biochem., 187, 307–314.

Lee, C.H., and Schaffner, C.P. (1966) Perimycin: chemistry of perosamine. Tetrahedron Lett., 47, 5837–5840.

Lee, C.H., and Schaffner, C.P. (1969) Perimycin: the structure of some degradation products. Tetrahedron, 25, 2229–2232.

Liu, H.-W., and Thorson, J.S. (1994) Pathways and mechanisms in the biogenesis of novel deoxysugars by bacteria. Ann. Rev. Microbiol., 48, 223–256.

Manning, P.A., Stroeher, U.H., Karageorgos, L.E., and Morona, R. (1995) Putative O-antigen transport genes within the rfb region of *Vibrio cholerae* O1 are homologous to those for capsule transport. Gene, 158, 1–7.

Miller, J.H. (1972) Experiments in molecular genetics. Cold Spring Harbor Laboratory, New York.

Okazaki, R., Okazaki, T., Strominger, J.T., and Michelson, A.M. (1962) Thymidine diphosphate 4-keto-6-deoxy-D-glucose, an intermediate in thymidine diphosphate L-rhamnose synthesis in *Escherichia coli* strains. J. Biol. Chem., 237, 3014–3026.

Pastuszak, I., Ketchum, C., Hermanson, G., Sjoberg, E.J., Drake, R., and Elbein, A.D. (1998) GDP-L-fucose pyrophosphorylase. Purification, cDNA cloning, and properties of the enzyme. J. Biol. Chem., 273, 30165–30174.

Payne, S.H., and Ames, B.N. (1982) A procedure for rapid extraction and high-pressure liquid chromatographic separation of the nucleotides and other small molecules from bacterial cells. Anal. Biochem., 123, 151–161.

Piepersberg, W. (1994) Pathway engineering in secondary metabolite-producing actinomycetes. Crit. Rev. Biotechnol., 14, 251–285.

Piepersberg, W., and Distler, J. (1997) Aminoglycosides and sugar components in other secondary metabolites. In Rehm, H.-J., Reed, G., Pühler, A., and Stadler, P., eds., Biotechnology, 2nd ed., vol. 7. VCH-Verlagsgesellschaft, Weinheim, pp. 397–488.

Sambrook, J., Fritsch, E.F., and Maniatis, T. (1989) Molecular cloning: a laboratory manual. Cold Spring Harbor Laboratory, New York.

Sanger, F., Nicklen, S., and Coulson, A.R. (1977) DNA sequencing with chain-terminating inhibitors. Proc. Natl Acad. Sci. USA, 74, 5463–5467.

Schmitt, J., Hess, H., and Stunneberg, H.G. (1993) Affinity purification of histidine-tagged proteins. Mol. Biol. Rep., 18, 223–230.

Stroeher, U.H., Karageorgos, L.E., Morona, R., and Manning, P.A. (1992) Serotype conversion in *Vibrio cholerae* O1. Proc. Natl Acad. Sci. USA, 89, 2566–2570.

Stroeher, U.H., Karageorgos, L.E., Brown, M.H., Morona, R., and Manning, P.A. (1995) A putative pathway for perosamine biosynthesis is the first function encoded within the rfb region of *Vibrio cholerae* O1. Gene, 166, 33–42.

Stroeher, U.H., Jedani, K.E., and Manning, P.A. (1998) Genetic organization of the regions associated with surface polysaccharide synthesis in *Vibrio cholerae* O1, O139 and *Vibrio anguillarum* O1 and O2: a review. Gene, 223, 269–282.

Sturla, L., Bisso, A., Zanaridi, D., Benatti, U., De Flora, A., and Tonetti, M. (1997) Expression, purification and characterization of GDP-D-mannose 4,6-dehydratase from *Escherichia coli*. FEBS Lett., 412, 126–130.

Sullivan, F.X., Kumar, R., Kriz, R., Stahl, M., Xu, GY., Rouse, J., Chang, X.J., Boodhoo, A., Potvin, B., and Cumming, D.A. (1998) Molecular cloning of human GDP-mannose 4,6-dehydratase and reconstitution of GDP-fucose biosynthesis in vitro. J. Biol. Chem., 273, 8193–8202.

Tatusov, R.L., Natale, D.A., Garkavtsev, I.V., Tatusova, T.A., Shankavaram, U.T., Rao, B.S., Kiryutin, B., Galperin, M.Y., Fedorova, N.D., and Koonin, E.V. (2001) The COG database: new developments in phylogenetic classification of proteins from complete genomes. Nucleic Acids Res., 29, 22–28.

Villeneuve, S., Souchon, H., Riottot, M.-M., Mazie, J.C., Lei P., Glademans, C.P., Kovac, P., Fournier, J.M., and Alzari, P.M. (2000) Crystal structure of an anti-carbohydrate antibody directed against *Vibrio cholerae* O1 in complex with antigen: molecular basis for serotype specificity. Proc. Natl Acad. Sci. USA, 97, 8433–8438.

Wu, A.M., and Mackenzie, N.E. (1987) Structural and immunochemical characterization of the O-haptens of *Brucella abortus* lipopolysaccharides from strains 19 and 2308. Mol. Cell. Biochem., 75, 103–111.
