
Reciprocal Regulation Between TOC1 and LHY/CCA1 Within the Arabidopsis Circadian Clock

David Alabadi? et al.

Science 293, 880 (2001);
DOI: 10.1126/science.1061320

This copy is for your personal, non-commercial use only.

If you wish to distribute this article to others, you can order high-quality copies for your colleagues, clients, or customers by clicking here.

Permission to republish or repurpose articles or portions of articles can be obtained by following the guidelines here.

The following resources related to this article are available online at www.sciencemag.org (this information is current as of February 24, 2013):

Updated information and services, including high-resolution figures, can be found in the online version of this article at:
http://www.sciencemag.org/content/293/5531/880.full.html

Supporting Online Material can be found at:
http://www.sciencemag.org/content/suppl/2001/08/02/293.5531.880.DC1.html

This article cites 19 articles, 13 of which can be accessed free:
http://www.sciencemag.org/content/293/5531/880.full.html#ref-list-1

This article has been cited by 285 article(s) on the ISI Web of Science

This article has been cited by 100 articles hosted by HighWire Press; see:
http://www.sciencemag.org/content/293/5531/880.full.html#related-urls

This article appears in the following subject collections:
Physiology
http://www.sciencemag.org/cgi/collection/physiology

Science (print ISSN 0036-8075; online ISSN 1095-9203) is published weekly, except the last week in December, by the American Association for the Advancement of Science, 1200 New York Avenue NW, Washington, DC 20005. Copyright 2001 by the American Association for the Advancement of Science; all rights reserved. The title Science is a registered trademark of AAAS.

similarity to a Thr → Ile change in v-Src versus c-Src at position 338, which corresponds to Thr³¹⁵ in c-Abl. Despite the fact that v-Src and c-Src have almost identical kinase domain sequences (98% identity), v-Src is about 50-fold more resistant than c-Src to kinase inhibition by the Src inhibitor PP1 (35).

Although the development of STI-571 resistance presents new therapeutic challenges, the fact that BCR-ABL remains active in STI-571–resistant cells suggests that the chimeric oncprotein remains a rational drug target. Because several patients examined to date share an identical mutation associated with drug resistance, it may be possible to identify an inhibitor of the mutant BCR-ABL allele that would have broad utility. In addition, knowledge of this mutation should permit the development of assays to detect drug-resistant clones before clinical relapse. It should be noted that this study has focused on a small, selected group of patients. Analyses of larger sample sizes are required to determine the true frequency of this mutation among resistant patients.

**References and Notes**

1. L. Chin *et al.*, Nature **400**, 468 (1999).
2. D. W. Felsher, J. M. Bishop, Mol. Cell **4**, 199 (1999).
3. C. S. Huettnert *et al.*, Nature Genet. **24**, 57 (2000).
4. B. J. Druker *et al.*, N. Engl. J. Med. **344**, 1038 (2001).
5. B. J. Druker *et al.*, N. Engl. J. Med. **344**, 1031 (2001).
6. C. L. Sawyers, N. Engl. J. Med. **340**, 1330 (1999).
7. S. Faderl *et al.*, N. Engl. J. Med. **341**, 164 (1999).
8. J. B. Konopka *et al.*, Proc. Natl. Acad. Sci. U.S.A. **82**, 1810 (1985).
9. G. Q. Daley *et al.*, Science **247**, 824 (1990).
10. N. Heisterkamp *et al.*, Nature **344**, 251 (1990).
11. B. J. Druker *et al.*, Nature Med. **2**, 561 (1996).
12. Peripheral blood and/or bone marrow samples were obtained with appropriate informed consent from CML and Ph-positive ALL patients at the University of California Los Angeles who were enrolled in multicenter clinical trials of STI-571 (sponsored by Novartis Pharmaceuticals, Basel, Switzerland). All patients had >30% blasts in the marrow before treatment. Responding patients had reduction in the percentage of bone marrow blasts to <15% (partial) or <5% (complete), as described (5). Progressive disease was defined as an increase in percentage of blasts after an initial response, despite continued STI-571 treatment. Mononuclear cells were isolated by centrifugation through Ficoll-Hypaque, washed twice in phosphate-buffered saline, counted, and used immediately or cryopreserved.
13. S. S. Clark *et al.*, Science **235**, 85 (1987).
14. S. S. Clark *et al.*, Science **239**, 775 (1988).
15. M. E. Gorre, C. L. Sawyers, unpublished data.
16. J. ten Hoeve *et al.*, Blood **84**, 1731 (1994).
17. T. Oda *et al.*, J. Biol. Chem. **269**, 22925 (1994).
18. G. L. Nichols *et al.*, Blood **84**, 2912 (1994).
19. K. Senechal *et al.*, J. Biol. Chem. **271**, 23255 (1996).
20. Cells were lysed in 1% Triton X-100 buffer with protease and phosphatase inhibitors (36). Equal amounts of protein, as determined by the BioRad DC protein assay (Bio-Rad, Hercules, CA), were separated by SDS-PAGE, transferred to nitrocellulose, and immunoblotted with phosphotyrosine antibody (4G10, Upstate Biotechnologies, Lake Placid, NY), Abl antibody [pex5, (36)], β-actin antibody (Sigma, St. Louis, MO), or Crkl antiserum (Santa Cruz Biotechnology, Santa Cruz, CA). Immunoreactive bands were visualized by ECL (Amersham Pharmacia Biotech, Piscataway, NJ). Several exposures were obtained to ensure linear range of signal intensity. Optimal exposures were quantified by densitometry with ImageQuant software (Molecular Dynamics, Sunnyvale, CA).
21. M. E. Gorre, C. L. Sawyers, data not shown.
22. C. Gambacorti-Passerini *et al.*, J. Natl. Cancer Inst. **92**, 1641 (2000).
23. Purified cells were plated at 1 × 10⁶/ml to 10 × 10⁶/ml in RPMI-1640 + 10% human AB serum with varying concentrations of STI-571 for 24 hours. Proteins were extracted and subjected to immunoblot analysis.
24. E. Weisberg, J. D. Griffin, Blood **95**, 3498 (2000).
25. P. le Coutre *et al.*, Blood **95**, 1758 (2000).
26. F. X. Mahon *et al.*, Blood **96**, 1070 (2000).
27. Interphase and metaphase cells were prepared (37) and examined with Locus Specific Identifier BCR-ABL dual color translocation probe (Vysis, Downers Grove, IL).
28. gDNA was extracted from purified bone marrow or peripheral blood cells with the QiaAMP Blood Mini Kit (Qiagen, Valencia, CA). We subjected 10 ng of total gDNA to real-time PCR analysis with the iCycler iQ system (Bio-Rad, Hercules, CA). A 361-bp gDNA fragment including ABL exon 3 was amplified with two primers (5′-CAGAGTCAGAATCCTTCAG-3′ and 5′-TTTGTAAGGCTGCCGGC-3′), which are specific for intron sequences 5′ and 3′ of ABL exon 3, respectively. A 472-bp gDNA fragment of glyceraldehyde-3-phosphate dehydrogenase (GAPDH) was amplified with two primers (5′-TTCACCACCATGGAGAAGGC-3′ and 5′-CAGGAAATGAGCTTGACAAA-3′), which are specific for sequences in exon 5 and exon 8 of GAPDH, respectively. Fold increase in ABL copy number was determined by calculating the difference between threshold cycle numbers of ABL and GAPDH for each sample (∆Ct). With control LB3 as reference sample, ∆Ct from each sample was subtracted from ∆Ct of control to determine ∆(∆Ct). Fold increase was calculated as 2⁻^(∆(∆Ct)).
29. RNA was extracted from purified peripheral blood or bone marrow cells with Trizol (Molecular Research Center, Cincinnati, OH). Two milligrams of total RNA was subjected to reverse transcriptase (RT)-PCR with Oligo dT primers. A 1327-bp cDNA fragment was amplified by PCR with a 5′BCR-specific primer (5′-GAAGCTTCTCCCTGGCATCCGT-3′) and a 3′ ABL-specific primer (5′-CCAGGCTCTCGGGGTG-CAGTCC-3′). In two patients, the BCR-ABL fragment could not be amplified; therefore, a 579-bp fragment was amplified with an alternative 5′ ABL-specific primer (5′-GCGCAACAAGCCCACTGTCTATGG-3′) and the same 3′ ABL primer. PCR products were cloned into the pCR2.1 TA cloning vector (Invitrogen, Carlsbad, CA). Both strands of a 579-bp region were sequenced with the 5′ ABL primer and M13 forward primer or M13 forward and reverse primer set for the 1327-bp and the 579-bp fragments, respectively, on an ABI prism 377 automated DNA sequencer (PE Applied Biosystems, Foster City, CA). Sequence analysis was performed with the ClustalW alignment algorithm (38).
30. gDNA was extracted from purified bone marrow or peripheral blood cells with the QiaAMP Blood Mini Kit (Qiagen). A 361-bp DNA fragment was amplified by PCR with two primers (5′-GCAGAGTCAGAATCCTTCAG-3′ and 5′-TTTGTAAGGCTGCCGGC-3′), which are specific for intron sequences 5′ and 3′ of ABL exon 3, respectively. PCR products were cloned and sequenced.
31. T. Schindler *et al.*, Science **289**, 1938 (2000).
32. The structure of the kinase domain of Hck in complex with an ATP analog (AMP-PNP) was superimposed onto the model of the Ile315 Abl kinase domain.
33. Full-length p210 BCR-ABL was subcloned into the pSRApSVtkNeo retrovirus vector (39). A fragment containing the C → T mutation at ABL nt 944 was made by PCR and swapped with the corresponding sequence in pSRApSVtkNeo p210 BCR-ABL wild-type to create the pSRApSVtkNeo p210 BCR-ABL T315I mutant. The resulting construct was confirmed by sequencing.
34. Transient transfection of 293T cells was performed with CaCl₂ (39). After a 24-hour transfection, cells were incubated with varying concentrations of STI-571 (Novartis) for 2 hours. Proteins were extracted and subjected to immunoblot analysis.
35. Y. Liu *et al.*, Chem. Biol. **6**, 671 (1999).
36. A. Goga *et al.*, Cell **82**, 981 (1995).
37. E. Abruzzese *et al.*, Cancer Genet. Cytogenet. **105**, 164 (1998).
38. J. D. Thompson *et al.*, Nucleic Acids Res. **25**, 4876 (1997).
39. A. J. Muller *et al.*, Mol. Cell. Biol. **11**, 1785 (1991).
40. Single-letter abbreviations for the amino acid residues are as follows: A, Ala; C, Cys; D, Asp; E, Glu; F, Phe; G, Gly; H, His; I, Ile; K, Lys; L, Leu; M, Met; N, Asn; P, Pro; Q, Gln; R, Arg; S, Ser; T, Thr; V, Val; W, Trp; and Y, Tyr.
41. We thank J. Kuriyan and B. Nagar for providing structural modeling data; E. Buchdunger for providing STI-571; K. Shannon, K. Shuai, and O. Witte for helpful comments; K. Banks for technical assistance; and L. Rose for manuscript preparation. Supported by grants from the Leukemia and Lymphoma Society, the National Cancer Institute (C.L.S.) and a United States Public Health Service National Research Service Award GM07185 (M.E.G.).

15 May 2001; accepted 12 June 2001 Published online 21 June 2001; 10.1126/science.1062538 Include this information when citing this paper.

---

**Reciprocal Regulation Between TOC1 and LHY/CCA1 Within the Arabidopsis Circadian Clock**

David Alabadi,¹* Tokitaka Oyama,¹† Marcelo J. Yanovsky,¹* Franklin G. Harmon,¹ Paloma Más,¹ Steve A. Kay¹‡

The interactive regulation between clock genes is central for oscillator function. Here, we show interactions between the Arabidopsis clock genes LATE ELONGATED HYPOCOTYL (LHY), CIRCADIAN CLOCK ASSOCIATED 1 (CCA1), and TIMING OF CAB EXPRESSION 1 (TOC1). The MYB transcription factors LHY and CCA1 negatively regulate TOC1 expression. We show that both proteins bind to a region in the TOC1 promoter that is critical for its clock regulation. Conversely, TOC1 appears to participate in the positive regulation of LHY and CCA1 expression. Our results indicate that these interactions form a loop critical for clock function in Arabidopsis.

Circadian clocks represent a widespread endogenous mechanism that allows organisms to time different processes appropriately throughout the day-night cycle. Among the activities controlled by the circadian clock are the regulation of transcription in cyanobacteria, the rhythmic movement of leaves in plants, and more complex activities such as

880 3 AUGUST 2001 VOL 293 SCIENCE www.sciencemag.org

control of feeding behavior in flies and control of the sleep-wake cycle in humans. The basic molecular mechanisms underlying the generation of circadian rhythms have been significantly deciphered in *Synechococcus elongatus*, *Neurospora crassa*, *Drosophila melanogaster*, and the mouse (1). In these four model systems, the oscillator is based on transcriptional-translational negative feedback loops involving a group of clock genes. In *Arabidopsis*, three genes have been suggested as components of the oscillator: *LHY*, *CCA1*, and *TOC1* (2). *LHY* and *CCA1* encode highly conserved single-MYB transcription factors which, when expressed at high and constitutive levels, disrupt the normal functioning of the clock (3, 4). The third gene, *TOC1*, was initially identified as the *toc1-1* mutant in a screen for plants with altered period length of *cab2::luc* expression (5). *toc1-1* is a short-period mutant with altered clock function through the entire life cycle of the plant. Moreover, the effect is independent of light quantity, suggesting a role for *TOC1* in the core of the oscillator (6, 7). The recent cloning of *TOC1* has allowed us a more detailed characterization of its molecular phenotypes (7). Here, we investigate potential interactions between *TOC1* and the MYB genes *LHY* and *CCA1*, because those interactions are likely to shape the *Arabidopsis* oscillator.

We investigated the *TOC1* expression patterns in *lhy* and *CCA1-OX* plants, which constitutively overexpress either *LHY* or *CCA1* from the strong promoter *CaMV 35S* (Fig. 1, A and B). *TOC1* mRNA oscillated with high amplitude in the wild-type parental lines in constant light (LL). The *TOC1* mRNA level was constant in both *lhy* and *CCA1-OX* plants under the same light conditions, as expected for plants in which the oscillator function is mostly disrupted. The transcript level in these mutants was similar to or lower than the trough of wild-type expression. Therefore, high and constant expression levels of either *LHY* or *CCA1* result in low and constant levels of *TOC1* transcript in LL. *LHY* and *CCA1* appear to be negative regulators of *TOC1*. The fact that *TOC1* transcript oscillates 12 hours out of phase with both the *LHY* and *CCA1* transcripts in wild-type plants further supports this idea. The *CCA1* protein has been found to cycle with little lag with respect to its transcript (4), indicating that maximum *CCA1* level coincides with minimum

---

**Department of Cell Biology and Institute for Childhood and Neglected Diseases, The Scripps Research Institute, La Jolla, CA 92037, USA.**

*These authors contributed equally to this work.

†Present address: Division of Biological Science, Graduate School of Science, Nagoya University, Furo-cho, Chikusa-ku, Nagoya, Aichi 464-8602, Japan.

‡To whom correspondence should be addressed. E-mail: stevek@scripps.edu

---

TOC1 mRNA level, and vice versa.

elf3-1 plants become arrhythmic due to an arrest of its circadian clock in continuous light (8, 9). We investigated the *TOC1* expression profiles in the *elf3-1* mutant under LL conditions (Fig. 1C). Contrary to what we observed for the wild type, the transcript level remained elevated and unchanging in the mutant in free-running conditions. The arrhythmic phenotypes observed in *elf3-1* and in *lhy* and *CCA1-OX* plants in LL are associated with different levels of *TOC1* transcript. Both the *LHY* and *CCA1* mRNA levels were found to be reduced in *elf3-1* plants in LL (3, 10), consistent with the elevated *TOC1* transcript levels in this mutant arising from a significant reduction in the amount of the negative regulators *LHY* and *CCA1*.

If *LHY* and *CCA1* act directly to negatively regulate *TOC1* expression, they might bind to a functionally important region of the *TOC1* promoter. Accordingly, we sought to identify regulatory regions of the *TOC1* promoter by generating a series of deletions fused to the luciferase gene and analyzing expression of the reporter in *Arabidopsis* plants transformed with these constructs (Fig. 2, A and B). Progressive deletions from –2340 (base pairs from the starting ATG) to –834 had only a slight effect on the mean expression level of the reporter and did not affect its oscillation (Fig. 2A) (10). On the other hand, deletion from –834 to –620 abolished rhythmicity. The region from –834 to –620 was sufficient to confer rhythmic expression to the reporter (Fig. 2B). To assess the possibility that *LHY* and/or *CCA1* utilize this region to regulate *TOC1* expression, the ability of these two proteins to bind to the –834 to –620 fragment was tested by performing electrophoretic mobility shift assay (Fig. 2, C and D) (11). Addition of extracts from *Escherichia coli* expressing either GST-CCA1 or GST-LHY to a probe corresponding to the –834 to –620 region of the *TOC1* promoter produced DNA species with retarded mobility (11); bacterial extracts harboring the empty glutathione S-transferase (GST) vector did not produce comparable bands. Competition experiments show that GST-CCA1 and GST-LHY specifically bind to a DNA fragment corresponding to this part of the promoter (Fig. 2, C and D). To delimit further this region, three more deletions were analyzed for the expression of the reporter (Fig. 2B). Removal of the region between –734 and –687 greatly reduced rhythmicity. Therefore, the –734 to –687 region contains cis elements important for clock regulation of *TOC1*. Recently, a novel sequence element (evening element or EE) has been identified in the promoter region of 31 cycling genes peaking at the end of the subjective day (12). The importance of that element for circadian regulation in vivo was shown for one of those genes, *CCR2*. We found one EE in the –734 to –687 fragment of the *TOC1* promoter. Mutation of the EE in the context of the –834 to –620 fragment caused strong reduction in rhythmicity (Fig. 2B). This indicates that the EE is important for clock regulation of *TOC1*. Given the high similarity between the EE (AAAATATCT) and the motif that CCA1 binds (AAAAATCT) (13), we investigated whether the EE was the actual target used by LHY and CCA1 to regulate *TOC1*. Figure 2, C and D, show the specific interaction of both transcription factors with the EE. Taken together, these results strongly suggest that LHY and CCA1 negatively regulate *TOC1* expression by binding to the EE in its promoter region.

To investigate whether the regulation is reciprocal, we analyzed *LHY* and *CCA1* expression patterns in the *TOC1* alleles *toc1-1* and *toc1-2* (Fig. 3) (5, 7). Both the *LHY* and *CCA1* messages cycled with a shorter period length in the two mutants than in wild-type plants, consistent with the short period observed for the expression of

![Figure 1](#fig1)

Fig. 1. TOC1 expression is strongly affected in *lhy*, *CCA1-OX*, and *elf3-1* plants. *lhy* plants and the parental wild type (*Ler*) (A), *CCA1-OX* (line o34) and *elf3-1* plants, and the parental wild type (*Col-0*) [(B) and (C), respectively] were germinated and entrained to cycles of 12 hours light, 12 hours dark [LD (12, 12)] for 7 days and then released into constant light (LL). TOC1 RNA levels were analyzed by Northern blot. The dark and white bars at the bottom of each graph represent lights off and on, respectively. Representative data are shown.

Fig. 2. Identification of a region in the TOC1 promoter that is essential for its circadian oscillation and is a target of LHY and CCA1. (A and B) Analysis of the TOC1 promoter. The different deleted versions of the promoter were fused to the firefly luciferase+ gene and the resulting constructs were introduced into Arabidopsis by Agrobacterium-mediated DNA transfer (27). Primary transgenic plants were selected under LD (12, 12) cycles for 8 days, transferred to media without selection, and imaged and analyzed as described (7). In the mutant promoter [m-834/-620 in (B)], the last six nucleotides of the EE were changed to CTGCAG. Error bars in (A) represent the standard error of the mean. The numbers in the upper left corner of each graph (A) and in the x axis (B) indicate the fragment of the TOC1 promoter analyzed. n, number of independent T1 seedlings analyzed. In (B), seedlings with a relative amplitude error lower than 0.6 were scored as rhythmic; values close to 1.0 indicate weak rhythms (9). Competition experiments reveal that the GST-CCA1 (C) and GST-LHY (D) proteins specifically bind to the EE in the -834/-620 region of the TOC1 promoter. No competitor DNA was added in lanes a and j. The unlabeled competitor DNA corresponded to the -834/-620 region of TOC1 (-834/-620 TOC1; lanes b through e) or the 379 to 481 region of the pUC18 (379/481 pUC18; lanes f through i). In lanes k through n, the unlabeled competitor DNA corresponded to the -734/-687 region of TOC1 (-734/-687 wt EE), and the competitor DNA in lanes o through r was the same as in lanes k through n but with the last six nucleotides of the EE changed to CTGCAG (-734/-687 mut EE). Black triangles represent increasing amount of competitor, which was present at 1× (lanes b, f, k, and o), 10× (lanes c, g, l, and p), 100× (lanes d, h, m, and q), and 200× (lanes e, i, n, and r) molar excess of the labeled probe. The arrowhead indicates the protein-DNA complexes.

other genes (5, 7). The peak levels of LHY and CCA1 expression were reduced by more than 50% in the toc1-2 background, and, consequently, the amplitude of oscillation for both transcripts was reduced. The different effects of the toc1-1 and toc1-2 mutations on the expression levels of LHY and CCA1 likely reflect the different nature of these alleles. toc1-1 is semidominant and appears to retain some of the TOC1 function, whereas toc1-2 is recessive and presumably represents a loss-of-function allele (5, 7). The low expression level of both genes might be the cause of the short-period phenotype of toc1-2. It could also explain the same phenotype in the CCA1 loss-of-function allele ccal-1 (14). The semidominant nature of toc1-1 suggests distinct biochemical abnormalities, such as altered protein-protein interaction, for its defect in period length. The reduction of the transcript peak level for both LHY and CCA1 in toc1-2 strongly suggests a positive effect of TOC1 on the expression of both genes. Several features of TOC1 protein support this idea: It is localized in nuclear speckles that may represent transcriptional complexes (7); it has an acidic-rich region at the COOH-terminal end of the protein, which is commonly found among several

Fig. 3. LHY and CCA1 expression is reduced in toc1-2 but not in toc1-1. toc1-1 (A and B), toc1-2 (C and D), and wild-type plants (C24) were grown in LD (12, 12) cycles for 7 days and then released into LL. LHY and CCA1 RNA levels were analyzed by Northern blot. Representative data are shown.

types of eukaryotic transcriptional activators (7, 15); and it bears homology to the CONSTANS protein of Arabidopsis, which has been implicated in transcriptional regulation (16). The ability of two Arabidopsis response regulators to activate transcription has been described (17). The fact that both LHY and CCA1 transcripts continue to oscillate in toc1-2 indicates that TOC1 is not the only factor controlling expression of both genes. It has also been suggested that the basic helix-loop-helix transcription fac-

found in these plants (4). However, the intermediate levels of endogenous $LHY$ transcript found in the $lhy$ allele $lhy^{TN104}$ (3) suggest that positive elements other than TOC1 may still be active; an alternative, but not mutually exclusive explanation would be that the effects of $lhy$ and $lhy^{TN104}$ on $TOC1$ expression are different.

As mentioned above, high levels of $TOC1$ expression correlate with low levels of $LHY$ and $CCA1$ transcripts in the $elf3-1$ mutant in LL (Fig. 1C) (3, 10). However, the fact that the level of both transcripts remains low suggests that TOC1 fails to regulate positively their expression in the absence of the ELF3 function. Otherwise, the steady-state levels of $TOC1$, $LHY$, and $CCA1$ transcripts would reach intermediate values due to the simultaneous action of positive (TOC1) and negative ($LHY$ and $CCA1$) factors.

The results presented here show interactions among genes involved in the Arabidopsis oscillator. The data reveal different roles for $LHY$ and $CCA1$ versus $TOC1$. The MYB factors act as negative elements that repress $TOC1$ expression, and, conversely, $TOC1$ appears to be a positive element for $LHY$ and $CCA1$ expression (Fig. 4). $LHY$ and $CCA1$ appear to have a dual role in the Arabidopsis clock. It has been suggested that $CCA1$ acts as a positive regulator of $CAB$ gene expression (4, 13, 14). Given the overall similarity between $CCA1$ and $LHY$, it is likely that the $LHY$ protein is also a positive regulator of $CAB$ genes. Furthermore, both of these proteins also act as negative regulators of one evening-phased gene, $TOC1$. We predict that $LHY$, $CCA1$, or both may also negatively regulate other genes having EE in their promoter regions, which include, for example, $GIGANTEA$ and $CCR2$ (12). Similar to a mechanism proposed for Drosophila (20), we suggest a model in which these

tor PIF3 acts as a positive element in $LHY$ and $CCA1$ expression, at least in etiolated seedlings in response to light pulses (18). The possibility remains that $TOC1$ function is, to some degree, redundant with some of the $TOC1$-like genes found in Arabidopsis, one of which oscillates with a phase very similar to $TOC1$ (19). The low level of $TOC1$ function expected in $CCA1-OX$ could explain, in part, the low level of endogenous $CCA1$ and $LHY$ messages

MYB factors simultaneously regulate genes that are phased to different times of the day. The analysis of $TOC1$ biochemical properties and the identification of $TOC1$-interacting proteins will help us to close the loop. Our results suggest that the interactive regulation between these genes define the basic framework for the clock mechanism in Arabidopsis.

### References and Notes

1. J. C. Dunlap, *Cell* **96**, 271 (1999).
2. D. E. Somers, *Plant Physiol.* **121**, 9 (1999).
3. R. Schaffer et al., *Cell* **93**, 1219 (1998).
4. Z. Y. Wang, E. M. Tobin, *Cell* **93**, 1207 (1998).
5. A. J. Millar, I. A. Carré, C. A. Strayer, N.-H. Chua, S. A. Kay, *Science* **267**, 1161 (1995).
6. D. E. Somers, A. A. R. Webb, M. Pearson, S. Kay, *Development* **125**, 485 (1998).
7. C. A. Strayer et al., *Science* **289**, 768 (2000).
8. K. A. Hicks et al., *Science* **274**, 790 (1996).
9. H. G. McWatters, R. M. Bastow, A. Hall, A. J. Millar, *Nature* **408**, 716 (2000).
10. D. Alabadí, M. J. Yanovsky, P. Más, S. A. Kay, unpublished data.
11. Supplementary material is available at Science Online at www.sciencemag.org/cgi/content/full/293/5531/880/DC1.
12. S. L. Harmer et al., *Science* **290**, 2110 (2000).
13. Z. Y. Wang et al., *Plant Cell* **9**, 491 (1997).
14. R. M. Green, E. M. Tobin, *Proc. Natl. Acad. Sci. U.S.A.* **96**, 4176 (1999).
15. K. B. Singh, *Plant Physiol.* **118**, 1111 (1998).
16. A. Samach et al., *Science* **288**, 1613 (2000).
17. H. Sakai, T. Aoyama, A. Oka, *Plant J.* **24**, 703 (2000).
18. J. F. Martínez-García, E. Huq, P. H. Quail, *Science* **288**, 859 (2000).
19. A. Matsushika, S. Makino, M. Kojima, T. Mizuno, *Plant Cell Physiol.* **41**, 1002 (2000).
20. N. R. Glossop, L. C. Lyons, P. E. Hardin, *Science* **286**, 766 (1999).
21. S. J. Clough, A. F. Bent, *Plant J.* **16**, 735 (1998).
22. We thank M. A. Blázquez, S. L. Harmer, and C. A. Strayer for critical reading of the manuscript; A. Schopke for technical assistance; and C. Orozco for his help with computers. Supported by NIH grant GM56006 (S.A.K.), and postdoctoral fellowships from the Spanish Ministerio de Educación y Cultura (D.A.), Human Frontier Science Program (T.O.), and Pew Foundation (M.J.Y.). F.G.H is a Department of Energy-Energy Biosciences Research Fellow of the Life Sciences Research Foundation.

4 April 2001; accepted 21 June 2001

---

**POWERSURGE**

NEW! Science Online’s Content Alert Service: Knowledge is power. If you’d like more of both, there’s only one source that delivers instant updates on breaking science news and research findings: Science’s Content Alert Service. This free enhancement to your Science Online subscription delivers e-mail summaries of the latest news and research articles published weekly in Science – instantly. To sign up for the Content Alert service, go to Science Online – but make sure your surge protector is working first.

---

**Science**

www.sciencemag.org

For more information about Content Alerts go to www.sciencemag.org. Click on Subscription button, then click on Content Alert button.

www.sciencemag.org SCIENCE VOL 293 3 AUGUST 2001 883
