
Biochimica et Biophysica Acta 1520 (2001) 154–162

**Sequencing, phylogenetic and transcriptional analysis of the glyoxylate bypass operon (ace) in the halophilic archaeon Haloferax volcanii**

Juan Antonio Serrano, María José Bonete*

División de Bioquímica, Facultad de Ciencias, Universidad de Alicante, Ap. 99, Alicante E-03080, Spain

Received 28 February 2001; received in revised form 30 May 2001; accepted 11 June 2001

---

### Abstract

The glyoxylate cycle occurs in the three domains of living organisms: Eukarya, Bacteria and Archaea. We have isolated and sequenced the ace (acetate assimilation) gene operon, comprising the glyoxylate cycle key enzymes isocitrate lyase and malate synthase genes (*icl* or *aceA* and *ms* or *aceB*), from the halophilic archaeon *Haloferax volcanii*. This is the first time that these genes are sequenced in an organism from the domain Archaea. Phylogenetic analysis of the sequenced genes revealed that isocitrate lyase shows a significant identity with isocitrate lyases from Eukarya and Bacteria, but it is not more closely related to eukaryal or bacterial enzymes, and that malate synthase from *H. volcanii* has very little identity with any other known protein. This enzyme forms a new class of malate synthases. Transcriptional analysis indicated that both genes are cotranscribed in a single 2.7 kb mRNA molecule. The genes were transcribed only when acetate was the carbon source, indicating transcriptional regulation. Two sets of palindromic sequences were found in the promoter region, possibly involved in binding of transcriptional regulators (repressors and/or activators). © 2001 Elsevier Science B.V. All rights reserved.

**Keywords:** Glyoxylate cycle; Isocitrate lyase; Malate synthase; Transcriptional regulation; Haloferax; Archaea

---

### 1. Introduction

The glyoxylate cycle is a metabolic pathway by which organisms can synthesize carbohydrates from C2 compounds. This cycle has some common enzymes with the citrate cycle, and there are two exclusive enzymes: isocitrate lyase (EC 4.1.3.1), which catalyzes the cleavage of D-isocitrate to glyoxylate and succinate, and malate synthase (EC 4.1.3.2), which catalyzes the formation of L-malate from glyoxylate and acetyl-CoA. The net function of the glyoxylate cycle is the formation of a C4 compound from two acetyl-CoA (C2) molecules in each cycle.

Taking into account the three domains of living organisms [1], the key enzymes of this cycle (isocitrate lyase and malate synthase) have been detected in all of them: Bacteria, Eukarya (fungi, plants, some invertebrates [2], and even in some vertebrates [3,4]) and Archaea [5]. In Archaea, the operation of the glyoxylate cycle has been reported only in *Haloferax volcanii* at present [5]. In Bacteria, Archaea and unicellular Eukarya, the glyoxylate cycle mainly operates when the principal or sole carbon source

* Corresponding author. Fax: +34-965-90-38-80.
E-mail address: mjbonete@ua.es (M.J. Bonete).

© 2001 Elsevier Science B.V. All rights reserved.
PII: S0167-4781(01)00263-9

BBAEXP 93553 3-8-01

J.A. Serrano, M.J. Bonete / Biochimica et Biophysica Acta 1520 (2001) 154–162

1 GTACAGAATTAAGTCATTGGCTAATATTATCATTATCAATCATTAACTATTTGTACGACT
61 CGGCAGTCGAACCGGGTGAGGAGACATCCGGATGGTAGTATCCGCAGCACACCACGCGTCC
121 AACC GCTCGACGACCGCGATACGAACCGACC GCTACCGACCGGACCTC ACCAACG TAA
181 CCGATGACAAAACCGCGACAACACGACCCGAAACCGACACGACCCGACGCCGACCACC
241 GCTCGCTCGACGCCGACTCGTCCCCCCC GCCGTGCGGCGCCTACCCCAAACAGGGAGAAC
301 ATGGACGCGCCGCTCGTCCCCCTTCATCCAGCGGGCGGGGACGACGAGATGACGACG
361 ATTCGGCGCGAACTCGCCGCGCTCCGCGCCCAA CTGAACCGCATCGAGCGAACGATGCAG
icl (aceA)
421 TCCACCGACACCCAATGAATCCGACC GAAACTCGACAGCGACGTCTTCGCACAGGATGTTG
M N P T E L D S D V F A Q D V D
M N P T E L D S X V F A Q D V D
481 ACAACCAGAAGGCCCCGCAACTGCGGGGAGATGCTGAAACACGCAGGACTTCGTGTT CGCCC
N Q K A R E L R E M L N T Q D F V F A P
N Q
541 CCGGCATGTACCACGCGCTCGACGCCCGCCTCGCCGAGATGACGGGCCACGACGCCGCCT
G M Y H A L D A R L A E M T G H D A A Y
601 ACATGTCCGGCTACTCGACC GTCCCTCGGGCAGTT CGGCTTCCCCGACCTGGAGATGGTCA
M S G Y S T V L G Q F G F P D L E E M V T
661 CGATGACC GAGATGGTCGAGAACGCAAAGCGCATGGTCGAGGCGACGAACTTGCCGGTCA
M T E M V E N A K R M V E A T N L P V I
721 TCGCCGACTGCGACACGGGCTACGGG GCATCCACAAACGTCCGTCGCGCCGTCCGCGAGT
A D C D T G Y G G I H N V R R A V R E Y
781 ACGAGAAGGCCGGCGTCGCCCGTCCACATCGAAGACCAGACGACCCCAAGCGCTGCG
E K A G V A A V H I E D Q T T P K R C G
841 GCCACATCGGGGCAAGCAAATCGTCTCCC GCGAGAAGGCCAAGGCTCGCTTCGAGGCCG
H I A G K Q I V S R E K A K A R F E A A
901 CCGTCGACGCCAAGCAGTCGGAGGACACGGTCGT CATCGCCC GCACC GACGCCTACGGCT
V D A K Q S E D T V V I A R T D A Y G S
961 CTTCGAA CGGAGACTGGGACGAGCACGT CGAGCGGCGCCGCATCTACGCGGACGCGGGCG
S N G D W D E H V E R G R I Y A D A G V
1021 TCGACATCGTCTGGCCC CGAGATGCCGAACCCCC TGCGCGGAGGACGCGT CGCCCTACGCCG
D I V W P E M P N P S R E D A V A Y A E
1081 AGGAGATTCACGAGACGCATCCGGACCTGAAAGCTCGCGTTCAACTACTCGTCGT CGTTCG
E I H E T H P D L K L A F N Y S S F A
1141 CGTGGTCCGAAAGAGGAGGACCCGTTGACGTTCCAAGAGCTGGGCGACCTCGGCTACAAGT
W S E E E D P L T F Q E L G D L G Y K Y
1201 ACATCTT CATCACGCTGTT CGGGCCTCCACTCGGGCGCGCACGCGTCTACGAGGACTTCA
I F I T L F G L H S G A H A V Y E D F K
F I T L A G
1261 AGAAGCTCGCCGAAACGGACGAGGAAAGGCCAGTT CGAACCTCGAAACAGCGCTACCTCGACC
K L A E Q D E E G Q F D L E Q R Y L D H
1321 ACCCGACC GAGAGCCACCACGAGCTGTCGTT CGTCGCGGTACCAGGACATCGAGACGG
P T E S H H E L S F V S R Y Q D I E T E
1381 AGTTCGACCCCGAGGCGCGCCGCA TC GAGGAGTCCGAAGGTT CAGCGAGGAGCAGG
F D P E A R R R I E E S E G F S E E Q A
ms (aceB)
1441 CCGACCCCATC ACGAGCAA CGACGATGACTGAGCGCAGACACGACAGGGAGTT CGTGC GG
D P I T S N D D D *
M T E R R H D R E F V R
T E R R H D R E F V
1501 ACGTTCTTCA CCCTCGCCGACGGCCGT CGGAGGGCGAAAGACGACTCCGC GAAAGATGCTCCGA
T F F T S P T A V E G E D D S A K M L R
1561 CGGGCGGGCCGGCCTCCGCGGGATGCAAGCGCCC GACGTGTGGGTCCCCGACAACGAGGAC
R A A G L R G M Q A P D V W V P D N E D
1621 GCCACC GCGCCGTCGATGCGCGACGAGGGAGCCGAGAACATCGT CGAAAGTCATCTCCGAA
A T A P S M R D E G A E N I V E V I S E
1681 CAGGGCGCGGAGTTCCCCGGGCGAGATTCACCCCCGGATGGTCTGGCACCGCGACAGCCCC
Q G A E F P G E I H P R M V W H R D S P
1741 GAGACCCGATATCAGGGCTTCCAGCACATGCTCGACATCACC GAGCGCGGGGGCG
E T R Y Q G F Q H M L D I T D P E R G A
1801 GTCGAGCACATCCACGGCTTCGTCATCCCCGAGGT CGGGGCATCGACGACTGGAAAGAG
V E H I H G F V I P E V G G I D D W K K
1861 GCCGACGAGTTCTTCA CCATCGTCGAGCAGCAGGGCCTCGACGAGGGGAGCCTCGCC
A D E F F T I V E H E H G L D E G S L A
1921 ATGTCGGTCATCATCGAGAGCGGCGAGGCGA ACTCGCCATGGGCGACCTCCGCGACGAG
M S V I I E S G E A E L A M G D L R D E
1981 ATGGGCAAGCCGACGAAACAACCTCGAACGGCTGTT CCTCTCGTCGACGGCGAGGTCGAC
M G K P T N N L E R L L L V D G E V D
2041 TACACGAAAGGACATGCGCGCCATGACGCGGCGAGGGGCGAACTCCC GCGGTGGCGGAGCTT
Y T K D M R A M T P T G E L P A W P E L
2101 CGACACAACACCTCGCGCGGGGCCAGCGCCGCGGCTGTGTGTCGCGGTGGACGGCCCCCTAC

Fig. 1. Nucleic acid sequences and predicted amino acid sequences of *icl* (*aceA*) and *ms* (*aceB*). The predicted amino acid sequences are shown in single-letter code below the first base of each codon. The N-terminal sequences experimentally determined for isocitrate lyase and malate synthase from *H. volcanii*, and the conserved region of isocitrate lyases used to design ICL2, are represented below the amino acid sequences predicted, where X is an uncertain assignment. The putative TATA box (double underlined), the putative ribosome binding site for *icl* (single underlined) and the putative transcription termination site (dashed underlined) are indicated.

R H N T S R G A S A A G C V A V D G P Y
2161 GACGACATCCGCGACGTGGAGGGCTACCGGCAGCGCATGACCACAATCAGGCGAAGGGG
D D I R D V E G Y R E R M T D N Q A K G
2221 ATGCTCGGCATCTGGTCGCTCACGCCCGGACAGGTGTCGAGGCAAAACACTCGCCGCTC
M L G I W S L T P G Q V V E A N T S P L
2281 CCGCCGAAAGACCGGCGAGTTGGCTCCTCGACGGGACGGCGAGGAGTCTGAACCTCGCGTCC
P P K T G S W L L D A D G E E E V E L A S
2341 GAGGACGGCGTCGAGGCGTACGACGGGACCCGCTCTCGCTCGAAGCCACCGACGGCGGC
E D G V E A Y D G D R L S L E A T D G G
2401 TACGAAACTCCGGTGGGCGGCGACGGCGGGAGCTCACCGTCGACGAAACTCCGGAGGAA
Y E L R V G G D A R E L T V D E L R E E
2461 CTGCTGGGTCTGACCTCGTACGTCCCAGCATGGACGACATCGTCGACTCCCATGGAGGAG
L L G L T S Y V P S M D D I V D S M E E
2521 TTCGAGGCGGCCAAGGAGGCCCCGGCGGCGCTATCGCCATGACGCAGTCCGCGACGCTC
F E A A K E A G R G A I A M T Q S A T L
2581 CGCATCGGCGGCAACGAAATCGACATCGAAAAAAGACGGATGTGGGACGAAGCGACCTAT
R I G G T E I D I E K D R M W D E A T Y
2641 CAGGCCGCGATGACGCCCATCAGCCTGTTCAAAGACGTGTACGAGAACCGCCCAGACCAA
Q A A M T P I S L F Q D V Y E N R P D Q
2701 CACGAAAGAACTGGAAGAACGCTACGGCGGGGCGTCGAGCGCGGATGGAGGTGGGT
H E E L E E R Y G A G V V E R A M E V G
2761 CTGTAAGCGCGGAAGGCGTCCGCCACAGGGGGTCCGCGGCCCCGGTTTCCGGCTT
L *
2821 GGTCAGACGACGGCGTCCGGTTTCGGAGCTCTGCGGCGCGCGCCCGAACGTGGGGGC
2881 GGCGCGCCTCGCTTTTTTGGGGGTCTCGCGCGGGTCCGCTTCGACTCGACAGGCCCCTCTA
2941 TCGACGCTAGCCTCAAGGTCCGTGCGGCGGACTCGACGTGTGTGAACGA

Fig. 1 (continued).

tioned. Phylogenies based on 16S rRNA sequences and some proteins indicate that Archaea are closer related to Eukarya than to Bacteria, but with the majority of available protein sequences, Archaea appear generally closer related to Gram-positive bacteria [13].

2. Materials and methods

2.1. Strain and growth conditions

H. volcanii DSM 3757 was grown aerobically at 37°C, in a chemically defined medium [14], with 0.1% sodium acetate or 0.5% sodium lactate as carbon source.

2.2. Purification of the enzymes, mass spectrometry and N-terminal sequencing

Isocitrate lyase and malate synthase were purified from cell extracts of H. volcanii grown on acetate with a three-step procedure, as described previously [5]. The subunit molecular weight of isocitrate lyase was determined on a Fisons VG Platform electrospray mass spectrometer (ES-MS). The sample was 20 μg of pure protein in water. The operating conditions for ES were as follows: the source was operated in the positive ion mode, the electrospray needle was held at 4.25 kV, and the heated capillary was operated at 90°C. Calibration was performed using a standard solution of horse heart myoglobin (Sigma). Full scan data acquisition was performed scanning from 670 to 2430 m/z in continuous mode, using a cycle time of 3.6 s and an interscan time of 0.13 s.

The N-terminal amino acid sequences of the subunit of both enzymes were determined by automated Edman deg-

separated electrophoretically in SDS-polyacrylamide gels (12%), carried out according to Laemmli [15], and gels were transferred to a polyvinylidene difluoride (Immobilon-PSQ, Millipore) membrane. Membranes were stained with Coomassie blue R-250, and bands corresponding to each enzyme were cut and sequenced on an automated protein sequencer.

2.3. Isolation and sequencing of genes for isocitrate lyase (icl or aceA) and malate synthase (ms or aceB)

Two synthetic oligonucleotides were constructed for specific amplification of a fragment of icl by PCR from genomic DNA of H. volcanii. The oligonucleotide used as forward primer was ICL-Nter (5'-ATGAACCC(C/G)GAGCTCGA-3'), based on the determined N-terminal amino acid sequence of isocitrate lyase from H. volcanii. The reverse primer was ICL2 (5'-CC(C/G)GC(C/G)AG(C/G)GTGATGAA-3'), based on a conserved region of isocitrate lyases, determined previously [16] by aligning known amino acid sequences of this protein from several organisms of the domains Eukarya and Bacteria. Both oligonucleotides were constructed based on H. volcanii codon usage.

The PCR conditions were as follow: 100 pmol of each primer, 200 ng of genomic DNA from H. volcanii, isolated with cetyltrimethylammonium bromide and chloroform

![Diagram](https://i.imgur.com/1234567.png)

Fig. 2. Schematic representation of the ace operon of H. volcanii.

extraction after lysis with SDS and proteinase K digestion [17], 2 mM MgCl₂, 1.2% DMSO and 2 U of Taq polymerase, in 100 μl of total volume. The cycle conditions were: 5 min at 96°C, followed by 30 cycles of 96°C, 1 min; 50°C, 1 min; 72°C, 1 min, and a final extension period of 7 min at 72°C.

The complete sequence of the PCR fragment generated with the combination of the forward and the reverse primer was determined by cycle sequencing with dye terminators (Perkin-Elmer), and sequencing reactions were run on a Perkin-Elmer ABI PRISM 310 DNA sequencer.

The PCR fragment was labeled nonradioactively with digoxigenin-11-dUTP, incorporated during PCR, with the PCR DIG Probe Synthesis kit (Boehringer Mannheim). This fragment was used as a probe in a library of genomic DNA of *H. volcanii* in the λ phage derived vector EMBL3.

The sequences of genes for isocitrate lyase and malate synthase were determined by sequencing a positive recombinant λ clone on both strands by primer walking. The sequencing was done as described above, using modified reaction conditions as in [18].

### 2.4. Sequence analysis

Nucleotide sequences were analyzed with the MAP program (Genetics Computer Group (GCG)). Deduced amino acid sequences were compared to protein sequences available in databases using FASTA, BESTFIT (GCG programs) and the CLUSTAL-W program package [19]. Phylogenetic analyses were done with the CLUSTAL-W program package, using the neighbor-joining method [20]. Trees were visualized using the TreeView 1.6.1. program [21], distributed by the University of Glasgow, UK.

### 2.5. Analysis of mRNA

Total RNA was isolated from *H. volcanii* cells grown on acetate or lactate by the guanidinium thiocyanate method [22], using Ultraspec RNA (Biotecx).

Total RNA (5 μg per well) was fractionated by electrophoresis on a 1× MOPS, 0.66 M formaldehyde, 1% agarose gel. Northern blotting was performed by downward alkaline capillary transfer [23] onto positively charged nylon membranes (Hybond-N⁺, Amersham). Northern hybridization probes were a 788 bp DNA fragment of the *icl* gene, amplified with the oligonucleotides ICL-Nter and ICL2 by PCR, spanning nucleotides 1–788 of the *icl* gene and a 497 bp DNA fragment of the *ms* gene, amplified with the oligonucleotides MS3 (5′-ATGTCCGGTCAT-CATCGAG-3′) and MSR2 (5′-CCAACGCGGGAGTTC-GTAG-3′), spanning nucleotides 398–894 of the *ms* gene. Both DNA fragments were labeled nonradioactively with DIG-11-dUTP as described above.

Northern hybridization was carried out as described in the manufacturer’s protocols [24]. Membranes were hybridized overnight at 50°C in High SDS buffer. Post-hybridization washes were: 2× SSC/0.1% SDS, 15 min with rotation at room temperature, twice; 0.2× SSC/0.1% SDS, 15 min with rotation at 50°C, twice.

### 2.6. Nucleotide sequence accession numbers

The sequences of *icl* (*aceA*) and *ms* (*aceB*) have been deposited in the EMBL Nucleotide Sequence database under accession Nos. AJ250922 for *icl* (*aceA*) and AJ250923 for *ms* (*aceB*).

## 3. Results and discussion

### 3.1. N-Terminal sequence and construction of a probe for *icl* and *ms*

The N-terminal amino acid sequences of the subunits of purified isocitrate lyase and malate synthase from *H. volcanii* were MNPTELDSXFAQDVDNQ and TERRHDREFV, respectively. The first amino acid of malate synthase is threonine, indicating a posttranslational modification of the enzyme.

Using ICL-Nter and ICL2a 788 bp DNA fragment with significant identity to known isocitrate lyase sequences was obtained and completely sequenced. This was used as a probe for detection of *icl* in the library of genomic DNA of *H. volcanii*. ICL-Nter is based on the N-terminal amino acid sequence MNPTELD of isocitrate lyase from *H. volcanii*, and ICL2 on the conserved region of isocitrate lyase FITLAG [16].

### 3.2. Isolation and sequencing of *icl* and *ms*

2989 bp of the positive recombinant λ clone were sequenced, which included the putative *icl* and *ms* genes (Figs. 1 and 2).

The N-terminal amino acid sequences of isocitrate lyase and malate synthase obtained by protein sequencing correspond with the deduced amino acid sequence of the sequenced genes (the first methionine residue is lacking in malate synthase). It can be concluded that those genes are the genes for isocitrate lyase and malate synthase from *H. volcanii* (Fig. 1).

Both isocitrate lyase and malate synthase are clustered in the genome of *H. volcanii*, even overlapping, as the start codon for *ms* is located five bases before the stop codon for *icl* (Figs. 1 and 2). Any other open reading frame was not found upstream from *icl* or downstream from *ms*.

Transcription in halophilic Archaea typically initiates 24 bp downstream from a TATA box sequence, with the consensus 5′-(T/C)T(T/A)(T/A)AN-3′ [25], preferably at a purine located in a consensus sequence –1 (T/A)(A/G)NG +3 [26]. Only a putative TATA box was found (5′-CTTCAT-3′), located 108 bp upstream from the trans-

295 GAGAACATGGACGCGCCGCTCGTCCC**CTTCAT**CCA
330 GCGGCCGGGCGAGGACGAGATGACGACTTC
365 GGCGCGAACTCGCCGCGCTCCGCCCCAACTGAAC
400 CGCATCGAGCGAACGTGCAGTCCACCGACACCCA
     icl →
435 ATGAATCCGACCGAACTCGACAGCGACGTCTTCGC
     M  N  P  T  E  L  D  S  D  V  F  A

Fig. 3. Promoter structure of the *ace* operon of *H. volcanii*. Double underlined, putative TATA box; single underlined, putative ribosome binding site for *icl* mRNA; bent arrow, putative transcriptional start site; →←, palindromic sequences.

lation start codon for *icl* (Figs. 1 and 3). The transcription initiation would be one of the nucleotides marked in Fig. 3. There would be a leader sequence in the mRNA of about 85 nucleotides between the 5′ end and the translation start codon of *icl*.

In Archaea, the start codons are usually preceded at a 3–10 nucleotide distance by at least a four-base complementary sequence to the 3′ end of 16S rRNA (Shine-Dalgarno sequences) [27]. Only the ATG start codon for the isocitrate lyase gene is preceded at 10 nucleotides by a Shine-Dalgarno-like sequence: 5′-TCCA-3′ (Figs. 1 and 3), complementary to the 3′ end of *H. volcanii* 16S rRNA (5′-UGGAUCACCUCUU-3′), although Shine-Dalgarno sequences are usually complementary to the pyrimidine-rich region of rRNA (5′-AUCACCUCUU-3′). The gene for malate synthase lacks a clear Shine-Dalgarno sequence. There is only a three-base complementary match at an eight-nucleotide distance from the ATG start codon: 5′-GAG-3′.

There is a putative transcription termination site located 130 bp downstream from the *ms* stop codon (Fig. 1) [28].

An interesting feature of the promoter region is the presence of two sets of palindromic sequences (Fig. 3), with a length and position similar to other sequences involved in transcriptional regulation in Archaea.

• There is a palindromic sequence preceding the putative TATA box, with five nucleotides per half, and with a position and structure similar to the transcriptional activator GvpE binding site. This protein is involved in the regulation of gas vesicle synthesis in *Haloferax mediterranei* and *Halobacterium salinarum* [29].
• Another palindromic sequence, with six nucleotides per half, is located between the TATA box and the putative transcription initiation. The location and structure of this sequence are similar to transcriptional repressor binding sites in other archaeal organisms [30–32].

### 3.3. Gene for isocitrate lyase

The isocitrate lyase gene encodes a 345-amino acid polypeptide, with a calculated molecular mass of 39,050 Da, which is in good agreement with the \(M_r\) calculated for the isocitrate lyase subunit by mass spectrometry (38,690 ± 10). This value is about 25% less than the \(M_r\) calculated by SDS-PAGE (47,000 ± 4,000) [5], because the molecular mass of halophilic proteins is overestimated by about 25% using this method, due to the excess of acidic residues of these proteins, which produces a retardation in SDS-PAGE [33]. This \(M_r\) (39,050) suggests that the enzyme is a homotetramer, as the native \(M_r\) is 140,000 ± 20,000 [5]. This subunit composition of the enzyme is the same as that of all other known isocitrate lyases [34].

The enzyme possesses a high content of acidic amino acids (11.59% glutamic acid, 9.28% aspartic acid, 20.87% total). This is a characteristic of enzymes from halophilic archaea involved in the adaptation to a high intracellular salt concentration [35].

The isocitrate lyase subunit from *H. volcanii* has a low but significant identity with isocitrate lyases from Eukarya and Bacteria (for example, 20.6% and 22.0% identity with isocitrate lyase from *Saccharomyces cerevisiae* and *Escherichia coli*, respectively), despite the difference in molecular mass (60–64 kDa and 47 kDa for eukaryal and bacterial isocitrate lyases, respectively). This variation of \(M_r\) among all isocitrate lyases is due to different domain composition. Five domains or regions can be distinguished in isocitrate lyase amino acid sequences (Fig. 4):

1. An N-terminal region, with a certain degree of identity among all known isocitrate lyase sequences, and with a

![Diagram](attachment:diagram.png)

Fig. 4. Schematic representation of domain organization of known isocitrate lyase amino acid sequences in Bacteria, Eukarya and Archaea.

J.A. Serrano, M.J. Bonete / Biochimica et Biophysica Acta 1520 (2001) 154–162

A


Trichotomy


- Bacillus halodurans (B)
- Deinococcus radiodurans (B)
- Escherichia coli (B)
- Vibrio cholerae (B)
- Pseudomonas putida (B)
- Myxococcus xanthus (B)
- Chlamydomonas reinhardtii (E)
- Mycobacterium tuberculosis (B)
- Corynebacterium glutamicum (B)
- Rhodococcus fascians (B)
- Caenorhabditis elegans GCP (E)
- Strongyloides stercoralis (E)
- Hyphomicrobium methylovorum (B)
- Mycobacterium leprae (B)
- Haloferax volcanii (A)
- Schizosaccharomyces pombe (E)
- Coprinus cinereus (E)
- Ashbya gossypii (E)
- Saccharomyces cerevisiae ICL1(E)
- Emericella nidulans (E)
- Neurospora crassa (E)
- Yarrowia lipolytica (E)
- Candida tropicalis (E)
- Candida albicans (E)
- Pinus taeda (E)
- Dendrobium crumenatum (E)
- Brassica napus (E)
- Arabidopsis thaliana (E)
- Lycopersicon esculentum (E)
- Gossypium hirsutum (E)
- Ricinus communis (E)
- Cucurbita maxima (E)
- Cucumis sativus (E)

0.1 substitutions per site

Fig. 5. Phylogenetic trees constructed using the neighbor-joining method of known isocitrate lyase (A) and known malate synthase (B) amino acid sequences. Organisms from the domains Bacteria (B), Eukarya (E) and Archaea (A) are indicated.

length of about 150 amino acids, with the exception of the enzyme from *H. volcanii*, that possesses a shorter N-terminal region (about 90 amino acids) with very little identity with the N-terminal region of other isocitrate lyases.
2. A highly conserved region about 100 amino acids long, which is present in all known isocitrate lyases.
3. A linker region between two highly conserved domains. In Eukarya this region has about 150 amino acids, with

the exception of the enzymes from *Caenorhabditis elegans*, *Strongyloides stercoralis* and *Chlamydomonas reinhardtii* which have about 50 amino acids. In Bacteria this region is shorter, about 50 amino acids long, with the exception of the enzyme from *Mycobacterium leprae*, which has about 200 amino acids. This region is almost absent in isocitrate lyase from *H. volcanii*.
4. A second highly conserved region, with about 100 amino acids, present in all known isocitrate lyases, with the

BBAEXP 93553 3-8-01

J.A. Serrano, M.J. Bonete / Biochimica et Biophysica Acta 1520 (2001) 154–162

B


Trichotomy


| 1000 | Candida tropicalis (E) |
| --- | --- |
| 1000 | Candida albicans (E) |
| 1000 | Emericella nidulans (E) |
| 700 | Neurospora crassa (E) |
| 984 | Pichia angusta (E) |
| 1000 | Saccharomyces cerevisiae MLS1 (E) |
| 1000 | Saccharomyces cerevisiae MSL2 (E) |
| 761 | Caenorhabditis elegans GCP (E) |
| 1000 | Escherichia coli MSA (B) |
| 664 | Vibrio cholerae (B) |
| 846 | Myxococcus xanthus (B) |
| Deinococcus radiodurans (B) |
| 910 | Zea mays (E) |
| 1000 | Brassica napus (E) |
| 964 | Raphanus sativus (E) |
| Asparagus officinalis (E) |
| 664 | Ricinus communis (E) |
| 415 | Glycine max (E) |
| 987 | Gossypium hirsutum (E) |
| 539 | Cucurbita maxima (E) |
| 1000 | Cucumis sativus (E) |
| 1000 | Streptomyces clavuligerus (B) |
| 744 | Streptomyces arenae (B) |
| Streptomyces coelicolor (B) |
| 777 | Haloferax volcanii (A) |
| 1000 | Escherichia coli MSG (B) |
| 651 | Bacillus halodurans (B) |
| 1000 | Corynebacterium glutamicum (B) |
| 715 | Pseudomonas fluorescens (B) |
| 987 | Mycobacterium leprae (B) |
| 1000 | Mycobacterium tuberculosis (B) |

0.1 substitutions per site

Fig. 5 (continued).

exception of the enzyme from *Hyphomicrobium methyl-
ovorum*.

5. A C-terminal region, about 30 amino acids long, with a certain degree of identity among all organisms.

As isocitrate lyase has been reported to be present in the three domains of living organisms, and this is the first archaeal isocitrate lyase sequenced, it is interesting to investigate the phylogenetic relationships among all of them. A phylogenetic tree including all known sequenced isocitrate lyases was constructed using the neighbor-joining method (Fig. 5A). There is only one class of isocitrate lyase, formed by all known sequenced enzymes. The identity ranges from 15.6% to 97.7% within all isocitrate lyases. The enzymes from Eukarya and Bacteria form two different groups (identity ranges from 46.1% to 97.7% and 54.8% to 83.4% within them, respectively), with an identity between them in the range 27.3–36.6%. There are several exceptions: the enzymes from the bacteria *M. leprae* and *H. methyllovorum*, which do not group with any of those two groups, and the enzymes from the eukarya *C. elegans*, *S. stercoralis* and *C. reinhardtii*, which

two classes). Malate synthase from *H. volcanii* forms a new class of malate synthases, diverged from both class A and class G (10.2–14.1% and 10.5–12.0% identity with class A and class G enzymes, respectively).

The molecular mass of the subunit from *H. volcanii* (48 kDa) is lower than that of class A (60–65 kDa) and class G enzymes (about 80 kDa). The subunit composition of the enzyme (homotetramer) is similar to some class A malate synthases [36], although there are also homodimers, homotrimers and homoctamers in this class [37–39]. Class G malate synthases are monomeric enzymes [40].

Interestingly, the amino acid sequence of malate synthase from *H. volcanii* shows a high degree of identity (77.5%) with a previously sequenced fragment of plasmid pHV3 from *H. volcanii* [41]. High identity is restricted to the malate synthase gene; neither the isocitrate lyase gene nor the sequence downstream the TAA stop codon for *ms* shows homology with that region of pHV3. This fragment of pHV3 is an open reading frame of 1308 nucleotides, but it is not known whether it encodes a protein or not.

There is no homolog protein in the genome of the closely related *Halobacterium* sp. NRC1.

Several evidences show that the sequenced gene is indeed a gene for a malate synthase, although the sequence of the predicted protein lacks any sequence homology to malate synthases from other species.

### 3.4. Gene for malate synthase

The malate synthase gene encodes a 433-amino acid protein, with a calculated molecular mass of 48,046 Da. This value is about 25% less than the \(M_r\) calculated by SDS-PAGE (67,000 ± 4,000) [5], due to the reason explained above. This \(M_r\) suggests that the enzyme is a homotetramer, as the native \(M_r\) is 200,000 ± 30,000 [5].

Malate synthase from *H. volcanii* has very little identity with any other known malate synthase, and even with any other known protein. There are two distinct classes of malate synthase genes, based on homology of protein sequences: class A, the malate synthase from Eukarya and some bacteria, and class G, present only in some bacteria. Both classes are related to acetate metabolism (glyoxylate cycle), but when both classes are present in an organism (for example *E. coli*), they are differentially regulated, and class G is related to utilization of glycolate as carbon source.

Class A and class G malate synthases have highly conserved regions, but there are no common conserved regions between the two classes. None of the conserved regions of class A or class G is present in the amino acid sequence of malate synthase from *H. volcanii*.

Analysis of the phylogenetic tree including all known sequenced malate synthase genes (Fig. 5B) shows that the enzymes of a same class are closely related (34.3–91.9% and 53.2–81.0% identity within class A and within class G, respectively), but the enzymes from the two classes are diverged (10.2–15.7% identity between the

1. The determined N-terminal amino acid sequence of the subunit of purified malate synthase from *H. volcanii* corresponds with the deduced N-terminal amino acid sequence from the sequenced gene.
2. This gene is clustered with the gene for isocitrate lyase in the genome of *H. volcanii*, and both genes are co-transcribed (see Section 3.5), as in some species of Bacteria [7,42].
3. The pattern of expression of this gene (see Section 3.5) corresponds with the pattern of malate synthase activity in cells of *H. volcanii* [5].
4. As mentioned above, there is a second copy of this gene in the genome of *H. volcanii*, with a high degree of identity (77.5% in the amino acid sequence) with the sequenced gene. This supports that the sequence of *ms* was determined accurately, and that there is no mutation in the recombinant clone which can change the reading frame.

### 3.5. Transcriptional analysis of *icl* (*aceA*) and *ms* (*aceB*)

The transcription patterns of *icl* (*aceA*) and *ms* (*aceB*) were analyzed by Northern blotting, with a probe for each gene.

A single 2.7 kb transcript was detected using both probes (Fig. 6), which indicates that *icl* and *ms* are cotranscribed in a single mRNA molecule. The size of the transcript approximately corresponds to the size of the leader

sequence, *icl*, *ms* and the region downstream from *ms* up to the transcription termination (2.5 kb).

The genes for isocitrate lyase and malate synthase were transcribed only when acetate was the carbon source (Fig. 6), which corresponds with the pattern of enzyme activities [5]. These results show that genes for the glyoxylate cycle key enzymes form an operon in the genome of *H. volcanii* (*ace* operon, for acetate assimilation) (Fig. 2), and that the control of the activities of these enzymes mainly occurs by transcriptional regulation.

### 3.6. Comment

The presence of a glyoxylate cycle in *H. volcanii* has an important ecological role in the nutrition of natural communities of halophilic Archaea, as acetate is produced from glycerol (the main carbon source in nature) in hypersaline lakes by some species of halophiles [43].

### Acknowledgements

We are grateful to Dr. J. Berenguer for N-terminal sequencing of isocitrate lyase and malate synthase. We thank M.J. Danson and D.W. Hough for preparation of genomic DNA from *H. volcanii* for λ library construction. We also thank M.L. Dyall-Smith for helpful comments. This work was supported by CICYT Grants PB95-0695 and PB98-0969. J.A.S. was the recipient of predoctoral fellowships from Programa de Formación de Personal Investigador (Ministerio de Educación y Cultura).

### References

[1] C.R. Woese, O. Kandler, M.L. Wheelis, Proc. Natl. Acad. Sci. USA 87 (1990) 4576–4579.

[2] F. Liu, J.D. Thatcher, H.F. Epstein, Biochemistry 36 (1997) 255–260.

[3] W.L. Davis, D.B.P. Goodman, L.A. Crawford, O.J. Cooper, J.L. Matthews, Biochim. Biophys. Acta 1051 (1990) 276–278.

[4] V.N. Popov, S.V. Volvenkin, A.T. Eprintsev, A.U. Igamberdiev, FEBS Lett. 440 (1998) 55–58.

[5] J.A. Serrano, M. Camacho, M.J. Bonete, FEBS Lett. 434 (1998) 13–16.

[6] F. Caspary, A. Hartig, H.J. Schüller, Mol. Gen. Genet. 255 (1997) 619–627.

[7] T. Chung, Mol. Cell 5 (1995) 140–145.

[8] D.C. LaPorte, J. Cell. Biochem. 51 (1993) 14–18.

[9] R.T. Mullen, D.J. Gifford, Plant Mol. Biol. 33 (1997) 593–604.

[10] I. Ordiz, P. Herrero, R. Rodicio, F. Moreno, FEBS Lett. 385 (1996) 43–46.

[11] A. Sunnarborg, D. Klumpp, T. Chung, D.C. LaPorte, J. Bacteriol. 172 (1990) 2642–2649.

[12] J.A. Leigh, Curr. Opin. Microbiol. 2 (1999) 131–134.

[13] R.S. Gupta, Mol. Microbiol. 29 (1998) 695–707.

[14] T. Kauri, R. Wallace, D.J. Kushner, Syst. Appl. Microbiol. 13 (1990) 14–18.

[15] U.K. Laemmli, Nature 227 (1970) 680–685.

[16] A. Rehman, B.A. McFadden, Arch. Biochem. Biophys. 336 (1996) 309–315.

[17] F.M. Ausubel, R. Brent, R.E. Kingston, D.D. Moore, J.G. Seidman, J.A. Smith, K. Struhl, Short Protocols in Molecular Biology, John Wiley and Sons, New York, 1995.

[18] R.J. Harvey, S. Harder, M.G. Darlison, Elsevier Trends Journals Technical Tips Online, No. T01612 (1999).

[19] J.D. Thompson, D.G. Higgins, T.J. Gibson, Nucleic Acids Res. 22 (1994) 4673–4680.

[20] N. Saitou, M. Nei, Mol. Biol. Evol. 4 (1987) 406–425.

[21] R.D.M. Page, Comput. Appl. Biosci. 12 (1996) 357–358.

[22] P. Chomczynski, BioTechniques 15 (1993) 532–534.

[23] P. Chomczynski, Anal. Biochem. 201 (1992) 134–139.

[24] Boehringer Mannheim, Biochemica, The DIG System User’s Guide for Filter Hybridization, 1995.

[25] J.R. Palmer, C.J. Daniels, J. Bacteriol. 177 (1995) 1844–1849.

[26] J. Soppa, Mol. Microbiol. 31 (1999) 1589–1601.

[27] P.P. Dennis, Cell 89 (1997) 1007–1010.

[28] D.K. Thompson, J.R. Palmer, C.J. Daniels, Mol. Microbiol. 33 (1999) 1081–1092.

[29] K. Krüger, T. Hermann, V. Armbruster, F. Pfeifer, J. Mol. Biol. 279 (1998) 761–771.

[30] R. Cohen-Kupiec, C. Blank, J.A. Leigh, Proc. Natl. Acad. Sci. USA 94 (1997) 1316–1320.

[31] R. Cohen-Kupiec, C.J. Marx, J.A. Leigh, J. Bacteriol. 181 (1999) 256–261.

[32] R. Ken, N.R. Hackett, J. Bacteriol. 173 (1991) 955–960.

[33] M.J. Bonete, C. Pire, F.I. Llorca, M.L. Camacho, FEBS Lett. 383 (1996) 227–229.

[34] P. Vanni, E. Giachetti, G. Pinzauti, B.A. McFadden, Comp. Biochem. Physiol. 95B (1990) 431–458.

[35] M.J. Danson, D.W. Hough, Comp. Biochem. Physiol. 117A (1997) 307–312.

[36] H. Okada, M. Ueda, A. Tanaka, Arch. Microbiol. 144 (1986) 137–141.

[37] H. Durchschlag, G. Biedermann, H. Eggerer, Eur. J. Biochem. 114 (1981) 255–262.

[38] N. Guex, H. Henry, J. Flach, H. Richter, F. Widmer, Planta 197 (1995) 369–375.

[39] S. Hüttner, D. Mecke, K.U. Fröhlich, Gene 188 (1997) 239–246.

[40] I. Molina, M.T. Pellicer, J. Badia, J. Aguilar, L. Baldomá, Eur. J. Biochem. 224 (1994) 541–548.

[41] J.-C. Imbeault, R.L. Charlebois, Direct submission to GenBank, accession No. U95373, 1997.

[42] J.F. Heidelberg, J.A. Eisen, W.C. Nelson, R.A. Claydon, M.L. Gwinn, R.J. Dodson, D.H. Haft, E.K. Hickey, J.D. Peterson, L. Umayam, S.R. Gill, K.E. Nelson, T.D. Read, H. Tettelin, D. Richardson, M.D. Ermolaeva et al., Nature 406 (2000) 477–483.

[43] A. Oren, P. Gurevich, FEMS Microbiol. Ecol. 14 (1994) 147–156.
