
JOURNAL OF BACTERIOLOGY, Sept. 2001, p. 5262–5267
0021-9193/01/$04.00+0   DOI: 10.1128/JB.183.18.5262–5267.2001
Copyright © 2001, American Society for Microbiology. All Rights Reserved.

**Gene Cluster on pAO1 of *Arthrobacter nicotinovorans* Involved in Degradation of the Plant Alkaloid Nicotine: Cloning, Purification, and Characterization of 2,6-Dihydroxypyridine 3-Hydroxylase**

DANIEL BAITSCH,$^{1}$ CRISTINEL SANDU,$^{1}$ RODERICH BRANDSCH,$^{1\ast}$ AND GABOR L. IGLOI$^{2}$

*Institute of Biochemistry and Molecular Biology,*$^{1}$ *and Institute of Biology III,*$^{2}$  
*University of Freiburg, Freiburg, Germany*

Received 3 April 2001/Accepted 29 June 2001

A 27,690-bp gene cluster involved in the degradation of the plant alkaloid nicotine was characterized from the plasmid pAO1 of *Arthrobacter nicotinovorans*. The genes of the heterotrimeric, molybdopterin cofactor (MoCo)-, flavin adenine dinucleotide (FAD)-, and [Fe-S] cluster-dependent 6-hydroxypseudooxynicotine (ketone) dehydrogenase (KDH) were identified within this cluster. The gene of the large MoCo subunit of KDH was located 4,266 bp from the FAD and [Fe-S] cluster subunit genes. Deduced functions of proteins encoded by open reading frames (ORFs) of the cluster were correlated to individual steps in nicotine degradation. The gene for 2,6-dihydroxypyridine 3-hydroxylase was cloned and expressed in *Escherichia coli*. The purified homodimeric enzyme of 90 kDa contained 2 mol of tightly bound FAD per mol of dimer. Enzyme activity was strictly NADH-dependent and specific for 2,6-dihydroxypyridine. 2,3-Dihydroxypyridine and 2,6-dimethoxypyridine acted as irreversible inhibitors. Additional ORFs were shown to encode hypothetical proteins presumably required for holoenzyme assembly, interaction with the cell membrane, and transcriptional regulation, including a MobA homologue predicted to be specific for the synthesis of the molybdopterin cytidine dinucleotide cofactor.

The gram-positive soil bacterium *Arthrobacter nicotinovorans* (formerly known as *Arthrobacter oxidans* and reclassified by Kodama et al. [22]) has the ability to use nicotine as its sole carbon and energy source (8, 10, 15, 17). Nicotine, the alkaloid of the tobacco plant, is synthesized as the L-isomer, and the first enzyme to attack L-nicotine is a trimeric, molybdopterin cofactor (MoCo) (most probably in its dinucleotide form [18])- , flavin adenine dinucleotide (FAD)-, and [Fe-S] cluster-containing nicotine dehydrogenase (NDH), which hydroxylates the pyridine ring at position 6 (Fig. 1, step I). The pyrrolidine ring of 6-hydroxy-L-nicotine is then oxidized by 6-hydroxy-L-nicotine oxidase (6HLNO) (7) (Fig. 1, step II), and the 6-hydroxypseudooxynicotine formed is once more hydroxylated at position 2 of the pyridine ring by ketone dehydrogenase (KDH) (8, 30) (Fig. 1, step III), an enzyme structured similarly to NDH. The side chain of N-methylaminopropyl-(2,6-dihydroxypyridyl-3)-ketone is cleaved off, resulting in 2,6-dihydroxypyridine and γ-methylaminobutyrate (Fig. 1, step IV). These two compounds have been identified as degradation products of nicotine by *A. nicotinovorans* (15), but no corresponding enzyme has been identified as yet. 2,6-Dihydroxypyridine is transformed into 2,3,6-trihydroxypyridine by 2,6-dihydroxypyridine 3-hydroxylase (2,6-DHPH) (Fig. 1, step V), an FAD-dependent enzyme that had been partially purified before (19). In the presence of oxygen, 2,3,6-trihydroxypyridine spontaneously forms a blue pigment (20) (Fig. 1), and liquid cultures of *A. nicotinovorans* grown on nicotine turn dark blue because of the nicotine blue secreted by the bacteria. The genes encoding these enzymes are located on the 160-kb catabolic plasmid pAO1, and the ability to degrade and grow on nicotine is lost when the plasmid is cured from the bacterium (5). The genes *ndh* and *6hlno* have been cloned and sequenced (16, 31). Of the *kdh* genes, the gene of the small [Fe-S] subunit and of the medium FAD subunit have been cloned and sequenced (30). The gene of the large MoCo subunit, however, could not be established unequivocally, and the organization of the genes on pAO1 has not been identified. It has been shown before that adjacent to the *ndh* and *6hlno* genes, and separated by the insertion sequence IS1473 (26), is a gene cluster consisting of a molybdate ABC transporter and genes involved in MoCo biosynthesis (27).

Here we present the organization of the genes on pAO1 involved in the degradation of nicotine, the identification and cloning of the 2,6-DHPH gene in *Escherichia coli*, and the purification and characterization of the enzyme.

---

*MATERIALS AND METHODS*

DNA sequencing and sequence analysis. pAO1 of *A. nicotinovorans* was isolated according to published methods (5) and partially digested with the restriction enzyme Sau3A. The digest was electrophoresed on a 0.8% agarose gel, and DNA fragments of 2.5 to 10 kb were isolated and cloned into pBlueScript SK (Stratagene, Heidelberg, Germany). In addition, a pAO1 library constructed in the lambda Zap Express vector (Stratagene) was kindly provided by K. Decker (30). Starting from a 10-kb pAO1 fragment in pBlueScript SK (27), overlapping clones were identified by sequencing individual clones and gaps were filled by sequencing PCR products obtained with pAO1 DNA as a template and the Expand Long Template PCR system of Roche Molecular Biochemicals (Mannheim, Germany). Contigs were assembled and edited using the Staden software

* Corresponding author. Mailing address: Institute of Biochemistry and Molecular Biology, Hermann-Herder-Str. 7, D-79104 Freiburg, Germany. Phone: 49-761-203-5231. Fax: 49-761-203-5253. E-mail: brandsch@ruf.uni-freiburg.de.

VOL. 183, 2001

nic GENE CLUSTER OF pAO1 5263

L-Nicotine $\stackrel{\text{I}}{\longrightarrow}$ NDH $\stackrel{\text{II}}{\longrightarrow}$ 6HLNO $\stackrel{\text{III}}{\longrightarrow}$ KDH

$\begin{aligned} & \text { N-Methylaminopropyl- } \\ & (6\text{-Hydroxypyridyl-3)-Ketone } \\ & (\text { Pseudooxynicotine }) \end{aligned}$

$\begin{aligned} & \text { N-Methylaminopropyl- } \\ & (2,6\text{-Dihydroxypyridyl-3)-Ketone } \end{aligned}$

IV enzyme unknown

hypothetical ring cleavage

$2,3,6\text{-Trihydroxypyridine}$

$2,6\text{-Dihydroxypyridine } \gamma\text{-N-Methylaminobutyric acid}$

O₂ spontaneously

blue pigment

FIG. 1. Overview of the steps in nicotine degradation to blue pigment by *A. nicotinovorans* pAO1. The reaction catalyzed by 2,6-DHPH, cloned and purified in this work, is framed. NDH, nicotine dehydrogenase; 6HLNO, 6-hydroxy-L-nicotine oxidase; KDH, ketone dehydrogenase.

package (3). Comparisons of DNA sequences and their derived amino acid sequences were performed with the BLAST family of programs (1).

Cloning and purification of 2,6-DHPH. The DNA fragment bearing the 2,6- *dhph* gene was amplified with *Pfu* polymerase (Stratagene) from the pAO1 DNA template, digested with *Xma*I and *Kpn*I, ligated into the *Xma*I-*Kpn*I sites of pH6EX3 (2), and transformed into *E. coli* XL-1 blue bacteria. His₆-tagged 2,6-DHPH was purified from bacterial lysates by Ni²⁺-chelating Sepharose chromatography as recommended by the supplier (Amersham Pharmacia Biotech, Freiburg, Germany).

Enzyme assays. 2,6-DHPH activity was assayed photometrically at 25°C in 50 mM potassium phosphate buffer (pH 7.1) in the presence of 0.2 mM NADH and 0.1 mM 2,6-dihydroxypyridine using a digital photometer (Eppendorf, Hamburg, Germany). Enzyme activity was recorded either at 334 nm as the decrease in NADH consumed in the reaction or as the increase in absorption at 578 nm due to the formation of the blue pigment, which was generated from the reaction product 2,3,6-trihydroxypyridine. One unit of enzyme was defined as the amount of enzyme that catalyzed the oxidation of 1 μmol of NADH per min under the assay conditions. The $K_m$ of the enzyme for 2,6-dihydroxypyridine was determined in assays recorded at 334 nm at seven substrate concentrations ranging from 100 μM to 500 nM. The $K_d$ of the enzyme for NADH was determined at a substrate concentration of 100 μM and at seven NADH concentrations ranging from 200 to 10 μM. The substrate specificity of 2,6-DHPH was tested with various compounds obtained from Sigma (Munich, Germany) (Table 1) at a concentration of 100 μM in the enzyme assay with NADH. The pH optimum of 2,6-DHPH was determined at pH values from 10.0 to 5.0 in 50 mM potassium phosphate buffer.

Preparation of apo-His₆-tagged 2,6-DHPH. To estimate the $K_d$ for FAD, apo-His₆-tagged 2,6-DHPH was prepared by precipitation with 50% $(NH_4)_2SO_4$ at pH 2.0 for 30 min on ice. The precipitated apoenzyme was collected by centrifugation at 10,000 × g for 15 min at 4°C and resuspended in 50 mM potassium phosphate buffer (pH 7.1). This preparation was employed to reconstitute 2,6-DHPH holoenzyme by incubation of the apoenzyme for 1 h on ice with FAD concentrations ranging from 500 nM to 500 μM.

Nucleotide sequence accession number. The 27,690-bp sequence of pAO1 DNA was deposited at GenBank with accession number AF373840.

RESULTS

A gene cluster on pAO1 of *A. nicotinovorans* is involved in nicotine degradation. The genes and open reading frames (ORFs) identified on the 27,690-bp pAO1 DNA fragment are presented in Fig. 2. The gene cluster starts with the *ndhM*, *ndhS*, *ndhL*, and *6hlno* genes (16). For more clarity, we have used the terms large (L), medium (M), and small (S) instead of C, A, and B for the nomenclature of the subunits of the trimeric MoCo, FAD, [Fe-S] cluster enzymes. The ORFs of *kdhs* and *kdhm* were identified further downstream as being transcribed in the opposite direction to the *ndh* and *6hlno* genes. The deduced amino-terminal sequences of KDHS and KDHM correspond to the experimentally determined amino acid sequences XAFRLTVEVNGVTH and MKPPSFDYVVADSVEHALRLLADG, respectively (29). X in the sequence of the small subunit stands for asparagine. The start of KDHM is extended from the previously assumed start site by 43 amino acid residues and contains the experimentally determined amino-terminal sequence MKPPSFDYVVADSVEHALRLLADG (29), which results in a protein with a calculated $M_r$ of 31,429. The *ndh* and *6hlno* genes are separated from the *kdhs* and *kdhm* genes by small, hypothetical ORFs (Fig. 2). Contrary to expectations, *kdhm* was not preceded by an ORF encoding a protein with similarity to known large subunits of MoCo enzymes. However, a corresponding ORF (Fig. 2, *kdhl*) was located 4,266 bp downstream from *kdhm* and transcribed in the opposite direction. The deduced amino acid sequence starts with MMAKAKALIPDNGRA and contains the sequence ALIPDNXXA, which had been experimentally de-

TABLE 1. Substrate specificity of 2,6-DHPH$^a$

| Substrate                     | Specific activity (U/mg of protein) | % Activity in presence of 2,6-DHP |
|------------------------------|------------------------------------|-----------------------------------|
| 2,6-Dihydroxypyridine         | 90                                 | 100                               |
| 2-Hydroxypyridine             | 0                                  | 52                                |
| 2,3-Dihydroxypyridine         | 0                                  | 0                                 |
| 2,6-Dimethoxypyridine         | 0                                  | 0                                 |
| Resorcinol                    | 0                                  | 49                                |
| 2,6-Dihydroxynicotinamide     | 0                                  | 45                                |

$a$ One unit is defined as the conversion of 1 μmol of 2,6-dihydroxypyridine (2,6-DHP) to 1 μmol of 2,3,6-trihydroxypyridine per min.

VOL. 183, 2001                                                                                             nic GENE CLUSTER OF pAO1 5265

![Diagram](attachment:diagram.png)

FIG. 2. Characterization of the 27,690-bp region of *A. nicotinovorans* pAO1 involved in nicotine degradation, showing the physical and genetic map of the *nic* gene cluster. Genes encoding known enzymes of nicotine degradation are shaded dark gray and are indicated by the corresponding abbreviation. ORFs proposed to encode proteins involved in nicotine degradation are shaded light gray, ORFs encoding hypothetical transcription factors are indicated in black, and ORFs with no similarity to known ORFs in data banks are unshaded. Gaps between gene subclusters are indicated in base pairs (bp) above the schematically drawn ORFs. Tn, transposase.

U/mg at the pH optimum of 8.0 and at the temperature optimum of 20°C. Under these conditions, the enzyme showed a \( K_m \) of \( 8.3 \times 10^{-6} \) M, a \( V_{\text{max}} \) of \( 4.7 \times 10^{-5} \) mol/min, and a \( k_{\text{cat}} \) of \( 3.9 \times 10^{-6} \) s\(^{-1}\). In contrast to what was observed by Holmes and Rittenberg (19) with a partially purified enzyme preparation, the purified enzyme was stable at 4°C and no rapid inactivation at 30°C was observed. The enzyme was inactivated at 52°C. Table 1 summarizes the enzyme activities obtained with substrate analogs. Only the pyridine ring hydroxylated in positions 2 and 6 served as a substrate. 2,3-Dihydroxypyridine and 2,6-dimethoxypyridine acted as irreversible inhibitors.

DISCUSSION

The deduced protein sequences of the ORFs of the gene cluster of pAO1 of *A. nicotinovorans* show a significant degree of similarity to known or hypothetical proteins of *Streptomyces* and *Mycobacterium* species. This finding may reflect the general relatedness of the genus *Arthrobacter* with *Streptomyces* and *Mycobacterium*. It may, however, also reflect gene transfer by catabolic plasmids, like pAO1, between species of these genera of soil bacteria. A gene cluster structured similarly to the *nic* gene cluster on pAO1 may be found on the *Mycobacterium tuberculosis* chromosome. It consists of the ORFs Rv0374c, Rv0375c, and Rv0373c, encoding the large, small,

TABLE 2. Features associated with ORFs of the *nic* gene cluster

| Designation\(^a\) | Basis of comparison | % Similarity/% identity to published sequence (accession no.) in: |
|--------------------|---------------------|---------------------------------------------------------------|
|                    |                     | Streptomyces                                                   |
| ORF 124           | Secreted hypothetical protein | 80/39in112aa (Q9KXP7)                                      |
| ORF 78            | Secreted hypothetical protein | 61/35in70aa (Q9KXP8)                                       |
| ORF 64            | None                |                                                               |
| ORF 103           | None                |                                                               |
| ORF 393           | Putative regulatory protein | 67/37in100aa (Q9XAJ6)                                      |
| ORF 407           | LuxR-type activator | 64/37in124aa (Q9XAJ6)                                         |
| ORF 116           | Hypothetical protein | 61/28in99aa (Q9KZG3)                                          |
| ORF 367           | Rapamycin resistance | 60/31in207aa (Q54287)                                         |
| ORF 106           | Putative hydrolase   | 74/53in19aa (Q9RK75)                                          |
| ORF 310           | Polyketide cyclase   | 64/31in85aa (Q9RN54)                                          |
| ORF 235           | CoxG                 | 78/40in151aa (O53704)                                         |
| ORF 294           | C-N hydrolase        | 70/43in221aa (Q9XA70)                                         |
| ORF 297           | MoxR                 | 76/47in278aa (O53705)                                         |
| ORF 363           | CoxE                 | 66/35in216aa (O53703)                                         |
| ORF 377           | CoxI, CoxF           | 77/48in168aa (Q9ZBN5)                                         |
| ORF 117           | None                |                                                               |
| ORF 204           | MobA                 | 74/45in188aa (Q9RKU8)                                         |
| ORF 70            | None                |                                                               |
| ORF 223           | None                |                                                               |
| ORF 96            | Δ Transposase       | 66/41in53aa (Q49592)                                          |

| Mycobacterium      | Other genus         |
|--------------------|---------------------|
| 63/37in99aa (Q06829) |                     |
| 60/28in261aa (Q50018) |                     |
|                     |                     |
|                     |                     |
|                     |                     |
|                     |                     |
|                     |                     |
|                     |                     |
|                     |                     |
|                     |                     |
| 65/23in138aa\(^b\) (Q9KX24) |               |
|                     | 82/50in273aa\(^c\) (Q9RWP5) |
|                     | 63/31in348aa\(^b\) (QKX26)  |
|                     | 45/29in98aa\(^b\) (Q9KX25)  |
|                     | 53/32in94aa\(^b\) (Q9KX22)  |
| 68/36in112aa (O53706) |                     |
|                     |                     |
|                     |                     |

\(^a\) The locations of the ORFs within the DNA fragment are documented in GenBank, accession no. AF373840. aa, amino acids.

\(^b\) Sequence from *Oligotropha carboxidovorans*.

\(^c\) Sequence from *D. radiodurans*.

and medium-sized subunits, respectively, of a hypothetical molybdenum enzyme, and the ORFs Rv0368c, Rv0369c, Rv0370c, Rv0371c, and Rv0376c (6) with similarity to the pAO1 ORF 368, ORF 235, ORF 363, MobA, and ORF 377 (Table 2). The IS1473 element at one end of the gene cluster and the transposase-similar ORF on the other end of the gene cluster make a transposition event in the origin of the gene cluster on pAO1 likely.

The known and the deduced enzymatic functions of the predicted products of the ORFs of the *nic* gene cluster may be correlated to individual steps in nicotine degradation. The first two hydroxylations of the pyridine ring of nicotine in positions 6 and 2 (Fig. 1, steps I and III) are performed by the related, heterotrimeric, FAD-, [Fe-S] cluster-, and MoCo-dependent enzymes NDH and KDH. Oxidation of the pyrrolidine ring is performed by 6HLNO (Fig. 1, step II). *A. nicotinovorans* extracts prepared from bacteria grown on D,L-nicotine contain 6-hydroxy-D-nicotine oxidase activity (9). The gene of this enzyme is not part of the gene cluster described here, but it was located on pAO1, 15,738 bp downstream from the IS1473 (G. L. Igloi and R. Brandsch, unpublished results). Since D-nicotine is not synthesized by the plant, the natural substrate of 6-hydroxy-D-nicotine oxidase remains speculative. Cleavage of the side chain of 2,6-dihydroxypseudooxynicotine (Fig. 1, step IV) may be performed by the products of ORF106 and ORF310 preceding the ORF for 2,6-DHPH, with which they may form a translational unit (Fig. 2). γ-Methylaminobutyrate and 2,6-dihydroxypyridine were identified as the products of this reaction (15, 20). Gherna et al. (15) formulated the reaction as a hydrolysis and pointed out that, although the biochemical hydrolytic replacement of the side chain of an aromatic ring is unusual, the reaction has precedence in the thiolysis of β-keto-acyl-coenzyme A compounds. The similarity of the hypothetical protein of ORF310 preceding 2,6-DHPH to thiol esterases fits this assumption. The gene encoding 2,6-DHPH, which performs the third hydroxylation of the pyridine ring (Fig. 1, step V), has been identified during this work, cloned, and overexpressed in *E. coli*. The purified protein was shown to be 2,6-DHPH. We propose that ring opening of 2,3,6-trihydroxypyridine is performed by the hypothetical protein, similar to endopeptidases (Table 2, ORF367) at the peptide bond of the hydroxylated pyridine ring in its amide resonance form (Fig. 1). Ring opening was proposed to be performed by a dioxygenase, in analogy to 2,5-dihydroxypyridine ring opening by 2,5-dihydroxypyridine oxygenase in the degradation of nicotinic acid (14). However, no ORF encoding a putative dioxygenase was identified in this pAO1 gene cluster. It cannot be excluded that such a dioxygenase is encoded on pAO1, but from the organization of the gene cluster we would expect such a gene to be part of the cluster. The hypothetical nitrilase would then remove the amino group from the linearized compound and the carbon skeleton would enter the general metabolism.

The identification of the *kdh* gene for the large subunit revealed that it forms a separate transcriptional unit from the *kdhM* and *kdhS* genes. To our knowledge, this is the first instance that such a gene arrangement has been found for a bacterial trimeric molybdoenzyme. This finding raises the question of how the coordinated expression of the KDH subunits is regulated. This regulation appears to be complex, given

the fact that two divergently transcribed putative transcriptional regulators are positioned within the cluster of genes related to enzymes of nicotine degradation. None of these regulators shows similarity to the molybdenum-dependent transcriptional regulator ModE of *E. coli* (25, 32), although expression of *ndh* was shown to be molybdenum dependent (16).

The hypothetical MobA protein for the molybdopterin cytidine dinucleotide cofactor is encoded by an ORF which may form a transcriptional unit with ORFs of no known functions but that are presumed to be involved in the assembly of the MoCo holoenzymes, attachment of the enzymes to the cell membrane, and interaction with the respiratory chain (18, 28). MobA could deliver, in the context of these proteins, the molybdopterin cytidine dinucleotide cofactor efficiently to the membrane-associated KDH and NDH apoenzymes (16).

2,6-DHPH is specific for the heterocyclic aromatic compound 2,6-dihydroxypyridine. 3-Hydroxyphenol (resorcinol) was not a substrate. In addition, the substrate must be hydroxylated in positions 2 and 6. 2,3-Dihydroxypyridine was not accepted as substrate and acted, as did 2,6-dimethoxypyridine, as an irreversible inhibitor. The monohydroxylated compound 2-hydroxypyridine was found to be a reversible inhibitor of the reaction. Thus, the enzyme showed a narrow substrate specificity. Flavoprotein hydroxylases belong to an enzyme family characterized by three amino acid fingerprint motifs involved in FAD and NAD(P)H binding (11, 12). 2,6-DHPH clearly belongs to the family of flavoprotein hydroxylases. However, some remarkable differences are also evident. The fingerprint motif GXGXXG of FAD- and NAD-dependent enzymes is altered to GXSXXG. In the second characteristic motif of this family, DXXXGXDGXK, which is involved in both FAD and NAD(P)H binding, two charged residues are replaced by uncharged polar (D→N) or uncharged hydrophobic (K→A) residues. It has been shown by chemical modification that the K in salicylate hydroxylase is important for NADH binding (34), and similar results were obtained with mutants of parahydroxybenzoate hydroxylase (13). In the third fingerprint motif of this family GDAAH, the H residue is replaced in 2,6-DHPH by V. Despite these alterations in residues shown to be important for cofactor binding, FAD is nevertheless tightly bound by the enzyme. There are several FAD-dependent hydroxylases known, with a loosely bound flavin cofactor, like 4-hydroxybenzoate hydroxylase (33). However, this enzyme shows the same conserved amino acid sequences as those with tightly bound FAD (11, 12). Apparently, additional amino acid residues other than those of the deduced fingerprint motifs may stabilize the interaction of the apoenzymes with FAD in some enzymes of this family.

### ACKNOWLEDGMENTS

We thank E. Schiefermayr and I. Deuchler for excellent technical assistance.

This work was supported by a grant of the Deutsche Forschungsgemeinschaft, GRK 434, to R.B. and, in part, by SFB 388 to G.L.I.

### REFERENCES

1. Altschul, S. F., W. Gish, W. Miller, E. W. Myers, and D. J. Lipman. 1990. Basic local alignment search tool. J. Mol. Biol. 215:403–410.
2. Berthold, H., M. Scanarini, C. C. Abney, B. Frorath, and W. Northemann. 1992. Purification of recombinant antigenic epitopes on the human 68-kDa

(U1) ribonucleoprotein antigen using the expression system pH6EX3 followed by metal chelating affinity chromatography. Protein Expr. Purif. 3:50–56.

3. Bonfield, J. K., K. F. Smith, and R. Staden. 1965. A new DNA sequence assembly program. Nucleic Acids Res. 23:4992–4999.
4. Bork, P., and E. V. Koonin. 1994. A new family of carbon-nitrogen hydrolases. Protein Sci. 3:1344–1346.
5. Brandsch, R., and K. Decker. 1984. Isolation and partial characterization of plasmid DNA from *Arthrobacter oxydans*. Arch. Microbiol. 138:15–17.
6. Cole, S. T., R. Brosch, J. Parkhill, et al. 1998. Deciphering the biology of *Mycobacterium tuberculosis* from the complete genome sequence. Nature 393:537–544.
7. Dai, V. D., K. Decker, and H. Sund. 1968. Purification and properties of L-6-hydroxynicotine oxidase. Eur. J. Biochem. 4:95–102.
8. Decker, K., F. A. Gries, and M. Brühmüller. 1961. Über den Abbau des Nikotins durch Bakterienenzyme. III. Stoffwechselstudien an Zellfreien Extrakten. Hoppe-Seyler’s Z. Physiol. Chem. 323:249–263.
9. Decker, K., and H. Bleeg. 1965. Induction and purification of stereospecific nicotine oxidizing enzymes from *Arthrobacter oxydans*. Biochim. Biophys. Acta 105:313–334.
10. Eberwein, H., F. A. Gries, and K. Decker. 1961. Über den Abbau des Nikotins durch Bakterienenzyme. II. Isolierung und Charakterisierung eines Nikotinabbauenden Bodenbakteriums. Hoppe-Seyler’s Z. Physiol. Chem. 323:236–248.
11. Eppink, M. H. M., A. H. Schreuder, and W. J. H. van Berkel. 1997. Identification of a novel conserved sequence motif in flavoprotein hydroxylases with a putative dual function in FAD/NAD(P)H binding. Protein Sci. 6:2454–2458.
12. Eppink, M. H. M., S. A. Boeren, J. Vervoort, and W. J. H. van Berkel. 1997. Purification and properties of 4-hydroxybenzoate 1-hydroxylase (decarboxylating), a novel flavin adenine dinucleotide-dependent monooxygenase from *Candida parapsilosis* CBS604. J. Bacteriol. 179:6680–6687.
13. Eppink, M. H. M., C. Bunthof, H. A. Schreuder, and W. J. H. van Berkel. 1999. Phe161 and Arg166 variants of p-hydroxybenzoate hydroxylase implications for NADPH recognition and structural stability. FEBS Lett. 443:251–255.
14. Gauthier, J. J., and S. C. Rittenberg. 1971. The metabolism of nicotinic acid. I. Purification and properties of 2,5-dihydroxypyridine oxygenase from *Pseudomonas putida* N-9. J. Biol. Chem. 246:3737–3742.
15. Gherna, R. L., S. H. Richardson, and S. C. Rittenberg. 1965. The bacterial oxidation of nicotine. VI. The metabolism of 2,6-dihydroxypseudooxynicotine. J. Biol. Chem. 240:3669–3674.
16. Grether-Beck, S., G. L. Igloi, S. Pust, E. K. Schiltz, and R. Brandsch. 1994. Structural analysis and molybdenum-dependent expression of the pAO1-encoded nicotine dehydrogenase genes of *Arthrobacter nicotinovorans*. Mol. Microbiol. 13:929–936.
17. Hamm, H.-H., and K. Decker. 1978. Regulation of flavoprotein synthesis in vivo in a riboflavin-requiring mutant of *Arthrobacter oxydans*. Arch. Microbiol. 119:65–70.
18. Hänselmann, P., and O. Meyer. 1998. Effect of molybdate and tungstate on the biosynthesis of CO dehydrogenase and the molybdopterin cytosine-dinucleotide-type of molybdenum cofactor in *Hydrogenophaga pseudoflava*. Eur. J. Biochem. 255:755–765.
19. Holmes, P. E., and S. C. Rittenberg. 1972. The bacterial oxidation of nicotine. VII. Partial purification and properties of 2,6-dihydroxypyridine oxidase. J. Biol. Chem. 247:7622–7627.
20. Holmes, P. E., and S. C. Rittenberg. 1972. The bacterial oxidation of nicotine. VIII. Synthesis of 2,3,6-trihydroxypyridine and accumulation and partial characterization of the product of 2,6-dihydroxypyridine oxidation. J. Biol. Chem. 247:7628–7633.
21. Husain, M., and V. Massey. 1978. Reversible resolution and reconstitution of flavoproteins into apoproteins and free flavin. Methods Enzymol. 53:429–437.
22. Kodama, Y., H. Yamamoto, N. Amano, and T. Amachi. 1992. Reclassification of two strains of *Arthrobacter oxydans* and proposal of *Arthrobacter nicotinovorans* sp. nov. Int. J. Syst. Bacteriol. 42:234–239.
23. Lomovskaya, N., Y. Doi-Katayama, S. Filippini, C. Nastro, L. Fonstein, M. Gallo, A. L. Colombo, and C. R. Hutchinson. 1998. The *Streptomyces peucetius dpsY* and *dnrX* genes govern early and late steps of daunorubicin and doxorubicin biosynthesis. J. Bacteriol. 180:2379–2386.
24. Marti, T., Z. Hu, N. L. Pohl, A. N. Shah, and C. Khosla. 2000. Cloning, nucleotide sequence, and heterologous expression of the biosynthetic gene cluster for R1128, a non-steroidal estrogen receptor antagonist. Insights into an unusual priming mechanism. J. Biol. Chem. 275:33443–33448.
25. McNicholas, P. M., M. M. Mazotta, S. A. Reich, and R. P. Gunsalus. 1998. Functional dissection of the molybdate-responsive transcription regulator, ModE, from *Escherichia coli*. J. Bacteriol. 180:4638–4643.
26. Menéndez, C., G. L. Igloi, and R. Brandsch. 1997. IS1473, a putative insertion sequence identified in the plasmid pAO1 from *Arthrobacter nicotinovorans*: isolation, characterization and distribution among *Arthrobacter* species. Plasmid 37:35–41.
27. Menéndez, C., A. Otto, G. L. Igloi, P. Nick, R. J. Brandsch, B. Schubach, B. Böttcher, and R. K. Brandsch. 1997. Molybdate-uptake genes and molybdopterin-biosynthesis genes on a bacterial plasmid. Eur. J. Biochem. 250:524–531.
28. Santiago, B., U. Schübel, C. Egelseer, and O. Meyer. 1999. Sequence analysis, characterization and CO-specific transcription of the *cox* gene cluster on the megaplasmid pHCG3 of *Oligotropha carboxidovorans*. Gene 236:115–124.
29. Schelling, U. 1995. Ph.D. dissertation. Albert Ludwig University, Freiburg, Germany.
30. Schenk, S., A. Hoelz, B. Kraus, and K. Decker. 1998. Gene structure and properties of enzymes of the plasmid-encoded nicotine catabolism of *Arthrobacter nicotinovorans*. J. Mol. Biol. 284:1323–1339.
31. Schenk, S., and K. Decker. 1999. Horizontal gene transfer involved in the convergent evolution of the plasmid-encoded enantioselective 6-hydroxynicotine oxidases. J. Mol. Evol. 48:178–186.
32. Self, W. T., A. M. Grunden, A. Hasona, and K. T. Shanmugam. 1999. Transcriptional regulation of molybdoenzyme synthesis in *Escherichia coli* in response to molybdenum: ModE-molybdate, a repressor of the *modABCD* (molybdate transport) operon is a secondary transcriptional activator for the *hyc* and *nar* operons. Microbiology 145:41–55.
33. Siebold, B., M. Matthes, M. H. M. Eppink, F. Lingens, W. J. H. van Berkel, and R. Müller. 1996. 4-Hydroxybenzoate hydroxylase from *Pseudomonas* sp. CBS3. Purification, characterization, gene cloning, sequence analysis and assignment of structural features determining the coenzyme specificity. Eur. J. Biochem. 239:469–478.
34. Suzuki, K., E. Asao, Y. Nakamura, K. Ohnishi, S. Fukuda. 2000. Overexpression of salicylate hydroxylase and the crucial role of lys(163) as its NADH binding site. J. Biochem. 128:293–299.
35. Torkkell, S., K. Ylihonko, J. Hakala, M. Skurnik, and P. Mantsala. 1997. Characterization of *Streptomyces nogalater* genes encoding enzymes involved in glycosylation steps in nogalamycin biosynthesis. Mol. Gen. Genet. 256:203–209.
