
Molecular Microbiology (2001) 41(5), 1159–1172

# A novel two-component regulatory system in *Bacillus subtilis* for the survival of severe secretion stress

Hanne-Leena Hyyryläinen,$^{1}$ Albert Bolhuis,$^{2†}$ Elise Darmon,$^{2}$ Leila Muukkonen,$^{1}$ Pertti Koski,$^{1}$ Marika Vitikainen,$^{1}$ Matti Sarvas,$^{1}$ Zoltán Prágai,$^{3}$ Sierd Bron,$^{2}$ Jan Maarten van Dijl$^{4*}$ and Vesa P. Kontinen$^{1}$

$^{1}$ Laboratory of Vaccine Development, National Public Health Institute, FIN-00300, Helsinki, Finland.  
$^{2}$ Department of Genetics, Groningen Biomolecular Sciences and Biotechnology Institute, Kerklaan 30, 9751 NN Haren, The Netherlands.  
$^{3}$ Department of Microbiology and Immunology, The Medical School, University of Newcastle upon Tyne, Newcastle upon Tyne NE2 4HH, UK.  
$^{4}$ Department of Pharmaceutical Biology, University of Groningen, Antonius Deusinglaan 1, 9713 AV Groningen, The Netherlands.

## Summary

The Gram-positive eubacterium *Bacillus subtilis* is well known for its high capacity to secrete proteins into the environment. Even though high-level secretion of proteins is an efficient process, it imposes stress on the cell. The present studies were aimed at the identification of systems required to combat this so-called secretion stress. A two-component regulatory system, named CssR–CssS, was identified, which bears resemblance to the CpxR–CpxA system of *Escherichia coli*. The results show that the CssR/S system is required for the cell to survive the severe secretion stress caused by a combination of high-level production of the α-amylase AmyQ and reduced levels of the extracytoplasmic folding factor PrsA. As shown with a *prsA3* mutation, the Css system is required to degrade misfolded exported proteins at the membrane–cell wall interface. This view is supported by the observation that transcription of the *htrA* gene, encoding a predicted membrane-bound protease of *B. subtilis*, is strictly controlled by CssS. Notably, CssS represents the first identified sensor for extracytoplasmic protein misfolding in a Gram-positive eubacterium. In conclusion, the results show that quality control systems for extracytoplasmic protein folding are not exclusively present in the periplasm of Gram-negative eubacteria, but also in the Gram-positive cell envelope.

## Introduction

Eubacteria have various regulatory systems to cope with stresses that cause protein misfolding or damage in the cytoplasm and cell envelope. Cytoplasmic stress response systems direct the upregulation of genes encoding heat shock proteins, molecular chaperones and proteases. Such systems have been studied for many years, and their modes of action are fairly well understood (reviewed by Hecker *et al.*, 1996; Yura and Nakahigashi, 1999). Compared with the cytoplasmic stress response systems, relatively little is known about extracytoplasmic stress response systems. In fact, studies describing extracytoplasmic stress response systems have, thus far, been limited to Gram-negative eubacteria, *Escherichia coli* in particular.

In the cell envelope of *E. coli*, the accumulation of misfolded or aggregated proteins is sensed and combated by two partially overlapping signal transduction pathways, the Cpx two-component regulatory system and the σ<sup>E</sup>-mediated stress response pathway (Connolly *et al.*, 1997; Danese and Silhavy, 1997; Pogliano *et al.*, 1997). The Cpx system consists of the sensor histidine kinase CpxA and the cognate response regulator CpxR. At the transcriptional level, it controls the synthesis of periplasmic proteins required for the efficient folding of periplasmic and outer membrane proteins and the degradation of their misfolded or aggregated forms. Thus, the synthesis of proteins catalysing disulphide bond formation (i.e. DsbA) and peptidyl-prolyl isomerization (i.e. PpiA, RotA and PpiD), as well as the synthesis of the periplasmic protease HtrA (also known as DegP), is elevated when the Cpx system is activated (Danese *et al.*, 1997; Missiakas and Raina, 1997; Pogliano *et al.*, 1997; Dartigalongue and Raina, 1998). Interestingly, HtrA also acts as a chaperone at low temperatures (Spiess *et al.*, 1999). Taken together, these observations indicate that the Cpx system plays a major role in the quality control of exported proteins. In contrast to the positively regulated target genes involved in protein quality control, the Cpx system negatively

Accepted 13 June, 2001. *For correspondence. E-mail J.M.van.Dijl@farm.rug.nl; Tel. (+31) 50 363 3079; Fax (+31) 50 363 3000. †Present address: Molecular Cell Biology, Department of Biological Sciences, University of Warwick, Coventry CV4 7AL, UK.*

© 2001 Blackwell Science Ltd

1160 H.-L. Hyyryläinen et al.

regulates the expression of genes for proteins involved in cell motility and chemotaxis (De Wulf *et al.*, 1999). Furthermore, the Cpx system of *Shigella sonnei* has been shown to regulate the expression of certain virulence genes (Nakayama and Watanabe, 1998).

The sigma factor $\sigma^E$, the cognate anti-sigma factor RseA and the periplasmic protein RseB are the key components of the $\sigma^E$-mediated stress response pathway (reviewed by Pallen and Wren, 1997; Missiakas and Raina, 1998). According to current models, the inner membrane-bound RseA protein sequestrates $\sigma^E$ in the absence of extracellular stress (De Las Peñas *et al.*, 1997; Missiakas *et al.*, 1997). Under certain stress conditions, RseB senses the accumulation of misfolded proteins in the periplasm and triggers the release of $\sigma^E$ from RseA by an as yet unknown mechanism. The released $\sigma^E$ then activates genes of the $\sigma^E$ regulon, one of which is *htrA* (Raina *et al.*, 1995; Rouvière *et al.*, 1995).

In contrast to the Gram-negative eubacterium *E. coli*, the Gram-positive eubacterium *B. subtilis* lacks an outer membrane, having a thick matrix of cell wall polymers instead (Archibald *et al.*, 1993). As a consequence, a periplasmic space confined by two membranes is absent from this organism. Upon translocation across the cytoplasmic membrane, secretory proteins fold into their native conformation at the membrane–cell wall interface, which is followed by their passage through the wall and release into the environment. The foldase PrsA (Kontinen *et al.*, 1991; Jacobs *et al.*, 1993; Leskelä *et al.*, 1999a), thiol-disulphide oxidoreductases (Bolhuis *et al.*, 1999a) and negatively charged cell wall polymers (Stephenson *et al.*, 1998; Hyyryläinen *et al.*, 2000) influence these post-translocational phases in protein secretion (for a review, see Tjalsma *et al.*, 2000).

The presence of three paralogous genes (*ykdA*, *yvtA* and *yyxA*) for putative membrane-bound HtrA-like proteases in *B. subtilis* suggests that, as in Gram-positive eubacteria, misfolded or aggregated proteins are removed

A  
**B. subtilis cssS::pMutin2**

integrated pMutin2  

![Diagram A](attachment://diagram_a.png)

B  
**B. subtilis cssS::Sp**

integrated cssS::Sp fragment  

![Diagram B](attachment://diagram_b.png)

C  
**B. subtilis htrA::pMutin2**

integrated pMutin2  

![Diagram C](attachment://diagram_c.png)

Fig. 1. Construction of *cssS* and *htrA* mutant strains of *B. subtilis*.  
A. Schematic presentation of the *cssR–cssS* region of the chromosome of strains containing a *cssS::pMutin2* knock-out mutation. By a single cross-over event (Campbell-type integration), the *cssS* gene was disrupted with pMutin2. Simultaneously, the *spoVG-lacZ* reporter gene of pMutin2 was placed under the transcriptional control of the *cssRS* promoter region ($P_{cssRS}$). The chromosomal fragment from the *cssRS* region, which was amplified by PCR and cloned into pMutin2, is indicated by black bars. Only the restriction sites relevant for the construction are shown (H, *HindIII*; B, *BamHI*). *lacI*, *E. coli lacI* gene; ori pBR322, replication functions of pBR322; *Ap'*, ampicillin resistance marker; *Em'*, erythromycin resistance marker; $T_1T_2$, transcriptional terminators on pMutin2; $P_{spac}$, IPTG-dependent promoter; *cssS'*, 3′ fragment of the disrupted *cssS* gene; *'cssS*, 5′ fragment of the disrupted *cssS* gene.  
B. Schematic presentation of the *cssR–cssS* region of strains containing the *cssS::Sp* mutation. By a double cross-over event, the *cssS* gene was disrupted with a *Sp'* marker. The part of the *cssS* gene that was amplified by PCR is indicated by black bars. Only the *StuI* (S) restriction sites, used for the construction, are shown.  
C. Schematic presentation of the *htrA* region of the chromosome of strains containing *htrA::pMutin2*. By a single cross-over event (Campbell-type integration), the *htrA* gene was disrupted with pMutin2. Simultaneously, the *spoVG-lacZ* reporter gene of pMutin2 was placed under the transcriptional control of the *htrA* promoter region ($P_{htrA}$). The chromosomal fragment from the *htrA* gene, which was amplified by PCR and cloned into pMutin2, is indicated by black bars. Only the restriction sites relevant for the construction are shown (H, *HindIII*; B, *BamHI*). *htrA'*, 3′ fragment of the disrupted *htrA* gene; *'htrA*, 5′ fragment of the disrupted *htrA* gene.

© 2001 Blackwell Science Ltd, *Molecular Microbiology*, 41, 1159–1172

from the membrane–cell wall interface by proteases with a
‘cleaning function’ (Tjalsma et al., 2000). The transcription
of one of these genes, ykdA (also known as htrA), was
recently shown to be heat shock inducible (Noone et al.,
2000), suggesting that B. subtilis contains signal trans-
duction systems that, in response to extracytoplasmic
protein misfolding, upregulate the expression of at least
one cell envelope-associated protease. Nevertheless, the
identification of such regulatory systems has not been
documented thus far. The aim of the present studies
was to identify regulators of extracytoplasmic proteolysis
in B. subtilis. In a search for Cpx-like two-component
regulatory systems in this organism, a novel two-
component system (designated CssR/S) was identified,

which is critical for the survival of severe secretion stress.
Our results show that the CssR/S system of B. subtilis
determines the level of proteolytic activity in the cell
envelope. Specifically, the transcription of the htrA gene,
encoding a putative ‘cleaning protease’, was shown to be
upregulated by secretion stress in a CssS-dependent
manner.

Results

Identification of the cssR–cssS genes

Based on the complete genome sequence of B. subtilis
(Kunst et al., 1997), a search for homologues of the

![Picture 1]



Fig. 2. Growth of cssS-prsA and cssR-prsA mutant strains during severe secretion stress. Cells containing pKTH3339 were grown in modified 2× L broth. To induce the expression of the P<sub>xy</sub><sub>n</sub>-amyQ gene of pKTH3339 at levels that saturate the secretion apparatus with AmyQ (A, B, D and E), 0.2% xylose was added to cultures at a cell density of about 80 Klett units (marked by arrows). To induce P<sub>xy</sub><sub>n</sub>-amyQ at non-saturating levels of AmyQ production (C), cultures were supplemented with 0.02% xylose (marked by an arrow). As a control, strains were grown in the absence of xylose (A, B, D and E). Furthermore, all cssR<sup>+</sup>-cssS<sup>+</sup> strains contained an insertion of pMutin2 in the yhcT gene, which is not required for growth and protein secretion (Bolhuis et al., 1999d).

A. B. subtilis cssS::pMutin2, pKTH3339 (IH7574; squares) and B. subtilis yhcT::pMutin2, pKTH3339 (IH7665; diamonds) grown in the presence (open symbols) or absence (closed symbols) of 0.2% xylose.

B. B. subtilis prsA3-cssS::pMutin2, pKTH3339 (IH7576; triangles) and B. subtilis prsA3-yhcT::pMutin2, pKTH3339 (IH7663; circles) grown in the presence (open symbols) or absence (closed symbols) of 0.2% xylose.

C. B. subtilis cssS::pMutin2, pKTH3339 (IH7574; squares), B. subtilis yhcT::pMutin2, pKTH3339 (IH7665; diamonds), B. subtilis prsA3-cssS::pMutin2, pKTH3339 (IH7576; triangles) and B. subtilis prsA3-yhcT::pMutin2, pKTH3339 (IH7663; circles) grown in the presence of 0.02% xylose.

D. B. subtilis cssR::pMutin4, pKTH3339 (IH8054; squares) and B. subtilis yhcT::pMutin2, pKTH3339 (IH7665; diamonds) grown in the presence (open symbols) or absence (closed symbols) of 0.2% xylose.

E. B. subtilis prsA3-cssR::pMutin4, pKTH3339 (IH8060; triangles) and B. subtilis prsA3-yhcT::pMutin2, pKTH3339 (IH7663; circles) grown in the presence (open symbols) or absence (closed symbols) of 0.2% xylose.

© 2001 Blackwell Science Ltd, Molecular Microbiology, 41, 1159–1172

two-component sensor histidine kinase CpxA of *E. coli* was performed. Significant similarities to about 20 of the 34 (potential) histidine kinases of *B. subtilis* were observed. Specifically, CpxA showed the highest levels of similarity to PhoR (35% identical residues in a stretch of 254 residues), YkoH (24% identical residues in a stretch of 347 residues), YvqB (23% identical residues in a stretch of 447 residues) and ResE (30% identical residues in a stretch of 237 residues). Notably, YvqB was the only histidine kinase showing sequence similarity to CpxA over the entire length of both proteins (46% identical residues and conservative replacements; data not shown). For this reason, the present studies were focused on YvqB.

The *yvqB* gene, located at 289.2° on the *B. subtilis* chromosome, specifies a protein of 451 residues (calculated molecular mass of 51 918). Like CpxA, the YvqB protein has two putative transmembrane segments. According to the predicted topology of these transmembrane segments, the amino- and carboxyl-termini of YvqB have a cytoplasmic localization (data not shown). The *yvqA* gene, located immediately upstream of *yvqB*, specifies a protein (225 residues; calculated molecular mass of 26 064) with significant sequence similarity to the two-component response regulator CpxR of *E. coli* (54% identical residues and conservative replacements; data not shown). In what follows, we show that *yvqB* is required to control the cellular response to secretion stress and, therefore, the *yvqA* and *yvqB* genes were renamed *cssR* (regulator) and *cssS* (sensor) respectively.

### CssS is required to combat secretion stress

A *cssS* knock-out mutant was constructed by inserting the integrative plasmid pMutin2 into the *cssS* gene (strain BV2006; Fig. 1A). This mutant was used to investigate the potential role(s) of the CssS protein in combating secretion stress and sensing misfolded proteins at the membrane–cell wall interface. For this purpose, the *cssS* mutant cells were subjected to secretion stress by the high-level expression of a secretory protein, the α-amylase AmyQ of *Bacillus amyloliquefaciens*. In addition, secretion stress was imposed on the *B. subtilis* cell by introduction of the *prsA3* mutation, which interferes with the post-translational folding of various secretory proteins, such as AmyQ (Kontinen et al., 1991; Jacobs et al., 1993; Kontinen and Sarvas, 1993; Leskelä et al., 1999a).

The *cssS::pMutin2* mutation had no effect on growth in liquid media nor the morphology of colonies on plates. In contrast, *cssS::pMutin2* cells transformed with plasmid pKTH10 (strain IH7552), which constitutively overproduce AmyQ, showed a slightly retarded growth in liquid media and a decreased colony size on plates compared with the parental strain and *B. subtilis* 168 (data not shown).

Notably, a high rate of reversion to variants that do not secrete AmyQ and have a wild-type colony morphology was observed. To be able to manipulate the level of secretion stress imposed on the cells, plasmid pKTH3339 was used, which carries the *amyQ* gene under the control of the xylose-inducible P<sub>xy</sub><sub>n</sub> promoter (P<sub>xy</sub><sub>n</sub>-amyQ). When P<sub>xy</sub><sub>n</sub>-amyQ was induced with 0.2% xylose, the secretion apparatus of *B. subtilis* 168 was saturated with AmyQ, as shown by the accumulation of significant amounts of preAmyQ in the cells. In contrast, induction with 0.02% xylose resulted in non-saturating AmyQ production levels (Vitikainen et al., 2001). When AmyQ production was induced to saturating levels (0.2% xylose) in pKTH3339-containing *cssS* mutant cells (strain IH7574), a level of growth retardation in liquid media was observed similar to that described above for the pKTH10-containing *cssS* mutant cells (Fig. 2A; open squares). In contrast, the growth of pKTH3339-containing *cssS* mutant cells was not affected when AmyQ synthesis was induced to non-saturating levels (0.02% xylose; Fig. 2C) or in the absence of AmyQ expression (Fig. 2A; closed squares).

A *prsA3-cssS::pMutin2* double mutant (IH7548) was constructed by transformation of a *prsA3* mutant strain (IH7171) with DNA from the *cssS::pMutin2* strain (BV2006). Notably, the growth of the double mutant in liquid medium and its colony morphology on plates were not affected compared with the corresponding single mutant strains if no secretion stress was applied (data not shown). To subject the *prsA3-cssS* double mutant to secretion stress, we first tried to transform this strain with pKTH10, but no AmyQ-secreting transformants were obtained. This finding suggests that high-level expression of *amyQ* in the double mutant is lethal, as neither the *prsA3* nor the *cssS::pMutin2* single mutation interfere with transformability (data not shown). This idea was verified with the pKTH3339-based xylose-inducible *amyQ* expression system. For this purpose, a *prsA3* mutant containing plasmid pKTH3339 (strain IH7558) was transformed with chromosomal DNA containing the *cssS::pMutin2* mutation. To avoid secretion stress and the eventual accumulation of suppressor mutations during the construction phase, *prsA3-cssS::pMutin2* transformants containing pKTH3339 (strain IH7576) were selected in the absence of xylose. As expected, in the absence of xylose, the *prsA3-cssS::pMutin2* strain with pKTH3339 showed similar growth characteristics to the *cssS*<sup>+</sup> control strain. In contrast, the growth of this strain was severely retarded on 0.2% xylose-containing plates on which the high-level expression of P<sub>xy</sub><sub>n</sub>-amyQ is induced (data not shown). The transfer of these growth-inhibited cells to fresh 0.2% xylose-containing plates resulted, at a high frequency, in wild-type-sized colonies that did not secrete AmyQ, suggesting that the constructed strain is genetically unstable under the imposed secretion

stress and that suppressor mutations accumulate. To study this growth inhibition in liquid medium, the prsA3-cssS::pMutin2 strain containing pKTH3339 was first grown in the absence of xylose until the mid-exponential growth phase. Next, 0.2% xylose was added to induce the P<sub>xy</sub><sub>n</sub>-amyQ gene. As shown in Fig. 2B (open triangles), shortly after the addition of xylose, the optical density of the culture decreased. This decrease was most probably caused by cell lysis because, 3.5 h after the induction of AmyQ production, the number of viable cells decreased about 100-fold (data not shown). After 4 h of induction, some cells restored their ability to grow in the presence of 0.2% xylose (Fig. 2B), but these were

genetically different from the original strain, as indicated by the formation of large colonies when transferred to xylose-containing plates. The growth retardation observed with the prsA3-cssS::pMutin2 double mutant strain containing pKTH3339 was clearly stronger than that observed with the prsA<sup>+</sup>-cssS::pMutin2 strain containing pKTH3339 (compare Fig. 2A and B), showing that the prsA3 mutation significantly increases secretion stress when combined with a cssS mutation. This idea was confirmed by the observation that the growth of the prsA3-cssS<sup>+</sup> strain containing pKTH3339 was only slightly inhibited upon the induction of amyQ expression with 0.2% xylose (Fig. 2B; open circles). In contrast, in the presence of 0.02% xylose, resulting in non-saturating levels of AmyQ production, the prsA3-cssS::pMutin2 strain containing pKTH3339 showed similar growth characteristics to the cssS<sup>+</sup>-prsA<sup>+</sup> control strain (Fig. 2C).

To investigate the importance of the response regulator CssR, a cssR::pMutin4 strain (BFA2461) was transformed with pKTH3339. In contrast to the cssS<sup>-</sup> prsA<sup>+</sup> mutant strain described above, the cssR::pMutin4 strain stopped growing and lysed upon the induction of AmyQ synthesis with 0.2% xylose (Fig. 2D). Surprisingly, the sensitivity of the cssR mutant cells to the overproduction of AmyQ was not exacerbated by the prsA3 mutation (Fig. 2E). Moreover, irrespective of the prsA3 mutation, the cssR mutant cells started to grow again after 2 h of induction, suggesting that suppressor mutations had accumulated rapidly. Taken together, these observations show that CssR and CssS are required to combat secretion stress.

Fig. 3. Increased levels of PrsA3 in the absence of CssS.
A. Cells of B. subtilis prsA3, pkTH3339 (IH7558; lanes 1–2), B. subtilis prsA3-cssS::pMutin2, pkTH3339 (IH7576; lanes 3–4) and B. subtilis prsA3 (overexpression) (IH7461; lane 5) were grown in modified 2× L broth in the presence (+) or absence (−) of 0.02% xylose. Whole-cell samples for SDS–PAGE and Western blotting were prepared from cells in the late exponential growth phase, and equal amounts of protein were loaded in each lane. PrsA3 was visualized with specific antibodies. The positions of full-length mature PrsA3 and its degradation product (d) are indicated.
B. A transcriptional prsA-lacZ gene fusion was used to determine the time course of prsA expression in cells grown at 37°C in modified 2× L broth. The strains used for the analyses are: B. subtilis prsA::pKTH3383 (pKTH3339) (IH7959; triangles); B. subtilis prsA::pKTH3383 cssS::Sp (pKTH3339) (IH7957; squares). β-Galactosidase activities are indicated in nmol mgprot<sup>−1</sup> min<sup>−1</sup>. Zero time (t = 0) indicates the transition point between the exponential and post-exponential growth phases.
C. The synthesis of the PrsA3 protein in B. subtilis cssS::pMutin2 (IH8056; lanes 1 and 2) and the corresponding cssS<sup>+</sup> control strain (IH8058; lanes 3 and 4) was analysed by pulse-chase labelling at 37°C and subsequent immunoprecipitation, SDS–PAGE and fluorography. Cells were labelled with [35S]-methionine for 45 s before chase with excess non-radioactive methionine. Samples were withdrawn after the chase at the times indicated. The positions of pre-PrsA3 (p) and mature PrsA3 (m) are indicated.

© 2001 Blackwell Science Ltd, Molecular Microbiology, 41, 1159–1172

1164 H.-L. Hyyryläinen et al.

Increased accumulation of active PrsA3 in the absence of CssS

Like wild-type PrsA, the PrsA3 mutant protein is a membrane-bound lipoprotein. However, as a result of the replacement of Asp-268 with Asn (Kontinen and Sarvas, 1993), PrsA3 is highly sensitive to degradation by as yet unidentified membrane- or cell wall-bound proteases. Thus, the PrsA level in *prsA3* mutant cells is more than 10-fold reduced, and a 2.5 kDa fragment is removed from the carboxyl-terminus of the PrsA3 protein (Kontinen and Sarvas, 1993; Hyyryläinen *et al.*, 2000). Notably, the protein secretion defect of *prsA3* mutant strains can be fully attributed to their decreased PrsA levels (data not shown). For these reasons, PrsA3 appeared to be a suitable reporter protein to investigate the possible role of CssS in the proteolysis of abnormal proteins at the membrane–cell wall interface.

As shown in Fig. 3A, the *cssS::pMutin2* mutation resulted in significantly increased levels of the PrsA3 protein. Moreover, this effect of the *cssS* mutation on PrsA3 was independent of the synthesis of AmyQ at non-saturating levels (0.02% xylose). As neither the transcription of *prsA* (shown with a transcriptional *lacZ* fusion; Fig. 3B) nor the synthesis of the PrsA3 protein (shown by pulse-chase labelling; Fig. 3C) were affected by the disruption of *cssS*, the CssS protein is most probably involved in the proteolysis of abnormal or misfolded proteins, such as PrsA3, at the membrane–cell wall interface. Nevertheless, the *cssS* mutation did not prevent the carboxyl-terminal cleavage of PrsA3, the ratio between the mature PrsA3 and its degradation product (d) remaining constant (Fig. 3A). This suggests that synthesis of the protease(s) responsible for carboxyl-terminal PrsA3 cleavage is independent of the Css system.

To investigate whether the PrsA3 that accumulates in the absence of CssS is active, the secretion of AmyQ by *cssS* mutant strains was analysed. α-Amylase activities in the medium of *B. subtilis cssS::pMutin2* with pKTH3339 (strain IH7574) and *B. subtilis prsA3-cssS::pMutin2* with pKTH3339 (strain IH7576) were determined as a function of growth. To avoid growth inhibition, P<sub>xy</sub><sub>n</sub>-amyQ was expressed at non-saturating levels (0.02% xylose). Compared with the *prsA3-cssS<sup>+</sup>* control strain (IH7558), the *prsA3-cssS::pMutin2* double mutant secreted about 2.5-fold more AmyQ into the medium (Fig. 4A). Notably, the *cssS* mutation enhanced the secretion of AmyQ to a much lower extent (about 30%) in the *prsA<sup>+</sup>* background (Fig. 4B). Taken together, these observations suggest that a significant fraction of the PrsA3, accumulating in the absence of CssS, is involved in the post-translational folding of AmyQ at the membrane–cell wall interface.

Accumulation of misfolded AmyQ in the envelope of cells lacking CssS

As the *cssS* mutation resulted in the enhanced secretion of AmyQ, particularly in the *prsA3* mutant background, the effects of these mutations on the accumulation of cell-associated forms of AmyQ were also investigated. To this end, P<sub>xy</sub><sub>n</sub>-amyQ was expressed at non-saturating levels (0.02% xylose) in pKTH3339-containing cells. As shown in Fig. 5, cell-associated mature AmyQ was observed in the *prsA3-cssS::pMutin2* double mutant strain (IH7576), but not in the *prsA3* or *cssS* single mutant control strains. Upon lysozyme treatment, about 70% of the cell-associated mature AmyQ in the *prsA3-cssS::pMutin2* double mutant remained protoplast associated. Moreover, a significant fraction of the mature AmyQ that was released into the protoplast supernatant (about 30%) was probably released as a result of lysis (data not shown). Thus, most of the mature AmyQ that accumulates in the *prsA3-cssS::pMutin2* double mutant appears to be associated with the cell membrane.


![Figure 4](https://i.imgur.com/yourimageurl.png)

Fig. 4. Enhanced secretion of AmyQ in the absence of *cssS*. Cells were grown in modified 2× L broth supplemented with 0.02% xylose (P<sub>xy</sub><sub>n</sub>-amyQ of pKTH3339 induced). Optical density of the culture (closed symbols) and the accumulation of α-amylase to the culture medium (open symbols) were measured at various intervals.
A. *B. subtilis prsA3-cssS::pMutin2*, pKTH3339 (IH7576; triangles),
B. *B. subtilis prsA3*, pKTH3339 (IH7558; squares).
B. *B. subtilis cssS::pMutin2*, pKTH3339 (IH7574; triangles)
and *B. subtilis*, pKTH3339 (IH7554; squares).

© 2001 Blackwell Science Ltd, Molecular Microbiology, 41, 1159–1172

is predicted to specify a membrane-bound protease in *B. subtilis* (Tjalsma et al., 2000), was investigated. For this purpose, a chromosomally located transcriptional *htrA–lacZ* gene fusion was constructed, using plasmid pMutin2 (Fig. 1C). Notably, the introduction of the *htrA–lacZ* gene fusion resulted in the inactivation of the *htrA* gene, which is negatively autoregulated (Noone et al., 2000). Furthermore, in order to combine the *htrA–lacZ* fusion with a *cssS* mutation, the *cssS* gene was disrupted with a spectinomycin resistance cassette (*Sp<sup>r</sup>*; Fig. 1B). Next, the transcription of *htrA–lacZ* was studied by measuring β-galactosidase activities as a function of growth. As shown in Fig. 7A (open squares), the *htrA–lacZ* gene fusion in cells of the *htrA::pMutin2* strain (BV2003) was transcribed at moderate levels during early exponential growth in TY medium. When these cells entered the mid-exponential growth phase, *htrA–lacZ* transcription was increased, showing a maximum about 1 h before the transition point (*t = 0*) between exponential and post-exponential growth. Strikingly, the disruption of *cssS* resulted in a severe reduction in *htrA–lacZ* transcription, the β-galactosidase levels in the *cssS::Sp-htrA::pMutin2* strain (BV2002; open circles in Fig. 7A) being close to the background levels in the parental strain 168. Essentially the same results were obtained when cells were grown in minimal medium (Fig. 7B), even though the *htrA–lacZ* expression profiles were slightly different from those observed in TY medium.

To investigate whether the AmyQ that accumulated in the cell envelope of the *prsA3-cssS::pMutin2* strain containing pKTH3339 (IH7576) was properly folded, the activity of the AmyQ released upon protoplasting was compared with that of AmyQ secreted by a *prsA<sup>+</sup>-cssS<sup>+</sup>* control strain (IH7665). In parallel, the activities of AmyQ derived from the cell envelopes of the control strain and the *cssS* single mutant (IH7574), both containing pKTH3339, were determined. To induce the accumulation of AmyQ in cell envelopes of the latter strains, the *P<sub>xyn</sub>-amyQ* gene was maximally induced with 0.2% xylose. Strikingly, the cell envelope-associated AmyQ derived from the *prsA3-cssS* double mutant displayed a fivefold reduced activity compared with the AmyQ secreted by the control strain (Fig. 6). Similarly, the activity of AmyQ released from the cell envelope of the *cssS* single mutant was about fivefold reduced (Fig. 6B). In contrast, the activity of AmyQ derived from the cell envelope of the *prsA<sup>+</sup>-cssS<sup>+</sup>* control strain was merely reduced by about 30% compared with the activity of the AmyQ secreted by this strain (Fig. 6B). In summary, these observations show that misfolded AmyQ accumulates in the cells lacking CssS. This implies that the Css system is required for the removal of misfolded protein molecules from the cell envelope of *B. subtilis*.

### CssS-dependent transcription of *htrA*

As the absence of CssS appeared to result in reduced levels of proteolysis at the membrane–cell wall interface, the role of CssS in the expression of the *htrA* gene, which

© 2001 Blackwell Science Ltd, *Molecular Microbiology*, **41**, 1159–1172

1166 H.-L. Hyyryläinen et al.

A

| Sample volume (µl) | 120 | 240 | 480 | 1,5 | 3 | 6 |
|--------------------|-----|-----|-----|-----|---|---|
| Source of AmyQ     | ce  | ce  | ce  | m   | m | m |

AmyQ activity (U)

| Amount of AmyQ (IOD) |
|-----------------------|
| 0                     |
| 0,2                   |
| 0,4                   |
| 0,6                   |
| 0,8                   |

B

| Sample volume (µl) | 160 | 320 | 80 | 160 | 240 | 480 | 3 | 6 |
|--------------------|-----|-----|----|-----|-----|-----|---|---|
| Source of AmyQ     | ce  | ce  | ce | ce  | ce  | ce  | m | m |
| Xylose conc. (%)   | 0.2 | 0.2 | 0.2| 0.2 | 0.02| 0.02| 0.02| 0.02|

U / IOD

Fig. 6. Analysis of AmyQ activity in cssS mutants.

A. The B. subtilis prsA3-cssS::pMutin2, pKTH3339 double mutant (IH7576; triangles) and the yhcT::pMutin2, pKTH3339 control strain (IH7665; squares) were grown in the presence of 0.02% xylose to induce P<sub>xy</sub><sub>n</sub>-amyQ. Different amounts of cell envelope-associated AmyQ from prsA3-cssS::pMutin2, pKTH3339 (protoplast supernatant fractions; see Experimental procedures) and the secreted AmyQ from yhcT::pMutin2, pKTH3339 (growth medium fractions) were subjected to SDS–PAGE and Western blotting (top). The indicated amounts relate to the samples withdrawn from the corresponding cultures. AmyQ bands were visualized with specific antibodies, and integrated optical densities (IOD) of the bands were determined by scanning. Equivalent relative amounts of AmyQ were used to measure α-amylase activities. At the bottom, the amounts of AmyQ estimated by optical scanning (IOD) are plotted against α-amylase activities, which are presented as arbitrary units (U): the absorbance in the wavelength range 616–624 nm. ce, cell envelope; m, medium.

B. Immunoblotting (top) and α-amylase activity assays (bottom) were carried out with the cell envelope-associated AmyQ from the B. subtilis cssS::pMutin2, pKTH3339 (IH7574), B. subtilis prsA3-cssS::pMutin2, pKTH3339 (IH7576) and the control strain B. subtilis yhcT::pMutin2, pKTH3339 (IH7665) as described in (A). Furthermore, AmyQ from the medium of the control strain was included in the analyses. Upon scanning of the AmyQ bands and the determination of α-amylase activities, the U/IOD ratios were calculated and plotted in a histogram (bottom). Bars corresponding to different samples from the same culture are labelled black and grey respectively. ce, cell envelope; m, medium.

increased by the prsA3 mutation, and it was increased even further in the prsA3 mutant strain secreting AmyQ. These findings show that the transcription of the htrA gene is CssS dependent, the level of transcription being increased in response to increased protein secretion stress.

Discussion

The present studies show for the first time that the cell envelope of a Gram-positive eubacterium contains a typical sensor protein, CssS, that monitors the (over)-production of at least one secretory protein and controls the synthesis of extracytoplasmic ‘cleaning proteases’. In fact, CssS of B. subtilis and the cognate response regulator CssR are required to combat the detrimental effects of severe secretion stress, as shown by the observation that the high-level overproduction of the α-amylase AmyQ caused significant growth inhibition of cssS mutant cells and lysis of cssR mutant cells. Furthermore, cells lacking CssS and containing the mutant

© 2001 Blackwell Science Ltd, Molecular Microbiology, 41, 1159–1172

Thirdly, cssS mutant cells accumulated misfolded mature AmyQ in the cell envelope. Taken together, these observations suggest that CssS can detect misfolded proteins at the membrane–cell wall interface. Thus, it seems that CssS of B. subtilis is, at least in part, functionally equivalent to CpxA of E. coli, which is involved in the control of periplasmic proteolysis. Consistent with this idea is the fact that both proteins show significant sequence similarity.

It is presently not clear why the reduced levels of proteolysis at the membrane–cell wall interface and the concomitant accumulation of misfolded proteins in the cell wall are highly detrimental for cssS mutant cells. However, it is conceivable that misfolded proteins interfere with essential processes, such as protein translocation via the Sec machinery or the folding and activity of essential cell wall proteins. In fact, this could also explain why the foldase PrsA is essential for viability (Kontinen and Sarvas, 1993; Vitikainen et al., 2001). In principle, the cell can prevent the detrimental accumulation of misfolded proteins by increasing the levels of proteases, foldases or both. The present data show that CssS is involved in the regulation of proteolysis. Thus far, it is not known whether, or to what extent, CssS can influence the folding of exported proteins. Notably, the transcription of the prsA gene is not affected by a cssS mutation, demonstrating that a major post-translocational protein folding activity is not modulated by the Css system. Even so, the observation that htrA transcription is controlled by CssS could imply a role for this sensor in the regulation of protein folding, as chaperone-like activities have been demonstrated for HtrA of E. coli (Spiess et al., 1999).

Despite the accumulation of misfolded AmyQ in the cell envelope of the prsA3–cssS double mutant, the secretion of enzymatically active α-amylase by this strain was increased compared with the prsA3 single mutant. This effect of the cssS mutation on AmyQ secretion can be attributed, at least in part, to the increased levels of the PrsA3 protein in the double mutant. The reason for this is that the Asp-268Asn replacement in PrsA3 affects the protease resistance of this protein, but not its foldase activity (data not shown). In contrast, the increase in AmyQ secretion by about 30% in a prsA⁺–cssS⁻ strain is probably caused by reduced degradation of this secretory protein before its post-translocational folding. This would be in line with recent studies, showing that various Bacillus α-amylases are subject to proteolysis shortly after their translocation across the membrane (Stephenson et al., 1998; Bolhuis et al., 1999b). One of the proteases that is responsible for protein degradation at the membrane–cell wall interface is the WprA-derived cell wall-bound protein CWBP52, which exhibits serine protease activity (Margot and Karamata, 1996). Notably, in addition to CWBP52, other as yet unidentified proteases appear to be active at

Fig. 7. Analysis of htrA expression with a transcriptional lacZ gene fusion. The transcriptional htrA–lacZ gene fusion shown schematically in Fig. 1C was used to determine the time courses of htrA expression in cells grown at 37°C in TY medium (A) or minimal medium (B). The strains used for the analyses are: B. subtilis htrA::pMutin2 (BV2003; open squares); B. subtilis htrA::pMutin2, pKTH10L (BV2013; closed squares); B. subtilis cssS::Sp-htrA::pMutin2 (BV2002; open circles) and B. subtilis cssS::Sp-htrA::pMutin2, pKTH10L (BV2014; closed circles). The parental strain 168 was used as a negative control (closed triangles). β-Galactosidase activities are indicated in nmol min⁻¹ OD600⁻¹. Zero time (t = 0) indicates the transition point between the exponential and post-exponential growth phases.

foldase PrsA3 stopped growing and started to lyse upon AmyQ overproduction. These observations indicate that a mutation in cssR makes the cells more sensitive to secretion stress than a mutation in cssS but, notably, the present cssR mutation may have a polar effect on the expression of the cssS gene. The conclusion that CssS is involved in the control of proteolysis at the membrane–cell wall interface is supported by three observations. Firstly, CssS was shown to be required for transcription of the htrA gene that specifies a, most probably, membrane-associated protease of B. subtilis. Secondly, the absence of CssS resulted in the increased accumulation of an unstable mutant of PrsA, known as PrsA3.

1168 H.-L. Hyyryläinen et al.

Table 1. Analysis of *htrA* expression with a transcriptional *lacZ* gene fusion.

| Phase of growth | IH7597 (pUB110) | IH7582 (pKTH10)<sup>b</sup> | IH7590 (*prsA3*, pUB110) | IH7520 (*prsA3*, pKTH10)<sup>b</sup> |
|-----------------|------------------|-----------------------------|---------------------------|------------------------------------|
| Klett 50        | 148              | 572                         | 331                       | 699                                |
| Klett 100       | 141              | 677                         | 429                       | 805                                |
| Klett 100+1h    | 392              | 1365                        | 1015                      | 1767                               |
| Klett 100+3h    | 528              | 1583                        | 1139                      | 2671                               |
| Klett 100+5h    | 631              | 1595                        | 936                       | 2532                               |

a. *B. subtilis* strains with a chromosomal *htrA-lacZ* fusion (*htrA::pMutin2*) were grown in modified 2× L broth, and β-galactosidase activities were determined in samples withdrawn during different phases of growth.
b. The plasmid pKTH10 has been constructed by cloning the *amyQ* gene of *B. amyloliquefaciens* in plasmid pUB110.

this subcellular location, as shown by the fact that the degradation of the α-amylase AmyL and an unstable mutant form of the signal peptidase SipS was not completely abolished in a *wprA* mutant (Stephenson and Harwood, 1998; Bolhuis *et al.*, 1999c). Interestingly, a *wprA* mutation does not reduce the degradation of PrsA3 (data not shown). Nevertheless, this does not exclude the possibility that the expression of the *wprA* gene is regulated by the Css system, because a disruption of the *htrA* gene, which is controlled by CssS, did not result in increased levels of the unstable PrsA3 protein. The latter observation, in particular, indicates that HtrA is not the only protease at the membrane–cell wall interface whose the synthesis is controlled by the Css system. Conversely, it seems that CssR/S does not regulate the synthesis of all proteases at this subcellular location. This view emerges from the observation that carboxyl-terminal cleavage of PrsA3 was not affected in the cssS mutant.

Even though the Css system bears some resemblance to the Cpx system of *E. coli*, at least one major difference between these two systems exists: the known CpxR binding site appears not to be conserved in *B. subtilis*, and it is clearly absent from the upstream region of the *htrA* gene. It is the identification of the CssR binding site that is needed to obtain a full definition of the CssR/S regulon of *B. subtilis*. This could result in the discovery of novel components or systems that are used by the cell to combat secretion stress.

### DNA techniques

Procedures for DNA purification, restriction, ligation, agarose gel electrophoresis and transformation of competent *E. coli* cells were carried out as described by Sambrook *et al.* (1989). *B. subtilis* was transformed as described by Kunst and Rapoport (1995). Polymerase chain reaction (PCR) was carried out as described by van Dijl *et al.* (1995). The nucleotide sequences of primers used for PCR (5′–3′) are listed below; restriction sites used for cloning are underlined. Enzymes were from Roche Molecular Biochemicals. Protein sequence alignments were carried out with the CLUSTALW program (Thompson *et al.*, 1994) using the Blosum matrices.

To construct plasmid pKTH3339 for xylose-inducible *amyQ* expression, a fragment containing the ribosome binding site, coding region and transcription termination site of the *amyQ* gene was amplified by PCR with primers 6104 (CAAATCGAT AAAATGAGAGGGAGAGGAAAC) and 6095 (CAAGTCGA CGTATTGTGCACACCCTGATG). This fragment was cloned between the *ClaI* and *SalI* sites of plasmid pSX50 (Hastrup and Jacobs, 1990), placing *amyQ* under the transcriptional control of the xylose-inducible *P<sub>xyn</sub>* promoter. To construct the *cssS::pMutin2* insertion mutation, an internal fragment (330 nucleotides) of the *cssS* (*yvqB*) gene was amplified with the primers AB50.cssS (GATGCGCCAAAAGCTTG) and AB51.cssS (AAGGATCCTTCCCACGATAGTTCTTTC) and cloned into plasmid pMutin2. The *cssS::pMutin2* mutation was obtained by Campbell-type (single cross-over) integration of the resulting plasmid into the chromosome of *B. subtilis* 168. To obtain *prsA3-cssS::pMutin2* double mutants, *prsA3* mutant strains were transformed with chromosomal DNA from a *cssS::pMutin2* strain. The *cssS::Sp* mutation was constructed by insertion of a Sp<sup>r</sup> marker, derived from plasmid pDG1727, into the chromosomal *cssS* gene via double cross-over recombination. To this end, an internal fragment of the *cssS* gene was amplified by PCR with primers cssS1 (CCG GAATTCTATTACAGCGAGGAAGC) and cssS2 (CCGGAA TTCAGGCTCATAGAGGCTG) and ligated into the unique *EcoRI* site of plasmid pSK1. Next, the *cssS* fragment in the resulting plasmid, denoted pSK1cssS, was disrupted by ligating the Sp<sup>r</sup> marker into the unique *StuI* site. This resulted in plasmid pDG172721. Upon transformation of *B. subtilis* 168 with pDG172721, the *cssS* gene was disrupted with the Sp<sup>r</sup> marker via a double cross-over recombination event. *B. subtilis cssR::pMutin4* (strain BFA2461) was constructed within the *Bacillus subtilis* systematic gene function analysis

### Experimental procedures

#### Plasmids, bacterial strains and media

Tables 2 and 3 list the bacterial strains and plasmids used. Bacteria were grown either in modified 2× L broth (2% tryptone, 1% yeast extract and 1% NaCl) or in TY medium (1% tryptone, 0.5% yeast extract and 1% NaCl). Minimal medium (GCHE medium) was prepared as described by Kunst and Rapoport (1995). Antibiotics were used at the following concentrations: ampicillin (Ap), 50 μg ml<sup>−1</sup>; erythromycin (Em), 1 μg ml<sup>−1</sup>; kanamycin (Km), 10 μg ml<sup>−1</sup>; spectinomycin (Sp), 100 μg ml<sup>−1</sup>. IPTG was used at 1 mM.

Table 2. Bacterial strains.

| Strain           | Relevant genotype                                                                 | Reference                          |
|------------------|-----------------------------------------------------------------------------------|------------------------------------|
| B. subtilis      |                                                                                   |                                    |
| 168              | trpC2; parental strain used in the present study                                  | Kunst et al. (1997)               |
| BFA1620          | yhcT::pMutin2                                                                    | Bolhuis et al. (1999d)            |
| BFA2461          | cssR(yvqA)::pMutin4                                                              | This study                        |
| BV2001           | cssS(yvqB)::Sp                                                                   | This study                        |
| BV2002           | cssS::Sp htrA(ykdA)::pMutin2                                                     | This study                        |
| BV2003           | htrA::pMutin2                                                                    | This study                        |
| BV2006           | cssS::pMutin2                                                                    | This study                        |
| BV2013           | htrA::pMutin2 (pKTH10L)                                                          | This study                        |
| BV2014           | cssS::Sp htrA::pMutin2 (pKTH10L)                                                 | This study                        |
| IH6538           | glyB133 hisA1 trpC2                                                               | Kontinen et al. (1991)            |
| IH7171           | prsA3 glyB133 hisA1                                                              | This study                        |
| IH7461           | prsA3 glyB133 hisA1 (pKTH3461)                                                   | Hyryläinen et al. (2000)          |
| IH7520           | prsA3 glyB133 hisA1 htrA::pMutin2 (pKTH10)                                        | This study                        |
| IH7535           | yhcT::pMutin2 (pKTH10)                                                           | This study                        |
| IH7537           | prsA3 glyB133 hisA1 yhcT::pMutin2 (pKTH10)                                        | This study                        |
| IH7548           | prsA3 glyB133 hisA1 cssS::pMutin2                                                | This study                        |
| IH7552           | cssS::pMutin2 (pKTH10)                                                           | This study                        |
| IH7554           | glyB133 hisA1 (pKTH3339)                                                         | This study                        |
| IH7558           | prsA3 glyB133 hisA1 (pKTH3339)                                                   | This study                        |
| IH7574           | cssS::pMutin2 (pKTH3339)                                                         | This study                        |
| IH7576           | prsA3 glyB133 hisA1 cssS::pMutin2 (pKTH3339)                                      | This study                        |
| IH7582           | htrA::pMutin2 (pKTH10)                                                           | This study                        |
| IH7590           | prsA3 glyB133 hisA1 htrA::pMutin2 (pUB110)                                        | This study                        |
| IH7597           | htrA::pMutin2 (pUB110)                                                           | This study                        |
| IH7659           | prsA3 glyB133 hisA1 yhcT::pMutin2                                                | This study                        |
| IH7661           | yhcT::pMutin2                                                                    | This study                        |
| IH7663           | prsA3 glyB133 hisA1 yhcT::pMutin2 (pKTH3339)                                      | This study                        |
| IH7665           | yhcT::pMutin2 (pKTH3339)                                                         | This study                        |
| IH7957           | prsA::pKTH3383; PprsA-lacZ cssS::Sp (pKTH3339)                                     | This study                        |
| IH7959           | prsA::pKTH3383; PprsA-lacZ (pKTH3339)                                             | This study                        |
| IH8054           | cssR::pMutin4 (pKTH3339)                                                         | This study                        |
| IH8056           | prsA3 glyB133 hisA1 cssS::pMutin2 (pKTH3461)                                      | This study                        |
| IH8058           | prsA3 glyB133 hisA1 yhcT::pMutin2 (pKTH3461)                                      | This study                        |
| IH8060           | prsA3 glyB133 hisA1 cssR::pMutin4 (pKTH3339)                                       | This study                        |
| E. coli          |                                                                                   |                                    |
| MC1061           | F⁻, araD139, Δ(ara-leu) 7696, Δ(lac)X74, galU, galK, hsdR2, mcrA, mcrB1, rspL     | Wertman et al. (1986)             |

Table 3. Plasmids.

| Strain    | Characteristics                                                                                     | Source/reference                              |
|-----------|-----------------------------------------------------------------------------------------------------|-----------------------------------------------|
| Plasmids  |                                                                                                     |                                               |
| pKTH10    | PamyQ-amyQ; Kmʳ                                                                                    | Palva (1982)                                  |
| pKTH10L   | pKTH10 derivative; Kmʳ                                                                              | This study                                   |
| pKTH3339  | Pxyr-amyQ; Cmʳ                                                                                     | Vitikainen et al. (2001)                     |
| pKTH3383  | Integrative vector containing a transcriptional prsA-lacZ fusion; Apʳ, Emʳ                           | This study                                   |
| pKTH3461  | Pspac-prsA3-HIS6; Kmʳ                                                                               | Hyryläinen et al. (2000)                      |
| pMutin1   | Integrative vector; Apʳ, Emʳ                                                                        | Vagner et al. (1998)                         |
| pMutin2   | Integrative vector; Apʳ, Emʳ                                                                        | Vagner et al. (1998)                         |
| pMutin4   | Integrative vector; Apʳ, Emʳ                                                                        | Vagner et al. (1998)                         |
| pSK1      | pACYC 184 derivative; Cmʳ, Kmʳ                                                                      | M. Noack (Department of Genetics, University |
|           |                                                                                                     | of Groningen)                                |
| pSK1cssS  | pSK1 derivative containing an internal fragment of the cssS gene; Cmʳ, Kmʳ                            | This study                                   |
| pDG1727   | pMTL23 derivative; Apˡ, Spˡ                                                                         | Guérout-Fleury et al. (1995)                 |
| pDG172721 | pSKcssS derivative; Cmʳ, Kmʳ, Spʳ                                                                   | This study                                   |

1170 H.-L. Hyyryläinen et al.

(BSFA) program. To construct this mutant, an internal fragment (221 nucleotides) of the *cssR* (*yvqA*) gene was amplified with the primers YVQAK-FOR (CGCGGATCCAT TCTCGATATCATGCTG) and YVQAK-REV (CGCGGATCC TCCTTATATACGAGCTGC) and cloned into plasmid pMutin4. The *cssR::pMutin4* mutation was obtained by Campbell-type (single cross-over) integration of the resulting plasmid into the chromosome of *B. subtilis* 168. To obtain a *prsA3-cssR::pMutin4* double mutant, the *prsA3* mutant strain was transformed with chromosomal DNA from the *cssR::pMutin4* strain. To construct a transcriptional *prsA–lacZ* fusion, a 3′ fragment of the *prsA* gene (376 nucleotides) was amplified by PCR with the primers 61151 (AACAAAGCTTAAAGAATAC TCAACAGAGAC) and 61152 (CAAGGATCCTTTTAGAATTG CTTGAAG) and cloned into pMutin1, resulting in pKTH3383. The chromosomal *prsA–lacZ* fusion was obtained by Campbell-type integration of pKTH3383 into the *prsA* region of *B. subtilis* IH6538, placing *lacZ* under the control of the *P<sub>prsA</sub>* promoter. This *prsA–lacZ* strain was subsequently transformed with pKTH3339, resulting in *B. subtilis* IH7959. To construct a *cssS* mutant strain containing *prsA–lacZ* (IH7957), strain IH7554 was first transformed with chromosomal DNA from BV2001 and then with the plasmid pKTH3383. To construct the *htrA::pMutin2* mutation and the transcriptional *htrA–lacZ* fusion, an internal fragment (430 nucleotides) of the *htrA* (*ykdA*) gene was amplified by PCR with the oligonucleotides AB58.htrA (AAAAGCTTGGCTCAA GCAGCTCATCTG) and AB59.htrA (CAAGCGGATCCCC AATC) and cloned into plasmid pMutin2. The *htrA::pMutin2* construct was obtained by Campbell-type integration of the resulting plasmid into the chromosome of *B. subtilis* 168. To construct the *cssS::Sp-htrA::pMutin2* double mutant, the *cssS::Sp* mutant was transformed with chromosomal DNA from the *htrA::pMutin2* mutant. The correct integration of plasmids in the chromosome of *B. subtilis* was verified by Southern hybridization.

### Immunoblotting and isolation of the cell envelope-associated α-amylase

Immunoblotting was performed as described previously (Leskelä et al., 1999b). The visualization of protein bands was carried out with specific antisera, horseradish peroxidase-conjugated goat anti-rabbit IgG and the ECL immunoblotting detection reagents (Amersham). Densities of protein bands were determined by optical scanning with a Bio Image system (Milligen/Biosearch). To isolate cell envelope-associated forms of the α-amylase AmyQ, the expression of *P<sub>xyn</sub>-amyQ* was induced by adding 0.02% or 0.2% xylose to cells growing exponentially (Klett 80) in 40 ml of modified 2× L-broth. After 1 h of induction, cells were collected by centrifugation (10 000 g for 5 min), resuspended in protoplast buffer (20 mM potassium phosphate, pH 7.5, 15 mM MgCl<sub>2</sub>, 20% sucrose and 1 mg ml<sup>−1</sup> lysozyme) and incubated at 37°C for 30 min. The resulting protoplasts were pelleted by centrifugation (2200 g for 8 min), and the protoplast supernatant was used to determine the α-amylase activity of AmyQ released from the cell envelope.

### Pulse-chase labelling

Pulse-chase labelling experiments were performed as described previously (Leskelä et al., 1999b). Cells were labelled with \[^{35}S\]-methionine for 45 s and chased with non-radioactive methionine for 0 or 1 min.

### Assays

Growth was monitored by optical density (OD) readings with a Klett-Summerson colorimeter using a no. 66 filter. α-Amylase activities were determined as described previously (Leskelä et al., 1999b). Briefly, one quarter of a Phadebas (Pharmacia and Upjohn) tablet was dispersed in 1 ml of sample appropriately diluted in assay buffer. After incubation at 37°C for 1 h, the reaction mixture was filtered through Whatman no. 1 filter paper, and the absorbance of the filtrate was measured in the range of 616–624 nm, the absorbance in the 800–804 nm range being used as a control. The enzyme concentration (mg l<sup>−1</sup>) was calculated from the activity of purified *B. amyloliquefaciens* α-amylase (Sigma) or, alternatively, the activity of α-amylase was expressed in absorbance units. To assay β-galactosidase activities, overnight cultures were diluted 100-fold in fresh medium, and samples were taken at hourly intervals for OD readings at 600 nm and β-galactosidase activity determinations. The β-galactosidase assay and the calculation of β-galactosidase units (Miller units: nmol OD600<sup>−1</sup> min<sup>−1</sup>) was performed as described by Miller (1982). Alternatively, a modification of this method was used as described in http://locus.jouy.inra.fr/genmic/madbase/mutant/galang.html. In the latter case, cells were collected from 1 ml samples by centrifugation (12 000 g, 2 min) and resuspended in 0.5 ml of Z buffer (60 mM Na<sub>2</sub>HPO<sub>4</sub> × 2H<sub>2</sub>O, 40 mM NaH<sub>2</sub>PO<sub>4</sub> × H<sub>2</sub>O, 10 mM KCl and 1 mM MgSO<sub>4</sub> × 7H<sub>2</sub>O) supplemented with 1 mM dithiothreitol (DTT), DNase (0.01 mg ml<sup>−1</sup>) and lysozyme (0.1 mg ml<sup>−1</sup>). The cell suspension was incubated at 37°C for 20 min, followed by centrifugation at 12 000 g for 5 min. The supernatants were transferred to clean tubes and kept on ice until assayed. The β-galactosidase activity was assayed by mixing 200 μl of supernatant with 600 μl of Z buffer containing 1 mM DTT and 200 μl of o-nitrophenyl-β-D-galactoside (ONPG; 4 mg dissolved in 0.1 M sodium phosphate, pH 7.5), followed by incubation of the mixture at 28°C. The reaction was terminated once the mixture turned yellow by adding 0.5 ml of 1 M Na<sub>2</sub>CO<sub>3</sub>, and the incubation time was noted. Protein concentrations of the supernatants were determined with the Bio-Rad Protein Assay and Bio-Rad Protein Assay Standard I (Bio-Rad). β-Galactosidase activities were expressed in nmol of ONP produced mgprot<sup>−1</sup> min<sup>−1</sup>.

### Acknowledgements

We thank Dr M. Noback for contributing plasmid pSK1, and Drs C. Eschevins, O. Kuipers, H. Tjalsma and other members of the Groningen/European *Bacillus* Secretion Groups for stimulating discussions. H.-L.H., A.B., L.M., P.K., M.V., M.S., S.B., J.M.v.D. and V.P.K. were supported in part by European Union (EU) Biotechnology Grants Bio2-CT93-0254, Bio2-CT93-0272, Bio4-CT96-0097 and Bio4-CT95-0278, and EU ‘Quality of Life and Management of Living Resources’ Grants QLK3-CT-1999-00413 and QLK3-CT-1999-00917.

E.D. was supported by the Ubbo Emmius Foundation of the University of Groningen.

**References**

Archibald, A.R., Hancock, I.C., and Harwood, C.R. (1993) Cell wall structure, synthesis and turnover. In *Bacillus subtilis and Other Gram-Positive Bacteria: Biochemistry, Physiology, and Molecular Genetics*. Sonenshein, A.L., Hoch, J.A., and Losick, R. (eds). Washington, DC: American Society for Microbiology Press, pp. 381–410.

Bolhuis, A., Venema, G., Quax, W.J., Bron, S., and van Dijl, J.M. (1999a) Functional analysis of paralogous thiol-disulfide oxidoreductases in *Bacillus subtilis*. *J Biol Chem* 274: 24531–24538.

Bolhuis, A., Tjalsma, H., Smith, H.E., de Jong, A., Meima, R., Venema, G., *et al.* (1999b) Evaluation of bottlenecks in the late stages of protein secretion in *Bacillus subtilis*. *Appl Environ Microbiol* 65: 2934–2941.

Bolhuis, A., Tjalsma, H., Stephenson, K., Harwood, C.R., Venema, G., Bron, S., and van Dijl, J.M. (1999c) Different mechanisms for thermal inactivation of *Bacillus subtilis* signal peptidase mutants. *J Biol Chem* 274: 15865–15868.

Bolhuis, A., Matzen, A., Hyyryläinen, H.-L., Kontinen, V.P., Meima, R., Chapuis, J., *et al.* (1999d) Signal peptide peptidase- and ClpP-like proteins of *Bacillus subtilis* required for efficient translocation and processing of secretory proteins. *J Biol Chem* 274: 24585–24592.

Connolly, L., De Las Peñas, A., Alba, B.M., and Gross, C.A. (1997) The response to extracytoplasmic stress in *Escherichia coli* is controlled by partially overlapping pathways. *Genes Dev* 11: 2012–2021.

Danese, P.N., and Silhavy, T.J. (1997) The σ^E^ and the Cpx signal transduction system control the synthesis of periplasmic protein-folding enzymes in *Escherichia coli*. *Genes Dev* 11: 1183–1193.

Dartigalongue, C., and Raina, S. (1998) A new heat-shock gene, ppiD, encodes a peptidyl-prolyl isomerase required for folding of outer membrane proteins in *Escherichia coli*. *EMBO J* 17: 3968–3980.

De Las Peñas, A., Connolly, L., and Gross, C.A. (1997) The σ^E^-mediated response to extracytoplasmic stress in *Escherichia coli* is transduced by RseA and RseB, two negative regulators of σ^E^. *Mol Microbiol* 24: 373–385.

De Wulf, P., Kwon, O., and Lin, E.C. (1999) The CpxRA signal transduction system of *Escherichia coli*: growth-related autoactivation and control of unanticipated target operons. *J Bacteriol* 181: 6772–6778.

van Dijl, J.M., de Jong, A., Venema, G., and Bron, S. (1995) Identification of the potential active site of the signal peptidase SipS of *Bacillus subtilis*. *J Biol Chem* 270: 3611–3618.

Guérout-Fleury, A.M., Shazand, K., Frandsen, N., and Stragier, P. (1995) Antibiotic-resistance cassettes for *Bacillus subtilis*. *Gene* 167: 335–336.

Hastrup, S., and Jacobs, M.F. (1990) Lethal phenotype conferred by xylose-induced overproduction of April-LacZ fusion protein. In *Genetics and Biotechnology of Bacilli*. Zukowski, M.M., Ganesan, A.T., and Hoch, J.A. (eds). San Diego: Academic Press, pp. 33–41.

Hecker, M., Schumann, W., and Volker, U. (1996) Heat-shock

Control of secretion stress in B. subtilis 1171 and general stress response in *Bacillus subtilis*. *Mol Microbiol* 19: 417–428.

Hyyryläinen, H.-L., Vitikainen, M., Thwaite, J., Wu, H., Sarvas, M., Harwood, C.R., *et al.* (2000) D-alanine substitution of teichoic acids as a modulator of protein folding and stability at the cytoplasmic membrane/cell wall interface of *Bacillus subtilis*. *J Biol Chem* 275: 26696–26703.

Jacobs, M., Anderssen, J.B., Kontinen, V., and Sarvas, M. (1993) *Bacillus subtilis* PrsA is required *in vivo* as an extracytoplasmic chaperone for secretion of active enzymes synthesized either with or without pro-sequence. *Mol Microbiol* 8: 957–966.

Kontinen, V.P., and Sarvas, M. (1993) The PrsA lipoprotein is essential for protein secretion in *Bacillus subtilis* and sets a limit for high-level secretion. *Mol Microbiol* 8: 727–737.

Kontinen, V.P., Saris, P., and Sarvas, M. (1991) A gene (*prsA*) of *Bacillus subtilis* involved in a novel, late stage of protein export. *Mol Microbiol* 5: 1273–1283.

Kunst, F., and Rapoport, G. (1995) Salt stress is an environmental signal affecting degradative enzyme synthesis in *Bacillus subtilis*. *J Bacteriol* 177: 2403–2407.

Kunst, F., Ogasawara, N., Moszer, I., Albertini, A.M., Alloni, G., Azevedo, V., *et al.* (1997) The complete genome sequence of the gram-positive bacterium *Bacillus subtilis*. *Nature* 390: 249–256.

Leskelä, S., Wahlström, E., Kontinen, V.P., and Sarvas, M. (1999a) Lipid modification of prelipoproteins is dispensable for growth but essential for efficient protein secretion in *Bacillus subtilis*: characterization of the *Igt* gene. *Mol Microbiol* 31: 1075–1085.

Leskelä, S., Wahlström, E., Hyyryläinen, H.-L., Jacobs, M., Palva, A., Sarvas, M., and Kontinen, V.P. (1999b) Ecs, an ABC transporter of *Bacillus subtilis*: dual signal transduction functions affecting expression of secreted proteins, as well as their secretion. *Mol Microbiol* 31: 533–543.

Margot, P., and Karamata, D. (1996) The *wprA* gene of *Bacillus subtilis* 168, expressed during exponential growth, encodes a cell-wall-associated protease. *Microbiology* 142: 3437–3444.

Miller, J.H. (1982) *Experiments in Molecular Genetics*. Cold Spring Harbor, NY. Cold Spring Harbor Laboratory Press.

Missiakas, D., and Raina, S. (1997) Protein misfolding in the cell envelope of *Escherichia coli*: new signaling pathways. *Trends Biochem Sci* 22: 59–63.

Missiakas, D., and Raina, S. (1998) The extracytoplasmic function sigma factors: role and regulation. *Mol Microbiol* 28: 1059–1066.

Missiakas, D., Mayer, M.P., Lemaire, M., Georgopoulos, C., and Raina, S. (1997) Modulation of the *Escherichia coli* σ^E^ (RpoE) heat-shock transcription-factor activity by the RseA, RseB and RseC proteins. *Mol Microbiol* 24: 355–371.

Nakayama, S., and Watanabe, H. (1998) Identification of *cpxR* as a positive regulator essential for expression of the *Shigella sonnei virF* gene. *J Bacteriol* 180: 3522–3528.

Noone, D., Howell, A., and Devine, K.M. (2000) Expression of *ykdA*, encoding a *Bacillus subtilis* homologue of HtrA, is heat shock inducible and negatively autoregulated. *J Bacteriol* 182: 1592–1599.

Pallen, M.J., and Wren, B.W. (1997) The HtrA family of serine proteases. *Mol Microbiol* 26: 209–221.

Palva, I. (1982) Molecular cloning of α-amylase gene from

© 2001 Blackwell Science Ltd, *Molecular Microbiology*, 41, 1159–1172

Bacillus amyloliquefaciens and its expression in Bacillus subtilis. Gene 19: 81–87.

Pogliano, J., Lynch, A.S., Belin, D., Lin, E.C.C., and Beckwith, J. (1997) Regulation of Escherichia coli cell envelope proteins involved in protein folding and degradation by the Cpx two-component system. Genes Dev 11: 1169–1182.

Raina, S., Missiakas, D., and Georgopoulos, C. (1995) The rpoE gene encoding the σ<sup>E</sup> (σ<sup>24</sup>) heat shock sigma factor of Escherichia coli. EMBO J 14: 1043–1055.

Rouvière, P.E., De Las Peñas, A., Mecsas, J., Lu, C.Z., Rudd, K.E., and Gross, C.A. (1995) rpoE, the gene encoding the second heat-shock sigma factor, σ<sup>E</sup>, in Escherichia coli. EMBO J 14: 1032–1042.

Sambrook, J., Fritsch, E.F., and Maniatis, T. (1989) Molecular Cloning: a Laboratory Manual. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.

Spiess, C., Beil, A., and Ehrmann, M. (1999) A temperature-dependent switch from chaperone to protease in a widely conserved heat shock protein. Cell 97: 339–347.

Stephenson, K., and Harwood, C.R. (1998) Influence of a cell-wall-associated protease on production of α-amylase by Bacillus subtilis. Appl Environ Microbiol 64: 2875–2881.

Stephenson, K., Carter, N.M., Harwood, C.R., Petit-Glatron, M.F., and Chambert, R. (1998) The influence of protein

folding on late stages of the secretion of α-amylases from Bacillus subtilis. FEBS Lett 430: 385–389.

Thompson, J.D., Higgins, D.G., and Gibson, T.J. (1994) CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res 22: 4673–4680.

Tjalsma, H., Bolhuis, A., Jongbloed, J.D.H., Bron, S., and van Dijl, J.M. (2000) Signal peptide-dependent protein transport in Bacillus subtilis: a genome-based survey of the secretome. Microbiol Mol Biol Rev 64: 515–547.

Vagner, V., Dervyn, E., and Ehrlich, S.D. (1998) A vector for systematic gene inactivation in Bacillus subtilis. Microbiology 144: 3097–3104.

Vitikainen, M., Pummi, T., Airaksinen, U., Wahlström, E., Wu, H., Sarvas, M., and Kontinen, V.P., (2001) Quantitation of the capacity of the secretion apparatus and the requirement for PrsA in the growth and secretion of α-amylase in Bacillus subtilis. J Bacteriol 183: 1881–1890.

Wertman, K.F., Wyman, A.R., and Botstein, D. (1986) Host/vector interactions which affect the viability of recombinant phage lambda clones. Gene 49: 253–262.

Yura, T., and Nakahigashi, K. (1999) Regulation of the heat-shock response. Curr Opin Microbiol 2: 153–158.
