
MECHANISMS OF SIGNAL
TRANSDUCTION:
Disturbed Activation of Endoplasmic
Reticulum Stress Transducers by Familial
Alzheimer's Disease-linked Presenilin-1
Mutations

Taiichi Katayama, Kazunori Imaizumi, Akiko
Honda, Takunari Yoneda, Takashi Kudo,
Masatoshi Takeda, Kazutoshi Mori, Richard
Rozmahel, Paul Fraser, Peter St.
George-Hyslop and Masaya Tohyama
J. Biol. Chem. 2001, 276:43446-43454.
doi: 10.1074/jbc.M104096200 originally published online September 10, 2001

---

Access the most updated version of this article at doi: [10.1074/jbc.M104096200](https://doi.org/10.1074/jbc.M104096200)

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinity-sites).

Alerts:
- [When this article is cited](https://www.jbc.org/site/alerts)
- [When a correction for this article is posted](https://www.jbc.org/site/alerts)

[Click here](https://www.jbc.org/site/alerts) to choose from all of JBC's e-mail alerts

This article cites 38 references, 14 of which can be accessed free at
[http://www.jbc.org/content/276/46/43446.full.html#ref-list-1](http://www.jbc.org/content/276/46/43446.full.html#ref-list-1)

THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2001 by The American Society for Biochemistry and Molecular Biology, Inc.

Disturbed Activation of Endoplasmic Reticulum Stress Transducers
by Familial Alzheimer's Disease-linked Presenilin-1 Mutations*

Received for publication, May 7, 2001, and in revised form, September 6, 2001
Published, JBC Papers in Press, September 10, 2001, DOI 10.1074/jbc.M104096200

Taiichi Katayama‡§∥, Kazunori Imaizumi¶****, Akiko Honda‡§∥, Takunari Yoneda‡$, Takashi Kudo§§, Masatoshi Takeda§§, Kazutoshi Mori¶¶, Richard Rozmahel∥∥, Paul Fraser∥∥, Peter St. George-Hyslop∥∥, and Masaya Tohyama‡§

From the ‡Department of Anatomy and Neuroscience, Graduate School of Medicine, Osaka University, 2-2 Yamadaoka, Suita, Osaka 565-0871, Japan, **Division of Structural Cell Biology, Nara Institute of Science and Technology (NAIST), 8916-5 Takayama, Ikoma, Nara 630-0101, Japan, ||Discovery Research Laboratory, Tanabe Seiyaku Co. Ltd., 3-16-89 Kashima, Yodogawaku, Osaka 532-0031, Japan, §§Department of Clinical Neuroscience, Psychiatry, Graduate School of Medicine, Osaka University, 2-2 Yamadaoka, Suita, Osaka 565-0871, Japan, ¶¶Graduate School of Biostudies, Kyoto University, 46-29 Yoshida-Shimoadachi, Sakyo-ku, Kyoto 606-8304, Japan, ∥∥Center for Research into Neurodegenerative Diseases, Department of Medicine (Neurology), University of Toronto, Toronto, Ontario, M5S 1A8, Canada, and $CREST of Japan Science and Technology Corporation (JST), Kawaguchi, Saitama 332-0012, Japan

Recent studies have shown independently that presenilin-1 (PS1) null mutants and familial Alzheimer's disease (FAD)-linked mutants should both down-regulate signaling of the unfolded protein response (UPR). However, it is difficult to accept that both mutants possess the same effects on the UPR. Furthermore, contrary to these observations, neither loss of PS1 and PS2 function nor expression of FAD-linked PS1 mutants were reported to have a discernable impact on the UPR. Therefore, re-examination and detailed analyses are needed to clarify the relationship between PS1 function and UPR signaling. Here, we report that PS1/PS2 null and dominant negative PS1 mutants, which are mutated at aspartate residue 257 or 385, did not affect signaling of the UPR. In contrast, FAD-linked PS1 mutants were confirmed to disturb UPR signaling by inhibiting activation of both Ire1α and ATF6, both of which are endoplasmic reticulum (ER) stress transducers in the UPR. Furthermore, PS1 mutants also disturbed activation of PERK (PKR-like ER kinase), which plays a crucial role in inhibiting translation during ER stress. Taken together, these observations suggested that PS1 mutations could affect signaling pathways controlled by each of the respective ER-stress transducers, possibly through a gain-of-function.

(APP) gene located on chromosome 21 (2–4), presenilin-1 (PS1) found on chromosome 14 (5), and presenilin-2 (PS2) located on chromosome 1 (6–8). Mutations in the gene encoding PS1 are responsible for many cases of FAD. Although the mechanisms by which mutations in PS1 predispose individuals to FAD have not yet been determined, FAD-linked PS1 mutations have been shown to be associated with altered proteolytic processing of APP, such as increases in the production of the highly amyloidogenic Aβ peptide, Aβ42 (9, 10). Aβ production accompanied by the accumulation of carboxyl-terminal fragments of APP, which are generated by cleavage of β- and γ-secretases for APP, was inhibited in PS1-deficient cells (11). This finding indicated that PS1 is important in the γ-secretase-mediated cleavage process involved in generating Aβ.

Mutations in PS1 increase cellular susceptibility to apoptosis induced by various insults, including the withdrawal of trophic factors and exposure to Aβ (12, 13). Previously, we reported that the mechanism by which PS1 mutations promote cell death could contribute to increasing vulnerability to endoplasmic reticulum (ER) stress by altering the signaling pathway of the unfolded protein response (UPR) (14). The accumulation of misfolded polypeptides in the lumen of the ER results in activation of the UPR, inducing transcriptional up-regulation of genes encoding molecular chaperones and folding catalysts present in the ER (15). One of the mediators of the UPR is a transducer of ER stress, the ER-resident transmembrane kinase Ire1 (16, 17). Ire1 senses the perturbed environment in the ER and leads to downstream signaling by a process that is thought to depend on oligomerization and autophosphorylation of its kinase domain. The down-regulation of UPR signaling by PS1 mutations is caused by disruption of the function of Ire1, i.e. PS1 mutations decrease the levels of phosphorylated Ire1 (14). Recently, Niwa et al. (18) reported that PS1-deficient cells show a reduced ability to mount the UPR, with a reduction in nuclear accumulation of carboxyl-terminal fragments of Ire1. They speculated that impairment of the UPR is based on a defect in UPR-induced proteolytic processing of Ire1 at its intramembranous region by defects of γ-cleaving activities associated with PS1. Contrary to these observations, Sato et al. (19) have reported that neither PS1 null nor FAD-linked mutations affect the UPR signaling. Therefore, a re-examination and detailed analyses of relationships between PS1 functions and the UPR should be performed.

ATF6 is an ER transmembrane protein, recently isolated as

* The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
† These authors contributed equally to this work.
‡ To whom correspondence should be addressed. Tel.: 81-743-72-5411; Fax: 81-743-72-5419; E-mail: imaizumi@bs.aist-nara.ac.jp.
¹ The abbreviations used are: Aβ, amyloid β-protein; APP, amyloid precursor protein; ER, endoplasmic reticulum; FAD, familial Alzheimer's disease; PS1 and -2, presenilin-1 and -2; UPR, unfolded protein response; eIF2α, eukaryotic initiation factor-2α; Tm, tunicamycin; TG, thapsigargin; DTT, dithiothreitol; ATF6, activating transcription factor 6; PERK, PKR-like ER kinase; BiP, immunoglobulin-binding protein.

a possible mammalian UPR-specific transcription factor (20–22). ATF6 is cleaved at or close to the cytosolic face of the membrane in response to ER stress. The amino-terminal cytoplasmic domain, which contains the DNA-binding, dimerization, and transactivation domains, is translocated into the nucleus and activates transcription of ER molecular chaperone genes containing the ER stress response element (ERSE) (20, 23), which is thought to be a regulatory element of promoter regions conserved in ER molecular chaperone genes in mammalian cells. However, it has not been determined whether the luminal domain of ATF6 can itself sense the accumulation of unfolded proteins in the ER or whether cleavage of ATF6 requires ER stress-dependent activation of Ire1 proteins. If PS1 null or FAD-linked mutations affect the induction of ER molecular chaperones as described previously, these mutations could disturb the function of ATF6 as well as Ire1.

In the present study, to make it clear whether PS1 is associated with UPR signaling, we re-examined the effects of PS1 on induction of BiP/GRP78 mRNA and the ER stress response using PS1-deficient cells and cell lines expressing dominant negative PS1 mutants, with mutations of aspartate residues located in transmembrane domains 6 and 7 of PS1 (24), and expressing FAD-linked PS1 mutants. Furthermore, to clarify the mechanisms by which mutant PS1 down-regulates signaling of the UPR, we examined whether PS1 mutants affected the signaling pathways mediated by other ER stress transducers such as ATF6 or PKR-like ER kinase (PERK), which is an ER stress transducer and may play a role in inhibiting translation during ER stress (25, 26).

### EXPERIMENTAL PROCEDURES

#### Cell Culture and Induction of ER Stress—Human neuroblastoma SK-N-SH cells and mouse embryonic fibroblasts were cultured in α-minimal essential medium (Life Technologies, Inc.) supplemented with 10% fetal calf serum and Dulbecco’s modified Eagle’s medium (DMEM, Life Technologies, Inc.) supplemented with 20% fetal calf serum, respectively. Two days before the stimulation by various ER stressors, \(3 \times 10^5\) cells were plated in 10-cm-diameter dishes (for Western and Northern blotting analyses) or \(1 \times 10^5\) cells in 6-cm-diameter dishes (for cell death assay), with the media changed for fresh ones the next day. In all experiments of ER stress response, we used only culture dishes that were grown at 70–80% confluence to avoid various stresses induced by overgrowth. On the day of stimulation, cells were placed in fresh media for more than 1 h before treatment with ER stressors to create the same conditions in each dish. We used tunicamycin, thapsigargin, and DTT (all from Sigma) as ER stress inducers at indicated doses and times. The control dishes were treated by just changing the media.

Before collecting, cells were washed in ice-cold phosphate-buffered saline and then lysed in aliquots of 200 μl of Triton buffer (20 mM HEPES, pH 7.5, 150 mM NaCl, 1% Triton X-100, 10% glycerol, 1 mM EDTA, 10 mM tetrasodium pyrophosphate, 100 mM NaF, 17.5 mM β-glycerophosphate, 1 mM phenylmethylsulfonyl fluoride, 4 mg/ml aprotinin, and 2 mg/ml pepstatin A) for detection of Ire1α and PERK. For detection of ATF6, the cells were lysed in equal volumes of hot-SDS buffer (0.9% SDS, 15 mM EDTA, 8 mM methionine, and 1,000 units of Trasylol were incubated in a boiling water bath for 10 min, cooled, diluted to 0.3% SDS, and adjusted to contain 33 mM Tris acetate, pH 8.5, and 1.7% Triton X-100). All samples were centrifuged at 4 °C for 10 min after standing on ice for 1 h, and then the supernatants were collected in 1.5-ml Eppendorf tubes.

#### Antibodies—Rabbit antisera against a synthetic peptide corresponding to residues 1–14 of human PS1 have been described previously (14). Anti-PERK antibody was raised against a synthetic peptide corresponding to residues of mouse PERK (residues 1094–1114) and was affinity-purified by ProtOn Kit1 (Multiple Peptide System, San Diego). Anti-ATF6 antibodies against recombinant ATF6 (amino acids 6–307) fused to *Escherichia coli* maltose-binding protein have also been described previously (21). Anti-Ire1α antibodies were provided by Prof. D. Ron (New York University School of Medicine). Anti-phosphorylated eIF2α polyclonal and anti-β-actin monoclonal antibodies were purchased from Research Genetics Co. and Chemicon International Inc., respectively.

#### Expression Plasmids and Gene Transfer into Cells—Wild-type, A246E, and ΔE9 PS1 cDNAs were cloned into pCDNA3 (Invitrogen) as described previously (14). Human SK-N-SH neuroblastoma cells or mouse Neuro 2a cells were transfected with each expression plasmid using LipofectAMINE (Life Technologies, Inc.), and each stable transformant was established by selection with G418. Dominant negative PS1 mutants, which were mutated at aspartate residue 257 (D257A) or 385 (D385A), were cloned into pCDNA3.1/Zeo (Invitrogen) at EcoRI sites.

#### Western Blotting—For Western blotting of Ire1α and PERK, cells were lysed in 1% Triton lysis buffer. Samples were loaded onto a 7% SDS-polyacrylamide gels. For Western blotting of ATF6, cells were lysed in Laemmli SDS sample buffer and then loaded onto 10% SDS-polyacrylamide gels. The protein concentration of each sample was quantified by Lowry assay (DC protein assay, Bio-Rad). Protein-equivalent samples were subjected to Western blotting. As an internal control, the levels of β-actin were examined on the same blot at the same time as the other proteins of interest.

#### Northern Blotting—Total RNA was extracted from cells treated with ER stress inducers. Aliquots of 10 μg of each total RNA were fractionated by electrophoresis through 1.0% agarose-formaldehyde gels and transferred onto Immobilon-N membranes (Millipore, Bedford, MA). The membranes were hybridized with ³²P-labeled human BiP cDNA probe. After washing in 2× SSC, 0.1% SDS and 0.1× SSC, 0.1% SDS, the membranes were dried and autoradiographed. The human BiP cDNA was a gift from Dr. S. Kaneda (Heat Shock Protein Research Institute).

#### Fluorescence Microscopy—Fibroblasts from PS1 mutant knock-in mice or SK-N-SH cells stably transfected with each PS1 construct were incubated with or without 3 μg/ml Tm for the specified times. Cells were fixed in 4% paraformaldehyde and permeabilized in 0.3% Triton X-100. Anti-ATF6 antibody was used at a dilution of 1:1000. Stained cells were viewed with a confocal microscope (LSM 510, Carl Zeiss). To estimate the relative nuclear fluorescence intensity, cells were co-stained with 4,6-diamidino-2-phenylindole to demarcate the nuclear boundaries. Then, cytoplasmic (C), nuclear (N), or both (N/C) types of staining of ATF6 were measured independently in 10 fields. Each value was represented by the percentage of positive cells in each fraction, and the total of C, N, and N/C was 100%.

#### Presenilin Null Mice—Mice lacking functional expression of PS1 were generated using homologous recombination to introduce a neomycin cassette flanked by loxP sequences ~200 base pairs downstream of exon 5 of the murine PS1 gene. When the loxP flanked neomycin cassette is not removed by cre recombinase, the presence of cryptic splice sites within the neomycin cassette results in the generation of extremely low quantities of unstable transcripts lacking an open reading frame after exon 5.² These transcripts result in no detectable PS1 protein expression in embryonic stem cells or in the brains of animals homozygous for this construct. As expected, homozygous PS1-deficient mice (PS1⁻/⁻) displayed the previously reported skeletal abnormalities (27, 28). Embryonic fibroblasts from PS1/PS2 double homozygous-deficient mice (29) and the littermates were kindly provided by Dr. B. De Strooper.

#### Culture of Fibroblasts from PS1-deficient and PS1 Mutant Knock-in Mice—Primary fibroblasts were cultured from PS1⁺/⁻ and PS1⁻/⁻ embryos on embryonic day 14.5. Fibroblast cultures were also prepared from fetal mice at embryonic day 14.5, generated by mating heterozygous PS1 mutation (I213T) knock-in mice as described (30). These cells were grown in Dulbecco’s modified Eagle’s medium supplemented with 10% fetal calf serum and were treated with 2–3 μg/ml Tm for the specified times. For evaluation of cell death, lactate dehydrogenase levels in culture medium were quantified as described previously (14).

---

### RESULTS

#### UPR Signaling in PS1 Mutant Cells—PS1⁻/⁻ cells have been reported to show impaired induction of BiP/GRP78, an ER molecular chaperone (18), suggesting that native PS1 is involved in UPR signaling. We re-examined the induction of BiP mRNA in PS1⁻/⁻, PS1/PS2⁻/⁻ fibroblasts, and SK-N-SH cell lines expressing PS1 dominant negative mutants (D257A and D385A). The cells were stimulated with 3 μg/ml tunicamycin (Tm), which induces ER stress by preventing protein glycosylation, for specified times. Northern blotting showed that there was no difference in the induction of BiP mRNA between

² R. Rozmahel, P. Fraser, and P. St. George-Hyslop, manuscript in preparation.

43448

**Disturbance of ER Stress Transducers by PS1 Mutants**

**a**
PS1(+/-)  
0 1 2 3 6  

PS1(-/-)  
0 1 2 3 6 hr  

← BiP  
← CHOP  
← β-actin  

**PS1 Knock-out mice fibroblast**  

**b**
PS1/2(+/-)  
0 1 2 3 6  

PS1/2(-/-)  
0 1 2 3 6 hr  

← BiP  
← β-actin  

**c**
Wild  
0 1 2 3 6  

D257A  
0 1 2 3 6  

D385A  
0 1 2 3 6 hr  

← BiP  
← β-actin  

FIG. 1. Induction of BiP mRNA in PS1⁻⁻ or PS1 dominant negative cells. Northern blotting of BiP/GRP78 and CHOP mRNAs. Primary cultured fibroblasts from PS1 +/– and –/– (a) and PS1/2 +/+ and –/– (b) mice were exposed to Tm at a concentration of 3 μg/ml for the indicated times. Total RNA was isolated from each culture and subjected to Northern blotting with probes for BiP, CHOP, or β-actin mRNA. Lower panel, left, the amounts of BiP mRNA in each cell are shown. Quantitative analyses of BiP mRNA levels as shown in autoradiographs were performed as described previously (14). Values are arbitrary intensities and represent the mean ± S.D. of data from four independent experiments. Right, the expression of PS1 in fibroblasts from PS1-deficient mice. Full-length and amino-terminal fragments of PS1 were absent in PS1-deficient cells. c, SK-N-SH cells were stably transfected with dominant negative PS1 mutants mutated at aspartate residues 257 or 385. Wild, cells stably transfected with wild-type PS1. The lower panel shows quantitative analyses of BiP mRNA levels as shown in autoradiographs and Western blotting of PS1 in these cells. Note that mutant cells showed accumulation of full-length PS1 (arrow).

PS1⁻⁻ and PS1⁺⁺ fibroblasts, PS1/PS2⁻⁻ and PS1/PS2⁺⁺ fibroblasts, or SK-N-SH cells expressing wild-type PS1 and dominant negative mutants (Fig. 1). No effects of these mutations on the induction of BiP mRNA were confirmed at concentrations from 0.5 to 10 μg/ml of Tm. In contrast, fibroblasts from FAD-linked PS1 mutant knock-in mice, which express the PS1 mutant (I213T) at normal physiological levels (30), and

SK-N-SH cells expressing PS1 mutant (delta exon 9, PS1Δ9) revealed that the induction of BiP mRNA was delayed and attenuated compared with wild-type cells at each dose from 0.5 to 3 μg/ml Tm (Fig. 2). Furthermore, we examined the changes in induction of CHOP, which is also induced during ER stress (31, 32). We detected no differences in the induction of CHOP mRNA in PS1⁻⁻ fibroblasts (Fig. 1) or aspartate mutant-

Disturbance of ER Stress Transducers by PS1 Mutants

**FIG. 2. BiP mRNA in PS1 mutation knock-in or transfected cells.** $a$, primary cultured fibroblasts from PS1 mutation ($I213T$) knock-in mice (wild and homozygous ($Homo$)). Time courses of the levels of BiP, CHOP, and $\beta$-actin mRNA after treatment with Tm are shown. $b$, SK-N-SH cells were stably transfected with each PS1 construct (Wild, wild-type; $\Delta E9$, mutant deleted exon 9). Induction of BiP and/or CHOP mRNA was attenuated in homozygous fibroblasts and $\Delta E9$-expressing SK-N-SH cells when cells were treated with $3 \mu g/ml$ Tm. These results were obtained for doses from $0.5$ to $3 \mu g/ml$ Tm.

expressing neuroblastomas (data not shown). Cells expressing FAD-linked PS1 mutants exhibited a delay in CHOP mRNA induction similar to that in BiP mRNA induction (Fig. 2). The effects of FAD-linked PS1 mutations on the induction of BiP and CHOP mRNA were observed at each dose of Tm from $0.5$ to $5 \mu g/ml$, but they could not be detected at $10 \mu g/ml$ because of the toxic effects of Tm, suggesting that higher doses of ER stress inducers could mask the down-regulation of UPR induced by PS1 mutation. Down-regulation of BiP mRNA induction by FAD-linked PS1 mutation was also observed in treatments with thapsigargin (TG) and dithiothreitol (DTT) (data not shown). These results indicated that although native PS1 is not involved in UPR signaling, FAD-linked PS1 mutants do have an effect that may be attributable to a gain of aberrant functions.

**Vulnerability to ER Stress in Cells Expressing PS1 Mutants—** We previously demonstrated that cells expressing PS1 mutants exhibited increased vulnerability to various ER stresses, the mechanism of which was the down-regulation of UPR signaling (14). If native PS1 is not relevant to UPR signaling, sensitivity to ER stress in PS1-deficient cells should be equivalent to that in wild-type cells. Therefore, we carried out cell death assays using various mutant cells treated with Tm. Cell death induced by Tm was significantly promoted in PS1 mutation knock-in fibroblasts and SK-N-SH cells expressing PS1$\Delta9$ (Fig. 3c), findings consistent with those observed previously. In contrast, PS1$^{-/-}$ fibroblasts and SK-N-SH cells expressing PS1 dominant negative mutants showed no differences in cell death induced by ER stress as compared with the respective controls (Fig. 3, $a$ and $b$). Thus, we confirmed that a deficiency of PS1 does not affect susceptibility to ER stress and that native PS1 is not associated with UPR signaling.

**Mutant PS1 Alters Proteolytic Cleavage and Translocation of ATF6—** ATF6 is cleaved at or close to the cytosolic face of the membrane, and its amino-terminal portion is translocated into

**FIG. 3. Vulnerability to ER stress in each PS1 mutant-expressing cell.** PS1-deficient fibroblasts ($a$), SK-N-SH cells stably transfected with each PS1 construct ($b$), and PS1 mutation (I213T) knock-in fibroblasts ($c$) were treated with Tm (fibroblasts, $3 \mu g/ml$; SK-N-SH cells, $0.5 \mu g/ml$) for the indicated times. Supernatants of each culture were collected, and the activities of released lactate dehydrogenase (LDH) were determined. The data are means $\pm$ S.D. of four independent experiments. Only cells expressing FAD-linked PS1 mutations exhibited increased vulnerability to ER stress. *, $p < 0.05$; **, $p < 0.01$ relative to the controls (wild-type cells); significance was calculated by Student’s $t$ test.

the nucleus during ER stress. The amino-terminal domain activates transcription of ER molecular chaperones such as BiP and GRP94 (20, 21, 22). We examined whether FAD-linked PS1

43450

**Disturbance of ER Stress Transducers by PS1 Mutants**

a  
Knock out mice  
+/–  
–/–  

kDa  
97–  
66–  
46–  

actin  

p50ATF6/β-actin  
(arbitrary unit)  
80  
60  
40  
20  
0  

b  
Knock-in mice  
Wild  
Homo  

kDa  
97–  
66–  
46–  

actin  

p50ATF6/β-actin  
(arbitrary unit)  
80  
60  
40  
20  
0  

c  

SK-N-SH cell line  

arbitrary unit  
125  
100  
75  
50  
25  
0  

wild  
A246E  

wild  
ΔE9  

wild  
D385A  

FIG. 4. Conversion of ATF6 to p50ATF6 after ER stress. Primary cultured fibroblasts from PS1 knock-out (a) and PS1 mutation knock-in (b) mice were treated with 2 μg/ml Tm for the indicated times, and cells were lysed as described under “Experimental Procedures.” Then, aliquots of lysates were subjected to Western blotting of ATF6. Wild-type cells showed accumulation of p50ATF6 band (arrowhead) after treatment with Tm. a, the amounts of p50ATF6 in PS1⁻/⁻ cells were almost equivalent to those in PS1⁺/⁻ cells. Middle panel, Western blotting of β-actin. Lower panel, the amounts of accumulated p50ATF6 fragments were analyzed quantitatively and normalized to expression levels of β-actin protein. Values are arbitrary intensities and represent the mean ± S.D. of data from four independent experiments. b, homozygous PS1 mutation knock-in cells (Homo) showed lower levels of p50ATF6 than those in wild-type cells at each time point. c, quantitative analysis of accumulated p50ATF6 fragments in each stable transfectant of SK-N-SH cells after treatment with Tm. FAD-linked PS1 mutation (A246E, ΔE9)-expressing cells showed that the amounts of accumulated p50ATF6 fragments were decreased at each time point compared with wild type-expressing cells. Values are arbitrary intensities and represent the means ± S.D. of data from four independent experiments.

mutants also impaired activation of the other UPR transducer protein, ATF6, similarly to Ire1 as has been reported previously (14). Fibroblasts from PS1 mutant knock-in mice were stimulated by the addition of 2 μg/ml Tm. Cells were lysed, and Western blotting was performed using anti-ATF6 antibodies raised against the amino-terminal region of ATF6 (amino acids 6–307). ATF6 was constitutively expressed in wild-type fibroblasts as a 90-kDa protein (p90ATF6) (Fig. 4b). The p90ATF6 levels decreased, and instead a new band of 50-kDa (p50ATF6) appeared 2 h after treatment with Tm, with p50ATF6 gradually accumulating after this time. The homozygous fibroblasts from the PS1 mutant knock-in mice showed that the production of p50ATF6 was delayed and that the levels of the fragments were decreased by ~50% compared with those from wild-type mice from 2 to 6 h after treatment with Tm (Fig. 4b). We also confirmed that the conversion of p90ATF6 to p50ATF6 was inhibited in SK-N-SH neuroblastoma cells stably transfected with FAD-linked PS1 mutants, such as PS1Δ9 or PS1 with an alanine-to-glutamate mutation (A246E) (Fig. 4c). These findings indicated that PS1 mutations disturb the conversion of

ATF6 under conditions of ER stress. In contrast, these effects on ATF6 were not observed in either PS1-deficient cells (Fig. 4a) or in PS1 dominant negative-expressing cells (Fig. 4c). Thus, native PS1 is not considered to play a role in the processing of ATF6 at or close to the cytosolic face of the ER membrane, findings that are likely to be different in the case of processing at the γ-site of APP.

ATF6 has been reported to be constitutively expressed at the perinuclear region under non-ER stress conditions. ER stress causes the conversion of p90ATF6 to p50ATF6, which is translocated into the nucleus. We performed an immunocytochemical analysis of the subcellular localization of ATF6 in PS1 mutant cells under normal or ER stress conditions. For evaluation of ATF6 translocation, the fluorescence signals of cells stained with anti-ATF6 antibodies were divided into cytoplasmic (C), nuclear (N), or both regions (N/C). N/C shows that ATF6 was shifting from the ER to the nucleus (Fig. 5a). Under non-inducing conditions, the distribution of ATF6 (p90ATF6) in PS1 mutant knock-in cells resembled that seen in fibroblasts expressing wild-type PS1 (Fig. 5b). p90ATF6 was localized

Disturbance of ER Stress Transducers by PS1 Mutants 43451

**FIG. 5. Translocation of ATF6 into nuclei from the ER.** $a$, left, immunoreactivity of ATF6 under non-ER stress conditions. $N$, nuclear; $C$, cytosolic. Right, immunoreactivity of ATF6 during ER stress. The fluorescence signals of cells stained with anti-ATF6 antibodies were divided into cytosolic ($C$), nuclear ($N$), or both regions ($N/C$) as described under “Experimental Procedures.” $b$, PS1 mutation knock-in cells were exposed to Tm for the indicated times. Nuclear translocation of ATF6 fragments was delayed in homozygous cells (Homo) compared with that in wild-type cells (Wild). $c$, quantification of immunofluorescence of ATF6. Upper panel, PS1 knock-out fibroblast. Middle panel, PS1 mutation knock-in fibroblast. Lower panel, SK-N-SH cell lines. Values show the frequency of cells that were localized in each region. Values represent the means ± S.D. as described under “Experimental Procedures.”

predominantly in the ER, and a small fraction of cells (~10%) showed nuclear staining. Nuclear immunoreactivity of ATF6, which has been demonstrated previously as p50ATF6 fragments (21), increased in wild-type cells from 2 h after the induction of ER stress by treatment with Tm and reached a maximum within 4 h (Fig. 5, $b$ and $c$). The changes in ATF6 immunoreactivity were consistent with the results of Western blotting. PS1 mutant knock-in cells showed slight nuclear accumulation of p50ATF6 2 h after induction (Fig. 5, $b$ and $c$). After 4 h, it gradually accumulated in the nucleus, and after 8 h, the immunoreactivity was almost equivalent to that in wild-type fibroblasts. Similar findings were obtained in SK-N-SH cells expressing PS1Δ9 (Fig. 5$c$). Thus, mutation in PS1 obviously delayed nuclear accumulation of p50ATF6, which could be ascribed to the delayed processing of p90ATF6 at the ER membrane (see Fig. 4$b$). In PS1-deficient cells, subcellular localization of ATF6 was not altered compared with that of control cells (Fig. 5$c$).

**Inhibited Activation of Endogenous ER Stress Transducers by PS1 Mutants—**The above findings demonstrated that PS1 mutants affect the activation of endogenous ATF6 after ER stress. To assess whether functions of the other ER stress transducers, Ire1α and PERK, are disturbed by PS1 mutants, we examined the activation of these endogenous molecules during ER stress in cells expressing PS1 mutation. Ire1 and PERK activation during ER stress were correlated with autophosphorylation of their cytoplasmic kinase domains. Initially, to examine the effects of PS1 mutants on the activation of Ire1α, Western blotting was performed using lysates of knock-in fibroblasts stimulated with ER stress inducers. Phosphorylation of Ire1α retards their mobility on SDS-polyacryl-

43452

**Disturbance of ER Stress Transducers by PS1 Mutants**

a

DTT 0.5mM

| 0 | 15 | 30 min |
| --- | --- | --- |
| Wt | Ho | Wt | Ho | Wt | Ho |

![Image](image1)

FIG. 6. Inhibited activation of endogenous ER stress transducers. $a$, fibroblasts from PS1 mutant knock-in mice were treated with 0.5 mM DTT for the indicated times. Upper panel, cells were lysed in 1% Triton buffer, and then Western blotting of Ire1$\alpha$ was carried out. The bands of Ire1$\alpha$ were shifted to a higher apparent molecular weight after treatment with DTT in wild-type (Wt) fibroblasts. The higher molecular weight bands of Ire1$\alpha$ were demonstrated to be derived from phosphorylated forms, as described previously (33). P-Ire1, phosphorylated-Ire1$\alpha$. Phosphorylation of Ire1$\alpha$ was completely inhibited in homozygous mutant fibroblasts (Ho). Lower panel, Western blotting of PERK after treatment of fibroblasts with DTT. Phosphorylation of PERK was also inhibited in homozygous mutant fibroblasts. P-PERK, phosphorylated-PERK.

b

| PS1W | dE9 |
| --- | --- |
| 0 | 15 | 30 | 60 | 0 | 15 | 30 | 60 |

Tg 1$\mu$M

![Image](image2)

When cells were treated with 1 $\mu$M TG (upper panel), the bands of Ire1$\alpha$ were completely shifted (phosphorylated-Ire1$\alpha$) within 15 min in cells expressing wild-type PS1 (PS1W). In contrast, no shift of Ire1$\alpha$-immunoreactive bands was seen in PS1$\Delta$E9 cells treated with the same dose of TG. When cells were treated with 5 $\mu$M TG (lower panel), both cells showed that phosphorylation of Ire1$\alpha$ occurred within 15 min, and there were no differences of the phosphorylation kinetics.

c

| PS1W | dE9 |
| --- | --- |
| 0 | 15 | 30 | 60 | 0 | 15 | 30 | 60 |

Tg 1$\mu$M

![Image](image3)

P-PERK
PERK

| PS1W | dE9 |
| --- | --- |
| 0 | 15 | 30 | 60 | 0 | 15 | 30 | 60 |

5$\mu$M

![Image](image4)

P-PERK
PERK

Wild
$\Delta$E9

| 0 | 5 | 15 | 30 | 0 | 5 | 15 | 30 (min) |
| --- | --- | --- | --- | --- | --- | --- | --- |
|  |  |  |  |  |  |  |  |

P-eIF2$\alpha$

$\beta$-actin

not shown). In PS1 mutant knock-in cells, about 50 or 30% of the PERK molecules remained nonphosphorylated at 15 or 30 min, respectively, after ER stress (Fig. 6a). In SK-N-SH cells stably transfected with PS1 mutants (PS1$\Delta$9), phosphorylation of PERK was severely inhibited within 30 min after treatment with 1 $\mu$M TG compared with that in cells expressing wild-type

PS1 (Fig. 6c) similar to the findings in PS1 mutation knock-in cells. At 60 min after ER stress, PERK was completely phosphorylated in both wild-type and mutant PS1-expressing cells. These results indicated that activation of PERK was delayed by PS1 mutation at the dose of \(1 \mu \mathrm{M}\) TG. When cells were treated with \(5 \mu \mathrm{M}\) TG, PERK was completely phosphorylated in these cells within \(15 \mathrm{~min}\), indicating that excessive doses of ER stress inducers could mask the effects of PS1 mutation on the delay of activation of PERK.

As described above, the disturbed function of PERK is known to cause the down-regulation of phosphorylation of eIF2α (25, 26). Therefore, we examined the levels of phosphorylated eIF2α after ER stress in SK-N-SH cells expressing PS1 mutant. As expected, PS1 mutation inhibited phosphorylation of eIF2α (Fig. 6c). Thus, activation of PERK and the resultant phosphorylation of eIF2α were disturbed by expression of a PS1 mutation. Taken together with the results of inhibited activation of Ire1α, it is possible to conclude that PS1 mutants disturb the signaling pathways from ER stress transducers such as Ire1α, ATF6, and PERK simultaneously.

### DISCUSSION

Our recent study showed that a missense mutation in PS1 leads to attenuation of the induction of BiP under ER stress conditions (14). In the present study, to determine whether the function of wild-type PS1 is essential for or is associated with the UPR signaling system, we analyzed the expression of BiP and CHOP, which are known to be induced by ER stress, using PS1-deficient cells. However, we found no attenuation of BiP or CHOP induction in fibroblasts from PS1-deficient mice in contrast to PS1 mutant knock-in cells. Furthermore, the UPR was not down-regulated in cells expressing dominant negative PS1, which was mutated in aspartate at residues 257 or 385. These findings suggest that wild-type PS1 is not associated with UPR signaling. Previously, PS1−/− cells were reported to show reduced induction of BiP after ER stress (18). In this paper, it was speculated that the mechanisms of attenuated induction of BiP were due to the disturbance of UPR-dependent proteolytic processing of Ire1α by a defect in the γ-secretase activity of PS1. We tried to detect the proteolytic processing of Ire1α and the nuclear accumulation of Ire1α fragments after ER stress in both normal and PS1-deficient cells. However, we did not observe the cleaved fragments of Ire1α in both cells. The cause may be attributed to the instability of Ire1α fragments, i.e., such fragments might be short-lived and therefore difficult to detect. The other possible reason for the nondetection of cleaved fragments of Ire1α is the differences in experimental conditions, such as methods of cell culture and stimuli by ER stress inducers.

Presenilin proteins seem to be important in the cleavage of APP at the γ-site because inhibition of Aβ production is accompanied by the accumulation of carboxyl-terminal fragments of β-APP in PS1−/− cells (11). ATF6 is synthesized as an ER transmembrane protein and is cleaved at or close to the cytosolic face of the membrane. The biochemical characteristics of ATF6 suggest that PS1 may be associated with the cleavage of ATF6 on the ER membrane, similar to the cleavage of APP at the γ-site. However, the conversion of p90ATF6 to p50ATF6 was intact in PS1−/− cells, suggesting that PS1 is not associated with the proteolytic process of ATF6, although PS1 has been shown recently to be required for processing of some of the type 1 transmembrane proteins such as APP (11), Notch (34), and Ire1 (18), and that the cleavage is different from the processing of APP at the γ-site. This conclusion was supported by the lack of changes in the processing of ATF6 in cell lines expressing dominant negative PS1 that was mutated in aspartate at residues 257 or 385. In contrast to native PS1, FAD-linked PS1 mutants inhibited the cleavage of p90ATF6. Although the detailed mechanisms are unclear, mutation of PS1 may affect the processing enzymes of ATF6 on the ER membrane. Recently, ATF6 was demonstrated to be processed by Site-1 protease (S1P) and Site-2 protease (S2P), which are processing enzymes known as sterol regulatory element-binding proteins (SREBPs) (35). FAD-linked PS1 mutations have not been reported to inhibit the processing of SREBPs. Therefore, it is inconceivable to consider that FAD-linked PS1 mutants inhibit the activities of S1P and S2P. Ire1α is known to be sufficient to activate the ATF6 reporter gene, whereas a dominant negative form of Ire1α blocks ER stress activation (36), suggesting that ATF6 is downstream of Ire1α in the ER stress signaling pathway. Our study also showed that FAD-linked PS1 mutants inhibited activation of Ire1α. Taken together, it is likely that disturbances of processing and nuclear transports of ATF6 could be caused by the dysfunction of Ire1α.

Neither PS1 null nor dominant negative mutants affected the signaling of the UPR. In contrast, FAD-linked PS1 mutants did affect it, suggesting that its abilities could be caused by gaining aberrant function. However, it is unclear whether the effects of mutant PS1 on the UPR could be associated with its γ-secretase activities. To address this issue, it a study will need to be made of the ability of PS1 mutants to attenuate the UPR after introducing second mutations into the aspartic acid residues in the catalytic portion of the protein.

The present study has shown that down-regulation of BiP induction by FAD-linked PS1 mutant is due to attenuated signaling of the UPR through decreased levels of phosphorylated Ire1 and inhibition of activation of ATF6 under ER stress conditions. Moreover, PS1 mutants also inhibited the phosphorylation of PERK, which is another ER stress transducer. Therefore, it is possible that mutant PS1 perturbs the functions of each ER stress transducer and inhibits its downstream signal. However, it remains unclear why mutant PS1 inhibits the activation of ER stress transducers. A recent study suggested that activation of the signaling-mediated ER stress transducers could be triggered by dissociation of BiP from stress transducers (33, 36, 37). This dissociation would lead to oligomerization of stress transducers inducing autophosphorylation and the resultant activation of downstream signaling. Previously, we reported that PS1 was associated with Ire1 on the ER membrane (14). If FAD-linked PS1 mutants form malfolded structures, BiP may constitutively bind to PS1 molecules to promote its folding. The complex formation of PS1, Ire1 (and/or PERK), and BiP may inhibit the dissociation of BiP from Ire1 (and/or PERK) under ER stress conditions. However, to clarify the detailed mechanisms responsible for perturbation of ER stress signaling by PS1 mutants, further studies focusing on the structures of PS1 molecules bearing missense mutations and the regulation of ER stress sensing are needed.

Sato et al. (19) recently reported that FAD-linked PS1 mutations do not have a discernible impact on the ER stress response, a finding that is not consistent with the present study. Their inability to reproduce the previous results, in which we showed that FAD-linked PS1 mutation causes the down-regulation of UPR signaling, is likely to be due to their use of cells (HEK293 and Neuro2a cells) that were, in comparison, not sensitive to ER stress and to their use of time points that were too late. They examined activation of PERK at \(5 \mathrm{~h}\) after ER stress. As described in our previous study (38) and present studies, FAD-linked PS1 mutants delayed the activation of stress transducers during ER stress, but the effects of PS1 mutations could be masked by treatment with excessive doses of ER stress inducers or stimulation for too long a time. In addition, it is most important to pre-incubate for more than \(1 \mathrm{~h}\) with fresh medium before treatment with agents to gain

constant data in the ER stress response (as has been described above under “Experimental Procedures”), because the induction levels of BiP mRNA are altered considerably if the changes are not conducted. The decrease of BiP mRNA induction in cells expressing FAD PS1 mutations was ~30 or 50% compared with the controls in the PS1 mutation knock-in fibroblasts or stably transfected SK-N-SH cells, respectively. To detect these subtle defects in the UPR in PS1 mutation-expressing cells, the cells should be handled carefully under the same experimental conditions.

In summary, wild-type PS1 does not play a role in signaling of the UPR, but FAD-linked PS1 mutants do affect it, suggesting that PS1 mutations cause perturbation of the ER stress response through a gain-of-function. The possible mechanism of a perturbed ER stress response could be the inhibition of activation of ER stress transducers such as Ire1α, ATF6, and PERK. The disturbance of these protective systems to ER stress in PS1 mutation-bearing cells may lead to susceptibility to various cellular stresses.

Acknowledgments—We thank Prof. D. Ron (New York University School of Medicine) for antibodies against Ire1α and PERK and Dr. B. De Strooper (Katholieke Universiteit Leuven and Flanders Institute for Biotechnology) for providing PS1/PS2 double knock-out fibroblasts.

**REFERENCES**

1. Selkoe, D. J. (1994) *Annu. Rev. Neurosci.* **17**, 489–517
2. Chartier-Harlin, M. C., Crawford, F., Houlden, H., Warren, A., Hughes, D., Fidani, L., Goate, A., Rossor, M., Roques, P., Hardy, J. & Mullan, M. (1991) *Nature* **353**, 844–846
3. Goate, A., Chartier-Harlin, M. C., Mullan, M., Brown, J., Crawford, F., Fidani, L., Giuffra, L., Haynes, A., Irving, N., James, L., *et al.* (1991) *Nature* **349**, 704–706
4. Mullan, M., Mullan, M., Crawford, F., Axelman, K., Houlden, H., Lilius, L., Winblad, B. & Lannfelt, L. (1992) *Nat. Genet.* **1**, 345–347
5. Sherrington, R., Rogaev, E. I., Liang, Y., Rogaeva, E. A., Levesque, G., Ikeda, M., Chi, H., Lin, C., Li, G., Holman, K., Tsuda, T., Mar, L., Foncin, J.-F., Bruni, A. C., Montesi, M. P., *et al.* (1995) *Nature* **375**, 754–760
6. Levy-Lahad, E., Wijsman, E. M., Nemens, E., Anderson, L., Goddard, K. A., Weber, J. L., Bird, T. D. & Schellenberg, G. D. (1995) *Science* **269**, 970–973
7. Levy-Lahad, E., Wasco, W., Poorkaj, P., Romano, D. M., Oshima, J., Pettingell, W. H., Yu, C. E., Jondro, P. D., Schmidt, S. D., Wang, K., Crowley, A. C., Fu, Y.-H., Guenette, S. Y., Galas, D., Nemens, E., *et al.* (1995) *Science* **269**, 973–977
8. Rogaev, E. I., Sherrington, R., Rogaeva, E. A., Levesque, G., Ikeda, M., Liang, Y., Chi, H., Lin, C., Holman, K., Tsuda, T., Mar, L., Sorbi, S., Nacmias, B., Piacentini, S., Amaducci, L., *et al.* (1995) *Nature* **376**, 775–778
9. Borchelt, D. R., Thinakaran, G., Eckman, C. B., Lee, M. K., Davenport, F., Ratovitsky, T., Prada, C. M., Kim, G., Seekins, S., Yager, D., Slunt, H. H., Wang, R., Seeger, M., Levey, A. I., Gandy, S. E., Copeland, N. G., Jenkins, N. A., Price, D. L., Younkin, S. G. & Sisodia, S. S. (1996) *Neuron* **17**, 1005–1013
10. Duff, K., Eckman, C., Zehr, C., Yu, X., Prada, C. M., Perez-tur, J., Hutton, M., Buee, L., Harigaya, Y., Yager, D., Morgan, D., Gordon, M. N., Holcomb, L.,
Refolo, L., Zenk, B., Hardy, J. & Younkin, S. (1996) *Nature* **383**, 710–713
11. De Strooper, B., Saftig, P., Craessaerts, K., Vanderstichele, H., Guhde, G., Annaert, W., Figura, K. V. & Leuven, F. V. (1998) *Nature* **391**, 387–390
12. Guo, Q., Sopher, B. L., Furukawa, K., Pham, D. G., Robinson, N., Martin, G. M. & Mattson, M. P. (1997) *J. Neurosci.* **17**, 4212–4222
13. Guo, Q., Fu, W., Sopher, B. L., Miller, M. W., Ware, C. B., Martin, G. M. & Mattson, M. P. (1999) *Nat. Med.* **5**, 101–106
14. Katayama, T., Imaizumi, K., Sato, N., Miyoshi, K., Kudo, T., Hitomi, J., Morihara, T., Yoneda, T., Gomi, F., Mori, Y., Nakano, Y., Takeda, J., Tsuda, T., Itoyama, Y., Murayama, O., Takashima, A., St. George-Hyslop, P., Takeda, M. & Tohyama, M. (1999) *Nat. Cell Biol.* **1**, 479–485
15. Sidrauski, C., Chapman, R. & Walter, P. (1998) *Trends Cell Biol.* **8**, 245–249
16. Tirasophon, W., Welihinda, A. A. & Kaufman, R. J. (1998) *Genes Dev.* **12**, 1812–1824
17. Wang, X-Z., Harding, H. P., Zhang, Y., Jolicoeur, E. M., Kuroda, M. & Ron, D (1998) *EMBO J.* **17**, 5708–5717
18. Niwa, M., Sidrauski, C., Kaufman, R. J. & Walter, P. (1999) *Cell* **99**, 691–702
19. Sato, N., Urano, F., Yoon, Leem, Y. J., Kim, S. H., Li, M., Donoviel, D., Bernstein, A., Lee, A. S., Ron, D., Veselits, M. L., Sisodia, S. S. & Thinakaran, G. (2000) *Nat. Cell Biol.* **2**, 863–870
20. Yoshida, H., Haze, K., Yanagi, H., Yura, T. & Mori, K. (1998) *J. Biol. Chem.* **273**, 33741–33749
21. Haze, K., Yoshida, H., Yanagi, H., Yura, T. & Mori, K. (1999) *Mol. Biol. Cell* **10**, 3787–3799
22. Yoshida, H., Okada, T., Haze, K., Yanagi, H., Yura, T., Negishi, M. & Mori, K (2000) *Mol. Cell. Biol.* **20**, 6755–6767
23. Roy, B. & Lee, A. S. (1999) *Nucleic Acids Res.* **27**, 1437–1443
24. Wolfe, M. S., Xia, W., Ostaszewski, B. L., Diehl, T. S., Kimberly, W. T. & Selkoe, D. J. (1999) *Nature* **398**, 513–517
25. Harding, H. P., Zhang, Y. & Ron, D. (1999) *Nature.* **397**, 271–274
26. Harding, H. P., Zhang, Y., Bertolotti, A., Zeng, H. & Ron, D. (2000) *Mol. Cell* **5**, 897–904
27. Shen, J., Bronson, R. T., Chen, D. F., Xia, W., Selkoe, D. J. & Tonegawa, S. (1997) *Cell* **89**, 629–639
28. Wong, P. C., Zheng, H., Chen, H., Becher, M. W., Sirinathsinghji, D. J., Trumbauer, M. E., Chen, H. Y., Price, D. L., Van der Ploeg, L. H. & Sisodia, S. S. (1997) *Nature* **387**, 288–292
29. Herreman, A., Hartmann, D., Annaert, W., Saftig, P., Craessaerts, K., Serneels, L., Umans, L., Schrijvers, V., Checler, F., Vanderstichele, H., Baekelandt, V., Dressel, R., Cupers, P., Huylebroeck, D., Zwijsen, A., Van Leuven, F. & De Strooper, B. (1999) *Proc. Natl. Acad. Sci. U. S. A.* **96**, 11872–11877
30. Nakano, Y., Kondoh, G., Kudo, T., Imaizumi, K., Kato, M., Miyazaki, J. I., Tohyama, M., Takeda, J. & Takeda, M. (1999) *Eur. J. Neurosci.* **11**, 2577–2581
31. Wang, X. Z., Lawson, B., Brewer, J. W., Zinszner, H., Sanjay, A., Mi, L. J., Boorstein, R., Kreibich, G., Hendershot, L. M. & Ron, D. (1996) *Mol. Cell. Biol.* **16**, 4273–4280
32. Kaufman, R. J. (1999) *Genes Dev.* **13**, 1211–1233
33. Bertolotti, A., Zhang, Y., Hendershot, L. M., Harding, H. P. & Ron, D. (2000) *Nat. Cell Biol.* **2**, 326–332
34. De Strooper, B., Annaert, W., Cupers, P., Saftig, P., Craessaerts, K., Mumm, J. S., Schroeter, E. H., Schrijvers, V., Wolfe, M. S., Ray, W. J., Goate, A. & Kopan, R. (1999) *Nature* **398**, 518–522
35. Ye, J., Rawson, R. B., Komuro, R., Chen, X., Dave, U. P., Prywes, R., Brown, M. S. & Goldstein, J. L. (2000) *Mol. Cell* **6**, 1355–1364
36. Wang, Y., Shen, J., Arenzana, N., Tirasophon, W., Kaufman, R. J. & Prywes, R. (2000) *J. Biol. Chem.* **275**, 27013–27020
37. Liu, C. Y., Schroder, M. & Kaufman, R. J. (2000) *J. Biol. Chem.* **275**, 24881–24885
38. Imaizumi, K., Katayama, T. & Tohyama, M. (2001) *Nat. Cell Biol.* **3**, E104
