
THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2001 by The American Society for Biochemistry and Molecular Biology, Inc.
Vol. 276, No. 51, Issue of December 21, pp. 48189–48195, 2001
Printed in U.S.A.

# Biosynthesis of the Linkage Region of Glycosaminoglycans

## CLONING AND ACTIVITY OF GALACTOSYLTRANSFERASE II, THE SIXTH MEMBER OF THE β1,3-GALACTOSYLTRANSFERASE FAMILY (β3GalT6)*

Received for publication, August 1, 2001, and in revised form, September 6, 2001  
Published, JBC Papers in Press, September 10, 2001, DOI 10.1074/jbc.M107339200

Xiaomei Bai‡§¶, Dapeng Zhou¶, Jillian R. Brown‡§¶, Brett E. Crawford‡, Thierry Hennet∥,  
and Jeffrey D. Esko‡**

From the ‡Department of Cellular and Molecular Medicine, Glycobiology Research and Training Center,  
University of California, San Diego, La Jolla, California 92093-0687 and the ∥University of Zurich,  
Institute of Physiology, Winterthurerstrasse 190, Zurich CH-8057, Switzerland

A family of five β1,3-galactosyltransferases has been characterized that catalyze the formation of Galβ1, 3GlcNAcβ and Galβ1,3GalNAcβ linkages present in glycoproteins and glycolipids (β3GalT1, -2, -3, -4, and -5). We now report a new member of the family (β3GalT6), involved in glycosaminoglycan biosynthesis. The human and mouse genes were located on chromosomes 1p36.3 and 4E2, respectively, and homologs are found in *Drosophila melanogaster* and *Caenorhabditis elegans*. Unlike other members of the family, β3GalT6 showed a broad mRNA expression pattern by Northern blot analysis. Although a high degree of homology across several subdomains exists among other members of the β3-galactosyltransferase family, recombinant enzyme did not utilize glucosamine- or galactosamine-containing acceptors. Instead, the enzyme transferred galactose from UDP-galactose to acceptors containing a terminal β-linked galactose residue. This product, Galβ1,3Galβ is found in the linkage region of heparan sulfate and chondroitin sulfate (GlcAβ1,3Galβ1,4Xylβ-O-Ser), indicating that β3GalT6 is the so-called galactosyltransferase II involved in glycosaminoglycan biosynthesis. Its identity was confirmed *in vivo* by siRNA-mediated inhibition of glycosaminoglycan synthesis in HeLa S3 cells. Its localization in the medial Golgi indicates that this is the major site for assembly of the linkage region.

ducing end. Xylosyltransferase (2), galactosyltransferase I (GalTI)¹ (3, 4), and glucuronosyltransferase (GlcATI) (5, 6) have been cloned and partially characterized. In addition, mutant cell lines have been identified that contain defects in each of these enzymatic steps (7–9), and *C. elegans* mutants in GalTI (*sqv3*) and the GlcATI (*sqv8*) have been described, demonstrating the importance of these reactions and glycosaminoglycans in cellular processes and organismal development (10, 11). The enzyme that transfers the second galactose unit, galactosyltransferase II (GalTII) has not yet been identified. Previous studies have described the appropriate enzyme activity in tissue extracts (12), but the enzyme has not yet been purified or cloned. Presumably, GalTII should be distinct from GalTI based on linkage (β1,3 versus β1,4) and substrate differences (galactose versus xylose).

GalTII catalyzes the formation of Galβ1,3Gal linkage; therefore, it belongs to a family of β1,3-galactosyltransferases. Five members of this family have already been described (13–20), with a possible homolog in *Drosophila* (*Braniac*) (21). None of the β3GalT enzymes characterized to date have activity with galactose-terminated oligosaccharide acceptors, suggesting that another member of the family must exist in order to catalyze the formation of the linkage region of glycosaminoglycans.

In this report, we describe the identification of GalTII and show that it is the sixth member of confirmed β1,3-galactosyltransferases. This enzyme was actually identified in a previous publication but was designated a β1,3 N-acetylglucosaminyltransferase because of a clerical error (22). Here we demonstrate that the expressed protein catalyzes the formation of Galβ1,3Gal linkages. The protein sequence is a close homolog of other β1,3-galactosyltransferase, and orthologs are present in *Drosophila melanogaster* and *Caenorhabditis elegans* genomes, consistent with the idea that this enzyme is broadly distributed as might be expected for an enzyme involved in glycosaminoglycan biosynthesis.

---

**Glycosaminoglycan biosynthesis initiates by the formation of the linkage tetrasaccharide, GlcAβ1,3Galβ1,4Xylβ-O-Ser, which serves as a primer for the addition of alternating GlcNAc or GalNAc and glucuronic acid (1). The assembly process involves the sequential transfer of monosaccharide residues from the corresponding nucleotide sugars starting at the reducing end.**

---

* This work was funded in part by the Swiss National Science Foundation Grant 31-58577.99 (to T.H.) and National Institutes of Health Grant GM33063 (to J.D.E.). The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
* The nucleotide sequence(s) reported in this paper has been submitted to the GenBank™/EBI Data Bank with accession number(s) AY050569 and AY050570.
* § Present address: Neose West, 10901 North Torrey Pines Rd., Bldg. 7, Rm. 7102, La Jolla, CA 92037.
* ¶ These authors contributed equally to this work.
* ** To whom correspondence should be addressed: Dept. of Cellular and Molecular Medicine, Glycobiology Research and Training Center, University of California, San Diego, 9500 Gilman Dr., La Jolla, CA 92093-0687. Tel.: 858-822-1100; Fax: 858-534-5611; E-mail: jesko@ucsd.edu.

---

### MATERIALS AND METHODS

Cell Culture—Chinese hamster ovary cells (CHO-K1), COS-7, and HeLa S3 cells were obtained from the American Type Culture Collection.

---

¹ The abbreviations used are: GalTI, galactosyltransferase I; GalTII, UDP-galactose-galactose β1,3-galactosyltransferase; GlcATI, glucuronosyltransferase; β3GalT, a member of the β1,3-galactosyltransferase family; Bn, benzyl; BSA, bovine serum albumin; GFP, green fluorescent protein; NM, 2-naphthylmethyl; NP, nitrophenyl; PBS, phosphate-buffered saline; PCR, polymerase chain reaction; Xyl, xylosyltransferase; MES, 2-(N-morpholino)ethanesulfonic acid; GAG, glycosaminoglycan; siRNA, small interfering RNA.

This paper is available on line at http://www.jbc.org 48189

48190

# Biosynthesis of the Linkage Region of Glycosaminoglycans

GaTII 1 - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
Biosynthesis of the Linkage Region of Glycosaminoglycans

Product Identification—Twenty small scale enzymatic assays were performed in parallel for 4 h using Galβ1,4Xylβ-O-Bn as substrate and nonradioactive UDP-galactose as donor. After the clean up on a Sep-Pak C18 cartridge, the product was separated from the large amounts of unreacted disaccharide acceptor by silica gel chromatography using dichloromethane/methanol (10:3, v/v). Samples were analyzed on silica gel 60 aluminum-backed high performance thin layer chromatography plates (Merck). A section of the plate was stained with α-naphthol reagent and heated at 110 °C for 5 min to detect the product (26).

The disaccharide substrate and purified trisaccharide product were repeatedly exchanged in D₂O (100%; Sigma) with intermediate lyophilization. The trisaccharide was dissolved in a final volume of 40 μl of 100% D₂O and transferred into a 40-μl nanocell. The disaccharide standard was dissolved in a final volume of 0.6–0.7 ml and transferred to a 5-mm tube (Wilmad 528-PP). ¹H NMR experiments were carried out on a Varian Unity Inova 500-MHz spectrometer at 25 °C. A SUN Microsystems Ultra-10 computer running Varian’s VNMR software (version 6.1B) controlled data acquisition. Chemical shifts are relative to 4,4-dimethyl-4-silapentane-1-sulfonate; they were typically measured relative to the residual acetate peak (δ 1.908 ppm at 22–30 °C and pD 6–8) and compared with literature values (27–31).

Small Interfering RNA (siRNA) Inhibition of GalTII—Two small interfering RNAs (siRNA GalTII-A and -B) were designed to target GalTII based on the method of Elbashir and co-workers (32). siRNA GalTII-A targeted nucleotides 461–482 numbered from the start codon (AY050570, sense: 5′-GGCGGACGACGUCCUUUCTT-3′; antisense: 5′-GAAGGAGUCGUCGCUCCGCCT-3′). siRNA GalTII-B targeted nucleotides 65–87 (sense: 5′-GCACGCGACGCUGGCGCGCTT-3′; antisense: 5′-GCGCGCCAGCGUCGCGUGCTT-3′). As a control for nonspecific effects of siRNA, oligonucleotides were designed to target mouse lamin A (mLaminA) using nucleotides 260–280 (NM019390; sense: 5′-GAAGCAGCUUUCAGGAUGAGAU-3′; antisense: 5′-CUCAUCCUGAACUGCUCUU-3′). siRNAs were obtained from Genset Oligos (Paris), and the complementary oligonucleotides were annealed as described previously (32). For each 30-mm culture dish of cells, 12 μl of the 20 μM stock siRNA duplex was mixed with 200 μl of Opti-MEM (Life Technologies, Inc.). This mixture was gently added to a solution containing 12 μl of Oligofectamine (Life Technologies, Inc.) in 48 μl of Opti-MEM. After 20 min at room temperature, 128 μl of Opti-MEM was added. This solution was gently overlaid onto 10–20% confluent HeLa S3 cells, which had been previously washed with Opti-MEM. After 5 h, 2 ml of Dulbecco’s modified Eagle’s medium with 30% fetal calf serum was added without removing the transfection media. Four days later, the cells were treated with trypsin and reseeded in 30-mm dishes. Three rounds of siRNA transfection were required to decrease glycosaminoglycan biosynthesis as determined by ³⁵SO₄ incorporation (33).

Fluorescence Microscopy—A green fluorescent protein (GFP)-fused form of GalTII was prepared by amplifying the full-length cDNA from the original pcDNA3.1(+) clone. An EcoRI restriction site at the 5′-end (5′-GCAGGAATTCTAGCCACATTCCC-3′) and BamHI restriction site at the 3′-end (5′-CTGGATCCTCCCTTCTGGCAC-3′) were introduced. After digestion with the appropriate enzymes, the fragment was inserted in frame with GFP at the C terminus in pEGFP-N1. Cells grown on 24-well glass microscope slides were transfected with pEGFPN1-GalT II using LipofectAMINE (Life Technologies) in accordance with the manufacturer’s instructions. After 48 h, the cells were rinsed with phosphate-buffered saline (PBS) (34), fixed for 1 h with 2% paraformaldehyde in 75 mM phosphate buffer (pH 7.3), and permeabilized with 0.1% Triton-X in PBS containing 0.1% bovine serum albumin (BSA). Primary rabbit antibodies against α-mannosidase II and CAL-NUC (medial and cis Golgi markers, respectively) were a gift from M. G. Farquhar (University of California, San Diego). The antiserum was diluted 1:500 in PBS containing 1% BSA and incubated with fixed cells in a humid chamber for 1 h. After several washes, the cells were incubated with 4′,6′-diamidino-2-phenylindole to stain nuclei and a secondary antibody (Alexa Fluor 488 goat anti-rabbit IgG, Molecular Probes, Inc., Eugene, OR) diluted 1:200 in PBS containing 1% BSA. The coverslips were washed several times with PBS containing 0.1% BSA and mounted with Vectashield mounting medium (Vector Laboratories, CA). Images were captured with a Photometrics CCD mounted on a Nikon microscope adapted to a DeltaVision (Applied Precision, Inc.) deconvolution imaging system. The data sets were deconvolved and analyzed using SoftWorx software (Applied Precision, Inc.) on a Silicon Graphics Octane work station.

FIG. 2. A ClustalX alignment of human β3GalT6 proteins with the putative orthologs in murine (AY050569), *C. elegans* (AAF-60408), and *Drosophila melanogaster* (AAF59065).

| Human | MNLLRRRAWRRRAAALGLGLTALAACGAALL |
| --- | --- |
| Murine | MKVFRRAWRHRVALGLGLLAFCGTLLL |
| C.elegans | ----MRFYRTYLVAAGAFCSLCTLAVI |
| Drosophila | ----MBRLNNLVTFFTAITAFFFGSFL |

| Human | Y-LARCAAEEPGDPRAMSQRSPPPAPA |
| --- | --- |
| Murine | Y-LARCASEGETPSASGAAR---P--- |
| C.elegans | FNCGWDDSPPATPSAINGGGSNAPLIS |
| Drosophila | T--KILSSVDQCPAHRSRIPHELP--- |

| Human | ----RAAAFLAVLASAPRRAEERRSVI |
| --- | --- |
| Murine | ----RAKFLAVLASAPRAVERTTAV |
| C.elegans | SPTNLPETFLYISTLTSPNETERRQNV |
| Drosophila | ----HPNLFLMLVLSAAPHNADERNAM |

| Human | RSTWLA---- |
| --- | --- |
| Murine | RSTWLAP---- |
| C.elegans | RDTWFR-L--- |
| Drosophila | RRITWLANAGQSIAQPYLPEELIYLPTF |

| Human | ---- |
| --- | --- |
| Murine | ---- |
| C.elegans | ---- |
| Drosophila | NAQGHLQVELVAEQASRLRQYTNWQQS |

| Human | -RRGAPG-----DVWARFAVGTAAGLG |
| --- | --- |
| Murine | -RRGGPE-----DVWARFAVGTGGLG |
| C.elegans | -STKGPS-----VFIAKFAVGTMGLA |
| Drosophila | LLTEGPPRTKRLITVKHVFSIGTLDLS |

| Human | AEERRALEREQA RHGDLLLLPAMRDAY |
| --- | --- |
| Murine | SEERRALELEQA QHGDLLLLPALRDAY |
| C.elegans | AEORLLLAEEEN EKFGDLALLDRHEESY |
| Drosophila | SSALAELEKEQN QNDLLLNRHHDITY |

| Human | ENLTAKV LAMLAWLDEHVAFEFMLKAD |
| --- | --- |
| Murine | MLTAKV LAMLTWLDERVDFFVMLKAD |
| C.elegans | RLAKKT LAACFVHAFAFNFKFKFLKTD |
| Drosophila | KNLIAKLMQSQSYIRRHYEFSYMLKVVD |

| Human | DDSFARLDA LLAELAREPARRR |
| --- | --- |
| Murine | DDSFARLDA TLVDLRAREPARRR |
| C.elegans | DDSFVRTTP LIINLKQIQDP--- |
| Drosophila | DDITYVK LDSLVNTLVSYDRKLLKRSE |

| Human | ----RLYWGFFSGRRGRVKPGGRRR |
| --- | --- |
| Murine | ----RLYWGFESGRRGRVKPGGRRR |
| C.elegans | ----MLYWGFLDGRAKPFRKGKWK |
| Drosophila | YRDHVLPOLYWGYFNNGRSTIKTKGQWK |

| Human | EAAMQLCDYYYPYALGGGYYLSADLVH |
| --- | --- |
| Murine | EAAWQLCDYYYPYALGGGYYLSADLVH |
| C.elegans | EPEWNLCDRYLPYOLGGGYYLSYELIR |
| Drosophila | ESSYYLSKNYLPYALGGGYYLSRSLCD |

| Human | YLRLSRDYLR AWHSEDVS LGAWLAPV- |
| --- | --- |
| Murine | YLRLSREYLR AWHSEDVSL GTWLAPV- |
| C.elegans | FLAINAQLFR HYRNEDVSVG AWIGGL- |
| Drosophila | YIVNNSQLCS HYGSEDVSVG TWLAPLR |

| Human | DVQREFHDRF DTEYRSRGCSNQYLVT |
| --- | --- |
| Murine | DVQREFHDRF DTEYKSRCGN NQYLVT |
| C.elegans | DVKYVHDRFD TEWRSSRGCN NEYLITH |
| Drosophila | HMVYRWHDRF DTSMAPRKCR SYHMVLH |

| Human | KQSLEDMLEKHATLAREGRCLKREVQL |
| --- | --- |
| Murine | KQSPEDMLEK QQMLLHEGBRLCKHEVQL |
| C.elegans | HTEQEMQEMF ENLKKTGKLCAKEFQK |
| Drosophila | RNGQMMRD---IHDGELOSGIGSS |

| Human | RLS-YVYDWSAPP SQCCQRREG-IP |
| --- | --- |
| Murine | RLS-YVYDWSAPP SQCCQRKEG-VP |
| C.elegans | HPS-YVYDFS KAPSECCTRBNGSNP |

FIG. 3. Expression pattern of the GalTII gene in adult human tissues determined by Northern blot analysis. Each lane represents about 2 μg of poly(A)⁺ RNA. The sizes of the RNA markers are indicated in kilobases to the left of the blot.

cells, the standard reaction (25 μl) contained 5 μl of protein A bead slurry (50%) containing immobilized enzyme, 50 mM MES, pH 6.0, 0.3 μCi of UDP-[1-³H]galactose (PerkinElmer Life Sciences), 150 μM UDP-galactose, 5 mM acceptor, 15 mM MnCl₂, and 50 mM KCl. After incubation at 37 °C for 1 h, the reaction products were diluted with 1 ml of 0.5 M NaCl and applied to a Sep-Pak C18 cartridge (100 mg; Waters). After washing the cartridge with 5 ml of water, the products were eluted with 50% methanol, dried, and counted by liquid scintillation.

48192

**Biosynthesis of the Linkage Region of Glycosaminoglycans**

FIG. 4. Phylogenetic trees. Unrooted phylogenetic trees were produced with phylogenetic analysis using Parsimony version 4.0b8, based on ClustalX multiple amino acid sequence alignment analysis. A, dendrogram using full coding sequence of all β3GalT genes. B, dendrogram using nucleotide sequences.

A  
GalT2  
GalT1  
GalT4  
C.elegans  
Drosophila  
Murine  
Human  

B  
GalT2  
GalT1  
GalT4  
GalT5  
C.elegans  
Murine  
Human  
Drosophila  

---  
50 changes  
100 changes  

---

**RESULTS AND DISCUSSION**

A large family of β1,3-galactosyltransferases has been described based on expressed sequence tags exhibiting a high degree of homology and by use of PCR-based cloning strategies (13–20). β3GalT1, -2, -3, and -5 utilize GlcNAc-terminated oligosaccharide as acceptor substrates, producing linkages characteristic of type 1 N-acetyllactosamine repeat units on N- and O-linked oligosaccharides and glycolipids, whereas β3GalT4 transfers galactose to the terminal GalNAc unit of the ganglioside series GM2, GD2, and GA2 acceptors (35). In addition to GlcNAcβ-based acceptors, β3GalT5 is also capable of transferring galactose to the terminal GalNAc unit of the globoside Gb4 (20). In a search for additional family members, we used the mouse β3GalT1, -2, and -3 (15) protein sequences to query the expressed sequence tag division of GenBank™, and the fragment AA150140 was identified. This fragment was used as a probe to isolate the corresponding full-length cDNAs from human and mouse libraries, which corresponded to open reading frames encoding proteins of 329 and 325 amino acids, respectively. This cDNA was previously associated with a β1,3-N-acetylglucosaminyltransferase activity (22), but the actual sequence responsible for the glucosaminyltransferase activity reported previously is encoded by AF092050 and AF092051. These correct designations have been posted in the GenBank™ data base. As shown below the cDNA actually encodes a β1,3-galactosyltransferase that participates in the formation of the linkage region of glycosaminoglycans. As the newest confirmed member of the β3GalT family, it is designated β3GalT6. However, in this paper it will be referred to as GAG GalTII after the nomenclature used in the proteoglycan field (1).

As shown in Fig. 1, GAG GalTII shows extensive homology to four other members of the human β1,3-galactosyltransferase family that have confirmed preferences for GlcNAc-terminated oligosaccharides. Like the other β1,3-galactosyltransferases, GAG GalTII has a typical type II transmembrane orientation. The tentative transmembrane domain consists of 19 amino acids starting at residue 12 from the N terminus. In the transmembrane domain, the GAG GalTII has a cysteine residue that is conserved in mouse, human, and C. elegans (Fig. 2). Interestingly, β3GalT1, β3GalT5, and the homologous Drosophila protein Brainiac also contain Cys residues in their putative transmembrane domains, but the function of these residues is unknown. One possibility is that they may affect the oligomerization of the protein as recently reported for α2,6-sialyltransferase I (36). The transmembrane domain is followed by a segment rich in Ser, Pro, Ala, and Gly, which has been described as the “SPLAG” domain in other transferases (37), and presumably represents a stem region that determines the distance between the catalytic domain and the membrane. All of the other motifs identified previously in the β3GalTs presumably define essential folds in the catalytic domain (13, 15, 22). These were also found in similar locations in GAG GalTII, but the conserved cysteine residues are either absent or located differently.

The chromosomal location of human and mouse GAG GalTII was determined using both the public genome data base and the partially annotated mouse genome. Human GalTII maps to chromosome 1p36.3 and occupies a similar position on the mouse chromosome at 4E2. Inspection of the sequence indicates that GalTII, like the other members of the β3GalTs, is encoded by a single exon in the human, but its organization in the mouse genome is unclear from the existing annotated data base. However, the gene appears to contain multiple exons in *D. melanogaster* and *C. elegans*. GalTII mRNA is expressed broadly across both human and mouse tissues (Fig. 3 and Ref. 22), revealing three different transcripts of ~1.6, 2.4, and 3.3 kilobases. This pattern differs from the expression profiles of the other β3GalTs, which tend to be restricted to specific tissues (13, 15, 16, 18, 19). The presence of multiple transcripts suggests that additional exons exist encoding 5′- and 3′-untranslated regions.

Differences in nucleotide and amino acid sequences were used to measure the evolutionary relationship of GalTII and the other members of the β3GalT family (Fig. 4). Both dendrograms indicated a “starburst” relationship among the human β3GalT genes; i.e. all members of the family are relatively equidistant from each other (the internodal distances have high bootstrap support). This observation suggests that the β3GalT family emerged early in evolution and has not undergone recent duplications and radiations. Human GalTII has

Biosynthesis of the Linkage Region of Glycosaminoglycans

TABLE II  
Chemical shift and coupling constants for diagnostic protons of the enzymatic product of GalTII  

| Residue | Proton | Chemical shift (pm) and coupling constant ($^3J_{vic}$) |
|---------|--------|-------------------------------------------------------|
|         |        | Galβ1,4Xylβ-O-Bn                                      |
| Xyl-1   | H-1    | 4.468, $^3J_{1,2}$ 8.0                               |
|         | H-5eq  | 4.107, $^3J_{4,5eq}$ 5.0                             |
| Gal-2   | H-1    | 4.515, $^3J_{1,2}$ 8.0                               |
|         | H-4    | 3.910, $^3J_{3,4}$ 3.0                               |
| Gal-3   | H-1    | 4.605, $^3J_{1,2}$ 8.0                               |
|         | H-4    | 3.919, $^3J_{3,4}$ 3.0                               |

|         |        | Galβ1,3Galβ1,4Xylβ-O-Bn                              |
|---------|--------|-------------------------------------------------------|
| Xyl-1   | H-1    | 4.519, $^3J_{1,2}$ 8.0                               |
|         | H-5eq  | 4.113, $^3J_{4,5eq}$ 5.5                             |
| Gal-2   | H-1    | 4.530, $^3J_{1,2}$ 8.0                               |
|         | H-4    | 4.189, $^3J_{3,4}$ 3.0                               |
| Gal-3   | H-1    | 4.615, $^3J_{1,2}$ 8.0                               |
|         | H-4    | 3.927, $^3J_{3,4}$ 3.4                               |

|         |        | Galβ1,3Galβ1,4Xylβ-O-Ser$^a$                         |
|---------|--------|-------------------------------------------------------|
| Xyl-1   | H-1    | 4.470, $^3J_{1,2}$ 7.2                               |
|         | H-5eq  | 4.124, $^3J_{4,5eq}$ 5.3                             |
| Gal-2   | H-1    | 4.534, $^3J_{1,2}$ 7.8                               |
|         | H-4    | 4.194, $^3J_{3,4}$ 3.4                               |
| Gal-3   | H-1    | 4.615, $^3J_{1,2}$ 7.8                               |
|         | H-4    | 3.927, $^3J_{3,4}$ 3.4                               |

$a$ The analytical data for Galβ1,3Galβ1,4Xylβ-O-Ser are from Refs. 27–31.

FIG. 6. ${}^{1}\mathrm{H}$ NMR spectrum of synthetic acceptor, Galβ1,4Xylβ-O-Bn and the enzymatic product, Gal β1,3Galβ1,4Xylβ-O-Bn, generated using recombinant GalTII (see "Materials and Methods"). The trisaccharide shown in A, Galβ1,3Galβ1,4Xylβ-O-Bn, was compared with the chemical shifts for the disaccharide starting material shown in B, Galβ1,4Xylβ-O-Bn, as well as the literature compound, Galβ1,3 Galβ1,4Xylβ-Ser.

Conclusions—In this report, we have identified the cDNA for GAG GalTII and report that it is the previously described β1,3-N-acetylgalactosaminyltransferase (22). This enzyme is part of a gene family that encodes at least six β1,3-galactosyltransferases involved in glycoprotein, glycolipid, and proteoglycan processing. These enzymes in fact contain a motif in common with other β1,3-glycosyltransferases. One of the enzymes, β3GalT3, has been shown to also transfer GalNAc in β1,3-linkage to globosides (42), and four structurally related β1,3-N-acetylgalactosaminyltransferases have been recently cloned based on sequence homology to β3GalT1 (43–45). In contrast to these enzymes, GAG GalTII (β3GalT6) preferentially acts on Galβ1,4Xyl, which is found in the linkage region of glycosaminoglycans. Furthermore, silencing of the gene by siRNA blocked glycosaminoglycan assembly *in vivo*, verifying the identity of this cDNA as GAG GalTII. Its localization to the medial Golgi along with GalTI and GlcATI is consistent with the idea that linkage region formation takes place in this compartment.

Acknowledgments—We thank Herman Van Halbeek and the Glycotechnology Core of the Glycobiology Research and Training Center at UCSD for help with NMR. We also thank Sara Olson for help with consensus alignment.

REFERENCES

1. Esko, J. D., and Lindahl, U. (2001) *J. Clin. Invest.* 108, 169–173
2. Götting, C., Kuhn, J., Zahn, R., Brinkmann, T., and Kleesiek, K. (2000) *J. Mol. Biol.* 304, 517–528
3. Okajima, T., Yoshida, K., Kondo, T., and Furukawa, K. (1999) *J. Biol. Chem.* 274, 22915–22918
4. Almeida, R., Levery, S. B., Mandel, U., Kresse, H., Schwientek, T., Bennett, E. P., and Clausen, H. (1999) *J. Biol. Chem.* 274, 26165–26171
5. Kitagawa, H., Tone, Y., Tamura, J., Neumann, K. W., Ogawa, T., Oka, S., Kawasaki, T., and Sugahara, K. (1998) *J. Biol. Chem.* 273, 6615–6618
6. Wei, G., Bai, X. M., Sarkar, A. K., and Esko, J. D. (1999) *J. Biol. Chem.* 274, 7857–7864
7. Esko, J. D., Stewart, T. E., and Taylor, W. H. (1985) *Proc. Natl. Acad. Sci. U. S. A.* 82, 3197–3201
8. Esko, J. D., Weinke, J. L., Taylor, W. H., Ekborg, G., Rodén, L., Anantharamaiah, G., and Gawish, A. (1987) *J. Biol. Chem.* 262, 12189–12195
9. Bai, X. M., Wei, G., Sinha, A., and Esko, J. D. (1999) *J. Biol. Chem.* 274, 13017–13024
10. Herman, T., and Horvitz, H. R. (1999) *Proc. Natl. Acad. Sci. U. S. A.* 96, 974–979
11. Bulik, D. A., Wei, G., Toyoda, H., Kinoshita-Toyoda, A., Waldrip, W. R., Esko, J. D., Robbins, P. W., and Selleck, S. B. (2000) *Proc. Natl. Acad. Sci. U. S. A.* 97, 10838–10843
12. Helting, T., and Rodén, L. (1969) *J. Biol. Chem.* 244, 2790–2798
13. Amado, M., Almeida, R., Carneiro, F., Levery, S. B., Holmes, E. H., Nomoto, M., Hollingsworth, M. A., Hassan, H., Schwientek, T., Nielsen, P. A., Bennett, E. P., and Clausen, H. (1998) *J. Biol. Chem.* 273, 12770–12778
14. Amado, M., Almeida, R., Schwientek, T., and Clausen, H. (1999) *Biochim. Biophys. Acta Gen. Subj.* 1473, 35–53
15. Hennet, T., Dinter, A., Kuhnert, P., Mattu, T. S., Rudd, P. M., and Berger, E. G. (1998) *J. Biol. Chem.* 273, 58–65
16. Kolbinger, F., Streiff, M. B., and Katopodis, A. G. (1998) *J. Biol. Chem.* 273, 433–440
17. Salvini, R., Bardoni, A., Valli, M., and Trinchera, M. (2001) *J. Biol. Chem.* 276, 3564–3573
18. Zhou, D., Berger, E. G., and Hennet, T. (1999) *Eur. J. Biochem.* 263, 571–576
19. Isshiki, S., Togayachi, A., Kudo, T., Nishihara, S., Watanabe, M., Kubota, T., Kitajima, M., Shiraishi, N., Sasaki, K., Andoh, T., and Narimatsu, H. (1999) *J. Biol. Chem.* 274, 12499–12507
20. Zhou, D., Henion, T. R., Jungalwala, F. B., Berger, E. G., and Hennet, T. (2000) *J. Biol. Chem.* 275, 22631–22634
21. Yuan, Y. P., Schultz, J., Mlodzik, M., and Bork, P. (1997) *Cell* 88, 9–11
22. Zhou, D., Dinter, A., Gutierrez Gallego, R., Kamerling, J. P., Vliegenthart, J. F., Berger, E. G., and Hennet, T. (1999) *Proc. Natl. Acad. Sci. U. S. A.* 96, 406–411; Correction (2000) *Proc. Natl. Acad. Sci. U. S. A.* 97, 11673–11675
23. Sarkar, A. K., Brown, J. R., and Esko, J. D. (2000) *Carbohydr. Res.* 329, 287–300
24. Lugemwa, F. N., Sarkar, A. K., and Esko, J. D. (1996) *J. Biol. Chem.* 271, 19159–19165
25. Brown, J. R., Field, R. A., Barker, A., Guy, M., Grewal, R., Khoo, K. H., Brennan, P. J., Besra, G. S., and Chatterjee, D. (2001) *Bioorg. Med. Chem.* 9, 815–824
26. Siakotos, A. N. (1965) *J. Am. Oil Chem. Soc.* 42, 913–919
27. Wieruszewski, J. M., Michalski, J. C., Montreuil, J., Strecker, G., Peter-Katalinic, J., Egge, H., van Halbeek, H., Mutsaers, J. H., and Vliegenthart, J. F. (1987) *J. Biol. Chem.* 262, 6650–6657
28. Van Halbeek, H., Dorland, L., Veldink, G. A., Vliegenthart, J. F., Garegg, P. J., Norberg, T., and Lindberg, B. (1982) *Eur. J. Biochem.* 127, 1–6
29. Krishna, N. R., Choe, B.-Y., Prabhakaran, M., Ekborg, G. C., Rodén, L., and


The OCR conversion is complete. Let me know if you need further assistance!
Harvey, S. C. (1990) *J. Biol. Chem.* **265**, 18256–18262

30. Agrawal, P. K., Jacquinet, J. C., and Krishna, N. R. (1999) *Glycobiology* **9**, 669–677

31. Krishna, N. R., and Agrawal, P. K. (2000) *Adv. Carbohydr. Chem. Biochem.* **56**, 201–234

32. Elbashir, S. M., Harborth, J., Lendeckel, W., Yalcin, A., Weber, K., and Tuschl, T. (2000) *Nature* **411**, 494–498

33. Bame, K. J., and Esko, J. D. (1989) *J. Biol. Chem.* **264**, 8059–8065

34. Dulbecco, R., and Vogt, M. (1954) *J. Exp. Med.* **99**, 167–182

35. Miyazaki, H., Fukumoto, S., Okada, M., Hasegawa, T., and Furukawa, K. (1997) *J. Biol. Chem.* **272**, 24794–24799

36. Qian, R., Chen, C., and Colley, K. J. (2001) *J. Biol. Chem.* **276**, 28641–28649

37. Shworak, N. W., Liu, J. A., Petros, L. M., Zhang, L. J., Kobayashi, M., Copeland, N. G., Jenkins, N. A., and Rosenberg, R. D. (1999) *J. Biol. Chem.* **274**, 5170–5184

38. Velasco, A., Hendricks, L., Moremen, K. W., Tulsiani, D. R., Touster, O., and Farquhar, M. G. (1993) *J. Cell Biol.* **122**, 39–51

39. Lin, P., Le-Niculescu, H., Hofmeister, R., McCaffery, J. M., Jin, M., Hennemann, H., McQuistan, T., De Vries, L., and Farquhar, M. G. (1998) *J. Cell Biol.* **141**, 1515–1527

40. Pinhal, M. A. S., Smith, B., Olson, S. K., Aikawa, J., Kimata, K., and Esko, J. D. (2001) *Proc. Natl. Acad. Sci. U. S. A.* **98**, 12984–12989

41. Vertel, B. M., Walters, L. M., Flay, N., Kearns, A. E., and Schwartz, N. B. (1993) *J. Biol. Chem.* **268**, 11105–11112

42. Okajima, T., Nakamura, Y., Uchikawa, M., Haslam, D. B., Numata, S. I., Furukawa, K., Urano, T., and Furukawa, K. (2000) *J. Biol. Chem.* **275**, 40498–40503

43. Shiraishi, N., Natsume, A., Togayachi, A., Endo, T., Akashima, T., Yamada, Y., Imai, N., Nakagawa, S., Koizumi, S., Sekine, S., Narimatsu, H., and Sasaki, K. (2001) *J. Biol. Chem.* **276**, 3498–3507

44. Togayachi, A., Akashima, T., Ookubo, R., Kudo, T., Nishihara, S., Iwasaki, H., Natsume, A., Mio, H., Inokuchi, J., Irimura, T., Sasaki, K., and Narimatsu, H. (2001) *J. Biol. Chem.* **276**, 22032–22040

45. Henion, T. R., Zhou, D., Wolfer, D. P., Jungalwala, F. B., and Hennet, T. (2001) *J. Biol. Chem.* **276**, 30261–30269

Biosynthesis of the Linkage Region of Glycosaminoglycans: CLONING AND
ACTIVITY OF GALACTOSYLTRANSFERASE II, THE SIXTH MEMBER OF
THE β1,3-GALACTOSYLTRANSFERASE FAMILY (β3GalT6)
Xiaomei Bai, Dapeng Zhou, Jillian R. Brown, Brett E. Crawford, Thierry Henriet and
Jeffrey D. Esko

J. Biol. Chem. 2001, 276:48189-48195.
doi:10.1074/jbc.M107339200 originally published online September 10, 2001

---

Access the most updated version of this article at doi: [10.1074/jbc.M107339200](https://doi.org/10.1074/jbc.M107339200)

Alerts:
- When this article is cited
- When a correction for this article is posted

[Click here to choose from all of JBC's e-mail alerts](#)

This article cites 45 references, 33 of which can be accessed free at
[http://www.jbc.org/content/276/51/48189.full.html#ref-list-1](http://www.jbc.org/content/276/51/48189.full.html#ref-list-1)
