
FEMS Microbiology Letters 203 (2001) 131–139

MiniReview

**Ubiquinone biosynthesis in microorganisms**

R. Meganathan*

Department of Biological Sciences, Northern Illinois University, DeKalb, IL 60115, USA

Received 22 May 2001; received in revised form 3 July 2001; accepted 3 July 2001

First published online 2 August 2001

---

### Abstract

The quinoid nucleus of the benzoquinone, ubiquinone (coenzyme Q; Q), is derived from the shikimate pathway in bacteria and eukaryotic microorganisms. Ubiquinone is not considered a vitamin since mammals synthesize it from the essential amino acid tyrosine. *Escherichia coli* and other Gram-negative bacteria derive the 4-hydroxybenzoate required for the biosynthesis of Q directly from chorismate. The yeast, *Saccharomyces cerevisiae*, can either form 4-hydroxybenzoate from chorismate or tyrosine. However, unlike mammals, *S. cerevisiae* synthesizes tyrosine in vivo by the shikimate pathway. While the reactions of the pathway leading from 4-hydroxybenzoate to Q are the same in both organisms, the order in which they occur differs. The 4-hydroxybenzoate undergoes a prenylation, a decarboxylation, and three hydroxylations alternating with three methylation reactions, resulting in the formation of Q. The methyl groups for the methylation reactions are derived from S-adenosylmethionine. While the prenyl side chain is formed by the 2-C-methyl-D-erythritol 4-phosphate (non-mevalonate) pathway in *E. coli*, it is formed by the mevalonate pathway in the yeast. © 2001 Federation of European Microbiological Societies. Published by Elsevier Science B.V. All rights reserved.

Keywords: Ubiquinone; Coenzyme Q; Q biosynthesis; Isoprenoid; Mevalonate; Non-mevalonate

---

### 1. Introduction

The isoprenoid quinones of the benzene and naphthalene series are widespread in microorganisms. Aerobic Gram-negative bacteria and eukaryotes contain the benzoquinone, ubiquinone, as the sole quinone, while the facultative anaerobic bacteria such as *Escherichia coli* contain the naphthoquinones, demethylmenaquinone (DMK) and menaquinone (MK) in addition to ubiquinone [1]. The archaea, as a group, lack ubiquinone. In bacteria and eukaryotes, including the fungi, ubiquinone functions as an electron carrier in the aerobic respiratory chain.

This MiniReview is concerned with the biosynthesis of the benzoquinone, ubiquinone (coenzyme Q). According to the IUPAC commission on nomenclature, both names are recognized. In this MiniReview and in biosynthetic studies, the newer ubiquinone is generally used, while in biomedical studies, coenzyme Q is used. Abbreviations are Q or Q-n where 'n' refers to the number of prenyl units present in the side chain (Fig. 1). *E. coli* contains Q-8 (n = 8, a 40 carbon isoprenoid side chain) as the predominant quinone with minor amounts of Q-1 to Q-7 and Q-9 [1]. The predominant side chain length is a constant depending on the species. Thus, for example, *Gluconobacter suboxydans*, *Rhodobacter capsulatus*, *E. coli*, and the yeast, *Saccharomyces cerevisiae* have side chain lengths of n = 10, 9, 8, and 6, respectively.

### 2. Structure and precursors

The structure of ubiquinone (4) is shown in Fig. 1. The quinoid nucleus is derived from the shikimate pathway via chorismate (1) in bacteria or tyrosine (2) in higher eukaryotes [2–5]. The eukaryotic microorganism, *S. cerevisiae*, can synthesize Q from either chorismate (1) or tyrosine (2) [4,5]. While in yeast and probably other fungi, the tyrosine is derived from chorismate in vivo, it has to be provided as an essential amino acid for mammals, which lack the shikimate pathway and, hence, Q is not considered a vitamin. The methyl groups on the benzene ring are derived from S-adenosylmethionine (SAM). In eukaryotes, the isopentenyl diphosphate required for the formation of the polyisoprenyl side chain is derived from acetate via the mevalonate pathway. In bacteria, it is formed by the newly discovered 2-C-methyl-D-erythritol 4-phosphate (MEP)

* Tel.: +1-(815) 753 7803; Fax: +1-(815) 753 0461.
* E-mail address: rmeganathan@niu.edu (R. Meganathan).

0378-1097/01/$20.00 © 2001 Federation of European Microbiological Societies. Published by Elsevier Science B.V. All rights reserved.
PII: S0378-1097(01)00330-5

FEMSLE 10080 21-9-01

R. Meganathan / FEMS Microbiology Letters 203 (2001) 131–139

**Bacteria & Yeast**

![Diagram](attachment:diagram.png)

Fig. 1. Biosynthetic precursors of ubiquinone. 1, chorismate; 2, tyrosine; 3, 4-hydroxybenzoate; 4, ubiquinone; SAM, S-adenosylmethionine; IPP, isopentenyl diphosphate; DMAPP, dimethylallyl diphosphate; MEP, 2-C-methyl-d-erythritol 4-phosphate.

specialized and address specific aspects of Q biosynthesis. For example, there are separate reviews covering the biosynthesis of the quinoid ring from chorismate in *E. coli*, the reaction mechanisms for the various enzymes, the biosynthesis of isopentenyl diphosphate by the mevalonate pathway in eukaryotes and the MEP pathway in bacteria and the path of assembly of the polyisoprenyl side chain. This MiniReview brings together all these pathways succinctly so as to provide a guide to further in depth reading.

The biosynthesis of the 'quinoid' head of Q is first discussed. This is followed by a description of the two pathways for the biosynthesis of dimethylallyl diphosphate (DMAPP) and/or isopentenyl diphosphate (IPP), and finally, the conversion of IPP into the side chain ('tail') precursor, polyprenyl diphosphate (PPP).

### 3. Biosynthesis of the quinoid ring

The biosynthesis of the quinoid ring and the various ring modification reactions in *E. coli* have been reviewed [8,9]. A mechanistic perspective on the various reactions has been provided [10,11].

#### 3.1. Biosynthesis of 4-hydroxybenzoate in *E. coli*

The formation of 4-hydroxybenzoate (3) from chorismate (1) is the first committed step in the biosynthesis of Q. This aromatizing reaction is catalyzed by the enzyme chorismate pyruvate-lyase encoded by the *ubiC* gene. Mutants blocked in this conversion have been isolated and characterized. The *ubiC* mutants are deficient in the formation of Q and are characterized by their inability to grow aerobically on oxidizable substrates such as succinate. The *ubiC* gene has been overexpressed, and the enzyme purified to homogeneity and characterized.

#### 3.2. Biosynthesis of 4-hydroxybenzoate in yeast

Yeast seems to produce 4-hydroxybenzoate by two different ways. It may be produced directly from chorismate by the chorismate pyruvate-lyase reaction similar to *E. coli*, or alternately from tyrosine, similar to higher eukaryotes. Consistent with the existence of two different routes is the fact that yeast mutants blocked in the formation of 4-hydroxybenzoate have never been isolated. Evidence for the presence of the two alternate routes was obtained using shikimate pathway mutants. Mutants blocked in the formation of shikimate or chorismate are expected to be deficient in the formation of Q, due to their inability to form 4-hydroxybenzoate. Addition of tyrosine to the growth medium restored the ability of these mutants to form 4-hydroxybenzoate and Q. It was further shown that wild-type yeast normally uses the conversion of chorismate (1) to 4-hydroxybenzoate (3) as the source of precursor for Q. However, in the shikimate pathway mutants,

R. Meganathan / FEMS Microbiology Letters 203 (2001) 131–139

can use a variety of prenyl diphosphates as side chain precursors. The lack of specificity of the enzyme also extends to the aromatic substrate. Thus, the enzyme tolerates substitutions by various groups at different positions on the benzene ring of Q.

The length of the prenyl side chain is a constant for each organism. This constancy of length is determined by the availability of the prenyl diphosphate in the cell (see Sections 7 and 8).

### 4. Reactions of ring modification in *E. coli*

#### 4.1. Prenylation

The prenylation of 4-hydroxybenzoate (3) to 3-octaprenyl-4-hydroxybenzoate (11) is carried out by the membrane-bound enzyme 4-hydroxybenzoate octaprenyltransferase in *E. coli* (Fig. 3). The enzyme is non-specific and can use a variety of prenyl diphosphates as side chain precursors. The lack of specificity of the enzyme also extends to the aromatic substrate. Thus, the enzyme tolerates substitutions by various groups at different positions on the benzene ring of Q.

The length of the prenyl side chain is a constant for each organism. This constancy of length is determined by the availability of the prenyl diphosphate in the cell (see Sections 7 and 8).

#### 4.2. Decarboxylation

The 3-octaprenyl-4-hydroxybenzoate (11) is decarboxylated to 2-octaprenylphenol (12) by the enzyme 3-octaprenyl-4-hydroxybenzoate decarboxylase. In *Salmonella enterica* and *E. coli*, two genes designated as *ubiD* and *ubiX* encode the two decarboxylases carrying out this reaction. The open reading frames (*orf*) encoding these two enzymes have been recently identified [11,14–16].

#### 4.3. Hydroxylations and methylations

The 2-octaprenylphenol (12) undergoes three hydroxylations alternating with three methylation reactions resulting in the formation of ubiquinol (20) and then Q (4). For convenience, the hydroxylations are considered together followed by the methylation reactions.

##### 4.3.1. Hydroxylations

Three monooxygenases are involved in the introduction of three hydroxyl groups at positions C-6, C-4, and C-5 of the benzene ring, respectively.

The three reactions are:

1. 2-octaprenylphenol (12) → 2-octaprenyl-6-hydroxyphenol (13);
2. 2-octaprenyl-6-methoxyphenol (16) → 2-octaprenyl-6-methoxy-1,4-benzoquinol (17);
3. 2-octaprenyl-3-methyl-6-methoxy-1,4-benzoquinol (18) → 2-octaprenyl-3-methyl-5-hydroxy-6-methoxy-1,4-benzoquinol (19).

Mutants deficient in each of the hydroxylations have been isolated and designated as *ubiB* (12 → 13), *ubiH* (16 → 17), and *ubiF* (18 → 19), respectively. The open reading frames encoding these reactions have been identified [11,17,18]. These mutants accumulate the intermediate before the block, namely (12), (16), (18) respectively.

When wild-type *E. coli* is grown anaerobically on the oxidizable substrate glycerol with fumarate as an electron acceptor, it forms considerable quantities of Q (50–70% of aerobically grown cells). Mutants blocked in the non-hydroxylating reactions of the pathway, such as *ubiA* (3 → 11), *ubiD* (11 → 12), and *ubiE* (17 → 18), are Q-deficient under aerobic as well as anaerobic conditions, thus establishing that the same genes and enzymes are involved

Fig. 2. Proposed pathway for the biosynthesis of 4-hydroxybenzoate from tyrosine. 2, tyrosine; 5, 4-hydroxyphenylpyruvate; 6, 4-hydroxyphenyllactate; 7, 4-hydroxycinnamate or *p*-coumarate; 8, *p*-coumaroyl-CoA; 9, 4-hydroxybenzoyl-CoA; 3, 4-hydroxybenzoate.

tyrosine (2) is able to compensate fully by providing the 4-hydroxybenzoate (3) required for Q biosynthesis [4,5]. It seems likely that many of the lower eukaryotes that have the shikimate pathway may contain dual pathways for the biosynthesis of 4-hydroxybenzoate (3). In animal cells, 4-hydroxybenzoate (3) is formed from the essential amino acid tyrosine (2). A pathway for the conversion of tyrosine (2) to 4-hydroxybenzoate (3) was proposed by Booth et al. [12] following urinary excretion studies on animals administered with various phenolic acids. Based on these studies and on the incorporation of radiolabeled phenolic acids into Q by liver and yeast, the following pathway was proposed (Fig. 2). Tyrosine → 4-hydroxyphenylpyruvate → 4-hydroxyphenyllactate → 4-hydroxycinnamate → 4-hydroxybenzoate. The in vitro conversion of tyrosine (2) to 4-hydroxyphenylpyruvate (5) and its subsequent reduction to 4-hydroxyphenyllactate (6) has been shown in extracts of rat liver and yeast. Further reactions involved in the conversion of 4-hydroxyphenyllactate (6) to 4-hydroxybenzoate (3) are not known. However, it has been suggested that 4-hydroxyphenyllactate (6) is converted to 4-hydroxycinnamate (7), followed by β-oxidation as the coenzyme A derivative, resulting in the formation of 4-hydroxybenzoate (3) [4]. Evidence in support of this pathway has recently been obtained in higher plants [13] (Fig. 2).

R. Meganathan / FEMS Microbiology Letters 203 (2001) 131–139

E. coli & Yeast

![Diagram]

Fig. 3. Ubiquinone biosynthetic pathway in *E. coli* and yeast. It should be noted that the chemical numbering system locates the prenyl side chain in compound 11 at the C-3 carbon and in compounds 14 and 15 at the C-5 carbon. In compounds 12, 13, 16 and subsequent intermediates, the prenyl side chain is assigned to C-2. Compounds 17–20 are drawn in the quinol form. Some authors draw these structures in the quinone form. The chemical names for the intermediates of the pathway are as follows: 1, chorismate; 2, tyrosine; 3, 4-hydroxybenzoate; 11, 3-polyprenyl-4-hydroxybenzoate; 12, 2-polyprenylphenol; 13, 2-polyprenyl-6-hydroxyphenol; 14, 3,4-dihydroxy-5-polyprenylbenzoate; 15, 3-methoxy-4-hydroxy-5-polyprenylbenzoate; 16, 2-polyprenyl-6-methoxyphenol; 17, 2-polyprenyl-6-methoxy-1,4-benzoquinol; 18, 2-polyprenyl-3-methyl-6-methoxy-1,4-benzoquinol; 19, 2-polyprenyl-3-methyl-5-hydroxy-6-methoxy-1,4-benzoquinol; 20, ubiquinol; Q, ubiquinone. The conversion of 20, ubiquinol, to Q is thought to be non-enzymatic. For *E. coli*, polyprenyl = octaprenyl; for yeast, polyprenyl = hexaprenyl. The genes for *E. coli* are the *ubi* genes and the yeast genes are the *COQ* genes.

in both conditions. However, the mutants blocked in the oxygenases discussed above, namely *ubiB*, *ubiH*, and *ubiF*, synthesize Q under anaerobic conditions, providing evidence that there are alternate hydroxylases for the synthesis of Q anaerobically. These anaerobic hydroxylases likely derive the hydroxyl groups from the solvent water [8,11].

### 4.3.2. Methylations

There are three methylation reactions in the pathway, of which two are on O and one on C. These reactions are as follows:

1. 2-octaprenyl-6-hydroxyphenol (13) → 2-octaprenyl-6-methoxyphenol (16) (i.e., O-methylation);
2. 2-octaprenyl-6-methoxy-1,4-benzoquinol (17) → 2-octaprenyl-3-methyl-6-methoxy-1,4-benzoquinol (18) (i.e., C-methylation);
3. 2-octaprenyl-3-methyl-5-hydroxy-6-methoxy-1,4-benzoquinol (19) → ubiquinol (20) (i.e., O-methylation).

As mentioned above, these three methylations alternate with hydroxylations, introducing methyl groups at the 6-OH, at the ring C-3, and at the 5-OH respectively. S-Adenosylmethionine is the actual methyl donor being converted to S-adenosylhomocysteine (SAH).

A non-specific C-methyltransferase encoded by the *ubiE* gene methylates the ring C-3 resulting in the conversion of (17 → 18). In addition, this methyltransferase also methylates demethylmenaquinone to menaquinone.

Both O-methylations mentioned above, namely the methylation of 6-OH and 5-OH, are carried out by the same O-methyltransferase encoded by the *ubiG* gene. This non-specific O-methyltransferase, in addition to carrying out the methylation of (13 → 16) and (19 → 20), also carries out the O-methylation of the *S. cerevisiae* Q biosynthetic intermediate (14 → 15) [11,16].

## 5. Reactions of ring modification in yeast

In yeast, the side chain precursor used for the prenyltransferase is hexaprenyl diphosphate. The product of the reaction 3-hexaprenyl-4-hydroxybenzoate (11) undergoes further ring modification reactions in a different order than that of *E. coli* just described. Thus, compound (11) first undergoes hydroxylation to (14) followed by methylation to (15) and finally decarboxylation to (16) (Fig. 3). The O-methylase encoded by the COQ3 gene like UbiG is non-specific and it catalyzes the conversion of (14 → 15) and (19 → 20). In addition, it converts the *E. coli* Q biosynthetic intermediate (13 → 16). The *E. coli* UbiG can function in vivo provided the protein contains the yeast mitochondrial import leader sequence [11,16].

## 6. Biosynthesis of the polyprenyl tail precursors

The biosynthetic precursors for the polyprenyl tail are isopentenyl diphosphate (IPP) (26) and DMAPP (27). There are two different pathways for the biosynthesis of these precursors (Fig. 4). Those bacteria that possess Q appear to use the MEP pathway (Fig. 4B), while the eukaryotic microorganisms use the mevalonate pathway (Fig. 4A) [19,20]. However, there are exceptions; certain eukaryotic microbes, like the green algae and the malarial parasite *Plasmodium falciparum*, appear to possess the MEP pathway, while a few bacteria contain the mevalonate pathway [21]. These exceptions to the general distribution have been attributed to horizontal gene transfer. Complete lists of the distribution of the pathways in various microorganisms are available [22,23].

### 6.1. MEP pathway

Two glycolytic intermediates, pyruvate (28) and D-glyceraldehyde 3-phosphate (30), are the starting precursors for the biosynthesis of IPP (26) and DMAPP (27) by the MEP pathway (Fig. 4B). The pyruvate is decarboxylated in a TPP-requiring reaction by a mechanism analogous to the pyruvate dehydrogenase (E₁) of the pyruvate dehydrogenase complex. The product of decarboxylation, hydroxyethyl-TPP anion (29), condenses with the aldehyde group of D-glyceraldehyde 3-phosphate (30), resulting in the formation of 1-deoxy-D-xylulose 5-phosphate (DXP) (31). The enzyme catalyzing the reaction is DXP synthase encoded by the *dxs* gene located at 9 min on the *E. coli* linkage map.

In the next step of the pathway, the DXP undergoes a benzyllic type rearrangement and reduction resulting in the formation of MEP (32). This conversion is mediated by an NADPH-dependent reductoisomerase encoded by the *dxr* gene located at 4.2 min on the *E. coli* linkage map.

Further studies established that the MEP (32) is converted to 4-diphosphocytidyl-2-C-methylerythritol (33) in a novel CTP-dependent reaction. The enzyme catalyzing the reaction, designated as 4-diphosphocytidyl-2-C-methylerythritol synthase, is encoded by the *ispD* gene. Subsequently, compound (33) is phosphorylated at the 2-position by an ATP-dependent kinase encoded by the *ispE* gene resulting in the formation of compound (34). In the next step of the pathway, an enzyme designated as 2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase encoded by the *ispF* gene eliminates CMP from compound (34) resulting in the formation of 2-C-methyl-D-erythritol 2,4-cyclodiphosphate (35) [11,21]. It is worth mentioning that compound (35) had been shown to accumulate in bacteria under oxidative stress [11].

The genes, enzymes, intermediates, and reactions involved in the conversion of the cyclodiphosphate (35) to IPP (26) and DMAPP (27) are yet to be determined. However, evidence has been obtained in support of the biosynthesis of both IPP (26) and DMAPP (27) by independent mechanisms in the terminal steps of the pathway [21,24]. An IPP isomerase encoded by the *idi* gene interconverts IPP (26) and DMAPP (27). It has been postulated that these terminal steps for the formation of IPP (26) and DMAPP (27) from the cyclic diphosphate (35) may involve a ring opening reaction, two dehydrations, and two reduction steps [21]. It is likely that the recently identified *lytB* gene of *E. coli* is involved in one of the unidentified late reactions just mentioned [25].

#### 6.2. Mevalonate pathway

Eukaryotic microorganisms like the fungi and yeasts lack the MEP pathway and rely on the mevalonate pathway with the few exceptions mentioned above (Fig. 4A). The starting precursor for the mevalonate pathway is acetyl-CoA. The pathway is initiated by the transfer of an acetyl group from one acetyl-CoA (10) to the methyl car-

Fig. 4. Pathways for the biosynthesis of isopentenyl diphosphate and dimethylallyl diphosphate. A: Mevalonate pathway. 10, acetyl-CoA; 21, acetoacetyl-CoA; 22, HMG-CoA; 23, mevalonate; 24, mevalonate 5-phosphate; 25, mevalonate 5-diphosphate; 26, isopentenyl diphosphate; 27, dimethylallyl diphosphate. B: 2-C-Methyl-D-erythritol 4-phosphate (MEP) pathway. 28, pyruvate; 29, hydroxyethyl-TPP anion; 30, D-glyceraldehyde-3-phosphate; 31, l-deoxy-D-xylulose 5-phosphate (DXP); 32, 2-C-methyl-D-erythritol 4-phosphate (MEP); 33, 4-diphosphocytidyl-2-C-methylerythritol; 34, 4-diphosphocytidyl-2-C-methyl-D-erythritol 2-phosphate; 35, 2-C-methyl-D-erythritol 2,4-cyclodiphosphate; 26, isopentenyl diphosphate; 27, dimethylallyl diphosphate. Multiple arrows indicate unknown reactions. The names of the genes are shown in italics.

bon of a second acetyl-CoA (10) resulting in the formation of acetoacetyl-CoA (21). In the next step of the pathway, another molecule of acetyl-CoA condenses with (21), resulting in the formation of 3-hydroxy-3-methylglutaryl-CoA (HMG-CoA) (22). The HMG-CoA is reduced in a two-step reaction requiring 2 moles of NADPH to mevalonate (23). The mevalonate (23) is converted to mevalonate diphosphate (25) by two phosphorylations mediated by mevalonate kinase and phosphomevalonate kinase, respectively. In the subsequent step of the pathway, the mevalonate diphosphate (25) undergoes dehydration-decarboxylation in an ATP requiring reaction, resulting in IPP (26). An IPP isomerase catalyzes the isomerization of IPP (26) → DMAPP (27) [26,27].

R. Meganathan / FEMS Microbiology Letters 203 (2001) 131–139

$\begin{aligned}
& \text { OPP (DMAPP, C$_5$) } \underset{\text { idi }}{\longleftrightarrow} \text { OPP (IPP, C$_5$)} \\
& \downarrow \quad \text { ispA } \\
& \text { IPP } \\
& \text { OPP (GPP, C$_{10}$) } \\
& \downarrow \quad \text { ispA } \\
& \text { IPP } \\
& \text { OPP (FPP, C$_{15}$) } \\
& \downarrow \quad \text { ispB } \\
& \text { IPP } \\
& \text { OPP (GGPP, C$_{20}$) } \\
& \downarrow \quad \text { ispB } \\
& \text { IPP } \\
& \text { OPP (GFPP, C$_{25}$) } \\
& \downarrow \quad \text { ispB } \\
& \text { IPP } \\
& \text { OPP (HexPP, C$_{30}$) } \\
& \downarrow \quad \text { ispB } \\
& \text { IPP } \\
& \text { OPP (HepPP, C$_{35}$) } \\
& \downarrow \quad \text { ispB } \\
& \text { IPP } \\
& \text { OPP (OPP, C$_{40}$) } \\
& \downarrow \quad \text { IPP } \\
& \text { OPP (SPP, C$_{45}$) } \\
& \downarrow \quad \text { IPP } \\
& \text { OPP(DPP,C$_{50}$) }
\end{aligned}$

Q-6 → S. cerevisiae  
MK-7 → Bacillus sp.  
Q-8 MK-8 → E. coli  
Q-9 → R. capsulatus  
Q-10 → G. suboxidans  

Fig. 5. Pathway for the biosynthesis of polyprenyl diphosphate from isopentenyl diphosphate. The names of compounds are as follows: IPP, isopentenyl diphosphate; DMAPP, dimethylallyl diphosphate; GPP, geranyl diphosphate; FPP, farnesyl diphosphate; GGPP, geranylgeranyl diphosphate; GFPP, geranylgeranylfarnesyl diphosphate; HexPP, hexaprenyl diphosphate; HepPP, heptaprenyl diphosphate; OPP, octaprenyl diphosphate; SPP, solanesyl diphosphate; DPP, decaprenyl diphosphate. *E. coli* IspB produces the (C$_{40}$) octaprenyl side chain. However, if the *ispB* mutant is complemented with homologous genes from organisms having different side chain lengths, then the side chain of Q can be altered.

---

### 7. Assembly and length of the polyprenyl tail in bacteria

#### 7.1. Assembly of the polyprenyl tail in bacteria

As described above, bacteria produce the two building blocks, IPP (26) and DMAPP (27), by the MEP pathway (Fig. 4B). For the building of the chain, two enzymes, namely farnesyl diphosphate (FPP) synthase and PPP synthase, are required. For the chain elongation, the DMAPP (27) serves as the primer to which an IPP (26) is added by head-to-tail condensation resulting in the formation of C$_{10}$ geranyl diphosphate (GPP) (36). In the next step, the GPP (36) condenses with another molecule of IPP (26) with the formation of C$_{15}$ FPP (37). FPP synthase is responsible for both the reactions. In *E. coli*, the gene responsible for this enzyme has been designated as *ispA* (Fig. 5) [28].

The FPP formed by the FPP synthase is further elongated by PPP synthase. As already pointed out, the precise length of the polyprenyl side chain present in an organism is a constant and is determined by the nature of the PPP

#### 7.2. Tail length determination

The availability of the insertion mutant described above made it possible to alter the chain length of the tail precisely and determine its effect on the physiology of the cell. The genes encoding polyprenyl diphosphates of varying chain length were cloned from different organisms and plasmids containing these genes were introduced into the *ispB::Cm*${}^r$ mutant by plasmid swapping (replacing the resident plasmid containing the *ispB* gene with the heterologous plasmid). When the genes for PPP synthase from *Haemophilus influenzae*, *R. capsulatus* or *G. suboxidans* were introduced, the transformed *ispB* insertion mutant

produced Q-7, Q-9 and Q-10, respectively. The Q containing the shorter or longer chain lengths had no adverse effect on the host. However, the *ispB* mutant was unable to express the genes encoding hexaprenyl diphosphate (HexPP) synthase from *S. cerevisiae* and decaprenyl diphosphate (DPP) synthase from *Schizosaccharomyces pombe*. The reason for this inability of *E. coli* to express the genes from the two yeasts is not understood [30].

Having established that *E. coli* is apparently indifferent to the length of the side chain of Q, Kainou et al. [31] investigated the precise factors that determine the chain length. Alignment of the amino acid sequence of the long chain PPP synthases from various organisms reveals seven conserved regions. Two of these conserved regions contain DDXXD motifs that are the binding sites for substrates in the presence of Mg²⁺. They engineered 15 different mutations into *ispB* gene resulting in amino acid replacements at the critical positions of the protein. The selection of critical amino acids was greatly aided by previous mutagenic studies on GGPP and FPP synthases of *Sulfolobus acidocaldarius* and *Bacillus stearothermophilus*, respectively [32,33]. The chain length of Q synthesized by various mutant enzymes when complementing the *ispB* null mutant of *E. coli* was determined. Four of the mutant plasmids, Y37A, A79Y, K170G, and K170A, failed to complement the null mutant indicating that they had lost functional activity. In contrast, mutant Y38A produced Q-7 with minor amounts of Q-6 and Q-8. A double mutant Y37A/Y38A complemented strain contained Q-6 and Q-7. Two other strains, R321V and Y38A/R321V, contained Q-6 and minor amounts of Q-5 and Q-7. Mutants K235L and R321A produced Q-8 as the major product and Q-7 as the minor product; however, the level of Q-7 was greater than that of wild-type IspB. On the basis of these results, the amino acids Tyr-38, Lys-235, and Arg-321 were implicated in chain length determination. Further, two Arg-321 mutants (R321A and R321D) showed a thermosensitive growth phenotype. These authors further demonstrated that dimerization is important in chain length determination [31].

### 8. Assembly and length of the polyprenyl tail in yeast

#### 8.1. Assembly of the polyprenyl tail in yeast

The yeast *Saccharomyces* and other fungi produce the IPP (26) required for polyprenyl diphosphate synthesis by the MVA pathway (Fig. 4A) [19]. The IPP (26) produced is isomerized to DMAPP (27). In the next step of the pathway, the enzyme FPP synthase catalyzes the sequential 1′-4 coupling of IPP (26) with DMAPP (27) and GPP (36) resulting in the formation of GPP (36) and FPP (37), respectively. The FPP synthase contains two aspartate rich motifs (domains I and II) where the two substrates bind, and, in addition, a pair of arginine residues near domain I and a lysine near domain II. The amino acids of the two motifs and the conserved arginine and lysine residues have been subjected to site-directed mutagenic studies. The effect of these studies on the catalysis by the enzyme has been documented [34].

The product of the FPP synthase is elongated further by the enzyme hexaprenyl diphosphate synthase, resulting in the formation of HexPP (40). Mutants deficient in the enzyme have been designated as *coq1* [16].

#### 8.2. Tail length determination

As mentioned above, yeast contains Q-6 with the 30-carbon hexaprenyl side chain. In order to determine the effect of varying the length of the side chain, polyprenyl diphosphate synthase genes from *S. acidocaldarius, S. cerevisiae, H. influenzae, E. coli, R. capsulatus* and *G. suboxydans* were cloned and fused with the sequence encoding the mitochondrial import leader peptide of *S. cerevisiae*. When these genes were expressed in the *coq1* mutant, it produced the corresponding Q-5, Q-6, Q-7, Q-8, Q-9, and Q-10 respectively. Growth properties of these strains carrying Q with different chain lengths were the same as those of the parent strain [35].

To investigate further the effect of varying chain length, Kainou et al. [31] fused the mutant *ispB* genes of *E. coli* to a sequence encoding the 53 amino acids of the yeast COQ1 mitochondrial import signal. These mutant plasmids were introduced into the yeast null mutant (*COQ1::URA3*) and the Q synthesized was determined. The species of Q produced by the mutant enzymes was found to be similar to that of *E. coli* described above. However, in some cases the Q produced had a shorter side chain length compared to that of *E. coli* just described above. Again, yeast was indifferent to prenyl chain length as was the case for *E. coli* described above.

### 9. Conclusion

The various reactions involved in the biosynthesis of the quinoid nucleus of Q from 4-hydroxybenzoate in both bacteria and yeast are reasonably well understood. As pointed out above, both *E. coli* and yeast form 4-hydroxybenzoate from chorismate by the chorismate pyruvate lyase reaction. However, the reactions involved in the formation of 4-hydroxybenzoate from tyrosine in eukaryotes remain to be elucidated. The various enzymes involved in the biosynthesis of the benzoquinone ring from tyrosine are yet to be characterized from any organism. Purification, characterization, structure determination and mechanistic studies on these enzymes are fertile areas for future research. Further, the terminal steps involved in the MEP pathway for the biosynthesis of IPP and DMAPP remain to be elucidated. Mechanistic studies on the enzymes of the pathway involving many of the novel reactions remain to

be pursued. Finally, the regulation of the enzymes involved in the biosynthetic pathways that lead to the head and tail of Q are in need of further investigation.

**Acknowledgements**

I thank Prof. Ronald Bentley for critical comments on the manuscript.

**References**

[1] Collins, M.D. and Jones, D. (1981) Distribution of isoprenoid quinone structural types in bacteria and their taxonomic implication. Microbiol. Rev. 45, 316–354.

[2] Bentley, R. (1990) The shikimate pathway – a metabolic tree with many branches. Crit. Rev. Biochem. Mol. Biol. 25, 307–384.

[3] Haslam, E. (1996) The Shikimic Acid: Metabolism and Metabolites, 387 pp. Wiley, New York.

[4] Olson, R.E. and Rudney, H. (1983) Biosynthesis of ubiquinone. Vitam. Horm. 40, 1–43.

[5] Pennock, J.F. and Threlfall, D.R. (1983) Biosynthesis of ubiquinone and related compounds. In: Biosynthesis of Isoprenoid Compounds, Vol. 2 (Porter, J.W. and Spurgeon, S.L., Eds.), pp. 191–303. John Wiley and Sons, New York.

[6] Gibson, F. (1973) Chemical and genetic studies on the biosynthesis of ubiquinone by *Escherichia coli*. Biochem. Soc. Trans. 1, 317–326.

[7] Gibson, F. and Young, I.G. (1978) Isolation and characterization of intermediates in ubiquinone biosynthesis. Methods Enzymol. 53, 600–609.

[8] Meganathan, R. (1996) Biosynthesis of the isoprenoid quinones menaquinone (vitamin K₂) and ubiquinone (coenzyme Q). In: *Escherichia coli* and *Salmonella*: Cellular and Molecular Biology, Vol. I (Neidhardt, F.C., Curtiss, R. III, Ingraham, J.L., Lin, E.C.C., Low, K.B., Magasanik, B., Reznikoff, W.S., Riley, M., Schaechter, M. and Umbarger, H.E., Eds.), pp. 642–656. American Society for Microbiology, Washington, DC.

[9] Soballe, B. and Poole, R.K. (1999) Microbial ubiquinones: multiple roles in respiration, gene regulation and oxidative stress management. Microbiology 145, 1817–1830.

[10] Begley, T.P., Kinsland, C., Taylor, S., Tandon, M., Nicewonger, R., Wu, M., Chiu, H.J., Kelleher, N., Campobasso, N. and Zhang, Y. (1998) Cofactor biosynthesis: a mechanistic perspective. Curr. Chem. 195, 93–142.

[11] Meganathan, R. (2001) Biosynthesis of menaquinone (vitamin K₂) and ubiquinone (coenzyme Q): a perspective on enzymatic mechanisms. Vitam. Horm. 61, 173–218.

[12] Booth, A.N., Masri, M.S., Robbins, D.J., Emerson, O.H., Jones, F.T. and DeEds, F. (1960) Urinary phenolic metabolites of tyrosine. J. Biol. Chem. 235, 2649–2652.

[13] Loscher, R. and Heide, L. (1994) Biosynthesis of p-hydroxybenzoate from p-coumarate and p-coumaroyl-coenzyme A in cell-free extracts of *Lithospermum erythrorhizon* cell cultures. Plant Physiol. 106, 271–279.

[14] Zhang, H. and Javor, G.T. (2000) Identification of the *ubiD* gene on the *Escherichia coli* chromosome. J. Bacteriol. 182, 6243–6246.

[15] Breinig, S., Schiltz, E. and Fuchs, G. (2000) Genes involved in anaerobic metabolism of phenol in the bacterium *Thauera aromatica*. J. Bacteriol. 182, 5849–5863.

[16] Jonassen, T. and Clarke, C. (2001) Genetic analysis of coenzyme Q biosynthesis. In: Coenzyme Q: Molecular Mechanisms in Health and Disease (Kagan, V.E. and Quinn, J., Eds.), pp. 185–208. CRC Press, Boca Raton, FL.

[17] Poon, W.W., Davis, D.E., Ha, H.T., Jonassen, T., Rather, P.N. and Clarke, C.F. (2000) Identification of *Escherichia coli ubiB*, a gene required for the first monooxygenase step in ubiquinone biosynthesis. J. Bacteriol. 182, 5139–5146.

[18] Kwon, O., Kotsakis, A. and Meganathan, R. (2000) Ubiquinone (coenzyme Q) biosynthesis in *Escherichia coli*: identification of the *ubiF* gene. FEMS Microbiol. Lett. 186, 157–161.

[19] Disch, A. and Rohmer, M. (1998) On the absence of the glyceraldehyde 3-phosphate/pyruvate pathway for isoprenoid biosynthesis in fungi and yeasts. FEMS Microbiol. Lett. 168, 201–208.

[20] Rohmer, M. (1999) The discovery of a mevalonate-independent pathway for isoprenoid biosynthesis in bacteria, algae and higher plants. Nat. Prod. Rep. 16, 565–574.

[21] Eisenreich, W., Rohdich, F. and Bacher, A. (2001) Deoxyxylulose phosphate pathway to terpenoids. Trends Plant Sci. 6, 78–84.

[22] Lange, B.M., Rujan, T., Martin, W. and Croteau, R. (2000) Isoprenoid biosynthesis: the evolution of two ancient and distinct pathways across genomes. Proc. Natl. Acad. Sci. USA 97, 13172–13177.

[23] Boucher, Y. and Doolittle, W.F. (2000) The role of lateral gene transfer in the evolution of isoprenoid biosynthesis pathways. Mol. Microbiol. 37, 703–716.

[24] Rodriguez-Concepcion, M., Campos, N., Maria Lois, L., Maldonado, C., Hoeffler, J.F., Grosdemange-Billiard, C., Rohmer, M. and Boronat, A. (2000) Genetic evidence of branching in the isoprenoid pathway for the production of isopentenyl diphosphate and dimethylallyl diphosphate in *Escherichia coli*. FEBS Lett. 473, 328–332.

[25] Cunningham Jr., F.X., Lafond, T.P. and Gantt, E. (2000) Evidence of a role for *LytB* in the nonmevalonate pathway of isoprenoid biosynthesis. J. Bacteriol. 182, 5841–5848.

[26] Grunler, J., Ericsson, J. and Dallner, G. (1994) Branch-point reactions in the biosynthesis of cholesterol, dolichol, ubiquinone and prenylated proteins. Biochim. Biophys. Acta 1212, 259–277.

[27] Qureshi, N. and Porter, J.W. (1981) Conversion of acetyl-coenzyme A to isopentenyl pyrophosphate. In: Biosynthesis of Isoprenoid Compounds, Vol. 1 (Porter, J.W. and Spurgeon, S.L., Eds.), pp. 47–94. Wiley, New York.

[28] Fujisaki, S., Nishino, T., Katsuki, H., Hara, H., Nishimura, Y. and Hirota, Y. (1989) Isolation and characterization of an *Escherichia coli* mutant having temperature-sensitive farnesyl diphosphate synthase. J. Bacteriol. 171, 5654–5658.

[29] Okada, K., Minehira, M., Zhu, X., Suzuki, K., Nakagawa, T., Matsuda, H. and Kawamukai, M. (1997) The *ispB* gene encoding octaprenyl diphosphate synthase is essential for growth of *Escherichia coli*. J. Bacteriol. 179, 3058–3060.

[30] Okada, K., Kamiya, Y., Zhu, X., Suzuki, K., Tanaka, K., Nakagawa, T., Matsuda, H. and Kawamukai, M. (1997) Cloning of the *sdsA* gene encoding solanesyl diphosphate synthase from *Rhodobacter capsulatus* and its functional expression in *Escherichia coli* and *Saccharomyces cerevisiae*. J. Bacteriol. 179, 5992–5998.

[31] Kainou, T., Okada, K., Suzuki, K., Nakagawa, T., Matsuda, H. and Kawamukai, M. (2001) Dimer formation of octaprenyl diphosphate synthase (IspB) is essential for chain length determination of ubiquinone. J. Biol. Chem. 276, 7876–7883.

[32] Ohnuma, S., Hirooka, K., Tsuruoka, N., Yano, M., Ohto, C., Nakane, H. and Nishino, T. (1998) A pathway where polyprenyl diphosphate elongates in prenyltransferase. Insight into a common mechanism of chain length determination of prenyltransferases. J. Biol. Chem. 273, 26705–26713.

[33] Ogura, K. and Koyama, T. (1998) Enzymatic aspects of isoprenoid chain elongation. Chem. Rev. 98, 1263–1276.

[34] Song, L. and Poulter, C.D. (1994) Yeast farnesyl-diphosphate synthase: site-directed mutagenesis of residues in highly conserved prenyltransferase domains I and II. Proc. Natl. Acad. Sci. USA 91, 3044–3048.

[35] Okada, K., Kainou, T., Matsuda, H. and Kawamukai, M. (1998) Biological significance of the side chain length of ubiquinone in *Saccharomyces cerevisiae*. FEBS Lett. 431, 241–244.
