
ENZYME CATALYSIS AND
REGULATION:
Malonyl-CoA:Anthocyanin 5-O
-Glucoside-6?-O-Malonyltransferase from
Scarlet Sage ( Salvia splendens ) Flowers:
ENZYME PURIFICATION, GENE
CLONING, EXPRESSION, AND
CHARACTERIZATION

Hirokazu Suzuki, Toru Nakayama, Keiko
Yonekura-Sakakibara, Yuko Fukui, Noriko
Nakamura, Masahiro Nakao, Yoshikazu
Tanaka, Masa-atsu Yamaguchi, Takaaki
Kusumi and Tokuzo Nishino
J. Biol. Chem. 2001, 276:49013-49019.
doi: 10.1074/jbc.M108444200 originally published online October 11, 2001

Access the most updated version of this article at doi: [10.1074/jbc.M108444200](http://dx.doi.org/10.1074/jbc.M108444200)

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinitysites).

Alerts:
- [When this article is cited](#)
- [When a correction for this article is posted](#)

[Click here](#) to choose from all of JBC's e-mail alerts

Supplemental material:
[http://www.jbc.org/content/suppl/2001/12/21/276.52.49013.DC1.html](http://www.jbc.org/content/suppl/2001/12/21/276.52.49013.DC1.html)

This article cites 25 references, 8 of which can be accessed free at
[http://www.jbc.org/content/276/52/49013.full.html#ref-list-1](http://www.jbc.org/content/276/52/49013.full.html#ref-list-1)

THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2001 by The American Society for Biochemistry and Molecular Biology, Inc.

Malonyl-CoA:Anthocyanin 5-O-Glucoside-6″-O-Malonyltransferase from Scarlet Sage (Salvia splendens) Flowers

ENZYME PURIFICATION, GENE CLONING, EXPRESSION, AND CHARACTERIZATION*[S]

Received for publication, August 31, 2001, and in revised form, October 10, 2001  
Published, JBC Papers in Press, October 11, 2001, DOI 10.1074/jbc.M108444200

Hirokazu Suzuki‡, Toru Nakayama‡, Keiko Yonekura-Sakakibara§∥, Yuko Fukui§, Noriko Nakamura§, Masahiro Nakao§, Yoshikazu Tanaka§, Masa-atsu Yamaguchi∥, Takaaki Kusumi§, and Tokuzo Nishino***

From the ‡Department of Biomolecular Engineering, Graduate School of Engineering, Tohoku University, Aoba-yama 07, Sendai 980-8579, Japan, the §Institute for Fundamental Research, Suntory Ltd., 1-1-1, Wakayamadai, Shimamoto-cho, Mishima-gun, Osaka 618-8503, Japan, and the ∥Department of Food Science and Technology, Minami-Kyushu University, Miyazaki 884-0003, Japan

The orange to blue coloration of flowers in nature is, in most cases, provided by anthocyanins, a class of plant flavonoids, many of which are modified by malonyl group(s). However, the identity of the enzyme catalyzing the malonylation reaction remains to be established. Here, we describe for the first time the purification, characterization, and cDNA cloning of an anthocyanin malonyltransferase from scarlet sage (Salvia splendens) flowers. The purified enzyme (termed Ss5MaT1) was a monomeric 50-kDa protein catalyzing the regiospecific transfer of the malonyl group from malonyl-CoA to the 6″-hydroxyl group of the 5-glucosyl moiety of anthocyanins. Ss5MaT1 showed a *k*~cat~ value of 7.8 s⁻¹ at 30 °C and pH 7.0 for the malonylation of bisdemalonylsalvianin (pelargonidin 3-(6″-O-caffeyl-β-glucopyranoside)-5-β-glucopyranoside) and *K*~m~ values of 101 μM and 57 μM for bisdemalonylsalvianin and malonyl-CoA, respectively. *p*-Coumaric acid, which mimics an aromatic acyl group linked to the 3-glucosidic moiety of an anthocyanin substrate, was a competitive inhibitor with respect to the substrate. This strongly suggests that the presence of an aromatic acyl group at the 3-glucosidic moiety of anthocyanin is important for substrate recognition by the enzyme. On the basis of the partial amino acid sequences of the purified enzyme, we isolated a cDNA encoding Ss5MaT1. Ss5MaT1 consisted of 462 amino acids and shared motifs that are commonly found among members of a versatile plant acyltransferase family, which was recently shown to include numerous homologs of unknown biochemical functions. Northern blot analysis revealed that the transcripts of Ss5MaT1 were detected in petals, sepals, bracts, and red stems, in accordance with the pigment accumulation patterns. Phylogenetic analysis suggests that the aliphatic and aromatic acylations of anthocyanins are generally catalyzed by subfamily members of the plant acyltransferase family.

Most orange to blue colors of flowers result from anthocyanins, a class of plant flavonoids that exist as glycosylated, acylated, and/or methylated forms. Despite a 100-year history of anthocyanin research, the widespread existence of acylated anthocyanins in plants has been recognized only during the past 2 decades, mainly because of their lability during the previously employed isolation processes of the pigments (1). Two major types of acyl substituents of anthocyanins exist (*i.e.* aromatic and aliphatic acyl groups), both of which are commonly linked to a hydroxy group of a glycosyl moiety of anthocyanins. It has been proposed that aromatic acylation makes anthocyanin bluer by intramolecular stacking of the anthocyanins with polyphenols (2, 3), whereas aliphatic acylation has no such effect but is believed to be important for enhancing pigment solubility in water, protecting glycosides from enzymatic degradation, stabilizing anthocyanin structures, or facilitating the uptake of anthocyanins into the vacuoles (4).

The biosynthetic pathways leading to anthocyanidin 3-O-glucosides have been well characterized and are conserved in plant species. On the other hand, modification, including acylation, of anthocyanidin 3-O-glucosides is variable and has not been understood well (4–6). In the crude extracts of a variety of plant tissues, the acylation of anthocyanins has been shown to be catalyzed by flavonoid-specific acyltransferases, which use coenzyme A (CoA) esters as acyl donors (4). Recently, two aromatic acyltransferases (*i.e.* hydroxycinnamoyl-CoA:anthocyanin 5-O-glucoside-6″-O-acyltransferase from the petals of gentian (Gentiana triflora, Gt5AT) (7) and hydroxycinnamoyl-CoA:anthocyanin 3-O-glucoside-6″-O-acyltransferase from Perilla frutescens (Pf3AT) (8)) have been cloned and characterized. Sequence comparison studies suggest that these aromatic acyltransferases belong to a versatile plant acyltransferase family, which is a large, recently identified protein family including numerous acyltransferase homologs of as yet unidentified biochemical functions in plants (8–10).

Malonylation is probably the most common aliphatic acylation of flavonoids and is exemplified by salvianin (*i.e.* pelargonidin 3-(6′-O-caffeyl-β-glucopyranoside)-5-(4″,6″-O-dimalonyl-β-glucopyranoside)) of scarlet sage (Salvia splendens) flowers (Fig. 1, compound 1a) (11), malonylshisonin (*i.e.* cyanidin 3-(6″-O-p-coumaryl-β-glucopyranoside)-5-(6″-O-malonyl-β-glucopyranoside)) of the red forma *P. frutescens* leaves (compound 2a) (12), and cyanidin 3-O-(3″,6″-O-dimalonyl-β-glucopyranoside) of chrysanthemum flowers (13). Putative roles of anthocyanin acylation in the translocation and stabilization of the pigments suggest that the anthocyanin malonyltransferase gene may be one of the key tools for controlling flower colors by genetic

* The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
* The nucleotide sequence(s) reported in this paper has been submitted to the GenBank™/EBI Data Bank with accession number(s) AF405707 (for Ss5MaT1) and AF405204 (for Pf5MaT1).
* [S] The on-line version of this article (available at http://www.jbc.org) contains Figs. 1S, 2S, and 3S.
* ∥ Present address: Plant Science Center, RIKEN (The Institute of Physical and Chemical Research), 2-1, Hirosawa, Wako-shi, Saitama 351-0198, Japan.
* **To whom correspondence should be addressed. Fax: 81-22-217-7270; E-mail: nishino@mail.cc.tohoku.ac.jp.

This paper is available on line at http://www.jbc.org 49013

engineering approaches (5). However, the identity of the anthocyanin malonyltransferases and aliphatic acyltransferases remains to be established, mainly due to the difficulty of their purification (6).

To enhance knowledge of the enzymology of malonylation (or aliphatic acylation) of anthocyanins in plants and shed light on the structure/function relationship of anthocyanin acyltransferases, we have undertaken to study the anthocyanin malonyltransferase of *S. splendens*. We describe here the first isolation, characterization, and gene cloning of an anthocyanin malonyltransferase to show that it is a member of the versatile plant acyltransferase family with a divergent phylogenetic relationship in the family.

### EXPERIMENTAL PROCEDURES

#### Plant Materials and Anthocyanins

Scarlet sage plants (*S. splendens*) were grown in a standard greenhouse at Suntory Ltd. (Osaka, Japan), and recently opened flowers (including petals and sepals; less than 2 cm in length) were collected and stored at –80°C until use. Anthocyanins were isolated and purified at Minami-Kyushu University (Miyazaki, Japan) and were used as authentic samples after confirmation of their structures by instrumental analyses (see legend to Fig. 1S for details).

#### Enzyme and Protein Assays

The standard reaction mixture (a final volume, 100 μl) consisted of 20 mM potassium Pi, pH 7.0, 120 μM anthocyanin substrate, 60 μM malonyl-CoA (final concentrations), and enzyme. For routine assays, shisonin (Fig. 1, compound 2b) was used as an anthocyanin substrate. The mixture without enzyme was preincubated at 30°C, and the reaction was started by the addition of the enzyme. After incubation at 30°C for 20 min, the reaction was stopped by the addition of 200 μl of ice-cold 0.5% (v/v) trifluoroacetic acid.

Anthocyanins in the reaction mixture were routinely analyzed by reversed-phase HPLC¹ using a RAININ Dynamax HPLC system (Rainin Instruments Co., Woburn, MA) equipped with a SHIMADZU SPD-10A VP UV-visible detector (column, Shodex Asahipak ODP-50 4E (4.6 × 250 mm); flow rate, 0.7 ml/min; solvent A, 0.5% (v/v) trifluoroacetic acid; solvent B, 0.5% (v/v) trifluoroacetic acid in 50% (v/v) acetonitrile). After injection (100 μl) onto a column that was equilibrated with 45% B, the column was initially developed isocratically for 3 min, followed by linear gradients from 45% B to 55% B in 15 min and from 55% B to 100% B in 1 min. The column was then washed isocratically with 100% B for 5 min, followed by a linear gradient from 100% B to 45% B in 1 min. The chromatograms were obtained with detection at 520 nm, and anthocyanins were identified by comparing their retention times with those of authentic samples (bisdemalonylsalvianin, 9.6 min; monodemalonylsalvianin, 11.1 min; shisonin, 11.7 min; 6"-malonylshisonin, 14.4 min). The amounts of anthocyanins were determined from peak integrals using authentic samples, which were used for calibration. Kinetic parameters and their S.E. values were estimated by fitting the initial velocity data to the Michaelis-Menten equation by nonlinear regression analysis (14). Protein was quantified by the method of Bradford (15) using a kit (Bio-Rad) with bovine serum albumin as the standard.

#### Purification of Ss5MaT1 from *S. splendens* Petals

All operations were performed at 4°C. All buffers used throughout the enzyme purification contained 30 mM 2-mercaptoethanol, and the buffer pH was set at 7.0.

**Step 1: Ammonium Sulfate Fractionation—** The flowers (less than 2 cm; petals and sepals, 2.7 kg) of *S. splendens* were suspended in 10,800 ml of 100 mM potassium Pi containing 5 mM EDTA, 0.5 mM phenylmethylsulfonyl fluoride, and 5% (w/v) polyvinylpolypyrrolidone and were disrupted for 30 s in a Waring blender, followed by centrifugation at 8,000 × g for 20 min. The supernatant was subjected to ammonium sulfate fractionation, and the precipitate of a 20–50% saturation fraction was dissolved in 3,000 ml of 100 mM potassium Pi containing 1 mM EDTA and 0.1 mM phenylmethylsulfonyl fluoride.

**Step 2: Octyl-Sepharose—** To the enzyme solution, 280 ml of octyl-

**Step 3: MIMETIC Yellow 2—** The enzyme solution (155 ml) was applied to a MIMETIC Yellow 2 (100 ml; Nacalai Tesque, Kyoto, Japan) column equilibrated with Buffer 3. The column was washed with the same buffer. The enzyme was eluted with 20 mM potassium Pi containing 0.05% CHAPS.

**Step 4: MIMETIC Red 3—** The enzyme solution (32 ml) was applied to a MIMETIC Red 3 column (50 ml; Nacalai Tesque) equilibrated with Buffer 4 (5 mM potassium Pi, pH 7.0, containing 0.03% Triton X-100). The column was washed with the same buffer. The enzyme was eluted with Buffer 4 containing 0.1 mM acetyl-CoA and was concentrated by ultrafiltration.

**Step 5: Mono Q FPLC—** The enzyme solution (1.5 ml) was applied to a Mono Q 5/5 column (Amersham Pharmacia Biotech) equilibrated with Buffer 4 containing 0.1 mM acetyl-CoA at a flow rate of 0.05 ml/min, using a Fast Protein Liquid Chromatography (FPLC) system (Amersham Pharmacia Biotech). The column was washed with Buffer 4. The enzyme was eluted with a linear gradient (in 400 min) of 0–1 M NaCl in Buffer 4.

**Step 6: Phenyl-Superose FPLC—** The enzyme solution (0.8 ml) was applied to a phenyl-Superose HR 5/5 column (Amersham Pharmacia Biotech) equilibrated with Buffer 1 at a flow rate of 0.02 ml/min using an FPLC system. The column was washed with the same buffer. The enzyme was eluted with a linear gradient (0–100% in 500 min) of Buffer 2.

SDS-PAGE was carried out according to the method of Laemmli (16). The proteins on the gels were visualized by silver staining.

### cDNA Cloning of Ss5MaT1

Total RNA was isolated from the *S. splendens* petals as described previously (17) and was used for the isolation of poly(A)+-RNA using a kit (Oligotex™-dT30 Super Kit; Roche Molecular Biochemicals). A cDNA library of *S. splendens* petals was constructed using a ZAP Express cDNA Synthesis Kit (Stratagene) following the manufacturer’s guidelines. Partial amino acid sequences of the purified enzyme (termed Ss5MaT1) were determined essentially as described previously (7). Based on the amino acid sequences determined, degenerate oligonucleotide primers were synthesized (see also Fig. 3): primer I (5′-TAYG-CIGCIGGIGAYTCIGTICCGT-3′), primer II (5′-GTICCGIGTIACIAT-HGCIGC-3′), and primer III (5′-YTTICCCCAICCRAARTCIGC-3′), where I indicates inosine and R, H, and Y indicate degenerate sites (R, A/G; H, A/C/T; and Y, C/T). The cDNA library was then used as a template for PCR amplification of partial Ss5MaT1 cDNA using primers I and III, and an amplified fragment was further subjected to nested PCR using primers II and III. The PCR amplifications were completed using 1× TaKaRa PCR buffer, 0.2 mM dNTPs, 1 μM each of primers, and 2.5 units of rTaq DNA polymerase (TaKaRa) with 100 ng of DNA template. Thermal cycling conditions were 96°C for 1 min for denaturation, 42°C for 2 min for annealing, and 72°C for 7 min. The nested PCR was completed in the same way as above except that the annealing temperature was 50°C.

The cDNA library, ~200,000 plaques, was screened using an amplified DNA fragment encoding determined partial amino acid sequences as a probe. The DIG DNA Labeling and Detection Kit (Roche Molecular Biochemicals) was used to label and detect DNA following the manufacturer’s guidelines. Because the longest cDNA clone was found to lack a translation initiation codon, we further analyzed the genomic DNA clone containing the 5′-terminal region of the Ss5MaT1 using an LA PCR in vitro Cloning Kit (TaKaRa) (18). The BamHI-digested *S. splendens* genomic DNA, primer S1 (5′-AGGCAGTGGGTGATTGGAGAAACCGATGCAG-3′), primer S2 (5′-TTCGGTGTTGGACGGGTAGAGGAGGTTGCC-3′), and primer S3 (5′-TGGAAACGATTGTGTGGAGGAATTGGGATTGGGAGG-3′) were used for the analysis.

The nucleotide sequences of complementary and genomic DNAs were determined using an automated DNA sequencer (model 373A; Applied Biosystems). Geneworks (IntelliGenetics) and GENETYX (Software Development) programs were used for analysis of nucleotide and deduced amino acid sequences.

---

¹ The abbreviations used are: HPLC, high pressure liquid chromatography; FPLC, fast protein liquid chromatography; CHAPS, 3-[(3-cholamidopropyl)dimethylammonio]-1-propanesulfonic acid.

Anthocyanin Malonyltransferase of Scarlet Sage Flowers

![Chemical Structures](https://i.imgur.com/chemical_structures.png)

FIG. 1. Structures of salvianin, shisonin, and related anthocyanins. Names of anthocyanins are salvianin (1a), monodemalonylsalvianin (1b), bisdemalonylsalvianin (1c), malonylshisonin (2a), shisonin (2b), pelargonidin 3,5-diglucoside (3a), and delphinidin 3,5-diglucoside (3b). Key positional numberings are also labeled on 1.

| R₁       | R₂       |
|----------|----------|
| 1a       | -OCCH₂COOH | -OCCH₂COOH |
| 1b       | -OCCH₂COOH | -H         |
| 1c       | -H        | -H         |

| R        |
|----------|
| 2a       | -OCCH₂COOH |
| 2b       | -H         |

| R        |
|----------|
| 3a       | -H         |
| 3b       | -OH        |

---

### Expression and Purification of the Recombinant Ss5MaT1

The Ss5MaT1 cDNA that was obtained lacked the 5′-terminal portion by 4 bp, as was revealed by sequencing of the genomic Ss5MaT1 DNA. The four bases were added by the PCR-based strategy, and the resultant full-length Ss5MaT1 cDNA was ligated into the pQE-30 expression vector (Qiagen), as described in the following. The Ss5MaT1 cDNA excised into a pBK-CMV phagemid vector was termed pBK-CMV/Ss5MaT1. To inactivate BamHI and XmnI sites in the Ss5MaT1 gene, a partial Ss5MaT1 cDNA (0.5 kb in length) was first amplified by PCR using pBK-CMV/Ss5MaT1 as a template and primers (5′-GGACCCG-CCGATACCGGAAAATTACTTC and 5′-GGATCCTTAGAATGGTTTG-ACGAGCGCCGGAGA-3′), where the underlining indicates the BamHI site and the double underlining indicates the translation termination codon. Using a mixture of this 0.5-kb cDNA fragment and upstream primer (5′-GGATCCATCGAGGGACGCATGACAACAACAACAAAC-3′; double underlining indicates the translation initiation codon), a full-length Ss5MaT1 DNA (1.4 kb in length) was amplified by PCR with pBK-CMV/Ss5MaT1 as a template. The upstream primer was designed on the basis of the genomic Ss5MaT1 sequence determined by the LA PCR strategy (see above) in order to add the lacking 5′-terminal portion (ATGA). The nucleotide sequence of the amplified DNA was confirmed by DNA sequencing. The amplified DNA fragment (digested with BamHI) was ligated with a BamHI-digested pQE-30 vector to obtain the plasmid pSs5MaT1, which encodes an N-terminal in-frame fusion of Ss5MaT1 with a His₆ tag. Escherichia coli JM109 was used as a host for the expression of pSs5MaT1.

Transformant cells were cultivated at 30°C in 600 ml of an LB medium supplemented with 50 μg/ml of ampicillin until A₆₀₀ reached 0.5. Isopropyl-β-D-thiogalactoside was then added to a final concentration of 0.5 mM, followed by further cultivation for 4 h at 30°C. Cells were harvested by centrifugation. All subsequent operations were carried out at 4°C. Cells were suspended in Buffer 5 (100 mM potassium Pᵢ, pH 7.5, containing 15 mM 2-mercaptoethanol, 0.5 M NaCl, 10% glycerol, and 10 mM imidazole) containing 0.02% Triton X-100, 0.5 mM phenylmethylsulfonyl fluoride, and 2 mg/ml lysozyme. After incubation of the mixture on ice for 30 min, the cell suspension was sonicated for 3 min. Polyethyleneimine was added to a final concentration of 0.1%, and the mixture was gently stirred for 10 min. After centrifugation (10,000 × g, 20 min), the supernatant was applied to a column (1 ml) of Ni²⁺-nitrilotriacetic acid-agarose (Qiagen) equilibrated with Buffer 5. The column was washed with Buffer 6 (100 mM potassium Pᵢ, pH 7.5, 15 mM 2-mercaptoethanol) containing 20 mM imidazole, followed by elution with Buffer 6 containing 200 mM imidazole. The eluate was applied to a High-Q column (Bio-Rad), and the column was washed with the same buffer. The flow-through and washing fractions were pooled and 20% saturated with ammonium sulfate. The enzyme solution was subjected to FPLC on phenyl-Superose HR 5/5 as described above.

---

### RESULTS

#### Purification of Ss5MaT1 from S. splendens Flowers—Salvianin (Fig. 1, compound 1a), which is the predominant anthocyanin responsible for the scarlet coloration of the S. splendens flowers,² has two malonyl groups linked to the 4″- and 6″-hydroxyl functions of the 5-glucosyl moiety. In the crude extracts of S. splendens flowers, we could identify two distinct malonyltransferase activities. One catalyzed the monomalonylation of bisdemalonylsalvianin (Fig. 1, 1c) (termed first malonylation), whereas the other catalyzed the further malonylation of the product of the first malonylation reaction to produce salvianin (second malonylation). These activities could be easily separated from each other by ammonium sulfate fractionation and column chromatographies (data not shown). An enzyme catalyzing the first malonylation could not catalyze the second malonylation and vice versa, indicating that the first and second malonylations of anthocyanin are catalyzed by distinct enzymes. In this study, we have purified the enzyme responsible for the first malonylation (termed Ss5MaT1).

Preliminary studies showed that, during the development of S. splendens flowers, a high specific activity of Ss5MaT1 was found in recently opened flowers (less than 2 cm in length), and a large amount of such flowers (sepals and petals, 2.7 kg) was used as a starting material to purify the enzyme. The enzyme was extremely unstable to oxidation, and the addition of 2-mercaptoethanol in the buffer was essential for efficient purification of the enzyme. Furthermore, we found that the enzyme was specifically eluted from the MIMETIC Red 3 column with acetyl-CoA, allowing us to effectively eliminate contaminant proteins, which were difficult to remove by conventional procedures such as ion exchange and hydrophobic interaction chromatographies. Finally, the enzyme could be purified 685-fold with an activity yield of 0.3% after seven purification steps (Table I). Purified Ss5MaT1 showed a single protein band with a molecular mass of 50 kDa on SDS-PAGE gels (Fig. 2). The native molecular mass of the purified Ss5MaT1 was estimated to be 46 kDa by gel filtration chromatography on a Superdex 200HR column, indicating that the enzyme was monomeric.

The enzyme catalyzed the malonylation of shisonin to exclusively produce a monomalonylated shisonin, as shown by MS analysis of the product (*m/z* 843[M]⁺); however, the enzyme did not catalyze further malonylation of the product. NMR analyses of the product unambiguously showed that it was cyanidin.

---

² Relative contents (mol %) of anthocyanins in the S. splendens flowers were estimated by LC-MS analysis as follows: salvianin/monodemalonylsalvianin/pelargonidin 3-(p-coumaryl-β-glucopyranoside)-5-(dimalonyl-β-glucopyranoside)/pelargonidin 3-(p-coumaryl-β-glucopyranoside)-5-(malonyl-β-glucopyranoside) = 53:22:19:6 (for petals) and 50:17:27:6 (for sepals).

49016

**Anthocyanin Malonyltransferase of Scarlet Sage Flowers**

3-(6″-O-p-coumaryl-β-glucopyranoside)-5-(6″-O-malonyl-β-glucopyranoside) (see Fig. 1S), indicating that the purified enzyme catalyzes the regiospecific transfer of the malonyl group to the 6″-hydroxyl group of the 5-glucosyl moiety of the substrate anthocyanins (Scheme I).

**cDNA Cloning and Sequence Comparison of Ss5MaT1—** We designed the PCR primers on the basis of the determined amino acid sequences and executed first-round PCR using the petal cDNA library of *S. splendens* as a template with primers I and III (see “Experimental Procedures” and Fig. 3). The amplified fragment was further subjected to nested PCR using primers II and III, yielding a PCR product of 880 bp encoding partial amino acid sequences that were determined from the purified enzyme. Using this fragment as a probe, the cDNA library (~200,000 plaques) was screened for Ss5MaT1 under high stringency conditions to obtain 10 positive clones. The cDNAs from these clones were sequenced from the 5′-end, and the longest cDNA was completely sequenced. The deduced amino acid sequence of the clone contained all of the partial amino acid sequences determined from the purified enzyme (Fig. 3). However, it was found to lack the N-terminal region of an open reading frame. The lacking N-terminal region (4 bp in length) was revealed from the sequence of Ss5MaT1 genomic DNA, and the full-length cDNA, Ss5MaT1, was obtained with a PCR-based strategy (see “Experimental Procedures” for details).

The full-length Ss5MaT1 cDNA encoded a protein (calculated mass, 50,723) of 462 amino acids. The deduced amino acid sequence of Ss5MaT1 showed similarity to those of Pf3AT (40% identity (8)) and Gt5AT (34% (7)). Lower sequence identities were also found with 10-deacetylbaucatin III-10-O-acetyltransferase (21% (19)) and taxadienol acetyltransferase (22% (20)) of *Taxus cuspidata*, acetyl-CoA:benzylalcohol acetyltransferase of *Clarkia breweri* (21% (21)), deacetylviolone 4-O-acetyltransferase of *Catharanthus roseus* (23% (9)), hydroxycinnamoyl/benzyl-CoA:anthranilate N-hydroxycinnamoyl/benzoyltransferase of *Dianthus caryophyllus* L. (24% (22)), salutaridiol 7-O-acetyltransferase of *Papaver somniferum* (20% (23)), alcohol acyltransferase of *Fragaria xananassa* (20% (24)), and trichothecene 3-O-acetyltransferase of *Fusarium graminearum* (20% (25)). Ss5MaT1 and these acyltransferases share the conserved sequences, -His-Xaa₃-Asp- (motif 1) and -Asp-Phe-Gly-Trp-Gly- (motif 3; see Fig. 2S).

Ss5MaT1 was expressed under the control of the T5 promoter in *E. coli* JM109 cells as a soluble, catalytically active protein at a level of 30 μg/liter of culture broth. The recombinant Ss5MaT1 could be easily purified to homogeneity from crude extracts of transformant cells with an activity yield of 8%. The recombinant Ss5MaT1 showed catalytic and molecular properties that were essentially identical to those of the native Ss5MaT1 (see below), allowing us to further confirm that the cloned cDNA codes for Ss5MaT1.

**Catalytic Properties and Stability of the Native and Recombinant Ss5MaT1—** We extensively analyzed the catalytic and molecular properties of the native and recombinant Ss5MaT1. In addition to shisonin (see above), the native and recombinant enzymes effectively acted on bisdemalonylsalvianin (Fig. 1, 1c) to produce monodemalonylsalvianin (Fig. 1, 1b); however, monodemalonylsalvianin, salvianin, pelargonidin 3,5-diglucoside (3a), and delphinidin 3,5-diglucoside (3b) could not serve as substrates for either the native or recombinant Ss5MaT1. The kinetic parameters of the malonylation reactions of shisonin and bisdemalonylsalvianin are summarized in Table II. We also examined the acyl donor specificity using the recombinant Ss5MaT1 and shisonin as an acyl acceptor; the relative activities of acetyl-CoA (retention time of reaction product, 18.6 min), methylmalonyl-CoA (19.9 min), and succinyl-CoA (17.7 min) were 2, 3, and 49%, respectively, of that of malonyl-CoA. The recombinant enzyme could not, however, catalyze the

---

**TABLE I**

**Purification of Ss5MaT1 from *S. splendens* flowers**

| Purification step       | Total protein   | Specific activity     | Purification fold | Yield % |
|--------------------------|------------------|------------------------|--------------------|---------|
|                          |                  | nanokatals/mg protein |                    |         |
| Crude extract           | 14,000 mg        | 0.13                   | 1                  | 100     |
| 20–50% (NH₄)₂SO₄        | 7,120 mg         | 0.21                   | 1.7                | 85      |
| Octyl-Sephrose          | 1,830 mg         | 0.38                   | 3.1                | 40      |
| MIMETIC Yellow 2        | 217 mg           | 1.17                   | 9.3                | 14      |
| MIMETIC Red 3           | 49.5 mg          | 6.85                   | 55                 | 19      |
| Mono Q                  | 0.66 mg          | 10.6                   | 84                 | 0.4     |
| Phenyl-Superose         | 0.079 mg         | 86.1<sup>a</sup>       | 685                | 0.3     |

<sup>a</sup> This specific activity corresponds to a molecular activity of 4.34 s⁻¹.

---

**FIG. 2. SDS-PAGE of purified Ss5MaT1 from *S. splendens* flowers.** The purified Ss5MaT1 (lane 2) and marker proteins (lane 1) were simultaneously electrophoresed under the conditions of Laemmli (16) and stained by silver staining.

---

**SCHEME 1. Ss5MaT1-catalyzed malonyl transfer to anthocyanins.** Substrate names are bisdemalonylsalvianin (R₁ = H, R₂ = OH) and shisonin (R₁ = OH, R₂ = H).

Anthocyanin Malonyltransferase of Scarlet Sage Flowers

MTTTTILET CHIPPPPAAN DLSIPLSFFD IKWLHYHPVR RLLFYHHPSS 50

KSQFLHTIVP HLKQSLSLAL THYLPVAGNL LYPSNTEKFP QLRYAAGDSV 100

PVTIAESNSD FESLTGNHTR DADQFYDLLP PIPPIEEESD WKLINIFAVQ 150

ITLFPGEGIC IGFSNHCLG DARSIVGFIS AWGEINGIGG YEGFLSNHSD 200

SLSLPIFDRS FINDPNKIDA IFWKVLRNIP LKTASFPLPT NRVRSTFLLR 250

RSDIEKLKTA TKSPASSFVA AAAFVWSCMV KSGDKSDENA PELFIIPADA 300

RGRVDPPPIPE NYFGNCIVSS VAQVERGKLA AEDGFAVAAE AIGGEIEGKL 350

KNRDEILRGA ENWMSDIFKC FGMSVLGVSG SPKFDLLKA D FGWKKARKLE 400

VLSIDGENHS MSLCSSSDFN GGLEVGLSLP RERMAAFEEV FRASIMAASG 450

PARRSPALVE PL 462

FIG. 3. Deduced amino acid sequence of the Ss5MaT1. The deduced amino acid sequence is shown with a one-letter notation. Partial amino acid sequences determined by an automated Edman degradation are underlined, and those used for designing oligonucleotide primers (I, II, and III) for amplifications of the partial Ss5MaT1 cDNA are indicated with Roman numerals. Amino acid sequences of motifs 1–3 are boxed with numerals.

TABLE II  
Kinetic parameters  

Kinetic parameters for malonyl transfer reaction catalyzed by native and recombinant Ss5MaT1 were determined at 30°C for 20 min as described under “Experimental Procedures.” Concentrations of counter substrates were set at saturating levels.

| Substrate                | $k_{cat}$ ($s^{-1}$) | $K_m$ ($\mu M$) | Counter substrate |
|--------------------------|-----------------------|------------------|-------------------|
| Bisdemalonylsalvianin    | 7.8 ± 0.6             | 101.0 ± 11.0     | Malonyl-CoA       |
| Shisonin                 | 4.3 ± 0.2             | 40.2 ± 3.3       | Malonyl-CoA       |
|                          | 8.7 ± 0.5$^a$         | 35.5 ± 4.7$^a$   | Malonyl-CoA       |
| Malonyl-CoA              |                       | 57.0 ± 5.7       | Bisdemalonyl      |
|                          |                       |                  | salvianin         |
|                          | 14.1 ± 1.2            |                  | Shisonin          |
|                          | 21.9 ± 2.6$^a$        |                  | Shisonin          |

$a$ The kinetic parameters of recombinant Ss5MaT1.

Transfer of either the caffeoyl or the $p$-coumaroyl group from their CoA esters. The native and recombinant Ss5MaT1 was active over a pH range of 6.0–11.0 with a maximum activity at pH 8.0 and was stable at pH 7.5 (at 20°C for 20 h) and below 30°C (at pH 7.0 for 20 min).

Inhibition Studies—The native and recombinant Ss5MaT1 activity was completely inhibited by 0.1 mM Cu$^{2+}$, 0.1 mM Hg$^{2+}$, 10 mM $p$-coumaric acid, and 10 mM caffeic acid. The enzyme was also partially inhibited by 5 mM N-ethylmaleimide (residual activity 47%, after incubation at 20°C for 20 min), 1 mM diethylpyrocarbonate (49%, after incubation at 20°C for 20 min), and 1 mM acetyl-CoA (52%). Other 0.1 mM metal ions (Ca$^{2+}$, Cd$^{2+}$, Co$^{2+}$, Fe$^{2+}$, Mg$^{2+}$, Mn$^{2+}$, Ni$^{2+}$, Sn$^{2+}$, and Zn$^{2+}$) and chemicals (1 mM dithiothreitol, 1 mM EDTA, 1% Tween 20, 1% Triton X-100, 0.1% CHAPS, 0.1 mM pelargonidin 3,5-diglucoside, and 10 mM glucose) had negligible effects on the catalytic activity.

We then kinetically analyzed the inhibition of Ss5MaT1 by $p$-coumaric acid in more detail. Preliminary studies showed that Ss5MaT1 did not catalyze the malonylation of the inhibitor and that the inhibition was a rapid reversible process (data not shown); therefore, $p$-coumaric acid was a dead end inhibitor. To find the type of Ss5MaT1 inhibition by $p$-coumaric acid, the initial velocity data were plotted in double reciprocal forms with $1/v$ versus $1/$[malonyl-CoA] or $1/v$ versus $1/$[shisonin] at several

FIG. 4. Northern blot analysis of the Ss5MaT1 transcript. Total RNA was prepared from leaves, stems, roots, bracts, and sepals and petals from flowers of four developmental stages of *S. splendens* by the method of McGookin (17). Stages of sepals and sepals are as follows: 1, closed bud with no pigmentation (0–5 mm in length); 2, closed bud with first signs of pigmentation (5–10 mm in length); 3, fully pigmented, closed bud (10–20 mm in length); 4, fully opened flower with all anthers dehisced. Northern analysis was carried out as previously described (7), using a DIG-labeled 0.88-kb fragment of Ss5MaT1 cDNA (see “Experimental Procedures”).

fixed concentrations of the inhibitor. A series of straight parallel lines was obtained in the $1/v$ versus $1/$[malonyl-CoA] plots, indicating an uncompetitive inhibition by $p$-coumaric acid for malonyl-CoA. Also, a series of straight lines intersecting on the $1/v$ axis was obtained with the $1/v$ versus $1/$[shisonin] plots, indicating a competitive inhibition by $p$-coumaric acid for shisonin.

Northern Blot Analysis—Northern blot analysis showed that the transcripts for Ss5MaT1 were mainly accumulated in young to opening petals and sepals but were also present at a lower level in bracts and red stem (Fig. 4). The Ss5MaT1 transcripts showed a similar profile to that of the red color development in *S. splendens*.

DISCUSSION

Despite the widespread occurrence of malonylated anthocyanins, the identity of the enzyme responsible for anthocyanin malonylation has long remained unclear, mainly because of the difficulty associated with the purification of the enzyme (6). Indeed, the present purification of Ss5MaT1 from *S. splendens* flowers was not easy either, as shown by the fact that the overall yield of the enzyme activity throughout the purification was very low (0.3%). However, we overcame these circumstances by obtaining a large amount (2.7 kg) of flowers at stages with high specific activities as a starting material as well as by successfully completing affinity chromatography on MIMETIC Red 3 to obtain a sufficient amount (79 $\mu$g) of homogeneous Ss5MaT1.

Ss5MaT1 showed a strong acyl donor preference for malonyl-CoA over CoA esters of the other aliphatic and aromatic acids, being consistent with the notion that it is a “malonyltransferase.” The calculated $k_{cat}/K_m$ values of native Ss5MaT1 for bisdemalonylsalvianin and shisonin were 0.077 and 0.108 $s^{-1} \mu M^{-1}$, respectively. Thus, the enzyme could catalyze the malonylation of both pelargonidin and cyanidin types of anthocyanins, despite the absence of cyanidin type anthocyanins in the *S. splendens* flowers used in this study. In this connection, many of the flavonoid glucosyltransferases and acyltransferases are known to have broad substrate specificity as far as the structure of the B-ring is concerned (6–8), and this is also the case with Ss5MaT1. The $K_m$ values of Ss5MaT1 for anthocyanin substrates were comparable with those of Pf3AT (6–624 $\mu M$ (8, 26)) and Gt5AT (113–180 $\mu M$ (7)). The important fact that the enzyme could not catalyze the malonylation of the 3,5-diglucosides of pelargonidin and delphinidin (Fig. 1, compounds 3a and 3b) suggests that the presence of an aromatic acyl group at the 3-glucosyl moiety of anthocyanin should be a

49018

Anthocyanin Malonyltransferase of Scarlet Sage Flowers

FIG. 5. Molecular phylogenetic tree of amino acid sequences of the versatile plant acyltransferase family. The tree was constructed by the neighbor-joining method. The lengths of lines indicate the relative distances between nodes. Enzymes used for alignment are Ss5MaT1 (GenBank™ accession number, AF405707; this study), Gt5AT (hydroxycinnamoyl-CoA:anthocyanin 5-O-glucoside-6"-O-acyltransferase of *G. triflora* (BAA74428 (7))), Pf3AT (hydroxycinnamoyl-CoA:anthocyanin 3-O-glucoside-6"-O-acyltransferase of *P. frutescens* (BAA93475 (8))), Pf5MaT (malonyl-CoA:anthocyanin 5-O-glucoside-6"-O-malonyltransferase of *P. frutescens* (AF405204)), ScAT8 (a putative anthocyanin acyltransferase of *S. cruentus* (E12757)), TcBAT (10-deacetylbaccatin III-10-O-acetyltransferase of *T. cuspidata* (AAF27621 (19))), TcTAT (taxadienol acetyltransferase of *T. cuspidata* (AAF34254 (20))), BEAT (acetyl-CoA:benzylalcohol acetyltransferase of *C. breweri* (AAC18062 (21))), DAT (deacetylvinidoline 4-O-acrtyltransferase of *C. roseus* (AAC99311 (9))), HCBT (hydroxycinnamoyl/benzoyl-CoA:anthranilate N-hydroxycinnamoyl/benzoyltransferase of *D. caryophyllus* (CAB06430 (22))), SalAT (salutaridiol 7-O-acetyltransferase of *P. somniferum* (AAK73661 (23))), AAT (alcohol acyltransferase of *F. xananassa* (AF193789 (24))), and NTSHR (Hsr201 protein of *Nicotiana tabacum* (T03274)). The following are putative acyltransferases whose biochemical functions are not clarified: *Arabidopsis thaliana* (BAB01191, BAB02518, BAB01203, BAB02519, BAB10378, BAB10831, BAB10830, BAB108329, BAB10449, AAB95283, and T00908); MEL2 of *Cucumis melo* (Z70521 (28)); PhAT48 of *Petunia hybrid*a (BAA93453 (8)); and GAT106 of *G. triflora* (BAA93452 (8)).

prerequisite for anthocyanin’s ability to act as a substrate for this enzyme. This was further confirmed by the fact that native and recombinant Ss5MaT1 are strongly inhibited by p-coumaric and caffeic acids. p-Coumaric acid was an uncompetitive inhibitor with respect to malonyl-CoA and a competitive inhibitor with respect to the substrate anthocyanin, suggesting that this inhibitor mimics the aromatic acyl moiety of the substrate anthocyanin and competes with it for binding to the enzyme. It should be mentioned that the observed types of inhibitions of Ss5MaT1 by p-coumaric acid were consistent with a sequential mechanism of Ss5MaT1-catalyzed acyl transfer. Detailed kinetic studies of the Ss5MaT1-catalyzed acyl transfer will be reported elsewhere.

Northern blot analysis showed that the expression of Ss5MaT1 was temporally regulated during the color development of the flower and was consistent with the role of Ss5MaT1 during coloration of the *S. splendens* flowers. The observed pattern of the temporal expression of Ss5MaT1 was consistent with our observation that Ss5MaT1 activities during the early stages of flower development were higher than those of fully opened flowers. These results indicate that anthocyanin biosynthesis is transcriptionally regulated also in *Salvia* as reported in many plant species.

The specificity of Ss5MaT1, along with the current knowledge on the pathway of shisonin biosynthesis in the leaves of *P. frutescens* (6), allowed us to propose the following order of late events of the salvianin biosynthesis in the *S. splendens* flowers (Fig. 3S). Pelargonidin 3-O-glucoside, which has been formed through the early stage of the biosynthesis, undergoes the 5-glucosylation and the 6"-acylation with a caffeoyl group probably through a “metabolic grid,” as shown in the shisonin biosynthesis (6). The product, bisdemalonylsalvianin, then undergoes the first malonylation at its 6"-position by the action of Ss5MaT1, followed by the second malonylation at its 4"-position by another malonyltransferase (termed Ss5MaT2) to produce salvianin.

Although anthocyanins are generally known to be accumulated in the vacuoles of plant cells (2), vacuolar localization of Ss5MaT1 is highly unlikely, considering the lack of a known signal sequence for the translocation to the vacuoles (27) in the deduced N-terminal amino acid sequence of Ss5MaT1, the suggested absence of the sugar chain in the native Ss5MaT1 molecule (data not shown), and the pH activity and pH stability profiles of Ss5MaT1, which are, instead, all consistent with the enzyme’s localization in the cytosol.

The deduced amino acid sequence of Ss5MaT1 shared the consensus sequences, -His-Xaa₃-Asp- (motif 1; Figs. 3 and 2S)

and -Asp-Phe-Gly-Trp-Gly- (motif 3), which have been identified in the sequences of members of the versatile plant acyltransferase family (9), indicating that Ss5MaT1 is also a member of this family. This family is a large protein family that is suggested to consist of acyltransferases with diverse biochemical functions in the secondary metabolisms in plants (10, 19–24). Biochemically characterized members of this family should represent only a small fraction of the total member of the family because it was recently estimated that *Arabidopsis* contains about 70 related genes of the family (10), and most of their biochemical roles are yet to be determined. The entire amino acid sequence of Ss5MaT1 showed only a limited similarity to those of the other members of this family. This, along with phylogenetic tree constructed on the basis of sequences of biochemically characterized members as well as those of functionally unidentified proteins of this family (Fig. 5), also corroborates that this family covers divergent members of acyltransferases with versatile biochemical roles in plants. It is worth noting that Ss5MaT1 shared a sequence, -Asn-Tyr-Phe-Gly-Asn-Cys- (motif 2; see Figs. 3 and 2S), which has uniquely been identified in the sequences of the members involved in the anthocyanin metabolism, such as anthocyanin aromatic acyltransferases from *P. frutescens* and *G. triflora* (*i.e.* Gt5AT (7) and Pf3AT (8), respectively), an anthocyanin malonyltransferase of *P. frutescens* (Fig. 2S), and a putative anthocyanin acyltransferase of *Sinecio cruentus* (Genbank™ accession number E12757).³ These results suggest that these aromatic and aliphatic acyltransferases for anthocyanins may be categorized into a subfamily of the versatile plant acyltransferase family (Fig. 5).

Finally, it must be mentioned that Ss5MaT1 cDNA obtained in this study will be an important tool for controlling flower colors by metabolic engineering of anthocyanin biosynthesis, because malonylated anthocyanins are ultimate stable forms of anthocyanins in many plant species.

---

³ H. Suzuki, T. Nakayama, K. Yonekura-Sakakibara, Y. Fukui, N. Nakamura, M. Nakao, Y. Tanaka, M. Yamaguchi, T. Kusumi, and T. Nishino, unpublished results.

---

**REFERENCES**

1. Strack, D., and Wray, V. (1994) in *The Flavonoids* (Harborne, J. B., ed) pp. 1–22, Chapman & Hall/CRC, Washington, D. C.
2. Brouillard, R., and Dangles, O. (1994) in *The Flavonoids* (Harborne, J. B., ed) pp. 565–588, Chapman & Hall/CRC, Washington, D. C.
3. Goto, T., and Kondo, T. (1991) *Angew. Chem. Int. Ed. Engl.* **30**, 17–33
4. Heller, W., and Forkmann, G. (1994) in *The Flavonoids* (Harborne, J. B., ed) pp. 499–535, Chapman & Hall/CRC, Washington, D. C.
5. Tanaka, Y., Tsuda, S., and Kusumi, T. (1998) *Plant Cell Physiol.* **39**, 1119–1126
6. Yamazaki, M., Gong, Z., Fukuchi-Mizutani, M., Fukui, Y., Tanaka, Y., Kusumi, T., and Saito, K. (1999) *J. Biol. Chem.* **274**, 7405–7411
7. Fujiwara, H., Tanaka, Y., Yonekura-Sakakibara, K., Fukuchi-Mizutani, M., Nakao, M., Fukui, Y., Yamaguchi, M., Ashikari, T., and Kusumi, T. (1998) *Plant J.* **16**, 421–431
8. Yonekura-Sakakibara, K., Tanaka, Y., Fukuchi-Mizutani, M., Fujiwara, H., Fukui, Y., Ashikari, T., Murakami, Y., Yamaguchi, M., and Kusumi, T. (2000) *Plant Cell Physiol.* **41**, 495–502
9. St-Pierre, B., Laflamme, P., Alarco, A.-M., and De Luca, V. (1998) *Plant J.* **14**, 703–713
10. Dudareva, N., and Pichersky, E. (2000) *Plant Physiol.* **122**, 627–633
11. Kondo, T., Yoshikane, M., Goto, T., and Yoshida, K. (1989) *Tetrahedron Lett.* **30**, 6729–6732
12. Takeda, K., Harborne, J. B., and Self, R. (1986) *Phytochemistry* **25**, 2191–2192
13. Nakayama, M., Koshioka, M., Shibata, M., Hiradate, S., Sugie, H., and Yamaguchi, M.-A. (1997) *Biosci. Biotech. Biochem.* **61**, 1607–1608
14. Leatherbarrow, R. J. (1990) *Trends Biochem. Sci.* **15**, 455–458
15. Bradford, M. M. (1976) *Anal. Biochem.* **72**, 248–254
16. Laemmli, U. K. (1970) *Nature* **227**, 680–685
17. McGookin, R. (1984) in *Methods in Molecular Biology* (Walker, J. M., ed) Vol. 2, pp. 113–116, Humana Press Inc., Totowa, NJ
18. Isegawa, Y., Sheng, J., Sokowa, Y., Yamanishi, K., Nakagomi, O., and Shigeharu, U. (1992) *Mol. Cell. Probes* **6**, 467–475
19. Walker, K., and Croteau, R. (2000) *Proc. Natl. Acad. Sci. U. S. A.* **97**, 583–587
20. Walker, K., Schoendorf, A., and Croteau, R. (2000) *Arch. Biochem. Biophys.* **374**, 371–380
21. Dudareva, N., D'Auria, J. C., Nam, K. H., Raguso, R. A., and Pichersky, E. (1998) *Plant J.* **14**, 297–304
22. Yang, Q., Reinhard, K., Schiltz, E., and Matern, U. (1997) *Plant Mol. Biol.* **35**, 777–789
23. Grothe, T., Lenz, R., and Kutchan, T. M. (2001) *J. Biol. Chem.* **276**, 30717–30723
24. Aharoni, A., Keizer, L. C., Bouwmeester, H. J., Sun, Z., Alvarez-Huerta, M., Verhoeven, H. A., Blaas, J., van Houwelingen, A. M., De Vos, R. C., van der Voet, H., Jansen, R. C., Guis, M., Mol, J., Davis, R. W., Schena, M., van Tunen, A. J., and O'Connell, A. P. (2000) *Plant Cell* **12**, 647–662
25. Kimura, M., Kaneko, E., Komiyama, M., Takatsuki, A., Koshinoi, H., Yoneyama, K., and Yamaguchi, I. (1998) *J. Biol. Chem.* **273**, 1654–1661
26. Fujiwara, H., Tanaka, Y., Fukui, Y., Ashikari, T., Yamaguchi, M., and Kusumi, T. (1998) *Plant Sci.* **137**, 87–94
27. Neuhaus, J.-M., and Rogers, J. C. (1998) *Plant Mol. Biol.* **37**, 127–144
28. Aggelis, A., John, I., Karvounis, Z., and Grierson, G. (1997) *Plant Mol. Biol.* **33**, 313–322
