
# Signaling networks in the plant circadian system

**Marcelo J Yanovsky* and Steve A Kay†**

Significant advances have been made during the past year in the genetic and molecular dissection of the plant circadian system. Several proteins involved in circadian clock regulation have been identified, and the way that their interactions contribute to temporal organization is starting to emerge. In addition, genomic approaches have identified hundreds of genes under clock control, providing a molecular basis to our understanding of how the clock coordinates plant physiology and development with daily and seasonal environmental cycles.

### Addresses

The Scripps Research Institute, 10550 North Torrey Pines Road, La Jolla, California 92037, USA  
*e-mail: yanovsky@scripps.edu*  
†e-mail: stevek@scripps.edu  

### Current Opinion in Plant Biology 2001, 4:429-435  
1369-5266/01/$—see front matter  
© 2001 Elsevier Science Ltd. All rights reserved.

### Abbreviations

- **CAB2**: CHLOROPHYLL A/B-BINDING PROTEIN 2
- **CCA1**: CIRCADIAN CLOCK ASSOCIATED 1
- **CO**: CONSTANS
- **cry**: cryptochrome
- **EE**: EVENING ELEMENT
- **ELF3**: EARLY FLOWERING 3
- **FKF1**: FLAVIN-BINDING, KELCH REPEAT, FBOX 1
- **GI**: GIGANTEA
- **LHY**: LATE ELONGATED HYPOCOTYL
- **LKP2**: LOV, KELCH PROTEIN 2
- **phy**: phytochrome
- **toc1**: timing of CAB expression 1
- **ZTL**: ZEITLUPE

### Introduction

Circadian systems are complex signaling networks that allow organisms to adjust cellular and physiological activities in anticipation of periodic changes in the environment. At the heart of the system, a central oscillator is responsible for generating circadian rhythms. Multiple pathways couple the oscillator to the physiological processes under its control and to the light and temperature cycles that adjust its pace on a daily basis [1].

This review summarizes the progress made during the past year in identifying the molecules that shape the plant circadian system and discusses how they interact to ensure proper coordination of events within the organism and between the organism and the environment.

### Molecular interactions at the heart of the Arabidopsis circadian system

In *Drosophila*, *Neurospora*, cyanobacteria, and mice, circadian clocks are based on transcriptional feedback loops in which cycling gene products negatively control their own expression by antagonizing the action of positive factors [1]. *Arabidopsis* does not contain true homologs of any of the clock genes found in these organisms, but forward and reverse genetic approaches are identifying molecules that are closely associated with the oscillator.

The timing of *CAB expression 1 (toc1)* mutant was the first circadian mutant to be characterized in *Arabidopsis* [2]. It exhibits a short period for all processes analyzed, and this phenotype is observed under a wide range of temperature and light conditions [3], even in complete darkness [4**]. This suggests that TOC1 is a component of the central oscillator, rather than part of the signaling pathways coupling the oscillator to the environment. TOC1 encodes a nuclear protein containing a receiver domain similar to that of response regulators from two-component signaling systems, and a basic motif, which is conserved within the *CONSTANS (CO)* family of transcription factors [4**]. Some of these features suggest a role for TOC1 in transcriptional regulation, but further characterization of the protein will be required to address this issue. Expression of *TOC1* cycles, peaking towards the end of the day. In addition, the period of this oscillation is shortened in the *toc1-1* mutant, indicating that TOC1 is part of an auto-regulatory feedback loop.

*CIRCADIAN CLOCK ASSOCIATED 1 (CCA1)* and *LATE ELONGATED HYPOCOTYL (LHY)* are two MYB-related transcription factors, originally identified for their effects on the light regulation of gene expression and flowering time [5, 6]. Interestingly, when the *CCA1* and *LHY* genes are overexpressed from a constitutive promoter, they repress their own transcription and induce a generalized arrhythmia, indicating that they are part of a negative feedback loop that is closely associated with the central oscillator [6, 7].

Are TOC1, CCA1, and LHY part of the same feedback loop? The *TOC1* transcript oscillates 12 hours out of phase with *LHY* and *CCA1* transcripts, and, given that the CCA1 protein cycles with little lag in relation to its transcript, CCA1 and LHY could act as repressors of *TOC1* [4**]. In agreement with this hypothesis, over-expression of either *LHY* or *CCA1* results in low and constant levels of *TOC1* transcript, suggesting that both transcription factors negatively regulate *TOC1* expression [8**]. Furthermore, they bind to a region of the *TOC1* promoter that is crucial for its circadian regulation. In addition, *LHY* and *CCA1* transcript levels are strongly reduced in the *toc1-2* mutant, presumably a loss-of-function allele, suggesting that TOC1 acts as a positive regulator of both genes [8**]. The reciprocal regulation of these genes, therefore, forms a negative feedback loop that is crucial for clock function in *Arabidopsis* (Figure 1). The fact that *LHY* and *CCA1* expression still oscillates in the *toc1* mutants, although with shortened period [8**], suggests that some *TOC1*-like genes [4**, 9*, 10*] could act redundantly with *TOC1* in the control of their expression. Finding out how TOC1 regulates *CCA1* and *LHY* expression, and which

430 Cell signalling and gene regulation

Figure 1

Molecular interactions within the *Arabidopsis* circadian system. The effects of light/dark cycles on the phase and period of the oscillator are mediated by multiple photoreceptors, including phytochromes (phys) A, B, D, and E, cryptochromes (crys) 1 and 2, and molecules such as ZTL, FKF1 and LKP2, which might function as either novel photoreceptors or photo-signaling components that target clock-associated proteins for degradation. Light effects on the clock are also modulated by the clock itself through its regulation of both positive and negative light-signaling components such as GI and ELF3. At the heart of the system, a feedback loop generated by the reciprocal regulation between the MYB-related transcription factors CCA1 and LHY, and the pseudo-response regulator TOC1 is responsible for generating circadian rhythms. These rhythms result in part from the direct transcriptional regulation of genes outside the oscillator by CCA1 and LHY. These MYB factors could function as activators of genes peaking at dawn and repressors of genes peaking at dusk, through their binding to the EE present in the promoter of these genes. Finally, clock-controlled expression of genes involved in the photosynthetic process, lipid metabolism, carbon and nitrogen assimilation, cell elongation, light-signaling pathways and flowering-time regulation, whether direct or not, provides the organism with the ability to anticipate and adapt itself to periodic changes in the environment. The gray shaded circle encompasses the feedback loop that is central for clock function in *Arabidopsis*. Broken arrows indicate unknown regulatory mechanisms. CHS, CHALCONE SYNTHASE; EXP, EXPANSIN.

genes in addition to TOC1 contribute to their circadian regulation, will be important to fully understand the molecular basis of the critical negative feed-back loop generated by this group of genes.

In addition to transcriptional regulation, post-transcriptional modifications such as phosphorylation and regulation of protein stability are also emerging as critical control steps in the plant circadian oscillator. CKB3, a subunit of casein kinase II, interacts with CCA1 *in vitro* [11], and its over-expression shortens the period of multiple circadian rhythms [12]. More recently, forward and reverse genetic screens have uncovered a family of proteins that are likely to target components of the circadian system for degradation (see below).

### Interactive signaling between photo-transduction pathways and the clock

Light controls the period and phase of the oscillator, allowing precise phase relationships between clock-controlled processes and the solar cycle to be maintained throughout the year [13]. *Arabidopsis* has five phytochromes (phyA to phyE), and four of them have been implicated in clock entrainment [13]. PhyA mediates clock responses to very low fluences of red and blue light and also to high fluences of far-red light, whilst phyB, phyD and phyE act at high fluences of red light [14, 15**, 16]. The blue-light photoreceptor, cryptochrome (cry), controls clock function not only in *Arabidopsis* but also in insects and mammals [13]. In plants, cry1 acts as a clock photoreceptor for high and low fluences of blue light, and cry1 and cry2 act redundantly at intermediate fluences [14, 15**]. In mice, the double *cry1 cry2* mutant is arrhythmic, suggesting that cryptochromes are central components of the mammalian clock [13]. In *Arabidopsis*, mutants lacking *cry1* and *cry2*, and even quadruple mutants lacking *phyA, phyB, cry1* and *cry2*, retain robust circadian rhythmicity, indicating that cryptochromes and phytochromes are not essential components of the oscillator [15**, 17**]. Surprisingly, *cry1* mutants and *cry1 cry2* double mutants also exhibit altered entrainment under low fluences of red light [15**]. As cryptochromes do not absorb red light, they are likely to be acting as intermediates in phytochrome signaling to the clock. Mutations in *cry2* also affect entrainment at intermediate fluences of

white light but have negligible effects under blue or red
light alone, suggesting that cry2 must interact with activated
phytochromes to be fully effective [18**]. Indeed, using
fluorescent resonance energy transfer microscopy, phyB
and cry2 have been shown to interact *in vivo*, co-localizing
in nuclear speckles in response to light treatments [18**].

In *Neurospora*, mice and *Drosophila*, light resetting of the
clock ultimately results from the regulation of gene
expression of a clock component or its degradation [13]. In
etiolated seedlings, *CCA1* and *LHY* expression is regulated
by light through the interaction of phyB with PHYTO-
CHROME INTERACTING FACTOR3 (PIF3), a tran-
scription factor that binds to the promoter of these genes
[19**]. Whether this regulation mediates clock entrainment,
however, has not been explicitly tested.

Forward and reverse genetic approaches in *Arabidopsis* have
recently identified ZEITLUPE (*ZTL*), FLAVIN-BINDING,
KELCH REPEAT, F-BOX1 (*FKF1*) and LOV, KELCH
PROTEIN2 (*LKP2*) [20**, 21**, 22*]. These genes encode
proteins that share a unique combination of motifs, which
include a PAS domain, an F-box domain and six kelch
repeats. PAS domains have been shown to mediate pro-
tein–protein interactions and the binding of chromophores
to some photoreceptors [23]. F-boxes have been implicated
in the recruitment of proteins for ubiquitination and sub-
sequent degradation [24]. The kelch repeats are likely to
mediate the specific interaction of these proteins with their
targets [25]. The *ztl* mutant is a long period mutant, the
phenotype of which depends upon the quantity of light
[20**]. The *fkf1* mutant was identified by its late flowering
phenotype but it also has minor circadian defects [21**].
Overexpression of *LKP2* results in arrhythmic leaf movement
and the expression of several clock-controlled genes
(T Schultz, T Kiyosue, MJ Yanovsky, M Wada, SA Kay,
unpublished data). The structural characteristics of these
proteins, together with their mutant and overexpressor
phenotypes, suggest that they could act as novel photo-
receptors that target clock components for degradation.
However, whether they bind chromophores and function as
photoreceptors themselves is still unknown. Alternatively,
the involvement of these proteins in photoentrainment
could result from their direct interaction with known photo-
receptors. Indeed, *ZTL* has been shown to interact *in vitro*
with the carboxyl terminus of *PHYB* and *CRY1* [26*];
however, it remains to be determined how this interaction
relates to the function of these proteins *in vivo*.

The effectiveness of light signaling pathways is rhythmically
controlled, adding another level of complexity to the
mechanism of clock entrainment. The acute induction of
CHLOROPHYLL A/B-BINDING PROTEIN2 (*CAB2*) gene
expression by a light pulse, for instance, has been shown to
follow a circadian rhythm, a phenomenon also known as
gating [27]. There is increasing evidence that this phe-
nomenon results from the circadian regulation of positive
and negative elements of the light-signaling pathway. The

expression of several photoreceptors is circadianly regulated
[28, 29**]. Another light-signaling component regulated by
the clock at the transcriptional level is GIGANTEA (GI),
a nuclear protein that affects flowering time, circadian
rhythms and hypocotyl elongation [30, 31, 32*]. The circadian
phenotype of *gi-1* is fluence-rate dependent and a novel
allele was identified in a screening for plants showing long
hypocotyls under red light, suggesting that GI is a shared
component of phyB signaling pathways to the clock and
other developmental processes [30, 32*]. Recent studies also
show that EARLY FLOWERING 3 (*ELF3*) is an important
component in the gating process. The *elf3* mutant, identified
by its early flowering phenotype, is arrhythmic in continuous
light but not in darkness [33, 34]. McWatters *et al.* [35**]
have shown that gating of light induction of *CAB2* is absent
in *elf3* mutants and the conditional arrhythmia results from
a direct effect of light on the clock itself. *ELF3* has been
cloned and encodes a protein with no detectable homology
to proteins with known function in the database [36*]. Both
the transcript and the protein cycle, peaking during
the night [36*, 37**]. A role for ELF3 in antagonizing light
signaling pathways has been suggested by Covington *et al.*
[38**], who observed that light induction of *CAB2* is enhanced
in *elf3* mutants. More interestingly, ELF3 antagonizes light
effects on the clock itself, as *elf3* mutants become arrhythmic
in continuous darkness after exposure to light pulses in the
middle of the night, a treatment that induces maximal
phase-shifts in the wild type [38**]. In agreement with
these observations, overexpression of *ELF3* attenuates light
induction of *CAB2*, increases period length in a light-
dependent manner, and reduces the magnitude of
phase-shifts elicited by light pulses [38**]. *ELF3* directly
interacts with phyB *in vitro*, suggesting a mechanistic basis
for its effect on light signaling [37**]. The phenotype of
*elf3 phyB* double mutants in some conditions is stronger
than that of the single mutants, indicating that ELF3 also
affects the actions of other photoreceptors [37**, 39*]. Together, the circadian regulation of *ELF3* expression and
the negative effect of ELF3 on light signaling to the clock
constitute a mechanism by which the clock modulates its
own resetting by light (Figure 1).

### From clock control of gene expression to physiological and developmental rhythms

The temporal organization of metabolic and developmental
activities results, at least in part, from the control of gene
expression. Genome-wide expression analyses are expand-
ing the number of activities known to be under clock
control and revealing the molecular rhythms underlying
physiological oscillations [29**, 40**, 41]. Using oligonu-
cleotide microarrays representing 8200 different genes,
Harmer *et al.* [29**] found 453 genes that are regulated by
the clock. Functional clustering of the genes on the basis of
their known or predicted physiological functions revealed
several examples of how the clock helps plants to anticipate
and adapt to the daily and seasonal fluctuations in light and
temperature. Aside from the known circadian regulation of
photosynthetic genes, the expression of 23 genes encoding

enzymes involved in the biosynthesis of photoprotective
pigments peaks coordinately before dawn, presumably to
protect plants from the damaging effects of UV-B [29**]. A
probable candidate to mediate communication between
the clock and these genes is PAP1, a transcription factor
cycling with the same phase, whose overexpression results
in enhanced accumulation of flavonoids [42]. Another inter-
esting cluster of coordinately expressed genes represents a
group involved in lipid modification; the expression of
these genes peaks just before dusk [29**]. The products of
these genes could be responsible for altering desaturation
levels of lipids, which have been shown to oscillate and
correlate with chilling tolerance.

Clock control of gene expression is also an important
component of the photoperiodic timing mechanism.
Photoperiodic time measurement, or day-length perception,
allows organisms to anticipate seasonal changes and to
coordinate their development with their environment
accordingly [43]. Several genes affecting photoperiodic
responses have been shown to be under clock control
[6,7,21**,30,31,36*,44**,45*]. Expression of CO, for instance,
oscillates under light/dark cycles and the waveform of
the oscillation changes with the photoperiod, ensuring a
coincidence between the presence of light and high levels
of CO under long days but not short days [44**]. CO, there-
fore, is likely to act at a point at which the action of light
and the clock are integrated to measure day-length.
Interestingly, two Arabidopsis genes isolated on the basis of
their similarity to CO, CONSTANS-LIKE (COL) 1 and 2,
are also under clock control. Overexpression of COL1,
unlike that of CO, has little effect on flowering time but
shortens the circadian period of CAB2 expression in a
light-dependent manner, suggesting a role for COL1 in
light signaling to the clock [46*].

Microarray experiments combined with computational
analysis can also help identify regulatory components of
the circadian system. For instance, Schaffer et al. [40**] have
analyzed cDNA microarrays representing 7800 unique
genes and found that 11% of the genes were regulated
diurnally and 2% were under clock control. By comparing
their data with 47 other unrelated experiments, they
identified a cluster of genes mainly regulated by the clock
[40**]. This cluster is likely to include novel proteins
important for clock regulation, as the expression of these
genes should not be affected by environmental conditions
other than those involved in clock entrainment. Another
approach was used by Harmer et al. [29**] who, searching
the upstream regions of co-regulated genes for common
motifs, found a nine base-pair EVENING ELEMENT (EE)
over-represented in a cluster of 31 genes whose expression
peaked at the end of the day. Mutational analysis confirmed
the importance of the EE for robust cycling of COLD,
CIRCADIAN RHYTHM 2 (CCR2) [29**], a gene encoding a
glycine-rich RNA-binding protein that is regulated by the
circadian clock and by cold treatments. This is in agreement
with the observation that a 56-base-pair region of the CCR2

promoter containing the EE confers circadian rhythmicity
to the β-glucuronidase reporter gene [47]. The sequence of
the EE (AAAATATCT) is identical to that recognized by
CCA1 and LHY in the TOC1 promoter [8**], where they
act as transcriptional repressors. The sequence is also
highly similar to that recognized by CCA1 in the promoter
of the CAB1 gene (AAAAATCT), where it acts as a positive
regulator [5]. This strongly suggests that CCA1 and LHY
directly control the expression of groups of genes with
opposite phases, and also reveals that the same mechanism
that regulates rhythmic transcription within the central
feedback loop is used to control the rhythmic expression of
genes outside the oscillator (Figure 1).

Second messengers such as Ca²⁺ are also likely to play a
regulatory role transducing timing information from the
clock to different Ca²⁺-sensitive processes. Interestingly,
cytosolic but not nuclear Ca²⁺ concentrations have been
shown to oscillate, suggesting that if Ca²⁺ oscillations play
a role in transcriptional regulation they might operate
through a cytoplasmic transduction mechanism [48*].

# The circadian system at the organismal level

The organization of the circadian system at the whole-plant
level has only recently started to be addressed. For instance,
how are the clocks of individual cells synchronized to
achieve the coordinated actions required for rhythmic
processes at the tissue or organ level? Thain et al. [49**]
have recently shown that the rhythmic expression of a
given gene in a single plant can be set simultaneously to
different phases in various anatomical locations and even
within different areas of the same leaf using localized
light/dark treatments. The initial phases are maintained
after ending the entraining treatments, which indicates
that the clock within each cell is entrained and functions
independently of those in neighboring cells. This demon-
strates that plants, unlike mammals, lack a central
pacemaker organ synchronizing the clocks of peripheral
tissues [49**]. Therefore, the coordinated activities of
plants must result from the simultaneous but individual
entrainment of each cell by environmental cues.

# Conclusions and perspectives

Our knowledge of the plant circadian system has increased
significantly in recent years. The reciprocal regulation
between TOC1, CCA1 and LHY constitutes a negative
feedback loop that is central to clock function, revealing
that the overall architecture of the plant circadian system is
similar to that of other organisms, although the individual
components are distinctly different. In addition, phyto-
chromes, cryptochromes and novel light-signaling
elements have been shown to couple the oscillator to
light/dark cycles, providing a remarkable homeostasis in
clock entrainment that allows plants to cope with the
varying light environments they experience throughout
their life cycle. Genome-wide expression analyses are
also allowing dramatic advances in our understanding of
how the clock ultimately regulates physiological and

developmental processes. Like the clock in mammals [50], the Arabidopsis clock appears to utilize components of the central feedback loop to control the expression of genes associated with rhythms in different biological activities. Finding the regulatory elements responsible for expressing genes at different phases will be an important step towards understanding the mechanism of timing physiological processes throughout the entire day.

**Update**

Two genes that play a role in daily and seasonal timing have recently been identified. ZGT is a novel clock and light regulated gene from tobacco whose over-expression increases the sensitivity of plants to light pulses that reset the clock, and to sustained oscillation of LHCB1*1 expression in constant darkness [51•]. Hd6 was originally identified as a minor quantitative trait locus (QTL) affecting photoperiodic sensitivity in rice. Hd6 was cloned and shown to encode the subunit of CK2, providing further evidence for a role of CK2 in biological timing [52••].

**Acknowledgements**

Thanks to S Harmer, J Casal, F Harmon, D Alabadi and F Ceriani for critical reading of this review.

**References and recommended reading**

Papers of particular interest, published within the annual period of review, have been highlighted as:

- • of special interest
- •• of outstanding interest

1. Dunlap JC: Molecular bases for circadian clocks. *Cell* 1999, 96:271-290.
2. Millar AJ, Carré IA, Strayer CA, Chua NH, Kay SA: Circadian clock mutants in *Arabidopsis* identified by luciferase imaging. *Science* 1995, 267:1161-1163.
3. Somers DE, Webb AAR, Pearson M, Kay S: The short-period mutant, *toc1-1*, alters circadian clock regulation of multiple outputs throughout development in *Arabidopsis thaliana*. *Development* 1998, 125:485-494.
4. Strayer CA, Oyama T, Schultz TF, Raman R, Somers DE, Mas P, Panda S, Kreps JA, Kay SA: Cloning of the *Arabidopsis* clock gene *TOC1*, an autoregulatory response regulator homolog. *Science* 2000, 289:768-771. The authors of this paper describe the cloning of *TOC1* and extend the phenotypic characterization of the mutant. The *toc1* mutant shortens period length in complete darkness, indicating that *TOC1* acts very close to the central pacemaker, rather than on a light-input pathway to the clock. They also show that *TOC1* effects on flowering time are a direct consequence of its circadian phenotype. *TOC1* encodes a nuclear protein with structural motifs, suggesting a role in transcriptional regulation. *TOC1* is regulated circadianly and this regulation is altered in the *toc1* mutant, implicating *TOC1* in a feedback loop controlling its own expression.
5. Wang ZY, Kenigsbuch D, Sun L, Harel E, Ong MS, Tobin EM: A Myb-related transcription factor is involved in the phytochrome regulation of an *Arabidopsis Lhcb* gene. *Plant Cell* 1997, 9:491-507.
6. Schaffer R, Ramsay N, Samach A, Corden S, Putterill J, Carré IA, Coupland G: The late elongated hypocotyl mutation of *Arabidopsis* disrupts circadian rhythms and the photoperiodic control of flowering. *Cell* 1998, 93:1219-1229.
7. Wang ZY, Tobin EM: Constitutive expression of the circadian clock associated 1 (*CCA1*) gene disrupts circadian rhythms and suppresses its own expression. *Cell* 1998, 93:1207-1217.
8. Alabadi D, Oyama T, Yanovsky MJ, Harmon FG, Mas P, Kay SA: Reciprocal regulation between *TOC1* and *LHY/CCA1* within the *Arabidopsis* circadian clock. *Science* 2001, in press. The interactions between the clock genes *LHY*, *CCA1*, and *TOC1* are described for the first time. *LHY* and *CCA1* negatively regulate *TOC1* expression through their binding to the EE present in the *TOC1* promoter. In addition, *TOC1* appears to positively regulate *LHY* and *CCA1* expression. These interactions define a negative feedback loop that is central to clock function in *Arabidopsis*.

9. Makino S, Kiba T, Imamura A, Hanaki N, Nakamura A, Suzuki T, Taniguchi M, Ueguchi C, Sugiyama T, Mizuno T: Genes encoding pseudo-response regulators: insight into His-to-Asp phosphorelay and circadian rhythm in *Arabidopsis thaliana*. *Plant Cell Physiol* 2000, 41:791-803. The authors characterized a family of *Arabidopsis* pseudo-response regulators that lack the invariant phospho-accepting aspartate site involved in the phosphorelay mechanism, one of which is identical to *TOC1*.
10. Matsushika A, Makino S, Kojima M, Mizuno T: Circadian waves of expression of the *APRR1/TOC1* family of pseudo-response regulators in *Arabidopsis thaliana*: insight into the plant circadian clock. *Plant Cell Physiol* 2000, 41:1002-1012. This paper characterizes the expression of the five members of the *TOC1/APRR1* family of pseudo-response regulators. The five genes are circadianly regulated at the transcript level and sequentially expressed throughout the day. Whether members of this family other than *TOC1/APRR1* have a role in clock regulation remains to be tested.
11. Sugano S, Andronis C, Green RM, Wang ZY, Tobin EM: Protein kinase CK2 interacts with and phosphorylates the *Arabidopsis* circadian clock-associated protein. *Proc Natl Acad Sci USA* 1998, 95:11020-11025.
12. Sugano S, Andronis C, Ong MS, Green RM, Tobin EM: The protein kinase CK2 is involved in regulation of circadian rhythms in *Arabidopsis*. *Proc Natl Acad Sci USA* 1999, 96:12362-12366.
13. Devlin PF, Kay S: Circadian photoperception. *Annu Rev Physiol* 2001, 63:677-694.
14. Somers DE, Devlin PF, Kay SA: Phytochromes and cryptochromes in the entrainment of the *Arabidopsis* circadian clock. *Science* 1998, 282:1488-1490.
15. Devlin PF, Kay SA: Cryptochromes are required for phytochrome signaling to the circadian clock but not for rhythmicity. *Plant Cell* 2000, 12:2499-2510. This paper demonstrates that phyD and phyE are involved in red-light input to the clock and that cry1 and cry2 act redundantly at intermediate fluences of blue light. More interestingly, it also shows that cryptochrome mutants are defective in red-light signaling to the clock, implying that they are part of a phytochrome signaling pathway that mediates photoentrainment.
16. Yanovsky MJ, Izaguirre M, Wagmaister JA, Gatz C, Jackson SD, Thomas B, Casal JJ: Phytochrome A resets the circadian clock and delays tuber formation under long days in potato. *Plant J* 2000, 23:223-232.
17. Yanovsky MJ, Mazzella MA, Casal JJ: A quadruple photoreceptor mutant still keeps track of time. *Curr Biol* 2000, 10:1013-1015. An *Arabidopsis* quadruple mutant lacking cry1, cry2, phyA and phyB retains robust circadian rhythmicity and the ability to be entrained by light/dark cycles. Other photoreceptors, probably phyD and/or phyE or members of the ZTL family, therefore regulate the pace of the clock in the absence of the four major photoreceptors.
18. Mas P, Devlin PF, Panda S, Kay SA: Functional interaction of phytochrome B and cryptochrome 2. *Nature* 2000, 408:207-211. Using fluorescent resonance energy transfer microscopy and co-immunoprecipitation, the authors demonstrate that the interactions between cry2 and phyB observed for the control of flowering time, hypocotyl elongation and circadian rhythms are mediated by a light-dependent physical association of these two photoreceptors.
19. Martinez-Garcia JF, Huq E, Quail PH: Direct targeting of light signals to a promoter element-bound transcription factor. *Science* 2000, 288:859-863. PIF3 is shown to bind specifically to the G-boxes present in various light-regulated promoters, including those of *CCA1* and *LHY*, and light induction of these genes is reduced in PIF3-antisense plants. Whether PIF3 mediates clock entrainment remains to be tested.
20. Somers DE, Schultz TF, Milnamow M, Kay SA: Zeitlupe encodes a novel clock-associated PAS protein from *Arabidopsis*. *Cell* 2000, 101:319-329. The *ztl* mutant lengthens the circadian period of multiple rhythms and affects photoperiodic time measurement. The period phenotype is dependent on wavelength and fluence rate, suggesting a role for ZTL in light signaling to the clock. ZTL encodes a protein containing a PAS domain, an F-box region and six terminal Kelch repeats, suggesting that it may function as a light-regulated switch that triggers clock-associated proteins for degradation.

434 Cell signalling and gene regulation

21. Nelson DC, Lasswell J, Rogg LE, Cohen MA, Bartel B: FKF1, a clock-controlled gene that regulates the transition to flowering in *Arabidopsis*. *Cell* 2000, 101:331-340.

The FKF1 gene was identified by its role in flowering time determination and encodes a protein that is closely related to ZTL and LKP2 [20**]. Besides its effect on flowering time, FKF1 affects hypocotyl elongation and the pattern of expression of some clock-controlled genes. FKF1, in contrast to ZTL and LKP2, is regulated circadianly.

22. Kiyosue T, Wada M: LKP1 (LOV kelch protein 1): a factor involved in the regulation of flowering time in *Arabidopsis*. *Plant J* 2000, 23:807-815.

LKP1 (which is identical to ZTL) was identified by the similarity of its LOV domain to that of NPH1, the photoreceptor mediating phototropic responses in *Arabidopsis*. Plants overexpressing LKP1/ZTL are late flowering in long days and have long hypocotyls. The reason why ztl mutants and plants over-expressing LKP1/ZTL are both late flowering is unclear.

23. Gu YZ, Hogenesch JB, Bradfield CA: The PAS superfamily: sensors of environmental and developmental signals. *Annu Rev Pharmacol Toxicol* 2000, 40:519-561.

24. Callis J, Vierstra RD: Protein degradation in signaling. *Curr Opin Plant Biol* 2000, 3:381-386.

25. Adams J, Kelso R, Cooley L: The kelch repeat superfamily of proteins: propellers of cell function. *Trends Cell Biol* 2000, 10:17-24.

26. Jarillo JA, Capel J, Tang RH, Yang HQ, Alonso JM, Ecker JR, Cashmore AR: An *Arabidopsis* circadian clock component interacts with both CRY1 and phyB. *Nature* 2001, 410:487-490.

The authors report the phenotypic characterization of a novel allele of ZTL [20**], called *ado1* here, resulting from a T-DNA insertion into the ZTL gene. The mutant is long period for leaf movement and gene expression in white and blue light, and arrhythmic for leaf movement in red light. ZTL is shown to interact in vitro with the carboxyl terminus of phyB and cry1.

27. Millar AJ, Kay SA: Integration of circadian and phototransduction pathways in the network controlling CAB gene transcription in *Arabidopsis*. *Proc Natl Acad Sci USA* 1996, 93:15491-15496.

28. Bognar LK, Hall A, Adam E, Thain SC, Nagy F, Millar AJ: The circadian clock controls the expression pattern of the circadian input photoreceptor, phytochrome B. *Proc Natl Acad Sci USA* 1999, 96:14652-14657.

29. Harmer SL, Hogenesch JB, Straume M, Chang HS, Han B, Zhu T, Wang X, Kreps JA, Kay SA: Orchestrated transcription of key pathways in *Arabidopsis* by the circadian clock. *Science* 2000, 290:2110-2113.

Using oligonucleotide-based microarrays, this study identified 453 genes that are regulated by the circadian clock, including clusters of genes involved in the photosynthetic process, lipid metabolism, carbon and nitrogen assimilation and partitioning, cell elongation, light-signaling pathways and flowering-time regulation. Sequence analysis followed up by mutational analysis allowed the identification of a common regulatory sequence in the promoters of evening-phased genes, which is critical for the circadian regulation of gene expression.

30. Park DH, Somers DE, Kim YS, Choy YH, Lim HK, Soh MS, Kim HJ, Kay SA, Nam HG: Control of circadian rhythms and photoperiodic flowering by the *Arabidopsis* GIGANTEA gene. *Science* 1999, 285:1579-1582.

31. Fowler S, Lee K, Onouchi H, Samach A, Richardson K, Morris B, Coupland G, Putterill J: GIGANTEA: a circadian clock-controlled gene that regulates photoperiodic flowering in *Arabidopsis* and encodes a protein with several possible membrane-spanning domains. *EMBO J* 1999, 18:4679-4688.

32. Huq E, Tepperman JM, Quail PH: GIGANTEA is a nuclear protein involved in phytochrome signaling in *Arabidopsis*. *Proc Natl Acad Sci USA* 2000, 97:9789-9794.

A novel allele of *gi* (*gi*-100) was identified in a screen for mutants with long hypocotyls under red light. This and other *gi* alleles display reduced de-etiolation under red, but not far-red light, suggesting that GI is a specific component of a phyB signaling pathway. In contrast to previous computer-based predictions, GI-reporter protein fusions are constitutively targeted to the nucleus rather than to plasmatic-membranes.

33. Hicks KA, Millar AJ, Carré IA, Somers DE, Straume M, Meeks-Wagner R, Kay SA: Conditional circadian dysfunction of the *Arabidopsis early-flowering 3* mutant. *Science* 1996, 274:790-792.

34. Zagotta MT, Hicks KA, Jacobs CI, Young JC, Hangarter RP, Meeks-Wagner DR: The *Arabidopsis ELF3* gene regulates vegetative photomorphogenesis and the photoperiodic induction of flowering. *Plant J* 1996, 10:691-702.

35. McWatters HG, Bastow RM, Hall A, Millar AJ: The ELF3 zeitnehmer regulates light signaling to the circadian clock. *Nature* 2000, 408:716-720.

This paper demonstrates that ELF3 is involved in gating the light signaling pathways that regulate CAB2 expression and suggests that the absence of gating of light effects on the clock itself is the basis for the arrhythmic phenotype of the *elf3* mutant. Under certain conditions, temperature cycles can entrain the clock of *elf3* mutants, implying that ELF3 specifically affects the light-input pathway.

36. Hicks KA, Albertson TM, Foss H, Wagner RD: *Early flowering 3* (*ELF3*) encodes a novel gene that regulates circadian clock function and flowering in *Arabidopsis*. *Plant Cell* 2001, 13:1281-1292.

This paper describes the cloning of *ELF3* by positional methods. *ELF3* encodes a novel 695-amino-acid protein whose transcript is under circadian control. Surprisingly, *ELF3* transcript still oscillates in the *lhy* mutant under constant light conditions, suggesting that the clock controlling its expression might be different from that associated with *LHY*.

37. Liu XL, Covington MF, Fankhauser C, Chory J, Wagner DR: *ELF3* encodes a circadian-regulated nuclear protein that functions in an *Arabidopsis PHYB* signal transduction pathway. *Plant Cell* 2001, 13:1293-1304.

This paper shows that the ELF3 protein is rhythmically controlled and follows a pattern of expression similar to that of the *ELF3* transcript, peaking during the night. In addition, it shows that ELF3 and phyB interact in the yeast two hybrid assay and in vitro, suggesting that a direct association of these molecules facilitates their interaction in the control of early photomorphogenic events and the circadian clock.

38. Covington MF, Panda S, Liu XL, Strayer CA, Wagner DR, Kay SA: *ELF3* modulates resetting of the circadian clock in *Arabidopsis*. *Plant Cell* 2001, 13:1305-1316.

The authors show that *elf3* mutants become arrhythmic after exposure to light pulses that elicit maximal phase-shifts in wild-type plants, whereas plants that overexpress *ELF3* show reduced phase shifts. Overexpression of *ELF3* also lengthens period under high fluences of light, which is consistent with a role for *ELF3* in antagonizing light signaling to the clock. In addition, light induction of *CAB2* is enhanced in *elf3* mutants and reduced in *ELF3* over-expressors, supporting the idea that *ELF3* acts, at least in part, by antagonizing light effects on the clock and other developmental processes.

39. Reed JW, Nagpal P, Bastow RM, Solomon KS, Dowson-Day MJ, Elumalai RP, Millar AJ: Independent action of *ELF3* and phyB to control hypocotyl elongation and flowering time. *Plant Physiol* 2000, 122:1149-1160.

Two new *elf3* alleles were identified for their long hypocotyls in red light. The phenotype of the *elf3 phyB* double mutant is stronger than that of the single mutants, indicating that the effect of *ELF3* is, at least in part, independent of phyB action.

40. Schaffer R, Landgraf J, Accerbi M, Simon V, Larson M, Wisman E: Microarray analysis of diurnal and circadian-regulated genes in *Arabidopsis*. *Plant Cell* 2001, 13:113-123.

The authors used cDNA microarrays to analyze diurnal and circadian regulation of gene expression in *Arabidopsis* and, by comparing their data with those of 47 other experiments, they identified a cluster of genes that are mainly regulated by the clock, which might lead to the discovery of novel genes involved in clock regulation.

41. Kreps JA, Muramatsu T, Furuya M, Kay SA: Fluorescent differential display identifies circadian clock-regulated genes in *Arabidopsis thaliana*. *J Biol Rhythms* 2000, 15:208-217.

42. Borevitz JO, Xia Y, Blount J, Dixon RA, Lamb C: Activation tagging identifies a conserved MYB regulator of phenylpropanoid biosynthesis. *Plant Cell* 2000, 12:2383-2394.

43. Samach A, Coupland G: Time measurement and the control of flowering in plants. *Bioessays* 2000, 22:38-47.

44. Suarez-Lopez P, Wheatley K, Robson F, Onuchi H, Valverde F, Coupland G: CONSTANS mediates between the circadian clock and the control of flowering in *Arabidopsis*. *Nature* 2001, 410:1116-1120.

This paper reports the circadian expression of *CO*, a gene required for the promotion of flowering in *Arabidopsis* under long days. The waveform of *CO* expression is modulated by day-length in such a way that there is a coincidence between peak levels of *CO* mRNA and light under long days but not under short days. This result strongly suggests that *CO* could be a key component of the molecular rhythm of sensitivity to light that is responsible for photoperiodic time measurement.

45. Liu J, Yu J, McIntosh L, Kende H, Zeevaart JAD: Isolation of a CONSTANS ortholog from *Pharbitis nil* and its role in flowering. *Plant Physiol* 2001, 125:1821-1830.

The *Pharbitis nil* ortholog of *CONSTANS* (*CO*) was identified in an attempt to isolate genes whose transcript levels were increased under short-day conditions that promoted the induction of flowering. Similar to the situation in *Arabidopsis*, *CO* expression is under circadian control in *Pharbitis nil*.
A detailed kinetic analysis is required to determine whether different levels of CO accumulate under short compared to long days, or whether the different pattern of expression is due to differential entrainment under different photoperiods.

46. Ledger S, Strayer C, Ashton F, Kay SA, Putterill J: Analysis of the function of two circadian-regulated CONSTANS-LIKE genes. *Plant J* 2001, 26:15-22.

COL1 and COL2, two genes identified for their similarity to CO, are shown to be under circadian control at the mRNA level. Overexpression of neither COL1 nor COL2 alters flowering time, but overexpression of COL1 shortens the period of the leaf movement and CAB2 expression rhythms. This phenotype is dependent on fluence rate, suggesting a role for COL1 in the light-signaling pathway to the clock.

47. Staiger D, Apel K: Circadian clock-regulated expression of an RNA-binding protein in *Arabidopsis*: characterization of a minimal promoter element. *Mol Gen Genet* 1999, 261:811-819.

48. Wood NT, Haley A, Viry-Moussaid M, Johnson CH, van der Luit AH, Trewavas A: The calcium rhythms of different cell types oscillate with different circadian phases. *Plant Physiol* 2001, 125:787-796.

By analyzing plants transformed with aequorin under the control of different promoters or fused to nucleoplasmin, the authors conclude that cytosolic but not nuclear Ca²⁺ concentrations exhibit circadian rhythmicity and that Ca²⁺ concentration oscillates with different phases in different tissues.

49. Thain SC, Hall A, Millar AJ: Functional independence of circadian clocks that regulate plant gene expression. *Curr Biol* 2000, 10:951-956.

Following circadian rhythms in luciferase activity driven by different promoters and in different tissues types, in combination with localized light/dark treatments,

these authors show that the plant circadian system is composed of multiple autonomous circadian clocks distributed throughout the plant.

50. Jin J, Shearman LP, Weaver DR, Zylka MJ, De Vries GJ, Reppert SM: A molecular mechanism regulating rhythmic output from the suprachiasmatic circadian clock. *Cell* 1999, 96:57-68.

51. Xu Y, Johnson CH: A clock and light-regulated gene that links the circadian oscillator to LHCB gene expression. *Plant Cell* 2001, 13:1411-1425.

The authors of this paper describe the cloning of a novel clock- and light-regulated gene from tobacco called ZGT. ZGT encodes a novel protein with no similarity to proteins with known function in the database. Plants overexpressing ZGT have an increased sensitivity to light pulses that reset the clock, and sustain the circadian control of the LHCB1*1 promoter in constant darkness. These results suggest that ZGT is another component of the complex interplay between light-signalling pathways and the clock.

52. Takahashi Y, Shimura A, Sasaki T, Yano M: *Hd6*, a rice quantitative trait locus involved in photoperiod sensitivity, encodes the α subunit of protein kinase CK2. *Proc Natl Acad Sci USA* 2001, 98:7922-7927.

This paper reports the cloning of *Hd6*, a minor quantitative trait locus (QTL) affecting photoperiodic sensitivity in rice. *Hd6* was cloned by combining the generation of near-isogenic lines with high-resolution and fine-scale mapping. *Hd6* encodes the α subunit of CK2. Overexpression of the β subunit of CK2 shortens the circadian period of several rhythms in *Arabidopsis*. Hence, it will be interesting to see whether the reduced photoperiodic sensitivity associated with the loss-of-function CK2α allele is caused by a lengthening of circadian periods.