
Recent advances in the formation of the bacterial peptidoglycan monomer unit

Jean van Heijenoort

Biochimie, Bat 430, Université Paris-Sud, Orsay, F-91405, France

Received (in Cambridge, UK) 26th April 2001  
First published as an Advance Article on the web 14th August 2001

Covering: 1985–2000.

1 Introduction
1.1 Biosynthesis of bacterial peptidoglycan
1.2 Assembly of the monomer unit
2 Formation of UDP-N-acetylglucosamine
2.1 GlmS synthase
2.2 Phosphoglucosamine mutase
2.3 GlmU synthase
3 Formation of UDP-N-acetylmuramic acid
3.1 Transferase MurA
3.2 MurB reductase
4 Formation of UDP-MurNAc-peptides
4.1 MurC synthetase
4.2 MurD synthetase
4.3 MurE synthetase
4.4 MurF synthetase
5 Formation of the lipid intermediates
5.1 Phospho-N-acetylmuramoyl-pentapeptide transferase (MraY)
5.2 N-Acetylglucosaminyl transferase MurG
6 Side pathways
6.1 Formation of D-glutamate
6.2 Formation of dipeptide D-alanyl-D-X
6.3 Undecaprenyl phosphate cycle
7 In vivo functioning of the monomer unit assembly
8 Structural variations in the monomer unit
9 Flexibility in peptidoglycan synthesis
10 Concluding remarks
11 Acknowledgements
12 References

1 Introduction

Peptidoglycan (or murein) is a continuous covalent macro- molecular structure found on the outside of the cytoplasmic membrane of almost all eubacteria and exclusively in these organisms. Its main function is to preserve cell integrity by

Jean van Heijenoort is a CNRS Director of Research at the Department of Biochemistry of the University of Orsay, France. His work on bacterial cell walls has been mainly devoted to the study of the structure and metabolism of peptidoglycan, and to that of antibiotics inhibiting its biosynthesis.

Fig. 1 Scheme of the primary structure of bacterial peptidoglycan. Abbreviations: GlcNAc: N-acetylglucosamine; MurNAc: N-acetyl- muramic acid; DA: diamino acid (generally diaminopimelic acid or L-lysine); n: number of amino acids in the cross-bridge depending on the organism; (D-Ala): often missing in the peptidoglycan of many organisms; $\longrightarrow$: CO-NH-.

withstanding the internal osmotic pressure. Peptidoglycan is also responsible for the maintenance of a defined cell shape, and it is intimately involved in cell growth and cell division. Its absence from cells will lead in a hypotonic medium to their swelling and to the rupture of the cytoplasmic membrane. Under certain conditions, cells lacking peptidoglycan can be maintained as protoplasts or spheroplasts, but the cell shape is lost and cell division is greatly perturbed. The main structural features of this giant macromolecule (Fig. 1) are linear glycan chains interlinked by short peptides. The glycan chains are composed of alternating units of N-acetylglucosamine (Glc- NAc) and N-acetylmuramic acid (MurNAc), all linkages between sugars being β,1→4. The carboxy group of each N- acetylmuramic acid residue is substituted by a peptide subunit, which is most often L-alanyl-γ-D-glutamyl-diaminopimelyl(or L-lysyl)-D-alanyl-D-alanine in nascent peptidoglycan, and which subsequently loses one or both D-alanine residues in mature peptidoglycan. Neighboring glycan chains are interlinked either by a direct peptide linkage between a peptide subunit of a chain with one of another chain, or by a short peptide bridge between two peptide subunits.

DOI: 10.1039/a804532a

Nat. Prod. Rep., 2001, 18, 503–519

This journal is © The Royal Society of Chemistry 2001

1.1 Biosynthesis of bacterial peptidoglycan

The biosynthesis of bacterial peptidoglycan is now a half-century old story. Its nucleotide precursors were first isolated from penicillin-treated *Staphylococcus aureus* and characterized by Park<sup>7</sup> at a time when its existence as an essential cell wall macromolecule was not yet recognized. The various steps have now been studied in different species and an overall view of the pathway valid for all eubacteria has emerged.<sup>1,8</sup> It is a complex two-stage process. The first stage concerns the assembly of its monomer unit by enzymes located in the cytoplasm or at the inner side of the cytoplasmic membrane.<sup>9-11</sup> The final product is the lipid intermediate disaccharide-(peptide)-pyrophosphate undecaprenol. The second stage involves polymerization reactions taking place at the outer side of the cytoplasmic membrane and using as substrate the lipid intermediate which has been translocated through the membrane. Two major types of membrane-bound activities are involved in polymerization: glycosyltransferases that catalyse the formation of the linear glycan chains<sup>12,13</sup> and transpeptidases that catalyse the formation of the peptide cross-bridges between the new chains and the binding to the preexisting cell wall.<sup>1,8,12,14,15</sup>

Another historically important aspect of the metabolism of peptidoglycan is the specific inhibition of certain steps by various families of antibiotics, some of which are well-known drugs in clinical use (β-lactams, glycopeptides, fosfomycin, bacitracin, etc.) or in use as animal growth promoters (moenomycin etc.). Their targets, mode of action, and effects on bacterial cells have been extensively investigated in close correlation with that of the metabolism of peptidoglycan (references in refs. 1, 8, 9, 12, 13, 16). The continued interest for this system in recent years is primarily due to: (i) the emergence of new resistance mechanisms against β-lactam and glycopeptide antibiotics involving subtle modifications in peptidoglycan synthesis; (ii) the need to overcome resistance mechanisms and to use specific targets for the search of novel antibacterials; and (iii) the steady progress in the difficult problems of correlating peptidoglycan metabolism with cell growth and division. The aim of this review is to focus more specifically on the abundant genetic, biochemical, and physiological data published over the past ten years on the biosynthesis of the peptidoglycan monomer unit.

1.2 Assembly of the monomer unit

The assembly of the monomer unit proceeds by a well-defined linear sequence of reactions from fructose-6-P to the final lipid intermediate via UDP-N-acetylglucosamine (Fig. 2). This pathway was established by characterizing its precursors and by developing a specific *in vitro* enzymatic assay for each step.<sup>1,8-11</sup> In particular, convenient methods for the isolation, quantitative analysis, and preparation of the various nucleotide precursors were developed (refs. in ref. 10). Two essential features characterize the pathway. First, the high specificity of each step reflects the unusual structural characteristics of peptidoglycan, many of which are already encountered in its monomer unit (presence of muramic acid, of D-glutamic acid involved in a γ linkage, of a diamino acid, and of alternating D and L residues in peptide linkages). Secondly, the monomer unit is transferred from the cytoplasm to the externally located sites of polymerization. This implies a passage of the lipid intermediate through the hydrophobic environment of the membrane. At least in *Escherichia coli*, all the genes directly involved in the assembly of the monomer unit have been identified, cloned, and sequenced. Most of their products have been overproduced, purified to homogeneity, and characterized. The availability of large amounts of purified enzymes has greatly facilitated the development of structural and mechanistic studies. Owing to their high specificity and their occurrence only in bacteria, they are potential targets of particular interest for the search of novel antibacterial agents and important efforts have been made to elucidate their catalytic mechanisms. In the assembly

Fig. 2 Stepwise assembly of the peptidoglycan monomer unit.

of the monomer unit (Fig. 2) four successive groups of reactions will be considered: formation of UDP-N-acetylglucosamine (steps GlmS, GlmM, and GlmU); formation of UDP-N-acetylmuramic acid (steps MurA and MurB); formation of the UDP-MurNAc-peptides (steps MurC to MurF); and formation of the lipid intermediates (steps MraY and MurG).

2 Formation of UDP-N-acetylglucosamine

In bacteria UDP-N-acetylglucosamine (UDP-GlcNAc) is not only an essential peptidoglycan precursor but is also used in the synthesis of many other cell wall N-acetylglucosamine-containing macromolecules such as, for instance, teichoic

acids<sup>17</sup> in Gram-positive organisms, or lipopolysaccharides<sup>18</sup> and the enterobacterial common antigen<sup>19</sup> in Gram-negative organisms. Four successive steps (1 to 4 in Fig. 3) are required for its synthesis from fructose-6-phosphate.<sup>20,21</sup> They have been investigated in detail at both genetic and biochemical levels.

### 2.1 GlmS synthase

In the first step, the conversion of D-fructose-6-phosphate into glucosamine-6-phosphate is catalysed by L-glutamine : D-fructose-6-phosphate amidotransferase, which is encoded in *E. coli* by the *glmS* gene.<sup>22</sup> The amide functionality of glutamine is used as ammonia source. The GlmS transferase was purified to homogeneity from *E. coli* and *Thermus thermophilus*, and characterized.<sup>23,24</sup> It is a bienzyme complex with two structurally and functionally distinct domains. The N-terminal glutaminase domain catalyses hydrolysis of glutamine to glutamate and ammonia, whereas the C-terminal isomerase domain utilizes the ammonia for fructose-6-phosphate to glucosamine-6-phosphate conversion. Each domain was overproduced, crystallized and its 3D-structure determined at high resolution.<sup>25,26</sup> The glutamine and fructose-6-P binding sites were studied using substrate analogues and the residues involved in their binding were identified.<sup>27–29</sup> In particular, there is evidence

![Diagram](attachment:diagram.png)

Fig. 3 Metabolic pathway leading to UDP-N-acetylglucosamine (UDP-GlcNAc) in bacteria.

for the close proximity of the two sites. A chemical mechanism for the isomerase activity of GlmS was proposed.<sup>30</sup> Naturally occurring and synthetic inhibitors of the GlmS synthase have been described (refs. in refs. 9, 16 and 29). Recently, novel electrophilic glutamine analogues based on 6-diazo-5-oxonorleucine were reported as very potent inhibitors.<sup>31</sup>

### 2.2 Phosphoglucosamine mutase

In the second step the phosphoglucosamine mutase catalyses the interconversion of glucosamine-6-phosphate and glucosamine-1-phosphate (Fig. 3). In *E. coli* it was initially characterized as the product of the *glmM* gene<sup>32</sup> which was later identified as the *ureC* gene in *Helicobacter pylori*,<sup>33</sup> as the *femD* or *femR315* gene in *Staphylococcus aureus*,<sup>34</sup> and as ORF540 in *Pseudomonas aeruginosa*.<sup>35</sup> The *glmM* gene was shown to be essential in *E. coli*,<sup>32</sup> but genetic data recently suggested that in *S. aureus* there could be an alternative pathway for glucosamine-1-phosphate.<sup>36</sup> Purified *E. coli* GlmM was shown to be active only in a phosphorylated form containing one bound phosphate and separable from the unphosphorylated form.<sup>32,37</sup> Phosphorylation could be a factor regulating the flow of metabolites in the pathway. The enzyme catalyses the reaction according to a ping-pong bi-bi mechanism involving GlcNAc-1,6-diphosphate as intermediate.<sup>37</sup> The phosphorylation site was clearly identified as Ser-102 and GlmM also catalyses the interconversion of glucose-1-phosphate and glucose-6-phosphate.<sup>37</sup> *In vitro* GlmM can undergo autophosphorylation with ATP.<sup>38</sup>

### 2.3 GlmU synthase

The last two steps, acetylation and uridylation (Fig. 3), are catalysed in *E. coli* by the product of the *glmU* gene located just upstream from *glmS* at 84 min on the chromosome.<sup>39,40</sup> The gene has now been identified in several other bacterial species (refs. in ref. 41). The GlmU protein from *E. coli* is a bifunctional enzyme which was overproduced and purified.<sup>40,42</sup> Its C-terminal domain catalyses acetylation of glucosamine-1-phosphate into N-acetylglucosamine-1-phosphate, whereas its N-terminal domain catalyses uridylation to yield UDP-GlcNAc.<sup>40,42</sup> The two domains are functionally independent<sup>40,42</sup> and each one is essential for cell viability as clearly established, in particular, by the study of N- and C-terminal truncated forms.<sup>41,42</sup> Acetylation precedes uridylation with apparently no cooperativity between them.<sup>40,42</sup> By using radiolabelled substrates, it was demonstrated that intermediate GlcNAc-1-P is released from the acetyltransferase domain prior to transformation by the uridyltransferase domain.<sup>42</sup> The substrates, products, and effectors of the acetyltransferase reaction have no effect on the uridyltransferase activity and *vice versa*.<sup>40,42</sup> Nonetheless, it is noteworthy that GlmU catalyses *in vitro* the uridylation of glucosamine-1-phosphate, although at a 15-fold slower rate than GlcNAc-1-P.<sup>41</sup> The acetyltransferase, but not the uridyltransferase, is inactivated by thiol-specific reagents.<sup>40</sup> The GlmU cysteine residues are exclusively located in the acetyltransferase domain and their possible role in the catalytic process was studied.<sup>43</sup> A truncated form of *E. coli* GlmU carrying the complete uridylation N-terminal domain but only part of the acetylation C-terminal domain was crystallized and its structure determined.<sup>44</sup> The molecule is composed of two distinct domains connected by a long α-helical arm and three GlmU molecules assemble into a trimeric arrangement. In addition, the structure of the GlmU : UDP-GlcNAc complex revealed the structural bases required for the uridyltransferase activity. Recently, the crystal structures of *Streptococcus pneumoniae* GlmU and its complex with UDP-GlcNAc and Mg<sup>2+</sup> were determined.<sup>45</sup> Here too GlmU forms trimers. UDP-GlcNAc and Mg<sup>2+</sup> are bound at the uridyltransferase active site which is in a closed form. A uridyltransferase mechanism was proposed. Chromatography of the *E. coli* GlmU protein on gel filtration was in agreement with a trimeric arrangement.<sup>41</sup>

Table 1 Leading references relevant to the genes, overproductions, purifications, crystallizations, and 3D structures of the various enzymes of the assembly of the peptidoglycan monomer unit

| Enzyme (gene)                          | Gene identification | Purification/characterization | Crystallization | 3D structure |
|----------------------------------------|---------------------|-------------------------------|-----------------|--------------|
| GlcN-6-P synthase                      | 22                  | 23, 24                        | 25              | 26           |
| (glmS)                                 |                     |                               |                 |              |
| GlcN-1-P mutase                        | 32–35               | 32, 37                        | —               | —            |
| (glmM)                                 |                     |                               |                 |              |
| UDP-GlcNAc synthase                    | 39, 40              | 40, 42, 45                    | 44, 45          | 44, 45       |
| (glmU)                                 |                     |                               |                 |              |
| Transferase                            | 46, 51, 52          | 51, 54–57                     | 63, 64          | 64, 67       |
| (murA)                                 |                     |                               |                 |              |
| Reductase                              | 47–50               | 58–62                         | 62, 65, 66      | 62, 68, 69   |
| (murB)                                 |                     |                               |                 |              |
| L-Ala-adding enzyme                    | 48, 88–92, 115      | 115, 121–130                  | 128             | —            |
| (murC)                                 |                     |                               |                 |              |
| D-Glu-adding enzyme                    | 93–99               | 131–135                       | 133             | 145–147, 165 |
| (murD)                                 |                     |                               |                 |              |
| A<sub>2</sub>pm or Lys-adding enzyme    | 100–106             | 136–141                       | 141             | 141          |
| (murE)                                 |                     |                               |                 |              |
| D-Ala-D-X-adding enzyme                | 88, 100, 107–109    | 109, 142–144                  | 144             | 148          |
| (murF)                                 |                     |                               |                 |              |
| Transferase                            | 190, 191            | —                             | —               | —            |
| (mraY)                                 |                     |                               |                 |              |
| GlcNAc transferase                     | 89, 198, 213, 214   | 219, 220                      | 221             | 221          |
| (murG)                                 |                     |                               |                 |              |

study of heterotrimers formed in the presence of truncated forms showed that trimerization is absolutely required for acetyltransferase activity and that its catalytic site involves regions of contact between adjacent monomers. Trimerization is determined by the acetyltransferase domain, and it is clearly not essential for uridyltransferase activity.

### 3 Formation of UDP-N-acetylmuramic acid

The formation of UDP-N-acetylmuramic acid (UDP-MurNAc) is a two-step process (MurA and MurB in Fig. 2). First, the transfer of enolpyruvate from PEP to position 3 of the GlcNAc residue is catalysed by a transferase to yield UDP-GlcNAc-enolpyruvate. In the second step, the reduction of the enolpyruvate moiety to D-lactoyl is catalysed by a reductase to yield UDP-MurNAc. Early genetic studies indicated that the murA and murB genes coding for the transferase and the reductase, respectively, were both located at ca. 90 min on the E.coli chromosome (refs. in Table 1). The position of murB at 89 min was confirmed after its physical and genetic mapping in the 15kb btuB-rpoBC cluster which contained no identifiable murA gene. Finally, a murA gene encoding a UDP-GlcNAc enolpyruvoyl transferase was cloned, located at 69 min and found to be essential. Sequence analyses have now shown more generally that Gram-negative bacteria have only one murA gene, whereas low-G+C Gram-positive bacteria have two distinct genes.

Transferase MurA and reductase MurB were initially purified from various bacterial species (Table 1). More recently, MurA from E. coli and E. cloacae as well as MurB from E. coli and S. aureus were overproduced and purified. This enabled their crystallization which subsequently led to the determination of their 3D structure. NMR studies of MurB have also been developed. The ready availability of MurA and MurB has also been useful for the large-scale preparation of UDP-GlcNAc-enolpyruvate, which is at a very low pool level, and for the preparation of labelled UDP-MurNAc.

#### 3.1 Transferase MurA

The reaction pathway of E. coli MurA has been investigated using rapid kinetics, PEP analogues and site-directed mutagenesis (ref. 74 and refs. therein). From these studies an addition-elimination mechanism was proposed which proceeds through a tetrahedral intermediate, where C-3 of PEP becomes

a methyl group and C-2 a ketal with phosphate and UDP-GlcNAc substituents (Fig. 4). The ketal adduct phospholactoyl-UDP-GlcNAc formed between the two substrates is non-covalently bound to the enzyme. Its structure and its chirality were determined from the structure of the complex between the fluoromethyl tetrahedral ketal analogue and the C115A mutant of MurA. The stereochemical course of the enzymatic enolpyruvyl transfer was determined by use of PEP analogues. Addition to the double bond of PEP is anti whereas elimination of H⁺ from C-3 and Pᵢ from C-2 is syn. Another tetrahedral intermediate covalently bound to the enzyme has also been demonstrated. In this covalent enzyme intermediate Cys-115 is attached to the C-2 of PEP to form an O-phosphothio ketal intermediate. A mechanism was proposed in which the formation of the covalent adduct precedes that of the non-covalent one with Cys-115 participating first as an enzyme nucleophile and thereafter as a general acid. However, it was proved that the covalent intermediate is dispensable for catalysis and appears to be off the main catalytic pathway.

Small angle X-ray scattering fluorescence experiments carried out with E. cloacae MurA and comparison of the structures of various E. coli MurA crystal complexes indicated that the 112–121 loop with Cys-115 is flexible and can assume three conformational states which are likely to correspond to different stages of the catalytic process. The open form would be nonligated MurA, the intermediate form one arising after ligand binding just before the addition reaction leading to the tetrahedral adduct, and the closed form present before the proton transfer and elimination of Pᵢ. It was suggested that Lys-22, which is strictly conserved in the active centre of MurAs, is involved directly in the binding of PEP and participates in the conformational change leading to the catalytically competent enzyme complex. Recently, the homogeneous isomerization of Asn-67 to an L-isaspartate residue was recognized and it was suggested to be critical for the induced-fit mechanism. Furthermore, site-directed mutageneses in E. cloacae MurA emphasized the dual role of Asp-305 as a general base and as an essential binding partner of UDP-GlcNAc in the active site. The best known inhibitor of MurA is fosfomycin which is a PEP analogue clinically used as antibiotic. The inactivation of MurA by fosfomycin results from its covalent linkage to the active site cysteine residue. Fosfomycin resistance in Mycobacterium tuberculosis was recently shown to be due to the replacement of the cysteine residue by an aspartate residue.

Fig. 4 Mechanism of MurA. EP-UDP-GlcNAc: UDP-N-acetylglucosamine-enolpyruvate.

### 3.2 MurB reductase

The MurB reductase from *E. coli* is a flavoprotein and its chemical mechanism (Fig. 5) involves a sequence of two-half-reactions. In the first one, FAD is reduced by the two-electron transfer from NADPH to the tightly bound flavin. In the second one, the same two electrons are transferred from E-FADH₂ to the C-3 of the enol ether and the reduction of the vinyl bond is completed by quenching at C-2 of the carbanion equivalent with a solvent-exchangeable proton. Active-site Ser-229 is a general acid catalyst serving as the proton donor to quench the carbanion/enol intermediate and delivers the proton to C-2 to yield the D-configuration of the lactyl ether product. A study of the kinetic mechanism of the *E. coli* MurB reductase was undertaken by characterizing the steady state initial velocity patterns and determining the inhibition constants of reaction products. These experiments suggested the absence of simultaneous binding of NADPH and UDP-GlcNAc-EP to MurB. The NMR study of Mur B indicated that NADPH binds to the same pocket as UDP-GlcNAc-EP and that NADPH transfers a hydride to the si face of the FAD iso-

Fig. 5 Mechanism of MurB: reductive and oxidative half-reactions according to Benson et al.

alloxazine ring. Furthermore, the binding of NADPH induces structural changes in MurB. To date there are few known potent inhibitors of MurB.

## 4 Formation of UDP-MurNAc-peptides

The assembly of the peptide of the monomer unit proceeds by the stepwise addition of L-alanine, D-glutamic acid, a diamino acid (usually diaminopimelic acid or L-lysine), and a dipeptide D-alanine-X (usually X is D-alanine, less frequently D-lactate or D-serine) onto the D-lactoyl group of UDP-MurNAc (steps MurC to MurF in Fig. 2). Each step is catalysed by a highly specific cytoplasmic ADP/peptide-forming enzyme using ATP/Mg²⁺ and designated as Mur synthetase, ligase or adding-enzyme (Fig. 6(A)). The biochemical study of these activities as well as the isolation of conditional-lethal mutants, characterized by a cell-lysis phenotype, were developed in both Gram-positive and Gram-negative organisms (refs. in refs. 1, 8, 9, 10, 11). However, the Mur synthetases have been studied in greater detail in *E. coli* where their genes (*murC*, *murD*, *murE* and *murF*) have been identified, cloned, sequenced, and characterized as unique and essential for viability (refs. in Table 1). They are located at 2 min on the *E. coli* chromosome in the large *mra* cluster that contains both peptidoglycan synthesis and cell division genes (refs. in refs. 110 and 111). Their

A. Stepwise assembly of the peptide subunit

MurC UDP-MurNAc + L-Ala + ATP → UDP-MurNAc-L-Ala + ADP + Pi  
MurD UDP-MurNAc-L-Ala + D-Glu + ATP → UDP-MurNAc-L-Ala-D-Glu + ADP + Pi  
MurE UDP-MurNAc-L-Ala-D-Glu + A₂pm (or L-Lys) + ATP → UDP-MurNAc-tripeptide + ADP + Pi  
MurF UDP-MurNAc-tripeptide + D-Ala-D-X + ATP → UDP-MurNAc-pentapeptide + ADP + Pi  

B. Mechanism of the Mur synthetases

![Mechanism Diagram](https://i.imgur.com/1234567.png)

Fig. 6 Synthesis of the peptide subunit (A) and mechanism of the Mur synthetases (B). R: nucleotide substrate. R′–NH₂: amino acid or dipeptide substrate.

expression was shown to be regulated by the Pₘᵣₐ promoter.¹¹⁰,¹¹¹ Cognate mur gene sequences from a wide variety of bacterial genera are now known. Some have been identified and cloned (refs. in Table 1). As observed in the bacterial genomes so far available many of them also belong to a division-cell wall (dcw) cluster similar to mra (refs. 92, 96–99, 112–115, and refs. therein). In E. coli there is yet another Mur synthetase, Mpl, which is not essential and which catalyses the addition of tripeptide L-Ala-γ-Glu-meso-A₂pm to UDP-MurNAc in the peptidoglycan recycling process.¹¹⁶,¹¹⁷

Comparison of the amino acid sequences of various Mur synthetases and other related enzymes revealed the existence of common invariants: seven amino acids plus the ATP-binding consensus sequence.¹¹⁸–¹²⁰ Moreover, the conservation of constant lengths between certain invariants suggested possible common structural motifs.¹²⁰ Among the other known ADP-peptide forming synthetases, only folylpoly-γ-L-glutamate synthetase (FPGS), the capB gene product involved in the synthesis of capsular poly-γ-D-glutamate, and part of the cyanophycin synthetase share the same conserved amino acids (refs. in refs. 119 and 120). Wild type and fusion forms of the four Mur synthetases from E. coli and some from other organisms have been overproduced, purified to homogeneity, and characterized (Table 1). Crystallization of all four synthetases has as yet led to the determination of the 3D structure of different forms of MurD, MurE and MurF (Table 1). All three differ from one another by the topology of their N-terminal domain. The resulting high resolution models of the closed active forms of MurD and MurE reveal that they share the same three-domain topology and a similar active site architecture. The remarkable structural similarity with the FPGS of Lactobacillus casei¹⁴⁹ clearly suggests that the Mur synthetases and FPGS are all members of the same superfamily.

The kinetic properties and the catalytic mechanism of the Mur synthetases have been investigated in detail. They all catalyse the formation of an amide or peptide bond with concomitant cleavage of ATP into ADP and inorganic phosphate (Fig. 6(A)). They operate by an essentially similar chemical mechanism (Fig. 6(B)). This entails the carboxy activation of a C-terminal amino acid residue of the nucleotide substrate to an acyl phosphate intermediate followed by nucleophilic attack by the amino group of the condensing amino acid or dipeptide, with the elimination of phosphate and subsequent peptide bond formation. The existence of an acyl phosphate intermediate was substantiated by enzymatic kinetics and chemical approaches.⁹,¹²⁴,¹²⁶,¹⁵⁰–¹⁵² The reversibility of the reaction⁹,¹²³,¹³¹,¹³²,¹³⁷,¹⁴⁰ as well as the catalysis of an exchange reaction between the amino acid or dipeptide substrate and the nucleotide reaction product, in the presence¹²³,¹³⁷,¹⁴⁰,¹⁵³ or absence⁹,¹⁵⁰ of ADP, have been established. These results are consistent with the reversible formation of an acyl phosphate. The formation of adenosine 5′-tetraphosphate observed with MurD in the absence of D-glutamate, but not with the other Mur synthetases in the absence of their amino acid or dipeptide substrate, further confirmed the existence of an acyl phosphate.¹⁵² Phosphinate analogues of the nucleotide substrates were found to be potent competitive inhibitors.¹⁵⁴–¹⁵⁶ The tight binding of compounds which mimic the structure of the putative tetrahedral intermediate strongly suggests that a tetrahedral transition state follows the acyl phosphate in the reaction scheme (Fig 6(B)).

The Mur synthetases thus appear as a well-defined family of closely structurally and functionally related proteins, presumably originating from a common ancestor. However, it is noteworthy that the enzymatic properties of a given Mur synthetase can vary to some extent with the bacterial species as recently established with MurD.¹³⁵ As far as we are aware no natural inhibitor of these synthetases has yet been identified.

### 4.1 MurC synthetase

Purified E. coli MurC exists in equilibrium between monomeric and dimeric forms and its specific activity is independent of its oligomerization state.¹²⁷ The specificity of various MurC synthetases for L-alanine and UDP-MurNAc was investigated with closely structurally related analogues accepted as substrate or functioning as competitive inhibitors.¹²¹–¹²³,¹²⁵,¹²⁸,¹⁵⁷–¹⁵⁹ Several compounds structurally related to L-alanine are good inhibitors. Although MurC can efficiently ligate serine and glycine to its nucleotide substrate, the preferential in vivo use of alanine is presumably due to its higher intracellular pool. However, in Corynebacterium¹⁶⁰ and Mycobacterium spp¹¹⁵ glycine is found in position 1 of the peptide subunit. In the first case this corresponds to a high specificity of the MurC synthetase for glycine, whereas in the second case it appears to be due to growth conditions. The nucleotide products formed with alanine analogues functioning as alternative substrates were isolated and characterized.¹²¹,¹²⁸ The kinetic mechanism of MurC was investigated by initial velocity methods and the data were consistent with an ordered mechanism in which ATP binds first, UDP-MurNAc binds second, L-alanine binds last, and the last product released is ADP.¹⁵¹ The replacement in E. coli MurC of amino acid residues well conserved among 20 Mur paralogues was undertaken by site-directed mutagenesis.¹²⁰ Some residues were essential for the catalytic process, whereas others led to variations of the Kₘ values for one or more substrates, thereby indicating their involvement in the structure of the active site. The observed variations of Kₘ values confirmed the binding-order of substrates. The replacement of the two cysteine residues of E. coli MurC showed they played no role in the catalytic process.¹⁶¹ A phylogenetic analysis of the amino acid sequence of MurC orthologues was recently undertaken.¹³⁰

### 4.2 MurD synthetase

Practically, only a D-glutamic acid residue is found in position 2 of the peptide subunit.⁶ The high specificity of MurD for D-glutamate was confirmed by studying structurally related analogues tested as substrates or inhibitors.¹⁵⁹,¹⁶² In particular, a D-glutamate configuration, possibly recognized by MurD, was proposed. In a similar way, but to a lesser extent, the specificity of MurD for the UDP-MurNAc-L-Ala substrate was studied.¹⁵⁹,¹⁶² The specificity was found not to be too strict since phospho-MurNAc-L-Ala was substrate. Various transition-

state analogues of the nucleotide substrate have been designed.¹⁵⁴,¹⁵⁶,¹⁶³,¹⁶⁴ The best IC₅₀ value (<1 nM) was obtained with a phosphinate compound mimicking very closely the transition state.¹⁵⁶ Conversely, the fact that phosphonic or phosphinic derivatives of alanine are nearly devoid of inhibitory activity¹⁶³,¹⁶⁴ indicates that both the L-Ala and D-Glu moieties are essential for good inhibition. The specificity for the nucleoside triphosphate was examined with *S. aureus* MurD.¹³¹ The crystal structures of six different forms of MurD have now been solved.¹⁴⁵–¹⁴⁷,¹⁶⁵ The structure is composed of three domains of topologies reminiscent of a nucleotide-binding fold. The N- and C-terminal domains have a dinucleotide-binding fold whereas the central domain has a mononucleotide-binding fold. The structure reveals the binding site for UDP-MurNAc-L-Ala, and comparison with known NTP complexes allows the identification of residues interacting with ATP. A comparison of the MurD structures suggests large C-terminal rotation, loop rearrangement and subdomain movements occurring on substrate binding. Similarly, conformational changes induced by substrate binding in the reaction mechanism of folylpolyglutamate synthetase results in the movement of the domains towards each other.¹⁴⁹ In parallel to structural studies, site-directed mutagenesis showed that the invariant amino acids common to all orthologues are all located in the active site.¹⁶⁶

### 4.3 MurE synthetase

Depending on the organism, generally either an A₂pm or L-lysine residue is located at position 3 in the peptide subunit, more rarely an ornithine, diaminobutyric acid, homoserine, lanthionine, or 3-hydroxy-A₂pm residue (refs. in refs. 6, 167, 168). By its ε-amino group, A₂pm or lysine is involved in the cross-linking between the peptide subunits of the glycan chains and thus plays a key role in the integrity of peptidoglycan. In a given organism A₂pm and L-lysine are both present as cell metabolites, but its MurE synthetase generally efficiently discriminates between the two amino acids by catalysing the addition of only one of them to the UDP-MurNAc-L-Ala-D-Glu precursor.¹³⁶,¹³⁷,¹³⁹,¹⁶⁹–¹⁷¹ This is also true in *Corynebacterium poinsettiae* where L-homoserine is specifically incorporated in position 3,¹⁶⁰ but not in *Bifidobacterium globosum* where the same enzyme catalyses the incorporation of both L-ornithine and L-lysine.¹³⁸ *B. sphaericus* is unique in its choice of the diamino acid.¹³⁹,¹⁶⁹,¹⁷² During vegetative growth a MurE synthetase catalyses the incorporation of L-lysine whereas during spore cortex formation another enzyme catalyses the incorporation of meso-A₂pm. The specificity of *E. coli* MurE for meso-A₂pm has been assessed with various analogues both in vivo and in vitro (refs. 140, 168, 173–175, and refs. therein). Although very high, certain A₂pm analogues can be accepted as substrate in vivo and in vitro, whereas others have an inhibitory effect. Similarly, hydroxyllysine can replace L-lysine at position 3 in a number of Gram-positive bacteria grown in its presence (refs. in ref. 176).

The specificity of *E. coli* MurE with respect to UDP-MurNAc-L-Ala-D-Glu was shown to be high.¹⁷⁷ It was studied with structural analogues mimicking to various extents either end of this complex linear substrate. None was substrate and only a few had a limited inhibitory effect, which suggested that no particular portion of the molecule is predominantly responsible for its recognition by the enzyme. Multiple sites located over the whole molecule are thus required for a proper recognition and determine the high specificity of the activity. The search for irreversible or transition state inhibitors was subsequently undertaken.¹⁵⁵,¹⁷⁸,¹⁷⁹ The best IC₅₀ value was observed with a phosphinate analogue.¹⁵⁵

The crystal structure of *E. coli* MurE complexed with its nucleotide reaction product is now solved to 2.0 Å resolution.¹⁴¹ Comparison with the known structures of MurD has allowed the identification of residues involved in the enzymatic mech-

![Figure 7](#fig7)

Fig. 7 Formation of lipid intermediates I and II. GlcNAc: *N*-acetyl-glucosamine; MurNAc: *N*-acetylmuramic acid; DA: diamino acid; pep: pentapeptide; P~~~~~ undecaprenyl phosphate.

anism. Interestingly, as observed with Lys-198 of MurD,¹⁴⁵ homologous Lys-224 of MurE is carbamylated. It was proposed that this modification is crucial for Mg²⁺ binding and thus for the positioning of the γ phosphate of ATP. Furthermore, a structural determinant responsible for the specificity for the diamino acid substrate was identified.

### 4.4 MurF synthetase

Initial velocity and dead-end inhibitor studies with MurF are consistent with a sequential ordered kinetic mechanism for the forward reaction in which ATP binds to free enzyme, followed by UDP-MurNAc-tripeptide and D-Ala-D-Ala in sequence prior to product release.¹⁰⁹ Substrate inhibition by UDP-MurNAc-tripeptide is suppressed by 0.5 M NaCl¹⁰⁹ but no product inhibition is observed.¹⁴³,¹⁵⁹ ADP-ATP or ATP-Pᵢ exchange reactions were described.¹³⁷,¹⁴² The specificity of MurF with respect to the UDP-MurNAc-tripeptide substrate has been investigated to a limited extent.¹⁵⁹,¹⁶⁰,¹⁶⁸ However, its insensitivity to various replacements of the amino acid in position 3 of the nucleotide substrate was clearly assessed.¹⁰⁶,¹⁶⁰,¹⁶⁸,¹⁷³,¹⁷⁶ The specificity profile for the dipeptide substrate has been studied with D-Ala-D-Ala analogues accepted as substrates or functioning as inhibitors, and with different in vivo systems.⁹,¹⁴³,¹⁵⁸,¹⁷⁶,¹⁸⁰–¹⁸⁵ The C-terminal D-alanine can be replaced by D-lactate, D-hydroxybutyrate, glycine, or various D-amino acids, whereas the N-terminal D-alanine can be replaced by glycine, D-aminobutyrate, D-serine or D-valine. Pseudo-tri- and tetrapeptide aminoalkylphosphinic acids of general structure X-LysΨ(PO₂H-CH₂)-Gly-Ala were synthesized as transition state analogues for MurF.¹⁸⁶ Kinetic assays revealed they act as inhibitors of MurF with *K*ᵢ values in the range 200–700 μM.

The crystal structure of the *E. coli* MurF apoenzyme was determined to 2.3 Å resolution and revealed an open conformation with three α/β-sheet domains juxtaposed in a crescent-like arrangement.¹⁴⁸ The substrate-free MurF resembles an open form of substrate bound-MurD. It was proposed that ATP, the first substrate to bind, may be responsible for inducing the proper domain rearrangement required for forming a functional enzyme-substrate complex.¹⁰⁹

## 5 Formation of the lipid intermediates

The first membrane step in the formation of the lipid intermediates (MraY in Figs. 2 and 7) involves a phospho-*N*-acetylmuramoyl-pentapeptide transferase (or translocase) which catalyses the transfer of the phospho-MurNAc-pentapeptide moiety of UDP-MurNAc-pentapeptide to membrane acceptor undecaprenyl phosphate to yield MurNAc-(pentapeptide)-pyrophosphoryl undecaprenol (lipid I). Thereafter, an *N*-acetylglucosamine transferase (MurG in Figs. 2 and 7) catalyses the addition of *N*-acetylglucosamine yielding GlcNAc-MurNAc-(pentapeptide)-pyrophosphoryl undecaprenol (lipid II). After its transfer to the outer side of the cytoplasmic membrane, lipid

II is used as the substrate for the polymerization reactions.¹³ Early work on both transferases has been reviewed.¹⁸⁻²⁰

### 5.1 Phospho-N-acetylmuramoyl-pentapeptide transferase (MraY)

The transferase catalysing the formation of lipid I was initially investigated in *E. coli*, *Micrococcus luteus* and *S. aureus*.¹⁸⁷⁻¹⁸⁹ In *E. coli* it was identified as the product of the *mraY* gene which was located in the *dcw* cluster at 2 min on the chromosome, cloned, and sequenced.¹⁹⁰,¹⁹¹ The *mraY* gene is dependent on the P<sub>*mra*</sub> promoter.¹¹⁰,¹¹¹ Moreover, the MraY protein was demonstrated to be essential¹⁹² and unique in all the eubacterial genomes so far available.¹⁹³ The high hydrophobicity of the *E. coli* MraY protein was clearly substantiated by its amino acid sequence, which has repeated hydrophobic and hydrophilic domains,¹⁹⁰ and by the requirement of a lipid microenvironment for its activity.¹⁸⁷,¹⁸⁹,¹⁹⁴ Recently, a common two-dimensional membrane topology model was established for the *E. coli* and *S. aureus* MraY transferases.¹⁹³ It possesses ten transmembrane segments, five cytoplasmic domains and six periplasmic domains including the N- and C-terminal ends. The agreement between the topologies of *E. coli* and *S. aureus*, their agreement to a fair extent with predicted models and a number of features arising from the comparative analysis of 25 orthologue sequences strongly suggested the validity of the model for all eubacterial MraYs. The primary structure of the ten transmembrane segments diverged among orthologues, but they retained their hydrophobicity, number and size. The similarity of the sequences and distribution of the five cytoplasmic domains in both models, as well as their conservation among the MraY orthologues, point to their possible involvement in substrate recognition and catalysis.

MraY was solubilized from *E. coli*, *M. luteus* and *S. aureus* by treatment with Triton X-100 or by repeated freezing and thawing.¹⁸⁷⁻¹⁸⁹ Except for a radiolabelled *in vitro* translation product,¹⁹² to date no MraY protein has been overproduced up to levels detectable by SDS-PAGE.¹⁹³,¹⁹⁵ The transferase activity is fully reversible and was found to catalyse also an exchange reaction between UMP and UDP-MurNAc-pentapeptide in the absence of undecaprenyl phosphate. Both reactions have been extensively studied and mechanisms proposed.¹⁸⁸,¹⁹⁶ In *E. coli* the *in vivo* equilibrium was shown to be greatly in favour of the nucleotide precursor.¹⁹⁷,¹⁹⁸ MraY has two substrates: undecaprenyl phosphate and UDP-MurNAc-pentapeptide. The first one is assumed to be the same in all eubacteria, but the nucleotide substrate can vary and the specificity profile of MraY towards the peptide subunit is well documented. On the basis of glycine substitution MraY was shown to have a high specificity for L-alanine in position 1 and D-alanine in position 4 but a low one for D-alanine in position 5.¹⁹⁹,²⁰⁰ On the contrary, a low specificity was observed for the diamino acid in position 3 as exemplified by the acceptance of A₂pm or L-lysine analogues (refs. 106, 168, 173, 193, 199, refs. in refs. 168 and 199) or acylated forms.²⁰¹⁻²⁰³ For instance, *S. aureus* MraY will accept the A₂pm-containing nucleotide as substrate¹⁹³ and *vice versa* *E. coli* MraY the L-lysine-containing nucleotide.¹⁰⁶ It is also noteworthy that shorter or longer peptide subunits can be accepted.¹⁰³,²⁰⁰,²⁰⁴⁻²⁰⁶

There are a number of efficient naturally occurring inhibitors of the MraY transferase activity (refs. in refs. 207–209): tunicamycin, liposidomycins, peptidylnucleotide antibiotics (mureidomycins, pacidamycins, napsamycins). None are clinically used as antibiotics for reasons of toxicity, or lack of activity against whole bacteria, owing to a too high hydrophobicity presumably impeding cell penetration. However, considering its specificity and its ubiquity limited to bacteria, MraY remains an interesting potential target for the search of novel antibacterial agents. Several studies of the mode of action and of structure–activity relationships of the peptidyl nucleotide

### 5.2 N-Acetylglucosaminyl transferase MurG

The *E. coli* N-acetylglucosaminyl transferase was identified as the product of the *murG* gene¹⁹⁸,²¹³ which had been cloned and sequenced.⁸⁹,²¹⁴ Its transcription is mainly dependent on the P<sub>*mra*</sub> promoter.¹¹⁰,¹¹¹ Moreover, as MraY, it appears to be unique and essential, since a strain with a *murG* amber mutation in a thermosensitive mutation suppressor background was found to lead to ovoid cells and lysis at the non-permissive temperature.¹⁹⁸,²¹⁵ *E. coli* MurG was shown to be associated with the cytoplasmic face of the cytoplasmic membrane,²¹⁶ thereby establishing that the entire peptidoglycan monomer unit is assembled before being transferred across the membrane (Fig. 7).

The N-acetylglucosaminyl transferase was initially solubilized from *Bacillus megaterium* by LiCl treatment and purified.²¹⁷,²¹⁸ Recently, wild type and His-tagged MurG proteins from *E. coli* were overproduced.²¹⁹,²²⁰ Although solubilized by detergents, they did not appear to be intrinsic membrane proteins since they could be purified without detergent. This is in agreement with the absence of any large hydrophobic segment in its sequence.⁸⁹,²¹⁴ Amino acid sequences of various MurG orthologues confirm its extrinsic and cationic characters. The preparation of a soluble dimer form of *E. coli* MurG enabled its crystallization and the determination of a 1.9 Å crystal structure consisting of two domains separated by a deep cleft.²²¹ The interdomain linker and the peptide segment joining the last helix of the C-domain to the last helix of the N-terminal domain define the floor of the cleft. Amino acid sequence analysis of MurG orthologues indicates that there are several invariant and highly conserved residues. The location of most of them at or near the cleft between the two domains suggests their implication in substrate binding and catalysis. In the N-terminal domain there is a hydrophobic patch surrounded by basic residues. It was proposed that this is the membrane association site and that the association involves both hydrophobic and electrostatic interactions with the negatively charged membrane. Functioning as a moderately hydrophobic cationic peripheral protein, MurG is thus a key enzyme at the junction between the two stages of peptidoglycan synthesis.

MurG catalyses the coupling of soluble donor UDP-GlcNAc to membrane-anchored acceptor lipid I. Attempts to develop an efficient one-step assay for MurG using directly lipid I as substrate have unfortunately been hampered by the difficulty in obtaining large amounts of this intermediate¹⁸⁸,²¹⁸ and assays were initially based on a reaction coupled with MraY.¹⁹⁸,²²²,²²³ Only recently have direct assays been developed with synthetic analogues of lipid I containing C₁₀ and C₃₅ chains.²²⁰,²²⁴,²²⁵ The availability of reasonable amounts of such substrates has led to the determination of kinetic parameters and to the study of enzymatic specificities.²²⁰ In particular, both the acceptor and donor specificities of *E. coli* MurG were studied with substrate analogues.²²⁰ Interestingly, UDP-MurNAc-pentapeptide functions as an acceptor substrate, albeit much less efficiently than lipid I. Structural considerations suggest that the C-terminal domain of *E. coli* MurG is the UDP-GlcNAc binding site, whereas the primary acceptor binding site is located in the N-terminal domain.²²¹ Ramoplanin is a glycolipodepsipeptide antibiotic which inhibits cell wall peptidoglycan biosynthesis in Gram-positive bacteria.²²⁶⁻²²⁸ Its target is the MurG reaction as shown with cell-free systems²²² and with purified MurG.²¹⁹ The ability of ramoplanin to interact also with lipid II and to thereby function as an inhibitor of polymerization was recently reported.²²⁹ An evaluation of certain lipid I analogues as inhibitors of MurG was recently made.²³⁰

510 Nat. Prod. Rep., 2001, 18, 503–519

acid-adding enzyme MurD.²⁴¹,²⁴² The affinity of this activator for racemase MurI is particularly high (K<sub>D</sub> = 4 μM) and is most likely the basis for a physiological mechanism regulating its in vivo activity. The formation of D-glutamic acid is thus adjusted to the requirements of peptidoglycan synthesis, thereby avoiding excessive racemization of the L-glutamic acid pool. No such activating mechanism was observed with the glutamate racemase from *P. pentosaceus*.²⁴⁶ A synthetic analogue, aziridino-glutamate, was shown to irreversibly inhibit glutamate racemase from *Lactobacillus*.²⁵⁴ The racemase from *P. pentosaceus* is inactivated with L-serine O-sulfate.²⁵⁵

The transaminase route (Fig. 8(C)) has been demonstrated in various bacterial species which possess a D-amino acid transaminase capable of synthesizing D-glutamate, among a broad range of D-amino acids, from a keto acid precursor using D-alanine as amino donor (refs. in refs. 256 and 257). The enzymes from *Bacillus subtilis*, *B. sphaericus* and *B.* sp. YM-1 were purified and characterized as pyridoxal phosphate dependent. The 3D structures of various forms of the D-amino acid transaminase from *B.* sp. YM-1 were determined and have allowed a description of the enzymatic reaction in structural terms (ref. 258 and refs. therein). A number of compounds (β-chloro-D-alanine, D-vinylglycine, D- and L-serine O-sulfate) have been shown to be potent inhibitors of the D-amino acid transaminase activity (refs. in refs. 259 and 260).

*B. subtilis*, *B. sphaericus*, and staphylococci were shown to possess both routes of D-glutamate synthesis.²⁵⁷,²⁶¹,²⁶² The genes for the racemase and transaminase activities were cloned, and each one was functional in a D-Glu-requiring *E. coli* mutant.²⁵⁷,²⁶¹ In *B. sphaericus* it was shown that either enzyme may be able to synthesize sufficient D-glutamate to sustain cell growth.²⁵⁷ It was speculated that the glutamate racemase may be sufficient to provide the necessary D-glutamate for peptidoglycan synthesis, whereas the D-amino acid transaminase is required to provide a broader range of D-amino acids for secondary metabolite syntheses. It is also noteworthy that certain bacilli are susceptible to produce D-glutamate-containing exopolypeptides.²³⁴

### 6 Side pathways

The functioning of the main pathway (Fig. 2) is dependent on a number of secondary metabolites (coenzyme A, UTP, PEP, ATP, D- and L-alanine, D-glutamic acid, diaminopimelic acid or lysine, dipeptide D-Ala-X, undecaprenyl phosphate) used as substrate at one step or another. Many are involved in various other metabolic reactions. However, the formations of D-glutamic acid and dipeptide D-Ala-X are more specific of peptidoglycan biosynthesis and that of undecaprenyl phosphate of cell wall polymer syntheses. Only these three side-pathways will be considered here. The formation of A<sub>2</sub>pm, which is the precursor of L-lysine in bacteria and which is incorporated into the peptidoglycan pathway at the MurE step in Gram-negative and in some Gram-positive organisms, has been critically reviewed.²³¹⁻²³³

#### 6.1 Formation of D-glutamate

D-Glutamic acid is a constituent unique to bacteria, not only found in position 2 of the peptide subunit⁶ but also in the capsular exopolypeptides of a few organisms.²³⁴ Two different routes (Fig. 8) have been identified for its synthesis: a direct racemase-catalysed conversion of L-glutamic acid into D-glutamic acid or a transamination process catalysed by a D-alanine aminotransferase using D-alanine and α-ketoglutarate.

Glutamate racemase activity has been detected in various bacteria and to date only in bacteria (refs. in ref. 235). Racemases from five organisms were more specifically investigated: *Pediococcus pentosaceus*,²³⁶,²³⁷ *Lactobacilli fermenti*²³⁸ and *brevis*,²³⁹ *E. coli*,²⁴⁰⁻²⁴⁶ and *Aquifex pyrophilus*.²⁴⁷ This involved the identification of a glutamate racemase gene, its cloning, its sequencing, and the overproduction of its product followed by purification to homogeneity. In the case of *E. coli* the racemase gene *murI* was shown to be essential.²⁴¹ The enzyme from *A. pyrophilus* was recently crystallized²⁴⁸ and the 3D structures of the apoenzyme and of the enzyme complexed with D-glutamine were resolved.²⁴⁹ The enzyme forms a dimer with each monomer consisting of two α/β fold domains.

Contrary to most amino acid racemases, glutamate racemases are cofactor-independent, and a two-base mechanism involving two cysteines as essential catalytic residues (Fig. 8(A)), was identified for the enzymes from *P. pentosaceus*²⁵⁰ and *L. fermenti*.²⁵¹,²⁵² A similar mechanism was established for the *E. coli* racemase.²⁴⁶,²⁵³ The involvement of two cysteine residues was also confirmed by the structural and mutational analyses of MurI from *A. pyrophilus*.²⁴⁹ Furthermore, the *E. coli* MurI racemase was shown to have an absolute requirement for UDP-MurNAc-L-alanine (Fig. 8(B)), the substrate of the D-glutamic

#### 6.2 Formation of dipeptide D-alanyl-D-X

D-Alanyl-D-alanine is formed by condensation of two molecules of D-alanine catalysed by a specific ATP-dependent amide bond forming ligase as first identified in *E. coli*, *Streptococcus faecalis* and *S. aureus* (refs. in ref. 263). Detailed kinetic and specificity studies of the reaction were initially carried out with the ligase purified from *S. faecalis*²⁶⁴ and provided evidence for two D-alanine-binding sites which have different specificity patterns and Michaelis constants.²⁶⁵ In *E. coli* a ligase gene was identified as *ddl* in the *mra* cluster.²⁶⁶,²⁶⁷ Further studies demonstrated the existence in *E. coli* and *Salmonella typhimurium* of two distinct genes, *ddlA* and *ddlB*.²⁶⁸,²⁶⁹ Both were cloned, their products overproduced and purified, and their very similar kinetic characteristics described.²⁶⁸⁻²⁷⁰ In some other bacteria it was shown that there is only one Ddl ligase.²⁷¹

Until recently, D-alanyl-D-alanine was assumed to be the only in vivo dipeptide substrate of the MurF synthetase.¹,⁶,⁸ However, it has now been established that the inducible resistance to glycopeptide antibiotics such as vancomycin was due in enterococci to the partial or complete replacement of D-alanyl-D-alanine by D-alanyl-D-lactate or D-alanyl-D-serine.⁹,²⁷²,²⁷³ Depsipeptide D-alanyl-D-lactate has also been elicited in a number of non-clinical naturally occurring vancomycin-resistant Gram-positive bacteria.²⁷⁴⁻²⁷⁶ In a few low-level vancomycin resistant enterococcal species D-alanyl-D-alanine is partially replaced by D-alanyl-D-serine.²⁷⁴,²⁷⁷ Depsipeptide D-alanyl-D-lactate and dipeptide D-alanyl-D-serine are synthesized by different ligases, which were designated as VanA, VanB, and VanC according to the vancomycin-resistance phenotype considered,²⁷³,²⁷⁸ and which were purified.²⁷⁹⁻²⁸¹ Homology analysis

analogues of D-Ala-D-X.²⁹⁹⁻³⁰³ Some of them act as ATP-dependent slow-binding inhibitors. The mechanism of the slow-binding inhibition was determined to proceed via phosphorylation at the active site to give a phosphinophosphate transition-state analogue, which dissociates very slowly from the enzyme.³⁰³ Phosphonic acid analogues of the acyl phosphate intermediate are also effective ligase inhibitors.³⁰⁴

### 6.3 Undecaprenyl phosphate cycle

In bacteria undecaprenyl phosphate is a key intermediate not only for the synthesis of lipid I but also for that of the glycosyl carrier lipids involved in the synthesis of many other cell wall polymers (refs. 1, 17, 18, 19, refs. in ref. 305). Our present knowledge of its metabolism (Fig. 10) is based on fragmentary data from various organisms, most of which was critically reviewed.¹ It originates by dephosphorylation of undecaprenyl pyrophosphate (step 2 in Fig. 10) or by phosphorylation of undecaprenol (step 3 in Fig. 10). Undecaprenyl pyrophosphate is synthesized by addition of isoprene units to farnesyl pyrophosphate (step 1 in Fig. 10) and it is also generated in the course of peptidoglycan polymerization when the disaccharide-peptide units are transferred to the extending glycan strands.¹³ In one instance, it was established that 63% of the undecaprenyl pyrophosphate originates as a recycled product of peptidoglycan polymerization and 37% by *de novo* synthesis.³⁰⁶ Therefore, undecaprenyl phosphate and undecaprenyl pyrophosphate form a cycle in which step 1 has been more specifically studied. A *cis*-prenyl pyrophosphate synthase catalyses Z-prenyl chain elongation onto (all-*E*)-farnesyl diphosphate to yield undecaprenyl pyrophosphate with *E*,*Z*-mixed stereochemistry (step 1). This activity has been described in several bacteria and most extensively studied in *Lactobacillus plantarum*.³⁰⁷ More recently, its gene was identified in *E. coli*³⁰⁸,³⁰⁹ and *Micrococcus luteus*,³¹⁰ and it was shown to be essential. Soluble over-produced forms of the enzyme from *E. coli*³⁰⁸ and *M. luteus*³¹⁰ have now been purified to homogeneity. Subsequently, the *M. luteus* synthase was crystallized³¹¹ and a site-directed mutagenesis analysis developed with the *E. coli* enzyme.³¹² Steps 2, 3, and 4 have been characterized to a very limited extent (refs. 313–315, refs. in ref. 316). Recently, the *E. coli* gene for the undecaprenol phosphokinase catalysing step 3 was tentatively identified as *bacA*.³¹⁷ Homologues of *bacA* have since been identified in the genomes of *S. aureus* and *S. pneumoniae*.³¹⁸

Although the various intermediates of the cycle (undecaprenol, undecaprenyl phosphate, undecaprenyl pyrophosphate, glycosyl carrier lipids) have been identified, their pool levels were estimated in only a few cases and never all in the same organism (refs. in ref. 1). The presence of free undecaprenol is intriguing. It has been suggested that it is a reserve pool for regulating the undecaprenyl phosphate pool by control mechanisms involving steps 3 and 4. The pool level of undecaprenyl phosphate is critical as exemplified by the different proposed mechanisms of bacitracin resistance.³⁰⁵,³¹⁷,³¹⁹ By forming a high-affinity complex with undecaprenyl pyrophosphate and a divalent cation (refs. in refs. 1, 16, 306) this antibiotic prevents dephosphorylation to undecaprenyl phosphate (step 2), thereby limiting its pool and subsequently peptidoglycan synthesis. Two resistance mechanisms imply a large increase of the undecaprenyl phosphate pool. In the first case,³¹⁷ the amplification of the *bacA* gene was shown to be sufficient to confer bacitracin resistance and correlates with an increase in membrane-associated undecaprenol kinase activity which presumably leads to an increased undecaprenyl-phosphate pool. It is noteworthy that mutations in *S. aureus* and *S. pneumoniae bacA* lead to increased susceptibility to bacitracin.³¹⁸ It was hypothesized that the absence of undecaprenol kinase leads to a reduced undecaprenyl phosphate pool. In the second case³⁰⁵ mutations confer resistance by eliminating the synthesis of cell wall exopolysaccharides competing for the undecaprenyl phosphate

Fig. 10 Undecaprenyl phosphate pathway.

pool. Similarly, partial resistance to bacitracin had been observed in *E. coli* failing to synthesize membrane-derived oligosaccharides.³²⁰ Finally, it was shown that the bacitracin resistance of *Bacillus licheniformis*, a producer of bacitracin, is mediated by the BcrC protein of the ABC transporter Bcr by binding to the antibiotic.³¹⁹

7 **In vivo functioning of the monomer unit assembly**

Most genes of the main pathway (Fig. 2) are unique and essential as initially suggested by the study of temperature-sensitive alleles.⁴⁸,³²¹ This is now also substantiated by the analysis of various bacterial genomes. In a few cases their essentiality was firmly established by null experiments.³²,⁴¹,⁴²,⁴⁹,⁵⁰,⁵²,¹⁹² The presence of two *murA* genes in low-C + G Gram-positive organisms appears as an exception.⁵³ The reasons for the existence of isoenzymes or alternative routes in the side-pathways leading to D-glutamate,²⁵⁷,²⁶¹,²⁶² A₂pm,²³³ or D-alanyl-D-alanine²⁶⁸,²⁶⁹ remains elusive, all the more so that it is not always clear which enzyme or route is preferentially functioning *in vivo*. The possibility of quantitatively analysing precursor pool levels and the availability of an accurate enzymatic *in vitro* assay for each step of the pathway have opened the way to more detailed physiological studies. The variations of the precursor pool levels and of the specific activities of many enzymes have been studied under various growth conditions, specific antibiotic treatments, or with mutants. Interesting data have also been obtained by comparing pool levels, *K*ₘ values, and rates of peptidoglycan synthesis. To date, most of the work has concerned the *E. coli* system and the overview of its *in vivo* functioning that has emerged was critically reviewed.¹⁰,¹¹ Briefly, it can be stated that many enzymes of the pathway were found to be more or less constitutive, their specific activity varying little with growth rate and only to a certain extent with growth phase.²⁸⁶,²⁸⁷,³²²,³²³ When compared with the rate of peptidoglycan synthesis, which can vary several-fold between fast and slow-growing cells,³²³ these specific activities are in excess or at least adjusted to the requirements of fast-growing cells.³²,³⁹,⁷²,¹²²,²⁸⁷,³²⁵ A fairly high capacity for peptidoglycan synthesis is maintained in slow-growing and stationary phase cells. The regulation of the pathway could thus involve specific cell effectors. A number of control mechanisms by feed-back inhibition have been proposed for different steps of the assembly of the monomer unit⁴⁰,⁵⁴,⁵⁶,⁷²,³²⁴–³²⁶ and their possible physiological significance discussed.¹,⁸,¹⁰,¹¹ Furthermore, the functioning of the pathway from UDP-GlcNAc to UDP-MurNAc-pentapeptide was shown to be unrestricted with a dependence upon the rate of input of UDP-GlcNAc, which is

controlled at some yet unidentified upstream step.³²⁷ Interestingly, the inhibition of protein synthesis leads to considerable increases of the UDP-GlcNAc and UDP-MurNAc-pentapeptide pool levels, and this effect is to some extent dependent on the relA gene product.³²⁷,³²⁸

### 8 Structural variations in the monomer unit

In principle, the most simple structure of bacterial peptidoglycan is that of an heteropolymer in which the linear glycan chains have repeating GlcNAc-β,1→4-MurNAc-peptide units and in which D-Ala→A₂pm or D-Ala→L-Lys cross-bridges are directly established between peptide subunits. However, in most bacteria, if not all, a great variety of additional structural features are encountered as revealed by detailed analyses of their peptidoglycan by chemical and enzymatic methods.¹,⁶,³²⁶ Such analyses have now been greatly facilitated by the use of reversed-phase high-performance liquid chromatography³²⁹ and, more recently, by its combination with mass spectrometry³³⁰,³³¹ and nuclear magnetic resonance spectroscopy.³³¹ Modifications vary from one organism to another and can concern the hexosamine residues (*O*-acetylation, de-*N*-acetylation, *O*-phosphorylation, *N*-glycolylation, 1,6-anhydroMurNAc cyclization, etc.) or the peptide subunit (amidation, addition of extra amino acids, etc.). Furthermore, under different circumstances (growth conditions, antibiotic treatments, mutations) the peptidoglycan of a given bacteria can undergo important modifications in the structure of its monomer unit, in glycan chain length, in extent of cross-linking, and in morphology. Examples are abundant throughout the entire peptidoglycan literature, but unfortunately only the oldest ones have as yet been reviewed.¹⁷⁶

Structural variations take place at different steps of peptidoglycan synthesis: prior to, during, or after polymerization. Those observed in the course of the assembly of the monomer unit concern more specifically the peptide subunit. In one case, a modification of the muramic acid residue in the nucleotide precursor was described.³³² As reported here and in previous reviews,¹,⁶,¹⁰ variations in the specificity of the Mur synthetases are mainly responsible for the variations observed in the amino acids of the peptide subunit. However, growth in the presence of glycine,¹⁵⁸ D-serine,¹⁸¹ A₂pm analogues (refs. 168, 173 and refs. therein), or hydroxylysine (refs. in ref. 176) can also lead to more or less specific replacements. Additional specific activities catalyse modifications in the peptide subunit such as the amidation of a carboxy group or the addition of extra amino acids. Although certain modifications involve the UDP-MurNAc-pentapeptide,³³³,³³⁴ most of them presumably occur at the level of the lipid intermediates as established in some cases.³³⁵–³³⁸ It is not always clear whether modifications take place on lipid I or lipid II, or on both. This probably leads to complex pools of the lipid intermediates which have not yet been in any way analysed.²⁰⁶

The presence of peptide cross-bridges between two peptide subunits (*n* > 0 in Fig. 1) is an important structural feature of the peptidoglycan of a number of Gram-positive organisms.¹,⁶,³²⁶ The actual cross-linking takes place by transpeptidation in the last steps of peptidoglycan synthesis,¹,⁸,¹²,¹⁴,¹⁵,³²⁶ but the cross-bridging peptides are first assembled at the level of the precursors by the stepwise addition of the branching amino acids to the ε-amino group of the L-lysine residue of the pentapeptide subunit. Two different mechanisms have been described for the formation of the branched peptide subunits. For instance, in enterococci it was found that an enzyme preparation could catalyse the *in vitro* addition of aspartate to UDP-MurNAc-pentapeptide in the presence of ATP.³³⁴ In staphylococci, depending on the species,¹,⁶,⁸,³²⁶ several glycine, alanine and serine residues are added from an aminoacyl tRNA intermediate to lipids I and II.³³⁶,³³⁷ The products of the *femA*, *femB*, and *fmhB* genes of *S.* aureus were shown to be required

---

**Fig. 11** Cross-linking of two peptide subunits by transpeptidation.  
GlcNAc: *N*-acetylglucosamine; MurNAc: *N*-acetylmuramic acid; DA: diamino acid; PBP: transpeptidase; →: –CO–NH–; D-Ala----→DA: direct linkage or by an interpeptide.

for incorporation of the branching glycine residues to the pentapeptide subunit: FmhB for the first glycine, FemA for the second and third ones, and FemB for the fourth and fifth ones.³³⁹,³⁴⁰ Similarly, in *S.* pneumoniae the *murM* and *murN* genes, homologous to *fmhB*, were found to be involved in the formation of the branched precursors.³⁴¹ A soluble UDP-MurNAc-pentapeptide : L-alanine ligase was initially purified and characterized from *Weissella viridescens*.³³³ Recently, transpeptidases catalysing the addition of the branching amino acids in *W.* viridescens and *Enterococcus faecalis* were overproduced and purified.³⁴²,³⁴³

### 9 Flexibility in peptidoglycan synthesis

Since the assembly of the monomer unit from glucosamine-6-phosphate to lipid II is essentially a unique sequential process, any variation at a given step must be accepted by the following ones. However, the specificity requirements can differ from one step to another in the pathway as reviewed here and previously.¹,⁸,¹⁰ Therefore, the cumulative effect of the different specificities along the pathway has a restrictive effect limiting the structural variability of the complete monomer unit. This is particularly true for the Mur synthetases catalysing the formation of the peptide subunit where only a limited number of structural variations are encountered in its five positions. Although studied to a lesser extent, we should have a similar situation with the subsequent steps of peptidoglycan synthesis. However, it has been shown that various *in vivo* and cell-free systems catalysing the synthesis of peptidoglycan from UDP-MurNAc-pentapeptide will accept important variations of the diamino acid in position 3,¹⁰⁶,¹⁶⁸,¹⁷³,¹⁹³,¹⁹⁹,²⁰¹–²⁰³ or of the peptide chain length.¹⁰³,²⁰⁰,²⁰⁴–²⁰⁶,³⁴⁴ This indicates that the MraY and MurG transferases as well as the glycosyltransferases catalysing the formation of the glycan chains¹³ have the same low-specificity profile for these two structural features of the peptide subunit.

In contrast, the specificity of the transpeptidation step is far more dependent on the structure of the peptide subunit. This is presumably due to the implication of two different parts of the peptide subunit in the transpeptidation reaction. Transpeptidases¹,⁸,¹²,¹⁴,¹⁵ catalyse the cross-linking between two peptide subunits by the formation of a peptide bond between the carboxy group of the D-alanine in position 4 of a donor peptide subunit and the amino group of an acceptor peptide subunit, concomitantly with the release of the donor C-terminal D-Ala (Fig. 11). Understandably, their specificity for the donor subunit is determined mainly by the C-terminal dipeptide, whereas that for the acceptor subunit is determined mainly by the diamino

514 Nat. Prod. Rep., 2001, 18, 503–519

acid or the N-terminus of the branching peptide. This is clearly apparent when two different peptide subunits are involved in transpeptidation as observed in some particular cases. For instance, the high specificity for the diamino acid in the acceptor is illustrated in *E. coli* where partial replacement of meso-A<sub>2</sub>pm by LL-A<sub>2</sub>pm, L-lysine or analogues results in peptide subunits which can be used as donor but not as acceptor.<sup>106,168,173</sup> This replacement results in a decrease of the extent of cross-linking. Moreover, peptide subunits with a blocked ε-amino A<sub>2</sub>pm can still function as donors.<sup>201</sup> Conversely, tetrapeptide or tripeptide subunits lacking one or both C-terminal D-alanine residues can still function as acceptors.<sup>200,345</sup> When the C-terminal positions 4 and 5 of the subunit are considered, generally transpeptidases have a higher degree of specificity for D-alanine in position 4 than for the one in position 5. This was observed with the peptidoglycan-synthesizing system from *Gaffkya homari*<sup>346</sup> and with many organisms grown in the presence of glycine or D amino acids which replace D-alanine.<sup>158,181,347</sup> Here too, the replacement of D-alanine in position 4 often leads to decreased cross-linking. The lower specificity for the C-terminal position was also illustrated by the replacement of the D-Ala-D-Ala dipeptide by D-Ala-D-lactate or D-Ala-D-Ser, which has little effect on transpeptidation, as observed in vancomycin-resistant Gram-positive organisms.<sup>9,273</sup> Aside these different examples, it was found that transpeptidation could depend on a much more subtle structural feature of the peptide subunit, such as the amidation of the α-carboxy of the D-glutamate residue. In *G. homari* enhanced deamidation of the acceptor tetrapeptide was correlated with the inhibition of the main cross-linking.<sup>348</sup>

Transpeptidases have been characterized as penicillin-binding proteins (PBP) since they are specifically inhibited by the covalent binding of β-lactam antibiotics to their active site.<sup>1,8,12,14,15</sup> Among the various PBPs of a given organism, it is likely that more than one PBP is involved in the transpeptidation reactions. Moreover, the specificities for the acceptor and donor subunits as well as the affinities for β-lactam antibiotics can differ from one PBP to another PBP. This opens the way to many possible correlations between the synthesis of the peptidoglycan precursors, the *in vivo* functioning of the transpeptidases and the susceptibility to β-lactams.<sup>349</sup> In fact, changes in the biosynthesis of the precursors (variations of pool levels, shortening of the peptide cross-bridge, etc.) have now been correlated with changes in the susceptibility to β-lactam antibiotics. This is well illustrated in staphylococci where some of the numerous genes essential for high-level resistance to methicillin are involved in the assembly of the monomer unit.<sup>350</sup> The mechanisms underlying these correlations are yet poorly understood. Similarly, in *S. pneumoniae*<sup>341</sup> and *E. faecium*<sup>351</sup> the shortening of the peptide cross-bridges, subsequent to variations in growth conditions or mutations, also results in changes in the susceptibility to β-lactams. In certain of these examples, where low and high affinity PBPs are present simultaneously, the decrease in penicillin resistance is presumably due to the preferential use of the modified peptide subunit by high affinity PBPs rather than by the low affinity ones. Provided that these variations are compatible with the survival of the bacteria and that one or more of these transpeptidases are also essential targets, their susceptibility versus resistance behaviour towards β-lactam antibiotics will be modified.

### 10 Concluding remarks

In the last 10–15 years rapid and important progress has been made in the genetic and biochemical study of the various steps of the assembly of the peptidoglycan monomer unit. Out of the twelve enzymes of the main pathway, only the MraY transferase has not yet been overproduced to a useful level for purification, owing presumably to the deleterious effect of its hydrophobicity. Furthermore, purified GlmM mutase remains to be crystallized and the 3D structure of crystallized MurC to be resolved. The elucidation of the mechanisms of most of these enzymes, together with that of their 3D structures will undoubtedly be useful for developing the rational design of specific and potent inhibitors.

Only a few control mechanisms of possible physiological significance have been proposed and the regulation of the pathway as a whole is still poorly understood. In particular, the regulation of the UDP-GlcNAc pool, which is an important branching point in many organisms for the synthesis of the cell wall polymers, remains to be studied, all the more so that its four-step synthesis from the intermediate metabolism is now well established. Progress in the study of the formation of the membrane intermediates has been slower than with that of the cytoplasmic precursors. However, the determination of the topology of the MraY transferase, the resolution of the MurG structure, the biochemical characterization of the transferases catalysing the formation of the peptide cross-bridges, and the availability of synthetic analogues of lipid I are encouraging results. They should help to develop the study of the membrane organization of the various transferases and that of the yet unexplored translocation of lipid II to the sites of polymerization. Appropriate methods for analysing the pools of the different lipid intermediates as well as for the large-scale preparation of the lipid intermediates are crucial for the development of the study of the membrane steps.

The elucidation of different subtle mechanisms of drug resistance and genetic engineering experiments have extended our knowledge of the limits of the flexibility of peptidoglycan synthesis, which was previously only accessible by analysis of peptidoglycan structures. A better understanding of this flexibility is important not only for evaluating the extent of variability of its structure, but also for determining mechanisms of drug resistance and foreseeing potential resistance mechanisms which could arise upon the use of new drugs.

### 11 Acknowledgements

This work was supported by grant UMR8619 from the Centre National de la Recherche Scientifique and grant PRFMMIP from the Ministère de l’Education Nationale, de la Recherche et de la Technologie. I thank Didier Blanot (Université Paris-Sud, France) and Dominique Mengin-Lecreulx (Université Paris-Sud, France) for their comments and suggestions.

### 12 References

1. H. J. Rogers, H. R. Perkins and J. B. Ward, *Microbial Cell Walls and Membranes*, Chapman & Hall, London, 1980.
2. M. R. J. Salton, in *Bacterial Cell Wall*, ed. J.-M. Ghuysen and R. Hakenbeck, Elsevier Science B.V., Amsterdam, 1994, p. 1.
3. J. T. Park, in *Escherichia coli and Salmonella: Cellular and Molecular Biology*, ed. F. C. Neidhardt, R. Curtis III, J. L. Ingraham, E. C. C. Lin, K. B. Low, B. Magasanik, W. S. Reznikoff, M. Riley, M. Schaechter and H. E. Umbarger, ASM Press, Washington, D.C., 1996, p. 48.
4. N. Nanninga, *Microbiol. Mol. Biol. Rev.*, 1998, **62**, 110.
5. L. B. Guze, *Microbial Protoplasts, Spheroplasts and L-forms*, Williams & Wilkins, Baltimore, MD, 1968.
6. K. H. Schleifer and O. Kandler, *Bacteriol. Rev.*, 1972, **36**, 407.
7. J. T. Park, *J. Biol. Chem.*, 1952, **194**, 877.
8. B. Ward, *Pharmacol. Ther.*, 1984, **25**, 327.
9. T. D. H. Bugg and C. T. Walsh, *Nat. Prod. Rep.*, 1992, **9**, 199.
10. J. van Heijenoort, in *Bacterial Cell Wall*, ed. J.-M. Ghuysen and R. Hakenbeck, Elsevier Science B.V., Amsterdam, 1994, p. 39.
11. J. van Heijenoort, in *Escherichia coli and Salmonella: Cellular and Molecular Biology*, ed. F. C. Neidhardt, R. Curtis III, J. L. Ingraham, E. C. C. Lin, K. B. Low, B. Magasanik, W. S. Reznikoff, M. Riley, M. Schaechter and H. E. Umbarger, ASM Press, Washington, D.C., 1996, p. 1025.
12. C. Goffin and J.-M. Ghuysen, *Microbiol. Mol. Biol. Rev.*, 1998, **62**, 1079.
13. J. van Heijenoort, *Glycobiology*, 2001, **11**, 25R.
14. M. Matsuhashi, in *Bacterial Cell Wall*, ed. J.-M. Ghuysen and R. Hakenbeck, Elsevier Science B.V., Amsterdam, 1994, p. 55.

15 J.-M. Ghuyse and G. Dive, in *Bacterial Cell Wall*, ed. J.-M. Ghuyse and R. Hakenbeck, Elsevier Science B.V., Amsterdam, 1994, p. 103.

16 E. F. Gale, E. Cundliffe, P. E. Reynolds, M. H. Richmond and M. J. Waring, *The Molecular Basis of Antibiotic Action*, John Wiley and Sons, London, 1981.

17 H. M. Pooley and D. Karamata, in *Bacterial Cell Wall*, ed. J.-M. Ghuyse and R. Hakenbeck, Elsevier Science B.V., Amsterdam, 1994, p. 187.

18 C. R. H. Raetz, in *Escherichia coli and Salmonella: Cellular and Molecular Biology*, ed. F. C. Neidhardt, R. Curtis III, J. L. Ingraham, E. C. C. Lin, K. B. Low, B. Magasanik, W. S. Reznikoff, M. Riley, M. Schaechter and H. E. Umbarger, ASM Press, Washington, D.C., 1996, p. 1035.

19 P. D. Rick and R. P. Silver, in *Escherichia coli and Salmonella: Cellular and Molecular Biology*, ed. F. C. Neidhardt, R. Curtis III, J. L. Ingraham, E. C. C. Lin, K. B. Low, B. Magasanik, W. S. Reznikoff, M. Riley, M. Schaechter and H. E. Umbarger, ASM Press, Washington, D.C., 1996, p. 104.

20 R. J. White, *Biochem. J.*, 1968, **106**, 847.

21 W. J. Dobrogosz, *J. Bacteriol.*, 1968, **95**, 578.

22 H. C. Wu and T. C. Wu, *J. Bacteriol.*, 1971, **105**, 455.

23 B. Badet, P. Vermoote, P.-Y. Haumont, F. Lederer and F. Le Goffic, *Biochemistry*, 1987, **26**, 1940.

24 M.-A. Badet-Denisot, L. A. Fernandez-Herrero, J. Berenguer, T. Ooi and B. Badet, *Arch. Biochem. Biophys.*, 1997, **337**, 129.

25 M. N. Isupov, G. Obmolova, S. Butterworth, M.-A. Badet-Denisot, B. Badet, I. Polikarpov, J. A. Littlechild and A. Teplyakov, *Structure*, 1996, **4**, 801.

26 A. Teplyakov, G. Obmolova, M.-A. Badet-Denisot, B. Badet and I. Polikarpov, *Structure*, 1998, **6**, 1047.

27 S. L. Bearne, *J. Biol. Chem.*, 1996, **271**, 3052.

28 C. Leriche, M.-A. Badet-Denisot and B. Badet, *Eur. J. Biochem.*, 1997, **245**, 418.

29 S. L. Bearne and C. Blouin, *J. Biol. Chem.*, 2000, **275**, 135.

30 A. Teplyakov, G. Obmolova, M.-A. Badet-Denisot and B. Badet, *Protein Sci.*, 1999, **8**, 596.

31 B. Walker, M. F. Brown, J. F. Lynas, S. L. Martin, A. McDowell, B. Badet and A. J. Hill, *Bioorg. Med. Chem. Lett.*, 2000, **10**, 2795.

32 D. Mengin-Lecreulx and J. van Heijenoort, *J. Biol. Chem.*, 1996, **271**, 32.

33 H. de Reuse, A. Labigne and D. Mengin-Lecreulx, *J. Bacteriol.*, 1997, **179**, 3488.

34 L. Jolly, S. Wu, J. van Heijenoort, H. de Lencastre, D. Mengin-Lecreulx and A. Tomasz, *J. Bacteriol.*, 1997, **179**, 5321.

35 I. M. Tavares, L. Jolly, F. Pompeo, J. H. Leitao, A. M. Fialho, I. Sa-Correia and D. Mengin-Lecreulx, *J. Bacteriol.*, 2000, **182**, 4453.

36 P. Glanzmann, J. Gustafson, H. Komatsuzawa, K. Ohta and B. Berger-Bächi, *Antimicrob. Agents Chemother.*, 1999, **43**, 240.

37 L. Jolly, P. Ferrari, D. Blanot, J. van Heijenoort, F. Fassy and D. Mengin-Lecreulx, *Eur. J. Biochem.*, 1999, **262**, 202.

38 L. Jolly, F. Pompeo, J. van Heijenoort, F. Fassy and M. Mengin-Lecreulx, *J. Bacteriol.*, 2000, **182**, 1280.

39 D. Mengin-Lecreulx and J. van Heijenoort, *J. Bacteriol.*, 1993, **175**, 6150.

40 D. Mengin-Lecreulx and J. van Heijenoort, *J. Bacteriol.*, 1994, **176**, 5788.

41 F. Pompeo, Y. Bourne, J. van Heijenoort, F. Fassy and D. Mengin-Lecreulx, *J. Biol. Chem.*, 2001, **276**, 3833.

42 A. M. Gehring, W. J. Lees, D. J. Mindiola, C. T. Walsh and E. D. Brown, *Biochemistry*, 1996, **35**, 579.

43 F. Pompeo, J. van Heijenoort and D. Mengin-Lecreulx, *J. Bacteriol.*, 1998, **180**, 4799.

44 K. Brown, F. Pompeo, S. Dixon, D. Mengin-Lecreulx, C. Cambillau and Y. Bourne, *EMBO J.*, 1999, **18**, 4096.

45 D. Kostrewa, A. D’Arcy, B. Takacs and M. Kamber, *J. Mol. Biol.*, 2001, **305**, 279.

46 H. C. Wu and P. S. Venkateswaran, *Ann. N.Y. Acad. Sci.*, 1974, **235**, 587.

47 H. Matsuzawa, M. Matsuhashi, A. Oka and Y. Sugino, *Biochem. Biophys. Res. Commun.*, 1969, **36**, 682.

48 T. Miyakawa, H. Matsuzawa, M. Matsuhashi and Y. Sugino, *J. Bacteriol.*, 1972, **112**, 950.

49 M. J. Pucci, L. F. Discotto and T. J. Dougherty, *J. Bacteriol.*, 1992, **174**, 1690.

50 P. Doublet, J. van Heijenoort, D. Mengin-Lecreulx, *Bacterial Growth and Lysis, Metabolism and Structure of the Bacterial Sacculus*, ed. M. A. de Pedro, J.-V. Höltje and W. Löffelhardt, Plenum Press, New York, 1993, p. 139.

51 J. L. Marquardt, D. A. Siegele, R. Kolter and C. T. Walsh, *J. Bacteriol.*, 1992, **174**, 5748.

52 E. D. Brown, E. I. Vivas, C. T. Walsh and R. Kolter, *J. Bacteriol.*, 1995, **177**, 4194.

53 W. Du, J. R. Brown, D. R. Sylvester, J. Huang, A. F. Chalker, C. Y. So, D. J. Holmes, D. J. Payne and N. G. Wallis, *J. Bacteriol.*, 2000, **182**, 4146.

54 G. G. Wickus and J. L. Strominger, *J. Bacteriol.*, 1973, **113**, 287.

55 P. J. Cassidy and F. M. Kahan, *Biochemistry*, 1973, **12**, 1364.

56 R. I. Zemell and R. A. Anwar, *J. Biol. Chem.*, 1975, **250**, 3185.

57 C. Wanke, R. Falchetto and N. Amrhein, *FEBS Lett.*, 1992, **301**, 271.

58 G. G. Wickus, P. A. Rubenstein, A. D. Warth and J. L. Strominger, *J. Bacteriol.*, 1973, **113**, 291.

59 R. A. Anwar and M. Vlaovic, *Can. J. Biochem.*, 1979, **57**, 188.

60 T. E. Benson, J. L. Marquardt, A. C. Marquardt, F. A. Etzkorn and C. T. Walsh, *Biochemistry*, 1993, **32**, 2024.

61 M. A. Tayeh, G. D. Dotson, J. C. Clemens and R. W. Woodward, *Protein Express. Purif.*, 1995, **6**, 757.

62 T. E. Benson, M. S. Harris, G. H. Choi, J. I. Cialdella, J. T. Herberg, J. P. Martin and E. T. Baldwin, *Biochemistry*, 2001, **40**, 2340.

63 S. Sack, Z. Dauter, C. Wanke and N. Amrhein, *J. Struct. Biol.*, 1996, **117**, 73.

64 T. Skarzynski, A. Mistry, A. Wonacott, S. E. Hutchinson, V. A. Kelly and K. Duncan, *Structure*, 1996, **4**, 1465.

65 T. E. Benson, C. T. Walsh and J. M. Hogle, *Protein Sci.*, 1994, **3**, 1125.

66 S. L. Ohringer, C. Y. Chang, S. Sheriff, H. E. Klei, J. S. Sack, B. L. Jacobson, J. Yanchunas, T. Lavoie, A. M. Dhalla, J. G. Robertson and H. M. Einspahr, *Acta Crystallogr., Sect. D*, 1996, **52**, 586.

67 E. Schönbrunn, S. Sack, S. Eschenburg, A. Perrakis, F. Krekel, N. Amrhein and E. Mandelkow, *Structure*, 1996, **4**, 1065.

68 T. E. Benson, D. J. Filman, C. T. Walsh and J. M. Hogle, *Nat. Struct. Biol.*, 1995, **2**, 644.

69 T. E. Benson, C. T. Walsh and J. M. Hogle, *Structure*, 1996, **4**, 47.

70 B. T. Farmer, K. L. Constantine, V. Goldfarb, M. S. Friedrichs, M. Wittekind, J. Yanchunas, J. G. Robertson and L. Mueller, *Nat. Struct. Biol.*, 1996, **3**, 995.

71 K. L. Constantine, L. Mueller, V. Goldfarb, M. Wittekind, W. J. Metzler, J. Yanchunas, J. G. Robertson, M. F. Malley, M. S. Friedrichs and B. T. Farmer, *J. Mol. Biol.*, 1997, **267**, 1223.

72 D. Mengin-Lecreulx, B. Flouret and J. van Heijenoort, *J. Bacteriol.*, 1983, **154**, 1284.

73 A. Bouhss, S. Dementin, J. van Heijenoort, C. Parquet and D. Blanot, *FEBS Lett.*, 1999, **453**, 15.

74 T. Skarzynski, D. H. Kim, W. J. Lees, C. T. Walsh and K. Duncan, *Biochemistry*, 1998, **37**, 2572.

75 E. D. Brown, J. L. Marquardt, J. P. Lee, C. T. Walsh and K. S. Anderson, *Biochemistry*, 1994, **33**, 10638.

76 C. Wanke and N. Amrhein, *Eur. J. Biochem.*, 1993, **218**, 861.

77 H. M. Salleh, G. D. Dotson and R. W. Woodward, *Bioorg. Med. Chem. Lett.*, 1996, **6**, 133.

78 D. H. Kim, W. J. Lees, K. E. Kempsell, W. S. Lane, K. Duncan and C. T. Walsh, *Biochemistry*, 1996, **35**, 4923.

79 E. Schönbrunn, D. I. Svergun, N. Amrhein and M. H. J. Koch, *Eur. J. Biochem.*, 1998, **253**, 406.

80 A. K. Samland, N. Amrhein and P. Macheroux, *Biochemistry*, 1999, **38**, 13162.

81 S. Eschenburg and E. Schönbrunn, *Proteins*, 2000, **40**, 290.

82 A. K. Samland, T. Etezady-Esfarjani, N. Amrhein and P. Macheroux, *Biochemistry*, 2001, **40**, 1550.

83 F. M. Kahan, J. S. Kahan, P. J. Cassidy and H. Kropp, *Ann. N.Y. Acad. Sci.*, 1974, **235**, 364.

84 K. A. L. De Smet, K. E. Kempsell, A. Gallagher, K. Duncan and D. B. Young, *Microbiol.*, 1999, **145**, 3177.

85 T. E. Benson, C. T. Walsh and V. Massey, *Biochemistry*, 1997, **36**, 796.

86 A. M. Dhalla, J. Yanchunas, H.-T. Ho, P. J. Falk, J. J. Villafranca and J. G. Robertson, *Biochemistry*, 1995, **34**, 5390.

87 C. J. Andres, J. J. Bronson, S. V. D’Andrea, M. S. Deshpande, P. J. Falk, K. A. Grant-Young, W. E. Harte, H.-T. Ho, P. F. Misco, J. G. Robertson, D. Stock, Y. Sun and A. W. Walsh, *Bioorg. Med. Chem. Lett.*, 2000, **10**, 715.

88 E. J. J. Lugtenberg and A. van Schijndel, *J. Bacteriol.*, 1972, **110**, 35.

89 M. Ikeda, M. Wachi, H. K. Jung, F. Ishino and M. Matsuhashi, *Nucleic Acids Res.*, 1990, **18**, 4014.

90 T. Ansai, Y. Yamashita, S. Awano, Y. Shibata, M. Wachi, K. Nagai and T. Takehara, *Microbiology*, 1995, **141**, 2047.

91 A. M. Lowe and R. L. Deresiewicz, *DNA Sequence*, 1999, **10**, 19.

92 M. Wachi, C. D. Wijayarathna, H. Teraoka and K. Nagai, *Appl. Microbiol. Biotechnol.*, 1999, **51**, 223.

93 D. Mengin-Lecreulx, C. Parquet, L. R. Desviat, J. Plá, B. Flouret, J. Ayala and J. van Heijenoort, *J. Bacteriol.*, 1989, **171**, 6126.

94 D. Mengin-Lecreulx and J. van Heijenoort, *Nucleic Acids Res.*, 1990, **18**, 183.

95 M. Ikeda, M. Wachi, F. Ishino and M. Matsushima, *Nucleic Acids Res.*, 1990, **18**, 1058.

96 A. O. Henriques, H. de Lencastre and P. J. Piggot, *Biochimie*, 1992, **74**, 735.

97 R. A. Daniel and J. Errington, *J. Gen. Microbiol.*, 1993, **139**, 361.

98 M. J. Pucci, J. A. Thanassi, L. F. Discotto, R. E. Kessler and T. J. Dougherty, *J. Bacteriol.*, 1997, **179**, 5632.

99 M. El Sherbeini, W. M. Geissler, J. Pittman, X. Yuan, K. K. Wong and D. L. Pompliano, *Gene*, 2000, **210**, 117.

100 I. N. Maruyama, A. H. Yamamoto and Y. Hirota, *J. Bacteriol.*, 1988, **170**, 3786.

101 J. S. Tao and E. E. Ishiguro, *Can. J. Microbiol.*, 1989, **35**, 1051.

102 C. Michaud, C. Parquet, B. Flouret, D. Blanot and J. van Heijenoort, *Biochem. J.*, 1990, **269**, 277.

103 A. Ornelas-Soares, H. de Lencastre, B. J. M. de Jonge and A. Tomasz, *J. Biol. Chem.*, 1994, **269**, 27246.

104 X. Liao, I. Charlebois, C. Ouellet, M. J. Morency, K. Dewar, J. Lightfoot, J. Foster, R. Siehnel, H. Schweitzer, J. S. Lam, R. E. Hancock and R. C. Levesque, *Microbiology*, 1996, **142**, 79.

105 A. M. Ludovice, S. W. Wu and H. de Lencastre, *Microb. Drug Resist.*, 1998, **4**, 85.

106 D. Mengin-Lecreulx, T. Falla, D. Blanot, J. van Heijenoort, D. J. Adams and I. Chopra, *J. Bacteriol.*, 1999, **181**, 5909.

107 C. Parquet, B. Flouret, D. Mengin-Lecreulx and J. van Heijenoort, *Nucleic Acid Res.*, 1989, **17**, 5379.

108 M. P. Malakhov, D. A. Los, H. Wada, V. E. Semenenko and N. Murata, *Microbiology*, 1995, **141**, 163.

109 M. S. Anderson, S. S. Eveland, H. R. Onishi and D. L. Pompliano, *Biochemistry*, 1996, **35**, 16264.

110 H. Hara, S. Yasuda, K. Horiuchi and J. T. Park, *J. Bacteriol.*, 1997, **179**, 5802.

111 D. Mengin-Lecreulx, J. Ayala, A. Bouhss, J. van Heijenoort, C. Parquet and H. Hara, *J. Bacteriol.*, 1998, **180**, 4406.

112 C. Duez, I. Thamm, F. Sapunarić, J. Coyette and J.-M. Ghuyse, *DNA Sequence*, 1998, **9**, 149.

113 O. Massidda, D. Anderluzzi, L. Friedli and G. Feger, *Microbiology*, 1998, **144**, 3069.

114 F. Francis, S. Ramirez-Arcos, H. Salimnia, C. Victor and J. R. Dillon, *Gene*, 2000, **251**, 141.

115 S. Mahapatra, D. C. Crick and P. J. Brennan, *J. Bacteriol.*, 2000, **182**, 6827.

116 D. Mengin-Lecreulx, J. van Heijenoort and J. T. Park, *J. Bacteriol.*, 1996, **178**, 5347.

117 J. T. Park, D. Raychaudhuri, H. Li, S. Normark and D. Mengin-Lecreulx, *J. Bacteriol.*, 1998, **180**, 1215.

118 M. Ikeda, M. Wachi, H. K. Jung, F. Ishino and M. Matsushima, *J. Gen. Appl. Microbiol.*, 1990, **36**, 179.

119 S. S. Eveland, D. L. Pompliano and M. S. Anderson, *Biochemistry*, 1997, **36**, 6223.

120 A. Bouhss, D. Mengin-Lecreulx, D. Blanot, J. van Heijenoort and C. Parquet, *Biochemistry*, 1997, **36**, 11556.

121 Y. Mizuno, M. Yaegashi and E. Ito, *J. Biochem.*, 1973, **74**, 525.

122 D. Liger, D. Blanot and J. van Heijenoort, *FEMS Microbiol. Lett.*, 1991, **80**, 111.

123 D. Liger, A. Masson, D. Blanot, J. van Heijenoort and C. Parquet, *Eur. J. Biochem.*, 1995, **230**, 80.

124 D. Liger, A. Masson, D. Blanot, J. van Heijenoort and C. Parquet, *Microb. Drug Resist.*, 1996, **2**, 25.

125 M. Gubler, Y. Appoldt and W. Keck, *J. Bacteriol.*, 1996, **178**, 906.

126 P. J. Falk, K. M. Ervin, K. S. Volk and H.-T. Ho, *Biochemistry*, 1996, **35**, 1417.

127 H. Jin, J. J. Emanuele, R. Fairman, J. G. Robertson, M. E. Hail, H.-T. Ho, P. J. Falk and J. J. Villafranca, *Biochemistry*, 1996, **35**, 1423.

128 J. J. Emanuele, H. Jin, B. L. Jacobson, C. Y. Chang, H. M. Einspahr and J. J. Villafranca, *Protein Sci.*, 1996, **5**, 2566.

129 K. D. Pryor and B. Leiting, *Protein Express. Purif.*, 1997, **10**, 309.

130 A. El Zoeiby, F. Sanschagrin, J. Lamoureux, A. Darveau and R. C. Levesque, *FEMS Microbiol. Lett.*, 2000, **183**, 281.

131 S. G. Nathenson, J. L. Strominger and E. Ito, *J. Biol. Chem.*, 1964, **239**, 1773.

132 F. Pratviel-Sosa, D. Mengin-Lecreulx and J. van Heijenoort, *Eur. J. Biochem.*, 1991, **202**, 1169.

133 G. Auger, L. Martin, J. Bertrand, P. Ferrari, E. Fanchon, S. Vaganay, Y. Pétillot, J. van Heijenoort, D. Blanot and O. Dideberg, *Protein Express. Purif.*, 1998, **13**, 23.

134 M. El-Sherbeini, W. M. Geissler, J. Pittman, X. Yuan, K. K. Wong and D. L. Pompliano, *Gene*, 1998, **210**, 117.

135 A. W. Walsh, P. J. Falk, J. Thanassi, L. Discotto, M. J. Pucci and H.-T. Ho, *J. Bacteriol.*, 1999, **181**, 5395.

136 E. Ito and J. L. Strominger, *J. Biol. Chem.*, 1964, **239**, 210.

137 Y. Mizuno and E. Ito, *J. Biol. Chem.*, 1968, **243**, 2665.

138 W. P. Hammes, R. Neukam and O. Kandler, *Arch. Microbiol.*, 1977, **115**, 95.

139 R. A. Anwar and M. Vlaovic, *Biochem. Cell Biol.*, 1986, **64**, 297.

140 C. Michaud, D. Mengin-Lecreulx, J. van Heijenoort and D. Blanot, *Eur. J. Biochem.*, 1990, **194**, 853.

141 E. Gordon, B. Flouret, L. Chantlat, J. van Heijenoort, D. Mengin-Lecreulx and O. Dideberg, *J. Biol. Chem.*, 2001, **276**, 10999.

142 D. G. Comb, *J. Biol. Chem.*, 1962, **237**, 1601.

143 K. Duncan, J. van Heijenoort and C. T. Walsh, *Biochemistry*, 1990, **29**, 2379.

144 Y. Yan, S. Munshi, Y. Li, K. A. Pryor, F. Marsilio and B. Leiting, *Acta Crystallogr.*, Sect. D, 1999, **55**, 2033.

145 J. A. Bertrand, G. Auger, E. Fanchon, L. Martin, D. Blanot, J. van Heijenoort and O. Dideberg, *EMBO J.*, 1997, **16**, 3416.

146 J. A. Bertrand, G. Auger, L. Martin, E. Fanchon, D. Blanot, D. L. Le Beller, J. van Heijenoort and O. Dideberg, *J. Mol. Biol.*, 1999, **289**, 579.

147 N. Y. Chirgadze, G. M. Birch, L. C. Blaszczak, S. L. Briggs, D. K. Clawson, N. G. Halligan, J. E. Mcgee, S. Sigmund, M. C. Smith and R. W. Schevitz, *Proceedings of the XVIII World Crystallography Congress*, Glasgow, 1999, p. 370.

148 Y. Yan, S. Munshi, B. Leiting, M. S. Anderson, J. Chrzas and Z. Chen, *J. Mol. Biol.*, 2000, **304**, 435.

149 Y. Sheng, X. Sun, Y. Shen, A. L. Bognar, E. N. Baker and C. A. Smith, *J. Mol. Biol.*, 2000, **302**, 427.

150 S. Vaganay, M. E. Tanner, J. van Heijenoort and D. Blanot, *Microb. Drug Resist.*, 1996, **2**, 51.

151 J. J. Emanuele, H. Jin, J. Yanchunas and J. J. Villafranca, *Biochemistry*, 1997, **36**, 7264.

152 A. Bouhss, S. Dementin, J. van Heijenoort, C. Parquet and D. Blanot, *FEBS Lett.*, 1999, **453**, 15.

153 A. Egan, P. Lawrence and J. L. Strominger, *J. Biol. Chem.*, 1973, **248**, 3122.

154 M. E. Tanner, S. Vaganay, J. van Heijenoort and D. Blanot, *J. Org. Chem.*, 1996, **61**, 1756.

155 B. Zeng, K. K. Wong, D. L. Pompliano, S. Reddy and M. E. Tanner, *J. Org. Chem.*, 1998, **63**, 10081.

156 L. D. Gegnas, S. T. Waddell, R. M. Chabin, S. Reddy and K. K. Wong, *Bioorg. Med. Chem. Lett.*, 1998, **8**, 1643.

157 E. E. Ishiguro, *Can. J. Microbiol.*, 1982, **28**, 654.

158 W. Hammes, K. H. Schleifer and O. Kandler, *J. Bacteriol.*, 1973, **116**, 1029.

159 C. Michaud, D. Blanot, B. Flouret and J. van Heijenoort, *Eur. J. Biochem.*, 1987, **166**, 631.

160 A. W. Wyke and H. R. Perkins, *J. Gen. Microbiol.*, 1975, **88**, 159.

161 F. Nosal, A. Masson, R. Legrand, D. Blanot, B. Schoot, J. van Heijenoort and C. Parquet, *FEBS Lett.*, 1998, **426**, 309.

162 F. Pratviel-Sosa, F. Acher, F. Trigalo, D. Blanot, R. Azerad and J. van Heijenoort, *FEMS Lett.*, 1994, **115**, 223.

163 G. Auger, J. van Heijenoort and D. Blanot, *J. Prakt. Chem.*, 1995, **337**, 351.

164 S. Gobec, U. Urleb, G. Auger and D. Blanot, *Pharmacie*, 2001, **56**, 295.

165 J. A. Bertrand, E. Fanchon, L. Martin, L. Chantlat, G. Auger, D. Blanot, J. van Heijenoort and O. Dideberg, *J. Mol. Biol.*, 2000, **301**, 1257.

166 A. Bouhss, S. Dementin, C. Parquet, D. Mengin-Lecreulx, J. A. Bertrand, D. Le Beller, O. Dideberg, J. van Heijenoort and D. Blanot, *Biochemistry*, 1999, **38**, 12240.

167 A. Fredriksen, E. N. Vassstrand and H. B. Jensen, *J. Bacteriol.*, 1991, **173**, 900.

168 D. Mengin-Lecreulx, D. Blanot and J. van Heijenoort, *J. Bacteriol.*, 1994, **176**, 4321.

169 D. J. Tipper and I. Pratt, *J. Bacteriol.*, 1970, **103**, 305.

170 D. Dietzler, R. Threnn, E. Ito and J. L. Strominger, *An. Asoc. Quím. Argent.*, 1972, **60**, 141.

171 E. Ito and J. L. Strominger, *J. Biol. Chem.*, 1973, **248**, 3131.

172 C. E. Buchanan, A. O. Henriques and P. J. Piggot, in *Bacterial Cell Wall*, ed. J.-M. Ghuyse and R. Hakenbeck, Elsevier Science B.V., Amsterdam, 1994, p. 167.

173 D. Mengin-Lecreulx, C. Michaud, C. Richaud, D. Blanot and J. van Heijenoort, *J. Bacteriol.*, 1988, **170**, 2031.

174 I. van Assche, M. Soroka, A. Haemers, M. Hooper, D. Blanot and J. van Heijenoort, *Eur. J. Med. Chem.*, 1991, **26**, 505.

175 G. Auger, J. van Heijenoort, J. C. Vederas and D. Blanot, *FEBS Lett.*, 1996, **391**, 171.

176 K. H. Schleifer, W. P. Hammes and O. Kandler, *Adv. Microbiol. Physiol.*, 1976, **13**, 245.

177 M. Abo-Ghalia, C. Michaud, D. Blanot and J. van Heijenoort, *Eur. J. Biochem.*, 1985, **153**, 81.

178 M. Abo-Ghalia, M. Flegel, D. Blanot and J. van Heijenoort, *Int. J. Peptide Protein Res.*, 1988, **32**, 208.

179 P. Le Roux, G. Auger, J. van Heijenoort and D. Blanot, *Eur. J. Med. Chem.*, 1992, **27**, 899.

180 F. C. Neuhaus and W. G. Struve, *Biochemistry*, 1965, **4**, 120.

181 B. Trippen, W. P. Hammes, K.-H. Schleifer and O. Kandler, *Arch. Microbiol.*, 1976, **109**, 247.

182 T. D. H. Bugg, G. D. Wright, S. Dutka-Malen, M. Arthur, P. Courvalin and C. T. Walsh, *Biochemistry*, 1991, **30**, 10408.

183 D. Billot-Klein, L. Gutmann, S. Sablé, E. Guittet and J. van Heijenoort, *J. Bacteriol.*, 1994, **176**, 2398.

184 P. E. Reynolds, H. A. Snaith, A. J. Maguire, S. Dutka-Malen and P. Courvalin, *Biochem. J.*, 1994, **301**, 5.

185 M. Arthur, P. Reynolds and P. Courvalin, *Trends Microbiol.*, 1996, **4**, 401.

186 D. J. Miller, S. M. Hammond, D. Anderluzzi and T. D. H. Bugg, *J. Chem. Soc., Perkin Trans. 1*, 1998, 131.

187 J. N. Umbreit and J. L. Strominger, *Proc. Natl. Acad. Sci. U.S.A.*, 1972, **69**, 1972.

188 D. D. Pless and F. C. Neuhaus, *J. Biol. Chem.*, 1973, **248**, 1568.

189 A. Geis and R. Plapp, *Biochim. Biophys. Acta*, 1978, **527**, 414.

190 M. Ikeda, M. Wachi, H. K. Jung, F. Ishino and M. Matsuhashi, *J. Bacteriol.*, 1991, **173**, 1021.

191 M. Ikeda, M. Wachi, F. Ishino and M. Matsuhashi, *Nucleic Acids Res.*, 1990, **18**, 1058.

192 D. S. Boyle and W. D. Donachie, *J. Bacteriol.*, 1998, **180**, 6429.

193 A. Bouhss, D. Mengin-Lecreux, D. Le Beller and J. van Heijenoort, *Mol. Microbiol.*, 1999, **34**, 576.

194 W. A. Weppner and F. C. Neuhaus, *Biochem. Biophys. Acta*, 1979, **552**, 418.

195 P. E. Brandish, M. K. Burnham, J. T. Lonsdale, R. Southgate, M. Inukai and T. D. H. Bugg, *J. Biol. Chem.*, 1996, **271**, 7609.

196 F. C. Neuhaus, *Acc. Chem. Res.*, 1971, **4**, 297.

197 U. Kohlrausch, F. B. Wientjes and J.-V. Höltje, *J. Gen. Microbiol.*, 1989, **135**, 1499.

198 D. Mengin-Lecreux, L. Texier, M. Rousseau and J. van Heijenoort, *J. Bacteriol.*, 1991, **173**, 4625.

199 W. P. Hammes and F. C. Neuhaus, *J. Biol. Chem.*, 1974, **249**, 3140.

200 W. P. Hammes and F. C. Neuhaus, *J. Bacteriol.*, 1974, **120**, 210.

201 J. B. Ward and H. R. Perkins, *Biochem. J.*, 1974, **139**, 781.

202 W. P. Hammes and O. Kandler, *Eur. J. Biochem.*, 1976, **70**, 97.

203 W. A. Weppner and F. C. Neuhaus, *J. Biol. Chem.*, 1977, **252**, 2296.

204 A. G. Pisabarro, R. Prats, D. Vasquez and A. Rodrigues-Tebar, *J. Bacteriol.*, 1986, **168**, 199.

205 Y. van Heijenoort, M. Gomez, M. Derrien, J. Ayala and J. van Heijenoort, *J. Bacteriol.*, 1992, **174**, 3549.

206 D. Billot-Klein, D. Shlaes, D. Bryant, D. Bell, R. Legrand, L. Gutmann and J. van Heijenoort, *J. Bacteriol.*, 1997, **179**, 4684.

207 K. Kimura, Y. Ikeda, S. Kagami, M. Yoshihama, K. Suzuki, H. Osada and K. Isono, *J. Antibiot.*, 1998, **51**, 1099.

208 V. J. Lee and S. J. Hecker, *Med. Res. Rev.*, 1999, **19**, 521.

209 P. E. Brandish, K.-I. Kimura, M. Inukai, R. Southgate, J. T. Lonsdale and T. D. H. Bugg, *Antimicrob. Agents Chemother.*, 1996, **40**, 1640.

210 C. A. Gentle and T. D. H. Bugg, *J. Chem. Soc., Perkin Trans. 1*, 1999, 1279.

211 C. A. Gentle, S. A. Harrison, M. Inukai and T. D. H. Bugg, *J. Chem. Soc., Perkin Trans. 1*, 1999, 1287.

212 C. Dini, P. Collette, N. Drochon, J. C. Guillot, G. Lemoine, P. Mauvais and J. Aszodi, *Bioorg. Med. Chem. Lett.*, 2000, **10**, 1839.

213 M. Ikeda, M. Wachi and M. Matsuhashi, *J. Gen. Appl. Microbiol.*, 1992, **38**, 53.

214 D. Mengin-Lecreux, L. Texier and J. van Heijenoort, *Nucleic Acids Res.*, 1990, **18**, 2810.

215 G. P. Salmond, J. F. Lutkenhaus and W. D. Donachie, *J. Bacteriol.*, 1980, **144**, 438.

216 K. Bupp and J. van Heijenoort, *J. Bacteriol.*, 1993, **175**, 1841.

217 A. Taku and D. P. Fan, *J. Biol. Chem.*, 1976, **251**, 1889.

218 A. Taku and D. P. Fan, *J. Biol. Chem.*, 1976, **251**, 6154.

219 M. Crouvoisier, D. Mengin-Lecreux and J. van Heijenoort, *FEBS Lett.*, 1999, **449**, 289.

220 S. Ha, E. Chang, M.-C. Lo, H. Men, P. Park, M. Ge and S. Walker, *J. Am. Chem. Soc.*, 1999, **121**, 8415.

221 S. Ha, D. Walker, Y. Shi and S. Walker, *Protein Sci.*, 2000, **9**, 1045.

222 E. A. Somner and P. E. Reynolds, *Antimicrob. Agents Chemother.*, 1990, **34**, 413.

223 A. A. Branstrom, S. Midha, C. B. Longley, K. Han, E. R. Baizman and H. R. Axelrod, *Anal. Biochem.*, 2000, **280**, 315.

224 G. Auger, M. Crouvoisier, M. Caroff, J. van Heijenoort and D. Blanot, *Lett. Peptide Sci.*, 1997, **4**, 371.

225 H. Men, P. Park, M. Ge and S. Walker, *J. Am. Chem. Soc.*, 1998, **120**, 2484.

226 B. Cavalleri, H. Pagani, G. Volpe, E. Selva and F. Parenti, *J. Antibiot.*, 1984, **37**, 309.

227 R. Ciabatti, J. K. Kettenring, G. Winters, G. Tuan, L. Zerilli and B. Cavalleri, *J. Antibiot.*, 1989, **42**, 254.

228 M. Kurz and W. Guba, *Biochemistry*, 1996, **35**, 12570.

229 M.-C. Lo, H. Men, A. Branstrom, J. Helm, N. Yao, R. Goldman and S. Walker, *J. Am. Chem. Soc.*, 2000, **122**, 3540.

230 D. J. Silva, C. L. Bowe, A. A. Branstrom, E. R. Baizman and M. J. Sofia, *Bioorg. Med. Chem. Lett.*, 2000, **10**, 2811.

231 R. J. Cox, *Nat. Prod. Rep.*, 1996, **13**, 29.

232 T. L. Born and J. S. Blanchard, *Curr. Opin. Chem. Biol.*, 1999, **3**, 607.

233 R. J. Cox, A. Sutherland and J. C. Vederas, *Bioorg. Med. Chem.*, 2000, **8**, 843.

234 H. König and R. Niemetz, *FEMS Microbiol. Rev.*, 1997, **20**, 36.

235 K. Nakajima, H. Tanizawa, H. Tanaka and K. Soda, *Agric. Biol. Chem.*, 1986, **50**, 2823.

236 S.-Y. Choi, N. Esaki, T. Yoshimura and K. Soda, *Protein Express. Purific.*, 1991, **2**, 90.

237 S.-Y. Choi, N. Esaki, M. Ashiuchi, T. Yoshimura and K. Soda, *Proc. Natl. Acad. Sci. U.S.A.*, 1994, **91**, 10144.

238 K. A. Gallo and J. R. Knowles, *Biochemistry*, 1993, **32**, 3981.

239 M. Yagasaki, K. Iwata, S. Ishino, M. Azuma and A. Ozaki, *Biosci. Biotechnol. Biochem.*, 1995, **59**, 610.

240 P. Doublet, J. van Heijenoort and D. Mengin-Lecreux, *J. Bacteriol.*, 1992, **174**, 5772.

241 P. Doublet, J. van Heijenoort, J.-P. Bohin and D. Mengin-Lecreux, *J. Bacteriol.*, 1993, **175**, 2970.

242 P. Doublet, J. van Heijenoort and D. Mengin-Lecreux, *Biochemistry*, 1994, **33**, 5285.

243 T. J. Dougerthy, J. A. Thanassi and M. J. Pucci, *J. Bacteriol.*, 1993, **175**, 111.

244 T. Yoshimura, M. Ashiuchi, N. Esaki, C. Kobatake, S.-Y. Choi and K. Soda, *J. Biol. Chem.*, 1993, **268**, 24242.

245 M. J. Pucci, J. Novotny, L. F. Discotto and T. J. Dougherty, *J. Bacteriol.*, 1994, **176**, 528.

246 H.-T. Ho, P. J. Falk, K. M. Ervin, B. S. Krishnan, L. F. Discotto, T. J. Dougherty and M. J. Pucci, *Biochemistry*, 1995, **34**, 2464.

247 S. S. Kim, I. G. Choi, S. H. Kim and Y. G. Yu, *Extremophiles*, 1999, **3**, 175.

248 K. Y. Hwang, C.-S. Cho, S. S. Kim, K. Baek, S.-H. Kim, Y. G. Yu and Y. Cho, *Acta Crystallogr., Sect. D*, 1999, **55**, 927.

249 K. Y. Hwang, C.-S. Cho, S. S. Kim, H.-C. Sung, Y. G. Yu and Y. Cho, *Nat. Struct. Biol.*, 1999, **6**, 422.

250 S.-Y. Choi, N. Esaki, T. Yoshimura and K. Soda, *J. Biochem.*, 1992, **112**, 139.

251 K. A. Gallo, M. E. Tanner and J. R. Knowles, *Biochemistry*, 1993, **32**, 3991.

252 M. E. Tanner, K. A. Gallo and J. R. Knowles, *Biochemistry*, 1993, **32**, 3998.

253 P. Doublet, J. van Heijenoort and D. Mengin-Lecreux, *Microb. Drug Resist.*, 1996, **2**, 43.

254 M. E. Tanner and S. Miao, *Tetrahedron Lett.*, 1994, **35**, 4073.

255 M. Ashiuchi, T. Yoshimura, N. Esaki, H. Ueno and K. Soda, *Biosci. Biotech. Biochem.*, 1993, **57**, 1978.

256 K. Tanizawa, Y. Masu, S. Asano, H. Tanaka and K. Soda, *J. Biol. Chem.*, 1989, **264**, 2445.

257 I. G. Fotheringham, S. A. Bledig and P. P. Taylor, *J. Bacteriol.*, 1998, **180**, 4319.

258 D. Peisach, D. M. Chipman, P. W. Van Ophem, J. M. Manning and D. Ringe, *Biochemistry*, 1998, **37**, 4958.

259 F. C. Neuhaus and W. P. Hammes, *Pharmacol. Ther.*, 1981, **14**, 265.

260 B. S. Axelsson, H. G. Floss, S. Lee, A. Saeed, P. A. Spencer and D. W. Young, *J. Chem. Soc., Perkin Trans. 1*, 1994, 2137.

261 M. J. Pucci, J. A. Thanassi, H.-T. Ho, P. J. Falk and T. J. Dougherty, *J. Bacteriol.*, 1995, **177**, 336.

262 A. Wipat, N. M. Carter, S. C. Brignell, B. J. Guy, K. Piper, J. Sanders, P. T. Emmerson and C. R. Harwood, *Microbiology*, 1996, **142**, 3067.

263 F. C. Neuhaus and J. L. Lynch, *Biochemistry*, 1964, **3**, 471.

264 F. C. Neuhaus, *J. Biol. Chem.*, 1962, **237**, 3128.

265 F. C. Neuhaus, *J. Biol. Chem.*, 1962, **237**, 778.

266 E. J. J. Lugtenberg and A. van Schijndel-van Dam, *J. Bacteriol.*, 1973, **113**, 96.

267 A. C. Robinson, D. J. Kenan, J. Sweeney and W. D. Donachie, *J. Bacteriol.*, 1986, **167**, 809.

268 L. E. Zawadzke, T. D. H. Bugg and C. T. Walsh, *Biochemistry*, 1991, **30**, 1673.

269 E. Daub, L. E. Zawadzke, D. Botstein and C. T. Walsh, *Biochemistry*, 1988, **27**, 3701.

270 O. A. M. Al-bar, D. O'Connor, I. G. Giles and M. Akhtar, *Biochem. J.*, 1992, **282**, 747.

271 A. E. Belanger, J. C. Porter and G. F. Hatfull, *J. Bacteriol.*, 2000, **182**, 6854.

272 G. D. Wright and C. T. Walsh, *Acc. Chem. Res.*, 1992, **25**, 468.

273 M. Arthur, P. Reynolds and P. Courvalin, *Trends Microbiol.*, 1996, **4**, 401.

274 D. Billot-Klein, L. Gutmann, S. Sable, E. Guittet and J. van Heijenoort, *J. Bacteriol.*, 1994, **176**, 2398.

275 S. Handwerger, M. J. Pucci, K. J. Volk, J. Liu and M. S. Lee, *J. Bacteriol.*, 1994, **176**, 260.

276 T. Ferain, J. N. Hobbs, J. Richardson, N. Bernard, D. Garmyn, P. Hols, N. E. Allen and J. Delcour, *J. Bacteriol.*, 1996, **178**, 5431.

277 P. E. Reynolds, H. A. Snaith, A. J. Maguire, S. Dutka-Malen and P. Courvalin, *Biochem. J.*, 1994, **301**, 5.

278 V. L. Healy, I. A. D. Lessard, D. I. Roper, J. R. Knox and C. T. Walsh, *Chem. Biol.*, 2000, **7**, R109.

279 T. D. H. Bugg, S. Dutka-Malen, M. Arthur, P. Courvalin and C. T. Walsh, *Biochemistry*, 1991, **30**, 2017.

280 D. Meziane-Cherif, M.-A. Badet-Denisot, S. Evers, P. Courvalin and B. Badet, *FEBS Lett.*, 1994, **354**, 140.

281 I.-S. Park, C.-H. Lin and C. T. Walsh, *Proc. Natl. Acad. Sci. U.S.A.*, 1997, **94**, 10040.

282 S. Evers, B. Casadewall, M. Charles, S. Dutka-Malen, M. Galimand and P. Courvalin, *J. Mol. Evol.*, 1996, **42**, 706.

283 L. S. Mullins, L. E. Zawadzke, C. T. Walsh and F. M. Raushel, *J. Biol. Chem.*, 1990, **265**, 8993.

284 V. L. Healy, L. S. Mullins, X. Li, S. E. Hall, F. M. Raushel and C. T. Walsh, *Chem. Biol.*, 2000, **7**, 505.

285 F. C. Neuhaus, C. V. Carpenter, J. L. Miller, N. M. Lee, M. Gragg and R. A. Stickgold, *Biochemistry*, 1969, **8**, 5119.

286 E. J. J. Lugtenberg, *J. Bacteriol.*, 1972, **110**, 26.

287 M.-R. de Roubin, D. Mengin-Lecreulx and J. van Heijenoort, *J. Gen. Microbiol.*, 1992, **138**, 1751.

288 C. Fan, P. C. Moews, C. T. Walsh and J. R. Knox, *Science*, 1994, **266**, 439.

289 C. Fan, I.-S. Park, C. T. Walsh and J. R. Knox, *Biochemistry*, 1997, **36**, 2531.

290 T. Huyton and D. I. Roper, *Acta Crystallogr.*, Sect. D, 1999, **55**, 1481.

291 D. I. Roper, T. Huyton, A. Vagin and G. Dodson, *Proc. Natl. Acad. Sci. U.S.A.*, 2000, **97**, 8921.

292 A. P. Kuzin, T. Sun, J. Jorczak-Baillass, V. L. Healy, C. T. Walsh and J. R. Knox, *Structure*, 2000, **8**, 463.

293 Y. Shi and C. T. Walsh, *Biochemistry*, 1995, **34**, 2768.

294 I.-S. Park, C. H. Lin and C. T. Walsh, *Biochemistry*, 1996, **35**, 10464.

295 Y. Vo-Quang, D. Carniato, L. Vo-Quang, A. M. Lacoste, E. Neuzil and F. Le Goffic, *J. Med. Chem.*, 1986, **29**, 148.

296 Y. Vo-Quang, D. Carniato, L. Vo-Quang, A. M. Lacoste, E. Neuzil and F. Le Goffic, *J. Med. Chem.*, 1986, **29**, 579.

297 K. Duncan and C. T. Walsh, *Biochemistry*, 1988, **27**, 3709.

298 K. Duncan, W. S. Faraci, D. S. Matteson and C. T. Walsh, *Biochemistry*, 1989, **28**, 3541.

299 Y. Vo-Quang, A. M. Gravey, R. Simonneau, L. Vo-Quang, A. M. Lacoste and F. Le Goffic, *Tetrahedron Lett.*, 1987, **28**, 6167.

300 W. H. Parsons, A. A. Pachett, H. G. Bull, W. R. Schoen, D. Taub, J. Davidson, P. L. Combs, J. P. Springer, H. Gadebusch, B. Weissberger, M. E. Valiant, T. N. Mellin and R. D. Busch, *J. Med. Chem.*, 1988, **31**, 1772.

301 M. Lacoste, A. M. Chollet-Gravey, L. Vo-Quang, Y. Vo-Quang and F. Le Goffic, *Eur J. Med. Chem.*, 1991, **26**, 255.

302 B. A. Ellsworth, N. T. Tom and P. A. Bartlett, *Chem. Biol.*, 1996, **3**, 37.

303 A. E. McDermott, F. Creuzet and R. G. Griffin, *Biochemistry*, 1990, **29**, 5767.

304 P. K. Chakravarty, W. J. Greenlee, W. H. Parsons, A. A. Patchett, P. Combs, A. Roth, R. D. Busch and T. N. Mellin, *J. Med. Chem.*, 1989, **32**, 1886.

305 T. J. Pollock, L. Thorne, M. Yamazaki, M. J. Mikolajczak and R. W. Armentrout, *J. Bacteriol.*, 1994, **176**, 6229.

306 G. Siewert and J. L. Strominger, *Proc. Natl. Acad. Sci. U.S.A.*, 1967, **57**, 767.

307 C. M. Allen, *Methods Enzymol.*, 1985, **110**, 281.

308 C. M. Apfel, B. Takacs, M. Fountoulakis, M. Stieger and W. Keck, *J. Bacteriol.*, 1999, **181**, 483.

309 J.-I. Kato, S. Fujisaki, K.-I. Nakajima, Y. Nishimura, M. Sato and A. Nakano, *J. Bacteriol.*, 1999, **181**, 2733.

310 N. Shimazu, T. Koyama and K. Ogura, *J. Biol. Chem.*, 1998, **273**, 19476.

311 M. Fujihashi, N. Shimizu, Y. W. Zhang, T. Koyama and K. Miki, *Acta Crystallogr.*, Sect. D, 1999, **55**, 1606.

312 J.-J. Pan, L.-W. Yang and P.-H. Liang, *Biochemistry*, 2000, **39**, 13856.

313 R. Goldman and J. L. Strominger, *J. Biol. Chem.*, 1972, **247**, 5116.

314 E. Willoughby, Y. Highasi and J. L. Strominger, *J. Biol. Chem.*, 1972, **247**, 5113.

315 R. B. Gennis and J. L. Strominger, *J. Biol. Chem.*, 1976, **251**, 1277.

316 H. Sandermann, *Eur. J. Biochem.*, 1976, **62**, 479.

317 B. D. Cain, P. J. Norton, W. Eubanks, H. S. Nick and C. M. Allen, *J. Bacteriol.*, 1993, **175**, 3784.

318 A. F. Chalker, K. A. Ingham, R. D. Lunsford, A. P. Bryant, J. Bryant, N. G. Wallis, J. P. Broskey, S. C. Pearson and D. J. Holmes, *Microbiol.*, 2000, **146**, 1547.

319 Z. Podlesek, A. Comino, B. Herzog-Velikonja and M. Grabnar, *FEMS Microbiol. Lett.*, 2000, **188**, 103.

320 W. Fiedler and H. Rotering, *J. Biol. Chem.*, 1988, **263**, 14684.

321 E. J. J. Lugtenberg, L. de Haas-Menger and W. H. M. Ruyters, *J. Bacteriol.*, **109**, 326.

322 E. Ito and J. L. Strominger, *J. Biol. Chem.*, 1962, **237**, 2696.

323 D. Mengin-Lecreulx and J. van Heijenoort, *J. Bacteriol.*, 1985, **163**, 208.

324 P. S. Venkateswaran, E. J. J. Lugtenberg and H. C. Wu, *Biochim. Biophys. Acta*, 1973, **293**, 570.

325 D. Mengin-Lecreulx, B. Flouret and J. van Heijenoort, *J. Bacteriol.*, 1982, **151**, 1109.

326 D. J. Tipper and A. Wright, in *The Bacteria*, ed. J. C. Gunsalus, Academic Press, New York, 1980, Vol. 7, p. 291.

327 D. Mengin-Lecreulx, E. Siegel and J. van Heijenoort, *J. Bacteriol.*, 1989, **171**, 3282.

328 E. E. Ishiguro and W. D. Ramey, *J. Bacteriol.*, 1978, **135**, 766.

329 B. Glauner, *Anal. Biochem.*, 1988, **172**, 451.

330 D. Billot-Klein, D. Shlaes, D. Bryant, D. Bell, J. van Heijenoort and L. Gutmann, *Biochem. J.*, 1996, **313**, 711.

331 A. Atrih, G. Bacher, G. Allmaier, M. P. Williamson and S. J. Foster, *J. Bacteriol.*, 1999, **181**, 3956.

332 O. Gateau, C. Bordet and G. Michel, *Biochim. Biophys. Acta*, 1976, **421**, 395.

333 R. Plapp and J. L. Strominger, *J. Biol. Chem.*, 1970, **245**, 3675.

334 W. Staudenbauer and J. L. Strominger, *J. Biol. Chem.*, 1972, **247**, 5095.

335 W. Katz, M. Matsuhashi, C. P. Dietrich and J. L. Strominger, *J. Biol. Chem.*, 1967, **242**, 3207.

336 M. Matsuhashi, C. P. Dietrich and J. L. Strominger, *Proc. Natl. Acad. Sci. U.S.A.*, 1965, **54**, 587.

337 M. Matsuhashi, C. P. Dietrich and J. L. Strominger, *J. Biol. Chem.*, 1967, **242**, 3191.

338 G. Siewert and J. L. Strominger, *J. Biol. Chem.*, 1968, **243**, 783.

339 A. M. Stranden, K. Ehlert, H. Labischinski and B. Berger-Bächi, *J. Bacteriol.*, 1997, **179**, 9.

340 S. Rohrer, K. Ehlert, M. Tschierske, H. Labischinski and B. Berger-Bächi, *Proc. Natl. Acad. Sci. U.S.A.*, 1999, **96**, 9351.

341 S. R. Filipe and A. Tomasz, *Proc. Natl. Acad. Sci. U.S.A.*, 2000, **97**, 4891.

342 S. S. Hedge and T. E. Shrader, *J. Biol. Chem.*, 2001, **276**, 6998.

343 A. Bouhss, N. Josseaume, D. Allanic, M. Crouvoisier, L. Gutmann, J.-L. Mainardi, D. Mengin-Lecreulx, J. van Heijenoort and M. Arthur, *J. Bacteriol.*, 2001, **183**, 5122.

344 J. C. Swenson and F. C. Neuhaus, *J. Bacteriol.*, 1976, **125**, 626.

345 W. Goodell and U. Schwarz, *J. Bacteriol.*, 1983, **156**, 136.

346 C. V. Carpenter, S. Goyer and F. C. Neuhaus, *Biochemistry*, 1976, **15**, 3146.

347 B. L. M. de Jonge, Y.-S. Chang, N. Xu and D. Gage, *Antimicrob. Agents Chemother.*, 1996, **40**, 1498.

348 R. K. Sinha and F. C. Neuhaus, *Antimicrob. Agents Chemother.*, 1991, **35**, 1753.

349 J. van Heijenoort and L. Gutmann, *Proc. Natl. Acad. Sci. U.S.A.*, 2000, **97**, 5028.

350 H. de Lencastre, S. W. Wu, M. G. Pinho, A. M. Ludovice, S. Filipe, S. Gardete, R. Sobral, S. Gill, M. Chung and A. Tomasz, *Microb. Drug Resist.*, 1999, **5**, 163.

351 J.-L. Mainardi, D. Billot-Klein, A. Coutrot, R. Legrand, B. Schoot and L. Gutmann, *Microbiology*, 1998, **144**, 2679.
