
Downloaded from http://rstb.royalsocietypublishing.org on March 29, 2015

**THE ROYAL SOCIETY**

---

# Picking out parallels: plant circadian clocks in context

Harriet G. McWatters${}^{1*}$, Laura C. Roden${}^{1}$ and Dorothee Staiger${}^{2}$

${}^{1}$Department of Biological Sciences, University of Warwick, Gibbet Hill Road, Coventry CV4 7AL, UK  
${}^{2}$Institute for Plant Sciences, Swiss Federal Institute of Technology, CH-8092 Zurich, Switzerland

Molecular models have been described for the circadian clocks of representatives of several different taxa. Much of the work on the plant circadian system has been carried out using the thale cress, *Arabidopsis thaliana*, as a model. We discuss the roles of genes implicated in the plant circadian system, with special emphasis on *Arabidopsis*. Plants have an endogenous clock that regulates many aspects of circadian and photoperiodic behaviour. Despite the discovery of components that resemble those involved in the clocks of animals or fungi, no coherent model of the plant clock has yet been proposed. In this review, we aim to provide an overview of studies of the *Arabidopsis* circadian system. We shall compare these with results from different taxa and discuss them in the context of what is known about clocks in other organisms.

**Keywords:** *Arabidopsis thaliana*; circadian rhythms; flowering time

## 1. INTRODUCTION

Circadian clocks, it has been frequently noted, are ubiquitous, reflecting a common challenge facing life on earth: adapting to the 24 h changing pattern of day and night. The Complex Clocks meeting in Edinburgh brought together researchers who between them described a fascinating series of studies on a variety of organisms, including plants.

Plants are unable to move to avoid adverse environmental conditions, such as drought and cold, or stresses, such as an excess of light. It is therefore very important for them to be able to anticipate changes in the environment. Possession of an endogenous circadian clock allows the prediction of daily and, as will be discussed, seasonal changes in light and temperature (see also Putterill 2001). The circadian system provides an internal measure or estimate of external time.

Circadian rhythms of biological processes, for example leaf movements in plants, persist in constant conditions, i.e. constant temperature and either constant light or dark. The period of the rhythm in constant conditions, in the absence of environmental stimuli that can act as time cues (*zeitgebers*) or entrainment signals, is called the free-running period (FRP). The FRP is not exactly 24 h. The cardinal features of the circadian system are an FRP of about 24 h, entrainment of the oscillator through photic and non-photic *zeitgebers*, and temperature compensation of the FRP.

Overt rhythms are studied to give an estimate of how the clockwork mechanism is behaving. There are many such rhythms in plants; for example, leaf and petal movements, photosynthetic capacity, fragrance emissions, cell division, stomatal movements and gene expression (Millar 1999; McClung 2000). A great aid to the measurement of circadian rhythms in plants has been the development of reporter gene fusions, most notably to the firefly luciferase (*luc*) gene. Millar et al. (1992) fused part of the promoter of the *Arabidopsis chlorophyll A/B binding protein 2* (*CAB2*; also called *Lhcb*I) gene to *luc* to make a real-time, non-invasive reporter for circadian-regulated gene expression. (Throughout, all gene names are in uppercase italic, mutant gene names in lowercase italic and protein names in uppercase.) This *CAB2::luc* marker provides us with another circadian phenotype that has aided in the isolation of mutants in the circadian clock, such as the *toc* (*timing of CAB*) mutants (Millar et al. 1995). Since then, other clock-related gene promoter-luciferase fusions have been made so that the expression of other genes can be inferred (Bognar et al. 1999; I. Carré and J.-Y. Kim, personal communication). Results from these experiments will be discussed later (§§ 2, 3, 8).

*Arabidopsis* is widely used as a model flowering plant in many fields of plant molecular genetic research. There are many reasons for this. *Arabidopsis* has a small genome of ca. 125 Mb, which is much smaller than most other plants, and has now been completely sequenced (*Arabidopsis* Genome Initiative 2000). It is a physically small plant, allowing many individuals to be grown in a small area. The generation time is short: seed-to-seed in 6–8 weeks, and thousands of progeny can be produced per generation. *Arabidopsis* is self-fertilizing, but it is possible to cross-pollinate plants with other genotypes and easy to generate transgenic plants using *Agrobacterium* as a vector to introduce genes of interest. The focus on *Arabidopsis* in this review reflects the ease of molecular genetic work in *Arabidopsis* compared with other plants. What is learned from *Arabidopsis* can be used as a base to study other plant systems.

The most rewarding approach so far in unravelling the molecular intricacies of the circadian clock, has been the genetic approach. Mutagenized populations are screened

*Author for correspondence (hmcwatters@bio.warwick.ac.uk)*

Phil. Trans. R. Soc. Lond. B (2001) 356, 1735–1743

Downloaded from http://rstb.royalsocietypublishing.org/ on March 29, 2015

1736 H. G. McWatters and others Plant circadian clocks

(a)

![image](https://i.imgur.com/1234567.png)

(b)

![image](https://i.imgur.com/890abcde.png)

Figure 1. (a) The three-component oscillator model—a linear pathway describing the minimum circadian system. (b) A refinement of the original model with a *zeitnehmer* loop feeding back from the clock to regulate input from the photoreceptors.

for aberrant clock phenotypes, usually in constant conditions. The genes are then cloned and their expression patterns and molecular interactions characterized. These types of screen generally look for altered periodicity, and may lead to only particular types of element of the circadian system being identified. Mutants defective in some aspect of the circadian clock were also identified as flowering-time mutants (*Zagotta et al.* 1996; *Schaffer et al.* 1998). Subtractive hybridization has led to the identification of two rhythmically expressed genes, one of which subsequently turned out to be part of an autoregulatory feedback loop (*Heintzen et al.* 1994; *Staiger* 2001). With recent advances in microarray technology (*Harmer et al.* 2000), it will be possible to discover many more rhythmically expressed genes, which may potentially lead to clock components. As the sequencing of the *Arabidopsis* genome is now complete (*Arabidopsis Genome Initiative* 2000) reverse genetic approaches matching gene sequence to predicted function will help to identify candidate clock genes.

### 2. CLOCK COMPONENTS AND FEEDBACK LOOPS

The ‘three-component model’ is a simple conceptual framework of a circadian clock (figure 1a). It comprises receptors, usually photoreceptors, that receive and transduce the environmental signal before passing information to the oscillator, which entrains and so couples environment and endogenous rhythmicity. Output from the oscillator is manifested in a variety of phenomena, such as locomotor activity of animals or leaf movement of plants, and it is usually these overt rhythms that are assayed in experiments.

This model has led to the search for genes the products of which function as components of the circadian clock. Various genes from a variety of organisms have been characterized that seem to have such a role, for example *FREQUENCY* (*FRQ*) from the filamentous fungus, *Neurospora crassa*, or *PERIOD* (*dPER*) from the fruitfly, *Drosophila melanogaster*. Such genes are expressed with a circadian rhythm under constant conditions, null mutations produce arrhythmia with other alleles producing long or short periods, and are self-regulated via negative feedback between the transcript and the protein.

Genes with similar characteristics are known from *Arabidopsis*. At present, the most likely candidates for plant central oscillator components, insofar as they possess many characteristics of clock genes in other organisms, are *TOC1* (*Strayer et al.* 2000; *Somers* 2001), *CIRCADIAN CLOCK ASSOCIATED 1* (*CCA1*) (*Wang & Tobin* 1998) and the closely related, *LATE ELONGATED HYPOCOTYL* (*LHY*) (*Schaffer et al.* 1998). The recently characterized *TOC1* has a circadian expression pattern with feedback between the transcript and protein (*Strayer et al.* 2000). The *toc1-1* mutation shortens the period of *CAB2::luc* expression in both red and blue light, suggesting that it acts downstream of input. *toc1-1* has identical effects on *CAB2* and *cold and circadian regulated-2* (*CCR2*), also known as *AtGRP7* (see the end of this section) (*Van Nocker & Vierstra* 1993; *Heintzen et al.* 1997), separate clock-controlled output genes with different temporal expression patterns. This suggests that the role of TOC1 is very close to, if not actually part of, the central clock mechanism (*Somers et al.* 1998a; *Somers* 2001).

*CCA1* is induced by light and, like TOC1, regulates its own production through a negative feedback loop between its RNA and protein in a similar manner to clock genes such as *FRQ* or *dPER*, although there is little delay between the appearance of the *CCA1* transcript and the protein. Constitutive expression of *CCA1* or *LHY* abolishes circadian period in the expression of *CAB2*, *CAT3*, *CCR2* and leaf movements in constant light and disrupts their expression in darkness (*Wang & Tobin* 1998; *Schaffer et al.* 1998). However, *CCA1* is not necessary for sustained oscillations, and expression of *CAB2*, *LHY*, *CAT2* or *CCR2* in the null mutant, *cca1-1*, is rhythmic with a short period (*Green & Tobin* 1999). These observations are probably due to a certain overlap of function between the highly homologous LHY and CCA1. It is also worth noting that CCA1 and LHY belong to a large family of closely related Myb-like transcription factors called the Reveille (RVE) family (*Andersson et al.* 1999), within which there is likely to be redundancy and overlap of function. The transcripts of many of the members of the RVE family cycle with a circadian rhythm.

Overexpression of either *CCA1* or *LHY* represses both genes (*Fowler et al.* 1999; *Wang & Tobin* 1998; *Schaffer et al.* 1998) indicating that their functions are closely tied.

CCAl also appears to be on an input pathway to the clock as it interacts with phytochrome (PHY) (Green & Tobin 1999). In addition to their input function, both CCAl and LHY interact with output markers for the clock such as CCR2 and CAB2. CCAl binds to the CAB2 promoter (Wang & Tobin 1998). CCAl interacted with a regulatory β-subunit (CKB3) of casein kinase 2 (CK2) in a yeast two-hybrid screen (Sugano et al. 1998). In addition, *in vitro* assays showed that CK2 was able to phosphorylate CCAl, and CK2-like activity in plant extracts affected the formation of a DNA–protein complex containing CCAl (Sugano et al. 1998). The activity of many transcription factors is regulated by their phosphorylation status (Whitmarsh & Davis 2000). Changes in protein phosphorylation levels can regulate cellular localization, protein stability and protein–protein interactions, and phosphorylation can be a prerequisite for DNA binding (Whitmarsh & Davis 2000). Intriguingly, phosphorylation of clock components is also a crucial part of both mammalian and *Drosophila* circadian feedback loops (Kloss et al. 1998; Lowrey et al. 2000). Furthermore, phosphorylation of FRQ is important in determining period length in *Neurospora* (Liu et al. 2000).

Overexpression of *CKB3* led to increased CK2 activity, shorter periods of the rhythms of *LHY* and *CCA1* expression, and loss of photoperiodic flowering response (Sugano et al. 1999). These results indicate that CK2 is involved in the regulation of circadian rhythms in *Arabidopsis*. All of these results taken together are consistent with CCAl and LHY feedback loops acting as part of the mechanism of the clock.

The *AtGRP7* (*Arabidopsis thaliana GLYCINE-RICH PROTEIN 7*; also known as CCR2 (Carpenter et al. 1994)) transcript encodes a predicted RNA-binding protein that is also part of a negative autoregulatory circuit. Overexpression of *AtGRP7* depresses the oscillation of the endogenous transcript, although overexpression of a mutated version containing a stop codon that does not give rise to a functional protein fails to do so (Heintzen et al. 1997; Staiger 2001). The *AtGRP7* promoter can confer circadian rhythmicity on a reporter gene, β-glucuronidase (Staiger & Apel 1999). Rhythmic activation by the clock persists in the presence of a constitutive high *AtGRP7* protein level, indicating that the promoter does not mediate the negative autoregulation. Rather, *AtGRP7* seems to influence oscillations of its transcript at the post-transcriptional level. In this respect, the *AtGRP7* feedback loop differs from known circadian oscillators in the fruitfly and mammals that are based on oscillating clock proteins rhythmically repressing transcription of their own genes. The role of *AtGRP7*, as well as other RNA-binding proteins, in circadian rhythms is discussed in more detail in Staiger & Heintzen (1999) and Staiger (2001).

### 3. PHOTORECEPTOR INPUT LOOPS

PHY, a red-light photoreceptor, regulates almost every aspect of plant development from seed germination to flowering to senescence. Five different PHYs (PHYA–PHYE) and several blue–ultraviolet (UV) absorbing pigments, among them the cryptochromes (CRY1 and CRY2), have been identified in *Arabidopsis*. There have been some excellent discussions of photoreceptors and

their roles in circadian rhythms (e.g. Deng & Quail 1999; Somers et al. 1998b; Lumsden 1991), to which the reader should refer for more detailed information.

In *Arabidopsis*, PHYs and CRYs provide light input to the circadian clock (Somers et al. 1998b). That the clock receives continuous input from light is shown by the fact that the periodicity of the *CAB2::luc* rhythm is lengthened in low-fluence light. PHYA and PHYB provide red-light input to the clock at low- and high-fluence rates, respectively; PHYA also transduces low-fluence blue light. CRY1 mediates the input of both high- and low-fluence blue light (Somers et al. 1998b). This plethora of photoreceptors may meet the need of plants to sense changes in light intensity and spectral composition during the day and across the year.

PHYB is translocated to the nucleus in a light-dependent manner (Sakamoto & Nagatani 1996) and interacts there with phytochrome interacting factor 3 (PIF3) (Nagy & Schafer 2000). Together, PHY and PIF3 can bind to promoters of other genes, including *CCA1* (Martinez-Garcia et al. 2000). Thus, a simple pathway transduces the light signal to *CCA1* and this pathway may yet prove central to the light regulation of the *Arabidopsis* clock.

A *PHYB* knockout does not lead to arrhythmicity (Millar et al. 1995; Somers et al. 1998b), nor does constitutive overexpression of *PHYB* (Somers et al. 1998b). Remarkably, a quadruple photoreceptor mutant lacking PHYA, PHYB, CRY1 and CRY2 retained circadian rhythmicity of leaf movements and was able to entrain to light signals (Yanovsky et al. 2000). However, given the number of plant photoreceptors represented by PHYs and CRYs alone (five *PHY* and two *CRY* genes), and the potential for blue-light responsiveness in the recently characterized ZEITLUPE (*ZTL*) (Somers et al. 2000; Somers 2001) and flavin-binding, kelch repeat, F-box protein (FKF1) (Nelson et al. 2000), some overlap of function and redundancy is almost inevitable.

The circadian clock regulates expression of the *PHYB* gene in tobacco and *Arabidopsis* (Bognar et al. 1999). These authors used luciferase–reporter constructs to demonstrate that *PHYA*, *PHYB*, *PHYC* and *CRY1* but not *CRY2*, had clear circadian rhythms of expression, and the regulation is maintained at the level of messenger RNA accumulation (Bognar et al. 2000). Their data demonstrated that the plant circadian clock controls the expression of the input receptors, adding a further regulatory loop on the light-input pathway. However, PHYB protein, the most abundant *Arabidopsis* PHY, does not show circadian variation in its absolute level, although new synthesis is rhythmic (Bognar et al. 1999).

Although the simplicity of the three-component model (figure 1a) has provided a paradigm for thought about the circadian clock, its linear pathway is too simple to reflect accurately the real world. In the initial oscillator model, the light-input pathway ran only from the photoreceptor to the oscillator. More recently, it has become clear that photoreceptors are much more intimately linked to clock function and may themselves show circadian rhythmicity. For instance, *Drosophila* CRYPTOCHROME (dCRY) mediates the light-induced degradation of TIMELESS (dTIM) (Ceriani et al. 1999; Naidoo et al. 1999); and the *WHITE-COLLAR* (*WC*) genes of *Neurospora* are involved

(a) absence of an oscillator

(b) masking of an oscillator output

Figure 2. Possible causes of arrhythmia: either the clock can stop, thus output becomes arrhythmic, or masking interferes with the output from a rhythmic oscillator.

in the light regulation of genes, including $FRQ$ (Crosthwaite et al. 1997).

Many of the clock genes identified in different organisms are associated with light-input pathways ($dTIM$, $WC-1$, $dCRY$). It has been suggested that the circadian clock may have arisen from light-input pathways (Crosthwaite et al. 1997). Separation of oscillator components and rhythmically acting input has been difficult (Roenneberg & Merrow 1998, 1999), but recent plant work on arrhythmicity (Millar et al. 1995; Somers et al. 1998b; Yanovsky et al. 2000) gives an indication that it is not an insurmountable problem. The discovery that, in plants, photoreceptors contribute clock properties may be providing a valuable hint to clock construction.

### 4. 'ZEITNEHMERS': CLOCKS MAY REGULATE THEIR OWN INPUT

The rhythmic transcription of the photoreceptors suggests that they are themselves regulated by the clock, implying a two-way trafficking of information between clock and photoreceptor (figure 1b). This would mean that the clock is capable of generating rhythmic input even under constant conditions through gating phototransduction of the light signal. This regulation of input by the clock has been called a *zeitnehmer* ('time-taker') loop (Roenneberg & Merrow 1998, 1999, 2001). A *zeitnehmer* is an input pathway regulated by feedback from an oscillator, creating rhythmic input under constant conditions.

Some mutations show conditional arrhythmic outputs, such as those of *early flowering 3* (*elf3*), that are arrhythmic in extended periods of light but rhythmic in darkness (Hicks et al. 1996). These have been found to retain a circadian oscillator that will entrain to light or temperature cycles under particular conditions (Hicks et al. 1996; McWatters et al. 2000). The *elf3* mutants of *Arabidopsis* demonstrate the stopped clock versus masked output conundrum. In this context, it is important to distinguish between different causes of arrhythmia. In true arrhythmia, the clock itself is stopped at some point in the circadian cycle or the oscillator has been driven to a fixed point (limit cycle) from which rhythmic output cannot be observed (Lumsden 1998). By contrast, arrhythmia of output pathways may also arise when outputs have become uncoupled from the clock; when the clock is bypassed or ‘masked’, and output receives a continuous input signal (figure 2). Under these conditions, no rhythmicity is seen even though the clock continues to run.

Studies of two *elf3* mutants, a null *elf3-1* and a partial mutant *elf3-7*, showed that the circadian clock in these plants operates beyond the onset of arrhythmicity indicating that the arrhythmia observed in constant light in *elf3* mutants was due, at least initially, to masking (McWatters et al. 2000). This masking is caused by an aberrant response to light. Wild-type *Arabidopsis* show circadian gating of the acute response to a light pulse; it is high during the subjective day and low in subjective night. Mutations of *ELF3* cause a defect in this gating that results in a continuously high acute response, thus in light there is a constant level of phototransduction (McWatters et al. 2000). This continuous input masks the remaining clock function in the *elf3* mutants. *ELF3* has been identified as a *zeitnehmer*, gating the light-input pathway to the circadian clock in the evening.

### 5. INTERACTIONS BETWEEN CIRCADIAN RHYTHMS AND PHOTOPERIODIC FLOWERING RESPONSES

There are two distinctive phases of development in *Arabidopsis*: vegetative and reproductive. The vegetative stage results in the formation of rosette leaves. The transition to the reproductive or flowering stage suppresses leaf production. The total leaf number and the time-to-flowering are quantified to assess flowering time. *Arabidopsis* is a facultative long-day plant; that is, it flowers earlier (i.e. with fewer leaves) in long days than in short days (Koornneef et al. 1998; Piñeiro & Coupland 1998).

Many mutations of *Arabidopsis* linked to the circadian clock, including *lhy* and *elf3*, were first identified because of their flowering-time phenotypes. Other mutants initially identified by their circadian phenotype also exhibit changes to the photoperiodic response; for instance, the *toc1-1* mutation leads to a diminished effect of daylength

on flowering in the C24 ecotype, and to photoperiodic  
flowering (flowers early regardless of daylength) in the  
Landsberg *erecta* background. This suggests that the same  
system underlies both circadian and photoperiodic time  
measurement, an idea first proposed by Bünning more than  
60 years ago (Bünning 1936).

Flowering time in *Arabidopsis* is under the control of at  
least two pathways: the long-day pathway, which is sensitive  
to photoperiod, and the autonomous pathway, which ensures  
that plants flower eventually even under short days (Koornneef *et al.* 1998; Blázquez & Weigel 2000; Putterill 2001). Mutations in the long-day pathway show delayed flowering under long days but not under short days. Mutations in the autonomous pathway cause plants to flower late under both long and short days.

*GIGANTEA* (*GI*) has a role in the long-day pathway;  
*gi* mutants are late flowering only on long days. *GI*  
mutants also have an altered circadian phenotype, although  
the details of the anomalies vary with the different *gi* alleles. Daily *GI* expression is rhythmic but the timing, peak level and peak duration are affected by daylength (Fowler *et al.* 1999). Putterill and colleagues (Fowler *et al.* 1999) examined the expression of *GI* in flowering-time mutants including *elf3*, *lhy* and *CCA1-OX*. They found that in an *elf3* background, expression of *GI* was increased but the amplitude of its rhythms was decreased. Overexpression of *CCA1* or *LHY* abolished rhythmicity of *GI*, whereas the *gi-3* mutation lowered the expression of *LHY* and *CCA1*, therefore *GI*, *CCA1* and *LHY* mutually feed back to affect each other's expression. The Nam and Kay laboratories (Park *et al.* 1999) have also investigated the role of *GI* in the circadian clock. They found that the *gi-1* mutation shortens periods of leaf movement, *CAB2::luc*, *CCA1* and *LHY*, whereas *gi-2* shortens the period of leaf movement but lengthens the periods of the other rhythms. *GI* has been placed on an input pathway because of the light-dependent effects of the *gi* mutations. Because *GI* expression is rhythmic, it must itself be under circadian control.

Night-break experiments on short-day plants such as the morning glory, *Pharbitis nil*, show that their photoperiodic response rhythm is based on a circadian rhythm that measures the duration of darkness and appears to be stopped in light: flowering occurs when the night exceeds a particular length (Lumsden 1998). The residual circadian behaviour of *elf3* mutants of *Arabidopsis* resembles that of short-day plants, although *elf3* has an early, flowering phenotype in 24 h T-cycles (explained below). In *elf3* plants, the pattern of *CAB2::luc* activity on being transferred to darkness after a variable time in light indicates that their clock stops at the end of the subjective day (McWatters *et al.* 2000).

Protocols that vary the total length (*T*) of the light–dark (LD) cycle, altering it from the natural external *T*=24h cycle, provide a means to synchronize external LD patterns with an organism’s internal circadian oscillator that has an endogenous circadian period, defined as *τ*. These are sometimes called resonance experiments because the length of the LD cycle is altered to ‘resonate’ optimally with the biological clock (Johnson *et al.* 1998).

It had been noted that the *elf3* mutant exhibited rhythms of *CAB2::luc* expression under LD cycles that were distinct from those of wild-type plants (Hicks *et al.* 1996). Roden and Carré are trying to drive the rhythms in a wild-type plant to a phase relationship with the LD cycle that is similar to that of *elf3* when *T*=24 h. Reciprocally, they are trying to restore responsiveness to photoperiod in photoperiod-insensitive mutants by driving the rhythms to the appropriate phase relative to the LD cycle. They have found that the photoperiodic responsiveness in *elf3-5*, a null mutant of *elf3*, can be restored in short *T*-cycles, and that under *T*=28 h, the timing of the *CAB2::luc* rhythm in wild-type plants was similar to that of *elf3-1* under *T*=24 h. This altered timing of *CAB2::luc* expression correlated with accelerated flowering (L. Roden and I. Carré, unpublished data).

Strayer *et al.* (2000) have shown that normal photoperiodic responses can be restored to the *timing of cab2 expression* (*toc1-I*) mutant, which displays a 21 h FRP of *CAB2::luc* expression, by growing the plants in 21 h *T*-cycles of long days (14 L:7 D) and short days (7 L:14 D), i.e. when *T=τ*.

# 6. PARALLELS BETWEEN CLOCKS IN DIFFERENT TAXA

The current molecular models of circadian function developed for mammals, *Drosophila*, *Neurospora* and cyanobacteria are all based on certain common features; most importantly, they are based on feedback loops, whereby clock proteins regulate their own production via positive and negative feedback elements (Dunlap 1999). In addition to transcriptional regulation of clock genes, post-translational modifications, such as protein phosphorylation, nuclear entry and targeted proteolysis, are involved in regulating the circadian period of activity of clock proteins and the clock (Edery *et al.* 1994; Kloss *et al.* 1998; Dunlap 1999; Liu *et al.* 2000; Nishiwaki *et al.* 2000). As discussed earlier (§2), *CCA1* and *LHY* can regulate their own (and each other’s) production through negative feedback loops. *CCA1* can be phosphorylated by CK2 (Sugano *et al.* 1998), and increased CK2 activity leads to shorter periods of *LHY* and *CCA1* expression, and loss of photoperiodic flowering response (Sugano *et al.* 1999). This suggests that similar mechanisms are employed to generate rhythmicity in disparate taxa. Does this mean that all circadian clocks share a single, common origin?

A first scan of the molecular evidence might seem to support this idea. Work in *Neurospora* and *Drosophila* has demonstrated common structural elements within the respective feedback loops: proteins containing the PAS domain. The PAS domain is so-named because of the proteins in which it was first found: *Drosophila* dPER, vertebrate aryl hydrocarbon receptor nuclear translocator, and *Drosophila* single-minded protein. PAS domains can be signalling modules involved in monitoring changes in light, redox potential, oxygen and small ligands within cells (Taylor & Zhulin 1999). Protein–protein interactions are mediated through the PAS domain (Taylor & Zhulin 1999). The LOV (light, oxygen, voltage) domain (Huala *et al.* 1997) is a type of PAS domain. It was first defined in the *Arabidopsis* NPH1 photoreceptor involved in blue-light phototropism, and has since been found in some blue-light photoreceptors, including *Neurospora* WHITE-COLLAR 1 (WC-1), and other sensory proteins. How does this bear on plant

Phil. Trans. R. Soc. Lond. B (2001)

clocks? Are there structural homologies between the clocks of plants and animals?

Some common elements can be identified; PHYs contain PAS domains and NPHI has a LOV domain. The mammalian CLOCK and CYCLE proteins contain PAS domains. Heterodimerization of mammalian CLOCK and CYCLE and dimerization of plant PHYs are mediated via their PAS domains (Taylor & Zhulin 1999). The recently characterized ZTL, a clock-associated gene from *Arabidopsis*, encodes a predicted LOV motif with 40% amino-acid identity to that of *Neurospora* WC-1 (Somers *et al.* 2000, 2001). Red and blue light shorten the period of *ztl-1* plants equally, implying that it acts after the convergence of red- and blue-input pathways. Does this add to the hypothesis that clock proteins are derived from photoreceptors?

Despite these tantalizing common features, true evidence for the underlying homology and conservation of circadian clocks between widely divergent taxa is harder to come by. The CRYs of plants may not be truly orthologous to those of animals, as the most parsimonious phylogeny indicates that the two have evolved separately from DNA-repair enzymes, photolyases, after the split between plant and animal lineages (Cashmore *et al.* 1999). Mammalian and fruitfly *CRY* genes are more closely related to 6,4-photolyases than they are to plant *CRY* genes. Therefore it would seem that two independent evolutionary routes have produced animal *CRY* and plant *CRY*.

More conclusively, homologues of cyanobacterial clock genes have not been found in *Arabidopsis*, or vice versa, despite our increasing knowledge of the genomes of these organisms. By contrast, homologues of many circadian genes have been found in insects and mammals, indicating that extant animal clocks are derived from a common ancestor. The similarities between the current clock models are more likely to be due to convergent evolution, in response to similar challenges, rather than to conserved homology from a common origin. After all, negative feedback loops, post-translational modification and protein–protein interactions are widespread regulatory mechanisms.

## 7. LOCAL AUTONOMY: ORGANIZATION OF CIRCADIAN SYSTEMS

It has long been known that in mammals two small group of cells, the paired suprachiasmatic nuclei (SCN) at the base of the brain, act as brain clocks that set time for the entire animal. Ablation of these nuclei causes arrhythmia, and SCN transfer from a rhythmic to an arrhythmic animal leads to the rhythms of the donor being expressed (Ralph *et al.* 1990; Ralph & Menaker 1988). It has recently become clear that individual animal cells contain all the parts of a functioning circadian clock, and that SCN cells in culture can produce rhythmic output without connections to other cells (Lui *et al.* 1997). Other cell types can also be rhythmic in tissue culture after synchronization by a serum pulse (Balsalobre *et al.* 1998). However, animals are organized into a differentiated system of organs, each organ having a specialized purpose. A similar hierarchy to that of mammals has been found in insects: the ventral lateral neurons of *Drosophila* seem to fulfil a related role in coordinating the timing of locomotor activity (Helfrich-Forster 1998; Helfrich-Forster *et al.* 1998). However, local autonomy of cellular clocks is found in animals, including some vertebrates: in *Drosophila*, clocks in the Malpighian tubules can be reset by light independently of the brain, despite the brain's role in timing locomotor activity (Giebultowicz *et al.* 2000); in the zebrafish, tissue clocks are also independent of the brain (Whitmore *et al.* 2000).

Like animals, plants are differentiated into organ systems and can coordinate organism-wide responses, for instance the response to attack by herbivores. However, unlike the hierarchical organization of mammalian clocks, the cellular clocks of plants act as stand-alone systems and are not coordinated (Thain *et al.* 2000). Thain *et al.* (2000) exposed the two cotyledons of a seedling to opposite LD schedules then allowed them to free-run and found that the free-running rhythms continued out of phase with each other. Thus, there is no indication of coupling between leaves. Experiments with roots and stems also failed to find any evidence of coupling between plant organs. Furthermore, when proximal and distal portions of a single leaf were subjected to similar treatment the same result was obtained. Thus, even within a single leaf there is no communication between the clocks of different cells. This strongly suggests that there is no central oscillator in plants with a role paralleling that of the SCN, and that there are no signals between clock systems, even in adjacent cells, that would lead to coordination.

Thus, plants stand at one end of a continuum that ranges from complete devolution of the circadian response to cellular clocks, to the complete control of circadian function by the nervous system. One can speculate on the reasons why no clock organ has developed in plants. First, each plant cell appears to contain all the photoreceptors that provide input to the system, and light effectively serves as a coordinating function negating the need for further coordination within the plant. This, however, appears to be true also for insects and some vertebrates in which it exists alongside brain-governed resetting so this is not in itself a sufficient explanation. Unlike the majority of animals, most plant cells remain totipotent throughout the life of the organism and so retain the ability to divide, differentiate and regenerate. In addition, plants are photosynthetic and have become adapted for efficient use of light. Plants cannot move to avoid adverse conditions; instead different parts may vary in their physical or biochemical make-up in response to variations in the local light environment. A stationary diffuse organism might not be under pressure to centralize circadian function but instead maintain internal synchrony by simultaneous entrainment of all its clocks by the environmental cycle. Thus, internal coordination is obtained in parallel in plants rather than in series as it is in mammals.

## 8. FUTURE DIRECTIONS

For those interested in complex clocks, the current pace of progress in the *Arabidopsis* model is very exciting. However, much work remains to be done before a clear picture of the *Arabidopsis* circadian system emerges. An area of circadian biology that has been hitherto neglected

Plant circadian clocks H. G. McWatters and others 1741

inputs                                                                 'Clock'                                                                 outputs

phytochromes A, B, C, D & E  
light  
cryptochrome 1  
cryptochrome 2  
ZTL  

ELF3  
PIF3  
TOC1  
CCA1  
LHY  
RVEs?  

GI  
flowering  
CCR2 (AtGRP7)  
CAB2  

Figure 3. A summary of the interactions between genes and their products thought to be involved in the *Arabidopsis* circadian system. Bold arrows indicate connections known from experimental data, dotted arrows represent postulated connections. Underlined text indicates rhythmically expressed (at the level of RNA or protein, or both) species; red text indicates light-induced species. References in text (§§ 2, 3, 4, 5 & 6).

in *Arabidopsis* studies is that of ‘real world’ experiments. In these, the properties of the pacemaker are studied under conditions of entrainment to LD or temperature cycles rather than under constant conditions. This useful approach, which is now attracting more attention, involves experiments that combine molecular genetic approaches with classical circadian protocols to understand how genetic lesions affect circadian physiology. Thus, function can be assigned to the different genes known to be involved in the circadian system. This approach has been found especially informative when phenotypes caused by several alleles of the same gene can be compared. Experiments of this type have proved rewarding in both *Arabidopsis* (e.g. McWatters *et al.* 2000) and other organisms, for instance *Neurospora* (Merrow *et al.* 1999; discussed in McWatters *et al.* 1999), for determining the role of the products of a particular gene, previously identified through circadian screening, in the organization of the circadian system.

In *Arabidopsis*, an important issue will be regulation of genes involved in photoperiodic flower induction in response to different photoperiods (Putterill 2001). The discovery of mutants disrupted in both circadian rhythmicity and photoperiodic flower induction in *Arabidopsis* opens a way to investigate the molecular basis of photoperiodic time measurement, i.e. how a clock with a period of ca. 24 h can determine responses to changes that occur over weeks or months. The behaviour (amplitude of the rhythm and phase of oscillation relative to the LD cycle) of clock-associated transcripts in different photoperiods should be correlated with effects of these photoperiods on flowering time. In *Arabidopsis*, studies of the regulation of genes that control flowering time, such as *GIGANTEA* (*GI*), has provided some interesting information regarding response to different daylengths (Fowler *et al.* 1999). At present, further studies are using this methodology to compare the phase of flowering-time genes with different

parts of the photoperiod and investigate how this has an effect on flowering time (Putterill 2001). Conducting experiments of this type will allow results obtained from *Arabidopsis* to be compared with other forms of photoperiodism such as that exhibited by short-day plants (Lumsden 1998), or to the insights obtained from classical studies of photoperiodism in other organisms.

The development of new markers, in particular *AtGRP7::luc* and *CHS::luc*, and improvements to automated methods of high-throughput screening to monitor luciferase activity or leaf movement, may provide a higher resolution to re-address some of the old questions that are the keys to understanding the circadian system of any organism: for instance, how many clocks exist per cell or how different clocks interact (Thain *et al.* 2000). The precise role of clock candidate genes needs further elucidation. It will be important to assign a biochemical function to the TOCl, ZTL and FKF proteins, all shown to be crucial for rhythmicity by genetic means (Somers 2001; Strayer *et al.* 2000; Nelson *et al.* 2000), in order to define their exact role in the oscillator mechanism. In addition, the role of photoreceptors such as PHY must be determined, in particular whether they provide input alone or are also clock components themselves.

### 9. CONCLUSIONS

Plants epitomize complex clocks. Clock research in plants seems to be poised between the knowledge of a variety of genes with circadian or photoperiodic effects and a model of their interactions to form the clock itself. Many pieces of the puzzle are already on the table—PHYs and CRYs provide input, *CCA1* and *LHY* are likely candidates for clock components, as is *TOC1*—but they have not yet been fitted together to provide a complete picture of the clock. A representation of the interactions

Phil. Trans. R. Soc. Lond. B (2001)

between the proteins discussed during the Complex Clocks meeting in Edinburgh is given in figure 3. Some pathways are becoming clear: a molecular mechanism for light regulation of genes such as CAB2 via PHY and CCA1 has been recently described. However, central clock components still need to be identified and the role of known genes made clear. The extent of overlap in the roles of similar genes and the nature of apparent redundancy must be further investigated, for instance that between LHY and CCA1. In addition, the interplay between photoreceptors and the clock itself needs to be further clarified.

This review is dedicated to Professor David Saunders at his retirement. We thank Ruth Bastow, Anthony Hall, Andrew Millar (Warwick University, UK), Jo Putterill (University of Auckland, New Zealand), Peter Lumsden, David Somers (Ohio State University, USA), Carl Strayer (Center for Biological Timing, Virginia, USA) and Simon Thain (University of Edinburgh, UK) for constructive advice and discussion.

---

**REFERENCES**

Andersson, C. R., Harmer, S. L., Schultz, T. F. & Kay, S. A. 1999 The Reveille (RVE) family of DNA binding proteins and the circadian clock. Abstract from *10th International Conference on Arabidopsis Research*, Australia.

Arabidopsis Genome Initiative 2000 Analysis of the genome sequence of the flowering plant *Arabidopsis thaliana*. *Nature* **408**, 796–815.

Balsalobre, A., Damiola, F. & Schibler, U. 1998 A serum shock induces circadian gene expression in mammalian tissue culture cells. *Cell* **93**, 929–937.

Blázquez, M. A. & Weigel, D. 2000 Integration of floral inductive signals in *Arabidopsis*. *Nature* **404**, 889–892.

Bognar, L. K., Hall, A., Adam, E., Thain, S. C., Nagy, F. & Millar, A. J. 1999 The circadian clock controls the expression pattern of the circadian input photoreceptor, phytochrome B. *Proc. Natl Acad. Sci. USA* **96**, 14652–14657.

Bognar, L. K., Toth, R., Kevei, E. & Nagy, F. 2000 Circadian clock-regulated expression of the input photoreceptors in higher plants. Abstract from *Complex Clocks Mtng, Edinburgh, UK, 21–23 March 2000*.

Bünning, E. 1936 Die endogene Tagesrhythmik als Grundlage der photoperiodischen Reaktion. *Ber. Dtsch. Bot. Ges.* **54**, 590–607.

Carpenter, C. D., Kreps, J. A. & Simon, A. E. 1994 Genes encoding glycine-rich *Arabidopsis thaliana* proteins with RNA-binding motifs are influenced by cold treatment and an endogenous circadian rhythm. *Plant Physiol.* **104**, 1015–1025.

Cashmore, A. R., Jarillo, J. A., Wu, Y. J. & Liu, D. M. 1999 Cryptochromes: blue light receptors for plants and animals. *Science* **284**, 760–765.

Ceriani, M. F., Darlington, T. K., Staknis, D., Mas, P., Petti, A. A., Weitz, C. J. & Kay, S. A. 1999 Light-dependent sequestration of TIMELESS by CRYPTOCHROME. *Science* **285**, 553–556.

Crosthwaite, S. K., Dunlap, J. C. & Loros, J. J. 1997 *Neurospora wc-1* and *wc-2*: transcription, photoresponses, and the origins of circadian rhythmicity. *Science* **276**, 763–769.

Deng, X. W. & Quail, P. H. 1999 Signalling in light-controlled development. *Semin. Cell Dev. Biol.* **10**, 121–129.

Dunlap, J. C. 1999 Molecular bases for circadian clocks. *Cell* **96**, 271–290.

Edery, I., Zwiebel, L. J., Dembinska, M. E. & Rosbash, M. 1994 Temporal phosphorylation of the *Drosophila* period protein. *Proc. Natl Acad. Sci. USA* **91**, 2260–2264.

Fowler, S., Lee, K., Onouchi, H., Samach, A., Richardson, K., Morris, B., Coupland, G. & Putterill, J. 1999 *GIGANTEA*: a circadian clock-controlled gene that regulates photoperiodic flowering in *Arabidopsis* and encodes a protein with several possible membrane spanning domains. *EMBO J.* **18**, 4679–4688.

Giebultowicz, J. M., Stanewsky, R., Hall, J. C. & Hege, D. M. 2000 Transplanted *Drosophila* excretory tubules maintain circadian clock cycling out of phase with the host. *Curr. Biol.* **10**, 107–110.

Green, R. M. & Tobin, E. M. 1999 Loss of the circadian clock-associated protein I in *Arabidopsis* results in altered clock-regulated gene expression. *Proc. Natl Acad. Sci. USA* **96**, 4176–4179.

Harmer, S. L., Hogenesch, J. B., Straume, M., Chang, H.-S., Han, B., Wang, X., Kreps, J. A. & Kay, S. A. 2000 Orchestrated transcription in *Arabidopsis* by the circadian clock. *Science* **290**, 2110–2113.

Heintzen, C., Melzer, S., Fischer, R., Kappeler, S., Apel, K. & Staiger, D. 1994 A light- and temperature-entrained circadian clock controls expression of transcripts encoding nuclear proteins with homology to RNA-binding proteins in meristematic tissue. *Plant J.* **5**, 799–813.

Heintzen, C., Nater, M., Apel, K. & Staiger, D. 1997 *AtGRP7*, a nuclear RNA-binding protein as a component of a circadian-regulated negative feedback loop in *Arabidopsis thaliana*. *Proc. Natl Acad. Sci. USA* **94**, 8515–8520.

Helfrich-Förster, C. 1998 Robust circadian rhythmicity of *Drosophila melanogaster* requires the presence of lateral neurons: a brain-behavioral study of disconnected mutants. *J. Comp. Physiol. A* **182**, 435–453.

Helfrich-Förster, C., Stengl, M. & Homberg, U. 1998 Organization of the circadian system in insects. *Chronobiol. Int.* **15**, 567–594.

Hicks, K. A., Millar, A. J., Carré, I. A., Somers, D. E., Straume, M., Meeks-Wagner, D. R. & Kay, S. A. 1996 Conditional circadian dysfunction of the *Arabidopsis early-flowering 3* mutant. *Science* **274**, 790–792.

Huala, E., Oeller, P. W., Liscum, E., Han, I. S., Larsen, E. & Briggs, W. R. 1997 *Arabidopsis* NPH1: a protein kinase with a putative redox-sensing domain. *Science* **278**, 2120–2123.

Johnson, C. H., Knight, M., Trewavas, A. & Kondo, T. 1998 A clockwork green: circadian programs in photosynthetic organisms. In *Biological rhythms and photoperiodism in plants* (ed. P. J. Lumsden & A. J. Millar), pp. 1–34. Oxford, UK: Bios.

Kloss, B., Price, J. L., Saez, L., Blau, J., Rothenfluh, A., Wesley, C. S. & Young, M. W. 1998 The *Drosophila* clock gene *double-time* encodes a protein closely related to human casein kinase I epsilon. *Cell* **94**, 97–107.

Koornneef, M., Alonso-Blanco, C., Peeters, A. J. M. & Soppe, W. 1998 Genetic control of flowering time in *Arabidopsis*. *A. Rev. Plant Physiol. Plant Mol. Biol.* **49**, 345–370.

Liu, C., Weaver, D. R., Strogatz, S. H. & Reppert, S. M. 1997 Cellular construction of a circadian clock: period determination in the suprachiasmatic nuclei. *Cell* **91**, 855–860.

Liu, Y., Loros, J. & Dunlap, J. C. 2000 Phosphorylation of the *Neurospora* clock protein FREQUENCY determines its degradation rate and strongly influences the period length of the circadian clock. *Proc. Natl Acad. Sci. USA* **97**, 234–239.

Lowrey, P. L., Shimomura, K., Antoch, M. P., Yamazaki, S., Zemenides, P. D., Ralph, M. R., Menaker, M. & Takahashi, J. S. 2000 Positional syntenic cloning and functional characterization of the mammalian circadian mutation tau. *Science* **288**, 483–491.

Lumsden, P. J. 1991 Circadian rhythms and phytochrome. *A. Rev. Plant Physiol. Plant Mol. Biol.* **42**, 351–371.

Lumsden, P. J. 1998 Photoperiodic induction in short-day plants. In *Biological rhythms and photoperiodism in plants* (ed. P. J. Lumsden & A. J. Millar), pp. 167–181. Oxford, UK: Bios.

McClung, R. 2000 Circadian rhythms in plants: a millennial view. *Physiol. Plant.* **109**, 359–371.

McWatters, H. G., Dunlap, J. C. & Millar, A. J. 1999 Circadian biology: clocks for the real world. *Curr. Biol.* **9**, R633–R635.

McWatters, H. G., Bastow, R. M., Hall, A. & Millar, A. J. 2000 The *ELF3* Zeitnehmer regulates light signalling to the circadian clock. *Nature* **408** 716–720.

Martinez-Garcia, J. F., Huq, E. & Quail, P. H. 2000 Direct targeting of light signals to a promoter element-bound transcription factor. *Science* **288**, 859–863.

Merrow, M., Brunner, M. & Roenneberg, T. 1999 Assignment of circadian function for the *Neurospora* clock gene *frequency*. *Nature* **399**, 584–586.

Millar, A. J. 1999 Biological clocks in *Arabidopsis thaliana*. *New Phytol.* **141**, 175–197.

Millar, A. J., Short, S. R., Chua, N.-H. & Kay, S. A. 1992 A novel circadian phenotype based on firefly luciferase expression in transgenic plants. *Plant Cell* **4**, 1075–1087.

Millar, A. J., Straume, M., Chory, J., Chua, N.-H. & Kay, S. A. 1995 The regulation of circadian period by phototransduction pathways in *Arabidopsis*. *Science* **267**, 1163–1166.

Nagy, F. & Schafer, E. 2000 Nuclear and cytosolic events of light-induced, phytochrome-regulated signaling in higher plants. *EMBO J.* **19**, 157–163.

Naidoo, N., Song, W., Hunter Ensor, M. & Sehgal, A. 1999 A role for the proteasome in the light response of the timeless clock protein. *Science* **285**, 1737–1741.

Nelson, D. C., Lasswell, J., Rogg, L. E., Cohen, M. A. & Bartel, B. 2000 *FKF1*, a clock-controlled gene that regulates the transition to flowering in *Arabidopsis*. *Cell* **101**, 331–340.

Nishiwaki, T., Iwasaki, H., Ishiura, M. & Kondo, T. 2000 Nucleotide binding and autophosphorylation of the clock protein KaiC as a circadian timing process of cyanobacteria. *Proc. Natl Acad. Sci. USA* **97**, 495–499.

Park, D. H., Somers, D. E., Kim, Y. S., Choy, Y. H., Lim, H. K., Soh, M. S., Kim, H. J., Kay, S. A. & Nam, H. G. 1999 Control of circadian rhythms and photoperiodic flowering by the *Arabidopsis GIGANTEA* gene. *Science* **285**, 1579–1582.

Piñeiro, M. & Coupland, G. 1998 The control of flowering time and floral identity in *Arabidopsis*. *Plant Physiol.* **11**, 1–8.

Putterill, J. 2001 Flowering in time: genes controlling photoperiodic flowering in *Arabidopsis*. *Phil. Trans R. Soc. Lond. B* **356**, 1761–1767. (DOI 10.1098/rstb.2001.0963.)

Ralph, M. R. & Menaker, M. 1988 A mutation of the circadian system in golden hamsters. *Science* **241**, 1225–1227.

Ralph, M. R., Foster, R. G., Davis, F. C. & Menaker, M. 1990 Transplanted suprachiasmatic nucleus determines circadian period. *Science* **247**, 975–978.

Roenneberg, T. & Merrow, M. 1998 Molecular circadian oscillators: an alternative hypothesis. *J. Biol. Rhythms* **13**, 167–179.

Roenneberg, T. & Merrow, M. 1999 Circadian systems and metabolism. *J. Biol. Rhythms* **14**, 449–459.

Roenneberg, T. & Merrow, M. 2001 Circadian systems: different levels of complexity. *Phil. Trans. R. Soc. Lond. B* **356**, 1687–1696. (DOI 10.1098/rstb.2001.0969.)

Sakamoto, K. & Nagatani, A. 1996 Nuclear localization activity of phytochrome B. *Plant J.* **10**, 859–868.

Schaffer, R., Ramsay, N., Samach, A., Corden, S., Putterill, J., Carré, I. A. & Coupland, G. 1998 The *late elongated hypocotyl* mutation of *Arabidopsis* disrupts circadian rhythms and the photoperiodic control of flowering. *Cell* **93**, 1219–1229.

Somers, D. E. 2001 Clock-associated genes in *Arabidopsis*: a family affair. *Phil. Trans. R. Soc. Lond. B* **356**, 1745–1753. (DOI 10.1098/rstb.2001.0965.)

Somers, D. E., Webb, A. A. R., Pearson, M. & Kay, S. A. 1998a The short-period mutant, *toc1-1*, alters circadian clock regulation of multiple outputs throughout development in *Arabidopsis thaliana*. *Development* **125**, 485–494.

Somers, D. E., Devlin, P. F. & Kay, S. A. 1998b Phytochromes and cryptochromes in the entrainment of the *Arabidopsis* circadian clock. *Science* **282**, 1488–1490.

Somers, D. E., Schultz, T. F., Milnamow, M. & Kay, S. A. 2000 ZEITLUPE encodes a novel clock-associated PAS protein from *Arabidopsis*. *Cell* **101**, 319–329.

Staiger, D. 2001 RNA-binding proteins and circadian rhythms in *Arabidopsis thaliana*. *Phil. Trans. R. Soc. Lond. B* **356**, 1755–1759. (DOI 10.1098/rstb.2001.0964.)

Staiger, D. & Apel, K. 1999 Circadian clock-regulated expression of an RNA-binding protein in *Arabidopsis*: characterization of a minimal promoter element that activates *Atgrp7* transcription at the end of the daily light phase. *Mol. Gen. Genet.* **261**, 811–819.

Staiger, D. & Heintzen, C. 1999 The circadian system of *Arabidopsis thaliana*—forward and reverse genetic approaches. *Chronobiol. Int.* **16**, 1–16.

Strayer, C., Oyama, T., Schultz, T. F., Raman, R., Somers, D. E., Mas, P., Panda, S., Kreps, J. A. & Kay, S. A. 2000 Cloning of the *Arabidopsis* clock gene *TOC1*, an autoregulatory response regulator homolog. *Science* **289**, 768–771.

Sugano, S., Andronis, C., Green, R. M., Wang, Z. Y. & Tobin, E. M. 1998 Protein kinase CK2 interacts with and phosphorylates the *Arabidopsis* circadian clock-associated 1 protein. *Proc. Natl Acad. Sci. USA* **95**, 11020–11025.

Sugano, S., Andronis, C., Ong, M. S., Green, R. M. & Tobin, E. M. 1999 The protein kinase CK2 is involved in the regulation of circadian rhythms in *Arabidopsis*. *Proc. Natl Acad. Sci. USA* **96**, 12362–12366.

Taylor, B. L. & Zhulin, I. B. 1999 PAS domains: internal sensors of oxygen, redox potential, and light. *Microbiol. Mol. Biol. Rev.* **63**, 479–506.

Thain, S., Hall, A. & Millar, A. 2000 Functional independence of multiple circadian clocks that regulate plant gene expression. *Curr. Biol.* **10**, 951–956.

Van Nocker, S. & Vierstra, R. 1993 Two cDNAs from *Arabidopsis thaliana* encode putative RNA binding proteins containing glycine-rich domains. *Plant Mol. Biol.* **21**, 695–699.

Wang, Z.-Y. & Tobin, E. M. 1998 Constitutive expression of the *Circadian Clock Associated 1 (CCA1)* gene disrupts circadian rhythms and suppresses its own expression. *Cell* **93**, 1207–1217.

Whitmarsh, A. J. & Davis, R. J. 2000 Regulation of transcription factor function by phosphorylation. *Cell. Mol. Life Sci.* **57**, 1172–1183.

Whitmore, D., Foulkes, N. S. & Sassone-Corsi, P. 2000 Light acts directly on organs and cells in culture to set the vertebrate circadian clock. *Nature* **404**, 87–91.

Yanovsky, M. J., Mazzella, M. A. & Casal, J. J. 2000 A quadruple photoreceptor mutant still keeps track of time. *Curr. Biol.* **10**, 1013–1015.

Zagotta, M. T., Hicks, K. A., Jacobs, C. I., Young, J. C., Hangartner, R. P. & Meeks-Wagner, D. R. 1996 The *Arabidopsis ELF3* gene regulates vegetative photomorphogenesis and the photoperiodic induction of flowering. *Plant J.* **10**, 691–702.
