
Progress in
Hematology

The Role of the AML1 Transcription Factor
in Leukemogenesis

Robert B. Lorsbach, James R. Downing

Department of Pathology, St. Jude Children's Research Hospital, Memphis, Tennessee

Received June 19, 2001; accepted June 28, 2001

---

### Abstract

Chromosomal translocations are one of the hallmarks of human leukemias. These structural abnormalities result in the generation of genetic mutations that play a direct role in the transformation of hematopoietic stem cells. Some of the most common targets of these chromosomal rearrangements are the genes that encode the AML1/CBFβ transcription factor complex. The AML1/CBFβ complex plays a critical role in normal hematopoiesis, controlling the initiation of a transcriptional cascade required for the formation of definitive hematopoietic stem cells. Understanding how alterations in the normal biologic activity of this transcription factor complex lead to the initiation of leukemia will provide critical insights in the molecular pathogenesis of this disease. These insights in turn are likely to lead to the development of more rational approaches to the treatment of acute leukemia. In this review, we will summarize our current understanding of the mechanisms by which alterations in the activity of AML1/CBFβ contribute to the development of leukemia. *Int J Hematol.* 2001;74:258-265.

© 2001 The Japanese Society of Hematology

#### Keywords: AML1; Runx1; Acute leukemia; ETO; AML1-ETO

---

### 1. Introduction

The core binding factors comprise a small family of transcription factors that play important roles in lineage-specific gene transcription in several cell types. This family consists of 3 DNA-binding α subunits, each of which is encoded by a distinct gene: *Runx1* (here we use the designation *AML1*), *Runx2*, and *Runx3*. Each of the α subunits can form a heterodimer with the common β subunit (*CBFβ*). *Runx2* and *Runx3* have been the subject of recent reviews and will not be further discussed [1,2]. Each *Runx* family member contains a runt domain (RD), so named because of its homology to the *Drosophila* protein Runt [3,4]. The RD mediates both heterodimerization with *CBFβ* and DNA binding of *Runx* proteins to the core enhancer sequence (CES), TGTGGT. The central role of AML1/CBFβ in developmental hematopoiesis is demonstrated by the striking phenotype of mice deficient in either *AML1* or *CBFβ* [5,6]. Homozygous *AML1* or *CBFβ* knock-out embryos die during mid-gestation and are characterized by anemia and extensive hemorrhage within the central nervous system and soft tissue. These embryos show a complete absence of fetal liver hematopoietic cells due to defective formation of definitive hematopoietic stem cells (HSCs) [5-8]. In addition, *AML1/CBFβ* has an important function in both lineage-committed progenitors and in mature hematopoietic effector cells.

*AML1* is one of the most commonly targeted genes involved in chromosomal translocations in human leukemia and was first identified and cloned by virtue of its involvement in the t(8;21)(q22;q22) seen in approximately 15% of acute myeloid leukemias (AMLs). This translocation results in the formation of a chimeric gene between *AML1* on chromosome 21 and *ETO* (for *Eight-Twenty-One*; also known as *MTG8*) on chromosome 8. The resultant fusion protein consists of the N-terminal portion of *AML1*, including the RD, fused in-frame to almost the complete *ETO* protein. *AML1* has been identified as a partner gene in several other leukemia-associated chromosomal translocations as well, including the t(16;21) [*AML1-MTG16*], t(3;21) [*AML1-EVI1*], and t(12;21) [*TEL-AML1*][9]. In addition, the AML-associated inv(16) results in a chimeric gene and encoded fusion protein between *CBFβ* and the smooth muscle myosin chain gene *MYH11*. In this review, we will summarize our current understanding of the role of *AML1* fusion proteins in leukemogenesis. For the sake of brevity, we will largely confine our comments to the *AML1-ETO* fusion protein, one of the most common and perhaps the best characterized of the *AML1* fusion proteins.

---

Correspondence and reprint requests: James R. Downing, MD,  
Department of Pathology, St. Jude Children's Research Hospital,  
332 North Lauderdale, Memphis, TN 38105; (901) 495-3510; fax:  
(901) 495-3749 (e-mail: jim.downing@stjude.org).

Role of AML1 Transcription Factor in Leukemia

2. The AML1/CBFβ Transcription Complex: Normal Function

AML1/CBFβ regulates the expression of a large number of hematopoietic-specific genes, including those for interleukin-3 (IL-3), the CSF-1 receptor, myeloperoxidase, neutrophil elastase, granzyme B, and the T-cell receptors (TCRs) α, β, δ, and γ [9]. AML1 transactivation is initiated by its binding to the CES present within the enhancers or promoters of AML1-responsive genes. AML1 binds to the CES through its central RD, which contains an immunoglobulin fold similar to motifs present in several other structurally related DNA-binding proteins including p53, NF-κB, NFAT, STAT1, and STAT3β [10,11]. The formation of heterodimers with CBFβ greatly enhances the DNA-binding affinity of AML1, by inducing conformational changes within the DNA-contact loops of the RD [12,13]. Heterodimerization with CBFβ also stabilizes AML1 and renders it more resistant to ubiquitin-proteosome-mediated degradation [14].

In general, binding of AML1/CBFβ alone is necessary but insufficient to effect transcriptional activation. Within promoters that contain the CES, adjacent DNA-binding sites for other transcription factors are often present, including C/EBPα, Myb, Ets family members, and AP-1. Moreover, AML1 can stabilize the DNA binding of several of these factors. AML1/CBFβ may, therefore, function as an organizing protein that promotes the assembly of transcriptional activation complexes (Figure 1). Indeed, AML1 associates with several coactivator proteins, including p300 and the CREB-binding protein (CBP) [15]. p300 possesses both intrinsic histone acetyl transferase (HAT) activity as well as domains through which it can interact with other HATs such as P/CAF and SRC-1. Thus, transactivation by AML1 is achieved through the recruitment of coactivators that, in turn, modulate chromatin structure and render a given promoter more accessible to the basal transcriptional machinery. AML1/CBFβ also interacts with other coactivator proteins, including Yes-associated protein (YAP), which

![Diagram](attachment:diagram.png)

**Figure 1.** A, AML1 is a modular protein that contains a central Runt DNA-binding domain (RD), a proline/serine/threonine-rich region (PST), a nuclear matrix attachment motif (NM), and a C-terminal transcriptional transactivation domain (TA). An additional transactivation domain has been defined that encompasses the PST and NM motifs. The VWRPY domain located in the C-terminus binds the Groucho corepressor. The arrow denotes the position of the chromosomal breakpoint in the t(8;21). B, Several factors dictate whether AML1 functions as either a transcriptional activator or repressor. These include the relative levels of cell lineage-specific transcription factors (eg, Ets, myb, and C/EBPα) and corepressors (eg, TLE1 and Ear2).

associates with AML1 via a proline-rich domain in AML1, and ALY, a protein that enhances TCRα promoter activity in cooperation with LEF-1, Ets-1, and AML1 [16, 17]. Finally, AML1 can function as a transcriptional repressor in some circumstances through the recruitment of corepressors, including the mammalian homologs of the *Drosophila* protein Groucho, termed transducin-like enhancer of split-1 (TLE1), the nuclear hormone receptor Ear-2, and Sin3 corepressors [18-21]. Whether AML1 functions as an activator or repressor of transcription will depend on the relative levels of coactivators and corepressors, the specific isoform of AML1 expressed, and the presence or absence of specific posttranslational modifications, eg, phosphorylation (Figure 1).

### 3. The AML1-ETO Fusion Oncoprotein

The t(8;21) chromosomal translocation results in the formation of an AML1-ETO fusion protein. This protein consists of the N-terminal half of AML1, including the RD, fused in-frame to all but the first 30 N-terminal amino acids of ETO. ETO is the mammalian homolog of the *Drosophila* protein Nervy and is normally expressed in the brain and immature hematopoietic progenitors [22, 23]. Although ETO is a nuclear phosphoprotein, it does not appear to directly bind DNA, but instead alters the activity of other transcription factors through protein-protein interactions [24]. ETO is a member of an extended gene family that includes MTG16 and MTGR1 [25, 26]. Remarkably, MTG16 was initially identified as part of an AML-MTG16 fusion protein formed as a result of the leukemia-associated t(16;21) [25]. Sequence comparison between ETO family members highlights 4 domains of high sequence conservation: a region with homology to the transcriptional coactivator TAF110; a hydrophobic heptad repeat (HHR); a small region termed Nervy homology region 3 (NHR3); and 2 C-terminal zinc-finger motifs termed the MYND domain (Figure 2). Recent studies suggest that ETO family members simultaneously interact with transcription factors and directly bind transcriptional corepressors [27-29]. Homo- and heterodimerization of ETO family members occur through the HHR domain, whereas binding of corepressors including NCoR, Sin3a/b, and histone deacetylase (HDAC) appears to involve multiple domains, with the corepressors binding independently of one another [28, 30, 31]. The binding of NCoR to ETO is mediated primarily through the MYND domain, whereas Sin3 appears to contact multiple sites within AML1 [28]. Thus, the ETO proteins appear to function as adapters or scaffolding proteins that recruit transcriptional corepressors to DNA-bound transcription factors, thereby inhibiting their ability to activate transcription. Consistent with this interpretation, a number of transcription factors have been shown to interact directly with ETO including PLZF, a target of the t(11;17) translocation associated with acute promyelocytic leukemia [32, 33].

In the AML1-ETO protein, the transcriptional activation domains of AML1 are replaced with almost the entire ETO protein, including its dimerization and corepressor interaction domains (Figure 3). Thus, based on its structure, AML1-

ETO would be predicted to retain its ability to bind to the CES and heterodimerize with CBFβ, but instead of activating transcription should, in a dominant fashion, repress transcription. Biochemical studies have demonstrated that AML1-ETO functions as a constitutive repressor dominantly inhibiting AML1. Moreover, deletion of the MYND domain ablates the recruitment of HDAC by AML1-ETO and markedly diminishes its ability to repress AML1-induced transcription [28]. Thus, the recruitment of HDAC is critical to the repression of gene transcription effected by AML1-ETO. Further supporting this interpretation, preliminary in vitro studies indicate that HDAC inhibitors can partially reverse the inhibition of differentiation induced by AML1-ETO [34, 35].

Although the majority of data supports a role of AML1-ETO as a transcriptional repressor, several other studies suggest that under some conditions it can activate transcription. Specifically, the promoters of the CSF-1 receptor, G-CSF receptor, and bcl-2 genes are activated by AML1-ETO [36-38]. Additionally, analysis of gene expression in a murine myeloid cell line overexpressing AML1-ETO has identified several genes whose level of expression was increased relative to that observed in the parental cell line [39]. Whether this increase is a direct effect of AML-ETO on the promoter/enhancer of these genes or results from indirect effects remains to be determined.

The role of AML1-ETO in hematopoietic cell growth and differentiation has been extensively investigated in vitro. AML1-ETO blocks the terminal differentiation in several cell lines of different hematopoietic lineages, an effect that may play a role in human leukemogenesis [9]. However, because these cell lines are likely to harbor mutations in other signaling pathways as a result of their in vitro propagation, these findings must be confirmed and extended with more rigorous analyses performed on primary leukemic samples.

To overcome the limitations of studies performed using cell lines, mice containing an *AML1-ETO* knocked-in allele have been developed [40, 41]. The advantage of this approach is that expression of AML1-ETO is regulated by the endogenous *AML1* promoter ensuring that its expression occurs both at a biologically relevant level and within cells committed to hematopoietic development. These mice have an embryonic lethal phenotype that is nearly identical to that of AML1-deficient mice. However, in contrast to AML1-deficient embryos, cultures of fetal liver cells from AML1-ETO mice yield dysplastic mixed-lineage colonies [41]. Moreover, when these hematopoietic colonies are serially cultured in vitro, they continue to form mixed colonies long after cultures of wild-type cells have ceased to grow. Comparable results are also observed in serial cultures of bone marrow cells isolated from transgenic mice in which a tetracycline-responsive promoter regulates the expression of AML1-ETO [42]. These findings suggest that AML1-ETO confers an enhanced capacity for self-renewal within the HSC compartment. Importantly, however, these AML1-ETO expressing cells do not show a growth advantage in vivo and fail to result in leukemia. Thus, the expression of AML1-ETO alone does not confer on cells full leukemogenic potential.

Role of AML1 Transcription Factor in Leukemia

A.
ETO
TAF HHR NHR3 MYND

B.

ETO Sin3 HDAC
NCOR
ETO Sin3 HDAC
NCOR
Trans. Factor X
Target Genes
XXXXX

CBFβ
AML1-ETO Sin3 HDAC
NCOR
TGTGGT
Target Genes

CBFβ
AML1-ETO Sin3 HDAC
NCOR
C/EBPα or Ets
Target Genes

Figure 2. A, ETO shares 4 regions of homology with other ETO family members, including an N-terminal region with homology to the transcriptional coactivator TAF110 (TAF), a hydrophobic heptad repeat (HHR), a small motif termed Nervy homology region 3 (NHR3), and 2 C-terminal zinc-finger motifs termed the MYND domain. The arrow denotes the position of the chromosomal breakpoint in the t(8;21). B, ETO, whether native protein or as a component of AML1-ETO, can homo- or heterodimerize with other ETO members (MTGR1 and MTG16), a function critical to the transcriptional repression effected by ETO or AML1-ETO. ETO also binds directly to the nuclear corepressors NCoR and Sin3a/b, which ultimately results in the recruitment of histone deacetylase (HDAC). ETO family proteins can interact with transcription factors (Trans. Factor X), including PLZF. By virtue of the Runt domain, AML1-ETO directly binds the core enhancer sequence or interacts with C/EBPα or Ets to effect transcriptional repression. In all cases, ETO functions to link the recruitment of transcriptional corepressors to DNA-bound transcription factors.

To bypass the embryonic lethality of the conventionally knocked-in AML1 allele, we have recently developed a mouse strain with a conditional AML1-ETO knock-in allele in which a transcriptional stop cassette bracketed by loxP sites was inserted in the intron upstream of the site of fusion [43]. This allele was shown to be silent in its intact state. To activate this allele, these mice were crossed with Mx1-Cre transgenic mice in which the expression of the Cre transgene is under the control of the interferon-inducible Mx1 promoter. Following Cre expression, AML1-ETO was detected in the bone marrow of double-transgenic mice for up to 1.5 years. Despite the expression of AML1-ETO, however, these mice had essentially normal peripheral blood counts and failed to develop leukemia. Thus,

expression of AML1-ETO in multilineage progenitors in the adult neither blocked differentiation nor resulted in leukemia. By contrast, treatment of the AML-ETO expressing double-transgenic mice with the potent mutagen N-ethyl-N-nitrosurea (ENU) resulted in the development of AML presenting as granulocytic sarcomas (solid tumors composed of leukemic myeloblasts) in 30% to 40% of the mice. No such tumors developed in control ENU-treated animals that lacked AML1-ETO expression. Furthermore, these tumors could be transplanted into syngeneic recipients and exhibited factor-independent growth in vitro. Thus, AML1-ETO in combination with cooperating mutations effectively induces a leukemia with features that resemble human t(8;21) disease.

A. AML1/ETO
![Diagram](https://i.imgur.com/yourimageurl.png)

B.
![Diagram](https://i.imgur.com/yourimageurl.png)

Figure 3. A, The structure of AML1-ETO consists of the N-terminal portion of AML1, including the Runt domain, fused to almost the entire ETO protein. AML1-ETO retains the ability to bind to the core enhancer sequence and heterodimerize with CBFβ; however, it lacks the transactivation domain of AML1. The position of the chromosomal breakpoint in the t(8;21) is indicated by an arrow. B, In contrast to native AML1, AML1-ETO can homo- or heterodimerize with other members of the ETO family, resulting in the recruitment of a corepressor complex consisting of NCoR, Sin3, and histone deacetylase (HDAC). This complex in turn represses the expression of genes normally regulated transcriptionally by the AML1/CBFβ complex.

An intriguing clinical observation in patients with AML1-ETO-expressing AML is relevant to this discussion. Several studies have documented that *AML1-ETO* transcripts are readily detectable by reverse transcriptase polymerase chain reaction in the bone marrow of many patients with t(8;21) AML who are in long-term remission [44-47]. Moreover, hematopoietic progenitor assays performed in vitro using highly purified stem cells from these patients have revealed the presence of t(8;21)-positive cells with the capacity for myeloid, macrophage, erythroid, and B-cell lineage differentiation [44,46,48].

Several important conclusions emerge from these studies in both humans and in murine models. First, acquisition of this chromosomal translocation appears to occur within the stem cell compartment in most, if not all, patients with t(8;21) AML. In contrast to the results from in vitro analyses of leukemic cell lines, the expression of AML1-ETO does not appear to block stem cell differentiation, because AML1-ETO-expressing HSCs from both patients with t(8;21)-positive AML as well as AML1-ETO knock-in mice have the capacity to differentiate along most hematopoietic lineages. Most importantly, these findings suggest that the expression of AML1-ETO alone is insufficient for leukemogenesis and that additional mutations, targeting as yet undefined signaling pathways, are required for leukemic transformation.

### 4. Insights Into AML1 Biology Obtained From the Analysis of Human Leukemia

In addition to chromosomal translocations, recent studies have identified other genetic mechanisms that also perturb the function of AML1 and likely play a central role in the pathogenesis of a subset of acute leukemias. Several somatically acquired point mutations in *AML1* have recently been identified in leukemic cells from patients with de novo and secondary AML, as well as a variety of chronic myeloproliferative disorders [49-51]. These mutations are primarily within the RD and include both missense mutations and nonsense mutations, the latter resulting in premature termination of the AML1 protein. Most of the missense mutations map to residues predicted to participate in sequence-specific DNA recognition mediated by the RD [52,53]. Not unexpectedly, AML1 mutants harboring these mutations have reduced or ablated DNA-binding activity and transcriptional transactivation. Many, but not all, of these mutants can, however, still heterodimerize with CBFβ [49,50]. By contrast, AML1 mutants resulting from premature termination generally lack both DNA-binding and heterodimerization activities. Interestingly, there appears to be a particularly high incidence of point mutations in the RD of the AML1 gene in poorly differentiated AMLs (classified as AML-M0 by French-American-British criteria) [49,51]. In our analysis of

40 cases of AML with M0 morphology, mutations of AML1 were detected in 20% (8/40) of cases [54]. Biallelic AML1 mutations were uncommon, present in only 1/8 cases, in contrast to the findings of an earlier report [51]. In addition to mutations affecting the RD, 3 mutations were identified in the coding sequences in the C-terminal portion of AML1, each of which causes a frame shift resulting in the addition of 59 to 135 amino acid residues, and are predicted to interfere with the transactivation function of AML1.

A rare clinical entity, familial platelet disorder with predisposition to acute myelogenous leukemia (FPD/AML), may provide significant insight into the mechanism by which these mutations in AML1 ultimately lead to the development of leukemia. FPD/AML is an autosomal dominant disorder linked to chromosome 21q22.1-22.2 [55-58]. Afflicted individuals have quantitative and functional platelet abnormalities as well as a markedly increased risk for the development of acute leukemia, primarily AMLs. Analysis of several FPD/AML pedigrees revealed the presence of either missense or nonsense mutations or intragenic deletions in one of the AML1 alleles [59]. The AML1 mutations cosegregated with the disease in each of the 6 pedigrees examined. These mutations resulted in either the complete deletion of one AML1 allele, premature termination of the coding sequence yielding a truncated AML1 protein, or mutation of critical amino acid residues within the RD that are involved in DNA binding [52,53]. Sequence analysis of the remaining AML1 allele revealed no mutations in any of the cases examined. Although some of the mutant AML1 proteins are predicted to function in a dominant-negative manner, several lines of evidence suggest that this may not be the case in every pedigree. As discussed above, mice expressing dominant-negative forms of AML1 uniformly manifest embryonic lethality, whereas patients with FPD/AML are born at the expected frequency [40,41]. Additionally, steady-state levels of mutant AML1 transcripts were much lower than those of wild-type AML1, as would be expected from a truncated transcript with reduced messenger RNA stability. Lastly, one pedigree was identified that contained an intrachromosomal deletion that eliminated one AML1 allele. Clearly, the loss of function of the remaining AML1 allele through other mechanisms must be more rigorously excluded. However, these findings raise the possibility that haploinsufficiency for AML1 is the underlying mechanism for the predisposition to AML. Thus, AML1 mutations may work through a mechanism distinct from that of the classic tumor-suppressor paradigm in which there is homozygous gene inactivation. How haploinsufficiency of AML1 predisposes to AML remains to be defined. Solution of the crystal structure of the AML1 RD-CBFβ-DNA ternary complex has recently been reported and provides a structural basis for the effects of these mutations on the biologic function of the AML1 protein [52,53]. The structure of the RD is that of a 12-stranded β-barrel oriented with the long axis perpendicular to that of the DNA helix, with DNA binding mediated by one facet of the β-barrel. Many of the contact points of the RD are centered in the βE′-F loop and the C-terminal portion of the RD, which appear to form a continuous contact surface that “clamps” the phosphate backbone between the major and minor grooves of the bound DNA [53]. Several of the somatically acquired and

germline mutations in AML1 discussed above, as well as germline mutations of RUNX2 that are associated with cleidocranial dysplasia, occur within these segments of the RD and would be predicted to disrupt DNA binding [60-63].

An alternative mechanism of altering AML1 function, gene amplification, has recently been described in several cases of acute lymphoblastic leukemia, AML, and myelodysplastic syndrome [64-66]. Trisomy 21 occurs at a relatively high frequency in acute leukemia [67-69]. Furthermore, the incidence of acute leukemia is 10-to-20-fold higher in children with Down syndrome than in the general pediatric population [70]. Trisomy 21 provides a quantitative increase in the dose of all genes residing on this chromosome, including AML1. Implicating AML1 as one of the critical targets is the recent finding of tandem duplications of AML1 in several AMLs with trisomy 21 as an acquired lesion. Furthermore, overexpression of AML1 as a result of retroviral insertion has been documented in several murine leukemias [71]. These observations raise the possibility that the overexpression of AML1 may also dysregulate the normal activity of AML1/CBFβ and lead to the initiation of leukemia. Clearly the normal hematopoietic function of AML1/CBFβ is critically sensitive to the dose of the active complex, and alterations in the functional level of AML1/CBFβ due to either haploinsufficiency or expression of dominant interfering fusion proteins can result in aberrations within HSCs that predispose them to cooperating mutations that lead to leukemia.

## 5. Summary

Alterations of the AML1 gene represent some of the most common genetic lesions in acute leukemia. In addition to translocations, point mutations, deletions, and gene amplification events target the AML1 gene and play a critical role in leukemogenesis. With the integration of data from structural and biochemical analyses, a more complete understanding is emerging on the molecular basis by which these mutations ultimately influence AML1-mediated transcriptional activation. A major challenge of future investigations is to translate our knowledge of the biochemical and biologic properties of the AML transcriptional pathway into the development of novel therapeutic agents for the treatment of leukemias caused by alterations in AML1.

## Acknowledgments

This work was supported in part by National Institutes of Health grants PO1 CA71907-6 and CA84221-03, Cancer Center CORE grant CA21765, and by the American Lebanese and Syrian Associated Charities (ALSAC) of St. Jude Children’s Research Hospital.

We would like to thank Dr. Noel Lenny for critically reading this manuscript.

## References

1. Ducy P. Cbfal: a molecular switch in osteoblast biology. *Dev Dyn*. 2000;219:461-471.
2. Westendorf JJ, Hiebert SW. Mammalian runt-domain proteins and

their roles in hematopoiesis, osteogenesis, and leukemia. J Cell
Biochem. 1999; (suppl) 32-33:51-58.
3. Ogawa E, Maruyama M, Kagoshima H, et al. PEBP2/PEA2 represents a family of transcription factors homologous to the products
of the Drosophila runt gene and the human AML1 gene. Proc Natl
Acad Sci USA. 1993; 90:6859-6863.
4. Kagoshima H, Shigesada K, Satake M, et al. The Runt domain
identifies a new family of heteromeric transcriptional regulators.
Trends Genet. 1993; 9:338-341.
5. Okuda T, van Deursen J, Hiebert SW, Grosveld G, Downing JR.
AML1, the target of multiple chromosomal translocations in
human leukemia, is essential for normal fetal liver hematopoiesis.
Cell. 1996; 84:321-330.
6. Wang Q, Stacy T, Binder M, Marin-Padilla M, Sharpe AH, Speck
NA. Disruption of the Cbfa2 gene causes necrosis and hemorrhaging in the central nervous system and blocks definitive hematopoiesis. Proc Natl Acad Sci USA. 1996; 93:3444-3449.
7. Sasaki K, Yagi H, Bronson RT, et al. Absence of fetal liver hematopoiesis in mice deficient in transcriptional coactivator core binding
factor beta. Proc Natl Acad Sci USA. 1996; 93:12359-12363.
8. Wang Q, Stacy T, Miller JD, et al. The CBFbeta subunit is essential
for CBFA2 (AML1) function in vivo. Cell. 1996; 87:697-708.
9. Downing JR. The AML1-ETO chimaeric transcription factor in
acute myeloid leukaemia: biology and clinical significance. Br J
Haematol. 1999; 106:296-308.
10. Nagata T, Gupta V, Sorce D, et al. Immunoglobulin motif DNA
recognition and heterodimerization of the PEBP2/CBF Runt
domain. Nat Struct Biol. 1999; 6:615-619.
11. Berardi MJ, Sun C, Zehr M, et al. The Ig fold of the core binding
factor alpha Runt domain is a member of a family of structurally
and functionally related Ig-fold DNA-binding domains. Structure
Fold Des. 1999; 7:1247-1256.
12. Ogawa E, Inuzuka M, Maruyama M, et al. Molecular cloning and
characterization of PEBP2 beta, the heterodimeric partner of a
novel Drosophila runt-related DNA binding protein PEBP2 alpha.
Virology. 1993; 194:314-331.
13. Wang S, Wang Q, Crute BE, Melnikova IN, Keller SR, Speck NA.
Cloning and characterization of subunits of the T-cell receptor and
murine leukemia virus enhancer core-binding factor. Mol Cell Biol.
1993; 13:3324-3339.
14. Huang G, Shigesada K, Ito K, Wee HJ, Yokomizo T, Ito Y. Dimerization with PEBP2beta protects RUNX1/AML1 from ubiquitin-
proteasome-mediated degradation. EMBO J. 2001; 20:723-733.
15. Kitabayashi I, Yokoyama A, Shimizu K, Ohki M. Interaction and
functional cooperation of the leukemia-associated factors AML1
and p300 in myeloid cell differentiation. EMBO J. 1998; 17:
2994-3004.
16. Yagi R, Chen LF, Shigesada K, Murakami Y, Ito Y. A WW domain-
containing yes-associated protein (YAP) is a novel transcriptional
co-activator. EMBO J. 1999; 18:2551-2562.
17. Bruhn L, Munnerlyn A, Grosschedl R. ALY, a context-dependent
coactivator of LEF-1 and AML-1, is required for TCRalpha
enhancer function. Genes Dev. 1997; 11:640-653.
18. Aronson BD, Fisher AL, Blechman K, Caudy M, Gergen JP. Groucho-dependent and -independent repression activities of Runt
domain proteins. Mol Cell Biol. 1997; 17:5581-5587.
19. Levanon D, Goldstein RE, Bernstein Y, et al. Transcriptional
repression by AML1 and LEF-1 is mediated by the TLE/Groucho
corepressors. Proc Natl Acad Sci USA. 1998; 95:11590-11595.
20. Lutterbach B, Westendorf JJ, Linggi B, Isaac S, Seto E, Hiebert SW.
A mechanism of repression by acute myeloid leukemia-1, the target
of multiple chromosomal translocations in acute leukemia.
J Biol Chem. 2000; 275:651-656.
21. Torchia J, Glass C, Rosenfeld MG. Co-activators and co-repressors
in the integration of transcriptional responses. Curr Opin Cell Biol.
1998; 10:373-383.
22. Feinstein PG, Kornfeld K, Hogness DS, Mann RS. Identification of
homeotic target genes in Drosophila melanogaster including nervy,
a proto-oncogene homologue. Genetics. 1995; 140:573-586.

23. Era T, Asou N, Kunisada T, et al. Identification of 2 transcripts of
AML1/ETO-fused gene in t(8;21) leukemic cells and expression of
wild-type ETO gene in hematopoietic cells. Genes Chromosomes
Cancer. 1995; 13:25-33.
24. Erickson PF, Dessev G, Lasher RS, Philips G, Robinson M, Drabkin HA. ETO and AML1 phosphoproteins are expressed in
CD34+ hematopoietic progenitors: implications for t(8;21) leukemogenesis and monitoring residual disease. Blood. 1996; 88:
1813-1823.
25. Gamou T, Kitamura E, Hosoda F, et al. The partner gene of AML1
in t(16;21) myeloid malignancies is a novel member of the
MTG8(ETO) family. Blood. 1998; 91:4028-4037.
26. Kitabayashi I, Ida K, Morohoshi F, et al. The AML1-MTG8
leukemic fusion protein forms a complex with a novel member of
the MTG8(ETO/CDR) family, MTGR1. Mol Cell Biol. 1998; 18:
846-858.
27. Hildebrand D, Tiefenbach J, Heinzel T, Grez M, Maurer AB. Multiple regions of eto cooperate in transcriptional repression. J Biol
Chem. 2001; 276:9889-9895.
28. Lutterbach B, Sun D, Schuetz J, Hiebert SW. The MYND motif is
required for repression of basal transcription from the multidrug
resistance 1 promoter by the t(8;21) fusion protein. Mol Cell Biol.
1998; 18:3604-3611.
29. Wang J, Hoshino T, Redner RL, Kajigaya S, Liu JM. ETO, fusion
partner in t(8;21) acute myeloid leukemia, represses transcription
by interaction with the human N-CoR/mSin3/HDAC1 complex.
Proc Natl Acad Sci USA. 1998; 95:10860-10865.
30. Zhang J, Hug BA, Huang EY, et al. Oligomerization of ETO is
obligatory for corepressor interaction. Mol Cell Biol. 2001; 21:
156-163.
31. Minucci S, Maccarana M, Ciocè M, et al. Oligomerization of RAR
and AML1 transcription factors as a novel mechanism of oncogenic activation. Mol Cell. 2000; 5:811-820.
32. Melnick A, Carlile GW, McConnell MJ, Polinger A, Hiebert SW, Licht JD. AML-1/ETO fusion protein is a dominant negative
inhibitor of transcriptional repression by the promyelocytic
leukemia zinc finger protein. Blood. 2000; 96:3939-3947.
33. Melnick AM, Westendorf JJ, Polinger A, et al. The ETO protein disrupted in t(8;21)-associated acute myeloid leukemia is a corepressor for the promyelocytic leukemia zinc finger protein. Mol Cell
Biol. 2000; 20:2075-2086.
34. Wang J, Saunthararajah Y, Redner RL, Liu JM. Inhibitors of histone deacetylase relieve ETO-mediated repression and induce differentiation of AML1-ETO leukemia cells. Cancer Res. 1999; 59:
2766-2769.
35. Ferrara FF, Fazi F, Bianchini A, et al. Histone deacetylase-targeted
treatment restores retinoic acid signaling and differentiation in
acute myeloid leukemia. Cancer Res. 2001; 61:2-7.
36. Rhoades KL, Hetherington CJ, Rowley JD, et al. Synergistic upregulation of the myeloid-specific promoter for the macrophage
colony-stimulating factor receptor by AML1 and the t(8;21)
fusion protein may contribute to leukemogenesis. Proc Natl Acad
Sci USA. 1996; 93:11895-11900.
37. Klampfer L, Zhang J, Zelenetz AO, Uchida H, Nimer SD. The
AML1/ETO fusion protein activates transcription of BCL-2. Proc
Natl Acad Sci USA. 1996; 93:14059-14064.
38. Shimizu K, Kitabayashi I, Kamada N, et al. AML1-MTG8 leukemic
protein induces the expression of granulocyte colony-stimulating
factor (G-CSF) receptor through the up-regulation of CCAAT/
enhancer binding protein epsilon. Blood. 2000; 96:288-296.
39. Shimada H, Ichikawa H, Nakamura S, et al. Analysis of genes
under the downstream control of the t(8;21) fusion protein AML1-
MTG8: overexpression of the TIS11b (ERF-1, cMG1) gene
induces myeloid cell proliferation in response to G-CSF. Blood.
2000; 96:655-663.
40. Yergeau DA, Hetherington CJ, Wang Q, et al. Embryonic lethality
and impairment of haematopoiesis in mice heterozygous for an
AML1-ETO fusion gene. Nat Genet. 1997; 15:303-306.

41. Okuda T, Cai Z, Yang S, et al. Expression of a knocked-in AML1-ETO leukemia gene inhibits the establishment of normal definitive hematopoiesis and directly generates dysplastic hematopoietic progenitors. *Blood*. 1998;91:3134-3143.

42. Rhoades KL, Hetherington CJ, Harakawa N, et al. Analysis of the role of AML1-ETO in leukemogenesis, using an inducible transgenic mouse model. *Blood*. 2000;96:2108-2115.

43. Higuchi M, O'Brien D, Lenny N, Yang S, Cai Z, Downing JR. Expression of AML1-ETO immortalizes myeloid progenitors and cooperates with secondary mutations to induce granulocytic sarcoma/acute myeloid leukemia [abstract]. *Blood*. 2001;96:222a.

44. Miyamoto T, Nagafuji K, Akashi K, et al. Persistence of multipotent progenitors expressing AML1/ETO transcripts in long-term remission patients with t(8;21) acute myelogenous leukemia. *Blood*. 1996;87:4789-4796.

45. Miyamoto T, Nagafuji K, Harada M, et al. Quantitative analysis of AML1/ETO transcripts in peripheral blood stem cell harvests from patients with t(8;21) acute myelogenous leukaemia. *Br J Haematol*. 1995;91:132-138.

46. Kusec R, Laczika K, Knobl P, et al. AML1/ETO fusion mRNA can be detected in remission blood samples of all patients with t(8;21) acute myeloid leukemia after chemotherapy or autologous bone marrow transplantation. *Leukemia*. 1994;8:735-739.

47. Nucifora G, Larson RA, Rowley JD. Persistence of the 8;21 translocation in patients with acute myeloid leukemia type M2 in long-term remission. *Blood*. 1993;82:712-715.

48. Miyamoto T, Weissman IL, Akashi K. AML1/ETO-expressing non-leukemic stem cells in acute myelogenous leukemia with 8;21 chromosomal translocation. *Proc Natl Acad Sci USA*. 2000;97:7521-7526.

49. Osato M, Asou N, Abdalla E, et al. Biallelic and heterozygous point mutations in the runt domain of the AML1/PEBP2alphaB gene associated with myeloblastic leukemias. *Blood*. 1999;93:1817-1824.

50. Imai Y, Kurokawa M, Izutsu K, et al. Mutations of the AML1 gene in myelodysplastic syndrome and their functional implications in leukemogenesis. *Blood*. 2000;96:3154-3160.

51. Preudhomme C, Warot-Loze D, Roumier C, et al. High incidence of biallelic point mutations in the Runt domain of the AML1/PEBP2 alpha B gene in Mo acute myeloid leukemia and in myeloid malignancies with acquired trisomy 21. *Blood*. 2000;96:2862-2869.

52. Tahirov TH, Inoue-Bungo T, Morii H, et al. Structural analyses of DNA recognition by the AML1/Runx-1 Runt domain and its allosteric control by CBFbeta. *Cell*. 2001;104:755-767.

53. Bravo J, Li Z, Speck NA, Warren AJ. The leukemia-associated AML1 (Runx1)—CBF beta complex functions as a DNA-induced molecular clamp. *Nat Struct Biol*. 2001;8:371-378.

54. Yeoh AE, Williams K, Behm F, et al. Somatic mutations of the AML-1 gene are frequent in acute myeloid leukemia with FAB M0 morphology [abstract]. *Blood*. 2000;96:91a.

55. Dowton SB, Beardsley D, Jamison D, Blattner S, Li FP. Studies of a familial platelet disorder. *Blood*. 1985;65:557-563.

56. Ho CY, Otterud B, Legare RD, et al. Linkage of a familial platelet

disorder with a propensity to develop myeloid malignancies to human chromosome 21q22.1-22.2. *Blood*. 1996;87:5218-5224.

57. Arepally G, Rebbeck TR, Song W, Gilliland G, Maris JM, Poncz M. Evidence for genetic homogeneity in a familial platelet disorder with predisposition to acute myelogenous leukemia (FPD/AML). *Blood*. 1998;92:2600-2602.

58. Luddy RE, Champion LA, Schwartz AD. A fatal myeloproliferative syndrome in a family with thrombocytopenia and platelet dysfunction. *Cancer*. 1978;41:1959-1963.

59. Song WJ, Sullivan MG, Legare RD, et al. Haploinsufficiency of CBFA2 causes familial thrombocytopenia with propensity to develop acute myelogenous leukaemia. *Nat Genet*. 1999;23:166-175.

60. Mundlos S, Otto F, Mundlos C, et al. Mutations involving the transcription factor CBFA1 cause cleidocranial dysplasia. *Cell* 1997;89:773-779.

61. Lee B, Thirunavukkarasu K, Zhou L, et al. Missense mutations abolishing DNA binding of the osteoblast-specific transcription factor OSF2/CBFA1 in cleidocranial dysplasia. *Nat Genet*. 1997;16:307-310.

62. Quack I, Vonderstrass B, Stock M, et al. Mutation analysis of core binding factor A1 in patients with cleidocranial dysplasia. *Am J Hum Genet*. 1999;65:1268-1278.

63. Zhou G, Chen Y, Zhou L, et al. CBFA1 mutation analysis and functional correlation with phenotypic variability in cleidocranial dysplasia. *Hum Mol Genet*. 1999;8:2311-2316.

64. Streubel B, Valent P, Lechner K, Fonatsch C. Amplification of the AML1(CBFA2) gene on ring chromosomes in a patient with acute myeloid leukemia and a constitutional ring chromosome 21. *Cancer Genet Cytogenet*. 2001;124:42-46.

65. Dal Cin P, Atkins L, Ford C, et al. Amplification of AML1 in childhood acute lymphoblastic leukemias. *Genes Chromosomes Cancer*. 2001;30:407-409.

66. Niini T, Kanerva J, Vettenranta K, Saarinen-Pihkala UM, Knuutila S. AML1 gene amplification: a novel finding in childhood acute lymphoblastic leukemia. *Haematologica*. 2000;85:362-366.

67. Raimondi SC, Chang MN, Ravindranath Y, et al. Chromosomal abnormalities in 478 children with acute myeloid leukemia: clinical characteristics and treatment outcome in a cooperative pediatric oncology group study-POG 8821. *Blood*. 1999;94:3707-3716.

68. Raimondi SC, Pui CH, Head D, et al. Trisomy 21 as the sole acquired chromosomal abnormality in children with acute lymphoblastic leukemia. *Leukemia*. 1992;6:171-175.

69. Watson MS, Carroll AJ, Shuster JJ, et al. Trisomy 21 in childhood acute lymphoblastic leukemia: a Pediatric Oncology Group study (8602). *Blood*. 1993;82:3098-3102.

70. Lange B. The management of neoplastic disorders of haematopoiesis in children with Down's syndrome. *Br J Haematol*. 2000;110:512-524.

71. Li J, Shen H, Himmel KL, et al. Leukaemia disease genes: large-scale cloning and pathway predictions. *Nat Genet*. 1999;23:348-353.
