
Letter

Identification of Genes Encoding Amino Acid Permeases by Inactivation of Selected ORFs from the Synechocystis Genomic Sequence

María José Quintero, María Luz Montesinos, Antonia Herrero, and Enrique Flores¹

Instituto de Bioquímica Vegetal y Fotosíntesis, Consejo Superior de Investigaciones Científicas-Universidad de Sevilla, E-41092 Seville, Spain

Genes encoding elements of four amino acid permeases were identified by insertional inactivation of ORFs from the genomic sequence of the cyanobacterium *Synechocystis* sp. strain PCC 6803 whose putative products are homologous to amino acid permease proteins from other bacteria. A transport system for neutral amino acids and histidine and a transport system for basic amino acids and glutamine were identified as ABC-type transporters, whereas Na⁺-dependent transport of glutamate was found to be mediated by at least two systems, the secondary permease GltS and a TRAP-type transporter. Except for GltS, substrate specificities of the identified permeases do not match those of previously characterized systems homologous to these permeases.

Availability of sequenced genomes permits new ways of identifying all of the genes required for a physiological function or a set of functions in the cell. In the case of membrane transporters, substrate specificity often correlates with phylogeny (Paulsen et al. 2000) making it possible to attempt the identification of genes encoding putative transporters for particular substrates by sequence comparisons (i.e., by homology searching). However, the number of sequenced bacterial genomes is outgrowing that of bacteria for which a substantial number of transporters has been characterized. Thus, it is common that a protein family including a considerable number of transporters is defined, with regard to substrate specificity, based only on one or a few family members that have been experimentally characterized. To define their substrates, it is therefore of interest to inactivate genes encoding putative membrane transporters in other than the most commonly studied bacteria. In this paper, we present the identification of ORFs from the genomic sequence of the cyanobacterium *Synechocystis* sp. strain PCC 6803 that encode elements of amino acid permeases with particular substrate specificities.

The cyanobacteria are phototrophic prokaryotes that carry out oxygenic photosynthesis. They mainly use inorganic sources of carbon (CO₂) and nitrogen for growth. The complete sequence of the chromosome of the unicellular cyanobacterium *Synechocystis* sp. strain PCC 6803 has been determined (Kaneko et al. 1996). Consistent with the mostly inorganic nutrition of cyanobacteria, only a low percentage, ~25%, of the putative transporters that would be encoded in that sequence are predicted to have organic compounds as substrates (Paulsen et al. 2000). The cyanobacteria for which amino acid transport has been investigated generally exhibit transport activities for neutral and, in some strains, basic and/or acidic amino acids (Montesinos et al. 1997). *Synechocystis* sp. strain PCC 6803 shows the highest activity of basic amino acid transport detected for any cyanobacterium and is unique in that it exhibits a high activity of glutamate transport whereas it shows a negligible aspartate transport activity (Labarre et al. 1987; Montesinos et al. 1997). This cyanobacterium has been described to bear three amino acid transport systems: one for basic amino acids with high affinity for arginine (*K*ₛ <1 μM; Flores and Muro-Pastor 1990) that would also transport glutamine with low affinity (Labarre et al. 1987; Flores and Muro-Pastor 1990); one for neutral (both polar and hydrophobic) amino acids excluding glutamine (Labarre et al. 1987); and one for glutamate that would also be a high-affinity glutamine transporter (Labarre et al. 1987). The latter system was, however, not supported by further data that indicated the presence of a transport system specific for glutamate and different from the high-affinity glutamine transport system (Montesinos et al. 1997). Among these systems, some molecular data are available only for the neutral amino acid transporter for which two genes (*natA* and *natB*) encoding elements, an ATP-binding subunit and a periplasmic substrate-binding protein, respectively, of an ABC-type permease have been identified (Montesinos et al. 1997). Here we report the identification of other *Synechocystis* genes encoding amino acid transport elements. The *Synechocystis* sequences used can be obtained from the CYANOBASE Web page (Kaneko et al. 1996; http://www.kazusa.or.jp/cyano/cyano.html).

RESULTS

Neutral Amino Acid Permease

The previously characterized NatA and NatB components of the neutral amino acid transporter of *Synechocystis* sp. strain PCC 6803 (Montesinos et al. 1997) are most similar to the BraF (ATP-binding subunit) and BraC (periplasmic substrate-binding protein) components, respectively, of the high-affinity branched-chain amino acid transport system of *Pseudomonas aeruginosa* (Hoshino and Kose 1990). In the *Synechocystis* genome, neither *natA* (*slr0467*) nor *natB* (*slr0559*) is clustered with other permease genes (Kaneko et al. 1996). Based on the reported observation that the constituents of ABC-type permeases appear to have evolved with minimal shuffling of constituents between systems (Saier 1998), we

¹ Corresponding author.
E-MAIL flores@cica.es; FAX 34 95 446 0065.
Article and publication are at http://www.genome.org/cgi/doi/10.1101/gr.196301.

2034 Genome Research
www.genome.org
11:2034–2040 ©2001 by Cold Spring Harbor Laboratory Press ISSN 1088-9051/01 $5.00; www.genome.org

sought the inactivation of those *Synechocystis* ORFs, *slr0949*, *slr1881*, and *sll0146*, whose putative products are most similar to the other subunits of the Bra system, BraD (transmembrane protein), BraG (ATP-binding subunit), and BraE (transmembrane protein), respectively. Inactivation of *slr0949*, *slr1881*, and *sll0146* with gene-cassette C.K3 (see Methods) rendered strains CSMJ36, CSMJ37, and CSMJ41, respectively. These mutant strains were severely affected in the uptake of all the neutral amino acids tested, except glutamine, whereas uptake of basic (arginine and lysine) or acidic (glutamate) amino acids was much less affected (Table 1). These results indicate that the products of the three ORFs are necessary, together with NatA and NatB, to constitute a functional transporter for neutral amino acids. We have named *slr0949* as *natD*, *slr1881* as *natE*, and *sll0146* as *natC*.

### Basic Amino acid and Glutamine Permease

The putative product of ORF *sll1270* from the *Synechocystis* genome resembles a fusion protein between a periplasmic substrate-binding protein (N-terminal half) most similar to *Escherichia coli* GlnH (a glutamine-binding protein) and an integral membrane protein (C-terminal half) most similar to *E. coli* GlnP (a transmembrane protein of a glutamine ABC-type transporter). A *sll1270::C.C1* mutant, strain CSMJ7, was constructed. This mutant showed a low activity of uptake of arginine, histidine, and lysine, whereas uptake of the tested neutral amino acids (except glutamine) and of glutamate was much less affected (Table 1). Glutamine uptake was also severely affected. These results identify the product of *sll1270* as an element of the previously described *Synechocystis* transport system for basic amino acids and glutamine (Labarre et al. 1987; Flores and Muro-Pastor 1990). The third element, an ATP-binding subunit, of the *E. coli* Gln system is GlnQ, and we inactivated the *Synechocystis* ORF whose putative product is most similar to GlnQ, *slr1735*. Strain CSMJ8 carrying *slr1735::C.K3* showed a phenotype of amino acid uptake identical to that of the *sll1270* mutant, as did strain CSMJ11, a *slr1735::C.K3 sll1270::C.C1* double mutant (Table 1). These results imply that the products of the two ORFs encode elements of the same transporter. We have named *slr1735* as *bgtA* and *sll1270* as *bgtB* (*bgt* stands for basic amino acid and glutamine transport). BgtA would represent an ATP-binding subunit and BgtB would carry periplasmic substrate-binding protein and transmembrane protein domains of an ABC-type permease.

Uptake was also tested for arginine, glutamine, histidine, and lysine concentrations of 1 μM and 100 μM. For both strains CSMJ7 and CSMJ8, lysine uptake was always ≤1% of the wild-type levels, whereas uptake was <1% and ~20% of the wild-type levels for 1 μM and 100 μM arginine, respectively. Another arginine transport system, with lower affinity for arginine than Bgt, appears therefore to be expressed in *Synechocystis* sp. strain PCC 6803. Consistently, strains CSMJ7, CSMJ8, and CSMJ11 were still able to grow using 5 mM L-arginine as the nitrogen source, but their growth rate constants (determined as described in Montesinos et al. 1995) were ~20% that of the wild-type strain. Glutamine uptake in strains CSMJ7 and CSMJ8 was most severely affected at 100 μM (~5% of the wild-type levels) than at 1 μM (~40%). These results are consistent with previously reported data (Labarre et al. 1987; Flores and Muro-Pastor 1990) and suggest that Bgt, which is a high-affinity system for basic amino acids, represents a low-affinity, high-capacity system for glutamine. Up-

take of histidine in the *bgt* mutants was also more severely affected at 100 μM (~4% of the wild-type levels) than at 1 μM (~25%). Therefore, transport system(s) exhibiting a higher affinity than Bgt for both glutamine and histidine appear to be expressed in *Synechocystis* sp. strain PCC 6803. Transport assays carried out with strains CSMJ7 and CSMJ8 for glutamine concentrations of 1, 5, 10, 15, and 20 μM indicated a \(K_s\) (glutamine) of ~2 μM.

#### nat bgt Double Mutants

Strains CSX28a (*natA::C.K3*) and CSX18a (*natB::C.K3*) (Montesinos et al. 1997) were transformed with plasmids carrying *bgtA(slr1735)::C.C1* or *bgtB(sll1270)::C.C1* constructs, and *natA bgtA* (strain CSMJ32), *natA bgtB* (CSMJ30), *natB bgtA* (CSMJ26), and *natB bgtB* (CSMJ24) double mutants were obtained. Amino acid uptake assays carried out with these strains (Table 1) indicated that mutation of the two systems resulted in a very low transport activity of any of the basic or neutral amino acids tested, whereas the glutamate transport activity was much less affected. Additionally, uptake of glutamine and histidine was also tested for substrate concentrations of 1 μM and 100 μM, and activities ≤3% of the wild-type activities were observed in every case. These results indicate that the Nat system is responsible for the high-affinity transport activities of glutamine and histidine discussed above and that the Bgt and Nat transporters together account for most of the transport activity detected for the tested amino acids except glutamate in *Synechocystis* sp. strain PCC 6803.

### Glutamate Permeases

ORF *slr1145* of *Synechocystis* sp. strain PCC 6803 (Kaneko et al. 1996) would encode a protein with 42% identity to the GltS Na+/glutamate permease of *Escherichia coli* (Deguchi et al. 1990). Our amino acid uptake assays are normally performed in the presence of 12.5 mM of Na+ ions, and under these conditions *Synechocystis* sp. strain PCC 6803 shows a glutamate uptake activity with an apparent \(K_s\) of 49 μM and a \(V_{\text{max}}\) of 529 nmole/min per mg of Chl (Montesinos et al. 1997). We have now observed that glutamate uptake in this cyanobacterium is dependent on the presence of Na+ in the incubation buffer. When Tricine-KOH substituted for Tricine-NaOH as the incubation buffer, maximal activity of transport of 10 μM glutamate was observed when the cells were supplemented with ~50 mM NaCl, ≤5% of the maximal activity was observed without added NaCl, and ~50% of the maximal activity was observed in the presence of 150 mM NaCl (results not shown).

A *Synechocystis slr1145::C.K3* mutant was generated and named strain CSMJ6. Assays of uptake of Arg, Lys, Ala, Gln, Gly, Pro, and Ser indicated that the transport of none of them was impaired in this mutant (data not shown). However, uptake of glutamate in Tricine-NaOH buffer (~12.5 mM Na+) was reduced in CSMJ6, depending on the concentration of glutamate used, to 56% to 69% of the values exhibited by the wild-type strain (4.8 ± 0.8 and 8.5 ± 2.7 nmole of glutamate/min per mg of Chl for strains CSMJ6 and PCC 6803, respectively, for 1 μM glutamate; 51.9 ± 9.7 and 76.2 ± 20.3 for strains CSMJ6 and PCC 6803, respectively, for 10 μM glutamate; and 227.5 ± 41.2 and 330.9 ± 27.8 for strains CSMJ6 and PCC 6803, respectively, for 100 μM glutamate; data are the mean and standard deviation from four to seven independent determinations). A similar decrease in uptake was found for the CSMJ6 mutant when glutamate uptake was tested with

Table 1. Amino Acid Transport Activities of Synechocystis sp. Strain PCC 6803 and Some Mutants

| Amino acid transport in strain PCC 6803<sup>a</sup> | Substrate | CSMJ36 natD (slr0949) | CSMJ37 natE (slr1881) | CSMJ41 natC (slr0146) | CSMJ7 bgtB (sll1270) | CSMJ8 bgtA (slr1735) | CSMJ11 bgTB bgTA (slr1270 slr1735) | CSMJ24 natB bgTB (slr0559 sll1270) | CSMJ26 natA bgTA (slr0467 slr1735) |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| (nmol · min<sup>-1</sup>) | Arg | 426.7 | 90 | n.d.<sup>c</sup> | 92 | n.d. | 3 | 3 | <1 |
| (.mg Chl<sup>-1</sup>) | His | 6.2 | 5.9 | 95 | 88 | 87 | 9 | 12 | 10 |
| Lys | 52.8 | 10.2 | 98 | 86 | 89 | <1 | <1 | <1 | <1 |
| Neutral | Ala | 5.9 | 9.4 | 1 | 1 | 1 | 80 | 91 | 97 |
| Gln | 9.4 | 7.7 | 1 | 1 | <1 | 19 | 22 | 21 |
| Gly | 7.7 | 6.1 | 2 | 2 | 1 | 80 | 89 | 93 |
| Leu | 6.1 | 7.0 | 1 | 1 | 1 | n.d. | n.d. | n.d. |
| Pro | 7.0 | 2 | 1 | 1 | 1 | 79 | 99 | 85 |
| Ser | 2 | 2 | 2 | 2 | 2 | 78 | 80 | 96 |
| Acidic | Glu | 60.9 | 87 | 98 | 71 | 99 | 74 | 97 | 95 |

| Amino acid transport (% of the wild-type activity)<sup>b</sup> |  |  |  |  |  |  |  |  |  |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
|  | Basic | 1 | 2 | 5 | 1 | 2 | 3 | 1 | 1 |
|  | His | <1 | <1 | <1 | <1 | <1 | <1 | <1 | <1 |
|  | Lys | 1 | 2 | 3 | 1 | 1 | 2 | 2 | 1 |
|  | Neutral | Ala | 1 | 2 | 2 | 1 | 1 | 1 | 1 |
|  | Gln | 3 | 2 | 1 | 1 | 1 | 1 | 1 | 1 |
|  | Gly | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1 |
|  | Leu | n.d. | n.d. | n.d. | n.d. | n.d. | n.d. | n.d. | n.d. |
|  | Pro | <1 | <1 | <1 | <1 | <1 | <1 | <1 | <1 |
|  | Ser | 2 | 2 | 2 | 2 | 2 | 2 | 2 | 2 |
|  | Acidic | Glu | 84 | 85 | 89 | 95 | 97 | 99 | 95 | 84 |

a Data are the mean of 6–12 determinations with similar results (within 20% of the mean).  
b Two to four determinations were performed for each mutant, with similar results. For simplicity, the data are presented as a percent of the wild-type activity.  
c n.d., not determined.
different Na<sup>+</sup> concentrations (Table 2). Consistent with its homology to *E. coli* GltS, we conclude that the ORF *slr1145* product is a Na<sup>+</sup>-dependent glutamate transporter and that ORF *slr1145* represents a *Synechocystis* sp. strain PCC 6803 *gltS* gene. However, at least another glutamate transporter appears to be expressed in this cyanobacterium.

TRAP permeases are binding protein-dependent secondary transport systems that consist of a periplasmic substrate-binding protein and two different integral membrane proteins (Forward et al. 1997; Rabus et al. 1999). ORFs *sll1102* and *sll1103* of *Synechocystis* sp. strain PCC 6803 (Kaneko et al. 1996) would encode proteins homologous to the membrane components, DctQ and DctM, respectively, of a *Rhodobacter capsulatus* TRAP transporter (Forward et al. 1997), whereas ORF *sll1104* would encode a polypeptide homologous to glutamine (26.7% identity to *E. coli* GlnH), glutamate (24.2% identity to *Corynebacterium glutamicum* GluB) and glutamate/glutamine/aspartate/asparagine (22.7% identity to *R. capsulatus* BztA) binding proteins of ABC-type transporters. Because *sll1102*, *sll1103*, and *sll1104* are tightly clustered in the *Synechocystis* genome, it has been suggested that they might together encode a TRAP-type glutamine transport system (Forward et al. 1997). A *sll1102-sll1103* mutant, strain CSX67, was generated in which a 370-bp *BamHI* fragment, including part of *sll1102* and part of *sll1103*, was substituted by the C.K3 gene-cassette, and a *sll1104::C.K3* mutant was also generated and named strain CSX68. Uptake assays with Arg, Lys, Ala, Gln, Gly, Pro, Ser, and Leu indicated that CSX67 and CSX68 were not impaired in the transport of any of these amino acids (not shown). Because of the homology of the putative ORF *sll1104* product to glutamine-binding proteins, glutamine uptake was further tested in the mutants at substrate concentrations of 1, 2, 10, 20, 100, and 200 μM, but no effect on the uptake rates, as compared with those exhibited by the wild type, was observed (not shown). However, Na<sup>+</sup>-dependent glutamate uptake was reduced in the CSX67 and CSX68 mutants to ~70%–80% of the wild-type values (Table 2). These values represent a small but reproducible decrease in activity. We suggest that ORFs *sll1102*, *sll1103*, and *sll1104* encode a TRAP-type Na<sup>+</sup>-dependent glutamate transporter and will name them as *gtrA*, *gtrB*, and *gtrC*, respectively (*gtr* stands for glutamate and TRAP).

### gltS gtr Double Mutants

Using mutants CSX67 and CSX68 as parental strains, *gltS gtr* double mutants were generated. A *gltS(slr1145):C.C1* construct was transferred to strains CSX67 and CSX68 and mutant strains CSMJ20 (derivative of CSX67) and CSMJ22 (derivative of CSX68) were obtained. Na<sup>+</sup>-dependent glutamate uptake was significantly decreased in the CSMJ20 and CSMJ22 mutants (Table 2). The effects of the *slr1145* and *sll1104* mutations were additive, that is, mutant CSMJ22 exhibited the expected activity for a strain carrying mutations in the two transporters, GltS and Gtr, if they worked independently. No such additive effect was observed for the *slr1145* and *sll1102/sll1103* mutations in strain CSMJ20, although its glutamate uptake activity was lower than those exhibited by mutants CSMJ6 and CSX67. The reason for the lack of additivity of the mutations in strain CSMJ20 was not further investigated.

## DISCUSSION

### Synechocystis Amino Acid Permeases

The four amino acid permeases identified in *Synechocystis* sp. strain PCC 6803 are summarized in Table 3, where the transporter classification (TC) family (Saier 2000) to which each of those permeases belongs is also indicated. Nat is a transport system for neutral amino acids and histidine that seems to be widespread in cyanobacteria (Montesinos et al. 1997). In contrast with earlier suggestions (Labarre et al. 1987), our results show that Nat also transports glutamine and that it does so with high affinity (*K*<sub>s</sub>, 2 μM). Thus, the high-affinity glutamine transport system described by Labarre et al. (1987) is Nat itself. Five genes encoding subunits of the Nat permease have been identified whose putative products would constitute a typical ABC-type transporter: a periplasmic substrate-binding protein, two transmembrane subunits, and two ATP-binding subunits. The Bgt system is an ABC-type permease that corresponds to the high-affinity basic amino acid transport system that also transports glutamine with low affinity (Labarre et al. 1987; Flores and Muro-Pastor 1990). One of its components, BgtB, represents an unusual fusion between a periplasmic substrate-binding protein and an integral membrane protein with five or six putative transmembrane segments. We do not know whether the functional Bgt permease is made up of BgtA and BgtB homodimers or whether other genes encode complementary transmembrane and ATP-binding subunits. In contrast to the operon structure commonly observed for genes encoding ABC-type permeases in many bacteria, the *nat*

### Table 2. Na<sup>+</sup>-Dependent Glutamate Transport in *Synechocystis* sp. Strain PCC 6803 and *gltS* and *gtr* Mutants

| Strain   | Genotype                          | Glutamate uptake<sup>a</sup> (nmol · mg Chl<sup>−1</sup>) |
|----------|-----------------------------------|---------------------------------------------------------|
|          |                                   | 16 mM NaCl                                             | 46 mM NaCl                                             |
| PCC 6803 | Wild type                         | 271 ± 99                                                | 301 ± 86                                                |
| CSMJ6    | *gltS(slr1145)*                   | 164 ± 39                                                | 184 ± 50                                                |
| CSX67    | *gtrAB(sll1102/sll1103)*          | 198 ± 57                                                | 212 ± 39                                                |
| CSX68    | *gtrC(sll1104)*                   | 203 ± 47                                                | 243 ± 66                                                |
| CSMJ20   | *gltS gtrAB(slr1145 sll1102/sll1103)* | 151 ± 51                                                | 160 ± 40                                                |
| CSMJ22   | *gltS gtrC(slr1145 sll1104)*      | 107 ± 28                                                | 131 ± 39                                                |

<sup>a</sup>The assays were performed for 10 min with 10 μM [¹⁴C]glutamate in 25 mM Tricine-KOH (pH 8.1) buffer supplemented with the indicated NaCl concentrations. The figures shown are the mean and standard deviation of the data from four or five independent experiments. With no added NaCl, activities were ~15 nmol · mg Chl<sup>−1</sup> for the wild type and negligible for the mutants.
Table 3. Identified Amino Acid Permeases of Synechocystis sp. Strain PCC 6803

| Permease | TC family<sup>a</sup> | Substrate(s) | ORF | Gene | Putative gene-product |
|----------|---------------------|--------------|-----|------|-----------------------|
| Nat      | 3.A.1              | Neutral amino acids and histidine | slr0467 | natA | ATP-binding subunit |
|          |                    |              | slr0559 | natB | Periplasmic substrate-binding protein |
|          |                    |              | sll0146 | natC | Integral membrane protein |
|          |                    |              | slr0949 | natD | Integral membrane protein |
|          |                    |              | slr1881 | natE | ATP-binding subunit |
| Bgt      | 3.A.1              | Basic amino acids and glutamine | slr1735 | bgtA | ATP-binding subunit |
|          |                    |              | sll1270 | bgtB | Periplasmic substrate-binding and integral membrane protein |
| GltS     | 2.A.27             | Glutamate    | slr1145 | gltS | Monocomponent permease |
| Gtr      | 2.A.56             | Glutamate    | sll1102 | gtrA | Integral membrane protein |
|          |                    |              | sll1103 | gtrB | Integral membrane protein |
|          |                    |              | sll1104 | gtrC | Periplasmic substrate-binding protein |

<sup>a</sup>Transporter classification (TC) family according to Saier (2000): 3.A.1, ABC-type uptake permeases; 2.A.27, glutamate: Na<sup>+</sup> symporter family; 2.A.56, tripartite ATP-independent periplasmic transporter family.

and *bgt* genes are spread in the *Synechocystis* genome (Kaneko et al. 1996). As pointed out by Tomii and Kanehisa (1998), in the *Synechocystis* genome there are less cases of conserved operons, but more cases of multiple components of a transporter being fused into a single gene, than in other genomes.

The Bgt and Nat systems together account for most of the transport activities of amino acids (other than glutamate) that have been observed in *Synechocystis* sp. strain PCC 6803, although the permease responsible for the low-affinity transport of arginine observed in *bgt* mutants has not yet been identified. The Nat system appears to have a role in recapture of hydrophobic amino acids leaked from the cells (Labarre et al. 1987; Montesinos et al. 1997), and it might also have a nutritional role in uptake of amino acids like proline that can be used as a poor nitrogen source by *Synechocystis* sp. strain PCC 6803 (M.J. Quintero, A. Herrero, and E. Flores, unpubl.). Bgt can also have a nutritional role, for instance in the uptake of arginine that can serve as a nitrogen source for this cyanobacterium (Flores and Muro-Pastor 1990). Because this permease shows an affinity much higher for arginine than for any other amino acid, it has been suggested that it should be considered to represent an arginine transporter (O. Köster, H. Ullrich, and F. Jüttner, unpubl.).

We have found that the glutamate transport activity of *Synechocystis* sp. strain PCC 6803 is Na<sup>+</sup>-dependent and have identified two permeases involved in this activity: GltS, a monocomponent secondary permease, and GtrABC, a TRAP-type transporter. Because GltS and GtrABC would together account for no more than about 60% of the wild-type activity, other Na<sup>+</sup>-dependent glutamate transport system(s) must be present in this cyanobacterium. Some of the *nat* mutants that have been analyzed are somewhat affected in glutamate uptake (Table 1, see also Montesinos et al. 1997), but we do not know whether the Nat system might represent a significant way for glutamate uptake in strain PCC 6803. The relatively high activity of glutamate transport in strain PCC 6803 appears therefore to result from addition of the activities of several independent transporters which, nonetheless, are functionally similar as all of them are Na<sup>+</sup> dependent and appear to work with a similar range of glutamate concentrations. In the genome sequence of *Anabaena* sp. strain PCC 7120, a cyanobacterium that exhibits a glutamate transport activity much lower (about 1.2%) than that of *Synechocystis* sp. strain PCC 6803 (Montesinos et al. 1997), no homologs to *gltS* or *gtr*

are found (Kazusa DNA Research Institute, http://www.kazusa.or.jp/cyano/anabaena/). The physiological role(s) of the *Synechocystis* Na<sup>+</sup>-dependent glutamate transport activities are currently unknown.

Apart from the ORFs listed in Table 3, the other ORFs from the *Synechocystis* genome that are predicted to encode amino acid permeases (see I.T. Paulsen, M.K. Sliwinski, and J. Garg, http://www.biology.ucsd.edu/~ipaulsen/transport/) are a set of ORFs (*slr0447*, *slr1200*, *slr1201*, *sll0764*, and *sll0374*), which have been found to constitute an ABC-type permease for urea rather than for amino acids (A. Valladares, M.L. Montesinos, A. Herrero, and E. Flores, unpubl.).

### Permease Substrate Specificities

Homology searches have permitted the identification of ORFs encoding amino acid permeases in *Synechocystis* sp. strain PCC 6803. However, the precise specificity of an identified transporter did not generally correspond to those of the previously characterized permeases that are most similar to the investigated *Synechocystis* permease. Thus, Bgt is most similar to GlnHPQ of *E. coli*, a glutamine permease, Nat is most similar to Bra of *P. aeruginosa*, a branched-chain amino acid permease, and GtrC, the periplasmic substrate-binding protein of the TRAP-type Gtr permease, is most similar to GlnH. Only one member of the family of TRAP transporters, the DctPQM system for transport of C<sub>4</sub>-dicarboxylates in *R. capsulatus* (Forward et al. 1997), had been previously characterized both molecularly and functionally. Our results showing that the *Synechocystis* GtrABC system is involved in glutamate uptake support the contention that TRAP permeases can mediate transport of substrates other than C<sub>4</sub>-dicarboxylates (Jacobs et al. 1996; Rabus et al. 1999). On the other hand, *Synechocystis* GltS, which is the second GltS permease that has been characterized, has the same substrate specificity as *E. coli* GltS (Deguchi et al. 1990).

### METHODS

#### Generation of Mutants

The ORFs of the *Synechocystis* sp. strain PCC 6803 chromosome (Kaneko et al. 1996) inactivated in this work are summarized in Table 4. DNA fragments corresponding to those ORFs were amplified by PCR using primers whose coordinates in the strain PCC 6803 chromosome are indicated in Table 4.

Downloaded from genome.cshlp.org on June 7, 2015 - Published by Cold Spring Harbor Laboratory Press

# Synecchocystis Amino Acid Permeases

## Table 4. ORFs of *Synecchocystis* sp. Strain PCC 6803 Inactivated in This Work$^a$

| ORF         | ORF coordinates                | Oligonucleotides coordinates                                      | Inserted cassette and insertion site                     |
|-------------|-------------------------------|------------------------------------------------------------------|---------------------------------------------------------|
| *slr1145*   | 809450–810658                 | 809342–809362, 810371–810351                                     | C.K3 or C.C1, Ncol (Klenow-filled)                      |
| *sll1270*   | 1116021–1114429               | 1116052–1116033, 1115052–1115072                                 | C.C1, KpnI                                              |
| *slr1735*   | 1317712–1318470               | 1317628–1317647, 1318781–1318761                                 | C.K3 or C.C1, SmaI                                       |
| *slr0949*   | 2017104–2017964               | 2017139–2017158, 2018372–2018353                                 | C.K3, HpaI                                              |
| *slr1881*   | 1239490–1240212               | 1239564–1239583, 1240494–1240475                                 | C.K3, Stul                                              |
| *sll0146*   | 2192051–2190933               | 2191877–2191858, 2190922–2190941                                 | C.K3, SmaI                                              |
| *sll1102*, *sll1103* | 1914652–1914113, 1914113–1912776 | 1914799–1914779, 1913403–1913423                             | C.K3, 2BamHI sites                                      |
| *sll1104*   | 1912786–1911896               | 1912674–1912654, 1911573–1911593                                 | C.K3, Xbal                                              |

$^a$ The names and chromosome coordinates (Kaneko et al. 1996) of the inactivated ORFs are indicated in the first and second columns, respectively. The coordinates of the oligonucleotides used for PCR amplification of those ORFs are indicated in the third column. The inserted antibiotic resistance-encoding gene cassette and the restriction endonuclease insertion site(s) are indicated in the last column.

Isolation of genomic DNA from cyanobacteria (Cai and Wolk 1990) and PCR amplification (Quintero et al. 2000) were carried out as described previously. The PCR products were cloned in vector pGEM-T (Promega). The identity of the cloned fragment was verified by restriction endonuclease analysis or by sequencing. Gene-cassette C.K3 or C.C1 (Elhai and Wolk 1988) was inserted by standard procedures into the endonuclease restriction site(s) indicated in Table 4 for each ORF. These restriction sites were unique in the corresponding DNA fragment except for the two BamHI sites in the insert carrying part of *sll1102* and *sll1103*; in this case, a deletion of 370 bp accompanied the insertion of the C.K3 cassette. Growth of *E. coli* DH5α, isolation of plasmid DNA, and DNA restriction and ligation were carried out by standard methods (Sambrook et al. 1989).

Transformation of *Synecchocystis* sp. with plasmids carrying the disrupted DNA fragments was carried out as described previously (Quintero et al. 2000). Transformants were selected in BG11 solid medium supplemented with 25 μg of kanamycin (Km)/mL for clones carrying the C.K3 cassette or 10 μg of chloramphenicol (Cm)/mL for those carrying C.C1. To facilitate segregation of the mutant chromosomes, individual colonies were re-isolated from Km<sup>r</sup> or Cm<sup>r</sup> transformants and grown in liquid medium supplemented with up to 300 μg of Km/mL or 20 μg of Cm/mL, respectively. To test whether the resulting mutant strains were homozygous for the mutant chromosomes, PCR amplification using the corresponding primers and genomic DNA from each mutant as template and/or Southern blot using the corresponding PCR-amplified DNA fragments as probes were carried out (Quintero et al. 2000). Strains homozygous for the mutated chromosome were obtained for all the disrupted ORFs. Double mutants were obtained following the same methodology using a single mutant as parental strain.

The orientation (direct or inverse) of the inserted cassette with respect to that of the inactivated *Synecchocystis* ORF was determined for strains CSMJ6 (*slr1145::C.K3*, direct), CSMJ8 (*slr1735::C.K3*, direct), CSMJ36 (*slr0949::C.K3*, inverse), CSMJ37 (*slr1881::C.K3*, inverse), and CSMJ41 (*sll0146::C.K3*, direct). For ORFs *slr1735* and *sll0146*, mutants with the cassette inserted in the inverse orientation were also analyzed, and results identical to those described in this work for the direct orientation were obtained. In any case, no transport phenotypes resulting from polar effects are expected for the mutants generated in this work, because, with the exception of the *sll1102-sll1103-sll1104* cluster, the inactivated ORFs are not clustered with other putative transporter-encoding genes (Kaneko et al. 1996).

### Uptake Assays

Cells from shaken cultures grown at 30°C in the light in BG11 medium (supplemented with antibiotics, 25 μg of Km/mL and/or 10 μg of Cm/mL, in the case of the mutants) were harvested by low-speed centrifugation at room temperature, washed twice with 25 mM N-tris(hydroxymethyl)-methylglycine (Tricine)-NaOH buffer (pH 8.1), and resuspended in the same buffer. To test the effect of Na<sup>+</sup>, Tricine-KOH buffer was used. The concentration of chlorophyll *a* (Chl) in the cell suspension was determined in methanolic extracts (Mackinney 1941). The uptake assays were performed for 10 min (1 min in the case of arginine and glutamate), unless otherwise indicated, at 30°C in the light (white light from fluorescent lamps) and were started by mixing a suspension (1.0 mL) of cells containing 4–10 μg of Chl with a solution (0.1 mL) of L-[U-<sup>14</sup>C]amino acid (4.5 to 70 μCi/μmole) (radioactive amino acids were from Amersham or New England Nuclear). Final concentration of amino acid in the assay was 10 μM unless otherwise indicated. At the end of the incubation, a 1-mL sample was filtered (0.45-μm pore size Millipore HA filters were used) and the cells on the filters were washed with 5–10 mL Tricine buffer. The filters carrying the cells were then immersed in a scintillation cocktail, and their radioactivity was measured. Retention of radioactivity by boiled cells was used as a blank.

---

### ACKNOWLEDGMENTS

This work was supported by grants PB97–1137 and PB98–0481 from the Ministerio de Ciencia y Tecnología, Spain.

The publication costs of this article were defrayed in part by payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 USC section 1734 solely to indicate this fact.

---

### REFERENCES

Cai, Y. and Wolk, C.P. 1990. Use of a conditionally lethal gene in *Anabaena* sp. strain PCC 7120 to select for double recombinants and to entrap insertion sequence. *J. Bacteriol.* **172**: 3138–3145.

Deguchi, Y., Yamato, I., and Anraku, Y. 1990. Nucleotide sequence of *gltS*, the Na<sup>+</sup>/glutamate symport carrier gene of *Escherichia coli*. *B. J. Biol. Chem.* **265**: 21704–21708.

Elhai, J. and Wolk, C.P. 1988. A versatile class of positive-selection vectors based on the nonviability of palindrome-containing plasmids that allows cloning into long polylinkers. *Gene* **68**: 119–138.

Flores, E. and Muro-Pastor, A.M. 1990. Mutational and kinetic analysis of basic amino acid transport in the cyanobacterium *Synechocystis* sp. PCC 6803. *Arch. Microbiol.* **154**: 521–527.

Forward, J.A., Behrendt, M.C., Wyborn, N.R., Cross, R., and Kelly D.J. 1997. TRAP transporters: A new family of periplasmic solute transport systems encoded by the *dctPQM* genes of *Rhodobacter*

capsulatus and by homologs in diverse Gram-negative bacteria. *J. Bacteriol.* **179**: 5482–5493.

Hoshino, T. and Kose, K. 1990. Cloning, nucleotide sequences, and identification of products of the *Pseudomonas aeruginosa* PAO bra genes, which encode the high-affinity branched-chain amino acid transport system. *J. Bacteriol.* **172**: 5531–5539.

Jacobs, M.H.J., van der Heide, T., Driessen, A.J.M., and Konings, W.N. 1996. Glutamate transport in *Rhodobacter sphaeroides* is mediated by a novel binding protein-dependent secondary transport system. *Proc. Natl. Acad. Sci.* **93**: 12786–12790.

Kaneko, T., Sato, S., Kotani, H., Tanaka, A., Asamizu, E., Nakamura, Y., Miyajima, N., Hirosawa, M., Sugiura, M., Sasamoto, S., et al. 1996. Sequence analysis of the genome of the unicellular cyanobacterium *Synechocystis* sp. strain PCC6803. II. Sequence determination of the entire genome and assignment of potential protein-coding regions. *DNA Res.* **3**: 109–136.

Labarre, J., Thuriaux, P., and Chauvat, F. 1987. Genetic analysis of amino acid transport in the facultatively heterotrophic cyanobacterium *Synechocystis* sp. strain 6803. *J. Bacteriol.* **169**: 4668–4673.

Mackinney, G. 1941. Absorption of light by chlorophyll solutions. *J. Biol. Chem.* **140**: 315–322.

Montesinos, M.L., Herrero, A., and Flores, E. 1995. Amino acid transport systems required for diazotrophic growth in the cyanobacterium *Anabaena* sp. strain PCC 7120. *J. Bacteriol.* **177**: 3150–3157.

———. 1997. Amino acid transport in taxonomically diverse cyanobacteria and identification of two genes encoding elements of a neutral amino acid permease putatively involved in

recapture of leaked hydrophobic amino acids. *J. Bacteriol.* **179**: 853–862.

Paulsen, I.T., Nguyen, L., Sliwinski, M.K., Rabus, R., and Saier, M.H., Jr. 2000. Microbial genome analyses: Comparative transport capabilities in eighteen prokaryotes. *J. Mol. Biol.* **301**: 75–100.

Quintero, M.J., Muro-Pastor, A.M., Herrero, A., and Flores, E. 2000. Arginine catabolism in the cyanobacterium *Synechocystis* sp. strain PCC 6803 involves the urea cycle and arginase pathway. *J. Bacteriol.* **182**: 1008–1015.

Rabus, R., Jack, D.L., Kelly, D.J., and Saier, M.H., Jr. 1999. TRAP transporters: An ancient family of extracytoplasmic solute-receptor-dependent secondary active transporters. *Microbiology* **145**: 3431–3445.

Saier, M.H., Jr. 1998. Molecular phylogeny as a basis for the classification of transport proteins from Bacteria, Archaea and Eukarya. *Adv. Microbial Physiol.* **40**: 81–136.

———. 2000. A functional-phylogenetic classification system for transmembrane solute transporters. *Microbiol. Mol. Biol. Rev.* **64**: 354–411

Sambrook, J., Fritsch, E.F., and Maniatis, T. 1989. Molecular cloning: A laboratory manual. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY.

Tomii, K. and Kanehisa, M. 1998. A comparative analysis of ABC transporters in complete microbial genomes. *Genome Res.* **8**: 1048–1059.

Received May 17, 2001; accepted in revised form September 10, 2001.

Downloaded from genome.cshlp.org on June 7, 2015 - Published by Cold Spring Harbor Laboratory Press

![CSH PRESS Logo](https://example.com/csh_press_logo.png) GENOME  
RESEARCH  

Identification of Genes Encoding Amino Acid Permeases by  
Inactivation of Selected ORFs from the Synecocystis Genomic  
Sequence  

Mari?a José Quintero, Mari?a Luz Montesinos, Antonia Herrero, et al.  

Genome Res. 2001 11: 2034-2040  
Access the most recent version at doi:10.1101/gr.196301  

---

**References**  
This article cites 17 articles, 14 of which can be accessed free at:  
[http://genome.cshlp.org/content/11/12/2034.full.html#ref-list-1](http://genome.cshlp.org/content/11/12/2034.full.html#ref-list-1)

**Creative Commons License**  
This article is distributed exclusively by Cold Spring Harbor Laboratory Press for the first six months after the full-issue publication date (see [http://genome.cshlp.org/site/misc/terms.xhtml](http://genome.cshlp.org/site/misc/terms.xhtml)). After six months, it is available under a Creative Commons License (Attribution-NonCommercial 3.0 Unported License), as described at [http://creativecommons.org/licenses/by-nc/3.0/](http://creativecommons.org/licenses/by-nc/3.0/).

**Email Alerting Service**  
Receive free email alerts when new articles cite this article - sign up in the box at the top right corner of the article or [click here](http://www.example.com).

---

**Gene Link™**  
All Modifications and  
Oligo Types Synthesized  
Long Oligos • Fluorescent • Chimeric • DNA • RNA • Antisense  
*Your wish is our command.*

---

To subscribe to Genome Research go to:  
[http://genome.cshlp.org/subscriptions](http://genome.cshlp.org/subscriptions)

---

Cold Spring Harbor Laboratory Press
