
Chem. Rev. 2001, 101, 2449–2476

MAP Kinases

Zhu Chen,† Tara Beers Gibson, Fred Robinson, Loraine Silvestro, Gray Pearson, Bing-e Xu, Angeline Wright, Colleen Vanderbilt, and Melanie H. Cobb*

Department of Pharmacology, University of Texas Southwestern Medical Center, 5323 Harry Hines Boulevard, Dallas, Texas 75390

Received March 26, 2001

Contents

I. Introduction                                                                                       2449
II. ERK1 and ERK2                                                                                     2450
   A. Identification                                                                                  2450
   B. Structure and Activation Mechanism                                                              2451
   C. MEKs (Also Known as MAP Kinase Kinases or MKKs)                                                 2452
   D. Raf Isoforms and Small G Proteins                                                               2453
   E. Activation from the Cell Surface                                                                2454
   F. Regulation by Nuclear Hormone Receptors                                                        2454
   G. Protein Complexes                                                                               2455
   H. Regulation of Subcellular Localization                                                          2456
   I. Pathway Inhibitors                                                                              2457
   J. Substrates and Targeting Motifs.                                                                2457
      1. Substrate Specificity and Targeting Motifs                                                   2457
      2. Membrane Proteins                                                                            2458
      3. Cytosolic, Cytoskeletal, and Mitochondrial Proteins                                          2458
      4. Nuclear Proteins                                                                             2459
      5. Targets of Downstream Protein Kinases                                                       2459
K. Disruption of Genes in the Cascade                                                                 2459
   1. Raf-1                                                                                           2460
   2. B-Raf                                                                                           2460
   3. Raf-1 and B-Raf                                                                                 2460
   4. A-Raf                                                                                           2460
   5. MEK1                                                                                            2461
   6. ERK1                                                                                            2461
III. JNK/SAPKs                                                                                        2461
   A. Identification                                                                                  2461
   B. Structure                                                                                       2461
   C. MEKs                                                                                            2461
   D. MEKKs and Small G Proteins                                                                      2462
   E. Activation from the Cell Surface                                                                2462
   F. Interactions with Nuclear Receptor Signaling Pathways                                          2462
   G. Protein Complexes                                                                               2463
   H. Regulation of Subcellular Localization                                                          2463
   I. Pathway Inhibitors                                                                              2463
   J. Substrates and Targeting Domains                                                                2464
      1. Specificity and Targeting Domains                                                            2464
      2. Membrane Substrates                                                                          2464
      3. Cytoplasmic and Cytoskeletal Substrates                                                      2464
      4. Nuclear Substrates                                                                           2464

K. Functions Deduced from Gene Disruption                                                             2465
   1. MEKK1                                                                                           2465
   2. MKK4/SEK1                                                                                       2465
   3. JNK1                                                                                            2465
   4. JNK2                                                                                            2465
   5. JNK3                                                                                            2465
   6. JNK Double Knockouts                                                                            2465
IV. p38 MAP Kinases                                                                                   2466
   A. Identification                                                                                  2466
   B. Structure                                                                                       2466
   C. MEKs                                                                                            2466
   D. MEKKs and Upstream Regulators                                                                   2466
   E. Activation from the Cell Surface                                                                2467
   F. Subcellular Localization                                                                        2468
   G. p38 Inhibitors                                                                                  2468
   H. Substrates                                                                                      2468
      1. Cytoskeletal and Cytosolic Proteins                                                          2468
      2. Nuclear Substrates                                                                           2468
      3. Substrates of Downstream Protein Kinases                                                    2468
      4. Gene Induction and mRNA Stabilization                                                        2468
I. Functions Deduced from Gene Disruption                                                             2469
   1. MKK3                                                                                            2469
   2. p38α                                                                                            2469
V. ERK5                                                                                              2469
   A. Identification                                                                                  2469
   B. MEKs                                                                                            2469
   C. Activation and Upstream Regulators                                                              2469
   D. Pathway Inhibitors                                                                              2469
   E. Substrates                                                                                      2470
   F. Disruption of a Gene Encoding a Probable Pathway Component                                      2470
   1. MEKK3                                                                                           2470
VI. Orphan MAP Kinases                                                                                2470
   A. ERK3 Isoforms                                                                                   2470
   B. ERK7                                                                                            2470
   C. NLK                                                                                             2470
   D. MOK                                                                                             2470
VII. Acknowledgments                                                                                  2470
VIII. References                                                                                      2470

I. Introduction

MAP kinases control many cellular events from complex programs, such as embryogenesis, cell differentiation, cell proliferation, and cell death, to short-term changes required for homeostasis and

*To whom correspondence should be addressed. Phone: (214) 648-3627. Fax: (214) 648-3811.
† Current address: Rockefeller University.

10.1021/cr000241p CCC: $36.00 © 2001 American Chemical Society
Published on Web 07/19/2001

Zhu Chen received her B.S. degree from Beijing University, P. R. China in Biochemistry and Molecular Biology. She received her Ph.D. degree from The University of Texas Southwestern Medical Center at Dallas. She is currently a postdoctoral associate in Dr. Jeffrey Friedman's laboratory at The Rockefeller University. No photograph available.

Tara Beers Gibson received her B.A. degree in Biological Chemistry at Wellesley College and her Ph.D. degree from The University of Pennsylvania in Biochemistry and Molecular/Biophysics. She is currently a postdoctoral fellow in the Department of Pharmacology at The University of Texas Southwestern Medical Center at Dallas. No photograph available.

Fred L. Robinson is a Ph.D. candidate in the Biological Chemistry Graduate Program at The University of Texas Southwestern Medical Center at Dallas. He received his bachelor’s degree from Oregon State University. No photograph available.

Loraine Silvestro is a postdoctoral fellow in the laboratory of Melanie Cobb at UT Southwestern Medical Center at Dallas. She earned a bachelor’s degree in Chemistry from the State University of New York at Buffalo and received her Ph.D. degree from the University of Pennsylvania in 2000. No photograph available.

Gray Pearson completed his undergraduate work at Cornell University in Ithaca, NY, where he performed research in the laboratory of Richard Cerione. He is currently a graduate student in the Department of Pharmacology at UT Southwestern Medical Center in Dallas. He works on various aspects of MAP kinase signaling. No photograph available.

Bing-e Xu received his B.S. degree majoring in Genetics and Genetic Engineering from Fudan University, China. He received his Ph.D. degree from the Department of Microbiology and Molecular Genetics, University of Vermont. He is now a postdoctoral fellow at UT Southwestern Medical Center at Dallas. No photograph available.

Angelique Wright is a third-year graduate student working with Melanie Cobb. She is currently investigating the mechanism and regulation of ERK import into the nucleus. She graduated from Virginia Polytechnic Institute and State University in 1997 with her B.S. degree in Biochemistry and a B.A. degree in Chemistry. No photograph available.

Colleen Vanderbilt received her B.S. degree in Microbiology at Kansas State University. She is currently a Research Associate in the Department of Pharmacology in the laboratory of Melanie Cobb at UT Southwestern Medical Center at Dallas. No photograph available.

Melanie Cobb received her undergraduate degree in Biochemistry from the University of Chicago and her Ph.D. degree in Biological Chemistry from Washington University in St. Louis in the laboratory of Garland Marshall. Following postdoctoral work with Ora Rosen at the Albert Einstein College of Medicine in New York, she joined the Department of Pharmacology at UT Southwestern Medical Center in Dallas, where she is currently a Professor and the holder of the Jane and Bill Browning, Jr. Chair in Medical Sciences. Her interests are in cellular regulatory mechanisms. No photograph available.

acute hormonal responses.¹ The C. elegans genome sequence contains 15 MAP kinase-related genes.² Five of these genes encode obvious homologues of the known mammalian enzymes; the others resemble MAP kinases but may not be functionally similar to the characterized mammalian kinases. cDNAs encoding nearly 20 MAP kinases have been found in mammals. The increased complexity of mammalian genomes relative to that of worm suggests that this number accounts for fewer than one-half of the MAP kinases in mammals. These statements are all based on the assignment of a kinase to the MAP kinase

subfamily based on its overall sequence identity (Table 1). Some of these related kinases, although as similar by sequence, lack the dual-phosphorylation motif (Thr-Xxx-Tyr) that has been considered a hallmark of the MAP kinases. The sequence relationships among these kinases and some of their upstream regulators have been analyzed, suggesting the manner in which some of these pathways may have arisen.³⁻⁵

Although each MAP kinase has unique characteristics, a number of features are shared by the MAP kinase pathways studied to date. MAP kinases are activated by protein kinase cascades that contain at least two upstream kinases (Figure 1). The kinases immediately upstream are members of the MAP kinase/ERK kinase or MEK family. MAP kinases require tyrosine and threonine phosphorylation, both catalyzed by MEKs, to become highly active.⁶,⁷ As a consequence, these kinases are inactivated by all three major groups of protein phosphatases: those removing phosphate from serine/threonine or from tyrosine and the dual-specificity phosphatases which remove phosphate from both.¹,⁸ A major subgroup of the dual-specificity phosphatases is known as MAP kinase phosphatases or MKPs because it specifically dephosphorylates members of the MAP kinase family (for an extensive review see ref 9 and an article by Guan and co-workers in this issue). MAP kinases phosphorylate serine or threonine residues most often followed by proline residues.¹ Substrates usually contain substrate-interaction or docking domains that are required for high-affinity interactions with the enzymes.¹⁰ Finally, MAP kinase cascades are organized in complexes by scaffolding or linker proteins, the prototype being Ste5p identified in the pheromone-induced mating pathway of budding yeast, that contribute to their specificity and localization.¹¹⁻¹⁵

This review contains an overview of the regulation and functions of mammalian MAP kinases including literature through early fall of 2000. Topics presented include control of localization, protein complexes, interactions with nuclear receptors, newly identified substrates, and gene disruptions.

## II. ERK1 and ERK2

### A. Identification

ERK1 and ERK2, 43 and 41 kDa, are 83% identical, with most differences outside the kinase core (Table 1, Figures 2 and 3.)¹⁶,¹⁷ They are expressed to varying extents in all tissues, including terminally differentiated cells, and have been estimated to be present in the range of 100–500 nM in tissues of greatest abundance. In fibroblasts, they are activated strongly by growth factors, serum, and phorbol esters and also to a lesser degree by ligands for heterotrimeric G protein-coupled receptors, cytokines, transforming growth factors, osmotic stress, and microtubule disorganization.¹ In differentiated cells they are often activated by the primary stimuli that regulate tissue-specific functions, e.g., glucose in islets, transmitters in brain, and secretagogues in endocrine tissues.¹⁸⁻²²

MAP Kinases

Table 1. Mammalian MAP Kinases

| MAP kinase | other names | P site motif | sequence identity to ERK2<sup>a</sup> in % | sequence identity listed relative in % | ref |
|------------|-------------|--------------|---------------------------------------|----------------------------------------|-----|
| ERK1<sup>b</sup> | p44MAPK | TEY | 88 | — | 16 |
| ERK2<sup>b</sup> | p42MAPK | TEY | 100 | — | 17 |
| ERK3α<sup>b</sup> | p63, rat ERK3 | SEG | 43 | ERK3β, 74 | 17 |
| ERK3β<sup>c</sup> | human ERK3 | SEG | 42 | ERK3α, 74 | 468 |
| JNK1<sup>b</sup> | SAPKy | TPY | 40 | JNK2, 87 | 223–225 |
| JNK2<sup>b</sup> | SAPKa | TPY | 41 | — | 223–225 |
| JNK3<sup>b</sup> | SAPKβ | TPY | 40 | JNK2, 88 | 223–225 |
| p38α<sup>d</sup> | p38; CSBP | TGY | 50 | — | 322–324 |
| p38β<sup>c</sup> | p38-2 | TGY | 47 | p38α, 75 | 325, 326, 331 |
| p38γ<sup>c</sup> | ERK6; SAPK3 | TGY | 44 | p38α, 62 | 327, 328 |
| p38δ<sup>d</sup> | SAPK4 | TGY | 42 | p38α, 64 | 329–331 |
| ERK5<sup>b</sup> | BMK1 | TEY | 51 | — | 445, 446 |
| ERK7<sup>b</sup> | — | TEY | 41 | — | 459 |

relative

| NLK | Nemo-like kinase | TQE<sup>e</sup> | 46 | — | 461 |
| MAK<sup>b</sup> | Male germ cell-associated kinase | TDY | 37 | — | 466 |
| MRK<sup>b</sup> | MAK-related kinase | TDY | 37 | — | 467 |
| MOK<sup>c</sup> | — | TEY | 30 | — | 464 |
| KKIALRE<sup>c</sup> | — | TDY | 32 | — | 469 |

<sup>a</sup> Identity comparing the catalytic core only, residues 16–350 of ERK2. <sup>b</sup> Rat. <sup>c</sup> Human. <sup>d</sup> Mouse. <sup>e</sup> Similar to CDKs.

---

A MAP kinase cascade

| Ras | Rac,? | Rac,? | Ras,? |
|-----|-------|-------|-------|
| MEKK | MEKK1-4 | TAK, ASK, TAO | MEKK2,3 |
| ↓   | ↓     | ↓     | ↓     |
| Raf | MEK1/2 | MEK4/7 | MEK3,6 |
| ↓   | ↓     | ↓     | ↓     |
| MEK | MEK1/2 | MEK4/7 | MEK3,6 |
| ↓   | ↓     | ↓     | ↓     |
| MAPK | ERK1/2 | JNK/SAPK | p38 |
| ↓   | ↓     | ↓     | ↓     |
| Rsk, Mnk, | Jun, TCFs, | Mnk, Msk | TCFs, |
| Msk, TCFs | ATFs, IRS1, | MAPKAPs, | MEF2s |
|         | Bcl-2 | TCFs, ATFs, |       |
|         |       | MEF2s |       |

Figure 1. MAP kinase cascades. Enzyme cascades shown are described in the text. A few substrates of each pathway are noted. Abbreviations: ERK, extracellular signal-regulated kinase; MEK, MAP/ERK kinase; MEKK, MEK kinase; TCF, ternary complex factor; ATF, activating transcription factor; MEF, myocyte-enhancing factor; IRS1, insulin receptor substrate 1; Rsk, ribosomal S6 kinase; Mnk, MAP kinase-interacting kinase; Msk, mitogen-and stress-activated protein kinase; MAPKAP, MAP kinase-activated protein kinase.

B. Structure and Activation Mechanism

The three-dimensional structures have been solved for the low-activity, unphosphorylated form and high-activity, phosphorylated form of ERK2.<sup>23,24</sup> Like other protein kinases, ERK2 has two domains (Figure 2). The N-terminal domain is composed primarily of β strands along with two helices, α helix C and α helix L16 contributed by a C-terminal extension to the catalytic core. The C-terminal domain consists primarily of α helices with four short β strands that contain several residues involved in catalysis. A flexible linker allows the domains to rotate apart to release substrates or together to close the active site. The active site is at the domain interface.<sup>23,24</sup> A surface loop, L12, at the exterior of the active site controls the conformational transitions of the protein kinase. L12 is referred to as the activation loop or phosphorylation lip because it contains the activating sites of phosphorylation. The two phosphoacceptor sites, Y185 and T183 in ERK2, are in this loop and must both be phosphorylated to activate the ki-

nases.<sup>6,7</sup> Protein substrates contact the surface of the α-helical domain as well as parts of L12 in the immediate vicinity of the tyrosine phosphorylation site. L12 is shorter in several of the other MAP kinases and is in a different conformation in each of the MAP kinase structures presently available. In unphosphorylated ERK2, L12 partially blocks the binding site for protein substrates. This appears to be a common feature in the function of L12 in MAP kinases. In ERK2, L12 also contacts the MAP kinase insert, an α-helical insert present in MAP kinases and the cyclin-dependent kinases (CDKs).

Phosphorylation increases ERK2 activity substantially, yielding a turnover number of 40–100 min<sup>-1</sup> with myelin basic protein or the peptide ERKtide as substrate and a 50,000-fold increase in k<sub>cat</sub>.<sup>25</sup> ERK1/2 are phosphorylated on tyrosine before threonine in vitro and in cells.<sup>26,27</sup> Phosphorylation is nonprocessive in that there are apparently two separate enzyme-substrate interactions required.<sup>28,29</sup> Once ERK2 is tyrosine-phosphorylated, threonine phos-

the basis of both the structure and mutagenesis experiments, phosphotyrosine 185 appears to induce a rearrangement of L12. This residue binds to two basic residues in a pocket on the exterior of the C-terminal domain of the kinase core, relieving the interference at the protein substrate binding site. As a consequence of the refolding of L12, Y185 forms part of the surface to bind the proline at the P + 1 residue of the protein substrate. Phosphothreonine 183, like the stable phosphothreonine residue in L12 of cyclic AMP-dependent protein kinase (PKA), aids in orienting active site residues in the N-terminal domain in part through interactions with an arginine residue in helix C. Given the nature of these changes, it is not surprising that incorporation of acidic residues in place of the two ERK2 phosphorylation sites does not constitutively activate ERK2 or other MAP kinases.

### C. MEKs (Also Known as MAP Kinase Kinases or MKKs)

MEK1 and MEK2, the first identified members of this dual-specificity family of protein kinases, activate ERK1/2 in vitro and in transfected cells. Like MAP kinases, MEKs are activated by over 1000-fold by phosphorylation of two residues. However, both phosphorylations are on either serine or threonine, and each of the two will partially increase the activity of MEK1/2. Unlike the MAP kinases, these enzymes have proven amenable to activation by mutagenesis, which has been examined in some detail. Both replacement of the two sites of phosphorylation with acidic residues (often called DD, EE, or DE mutants) and deletions in the N-terminus yield proteins with significantly elevated activity. Because the MEKs, but not the ERKs, have been activated by these acidic substitutions, MEK mutants have been used extensively to probe the functions of ERK1/2. In general, MEK family members are among the most selective protein kinases known in that they phosphorylate very few substrates. MEK1/2 have not been shown to phosphorylate any other MAP kinases or other proteins of physiologic relevance. However, another MEK family member, MEK4, has been shown to phosphorylate at least one protein other than MAP kinases, suggesting that other MEKs may also have a few non-MAP kinase substrates.

MEK1 and MEK2 also contain three nonenzymatic domains that are important for their functions: an ERK1/2 binding site, a proline-rich domain, and a nuclear export sequence (NES; discussed further in section II.H; see Figure 3). MEKs, like a number of other protein kinases, have been shown to form complexes with substrates through binding sites that are distinct from the active site. The impact of these docking sites on the formation of Michaelis complexes has not been thoroughly explored but is believed to facilitate the association of the inactive forms of MEKs with their MAP kinase substrates. The ERK1/2 binding site is located near the MEK N-terminus (Figure 3). This binding site is known as a docking or D domain and is also present in MAP kinase substrates and other proteins as described later. The binding site on MEK1/2 contains basic and

**Figure 2.** Three-dimensional structure of ERK2. The active site of protein kinases are formed at the interface of the two folding domains. ATP is represented in the ERK2 active site. Phosphorylation sites (Y185 and T183) within the activation loop, the MAP kinase insert, the common docking or CD domain, α helix C, and the C-terminal extension, L16, are indicated.

**Figure 3.** Schematic diagrams of Raf-1, MEK1, and ERK2. (Top) Raf-1. In Raf-1, the kinase domain and the two cysteine-rich domains (CR1 and CR2) are shown. Several phosphorylation sites are indicated by P. Serines 259 and 621, which are 14–3–3 binding sites, are indicated with the number sign (#). The serine/tyrosine cluster, residues 338–341, which have been shown to activate Raf-1 when phosphorylated, are indicated by an asterisk (*). Additional phosphorylation sites are indicated in the kinase activation loop. (Middle) MEK1. The docking or D domain, nuclear export sequence (NES), proline-rich domain, and kinase domain are indicated. Activating phosphorylation sites are in the activation loop, and modulatory sites are in the proline-rich domain. (Bottom) ERK2. The kinase domain is shown. The common docking or CD domain and the C-terminal extension, L16, are also shown.

MAP Kinases

A

![Diagram A](#)

B

![Diagram B](#)

Figure 4. Ras activation complexes. (A) MP1 links ERK1 and MEK1 to enhance their activation by Raf-1. (B) KSR also promotes activation of the ERK1/2 cascade through its interactions with the kinase components of the cascade.

was isolated in a binding screen using the tyrosine-phosphorylated, C-terminus of the EGF receptor. Grb10 may target MEK1 and the ERK1/2 cascade to mitochondria. It has been proposed that from this location it may allow transmission of ERK1/2-dependent cell survival signals.⁵⁷

D. Raf Isoforms and Small G Proteins

Kinases that phosphorylate MEKs are known as MEKKs. Several will phosphorylate MEK1/2 in vitro. However, outside of oocytes, where Mos is the relevant MEKK for the ERK1/2 pathway, Raf isoforms appear to be the major or perhaps only catalysts of MEK1/2 activation.⁵⁸⁻⁶⁰ There are three isoforms: A-Raf, B-Raf, and Raf-1. Each is composed of three domains (Figure 3): the kinase domain at the C-terminus and the two N-terminal regulatory domains. An activated mutant of Raf, RafBXB, lacks the majority of these regulatory domains.⁶¹

Because Raf-1 is widely distributed in the body and expressed in most cultured cells, it has received the most attention. The other isoforms are more restricted in expression, with B-Raf expressed particularly in neuronal tissues. Regulation of the MEKK level enzymes is generally more complex than that of MEKs and MAP kinases. Two major events, binding to GTP-liganded Ras and phosphorylation, activate Raf-1. Under most circumstances, association with Ras is required for activation.⁶²⁻⁶⁶

Phosphorylation may occur on several residues.⁶⁶⁻⁷³ A cluster of residues, Ser338, Ser339, Tyr340, and Tyr341, N-terminal to the kinase domain of Raf-1 appear to be most important for activation. One or more of these sites are phosphorylated by Src, the p21-activated protein kinase PAK, and, most likely, other kinases. It is not yet clear if activity increases in proportion to the number of sites that are phosphorylated nor if Ras is absolutely required in all cases, although it clearly is in some.

Two additional sites, Ser259 and Ser621, are thought to regulate association of Raf-1 with 14-3-3.⁶⁷,⁷⁴⁻⁸⁴ 14-3-3 proteins bind certain sequences containing phosphoserine in a number of proteins and may sequester the proteins or aid in formation of appropriate complexes. Mutation of Ser621 destroys Raf activity, leading to the conclusion that binding of 14-3-3 to this site stabilizes Raf activity. Mutation of Ser259, on the other hand, greatly increases Raf activity, suggesting that 14-3-3 may also suppress Raf activity directly or indirectly. This residue has been proposed as a site for phosphorylation by Akt.⁸⁵⁻⁸⁷ Multiple Akt sites have been found on B-Raf.⁸⁷ Raf also forms complexes with hsp90 and p50. Their significance in Raf function has been inferred in part from the actions of geldanamycin, which disrupts their interaction with Raf.⁸¹,⁸²,⁸⁸

There are differences in regulation of Raf-1 and B-Raf, particularly due to apparent distinctions in effects of the small G proteins Ras and Rap1a.⁸⁹⁻⁹³ While Raf-1 seems to be responsive to H-, K-, and N-Ras, B-Raf is also activated by Rap1a.⁸⁹,⁹²,⁹⁴ This has been studied most thoroughly in PC12 cells. Another Ras family member, TC21, may interact with Raf-1 and B-Raf, apparently activating them.⁹⁵

The first experiments demonstrating interaction of Ras and Raf also showed that MEK1 was in the same complex. The domains of Raf and MEK that interact have not yet been identified; however, a mutation in the activation loop of Raf-1 has been found that ablates its ability to bind MEK1 by two-hybrid analysis. In addition, changes in the MEK1 proline-rich insert influence Raf association, although the insert does not directly bind to Raf. The stable association of Raf-1 with MEK appears to be another essential interaction in the ERK1/2 cascade. Raf-1 also binds to the Raf-1 kinase-inhibitory protein (RKIP), which interferes with Raf-MEK association and blocks MEK phosphorylation and activation by Raf.

### E. Activation from the Cell Surface

Cell surface receptors transmit activating signals to MAP kinases through a variety of mechanisms. Two examples are the pathways involved in tyrosine kinase receptor activation by growth factors and activation by G protein-coupled receptor (GPCR) signaling. Ligand-bound receptor tyrosine kinases autophosphorylate. Adaptors which contain modular protein-protein interaction domains, such as Shc and Grb2, then mediate the formation of multiprotein complexes through two types of protein-protein interactions. The first consists of interactions between the SH2 domains of the modular proteins and phosphotyrosine motifs on the receptors or their substrates. The second is the recognition of proline-rich motifs on the modular proteins by SH3 domains located on son of sevenless (Sos) and other Ras-activating guanine nucleotide exchange factors (GEFs). These GEFs stimulate Ras to exchange GDP for GTP, which allows Ras to interact not only with Raf to activate the kinase cascade, but with a wide range of other downstream effectors as well.

GPCRs also stimulate ERKs through increasing the activity of a Raf isoform; however, they do so by a more diverse array of mechanisms (Figure 5). The mechanism appears to depend on the class of G protein, cell type, and expression of effector molecules. The mechanism utilized by Gs-coupled receptors has been particularly difficult to discern. GTP-liganded Gas engages adenylyl cyclase to increase the production of cAMP. However, increasing cAMP concentration may increase, decrease, or have no effect on ERK1/2 activity depending on cell type. cAMP activates PKA, which can apparently reduce Raf-1 activity through direct phosphorylation. However, in neuronal cells, PKA may also phosphorylate and activate the small G protein, Rap1a, which can increase ERK1/2 activity through the activation of B-Raf. In addition, cAMP can influence Rap1a activity through the activation of a cAMP-binding Rap1a GEF. Gs activation of ERK1/2 has also been proposed to involve Src. In one model, Gs activation of Src occurs through a PKA-dependent switching of receptor coupling from Gs to Gi. Alternatively, Gas or Gai but not other Ga subunits can directly interact with Src, causing an increase in Src activity toward substrate in vitro and in cells (Figure 5).

---

**Figure 5.** Activation of ERK1/2 by GPCRs through $\beta\gamma$ subunits of heterotrimeric G proteins. (A) One model supports the role of PI-3 kinase and Src, along with one or more other tyrosine kinases (see text). (B) A second model is based on direct activation of Src by the $\alpha$ subunits of Gs or Gi (see text).

There is also evidence that free $\beta\gamma$ subunits resulting from G$\alpha$i activation are active signal transducers involved in ERK1/2 activation (Figure 5). In contrast to the direct mechanism shown for G$\alpha$s and G$\alpha$i activation of Src, it is believed that G-$\beta\gamma$ subunit activation of Src involves phosphatidylinositol-3 kinase (PI-3 kinase). Activated Src then phosphorylates receptor tyrosine kinases, PYK2 or FAK to stimulate the recruitment of Ras activating complexes to the membrane in an analogous mechanism to that just described for growth factor receptors.

Active G$\alpha$q subunits directly interact with PLC$\beta$ to stimulate the production of diacylglycerol (DAG) and IP$_3$. DAG activates the several PKC isoforms in a calcium-dependent or independent manner, thereby increasing their activity. PKC stimulation of ERK1/2 has been observed in a number of contexts and may involve the direct phosphorylation of Raf-1, although the sites of phosphorylation are still in dispute. Additional G$\alpha$q mechanisms have also been described; however, it is not clear if they act in concert with or instead of PKC. GTPase-deficient mutants of G$\alpha$12 and G$\alpha$13 can inhibit ERK activation in some situations. These mutants can also increase the activities of other MAP kinases, the c-Jun N-terminal kinase/stress-activated protein kinase (JNK/SAPK), and p38, possibly through the small G protein Rac.

### F. Regulation by Nuclear Hormone Receptors

Some nuclear receptors regulate the ERK1/2 MAP kinase pathway through direct interactions with signaling elements upstream in the Ras activation pathway; the mechanism shares components and bears a strong resemblance to that used by growth factor receptors to control ERK1/2. A possible model is shown in Figure 6. Estradiol treatment of human mammary cancer MCF-7 cells or Cos cells transiently transfected with human wild-type estradiol receptor $\alpha$ (ER$\alpha$) caused rapid and transient tyrosine phos-

MAP Kinases

Chemical Reviews, 2001, Vol. 101, No. 8 2455

![Diagram](https://i.imgur.com/yourimageurl.png)

Figure 6. Regulation of ERK2 by nuclear receptors. Nuclear receptors may regulate ERK1/2 through complexes composed of the estrogen receptor, Src and Shc.

phorylation of Shc, accompanied by activation of p21ras, ERK1, and ERK2. ${ }^{114}$

Estradiol stimulated progression of NIH 3T3 cells to S-phase of the cell cycle equally well if cells were transiently transfected with a transcriptionally inactive mutant ERα or the wild-type ERα, supporting the idea that a nontranscriptional mechanism was involved. ${ }^{115}$ Microinjection of cells with various antibodies or inhibitors of the MAP kinase pathway inhibited the proliferative response to progesterin or estrogen.

Interestingly, progestins activate the ERK1/2 cascade in a manner that requires binding to the progesterin receptor B (PRB) but also requires ERα and can be blocked by the anti-estrogen ICI 182,780 as well as the anti-progestin RU486. ${ }^{116}$ Immunoprecipitation of PR also brought down ER, both before and after progesterin or estrogen treatment of cells. Anti-ER antibodies co-immunoprecipitated PR and c-Src only after progesterin or estrogen treatment. PR did not associate with c-Src in the absence of ER, providing evidence that the ER interacts with Src.

Improta-Brears et al. showed that estradiol treatment of MCF-7 cells led to a rapid activation of MAP kinase that was coincident with mobilization of intracellular calcium stores but did not result in activation of Raf-1. ${ }^{117}$ In other work, explants of rat cerebral cortex, activation of the MAP kinase pathway by estradiol was blocked by the MEK1 inhibitor PD98059 but not by the ER antagonists ICI 182,780 and ICI 164,384. ${ }^{118}$ In these cells, the ER co-immunoprecipitated with hsp90 and BRaf but not MEK1. Other researchers have observed rapid activation of MAP kinase pathways by estrogen but have yet to demonstrate any interaction between the estrogen receptors and proteins of the MAP kinase cascade. ${ }^{119-124}$

Davis et al. demonstrated in 293T cells that treatment with thyroxine (T4) increased translocation of its receptor TRβ1 to the nucleus and resulted in phosphorylation of ERK1 and ERK2. ${ }^{125}$ ERK1/2 and TRβ1 were co-immunoprecipitated from the nucleus after cells were stimulated with T4. This association was blocked by the MEK inhibitor PD98059 and the T4 inhibitors tetrac and triac. T4 activated TRβ1 by ERK1/2-mediated serine phosphorylation (in vitro) and dissociation of the co-repressor, SMRT (cells). The region of TRβ1 that binds to ERK1/2 was the second zinc finger domain; the complementary region of ERK1/2 was not examined.

Song et al. were able to separate rapid activation of ERK2 and genomic responses to 1α,25-dihydroxyvitamin D3 in a human acute promyelocytic leukemia cell line (NB4) by using constrained analogues of 1α,25-dihydroxyvitamin D3. ${ }^{126}$ As these analogues had a low affinity for the 1α,25-dihydroxyvitamin D3 receptor, the authors pursued explanations other than direct interaction with MAP kinase proteins (for an issue devoted to rapid responses of steroid hormones, see ref 127).

### G. Protein Complexes

As suggested earlier, the formation of complexes in the ERK1/2 cascade is essential for signal transmission and function of the pathway. However, some of the proteins that mediate formation of complexes have only recently been identified, and their importance to cascade function has only been evaluated in a few contexts. Furthermore, it seems likely at this time that there are several distinct complexes that may lead to ERK1/2 activation. Perhaps these complexes mediate responses to different ligands and are directed to distinct sites of action in cells. Kinase suppressor of Ras (KSR), Sur-8, and connector enhancer of KSR (CNK) have been implicated in Ras activation of the ERK1/2 pathway through genetic exploration in the fruit fly and the nematode; their functions in mammals are being probed as homologues have been identified.

KSR was discovered in the two pathways that have been most often employed to delineate the growth factor receptor/Ras pathway as it controls cell fate: the photoreceptor system in *Drosophila* and the vulval induction pathway in *C. elegans*. A screen for mutants with impaired signaling between Ras and Raf led to the discovery of KSR in each system. ${ }^{128-130}$ The mechanism of action of KSR was initially mysterious. KSR contains a protein kinase-like domain, with all conserved residues except the catalytic lysine normally found in subdomain II of the protein kinases (equivalent to K52 of ERK2). Interestingly, its domain organization most resembles Raf in that it has an N-terminal cysteine-rich region and a C-terminal kinase domain. Although the question of whether KSR is actually a protein kinase has not been unequivocally resolved, its major function appears to be to bind the kinases of the ERK1/2 MAP kinase module (Figure 4B ${ }^{131-135}$ ). Its kinase-like domain binds to both Raf-1 and MEK1. Binding to MEK1 is essential for its function. ${ }^{135}$ KSR binds ERK2 through its cysteine-rich domain. ${ }^{131-133}$ While KSR is assumed to act in the Ras pathway, KSR also binds to γ subunits of heterotrimeric G proteins, suggesting that KSR may also act in signaling by G protein-coupled receptors. ${ }^{136}$

Sur-8 was identified in a screen for suppressors of an activated Ras allele in *C. elegans* which causes a multivulval phenotype. ${ }^{137,138}$ Depressed Sur-8 function did not yield any obvious phenotype, except if there was another defect in the nematode ERK pathway. Nevertheless, overexpression of Sur-8 enhanced the Ras-induced phenotype and increased Raf activity. Overexpressed Sur-8 had no effect on the pathway if activated mutants of Raf and MEK were used to induce the phenotype. Under some circumstances, Sur-8 co-immunoprecipitated with Ras and

2456 Chemical Reviews, 2001, Vol. 101, No. 8

![Diagram](attachment:diagram.png)

Figure 7. Model for regulation of the nuclear localization of ERK2. ERK2 may be actively imported into the nucleus in a dimeric form or by diffusion. ERK2 is actively exported in a CRM1-dependent manner. Binding to MEK1 appears to be a major mechanism for cytoplasmic retention via active nuclear export. Other proteins may also anchor ERK2 in the cytoplasm and in the nucleus.

Raf; thus, Sur-8 may enhance signaling through this cascade by promoting the interaction of Ras with Raf. In contrast to KSR, it does not appear to bind to the other components of the kinase cascade.

CNK was identified in a screen for enhancers and suppressors of KSR in the fly photoreceptor system. CNK enhanced the rough eye phenotype caused by expression of a fragment of KSR and suppressed phenotypes caused by activated alleles of ras. Drosophila CNK is a large protein with several modular protein interaction domains and two SH3 binding sites. Drosophila CNK binds to the kinase domain of Raf-1; its overexpression inhibits Raf function. A possible mammalian homologue of CNK, also known as membrane-associated guanylate kinase-interacting protein (MAGUIN)-1, lacks the domain of the fly protein that binds Raf, yet it apparently retains this binding capacity through its central PH domain. Its role in the mammalian pathway is unclear.

MEKK1 is a 195 kDa protein which binds several components of MAP kinase pathways. Its relationship to the JNK/SAPK is described in section IV. MEKK1 also binds to ERK2, MEK1, and Raf-1. Because MEKK1 is strongly associated with the actin, microtubule, and intermediate filament cytoskeleton, it may serve to connect a small fraction of the ERK1/2 cascade to this cellular compartment. Gene disruption experiments suggest that MEKK1 has a limited impact on the ERK1/2 pathway.

H. Regulation of Subcellular Localization

ERK1/2 are distributed throughout cells. In unstimulated fibroblasts, the majority of these proteins appear to associate with the microtubule cytoskeleton. Stimulation of cells often causes a significant population of ERK1/2 to accumulate in the nucleus, in contrast to their relative exclusion from the nucleus in resting cells. In other cell types, ERK1/2 may be localized to membrane specializations and not enter the nucleus even when activated. Proteins that bind to ERK2 in the cytoplasm, such as the smooth muscle protein calponin, may alter its activation-induced redistribution.

Underlining the importance of localization of MAP kinases for their function, the nuclear localization of ERK1/2 is essential for morphological transformation of 3T3 fibroblasts and neurite extension in PC12 cells, a model system for neuronal differentiation. Studies of ERK in presenescent and senescent fibroblasts show that ERK1/2 become inactivated and cytoplasmic in senescent cells. Findings discussed below suggest that multiple regulated events and the relative concentrations of several proteins in cells will impact the nuclear accumulation of ERK1/2. These are represented in the model in Figure 7 and include the following: binding to proteins retained in the cytoplasm; diffusional entry into and exit from the nucleus; phosphorylation-induced dimerization; active nuclear import; nuclear export; and binding to nuclear proteins.

One of the most important proteins in determining the localization of ERK1/2 is MEK1. MEK1 is as abundant as ERK2 and is usually found in the cytoplasm due to a very active nuclear export sequence (NES). Deletion of this NES causes accumulation of MEK1 in the nucleus. In contrast to the behavior of endogenous ERK2, when ERK2 is overexpressed at a high level in fibroblasts, it is more readily detected in the nucleus as well as the cytoplasm. This is consistent with the conclusions of Ferrell and co-workers, who found that ERK2 was present at equal concentrations in both cytoplasmic and nuclear compartments of Xenopus oocytes (personal communication). Coexpression of MEK1 causes ERK2 to be excluded from the nucleus as long as cells are quiescent. This indicates that MEK1 is sufficient to exclude ERK2 from the nucleus.

MAP Kinases

Chemical Reviews, 2001, Vol. 101, No. 8 2457

sufficient to maintain the expected localization of ERK2, although this does not prove that MEK1 is exclusively responsible for ERK2 export. Leptomycin, an inhibitor of nuclear export via the export receptor CRM1, causes the accumulation of ERK2 in the nucleus, also consistent with the idea that ERK2 is exported via an NES, perhaps that of MEK1. ${}^{157,159}$ Fusion of ERK2 to MEK1 yields a protein which is excluded from the nucleus, unless the NES is inactivated by mutagenesis, also indicating the dominance of the MEK1 NES. ${}^{155}$

The nuclear uptake of MEK1 itself is also regulated, although its NES has made the regulation of its localization more difficult to analyze. MEK1 uptake was increased when cells were stimulated with growth factors. Phosphorylation or mutation of the phosphoacceptor sites to acidic residues increased nuclear uptake. Mutations that impair MEK1 kinase activity were without effect; however, blocking MEK activity pharmacologically interfered with uptake. These studies indicate that phosphorylation of MEK1 and downstream signaling induced by it are important for its nuclear accumulation. ${}^{164}$ This is consistent with the idea that active MEK1 may enter cells to activate nuclear pools of ERK2.

Other proteins may impact ERK2 localization in a similar manner. In smooth muscle calponin, which is highly expressed in muscle, is thought to retain ERK2 in the cytoplasm due to their stable association, apparently through the calponin CH1 domain. ${}^{165}$ The phosphatase PTP-SL has also been proposed to prevent entry of ERK2 into the nucleus. ${}^{166}$ Like MEK1, PTP-SL contains a D domain-like sequence which is required for interaction with ERK2. Activation of ERK2 has been shown to favor dissociation from both MEK1 and PTP-SL, thereby enhancing ERK2 nuclear accumulation. Introduction of peptides containing this D domain-related sequence from either MEK1 or PTP-SL promotes nuclear accumulation of ERK2, presumably by competing for association with these or other D domain-containing proteins. PKA phosphorylates PTP-SL within the ERK2 binding site, inhibiting its association with ERK2 and enhancing its nuclear entry. ${}^{166}$ Control of nuclear transport may not be a linear response, i.e., from upstream activators to ERK1/2, but may also involve molecules downstream in the pathway and those that are affected by multiple signaling cascades.

ERK2 forms dimers upon phosphorylation which promote its nuclear accumulation. ${}^{150}$ Although the mechanism has not been determined, ERK2 mutants that dimerize less well are less able to accumulate in the nucleus. As a monomer, ERK2 is small enough to enter and exit the nucleus by diffusion through nuclear pores. To examine non-diffusional entry, ERK2 was fused to β-galactosidase. This β-galactosidase fusion protein was unable to translocate to the nucleus in the presence of active transport inhibitors such as wheat germ agglutinin, which blocks nuclear pores, or mutant nuclear import factors, implying an active import mechanism. ${}^{158}$ An ERK2 fusion protein mutant that failed to dimerize also failed to enter the nucleus, additional evidence that dimerization is important for nuclear entry or retention. ${}^{150}$ Analysis

of the crystal structure of phosphorylated ERK2 revealed that dimerization is mediated by mostly hydrophobic interactions between the activation loop and L16.

A number of nuclear binding sites for ERK2 have been proposed that have been studied primarily in relation to the role of ERKs in cell cycle progression. These include topoisomerase IIα and components of the nuclear spindle assembly. ${}^{167}$ In several cell types, ERK1/2 are activated in the nucleus during mitosis and found on kinetochores, asters, and the midbody during mitosis. Among several possible ERK1/2 functions specific to this localization, the kinetochore motor protein CENP-E was identified as a potential ERK1/2 substrate. ${}^{168,169}$

### I. Pathway Inhibitors

Highly selective inhibitors of ERK1/2 have not been reported; however, drugs that are related to the microtubule drug taxol inhibit these kinases. ${}^{170}$ In contrast, inhibitors of MEK1/2 are in widespread use. PD98059, found in an in vitro screen for blockers of ERK activation, ${}^{171,172}$ and U0126, identified in a cell-based screen for inhibitors of activating protein (AP)-1 transactivation, ${}^{173}$ are noncompetitive inhibitors with respect to both substrates. It has been suggested that these drugs bind and stabilize the low-activity structures of MEK1/2. The use of these drugs, in addition to kinase-deficient forms of ERK1/2, has implicated ERK1/2 in the regulation of proliferation of fibroblasts and some cancer cells by growth factors and steroid hormones, control of cell motility, long-term potentiation in neurons, and circadian rhythm. ${}^{1,8,174}$ Because these drugs are noncompetitive with ATP, it has been assumed that they were among the most selective inhibitors available. ${}^{171,173}$ However, although they have no effects on most other MEK family members, they inhibit MEK5 of the ERK5 pathway. ${}^{175}$ Blockade of MEK5 occurs at concentrations just above those that block MEK1/2. A second-generation MEK inhibitor with a different chemistry (PD184352) is currently in clinical trials for treatment of cancer. ${}^{176}$

### J. Substrates and Targeting Motifs

#### 1. Substrate Specificity and Targeting Motifs

ERK1/2, like other MAP kinases, phosphorylate serine/threonine residues followed by proline residues in their substrates. ERK1/2 may also have a secondary preference for proline at the P-2 position, such that the consensus sequence is PXS/TP. Many MAP kinases form tight complexes with their substrates through docking sites within the substrates. These docking sites increase not only the specificity of substrate recognition, but also the efficiency of phosphorylation and activation of substrates by MAP kinases.

The docking or D domains found in transcription factors, such as Elk-1 and the MEF2 family, MEKs and PTPs, resemble the delta domain, which was the first domain identified that mediated the stable association of a substrate with any MAP kinase family member. ${}^{43,45,177,178}$ The typical D domain con-

2458 Chemical Reviews, 2001, Vol. 101, No. 8

consists of two or more basic residues (Lys or Arg) followed by a (L/I)X(L/I) motif. Both basic and hydrophobic residues are important for binding to ERK1/2. The D domain can lie either upstream or downstream of the phosphoacceptor site of the substrate to mediate the stable interaction with MAP kinases. The D domain is recognized by ERK1/2, JNK/SAPKs, and p38 MAP kinases, probably through interaction with their common docking (CD) domains, ${}^{43,46}$ discussed below.

Jacobs et al. identified a different docking motif FXFP in transcription factors LIN-1, SAP-1, and Elk-1. Unlike the D domain, FXFP is, thus far, reported to be recognized only by ERK1/2. ${}^{45,177,179}$ This motif is found in the sequences of substrates not only of mammalian ERK1/2, but also of the C. elegans and Drosophila ERK1/2 orthologs. Currently, it is not clear which sequences in ERK1/2 interact with the FXFP motif. Some proteins such as Elk-1 have both the D domain and the FXFP motif. These motifs can work in combination to give higher affinity for ERK1/2. D domains may play a bigger role than the FXFP motif because mutations in the D domain cause more severe effects than the those in the FXFP motif.

Several protein kinase substrates of ERK1/2 including Rsk and Mnk contain the docking motif LAQRR. ${}^{180,181}$ The motif in Rsk and Mnk2 is recognized only by ERK1/2. The LAQRR binding site on ERK1/2 is believed to be the CD domain. One variant motif LA(K/R)RR found in Mnk1 and MSK1 interacts with both ERK1/2 and p38. Another variant sequence LX(K/R)(R/K)RK, found in MSK2, PRAK, and MAP kinase-activated protein (MAPKAP) kinases 2 and 3, interacts only with p38.

The CD domain is an acidic cluster DXX(D/E) located just following the kinase domain in ERK1/2 and is conserved in all MAP kinases. ${}^{43,46}$ The acidic residues probably bind to basic residues in the D domain and other docking motifs. The CD motif in ERK2 serves as a docking site for MEK1, MKP-3, and Mnk1 and is required for efficient enzymatic reactions. The electrostatic interaction is not the only determinant of MAP kinase-substrate interaction. Two hydrophobic residues in the D domain are also important for its function.

Since ERK1/2 were first identified as kinase activities, there has been an intense search for physiological substrates. ERK1/2 target cytoplasmic proteins, membrane proteins, cytoskeletal proteins, and nuclear proteins. One subset of ERK1/2 targets are protein kinases, which include Rsk1–3, MAPKAP kinase-2 and -3, Mnk1, and Mnk2. ${}^{181-187}$ In addition to enhancing gene expression via intermediary kinases which increase the accessibility of DNA, MAP kinases also phosphorylate transcription factors directly and increase their activity. ERK1/2 phosphorylate ternary complex factors such as Elk1, c-Fos, c-Jun, and the estrogen receptor, among others. ${}^{188-192}$ Membrane protein substrates include cytosolic phospholipase A2 and the epidermal growth factor receptor. ${}^{193,194}$ Information on these ERK1/2 substrates has been reviewed elsewhere multiple times. ${}^{1}$ The rest of this section will focus on substrates identified over the last 3 years.

---

Chen et al.

### 2. Membrane Proteins

Binding of tumor necrosis factor-$\alpha$ (TNF) to its receptor CD120a (p55) leads to apoptosis and transcription of proinflammatory genes as well as activation of the ERK1/2 pathway. CD120a is phosphorylated by ERK2 on Thr236 and Ser270 within the membrane proximal region of the cytoplasmic domain. ${}^{195,196}$ When phosphorylated, the receptor moves from the plasma membrane and Golgi to the endoplasmic reticulum. Cotransfection of receptor and constitutively active MEK1 led to receptor phosphorylation in vivo, supporting the idea that this functional change is caused by ERK1/2.

Syk is a protein tyrosine kinase that functions downstream of antigen receptors such as the type 1 Fc$\alpha$ receptor on mast cells. Syk activation leads to degranulation, and phosphorylation on tyrosine and serine enhanced this activity. ERK1 formed a complex with Syk and phosphorylated it in rat mucosal type mast cells, RBL-2H3. ${}^{197,198}$ MEK inhibitors decreased the antigen-induced phosphorylation of Syk and suppressed degranulation of cells. This is one example of ERKs functioning in their own positive feedback regulation.

ERKs induce negative feedback on their activation by phosphorylation of upstream components of the kinase cascade including a number of tyrosine kinase receptors, Sos, Raf, and MEK. ${}^{1}$ Ogier-Denis et al. recently reported that ERK2 phosphorylated G alpha interacting protein or GAIP, a regulator of G protein signaling (RGS) that acts as a GTPase activating protein (GAP) for Gai3. ${}^{199}$ The likely site of phosphorylation was Ser151, which lies in the GAIP RGS domain. GAIP and Gai3 are involved in regulating lysosomal-autophagic catabolism signaling in human colon cancer HT-29 cells. The GAP activity of GAIP was stimulated by ERK2 phosphorylation and diminished when Ser151 of GAIP was mutated to alanine. This study suggests that ERK1/2 cause negative feedback of not only tyrosine kinase signaling, but also heterotrimeric G protein signaling.

Calnexin is an endoplasmic reticulum type 1 integral membrane protein phosphorylated by ERK1 in vitro at the same Ser563 site phosphorylated in vivo. ${}^{200}$ Phosphorylation increased calnexin association with membrane-bound ribosomes. ERK activity coimmunoprecipitated with calnexin in Rat-2 cells, which was prevented by transfection of dominant negative MEK1.

---

### 3. Cytosolic, Cytoskeletal, and Mitochondrial Proteins

Other signaling enzymes in addition to protein kinases are ERK1/2 targets. Phosphodiesterase (PDE) 4D3 is a cAMP-specific phosphodiesterase which degrades intracellular cAMP to terminate cAMP-dependent responses. ERK2 phosphorylated PDE4D3 on Ser579 in its C-terminus, thereby reducing its PDE activity by 75%. ${}^{201}$ Stimulation of COS-1 cells with EGF promoted the phosphorylation of PDE4D3 and the concomitant decrease in its activity. The sensitivity of this phosphorylation to blockade by a MEK inhibitor supported the conclusion that ERK2 catalyzed phosphorylation of the PDE in cells.
MAP Kinases

ERK1/2 also phosphorylate structural proteins such as neurofilament subunits. Neurofilaments (NF) are neuron-specific intermediate filaments in large myelinated axons. Their phosphorylation is vital to the regulation of NF activity. KSP repeats in the C-termini of NF-M and NF-H are phosphorylated by ERK2 in vitro.²⁰² Cotransfection of constitutively active MEK into NIH3T3 cells with NF-M resulted in NF-M phosphorylation. Stimulation of cells with EGF resulted in phosphorylation of NF-M in vivo, suggesting that ERK1/2 may be relevant NF kinases under certain circumstances.

Paxillin is a multifunctional docking protein involved in cell adhesion. In vivo labeling studies in PMA-treated EL4 thymona cells led to phosphorylation of paxillin via PKC and ERK1/2.²⁰³ Consistent with a role for ERK1/2, MEK inhibitors reduced paxillin phosphorylation in these cells. Effects of phosphorylation on paxillin function were not delineated.

The rate-limiting step in the biosynthesis of pyrimidine nucleotides is performed by carbamoyl phosphate synthetase, which is in a multifunctional enzyme complex in mitochondria. Allosteric modulation of the activity of the complex by substrates and products is regulated by MAP kinase phosphorylation.²⁰⁴ Feedback inhibition, induced by uridine triphosphate, was lost upon phosphorylation by ERKs. In addition, activation by phosphoribosyl pyrophosphate was enhanced. Both of these effects were blocked by MAP kinase inhibition. These changes in allosteric control both increase the biosynthesis of pyrimidines which supports cell proliferation.

### 4. Nuclear Proteins

Among nuclear ERK2 substrates is steroid receptor coactivator-1 (SRC-1).²⁰⁵–²⁰⁷ SRC-1 is a coactivator involved in activation of genes induced by the nuclear receptor superfamily and is ubiquitously expressed in mammalian tissues. SRC-1 interacts with cAMP-response element binding protein (CBP) to enhance both estrogen receptor- and progesterone receptor-mediated gene activation. SRC-1 has intrinsic histone acetyltransferase activity (HAT) which is involved in chromatin remodeling. SRC-1 also recruits p300/CBP-associated factor (P/CAF). Two out of seven phosphorylation sites found in SRC-1 expressed in COS-1 cells have consensus sequences for ERK1/2.²⁰⁷ Phosphorylation of SRC-1 in vitro with ERK2 followed by tryptic phosphopeptide mapping revealed phosphorylation on Thr1179 and Ser1185 and to a lesser extent Ser395. Thr1179 and Ser1185 phosphorylation sites both lie within the intrinsic HAT region of SRC-1 and may be responsible for regulating this activity. Thus far, more direct evidence for regulation of SRC-1 by ERK1/2 activity in vivo is lacking.

Both ERK2 and p38 kinases, but not JNK, have been shown to phosphorylate the transcription factor Pax6 in vitro and in vivo.²⁰⁸,²⁰⁹ Pax6 is a member of the paired box-containing Pax gene family originally identified in *Drosophila*.²⁰⁸ These proteins are involved in organ development (eyes, nose, pancreas, and central nervous system) in mammals. Three out of four putative MAP kinase phosphorylation con-

Chemical Reviews, 2001, Vol. 101, No. 8 2459

sensus sites (Thr323, Ser376, and Ser413) within the transactivating domain of Pax6 were phosphorylated in vitro by both ERK2 and p38. Overexpressed Pax6 was phosphorylated on Ser413 in cells stimulated with serum or phorbol ester or coexpressing either MEK1 or MKK6b. Phosphorylated Pax6 displayed a transactivating activity that was increased 3- and 16-fold, respectively. When Ser413 was mutated to alanine in these experiments, p38 phosphorylation was abolished and ERK phosphorylation was reduced by 30%.

ERKs, along with JNK and p38, associate with and phosphorylate the transcriptional regulator nuclear factor of activated T cells (NFATc).²¹⁰–²¹² ERK, JNK, and p38 associate with the N-terminus of the regulatory domain in cells transfected with NFATc. These kinases phosphorylated NFATc on Ser172. Calcineurin dephosphorylates NFATc and causes its translocation to the nucleus where it induces expression of immunoregulatory genes. Overexpression of ERK2 inhibited ionomycin-induced NFATc nuclear localization in T cells, while MEK inhibitors had the opposite effect.

ERK2 has been shown to phosphorylate multiple members of the signal transducer and activator of transcription (STAT) family, notably STAT3. STAT5a coimmunoprecipitates with ERK2 via its C-terminal transactivation domain in unstimulated Chinese hamster ovary cells and is phosphorylated by the kinase on Ser780.²¹³ When activated by growth hormone, STAT5a was phosphorylated on multiple residues, which led to transcriptional activation of a reporter gene. MEK inhibitors were able to block STAT5a activity, demonstrating that phosphorylation by ERK1/2 was important for its maximal activation.

### 5. Targets of Downstream Protein Kinases

Rsk2 and MSK1, two other ERK1/2 substrates, have been shown to phosphorylate the conserved Ser10 residue of histone H3, a core protein of the nucleosome.²¹⁴–²¹⁶ When phosphorylated, H3 activates immediate early gene expression, chromatin remodeling, and chromatin condensation during mitosis. ERK1/2 and p38 also phosphorylate Ser10 in vitro. Inhibitors and dominant negative mutants of ERK2 and p38 both block H3 phosphorylation induced by ultraviolet B in the JB6 C1 41 mouse epidermal cell line, indicating that these kinases either directly or indirectly regulate the function of H3.

### K. Disruption of Genes in the Cascade

Mouse gene disruption experiments are playing a critical role in determining the physiological functions of specific components of the ERK1/2 cascade. Gene knockouts of Raf-1, BRaf, ARaf, MEK1, and ERK1 have been reported, as well as a double knockout of Raf-1 and BRaf. With the exception of ERK1, these gene disruptions adversely affect development and lead to death during embryogenesis or shortly after birth. In this section, reports describing the phenotypes of mice lacking kinases of the ERK1/2 cascade are summarized.
Table 2. Summary of Disruptions of Genes in MAP Kinase Pathways

| gene knockout | lethality/viability | phenotype | ref |
| --- | --- | --- | --- |
| A-Raf | 1–3 weeks; 50% survive to adulthood on 129/OLA genetic background | neurological defects; megacolon | 220 |
| B-Raf | E10.5–12.5 | vascular defects; abnormal apoptosis, especially of endothelial cells | 218 |
| Raf-1 | E10.5–12.5; survive to term on CD1 genetic background | abnormal placental development; abnormal development of skin, eyelids, and lungs | 217 |
| B-Raf/Raf-1 | E8.5 | abnormal development beyond blastocyst stage | 219 |
| Mekk1 | viable | defective JNK activation in response to many stress stimuli in Mekk1–/– ES cells; eyelid defect | 144, 145, 310 |
| Mekk3 | E11.5 | defective angiogenesis, esp. in labyrinthine placenta | 455 |
| Mek1 | E10.5 | defective angiogenesis in labyrinthine placenta | 221 |
| Mkk3 | viable | defective IL-12 production in LPS-stimulated macrophages and CD40-stimulated dendritic cells defective IL-1 and IL-6 secretion in TN α-stimulated Mkk3–/– fibroblasts | 350 |
| Mkk4/Sek1 | E10.5–12.5 | defective liver development; immunological defects in Mkk4/Sek1–/–, Rag2–/– chimeras | 313–315, 470–473 |
| Erk1 | viable | impaired thymocyte proliferation and maturation in response to TCR-ligation | 222 |
| Jnk1 | viable | activated Jnk1–/– CD4+ T cells preferentially differentiate into Th2 effector cells | 316 |
| Jnk2 | viable | altered T cell activation, differentiation and apoptosis | 317, 318 |
| Jnk3 | viable | resistance to excitotoxicity-induced hippocampal apoptosis | 319 |
| Jnk1/Jnk2 | E10.5–11.5 | abnormal apoptosis during embryonic brain development; exencephaly | 320, 321 |
| p38α | E10.5–11.5 (some survive to E16.5) | defective angiogenesis in labyrinthine placenta defective definitive erythropoiesis; defective erythropoietin expression defective MAPKAP-K2 activation in arsenite-treated p38α–/– ES cells; decreased IL-1-stimulated IL-6 secretion in differentiated p38α–/– ES cells | 442–444 443 441 |

---

### 1. Raf-1
The Raf-1 (c-Raf) gene was disrupted by replacing the first coding exon with a drug-resistance gene. This strategy blocked production of the full-length (74 kDa) Raf-1 protein but unexpectedly generated a 62 kDa N-terminally truncated Raf-1 protein. This mutant form of Raf-1 was found at a low level and was about 10% as active as wild-type Raf-1 from serum- or phorbol ester-treated cells. Mouse embryos homozygous for the mutated Raf-1 allele grew more slowly by embryonic (E) day 10.5 and died by E 12.5. The histology of mutant placentae revealed gross defects, in particular, reduced spongiotrophoblast and labyrinthine layers, leading to the conclusion that embryonic lethality in Raf-1 mutant mice likely results from poor placental function.

The genetic background profoundly affected the severity of developmental defects observed in the Raf-1 mutant mice. Most Raf-1 mut/mut embryos died by E 12.5 on a 129/C57B6 background. In contrast, when the mutated Raf-1 allele was evaluated on the outbred CD1 background, two-thirds of the homozygous mutant mice survived until birth. Developmental defects were observed in Raf-1 mut/mut mice after midgestation. Certain epithelial tissues were abnormal, including the eyelids and skin. The cause of death of these animals was attributed to failed lung maturation. Fibroblasts derived from these embryos did not proliferate normally in response to serum growth factors.

---

### 2. B-Raf
The gene encoding B-Raf was disrupted by eliminating the exon encoding part of the Ras-binding domain. B-Raf–/– mice died in utero (E 10.5–12.5) from vascular deficiencies including enlarged and poorly formed vessels, which were prone to ruptures, indicating that its major actions were on the normal development of the vasculature. This was somewhat surprising given the high expression of B-Raf in neuronal and neuroendocrine tissues. The elimination of B-Raf also led to abnormal patterns of apoptosis throughout the developing embryos; however, only the vascular endothelium displayed greater total cell loss.

---

### 3. Raf-1 and B-Raf
The phenotypes of mice lacking either Raf-1 or B-Raf imply that they are required in distinct developmental programs. Mice heterozygous for the disrupted Raf-1 and B-Raf alleles were crossed to probe for possible overlapping functions of these two Raf genes. Raf-1/B-Raf-deficient embryos underwent normal development only to the blastocyst stage, revealing a function in early development that was shared by these two Raf isoforms. The authors also found gene dosage effects on development, in that defects in mice lacking three of these four Raf alleles were more severe than in animals lacking only two of the four alleles.

---

### 4. A-Raf
Animals in which the A-Raf gene had been disrupted did not express detectable A-Raf protein. As with Raf-1, the phenotype was influenced by mouse strain. On a C57B6 background, A-Raf–/– animals were normal in size until 2–3 days after birth and survived no more than 3 weeks. These

young A-Raf -/- animals displayed neurological defects by behavioral measures, without identifiable histological abnormalities, and megacolon. In contrast, on a largely 129/OLA genetic background, one-half of the A-Raf-deficient mice, although small, survived to adulthood. The abnormalities were largely absent in the adult survivors.

5. MEK1

Insertional mutagenesis was used to disrupt the mouse MEK1 gene. The MEK1 -/- embryos, which died around E 10.5, were small with swollen yolk sac blood vessels that lacked red blood cells.²²¹ The abnormal placentae contained an ill-defined spongiotrophoblast layer and a compressed labyrinthine layer. The histological evidence indicated a failure of angiogenesis into the labyrinthine layer. In addition to the apparent role of ERK1/2 in angiogenesis, analysis of migration of fibroblasts from these embryos suggested a requirement for ERK1/2 in the stimulation of migration by extracellular matrix under some but not all conditions.

6. ERK1

The ERK1 gene was disrupted by deleting residues encoding a portion of the catalytic core.²²² Consistent with the selective loss of ERK1 but not ERK2 protein, ERK activity in lysates from -/- embryonic fibroblasts stimulated with serum was one-half that of wild-type cells. The kinetics of ERK2 activation by serum in these cells were altered in that activation was prolonged. ERK1 -/- mice displayed no obvious differences from the wild-type controls. The only changes in proliferation of various types of cells from ERK1 -/- animals were noted in thymocytes. The thymocytes proliferated less robustly upon activation of the T cell receptor and matured into single-positive (CD8+ or CD4+) thymocytes more slowly. The defect in maturation may have been due to reduced positive thymic selection, because apoptosis in thymocytes from the ERK1-deficient mice was unchanged.

III. JNK/SAPKs

A. Identification

JNK1, -2, and -3, also known as SAPKγ, SAPKα, and SAPKβ, exist as 10 or more spliced forms.²²³⁻²²⁵ Over the MAP kinase catalytic core they are 87–88% identical to one another (Table 1). They are ubiquitous, although JNK3 is present primarily in brain. They are identified as stress-activated protein kinases because their activities increase in response to cytokines, UV irradiation, growth factor deprivation, and agents that interfere with DNA and protein synthesis, for example, and less robustly to ligands for some G protein-coupled receptors, serum, and growth factors. They are involved in cytokine production and other aspects of the inflammatory response, more generally in the function of the immune system, stress-induced and developmentally programmed apoptosis, actin reorganization, and in cell transformation.²²⁶ These kinases have been reviewed in detail elsewhere (Kyriakis, J. M.; Avruch, J. Physiol. Rev., in press).

B. Structure

Like ERK1/2, JNK/SAPKs are activated by phosphorylation of a tyrosine and a threonine residue, which in all JNK/SAPKs are separated by proline, in their activation loops. JNK/SAPKs contain the same core structure with interesting differences in the conformation of the activation loop which reveal differences in mechanism of regulation.²²⁷ The N- and C-terminal domains of the kinase are rotated farther apart in JNK3 relative to ERK2.²³,²⁵,²²⁷⁻²²⁹ As with ERK2, the activation loop is not positioned appropriately for protein substrate binding, but in JNK3 this loop is four residues shorter and folded up between the domains rather than extended outward. The tyrosine phosphorylation site, Tyr223, is solvent exposed in JNK3; the comparable residue in ERK2 is buried. In JNK3, the P+1 site, which binds proline following the phosphoacceptor site in substrates, is blocked by Arg 230, which just follows the loop. The phosphate anchor ribbon which interacts with ATP is better ordered in JNK3 compared to ERK2 and the ATP binding mode similar to that of cAMP-dependent protein kinase.³⁰,³¹,²³⁰

C. MEKs

MKK4 (SEK1, MEK4, JNKK1, SKK1) and MKK7 (MEK7, JNKK2, SKK4) are the predominant MEKs in JNK/SAPK pathways. Their specificity is somewhat relaxed compared to MEK1/2 because they will also phosphorylate p38 MAP kinases in vitro, and some evidence suggests they may do so in cells under certain circumstances.²³¹ Although MKK4 has the capacity to activate JNK/SAPKs and p38s by phosphorylating both threonine and tyrosine phosphorylation sites, in vitro a preference for tyrosine is revealed. Perhaps MKK7 displays complementary behavior in that it displays a preference for the threonine residue. These kinases may cooperate to activate JNK/SAPKs in cells.²³²,²³³ Threonine phosphorylation may be most important for activity changes of JNK3.²³³,²³⁴ These MEKs are phosphorylated on two sites, a serine and a threonine, in their activation loops. Like MEK1/2, replacement of these residues with acids generates forms which have enhanced activity in cells.

Although MAP kinases are the best characterized substrates of MEKs, MKK4 has an additional substrate, the retinoid X receptor (RXR). This receptor forms heterodimers with the retinoic acid receptor to regulate gene transcription induced by retinoic acid. Cell stresses, such as anisomycin, or constitutively active MKK4 caused the phosphorylation of the RXR, inhibition of subsequent gene expression induced by the RXR, and accelerated RXR degradation by the proteasome.⁴² Inhibitory forms of JNK/SAPKs did not block the effects of constitutively active MKK4, indicating that JNK/SAPKs were not required. These effects of cell stresses on RXR were significantly reduced in cells lacking MKK4 and were restored by reintroduction of MKK4 by transfection. Thus, phosphorylation of the RXR by MKK4 mediates the inhibition of retinoid signaling by cell stresses.

D. MEKKs and Small G Proteins

The MEKKs in the JNK/SAPK pathway have been identified by cDNA cloning and expression studies. Because overexpression of a number of kinases leads to activation of JNK/SAPKs, confirmation that these MEKKs are important for JNK/SAPK regulation has been more difficult to obtain and has relied significantly on gene disruption. Enzymes with MEKK activity include relatives of the yeast MEKK, Ste11p, and a more upstream kinase in the yeast pheromone response pathway, Ste20p. ${ }^{235-237}$

Among homologues of Ste11p that activate JNK/SAPKs when overexpressed are MEKK1–4. ${ }^{238-241}$ These kinases range in size from 70 to nearly 200 kDa. Like Raf, these kinases have C-terminal catalytic domains and N-terminal regulatory domains. ${ }^{235}$ In their kinase domains, MEKKs2–4 are ~50% identical to MEKK1. ${ }^{238-241}$ Other MEKKs for JNK/SAPK pathways are much less closely related. The mixed lineage kinase MLK3/PTK-1/SPRK is strongly linked to JNK/SAPK activation, not only because its overexpression activates the pathway, but also because it binds to JNK pathway scaffolding proteins, as described more fully below. ${ }^{242,243}$

Other MEKKs that activate JNK/SAPKs based primarily on overexpression in transfected cells are as follows: MAP three kinase (MTK1), ${ }^{238,240,241,244,245}$ tumor progression locus 2 (Tpl-2/Cot), ${ }^{246}$ dual-leucine zipper kinase (DLK), ${ }^{247}$ mixed lineage kinase MLK2/MST, ${ }^{248}$ TGF$\beta$-activated kinase (TAK1), ${ }^{249}$ apoptosis signal regulating kinases (ASK1)/MAPKKK5 ${ }^{250,251}$ and ASK2/MAPKKK6, ${ }^{252}$ dendritic cell-derived protein kinase (DPK), ${ }^{253}$ and thousand and one amino acid kinases 1,2 (TAOs1,2). ${ }^{254,255}$ Several of these kinases also activate other MAP kinase pathways as well as other signaling cascades such as that activating NF-$\kappa$B. Disruption of the MEKK1 gene indicates that it is important in regulation of JNK/SAPKs by some but not all stimuli. ${ }^{144}$

Several groups linked JNK/SAPK activation to small G proteins, especially Rac and Cdc42, in a manner that is variably related to the p21-activated kinase PAK or to MLK3. ${ }^{256-260}$ The v-Crk oncogene activates JNK through the R-Ras in fibroblasts. ${ }^{261}$ Constitutively active mutants of R-Ras, H-Ras, and Rac1 enhanced JNK activity. Likewise, dominant-negative mutants of these proteins, alone with kinase-dead MLK3, blocked JNK activation by v-Crk. On the basis of the use of dominant negative mutants, Vav1-dependent stimulation of JNK/SAPKs was mediated by a pathway consisting of Rac, hematopoietic progenitor kinase-1 (HPK1), MLK3, and MKK7. ${ }^{262}$ Rho has also been linked to JNK activation. ${ }^{263}$ Several other kinases that appear to be upstream of MEKKs in the JNK pathway are the Nck-interacting kinase (NIK), ${ }^{264}$ HPK, ${ }^{265,266}$ germinal center kinase (GCK), ${ }^{267}$ NIK-like embryo-specific kinase NESK, ${ }^{268}$ and the HPK/GCK-like kinase HGK. ${ }^{269}$

E. Activation from the Cell Surface

Mechanisms for ligand- or stress-activation of JNK/SAPKs are best delineated for G protein-coupled receptors. A number of studies described next indicate a role of pertussis toxin-sensitive heterotrimeric G proteins, Gi and Go, although the steps between G proteins and the JNKs are less well explored. ${ }^{270}$ G12, G13, Gq, and G16 also activate JNK/SAPKs. ${ }^{112,271,272}$

Although Gi is a frequent element in JNK activation, the downstream mechanisms linking to JNK/SAPKs, particularly the effects of calcium, may be cell-type- or ligand-dependent. Activation of JNK1 by endothelin in Rat 1 cells was partially blocked by pertussis toxin, suggesting that it is regulated in part by Gi or Go. Neither PKC nor Ca $^{2+}$ was required for JNK activation in this system. ${ }^{273}$ Thyrotropin-stimulating hormone (TSH) also activated JNK in a pertussis toxin-sensitive manner in thyroid cells. The increase in JNK activity was unaffected by PI-3 kinase inhibitors but was blocked by inhibitors of protein kinase C. ${ }^{274}$ Activation of the m3, but not the m2, acetylcholine receptor stimulated JNK activity in CHO cells overexpressing these receptor types. ${ }^{275}$ Agonist stimulation was pertussis toxin-sensitive, consistent with mediation by Gi or Go, but was reduced by preventing the influx of extracellular Ca $^{2+}$. Two studies have demonstrated that gonadotropin releasing hormone stimulated JNK activity in rat pituitary gonadotrophs, either alphaT3–1 or primary cells. ${ }^{276,277}$ There was agreement that Cdc42 was required but disagreement about PKC which was only required in one case. In addition, one study found that Src and MEKK1 were also required but that Ras was not, while the other concluded that JNK activation was dependent on PAK1 and intracellular calcium but independent from extracellular calcium. JNK/SAPKs may be activated by $\beta \gamma$ subunits or $\alpha$ subunits in a manner that may depend on tyrosine kinases. ${ }^{278-280}$ Constitutively activated mutants of G$\alpha$i1, G$\alpha$i2, and G$\alpha$i3 activated JNK in 293 cells, while comparable mutants of G$\alpha$o and G$\alpha$z did not. Activation of JNK by G$\alpha$i was inhibited by dominant-negative mutants of the small G proteins Rho and Cdc42 as well as tyrosine kinase inhibitors but not by dominant-negative Rac or inhibitors of PI-3 kinase. The microtubule-disrupting drug vincristine activated JNK1 in HL60 cells. ${ }^{280}$ JNK1 activation and IL-1 production were blocked by piceatannol, an inhibitor of the tyrosine kinase Syk, but not by an inhibitor of Src, suggesting that Syk is upstream.

F. Interactions with Nuclear Receptor Signaling Pathways

Like ERK1/2, the activities of JNK/SAPKs are influenced by nuclear hormones. Swantek initially demonstrated that activation of JNK/SAPKs in macrophages and other cell types is inhibited by glucocorticoids in a manner that depends on the presence of nuclear receptors but not on transcription. ${ }^{281}$ The mechanism has not been elucidated, but the site of inhibition appears to lie upstream of MEKK7. Dexamethasone also inhibited TNF-$\alpha$-induced JNK phosphorylation in HeLa cells, based on immunofluorescence with anti-phospho JNK antibodies. ${ }^{282}$ A possibly similar effect has been observed with retinoic acid. ${ }^{283,284}$ The transcriptional activation function of

MAP Kinases

![Diagram](#)

Figure 8. Multiple mechanisms may assemble components of the JNK/SAPK pathway. One major group of complexes may be organized by JIP and similar proteins. Kinases themselves, such as MEKK1, may organize signaling complexes in this cascade.

the retinoic acid receptor was required for the long term but not the acute suppression of JNK/SAPK activity. The effect could be detected at the level of MKK4, which was also inhibited. At least part of the suppression was due to the induction of MKP-1.

### G. Protein Complexes

An answer to questions about specificity in MAP kinase pathways was suggested with the discovery that the Ste5p protein of the yeast pheromone response pathway is a scaffold for the components of the MAP kinase cascade in that system. Since that time, a major focus has been the identification of scaffolds for the mammalian cascades that might prevent inadvertent crosstalk between pathways.¹⁵ While nothing like Ste5p has been discovered in the ERK1/2 pathway, proteins that appear to function primarily as scaffolds have been found for the stress kinase cascades. The first of these was JNK inhibitory protein (JIP) 1. Several other JIPs and proteins known as JSAPs have been found, some with sequences related to JIP1 and others with unrelated sequences.²⁸⁵–²⁹⁰ In general, these proteins bind some or all of the JNK/SAPKs, MKK4 or MKK7 (but each scaffold probably binds only one or the other), and MEKK level enzymes of the mixed-lineage kinase subgroup, e.g., MLK3 or DLK, or MEKK1 (Figure 8). They may also bind kinases that are thought to lie upstream of the MEKK level enzymes such as HPK1. These scaffolds may also contain other modular protein-protein interaction domains such as SH3 domains and may homo- or hetero-oligomerize with other similar proteins.

At least two of the MEKKs that regulate JNK/SAPKs, MEKK1 and MEKK2, possess their own binding sites for multiple components of the JNK/SAPK pathway. Thus, the MEKKs themselves may also contribute significantly to organization, localization, and function of the downstream MAP kinases. For example, MEKK1 is localized largely on actin fibers, focal adhesions, and microtubules and may provide a mechanism for regulation of the JNK/SAPK cascade on the cytoskeleton.¹⁴³ MEKK1 also binds a possible upstream kinase, the Nck-interacting kinase NIK.²⁶⁴ Nck, through NIK and MEKK1, may link tyrosine phosphoproteins and Sos to JNK/SAPKs.²⁹¹

---

### H. Regulation of Subcellular Localization

Like ERK1/2, JNK/SAPKs may redistribute from cytoplasm to the nucleus upon activation. In unstimulated cells, JNK/SAPKs may be more readily detected in nuclear fractions, although it may be more difficult to define a low-activity condition for these kinases. Using antibodies to phosphorylated JNK, the active form was localized to punctate structures along microtubules and to nuclear speckle populations in neurons.²⁹² Adriamycin stimulated JNK activity and caused an increase in nuclear immunostaining particularly on nuclear speckles with the antiphospho JNK antibody. The active kinase colocalized with snRNPs, suggesting some function in the regulation of splicing factors. Ischemia was also found to promote the translocation of JNK1 from the cytoplasm to the nucleus in rat heart.²⁹³ Reperfusion following ischemia blocked further nuclear translocation of JNK1.

There are differences in localization of JNK/SAPK isoforms. In a different study, the three major forms were found widely distributed in brain, although JNK2 was present at lower concentrations than the other two. JNK2 was present in the nucleus and cytoplasm of neurons of adult mice, while JNK1 was primarily in the cytoplasm and dendrites.²⁹⁴

---

### I. Pathway Inhibitors

Pharmacological inhibitors specific for the JNK/SAPK pathway are not yet available. An inhibitor of JNK/SAPK activation, CEP-1347, was identified in a screen for molecules that promoted neuronal survival.²⁹⁵,²⁹⁶ CEP-1347 prevented death of neurons that had been deprived of growth factors and also inhibited JNK1 activation with equivalent IC50s; ERK1/2 activity was unaffected. This inhibitor also inhibited JNK1 activation in fibroblasts by ultraviolet irradiation, osmotic shock, and glycosylation inhibitors. CEP-1347 did not inhibit a kinase downstream from p38 MAP kinase, indicating that neither the ERK nor p38 pathways are affected by the drug. A concern is that the compound did not inhibit JNK1 activation by MEKK1 in fibroblasts, consistent with the conclusion that neither JNKs nor MEKs 4 or 7 are affected. The target appears to be further upstream. Pathways branching from its upstream target, in addition to JNKs, may also be affected. Because CEP-1347 is a derivative of K-252a, a broad specificity protein kinase inhibitor, it may be expected to have a number of secondary targets.

Most studies have relied on transfection of kinase dead forms of JNK/SAPKs or inhibitors that may also have additional actions.²⁸¹,²⁹⁷ Using a catalytically defective mutant, Swantek and colleagues showed that JNK/SAPKs were required for translational induction of TNF-α by lipopolysaccharide.²⁸¹ The p38 inhibitors SB203580 and SB202190, described below, also block JNK activity at concentrations above those necessary to block p38. Using this strategy, Le-Niculescu and co-workers provided evidence that JNK-induced apoptosis of neuronal cells, caused by growth factor withdrawal or by KCl, was due at least in part to the induction of Fas ligand in a manner that required phosphorylation of c-Jun.

J. Substrates and Targeting Domains

1. Specificity and Targeting Domains

The idea that MAP kinases bind stably to their substrates arose with studies of JNK/SAPKs, which were extracted with c-Jun.²⁹⁸,²⁹⁹ A motif in c-Jun called the delta domain was required for its stable association with JNK/SAPKs. The site to which the delta domain binds on JNK/SAPKs is believed to be an insert between subdomain IX and X of certain JNK/SAPK splice variants not present in other MAP kinases. JNK/SAPKs, like ERK1/2, also bind to D domains, presumably through their CD motifs. Also like ERK1/2, JNK/SAPKs phosphorylate substrates in membranes, in the cytoplasm, on the cytoskeleton, and in the nucleus. This section of the review is limited to substrates or functions identified in the last 3 years.

2. Membrane Substrates

Insulin receptor substrate-1 (IRS-1) is tyrosine-phosphorylated in response to insulin receptor activation and mediates much of the insulin-signaling in cells. TNF-α inhibits insulin action by inhibiting tyrosine phosphorylation of IRS-1.³⁰⁰ This effect appears to be mediated by activation of JNK which phosphorylated IRS-1 on serine residues. The site of IRS-1 phosphorylation in anisomycin-activated cells was Ser307, near its PTB domain.³⁰⁰ IRS-1 has 14 potential JNK consensus binding motifs, based on the binding site identified in JIPs. Binding of JNK to IRS-1 was suggested by coimmunoprecipitation with a fragment of IRS-1 containing amino acid residues 555–898.

3. Cytoplasmic and Cytoskeletal Substrates

In KB-3 cells treated with vinblastine (a microtubule-damaging agent), JNK1 and JNK2 were activated and antiapoptotic proteins, Bcl-2 and Bcl-xL, were phosphorylated.³⁰¹ Specific JNK phosphorothioate antisense oligonucleotides were able to block this vinblastine-induced phosphorylation of both Bcl-2 and Bcl-xL by 85% and 65%, respectively. When phosphorylated, Bcl-2 becomes inactivated and apoptosis occurs. It is unclear what the outcome of Bcl-xL phosphorylation is, but it is assumed that it is involved in its regulatory activity in some manner.

In a yeast two-hybrid screen searching for *Drosophila* JNK (DJNK) interacting partners, a novel protein named p150-Spir was cloned.²²⁶ p150-Spir belongs to the Wiscott-Aldrich syndrome protein (WASP) homology domain 2 (WH2) family involved in actin reorganization. p150-Spir contains four WH2 domains and one motif related to the FXFP docking motif. In NIH3T3 cells, JNK and p150-Spir colocalized to discrete regions around the nucleus and JNK phosphorylated residues in the C-terminus of p150-Spir in vivo and in vitro.

4. Nuclear Substrates

Yeast two-hybrid experiments using JNK3 as bait in a human brain cDNA library identified the first novel substrate for JNK3.³⁰² DENN/MADD is a splice variant of mitogen-activated kinase activating death domain that is differentially expressed in neoplastic vs normal cells. Both DENN/MADD and JNK3 were colocalized to large pyramidal neurons in human hippocampus by in situ hybridization. In these hypoxia-sensitive neurons, DENN/MADD and JNK3 both translocated to the nucleus under hypoxia/ischemic stress. JNK3 phosphorylated DENN/MADD, although the sites were not determined.

The nuclear localization of NFATc1 is regulated by JNK. Calcineurin, a protein phosphatase that mediates calcium signals during T cell activation, dephosphorylates cytoplasmic NFATc1 and causes its nuclear translocation and activation of cytokine expression.³⁰³ Calcineurin interacts with NFATc1 via the PXIXIT motif within the N-terminus.³⁰⁴ JNK, however, phosphorylates Ser117 and Ser172 of NFATc1α and β isoforms as determined by in vitro mutational analysis and in vivo labeled-phosphopeptide mapping.³⁰⁵ Residues 126–138 were also required for JNK binding. These phosphorylation and binding sites are near or within the calcineurin-targeting domain of NFATc1; phosphorylation by JNK inhibited calcineurin-stimulated nuclear accumulation of NFATc1 specifically in T cells. Thus, activation of JNK inhibits NFATc1 activity by preventing its nuclear translocation.

Activation of JNK and p38 are capable of increasing the stability of activating transcription factor-2 (ATF-2) factor by preventing ubiquitination and degradation.³⁰⁶ Phosphorylation on Thr69 and Thr71 in the N-terminus of ATF-2 is presumed to cause a conformational change that promotes ATF-2 homo- and heterodimerization and DNA binding capability. In human fibroblasts stimulated with TNF-α, okadaic acid, which inhibits phosphatase activity, increased the duration of JNK phosphorylation of ATF-2 and increased its transactivation activity by blocking proteasome-mediated degradation.

Several proteins, including heat shock transcription factor-1 (HSF-1), are substrates for both ERK1/2 and JNK/SAPKs.³⁰⁷ HSF-1 regulates the expression of heat shock proteins to promote cell survival. Normally, HSF-1 is a phosphorylated monomer in the cytoplasm; during environmental stress, however, HSF-1 becomes a more highly phosphorylated trimer and translocates to the nucleus. As a result, HSF-1 binds DNA and increases its transcription of target genes. JNK immunoprecipitated from heat shock-activated HeLa cells phosphorylated HSF-1 on Ser363 within its regulatory domain. Mutation of this site to alanine reduced JNK phosphorylation. JNK also bound to residues 203–224 of HSF-1, within its conserved D domain. JNK binding and phosphorylation resulted in rapid clearance of HSF-1 punctate granules from the nucleus and subsequent suppression of transcriptional activity.

STAT3 is another substrate phosphorylated by both ERK1/2 and JNK/SAPKs.³⁰⁸,³⁰⁹ In EGF-stimulated cells STAT3 is phosphorylated by ERK2 on Ser727, enhancing STAT3 activity.³⁰⁸ JNK isolated from cells treated with TNF-α, UV, anisomycin, or transfected with MEKK1 phosphorylated STAT3 in vitro, resulting in negative regulation of STAT3 DNA binding and transcriptional activity. PD98059 had no effect on this inhibitory regulation by JNK.

MAP Kinases

K. Functions Deduced from Gene Disruption

The physiological functions of a number of the kinases of the JNK/SAPK cascade have been examined by gene disruptions in mice. Specifically, mouse knockouts of MEKK1, MKK4/SEK1, JNK1 (SAPKg), JNK2 (SAPKa), and JNK3 (SAPKb) have been reported. These studies are providing critical insights into the relative roles various JNK/SAPK cascade components have in the regulation of apoptosis and in immune cell biology.

### 1. MEKK1

The mouse MEKK1 gene has been knocked out by two groups of investigators.¹⁴⁴,¹⁴⁵,³¹⁰ While the initial report suggested that the gene was essential for survival, later reports agreed that animals were viable but displayed a defect in eyelid closure. MEKK1⁻/⁻ ES cells were used to examine the essentiality of MEKK1 for the activation of JNK by various stress signals or pro-inflammatory stimuli as well as the role of MEKK1 in apoptosis. When treated with nocodazole or sorbitol, MEKK1⁻/⁻ embryonic stem (ES) cells displayed higher rates of apoptosis than wild-type cells, supporting the conclusion that MEKK1 provides an antiapoptotic signal. Opposite conclusions have been drawn from experiments with overexpressed MEKK1, where, following caspase cleavage, MEKK1 acts to promote apoptosis.³¹¹,³¹² In the absence of MEKK1, activation of JNK/SAPKs in ES cells by UV, anisomycin, or heat shock was unchanged, but significant decreases in activation were noted with serum, LPA, cold stress, and nocodazole.¹⁴⁴ A partial loss of ERK1/2 activation by sorbitol, serum, and LPA was observed, while no effects on p38 were identified. The other group found that MEKK1 was required for JNK activation by growth factors, TNFα, IL-1, double-stranded RNA, and LPS and no loss of ERK1/2 activation by growth factors.¹⁴⁵

### 2. MKK4/SEK1

Three groups of investigators have knocked out the mouse gene for MKK4/SEK1; each has reported that MKK4/SEK1⁻/⁻ mice die by E 12.5.³¹³⁻³¹⁵ In two cases, investigators found that although yolk sac hematopoiesis appeared normal, MKK4/SEK1⁻/⁻ embryos were anemic.³¹³,³¹⁴ Histological examination of livers from MKK4/SEK1⁻/⁻ embryos revealed a paucity of hepatocytes relative to the livers of wild-type embryos. Rates of apoptosis in livers or cultured hepatocytes from MKK4/SEK1⁻/⁻ mice were increased as compared to wild-type animals. These findings led to the conclusion that MKK4/SEK1 is essential for the survival and/or proliferation of hepatocytes in vivo.

The role of MKK4/SEK1 in JNK/SAPK and p38 activation was investigated using MKK4/SEK1⁻/⁻ ES as well as mouse embryo fibroblasts (MEFs) from MKK4/SEK1⁻/⁻ mice.³¹⁴ JNK/SAPK activation was eliminated or drastically reduced in MKK4/SEK1⁻/⁻ ES cells either treated with anisomycin or exposed to UV radiation or heat shock. Under these same cellular stress conditions, MKK4/SEK1⁻/⁻ ES cells had normal p38 activation profiles. Experiments with MKK4/SEK1⁻/⁻ MEF cells showed dramatic decreases in both JNK/SAPK and p38 activities following exposure to IL-1 or TNFα.³¹⁴ Anisomycin-triggered activation of both JNK/SAPK and p38 was abolished in MKK4/SEK1⁻/⁻ MEF cells. While p38 activation in response to sorbitol was unchanged in MKK4/SEK1⁻/⁻ MEF cells, JNK/SAPK activation was eliminated in response to this osmotic stressor.

### 3. JNK1

Four exons of the JNK1 gene were deleted to disrupt it.³¹⁶ The homozygous null mice had no obvious defects and were fertile. They displayed differences in differentiation of T cell populations.

### 4. JNK2

Two groups knocked out the JNK2 gene.³¹⁷,³¹⁸ In both cases the JNK2⁻/⁻ mice were viable and fertile. The conclusions of the two groups regarding the defects observed were different. Yang et al. found that mature (peripheral) CD4+ T cells from JNK2⁻/⁻ mice are impaired in differentiation into the Th1 effector cells following antigen stimulation.³¹⁷ IL-12-induced secretion of the Th1-specific cytokine interferon (IFN)γ was reduced. Sabapathy et al. found that T cell activation and apoptosis were impaired in response to antigen stimulation.³¹⁸ Secretion of IL-2, IL-4 (Th2-specific), and IFNγ (Th1-specific) from peripheral T cells was reduced in response to in vitro stimulation with anti-CD3 antibodies. Sabapathy et al. also compared effects of anti-CD3 antibody on wild-type and JNK2⁻/⁻ mice. The immature CD4+, CD8+ thymocytes did not undergo apoptosis normally induced by the anti-CD3 antibody but did demonstrate normal apoptosis when exposed to UV radiation, dexamethasone, or anti-Fas antibody.

### 5. JNK3

JNK3⁻/⁻ mice were apparently normal and fertile.³¹⁹ Because JNK3 is expressed primarily in brain, this organ was examined in detail and displayed normal histology. To test any possible abnormalities in response to stress, kainate, which activates JNKs in neuronal cultures, was used to induce seizures and neuronal damage. Loss of JNK3 resulted in either less severe kainate-induced seizures or a higher survival rate following seizures than in normal mice. As expected, kainate-induced c-Jun phosphorylation and AP-1 activity were reduced in the hippocampus of JNK3⁻/⁻ mice, as was apoptosis. Thus, JNK3 appears to be essential for neuronal apoptosis induced by excitotoxic stress.

### 6. JNK Double Knockouts

Animals lacking individual JNK genes were crossed to examine effects of eliminating multiple JNKs.³²⁰,³²¹ JNK1/JNK3 and JNK2/JNK3 double mutant mice, like the individual knockouts, survived to term and appeared normal.³²⁰ JNK1/JNK2 double mutants displayed defects in developmentally programmed neuronal apoptosis and died by E 11.5. A major defect was hindbrain exencephaly. One group concluded

that either J NK1 or J NK2 was essential for apoptosis in the hindbrain.³²⁰ The second group found increased hindbrain apoptosis in the mice lacking J NK1 and J NK2, suggesting the opposite conclusion, that J NK1/2 were essential to prevent inappropriate apoptosis.³²¹

### IV. p38 MAP Kinases

#### A. Identification

Four members of the p38 MAP kinase family have been cloned and characterized (Table 1). p38α is 50% identical to ERK2 and bears significant identity to the yeast kinase Hog1p involved in the response to hyperosmolarity.³²²⁻³²⁴ Because the human p38α homologues were identified as molecules that bind to pyridinyl imidazole compounds, which were known to inhibit the release of proinflammatory cytokines from LPS-stimulated monocytes, these p38α proteins were also designated as cytokine suppressive anti-inflammatory drug binding proteins (CSBPs).³²³ Three additional p38 family members, p38β,³²⁵,³²⁶ p38γ,³²⁷,³²⁸ and p38δ,³²⁹⁻³³¹ range from 47% to 42% identical to ERK2 but are 75%, 62%, and 64% identical to p38α, respectively. Alternatively spliced variants of p38 and p38β have been reported, including one named Mxi, which lacks the last 80 residues of p38α and contains in their place 17 unrelated residues.³³¹,³³² This form of p38α was isolated from a two-hybrid screen with Max, a protein that heterodimerizes with the transcriptional regulator c-Myc.

The p38 subgroup containing the ubiquitously expressed p38α and p38β is inhibited by pyridinyl imidazole drugs, while the other two p38 kinases, p38γ and p38δ, are insensitive to these drugs.³³³ p38γ and p38δ, with 67% identity to each other, have distinct expression patterns. p38γ is primarily expressed in skeletal muscle, and its expression is upregulated during muscle differentiation.³²⁷ p38δ expression is also developmentally regulated and is most highly detected in lung, kidney, endocrine organs, and small intestine.³³⁴

Differential expression, activation, and substrate specificity of different p38 isoforms result in their divergent physiological functions in different cellular contexts. For example, the MADS-box transcription factors MEF2A and MEF2C are preferentially phosphorylated and activated by p38α and p38β2 but not by p38γ or p38δ.¹⁷⁸ In human neutrophils, only p38α and p38δ could be detected and p38α specifically mediated LPS-induced cellular events, such as adhesion, NF-κB activation, and synthesis of TNFα.³³⁵ In cardiomyocytes, p38β is involved in a hypertrophic response whereas p38α seems to be the primary p38 isoform mediating apoptosis.³³⁶ Studies in HeLa cells indicated that while p38α induces apoptosis, p38β2 promotes cell survival.³³⁷ p38 isoforms, like other MAP kinases, have overlapping and distinct physiological roles.

#### B. Structure

p38 contains little more than the kinase catalytic domain. The short N- and C-terminal extensions bear little homology to other known proteins except the

related p38 family members and J NK isoforms. The structures of unphosphorylated and phosphorylated p38 have been solved by X-ray crystallography, providing insight into the mechanism controlling activation and substrate specificity.²²⁸,²²⁹,³³⁸⁻³⁴⁰ The fold and topology of p38 is similar to that of ERK2, despite significant differences in the conformation of the activation loop.²³,²²⁹ The C helix of p38 is also rotated 6 D away from its location in ERK2. The interactions between L16 and the N-terminal domain are not as strong. The result is a relatively open ATP binding site in p38. The phosphorylation lip sequence in p38 is six residues shorter than in ERK2, placing the Thr and Tyr phosphorylation sites in distinct positions in p38. In p38, as in JNK3, this sequence is folded up between the two domains. The Gly residue in the Thr-Gly-Tyr (TGY) dual-phosphorylation motif, as well as the length of the loop, contributes to p38 substrate specificity. The P+1 specificity pocket is blocked by the C-terminus of the lip in p38. The length of the activation loop also plays an important role in controlling autophosphorylation, yet modifications of the loop structure do not result in any change in the selection of p38 by MEKs.³⁴¹ p38 specificity is also influenced by docking sites within various substrates as described for ERK2.¹⁷⁸,¹⁷⁹,¹⁸¹,²⁹⁹

#### C. MEKs

MEK3 and MEK6 are thought to be the major kinases responsible for p38 activation in cells (Figure 1342–345). MEK4 also displays activity toward p38 in vitro.³⁴² In MEK4 –/– fibroblasts both JNK and p38 lost their responsiveness to TNFα, interleukin-1, and hyperosmotic stress, suggesting that crosstalk exists between these two stress-sensitive pathways.³¹⁴ There was also evidence that MEK7 activates p38δ, indicating that MEK7 may be an additional MEK family member that regulates p38.³³⁴

Reflecting the specificity observed for MEKs and the relatively low sequence identity among p38 isoforms, selective activation of different p38 isoforms by distinct MEKs was observed. For instance, MEK6 is a common activator for all four p38 isoforms, whereas MEK3 activates only p38α, γ, and δ.³²⁵,³²⁹,³³⁰,³⁴⁶⁻³⁴⁸ The difference in substrate selectivity of MEK3 and MEK6 may account, in part, for their nonredundant functions, revealed by targeted gene disruption studies in mice.³⁴⁹,³⁵⁰ Furthermore, expression of activated mutants of MEK3b (MEK3bE) and MEK6b (MEK6bE) elicited different biological responses in cardiomyocytes,³³⁶ suggesting that this signaling specificity is crucial for the generation of appropriate biological responses by the p38 pathway. Consistent with the emerging theme of MAP kinase docking sites contributing to signaling specificity, selective docking interactions between MEK3/6 and p38 MAP kinases may also contribute to the signaling specificity of the p38 pathway.³⁴³

#### D. MEKKs and Upstream Regulators

MEKs 3 and 6 are phosphorylated and activated by a plethora of MEKKs and MLKs, yet details regarding their relative contribution to the p38

pathway are poorly understood. MEKKs 1–3 have been implicated in p38 activation, although they preferentially regulate JNKs and ERKs.²⁴¹,³⁵¹,³⁵² MEK4 may transmit the MEKK1 signal to p38 in 3T3 cells,³⁵² and MEK3 may transmit the MEKK3 signal as shown by cotransfection studies.³⁵¹

Members of the MLK family contain an SH3 domain, leucine zippers, and a small GTPase binding domain.²⁴² Protein-protein interactions through these domains facilitate integration of signals by MLKs from upstream regulators to the downstream MAP kinases. Four MLK family members have been implicated upstream of p38: MUK/ZPK/DLK,²⁴⁷,³⁵³ MTK1/MEKK4,³⁵⁴ MLK2/MST,³⁵³⁻³⁵⁵ and MLK3/PTK/SPRK.²⁴²,²⁴³ Overexpression of these MLK family members always leads to coactivation of JNK and p38. The relevant contribution to cellular responses has not been fully resolved, although in COS-7 and HeLa cells MTK1-mediated activation of p38 seemed to be the primary pathway triggered by environmental stress (osmotic shock, UV radiation, and anisomycin).³⁵⁴

Other kinases that may regulate p38 include Tpl2,²⁴⁶ ASK1,²⁵⁰ and TAK1.²⁴⁹,³⁵⁶ Tpl2 may preferentially regulate JNK, whereas overexpressed ASK1 and TAK1 are more effective activators of the p38 pathway. Mammalian homologues of the yeast Ste20p also regulate stress-responsive pathways. Although the majority of the family members, such as GCK, HPK1, NIK, and GLK, have only displayed activities on the JNK pathway,²⁶⁴,²⁶⁶,²⁶⁷,³⁵⁷ TAO1/2 and PAKs have been implicated in both the JNK and the p38 pathways.²⁵⁴,²⁵⁵,³⁵⁸⁻³⁶¹ PAK1 mediates IL-1 activation of p38, whereas ASK1 plays a key role in p38-mediated apoptosis induced by TNF-α.²⁵⁰ The proteins that couple these kinases to extracellular cues remain to be fully identified.

TAO1 and 2 have MEKK activity and can activate MEKs 3, 4, and 6 in vitro.²⁵⁴,²⁵⁵ TAO1/2 bind to MEK3 and MEK6 but not to other known MEKs such as MEK4, although MEK4 is an in vitro substrate. The stable association of MEK3 or MEK6 with TAO proteins may link their physiological functions to p38 as binding to MEKK1/2 may link them to JNK/SAPK pathways. A third TAO-like kinase, JNK inhibitory kinase or JIK, is nearly 90% identical to TAO1 in its catalytic domain; effects of JIK on the p38 pathway have not been reported.³⁶²

Rac and Cdc42, the Rho family small GTPases, are potential regulators of the p38 pathway. Dominant-negative Rac or Cdc42 inhibited p38 activity in response to IL-1 stimulation,³⁵⁸ suggesting that these molecules are important signaling intermediates. PAK appeared to be one of the mediators between Rac/Cdc42 and p38, since PAK binds to the Cdc42/Rac GTPase-binding (CRIB) domains and dominant-negative PAK can inhibit p38 activation by Rac/Cdc42.³⁵⁹ MTK1/MEKK4, MLK2, and MLK3 have also been shown to bind Rac and Cdc42 in their CRIB domains,²⁴¹,³⁶³ constituting alternative pathways linking small GTPases to p38.

E. Activation from the Cell Surface

The mechanisms by which heterotrimeric G proteins activate the p38 signaling cascade are dependent on the cellular system. For example, in human embryonic kidney 293 cells, p38 was activated through m1 and m2 muscarinic acetylcholine and β-adrenergic receptors and apparently through Gβγ and Gαq/11.³⁶⁴ In α1A-adrenergic receptor (α1A-AR)-transfected PC12 cells, both JNK and p38 were significantly activated by norepinephrine.³⁶⁵ In adult rat ventricular myocytes (ARVMs), activation of p38 played a protective role in β-AR-stimulated apoptosis and the protective effects of Gi were mediated by p38.³⁶⁶ In contrast, activation of p38 by the somatostatin sst2-(a) receptor isoform was essential in mediating antiproliferative events.³⁶⁷ In primary cultures of cardiac myocytes, activation of p38 by a G protein-coupled receptor agonist, endothelin-1, involved protein kinase C but not Gi or Go.³⁶⁸ In human neutrophils, interaction of chemokines such as FMLP or platelet-activating factor (PAF) with their respective GPCRs selectively activated p38 but not JNK.³⁶⁹ In CHO cells, PAF-induced activation of p38 was attenuated by RGS16, and this attenuation was prevented by a constitutively active form of Gα11 Q209L, suggesting that Gαq/11 is involved in p38 activation by PAF.³⁷⁰ In aT3–1 cells, gonadotropin-releasing hormone (GnRH) induction of immediate early genes involved PKC-dependent activation of the p38 pathway.³⁷¹

IL-1 is one of the best characterized agonists for the stress-responsive pathways. Upon IL-1 stimulation, the IL-1 receptor-associated kinase (IRAK) and MyD88 form a complex, from which IRAK is subsequently released. IRAK then binds to TNF receptor-associated factor 6 (TRAF6), resulting in the interaction between TRAF6 and TAK1. TAK1 then transduces the signal to the stress-responsive MEKs.³⁷²⁻³⁷⁴ The TNFα receptor also sequentially recruits intermediate molecules, such as TNF receptor-associated death domain protein (TRADD), Fas-associated death domain protein (FADD) and TNFα receptor associated protein 2, 5, 6 (TRAF2, 5, 6) in a ligand-dependent manner. These molecules then activate JNK and p38 through mediation of caspases or ASK1.²⁴⁵,³⁷⁵⁻³⁷⁸ A novel adaptor protein, Daxx, was recently shown to directly associate with both the cell death-inducing receptor, Fas, and ASK1 and is therefore, a crucial signaling molecule regulating JNK and p38.³⁷⁹,³⁸⁰ A collagen receptor, integrin α2β1, upregulates collagen gene transcription through p38α,³⁸¹ although the exact mechanism is ill defined. Intermediate molecules transducing signals from other extracellular stimuli, such as UV irradiation, high osmotic stress, and heat shock, to the p38 MAP kinase remain to be elucidated.

p38 can be activated by various physical and chemical stresses, such as oxidative stress in macrophages,³⁸²,³⁸³ hypo-osmolarity in human embryonic kidney 293 cells,³³¹ UV irradiation in PC12 cells,³⁸⁴ hypoxia in bovine pulmonary artery fibroblasts,³⁸⁵ cyclic stretch in mesangial cells,³⁸⁶ and ischemia/reperfusion in perfused heart.³⁸⁷ p38 is activated by IL-1 in KB cells,³⁸⁸ IL-17 stimulation of articular chondrocytes,³⁸⁹ and IL-18 stimulated U1 monocytic cells.³⁹⁰ p38 is also involved in the immune responses to pathogens in various cells, for example, baby hamster kidney (BHK) cells infected by Herpes

2468 Chemical Reviews, 2001, Vol. 101, No. 8

simplex virus type-1,${}^{391}$ Rat-1 cells stimulated by Clostridium botulinum C3 toxin,${}^{392}$ and macrophages stimulated by soluble staphylococcal peptidoglycan.${}^{393}$

### F. Subcellular Localization

The subcellular localization of p38 and its movements following activation are little understood. Immunofluorescence microscopy demonstrated that p38 is present in both the nucleus and cytoplasm of activated cells. ${}^{394}$ p38 was also reported to translocate from the cytosol to the nucleus in response to certain stresses, such as myocardial ischemic stress. ${}^{395}$ A p38 kinase substrate, MAPKAP kinase-2 mediated nuclear export of p38 following stimulation. ${}^{396}$ Tyrosine dephosphorylation of p38α regulated directly by PTP-SL tyrosine phosphatase and indirectly by PKA provides another mechanism regulation cytoplasmic retention of p38α. ${}^{166}$ Differential subcellular localization of p38α and p38β was observed in adult mouse brain, suggesting that these p38 isoforms may have differential physiological functions and mechanisms of subcellular localization. ${}^{397}$

### G. p38 Inhibitors

Pyridinyl imidazole compounds are well characterized inhibitors of p38α. They also work on p38β but not on γ or δ isoforms. The structures of unphosphorylated p38α alone and in complex with five different drug inhibitors have been solved by X-ray crystallography, providing insight into the mechanism controlling activator and substrate specificity. ${}^{228,229,338–340}$ Inhibitors bind in an extended pocket in the ATP-binding site, thereby inhibiting p38 activity. These structures have facilitated the design of more selective and tight binding inhibitors for the treatment of inflammation and other diseases.

### H. Substrates

#### 1. Cytoskeletal and Cytosolic Proteins

p38 phosphorylates the microtubule-associate protein (tau) at physiologically relevant sites, suggesting that cellular stress may contribute to tau hyperphosphorylation during the development of tau pathology. ${}^{398}$ p38δ phosphorylates stathmin, linking stress-induced intracellular signaling cascades to regulation of microtubule dynamics. ${}^{399}$ In stimulated platelets, p38 is involved in early phosphorylation of cytosolic phospholipase A2 (cPLA2). ${}^{400}$ By phosphorylating the Na⁺/H⁺ exchange isoform-1 (NHE−1), p38 plays an important role in angiotensin II-mediated regulation of NHE−1 in vascular smooth muscle cells. ${}^{401}$

#### 2. Nuclear Substrates

p38 phosphorylates and enhances the activity of many transcription factors, including ATF-1/2, ${}^{384,394,402,403}$ myocyte enhancer factor 2C (MEF2C), ${}^{404}$ and MEF2A, ${}^{178,405}$ Sap-1, ${}^{406–408}$ Elk-1, ${}^{407,408}$ NF-κB, ${}^{409,410}$ CHOP/GADD153, ${}^{411}$ Ets-1, ${}^{412}$ the Myc binding partner MAX, ${}^{332}$ heat shock transcription factor-1 (HSF-1), ${}^{413}$ and p53. ${}^{414}$ p38 controls an important cis-element, the AP-1 B binding site through different mechanisms, and thereby regulates the expression

of many genes, including c-jun. p38 phosphorylates ATF2 and promotes its subsequent heterodimerization with Jun family transcription factors and association with the AP-1 binding site.

p38 also indirectly regulates AP-1 activity through induction of c-fos and c-jun. By phosphorylating the ternary complex factors Elk-1 and Sap-1, p38 upregulates the SRE-dependent c-fos gene. c-fos is also induced by binding of cyclic AMP response element-binding protein (CREB) to its CRE consensus sequence following phosphorylation of MAPKAP-kinase 2, another substrate of p38. ${}^{403}$ c-jun transcription, on the other hand, can be induced by phosphorylation of MEF2C and MEF2A by p38.

CHOP10/Gadd153 ${}^{411}$ and C/EBPb, ${}^{415}$ members of the C/EBP family of transcription factors, are also controlled by p38 in mediating the effects of cellular stress on growth and differentiation. Phosphorylation of p53 by p38 mediates N-terminal phosphorylation and apoptosis in response to UV radiation, ${}^{416}$ providing one mechanism of p38 regulation of apoptosis. In newly generated postmitotic neurons, MEF2 is activated by p38 in response to calcium influx; thus, it regulates neuronal survival, defining a function for the p38-MEF2 pathway during nervous system development. ${}^{417}$

### 3. Substrates of Downstream Protein Kinases

While some transcription factors are direct targets of p38, others are phosphorylated by downstream protein kinases that are themselves activated by p38. For example, MAPKAP kinase-2 and a closely related protein kinase, MAPKAP kinase-3, were isolated as substrates of p38α. ${}^{324,418,419}$ Activated MAPKAP kinase-2 and MAPKAP kinase-3 phosphorylate various substrates such as CREB, ${}^{403}$ ATF1, ${}^{403}$ SRF, ${}^{420}$ small heat shock protein 27 (HSP27), ${}^{419}$ and tyrosine hydroxylase. ${}^{421}$ MAPKAP kinase-5 is another homologue that can be activated by p38 in vitro. ${}^{422}$ Other protein kinases that were identified as downstream substrates of p38 include MNK1, ${}^{184,185}$ MSK1 (or RSK-B or RLPK), ${}^{423–425}$ and p38-regulated/activated kinase (PRAK). ${}^{426}$ MNK1 strongly binds to p38 and phosphorylates eukaryotic initiation factor 4E (eIF-4E). Activation of MNK1 by mitogen or stress can be blocked by inhibitors of either p38 or MEK1. MNK1 may therefore be a convergence point between growth factor-activated and stress-activated protein kinase cascades. MSK1 is another protein kinase that mediates both mitogen- and stress-induced responses. Stress-induced activation of CREB and ATF1 by MSK1 is sensitive to SB203580, again linking p38 to these transcription factors. PRAK activity is upregulated by p38α and p38β both in vitro and in vivo. In-gel kinase assays demonstrated that PRAK is a major stress-activated protein kinase that phosphorylates HSP27, providing an alternative mechanism of small heat shock protein phosphorylation by p38.

### 4. Gene Induction and mRNA Stabilization

Studies on p38-regulated gene expression have been greatly facilitated by the use of p38 inhibitors as well as inactive and constitutively active forms of

MAP Kinases

MEK3 and MEK6. p38 regulates the expression of many cytokines, transcription receptors, and cell surface receptors. In stimulated human macrophages and murine microphages, p38 mediates the induction of mRNA synthesis and protein secretion of IL-1β and TNFα.427 p38 also mediates the production of IL-6 in human fibroblast-like synoviocytes428 and IL-8 in human monocytes and polymophonuclear cells.429 In rat pulmonary myofibroblasts, p38 regulates IL-1β-induced platelet-derived growth factor receptor-α expression at the translational level by signaling the synthesis of an mRNA-stabilizing protein.430 p38 was also reported to regulate jun-B, c-jun and c-fos.371,431 Other genes that are upregulated by p38 include cyclooxygenase (COX)-2 in human monocytes and human mammary epithelial cells,432,433 HIV-LTR in hela cells,434 CD23 in U937 cells,435 Fgl-2 in macrophages,436 phosphoenolpyruvate carboxykinase in hepatoma cells,437 inducible nitric oxide synthase (iNOS) in chondrocytes389 and oligodendrocytes,438 and vascular cell adhesion molecule (VCAM)-1 in human umbilical vein endothelial cells.439 Negative regulation of cyclin D1 gene by p38 was observed in CCL 39 cells.440 Future identification of additional genes regulated by p38 will provide more insight into the physiological functions of p38.

I. Functions Deduced from Gene Disruption

1. MKK3

MKK3 −/− mice appeared normal at birth and were fertile.350 p38 was less sensitive to LPS, but equally responsive to sorbitol, in macrophages and dendritic cells from these animals compared to wild-type. JNK activation by LPS was not impaired. Consistent with reduced p38 activation, the LPS-stimulated increases in IL-12 mRNA and secretion from macrophages were impaired, while no changes were observed in secretion of TNFα and IL-6. Deficiencies in cytokine production were also noted in other cell types. Activation of both p38 and JNK by UV, sorbitol, and IL-1 was normal in MEF cells from MKK3 −/− mice.349 In MEFs, activation of p38 by TNFα was reduced and the secretion of IL-1 and IL-6 was severely impaired. These studies confirm a link between MKK3 and p38 in the cytokine biosynthesis.349,350

2. p38α

Four groups of investigators have recently published papers describing the phenotype of p38α-deficient mice; each group concluded that the loss of p38α leads to death during embryogenesis.441–444 Three of the four reports were in agreement that p38α-deficient mice die at E 10.5–11.5 as a result of defective angiogenesis within the labyrinthine placenta.442–444 Allen et al. described experiments using cells derived from the knockout mice and did not report on the time or cause of death.441 Mudgett et al. found that in addition to the placental defect, angiogenesis was also abnormal in the yolk sac and the embryo itself.444 Adams et al. reported severe underdevelopment of the embryonic myocardium and abnormal blood vessel formation in the head but concluded that these

abnormalities are secondary effects caused by poor placental function.442 These authors succeeded in rescuing p38α −/− embryos from midgestational death by fusing morula-stage p38α−/− embryos with tetraploid p38α +/+ cells (this strategy yields p38α −/− embryos bearing genetically wild-type placenta). Mutant mice rescued in this manner displayed normal heart and vessel development and survived to term, leading the authors to conclude that the placenta is likely the only organ in which p38α function is critical for proper embryonic development. Tamura et al. found that although many p38α −/− embryos died at around E 11.5, some survive to as late as E 16.5, appearing normal but slightly pale.443 These authors carefully evaluated the hematopoietic profiles of these longer-lived embryos and concluded that the mice have a defect in definitive erythropoiesis resulting from defective erythropoietin expression.

V. ERK5

A. Identification

ERK5 is a ~100 kDa protein, one of the largest known MAP kinase family members.445,446 The core catalytic domain is at its N-terminus and 51% identical to ERK2. The 400 amino acids C-terminal to the catalytic domain are not required for activity and, as of yet, have no defined function. Like ERK2, ERK5 contains a TEY motif in the activation loop.

B. MEKs

Tyrosine and threonine are phosphorylated by the upstream activator, MEK5, which may be found as α or β isoforms, depending on tissue type.445,447 The other known MEKs do not appear to influence ERK5 activity.

C. Activation and Upstream Regulators

ERK5 activity increases in response to stimuli inducing both cell proliferation, such as growth factors and serum, and stress responses. Initial attempts to delineate the signal transduction pathways involved in ERK5 activation have implicated Src and Ras as regulators of ERK5 signaling in some circumstances.448–450 Receptors that couple to Gq and G12/13 but not Gi also regulate ERK5 activity.451

In addition, recent reports have identified MEKK3 and Tpl2 (Cot-1) as MEK5 kinases. MEKK3 immunoprecipitated from transfected 293 cells phosphorylated MEK5 in vitro and interacted with MEK5 in a two-hybrid assay; however, a similar direct link between MEK5 and Tpl2 remains to be established.452,453

D. Pathway Inhibitors

While no inhibitors have been reported that selectively block ERK5 activation or activity, the MEK1/2 inhibitors discussed earlier, PD98059 and U0126, inhibit MEK5 at concentrations only slightly above the effective range for inhibition of MEK1/2.175 Activation of ERK5 by EGF was blocked by both inhibitors. On the basis of in vivo studies, ERK5 is

also dephosphorylated by MKP-1 and MKP-3, which have a preference for ERK1/2 compared to the stress-responsive kinases.¹⁷⁵

### E. Substrates

ERK5 substrates include the MEF2 family members, MEF2A, C, and D, and the Ets-like transcription factor Sap1a.¹⁷⁵,⁴⁵⁴ ERK5 is believed to influence c-Jun-mediated transcription in a MEF2-dependent manner. ERK5 contains a sequence that may function as a possible CD domain, as discussed above for ERK1/2; however, it is not known whether it is required for recognition of substrates by ERK5.

### F. Disruption of a Gene Encoding a Probable Pathway Component

#### 1. MEKK3

To disrupt the MEKK3 gene, sequence encoding a portion of its catalytic domain was removed. The MEKK3⁻/⁻ mice died around E11 apparently of defects in angiogenesis.⁴⁵⁵ A major site of this failure was the labyrinthine layer of the placenta; defective angiogenesis prevented the fetal blood vessels from sufficient contact with the maternal blood supply.

## VI. Orphan MAP Kinases

### A. ERK3 Isoforms

ERK3 is among the least studied of the MAP kinase family because substrates have not been identified. Two ~63k forms, which are being designated ERK3α and ERK3β, are 74% identical in their catalytic domains and appear to be encoded by different genes. Multiple longer forms of ERK3α have been found.⁴⁵⁶,⁴⁵⁷ ERK3 contains a single serine phosphoacceptor site in its activation loop. ERK3α is constitutively localized to the nucleus.⁴⁵⁶ A kinase that may lie upstream of ERK3 in a cascade has been characterized.⁴⁵⁸

### B. ERK7

ERK7 at 61 kDa is 40% identical to ERK2 in the catalytic domain and contains the same TEY motif in the activation loop.⁴⁵⁹ Regulators of ERK7 have not been identified but are likely distinct from those that activate ERK2 or other characterized MAP kinases. Autoregulation has been suggested. Although its substrate specificity is not known in detail, the chloride channel-related protein CLIC3 binds to the noncatalytic tail of ERK7.⁴⁶⁰

### C. NLK

The nemo-like kinase NLK, a homologue of *Drosophila* nemo, is over 45% identical to ERK2 in its kinase domain.⁴⁶¹ Despite this identity, like ERK3 it lacks the tyrosine phosphoacceptor site in the activation loop. Evidence suggesting that NLK must be considered a MAP kinase family member comes from the demonstration that an activated mutant of MEK6 activates NLK. The *C. elegans* NLK homologue lit-1 is downstream of Mom-4, the worm homologue of

### D. MOK

MOK is 30% identical to ERK2.⁴⁶⁴ In contrast to NLK, MOK has both the tyrosine and threonine phosphoacceptor sites in its activation loop. Phorbol ester and okadaic acid increase its activity, making it appear more like ERK1/2 in its sensitivity. Other kinases more similar to MOK are male germ cell associated kinase (MAK) and the MAK-related kinase, MRK.⁴⁶⁵–⁴⁶⁷

## VII. Acknowledgments

The authors thank members of the Cobb laboratory for suggestions and comments about the manuscript and Dionne Ware for administrative assistance. Work from the authors’ laboratory was supported by grants from the National Institutes of Health, the Juvenile Diabetes Foundation, and the Welch Foundation. T.B.G. has been supported by a fellowship from the American Diabetes Association. G.P. has been supported by NIH Pharmacological Sciences Training Grant T32 GM07062.

## VIII. References

(1) Lewis, T. S.; Shapiro, P. S.; Ahn, N. G. *Adv. Cancer Res.* **1998**, 74, 49.

(2) Plowman, G. D.; Sudarsanam, S.; Bingham, J.; Whyte, D.; Hunter, T. *Proc. Natl. Acad. Sci. U.S.A.* **1999**, 96, 13603.

(3) Kultz, D. *J. Mol. Evol.* **1998**, 46, 571.

(4) Caffrey, D. R.; O'Neill, L. A.; Shields, D. C. *J. Mol. Evol.* **1999**, 49, 567.

(5) Caffrey, D. R.; O'Neill, L. A.; Shields, D. C. *Protein Sci.* **2000**, 9, 655.

(6) Payne, D. M.; Rossomando, A. J.; Martino, P.; Erickson, A. K.; Her, J.-H.; Shananowitz, J.; Hunt, D. F.; Weber, M. J.; Sturgill, T. W. *EMBO J.* **1991**, 10, 885.

(7) Robbins, D. J.; Zhen, E.; Owaki, H.; Vanderbilt, C.; Ebert, D.; Geppert, T. D.; Cobb, M. H. *Biol. Chem.* **1993**, 268, 5097.

(8) Cobb, M. H. *Prog. Biophys. Mol. Biol.* **1999**, 71, 479.

(9) Camps, M.; Nichols, A.; Arkinstall, S. *FASEB J.* **2000**, 14, 6.

(10) Sharrocks, A. D.; Yang, S. H.; Galanis, A. *Trends Biochem. Sci.* **2000**, 25, 448.

(11) Whitmarsh, A. J.; Davis, R. J. *Trends Biochem. Sci.* **1998**, 23, 481.

(12) Karandikar, M.; Cobb, M. H. *Cell Calcium* **1999**, 26, 219.

(13) Garrington, T. P.; Johnson, G. L. *Curr. Opin. Cell Biol.* **1999**, 11, 211.

(14) Holland, P. M.; Cooper, J. A. *Curr. Biol.* **1999**, 9, R329.

(15) Levchenko, A.; Bruck, J.; Sternberg, P. W. *Proc. Natl. Acad. Sci. U.S.A.* **2000**, 97, 5818.

(16) Boulton, T. G.; Yancopoulos, G. D.; Gregory, J. S.; Slaughter, C.; Moomaw, C.; Hsu, J.; Cobb, M. H. *Science* **1990**, 249, 64.

(17) Boulton, T. G.; Nye, S. H.; Robbins, D. J.; Ip, N. Y.; Radziejewska, E.; Morgenbesser, S. D.; DePinho, R. A.; Panayotatos, N.; Cobb, M. H.; Yancopoulos, G. D. *Cell* **1991**, 65, 663.

(18) Frödin, M.; Sekine, N.; Roche, E.; Filloux, C.; Prentki, M.; Wollheim, C. B.; Van Obberghen, E. *J. Biol. Chem.* **1995**, 270, 7882.

(19) Khoo, S.; Cobb, M. H. *Proc. Natl. Acad. Sci. U.S.A.* **1997**, 94, 5599.

(20) English, J. D.; Sweatt, J. D. *J. Biol. Chem.* **1996**, 271, 24329.

(21) Atkins, C. M.; Selcher, J. C.; Petraitis, J. J.; Trzaskos, J. M.; Sweatt, J. D. *Nat. Neurosci.* **1998**, 1, 602.

(22) Rossi-Arnaud, C.; Grant, S. G.; Chapman, P. F.; Lipp, H. P.; Sturani, E.; Klein, R. *Nature* **1997**, 390, 281.

(23) Zhang, F.; Strand, A.; Robbins, D.; Cobb, M. H.; Goldsmith, E. *J. Nature* **1994**, 367, 704.

(24) Canagarajah, B. J.; Khokhlatchev, A.; Cobb, M. H.; Goldsmith, E. *Cell* **1997**, 90, 859.

(25) Prowse, C. N.; Hagopian, J. C.; Cobb, M. H.; Ahn, N. G.; Lew, J. *Biochemistry* **2000**, 39, 6258–6266.

(26) Robbins, D. J.; Cobb, M. H. *Mol. Biol. Cell* **1992**, 3, 299.

(27) Ferrell, J. E.; Bhatt, R. R. *J. Biol. Chem.* **1997**, 272, 19008.

(28) Ferrell, J. E. *TIBS* **1997**, 22, 288.

MAP Kinases

(29) Ferrell, J. E. J. BioEssays 1999, 21, 866.

(30) Knighton, D. R.; Zheng, J.; Ten Eyck, L. F.; Ashford, V. A.; Xuong, N.-H.; Taylor, S. S.; Sowadski, J. M. Science 1991, 253, 407.

(31) Knighton, D. R.; Zheng, J.; Ten Eyck, L. F.; Xuong, N.-H.; Taylor, S. S.; Sowadski, J. M. Science 1991, 253, 414.

(32) Zhang, J.; Zhang, F.; Ebert, D.; Cobb, M. H.; Goldsmith, E. J. Structure 1995, 3, 299.

(33) Seger, R.; Seger, D.; Lozeman, F. J.; Ahn, N. G.; Graves, L. M.; Campbell, J. S.; Ericsson, L.; Harrylock, M.; Jensen, A. M.; Krebs, E. G. J. Biol. Chem. 1992, 267, 25628.

(34) Crews, C.; Alessandrini, A.; Erikson, R. Science 1992, 258, 478.

(35) Kosako, H.; Gotoh, Y.; Matsuda, S.; Ishikawa, M.; Nishida, E. EMBO J. 1992, 11, 2903.

(36) Zheng, C.-F.; Guan, K. J. Biol. Chem. 1993, 268, 11435.

(37) Alessi, D. R.; Saito, Y.; Campbell, D. G.; Cohen, P.; Sithanandam, G.; Rapp, U.; Ashworth, A.; Marshall, C. J.; Cowley, S. EMBO J. 1994, 13, 1610.

(38) Zheng, C.-F.; Guan, K. EMBO J. 1994, 13, 1123.

(39) Mansour, S. J.; Candia, J. M.; Matsuura, J. E.; Manning, M. C.; Ahn, N. G. Biochemistry 1996, 35, 15529.

(40) Mansour, S. J.; Matten, W. T.; Hermann, A. S.; Candia, J. M.; Rong, S.; Fukasawa, K.; Vande Woude, G. F.; Ahn, N. G. Science 1994, 265, 966.

(41) Whalen, A. M.; Galasinski, S. C.; Shapiro, P. S.; Nahreini, T. S.; Ahn, N. G. Mol. Cell Biol. 1997, 17, 1947.

(42) Lee, H. Y.; Suh, Y.; Robinson, M. J.; Clifford, J. L.; Hong, W. K.; Woodgett, J. R.; Cobb, M. H.; Mangelsdorf, D. J.; Kurie, J. M. J. Biol. Chem. 2000, 275, 32193.

(43) Tanoue, T.; Adachi, M.; Moriguchi, T.; Nishida, E. Nat. Cell Biol. 2000, 2, 110.

(44) Xu, B.; Wilsbacher, J. L.; Collisson, T.; Cobb, M. H. J. Biol. Chem. 1999, 274, 34029.

(45) Yang, S. H.; Whitmarsh, A. J.; Davis, R. J.; Sharrocks, A. D. EMBO J. 1998, 17, 1740.

(46) Rubinfeld, H.; Hanoch, T.; Seger, R. J. Biol. Chem. 1999, 274, 30349.

(47) Duesbery, N. S.; Webb, C. P.; Leppla, S. H.; Gordon, V. M.; Klimpel, K. R.; Copeland, T. D.; Ahn, N. G.; Oskarsson, M. K.; Fukasawa, K.; Paull, K. D.; Vande Woude G. F. Science 1998, 280, 734.

(48) Gotoh, I.; Fukuda, M.; Adachi, M.; Nishida, E. J. Biol. Chem. 1999, 274, 11874.

(49) Catling, A. D.; Schaeffer, H.-J.; Reuter, C. W. M.; Reddy, G. R.; Weber, M. J. Mol. Cell. Biol. 1995, 15, 5214.

(50) Dang, A.; Frost, J. A.; Cobb, M. H. J. Biol. Chem. 1998, 273, 19909.

(51) Mansour, S. J.; Resing, K. A.; Candia, J. M.; Hermann, A. S.; Gloor, J. W.; Herskind, K. R.; Wartmann, M.; Davis, R. J.; Ahn, N. G. J. Biochem. 1994, 116, 304.

(52) Gardner, A. M.; Vaillancourt, R. R.; Lange-Carter, C. A.; Johnson, G. L. Mol. Biol. Cell 1994, 5, 193.

(53) Cobb, M. H.; Xu, S.; Cheng, M.; Ebert, D.; Robbins, D.; Goldsmith, E.; Robinson, M. Adv. Pharmacol. (San Diego) 1996, 7, 49.

(54) Frost, J. A.; Steen, H.; Shapiro, P. S.; Lewis, R.; Ahn, J.; Shaw, P. E.; Cobb, M. H. EMBO J. 1997, 16, 6426.

(55) Schaeffer, H.J.; Catling, A. D.; Eblen, S. T.; Collier, L. S.; Krauss, A.; Weber, M. J. Science 1998, 281, 1668.

(56) Nantel, A.; Mohammad-Ali, K.; Sherk, J.; Posner, B. I.; Thomas, D. Y. J. Biol. Chem. 1998, 273, 10475.

(57) Nantel, A.; Huber, M.; Thomas, D. Y. J. Biol. Chem. 1999, 274, 35719.

(58) Kyriakis, J. M.; App, H.; Zhang, X.-F.; Banerjee, P.; Brautigan, D. L.; Rapp, U. R.; Avruch, J. Nature 1992, 358, 417.

(59) Dent, P.; Haser, W.; Haystead, T. A. J.; Vincent, L. A.; Roberts, T. M.; Sturgill, T. W. Science 1992, 257, 1404.

(60) Force, T.; Bonventre, J. V.; Heidecker, G.; Rapp, U.; Avruch, J.; Kyriakis, J. M. Proc. Natl. Acad. Sci. U.S.A. 1994, 91, 1270.

(61) Leevers, S. J.; Paterson, H. F.; Marshall, C. J. Nature 1994, 369, 411.

(62) Moodie, S. A.; Willumsen, B. M.; Weber, M. J.; Wolfman, A. Science 1993, 260, 1658.

(63) Vojtek, A. B.; Hollenberg, S. M.; Cooper, J. A. Cell 1993, 74, 205.

(64) Zhang, X.; Settleman, J.; Kyriakis, J. M.; Takeuchi-Suzuki, E.; Elledge, S. J.; Marshall, M. S.; Bruder, J. T.; Rapp, U. R.; Avruch, J. Nature 1993, 364, 308.

(65) Warne, P. H.; Viciana, P. R.; Downward, J. Nature 1993, 364, 353.

(66) Dent, P.; Reardon, D. B.; Morrison, D. K.; Sturgill, T. W. Mol. Cell Biol. 1995, 15, 4125.

(67) Morrison, D. K.; Heidecker, G.; Rapp, U. R.; Copeland, T. D. J. Biol. Chem. 1993, 268, 17309.

(68) Fabian, J. R.; Daar, I. O.; Morrison, D. K. Mol. Cell. Biol. 1993, 13, 7170.

(69) Kolch, W.; Heldecker, G.; Kochs, G.; Hummel, R.; Vahidi, H.; Mischak, H.; Finkenzeller, G.; Marmé, D.; Rapp, U. R. Nature 1993, 364, 249.


Chemical Reviews, 2001, Vol. 101, No. 8 2471

(70) Diaz, B.; Barnard, D.; Filson, A.; MacDonald, S.; King, A.; Marshall, M. Mol. Cell Biol. 1997, 17, 4509.

(71) Chaudhary, A.; King, W. G.; Mattaliano, M. D.; Frost, J. A.; Diaz, B.; Morrison, D. K.; Cobb, M. H.; Marshall, M. S.; Brugge, J. S. Curr. Biol. 2000, 10, 551.

(72) Mason, C. S.; Springer, C. J.; Cooper, R. G.; Superti-Furga, G.; Marshall, C. J.; Marais, R. EMBO J. 1999, 18, 2137.

(73) Sidovar, M. F.; Kozlowski, P.; Lee, J. W.; Collins, M. A.; He, Y.; Graves, L. M. J. Biol. Chem. 2000, 275, 28688.

(74) Thorson, J. A.; Yu, L. W.; Hsu, A. L.; Shih, N. Y.; Graves, P. R.; Tanner, J. W.; Allen, P. M.; Piwnica-Worms, H.; Shaw, A. S. Mol. Cell Biol. 1998, 18, 5229.

(75) Muslin, A. J.; Tanner, J. W.; Allen, P. M.; Shaw, A. S. Cell 1996, 84, 889.

(76) Yaffe, M. B.; Rittinger, K.; Volinia, S.; Caron, P. R.; Aitken, A.; Leffers, H.; Gamblin, S. J.; Smerdon, S. J.; Cantley, L. C. Cell 1997, 91, 961.

(77) Stancato, L. F.; Chow, Y. H.; Hutchison, K. A.; Perdew, G. H.; Jove, R.; Pratt, W. B. J. Biol. Chem. 1993, 268, 21711.

(78) Fantl, W. J.; Muslin, A. J.; Kikuchi, A.; Martin, J. A.; MacNicol, A. M.; Gross, R. W.; Williams, L. T. Nature 1994, 371, 612.

(79) Freed, E.; Symons, M.; MacDonald, S. G.; McCormick, F.; Ruggieri, R. Science 1994, 265, 1713.

(80) Michaud, N. R.; Fabian, J. R.; Mathes, K. D.; Morrison, D. K. Mol. Cell. Biol. 1995, 15, 3390.

(81) Stancato, L. F.; Silverstein, A. M.; Owens-Grillo, J. K.; Chow, Y. H.; Jove, R.; Pratt, W. B. J. Biol. Chem. 1997, 272, 4013.

(82) Schulte, T. W.; Blagosklonny, M. V.; Romanova, L.; Mushinski, J. F.; Monia, B. P.; Johnston, J. F.; Nguyen, P.; Trepel, J.; Neckers, L. M. Mol. Cell Biol. 1996, 16, 5839.

(83) Tzivion, G.; Luo, Z.; Avruch, J. Nature 1998, 394, 88.

(84) Jaiswal, R. K.; Weissinger, E.; Kolch, W.; Landreth, G. E. J. Biol. Chem. 1996, 271, 23626.

(85) Zimmermann, S.; Moelling, K. Science 1999, 286, 1741.

(86) Rommel, C.; Clarke, B. A.; Zimmermann, S.; Nunez, L.; Rossman, R.; Reid, K.; Moelling, K.; Yancopoulos, G. D.; Glass, D. J. Science 1999, 286, 1738.

(87) Guan, K. L.; Figueroa, C.; Brtva, T. R.; Zhu, T.; Taylor, J.; Barber, T. D.; Vojtek, A. B. J. Biol. Chem. 2000, 275, 27354.

(88) Schulte, T. W.; Blagosklonny, M. V.; Ingui, C.; Neckers, L. J. Biol. Chem. 1995, 270, 24585.

(89) Vossler, M. R.; Yao, H.; York, R. D.; Pan, M. G.; Rim, C. S.; Stork, P. J. Cell 1997, 89, 73.

(90) Seidel, M. G.; Klinger, M.; Freissmuth, M.; Ille, C. J. Biol. Chem. 1999, 274, 25833.

(91) de Rooij, J.; Zwartkruis, F. J.; Verheijen, M. H.; Cool, R. H.; Nijman, S. M.; Wittinghofer, A.; Bos, J. L. Nature 1998, 396, 474.

(92) Zwartkruis, F. J.; Wolthuis, R. M.; Nabben, N. M.; Franke, B.; Bos, J. L. EMBO J. 1998, 17, 5905.

(93) Erhardt, P.; Troppmair, J.; Rapp, U. R.; Cooper, G. M. Mol. Cell Biol. 1995, 15, 5524.

(94) Okada, T.; Hu, C. D.; Jin, T. G.; Kariya, K.; Yamawaki-Kataoka, Y.; Kataoka, T. Mol. Cell Biol. 1999, 19, 6057.

(95) Rosario, M.; Paterson, H. F.; Marshall, C. J. EMBO J. 1999, 18, 1270.

(96) Pearson, G.; Bumeister, R.; Henry, D. O.; Cobb, M. H.; White, M. A. J. Biol. Chem. 2000, 275, 37303.

(97) Yeung, K.; Seitz, T.; Li, S.; Janosch, P.; McFerran, B.; Kaiser, C.; Fee, F.; Katsanakis, K. D.; Rose, D. W.; Mischak, H.; Sedivy, J. M.; Kolch, W. Nature 1999, 401, 173.

(98) Pawson, T.; Scott, J. D. Science 1997, 278, 2075.

(99) Hunter, T. Cell 1995, 80, 225.

(100) Mischak, H.; Seitz, T.; Janosch, P.; Eulitz, M.; Steen, H.; Schellerer, M.; Philipp, A.; Kolch, W. Mol. Cell Biol. 1996, 16, 5409.

(101) Kikuchi, A.; Williams, L. T. J. Biol. Chem. 1996, 271, 588.

(102) Grewal, S. S.; Horgan, A. M.; York, R. D.; Withers, G. S.; Banker, G. A.; Stork, P. J. J. Biol. Chem. 2000, 275, 3722.

(103) Kawasaki, H.; Springett, G. M.; Mochizuki, N.; Toki, S.; Nakaya, M.; Matsuda, M.; Housman, D. E.; Graybiel, A. M. Science 1998, 282, 2275.

(104) Daaka, Y.; Luttrel, L. M.; Lefkowitz, R. J. Nature 1997, 390, 88.

(105) Ma, Y. C.; Huang, J.; Ali, S.; Lowry, W.; Huang, X. Y. Cell 2000, 102, 635.

(106) Hedin, K. E.; Bell, M. P.; Huntoon, C. J.; Karnitz, L. M.; McKean, D. J. J. Biol. Chem. 1999, 274, 19992.

(107) Luttrel, L. M.; Hawes, B. E.; Vanbiesen, T.; Luttrel, D. K.; Lansing, T. J.; Lefkowitz, R. J. J. Biol. Chem. 1996, 271, 19443.

(108) Lopez-Ilasaca, M.; Crespo, P.; Pellici, P. G.; Gutkind, J. S.; Wetzker, R. Science 1997, 275, 394.

(109) Dikic, I.; Tokiwa, G.; Lev, S.; Courtneidge, S. A.; Schlessinger, J. Nature 1996, 383, 547.

(110) Dell'aroccia GJ.; Maudsley, S.; Daaka, Y.; Lefkowitz, R. J.; Luttrel, L. M. J. Biol. Chem. 1999, 274, 13978.

(111) Lev, S.; Moreno, H.; Martinez, R.; Canoll, P.; Peles, E.; Musachio, J. M.; Plowman, G. D.; Rudy, B.; Schlessinger, J. Nature 1995, 376, 737.

(112) Collins, L. R.; Minden, A.; Karin, M.; Brown, J. H. *J. Biol. Chem.* 1996, 271, 17349.

(113) Wilk-Blaszczak, M. A.; Stein, B.; Xu, S.; Barbosa, M. S.; Cobb, M. H.; Belardetti, F. *J. Neurosci.* 1998, 18, 112.

(114) Migliaccio, A.; Di Domenico, M.; Castoria, G.; Defalco, A.; Bon-tempo, P.; Nola, E.; Auricchio, F. *EMBO J.* 1996, 15, 1292.

(115) Castoria, G.; Barone, M. V.; Di Domenico, M.; Bilancio, A.; Ametrano, D.; Migliaccio, A.; Auricchio, F. *EMBO J.* 1999, 18, 2500.

(116) Migliaccio, A.; Piccolo, D.; Castoria, G.; Di Domenico, M.; Bilancio, A.; Lombardi, M.; Gong, W.; Beato, M.; Auricchio, F. *EMBO J.* 1998, 17, 2008.

(117) Improta-Brears, T.; Whorton, A. R.; Codazzi, F.; York, J. D.; Meyer, T.; McDonnell, D. P. *Proc. Natl. Acad. Sci. U.S.A.* 1999, 96, 4686.

(118) Singh, M.; Setalo, G. J.; Guan, X.; Warren, M.; Toran-Allerand, C. D. *J. Neurosci.* 1999, 19, 1179.

(119) Watters, J. J.; Campbell, J. S.; Cunningham, M. J.; Krebs, E. G.; Dorsa, D. M. *Endocrinology* 1997, 138, 4030.

(120) Razandi, M.; Pedram, A.; Greene, G. L.; Levin, E. R. *Mol. Endocrinol.* 1999, 13, 307.

(121) Chen, Z.; Yuhanna, I. S.; Galcheva-Gargova, Z.; Karas, R. H.; Mendelsohn, M. E.; Shaul, P. W. *J. Clin. Invest.* 1999, 103, 401.

(122) Zhang, C. C.; Shapiro, D. *J. Biol. Chem.* 2000, 275, 479.

(123) Singer, C. A.; Figueroa-Masot, X. A.; Batchelor, R. H.; Dorsa, D. M. *J. Neurosci.* 1999, 19, 2455.

(124) Bi, R.; Broutman, G.; Foy, M. R.; Thompson, R. F.; Baudry, M. *Proc. Natl. Acad. Sci. U.S.A.* 2000, 97, 3602.

(125) Davis, P. J.; Shih, A.; Lin, H. Y.; Martino, L. J.; Davis, F. B. *J. Biol. Chem.* 2000, 275, 38032.

(126) Song, X.; Bishop, J. E.; Okamura, W. H.; Norman, A. W. *Endocrinology* 1998, 139, 457.

(127) Norman, A. W.; Wehling, M. *Steroids* 1999, 64, 3.

(128) Therrien, M.; Chang, H. C.; Solomon, N. M.; Karim, F. D.; Wassarman, D. A.; Rubin, G. M. *Cell* 1995, 83, 879.

(129) Sundaram, M.; Han, M. *Cell* 1995, 83, 889.

(130) Kornfeld, K.; Hom, D. B.; Horvitz, H. R. *Cell* 1995, 83, 903.

(131) Therrien, M.; Michaud, N. R.; Rubin, G. M.; Morrison, D. K. *Genes Dev.* 1996, 10, 2684.

(132) Michaud, N. R.; Therrien, M.; Cacace, A.; Edsall, L. C.; Spiegel, S.; Rubin, G. M.; Morrison, D. K. *Proc. Natl. Acad. Sci. U.S.A.* 1997, 94, 12792.

(133) Yu, W.; Fantl, W. J.; Harrowe, G.; Williams, L. T. *Curr. Biol.* 1998, 8, 56.

(134) Denouel-Galy, A.; Douville, E. M.; Warne, P. H.; Papin, C.; Laugier, D.; Calothy, G.; Downward, J.; Eychene, A. *Curr. Biol.* 1998, 8, 46.

(135) Stewart, S.; Sundaram, M.; Zhang, Y.; Lee, J.; Han, M.; Guan, K. L. *Mol. Cell Biol.* 1999, 19, 5523.

(136) Bell, B.; Xing, H.; Yan, K.; Gautam, N.; Muslin, A. *J. Biol. Chem.* 1999, 274, 7982.

(137) Sieburth, D. S.; Sun, Q.; Han, M. *Cell* 1998, 94, 119.

(138) Li, W.; Han, M.; Guan, K. L. *Genes Dev.* 2000, 14, 895.

(139) Therrien, M.; Wong, A. M.; Rubin, G. M. *Cell* 1998, 95, 343.

(140) Therrien, M.; Wong, A. M.; Kwan, E.; Rubin, G. M. *Proc. Natl. Acad. Sci. U.S.A.* 1999, 96, 13259.

(141) Yao, I.; Ohtsuka, T.; Kawabe, H.; Matsuura, Y.; Takai, Y.; Hata, Y. *Biochem. Biophys. Res. Commun.* 2000, 270, 538.

(142) Karandikar, M.; Xu, S.; Cobb, M. H. *J. Biol. Chem.* 2000, 275, 40120.

(143) Christerson, L. B.; Vanderbilt, C. A.; Cobb, M. H. *Cell Motil. Cytoskeleton* 1999, 43, 186.

(144) Yujiri, T.; Sather, S.; Fanger, G. R.; Johnson, G. L. *Science* 1998, 282, 1911.

(145) Xia, Y.; Makris, C.; Su, B.; Li, E.; Yang, J.; Nemerow, G. R.; Karin, M. *Proc. Natl. Acad. Sci. U.S.A.* 2000, 97, 5243.

(146) Reszka, A. A.; Seger, R.; Diltz, C. D.; Krebs, E. G.; Fischer, E. H. *Proc. Natl. Acad. Sci. U.S.A.* 1995, 92, 8881.

(147) Reszka, A. A.; Bulinski, J. C.; Krebs, E. G.; Fischer, E. H. *Mol. Cell Biol.* 1997, 8, 1219.

(148) Chen, R.-H.; Sarnecki, C.; Blenis, J. *Mol. Cell. Biol.* 1992, 12, 915.

(149) Lenormand, P.; Brondello, J. M.; Brunet, A.; Pouyssegur, J. *J. Cell Biol.* 1998, 142, 625.

(150) Khokhlatchev, A.; Canagarajah, B.; Wilsbacher, J. L.; Robinson, M.; Atkinson, M.; Goldsmith, E.; Cobb, M. H. *Cell* 1998, 93, 605.

(151) Mineo, C.; Anderson, R. W.; White, M. A. *J. Biol. Chem.* 1997, 272, 10345.

(152) Furuchi, T.; Anderson, R. G. *J. Biol. Chem.* 1998, 273, 21099.

(153) Menice, C. B.; Hulvershorn, J.; Adam, L. P.; Wang, C. A.; Morgan, K. G. *J. Biol. Chem.* 1997, 272, 25157.

(154) Cowley, S.; Paterson, H.; Kemp, P.; Marshall, C. *J. Cell* 1994, 77, 841.

(155) Robinson, M. J.; Stippec, S. A.; Goldsmith, E.; White, M. A.; Cobb, M. H. *Curr. Biol.* 1998, 8, 1141.

(156) Kim-Kaneyama, J.; Nose, K.; Shibanuma, M. *J. Biol. Chem.* 2000, 275, 20685.

(157) Fukuda, M.; Gotoh, Y.; Nishida, E. *EMBO J.* 1997, 16, 1901.

(158) Adachi, M.; Fukuda, M.; Nishida, E. *EMBO J.* 1999, 18, 5347.

(159) Adachi, M.; Fukuda, M.; Nishida, E. *J. Cell Biol.* 2000, 148, 849.

(160) Fukuda, M.; Gotoh, I.; Adachi, M.; Gotoh, Y.; Nishida, E. *J. Biol. Chem.* 1997, 272, 32642.

(161) Fukuda, M.; Gotoh, I.; Gotoh, Y.; Nishida, E. *J. Biol. Chem.* 1996, 271, 20024.

(162) Zheng, C.-F.; Guan, K. *J. Biol. Chem.* 1994, 269, 19947.

(163) Lenormand, P.; Sardet, C.; Pages, G.; L’Allemand, G.; Brunet, A.; Pouysségur, J. *J. Cell Biol.* 1993, 122, 1079.

(164) Tolwinski, N. S.; Shapiro, P. S.; Goueli, S.; Ahn, N. G. *J. Biol. Chem.* 1999, 274, 6168.

(165) Leinweber, B. D.; Leavis, P. C.; Grabarek, Z.; Wang, C. L.; Morgan, K. G. *Biochem. J.* 1999, 344, 117.

(166) Blanco-Aparicio, C.; Torres, J.; Pulido, R. *J. Cell Biol.* 1999, 147, 1129.

(167) Shapiro, P. S.; Whalen, A. M.; Tolwinski, N. S.; Froelich-Ammon, S. J.; Garcia, M.; Osheroff, N.; Ahn, N. G. *Mol. Cell Biol.* 1999, 19, 3551.

(168) Shapiro, P. S.; Vaisberg, E.; Hunt, A. J.; Tolwinski, N. S.; Whalen, A. M.; McIntosh, J. R.; Ahn, N. G. *J. Cell Biol.* 1998, 142, 1533.

(169) Zecevic, M.; Catling, A. D.; Eblen, S. T.; Renzi, L.; Hittle, J. C.; Yen, T. J.; Gorbsky, G. J.; Weber, M. *J. J. Cell Biol.* 1998, 142, 1547.

(170) Olsen, M. K.; Reszka, A. A.; Abraham, I. *J. Cell. Physiol.* 1998, 176, 525.

(171) Dudley, D. T.; Pang, L.; Decker, S. J.; Bridges, A. J.; Saltiel, A. R. *Proc. Natl. Acad. Sci. U.S.A.* 1995, 92, 7686.

(172) Alessi, D. R.; Cuenda, A.; Cohen, P.; Dudley, D. T.; Saltiel, A. R. *J. Biol. Chem.* 1995, 270, 27489.

(173) Favata, M. F.; Horiuchi, K. Y.; Manos, E. J.; Daulerio, A. J.; Stradley, D. A.; Feesser, W. S.; Van Dyk, D. E.; Pitts, W. J.; Earl, R. A.; Hobbs, F.; Copeland, R. A.; Magolda, R. L.; Scherle, P. A.; Trzaskos, J. M. *J. Biol. Chem.* 1998, 273, 18623.

(174) Akashi, M.; Nishida, E. *Genes Dev.* 2000, 14, 645.

(175) Kamakura, S.; Moriguchi, T.; Nishida, E. *J. Biol. Chem.* 1999, 274, 26563.

(176) Sebolt-Leopold, J. S.; Dudley, D. T.; Herrera, R.; Van Becelaere, K.; Wiland, A.; Gowan, R. C.; Tecle, H.; Barrett, S. D.; Bridges, A.; Przybranowski, S.; Leopold, W. R.; Saltiel, A. R. *Nat. Med.* 1999, 5, 810.

(177) Yang, S. H.; Yates, P. R.; Whitmarsh, A. J.; Davis, R. J.; Sharrocks, A. D. *Mol. Cell Biol.* 1998, 18, 710.

(178) Yang, S. H.; Galanis, A.; Sharrocks, A. D. *Mol. Cell Biol.* 1999, 19, 4028.

(179) Jacobs, D.; Glossip, D.; Xing, H.; Muslin, A. J.; Kornfeld, K. *Genes Dev.* 1999, 13, 163.

(180) Smith, J. A.; Poteet-Smith, C. E.; Malarkey, K.; Sturgill, T. W. *J. Biol. Chem.* 1999, 274, 2893.

(181) Gavin, A. C.; Nebreda, A. R. *Curr. Biol.* 1999, 9, 281.

(182) Sturgill, T. W.; Ray, L. B.; Erikson, E.; Maller, J. *Nature* 1988, 334, 715.

(183) Stokoe, D.; Campbell, D. G.; Nakielny, S.; Hidaka, H.; Leavers, S. J.; Marshall, C.; Cohen, P. *EMBO J.* 1992, 11, 3985.

(184) Waskiewicz, A. J.; Flynn, A.; Proud, C. G.; Cooper, J. A. *EMBO J.* 1997, 16, 1909.

(185) Fukunaga, R.; Hunter, T. *EMBO J.* 1997, 16, 1921.

(186) Zhao, Y.; Bjørbaek, C.; Møller, D. E. *J. Biol. Chem.* 1996, 271, 29773.

(187) Hsiao, K.-M.; Chou, S.; Shih, S.-J.; Ferrell, J. E., Jr. *Proc. Natl. Acad. Sci. U.S.A.* 1994, 91, 5480.

(188) Rivera, V. M.; Miranti, C. K.; Misra, R. P.; Ginty, D. D.; Chen, R. H.; Blenis, J.; Greenberg, M. E. *Mol. Cell Biol.* 1993, 13, 6260.

(189) Pende, M.; Fisher, T. L.; Simpson, P. B.; Russell, J. T.; Blenis, J.; Gallo, V. *J. Neurosci.* 1997, 17, 1291.

(190) Chen, R. H.; Juo, P. C.; Curran, T.; Blenis, J. *Oncogene* 1996, 12, 1493.

(191) Chen, R. H.; Abate, C.; Blenis, J. *Proc. Natl. Acad. Sci. U.S.A.* 1993, 90, 10952.

(192) Joel, P. B.; Smith, J.; Sturgill, T. W.; Fisher, T. L.; Blenis, J.; Lannigan, D. A. *Mol. Cell Biol.* 1998, 18, 1978.

(193) Lin, L. L.; Wartmann, M.; Lin, A. Y.; Knopf, J. L.; Seth, A.; Davis, R. J. *Cell* 1993, 72, 269.

(194) Alvarez, E.; Northwood, I. C.; Gonzalez, F. A.; Latour, D. A.; Seth, A.; Abate, C.; Curran, T.; Davis, R. J. *J. Biol. Chem.* 1991, 266, 15277.

(195) Cottin, V.; Van Linden, A.; Riches, D. W. *J. Biol. Chem.* 1999, 274, 32975.

(196) Van Linden, A. A.; Cottin, V.; Leu, C.; Riches; DW *J. Biol. Chem.* 2000, 275, 6996.

(197) Zhang, J.; Berenstein, E. H.; Evans, R. L.; Siraganian, R. P. *J. Exp. Med.* 1996, 184, 71.

(198) Xu, R.; Seger, R.; Pecht, I. *J. Immunol.* 1999, 163, 1110.

(199) Ogier-Denis, E.; Pattingre, S.; El Benna, J.; Codogno, P. *J. Biol. Chem.* 2000, 275, 39090.

(200) Chevet, E.; Wong, H. N.; Gerber, D.; Cochet, C.; Fazel, A.; Cameron; Gushue, J. N.; Thomas, D. Y.; Bergeron, J. J. *EMBO J.* 1999, 18, 3655.

(201) Hoffmann, R.; Baillie, G. S.; MacKenzie, S. J.; Yarwood, S. J.; Houslay, M. D. *EMBO J.* 1999, 18, 893.

(202) Veeranna; Amin, N. D.; Ahn, N. G.; Jaffe, H.; Winters, C. A.; Grant, P.; Pant, H. C. *J. Neurosci.* 1998, 18, 4008.

(203) Ku, H.; Meier, K. E. *J. Biol. Chem.* 2000, 275, 11333.

(204) Graves, L. M.; Guy, H. I.; Kozlowski, P.; Huang, M.; Lazarowski, E.; Pope, R. M.; Collins, M. A.; Dahlstrand, E. N.; Earp, H. S.; Evans, D. R. *Nature* 2000, 403, 328.

(205) Smith, C. L.; Onate, S. A.; Tsai, M. J.; O'Malley, B. W. *Proc. Natl. Acad. Sci. U.S.A.* 1996, 93, 8884.

(206) Spencer, T. E.; Jenster, G.; Burcin, M. M.; Allis, C. D.; Zhou, J.; Mizzen, C. A.; McKenna, N. J.; Onate, S. A.; Tsai, S. Y.; Tsai, M. J.; O'Malley, B. W. *Nature* 1997, 389, 194.

(207) Rowan, B. G.; Weigel, N. L.; O'Malley, B. W. *J. Biol. Chem.* 2000, 275, 4475.

(208) Walther, C.; Gruss, P. *Development* 1991, 113, 1435.

(209) Mikkola, I.; Bruun, J. A.; Bjorkoy, G.; Holm, T.; Johansen, T. J. *Biol. Chem.* 1999, 274, 15115.

(210) Porter, C. M.; Havens, M. A.; Clipstone, N. A. *J. Biol. Chem.* 2000, 275, 3543.

(211) Chow, C. W.; Dong, C.; Flavell, R. A.; Davis, R. J. *Mol. Cell Biol.* 2000, 20, 5227.

(212) Gomez, d. A.; Martinez-Martinez, S.; Maldonado, J. L.; Ortega-Perez, I.; Redondo, J. M. *J. Biol. Chem.* 2000, 275, 13872.

(213) Pircher, T. J.; Petersen, H.; Gustafsson, J. A.; Haldosen, L. A. *Mol. Endocrinol.* 1999, 13, 555.

(214) Zhong, S. P.; Ma, W. Y.; Dong, Z. *J. Biol. Chem.* 2000, 275, 20980.

(215) Sassone-Corsi, P.; Mizzen, C. A.; Cheung, P.; Crosio, C.; Monaco, L.; Jacquot, S.; Hanauer, A.; Allis, C. D. *Science* 1999, 285, 886.

(216) Thomson, S.; Clayton, A. L.; Hazzalin, C. A.; Rose, S.; Barratt, M. J.; Mahadevan, L. C. *EMBO J.* 1999, 18, 4779.

(217) Wojnowski, L.; Stancato, L. F.; Zimmer, A. M.; Hahn, H.; Beck, T. W.; Larner, A. C.; Rapp, U. R.; Zimmer, A. *Mech. Dev.* 1998, 76, 141.

(218) Wojnowski, L.; Zimmer, A. M.; Beck, T. W.; Hahn, H.; Bernal, R.; Rapp, U. R.; Zimmer, A. *Nat. Genet.* 1997, 16, 293.

(219) Wojnowski, L.; Stancato, L. F.; Larner, A. C.; Rapp, U. R.; Zimmer, A. *Mech. Dev.* 2000, 91, 97.

(220) Pritchard, C. A.; Bolin, L.; Slattery, R.; Murray, R.; McMahon, M. *Curr. Biol.* 1996, 6, 614.

(221) Giroux, S.; Tremblay, M.; Bernard, D.; Cardin-Girard, J. F.; Aubry, S.; Larouche, L.; Rousseau, S.; Huot, J.; Landry, J.; Jeannotte, L.; Charron, J. *Curr. Biol.* 1999, 9, 369.

(222) Pages, G.; Guerin, S.; Grall, D.; Bonino, F.; Smith, A.; Anjuere, F.; Auberger, P.; Pouyssegur, J. *Science* 1999, 286, 1374.

(223) Kyriakis, J. M.; Banerjee, P.; Nikolakaki, E.; Dai, T.; Rubie, E. A.; Ahmad, M. F.; Avruch, J.; Woodgett, J. R. *Nature* 1994, 369, 156.

(224) Derijard, B.; Hibi, M.; Wu, I.-H.; Barrett, T.; Su, B.; Deng, T.; Karin, M.; Davis, R. J. *Cell* 1994, 76, 1025.

(225) Gupta, S.; Barrett, T.; Whitmarsh, A. J.; Cavanagh, J.; Sluss, H. K.; Derijard, B.; Davis, R. J. *EMBO J.* 1996, 15, 2760.

(226) Otto, I. M.; Raabe, T.; Rennefahrt, U. E.; Bork, P.; Rapp, U. R.; Kerkhoff, E. *Curr. Biol.* 2000, 10, 345.

(227) Xie, X.; Gu, Y.; Fox, T.; Coll, J. T.; Fleming, M. A.; Markland, W.; Caron, P. R.; Wilson, K. P.; Su, M. S. *Structure* 1998, 6, 983.

(228) Wilson, K. P.; Fitzgibbon, M. J.; Caron, P. R.; Griffith, J. P.; Chen, W.; McCaffrey, P. G.; Chambers, S. P.; Su, M. S. *J. Biol. Chem.* 1996, 271, 27696.

(229) Wang, Z.; Harkins, P. C.; Ulevitch, R. J.; Han, J.; Cobb, M. H.; Goldsmith, E. J. *Proc. Natl. Acad. Sci. U.S.A.* 1997, 94, 2327.

(230) Taylor, S. S.; Radzio-Andzelm, E. *Curr. Opin. Chem. Biol.* 1997, 1, 219.

(231) Meier, R.; Rouse, J.; Cuenda, A.; Nebreda, A. R.; Cohen, P. *Eur. J. Biochem.* 1996, 236, 796.

(232) Lawler, S.; Fleming, Y.; Goedert, M.; Cohen, P. *Curr. Biol.* 1998, 8, 1387.

(233) Lisnock, J.; Griffin, P.; Calaycay, J.; Frantz, B.; Parsons, J.; O'Keefe, S. J.; LoGrasso, P. *Biochemistry* 2000, 39, 3141.

(234) Khokhlatchev, A.; Xu, S.; English, J.; Wu, P.; Schaefer, E.; Cobb, M. H. *J. Biol. Chem.* 1997, 272, 11057.

(235) Rhodes, N.; Connell, L.; Errede, B. *Genes Dev.* 1990, 4, 1862.

(236) Ramer, S. W.; Davis, R. W. *Proc. Natl. Acad. Sci. U.S.A.* 1993, 90, 452.

(237) Leberer, E.; Dignard, D.; Harcus, D.; Thomas, D. Y.; Whiteway, M. *EMBO J.* 1992, 11, 4815.

(238) Lange-Carter, C. A.; Pleiman, C. M.; Gardner, A. M.; Blumer, K. J.; Johnson, G. L. *Science* 1993, 260, 315.

(239) Xu, S.; Robbins, D. J.; Christerson, L. B.; English, J. M.; Vanderbilt, C. A.; Cobb, M. H. *Proc. Natl. Acad. Sci. U.S.A.* 1996, 93, 5291.

(240) Blank, J. L.; Gerwins, P.; Elliott, E. M.; Sather, S.; Johnson, G. L. *J. Biol. Chem.* 1996, 271, 5361.

(241) Gerwins, P.; Blank, J. L.; Johnson, G. L. *J. Biol. Chem.* 1997, 272, 8288.

(242) Rana, A.; Gallo, K.; Godowski, P.; Hirai, S.; Ohno, S.; Zon, L. I.; Kyriakis, J. M.; Avruch, J. *J. Biol. Chem.* 1996, 271, 19025.

(243) Tibbles, L. A.; Ing, Y. L.; Kiefer, F.; Chan, J.; Iscove, N.; Woodgett, J. R.; Lassam, N. J. *EMBO J.* 1996, 15, 7026.

(244) Chou, M. M.; Hanafusa, H. *J. Biol. Chem.* 1995, 270, 7359.

(245) Graves, J. D.; Gotoh, Y.; Draves, K. E.; Ambrose, D.; Han, D. K.; Wright, M.; Chernoff, J.; Clark, E. A.; Krebs, E. G. *EMBO J.* 1998, 17, 2224.

(246) Salmerón, A.; Ahmad, T. B.; Carlile, G. W.; Pappin, D.; Narasimhan, R. P.; Ley, S. C. *EMBO J.* 1996, 15, 817.

(247) Fan, G.; Merritt, S. E.; Kortenjann, M.; Shaw, P. E.; Holzman, L. R. *J. Biol. Chem.* 1996, 271, 24788.

(248) Hirai, S.; Katoh, M.; Terada, M.; Kyriakis, J. M.; Zon, L. I.; Rana, A.; Avruch, J.; Ohno, S. *J. Biol. Chem.* 1997, 272, 15167.

(249) Yamaguchi, K.; Shirakabe, K.; Shibuya, H.; Irie, K.; Oishi, I.; Ueno, N.; Taniguchi, T.; Nishida, E.; Matsumoto, K. *Science* 1995, 270, 2008.

(250) Ichijo, H.; Nishida, E.; Irie, K.; ten Dijke, P.; Saitoh, M.; Moriguchi, T.; Takagi, M.; Matsumoto, K.; Miyazono, K.; Gotoh, Y. *Science* 1997, 275, 90.

(251) Wang, X. S.; Diener, K.; Jannuzzi, D.; Trollinger, D.; Tan, T. H.; Lichenstein, H.; Zukowski, M.; Yao, Z. *J. Biol. Chem.* 1996, 271, 31607.

(252) Wang, X. S.; Diener, K.; Tan, T. H.; Yao, Z. *Biochem. Biophys. Res. Commun.* 1998, 253, 33.

(253) Zhang, W.; Chen, T.; Wan, T.; He, L.; Li, N.; Yuan, Z.; Cao, X. *Biochem. Biophys. Res. Commun.* 2000, 274, 872.

(254) Hutchison, M.; Berman, K.; Cobb, M. H. *J. Biol. Chem.* 1998, 273, 28625.

(255) Chen, Z.; Hutchison, M.; Cobb, M. H. *J. Biol. Chem.* 1999, 274, 28803.

(256) Minden, A.; Lin, A.; Claret, F.-X.; Abo, A.; Karin, M. *Cell* 1995, 81, 1147.

(257) Coso, O. A.; Chiariello, M.; Yu, J.-C.; Teramoto, H.; Crespo, P.; Xu, N.; Miki, T.; Gutkind, J. S. *Cell* 1995, 81, 1137.

(258) Polverino, A.; Frost, J.; Yang, P.; Hutchison, M.; Neiman, A. M.; Cobb, M. H.; Marcus, S. *J. Biol. Chem.* 1995, 270, 26067.

(259) Crespo, P.; Bustelo, X. R.; Aaronson, D. S.; Coso, O. A.; Lopez-Barahona, M.; Barbacid, M.; Gutkind, J. S. *Oncogene* 1996, 13, 455.

(260) Teramoto, H.; Coso, O. A.; Miyata, H.; Igishi, T.; Miki, T.; Gutkind, J. S. *J. Biol. Chem.* 1996, 271, 27225.

(261) Mochizuki, N.; Ohba, Y.; Kobayashi, S.; Otsuka, N.; Graybiel, A. M.; Tanaka, S.; Matsuda, M. *J. Biol. Chem.* 2000, 275, 12667.

(262) Hehner, S. P.; Hofmann, T. G.; Dienz, O.; Droge, W.; Schmitz, M. L. *J. Biol. Chem.* 2000, 275, 18160.

(263) Teramoto, H.; Crespo, P.; Coso, O. A.; Igishi, T.; Xu, N.; Gutkind, J. S. *J. Biol. Chem.* 1996, 271, 25731.

(264) Su, Y.-C.; Han, J.; Xu, S.; Cobb, M.; Skolnik, E. Y. *EMBO J.* 1997, 16, 1279.

(265) Hu, M. C.; Qiu, W. R.; Wang, X.; Meyer, C. F.; Tan, T. H. *Genes Dev.* 1996, 10, 2251.

(266) Kiefer, F.; Tibbles, L. A.; Anafi, M.; Janssen, A.; Zanke, B. W.; Lassam, N. J.; Pawson, T.; Woodgett, J. R.; Iscove, N. *EMBO J.* 1996, 15, 7013.

(267) Pombo, C. M.; Kehrl, J. H.; Sánchez, I.; Katz, P.; Avruch, J.; Zon, L. I.; Woodgett, J. R.; Force, T.; Kyriakis, J. M. *Nature* 1995, 377, 750.

(268) Nakano, K.; Yamauchi, J.; Nakagawa, K.; Itoh, H.; Kitamura, N. *J. Biol. Chem.* 2000, 275, 20533.

(269) Yao, Z.; Zhou, G.; Wang, X. S.; Brown, A.; Diener, K.; Gan, H.; Tan, T. H. *J. Biol. Chem.* 1999, 274, 2118.

(270) Coso, O. A.; Chiariello, M.; Kalinec, G.; Kyriakis, J. M.; Woodgett, J.; Gutkind, J. S. *J. Biol. Chem.* 1995, 270, 5620.

(271) Prasad, M. V.; Dermott, J. M.; Heasley, L. E.; Johnson, G. L.; Dhanasekaran, N. *J. Biol. Chem.* 1995, 270, 18655.

(272) Heasley, L. E.; Storey, B.; Fanger, G. R.; Butterfield, L.; Zamarripa, J.; Blumberg, D.; Maue, R. A. *Mol. Cell Biol.* 1996, 16, 648.

(273) Cadwallader, K.; Beltman, J.; McCormick, F.; Cook, S. *Biochem J.* 1997, 321, 795.

(274) Hara, T.; Namba, H.; Takamura, N.; Yang, T. T.; Nagayama, Y.; Fukata, S.; Kuma, K.; Ishikawa, N.; Ito, K.; Yamashita, S. *Endocrinology* 1999, 140, 1724.

(275) Wylie, P. G.; Challiss, R. A.; Blank, J. L. *Biochem. J.* 1999, 338, 619.

(276) Mulvaney, J. M.; Roberson, M. S. *J. Biol. Chem.* 2000, 275, 14182.

(277) Levi, N. L.; Hanoch, T.; Benard, O.; Rozenblat, M.; Harris, D.; Reiss, N.; Naor, Z.; Seger, R. *Mol. Endocrinol.* 1998, 12, 815.

(278) Coso, O. A.; Teramoto, H.; Simonds, W. F.; Gutkind, J. S. *J. Biol. Chem.* 1996, 271, 3963.

(279) Yamauchi, J.; Kawano, T.; Nagao, M.; Kaziro, Y.; Itoh, H. *J. Biol. Chem.* 2000, 275, 7633.

(280) Cambien, B.; Millet, M. A.; Schmid-Antomarchi, H.; Brossette, N.; Rossi, B.; Schmid-Alliana, A. *J. Immunol.* 1999, 163, 5079.

(281) Swantek, J. L.; Cobb, M. H.; Geppert, T. D. *Mol. Cell Biol.* 1997, 17, 6274.

(282) Gonzalez, M. V.; Jimenez, B.; Berciano, M. T.; Gonzalez-Sancho, J. M.; Caelles, C.; Lafarga, M.; Munoz, A. *J. Cell Biol.* 2000, 150, 1199.

(283) Lee, H. Y.; Walsh, G. L.; Dawson, M. I.; Hong, W. K.; Kurie, J. M. *J. Biol. Chem.* 1998, 273, 7066.

(284) Lee, H. Y.; Sueoka, N.; Hong, W. K.; Mangelsdorf, D. J.; Claret, F. X.; Kurie, J. M. Mol. Cell Biol. 1999, 19, 1973.

(285) Dickens, M.; Rogers, J. S.; Cavanagh, J.; Raitano, A.; Xia, Z.; Halpern, J. R.; Greenberg, M. E.; Sawyers, C. L.; Davis, R. J. Science 1997, 277, 693.

(286) Whitmarsh, A. J.; Cavanagh, J.; Tournier, C.; Yasuda, J.; Davis, R. J. Science 1998, 281, 1671.

(287) Yasuda, J.; Whitmarsh, A. J.; Cavanagh, J.; Sharma, M.; Davis, R. J. Mol. Cell Biol. 1999, 19, 7245.

(288) Kelkar, N.; Gupta, S.; Dickens, M.; Davis, R. J. Mol. Cell Biol. 2000, 20, 1030.

(289) Ito, M.; Yoshioka, K.; Akechi, M.; Yamashita, S.; Takamatsu, N.; Sugiyama, K.; Hibi, M.; Nakabeppu, Y.; Shiba, T.; Yamamoto, K. I. Mol. Cell Biol. 1999, 19, 7539.

(290) Koyano, S.; Ito, M.; Takamatsu, N.; Shiba, T.; Yamamoto, K.; Yoshioka, K. FEBS Lett. 1999, 457, 385.

(291) Hu, Q.; Milfay, D.; Williams, L. T. Mol. Cell. Biol. 1995, 15, 1169.

(292) Pena, E.; Berciano, M. T.; Fernandez, R.; Crespo, P.; Lafarga, M. Exp. Cell Res. 2000, 256, 179.

(293) Mizukami, Y.; Yoshioka, K.; Morimoto, S.; Yoshida, K. J. Biol. Chem. 1997, 272, 16657.

(294) Lee, J. K.; Park, J.; Lee, Y. D.; Lee, S. H.; Han, P. L. Brain Res. Mol. Brain Res. 1999, 70, 116.

(295) Maroney, A. C.; Glicksman, M. A.; Basma, A. N.; Walton, K. M.; Knight, E. J.; Murphy, C. A.; Bartlett, B. A.; Finn, J. P.; Angeles, T.; Matsuda, Y.; Neff, N. T.; Dionne, C. A. J. Neurosci. 1998, 18, 104.

(296) Saporito, M. S.; Brown, E. M.; Miller, M. S.; Carswell, S. J. Pharmacol. Exp. Ther. 1999, 288, 421.

(297) Le-Niculescu, H.; Bonfoco, E.; Kasuya, Y.; Claret, F. X.; Green, D. R.; Karin, M. Mol. Cell Biol. 1999, 19, 751.

(298) Kallunki, T.; Su, B.; Tsigelny, I.; Sluss, H. K.; Dérijard, B.; Moore, G.; Davis, R.; Karin, M. Genes Dev. 1994, 8, 2996.

(299) Kallunki, T.; Deng, T.; Hibi, M.; Karin, M. Cell 1996, 87, 929.

(300) Aguirre, V.; Uchida, T.; Yenush, L.; Davis, R.; White, M. F. J. Biol. Chem. 2000, 275, 9047.

(301) Fan, M.; Goodwin, M.; Vu, T.; Brantley-Finley, C.; Gaarde, W. A.; Chambers, T. C. J. Biol. Chem. 2000, 275, 29980.

(302) Zhang, Y.; Zhou, L.; Miller, C. A. Proc. Natl. Acad. Sci. U.S.A. 1998, 95, 2586.

(303) Crabtree, G. R. Cell 1999, 96, 611.

(304) Aramburu, J.; Garcia-Cozar, F.; Raghavan, A.; Okamura, H.; Rao; Hogan, P. G. Mol. Cell 1998, 1, 627.

(305) Yamashita, S.; Mochizuki, N.; Ohba, Y.; Tobiume, M.; Okada, Y.; Sawa, H.; Nagashima, K.; Matsuda, M. J. Biol. Chem. 2000, 275, 25488.

(306) Fuchs, S. Y.; Tappin, I.; Ronai, Z. J. Biol. Chem. 2000, 275, 12560.

(307) Dai, R.; Frejtag, W.; He, B.; Zhang, Y.; Mivechi, N. F. J. Biol. Chem. 2000, 275, 18210.

(308) Chung, J.; Uchida, E.; Grammer, T. C.; Blenis, J. Mol. Cell Biol. 1997, 17, 6508.

(309) Lim, C. P.; Cao, X. J. Biol. Chem. 1999, 274, 31055.

(310) Yujiri, T.; Ware, M.; Widmann, C.; Oyer, R.; Russell, D.; Chan, E.; Zaitsu, Y.; Clarke, P.; Tyler, K.; Oka, Y.; Fanger, G. R.; Henson, P.; Johnson, G. L. Proc. Natl. Acad. Sci. U.S.A. 2000, 97, 7272.

(311) Turner, C. E.; Miller, J. T. J. Cell Sci. 1994, 107, 1583.

(312) Cardone, M. H.; Salvesen, G. S.; Widmann, C.; Johnson, G.; Frisch, S. M. Cell 1997, 90, 315.

(313) Nishina, H.; Vaz, C.; Billia, P.; Nghiem, M.; Sasaki, T.; de la Pompa, J. L.; Furlonger, K.; Paige, C.; Hui, C.; Fischer, K. D.; Kishimoto, H.; Iwatsubo, T.; Katada, T.; Woodgett, J. R.; Penninger, J. M. Development 1999, 126, 505.

(314) Ganiatsas, S.; Kwee, L.; Fujiwara, Y.; Perkins, A.; Ikeda, T.; Labow, M. A.; Zon, L. I. Proc. Natl. Acad. Sci. U.S.A. 1998, 95, 6881.

(315) Yang, D.; Tournier, C.; Wysk, M.; Lu, H. T.; Xu, J.; Davis, R. J.; Flavell, R. A. Proc. Natl. Acad. Sci. U.S.A. 1997, 94, 3004.

(316) Dong, C.; Yang, D. D.; Wysk, M.; Whitmarsh, A. J.; Davis, R. J.; Flavell, R. A. Science 1998, 282, 2092.

(317) Yang, D. D.; Conze, D.; Whitmarsh, A. J.; Barrett, T.; Davis, R. J.; Rincon, M.; Flavell, R. A. Immunity 1998, 9, 575.

(318) Sabapathy, K.; Hu, Y.; Kallunki, T.; Schreiber, M.; David, J. P.; Jochum, W.; Wagner, E. F.; Karin, M. Curr. Biol. 1999, 9, 116.

(319) Yang, D. D.; Kuan, C. Y.; Whitmarsh, A. J.; Rincon, M.; Zheng, T. S.; Davis, R. J.; Rakic, P.; Flavell, R. A. Nature 1997, 389, 865.

(320) Kuan, C. Y.; Yang, D. D.; Samanta, R. D.; Davis, R. J.; Rakic, P.; Flavell, R. A. Neuron 1999, 22, 667.

(321) Sabapathy, K.; Jochum, W.; Hochedlinger, K.; Chang, L.; Karin, M.; Wagner, E. F. Mech. Dev. 1999, 89, 115.

(322) Han, J.; Lee, J.-D.; Bibbs, L.; Ulevitch, R. J. Science 1994, 265, 808.

(323) Lee, J. C.; Laydon, J. T.; McDonnell, P. C.; Gallagher, T. F.; Kumar, S.; Green, D.; McNulty, D.; Blumenthal, M. J.; Heys, J. R.; Landvatter, S. W. Nature 1994, 372, 739.


(324) Rouse, J.; Cohen, P.; Trigon, S.; Morange, M.; Alonso-Llamazares, A.; Zamanillo, D.; Hunt, T.; Nebreda, A. R. Cell 1994, 78, 1027.

(325) Jiang, Y.; Chen, C.; Li, Z.; Guo, W.; Gegner, J. A.; Lin, S.; Han, J. J. Biol. Chem. 1996, 271, 17920.

(326) Stein, B.; Yang, M. X.; Young, D. B.; Janknecht, R.; Hunter, T.; Murray, B. W.; Barbosa, M. S. J. Biol. Chem. 1997, 272, 19509.

(327) Lechner, C.; Zahalka, M. A.; Giot, J. F.; Moller, N. P.; Ullrich, A. Proc. Natl. Acad. Sci. U.S.A. 1996, 93, 4355.

(328) Li, Z.; Jiang, Y.; Ulevitch, R. J.; Han, J. Biochem. Biophys. Res. Commun. 1996, 228, 334.

(329) Jiang, Y.; Gram, H.; Zhao, M.; New, L.; Gu, J.; Feng, L.; Di Padova, F.; Ulevitch, R. J.; Han, J. J. Biol. Chem. 1997, 272, 30122.

(330) Goedert, M.; Cuenda, A.; Craxton, M.; Jakes, R.; Cohen, P. EMBO J. 1997, 16, 3563.

(331) Kumar, S.; McDonnell, P. C.; Gum, R. J.; Hand, A. T.; Lee, J. C.; Young, P. R. Biochem. Biophys. Res. Commun. 1997, 235, 533.

(332) Zervos, A. S.; Faccio, L.; Gatto, J. P.; Kyriakis, J. M.; Brent, R. Proc. Natl. Acad. Sci. U.S.A. 1995, 92, 10531.

(333) Lee, J. C.; Kassis, S.; Kumar, S.; Badger, A.; Adams, J. L. Pharmacol. Ther. 1999, 82, 389.

(334) Hu, M. C.; Wang, Y. P.; Mikhail, A.; Qiu, W. R.; Tan, T. H. J. Biol. Chem. 1999, 274, 7095.

(335) Nick, J. A.; Avdi, N. J.; Young, S. K.; Lehman, L. A.; McDonald, P. P.; Frasch, S. C.; Billstrom, M. A.; Henson, P. M.; Johnson, G. L.; Worthen, G. S. J. Clin. Invest. 1999, 103, 851.

(336) Wang, Y.; Huang, S.; Sah, V. P.; Ross, J. J.; Brown, J. H.; Han, J.; Chien, K. R. J. Biol. Chem. 1998, 273, 2161.

(337) Nemoto, S.; Xiang, J.; Huang, S.; Lin, A. J. Biol. Chem. 1998, 273, 16415.

(338) Tong, L.; Pav, S.; White, D. M.; Rogers, S.; Crane, K. M.; Cywin, C. L.; Brown, M. L.; Pargellis, C. A. Nat. Struct. Biol. 1997, 4, 311.

(339) Wang, Z.; Canagarajah, B. J.; Boehm, J. C.; Kassisa, S.; Cobb, M. H.; Young, P. R.; Abdel-Meguid, S.; Adams, J. L.; Goldsmith, E. J. Structure 1998, 6, 1117.

(340) Wilson, K. P.; McCaffrey, P. G.; Hsiao, K.; Pazhanisamy, S.; Galullo, V.; Bemis, G. W.; Fitzgibbon, M. J.; Caron, P. R.; Murcko, M. A.; Su, M. S. Chem. Biol. 1997, 4, 423.

(341) Jiang, Y.; Li, Z.; Schwarz, E. M.; Lin, A.; Guan, K.; Ulevitch, R. J.; Han, J. J. Biol. Chem. 1997, 272, 11096.

(342) Dérijard, B.; Raingeaud, J.; Barrett, T.; Wu, I.-H.; Han, J.; Ulevitch, R. J.; Davis, R. J. Science 1995, 267, 682.

(343) Enslin, H.; Brancho, D. M.; Davis, R. J. EMBO J. 2000, 19, 1301.

(344) Stein, B.; Brady, H.; Yang, M. X.; Young, D. B.; Barbosa, M. S. J. Biol. Chem. 1996, 271, 11427.

(345) Han, J.; Lee, J. D.; Jiang, Y.; Li, Z.; Feng, L.; Ulevitch, R. J. J. Biol. Chem. 1996, 271, 2886.

(346) Wang, X. S.; Diener, K.; Manthey, C. L.; Wang, S.; Rosenzweig, B.; Bray, J.; Delaney, J.; Cole, C. N.; Chan-Hui, P. Y.; Mantlo, N.; Lichenstein, H. S.; Zukowski, M.; Yao, Z. J. Biol. Chem. 1997, 272, 23668.

(347) Enslin, H.; Raingeaud, J.; Davis, R. J. J. Biol. Chem. 1998, 273, 1741.

(348) Cuenda, A.; Cohen, P.; Buee-Scherrer, V.; Goedert, M. EMBO J. 1997, 16, 295.

(349) Wysk, M.; Yang, D. D.; Lu, H. T.; Flavell, R. A.; Davis, R. J. Proc. Natl. Acad. Sci. U.S.A. 1999, 96, 3763.

(350) Lu, H. T.; Yang, D. D.; Wysk, M.; Gatti, E.; Mellman, I.; Davis, R. J.; Flavell, R. A. EMBO J. 1999, 18, 1845.

(351) Deacon, K.; Blank, J. L. J. Biol. Chem. 1999, 274, 16604.

(352) Guan, Z.; Buckman, S. Y.; Miller, B. W.; Springer, L. D.; Morrison, A. R. J. Biol. Chem. 1998, 273, 28670.

(353) Hirai, S.; Izawa, M.; Osada, S.; Spyrou, G.; Ohno, S. Oncogene 1996, 12, 641.

(354) Takekawa, M.; Posas, F.; Saito, H. EMBO J. 1997, 16, 4973.

(355) Cuenda, A.; Dorow, D. S. Biochem J. 1998, 333, 11.

(356) Moriguchi, T.; Kuroyanagi, N.; Yamaguchi, K.; Gotoh, Y.; Irie, K.; Kano, T.; Shirakabe, K.; Muro, Y.; Shibuya, H.; Matsumoto, K.; Nishida, E.; Hagiwara, M. J. Biol. Chem. 1996, 271, 13675.

(357) Diener, K.; Wang, X. S.; Chen, C.; Meyer, C. F.; Keesler, G.; Zukowski, M.; Tan, T. H.; Yao, Z. Proc. Natl. Acad. Sci. U.S.A. 1997, 94, 9687.

(358) Bagrodia, S.; Dérijard, B.; Davis, R. J.; Cerione, R. A. J. Biol. Chem. 1995, 270, 27995.

(359) Zhang, S.; Huan, J.; Sells, M. A.; Chernoff, J.; Knaus, U. G.; Ulevitch, R. J.; Bokoch, G. M. J. Biol. Chem. 1995, 270, 23934.

(360) Lim, L.; Manser, E.; Leung, T.; Hall, C. Eur. J. Biochem. 1996, 242, 171.

(361) Knaus, U. G.; Morris, S.; Dong, H. J.; Chernoff, J.; Bokoch, G. M. Science 1995, 269, 221.

(362) Tassi, E.; Biesova, Z.; Di Fiore, P. P.; Gutkind, J. S.; Wong, W. T. J. Biol. Chem. 1999, 274, 33287.

(363) Nagata, K.; Puls, A.; Futter, C.; Aspenstrom, P.; Schaefer, E.; Nakata, T.; Hirokawa, N.; Hall, A. EMBO J. 1998, 17, 149.

(364) Yamauchi, J.; Nagao, M.; Kaziro, Y.; Itoh, H. J. Biol. Chem. 1997, 272, 27771.

MAP Kinases

(365) Williams, N. G.; Zhong, H.; Minneman, K. P. J. Biol. Chem. 1998, 273, 24624.

(366) Communal, C.; Colucci, W. S.; Singh, K. J. Biol. Chem. 2000, 275, 19395.

(367) Sellers, L. A.; Alderton, F.; Carruthers, A. M.; Schindler, M.; Humphrey, P. P. Mol. Cell Biol. 2000, 20, 5974.

(368) Clerk, A.; Michael, A.; Sugden, P. H. J. Cell Biol. 1998, 142, 523.

(369) Nick, J. A.; Avdi, N. J.; Young, S. K.; Knall, C.; Gerwins, P.; Johnson, G. L.; Worthen, G. S. J. Clin. Invest. 1997, 99, 975.

(370) Zhang, Y.; Neo, S. Y.; Han, J.; Yaw, L. P.; Lin, S. C. J. Biol. Chem. 1999, 274, 2851.

(371) Roberson, M. S.; Zhang, T.; Li, H. L.; Mulvaney, J. M. Endocrinology 1999, 140, 1310.

(372) Muzio, M.; Ni, J.; Feng, P.; Dixit, V. M. Science 1997, 278, 1612.

(373) Burns, K.; Martinon, F.; Esslinger, D.; Pahl, H.; Schneider, P.; Bodmer, J.-L.; DiMarco, F.; French, L.; Tschopp, J. J. Biol. Chem. 1998, 273, 12203.

(374) Kanakaraj, P.; Schafer, P. H.; Cavender, D. E.; Wu, Y.; Ngo, K.; Grealish, P. F.; Wadsworth, S. A.; Peterson, P. A.; Siekierka, J. J.; Harris, C. A.; Fung-Leung, W. P. J. Exp. Med. 1998, 187, 2073.

(375) Liu, Z. G.; Hsu, H.; Goeddel, D. V.; Karin, M. Cell 1996, 87, 565.

(376) Natoli, G.; Costanzo, A.; Ianni, A.; Templeton, D. J.; Woodgett, J. R.; Balsano, C.; Levrero, M. Science 1997, 275, 200.

(377) Reinhard, C.; Shamoon, B.; Shyamala, V.; Williams, L. T. EMBO J. 1997, 16, 1080.

(378) Nishitoh, H.; Saitoh, M.; Mochida, Y.; Takeda, K.; Nakano, H.; Rothe, M.; Miyazono, K.; Ichijo, H. Mol. Cell 1998, 2, 389.

(379) Chang, H. Y.; Nishitoh, H.; Yang, X.; Ichijo, H.; Baltimore, D. Science 1998, 281, 1860.

(380) Yang, X.; Khosravi-Far, R.; Chang, H. Y.; Baltimore, D. Cell 1997, 89, 1067.

(381) Ivaska, J.; Reunanen, H.; Westermarck, J.; Koivisto, L.; Kahari, V. M.; Heino, J. J. Cell Biol. 1999, 147, 401.

(382) Brewster, J. L.; de Valoir, T.; Dwyer, N. D.; Winter, E.; Gustin, M. C. Science 1993, 259, 1760.

(383) Shiozaki, K.; Russell, P. Nature 1995, 378, 739.

(384) Hazzalin, C. A.; Cano, E.; Cuenda, A.; Barratt, M. J.; Cohen, P.; Mahadevan, L. C. Curr. Biol. 1996, 6, 1028.

(385) Scott, P. H.; Paul, A.; Belham, C. M.; Peacock, A. J.; Wadsworth, R. M.; Gould, G. W.; Welsh, D.; Plevin, R. Am. J. Respir. Crit. Care Med. 1998, 158, 958.

(386) Ingram, A. J.; Ly, H.; Thai, K.; Kang, M.; Scholey, J. W. Kidney Int. 1999, 55, 476.

(387) Bogoyevitch, M. A.; Gillespie-Brown, J.; Ketterman, A. J.; Fuller, S. J.; Ben-Levy, R.; Ashworth, A.; Marshall, C. J.; Sugden, P. H. Circ. Res. 1996, 79, 162.

(388) Gould, G. W.; Cuenda, A.; Thomson, F. J.; Cohen, P. Biochem. J. 1995, 311, 735.

(389) Shalom-Barak, T.; Quach, J.; Lotz, M. J. Biol. Chem. 1998, 273, 27467.

(390) Shapiro, L.; Puren, A. J.; Barton, H. A.; Novick, D.; Peskind, R. L.; Shenkar, R.; Gu, Y.; Su, M. S.; Dinarello, C. A. Proc. Natl. Acad. Sci. U.S.A. 1998, 95, 12550.

(391) Zachos, G.; Clements, B.; Conner, J. J. Biol. Chem. 1999, 274, 5097.

(392) Beltman, J.; Erickson, J. R.; Martin, G. A.; Lyons, J. F.; Cook, S. J. J. Biol. Chem. 1999, 274, 3772.

(393) Dziarski, R.; Jin, Y. P.; Gupta, D. J. Infect. Dis. 1996, 174, 777.

(394) Raingeaud, J.; Gupta, S.; Rogers, J. S.; Dickens, M.; Han, J.; Ulevitch, R. J.; Davis, R. J. J. Biol. Chem. 1995, 270, 7420.

(395) Maulik, N.; Yoshida, T.; Zu, Y. L.; Sato, M.; Banerjee, A.; Das, D. K. Am. J. Physiol. 1998, 275, H1857.

(396) Ben-Levy, R.; Hooper, S.; Wilson, R.; Paterson, H. F.; Marshall, C. J. Curr. Biol. 1998, 8, 1049.

(397) Lee, S. H.; Park, J.; Che, Y.; Han, P. L.; Lee, J. K. J. Neurosci. Res. 2000, 60, 623.

(398) Reynolds, C. H.; Nebreda, A. R.; Gibb, G. M.; Utton, M. A.; Anderton, B. H. J. Neurochem. 1997, 69, 191.

(399) Parker, C. G.; Hunt, J.; Diener, K.; McGinley, M.; Soriano, B.; Keesler, G. A.; Bray, J.; Yao, Z.; Wang, X. S.; Kohno, T.; Lichenstein, H. S. Biochem. Biophys. Res. Commun. 1998, 249, 791.

(400) Kramer, R. M.; Roberts, E. F.; Um, S. L.; Borsch-Haubold, A. G.; Watson, S. P.; Fisher, M. J.; Jakubowski, J. A. J. Biol. Chem. 1996, 271, 27723.

(401) Kusuhara, M.; Takahashi, E.; Peterson, T. E.; Abe, J.; Ishida, M.; Han, J.; Ulevitch, R.; Berk, B. C. Circ. Res. 1998, 83, 824.

(402) Raingeaud, J.; Whitmarsh, A. J.; Barrett, T.; Derijard, B.; Davis, R. J. Mol. Cell Biol. 1996, 16, 1247.

(403) Tan, Y.; Rouse, J.; Zhang, A.; Cariati, S.; Cohen, P.; Comb, M. J. EMBO J. 1996, 15, 4629.

(404) Han, J.; Jiang, Y.; Li, Z.; Kravchenko, V. V.; Ulevitch, R. J. Nature 1997, 386, 296.

(405) Zhao, M.; New, L.; Kravchenko, V. V.; Kato, Y.; Gram, H.; Di Padova, F.; Olson, E. N.; Ulevitch, R. J.; Han, J. Mol. Cell Biol. 1999, 19, 21.

(406) Janknecht, R.; Hunter, T. EMBO J. 1997, 16, 1620.


Chemical Reviews, 2001, Vol. 101, No. 8 2475

(407) Whitmarsh, A. J.; Shore, P.; Sharrocks, A. D.; Davis, R. J. Science 1995, 269, 403.

(408) Price, M. A.; Cruzalegui, F. H.; Treisman, R. EMBO J. 1996, 15, 6552.

(409) Huang, C.; Chen, N.; Ma, W. Y.; Dong, Z. Int. J. Oncol. 1998, 13, 711.

(410) Zechner, D.; Craig, R.; Hanford, D. S.; McDonough, P. M.; Sabbadini, R. A.; Glembotski, C. C. J. Biol. Chem. 1998, 273, 8232.

(411) Wang, X. Z.; Ron, D. Science 1996, 272, 1347.

(412) Tanaka, K.; Oda, N.; Iwasaka, C.; Abe, M.; Sato, Y. J. Cell Physiol. 1998, 176, 235.

(413) Kim, J.; Nueda, A.; Meng, Y. H.; Dynan, W. S.; Mivechi, N. F. J. Cell Biochem. 1997, 67, 43.

(414) Huang, C.; Ma, W. Y.; Maxiner, A.; Sun, Y.; Dong, Z. J. Biol. Chem. 1999, 274, 12229.

(415) Engelman, J. A.; Lisanti, M. P.; Scherer, P. E. J. Biol. Chem. 1998, 273, 32111.

(416) Bulavin, D. V.; Saito, S.; Hollander, M. C.; Sakaguchi, K.; Anderson, C. W.; Appella, E.; Fornace, A. J. J. EMBO J. 1999, 18, 6845.

(417) Mao, Z.; Bonni, A.; Xia, F.; Nadal-Vicens, M.; Greenberg, M. E. Science 1999, 286, 785.

(418) Freshney, N. W.; Rawlinson, L.; Guesdon, F.; Jones, E.; Cowley, S.; Hsuan, J.; Saklatvala, J. Cell 1994, 78, 1039.

(419) McLaughlin, M. M.; Kumar, S.; Mcdonnell, P. C.; Van Horn, S.; Lee, J. C.; Livi, G. P.; Young, P. R. J. Biol. Chem. 1996, 271, 8488.

(420) Heidenreich, O.; Neininger, A.; Schratt, G.; Zinck, R.; Cahill, M. A.; Engel, K.; Kotlyarov, A.; Kraft, R.; Kostka, S.; Gaestel, M.; Nordheim, A. J. Biol. Chem. 1999, 274, 14434.

(421) Thomas, G.; Haavik, J.; Cohen, P. Eur. J. Biochem. 1997, 247, 1180.

(422) Ni, H.; Wang, X. S.; Diener, K.; Yao, Z. Biochem. Biophys. Res. Commun. 1998, 243, 492.

(423) Deak, M.; Clifton, A. D.; Lucocq, L. M.; Alessi, D. R. EMBO J. 1998, 17, 4426.

(424) New, L.; Zhao, M.; Li, Y.; Bassett, W. W.; Feng, Y.; Ludwig, S.; Padova, F. D.; Gram, H.; Han, J. J. Biol. Chem. 1999, 274, 1026.

(425) Pierrat, B.; Correia, J. S.; Mary, J. L.; Tomas-Zuber, M.; Lesslauer, W. J. Biol. Chem. 1998, 273, 29661.

(426) New, L.; Jiang, Y.; Zhao, M.; Liu, K.; Zhu, W.; Flood, L. J.; Kato, Y.; Parry, G. C.; Han, J. EMBO J. 1998, 17, 3372.

(427) Garcia, J.; Lemercier, B.; Roman-Roman, S.; Rawadi, G. J. Biol. Chem. 1998, 273, 34391.

(428) Miyazawa, K.; Mori, A.; Miyata, H.; Akahane, M.; Ajisawa, Y.; Okudaira, H. J. Biol. Chem. 1998, 273, 24832.

(429) Marie, C.; Roman-Roman, S.; Rawadi, G. Infect. Immun. 1999, 67, 688.

(430) Wang, Y. Z.; Zhang, P.; Rice, A. B.; Bonner, J. C. J. Biol. Chem. 2000, 275, 22550.

(431) Huttunen, P.; Hyypia, T.; Vihinen, P.; Nissinen, L.; Heino, J. Virology 1998, 250, 85.

(432) Subbaramaih, K.; Hart, J. C.; Norton, L.; Dannenberg, A. J. J. Biol. Chem. 2000, 275, 14838.

(433) Lasa, M.; Mahtani, K. R.; Finch, A.; Brewer, G.; Saklatvala, J.; Clark, A. R. Mol. Cell Biol. 2000, 20, 4265.

(434) Kumar, S.; Orsini, M. J.; Lee, J. C.; Mcdonnell, P. C.; Debouck, C.; Young, P. R. J. Biol. Chem. 1996, 271, 30864.

(435) Marshall, L. A.; Hansbury, M. J.; Bolognese, B. J.; Gum, R. J.; Young, P. R.; Mayer, R. J. J. Immunol. 1998, 161, 6005.

(436) McGilvray, I. D.; Lu, Z.; Wei, A. C.; Dackiw, A. P.; Marshall, J. C.; Kapus, A.; Levy, G.; Rotstein, O. D. J. Biol. Chem. 1998, 273, 32222.

(437) Cheong, J.; Coligan, J. E.; Shuman, J. D. J. Biol. Chem. 1998, 273, 22714.

(438) Bhat, N. R.; Zhang, P.; Bhat, A. N. J. Neurochem. 1999, 72, 472.

(439) Pietersma, A.; Tilly, B. C.; Gaestel, M.; de Jong, N.; Lee, J. C.; Koster, J. F.; Sluiter, W. Biochem. Biophys. Res. Commun. 1997, 230, 44.

(440) Lavoie, J. N.; L'Allemand, G.; Brunet, A.; Muller, R.; Pouyssegur, J. J. Biol. Chem. 1998, 271, 20608.

(441) Allen, M.; Svensson, L.; Roach, M.; Hambor, J.; McNeish, J.; Gabel, C. A. J. Exp. Med. 2000, 191, 859.

(442) Adams, R. H.; Porras, A.; Alonso, J.; Jones, M.; Vintersten, K.; Panelli, S.; Valladares, A.; Perez, L.; Klein, R.; Nebreda, A. R. Mol. Cell 2000, 6, 109.

(443) Tamura, K.; Sudo, T.; Senftleben, U.; Dadak, A. M.; Johnson, R.; Karin, M. Cell 2000, 102, 221.

(444) Mudgett, J. S.; Ding, J.; Guh-Siesel, L.; Chartrain, N. A.; Yang, L.; Gopal, S.; Shen, M. M. Proc. Natl. Acad. Sci. U.S.A. 2000, 97, 10454.

(445) Zhou, G.; Bao, Z. Q.; Dixon, J. E. J. Biol. Chem. 1995, 270, 12665.

(446) Lee, J.-D.; Ulevitch, R. J.; Han, J. Biochem. Biophys. Res. Comm. 1995, 213, 715.

(447) English, J. M.; Vanderbilt, C. A.; Xu, S.; Marcus, S.; Cobb, M. H. J. Biol. Chem. 1995, 270, 28897.

(448) Abe, J.-I.; Takahashi, M.; Ishida, M.; Lee, J.-D.; Berk, B. C. J. Biol. Chem. 1997, 272, 20389.

(449) English, J. M.; Pearson, G.; Baer, R.; Cobb, M. H. *J. Biol. Chem.* 1998, 273, 3854.

(450) English, J. M.; Pearson, G.; Hockenberry, T.; Shivakumar, L.; White, M. A.; Cobb, M. H. *J. Biol. Chem.* 1999, 274, 31588.

(451) Fukuhara, S.; Marinissen, M. J.; Chiariello, M.; Gutkind, J. S. *J. Biol. Chem.* 2000, 275, 21730.

(452) Chao, T. H.; Hayashi, M.; Tapping, R. I.; Kato, Y.; Lee, J. D. J. *Biol. Chem.* 1999, 274, 36035.

(453) Chiariello, M.; Marinissen, M. J.; Gutkind, J. S. *Mol. Cell Biol.* 2000, 20, 1747.

(454) Kato, Y.; Kravchenko, V. V.; Tapping, R. I.; Han, J.; Ulevitch, R. J.; Lee, J. D. *EMBO J.* 1997, 16, 7054.

(455) Yang, J.; Boerm, M.; McCarty, M.; Bucana, C.; Fidler, I. J.; Zhuang, Y.; Su, B. *Nat. Genet.* 2000, 24, 309.

(456) Cheng, M.; Boulton, T. G.; Cobb, M. H. *J. Biol. Chem.* 1996, 271, 8951.

(457) Turgeon, B.; Saba-El-Leil, M. K.; Meloche, S. *Biochem. J.* 2000, 346, 169.

(458) Cheng, M.; Zhen, E.; Robinson, M. J.; Ebert, D.; Goldsmith, E.; Cobb, M. H. *J. Biol. Chem.* 1996, 271, 12057.

(459) Abe, M. K.; Kuo, W. L.; Hershenson, M. B.; Rosner, M. R. *Mol. Cell Biol.* 1999, 19, 1301.

(460) Qian, Z.; Okuhara, D.; Abe, M. K.; Rosner, M. R. *J. Biol. Chem.* 1999, 274, 1621.

(461) Brott, B. K.; Pinsky, B. A.; Erikson, R. L. *Proc. Natl. Acad. Sci. U.S.A.* 1998, 95, 963.

(462) Ishitani, T.; Ninomiya-Tsuji, J.; Nagai, S.; Nishita, M.; Mene- ghini, M.; Barker, N.; Waterman, M.; Bowerman, B.; Clevers, H.; Shibuya, H.; Matsumoto, K. *Nature* 1999, 399, 798.

(463) Meneghini, M. D.; Ishitani, T.; Carter, J. C.; Hisamoto, N.; Ninomiya-Tsuji, J.; Thorpe, C. J.; Hamill, D. R.; Matsumoto, K.; Bowerman, B. *Nature* 1999, 399, 793.

(464) Miyata, Y.; Akashi, M.; Nishida, E. *Genes Cells* 1999, 4, 299.

(465) Miyata, Y.; Nishida, E. *Biochem. Biophys. Res. Commun.* 1999, 266, 291.

(466) Matsushime, H.; Jinno, A.; Takagi, N.; Shibuya, M. *Mol. Cell Biol.* 1990, 10, 2261.

(467) Abe, S.; Yagi, T.; Ishiyama, S.; Hiroe, M.; Marumo, F.; Ikawa, Y. *Oncogene* 1995, 11, 2187.

(468) Gonzalez, F. A.; Raden, D. L.; Rigby, M. R.; Davis, R. J. *FEBS Lett.* 1992, 304, 170.

(469) Meyerson, M.; Enders, G. H.; Wu, C. L.; Su, L. K.; Gorka, C.; Nelson, C.; Harlow, E.; Tsai, L. H. *EMBO J.* 1992, 11, 2909.

(470) Nishina, H.; Fischer, K. D.; Radvanyi, L.; Shahinian, A.; Hakem, R.; Rubie, E. A.; Bernstein, A.; Mak, T. W.; Woodgett, J. R.; Penninger, J. M. *Nature* 1997, 385, 350.

(471) Nishina, H.; Bachmann, M.; Oliveira, d. S. A.; Kozieradzki, I.; Fischer, K. D.; Odermatt, B.; Wakeham, A.; Shahinian, A.; Takimoto, H.; Bernstein, A.; Mak, T. W.; Woodgett, J. R.; Ohashi, P. S.; Penninger, J. M. *J. Exp. Med* 1997, 186, 941.

(472) Nishina, H.; Radvanyi, L.; Raju, K.; Sasaki, T.; Kozieradzki, I.; Penninger, J. M. *J. Immunol.* 1998, 161, 3416.

(473) Swat, W.; Fujikawa, K.; Ganiatsas, S.; Yang, D.; Xavier, R. J.; Harris, N. L.; Davidson, L.; Ferrini, R.; Davis, R. J.; Labow, M. A.; Flavell, R. A.; Zon, L. I.; Alt, F. W. *Immunity* 1998, 8, 625.
