
# Circadian Clock-Regulated Expression of Phytochrome and Cryptochrome Genes in Arabidopsis¹

Réka Tóth, Éva Kevei, Anthony Hall, Andrew J. Millar, Ferenc Nagy, and László Kozma-Bognár*

Institute of Plant Biology, Biological Research Center of the Hungarian Academy of Sciences, P.O. Box 521, H–6701 Szeged, Hungary (R.T., É.K., F.N., L.K.-B.); and Department of Biological Sciences, University of Warwick, Coventry CV4 7AL, United Kingdom (A.H., A.J.M.)

Many physiological and biochemical processes in plants exhibit endogenous rhythms with a period of about 24 h. Endogenous oscillators called circadian clocks regulate these rhythms. The circadian clocks are synchronized to the periodic environmental changes (e.g. day/night cycles) by specific stimuli; among these, the most important is the light. Photoreceptors, phytochromes, and cryptochromes are involved in setting the clock by transducing the light signal to the central oscillator. In this work, we analyzed the spatial, temporal, and long-term light-regulated expression patterns of the Arabidopsis phytochrome (*PHYA* to *PHYE*) and cryptochrome (*CRY1* and *CRY2*) promoters fused to the luciferase (*LUC⁺*) reporter gene. The results revealed new details of the tissue-specific expression and light regulation of the *PHYC* and *CRY1* and 2 promoters. More importantly, the data obtained demonstrate that the activities of the promoter::*LUC⁺* constructs, with the exception of *PHYC*::*LUC⁺*, display circadian oscillations under constant conditions. In addition, it is shown by measuring the mRNA abundance of *PHY* and *CRY* genes under constant light conditions that the circadian control is also maintained at the level of mRNA accumulation. These observations indicate that the plant circadian clock controls the expression of these photoreceptors, revealing the formation of a new regulatory loop that could modulate gating and resetting of the circadian clock.

The periodic succession of days and nights is an eternally recurring environmental factor ever since life has appeared on the Earth. It is postulated that organisms possessing the ability to adapt to the predictable changes of the environment have an evolutionary advantage and that this benefit has promoted the development of timekeeping mechanisms (endogenous clocks). The biological clocks that generate and maintain oscillations of many physiological and molecular processes with a period length close to 24 h are also referred to as circadian clocks. Circadian rhythms persist under constant conditions; however, to function reliably and to be useful for the organisms, the clocks must operate in harmony with the periodic changes of the outer environment. To achieve this synchrony, the circadian clock is reset to the local time by specific stimuli perceived at dawn and dusk. The most important entraining factors are light and temperature. Light signals are perceived and transduced to the central oscillator via specialized photoreceptors. In plants, the photoreceptor phytochrome and cryptochrome have been shown to be involved in this process (Somers et al., 1998; Devlin and Kay, 2000).

Phytochromes are chromoproteins that contain a covalently linked linear tetrapyrrole chromophore per molecule and exist as homodimers. These photoreceptor molecules absorb red and far-red light, which activates or inactivates them, respectively (Quail et al., 1995). In higher plants, small multigene families encode these molecules. In Arabidopsis, five genes (*PHYA–E*) have been isolated (Sharrock and Quail, 1989; Clack et al., 1994). *PHYA* is a photolabile molecule degrading rapidly upon exposure to light. It is the dominant phytochrome in etiolated seedlings and it mediates responses to very low fluences of red and far-red light. *PHYB*, *C*, *D*, and *E* are relatively photostable molecules; in green seedlings, *PHYB* is the dominant phytochrome photoreceptor. They mediate responses to low and high fluences of red light (for review, see Furuya and Schäfer, 1996; Casal et al., 1998; Neff et al., 2000). Recently, it was shown that *PHYA* and *PHYB* translocate to the nucleus in a light-dependent manner (Kircher et al., 1999; Yamaguchi et al., 1999). It was also suggested that the regulated nuclear import of these receptors could be a key element of the phytochrome signal transduction pathway (Nagy and Schäfer, 2000a).

Plant cryptochromes are FAD and pterin-containing chromoproteins showing significant homology to DNA photolyases, but lacking photolyase activity. Cryptochromes absorb in the blue region of the spectrum. To date, two members of the cryptochrome family, *CRY1* and 2, have been identified in Arabidopsis (Ahmad and Cashmore, 1993; Lin et al., 1998). The *CRY2* protein shows rapid blue light-induced degradation and functions primarily at low light intensities (Lin et al., 1998). The *CRY1* protein is

---

¹ This work was supported by the Hungarian Scientific Research Fund (grant nos. F–029163 to L.K.-B. and T–032565 to F.N.), by the Howard Hughes Medical Institute (grant no. 55000325 to F.N.), by the Human Frontier Research Program (to A.J.M. and F.N.), and by the Biotechnology and Biological Sciences Research Council (grant no. G08667 to A.J.M.).

* Corresponding author; e-mail kozmab@nucleus.szbk.u-szeged.hu; fax 36–62–433–434.

Article, publication date, and citation information can be found at www.plantphysiol.org/cgi/doi/10.1104/pp.010467.

Tóth et al.

relatively stable in light and mediates responses to higher fluences of blue light (Lin et al., 1998). More recently, cryptochrome photoreceptors were also identified in animals. In *Drosophila melanogaster*, input light signals absorbed by the CRY photoreceptors promote the degradation of the TIMELESS protein, a key element of the fly's circadian oscillator, thereby resetting the clock (Ceriani et al., 1999; Emery et al., 2000). Although it is unclear whether the CRY protein is a functional input receptor to the mammalian clock, it was proven that it is an indispensable part of the central oscillator in mammals (van der Horst et al., 1999).

According to the simplest model of the circadian system, the central oscillator generates an oscillation with a period of approximately 24 h, based on negative feedback loops formed by the clock genes and proteins, and it regulates the expression of genes through the output pathway. On the other side of the system, light signals absorbed by photoreceptors reach the central oscillator through the input pathway and synchronize its phase to the actual periodic environmental changes. In this model, there is a one-way relationship between the input receptors and the oscillator without any feedback mechanisms. However, it has been shown that the oscillator controls the expression of cryptochrome receptors in *D. melanogaster* and in mouse (*Mus musculus*; Glossop et al., 1999; Shearman et al., 2000). We have demonstrated previously that the Arabidopsis circadian clock regulates the expression of the PHYB photoreceptor (Kozma-Bognár et al., 1999). Very recently, Harmer and coworkers (Harmer et al., 2000) demonstrated that the mRNA levels of the Arabidopsis CRY1 and CRY2 genes oscillate with a circadian rhythm under constant light (LL) conditions.

In this work, we performed a detailed, comparative analysis of the expression patterns of all phytochrome and cryptochrome genes in Arabidopsis with respect to their circadian regulation. Utilizing the luciferase reporter system, we demonstrate that the circadian clock controls the promoter activity of all CRY and PHY genes except for that of PHYC. We also show that the circadian modulation of the promoter activities is reflected at the level of mRNA accumulation. Regarding the PHYC gene, we demonstrate that the circadian clock regulates only the accumulation of PHYC mRNA. Moreover, by using the luminescent reporters, we also present in vivo data on tissue specificity and light-regulated expression of all PHY and CRY genes.

with 5 mM luciferin solution, and the bioluminescence patterns were characterized by in vivo imaging. The expression patterns of the PHYA::LUC⁺ and CRY2::LUC⁺ chimeric genes were nearly identical: A high level of luciferase activity was measured in the shoot meristems and root tips, and less but significant activity was found in the cotyledons, hypocotyls, and roots (Fig. 1, B and H). CAB2::LUC⁺ and CRY1::LUC⁺ were actively transcribed in the aerial tissues (in cotyledons and leaf primordia), but no activity was detected in the roots (Fig. 1, A and G). PHYB::LUC⁺ was luminescent in all tissues with highest activity in the cells of the shoot meristem and root tips (Fig. 1C). The expression pattern of PHYD::LUC⁺ and PHYE::LUC⁺ was similar to that of PHYB::LUC⁺, but displayed relatively lower expression in the shoot meristem (Fig. 1, E and F). PHYC::LUC⁺ was active mainly in the cotyledons and root tips, but lower expression was detected in the leaf primordia and in the root (Fig. 1D). The data presented here for the organ-specific expression of the PHYA, PHYB, PHYD, and PHYE genes are consistent with earlier reports based on β-glucuronidase (GUS) reporter fusions (Somers and Quail, 1995; Goosey et al., 1997) and mRNA accumulation (Clack et al., 1994). Moreover, they confirm the results of mRNA analysis (Ahmad and Cashmore, 1993; Lin et al., 1998) and extend our knowledge concerning the organ- or tissue-specific expression of CRY1-2 and PHYC genes.

Diurnal and Circadian Regulation of Phytochrome and Cryptochrome Promoter Activity

Transgenic seedlings carrying the various promoter::luciferase chimeric genes, including the CAB2::LUC⁺ control, were grown under 12-h-light/12-h-dark photoperiods for 1 week and were imaged under the same conditions. All of these plants showed diurnal rhythms with activity peaking during the light phase, although with there were differences regarding amplitudes and phases (Fig. 2). The luminescence rhythm of PHYA::LUC⁺ displayed a biphasic curve. The first peak appeared just after the lights-on signal and was followed by a second peak occurring a few hours before the lights-off signal (Fig. 2A). The amplitude of the oscillation of PHYB::LUC⁺ activity was very similar to that of PHYA::LUC⁺; however, its expression peaked earlier at about 4 h after the lights were on (Fig. 2A). The activity of PHYC::LUC⁺ was again higher during the light phase and lower in dark phase, but it showed a low amplitude rhythm. It peaked about 2 h before the lights were turned off and promptly decreased to a lower, flattened level in the dark (Fig. 2B). The diurnal rhythms of PHYD::LUC⁺ and PHYE::LUC⁺ were quite similar: They showed the same phase of maximal expression (4–6 h after the lights were on) and had a relatively low amplitude (Fig. 2C). The expres-

RESULTS

Spatial Expression Pattern of the Luminescent Reporter Constructs

Transgenic seedlings expressing the various promoter::luciferase constructs were grown in light-dark cycles for 1 week. The seedlings were sprayed

Circadian Regulation of AtPHYA–E and AtCRY1-2 Gene Expression

Figure 1. Tissue-specific expression of the various luminescent reporter constructs in Arabidopsis seedlings. Plants were grown under 12-h-light (60–70 $\mu$M m${}^{-2}$ s${}^{-1}$, white fluorescent)/12-h-dark photoperiods for 7 d. Images were taken during the light phase (between 4 and 8 h after the lights were on) on the 8th d after germination. Pictures are arranged as pairs of corresponding images. Right, Reflected-light image; left, false-colored luminescence image of the same seedling carrying the given transgene. A, CAB2::LUC${}^{+}$; B, PHYA::LUC${}^{+}$; C, PHYB:LUC${}^{+}$; D, PHYC::LUC${}^{+}$; E, PHYD::LUC${}^{+}$; F, PHYE::LUC${}^{+}$; G, CRY1::LUC${}^{+}$; H, CRY2::LUC${}^{+}$. The false-color scale goes from blue (low activity) to red and white (high activity).

sion of CRY1::LUC${}^{+}$ and CAB2::LUC${}^{+}$ had the same phase and similar amplitude (Fig. 2, D and B). In contrast, the CRY2::LUC${}^{+}$ rhythm showed a much lower amplitude and a late phase, with maximal expression around the end of the light phase (Fig. 2D). In all cases, except for PHYC::LUC${}^{+}$, the observed diurnal rhythms anticipated the lights-on and lights-off signals, suggesting a role for the circadian clock in the regulation of the expression of these genes.

The most reliable diagnostic feature of circadian rhythms is that they persist under constant conditions. Therefore, we measured the luminescence of the same transgenic seedlings entrained as described above, in LL (Fig. 3) and constant dark (DD; Fig. 4). In LL, most of the chimeric genes showed rhythmic

Plant Physiol. Vol. 127, 2©Downloaded from www.plantphysiol.org on May 12, 2015 - Published by www.plant.org 1609  
Copyright © 2001 American Society of Plant Biologists. All rights reserved.

its rhythm did not dampen rapidly and exhibited about 2-fold lower amplitude compared with the corresponding LL data (Fig. 3A). The amplitude of the CRY1::LUC⁺ rhythm decreased day by day during the measurement (Fig. 4D). The extended dark period did not significantly reduce the amplitude of the PHYD,E::LUC⁺ rhythms, as shown in Figure 4C. The expression level of PHYC::LUC⁺ decreased continuously in the dark and had an extremely low amplitude with two maxima at ZT 24 and ZT 48 (Fig. 4B), similar to LL (Fig. 3B).

### Long-Term Regulation of the Mean Level of Phytochrome and Cryptochrome Promoter Activities by Light

Seedlings carrying the various promoter::luciferase reporter constructs were grown and entrained in LD cycles for 1 week and were subsequently transferred to LL or DD. Luminescence of individual seedlings was measured using the TopCount luminometer (Packard Instruments, Meriden, CT) for 4d as described in Figure 5. The average of the counts collected over 4 d is shown.

Figure 2. Diurnal regulation of phytochrome and cryptochrome gene expression in Arabidopsis seedlings. Seedlings were grown under 12-h-light/12-h-dark cycles for 1 week, and were then imaged under the same conditions. A, PHYA::LUC⁺ (○), PHYB::LUC⁺ (▲); B, PHYC::LUC⁺ (○), CAB2::LUC⁺ (▲); C, PHYD::LUC⁺ (○), PHYE::LUC⁺ (▲); D, CRY1::LUC⁺ (○), CRY2::LUC⁺ (▲). White box on time axis, Light interval; black box, dark interval.

expression. These data indicate that the circadian clock controls the expression of these genes. However, as compared with light/dark conditions (LD), we detected minor changes in the phase of the peaks (1–2 h) in LL. For example, in LL, PHYA and PHYB::LUC⁺ reached their maximum activities later (Fig. 3A versus Fig. 2A), whereas the peak luminescence of PHYD and PHYE::LUC⁺ was shifted to an earlier time (Fig. 3C versus Fig. 2C) than in LD. As an exception, the expression of PHYC::LUC⁺ showed a very weak rhythmicity, if any, in LL (Fig. 3B). Its pattern is characterized by a much-reduced amplitude and a 12-h phase-shift (from Zeitgeber time [ZT] 12 to ZT 24).

In DD, many of the circadian processes dampen rapidly in plants. For example, the rhythmic expression of CAB2::LUC⁺ dampened to a low level in DD (Fig. 4B). The opposite effect was observed for the PHYA::LUC⁺ and CRY2::LUC⁺ constructs. The activity of these transgenes dampened to a high level after a rapid initial increase during the first subjective day (Fig. 4, A and D). The luminescence of PHYB::LUC⁺ also showed this initial increase (Fig. 4A); however,

Figure 3. Circadian regulation of phytochrome and cryptochrome gene expression in LL. Seedlings were grown and entrained as in Figure 2, but were imaged after transfer to LL. A, PHYA::LUC⁺ (○), PHYB::LUC⁺ (▲); B, PHYC::LUC⁺ (○), CAB2::LUC⁺ (▲); C, PHYD::LUC⁺ (○), PHYE::LUC⁺ (▲); D, CRY1::LUC⁺ (○), CRY2::LUC⁺ (▲). White box on time axis, Light interval; striped box, subjective dark interval.

Circadian Regulation of AtPHYA–E and AtCRY1-2 Gene Expression

in 4-h intervals. The abundance of phytochrome and cryptochrome mRNA molecules was determined as described in "Materials and Methods."

In all cases, the mRNA levels of the various PHY and CRY genes displayed clear circadian oscillations (Fig. 6). In general, the phases of peaks for mRNA levels were the same or 2 to 4 h earlier as compared with the corresponding luminescence data (Fig. 3). Moreover, rhythms in mRNA levels of the PHYA, B (Fig. 6A) and of the CRY1,2 (Fig. 6D) genes had amplitudes similar to those of the corresponding luminescence rhythms (Fig. 3, A and D). These data suggest that the circadian rhythms originating from the clock-regulated promoter activity of these genes exist, without significant modifications, at the level of the mRNA accumulation. However, the oscillation of the PHYD and PHYE mRNA levels exhibited significantly higher (3- to 4-fold) amplitudes (Fig. 6C) than the rhythms of their promoter activity (Fig. 3C), indicating that the circadian clock has an additional effect on the mRNA synthesis/stability of these genes. The luminescence of the PHYC::LUC+ construct showed very weak rhythm with extremely low amplitude in LL (Fig. 3B). To our surprise, however, the PHYC mRNA level displayed clear circadian oscillations with an amplitude comparable with that of the PHYD,E mRNA levels (Fig. 6B). This fact indicates that the circadian regulation of the PHYC gene

Figure 4. Circadian regulation of phytochrome and cryptochrome gene expression in DD. Seedlings were grown and entrained as in Figure 2 and 3, but were imaged after transfer to DD. A, PHYA::LUC+ (○), PHYB::LUC+ (▲); B, PHYC::LUC+ (○), CAB2::LUC+ (▲); C, PHYD::LUC+ (○), PHYE::LUC+ (▲); D, CRY1::LUC+ (○), CRY2::LUC+ (▲). White box on time axis, Light interval; black box, dark interval; gray box, subjective light intervals.

lected during the whole measurement from seedlings with the same transgene was calculated and is referred to as the mean expression level (Fig. 5). The expression levels of the PHYA,B::LUC+ and of the CRY2::LUC+ constructs in the dark were 1.5- to 2.5-fold higher than in the light; thus, the activity of these promoters is down-regulated by light. In contrast, the luminescence of PHYC::LUC+, CAB2::LUC+, and CRY1::LUC+ was 2- to 3-fold higher in the light than in the dark. The activity of the PHYD,E::LUC+ constructs showed very weak light dependence because their expression level was only 1.2- to 1.3-fold higher in the light.

Circadian Accumulation of the Phytochrome and Cryptochrome mRNA Molecules

Wild-type Arabidopsis seeds (Wassilewskija [WS] ecotype) were germinated and grown on sterile Murashige and Skoog medium under 12-h-light/12-h-dark cycles for 1 week and were then transferred to LL. Total RNA was isolated from samples harvested

Plant Physiol. Vol. 127, 20Downloaded from www.plantphysiol.org on May 12, 2015 - Published by www.plant.org 1611 Copyright © 2001 American Society of Plant Biologists. All rights reserved.

Tóth et al.

![Image Description](#)

expression is exerted at the level of mRNA accumulation rather than at the level of the promoter activity.

## DISCUSSION

### Tissue-Specific Expression

To study spatial and temporal regulation of Arabidopsis *PHY* and *CRY* genes at the level of promoter activity, we constructed a series of *PHY* and *CRY* promoter::luciferase chimeric genes and regenerated a large number of transgenic Arabidopsis plants expressing these reporters. Tissue- and organ-specific expression of the transgenes was determined at the same developmental stage (LD-grown, 7-d-old seedlings; Fig. 1) when time course measurements for circadian rhythmicity started. Spatial expression patterns of the *PHYA*, *PHYB*, *PHYD*, and *PHYE* promoter:luciferase transgenes were identical to those reported earlier based on GUS reporter data (Somers and Quail, 1995; Goosey et al., 1997). The difference between the GUS and LUC⁺ reporter data concerning the activity of the *PHYE* promoter in the root tips can be explained by the different lengths of promoter fragments used in the constructs and/or by the different growth conditions. Moreover, similar to Clack et al. (1994), we were able to detect significant amounts of *PHYE* transcripts in the roots of 1-week-old plants (data not shown), demonstrating that under our conditions, *PHYE* is expressed in the root tissue. On the one hand, these observations verify that luminescent data collected from our plants correctly reflect the regulation of these promoters. On the other hand, the data presented here offer a better resolution than those obtained by measuring mRNA accumulation of *PHYC* (Clack et al., 1994) and *CRY1* and *CRY2* (Ahmad and Cashmore, 1993; Hoffman et al., 1996). In addition, it is demonstrated clearly that *CRY1* is expressed mostly in the aerial tissues (similar to *CAB2*), whereas the highest activity of *CRY2::LUC⁺* is found in the leaf primordia and the root tip, and it is also clearly detected in other tissues including the cotyledons (similar to *PHYA*). The expression pattern of *PHYC::LUC⁺* closely resembles that of *CRY2::LUC⁺*, with the difference that the highest level of expression was found in the cotyledons rather than in the shoot meristem.

### Light-Regulated Expression

Taking advantage of the luciferase reporter system, we also determined the effect of light on the activities of the *PHY* and *CRY* promoters in vivo. To this end, plants were grown on LD cycles for 1 week, and they were then transferred to light or dark for the extended period of the measurement, during which luminescence data were collected in 1-h intervals. We found that the activities of the *PHYA*, *PHYB*, and *CRY2* promoters are down-regulated, whereas the activities of the *PHYC* and *CRY1* promoters are up-

Figure 6. Circadian accumulation of phytochrome and cryptochrome mRNA in LL. Wild-type Arabidopsis seedlings (WS ecotype) were grown under LD cycles for 1 week, and were then transferred to LL. Abundance of the phytochrome- and cryptochrome-specific mRNAs was measured in samples harvested in 4-h intervals by RNase protection assays using 30 μg of total RNA per lane. For *CAB2* mRNA determination, 15 μg of total RNA was analyzed by northern blots hybridized with the coding region of the *CAB2* gene (B). The measurement of the *UBQ10* mRNA abundance was included in all experiments as an internal control. The radioactive signals of the protected fragments were quantified by PhosphorImager and normalized to the corresponding *UBQ10* signals, and then to the highest value of the normalized test gene signals. Because the experiments were highly reproducible, only one set of the autoradiograms is shown for each gene. White box on time axis, Light interval; gray box, subjective dark interval.

regulated by light (Fig. 5). The same figure also shows that the expression of the PHYD and PHYE promoters was not affected significantly by the changes of light conditions. These observations are consistent with the results of earlier studies employing promoter:GUS reporters to study the light-regulated expression of the PHY promoters (Somers and Quail, 1995; Goosey et al., 1997). However, our data concerning the effect of light on the activities of the PHYC and CRY1, CRY2 promoters differ from previous results published by Clack et al. (1994), Ahmad and Cashmore (1993), and Lin et al. (1998). These authors reported that the accumulation of mRNA transcribed from these genes is unaffected by light. This apparent difference may be due to the different experimental setups (developmental stage of seedlings, frequency of sampling, and higher sensitivity of the assay used for the present study) or to the fact that light differentially regulates mRNA accumulation and promoter activity of these genes.

### Diurnal- and Circadian-Regulated Expression

We demonstrate that the promoter activities of the PHY and CRY genes follow a diurnal rhythm and exhibit maximum expression in the light phase (Fig. 2). Furthermore, we show that these oscillations persist under LL and DD conditions with a period close to 24 h, proving that a circadian clock regulates the expression of these promoters (Figs. 3 and 4). Measurements of mRNA transcribed from these genes in seedlings transferred to LL indicate that the rhythmic expression is maintained at the level of mRNA accumulation (Fig. 6). These findings are consistent with our earlier results regarding PHYB (Kozma-Bognár et al., 1999) and with more recent data on CRY1, CRY2, and PHYA, PHYB mRNA levels derived from microarray experiments (Harmer et al., 2000; Schaffer et al., 2001).

We present the first evidence for the circadian regulation of PHYC, PHYD, and PHYE genes in this study. Based on our results, these genes can be used as new molecular markers to study circadian-regulated gene expression in Arabidopsis. However, the PHY and CRY genes form a special subgroup of the circadian markers because they can be placed in the input and the output pathways, as well. The temporal regulation of PHYC expression is different from that of the other PHY genes. The diurnal rhythm of PHYC::LUC⁺ activity indicates direct regulation by light rather than anticipation of light transitions. The circadian expression pattern of this construct is characterized by a very low amplitude with a peak at ZT 24 (versus ZT 10–12 for the diurnal rhythm) and by continuously increasing or decreasing expression levels in LL or DD, respectively. In contrast, PHYC mRNA accumulation exhibits a circadian rhythm with remarkable amplitude and peaks of expression at ZT 24, ZT 48, etc. These data suggest

Circadian Regulation of AtPHYA-E and AtCRY1-2 Gene Expression

that the relatively strong light dependence of PHYC expression probably masks the effect of circadian regulation on the level of promoter activity and that circadian regulation affects accumulation of PHYC mRNA more dominantly, acting most probably on RNA stability.

Our data revealed striking similarities between the expression patterns of the PHYA and CRY2 promoters. The activities of these promoters follow the same spatial pattern. They are down-regulated by light and exhibit circadian oscillations with nearly identical characteristics, including amplitude and phase. Moreover, to enumerate further similarities, both proteins were shown to be light labile. These observations indicate that these photoreceptors are functioning primarily at low-light intensities.

Comparison of timing of maximum level expression of the various promoter::luciferase constructs allowed us to position the circadian phases of phytochrome and cryptochrome gene expression around a clock (Fig. 7). It is interesting that this comparison shows that the photoreceptor genes coding for relatively light-stable proteins (PHYC, PHYD, and PHYE) are intensively transcribed at the beginning or in the first one-half (PHYB and CRY1) of the light phase. It follows that the dramatic increase in light intensity at the beginning of the light phase can be accompanied by an increased accumulation of these receptors. We speculate that the newly synthesized masses of photoreceptors then mediate efficient adaptation of a variety of light-dependent processes (inhibition of hypocotyl and stem elongation, induction of genes coding for components of the photosynthetic machinery, and resetting the circadian clock) to these light conditions. By contrast, the expression of the PHYA and CRY2 genes coding for photolabile receptors reaches maximum close to the end of the light interval. It is characteristic of light signals, at

![Figure 7](#fig7)

**Figure 7.** A circle diagram illustrating the relative phases of peak activity of the various reporter constructs as determined in the measurements under LL (see Fig. 2). The time of the day is presented as the face of a 24-h clock. ZT 0 to ZT 12, Light interval; ZT 12 to ZT 24, subjective dark interval. Genes with similar timing of peak expression are grouped and boxed. Arrows point to the specific time of the peak activity of the individual groups.

Tóth et al.

this part of the day, that they have low intensities yet regulate important physiological responses (e.g. end-of-the-day far-red response). Therefore, the specifically timed maximum level transcription of the PHYA and CRY2 genes may mediate optimal adaptation of plants to low-intensity light conditions.

We demonstrate here that the circadian clock regulates promoter activity and/or mRNA accumulation of PHY and CRY genes. This observation indicates the presence of an additional regulatory loop within the plant circadian system (Fig. 8). It is proposed that this regulatory loop ensures maximal efficiency in the perception of the resetting light signals at the right times and neutralization of signals from non-predictable environmental cues, which could cause resetting of the circadian clock. In addition, this postulated regulatory loop can also mediate the generation of more robust rhythms with higher amplitude under relatively constant conditions.

An intimate association between the oscillator and the components of the input pathway has been described in a number of organisms such as cyanobacteria (Iwasaki et al., 2000), *Neurospora crassa* (Heintzen et al., 2001; Merrow et al., 2001), *D. melanogaster* (Ceriani et al., 1999), and mouse (Shigeyoshi et al., 1997). For example, in *N. crassa*, this is manifested in the formation of a variety of feedback loops whose exact relation to each other and function in the circadian system is still debated. Regarding higher plants, our data suggest the existence of similarly complex regulatory circuit(s). It has been shown that light-controlled nucleo/cytoplasmic partitioning of PHYA and PHYB is an important regulatory step in phototransduction mediated by these photoreceptors (Nagy and Schäfer, 2000b). To fulfill the above-proposed functions, the postulated regulatory loop should also be operative at the level of photoreceptor accumulation and/or subcellular localization. Data published so far indicate that the total amount of PHYA and PHYB proteins does not vary significantly under extended LD, LL, or DD conditions (Kozma-Bognár et al., 1999). It has been shown, however, that PHYB interacts with a variety of molecules, including the input regulators CRY2 (Mas et al., 2000), ADA-GIO (Jarillo et al., 2001), and the transcription factor PIF3 (Ni et al., 1999). Recent data indicate that ELF3, as a part of the so-called zeitnehmer loop, is also an important factor generating rhythmic light input to

the oscillator even under relative constant conditions (McWatters et al., 2000). Our data suggest a similar (but not the same) role for the PHY molecules, but the underlying molecular mechanism remains to be elucidated.

## MATERIALS AND METHODS

### Promoter::Luciferase Fusions, Plant Materials, and Growth Conditions

The PHYA-E and CRY1 promoter fragments were obtained by PCR reactions performed on genomic DNA isolated from Arabidopsis (WS ecotype) plants. Unique restriction sites were designed at the 5′ and 3′ ends of the promoter fragments to facilitate cloning in the pPCV 812 binary vector (Koncz and Schell, 1986) containing the modified luciferase (*LUC⁺*) reporter gene (Promega, Madison, WI) with the 3′-terminator sequences of the nopaline synthase gene. All of the amplified fragments contained the entire 5′-untranslated region, but not the ATG of the corresponding genes. The fragment lengths and the unique restriction sites at the 5′ and 3′ ends were the following: PHYA, 2,357 bp, EcoRI-BamHI; PHYB, 2,292 bp, HindIII-BamHI; PHYC, 2,385 bp, EcoRI-SmaI; PHYD, 2,310 bp, SalI-SmaI; PHYE, 2,883 bp, HindIII-BamHI; and CRY2, 2,901 bp, HindIII-BamHI. The CRY1 promoter fragment, 1,004 bp in length bordered by EcoRI and StuI sites, was a gift of Anthony R. Cashmore (Plant Science Institute, Department of Biology, University of Pennsylvania, Philadelphia). The identity of the promoter fragments was verified by restriction digests and sequencing. The constructs were transformed into Arabidopsis (WS) plants by the *Agrobacterium tumefaciens*-mediated transformation method (Clough and Bent, 1998). Transformants were selected on Murashige and Skoog medium supplemented with 15 μg mL⁻¹ of hygromycin. Ten to 15 independent transformants from each group were planted in soil, selfed, and the individuals of the F₂ progeny were used for luminescence assays. It is notable that differences in spatial and temporal expression pattern among the independent transformant lines for a given construct have not been observed, but there were strong variations in the level of expression among the lines. Lines for the analysis were chosen to represent the entire range of expression levels: usually one low-, two medium-, and one high-expressing line of a given construct were included in each experiment.

Transgenic seeds carrying the PHYA-E::LUC⁺ and CRY1-2::LUC⁺ chimeric transgenes were surface sterilized, sowed on sterile Murashige and Skoog medium containing 3% (w/v) Suc, solidified with 1% (w/v) agar, and were then stratified at 4°C for 2 d. Seedlings were then grown in a phytochamber (MLR-350, Sanyo, Gallenkamp, UK) at 22°C with 12-h-light (60–70 μM m⁻² s⁻¹, white fluorescent)/12-h-dark photoperiods for 7 d.

### Luminescence Assays

Luciferase luminescence was measured by low-light video imaging using a liquid nitrogen-cooled CCD camera

Figure 8. A working model of the plant circadian system incorporating the regulatory loop from the output to the input photoreceptors.
(Princeton Instruments, Trenton, NJ). Groups of 20 to 25 seedlings were germinated on Murashige and Skoog plates and were entrained as described above. Plants were sprayed with 5 mM luciferin solution (Biosynth AG, Switzerland) three times, 36, 24, and 12 h prior to the start of the imaging. Imaging started on the 8th d after germination at the beginning of the light phase (ZT 0). During the LL experiments, the seedlings were transferred to white LL (60–70 μM m⁻² s⁻¹) at ZT 0. Alternatively, during the DD measurements, seedlings were transferred to DD at ZT 12. All experiments were performed at constant (22°C) temperature. Images were taken every 2 h (hourly for DD data acquisition), and exposure times were 15 min for PHYA,C::LUC⁺, CRY2::LUC⁺, and CAB2::LUC⁺ plants, or 25 min for PHYB,D,E::LUC⁺ and CRY1::LUC⁺ plants. Brightness of areas containing groups of seedlings was measured by the MetaView software, corrected for background counts, and was normalized to the average value of luminescence of individual lines, as detected during the measurements. Normalized data were graphed as a function of time using Excel (Microsoft, Redmond, WA). Measurements were repeated at least three times on three or four independent transgenic lines for each construct with very similar results. Alternatively, to study the long-term light regulation of the various constructs, the luciferase activity of individual seedlings was measured in a Topcount NXT luminometer (Packard Instruments) as described by Carré and Kay (1995).

### RNA Assays

Total RNA was extracted as described (Adam et al., 1994) from whole seedlings entrained for 7 d and was then transferred to LL as described above. Samples were harvested every 4 h. The abundance of the specific mRNAs was determined by RNase protection assays. The gene-specific probes were obtained as short fragments of the coding region individual genes, amplified by PCR, and then cloned in pBluescript plasmid. The 5′- and 3′-end positions of the probe fragments referring to the nucleotides in the appropriate GenBank data files were: PHYA, 3,108 through 3,347 (X17341); PHYB, 3,348 through 3,565 (X17342); PHYC, 291 through 565 (X17343); PHYD, 2,888 through 3,177 (X76609); PHYE, 804 through 1,126 (X76610); CRY1, 111,223 through 111,397 (AL161513); CRY2, 1,261 through 1,542 (U43397); and UBQ10, 1,008 through 1,151 (L05361). Labeling of antisense RNA probes and subsequent steps of the RNase protection assays were performed as described by Adam et al. (1996). Thirty micrograms of total RNA was hybridized with the mix of the necessary gene probe and the UBQ10 probe. For CAB2 mRNA measurements, 15 μg of total RNA per lane was analyzed by northern hybridization using the CAB2 coding region probe (Millar et al., 1992). After exposure, blots were washed and rehybridized with the UBQ10 coding region probe. Radioactive signals were visualized in a PhosphorImager (Molecular Dynamics, Sunnyvale, CA) and were quantified using the ImageQuant 1.1 software. Ratios between the individual PHY or CRY signals and the corresponding UBQ10 signals were calculated and normalized to the highest value. Experiments were performed two or three times and were highly reproducible; one representative set of data is shown in Figure 6.

## ACKNOWLEDGMENTS

We are grateful to Rózsa Nagy for excellent technical assistance and Erzsébet Fejes for critical reading of the manuscript. We thank Anthony Cashmore and Alfred Batschauer for providing the CRY1 promoter fragment and a genomic clone containing the CRY2 promoter, respectively.

Received May 25, 2001; returned for revision July 30, 2001; accepted September 1, 2001.

## LITERATURE CITED

Adam E, Kozma-Bognár L, Kolar C, Schäfer E, Nagy F (1996) The tissue-specific expression of tobacco phytochrome B gene. Plant Physiol 110: 1081–1088

Adam E, Szell M, Szekeres M, Schäfer E, Nagy F (1994) The developmental and tissue-specific expression of tobacco phytochrome A genes. Plant J 6: 283–293

Ahmad M, Cashmore AR (1993) HY4 gene of *A. thaliana* encodes a protein with characteristics of blue-light photoreceptor. Nature 366: 162–166

Carré IA, Kay SA (1995) Multiple DNA-protein complexes at a circadian-regulated promoter element. Plant Cell 7: 2039–2051

Casal JJ, Cerdan PD, Staneloni RJ, Cattaneo L (1998) Different phototransduction kinetics of phytochrome A and phytochrome B in *Arabidopsis thaliana*. Plant Physiol 116: 1533–1538

Ceriani MF, Darlington TK, Staknis D, Mas P, Petti AA, Weitz CJ, Kay SA (1999) Light-dependent sequestration of TIMELESS by CRYPTOCHROME. Science 285: 553–556

Clack T, Mathews S, Sharrock RA (1994) The phytochrome apoprotein family in *Arabidopsis* is encoded by five genes: the sequences and expression of PHYD and PHYE. Plant Mol Biol 25: 413–427

Clough SJ, Bent AF (1998) Floral dip: a simplified method for Agrobacterium-mediated transformation of *Arabidopsis thaliana*. Plant J 16: 735–743

Devlin PF, Kay SA (2000) Cryptochromes are required for phytochrome signaling to the circadian clock but not for rhythmicity. Plant Cell 12: 2499–2510

Emery P, Stanewsky R, Hall JC, Rosbash M (2000) A unique circadian-rhythm photoreceptor. Nature 404: 456–457

Furuya M, Schäfer E (1996) Photoperception and signaling of induction reactions by different phytochromes. Trends Plant Sci 1: 301–307

Glossop NR, Lyons LC, Hardin PE (1999) Interlocked feedback loops within the *Drosophila* circadian oscillator. Science 286: 766–768

Goosey L, Palecanda L, Sharrock RA (1997) Differential patterns of expression of the *Arabidopsis* PHYB, PHYD,
Tóth et al.

and PHYE phytochrome genes. Plant Physiol 115: 959–969

Harmer SL, Hogenesch JB, Straume M, Chang HS, Han B, Zhu T, Wang X, Kreps JA, Kay SA (2000) Orchestrated transcription of key pathways in *Arabidopsis* by the circadian clock. Science 290: 2110–2113

Heintzen C, Loros JJ, Dunlap JC (2001) The PAS protein VIVID defines a clock-associated feedback loop that represses light input, modulates gating, and regulates clock resetting. Cell 104: 453–464

Hoffman PD, Batschauer A, Hays JB (1996) PHH1, a novel gene from *Arabidopsis thaliana* that encodes a protein similar to plant blue-light photoreceptors and microbial photolyases. Mol Gen Genet 253: 259–265

Iwasaki H, Williams SB, Kitayama Y, Ishiura M, Golden SS, Kondo T (2000) A kaiC-interacting sensory histidine kinase, SasA, necessary to sustain robust circadian oscillation in cyanobacteria. Cell 101: 223–233

Jarillo JA, Capel J, Tang RH, Yang HQ, Alonso JM, Ecker JR, Cashmore AR (2001) An *Arabidopsis* circadian clock component interacts with both CRY1 and phyB. Nature 410: 487–490

Kircher S, Kozma-Bognár L, Kim L, Adam E, Harter K, Schäfer E, Nagy F (1999) Light quality-dependent nuclear import of the plant photoreceptors phytochrome A and B. Plant Cell 11: 1445–1456

Koncz C, Schell J (1986) The promoter of T-DNA gene 5 controls the tissue-specific expression of chimeric genes carried by a novel type of Agrobacterium binary vector. Mol Gen Genet 204: 383–396

Kozma-Bognár L, Hall A, Adam E, Thain SC, Nagy F, Millar AJ (1999) The circadian clock controls the expression pattern of the circadian input photoreceptor, phytochrome B. Proc Natl Acad Sci USA 96: 14652–14657

Lin C, Yang H, Guo H, Mockler T, Chen J, Cashmore AR (1998) Enhancement of blue-light sensitivity of *Arabidopsis* seedlings by a blue light receptor cryptochrome 2. Proc Natl Acad Sci USA 95: 2686–2690

Mas P, Devlin PF, Panda S, Kay SA (2000) Functional interaction of phytochrome B and cryptochrome 2. Nature 408: 207–211

Merrow M, Franchi L, Dragovic Z, Gorl M, Johnson J, Brunner M, Macino G, Roenneberg T (2001) Circadian regulation of the light input pathway in *Neurospora crassa*. EMBO J 20: 307–315

Millar AJ, Short SR, Chua NH, Kay SA (1992) A novel circadian phenotype based on firefly luciferase expression in transgenic plants. Plant Cell 4: 1075–1087

McWatters HG, Bastow RM, Hall A, Millar AJ (2000) The ELF3 zeitnehmer regulates light signaling to the circadian clock. Nature 408: 716–720

Nagy F, Schäfer E (2000a) Nuclear and cytosolic events of light-induced, phytochrome-regulated signaling in higher plants. EMBO J 19: 157–163

Nagy F, Schäfer E (2000b) Control of nuclear import and phytochromes. Curr Opin Plant Biol 3: 450–454

Neff MM, Fankhauser C, Chory J (2000) Light: an indicator of time and place. Genes Dev 14: 257–271

Ni M, Tepperman JM, Quail PH (1999) Binding of phytochrome B to its nuclear signalling partner PIF3 is reversibly induced by light. Nature 400: 781–784

Quail PH, Boylan MT, Parks BM, Short TW, Xu Y, Wagner D (1995) Phytochromes: photosensory perception and signal transduction. Science 268: 675–680

Schaffer R, Landgraf J, Accerbi M, Simon V, Larson M, Wisman E (2001) Microarray analysis of diurnal and circadian-regulated genes in *Arabidopsis*. Plant Cell 13: 113–123

Sharrock RA, Quail PH (1989) Novel phytochrome sequences in *Arabidopsis thaliana*: structure, evolution and differential expression of a plant regulatory receptor family. Genes Dev 3: 1745–1757

Shearman LP, Sriram S, Weaver DR, Maywood ES, Chaves I, Zheng B, Kume K, Lee CC, van der Horst GT, Hastings MH et al. (2000) Interacting molecular loops in the mammalian circadian clock. Science 288: 1013–1019

Shigeyoshi Y, Taguchi K, Yamamoto S, Takekida S, Yan L, Tei H, Moriya T, Shibata S, Loros JJ, Dunlap JC et al. (1997) Light-induced resetting of a mammalian circadian clock is associated with rapid induction of the mPer1 transcript. Cell 91: 1043–1053

Somers DE, Devlin PF, Kay SA (1998) Phytochromes and cryptochromes in the entrainment of the *Arabidopsis* circadian clock. Science 282: 1488–1490

Somers DE, Quail PH (1995) Temporal and spatial expression patterns of PHYA and PHYB genes in *Arabidopsis*. Plant J 7: 413–427

Yamaguchi R, Nakamura M, Mochizuki N, Kay SA, Nagatani A (1999) Light-dependent translocation of a phytochrome B-GFP fusion protein to the nucleus in transgenic *Arabidopsis*. J Cell Biol 3: 437–445

van der Horst GT, Muijtjens M, Kobayashi K, Takano R, Kanno S, Takao M, de Wit J, Verkerk A, Eker AP, van Leenen D et al. (1999) Mammalian Cry1 and Cry2 are essential for maintenance of circadian rhythms. Nature 398: 627–630
