
Res. Microbiol. 152 (2001) 861–872  
© 2001 Éditions scientifiques et médicales Elsevier SAS. All rights reserved  
S0923-2508(01)01269-4/FLA

# Fluorene degradation by *Sphingomonas* sp. LB126 proceeds through protocatechuic acid: a genetic analysis

Pierre Wattiau${}^{a*}$, Leen Bastiaens${}^{b}$, René van Herwijnen${}^{c}$, Ludwin Daal${}^{c}$, John R. Parsons${}^{c}$, Marie-Eve Renard${}^{a}$, Dirk Springael${}^{b}$, Guy R. Cornelis${}^{a}$

${}^{a}$Catholic University of Louvain (UCL), Microbial Pathogenesis Unit, Av. Hippocrate 74, box MIPA 7449, 1200 Brussels, Belgium  
${}^{b}$Flemish Institute for Technological Research (VITO), Boeretang 200, 2400 Mol, Belgium  
${}^{c}$University of Amsterdam, Department of Environmental and Toxicological Chemistry (IBED-ARISE), Nieuwe Achtergracht 166, 1018WV Amsterdam, The Netherlands  

Received 17 January 2001; accepted 21 June 2001

## Abstract – *Sphingomonas* sp. LB126 is able to utilize fluorene as sole source of carbon and energy. In the present study, a mutagenic vector was constructed and a “plasmid rescue” strategy was set up to isolate a 16.5-kb DNA fragment containing genes required for fluorene degradation. A 14.5-kb portion of the cloned DNA was sequenced revealing thirteen open reading frames. Two encoded hypothetical proteins (FldE and FldY) similar to transcriptional regulators and one (ORF360) located on an IS-like element (ISSpI26) encoded a putative transposase. Three other putative proteins (FldB, FldU and FldV) displayed strong similarity with enzymes of the protocatechuate 4,5-degradation pathway utilized by *Sphingomonas paucimobilis* SYK-6 for the degradation of lignin breakdown products. The remaining hypothetical proteins displayed only limited similarity with enzyme sequences available from databases. Suicide plasmid-directed mutagenesis and genetic complementations showed that integrity of the protocatechuate catabolic pathway was an absolute requirement for fluorene degradation to proceed. These findings were further supported by the analysis of metabolites in bacterial culture supernatants obtained from appropriate mutants. The results presented here demonstrated the suitability of the genetic tool constructed and supplied the first genetic evidence for the participation of a protocatechuate 4,5-degradation pathway in a bacterial fluorene degradation pathway. © 2001 Éditions scientifiques et médicales Elsevier SAS

### polyaromatic hydrocarbons / *Sphingomonas* / fluorene / protocatechuic acid / microbial genetics

## 1. Introduction

Polyaromatic hydrocarbons (PAHs) are hydrophobic pollutants that can persist in the environment for many years due to their low water solubility and their strong tendency to sorb to soil particles as well as to their low degradability. Because of its abundance and toxicity, fluorene has been classified as a priority pollutant by the US Environmental Protection Agency together with 15 other PAHs [16]. Several bacteria able to use fluorene as sole carbon source have been isolated and three major degradative routes have been described. *Arthrobacter* sp. F101 [8, 12], *Burkholderia cepacia* F297 [14] and *Mycobacterium* sp. BB1 [6] initiate fluorene metabolism by a 3,4-dioxygenation of the fluorene molecule which leads to salicylate formation, while *Arthrobacter* sp. F101 and *Staphylococcus auriculans* DBF63 [8, 22] initiate fluorene breakdown by a 1,2-dioxygenation. *Brevibacterium* sp. DPO1361 [36] and *Pseudomonas* sp. F274 [13] initiate monooxygenation at C-9. This route is only productive if a subsequent angular-carbon dioxygenation occurs (*figure 1*), leading to the formation of phthalic acid, which is further metabolized to protocatechuic acid (PCA). The fluorene degradation pathways were proposed on the basis of metabolite analysis. Very little is known about the enzymes participating in fluorene degradation in the relevant bacteria and no information is available on the corresponding genes. Recently, the complete sequence of the 184-kb catabolic plasmid pNL1 from

* Correspondence and reprints.  
Present address: Catholic University of Louvain, Bioengineering Unit, Place Croix du Sud 2 P. box 19, B-1348 Louvain-la-Neuve, Belgium.  
E-mail address: wattiau@gebi.ucl.ac.be (P. Wattiau).

P. Wattiau et al. / Res. Microbiol. 152 (2001) 861–872

![Chemical Structures](https://i.imgur.com/chemical_structures.png)

Figure 1. Putative fluorene degradation pathway used by *Sphingomonas* sp. LB126. PDC: 2-pyrone-4,6-dicarboxylic acid, CHMS: 4-carboxy-2-hydroxymuconate-6-semialdehyde, CHM: 4-carboxy-2-hydroxymuconic acid. Not all intermediate metabolites are drawn. The part of the pathway leading to PCA is also present in *Brevibacterium* sp. DPO1361 [36] and *Pseudomonas* sp. F274 [13], whereas the degradation of PCA by 4,5-dioxygenation is common to *Pseudomonas* sp. F274 [13] and to *Sphingomonas* sp. SYK-6 [23].

*Sphingomonas aromaticivorans* F199 has been published [29]. This bacterium is able to degrade toluene and several PAHs including fluorene. However, to date the function of the pNL1 catabolic genes is still unknown. *Sphingomonas* sp. LB126 is able to assimilate fluorene but not other PAHs with similar structure including dibenzothiophene and fluoranthene. In this study, 16.9 kb of contiguous genomic DNA from *Sphingomonas* sp. LB126 were isolated, of which 14.5 kb were sequenced. Two transposon insertion mutants impaired in fluorene degradation were isolated (see accompanying paper for details [4]). They were mapped and the role of some other relevant genes was analyzed. This genetic analysis demonstrated the existence of a pathway in which PCA is an obligatory intermediate metabolite.

### 2. Materials and methods

#### 2.1. DNA manipulation, enzymes and chemicals

Molecular DNA techniques were carried out essentially as described [31]. All molecular biology enzymes used were purchased from Amersham, except T4 DNA ligase and Taq DNA polymerase (Roche Molecular Biochemicals). Unless specified, all chemicals were purchased from Sigma.

#### 2.2. Bacterial strains, antibiotic selection and growth conditions

Plasmids used are listed in table I. *Sphingomonas* sp. LB126 was isolated from a PAH-contaminated soil by liquid enrichment using fluorene as sole source of carbon and energy [3]. LB126 is naturally resistant to streptomycin. It was grown routinely in phosphate minimal medium, which is identical to Tris medium [20] but with 50 mM phosphate (pH 7.4) instead of Tris and supplemented with either 0.4% (w/v) glucose or with fluorene as sole carbon source. Growth on fluorene on phosphate agar (15 g L⁻¹) plates was assessed by directly spreading gross fluorene crystals on top of the agar streaked with the relevant *Sphingomonas* strains. Growth on PCA was assessed in liquid phosphate medium supplemented with 500 mg L⁻¹ of PCA (pH 7.3). L-132 and L-134 are mini-Tn5-luxAB-tet transposon [9] insertion mutants of LB126, the construction of which is described in the accompanying paper [4]. *Escherichia*

Table I. List of plasmids used in this study.

| Plasmid       | Size    | Characteristics/construction                                                                                          | Reference     |
|---------------|---------|---------------------------------------------------------------------------------------------------------------------|---------------|
| pALTER-1      | 5.68 kb | $ori_{pBR322}$, $ori_{F1}$, $lacZ'$, tet, $bla'$, Tet$^R$, Amp$^S$                                                   | Promega$^{TM}$|
| pPW62         | 6.45 kb | $oriT_{RK2}$ from pTJS82 [32] cloned in ScaI of pALTER-1 as a BamHI fragment made blunt end with Klenow enzyme            | This work     |
| pPW77         | 6.45 kb | pPW62 with a disrupted $lacZ'$ following XbaI-HindIII deletion and T4 DNA polymerase polishing                           | This work     |
| pPW78         | 6.45 kb | pPW77 with a functional $lacZ'$ following EcoRI digestion and Mung bean nuclease treatment                               | This work     |
| pPW1321       | 5.4 kb  | pBlueScript KSII(−) (Stratagene$^{TM}$) + 2.5 kb NotI fragment from L-132                                              | This work     |
| pPW1325       | 6.95 kb | pPW78 + $fldA$ intragenic fragment (coordinates 5745-6114*); $fldA$ mutator plasmid                                     | This work     |
| pPW82         | 23.5 kb | rescued by BglII digestion and ligation of L-613 ($fldA$: pPW1325) genomic DNA                                         | This work     |
| pPW101        | 3.5 kb  | pBlueScript KSII(−) + 1.58 kb NotI fragment from pPW82 containing $fldB$ and part of $fldA$ (coordinates 6107-7689*)      | This work     |
| pPW121        | 6.75 kb | pPW78 + $fldB$ intragenic fragment (coordinates 6555-6776*); $fldB$ mutator plasmid                                    | This work     |
| pPW122        | 6.95 kb | pPW78 + $fldC$ intragenic fragment (coordinates 7922-8389*); $fldC$ mutator plasmid                                    | This work     |
| pPW123        | 6.85 kb | pPW78 + $fldD$ intragenic fragment (coordinates 9716-10083*); $fldD$ mutator plasmid                                   | This work     |
| pBBR1MCS-2    | 5.14 kb | $ori_{pBBR1}$, $lacZ'$, $npt$, $oriT_{RK2}$, Km$^R$                                                                  | [2]           |
| pPW131        | 6.15 kb | pBBR1MCS-2 + PCR-amplified $fldA$                                                                                  | This work     |
| pPW132        | 6.05 kb | pBBR1MCS-2 + PCR-amplified $fldB$                                                                                  | This work     |

*Sequence coordinates refer to the databank entry AJ277295.*

coli LK111 is a F$^+$ derivative of C600 carrying the $lacZ\Delta M15$ mutation and was used as the host strain in all cloning experiments (received from M. Zabeau, Ghent, Belgium). The conjugative strain E. coli SM10$\lambda$pir [21] was used to deliver the mini-transposon and the pPW78 derivatives to LB126, whereas E. coli S17-1 [33] was used to deliver the pBBR1MCS-2 derivatives. Antibiotics were used at the following concentrations: tetracycline, 4 $\mu$g mL$^{-1}$ (Sphingomonas) or 12 $\mu$g mL$^{-1}$ (E. coli); kanamycin, 15 $\mu$g mL$^{-1}$ (Sphingomonas) or 50 $\mu$g mL$^{-1}$ (E. coli); streptomycin, 100 $\mu$g mL$^{-1}$ (both E. coli and Sphingomonas); ampicillin, 200 $\mu$g mL$^{-1}$ (E. coli). All E. coli strains were grown on tryptic soy agar or in tryptic soy broth (Difco).

### 2.3. Conjugational transfers

Transfer of mobilizable plasmids was obtained by biparental mating using the conjugative E. coli strains SM10$\lambda$pir and S17-1. One hundred microliters of highly concentrated cell suspensions (OD$_{600}$ approximately equal to 30) derived from fresh cultures of each donor and recipient strain were gently mixed and laid on a TSA agar plate at 30°C. After overnight incubation, the resulting spots were collected and homogenized in 500 $\mu$L of 0.9% (w/v) NaCl. Serial dilutions were plated on phosphate agar plates supplemented with 0.2% (w/v) glucose and the relevant antibiotics. Colonies of Sphingomonas sp. LB126 recombinants appeared after 4–7 days and were purified on the same medium before being further characterized.

### 2.4. Construction of a mutagenic vector for Sphingomonas

A mobilizable derivative of pALTER-1 (Promega) was constructed by cloning the minimal mobilization region of RK2/RP4 borne on pTJS82 [32] in the unique ScaI site of pALTER-1. The resulting plasmid, pPW62, was further modified by a XbaI-HindIII deletion followed by T4 DNA polymerase polishing, leading to pPW77 which carries a disrupted $\alpha$-peptide. In the final construct pPW78, the correct reading frame of the $\alpha$-peptide gene was restored by polishing the EcoRI restriction site of pPW77 with mung bean nuclease.

### 2.5. Suicide plasmid-directed mutagenesis and DNA rescuing

A 2.5-kb DNA fragment recovered from the L-132 chromosome was cloned as a NotI fragment in pBlue-

Script KSII(−) (Stratagene) to yield pPW1321. The inserted DNA contained the tetracycline resistance gene originating from the mini-Tn5-luxAB transposon and 0.37 kb of contiguous chromosomal DNA. This fragment was further subcloned in pPW78, resulting in the mutator plasmid pPW1325. Mobilization of pPW1325 in *Sphingomonas* sp. LB126 and integration of the plasmid in the new host genome was achieved as described above and resulted in the *fldA* mutant L-613. To recover the pPW1325 plasmid from its integrated form in L-613, 8 μg of genomic DNA obtained by the method of Marmur [17] were digested with BglII in a total volume of 100 μL. After phenol/chloroform extraction and ethanol precipitation, the digested DNA was incubated overnight at 16°C with 5 units of T4 DNA ligase in 200 μL of the appropriate buffer supplied by the manufacturer (Boehringer). The ligated DNA was electroporated in freshly prepared *E. coli* LK111 cells. The resulting circular plasmid (pPW82) was 23.5 kb in size. The insert of this plasmid was subcloned to allow nucleotide sequencing and construction of mutator plasmids.

### 2.6. Construction of *fldA* and *fldB* complementing plasmids

Two sets of primers with the following sequences were used to amplify the *fldA* and *fldB* genes by PCR:

- FLDAF-832: AAAAAGCTTAAGAAGGAGATA-TACATATGTCTGATTCCGCCCGC
- FLDAR-833: GGGGGATCCTCATACGAAAAC-CTCTCCATCCATCAA
- FLDBF-831: AAAAAGCTTAAGAAGGAGATA-TACATATGAGTGCCCAAGACACC GCC
- FLDBR-834: GGGGGATCCTTATTCGCCC GGG-CCAGTAGAGG

The PCR products were digested with BamHI and HindIII and cloned in the corresponding sites of pBBR1-MCS2 [2]. The resulting plasmids, pPW131 and pPW132, were transferred to *Sphingomonas* by conjugation as described above. They respectively expressed *fldA* and *fldB* from both the *lac* and kanamycin resistance gene promoters of the vector, downstream of the *phi10* ribosome binding site (sequence included in the amplimer).

### 2.7. Nucleotide sequencing and computing

The nucleotide sequence of a 14,034-bp segment of LB126 chromosomal DNA borne on pPW82 was determined. This sequence extended from the *BglII* site located 5401 bp upstream of the transposon insertion in L-132, to the next *XbaI* site. The nucleotide sequence of an extra 343-bp DNA fragment was obtained by sequencing a PCR product amplified with a set of primers corresponding to the right end of the mini-transposon inserted in L-134 (primer MIPA417: CCATCGCGTCCGCCATCTCCAG) and to the 5′ coding sequence of *fldU* (primer MIPA763: CTGGCATATCGCCCAGAG). The nucleotide sequence of *fldA* was confirmed by direct sequencing of the PCR product resulting from its amplification. The reconstructed 14,377-bp sequence is available from the EMBL database (accession AJ277295). The pPW78 vector sequence (accession AJ272004) was extrapolated from the nucleotide sequence of the parent plasmids and from the sequences obtained for the modified regions. Protein sequence comparisons were performed using BLAST [1]. Transmembrane domains were predicted using TopPred [37].

### 2.8. Metabolite analysis

The mutant strains L-132 and L-134 were grown in 250 mL phosphate minimal medium supplemented with 0.4% glucose, 0.1 mg mL⁻¹ fluorene and the relevant antibiotics. Cultures were sampled while in exponential growth phase. The filtrate was extracted with 50% (v/v) ethyl acetate at pH 6.8 and at pH 1. Derivatization was performed in 50 mL acetone with 2.6 g K₂CO₃ and 3 mL iodomethane and followed by a clean-up with 0.8 g florisil using pentane and dichloromethane (10 mL each) as eluants. Fractions were collected separately, concentrated and dissolved in 2,2,4-trimethylpentane before being analyzed on an HP 5960 gas chromatograph (Hewlett-Packard) using a DB-5 column (J&W Scientific) and an HP 5970 mass selective detector (Hewlett-Packard).

## 3. Results

### 3.1. Cloning of genes participating in fluorene degradation

Genes required for fluorene degradation by *Sphingomonas* sp. LB126 were isolated from the L-132 mini-Tn5-luxAB-tet transposon insertion mutant described in the accompanying paper [4]. Growth of the L-132 mutant with fluorene as sole carbon source was drastically reduced as a consequence of the mini-transposon insertion (the doubling time was estimated

to be five times slower compared to the wild type). In addition, the promoterless *luxAB* reporter gene was induced in response to fluorene but not to glucose, suggesting that the transposon inserted in a gene whose expression was specifically associated with fluorene degradation [4]. A so-called “plasmid rescue” strategy was chosen to recover the chromosomal genes mutated in L-132. In a preliminary experiment, we noticed that plasmids carrying the pBBR1 replication origin [2] could be introduced by conjugation and maintained in *Sphingomonas* sp. LB126 while those with the pBR322 origin could not. pBR322-based plasmids were therefore assumed to be effective “suicide vectors”, which could only be maintained in *Sphingomonas* if a recombination event occurred that integrated the plasmid in the host genomic DNA. This strategy was similar to the one developed for the construction of mutagenic plasmids based on defective replication origins [21]. pPW78, a mobilizable suicide vector for *Sphingomonas* spp., was constructed in which the number of restriction sites available for cloning was intentionally reduced, so that recombinant derivatives integrated in the target genome could be excised and rescued using a variety of restriction enzymes. A small genomic fragment recovered from the L-132 mutant was cloned in pPW78. This fragment corresponded to an internal portion of the unknown gene affected by the transposon insertion in L-132. The resulting mutator plasmid, called pPW1325, was integrated in the wild-type strain *Sphingomonas* sp. LB126 as described in Materials and methods, which provided the mutant strain L-613. Southern hybridization confirmed that the mutagenic plasmid pPW1325 integrated at the expected locus in L-613 by a single cross-over (figure 2A). As for L-132, growth of L-613 on fluorene was about five times slower compared to the wild-type strain, confirming that the genes affected by the mutagenic insertions engineered in both strains were required for fluorene degradation. After *BglII* digestion of the L-613 genomic DNA and ligation, a 23.5-kb derivative of the integrated mutagenic plasmid was recovered in *E. coli*. A summary of the mutagenesis/plasmid rescue strategy is shown in figure 3A.

### 3.2. Sequence analysis of the *fld* gene cluster of *Sphingomonas* sp. LB126

The nucleotide sequence of a large contiguous segment of pPW82 was determined and completed with the nucleotide sequence resulting from the mapping

---

**Figure 2.** Southern hybridization of genomic DNA from *Sphingomonas* LB126 mutant derivatives. A, total DNA from wild-type LB126 (lane 1) and *fldA* mutant L-613 (lane 2) digested by *BstEII/DraI* and hybridized with a *fldA* probe; B, total DNA from wild-type LB126 (lane 3) and from *fldB* mutant L-121 (lane 4) digested by *NotI* and hybridized with a *fldB* probe; C, total DNA from wild-type LB126 (lane 5), from *fldC* mutant L-122 (lane 6) and from *fldD* mutant L-123 (lane 7) digested by *MscI* and hybridized with a *fldBCD* probe. Labels along the left side of the pictures are molecular weight standards of 1.6 kb (bottom), 2 kb (middle) and 4 kb (top).

of the L-134 transposon insertion mutant (see Materials and methods). Fourteen open reading frames were identified and called *fld*. *fldA* was the target of the mutation in L-132 and L-613. The genetic organization of the sequenced DNA is depicted in figure 3B and the gene features are summarized in table II. *fldA* encoded a 37.3-kDa protein similar to a hypothetical protein of unknown function (ORF350) identified during the *E. coli* genome sequencing project [5]. The hypothetical FldA protein was predicted to contain at least one transmembrane domain spanning residue 279 to 299. Transmembrane domains were also predicted for the *E. coli* counterpart ORF350. The nucleotide sequence downstream of *fldA* was determined until the next *XbaI* restriction site and turned out to contain 6 genes (see figure 3B). *fldB* shared strong similarity with the *ligI* gene encoding 2-pyrone-4,6-dicarboxylic acid hydrolase (PDC hydrolase), an enzyme participating in PCA degradation by *Sphingomonas paucimobilis* SYK-6 [18]. The start codon of *fldB* overlapped the stop codon of *fldA*, suggesting that the 2 genes were part of an operon. *fldC* and *fldD* encoded proteins of the alcohol and aldehyde dehydrogenase families.

Table II. Open reading frames encoded in the 14,377-bp gene cluster isolated from *Sphingomonas* sp. LB126.

| Gene    | Amino acids | Mol wt | Function                                      | Closest relative                                                                                          | % Identity (overlap) |
|---------|-------------|--------|-----------------------------------------------|-----------------------------------------------------------------------------------------------------------|----------------------|
| $fldA$  | 356         | 37,301 | Unknown                                       | ORF350, *E. coli* K12 [5]                                                                                 | 44.9 (343)           |
| $fldB$  | 295         | 32,921 | Probable PDC hydrolase                        | LigI, PDC hydrolase (*S. paucimobilis* SYK-6) [18]                                                        | 82.3 (288)           |
| $fldC$  | 533         | 57,774 | Putative alcohol dehydrogenase                | Sorbose dehydrogenase (*Gluconobacter oxydans* T-100) [30]                                               | 42.2 (403)           |
| $fldD$  | 504         | 53,424 | Putative aldehyde dehydrogenase               | CymC, Cumic aldehyde dehydrogenase (*P. putida* F1) [10]                                                 | 49.7 (463)           |
| $fldE$  | 293         | 32,672 | Probable transcriptional regulator of the AraC family | PobR regulatory protein (*Rhizobium leguminosarum* B155) [38]                                            | 33.9 (289)           |
| $fldF'$ | 253         | 27,899 | Putative $p$-hydroxybenzoate hydroxylase, truncated | PobA, benzoate hydroxylase (*Rhizobium leguminosarum* B155) [38]                                        | 59.6 (250)           |
| $Orf360$ | 360        | 40,361 | Putative transposase of ISSsp126              | Transposase of IS1168 from *Bacteroides vulgatus* [15]                                                   | 39.0 (348)           |
| $fldT$  | NA          | NA     | Unknown, partial sequence only                | NA                                                                                                        | NA                   |
| $fldU$  | 281         | 31,311 | Probable PCA dioxygenase ($\beta$ subunit)    | LigB, PCA 4,5-dioxygenase $\beta$ chain (*S. paucimobilis* SYK-6) [23]                                  | 70.8 (282)           |
| $fldV$  | 134$^a$     | 15,461 | Probable PCA dioxygenase ($\alpha$ subunit)   | LigA, PCA 4,5-dioxygenase $\alpha$ chain (*S. paucimobilis* SYK-6) [23]                                | 71.3 (129)           |
| $fldW$  | 340         | 37,722 | Putative hydratase                             | LigY, OH-DDVA meta-cleavage compound hydrolase (*S. paucimobilis* SYK-6) [26]                           | 35.9 (334)           |
| $fldX$  | 299         | 31,140 | Putative dehydrogenase                        | MTH1747 unknown protein (*Methanobacterium thermoautotrophicum* $\delta$H) [34]                         | 31.4 (239)           |
| $fldY$  | 383         | 41,231 | Probable regulator of the LysR family         | PcaQ, regulator of the *pca* operon (*A. tumefaciens*) [25]                                              | 24.5 (302)           |
| $fldZ$  | 224         | 23,557 | Putative acyl transferase                     | Unknown ORF (*Pseudomonas* sp. SE83) [19]                                                                | 32.7 (202)           |

NA = not available, PCA = protocatechuic acid, PDC = 2-pyrone-4,6-dicarboxylic acid, OH-DDVA = 2,2',3-trihydroxy-3'-methoxy-5,5'-dicarboxybiphenyl.  
$^a$ GTG start codon; $^b$ alternative translation start.

P. Wattiau et al. / Res. Microbiol. 152 (2001) 861–872

A

B

L-134
Bgl II
T' U V W X Y Z fldA B C D E F' orf360 F"
L-132
L-613
Not I
L-121
L-122
L-123
Xba I
Bgl II
1 kb

Figure 3. A. Mutagenesis and plasmid rescue strategy. Gene disruption was obtained by recombination between a “mutator” plasmid that carried a cloned internal fragment of the gene to be disrupted and the intact copy of that gene in the target genome. The target gene is disrupted by integration of the plasmid. The integrated plasmid flanked by host DNA can be subsequently recovered by restriction of genomic DNA, ligation and transformation in a suitable host (*E. coli*). Bold arrows point to restriction sites that could be used. These should be absent from the plasmid itself. B. Genetic organization of the *fld* genes. Genes are symbolized by block arrows. Vertical flags and pins respectively show the position of the insertion of mini-Tn5-luxAB-tet and suicide plasmids in the mutants used in this study. Flags are drawn according to the orientation of the *luxAB* reporter gene borne on the mini-Tn5 transposon. The solid box in *fldA* corresponds to the DNA fragment cloned in the mutagenic plasmid pPW1325 used as an initial target for the “plasmid rescuing” experiment. The open box delineates ISSsp126 containing the putative transposase ORF360. The nucleotide sequence of the DNA portion lying between *XbaI* and the right-*BglII* site has not been determined.

hyde dehydrogenases families, respectively, and were also likely to be part of an operon. *fldE* encoded a putative transcriptional regulator of the AraC family. The most significant similarity was found with regulators of catabolic genes required for the degradation of *p*-hydroxybenzoate by *Pseudomonas* sp. HR199 and *Rhizobium leguminosarum* [38]. Consistent with this finding, *fldF*, the next gene in our sequence, encoded a putative *p*-hydroxybenzoate hydroxylase fairly similar to PobA of *Rhizobium leguminosarum* [24] and *Azotobacter chroococcum* (J.A. Quinn and B. Entsch, unpublished data). However, the reading frame of *fldF* was interrupted by an insertion sequence (ISSsp126) and hence should not be functional. ISSsp126 encoded ORF360 resembling transposases of the IS5 family found in *Bacteroides* species, in *Streptococcus thermophilus* and *Pseudomonas putida* [7, 15, 27, 28]. The presence of ISSsp126 at the same position in the wild-type strain

LB126 was confirmed by PCR (data not shown), ruling out the possibility that this element integrated in *fldF* during the mutagenesis and cloning processes. Moreover, *Sphingomonas* sp. LB126 is unable to utilize *p*-hydroxybenzoate as sole carbon source [4], suggesting that a functional *p*-hydroxybenzoate hydroxylase was not present in LB126. Downstream of the putative transposase encoded by *orf360*, the 3′ portion of the *p*-hydroxybenzoate hydroxylase gene was found, confirming the hypothesis that the ISSsp126 insertion split an ancestral gene. Upstream of *fldA* the nucleotide sequence revealed 6 open reading frames, which were called *fldU*, *V*, *W*, *X*, *Y* and *Z* followed by the 5 first codons of a seventh, *fldT*. The genetic organization suggested that *fldYXWVU* formed an operon. The hypothetical proteins encoded by *fldV* and *fldU* were highly similar to LigA and LigB, the α and β components of the PCA 4,5-dioxygenase of *S.* *paucimobilis* SYK-6 [23]. The pu-

orientation relative to the direction of transcription of
the local genes.

3.4. Role of *fldA*, *fldB*, *fldC* and *fldD*
in fluorene degradation

Growth of the *fldA* mutants L-132 and L-613 on
fluorene was about five times slower than the wild-
type strain. This phenotype could be due to the lack
of a functional *fldA* gene or to a polar effect on
the transcription of the downstream genes. A se-
ries of mutations and complementation experiments
were therefore engineered to test for the participa-
tion of the downstream *fldB*, *fldC* and *fldD* genes in
fluorene degradation by *Sphingomonas* sp. LB216.
Three mutator plasmids derived from the suicide vec-
tor pPW78 were constructed by cloning intragenic
portions of *fldB* (pPW121), *fldC* (pPW122) and *fldD*
(pPW123). After conjugation and integrative disrup-
tion, the genotype of each mutant was confirmed by
Southern hybridization (*figure 2B, C*). The *fldB121*
mutation completely suppressed the capacity to grow
on fluorene, whereas the *fldC122* and *fldD123* muta-
tions had no effect. To confirm the specificity of the
*fldA132*, *fldA613* and *fldB121* mutations, the mutants
were complemented with either pPW131 or pPW132,
two pBBR1MCS-2 [2] derived plasmids, which con-
stitutively expressed intact *fldA* and *fldB* genes. As
summarized in *table III*, the ability to assimilate fluo-
rene was restored in the recombinant strain P-12132,
which carried the *fldB121* mutation complemented
with a plasmid-borne *fldB*. The other complemented
strains remained unable to grow with fluorene. These
results demonstrated that *fldA* and *fldB*, but not *fldC*
and *fldD*, were essential for fluorene degradation by
*Sphingomonas* sp. LB126. In addition, since the *fldA*
mutants failed to be complemented with an FldA⁺
plasmid, we concluded that the mutations engineered
in both L-132 and L-613 had a polar effect on the
transcription of the downstream *fldB* gene. This result
also validated the operon organization of *fldA* and *fldB*
suggested by the nucleotide sequence.

3.5. PCA is an essential metabolite formed
in the course of fluorene degradation

To gain a more precise picture of the enzymatic
step blocked in the mutants described above, the
metabolites found in liquid cultures of these mutants
incubated with fluorene were extracted and analyzed
by GC-MSD. The chromatograms of the L-132 and

Table III. Characteristics of the mutant derivatives of Sphingomonas sp. LB 126.

| Strain   | Plasmid    | Genotype                                                                 | Growth on fluorene | Characteristics |
|----------|------------|-------------------------------------------------------------------------|--------------------|------------------|
| LB126    | –          | wild-type                                                              | +                  | –                |
| L-132    | –          | fldA132(fldA::mini-Tn5-luxAB-tet)                                       | –(1)               | FldA⁻FldB⁻       |
| P-13231  | pPW131     | fldA132+plasmid-borne fldA                                              | –(1)               | FldA⁺FldB⁻       |
| P-13232  | pPW132     | fldA132+plasmid-borne fldB                                              | –(1)               | FldA⁻FldB⁺       |
| L-613    | –          | fldA613(fldA::pPW1325)                                                 | –(1)               | FldA⁻FldB⁻       |
| P-61331  | pPW131     | fldA613+plasmid-borne fldA                                              | –(1)               | FldA⁺FldB⁻       |
| P-61332  | pPW132     | fldA613+plasmid-borne fldB                                              | –(1)               | FldA⁻FldB⁺       |
| L-121    | –          | fldB121(fldB::pPW121)                                                  | –                  | FldA⁺FldB⁻       |
| P-12131  | pPW131     | fldB121+plasmid-borne fldA                                              | –                  | FldA⁺FldB⁻       |
| P-12132  | pPW132     | fldB121+plasmid-borne fldB                                              | +                  | FldA⁺FldB⁺       |
| L-122    | –          | fldC122(fldC::pPW122)                                                  | +                  | FldC⁻            |
| L-123    | –          | fldD123(fldD::pPW123)                                                  | +                  | FldD⁻            |
| L-134    | –          | fldT134(fldT::mini-Tn5-luxAB-tet)                                       | –                  | FldT⁻            |

(1) Very slow growth (about five times slower compared to wild type).

Parsons, unpublished data). LB26 might therefore be used as an alternative organism to study the genetic bases associated with this peculiar, poorly characterized pathway.

Sequence analysis of the relevant genes in *Sphingomonas* sp. LB126 revealed that FldW shared limited similarity with LigY, a hydrolase which catalyzes the transformation of the biphenyl degradation product formed during the breakdown of lignin derivatives by *S. paucimobilis* SYK-6 [26]. Surprisingly, FldW was not similar to other hydrolases catalyzing the corresponding reaction in well-characterized biphenyl degrading strains, which is also the case for LigY [26]. The relevance of the homology between FldW and LigY is therefore questionable and might reflect a similarity in the reaction mechanisms rather than in the substrate specificity.

*Sphingomonas* sp. LB126 genes *fldC* and *fldD* found in the DNA segment we cloned would respectively encode enzymes of the alcohol and aldehyde dehydrogenase family. Their disruption did not affect the efficiency of fluorene degradation, leading us to conclude that they were not part of the fluorene degradation pathway. The amino acid sequence of FldD was similar to a *Pseudomonas putida* enzyme, which participates in the degradation of the monoaromatic hydrocarbon cymene [10]. Based on this similarity, FldD and FldC could be part of an unidentified degradation pathway, possibly for monoaromatic compounds, that would supply additional entry routes for the PCA degradation pathway by *Sphingomonas* sp. LB126.

Organization of the *fld* genes suggested that all of them but *fldE* could be organized into three operons, *fldZAB*, *fldCD* and *fldYXWVUT*. The polar effect of a *fldA* mutation on the expression of the downstream *fldB* gene was demonstrated, but the two other operons remain to be validated experimentally. An insertion sequence (ISSsp126) was discovered in the coding sequence of a putative *p*-hydroxybenzoate hydroxylase (*fldF*'). LB126 is unable to grow on *p*-hydroxybenzoate [4] supporting the conclusion that transposition of this newly described IS inactivated the *fldF* gene product. Insertion sequences often border catabolic gene clusters, which led to hypothesize that they contribute to the dissemination of catabolic pathways among microorganisms by horizontal gene transfer [11, 35]. Whether another copy of ISSsp126 lies at the other side of the PCA pathway in the LB126 chromosome remains to be established.

From the evolutionary point of view, it is interesting that numerous similarities reported in this study occur between *S. paucimobilis* SYK-6 PCA genes involved in lignin degradation and *Sphingomonas* sp. LB126 genes, which participate in the degradation of fluorene. These similarities suggest that the ability to degrade polyaromatic hydrocarbons derived from fossil fuel or lignin components relies on the evolution of a common ancestral pathway. Since bacteria of the *Sphingomonas* genus are very often encountered in PAH-contaminated biotopes, the special metabolic capabilities displayed by members of this genus might reflect an innate disposition for the removal of PAHs and justify their interesting potential as bioremediating microorganisms.

### Acknowledgements

The authors wish to thank C. Locht for the gift of plasmid pBBR1-MCS2, W. De Jonghe for assistance in its use, and S. Wuertz for a critical reading of the manuscript. This work was supported by a grant from the European Community BIOTECH program BIO4-CT97-2015.

### References

[1] Altschul S., Madden T., Schaffer A., Zhang J.H., Zhang Z., Miller W., Lipman D., Gapped BLAST and PSI-BLAST: A new generation of protein database search programs, Faseb J. 12 (1998) 102.

[2] Antoine R., Locht C., Isolation and molecular characterization of a novel broad-host-range plasmid from *Bordetella bronchiseptica* with sequence similarities to plasmids from gram-positive organisms, Mol. Microbiol. 6 (1992) 1785–1799.

[3] Bastiaens L., Springael D., Wattiau P., Harms H., de Wachter R., Verachtert H., Diels L., Isolation of adherent polycyclic aromatic hydrocarbon (PAH)-degrading bacteria using PAH-sorbing carriers, Appl. Environ. Microbiol. 66 (2000) 1834–1843.

[4] Bastiaens L., Springael D., Dejonghe W., Wattiau P., Verachtert H., Diels L., Construction of a transcriptional *luxAB* reporter fusion responding to fluorene in *Sphingomonas* sp. LB126 and its initial characterisation for whole-cell bioreporter purposes, Accompanying paper (2001).

[5] Blattner F.R., Plunkett G., 3rd, Bloch C.A., Perna N.T., Burland V., Riley M., Collado-Vides J., Glasner J.D., Rode C.K., Mayhew G.F., Gregor J., Davis N.W., Kirkpatrick H.A., Goeden M.A., Rose D.J., Mau B., Shao Y., The complete genome sequence of *Escherichia coli* K-12, Science 277 (1997) 1453–1474.

[6] Boldrin B., Tiehm A., Fritzshe C., Degradation of phenanthrene, fluorene, fluoranthene, and pyrene by a *Mycobacterium* sp, Appl. Environ. Microbiol. 59 (1993) 1927–1930.

[7] Bourgin F., Guedon G., Gintz B., Decaris B., Characterization of a novel insertion sequence, IS1194, in *Streptococcus thermophilus*, Plasmid 40 (1998) 44–49.

[8] Casellas M., Grifoll M., Bayona J.M., Solanas A.M., New metabolites in the degradation of fluorene by *Arthrobacter* sp. strain F101, Appl. Environ. Microbiol. 63 (1997) 819–826.

[9] de Lorenzo V., Herrero M., Jakubzik U., Timmis K.N., Mini-Tn5 transposon derivatives for insertion mutagenesis, promoter probing, and chromosomal insertion of cloned DNA in gram-negative eubacteria, J. Bacteriol. 172 (1990) 6568–6572.

[10] Eaton R.W., p-cumate catabolic pathway in *Pseudomonas putida* Fl: cloning and characterization of DNA carrying the *cmt* operon, J. Bacteriol. 178 (1996) 1351–1362.

[11] Fulthorpe R.R., Wyndham R.C., Involvement of a chlorobenzoate-catabolic transposon, Tn5271, in community adaptation to chlorobiphenyl, chloroaniline, and 2,4-dichlorophenoxyacetic acid in a freshwater ecosystem, Appl. Environ. Microbiol. 58 (1992) 314–325.

[12] Grifoll M., Casellas M., Bayona J.M., Solanas A.M., Isolation and characterization of a fluorene-degrading bacterium – identification of ring oxidation and ring fission-products, Appl. Environ. Microbiol. 58 (1992) 2910–2917.

[13] Grifoll M., Selifonov S.A., Chapman P.J., Evidence for a novel pathway in the degradation of fluorene by *Pseudomonas* sp. strain F274, Appl. Environ. Microbiol. 60 (1994) 2438–2449.

[14] Grifoll M., Selifonov S.A., Gatlin C.V., Chapman P.J., Actions of a versatile fluorene-degrading bacterial isolate on polycyclic aromatic compounds, Appl. Environ. Microbiol. 61 (1995) 3711–3723.

[15] Haggoud A., Reysset G., Azeddoug H., Sebald M., Nucleotide sequence analysis of two 5-nitroimidazole resistance determinants from *Bacteroides* strains and of a new insertion sequence upstream of the two genes, Antimicrob. Agents Chemother. 38 (1994) 1047–1051.

[16] Keith L.H., Telliard W.A., Priority pollutants. I. A perspective view, Environ. Sci. Technol. 13 (1979) 416–423.

[17] Marmur J., A procedure for the isolation of deoxyribonucleic acid from microorganisms, J. Mol. Biol. 3 (1961) 208–218.

[18] Masai E., Shinohara S., Hara H., Nishikawa S., Katayama Y., Fukuda M., Genetic and biochemical characterization of a 2-pyrone-4,6-dicarboxylic acid hydrolase involved in the protocatechuate 4,5-cleavage pathway of *Sphingomonas paucimobilis* SYK-6, J. Bacteriol. 181 (1999) 55–62.

[19] Matsuda A., Toma K., Komatsu K., Nucleotide sequences of the genes for two distinct cephalosporin acylases from a *Pseudomonas* strain, J. Bacteriol. 169 (1987) 5821–5826.

[20] Mergeay M., Nies D., Schlegel H.G., Gerits J., Charles P., Van Gijsegem F., *Alcaligenes eutrophus* CH34 is a facultative chemolithotroph with plasmid-bound resistance to heavy metals, J. Bacteriol. 162 (1985) 328–334.

[21] Miller V.L., Mekalanos J.J., A novel suicide vector and its use in construction of insertion mutations: osmoregulation of outer membrane proteins and virulence determinants in

Vibrio cholerae requires toxR, J. Bacteriol. 170 (1988) 2575–2583.

[22] Monna L., Omori T., Kodama T., Microbial degradation of dibenzofuran, fluorene, and dibenzo-p-dioxin by Staphylococcus auriculans Dbf63, Appl. Environ. Microbiol. 59 (1993) 285–289.

[23] Noda Y., Nishikawa S., Shiozuka K., Kadokura H., Nakajima H., Yoda K., Katayama Y., Morohoshi N., Haraguchi T., Yamasaki M., Molecular cloning of the protocatechuate 4,5-dioxygenase genes of Pseudomonas paucimobilis, J. Bacteriol. 172 (1990) 2704–2709.

[24] Overhage J., Kresse A.U., Priefert H., Sommer H., Krammer G., Rabenhorst J., Steinbuchel A., Molecular characterization of the genes pcaG and pcaH, encoding protocatechuate 3,4-dioxygenase, which are essential for vanillin catabolism in *Pseudomonas* sp. strain HR199, Appl. Environ. Microbiol. 65 (1999) 951–960.

[25] Parke D., Characterization of PcaQ, a LysR-type transcriptional activator required for catabolism of phenolic compounds, from Agrobacterium tumefaciens, J. Bacteriol. 178 (1996) 266–272.

[26] Peng X., Masai E., Katayama Y., Fukuda M., Characterization of the meta-cleavage compound hydrolase gene involved in degradation of the lignin-related biphenyl structure by Sphingomonas paucimobilis SYK-6, Appl. Environ. Microbiol. 65 (1999) 2789–2793.

[27] Podglajen I., Breuil J., Collatz E., Insertion of a novel DNA sequence, IS1186, upstream of the silent carbapenemase gene *cfiA*, promotes expression of carbapenem resistance in clinical isolates of *Bacteroides fragilis*, Mol. Microbiol. 12 (1994) 105–114.

[28] Reddy B.R., Shaw L.E., Sayers J.R., Williams P.A., Two identical copies of IS1246, a 1275 base pair sequence related to other bacterial insertion sequences, enclose the *xyl* genes on TOL plasmid pWW0, Microbiology (UK) 140 (1994) 2305–2307.

[29] Romine M.F., Stillwell L.C., Wong K.K., Thurston S.J., Sisk E.C., Sensen C., Gaasterland T., Fredrickson J.K., Saffer J.D., Complete sequence of a 184-kilobase catabolic plasmid from Sphingomonas aromaticivorans F199, J. Bacteriol. 181 (1999) 1585–1602.

[30] Saito Y., Ishii Y., Hayashi H., Imao Y., Akashi T., Yoshikawa K., Noguchi Y., Soeda S., Yoshida M., Niwa M., Hosoda J., Shimomura K., Cloning of genes coding for L-sorbose and

L-sorbosone dehydrogenases from Gluconobacter oxydans and microbial production of 2-Keto-L-Gulonate, a precursor of L-ascorbic acid, in a recombinant G. oxydans strain, Appl. Environ. Microbiol. 63 (1997) 454–460.

[31] Sambrook J., Russel D. (Eds.), Molecular cloning, a laboratory manual, Third Edition, Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 2001.

[32] Schmidhauser T.J., Helinski D.R., Regions of broad-host-range plasmid RK2 involved in replication and stable maintenance in nine species of gram-negative bacteria, J. Bacteriol. 164 (1985) 446–455.

[33] Simon R., Priefer U., Pühler A., A broad host range mobilization system for in vivo genetic engineering: transposon mutagenesis in Gram negative bacteria, Biotechnology 1 (1983) 784–791.

[34] Smith D.R., DoucetteStamm L.A., Deloughery C., Lee H.M., Dubois J., Aldredge T., Bashirzadeh R., Blakely D., Cook R., Gilbert K., Harrison D., Hoang L., Keagle P., Lumm W., Pothier B., Qiu D.Y., Spadafora R., Vicaire R., Wang Y., Wierzbowski J., Gibson R., Jiwani N., Caruso A., Bush D., Safer H., Patwell D., Prabhakar S., McDougall S., Shimer G., Goyal A., Pietrokovski S., Church G.M., Daniels C.J., Mao J.I., Rice P., Nolling J., Reeve J.N., Complete genome sequence of Methanobacterium thermoautotrophicum Delta H: Functional analysis and comparative genomics, J. Bacteriol. 179 (1997) 7135–7155.

[35] Springael D., vanThor J., Goorissen H., Ryngaert A., DeBaere R., VanHauwe P., Commandeur L.C.M., Parsons J.R., DeWachter R., Mergeay M., RP4::Mu3A-mediated in vivo cloning and transfer of a chlorobiphenyl catabolic pathway, Microbiology (UK) 142 (1996) 3283–3293.

[36] Trenz S.P., Engesser K.H., Fischer P., Knackmuss H.J., Degradation of fluorene by Brevibacterium sp. strain DPO 1361: a novel C–C bond cleavage mechanism via 1,10-dihydro-1,10-dihydroxyfluoren-9-one, J. Bacteriol. 176 (1994) 789–795.

[37] von Heijne G., Membrane protein structure prediction. Hydrophobicity analysis and the positive-inside rule, J. Mol. Biol. 225 (1992) 487–494.

[38] Wong C.M., Dilworth M.J., Glenn A.R., Cloning and sequencing show that 4-hydroxybenzoate hydroxylase (PobA) is required for uptake of 4-hydroxybenzoate in Rhizobium leguminosarum, Microbiology (UK) 140 (1994) 2775–2786.
