
Eur. J. Biochem. 269, 519–526 (2002) © FEBS 2002

# Biosynthesis of riboflavin

## 6,7-Dimethyl-8-ribityllumazine synthase of *Schizosaccharomyces pombe*

Markus Fischer${}^{1}$, Ilka Haase${}^{1}$, Richard Feicht${}^{1}$, Gerald Richter${}^{1}$, Stefan Gerhardt${}^{2}$, Jean-Pierre Changeux${}^{3}$, Robert Huber${}^{2}$ and Adelbert Bacher${}^{1}$

${}^{1}$Institut für Organische Chemie und Biochemie, Technische Universität München, Germany; ${}^{2}$Department of Protein Crystallography, Max-Planck-Institute of Biochemistry, Martinsried, Germany; ${}^{3}$Department of Molecular Neurobiology, Institut Pasteur, Paris, France

A cDNA sequence from *Schizosaccharomyces pombe* with similarity to 6,7-dimethyl-8-ribityllumazine synthase was expressed in a recombinant *Escherichia coli* strain. The recombinant protein is a homopentamer of 17-kDa subunits with an apparent molecular mass of 87 kDa as determined by sedimentation equilibrium centrifugation (it sediments at an apparent velocity of 5.0 S at 20 °C). The protein has been crystallized in space group C222${}_{1}$. The crystals diffract to a resolution of 2.4 Å. The enzyme catalyses the formation of 6,7-dimethyl-8-ribityllumazine from 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione and 3,4-dihydroxy-2-butanone 4-phosphate. Steady-state kinetic analysis afforded a $v_{\text{max}}$ value of 13 000 nmol·mg${}^{-1}$·h${}^{-1}$ and $K_{\text{m}}$ values of 5 and 67 μM for 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione and 3,4-dihydroxy-2-butanone 4-phosphate, respectively. The enzyme binds riboflavin with a $K_{\text{d}}$ of 1.2 μM. The fluorescence quantum yield of enzyme-bound riboflavin is < 2% as compared with that of free riboflavin. The protein/riboflavin complex displays an optical transition centered around 530 nm as shown by absorbance and CD spectrometry which may indicate a charge transfer complex. Replacement of tryptophan 27 by tyrosine or phenylalanine had only minor effects on the kinetic properties, but complexes of the mutant proteins did not show the anomalous long wavelength absorbance of the wild-type protein. The replacement of tryptophan 27 by aliphatic amino acids substantially reduced the affinity of the enzyme for riboflavin and for the substrate, 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione.

**Keywords:** biosynthesis of riboflavin; crystallization; 6,7-dimethyl-8-ribityllumazine synthase; mutagenesis; riboflavin binding.

The biosynthetic precursor of riboflavin (4), where numbers refer to those in Fig. 1, 6,7-dimethyl-8-ribityllumazine (3), is biosynthesized by condensation of 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione (1) with 3,4-dihydroxy-2-butanone 4-phosphate (2) [1–4]. The reaction is catalysed by the enzyme 6,7-dimethyl-8-ribityllumazine synthase (Fig. 1A).

The structures of lumazine synthases from several species have been studied by X-ray diffraction analysis. The enzymes from *Bacillus subtilis*, *Escherichia coli* and *Spinacia oleracea* (spinach) were shown to form capsids of 60 identical subunits with icosahedral 532 symmetry which are best described as dodecamers of pentamers [5–11]. The icosahedral lumazine synthases from Bacillaceae form a complex with riboflavin synthase which is enclosed in the central core of the icosahedral capsid [12–14].

The lumazine synthases of *Saccharomyces cerevisiae*, *Magnaporthe grisea* and *Brucella abortus* are homopentamers of ≈ 85 kDa [10,15,16]. Their subunit folds are closely similar to those of the icosahedral enzymes.

The five and, respectively, the 60 equivalent active sites of the pentameric and icosahedral lumazine synthases are all located at interfaces between adjacent subunits in the pentameric motifs [7,8,11].

The riboflavin pathway is a potential target for anti-infective chemotherapy as Gram-negative bacteria and possibly pathogenic yeasts are unable to absorb riboflavin or flavoenzymes from the environment and are thus absolutely dependent on the endogenous synthesis of the vitamin. This paper reports the heterologous expression of lumazine synthase from the yeast, *Schizosaccharomyces pombe*, which was found to bind riboflavin with relatively high affinity.

---

## EXPERIMENTAL PROCEDURES

### Materials

5-Amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione (1) and 6,7-dimethyl-8-ribityllumazine (3) were synthesized according to published procedures [5,17]. Recombinant 3,4-dihydroxy-2-butanone 4-phosphate synthase of *E. coli* [18] was used for preparation of 3,4-dihydroxy-2-butanone 4-phosphate (2) [4]. Riboflavin and FMN were from Sigma. Restriction enzymes were from Pharmacia Biotech. T4 DNA ligase and reverse transcriptase (SuperScript™II) were from Gibco BRL. Oligonucleotides were synthesized

Correspondence to M. Fischer, Institut für Organische Chemie und Biochemie, Technische Universität München, Lichtenbergstr. 4, D-85747 Garching, Germany. Fax: + 49 89 28 91 33 63, Tel.: + 49 89 28 91 33 36, E-mail: markus.fischer@ch.tum.de (Received 28 June 2001, revised 19 October 2001, accepted 15 November 2001)

520 M. Fischer *et al.* (Eur. J. Biochem. 269)

of the cell mash was added. The mixture was centrifuged (Beckman SW40 rotor, 31 000 r.p.m., 18 h, 20 °C). The pellet was dissolved in 200 μL sterile water. RNA was precipitated by the addition of 10 μL 3 M sodium acetate pH 5.0, and 250 μL ethanol. The mixture was centrifuged (Jouan AB 2.14 rotor, 17 000 r.p.m., 30 min, 4 °C). The pellet was washed twice with 200 μL ice-cold 70% ethanol and dried. It was then dissolved in 200 μL sterile water. RNA concentration was determined photometrically (260 nm).

Preparation of cDNA

A reaction mixture (20 μL) containing 50 mM Tris/HCl pH 8.3, 75 mM potassium chloride, 3 mM MgCl₂, 10 mM dithiothreitol, 0.5 mM dNTPs, 0.5 μg Oligo-(dT)-15, 2 μg *S. pombe* total RNA, and 200 U reverse transcriptase was incubated at 37 °C for 15 min and subsequently at 48 °C for 30 min. The mixture was heated at 95 °C for 5 min.

Construction of a hyperexpression plasmid

*S. pombe* cDNA was used as template for PCR amplification and the oligonucleotides A-1 and A-2 as primers (Table 2). The amplificate (525 bp) was purified with the Purification Kit from Qiagen and was digested with the restriction endonucleases EcoRI and BamHI and was ligated into the expression vector pNCO113 [20] which had been digested with the same enzymes yielding the plasmid designated pNCO-SSP-RIB4-WT.

Site-directed mutagenesis

PCR-amplification using the plasmid pNCO-SSP-RIB4-WT as template and the oligonucleotides shown in Table 2 as primers (primer combinations: W27G/A-3, W27I/A-3, W27S/A-3, W27H/A-3, W27F/A-3, W27Y/A-3) afforded DNA fragments that served as templates for a second round of PCR amplification using the oligonucleotides A-3 and A-4 as primers. For the verification of mutations, primers were designed to introduce recognition sites for specific restriction endonucleases (Table 2). Restriction and ligation of the vector pNCO113 and the purified PCR product were performed as described above.

Strains and plasmids

Bacterial strains and plasmids used in this study are summarized in Table 1.

Isolation of RNA

*Schizosaccharomyces pombe var. pombe* Lindner (ATCC 16491) was cultured in medium containing 0.3 g yeast extract, 0.3 g malt extract, 0.5 g peptone and 1 g glucose per litre. Cultures were incubated for 72 h at 24 °C with shaking. The cells were harvested by centrifugation (5000 r.p.m., 15 min, 4 °C, Sorvall GSA rotor). The isolation of total RNA was carried out using a method modified after Chirgwin *et al.* [19]. The cell mass (1 g) was frozen in liquid nitrogen. A solution (10 mL) containing 4.23 M guanidinium thiocyanate, 25 mM sodium citrate, 100 mM mercaptoethanol, 0.5% lauryl sarcosine and 10 μL Antifoam A was added. The mixture was crushed, the resulting powder was thawed, and the suspension was passed through a hypodermic needle (internal diameter, 1 mm). A solution (3 mL) containing 5.7 M CsCl and 0.1 M EDTA pH 7.0, was placed into a centrifuge tube, and 7 mL

Table 1. Bacterial strains and plasmids.

| Strain or plasmid | Relevant characteristics | Source |
|-------------------|--------------------------|--------|
| *E. coli* strain XL-1-Blue | recA1, endA1, gyrA96, thi-1, hsdR17, supE44, relA1, lac[F′], proAB, lacIqZΔM15, Tn10(tet')] | [21] |
| Plasmids for the RIB4 gene of *S. pombe* |  |  |
| pNCO113 | Expression vector | [20] |
| pNCO-SSP-RIB4-WT | RIB4 gene wild-type | This study |
| pNCO-SSP-RIB4-W27G | RIB4 gene W27G mutant | This study |
| pNCO-SSP-RIB4-W27I | RIB4 gene W27I mutant | This study |
| pNCO-SSP-RIB4-W27S | RIB4 gene W27S mutant | This study |
| pNCO-SSP-RIB4-W27H | RIB4 gene W27H mutant | This study |
| pNCO-SSP-RIB4-W27F | RIB4 gene W27F mutant | This study |
| pNCO-SSP-RIB4-W27Y | RIB4 gene W27Y mutant | This study |

Fig. 1. Terminal reactions in the pathway of riboflavin biosynthesis. (A) Lumazine synthase; (B) riboflavin synthase.

© FEBS 2002                                                                 Biosynthesis of riboflavin (Eur. J. Biochem. 269) 521

Table 2. Oligonucleotides used for construction of expression plasmids. Mutated bases are shown in bold type and recognition sites for detection of the mutations are underlined.

| Designation | Endonuclease | Sequence |
|-------------|--------------|----------|
| A-1         |              | 5'ataatagaattcattaagaggagaaattaactatgttcagttggtattaaaggccctaac3' |
| A-2         |              | 5'tattatggatccttaatacaaaagcttcaatcccattc3' |
| W27G        | SacII        | 5'aaaggccctaacccttcagacttaagggaaccagaattgcgcattcttattgtccatgcccggttaatcttcaag3' |
| W27I        | Asel         | 5'aaaggccctaacccttcagacttaagggaaccagaattgcgcattcttattgtccatgcccgcttaatcttcaag3' |
| W27S        | AsuII        | 5'aaaggccctaacccttcagacttaagggaaccagaattgcgcattcttattgtccatgcccgttcaatcttcaag3' |
| W27H        | SacI         | 5'aaaggccctaacccttcagacttaagggaaccagacttcgcattcttattgtccatgcccgccataatcttcaag3' |
| W27F        | SphI         | 5'aaaggccctaacccttcagacttaagggaaccagaattgcgcattcttattgtcatgcccgctttaatcttcaag3' |
| W27Y        | ApaI         | 5'aaaggccctaacccttcagacttaagggccccgaattgcgcattcttattgtccatgcccgctacaatcttcaag3' |
| A-3         |              | 5'ctccatttttagcttccttagctcctg3' |
| A-4         |              | 5'ataatagaattcattaagaggagaaattaactatgttcagttggtattaaaggccctaacccttcagacttaag3' |

Transformation of *E. coli* XL1-Blue cells

*E. coli* XL-1 Blue cells were transformed according to Bullock *et al.* 1987 [21]. Transformants were selected on Luria-Bertani (LB) agar plates supplemented with ampicillin (150 mg·L⁻¹). The constructs were monitored by restriction analysis and by DNA sequencing. In the expression plasmids, the lumazine synthase gene is under control of the T5 promoter and the *lac* operator. Protein expression was induced by the addition of 2 mM isopropyl thio-β-D-galactoside.

Estimation of protein concentration

Protein concentration was estimated by the modified Bradford procedure reported by Read and Northcote [23].

SDS/PAGE

SDS/PAGE using 16% polyacrylamide gels was performed as described by Laemmli [24]. Molecular mass standards were supplied by Sigma.

DNA sequencing

Sequencing was performed by the dideoxy chain termination method [22] using a model 377A DNA sequencer (Applied Biosystems). Plasmid DNA was isolated from cultures (5 mL) of XL-1 Blue strains grown overnight in LB medium containing ampicillin (150 mg·L⁻¹) using Nucleo-bond AX20 columns (Macherey und Nagel, Düren, Germany).

Protein sequencing

Sequence determination was performed according to the automated Edman method using a 471A Protein Sequencer (PerkinElmer).

HPLC

Protein was denatured with 15% (w/v) trichloroacetic acid. The mixture was centrifuged, and the supernatant was analysed by HPLC. RP-HPLC was performed with a column of Hypersil ODS 5 μ. The eluent contained 100 mM ammonium formate and 40% (v/v) methanol. The effluent was monitored fluorometrically (6,7-dimethyl-8-ribityllumazine: excitation, 408 nm; emission, 487 nm; flavins: excitation, 445 nm; emission, 520 nm).

Preparation of ligand-free 6,7-dimethyl-8-ribityllumazine synthase

Urea was added to a final concentration of 5 M to the yellow coloured protein solution. The solution was dialysed against 50 mM potassium phosphate pH 7.0 containing 0.02% sodium azide and 5 M urea and subsequently against 50 mM potassium phosphate pH 7.0.

Fluorescence titration

Experiments were performed with a F-2000 spectrofluorimeter from Hitachi at room temperature in a 10-mm quartz cell. Concentrated stock solutions of riboflavin, FMN, 6,7-dimethyl-8-ribityllumazine, 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione and 5-nitro-6-ribitylamino-2,4(1H,3H)-pyrimidinedione were prepared freshly before

Protein purification

Recombinant *E. coli* strains were grown in LB medium containing ampicillin (150 mg·L⁻¹) at 37°C with shaking. At an optical density of 0.6 (600 nm), isopropyl thio-β-D-thiogalactoside was added to a final concentration of 2 mM, and incubation was continued for 6 h. The cells were harvested by centrifugation, washed with 0.9% NaCl and stored at –20°C. The cell mass was thawed in lysis buffer (50 mM potassium phosphate pH 6.9, 0.5 mM EDTA, 0.5 mM sodium sulfite, 0.02% sodium azide). The suspension was cooled on ice and was subjected to ultrasonic treatment. The supernatant was placed on top of a Q-Sepharose column (92 mL) which had been equilibrated with 20 mM potassium phosphate pH 6.9. The column was developed with a linear gradient of 0–1.0 M potassium chloride in 20 mM potassium phosphate pH 7.0. Fractions were combined, and ammonium sulfate was added to a final concentration of 2.46 M. The precipitate was harvested and redissolved in 20 mM potassium phosphate pH 7.0. The solution was placed on top of a Superdex-200 column which was developed with 20 mM potassium phosphate pH 7.0 containing 100 mM potassium chloride. Fractions were combined and concentrated by ultrafiltration.

each experiment and were calibrated photometrically [riboflavin resp. FMN, ε₄₄₅ = 12 500 M⁻¹·cm⁻¹ (pH 7.0); 6,7-dimethyl-8-ribityllumazine, ε₄₀₈ = 12 100 M⁻¹·cm⁻¹ (pH 7.0), 5-amino-6-ribitylamino-4(1H,3H)-pyrimidinedione, ε₂₆₈ = 24 500 M⁻¹·cm⁻¹ (pH 1.0), 5-nitro-6-ribityl-amino-2,4(1H,3H)-pyrimidinedione, ε₃₂₃ = 14 200 M⁻¹·cm⁻¹ (pH 1.0)]. Titrations were performed by adding 50 μL ligand solution in 5 μL steps to 1 mL of protein solution. Control experiments were performed with 1 mL 50 mM potassium phosphate pH 7.0.

### Equilibrium dialysis

Equilibrium dialysis experiments were performed with a DIANORM microcell system (Bachofer, Reutlingen, Germany). Enzyme solution (150 μM) was dialysed against flavin solution for 2 h at 4 °C. Protein was precipitated by the addition of 15% (w/v) trichloroacetic acid (1 : 1). The flavin concentration of each cell was determined by HPLC.

### Steady-state kinetics

Assay mixtures contained 100 mM phosphate pH 7.0, 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione, 3,4-dihydroxy-2-butanone 4-phosphate and protein, as indicated. The reaction was monitored photometrically at 410 nm and 37 °C [2,3].

### CD

Measurements were performed with a spectropolarimeter JASCO J-715 using 5- or 10-mm quartz cells. Protein solutions (145 μM) and riboflavin solutions (145 μM) were measured against 50 mM potassium phosphate pH 7.0, at 20 °C.

### Electrospray MS

Experiments were performed as described by Mann & Wilm [25] using a triple quadrupol ion spray mass spectrometer API365 (SciEx, Thornhill, Ontario, Canada).

### Analytical ultracentrifugation

Experiments were performed with an analytical ultracentrifuge Optima XL-A from Beckman Instruments equipped with absorbance optics. Aluminum double sector cells equipped with quartz windows were used throughout. Protein solutions were dialysed against 50 mM potassium phosphate pH 7.0. The partial specific volume was estimated from the amino acid composition yielding a value of 0.741 mL·g⁻¹ [26].

For boundary sedimentation experiments 50 mM potassium phosphate pH 7.0 containing 1.1 mg protein·mL⁻¹ was centrifuged at 59 000 r.p.m. and 20 °C.

Sedimentation equilibrium experiments were performed with 50 mM potassium phosphate pH 7.0 containing 0.44 mg protein·mL⁻¹ and centrifuged at 10 000 r.p.m. and 4 °C for 72 h.

Protein concentrations were monitored photometrically at 280 nm in both cases.

### Crystallization

Crystallization was performed by the sitting-drop vapour diffusion method. A solution containing 20 mM potassium phosphate pH 7.0, 50 mM KCl, and 10 mg protein·mL⁻¹ was mixed with an equal amount of a solution containing 0.1 M citrate pH 4.9–5.2 and 1.5 M sodium formate. The reservoir buffer contained 0.1 M citrate pH 4.9–5.2 and 1.5 M sodium formate.

## RESULTS

A hypothetical gene of *S. pombe* assumed to specify 6,7-dimethyl-8-ribityllumazine synthase (accession number, CAB52615) had been proposed to contain one putative intron of 288 bp. The putative reading frame was amplified from *S. pombe* cDNA, and the amplificate was cloned into the expression vector pNCO 113. Sequencing confirmed the open reading frame which had been predicted earlier on basis of the genomic data (Fig. 2).

A recombinant *E. coli* strain carrying the *S. pombe* gene under the control of a T5 promoter and a *lac* operator expressed a recombinant 17 kDa protein (≈ 10% of the total cell protein), which was isolated in pure form by two chromatographic steps as described in Materials and methods. The pure protein solution showed intense yellow colour but appeared nonfluorescent under ultraviolet light. Electrospray MS afforded a molecular mass of 17 189 Da in close agreement with the predicted mass of 17 188 Da. Edman degradation of the N-terminus afforded the sequence MFSGIKGPNPSDLKG in agreement with the translated open reading frame.

The enzyme sedimented in the analytical ultracentrifuge as a single, symmetrical boundary. The apparent sedimentation velocity at 20 °C in 50 mM potassium phosphate pH 7.0 was 5.0 S. For comparison, it should be noted that the lumazine synthase of *S. cerevisiae* has an apparent sedimentation coefficient of *s*₂₀ = 5.5 S [9]. Sedimentation equilibrium experiments indicated a molecular mass of 87 kDa using an ideal mono-disperse model for calculation. The residuals show close agreement between the model and the experimental data. The subunit molecular mass of 17 188 Da implicates a pentamer mass of 85.9 kDa in excellent agreement with the experimental data.

Crystallization experiments were performed as described in Methods. Crystals of 0.4 × 0.2 × 0.2 mm³ appeared within few days. They diffract X-rays to a resolution of 2.4 Å and belong to the space group C222₁ with cell constants *a* = 111.50 Å, *b* = 145.52 Å, *c* = 128.70 Å, α = β = γ = 90°. The asymmetric unit contains one pentamer resulting in a Matthews coefficient of 3.04 Å³ [27].

Enzyme assays confirmed that the protein catalyses the formation of 6,7-dimethyl-8-ribityllumazine from 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione and 3,4-dihydroxy-2-butanone 4-phosphate. Steady-state kinetic analysis afforded a *v*ₘₐₓ value of 13 000 nmol·mg⁻¹·h⁻¹ and and *K*ₘ values of 5 and 67 μM for 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione and 3,4-dihydroxy-2-butanone 4-phosphate, respectively (Table 3). Riboflavin acted as a competitive inhibitor of the enzyme with a *K*ᵢ of 17 μM.

In order to identify the yellow chromophore present in the purified protein solution, aliquots of various batches were treated with trichloroacetic acid, and the supernatant

© FEBS 2002                                                                 Biosynthesis of riboflavin (Eur. J. Biochem. 269) 523

| S. cerevisiae | M A V K G L G K P D Q V Y D G S K I R V G I I H A R W N R V I D |
| --- | --- |
| S. pombe | M F S G I K G P N P S D . . L K G P E L R I L V H A R W N L Q A I E |
| B. subtilis | . . . . . M N I I Q G N . . . L V G T G L K I G I V V G R F N D F I T S |

$\beta 2$

| S. cerevisiae | A L V K G A I E R M A S . L G V E E N N I I E T V P G S Y E L P W G |
| --- | --- |
| S. pombe | P L V K G A V E T M I E K H D V K L E N I D I E S V P G S W E L P Q G |
| B. subtilis | K L L S G A E D A L L R H G . V D T N D I D V A W V P G A F E I P F A |

$\alpha 1$ $\beta 3$

| S. cerevisiae | T K R F V D R Q A K L G K P L D V V I P I G V L I K G S T M H F E Y I |
| --- | --- |
| S. pombe | I R A S I A R N . T . . . Y D A V I G I G V L I K G S T M H F E Y I |
| B. subtilis | A K K M A E T K . . . . K Y D A I I T L G T V I R G A T T H Y D Y V |

$\alpha 2$ $\beta 4$

| S. cerevisiae | S D S T T H A L M N L Q E K V D M P V I F G L L T C M T E E Q A L A R |
| --- | --- |
| S. pombe | S E A V V H G L M R V G L D S G V P V I L G L L T V L N E E E Q A L Y R |
| B. subtilis | C N E A A K G I A Q A A N T T G V P V I F G I V T T E N I E Q A I E R |

$\alpha 3$ $\beta 5$ $\alpha 4$

| S. cerevisiae | A G I D E A H S M H N H G E D W G A A A V E M A V K F G K N A F |
| --- | --- |
| S. pombe | A G . . . L N G G H N H G N D W G S A A V E M G L K A L Y . . . |
| B. subtilis | A G . . . T K A G N K G V D C A V S A I E M A N L N R S . F E |

$\alpha 5$

Fig. 2. Sequence alignment of lumazine synthases. Elements of secondary structure found in *S. cerevisiae* [15] are indicated below the sequences. Numbers refer to lumazine synthase from *S. pombe*. Highly conserved residues, light grey; amino acids with similar polarity, dark grey; amino acid residues which are involved in the active site are indicated by ▲; insertions between the helices $\alpha_4$ and $\alpha_5$ are indicated in bold type.

was analysed by HPLC. Riboflavin was found in concentrations ranging from 0.17 to 0.21 μmol·μmol⁻¹ protein subunit. Moreover, 6,7-dimethyl-8-ribityllumazine was detected in the range of 0.028–0.032 μmol·μmol⁻¹ protein subunits.

To study the optical properties of the riboflavin/enzyme complex, the protein solution was treated with a large excess of riboflavin and was subsequently dialysed extensively against 50 mM potassium phosphate. The absorption spectrum of the complex differed substantially from that of free riboflavin in several respects. The absorption band of riboflavin at 370 nm showed a bathochromatic shift of about 20 nm (Fig. 3). The maximum of the long wavelength band at 445 nm was not shifted significantly, but the relative intensities of the two bands had changed substantially in comparison with the spectrum of free riboflavin. Most notably, however, the long wavelength band of the complex showed trailing on the long wavelength side which extends at least to 600 nm.

In order to analyse the optical transitions involved in more detail, CD spectra were recorded in the long wavelength range for the purified protein with ≈20% riboflavin (data not shown) as well as for the protein solution treated with a large excess of riboflavin and subsequently dialysed extensively against 50 mM potassium phosphate (Fig. 4A). In both cases the CD spectra of the enzyme/riboflavin complex showed positive Cotton effects centred at 530 nm and 405 nm and negative Cotton effects of lower intensity at 460 nm and 360 nm. Riboflavin was analysed for comparison and showed a negative Cotton effect at 450 nm and a positive Cotton effect at 340 nm in agreement with earlier measurements [28]. In conjunction with the absorption spectra described above, the data suggested the involvement of a charge transfer complex.

Table 3. Steady-state kinetic analysis of wild-type and mutant lumazine synthases.

| Enzyme    | $v_{\text{max}}$ (nmol mg⁻¹·h⁻¹) | $K_{\text{m}}^{\text{a}}$ (μM) | $K_{\text{m}}^{\text{b}}$ (μM) | $K_{\text{d}}^{\text{c}}$ (μM) | $K_{\text{i}}^{\text{d}}$ (μM) |
|-----------|----------------------------------|-------------------------------|-------------------------------|-------------------------------|-------------------------------|
| Wild-type | 13 000                           | 5                             | 67                            | 1.2                          | 17                            |
| W27Y      | 14 000                           | 3                             | 86                            | 12.0                         |                               |
| W27F      | 10 000                           | 3                             | 65                            |                              |                               |
| W27H      | 4000                             | 400                           | 145                           |                              |                               |
| W27S      | 4300                             | 460                           | 187                           |                              |                               |
| W27I      | 5500                             | 230                           | 137                           |                              |                               |
| W27G      | 5400                             | 430                           | 168                           |                              |                               |

${}^{\text{a}} K_{\text{m}}$ for 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione. ${}^{\text{b}} K_{\text{m}}$ for 3,4-dihydroxy-2-butanone 4-phosphate. ${}^{\text{c}} K_{\text{d}}$ for riboflavin. ${}^{\text{d}} K_{\text{i}}$ for riboflavin.

524 M. Fischer et al. (Eur. J. Biochem. 269)

Fig. 3. Absorption spectra obtained in 50 mM potassium phosphate pH 7.0. Solid line, wild-type enzyme; dotted line, W27Y mutant; dashed line, riboflavin.

Fig. 4. CD. Measurements were performed in 50 mM potassium phosphate pH 7.0. (A) Wild-type enzyme; (B) W27Y mutant; (C) riboflavin.

Relatively drastic denaturing conditions were required in order to remove riboflavin completely from the protein. Specifically, the protein was dialysed against 5 M urea in 50 mM potassium phosphate and was then dialysed against 50 mM phosphate pH 7.0. The resulting colourless protein showed full catalytic activity.

Fluorescence titration experiments with riboflavin showed a dissociation constant of 1.3 μM. A similar \( K_d \) value of 1.2 μM was observed in equilibrium dialysis experiments (Fig. 5). The relative fluorescence quantum yield of bound riboflavin as compared to free riboflavin was < 2%.

6,7-Dimethyl-8-ribityllumazine was found to bind to the enzyme with a \( K_d \) of 2 μM as shown by fluorescence titration. Riboflavin-5'-phosphate (FMN) was bound with a \( K_d \) of 16 μM.

Riboflavin can be displaced from the enzyme by the substrate, 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione, as well as by the substrate analogue, 5-nitro-6-ribitylamino-2,4(1H,3H)-pyrimidinedione (compound 5, Fig. 6). The second substrate, 3,4-dihydroxy-2-butanone 4-phosphate, could not displace enzyme-bound riboflavin. However, it facilitated the displacement of riboflavin by the substrate analogue, 5-nitro-6-ribitylamino-2,4(1H,3H)-pyrimidinedione (Fig. 6).

The active sites of riboflavin synthases from *S. cerevisiae* and of *B. subtilis* have been studied in some detail by X-ray crystallography [7,8,11,15]. The heterocyclic moiety of the substrate, 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione, has been shown to form a coplanar complex with phenylalanine 22 in case of the *B. subtilis* enzyme and with tryptophan 27 in case of the yeast enzyme. The most likely positional equivalent of these respective amino acids in the *S. pombe* enzyme is tryptophan 27.

Based on the hypothesis that the unexpected optical properties of the riboflavin/enzyme complex are related to the non-covalent interaction of riboflavin with an aromatic amino acid moiety at the active site, we decided to modify tryptophan 27 by site-directed mutagenesis (Table 1). Replacement of tryptophan 27 by phenylalanine or tyrosine did not significantly affect the kinetic properties (Table 3). The replacement of tryptophan 27 by various other amino acids (glycine, serine, histidine, isoleucine) decreased the maximum catalytic rate by factors up to threefold but had little impact on the maximum catalytic rate. The \( K_m \) value for 5-amino-6-ribitylamino-2,4(1H,3H)-pyrimidinedione

Fig. 5. Equilibrium dialysis. Wild-type enzyme, ▲; W27Y mutant, ●; for details see Methods. r, Number of bound riboflavin molecules per protein subunit; L, concentration of free ligand.

© FEBS 2002

Biosynthesis of riboflavin (Eur. J. Biochem. 269) 525

![Diagram](https://i.imgur.com/1234567.png)

Fig. 6. Displacement of riboflavin by 5-nitro-6-ribitylamino-2,4(1H,3H)-pyrimidinedione (5) in wild-type enzyme treated with a large excess of riboflavin and subsequently dialysed extensively against 50 mM potassium phosphate pH 7.0. The substrate analogue was titrated in 1 mL lumazine synthase (60.2 μM) in 50 mM potassium phosphate pH 7.0 (▲) and in 1 mL lumazine synthase (60.2 μM) in 50 mM

was increased by approximately two orders of magnitude by these mutations, whereas the \( K_m \) for 3,4-dihydroxy-2-butanone 4-phosphate increased only by a factor of about three (Table 3).

As expected, the mutations had major impact on the affinity for riboflavin. Only the wild-type and the W27Y mutant were obtained with bound riboflavin after chromatographic purification. The other mutants were obtained as colourless proteins.

Even in case of the W27Y mutant, the absorption and CD spectra of the riboflavin/enzyme complex differed substantially from those of the wild-type protein (Figs 3 and 4B). Whereas the general shape of the two long-wave absorption bands was similar to that of the wild-type, the long wavelength trail was much weaker in case of the mutant protein. The CD spectrum of the mutant showed a positive Cotton effect at ≈475 nm and a negative Cotton effect at ≈365 nm. In contrast with the wild-type protein, no significant ellipticity was noticed at wavelengths > 550 nm. Equilibrium dialysis experiments afforded a \( K_d \) of 12 μM for riboflavin (Fig. 5).

## DISCUSSION

The structures of lumazine synthases from three bacterial species, three fungi and one plant have been determined at near-atomic resolution. The representatives from fungi, *M. grisea*, *S. cerevisiae*, *S. pombe* and from the bacterium, *Brucella abortus*, are pentameric, whereas the enzymes from Bacillaceae, *Aquifex aeolicus*, *E. coli* and the plant *Spinacia oleracea* form icosahedral capsids [5–12,15,16,29,30]. The pentameric enzymes of *S. cerevisiae* and *Brucella abortus* contain inserts of four amino acids between the helices \( \alpha_4 \) and \( \alpha_5 \) which have been hypothesized to be responsible for the inability of this protein to form an icosahedral capsid as a consequence of steric hindrance [15,16]. The *S. pombe* enzyme contains only a single added leucine residue in this

location by comparison with the icosahedral enzymes studied (Fig. 2).

The purified *S. pombe* lumazine synthase was characterized by bright yellow colour, in contrast with all other lumazine synthases studied in our laboratory which were obtained as colourless proteins. The yellow colour was caused by noncovalent binding of riboflavin together with small amounts of 6,7-dimethyl-8-ribityllumazine. The situation is reminiscent of earlier observations by Plaut and coworkers who obtained riboflavin synthase from bakers' yeast as a complex with bound riboflavin even after extensive purification [31].

Dissociating conditions were required to remove the bound riboflavin from the *S. pombe* enzyme. This observation is well in line with the \( K_d \) value of 1.2 μM observed for riboflavin.

The optical spectrum of riboflavin bound to lumazine synthase from *S. pombe* is characterized by a marked change in the relative intensities of the transition centred at 445 nm and 370 nm. Moreover, a significant absorbance is found in the wavelength range at least up to 550 nm and is accompanied by a Cotton effect at ≈525 nm. This optical anomaly is less pronounced when tryptophan 27 is replaced by tyrosine. Based on comparisons of sequences and three-dimensional structures, it is almost certain that tryptophan 27 is within van der Waals’ distance of the bound riboflavin and is the determining factor for the unexpected riboflavin affinity of the *S. pombe* enzyme. Thus, we suggest tentatively that the optical anomalies described indicate a charge transfer complex involving the isoalloxazine moiety of riboflavin and the indole ring system of tryptophan 27.

## ACKNOWLEDGEMENTS

We thank H. Rau for helpful discussions, N. Schramek for help with the preparation of the manuscript, P. Köhler for protein sequencing, L. Schulte for skillful assistance and A. van Loon for plasmids. This work was supported by the Deutsche Forschungsgemeinschaft and the Fonds der Chemischen Industrie.

## REFERENCES

1. Neuberger, G. & Bacher, A. (1986) Biosynthesis of riboflavin. Enzymatic formation of 6,7-dimethyl-8-ribityllumazine by heavy riboflavin synthase from *Bacillus subtilis*. *Biochem. Biophys. Res. Commun.* **139**, 1111–1116.
2. Kis, K. & Bacher, A. (1995) Substrate channeling in the lumazine synthase/riboflavin synthase complex of *Bacillus subtilis*. *J. Biol. Chem.* **270**, 16788–16795.
3. Kis, K., Volk, R. & Bacher, A. (1995) Biosynthesis of riboflavin. Studies on the reaction mechanism of 6,7-dimethyl-8-ribityllumazine synthase. *Biochemistry* **34**, 2883–2892.
4. Volk, R. & Bacher, A. (1990) Studies on the four carbon precursor in the biosynthesis of riboflavin. Purification and properties of L-3,4-dihydroxy-2-butanone 4-phosphate synthase. *J. Biol. Chem.* **265**, 19479–19485.
5. Bacher, A. (1986) Heavy riboflavin synthase from *Bacillus subtilis*. *Methods Enzymol.* **122**, 192–199.
6. Ladenstein, R., Meyer, B., Huber, R., Labischinski, H., Bartels, K., Bartunik, H.D., Bachmann, L., Ludwig, H.C. & Bacher, A. (1986) Heavy riboflavin synthase from *Bacillus subtilis*. Particle dimensions, crystal packing and molecular symmetry. *J. Mol. Biol.* **187**, 87–100.
7. Ladenstein, R., Schneider, M., Huber, R., Schott, K. & Bacher, A. (1988) The structure of the icosahedral \( \alpha_{60} \) capsid of heavy

riboflavin synthase from *Bacillus subtilis*. Z. Kristallographie 185, 122–124.

8. Ladenstein, R., Ritsert, K., Huber, R., Richter, G. & Bacher, A. (1994) The lumazine synthase/riboflavin synthase complex of *Bacillus subtilis*: X-ray structure analysis of reconstituted α₆₀ capsid at 3.2 Å resolution. *Eur. J. Biochem.* 223, 1007–1017.

9. Mörtl, S., Fischer, M., Richter, G., Tack, J., Weinkauf, S. & Bacher, A. (1996) Biosynthesis of riboflavin. Lumazine synthase of *Escherichia coli*. *J. Biol. Chem.* 271, 33201–33207.

10. Persson, K., Schneider, G., Douglas, B.J., Viitanen, P.V. & Sandalova, T. (1999) Crystal structure analysis of a pentameric fungal and an icosahedral plant lumazine synthase reveals the structural basis for differences in assembly. *Protein Sci.* 8, 2355–2365.

11. Ritsert, K., Turk, D., Huber, R., Ladenstein, R., Schmidt-Bäse, K. & Bacher, A. (1995) Studies on the lumazine synthase/riboflavin synthase complex of *Bacillus subtilis*. Crystal structure analysis of reconstituted, icosahedral α subunit capsid at 2.4 Å resolution. *J. Mol. Biol.* 253, 151–167.

12. Ladenstein, R., Ludwig, H.C. & Bacher, A. (1983) Crystallisation and preliminary X-ray diffraction study of heavy riboflavin synthase from *Bacillus subtilis*. *J. Biol. Chem.* 258, 11981–11983.

13. Bacher, A., Baur, R., Eggers, U., Harders, H., Otto, M.K. & Schnepple, H. (1980) Riboflavin synthases of *Bacillus subtilis*. Purification and properties. *J. Biol. Chem.* 255, 632–637.

14. Bacher, A., Schnepple, H., Mailänder, B., Otto, M.K. & Ben-Shaul, J. (1980) Structure and function of the riboflavin synthase complex of *Bacillus subtilis*. In *Flavins and Flavoproteins* (Yagi, K., ed.). pp. 579–586. Japan Sci. Soc./T. Yamano, Tokyo.

15. Meining, W., Mörtl, S., Fischer, M., Cushman, M., Bacher, A. & Ladenstein, R. (2000) Crystal structure analysis of a pentameric fungal and an icosahedral plant lumazine synthase reveals the structural basis for differences in assembly. *J. Mol. Biol.* 299, 181–197.

16. Braden, B.C., Velikovsky, C.A., Cauerhff, A.A., Polikarpov, I. & Goldbaum, F.A. (2000) Divergence in macromolecular assembly: X-ray crystallographic structure analysis of lumazine synthase from *Brucella abortus*. *J. Mol. Biol.* 297, 1031–1036.

17. Sedlmaier, H., Müller, F., Keller, P.J. & Bacher, A. (1987) Enzymatic synthesis of riboflavin and FMN specifically labelled with ¹³C in the xylene ring. *Z. Naturforsch.* 42, 425–429.

18. Richter, G., Volk, R., Krieger, C., Lahm, H., Röthlisberger, U. & Bacher, A. (1992) Biosynthesis of riboflavin. Cloning sequencing and expression of the gene coding for 3,4-dihydroxy-2-butanone 4-phosphate synthase of *Escherichia coli*. *J. Bacteriol.* 174, 4050–4056.

19. Chirgwin, J.M., Przybyla, A.E., MacDonald, R.J. & Rutter, W.J. (1979) Isolation of biologically active ribonucleic acid from sources enriched in ribonuclease. *Biochemistry* 18, 5294–5299.

20. Stüber, D., Matile, H. & Garotta, G. (1990) System for high level production in *E. coli* and rapid purification of recombinant proteins: application to epitope mapping, preparation of antibodies and structure function analysis. In *Immunological Methods*, IV (Lefkovits, I. & Pernis, P., eds), pp. 121–152.

21. Bullock, W.O., Fernandez, J.M. & Short, J.M. (1987) XL1-Blue: a high efficiency plasmid transforming recA *Escherichia coli* strain with α-galactosidase selection. *Biotechniques* 5, 376–379.

22. Sanger, F., Niklen, S. & Coulson, A.R. (1977) DNA sequencing with chain-terminating inhibitors. *Proc. Natl Acad. Sci. USA* 74, 5463–5467.

23. Read, S.M. & Northcote, D.H. (1981) Minimization of variation in the response to different proteins of the Coomassie blue G dye-binding assay for protein. *Anal. Biochem.* 116, 53–64.

24. Laemmli, U.K. (1970) Cleavage of structural proteins during the assembly of the head of bacteriophage T4. *Nature* 227, 680–685.

25. Mann, M. & Wilm, M. (1995) Electrospray mass spectrometry for protein characterization. *Trends Biochem. Sci.* 20, 219–224.

26. Laue, T.M., Shah, B.D., Ridgeway, T.M. & Pelletier, S.L. (1992) Computer-aided interpretation of analytical sedimentation data for proteins. In *Analytical Ultracentrifugation in Biochemistry and Polymer Science* (Harding, S.E., Rowe, A.J. & Horton, J.C., eds), pp. 90–125. Royal Society of Chemistry, Cambridge.

27. Matthews, B.W. (1968) Solvent content of protein crystals. *J. Mol. Biol.* 33, 491–497.

28. Harders, H., Fäster, S., Voelter, W. & Bacher, A. (1974) Problems in electronic state assignment based on circular dichroism. Optical activity of flavines and 8-substituted lumazines. *Biochemistry* 13, 3360–3364.

29. Zhang, X., Meining, W., Fischer, M., Bacher, A. & Ladenstein, R. (2001) X-ray structure analysis and crystallographic refinement of lumazine synthase from the hyperthermophile *Aquifex aeolicus* at 1.6 Å resolution: determinants of thermostability revealed from structural comparisons. *J. Mol. Biol.* 306, 1099–1114.

30. Bacher, A., Mailänder, B., Baur, R., Eggers, U., Harders, H. & Schnepple, H. (1975) Studies on the biosynthesis of riboflavin. In *Chemistry and Biology of Pteridines* (Pfleiderer, W., ed.), pp. 285–290. Walter de Gruyter Verlag, Berlin & New York.

31. Plaut, G.W.E., Beach, R.L. & Aogaichi, T. (1970) Studies on the mechanism of elimination of protons from the methyl groups of 6,7-dimethyl-8-ribityllumazine by riboflavin synthetase. *Biochemistry* 9, 771–785.
