
Multidrug Efflux in *Pseudomonas aeruginosa*: Components, Mechanisms and Clinical Significance

Keith Poole* and Ramakrishnan Srikumar

Department of Microbiology and Immunology, Queen's University, Kingston, Ontario K7L 3N6, Canada

Abstract: *Pseudomonas aeruginosa* is an opportunistic human pathogen characterized by an intrinsic resistance to multiple antimicrobial agents and the ability to develop high-level (acquired) multidrug resistance during antibiotic therapy. Much of this resistance is promoted by highly homologous three-component efflux systems of broad substrate specificity, of which four have been identified to date. These include MexA-MexB-OprM and MexX-MexY-OprM, which are expressed constitutively in wild type cells and, thus, provide for intrinsic multidrug resistance, and MexC-MexD-OprJ and MexE-MexF-OprN, whose expression so far has only been seen in acquired multidrug resistant mutant strains. Additional homologues of these efflux systems are identifiable in the recently released genome sequence, though their roles, if any, in antimicrobial efflux are unknown. These tripartite pumps are composed of an integral cytoplasmic membrane drug-proton antiporter of the resistance-nodulation-cell division (RND) family of exporters, a channel-forming outer membrane efflux protein (or outer membrane factor [OMF]) and a periplasmic membrane fusion protein (MFP) that links the other two. In addition to a number of antimicrobials of clinical significance, these pumps also export dyes, detergents, disinfectants, organic solvents and acylated homoserine lactones involved in quorum-sensing. While the natural functional of these pumps remains undefined, the fact that they contribute to antimicrobial resistance in *P. aeruginosa* makes them reasonable targets for therapeutic intervention.

INTRODUCTION

The current problem of resistance to multiple antimicrobial agents (multidrug resistance) in many important human pathogens is the direct result of the misuse of antimicrobials over the past several decades. Indeed, the overreliance on antimicrobial agents continues to select for resistance in target bacteria, and to enrich for organisms that are innately resistant. A number of resistance mechanisms contribute to bacterial antibiotic resistance, not the least of which are chromosomally-encoded broadly-specific antimicrobial (so-called multidrug) efflux systems found in a variety of Gram-positive and Gram-negative organisms [1-3]. Bacterial multidrug efflux transporters are generally grouped into five super-families, primarily on the basis of amino acid sequence homology. These include the major facilitator superfamily (MFS) [4], the ATP-binding cassette (ABC) family [5], the resistance-nodulation-division (RND) family [6-8], the small multidrug resistance (SMR) protein family [9] and, very recently, the multidrug and toxic compound extrusion (MATE) family [10] (see Fig. 1). Efflux pumps accommodating clinically relevant antimicrobials fall into the RND or MFS groups and utilize the energy of the proton motive force to export antibiotics from the cell [4,7,11]. RND family transporters are prevalent in Gram-negative bacteria and typically work in conjunction with a periplasmic membrane fusion protein (MFP) [6,12], [also called a periplasmic efflux protein (PEP) [13]], and an outer membrane protein [7] [also called outer membrane efflux protein (OMF) [13] or simply outer membrane factor (OMF)]. This organization provides for efflux of antibiotics across both membranes of the typical Gram-negative organism.

THE MULTIDRUG EFFLUX SYSTEMS OF *P. AERUGINOSA*

*P. aeruginosa* is an opportunistic human pathogen associated with infections of individuals immunocompromised as a result of burns or other severe trauma, underlying diseases, including cancer, diabetes and cystic fibrosis, deliberate immunosuppression and major surgery [14]. A major nosocomial pathogen, the prevalence of *P. aeruginosa* in hospitals owes much to the innate resistance of the organism to multiple antimicrobial agents, antimicrobial use being comparatively high in these institutions. This resistance has and continues to complicate antipseudomonal therapy [15].

Traditionally attributed to the limited permeability of the *P. aeruginosa* outer membrane [16], it is now clear that the organism’s intrinsic multidrug resistance also owes much to the operation of broadly specific antimicrobial efflux systems [17]. Indeed, the contribution of limited drug influx and active drug efflux work synergistically to manifest this resistance, and compromising of either effectively reduces efflux and intrinsic resistance [18,19]. Four multidrug efflux systems have been described to date in *P. aeruginosa*, two of which, MexAB-OprM and MexXY-OprM contribute to

*Address correspondence to this author at the Department of Microbiology and Immunology, Queen’s University, Kingston, Ontario K7L 3N6, Canada; Tel: (613) 533-6677; Fax: (613) 533-6796; E-mail: poolek@post.queensu.ca

OUT

![Diagram](#)

Gram-positive

Gram-negative

IN

Fig. (1). Schematic demonstrating the organization and operation of bacterial multidrug efflux pumps. OM, outer membrane; PP, periplasmic space; CM, cytoplasmic membrane.

intrinsic multidrug resistance. The others, MexCD-OprJ and MexEF-OprN are not expressed in wild type cells but are hyperexpressed in and contribute to the antimicrobial resistance of acquired multidrug resistant mutants.

The *P. aeruginosa* multidrug efflux systems are of the RND-MFP-OMF type (Fig. 1) and show a high degree of amino acid sequence conservation (Fig. 2). A number of homologues of these systems are identifiable within the recently completed *P. aeruginosa* genome sequence [20] (Fig. 2) although their contribution, if any, to antimicrobial resistance remains to be demonstrated. Related RND-MFP-OMF multidrug efflux systems have also been described in other bacteria including *Escherichia coli* [21,22-25], *Salmonella enterica* serovar Typhimurium [26,27], *Haemophilus influenzae* [28], *Neisseria gonorrhoeae* [29], *Burkholderia cepacia* [30] (Burns, J.L., Pritchlaff, C., Barry, J., Charron, M., Cieri, M. Abstr. 98th Gen. Meet. Amer. Soc. Microbiol., abstr. V-108, 1998), *Burkholderia pseudomallei* [31], *Pseudomonas putida* [32] and *Stenotrophomonas maltophilia* (Zhang, L., Li, X.-Z., and Poole, K., unpublished; GenBank accession number AF173226)[33]. Although these systems tend to be chromosomally-encoded, the recent identification of an RND type efflux gene on a plasmid [34] indicates that efflux-mediated multidrug resistance may be plasmid-determined as well.

### MexAB-OprM

The first multidrug efflux system to be described in *P. aeruginosa* is encoded by the *mexAB-oprM* operon (previously called *mexAB-oprK*) [17,35-37]. Expressed constitutively in cells grown under standard laboratory conditions, this system contributes to the intrinsic resistance

Multidrug Efflux in Pseudomonas aeruginosa

| REG | MFP | RND | OMF | Substrates |
| --- | --- | --- | --- | --- |
| $\leftarrow$ | $\rightarrow$ | $\rightarrow$ | $\rightarrow$ | antibiotics, solvents, dyes, detergents, disinfectants, homoserine lactones |
| $mexR$ | $mexA$ | $mexB$ | $oprM$ |  |
| $\leftarrow$ | $\rightarrow$ | $\rightarrow$ | $\rightarrow$ | antibiotics, solvents, dyes, detergents |
| $nfxB$ | $mexC$ | $mexD$ | $oprJ$ |  |
|  | $(43.4)$ | $(49.8)$ | $(46.1)$ |  |
| $\rightarrow$ | $\rightarrow$ | $\rightarrow$ | $\rightarrow$ | antibiotics, solvents |
| $mexT$ | $mexE$ | $mexF$ | $oprN$ |  |
|  | $(32.2)$ | $(41.4)$ | $(30.3)$ |  |
| $\leftarrow$ | $\rightarrow$ | $\rightarrow$ |  | antibiotics, dyes |
| $mexZ$ | $mexX$ | $mexY$ |  |  |
|  | $(36.3)$ | $(46.8)$ |  |  |
| $\rightarrow$ | $\rightarrow$ |  |  | ? |
| $pa1435$ | $pa1436$ |  |  |  |
| $(30.4)$ | $(29.6)$ |  |  |  |
| $\rightarrow$ | $\rightarrow$ |  |  | ? |
| $pa3677$ | $pa3676$ |  |  |  |
| $(23.4)$ | $(24.9)$ |  |  |  |
| $\rightarrow$ | $\rightarrow$ |  |  | ? |
| $pa4374$ | $pa4375$ |  |  |  |
| $(24.4)$ | $(33.3)$ |  |  |  |
| $\rightarrow$ | $\rightarrow$ | $\rightarrow$ |  | ? |
| $pa3523$ | $pa3522$ | $pa3521$ |  |  |
| $(30.4)$ | $(39.6)$ | $(27.9)$ |  |  |
| $\rightarrow$ | $\rightarrow$ | $\rightarrow$ |  | ? |
| $pa4206$ | $pa4207$ | $pa4208$ |  |  |
| $(23.9)$ | $(30.0)$ | $(29.7)$ |  |  |
| $\rightarrow$ | $\cdots\cdots$ | $\rightarrow$ | $\rightarrow$ | pyoverdine? |
| $pa2389$ | $pa2390$ | $pa2391$ |  |  |
| $(23.8)$ | $(-)$ | $(30.9)$ |  |  |

Fig. (2). Genetic organization of multidrug efflux genes of *P. aeruginosa*. The organization of genes encoding known and homologues of multidrug efflux systems are indicated under headings describing the family (REG, regulatory protein; MFP, membrane fusion protein; RND, resistance-nodulation-cell division; OMF, outer membrane factor) to which the gene products belong. Known genes are designated while uncharacterized ORFs identified in the *P. aeruginosa* genome sequence are described using the nomenclature of the *Pseudomonas* genome project (http://www.pseudomonas.com/). Substrates of the various multidrug efflux systems, where known, are indicated at right. Numbers in parentheses represent the percent amino acid identity of the indicated gene product relative to MexA (for the MFP homologues), MexB (for the RND homologues) or OprM (for the OMF homologues). Note that ORF pa2390 actually encodes a transporter of the ABC family and that the putative efflux operon pa2389/90/91 occurs within the *pvd* locus (see Fig. 4)

although MexAB-OprM-mediated resistance to meropenem has been demonstrated [44,60,61].

Upstream of $mexAB-oprM$ lies a gene, $mexR$, that encodes a repressor of $mexAB-oprM$ expression [38,62]. MexR is a member of a family of regulators that includes MarR [63], the product of the first gene of the $marRAB$ operon (or $mar$ locus as it is also known) in *E. coli*. MarR negatively regulates $marRAB$ expression, thereby controlling expression of the transcriptional activator MarA and, thus, several MarA-regulated genes associated with resistance including the $acrAB$ multidrug efflux operon [64]. Two MexR binding sites have been identified within the $mexR$-$mexA$ intergenic region, overlapping promoters for $mexR$ and

$mexAB-oprM$ (Evans, K., Adewoye, L., and Poole, K., manuscript submitted). Hyperexpression of $mexAB-oprM$ in $nalB$ mutants results from mutations in $mexR$ [38] and is the target of mutations in $nalB$ strains [38,50,62,65,66], although $mexAB-oprM$ hyperexpression can also occur independent of mutations within $mexR$ or the promoters of $mexR$ and $mexA$ [50,62]. These so-called $nalC$ mutants [62] presumably carry a mutation in an unidentified regulator of $mexAB-oprM$ expression. The observation that MexAB-OprM expression is also growth phase regulated, independent of $mexR$ (i.e. in both wild type and $nalB$ strains of *P. aeruginosa*) [67], suggests that yet additional regulator(s) are involved in controlling $mexAB-oprM$ expression.

The MexCD-OprJ multidrug efflux system [68] is not expressed in wild type cells under typical laboratory growth conditions [39, 69] and, thus, does not contribute to the intrinsic antibiotic resistance of the organism. Expression of this efflux system is seen, however, in *nfxB* mutants where it is responsible for the multidrug resistance of these mutants [43, 68, 70]. Two classes of *nfxB* mutants have been described, expressing moderate (type A) or high (type B) levels of the efflux system, with resistance levels correlating with efflux gene expression [71]. Hyperexpression of *mexCD-oprJ* results from mutations in a gene, *nfxB* [72, 73], which occurs upstream of the efflux genes and encodes a repressor of *mexCD-oprJ* expression)[68]. NfxB also negatively regulates its own gene [74].

Originally identified as a determinant of fluoroquinolone resistance [70], MexCD-OprJ accommodates a variety of antimicrobial agents including macrolides, chloramphenicol, novobiocin, tetracycline, trimethoprim and some β-lactams [39, 40, 71, 75]. Although the MexCD-OprJ export of β-lactams was originally reported to be limited to 4th generation cephalosporins (e.g., cefpirome and cefepime) [68, 71], more recent studies using mutants lacking MexAB-OprM but expressing MexCD-OprJ have confirmed the ability of this efflux system to accommodate ordinary cephalosporins such as cefoperazone and ceftazidime [39, 75]. MexCD-OprJ is still distinguishable from MexAB-OprM, however, by the latter's inability to export additional β-lactams such as carbenicillin and aztreonam [39, 75].

An interesting feature of *nfxB* strains is their hypersusceptibility to β-lactams such as carbenicillin [39, 70, 75], which apparently results from the reduced expression of MexAB-OprM in these mutants [75]. *nfxB* Mutants are also hypersusceptible to aminoglycosides [68, 70], a major substrate for the MexXY-OprM multidrug efflux system (see below). As such, this latter efflux system may also be downregulated in *nfxB* mutants. The coordinated expression of multidrug efflux systems in *P. aeruginosa* implied by these results was recently confirmed by studies assessing *mexCD-oprJ* and *mexEF-oprN* expression in response to the presence or absence of MexAB-OprM [76]. Li and co-workers provided evidence for an inverse relationship between expression of MexAB-OprM and the other two efflux systems [76]. While the natural substrate(s) of these efflux systems are as yet undefined, the overlap in antimicrobial substrate specificity would suggest that any cell-associated products exported by these systems could be accommodated by more than one efflux system. Thus, the ability to maintain an export competent phenotype with respect to these substrates would not require all systems to be expressed concomitantly, just that a basal level of efflux activity be retained.

strains [43, 77, 78]. Originally selected as fluoroquinolone-resistant mutants [43, 78], *nfxC* mutants have also been isolated on media containing tetracycline or chloramphenicol [48]. These mutants are resistant to fluoroquinolones, chloramphenicol, trimethoprim and the carbapenem imipenem [77, 78]. Resistance to imipenem in *nfxC* strains does not, however, result from MexEF-OprN expression [77] but rather the concomitant decrease in outer membrane protein OprD in these mutants [43, 78]. OprD is a channel-forming porin protein that promotes imipenem entry into *P. aeruginosa* [79], and mutational loss of this protein is the most frequent cause of imipenem-resistance [60, 80]. The hypersusceptibility of *nfxC* strains to β-lactams and aminoglycosides [78] may, as suggested for *nfxB* mutants which share this phenotype, result from decreased expression of MexAB-OprM and MexXY-OprM. The hyperexpression of MexEF-OprN in *nfxC* mutants is known to require the product of the *mexT* gene, although this gene is not the target of mutation in these mutants. MexT is a positive regulator of *mexEF-oprN* expression [81, 82] and is also responsible for the decrease in OprD expression seen in *nfxC* strains [81, 82]. This latter effect is manifested at the level of *oprD* transcription [81, 82] although posttranscriptional effects on OprD production have also been reported [81].

### MexXY-OprM

In contrast to the aforementioned efflux operons, the recently described *mexXY* system [83] (also called *amrAB* [84]) lacks a linked outer membrane gene [83]. This is reminiscent of the *acrAB* MDR efflux operon of *E. coli* whose outer membrane gene, *tolC*, is also located elsewhere on the chromosome [21-23]. MexXY apparently utilizes OprM as its outer membrane constituent [83, 85]. Strains deleted for *mexXY* are more susceptible to aminoglycosides, tetracycline and erythromycin [85], indicating that this efflux system contributes to the intrinsic resistance of *P. aeruginosa* to these agents. This resistance is, however, dependent upon induction of MexXY in wild type strains by these agents [86]. Interestingly, the cloned genes also promote resistance to fluoroquinolones [83, 85] although this efflux system does not contribute to intrinsic resistance to these agents. This appears to be due to the failure of these agents to induce MexXY expression in wild type cells, since mutants hyperexpressing MexXY demonstrate enhanced fluoroquinolone resistance [86]. Hyperexpression of MexXY/AmrAB is seen in several impermeability type aminoglycoside-resistant strains of *P. aeruginosa*, and elimination of *amrB* (*mexY*) compromises this resistance, confirming the role of AmrAB/MexXY in the amino-glycoside resistance of these mutants [84].

A gene, *mexZ* (also called *amrR* [84]), has been identified upstream of *mexXY* and apparently encodes a repressor of *mexXY/amrAB* expression [84, 85]. Deletion of this gene enhances *amrAB/mexXY* transcription, although a Δ*amrR* strain is not aminoglycoside-resistant [84], presumably because OprM is not hyperexpressed. Aminoglycoside resistance attributable to MexXY/AmrAB hyperexpression in impermeability type mutants must, therefore, rely on mutations in genes in addition to or besides *amrR*.

### MexEF-OprN

The MexEF-OprN system also appears to be quiescent in wild type cells under the usual laboratory growth conditions [77], although it is expressed in *nfxC* type multidrug resistant

Multidrug Efflux in Pseudomonas aeruginosa

COMPONENTS OF MULTIDRUG EFFLUX

The challenge for multidrug efflux systems in Gram-negative bacteria such as *P. aeruginosa* is the need to traverse two membranes in removing antimicrobials from the bacterial cell. Unlike single component drug transporters such as TetA and related tetracycline efflux transporters, which mediate resistance merely by pumping antibiotics to the periplasm [87], it appears that RND-MFP-OMF type efflux systems need to deliver antimicrobials to the extracellular milieu in order to promote resistance. Apparently, the low $V_{\text{max}}$ of RND transporters [87] would not permit these pumps to counter the rapid re-entry of antimicrobials across the highly permeable cytoplasmic membrane, should they deliver antimicrobials to the periplasm only. Export of antimicrobials outside the cell, however, allows these low $V_{\text{max}}$ pumps to effect meaningful resistance, as re-entry of antimicrobials will be restricted by the limited permeability of the outer membrane [7, 87]. Thus, the OMF and MFP components of the typical RND-MFP-OMF pump play a crucial role in promoting the extracellular release of antimicrobials, the former by forming a channel through the periplasm and outer membrane [88], and the latter by joining the RND and OMF components [89].

RND Efflux Transporters

Multidrug transporters of the RND type are predicted to catalyze drug proton antiport across the cytoplasmic membrane [11], although this has only been demonstrated for the AcrB component of the AcrAB-TolC multidrug efflux system of *E. coli* [90]. Still, the high degree of homology between AcrB and the MexB, MexD, MexF and MexY components of the *P. aeruginosa* efflux systems and the observation that protonophores such as CCCP abolish efflux-mediated resistance promoted by the *P. aeruginosa* pumps suggest that these are also energized by the proton motive force. Topological analysis of MexB [91] and MexD [92] indicate that these integral cytoplasmic membrane proteins possess 12 membrane-spanning helices with large periplasmic loops of ca. 300 residues occurring between helices 1 and 2, and 7 and 8. The exact function of these extensive periplasmic loops is unknown although they may promote pump assembly by interacting with the corresponding MFP (MexA, MexC) and/or OMF (OprM, OprJ) components. The in vivo association of MexA and MexB has been demonstrated (Tibbo, J. and Poole, K., unpublished data), in agreement with recent results demonstrating that AcrA (a MexA homologue) and AcrB (a MexB homologue) of the *E. coli* AcrAB-TolC multidrug efflux system also interact in vivo [93]. In contrast to studies (see below) demonstrating the interchangeability of OMF components of the various *P. aeruginosa* multidrug efflux systems, exchanging the inner membrane or periplasmic components of MexAB-OprM with their MexCD-OprJ counterparts failed to yield functional export systems [94]. Thus, a functional complex may be restricted to cognate inner membrane and periplasmic components, due presumably to the specificity of MexC interaction with MexD and MexA interaction with MexB. Still, a recent report indicated that MexB could, in fact, replace MexD in producing a functional MexC-MexB-OprJ chimera (Gotoh,

N., Kuwagaki, M., Murata, T., Shin, T., Takahashi, M., and Nishino, T. Abstr. 40${}^{\text{th}}$ Intersci. Conf. Antimicrob. Agents Chemother., abstr. C-440, 2000).

A number of studies have shown that the cytoplasmic membrane components of the multidrug efflux systems are responsible for substrate recognition [39, 95, 96]. Still, it wasn’t clear whether the integral cytoplasmic membrane RND component was solely responsible or whether the periplasmic but cytoplasmic membrane-anchored MFP component also played a role. The recent observation, however, that the substrate specificity of a functional MexC-MexB-OprJ chimera was reminiscent of MexAB-OprM argues that MexB (i.e. the RND component) determines substrate specificity (Gotoh, N., Kuwagaki, M., Murata, T., Shin, T., Takahashi, M., and Nishino, T. Abstr. 40${}^{\text{th}}$ Intersci. Conf. Antimicrob. Agents Chemother., abstr. C-440, 2000). The physicochemical basis for this broad substrate-specificity is unknown, although there are suggestions that substrate is accessed by the pumps at least partially from within the cytoplasmic membrane bilayer [1, 97, 98]. A study of efflux-mediated resistance to β-lactams mediated by the AcrAB system of *S. enterica* sv. Typhimurium, for example, showed a correlation between resistance and hydrophobicity of the β-lactams [26], consistent with the need for the antibiotics to partition within the cytoplasmic membrane to be exported by this system. The observation, too, that the only carbapenem antibiotic which is readily exported by MexAB-OprM, meropenem, is also the most amphiphilic [60] is consistent with the need for efflux substrates to enter the cytoplasmic membrane. Finally, the fact that organic solvents are exported by a number of RND-MFP-OMF type efflux systems [32, 99-105], including the *P. aeruginosa* multidrug efflux systems [53, 54] strongly favour the membrane as being a primary site for substrate recognition by these pumps, since solvents are lipophilic agents which target biological membranes [106].

An enduring curiosity regarding multidrug transporters is their ability to recognize and pump a variety of structurally diverse compounds. Although there is as yet no data on the mechanism of multiple substrate recognition by RND transporters, studies of mammalian ABC type multidrug transporter P-glycoprotein [107] and the MFS type LmrP multidrug transporter of *Lactococcus lactis* [108] reveal that these transporters possess multiple substrate-binding sites. In contrast, the BmrR protein, which regulates expression of the Bmr multidrug transporter in *Bacillus subtilis* and binds many of the same substrates exported by Bmr [109], has a single flexible interaction site, where drug binding is mediated by different sets of agent-specific electrostatic and van der Waals interactions [110-112]. While the latter is not a multidrug transporter, it does provide an example of how multiple substrates can be bound at a single site. Whether RND transporters possess a single or multiple sites, and the nature of these site(s) remains to be elucidated.

Outer Membrane Efflux Proteins

The OMF component of RND-MFP-OMF multidrug efflux systems is responsible for antimicrobial export across the outer membrane. Presumed to be channel-forming

proteins, channel formation by OprM has recently been demonstrated in planar lipid bilayer membranes [113]. The related TolC OMF of the AcrAB-TolC pump, too, has been shown to form channels in lipid bilayer membranes [114]. Still, OMFs are seemingly unrelated to the porin family of channel-forming proteins and likely differ structurally from these β-sheet rich proteins [13]. Indeed, the crystal structure of the TolC channel has been solved and describes a predominantly α-helical trimeric protein that spans both the outer membrane (as a β-barrel) and the periplasm (as an α-helical barrel) [88]. Despite somewhat limited homology between TolC and OprM, OprM is predicted to be predominantly α-helical and structurally reminiscent of TolC [115, 116], and suggestions of a porin-like OprM comprised of 16 membrane-spanning β-strands [113] are likely incorrect. The suggestion, too, that OprM is not the outer membrane export channel of the MexAB-OprM pump [117] is probably also in error. The OMF components of the RND-MFP-OMF pumps are, thus, responsible for antimicrobial export across both the periplasm and the outer membrane and, as such, probably interact with the RND exporters of the cytoplasmic membrane. In this scenario, the MFP component promotes association of the OMF and the RND transporter components, and is not part of the export conduit itself (see Fig. 1).

OMFs do not appear to maintain a constant association with the other efflux components. In contrast to the RND and MFP components which are stably associated in vivo (see above), evidence for OprM or TolC association with the MexAB/AcrAB components is lacking. This makes sense since many OMFs, including OprM [94, 118] and TolC [7, 119, 120] function as components of multiple export systems and, thus, cannot be sequestered by any one system. Possibly, the OMF component is recruited by the RND-MFP

components only when an antimicrobial substrate is actively being pumped. TolC is, for example, also the OMF of the HlyBD-TolC hemolysin export system and associates with the HlyBD components (members of the ABC and MFP families, respectively) only when hemolysin is actively being exported. In contrast, HlyB and HlyD remain associated independent of export activity [121]. Thus, OprM association with MexAB or MexXY may be transient and dependent upon the export activity of these pumps.

OprM, unlike TolC, contains an N-terminal signature sequence for acylation (so-called lipoprotein box [122]) within which is a conserved cysteine residue, the presumed site of acylation (Fig. 3). Recently, acylation of OprM has been demonstrated [117] although acylation appears somewhat dispensable as regards OprM function, and OprM derivatives lacking the N-terminal lipid tail are active [115, 117]. While the significance of OprM acylation is unknown, it is noteworthy that the other OMFs in *P. aeruginosa* as well as a number of OprM homologues encoded as part of three-component efflux operons (Fig. 2) also possess N-terminal lipoprotein boxes (Fig. 3). This conserved lipid moiety likely plays a role in the disposition of the N-terminal amino acids of these proteins, residues that are lacking in TolC. Whether this additional sequence in the *P. aeruginosa* OMFs is uniquely important for their function is unknown. Still, efflux-mediated multidrug resistance attributable to MexAB-OprM and MexCD-OprJ appears to require the product of the *tonB* gene (now called *tonBI*) [123], a protein whose *E. coli* counterpart is known to interact with certain outer membrane channel-forming proteins [i.e., those involved in the uptake of ferric siderophores and vitamin B12 [124, 125]] at the N-terminus [126, 127]. Such interaction is energy-dependent and promotes conformational changes in the channel-forming


oprJ    MKKPAFGVSALLIALGACSMAPTYCRPAAPVADWSWG--AAAQROQGAAD-- 50
oprM    MKR-SF-LSLAVAAVYLSQCSLIPOYQPEAPVARAYPQQQAYQNTGAAAVPA 52
oprN    MIFAQSIRSGLASALGLFSLLMLSAQTWGPDYTPOTAANKIDATAASKPYDRSREFSL-- 59
PA4208  MKRSYPMLSRLALALAAG--YGLAACSVGPDYORFPSPPRVASSEHLGEFSGERREAP-- 56
PA3521  --MKPYLASSLSALILLLQQCAVGFOYAPPSSASAPASFGAMPAGIDGSQVEIK- 51
PA2391  --MSMRNLSLISACLILIGACOSTPALDSGLAAPSQWRYLAAGRSADSDIRQ-- 50
PA2525  -MKETPSLLALALVAALGQQAMIGPDYRPDLAYPAEFFKLADGWRRALFRDVYQR 53
        *: :*: **:: ::   .: .:.*:
oprJ    -TLDWKSFIVDAELRRLYVQALDNNRSRLQTLLDEAARAQYRIQRADRVPGCLNAATCN 109
oprM    ADIGWEEFFDPDQLQOLIGVALDNRSDLKVAAINVEAFRAMQYRIQRADLFPRIGVDGSGT 112
oprN    ----WWKQFDFTLMQLVVEQSLSGNEDLKVFARLEAAHRLDOVANDFFPVYTSPASAD 114
PA4208  ----WWSEFDDGQLYRLDLQALARNHDIPERANLRNSARALFDOKWLDDLQLPQTSQAGYS 112
PA3521  ----WWRGEDIPALESLIQRALAANLDEIAGARLDEAKALLPEREEFLPLPRGGPAFDYQ 107
PA2391  ----WWKAFGAPFELDSLQLALLNSQDLGAAVAVERQAQASAVIAGAPLLPELNATLGAS 106
PA2525  --GAWWELEYGDQTINDLQMHELERSQTILAQSVAQFRQALALVRGARAAFTPSTIQVGTK 111
        *: :*: **:: ::   .: .:.*:
oprJ    RQRQPADL---------SAGNRSTVASSYQVGLALPEYELDLFGRYKSLTDAALQQYLA 159
oprM    RQRLPGDL----------STZGSPAISSEQYGVTIGTTAMELDLFGRLPSLFDQALECYLA 162
oprN    LGKQQQPG----------VTGD--FVNSERYDLGIDSAMELDFGRIRRQLESSDALSEA 162
PA4208  BSIEQQLD----------YDGEPRRRLAESYRAGFDAQWIEDLFCRLCRLSDAALAPAEA 162
PA3521  AFERGEV2----------TPAC-QQRDIEBTYRGALDASWEIDLFGVRORSVEEMMAGGS 156
PA2391  RQKLLRDSG--------YSGTDATSUNDAAVDSFASGLSASTYDFWGRQJAYRSALESLRA 160
PA2525  RSQQOOODSTVLLPQSTSVSSCGSGAISTSYSTNLVSWSVDLWGKRLOLEANQASLBA 171
        :*: *::*
oprJ    SEEAARAARIALVAEV3QAYLSYDGALERLALTRQTLYSRIYSFALIDQRAAAGAATALD 219
oprM    TQACQRSACTTLVASVATAYLTLKADQAJQLQLTDILGTYQSFDLTQRSYDVGVASALD 222
oprN    AEADLQQQLQVSLLABVDAYQOLRGAQLFKIALSNLEMQKLSRQLTEQRLDAGVGAELO 222
PA4208  ADADLRVLRLSIADDTAPAYTIIOQYQERLDVARQVSRWDTLELTRSSLQIGSCLPED 222
PA3521  REALLRNQASVAATVAMSWEQLQGIJEALAVWDIAQNQFDSELMERLVSEAGSAHEFD 216
PA2391  SEYDRATVELTLSLVCANSYLQVLLALREQQRIARLNLDNEREVLRLVEIERAAGSATALE 220
PA2525  SAADLAAVORLSQQSQLAQNYLQHRVHDSQIRLLNDTVTAYFRSLKVAENKYRAGITYTRAD 231
        : : : : : : : :
oprJ    YQKALGLVEQRAEQERNLRQKQQAFNALVLLLGSDDAAQAIIRS-POQRPKLLQQIAPG 278
oprM    LRQAQTAEGARATILAQYTRLVAQQQNALVLLLLGSG-IPANLBQG-LGLDQZLLLTEVPAG 280
oprN    VLRADARLLATAASVPQLQAZAEPRERIATLLLGQSPPEELTVDL5-PRDLPAYTKALPIG 281
PA4208  VENQAQANLLRSEAAIPPLTTAALESARYRLYRLEAGPSGAFILD-GGAAPALPKNNLPQ 281
PA3521  RLRAEALLHENVEAAVPDLERPEATPNALVLLAAAPQATSFWARASGERLTLLTGVG 276
PA2391  VAQQSSILVAQKKQLPULLSQQABEALITLATLIGEPVQALICQAEER--PFDSLRWPSTCAG 278
PA2525  VAQARTQLKSQQAQIDLDLKQSAQLEHAIAVLVGLPFAQFNLPFY---ASVPKLPDLPAY 288
        : : * .


Fig. (3). Multiple alignment of OprM and its OMF homologues in *P. aeruginosa*. Known OMFs as well as putative OMFs identified from BLAST searches of the recently completed genome project (here designated with their PA identifier number) were aligned using CLUSTALW. A highly conserved lipoprotein box is boxed at the N-terminus. Regions of OprM predicted to occur as outer membrane spanning _-sheets are underlined. Several regions of high sequence conservation are obvious and deletions in two of these (boxed) were previously shown to abrogate OprM expression or activity [115].

proteins necessary for ligand uptake across the outer
membrane [128]. Thus, tonB1 mutants of P. aeruginosa are
defective in ferric-siderophore uptake [129, 130]. The
observation, then, that efflux-mediated multidrug resistance
is similarly compromised in such mutants suggests that the
P. aeruginosa OMF components, too, may form TonB-
dependent channels. TonB interaction with the OMFs,
perhaps at the unique N-termini of the proteins, would in this
instance, however, promote drug export across the outer
membrane. While it is true that purified OprM forms
channels independent of TonB [113], such channels are too
small to accommodate the known antimicrobial substrates of
MexAB-OprM and, thus, TonB-promoted conformational
changes in OprM may be necessary for full channel opening
and drug extrusion across the outer membrane. Alternatively,
the tonB defect may have an indirect effect on pump activity,
unrelated to an influence on OMF channel activity, and OMF
interaction with the RND and/or MFP components may
provide the impetus for the necessary conformational
changes.

An interesting phenomenon as regards the OMFs is the
apparent flexibility of the RND-MFP-OMF pumps with
respect to the OMF component that can be used. Originally
constructed to assess the involvement of OprM in
determining the β-lactam specificity of MexAB-OprM,
MexAB-OprJ and MexCD-OprM chimeras were, for
example, active with respect to efflux-mediated resistance
[39]. This ability of OprM to replace OprJ has also been
confirmed by other researchers [95]. Similarly, OprM is able
to replace OprN in producing a functional MexEF-OprM
chimera [96]. Finally, the MexCD components are able to
function with the E.coli TolC protein [42].

Outer membrane efflux proteins are not functional in the
absence of their RND-MFP counterparts [131]. The
demonstration, then, that the cloned oprM gene was able to

promote efflux-mediated antibiotic resistance in a mexAB-
oprM deletion strains was interpreted as OprM operating as
the outer membrane component of additional antibiotic
efflux systems [118, 132]. Indeed, it is now known to
function as the OMF of the MexXY-OprM efflux system
[83, 85]. Intriguingly, strains lacking OprM (e.g. a Δ-mexAB-
oprM mutant) are more susceptible to fluoroquinolones and
tetracyclines than is a _mexB_mexXY double mutant (Table
1), arguing that OprM likely functions as the OMF of
additional, as yet unidentified drug efflux systems. In this
regard, there are a number of gene pairs in the P. aeruginosa
genome sequence encoding RND-MFP homologues that lack
a linked OMF gene (Fig. 2). One or more of these might
function with OprM in mediating the observed MexAB/
MexXY-independent, OprM-dependent resistance to fluoro-
quinolones and tetracyclines.

Examination of the P. aeruginosa genome sequence has
also revealed a gene for an OprM homologue within the
previously described pvd locus of pyoverdine biosynthesis
and uptake [133]. Interestingly, the gene appears to form part
of a three component export operon which includes an MFP
gene and, in place of an RND exporter gene, a gene encoding
an ABC family transporter (Fig. 4). It is tempting to
speculate that this operon encodes an export system for
pyoverdine.

MFP Components

MFPs were so-named owing to the homology of their C-
termini with viral membrane fusion proteins [12]. They have
been predicted to promote the association of the OMF and
RND components of the typical RND-MFP-OMF pump [89]
although direct experimental evidence for this has yet to be
provided. A member of the MFP class of proteins, AcrA of
E.coli, has been shown to promote the close association of

Table 1. Antibiotic Susceptibility of P. aeruginosa Efflux Mutants*

| Antibiotic | PAO1 | ΔmexB | ΔmexXY | ΔmexB, ΔmexXY | ΔmexAB-oprM |
|------------|------|-------|--------|---------------|--------------|
| Carbenicillin | 64 | 1 | 64 | 1 | 1 |
| Chloramphenicol | 16 | 2 | 16 | 2 | 2 |
| Erythromycin | 512 | 512 | 64 | 32 | 32 |
| Gentamicin | 4 | 4 | 1 | 1 | 1 |
| Sparfloxacin | 0.5 | 0.25 | 0.25 | 0.125 | 0.031 |
| Trovafloxacin | 0.5 | 0.25 | 0.25 | 0.125 | 0.031 |
| Clinafloxacin | 0.062 | 0.031 | 0.062 | 0.016 | 0.004 |
| Moxifloxacin | 1 | 0.5 | 0.5 | 0.25 | 0.06 |
| Tetracycline | 8 | 8 | 8 | 1 | 0.5 |
| Doxycycline | 8 | 8 | 8 | 1 | 0.5 |

*The susceptibility of P. aeruginosa deleted for the indicated efflux genes was determined by assessing the growth of the bacteria in media containing serial two-fold dilutions of the
indicated antibiotics. The MIC or minimum inhibitory concentration represented the lowest concentration of antibiotic that prevented visible growth of the bacteria. Note that
deletion of the mexAB-oprM genes will compromise MexAB-OprM as well as additional efflux systems that utilize OprM as the OMF component.

$pvdA$  
$pvdR$  
$pvdI$  

MFP  
ABC  
OMF  

$pvdE$  
$fpvA$  
$pvdD$  

$pvd2389$  
$pvd2390$  
$pvd2391$  
$pvd2392$  
$pvd2393$  
$pvd2394$  
$pvd2395$  

Fig. (4). Genetic organization of the $pvd$ locus highlighting genes encoding a putative three-component efflux system (in black). The *Pseudomonas* genome project designations are provided for each of the three ORFs, as is the family of the deduced product of these ORFs (MFP, membrane fusion protein; ABC, ATP-binding cassette; OMF, outer membrane factor). Genes known to play a role in pyoverdine biosynthesis are indicated in white with the appropriate gene designations. Genes encoding a putative two-component sensor ($pvdR$) and extracytoplasmic function (ECF) sigma factor ($pvdI$) are highlighted in gray. Additional ORFs identified by the Pseudomonas genome project are shown as striped boxes with the identifier numbers provided by the project.

adjacent membranes [90], consistent with a role in linking the outer membrane OMF and cytoplasmic membrane RND components. There are indications that MFPs may operate as oligomers, with multimers of AcrA [93] and MexA (Li, X.-Z., and Poole, K., unpublished data) observed in cross-linking studies intended to assess AcrA/MexA interactions with their pump counterparts.

Many, but certainly not all members of the MFP family are lipoproteins, although all of the MFPs of the RND-MFP-OMF multidrug efflux systems are likely acylated. The MexA MFP is a lipoprotein although acylation of this protein is dispensable for function [134]. Similarly, an unacylated version of the normally acylated AcrA is also active in drug efflux [135]. This dispensability of MexA acylation is intriguing since MexA is a periplasmic protein apparently anchored to the cytoplasmic membrane solely via its lipid tail [134]. Perhaps, a cytoplasmic membrane association is only necessary to facilitate an interaction with the MexB component. Under conditions where an unacylated MexA is hyperexpressed (from a multicopy plasmid-borne gene as in [134]), sufficient MexA-MexB complexes may form to promote a wild type level of efflux activity despite the lack of membrane anchoring. Association of wild type (i.e. acylated) MexA and MexB has been demonstrated using an in vivo cross-linking approach (Tibbo, J., and Poole, K., unpublished data) although no studies to date have examined MexB association with a lipid-free MexA.

**CLINICAL SIGNIFICANCE OF MULTIDRUG EFFLUX SYSTEMS**

Despite their broad substrate specificity and, thus, ability to promote resistance to multiple antimicrobial agents, the multidrug efflux systems of *P. aeruginosa* are seen primarily as determinants of resistance to fluoroquinolones [2]. Indeed, there are numerous reports in the literature of fluoroquinolone-resistant strains exhibiting cross-resistance to unrelated antimicrobials [44, 45, 70, 78, 136–142], a phenotype now attributable to hyperexpression of multidrug efflux systems. And while target site (i.e. topoisomerase) mutations prevail in fluoroquinolone-resistant bacteria in general [143], in vitro-selected fluoroquinolone-resistant

strains of *P. aeruginosa* typically hyperexpress multidrug efflux systems [144–147]. Perhaps most importantly, however, multidrug resistant clinical strains of *P. aeruginosa* hyperexpressing multidrug efflux systems have been described, including *nalB* [50, 65, 141, 148] (or *cfxB* [136]), *nfxB* [65, 142, 144, 148] and *nfxC* [148, 149] mutants. Hyperexpression of MexXY/AmrAB is seen in a number of so-called impermeability type aminoglycoside-resistant strains of *P. aeruginosa*, and elimination of *amrB* (*mexY*) compromises this resistance, confirming the role of AmrAB/MexXY in the aminoglycoside resistance of these mutants [84]. There is even a report of a clinical *P. aeruginosa* strain expressing two multidrug efflux systems simultaneously [150]. Still, the multidrug efflux systems appear to contribute minimally to the resistance of *P. aeruginosa* in biofilms [151](de Kieviet, T.R., Parkins, M.E., Gillis, R.J., Ceri, H., Srikumar, R., Poole, K., Iglewski, B.H., and Storey, D.G., manuscript submitted), a common mode of growth of the organism in vivo. Similarly, AcrAB is not a significant determinant of biofilm resistance to ciprofloxacin in *E. coli* [152]. Intriguingly, recent evidence points to multidrug efflux systems also contributing to the virulence of *P. aeruginosa*, inasmuch as mutants lacking certain of these systems are attenuated in animal and tissue culture models of infection (Srikumar, R., Hirakata, Y., Yan, H., Hancock, R.E.W., and Poole, K. Abstr. 40<sup>th</sup> Intersci. Conf. Antimicrob. Agents Chemother., abstr. C-441, 2000). It may be, however, that loss of these systems compromises the fitness of the organism rather than production of a particular virulence factor. Hyperexpression of multidrug efflux pumps in *P. aeruginosa* also correlates with a decrease in virulence (Köhler, T., Join-Lambert, O., Faurisson, F., Kocjancic-Curty, L., Pechère, J.C., and Carbon, C. Abstr. 40<sup>th</sup> Intersci. Conf. Antimicrob. Agents Chemother., abstr. B-22, 2000) perhaps reflecting the negative impact such hyperexpression has on quorum-sensing-dependent expression of several virulence factors in this organism [55].

Given the contribution of the multidrug efflux systems to acquired resistance to an important class of antipseudomonal agent, the fluoroquinolones, and their importance vis-a-vis intrinsic resistance to multiple agents, it seems logical that they be targets for therapeutic intervention. Indeed, a variety

of genetic and inhibitor studies have confirmed the usefulness of pump inactivation in increasing bacterial susceptibility to fluoroquinolones (and other antimicrobials) and preventing emergence of fluoroquinolone resistance. Triple mutants of *P. aeruginosa* deleted for *mexAB-oprM*, *mexCD-oprJ* and *mexEF-oprN* were, for example, markedly fluoroquinolone hypersusceptible, and fluoroquinolone-resistant derivatives of this deletion strain could not be selected in vitro at clinically relevant fluoroquinolone concentrations [153]. Moreover, elimination of the multidrug efflux systems in this organism compromised resistance mediated by target site (i.e. *gyrA*) mutations [153] (Poole, K., unpublished data). This is reminiscent of *E. coli* where loss of the RND-MFP-OMF type AcrAB-TolC multidrug efflux system rendered topoisomerase mutations generally inconsequential as regards clinical fluoroquinolone resistance [154]. Recently, the first examples of inhibitors of the Mex efflux systems of *P. aeruginosa* have been reported [155]. These inhibitors potentiated the activity of a number of antibiotics in vitro [155] and effectively reversed acquired fluoroquinolone resistance attributable to efflux or target site mutations (Lomovskaya, O., Hoshino, K., Ishida, H., Lee, A., Warren, M., Galazzo, J., and Lomovskaya, O. Abstr. 39th Intersci. Conf. Antimicrob. Agents Chemother., abstr. F-1264, 1999).

The suggestion has also been made that naturally-occurring cell-associated compounds are the intended substrates for multidrug efflux systems [156,160,161]. Certainly, cell-derived compounds have been identified as substrates for these efflux systems, including homoserine lactone autoinducers in *P. aeruginosa* (MexAB-OprM) [55,56] and indole (a precursor of tryptophan) in *E. coli* (AcrEF) (Sato, K., Shibayama, K., Horii, T., Arakawa,Y., and Ohta, M. Abstr. 38th Intersci. Conf. Antimicrob. Agents Chemother., abstr. C-126, 1998), although it is far from clear that these are the intended substrates. MexEF-OprN expression in *P. aeruginosa*, for example, is associated with a decrease in pyocyanin production [77], a phenotype also seen in *nalB* strains and attributed to homoserine lactone efflux by MexAB-OprM [55]. Thus, homoserine lactones may be just another in a line of shared substrates for these highly accommodating multidrug efflux systems in *P. aeruginosa*.

### CONCLUDING REMARKS

The physiological basis for the broad substrate specificity of the multidrug efflux systems of *P. aeruginosa* and the need for such multi-substrate capability remain to be elucidated. While the overlap in antimicrobial substrate specificity amongst the various multidrug efflux systems of *P. aeruginosa* suggests a likely overlap vis-a-vis the intended physiological substrate(s), this remains to be determined, as does the identification of the intended substrates. It has been proposed that flexibility vis-a-vis substrate recognition may reflect a role in the export of the byproducts of metabolism [160], the probably vast numbers of which would preclude specific exporter systems. Certainly, the excretion of metabolic waste is likely to be important for overall cell health, although this is an area that has received scant attention to date. The multiplicity of multidrug efflux systems in *P. aeruginosa* may well reflect the well-known metabolic diversity of this organism, its many metabolic pathways necessitating, perhaps, multiple waste excretion systems. Given the hydrophobicity/amphipathicity of most efflux substrates identified to date, these systems may also function to keep membranes clear of cellular products that have improperly targeted the cytoplasmic membrane. What is certain is that the search for intended substrates and, thus, the elucidation of the natural function of these efflux systems represents a major challenge. The observation that virulence is compromised in pump-deficient strains and the likelihood, therefore, that a component of virulence is excreted by these pumps does, however, provide an avenue for identifying a possible natural substrate.

Much remains to be elucidated, including the identification of signals responsible for naturally stimulating efflux gene expression (vs. mutation of regulatory genes). Indeed, this might provide additional clues as to the natural function of specific multidrug efflux systems. The mechanism of RND pump recognition and export of antimicrobials is an area yet to be mined, although clues may be found in the study of a related RND type heavy metal exporter [162]. The structure of these tripartite pumps, including the contribution made by the MFP components and

the nature of OMF association with the RND-MFP components also needs to be studied. Finally, the functional significance of MFP and OMF acylation and the role, if any, that TonB plays in efflux pump activity all need to be addressed. While the conservation of these multidrug efflux systems in a variety of clinically-significant organisms highlights their significance as determinants of antimicrobial resistance in bacteria, it also highlights the probable importance of this family of exporters vis-a-vis some hitherto undetermined housekeeping function(s).

### ACKNOWLEDGEMENTS

Work on the multidrug efflux systems of *P. aeruginosa* in the K.P.'s laboratory is supported by grants from the Canadian Cystic Fibrosis Foundation (CCFF) and the Canadian Bacterial Diseases Network (one of the Networks of Centres of Excellence). K.P. holds a scholarship from the CCFF. R.S. is a CCFF fellow.

### ADDENDUM

The identification of MexR binding sites within the *mexR-mexA* intergenic region has now been published (Evans *et al.* J. Bacteriol. 2001, 183, 807). A recent report highlights the activity of a broad-spectrum inhibitor of the multidrug efflux systems of *P. aeruginosa* that both reduces fluoroquinolone resistance in efflux and target site mutants and prevents the emergence of highly fluoroquinolone-resistant strains in vitro (Lomovskaya *et al.* Antimicrob. Agents Chemother. 2001, 45, 105). Using well-defined efflux mutants, the export of β-lactams by MexAB-OprM, MexCD-OprJ and MexXY-OprM was confirmed, although MexXY-OprM was a much less effective exporter of β-lactams than the other two, and MexAB-OprM was superior to MexCD-OprJ as regards export of penicillins and oxacephems (Masuda *et al.* Antimicrob. Agents Chemother. 2000, 44, 3322). The role of MexCD-OprJ and MexEF-OprN, but not MexXY-OprM in the export of and resistance to triclosan, a fatty acid biosynthesis inhibitor with broad-spectrum antimicrobial activity has also been reported (Chuanchuen *et al.* Antimicrob. Agents Chemother. 2001, 45, 428). In vivo selection of multidrug resistant strains of *P. aeruginosa* hyperexpressing MexCD-OprJ or MexEF-OprN has been reported in an acute pneumonia rat model of infection (Join-Lambert *et al.* Antimicrob. Agents Chemother. 2001, 45, 571). The noted difficulty in selecting MexEF-OprN-hyperexpressing *nfxC* mutants from certain strains of *P. aeruginosa* was recently explained by the observation that several so-called wild type strains carry mutations in the *mexT* gene required for *mexEF-oprN* hyperexpression (Maseda *et al.* FEMS Microbiol. Lett. 2000, 192, 107). Finally, the lone prokaryotic representative of an ABC type multidrug transporter, the LmrA pump of *Lactococcus lactis*, has now been shown to play a role in resistance to clinically relevant antimicrobials (Putman *et al.* Mol. Microbiol. 2000, 36, 772). It is unlikely, however, that it has any real significance vis-a-vis antimicrobial resistance in this organism.

---

### REFERENCES

[1] Nikaido, H. *Curr.Opin.Microbiol.* 1998, 1, 516.

[2] Poole, K. *Antimicrob.Agents Chemother.* 2000, 44, 2233.

[3] Poole, K. *Antimicrob Agents Chemother* 2000, 44, 2595.

[4] Pao, S. S.; Paulsen, I. T.; Saier, M. H., Jr. *Microbiol.Mol.Biol.Rev.* 1998, 62, 1.

[5] van Veen, H. W.; Konings, W. N. *Biochim.Biophys.Acta* 1998, 1365, 31.

[6] Saier, M. H.; Tam, R.; Reizer, A.; Reizer, J. *Mol.Microbiol.* 1994, 11, 841.

[7] Nikaido, H. *Clin.Infect.Dis.* 1998, 27(Suppl 1), S32.

[8] Tseng, T. T.; Gratwick, K. S.; Kollman, J.; Park, D.; Nies, D. H.; Goffeau, A.; Saier, M. H., Jr. *J Mol.Microbiol.Biotechnol.* 1999, 1, 107.

[9] Paulsen, I. T.; Skurray, R. A.; Tam, R.; Saier, M. H., Jr.; Turner, R. J.; Weiner, J. H.; Goldberg, E. B.; Grinius, L. L. *Mol.Microbiol.* 1996, 19, 1175.

[10] Brown, M. H.; Paulsen, I. T.; Skurray, R. A. *Mol.Microbiol.* 1999, 31, 393.

[11] Paulsen, I. T.; Brown, M. H.; Skurray, R. A. *Microbiol.Rev.* 1996, 60, 575.

[12] Dinh, T.; Paulsen, I. T.; Saier, M. H., Jr. *J.Bacteriol.* 1994, 176, 3825.

[13] Johnson, J. M.; Church, G. M. *J.Mol.Biol.* 1999, 287, 695.

[14] Botzenhart, K.; Döring, G. In *Pseudomonas aeruginosa as an opportunistic pathogen;* M. Campa, M. Bendinelli and H. Friedman, Eds.; Plenum Press: New York, 1993, pp 1-18.

[15] Dobbin, C. J.; Soni, R.; Jelihovsky, T.; Bye, P. T. *Aust.N.Z.J Med.* 2000, 30, 288.

[16] Nikaido, H. *Antimicrob.Agents Chemother.* 1989, 33, 1831.

[17] Poole, K.; Krebes, K.; McNally, C.; Neshat, S. *J.Bacteriol.* 1993, 175, 7363.

[18] Germ, M.; Yoshihara, E.; Yoneyama, H.; Nakae, T. *Biochem Biophys Res Commun* 1999, 261, 452.

[19] Li, X.-Z.; Zhang, L.; Poole, K. *J.Antimicrob.Chemother.* 2000, 45, 433.

[20] Stover, C. K.; Pham, X. Q.; Erwin, A. L.; Mizoguchi, S. D.; Warrener, P.; Hickey, M. J.; Brinkman, F. S.; Hufnagle, W. O.; Kowalik, D. J.; Lagrou, M.; Garber, R. L.; Goltry, L.; Tolentino, E.; Westbrook-Wadman, S.; Yuan, Y.; Brody, L. L.; Coulter, S. N.; Folger, K. R.; Kas, A.; Larbig, K.; Lim, R.; Smith, K.; Spencer, D.; Wong, G. K.; Wu, Z.; Paulsen, I. T.; Reizer, J.; Saier, M.H.; Hancock, R.E.W.; Lory, S.; Olson, M.V. *Nature* 2000, 406, 959.

[21] Ma, D.; Cook, D. N.; Alberti, M.; Pon, N. G.; Nikaido, H.; Hearst, J. E. *J.Bacteriol.* 1993, 175, 6299.

[22] Ma, D.; Cook, D. N.; Hearst, J. E.; Nikaido, H. *Trends in Microbiology* 1994, 2, 489.

[23] Fralick, J. A. *J.Bacteriol.* 1996, 178, 5803.

Multidrug Efflux in Pseudomonas aeruginosa

[24] Klein, J. R.; Henrich, B.; Plapp, R. Mol. Gen. Genet. 1991, 230, 230.

[25] Rosenberg, E. Y.; Ma, D.; Nikaido, H. J Bacteriol. 2000, 182, 1754.

[26] Nikaido, H.; Basina, M.; Nguyen, V.; Rosenberg, E. Y. J. Bacteriol. 1998, 180, 4686.

[27] Lacroix, F. J. C.; Cloeckaert, A.; Grépinet, O.; Pinault, C.; Popoff, M. Y.; Waxin, H.; Pardon, P. FEMS Microbiol. Lett. 1996, 135, 161.

[28] Sanchez L; Pan, W.; Vinas, M.; Nikaido, H. J. Bacteriol. 1997, 179, 6855.

[29] Hagman, K. E.; Pan, W.; Spratt, B. G.; Balthazar, J. T.; Judd, R. C.; Shafer, W. M. Microbiology 1995, 141, 611.

[30] Burns, J. L.; Wadsworth, C. D.; Barry, J. J.; Goodall, C. P. Antimicrob. Agents Chemother. 1996, 40, 307.

[31] Moore, R. A.; DeShazer, D.; Reckseidler, S.; Weissman, A.; Woods, D. E. Antimicrob. Agents Chemother. 1999, 43, 465.

[32] Fukumori, F.; Hirayama, H.; Takami, H.; Inoue, A.; Horikoshi, K. Extremophiles. 1998, 2, 395.

[33] Alonso, A.; Martinez, J.L. Antimicrob. Agents Chemother. 2000, 44, 3079.

[34] Droge, M.; Puhler, A.; Selbitschka, W. Mol. Gen. Genet. 2000, 263, 471.

[35] Poole, K.; Heinrichs, D. E.; Neshat, S. Mol. Microbiol. 1993, 10, 529.

[36] Gotoh, N.; Tsujimoto, H.; Poole, K.; Yamagishi, J.-I.; Nishino, T. Antimicrob. Agents Chemother. 1995, 39, 2567.

[37] Li, X.-Z.; Nikaido, H.; Poole, K. Antimicrob. Agents Chemother. 1995, 39, 1948.

[38] Poole, K.; Tetro, K.; Zhao, Q.; Neshat, S.; Heinrichs, D.; Bianco, N. Antimicrob. Agents Chemother. 1996, 40, 2021.

[39] Srikumar, R.; Li, X.-Z.; Poole, K. J. Bacteriol. 1997, 179, 7875.

[40] Köhler, T.; Kok, M.; Michea-Hamzehpour, M.; Plesiat, P.; Gotoh, N.; Nishino, T.; Kocjanici Curty, L.; Pechere, J.-C. Antimicrob. Agents Chemother. 1996, 40, 2288.

[41] Li, X.-Z.; Zhang, L.; Srikumar, R.; Poole, K. Antimicrob. Agents Chemother. 1998, 42, 399.

[42] Srikumar, R.; Kon, T.; Gotoh, N.; Poole, K. Antimicrob. Agents Chemother. 1998, 42, 65.

[43] Masuda, N.; Sakagawa, E.; Ohya, S. Antimicrob. Agents Chemother. 1995, 39, 645.

[44] Masuda, N.; Ohya, S. Antimicrob. Agents Chemother. 1992, 36, 1847.

[45] Rella, M.; Haas, D. Antimicrob. Agents Chemother. 1982, 22, 242.

[46] Hamzehpour, M. M.; Pechere, J.-C.; Plesiat, P.; Köhler, T. Antimicrob. Agents Chemother. 1995, 39, 2392.

[47] Alonso, A.; Campanario, E.; Martinez, J. L. Microbiology 1999, 145, 2857.

[48] Jalal, S.; Wretlind, G.; Gotoh, N.; Wretlind, B. APMIS 1999, 107, 1109.

[49] Cavallo, J. D.; Fabre, R.; Leblanc, F.; Nicolas-Chanoine, M. H.; Thabaut, A. J Antimicrob Chemother 2000, 46, 133.

[50] Ziha-Zarifi, I.; Llanes, C.; Köhler, T.; Pechere, J.-C.; Plesiat, P. Antimicrob. Agents Chemother. 1999, 43, 287.

[51] Srikumar, R.; Poole, K. Clin. Microbiol. Infect. 1999, 5, 5S58.

[52] Schweizer, H. P. Antimicrob. Agents Chemother. 1998, 42, 394.

[53] Li, X.-Z.; Zhang, L.; Poole, K. J. Bacteriol. 1998, 180, 2987.

[54] Li, X.-Z.; Poole, K. Can J Microbiol 1999, 45, 18.

[55] Evans, K.; Passador, L.; Srikumar, R.; Tsang, E.; Nezezon, J.; Poole, K. J. Bacteriol. 1998, 180, 5443.

[56] Pearson, J. P.; Van Delden, C.; Iglewski, B. H. J. Bacteriol 1999, 181, 1203.

[57] Masuda, N.; Gotoh, N.; Ishii, C.; Sakagawa, E.; Ohya, S.; Nishino, T. Antimicrob Agents Chemother 1999, 43, 400.

[58] Nakae, T.; Nakajima, A.; Ono, T.; Saito, K.; Yoneyama, H. Antimicrob Agents Chemother 1999, 43, 1301.

[59] Srikumar, R.; Tsang, E.; Poole, K. J. Antimicrob Chemother 1999, 44, 537.

[60] Kohler, T.; Michea-Hamzehpour, M.; Epp, S. F.; Pechere, J. C. Antimicrob Agents Chemother 1999, 43, 424.

[61] Li, X.-Z.; Ma, D.; Livermore, D. M.; Nikaido, H. Antimicrob. Agents Chemother. 1994, 38, 1742.

[62] Srikumar, R.; Paul, C. J.; Poole, K. J. Bacteriol. 2000, 182, 1410.

[63] Miller, P. F.; Sulavik, M. C. Mol. Microbiol. 1996, 21, 441.

[64] Alekshun, M. N.; Levy, S. B. Trends Microbiol. 1999, 7, 410.

[65] Jalal, S.; Wretlind, B. Microbiol. Drug Resist. 1998, 4, 257.

[66] Saito, K.; Yoneyama, H.; Nakae, T. FEMS Microbiol Lett. 1999, 179 , 67.

[67] Evans, K.; Poole, K. FEMS Microbiol. Lett. 1999, 173, 35.

[68] Poole, K.; Gotoh, N.; Tsujimoto, H.; Zhao, Q.; Wada, A.; Yamasaki, T.; Neshat, S.; Yamagishi, J.-I.; Li, X.-Z.; Nishino, T. Mol. Microbiol. 1996, 21, 713.

[69] Hosaka, M.; Gotoh, N.; Nishino, T. Antimicrob. Agents Chemother. 1995, 39, 1731.

[70] Hirai, K.; Suzue, S.; Irikura, T.; Iyobe, S.; Mitsuhashi, S. Antimicrob. Agents Chemother. 1987, 31, 582.

[71] Masuda, N.; Gotoh, N.; Ohya, S.; Nishino, T. Antimicrob. Agents Chemother. 1996, 40, 909.

[72] Okazaki, T.; Iyobe, S.; Hashimoto, H.; Hirai, K. FEMS Microbiol. Lett. 1991, 79, 31.

[73] Okazaki, T.; Hirai, K. FEMS Microbiol. Lett. 1992, 97, 197.
[74] Shiba, T.; Ishiguro, K.; Takemoto, N.; Koibuchi, H.; Sugimoto, K. J. Bacteriol. 1995, 177, 5872.
[75] Gotoh, N.; Tsujimoto, H.; Tsuda, M.; Okamoto, K.; Nomura, A.; Wada, T.; Nakahashi, M.; Nishino, T. Antimicrob. Agents Chemother. 1998, 42, 1938.
[76] Li, X.-Z.; Barré, N.; Poole, K. J. Antimicrob. Chemother. 2000, 46, 885.
[77] Köhler, T.; Michea-Hamzehpour, M.; Henze, U.; Gotoh, N.; Curty, L. K.; Pechere, J.-C. Mol. Microbiol. 1997, 23, 345.
[78] Fukuda, H.; Hosaka, M.; Hirai, K.; Iyobe, S. Antimicrob. Agents Chemother. 1990, 34, 1757.
[79] Trias, J.; Nikaido, H. Antimicrob Agents Chemother 1990, 34, 52.
[80] Ballestero, S.; Fernandez-Rodriguez, A.; Villaverde, R.; Escobar, H.; Perez-Diaz, J. C.; Baquero, F. J Antimicrob Chemother 1996, 38, 39.
[81] Köhler, T.; Epp, S. F.; Curty, L. K.; Pechére, J.-C. J Bacteriol 1999, 181, 6300.
[82] Ochs, M. M.; McCusker, M. P.; Bains, M.; Hancock, R. E. Antimicrob Agents Chemother 1999, 43, 1085.
[83] Mine, T.; Morita, Y.; Kataoka, A.; Mitzushima, T.; Tsuchiya, T. Antimicrob. Agents Chemother. 1999, 43, 415.
[84] Westbrock-Wadman, S.; Sherman, D. R.; Hickey, M. J.; Coulter, S. N.; Zhu, Y. Q.; Warrener, P.; Nguyen, L. Y.; Shawar, R. M.; Folger, K. R.; Stover, C. K. Antimicrob. Agents Chemother. 1999, 43, 2975.
[85] Aires, J. R.; Köhler, T.; Nikaido, H.; Plesiat, P. Antimicrob. Agents Chemother. 1999, 43, 2624.
[86] Masuda, N.; Sakagawa, E.; Ohya, S.; Gotoh, N.; Tsujimoto, H.; Nishino, T. Antimicrob Agents Chemother 2000, 44, 2242.
[87] Thanassi, D. G.; Suh, G. S.; Nikaido, H. J. Bacteriol. 1995, 177, 998.
[88] Koronakis, V.; Sharff, A.; Koronakis, E.; Luisi, B.; Hughes, C. Nature 2000, 405, 914.
[89] Zgurskaya, H. I.; Nikaido, H. Mol. Microbiol. 2000, 37, 219.
[90] Zgurskaya, H. I.; Nikaido, H. Proc. Natl. Acad. Sci. U S A 1999, 96, 7190.
[91] Guan, L.; Ehrmann, M.; Yoneyama, H.; Nakae, T. J. Biol. Chem. 1999, 274, 10517.
[92] Gotoh, N.; Kusumi, T.; Tsujimoto, H.; Wada, T.; Nishino, T. FEBS Lett. 1999, 458, 32.
[93] Zgurskaya, H. I.; Nikaido, H. J Bacteriol. 2000, 182, 4264.
[94] Yoneyama, H.; Ocaktan, A.; Gotoh, N.; Nishino, T.; Nakae, T. Biochem. Biophys. Res. Commun. 1998, 244, 898.
[95] Gotoh, N.; Tsujimoto, H.; Nomura, A.; Okamoto, K.; Tsuda, M.; Nishino, T. FEMS Microbiol. Lett. 1998, 165, 21.

[96] Maseda, H.; Yoneyama, H.; Nakae, T. Antimicrob. Agents Chemother. 2000, 44, 658.
[97] Ocaktan, A.; Yoneyama, H.; Nakae, T. J. Biol. Chem. 1997, 272, 21964.
[98] Nikaido, H. J. Bacteriol. 1996, 178, 5853.
[99] Kieboom, J.; Dennis, J. J.; de Bont, J. A.; Zylstra, G. J. J. Biol. Chem. 1998, 273, 85.
[100] Kim, K.; Lee, S.; Lee, K.; Lim, D. J. Bacteriol. 1998, 180, 3692.
[101] Ramos, J. L.; Duque, E.; Godoy, P.; Segura, A. J. Bacteriol. 1998, 180, 3323.
[102] Mosqueda, G.; Ramos, J. L. J. Bacteriol. 2000, 182, 937.
[103] White, D. G.; Goldman, J. D.; Demple, B.; Levy, S. B. J. Bacteriol. 1997, 179, 6122.
[104] Aono, R. Extremophiles 1998, 2, 239.
[105] Aono, R.; Tsukagoshi, N.; Yamamoto, M. J Bacteriol 1998, 180, 938.
[106] Weber, F. J.; de Bont, J. A. Biochim. Biophys. Acta 1996, 1286, 225.
[107] Martin, C.; Berridge, G.; Higgins, C. F.; Mistry, P.; Charlton, P.; Callaghan, R. Mol. Pharmacol. 2000, 58, 624.
[108] Putman, M.; Koole, L. A.; van Veen, H. W.; Konings, W. N. Biochemistry 1999, 38, 13900.
[109] Ahmed, M.; Borsch, C. M.; Taylor, S. S.; Vasquez-Laslop, N.; Neyfakh, A. A. J. Biol. Chem. 1994, 269, 28506.
[110] Zheleznova, E. E.; Markham, P. N.; Neyfakh, A. A.; Brennan, R. G. Cell 1999, 96, 353.
[111] Zheleznova, E. E.; Markham, P.; Edgar, R.; Bibi, E.; Neyfakh, A. A.; Brennan, R. G. Trends Biochem. Sci. 2000, 25, 39.
[112] Vazquez-Laslop, N.; Zheleznova, E. E.; Markham, P. N.; Brennan, R. G.; Neyfakh, A. A. Biochem. Soc. Trans. 2000, 28, 517.
[113] Wong, K. K. Y.; Hancock, R. E. W. J. Bacteriol. 2000, 182, 2402.
[114] Benz, R.; Maier, E.; Gentshev, I. J. Bacteriol. 1993, 178, 5803.
[115] Li, X.-Z.; Poole, K. J. Bacteriol. 2000, 183, 12.
[116] Wong, K. K. Y.; Brinkman, F. S.; Benz, R. S.; Hancock, R. E. W. J. Bacteriol. 2000, 183, 367.
[117] Nakajima, A.; Sugimoto, Y.; Yoneyama, H.; Nakae, T. J. Biol. Chem. 2000, 275, 30064.
[118] Zhao, Q.; Li, X.-Z.; Srikumar, R.; Poole, K. Antimicrob. Agents Chemother. 1998, 42, 1682.
[119] Hwang, J.; Zhong, X.; Tai, P. C. J. Bacteriol. 1997, 179, 6264.
[120] Wandersman, C.; Delepelaire, P. Proc. Natl. Acad. Sci. USA 1990, 87, 4776.

[121] Thanabalu, T.; Koronakis, E.; Hughes, C.; Koronakis, V. EMBO J 1998, 17, 6487.

[122] Braun, V.; Wu, H. C. Bacterial Cell Wall; Elsevier Science B.V.: Amsterdam, 1994, pp 319-341.

[123] Zhao, Q.; Li, X.-Z.; Mistry, A.; Srikumar, R.; Zhang, L.; Lomovskaya, O.; Poole, K. Antimicrob. Agents Chemother. 1998, 42, 2225.

[124] Postle, K. J. Bioenerget. Biomembr. 1993, 25, 591.

[125] Kadner, R. J. Mol. Microbiol. 1990, 4, 2027.

[126] Heller, K. J.; Kadner, R. J.; Günther, K. Gene 1988, 64, 147.

[127] Cadieux, N.; Kadner, R. J. Proc. Natl. Acad. Sci. USA 1999, 96, 10673.

[128] Moeck, G. S.; Coulton, J. W.; Postle, K. J. Biol. Chem., 1997, 272, 28391.

[129] Poole, K.; Zhao, Q.; Neshat, S.; Heinrichs, D. E.; Dean, C. R. Microbiology 1996, 142, 1449.

[130] Takase, H.; Nitanai, H.; Hoshino, K.; Otani, T. Infect. Immun. 2000, 68, 4498.

[131] Wong, K. K. Y.; Poole, K.; Gotoh, N.; Hancock, R. E. W. Antimicrob. Agents Chemother. 1997, 41, 2009.

[132] Yoneyama, H.; Ocaktan, A.; Tsuda, M.; Nakae, T. Biochem. Biophys. Res. Commun. 1997, 233, 611.

[133] Tsuda, M.; Miyazaki, H.; Nakazawa, T. J. Bacteriol. 1995, 177, 423.

[134] Yoneyama, H.; Maseda, H.; Kamiguchi, H.; Nakae, T. J. Biol. Chem. 2000, 275, 4628.

[135] Zgurskaya, H. I.; Nikaido, H. J. Mol. Biol. 1999, 285, 409.

[136] Celesk, R. A.; Robillard, R. A. Antimicrob. Agents Chemother. 1989, 33, 1921.

[137] Radberg, G.; Nilsson, L. E.; Svensson, S. Antimicrob. Agents Chemother. 1990, 34, 2142.

[138] Legakis, N. J.; Tzouvelekis, L. S.; Makris, A.; Kotsifaki, H. Antimicrob. Agents Chemother. 1989, 33, 124.

[139] Robillard, R. A.; Scarpa, A. L. Antimicrob. Agents Chemother. 1988, 32, 535.

[140] Lei, Y.; Sato, K.; Nakae, T. Biochem. Biophys. Res. Commun. 1991, 178, 1043.

[141] Piddock, L. J. V.; Hall, M. C.; Bellido, F.; Bains, M.; Hancock, R. E. W. Antimicrob. Agents Chemother. 1992, 36, 1057.

[142] Yoshida, T.; Muratani, T.; Iyobe, S.; Mitsuhashi, s. Antimicrob. Agents Chemother. 1994, 38, 1466.

[143] Koehler, T.; Pechere, J.-C. In The Quinolones; V.T. Andriole, Ed.; Academic Press: San Diego, 1998, pp 117-142.

[144] Jakics, E. B.; Iyobe, S.; Hirai, K.; Fukuda, H.; Hashimoto, H. Antimicrob. Agents Chemother. 1992, 36, 2562.

[145] Yoshida, H.; Nakamura, M.; Bogaki, M.; Nakamura, S. Antimicrob. Agents Chemother. 1990, 34, 1273.

[146] Iyobe, S.; Hirai, K.; Hashimoto, H. Chemotherapy (Basel) 1991, 44, 209.

[147] Koehler, T.; Michea-Hamzehpour, M.; Plesiat, P.; Kahr, A.-L.; Pechere, J.-C. Antimicrob. Agents Chemother. 1997, 41, 2543.

[148] Jalal, S.; Ciofu, O.; Hoiby, N.; Gotoh, N.; Wretlind, B. Antimicrob. Agents Chemother. 2000, 44, 710.

[149] Fukuda, H.; Hosaka, M.; Iyobe, S.; Gotoh, N.; Nishino, T.; Hirai, K. Antimicrob. Agents Chemother. 1995, 39, 790.

[150] Pumbwe, L.; Piddock, L. J. Antimicrob Agents Chemother 2000, 44 , 2861.

[151] Brooun, A.; Liu, S.; Lewis, K. Antimicrob Agents Chemother 2000, 44, 640.

[152] Maira-Litran, T.; Allison, D. G.; Gilbert, P. J. Antimicrob Chemother 2000, 45, 789.

[153] Lomovskaya, O.; Lee, A.; Hoshino, K.; Ishida, H.; Mistry, A.; Warren, M. S.; Boyer, E.; Chamberland, S.; Lee, V. J. Antimicrob. Agents Chemother. 1999, 43, 1340.

[154] Oethinger, M.; Kern, W. V.; Jellen-Ritter, A. S.; McMurry, L. M.; Levy, S. B. Antimicrob. Agents Chemother. 2000, 44, 10.

[155] Renau, T. E.; Leger, R.; Flamme, E. M.; Sangalang, J.; She, M. W.; Yen, R.; Gannon, C. L.; Griffith, D.; Chamberland, S.; Lomovskaya, O.; Hecker, S. J.; Lee, V. J.; Ohta, T.; Nakayama, K. J. Med. Chem. 1999, 42, 4928.

[156] Neyfakh, A. A. Trends Microbiol. 1997, 5, 309.

[157] Ma, D.; Cook, D. N.; Alberti, M.; Pon, N. G.; Nikaido, H.; Hearst, J. E. Mol. Microbiol. 1995, 16, 45.

[158] Thanassi, D. G.; Cheng, L. W.; Nikaido, H. J. Bacteriol. 1997, 179, 2512.

[159] Shafer, W. M.; Balthazar, J. T.; Hagman, K. E.; Morse, S. A. Microbiology 1995, 141, 907.

[160] Poole, K. J. Antimicrob. Chemother. 1994, 34, 453.

[161] Woolridge, D. P.; Vasquez-Laslop, N.; Markham, P. N.; Chevalier, M. S.; Neyfakh, A. A. J. Biol. Chem. 1997, 272, 8864.

[162] Goldberg, M.; Pribyl, T.; Juhnke, S.; Nies, D.H. J. Biol. Chem. 1999, 274, 26065.
