
CELLULAR
SIGNALLING

www.elsevier.com/locate/cellsig

Cellular Signalling 14 (2002) 477–492

Review article

TNF receptor subtype signalling:
Differences and cellular consequences

David J. MacEwan*

Department of Biomedical Sciences, Institute of Medical Sciences, University of Aberdeen, Aberdeen, AB25 2ZD, UK

Received 26 August 2001; accepted 3 October 2001

Abstract

Tumour necrosis factor-α (TNFα) is a multifunctional cytokine belonging to a family of ligands with an associated family of receptor proteins. The pleiotropic actions of TNF range from proliferative responses such as cell growth and differentiation, to inflammatory effects and the mediation of immune responses, to destructive cellular outcomes such as apoptotic and necrotic cell death mechanisms. Activated TNF receptors mediate the association of distinct adaptor proteins that regulate a variety of signalling processes including kinase or phosphatase activation, lipase stimulation, and protease induction. Moreover, the cytokine regulates the activities of transcription factors, heterotrimeric or monomeric G-proteins and calcium ion homeostasis in order to orchestrate its cellular functions. This review addresses the structural basis of TNF signalling, the pathways employed with their cellular consequences, and focuses on the specific role played by each of the two TNF receptor isoforms, TNFR1 and TNFR2. © 2002 Elsevier Science Inc. All rights reserved.

Keywords: Cytokine; Receptor; Signal transduction; Lipase; Kinase; Phosphatase; Protease; Calcium homeostasis; G-protein; Apoptosis; Necrosis; Proliferation; Inflammation; Immune response

1. Introduction

Tumour necrosis factor (TNF) signalling is a very complex process indeed, of that there can be no doubt. Classical receptors such as ion channels respond to their extracellular stimulus by increasing the probability of opening and allowing the flux of ions which is chiefly governed by the membrane potential and presence of cofactors in its environment which may modulate channel activity [1]. G-protein-coupled receptors are more intricate in that they acquire intracellular proteins that can have multiple signalling functions [2]. TNF receptors themselves have no metabolic capability and must bind intracellular proteins to control the signalling of its target cell's biochemistry. In little over a decade, since the original identification of TNF receptor proteins [3–15] and their subsequent cloning [16–24], modern molecular techniques have revealed a bewildering array of potential proteins that may interact with the TNF receptors, and a staggering arrangement of signalling pathway activation and interplay. To understand the complexity of the proteins and pathways that are implicated, we must understand how these new entities are elucidated. The use of techniques such as yeast two hybrid screening allows the identification of potential proteins that may interact with a target protein used as bait. Identification of an interacting protein often leads to its molecular cloning then its ability to modulate a downstream signalling pathway based on the transfection of its cDNA into cultured mammalian cells. Superexpression of the cloned protein often leads to dramatic cellular consequences independent of ligand stimulation and a mutated version of the cloned protein may be introduced into the cultured cells to verify the purpose of the protein in inhibiting the ligand-dependent signalling end point. Ultimately, the use of gene recombination in knockout mice may provide an insight into the physiological role of the newly identified protein when it is totally absent from all cells in the whole animal. These techniques are extremely powerful, although quite obviously not foolproof, in allowing the elucidation of the possible proteins that interact with TNF receptors and the signalling pathways controlled by TNF ligand.

* Tel.: +44-1224-273154; fax: +44-1224-273019.
E-mail address: david.macewan@abdn.ac.uk (D.J. MacEwan).

0898-6568/01/$ – see front matter © 2002 Elsevier Science Inc. All rights reserved.
PII: S0898-6568(01)00262-5

2. TNF ligand

TNF was the unknown factor that was named after its ability to regress tumour masses in patients whose tumour has become infected with a bacterial infection [25]. TNF was isolated in 1984 and since has been found to be a pleiotropic cytokine produced primarily by activated macrophages expressed as a 26-kDa transmembrane protein that can be cleaved to release a 17-kDa soluble TNF form [26]. TNF is also referred to as TNFα, cachectin, or differentiation-inducing factor (DIF) [27]. It belongs to a family of nearly 20 different homologues including TNFβ/lymphotoxinα, lymphotoxinβ, CD40 ligand, and Fas ligand [28]. TNF as well as causing necrotic cell death may also cause apoptotic cell death, cellular proliferation, differentiation, inflammation, tumourigenesis, and viral replication. TNF's primary role is in the regulation of immune cells, but is known to be heavily involved in pathogenic disorders such as rheumatoid arthritis, asthma, septic shock, irritable bowel disorder, haemorrhagic fever, and cachexia, the tissue wasting disorder often seen in cancer patients [29].

molecule is the silencer of DD protein SODD, which binds to the DD in TNFR1 and prevents other DD-interacting proteins from taking hold [35]. SODD dissociates from the TNFR1 DD upon TNF stimulation allowing other activator proteins to access the DD receptor module. TNFR1 also contains an intracellular sequence at Positions 309–319 which are known to bind the adaptor protein FAN, a key stimulatory element in the activation of neutral sphingomyelinase (SMase) [36,37] which catalyses the degradation of sphingolipids into smaller ceramide-containing molecules which are key signalling intermediates [38]. Similarly, a stress-responsive 44-kDa protein highly expressed in brain and reproductive organs, BRE, was found to specifically interact with the juxtamembrane region of TNFR1 [39]. BRE, like SODD, acts to inhibit and dampen TNFR1-stimulated signalling. Both TNFR1 and TNFR2 possess sequences that are capable of binding intracellular adaptor proteins that link TNF receptor stimulation to activation of many signalling processes. These TNF receptor-associating factors (TRAFs) and adaptors are what transduce the TNF signal from the biochemically inert receptors to dramatic changes of the signalling molecules within target cells.

3. TNF receptors

TNF receptors are an ever increasing superfamily of proteins that consist of at least 41 members characterised by their extracellular sequence homology, and include TNF receptors, Fas, CD40, the low-affinity nerve growth factor receptor, TRAIL receptors, RANK and death and decoy receptors [28,30]. Many of these members go by multiple names and are activated by specific ligands, however, TNF only has the ability to bind two of these receptors. TNF ligand achieves all its different cellular and pathological effects by its binding to TNFR1 (also known as p55TNFR, p60, CD120a, TNFRSF1a) and TNFR2 (also known as p75TNFR, p80, CD120b, TNFRSF1b). These TNF receptor subtypes are single transmembrane glycoproteins with 28% homology mostly in their extracellular domain with both containing four tandemly repeated cysteine-rich motifs. Their intracellular sequences are largely unrelated with almost no homology between each other, and early work suggested delineation of their signalling functions [31]. They contain several motifs with known functional significance. Both TNFR1 and TNFR2 contain an extracellular pre-ligand-binding assembly domain (PLAD) (distinct from ligand binding regions) that pre-complexes receptors and encourages them to trimerize particularly upon activation by TNF ligand [32], which itself is presented in a trimerized conformation [29,33]. TNFR1 contains a so-called death domain (DD) region of approximately 80 amino acids in length towards the carboxyl-end of the receptor [34]. The DD is present on a number of associating proteins and related molecules that are primarily involved in signalling for cell death. One such

4. TRAFs and adaptor proteins

The role of TRAF proteins and their selective interaction with TNF receptors has been expertly reviewed in a recent article in this journal [40]. TRAF molecules all contain a RING finger and zinc finger motifs in their N-terminal, with their C-terminal regions possessing a TRAF domain sequence, and to date, six mammalian TRAFs have been identified. The first TRAFs to be uncovered, TRAF1 and TRAF2, were discovered by their ability to directly interact with the cytoplasmic domain of TNFR2 [41]. Work by the same group also identified the apoptotic adaptor proteins c-IAP1 and c-IAP2 that bind to the TNF receptor via a TRAF1/TRAF2 heterocomplex [42]. Since then, it is now thought that mostly TRAF2 interacts with TNFR2 directly, with TRAF1 interacting indirectly and TRAF3 also able to associate. TRAF2 is recruited to TNFR1 indirectly through a specific interaction with the protein TNF receptor-associated death domain (TRADD), a 34-kDa cytosolic adaptor protein that directly binds to TNFR1 through its own DD sequence [43]. TRADD recruits the downstream signalling adaptor molecules Fas-associated death domain (FADD) and receptor interacting protein (RIP). RIP originally identified as a Fas-associating molecule [44] also interacts with TNF receptors [45,46]. RIP contains a kinase sequence, but its role as a kinase enzyme is unclear at present. FADD contains a death effector domain (DED) sequence [47], which interacts with the DED domain in caspase-8 and a number of other DED-containing molecules that regulate cell death mechanisms [48]. Another DD-containing molecule RAIDD is recruited to TNFR1 and interacts with RIP and caspase-2 to allow its activation. RIP and FADD are
also thought capable under certain conditions to be able to indirectly bind to TNFR2 via TRAF2 [49].

TRAF2 is also capable of interacting with downstream signalling molecules such as NFκB-inducing kinase (NIK), which is a member of the serine/threonine mitogen-activated protein kinase (MAPK) kinase (MEKK) family [46, 49, 50]. NIK phosphorylates its target at serine 176, the enzyme named inhibitor of κB (IκB) kinase (IKK) which is implicated in the activation of NFκB transcription factor involved in transcriptional responses to stress and anti-apoptotic cellular action [51]. NIK is also capable of interacting with TRAFs 1, 3, 5, and 6 [40]. One of the genes under the transcriptional control of NFκB is the cellular inhibitor of apoptosis protein 2 (c-IAP2) which binds to TRAF2 and is capable of blocking caspase-8 activation and apoptosis. Similarly, A20 is an 80-kDa inducible protein that binds TRAF2 and is anti-apoptotic [52]. RIP and NIK are not the only kinases to interact with TRAF2, apoptosis-stimulating kinase (ASK1) [53], germinal centre kinase (GCK) [54, 55], and MEKK1 [56], have all been implicated in the activation of p38MAPK and c-Jun N-terminal kinase (JNK) stress kinases, as well as AP-1 and NFκB transcriptional activation processes.

The activation of the MAPK family of kinase enzymes can be achieved by TNF receptor interaction with the factor associated with neutral SMase activation (FAN) adaptor protein [36, 37, 57]. FAN is responsible for neutral SMase-mediated generation of ceramide-containing sphingolipids [36], which are capable of activating ceramide-activated protein kinases (CAPK) [58, 59] [also known as kinase suppressor of ras (KSR) [60]] which activates Raf kinase [61], an upstream activator of the MEK and MAPK serine/threonine kinase family [62]. Interestingly, TNF receptors have been found to interact with the Grb2 adaptor and son of sevenless (SOS) exchange factor [37, 63]. Activated Grb2 binds through its SH3 domain to a motif within the TNF receptor, allowing this activated complex to stimulate c-Raf-1 kinase. Another recently identified protein that interacts with FADD through a DED domain is the FLICE-like inhibitory protein (FLIP [64, 65]), which blocks caspase-8 recruitment and activation. FLIP has been identified as being able to interact with TRAFs and RIP to switch signalling pathways through more anti-apoptotic pathways such as NFκB and Raf-1 kinase, resulting in marked MAPK activation [66]. The DD within TNFR1 is also able to bind MADD adaptor protein, which contains a DD sequence. MADD activates its downstream kinase MAPK when over-expressed in cultured mammalian cells [66–68].

processes within cells initiating its programmed suicide [70]. Cysteine-aspartate-directed proteases (caspases) are a group of at least 14 different enzyme forms, which play a key role in the induction of most types of apoptotic cell death. As mentioned above, caspase-8 is part of the TNFR1 death-inducing signalling complex (DISC) by virtue of its DED domain allowing its interaction with the DD-containing TRADD/FADD machinery [29]. Caspases-2, -8, -9, and -10 are known as apoptotic initiator caspases that lie upstream of executioner procaspases-3, -6, and -7, which exist in a latent form until activated by its initiator through a process of cleavage, oligomerisation, and autoactivation [70]. Caspase-1 is the enzyme that processes the proteolytic maturation of pro-interleukin-1β, and together with caspase-11, these enzymes are predominantly responsible for cytokine processing. Less is known about the function of caspases-4, -5, -12, -13, and -14. Evidence suggests a crucial role of caspases-3, -8, and -10 in TNF-induced apoptotic mechanisms with some viral products, such as CrmA cowpox modifier protein, blocking specific caspase forms and showing their absolute importance in TNF-mediated apoptotic [43, 71], but not necrotic [72, 73] cell death. Much of the importance of caspase members in cell death has arisen from the use of cell-permeable pharmacological inhibitors of caspases (such as zVAD-FMK) which have shown dramatic effects at blocking ligand-induced cell death [74]. As with all pharmacological data, however, the claimed specificity of these agents does not match up to their actual inhibitions at the concentrations in which they are used experimentally [75], and as such, much of the precise role of each caspase family member remains uncertain. Transgenic mice deficient in caspases-1, -2, -3, and -9 showed TNF-induced apoptotic death that was largely unaffected, indicating multiple caspase pathways that may be commissioned by activated TNF receptors in each particular cell type [76–79]. Much of the present thinking implicates only TNFR1 in its ability to activate caspase proteases [80]. Recent evidence investigating the variable ability of RIP adaptor protein [81] (itself a target for caspase-mediated degradation [82]) to associate with TRAF2 and TNFR1 or TNFR2 suggests that cellular conditions may favour the switching of TNFR2 between anti-apoptotic NFκB activation signalling and death induction through caspase mechanisms [49, 83]. This new mechanism by which non-DD-containing receptors such as TNFR2 can signal for cell death is an interesting and understudied area of TNFR signalling research (Fig. 1).

Caspases initiate cell suicide by the controlled destruction of the cell’s own repair mechanisms [84]. Mitochondria, the powerhouses of cells, sense apoptotic signals via a family of Bcl-2 proteins that either inhibit (Bcl-2, Bcl-xL) or promote (Bax, Bad, Bak, Bik, Bid) apoptosis [85]. Caspase-8-mediated cleavage of Bid produces a truncated form (tBid) that translocates from the cytosol to the mitochondria. This caspase-mediated Bid activation step leads to reduced mitochondrial membrane potential and the release of

5. Caspases

TNF receptors are capable of many cellular responses but are chiefly associated with the induction of cell death mechanisms [69]. These so-called death receptors are named as such because of their ability to stimulate destructive
isotype, which stimulated downstream acidic membrane-bound SMase activity. Stimulation of PC-PLC activity degrades phosphatidylcholine into choline (a molecule with no known signalling function) and diacylglycerol, the physiological activator of most protein kinase C (PKC) isoforms. Since choline production was unable to alter intracellular Ca²⁺ concentrations, it was assumed that the Ca²⁺-independent forms of PKC would be more selectively activated by PC-PLC action [111]. Less extensive reporting has also suggested that TNF ligand could stimulate phospholipase D (PLD), creating phosphatic acid [112–114], which may be converted to DAG by phosphatic acid phosphohydrolase. Others have implied TNF-stimulated phosphatidylinositol-specific phospholipase C (PI-PLC, generating inositol-1,4,5-trisphosphate and diacylglycerol second messengers) activity [115], with others indicating an inhibitory role of TNF on inositol lipid signalling [116–120]. The functional significance of these findings have not stood the test of time, however, their importance in TNF signalling may be critical in certain cell types or cellular conditions as yet unappreciated.

Much interest has centred on the ability of TNF to stimulate SMase of which there are two types, neutral and acidic [38]. These SMase enzymes stimulate the cleavage of membrane sphingolipids into ceramide, which can then be converted into other ceramide-containing lipids such as sphingosine and sphingosine-1-phosphate (S-1-P) [121]. TNF was the first stimulus found to induce ceramide generation in HL-60, A-431, and in a cell-free system [58, 59, 122, 123]. The proximal half of TNFR1 was found to be responsible for the stimulation of membrane-associated neutral SMase [108, 124] partially via ceramide interconverted from DAG [122, 125–128], but also via the more recently discovered TNFR1 adaptor protein FAN, which stimulates neutral SMase more directly [37]. Profiles of ceramide generation and its associated signalling may be rapid, intermediary, or long-term dependent on the cell type stimulated or the metabolic processing enzymes present [129, 130]. Exogenous ceramide is a highly specific yet destructive chemical in most cells with rapid stimulation of apoptotic cell death processes. These apoptosis-signalling processes include the sequential activation of Bad [131], then activation of CAPK which stimulates raf kinase [61] and MAPK activity [132, 133]. TNF-induced ceramide production also stimulates the stress-activated kinase JNK in rat glomerular mesangial cells [134]. Sphingolipids are able to modulate PKC isoforms. For example, human leukaemia cells treated with ceramide or TNF caused the rapid translocation to the cytosol of PKC-δ and PKC-ε [135]. Similarly, in L929 rat fibroblasts, ceramide and TNF blocked PKC-α activity and translocation to the plasma membrane [136, 137]. TNF-stimulated ceramide generation and apoptosis in HL-60 leukaemia cells was found to be crucially dependent on PKC-β [138], while ceramide has also been found to bind directly to the atypical isoform PKC-ζ which can activate ras, raf, MEK, MAPK,

cytochrome c, which binds the adaptor protein Apaf-1, a 130-kDa protein that contains an N-terminal caspase recruitment domain (CARD). With dATP/ATP and cytochrome c acting as cofactors, Apaf-1 self-oligomerises and binds procaspase-9 to form the apoptosome complex, which activates the executioner caspases-3 and -7 then other caspases to mediate apoptotic proteolysis of key repair and housekeeping proteins [86]. For example, caspase-dependent proteolysis cleaves and activates PKC-δ [87], and inactivates polyadenosine ribosyl polymerase (PARP) repair enzyme as well as the proteolytic activation of caspase-activated DNAses (CADs) that destroy the genome by excising genes [88] leading to the characteristic DNA fragmentation and laddering associated with apoptotic death [89].

Caspases are not the only type of protease enzyme known to be implicated in cell death mechanisms. Some recent work on the Ca²⁺-sensitive calpain group of proteases and cathepsins and granzymes has shown that these destructive enzymes are brought into play during cell death [90–92]. They may be more important in forms of death associated with Ca²⁺ overload and excitotoxicity [93–95], but as such, their role in general TNF signalling has been implicated [96, 97], and it may be that their role in TNF-induced cell death mechanisms is understudied.

## 6. Lipases

Some of the earliest TNF signalling work performed revealed the modulation of several lipase activities [98–104]. Some early detailed reports of a signalling enzyme found to be activated by TNF receptors was phosphatidylcholine-specific phospholipase C (PC-PLC) [105–110]. TNFR1 activation of PC-PLC occurs through a defined portion of the intracellular face of the receptor
and NFκB transcription [139–143]. TNF-stimulated sphingolipids are also capable of being converted into S-1-P, which has intracellular actions including motivating rises in \[Ca^{2+}\] from stores and stimulating MAPK activity [144]. As well as its intracellular actions, S-1-P is able to diffuse from the cell and act in an autocrine manner by stimulating endothelial differentiation gene (EDG) cell surface receptors, which are a family of stimulatory G-protein-coupled receptors linked to stimulation of Gq/11α, Giα, G12/13α heterotrimeric G-proteins as well as rho monomeric G-protein regulation [144].

Another lipase strongly stimulated by TNF receptors is the 110-kDa hormone-sensitive, Ca²⁺-dependent cytosolic phospholipase A₂ (cPLA₂) [145]. cPLA₂ is responsible for the liberation of arachidonic acid from the sn-2 position of mainly phosphatidylcholine. The arachidonic acid again acts in an autocrine fashion and within its own cell of generation, to be converted into prostaglandins and leukotrienes to stimulate eicosanoid-sensing receptors. Moreover, these eicosanoids are responsible for the generation of oxygen radical-containing lipids and reactive oxygen species (ROS) that disrupt mitochondrial integrity and set in motion cell death mechanisms indicated above [146]. Protective cellular enzymes such as superoxide dismutase counteract these disruptive ROS molecules [147, 148]. TNF receptor stimulation of MAPK and PKC isoforms leads to the phosphorylation and activation of cPLA₂ at specific residues [149, 150] (most notably serine 505 in the case of MAPK [151]) resulting in a rapid liberation of arachidonic acid. Cytokine receptor activation of cPLA₂ through p38MAPK has also been implicated [152]. p38MAPK phosphorylation of cPLA₂ occurs at serine residue 727 and leads to activation of the lipase [153, 154], however, it may be that p38MAPK phosphorylation and activation of cPLA₂ have only really been observed in platelets leading to their priming. Whether TNF-stimulated p38MAPK-mediated activation of cPLA₂ occurs in other cell types is not clear and this may represent a cell type-specific form of TNFR signalling. A secondary mechanism of TNF-stimulated cPLA₂ gene induction also occurs, leading to the increased expression of cPLA₂ protein and activity which is sensitive to inhibition by protein synthesis blockers and glucocorticoids [155]. It has been shown in several cell types that cPLA₂ protein and activity is crucial for TNF-mediated cell death [112, 156–163]. The mechanism of cPLA₂ stimulation by TNF is thought to be TNFR1-specific as only this receptor has shown the ability to activate the kinases involved in its phosphorylation [68, 164–167] with TNFR2 having a role in the regulation of cPLA₂ expression [168]. More recently, cPLA₂ has been shown to be the target of caspase proteases, which cleave cPLA₂ into fragments [169–173]. The relative activity of these cPLA₂ fragments is not clear with the existence of conflicting reports, however, this novel secondary mechanism of lipase modulation by TNF signalling provides a potential role of cPLA₂ for any receptors capable of caspase proteolysis.

### 7. Kinases and phosphatases

Most signalling mechanisms relaying the transduction of messages from outside a cell to its cytosol and nucleus are based on the tight regulation of protein phosphorylation and dephosphorylation cascades. Through activation of PC-PLC, TNF receptors are capable of diacylglycerol generation and subsequent PKC activation, as some of the early signalling research on TNF showed [108, 174–178]. There exist at least 12 forms of PKC with differential activation characteristics and distinct tissue distribution, and some of these isoforms have been shown to be TNFR-stimulated. For example, PKC-α activation by TNF can occur in L929 fibroblasts through an indirect activation process involving ceramide production [137]. A variant of the HL60 human myeloid leukaemia cell line, HL525, which is deficient in PKC-β, is resistant to TNF-induced apoptosis which can be reinstated by reestablishing PKC-β protein expression [138]. PKC-δ is a substrate for TNF-stimulated caspase-dependent cleavage [84] and has also been shown to cause the serine phosphorylation of TNFR1 protein itself [179]. The atypical PKC-ζ, -λ, and -η have also been shown to be regulated by more complex TNF-stimulated mechanisms that may involve lipid intermediates [139, 180, 181] and, as mentioned above, in particular, PKC-ζ directly binds TNF-generated ceramide to cause the subsequent activation of ras, raf, MEK, MAPK, and NFκB.

Much work has been concerned with the activation by TNF receptors of the extracellular signal-regulated protein kinase (ERK) superfamily of kinases, responsible for much of a cell’s reaction to a variety of mitogenic stimuli and stress responses [182]. These enzymes are characterised by their activation process of dual phosphorylation on threonine and tyrosine residues in their sequences, with the motif Thr-X-Tyr, where X can be Glu for MAPK members, Gly for p38MAPK, or Pro for the JNK family of stress-responsive kinases [166]. All three of these ERK families are readily stimulated by TNF. Upstream kinases (MEKs) control their activation, which are themselves under the control of MEKKs. They have a range of known substrates activated in an ERK family-selective manner, including transcription factors and downstream kinases [183]. The first members of the ERK family to be identified, p42 and p44MAPK, are transiently activated by TNF through MEK-1 and MEK-2 phosphorylation [184, 185], and one report in fibroblasts from TNFR-deficient mice indicates that both receptor subtypes are capable of MAPK stimulation [186] but more recent work has shown that MAPK activation occurs through the TNFR1 receptor only [165]. Several pharmacological inhibitors of these MEKs, such as PD98059 and U0126, have helped reveal a role for p42 and p44MAPK activation in TNF modulation of cell death [187–189].

The p38MAPK and JNKs are termed stress kinases as they are transiently activated by a range of stress stimuli including cytokines, heat- or osmotic-shock, and UV
irradiation. Upstream kinases responsible for p38MAPK activation include MEK-3 and MEK-6 [182, 190]. The pharmacological tools SB203580 and SE239063 are relatively specific inhibitors of p38MAPK and have shown the role of p38MAPK in many TNF-induced cellular responses [191]. One of the first reports to implicate p38MAPK in TNF signalling conclusively showed the role of the kinase in TNF-stimulated phosphorylation of heat-shock protein hsp27 and induction of NFκB activity and interleukin-6 [166, 192]. Indeed, many of the actions of p38MAPK are considered proinflammatory [193, 194], although the role of p38MAPK in cell death and survival is not fully understood [195–200]. The JNK family of ERKs is a highly active arm of TNF receptor signalling mechanisms. JNK is activated mainly by MEK-4 (SEK-1), but also by MEK-7 [183]. Both TNFR1 and TNFR2 are proficient activators of JNK [165, 201, 202], however, whether both receptors activate JNK through overlapping mechanisms is currently under investigation. It is known that TRAF2 activation leads to JNK stimulation (see above and Ref. [203]) and that caspase action may lead to JNK activity [204–207]. Unfortunately, no commercially available JNK pathway inhibitor presently exists but the use of tools such as a dominant-negative form of MEK-4 has revealed an important role for JNK in TNF-mediated cellular processes including a probable role in apoptosis [46, 197, 208–212].

An area of recent intense research are the signalling pathways employed by TNF to activate NFκB transcription factor (Fig. 2). Because of its proinflammatory effects, TNF plays a key role in the pathology of several immune diseases including rheumatoid arthritis [213]. Its induction of other inflammatory mediators and processes are under the control of several promoter gene sequences including NFκB [27]. The activation of NFκB is controlled by its association with IκBα [214], whose degradation is controlled by its phosphorylation on serine 176 by NIK [51] and which is activated by the TNF stimulus [50]. Several of the TNF receptor-associated molecules, including TRAF2 [215] and RIP [44–46], have been implicated as upstream NFκB activators. TRAF2 was the first of these entities identified to activate NFκB, with dominant-negative forms of TRAF2 (but not TRAF1 or TRAF3) able to block TNF-induced NFκB activity [216]. TRAF2 associated with NIK and was suggested to be crucial to NFκB activation, however, embryonic fibroblasts from TRAF2-deficient knockout mice were still capable of TNF-induced NFκB activation, with cytokine-stimulated JNK activity blocked in this mouse model [217, 218]. RIP-deficient mice on the other hand were incapable of TNF-stimulated NFκB activity, with JNK activation and apoptosis unaffected [44, 83]. It appeared that RIP, but not TRAF2, was required for TNF-stimulated NFκB activation. A recent study has implicated both TRAF2 and RIP as essential in TNF-induced NFκB activation, with TRAF2 needed for IKK recruitment and RIP responsible for IKK activation [219]. RIP2 (also called CARDIAK [220, 221]) and RIP3 [222] are related adaptor

molecules capable of NFκB modulation and MAPK activation. It has been known for some time, and in a wide range of cell types, that both TNFR1 and TNFR2 are capable of NFκB activation [41, 167, 202, 223–229]. Recently, RIP has thought to have a possible switching capability where it binds to both TNFRs and regulates their caspase or NFκB signalling [49, 83]. The role of NIK in NFκB activation processes has been brought into question by several studies [56, 214, 230, 231] including the normal activation of NFκB by TNF in NIK-deficient mice [232]. Furthermore, a transactivation process of NFκB stimulation has been identified whereby phosphorylation of IkB (or other signalling components of its activation machinery) results in its activation not necessarily with any characteristic proteolytic degradation [233–238]. Thus, it appears that TNF-induced NFκB activation processes may occur through several signalling means, with the exact conditions for any particular cell still to be delineated.

Protein kinase B (PKB) (Akt) is a more recently discovered kinase that is recruited to the plasma membrane by phosphatidylinositol-3,4,5-trisphosphate (PtdIns(3,4,5)P₃) generated by phosphoinositide 3-kinases (PI3Ks) [239]. PKB activation is completed by its phosphorylation by 3′-phosphoinositide-dependent kinase 1 (PDK1). PKB modulates signalling molecules controlling insulin actions as well as components thought to be important in regulating cell survival responses, including Bad, caspase-9, IKK, raf, and ERK activities [239, 240]. Destruction of PKB by caspases allowed inhibition of its survival signal [241]. Inhibition of PKB activity is also achieved by ceramide lipids allowing apoptotic mechanisms to take hold [242–245]. It was uncovered that PKB plays a role in TNF-induced stimulation of anti-apoptotic NFκB activity [238, 246, 247], with the activation of NIK dependent on PKB which phosphorylates IKKα at threonine 23 [238]. Furthermore, cyclic AMP- and cyclic GMP-mediated inhibition of TNF-induced apoptosis in primary cultured hepatocytes was partly mediated by the stimulation of PKB survival signals [248]. PKB is involved in interleukin-13 reversal of TNF-stimulated death in epithelial cells [249], but is thought to be involved in the TNF-induced death of endothelial cells [250, 251] or synovial fibroblasts [252]. Treatment of cardiac myocytes with TNF does not alter their viability, but leads to activation of a variety of genes through a mechanism involving PKB stimulation [253].

A newly discovered isoform of phosphatidylinositol-4-phosphate 5-kinase (PIP5K-IIβ) responsible for the generation of phosphatidylinositol-4,5-bisphosphate (substrate for PI-PLC-mediated inositol-1,4,5-trisphosphate and diacylglycerol second messengers) was found to interact with and be stimulated by TNFR1, but not TNFR2 [254]. This interaction of PIP5K-IIβ with TNFR1 was through the receptor’s juxtamembrane region. Clearly, TNFR1 is capable of interacting with a wide variety of signalling machinery, and this interesting finding opens up new possibilities for the control of classical phosphatidylinositol

cysteine
-rich motif

death domain

adaptor protein

caspase

kinase

lipase

G-protein

soluble
TNF

membrane
-bound TNF

Internalization Internalization

Ligand Passing

Shedding

BRE
SODD
Grb2
MADD
FAN
PIP5K
p60TRAK

BRE
SODD
Grb2
MADD
FAN
PIP5K
p60TRAK

TRADD
RAIDD RIP
FADD
caspase-2
caspase-8

TRADD
TRAF2
RIP
FADD
or

TRAF2

NIK
RIP
GCK
ASK
caspases

IKK
IkB
NFkB

IKK
IkB
NFkB

raf
JNK
MEK
p38
MAPK
PKCs
PKB

SMase
CPLA₂
PC-PLC
IRS-1

ceramide
CAPK
PKCζ

AA
DAG
ras cdc42
rac rho
Giα Gq/11α
Ca²⁺ InsP₃

Fig. 2. Signalling pathways modulated by TNF receptor subtypes.

signalling by TNFR1, as well as Gqα-linked forms of
G-protein-coupled receptors.

As well as TNF stimulating the production of ceram-
ide to stimulate CAPK (see above), ceramide also has a
role in the activation of ceramide-activated protein phos-
phatase (CAPP) of the Type 2A [255–257]. CAPP was
found to be important in TNF-induced c-myc down-
regulation [257] as well as the dephosphorylation of c-
Jun in TNF-stimulated A431 human epithelial cells [258].
Other reports have also indicated the possible involve-
ment of serine/threonine- [259,260] or tyrosine-specific
[261,262] phosphatases in the regulation of cellular TNF

responses. In particular, tyrosine phosphatases may be involved in the regulation of NFκB activation processes that are controlled by TNF stimulation [263–267]. Dephosphorylation of signalling proteins by CAPP, serine/threonine, and tyrosine phosphatases provides a new perspective on signalling regulation by TNF receptors which remains underresearched.

TNF is also known as cachectin because of its primary role in the muscle wasting disorder cachexia [268, 269]. It is now becoming apparent that TNF may play an important role in metabolic disorders including Type II diabetes mellitus [270–273]. Early work on this topic revealed that TNF could interfere with insulin signalling mechanisms, particularly by inhibiting the tyrosine kinase activities of the insulin receptor and serine phosphorylation of the insulin receptor substrate 1 (IRS-1) [274–277]. It has since been shown that the TNFR1 isoform plays a major role in the TNF-mediated insulin resistance [278–280], which occurs in a variety of lipid-handling tissues [274, 275, 281] suggesting that anti-TNF therapy may be useful in diabetes [282]. The exact signalling mechanism for these insulin-modulating effects of TNF is not fully clear but has been proposed to include PLC-γ, PKC-ζ, PKB, and the STAT5 transcription factor that controls interferon-stimulated gene activity [283–285].

protein (see above) that coordinates ras activation to stimulate the raf/MEK/MAPK signalling axis [63]. RIP2 adaptor protein acts in a similar fashion to ras in stimulating raf kinase activity [300]. TNF-generated ceramides are capable of binding to and activating ras protein [301]. Indeed, CAPK is a regulator of ras [60] and is necessary for TNF stimulation of MAPK in intestinal epithelial cells [302]. TNF has also recently been shown to stimulate cdc42 in fibroblasts [303], and an Rac/cdc42 [304] and rho [305] monomeric G-protein pathway in endothelial cells. Such TNF-induced changes in endothelial cell cytoskeletal structures have also been reported by others to involve Rac, cdc42, and p21 rho monomeric G-protein [306]. TNF-induced cytoskeletal rearrangement is thought to occur by specific regions (similar to the FAN-associating site) of TNFR1 [307, 308]. In airway smooth muscle cells, TNF stimulates a rho-activation pathway that is involved in myosin light chain phosphorylation and contractile sensitivity [309]. Activation by TNF of transcription factors, such as NFκB and c-fos serum response element, has been shown to involve monomeric G-proteins such as rho, Rac, and cdc42 [310, 311]. Thus, many aspects of TNFR signalling are probably significantly regulated by monomeric as well as heterotrimeric G-proteins.

A role for inositol phosphates and Ca²⁺ in TNF-stimulated cellular responses has been suggested for some time [115, 312]. TNF was shown to inhibit inositol phosphate action and cellular Ca²⁺ handling processes in a variety of cell types [120, 313, 314], with a possible regulation of Ca²⁺-mobilizing InsP₃ receptor subtypes by caspase-dependent processes [96]. In cardiac myocytes, TNF regulated Ca²⁺ mobilization [315] and in dentate gyrus, long-term potentiation (a Ca²⁺ handling phenomenon) was inhibited by TNF treatment [316]. TNF is thought to have a primary role in the onset of asthmatic conditions [317, 318]. In airway smooth muscle, TNF enhances Ca²⁺ mobilization in a process that is thought to be crucial to the airway hyperresponsiveness [319, 320]. TNFR1 is the receptor isoform responsible for the observed hyperresponsiveness [321, 322] which also signals for MAPK and NFκB activities [167, 323]. It was uncovered that TNF was rapidly stimulating a novel pathway resulting in the enhanced phosphorylation of myosin light chain₂₀ that resulted in greater contractile force [324]. TNF itself did not alter [Ca²⁺]i but enhanced the activity of other Ca²⁺-mobilizing agents [324, 325] and could be considered a pure Ca²⁺-sensitizing agent. Prolonged TNF stimulation has itself been shown to raise [Ca²⁺]i in L929 fibroblasts, which undergo TNF-induced cytotoxicity [326]; however, these Ca²⁺-raising effects of TNF are not evident in a range of cell types which undergo TNF-induced cell death [296, 327]. Interestingly in sensory neurones, TNF has the ability to stimulate rapid transient waves of Ca²⁺ mobilization [328]. Although similar to InsP₃-induced release of intracellular Ca²⁺ stores, these TNF-induced Ca²⁺ spikes are probably through ryanodine-sensitive stores, brought about by S-1-P converted from ceramide.

8. Other signalling mechanisms

Other signalling means by which TNF receptors can control cellular function have been demonstrated. Although not of the seven transmembrane G-protein-coupled class of receptors, TNF receptors can influence heterotrimeric G-proteins. Bacterial toxin experiments in osteoblasts, breast cancer cells, and hepatocytes implicated pertussis toxin-sensitive G-proteins in the sensitivity of these cells to TNF treatment [286–288]. In neutrophils, the involvement of the Giα class of G-proteins has been implicated in TNF responses [289, 290]. TNF priming of leukocytes also leads to regulation of G-protein activity and levels, particularly of the Gi₂α class [291–295], whereas in airway smooth muscle cells, TNF treatment could lead to the regulation of Gi₂α and Gi₃α levels. TNF was recently found to selectively regulate the degradation of Gqα/G₁₁α class of G-protein in HeLa and L929 cell models of cytotoxicity [296]. Proteins that modulate G-protein function, such as GSP2 (G-protein pathway suppressor) and RGS16 (regulator of G-protein signalling), have also been found to be regulated by TNF activation [297, 298]. Hence, although not G-protein-coupled themselves, TNFRs can modulate heterotrimeric G-protein function through a range of signalling means.

Monomeric small G-proteins such as ras probably play a role in TNF signalling. For example, inhibition of ras activity by rapla tumour suppressor gene or rasN17 dominant-negative mutant blocked TNF-induced apoptosis in fibroblasts [299]. TNFR1 is able to bind Grb2 adaptor

D.J. MacEwan / Cellular Signalling 14 (2002) 477–492

9. TNFR1

More information is known about TNFR1 than TNFR2. This is due to a number of factors including differences in affinity and efficacy of its ligands. The affinity of TNF ligand for TNFR1 receptor varies depending on the study from approximately 100 pM on native receptor [8,329,330], to estimations on cloned receptors from 100–600 pM [20,331–335]. Another study on native receptor indicated that soluble TNF, as is used experimentally, rapidly binds to TNFR1 with high affinity (Kd of 19 pM) and a slow dissociation from the receptor once bound (t1/2 = 33 min), a process which efficiently activates the receptor [336]. Such kinetics are different from TNFR2 association (see below). Phosphorylation of TNFR1 can occur at a consensus MAPK site on its cytoplasmic domain and through tyrosine phosphorylation [337–340], although it is not fully understood how these phosphorylations control receptor processing. TNFR1 is obviously a cell surface receptor, but large amounts of the receptor can be found localised at the perinuclear-Golgi complex, as is TRADD, but which only associates with TNFR1 once at the plasma membrane [341,342]. TNFR1 has the ability to be cleaved as its soluble form is found in the sera of healthy patients, with higher levels in patients suffering leukaemia [343] plus these shed forms of TNFRs may play a possible role in arthritis [213]. Stimulation of TNFR1 leads to the internalization of the receptor [344] which leads to an inhibition of its long-term actions.

Almost all of the signalling mentioned above can or has been ascribed to TNFR1, with much of the research focus primarily being about the TNFR1 subtype [80]. As the archetypal death receptor, TNFR1 has been heavily investigated worldwide in attempts to understand the controlling of cell death mechanisms as possible therapies for many diseases. So what is specific about TNFR1 that cannot be accessed by TNFR2? There is no doubt about TNFR1’s ability to induce cell death. Many of its intracellular regions have been mapped and shown to specifically interact with a host of apoptosis-related proteins. Specifically, TNFR1 association with TRADD/FADD occurs through the DD which exists on TNFR1, but not TNFR2. Moreover, generally, it is thought that caspase activation occurs through TNFR1-specific sequences of events, initiated by the receptor’s DD. Stimulation of ceramide production is a pro-apoptotic response in any cell which is mediated by the initial signalling of the FAN adaptor protein, once more a TNFR1-specific signal [37]. Similarly, BRE, PIP5K, and Grb2 association with TNFR1 has been reported to occur through this receptor subtype only. Grb2 activation would stimulate downstream ERKs, as would the MADD adaptor protein, which associates specifically with TNFR1 to stimulate MAPK [67,68], a kinase which has been reported to be activated in a TNFR1-specific nature [165] although TNFR-deficient mice revealed that TNFR2 could activate MAPK, but with different profiles of activity distinct from TNFR1 stimulation [186].

Transgenic mice deficient in TNFR1 display greater sensitivity to infection by Listeria monocytogenes but are resistant to TNF or interleukin-1-mediated in vivo lethality, plus were resistant to models of endotoxic shock induced by lipopolysaccharide and D-galactosamine [345,346]. TNFR1 has also been shown to control early graft versus host disease [347]. Deletion of FADD, caspase-3, or caspase-9 was found to be lethal in mice [348–350] (but can be overcome [79,351]) suggesting a critical role for their development in mice, though not necessarily through their activation by TNFR1.

10. TNFR2

TNFR2 was fully cloned after TNFR1 and its structural and functional characterisation is less well understood than its sister receptor. Part of the reason for the relative lack of signalling information about TNFR2 is that, generally, it is not efficiently activated in the laboratory. It was initially assumed that recombinant 17-kDa soluble TNF (as is provided commercially) was an efficient activator of TNFR2. However, it was uncovered that the membrane-bound 26-kDa form of TNF (mTNF) was vastly superior to soluble TNF in activating TNFR2 [352] leading to qualitatively different responses and new insight into TNFR2 function [353]. TNFR1 is activated equally well by soluble and mTNF. TNF ligand acts primarily in the immune system whereby it would activate TNFRs through cell-cell interactions. As such, most of the TNF effects in vivo may be mediated by mTNF (TNFR1=TNFR2 activation) rather than soluble TNF (TNFR1>TNFR2 activation) and the physiological role of TNFR2 may be underestimated by most TNF research conducted in the laboratory which uses soluble TNF as the stimuli. Soluble TNF acts similar to a partial agonist on TNFR2 in that it binds to the receptor, but is not highly efficacious and efficient in its activation. Such limitations of stimulation are overcome by the use of agonistic antibodies capable of efficiently stimulating TNFRs [165,202,224,334,354–363], although how these functionally relate to natural forms of TNF can only be assumed.

It is accepted that the kinetics of TNFR2 activation by TNF are different from TNFR1 [80]. The dissociation kinetics of TNF from native TNFR2 is approximately 20–30-fold faster than from TNFR1 [336], with workers finding the affinity of TNF for TNFR2 significantly greater [364] or less [336] than the ligand’s affinity for TNFR1. It is not clear how the binding characteristics of membrane-bound TNF at TNFR1 and TNFR2 compare to soluble TNF. Slight structural changes in the TNF sequence can lead to dramatic changes in its binding characteristics to TNF receptors. For example, murine TNF activates mouse TNFR1 and TNFR2 equally well, whereas human TNF acts on mouse TNFR1 but does not bind to mouse TNFR2 [23]. Such observations led to studies in which mutant protein ('muteins') forms of soluble TNF were developed which displayed reduced affinity
towards TNFRs compared to wild-type TNF, but marked selectivity between TNFR1 and TNFR2 [330, 333, 365–367] leading to in vivo information on the role of each TNFR [119, 333, 368–370].

Once activated, TNFR2 is readily cleaved by metalloproteases into a soluble shed form of the receptor which is still capable of TNF binding [344, 371–373]. The role of TNFR2 in cellular responses is not fully understood. Activation of TNFR2 in some cells (mainly T-cells and thymocytes) has been shown to be proliferative [352, 356, 362, 368, 374–378], but it is also known to have a function in regulating TNF-induced apoptosis [202, 228, 352, 354, 378, 379]. TNFR2 does not contain a DD motif but still recruits adaptor proteins. TNFR2 is thought to be able to signal apoptosis directly [31, 202, 380–382] or through a so-called ‘ligand-passing’ mechanism by which TNFR2’s greater affinity and half-life of TNF binding, holds ligand, increases the local TNF concentration in the vicinity of TNFR1 receptors which accept the TNF ligand from TNFR2 and are themselves activated, signalling the TNFR1 apoptotic machinery [364, 383]. Others believe that, additionally, TNFR2 signals for cell death through its cytoplasmic domain to induce endogenous mTNF expression, which then signals for apoptosis on itself through its own expressed TNFR1 [201, 355, 384–386], a mechanism which in some cell types requires the disinhibition of protective cellular factors by agents which block the transcription/translation of such protective proteins [40, 361].

Although regulation of TNFR protein expression is not restricted to TNFR2 alone [387–391], generally the more restrictive tissue distribution of TNFR2 and flexible regulation by numerous stimuli, suggest that modulated TNFR2 protein levels may provide insight into TNF function in cellular situations or pathological conditions [80, 387, 388, 390, 392, 393]. Although TNFRs form homotrimers upon activation by TNF without the assembly of heterotrimers [26, 29, 335], the TNFR1:TNFR2 protein ratio has been found to be important in the way a cell predetermines its TNF response [354, 378, 394, 395]. Thus, largely unmodulated TNFR1 expression coupled to changeable TNFR2 levels in cells alters the TNFR1:TNFR2 ratio and controls the functional outcome of the cell when it encounters a TNF stimulus.

TNFR2 receptors are known targets for certain kinase activities. For example, TNF receptors can stimulate a range of kinases including casein kinase activity [184, 396–400]. TNFR2 is itself a target for phosphorylation by kinases [371], notably casein kinase [401]. Casein kinase phosphorylation of TNFR2 leads to an inhibition of TNF-induced apoptosis and such events may be important in the process of ‘reverse signalling,’ whereby information is transduced into the ligand-containing cell by binding of the TNF receptor [402]. A 59-kDa TNFR2-associated serine/threonine kinase p80TRAK has been described that is TNF-stimulated [403]. p80TRAK binds to a 44 amino acid site in TNFR2 cytoplasmic domain in a similar fashion to casein

kinase [396]. Apart from these processes, none of the signalling thus far discovered is specific for TNFR2. TRAF1 and TRAF2 (as well as TRAF3) association was first uncovered by its association to TNFR2 [41], however, although these TRAFs bind relatively well to TNFR2, it has subsequently been found that they also bind to TNFR1 to varying degrees [80]. Likewise, NFκB and JNK activation occurs through TNFR2 as well as TNFR1, but given the range of signalling cascades accessed by TNFR1, the signalling capabilities of TNFR2 are limited by comparison. Recent work on the association of RIP [49, 180, 219–222, 300, 404], which can bind TRAF2 and differentially associate with TNFR1 and TNFR2, may provide new insight into the molecular mechanisms by which TNFR1 and TNFR2 control such a variety of cellular responses.

Deletion of TNFR2 in transgenic mice has uncovered that this receptor subtype is important in low dose TNF-induced lethality [405]. In addition to a role in thymocyte proliferation [375], TNFR2 plays an important role in models of cerebral malaria and microvascular endothelial cell damage [379, 406, 407]. Langerhans cell migration was depressed in mice lacking TNFR2 [408], whereas TNFR2 plays a critical role in multiorgan inflammation [409]. Experimental hepatitis involves both TNFRs [410] and TNFR2 was seen to have a minor role in *Mycobacterium bovis* (BCG) immunity in knockout mice [411]. Clearly, TNFR2 has a role in certain tissues and diseased states, however, the validity of direct comparisons between TNFR-null transgenic mice and normal cells and tissues, which ubiquitously express TNFRs at altering TNFR1:TNFR2 ratios, always has to be considered when interpreting the physiological role of TNFRs.

### 11. Conclusions

A bewildering array of proteins interact with TNF receptors. Whether this is the tip of the iceberg is unknown, but advances in molecular biology have driven our understanding of TNF-accessible signalling pathways. There is no doubt much still to learn about the molecular mechanisms of TNFR1 and TNFR2, with many advances coming from studies on related receptors and ligands. The signalling processes are vastly more complicated than was first imagined. Clearly, much has been learned about TNF receptors and their signal transduction patterns since a review in this journal just over a decade ago [412]. This is due in part to technological improvements in the research, but it has also been driven by the quest to understand how TNF and its related cytokines can signal for such a range of assorted cellular actions. It is intriguing how one ligand acting at two receptors can generate such a variety of cellular responses. It is probable that the TNF receptors do not always act in exactly the same way each time they are activated. The subtle differences in expression patterns of associated machinery and the modulation
of the proteins it interacts with in its immediate environment, will account for much of the observed signalling diversity in different cell types and tissues. It is now up to the research to more fully understand these refined cellular actions, so that we may properly understand TNF actions and master its effects in knowledgeable strategies for therapeutic intervention for diseases associated with TNF and its signalling.

Acknowledgments

Thanks to Renu Heller for sparking my interest in TNF receptor signalling.

References

[1] Colquhoun D. Br J Pharmacol 1998;125:924–47.
[2] Kenakin T. FASEB J 2001;15:598–611.
[3] Aggarwal BB, Eessalu TE, Hass PE. Nature 1985;318:665–7.
[4] Hass PE, Aggarwal BB. Fed Proc 1985;44:1686.
[5] Rubin BY, Anderson SL, Sullivan SA, Williamson BD, Carswell EA, Old LJ. J Exp Med 1985;162:1099–104.
[6] Creasey AA, Yamamoto R, Vitt CR. Proc Natl Acad Sci USA 1987;84:3293–7.
[7] Hohmann HP, Remy R, Poschl B, van Loon AP. J Biol Chem 1990;265:15183–8.
[8] Kull FC, Jacobs S, Cuatrecasas P. Proc Natl Acad Sci USA 1985;82:5756–60.
[9] Stauber GB, Aiyer RA, Aggarwal BB. J Biol Chem 1988;263:19098–104.
[10] Imamura K, Spriggs D, Kufe D. J Immunol 1987;139:2989–92.
[11] Vitt CR, Yamamoto R, Creasey AA. Fed Proc 1987;46:2117.
[12] Shalaby MR, Palladino MA, Hirabayashi SE, Eessalu TE, Lewis GD, Shepard HM, Aggarwal BB. J Leukocyte Biol 1987;41:196–204.
[13] Hass PE, Hotchkiss A, Mohler M, Aggarwal BB. J Biol Chem 1985;260:2214–8.
[14] Eades DK, Cornelius P, Pekala PH. Placenta 1988;9:247–51.
[15] Kull FC. Biochem Biophys Res Commun 1988;153:402–9.
[16] Heller RA, Song K, Onasch MA, Fischer WH, Chang D, Ringold GM. Proc Natl Acad Sci USA 1990;87:6151–5.
[17] Himmler A, Maurerfogy I, Kronke M, Scheurich P, Pfizenmaier K, Lantz M, Olsson I, Hauptmann R, Stratowa C, Adolf GR. DNA Cell Biol 1990;9:705–15.
[18] Gray PW, Barrett K, Chantry D, Turner M, Feldmann M. Proc Natl Acad Sci USA 1990;87:7380–4.
[19] Loetscher H, Pan YCE, Lahm HW, Gentz R, Brockhaus M, Tabuchi H, Lesslauer W. Cell 1990;61:351–9.
[20] Schall TJ, Lewis M, Koller KJ, Lee A, Rice GC, Wong GHW, Gatanaga T, Granger GA, Lentz R, Raab H, Kohr WJ, Goeddel DV. Cell 1990;61:361–70.
[21] Barrett K, Taylorfishwick DA, Cope AP, Kissonerghis AM, Gray M, Feldmann M, Foxwell BMJ. Eur J Immunol 1991;21:1649–56.
[22] Goodwin RG, Anderson D, Jerzy R, Davis T, Brannan CI, Copeland NG, Jenkins NA, Smith CA. Mol Cell Biol 1991;11:3020–6.
[23] Lewis M, Tartaglia LA, Lee A, Bennett GL, Rice GC, Wong GHW, Chen EY, Goeddel DV. Proc Natl Acad Sci USA 1991;88:2830–4.
[24] Rothe JG, Brockhaus M, Gentz R, Lesslauer W. Immunogenetics 1991;34:338–40.
[25] Old LJ. Science 1985;230:630–2.
[26] Idriss HT, Naismith JH. Microsc Res Tech 2000;50:184–95.
[27] Aggarwal BB. Ann Rheum Dis 2000;59:6–16.
[28] Darnay BG, Aggarwal BB. Ann Rheum Dis 1999;58:2–13.
[29] Locksley RM, Killeen N, Lenardo MJ. Cell 2001;104:487–501.
[30] Inoue J, Ishida T, Tsukamoto N, Kobayashi N, Naito A, Azuma S, Yamamoto T. Exp Cell Res 2000;254:14–24.
[31] Grell M, Zimmermann G, Hulser D, Pfizenmaier K, Scheurich P. J Immunol 1994;153:1963–72.
[32] Chan FKM, Chun HJ, Zheng LX, Siegel RM, Bui KL, Lenardo MJ. Science 2000;288:2351–4.
[33] Jones EY. Curr Opin Struct Biol 2000;10:644–8.
[34] Tartaglia LA, Ayres TM, Wong GHW, Goeddel DV. Cell 1993;74:845–53.
[35] Jiang YP, Woronicz JD, Liu W, Goeddel DV. Science 1999;283:543–6.
[36] Adamklages S, Schwandner R, Adam D, Kreder D, Bernardo K, Kronke M. J Leukocyte Biol 1998;63:678–82.
[37] Adam D, Adamklages S, Kronke M. J Inflammation 1996;47:61–6.
[38] Kolesnick RN, Kronke M. Annu Rev Physiol 1998;60:643–65.
[39] Gu CH, Castellino A, Chan JYH, Chao MV. FASEB J 1998;12:1101–8.
[40] Wajant H, Henkler F, Scheurich P. Cell Signal 2001;13:389–400.
[41] Rothe M, Wong SC, Henzel WJ, Goeddel DV. Cell 1994;78:681–92.
[42] Rothe M, Pan MG, Henzel WJ, Ayres TM, Goeddel DV. Cell 1995;83:1243–52.
[43] Hsu HL, Xiong J, Goeddel DV. Cell 1995;81:495–504.
[44] Stanger BZ, Leder P, Lee TH, Kim E, Seed B. Cell 1995;81:513–23.
[45] Hsu HL, Huang JN, Shu HB, Baichwal V, Goeddel DV. Immunity 1996;4:387–96.
[46] Liu ZG, Hsu HL, Goeddel DV, Karin M. Cell 1996;87:565–76.
[47] Chinnaiyan AM, O'Rourke K, Tewari M, Dixit VM. Cell 1995;81:505–12.
[48] Muzio M, Chinnaiyan AM, Kischkel FC, O'Rourke K, Shevchenko A, Ni J, Scaffidi C, Bretz JD, Zhang M, Grentz R, Mann M, Krammer PH, Peter ME, Dixit VM. Cell 1996;85:817–27.
[49] Pimentel-Muinos FX, Seed B. Immunity 1999;11:783–93.
[50] Malinin NL, Boldin MP, Kovalenko AV, Wallach D. Nature 1997;385:540–4.
[51] Ling L, Cao ZD, Goeddel DV. Proc Natl Acad Sci USA 1998;95:3792–7.
[52] Song HY, Rothe M, Goeddel DV. Proc Natl Acad Sci USA 1996;93:6721–5.
[53] Nishitoh H, Saitoh M, Mochida Y, Takeda K, Nakanon H, Rothe M, Miyazono K, Ichijo H. Mol Cell 1998;2:1–20.
[54] Yuasa T, Ohno S, Kehrl JH, Kyriakis JM. J Biol Chem 1998;273:22681–92.
[55] Shi CS, Kehrl JH. J Biol Chem 1997;272:32102–7.
[56] Baud V, Liu Z-G, Bennett B, Suzuki N, Xia Y, Karin M. Genes Dev 1999;13:1297–308.
[57] Kreder D, Schwandner R, Wiegmann K, Adam-Klage S, Schere G, Bernardo K, Pfeffer K, Kronke M. J Interferon Cytokine Res 1998;18:A-54.
[58] Dressler KA, Mathias S, Kolesnick RN. Science 1992;255:1715–8.
[59] Mathias S, Dressler KA, Kolesnick RN. Proc Natl Acad Sci USA 1991;88:10009–13.
[60] Zhang YH, Yao B, Delikat S, Bayoumy S, Lin XH, Basu S, McGinley M, Chanhui PY, Lichenstein H, Kolesnick R. Cell 1997;89:63–72.
[61] Yao B, Zhang YH, Delikat S, Mathias S, Basu S, Kolesnick R. Nature 1995;378:307–10.
[62] Mathias S, Pena LA, Kolesnick RN. Biochem J 1998;335:465–80.
[63] Hildt E, Oess S. J Exp Med 1999;189:1707–14.
[64] Irmler M, Thome M, Hahne M, Schneider P, Hofmann B, Steiner V, Bodmer JL, Schroter M, Burns K, Mattmann C, Rimoldi D, French LE, Tschopp J. Nature 1997;388:190–5.
[65] Thome M, Schneider P, Hofmann K, Fickenscher H, Meil E, Neipel F, Mattmann C, Burns K, Bodmer JL, Schroter M, Scaffidi C, Krammer PH, Peter ME, Tschopp J. Nature 1997;386:517–21.
[66] Kataoka T, Budd RC, Holler N, Thome M, Martinon F, Irmler M,

Burns K, Hahne M, Kennedy N, Kovacsovic M, Tschopp J. Curr Biol 2000;10:640–8.

[67] Schievella AR, Chen JH, Graham JR, Lin L-L. J Biol Chem 1997; 272:12069–75.

[68] Brinkman BMN, Telliez JB, Schievella AR, Lin LL, Goldfeld AE. J Biol Chem 1999;274:30882–6.

[69] Wyllie AH. Eur J Cell Biol 1997;73:189–97.

[70] Denecker G, Vercammen D, Declercq W, Vandenberghe P. Cell Mol Life Sci 2001;58:356–70.

[71] Tewari M, Quan LT, O'Rourke K, Desnoyers S, Zeng Z, Beidler DR, Poirier GG, Salvesen GS, Dixit VM. Cell 1995;81:801–9.

[72] Vercammen D, Brouckaert G, Denecker G, Vandecraen M, Declercq W, Fiers W, Vandenberghe P. J Exp Med 1998;188:919–30.

[73] Vercammen D, Beyaert R, Denecker G, Goossens V, Vanloo G, Declercq W, Grooten J, Fiers W, Vandenberghe P. J Exp Med 1998; 187:1477–85.

[74] Kinloch RA, Treherne M, Furness M, Hajimohamadreza I. TIPS 1999;20:35–42.

[75] Schotte P, Declercq W, Vanhuffel S, Vandenberghe P, Beyaert R. FEBS Lett 1999;442:117–21.

[76] Kuida K, Haydar TF, Kuan CY, Gu Y, Taya C, Karasuyama H, Su MSS, Rakic P, Flavell RA. Cell 1998;94:325–37.

[77] Kuida K, Zheng TS, Na SQ, Kuan CY, Yang D, Karasuyama H, Rakic P, Flavell RA. Nature 1996;384:368–72.

[78] Kuida K, Lippke JA, Ku G, Harding MW, Livingston DJ, Su MSS, Flavell RA. Science 1995;267:2000–3.

[79] Zheng TS, Hunot S, Kuida K, Flavell RA. Cell Death Differ 1999; 6:1043–53.

[80] Vandenberghe P, Declercq W, Beyaert R, Fiers W. Trends Cell Biol 1995;5:392–9.

[81] Holler N, Zaru R, Micheau O, Thome M, Attinger A, Valitutti S, Bodmer JL, Schneider P, Seed B, Tschopp J. Nat Immunol 2000;1: 489–95.

[82] Lin Y, Devin A, Rodriguez Y, Liu ZG. Genes Dev 1999;13:2514–26.

[83] Kelliher MA, Grimm S, Ishida Y, Kuo F, Stanger BZ, Leder P. Immunity 1998;8:297–303.

[84] Cohen GM. Biochem J 1997;326:1–16.

[85] Chao DT, Korsmeyer SJ. Annu Rev Immunol 1998;16:395–419.

[86] Screaton G, Xu XN. Curr Opin Immunol 2000;12:316–22.

[87] Emoto Y, Manome Y, Meinhardt G, Kisaki H, Kharbanda S, Robertson M, Ghayur T, Wong WW, Kamen R, Weichselbaum R, Kufe D. EMBO J 1995;14:6148–56.

[88] Rich T, Allen RL, Wyllie AH. Nature 2000;407:777–83.

[89] Kerr JFR, Wyllie AH, Currie AR. Br J Cancer 1972;28:239–57.

[90] Johnson DE. Leukemia 2000;14:1695–703.

[91] Utz PJ, Anderson P. Cell Death Differ 2000;7:589–602.

[92] Squier MKT, Cohen JJ. Cell Death Differ 1996;3:275–83.

[93] Pomaress MI, Ares MPS, Orrenius S. Toxicol in Vitro 1998;12: 539–43.

[94] Pomaress MI, Samali A, Orrenius S. Cell Death Differ 1998;5: 1028–33.

[95] Hajnoczky G, Csordas G, Madesh M, Pacher P. Cell Calcium 2000; 28:349–63.

[96] Diaz F, Bourguignon LYW. Cell Calcium 2000;27:315–28.

[97] Han YQ, Weinman S, Boldogh I, Walker RK, Brasier AR. J Biol Chem 1999;274:787–94.

[98] Dayer JM, Beutler B, Cerami A. J Exp Med 1985;162:2163–8.

[99] Marquet RL, Debruin RWF, Fiers W, Jeekel J. Immunobiology 1987;175:54–5.

[100] Beyaert R, Suffys P, Van Roy F, Fiers W. Immunobiology 1987;175:3.

[101] Suffys P, Beyaert R, Van Roy F, Fiers W. Biochem Biophys Res Commun 1987;149:735–43.

[102] Clark MA, Chen MJ, Crooke ST, Bomalaski JS. Biochem J 1988; 250:125–32.

[103] Clark MA, Chen MJ, Bomalaski JS. Fed Proc 1987;46:1946.

[104] Godfrey RW, Johnson WJ, Hoffstein ST. Biochem Biophys Res Commun 1987;142:235–41.

[105] Heller RA, Kronke M. J Cell Biol 1994;126:5–9.

[106] Schutze S, Potthoff K, Machleidt T, Berkovic D, Wiegmann K, Kronke M. Cell 1992;71:765–76.

[107] Schutze S, Berkovic D, Tomsing O, Unger C, Kronke M. J Exp Med 1991;174:975–88.

[108] Wiegmann K, Schutze S, Kampen E, Himmler A, Machleidt T, Kronke M. J Biol Chem 1992;267:17997–8001.

[109] Kronke M, Berkovic D, Unger C, Schutze S. Lymphokine Res 1990;9:549.

[110] Machleidt T, Kramer B, Adam D, Neumann B, Schutze S, Wieg- mann K, Kronke M. J Exp Med 1996;184:725–33.

[111] Nishizuka Y. Science 1992;258:607–14.

[112] Devalck D, Vercammen D, Fiers W, Beyaert R. J Cell Biochem 1998;71:392–9.

[113] Plo I, Lautier D, Levade T, Sekouri H, Jaffrezou JP, Laurent G, Bettaieb A. Biochem J 2000;351:459–67.

[114] Kang JH, Shin I, Han JS. Exp Mol Med 1998;30:21–7.

[115] Beyaert R, Heyninck K, Devalck D, Boeykens F, Vanroy F, Fiers W. J Immunol 1993;151:291–300.

[116] Condliffe AM, Hawkins PT, Stephens LR, Haslett C, Chilvers ER. FEBS Lett 1998;439:147–51.

[117] Yorek MA, Dunlap JA, Thomas MJ, Cammarata PR, Zhou C, Lowe WL. Am J Physiol: Cell Physiol 1998;43:C58–71.

[118] Kozawa O, Suzuki A, Kaida T, Tokuda H, Uematsu T. J Biol Chem 1997;272:25099–104.

[119] Murray J, Barbara JAJ, Dunkley SA, Lopez AF, Vanostade X, Con- dliffe AM, Dransfield I, Haslett C, Chilvers ER. Blood 1997;90: 2772–83.

[120] Reithmann C, Werdan K. Naunyn-Schmiedeberg's Arch Pharmacol 1994;349:175–82.

[121] Hannun YA. J Biol Chem 1994;269:3125–8.

[122] Okazaki T, Bell RM, Hannun YA. J Biol Chem 1989;264:19076–80.

[123] Kim MY, Linardic C, Obeid L, Hannun Y. J Biol Chem 1991;266: 484–9.

[124] Wiegmann K, Schutze S, Machleidt T, Witte D, Kronke M. Cell 1994;78:1005–15.

[125] Hannun YA, Bell RM. Science 1989;243:500–7.

[126] Kolesnick RN, Hemer MR. J Biol Chem 1990;265:18803–8.

[127] Dressler KA, Kolesnick RN. J Biol Chem 1990;265:14917–21.

[128] Kolesnick RN, Hemer MR. J Biol Chem 1990;265:10900–4.

[129] Okazaki T, Kondo T, Kitano T, Tashima M. Cell Signal 1998; 10:685–92.

[130] Hannun YA, Luberto C, Argraves KM. Biochem 2001;40:4893–903.

[131] Basu S, Bayoumy S, Zhang Y, Lozano J, Kolesnick R. J Biol Chem 1998;273:30419–26.

[132] Raines MA, Kolesnick RN, Golde DW. J Biol Chem 1993;268: 14572–5.

[133] Bird TA, Kyriakis JM, Tyshler L, Gayle M, Milne A, Virca GD. J Biol Chem 1994;269:31836–44.

[134] Coroneos E, Wang YZ, Panuska JR, Templeton DJ, Kester M. Bio- chem J 1996;316:13–7.

[135] Sawai H, Okazaki T, Takeda Y, Tashima M, Sawada H, Okuma M, Kishi S, Umehara H, Domae N. J Biol Chem 1997;272:2452–8.

[136] Lee JY, Hannun YA, Obeid LM. J Biol Chem 1996;271:13169–74.

[137] Lee JY, Hannun YA, Obeid LM. J Biol Chem 2000;275:29290–8.

[138] Laouar A, Glesne D, Huberman E. J Biol Chem 1999;274: 23526–34.

[139] Muller G, Ayoub M, Storz P, Rennecke J, Fabbro D, Pfizenmaier K. EMBO J 1995;14:1961–9.

[140] Diazmeco MT, Lozano J, Municio MM, Berra E, Frutos S, Sanz L, Moscat J. J Biol Chem 1994;269:31706–10.

[141] Lozano J, Berra E, Municio MM, Diazmeco MT, Dominguez I, Sanz L, Moscat J. J Biol Chem 1994;269:19200–2.

[142] Berra E, Diazmeco MT, Lozano J, Frutos S, Municio MM, Sanchez P, Sanz L, Moscat J. EMBO J 1995;14:6157–63.

[143] Conway AM, Pyne NJ, Pyne S. Cell Signal 2000;12:737–43.

[144] Pyne S, Pyne NJ. Biochem J 2000;349:385–402.

[145] Clark JD, Lin LL, Kriz RW, Ramesha CS, Sultzman LA, Lin AY, Milona N, Knopf JL. Cell 1991;65:1043–51.
[146] Chang DJ, Ringold GM, Heller RA. Biochem Biophys Res Commun 1992;188:538–46.
[147] Wong GHW, Elwell JH, Oberley LW, Goeddel DV. Cell 1989;58:923–31.
[148] Wong GHW, Goeddel DV. Science 1988;242:941–4.
[149] Nemenoff RA, Winitz S, Qian NX, Vanputten V, Johnson GL, Heasley LE. J Biol Chem 1993;268:1960–4.
[150] Lin W-W, Chen BC. Br J Pharmacol 1998;125:1601–9.
[151] Lin LL, Wartmann M, Lin AY, Knopf JL, Seth A, Davis RJ. Cell 1993;72:269–78.
[152] Kramer RM, Roberts EF, Um SL, Borschhaubold AG, Watson SP, Fisher MJ, Jakubowski JA. J Biol Chem 1996;271:27723–9.
[153] Borschhaubold AG, Bartoli F, Asselin J, Dudler T, Kramer RM, Apitzcastro R, Watson SP, Gelb MH. J Biol Chem 1998;273:4449–58.
[154] Borschhaubold AG, Kramer RM, Watson SP. Eur J Biochem 1997;245:751–9.
[155] Hoeck WG, Ramesha CS, Chang DJ, Fan N, Heller RA. Proc Natl Acad Sci USA 1993;90:4475–9.
[156] Hayakawa M, Ishida N, Takeuchi K, Shibamoto S, Hori T, Oku N, Ito F, Tsujimoto M. J Biol Chem 1993;268:11290–5.
[157] Wu YL, Jiang XR, Newland AC, Kelsey SM. J Immunol 1998;160:5929–35.
[158] Wu YL, Jiang XR, Lillington DM, Allen PD, Newland AC, Kelsey SM. Cancer Res 1998;58:633–40.
[159] Suffys P, Beyaert R, Devalck D, Vanhaesebroeck B, Vanroy F, Fiers W. Eur J Biochem 1991;195:465–75.
[160] O'Brien JB, Piddingtion DL, Voelkeljohnson C, Richards DJ, Hadley LA, Laster SM. J Immunol 1998;161:1525–32.
[161] Thorne TE, Voelkeljohnson C, Casey WM, Parks LW, Laster SM. J Virol 1996;70:8502–7.
[162] Voelkeljohnson C, Thorne TE, Laster SM. J Immunol 1996;156:201–7.
[163] Jayadev S, Hayter HL, Andrieu N, Gamard CJ, Liu B, Balu R, Hayakawa M, Ito F, Hannun YA. J Biol Chem 1997;272:17196–203.
[164] Boone E, Vandevoorde V, Dewilde G, Haegeman G. FEBS Lett 1998;441:275–80.
[165] Jupp OJ, McFarlane SM, Anderson HM, Littlejohn AF, McKay RH, Vandenabeele P, MacEwan DJ. Biochem J 2001;359:525–5.
[166] Fiers W, Beyaert R, Boone E, Cornelis S, Declercq W, Decoster E, Denecker G, Depuydt B, Devalck D, Dewilde G, Goossens V, Grooten J, Haegeman G, Heyninck K, Penning L, Plaisance S, Vancompernolle K, Vancriekinge W, Vandenabeele P, Vandenberghe W, Vandecraen M, Vandevoorde V, Vercammen D. J Inflammation 1996;47:67–75.
[167] McFarlane SM, Jupp OJ, Cobban HJ, Hunter I, Anderson HM, Vandenabeele P, Nixon GF, MacEwan DJ. Biochem Pharmacol 2001;61:749–59.
[168] MacEwan DJ. FEBS Lett 1996;379:77–81.
[169] Luschen S, Ussat S, Kronke M, Adamklages S. Biochem Biophys Res Commun 1998;253:92–8.
[170] Adamklages S, Schwandner R, Luschen S, Ussat S, Kreder D, Kronke M. J Immunol 1998;161:5687–94.
[171] Atsumi G, Tajima M, Hadano A, Nakatani Y, Murakami M, Kudo I. J Biol Chem 1998;273:13870–7.
[172] Wissing D, Mouritzen H, Egeblad M, Poirier GG, Jaattela M. Proc Natl Acad Sci USA 1997;94:5073–7.
[173] Voelkeljohnson C, Entingh AJ, Wold WSM, Gooding LR, Laster SM. J Immunol 1995;154:1707–16.
[174] Bermudez LE, Young LS. J Leukocyte Biol 1987;42:598–9.
[175] Schutze S, Nottrott S, Pfizenmaier K, Kronke M. J Immunol 1990;144:2604–8.
[176] Bereta J, Bereta M, Cohen S, Cohen MC. J Cell Biochem 1991;47:62–78.
[177] Matsubara N, Fuchimoto S, Orita K. Immunology 1991;73:457–9.
[178] Sanchotello M, Terranova PF. Endocrinology 1991;128:1223–8.
[179] Kilpatrick LE, Song YH, Rossi MW, Korchak HM. Am J Physiol: Cell Physiol 2000;279:C2011–8.
[180] Sanz L, Sanchez P, Lallena MJ, Diaz-Meco MT, Moscat J. EMBO J 1999;18:3044–53.
[181] Bonizzi G, Piette J, Schoonbroodt S, Merville MP, Bours V. Biochem Pharmacol 1999;57:713–20.
[182] Paul A, Wilson S, Belham CM, Robinson CJM, Scott PH, Gould R, Plevin R. Cell Signal 1997;9:403–10.
[183] Davis RJ. Cell Resp Stress 1999;1–12.
[184] Vanlint J, Agostinis P, Vandevoorde V, Haegeman G, Fiers W, Merlevede W, Vandenheede JR. J Biol Chem 1992;267:25916–21.
[185] Minden A, Lin A, McMahon M, Lange Carter C, Derijard B, Davis RJ, Johnson GL, Karin M. Science 1994;266:1719–23.
[186] Kalb A, Bluethmann H, Moore MW, Lesslauer W. J Biol Chem 1996;271:28097–104.
[187] Chang NS. Biochem Biophys Res Commun 2000;267:194–200.
[188] Rao KMK. J Leukocyte Biol 2001;69:3–10.
[189] Cuvillier O, Pirianov G, Kleuser B, Vanek PG, Coso OA, Gutkind JS, Spiegel S. Nature 1996;381:800–3.
[190] Enslen H, Raingeaud J, Davis RJ. J Biol Chem 1998;273:1741–8.
[191] Barone FC, Irving EA, Ray AM, Lee JC, Kassis S, Kumar S, Badger AM, Legos JJ, Erhardt JA, Ohlstein EH, Hunter AJ, Harrison DC, Philpott K, Smith BR, Adams JL, Parsons AA. Med Res Rev 2001;21:129–45.
[192] Beyaert R, Cuesta A, Vandenberghe W, Plaisance S, Lee JC, Haegeman G, Cohen P, Fiers W. EMBO J 1996;15:1914–23.
[193] Tsukahara K, Nakao A, Hiraguri M, Miike S, Mamura M, Saito Y, Iwamoto I. Int Arch Allergy Immunol 1999;120:54–9.
[194] Nakada S, Kawano T, Saito-Akita S, Iwase S, Horiguchi-Yamada J, Ohno T, Yamada H. Anticancer Res 2001;21:167–71.
[195] Raingeaud J, Gupta S, Rogers J, Dickens M, Han J, Ulevitch RJ, Davis RJ. J Biol Chem 1995;270:7420–6.
[196] Merritt C, Enslen H, Diehl N, Conze D, Davis RJ, Rincon M. Mol Cell Biol 2000;20:936–46.
[197] Xia Z, Dickens M, Raingeaud J, Davis RJ, Greenberg ME. Science 1995;270:1326–31.
[198] Eliopoulos AG, Gallagher NJ, Blake SMS, Dawson CW, Young LS. J Biol Chem 1999;274:16085–96.
[199] Schwenger P, Alpert D, Skolnik EY, Vilcek J. Mol Cell Biol 1998;18:78–84.
[200] Cross TG, Scheel-Toellner D, Henriquez NV, Deacon E, Salmon M, Lord JM. Exp Cell Res 2000;256:34–41.
[201] Weiss T, Grell M, Siemienski K, Muhlenbeck F, Durkop H, Pfizenmaier K, Scheurich P, Wajant H. J Immunol 1998;161:3136–42.
[202] Haridas V, Darnay BG, Natarajan K, Heller R, Aggarwal BB. J Immunol 1998;160:3152–62.
[203] Song HY, Regnier CH, Kirschning CJ, Goeddel DV, Rothe M. Proc Natl Acad Sci USA 1997;94:9792–6.
[204] Muhlenbeck F, Haas E, Schwenzer R, Schubert G, Grell M, Smith C, Scheurich P, Wajant H. J Biol Chem 1998;273:33091–8.
[205] Widmann C, Gerwins P, Johnson L, Jarpe MB, Johnson GL. Mol Cell Biol 1998;18:2416–29.
[206] Cardone MH, Salvesen GS, Widmann C, Johnson G, Frisch SM. Cell 1997;90:315–23.
[207] Avdi NJ, Nick JA, Whitlock BB, Billstrom MA, Henson PM, Johnson GL, Worthen GS. J Biol Chem 2001;276:2189–99.
[208] Kunz M, Ibrahim S, Koczán D, Thiesßen HJ, Köhler HJ, Acker T, Plate KH, Ludwig S, Rapp UR, Brocker EB, van Muijen GN, Flory E, Gross G. Cell Growth Differ 2001;12:137–45.
[209] Tournier C, Hess P, Yang DD, Xu J, Turner TK, Nimnual A, Bar-Sagi D, Jones SN, Flavell RA, Davis RJ. Science 2000;288:870–4.
[210] Doman RK, Perez M, Donato NJ. J Interferon Cytokine Res 1999;19:261–9.
[211] Verheij M, Bose R, Lin XH, Yao B, Jarvis WD, Grant S, Birrer MJ, Szabo E, Zon LI, Kyriakis JM, Haimovitz Friedman A, Fuks Z, Ko-lesnick RN. Nature 1996;380:75–9.

[212] Helms MJ, Mohamed AAA, MacEwan DJ. FEBS Lett 2001; 505: 68–74.

[213] Feldmann M, Maini RN. Annu Rev Immunol 2001; 19: 163–96.

[214] Karin M. J Biol Chem 1999; 274: 27339–42.

[215] Hsu HL, Shu HB, Pan MG, Goeddel DV. Cell 1996; 84: 299–308.

[216] Rothe M, Sarma V, Dixit VW, Goeddel DV. Science 1995; 269: 1424–7.

[217] Lee SY, Reichlin A, Santana A, Sokol KA, Nussenzweig MC, Choi Y. Immunity 1997; 7: 703–13.

[218] Yeh WC, Shahinian A, Speiser D, Kraunus J, Billia F, Wakeham A, Delapompa JL, Ferrick D, Hum B, Iscove N, Ohashi P, Rothe M, Goeddel DV, Mak TW. Immunity 1997; 7: 715–25.

[219] Devin A, Cook A, Lin Y, Rodriguez Y, Kelliher M, Liu ZG. Immunity 2000; 12: 419–29.

[220] McCarthy JV, Ni J, Dixit VM. J Biol Chem 1998; 273: 16968–75.

[221] Thome M, Hofmann K, Burns K, Martinon F, Bodmer JL, Mattmann C, Tschopp J. Curr Biol 1998; 8: 885–8.

[222] Sun XQ, Lee J, Navas T, Baldwin DT, Stewart TA, Dixit VM. J Biol Chem 1999; 274: 16871–5.

[223] Kruppa G, Thoma B, Machleidt T, Wiegmann K, Kronke M. J Immunol 1992; 148: 3152–7.

[224] Laegreid A, Medvedev A, Nonstad U, Bombara MP, Ranges G, Sundan A, Espevik T. J Biol Chem 1994; 269: 7785–91.

[225] Thommesen L, Sjursen W, Gasvik K, Hanssen W, Brekke OL, Skatetbol L, Holmeide AK, Espevik T, Johansen B, Laegreid A. J Immunol 1998; 161: 3421–30.

[226] Brink R, Lodish HF. J Biol Chem 1998; 273: 4129–34.

[227] Takeuchi M, Rothe M, Goeddel DV. J Biol Chem 1996; 271: 19935–42.

[228] Weiss T, Grell M, Hessabi B, Bourteele S, Muller G, Scheurich P, Wajant H. J Immunol 1997; 158: 2398–404.

[229] Amrani Y, Lazaar AL, Panettieri RA. J Immunol 1999; 163: 2128–34.

[230] Shinkura R, Kitada K, Matsuda F, Tashiro K, Ikuta K, Suzuki M, Kogishi K, Serikawa T, Honjo T. Nat Genet 1999; 22: 74–7.

[231] Matsushima A, Kaisho T, Rennert PD, Nakano H, Kurosawa K, Uchida D, Takeda K, Akira S, Matsumoto M. J Exp Med 2001; 193: 631–6.

[232] Yin L, Wu L, Wesche H, Arthur CD, White JM, Goeddel DV, Schreiber RD. Science 2001; 291: 2162–5.

[233] Nasuhara Y, Adcock IM, Catley M, Barnes PJ, Newton R. J Biol Chem 1999; 274: 19965–72.

[234] Vandenberghe W, Plaisance S, Boone E, Debosscher K, Schmitz ML, Fiers W, Haegeman G. J Biol Chem 1998; 273: 3285–90.

[235] Fischer C, Page S, Weber M, Eisele T, Neumeier D, Brand K. J Biol Chem 1999; 274: 24625–32.

[236] Heyninck K, Devalck D, Vandenberghe W, Vancriekinge W, Contreras R, Fiers W, Haegeman G, Beyaert R. J Cell Biol 1999; 145: 1471–82.

[237] Johannes FJ, Horn J, Link G, Haas E, Siemienski K, Wajant H, Pfizenmaier K. Eur J Biochem 1998; 257: 47–54.

[238] Ozes ON, Mayo LD, Gustin JA, Pfeffer SR, Pfeffer LM, Donner DB. Nature 1999; 401: 82–5.

[239] Vanhaesebroeck B, Alessi DR. Biochem J 2000; 346: 561–76.

[240] Scheid M, Duronio V. Proc Natl Acad Sci 1998; 95: 7439–44.

[241] Widmann C, Gibson S, Johnson GL. J Biol Chem 1998; 273: 7141–7.

[242] Zinda MJ, Vlahos CJ, Lai MT. Biochem Biophys Res Commun 2001; 280: 1107–15.

[243] Schubert KM, Scheid MP, Duronio V. J Biol Chem 2000; 275: 13330–5.

[244] Salinas M, Lopez-Valdaliso R, Martin D, Alvarez A, Cuadrado A. Mol Cell Neurosci 2000; 15: 156–69.

[245] Zhou HL, Summers SK, Birnbaum MJ, Pittman RN. J Biol Chem 1998; 273: 16568–75.

[246] Burow ME, Weldon CB, Melnik LI, Duong BN, Collins-Burow BM, Beckman BS, McLachlan JA. Biochem Biophys Res Commun 2000; 271: 342–5.

[247] Pastorino JG, Tafani M, Farber JL. J Biol Chem 1999; 274: 19411–6.

[248] Li JR, Yang SF, Billiar TR. J Biol Chem 2000; 275: 13026–34.

[249] Wright K, Kolios G, Westwick J, Ward SG. J Biol Chem 1999; 274: 17193–201.

[250] Madge LA, Pober JS. J Biol Chem 2000; 275: 15458–65.

[251] Murao K, Ohyama T, Imachi H, Ishida T, Cao WM, Namihira H, Sato M, Wong NCW, Takahara J. Biochem Biophys Res Commun 2000; 276: 791–6.

[252] Wang YM, Zhang HG, Xie JF, Xu L, Hsu HC, Liu D, Mountz JD. Arthritis Rheum 2000; 43: 1989.

[253] Hiraoka E, Kawashima S, Takahashi T, Rikitake Y, Kitamura T, Ogawa W, Yokoyama M. Am J Physiol: Heart Circ Physiol 2001; 280: H1861–8.

[254] Castellino AM, Parker GJ, Boronenkov IV, Anderson RA, Chao MV. J Biol Chem 1997; 272: 5861–70.

[255] Dobrowsky RT, Kamibayashi C, Mumby MC, Hannun YA. J Biol Chem 1993; 268: 15523–30.

[256] Dobrowsky RT, Hannun YA. J Biol Chem 1992; 267: 5048–51.

[257] Wolff RA, Dobrowsky RT, Bielawska A, Obeid LM, Hannun YA. J Biol Chem 1994; 269: 19605–9.

[258] Reyes JG, Robayna IG, Delgado PS, Gonzalez IH, Aguiar JQ, Rosas FE, Fanjul LF, Degalarreta CMR. J Biol Chem 1996; 271: 21375–80.

[259] Totpal K, Agarwal S, Aggarwal BB. Cancer Res 1992; 52: 2557–62.

[260] Barber SA, Perera PY, McNally R, Vogel SN. J Immunol 1995; 155: 1404–10.

[261] Guo DQ, Wu LW, Dunbar JD, Ozes ON, Mayo LD, Kessler KM, Gustin JA, Baerwald MR, Jaffe EA, Warren RS, Donner DB. J Biol Chem 2000; 275: 11216–21.

[262] Mishra S, Mathur R, Hamburger AW. Lymphokine Cytokine Res 1994; 13: 77–83.

[263] Natarajan K, Manna SK, Chaturvedi MM, Aggarwal BB. Arch Biochem Biophys 1998; 352: 59–70.

[264] Dhawan S, Singh S, Aggarwal BB. Eur J Immunol 1997; 27: 2172–9.

[265] Singh S, Aggarwal BB. J Biol Chem 1995; 270: 10631–9.

[266] Mukhopadhyay A, Manna SK, Aggarwal BB. J Biol Chem 2000; 275: 8549–55.

[267] You M, Flick LM, Yu DH, Feng GS. J Exp Med 2001; 193: 101–9.

[268] Vlassara H, Brownlee M, Pasagian A, Cerami A. Diabetes 1988; 37: A20.

[269] Beutler B, Greenwald D, Hulmes JD, Chang M, Pan YCE, Mathison J, Ulevitch R, Cerami A. Nature 1985; 316: 552–4.

[270] Kulseng B, Vatten L, Espevik T. Acta Diabetol 1999; 36: 99–105.

[271] Saltiel AR. Cell 2001; 104: 517–29.

[272] Moller DE. Trends Endocrinol Metab 2000; 11: 212–7.

[273] Spiegelman BM, Hotamisligil GS. Cell 1993; 73: 625–7.

[274] Hotamisligil GS, Peraldi P, Budavari A, Ellis R, White MF, Spiegelman BM. Science 1996; 271: 665–8.

[275] Hotamisligil GS, Budavari A, Murray D, Spiegelman BM. J Clin Invest 1994; 94: 1543–9.

[276] Hotamisligil GS, Murray DL, Choy LN, Spiegelman BM. Proc Natl Acad Sci USA 1994; 91: 4854–8.

[277] Hotamisligil GS, Shargill NS, Spiegelman BM. Science 1993; 259: 87–91.

[278] Peraldi P, Hotamisligil GS, Spiegelman BM. Diabetes 1996; 45: 161.

[279] Xu HY, Sethi JK, Hotamisligil GS. J Biol Chem 1999; 274: 26287–95.

[280] Sethi JK, Xu HY, Uysal T, Wiesbrock SM, Scheja L, Hotamisligil GS. FEBS Lett 2000; 469: 77–82.

[281] Navasa M, Gordon DA, Hariharan N, Jamil H, Shigenaga JK, Moser AA, Fiers W, Pollock A, Grunfeld C, Feingold KR. J Lipid Res 1998; 39: 1220–30.

[282] Uysal KT, Wiesbrock SM, Marino MW, Hotamisligil GS. Nature 1997; 389: 610–4.

[283] Storz P, Doppler H, Wernig A, Pfizenmaier K, Muller G. FEBS Lett 1998; 440: 41–5.

[284] Ravichandran LV, Esposito DL, Chen J, Quon MJ. J Biol Chem 2001; 276: 3543–9.

[285] Ermakova NV, Paulssen RH, Florholmen J. Diabetologia 1999; 42: 529.

[286] Branellec D, Decremoux P, Barreau P, Calvo F, Chouaib S. Eur J Immunol 1992;22:963–7.
[287] Yanaga F, Abe M, Koga T, Hirata M. J Biol Chem 1992;267:5114–21.
[288] Hernandezmunoz I, Delatorre P, Sanchezalcazar JA, Garcia I, Santiago E, Munozyague M, Solisherruzo JA. Gastroenterology 1997;113:625–40.
[289] McLeish KR, Klein JB, Lederer ED, Head KZ, Ward RA. Kidney Int 1996;50:407–16.
[290] McLeish KR, Klein JB, Schepers T, Sonnenfeld G. Biochem J 1991;279:455–60.
[291] Scherzer JA, Lin Y, McLeish KR, Klein JB. J Immunol 1997;158:913–8.
[292] Klein JB, Scherzer JA, Harding G, Jacobs AA, McLeish KR. J Leukocyte Biol 1995;57:500–6.
[293] Klein JB, Scherzer JA, Corpier CL, Ohara LC, McLeish KR. Clin Res 1994;42:A331.
[294] Scherzer J, Klein J. Clin Res 1993;41:A241.
[295] Klein JB, Scherzer JA, McLeish KR. Clin Res 1992;40:A745.
[296] Pollock VP, Lofthouse EJ, Jupp OJ, Gauld SB, Anderson HM, MacEwan DJ. Mol Cell Biochem 2000;206:67–74.
[297] Fong CW, Zhang Y, Neo SY, Lin SC. Biochem J 2000;352:747–53.
[298] Jin DY, Teramoto H, Giam CZ, Chun RF, Gutkind JS, Jeang KT. J Biol Chem 1997;272:25816–23.
[299] Trent JC, McConkey DJ, Loughlin SM, Harbison MT, Fernandez A, Ananthaswamy HN. EMBO J 1996;15:4497–505.
[300] Navas TA, Baldwin DT, Stewart TA. J Biol Chem 1999;274:33684–90.
[301] Muller G, Storz P, Bourteele S, Doppler H, Pfizenmaier K, Mischak H, Philipp A, Kaiser C, Kolch W. EMBO J 1998;17:732–42.
[302] Yan F, Polk DB. Cancer Res 2001;61:963–9.
[303] Puls A, Eliopoulos AG, Nobes CD, Bridges T, Young LS, Hall A. J Cell Sci 1999;112:2983–92.
[304] Min W, Pober JS. J Immunol 1997;159:3508–18.
[305] Petrache I, Verin AD, Crow MT, Birukova A, Liu F, Garcia JGN. Am J Physiol: Lung Cell Mol Physiol 2001;280:L1168–78.
[306] Wojciak-Stothard B, Entwistle A, Garg R, Ridley AJ. J Cell Physiol 1998;176:150–65.
[307] Peppelenbosch M, Boone E, Jones GE, Vandeventer SJH, Haegeman G, Fiers W, Grooten J, Ridley AJ. J Immunol 1999;162:837–45.
[308] Kieser A, Kaiser C, Hammerschmidt W. EMBO J 1999;18:2511–21.
[309] Hunter I, Cobban HJ, MacEwan DJ, Nixon GF. FASEB J 2001;15:A163.
[310] Perona R, Montaner S, Saniger L, Sanchezperez I, Bravo R, Lacal JC. Genes Dev 1997;11:463–75.
[311] Kim BC, Lee MN, Kim JY, Lee SS, Chang JD, Kim SS, Lee SY, Kim JH. J Biol Chem 1999;274:24372–7.
[312] Denecker G, Vandenameele P, Grooten J, Penning LC, Declercq W, Beyaert R, Buurman WA, Fiers W. Cytokine 1997;9:631–8.
[313] Yorek M, Jaipaul N, Dunlap J, Bielefeldt K. Arch Biochem Biophys 1999;361:241–51.
[314] Rosado JA, Rosenzweig I, Harding S, Sage SO. Am J Physiol: Cell Physiol 2001;280:C1636–44.
[315] Bick RJ, Liao JP, King TW, Lemaistre A, McMILLIN JB, Buja LM. Am J Physiol: Heart Circ Physiol 1997;41:H1937–44.
[316] Cunningham AJ, Murray CA, O’Neill LAJ, Lynch MA, O’Connor JJ. Neurosci Lett 1996;203:17–20.
[317] Thomas PS, Yates DH, Barnes PJ. Am J Respir Crit Care Med 1995;152:76–80.
[318] Thomas PS. Immunol Cell Biol 2001;79:132–40.
[319] Amrani Y, Bronner C. C R Acad Sci, Ser III 1993;316:1489–94.
[320] Amrani Y, Martinet N, Bronner C. Br J Pharmacol 1995;114:4–5.
[321] Amrani Y, Aubier M, Bronner C. Rev Mal Respir 1996;13:539–46.
[322] Amrani Y, Panettieri RA, Frossard N, Bronner C. Am J Respir Cell Mol Biol 1996;15:55–63.
[323] Amrani Y, Laazaar AL, Hoffman R, Amin K, Ousmer S, Panettieri RA. Mol Pharmacol 2000;58:237–45.
[324] Parris JRM, Cobban HJ, Littlejohn AF, MacEwan DJ, Nixon GF. J Physiol 1999;518:561–9.
[325] Nakatani Y, Nishimura Y, Nishiuma T, Maeda H, Yokoyama M. Eur J Pharmacol 2000;392:175–82.
[326] Kong SK, Fung KP, Choy YM, Lee CY. Oncology 1997;54:55–62.
[327] McFarlane SM, Anderson HM, Tucker SJ, Jupp OJ, MacEwan DJ. Mol Cell Biochem 2000;211:19–26.
[328] Pollock J, McFarlane SM, Connell MC, Zehavi U, Vandenameele P, MacEwan DJ, Scott RH. Neuropharmacology 2002;42:93–106.
[329] Tsujimoto M, Yip YK, Vilcek J. Proc Natl Acad Sci USA 1985;82:7626–30.
[330] Vanostade X, Vandenameele P, Everaerd B, Loetscher H, Gentz R, Brockhaus M, Lesslauer W, Tavernier J, Brouckaert P, Fiers W. Nature 1993;361:266–9.
[331] Hohmann HP, Brockhaus M, Baeuerle PA, Remy R, Kolbeck R, van Loon AP. J Biol Chem 1990;265:22409–17.
[332] Pennica D, Kohr WJ, Fendly BM, Shire SJ, Raab HE, Borchardt PE, Lewis M, Goeddel DV. Biochemistry 1992;31:1134–41.
[333] Loetscher H, Stueber D, Banner D, Mackay F, Lesslauer W. J Biol Chem 1993;268:26350–7.
[334] Grell M, Scheurich P, Meager A, Pfizenmaier K. Lymphokine Cytokine Res 1993;12:143–8.
[335] Moosmayer D, Dinkel A, Gerlach E, Hessabi B, Grell M, Pfizenmaier K, Scheurich P. Lymphokine Cytokine Res 1994;13:295–301.
[336] Grell M, Wajant H, Zimmermann G, Scheurich P. Proc Natl Acad Sci USA 1998;95:570–5.
[337] Cottin V, Van Linden AA, Riches DWH. J Biol Chem 2001;276:17252–60.
[338] Cottin V, Van Linden A, Riches DWH. J Biol Chem 1999;274:32975–87.
[339] Van Linden AA, Cottin V, Leu C, Riches DWH. J Biol Chem 2000;275:6996–7003.
[340] Darnay BG, Aggarwal BB. FEBS Lett 1997;410:361–7.
[341] Jones SJ, Ledgerwood EC, Prins JB, Galbraith J, Johnson DR, Pober JS, Bradley JR. J Immunol 1999;162:1042–8.
[342] Ledgerwood EC, Pober JS, Bradley JR. Lab Invest 1999;79:1041–50.
[343] Digel W, Porzsolt F, Schmid M, Herrmann F, Lesslauer W, Brockhaus M. J Clin Invest 1992;89:1690–3.
[344] Higuchi M, Aggarwal BB. J Immunol 1994;152:3550–8.
[345] Pfeffer K, Matsuyama T, Kundig TM, Wakeham A, Kishihara K, Shahinian A, Wiegmann K, Ohashi PS, Kronke M, Mak TW. Cell 1993;73:457–67.
[346] Rothe J, Lesslauer W, Lotscher H, Lang Y, Koebel P, Kontgen F, Althage A, Zinkernagel R, Steinmetz M, Bluethmann H. Nature 1993;364:798–802.
[347] Speiser DE, Bachmann MF, Frick TW, Mckallfaienza K, Griffiths E, Pfeffer K, Mak TW, Ohashi PS. J Immunol 1997;158:5185–90.
[348] Woo M, Hakem A, Elia AJ, Hakem R, Duncan GS, Patterson BJ, Mak TW. J Immunol 1999;163:4909–16.
[349] Hakem R, Hakem A, Duncan GS, Henderson JT, Woo M, Soengas MS, Elia A, De La Pompa JL, Kagi D, Khoo W, Potter J, Yoshida R, Kaufman SA, Lowe SW, Penninger JM, Mak TW. Cell 1998;94:339–52.
[350] Woo M, Hakem R, Soengas MS, Duncan GS, Shahinian A, Kagi D, Hakem A, McCurrach M, Khoo W, Kaufman SA, Senaldi G, Howard T, Lowe SW, Mak TW. Genes Dev 1998;12:806–19.
[351] Zheng TS, Hunot S, Kuida K, Momoi T, Srinivasan A, Nicholson DW, Lazebnik Y, Flavell RA. Nat Med 2000;6:1241–7.
[352] Grell M, Douni E, Wajant H, Lohden M, Clauss M, Maxeiner B, Georgopoulos S, Lesslauer W, Kollias G, Pfizenmaier K, Scheurich P. Cell 1995;83:793–802.
[353] Decoster E, Vanhaesebroeck B, Vandenameele P, Grooten J, Fiers W. J Biol Chem 1995;270:18473–8.
[354] Vandenameele P, Declercq W, Vanhaesebroeck B, Grooten J, Fiers W. J Immunol 1995;154:2904–13.
[355] Lazdins JK, Grell M, Walker MR, Woodscook K, Scheurich P, Pfizenmaier K. J Exp Med 1997;185:81–90.

[356] Tartaglia LA, Weber RF, Figari IS, Reynolds C, Palladino MA, Goeddel DV. Proc Natl Acad Sci USA 1991;88:9292–6.
[357] Leeuwenberg JFM, Vantits LJH, Jeunhomme TMAA, Buurman WA. Cytokine 1995;7:457–62.
[358] Paleolog EM, Delasalle SAJ, Buurman WA, Feldmann M. Blood 1994;84:2578–90.
[359] Gehr G, Gentz R, Brockhaus M, Loetscher H, Lesslauer W. J Immunol 1992;149:911–7.
[360] Belka C, Wiegmann K, Adam D, Holland R, Neuloh M, Herrmann F, Kronke M, Brach MA. EMBO J 1995;14:1156–65.
[361] Wajant H, Haas E, Schwenzer R, Muhlenbeck F, Kreuz S, Schubert G, Grell M, Smith C, Scheurich P. J Biol Chem 2000;275:24357–66.
[362] Borset M, Medvedev AE, Sundan A, Espevik T. Cytokine 1996;8:430–8.
[363] Medvedev AE, Sundan A, Espevik T. Eur J Immunol 1994;24:2842–9.
[364] Tartaglia LA, Pennica D, Goeddel DV. J Biol Chem 1993;268:18542–8.
[365] Vanostade X, Tavernier J, Fiers W. Protein Eng 1994;7:5–22.
[366] Vanostade X, Vandenameele P, Tavernier J, Fiers W. Eur J Biochem 1994;220:771–9.
[367] Barbara JAJ, Smith WB, Gamble JR, Vanostade X, Vandenameele P, Tavernier J, Fiers W, Vadas MA, Lopez AF. EMBO J 1994;13:843–50.
[368] Welborn MB, Vanzee K, Edwards PD, Pruitt JH, Kaibara A, Vauthey JN, Rogy M, Castleman WL, Lowry SF, Kenney JS, Stuber D, Ettlin U, Wipf B, Loetscher H, Copeland EM, Lesslauer W, Moldawer LL. J Exp Med 1996;184:165–71.
[369] Vanderpoll T, Jansen PM, Vanzee KJ, Welborn MB, Dejong I, Hack CE, Loetscher H, Lesslauer W, Lowry SF, Moldawer LL. Blood 1996;88:922–7.
[370] Vanzee KJ, Stackpole SA, Montegut WJ, Rogy MA, Calvano SE, Hsu KC, Chao M, Meschter CL, Loetscher H, Stuber D, Ettlin U, Wipf B, Lesslauer W, Lowry SF, Moldawer LL. J Exp Med 1994;179:1185–91.
[371] Pennica D, Lam VT, Mize NK, Weber RF, Lewis M, Fendly BM, Lipari MT, Goeddel DV. J Biol Chem 1992;267:21172–8.
[372] Porteu F, Hieblot C. J Biol Chem 1994;269:2834–40.
[373] Crowe PD, Walter BN, Mohler KM, Otten-Evans C, Black RA, Ware CF. J Exp Med 1995;181:1205–10.
[374] Vandenameele P, Declercq W, Vercammen D, Vandecraen M, Grooten J, Loetscher H, Brockhaus M, Lesslauer W, Fiers W. J Exp Med 1992;176:1015–24.
[375] Grell M, Becke FM, Wajant H, Mannel DN, Scheurich P. Eur J Immunol 1998;28:257–63.
[376] Tartaglia LA, Goeddel DV, Reynolds C, Figari IS, Weber RF, Fendly BM, Palladino MA. J Immunol 1993;151:4637–41.
[377] Rothe J, Mackay F, Bluethmann H, Zinkernagel R, Lesslauer W. Circ Shock 1994;44:51–6.
[378] Baxter GT, Kuo RC, Jupp OJ, Vandenameele P, MacEwan DJ. J Biol Chem 1999;274:9539–47.
[379] Lucas R, Garcia I, Donati YRA, Hribar M, Mandriota SJ, Giroud C, Buurman WA, Fransen L, Suter PM, Nunez G, Pepper MS, Grau GE. Eur J Immunol 1998;28:3577–86.
[380] Heller RA, Song K, Fan N, Chang DJ. Cell 1992;70:47–56.
[381] Heller RA, Song K, Fan N. Cell 1993;73:216.
[382] Declercq W, Vandenameele P, Fiers W. Cytokine 1995;7:701–9.
[383] Tartaglia LA, Rothe M, Hu YF, Goeddel DV. Cell 1993;73:213–6.
[384] Vercammen D, Vandenameele P, Declercq W, Vandecraen M, Grooten J, Fiers W. Cytokine 1995;7:463–70.
[385] Haas E, Grell M, Wajant H, Scheurich P. J Biol Chem 1999;274:18107–12.
[386] Grell M, Zimmermann G, Gottfried E, Chen CM, Grunwald U, Huang DCS, Lee YHW, Durkop H, Engelmann H, Scheurich P, Wajant H, Strasser A. EMBO J 1999;18:3034–43.
[387] Manna SK, Aggarwal BB. J Biol Chem 1998;273:33333–41.
[388] Totpal K, Chaturvedi MM, Lapushin R, Aggarwal BB. Blood 1995;85:3547–55.
[389] Aggarwal BB, Pandita R. FEBS Lett 1994;337:99–102.
[390] Aggarwal BB, Graff K, Samal B, Higuchi M, Liao WSL. Lymphokine Cytokine Res 1993;12:149–58.
[391] Higuchi M, Aggarwal BB. J Biol Chem 1992;267:20892–9.
[392] Pocsik E, Mihalik R, Penzes M, Loetscher H, Gallati H, Aggarwal BB. J Cell Biochem 1995;59:303–16.
[393] Pocsik E, Mihalik R, Aliosman F, Aggarwal BB. J Cell Biochem 1994;54:453–64.
[394] Medvedev AE, Espevik T, Ranges G, Sundan A. J Biol Chem 1996;271:9778–84.
[395] Declercq W, Denecker G, Fiers W, Vandenameele P. J Immunol 1998;161:390–9.
[396] Darnay BG, Singh S, Aggarwal BB. FEBS Lett 1997;406:101–5.
[397] Guesdon F, Freshney N, Waller RJ, Rawlinson L, Saklatvala J. J Biol Chem 1993;268:4236–43.
[398] Darnay BG, Reddy SAG, Aggarwal BB. J Biol Chem 1994;269:20299–304.
[399] Guesdon F, Waller RJ, Saklatvala J. Biochem J 1994;304:761–8.
[400] Guesdon F, Waller RJ, Saklatvala J. Cytokine 1997;9:471–9.
[401] Beyaert R, Vanhaesebroeck B, Declercq W, Vanlint J, Vandenameele P, Agostinis P, Vandeneede JR, Fiers W. J Biol Chem 1995;270:23293–9.
[402] Watts AD, Hunt NH, Wanigasekara Y, Bloomfield G, Wallach D, Roufogalis BD, Chaudhri G. EMBO J 1999;18:2119–26.
[403] Darnay BG, Reddy SAG, Aggarwal BB. J Biol Chem 1994;269:19687–90.
[404] Grimm S, Stanger BZ, Leder P. Proc Natl Acad Sci USA 1996;93:10923–7.
[405] Erickson SL, Desauvage FJ, Kikly K, Carvermoore K, Pittsmeek S, Gillett N, Sheehan KCF, Schreiber RD, Goeddel DV, Moore MW. Nature 1994;372:560–3.
[406] Lucas R, Juillard P, Decoster E, Redard M, Burger D, Donati Y, Giroud C, Monsohinard C, Dekesel T, Buurman WA, Moore MW, Dayer JM, Fiers W, Bluethmann H, Grau GE. Eur J Immunol 1997;27:1719–25.
[407] Lucas R, Lou JN, Juillard P, Moore M, Bluethmann H, Grau GE. J Neuroimmunol 1997;72:143–8.
[408] Wang B, Kondo S, Shivji GM, Fujisawa H, Mak TW, Sauder DN. Immunology 1996;88:284–8.
[409] Douni E, Kollias G. J Exp Med 1998;188:1343–52.
[410] Kusters S, Tiegs G, Alexopoulou L, Pasparakis M, Douni E, Kunstle G, Bluethmann H, Wendel A, Pfizenmaier K, Kollias G, Grell M. Eur J Immunol 1997;27:2870–5.
[411] Jacobs M, Brown N, Allie N, Chetty K, Ryffel B. Pathobiology 2000;68:68–75.
[412] Kronke M, Schutze S, Scheurich P, Meichele A, Hensel G, Thoma B, Kruppa G, Pfizenmaier K. Cell Signal 1990;2:1–8.
