
Biosynthesis and attachment of novel bacterial polyketide synthase starter units

Bradley S. Moore$^{a}$ and Christian Hertweck$^{b}$

$a$ Division of Medicinal Chemistry, College of Pharmacy, University of Arizona, Tucson, AZ 85721-0207, USA. E-mail: moore@pharmacy.arizona.edu  
$b$ Hans-Knoell-Institute for Natural Products Research, Beutenbergstr. 11a, 07745 Jena, Germany. E-mail: hertweck@pmail.hki-jena.de  

Received (in Cambridge, UK) 5th September 2001  
First published as an Advance Article on the web 6th December 2001  

Covering: up to early 2001.

The biosynthesis and mode of attachment of a wide range of polyketide synthase (PKS) starter units in bacteria are covered in this review. Natural, unnatural, and engineered starter units associated with type I and type II PKSs are reported. The literature through early 2001 is reviewed, and 240 references cited.

|  |  |  |
| --- | --- | --- |
| 1 | Introduction |  |
| 2 | Novel starter units associated with type I PKSs |  |
| 2.1 | Isobutyrate, isovalerate, and 2-methylbutyrate | 3,5-Dihydroxybenzoate (DHBA) |
| 2.2 | Glycolate | Phenylacetate |
| 2.3 | Glycerol-derived starter units | Amino acids |
| 2.4 | Cyclohexanecarboxylate (CHC) | Glycine |
| 2.5 | 3,4-Dihydroxycyclohexanecarboxylate (DHCHC) | β-Alanine |
| 2.6 | Benzoate | 4-Aminobutyrate and derivatives |
| 2.7 | p-Aminobenzoate (PABA) | 3-Amino-2-methylpropionate |
| 2.8 | 3-Amino-5-hydroxybenzoate | β-Phenylalanine |
| 2.9 | 3-Amino-4-hydroxybenzoate (3,4-AHBA) | Proline |
|  |  | Malonate and malonamate |

Born in Honolulu, Hawaii (1966), Bradley S. Moore received his BS in chemistry at the University of Hawaii, where he studied natural products with Professor R. E. Moore. He conducted graduate (PhD 1994 in bioorganic chemistry with Professor H. G. Floss at the University of Washington) and postdoctoral research (1995 with Professor J. A. Robinson at the University of Zurich) on the biosynthesis of bacterial natural products. He began his academic work as a research assistant professor of chemistry at the University of Washington (1996–1999) before moving to the University of Arizona where he is an assistant professor of medicinal chemistry in the College of Pharmacy. His research interests are in exploring and engineering natural product diversity from marine bacteria and in studying the involvement of marine invertebrate-associated microbes for their secondary metabolic capabilities. He has co-authored over 30 publications and is the recipient of the Matt Suffness Young Investigator Award from the American Society of Pharmacognosy (2001).

Christian Hertweck was born in Bonn, Germany, in 1969. After studying chemistry at the University of Bonn, he joined the bioorganic chemistry group of Professor Wilhelm Boland. During his PhD studies at the Kekulé Institute for Organic Chemistry and Biochemistry, Bonn, and at the Max-Planck-Institute for Chemical Ecology, Jena, he worked on the stereoselective synthesis of natural products, with focus on algae sex pheromones and sphingolipids. On completion of his PhD in 1999, he gained a Feodor Lynen fellowship and became Humboldt postdoctoral fellow of Professor Heinz G. Floss and Bradley S. Moore at the University of Washington, Seattle. With Professor Moore he worked on the elucidation of enterocin biosynthesis as well as on combinatorial biosynthesis with iterative polyketide synthases. Since December 2000, he has been habilitand and head of the Bioorganic Synthesis group at the Hans-Knoell-Institute for Natural Products Research, Jena. His current research interests aim at studying microbial biosynthetic pathways and generating molecular diversity through an amalgamation of organic synthesis and genetic engineering.

70 Nat. Prod. Rep., 2002, 19, 70–99  
DOI: 10.1039/b003939j  

This journal is © The Royal Society of Chemistry 2002

naphthalene). The biosynthesis and attachment of a number of uncommon PKS starter units associated with modular type I and iterative type II PKSs are reviewed in Sections 2 and 3, respectively. Type III PKSs are not discussed in this review due to limited information from this new group.

As a consequence of fundamental advances in the genetic engineering of bacterial polyketide biosynthetic genes, it is now possible to generate structurally diverse libraries that are virtually unavailable by conventional synthetic methods. In many cases, PKSs tolerate non-natural primers that result in novel chemistry. Engineering polyketides with unnatural starter units through precursor-directed biosynthesis and metabolic engineering is highlighted in Section 4. General labeling patterns consistently used throughout this report are outlined in Fig. 1.

- isolated methyl carbon from C2 of acetate
- intact acetate unit
- intact propionate unit
- methyl carbon from [methyl-13C] methionine
- intact glycolate unit, including C-O bond
- intact glycine unit, including C-N bond
- glycerol unit

Fig. 1

2 Novel starter units associated with type I PKSs

The loading modules of type I PKSs that utilize common acetate or propionate starter units fall into two groups. The N-terminus of the well-studied erythromycin PKS 6-deoxyerythronolide B synthase 1 (DEBS1) contains the loading module prior to the first two condensing modules and is composed of an acyltransferase (AT) and an ACP. Propionyl-CoA is preferentially loaded by AT_L (the loading AT) and transferred to the loading ACP (ACP_L) before movement to the active site cysteine of the ketosynthase (KS) in module 1 (KS_1) for polyketide assembly. Most other modular PKSs, however, additionally have a modified KS called a KSQ in which the active site cysteine has been mutated into a glutamine (Q) residue. Leadlay and coworkers demonstrated that these loading domains rather load dicarboxylated starter units such as malonyl-CoA and catalyze their decarboxylation directly on the megasynthase. In contrast, type I PKSs that utilize starter units other than acetate or propionate generally attach their specific primers in one of two ways based largely on the nature of their carboxyl group. Starter units synthesized as CoA thioesters appear to be loaded onto their respective PKSs by the erythromycin type AT_L/ACP_L loading didomain. On the other hand, starter units produced as free carboxylic acids seem to be activated and loaded by a non-ribosomal peptide synthetase (NRPS)-like adenylation-thiolation (A-T) didomain at the N-terminus of the PKS. Not surprisingly, the KSQ-type loading domain, which appears to be limited to malonyl-CoA derivatives, is absent from loading modules that specify structurally more complex primer units.

2.1 Isobutyrate, isovalerate, and 2-methylbutyrate

Short branched-chain carboxylic acids derived from the amino acids valine, leucine, and isoleucine commonly serve as fatty acid synthase (FAS) starter units that give rise to the iso and anteiso series of branched-chain fatty acids in bacteria. In a similar fashion, several modular type I PKSs utilize amino acid-derived short branched-chain carbonyl-CoAs such as

Nat. Prod. Rep., 2002, 19, 70–99

isobutyryl-CoA, isovaleryl-CoA, and 2-methylbutyryl-CoA as primer units, including those in the avermectin, myxothiazol, manumycin, and virginiamycin series (Scheme 1).

L-valine / L-isoleucine / L-leucine

[branched-chain amino acid transaminase]

α-ketoglutarate
glutamate

2-ketoisovaleric acid / 2-keto-3-methylvaleric acid / 2-ketoisocaproic acid

[branched-chain α-keto acid dehydrogenase (BCDH) complex]

NAD⁺, CoA
NADH, CO₂

isobutyryl-CoA
(S)-2-methylbutyryl-CoA
isovaleryl-CoA

virginiamycins
avermectins
manumycins
myxothiazol
butyrolactols
tautomycin

**Scheme 1**

The avermectins are related anthelmintic macrolides produced by *Streptomyces avermitilis* that structurally differ at three positions, C5, C22–23, and C26 (Fig. 2).²⁰ The avermectin

![Chemical Structure](https://i.imgur.com/chemical_structure.png)

R² = Et (series a)
Me (series b)

| Avermectin | R¹ | R² | X—Y |
|------------|----|----|------|
| Ala        | Me | Et | CH≡CH |
| Alb        | Me | Me | CH≡CH |
| A2a        | Me | Et | CH₂—CHOH |
| A2b        | Me | Me | CH₂—CHOH |
| B1a        | H  | Et | CH≡CH |
| B1b        | H  | Me | CH≡CH |
| B2a        | H  | Et | CH₂—CHOH |
| B2b        | H  | Me | CH₂—CHOH |

**Fig. 2**

biosynthetic gene cluster (*ave*) was recently cloned and sequenced, containing 18 open reading frames (ORFs) that span a distance of 82 kb.²¹ Four large type I PKS genes *aveAl–4* organized into 12 modules are convergently transcribed as two groups. Labeling studies demonstrated that the avermectin starting acyl group is derived from the α-methyl carboxylates 2-methylbutyrate or isobutyrate and gives rise to the structural differences at C26 in the “a” and “b” components, respectively.²²,²³ These carboxylic acids arise from the catabolism of L-isoleucine and L-valine via their respective α-keto acids (Scheme 1).²⁴,²⁵ A mutant strain that lacks branched-chain α-keto acid

dehydrogenase (BCDH) activity is only able to produce the natural avermectins when supplemented with (S)-(+) -2-methylbutyrate or isobutyrate, but not with L-isoleucine, L-valine, 2-keto-3-methylvalerate, or 2-ketoisovalerate.²⁴ The BCDH functions to provide the CoA thioesters (S)-2-methylbutyryl-CoA and isobutyryl-CoA for the starter unit of the avermectin PKS. The fatty acid profile of the *S. avermitilis bkd* mutant is consequently devoid of branched-chain fatty acids and as a consequence modulates its membrane fluidity with a significant increase in unsaturated straight-chain fatty acids.²⁶ *S. avermitilis* harbors at least two gene sets encoding the Ela, Elβ, and E2 subunits of the BCDH complex, the *bkdABC*²⁷ and the *bkdFGH*²⁸ gene clusters that are separated in the genome by approximately 12 kb. Disruption of *bkdF* resulted in a *S. avermitilis bkd* mutant lacking BCDH activity and the ability to produce natural avermectins without supplementation with (S)-2-methylbutyrate or isobutyrate.²⁸ Alternatively, inactivation of the *bkdABC* gene cluster did not result in a similar loss of activity, suggesting that these genes are either silent or that their functions are complemented by *bkdFGH* or other *S. avermitilis* genes. Taking advantage of the *S. avermitilis bkd* mutant, precursor directed biosynthesis with alternative carboxylic acid starter units resulted in the generation of a wide range of novel avermectin derivatives (see Section 4). The catabolism of short branched-chain carbonyl-CoAs has been studied in *S. avermitilis* and *Streptomyces coelicolor* and involves conversion into their α,β-unsaturated derivatives by an acyl-CoA dehydrogenase (AcdH) with broad substrate specificity.²⁹

The myxobacterial mixed PKS/NRPS product myxothiazol 1

![Myxothiazol Structure](https://i.imgur.com/myxothiazol_structure.png)

utilizes the leucine-derived starter group isovaleryl-CoA.³⁰ The biosynthetic gene cluster (*mta*) of this electron transport inhibitor has been cloned and sequenced and contains five PKS and three NRPS modules within the six genes *mtaB–G*.³¹ In comparison to the standard organization of the avermectin loading module of avermectin synthase 1 (AVES1), the modular organization of the 1 loading module in MtaB is quite different (Fig. 3). Instead of the starter unit specific AT and ACP preceding the first condensation module as found in AVES1 and DEBS1, the MtaB loading module and module 1 are intermixed. One AT presumably recognizes and transfers the starter unit isovaleryl-CoA to the loading (first) ACP, while the second AT presumably recognizes and transfers the extender unit malonyl-CoA to the second ACP. Müller and coworkers were not able to determine the specificities of these myxobacterial ATs by comparing them with known actinomycete ATs.

The manumycin-group compounds possess broad ranges of biological activities and are produced by several *Streptomyces* spp.³² These metabolites contain two short polyene chains of probable type I PKS origin. The structural diversity within the manumycins is largely based on the variation of the “upper” chain, involving mixed starter and extender units. On the basis of biosynthetic studies with manumycin A 2 and asukamycin 3,³³ the PKS(s) involved in the assembly of the “upper” chains utilize a mixture of linear, branched-chain, and cyclic carboxylic acid starter units, thus implying a very relaxed priming

72 Nat. Prod. Rep., 2002, 19, 70–99

Fig. 3

mechanism of the PKS. Two groups have been designated within the manumycin family based on the structural diversity of the “upper” chain. Manumycins such as 2 appear to utilize exclusively a butyrate starter unit followed by extensions with both malonyl-CoA and methylmalonyl-CoA, whereas other members of the series such as 3–6 include those with branched-chain (Scheme 1) and cyclohexylcarboxylic acid (see Section 2.4) starter units extended solely by malonyl-CoA. The fatty acid profile of the manumycin-group producing strain *Streptomyces nodosus* reflects this mixed use of starter units with mixtures of linear, branched-chain, and ω-cyclohexyl fatty acids.³⁴

Valine-derived isobutyryl-CoA is the PKS starter unit in several other actinomycete systems, including the linear polyketide tautomycin³⁵ 7, mixed polyketide-peptides belonging to the virginiamycin family such as virginiamycin M₁³⁶ 8, pristinamycin IIₚ³⁷ and antibiotic A2315A,³⁸ the macrolide antibiotic LL-F28249-α (nemadectin) 9,³⁹ and possibly in the antifungal polyketides butyrolactols A 10 and B 11.⁴⁰ The possibility of a *t*-valeryl starter unit derived from *t*-leucine is very attractive for the unusual terminal *t*-butyl group in 10, although it is probably derived from an isobutyrate starter unit that is subsequently methylated by S-adenosyl-L-methionine. The bicyclic depsipeptide salinamide A 12 from a marine streptomycete, on the other hand, contains two diketide units, one of which is initiated with isobutyrate and the other with 2-methylbutyrate-derived tiglic acid (2-methylbut-2-enoic acid).⁴¹ Short branched-chain carboxylates are also involved as starter units for cyanobacterial metabolites, including barbamide⁴² and apratoxin A.⁴³

### 2.2 Glycolate

Although glycolate has not yet been reported as a starter unit for a bacterial PKS, it serves as a starter unit in a few dinoflagellate polyketides such as the dinophysistoxin family of polyethers and possibly the antifungal macrolide goniodomycin A.⁴⁴ Glycolate or its methoxy derivative may, however, serve as a starter unit in the macrolide apotolidin 13 in *Nocardiopsis* sp.,⁴⁵ yet biosynthetic experiments have not been reported for this selective cytotoxic agent.

### 2.3 Glycerol-derived starter units

The biosynthesis of a group of boron-containing antibiotic macrodiolides in two streptomycetes and a myxobacterium involves glycerol-derived starter units. Feeding experiments demonstrated that aplasmomycin 14, boromycin 15, and tartrolon B 16 are acetate-derived polyketides whose methyl groups originate from methionine.⁴⁴ In each case, glycerol was shown to be an intact precursor of the three-carbon starter unit. Floss and coworkers demonstrated that the two hydrogens of the pro-R-hydroxymethyl group of (*1R*,*2R*)-[1-*²H*,*³H*]glycerol give rise to a chiral methyl group of *S* configuration at C17 of 14.⁴⁶ The stereochemical outcome of this feeding experiment rules out serine, methylglyoxal, pyruvate, and compounds derived

from these as the glycerol-derived starter unit. Rather, phosphoglycerate or phosphoenolpyruvate are the two proposed candidates. Neither, however, has been observed as a CoA ester or as a PKS starter unit.

tautomycin 7

virginiamycin M₁ 8

LL-F28249-α 9

butyrolactol A 10 R = Me  
butyrolactol B 11 R = H  

2-methyl-[1-¹³C]butyrate (Δ)  
2-methyl-[1-¹³C]but-2-enoyl-  
NAC thioester (O)

salinamide A 12

apoptolidin 13

C3 ?  

aplamomycin 14  

boromycin 15 valine  

tartrolon B 16  

### 2.4 Cyclohexanecarboxylate (CHC)

The streptomyces antibiotics asukamycin⁴⁷ 3 and several members of the phoslactomycin family⁴⁸⁻⁵⁰ 17–19 contain a saturated, monosubstituted cyclohexane ring at the terminal end of alkyl chains of probable type I PKS origin. In the 3-producing strain *S. nodosus*, the cyclohexane ring and adjacent carbon on the “upper” polyene chain originate from the shikimic acid (SA)-derived³³ starter unit cyclohexanecarboxylic acid (CHC) (Scheme 2).⁵¹ Although the conversion of SA into CHC has not been delineated in the asukamycin or phoslactomycin series, this biosynthetic pathway has been well characterized at the chemical, biochemical and genetic levels in two other model systems, which are reviewed here.

The side chain of the polyketide ansatrienin A⁵² 20 produced by *Streptomyces collinus* contains a CHC residue, while the thermoacidophilic bacterium *Alicyclobacillus acidocaldarius* contains large proportions of ω-cyclohexyl fatty acid-containing lipids⁵³ that are primed with CHC-CoA.⁵⁴ The stereochemistry of each reaction in the conversion of SA into CHC-CoA was delineated with the combined knowledge of the stereochemical fate of the carbon-bound hydrogens of SA in *S. collinus*⁵⁵ with the stereochemical information determined in two *A. acidocaldarius* blocked mutants.⁵⁴,⁵⁶,⁵⁷ This nine-step pathway involves a series of dehydrations and double bond reductions interspersed in such a way that no intermediate is ever aromatic (Scheme 3). As the CHC biosynthetic pathway was recently reviewed in detail by Floss and Moore,⁵⁸,⁵⁹ this

Nat. Prod. Rep., 2002, 19, 70–99

[U-13C]glycerol

$\begin{bmatrix}
\text{OH} \\
\text{HOOC} & \text{OH} \\
\text{OH} & \text{OH}
\end{bmatrix}$

SA

$\downarrow$

$\begin{bmatrix}
\text{HOOC} \\
\text{CHC}
\end{bmatrix}$

$\rightarrow$

$\begin{bmatrix}
\text{3}
\end{bmatrix}$

Scheme 2

$\begin{bmatrix}
\text{H}_2\text{N} & \text{O} \\
\text{HOOC} & \text{OH} \\
\text{OH} & \text{OH} \\
\text{Me} & \text{R}
\end{bmatrix}$

phoslactomycin B 17 R = H

phoslactomycin E 18 R = O₂C-C₆H₁₁

2-pyrone derivative I-i 19 R = O₂C-C₆H₁₁

$\begin{bmatrix}
\text{Me} & \text{O} \\
\text{HO} & \text{NH} \\
\text{Me} & \text{O} \\
\text{OMe} & \text{O}
\end{bmatrix}$

ansatrienin A 20

---

$\begin{bmatrix}
\text{COOH} \\
\text{H}_a & \text{H}_b \\
\text{OH} & \text{OH}
\end{bmatrix}$

SA

$\begin{bmatrix}
\text{COSCoA} \\
\text{H}_a & \text{OH}
\end{bmatrix}$

21

$\begin{bmatrix}
\text{COSCoA} \\
\text{H}_a & \text{OH}
\end{bmatrix}$

23

$\begin{bmatrix}
\text{COSCoA} \\
\text{H}_a & \text{OH}
\end{bmatrix}$

24

$\begin{bmatrix}
\text{COSCoA} \\
\text{H}_a & \text{OH}
\end{bmatrix}$

25

$\begin{bmatrix}
\text{COSCoA} \\
\text{H}_d & \text{H}_c
\end{bmatrix}$

27

$\begin{bmatrix}
\text{COSCoA} \\
\text{H}_d & \text{H}_c
\end{bmatrix}$

26

$\begin{bmatrix}
\text{COSCoA} \\
\text{H}_d & \text{H}_c
\end{bmatrix}$

20

CHC-CoA

Scheme 3

---

downstream of $chcA$ have been tentatively assigned functional roles on the basis of sequence homology (Scheme 3). Noticeably absent from this gene set is the 2-cyclohexenylcarbonyl-CoA isomerase-encoding gene ($chcB$) whose product catalyzes the penultimate reaction in the CHC pathway, the isomerization of 26 to 27. The 28 kDa ChcB was purified from *S. collinus* and the corresponding gene $chcB$ does not appear to be associated with the ansatrienin biosynthetic gene cluster. Disruption of $chcB$ resulted in the loss of isomerase activity and ansatrienin and ω-cyclohexyl fatty acid synthesis. ChcB is apparently present in other *Streptomyces* spp., indicating that this Δ³,Δ²-enoyl-CoA isomerase with broad substrate specificity may additionally serve a primary metabolic role.

### 2.5 3,4-Dihydroxycyclohexanecarboxylate (DHCHC)

The streptomycete immunosuppressants rapamycin FK520 (ascomycin) and FK506 (tacrolimus) are structurally related macrolides containing a more oxygenated CHC moiety derived from the proximal starter unit (1R,3R,4R)-3,4-dihydroxycyclohexanecarboxylic acid (DHCHC). Feeding experiments with labeled precursors

Nat. Prod. Rep., 2002, 19, 70–99

confirmed the polyketide origin of these macrolides. 72,73 The cloning and complete sequence analysis of the rapamycin 74–76 and FK520 77 biosynthetic gene clusters and a partial sequence of the FK506 78,79 gene cluster have been reported. Each cluster encodes a modular PKS as well as additional processing enzymes. Genes involved in the synthesis of the DHCHC starter unit, however, are not clustered with the rapamycin and FK520 biosynthetic gene clusters and have yet to be identified. The DHCHC-derived unit in FK506 undergoes O-methylation after polyketide assembly. 80

![Chemical Structure](https://i.imgur.com/1.png)

rapamycin 28

![Chemical Structure](https://i.imgur.com/2.png)

FK 520 29, R = CH₂CH₃  
FK 506 30, R = CH₂CH=CH₂  

The DHCHC moiety in rapamycin and FK520 is derived from all seven carbon atoms of SA, 73,81 as is the case for CHC (Section 2.4). Detailed feeding experiments have elucidated the steps between SA and DHCHC for rapamycin 82,83 and FK506 84,85 (Scheme 4). The pathway shares the first step with the CHC biosynthetic pathway, anti elimination of H₆R from SA, 83,85 but diverges at the stage of 21. Both 21 and 4,5-dihydroxycyclohex-1-enecarboxylic acid 32 were incorporated into FK520, implicating 31a or its C1 epimer 31b as a likely intermediate. However, deuterium labeling revealed that the enoyl reductions in this pathway proceed with a different stereochemistry than those in the pathway from SA to CHC (Scheme 3). Since the C1 configuration of the inferred intermediate 31 is not known, two alternative stereochemical pathways, one proceeding through 31a and the other through 31b, could not be distinguished (Scheme 4). 83,84 Consistent with the difference in enoyl reduction stereochemistry, the analysis of the rapamycin, FK520, and FK506 biosynthetic gene clusters suggest that the enoyl reductase (ER) in the initiation module is responsible for the last double bond reduction in the starter unit. 76 This is in contrast to the CHC pathway in which the l-cyclohexenylcarbonyl-CoA reductase is a distinct, separate enzyme with no homology to enoyl reductases. 86 Lowden et al.

76 Nat. Prod. Rep., 2002, 19, 70–99

![Scheme 4](https://i.imgur.com/3.png)

Scheme 4

recently reported that 32, rather than DHCHC, is the true starter unit for rapamycin, and by analogy FK520 and FK506, PKSs. 83 Precursors of 32 are speculatively formed as free acids on the basis of high specific incorporations before activation as an adenosine monophosphate by the NRPS-like A domain and loading onto the ACP domain of the PKS (Scheme 5).

![Scheme 5](https://i.imgur.com/4.png)

Scheme 5

Reduction by the ER domain to enzyme bound 32 and transfer of DHCHC to the KS₁ domain initiates chain elongation by the rapamycin, FK506, and FK520 PKSs. The direct incorporation of DHCHC into rapamycin suggests a broad substrate

specificity of the loading module. Furthermore, the carboxyl activation and stereochemical differences between the CHC and DHCHC pathway intermediates suggest that these two related pathways evolved independently.

### 2.6 Benzoate

The antifungal macrolide antibiotic soraphen A<sup>87</sup> 33 from the myxobacterium *Sorangium cellulosum* contains a starter unit putatively derived from benzoic acid (BA). Feeding experiments verified that the starter unit is derived from glycerol in a manner largely consistent with the shikimate pathway.<sup>88</sup> Hill *et al.* acknowledge unpublished data for the direct incorporation of BA,<sup>88</sup> while Höfle and Reichenbach report that phenylalanine but not BA or its derivatives is accepted as a precursor.<sup>89</sup> The soraphen biosynthesis gene cluster has been cloned and sequenced.<sup>90,91</sup> The complete DNA sequence, which was reported in a patent, indicates that 33 is synthesized by the products of two large type I PKS genes *sorA-B* that are organized into nine modules. The organization of the loading module and the first three extended modules in SorA resembles that of MtaB (Fig. 3) in which the loading AT resides within the first extender module. Although none of the genes or biosynthetic intermediates involved in the assembly of the BA residue in 33 have been identified, BA biosynthesis in *S. cellulosum* probably follows one of the two recently described bacterial BA biosynthetic pathways. These include the plant-like biosynthesis of benzoyl-CoA from L-phenylalanine as described in Section 3.4 for the biosynthesis of the type II PKS product enterocin<sup>92</sup> and the anaerobic pathway to benzoyl-CoA involving two successive α-oxidative decarboxylations of phenylpyruvate.<sup>93</sup>

![Structure of soraphen A 33](attachment:structure_soraphen_A_33.png)

Although biosynthetic studies have not been reported, the myxobacterial antifungal and cytotoxic antibiotics crocacinins A–C may utilize a benzoate or a trans-cinnamate starter unit.<sup>94</sup>

### 2.7 p-Aminobenzoate (PABA)

Candididin 34 and FR-008 are related heptaene macrolides whose aglycons are large macrolactones containing a p-aminocetophenone group derived from a p-aminobenzoic acid (PABA) starter unit.<sup>95</sup> Although DNA sequences for these polyene PKS gene clusters have not been reported to date, gene clusters of other polyene antibiotics have recently been sequenced, revealing a modular type I PKS organization.<sup>96,97</sup> The FR-008 PKS has been cloned and physically mapped to span ~105 kb, which is consistent with the expected 21 modules required for the synthesis of the macrolactone.<sup>98</sup> Feeding experiments revealed that the starter unit is PABA, probably as its CoA thioester. The PABA synthase gene *pabAB* of the 34-producing *Streptomyces griseus*, located between genes putatively involved in 34 biosynthesis, has been cloned, sequenced, and expressed.<sup>99,100</sup> The *pabAB* gene product possesses two catalytic domains: the N-terminus domain has glutamine aminotransferase activity whereas the C-terminus has PABA synthase activities. Southern hybridization experiments with this gene have been used to identify other PABA initiated polyene producing *Streptomyces* strains.<sup>101</sup> In Enterobacteria three discrete enzymes are involved in PABA biosynthesis. PabA and PabB, which share high sequence identity with the *S. griseus* bifunctional PabAB, convert chorismic acid into p-amino-p-deoxychorismic acid 35 and the lyase PabC converts this intermediate into PABA (Scheme 6).<sup>102</sup>

![Scheme 6](attachment:scheme_6.png)

The *pabC* gene of *S. griseus*, if present, is apparently separate from *pabAB*.<sup>100</sup> A partially sequenced gene downstream from *pabAB* may encode a PABA-CoA ligase, which would be required for PABA activation for use as a PKS starter unit. The production of 34 and *pabAB* gene expression is regulated by phosphate in *S. griseus* and in the readily transformable *Streptomyces acrimycini* JI2236.<sup>103</sup>

### 2.8 3-Amino-5-hydroxybenzoate

The large family of ansamycin<sup>104</sup> antibiotics contains a novel structural element, a mC<sub>7</sub>N unit, which is biosynthetically derived from 3-amino-5-hydroxybenzoic acid (3,5-AHBA). This group of macrolactam natural products is produced by a variety of microorganisms and plants and is subdivided into two subgroups based on the structure of the 3,5-AHBA-derived aromatic moiety. The benzenic ansamycins are mainly cytotoxic against eukaryotes and include ansatrienin A 20, ansamitocin P3 36, geldanamycin 37, and herbimycin A. Naphthalenic ansamycins, on the other hand, include rifamycin B 38, naphthomycin A 39, and tolypomycin Y, and are antibiotics that are active particularly against Gram-positive bacteria and *Mycobacterium tuberculosis*.

The biosynthesis of 3,5-AHBA has been studied at the chemical, biochemical, and genetic levels in organisms pro-

![Structure of candididin 34](attachment:structure_candididin_34.png)

Nat. Prod. Rep., 2002, 19, 70–99

ansamitocin P3 36

geldanamycin 37

rifamycin B 38

naphthomycin A 39

a preference for 3- and 3,5-disubstituted benzoates that closely resemble the biological substrate 3,5-AHBA, a variety of substituted benzoates were accepted, thus suggesting considerable substrate tolerance. Downstream of *rifE* lies a sub-cluster of genes responsible for the biosynthesis of the starter unit 3,5-AHBA (Fig. 4 and Table 1).<sup>107</sup> Floss and coworkers have probed the biosynthetic pathway of 3,5-AHBA formation through

**Scheme 7**

**Fig. 4** Organization of the 3,5-AHBA biosynthetic regions of the rifamycin B 38 (A), ansatrienin A 20 (B), the putative naphthomycin A 39 (C), and mitomycin C 40 (D) gene clusters. Each ORF associated with 3,5-AHBA formation is shown as a shaded arrow pointed in the direction of transcription. The proposed functions of each ORF is given in Table 1.

**Table 1** Proposed functions of homologous ORFs in the rifamycin B 38 (*rif*), ansatrienin A 20 (*ans*), naphthomycin A 39 (*nap*), and mitomycin C 40 (*mit*) biosynthetic gene clusters

| 3,5-AHBA biosynthesis gene | Proposed function |
|-----------------------------|--------------------|
| *rifG*, *ansA*, *napC*, *mitP* | aDHQ synthase |
| *rifH*, *ansI*, *napD*       | aDAHP synthase   |
| *rifI*, *napE*, *mitT*       | Aminoquinine dehydrogenase |
| *rifJ*, *ansE*, *mmcF*       | aDHQ dehydratase |
| *rifK*, *ansF*, *napF*, *mitA* | 3,5-AHBA synthase |
| *rifL*, *ansG*, *napG*, *mitG* | Oxidoreductase   |
| *rifM*, *ansH*, *napH*, *mitJ* | Phosphatase      |
| *rifN*, *ansB*, *napI*, *mitS* | Kinase           |

ducing various ansamycins and mitomycin C<sup>105</sup> 40, and was recently reviewed in the context of rifamycin B biosynthesis in *Amycolatopsis mediterranei* by Floss.<sup>58,106</sup> This section will therefore largely highlight new results published in the past few years.

Molecular analysis of ~95 kb of DNA containing the *rif* biosynthesis gene set in *A*. *mediterranei* S699 established the clustering of the rifamycin type I PKS and AHBA biosynthesis genes.<sup>107</sup> The five PKS proteins encoded by *rifA–rifE* contain ten separate modules that are typical of type I PKSs. RifA contains at its amino terminus the loading domain for AHBA, which consists of a NRPS-like A–T didomain (Scheme 7).<sup>108</sup> Adenylation and thiolation activities were reconstituted *in vitro* and shown to be independent of CoA, which was contrary to earlier proposals involving a CoA ligase in the loading module. Although the loading module exhibited

78 Nat. Prod. Rep., 2002, 19, 70–99

feeding experiments with stable isotopes, cell-free assays,^{109} and mutational analyses.^{10} Their results demonstrated the operation of a new variation of the shikimate pathway in the formation of 3,5-AHBA starting from phosphoenolpyruvate (PEP) and erythrose 4-phosphate (E4P) or a nitrogen-containing equivalent to 3,4-dideoxy-4-amino-D-arabino-heptulosonate-7-phosphate (aDAHP). The pathway then continues via 5-deoxy-5-amino-3-dehydroquinic acid (aDHQ) and 5-deoxy-5-amino-3-dehydroshikimic acid (aDHS) to 3,5-AHBA (Scheme 8). The terminal enzyme 3,5-AHBA synthase (RifK),

which catalyzes the aromatization of aDHS to 3,5-AHBA, is a dimeric, pyridoxal 5′-phosphate (PLP) dependent enzyme.^{11} Mechanistic studies demonstrated that the enzyme-bound PLP forms a Schiff's base with the amino group of aDHS and catalyzes the α,β-dehydration and stereospecific 1,4-enolation of the substrate (Scheme 9). The structure of 3,5-AHBA synthase from *A. mediterranei* was determined to 2.0 Å resolution with bound PLP and with PLP and the substrate analogue inhibitor gabaculine.^{12} The structural work showed that the enzyme belongs to the aspartate family of PLP-dependent enzymes and confirmed the proposed mechanism depicted in Scheme 9. The active site Lys188 forms an internal aldimine linkage with the PLP cofactor, and the active site is further composed of residues from two subunits of 3,5-AHBA synthase, thus indicating that the enzyme RifK is active as a dimer. Mutational analysis and reconstituted expression of *rifG-rifN* demonstrated that seven of these genes (excluding the aminoquinolate dehydrogenase gene *rifI*) are indeed necessary and sufficient for 3,5-AHBA formation.^{10} Genes involved in 3,5-AHBA formation were individually inactivated and the resulting mutants were analyzed by incubating cell-free extracts with known intermediates. The gene products of *rifH*, -G, and -J resemble shikimic acid biosynthesis enzymes and their inactivation led to mutants that produced rifamycin B 38 at 1%, ~100%, and 10%, respectively, of wild type yields, suggesting functional substitution to varying degrees with their corresponding normal shikimate pathway homologues (Fig. 4 and Scheme 8). The *rifL*, -M, and -N genes, which are located downstream of the *rifK* gene, are essential for 3,5-AHBA formation yet are not related to any shikimate pathway gene products. These genes were shown to be involved in the formation of intermediates leading to the first unique pathway intermediate aDAHP. Coexpression of *rifG-rifN* and -J genes in the engineered host strain *S. coelicolor* YU105 led to significant production of 3,5-AHBA, thus confirming that these seven genes are sufficient for 3,5-AHBA synthesis. The involvement of the individual *rif* genes in 3,5-AHBA formation was further elaborated through complementary expression studies involving incomplete expression cassettes, indicating that all of these genes encode proteins with catalytic rather than regulatory functions. The formation of aDAHP and the introduction of nitrogen are not presently clear due to the requirement for the three genes *rifL*, -M, and -N of unknown function in addition to the aDAHP synthase gene *rifH*. Nonetheless,

---

**Scheme 8**

![Chemical Structure](https://i.imgur.com/structure_scheme_8.png)

---

**Scheme 9**

![Mechanism Diagram](https://i.imgur.com/mechanism_scheme_9.png)

---

Nat. Prod. Rep., 2002, 19, 70–99

rifK has been used as a genetic probe to identify homologous 3,5-AHBA biosynthesis gene sets associated with the *S. collinus* ansamycins ansatrienin A 20 and naphthomycin A 39 and mitomycin C 40 from *S. lavendulae*. The relative organization of the 3,5-AHBA biosynthetic regions of the 20 and 38-40 gene clusters is shown in Fig. 4. Furthermore, degenerate PCR primers based on 3,5-AHBA synthase have been used to amplify a PCR product of high (85%) amino acid sequence homology to rifK in the ansamycin antibiotic rubradirin producing strain *Streptomyces achromogenes* var. *rubradiris* NRRL 3061.

### 2.9 3-Amino-4-hydroxybenzoate (3,4-AHBA)

As discussed in Sections 2.1 and 2.4, the “upper” chains of the manumycin group antibiotics 2–6 utilize a mixture of linear, branched-chain, and cyclic carboxylic acid starter units. The “upper” polyketide chain is linked through an mC₇N unit, which serves as the starter unit for the “lower” polyketide residue. Based on the structure of the central mC₇N unit, two types of manumycins exist. Type I manumycins have an oxirane at C5/C5/C6 whereas those with a hydroxyethylene group at C5/C6 belong to the type II manumycins. Hu and Floss demonstrated through production time-course studies and the biosynthetic conversion of ¹³C-labeled (type I) 3 into type II 3 that type II manumycins are not synthesized independently but are rather derived from the corresponding type I molecules (Scheme 10).

**Scheme 10**

![Chemical Structure](https://i.imgur.com/structure1.png)

Unlike the 3,5-AHBA-derived mC₇N unit in ansamycin antibiotics (Section 2.8), the mC₇N unit in the manumycins is specifically derived from 3,4-AHBA (Scheme 11) via the tricarboxylic acid cycle. Extensive feeding experiments established that the manumycin mC₇N unit originates from a 4-carbon unit derived from succinic acid and a 3-carbon unit derived from a glycerol-based triose phosphate, thus ruling out a shikimate pathway origin. The same labeling pattern and biosynthetic intermediacy of 3,4-AHBA has also been established for the iron-chelating metabolite 4-hydroxy-3-nitrosobenzamide 41 from *Streptomyces murayamaensis*. The nature of the 4-carbon unit was explored in *S. murayamaensis* with [1-¹³C]methionine, [1-¹³C]homoserine, and [4-¹³C]oxalacetate and revealed the intermediacy of only oxalacetate in the biosynthesis of 3,4-AHBA. Feeding experiments from the Floss and Gould groups suggest that oxalacetate directly condenses with either pyruvate or phosphoenolpyruvate to form 4-carboxy-4-hydroxy-2-oxoadipate which is further converted in a number of unspecified reactions into 3,4-AHBA (Scheme 12).

**Scheme 11**

![Chemical Structure](https://i.imgur.com/structure2.png)

**Scheme 12**

![Chemical Structure](https://i.imgur.com/structure3.png)

### 2.10 3,5-Dihydroxybenzoate (DHBA)

Kendomycin 42 is a structurally novel carbocyclic ansa-compound possessing antimicrobial and cytotoxic properties. General feeding experiments with the *Streptomyces violaceoruber* producing strain elucidated the origin of all carbon and oxygen atoms in 42. An acetate-derived penta-substituted benzoate is proposed as the starter unit to polyketide assembly following a type I PKS mechanism. Bode and Zeeck speculate that the symmetrically labeled benzoate is synthesized by a type III PKS that provides the intermediate 3,5-dihydroxyphenylacetate, which is the proposed intermediate of the non-proteinogenic amino acid 3,5-dihydroxyphenylglycine residue of the vancomycin type glycopeptides. Decarboxylation to 3,5-dihydroxybenzoate (DHBA) followed by oxidation and methylation would provide the proposed starter unit 2,3,5,6-tetrahydroxy-4-methylbenzoyl-CoA 43 (Scheme 13). This report is the first to speculate on the probability that a type III PKS product may serve as the starter unit for a type I PKS. The structural nature of the benzoate starter molecule was investigated in a series of precursor-directed biosynthetic experiments with unlabeled benzoate and 3,5-dihydroxybenzoate free acids and thioesters, but levels of 42 production were not increased nor were alternative starter units incorporated. Rather the N-acetylcysteamine thioester analogs as well as thiol nucleophiles gave modified structures at the electrophilic center C20. Oxidation of the starter unit to a benzquinone intermediate likely facilitates the ring closure of the carbocycle by an aldol condensation with the carboxyl terminus (Scheme 14).

The 60-membered macrolide quinolidomycin A₁¹²³,¹²⁴ 44 from the actinomycete *Micromonospora* sp. JY16¹²⁵ also contains a similarly structured benzoquinone chromophore that may also be derived from DHBA. Biosynthetic studies, however, have not been reported for the quinolidomics to support this hypothesis. Although 44 inhibited the growth of various tumor cells, the dihydro derivative quinolidomycin A₂ 45 lacked cytotoxicity, thus highlighting the structural importance of the starter unit towards biological activity.¹²⁵

### 2.11 Phenylacetate

A number of polyketides carrying a monosubstituted benzene ring utilize a phenylalanine-derived starter unit, most probably as phenylacetate or phenylacetyl-CoA rather than BA (Section 2.6). Feeding experiments with stable isotopes demonstrated that the starter unit in the myxobacterial macrolide ripostatin A 46 from *Sorangium cellulosum* is derived from phenylalanine.¹²⁶ Direct incorporation of the expected intermediate phenylacetate was, however, not observed. In a similar fashion, the starter unit of the polyketide moiety (2S,3S,8S,9S)-3-amino-9-methoxy-2,6,8-trimethyl-10-phenyl-4,6-decadienoic acid (Adda) from the peptide hepatotoxins microcystin-LR¹²⁷ 47 and nodularin¹²⁸ 48 was only enriched by phenylalanine and not phenylacetate. A further cyanobacterial metabolite, the potent antitumor agent cryptophycin 1 49, contains a probable phenylacetate-derived starter group in the polyketide fragment.¹²⁹ Although the putative phenylacetate starter unit is probably synthesized from L-phenylalanine via phenylpyruvate, its non-incorporation in all three tested systems indicates that the carboxylic acid may not be a free intermediate.

The microcystin biosynthetic gene cluster was recently cloned and sequenced from two strains of the cyanobacterium *Microcystis aeruginosa* independently by the Neilan¹³⁰ and Shirai¹³¹,¹³² groups showing a mixed PKS/NRPS system. The structural organization of the *mycDEFG* operon suggests its involvement in Adda biosynthesis and the incorporation of Adda and glutamic acid into the microcystin molecule. The NRPS-like loading module of MycG is related to that in rapamycin (Section 2.5) and rifamycin (Section 2.8). This didomain similarly contains adenylation and thiolation regions.

[1-13C]phenylacetate

ripostatin A 46

C2, C3 and Ph of L-phenylalanine

Adda

microcystin-LR 47

C2, C3 and Ph of L-phenylalanine

Adda

nodularin 48

cryptophycin 1 49

McyG
loading module 1
A ACP KS AT CM KR ACP

Fig. 5

polyketide-peptide metabolites that are synthesized by mixed NRPS/PKS modular enzymes. Several of the examples below include the macrolactams in which the amino group of the starter molecule is involved in an amide linkage with the terminal carboxyl group, a structural feature common to the ansamycins (Section 2.8).

### 2.12.1 Glycine

The biosynthesis of tolytoxin 50, a scytophycin-like polyketide from the cyanobacterium *Scytonema mirabile*, involves an intact glycine starter unit that is extended by 15 acetate units. All of the methyl groups are derived from the methyl group of methionine. Incorporation studies with \({}^{13}\)C- and \({}^{2}\)H-labeled glycine suggest that the starter unit of the 17-membered carbocyclic ring of lankacidin 51 is also formed from glycine. The proposed biosynthesis of this streptomyces antibiotic involves ring-contraction of an 18-membered ring intermediate by a Favorskii-like rearrangement as shown in Scheme 15. Support for this mechanism comes from the loss of both deuteriums from C2 of the glycine starter unit. A third polyketide involving glycine as a starter unit is the antibiotic myxopyronin A 52 from *Myxococcus fulvus*. The carbon

and appears to activate phenylacetate in an NRPS manner before its transfer into the Adda PKS pathway (Fig. 5). The thiolation domain may also be termed an ACP on the basis of its assumed function in the transfer of the aromatic carboxylic acid.

### 2.12 Amino acids

Numerous PKSs utilize amino acids and their derivatives as starter units that are presumably loaded by the action of an NRPS-like loading didomain such as that seen above for DHCHC (Section 2.5), 3,5-AHBA (Section 2.8) and phenylacetate (Section 2.11). Although the below examples are without biochemical and genetic support, this mixed NRPS/PKS model is supported by numerous examples of hybrid

tolytoxin 50

lankacidin C 51

Scheme 15

82 Nat. Prod. Rep., 2002, 19, 70–99

skeleton appears to be derived from two polyacetate chains, one of which incorporates glycine as its starter group.

### 2.12.2 β-Alanine

The fluvirucins are a family of 14-membered macrolactam antibiotics identified from several *Actinomadura* strains. These independently isolated antifungal agents are arranged into two homologous groups depending on the position of sugar residues at either C3 or C9 (as in fluvirucin B₁ (SCH 38516) 53).¹³⁸,¹³⁹ In both cases, feeding experiments yielded complementary results as summarized in Scheme 16.¹⁴⁰,¹⁴¹ The aglycones are largely constructed from acetate, propionate, and butyrate units, whereas the starter unit presumably derives from β-alanine or its equivalent. The amide nitrogen of the macrolactam was labeled from [¹⁵N]aspartate,¹⁴⁰ and carbons 11–13 also probably derive from this source. Intact acetate labeled C11–C12 whereas C2 of acetate labeled both C12 and C13, suggesting the involvement of the citric acid cycle in the formation of the starter unit. Oxaloacetate may be transaminated to aspartate and incorporated either directly or after decarboxylation to β-alanine.

#### Scheme 16

![Scheme 16](https://i.imgur.com/1234567.png)

### 2.12.3 4-Aminobutyrate and derivatives

A number of polyketides utilize starter units derived from ornithine or arginine, probably as 4-aminobutanoyl-CoA or the guanidino analogue. The marginolactones, which include the desertomycins and the oasomycins from *Streptoverticillium baldacii*, are macrolactones possessing a side chain derived from arginine or ornithine.¹⁴² The biosynthetic building blocks of the marginolactone oasomycin B 54 have been determined and show that the biosynthesis of this macrolide is probably initiated by ornithine-derived 4-aminobutanoyl-CoA and chain extended in a type I PKS fashion with 12 acetate and nine propionate units (Scheme 17).¹⁴³ Labels from ornithine, aspartic acid, glutamic acid, and 4-aminobutanoic acid were all incorporated into the polyketide starter unit of 54 and desertomycin A 55. The biosynthetic relationship between the various desertomycins and oasomycins, which differ in the nature of the starter unit-derived side chain at C41, was examined by analyzing the fermentation time course.¹⁴² Desertomycins A 55 and B 56, the first detectable biosynthetic intermediates, undergo post-PKS modifications to give lactonization of the side chain (Scheme 18).

#### Scheme 17

![Scheme 17](https://i.imgur.com/8901234.png)

#### Scheme 18

![Scheme 18](https://i.imgur.com/5678901.png)

Incorporation of ¹³C-labeled acetates and propionates established the biosynthetic origin for most of the carbons in the streptomycete macrolide antibiotics azalomycin F₄a 57 and guanidylfungin A 58. These compounds, in which the position of the malonate ester is uncertain, feature a disubstituted guanidine group at the terminus. The starter unit in each case is probably derived from arginine via oxidative deamination and decarboxylation, but this has not been established through feeding experiments. The structural features of 57 and 58 resemble a number of other streptomycete metabolites including scopafungin,¹⁴⁶ copiamycin,¹⁴⁷ and malolactomycin A.¹⁴⁸ An arginine-derived starter unit is also likely involved in the formation of the ω-guanidino fatty acid residue of the cyclic lipopeptides circulocins α–δ from *Bacillus circulans*.¹⁴⁹

The Streptomyces antibiotic linearmycin A 59 is an example of a linear polyene that also probably utilizes 4-aminobutanoyl-CoA as a starter unit.¹⁵⁰ Feeding experiments with ¹³C-labeled acetate and propionate verified the 24 acetate and four propionate building blocks and indirectly supported the involvement of an ornithine-derived 4-aminobutanoate starter.

### 2.12.4 3-Amino-2-methylpropionate

Biosynthetic studies of the antitumor antibiotic vicenistatin 60 in *Streptomyces halstedii* established that all of the elongating units of the macrolactam aglycon are derived from acetate and propionate in the standard manner for type I PKS assembly.¹⁵¹,¹⁵² The starter unit, on the other hand, rather

![Chemical Structure](image1.png)

azalomycin F₄a 57

![Chemical Structure](image2.png)

guanidylfungin A 58

originates from glutamate through 3-amino-2-methylpropionate or its equivalent. Feeding experiments with [1-¹³C]- and [1,2-¹³C₂]acetate and [2,3,3-²H₃]- and [¹⁵N]glutamate suggest the involvement of glutamate mutase and decarboxylase catalyzed reactions in the formation of the starter unit (Scheme 19). No incorporation of deuterium from C2 of glutamate into the amidomethylene group of 60 was observed and reasoned to possibly result from rapid loss through deprotonation-reprotonation via 2-oxoglutarate.

### 2.12.5 β-Phenylalanine

Hitachimycin 61 is an antiprotozoal antibiotic that contains a novel, 19-membered ring lactam skeleton.¹⁵³ Feeding experiments established that this actinomycete macrolactam is biosynthesized from a phenylalanine-derived starter unit, eight acetate residues, and one propionate residue.¹⁵⁴ Incorporation of D,L-[1-¹³C]- and D,L-[¹⁵N]phenylalanine into 61 suggests the intramolecular rearrangement of phenylalanine to the presumptive starter unit β-phenylalanine catalyzed by an amino-mutase (Scheme 20). Phenylalanine 2,3-aminomutase was recently characterized in the plant *Taxus brevifolia* where it is involved in the formation of the phenylisoserine side chain in taxol.¹⁵⁵ The structure of 61 is related to the 24-membered macrocyclic polyene lactam antibiotic viridenomycin.¹⁵⁶

![Chemical Structure](image3.png)

linearmycin A 59

### 2.12.6 Proline

The pyrrole ethers A23187 (calcimycin) 63 and X-14547A (indanomycin) 64 are streptomyces ionophore antibiotics that complex and transport divalent cations such as Ca²⁺ across biological membranes.¹⁵⁷ A23187 63 contains an α-ketopyrrole residue that arises from a proline-derived starter unit.¹⁵⁸,¹⁵⁹ The remainder of the polyketide fragment is assembled from four propionate units and an acetate, whereas the benzoxazole unit contains a shikimate-derived benzenoid ring. Proline is presumably aromatized to pyrrole-2-carboxylate, which may directly serve as the primer unit. Aromatization could conceivably take place on an NRPS-bound proline unit. A similar scenario is also envisaged for 64; however, biosynthetic experiments have not been reported. Proline-derived pyrrole-2-carboxylate has also been proposed as a type I PKS starter unit in the biosynthesis of pyoluteorin 65 in *Pseudomonas fluorescens*¹⁶⁰ and undecylprodigine 66 from *S. coelicolor* A3(2).¹⁶¹ Biosynthetic gene clusters of these largely unrelated natural products have recently been described and share several structural traits. In each case, proline, which is either covalently

**Nat. Prod. Rep., 2002, 19, 70–99**

![image](https://i.imgur.com/1234567.png)

**Scheme 20**

attached to CoA or to a PCP, is oxidized to the corresponding Δ²-pyrroline derivative before spontaneous oxidation to pyrrole-2-carboxylate. This activated pyrrole is then presumably transferred to the KS₁ domain of either PltB-PltC or RedL to eventually yield 65 and 66, respectively (Scheme 21).

### 3 Novel starter units associated with type II PKSs

Most aromatic polyketides are formally acetate-primed. However, recently it has been disclosed that, at least in actinorhodin¹⁶²,¹⁶³ and tetracenomycin¹⁶⁴ biosynthesis, malonyl-CoA and not acetyl-CoA is the proximate primer of the PKS. It is assumed that most type II PKSs are primed by decarboxylation of a malonyl unit to yield an acetyl-S-KS intermediate, which is subsequently elongated by the minimal PKS. Leadlay *et al.* demonstrated that the KSβ (CLF) has decarboxylase activity towards malonyl-CoA, in close analogy to the KSQ of modular PKSs.¹⁸ Despite the widespread occurrence of acetate-primed polyketides, a variety of bacterial aromatic PKSs deviate from the decarboxylation mechanism and are primed by different starters. In the case of benzoate or salicylate-derived polyketides (Sections 3.4 and 3.5), decarboxylation is mechanistically not feasible. In general, attachment of alternative primers on type II PKSs is less thoroughly understood than in the type I PKSs, and the molecular basis for the choice of starter units on aromatic PKSs is fully unknown. Based on recent findings, two alternative pathways are evolving. Like the ery case, oxytetracycline (Section 3.1) and enterocin (Section 3.4) biosynthesis gene sets contain a monofunctional CoA ligase and an AT that are putatively responsible for activation and transfer of the parent starters (malonamate and benzoate, respectively) onto the PKS. In the case of daunorubicin (Section 3.2), R1128, and frenolicin biosynthesis (Section 3.3), where the PKS is primed by short-chain fatty acids, an additional ACP

**Scheme 21**

and KSIII are involved, leading to a functional cross-talk between fatty acid and polyketide metabolism. In this section, the biosynthesis and attachment of non-acetate primers of type II PKSs is presented.

#### 3.1 Malonate and malonamate

An aberrant priming mechanism is observed in the biosynthetic pathways of the tetracycline and the xanthone families of

*Nat. Prod. Rep.*, 2002, **19**, 70–99

metabolites. Decarboxylation of malonyl-CoA does not occur as in actinorhodin and tetracenomycin biosynthesis, resulting in an intact incorporation of a C₃ unit in lieu of acetate. The quite extraordinary function of a complete malonate unit as a starter unit is even more obscure considering the concomitant amination of the malonate carboxyl in the tetracyclines 67–69 (Scheme 22). There has been an on-going discussion whether the

![Chemical Structure](https://i.imgur.com/1234567.png)

Scheme 22

carboxamido group is introduced at the start of the biosynthesis or during a post-PKS tailoring step.¹⁶⁵⁻¹⁶⁷ In order to determine whether malonate or malonamate is the true starter unit, studies at both chemical and genetic levels were undertaken. On the basis of isotopic labeling studies, Thomas and Williams hypothesized in 1983 that the starter unit of oxytetracycline 67, a broad-spectrum antibiotic produced by several actinomycetes, e.g., *S. aureofaciens* and *S. rimosus*, is malonamyl-CoA. However, the authors admitted that malonyl-CoA could also act as starter with amination taking place subsequent to completion of the polyketide backbone.¹⁶⁸ Cloning, sequencing, and heterologous expression of oxytetracycline (*otc*) biosynthetic genes was expected to shed some light on the nature of the true starter unit.¹⁶⁵ Expression experiments by Hopwood, Khosla, and coworkers showed that the minimal *otc* PKS is intrinsically capable of synthesizing an acetate-derived polyketide backbone in the absence of additional proteins encoded by the *otc* cluster.¹⁶⁹ A hybrid PKS composed of the *otc* minimal PKS and the actinorhodin (*act*) ketoreductase in *S. coelicolor* CH999 gave a decaketide derived from acetate only. These expression studies indicated that the *otc* PKS is promiscuous towards acetate, but left open the nature of the original starter unit. The best indication for malonamate over malonate as a primer comes from a *S. rimosus* mutant deficient in a functional genomic copy of the bifunctional cyclase/aromatase encoding gene *otcDI*.¹⁷⁰ Hunter and coworkers showed that the *otcDI* null mutant produces four novel malonamate primed polyketides (70–73) with different folding patterns and shorter chain lengths than 67 (Scheme 23). All novel shunt products have aminated termini, which are most probably derived from the same aminated precursor. The presence of the carboxamido group in all metabolites implies that the carboxamido group is already present in the primer of *otc* biosynthesis.

---

Rohr, Floss, and coworkers investigated the biosynthesis of lysolipin X 75, an antibacterial, cytotoxic xanthone antibiotic produced by *S. violaceoniger* Tü 96 (Scheme 24).¹⁷¹ Since [U-¹³C] malonic acid was incorporated intact, it was concluded that the nonaketide 74 is primed by malonyl-CoA. Amination of the polyketide backbone possibly takes place at a later stage of the biosynthesis. However, an equally acceptable scheme would involve nitrogen insertion via a malonamate primer unit as postulated for the tetracyclines.¹⁶⁸

### 3.2 Propionate

Incorporation of an intact C₃ starter unit in type II PKS systems has also been reported for a number of anthracycline antibiotics, such as aclacinomycin 79, daunorubicin 80, and doxorubicin 81.¹⁷² Although anthracyclines share structural features with the tetracyclines, labeling studies demonstrated a rather different folding of the propionate-primed polyketide backbone 76 (Scheme 25). Umezawa et al. showed that the common anthracycline intermediate aklavinone 78, a metabolite isolated from *S. galilaeus*, is derived from propionyl-CoA and nine malonyl-CoA extender units.¹⁷³ The best-studied use of propionate as starter is for daunorubicin 80 from *S. peuceti* and its C14-hydroxylated derivative doxorubicin 81 from *S. coeruleorubrum*, which are among the most important antitumor antibiotics in current use.¹⁷⁴,¹⁷⁵ In regard to the biosynthetic origin of propionyl-CoA, it is proposed that the primer is derived from multiple sources such as amino acid catabolism and degradation of odd-numbered fatty acids, but decarboxylation of methylmalonyl-CoA appears to be less likely.¹⁷⁶ Sequence analyses of the daunorubicin (*dnr*) and doxorubicin (*dox*) gene clusters by the Hutchinson¹⁷⁴ and Strohl¹⁷⁷ groups revealed an unusual arrangement of the PKS genes. Contrary to most iterative PKS gene clusters, the *dnr* ACP gene *dpsG* is not translationally coupled with the KSα/KSβ

S. violaceoniger  
Tü 96  

12 x malonate  

![Chemical Structure](https://i.imgur.com/structure1.png)  

R = OH or NH₂  

74  

![Chemical Structure](https://i.imgur.com/structure2.png)  

lysolipin X 75  

Scheme 24  

genes *dpsA* and *dbsB*. Furthermore, two additional PKS components, KAS and AT encoding genes *dpsC* and *dpsD* whose products may be relevant for chain initiation, were identified in the *dnr* gene cluster. Interestingly, DpsD has high similarity with the malonyl-CoA:ACP acetyltransferase (MCAT) enzymes of bacterial fatty acid metabolism, while DpsC is a homolog of FabH from bacterial FASs.¹⁷⁵ Reynolds *et al.* found that the *S. glaucescens* FabH functions as a KSIII, which catalyzes the first condensation of malonyl-CoA with acetyl-CoA to form acetoacetyl-CoA in the biosynthesis of fatty acids.¹⁷⁸ In analogy, it is assumed that DpsC functions as a priming KS that is solely dedicated to the first malonyl-CoA condensation cycle. Catalyzed by the AT DpsD, the diketide is putatively transferred onto the active site of the KSα-KSβ heterodimer for further malonyl-CoA elongations (Scheme 26).¹⁷⁵,¹⁷⁷ In fatty acid biosynthesis, FabH specifies the starter unit,¹⁷⁸ but in anthracycline biosynthesis, the situation appears to be more complex. In order to address which components of the *dnr* PKS are responsible for starter unit specificity, different combinations of the *dnr* PKS genes from *S. peucitius*¹⁷⁵ and *Streptomyces* sp. strain C5¹⁷⁷ were cloned and heterologously expressed. Hutchinson and coworkers found that DpsA and DpsB have relaxed starter unit specificity for acetyl-CoA instead of propionyl-CoA when expressed in *S. lividans* and *S. glaucescens*. It was thus concluded that DpsA and DpsB do not (at least solely) determine the choice of starter unit and that additional components, such as DpsC and DpsD, are involved in choice of the appropriate starter unit.¹⁷⁹ Conversely, Strohl and coworkers reported that heterologous expression of the daunorubicin and daunomycin PKSs in *S. lividans* can result in biosynthesis of propionyl-primed polyketides even in the absence of the putative propionyl transferase and priming KS; DpsC and DpsD.¹⁸⁰ Strohl *et al.* thus suggested that the minimal PKS would also show some selectivity toward the

![Chemical Structure](https://i.imgur.com/structure3.png)  

reduction  

9 x  
malonyl-  
CoA  

76  

![Chemical Structure](https://i.imgur.com/structure4.png)  

aklavinone 78  

![Chemical Structure](https://i.imgur.com/structure5.png)  

aklanonic acid 77  

![Chemical Structure](https://i.imgur.com/structure6.png)  

aclarubicin 79  

daunorubicin 80  

![Chemical Structure](https://i.imgur.com/structure7.png)  

doxorubicin 81  

Scheme 25  

cognate primer unit and that protein interactions between KAS/CLF/CYC/KR jointly specify the starter unit.¹⁸⁰ While *dpsD* mutants retained the ability to choose the correct starter unit in formation of aklanonic acid,¹⁷⁵ more recently, DpsC was found to exhibit a very high specific activity for propionyl-CoA. Hutchinson and coworkers showed by *in vitro* synthesis of propionate-primed polyketides that DpsC is indeed responsible for the choice of the correct starter unit for daunorubicin biosynthesis.¹⁸¹,¹⁸² It is noteworthy that in the biosynthetic gene cluster of aklavinone 78 (*S. galilaeus*) homologs of *dpsC* and *dpsD* have not been identified.¹⁸³  

### 3.3 Butyrate, isobutyrate and homologs  

As for several type I PKSs and FASs (Section 2.1), several short-chain carboxylic acids have been identified as starter units for aromatic PKSs. The anthraquinones R1128A–D (82–85) are non-steroidal estrogen receptor antagonists produced by *Streptomyces* sp. R1128.¹⁸⁴,¹⁸⁵ They show *in vitro* and *in vivo* antitumor activity and are thus promising leads for anti-

Polyketide (dnr) biosynthesis                                                                 Fatty acid biosynthesis

![Chemical Structure](https://i.imgur.com/structure1.png)

**Scheme 26**

hormone treatment for breast cancer. According to the various alkyl side chains, the polyketides are presumably primed by a number of different short-chain starter units. Recently, Khosla and coworkers demonstrated that the unusual 4-methylvaleryl side chain of R1128C 84 and HU235 86, a novel tetrahydro-anthracenone compound, is derived from valine. Feeding experiments in *S. lividans* K4-114/pHU235, a transformant harboring a cosmid clone with the entire biosynthetic gene cluster of the R1128 complex, gave evidence that valine is converted into isobutyryl-CoA via transamination, followed by decarboxylation and loss of C1 of [1-¹³C]valine (see Section 2.1, Scheme 1). The arrangement of the 17-kb R1128 biosynthetic gene cluster largely deviates from the common gene architecture of type II PKSs. Besides the ketosynthase heterodimer, consisting of KSα (ZhuB) and KSβ (ZhuA), sequence analyses of the 14 ORFs revealed additional genes putatively involved in starter unit biosynthesis and attachment. The ORFs encode an additional β-ketoacylsynthase (ZhuH), an AT (ZhuC), and two ACPs (ZhuG and ZhuN). ZhuH is a homolog of the FabH β-ketoacylsynthase III (KS III), and Khosla et al. thus propose that, as in the case of daunorubicin biosynthesis (Section 3.2), ACP-bound β-ketoacyl intermediates are formed by the concerted action of ZhuC, the primer-specific ZhuH and malonated ZhuG. Dependent of the primer, acetoacetyl-ACP, β-ketopentanoyl-ACP, or 2-oxo-4-methyl-pentanoyl-ACP are thus formed. In contrast to daunorubicin biosynthesis, ACP bound intermediates are then reduced into butyryl-ACP, valeryl-ACP or 4-methylvaleryl-ACP in analogy to bacterial fatty acid biosynthesis before transfer to the minimal PKS (Scheme 27). Since genes encoding a full reductive cycle were not located on the R1128 gene cluster, the R1128 synthase may recruit ketoreductase, dehydratase and enoylreductase enzymes from the endogenous FAS. A functional link between fatty acid biosynthesis and polyketide biosynthesis has also been proposed for alkanonic acid biosynthesis (Section 3.2), and is well established for actinorhodin biosynthesis in *S. coelicolor*, where the minimal PKS employs a MCAT from the endogenous FAS.

In several other examples for polyketides bearing putatively butyryl-derived moieties, the situation is less apparent than for R1128 biosynthesis. For example, the polyketide backbone of frenolicin B 87, an antibiotic produced by *S. roseofulvus*, is presumably derived by transfer of a butyryl unit on to the KSα-KSβ heterodimer followed by seven malonyl-CoA extensions (Scheme 28), while an analogous route is assumed to yield the related acetyl-CoA primed metabolite nanaomycin 88. On the other hand, it can not be ruled out that 87 is a nonaketide with a fully reduced methylene group, since nonaketides derived from nine acetates were produced by the frenolicin (*frn*) minimal PKS in *S. coelicolor*. Analysis of the *frn* biosynthetic gene cluster lends support for a similar priming mechanism as in R1128 biosynthesis. The rare occurrence of an additional ACP in both the R1128 and the frenolicin gene clusters is striking (58% identity of FrnN with ZhuN, 46% identity of FrnJ with ZhuG).

![Chemical Structure](https://i.imgur.com/structure2.png)

**Scheme 27**

88 Nat. Prod. Rep., 2002, 19, 70–99

A variety of other aromatic polyketides are putatively biosynthesized from butyrate starter units, such as murayaquinone 89, FD-594 90, and sulfurmycinone 91. However, the true nature of the starter unit in each case is vague. Furthermore, the structure of benastatin 92 even suggests the involvement of a hexanoate starter. As proposed in the case of the R1128 complex, a full reductive cycle is probably involved prior to PKS malonate extensions.

### 3.4 Benzoate

The bacteriostatic agents enterocin (vulgamycin) 93 and wailupemycins A–D 94–97 are members of a series of structurally diverse polyketides produced by the marine bacterium "Streptomyces maritimus" (Scheme 29). Common to all metabolites is a benzoyl-derived moiety. Classical feeding experiments by Seto and coworkers demonstrated that enterocin is derived from BA, seven acetate units, and the methyl group of methionine and undergoes a rare Favorskii-like carbon rearrangement. Recently, the 20 ORF enterocin (enc) biosynthesis gene cluster (~21 kbp) was cloned and sequenced in the Moore group. Seven ORFs putatively involved in the biosynthesis and attachment of the benzoyl-CoA starter unit are arranged on either side of the centrally located minimal enc type II PKS on four transcripts. Sequence analysis of these genes was very suggestive for a eukaryotic-like BA biosynthetic pathway. In the field of plant secondary metabolism, two pathways have been proposed that similarly involve conversion of L-phenylalanine into cinnamic acid by phenylalanine ammonia lyase (PAL) (Scheme 30). The routes diverge at this intermediate, one involving a β-oxidation pathway (route “a”) and the second a retro-aldol path through benzaldehyde (route “b”). Until recently, the only known bacterial benzoate pathway was anaerobic and involved transamination of phenylalanine to phenylpyruvate followed by two successive rounds of α-oxidative decarboxylations (route “c”). As shown in Scheme 31, a rare bacterial PAL putatively encoded by encP converts L-phenylalanine into trans-cinnamic acid. Cinnamate is then activated by the putative cinnamoyl-CoA ligase EncH to yield cinnamoyl-CoA. Feeding experiments using doubly labeled [ring-²H₅–¹,²–¹³C₂]phenylpropanoid biosynthetic intermediates were conducted in order to unequivocally differentiate between the β-oxidative and the retro-aldol routes, and to establish whether β-ketophenylpropionyl-CoA may serve as an alternative starter, omitting one malonyl-CoA extension. By MS analysis of the metabolites, Hertweck and Moore found that all probes were incorporated, yet only after loss of the side chain ¹³C-label. This result clearly indicates that the propionate side chain is cleaved before incorporation of the phenyl unit, and thus, benzoyl-CoA, and not 98, is the enc PKS starter unit. Furthermore, the labeling experiments gave strong evidence for the β-oxidative pathway. In accordance with the plant pathway, hydration of cinnamoyl-CoA by EncI, dehydration, and thiolysis by EncJ yields the benzoyl-CoA starter unit. Finally, the benzoyl-CoA starter is loaded onto the PKS by the aromatic acyl transferase EncL. The functional verification of each of these enzymatic processes was recently provided through gene inactivation experiments (L. Xiang and B. S. Moore, unpublished observations). Furthermore, the Moore and Oldham groups jointly disclosed that (3R)-3-hydroxy-3-phenylpropionyl-CoA 99 is an intermediate in the biosynthesis of BA in both bacteria (“S. maritimus”) and plants (*Nicotiana tabacum*). Similar observations have been made for cocaine biosynthesis, where also a dramatic preference for the *R*-enantiomer was exhibited. The stereochemistry of the hydroxypropionate is identical to its counterparts in fatty acid β-oxidation (Scheme 31).

and, interestingly, there is considerable amino acid similarity between the bacterial cinnamoyl-CoA hydratase EncI and various fatty acid enoyl-CoA hydratases from bacteria, mammals and plants.²⁰⁷

### 3.5 Salicylate

Thermorubin 100 is a polycyclic antibiotic produced by the thermotolerant, low G+C Gram-positive bacterium *Thermobactin vulgaris*. Feeding experiments with $^{13}$C-labeled acetates by Aragozzini and coworkers revealed a mixed biosynthetic origin of this polyaromatic polyketide of presumed type II PKS origin.²⁰⁹ The undecaketide moiety, which is entirely acetate derived, undergoes an interesting oxidative rearrangement in a later stage of the biosynthesis. Successful incorporation of [1-$^{13}$C]salicylate supports the involvement of salicyl-CoA as the novel priming unit (Scheme 32). Although the biosynthesis of salicylic acid has not been elucidated in *T. vulgaris*, bacterial salicylic acid is derived from the shikimate pathway via the loss of pyruvate from isochorismic acid.²¹⁰

### 4 Engineering novel polyketides with unnatural starter units

As demonstrated in Sections 2 and 3, in many cases the nature of the starter units defines biological activity of the polyketides produced. However, the natural product may be sub-optimal in regard of activity, selectivity, availability and unwanted side effects. In order to improve promising leads, recent approaches in metabolic engineering are aimed at generating novel polyketides.²,¹⁰,¹⁴ In this context, enlarging natural diversity by targeted manipulations of starter units is one attractive option. Several strategies have been described, in which either the native producer or a modified biosynthetic machinery is employed. Enzymes of secondary metabolism are generally less substrate specific than enzymes from primary metabolism.¹⁵ Thus, derivatization of a secondary metabolite can be possible by simply feeding biosynthetic precursor analogs to the fermentation broth of the producing microorganism. This process, named precursor-directed biosynthesis, can be a powerful alternative to chemical derivatization. Thiericke and Rohr thoroughly reviewed this technique in 1993 in this journal.²¹¹ Success and failure of precursor-directed biosynthesis using PKSs depend on tolerance and specificity of a variety of

**Scheme 29**

**Scheme 30**

Nat. Prod. Rep., 2002, 19, 70–99

Benzoate pathway ("S. maritimus" and N. tabacum)

L-Phe $\xrightarrow{\text{EncP}}$ $\xrightarrow{\text{PAL}}$ $\xrightarrow{\text{Encl}}$

Fatty acid degradation (bacteria, mammals and plants)

$\xrightarrow{\text{EncH}}$

Scheme 31

The following examples highlight the potential of metabolic engineering in the context of PKS priming.

### 4.1 Precursor-directed biosynthesis

#### 4.1.1 Erythromycin analogs

The modular megasynthase DEBS catalyzes the formation of 6-deoxyerythronolide B (6-dEB), the parent aglycon of the antibiotic erythromycin B 101. The PKS, arranged into the three large multidomain enzymes DEBS1, DEBS2 and DEBS3, is naturally primed by propionyl-CoA by means of the loading AT<sub>L</sub>–ACP<sub>L</sub> didomain. The AT<sub>L</sub> domain has influence on the choice of the starter unit incorporated. In addition to its natural propionyl-CoA substrate, a few alternative starters can be accepted. The wild type producer *Saccharopolyspora erythraea* NRRL 2338 has been shown to incorporate acetate and cyclopropyl carboxylic acid *in vivo* to form 15-nor-erythromycin C<sup>214,215</sup> 102 and 13-cyclopropylerythromycin B<sup>216</sup> 103, respectively. A derivative of the latter, 6-deoxy-13-cyclopropyl-erythromycin B 104 has also been produced by feeding cyclopropane carboxylic acid to *S. erythraea* NRRL 18643, a genetically engineered strain with a knock-out in the C6 hydroxylase.<sup>217</sup> In a cell-free system, it was shown that the AT<sub>L</sub> domain also tolerates *n*-butyryl-CoA and isobutyryl-CoA.<sup>218</sup> A truncated form of DEBS (DEBS1 + TE), in which DEBS1 is fused to the thioesterase domain from the C-terminal end of DEBS3, was used to investigate the substrate specificity. DEBS1 + TE from cell-free preparations of a recombinant streptomyces was found to exhibit a relaxed specificity for acetyl- and butyryl-CoA.<sup>219,220</sup> Functionally expressed DEBS1 + TE in *E. coli* can convert acetyl-, propionyl- and butyryl-CoA into the corresponding triketide lactones *in vitro* (Fig. 6).<sup>17</sup> Competition experiments using a broad range of substrates showed that the loading didomain of DEBS has a preference for unbranched alkyl chain substrates over branched alkyl chain, polar, aromatic and charged substrates.

#### 4.1.2 Avermectin analogs

The avermectin synthase (AVES) exhibits a large substrate tolerance towards non-natural starter units, as already mentioned in Section 2.1.<sup>23,221</sup> While avermectins naturally incorporate just two alternative primers, isobutyryl-CoA and 2-methylbutyryl-CoA,<sup>22</sup> the loading domain of the AVES is capable of loading at least 44 different starter units (see Section 4.2). For example, 2-pentyl and 2-hexyl avermectin analogs 105 and 106 with potent anthelmintic and insecticidal activity have been yielded through feeding 2-methylpentanoate and 2-methylhexanoate to the natural producing strain *S. avermitilis* MA 5502.<sup>221</sup>

enzymes. At first, the unnatural starter typically needs to be activated by a CoA ligase, if not administered *e.g.* as a NAC thioester. Subsequently, the primer has to be loaded onto the PKS by a rather nonselective AT. Once this bottleneck is passed, a number of downstream enzymes have to tolerate the non-natural intermediates in order to yield the final modified product. In order to “force-feed” a non-natural primer, analogs may be supplemented to a blocked mutant, by either adding a specific inhibitor of the primer biosynthesis, or by deleting particular biosynthetic genes. The latter technique is sometimes referred to as “mutasynthesis”.<sup>104,212,213</sup> Genetic engineering also offers the possibility of generating broader or different specificity of the loading domain and/or combining starter unit specific biosynthetic pathways with different PKSs.

erythromycin B 101 R = Me  
C 102 R = H  

103 R' = OH  
104 R' = H  

eryA1/eryA3  

translate  

DEBS1  
ery loading module 1 module 2 end  

AT ACP KS AT KR ACP KS AT KR ACP TE  

S  
R  
= O  

S  
R  
OH  

S  
R  
OH  

DEBS1/TE  

OH  

R  
O  
= O  

Fig. 6  

### 4.1.3 Ansamycin and manumycin analogs  

Biosynthesis and function of the central mC₇N units of the ansamycins, manumycins and asukamycin were discussed in Sections 2.8 and 2.9. Incorporation of 3-aminobenzoic acid (mABA) to the 20-producing strain S. collinus Tü 1892 gave the novel product 20,23-dideoxyansatrienin B 107 (Scheme 33). The manumycin family of metabolites has also been a target for precursor-directed biosynthesis. When feeding artificial precursors, such as substituted (amino)benzoic acids, aromatic amines and non-aromatic compounds to the manumycin-producing strain S. parvulus Tü 64, Thiericke and Zeeck observed a strong dependence of the metabolite pattern obtained (Scheme 34). While 7 mM mABA suppressed natural manumycin biosynthesis, increased concentrations of mABA  

(55 mM) lead to a manumycin variant 108 with replacement of the central mC₇N starter unit. Feeding PABA gave analog 109 in which the “upper” side chain was absent. Other amino- and hydroxy-substituted benzoic acids were also accepted by the manumycin PKS, providing related metabolites 110–113, yet lacking the “upper” side chain. 113 is formed by both administering the benzoic acid (BA) derivative as well as the cinnamic acid (CA) derivative, thus bypassing one malonyl-CoA extension of the parent primer.

### 4.1.4 Enterocin analogs  

As mentioned in Section 3.4, the enterocin PKS from "S. maritimus" harbors a dedicated benzoyl-CoA ligase and AT. In the enterocin-producing strain S. hygroscopicus A-5294, the benzoyl-CoA starter of the enc PKS can be replaced by feeding fluorinated benzoic acids. As monitored by ¹⁹F NMR spectroscopy, ortho-, meta- and para-fluorobenzoic acid, as well as 3,4-difluorobenzoic acid were incorporated, yielding the requisite fluorinated enterocins 114–117 (Scheme 35). The fluorinated analogs, however, did not show higher antibacterial activity than enterocin.

altered in the region of the starter unit. Khosla and coworkers generated the transformant *S. coelicolor* CH999/pJRRJ2, which harbors a KS₁ null mutant of DEBS, which is unable to produce 6-dEB 119 unless supplemented with the appropriate NAC diketide thioester 118.²²⁷ By incorporation of synthetic partial chains into the PKS downstream of KS1, a variety of macrolide aglycon analogs with unnatural substituents at C13 were produced. Diketide analogs such as 120 and 126 were introduced into heterologously expressed DEBS (KS1°), giving 15-ethyl-6-dEB 121 and 14-demethyl-14-phenyl-6-dEB 127, respectively, which were then converted by *S. erythraea* to give the corresponding erythromycin D analogs with antibacterial activity.²²⁷ In analogy, vinyl- and propargyl-substituted 6-dEB analogs 123 and 125 were produced by feeding the corresponding vinyl and propargyl diketides 122 and 124.²²⁸,²²⁹ Incorporation of the more advanced NAC thioester 128 resulted in the formation of 15-(8-oxononyl)-6-dEB 129 (Scheme 36).²²⁸ Staunton, Leadlay and coworkers

evaluated a related precursor-directed approach by means of purified DEBS1 + TE system *in vitro*. Feeding synthetic diketides to the truncated DEBS generated triketide lactones with structural variations at C5.²³⁰ Although the loading domain was bypassed in all experiments, the novel substituents replaced the natural primer, thus representing a technique for formally engineering novel starter units.

### 4.2.2 Rifamycin analogs

As mentioned in Section 2.8, the rifamycin loading module exhibits a relaxed specificity towards non-natural starter units. Its substrate tolerance is comparable with related bacterial benzoyl-CoA ligases.²³¹,²³² The *rifK(-)* mutant HGF003, in which the 3,5-AHBA synthase was inactivated, only produces rifamycin B 38 upon supplementation with the natural primer 3,5-AHBA.¹¹¹ Administration of the starter unit analogs 3-hydroxybenzoate (3-HBA) and 3,5-dihydroxybenzoate (3,5-DHBA) to this mutant strain gave the new tetraketides 130 and 131, respectively (Scheme 37).²³³,²³⁴ These compounds are structurally related to the tetraketide shunt product P8/1-OG 132, which was first described from a different mutant strain of *A. mediterranei*.²³³ The tetraketides 130 and 131 fail to fully elongate on the rifamycin PKS and terminate before the fourth module. Floss and coworkers propose that the dihydronaphthoquinone ring of 38 is formed between the tetraketide and

---

**4.2 Mutational biosynthesis**

A variant of precursor-directed biosynthesis, where non-natural primers are administered to suitably blocked mutants is referred to as mutational biosynthesis or mutasynthesis.²¹²,²¹³ Biosynthesis of the natural metabolite is abolished by inactivation of genes responsible for starter unit biosynthesis or processing of early intermediates, and can only be restored by providing synthetic (activated) precursors. By means of this approach, the natural biosynthesis is suppressed, making it possible to “force-feed” non-natural primers to the manipulated biosynthetic machinery.

#### 4.2.1 Erythromycin analogs

A mutational biosynthetic approach using a modified DEBS has succeeded in the production of erythromycin analogs

in RifB.²³⁵ Hence, module 4 of RifB may only chain extend naphthoquinoid (cyclized) tetraketides and thus discriminate against the unnatural tetraketides 130 and 131. More recently, eleven additional aromatic acids were identified as *in vitro* substrates of the rifamycin loading A–T didomain.¹⁰⁸

### 4.2.3 Avermectin Analogs

A randomly generated mutant of the avermectin producer *S. avermitilis*, which lacks branched-chain 2-oxo acid dehydrogenase activity, has been found suitable for mutational biosynthesis of novel avermectins.²⁴ When this mutant is grown in the absence of supplementary branched-chain carboxylic acids, no avermectins are produced. Avermectin biosynthesis can be restored, however, only upon supplementation.²⁸ Out of various potential precursors, more than 44 alternative primers were discovered that allow for production of avermectin analogs with altered C25 substituents.²³⁶,²²¹ However, as a result of additional random mutagenesis, the productivity of the mutant is low. A rationally designed *bkdF* mutant of *S. avermitilis* (Section 2.1) therefore was able to surmount decreased efficacy. Substrate feeding of alternate carboxylic acids to the *S. avermitilis* mutants demonstrated that the ave loading domain is capable of accepting a large range of substrates to produce novel C25 substituted avermectins (Fig. 7).²³⁶ The amino-terminal ATₗ/ACPₗ accommodates a wide range of non-natural starter units with (cyclo)alkyl

**Scheme 37**

pentaketide stage, which is coincident with the transfer of substrate from the last module of RifA to the first module

![Chemical Structures](https://i.imgur.com/chemical_structures.png)

**Fig. 7**

94 Nat. Prod. Rep., 2002, 19, 70–99

moieties, double bonds and triple bonds, as well as some oxygen and sulfur containing substituents. According to the spectrum of accepted substrates, the binding region of the ave loading domain appears to be small and hydrophobic. However, it is quite tolerant towards non-natural primers. All novel avermectins produced by mutational biosynthesis possess broad-spectrum antiparasitic activity in vitro. Feeding CHC to the S. avermitilis bkd mutant 237 yielded the commercially important analog doramectin, which is more active against parasites than the natural avermectins, especially for treatment of onchrozerkosis. 238

### 4.3 Engineering hybrid biosynthetic pathways

With the advent of recombinant techniques, a growing number of publications document novel opportunities for genetic pathway engineering. It is well documented that components of different polyketide pathways can be recombined in a mix-and-match fashion to produce new polyketides. Combinatorial biosynthesis approaches have been employed for manipulating the polyketide carbon backbone, cyclization and tailoring post-PKS steps. In order to engineer alternate starter units, viable options include transplanting loading domains (starter unit specificity) and/or biosynthetic gene cassettes that are responsible for the generation of distinct primer units.

#### 4.3.1 Engineering broader specificity into the loading module

The DEBS loading module has been successfully replaced in vivo with the loading module from the avermectin-producing PKS of S. avermitilis. Leadlay et al. demonstrated that substitution of the ery AT_L/ACP_L in DEBS1/TE with the ave AT_L/ACP_L resulted in a construct with broader substrate specificity (Fig. 8). S. coelicolor CH999/pIG1, harboring the modified DEBS1/TE, was capable of accepting propionyl-CoA and acetyl-CoA, as well as isobutyrate and sec-butyrate as starter units and providing the corresponding triketide lactones. 239 An equivalent loading module swap was carried out on the full DEBS in S. erythraea by homologous recombination. By integration into the chromosome, the ave loading domain replaced the natural ery loading domain. The resulting integrant S. erythraea strain NRRL 2338/pIG1 (S. erythraea ERMD1) proved to contain a functional hybrid PKS. Besides erythromycins A, B and D, novel analogs in which the natural C13 ethyl moiety was replaced by isopropyl and sec-butyl were generated (Scheme 38). 239 Leadlay et al. speculate that the mutant recruits branched chain carboxylic acids from endogenous fatty acids for production of isopropyl B and sec-butyl B analogs. Novel compounds 136 and 137 showed antibacterial activity against B. subtilis comparable to erythromycins A and B. Conversion of DEBS into a hybrid system of broader specificity has the potential of accepting a wider range of starter units comparable to that of the AVES system (Section 4.2). 236 In fact, a variety of novel erythromycins were obtained by fermentation of the engineered S. erythraea strain with exogenously supplied unnatural precursors (Scheme 39). 216 The C13 cyclopentyl analog showed particularly high activity against P. multocida and E. coli amongst the various hybrid metabolites. 216,240

#### 4.3.2 Engineering novel starter units via hybrid biosynthetic pathways

Adding synthetic precursors to fermentations for precursor-directed or mutational biosynthesis in order to yield structural analogs of the metabolites is a powerful, yet sometimes costly, technique. In addition, several other factors have to be considered, such as toxicity, cell permeability and activation.

Until recently, the important avermectin analog doramectin 139 was primarily produced by adding synthetic CHC to

fermentations of the S. avermitilis bkd mutant. 28,237 Considering the high demand for doramectin for treatment of onchrozerkosis, alternative ways were sought, such as fusing genes responsible for CHC starter unit biosynthesis with ave PKS genes. As mentioned in Section 2.4, ansatrienin A 20 contains a CHC-derived side chain. 55 The S. collinus CHC-CoA biosynthetic gene cassette pAC12 was introduced into the bkdF mutant of S. avermitilis, and the resulting mutant was capable of producing 139 in the absence of exogenous CHC (Scheme 40). 63 As the AVES loading domain allows for the

R¹—COOH

S. erythraea  
NRRL2338/pAVLD  
(ERMD1)

![Chemical Structure](https://i.imgur.com/structure1.png)

**Scheme 39**

![Chemical Structure](https://i.imgur.com/structure2.png)

**Scheme 40**

loading of CHC-CoA, which can be synthesized using the CHC biosynthetic gene cassette, Reynolds and coworkers have now paved the way for the generation of hybrid PKSs that could use CHC-CoA or a pathway intermediate for the *in vivo* generation of novel products.

### 5 Acknowledgements

Research on the biosynthesis and specificity of type II PKS starter units is generously supported by the National Institutes of Health (AI47818) to B.S.M. and by a Feodor-Lynen fellowship from the Alexander-von-Humboldt Foundation to C.H.

---

### 6 References

1. D. O'Hagan, *The Polyketide Metabolites*, Ellis Horwood, Chichester, 1991.
2. D. A. Hopwood, *Chem. Rev.*, 1997, **97**, 2465.
3. B. S. Moore and J. N. Hopke, *ChemBioChem*, 2001, **2**, 35.
4. B. J. Rawlings, *Nat. Prod. Rep.*, 2001, **18**, 190.
5. B. J. Rawlings, *Nat. Prod. Rep.*, 2001, **18**, 231.
6. B. J. Rawlings, *Nat. Prod. Rep.*, 1999, **16**, 425.
7. J. Staunton and K. J. Weissman, *Nat. Prod. Rev.*, 2001, **18**, 380.
8. B. A. Pfeifer and C. Khosla, *Microbiol. Mol. Biol Rev.*, 2001, **65**, 106.
9. C. R. Hutchinson, *Curr. Opin. Microbiol.*, 1998, **1**.
10. P. F. Leadlay, *Curr. Opin. Chem. Biol.*, 1997, **1**, 162.
11. L. Katz and R. McDaniel, *Med. Res. Rev.*, 1999, **19**, 543.
12. C. Khosla, *Chem. Rev.*, 1997, **97**, 2577.
13. C. Hertweck, *ChemBioChem*, 2000, **1**, 113.
14. D. E. Cane, C. T. Walsh and C. Khosla, *Science*, 1998, **282**, 63.
15. C. Khosla, R. S. Gokhale, J. R. Jacobsen and D. E. Cane, *Annu. Rev. Biochem.*, 1999, **68**, 219.
16. A. Pereda, R. G. Summers, D. L. Stassi, X. A. Ruan and L. Katz, *Microbiology*, 1998, **144**, 543.
17. J. Lau, D. E. Cane and C. Khosla, *Biochemistry*, 2000, **39**, 10514.
18. C. Bisang, P. F. Long, J. Cortes, J. Westcott, J. Crosby, A. L. Matharu, R. Cox, T. J. Simpson, J. Staunton and P. F. Leadlay, *Nature*, 1999, **401**, 502.
19. T. Kaneda, *Microbiol. Rev.*, 1991, **55**, 288.
20. H. Ikeda and S. Omura, *Chem. Rev.*, 1997, **97**, 2591.
21. H. Ikeda, T. Nonomiya, M. Usami, T. Ohta and S. Omura, *Proc. Natl. Acad. Sci., U.S.A.*, 1999, **96**, 9509.
22. D. E. Cane, T.-C. Liang, L. K. Kaplan, M. K. Nallin, M. D. Schulman, O. D. Hensens, A. W. Douglas and G. Albers-Schönberg, *J. Am. Chem. Soc.*, 1983, **105**, 4110.
23. T. S. Chen, B. H. Arison, V. P. Gullo and E. S. Inamine, *J. Ind. Microbiol.*, 1989, **4**, 231.
24. E. W. Hafner, B. W. Holley, K. S. Holdom, S. E. Lee, R. G. Wax, D. Beck, H. A. I. McArthur and W. C. Wernau, *J. Antibiot.*, 1991, **44**, 349.
25. S. Omura, H. Ikeda and H. Tanaka, *J. Antibiot.*, 1991, **44**, 560.
26. T. A. Cropp, A. A. Smogowicz, E. W. Hafner, C. D. Denoya, H. A. I. McArthur and K. A. Reynolds, *Can. J. Microbiol.*, 2000, **46**, 506.
27. D. D. Skinner, M. R. Morgenstern, R. W. Fedechko and C. D. Denoya, *J. Bacteriol.*, 1995, **177**, 183.
28. C. D. Denoya, R. W. Fedechko, E. W. Hafner, H. A. I. McArthur, M. R. Morgenstern, D. B. Skinner, K. Stutzman-Engwall, R. G. Wax and W. C. Wernau, *J. Bacteriol.*, 1995, **177**, 3504.
29. Y. X. Zhang, C. D. Denoya, D. D. Skinner, R. W. Fedechko, H. A. I. McArthur, M. R. Morgenstern, R. A. Davies, S. Lobo, K. A. Reynolds and C. R. Hutchinson, *Microbiology*, 1999, **145**, 2323.
30. W. Trowitzsch-Kienast, V. Wray, K. Gerth, H. Reichenbach and G. Höfle, *Liebigs Ann. Chem.*, 1986, 93.
31. B. Silakowski, H. U. Schairer, H. Ehret, B. Kunze, S. Weinig, G. Nordsiek, P. Brandt, H. Blocker, G. Höfle, S. Beyer and R. Müller, *J. Biol. Chem.*, 1999, **274**, 37391.
32. I. Sattler, R. Thiericke and A. Zeeck, *Nat. Prod. Rep.*, 1998, **15**, 221.
33. R. Thiericke, A. Zeeck, A. Nakagawa, S. Omura, R. E. Herrold, S. T. S. Wu, J. M. Beale and H. G. Floss, *J. Am. Chem. Soc.*, 1990, **112**, 3979.
34. Y. Hu, PhD Dissertation, University of Washington, Seattle, 2000.
35. M. Ubukata, X. -C. Cheng, J. Uzawa and K. Isono, *J. Chem. Soc., Perkin Trans. I*, 1995, 2399.
36. D. G. I. Kingston, M. X. Kolpak, J. W. LeFevre and I. Borup-Grochtmann, *J. Am. Chem. Soc.*, 1983, **105**, 5106.
37. N. Bamas-Jacques, S. Lorenzon, P. Lacroix, C. de Swetschin and J. Crouzet, *J. Appl. Microbiol.*, 1999, **87**, 939.
38. J. W. LeFevre and D. G. I. Kingston, *J. Org. Chem.*, 1984, **49**, 2588.
39. Z. H. Ahmed, R. R. Fiala and M. W. Bullock, *J. Antibiot.*, 1993, **46**, 614.
40. C. Kotake, T. Yamasaki, T. Moriyama, M. Shinoda, N. Komiyama, T. Furumai, M. Konishi and T. Oki, *J. Antibiot.*, 1992, **45**, 1442.
41. B. S. Moore and D. Seng, *Tetrahedron Lett.*, 1998, **39**, 3915.
42. N. Sitachitta, B. L. Márquez, R. T. Williamson, J. Rossi, M. A. Roberts, W. H. Gerwick, V.-A. Nguyen and C. L. Willis, *Tetrahedron*, 2000, **56**, 9103.
43. H. Luesch, W. Y. Yoshida, R. E. Moore, V. J. Paul and T. H. Corbett, *J. Am. Chem. Soc.*, 2001, **123**, 5418.
44. B. S. Moore, *Nat. Prod. Rep.*, 1999, **16**, 653.
45. Y. Hayakawa, J. W. Kim, H. S.-Y. Adachi, K. K.-I. Fujita and H. Seto, *J. Am. Chem. Soc.*, 1998, **120**, 3524.

46 J. J. Lee, P. M. Dewick, C. P. Gorst-Allman, F. Spreafico, C. Kowal, C.-j. Chang, A. G. McInnes, J. A. Walter, P. J. Keller and H. G. Floss, *J. Am. Chem. Soc.*, 1987, **109**, 5426.

47 K. Kakinuma, N. Ikekawa, A. Nakagawa and S. Omura, *J. Am. Chem. Soc.*, 1979, **101**, 3402.

48 S. Fushimi, K. Furihata and H. Seto, *J. Antibiot.*, 1989, **42**, 1026.

49 T. Ozasa, K. Tanaka, M. Sasamata, H. Kaniwa, M. Shimizu, H. Matsumoto and M. Iwanami, *J. Antibiot.*, 1989, **42**, 1339.

50 T. Kohama, T. Nakamura, T. Kinoshita, I. Kaneko and A. Shiraishi, *J. Antibiot.*, 1993, **46**, 1512.

51 Y. Hu and H. G. Floss, *J. Antibiot.*, 2001, **54**, 340.

52 M. Damberg, P. Russ and A. Zeeck, *Tetrahedron Lett.*, 1982, 59.

53 B. Hippchen, A. Roell and K. Poralla, *Arch. Microbiol.*, 1981, **129**, 53.

54 B. S. Moore, K. Poralla and H. G. Floss, *J. Am. Chem. Soc.*, 1993, **115**, 5267.

55 B. S. Moore, H. Cho, R. Casati, E. Kennedy, K. A. Reynolds, J. M. Beale, U. Mocek and H. G. Floss, *J. Am. Chem. Soc.*, 1993, **115**, 5254.

56 B. S. Moore and H. G. Floss, *J. Nat. Prod.*, 1994, **57**, 382.

57 S. Handa and H. G. Floss, *Chem. Commun.*, 1997, 153.

58 H. G. Floss, *Nat. Prod. Rep.*, 1997, **14**, 433.

59 B. S. Moore and H. G. Floss, in ‘Polyketides and Other Secondary Metabolites Including Fatty Acids and Their Derivatives’, ed. D. Barton, K. Nakanishi and O. Meth-Cohn, *Comprehensive Natural Products Chemistry*, Elsevier, New York, 1999, vol. 1, pp. 61–82.

60 K. A. Reynolds, K. M. Fox, Z. Yuan and Y. Lam, *J. Am. Chem. Soc.*, 1991, **113**, 4339.

61 P. Wang, C. D. Denoya, M. R. Morgenstern, D. D. Skinner, K. K. Wallace, R. Digate, S. Patton, N. Banavali, G. Schuler, M. K. Speedie and K. A. Reynolds, *J. Bacteriol.*, 1996, **178**, 6873.

62 S. Chen, D. von Bamberg, V. Hale, M. Breuer, B. Hardt, R. Müller, H. G. Floss, K. A. Reynolds and E. Leistner, *Eur. J. Biochem.*, 1999, **261**, 98.

63 T. A. Cropp, D. J. Wilson and K. A. Reynolds, *Nat. Biotechnol.*, 2000, **18**, 980.

64 S. M. Patton, T. A. Cropp and K. A. Reynolds, *Biochemistry*, 2000, **39**, 7595.

65 C. Vezina, A. Kudelski and S. N. Sehgal, *J. Antibiot.*, 1975, **28**, 721.

66 S. N. Sehgal, H. Baker and C. Vezina, *J. Antibiot.*, 1975, **28**, 727.

67 H. Hatanaka, M. Iwami, T. Kino, T. Goto and M. Okuhara, *J. Antibiot.*, 1988, **41**, 1586.

68 H. Hatanaka, T. Kino, S. Miyata, N. Imunara, A. Kuroda, T. Goto, H. Tanaka and M. Okuhara, *J. Antibiot.*, 1988, **41**, 1592.

69 T. Kino, H. Hatanaka, M. Hashimoto, M. Nishiyama, T. Goto, M. Okuhara, M. Kohsaka, H. Aoki and H. Imanaka, *J. Antibiot.*, 1987, **40**, 1249.

70 T. Kino, H. Hatanaka, S. Miyata, N. Inamura, M. Nishiyama, T. Yajima, T. Goto, M. Okuhara, M. Kohsaka, H. Aoki and T. Ochiai, *J. Antibiot.*, 1987, **40**, 1256.

71 H. Tanaka, A. Kuroda, H. Murusawa, H. Hatanaka, T. Kino, T. Goto, M. Hashimoto and S. T. Taga, *J. Am. Chem. Soc.*, 1987, **109**, 5031.

72 N. L. Paiva, A. L. Demain and M. F. Roberts, *J. Nat. Prod.*, 1991, **54**, 167.

73 K. M. Byrne, A. Shaffee, J. B. Nielsen, B. Arison, R. L. Monaghan and L. Kaplan, *Dev. Ind. Microbiol.*, 1993, **32**, 29.

74 T. Schwecke, J. F. Aparicio, I. Molnar, A. König, L. E. Khaw, S. F. Haydock, M. Oliynyk, P. Caffrey, J. Cortes, J. B. Lester, G. A. Böhm, J. Staunton and P. F. Leadlay, *Proc. Nat. Acad. Sci. U.S.A.*, 1995, **92**, 7839.

75 I. Molnar, J. F. Aparicio, S. F. Haydock, L. E. Khaw, T. Schwecke, A. König, J. Staunton and P. F. Leadlay, *Gene*, 1996, **169**, 1.

76 J. F. Aparicio, I. Molnar, T. Schwecke, A. König, S. F. Haydock, L. E. Khaw, J. Staunton and P. F. Leadlay, *Gene*, 1996, **169**, 9.

77 K. Wu, L. Chung, W. P. Revill, L. Katz and C. D. Reeves, *Gene*, 2000, **251**, 81.

78 H. Motamedi, S. J. Cai, A. Shafiee and K. O. Elliston, *Eur. J. Biochem.*, 1997, **244**, 74.

79 H. Motamedi and A. Shafiee, *Eur. J. Biochem.*, 1998, **256**, 528.

80 A. Shafiee, H. Motamedi and T. Chen, *Eur. J. Biochem.*, 1994, **225**, 755.

81 N. L. Paiva, M. F. Roberts and A. L. Demain, *J. Ind. Microbiol.*, 1993, **12**, 423.

82 P. A. S. Lowden, G. A. Böhm, J. Staunton and P. F. Leadlay, *Angew. Chem., Int. Ed.*, 1996, **35**, 2249.

83 P. A. S. Lowden, B. Wilkinson, G. A. Böhm, S. Handa, H. G. Floss, P. F. Leadlay and J. Staunton, *Angew. Chem., Int. Ed.*, 2001, **40**, 777.

84 K. K. Wallace, K. A. Reynolds, K. Koch, H. A. I. McArthur, M. S. Brown, R. G. Wax and B. S. Moore, *J. Am. Chem. Soc.*, 1994, **116**, 11600.

85 K. A. Reynolds, K. K. Wallace, S. Handa, M. S. Brown, H. A. I. McArthur and H. G. Floss, *J. Antibiot.*, 1997, **50**, 701.

86 K. A. Reynolds, P. Wang, K. M. Fox, M. K. Speedie, Y. Lam and H. G. Floss, *J. Bacteriol.*, 1992, **174**, 3850.

87 N. Bedorf, D. Schomburg, K. Gerth, H. Reichenbach and G. Höfle, *Liebigs Ann. Chem.*, 1993, 1017.

88 A. M. Hill, J. P. Harris and A. P. Siskos, *Chem. Commun.*, 1998, 2361.

89 G. Höfle and H. Reichenbach, in *Sekundärmetabolismus bei Mikroorganismen*, ed. W. Kuhn and H.-P. Fiedler, Attempto Verlag, Tübingen, Germany, 1995, pp. 61–78.

90 T. Schupp, C. Toupet, B. Cluzel, S. Neff, S. Hill, J. J. Beck and J. M. Ligon, *J. Bacteriol.*, 1995, **177**, 3673.

91 J. M. Ligon T. Schupp J. J. Beck D. S. Hill S. Neff and J. A. Ryals, *US Pat.*, 5716849, USA, 1998.

92 C. Hertweck and B. S. Moore, *Tetrahedron*, 2000, **56**, 9115.

93 S. Schneider, M. E. Mohamed and G. Fuchs, *Arch. Microbiol.*, 1997, **168**, 310.

94 R. Jansen, P. Washausen, B. Kunze, H. Reichenbach and G. Höfle, *Eur. J. Org. Chem.*, 1999, 1085.

95 J. F. Martin, *Annu. Rev. Microbiol.*, 1977, **31**, 13.

96 T. Brautaset, O. N. Sekurova, H. Sletta, T. E. Ellingsen, A. R. Strom, S. Valla and S. B. Zotchev, *Chem. Biol.*, 2000, **7**, 395.

97 J. F. Aparicio, R. Fouces, M. V. Mendes, N. Olivera and J. F. Martin, *Chem. Biol.*, 2000, **7**, 895.

98 Z. H. Hu, K. Boa, X. F. Zhou, Q. Zhou, D. A. Hopwood, T. Kieser and Z. H. Deng, *Mol. Microbiol.*, 1994, **14**, 163.

99 J. A. Gil and D. A. Hopwood, *Gene*, 1983, **25**, 119.

100 L. M. Criado, J. F. Martin and J. A. Gil, *Gene*, 1993, **126**, 135.

101 J. A. Gil, L. M. Criado, M. T. Alegre and J. F. Martin, *FEMS Microbiol. Lett.*, 1990, **70**, 15.

102 J. M. Green and B. P. Nichols, *J. Biol. Chem.*, 1991, **266**, 12972.

103 J. A. Asturias, J. F. Martin and P. Liras, *J. Ind. Microbiol.*, 1994, **13**, 183.

104 K. L. Rinehart Jr. and L. S. Shield, *Fortschr. Chem. Org. Naturst.*, 1976, **33**, 231.

105 Y. Q. Mao, M. Varoglu and D. H. Sherman, *Chem. Biol.*, 1999, **6**, 251.

106 H. G. Floss and T.-W. Yu, *Curr. Opin. Chem. Biol.*, 1999, **3**, 592.

107 P. R. August, L. Tang, Y. J. Yoon, S. Ning, R. Müller, T.-W. Yu, M. Taylor, D. Hoffman, C.-G. Kim, X. Zhang, C. R. Hutchinson and H. G. Floss, *Chem. Biol.*, 1998, **5**, 69.

108 S. J. Admiraal, C. T. Walsh and C. Khosla, *Biochemistry*, 2001, **40**, 6116.

109 C. G. Kim, A. Kirschning, P. Bergon, P. Zhou, E. Su, B. Sauerbrei, S. Ning, Y. Ahn, M. Breuer, E. Leistner and H. G. Floss, *J. Am. Chem. Soc.*, 1996, **118**, 7486.

110 T.-W. Yu, R. Müller, M. Müller, X. Zhang, G. Draeger, C.-G. Kim, E. Leistner and H. G. Floss, *J. Biol. Chem.*, 2001, **276**, 12546.

111 C. G. Kim, T.-W. Yu, C. B. Fryhle, S. Handa and H. G. Floss, *J. Biol. Chem.*, 1998, **273**, 6030.

112 J. C. Eads, M. Beeby, G. Scapin, T.-W. Yu and H. G. Floss, *Biochemistry*, 1999, **38**, 9840.

113 J. K. Sohng, T. J. Oh, J. J. Lee and C. G. Kim, *Mol. Cells*, 1997, **7**, 674.

114 Y. D. Hu, C. R. Melville, S. J. Gould and H. G. Floss, *J. Am. Chem. Soc.*, 1997, **119**, 4301.

115 H. Cho, I. Sattler, J. M. Beale, A. Zeeck and H. G. Floss, *J. Org. Chem.*, 1993, **58**, 7925.

116 S. J. Gould, C. R. Melville and M. C. Cone, *J. Am. Chem. Soc.*, 1996, **118**, 9228.

117 M. C. Cone, C. R. Melville, J. R. Carney, M. P. Gore and S. J. Gould, *Tetrahedron*, 1995, **51**, 3095.

118 Y. F. Li, S. J. Gould and P. J. Proteau, *Tetrahedron Lett.*, 2000, **41**, 5181.

119 H. B. Bode and A. Zeeck, *J. Chem. Soc.*, *Perkin Trans. 1*, 2000, 323.

120 H. B. Bode and A. Zeeck, *J. Chem. Soc.*, *Perkin Trans. 1*, 2000, 2665.

121 S. J. Hammond, M. P. Williamson, D. H. Williams, L. D. Boeck and G. G. Marconi, *J. Chem. Soc.*, *Chem. Commun.*, 1982, 344.

122 A. M. A. van Wageningen, P. N. Kirkpatrick, D. H. Williams, B. R. Harris, J. K. Kershaw, N. J. Lennard, M. Jones, S. J. M. Jones and P. J. Solenberg, *Chem. Biol.*, 1998, **5**, 155.

123 Y. Hayakawa, K. Shin-ya, K. Furihata and H. Seto, *J. Am. Chem. Soc.*, 1993, **115**, 3014.

124 Y. Hayakawa, K. Shin-ya, K. Furihata and H. Seto, *J. Antibiot.*, 1993, **46**, 1563.

125 Y. Hayakawa, M. Matsuoka, K. Shin-ya and H. Seto, *J. Antibiot.*, 1993, **46**, 1557.

126 H. Augustiniak, H. Irschik, H. Reichenbach and G. Höfle, *Liebigs Ann.*, 1996, 1657.

127 R. E. Moore, J. L. Chen, B. S. Moore, G. M. L. Patterson and W. W. Carmichael, *J. Am. Chem. Soc.*, 1991, **113**, 5083.

128 B. W. Choi, M. Namikoshi, F. Sun, K. L. Rinehart, W. W. Carmichael, A. M. Kaup, W. R. Evans and V. R. Beasley, *Tetrahedron Lett.*, 1993, **34**, 7881.

129 T. Golakoti, J. Ogino, C. E. Heltzel, T. L. Husebo, C. M. Jensen, L. K. Larsen, G. M. L. Patterson, R. E. Moore, S. L. Mooberry, T. M. Corbett and F. A. Valeriote, *J. Am. Chem. Soc.*, 1995, **117**, 12030.

130 D. Tillett, E. Dittmann, M. Erhard, H. von Döhlen, T. Börner and B. A. Neilan, *Chem. Biol.*, 2000, **7**, 753.

131 T. Nishizawa, M. Asayama, K. Fujii, K. Harada and M. Shirai, *J. Biochem.*, 1999, **126**, 520.

132 T. Nishizawa, A. Ueda, M. Asayama, K. Fujii, K. Harada, K. Ochi and M. Shirai, *J. Biochem.*, 2000, **127**, 779.

133 L. Du, C. Sanchez and B. Shen, *Metab. Eng.*, 2001, **3**, 78.

134 M. Ishibashi, R. E. Moore, G. M. L. Patterson, C. Xu and J. Clardy, *J. Org. Chem.*, 1986, **51**, 5300.

135 S. Carmeli, R. E. Moore, G. M. L. Patterson and W. Y. Yoshida, *Tetrahedron Lett.*, 1993, **34**, 5571.

136 K. Kakinuma, J. Uzawa and M. Uramoto, *Tetrahedron Lett.*, 1982, **23**, 5303.

137 W. Kohl, H. Irschik, H. Reichenbach and G. Höfle, *Liebigs Ann. Chem.*, 1984, 1088.

138 N. Naruse, M. Konishi, T. Oki, Y. Inouye and H. Kakisawa, *J. Antibiot.*, 1991, **44**, 756.

139 V. R. Hegde, M. G. Patel, V. P. Gullo, A. K. Ganguly, O. Sarre, M. S. Puar and A. T. McPhail, *J. Am. Chem. Soc.*, 1990, **112**, 6403.

140 N. Naruse, T. Tsuno, Y. Sawada, M. Konishi and T. Oki, *J. Antibiot.*, 1991, **44**, 741.

141 M. S. Puar, V. Gullo, I. Gunnarsson, V. Hegde, M. Patel and J. Schwartz, *Bioorg. Med. Chem. Lett.*, 1992, **2**, 575.

142 M. Mayer and R. Thiericke, *J. Chem. Soc., Perkin Trans. 1*, 1993, 2525.

143 M. Zerlin and R. Thiericke, *J. Org. Chem.*, 1994, **59**, 6986.

144 S. Iwasaki, K. Sasaki, M. Namikoshi and S. Okuda, *Heterocycles*, 1982, **17**, 331.

145 K. Takesako and T. Beppu, *J. Antibiot.*, 1984, **37**, 1170.

146 D. Samain, J. Cook and K. L. Rinehart Jr., *J. Am. Chem. Soc.*, 1982, **104**, 4129.

147 K. Fukushima, T. Arai, S. Iwasaki, M. Namikoshi and S. Okuda, *J. Antibiot.*, 1982, **35**, 1480.

148 H. Koshio, K. Koinata, J. Uzawa, M. Uramoto, K. Isono and H. Osada, *Tetrahedron*, 1993, **49**, 8827.

149 H. He, B. Shen, J. Korshalla and G. T. Carter, *Tetrahedron*, 2001, **57**, 1189.

150 S. Sakuda, U. Guce-Bigol, M. Itoh, T. Nishimura and Y. Yamada, *J. Chem. Soc., Perkin Trans. 1*, 1996, 2315.

151 M. Otsuka, T. Eguchi, K. Shindo and K. Kakinuma, *Tetrahedron Lett.*, 1998, **39**, 3185.

152 M. Otsuka, M. Fujita, Y. Matsushima, T. Eguchi, K. Shindo and K. Kakinuma, *Tetrahedron*, 2000, **56**, 8281.

153 A. B. Smith, T. A. Rano, N. Chida, G. A. Sulikowski and J. L. Wood, *J. Am. Chem. Soc.*, 1992, **114**, 8008.

154 S. Omura, A. Nakagawa, K. Shibata and H. Sano, *Tetrahedron Lett.*, 1982, **23**, 4713.

155 K. D. Walker and H. G. Floss, *J. Am. Chem. Soc.*, 1998, **120**, 5333.

156 M. Nakagawa, Y. Toda, K. Furihata, Y. Hayakawa and H. Seto, *J. Antibiot.*, 1992, **45**, 1133.

157 J. Bolte, C. Demuyck, G. Jeminet, J. Juillard and C. Tissier, *Can. J. Chem.*, 1982, **60**, 981.

158 L. David and S. Emadzadeh, *J. Antibiot.*, 1982, **35**, 1616.

159 M. J. Zmijewski, R. Wong, J. W. Paschal and D. E. Dorman, *Tetrahedron*, 1983, **39**, 1255.

160 B. Nowak-Thompson, N. Chaney, J. S. Wing, S. J. Gould and J. E. Loper, *J. Bacteriol.*, 1999, **181**, 2166.

161 A. M. Cerdeno, M. J. Bibb and G. L. Challis, *Chem. Biol.*, 2001, **8**, 817.

162 C. W. Carreras and C. Khosla, *Biochemistry*, 1998, **37**, 2084.

163 T. S. Hitchman, J. Crosby, K. J. Byrom, R. J. Cox and T. J. Simpson, *Chem. Biol.*, 1998, **5**, 35.

164 W. L. Bao, E. Wendt-Pienkowski and C. R. Hutchinson, *Biochemistry*, 1998, **37**, 8132.

165 C. Binnie, M. Warren and M. Butler, *J. Bacteriol.*, 1989, **171**, 887.

166 P. M. Rhodes, N. Winskill, E. J. Friend and M. Warren, *Gen. Microbiol.*, 1981, **124**, 329.

167 I. S. Hunter and R. A. Hill, in *Biotechnology of Antibiotics*, ed. W. R. Strohl, Marcel Dekker, Inc., New York, 2nd edn., 1997, vol. 82, pp. 659–682.

168 R. Thomas and D. J. Williams, *J. Chem. Soc., Chem. Commun.*, 1983, 667.

169 H. Fu, S. Ebert-Khosla, D. A. Hopwood and C. Khosla, *J. Am. Chem. Soc.*, 1994, **116**, 6443.

170 H. Petkovic, A. Thamchaipenet, L. H. Zhou, D. Hranueli, P. Raspor, P. G. Waterman and I. S. Hunter, *J. Biol. Chem.*, 1999, **274**, 32829.

171 H. Bockholt, G. Udvarnoki, J. Rohr, U. Mocek, J. M. Beale and H. G. Floss, *J. Org. Chem.*, 1994, **59**, 2064.

172 I. Fujii and Y. Ebizuka, *Chem. Rev.*, 1997, **97**, 2511.

173 A. Yoshimoto, T. Matzuzawa, T. Oki, T. Takeuchi and H. Umezawa, *J. Antibiot.*, 1981, **34**, 951.

174 C. R. Hutchinson, *Chem. Rev.*, 1997, **97**, 2525.

175 A. Grimm, K. Madduri, A. Ali and C. R. Hutchinson, *Gene*, 1994, **151**, 1.

176 K. J. Weissman, M. Bycroft, J. Staunton and P. F. Leadlay, *Biochemistry*, 1998, **37**, 11012.

177 J. Ye, M. L. Dickens, R. Plater, Y. Li, J. Lawrence and W. R. Strohl, *J. Bacteriol.*, 1994, **176**, 6270.

178 L. Han, S. Lobo and K. Reynolds, *J. Bacteriol.*, 1998, **180**, 4481.

179 G. Meurer and C. R. Hutchinson, *J. Am. Chem. Soc.*, 1995, **117**, 5899.

180 V. B. Rajgarhia and W. R. Strohl, *J. Bacteriol.*, 1997, **179**, 2690.

181 W. Bao, P. J. Sheldon, E. Wendt-Pienkowski and C. R. Hutchinson, *J. Bacteriol.*, 1999, **181**, 4690.

182 W. Bao, P. J. Sheldon and C. R. Hutchinson, *Biochemistry*, 1999, **38**, 9752.

183 R. G. Summers, A. Ali, B. Shen, W. A. Wessel and C. R. Hutchinson, *Biochemistry*, 1995, **34**, 9389.

184 Y. Hori, S. Takase, N. Shigematsu, T. Goto, M. Okuhara and M. Kohsaka, *J. Antibiot.*, 1993, **46**, 1063.

185 Y. Hori, Y. Abe, M. Ezaki, T. Goto, M. Okuhara and M. Kohsaka, *J. Antibiot.*, 1993, **46**, 1155.

186 T. Marti, Z. Hu, N. L. Pohl, A. N. Shah and C. Khosla, *J. Biol. Chem.*, 2000, **275**, 33443.

187 J. T. Tsay, W. Oh, T. J. Larson, S. Jackowski and C. O. Rock, *J. Biol. Chem.*, 1992, **267**, 6807.

188 M. J. Bibb, D. H. Sherman, S. Omura and D. A. Hopwood, *Gene*, 1994, **142**, 31.

189 K. Tsuzuki, Y. Iwai and S. Omura, *J. Antibiot.*, 1986, **39**, 1343.

190 R. McDaniel, S. Ebert-Khosla, D. A. Hopwood and C. Khosla, *J. Am. Chem. Soc.*, 1993, **115**, 11671.

191 R. McDaniel, S. Ebert-Khosla, D. A. Hopwood and C. Khosla, *Science*, 1993, **262**, 1546.

192 S. J. Gould, C. R. Melville and J. Chen, *Tetrahedron*, 1997, **53**, 4561.

193 K. Kondo, T. Eguchi, K. Kakinuma, K. Mizoue and Y.-F. Qiao, *J. Antibiot.*, 1998, **51**, 288.

194 A. Fujiwara, T. Hoshino, M. Tazoe and M. Fujiwara, *J. Antibiot.*, 1981, **34**, 608.

195 T. Aoyama, H. Naganawa, Y. Muraoka, T. Aoyagi and T. Takeuchi, *J. Antibiot.*, 1992, **45**, 1767.

196 N. Miyairi, H.-I. Sakai, T. Konomi and H. Imanaka, *J. Antibiot.*, 1976, **29**, 227.

197 Y. Tokuma, N. Miyairi and Y. Morimoto, *J. Antibiot.*, 1976, **29**, 1114.

198 H. Seto, T. Sato, S. Urano, J. Uzawa and H. Yonehara, *Tetrahedron Lett.*, 1976, **17**, 4367.

199 N. Sitachitta, M. Gadepalli and B. S. Davidson, *Tetrahedron*, 1996, **52**, 8073.

200 J. Piel, K. Hoang and B. S. Moore, *J. Am. Chem. Soc.*, 2000, **122**, 5415.

201 J. Piel, C. Hertweck, P. Shipley, D. S. Hunt, M. S. Newman and B. S. Moore, *Chem. Biol.*, 2000, **7**, 943.

202 D. M. Ribnicky, V. Shulaev and I. Raskin, *Plant Physiol.*, 1998, **118**, 565.

203 A. P. Jarvis, O. Schaaf and N. J. Oldham, *Planta*, 2000, **212**, 119.

204 H. Lee, J. Leon and I. Raskin, *Proc. Natl. Acad. Sci. U.S.A.*, 1995, **92**, 4076.

205 A. Mitra, Y. Kitamura, M. J. Gasson, A. Narbad, A. J. Parr, J. Payne, M. J. C. Rhodes, C. Sewter and N. J. Walton, *Arch. Biochem. Biophys.*, 1999, **365**, 10.

206 G. S. Bezanson, D. Desaty, A. V. Emes and L. C. Vining, *Can. J. Microbiol.*, 1970, **16**, 147.

207 C. Hertweck, A. P. Jarvis, L. Xiang, B. S. Moore and N. J. Oldham, *ChemBioChem*, 2001, **2**, 784.

208 J. A. Bjorklund and E. Leete, * Phytochemistry*, 1992, **31**, 3883.

209 F. Aragozzini, R. Craveri, E. Maconi, G. S. Ricca and C. Scolastico, *J. Chem. Soc., Perkin Trans. 1*, 1988, 1865.

210 L. Serino, C. Reimmann, H. Baur, M. Beyeler, P. Visca and D. Haas, *Mol. Gen. Genet.*, 1995, **249**, 217.

211 R. Thiericke and J. Rohr, *Nat. Prod. Rep.*, 1993, **10**, 265.

212 K. L. Rinehart, *Pure Appl. Chem.*, 1977, **49**, 1361.

213 S. J. Daum and J. R. Lemke, *Annu. Rev. Microbiol.*, 1979, **33**, 241.

214 I. O. Kibwage, G. Janssen, R. Busson, J. Hoogmartens and H. Vanderhaeghe, *J. Antibiot.*, 1986, **40**, 1.

215 J. J. Jacobsen, A. T. Keatinge-Clay, D. E. Cane and C. Khosla, *Bioorg. Med. Chem.*, 1998, **6**, 1171.

216 M. S. Pacey, J. P. Dirlam, R. W. Geldart, P. F. Leadlay, H. A. I. McArthur, E. L. McCormick, R. A. Monday, T. N. O'Connell, J. Staunton and T. J. Winchester, *J. Antibiot.*, 1998, **51**, 1029.

217 M. S. Brown, J. P. Dirlam, H. A. I. McArthur, E. L. McCormick, B. K. Morse, P. A. Murphy, T. O'Connell, M. Pacey, D. M. Rescek, J. Ruddock and R. G. Wax, *J. Antibiot.*, 1999, **52**, 742.

218 K. E. H. Wiesmann, J. Cortes, M. J. B. Brown, A. L. Cutter, J. Staunton and P. F. Leadlay, *Chem. Biol.*, 1995, **2**, 583.

219 R. Pieper, G. Luo, D. E. Cane and C. Khosla, *J. Am. Chem. Soc.*, 1995, **117**, 11373.

220 R. Pieper, S. Ebert-Khosla, D. Cane and C. Khosla, *Biochemistry*, 1996, **35**, 2054.

221 T. S. Chen, E. S. Inamine, O. D. Hensens, D. Zink and D. A. Ostlind, *Arch. Biochem. Biophys.*, 1989, **269**, 544.

222 M. D. Schulman, D. Valentino and O. Hensens, *J. Antibiot.*, 1986, **39**, 541.

223 R. Kind, PhD Dissertation, University of Göttingen, Germany, 1991.

224 R. Thiericke and A. Zeeck, *J. Chem. Soc., Perkin Trans. 1*, 1988, 2133.

225 R. Thiericke, H.-J. Langer and A. Zeeck, *J. Chem. Soc., Perkin Trans. 1*, 1989, 2133.

226 A. Kawashima, H. Seto, M. Kato, K. Uchida and N. Otake, *J. Antibiot.*, 1985, **38**, 1499.

227 J. Jacobsen, C. R. Hutchinson, D. E. Cane and C. Khosla, *Science*, 1997, **277**, 367.

228 D. Hunziker, N. Wu, K. Kenoshita, D. E. Cane and C. Khosla, *Tetrahedron Lett.*, 1999, **40**, 635.

229 T. Leaf, L. Cadapan, C. Carreras, R. Regentin, S. Ou, E. Woo, G. Ashley and P. Licari, *Biotech. Progr.*, 2000, **16**, 553.

230 K. J. Weissman, M. Bycroft, A. L. Cutter, U. Hanefeld, E. J. Frost, M. C. Timoney, R. Harris, S. Handa, M. Roddis, J. Staunton and P. F. Leadlay, *Chem. Biol.*, 1998, **5**, 743.

231 J. F. Geissler, C. S. Harwood and J. Gibson, *J. Bacteriol.*, 1988, **170**, 1709.

232 U. Altenschmidt, B. Oswald and G. Fuchs, *J. Bacteriol.*, 1991, **173**, 5494.

233 O. Ghisalba, H. Furrer, W. Richter and S. Moss, *J. Antibiot.*, 1981, **34**, 58.

234 D. Hunziker, T.-W. Yu, C. R. Hutchinson, H. G. Floss and C. Khosla, *J. Am. Chem. Soc.*, 1998, **120**, 1092.

235 T.-W. Yu, Y. Shen, Y. Doi-Katayama, L. Tang, C. Park, B. S. Moore, C. R. Hutchinson and H. G. Floss, *Proc. Natl. Acad. Sci. U.S.A.*, 1999, **96**, 9051.

236 C. J. Dutton, S. P. Gibson, A. C. Goudie, K. S. Holdom, M. S. Pacey, J. C. Ruddock, J. D. Bu'Lock and M. K. Richards, *J. Antibiot.*, 1991, **44**, 357.

237 H. A. I. McArthur, *Developments in Industrial Microbiology—BMP '97*, ed. C. R. Hutchinson and J. McAlpine, Society for Industrial Microbiology, Fairfax, VA, 1998, pp. 43–48.

238 W. Traeder, *Tierärztl. Umsch.*, 1994, **49**, 465.

239 A. F. A. Marsden, B. Wilkinson, J. Cortes, N. J. Dunster, J. Staunton and P. F. Leadlay, *Science*, 1998, **279**, 199.

240 I. C. Parsons, J. R. Everett, M. S. Pacey, J. C. Ruddock, A. G. Swanson and C. M. Thompson, *J. Antibiot.*, 1999, **52**, 190.
