Molecular Microbiology (2002) 43(5), 1215–1225

# Engineering specificity of starter unit selection by the erythromycin-producing polyketide synthase

Paul F. Long,$^{1†}$ Christopher J. Wilkinson,$^{1}$ Christian P. Bisang,$^{1‡}$ Jesús Cortés,$^{1§}$ Nicholas Dunster,$^{1\|}$ Marko Oliynyk,$^{1}$ Ellen McCormick,$^{2}$ Hamish McArthur,$^{2}$ Carmen Mendez,$^{3}$ José A. Salas,$^{3}$ James Staunton$^{4}$ and Peter F. Leadlay$^{1*}$

$^{1}$ Cambridge Centre for Molecular Recognition and Department of Biochemistry, University of Cambridge, Tennis Court Road, Cambridge CB2 1GA, UK.

$^{2}$ Bioprocess Research, Pfizer Inc., Central Research, Eastern Point Road, Groton, CT 06340, USA.

$^{3}$ Departamento de Biología Funcional, Universidad de Oviedo, 33006 Oviedo, Spain.

$^{4}$ Cambridge Centre for Molecular Recognition and Department of Chemistry, University of Cambridge, Lensfield Road, Cambridge CB2 9EY, UK.

## Summary

Chain initiation on many modular polyketide syntheses is mediated by acyl transfer from the CoA ester of a dicarboxylic acid, followed by decarboxylation *in situ* by KSQ, a ketosynthase-like decarboxylase domain. Consistent with this, the acyltransferase (AT) domains of all KSQ-containing loading modules are shown here to contain a key arginine residue at their active site. Site-specific replacement of this arginine residue in the oleandomycin (*ole*) loading AT domain effectively abolished AT activity, consistent with its importance for catalysis. Substitution of the *ole* PKS loading module, or of the tylosin PKS loading module, for the erythromycin (*ery*) loading module gave polyketide products almost wholly either acetate derived or propionate derived, respectively, instead of the mixture found normally. An authentic extension module AT domain, *rap* AT2 from the rapamycin PKS, functioned appropriately when engineered in the place of the *ole* loading AT domain, and gave rise to substantial amounts of C13-methylerythromycins, as predicted. The role of direct acylation of the ketosynthase domain of extension module 1 in chain initiation was confirmed by demonstrating that a mutant of the triketide synthase DEBS1-TE, in which the 4′-phosphopantetheine attachment site for starter acyl groups was specifically removed, produced triketide lactone products in detectable amounts.

## Introduction

Polyketides are a large and structurally diverse family of bioactive natural products whose biosynthesis, like that of fatty acids, involves the head-to-tail condensation of organic acid units catalysed by multifunctional enzymes called polyketide synthases (PKSs) (for a review, see Hopwood, 1997). The PKSs that synthesize macrolides and other complex polyketides are distinctive in consisting of giant multifunctional proteins containing discrete sets or modules of enzymes. Each module catalyses a single specific cycle of polyketide chain extension, as first shown for the erythromycin PKS of *Saccharopolyspora erythraea* (Cortés *et al.*, 1990; Donadio *et al.*, 1991). The first module of the erythromycin (*ery*) PKS was initially proposed to contain, at the N-terminus of the first multienzyme subunit DEBS1, an acyltransferase (AT) domain and an acylcarrier protein (ACP) domain involved in loading the propionate starter unit and providing this unit for condensation (Donadio *et al.*, 1991). However, subsequent work established that the N-terminal AT and ACP domains function together independently of extension module 1 and constitute a genuine loading module (Aparicio *et al.*, 1994) (Fig. 1), for which propionyl-CoA is the natural substrate. Under normal conditions *in vivo*, only propionate units are recruited by the DEBS loading module but, in recombinant cells or with the purified PKS *in vitro*, several other acyl-CoA esters are also accepted as starter units (Kao *et al.*, 1994; Brown *et al.*, 1995; Pieper *et al.*, 1995; Wiesmann *et al.*, 1995; Lau *et al.*, 2000).

Subsequent sequencing of genes from *Streptomyces avermitilis* governing production of the antiparasitic polyketide avermectin (MacNeil *et al.*, 1994; Marsden *et al.*, 1998) showed that the loading module for this PKS contained the same type of loading module, comprising an AT–ACP didomain N-terminal of the first extension module (Fig. 1). The avermectin-producing PKS is

Accepted 20 November, 2001. *For correspondence.* E-mail pfl10@mole.bio.cam.ac.uk; Tel. (+44) 1223 333656; Fax (+44) 1223 766091. Present addresses: $^{\dagger}$School of Pharmacy, University of London, 29/39 Brunswick Square, Bloomsbury, London WC1N 1AX, UK. $^{\ddagger}$Basilea Pharmaceutica Ltd, Postfach 3255, CH-4002 Basel, Switzerland. $^{\S}$Bioprocessing Unit, GlaxoSmithKline Research and Development, Medicines Research Centre, Gunnels Wood Road, Stevenage, Hertfordshire SG1 2NY, UK. $^{\|}$PanTherix Ltd, Kelvin Campus, West of Scotland Science Park, Glasgow G20 0SP, UK.

© 2002 Blackwell Science Ltd
the molecular basis for starter unit selection on these enzymes therefore has important consequences for our ability to manipulate these enzymes to produce new polyketide antibiotics.

An alternative paradigm for chain initiation on modular PKSs has been provided by the rapamycin PKS from Streptomyces hygroscopicus, in which a carboxylic acid is activated as the acyladenylate within the loading module (Schwecke et al., 1995; Aparicio et al., 1996; Lowden et al., 1996). The same strategy is seen for the initiation of pimaricin biosynthesis in Streptomyces natalensis (Aparicio et al., 1999) and for the biosynthesis of rifamycin in Amycolatopsis mediterranei (August et al., 1998; Schupp et al., 1998). However, the majority of type I modular PKS clusters sequenced so far possess loading modules that contain an additional N-terminal domain termed KSQ and resembling a ketosynthase domain. These clusters include those for niddamycin (Kakavas et al., 1997), tylosin (Kuhstoss et al., 1996; GenBank accession no. U78289), pikromycin/methymycin (Xue et al., 1998), spinosyn (Waldron et al., 2001) and the polyether antibiotic monensin (M. Oliynyk and P. F. Leadlay, manuscript in preparation). Each KSQ domain closely resembles a KS domain but, in all cases, the active-site cysteine residue within the conserved amino acid sequence VDTACSSS is replaced by glutamine (Q).

It has been proposed recently that polyketide chain initiation on these modular PKSs proceeds by way of KSQ-catalysed decarboxylation of CoA-esters of dicarboxylic acids that are recruited from the intracellular pool by the AT of the loading module (Bisang et al., 1999; Witkowski et al., 1999) (Fig. 1), and this has been demonstrated directly for the monensin PKS loading module (Bisang et al., 1999). This KSQ-containing loading module was shown to have a strict substrate specificity, even when studied *in vitro* with purified multienzyme, in contrast to the relaxed specificity of the ave or ery PKS loading modules, which lack the KSQ domain. We report here further experimental tests of the hypothesis that KSQ-containing modules load extender units, which are decarboxylated *in situ* to provide PKS-bound starter units. We also attempted to alter the substrate specificity of both the ery and the ole loading modules by site-specific mutagenesis of a key arginine residue previously implicated in the discrimination by fatty acid synthase between carboxylated and non-carboxylated acyl-CoA esters (Rangan and Smith, 1997).

Decarboxylation has been invoked previously by others as a significant mechanism for chain initiation on modular PKSs (Pieper et al., 1997), but that mechanism involved the action of the KS1 active site on methylmalonyl units bound in the first extension module. Subsequent investigation has shown that this mechanism

© 2002 Blackwell Science Ltd, Molecular Microbiology, **43**, 1215–1225
thetic group, which is required for covalent attachment of the starter acyl unit.

# Results

## The oleandomycin PKS loading module

Oleandomycin is a 14-membered macrolide whose precursor aglycone, 8,8a-deoxyoleandrolide, differs from 6-deoxyerythronolide B, the equivalent precursor of erythromycin A, only in having an acetate rather than a propionate as starter unit (Fig. 2). However, when we determined the DNA sequence of the loading module of the oleandomycin PKS, contrary to our expectations the loading module of the oleandomycin (ole) PKS contained a KSQ domain, in addition to an AT and ACP. Nearly 6.2 kb of continuous DNA from the oleandomycin cluster was sequenced as shown in Fig. 3. The presence of two incomplete open reading frames (ORFs) was readily deduced. More recently, the entire nucleotide sequence for one of these ORFs, designated *oleAI*, has been reported (Shah *et al.*, 2000; GenBank accession no. AF220951), so the sequence is not discussed here in detail. The two sequences differ by two bases: T5276 versus C4222; and C5184 versus T4130. Either of two methionine codons at nucleotide positions 1584 and 1677 could act as the start codon for *oleAI*. The second is more likely, based on the recognition of a purine-rich potential ribosome binding site motif 5′-¹⁶⁵⁸AGAGAGGTGG-3′ (Strohl, 1992) 19 bp upstream of this putative ATG start codon. Fingerprint motifs for KS, AT and ACP domains of

### oleandomycin biosynthetic cluster

![Diagram](https://i.imgur.com/placeholder.png)

Fig. 3. The oleandomycin biosynthetic gene cluster as published recently (GenBank accession nos AF055579; AF220951; SAPKS3ORF) (top) and the portion sequenced for this work. x = any; X = Arg, Ser, Ala or Glu; h = hydrophobic.

does not operate, at least *in vitro* (Weissman *et al.*, 1998); and separately Katz and colleagues (Pereda *et al.*, 1998) have provided evidence against decarboxylation by KS1 and in support of direct acylation of KS1 as a plausible alternative source of starter units. As part of the present study, we have reinvestigated the role of direct KS1 loading in the provision of starter units for the ery PKS, using site-specific mutagenesis of the loading module ACP domain to remove the 4′-phosphopantetheine pros-
PKSs were readily identifiable from the 4.5 kb of the remaining sequence available using the erythromycin-producing PKS as a reference for domain boundaries (Aparicio et al., 1994; 1996).

The second ORF starts with the ATG codon at position 1088. The full 2304 nucleotide sequence for this ORF, designated *oleR*, has been described recently, and its likely function as a β-glucosidase responsible for reactivation of glycosylated oleandomycin has been confirmed by in vitro studies (Quirós et al., 1998). Comparison with this sequence revealed two base mismatches with the deposited sequence at positions A979 (G14876) and G980 (A14877) (GenBank accession no. AF055579). We noted an alternative start codon to the one proposed previously (nucleotide position 872). The realization that the *ole* loading module contained a KSQ domain, despite the close resemblance of the product to erythromycin (Fig. 2), prompted the use of this KSQ domain in the experiments described below.

### Differences between the ATs of PKS loading modules

The availability of DNA sequence information from the *ole* PKS and from other PKSs containing KSQ-type loading domains now permits a detailed comparison between the primary sequences of the AT domains of these loading modules, which are hypothesized to be specific for CoA esters of dicarboxylic rather than monocarboxylic acids.

| -specificity motif-- | active site | 117 |
|----------------------|-------------|-----|
| RAPSAT08             | ETGYAQPALFALQVALFGLL ESWGVR.PD.AVV GHSVG ELAAGYVSGLWSLEDACTLVSA R A |
| RAPSAT12             | ETGYAQPALFAMQVALFGLL ESWGVR.PD.AVI GHSVG ELAAAYVSGVWSLEDACTLVSA R A |
| RAPSAT02             | ETGYAQPALFALQVALFGLL ESWGVR.PD.AVV GHSVG ELAAGYVSGLWSLEDACTLVSA R A |
| RAPSAT11             | ETGYAQPALFALQVALFGLL ESWGVR.PD.AVI GHSVG ELAAAYVSGVWSLEDACTLVSA R A |
| RAPSAT09             | ETGYAQPALFALQVALFGLL ESWGVR.PD.AVI GHSVG ELAAAYVSGLWSLEDACTLVSA R A |
| RAPSAT05             | ETGYAQPALFALQVALFGLL ESWGVR.PD.AVV GHSVG ELAAGYVSGLWSLEDACTLVSA R A |
| OLESAT00             | ETHYTQAALFALETALFRLI VQWGLK.PD.HLA GHSVG EIAAAHAAGILDLSDAAELVAT R G |
| SORSAT02             | QTAFTQPALFALEVALFELL QSFLAG.PA.LLL GHSIG ELVAAHVAGVLSLQDACTLVAA R A |
| RAPSAT14             | DTLYAQAGIFAMEAALFGLL EDWGVR.PD.FVA GHSIG EATAAYASGMLSLENVTTLIVA R G |
| AVESAT00             | RVDVVQPTLFAVMISLAALW RSQGVE.PC.AVL GHSLG EIAAAHVSAGGLSLADAARVVTL W S |
| DEBSAT00             | RVEVVQPALFAVQTSLAALW RSFGVT.PD.AVV GHSIG ELAAAHVCGGAAGAADAAARAAL W S |
| DEBSAT02             | RVDVVQPVLFAVMVSLARLW RACGAV.PS.AVI GHSQG EIAAAVVAGALSLEDGMRVVVAR R S |
| DEBSAT06             | RVDVVQPVLFSVMVSLARLW GACGVS.PS.AVI GHSQG EIAAAVVAGVLSLEDGVRVVAL R A |
| DEBSAT03             | RVDVVQPVFAVMVSLAELW RSYGVE.PA.AVV GHSQG EIAAAHVAGALTLEDAAKLVVG R S |
| DEBSAT05             | RVDVVQPALFAVMVSLAALW RSHGVE.PA.AVV GHSQG EIAAAHVAGALTLEDAAKLVAV R S |
| DEBSAT04             | RVDVLQPVLFSIMVSLAELW RAHGVT.PA.AVV GHSQG EIAAAHVAGALSLEAAKVVAL R S |
| DEBSAT01             | RVDVVQPVMFAVMVSLASMW RAHGVE.PA.AVI GHSQG EIAAACVAGALSLDDAARVVVAL R S |
| RAPSAT01             | RVDVVQPASWAVMVSLAAVW QAAGVR.PD.AVV GHSQG EIAAACVAGAVSLRDAARVVTL R S |
| RAPSAT07             | RVDVVQPASWAVMVSLAAVW QADGVR.PD.AVI GHSQG EIAAACVAGAVSLRDAARSVTL R S |
| RAPSAT03             | RVDVVQPASWAVMVSLAAVW QAAGVR.PD.AVI GHSQG EIAAACVAGAVSLRDAARIVTL R S |
| RAPSAT13             | RVDVVQPASWAMMVSLAAVW QAAGVR.PD.AVI GHSQG EIAAACVAGAVSLRDAARIVTL R S |
| RAPSAT06             | RVDVVQPASWRMMVSLAAVW QAAGVR.PD.AVI GHSQG EIAAACVAGAVSMRDAARIVTL R S |
| RAPSAT04             | RVDVVQPASWAVMVSLAAVW QAAGVR.PD.AVI GHSQG EIAAACVAGAVSLRDAARIVTL R S |
| RAPSAT10             | RVDVVQPASWAVMVSLAAVW QAAGVR.PD.AVI GHSQG EIAAACVAGAVSMRDAARIVTL R S |
| TYLSAT00             | RVDVVQPVTWAVMVSLARYW QAMGVD.VA.AVV GHSQG EIAAATVAGALSLEAAAVVAL R A |
| SORSAT01             | EIDVSLPAIISIEIALAAQW RAWGVE.PA.FVV GHSTG EIAAAHVAGVLSIEDAMRTICA Y G |
| EcoMCAT              | KTWQTQPALLTASVALYRVW QQGGKAPA.MMA GHSLG EYSALVCAGVIDFADAVRLVEM R G |
| RatMAAT              | DIVHSFVSLTAIQIALIDLL TSMGLKPDGFTII GHSLG EVACGYADGCLSQREAVLAAYW R G |

**consensus sequences:**

Propionate RVDVVxxxxxxMxSxAXhW  
Acetate     ETGYAxxxxxxQxAxFGLL  

**Fig. 4.** Comparison of the amino acid sequences of acyltransferase (AT) domains from modular polyketide synthases. RAPS, rapamycin; DEBS, erythromycin, AVES, avermectin, OLES, oleandomycin, TYLS, tylosin, SORS, soraphen, *Eco*MCAT, fatty acid synthase AT of *E. coli*; RatMAAT, AT of the rat fatty acid synthase. Sequences of AT domains from loading modules are in bold font. The specificity (Haydock et al., 1995), the active-site motif and position 117 (*E. coli* numbering) are labelled.

Table 1. Strains of S. erythraea and plasmids used in this work.

| Strains | Characteristics | Source |
| --- | --- | --- |
| NRRL 2338 | Wild-type erythromycin producer, red variant | Hessler et al. (1997) |
| JC2 | ΔeryA | Rowe et al. (1998) |

| Plasmids | Characteristics | Source |
| --- | --- | --- |
| pCJR24 | Integrative plasmid in which cloned genes are placed under the control of the heterologous actI/actII-ORF4 promoter | Rowe et al. (1998) |
| pNTEPH | pT7-7-based vector expressing DEBS1-TE | Böhm et al. (1998) |
| pKSW1 | pT7-7-based vector expressing DEBS1-TE in which the KS1 domain is flanked by unique PstI and HindIII sites | Böhm et al. (1998) |
| pCB117 | Derivative of pCJR24 which expresses DEBS1-TE containing entire ole loading module instead of eryAT0-eryACP0 | This study |
| pCJW111 | Derivative of pCJR24 which expresses DEBS1-TE containing entire tyl loading module instead of eryAT0-eryACP0 | This study |
| pND4 | Derivative of pCJR24 which expresses DEBS1-TE bearing the mutation S493A in the ACP domain of the loading module (ACP0) | This study |
| pCB151 | Derivative of pCJR24 which expresses DEBS1-TE containing ole loading module bearing KSQ (Q173A) mutation | This study |
| pCJW104 | pCJR24 which expresses DEBS1-TE containing ole loading module bearing KSQ (Q173A) mutation and AT0 (R117W) mutation | This study |
| pCJW86 | Derivative of pCJR24 which expresses DEBS1-TE bearing the mutation W117R in the AT0 domain | This study |
| pCJW89 | Derivative of pCJR24 which expresses DEBS1-TE containing the loading module tyl/KSQ eryAT0 (W117R) eryACP0 | This study |
| pPFL35 | Derivative of pCJR24 which expresses DEBS1-TE containing in the loading module ole KSQ rap AT2 instead of eryAT0 | This study |

---

ATs from KSQ-containing loading modules do in fact align well with extender ATs (Fig. 4) and, in particular, they contain a conserved arginine residue [residue 117 in *Escherichia coli* malonyl-CoA:ACP acyltransferase (MCAT) numbering], which Smith and colleagues have shown to be involved in substrate selection by the rat fatty acid synthase AT domain (Rangan and Smith, 1997). The X-ray crystal structure of the *E. coli* MCAT (Serre et al., 1995) has revealed that this arginine residue lies within the active site, and it probably interacts with the carboxyl group of methylmalonyl-CoA or malonyl-CoA (Rangan and Smith, 1997). Consistent with this, Fig. 4 shows that, in the sequences of the ATs of the loading modules of the erythromycin and avermectin PKSs, which are specific for the CoA esters of monocarboxylic acids, this conserved arginine residue is substituted by a hydrophobic tryptophan residue. The first two AT domains from the soraphen PKS, which are in an unusual tandem arrangement (Schupp et al., 1995), can also be tentatively identified on this basis. The first AT of the soraphen PKS (AT1a) has a tyrosine residue at position 117, consistent with its involvement in selection of the presumed precursor of the soraphen starter unit, benzoyl-CoA. The second AT of the soraphen PKS (AT1b) possesses an arginine residue and aligns well with the malonyl-CoA specific ATs in a region previously identified as diagnostic for the specificity of such ATs (Haydock et al., 1995), consistent with this AT domain governing the selection of malonyl-CoA to provide the first extension unit in soraphen biosynthesis.

A hybrid DEBS1-TE containing the ole loading module produces triketide lactone derived almost exclusively from an acetate starter unit

The ole loading module (KSQ-AT0-ACP0) was used to replace the DEBS loading module (AT0-ACP0) in the model PKS system DEBS1-TE (Wiesmann et al., 1995). This hybrid PKS was overexpressed from plasmid pCB117 (Table 1) in *S. erythraea* JC2, a strain from which the resident DEBS genes have been effectively deleted, and the polyketide products of fermentation (Fig. 2) were analysed by gas chromatography–mass spectrometry (GC-MS). The predicted triketide lactone 2 with acetate-derived starter unit was observed in good yield (essentially the same total production of triketide lactone as DEBS1-TE). Less than 2% of the triketide lactone produced was 1 containing starter units derived from propionate. In contrast, DEBS1-TE under the same conditions gives a molar ratio of 2:1 to 4:1 for triketide lactone products 1 and 2 (Fig. 2) respectively (Rowe et al., 1998).

A hybrid DEBS1-TE containing the tyl loading module produces triketide lactone exclusively derived from a propionate starter unit

We also replaced the loading module in DEBS1-TE with that from the tylosin PKS (Kuhstoss et al., 1996); GenBank accession no. U78289), which specifies a propionate starter unit for this 16-membered macrolide in its

natural context. Initial experiments, using the start codon of *tylGl* previously assigned in the published database sequence, failed (data not shown). However, the use of an alternative start codon, located 55 bp downstream, produced a functional protein. When this was overexpressed in *S*. erythraea JC2 from plasmid pCJW111, only triketide lactone 1 (Fig. 2) derived from the use of propionate as starter unit was observed, and in good yield (essentially the same levels of production of triketide lactone as DEBS1-TE, 50–60 mg l⁻¹). No triketide lactones derived from the use of acetate as starter unit could be detected.

Site-specific mutation of the 4′-phosphopantetheine attachment site in the ACP domain of the loading module of DEBS1-TE and its effects on triketide lactone biosynthesis

A mutation was introduced into DEBS1-TE (Wiesmann et al., 1995) using site-specific mutagenesis to change to alanine the specific serine residue (Ser-493) that provides the attachment site for the 4′-phosphopantetheine prosthetic group. The resulting triketide lactone synthase was therefore unable to provide propionate units from the loading module to KS1 by the normal mechanism. The mutant was expressed in *S*. erythraea JC2 from plasmid pND4 (*Experimental procedures*), and the products were analysed using GC-MS. Triketide lactones 1 (bearing a propionate starter unit) and 2 (containing an acetate starter unit) were detected at levels of about 2–5% of those produced under the same conditions by the parent triketide lactone synthase.

Site-specific mutation of the conserved arginine residue in the AT domain of the ole loading module

Starting from the gene encoding the hybrid triketide lactone synthase bearing the *ole* loading module, the active-site glutamine residue of the KSQ domain was specifically altered to an alanine residue (see *Experimental procedures* and *Supplementary material*) to obtain the expression plasmid pCB151. As expected on the basis of an analogous experiment carried out previously with the *mon* KSQ (Bisang et al., 1999), the hybrid PKS expressed from this plasmid was wholly inactive in triketide lactone biosynthesis in *S*. erythraea JC2 (data not shown). A second mutation (intended to be compensating) was also introduced by replacement of the conserved active-site arginine residue of the *ole* loading module AT domain (corresponding to residue 117 in the *E*. coli AT domain) with a tryptophan residue, as found at the corresponding position in both *ave* and *ery* loading module AT domains (Fig. 4).

The hybrid triketide lactone synthase (*Supplementary

material*) bearing the double mutation was expressed in *S*. erythraea JC2 from plasmid pCJW104 (*Supplementary material*), and the products were analysed using GC-MS. Triketide lactones 1 (propionate starter unit) and 2 (acetate starter unit) were detected in a ratio of about 8:1 and at levels about 1% of those produced under the same conditions by the unmodified triketide lactone synthase, DEBS1-TE.

Attempted conversion of the ery loading module into a KSQ-containing loading module

In a complementary experiment, we attempted to shift the substrate specificity of the AT domain of the *ery* PKS, in DEBS1-TE, from propionyl-CoA towards methylmalonyl-CoA, by first substituting the active-site tryptophan residue (corresponding to residue 117 in the *E*. coli AT domain) for an arginine residue, as found at the corresponding position in KSQ-containing loading module AT domains (Fig. 4). It was anticipated that the resulting DEBS1-TE (TKSCJW14) would be inactive, as there is no evidence that methylmalonate can be incorporated as a starter unit by this PKS. The sequence encoding DEBS loading domain AT was mutated from tryptophan 117 to arginine 117 by site-directed mutagenesis using standard polymerase chain reaction (PCR) and cloning procedures (*Supplementary material*).

To complete the attempted shift in specificity, we then constructed TKSCJW15, a derivative of TKSCJW14, in which the *tyl/KSQ* domain has additionally been fused to the N-terminus of the DEBS loading module. The domain organization in the loading domain of TKSCJW15 was *tyl/KSQ*, DEBS AT (W117R), DEBS ACP. The point of fusion between the *tyl/KSQ* domain and the DEBS AT domain was chosen to be the GQ amino acid sequence motif at the beginning of the conserved sequence of the AT domain. The expression plasmid pCJW89 containing TKSCJW15 was constructed using standard PCR and cloning procedures (*Supplementary material*). These two recombinant DEBS1-TE derivatives, TKSCJW14 and TKSCJW15, were expressed in *S*. erythraea JC2, and the polyketide products were analysed by GC-MS. In both cases, the products were found to be the triketide lactones 1 (propionate starter) and 2 (acetate starter) at levels about 1% of those seen with unmodified DEBS1-TE. The molar ratio of products 1:2 was found to be very similar, 6:1 and 9:1 respectively.

Specificity of a chimeric loading module consisting of the oleandomycin KSQ, acetate-specific rapamycin extension AT2 domain and DEBS ACP0 domain

To test the proposed functional equivalence of an extender AT with the AT of a KSQ loading domain, a hybrid

© 2002 Blackwell Science Ltd, *Molecular Microbiology*, **43**, 1215–1225

loading module was constructed consisting of a KSQ, extender AT and loading module ACP, all heterologous to each other. A module containing the KSQ from the oleandomycin PKS, AT2 from the rapamycin PKS (an extender AT specific for malonyl-CoA; Haydock *et al.*, 1995; Schwecke *et al.*, 1995; Aparicio *et al.*, 1996) and the loading module ACP0 from the erythromycin PKS was substituted for the loading module of DEBS1-TE by constructing plasmid pPFL35 (see Experimental procedures).

When the plasmid pPFL35 was used to transform wild-type S. erythraea NRRL 2338, integration into the genome in the region encoding DEBS1 occurred to give the gene arrangement shown in Fig. 5 (confirmed by Southern analysis; data not shown). Fermentation of this strain typically led to a mixture of triketide lactones 1 and 2 (not analysed in detail) produced by the DEBS1-TE enzyme that is present in this strain and also to a complex mixture of erythromycins apparently derived from the set of altered DEBS proteins whose genes are under the control of the heterologous actII-ORF4/PactI activator-promoter system (Fig. 5). One set of erythromycin derivatives observed had retention times and fragmentation patterns fully consistent with the recruitment of an acetate starter unit to yield C-13 methyl erythromycins (41 mg l⁻¹ C-13 methyl erythromycin A, 45 mg l⁻¹ C-13 methyl erythromycin B). These novel erythromycins (see also Bisang *et al.*, 1999) have not been

![Diagram](attachment:diagram.png)

Fig. 5. The mode of integration of DEBS1-TE-based plasmid pPFL35 into wild-type S. erythraea NRRL 2338. The integrant contains a copy of DEBS1-TE under the control of the ery promoter, and a copy of mutated DEBS1 (together with DEBS2 and DEBS3) under the control of the strong actII-ORF4/PactI activator-promoter.

reported from conventional erythromycin fermentations. Normal erythromycins (containing a propionate starter unit) (Fig. 1) were also detected (101-mg l⁻¹ erythromycin A, 31 mg l⁻¹ erythromycin B) at levels similar to the extract from the S. erythraea NRRL 2338 control (108 mg l⁻¹ erythromycin A, erythromycin B below the limit of detection, <3.0 mg l⁻¹).

### Discussion

The recent sequencing of numerous genes encoding modular PKSs and initial attempts to engineer the production of hybrid products have led to a re-evaluation of the normal mechanisms of chain initiation on these multienzyme systems and of the reasons for the observed substrate specificity for the starter units used. The AT-ACP type of loading module found in ery and ave PKSs has been studied more than most because it was identified first. In vitro studies with purified DEBS1-TE enzyme (Kao *et al.*, 1994; Wiesmann *et al.*, 1995) provide the clearest evidence that at least some alternative acyl-CoA esters can be recognized by the ery loading module. The apparently efficient discrimination between alternative CoA esters seen in vivo in the production of erythromycin A by S. erythraea reflects other factors, particularly an elevated level of intracellular propionyl-CoA compared with acetyl-CoA. For example, changes in the intracellular concentration of precursor acyl-CoAs (whether by changes in medium composition or by the specific addition of carboxylic acid precursor) and, in particular, an increase in the levels of PKS multienzyme can readily lead to the introduction of alternative starter units (Rowe *et al.*, 1998). Similarly, transfer of the DEBS1-TE to other host strains has been found to lead to an altered product ratio (Kao *et al.*, 1994; Brown *et al.*, 1995; Pieper *et al.*, 1995; Wiesmann *et al.*, 1995). For the ave PKS, the broad specificity is evident even in vivo, with the CoA esters of both 2-methylbutyrate and isobutyric acid serving efficiently as substrates, and the additional feeding of alternative carboxylic acids to the fermentation has led to a number of commercially useful novel avermectins (Dutton *et al.*, 1991). This broad specificity can even be transferred to other PKSs by transfer of the ave loading module (Marsden *et al.*, 1998).

In contrast, the only investigation so far of the substrate specificity in vitro of a KSQ-type loading module, that of the monensin PKS (Bisang *et al.*, 1999), has shown that, in a purified hybrid DEBS1-TE containing the mon loading module, the specificity towards malonyl-CoA rather than methylmalonyl-CoA is very high (only 1–2% of product triketide lactone had used acetate as starter). The results in this paper and also the work of Baltz and colleagues (Kuhstoss *et al.*, 1996) show that such high specificity is also a feature of other KSQ-containing loading modules,

at least *in vivo*. When the DEBS1-TE *ery* loading module was replaced by the *tyl/KSQ-AT-ACP* loading module, the product **1** was wholly propionate derived. When the *ole* KSQ-AT-ACP loading module was used, the product was almost (but not quite) exclusively the acetate starter triketide lactone **2**. Each natural KSQ-containing loading module was therefore able to impose its native specificity on the DEBS1-TE and, under the conditions used, the respective intracellular concentrations of acetyl- and propionyl-CoA in *S. erythraea* did not prevent this efficient switch. These experiments, together with those carried out by Baltz and colleagues (*Kuhstoss et al.*, 1996) to alter the starter unit introduced by the platenolid PKS, suggest that KSQ-containing loading modules are the loading modules of choice when seeking to obtain exclusively propionate or acetate starter units in an engineered hybrid modular PKS.

We carried out a direct test of the role of alternative mechanisms in providing starter units by mutagenesis of the essential 4′-phosphopantetheine binding site in the ACP domain of the loading module of DEBS1-TE. Triketide lactones **1** (bearing a propionate starter unit) and **2** (containing an acetate starter unit) were detected at levels about 2–5% of those produced under the same conditions by the parent triketide lactone synthase.

### 2–5% of normal triketide lactone levels produced by DEBS1-TE

These data support and extend previous conclusions (*Pereda et al.*, 1998) based on experiments with a truncated mutant DEBS, from which the entire loading module had been excised. It might have been objected that the evidence for direct loading seen in those experiments (*Pereda et al.*, 1998) was a direct result of a perturbation of the protein structure induced by this radical surgery. Here, only a single amino acid change is involved, so the evidence for the role of direct loading in the native DEBS is considerably strengthened. Our results place an upper limit on the contribution that direct acylation of KS1 (or any other route for supply of starter units to this enzyme) makes to chain initiation under these conditions *in vivo* because there is no competition between direct loading and the provision of starter units by the loading module. In the wild-type DEBS, it seems probable that a functional loading module should, by efficient supply of starter units to KS1, tend to suppress direct loading routes for starter unit provision. The other conclusion from our data is that direct acylation is sufficient to account for the residual quantities of triketide lactone produced (see above) by the triketide lactone synthase fronted by the mutated *ole* KSQ loading module; and it could also account (at least in part) for the production of propionate-derived polyketides in the experiments discussed below.

A wholly conserved arginine residue has been identified as an important distinguishing feature between the AT domains of KSQ-containing loading domains and the AT domains of *ery* and *ave*. Given that the alteration in the single active-site cysteine residue in the KS domain of the animal fatty acid synthase is enough to convert it into a decarboxylase (*Witkowski et al.*, 1999), it seemed possible that replacement of this single arginine residue might be sufficient to alter the substrate specificity of the AT active site from (methyl)malonyl-CoA towards propionyl-CoA/acetyl-CoA. We therefore engineered this change into DEBS1-TE bearing the *ole* loading domain, along with a second substitution, of the *ole* KSQ active site glutamine by alanine. Separately, we have shown that the latter change is sufficient to abolish polyketide production both *in vitro* and *in vivo* (data not shown). This shows that malonyl-CoA cannot itself be used as an alternative starter unit. Analysis of the fermentation products of the double mutant showed that only low levels of triketide lactone were obtained (1–2% of those produced by the parent DEBS1-TE bearing the *ole* loading module), and the product was largely the triketide lactone **1**, containing the propionate starter unit, with minor amounts of **2** produced. We interpret this result to mean that the loading module was not functional in this mutant, and that the product seen is derived by direct acylation of the DEBS1-TE KS1 domain.

Similarly, the attempt to shift the specificity of the *ery* loading module in DEBS1-TE towards methylmalonyl-CoA by site-specific replacement of the active-site tryptophan residue by arginine also failed, in that the mutant produced low levels of triketide lactones *in vivo* (1% of DEBS1-TE), which are consistent with the loading domain having been inactivated by this point mutation. Any modest contribution to chain initiation cannot be distinguished from the contribution via direct loading of KS1. When the *tyl/KSQ* domain was fused to the N-terminus of the altered AT domain to provide a pathway for decarboxylation of any ACP-bound methylmalonyl groups, the product analysis of this mutant gave essentially the same result. These results are fully consistent with the assumed role of arginine-117 in catalysis but, to clarify the exact reasons for the low activity of these mutants in polyketide synthesis, further work using purified multienzymes *in vitro* will be required.

However, taken together, the results from the experiments in this paper confirm that the greater selectivity seen with KSQ-containing domains arises naturally from the greater substrate discrimination possible, at an AT active site, between the CoA esters of dicarboxylic acids, as the ionic interaction with the conserved arginine side-chain imposes a strong additional spatial constraint on binding. Consistent with this, we have shown here that, when the AT domain of the *ole* loading module is replaced

© 2002 Blackwell Science Ltd, *Molecular Microbiology*, **43**, 1215–1225
by an authentic extender AT domain (the malonyl-CoA specific AT domain from module 2 of the rapamycin PKS) in a system (S. erythraea::pPFL35) that produces full-length erythromycins, the major products included C13-methyl erythromycins (Fig. 2), which must have arisen from the heterologous extender AT functioning appropriately in its new context as a loading AT domain, as predicted.

The significant levels of normal erythromycins also seen in the latter experiment are too high to be accounted for solely by direct loading of propionate onto the KS1. Without further work, we cannot be sure of the origin of this material. It may reflect the presence in the recombinant S. erythraea strain of a copy of unmutated DEBS1-TE, which might in some way be able to channel triketide intermediates towards erythromycin formation. However, the major significance of this experiment is that a chimeric KSQ-containing module, with KSQ and AT domains from different sources, has been shown to be fully functional. It remains to be seen whether a KSQ decarboxylase can discriminate between ACP-bound methylmalonyl- and malonyl- groups, for example by matching the ole KSQ with an AT domain specific for methylmalonyl-CoA; or the tyl KSQ with an AT domain specific for malonyl-CoA.

The sequence alignments in Fig. 4, together with the results of the site-directed mutagenesis carried out here, provide strong evidence that any loading module AT domain containing an arginine in position 117 (E. coli AT numbering) should be considered *a priori* capable of accepting a carboxylated acyl-CoA as substrate. It is therefore interesting to note that, in the recently sequenced loading modules for the polyenes pimaricin (Aparicio et al., 1999; Brautaset et al., 2000; Caffrey et al., 2001), there is such an AT domain, which might indicate that the adjacent KS-like domain, with a serine residue apparently replacing cysteine (or glutamine) in the active site, could also be functioning as a decarboxylase to provide starter units for these PKSs.

### Experimental procedures

#### Bacterial strains and plasmids

A red variant of S. erythraea NRRL 2338 was used as the host for the production of engineered erythromycins (Hessler et al., 1997). S. erythraea JC2 (Rowe et al., 1998) was used as the host for the production of triketide lactones. E. coli DH10B (Gibco BRL) was used as a host during routine cloning procedures. Routine subcloning and preparation of DNA sequencing templates used the high-copy-number plasmid pUC18 (Yanisch-Perron et al., 1985). The cosmids CosAB63 and CosA79 were from a chromosomal DNA library of the oleandomycin producer S. antibioticus (Rodríguez et al., 1993). Plasmid pkSW1 was derived from pT7-7 and DEBS1-TE, in which unique PstI and HindIII sites had been introduced to flank the KS1 of the first extension module

(Böhm et al., 1998). The integrative E. coli–Streptomyces suicide vector pCJR24 has been described previously (Rowe et al., 1998). Plasmids containing hybrid loading module and triketide lactone synthases were constructed using standard PCR and cloning procedures (Sambrook et al., 1989), as detailed in the Supplementary material. Briefly, a loading module consisting of ole KSQ, rap AT2 and ery ACP0 was constructed using splice sites that we have used previously (Oliynyk et al., 1996) The complete ole and tyl loading modules were substituted for that of DEBS1-TE using a splice site used previously for the ave loading module (Marsden et al., 1998). A summary of the plasmids used in this study is shown in Table 1, and details of their construction are given in the Supplementary material.

#### Media and growth conditions

Escherichia coli DH10B was grown using standard culture conditions for the manipulation of DNA (Sambrook et al., 1989). S. erythraea was transformed and maintained as described previously (Yamamoto et al., 1986; Gaisser et al., 1997; Rowe et al., 1998). Putative integrants were assessed for the production of novel erythromycins as described by Pacey et al. (1998). Putative integrants were assessed for the production of lactones using SSDM (Caffrey et al., 1992) or SM3 medium as production medium, as described previously (Ranganathan et al., 1999).

#### DNA sequencing

A series of sequencing templates of differing size but in defined order were generated by nested deletion of Cos63AB using the restriction endonucleases BamHI and Sau3A. End sequencing with commercially available pUC18 reverse primer to perform dideoxy terminator reactions was carried out according to the manufacturer’s instructions (PerkinElmer Biosystems). Synthetic oligonucleotides were used to complete the sequence. The SEQUENCE ASSEMBLY program (Staden, 1996) was used to assemble the data, which were then analysed using University of Wisconsin GCG software (Devereux et al., 1984).

#### Purification and characterization of polyketides

Medium from the growth of recombinant strains was extracted, and liquid chromatography (LC)-MS or GC-MS was performed on the crude extracts as described previously (Pacey et al., 1998; Ranganathan et al., 1999). Identification of polyketide products was made by comparison of retention times and fragmentation patterns with those from authentic samples.

### Acknowledgements

We thank the staff of the Wellcome Trust/BBSRC DNA Sequencing Facility, Department of Biochemistry, for their assistance. We also thank J. M. Weber for the kind gift of material. This work was supported by a project grant to P. F. Leadlay and J. Staunton, and studentships to P. F. Long

© 2002 Blackwell Science Ltd, *Molecular Microbiology*, **43**, 1215–1225
1224 P. F. Long et al.

and C. J. Wilkinson, from the Biotechnology and Biological Sciences Research Council (UK). C. P. Bisang was supported by a fellowship from the Swiss National Science Foundation.

## Supplementary material

The following material is available from http://www.blackwell-science.com/products/journals/suppmat/mole/mole2815/mmi2815sm.htm

- **Table S1.** Oligonucleotides used in this study: details of construction of the expression plasmids listed in Table 1.
- **Table S2.** Plasmids constructed during this study.

## References

Aparicio, J.F., Caffrey, P., Marsden, A.F.A., Staunton, J., and Leadlay, P.F. (1994) Limited proteolysis and active site studies of the first multienzyme component of the erythromycin-producing polyketide synthase. *J Biol Chem* 269: 8524–8528.

Aparicio, J.F., Molnár, I., Schwecke, T., König, A., Haydock, S.F., Khaw, L.E., *et al.* (1996) Organization of the biosynthetic gene cluster for rapamycin in *Streptomyces hygroscopicus*: analysis of the enzymatic domains in the modular polyketide synthase. *Gene* 169: 9–16.

Aparicio, J.F., Colina, A.J., Ceballos, E., and Martin, J.F. (1999) The biosynthetic gene cluster for the 26-membered ring polyene macrolide pimaricin — a new polyketide synthase organization encoded by two subclusters separated by functionalization genes. *J Biol Chem* 274: 10133–10139.

August, P.R., Tang, L., Yoon, Y.J., Ning, S., Muller, R., Yu, T.W., *et al.* (1998) Biosynthesis of the ansamycin antibiotic rifamycin: deductions from the molecular analysis of the *rif* biosynthetic gene cluster of *Amycolatopsis mediterranei* S699. *Chem Biol* 5: 69–79.

Bisang, C., Long, P.F., Cortes, J., Westcott, J., Crosby, J., Matharu, A.L., *et al.* (1999) A chain initiation factor common to both modular and aromatic polyketide synthases. *Nature* 401: 502–505.

Böhm, I., Holzbaur, I.E., Hanefeld, U., Cortés, J., Staunton, J., and Leadlay, P.F. (1998) Engineering of a minimal modular polyketide synthase, and targeted alteration of the stereospecificity of polyketide chain extension. *Chem Biol* 5: 407–412.

Brataaset, T., Sekurova, O.N., Sletta, H., Ellingsen, T.E., Strom, A.R., Valla, S., *et al.* (2000) Biosynthesis of the polyene antifungal antibiotic nystatin in *Streptomyces noursei* ATCC 11455: analysis of the gene cluster and deduction of the biosynthetic pathway. *Chem Biol* 7: 395–403.

Brown, M.J.B., Cortés, J., Cutter, A.L., Leadlay, P.F., and Staunton, J. (1995) A mutant generated by expression of an engineered DEBS1 protein from the erythromycin-producing polyketide synthase (PKS) in *Streptomyces coelicolor* produces the triketide as a lactone, but the major product is the nor-analogue derived from acetate as starter acid. *J Am Chem Soc* 15: 1517–1518.

Caffrey, P., Bevitt, D.J., Staunton, J., and Leadlay, P.F. (1992) Identification of DEBS 1, DEBS 2 and DEBS 3, the multienzyme polypeptides of the erythromycin-producing polyketide synthase from *Saccharopolyspora erythraea*. *FEBS Lett* 304: 225–228.

Caffrey, P., Lynch, S., Flood, E., Finnan, S., and Oliynyk, M. (2001) Amphoteracin biosynthesis in *Streptomyces nodosus*: deductions from analysis of polyketide synthase and late genes. *Chem Biol* 8: 701–711.

Cortés, J., Haydock, S.F., Roberts, G.A., Bevitt, D.J., and Leadlay, P.F. (1990) An unusually large multifunctional polypeptide in the erythromycin-producing polyketide synthase of *Saccharopolyspora erythraea*. *Nature* 348: 176–178.

Cortés, J., Wiesmann, K.E.H., Roberts, G.A., Brown, M.J.B., Staunton, J., and Leadlay, P.F. (1995) Repositioning of a domain in a modular polyketide synthase to promote specific chain cleavage. *Science* 268: 1487–1489.

Devereux, J., Haeberli, P., and Smithies, O. (1984) A comprehensive set of sequence analysis programs for the Vax. *Nucleic Acids Res* 12: 387–395.

Donadio, S., Staver, M.J., McAlpine, J.B., Swanson, S.J., and Katz, L. (1991) Modular organization of genes required for complex polyketide biosynthesis. *Science* 252: 675–679.

Dutton, C.J., Gibson, S.P., Goudie, A.C., Holdom, K.S., Pacey, M.S., Ruddock, J.C., *et al.* (1991) Novel avermectins produced by mutational biosynthesis. *J Antibiot* 44: 357–365.

Gaisser, S., Böhm, G.A., Cortés, J., and Leadlay, P.F. (1997) Analysis of seven genes from the *eryAI/eryK* region of the erythromycin biosynthetic gene cluster in *Saccharopolyspora erythraea*. *Mol Gen Genet* 256: 239–251.

Haydock, S.F., Aparicio, J.F., Molnár, I., Schwecke, T., Khaw, L.E., König, A., *et al.* (1995) Divergent sequence motifs correlated with the substrate specificity of (methyl)malonyl-CoA:acyl carrier protein transacylase domains in modular polyketide synthases. *FEBS Lett* 374: 246–248.

Hessler, P.E., Larsen, P.E., Constantinou, A.I., Schram, K.H., and Weber, J.M. (1997) Isolation of isoflavones from soy-based fermentations of the erythromycin-producing bacterium *Saccharopolyspora erythraea*. *Appl Microbiol Biotechnol* 47: 398–404.

Hopwood, D.A. (1997) Genetic contributions to understanding polyketide synthases. *Chem Rev* 97: 2465–2497.

Kakavas, S.J., Katz, L., and Stassi, D. (1997) Identification and characterization of the niddamycin polyketide synthase genes from *Streptomyces caelestis*. *J Bacteriol* 179: 7515–7522.

Kao, C.M., Katz, L., and Khosla, C. (1994) Engineered biosynthesis of a complete macrolactone in a heterologous host. *Science* 265: 509–512.

Kuhstoss, S., Huber, M., Turner, J.R., Paschal, J.W., and Rao, R.N. (1996) Production of a novel polyketide through the construction of a hybrid polyketide synthase. *Gene* 183: 231–236.

Lau, J., Cane, D.E., and Khosla, C. (2000) Substrate specificity of the loading didomain of the erythromycin polyketide synthase. *Biochemistry* 39: 10514–10520.

Lowden, P.A.S., Bohm, G.A., Staunton, J., and Leadlay, P.F. (1996) The nature of the starter unit for the rapamycin polyketide synthase. *Ang Chemie* 35: 2249–2251.

MacNeil, D.J., Occi, J.L., Gewain, K.M., and MacNeil, T. (1994) Correlation of the avermectin polyketide synthase genes to the avermectin structure. Implications for designing novel avermectins. Ann NY Acad Sci 721: 123–132.

Marsden, A.F.A., Wilkinson, B., Cortés, J., Dunster, N.J., Staunton, J., and Leadlay, P.F. (1998) Engineering broader specificity into an antibiotic-producing polyketide synthase. Science 279: 199–202.

Oliynyk, M., Brown, M.J.B., Cortés, J., Staunton, J., and Leadlay, P.F. (1996) A hybrid modular polyketide synthase obtained by domain swapping. Chem Biol 3: 833–839.

Pacey, M.S., Dirlam, J.P., Geldart, R.W., Leadlay, P.F., McArthur, H.A.I., McCormick, E.L., et al. (1998) Novel erythromycins from a recombinant Saccharopolyspora erythraea strain NRRL 2338 pIG1. I. Fermentation, isolation and biological activity. J Antibiot 51: 1029–1034.

Parsons, I.C., Everett, J.R., Pacey, M.S., Ruddock, J.C., Swanson, A.G., and Thompson, C.M. (1999) Structural elucidation of a novel erythromycin, 13-cyclopentyl 13-desethyl-erythromycin B, from a recombinant Saccharopolyspora erythraea strain, NRRL 2338 pIG/1. J Antibiot 52: 190–192.

Pereda, A., Summers, R.G., Stassi, D.L., Ruan, X.A., and Katz, L. (1998) The loading domain of the erythromycin polyketide synthase is not essential for erythromycin biosynthesis in Saccharopolyspora erythraea. Microbiol- ogy 144: 543–553.

Pieper, R., Luo, G.L., Cane, D.E., and Khosla, C. (1995) Remarkably broad substrate specificity of a modular polyketide synthase in a cell-free system. J Am Chem Soc 117: 11373–11374.

Pieper, R., Gokhale, R.S., Luo, G.L., Cane, D.E., and Khosla, C. (1997) Purification and characterization of bimodular and trimodular derivatives of the erythromycin polyketide synthase. Biochemistry 36: 1846–1851.

Quirós, L.M., Aguirrezabalaga, I., Olano, C., Méndez, C., and Salas, J.A. (1998) Two glycosyltransferases and a glycosidase are involved in oleandomycin modification during its biosynthesis by Streptomyces antibioticus. Mol Microbiol 28: 1177–1185.

Rangan, V.S., and Smith, S. (1997) Alteration of the substrate specificity of the malonyl-CoA/acetyl-CoA: acyl carrier protein S-acyltransferase domain of the multifunctional fatty acid synthase by mutation of a single arginine residue. J Biol Chem 272: 11975–11978.

Ranganathan, A., Timoney, M., Bycroft, M., Cortés, J., Thomas, I.P., Wilkinson, B., et al. (1999) Knowledge-based design of bimodular and trimodular polyketide synthases based on domain and module swaps: a route to simple statin analogues. Chem Biol 6: 731–741.

Rodríguez, A.M., Olano, C., Vilches, C., Méndez, C., and Salas, J.A. (1993) Streptomyces antibioticus contains at least three oleandomycin resistance determinants, one of which shows similarity with proteins of the ABC transporter superfamily. Mol Microbiol 8: 571–582.

Rowe, C.J., Cortés, J., Gaisser, S., Staunton, J., and Leadlay, P.F. (1998) Construction of new vectors for high-level expression in Actinomycetes. Gene 216: 215–223.

Sambrook, J., Fritsch, E.F., and Maniatis, T. (1989) Molecular Cloning: a Laboratory Manual, 2nd edn. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.

Schupp, T., Toupet, C., Cluzel, B., Neff, S., Hill, S., Beck, J.J., et al. (1995) A Sorangium cellulosum (myxobacterium) gene cluster for the biosynthesis of the macrolide antibiotic soraphen A: cloning, characterization, and homology to polyketide synthase genes from actinomycetes. J Bacteriol 177: 3673–3679.

Schupp, T., Toupet, C., Engel, N., and Goff, S. (1998) Cloning and sequence analysis of the putative rifamycin polyketide synthase gene cluster from Amycolatopsis mediterranei. FEMS Microbiol Lett 159: 201–207.

Schwecke, T., Aparicio, J.F., Molnár, I., König, A., Khaw, L.E., Haydock, S.F., et al. (1995) The biosynthetic gene cluster for the polyketide immunosuppressant rapamycin. Proc Natl Acad Sci USA 92: 7839–7843.

Serre, L., Verbee, E.C., Dauter, Z., Stuitje, A.R., and Derewenda, Z.S. (1995) The Escherichia coli malonyl-CoA:acyl carrier protein transacylase at 1.5-angstrom resolution – crystal structure of a fatty-acid synthase component. J Biol Chem 270: 12961–12964.

Shah, S., Xue, Q., Tang, L., Carney, J.R., Betlach, M., and McDaniel, R. (2000) Cloning, characterization and heterologous expression of a polyketide synthase and P-450 oxidase involved in the biosynthesis of the antibiotic oleandomycin. J Antibiot 53: 502–508.

Staden, R. (1996) The Staden sequence analysis package. Mol Biotechnol 5: 233–241.

Strohl, W.R. (1992) Compilation and analysis of DNA sequences associated with apparent Streptomycete promoters. Nucleic Acids Res 20: 961–974.

Waldron, C., Matsushima, P., Rosteck, P.R.J., Broughton, M.C., Turner, J., Madduri, K., et al. (2001) Cloning and analysis of the spinosad biosynthetic gene cluster of Saccharopolyspora erythraea. Chem Biol 94: 1–13.

Weissman, K.J., Bycroft, M., Staunton, J., and Leadlay, P.F. (1998) Origin of starter units for erythromycin biosynthesis. Biochemistry 37: 11012–11017.

Wiesmann, K.E.H., Cortés, J., Brown, M.J.B., Cutter, A.I., Staunton, J., and Leadlay, P.F. (1995) Polyketide synthesis in vitro on a modular polyketide synthase. Chem Biol 2: 583–589.

Witkowski, A., Joshi, A.K., Lindqvist, Y., and Smith, S. (1999) Conversion of a β-ketoacyl synthase to a malonyl decarboxylase by replacement of the active-site cysteine with glutamine. Biochemistry 38: 11643–11650.

Xue, Y.Q., Zhao, L.S., Liu, H.W., and Sherman, D.H. (1998) A gene cluster for macrolide antibiotic biosynthesis in Streptomyces venezuelae: architecture of metabolic diversity. Proc Natl Acad Sci USA 95: 12111–12116.

Yamamoto, H., Maurer, K.H., and Hutchinson, C.R. (1986) Transformation of Streptomyces erythraeus. J Antibiot 39: 1304–1313.

Yanisch-Perron, C., Vieira, J., and Messing, J. (1985) Improved M13 phage cloning vectors and host strains – nucleotide sequences of the M13MP18 and pUC19 vectors. Gene 33: 103–109.
