
METABOLISM AND BIOENERGETICS:
Assembly of Photosystem I: I.
INACTIVATION OF THE rubA GENE
ENCODING A
MEMBRANE-ASSOCIATED
RUBREDOXIN IN THE
CYANOBACTERIUMSYNECHOCOCCUS
SP. PCC 7002 CAUSES A LOSS OF
PHOTOSYSTEM I ACTIVITY

Gaozhong Shen, Jindong Zhao, Susan K.
Reimer, Mikhail L. Antonkine, Qun Cai,
Sharon M. Weiland, John H. Golbeck and
Donald A. Bryant
J. Biol. Chem. 2002, 277:20343-20354.

Access the most updated version of this article at [http://www.jbc.org/content/277/23/20343](http://www.jbc.org/content/277/23/20343)

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/affinity-sites).

Alerts:
- When this article is cited
- When a correction for this article is posted

Click here to choose from all of JBC's e-mail alerts

This article cites 55 references, 19 of which can be accessed free at
[http://www.jbc.org/content/277/23/20343.full.html#ref-list-1](http://www.jbc.org/content/277/23/20343.full.html#ref-list-1)

THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2002 by The American Society for Biochemistry and Molecular Biology, Inc.

**Assembly of Photosystem I**

I. INACTIVATION OF THE *rubA* GENE ENCODING A MEMBRANE-ASSOCIATED RUBREDOXIN IN THE CYANOBACTERIUM SYNECHOCOCCUS SP. PCC 7002 CAUSES A LOSS OF PHOTOSYSTEM I ACTIVITY*

Received for publication, February 2, 2002, and in revised form, March 13, 2002  
Published, JBC Papers in Press, March 25, 2002, DOI 10.1074/jbc.M201103200

Gaozhong Shen‡, Jindong Zhao§, Susan K. Reimer¶, Mikhail L. Antonkine‡, Qun Cai§, Sharon M. Weiland‡, John H. Golbeck‡, and Donald A. Bryant‡**

From the ‡Department of Biochemistry and Molecular Biology, Pennsylvania State University, University Park, Pennsylvania 16802, the §Department of Plant Molecular Biology, College of Life Sciences, Peking University, Beijing, China, and the ¶Department of Biology, St. Francis University, Loretto, Pennsylvania 15940

A 4.4-kb HindIII fragment, encoding an unusual rubredoxin (denoted RubA), a homolog of the Synechocystis sp. PCC 6803 gene *slr2034* and *Arabidopsis thaliana* HCF136, and the *psbEFLJ* operon, was cloned from the cyanobacterium *Synechococcus* sp. PCC 7002. Inactivation of the *slr2034* homolog produced a mutant with no detectable phenotype and wild-type photosystem (PS) II levels. Inactivation of the *rubA* gene of *Synechococcus* sp. PCC 7002 produced a mutant unable to grow photoautotrophically. RubA and PS I electron transport activity were completely absent in the mutant, although PS II activity was ~80% of the wild-type level. RubA contains a domain of ~50 amino acids with very high similarity to the rubredoxins of anaerobic bacteria and archaea, but it also contains a region of about 50 amino acids that is predicted to form a flexible hinge and a transmembrane α-helix at its C terminus. Overproduction of the water-soluble rubredoxin domain in *Escherichia coli* led to a product with the absorption and EPR spectra of typical rubredoxins. RubA was present in thylakoid but not plasma membranes of cyanobacteria and in chloroplast thylakoids isolated from spinach and *Chlamydomonas reinhardtii*. Fractionation studies suggest that RubA might transiently associate with PS I monomers, but no evidence for an association with PS I trimers or PS II was observed. PS I levels were significantly lower than in the wild type (~40%), but trimeric PS I complexes could be isolated from the *rubA* mutant. These PS I complexes completely lacked the stromal subunits PsaC, PsaD, and PsaE but contained all membrane-intrinsic subunits. The three missing proteins could be detected immunologically in whole cells, but their levels were greatly reduced, and degradation products were also detected. Our results indicate that RubA plays a specific role in the biogenesis of PS I.

In oxygen-evolving photosynthetic organisms, two photosystems (PS)¹ cooperate in the utilization of light energy to produce the reducing power and chemical energy required for carbon dioxide fixation. PS II carries out the light-dependent oxidation of water and produces a weak reductant, plastoquinol. PS I catalyzes the light-dependent oxidation of plastocyanin (or cytochrome *c*₆) and produces a strong reductant, reduced ferredoxin (or flavodoxin). The composition and general properties of PS I and II are rather well understood (for reviews, see Refs. 1–3), although important functional details continue to be missing. Using PS I and II complexes purified from the thermophilic cyanobacterium *Synechococcus elongatus*, structural details for both photosystems have recently been obtained by x-ray crystallography (4, 5).

The PS I reaction center is a large, multisubunit complex that forms trimers in cyanobacterial membranes. The structure of cyanobacterial PS I at 2.5-Å resolution shows that each monomer comprises 12 polypeptides (denoted PsaA–PsaF, PsaI–PsaM, and PsaX), 96 chlorophyll (Chl) *a* molecules, 22 β-carotene molecules, three [4Fe-4S] clusters, two phylloquinones, and five tightly bound lipids (5). Six redox centers are involved in light-induced electron transfer in PS I (Fig. 1). The PsaA/PsaB heterodimer harbors the primary electron donor P700 (a Chl *a* dimer), A₀ (a monomeric Chl *a*), A₁ (a phylloquinone), and Fₓ (an interpeptide [4Fe-4S] cluster). Two terminal electron acceptors, Fₐ and Fₔ, are [4Fe-4S] clusters that are located on the extrinsic PsaC protein (6). Although significant progress has been achieved in elucidating of the structure and function of the PS I, many questions still remain concerning the biogenesis and regulation of PS I in the thylakoid membrane.

The biogenesis of photosynthetic complexes in cyanobacteria and higher plant chloroplasts is a complex multistep process that includes apoprotein translation, protein folding and insertion into the thylakoid membrane, cofactor binding, iron-sulfur cluster assembly, and ordered association of the individual subunits to form multisubunit complexes. It has been demonstrated that all of these steps are highly regulated at the post-translational level, especially for the cofactor- and pigment-binding polypeptides. In recent years, several factors involved in either the assembly or the stability of PS I have been identified (e.g. Refs. 7–9). Nevertheless, the detailed molecular mechanisms of assembly of the photosynthetic apparatus in the thylakoid membranes remain poorly understood. The identification of genes whose products function in the assembly of the

* This work was supported by National Science Foundation Grants MCB-9723469 and MCB-0077586 (to D. A. B.) and United States Department of Agriculture Award 2001-35318-10125 (to J. H. G.). This work was also supported by China State Key Project G19980101 (to J. Z.). The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
** Supported by National Science Foundation ROA supplement award MCB-9723469.
*** To whom correspondence should be addressed: Dept. of Biochemistry and Molecular Biology, S-234 Frear Bldg., Pennsylvania State University, University Park, PA 16802. Tel.: 814-865-1992; Fax: 814-863-7024; E-mail: dab14@psu.edu.

---

1 The abbreviations used are: PS, photosystem; Chl, chlorophyll; DM, *n*-dodecyl-β-D-maltopyranoside; EPR, electron paramagnetic resonance; GST, glutathione S-transferase; MV, methyl viologen; ORF, open reading frame; μE, microeinsteins.

biotics as appropriate: kanamycin (100 μg ml⁻¹), erythromycin (25 μg ml⁻¹), streptomycin (100 μg ml⁻¹), spectinomycin (50 μg ml⁻¹), and ampicillin (10 μg ml⁻¹). For strains lacking PS I activity, the medium was supplemented with 10 mM glycerol as carbon and energy source, and cells were grown at low light intensity (~5–10 μE m⁻² s⁻¹). Growth was monitored turbidometrically at 730 nm with a Cary-14 spectrophotometer modified for computerized data acquisition by On-Line Instruments, Inc. (Bogart, GA).

Mutant Construction and Complementation—Two open reading frames (ORFs) with strong sequence similarity to ORFs *slr2033* and *slr2034* of *Synechocystis* sp. PCC 6803 are found upstream from the *psbEFLJ* operon of *Synechococcus* sp. PCC 7002 (see Fig. 2A). Each gene was inactivated by insertion of a 1.3-kb DNA fragment that encodes the *aphII* gene and confers resistance to kanamycin. ORF *slr2033* was inactivated by inserting the fragment at the unique *BglII* site within the coding sequence, whereas ORF *slr2034* was inactivated by inserting the resistance cassette into a *SmaI* site (see Fig. 2). Transformation of these constructions into strains of *Synechococcus* sp. PCC 7002 was performed as described (15, 16). After selection of kanamycin-resistant transformants and restreaking to allow segregation of alleles, segregation was verified by PCR using appropriate oligonucleotide primers and by Southern blot hybridization analyses.

To verify the phenotype of the *rubA::aphII* mutant strain, complementation studies were performed with the full-length *rubA* gene of *Synechococcus* sp. PCC 7002 and *Synechocystis* sp. PCC 6803. The complementing genes were introduced into the mutant strain in trans on plasmids pAQE/7002FL and pAQE/6803FL, respectively. To construct plasmid pAQE/7002FL, a 360-bp DNA fragment encoding the full-length *rubA* gene of *Synechococcus* sp. PCC 7002 was amplified by PCR and cloned into plasmid pET3d. This *rubA* gene was then excised by digestion with *BglII* and *EcoRI* and ligated into plasmid pAQE19 (17), which had also been digested with *BglII* and *EcoRI*. Similarly, the full-length *rubA* gene of *Synechocystis* sp. PCC 6803 was amplified by PCR and first cloned into plasmid pSE280 (18). A *BglII-EcoRI* fragment containing the *P*~trc~ promoter and the *rubA* gene was recloned into plasmid pAQE19 to produce plasmid pAQE/6803FL. The two complementation plasmids were introduced into the *rubA::aphII* mutant strain by transformation, and ampicillin- and kanamycin-resistant colonies were selected and characterized.

To determine whether the hydrophobic C-terminal extension of RubA was essential to its function, a 204-bp fragment encoding the N-terminal 68 amino acid residues of the *rubA* gene and a subsequent stop codon were generated by PCR amplification and cloned into plasmid pSE280 (18). An *EcoRI* fragment encoding the *P*~trc~ promoter and the truncated *rubA* gene was then recloned into an *EcoRI* site immediately upstream from a spectinomycin-resistance cassette in plasmid pLAT4. Plasmid pLAT4 was generated by cloning a 2.3-kb DNA *EcoRV* fragment encoding the dispensable *desE* gene of *Synechococcus* sp. PCC 7002 (19) into plasmid pBluescript KS(−). A 400-bp *HindIII-EcoRI* fragment was deleted and replaced with a 2-kb spectinomycin-resistance cartridge. The construction pLAT4/6803TR was used to transform the *rubA::aphII* mutant. Selection of kanamycin- and spectinomycin-resistant mutants produced a strain in which the truncated *rubA* gene and *P*~trc~ promoter were inserted into the *Synechococcus* sp. PCC 7002 genome at the *desE* locus.

To test whether the *Clostridium pasteurianum* rubredoxin could complement the *rubA::aphII* mutant strain, codons 1–48 of the *C. pasteurianum* rubredoxin gene (20) were amplified by PCR and fused to residues Gln⁶² to Asp¹¹⁵ of the *Synechococcus* sp. PCC 7002 RubA. Due to the introduction of an *EcoRI* site, residue Gln⁴⁹ of *C. pasteurianum* rubredoxin (Gln⁶² of RubA) was replaced by glutamate. To assemble this construct, an *EcoRI-BamHI* fragment encoding the C terminus of the RubA protein was cloned in pBluescript. An *NdeI-EcoRI* fragment encoding residues 1–48 of *C. pasteurianum* rubredoxin was cloned into pET3d, and subsequently the *EcoRI-BamHI* fragment encoding the C terminus of RubA was inserted into this plasmid as well. Finally, a *BglII-BamHI* fragment encoding the complete chimeric gene was cloned into pAQE19 that had been digested with *BglII* and *BamHI*. This plasmid was then transformed into the *rubA::aphII* mutant strain of *Synechococcus* sp. PCC 7002, and transformants resistant to ampicillin and kanamycin were selected and characterized.

Expression and Purification of the RubA Protein—The full-length *rubA* gene from *Synechococcus* sp. PCC 7002 was amplified by PCR using the following primers: 5′-ATACCCTCTGAAACATGTCCGATCGCCCCA-3′ and 5′-TTTGCTATGGGGATCCTTAGTCCAGACCAT-3′. The 380-bp amplified fragment was digested with *AflIII* and *BamHI* (sites are underlined in the primer sequences) and ligated into pET3d (Novagen, Madison, WI), which had been digested with *NcoI* and

FIG. 1. Structural model of PS I. Electron transport cofactors and the relative positions of the PsaA, PsaB, PsaC, PsaD, and PsaE subunits are shown. The primary electron donor, P700, is composed of Chl *a* molecule eC-B1 and Chl *a′* molecule eC-A1. The Chl *a* molecule spectroscopically identified as the primary acceptor *A₀* is probably eC-A3, and the phylloquinone molecule spectroscopically identified as the secondary acceptor *A₁* is probably *Q*~K~-A. The [4Fe-4S] cluster *F*~X~ is bound by two cysteines from PsaA and two cysteines from PsaB. The *F*~A~ and *F*~B~ [4Fe-4S] clusters are bound to the PsaC subunit. For additional details, see Ref. 5.

photosystem complexes should greatly increase our understanding of biogenesis and regulation photosynthesis specifically and protein complexes generally.

The search for proteins that function in the biogenesis of photosynthetic complexes has been greatly facilitated by sequencing of chloroplast genomes, cyanobacterial genomes, the genome of the model plant *Arabidopsis thaliana*, and the genomes of diverse bacteria. These data, in combination with screens for mutants defective in photosystem assembly in cyanobacteria (10, 11) and *Chlamydomonas reinhardtii* (12), have identified genes whose products function in photosystem biogenesis. The relatively simple, small genomes of cyanobacteria and the facile ability to manipulate these organisms genetically have proven to be significant advantages in the search for such assembly or stabilizing proteins.

In this study, a membrane-associated rubredoxin that is found in cyanobacteria and diverse eukaryotes is described. In contrast to the results of Wastl *et al.* (13), who concluded that a chloroplast-localized rubredoxin is associated with PS II, it is conclusively demonstrated here that this rubredoxin, denoted RubA, is absolutely required for assembly of functional PS I complexes in cyanobacteria. Mutants lacking RubA produce trimeric PS I complexes, which are inactive in electron transport to flavodoxin or the artificial acceptor methyl viologen. PS I complexes isolated from the *rubA* mutant contain all membrane-intrinsic subunits but are completely devoid of the stromal PsaC, PsaD, and PsaE subunits. A preliminary report of some of these results was presented at the XIth International Congress on Photosynthesis in Budapest, Hungary (14).

MATERIALS AND METHODS

Growth of Cultures—*Synechococcus* sp. PCC strain 7002 wild-type and mutant strains were grown in medium A supplemented with 1 mg⁻¹ NaNO₃ (15). Liquid cultures were grown at 38 °C under constant cool white fluorescent illumination; cultures were bubbled with air supplemented with 1.0% (v/v) CO₂. Agar plates were prepared by solidifying the same growth medium with 1.5% (w/v) Difco Bacto-agar supplemented with 0.3% (w/v) sodium thiosulfate and the following anti-

Assembly of Photosystem I

BamHI. The resulting expression plasmid was designated pET3d/ 7002FL. A similar construction strategy was employed for the expression of a truncated, water-soluble RubA fragment (residues 1–68; see Fig. 3), except that a stop codon was introduced at codon 69 of the gene. These plasmids were introduced into *E. coli* strain BL21(DE3); protein expression followed standard protocols (21).

For the production of antibodies to RubA, a truncated RubA protein that comprised residues 1–94 and lacked the 21-residue C-terminal hydrophobic region, was fused to the C terminus of glutathione S-transferase (GST). The primers used for amplification were 5′-AGGATCCG- ATGAGCGATCGCCCCATAG-3′ and 5′-CACC GGATCCTTAGTTTTT- TTGGTTGGAG-3′. The amplified fragment was digested with BamHI (sites are underlined in the primers), and the resulting fragment was ligated into the BamHI site of plasmid pGXT-4 (Amersham Biosciences). The resulting plasmid was denoted pGST-RubA and was transformed into *E. coli* strain BL21. An overnight culture of this strain was induced with isopropyl-β-D-thiogalactoside at 28°C for 6 h. Cells were harvested by centrifugation, washed with buffer, and disrupted by two passages through a French pressure cell at 124 megapascals at 4°C. Purification of the recombinant RubA protein followed the recommendations of the manufacturer. Cell debris was removed by centrifugation, and the supernatant was loaded onto a GST affinity column (Amersham Biosciences) at a flow rate of 15 ml h⁻¹. The eluted GST-RubA protein was digested with thrombin overnight at 37°C, and the RubA protein was separated from GST by gel exclusion chromatography on a Sephadex G-50 column. Fractions containing RubA were pooled and concentrated by lyophilization. The protein was sequenced with an automatic sequencer (ABI 491; Perkin-Elmer Life Sciences) to verify that the expected product had been obtained. Polyclonal antibodies to the purified RubA protein were prepared in rabbits as recommended by Harlow and Lane (22).

DNA Sequence Analysis—DNA sequence analyses were performed at the Nucleic Acid Facility of The Pennsylvania State University with a ABI PRISM model 377 DNA sequencer (Perkin-Elmer Life Sciences) using 3′ BigDye™-labeled dideoxynucleotide triphosphates. All plasmid constructions, site-directed mutants, and synthetic chimeric genes were verified completely by DNA sequence analysis. DNA sequencing of the 4.4-kb HindIII fragment encoding *rubA*, a homolog of *Synechocystis* sp. PCC 6803 ORF *slr2034*, and the *psbEFLJ* operon of *Synechococcus* sp. PCC 7002 was performed by primer walking. Synthetic oligonucleotides were synthesized after initial DNA sequencing with the M13 universal and reverse sequencing primers by visual inspection of the sequence data obtained. The fragment was completely sequenced on both strands, and the data were deposited in GenBank™ under the accession number AY075046. DNA and protein sequences were manipulated using MacVector version 7.0 software (Oxford Molecular Co., Ltd.).

Purification of PS I Complexes—Thylakoid membranes were prepared from cells in late exponential growth phase as previously described (16). Cells were harvested by centrifugation and washed once in 50 mM Tris-HCl, pH 8.0, buffer. Cells were broken by two passages through a French pressure cell at 124 megapascals at 4°C. Unbroken cells and debris were removed by low speed centrifugation at 3000 × g. Thylakoid membranes were pelleted by centrifugation at 50,000 × g for 45 min. The thylakoid membranes were resuspended in 50 mM Tris-HCl, pH 8.0, buffer.

Thylakoid membranes were diluted to a concentration of 0.4 mg of Chl ml⁻¹ in 50 mM Tris-HCl, pH 8.0, buffer and solubilized for 2 h at 4°C by the addition of *n*-dodecyl-β-D-maltopyranoside (DM) to a concentration of 1% (w/v). The solution was clarified by centrifugation for 20 min at 14,000 × g, and the supernatant was loaded onto a 5–20% (w/v) linear sucrose gradient prepared in 50 mM Tris-HCl, pH 8.0, buffer containing 0.03% (w/v) DM. The gradients were centrifuged for 16–18 h at 140,000 × g, and the lower, dark green band containing PS I trimers and the middle green band enriched in PS I monomers were collected. After concentration of each fraction using Centricon-50 concentrators (50,000 molecular weight cut-off membranes; Amicon, Beverly, MA), the PS I complexes were twice dialyzed against 50 mM Tris-HCl, pH 8.0, buffer containing 0.03% (w/v) DM for 2 h using a 50,000 molecular weight cut-off dialysis membrane. To purify the PS I trimers and monomers further, the trimeric PS I fraction and the monomer-enriched PS I fraction were again loaded onto 5–20% (w/v) linear sucrose gradients prepared with 50 mM Tris-HCl, pH 8.0, buffer with no added detergent. The gradients were centrifuged for 16–18 h at 140,000 × g, and PS I complexes were collected and resuspended in 50 mM Tris-HCl, pH 8.0, buffer containing 0.03% (w/v) DM and 15% (v/v) glycerol. PS I trimers are typically quite pure after the second sucrose gradient centrifugation step; however, the PS I monomer fraction typically still contains some PS II complexes and other contaminating proteins.

To purify the PS I monomers further, the monomeric PS I fraction from the second sucrose gradient was subjected to ion-exchange chromatography. A DEAE-Sepharose column (1.6 × 10 cm) was equilibrated with 50 mM Tris-HCl, pH 8.0, buffer containing 0.05% DM prior to loading the PS I monomer fraction. Proteins not bound by the column were eluted with several column volumes of equilibration buffer, and the column was then developed with a 0–400 mM NaCl linear gradient. Fractions were collected and concentrated using Centricon-50 centrifugal concentrators (50-kDa cut-off membrane, Amicon, Beverly, MA).

SDS-PAGE Analysis and Immunoblotting—Methods for SDS-PAGE and immunoblotting were identical to those described (16). To resolve the subunit compositions of PS I preparations from the wild type and mutants, a 16% (w/v) polyacrylamide separation gel containing 6 M urea was used with the Tricine/Tris discontinuous buffer system previously described (23). PS I samples containing 5 μg of Chl were solubilized in loading buffer and denatured by heating at 55°C for 5 min prior to loading. The resolved proteins were visualized by silver staining (24).

For immunoblots, proteins separated by SDS-PAGE were electrophoretically transferred onto nitrocellulose membranes (Schleicher & Schuell) using a semidry blotting system (Bio-Rad). The nitrocellulose membranes were subjected to immunoreaction with polyclonal rabbit antibodies as previously described (16, 22). Whole-cell extracts were prepared by harvesting cells by centrifugation, resuspending them in 50 mM Tris-HCl, pH 8.0, buffer, and disrupting them by sonication.

Chlorophyll Analysis and Oxygen Evolution Assay—Cellular Chl content and whole-chain electron transport activities based on oxygen evolution were calculated on the basis of equal cell numbers determined turbidometrically by the optical density at 730 nm (*OD*~730~). These measurements were made with cells harvested by centrifugation from the late exponential growth phase and resuspended in 25 mM HEPES-NaOH, pH 7.0 buffer. Chl was extracted from cells with 100% methanol, and its concentration was determined as described (25) using a Cary-14 spectrophotometer modified for computerized data acquisition by On-Line Instruments, Inc. (Bogart, GA).

Whole-chain oxygen evolution measurements on *Synechococcus* sp. PCC 7002 wild-type and mutant cells were performed using a Clark-type electrode as described (16). The actinic light from a tungsten-halogen lamp was filtered through 3 cm of water and a Corion 500-nm cut-on filter. The saturating light intensity used was about 2300 μE m⁻²s⁻¹. The temperature of the measuring chamber was maintained at 30°C by a circulating water bath. Cell samples were adjusted to a final concentration of 5 μg of Chl ml⁻¹ or 1.0 *OD*~730~ ml⁻¹ in 25 mM HEPES-NaOH, pH 7.0, buffer. For whole-chain electron transport measurements (H₂O to CO₂), 5 mM NaHCO₃ was added to the cell suspension. For electron transport measurements from H₂O to Qₐ, 0.5 mM K₃Fe(CN)₆ and 0.1 mM 2,5-dimethyl-*p*-benzoquinone were added as artificial electron acceptors.

Fluorescence Emission Spectra at 77 K—Fluorescence emission spectra at 77 K were measured using a SLM 8000C spectrofluorometer as described (16). Cells in exponential growth phase (*OD*~730~ = 0.6–0.7) were collected and resuspended in 25 mM HEPES/NaOH, pH 7.0, buffer. Glycerol was added to a final concentration 60% (v/v). Cells were frozen at a concentration of 1.0 *OD*~730~/ml. The excitation wavelength was 440 nm for Chl excitation. A long pass filter (transmitting at >600 nm) was used at the inlet of the emission monochromator to minimize contributions from scattered light.

PS I-mediated Electron Transport Rates—PS I-specific electron transport rates were measured for thylakoid membranes isolated from *Synechococcus* sp. PCC 7002 strains by measuring the rate of oxygen uptake via the Mehler reaction. Photosynthetic electron transport from 2,6-dichlorophenolindophenol to methyl viologen (MV) catalyzed by PS I was determined by measuring oxygen uptake under red light illumination (a Corion 600-nm cut-on filter was used for excitation light). Thylakoid membranes were resuspended in the 50 mM HEPES-NaOH, pH 8.0, buffer, containing 5 mM MgCl₂, 10 mM CaCl₂, 0.5% (v/v) dimethyl sulfoxide, and 15% (v/v) glycerol) for storage. PS I complexes were resuspended in 50 mM Tris-HCl, pH 8.0, buffer at a Chl concentration of 5 μg of Chl ml⁻¹. 2,6-Dichlorophenolindophenol (0.1 mM) and sodium ascorbate (1 mM) were added as electron donors, and MV (0.1 mM) was added as the electron acceptor for the Mehler reaction. The reaction mixture also contained 20 μM 3-(3,4-dichlorophenyl)-1,1-dimethylenurea and 5 mM NaN₃.

The electron transport activity of isolated PS I complexes was measured by determining the rate of flavodoxin reduction as described previously (26–28). PS I complexes were suspended in 50 mM Tris-HCl, pH 8.0, buffer at a Chl concentration of 5 μg ml⁻¹. The reaction mixture

20346

Assembly of Photosystem I

FIG. 2. Physical map of DNA fragments for cloning and mutagenesis of the rubA gene. A, restriction map of a 4.4-kb HindIII DNA fragments carrying the rubA-slr2034-(ORF341)-psbEFLJ genes from Synechococcus sp. PCC 7002. B, restriction map showing the interruption of the rubA gene through insertion of the aphII gene at the BglII site in the rubA gene. The small arrows flanking the rubA gene indicate the positions of primers for PCR amplification of the gene. C, fractionation of DNA fragments produced by PCR from chromosomal DNA of the Synechococcus sp. PCC 7002 WTREm and the rubA::aphII strains. The larger fragment size (about 1.6 kb) in the rubA mutant corresponds to the rubA gene with the inserted 1.3-kb aphII gene cassette (see map in Fig. 2B). D, restriction map showing the inactivation of the slr2034 homolog by insertion of a 1.3-kb kanamycin resistance cartridge at a unique SmaI site.

contained purified, recombinant flavodoxin from Synechococcus sp. PCC 7002 (15 μM) (28), cytochrome $c_{6}$ from Spirulina maxima (15 μM) (29), and sodium ascorbate (6 mM). The rate of flavodoxin reduction was monitored by the absorption change at 467 nm using a Cary model 219 spectrophotometer.

Electron Paramagnetic Resonance Spectroscopy—EPR studies were performed using a Bruker ECS-106 X-band spectrometer equipped with a standard mode resonator (ER/4102 ST). Cryogenic temperatures were maintained with a liquid helium cryostat and an ITC-4 temperature controller (Oxford Instruments). Sample temperatures were monitored by a calibrated thermocouple located 3 mm beneath the bottom of the quartz sample tube and referenced to liquid N₂. The microwave frequency was measured with a Hewlett-Packard 5340A frequency counter, and the magnetic field was calibrated using α,α′-diphenol-β-picryl hydrazyl as the standard.

RESULTS

Cloning the Gene Encoding the Rubredoxin of Synechococcus sp. PCC 7002—As shown in Fig. 2A, a 4.4-kb HindIII DNA fragment, encoding the psbEFLJ operon, was cloned and sequenced from the Synechococcus sp. PCC 7002 genome (GenBank™ accession number AY075046). Two open reading frames, encoding predicted proteins of 115 and 341 amino acids and exhibiting strong sequence similarity to ORFs slr2033 and slr2034 of Synechocystis sp. PCC 6803, were identified upstream from the psbEFLJ operon, which encodes the α and β subunits of cytochrome $b_{559}$ as well as two additional subunits of PS II. A. thaliana mutants lacking the nucleus-encoded, chloroplast-targeted homolog of slr2034 (HCF136) lack PS II activity (30). Insertional inactivation of slr2034 in Synechococcus sp. PCC 7002 produced a strain with normal PS II activity and no detectable phenotype under standard growth conditions (data not shown).

As shown in Fig. 3, the deduced amino acid sequence of the slr2033 homolog contains a 50-amino acid domain with very high sequence similarity to rubredoxins from numerous organisms. Rubredoxins are small non-heme iron proteins, which are widely distributed among eubacteria and archaea, especially anaerobes (31). The product of the Synechococcus sp. PCC 7002 slr2033 homolog has a calculated molecular mass of 12.5 kDa and exhibits highest sequence similarity to the rubredoxin-like proteins encoded by ORF slr2033 of Synechocystis sp. PCC 6803 (32) and alr3843 of Nostoc sp. PCC 7120 (33). Compared

with bacterial rubredoxins, the cyanobacterial rubredoxins have a charged, hydrophilic N-terminal extension of ~14 amino acid residues as well as a longer C-terminal extension of ~50 residues. This extension is predicted to form a hydrophobic, transmembrane α-helix at its end and is connected to the rubredoxin domain by a region predicted to form a flexible hinge (see Fig. 3). Based upon this sequence similarity to bacterial rubredoxins and upon the results from the characterization of the protein as presented below, we have assigned the slr2033 homolog of Synechococcus sp. PCC 7002 the gene locus designation rubA. Genes encoding proteins similar to rubA have also been identified in the nucleomorph genome of the eukaryotic cryptomonad Guillardia theta (34) and the nuclear genome of the higher plant A. thaliana (35) (see Fig. 3).

Overexpression of the rubA Gene and Characterization and Distribution of RubA—Attempts to overproduce the full-length RubA protein were not successful, but overproduction of a truncated form of the protein in which a stop codon was inserted at codon 69 was achieved. The resulting protein was partly purified by ion exchange chromatography, and the absorption spectrum of this protein is shown in Fig. 4. The spectrum contains characteristic absorption features of sulfur → iron charge-transfer transitions; is characterized by absorption maxima at 375, 475, and 570 nm; and is very similar to the spectra of typical bacterial rubredoxins (20, 36, 37). The EPR spectrum of oxidized RubA is also shown in Fig. 4. This spectrum features the characteristic signals from a ground $S = \frac{3}{2}$ spin state near the rhombic limit ($E/D$) of $\frac{1}{3}$. These spectra are characterized by a nearly isotropic resonance around g = 4.31 derived from the middle Kramers doublet and a resonance at g = 9.46 that represents the lowfield peak of a rhombic signal. The predicted g = 0.86 derivative and g = 0.61 trough are broadened, are not easily detected, and indeed were not observed here. These spectral features are identical to those observed for the rubredoxin of the anaerobic bacterium C. pasteurianum (38) and are similar to the soluble fragment derived from the nucleomorph-encoded rubredoxin of the cryptomonad alga G. theta (13, 39). These data indicate that the water-soluble, rubredoxin-like domain of cyanobacterial RubA

Assembly of Photosystem I

1 7002
2 6803
3 7937
4 7120
5 G.theta MFVN NLLIDCRFVKNTCHFKVKKK FNLIQVNSSN
6 A.thali. MASATFSFCPLSQSHPHSSL IKPTPFLRYGKKLHHHLHHHI FITISYPSHHSRYLAVSRDD AATPLSSSDQ
7 C.t.-Rd1
8 C.t.-Rd2
9 C.t.-Rd3
9 C. past.

Rubredoxin
1 10 20 30 40 50
1 7002 MSDRIEK TLAEKAPASYECPSCGYIYT PAKGESRTNTPPGTAFEDLP LIWNCPVCSV
2 6803 MSERPPEK TLAELASPNECRACGGYVYI PSQGDQKTSVSPGTPFEALP LNWKCPVCGA
3 7937 MSEQ AVENTVLDRFECRSCGYVYE PEKGDSKHDIAPETPFAELP INWRCPVCTA
4 7120 MSEQ AVENTVLDRFECRSCGYVYE PEKGDNKHDIAPETPFAELP INWRCPVCTA
5 Gu.theta LSDEEKSSRIDEILRLKNAE KFMEIDEGKYECEACGYIYE PEKGDKFAGIPPGTPFVDLS DSFMCPACRS
6 Ar.thali.TQETETKEVEDTIEKRRMEE KFAVLNTGIYECRSCGYKYD ESAGDPSYPIPPGFQFDKLP EDWRCPTCGA
7 C.t.-Rd1 MNVKESE PAQADLQASWMCAECGYIYD PAEGNLETNIRPGMPFDKLP DDWSCPVCNH
8 C.t.-Rd2 MEQWKCNICGYIYN PETGDPEGDIPAGTSFESLP DSWMCPVCGA
9 C.t.-Rd3 MQKWVCVPCGYEYD PADGDPENGIEPGTAFEDLP EDWVCPVCGV
10 C. pas. MKKYTCTVCGYIYN PEDGDPDNGVNPGTDFKDIP DDWVCPLCGV

********* hinge ********* -transmembrane helix-
60 70 80 90 100 110 a.a.
1 7002 DRAQFRNIGAADAPSGFEEN LKYGFGVNNLSSNQKNLLIF GAMGLAILFFLSLYGLD 115
2 6803 PRNYFISTGETDAPSGFAEN LNYGFGFNRMSSGKKNLLIF GSLFVIFLFFLSLYGMG 115
3 7937 KKAAFSNIGPAGTASGFREN LGYGLGVNKLTPAQKNILIF GALALGFLFFISLYGLQ 111
4 7120 KKAFTNIGPAGTASGFREN LGYGLGVNKLTPAQKNILIF GALALGFLFFISLYGLQ 111
5 G.theta PKNQFKSI--KKVIAGFAEN QKYGLGGNSLTSGQKNTLIY GVLVLLFLLLLSGYLIE 159
6 A.thali. AQSFFESK--MVEIAGFAQN QQYGLGGNALTSGQKTGLIF GSLLLLFFALFLSGYFIQ 195
7 C.t.-Rd1 PKNQFTKFISQL 68
8 C.t.-Rd2 GKEEFTKI 52
9 C.t.-Rd3 DKSFFEPVS 53
10 C. pas. GKDQFEEVEE 54

FIG. 3. Amino acid sequence comparison of rubredoxin-like proteins from cyanobacteria with other rubredoxin sequences. An alignment is shown of amino acid sequences of rubredoxins from Synechococcus sp. PCC 7002 (this work) (1); Synechocystis sp. PCC 6803 (32) (2); Anabaena sp. PCC 7937³ (3); *Nostoc* sp. PCC 7120 (33) (4); *G. theta* (34) (5); *A. thaliana* (35) (6); *C. tepidum* rubredoxins 1, 2, and 3 (56) (7–9); and *C. pasteurianum* (20) (10). The total number of amino acid residues for each polypeptide is shown at the end of each sequence. The domains (rubredoxin, hinge, and membrane anchor transmembrane α-helix) are indicated above the sequences, and the four iron-binding cysteine residues are indicated in boldface type. Hyphens indicate insertions/deletions to optimize the sequence similarity.

binds a non-heme iron atom and thus represents a typical rubredoxin.

For the production of antibodies, the water-soluble N-terminal domain of RubA was fused to GST. The overproduced protein was purified on a GST affinity column, digested with thrombin, and repurified by gel filtration. Fractions containing rubredoxin were pooled, concentrated, and lyophilized, and the protein was judged to be homogeneous by SDS-PAGE analysis. N-terminal amino acid sequencing was performed to verify the correctness of the purified protein, which was used to raise antibodies in rabbits.

Antibodies to the soluble, recombinant RubA protein were used to examine the subcellular localization of RubA. Antibodies to the RubA protein cross-reacted with a protein with an apparent mass of 13 kDa in total membranes isolated from *Synechococcus* sp. PCC 7002 (Fig. 5, B and C, lane 1) and from *Synechocystis* sp. PCC 6803 (Fig. 5A, lane 1). These results indicate that RubA is probably a membrane-intrinsic protein, as suggested by the presence of the predicted transmembrane α-helix at the C terminus. An immunoreactive protein with an apparent mass of about 8 kDa (see Fig. 5) was usually detected as well and probably represents a degradation product, since the amount of this species increased in samples that had been stored for increasing times. Interestingly, the antibodies also cross-reacted strongly with proteins of ~13 and 8 kDa in thylakoid membranes isolated from *C. reinhardtii* (Fig. 5C, lane 2) and spinach (Fig. 5C, lane 3). Using very highly purified plasma (Fig. 5A, lane 2) and thylakoid (Fig. 5A, lane 3) membranes from *Synechocystis* sp. PCC 6803 (40), it can be seen that RubA is only found in the thylakoid membrane fraction. Similar results were obtained with *Synechococcus* sp. PCC 7002 membrane fractions prepared by the same method (data not shown). The antibodies did not cross-react with proteins in highly purified PS I trimers prepared with Triton X-100 (Fig. 5B, lane 2) or DM (Fig. 5B, lane 3) but cross-reacted strongly with a fraction enriched in PS I monomers prepared with the latter detergent (Fig. 5B, lane 4).

Since a previous study had suggested that the rubredoxin might be associated with PS II in spinach (13), the PS I monomer fraction, which typically contains some contaminating PS II complexes, was subjected to further purification by ion exchange chromatography. A fraction from a sucrose gradient containing a mixture of PS I monomers, PS II, and other proteins was applied to a DEAE-Sepharose column, extensively washed with starting buffer, and developed with a linear sodium chloride gradient (0–400 mM). Fractions were collected and subjected to SDS-PAGE, and the separated proteins were transferred electrophoretically to a nitrocellulose membrane for immunoblotting analysis. As shown in Fig. 6, RubA was detected in fractions 2–6, and the fraction containing the greatest amount of the RubA protein was fraction 3. The distribution of PS I monomers was evaluated by simultaneously determining the elution pattern of the PS I-specific protein PsaD. PsaD was detected in fractions 1–6, with the greatest amount of this

20348

Assembly of Photosystem I

![Image Description](image.png)

FIG. 4. Absorption and EPR spectra of the purified soluble fragment of the *Synechococcus* sp. PCC 7002 RubA protein. The upper inset shows the UV-visible absorption spectrum of the soluble RubA protein with absorption maxima at 375, 475, and 570 nm. The EPR spectrum of the protein shows a large resonance at g = 4.31 and a small resonance at g = 9.46. The lower inset shows the g = 9.46 resonance after 20-fold expansion. EPR conditions were as follows: microwave frequency, 9.4733 GHz; microwave power, 5 mW; modulation amplitude, 10 G; receiver gain, \(6.3 \times 10^4\); temperature, 10 K. The spectrum represents the average of four scans.

protein occurring in fraction 3. The distribution of PsaD thus appears to be nearly identical to the distribution of RubA. However, the observed distribution of RubA and PsaD differs from that for CP43 (PsbC), a subunit of PS II, which was detected in fractions 4–7 and which was maximal in fraction 4 (see Fig. 6). These results demonstrate that the RubA protein copurifies with the PS I monomer fraction but not with PS II complexes. This result is consistent with, but does not necessarily prove, the hypothesis that RubA can bind to PS I monomers.

Mutagenesis of the rubA Gene—The rubA gene was inactivated by inserting a 1.3-kb *Bam*HI DNA fragment that encodes the *aphII* gene and confers kanamycin resistance into the unique *Bgl*II site within the *rubA* coding sequence (Fig. 2B). This construction was used to transform three *Synechococcus* sp. PCC 7002 strains: the wild type, a PS I-less strain created by complete deletion of the *psaA* and *psaB* genes (16), and a PS II-less strain created by inactivation of the *psbCD* and *psbB* genes.² Transformants carrying the insertional inactivated

² G. Shen and D. A. Bryant, unpublished results.

Assembly of Photosystem I

A 1 2 3  
B 1 2 3 4  
C 1 2 3  

kDa  
36.7—  
28.1—  
16.5—  
8.3—

RubA

FIG. 5. Localization of the RubA protein through immunoblot analysis. A, immunoblot analysis of the RubA protein in total membranes (lane 1), highly purified plasma membranes (lane 2), and highly purified thylakoid membranes (lane 3), which had been isolated from cells of *Synechocystis* sp. PCC 6803 (40 μg of total protein was loaded per lane). The immunoreactive protein indicated by the lower arrowhead is probably a degradation product. B, identification of the RubA protein in *Synechococcus* sp. PCC 7002 thylakoids (lane 1), trimeric PS I complexes prepared with Triton X-100 (lane 2), trimeric PS I complexes prepared with DM (lane 3), and monomeric PS I particles prepared with DM (lane 4). C, detection of the RubA protein in thylakoid membranes of *Synechococcus* sp. PCC 7002 (lane 1), *C. reinhardtii* (lane 2), and spinach (lane 3).

Fraction 1  
Fraction 2  
Fraction 3  
Fraction 4  
Fraction 5  
Fraction 6  
Fraction 7  
PS I-Trimers  

RubA →  
PsaD →  
PsbC →

FIG. 6. Immunoblot analysis showing coelution of RubA and PS I monomers. Chl-containing fractions from ion exchange (DEAE-Sepharose) chromatography of the crude PS II and PS I-monomer fraction from a sucrose gradient were probed with antibodies against the *Synechococcus* sp. PCC 7002 RubA protein, the *Synechococcus* sp. PCC 7002 PsaD protein, and the *Synechocystis* sp. PCC 6803 PsbC (CP43) protein. Proteins from these column fractions and PS I trimers were separated by SDS-PAGE and transferred onto a nitrocellulose membrane. After blocking and probing with the three antibodies, the nitrocellulose blot was immunostained using the alkaline phosphatase detection method (22).

rubA::aphII gene were easily and fully segregated in the PS I-less ΔpsaAB strain. Full segregation of the rubA::aphII and rubA alleles in the ΔpsaAB strain was verified both by Southern blot hybridization analyses (data not shown) and by PCR (Fig. 2C). As shown in Fig. 2C, PCR amplification with DNA from *Synechococcus* sp. PCC 7002 wild-type cells using primers flanking the rubA gene amplified a 330-bp DNA fragment. PCR amplification with DNA from the rubA mutant strain using the same primers amplified a 1.6-kb DNA fragment as expected (see Fig. 2, B and C).

The rapid segregation of the rubA and rubA::aphII alleles in the ΔpsaAB background suggested that RubA might play a role in PS I biogenesis. To test this possibility, the ΔpsaAB and ΔpsaAB rubA::aphII strains were complemented at the psaAB locus by transformation with plasmid pAQEEmr80. This plasmid contains the wild-type psaAB genes (41) with the erm gene, that confers resistance to erythromycin, inserted at the EcoNI site that occurs 112 bp downstream from the stop codon of the psaB gene. The resulting pseudo-wild-type strain, produced by complementation from the PS I-less ΔpsaAB strain, is referred to here as strain WTREm. This strain has a phenotype that is identical to that of the true wild-type strain (strain PR6000) of *Synechococcus* sp. PCC 7002 as indicated by its ability to grow photoautotrophically at wild-type rates, by its Chl content, and by its ability to assemble functional PS I reaction centers (see Table I, and see below). Because complementation of the ΔpsaAB strain results in a wild-type phenotype, complementation of the psaAB locus in the ΔpsaAB rubA::aphII strain should produce a strain that is equivalent to inactivation of the rubA gene in a wild-type genetic background.

Thylakoid membranes of the pseudo-wild-type WTREm and rubA::aphII strain were subjected to immunoblotting analysis with antibodies to RubA. As shown in Fig. 7, lane 1, thylakoids of strain WTREm exhibit a strong cross-reaction with the an-

Assembly of Photosystem I

TABLE I  
Growth rate, chlorophyll content, and electron transport measurements for the Synechococcus sp. PCC 7002 strains

| Photoautotrophic growth (h) | WT$^a$ | WTREm$^b$ | rubA mutant$^c$ |
|----------------------------|--------|------------|------------------|
| At 5–10 $\mu$E m$^{-2}$ s$^{-1}$ | $62 \pm 2.7$ | $64 \pm 3.4$ | No growth |
| At 250 $\mu$E m$^{-2}$ s$^{-1}$ | $4.4 \pm 0.3$ | $4.2 \pm 0.4$ | No growth |

| Photomixotrophic growth (h) |  |  |  |
|----------------------------|--------|------------|------------------|
| At 5–10 $\mu$E m$^{-2}$ s$^{-1}$ | $17 \pm 1.5$ | $16 \pm 1.8$ | $34 \pm 2.1$ |
| At 250 $\mu$E m$^{-2}$ s$^{-1}$ | $3.4 \pm 0.4$ | $3.3 \pm 0.3$ | No growth |

| Chlorophyll content$^d$ (mg of Chl OD$_{730}^{-1}$) | $4.46 \pm 0.28$ | $4.57 \pm 0.24$ | $2.49 \pm 0.21$ |

| Oxygen evolution$^e$ ($\mu$mol O$_2$ OD$_{730}^{-1}$ l$^{-1}$ h$^{-1}$) |  |  |  |
|----------------------------|--------|------------|------------------|
| Whole chain (H$_2$O to CO$_2$) | $1330 \pm 60$ | $1280 \pm 45$ | NM$^f$ |
| PS II-mediated | $1960 \pm 130$ | $1940 \pm 120$ | $1490 \pm 160$ |

| PS I-mediated electron transport$^g$ |  |  |  |
|----------------------------|--------|------------|------------------|
| DCPIP to MV ($\mu$mol O$_2$ mg of Chl$^{-1}$ h$^{-1}$) | $510 \pm 40$ | $530 \pm 57$ | NM |
| Cyt c$_6$ to flavodoxin ($\mu$mol mg of Chl$^{-1}$ h$^{-1}$) | $6710 \pm 340$ | $6650 \pm 390$ | NM |

$a$ Wild type.  
$b$ $\Delta psaAB$ mutant strain complemented with $psaAB$ genes.  
$c$ $rubA::aphII$.  
$d$ Chlorophyll was extracted from cells were grown photomixotrophically at light intensity of 5–10 $\mu$E m$^{-2}$ s$^{-1}$.  
$e$ Measurements were made on whole cells grown at the exponential phase.  
$f$ NM, not measurable.  
$g$ Measurements were made on thylakoid membranes.

FIG. 7. Immunoblot analysis with antibodies against the Synechococcus sp. PCC 7002 RubA protein. Lane 1, thylakoids of Synechococcus sp. PCC 7002 strain WTREm; lane 2, thylakoids of the $rubA$ mutant; lane 3, thylakoids of the $rubA$ mutant complemented with a plasmid encoding the full-length $rubA$ gene of Synechocystis sp. PCC 6803; lane 4, whole-cell extracts of the $rubA$ mutant complemented with a plasmid encoding a truncated, soluble fragment of Synechocystis sp. PCC 6803 RubA; lane 5, fraction eluted at 0.2 M NaCl from DEAE-Sephacel chromatography of the supernatant containing soluble proteins from the $rubA$ mutant complemented with a plasmid encoding a water-soluble fragment of Synechocystis sp. PCC 6803 RubA. Molecular mass markers (kDa) are indicated on the right. The arrows point to the RubA protein and truncated, soluble RubA domain (RubA$_{\text{TR}}$). 40 $\mu$g of protein was loaded per lane.

sensitivity to high light intensities, a phenotype often observed with mutants deficient in PS I (42).

Complementation of the $rubA$ Mutant—To verify the phenotype of the $rubA::aphII$ mutant, the mutant was complemented by transforming the strain with plasmid pAQE7002FL, which encodes the wild-type $rubA$ gene of Synechococcus sp. PCC 7002. The complemented mutant strain grew photoautotrophically at a rate equivalent to that of the wild-type strain (data not shown). Complementation of the $rubA::aphII$ mutant was also performed using plasmid pAQE19/6803FL, which encodes the wild-type $rubA$ gene of Synechocystis sp. PCC 6803. This strain also exhibited a growth rate under photoautotrophic conditions that was equal to that of the pseudo-wild-type strain WTREm (data not shown). Immunoblotting analysis (Fig. 7, lane 3) showed that the amount of RubA protein in this complemented strain was similar to that for the pseudo-wild-type strain WTREm (Fig. 7, lane 1). These results establish clearly that the growth defect observed in the $rubA::aphII$ mutant is only due to the $rubA$ mutation and rule out any possibility of a defect caused by a secondary mutation.

The primary difference between the cyanobacterial rubredoxin, RubA, and other bacterial rubredoxins is the additional 50 amino acids at the C terminus and in particular the predicted transmembrane $\alpha$-helix that occurs near the C terminus of the protein. To evaluate the role of this domain of RubA, the $rubA::aphII$ strain was transformed with plasmid pLAT4/6803TR, which carries the Synechocystis sp. PCC 6803 $rubA$ gene, which has been modified by site-directed mutagenesis to include a stop codon at residue 69 (see Fig. 3). Selection of spectinomycin- and kanamycin-resistant transformants resulted in a strain that can only produce the water-soluble, N-terminal rubredoxin-like domain of RubA. This strain was unable to grow photoautotrophically (data not shown), although the water-soluble, rubredoxin-like domain was detected in cell extracts by immunoblotting analysis (Fig. 7, lane 5). These results suggest that the hydrophobic C-terminal region of the RubA protein might have an important and specific function.

One additional protein was tested for its ability to complement the $rubA::aphII$ mutation. A chimeric rubredoxin gene was constructed from codons 1–48 of the rubredoxin gene of *C. pasteurianum* (a kind gift from Dr. Donald Kurtz, University of Georgia), and codons 63–115 of the $rubA$ gene of Synechococcus sp. PCC 7002. To facilitate the gene construction, residue 48 of *C. pasteurianum* rubredoxin was changed from glu-

Assembly of Photosystem I

FIG. 8. 77 K fluorescence emission spectra. Fluorescence emission spectra of the *Synechococcus* sp. PCC 7002 WTREm strain (solid line) and the *rubA* mutant (dotted line). Each spectrum is the average of five independent measurements. The excitation wavelength was 440 nm.

![Graph](https://i.imgur.com/yourimageurl.png)

WTREm  
$rubA$ mutant  

Wavelength (nm)

tamine to glutamate, and the six C-terminal residues of this rubredoxin were replaced by residues 63–115 (hinge and transmembrane α-helix) of RubA of *Synechococcus* sp. PCC 7002. When the *rubA::aphII* strain was complemented with this chimeric rubredoxin gene, the resulting strain could not grow photoautotrophically (data not shown). This result suggests that the C-terminal modified *C. pasteurianum* rubredoxin does not have the structural and/or functional properties necessary to replace the RubA protein of cyanobacteria. It is possible that the inability to complement the *rubA* mutant strain is due to the absence of the N-terminal extension that is found in the cyanobacterial RubA proteins (see Fig. 3) but that is missing in bacterial rubredoxins.

Chlorophyll Content and 77 K Fluorescence Emission of the *rubA* Mutant—Table I shows a comparison of the Chl contents of three strains: the wild type, the pseudo-wild-type WTREm, and the *rubA::aphII* mutant. The *rubA* mutant cells contained about half of the Chl of the control cells. The carotenoid content of the mutant cells was slightly higher than in the control cells, but phycobiliprotein levels were essentially identical in all three strains (data not shown).

Most of the Chl in cyanobacteria is associated with the two photosynthetic complexes, PS I and II (16). Thus, fluorescence emission at low temperature is a convenient way to detect differences in the PS I/PS II ratio in thylakoid membranes. Fig. 8 shows the fluorescence emission spectra at 77 K of equivalent numbers of cells of the *rubA* mutant and pseudo-wild-type WTREm strains. For the WTREm strain, excitation of whole cells at 440 nm resulted in a typical, large emission peak at 716–717 nm, which results from the excitation of PS I-associated Chl. PS II-associated Chl contribute to the emission maxima at 695 nm (mostly from the PsbB/CP47 protein) and at 685 nm (mostly from PsbC/CP43) (43). The smaller emission peaks at 647 and 663 nm are due to weak excitation of phycocyanin and allophycocyanin, respectively. The same emission peaks were detected in the *rubA* mutant, which indicates that PS I and II complexes were present in thylakoid membranes. However, a large decrease was observed in the amplitude of the PS I-associated fluorescence emission at 717 nm. In contrast, PS II seems to be present at nearly normal levels based upon the presence and amplitude of the Chl emission peaks at 685 and 695 nm (Fig. 8). The increased amplitude and 2-nm blue shift of the 683-nm emission peak is consistent with an increased amount of a minor Chl *a*-protein complex in cells that appears to be related to HliA and other small CAB-like proteins found in cyanobacteria (44, 45).² These results indicate that a deficiency in RubA did not appear to affect substantially the assembly of PS II reaction centers but led to a reduction in cellular PS I content.

Oxygen Evolution and Electron Transport Measurements—The low temperature fluorescence emission measurements suggested that PS II and I (albeit at reduced level) were still assembled in the *rubA* mutant. To determine whether the assembled photosynthetic complexes could carry out electron transport, oxygen evolution measurements were performed. The whole-chain electron transport rate of the pseudo-wild-type WTREm strain was similar to that of the wild-type strain (PR6000) of *Synechococcus* sp. PCC 7002 (Table I). No whole-chain oxygen evolution was detected for the *rubA* mutant, although a substantial PS II activity (77% of wild-type level) was measured when PS II-mediated oxygen evolution activity was assayed with ferricyanide and 2,5-dimethyl benzoquinone as electron acceptors. These results indicate that the photosynthesis defect in the *rubA* mutant is mostly likely due to a defect in PS I.

To explore this possibility further, the oxygen uptake rate for PS I-mediated electron transport from 2,6-dichlorophenolindophenol to methyl viologen (Mehler reaction) was measured for the WTREm and *rubA* mutant strains. As shown in Table I, the oxygen uptake rate (530 μmol of O₂ mg of Chl⁻¹ h⁻¹) for thylakoids from the pseudo-wild-type WTREm was essentially identical to that of the wild type strain, but thylakoids from the *rubA* mutant had no detectable PS I-mediated oxygen uptake. PS I activity was also measured by determining the rate of electron transfer from the physiological donor, cytochrome *c₆* to a physiologically relevant acceptor, flavodoxin. As shown in Table I, thylakoids from the WTREm strain had a PS I-mediated rate of flavodoxin reduction (6650 μmol mg of Chl⁻¹ h⁻¹) similar to that of wild-type thylakoids. However, consistent with the results from the Mehler reaction studies, thylakoids of the *rubA* mutant exhibited no measurable flavodoxin reduction. These results clearly establish that the PS I reaction centers of the *rubA* mutant strain are not functional in electron transport.

20352

**Assembly of Photosystem I**

show that the three stromal polypeptides of PS I were detectable in the rubA mutant, but their levels were very much reduced relative to those of the control strain. Similarly, low levels of PsaC, PsaD, and PsaE could be detected in the thylakoids of the rubA mutant (data not shown). Faster migrating, immunoreactive species were observed for both PsaC and PsaE (Fig. 9B, arrows). These data suggest that these proteins are unstable and are rapidly degraded in the rubA mutant cells. From these results, it is concluded that PsaC, PsaD, and PsaE are probably synthesized normally in the rubA mutant, but these proteins are more easily lost from PS I complexes during purification and are much less stable than in the wild type.

---

### DISCUSSION

Rubredoxins are typically very small (~6 kDa), water-soluble, mononuclear non-heme iron proteins that occur most frequently in anaerobic eubacteria and archaea. However, rubredoxins have also been found in a few aerobic bacteria (e.g., *Pseudomonas oleovorans* (46) and *Acinetobacter* sp. (47)) and more recently have even been found in oxygen-evolving prokaryotes and eukaryotes (see below). Rubredoxins have been spectroscopically, biophysically, and structurally well characterized for many years. The roles of these proteins in cellular metabolism and physiology remain poorly understood, although they have recently been implicated as electron donors or acceptors in diverse biochemical reactions. Reduced rubredoxins have been proposed or shown to serve as electron donors to nitrate reductase (48), hydrogenase (49), rubredoxin:oxygen reductase (50), alkane hydroxylase (47), ruberythrin (51), superoxide reductases (51, 52), and bacterioferritin (53). Several bacterial genomes have been shown to encode at least two rubredoxins, and it is possible that sequence variations in the vicinity of the iron-binding ligands could confer specific functionalities to the different proteins (47, 54–56). The physiological donor of electrons to rubredoxin for the above reactions is generally not known, with the exception of those organisms that have been shown to synthesize an NADH:rubredoxin oxidoredutase (see Ref. 37). Interestingly, in the photoautotrophic green sulfur bacterium *Chlorobium tepidum*, whose genome encodes three rubredoxins (Ref. 56; see Fig. 3), a rubredoxin serves as the electron acceptor when pyruvate is oxidized to acetyl-CoA and CO₂ by pyruvate:ferredoxin oxidoreductase (57).

The complete genome sequence analysis of the cyanobacterium *Synechocystis* sp. PCC 6803 showed that this organism has the potential to synthesize an unusual rubredoxin-like protein of 115 amino acids (Ref. 32; see Fig. 3). Similar proteins have now been identified in diverse oxygen-evolving prokaryotes including *Synechococcus* sp. PCC 7002 (this work), *Anabaena variabilis* strain PCC 7937, *Nostoc* sp. PCC 7120 (33), *Nostoc punctiforme,*³ *Prochlorococcus marinus* strains MED4 and MIT9313,³ and *Synechococcus* sp. WH8102.³ Rubredoxins with strong sequence similarity to the cyanobacterial rubredoxins were also recently discovered on chromosome 2 of the nucleomorph of the cryptomonad *G. theta* (34) and on chromosome 1 of the higher plant *A. thaliana* (35). This broad distribution of RubA homologs among cyanobacteria, prochlorophytes, cryptomonads (red algae), green algae, and higher plants strongly suggests that rubA homologs might be found in all oxygen-evolving photosynthetic organisms.

The nucleomorph represents the remnant nucleus of the red-algal endosymbiont acquired by the eukaryotic ancestor of the cryptomonad. The product of the nucleomorph-encoded rubredoxin gene includes a long leader sequence that is pre-

---

³ Available at the DOE-Joint Genome Institute site on the World Wide Web (spider.jgi-psf.org/JGI_microbial/html/).
sumed to direct the protein into the cryptomonad chloroplast. A water-soluble, recombinant form of the *G. theta* rubredoxin has the optical and EPR spectroscopic characteristics of typical rubredoxins (13, 39), and the structure of the water-soluble zinc-rubredoxin domain of the protein has been solved by NMR methods (59). This soluble domain of the recombinant rubredoxin is quite similar in structure to typical bacterial rubredoxins. The full-length rubredoxin of *G. theta* could be imported into pea chloroplasts (13), and by using antibodies to the *G. theta* rubredoxin in immunoelectron microscopy studies, the protein was shown to be localized to thylakoids in *G. theta* and pea chloroplasts. Although rubredoxin was also detected in thin sections of the cyanobacterium *Synechocystis* sp. PCC 6803, labeling of both the thylakoids and the cytoplasm was observed in this case (see Ref. 13; Fig. 4B). It was suggested that the rubredoxin might be associated with PS II, since the protein was detected immunologically in grana thylakoids and PS II core complexes isolated from spinach chloroplasts (13). However, no direct demonstration of a role for rubredoxin in PS II structure, function, or assembly was established in these studies.

The biochemical and spectroscopic properties of the cyanobacterial rubredoxin RubA reported here agree well with those reported for the rubredoxin of *G. theta* described above (13, 39, 59). The optical and EPR properties of the water-soluble recombinant rubredoxin domain from *Synechococcus* sp. PCC 7002 were nearly identical to those reported for the recombinant *G. theta* protein. The presence of an immunologically related protein in spinach thylakoids was confirmed here with an antiserum to a cyanobacterial RubA, and these antibodies also detected a protein of similar mass in thylakoid membranes of the green alga *C. reinhardtii*. RubA was shown to be present in purified thylakoid but not plasma membranes of *Synechocystis* sp. PCC 6803, and the important role of the putative transmembrane α-helix at the C terminus of the protein was established by complementation studies using a truncated form of the protein.

In typical rubredoxins, three β-strands are joined by two loops, each of which contains two cysteine residues (Cys-X-X-Cys) ligated to an iron atom to form a structure known as a "knuckle" (59). All structurally characterized rubredoxins and rubredoxin-like domains of more complex proteins share significant amino acid sequence identity (~50%; see Fig. 3) and have essentially isostructural [Fe(S-Cys)4] sites with distorted tetrahedral coordination geometry (48, 59). Surprisingly, however, the redox potentials of rubredoxins and rubredoxin-like domains vary over a considerable range from about −150 mV to about +230 mV (53). Although the redox potential of the RubA protein was not determined in this study, the soluble domain of the *G. theta* protein was shown to have a midpoint potential of +125 mV (13). The C-terminal extension found in RubA homologs from cyanobacteria and other oxygen-evolving organisms is also highly conserved. This suggests that the flexible hinge region and the membrane anchor are likely to be quite important in the function of the RubA protein. The N-terminal extensions of the cyanobacterial and the eukaryotic RubA homologs are somewhat variable in sequence, but all of these sequences are rather hydrophilic and highly charged. *C. pasteurianum* rubredoxin (midpoint potential ~−55 mV (38)) was unable to complement the *rubA* mutation, even when fused to the C-terminal extension of a cyanobacterial RubA. This could indicate that the redox potential of this protein is inappropriate to replace RubA function, that the N-terminal extension is very important in the functionality of RubA, that highly specific protein-protein interactions are required for RubA function, or that insufficient expression of the fusion protein was achieved

to effect complementation. Further studies will be required to determine which of these possibilities is correct.

The results presented here clearly establish that the loss of the RubA rubredoxin causes the complete loss of PS I activity as well as the destabilization of the three peripheral polypeptides, PsaC, PsaD, and PsaE, that are associated with the stromal surface of the PS I complex (5). Complementation studies with the *rubA* genes of *Synechococcus* sp. PCC 7002 and *Synechocystis* sp. PCC 6803 showed that no unexpected secondary mutations had occurred in the *rubA* mutant strain. PS II activities in the *rubA* mutant of *Synechococcus* sp. PCC 7002 were only slightly reduced on a per cell basis relative to the wild-type (76% of wild-type level; Table I). Our observations suggest that RubA is transiently associated with monomeric PS I complexes. Trimerization of PS I complexes requires the PsaL subunit (60), and mutational studies indicate that trimerization is stabilized by the presence of the PsaK subunit.⁴ It is likely that trimerization is one of the final steps in PS I biogenesis, since PS I monomers lacking PsaL and PsaK are fully active to perform electron transfer from cytochrome *c*₆ to flavodoxin (61). Thus, the presence of RubA in the monomer fraction suggests that it might normally exert its action prior to PS I trimerization. In the accompanying paper (58), trimeric PS I complexes from the *rubA* mutant strain are characterized in detail. These studies establish that the loss of PS I activity results from a specific defect: an inability to assemble the interpeptide [4Fe-4S] cluster Fₓ.

Acknowledgments—We thank Dr. Himadri Pakrasi (Washington University, St. Louis, MO) for providing highly purified samples of plasma and thylakoid membranes from *Synechocystis* sp. PCC 6803, Dr. Wim Vermaas (Arizona State University, Tempe, AZ) for antibodies for the PsbC protein of PS II, and Dr. Donald Kurtz (University of Georgia) for supplying the *rubA* gene from *C. pasteurianum*.

---

**REFERENCES**

1. Golbeck, J. H., and Bryant, D. A. (1991) in *Current Topics in Bioenergetics* (Lee, C. P., ed) Vol. 16, pp. 83–177, Academic Press, Inc., New York
2. Chitnis, P. R. (2001) *Annu. Rev. Plant Physiol. Plant Mol. Biol.* **52**, 593–626
3. Diner, B. A., and Babcock, G. T. (1996) in *Oxygenic Photosynthesis: The Light Reactions* (Ort, D. R., and Yocum, C. F., eds) pp. 213–247, Kluwer Academic Publishers, Dordrecht, The Netherlands
4. Zouni, A., Witt, H. T., Kern, J. Fromme, P., Krauss, N., Saenger, W., and Orth, P. (2001) *Nature* **409**, 739–743
5. Jordan, P., Fromme, P., Witt, H. T., Klukas, O., Saenger, W., and Krauß, N. (2001) *Nature* **411**, 909–917
6. Vassiliev, I. R., Antonkine, M. L., and Golbeck, J. H. (2001) *Biochim. Biophys. Acta* **1507**, 139–160
7. Bartsevich, V. V., and Pakrasi, H. B. (1997) *J. Biol. Chem.* **272**, 6382–6387
8. Boudreau, E., Takahashi, Y., Lemieux, D., Turmel, M., and Rochaix, J.-D. (1997) *EMBO J.* **16**, 6095–6104
9. Naver, H., Boudreau, E., and Rochaix, J.-D. (2001) *Plant Cell* **13**, 2731–2745
10. Vermaas, W. F. J. (1993) *Annu. Rev. Plant Physiol. Plant Mol. Biol.* **44**, 457–481
11. Pakrasi, H. B. (1995) *Annu. Rev. Genet.* **29**, 755–776
12. Togasaki R. K., and Surzycki, S. J. (1998) in *The Molecular Biology of Chloroplasts and Mitochondria in Chlamydomonas* (Rochaix, J.-D., Goldschmidt-Clermont, M., and Merchant, S., eds) pp. 13–23, Kluwer Academic Publishers, Dordrecht, The Netherlands
13. Wastl, J., Duin, E. C., Iuzzolino, L., Dörner, W., Link, T., Hoffmann, S., Sticht, H., Dau, H., Lingelbach, K., and Maier, U.-G. (2000) *J. Biol. Chem.* **275**, 30058–30063
14. Shen, G., Antonkine, M. L., Vassiliev, I. R., Golbeck, J. H., and Bryant, D. A. (1998) in *Photosynthesis: Mechanisms and Effects* (Garab, G., ed) Vol. IV, pp. 3147–3150, Kluwer Academic Publishers, Dordrecht, The Netherlands
15. Stevens, S. E., Jr., and Porter, R. D. (1980) *Proc. Natl. Acad. Sci. U. S. A.* **77**, 6052–6056
16. Shen, G., and Bryant, D. A. (1995) *Photosynth. Res.* **41**, 41–53
17. Buzby, J. S., Porter, R. D., and Stenven, S. E., Jr. (1985) *Science* **230**, 805–807
18. Brosius, J. (1989) *DNA* **8**, 759–777
19. Sakamoto, T., Stirewalt, V. L., and Bryant, D. A. (1997) in *Physiology, Biochemistry, and Molecular Biology of Plant Lipids* (Williams, J. P., Khan, M. U., and Lem, N. W., eds) pp. 380–382, Kluwer Academic Publishers, Dordrecht, The Netherlands
20. Mathieu, I., Meyer, J., and Moulis, J.-M. (1992) *Biochem. J.* **285**, 255–262
21. Studier, F. W., Rosenberg, A. H., Dunn, J. J., and Dubendorff, J. W. (1990) *Methods Enzymol.* **185**, 60–89

---

⁴ G. Shen, J. H. Golbeck, and D. A. Bryant, unpublished observations.
20354

**Assembly of Photosystem I**

22. Harlow, E., and Lane, D. (1988) *Antibodies: A Laboratory Manual*, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY

23. Schägger, H., and von Jagow, G. (1987) *Anal. Biochem.* **166**, 368–379

24. Blum, H., Beier, H., and Gross, H. J. (1987) *Electrophoresis* **8**, 93–99

25. Lichtenthaler, H. K. (1987) *Methods Enzymol.* **148**, 350–382

26. Jung, Y. S., Yu, L., and Golbeck, J. H. (1995) *Photosynth. Res.* **46**, 249–255

27. Golbeck, J. H. (1995) in *CRC Handbook of Organic Photochemistry and Photobiology* (Song, P. S., and Horspools., W., eds) pp. 1407–1419, CRC Press, Inc., Boca Raton, FL

28. Zhao, J., Li, R., and Bryant, D. A. (1998) *Anal. Biochem.* **264**, 263–270

29. Lojero, C. G., and Krogmann, D. W. (1996) *Photosynth. Res.* **47**, 293–299

30. Meurer, J., Plucken, H., Kowallik, K. V., and Westhoff, P. (1998) *EMBO J.* **17**, 5286–5297

31. Meyer, J., Gaganon, J., Seiker, L. C., Dorsselaer, A. V., and Moulis, J.-M. (1990) *Biochem. J.* **271**, 839–841

32. Kaneko, T., Sato, S., Kotani, H., Tanaka, A., Asamizu, E., Nakanmura, Y., Miyajima, N., Hirosewa, M., Sugiura, M., Sasamoto, S., Kimura, T., Hosouchi, T., Matsuno, A., Muraki, A., Nakazaki, N., Naruo, K., Okumura, S., Shimpo, S., Takeuchi, C., Wada, T., Watanabe, A., Yamada, M., Yasuda, M., and Tabata, S. (1996) *DNA Res.* **3**, 109–136

33. Kaneko, T., Nakamura, Y., Wolk, C. P., Kuritz, T., Sasamoto, S., Watanabe, A., Iriguchi, M., Ishikawa, A., Kawashima, K., Kimura, T., Kishida, Y., Kohara, M., Matsumoto, M., Matsuno, A., Muraki, A., Nakazaki, N., Shimpo, S., Sugimoto, M., Takazawa, M., Yamada, M., Yasuda, M., and Tabata, S. (2001) *DNA Res.* **8**, 227–253

34. Douglas, S., Zauner, S., Fraunholz, M., Beaton, M., Penny, S., Deng, L. T., Wu, X., Reith, M., Cavalier-Smith, T., Maier, U.-G. (2001) *Nature* **410**, 1091–1096

35. Theologis, A., Ecker, J. R., Palm, C. J., Federspiel, N. A., Kaul, S., White, O., Alonso, J., Altafi, H., Araujo, R., Bowman, C. L., Brooks, S. Y., Buehler, E., Chan, A., Chao, Q., Chen, H., Cheuk, R. F., Chin, C. W., Chung, M. K., Conn, L., Conway, A. B., Conway, A. R., Creasy, T. H., Dewar, K., Dunn, P., Etgu, P., Feldblyum, T. V., Feng, J., Fong, B., Fujii, C. Y., Gill, J. E., Goldsmith, A. D., Haas, B., Hansen, N. F., Hughes, B., Huizar, L., Hunter, J. L., Jenkins, J., Johnson-Hopson, C., Khan, S., Khaykin, E., Kim, C. J., Koo, H. L., Kremenetskaia, I., Kurtz, D. B., Kwan, A., Lam, B., Langin-Hooper, S., Lee, A., Lee, J. M., Lenz, C. A., Li, J. H., Li, Y., Lin, X., Liu, S. X., Liu, Z. A., Luros, J. S., Maiti, R., Marziali, A., Militscher, J., Miranda, M., Nguyen, M., Nierman, W. C., Osborne, B. I., Pai, G., Peterson, J., Pham, P. K., Rizzo, M., Rooney, T., Rowley, D., Sakano, H., Salzberg, S. L., Schwartz, J. R., Shinn, P., Southwick, A. M., Sun, H., Tallon, L. J., Tambunga, G., Toriumi, M. J., Town, C. D., Utterback, T., Van Aken, S., Vaysbergm M., Vysotskiam, V. S., Walker, M., Wu, D., Yu, G., Fraser, C. M., Venter, J. C., and Davis, R. W. (2000) *Nature* **408**, 816–820

36. Das, A., Coulter, E. D., Kurtz, D. M. Jr., and Ljungdahl, L. G. (2001) *J. Bacteriol.* **183**, 1560–1567

37. Guedon, E., and Petitdemange, H. (2002) *Biochem. Biophys. Res. Commun.* **285**, 496–502

38. Eidsness, M. K., Burded, A. E., Richie, K. A., Kurtz, D. M., Scott, R. A., Smith, E. T., Ichiyé, T., Beard, B., Min, T., and Kang, C. (1999) *Biochemistry* **38**, 14803–14809

39. Wastl, J., Sticht, H., Maier, U.-G., Rösch, P., and Hoffmann, S. (2000) *FEBS Lett.* **471**, 191–196

40. Norling, B., Zak, E., Andersson, B., and Pakrasi, H. (1998) *FEBS Lett.* **436**, 189–192

41. Cantrell, A., and Bryant, D. A. (1987) *Plant Mol. Biol.* **9**, 453–468

42. Johnson, T. W., Shen, G., Zybailov, B., Kolling, D., Reategui, R., Beauparlant, S., Vassiliev, I. R., Bryant, D. A., Jones, A. D., Golbeck, J. H., and Chitnis, P. R. (2001) *J. Biol. Chem.* **275**, 8523–8530

43. Shen, G., and Vermaas, W. F. J. (1994) *J. Biol. Chem.* **269**, 13904–13910

44. Funk, C., and Vermaas, W. (1999) *Biochemistry* **38**, 9397–9404

45. He, Q., Dolganov, N., Bjorkman, O., and Grossman, A. R. (2001) *J. Biol. Chem.* **276**, 306–314

46. Kok, M., Oldenhuis, R., van der Linden, M. P., Meulenberg, C. H., Kingma, J., and Witholt, B. (1989) *J. Biol. Chem.* **264**, 5442–5451

47. Geissdörfer, W., Frosch, S. C., Haspel, G., Ehrt, S., and Hillen, W. (1995) *Microbiology* **141**, 1425–1432

48. Seki, Y., Seki, S., Satoh, M., Ikeda, A., and Ishimoto, M. (1989) *J. Biochem. (Tokyo)* **106**, 336–341

49. Rey, L., Hidalgo, E., Palacios, J., and Ruiz-Argüeso (1992) *J. Mol. Biol.* **228**, 998–1002

50. Gomes, C. M., Silva, G., Oliveira, S., LeGall, J., Liu, M. Y., Xavier, A. V., Rodrigues-Pousada, C., and Teixeira, M. (1997) *J. Biol. Chem.* **272**, 22502–22508

51. Coulter, E. D., and Kurtz, D. M., Jr. (2001) *Arch. Biochem. Biophys.* **394**, 76–86

52. Jenny, F., Verhagen, M., Cui, W., and Adams, M. (1999) *Science* **286**, 306–309

53. da Costa, P. N., Romão, C. V., LeGall, J., Xavier, A. V., Melo, E., Teixeira, M., and Saralva, L. M. (2001) *Mol. Microbiol.* **41**, 217–227

54. LeGall, J., Liu, M., Gomes, C. M., Braga, V., Pacheco, I., Regalla, M., Xavier, A., and Reixeiira, M. (1998) *FEBS Lett.* **429**, 295–298

55. Lumppio, H. L., Shenvi, N. V., Summers, A. O., Voordouw, G., and Kurtz, D. M., Jr. (2001) *J. Bacteriol.* **183**, 101–108

56. Eisen, J. A., Nelson, K. E., Paulsen, I. T., Heidelberg, J. F., Wu, M., Dodson, R. J., Deboy, R., Gwinn, M. L., Nelson, W. C., Haft, D. H., Hickey, E. K., Peterson, J. D., Durkin, A. S., Kolonay, J. L., Yang, F., Holt, I., Umayam, L. A., Mason, T., Brenner, M., Shea, T. P., Parksey, D., Feldblyum, T. V., Hansen, C. L., Craven, M. B., Radune, D., Khouri, H., Fujii, C. Y., White, O., Venter J. C., Volfovsky, N., Gruber, T. M., Ketchum, K. A., Tettelin, H., Bryant, D. A., and Fraser, C. M. (2002) *Proc. Natl. Acad. Sci. U. S. A.*, in press

57. Yoon, K. S., Hille, R., Hemann, C., and Tabita, F. R. (1999) *J. Biol. Chem.* **274**, 29772–29778

58. Shen, G., Antonkine, M. L., van der Est, A., Vassiliev, I. R., Brettel, K., Bittl, R., Zhao, J., Stehlik, D., Bryant, D. A., and Golbeck, J. H. (2002) *J. Biol. Chem.* **277**, 20355–20366

59. Schweimer, K., Hoffmann, S., Wastl, J., Maier, U.-G., Rösch, P., and Sticht, H. (2000) *Protein Sci.* **9**, 1474–1486

60. Schluchter, W. M., Shen, G., Zhao, J., and Bryant, D. A. (1996) *Photochem. Photobiol.* **64**, 53–66

61. Yang, F., Shen, G., Schluchter, W., Ganago, A., Vassiliev, I., Bryant, D., and Golbeck, J. (1998) *J. Phys. Chem.* **102**, 8288–8299
