
Molecular Microbiology (2002) 43(3), 809–821

# Quorum sensing *Escherichia coli* regulators B and C (QseBC): a novel two-component regulatory system involved in the regulation of flagella and motility by quorum sensing in *E. coli*

Vanessa Sperandio,† Alfredo G. Torres and James B. Kaper*
Center for Vaccine Development and Department of Microbiology and Immunology, University of Maryland School of Medicine, 685 West Baltimore Street, Baltimore, MD 21201, USA.

## Summary

Quorum sensing is a cell-to-cell signalling mechanism in which bacteria secrete hormone-like compounds called autoinducers. When these autoinducers reach a certain threshold concentration, they interact with bacterial transcriptional regulators, thereby regulating gene expression. Enterohaemorrhagic *Escherichia coli* (EHEC) O157:H7 as well as *E. coli* K-12 produces the autoinducer-2 (AI-2), which is synthesized by the product of the *luxS* gene, and previous work from our laboratory has shown that genes encoding the EHEC type III secretion system were activated by quorum sensing. Recently, by hybridizing an *E. coli* K-12 gene array with cDNA synthesized from RNA extracted from EHEC strain 86-24 and its isogenic *luxS* mutant, we observed that other potential virulence-associated factors, such as genes encoding the expression and assembly of flagella, motility and chemotaxis, were also activated by quorum sensing. The array data also indicated that several genes encoding putative *E. coli* regulators were controlled by quorum sensing. In this report, we describe a two-component system regulated by quorum sensing that shares homology with *Salmonella typhimurium* PmrAB, which we have named quorum sensing *E. coli* regulator B and C (QseBC). The *qseBC* genes, previously identified only as open reading frames b3025 and b3026, are organized in an operon in the *E. coli* chromosome, with *qseB* encoding the response regulator and *qseC* the sensor kinase. We confirmed the regulation of *qseBC* by quorum sensing using *qseB::lacZ* transcriptional fusions and characterized the phenotypes of an isogenic *qseC* mutation in EHEC. This mutant expressed less flagellin and had reduced motility compared with the wild-type and complemented strains. Transcription of *flhD*, *fliA*, *motA* and *fliC::lacZ* fusions was decreased in the *qseC* mutant, suggesting that *qseBC* is a transcriptional regulator of flagella genes. A *qseC* mutant was also generated in *E. coli* K-12 strain MC1000 that showed the same phenotypes as the EHEC mutant, indicating that *qseBC* regulates flagella and motility by quorum sensing in both EHEC and K-12. QseBC activates transcription of *flhDC*, which is the master regulator for the flagella and motility genes and, in the absence of *flhD*, QseBC failed to activate the transcription of *fliA*. Motility of a *luxS*, but not of a *qseC*, mutant can be restored by providing AI-2 exogenously as preconditioned media, suggesting that the *qseC* mutant is unable to respond to AI-2. However, QseC has no effect on the expression of other quorum sensing-controlled genes such as those encoding for the type III secretion system. These data indicate that QseBC is one component of the quorum-sensing regulatory cascade in both EHEC and K-12 that is involved in the regulation of flagella and motility genes, but that additional regulators in this cascade remain to be characterized.

## Introduction

Enterohaemorrhagic *Escherichia coli* (EHEC) O157:H7 colonizes the large intestine, where it causes attaching and effacing (AE) lesions and produces Shiga toxin (Stx), which is responsible for the major symptoms of haemorrhagic colitis and haemolytic uraemic syndrome (reviewed by Kaper et al., 1998; Nataro and Kaper, 1998). The AE lesion is characterized by effacement of the intestinal epithelial cell microvilli and the rearrangement of the cytoskeleton to form a pedestal-like structure that cups the bacteria individually. The genes involved in the formation of the AE lesion are encoded within a chromosomal pathogenicity island named the locus of enterocyte effacement (LEE) (McDaniel et al., 1995). The LEE region

Accepted 14 November, 2001. *For correspondence.* E-mail jkaper@umaryland.edu; Tel. (+1) 410 706 2493; Fax (+1) 410 706 0182. †Present address: Department of Microbiology, University of Texas Southwestern Medical Center, Dallas, TX 75390-9048, USA.

© 2002 Blackwell Science Ltd

810 V. Sperandio, A. G. Torres and J. B. Kaper

contains five major operons: LEE1, LEE2, LEE3, tir (LEE5) and LEE4 (Mellies *et al.*, 1999). Operons LEE1, 2 and 3 encode the majority of the components of the type III secretion apparatus (Jarvis *et al.*, 1995); *tir*/LEE5 encodes the bacterial adhesin intimin (Jerse and Kaper, 1991) and the translocated intimin receptor (Kenny *et al.*, 1997), and LEE4 encodes the proteins EspA, EspB and EspD, which are secreted by the type III secretion system (Donnenberg *et al.*, 1993; Kenny *et al.*, 1996a,b; Lai *et al.*, 1997), including EspA, which forms filamentous-like structures involved in the translocation of EspB and Tir to the epithelial cell (Knutton *et al.*, 1998). The LEE genes are activated directly by the LEE-encoded regulator (Ler), which is the first gene in the LEE1 operon (Friedberg *et al.*, 1999; Mellies *et al.*, 1999; Elliott *et al.*, 2000; Sperandio *et al.*, 2000; Bustamante *et al.*, 2001). We reported recently that genes involved in the formation of the AE lesion were regulated by quorum sensing through the autoinducer-2 (AI-2), which is synthesized by the product of the *luxS* gene (Sperandio *et al.*, 1999). Quorum sensing *E. coli* regulator A (QseA), which is a member of the LysR family, activates transcription of *ler* and Ler, thereby leading to the activation of other LEE genes in a cascade fashion (Sperandio *et al.*, 2002). This quorum-sensing system is a global regulatory system that controls not only genes involved in pathogenesis, including flagellation, motility and Shiga toxin expression, but also genes involved in bacterial metabolism, DNA repair, nucleotide and protein biosynthesis, cell growth and division among others (Sperandio *et al.*, 2001).

The expression, synthesis, assembly and function of flagella and motility require the expression of more than 50 genes divided among at least 17 operons, constituting a large and co-ordinately regulated flagella regulon (reviewed by Macnab, 1996). The expression of these genes is regulated in response to several environmental cues (reviewed by Chilcott and Hughes, 2000), and among these is bacterial cell density (Sperandio *et al.*, 2001b). Within the regulon, the operons are divided into three temporally regulated, hierarchical transcriptional classes: class 1, class 2 and class 3 (Kutsukake *et al.*, 1990). The class 1 genes encode the master flagella regulator FlhDC, which activates the transcription of class 2 genes, including genes encoding for the structure and assembly of the hook–basal body and the transcriptional regulators FigM and FliA (*σ*²⁸). FliA regulates the transcription of class 3 genes, which include *fliC* (flagellin) and the *mot* operon (reviewed by Chilcott and Hughes, 2000).

The *luxS*/AI-2 quorum-sensing system activates the expression of genes involved in assembly of flagella and motility by activation of *flhDC* transcription (Sperandio *et al.*, 2001). In this study, we describe a two-component regulatory system that is involved in the regulation of *flhDC* and, consequently, the other flagella regulon genes by quorum sensing. This two-component system shares homology with *Salmonella typhimurium* PmrAB and was renamed quorum sensing *E. coli* regulator B and C (QseBC). QseBC is a previously undescribed activator of the flagella regulon, as well as a component of the *E. coli* quorum-sensing regulatory cascade.

## Results

Escherichia coli putative two-component system b3025 and b3026 is regulated by quorum sensing

We previously hybridized a DNA gene array from *E. coli* K-12 with cDNA synthesized from RNA extracted from the wild-type EHEC strain 86-24 and its isogenic *luxS* mutant VS94 (Sperandio *et al.*, 2001b). Through this array analysis, we found 19 putative *E. coli* transcriptional regulators, as defined by Blattner *et al.* (1997), that were either activated or repressed by quorum sensing (data not shown). One of these putative regulators was activated 17-fold by quorum sensing and is encoded by the open reading frames (ORFs) referred to as b3025 and b3026 in the K-12 (GenBank accession number AE000384) and in the EHEC (GenBank accession number AE005532) genomes (Fig. 1A). These genes seem to be organized in an operon and belong to the family of two-component regulators (Blattner *et al.*, 1997), with b3025 (or YgiX) being the response regulator and b3026 (or YgiY) the sensor kinase (Fig. 1B). These genes belong to a poorly characterized family of two-component systems present in *Pasteurella multocida*, *Haemophilus influenzae* and *Pseudomonas aeruginosa*; b3025 shares 46% identity, 62% similarity over 222 amino acids with PmrA from *S. typhimurium*, and b3026 shares 28% identity, 46% similarity over 440 amino acids with ArmS from *Pseudomonas* and 28% identity, 45% similarity over 356 amino acids with *S. typhimurium* PmrB. PmrAB is involved in regulating genes that confer resistance to antimicrobial peptides such as polymyxin (Gunn and Miller, 1996; Soncini and Groisman, 1996) and response to extracytoplasmic ferric iron (Wosten *et al.*, 2000). We then renamed b3025 and b3026 quorum sensing *E. coli* regulators B and C (QseBC). Both genes are present in EHEC and K-12; QseB has three amino acid changes, and QseC has eight amino acid changes between these strains.

To confirm the regulation of *qseBC* by quorum sensing, we generated a *qseB::lacZ* transcriptional fusion by amplifying the regulatory region of this operon (from bp 8041 to bp 8700 of the sequence in GenBank with accession number AE000384) and cloning it into a vector containing a promoterless *lacZ* gene, generating plasmid pVS159 (Table 1). Transcription of *qseB* was downregulated 2.5-fold in the *luxS* mutant (VS94) compared with wild-type (86-24) and complemented (VS95) strains (Fig. 1C).

QseBC regulates flagella and motility in E. coli 811

Fig. 1. Regulation of transcription of qseBC by quorum sensing.

A. Section of gene array containing b3026 hybridized with cDNA synthesized from RNA extracted from the wild-type strain (86-24) and the luxS mutant (VS94).

B. Organization of ORFs b3025 (YgiX) and b3026 (YgiY) in the E. coli chromosome; b3025 (QseB) is the predicted response regulator and b3026 (QseC) the predicted sensor kinase.

C. β-Galactosidase activity of a qseBC::lacZ fusion (pVS159) in 86-24 (wild-type), VS94 (luxS mutant) and VS95 (complemented) strain backgrounds in DMEM at an OD<sub>600</sub> of 1.0.

D. β-Galactosidase activity of a qseBC::lacZ fusion (pVS159) in VS94 (luxS mutant), in fresh DMEM and in DMEM preconditioned with strains 86-24, VS94 and VS95.

The 2.5-fold difference in the expression of qseB::lacZ seen with plasmid pVS159 compared with the 17-fold difference observed with the array may result from a copy number effect of the cloned qseB promoter on a multicopy plasmid. To confirm further that qseB transcription was upregulated by quorum sensing and not as a result of some metabolic change associated with a luxS mutation, we tested whether transcription of the qseB::lacZ fusion

Table 1. Bacterial strains and plasmids used in this study.

| Strains       | Genotype/description                                                                 | Reference                          |
|---------------|-------------------------------------------------------------------------------------|------------------------------------|
| MC1000        | araD139Δ(araABC-leu) 7679 galU galKΔ(lac) X74 rpsL thi                               | Silhavy and Beckwith (1984)        |
| 86-24         | Stx2+ EHEC strain (serotype O157:H7)                                                 | Griffin et al. (1988)             |
| VS94          | 86-24 luxS mutant                                                                   | Sperandio et al. (2001)            |
| VS95          | VS94 with plasmid pVS84                                                             | Sperandio et al. (2001)            |
| VS138         | 86-24 qseC mutant                                                                  | This study                         |
| VS184         | E. coli MC1000 qseC mutant                                                          | This study                         |
| VS156         | VS138 with plasmid pVS155                                                           | This study                         |
| VS179         | VS138 with plasmid pVS178                                                           | This study                         |
| VS185         | VS184 with plasmid pVS178                                                           | This study                         |
| BB170         | V. harveyi luxN::Tn5 (sensor-1<sup>-</sup> sensor-2<sup>+</sup>)                      | Surette and Bassler (1998)         |
| flhD::km      | MC1000 flhDC mutant                                                                | Matsumura                          |
| YK4131        | MC1000 flhD point mutation                                                          | Matsumura                          |

| Plasmids      | Description                                                                         | Source                             |
|---------------|-------------------------------------------------------------------------------------|------------------------------------|
| TOPO PCR blunt | PCR blunt cloning vector with topoisomerase                                        | Invitrogen                         |
| pBluescript-II KS | Cloning vector                                                                    | Stratagene                        |
| pBADMycHis    | C-terminal Myc-His-Tag vector                                                       | Invitrogen                         |
| pBAD33        | Cloning vector                                                                      | Guzman et al. (1995)              |
| pCVD442       | Suicide vector                                                                      | Donnenberg and Kaper (1991)       |
| pRS551        | lacZ reporter gene fusion vector                                                    | Simons et al. (1987)              |
| pVSAP         | bla::lacZ in pRS551                                                                 | Sperandio et al. (1999)           |
| pVS155        | MC1000 qseC in pBADMycHisA                                                          | This study                         |
| pVS161        | MC1000 qseBC in pBADMycHisA                                                         | This study                         |
| pVS178        | EHEC qseBC in pBAD33                                                                | This study                         |
| pVS129        | Upstream region and qseB in pBluescript-II KS                                       | This study                         |
| pVS131        | qseB::tet in pBluescript KSII                                                       | This study                         |
| pVS128        | Region downstream of qseC in TOPO PCR blunt                                         | This study                         |
| pVS132        | qseB::tet::region downstream of qseC in pCVD442                                     | This study                         |
| pVS175        | fliC::lacZ in pRS551                                                                | Sperandio et al. (2001)            |
| pVS176        | motA::lacZ in pRS551                                                                | Sperandio et al. (2001)            |
| pVS177        | fliAeh<sup>a</sup>::lacZ in pRS551                                                  | Sperandio et al. (2001)            |
| pVS182        | flhD::lacZ in pRS551                                                                | Sperandio et al. (2001)            |
| pVS183        | fliAehK12::lacZ in pRS551                                                           | This study                         |
| PVS159        | qseB::lacZ in pRS551                                                                | This study                         |

a. The eh stands for the fliA regulatory region containing the EHEC regulatory region.

© 2002 Blackwell Science Ltd, Molecular Microbiology, 43, 809–821

812 V. Sperandio, A. G. Torres and J. B. Kaper

**A**
LB + 0.2% Arabinose MEM + 0.2% Arabinose
86-24 VS138 VS156 86-24 VS138 VS156

**B**
Early log Mid log Late log stationary
86-24 VS138 86-24 VS138 86-24 VS138 86-24 VS138

Fig. 2. Western blots of whole-cell lysates with anti-H7 flagellin serum.
A. Western blots of whole-cell lysates of 86-24 (wild type), VS138 (qseC mutant) and VS156 (VS138 complemented with qseC in plasmid pVS155) in LB and DMEM supplemented with 0.2% arabinose grown at 37°C to an OD<sub>600</sub> of 1.0; equal amounts of proteins were loaded in each well.
B. Western blots of whole-cell lysates of strains 86-24 and VS138 grown at 37°C in LB in different growth phases (early, mid and late exponential and stationary phases); equal amounts of proteins were loaded in each well.

in a luxS mutant (VS94) was activated by exogenous AI-2. As the final structure of AI-2 is still not reported and this molecule proved to be very difficult to purify (Schauder et al., 2001), we used media preconditioned with the wild-type (86-24) and complemented (VS95) strains, which produce AI-2, in these tests. As a negative control, we used media preconditioned with the luxS mutant (VS94). We then observed that transcription of qseB was activated by both media preconditioned with the wild-type and complemented strains, but not with media preconditioned with the luxS mutant (Fig. 1D), indicating that the activation of qseB transcription resulted from AI-2 signalling and, therefore, quorum sensing. The activation of qseB transcription was greater (sevenfold) in media preconditioned with VS95 (complemented) strain, which produces more AI-2 than the wild-type strain (twofold).

Characterization of a qseC EHEC mutant

We generated a qseC (sensor) EHEC mutant by replacing qseC with a gene encoding tetracycline resistance, which left qseB intact, resulting in strain VS138. The qseC mutation was complemented with qseC cloned into a pBADMycHis vector (pVS155), with the transcription of qseC being driven from the pBAD vector promoter, thereby generating strain VS156. This mutation was also complemented with both qseBC genes cloned into a pBAD33 vector (pVS178), generating strain VS179. The qseC mutant was still able to produce AI-2, as supernatants prepared with this mutant induced the production of light in Vibrio harveyi strain BB170 (Surette and Bassler, 1998), which only produces light in response to AI-2 (data not shown).

We observed in Western blots of whole-cell lysates probed with a polyclonal antiflagellin antiserum that the qseC mutant (VS138) expressed considerably less flagellin than the wild-type (86-24) or the complemented (VS156) strains in both LB medium and Dulbecco's modified Eagle medium (DMEM) with 0.2% arabinose. Expression of flagellin in LB is higher than in DMEM for all these strains (Fig. 2A). Production of flagellin increased in the wild-type strain during middle and late exponential growth phases, and remained lower in the qseC mutant (VS138) throughout growth (Fig. 2B).

The qseC mutant (VS138) also formed smaller motility halos in tryptone + 0.2% arabinose semi-solid media than the wild-type (86-24) and complemented (VS179) strains (Fig. 3). After incubation at 37°C for 8 h, strains 86-24 and VS179 formed motility halos of 18 mm, whereas VS138 formed halos of 8 mm; after 16 h, strains 86–24 and VS179 formed halos of 32 mm and 28 mm, respectively, while VS138 formed a halo of 20 mm (Fig. 3). These motility experiments were performed in triplicate, and the measurement of halo diameters reflects the average of these experiments.

**A**
Tryptone + 0.2% Ara 8 h

**B**
Tryptone + 0.2% Ara 16 h

Fig. 3. Motility of 86-24 (wild type), VS138 (qseC mutant) and VS179 (VS138 complemented with qseBC in plasmid pVS178) in tryptone semi-solid agar supplemented with 0.2% arabinose.
A. After incubation at 37°C for 8 h.
B. After incubation at 37°C for 16 h.

© 2002 Blackwell Science Ltd, Molecular Microbiology, 43, 809–821

Regulation of transcription of flagella class 1, 2 and 3 promoters

As the *qseC* mutant (VS138) had reduced flagella production and motility (Figs 2 and 3), we examined whether the transcription of flagella class 1, 2 and 3 promoters was affected by QseBC. Using transcriptional fusions of the regulatory regions of *flhD* (class 1), *fliA* (class 2), *fliC* (class 3) and *motA* (class 3) to a *lacZ* reporter gene (Table 1), we observed that transcription of all these promoters was downregulated in VS138 compared with 86-24 (wild-type) and VS179 (complemented) strains in LB with 0.2% arabinose (Fig. 4). Transcription of *flhD* was downregulated twofold, and *fliA* was downregulated 17-fold in VS138 compared with 86-24 and VS179. Transcription of *fliC* was downregulated 28-fold in VS138 compared with 86-24 and 25.5-fold compared with VS179. Transcription of *motA* was downregulated fivefold compared with 86-24 and 12-fold compared with VS179 (Fig. 4A). The basal transcriptional level of vector alone (*pRS551*) and of a control β-lactamase promoter::*lacZ* (*bla::lacZ*) fusion did not change in the different strain backgrounds (data not shown).

QseBC activation of *fliA* transcription is higher in LB media containing arabinose (17-fold upregulation in 86-24 compared with VS138) than in LB alone (fivefold), and there was no difference in the activation of *fliA* in LB at either 37°C or 30°C. Transcriptional activation of *fliA* was reduced (twofold) in LB 0.4 M NaCl and DMEM with both high (Gibco BRL catalogue no. 13000) and low glucose (Gibco BRL catalogue no. 11054) (Fig. 4B). The basal transcriptional level of vector alone (*pRS551*) and of a control β-lactamase promoter::*lacZ* (*bla::lacZ*) fusion was the same in all the environmental conditions tested (Fig. 4B).

QseBC activation of *flhD* transcription is basically the same in all the conditions tested: twofold in LB, LB + 0.2% arabinose and DMEM high glucose; 2.5-fold in LB 0.4 M NaCl; and threefold in DMEM low glucose (Fig. 5C). However, the basal level transcription of *flhD* in VS138 is two- and threefold higher in DMEM low and high glucose, respectively, compared with all the other conditions (Fig. 4C).

The environmental transcriptional regulation of *fliA* (Fig. 4B) is consistent with the flagellin expression profile observed by Western blots, with *fliA* being better expressed in LB than in DMEM (Fig. 2A). However, activation of *flhD* transcription was unchanged in either LB or DMEM (Fig. 4C). Moreover, the basal level of *flhD* transcription was increased in DMEM. These results are contradictory to the flagellin expression profile by Western blot analysis; however, transcription of *flhD* is very complex and sensitive to several environmental conditions (reviewed by Chilcott and Hughes, 2000), which may account for this discrepancy.

QseBC regulates flagella and motility in E. coli 813

QseBC also activates the flagella regulon in E. coli K-12

As *qseBC* is present in both EHEC and K-12, we examined whether this regulator was also involved in the activation of the flagella regulon in K-12. As mentioned above, QseB has three amino acid changes between EHEC and K-12, and QseC has eight. However, complementation of a *qseC* mutation in EHEC can be achieved using *qseC* from either K-12 (strain VS156; Fig. 2) or EHEC (strain VS179; Figs 3 and 4), suggesting that these genes might have the same function in both strains and can be interchangeable. The regulatory region of *flhDC* is 98% identical between EHEC and K-12 [GenBank accession numbers: AE005411 (bp 5441–6410) and D90831 (bp 11066–11655)], and the regulatory region of *fliA* is 97% identical between EHEC and K-12 [GenBank accession numbers: AE005415 (bp 1381–1945) and AE000285 (bp 1383–1944)], suggesting that regulation of the flagella operon is conserved between these two strains. The *flhD::lacZ* fusion that we constructed (*pVS182*) contained only the conserved region between these two strains. However, the *fliA::lacZ* (*pVS177*) fusion that we constructed initially had the conserved regulatory region between EHEC and K-12 plus an additional 390 bp that were unique to EHEC, as they contained part of the structural gene of *fliC*. We then constructed a second *fliA::lacZ* fusion containing only the conserved region between the two strains (*pVS183*), and this fusion behaves exactly the same in 86-24, VS138 (*qseC* mutant) and VS179 (complemented) strain backgrounds (Fig. 5A). Both fusions (*pVS177* and *pVS183*) were downregulated 17-fold in VS138 compared with strains 86-24 and VS179 (Fig. 5A).

We observed that, in a strain MC1000 (K-12) background, transcription of *fliA::lacZ* (*pVS183*) was upregulated fivefold by overexpression of *qseBC* in a multicopy plasmid (*pVS178*) (Fig. 5B). We then generated a *qseC* MC1000 mutant strain (VS184), which we complemented using plasmid *pVS178*, generating strain VS185. Transcription of a *fliA::lacZ* fusion was downregulated 10- and 32.5-fold in strain VS184 (MC1000 *qseC* mutant) compared with MC1000 and VS185 (complemented) respectively (Fig. 5B). Transcriptional regulation of *fliA* by QseBC is dependent on the presence of a functional FlhDC, as QseBC failed to activate transcription of a *fliA::lacZ* fusion in an *flhDC* (MC1000*flhD::km*) and an *flhD* (YK4131) mutant (Fig. 5B). Furthermore, the MC1000 *qseC* mutant (VS184) is non-motile in tryptone with 0.2% arabinose on semi-solid agar plates, whereas motility was observed in MC1000 and VS185 (complemented) strains (data not shown). Taken together, these results indicate that QseBC activates the flagella regulon by activating transcription of *flhDC* in both EHEC and K-12 backgrounds.

814 V. Sperandio, A. G. Torres and J. B. Kaper

**LB + 0.2 % Arabinose**

Fig. 4. A. Transcription of *fliC*, *motA*, *fliA* and *flhD::lacZ* fusions in 86-24, VS138 (pBAD33) (*qseC* mutant with vector only) and VS179 (complemented *qseC* mutant) backgrounds in LB + 0.2% arabinose at 37°C grown to an OD<sub>600</sub> of 1.0.

B. Transcription of a *fliA::lacZ*, *bla::lacZ* (pVSAP) fusion and vector alone (pRS551) in 86-24 and VS138 backgrounds in different environmental conditions: LB ara (LB + 0.2% arabinose), LB 37 (LB at 37°C), LB 30 (LB at 30°C), LB 0.4 M NaCl, MLG (DMEM low glucose) and MHG (DMEM high glucose).

C. Transcription of a *flhD::lacZ* fusion in 86-24 and VS138 backgrounds in different environmental conditions.

The *qseC* mutant does not respond to exogenous AI-2

We reported recently that the EHEC *luxS* (VS94) mutant produced fewer flagella and had reduced motility compared with the wild-type strain. Transcription of flagella gene classes 1, 2 and 3 (*flhD*, *fliA*, *fliC* and *motA*) was also downregulated in the *luxS* mutant (Sperandio et al., 2001). The *qseC* mutant (VS138) has the same characteristics.

© 2002 Blackwell Science Ltd, Molecular Microbiology, 43, 809–821

QseBC regulates flagella and motility in E. coli 815

Fig. 5. A. Transcription of *fliA::lacZ* fusions pVS177 (containing the common regulatory region between EHEC and K-12 and also 390 bp present only in EHEC) and pVS183 (containing only the common regulatory region between EHEC and K-12) in 86-24, VS138 (pBAD33) and VS179 backgrounds in LB with 0.2% arabinose.

B. Transcription of *fliA::lacZ* (pVS183) in K-12 MC1000 (pBAD33), MC1000 (pVS178; *qseBC* in pBAD33), VS184 (MC1000 *qseC* mutant), VS185 (VS184 with plasmid pVS178), *flhD::km* (MC1000 *flhDC* mutant with pBAD33), *flhD::km* (pVS178), YK4131 (MC1000 *flhD* mutant with pBAD33) and YK4131 (pVS178) in LB with 0.2% arabinose.

istics as the *luxS* mutant (VS94) with regard to flagella expression and motility (Figs 2–4). Both *qseC* and *luxS* mutants form smaller motility halos in a semi-solid DMEM plate compared with strain 86-24 (Fig. 6A). Motility of the *luxS* mutant can be restored by giving AI-2 exogenously as media preconditioned by growth of strain 86-24 (which produces AI-2) (Sperandio *et al.*, 2001b) (Fig. 6B). However, this is not the case with the *qseC* mutant, which remains non-motile even in the presence of AI-2 (Fig. 6B), suggesting that a *qseC* mutant is unable to sense and respond to AI-2.

QseC does not affect the expression of other quorum-sensing regulated genes

The observation that a *qseC* mutant did not respond to AI-2 could be an indication that QseC was the sensor for AI-2 in *E. coli*. However, a *qseC* mutation did not have any effect on the transcription of other genes and phenotypes regulated by this quorum-sensing mechanism. We have reported previously that quorum sensing through the *luxS/AI-2* system controlled transcription of the type III secretion system in EHEC (Sperandio *et al.*, 1999). A *qseC* mutation has no effect on the secretion of proteins (EspA and EspB) in EHEC, which are secreted through

the type III secretion apparatus (Fig. 7A) and the production of AE lesions on cultured epithelial cells (data not shown). Overexpression of *qseBC* on a multicopy plasmid also had no effect on the transcription of *LEE1*, *LEE2*, *LEE3*, *tir* and *LEE4::lacZ* fusions (Fig. 7B). Taken together, these results may suggest that QseC is one of the members of this regulatory cascade but is not the true sensor, or that there is more than one sensor kinase interacting with AI-2, and QseC is the one responding for flagellation and motility.

Discussion

Quorum sensing is a very powerful means used by bacteria to assess not only the population of its own species but also the population of other species of bacteria in a given environment (Schauder and Bassler, 2001). EHEC colonizes the large intestine, where the bacterial flora is present in high cell density, making it logical that EHEC uses quorum sensing to regulate different genes necessary for its survival within the host, as well as virulence genes, such as those encoding the LEE type III secretion system, Shiga toxin production and the flagella regulon (Sperandio *et al.*, 1999; 2001). Given the hundreds of different bacterial species present in the large intestine, it

© 2002 Blackwell Science Ltd, Molecular Microbiology, 43, 809–821

816 V. Sperandio, A. G. Torres and J. B. Kaper

A D-MEM
B D-MEM + 50% PC 86-24

Fig. 6. Motility of strains 86-24 (wild-type EHEC), VS94 (86-24 luxS mutant) and VS138 (86-24 qseC mutant) in (A) DMEM and (B) DMEM + 50% DMEM preconditioned by growth of 86-24.

is also logical that the AI-2 quorum-sensing system, which is involved primarily in interspecies communication, would be used by EHEC rather than AI-1 or another intraspecies quorum-sensing system. The proteins comprising the type III secretion systems are homologous to the flagella basal body proteins and, as quorum sensing activates the type III secretion system in EHEC, it is perhaps not too surprising that it also activates the flagella regulon. Moreover, coupled regulation of type III secretion and flagella genes has been described recently in Salmonella (Eichelberg and Galan 2000; Goodier and Ahmer, 2001).

Our previous DNA array study of a luxS mutant (Sperandio et al., 2001) revealed 19 putative *E. coli* transcriptional regulators that were either activated or repressed by quorum sensing through the luxS system. One putative regulator that was highly activated by quorum sensing belongs to the family of two-component regulators and shares homology with *S.* typhimurium PmrAB. The genes encoding these putative *E. coli* regulators, renamed quorum sensing *E. coli* regulators B and C (QseBC), encode a predicted 24 kDa response regulator (QseB) and a predicted 50 kDa sensor kinase (QseC). We confirmed that transcription of *qseBC* was activated

by the AI-2 quorum-sensing system (Fig. 1), but we do not know whether the presence of AI-2 has any effect on the phosphorylation state of these proteins, which is one of the questions we intend to address in the future. An EHEC *qseC* mutant produces fewer flagella and is less motile than the wild-type and complemented strains (Figs 2 and 3). Transcription of flagella class 1 (*flhD*), class 2 (*fliA*) and class 3 (*fliC* and *motA*) genes was also downregulated in the *qseC* mutant (Fig. 4). An *E. coli* K-12 *qseC* mutant had the same phenotypes as the EHEC mutant (Fig. 5). Taken together, these data suggest that QseBC is a novel two-component system involved in regulation by quorum sensing of the flagella regulon in both EHEC and K-12 *E. coli* strains, and that it might act by activating transcription of *flhDC* by quorum sensing. We do not know at this point whether this effect is direct or indirect, and we intend to answer this question in the future with biochemical studies with purified QseB to address whether it interacts directly with the promoter region of *flhDC*.

FlhDC is the master flagella regulator, and transcriptional regulation of *flhDC* is sensitive to several environmental conditions and cell state sensors (reviewed by Chilcott and Hughes, 2000). The promoter of *flhDC*

Fig. 7. A. Western blots of secreted proteins from strains 86-24, VS138 and VS156 probed with antisera directed against EspA and EspB. B. Transcription of LEE1, LEE2, LEE3, *tir* and LEE4::lacZ fusions in single copy in the chromosome of strain TE2680 in the presence of vector alone (pBADMycHis) or *qseBC* cloned in pVS161.

defines the flagellar class 1 promoter and is a crucial point at which the decision to initiate or prevent flagellar biosynthesis occurs. This promoter is controlled by a number of global regulators such as CRP (Yokota and Gots, 1970; Silverman and Simon, 1974), heat shock proteins DnaK, DnaJ and GrpE (Shi et al., 1992) and H-NS (Bertin et al., 1994; Kutsukake, 1997; Yanagihara et al., 1999), as well as by several environmental signals such as temperature control (Qualding and Stocker, 1962; Adler and Templeton, 1967), high concentrations of inorganic salts, carbohydrates or alcohols (Li et al., 1993; Shi et al., 1993), growth phase (Nishimura and Hirota, 1989), DNA supercoiling (Li et al., 1993; Shi et al., 1993; Kutsukake, 1997; Das et al., 1998), phosphatidylethanolamine and phosphatidylglycerol synthesis (Shi et al., 1993; Mizushima et al., 1995), cell cycle control (Morgan et al., 1995; Prüß and Matsumura, 1996; Mizushima et al., 1997) and quorum sensing (Sperandio et al., 2002). Reflecting this complex regulation, six transcriptional start sites have been mapped in the class 1 promoter region (Yanagihara et al., 1999). Consequently, establishing conclusively whether QseB interacts directly with the promoter region of *flhDC* will not be a simple task. It will involve a comprehensive deletion analysis of this regulatory region, as well as competition assays with other DNA-binding proteins known to regulate *flhDC* expression.

Although QseBC is involved in the regulation of the flagella regulon by quorum sensing, it plays no role in the regulation of other quorum sensing-regulated phenotypes such as expression of the LEE-encoded type III secretion system (Fig. 7) and Shiga toxin production (data not shown). Quorum sensing regulatory cascades tend to be very complex and involve a variety of regulators and, in these cascades, the regulators themselves are usually regulated by quorum sensing (reviewed by de Kievit and Iglewski, 2000). In *V. harveyi*, which is the only organism in which the *luxS*/AI-2 quorum-sensing cascade has been characterized, the sensor for AI-2 is a periplasmic protein, LuxP, and the AI-2–LuxP complex interacts with the LuxQ sensor, which is part of a complex phosphorelay cascade

involved in the production of bioluminescence (reviewed by Schauder and Bassler, 2001). The LuxP homologue in *E. coli* is the small ribose-binding protein, and AI-2 complexed with this protein might interact with one or more sensor kinases in the *E. coli* membrane. If flagellation and motility were the only phenotypes known to be regulated by quorum sensing in EHEC, we might have assumed that QseC was the kinase interacting with the AI-2 small ribose-binding protein complex, as a *qseC* mutant does not respond to AI-2 in the motility phenotype (Fig. 6). However, a *qseC* mutation does not affect other quorum sensing-regulated phenotypes, suggesting that other regulators are involved in this cascade. Therefore, we propose two different hypotheses for the role of QseBC in this cascade: (i) QseC is one of several sensor kinases that detect AI-2 and is dedicated to transducing this signal towards regulation of the flagella regulon, whereas other AI-2 sensor kinases regulate different sets of genes; (ii) there is only one sensor kinase recognizing AI-2, and this regulator activates transcription and perhaps phosphorylation of the QseBC two-component system. We have described one regulator of the LysR family (QseA) that is involved in the regulation of the LEE genes, but not the flagella regulon by quorum sensing (Sperandio et al., 2002), and also have other candidates to be included in this regulatory cascade (Fig. 8).

The QseBC regulator described in this paper, which is involved in activation of the flagella regulon by quorum sensing, is one of the first members of the *E. coli* AI-2 regulatory cascade to be described. A better understanding of this regulatory system could lead to the identification of additional virulence factors and could potentially provide novel targets for vaccine and drug development.

### Experimental procedures

#### Bacterial strains and plasmids

Strains and plasmids used in this study are listed in Table 1. All *E. coli* strains were grown aerobically at 37°C in LB medium. Selective antibiotics were added at the following

![Diagram](attachment:diagram.png)

Fig. 8. Model of quorum-sensing regulation in EHEC. Quorum sensing activates the transcription of *qseA*, which activates transcription of the LEE genes by quorum sensing. There are also additional transcriptional factors, as yet undescribed, involved in the activation of the LEE genes by this mechanism. Quorum sensing activates transcription of *qseBC*, which in turn activates transcription of the flagella regulon.

© 2002 Blackwell Science Ltd, *Molecular Microbiology*, **43**, 809–821

818 V. Sperandio, A. G. Torres and J. B. Kaper

Table 2. Oligonucleotides used in this study.

| Name    | Sequence                                                                 |
|---------|--------------------------------------------------------------------------|
| K1979   | 5′-CGCGGATCCCAAGTCGTTACGCTGCCGTTC-3′                                     |
| K2026   | 5′-CCGGAATTCAGCCTGACGCCGAGACTAAG-3′                                      |
| K2027   | 5′-CGGGGTACCCGATCGCGCTACTCGTCTGG-3′                                      |
| K2028   | 5′-CCGCTCGAGAGTATGACTGGATGCTTGAG-3′                                      |
| K2163   | 5′-CGGGGTACCACCAAACGTCTTAGTCTGCGC-3′                                     |
| K2164   | 5′-CGGGGTACCCTGATAGAAGATGACATGCTG-3′                                     |
| K2165   | 5′-CCGGAATTCCCCCCAGCTTACCTTCGCCCTCAA-3′                                  |
| K2181   | 5′-CCGGAATTCTCCAGTGTCAGGGTTTGTCA-3′                                      |
| K2182   | 5′-CGCGGATCCCCCGTCGCAATCAGCATGTC-3′                                      |
| K2309   | 5′-CGCGGATCCTCACGATAAACAGCCCTGCG-3′                                      |
| K2332   | 5′-GCTCTAGAGACGGTTATCGCAGGGATGA-3′                                       |
| K2333   | 5′-CCCAAGCTTTTACCAGCTTACCTTCGCCCT-3′                                     |
| K2372   | 5′-CCGAATTCGAACAACACCACTACCAACC-3′                                       |

concentrations: ampicillin 100 μg ml⁻¹; tetracycline 25 μg ml⁻¹; and kanamycin, 50 μg ml⁻¹.

Recombinant DNA techniques

Plasmid purification, polymerase chain reaction (PCR), restriction, ligation, transformation and DNA gel electrophoresis were performed using standard methods (Sambrook et al., 1989). DNA sequence analysis was performed at the University of Maryland Biopolymer Facility on an ABI automated sequencer using DNA purified by Qiagen midi-columns. All oligonucleotide primers used are listed in Table 2.

Plasmid pVS155 was constructed by amplifying the qseC gene from K-12 strain MG1655 with Pfx DNA polymerase (Gibco BRL) using primers K2163 and K2165 and cloning into the EcoRI–KpnI site of vector pBADMycHisA (Invitrogen). Plasmid pVS161 was constructed by amplifying the qseBC genes from K-12 strain MG1655 with Pfx DNA polymerase (Gibco BRL) using primers K2164 and K2165 and cloning into the EcoRI–KpnI site of vector pBADMycHisA (Invitrogen). Plasmid pVS178 was generated by amplifying the qseBC gene from EHEC strain 86-24 with Pfx DNA polymerase (Gibco BRL) using primers K2332 and K2333 and cloning into the XbaI–HindIII site of vector pBAD33 (Guzman et al., 1995).

To generate a qseC mutation, we initially constructed plasmid pVS129 by amplifying the upstream region and the qseB gene from K-12 strain MG1655 with Pwo polymerase (Boehringer Mannheim) using primers K1979 and K2026 and cloning into the EcoRI–Smal site of vector pBluescript-II KS (Stratagene). A tetracycline resistance cassette derived from pBR322 was cloned into the EcoRI–KpnI site of pVS129, generating plasmid pVS131. Plasmid pVS128 was constructed by amplifying the downstream region of the qseC gene from K-12 strain MG1655 with Pwo polymerase (Boehringer Mannheim) using primers K2027 and K2028 and cloning into TOPO PCR blunt vector (Invitrogen) according to the manufacturer’s instructions. Plasmid pVS132 was constructed by cloning the qseB::Tc cassette digested from pVS129 with Xbal and KpnI, together with the downstream region of qseC digested with Xhol and KpnI from plasmid pVS128 into the R6K-based suicide vector pCVD442 (Donnenberg and Kaper, 1991) digested with Xbal and SalI.

The EHEC qseC mutant strain, named VS138, was generated by allelic exchange of the qseB gene and the downstream region of qseC in pVS132 using Tc and sucrose selection as described previously (Donnenberg and Kaper, 1991). The *E. coli* K-12 MC1000 qseC mutant strain (VS184) was also generated using plasmid pVS132 as described above. Strain VS156 is VS138 complemented with qseC cloned into pVS155. Strains VS179 and VS185 are strains VS138 and VS184, respectively, complemented with qseBC cloned into pVS178.

Construction of operon fusions with lacZ

Operon fusions to *lacZ* were constructed by amplifying the regulatory regions of the genes in strain 86-24 by PCR using Pwo polymerase and cloning into EcoRI and BamHI sites of plasmid pRS551, which contains a promoterless *lac* operon (Simons et al., 1987). The following primer pairs were used to produce specific *lacZ* fusions: primers K2181 and K2182 for qseBC::lacZ (pVS159) and primers K2372 and K2309 for fliA::lacZ (pVS183) (Tables 1 and 2).

β-Galactosidase assays

Bacteria containing the operon::lacZ fusions were grown at 250 r.p.m. for 18 h at 37°C in LB, diluted 1:100 in fresh LB, LB + 0.2% arabinose, DMEM low glucose (Gibco BRL catalogue no. 11054) and DMEM high glucose (Gibco BRL catalogue no. 13000) and grown at 37°C to an OD₆₀₀ of 1.0. These cultures were diluted 1:10 in Z buffer and assayed for β-galactosidase activity using ONPG as a substrate as described previously (Miller, 1972).

Preconditioned media

To prepare preconditioned media containing high levels of AI-2, an initial inoculum of strain 86-24 was grown shaking in DMEM for 18 h at 37°C. This culture was diluted 1:100 in fresh DMEM and grown to an OD₆₀₀ of 0.2, when it was once again diluted 1:100 and grown to an OD₆₀₀ of ≈ 1.0. After centrifugation (12 000 g, 4 min, 25°C) and filtration (0.2 μM filter), the pH of the supernatant was adjusted to 7.0.

Vibrio harveyi luminescence assay

The presence of AI-2 in the preconditioned media was assayed using the *V. harveyi* BB170 (*luxN*::Tn5) reporter strain, which only responds to AI-2 (Surette and Bassler, 1998). The luminescence assays were performed as described previously (Surette and Bassler, 1998), and the assays were read in a Wallac 1420 multilabel counter.

Western blotting

Total proteins were extracted from strains 86-24, VS138 and VS156 grown in LB or DMEM to the same OD₆₀₀. Briefly, 1 ml of culture was pelleted (12 000 g for 5 min at 4°C), resuspended in 400 μl of PBS and 100 μl of 5× sample buffer (20% SDS, 20% glycerol, 200 mM Tris base, pH 6.8, 0.001%

© 2002 Blackwell Science Ltd, Molecular Microbiology, 43, 809–821

bromophenol blue). Protein concentration was measured using the Lowry assay (Lowry *et al.*, 1951). Equal amounts of total proteins were electrophoresed in 12% SDS–PAGE (Wattiau *et al.*, 1996). Secreted proteins were extracted from strains 86-24, VS138 and VS156 grown at 250 r.p.m. in DMEM at 37°C to an OD<sub>600</sub> of 1.0 as described by Jarvis *et al.*, 1995). Western blotting procedures were performed as described previously (Sambrook *et al.*, 1989) and probed with polyclonal antisera directed against the H7 flagellin (J. A. Giron, A. G. Torres, E. Negrete, M. Rosales and J. B. Kaper, unpublished results) and total EPEC secreted proteins (Jarvis *et al.*, 1995).

### Motility assays

Motility assays were performed at 37°C on 0.3% agar plates containing: DMEM, DMEM + 50% DMEM preconditioned by growth of EHEC 86-24 or tryptone media (1% tryptone and 0.25% NaCl). The motility halos were measured at 8, 16 and 48 h.

### Acknowledgements

We would like to thank Dr Andres Campos and Dr Philip Matsumura from the University of Chicago for the MC1000 *flhDC* and *flhD* mutants, and Dr Bonnie Bassler from Princeton University for strain BB170. This work was supported by Public Health Service grants AI41325 and AI21657. A.G.T. was supported by a research supplement for under-represented minorities from the National Institute of Allergy and Infectious Diseases, NIH.

### References

Adler, J., and Templeton, B. (1967) The effect of environmental conditions on the motility of *Escherichia coli*. *J Gen Microbiol* **46**: 175–184.

Bertin, P., Terao, E., Lee, E.H., Lejeune, P., Colson, C., Danchin, A., and Collatz, E. (1994) The H-NS protein is involved in the biogenesis of flagella in *Escherichia coli*. *J Bacteriol* **176**: 5537–5540.

Blattner, F.R., Plunkett, G.I., Bloch, C.A., Perna, N.T., Burland, V., Riley, M., *et al.* (1997) The complete genome sequence of *Escherichia coli* K-12. *Science* **277**: 1453–1462.

Bustamante, V., Santana, F., Calva, E., and Puente, J. (2001) Transcriptional regulation of type III secretion genes in enteropathogenic *Escherichia coli*: Ler antagonizes H-NS-dependent repression. *Mol Microbiol* **39**: 664–678.

Chilcott, G.S., and Hughes, K.T. (2000) Coupling of flagellar gene expression to flagellar assembly in *Salmonella enterica* serovar typhimurium and *Escherichia coli*. *Microbiol Mol Biol Rev* **64**: 694–708.

Das, M., Chopra, A.K., Cantu, J.M., and Peterson, J.W. (1998) Antisera to selected outer membrane proteins of *Vibrio cholerae* protect against challenge with homologous and heterologous strains of *V. cholerae*. *FEMS Immunol Med* **22**: 303–308.

Donnenberg, M.S., and Kaper, J.B. (1991) Construction of an eae deletion mutant of enteropathogenic *Escherichia coli* by using a positive-selection suicide vector. *Infect Immun* **59**: 4310–4317.

Donnenberg, M.S., Yu, J., and Kaper, J.B. (1993) A second chromosomal gene necessary for intimate attachment of enteropathogenic *Escherichia coli* to epithelial cells. *J Bacteriol* **175**: 4670–4680.

Eichelberg, K., and Galan, J.E. (2000) The flagellar sigma factor FliA (sigma 28) regulates the expression of *Salmonella* genes associated with the centisome 63 type III secretion system. *Infect Immun* **68**: 2735–2743.

Elliott, S.J., Sperandio, V., Giron, J.A., Shin, S., Mellies, J.L., Wainwright, L.A., *et al.* (2000) The LEE-encoded regulator (Ler) controls the expression of both LEE and non-LEE encoded virulence factors in enteropathogenic and enterohemorrhagic *Escherichia coli*. *Infect Immun* **68**: 6115–6126.

Friedberg, D., Umanski, T., Fang, Y., and Rosenshine, I. (1999) Hierarchy in the expression of the locus of enterocyte effacement genes of enteropathogenic *Escherichia coli*. *Mol Microbiol* **34**: 941–952.

Goodier, R.I., and Ahmer, B.M. (2001) SirA orthologs affect both motility and virulence. *J Bacteriol* **183**: 2249–2258.

Griffin, P.M., Ostroff, S.M., Tauxe, R.V., Greene, K.D., Wells, J.G., Lewis, J.H., and Blake, P.A. (1988) Illnesses associated with *Escherichia coli* O157:H7 infections. *Ann Intern Med* **109**: 705–712.

Gunn, J.S., and Miller, S.I. (1996) PhoP–PhoQ activates transcription of *pmrAB*, encoding a two-component regulatory system involved in *Salmonella typhimurium* antimicrobial peptide resistance. *J Bacteriol* **178**: 6857–6864.

Guzman, L.M., Belin, D., Carson, M.J., and Beckwith, J. (1995) Tight regulation, modulation, and high-level expression by vectors containing the arabinose PBAD promoter. *J Bacteriol* **177**: 4121–4130.

Jarvis, K.G., Girón, J.A., Jerse, A.E., McDaniel, T.K., Donnenberg, M.S., and Kaper, J.B. (1995) Enteropathogenic *Escherichia coli* contains a specialized secretion system necessary for the export of proteins involved in attaching and effacing lesion formation. *Proc Natl Acad Sci USA* **92**: 7996–8000.

Jerse, A.E., and Kaper, J.B. (1991) The *eae* gene of enteropathogenic *Escherichia coli* encodes a 94-kilodalton membrane protein, the expression of which is influenced by the EAF plasmid. *Infect Immun* **59**: 4302–4309.

Kaper, J.B., Elliott, S., Sperandio, V., Perna, N.T., Mayhew, G.F., and Blattner, F.R. (1998) Attaching and effacing intestinal histopathology and the locus of enterocyte effacement. In *Escherichia coli O157:H7 and Other Shiga Toxin-Producing E. coli Strains*. Kaper, J.B., and O’Brien, A.D. (eds). Washington, DC: American Society for Microbiology Press, pp. 163–182.

Kenny, B., Lai, L.-C., Finlay, B.B., and Donnenberg, M.S. (1996a) EspA, a protein secreted by enteropathogenic *Escherichia coli*, is required to induce signals in epithelial cells. *Mol Microbiol* **20**: 313–324.

Kenny, B., Lai, L.C., Finlay, B.B., and Donnenberg, M.S. (1996b) EspA, a protein secreted by enteropathogenic *Escherichia coli*, is required to induce signals in epithelial cells. *Mol Microbiol* **20**: 313–323.

Kenny, B., DeVinney, R., Stein, M., Reinscheid, D.J., Frey, E.A., and Finlay, B.B. (1997) Enteropathogenic *E. coli* (EPEC) transfers its receptor for intimate adherence into mammalian cells. *Cell* 91: 511–520.

de Kievit, T.R., and Iglewski, B.H. (2000) Bacterial quorum sensing in pathogenic relationships. *Infect Immun* 68: 4839–4849.

Knutton, S., Rosenshine, I., Pallen, M.J., Nisan, L., Neves, B.C., Bain, C., *et al.* (1998) A novel EspA-associated surface organelle of enteropathogenic *Escherichia coli* involved in protein translocation into epithelial cells. *EMBO J* 17: 2166–2176.

Kutsukake, K. (1997) Autogenous and global control of the flagellar master operon, *flhD*, in *Salmonella typhimurium*. *Mol Gen Genet* 254: 440–448.

Kutsukake, K., Ohya, Y., and Iino, T. (1990) Transcriptional analysis of the flagellar regulon of *Salmonella typhimurium*. *J Bacteriol* 172: 741–747.

Lai, L.-C., Wainwright, L.A., Stone, K.D., and Donnenberg, M.S. (1997) A third secreted protein that is encoded by the enteropathogenic *Escherichia coli* pathogenicity island is required for transduction of signals and for attaching and effacing activities in host cells. *Infect Immun* 65: 2211–2217.

Li, C., Louise, C.J., Shi, W., and Adler, J. (1993) Adverse conditions which cause lack of flagella in *Escherichia coli*. *J Bacteriol* 175: 2229–2235.

Lowry, O.H., Rosebrough, N.J., Farr, A.L., and Randall, R.J. (1951) Protein measurement with the Folin phenol reagent. *J Biol Chem* 193: 265–275.

McDaniel, T.K., Jarvis, K.G., Donnenberg, M.S., and Kaper, J.B. (1995) A genetic locus of enterocyte effacement conserved among diverse enterobacterial pathogens. *Proc Natl Acad Sci USA* 92: 1664–1668.

Macnab, R.M. (1996) Flagella and motility. In *Escherichia coli and Salmonella*. Neidhardt, F.C., Curtiss, R.I., Ingraham, J.L., Lin, C.C., Low, K.B., Magasanik, B., *et al.* (eds). Washington, DC: American Society for Microbiology Press, pp. 123–145.

Mellies, J.L., Elliott, S.J., Sperandio, V., Donnenberg, M.S., and Kaper, J.B. (1999) The Per regulon of enteropathogenic *Escherichia coli*: identification of a regulatory cascade and a novel transcriptional activator, the locus of enterocyte effacement (LEE)-encoded regulator (Ler). *Mol Microbiol* 33: 296–306.

Miller, J.H. (1972) *Experiments in Molecular Genetics*. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.

Mizushima, T., Koyanagi, R., Suzuki, E., Tomura, A., Kutsukake, K., Miki, T., and Sekimizu, K. (1995) Control by phosphatidylglycerol of expression of the *flhD* gene in *Escherichia coli*. *Biochim Biophys Acta* 1245: 397–401.

Mizushima, T., Koyanagi, R., Katayama, T., Miki, T., and Sekimizu, K. (1997) Decrease in expression of the master operon of flagellin synthesis in a *dnaA46* mutant of *Escherichia coli*. *Biol Pharm Bull* 20: 327–331.

Morgan, D.G., Owen, C., Melanson, L.A., and DeRosier, D.J. (1995) Structure of bacterial flagellar filaments at 11 Å resolution: packing of the alpha-helices. *J Mol Biol* 249: 88–110.

Nataro, J.P., and Kaper, J.B. (1998) Diarrheagenic *Escherichia coli*. *Clin Microbiol Rev* 11: 142–201.

Nishimura, A., and Hirota, Y. (1989) A cell division regulatory mechanism controls the flagellar regulon in *Escherichia coli*. *Mol Gen Genet* 216: 340–346.

Prüß, B.M., and Matsumura, P. (1996) A regulator of the flagellar regulon of *Escherichia coli*, *flhD*, also affects cell division. *J Bacteriol* 178: 668–674.

Qualding, C., and Stocker, B.A.D. (1962) An environmentally-induced transition from the flagellated to the non-flagellated state in *Salmonella typhimurium*: the fate of parental flagella at cell division. *J Gen Microbiol* 28: 257–270.

Sambrook, J., Fritsch, E.F., and Maniatis, T. (1989) *Molecular Cloning, a Laboratory Manual*. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.

Schauder, S., and Bassler, B.L. (2001) The languages of bacteria. *Genes Dev* 15: 1468–1480.

Schauder, S., Shokat, K., Surette, M.G., and Bassler, B.L. (2001) The LuxS family of bacterial autoinducers: biosynthesis of a novel quorum-sensing signal molecule. *Mol Microbiol* 41: 463–476.

Shi, W., Zhou, Y., Wild, J., Adler, J., and Gross, C.A. (1992) DnaK, DnaJ, and GrpE are required for flagellum synthesis in *Escherichia coli*. *J Bacteriol* 174: 6256–6263.

Shi, W., Li, C., Louise, C.J., and Adler, J. (1993) Mechanism of adverse conditions causing lack of flagella in *Escherichia coli*. *J Bacteriol* 175: 2236–2240.

Silhavy, T.J., Berman, M.L., and Enquist, L.W. (1984) *Experiments with Gene Fusions*. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.

Silverman, M., and Simon, M. (1974) Characterization of *Escherichia coli* flagellar mutants that are insensitive to catabolite repression. *J Bacteriol* 120: 1196–1203.

Simons, R.W., Houman, F., and Kleckner, N. (1987) Improved single and multicopy lac-based cloning vectors for protein and operon fusions. *Gene* 53: 85–96.

Soncini, F.C., and Groisman, E.A. (1996) Two-component regulatory systems can interact to process multiple environmental signals. *J Bacteriol* 178: 6796–6801.

Sperandio, V., Mellies, J.L., Nguyen, W., Shin, S., and Kaper, J.B. (1999) Quorum sensing controls expression of the type III secretion gene transcription and protein secretion in enterohemorrhagic and enteropathogenic *Escherichia coli*. *Proc Natl Acad Sci USA* 96: 15196–15201.

Sperandio, V., Mellies, J.L., Delahay, R.M., Frankel, G., Crawford, J.A., Nguyen, W., and Kaper, J.B. (2000) Activation of enteropathogenic *Escherichia coli* (EPEC) LEE2 and LEE3 operons by Ler. *Mol Microbiol* 38: 781–793.

Sperandio, V., Li, C.C., and Kaper, J.B. (2002) Quorum-sensing *Escherichia coli* regulator A (QseA): a regulator of the LysR family involved in the regulation of the LEE pathogenicity island in enterohemorrhagic *Escherichia coli*. *Infect Immun*, in press.

Sperandio, V., Torres, A.G., Giron, J.A., and Kaper, J.B. (2001) Quorum sensing is a global regulatory mechanism in enterohemorrhagic *Escherichia coli* (EHEC) O157:H7. *J Bacteriol* 183: 5187–5197.

Surette, M.G., and Bassler, B.L. (1998) Quorum sensing in *Escherichia coli* and *Salmonella typhimurium*. *Proc Natl Acad Sci USA* 95: 7046–7050.

Wattiau, P., Woestyn, S., and Cornelis, G. (1996) Customized secretion chaperones in pathogenic bacteria. *Mol Microbiol* 20: 255–262.

Wosten, M.M., Kox, L.F., Chamnongpol, S., Soncini, F.C., and Groisman, E.A. (2000) A signal transduction system that responds to extracellular iron. *Cell* 103: 113–125.

Yanagihara, S., Iyoda, S., Ohnishi, K., Iino, T., and Kutsukake, K. (1999) Structure and transcriptional control

QseBC regulates flagella and motility in E. coli 821

of the flagellar master operon of *Salmonella typhimurium*. *Genes Genet Syst* 74: 105–111.

Yokota, T., and Gots, J.S. (1970) Requirement of adenosine 3′, 5′-cyclic phosphate for flagella formation in *Escherichia coli* and *Salmonella typhimurium*. *J Bacteriol* 103:513–516.

© 2002 Blackwell Science Ltd, *Molecular Microbiology*, 43, 809–821
