Molecular Microbiology (2002) 43(3), 703–715

# An ABC-type, high-affinity urea permease identified in cyanobacteria

**Ana Valladares, María Luz Montesinos, Antonia Herrero and Enrique Flores**

Instituto de Bioquímica Vegetal y Fotosíntesis, Consejo Superior de Investigaciones Científicas, Universidad de Sevilla, Avda Américo Vespucio s/n, E-41092 Sevilla, Spain.

## Summary

Urea is an important nitrogen source for many microorganisms, but urea active transporters have not been characterized at a molecular level in any bacterium. Cells of *Synechocystis* sp. PCC 6803 and *Anabaena* sp. PCC 7120 exhibited the capacity to take up \[^{14}C\]-urea from low-concentration (<1 μM) urea solutions. The \(K_s\) of *Anabaena* cells for urea was about 0.11 μM, and the observed uptake activity involved the transport and metabolism of urea. In contrast to urease, which was constitutively expressed, expression of the high-affinity urea uptake activity was subjected to nitrogen control. In an *Anabaena ureG* (urease⁻) mutant, a concentrative, active transport of urea could be demonstrated. We found that a mutant of open reading frame (ORF) *slr0374* from the *Synechocystis* genomic sequence lacked urea transport activity. This ORF encoded a conserved component of an ABC-type transporter, but it is not clustered together with any other possible transporter-encoding gene. An *Anabaena* homologue of *slr0374*, *urtE*, was isolated and found to be part of a cluster of genes, *urtABCDE*, putatively encoding all the elements of an ABC-type permease. Although the longest transcript that we could detect only covered *urtABC*, the impairment of urea transport by inactivation of *urtA*, *urtB* or *urtE* suggested that the whole gene cluster is expressed producing the urea permease. Expression was induced under nitrogen-limiting conditions, and a complex promoter regulated by the cyanobacterial global nitrogen control transcription factor NtcA was found upstream from *urtA*. Our work adds urea to the known substrates of the versatile class of ABC-type transporters and suggests the involvement of a transporter of this superfamily in urea scavenging by some bacteria in natural environments.

Accepted 26 October, 2001. *For correspondence. E-mail flores@ cica.es; Tel. (+34) 954 48 9523; Fax (+34) 954 46 0065.*

© 2002 Blackwell Science Ltd

## Introduction

Urea metabolism has been widely investigated, as the urea cycle is central to nitrogen metabolism in animals. Additionally, both higher plant (jack bean) and microbial ureases have been intensively characterized. In contrast, little is known about mechanisms of urea transport across biological membranes, probably reflecting the fact that diffusion of urea across lipid bilayers takes place readily, making it difficult to investigate urea transport. Nonetheless, urea channels have recently been characterized. The bacterium *Helicobacter pylori* expresses a urea-specific, H⁺-gated channel, the product of the *ureI* gene (Weeks et al., 2000), and channel proteins of the major intrinsic protein (MIP) family, which exhibit specificity for small neutral molecules including water, glycerol and urea, are ubiquitous (Saier, 1999). Flux through these channels, perhaps along with diffusion, may represent a method of urea translocation across biological membranes when this compound is present at relatively high concentrations (e.g. >1mM) inside or outside the cells.

Using urea as a nutrient may pose a different problem as, in this case, urea has to be taken up from the environment where it may be found at low concentrations (e.g. in the micromolar range). Urea can serve as a nitrogen source for many microorganisms including some bacteria, yeast, fungi and algae. Although some yeasts and algae degrade urea by means of the ATP- and biotin-dependent enzyme urea amidolyase, most microorganisms use the Ni²⁺-dependent urea amidohydrolase or urease, an intracellular enzyme that catalyses the hydrolysis of urea, finally rendering CO₂ and two molecules of ammonia, and exhibits a \(K_m\) for urea generally above 1mM (Mobley and Hausinger, 1989). Transporters with a relatively high affinity for urea (\(K_s\) in the range 10–40 μM) have been physiologically characterized in yeast (Cooper and Sumrada, 1975), fungi (Pateman et al., 1982), algae (McCarthy, 1972; Syrett and Bekheet, 1977) and a few bacteria (Healey, 1977; Jahns et al., 1988; Siewe et al., 1998). A common feature of the urea uptake activities reported for these organisms is that they are derepressed under nitrogen limitation. Only the urea active transporter of *Saccharomyces cerevisiae* has been characterized to date at the molecular level (ElBerry et al., 1993). This transporter is the product of the *DUR3* gene and consists of a polypeptide that exhibits 14 putative
704 A. Valladares, M. L. Montesinos, A. Herrero and E. Flores

transmembrane domains. In this report, we present the first molecular characterization of a bacterial urea active transporter, an ABC-type permease that we have identified in cyanobacteria.

The cyanobacteria are phototrophic eubacteria that perform oxygenic photosynthesis and fix CO₂ through the Calvin cycle. The nitrogen sources most commonly used by these organisms are ammonium, nitrate and urea and, in some species, dinitrogen. Although the cyanobacterial assimilatory nitrogen metabolism has been widely investigated (Flores and Herrero, 1994), information on urea assimilation is scarce and mostly related to urease (Ge et al., 1989; Argall et al., 1992; Jahns et al., 1995; Collier et al., 1999; Palinska et al., 2000). In cyanobacteria, expression of genes encoding the assimilatory systems for sources of nitrogen alternative to ammonium, such as nitrate or N₂, is subjected to positive regulation mediated by NtcA, a transcription factor of the CAP family (Herrero et al., 2001). In the absence of ammonium, NtcA activates transcription from promoters that bear a –10 box (TAN₃T) and an NtcA binding site (GTAN₈TAC), which is most frequently centred at –41.5 with regard to the transcription start point (Luque et al., 1994).

---

## Results

### Assimilation of urea

Cells of *Synechocystis* sp. PCC 6803, a non-nitrogen-fixing unicellular cyanobacterium, and of *Anabaena* sp. EF116, a Fox⁻ (non-nitrogen-fixing under aerobic conditions) mutant of the heterocyst-forming cyanobacterium *Anabaena* sp. PCC 7120, grew on urea at rates (µ, 0.84 and 0.82 day⁻¹ respectively) close to those observed with nitrate (1.03 and 1.01 day⁻¹ respectively) as the nitrogen source. However, no urea-dependent growth was detected with the non-nitrogen-fixing unicellular cyanobacterium *Synechococcus* sp. PCC 7942. Whereas the latter strain showed no detectable urease activity (see also Berns et al., 1966), strains PCC 6803 and PCC 7120 exhibited urease activities, determined in permeabilized cells, of about 3.6 and 1.6 U mg⁻¹ chlorophyll a (Chl) respectively. No significant differences in urease activity were observed when ammonium-grown *Ana-baena* or *Synechocystis* cells were incubated for 24 h with ammonium, urea, nitrate or no source of combined nitrogen. Constitutive urease expression was also observed for two other cyanobacteria that were tested, *Calothrix* sp. PCC 7601 and *Pseudanabaena* sp. PCC 6903 (not shown). Consistent with a lack of nitrogen control, urease activities in strain CSE2, an *ntcA* mutant of *Anabaena* sp. PCC 7120 (Frías et al., 1994), were similar to those observed with the wild type for the different nitrogen sources. The *K*ₘ of urease for urea, determined in permeabilized

---

Table 1. High-affinity [¹⁴C]-urea uptake in *Anabaena* sp.

| Nitrogen source | Strain PCC 7120 | Strain CSE2 (*ntcA*) |
|------------------|-----------------|-----------------------|
| Urea            | 1.24            | 0.16                  |
| NH₄⁺            | 1.30            | 0.23                  |
| NO₃⁻            | 2.96            | 0.66                  |
| –N              | 4.96            | 0.97                  |

Ammonium-grown cells were incubated for 10 h in BG11₀ medium with 17.6 mM NaNO₃, 2 mM NH₄Cl, 2 mM urea or no source of combined nitrogen. The cells were then harvested, washed and used in 10 min uptake assays with 1 µM [¹⁴C]-urea as described in Experimental procedures. Data are the mean of three independent determinations with similar results (standard deviation about 22% of the mean for the data for strain PCC 7120 and about 33% of the mean for the data for strain CSE2).

*Anabaena* cells, was 2.5 mM, and the urease activity was inhibited by more than 95% by a 40 min incubation of the *Anabaena* cells with 100 µM acetohydroxamic acid, a nickel chelator that has been shown to inhibit urease (Mobley and Hausinger, 1989).

#### Urea uptake

Cells of *Synechocystis* sp. PCC 6803 and *Anabaena* sp. PCC 7120 took up [¹⁴C]-urea efficiently when this substrate was supplied at a low concentration (≤1 µM). In contrast to urease, uptake activity at a low (1 µM) concentration of [¹⁴C]-urea was subjected to nitrogen control. Activity was highest in cells incubated in the absence of a source of combined nitrogen and lowest in cells incubated with ammonium or urea as a nitrogen source, and it was about fivefold lower in the *ntcA* mutant strain CSE2 (Table 1). The rate of [¹⁴C]-urea uptake was determined for urea concentrations of 0.05–1 µM in *Anabaena* cells that had been induced by incubation for 10 h with no source of combined nitrogen. Uptake was linear for 3–10 min, depending on the concentration of substrate, and a *K*ₛ value of 0.11 µM and a *V*ₘₐₓ of about 4–5 nmol min⁻¹ mg⁻¹ Chl were obtained. Detection of a regulated high-affinity uptake activity and of a high-affinity kinetic component suggested the presence of a urea permease in these cells.

In *Anabaena* cells treated for 1 h with 150 µM acetohydroxamic acid, a *K*ₛ (urea) similar to that obtained with untreated cells was found, but *V*ₘₐₓ was about 50%, suggesting that, in addition to transport, urea uptake involved metabolism through urease. Consistent with this observation, the radioactivity in the soluble fraction of extracts of *Anabaena* cells that had been incubated for 15 min with 1.1 µM [¹⁴C]-urea, analysed by thin-layer chromatography (TLC), was distributed among several radioactive spots (not shown).

© 2002 Blackwell Science Ltd, *Molecular Microbiology*, **43**, 703–715

When urea concentrations of 10 and 50 μM, which are above the saturation level for the high-affinity permease (i.e. above about 5 μM), were tested, no further increase in the rate of uptake was observed for acetohydroxamic acid-treated cells. However, higher than expected rates of uptake were observed for untreated cells. These urease-dependent increased uptake levels might correspond to diffusion of urea, but we did not pursue investigation of this phenomenon. Because some metabolism of urea was still observed in the acetohydroxamic acid-treated cells (not shown), we sought to isolate an Anabaena urease mutant to facilitate study of the high-affinity urea transporter.

**Urea transport in a ureG mutant**

[^14C]-Urea uptake experiments (2 h incubation, 1 μM [^14C]-urea as substrate) carried out with a collection of Tn-1063a-induced Anabaena mutants (see Experimental procedures) identified a clone, strain CSAV56, which exhibited <2% of the wild-type urea uptake activity, lacked urease activity and carried the transposon inserted into a gene encoding a protein homologous to UreG, an accessory protein required for the production of an active urease (Mobley *et al.*, 1995). To corroborate that the impairment in [^14C]-urea uptake resulted from inactivation of *ureG*, the Anabaena sp. PCC 7120 genomic region carrying this gene was cloned and mutated. A plasmid, pCSAV3, bearing a 4.6 kb insert of Anabaena DNA that carried the *ureC*, *ureE*, *ureF* and *ureG* genes was isolated; gene cassette C.S3 was introduced into pCSAV3 substituting for part of the *ureG* gene, and the resulting construct was used to generate an Anabaena mutant, strain CSAV1, which was homozygous for the mutant *ureG*:C.S3 chromosomes (see Experimental procedures). Strain CSAV1 lacked any detectable urease activity and was impaired in the uptake of 0.1 μM [^14C]-urea (2.8% of wild-type activity). These results confirmed that the *ureG* mutation causes an impairment in [^14C]-urea uptake.

The time course of uptake of 0.1 μM [^14C]-urea was investigated in strain CSAV56 (*ureG*:Tn1063a) and compared with that of the wild type (Fig. 1). Whereas uptake in the wild type proceeded for at least 45 min, the capacity for uptake in the mutant was saturated after 1 min. (In order to observe retention of radioactivity from [^14C]-urea in the mutant, the step of washing the cells on the filter after the uptake assays was avoided in these experiments.) We suggest that, whereas urea metabolism permits a high level of urea uptake in the wild type, in the mutant, net [^14C]-urea uptake will proceed only until the urea accumulated inside the cells reaches the highest level permitted by the capability of concentration of the transport system.

![PCC 7120](https://example.com/image1.png)
![CSAV56](https://example.com/image2.png)

Fig. 1. Time course of [^14C]-urea uptake in Anabaena sp. strains PCC 7120 and CSAV56 (*ureG*:Tn1063a). Nitrate-grown cells were incubated in BG11<sub>o</sub> medium for 10 h, harvested, washed and used in assays of uptake of 0.1 μM [^14C]-urea as described in Experimental procedures. The cell suspensions contained 1 μg · ml<sup>-1</sup> Chl, and the cells were not washed after the filtration step. Note the different scales for the two graphs.

To characterize this high-affinity [^14C]-urea transport activity further, we tested the effect of the F<sub>o</sub>F<sub>1</sub>ATPase inhibitor DCCD on the transport activity and analysed the fate of the [^14C]-urea taken up. In these experiments, nitrate-grown cells incubated for 10 h in BG11<sub>o</sub> medium (which lacks any source of combined nitrogen) were used, the washing of the cells after the filtration step was avoided, and retention of radioactivity by boiled cells was used as a blank. Uptake of [^14C]-urea by Synechococcus sp. PCC 7942, a strain unable to assimilate urea (see above), was also tested as a negative control. Using 1 μM [^14C]-urea as a substrate in 4 min uptake assays, no transport activity was detected for strain PCC 7942, whereas activities of 1.52 and 17.5 nmol mg<sup>-1</sup> Chl were observed for strains CSAV56 and PCC 7120 respectively. Incubation of cells of strain CSAV56 for 10 min with 50 μM DCCD led to a 97% inhibition of urea transport (tested with 1.1 μM [^14C]-urea for 4 min). The fate of [^14C]-urea taken up by strain CSAV56 was analysed by TLC. Only one spot that could be identified as [^14C]-urea was detected in the soluble fraction of cells that had been incubated with 1.1 μM [^14C]-urea for 15 min (not shown). The ratio of intracellular to extracellular concentration of [^14C]-urea at the end of the incubation was close to 20. These results demonstrated that Anabaena sp. PCC 7120 and its derivative strain CSAV56 bear a concentrative, active, high-affinity urea transporter that is not present in Synechococcus sp. PCC 7942.

**Cloning of the urt genes**

We have reported previously the inactivation of some open reading frames (ORFs) from the genome of Synechocystis sp. PCC 6803 whose putative products are

© 2002 Blackwell Science Ltd, *Molecular Microbiology*, **43**, 703–715

706 A. Valladares, M. L. Montesinos, A. Herrero and E. Flores

(25% identity), UrtD to the ATP-binding subunit BraF (31% identity) and UrtE to the ATP-binding subunit BraG (38% identity). The *P. aeruginosa bra* genes constitute an operon with a structure similar to that of the *Anabaena urt* gene cluster. With regard to *Synechocystis* sp. PCC 6803, the product of *sll0374*, the ORF that was used as a probe to identify the *Anabaena urt* gene cluster, would be homologous to UrtE (72% identity), the product of *slr0447* to UrtA (65% identity), that of *slr1200* to UrtB (62% identity), that of *slr1201* to UrtC (59%) and that of *sll0764* to UrtD (61% identity). Of these ORFs, only *slr1200* and *slr1201* are clustered together in the *Synechocystis* genome. Gene clusters that would encode proteins homologous to the Urt proteins are also present in the genomes of the heterocyst-forming cyanobacterium *Nostoc punctiforme* and the unicellular marine cyanobacteria *Synechococcus* sp. WH8102 and *Prochlorococcus* sp. strains MED4 and MIT9313 (http://www.jgi.doe.gov/). Interestingly, in these marine strains, the putative *urt* genes are clustered together with urease-encoding genes.

**Fig. 2.** The *urt* gene cluster of *Anabaena* sp. PCC 7120 (nucleotide sequence accession number AJ271599). The parts of the genomic region covered by the inserts of plasmids pCSAV8, pCSAV90 and pCSAV98 are indicated. The location of the *urtA/urtB* probe used to identify a clone carrying pCSAV98, the restriction sites used to insert the C.S3 gene cassette and the place where a *HindIII* site was created in *urtE* to insert the C.K3 gene cassette are also indicated. The *urtC* and *urtD* genes show a 26 bp overlap.

homologous to elements of amino acid transporters from other biological sources (Montesinos *et al.*, 1997). Inactivation was achieved by insertion of the C.K3 gene cassette (encoding resistance to kanamycin-neomycin) and, whereas some of the generated *Synechocystis* mutants were indeed impaired in amino acid transport, some others were not (Montesinos *et al.*, 1997). We checked the latter mutants for transport of other possible substrates and found that a *Synechocystis* strain mutated in ORF *sll0374* exhibited <2% of the wild-type urea transport activity (tested with 1.6 μM \[^{14}C\]-urea in 10 min assays). This ORF would encode a conserved component of an ABC-type transporter, but it is not clustered together with other possible transporter-encoding ORFs in the *Synechocystis* genome (Kaneko *et al.*, 1996).

The *Anabaena* sp. PCC 7120 genomic region carrying a homologue of *Synechocystis sll0374* was identified by hybridization, cloned and sequenced. The insert of plasmid pCSAV8 carried a whole ORF homologous to *sll0374* and part of an ORF that would encode another ABC-type transporter conserved component (Fig. 2). Plasmids pCSAV90 and pCSAV98 were then isolated as described in Experimental procedures, and their inserts were sequenced and found to cover a region of the *Anabaena* genome including five ORFs that would encode the elements of an ABC-type permease. Because they are involved in urea transport (see below), we have named these ORFs as *urt* genes (Fig. 2; nucleotide sequence accession number AJ271599). The proteins of known function that show highest similarities to the *Anabaena urt* gene products are the products of the *Pseudomonas aeruginosa bra* genes, which encode an ABC-type permease for branched-chain amino acids (Hoshino and Kose, 1989; 1990). UrtA is homologous to the periplasmic substrate-binding protein BraC (25% identity), UrtB to the transmembrane protein BraD (26% identity), UrtC to the transmembrane protein BraE

**urt and ureG urtE mutants**

To test the involvement of the *urt* gene cluster in urea transport in *Anabaena* sp. PCC 7120, some of the *urt* genes were inactivated. The C.S3 gene cassette, encoding resistance to Sm and Sp and carrying transcriptional terminators, was inserted into *urtA* (with the concomitant deletion of a 241 bp fragment) and *urtB*, and the C.K3 gene cassette, encoding resistance to Km and Nm, was inserted into *urtE*. The resulting constructs were transferred to *Anabaena* sp. PCC 7120, and clones homozygous for *urtA::C.S3*, *urtB::C.S3* and *urtE::C.K3* mutant chromosomes were obtained and named strain CSAV4, CSAV5 and CSAV2 respectively. The three mutant strains showed very low \[^{14}C\]-urea uptake activities, confirming that the *urt* genes encode a high-affinity urea transporter (Table 2).

**Table 2.** Uptake of \[^{14}C\]-urea in *Anabaena urtA*, *urtB*, *urtE* and *ureG urtE* mutants.

| Strain | Genotype       | \[^{14}C\]-Urea uptake (nmol min\(^{-1}\) mg\(^{-1}\) Chl) |
|--------|----------------|-------------------------------------------------------|
| PCC 7120 | Wild type     | 1.96                                                |
| CSAV4   | *urtA*         | 0.06                                                |
| CSAV5   | *urtB*         | 0.05                                                |
| CSAV2   | *urtE*         | 0.05                                                |
| CSAV3   | *ureG*, *urtE* | <0.01                                               |

Nitrate-grown cells were incubated for 10 h in BG11\(_o\) medium and used in 7 min uptake assays with 0.1 μM \[^{14}C\]-urea. The cells were not washed after the filtration step. Data are the mean of three independent experiments with similar results (standard deviation 13% of the mean for strain PCC 7120 and about 33% of the mean for the mutants).

© 2002 Blackwell Science Ltd, *Molecular Microbiology*, **43**, 703–715

However, some radioactivity from \[^{14}C\]-urea was still incorporated by the *urt* mutants. To test whether this incorporation was the result of some uptake, perhaps unspecific, pulled by urease, we generated a *urt ure* double mutant. Plasmid pCSAV12 (*ureG::C.S3*; see above) was conjugated to strain CSAV2 (*urtE::C.K3*), and a clone homozygous for the chromosomes carrying the second mutation (*ureG::C.S3*) was obtained and named strain CSAV3. This strain showed negligible urea uptake activity (Table 2), suggesting that the identified *urt* and *ure* genes are the only genes specifically involved in urea assimilation in *Anabaena* sp. PCC 7120.

### Expression of *urt* genes

The expression of the *urt* genes was investigated by Northern analysis using probes of each of the *urt* genes and RNA isolated from *Anabaena* cells grown with ammonium or grown with ammonium and incubated for 3 or 6 h with no combined nitrogen, nitrate or urea. Hybridization was observed only with the *urtA* probe (Fig. 3), and the size of the hybridizing band, 1.6 kb, could correspond to a transcript covering the *urtA* gene (1305 bp). However, the effect of the *urtB* and *urtE* mutations (see Table 2) suggests that the whole gene cluster is expressed. We therefore tried to identify a longer transcript by reverse transcriptase-polymerase chain reaction (RT-PCR). Retrotranscription was performed with oligonucleotides corresponding to the *urtC* (URT6 primer), *urtD* (URT2 primer) and *urtE* (URT28 primer) genes, and PCR of the generated cDNAs was carried out with an oligonucleotide corresponding to *urtA* (URT15, forward primer) and the same primers used for retrotranscription (see Fig. 4). We only observed amplification of the retrotranscription product generated with the *urtC* primer, for which an amplified fragment of 1.7 kb was expected (Fig. 4). Amplification was strictly dependent on the presence of RNA in the reaction (see Fig. 4, sample 2) and therefore could not be attributed to any contaminating DNA. The results indicate co-transcription of at least *urtA*, *urtB* and *urtC*.

The results in Fig. 3A also show that the expression of *urtA* is subjected to nitrogen control. Quantification of the hybridization signals indicated a five- and twofold induction by incubation for 3 h with no combined nitrogen and nitrate respectively. To test whether this induction was dependent on the NtcA transcriptional regulator, Northern analysis was carried out with RNA isolated from an *Anabaena ntcA* mutant, strain CSE2. Induction was impaired in the mutant, which showed only a basal level of *urtA* expression (Fig. 3B).

### Promoter structure

The putative transcription start point (tsp) of the *urt* operon

© 2002 Blackwell Science Ltd, *Molecular Microbiology*, **43**, 703–715

---

**Fig. 3.** Northern analysis of the expression of the *urtA* gene.
A. RNA was isolated from cells of *Anabaena* sp. PCC 7120 grown with ammonium (\(\text{NH}_4^+\)) or grown with ammonium and incubated for 3 or 6 h with no source of combined nitrogen (\(-\text{N}\)), nitrate (\(\text{NO}_3^-\)) or urea.
B. RNA was isolated from cells of *Anabaena* sp. PCC 7120 and the *ntcA* mutant, strain CSE2, grown with ammonium (0) or grown with ammonium and incubated for 3 or 6 h in a medium lacking combined nitrogen (3, 6). The *urtA* probe was generated by PCR using pCSAV98 as a template and oligonucleotides URT11 and URT18 as primers. The size of the hybridization band is indicated in kb. The filter used in panel B was also hybridized with a probe of the RNase P RNA gene, *rnpB*, as a loading and transfer control.

was investigated by primer extension using primers URT17 and URT22, which are located close to the 5′ end of *urtA* (see Fig. 5A). Similar results were obtained with the two primers, and those obtained with primer URT17

708 A. Valladares, M. L. Montesinos, A. Herrero and E. Flores

Discussion

Some cyanobacteria, such as *Synechocystis* sp. PCC 6803 and *Anabaena* sp. PCC 7120, are capable of taking up urea even when it is present at a low concentration (e.g. <1 μM) in the extracellular medium. We have shown in this work that an ABC-type permease, encoded by the *urt* genes, is essential for this uptake activity. The *urt* genes are clustered together in the *Anabaena* genome and exhibit the structure *urtABCDE*. Sequence homologies suggest that *urtA* encodes the periplasmic-binding protein, *urtB* and *urtC* the transmembrane proteins and *urtD* and *urtE* the ATP-binding subunits. This gene arrangement is similar to that commonly found for ABC-type permeases that are encoded by operons (Boos and Lucht, 1996). Although we have observed an abundant transcript only for *urtA* and have been able to detect by RT–PCR a transcript that covers only *urtABC*, the physiological effect of inactivation of *urtE* indicates that the last genes of the gene cluster are also expressed. Expression of this gene cluster appears therefore to decrease 5′ to 3′, probably rendering higher levels of the substrate-binding protein than of the other permease subunits. This is a common feature for ABC-type uptake permeases (Boos and Lucht, 1996).

Inactivation of *urtA*, *urtB* or *urtE* in *Anabaena* sp. PCC 7120 leads to a 97–98% reduction in the activity of uptake observed in the wild type for a urea concentration of 0.1 μM. The activity remaining in the *urt* mutants is similar to that observed for DCCD-treated *Anabaena* cells, suggesting that it corresponds to passive uptake that could take place by either diffusion of urea through the membrane or flux through a channel. Lack of any detectable uptake in an *ureG urtE* double mutant (Table 2) indicates that the uptake remaining in the *urt* single mutants is a consequence of metabolism of urea via urease. The radioactivity detected inside the cells would thus correspond to products of the fixation of the ^14^C-labelled CO₂ released in the urease reaction. On the other hand, in Urt⁺ *ureG* cells, an intracellular accumulation of urea was observed, consistent with the operation of an active transport system. The fact that the accumulation ratio observed for urea (up to about 20) was relatively low is probably a consequence of the easy diffusion of urea through bacterial membranes (Siewe *et al.*, 1998). Thus, a high level of urea uptake is dependent on urea metabolism (Fig. 1).

The ABC-type transporters commonly exhibit a higher affinity for their substrates than many other types of transporters. Consistently, the Urt system of *Anabaena* sp. PCC 7120 exhibits a lower *K*ₛ for urea (about 0.1 μM) than the urea permease of *Saccharomyces cerevisiae* (*K*ₛ, 14 μM) (Cooper and Sumrada, 1975; ElBerry *et al.*, 1993) or the urea carrier of *Corynebacterium glutamicum* (*K*ₛ,

© 2002 Blackwell Science Ltd, *Molecular Microbiology*, **43**, 703–715

A

URT19

TCTTGCGGAT GCTAGTTTAG ATACCTGTTA GGTCAACCCT TTAATATCTG CACAGCTATA AATATCACTC

TGAATTTTAG TTAATTAGAT TAATTTTTAT TTAAAGGAAT TAGAATTTAG TATCAAAAAT AACAAATCAA

-10 tsp2 tsp1 NtcA-binding site

TGGTTAAATA TC AAACTAAT ATCAAAATCT AAAGCTTGAC CTGTGAAACT CTCAGGGAGC TAGCTGATTC

-10 box

ATCAAGGGGA AATTTAACCA AGATATGAGT AGACGAATTA ACCGACGCAG ATTTTTAATT TACGGCTCTG

CAACTATTG GTAGCAGCAT TTTGCTGAAG GCTTGCGCTA ATAGCACCCA

URT22 URT17

B

PCC 7120 CSE2

T C G A 0 3 6 0 3 6

Fig. 5. Analysis of the *urtA* promoter.

A. Nucleotide sequence of the 5′ end and sequences upstream of *urtA*. The location of primers URT17, URT19 and URT22 as well as the possible tps and promoter elements for tsp1 (−10 box and NtcA binding site, underlined) and for tsp2 (−10 and −35 hexamers, double overlined) are indicated.

B. Primer extension using the URT17 primer and RNA isolated from Anabaena sp. PCC 7120 or strain CSE2 (*ntcA*) cells grown with ammonium (0) or grown with ammonium and incubated for 3 or 6 h without combined nitrogen.

C. Binding of NtcA to a 330 bp DNA fragment, containing the promoter region of *urtA*, generated by PCR using oligonucleotides URT17 and URT19 as primers and plasmid pCSAV98 as template. The 32P-labelled DNA fragment (lane 1) was incubated with histidine-tagged Anabaena NtcA protein (lanes 2–4). A 25-fold excess of the same unlabelled DNA fragment (lane 3) or of an unlabelled fragment carrying the P1 promoter of the Anabaena *glnA* gene (Frías et al., 1994; lane 4) was also added.

RNA₂
RNA₁

C

1 2 3 4

NtcA + Pᵤᵣₜ

Pᵤᵣₜ

9 μM) that has been suggested to be a proton motive force-dependent secondary transport system (Siewe et al., 1998). A binding protein with high affinity for urea and short-chain amides that may be a component of an

ABC-type transporter has also been characterized in Methylophilus methylotrophus (Mills et al., 1998). The gene encoding this binding protein is clustered together with genes that would encode proteins homologous to

© 2002 Blackwell Science Ltd, Molecular Microbiology, 43, 703–715

the BraD and BraE transmembrane proteins, respectively, but none of these genes has been mutated and, therefore, their physiological function has not been established. The presence of transporters with a high affinity for urea suggests that some cyanobacteria can take up the urea that may be found at very low concentrations in their natural environments. Recent determinations in some natural water bodies have indicated the presence of urea at 0.1–3 μM concentrations (Cho *et al.*, 1996; Mitamura *et al.*, 2000a,b), and it has been suggested that cyanobacterial phytoplankton use urea as a nitrogen source in a lake in which this substrate is found at 0.1–0.6 μM (Mitamura *et al.*, 2000a).

Urease expression is subjected to nitrogen control in some bacteria (Bender, 1991; Collins *et al.*, 1993; Goss and Bender, 1995), but it is constitutive in the cyanobacteria investigated in this work. In contrast, an induction of about two- and fivefold is observed by incubation of the Anabaena cells in medium with nitrate or lacking combined nitrogen, respectively, for both urea transport activity and *urtA* expression (Table 1 and Fig. 3). In an Anabaena *ntcA* mutant, transport activity was about fivefold lower and *urtA* expression was impaired, indicating that NtcA is involved in this regulation. NtcA is known to be capable of acting as a transcriptional activator or a transcriptional repressor in different promoters, although there have been more reports of NtcA-dependent activation (Herrero *et al.*, 2001). For *urtA*, NtcA appears to activate promoter P₁, which has a structure (GTAN₈AACN₂₃TAN₃T; see Fig. 5A) very close to that of the canonical NtcA-activated promoter, and to repress promoter P₂, which has a structure (GTATCAN₁₈TTAAAT) similar to the σ⁷⁰ canonical promoters and with which the NtcA binding site overlaps (see Fig. 5A). When activated, P₁ appears to be stronger than the derepressed P₂ (Fig. 5B), thus explaining the increased expression of *urtA* under N-limiting conditions. On the other hand, a nitrogen-dependent regulation of urea uptake is evident in the *ntcA* mutant (Table 1), but the molecular mechanism for this NtcA-independent regulation is currently unknown.

Nitrogen regulation of the Urt permease is consistent with the use of urea as a nutrient and would ensure expression of the *urt* gene cluster when ammonium or urea itself is not available at high concentrations. In contrast, constitutive expression of urease suggests a role for this enzyme in nitrogen metabolism independently of the presence of urea in the extracellular medium. The cyanobacterial strains for which we have described a constitutive urease can accumulate cyanophycin, a copolymer of arginine and aspartate unique to some cyanobacteria (Allen, 1984), and we have found recently that arginine catabolism involves the urea cycle and arginase pathways (Quintero *et al.*, 2000). It is therefore possible that the constitutive expression of urease

is related to a role in arginine catabolism and cyanophycin mobilization in cyanobacteria that synthesize cyanophycin.

As a group, the ABC-type transporters are known to translocate quite diverse substrates bidirectionally across the cytoplasmic membrane, from simple inorganic ions to proteins (Saier, 1998). The bacterial ABC uptake systems (TC no. 3.A.1 in the transporter classification of Saier, 2000) constitute a phylogenetically well-defined group of permeases within the ABC superfamily, but different phylogenetic clusters are observed within these ABC transporters (Saier, 1998). Permeases in each cluster transport chemically related or similar substrates, i.e. sugars, inorganic anions, amino acids, etc. Although most similar to transporters in cluster 4 of the ABC-type uptake permeases that have hydrophobic amino acids as substrates (TC no. 3.A.1.4; Saier, 2000), the cyanobacterial Urt transporters appear to represent a new family of ABC uptake permeases, as the percentage identity of Urt proteins to cluster 4 proteins is relatively low, and urea represents a new type of substrate for ABC transporters.

### Experimental procedures

#### Bacterial strains and growth conditions

Anabaena sp. PCC 7120, Synechocystis sp. PCC 6803 and Synechococcus sp. PCC 7942 were grown axenically in BG11 medium, which contains 17.6 mM NaNO₃ (Rippka *et al.*, 1979), or in BG11₀ (nitrogen-free) medium supplemented with 2 mM NH₄Cl and 4 mM TES-NaOH buffer (pH 7.5) or 2 mM urea and 4 mM TES-NaOH buffer (pH 7.5). For plates, the medium was solidified with 1% separately autoclaved agar (Difco). Cultures were grown at 30°C in the light (75 μEm⁻²s⁻¹), with shaking (80–90 r.p.m.) for liquid cultures. Anabaena sp. PCC 7120 mutants carrying gene cassette C.K3 (Elhai and Wolk, 1988) were grown routinely in medium supplemented with 25 μg ml⁻¹ Nm, and mutants carrying gene cassette C.S3 (Prentki and Krisch, 1984) were grown in medium supplemented with 2–5 μg ml⁻¹ Sp and Sm. The cultures used for RNA isolation were grown exponentially in BG11₀C (BG11₀ supplemented with 0.84 g l⁻¹ NaHCO₃) supplemented with 5–8 mM NH₄Cl and 10–16 mM TES-NaOH buffer (pH 7.5) and bubbled with a mixture of CO₂ (1.5% v/v) and air. To analyse the effect of different nitrogen sources on the expression of *urt* genes, ammonium-grown cells were harvested at room temperature, washed with BG11₀ and resuspended in BG11₀C medium supplemented with nitrate, urea or without combined nitrogen and incubated under culture conditions.

Growth rates were estimated from the increase in protein concentration in the cultures. Protein concentration was determined by a modified Lowry procedure (Markwell *et al.*, 1978) in 0.2 ml aliquots periodically withdrawn from the cultures. The growth rate constant (μ) corresponds to ln2/*t*~d~, where *t*~d~ represents the doubling time. The Chl content of the cultures was determined in methanolic extracts of the cells

© 2002 Blackwell Science Ltd, *Molecular Microbiology*, **43**, 703–715
(Mackinney, 1941). The amount of cells that contains 1 μg of Chl contains about 20–25 μg of protein.

Escherichia coli strains DH5α, HB101, MC1061 and GM48 were grown in LB medium with 50 μg ml⁻¹ Ap, 50 μg ml⁻¹ Km, 30 μg ml⁻¹ Cm, 25 μg ml⁻¹ Sm or 100 μg ml⁻¹ Sp when necessary.

### Urease activity

For the determination of urease activity, an amount of cells containing 17–20 μg of Chl was mixed with 40 mM tricine-NaOH buffer (pH 8.1), 25 mM urea and 0.1 mg ml⁻¹ MTA and incubated at 30°C. (MTA is mixed alkyltrimethylammonium bromide, a detergent added to make the cells permeable to small molecules.) Samples were taken after different incubation times, and the cells were removed by centrifugation at room temperature. To determine the ammonium produced, 0.1 ml of supernatant was supplemented with water and mixed with 0.5 ml of a commercial reagent (Sigma ammonia colour reagent) in a final volume of 1.5 ml, and the A₄₃₀ was measured. The reaction was linear for at least 60 min. One activity unit corresponds to 1 μmol of ammonium produced min⁻¹. For the determination of the \( K_{\mathrm{M}} \) of urease for urea, different urea concentrations were tested up to 0–100 mM.

### Urea uptake assays

Wild-type or mutant cells grown in nitrate-containing medium and, unless indicated otherwise, incubated in the absence of combined nitrogen for 10–12 h were harvested by centrifugation at room temperature, washed with 5 mM tricine-NaOH buffer (pH 8.1) and resuspended in the same buffer. After preincubation at 30°C in the light (180 μE m⁻² s⁻¹) for 5–30 min, the assays were started by mixing the suspension of cells (1–15 μg ml⁻¹ Chl) with a solution of [¹⁴C]-CO(NH₂)₂ (200–2000 Bq μmol⁻¹) in tricine-NaOH buffer. After different incubation times, 0.1–1 ml samples were filtered (0.45 μm pore size Millipore HA filters were used) and washed with 2–5 ml of tricine-NaOH buffer. In some experiments, as indicated, the washing step was omitted in order to avoid the exit of urea from the cells. The filters carrying the cells were then immersed in scintillation cocktail, and their radioactivity was measured. Retention of radioactivity by boiled cells was used as a blank. For the determination of the \( K_{\mathrm{s}} \) of the cells for urea, rates from the linear parts of the uptake assays with different urea concentrations were used to derive kinetic parameters from a Lineweaver–Burk representation.

### Intracellular accumulation of labelled urea

Filters containing cells that had been used in the uptake assays described above were immediately immersed in 2–3 ml of boiling water and incubated at 100°C for 5 min. The filters were removed, and the resulting suspensions were centrifuged. Samples from the supernatants were lyophilized and dissolved in a small volume of water. Metabolites present in these samples were resolved by two-dimensional TLC using 0.1 mm cellulose plates (20 cm by 20 cm; Merck). The solvents used were *n*-butanol–pyridine–water (6:4:3) and *n*-butanol–ethanol–water (2:1:1). The resulting

radioactive areas were quantified in an InstantImager scanner for β particles (Packard). To calculate the intracellular concentration of [¹⁴C]-urea, an intracellular volume of 125 μl mg⁻¹ Chl was assumed (see references in Quintero et al., 2000).

#### Transposon mutagenesis

Mutagenesis of Anabaena sp. PCC 7120 by Tn5-derived transposons was performed as described by Wolk et al. (1991), generating a collection of 426 clones carrying the transposon. Transposon Tn1063a and DNA contiguous with it were recovered from Anabaena strain CSAV56 by digestion with EcoRV and transformation of *E. coli*. The plasmid recovered was called pCSAV56. The DNA contiguous with each side of Tn1063a was sequenced using primers that were homologous to the corresponding ends of the transposon (LTn1063a: 5′-CAATGCTATCAATGAG-3′; RTn1063a: 5′-CACATGGAAATATCAG-3′).

#### Plasmids

Plasmid pCSAV3 contains a 4.6 kb DNA fragment from the *ure* region of Anabaena sp. PCC 7120 cloned into ClaI-digested vector pIC20R. This DNA fragment was identified by hybridization using pCSAV56 as a probe and extends from nucleotide –246 with respect to the translation start of *ureC* to the first *ClaI* site located downstream of *ureG* (see http://www.Kazusa.or.jp/cyano/anabaena/). For the generation of a *ureG* mutant, a 0.7 kb *HindIII* fragment from pCSAV3 containing the last 540 bp of *ureG* was substituted by *HindIII*-ended gene cassette C.S3 from plasmid pRL463 (Elhai and Wolk, 1988) rendering plasmid pCSAV11. Then, the *EcoRI* fragment from pCSAV11 filled with Klenow enzyme was cloned into the *sacB*-bearing vector pRL278 (Cai and Wolk, 1990) digested with *NruI*, rendering pCSAV12. The *sacB* gene determines sensitivity to sucrose and can be counter-selected for in Anabaena sp., allowing positive selection for double recombinants (Cai and Wolk, 1990).

To clone *urtE* from Anabaena sp. PCC 7120, a 0.5 kb DNA fragment containing part of the ORF *slr0374* from Synechocystis sp. PCC 6803 was used as a probe in heterologous hybridization with Southern blots of total DNA from strain PCC 7120. Hybridization was ascribed to a *DraI* fragment of ≈ 1.4 kb from the Anabaena genome. A subgenomic gene library from strain PCC 7120 was constructed by cloning *DraI* restriction fragments of 1–2 kb in cloning vector pIC20R. From this subgenomic library, a clone exhibiting hybridization to the 0.5 kb *Synechocystis* fragment was identified and named pCSAV8. The insert of pCSAV8 extends from a *DraI* site located 189 bp downstream from the translation start codon of *urtD* to the first *DraI* site located downstream of *urtE*. For inactivation of *urtE*, a *HindIII* restriction site was generated at position 493 with respect to the translation start of *urtE*, as described previously (Ausubel et al., 2001). The mutagenic oligonucleotides used were URT25 (5′-CGACGGACGGAAGCTTCAATT-3′) and URT26 (5′-GAAATTGAAGCTTCCGTCCGTG-3′), and the flanking ones were M13 forward and reverse primers; pCSAV8 was used as template. The plasmid generated, pCSAV14, was

© 2002 Blackwell Science Ltd, Molecular Microbiology, **43**, 703–715
712 A. Valladares, M. L. Montesinos, A. Herrero and E. Flores

digested with HindIII and ligated to the 1.1 kb Km<sup>R</sup> gene cassette C.K3, excised with BamHI and filled with Klenow enzyme, generating plasmid pCSAV15. The PvuII fragment from pCSAV15, containing the interrupted *urtE* gene, was ligated to the sacB vector pRL277 digested with SpeI and filled with Klenow enzyme rendering pCSAV16.

Plasmid pCSAV90 contains a 7 kb DNA fragment from the *urt* region of Anabaena sp. PCC 7120 that was cloned by transformation of *E. coli* MC1061 with a library of HindIII DNA fragments from Anabaena strain CSAV2 and selection for Km<sup>R</sup>. This fragment, which was cloned into HindIII-digested vector pIC20R, extends from a HindIII site located 640 bp downstream of the translation start of *urtA* to the first HindIII site located downstream of *urtE*. pCSAV93 was generated to inactivate *urtB*. This plasmid contains a 1.9 kb DNA fragment amplified by PCR using the oligonucleotides URT6 (complementary to nucleotides 275–253 with respect to the translation start of the *urtC* gene) and M13 forward primer; pCSAV90 was used as template. PCR products were cloned in pGEM-T vector (Promega). The 2 kb Sm<sup>R</sup> Sp<sup>R</sup> gene cassette C.S3 excised with EcoRV and SmaI was inserted into a unique Eco47III site that is present in the Anabaena DNA insert of pCSAV93 to generate plasmid pCSAV94. The PvuII fragment from pCSAV94, containing the interrupted *urtB* gene, was ligated to the sacB vector pRL278 digested with NruI rendering pCSAV96.

Plasmid pCSAV98 contains a 2.3 kb DNA fragment from the *urt* region of Anabaena sp. PCC 7120 isolated from a library of Eco47III DNA fragments cloned into EcoRV-digested low-copy-number vector pDUCA7RV. This fragment extends from an Eco47III site located 251 bp upstream from the translation start codon of *urtA* to the Eco47III site located 754 bp downstream of the translation start of *urtB*. pCSAV120, which was generated to inactivate *urtA*, contains a 1.84 kb DNA fragment amplified by PCR using the oligonucleotides URT9 (complementary to nucleotides 245–222 with respect to the translation start of *urtB*) and URT19 (corresponding to nucleotides –235 to –216 with respect to the translation start of *urtA*) and genomic DNA from strain PCC 7120 as template. PCR products were cloned in pGEM-T vector (Promega). The 2 kb Sm<sup>R</sup> Sp<sup>R</sup> gene cassette C.S3 excised with EcoRV and SmaI was inserted into the BclI, EcoRI sites that are present in the Anabaena DNA insert of pCSAV120 to generate plasmid pCSAV121. The PvuII fragment from pCSAV121, containing the interrupted *urtA* gene, was ligated to the sacB vector pRL278 digested with BglII and filled in with Klenow enzyme rendering pCSAV122.

pRL443 and thereafter with Anabaena sp. Exconjugants were isolated (Elhai and Wolk, 1988), and double recombinants were identified as clones resistant to the antibiotic for which resistance was encoded in the inserted gene cassette, resistant to sucrose and sensitive to the antibiotic for which the resistant determinant was present in the vector portion of the transferred plasmid and were confirmed by Southern or PCR analysis.

### DNA and RNA isolation and manipulation

Total DNA (Cai and Wolk, 1990) and RNA (García-Domínguez and Florencio, 1997; based on Golden *et al.*, 1987) from Anabaena sp. PCC 7120 and its derivatives was isolated as described previously. Sequencing was carried out by the dideoxy chain termination method, using a T<sup>7</sup> Sequencing kit (Amersham Pharmacia Biotech) and [α-<sup>35</sup>S]-thio-dATP. DNA fragments were purified from agarose gels with the Geneclean II kit (Bio101). Plasmid isolation from *E. coli*, transformation of *E. coli*, digestion of DNA with restriction endonucleases, ligation with T4 ligase and PCR were performed by standard procedures (Sambrook *et al.*, 1989; Ausubel *et al.*, 2001).

### Southern blotting and hybridization

Southern analysis was carried out as described previously (Montesinos *et al.*, 1997) using GeneScreen Plus membranes (Dupont). DNA probes used in the hybridizations were obtained by restriction of plasmids or by PCR and were labelled with a DNA labelling kit (Ready to Go; Amersham Pharmacia Biotech) and [α-<sup>32</sup>P]-dCTP.

### Northern blotting and hybridization

For Northern analysis, 70 μg of RNA was loaded per lane and electrophoresed in 1% agarose denaturing formaldehyde gels. Transfer and fixation to Hybond-N<sup>+</sup> membranes (Amersham Pharmacia Biotech) were carried out using 0.1 N NaOH. Hybridization was performed at 65°C according to the recommendations of the manufacturers of the membranes. The *urtA* probe was amplified by PCR using oligonucleotides URT11 (complementary to positions +1260 to +1235 relative to the translation start of *urtA*) and URT18 (corresponding to positions +38 to +59 relative to the translation start of *urtA*) and pCSAV98 as template. This DNA fragment was <sup>32</sup>P labelled with a Ready to Go DNA labelling kit (Amersham Pharmacia Biotech) using [α-<sup>32</sup>P]-dCTP. Images of radioactive filters were obtained and quantified using a Cyclone storage phosphor system and the OPTIQUANT image analysis software (Packard).

### Primer extension analysis

Primer extension analysis was carried out as described previously (Muro-Pastor *et al.*, 1999). Oligonucleotides used for analysis of the *urtA* transcript were URT17 (complementary to positions +94 to +74 relative to the translation start of *urtA*) and URT22 (complementary to positions +61 to +40 relative

© 2002 Blackwell Science Ltd, *Molecular Microbiology*, **43**, 703–715
to the translation start of *urtA*). Plasmid pCSAV98, which contains the upstream region of the *urtA* gene, was used to generate dideoxy sequencing ladders using the same primers.

### RT–PCR

For RT–PCR experiments, 10 μg of strain PCC 7120 total RNA was mixed with 40 pmol of oligonucleotide URT6 (see above), URT2 (complementary to positions +821 to +793 relative to the translation start codon of *urtD*) or URT28 (complementary to positions +146 to +125 relative to the translation start of *urtE*) in the presence of 10 mM Tris-HCl, pH 8.0, 150 mM KCl and 1 mM EDTA, heated for 2 min at 85°C and then at 50°C for 1 h for annealing. The extension reactions were carried out at 47°C for 1 h in a final volume of 45 μl containing the whole annealing reaction, 0.25 mM each deoxynucleotide triphosphate, 200 U of reverse transcriptase (Superscript II; Gibco BRL) and the buffer recommended by the transcriptase provider. To control for the presence of contaminating DNA, samples containing 10 μg of RNA, 40 pmol of oligonucleotides and 1 μg of RNase A (DNase free; Boehringer) were incubated in a 45 μl reaction volume at 37°C for 1 h. PCR was carried out with 3–5 μl of retrotranscription mixture or RNase-treated sample as the template and oligonucleotides URT6, URT2 or URT28 (see above) and URT15 (corresponding to nucleotides +1084 to +1102 with respect to the *urtA* translation start) as primers. Samples containing the same oligonucleotides and strain PCC 7120 genomic DNA as the template were run in parallel and used as standards. PCR was performed by standard procedures. Half of each sample was resolved by electrophoresis in a 0.7% agarose gel and transferred to membranes for Southern blot analysis. The *urtC* probe was amplified by PCR using oligonucleotides URT23 (corresponding to positions +45 to +60 relative to the translation start of *urtC*) and URT24 (complementary to positions +872 to +855 relative to the translation start of *urtC*) and pCSAV90 as template.

### Bandshift assays

DNA fragments to be used in electrophoretic mobility shift assays (EMSAs) were obtained by PCR amplification. Oligonucleotides URT17 and URT19 (see above) and plasmid pCSAV98 were used for PCR amplification of the *urtA* upstream region. In the case of the *glnA* upstream region, oligonucleotides GA3 (corresponding to positions –238 to –215 relative to the translation start of *glnA*) and GA6 (complementary to positions –70 to –87 relative to the translation start of *glnA*) and plasmid pAN503 (Tumer et al., 1983) were used. The same unlabelled DNA fragments were added in a 25-fold molar excess as competitors in some assays. DNA fragments were end-labelled with T4 polynucleotide kinase (Boehringer) and [γ-³²P]-dATP as described previously (Ausubel et al., 2001). Assays were carried out as described previously (Luque et al., 1994) with 0.05 pmol of labelled fragment and 5 pmol of histidine-tagged NtcA purified from an extract of *E. coli* strain BL21 (pCSAM70, pREP4) carrying the *Anabaena* sp. PCC 7120 *ntcA* gene (Muro-Pastor et al., 1999). Images of radioactive gels were obtained using a Cyclone storage phosphor system (Packard).

---

### Acknowledgements

We thank A. M. Muro-Pastor and J. E. Frías for their generous help throughout this work, and F. Jüttner for useful discussions. Preliminary sequence data obtained from the Kazusa DNA Research Institute (Japan) and the DOE Joint Genome Institute (USA) are acknowledged. This work was supported by grants PB97-1137 and PB98-0481 from the Ministerio de Ciencia y Tecnología (Spain).

### References

Allen, M.M. (1984) Cyanobacterial cell inclusions. *Annu Rev Microbiol* **38**: 1–25.

Argall, M.E., Smith, G.D., Patrick, N., Stamford, J., and Youens, B.N. (1992) Purification and properties of urease from the cyanobacterium *Anabaena cylindrica*. *Biochem Int* **6**: 1027–1036.

Ausubel, F.M., Brent, R., Kingston, R.E., Moore, D.D., Seidman, J.G., Smith, J.A., and Struhl, K. (eds) (2001) *Current Protocols in Molecular Biology*. New York: Greene/Wiley-Interscience.

Bender, R.A. (1991) The role of the NAC protein in the nitrogen regulation of *Klebsiella aerogenes*. *Mol Microbiol* **5**: 2575–2580.

Berns, D.S., Holohan, P., and Scott, E. (1966) Urease activity in blue-green algae. *Science* **152**: 1077–1078.

Boos, W., and Lucht, J.M. (1996) Periplasmic binding protein-dependent ABC transporters. In *Escherichia coli and Salmonella: Cellular and Molecular Biology*. Neidhardt, F.C. (ed. in chief). Washington, DC: American Society for Microbiology Press, pp. 1175–1209.

Cai, Y., and Wolk, C.P. (1990) Use of conditionally lethal gene in *Anabaena* sp. strain PCC 7120 to select for double recombinants and to entrap insertion sequences. *J Bacteriol* **172**: 3138–3145.

Cho, B.C., Park, M.G., Shim, J.H., and Azam, F. (1996) Significance of bacteria in urea dynamics in coastal surface waters. *Mar Ecol Prog Ser* **142**: 19–26.

Collier, J.L., Brahamsha, B., and Palenik, B. (1999) The marine cyanobacterium *Synechococcus* sp. WH7805 requires urease (urea amidohydrolase, EC 3.5.1.5) to utilize urea as a nitrogen source: molecular-genetic and biochemical analysis of the enzyme. *Microbiology* **145**: 447–459.

Collins, C.M., Gutman, D.M., and Laman, H. (1993) Identification of a nitrogen-regulated promoter controlling expression of *Klebsiella pneumoniae* urease genes. *Mol Microbiol* **8**: 187–198.

Cooper, T.G., and Sumrada, R. (1975) Urea transport in *Saccharomyces cerevisiae*. *J Bacteriol* **121**: 571–576.

ElBerry, H.M., Majumdar, M.L., Cunningham, T.S., Sumrada, R., and Cooper, T.G. (1993) Regulation of the urea active transport gene (*DUR3*) in *Saccharomyces cerevisiae*. *J Bacteriol* **175**: 4688–4698.

Elhai, J., and Wolk, C.P. (1988) A versatile class of positive-selection vectors based on the nonviability of palindrome-containing plasmids that allows cloning into long polylinkers. *Gene* **68**: 119–138.

Elhai, J., Cai, Y., and Wolk, C.P. (1994) Conduction of pEC22, a plasmid coding for MR. EcoT22I, mediated by a

© 2002 Blackwell Science Ltd, *Molecular Microbiology*, **43**, 703–715
resident Tn3-like transposon, Tn5396. J Bacteriol 176: 5059–5067.

Flores, E., and Herrero, A. (1994) Assimilatory nitrogen metabolism and its regulation. In *The Molecular Biology of Cyanobacteria*. Bryant, D.A. (ed.). Dordrecht, The Netherlands: Kluwer Academic Publishers, pp. 487–517.

Frías, J.E., Flores, E., and Herrero, A. (1994) Requirement of the regulatory protein NtcA for the expression of nitrogen assimilation and heterocyst development genes in the cyanobacterium Anabaena sp. PCC 7120. Mol Microbiol 14: 823–832.

García-Domínguez, M., and Florencio, F.J. (1997) Nitrogen availability and electron transport control the expression of *glnB* gene (encoding P<sub>II</sub> protein) in the cyanobacterium Synechocystis sp. PCC 6803. Plant Mol Biol 35: 723–734.

Ge, X., Cain, K., and Hirschberg, R. (1989) Urea metabolism and urease regulation in the cyanobacterium Anabaena variabilis. Can J Microbiol 36: 218–222.

Golden, S.S., Brusslan, J., and Haselkorn, R. (1987) Genetic engineering of the cyanobacterial chromosome. Methods Enzymol 153: 215–231.

Goss, T.J., and Bender, R.A. (1995) The nitrogen assimilation control protein, NAC, is a DNA transcription activator in Klebsiella aerogenes. J Bacteriol 177: 3546–3555.

Healey, F.P. (1977) Ammonium and urea uptake by some freshwater algae. Can J Bot 55: 61–69.

Herrero, A., Muro-Pastor, A.M., and Flores, E. (2001) Nitrogen control in cyanobacteria. J Bacteriol 183: 411–425.

Hoshino, T., and Kose, K. (1989) Cloning and nucleotide sequence of *braC*, the structural gene for the leucine-, isoleucine-, and valine-binding protein of Pseudomonas aeruginosa PAO. J Bacteriol 171: 6300–6306.

Hoshino, T., and Kose, K. (1990) Cloning, nucleotide sequences, and identification of products of the Pseudomonas aeruginosa *bra* genes, which encode the high-affinity branched-chain amino acid transport system. J Bacteriol 172: 5531–5539.

Jahns, T., Zobel, A., Kleiner, D., and Kaltwasser, H. (1988) Evidence for carrier-mediated, energy-dependent uptake of urea in some bacteria. Arch Microbiol 149: 377–383.

Jahns, T., Schäfer, U., and Kaltwasser, H. (1995) Heat-stable ureases from two filamentous cyanobacteria. Microbiology 141: 737–741.

Kaneko, T., Sato, S., Kotani, H., Tanaka, A., Asamizu, E., Nakamura, Y., et al. (1996) Sequence analysis of the genome of the unicellular cyanobacterium Synechocystis sp. strain PCC6803. II. Sequence determination of the entire genome and assignment of potential protein-coding regions. DNA Res 3: 109–136.

Luque, I., Flores, E., and Herrero, A. (1994) Molecular mechanism for the operation of nitrogen control in cyanobacteria. EMBO J 13: 2862–2869.

McCarthy, J.J. (1972) The uptake of urea by marine phytoplankton. J Phycol 8: 216–222.

Mackinney, G. (1941) Absorption of light by chlorophyll solutions. J Biol Chem 140: 315–322.

Markwell, M.A.K., Hass, S.M., Bieber, L.L., and Tolbert, N.E. (1978) A modification of the Lowry procedure to simplify protein determination in membrane and lipoprotein samples. Anal Biochem 87: 206–210.

Mills, J., Wyborn, N.R., Greenwood, J.A., Williams, S.G., and

Jones, C.W. (1998) Characterisation of a binding-protein-dependent, active transport system for short-chain amides and urea in the methylotrophic bacterium *Methylophilus methylotrophus*. Eur J Biochem 251: 45–53.

Mitamura, O., Kawashima, M., and Maeda, H. (2000a) Urea degradation by picophytoplankton in the euphotic zone of Lake Biwa. Limnology 1: 19–26.

Mitamura, O., Seike, Y., Kondo, K., Ishida, N., and Okumura, M. (2000b) Urea decomposing activity of fractionated brackish phytoplankton in Lake Nakaumi. Limnology 1: 75–80.

Mobley, H.L.T., and Hausinger, R.P. (1989) Microbial ureases: significance, regulation, and molecular characterization. Microbiol Rev 53: 85–108.

Mobley, H.L.T., Island, M.D., and Hausinger, R.P. (1995) Molecular biology of microbial ureases. Microbiol Rev 59: 451–480.

Montesinos, M.L., Herrero, A., and Flores, E. (1997) Amino acid transport in taxonomically diverse cyanobacteria and identification of two genes encoding elements of a neutral amino acid permease putatively involved in recapture of leaked hydrophobic amino acids. J Bacteriol 179: 853–862.

Muro-Pastor, A.M., Valladares, A., Flores, E., and Herrero, A. (1999) The *hetC* gene is a direct target of the NtcA transcriptional regulator in cyanobacterial heterocyst development. J Bacteriol 181: 6664–6669.

Palinska, K.A., Jahns, T., Rippka, R., and Tandeau de Marsac, N. (2000) *Prochlorococcus marinus* strain PCC 9511, a picoplanktonic cyanobacterium, synthesizes the smallest urease. Microbiology 146: 3099–3107.

Pateman, J.A., Dunn, E., and Mackay, E.M. (1982) Urea and thiourea transport in *Aspergillus nidulans*. Biochem Genet 20: 777–791.

Prentki, P., and Krisch, H.M. (1984) *In vitro* insertional mutagenesis with a selectable DNA fragment. Gene 29: 303–313.

Quintero, M.J., Muro-Pastor, A.M., Herrero, A., and Flores, E. (2000) Arginine catabolism in the cyanobacterium *Synechocystis* sp. strain PCC 6803 involves the urea cycle and arginase pathway. J Bacteriol 182: 1008–1015.

Rippka, R., Deruelles, J., Waterbury, J.B., Herdman, M., and Stanier, R.Y. (1979) Generic assignments, strain stories and properties of pure cultures of cyanobacteria. J Gen Microbiol 111: 1–61.

Saier, M.H., Jr (1998) Molecular phylogeny as a basis for the classification of transport proteins from Bacteria, Archaea and Eukarya. Adv Microb Physiol 40: 81–136.

Saier, M.H., Jr (1999) Genome archeology leading to the characterization and classification of transport proteins. Curr Opin Microbiol 2: 555–561.

Saier, M.H., Jr (2000) A functional-phylogenetic classification system for transmembrane solute transporters. Microbiol Mol Biol Rev 64: 354–411.

Sambrook, J., Fritsch, E.F., and Maniatis, T. (1989) *Molecular Cloning: a Laboratory Manual*, 2nd edn. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.

Siewe, R.M., Weil, B., Burkovski, A., Eggeling, L., Krämer, R., and Jahns, T. (1998) Urea uptake and urease activity in *Corynebacterium glutamicum*. Arch Microbiol 169: 411–416.

Syrett, P.J., and Bekheet, I.A. (1977) The uptake of thiourea by *Chlorella*. *New Phytol* **79**: 291–297.

Tumer, N.E., Robinson, S.J., and Haselkorn, R. (1983) Different promoters for the *Anabaena* glutamine synthetase gene during growth using molecular or fixed nitrogen. *Nature* **306**: 337–342.

Weeks, D.L., Eskandari, S., Scott, D.R., and Sachs, G.

(2000) A H⁺-gated urea channel: the link between *Helicobacter pylori* urease and gastric colonization. *Science* **287**: 482–485.

Wolk, C.P., Cai, Y., and Panoff, J.-M. (1991) Use of a transposon with luciferase as a reporter to identify environmentally responsive genes in a cyanobacterium. *Proc Natl Acad Sci USA* **88**: 5355–5359.
