
THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2002 by The American Society for Biochemistry and Molecular Biology, Inc.

**EnvZ-OmpR Interaction and Osmoregulation in *Escherichia coli***  
Received for publication, November 7, 2001, and in revised form, April 23, 2002  
Published, JBC Papers in Press, April 24, 2002, DOI 10.1074/jbc.M110715200

Sheng Jian Cai and Masayori Inouye‡  
*From the Department of Biochemistry, Robert Wood Johnson Medical School, Piscataway, New Jersey 08854*

EnvZ, a histidine kinase/phosphatase in *Escherichia coli*, responds to the osmolarity changes in the medium by regulating the phosphorylation state of the transcription factor OmpR, which controls the expression levels of outer membrane porin proteins OmpF and OmpC. Although both *ompR* and *envZ* genes are located on the *ompB* locus under the control of the *ompB* promoter and transcribed as a single polycistronic mRNA, the expression of *envZ* is known to be significantly less than *ompR*. However, to date no accurate estimation for the amounts of EnvZ and OmpR in the cell has been carried out. Here we examined the levels of EnvZ and OmpR in the wild-type strain MC4100 by quantitative Western blot analysis using anti-OmpR and anti-EnvZc (cytoplasmic domain of EnvZ) antisera. It was observed that during exponential growth in L-broth medium there were ~3500 and 100 molecules per cell of OmpR and EnvZ, respectively. The levels of OmpR and EnvZ in MC4100 cells grown in a high osmolarity medium (nutrient broth with 20% sucrose) were about the same as those grown in L-broth, whereas they were 1.7-fold higher than those in a low osmolarity medium (nutrient broth). With His<sub>10</sub>-OmpR, we also determined that the *K<sub>d</sub>* value for the EnvZc-OmpR complex formation is 1.20 ± 0.17 μM. On the basis of these results, the molecular mechanism of osmoregulation of *ompF* and *ompC* is discussed.

membrane, which allow passive diffusion of small hydrophilic molecules of less than 650 Da in size (9). EnvZ also possesses the OmpR-P phosphatase activity to dephosphorylate OmpR-P. It has been proposed that osmotic signal regulates the ratio of the kinase to the phosphatase activity of EnvZ to modulate the level of cellular OmpR-P by primarily altering the phosphatase activity (10). The cellular OmpR-P levels reciprocally regulate the transcription of *ompF* and *ompC*. At low medium osmolarity a reduced level of OmpR-P, due to the decreased kinase/phosphatase ratio of EnvZ, favors the transcription of *ompF*. At high medium osmolarity an elevated OmpR-P level, resulting from the increased kinase/phosphatase ratio of EnvZ, allows the activation of *ompC* transcription. On the other hand, more OmpR-P molecules bind to the *ompF* promoter upstream region causing repression of *ompF* expression.

EnvZ, consisting of 450 amino acid residues, exists as a dimer located in the inner cytoplasmic membrane of *E. coli*. The cytoplasmic domain of EnvZ (residues 180–450), EnvZc, also exists as a dimer and consists of the linker region (residues 180–222), domain A (or DHp domain, residues 223–289) and domain B (or CA domain, residues 290–450). It possesses both kinase and phosphatase activities similar to the intact EnvZ (11, 12). The linker region is considered to play an important role in transducing the signal from the periplasmic receptor domain to the cytoplasmic catalytic domain (13). The NMR solution structures of both domain A and domain B have been solved (14, 15). Recently it has been demonstrated that domain A by itself can serve as the OmpR-P phosphatase, and it has been proposed that the phosphatase activity of EnvZ is modulated by the spatial arrangement between domain A and domain B (16).

The response regulator OmpR consists of an N-terminal CheY-like receiver domain containing the highly conserved Asp-55 phosphorylation site and a C-terminal DNA-binding domain (17). The two domains are connected by a flexible linker (18). The x-ray structure of the DNA binding domain shows a winged helix-turn-helix motif structure (19, 20). The phosphorylation of OmpR at Asp-55 greatly enhances its binding affinity for the regulatory sequences upstream of the *ompF* and *ompC* promoters (21–24). The *in vitro* and *in vivo* footprinting studies have demonstrated that OmpR-P binds to the –100 to –38 region (C1, C2, and C3 sites) of *ompC* and the –380 to the –361 region (F4 site) and the –100 to –39 region (F1, F2, and F3 sites) of *ompF* (24–27). These OmpR binding sites consist of 20 base pairs each sharing a consensus sequence (28). It has been shown that two OmpR-P molecules bind to each site in a head-to-tail manner (28, 29). The binding affinities of OmpR-P to these sites are in a hierarchical order such that OmpR-P independently binds to F1 and C1 sites whereas OmpR-P cooperatively binds to other sites only after the upstream F1 and C1 sites are occupied by OmpR-P (21, 27).

The *ompB* operon of *E. coli* is comprised of two genes, *ompR* and *envZ*, which are co-transcribed as a polycistronic mRNA from a promoter located 5′ to the *ompR* gene (30). It has been shown that significantly fewer EnvZ molecules are produced

Bacteria primarily rely on the His-Asp phosphorelay signal transduction system (HAP system),<sup>1</sup> or the two-component signal transduction system, to acclimate in response to the changes of environmental conditions (1, 2). The HAP system basically consists of a histidine kinase, which serves as a signal receptor, and its cognate response regulator, which regulates specific gene expression or mediates cellular locomotion (3).

One of most extensively studied HAP systems is the EnvZ/OmpR system mediating signal transduction in response to environmental osmolarity changes in *Escherichia coli* (4–6). EnvZ, a histidine kinase, undergoes trans-autophosphorylation on the highly conserved His-243 residue (7, 8). The high energy phosphoryl group is subsequently transferred to the conserved Asp-55 residue of OmpR, a response regulator. Phosphorylated OmpR (OmpR-P) serves as a transcription factor differentially modulating the expression of the major outer membrane porin genes, *ompC* and *ompF*. OmpC and OmpF form channels in the outer

* This work was supported by Grant GM19043 from the National Institutes of Health. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
‡ To whom correspondence should be addressed. Tel.: 732-235-4115; Fax: 732-235-4559; E-mail: inouye@rwja.umdnj.edu.
<sup>1</sup> The abbreviations used are: HAP, His-Asp phosphorelay signal transduction system; NTA, nitrilotriacetic acid; NB, nutrient broth; Omp, outer membrane porin proteins; ADPNP, 5′-adenylyl-β,γ-imidodiphosphate.

This paper is available on line at http://www.jbc.org 24155

24156

**EnvZ-OmpR Interaction and Osmoregulation in E. coli**

compared with OmpR molecules in a cell, and it has been estimated that there are about 1000 molecules of OmpR whereas there are only 10 molecules of EnvZ per cell (5). In this report, we reexamined the cellular OmpR and EnvZ levels under different growth conditions by quantitative Western blot analysis. We also determined the dissociation constant of OmpR and EnvZ by using His<sub>10</sub>-OmpR and Ni-NTA-agarose. These findings provide new insights into the mechanism by which the EnvZ/OmpR system regulates the osmosensory pathway in *E. coli*.

### EXPERIMENTAL PROCEDURES

#### Bacteria Strains—MC4100 [F<sup>−</sup> araD139 Δ(argF-lac) U169 rpsL150 relA1 ftsB-5301 ptsF25 deoC1 thiA1] (31), and AR137 [MC4100Δ(malT-OmpB)pcn80] (32) were used.

#### Antisera—The rabbit anti-OmpR and -EnvZ polyclonal antibodies (Pocono Rabbit Farm & Laboratory) were used as the first antibody, and the anti-rabbit Ig horseradish peroxidase-linked F(ab)<sub>2</sub> fragment (Amersham Biosciences) was used as the second antibody in Western blot analysis.

#### Preparation of Whole Cell Lysates—The wild-type *E. coli* MC4100 and a control strain AR137, which is unable to produce OmpR and EnvZ, were grown at 37°C to mid-log phase in L-broth or nutrient broth (NB) medium with or without 20% sucrose. Three separate cultures for each strain were grown. The cells were concentrated 30-fold by centrifugation and were then resuspended in buffer A (50 mM Tris-HCl (pH 8.0), 50 mM KCl, 5 mM β-mercaptoethanol, 1 mM phenylmethylsulfonyl fluoride). The cell concentrations were determined using a Petroff-Hausser counter. All sixteen squares were counted. Each culture was counted three times, and the results were averaged. The cells were lysed twice with a French Press, and the cell lysates were kept at −80°C.

#### Membrane Fraction Preparation—The membrane fraction from each culture was prepared as previously described (10).

#### Protein Preparation—EnvZc, OmpR, and His<sub>10</sub>-OmpR were prepared as described previously (33, 34). The final preparations of OmpR and EnvZ were in buffer A with 10% glycerol and kept at −80°C. The protein concentrations were determined by measuring the UV absorbance at 280 nm using an extinction coefficient of 24870 M<sup>−1</sup> cm<sup>−1</sup> for EnvZc and 13490 M<sup>−1</sup> cm<sup>−1</sup> for OmpR and His<sub>10</sub>-OmpR, respectively.

#### Western Blot Analysis—The cell lysates (for OmpR) or the membrane fractions (for EnvZ) were mixed with one-fifth volume of the 6-fold sample loading buffer (300 mM Tris-HCl (pH 6.8), 6% SDS, 30% glycerol, 0.6% bromphenol blue, 1.2 M β-mercaptoethanol), and the mixtures were boiled for 5 min before being subjected to SDS-PAGE. 17.5 and 12.5% SDS-polyacrylamide gels were used in electrophoresis for OmpR and EnvZ analysis, respectively. Proteins were then electrotransferred onto Immobilon-P (Millipore, MA) following the manufacturer's protocol and were detected with immunoblotting using the antiserum against OmpR or EnvZc and the second antibody, horseradish peroxidase-conjugated anti-rabbit Ig F(ab)<sub>2</sub> fragment. The protein bands were visualized by the Western blot chemiluminescence reagent plus kit (PerkinElmer Life Sciences, Inc.). The densities of protein bands were determined by a densitometer. The amounts of individual protein bands were calculated from the standard curves derived from a set of purified OmpR or EnvZc protein standards run on the same blot.

#### Binding Assay—The determination of the *K*<sub>d</sub> value for the His<sub>10</sub>-OmpR and EnvZc complex formation was carried out as follows: 1.5 μM His<sub>10</sub>-OmpR was mixed with various amounts of EnvZc (0.25, 0.5, 1, 2, 4, 8, 16, and 32 μM) in 130 μl of buffer B (20 mM Tris-HCl (pH 8.0), 300 mM NaCl, 20 mM imidazole, 5 mM β-mercaptoethanol, and 5% glycerol) in the presence or absence of 1 mM ADP plus 5 mM Mg<sup>2+</sup> or 1 mM ADPNP plus 5 mM Mg<sup>2+</sup> and incubated at 25°C for 30 min. After incubation, 15 μl of Ni-NTA-agarose (50% slurry in buffer B, Qiagen) was added to each reaction mixture. The mixture was incubated at 4°C with constant shaking for 40 min and then transferred to an Ultra-free-MC centrifugal filter device (UFC30HV00, Millipore) and centrifuged at 6000 rpm for 10 s. The resulting Ni-NTA-agarose was washed three times with buffer B at 4°C by quick spinning at 6000 rpm for 10 s. 10 μl of elution buffer (20 mM Tris-HCl (pH 8.0), 300 mM NaCl, 5 mM β-mercaptoethanol, 250 mM imidazole, and 5% glycerol) was added to the resin in the filter tube and mixed well. The mixture was incubated at room temperature for 10 min and then centrifuged at 6000 rpm for 1 min. The elution step was repeated once, and the eluates were combined. The eluates were then subjected to SDS-PAGE. After staining

---

**FIG. 1. Quantitative analysis of OmpR in E.coli (MC4100).** A, Western blot quantitative analysis of OmpR of E.coli cells grown in L-broth medium. Western blot analysis was carried out as described under “Experimental Procedures.” Lane 1, the whole cell extract of 1.45 × 10<sup>8</sup> cells was applied; lane 2, the whole cell extract of a negative control strain AR137 (*ompB*<sup>−</sup>); lanes 3–7, the purified OmpR protein standards and the amounts of OmpR loaded (ng) are as indicated on the top of the gel. B, a standard curve of the OmpR protein samples in lanes 3–7. The density of each band was plotted against the amount of OmpR (ng) loaded. The arrow indicates the point on the standard curve corresponding to the density of OmpR band in lane 1 in A, and the amount of OmpR is estimated to be 21.3 ng.

---

**TABLE I**

**Western blot analysis determination of levels of OmpR in MC4100 (L-broth)**

| Culture | Number of cells blotted | OmpR | Average OmpR ± S.D. | Number of molecules per cell | Medium |
|---------|--------------------------|------|-----------------------|-------------------------------|--------|
|         |                          | ng   |                       |                               |        |
| 1       | 1.45 × 10<sup>8</sup>    | 21.3 | 20.7 ± 0.47           | 3108 ± 71                    | L-broth |
|         |                          | 20.5 |                       |                               |        |
|         |                          | 20.2 |                       |                               |        |
| 2       | 1.18 × 10<sup>8</sup>    | 21.2 | 21.3 ± 3.7            | 3936 ± 692                   | L-broth |
|         |                          | 26.5 |                       |                               |        |
|         |                          | 20.5 |                       |                               |        |
| 3       | 1.26 × 10<sup>8</sup>    | 23.4 | 20.43 ± 2.7           | 3524 ± 472                   | L-broth |
|         |                          | 18   |                       |                               |        |
|         |                          | 19.9 |                       |                               |        |
| Average |                          |      |                       | 3523                         | L-broth |

EnvZ-OmpR Interaction and Osmoregulation in E. coli

TABLE II  
Western blot analysis determination of levels of OmpR in MC4100 (NB + 20% sucrose)

| Culture | Number of cells blotted | OmpR | Average OmpR ± S.D. | Number of molecules per cell | Medium |
|---------|--------------------------|------|-----------------------|----------------------------|--------|
|         |                          | ng   |                       |                            |        |
| 1       | $1.42 \times 10^{8}$     | 22.3 | $21.7 \pm 0.6$        | $3325 \pm 90$              | NB     |
|         |                          | 21.4 |                       |                            | 20% sucrose |
|         |                          | 21.2 |                       |                            |        |
| 2       | $1.14 \times 10^{8}$     | 24.8 | $20.3 \pm 3.9$        | $3877 \pm 760$             | NB     |
|         |                          | 19.2 |                       |                            | 20% sucrose |
|         |                          | 17.1 |                       |                            |        |
| 3       | $1.81 \times 10^{8}$     | 30.6 | $28.1 \pm 2.3$        | $3366 \pm 272$             | NB     |
|         |                          | 27.5 |                       |                            | 20% sucrose |
|         |                          | 26.2 |                       |                            | NB     |
| Average |                          |      |                       | 3525                       | 20% sucrose |

where ER is the concentration of the His$_{10}$-OmpR/EnvZc complex; Et, total EnvZc concentration; and Rt, total OmpR concentration.

RESULTS

Determination of Cellular Levels of OmpR—Previously it has been reported that the OmpR concentration in MC4100 is about 1 $\mu$g/mg of the soluble extract of cells grown in NB medium and about 2.3 $\mu$g/mg of the soluble extract of cells grown in NB + 20% sucrose (22). In the present report we used whole cell lysates rather than the soluble extract to achieve a more accurate estimation of the cellular OmpR concentration. The level of OmpR in the wild-type strain MC4100 grown in L-broth medium in mid-log phase was determined by quantitative Western blot analysis. An example of Western blot analysis is shown in Fig. 1A. Densitometric analysis of the OmpR standards was plotted in Fig. 1B. The amount of OmpR was calculated from the standard curve. The cellular OmpR in the lysates migrated at the same position as that of purified OmpR in SDS-PAGE while no band was detected at this position in the control ($ompB^{-}$) strain AR137 (Fig. 1A, lane 2). Two other protein bands cross-reacted with OmpR antiserum in MC4100 cells (Fig. 1A, lane 1). As these bands were also observed in the negative control AR137 (Fig. 1A, lane 2), they were not related to OmpR. The results from this analysis show that the MC4100 strain contains ~3500 molecules of OmpR per cell (Table I). We also examined the effect of osmolarity changes on the expression of $ompR$ using NB medium in the presence or the absence of 20% sucrose. As shown in Table II and Table III, 3525 and 2043 molecules of OmpR per cell were found when cells were grown in the high osmolarity medium (NB + 20% sucrose) and the low osmolarity medium (NB), respectively.

The present results further confirmed that the cellular OmpR level of the cells grown under high osmolarity was significantly higher (~1.7-fold) than that of the cells grown in low osmolarity. It should be noted that the amount of OmpR in MC4100 grown in L-broth medium and NB + 20% sucrose were about the same, that is ~3500 molecules per cell.

Determination of the Cellular Levels of EnvZ—Although the $envZ$ gene is in the same operon as the $ompR$ gene, their expression levels are known to be significantly different. Previously the cellular level of EnvZ was estimated to be 10 molecules per cell (5). We used the membrane fraction rather than the whole cell lysate for Western blot analysis to determine the cellular EnvZ levels. An example of Western blot analysis is shown in Fig. 2A. Densitometric analysis of EnvZc standards was plotted as shown in Fig. 2B, and the amount of EnvZ was calculated from the standard curve. Similar to the results of the Western blot analysis shown in Tables IV, V, and VI, the wild-type strain MC4100 grown in L-broth medium in mid-log phase was estimated to contain about 102 EnvZ monomers per cell (or 51 dimers per cell), whereas 63 and 113 EnvZ monomers

TABLE III  
Western blot analysis determination of levels of OmpR in MC 4100 (NB)

| Culture | Number of cells blotted | OmpR | Average OmpR ± S.D. | Number of molecules per cell | Medium |
|---------|------------------------|------|---------------------|-------------------------------|--------|
| 1       | $1.37 \times 10^{8}$    | ng   | $11.8 \pm 1.4$      | $1877 \pm 222$                | NB     |
|         |                        | 13.4 |                     |                               |        |
|         |                        | 11.2 |                     |                               |        |
|         |                        | 10.9 |                     |                               |        |
| 2       | $0.98 \times 10^{8}$    | 7.5  | $8.9 \pm 1.2$       | $1990 \pm 268$                | NB     |
|         |                        | 9.9  |                     |                               |        |
| 3       | $1.32 \times 10^{8}$    | 9.4  | $13.8 \pm 1.3$      | $2264 \pm 222$                | NB     |
|         |                        | 14.4 |                     |                               |        |
|         |                        | 14.8 |                     |                               |        |
|         |                        | 112.3|                     |                               |        |
| Average |                        |      |                     | $2043$                       | NB     |

TABLE IV  
Western blot analysis determination of levels of EnvZ in MC 4100 (L-broth)

| Culture | Number of cells blotted | EnvZ | Average EnvZ ± S.D. | Number of molecules per cell | Medium   |
|---------|------------------------|------|---------------------|-------------------------------|----------|
| 1       | $2.29 \times 10^{9}$    | ng   | $14.1 \pm 1.19$     | $118 \pm 10$                  | L-broth  |
|         |                        | 15.1 |                     |                               |          |
|         |                        | 14.5 |                     |                               |          |
|         |                        | 12.8 |                     |                               |          |
| 2       | $2.35 \times 10^{9}$    | 12.9 | $11.17 \pm 1.57$    | $96 \pm 14$                   | L-broth  |
|         |                        | 10.5 |                     |                               |          |
| 3       | $2.52 \times 10^{9}$    | 10.1 | $11.37 \pm 1.36$    | $91 \pm 11$                   | L-broth  |
|         |                        | 11.2 |                     |                               |          |
|         |                        | 12.8 |                     |                               |          |
| Average |                        |      |                     | $102$                         | L-broth  |

TABLE V  
Western blot determination of levels of EnvZ in MC 4100 (NB + 20% sucrose)

| Culture | Number of cells blotted | EnvZ | Average EnvZ ± S.D. | Number of molecules per cell | Medium           |
|---------|------------------------|------|---------------------|-------------------------------|------------------|
| 1       | $2.83 \times 10^{9}$    | ng   | $17.1 \pm 0.4$      | $121 \pm 3$                  | NB + 20% sucrose |
|         |                        | 16.8 |                     |                               |                  |
|         |                        | 17.6 |                     |                               |                  |
|         |                        | 16.9 |                     |                               |                  |
| 2       | $2.77 \times 10^{9}$    | 15.3 | $14.7 \pm 0.6$      | $106 \pm 4$                  | NB + 20% sucrose |
|         |                        | 14.5 |                     |                               |                  |
|         |                        | 14.2 |                     |                               |                  |
| 3       | $2.41 \times 10^{9}$    | 14.2 | $13.4 \pm 0.8$      | $112 \pm 7$                  | NB + 20% sucrose |
|         |                        | 12.6 |                     |                               |                  |
|         |                        | 13.4 |                     |                               |                  |
| Average |                        |      |                     | $113$                        | NB + 20% sucrose |

TABLE VI  
Western analysis blot determination of levels of EnvZ in MC 4100 (NB)

| Culture | Number of cells blotted | EnvZ | Average EnvZ ± S.D. | Number of molecules per cell | Medium |
|---------|------------------------|------|---------------------|-------------------------------|--------|
| 1       | $4.25 \times 10^{9}$    | ng   | $14.3 \pm 0.9$      | $68 \pm 4$                   | NB     |
|         |                        | 14.9 |                     |                               |        |
|         |                        | 14.8 |                     |                               |        |
|         |                        | 13.3 |                     |                               |        |
| 2       | $2.59 \times 10^{9}$    | 7.5  | $7.3 \pm 0.4$       | $56 \pm 3$                   | NB     |
|         |                        | 6.5  |                     |                               |        |
| 3       | $3.63 \times 10^{9}$    | 7.8  | $11.9 \pm 0.6$      | $66 \pm 7$                   | NB     |
|         |                        | 12.4 |                     |                               |        |
|         |                        | 12.6 |                     |                               |        |
|         |                        | 13.4 |                     |                               |        |
| Average |                        |      |                     | $63$                         | NB     |

per cell (or 32 and 57 dimers per cell) are present when cells were grown in NB (low osmolarity) or in NB + 20% sucrose (high osmolarity), respectively. Note that as the antiserum used was raised against EnvZc (residues 180–450), the full-length EnvZ should be detected with the same sensitivity as EnvZc. The full-length EnvZ band could not be detected in the

EnvZ-negative control strain AR 137 (Fig. 1B, lane 2) while it existed in a positive control strain (BL21/pET11-envZ) in which EnvZ is overexpressed (data not shown). Although there are some weak cross-reactive bands shown in lane 1, they are also present in the negative control AR 137 lane. The cellular levels of EnvZ, when cells were grown in L-broth and NB + 20%

EnvZ-OmpR Interaction and Osmoregulation in E. coli 24159

A
0.25 0.5 1 2 4 8 16 32 32 (µM)
![Image Description](#)

B
1 2 3 4 5 6 7 8 9
![Image Description](#)

FIG. 3. SDS-PAGE analysis of the His<sub>10</sub>-OmpR/EnvZc binding assay. A, the eluates from a binding assay with binding buffer in the absence of nucleotides and Mg<sup>2+</sup> were analyzed by SDS-PAGE, and the gel was stained with Coomassie Brilliant Blue. The total amount of EnvZc added in each binding reaction is as indicated on the top of the gel (lanes 1–9). Lane 9 is a control reaction in which no His<sub>10</sub>-OmpR was added. B, 10 µl of the total reaction mixtures used in A before the addition of Ni-NTA-agarose were analyzed by SDS-PAGE and stained with Coomassie Brilliant Blue.

sucrose media, were found to be more than ten times higher than the amount previously estimated (5).

**Determination of the Dissociation Constant of EnvZ and His<sub>10</sub>-OmpR—His<sub>10</sub>-OmpR**, purified as described previously, serves as a substrate for both kinase and phosphatase activities of EnvZ, similar to the wild-type OmpR protein (33). After His<sub>10</sub>-OmpR and EnvZc were mixed in the binding assay, both the EnvZ/His<sub>10</sub>-OmpR complex and free His<sub>10</sub>-OmpR were isolated from the mixture with Ni-NTA-agarose. The proteins bound to Ni-NTA-agarose were eluted and analyzed by SDS-PAGE. The gels were stained with Coomassie Brilliant Blue, and the densities of the EnvZ and His<sub>10</sub>-OmpR bands were measured with a densitometer. The densities of EnvZ bands were normalized; the relative density of EnvZc was 91% that of His<sub>10</sub>-OmpR when equimolar amounts of EnvZc and His<sub>10</sub>-OmpR were subjected to SDS-PAGE and stained with Coomassie Brilliant Blue. Also note that the densities of EnvZc and OmpR stained with Coomassie Brilliant Blue are linear in a range from 150 to 4800 ng (data not shown). The densities of the EnvZc and OmpR bands in the SDS-PAGE gel for the binding assay were within this range. Because OmpR molecules bind to EnvZc monomers stoichiometrically in a 1:1 ratio (2 OmpR molecules/1 EnvZc dimer),<sup>2</sup> the percentage of the His<sub>10</sub>-OmpR bound to EnvZc can be estimated from the ratio of densities of EnvZc to His<sub>10</sub>-OmpR in the Coomassie Brilliant Blue-stained SDS-PAGE gel. Panel A in Fig. 3 shows one of such SDS-PAGE analyses of the formation of the His<sub>10</sub>-OmpR/EnvZc complex bound to Ni-NTA-agarose using a binding assay with binding buffer that lacks nucleotides and Mg<sup>2+</sup>. Panel B shows the SDS-PAGE analysis of the binding reaction mixture used in panel A before the addition of Ni-NTA resin. Note that when His<sub>10</sub>-OmpR was not added in the reaction, no EnvZc was trapped by Ni-NTA (Fig. 3A, lane 9). The concentrations of the His<sub>10</sub>-OmpR/EnvZc complex were plotted against total EnvZc concentrations as described under “Experimental Procedures.” The binding curves shown in Fig. 4, A–C represent the best fit of the data using Sigmaplot from one binding experiment under each of three binding conditions, respectively. The apparent *K*<sub>d</sub> value thus obtained for His<sub>10</sub>-OmpR and EnvZc from three independent experiments in buffer B was 1.20 ± 0.17 µM whereas the *K*<sub>d</sub> values in the presence of ADP plus Mg<sup>2+</sup> and ADPNP plus Mg<sup>2+</sup> were found to be 1.17 ± 0.15 µM and 1.23 ± 0.19 µM, respectively, indicating that there were no significant differences for the *K*<sub>d</sub> values under the three conditions. It was speculated that when nucleotides (ADP or ADPNP) bind to EnvZ in the presence of Mg<sup>2+</sup>, conformational changes may occur in the EnvZ molecule resulting in an increase of the

binding affinities of OmpR or OmpR-P to EnvZ. However the present binding assays showed that binding of the nucleotides do not affect the binding affinities of OmpR to EnvZc.

DISCUSSION

In the present study, OmpR and EnvZ levels in *E. coli* cells grown in L-broth at mid-log phase were determined to be 3500 and 100 molecules per cell, respectively. OmpR and EnvZ levels were almost the same from cells grown in L-broth medium or in a high osmolarity medium (NB + 20% sucrose). However, when cells were grown in a low osmolarity medium (NB), both OmpR and EnvZ levels were about 60% of those in the high osmolarity medium. The mechanism for the differential expression of the *ompB* locus under different osmolarities remains to be elucidated. The autoregulation of the *ompB* locus has been reported, where the *ompB* expression is regulated by OmpR, the first product of the *ompB* operon (35). It should be noted that under all the different growth conditions used, the ratio of OmpR to EnvZ was almost constant, and OmpR molecules existed in 30–35-fold excess over EnvZ monomers or in 60–70-fold excess compared with EnvZ dimers. It has been reported that translation of OmpR mRNA is ten times more efficient than that of EnvZ using *lacZ* as a reporter in *Salmonella typhimurium* (36). Because both *envZ* and *ompR* genes are transcribed as a single polycistronic mRNA (30), the expression of the two genes are differentially regulated at the level of translation initiation (36).

The apparent *K*<sub>d</sub> value of EnvZc and His<sub>10</sub>-OmpR interaction was 1.20 ± 0.17 µM under the conditions used. Assuming the cell volume is 10<sup>−15</sup> liters (37), the cellular concentrations of OmpR and EnvZ were calculated to be 6 and 0.18 µM, respectively. Therefore on the basis of the *K*<sub>d</sub> value thus obtained, ~85% of EnvZ molecules in a cell are estimated to be occupied with OmpR. It is important to note that as OmpR molecules exist in concentrations 60–70-fold higher than EnvZ dimers, OmpR-P molecules formed on EnvZ dimers are always competed with free OmpR molecules in the cytoplasmic fraction. It has been demonstrated that OmpR-P has a much higher affinity for the *ompF* and *ompC* promoter regions than OmpR, so that once the OmpR-P molecules released into the cytoplasm are bound to the *ompF* and *ompC* promoter regions they are readily sequestered from the EnvZ phosphatase activity (38).

Previous reports (27) have proposed that at low osmolarity, OmpR-P binds to the C1 site and cooperatively to F1, F2, and somewhat loosely to F3 sites resulting only in *ompF* expression, whereas at high osmolarity, the concentration of OmpR-P increases allowing C2, C3, and F4 sites (the repressor site for the *ompF* expression) to be also occupied by OmpR-P, resulting in the induction of *ompC* expression and repression of *ompF* ex-

24160 EnvZ-OmpR Interaction and Osmoregulation in E. coli

FIG. 4. EnvZc/His<sub>10</sub>-OmpR complex formation. His<sub>10</sub>-OmpR (1.5 μM) was mixed with various amounts of EnvZc in binding buffer with or without 1 mM ADP plus 5 mM Mg<sup>2+</sup> or 1 mM ADPNP plus 5 mM Mg<sup>2+</sup>. The His<sub>10</sub>-OmpR /EnvZc complex and free His<sub>10</sub>-OmpR were removed from the reaction mixture by Ni-NTA-agarose and subjected to SDS-PAGE as described under "Experimental Procedures." After Coomassie Brilliant Blue staining, the densities of EnvZc and His<sub>10</sub>-OmpR bands were analyzed by a densitometer. The concentrations of the His<sub>10</sub>-OmpR/EnvZc complex were plotted against the total EnvZc concentrations using the Sigmaplot program as described under "Experimental Procedures." The curves indicate the best fit of the data. The data plotted were the mean values of duplicate reactions at each point of one experiment. Three independent experiments were carried out for each binding condition. A, a binding assay in binding buffer in the absence of nucleotides and Mg<sup>2+</sup>; K<sub>d</sub> = 1.13 ± 0.16 μM. B, a binding assay in binding buffer supplemented with 1 mM ADP plus 5mM Mg<sup>2+</sup>; K<sub>d</sub> = 1.19 ± 0.15 μM. C, a binding assay in binding buffer supplemented with 1mM ADPNP plus 5 mM Mg<sup>2+</sup>; K<sub>d</sub> = 1.09 ± 0.20 μM.

pression. In this fashion, the reciprocal expression of *ompF* and *ompC* is achieved depending upon the medium osmolarity. According to the footprinting experiment, F1, F2, and a significant fraction of the F3 site were protected when the OmpR-P concentrations were between 108 and 135 nM whereas all C1, C2, and C3 sites became protected only when the OmpR-P

concentration reached 540 nM (27), assuming that the OmpR added was fully phosphorylated. Assuming that these results may be applied in vivo, at low osmolarity the phosphorylation of only 3.5% (120 nM or 70 OmpR-P molecules/cell) of total OmpR molecules in a cell (2024 OmpR molecules per cell) would be enough to activate the expression of ompF, whereas at high osmolarity the phosphorylation of about 10% (580 nM or 350 OmpR-P molecules/cell) of total OmpR molecules in a cell (3500 molecules per cell) would be sufficient to promote the expression of ompC and to repress the expression of ompF. Clearly the 60 EnvZ monomers or 30 EnvZ dimers per cell at low osmolarity and 100 EnvZ monomers or 50 EnvZ dimers per cell at high osmolarity are sufficient to fulfill the effective regulation of ompF and ompC. On the basis of OmpR amounts it has been shown that the OmpR-P level at high osmolarity was about 4-fold higher than that at low osmolarity (39). Considering that the cellular OmpR level is increased by more than 70% at high osmolarity than that at low osmolarity, the actual OmpR-P level could be increased more than 7-fold at high osmolarity in comparison with the level of OmpR-P at low osmolarity. In addition, the cell volume increases at low osmolarity whereas it decreases at high osmolarity resulting in even more dramatic changes in the OmpR-P concentration. This sharp change of the OmpR-P concentration appears to account well for the expression patterns of ompF and ompC at high and low osmolarity. However, the regulation of ompF and ompC in the cell may be more complex, as there are cross-talks among the His-Asp phosphorelay systems (40), and other genes besides ompF and ompC are known to be regulated by OmpR-P (41–43).

A similar quantitative Western blot analysis has been carried out with the proteins involved in chemotaxis signal transduction. The cellular contents of the total MCPs (methyl-accepting chemotaxis protein), CheA, CheY, and CheW were estimated to be 6000 molecules each per cell in E. coli (44, 45). However it is important to know that CheA belongs to Class II histidine kinases, whereas EnvZ belongs to Class I, the major signal transducing kinases in E. coli as 29 of 30 histidine kinases in E. coli belong to Class I (46). Class II kinases are involved in regulating flagella rotation whereas Class I kinases regulate transcription of specific genes (47). The present report is the first to accurately estimate the cellular contents of a Class I kinase and its cognate response regulator, suggesting that the Class I kinase system is substantially different from the Class II kinase system in their regulatory mechanisms.

The interaction between EnvZ and OmpR-P is not discussed in the present study. A recent report concerning this issue raised a question about the role of EnvZ phosphatase activity in osmoregulation (48). We have reexamined the OmpR-P interaction with EnvZc, and these results will be presented elsewhere. The characterization of the OmpR-P-EnvZ interaction provides important insights into the bi-functional roles of EnvZ in the osmoregulation of ompC and ompF.

Acknowledgments—We thank Ling Qin for supplying His<sub>10</sub>-OmpR and Yan Zhu for providing EnvZc. We are grateful to Ling Qin, Yan Zhu and Takeshi Yoshida for their critical reading of the manuscript.

---

**REFERENCES**

1. Hoch, J. A. (2000) *Curr. Opin. Microbiol.* **3**, 165–170
2. Stock, A. M., Robinson, V. L., and Goudreau, P. N. (2000) *Annu. Rev. Biochem.* **69**, 183–215
3. Grebe, T. W., and Stock, J. B. (1999) *Adv. Microb. Physiol.* **41**, 139–227
4. Egger, L. A., Park, H., and Inouye, M. (1997) *Genes Cells* **2**, 167–184
5. Pratt, L., and Silhavy, T. J. (1995) in *Two-component Signal Transduction*. (Hoch, J., and Silhavy, T. J., eds), pp. 105–107, American Society for Microbiology Press, Washington, D. C.
6. Forst, S. A., and Roberts, D. L. (1994) *Res. Microbiol.* **145**, 363–373
7. Yang, Y., and Inouye, M. (1991) *Proc. Natl. Acad. Sci. U. S. A.* **88**, 11057–11061
8. Yang, Y., and Inouye, M. (1993) *J. Mol. Biol.* **231**, 335–342
9. Nikaido, H., and Vaara, M. (1985) *Microbiol. Rev.* **49**, 1–32
10. Jin, T., and Inouye, M. (1993) *J. Mol. Biol.* **232**, 484–492
11. Park, H., Saha, S. K., and Inouye, M. (1998) *Proc. Natl. Acad. Sci. U. S. A.* **95**, 6728–6732
12. Dutta, R., Qin, L., and Inouye, M. (1999) *Mol. Microbiol.* **34**, 633–640
13. Park, H., and Inouye, M. (1997) *J. Bacteriol.* **179**, 4382–4390
14. Tomomori, C., Tanaka, T., Dutta, R., Park, H., Saha, S. K., Zhu, Y., Ishima, R., Liu, D., Tong, K. I., Kurokawa, H., Qian, H., Inouye, M., and Ikura, M. (1999) *Nat. Struct. Biol.* **6**, 729–734
15. Tanaka, T., Saha, S. K., Tomomori, C., Ishima, R., Liu, D., Tong, K. I., Park, H., Dutta, R., Qin, L., Swindells, M. B., Yamazaki, T., Ono, A. M., Kainosho, M., Inouye, M., and Ikura, M. (1998) *Nature* **396**, 88–92
16. Zhu, Y., Qin, L., Yoshida, T., and Inouye, M. (2000) *Proc. Natl. Acad. Sci. U. S. A.* **97**, 7808–7813
17. Kato, M., Aiba, H., Tate, S., Nishimura, Y., and Mizuno, T. (1989) *FEBS Lett.* **249**, 168–172
18. Kenney, L. J., Bauer, M. D., and Silhavy, T. J. (1995) *Proc. Natl. Acad. Sci. U. S. A.* **92**, 8866–8870
19. Martinez-Hackert, E., and Stock, A. M. (1997) *Structure* **5**, 109–124
20. Kondo, H., Nakagawa, A., Nishihiro, J., Nishimura, Y., Mizuno, T., and Tanaka, I. (1997) *Nat. Struct. Biol.* **4**, 28–31
21. Head, C. G., Tardy, A., and Kenney, L. J. (1998) *J. Mol. Biol.* **281**, 857–870
22. Forst, S., Delgado, J., and Inouye, M. (1989) *J. Bacteriol.* **171**, 2949–2955
23. Aiba, H., Nakasai, F., Mizushima, S., and Mizuno, T. (1989) *J. Biochem. (Tokyo)* **106**, 5–7
24. Huang, K. J., and Igo, M. M. (1996) *J. Mol. Biol.* **262**, 615–628
25. Tsung, K., Brissette, R. E., and Inouye, M. (1989) *J. Biol. Chem.* **264**, 10104–10109
26. Rampersaud, A., Harlocker, S. L., and Inouye, M. (1994) *J. Biol. Chem.* **269**, 12559–12566
27. Bergstrom, L. C., Qin, L., Harlocker, S. L., Egger, L. A., and Inouye, M. (1998) *Genes Cells* **3**, 777–788
28. Harlocker, S. L., Bergstrom, L., and Inouye, M. (1995) *J. Biol. Chem.* **270**, 26849–26856
29. Harrison-McMonagle, P., Denissova, N., Martinez-Hackert, E., Ebright, R. H., and Stock, A. M. (1999) *J. Mol. Biol.* **285**, 555–566
30. Hall, M. N., and Silhavy, T. J. (1981) *J. Mol. Biol.* **151**, 1–15
31. Hall, M. N., and Silhavy, T. J. (1981) *J. Mol. Biol.* **146**, 23–43
32. Harlocker, S. L., Rampersaud, A., Yang, W. P., and Inouye, M. (1993) *J. Bacteriol.* **175**, 1956–1960
33. Qin, L., Dutta, R., Kurokawa, H., Ikura, M., and Inouye, M. (2000) *Mol. Microbiol.* **36**, 24–32
34. Dutta, R., Yoshida, T., and Inouye, M. (2000) *J. Biol. Chem.* **275**, 38645–38653
35. Liljestrom, P., Laamanen, I., and Palva, E. T (1988) *J. Mol. Biol.* **201**, 663–673
36. Berman M. L., and Jackson D. E. (1984) *J. Bacteriol.* **159**, 750–756
37. Wanner, B. L. (1996) in *Escherichia coli and Salmonella* (Neidhardt, F. C., ed) Vol. 1, pp. 1359, American Society of Microbiology, Washington, D. C.
38. Qin, L., Yoshida, T., and Inouye, M. (2001) *Proc. Natl. Acad. Sci. U. S. A.* **98**, 908–913
39. Forst, S., Delgado, J., Rampersaud, A., and Inouye, M. (1990) *J. Bacteriol.* **172**, 3473–3477
40. Wanner, B. L. (1992) *J. Bacteriol.* **174**, 2053–2058
41. Verhoeft, C., Lugtenberg, B., van Boxtel, R., de Graaff, P., and Verheij, H. (1979) *Mol. Gen. Genet.* **169**, 137–146
42. Shin, S., and Park, C. (1995) *J. Bacteriol.* **177**, 4696–4702
43. Mao, W., and Siegle, D. A. (1998) *Mol. Microbiol.* **27**, 415–424
44. Gegner, J. A., Graham, D. R., Roth, A. F., and Dahlquist, F. W. (1992) *Cell* **70**, 975–982
45. Kuo, S. C., and Koshland, D. E., Jr. (1987) *J. Bacteriol.* **169**, 1307–1314
46. Mizuno, T. (1997) *DNA Res.* **4**, 161–168
47. Yamaguchi, S., Aizawa, S., Kihara, M., Isomura, M., Jones, C. J., and Macnab, R. M. (1986) *J. Bacteriol.* **168**, 1172–1179
48. Mattison, K., and Kenney, L. (2002) *J. Biol. Chem.* **277**, 11143–11148

MECHANISMS OF SIGNAL
TRANSDUCTION:
EnvZ-OmpR Interaction and
Osmoregulation in *Escherichia coli*

Sheng Jian Cai and Masayori Inouye
J. Biol. Chem. 2002, 277:24155-24161.
doi: 10.1074/jbc.M110715200 originally published online April 24, 2002

Access the most updated version of this article at doi: [10.1074/jbc.M110715200](http://dx.doi.org/10.1074/jbc.M110715200)

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinity-sites).

Alerts:
- When this article is cited
- When a correction for this article is posted

[Click here](https://www.jbc.org/site/alerts) to choose from all of JBC's e-mail alerts

This article cites 47 references, 22 of which can be accessed free at [http://www.jbc.org/content/277/27/24155.full.html#ref-list-1](http://www.jbc.org/content/277/27/24155.full.html#ref-list-1)
