
insight review articles

Circadian rhythms from flies to human

Satchidananda Panda*†, John B. Hogenesch† & Steve A. Kay*†

*Department of Cell Biology, The Scripps Research Institute, La Jolla, California 92037, USA (e-mail: stevek@scripps.edu)  
†Genomics Institute of Novartis Research Foundation, San Diego, California 92121, USA  

In this era of jet travel, our body ‘remembers’ the previous time zone, such that when we travel, our sleep-wake pattern, mental alertness, eating habits and many other physiological processes temporarily suffer the consequences of time displacement until we adjust to the new time zone. Although the existence of a circadian clock in humans had been postulated for decades, an understanding of the molecular mechanisms has required the full complement of research tools. To gain the initial insights into circadian mechanisms, researchers turned to genetically tractable model organisms such as *Drosophila*.

The rotation of the Earth causes predictable changes in light and temperature in our natural environment. Accordingly, natural selection has favoured the evolution of circadian (from the Latin *circa*, meaning ‘about’, and *dies*, meaning ‘day’) clocks or biological clocks — endogenous cellular mechanisms for keeping track of time. These clocks impart a survival advantage by enabling an organism to anticipate daily environmental changes and thus tailor its behaviour and physiology to the appropriate time of the day. The clock is synchronized by the day-night cycle, allowing the organism to accommodate not only the daily cycles of light and dark attributable to the Earth’s rotation, but also the alteration in relative span of day and night caused by the tilting of the Earth’s axis relative to the Sun. Thus, a circadian timing mechanism that undergoes daily adjustment is useful as a seasonal timer as well.

Constructing a true 24-hour clock, as opposed to a mere sand-timer, is not a trivial task for any organism to undertake during the course of its evolution. Neither is the analysis of how such a precise biological system is assembled and maintained. Genetics of circadian rhythms in flies has elucidated the working principles of the core clock, and provided the tools by which its conservation is also seen in mammals. More recently, genomic analysis of circadian rhythms in flies and mammals has revealed conservation of output physiology that has opened up new avenues in using flies as a model system in the understanding of the daily regulation of behaviour at the molecular level.

**Circadian behaviours in *Drosophila***

As several aspects of *Drosophila* physiology and behaviour are restricted to particular times of day, the organism became a natural model system for molecular investigation of circadian regulation. Adult flies emerge from their pupal cases (eclose) when it is cool and moist during the early morning, so minimizing the risk of desiccation as the emerging fly expands its folded wings and hardens its cuticle. Pupae exposed to a 12-h light:12-h dark cycle and subsequently kept in constant darkness also time their eclosion to when they expect dawn (subjective dawn), indicating the presence of an internal pacemaker¹. Once emerged, adult flies restrict flight, foraging and mating activities to the day (or subjective day), while they tend to ‘sleep’ (that is, they are relatively unresponsive to sensory stimuli and exhibit rest homeostasis²³) during the night.

Circadian regulation of such physiology and behaviour results from coordination of the activities of multiple tissues and cell types. An example is the consolidation of feeding behaviour to the day phase, which involves regulation of the sensitivity of chemosensory organs to locate food, activity of the wing muscles to move towards the food, and the action of the digestive system to assimilate nutrients. This locomotor activity rhythm is relatively refractory to acute or minor changes in light levels, such as during lightning and full moons, but is exquisitely sensitive to the timing of dawn and dusk to adapt to the seasonally changing day length.

**Discovery of period mutants in *Drosophila***

Early experiments in fly circadian biology established the endogenous nature of the eclosion rhythm. It was found that period length of this rhythm under constant darkness is dependent upon the genetic make-up of the fly strain (reviewed in ref. 1), thus suggesting specific sets of genes were involved in defining complex behaviours (a controversial idea three decades ago). These studies set the stage for a forward genetics approach using eclosion rhythms as a phenotype to identify clock components. Ron Konopka, while a graduate student in Seymour Benzer’s laboratory, performed a phenotype-based screen of mutagen-exposed flies and isolated three period mutants in eclosion rhythm, long (*perˡ*), short (*perˢ*) and arrhythmic (*per⁰*), which mapped to a single genetic locus⁴. These mutant flies exhibited similar defects in locomotor activity rhythms, the detection of which was facilitated by an automated screen developed by Yoshiki Hotta, also in Benzer’s laboratory. This more quantitative and persistent trait became the preferred phenotype for mutant screens for the next 30 years (Fig. 1), and a variant of this measure also became a powerful circadian phenotype in rodents.

The pleiotropic effect of each *per* mutation on both eclosion and locomotor activity rhythm established the existence of a single oscillator underlying different rhythms in different developmental stages of the animal. Restoration

of a short-period activity rhythm in *per*⁰ flies by transplantation of  
adult brain from *per*⁵ flies on the abdomen of the recipient demonstrated  
the presence of this oscillator in the fly brain⁵.  

Some years later in the mid-1980s, the *period* gene was cloned  
independently by the Young and Rosbash laboratories⁶,⁷ and shown to  
encode a large protein of more than 1,200 amino acids. In transgenic  
experiments, expression of a wild-type copy of this gene restored  
normal behavioural rhythms in arrhythmic *per*⁰ flies. Several early  
studies of these mutants shed light on the role of *per* in maintenance of  
circadian rhythms. Behavioural analyses of flies harbouring different  
copy numbers of *per* genes demonstrated dependence of period  
length on gene dosage—higher doses of *per* decreased period length⁸.  
Both *per* mRNA and protein levels exhibited rhythmic abundance,  
which reflected the behavioural rhythm: a near 24-h molecular  
rhythm in the wild-type flies, no rhythm in *per*⁰ flies, a short period  
rhythm in *per*⁵ flies, and along period rhythm in *per*⁴ flies.  

Subsequent characterization of these flies using genetic mosaic  
and transgenic approaches defined a group of 20–30 lateral neurons  
in the adult fly brain as the anatomic site controlling activity rhythm⁹  
(Fig. 2). Restricted expression of PER protein in these lateral neurons  
of *per*⁰ flies restored normal behavioural rhythms¹⁰. Homologues of  
the *Drosophila per* gene were subsequently cloned from several other  
insect species¹¹⁻¹⁵, and complementation studies of the *per*⁰ allele  
with genes derived from other insect species demonstrated conservation  
of rhythmicity¹⁶. Additionally, PER-like immunoreactivity was  
shown in different orders of animal species¹⁷,¹⁸, suggesting a  
functionally conserved clock throughout the animal kingdom.  
Characterization of *per* in the circadian clock mechanism established  
the first genetic and anatomic basis for an animal behaviour.  

**The transcriptional feedback model**  
The successful identification of *per* spawned subsequent genetic and  
biochemical screens to identify additional components of the circadian clock. Several additional period mutants were isolated in flies.  
One of these, *timeless (tim)*, exhibited characteristics indicative of a  
true clock component, including all three types of period defect that  
mapped to the same locus and, more importantly, an elevated level  
of cytoplasmic PER protein in the arrhythmic allele¹⁹,²⁰. Positional  
cloning was used²¹ to isolate *tim*; it exhibited molecular rhythms  
similar to PER, and its mRNA and protein levels were coincident  
with PER in the fly head. TIM protein was also isolated as an interaction  
partner of PER in a yeast two-hybrid assay²², a method for  
detecting direct protein-protein interactions.  
Detailed molecular genetic characterization of *per* and *tim*  
offered a skeletal clock mechanism that has been a subject of several  

reviews. Both genes are at the core of a transcriptional feedback loop  
in which their protein products, PER and TIM, dimerize and they accumulate in the cytoplasm during the day, then translocate into the  
nucleus in the evening to negatively regulate their own transcription.  
Both proteins are progressively phosphorylated, leading to their  
eventual degradation in the late night. Neither of these transcriptional  
inhibitors harbours any domain with similarity to known DNA-  
binding motifs. However, a domain was identified in PER that shared  
homology with a second fly protein, the single-minded gene product  
SIM, and a mammalian protein ARNT (aryl hydrocarbon nuclear  
translocator)²³,²⁴. In SIM and ARNT, this PAS (PER, ARNT and SIM)  
domain is accompanied by a basic region-helix-loop-helix (bHLH)  
domain, a DNA-binding and heterodimerization surface.  
PAS domains were shown to be dimerization surfaces, and given  
that bHLH proteins often function as heterodimeric pairs (and that  
PER lacked the bHLH domain), this suggested that PER could function  
as a transcriptional repressor of a bHLH-PAS heterodimeric  
pair²⁵. A key element of this mystery was revealed after analyses of *per*  
and *tim* promoters defined a clock box or E-box (a bHLH protein-  
binding site) regulatory element that conferred transcriptional  
cycling²⁶,²⁷. Two additional clock genes were identified in the fly with  
mammalian orthologues, *dClock (dClk)* and *cycle (cyc)*, which  
encode proteins containing both bHLH DNA-binding domains and  
PAS domains²⁷⁻²⁹. These two proteins were shown to heterodimerize,  
bind directly to E-box elements found in the *per* and *tim* promoters,  
and activate their expression. This activation is subsequently inhibited  
by PER and TIM, thus closing the molecular feedback loop²⁷.  
The identification of these new players resulted in the following  
model for the generation of molecular (and the resultant behavioural)  
rhythmicity (Fig. 3). The bHLH-PAS heterodimeric pair, dCLK  
and CYC, reside in the nucleus on the E-box elements in the *per* and  
*tim* structural genes, positively regulating their transcription. PER  
and TIM protein levels continue to rise throughout the day to their  
peak levels in the early evening—a few hours after the peak level of  
*per* and *tim* mRNAs. The two proteins heterodimerize and translocate into the nucleus where they inhibit the transcriptional activity of  
the dCLK/CYC complex, thus repressing their own transcription. As  
both PER and TIM proteins are degraded before dawn, this process is  
relieved, lifting repression of the dCLK/CYC complex, thereby  
starting another cycle of PER and TIM accumulation. Incredibly, this  
core mechanism and several of the above mentioned components  
are conserved between flies and mammals, over 600 million years of  
evolutionary time²⁷⁻³¹.  
*Drosophila* genetics identified three additional circadian components, *doubletime (dbt)*, *shaggy (sgg)* and *vrille (vri)*, which act to  
refine this simple transcriptional-translational feedback loop³²⁻³⁴.  

**Table 1 Properties of fly and mammalian clock genes and proteins**

| Drosophila Gene name | Properties                                                                                   | Mammal Gene name       | Properties                                                                                          |
|----------------------|----------------------------------------------------------------------------------------------|------------------------|-----------------------------------------------------------------------------------------------------|
| *period (per)*       | RNA and protein cycle. Binds to TIM. Inhibits dCLK/CYC function.                              | *Period 1 (Per 1)*     | RNA and proteins cycle. Physically associates with CRY and among PER proteins. Activator of BMAL 1 function. (Mutation in *Per2* is associated with FASPS.) |
|                      |                                  | *Period 2 (Per 2)*     |                                                                                                     |
|                      |                                  | *Period 3 (Per 3)*     |                                                                                                     |
| *timeless (tim)*     | RNA and protein cycle. Binds to PER and facilitates PER nuclear transport. Inhibits dCLK/CYC function. Degrades in response to light. | *Timeless (Tim)*      | Constitutively expressed. Closest *Drosophila* relative is *timeout*. Homozygous null mutant is lethal, making it impossible to conclusively establish clock function. |
| *doubletime (dbt)*   | Constitutively expressed. Ser/Thr kinase (CK1ε). Phosphorylates TIM-free PER, promoting its degradation. | *Casein kinase 1ε*     | Constitutively expressed. Protein kinase (CK1ε). Phosphorylates PER and affects PER stability.                     |
| *dClock (dClk)*      | RNA and protein cycle. bHLH-PAS protein. Heterodimerizes with CYC and promotes transcription from E-box. | *Circadian locomotor output cycle kaput (Clock)* | Constitutively expressed. Heterodimerizes with BMAL 1 and binds to E-box. Promotes transcription of *Per* and *Cry*. |
| *cycle (cyc)*        | Constitutively expressed. bHLH-PAS protein. Heterodimerizes with dCLK and promotes transcription from E-box. | *Bmal 1/MOP 3*         | RNA cycles. Heterodimerizes with CLOCK and binds to E-box. Promotes transcription of *Per* and *Cry*.           |
| *cryptochrome (cry)* | RNA cycles. Circadian photoreceptor. Promotes light-dependent degradation of TIM. Maybe essential for some peripheral clocks.          | *Cryptochrome 1 (Cry 1)* | RNA cycles. Mutations alter rhythmicity in mice, implying a central oscillator function. Physically associates with and stabilizes PER. Inhibits transcription of *Per* and *Cry*. |
|                      |                                  | *Cryptochrome 2 (Cry 2)* |                                                                                                     |
| *vrille*             | RNA and protein cycle. bZIP transcription factor. May repress *per* and *tim* transcription.    | *Nfil3/E4BP 4*         | In chicken, represses *cPer* expression⁷⁸. In mouse, suppresses *mPer1* expression in cell-culture assays⁷⁹. |
| *shaggy*             | Constitutively expressed. Ser/Thr kinase (GSK-3). Promotes TIM phosphorylation and nuclear localization of PER/TIM complex.            |                        |                                                                                                     |

© 2002 Macmillan Magazines Ltd
NATURE | VOL 417 | 16 MAY 2002 | www.nature.com

insight review articles

a
![](https://example.com/image1.png)
d
![](https://example.com/image2.png)

b
WT
![](https://example.com/image3.png)
e
WT (+/+) 
![](https://example.com/image4.png)

c
$per^0$
![](https://example.com/image5.png)
f
$Clock^{+/+}$
![](https://example.com/image6.png)

Figure 1 Assay of circadian activity rhythm in flies and mice. **a**, An infra-red beam and optical sensor automatically detect activity of a single fly placed inside a transparent tube and record it on an activity chart or actogram. Actogram of **b**, individual wild-type fly, and **c**, $per^0$ fly (bearing a loss of function mutation at $per$ locus) recorded over several days. The flies were maintained under a 12-h light:12-h dark cycle for few days and then transferred to complete darkness at the time indicated by a red arrow. The $per^0$ mutation stops the clock and results in arrhythmic activity under constant condition. **d**, Wheel-running activity of individually caged mice gives a measurement of circadian activity rhythm. **e**, Wild-type (+/+) and **f**, mutant mice (+/-) containing one copy of the mutated $Clock$ gene both show similar activity pattern under an entraining light:dark condition. After transfer to constant darkness, the wild-type mouse exhibits a rhythm slightly shorter than 24 h, whereas the mutant mouse has a longer period of activity rhythm. Robustness of the assay enabled detection of this mutant in the original screen. Homozygous $Clock/Clock$ mice behave in a way that is similar to $per^0$ flies under extended darkness.

The $dbt$ gene encodes the *Drosophila* homologue of the mammalian casein kinase Iε (CKIε), and is constitutively expressed [32]. DBT protein associates physically with both PER and PER/TIM complexes, and may phosphorylate PER [35]. Outside the PER/TIM complex, phosphorylated PER is unstable. The interplay among PER, TIM and DBT is critical in understanding some human circadian disorders. In early subjective day when monomeric PER is synthesized in the cytoplasm, DBT binds to PER and promotes its phosphorylation, leading to PER degradation and TIM accumulation. High concentrations of TIM promote formation of a stable DBT/PER/TIM complex that can enter the nucleus during early subjective night. Nuclear DBT/PER/TIM complexes are converted to DBT/PER complexes over a period of ~8–10 h. Concomitantly, progressive repression of $per$ and $tim$ transcription results in decreased accumulation and nuclear entry of the PER/TIM/DBT complex. DBT may progressively phosphorylate PER, leading to its nuclear degradation and contributing to time delays that comprise the circadian transcription-translation feedback loop by either delaying PER accumulation in the cytoplasm or delaying PER’s feedback on its own transcription [35–37]. A second constitutively produced kinase, SGG, promotes TIM phosphorylation, which regulates the timing of nuclear entry of the PER/TIM complex [34]. Finally, VRI — a basic leucine zipper (bZIP) transcription factor — cycles in the

a
![](https://example.com/image7.png)
- Dorsal neuron group 1 (DN1)
- DN2
- DN3
- Dorsal lateral neuron
- Small ventral lateral neuron
- Large ventral lateral neuron

b
![](https://example.com/image8.png)
Retina
Optic nerve

c
![](https://example.com/image9.png)
LV
SCNvl
SCNdm
opt

Figure 2 Schematic diagrams showing anatomic features of *Drosophila* and rodent central oscillator. **a**, Diagram of *Drosophila* brain showing the individual neurons expressing the *period* and *timeless* genes, as originally illustrated in ref. 80. Cell bodies are represented as circles, and neurites as lines. The number of circles represents the number of cell bodies, except for DN1 and DN3 (~15 and 40, respectively). Reprinted by permission of Wiley-Liss, Inc. **b**, Longitudinal view of the mouse brain illustrating input pathways to the SCN and outputs from the SCN. Light input from the retina may reach the SCN directly (above the optic chiasm (ox)) via the retinohypothalamic tract (RHT) or indirectly via the intergeniculate leaflet (IGL) of the lateral geniculate nucleus. Brain regions receiving projections from the SCN include subparaventricular zone of the hypothalamus (sPVz), dorsomedial nucleus of the hypothalamus (DMH), paraventricular nucleus of the thalamus (PV), and arcuate hypothalamic nucleus (Arc). These regions in turn mediate many aspects of circadian behaviour and physiology. **c**, Based on differences in morphology, afferent inputs and output projections, the SCN can be divided into the ventrolateral part of the SCN (SCNvl), or ‘core’ SCN, and the dorsomedial part of the SCN (SCNdm), or ‘shell’ SCN. See ref. 81 for an exhaustive review of SCN structure and function.

same phase as PER and TIM and has been implicated as a repressor of $per$ and $tim$ in clock function [33].

Contrary to its name, CYC in the fly does not cycle with any detectable amplitude at the RNA or protein level [38]. dCLK, however, does cycle with a phase almost opposite to that of PER and TIM [39]. PER has some role in promoting $dClk$ transcription, constituting another feedback loop [40], although the nature of transcriptional regulation of $dClk$ remains a subject of investigation.

Mammalian clockworks

The fundamental anticipatory and light-responsive properties of the circadian pacemaker are conserved between flies and rodents, raising the possibility that the underlying timekeeping mechanism might also be conserved. The first substantial genetic support for this came from a fortuitously isolated rhythm mutant, $tau$, in the golden hamster, which exhibited a short period rhythm in its wheel-running activities [41]. The identification of this mutant provided the first genetic tool for the anatomic definition of the circadian pacemaker in a manner similar to that in *Drosophila*. Studies from the early 1970s showed that ablation of the suprachiasmatic nucleus (SCN) — a bilateral pair of hypothalamic nuclei located just above the optic chiasm (Fig. 2) — resulted in

NATURE | VOL 417 | 16 MAY 2002 | www.nature.com
© 2002 Macmillan Magazines Ltd

insight review articles

![](image.png)

Figure 3 *Drosophila* and mammalian circadian clock. **a**, In *Drosophila*, a heterodimer of two bHLH-PAS domain-containing transcription factors, dCLK and CYC, binds to the E-box in *per* and *tim* promoters, promoting their transcription. DBT phosphorylates (Ph) cytoplasmic TIM-free PER protein (P) and triggers its degradation. As TIM (T) progressively accumulates, it binds to PER, prevents DBT activity, and stabilizes PER. Shaggy (S) phosphorylates TIM, and times the nuclear entry of the PER/TIM/DBT complex. TIM is later released from the nuclear PER/TIM/DBT complex, allowing repression of dCLK/CYC function and starting a new wave of transcription from the E-box. dCLK constitutes another feedback loop by repressing its own transcription. PER promotes *dClk* transcription, although the transcription factor(s) that binds to the *dClk* promoter is currently unknown (X). **b**, In mammals, a heterodimer of two bHLH-PAS domain-containing transcription factors, CLOCK and BMAL1, binds to the E-box in *Per* and *Cry* promoters, and promotes their transcription. CK1ε phosphorylates cytoplasmic PER protein (P) and triggers its degradation. Three different mammalian PER proteins can bind to two mammalian CRY proteins and translocate into the nucleus; here CRY strongly represses CLOCK/BMAL1 activity, whereas PER promotes *Bmal1* transcription. *Bmal1* levels cycle, and the factor(s) that binds to its promoter is currently unknown (Y). Dotted lines represent delays; '?' denotes uncertainty in the step.

complete arrhythmicity of locomotor activity in the rodent<sup>42</sup>. When SCNs from *tau* mutants were transplanted into wild-type hamsters with surgically ablated SCNs, the recipients adopted the short period characteristic of the *tau* mutation. Finally, SCN neurons in culture were shown to have persistent circadian rhythms in their spontaneous firing rate, extending several weeks in culture<sup>43</sup>.

A combination of forward mutagenesis screening in mice and the use of sequence comparisons with known components of the fly clock has produced a picture of the functional clock in mammals that is highly similar to that in flies (Fig. 3 and Table 1). The most similar components are CLOCK (circadian locomotor output cycle kaput) and BMAL1/MOP3, which are mammalian orthologues of fly dCLK and CYC, respectively. CLOCK and BMAL1/MOP3 were shown to heterodimerize, bind the E-box element (functionally conserved between flies and mammals), and transactivate mammalian genes that harbour this element<sup>31</sup>. The *Clock* mutant (a splice-site mutation resulting in a deletion of a portion of the transactivation surface) reduces *mPer* expression and lengthens the overt activity rhythm (eventually turning arrhythmic)<sup>44</sup>, whereas a loss-of-function *Bmal1/MOP3* mutant abolishes *mPer* expression and eliminates activity rhythms altogether<sup>45</sup>. Mutation of two of the three PER orthologues, *mPer1* and *mPer2*, results in aberrant circadian activity, and the double mutant abolishes rhythmicity<sup>46,47</sup>.

Although the clock components are conserved across species, their genetic and biochemical roles have diverged. For example, in the mouse mPER2 seems to activate transcription of *Bmal1*, and exactly opposite to that in flies, BMAL1 cycles in mice whereas CLOCK does not. Therefore, PER positively regulates the rhythmic production of CLOCK/BMAL1 complexes in both mice and flies, although its target has switched. Finally, PER protein products have been shown to weakly suppress CLOCK/BMAL1-dependent *mPer1* transcription in cultured mammalian cells<sup>44</sup>. These results would seem to support a role very similar to that seen for PER (PER/TIM complex) in *Drosophila*, as a negative regulator of its own transcription and a positive regulator of the dCLK/CYC complex. The putative orthologue of *Drosophila timeless*, *mTim*, was found to be a closer orthologue of a second fly gene, *timeout*, which is apparently not involved in maintenance of circadian rhythmicity. Instead, deletion of the *mTim* gene in the mouse causes lethality<sup>48</sup>. Finally, better repressors of CLOCK/BMAL1 molecular activity were isolated in the orthologues of a *Drosophila* photoreceptor called cryptochrome (CRY; see below).

### Resetting the clock

The circadian clock is sensitive to the timing of light exposure. During the middle of subjective day, when light is expected, it has no effect on phase. However, a light pulse administered around subjective dusk (or early night) causes a phase delay, whereas a light pulse near subjective dawn (or early morning) causes a phase advance. This differential sensitivity to light is known as the phase response curve, and is a hallmark of clock function across species. In nature, this property allows the clock to function as a timing device to measure day length, enabling organisms to synchronize their physiology with changing seasons (it also enables jet travellers to adjust to new timezones).

Fly research has provided clues as to how clock resetting occurs in other animals. Stability of TIM protein is light sensitive — even a brief light pulse can trigger its degradation — and this change in TIM level can reset the molecular clock and result in resetting of activity rhythm<sup>49-51</sup>. Because TIM protein does not possess any chromophore-binding site, the initial step of light perception must be mediated by a photoreceptor. Early experiments showed that rhodopsin is not the circadian photoreceptor in flies, as depletion of the rhodopsin chromophore (vitamin A)<sup>52</sup> or presence of the *norpA* mutation<sup>53</sup> has no effect on entrainment to an external light-dark cycle. Similarly, vitamin A-depleted mice<sup>54</sup> or mice bearing mutations in visual pathways also exhibit intact circadian entrainment. Even studies in humans have shown that many patients with no significant perception of light as a result of retinal diseases still retain circadian responses to light<sup>55</sup>. Thus, circadian photoperception may have evolved to use a separate mechanism and perhaps separate photoreceptor(s) to filter out weak light stimuli, such as lightning and moonlight, which would otherwise mimic weak light conditions such as dawn and dusk.

A genetic screen for altered rhythmicity identified a *Drosophila* mutant with a light-resetting defect. Designated *cry<sup>baby</sup>*, the mutant exhibits a normal activity rhythm, entrains properly to a 12-h light:12-h dark cycle, but, unlike wild-type flies, does not phase shift in response to a brief 10-min light pulse during subjective night. Most important, TIM levels are also insensitive to light pulse<sup>56</sup>. Molecular cloning of *cry<sup>baby</sup>* and subsequent characterization revealed the *dCry* gene encodes a protein that shares extensive sequence similarity with a previously known class of plant circadian photoreceptors, the cryptochromes. The *cry<sup>baby</sup>* mutation itself corresponds to a highly conserved position in a putative flavin-binding site identified by sequence homology searches<sup>57</sup>.

Biochemical characterization of dCRY showed that it interacted physically with TIM in the yeast two-hybrid system<sup>58</sup>, promoting phosphorylation, ubiquitination and subsequent degradation of TIM by the proteasome<sup>59</sup>. This light-mediated degradation of TIM was shown recently to be dependent on redox activity associated with flavin<sup>60</sup>. Thus, during early subjective night when TIM protein levels rise, light-induced TIM degradation promoted by dCRY delays the accumulation of TIM, which in turn delays the subsequent molecular events of the oscillator machinery, resulting in a phase delay. Conversely, light pulses administered during the late night, when TIM levels are decreasing, facilitates the rapid decline in TIM protein, and causes phase advances. This model assumes no central oscillator role of dCRY, although this protein has recently been implicated in such a role in some peripheral tissues in fly<sup>61</sup>.

This clock function of cryptochromes may be conserved in mammals<sup>62,63</sup>. Although dCRY and its interacting partner TIM are not functionally conserved between flies and mammals, their activity in flies elucidated the integration of two simple molecular mechanisms — a feedback loop and a simple light response — to produce a seemingly complex time-of-day-dependent response of circadian behaviour to light. This also establishes a model for circadian photoresponses in mammals. In contrast to the fly, cryptochrome-deficient mice exhibit circadian rhythm defects, but no conclusive light-resetting defect. Mice deficient in either *mCry1* or *mCry2* exhibit altered period length, and double-mutant mice are completely arrhythmic under constant darkness<sup>64</sup>. Molecular properties of mCRY mirrors fly TIM — cryptochrome mRNA and protein cycle in phase with mPER1 expression in the SCN and retina, mCRY and mPER interact, mCRY proteins inhibit CLOCK/MOP3 transactivation, and double-mutant mice accumulate elevated levels of mPER2, suggesting that the cryptochromes are repressors of mPER expression<sup>62</sup>. In short, cryptochromes have taken over the role of TIM in the mammalian pacemaker.

If CRY took up TIM's job in mice, who is doing CRY's job? The circadian photoreceptor in mammals is yet to be discovered, although the action spectra of circadian resetting support an opsin-based photoreceptor<sup>65</sup>. Genetic analyses have ruled out necessary circadian photoreceptor functions of rod or cone opsins in mammals, although they may have some redundant roles (reviewed in ref. 66).

---

### Clocks at the protein level

For a transcription-translation loop to generate a sustained rhythm, both RNA and protein products of oscillator components must undergo a controlled and rapid degradation. The effect of gene dosage on period length for cycling clock components such as *per* (ref. 8) and constant components such as *Clock* (ref. 67) suggested that tight regulation in their steady-state expression levels is required to generate and sustain a precise rhythm. Early studies showed that the PER and TIM protein products were progressively phosphorylated before degradation. *Drosophila* TIM provided the first clue of how this degradation took place. Pharmacological and *in vitro* studies suggested light-induced tyrosine phosphorylation of TIM is followed by its ubiquitination (a tag for proteasomal degradation)<sup>59</sup>. Identification of the serine/threonine protein kinases DBT and SGG indicated that phosphorylation is essential to trigger degradation of PER and TIM during the circadian cycle. Kinase mutants exhibited

---

### insight review articles

accumulation of the hypophosphorylated form of PER<sup>32</sup> or TIM<sup>34</sup>. The subsequent steps in degradation of PER and TIM are currently unknown. Phosphorylation of other clock components has been reported, but the respective kinases are yet to be discovered<sup>39,63</sup>.

#### Transcriptome analysis and behaviour

Although the molecular mechanism by which the central oscillator controls timekeeping is becoming increasingly clear, knowledge of how timing information is transmitted to regulate behaviour and physiology is only just emerging. A common theme in connecting the clock to physiological outputs has been the identification of cycling component(s), followed by molecular genetic and histological tests to establish a connection. Using this paradigm, two clock-controlled genes, *lark* and *pdf* (whose protein levels, but not RNA, oscillate), were shown to be key mediators of eclosion and activity rhythms in flies<sup>68,69</sup>. However, how the central oscillator controls rhythmic accumulation of a protein at the post-transcriptional level is entirely unknown.

Modern genomics tools are increasingly important in identification of the transcriptional outputs of the circadian clock. Early success came in a differential display screen for genes expressed in wild-type and clock-deficient flies. This screen identified a gene called *takeout*, whose mRNA is coincidentally expressed with TIM and encodes a lipophilic, ligand-binding protein. The gene is acutely induced in response to starvation in feeding-related organs in insects, and therefore may be important in establishing circadian feeding behaviour<sup>70</sup>. A similar genomics approach identified the *vri* clock component<sup>33</sup>.

These successes have encouraged systematic analysis of the circadian pattern of gene expression in different tissue types and genotypes of *Arabidopsis*, flies, mice and rats using DNA microarrays<sup>71–76</sup>. In each organism, the temporal gene-expression data sets detected cycling of hundreds of transcripts, many times more than previously identified. The list includes already known clock-controlled genes (thus validating the approach), candidate cycling genes involved in known cycling pathways and processes, new processes under circadian control, and key regulators orchestrating coordination of clock-controlled processes. Genomic characterization of mutant flies lacking an essential clock component abolished cycling of all clock-controlled genes, conclusively demonstrating the existence of only one central molecular oscillator in animals<sup>73,74</sup>.

The scenario seems more complex when we compare the cycling gene sets in different tissue types. Many genes that cycle in fly head do not cycle in the body and vice versa<sup>75</sup>. Similar comparison in more defined tissue types, such as mouse SCN and liver, reveals that most cycling transcripts are tissue specific, implying that circadian transcriptional output functions to temporally regulate physiology to a specific tissue or cell type<sup>76</sup>.

#### Complex regulation of gene expression

How are different phases of rhythmic gene expression generated from primarily two principal phases of the central clock? A computational approach analysing promoter regions of coordinately regulated transcripts in *Arabidopsis* has identified a *cis*-acting element that specifies the evening phase of cycling<sup>71</sup>. Molecular genetic analysis of this element has not only supported its key role in phase determination, but also identified cycling transcription factors binding to it<sup>77</sup>. In flies and mammals, an E-box promoter element has been implicated in rhythmic expression of *per* (and *tim*) phased genes. This E-box element is enriched in the 5′-upstream region of some genes cycling in phase with *per*<sup>73,74</sup>, indicating that cycling of E-box-containing genes may be controlled directly by the clock components. Both of these studies focus on the regulation of a small subset of circadian output genes, leaving the mechanism of clock-controlled mRNA expression at other phases unknown.

Overall, analysis of the circadian transcriptome is bringing many new challenges to the forefront of research. Which molecular clock outputs participate in transmitting timing information from the

insight review articles

---

**Box 1**

**Mutant flies and the genetics of human sleep disorders**

A popular chronicler<sup>82</sup> wrote of the *per<sup>S</sup>* mutants in flies, “they woke up about five hours too early; they did the same thing for the rest of their lives.” The *tau* mutant in hamster behaves similarly — the mutants wake up 2–4 hours early. The mutation was first narrowed down to a chromosomal region in hamster, and the respective mouse chromosome was found to contain a gene similar to fly *dbt*, *Ck1ε*, which was later shown to harbour a mutation<sup>83</sup>. Molecular threading of CK1ε on the crystal structure on the closely related kinase CK1δ suggests that the *tau* mutation causes a substitution in a highly conserved amino-acid residue involved in specific recognition of an anionic amino acid in target proteins. Similar to its fly counterpart, mammalian CK1ε interacts physically with and phosphorylates mammalian PER both *in vitro* and *in vivo*, and this phosphorylation destabilizes the PER protein. The *tau* mutation diminishes phosphorylation of PER, and therefore enhances PER’s stability<sup>83-85</sup>.

One human sleep disorder, familial advanced sleep phase syndrome (FASPS), is characterized by circadian rhythms resembling those of *per<sup>S</sup>* flies and *tau* hamsters. FASPS was linked to a single-gene mutation on human chromosome 2q in one large kindred. Sequence analysis of the candidate gene *hPer2*, resident in this chromosomal region revealed that the mutation<sup>86</sup>, a serine-to-glycine amino-acid substitution at residue 662, was a putative CK1ε recognition site in *hPer2*. This observation almost instantaneously elicited a testable hypothesis. CSNK1ε (RefSeq name of human CK1ε) may phosphorylate hPER2, resulting in its instability. A mutation in *hPer2* leading to reduced phosphorylation by CSNK1ε may result in its stabilization, which in turn may phase advance the clock, perhaps by activating *Bmal1* transcription.

---

central pacemaker cells in the lateral neurons in flies or SCN in mammals to other brain regions and peripheral organs? How does the circadian clock function to establish the spatiotemporal pattern of gene expression? How are multiple phases of gene expression generated?

---

**Conservation in clock-controlled processes**

Annotation of the genes under circadian control reveals the potentially adaptive functions of circadian rhythms that are well conserved through evolution. For example, the circadian oscillator synchronizes the consolidation of feeding behaviour to the activity (wake) phase. Clock regulation of transporters that channel nutrients intracellularly and rate-limiting enzymes of nutrient-utilization pathways occurs in flies and mice, thus coordinating the expression of proteins needed for efficient digestion to the time of day in which feeding occurs. Intermediate products of nutrient metabolism also supply important precursor molecules, which are more fully utilized owing to clock-controlled regulation of enzymes that convert intermediates to their final bioactive form (cholesterol to testosterone, for example). Finally, the act of feeding exposes an organism to various xenobiotics and pathogens. Clock regulation of several intermediate metabolic pathways, which inactivate and promote excretion of several xenobiotics and degradation products of endobiotics, may be an underlying defence mechanism against this chemical stress, initiated secondarily by the feeding behaviour<sup>72-76</sup>.

A pattern is emerging whereby evolutionary conservation in clock regulation of a specific physiology results from the regulation of key rate-limiting enzymes. For example, the rate-limiting enzymes in the biosynthesis of cholesterol, haem and bioactive amines exhibit circadian rhythms in mRNA accumulation in both flies and mice<sup>72-74</sup>. The rate-limiting nature of these proteins, coupled with circadian regulation of their transcription, may be an

adaptive mechanism suited for anticipated circadian rhythms in substrate availability or demand for the end product.

The evolutionary conservation in clock regulation of physiology has encouraged researchers to pursue flies as a model system to interrogate the temporal component of learning, alertness and sleep in mammals. Many clock-regulated genes in fly head also have their mammalian counterparts cycling in rodent SCN. The functional significance of such circadian regulation in nervous system and behaviour can be rapidly tested only in a model organism like the fly. For example, transcription of a calcium-activated large potassium channel, *slowpoke*, and its associated protein, slowpoke-binding protein, are clock controlled in flies. The mammalian homologue of *slowpoke*, *mSlo*, also exhibits circadian transcription of its mRNA, and importantly, its expression is enriched in the SCN. The availability of fly mutant stocks and quantitative phenotypic assays established the role of *slowpoke* in regulation of locomotor activity, as *slowpoke*-deficient flies are arrhythmic with no apparent reduction in total activity<sup>75</sup>. If fly-mammal circadian history is our guide, we may yet discover that *mSlo* is an important regulator of activity rhythms in mammals.

---

### From flies to humans

The study of chronobiology in the fly offers an excellent example of how a model organism can facilitate deciphering of the underlying molecular mechanism for a complex trait like the sleep-wake rhythm in humans (Box 1). The rapid progress in this field can justly be attributed to the focus of researchers on the underlying mechanism for generating an overt rhythm (and not the overt rhythm itself). Much of the future of circadian research should focus on connecting the central oscillator to circadian behaviour and physiology. Identification of cycling transcripts has just begun that process, and will be extended by the study of protein and small-molecule rhythms as enabling technologies emerge. The integration of these data will enable a more complete picture of the maintenance of circadian physiology and behaviour. The breadth and depth of circadian regulation now seems to present the perfect example of systems-level biology where a molecular oscillator ticking in a few key neurons in the brain orchestrates a large number of molecules in multiple tissues to generate overt behavioural rhythms.

Mapping the newly identified clock outputs to specific brain regions in flies and connecting them to the master oscillator in lateral neurons will be a step towards understanding how the master oscillator signals to peripheral tissues. Parallel progress in other branches of fly and mammalian neurobiology may help associate a given clock output with a specific clock-controlled behaviour, such as olfaction or feeding. Neuroendocrine signalling is emerging as an important component in the systemic control of clock functions. The systems-level orchestration of circadian physiology is already generating testable hypotheses at a rate that far exceeds current methods to test them in mammals. Model organisms such as the fly offer readily available genetic and genomic tools, rapid generation (or acquisition from public stock centres) of mutants, RNA interference technologies, and automated, quantitative phenotypic assays to rapidly sift through these hypotheses. The exciting possibility that complex behaviours can be described at the molecular level, and are well conserved across species, underscores the importance of the use of model organisms and comparative behavioural genomics.

---

1. Pittendrigh, C. S. Circadian systems. I. The driving oscillation and its assay in *Drosophila pseudoobscura*. *Proc. Natl Acad. Sci. USA* **58**, 1762–1767 (1967).
2. Shaw, P. J., Cirelli, C., Greenspan, R. J. & Tononi, G. Correlates of sleep and waking in *Drosophila melanogaster*. *Science* **287**, 1834–1837 (2000).
3. Hendricks, J. C. *et al.* Rest in *Drosophila* is a sleep-like state. *Neuron* **25**, 129–138 (2000).
4. Konopka, R. J. & Benzer, S. Clock mutants of *Drosophila melanogaster*. *Proc. Natl Acad. Sci. USA* **68**, 2112–2116 (1971).
5. Handler, A. M. & Konopka, R. J. Transplantation of a circadian pacemaker in *Drosophila*. *Nature* **279**, 236–238 (1979).
6. Bargiello, T. A., Jackson, F. R. & Young, M. W. Restoration of circadian behavioural rhythms by gene transfer in *Drosophila*. *Nature* **312**, 752–754 (1984).
7. Reddy, P. *et al.* Molecular analysis of the period locus in *Drosophila melanogaster* and identification of a transcript involved in biological rhythms. *Cell* **38**, 701–710 (1984).

---

334

© 2002 Macmillan Magazines Ltd

NATURE | VOL 417 | 16 MAY 2002 | www.nature.com

8. Baylies, M. K., Bargiello, T. A., Jackson, F. R. & Young, M. W. Changes in abundance or structure of the *per* gene product can alter periodicity of the *Drosophila* clock. *Nature* **326**, 390–392 (1987).

9. Ewer, J., Frisch, B., Hamblen-Coyle, M. J., Rosbash, M. & Hall, J. C. Expression of the period clock gene within different cell types in the brain of Drosophila adults and mosaic analysis of these cells' influence on circadian behavioral rhythms. *J. Neurosci.* **12**, 3321–3349 (1992).

10. Vosshall, L. B. & Young, M. W. Circadian rhythms in Drosophila can be driven by period expression in a restricted group of central brain cells. *Neuron* **15**, 345–360 (1995).

11. Colot, H. V., Hall, J. C. & Rosbash, M. Interspecific comparison of the period gene of Drosophila reveals large blocks of non-conserved coding DNA. *EMBO J.* **7**, 3929–3937 (1988).

12. Reppert, S. M., Tsai, T., Roca, A. L. & Sauman, I. Cloning of a structural and functional homolog of the circadian clock gene period from the giant silkmoth *Antheraea pernyi*. *Neuron* **13**, 1167–1176 (1994).

13. Sauman, I. & Reppert, S. M. Circadian clock neurons in the silkmoth *Antheraea pernyi*: novel mechanisms of Period protein regulation. *Neuron* **17**, 889–900 (1996).

14. Regier, J. C. *et al.* Evolution and phylogenetic utility of the period gene in Lepidoptera. *Mol. Biol. Evol.* **15**, 1172–1182 (1998).

15. Toma, D. P., Bloch, G., Moore, D. & Robinson, G. E. Changes in period mRNA levels in the brain and division of labor in honey bee colonies. *Proc. Natl Acad. Sci. USA* **97**, 6914–6919 (2000).

16. Levine, J. D., Sauman, I., Imbalzano, M., Reppert, S. M. & Jackson, F. R. Period protein from the giant silkmoth *Antheraea pernyi* functions as a circadian clock element in Drosophila melanogaster. *Neuron* **15**, 147–157 (1995).

17. Siwicki, K. K., Strack, S., Rosbash, M., Hall, J. C. & Jacklet, J. W. An antibody to the Drosophila period protein recognizes circadian pacemaker neurons in Aplysia and Bulla. *Neuron* **3**, 51–58 (1989).

18. Siwicki, K. K., Schwartz, W. J. & Hall, J. C. An antibody to the Drosophila period protein labels antigens in the suprachiasmatic nucleus of the rat. *J. Neurogenet.* **8**, 33–42 (1992).

19. Sehgal, A., Price, J. L., Man, B. & Young, M. W. Loss of circadian behavioral rhythms and *per* RNA oscillations in the *Drosophila* mutant timeless. *Science* **263**, 1603–1606 (1994).

20. Vosshall, L. B., Price, J. L., Sehgal, A., Saez, L. & Young, M. W. Block in nuclear localization of period protein by a second clock mutation, timeless. *Science* **263**, 1606–1609 (1994).

21. Sehgal, A. *et al.* Rhythmic expression of *timeless*: a basis for promoting circadian cycles in *period* gene autoregulation. *Science* **270**, 808–810 (1995).

22. Gekakis, N. *et al.* Isolation of *timeless* by PER protein interaction: defective interaction between *timeless* protein and long-period mutant PER<sup>L</sup>. *Science* **270**, 811–815 (1995).

23. Hoffman, E. C. *et al.* Cloning of a factor required for activity of the Ah (dioxin) receptor. *Science* **252**, 954–958 (1991).

24. Crews, S. T., Thomas, J. B. & Goodman, C. S. The Drosophila single-minded gene encodes a nuclear protein with sequence similarity to the *per* gene product. *Cell* **52**, 143–151 (1988).

25. Huang, Z. J., Edery, I. & Rosbash, M. PAS is a dimerization domain common to Drosophila Period and several transcription factors. *Nature* **364**, 259–262 (1993).

26. Hao, H., Allen, D. L. & Hardin, P. E. A circadian enhancer mediates PER-dependent mRNA cycling in *Drosophila melanogaster*. *Mol. Cell. Biol.* **17**, 3687–3693 (1997).

27. Darlington, T. K. *et al.* Closing the circadian loop: CLOCK-induced transcription of its own inhibitors *per* and *tim*. *Science* **280**, 1599–1603 (1998).

28. Rutila, J. E. *et al.* CYCLE is a second bHLH-PAS clock protein essential for circadian rhythmicity and transcription of *Drosophila period* and *timeless*. *Cell* **93**, 805–814 (1998).

29. Allada, R., White, N. E., So, W. V., Hall, J. C. & Rosbash, M. A mutant *Drosophila* homolog of mammalian *Clock* disrupts circadian rhythms and transcription of *period* and *timeless*. *Cell* **93**, 791–804 (1998).

30. Hogenesch, J. B., Gu, Y. Z., Jain, S. & Bradfield, C. A. The basic-helix-loop-helix-PAS orphan MOP3 forms transcriptionally active complexes with circadian and hypoxia factors. *Proc. Natl Acad. Sci. USA* **95**, 5474–5479 (1998).

31. Gekakis, N. *et al.* Role of the CLOCK protein in the mammalian circadian mechanism. *Science* **280**, 1564–1569 (1998).

32. Price, J. L. *et al.* *double-time* is a novel *Drosophila* clock gene that regulates PERIOD protein accumulation. *Cell* **94**, 83–95 (1998).

33. Blau, J. & Young, M. W. Cycling *vrille* expression is required for a functional *Drosophila* clock. *Cell* **99**, 661–671 (1999).

34. Martinek, S., Inonog, S., Manoukian, A. S. & Young, M. W. A role for the segment polarity gene *shaggy*/GSK-3 in the *Drosophila* circadian clock. *Cell* **105**, 769–779 (2001).

35. Kloss, B., Rothenfluh, A., Young, M. W. & Saez, L. Phosphorylation of PERIOD is influenced by cycling physical associations of DOUBLE-TIME, PERIOD, and TIMELESS in the *Drosophila* clock. *Neuron* **30**, 699–706 (2001).

36. Rothenfluh, A., Young, M. W. & Saez, L. A TIMELESS-independent function for PERIOD proteins in the *Drosophila* clock. *Neuron* **26**, 505–514 (2000).

37. Bao, S., Rihel, J., Bjes, E., Fan, J. Y. & Price, J. L. The *Drosophila double-time* mutation delays the nuclear accumulation of *period* protein and affects the feedback regulation of *period* mRNA. *J. Neurosci.* **21**, 7117–7126 (2001).

38. Bae, K., Lee, C., Sidote, D., Chuang, K. Y. & Edery, I. Circadian regulation of a *Drosophila* homolog of the mammalian *Clock* gene: PER and TIM function as positive regulators. *Mol. Cell. Biol.* **18**, 6142–6151 (1998).

39. Lee, C., Bae, K. & Edery, I. The *Drosophila* CLOCK protein undergoes daily rhythms in abundance, phosphorylation, and interactions with the PER-TIM complex. *Neuron* **21**, 857–867 (1998).

40. Glossop, N. R., Lyons, L. C. & Hardin, P. E. Interlocked feedback loops within the *Drosophila* circadian oscillator. *Science* **286**, 766–768 (1999).

41. Ralph, M. R. & Menaker, M. A mutation of the circadian system in golden hamsters. *Science* **241**, 1225–1227 (1988).

42. Moore, R. Y. & Eichler, V. B. Loss of a circadian adrenal corticosterone rhythm following suprachiasmatic lesions in the rat. *Brain Res.* **42**, 201–206 (1972).

43. Welsh, D. K., Logothetis, D. E., Meister, M. & Reppert, S. M. Individual neurons dissociated from rat suprachiasmatic nucleus express independently phased circadian firing rhythms. *Neuron* **14**, 697–706 (1995).

44. Jin, X. *et al.* A molecular mechanism regulating rhythmic output from the suprachiasmatic circadian clock. *Cell* **96**, 57–68 (1999).

45. Bunger, M. K. *et al.* *Mop3* is an essential component of the master circadian pacemaker in mammals. *Cell* **103**, 1009–1017 (2000).

46. Bae, K. *et al.* Differential functions of mPer1, mPer2, and mPer3 in the SCN circadian clock. *Neuron* **30**, 525–536 (2001).

47. Zheng, B. *et al.* Nonredundant roles of the mPer1 and mPer2 genes in the mammalian circadian clock. *Cell* **105**, 683–694 (2001).

48. Gotter, A. L. *et al.* A time-less function for mouse Timeless. *Nature Neurosci.* **3**, 755–756 (2000).

49. Lee, C., Parikh, V., Itukaichi, T., Bae, K. & Edery, I. Resetting the *Drosophila* clock by photic regulation of PER and a PER-TIM complex. *Science* **271**, 1740–1744 (1996).

50. Myers, M. P., Wager-Smith, K., Rothenfluh-Hilfiker, A. & Young, M. W. Light-induced degradation of TIMELESS and entrainment of the *Drosophila* circadian clock. *Science* **271**, 1736–1740 (1996).

51. Hunter-Ensor, M., Ousley, A. & Sehgal, A. Regulation of the *Drosophila* protein timeless suggests a mechanism for resetting the circadian clock by light. *Cell* **84**, 677–685 (1996).

52. Zimmerman, W. F. & Goldsmith, T. H. Photosensitivity of the circadian rhythm and of visual receptors in carotenoid-depleted *Drosophila*. *Science* **171**, 1167–1169 (1971).

53. Wheeler, D. A., Hamblen-Coyle, M. J., Dushay, M. S. & Hall, J. C. Behavior in light-dark cycles of *Drosophila* mutants that are arrhythmic, blind, or both. *J. Biol. Rhythms* **8**, 67–94 (1993).

54. Thompson, C. L. *et al.* Preservation of light signaling to the suprachiasmatic nucleus in vitamin A-deficient mice. *Proc. Natl Acad. Sci. USA* **98**, 11708–11713 (2001).

55. Czeisler, C. A. *et al.* Suppression of melatonin secretion in some blind patients by exposure to bright light. *N. Engl. J. Med.* **332**, 6–11 (1995).

56. Stanewsky, R. *et al.* The *cry<sup>b</sup>* mutation identifies cryptochrome as a circadian photoreceptor in *Drosophila*. *Cell* **95**, 681–692 (1998).

57. Emery, P., So, W. V., Kaneko, M., Hall, J. C. & Rosbash, M. CRY, a *Drosophila* clock and light-regulated cryptochrome, is a major contributor to circadian rhythm resetting and photosensitivity. *Cell* **95**, 669–679 (1998).

58. Ceriani, M. F. *et al.* Light-dependent sequestration of TIMELESS by CRYPTOCHROME. *Science* **285**, 553–556 (1999).

59. Naidoo, N., Song, W., Hunter-Ensor, M. & Sehgal, A. A role for the proteasome in the light response of the timeless clock protein. *Science* **285**, 1737–1741 (1999).

60. Lin, F. J., Song, W., Meyer-Bernstein, E., Naidoo, N. & Sehgal, A. Photic signaling by cryptochrome in the *Drosophila* circadian system. *Mol. Cell. Biol.* **21**, 7287–7294 (2001).

61. Krishnan, B. *et al.* A new role for cryptochrome in a *Drosophila* circadian oscillator. *Nature* **411**, 313–317 (2001).

62. Kume, K. *et al.* mCRY1 and mCRY2 are essential components of the negative limb of the circadian clock feedback loop. *Cell* **98**, 193–205 (1999).

63. Lee, C., Etchegaray, J. P., Cagampang, F. R., Loudon, A. S. & Reppert, S. M. Posttranslational mechanisms regulate the mammalian circadian clock. *Cell* **107**, 855–867 (2001).

64. van der Horst, G. T. *et al.* Mammalian Cry1 and Cry2 are essential for maintenance of circadian rhythms. *Nature* **398**, 627–630 (1999).

65. Takahashi, J. S., DeCoursey, P. J., Bauman, L. & Menaker, M. Spectral sensitivity of a novel photoreceptive system mediating entrainment of mammalian circadian rhythms. *Nature* **308**, 186–188 (1984).

66. Lucas, R. J. *et al.* Identifying the photoreceptive inputs to the mammalian circadian system using transgenic and retinally degenerate mice. *Behav. Brain Res.* **125**, 97–102 (2001).

67. Antoch, M. P. *et al.* Functional identification of the mouse circadian *Clock* gene by transgenic BAC rescue. *Cell* **89**, 655–667 (1997).

68. Newby, L. M. & Jackson, F. R. A new biological rhythm mutant of *Drosophila melanogaster* that identifies a gene with an essential embryonic function. *Genetics* **135**, 1077–1090 (1993).

69. Renn, S. C., Park, J. H., Rosbash, M., Hall, J. C. & Taghert, P. H. A *pdf* neuropeptide gene mutation and ablation of PDF neurons each cause severe abnormalities of behavioral circadian rhythms in *Drosophila*. *Cell* **99**, 791–802 (1999).

70. Sarov-Blat, L., So, W. V., Liu, L. & Rosbash, M. The *Drosophila takeout* gene is a novel molecular link between circadian rhythms and feeding behavior. *Cell* **101**, 647–656 (2000).

71. Harmer, S. L. *et al.* Orchestrated transcription of key pathways in *Arabidopsis* by the circadian clock. *Science* **290**, 2110–2113 (2000).

72. Grundschober, C. *et al.* Circadian regulation of diverse gene products revealed by mRNA expression profiling of synchronized fibroblasts. *J. Biol. Chem.* **276**, 46751–46758 (2001).

73. McDonald, M. J. & Rosbash, M. Microarray analysis and organization of circadian gene expression in *Drosophila*. *Cell* **107**, 567–578 (2001).

74. Claridge-Chang, A. *et al.* Circadian regulation of gene expression systems in the *Drosophila* head. *Neuron* **32**, 657–671 (2001).

75. Ceriani, M. F. *et al.* Genome-wide expression analysis in *Drosophila* predicts genes controlling circadian behavior. (Submitted)

76. Panda, S. *et al.* Coordinated transcription of key pathways in the mouse by the circadian clock. *Cell* (in the press).

77. Alabadi, D. *et al.* Reciprocal regulation between *TOC1* and *LHY/CCA1* within the *Arabidopsis* circadian clock. *Science* **293**, 880–883 (2001).

78. Doi, M., Nakajima, Y., Okano, T. & Fukada, Y. Light-induced phase-delay of the chicken pineal circadian clock is associated with the induction of cE4bp4, a potential transcriptional repressor of cPer2 gene. *Proc. Natl Acad. Sci. USA* **98**, 8089–8094 (2001).

79. Mitsui, S., Yamaguchi, S., Matsuo, T., Ishida, Y. & Okamura, H. Antagonistic role of E4BP4 and PAR proteins in the circadian oscillatory mechanism. *Genes Dev.* **15**, 995–1006 (2001).

80. Kaneko, M. & Hall, J. C. Neuroanatomy of cells expressing clock genes in Drosophila: transgenic manipulation of the period and timeless genes to mark the perikarya of circadian pacemaker neurons and their projections. *J. Comp. Neurol.* **422**, 66–94 (2000).

81. van Esseveldt, K. E., Lehman, M. N. & Boer, G. J. The suprachiasmatic nucleus and the circadian time-keeping system revisited. *Brain Res. Brain Res. Rev.* **33**, 34–77 (2000).

82. Weiner, J. *Time, Love, Memory* (Vintage Books, New York, 1999).

83. Lowrey, P. L. *et al.* Positional syntenic cloning and functional characterization of the mammalian circadian mutation tau. *Science* **288**, 483–492 (2000).

84. Keesler, G. A. *et al.* Phosphorylation and destabilization of human period 1 clock protein by human casein kinaseIε. *NeuroReport* **11**, 951–955 (2000).

85. Vielhaber, E., Eide, E., Rivers, A., Gao, Z. H. & Virshup, D. M. Nuclear entry of the circadian regulator mPER1 is controlled by mammalian casein kinaseIε. *Mol. Cell. Biol.* **20**, 4888–4899 (2000).

86. Toh, K. L. *et al.* An hPer2 phosphorylation site mutation in familial advanced sleep phase syndrome. *Science* **291**, 1040–1043 (2001).
