
BMC Microbiology

Research article

The methionine salvage pathway in Bacillus subtilis

Agnieszka Sekowska¹ and Antoine Danchin²*

Address: ¹HKU-Pasteur Research Centre, Dexter HC Man Building, 8, Sassoon Road, Pokfulam, Hong Kong, China and ²Génétique des Génomes Bactériens, Institut Pasteur, 28 rue du Docteur Roux, 75724 Paris, France

E-mail: Agnieszka Sekowska - sekowska@hkucc.hku.hk; Antoine Danchin* - adanchin@hkucc.hku.hk

*Corresponding author

Published: 25 April 2002
Received: 4 March 2002
Accepted: 25 April 2002

BMC Microbiology 2002, 2:8

This article is available from: http://www.biomedcentral.com/1471-2180/2/8

© 2002 Sekowska and Danchin; licensee BioMed Central Ltd. Verbatim copying and redistribution of this article are permitted in any medium for any purpose, provided this notice is preserved along with the article's original URL.

---

### Abstract

**Background:** Polyamine synthesis produces methylthioadenosine, which has to be disposed of. The cell recycles it into methionine through methylthioribose (MTR). Very little was known about MTR recycling for methionine salvage in *Bacillus subtilis*.

**Results:** Using *in silico* genome analysis and transposon mutagenesis in *B. subtilis* we have experimentally uncovered the major steps of the dioxygen-dependent methionine salvage pathway, which, although similar to that found in *Klebsiella pneumoniae*, recruited for its implementation some entirely different proteins. The promoters of the genes have been identified by primer extension, and gene expression was analyzed by Northern blotting and *lacZ* reporter gene expression. Among the most remarkable discoveries in this pathway is the role of an analog of ribulose diphosphate carboxylase (Rubisco, the plant enzyme used in the Calvin cycle which recovers carbon dioxide from the atmosphere) as a major step in MTR recycling.

**Conclusions:** A complete methionine salvage pathway exists in *B. subtilis*. This pathway is chemically similar to that in *K. pneumoniae*, but recruited different proteins to this purpose. In particular, a paralogue or Rubisco, *MtnW*, is used at one of the steps in the pathway. A major observation is that in the absence of *MtnW*, MTR becomes extremely toxic to the cell, opening an unexpected target for new antimicrobial drugs. In addition to methionine salvage, this pathway protects *B. subtilis* against dioxygen produced by its natural biotope, the surface of leaves (phylloplane).

---

### Background

The fate of methylthioribose (MTR), the end-product of spermidine and spermine metabolism, as well as of ethylene biosynthesis has not yet been fully explored in most organisms. In *Escherichia coli* this molecule is excreted in the medium [1] while in *Klebsiella pneumoniae* it constitutes the methionine salvage pathway, being metabolized back into methionine [2,3]. In eukaryotic parasites it is also recycled into methionine, presumably through a pathway similar to that in *K. pneumoniae*[4]. In *Bacillus subtilis* we found that MTR is an excellent sulfur source [5] and we unraveled some of the steps involved in its metabolism, which starts from phosphorylation of MTR, mediated by the *MtnK* protein [6].

It has been shown previously that the *ykrW* gene, annotated as similar to ribulose phosphate carboxylase/oxygenases (Rubisco) in the original sequence, has links with sulfur metabolism. Indeed, Henkin and co-workers found that the corresponding coding sequence (CDS) was preceded

BMC Microbiology 2002, 2

http://www.biomedcentral.com/1471-2180/2/8

![Diagram](https://i.imgur.com/yourimageurl.png)

Figure 1  
Location of transposon (Tn10) insertions in the *mtn* region. One insertion was localized 73 bp upstream of the translational start point of the *mtnK* gene [6], four were located into *mtnW* and six into the *mtnY* gene. The insertion situated 353 bp downstream of the *mtnW* translation start point (strain BSHP7064) and one situated 556 bp downstream of the *mtnY* translation start point (strain BSHP7065) are shown in the figure.

by a S-box typical of sulfur metabolism genes in *B. subtilis*[7] and Hanson and Tabita found that two classes of enzymes similar to Rubisco were associated with sulfur metabolism [8]. Interestingly, these authors, working with Chlorobium, which uses the reverse TCA cycle for CO₂ fixation, postulated these proteins as possible precursors of cyanobacteria, then plant Rubisco at an early time of the development of life on Earth.

This raised interesting questions about the origin of this pathway. In particular the YkrW gene origin could have been early in evolution, or resulting from lateral transfer from plants to bacilli. We demonstrate here that proteins YkrUWXYZ are needed for MTR recycling into methionine in *B. subtilis*, while YkrV, an aminotransferase, is probably more specific of methionine transamination, but is dispensable in the present conditions because of the presence of a variety of isozymes (up to nine amino acid transaminases are present in *B. subtilis*).

### Results

#### Transposon insertion mutations and phenotype of inactivated mutants

The MTR analog trifluoromethylthioribose (3F-MTR) is toxic if the methyl sulfur moiety of the molecule is recycled [9]. This molecule was therefore an excellent candidate to explore the steps needed for MTR recycling to methionine. Mutants were obtained by transformation of a wild type strain with a random transposon library, selecting for growth in the presence of 3F-MTR in the presence of sulfate as sulfur source. The mutants were subsequently tested for growth on plates lacking sulfur source but supplemented with MTR: only those that could not grow were retained for further study. In order to ascertain that the resistant phenotype was not coming from secondary mutations but was directly related to the transposon insert, the chromosome DNA was extracted from each putative mutant and back transformed into a wild type strain selecting for the transposon antibiotic marker. The 3F-MTR and MTR phenotypes were subsequently tested and only those mutants that passed the test were retained. The insertion positions of the transposons were then sequenced. As shown in Figure 1 we recovered mutants in several genes located in the close vicinity of each other. One mutant was located at the *mtnK* locus (previously named *ykrT*[6]), four were located into *ykrW* and six into the *ykrY* gene. One clone with transposon insertion into the *ykrW* gene (strain BSHP7064, insertion situated 353 bp downstream of *ykrW* translation start point) and one into the *ykrY* gene (strain BSHP7065, insertion situated 556 bp downstream of *ykrY* translation start point) were retained for further studies.

Using the collection of the *lacZ* transcriptional fusion mutants constructed during the *Bacillus subtilis* functional analysis program ([http://locus.jouy.inra.fr/cgi-bin/genmic/madbase/progs/madbase.operl] and [http://bacillus.genome.ad.jp], [10]) and constructing mutants which were not available in the collection, we tested all genes in the region for their phenotype of growth on MTR as the sole sulfur source. Table 1 displays the results obtained. As we can see, mutants in *mtnK* (previously identified as coding for MTR kinase [6], strain BFS1850), *mtnS* (strain BSHP7010 [6]), *ykrU* (renamed *mtnU*, strain BFS1851), *ykrW* (renamed *mtnW*, strain BSHP7014 allowing the expression of downstream genes) and *ykrY* (renamed *mtnY*, strain BSHP7016) failed to grow on the substrate. In the absence of IPTG *ykrX* (renamed *mtnX*, strain BFS1852) also failed to grow, but it recovered its growth properties when IPTG was added to the medium, suggesting some polar effect of the transposon insertion. The mutant of *ykrZ* (renamed *mtnZ*, strain BFS1853) presented only a

BMC Microbiology 2002, 2

Table 1: Phenotype of gene inactivation in the mtn region.

| Gene name          | Strain    | Growth on MTR as sole sulfur source |
|--------------------|-----------|-------------------------------------|
| Wild type +O2<sup>a</sup> | 168       | normal growth after four days     |
| Wild type -O2<sup>a</sup> | 168       | no growth after four days         |
| mtnS (ykrS)<sup>b</sup>   | BSHP7010  | no growth                         |
| mtnK (ykrT)        | BFS1850   | no growth                         |
| mtnU (ykrU)        | BFS1851   | no growth (numerous revertants)    |
| mtnV (ykrV)        | BSHP7020  | normal growth                     |
| mtnW (ykrW)        | BSHP7014  | no growth                         |
| mtnX (ykrX)        | BFS1852   | normal growth                     |
| mtnY (ykrY)        | BSHP7016  | no growth                         |
| mtnZ (ykrZ)        | BFS1853   | weak residual growth               |

a. See Materials and methods; nitrate was used as an electron acceptor. b. Former gene names are given in brackets.

very weak (residual) growth on MTR, suggesting the presence in the cell of some other enzymatic activity able to partially complement the lack of mtnZ gene product. Disruption of ykrV (renamed mtnV, strain BSHP7020) had no visible effect on growth on MTR as the sulfur source.

### Identification of promoters

Several genes in the region have been shown by Henkin and co-workers to be expressed from promoters regulated by the S-box attenuation system [7]. This is the case of mtnKS and mtnWXYZ transcription units. Some of the genes, however, are not regulated in this way. Expression of the mtnU and mtnV genes is not subject to that regulation since no S-box is present in their leader transcript. As shown in Figure 2A the promoter of mtnU is located 35 nt from the translation start point. Its start was found to lie 5 nt downstream from a putative -10 box identified in the sequence (TTAAAT). Upstream from this box separated by 18 nt is a -35 box (ATGATA) with sequence similar to the consensus sequence TTGACA that is typical of B. subtilis sigmaA-dependent promoters [11].

The promoter of mtnV is located 42 nt upstream from the translation start point. Its start lies 8 nt downstream from a putative -10 box identified in the sequence (TATGAT) separated by 17 nt from -35 box (TTTACT) (see Fig. 2.B). The mtnU and mtnV genes share the same promoter region (94 nt) but are transcribed in divergent orientation from overlapping promoters. Thus, the -10 box of the mtnV promoter is situated between the -10 and -35 boxes of the mtnU promoter and the -10 box of the mtnU promoter is situated between the -10 and -35 boxes of the mtnV promoter.

The mtnKS promoter region is 326 nt long. Its start was found to lie 7 nt downstream from a putative -10 box identified in the sequence (TACCAT) (see Fig. 2.C). Upstream from this box and separated by 18 nt is a -35 box (TTGACA), a typical B. subtilis sigmaA-dependent promoter. Downstream of this promoter lies an S-box regulatory sequence.

Genes mntWXYZ are expressed from two overlapping promoters that are situated in a 195 nt long region. The upstream P1 promoter's start was found to lie 7 nt downstream from a putative -10 box identified in the sequence (GATAAT) separated by 17 nt from a consensus-35 box (TTGACA). The P2 promoter's start was found to lie 7 nt downstream from a putative -10 box identified in the sequence (TAAAAT) upstream from which is a -35 box (ATGGGA) (see Fig. 2.D). This promoter region is also reminiscent of regions recognized by the developmental sigmaH transcriptional regulator. The -10 box of the P1's promoter and the -35 box of P2 are partly overlapping. The relative intensity of the signals indicates that transcription from the P1 promoter is more abundant than from P2 (see Fig. 2.D).

### Transcription organisation of the mtn locus

To further investigate transcription of the mtnVWXYZ genes, RNA synthesis was analyzed by Northern blotting. RNA was extracted from exponentially growing cells, in minimal medium containing either sulfate or methionine as sulfur source. As shown in Figure 3.A, a band of about 1200 nt, corresponding to the expected length of a transcript initiated at the mtnV promoter and terminating near its stop codon, was observed for the mtnV gene probe. An equal intensity of the signal was observed for mtnV transcripts prepared from cells either grown with sulfate or with methionine as sulfur source (lanes 1 and 2, Fig. 3.A).

When mtnW and mtnZ gene specific probes were used, two bands were revealed: one of about 2.5 kb and second of about 3.2 kb (Fig. 3.B and 3.C). The larger band corresponds to the expected length of a transcript initiated at the mtnW promoter and terminating in a stem and loop structure at the end of the mtnWXYZ transcriptional unit. The smaller band can possibly be the result of RNA processing at the end of the S-box regulatory sequence of the 5' extremity of the transcript. The intensity of bands when hybridizing RNA from cells grown with sulfate as sulfur source was higher than when using RNA from cells grown in the presence of methionine (lane 1 and 2 in Fig. 3.B and 3.C).

As shown previously, mtnK and mtnS are expressed as an operon, while mtnU is expressed independently [6].

### Regulatory features

To substantiate the results obtained with RNA analysis and further investigate the expression of genes from the

BMC Microbiology 2002, 2

http://www.biomedcentral.com/1471-2180/2/8

A
T C G A

B
T C G A

C
T C G A

D
T C G A

E
mtnU: tttcatctatttctcaccttctaaaaaatcttgaaATGATAgtcccatcatatatggTTAAATg
        -35                              -10
         aaaaagtaaagaaaaattttgggtgtaaaaag ATG AAA TGG - - -
         +1 mRNA

mtnV: catcttttttaacaccccaattttttcTTTACTtttttcattaaccataTATGATgatggactat
        -35                              -10
         catttcaagatttttttagaaggtgagaatag ATG AAA TTT - - -

mtnKS: aaaaatataTGACAactaactaattacctgtTACCATgttcatcaactgataaattagaat
          -35                          -10
           tgcacaacaagtgaatatattctcttatcgagaggtttggcgaggattttggccttttgaccca
           aaagcaaccgaccgtaattccatttgtgaaatggggcgcttttttcgcgccgagacgctggtc
           tcttaaggcacggtgctaattccattcagatctgatctgagagataagagaggcggacatagat
           gttaagcctccttctctctgagaaggaggcttttttacggccacatattaattaattacataat
           tggaggttatg ATG ATG ATG - - -

mtnW: caaactgaacctgttttacccaaaaaagcaTTGACAagaccttataacgATGGGATAATtcaaat
          -35                          -10
           aatttaTTAAAATctatatttttcttatcaagagacaggcaggggacgagcccgatgaagccccgca
           accgacttataaagcacggtgctaattccttgacgctagcggctgagagagataagattcgggacgaga
           aacgaaacctcttttagacgcgattgcagtttgaagaggttttttgat ATG GAT GAA - - -


Figure 2

Identification of the mtn region promoters by primer extension.

A. Identification of the transcription start site of the mtnU operon. The size of the extended product is compared to a DNA-sequencing ladder of the mtnU promoter region. Primer extension and sequencing reaction were performed with the same primer. The +1 site is marked by an arrow.

B. Identification of the transcription start site of the mtnV gene. The size of the extended product is compared to a DNA-sequencing ladder of the mtnV promoter region. Primer extension and sequencing reaction were performed with the same primer. The +1 site is marked by an arrow.

C. Identification of the transcription start site of the mtnKS gene. The size of the extended product is compared to a DNA-sequencing ladder of the mtnKS promoter region. Primer extension and sequencing reaction were performed with the same primer. The +1 site is marked by an arrow.

D. Identification of the transcription start site of the mtnWXYZ operon. The size of the extended product is compared to a DNA-sequencing ladder of the mtnWXYZ promoter region. Primer extension and sequencing reaction were performed with the same primer. Two +1 sites are marked by arrows.

E. Sequences of the corresponding promoter regions. Promoter sites are in capital letters and underlined (-35 and -10 boxes), and the transcription start sites are indicated by broken arrows (+1).

BMC Microbiology 2002, 2

http://www.biomedcentral.com/1471-2180/2/8

A
1 2
![Image A](image_a.png)
2,800 bp
1,900 bp
1,600 bp
1,000 bp

B
1 2
![Image B](image_b.png)
7,400 bp
5,300 bp
2,800 bp
1,900 bp
1,600 bp

C
1 2
![Image C](image_c.png)
7,400 bp
5,300 bp
2,800 bp
1,900 bp
1,600 bp

Figure 3  
Northern blot analysis of *B. subtilis* 168 *mtnVWXYZ* region. A total of 3 μg of RNA was used. A. Northern hybridization with *mtnV* gene specific probe. RNA corresponding to lane 1 was obtained from a culture grown in minimal medium with sulfate as a sulfur source, and for lane 2 from a culture grown in minimal medium with methionine as a sulfur source. B. Northern hybridization with *mtnW* gene specific probe. RNA corresponding to lane 1 was obtained from a culture grown in minimal medium with sulfate as a sulfur source, and for lane 2 from a culture grown in minimal medium with methionine as a sulfur source. C. Northern hybridization with *mtnZ* gene specific probe. RNA corresponding to lane 1 was obtained from a culture grown in minimal medium with sulfate as a sulfur source, and for lane 2 from a culture grown in minimal medium with methionine as a sulfur source.

---

mtn region, we constructed mutants carrying lacZ transcriptional fusions as well as used some mutants constructed during the functional analysis program (also corresponding to lacZ transcription fusions). Table 2 shows the results obtained with these strains when using sulfate or methionine as sulfur source. The mtnU gene (strain BFS1851, mtnU::lacZ) is expressed constitutively at a fairly low level and its expression is independent of the sulfur source used (62 U/mg of protein in the exponential growth phase in presence of sulfate and 53 U/mg of protein in the exponential growth phase in presence of methionine). In contrast, the mtnV gene (strain BSHP7020, mtnV::lacZ) although expressed in the similar way (constitutive and sulfur source independent expression) is expressed at a significantly higher level (217 U/mg of protein in the exponential growth phase in presence of sulfate and 181 U/mg of protein in the exponential growth phase in presence of methionine).

The genes from the mtnWXYZ transcriptional unit (strains BSHP7014, BFS1852, BSHP7016 and BFS1853 for mtnW::lacZ, mtnX::lacZ, mtnY::lacZ and mtnZ::lacZ, respectively) are expressed in a coordinated and sulfur source-dependent way. The expression of the first gene in the operon (mtnW) is higher than that of the last one (mtnZ) with intermediary values for intermediary genes mtnX and mtnY. This suggests the effect of some transcription polarity during the process of transcription (see Table 2). A 5-fold difference is observed between the expression of the mtnWXYZ genes in the presence of sulfate and that in the

presence of methionine (579 U/mg of protein in the exponential growth phase in the presence of sulfate and 113 U/mg of protein in the exponential growth phase in the presence of methionine for the mtnW::lacZ transcriptional fusion and 280 U/mg of protein in the exponential growth phase in presence of sulfate and 57 U/mg of protein in the exponential growth phase in presence of methionine for mtnZ::lacZ transcriptional fusion). This observation is in accordance with the presence of S-box regulatory element in the promoter region of mtnWXYZ operon which modulates gene expression as a function of methionine availability [7].

### Reconstruction of the metabolic pathway

In order to identify the methionine salvage pathway we made constructs allowing us to decipher the order of the gene products in the pathway, together with in silico, physiologic and genetic analysis of the effect of metabolites of the pathway. This is reminiscent of the way advocated by Koonin et al. for the use of in silico approaches as complement to in vivo experiments [12].

As a first goal we showed that the end product of the pathway is indeed methionine. This was demonstrated by showing that MTR, which is a good sulfur source, can be used as the methionine source in methionine auxotrophs (Fig. 4, Fig. 5 and data non shown).

Two genes in the pathway are dispensable, mtnV and mtnX. The first one encodes a transaminase of which there

BMC Microbiology 2002, 2

Table 2: Expression of mtn::lacZ transcriptional fusions.

| Strain | EDI medium with sulfate | EDI medium with methionine |
|--------|-------------------------|-----------------------------|
|        | exp<sup>b</sup>         | stat                       | exp                      | stat                     |
| BFS1851<sup>c</sup> | 62                        | 41                         | 53                        | 33                        |
| BSHP7020          | 217                       | 121                        | 181                       | 161                       |
| BSHP7014          | 579                       | 267                        | 113                       | 95                        |
| BFS1852           | 442                       | 251                        | 108                       | 92                        |
| BSHP7016          | 294                       | 139                        | 61                        | 47                        |
| BFS1853           | 280                       | 112                        | 57                        | 33                        |

a. for the β-galactosidase activity assay the bacteria were grown in the ED minimal medium with either sulfate or methionine as sulfur source. b. exp = exponential growth phase, stat= stationary growth phase. c. BFS1851 = mtnU::lacZ, BSHP7020 = mtnV::lacZ, BSHP7014 = mtnW::lacZ, BFS1852 = mtnX::lacZ, BSHP7016 = mtnY::lacZ, BFS1853 = mtnZ::lacZ.

are nine putative paralogs in the genome of B. subtilis (YwfG, AlaT, AspB, PatA, YhdR, YdfD, PatB, YisV, and HisC). In the same way, MtnX (YkrX) is a member of the phosphatase family pfam00702 ([13], Fig. 6), and therefore of a ubiquitous class of hydrolases (several phosphatase genes in particular are present in the genome of B. subtilis). This is likely to account for the lack of phenotype under our growth conditions. Inactivation of mtnZ provides only a very weak, residual growth on MTR. Inactivation of mtnK, mtnS, mtnY and mtnW result in resistance to 3F-MTR and lack of growth on MTR. Inactivation of mtnW with a polar effect on the distal genes (by insertion of a disrupting plasmid) has a phenotype similar to that of mtnY (i.e. lack of growth on MTR, and lack of influence of MTR on sulfate supplemented plates). In contrast, we discovered that MTR is toxic when the distal genes are present (when used as sole sulfur source or even in the presence of sulfate, see Fig. 7). Because of the weak phenotype of a mtnZ mutant and the absence of phenotype of a mtnX mutant, we can be confident that MtnY acts before MtnW (this is a regular feature in operons, where it is often observed that the more distal genes code for proteins acting in the more proximate steps of the pathway).

The methionine salvage pathway has been deciphered in K. pneumoniae. It is possible, combining this knowledge to the genetic and physiologic results just described, to use it at the basis for reconstructing in silico the corresponding metabolic pathway in B. subtilis. The first steps are similar in both organisms: methylthioadenosine is converted into MTR by a nucleosidase (MtnA, [5]). Subsequently, MTR is phosphorylated into MTR-1-phosphate by MtnK [6]. On the other end of the pathway, methionine is synthesized directly from its keto acid precursor, 2-keto-4-methylthiobutyrate, by a transaminase. MtnV is the likely preferred enzyme for this activity. In K. pneumoniae a dioxygenase is converting 2,3-diketo-5-methylthio-1-phosphopentane into 2-keto-4-methylthiobutyrate [2]. Using dynamic programming (FASTA) we compared the sequence of the corresponding protein to the complete proteome of B. subtilis. YkrZ comes out as the first hit, as the most similar enzyme present in the proteome. Furthermore, it displays a strong consensus similarity with the dioxygenases of the family pfam03079 (Fig. 8) [13]. In order to check whether dioxygen was indeed involved in the case of B. subtilis we grew the cells anaerobically, with nitrate as an electron acceptor, and tested for growth on MTR: while the wild type strain grew well when sulfate was the carbon source, it failed to grow with MTR (Table 1).

Since this dioxygenase is coded in the mtn operon we can infer that it indeed displays the corresponding activity [12], and we therefore renamed it MtnZ. In K. pneumoniae, the immediate precursor activity is that of a coupled phosphatase. The presence of MtnX, which belongs to family pfam00702 comprising phosphatases is strongly suggestive of its involvement at this step [13]. We are thus left with two enzymes, and two steps. We also know, from the genetic data, that the steps are catalyzed in the order Mt nY, MtnW. Finally, the reaction needed upstream of MtnZ is active on a molecule phosphorylated in position 1. MtnW is very similar to ribulosephosphate carboxylase oxygenase (Rubisco). It is therefore likely to be active on a ribulose-1-phosphate derivative. Hence MtnY, which is similar to the araD gene product of E. coli (ribulose-5-

BMC Microbiology 2002, 2

http://www.biomedcentral.com/1471-2180/2/8

Figure 4  
The MTR recycling pathway in B. subtilis.

phosphate epimerase) is most likely to be an epimerase that converts MTR-1-P into 5-thiomethyl-ribulose-1-phosphate, which is the substrate of MtnW. This is strongly supported by the list of similarities found about this gene at the SubtiList database [http://genolist.pasteur.fr/SubtiList].

At this stage it is difficult to explicitly identify the activity of MtnW. Even in the case of the paradigmatic Rubisco, with many crystal structures known, the exact mechanism of catalysis is still a matter of controversy. However we can note (as did [8], at a time when they could not propose a function for the protein) that all the residues involved in catalysis have been conserved, the only residues modified being those involved in the binding of the phosphate at position 5 of ribulose diphosphate. The reaction is that of a dehydratase, but the pathway of the reaction is not yet known. Further work will establish the details of the reaction.

Finally MtnU is also defective for MTR recycling. However, this protein is synthesized at a level much lower than that of the other components of the pathway. We can therefore surmise that it is involved in a regulatory step in the pathway.

### Discussion

Several genes in the vicinity of *mtnK* have been shown to have significant relationships with sulfur metabolism. In particular, it has been known for some time that genes *ykrWXYZ* were preceded by an S-box, typical of sulfur mediated regulation [7]. In addition, while analyzing the function of ribulose-1,5-diphosphate carboxylase (Rubisco), Hanson and Tabita discovered a class of highly related enzymes that were involved in sulfur metabolism [8].

Allowing cells to grow in the presence of the toxic MTR analog 3F-MTR, resistant mutants were found in genes *mtnK*, *mtnW* and *mtnY*. The trifluoromethyl group of 3F-MTR is the toxic group of the molecule and it was expected that

BMC Microbiology 2002, 2

http://www.biomedcentral.com/1471-2180/2/8

A
![Image Description](image_A.png)

B
![Image Description](image_B.png)

C
![Image Description](image_C.png)

D
![Image Description](image_D.png)

Figure 5

Growth of mutants from the mtn region with MTR as sole sulfur source. Panel A: EDI minimal medium plate with 1 mM IPTG containing 0.2 mM MTR as sole sulfur source WT, metl (BSIP1143), mtnS (BSHP7010), mtnK (BFS1850), mtnU (BFS1851), mtnV (BSHP7020), mtnW (BSHP7014), mtnX (BFS1852), mtnY (BSHP7016) and mtnZ (BFS1853) were inoculated for over-night growth at 37°C. No growth of mtnS, mtnK, mtnW and mtnY is represented by an example of absence of growth around a disc with MTR of mtnY mutant in panel B. Normal growth of mtnV and mtnX is represented by an example of normal growth around a disc with MTR of mtnV mutant in panel C. The partial growth of the mtnZ mutant is illustrated by its growth around a disc with MTR in panel C. Panel B: The mtnY strain (BSHP7016) was inoculated on EDI minimal medium plate with no added sulfur source. 10 μl of methionine (met) or MTR was put on paper discs and the plate was incubated over-night at 37°C. Panel C: The mtnV strain (BSHP7020) was inoculated on EDI minimal medium plate with no added sulfur source. 10 μl of methionine (met) or MTR was put on paper discs and the plate was incubated over-night at 37°C. Panel D: The mtnZ strain (BFS1853) was inoculated on EDI minimal medium plate with no added sulfur source. 10 μl of methionine (met) or MTR was adsorbed on paper discs and the plate was incubated over-night at 37°C. Methionine was used as a control.

BMC Microbiology 2002, 2

http://www.biomedcentral.com/1471-2180/2/8

pfam 00702
MtnX: 65 EEITSFVLEDAKIREGFREFVAFINEHEIPFYVISGGMDFFVYPLLEGIVEKDRIYCNHASF 122
Cons: 84 GEVLGLITALADKLYPGAREALKALKERGIKLAILTNGDRANAEAVLE---------LLGLADLF 134
MtnX: 123 DNDYIHIDWPHSCKGTCSNQCGCCKPSVIHELSEP----NOYTIMIGDSVTDEAAKLSDL 183
Cons: 135 DVIVDSDDVGVG-KP----------KPEIFLLALERLGVKPEEVLMGDGVNDA PALAAAGM 189

Figure 6
Alignment of MtnX with the consensus of pfam 00702 [http://www.ncbi.nlm.nih.gov/Structure/cdd/cddsrv.cgi?uid=pfam00702&version=v1.54], that includes L-2-haloacid dehalogenase, epoxide hydrolases and phosphatases. Red letters represent identities, blue letters conservative replacements (similarity classes: AGPST, ILMV, FWY, DENQ, HKR). A loop containing a metal (presumably iron, or an iron-sulfur cluster) is likely to be present in MtnX.

inactivation of any gene coding for the steps, including
permeation, that led from MTR to methionine (the ultimate methyl donor, since AdoMet is an essential metabolite for the cell) would result in a resistant phenotype. Remarkably, no permease gene was found, suggesting that MTR enter the cells via several entries. In addition, apart from the mtnKS and mtnWXYZ operons no other genes was found, suggesting that all essential steps for recycling are coded for by these genes (or that other steps are coded for by redundant genes). The first step of the metabolic pathway is phosphorylation of MTR. The last step, presumably, is transamination, with mtnV being the preferred transaminase.

Interestingly, the pathway described in this work, although similar to that found by the pioneering work of Abeles and co-workers, uses an enzyme, MtnW, which is extremely similar to Rubisco but not present in K. pneumoniae or K. oxytoca [14, 15]. While most of the genome sequence of this bacterium is known [http://wit.integratedgenomics.com/GOLD/] no counterpart of MtnW could be found (data not shown). The corresponding activity exists in K. pneumoniae, but no corresponding gene has yet been isolated. As discovered by Hanson and Tabita, MtnW counterparts constitute a special class (class IV) of Rubisco-like enzymes, which are involved in sulfur metabolism: we can presume that they are all part of the methionine salvage pathway in these organisms [8]. Interestingly, the expected reaction required to metabolize 5-thiomethyl-ribulose-1-phosphate is that of a dehydratase that may use a co-factor as a substrate for the reaction [16]. Rubisco, in the presence of carbon dioxide (resp. dioxygen), acts as a carboxylase (resp. dioxygenase) which cleaves the substrate. In the present case we expect that, instead of cleavage, we have maintenance of a five carbon molecule that is dephosphorylated (by MtnX) and subsequently cleaved by dioxygen in the reaction mediated by MtnZ. The counterpart of this activity in other genomes,

including those with a methionine salvage pathway is not known.

As a strong support of this schema, we found counterparts of MtnK and of MtnZ in K. pneumoniae, substantiating the proposed pathway. In this latter organism the counterpart of MtnY is not known, and the corresponding step (opening of the MTR-1-P ring with epimerisation) is not known in any organism yet. MtnY is part of a very wide family of aldolases-epimerases-transketolases and in silico prediction of function alone, at this stage is highly problematic (wrong assignment is frequent for similar functions [17]), but combination with genetic data make the prediction highly probable [12]. We therefore propose that MtnY be used as a basis for annotation of similar gene products. For example in Xylella fastidiosa, gene XF2209 and in Pseudomonas aeruginosa gene PA1683 probably encodes the cognate activity. Noticeably, a counterpart exists in the Human Genome, where a similar pathway operates.

Two gene products are not directly accounted for in the present schema, MtnU and MtnS. MtnU is expressed at a very low level (ten times lower) as compared to MtnW, and this would hardly fit with the expected stoichiometric enzyme concentration usually found in multistep metabolic pathways. In addition, we found in this work that its synthesis is not submitted to any regulation by the sulfur source. Similarly, MtnS, which is highly similar to an eukaryotic translation initiation factor eIF-2B involved in GTP/GDP exchange is a member typical of a class of GTP-dependent regulators. The presence of two regulator molecules in this pathway indicates that it must have an important role in the cell. B. subtilis is likely to strive on the phylloplane. It is therefore regularly submitted to very high local concentrations of oxygen, and we speculate that this pathway, in addition to providing an excellent means to recycle the energy costly methionine, is used as a means to protect the cell against oxygen.

BMC Microbiology 2002, 2 http://www.biomedcentral.com/1471-2180/2/8

Table 3: Bacterial strains and plasmids used in this study.

| Strain or plasmid | Genotype or description                                                                 | Source or reference                     |
|-------------------|----------------------------------------------------------------------------------------|-----------------------------------------|
| **Strains**       |                                                                                        |                                         |
| **Escherichia coli** |                                                                                        |                                         |
| TG1               | KI2supEhsdΔ5thiΔ (lac-proAB) F' [traD36 proA+ proB+ lacIq lacZΔM15]                      | Laboratory collection                  |
| XL1-Blue          | KI2supE44hsdR17recAlendAlgryA46thirelAl lac-F' [proAB+ lacIq lacZΔM15 TnlO(tetR)]         | Laboratory collection                  |
| **Bacillus subtilis** |                                                                                        |                                         |
| 168               | trpC2                                                                                  | [31]                                    |
| BSIP1143          | trpC2metl::spc                                                                         | [32]                                    |
| BSHP7010          | trpC2mtnS::spc                                                                         | [6]                                     |
| BFS1850           | trpC2mtnK::lacZ                                                                        | Functional analysis project^a^[6]      |
| BFS1851           | trpC2mtnU::lacZ                                                                        | Functional analysis project^a^[6]      |
| BFS1852           | trpC2mtnX::lacZ                                                                        | Functional analysis project^a^          |
| BFS1853           | trpC2mtnZ::lacZ                                                                        | Functional analysis project^a^          |
| BSHP7014          | trpC2mtnW::lacZamyE::(pxylmtnXYZ)                                                     | This work                               |
| BSHP7015          | trpC2amyE::(pxylmtnXYZ)                                                               | This work                               |
| BSHP7016          | trpC2mtnY::lacZ                                                                        | This work                               |
| BSHP7020          | trpC2mtnV::lacZ                                                                        | This work                               |
| BSHP7064          | trpC2mtnW::Tn10                                                                        | This work                               |
| BSHP7065          | trpC2mtnY::Tn10                                                                        | This work                               |
| **Plasmids**      |                                                                                        |                                         |
| pIC333            | mini-Tn10 delivery vector, Spc^R^, Ery^R^                                             | [27]                                    |
| pJM783            | cloning vector, Cm^R^, Amp^R^                                                         | [25]                                    |
| pX                | cloning vector, Cm^R^, Amp^R^, pxyl promoter, amyE locus integration                    | [24]                                    |
| pHPP7011          | pJMMtnV::lacZ                                                                          | This work                               |
| pHPP7014          | pJMMtnW::lacZ                                                                          | This work                               |
| pHPP7014bis       | pJMMtnW::lacZ (bla::spcb)                                                             | This work                               |
| pHPP7015          | pXpxylmtnXYZ                                                                           | This work                               |
| pHPP7016          | pJMMtnY::lacZ                                                                          | This work                               |

a. This strain has been constructed in the frame of the EC project for the functional characterization of the genome of *B. subtilis* in Europe. b. *spc* is the spectinomycin resistance gene from *Staphylococcus aureus*.

---

### Conclusions

This work demonstrates that a complete methionine salvage pathway exists in *B. subtilis*. This pathway is chemically similar to that in *K. pneumoniae*, but recruited different proteins to this purpose. In particular, a paralogue or Rubisco, MtnW, is used at one of the steps in the pathway. A major observation stemming from the present experiments is that in the absence of MtnW MTR becomes extremely toxic to the cell. This sensitivity opens an unexpected target for new antimicrobial drugs, since analogs of 5-methylthio-ribulose1-phosphate might have a strong inhibitory effect on growth on bacteria containing this methionine salvage pathway, including *Bacillus anthracis*.

### Materials and methods

#### Bacterial strains and plasmids, and growth media

E. coli and B. subtilis strains as well as plasmids used in this work are listed in Table 3. E. coli TG1 and XL1-Blue were used for cloning experiments (TG1 for single cross-over recombination and XL1-Blue for double cross-over recombination). Despite the fact that there are no public regulations yet in this domain in China, all experiments were performed in accordance with the European regulation requirements concerning the contained use of Genetically Modified Organisms of Group-I (French agreement N° 2735). E. coli and B. subtilis were grown in Luria-Bertani (LB) medium [18] and in ED minimal medium: K₂HPO₄, 8 mM; KH₂PO₄, 4.4 mM; glucose, 27 mM; Na₃-citrate, 0.3 mM; L-glutamine, 15 mM; L-tryptophan, 0.244 mM; ferric citrate, 33.5 μM; MgSO₄, 2 mM; MgCl₂, 0.61 mM; CaCl₂, 49.5 μM; FeCl₃, 49.9 μM; MnCl₂, 5.05 μM; ZnCl₂, 12.4 μM; CuCl₂, 2.52 μM; CoCl₂, 2.5 μM; Na₂MoO₄, 2.48 μM. When methionine was used as sulfur source (1 mM), MgSO₄ was replaced by MgCl₂ at the same magnesium concentration (2 mM). For assaying growth on plates, either the MgSO₄ containing medium or the sulfur-free basal medium was used (MgSO₄ was replaced by MgCl₂ as described above). In the latter case, 10 μl of the sulfur source under investigation was applied onto paper discs (MTR, 200 mM stock solution and methionine, 100 mM stock solution) deposited at the center of the plate, after bacteria had been uniformly spread at the

surface of the plate, and growth was measured around the disk. In some cases MTR was used directly in the plate as sulfur source (0.2 mM). When necessary IPTG was included at 1 mM concentration. When xylose was added to the medium (0.5%) in order to trigger the expression of genes under the control of PxyI inducible promoter, fructose was used as carbon source instead of glucose. LB and ED plates were prepared by addition of 17 g/liter Bacto agar or Agar Noble (Difco), respectively, to the medium. When included, antibiotics were added to the following concentrations: ampicillin, 100 mg/liter; chloramphenicol, 50 mg/liter; spectinomycin, 100 mg/liter; erythromycin plus lincomycin, 1 mg/liter and 25 mg/liter. Bacteria were grown at 37°C. The optical density (OD) of bacterial cultures was measured at 600 nm. MTR was prepared from MTA (Sigma, D5011) by acid hydrolysis as described by Schlenk [19]. 3-fluoromethylthiorybose (3F-MTR, 5-thio-5-S-trifluoromethyl-D-ribose) was synthesised according- ly to [6,20]. For mutant selection, 3F-MTR was used at 100 mg/liter concentration in the ED minimal medium plates containing magnesium sulfate as sulfur source and glucose as carbon source. When applied onto paper discs 10 μl of 100 mM stock solution of 3F-MTR was used. For anaerobic growth on plates, the Anaerocult A (Merck) within an anaerobiosis jar for CO2 production with simultaneous O2 absorption was used. Sulfur-free ED minimal medium plates were supplemented to 1% glucose final concentration and with 0.5% sodium pyruvate and 20 mM sodium nitrate as electron acceptor. Plates were incubated at 37°C for 4 days with the sulfur source under investigation.

**Transformation**

Standard procedures were used to transform *E. coli* [21] and transformants were selected on LB plates containing ampicillin, spectinomycin or ampicillin plus spectinomycin. *B. subtilis* cells were transformed with plasmid DNA following the two-step protocol described previously [22]. Transformants were selected on LB plates containing erythromycin plus lincomycin or spectinomycin or chloramphenicol.

**Molecular genetics procedures**

Plasmid DNA was prepared from *E. coli* by standard procedures [21]. *B. subtilis* chromosomal DNA was purified as described by Saunders [23]. Restriction enzymes and T4 DNA ligase were used as specified by manufacturers.

DNA fragments used for cloning experiments were prepared by PCR using PfuTurbo DNA polymerase (Stratagene). Amplified fragments were purified by QIAquick PCR Purification Kit (Qiagen). DNA fragments were purified from a gel using Spin-X columns from Corning Costar by subsequent centrifugation and precipitation.

---

![](https://via.placeholder.com/500x500.png)

Figure 7

Toxicity of MTR for BSHP7014 strain. Strain BSHP7014 (*mtnW::lacZ amyE::pxyl mtnXYZ*) was grown on ED1 minimal medium plates in the presence of sulfate as sulfur source (agar as sole sulfur source, panel A) or in the absence of any added sulfur source (panel B). Xylose was added to the medium in order to trigger the expression of *mtnXYZ* from the *pxyl* promoter. 10 μl of methionine (met) or MTR was adsorbed on paper discs and plates were incubated overnight at 37°C. Methionine was used as a control for growth and/or toxicity of the sulfur source.

The *mtnXYZ* region (nucleotides -31 relative to the *mtnX* translation start point and ending 3 bp after the stop codon of *mtnZ*) was amplified by PCR using primers introducing a SpeI cloning site at the 5' end and a BamHI cloning site at the 3' end of the fragment. This fragment was then inserted into the SpeI and BamHI sites of xylose-inducible pX plasmid [24] producing plasmid pHPP7015. Prior to transformation, this plasmid was linearized at its unique ScaI site. Complete integration of the plasmid was obtained by a double cross-over event at the *amyE* locus, giving strain BSHP7015.

The DNA downstream from the *mtnW* gene (nucleotides +41 to +257 relative to the translation start point) was amplified by PCR using primers introducing an EcoRI cloning site at the 5' end and a BamHI cloning site at the 3' end of the fragment, then inserted into the EcoRI and BamHI sites of plasmid pJM783 [25] producing plasmid pHPP7014. To introduce an additional antibiotic resistance gene into plasmid pHPP7014, a SmaI restricted spectinomycin resistance cassette [26] was inserted into the ScaI restriction site of the *bla* gene producing plasmid pHPP7014bis. The plasmid in which the *mtnW* gene was disrupted as well as fused (transcriptional fusion) with the *lacZ* gene was introduced into the chromosome of BSHP7015 strain by a single cross-over event, giving strain BSHP7014.

For transcriptional fusion of *mtnY* with the *lacZ* gene, a DNA segment downstream from the *mtnY* gene (nucleotides +57 to +264 relative to the translation start point) was amplified by PCR using primers introducing an EcoRI

BMC Microbiology 2002, 2

http://www.biomedcentral.com/1471-2180/2/8

pfam 03079

MtnZ: 3 TIRIHDEANTTIENQEEVASFLDSQEVIYEQWDITRILPEHLSEKYDLTEEEKQOILDTFE 62  
Cons: 1 QAWIMDDSEC--DQRLPHHTFPPEKAELDELAKLGVLWKLDADDEETAELLRIKYYRN 58  

MtnZ: 63 TEIKDISTRRGYKAQDVISLSDSNPKLDELLLENFKREHHHTDDEVRFIVSGHGIFVIOGQ 122  
Cons: 59 YLDKDI---------DVTVCPETTPNFDEKLKKFFEEHLHTDEEIRYIVEGTGYFVRDK 109  

MtnZ: 123 DGTFFDVRLNP GDLISVPENIRHYFTLQEDRKVAVRIFVTTEGWVP 169  
Cons: 110 DDVWIRLVVEKGDLISLPAGIYHRFTTTPDNFVKALRLFVGKP GWTA 156  

Figure 8  
Alignment of MtnZ with the consensus of pfam 03079 [http://www.ncbi.nlm.nih.gov/Structure/cdd/qrpsb.cgi?RID=1014604213-20481-4181], coding for aci-reductone enzymes. Red letters represent identities, blue letters conservative replacements (classes: AGPST, ILMV, FWY, DENQ, HKR).

cloning site at the 5' end and a BamHI cloning site at the  
3' end of the fragment, then inserted into the EcoRI and  
BamHI sites of plasmid pJM 783 producing plasmid  
pHPP 7016. The plasmid in which the mtnY gene was dis-  
rupted as well as fused (transcriptional fusion) with the  
lacZ gene was introduced into the chromosome by a sin-  
gle cross-over event, giving strain BSHP 7016.

To construct a mtnV transcriptional fusion with the lacZ  
gene, a DNA fragment downstream from the mtnV gene  
(nucleotides +44 to +259 relative to the translation start  
point) was amplified by PCR using primers introducing  
an EcoRI cloning site at the 5' end and a BamHI cloning  
site at the 3' end of the fragment, then inserted into the  
EcoRI and BamHI sites of plasmid pJM 783 producing plas-  
mid pHPP 7011. The plasmid in which the mtnV gene was  
disrupted as well as fused (transcriptional fusion) with the  
lacZ gene was introduced into the chromosome by a sin-  
gle cross-over event, giving strain BSHP 7020.

Within the framework of a European Union and Japanese  
projects for the functional analysis of the genome of B.  
subtilis, more than 2000 genes have been disrupted by fu-  
sion with the lacZ reporter gene ([http://locus.jouy.in-  
ra.fr/cgi-bin/genmic/madbase/progs/madbase.operl] and  
[http://bacillus.genome.ad.jp]). The strains from the col-  
lection used in this study, constructed by Dr S. Krogh, are  
listed in Table 3.

Transposon mutagenesis  
A transposon bank was constructed by introduction of the  
mini-Tn10 delivery vector pIC 333 [27] into the B. subtilis  
168 strain as described previously [28]. Several thousand  
independent clones were pooled together and 5 samples  
of chromosomal DNA were prepared for further use. To  
obtain 3F-MTR resistant clones, B. subtilis 168 was trans-  
formed with chromosomal DNA containing previously  
prepared transposon banks and clones were selected on  

LB plates containing spectinomycin. Then, using velvets  
replicas, clones were transferred onto minimal medium  
plates containing 3F-MTR at 100 μM concentration and  
allowed to grow for 24 hrs. The single transposon inser-  
tion event was confirmed by back-cross into strain 168  
and check for 3F-MTR resistance. To determine the loca-  
tion of the transposon insertion, chromosomal DNA was  
prepared, followed by subsequent digestion with HindIII,  
self ligation in E. coli XL1-Blue strain and plasmid se-  
quencing. The primers used for sequencing of transposon  
insertions were the followings: Tn10 left:  
5'GGCCGATTCATTAATGCAGGG3' and Tn10 right:  
5'CGATATTCACGGTTTACCCAC3'.

RNA isolation and manipulation  
Total RNA was obtained from cells growing on ED1 min-  
imal medium with sulfate or methionine as sulfur source  
to an OD600 of 0.5 using "High Pure RNA Isolation Kit"  
from Roche. The RNA concentration was determined by  
light absorption at 260nm and 280nm. 2 μg of RNA were  
loaded onto 1.2% agarose gel to check the RNA purity and  
integrity.

RNA molecules were separated on 1% agarose gels and  
transferred to nylon membranes (Hybond-N, Amersham).  
Efficiency of transfer was monitored by analysis of ethid-  
ium bromide-stained material. Membranes were prehy-  
bridized at 50°C for 1hr in DIG Easy Hyb buffer from  
Roche. Hybridization was performed under the same con-  
ditions with mtnV, mtnW or mtnZ specific probes using a  
non-radioactive DNA labeling and detection kit "Dig-UTP  
labeling" from Roche.

Primer extension analysis using reverse transcriptase AMV  
(Roche) was performed as described by [29] with two oli-  
gonucleotides for each promoter identification. For  
mtnKS promoter the followings primers were used:  
5'ACCAGCGTCTCGGGCGCGAAAAAAAAATGCCGCCCC3'

and 5'TCACAATGGAATTACGGTCGGTTGCTTTTGG3' (+137 to +169 and +172 to +203 with respect to the translation start point, respectively); for the mtnU promoter the followings primers were used:

5'AGTTCATCAAGATTGGCCCAGATCATATCCG3' and 5'CAGGCAGAAACAAGAATCAGCATGTTIGC' (-133 to -103 and -90 to -60 with respect to translation start point, respectively); for the mtnV promoter the followings primers were used: 5'GTTTCATCTCCTCAACAAATATGCTCAG-GAG' and 5'TCCCAGATTGATAACGTGATGTCCTTCTGC' (-166 to -146 and -114 to -84 with respect to the translation start point, respectively); for the mtnWXYZ promoter the followings primers were used: 5'CGTTTCTCGTC-CGAATTTATCTCTCAGCC' and 5'AGCTGCAAGAATT-AGCACCGTGCTTTATAAG' (+43 to +73 and +76 to +107 with respect to the translation start point, respectively).

The same primers were used for the generation of sequence ladders. Reaction products were separated on 7% denaturing polyacrylamide gel containing 8 M urea. DNA sequences were determined using Sanger's dideoxy chain termination method with "Thermo Sequenase radiolabeled terminator cycle sequencing kit" from Amersham Pharmacia Biotech.

### Enzyme assays

B. subtilis cells containing lacZ fusions were assayed for β-galactosidase activity as described previously [30]. Specific activity was expressed in Units per mg protein. The Unit used is equivalent to 0.28 nmols min-1 at 28°C. Protein concentration was determined by Bradford's method using a protein assay Kit (Bio-Rad Laboratories). At least two independent cultures were monitored.

Amylase activity was detected after growth of B. subtilis strains on Tryptose Blood Agar Base (TBAB, Difco) supplemented with 10 g/liter hydrolyzed starch (Sigma). Starch degradation was detected by sublimating iodine onto the plates.

### Abbreviations

bp: base pairs; CDS: coding sequence; IPTG: isopropyl β-D-thiogalactopyranoside; kb: kilobase; MTA: methylthioadenosine; MTR: methylthioribose; 3F-MTR: trifluoromethylthioribose; nt: nucleotides.

### Authors' contributions

AS carried out the experimental part of the study, discovered the MTR toxic effect in the mtnW and mtnU mutants, drafted and wrote several sections of the manuscript.

AD outlined the rationale for the experiments, carried out the in silico part of the study and wrote the bulk of the manuscript.

Both authors read and approved the final manuscript.

### Acknowledgements

We wish to thank Susan Krogh for the construction of the mtnK, mtnU, mtnX and mtnZ mutants during the Bacillus subtilis functional analysis project, and Sandrine Auger for providing the metI mutant.

### References

1. Schroeder HR, Barnes CJ, Bohinski RC, Mallette MF: Biological production of 5-methylthioribose. Can J Microbiol 1973, 19:1347-1354
2. Dai Y, Pochapsky TC, Abeles RH: Mechanistic studies of two dioxygenases in the methionine salvage pathway of Klebsiella pneumoniae. Biochemistry 2001, 40:6379-6387
3. Heilbronn J, Wilson J, Berger BJ: Tyrosine aminotransferase catalyzes the final step of methionine recycling in Klebsiella pneumoniae. J Bacteriol 1999, 181:1739-1747
4. Riscoe MK, Ferro AJ, Fitchen JH: Analogs of 5-methylthioribose, a novel class of antiprotozoal agents. Antimicrob Agents Chemother 1988, 32:1904-1906
5. Sekowska A, Danchin A: Identification of yrrU as the methylthioribose nucleosidase gene in Bacillus subtilis. DNA Res 1999, 6:255-264
6. Sekowska A, Mulard L, Krogh S, Tse JK, Danchin A: MtnK, methylthioribose kinase, is a starvation-induced protein in Bacillus subtilis. BMC Microbiol 2001, 1:15
7. Grundy FJ, Henkin TM: The S box regulon: a new global transcription termination control system for methionine and cysteine biosynthesis genes in gram-positive bacteria. Mol Microbiol 1998, 30:737-749
8. Hanson TE, Tabita FR: A ribulose-1,5-bisphosphate carboxylase/oxygenase (RubisCO)-like protein from Chlorobium tepidum that is involved with sulfur metabolism and the response to oxidative stress. Proc Natl Acad Sci USA 2001, 98:4397-4402
9. Gianotti AJ, Tower PA, Sheley JH, Conte PA, Spiro C, Ferro AJ, Fitch en JH, Riscoe MK: Selective killing of Klebsiella pneumoniae by 5-trifluoromethylthioribose. Chemotherapeutic exploitation of the enzyme 5-methylthioribose kinase. J Biol Chem 1990, 265:831-837
10. Vagner V, Dervyn E, Ehrlich SD: A vector for systematic gene inactivation in Bacillus subtilis. Microbiology 1998, 144:3097-3104
11. Moran CP: RNA polymerase and transcription factors. In: Bacillus subtilis and Others Gram-Positive Bacteria (Edited by: AL Sonenshein) Washington, D. C.: American Society for Microbiology 1993, 653-667
12. Iyer LM, Aravind L, Bork P, Hofmann K, Mushegian AR, Zhulin IB, Koonin EV: Quod erat demonstrandum? The mystery of experimental validation of apparently erroneous computational analyses of protein sequences. Genome Biol 2001, 2:RESEARCH0051
13. Bateman A, Birney E, Cerruti L, Durbin R, Eddy SR, Grif fiths-Jones S, Howe KL, Marshall M, Sonnhammer EL: The Pfam protein families database. Nucleic Acids Res 2002, 30:276-280
14. Furfine ES, Abeles RH: Intermediates in the conversion of 5'-S-methylthioadenosine to methionine in Klebsiella pneumoniae. J Biol Chem 1988, 263:9598-9606
15. Wray JW, Abeles RH: The methionine salvage pathway in Klebsiella pneumoniae and rat liver. Identification and characterization of two novel dioxygenases. J Biol Chem 1995, 270:3147-3153
16. Myers RW, Abeles RH: Conversion of 5-S-methyl-5-thio-D-ribose to methionine in Klebsiella pneumoniae. Stable isotope incorporation studies of the terminal enzymatic reactions in the pathway. J Biol Chem 1990, 265:16913-16921
17. Sekowska A, Danchin A, Risler JL: Phylogeny of related functions: the case of polyamine biosynthetic enzymes. Microbiology 2000, 146:1815-1828
18. Bertani G: Studies in lysogenesis I. The mode of phage liberation by lysogenic Escherichia coli. J Bacteriol 1951, 62:293-300
19. Schlenk F, Zydék-Cwick CR, Dainko JL: 5'-Methylthioadenosine and related compounds as precursors of S-adenosylmethionine in yeast. Biochimica et Biophysica Acta 1973, 320:357-362
20. Bouchu M, Large S, Steng M, Langlois B, Praly JP: An unprecedented access to trifluoromethylthiosugar derivatives from thiocyanate precursors upon treatment with trifluoromethylsilane. Carbohydr Res 1998, 314:37-45

BMC Microbiology 2002, 2

http://www.biomedcentral.com/1471-2180/2/8

21. Sambrook J, Fritsch E, Maniatis T: Molecular cloning: A laboratory manual. Cold Spring Harbour, New York: Cold Spring Harbour Laboratory Press 1989
22. Kunst F, Rapoport G: Salt stress in an environmental signal affecting degradative enzyme synthesis in Bacillus subtilis. J Bacteriol 1995, 177:2403-2407
23. Saunders C, Schmidt B, Morot M, Thompson L, Guyer M: Use of chromosomal integration in the establishment and expression of blaZ, a Staphylococcus aureus β-lactamase gene, in Bacillus subtilis. J Bacteriol 1984, 157:718-726
24. Kim L, Mogk A, Schumann W: A xylose-inducible Bacillus subtilis integration vector and its application. Gene 1996, 181:71-76
25. Perego M: Integrational vectors for genetic manipulation in Bacillus subtilis. In: Bacillus subtilis and other Gram-positive Bacteria (Edited by: AL Sonenshein, JA Hoch, R Losick) Washington, DC: American Society for Microbiology 1993, 615-624
26. Murphy E: Nucleotide sequence of a spectinomycin adenyltransferase AAD(9) determinant from Staphylococcus aureus and its relationship to AAD(3')(9). Mol Gen Genet 1985, 200:33-39
27. Steinmetz M, Richter R: Easy cloning of mini-Tn10 insertions from the Bacillus subtilis chromosome. J Bacteriol 1994, 176:1761-1763
28. Dartois V, Djavakhishvili T, Hoch JA: KapB is a lipoprotein required for KinB signal transduction and activation of the phosphorelay to sporulation in Bacillus subtilis. Mol Microbiol 1997, 26:1097-1108
29. Piekielny C, Rosbash M: mRNA splicing efficiency in yeast and contribution of non conserved sequences. Cell 1985, 41:119-126
30. Msadek T, Kunst F, Henner D, Klier A, Rapoport G, Dedonder R: Signal transduction pathway controlling synthesis of a class of degradative enzymes in Bacillus subtilis: expression of the regulatory genes and analysis of mutations in degS and degU. J Bacteriol 1990, 172:824-834
31. Spizizen J: Transformation of biochemically deficient strains of Bacillus subtilis by deoxyribonucleate. Proc Natl Acad Sci USA 1958, 44:1072-1078
32. Auger S, Yuen WH, Danchin A, Martin-Verstraete I: The metIC operon involved in methionine biosynthesis in Bacillus subtilis is controlled by transcription antitermination. Microbiology 2002, 148:507-518

---

Publish with **BioMed Central** and every scientist can read your work free of charge

*"BioMed central will be the most significant development for disseminating the results of biomedical research in our lifetime."*

Paul Nurse, Director-General, Imperial Cancer Research Fund

Publish with BMC and your research papers will be:
- available free of charge to the entire biomedical community
- peer reviewed and published immediately upon acceptance
- cited in PubMed and archived on PubMed Central
- yours - you keep the copyright

Submit your manuscript here:  
http://www.biomedcentral.com/manuscript/

editorial@biomedcentral.com
