
The Plant Cell, S111–S130, Supplement 2002, www.plantcell.org © 2002 American Society of Plant Biologists

# Control of Flowering Time: Interacting Pathways as a Basis for Diversity

Aidyn Mouradov,$^{1}$ Frédéric Cremer,$^{1}$ and George Coupland$^{2}$

Max-Planck-Institute for Plant Breeding, Carl-von-Linné-Weg 10, D-50829 Cologne, Germany

## INTRODUCTION

Flowering is controlled by environmental conditions and developmental regulation. The complexity of this regulation is created by an intricate network of signaling pathways. Arabidopsis is an excellent model system in which to approach this complexity, because it responds to many of the environmental conditions that control flowering in other species, and genetic tools are well developed. Studies in Arabidopsis have led to the identification of components within individual signaling pathways that affect flowering, and to their positioning within molecular hierarchies. Furthermore, distinct signaling pathways are known to converge on the activation of the same flowering-time genes. This convergence of pathways on a common set of genes may enable the integration of different responses, so that the plant can produce a coordinated flowering response under conditions in which multiple environmental parameters are changing simultaneously. Also, genetic analysis of Arabidopsis varieties showing natural variation in flowering time has demonstrated how the activity of these pathways can be altered in nature and how balancing the effects of different environmental stimuli on flowering time is important in plants adapting to growth in different geographical locations.

At present, the full complexity of the flowering network can only be approached in Arabidopsis where the necessary tools are available, and extensive efforts are being made to describe related pathways. For example, photoreception, circadian clock regulation, growth regulator synthesis and response, chromatin structure, and response to low temperatures play important roles in flowering-time control and are studied extensively in Arabidopsis. Nevertheless, there is a need to understand how the full diversity in flowering responses is generated. For example, Arabidopsis responds to photoperiod, but all ecotypes are long-day plants that flower earlier under long than short days, whereas many other species show the reverse response. Also, all *Arabidopsis thaliana* ecotypes are annual plants, and understanding the perennial habit will require a different model species. To understand the diversity in flowering responses, there is a need to look to other species, and here we compare detailed information from Arabidopsis with the emerging picture of the genetic control of flowering in short-day plants.

The control of flowering has been reviewed frequently over the last few years (Koornneef et al., 1998b; Simpson et al., 1999; Reeves and Coupland, 2000; Samach and Coupland, 2000; Araki, 2001). However, progress has been rapid, and in the following review we emphasize results that have emerged recently. In particular, we summarize the current understanding of the signaling pathways involved in flowering control in Arabidopsis, and stress how alterations in the balance of activity between pathways can give rise to dramatically different flowering behaviors, for example between ecotypes. We also address recent progress in describing how these pathways function in species that show different responses to environmental conditions than does Arabidopsis.

## PATHWAYS CONTROLLING FLOWERING TIME IN ARABIDOPSIS

### The Photoperiod Response Pathways

One of the most important factors controlling flowering time in temperate regions is the duration of the daily light period, or photoperiod. Arabidopsis is a facultative long-day plant, which flowers earlier under long days but eventually flowers under short days. Under laboratory conditions, Arabidopsis will flower in response to a single long day (Corbesier et al., 1996). Molecular-genetic approaches have identified genes required for the daylength response, and some of these encode regulatory proteins specifically involved in the regulation of flowering, while others encode components of light signal transduction pathways or are involved in circadian clock function. A representation of the relationships among

$^{1}$ These authors contributed equally to this work.

$^{2}$ To whom correspondence should be addressed. E-mail coupland@mpiz-koeln.mpg.de; fax 49-221-5062207.

Article, publication date, and citation information can be found at www.plantcell.org/cgi/doi/10.1105/tpc.001362.

S112 The Plant Cell

![Diagram](attachment:diagram.png)

Figure 1. Signaling Pathways Involved in the Regulation of Flowering by Photoperiod in Arabidopsis.

A diagram showing the putative relationships among genes involved in the photoperiod pathway. On the basis of the phenotypes of known mutants, genes shown in red generally repress flowering, whereas those in green promote it. Small upright arrows indicate the role of the genes as determined by overexpression in transgenic plants. Arrows between genes represent a promotive effect, whereas perpendicular lines represent a repressive effect, and simple lines represent protein–protein interactions. Arrows from the clock indicate that the expression of the gene is circadian clock controlled. Arrows to the clock indicate that the gene lengthens period length, while perpendicular lines indicate that it shortens period length. Numbers between brackets refer to the following publications, which support the indications provided in the diagram. (1) (Somers et al., 1998b) (2) (Swarup et al., 1999) (3) (Wang and Tobin, 1998) (4) (Schaffer et al., 1998) (5) (Mizoguchi et al., 2002) (6) (Goto et al., 1991) (7) (Samach et al., 2000) (8) (Onouchi et al., 2000) (9) (Alabadi et al., 2001) (10) (Strayer et al., 2000) (11) (Martinez-Garcia et al., 2000) (12) (Nelson et al., 2000) (13) (Jarillo et al., 2001) (14) (Millar et al., 1995a) (15) (Somers et al., 2000) (16) (Sugano et al., 1999) (17) (Zagotta et al., 1992) (18) (Sugano et al., 1998) (19) (Liu et al., 2001b) (20) (Hicks et al., 2001) (21) (Covington et al., 2001) (22) (Fowler et al., 1999) (23) (Park et al., 1999) (24) (Fowler et al., 1999; Park et al., 1999) (25) (Johnson et al., 1994) (26) (Bagnall et al., 1995) (27) (Guo et al., 1998) (28) (Putterill et al., 1995) (29) (Simon et al., 1996) (30) (Mozley and Thomas, 1995) (31) (Ahmad et al., 1998) (32) (Koornneef et al., 1980) (33) (Suarez-Lopez et al., 2001) (34) (Somers et al., 1998a).

these processes is shown in Figure 1 and is described in the following sections.

## A Pathway That Promotes Flowering of Arabidopsis in Response to Long Days

Among the flowering-time mutants of Arabidopsis, there is a group that is late flowering in long days but similar or identical to the wild type under short days. These mutants are weakly, or not at all, sensitive to vernalization, and were proposed to define a genetic pathway that promotes flowering specifically in response to long days. CONSTANS (CO), CRYPTOCHROME2/FHA (CRY2), GIGANTEA (GI), FT and FWA are part of this long day–promoting pathway (Koornneef et al., 1991). CO is the latest acting of the known genes that is specific to this pathway; all of the other genes also act in other pathways or have more general effects (Figure 1). FT and FWA act downstream of CO and in other pathways (Kardailsky et al., 1999; Kobayashi et al., 1999; Onouchi et al., 2000), whereas GI and CRY2 act upstream of CO (Guo et al., 1998; Suarez-Lopez et al., 2001).

The CO gene was proposed to encode a protein with two zinc fingers loosely related to those of GATA transcription

factors (Putterill et al., 1995) and contains a carboxy-terminal domain called CCT because it is present in CO, CO-like (COL), and TIMING OF CAB 1 (TOC1) proteins (Strayer et al., 2000; Robson et al., 2001). Comparison of 16 Arabidopsis COL genes that code for proteins sharing the zinc fingers and CCT domains (Robson et al., 2001) demonstrated that the zinc fingers are most similar to the B-box, a type of zinc finger identified in animal proteins and believed to mediate protein-protein interactions (Borden, 1998). The CCT domain is sufficient and necessary for nuclear localisation of GFP:CO or GFP:TOC1 fusion proteins (Strayer et al., 2000; Robson et al., 2001), but may have additional roles in protein-protein interactions (Kurup et al., 2000).

When constitutively expressed from a cauliflower mosaic virus 35S promoter, CO induces early flowering and loss of photoperiod sensitivity (Onouchi et al., 2000). Mutations in the FT, SUPPRESSOR OF OVEREXPRESSION OF CO1 (SOC1, also called AGAMOUS-LIKE 20 [AGL20]) and FWA genes partially suppress 35S::CO (Onouchi et al., 2000). In addition, FT and SOC1 expression levels are increased in 35S::CO plants or 35S::CO:GR plants in which CO activity is induced by dexamethasone (Samach et al., 2000), indicating that these genes act downstream of CO. In agreement with this, 35S::FT can suppress the effect of co mutations (Kardailsky et al., 1999; Kobayashi et al., 1999). However, no mutations were recovered that totally suppress the 35S::CO early-flowering phenotype, suggesting that parallel pathways are activated by CO (Onouchi et al., 2000).

The CO transcript level shows a diurnal rhythm in long days, with a broad biphasic peak between 12 and 24 h after dawn and maximum levels 16 and 24 h after dawn. This peak is narrower in short days, ending 4 h earlier (Suarez-Lopez et al., 2001). Plants entrained in long days show a circadian rhythm in CO transcript level when transferred to continuous light (LL), indicating that this rhythm is controlled by the circadian clock (Suarez-Lopez et al., 2001).

FT is an early target of CO (Samach et al., 2000), and its transcript level follows a circadian rhythm that peaks 20 h after dawn in long days (Suarez-Lopez et al., 2001). This peak is absent in the co mutant. In addition, the circadian clock-related mutants, LATE ELONGATED HYPOCOTYL (LHY), GI, and EARLY FLOWERING3 (ELF3), show an altered rhythm in CO transcript level that correlates with their flowering phenotype. Overall, CO appears to mediate between the circadian clock and the flowering-time gene FT (Suarez-Lopez et al., 2001). This model suggests that the circadian clock acts within the long-day pathway to regulate the expression of downstream genes such as CO and FT.

Two general models have been proposed for how flowering may be regulated by daylength (Thomas and Vince-Prue, 1997; Samach and Coupland, 2000; Samach and Gover, 2001). The external coincidence model proposes that light acts as an external signal that interacts with a light-sensitive rhythm at certain times of day. In long-day plants, such as Arabidopsis, if the plant is exposed to light at the crucial phase of the rhythm, flowering is promoted.

Another model, the internal coincidence model, suggests that in photoperiods that are inductive, two rhythms are brought into the same phase and interact to promote flowering, whereas in noninductive photoperiods, these rhythms are out of phase. Observing the activation of FT by CO, Suarez-Lopez et al. (2001) proposed that the expression pattern of CO may represent a light-sensitive rhythm that is exposed to light only under long-day conditions that promote flowering, and that post-transcriptional activation of CO by light may lead to the activation of FT transcription and ultimately to flowering. This proposal represents a version of the external coincidence model.

Candidates for photoreceptors that might be involved in the post-transcriptional activation of CO protein under long days are CRY2 and PHYTOCHROME A (PHYA). On the basis of the observation that phyA mutants are slightly late flowering in long days (Johnson et al., 1994) and transgenic plants overexpressing PHYA flower earlier than the wild type in short days (Bagnall et al., 1995), PHYA is a photoreceptor that promotes flowering. Similarly, the cry2 mutant is late flowering in long days and transgenic plants overexpressing CRY2 are early flowering in short days, indicating that CRY2 is involved in sensing photoperiod (Guo et al., 1999). Furthermore, a novel allele of CRY2 was recently isolated by positional cloning of a quantitative trait locus for early flowering identified between two Arabidopsis ecotypes (El-Assal et al., 2001). This new allele causes early flowering under short days because of a single amino acid substitution that impairs the light-induced downregulation of CRY2 under short days. Mutation in CO suppresses the early flowering caused by the CRY2 allele, suggesting that CRY2 does regulate flowering through CO. PHYA and CRY2 are therefore good candidates for photoreceptors that perceive the photoperiod in the external coincidence model.

## The Circadian Clock and the Central Oscillator

Circadian rhythms have a period length (the duration of one cycle) of ~24 h. These rhythms do not require daily transitions from light to dark, but continue under constant conditions. Circadian rhythms have been observed at different levels of organization, from leaf movement to stomatal aperture, CO₂ assimilation, or gene transcription. The mechanism that generates these rhythms is often described in three interrelated sections. These are input pathways that synchronise the clock mechanism to daily cycles of light and dark, a central oscillator that generates the 24-h time-keeping mechanism, and output pathways that regulate particular processes (Roenneberg and Merrow, 2000; McClung, 2001). The control of flowering via CO and FT represents one such output pathway (Suarez-Lopez et al., 2001), whereas many others have been described in detail using global gene expression assays (Harmer et al., 2000; Schaffer et al., 2001).

One output pathway, represented by the CHLOROPHYLL A/B BINDING PROTEIN2 (CAB2) gene, has enabled detailed analysis of clock regulation. The timing of CAB (toc) mutants were isolated from transgenic plants carrying the LU- CIFERASE (LUC) reporter gene under the control of the CAB2 promoter (Millar et al., 1995a). The toc1 mutant showed a 2- to 3-h shorter period length for CAB transcription and other gene expression rhythms, leaf movement (Millar et al., 1995a), and stomatal conductance (Kreps and Simon, 1997; Somers et al., 1998b). The toc1 mutant flowered early in short days (Kreps and Simon, 1997; Somers et al., 1998b), and this phenotype could be rescued by growing the plants in daily cycles of 21 h rather than 24 h, a reduction of 3 h corresponding to the shorter period length observed in toc1 (Strayer et al., 2000). This supports the proposal that shortened circadian period length is the cause of the early-flowering phenotype of toc1 mutants.

The TOC1 protein shows two characteristic domains. The first of these, at the N terminus, is similar to the receiver domain of response regulators from two-component signal transduction systems. However, invariant residues of this motif, including the Asp residue that is normally phosphorylated in response regulators, are substituted in TOC1, suggesting that TOC1 does not function as a classic response regulator. Indeed, TOC1, also known as APRR1, and other pseudo-response regulators, are not phosphorylated in an in vitro assay (Makino et al., 2000). The second motif, at the C terminus, is the CCT domain, shared with the CO family of transcriptional regulators. TOC1 transcript level cycles in light/dark cycles with a peak in the evening and shows a circadian rhythm in LL (continuous light; Strayer et al., 2000). In the toc1 mutant, the period of the circadian rhythm of TOC1 expression in LL is shorter, indicating that TOC1 controls its own rhythm of expression through a feedback loop. The rhythm in TOC1 transcript level, the period-shortening effect of the toc1 mutation with the identical influence of the mutation on multiple outputs, and the participation of TOC1 in a feedback loop make it a candidate component of the oscillator.

Other likely components of the oscillator are the CIRCADIAN CLOCK ASSOCIATED1 (CCA1) and LATE ELONGATED HYPOCOTYL (LHY) genes, which are also involved in the photoperiodic induction of flowering. These genes encode highly similar single Myb domain DNA binding proteins and are each regulated by the circadian clock, with a peak in their expression soon after dawn (Schaffer et al., 1998; Wang and Tobin, 1998). The overexpression of LHY or CCA1 disrupts all circadian rhythms tested, such as those in leaf movement and gene expression rhythms in different phases, including those in LHY or CCA1 expression (Schaffer et al., 1998; Wang and Tobin, 1998). Recent reports, nevertheless, suggest that LHY overexpression may not disrupt rhythms in expression of the circadian clock-regulated gene EARLY FLOWERING 3 (ELF3), which would be an unexpected exception to the effect of LHY (Hicks et al., 2001). Loss-of-function alleles of LHY or CCA1 cause circadian period length to be shortened by ~3 h, and inactivation of both genes prevents the maintenance of circadian rhythms under constant conditions (Green and Tobin, 1999; Mizoguchi et al., 2002). The observations that CCA1 and LHY present a circadian rhythm of expression, that they regulate their own transcripts through a feedback loop, and that they both suppress all observed rhythms when constitutively expressed suggest that they could be components of the oscillator. There is also a close relationship between the effect of LHY and CCA1 on clock regulation, and flowering time. Overexpression of either gene causes late flowering under long-day conditions, whereas loss-of-function alleles of CCA1 and LHY, which, like the toc1 mutant, show a shortening of the period length in LL, also resulted in early flowering in short days (Mizoguchi et al., 2002).

The proposal that TOC1 and LHY/CCA1 may be part of a central oscillator was recently strengthened by the demonstration that they participate in an autoregulatory loop (Alabadi et al., 2001). LHY and CCA1 bind to the TOC1 promoter in vitro and reduce TOC1 expression when overexpressed, whereas in toc1 mutants, the period length of LHY and CCA1 expression is shorter and their level of expression is reduced. Therefore, TOC1, LHY and CCA1 may act in a negative feedback loop, in which LHY expression and CCA1 expression rise in the morning and repress the expression of TOC1. According to this model, LHY and CCA1 feed back to repress their own expression, and as their protein levels fall, the expression of TOC1 rises. TOC1, in turn, promotes the expression of LHY and CCA1, so initiating another cycle (Alabadi et al., 2001). This feedback loop may regulate flowering time by determining the time of day that flowering-time genes in the long-day pathway, such as CO and GI, are expressed (Figure 1).

Entraining the Clock

Circadian rhythms must be synchronized with the daily rhythm of light and dark and of temperature. The phase of circadian rhythms can therefore be reset by environmental conditions such as light and temperature that fluctuate during day/night cycles. Mutations that affect entrainment of circadian rhythms by light signals have been described, and some of them directly affect photoreceptors, whereas others affect light signal transduction. Many of these mutations also have severe effects on flowering time, but for many of them, it is not clear whether their effect on flowering time is due to an impairment of circadian clock entrainment or to another aspect of light signal transduction. The intensity and quality of light will also affect flowering independently of the effects of light on circadian clock regulation, and the role of the blue light-receptor cryptochromes is reviewed in this volume (Lin, 2002).

For example, a general effect of phytochromes on flowering can be deduced from the hy1 and hy2 mutants, which are impaired in the synthesis of the chromophore of all phytochromes. Both mutations cause severe early flowering in

short days and in long days (Goto et al., 1991), indicating that phytochromes generally are repressors of flowering. However, only a part of this effect is likely to be due to the long circadian period also caused by these mutations (Millar et al., 1995b).

The photoreceptors PHYA, PHYB, CRY1, and CRY2 all influence clock entrainment under specific light conditions. However, the quadruple mutant *phyA phyB cry1 cry2* still shows robust circadian rhythms that can be entrained, indicating that either another phytochrome or an unknown photoreceptor is also part of the clock input machinery (Yanovsky et al., 2000). The influence of *PHYA* and *CRY2* on clock entrainment occurs only at low fluence rates (Somers et al., 1998a), suggesting that the late-flowering phenotype associated with mutations in these two photoreceptors probably is not caused by a direct effect on circadian rhythms (see also section describing the long-day pathway).

Mutations in a PHYB-interacting protein, ZEITLUPE (*ZTL*), also have dramatic effects on clock function and flowering time. A semidominant mutation in *ZTL* caused late flowering in long days and a 3-h increase in the period length of gene expression rhythms (Millar et al., 1995a; Somers et al., 2000), whereas the period of leaf movements was lengthened by 5 to 6 h. The lengthening of the period in CAB transcription, as followed using a *CAB::LUC* reporter, was shown to be strongly dependent on light intensity, but no effect of light quality was observed (Somers et al., 2000), whereas leaf movements were arrhythmic under red light (Jarillo et al., 2001). Constitutive overexpression of *ZTL* also gave a late-flowering phenotype in long days (Nelson et al., 2000). The ZTL protein contains a PAS domain, an F-box, and six repeated kelch motifs that are predicted to form a β-propeller. The β-propellers are believed to be implicated in protein–protein interactions (Adams et al., 2000). The F-box interacts with SKP1 within the Skp1p-Cdc53p-F-box protein (SCF) complex that acts as a ubiquitin ligase implicated in the ubiquitination of target proteins (Patton et al., 1998). The PAS domain mediates protein–protein interactions and has been found in a group of blue light photoreceptors (Briggs and Huala, 1999). The combination of domains within *ZTL* suggested that it might recruit proteins for degradation by the proteosome in a way that is influenced by light. Protein–protein interactions have been described between *ZTL* and *PHYB* and between *ZTL* and *CRY1* (Jarillo et al., 2001). Because the transcript level of *ZTL* is not regulated by the clock and its effect is dependent on light intensity (Somers et al., 2000), *ZTL* may affect circadian rhythms by the impairment of a light input pathway.

*ZTL* is a member of a family of three genes that also includes *FKF1* and *LKP2* (Somers et al., 2000). The *fkf1* mutant shares the late-flowering phenotype of *ztl*. However, its transcript level shows a circadian rhythm, and an *fkf1* null allele has no effect on the period length of *CAB* and *CCA1* expression; only a change in the form of the peak was observed (Nelson et al., 2000). The late-flowering phenotype of *fkf1* can be rescued by vernalization and gibberellin (GA) application, suggesting that *fkf1* may be associated with the autonomous flowering pathway (Nelson et al., 2000). The association of F-box proteins with circadian clock function is consistent with the importance of protein instability in circadian clock regulation.

Target protein recognition by F-box proteins is strictly dependent on their phosphorylation (del Pozo and Estelle, 2000). *CCA1* and *LHY* can be bound by CKB3, a regulatory subunit of the protein kinase CK2 (Sugano et al., 1998, 1999). Transgenic plants overexpressing CKB3 show a period length of *CCA1* and *LHY* expression under LL that is shorter by ~4 h, and are early flowering under long or short days. The period shortening was also detected in the rhythms in expression of several other genes, which peak at different times of the day, indicating the importance of CK2 in the regulation of the circadian clock. CK2 activity is increased in the transgenic plants, and this activity has been shown to phosphorylate *CCA1* and *LHY* in vitro. Thus, in plants overexpressing CKB3, *CCA1* may be phosphorylated more rapidly, perhaps causing a shorter period length due to an increased rate of degradation of the protein. This may involve F-box proteins and the ubiquitin pathway.

The *elf3* mutant was isolated as an early-flowering mutant insensitive to photoperiod (Zagotta et al., 1992), and this phenotype is likely to be due, at least in part, to increased expression of *CO* (Suarez-Lopez et al., 2001). *elf3* shows similarities to *phyB* mutants, including elongated hypocotyls and petioles, early flowering, and defects in the red light response (Reed et al., 2000). *ELF3* encodes a nuclear protein proposed to act as a transcriptional activator (Hicks et al., 2001; Liu et al., 2001b). The ELF3 protein interacts with *PHYB* and requires *PHYB* for the regulation of hypocotyl elongation but not for the control of flowering time, indicating that although *ELF3* is part of the *PHYB* signaling pathway, they control flowering time independently (Liu et al., 2001b). *ELF3* transcript level is circadian clock regulated, and peaks at the beginning of the subjective night (Hicks et al., 2001). The *elf3* mutant is arrhythmic for leaf movements and gene expression rhythms when grown in LL, but not in continuous darkness (DD). It also shows increased sensitivity to red light, as measured by light-induced expression of *CAB2* in etiolated seedlings, which is independent of circadian clock control, or induction of *CAB2* during the night, a process regulated by the circadian clock. *ELF3*-overexpressing plants show a reduced sensitivity to acute light induction and an increased period length of circadian rhythms. Taken together, these data suggest that *ELF3* is a repressor of light signaling that represses light input to the clock during the night. It also inhibits phytochrome-induced responses such as the acute induction of *CAB* (McWatters et al., 2000; Covington et al., 2001). *ELF3* illustrates the complexity of the function of genes that simultaneously control circadian rhythms and light regulation while they are themselves influenced by the clock.

The *gi* mutant is late flowering in long days, and *gi* seedlings show a loss of sensitivity to red light but not to far-red

light, typical of a disruption of the PHYB signal transduction pathway (Huq et al., 2000). However, how this role for GI in PHYB signal transduction relates to its late-flowering phenotype is unclear because in contrast to gi mutants, phyB mutants are early flowering. Mutant alleles of GI also cause alterations in period length of gene expression rhythms. The GI protein contains six putative transmembrane domains (Fowler et al., 1999; Park et al., 1999), suggesting a membrane protein, but GUS:GI and GFP:GI fusion proteins were targeted to the nucleus (Park et al., 1999). The GI transcript level is circadian clock regulated. In the elf3 mutant, the GI transcript level is increased in long days and short days, and the rhythm is disrupted in LL, suggesting that GI acts after ELF3 (Fowler et al., 1999). The late flowering of gi mutants is at least in part due to reduced amplitude of CO expression, but how this relates to the roles of GI in light signal transduction or in regulating circadian clock period length is unknown.

The Vernalization Response Pathway

Exposure to low temperatures for several weeks will often accelerate flowering. Susceptibility to this treatment can differ markedly between varieties of a species. For example, many naturally occurring Arabidopsis varieties will flower very late if they are not exposed to a vernalization treatment but flower early if exposed to low temperatures for 4 to 8 weeks (Michaels and Amasino, 2000). Such a requirement for vernalization is associated with a winter annual growth habit. In nature, these varieties do not flower until they have been exposed to winter conditions. They typically germinate in the summer, grow vegetatively through the winter until the following spring, and then flower, often in response to appropriate daylengths during the spring or summer. This is in contrast to summer annuals, which germinate and flower in the same summer without a requirement for vernalization. The genetic control of vernalization was addressed by crossing winter annual varieties that require vernalization with summer annual varieties that do not. These varieties differed at two loci, FLOWERING LOCUS C (FLC) and FRIGIDA (FRI), and dominant alleles at these loci in the winter annual are required to confer a vernalization requirement (Burn et al., 1993a; Lee and Amasino, 1993; Clarke and Dean, 1994).

FLC encodes a MADS box transcription factor (Michaels and Amasino, 1999b; Sheldon et al., 1999). The mRNA and protein of FLC are expressed at much higher levels in vernalization-requiring winter annual varieties of Arabidopsis than in early-flowering summer annual varieties. In addition, expression of FLC in summer annual varieties from the 35S promoter causes a dramatic late-flowering phenotype. Therefore, FLC encodes a repressor of flowering, and high-level expression of FLC correlates with the vernalization requirement of winter annual varieties (Michaels and Amasino, 1999b; Sheldon et al., 1999, 2000). The association between FLC and vernalization was strengthened by demonstration that the abundance of FLC mRNA falls when the plants are exposed to cold, and that this reduction occurs progressively in a way that is consistent with the progressive effect on flowering time. For example, treatment of plants at 4°C for 14 days causes a partial reduction in FLC expression, and incomplete vernalization as measured by flowering time (Sheldon et al., 2000). Flowering-time genes whose expression is repressed by FLC have been isolated (see below, Integration of Flowering Pathways).

Although FLC plays a central role in vernalization, and much of this response involves reductions in FLC mRNA, this probably does not explain the whole response to vernalization. Mutants carrying null alleles of FLC still show a response to vernalization (Michaels and Amasino, 2001). There is a clade of MADS box transcription factors closely related to FLC, and partial redundancy between these might explain the ability of flc mutants to respond to vernalization (Ratcliffe et al., 2001; Scortecci et al., 2001). Inactivation of at least one of these transcription factors, FLM, causes early flowering, and its overexpression delays flowering (Scortecci et al., 2001). However, none of these genes are regulated by vernalization in a way similar to FLC. There is also suggestive physiological and genetic evidence that vernalization does not simply act to reduce the expression of an inhibitor of flowering such as FLC, but that it more actively promotes flowering. For example, vernalization of flc mutants in a Columbia background causes early flowering in short days, suggesting that promotion of flowering by vernalization can overcome the requirement for long-day induction of flowering (Michaels and Amasino, 2001). Similarly, vernalization causes a co ga1 fca triple mutant to flower surprisingly early, suggesting that, even in this background, in which the three major pathways are impaired, vernalization will promote early flowering (Reeves and Coupland, 2001). However, the molecular nature of a pathway that promotes flowering via vernalization but independently of FLC is not known (Figure 2).

The other locus at which dominant alleles confer a vernalization requirement is FRI. The product of this gene somehow increases FLC mRNA abundance (Michaels and Amasino, 1999b; Sheldon et al., 1999). That this effect on FLC expression is required for FRI to delay flowering is supported by the observation that loss-of-function flc mutations suppress the effect of FRI on flowering time. The biochemical function of FRI protein is unknown, but it contains coiled-coil domains that may be involved in protein–protein interactions (Johanson et al., 2000). Most early-flowering varieties of Arabidopsis carry one of two deletions that disrupt the open reading frame of FRI, suggesting that these early-flowering varieties are derived from late-flowering ones by inactivation of the FRI gene (Johanson et al., 2000).

The molecular-genetic analysis of FRI/FLC suggests that vernalization acts by reducing FLC expression in response to extended exposure to cold. The effect of vernalization was previously shown to be stable through mitosis

Control of Flowering Time S117

![Diagram](https://i.imgur.com/5JZzvLX.png)

Figure 2. The Effects of Vernalization and the Autonomous Pathway on Flowering Time, Emphasizing the Central Role of FLC.

The autonomous pathway genes and vernalization promote flowering by repressing $FLC$ expression. Once $FLC$ expression is reduced by vernalization, it is stably repressed by the $VRN2$ gene product. Vernalization promotes flowering independently of $FLC$ as well as by $FLC$ repression. $HOS1$ seems to act as a repressor of the vernalization pathway. $FLC$ expression is promoted by $FRI$. $FLC$ represses flowering and, at least in part, this occurs through repressing the flowering time genes $SOC1$ and $FT$. The data used to derive this model are described in detail in the text.

(Michaels and Amasino, 2000), and once $FLC$ expression is reduced, it remains low even in plant organs that are formed subsequent to the vernalization treatment (Michaels and Amasino, 1999b; Sheldon et al., 1999). Similarly, the effect of vernalization is reset during meiosis, and high-level expression of $FLC$ is also restored in the progeny of vernalized plants. $FLC$ expression therefore shows features in common with the vernalized state. These features of vernalization show similarities to epigenetic control of gene expression, and led to the proposal that regulation of gene expression by methylation may be the basis of vernalization (Burn et al., 1993b). Consistent with this proposal, demethylating agents caused early flowering of late-flowering mutants of Arabidopsis, transient reductions in methylation occurred in vernalized plants, and reductions in methylation in transgenic plants carrying an antisense methyltransferase construct caused early flowering of late-flowering plants that respond to vernalization (Finnegan et al., 1998). $FLC$ expression was also reduced in a transgenic line carrying the antisense methylase transgene (Sheldon et al., 1999).

A genetic approach was also taken to identifying genes required for the vernalized state. In this case, vernalization-requiring plants ($fca$ mutants; see below, The Autonomous Pathway) were mutagenized to identify mutations that prevented early flowering after vernalization (Chandler et al., 1996). The resulting vernalization ($vrn$) mutants appeared to prevent downregulation of $FLC$ expression by exposure to cold, because when $fca\ vrn$ double mutant plants were vernalized and then grown at normal temperatures for 2 to 3 weeks, $FLC$ mRNA was expressed strongly in the double mutant but reduced to low levels in the $fca$ mutant (Sheldon et al., 1999). However, recently the role of $VRN2$ was described in more detail (Gendall et al., 2001). These experiments demonstrated that $FLC$ mRNA levels in an $fca\ vrn2$ double mutant were as low as in an $fca$ mutant immediately after vernalization, but that as plants were grown at normal growth temperatures, the abundance of $FLC$ mRNA rose in the double mutant but not in $fca$. This suggests that the role of $VRN2$ is not the reduction of $FLC$ expression in response to low temperatures, but the maintenance of the vernalized state (Gendall et al., 2001). Cloning of $VRN2$ showed that its product is related in sequence to a Drosophila Polycomb group protein, Su(z)12, that regulates gene expression by modifying chromatin structure (Birve et al., 2001; Gendall et al., 2001). A similar role for $VRN2$ was indicated by the demonstration of altered chromatin structure around $FLC$ in $vrn2$ mutants. The epigenetic regulation of vernalization may, then, be mediated by alterations in chromatin structure; how this relates to the effect of methylation is not clear.

The analysis of $VRN2$ gene function demonstrated that this protein is required for the maintenance of the vernalized state but not for the initial response to cold. A gene that appears to regulate the response to cold is HIGH EXPRESSION OF OSMOTICALLY RESPONSIVE GENES 1 ($HOS1$) (Lee et al., 2001). Mutations in this gene were identified by isolating mutants that showed enhanced expression of the cold-inducible gene $RD29$ in response to cold stress. The $hos1$ mutants were also early flowering, and showed reduced expression of $FLC$ (Ishitani et al., 1998; Lee et al., 2001). These observations are consistent with the idea that $HOS1$ encodes a negative regulator of cold signaling, and that in the absence of $HOS1$, enhanced activity of this pathway leads to increased expression of cold-induced genes such as $RD29$ and reduced expression of $FLC$, which is normally repressed by cold (Lee et al., 2001). The $hos1$ mutant provides a genetic link between vernalization and cold stress. The $HOS1$ gene encodes a RING finger protein that may serve as a ubiquitin ligase; these have been implicated as negative regulators of a number of other plant signal transduction pathways, but a role in cold signal transduction

has not been demonstrated (Bachmair et al., 2001). Further genes involved in vernalization and required for cold perception and/or response to the cold will surely emerge from further genetic screens, and the indication, based on the analysis of *HOS1*, that there may be similarities to the mechanism of cold stress is a tantalizing one.

### The Autonomous Pathway

The autonomous pathway was identified via a group of mutants that are late flowering under all photoperiods, and are highly responsive to vernalization (Martinez-Zapater and Somerville, 1990; Koornneef et al., 1991). These mutants include *fca*, *fy*, *fpa*, *luminidependens* (*ld*), and *fve*. The similarity between these mutants was further emphasized by the demonstration that they all contain much higher levels of *FLC* mRNA than do wild-type plants or late-flowering mutants affected in the long-day or GA pathways (Michaels and Amasino, 1999b; Sheldon et al., 1999). Thus, autonomous pathway mutations appear to delay flowering by causing increased expression of *FLC*, and therefore in wild-type plants this pathway can be considered to negatively regulate *FLC* expression (Figure 2). Such a role for autonomous pathway mutants was also demonstrated genetically, because loss-of-function *flc* alleles suppress the late-flowering phenotype of autonomous pathway mutants (Michaels and Amasino, 2001).

Nevertheless, the interaction of autonomous pathway mutants with naturally occurring alleles of *FLC* differs. The *fca*, *fy* and *fpa* mutations cause late flowering in a Landsberg erecta background, and the *FLC* allele present in this ecotype is therefore sensitive to loss-of-function alleles in these autonomous pathway genes. However, the *ld* mutation does not cause late flowering in Landsberg erecta, but does in other ecotypes such as Columbia, suggesting that the Columbia but not the Landsberg allele of *FLC* responds to loss-of-function of *LD* (Koornneef et al., 1994; Lee et al., 1994a). These allelic differences in the regulation of *FLC* remain to be explained.

Although all of the autonomous pathway mutations act by increasing *FLC* expression, genetic evidence suggests that they may not all act in a simple linear pathway (Figure 2). For example, *fca fpa* double mutants are much later flowering than would be expected as a result of a simple additive effect of these mutations, and combining *fpa* and *fy* mutations appears to be lethal, indicating a much wider role for these genes in plant development than simply regulating flowering time (Koornneef et al., 1998a).

Several of the genes within the autonomous pathway have been cloned. The predicted FCA protein contains two copies of an RNA binding domain, the RNP (also referred to as RNA recognition motif or consensus sequence RNA-binding domain), and a WW protein–protein interaction domain (Macknight et al., 1997). FCA was shown to bind RNA in vitro. Strikingly, FPA also encodes an RNA binding protein containing RNP motifs, suggesting that post-transcriptional regulation may play a general role in the pathway (Schomburg et al., 2001). The LD protein contains a homeobox and putative nuclear localization sequences, and may encode a transcription factor (Lee et al., 1994b). All three genes are expressed in a similar pattern, with maximal expression at the apex of the plant and inflorescences, and low-level expression in mature leaves and roots. Overexpression of *FPA* causes severe early flowering under short days, whereas increased expression of *FCA* had relatively minor effects on flowering time, but in this experiment, post-transcriptional regulation of *FCA* RNA splicing prevented large increases in the abundance of the fully processed *FCA* mRNA (Macknight et al., 1997; Schomburg et al., 2001).

The manner in which these proteins regulate *FLC* expression is not known. However, plants containing meristem sectors of *fca* homozygous mutant cells within the L2 and L3 layers of otherwise wild-type plants flowered at the same time as did the wild type. This suggests that FCA wild-type cells, either in adjacent L1, L2, or L3 cells within the meristem or in other tissues of the plant, were able to correct the phenotype of the *fca* mutant cells (Furner et al., 1996). Thus non-cell autonomous factors may act downstream of *FCA*.

### The GA Pathway

The growth regulator GA promotes flowering of Arabidopsis (Wilson et al., 1992; Putterill, et al., 1995; Blazquez et al., 1998). This was initially demonstrated by applications of exogenous GA (Langridge, 1957), and has been more recently studied using mutations that disrupt either GA biosynthesis or signaling (Wilson et al., 1992). These mutations also have effects on many other aspects of plant growth and development, including stem elongation, germination, and floral development. In this section, we summarize the impact of GA on flowering (Figure 3), and GA signaling is thoroughly reviewed by Olszewski et al. (2002).

Several mutations affecting GA biosynthesis have been identified in Arabidopsis, and the mutant genes cloned. Roles for the *GA1*, *GA4* and *GA5* genes in regulating flowering time have been described. *GA1* encodes copalyl diphosphate synthase, an enzyme that catalyzes the first committed step in GA biosynthesis (Sun and Kamiya, 1994). The corresponding mutant, *ga1-3*, is unable to flower in short days, and is later flowering than is the wild type under long days (Wilson et al., 1992). These plants are also severely dwarfed, do not germinate in the absence of exogenous GA, and exhibit reduced apical dominance.

In contrast to *ga1*, the *ga4* and *ga5* mutations have less-severe effects, giving rise to semidwarf plants that produce fertile flowers with normal siliques (Koornneef and van der Veen, 1980). The *ga4* and *ga5* mutants are defective in GA 3β-hydroxylase and GA 20-oxidase activity, respectively (Talon et al., 1990). GA 20-oxidase is regulated by environmental or physiological changes, suggesting that it may be

Environmental Signals  
Endogenous Signals  

↓  

GA Biosynthesis  

↓  

Bioactive GA  

↓  

GA receptor  

SPY  
PHOR1  

GAI/RGL1 (RGL2, RGL3)?  

RGA → Degradation  

↓  

GA signaling  

↓  

SOC1/FPF1/GAMYB-like  

↓  

LFY  

↓  

Flower development  

---

Figure 3. GA-Signaling Pathway That Regulates Flowering in Arabidopsis.

Activation of a hypothetical transmembrane receptor by GA inhibits repressors of GA signaling. These repressors are encoded by the RGA, GAI, and RGL genes. The SPY gene also represses GA signaling and genetically acts upstream of RGA and GAI. It may act to promote the activity of GAI/RGA/RGL by GlcNAc modification, in which case GA signaling may inhibit GAI/RGA/RGAL by repressing SPY function. PHOR1 has not been described in Arabidopsis, but has been shown to be involved in GA signaling in potato. Its possible involvement in ubiquitination and protein degradation leads to the tentative proposal that it is involved in the demonstrated degradation of the repressing protein RGA in response to GA. The floral meristem identity gene LFY is upregulated at the transcriptional level by GA. The flowering-time gene SOC1 is also upregulated by GA, whereas FPF1 and GA-MYB were proposed to mediate between GAs and the regulation of flowering time. These three genes may therefore act downstream of GAI/RGA/RGL but upstream of LFY. The data underlying this scheme are described in detail in the text.

involved in a key regulatory step in GA biosynthesis (Xu et al., 1995). Expression of this gene increases when plants are transferred from short days to long days, and therefore high-level expression correlates with conditions that induce early flowering (Xu et al., 1997). Furthermore, transgenic plants containing elevated levels of GA 20-oxidase also contained more GA4 and flowered earlier than did wild-type control plants under both long-day and short-day conditions (Huang et al., 1998; Coles et al., 1999). This suggests that GA levels are limiting on flowering time, and is consistent with previous observations that application of exogenous GA causes early flowering of wild-type plants.

A role for GA in the control of flowering is also suggested by studying genes involved in GA signaling. Three related genes with key roles in GA signaling, GIBBERELLIC ACID INSENSITIVE (GAI), REPRESSOR OF GA1-3 (RGA) and RGA-LIKE 1(RGL1) have overlapping functions. GAI and RGA proteins share 71% sequence identity with each other (Peng et al., 1997; Silverstone et al., 1998), and 61% identity to RGL1 (Sanchez-Fernandez et al., 1998; Wen and Chang, 2002). These proteins belong to the plant-specific GRAS (for GAI, RGA, SCARECROW) family of regulatory proteins (Pysh et al., 1999; Bolle et al., 2000). All GRAS family members contain highly conserved central (VHIID) and carboxy-terminal (RVER) regions (Silverstone et al., 1998). RGA, GAI and RGL1 also contain a conserved sequence near their N termini that was called DELLA after the amino acids making up the region, although in RGL1 an alanine-to-valine substitution changes the sequence to DELLV (Wen and Chang, 2002). This sequence is not present in most GRAS family members (Peng et al., 1997; Silverstone et al., 1998). GAI, RGA and RGL1 are believed to inhibit GA responses in the absence of active GA, and GA relieves this inhibition.

A prediction of this model is that GA responses would occur in the absence of GA if GAI, RGA, and RGL1 genes were inactivated. This prediction was recently confirmed by constructing ga1 mutant plants, which contain very low amounts of active GA, in which RGA and GAI were also inactivated. The introduction of rga gai loss-of-function alleles completely rescued the stem growth defects of ga1-3 and suppressed the nonflowering phenotype of ga1 mutants so that the triple mutant flowered slightly earlier than did the wild type under short days (Dill and Sun, 2001; King et al., 2001). This, together with the observation that rga gai double loss-of-function mutants are slightly earlier flowering than wild-type plants, indicates that the late-flowering phenotype shown by GA biosynthetic mutants is due to active repression of flowering by GAI and RGA. This is an important observation, because the rga gai mutations do not rescue all aspects of the ga1 phenotype. For example, the rga gai ga1 triple mutant retains the germination and impaired floral development phenotype of ga1. These aspects of the ga1 phenotype may be negatively regulated by RGL1 and other related genes, such as RGL2 and RGL3 (Wen and Chang, 2002). Indeed, flower development of transgenic plants in which RGL1 expression was reduced by co-suppression

was resistant to the effects of inhibitors of GA biosynthesis (Wen and Chang, 2002). Unlike GAI and RGA, expression of *RGL1* is almost restricted to the inflorescence. In flowers, *RGL1* expression was localized to ovules and developing anthers. However, *RGL1* may also be involved in sepal and petal development. In transgenic plants overexpressing a modified *RGL1* protein lacking the DELLA domain, sepals, petals, and stamens were underdeveloped and the flowers were male sterile. *RGL1* may therefore play a role in repressing GA responses in the inflorescence, where apparently GAI and RGA are less important.

The DELLA sequence appears to play an important role in the mechanism by which GA inhibits the function of GAI, RGA, and *RGL1*. This was suggested by the analysis of a dominant *gai* allele carrying a deletion within the DELLA domain, because this mutant form inhibited shoot growth and delayed flowering even in the presence of GA (Peng et al., 1997). Deletion of the DELLA domain may therefore cause GAI to become insensitive to GA, so that it continues to repress shoot growth and flowering even in the presence of GA (Peng et al., 1997). Further evidence for this model comes from recent analysis of RGA. The RGA protein accumulates in the nucleus in the absence of GA, but in the presence of GA is rapidly degraded (Silverstone et al., 2001). However, this effect is prevented by removal of the DELLA domain (Dill et al., 2001). Therefore, regulation of *RGA* by GA may be caused by RGA degradation through a mechanism that requires the DELLA domain of RGA. Transgenic plants expressing a modified *RGL1* protein lacking the DELLA domain also exhibited repression of GA responses and were phenotypically similar to GA-deficient plants (Wen and Chang, 2002). However, unlike GFP:RGA, GFP:*RGL1* was not degraded upon GA treatment, suggesting this might not be a universal mechanism for the regulation of these DELLA proteins.

The SPY locus encodes another negative regulator of GA responses that influences flowering time. Mutations in SPY cause partial suppression of the effects of reduced GA levels, whether these are due to mutations in GA biosynthetic genes or the presence of GA biosynthesis inhibitors (Jacobsen and Olszewski, 1993). The *spy* mutant is early flowering, probably due to the effects of increased activity of the GA-signaling pathway. Cloning of SPY (Jacobsen et al., 1996) and its homolog in barley, HvSPY (Robertson et al., 1998), revealed that SPY is highly similar to Ser/Thr O-linked N-acetylglucosamine transferases in rat and humans (Kreppel et al., 1997). The manner in which this enzyme acts to regulate GA responses is not known, but genetically it acts upstream of GAI, and may be required for GAI/RGA activity.

Other proteins implicated in GA signal transduction and flowering time regulation are PHOR1 (PHOTOPERIOD RESPONSIVE 1), FPF1, and SHI. PHOR1 was initially identified in potato, and has not been incorporated into the signal transduction pathway defined in Arabidopsis (Amador et al., 2001). However, PHOR1 is transported into the nucleus in response to GA. This protein belongs to a family of armadillo-related helical repeat proteins, which serve as scaffold- ing proteins on which other proteins and/or nucleotides can assemble, and contains a protein domain related to that present in components of the ubiquitination system. The GA-responsive nuclear import of PHOR1 and its relationship to the ubiquitination system suggest that it could act upstream of RGA and could be involved in its degradation in the nucleus in response to GA, although there is no direct demonstration of this (Figure 3). *FPF1* is upregulated in the shoot meristem at the transition to flowering, and when overexpressed causes early flowering (Kania et al., 1997; Melzer et al., 1999). On the basis of genetic tests performed by combining transgenes that overexpress *FPF1* with mutations that delay flowering, *FPF1* was proposed to promote flowering via the GA pathway. *FPF1* encodes a small protein, whose biochemical function is unknown. Finally, the dominant *shi* mutation causes a phenotype similar to a GA-deficient mutant, including late flowering, and is caused by overexpression of a zinc finger protein (Fridborg et al., 1999, 2001).

Several studies have described genetic interactions between the GA pathway and other flowering-time pathways. For example, genetic analysis suggests that the GA pathway probably acts in parallel to the long-day pathway because there is redundancy between mutations affecting the two pathways. The effect of mutations that impair the GA pathway is strongest under short days, and combining *ga1* with mutations that impair the long-day pathway, such as *co*, produced double mutants that often did not flower under long days (Putterill et al., 1995; Reeves and Coupland, 2001). This suggests that in short days, where the long-day pathway is not active, GA is the major flowering pathway and loss of function of this pathway can prevent flowering. Under long days, the effect of inactivation of the GA pathway is less severe because of the activity of the long-day pathway.

There is also recent evidence for a connection between FPA, a gene encoding an RNA binding protein that acts in the autonomous pathway, and GA (Meier et al., 2001). Two late-flowering mutants, *fpa1-3* and *fpa1-4*, markedly increased activity of a *GA5::LUC* transgene. These plants showed elevated levels of GA<sub>19</sub> and GA<sub>4</sub>, which is consistent with overexpression of the *GA5* gene (Coles et al., 1999). *fpa1-3* and *fpa1-4* plants also show reduced sensitivity to GA levels for floral promotion and germination (Meier et al., 2001). Finally, the early flowering caused by vernalization was proposed to be due to the GA pathway (Sheldon et al., 1999). This model indicated that FLC acts to repress GA activity at the apex and thereby to repress flowering, and vernalization overcomes this repression by reducing FLC levels. However, *ga1-3 FRI FLC* plants, which never flower under long days without vernalization, flower as early as do *ga1-3* single mutants after cold induction, suggesting the ability to respond to vernalization under long days does not require GA (Michaels and Amasino, 1999a). Although this suggested that the GA pathway is not the basis of vernalization, the recent demonstration that expression of the flower-

ing-time gene SOC1 is repressed by FLC and promoted by GA or the long-day pathway indicates that the GA pathway does regulate the same flowering-time genes as the other flowering-time pathways (Borner et al., 2000; Lee et al., 2000; Samach et al., 2000) (see also below, section on pathway integration).

One way in which GAs promote flowering is by increasing the transcriptional activity of the floral meristem identity gene LEAFY (LFY). Expression of LFY::GUS is reduced in mutants defective in GA biosynthesis, such as *ga1-3*, and increased in mutants with constitutive GA signaling, such as *spy* and *35S::FPF1* (Blazquez et al., 1998; Melzer et al., 1999). Overexpression of *LFY* also restores flowering of *ga1-3* in short days (Blazquez et al., 1998). The effect of GA on *LFY* transcription appears to act through a promoter motif that is similar to the consensus binding site for MYB transcription factors (Blazquez and Weigel, 2000). A particular MYB protein, AtMYB33, which resembles a MYB transcription factor of barley that mediates activation of amylase gene expression by GA, and is upregulated at the apex during floral initiation was proposed to interact with this *LFY* promoter motif in vivo (Gocal et al., 2001). Although *LFY* transcription is activated by GA, *35S::LFY ga1-3* plants still produced more leaves than did *35S::LFY* plants, suggesting that GA plays an additional role in the regulation of flowering time (Blazquez et al., 1998).

### Chromatin Structure and Floral Repression in Arabidopsis

Arabidopsis mutants have been described that flower without forming any adult leaves, but progress directly from embryonic development to flowering (Sung et al., 1992; Yang et al., 1995; Kinoshita et al., 2001). The original representative of this class of mutants was *embryonic flower 1* (*emf1*), which formed a reduced inflorescence and abnormal flowers that lacked petals without first forming any rosette (Sung et al., 1992). A second mutant with a similar phenotype, *emf2*, was identified subsequently (Yang et al., 1995). The embryonic flower mutations are recessive, and were interpreted as identifying genes that are required either to promote vegetative growth or to repress flowering during embryo and seedling development. The floral meristem identity gene *APETALA1* (*AP1*) and the *AGAMOUS* (*AG*) gene, which specifies carpel and stamen identity in the flower, are ectopically expressed in germinating seedlings of *emf* mutants (Chen et al., 1997). However, the mutant phenotype of *emf2* was not reduced in severity when combined in double mutants with the *ap1*, *ap2*, or *lfy* mutations, suggesting that ectopic expression of each of these genes is not essential for the *emf* phenotype.

A mechanism of action of the *EMF* genes was suggested by their cloning. The predicted EMF2 protein contains a zinc finger, an N-terminal basic domain and a C-terminal acidic domain (Yoshida et al., 2001). This protein shows similarity to two Arabidopsis proteins, VRN2 and FERTILISATION INDEPENDENT SEED 2 (FIS2), which were previously identified by mutations, and to a Drosophila protein, Su(z)12. The *Su(z)12* gene encodes a Polycomb group (PcG) protein involved in the repression of homeobox gene expression during development of Drosophila (Birve et al., 2001). The *VRN2* gene is required for maintenance of the vernalized state and continued repression of *FLC* expression after vernalization (Gendall et al., 2001), and mutations in *FIS2* allow partial development of the seed without fertilization (Chaudhury et al., 1997).

The similarity of EMF2, VRN2 and FIS2 to Su(z)12 suggests that these three plant proteins may act in a way similar to PcG proteins of animals. These proteins act in large protein complexes to repress transcription by altering chromatin structure. The EMF2 protein may then act as part of a protein complex that during embryo and seedling development represses the expression of genes that promote reproductive development.

The importance of PcG genes in repressing reproductive development was also emphasized by the demonstration that another Arabidopsis PcG gene, FERTILISATION INDEPENDENT ENDOSPERM (*FIE*), which is related to the PcG protein EXTRA SEX COMBS OF Drosophila, represses reproductive development in the seedling (Kinoshita et al., 2001). Loss-of-function *fie* alleles were originally described because they allow partial endosperm formation prior to fertilization. However, the effect of the mutation in the seedling or adult was not described, because maternal *FIE* alleles are essential for embryo development (Ohad et al., 1999). By expressing *FIE* from a defective *FIE* promoter that allows expression during seed development but not in the germinated seedling, it could be demonstrated that *FIE* is required for the repression of flowering in the seedling (Kinoshita et al., 2001). Plants homozygous for *fie* and expressing *FIE* from such a defective promoter initiate reproductive development as seedlings and resemble *emf* mutants.

The effects of *fie* mutations on embryo and seed development are similar to those of the *mea* and *fis2* mutations. MEDEA (*MEA*) encodes a SET-domain PcG group protein while *FIS2* encodes a Su(z)12 homolog (see above), and *MEA/FIS2/FIE* probably form a protein complex that regulates seed development (Luo et al., 1999; Spillane et al., 2000; Yadegari et al., 2000). However, *mea* and *fis2* mutant seedlings do not initiate reproductive development in the seedling. Therefore, *FIE* may interact in the seedling with other proteins related to *MEA* and *FIS2* to generate a protein complex that represses reproductive development. One candidate for such a protein is CURLY LEAF (*CLF*), a SET-domain protein implicated in the repression of homeotic gene expression during vegetative growth, and *clf* mutants show an early-flowering phenotype (Goodrich et al., 1997). The analysis of mutants that initiate flowering prematurely in the seedling therefore illustrates a requirement for active transcriptional repression to prevent reproductive development

occurring so early that no vegetative development can take place. The activation of flowering by environmental conditions such as photoperiod and vernalization must be able to overcome these repression mechanisms and ensure that flower development is initiated at appropriate times.

There is also evidence that chromatin modeling can affect flowering by changing methylation patterns. The decreased DNA methylation (*ddm1*) mutation affects a gene encoding a SWI2/SNF2-like protein that is most closely related to genes of the SNF2 family implicated in chromatin remodeling (Jeddeloh et al., 1999). Repeated self-fertilization of *ddm1* mutants creates epigenetic alleles, some of which affect flowering time (Kakutani et al., 1996). One of the loci affected by these alleles is *FWA* (Kakutani, 1997). Epigenetic alleles of *FWA* result in late flowering due to reduced methylation in the vicinity of the gene, which causes increased and ectopic expression of *FWA* (Soppe et al., 2000). *FWA* encodes a homeodomain containing transcription factor (Soppe et al., 2000), whose ectopic expression delays flowering. Analysis of *FWA*, together with the other loci causing early and late flowering in the *ddm* background, suggests that methylation levels regulate the expression of a number of genes involved in flowering-time regulation.

### Integration of Arabidopsis Flowering Pathways

Separate genetic pathways regulate flowering in response to different environmental signals, but these pathways eventually converge to regulate the expression of the same downstream genes. For example, all of the flowering-time pathways ultimately lead to the transcriptional activation of the same set of floral identity genes that act within the floral primordia (Pineiro and Coupland, 1998). *LFY* is the earliest of the known floral identity genes to be expressed, and directly activates at least one of the later genes, *AP1* (Wagner et al., 1999). Plants carrying fusions of the *LFY* promoter to the *GUS* marker gene were used to demonstrate that *LFY* expression responds both to the long-day flowering pathway and to GA. Furthermore, deletion of a putative MYB transcription factor binding site within the *LFY* promoter prevented activation by GA, but not by the long-day pathway (Blazquez and Weigel, 2000). Plants homozygous for the *lfy* mutation and carrying a *LFY* transgene in which this GA-responsive element was deleted appeared wild type under long days but exhibited the *lfy* mutant phenotype under short days, where *LFY* expression requires the GA pathway. This conditional activation of the mutant *LFY* transgene demonstrated that activation of *LFY* by each of these two flowering pathways is separable. Thus the GA and long-day pathways converge on the *LFY* promoter, rather than both pathways activating an earlier acting gene that in turn increases the expression of *LFY* (Blazquez and Weigel, 2000).

Convergence of the long-day and autonomous pathways has also been studied. These pathways are clearly separate until the *FLC* and *CO* genes (Figure 4). For example, expression of the floral repressor *FLC*, a component of the autonomous pathway, delays flowering but does not reduce expression of *CO*, a component of the long-day pathway (Suarez-Lopez et al., 2001). Similarly, mutations in the *CO* gene do not affect expression of *FLC* (Sheldon et al., 1999). Nevertheless, flowering-time genes have been identified that act downstream of both *CO* and *FLC*. The *SOC1* (or *AGL20*) gene that encodes a MADS box protein is both activated by *CO* and repressed by *FLC*, suggesting that the pathways represented by these genes converge on *SOC1* (Borner et al., 2000; Lee et al., 2000; Samach et al., 2000; Michaels and Amasino, 2001). *SOC1* is also regulated by GA, and therefore is a common target of all three flowering pathways (Borner et al., 2000). Mutations that inactivate *SOC1* delay flowering, while *SOC1* overexpression causes early flowering and suppresses the effect of mutations in the long-day and autonomous pathways. Expression of the *Sinapis alba* ortholog of *SOC1* is increased by daylength, as well as by applications of GA and cytokinin (Bonhomme et al., 2000). This is consistent with the results from Arabidopsis, and in addition indicates that *SOC1* responds to cytokinins. Expression of *FT*, a second flowering-time gene regulated by more than one pathway, is reduced by mutations that impair the function of the long-day and autonomous pathways and activated by overexpression of *CO*, indicating that *FT* acts downstream of both of these pathways (Kardailsky et al., 1999; Kobayashi et al., 1999; Samach et al., 2000).

The repression of *SOC1* and *FT* expression by *FLC*, even in the presence of wild-type *CO* represents an important aspect of the adaptation of flowering time to changing seasons. This adaptation involves responses to several environmental stimuli, including temperature and daylength, which need to be balanced to produce a coherent response. For example, the winter annual growth habit requires repressing the induction of flowering by daylength until the plant has been exposed to winter temperatures, after which the plant must respond to lengthening daylength the following spring. The antagonism between *FLC* and *CO* in the activation of downstream genes such as *SOC1* and *FT* can explain such an adaptation because the activation of these genes by *CO* may be prevented by *FLC* during the first summer, but reduction in *FLC* expression during winter would allow *CO* to activate expression of the downstream genes during the long photoperiods of the following summer.

The analysis of *SOC1* expression demonstrates convincingly that this gene represents a point of convergence of several flowering-time pathways. Although *SOC1* expression increases rapidly within the floral meristem in response to inductive conditions, such as exposure to inductive photoperiods, the manner in which its function is related to increases in expression of the floral meristem identity genes is not understood.

The relationship between *FT* and floral meristem identity gene expression has been studied at the genetic and molecular levels. The *ft* mutation did not reduce the expression

tion. The crystal structure of CEN confirmed its relationship with PEBP, and in particular demonstrated that it also contained the ligand binding site that is believed to interact with phosphate groups (Banfield and Brady, 2000). The sequence of $FT$ is therefore consistent with it controlling flowering by regulating a phosphorylation cascade.

Conservation of Flowering Pathways in Plants Showing Responses Different from Those of Arabidopsis

Plants that flower in response to photoperiod are classified as short day, long day or day-neutral. Arabidopsis is a facultative long-day plant, and genetic and biochemical analyses have identified a pathway responsible for early flowering in response to inductive long photoperiods, and a GA-mediated pathway that promotes flowering under short days. However, GAs have no florigenic effect in most short-day plants in noninductive photoperiods, and whereas day-length perception appears to occur mainly under darkness in short-day plants, it seems to occur predominately during the day in long-day plants (Thomas and Vince-Prue, 1997). These observations suggest that there may be major differences in the mechanisms of flower induction in the long-day plant Arabidopsis compared with most short-day plants, and that genetic models based on Arabidopsis may not be sufficient to explain photoperiodic responses in short-day plants. However, recent molecular-genetic experiments in short-day plants have demonstrated striking parallels between these and Arabidopsis.

Molecular and genetic approaches were used to study the transition from the vegetative to the reproductive phase in the classic short-day plants, rice, *Pharbitis nil*, and tobacco. In rice, flowering time (or heading date) is critical for the adaptation to different cultivation areas and cropping seasons and may be affected by environmental conditions such as photoperiod, temperature, and light intensity. There are several major genes affecting heading date that relate to vegetative growth or photoperiod sensitivity. Five quantitative trait loci, HEADING DATE 1 ($Hd1$) to $Hd5$, that control heading date were identified by comparing genetic differences between a *japonica* rice variety (Nipponbare) and an *indica* variety (Kasalath) (Yano et al., 1997). Genetic interactions among three loci involved in photoperiod sensitivity, $Hd1$, $Hd2$, and $Hd3$ identified epistatic interactions between some combinations, suggesting that they act in the same genetic pathway (Lin et al., 2000).

$Hd1$ was cloned on the basis of its map position. Sequence analysis revealed that $Hd1$ encodes a B-box zinc finger protein containing a C-terminal CCT domain and exhibits a high degree of similarity to the Arabidopsis CO gene (Yano et al., 2000). Experiments with various rice lines showed that the presence of a functional Nipponbare $Hd1$ allele was associated with a stronger photoperiod response, causing early heading under short days and later heading under long days. Lines homozygous for the recessive

Kasalath *Hd1* allele in a Nipponbare background flowered extremely early under long-day field conditions. Thus, the CO/*Hd1* gene is important in photoperiod response in long-day and short-day plants. However, unlike CO in Arabidopsis, *Hd1* appears to have two roles in rice, promoting heading under short days and delaying it under long days.

Detailed comparison of the sequences of wild-type and mutant forms of CO or *Hd1* also suggests that there may be differences in the biochemical mechanism of CO and *Hd1* function. For example, *co-3* is a strong mutant allele that converts a His residue predicted to be involved in zinc binding within the second B-box to a Tyr residue (Robson et al., 2001). Strikingly, the active Ginbouzu allele of *Hd1* has exactly the same change of His to Tyr, suggesting that in rice this does not impair protein function. Similarly, the *co-1* allele of Arabidopsis has a deletion of three residues within the second B-box, while the active Nipponbare allele has a deletion of 36 bp that removes two of these residues. Taken together, these data suggest that the second B-box is essential for CO but not for *Hd1* function. Detailed comparisons of COL proteins among species may help in determining their biochemical function. Although a COL gene displaying a high level of homology was found in gymnosperms (GenBank accession number AF001136), suggesting conservation of these sequences, there is also evidence that the family evolved rapidly (Lagercrantz and Axelsson, 2000).

Among the other major genes that affect heading date and are related to vegetative growth or photoperiod sensitivity in rice are *Hd6* and *SE5*, which were identified as the α subunit of protein kinase CK2 (Takahashi et al., 2001) and a putative heme oxygenase (Izawa et al., 2000), respectively. *Hd6* is a quantitative trait locus involved in rice photoperiod sensitivity, which was also detected in backcross progeny derived from a cross between the Nipponbare and Kasalath varieties (Yamamoto et al., 2000). The Nipponbare *Hd6* allele is nonfunctional. Introduction of the functional Kasalath allele delayed heading date of Nipponbare. CK2 belongs to the family of messenger-independent serine/threonine kinases that are present in all eukaryotic cells examined to date. This also suggests a relationship with photoperiod response in Arabidopsis. The expression of CK2 antisense RNA in Arabidopsis affected the expression of some light-regulated genes (Lee et al., 1999). CK2 interacts with and phosphorylates the Arabidopsis CCA1 protein in vitro (Sugano et al., 1998; Yamamoto et al., 2000). Overexpression of CK2 shortened the period of rhythmic expression of the CCA1 and *LHY* genes, and caused early flowering in both long- and short-day conditions (Sugano et al., 1999). Therefore, protein kinase CK2 may have a role in the photoperiod pathway of Arabidopsis and rice.

Finally, the rice photoperiodic sensitivity 5 (*se5*) mutation prevents the delay in flowering caused by exposure to noninductive long-day conditions (Izawa et al., 2000). *SE5* encodes a putative heme oxygenase, which shows 70% identity to *HY1* from Arabidopsis within the heme oxygenase domain (Davis et al., 1999; Muramoto et al., 1999). Both *hy1*

and *se5* mutants are deficient in phytochrome responses such as coleoptile responses to light pulses and seedling growth under continuous red and far-red light. The *hy1* mutant of Arabidopsis also flowers very early under noninductive conditions, emphasizing the involvement of the phytochromes in the repression of flowering under noninductive conditions in both long-day and short-day plants.

The Japanese Morning Glory, *P. nil* cv Violet, has also been used extensively as a model system for the physiology of flowering in short-day plants (Takeba and Takimoto, 1966; Vince-Prue and Gressel, 1985). A *P. nil* homolog of the CO gene, *PnCO* was isolated using differential display to identify genes with increased expression under short-day conditions (Liu et al., 2001a). The *PnCO* mRNA is inefficiently spliced, but the fully processed cDNA under the control of the cauliflower mosaic virus 35S promoter in Arabidopsis *co-1* complements the late-flowering phenotype of *co-1* mutants, confirming that *PnCO* has the same biochemical function as CO. *PnCO*, like CO, is under circadian clock control (Liu et al., 2001a). In continuous inductive darkness, the level of *PnCO* mRNA reaches a peak between 16 and 20 h, but it is not clear how this circadian clock control of *PnCO* regulates flowering in response to short days. Much lower accumulation of *PnCO* mRNA was detected after the noninductive 8, 10, and 12 h of darkness. Interestingly, the daily cycle in Arabidopsis CO expression under short days also shows a broad peak between 16 and 20 h with a strong peak at 20 h (Suarez-Lopez et al., 2001). In Arabidopsis, circadian clock control is thought to regulate exposure of CO protein to light, and post-transcriptional regulation of CO by light is proposed to promote flowering in long days. A night break (NB), 5 min of red light given 8 h into 14 h of darkness, inhibits flowering of *P. nil*. However, the level of *PnCO* RNA is only slightly reduced under these conditions, suggesting that the NB does not inhibit flowering by reducing transcription of *PnCO*. However, CO protein is very unstable in light (Suarez-Lopez et al., 2001), and *P. nil* *PnCO* protein could also be unstable and a target for degradation in response to NB.

The mechanism by which CO and its orthologs in short-day plants mediate the photoperiodic response is not clear. In Arabidopsis, CO promotes flowering under long-day conditions, activating at least two flowering-time genes, *SOC1* and *FT*, which have major roles in promoting flowering time (Samach et al., 2000). Light-activated post-transcriptional regulation of CO could be a part of this signal transduction pathway. It remains possible that differences in photoperiod-regulated activity of CO in short-day plants lie in the interaction of CO with different sets of proteins and subsequent upregulation of the target flowering-time genes. In addition, in short-day plants, CO may act in protein complexes that actively repress flowering under long days. The fact that the *Hd1* protein is involved in two opposite processes—activation of heading under short-day conditions and inhibition of heading under long-day conditions—supports this suggestion.

Isolation and characterization of homologs of the key genes involved in photoperiod-mediated transition to flowering in short-day plants as well as expression of the homologous genes in transgenic long-day and short-day plants will enable comparison of the molecular mechanisms that regulate the floral transition in short-day and long-day plants. In a study of this type, the SOC1 homolog from mustard, MADSA, was overexpressed in long-day (*Nicotiana sylvestris*), day-neutral (*N. tabacum*), and short-day tobacco (*N. tabacum* cv Maryland Mammoth) species (Borner et al., 2000). Analysis of the transgenic plants showed that only in the short-day cultivar could overexpression of MADSA overcome the photoperiodic barrier of floral induction. MADSA overexpression did not cause long-day and day-neutral cultivars to flower under noninductive photoperiod, suggesting a difference in the behavior of long- and short-day plants at the molecular level, even in downstream processes.

PERSPECTIVES

Genetic analysis in Arabidopsis has enabled the isolation of genes that control flowering time, and the identification of interacting pathways that promote flowering in response to different environmental conditions. However, our present understanding of these pathways represents only a skeletal framework, and future work in this area will concentrate on understanding the biochemical function of pathway components and the manner in which the signaling pathways convey information that ultimately regulates flowering time. Furthermore, the incorporation into these models of classic observations, such as the graft transmissibility of flowering signals formed in the leaf, is a major challenge.

The complexity of the data regarding the network of flowering pathways will also increase further. Some environmental effects on flowering have not been placed within the pathway structure. For example, it is not clear whether the effects of nutrient availability or light quality on flowering act through the pathways described here or through further pathways, although high sucrose levels delay flowering and reduce *FT* expression (Ohto et al., 2001). There is also physiological evidence that vernalization may act partly through an additional pathway that has so far not been described genetically (Figure 2). In addition, although there is strong genetic and molecular evidence for the separation of the photoperiod and autonomous pathways, as shown in Figure 4, there are also indications of interactions between the pathways at unexpected levels. For example, *FLC*, a major repressor within the autonomous pathway, influences circadian period length, a feature of the photoperiod pathway (Swarup et al., 1999).

Explaining the diversity in flowering-time responses also represents a great challenge. The analysis of natural variation in flowering time between ecotypes of Arabidopsis indicates that modifying the balance of the activity of the described flowering pathways can create different reproductive strategies, such as winter annual or summer annual types (Michaels and Amasino, 2000). For example, in comparing winter annual and summer annual varieties of Arabidopsis, changing the relative activities of the vernalization and photoperiod pathways generates a vernalization requirement. It seems likely that altering the balance of the activity of distinct pathways on the activation of genes such as *FT* and *SOC1* that integrate the different pathways may be a more general mechanism of generating diversity in flowering behavior. However, there are also fundamental differences among species in the roles of pathways. The comparison of photoperiod responses in long-day and short-day plants demonstrates that for this example of diversity in flowering behavior, the pathway described in Arabidopsis is likely to be relevant for species that behave very differently. However, the function of key pathway components is likely to be altered in short-day plants to change their target genes or the activity of the protein complexes in which they act. Similar changes may explain why GA promotes flowering in Arabidopsis but represses flowering in trees and some short-day plants. There are flowering strategies, however, for which Arabidopsis may be a less-effective model. For example, although Arabidopsis genes can be used to drive very early flowering of trees and to overcome developmental delays of flowering (Weigel and Nilsson, 1995; Pena et al., 2001), studies of an annual plant such as Arabidopsis will probably not reveal the molecular mechanisms underlying many of the unique features of perennial plants (Battey, 2000).

Received December 21, 2001; accepted March 4, 2002.

REFERENCES

Adams, J., Kelso, R., and Cooley, L. (2000). The kelch repeat superfamily of proteins: Propellers of cell function. Trends Cell Biol. **10**, 17–24.

Ahmad, M., Jarillo, J.A., and Cashmore, A.R. (1998). Chimeric proteins between cry1 and cry2 Arabidopsis blue light photoreceptors indicate overlapping functions and varying protein stability. Plant Cell **10**, 197–207.

Alabadi, D., Oyama, T., Yanovsky, M.J., Harmon, F.G., Mas, P., and Kay, S.A. (2001). Reciprocal regulation between TOC1 and LHY/CCA1 within the Arabidopsis circadian clock. Science **293**, 880–883.

Amador, V., Monte, E., Garcia-Martinez, J.L., and Prat, S. (2001). Gibberellins signal nuclear import of PHOR1, a photoperiod-responsive protein with homology to Drosophila armadillo. Cell **106**, 343–354.

Araki, T. (2001). Transition from vegetative to reproductive phase. Curr. Opin. Plant Biol. **4**, 63–68.

Bachmair, A., Novatchkova, M., Potuschak, T., and Eisenhaber, F. (2001). Ubiquitylation in plants: A post-genomic look at a post-translational modification. Trends Plant Sci. **6**, 463–470.

Bagnall, D.J., King, R.W., Whitelam, G.C., Boylan, M.T., Wagner, D., and Quail, P.H. (1995). Flowering responses to altered expression of phytochrome in mutants and transgenic lines of Arabidopsis thaliana (L.) Heynh. Plant Physiol. 108, 1495–1503.

Banfield, M.J., and Brady, R.L. (2000). The structure of Antirrhinum centroradialis protein (CEN) suggests a role as a kinase regulator. J. Mol. Biol. 297, 1159–1170.

Battey, N.H. (2000). Aspects of seasonality. J. Exp. Bot. 51, 1769–1780.

Birve, A., Sengupta, A.K., Beuchle, D., Larsson, J., Kennison, J.A., Rasmuson-Lestander, A., and Muller, J. (2001). Su(z)12, a novel Drosophila Polycomb group gene that is conserved in vertebrates and plants. Development 128, 3371–3379.

Blazquez, M.A., Green, R., Nilsson, O., Sussman, M.R., and Weigel, D. (1998). Gibberellins promote flowering of Arabidopsis by activating the LEAFY promoter. Plant Cell 10, 791–800.

Blazquez, M.A., and Weigel, D. (2000). Integration of floral inductive signals in Arabidopsis. Nature 404, 889–892.

Bolle, C., Koncz, C., and Chua, N.H. (2000). PAT1, a new member of the GRAS family, is involved in phytochrome A signal transduction. Genes Dev. 14, 1269–1278.

Bonhomme, F., Kurz, B., Melzer, S., Bernier, G., and Jacqmard, A. (2000). Cytokinin and gibberellin activate SaMADS A, a gene apparently involved in regulation of the floral transition in Sinapis alba. Plant J. 24, 103–111.

Borden, K.L.B. (1998). RING fingers and B-boxes: Zinc-binding protein-protein interaction domains. Biochem. Cell Biol. 76, 351–358.

Borner, R., Kampmann, G., Chandler, J., Gleissner, R., Wisman, E., Apel, K., and Melzer, S. (2000). A MADS domain gene involved in the transition to flowering in Arabidopsis. Plant J. 24, 591–599.

Bowman, J.L., Alvarez, J., Weigel, D., Meyerowitz, E.M., and Smyth, D.R. (1993). Control of flower development in Arabidopsis thaliana by APETALA1 and interacting genes. Development 119, 721–743.

Bradley, D., Ratcliffe, O., Vincent, C., Carpenter, R., and Coen, E. (1997). Inflorescence commitment and architecture in Arabidopsis. Science 275, 80–83.

Briggs, W.R., and Huala, E. (1999). Blue-light photoreceptors in higher plants. Annu. Rev. Cell Dev. Biol. 15, 33–62.

Burn, J.E., Smyth, D.R., Peacock, W.J., and Dennis, E.S. (1993a). Genes conferring late flowering in Arabidopsis thaliana. Genetica 90, 147–155.

Burn, J.E., Bagnall, D.J., Metzger, J.D., Dennis, E.S., and Peacock, W.J. (1993b). DNA methylation, vernalization, and the initiation of flowering. Proc. Natl. Acad. Sci. USA 90, 287–291.

Chandler, J., Wilson, A., and Dean, C. (1996). Arabidopsis mutants showing an altered response to vernalization. Plant J. 10, 637–644.

Chaudhury, A.M., Ming, L., Miller, C., Craig, S., Dennis, E.S., and Peacock, W.J. (1997). Fertilization-independent seed development in Arabidopsis thaliana. Proc. Natl. Acad. Sci. USA 94, 4223–4228.

Chen, L.J., Cheng, J.C., Castle, L., and Sung, Z.R. (1997). EMF genes regulate Arabidopsis inflorescence development. Plant Cell 9, 2011–2024.

Clarke, J.H., and Dean, C. (1994). Mapping FRI, a locus controlling

flowering time and vernalization response in Arabidopsis thaliana. Mol. Gen. Genet. 242, 81–89.

Coles, J.P., Phillips, A.L., Croker, S.J., Garcia-Lepe, R., Lewis, M.J., and Hedden, P. (1999). Modification of gibberellin production and plant development in Arabidopsis by sense and antisense expression of gibberellin 20-oxidase genes. Plant J. 17, 547–556.

Corbesier, L., Gadisseur, I., Silvestre, G., Jacqmard, A., and Bernier, G. (1996). Design in Arabidopsis thaliana of a synchronous system of floral induction by one long day. Plant J. 9, 947–952.

Covington, M.F., Panda, S., Liu, X.L., Strayer, C.A., Wagner, D.R., and Kay, S.A. (2001). ELF3 modulates resetting of the circadian clock in Arabidopsis. Plant Cell 13, 1305–1315.

Davis, S.J., Kurepa, J., and Vierstra, R.D. (1999). The Arabidopsis thaliana HY1 locus, required for phytochrome-chromophore biosynthesis, encodes a protein related to heme oxygenases. Proc. Natl. Acad. Sci. USA 96, 6541–6546.

del Pozo, J.C., and Estelle, M. (2000). F-box proteins and protein degradation: An emerging theme in cellular regulation. Plant Mol. Biol. 44, 123–128.

Dill, A., Jung, H.S., and Sun, T.P. (2001). The DELLA motif is essential for gibberellin-induced degradation of RGA. Proc. Natl. Acad. Sci. USA 98, 14162–14167.

Dill, A., and Sun, T.P. (2001). Synergistic derepression of gibberellin signaling by removing RGA and GAI function in Arabidopsis thaliana. Genetics 159, 777–785.

El-Assal, S.E.D., Alonso-Blanco, C., Peeters, A.J.M., Raz, V., and Koornneef, M. (2001). A QTL for flowering time in Arabidopsis reveals a novel allele of CRY2. Nat. Genet. 29, 435–440.

Finnegan, E.J., Genger, R.K., Kovac, K., Peacock, W.J., and Dennis, E.S. (1998). DNA methylation and the promotion of flowering by vernalization. Proc. Natl. Acad. Sci. USA 95, 5824–5829.

Fowler, S., Lee, K., Onouchi, H., Samach, A., Richardson, K., Coupland, G., and Putterill, J. (1999). GIGANTEA: A circadian clock-controlled gene that regulates photoperiodic flowering in Arabidopsis and encodes a protein with several possible membrane-spanning domains. EMBO J. 18, 4679–4688.

Fridborg, I., Kuusk, S., Moritz, T., and Sundberg, E. (1999). The Arabidopsis dwarf mutant shi exhibits reduced gibberellin responses conferred by overexpression of a new putative zinc finger protein. Plant Cell 11, 1019–1031.

Fridborg, I., Kuusk, S., Robertson, M., and Sundberg, E. (2001). The Arabidopsis protein SHI represses gibberellin responses in Arabidopsis and barley. Plant Physiol. 127, 937–948.

Furner, I.J., Ainscough, J.F.X., Pumfrey, J.A., and Petty, L.M. (1996). Clonal analysis of the late flowering fca mutant of Arabidopsis thaliana: Cell fate and cell autonomy. Development 122, 1041–1050.

Gendall, A.R., Levy, Y.Y., Wilson, A., and Dean, C. (2001). The VERNALIZATION 2 gene mediates the epigenetic regulation of vernalization in Arabidopsis. Cell 107, 525–535.

Gocal, G.F., Sheldon, C.C., Gubler, F., Moritz, T., Bagnall, D.J., MacMillan, C.P., Li, S.F., Parish, R.W., Dennis, E.S., Weigel, D., and King, R.W. (2001). GAMYB-like genes, flowering, and gibberellin signaling in Arabidopsis. Plant Physiol. 127, 1682–1693.

Goodrich, J., Puangsomlee, P., Martin, M., Long, D., Meyerowitz,

E.M., and Coupland, G. (1997). A polycomb-group gene regulates homeotic gene expression in Arabidopsis. Nature **386**, 44–51.

Goto, N., Kumagai, T., and Koornneef, M. (1991). Flowering responses to light-breaks in photomorphogenic mutants of Arabidopsis thaliana, a long-day plant. Plant Physiol. **83**, 209–215.

Green, R.M., and Tobin, E.M. (1999). Loss of the circadian clock-associated protein 1 in Arabidopsis results in altered clock-regulated gene expression. Proc. Natl. Acad. Sci. USA **96**, 4176–4179.

Guo, H.W., Duong, H., Ma, N., and Lin, C.T. (1999). The Arabidopsis blue light receptor cryptochrome 2 is a nuclear protein regulated by a blue light-dependent post-transcriptional mechanism. Plant J. **19**, 279–287.

Guo, H.W., Yang, W.Y., Mockler, T.C., and Lin, C.T. (1998). Regulations of flowering time by Arabidopsis photoreceptors. Science **279**, 1360–1363.

Harmer, S.L., Hogenesch, L.B., Straume, M., Chang, H.S., Han, B., Zhu, T., Wang, X., Kreps, J.A., and Kay, S.A. (2000). Orchestrated transcription of key pathways in Arabidopsis by the circadian clock. Science **290**, 2110–2113.

Hicks, K.A., Albertson, T.M., and Wagner, D.R. (2001). EARLY FLOWERING3 encodes a novel protein that regulates circadian clock function and flowering in Arabidopsis. Plant Cell **13**, 1281–1292.

Huang, S.S., Raman, A.S., Ream, J.E., Fujiwara, H., Cerny, R.E., and Brown, S.M. (1998). Overexpression of 20-oxidase confers a gibberellin-overproduction phenotype in Arabidopsis. Plant Physiol. **118**, 773–781.

Huq, E., Tepperman, J.M., and Quail, P.H. (2000). GIGANTEA is a nuclear protein involved in phytochrome signaling in Arabidopsis. Proc. Natl. Acad. Sci. USA **97**, 9789–9794.

Ishitani, M., Xiong, L.M., Lee, H.J., Stevenson, B., and Zhu, J.K. (1998). HOS1, a genetic locus involved in cold-responsive gene expression in Arabidopsis. Plant Cell **10**, 1151–1161.

Izawa, T., Oikawa, T., Tokutomi, S., Okuno, K., and Shimamoto, K. (2000). Phytochromes confer the photoperiodic control of flowering in rice (a short-day plant). Plant J. **22**, 391–399.

Jacobsen, S.E., Binkowski, K.A., and Olszewski, N.E. (1996). SPINDLY, a tetratricopeptide repeat protein involved in gibberellin signal transduction in Arabidopsis. Proc. Natl. Acad. Sci. USA **93**, 9292–9296.

Jacobsen, S.E., and Olszewski, N.E. (1993). Mutations at the SPINDLY locus of Arabidopsis alter gibberellin signal transduction. Plant Cell **5**, 887–896.

Jarillo, J.A., Capel, J., Tang, R.-H., Yang, H.-Q., Alonso, J.M., Ecker, J.R., and Cashmore, A.R. (2001). An Arabidopsis circadian clock component interacts with both CRY1 and phyB. Nature **410**, 487–490.

Jeddeloh, J.A., Stokes, T.L., and Richards, E.J. (1999). Maintenance of genomic methylation requires a SW12/SNF2-like protein. Nat. Genet. **22**, 94–97.

Johanson, U., West, J., Lister, C., Michaels, S., Amasino, R., and Dean, C. (2000). Molecular analysis of FRIGIDA, a major determinant of natural variation in Arabidopsis flowering time. Science **290**, 344–347.

Johnson, E., Bradley, M., Harberd, N.P., and Whitelam, G.C. (1994). Photoresponses of light-grown phyA mutants of Arabidopsis: Phytochrome A is required for the perception of daylength extensions. Plant Physiol. **105**, 141–149.

Kakutani, T. (1997). Genetic characterization of late-flowering traits induced by DNA hypomethylation mutation in Arabidopsis thaliana. Plant J. **12**, 1447–1451.

Kakutani, T., Jeddeloh, J.A., Flowers, S.K., Munakata, K., and Richards, E.J. (1996). Developmental abnormalities and epimutations associated with DNA hypomethylation mutations. Proc. Natl. Acad. Sci. USA **93**, 12406–12411.

Kania, T., Russenberger, D., Peng, S., Apel, K., and Melzer, S. (1997). FPF1 promotes flowering in Arabidopsis. Plant Cell **9**, 1327–1338.

Kardailsky, I., Shukla, V.K., Ahn, J.H., Dagenais, N., Christensen, S.K., Nguyen, J.T., Chory, J., Harrison, M.J., and Weigel, D. (1999). Activation tagging of the floral inducer FT. Science **286**, 1962–1965.

King, K.E., Moritz, T., and Harberd, N.P. (2001). Gibberellins are not required for normal stem growth in Arabidopsis thaliana in the absence of GAI and RGA. Genetics **159**, 767–776.

Kinoshita, T., Harada, J.J., Goldberg, R.B., and Fischer, R.L. (2001). Polycomb repression of flowering during early plant development. Proc. Natl. Acad. Sci. USA **98**, 14156–14161.

Kobayashi, Y., Kaya, H., Goto, K., Iwabuchi, M., and Araki, T. (1999). A pair of related genes with antagonistic roles in mediating flowering signals. Science **286**, 1960–1962.

Koornneef, M., Alonso-Blanco, C., Blankstijn-de Vries, H., Hanhart, C.J., and Peeters, A.J.M. (1998a). Genetic interactions among late-flowering mutants of Arabidopsis. Genetics **148**, 885–892.

Koornneef, M., Alonso-Blanco, C., Peeters, A.J.M., and Soppe, W. (1998b). Genetic control of flowering time in Arabidopsis. Annu. Rev. Plant Physiol. Plant Mol. Biol. **49**, 345–370.

Koornneef, M., Blankstijn-de Vries, H., Hanhart, C., Soppe, W., and Peeters, T. (1994). The phenotype of some late-flowering mutants is enhanced by a locus on chromosome 5 that is not effective in the Landsberg erecta wild-type. Plant J. **6**, 911–919.

Koornneef, M., Hanhart, C.J., and Van Der Veen, J.H. (1991). A genetic and physiological analysis of late flowering mutants in Arabidopsis thaliana. Mol. Gen. Genet. **229**, 57–66.

Koornneef, M., Rolff, E., and Spruit, C.J.P. (1980). Genetic control of light-inhibited hypocotyl elongation in *Arabidopsis thaliana* (L.) Heynh. Z. Pflanzenphysiol. **100**, 147–160.

Koornneef, M., and van der Veen, J.H. (1980). Induction and analysis of gibberellin sensitive mutants in *Arabidopsis thaliana* (L.) Heynh. Theor. Appl. Genet. **58**, 257–263.

Kreppel, L.K., Blomberg, M.A., and Hart, G.W. (1997). Dynamic glycosylation of nuclear and cytosolic proteins—cloning and characterization of a unique O-GlcNAc transferase with multiple tetratricopeptide repeats. J. Biol. Chem. **272**, 9308–9315.

Kreps, J.A., and Simon, A.E. (1997). Environmental and genetic effects on circadian clock-regulated gene expression in Arabidopsis. Plant Cell **9**, 297–304.

Kurup, S., Jones, H.D., and Holdsworth, M.J. (2000). Interactions of the developmental regulator ABI3 with proteins identified from developing Arabidopsis seeds. Plant J. **21**, 143–155.

Lagercrantz, U., and Axelsson, T. (2000). Rapid evolution of the

family of CONSTANS LIKE genes in plants. Mol. Biol. Evol. 17, 1499–1507.

Langridge, J. (1957). Effect of day-length and gibberellic acid on the flowering of Arabidopsis. Nature 180, 36–37.

Lee, H., Suh, S.-S., Park, E., Cho, E., Ahn, J.H., Kim, S.-G., Lee, J.S., Kwon, Y.M., and Lee, I. (2000). The AGAMOUS-LIKE 20 MADS domain protein integrates floral inductive pathways in Arabidopsis. Genes Dev. 14, 2366–2376.

Lee, H.J., Xiong, L.M., Gong, Z.Z., Ishitani, M., Stevenson, B., and Zhu, J.K. (2001). The Arabidopsis HOS1 gene negatively regulates cold signal transduction and encodes a RING finger protein that displays cold-regulated nucleo-cytoplasmic partitioning. Genes Dev. 15, 912–924.

Lee, I., Aukerman, M.J., Gore, S.L., Lohman, K.N., Michaels, S.D., Weaver, L.M., John, M.C., Feldmann, K.A., and Amasino, R.M. (1994b). Isolation of LUMINIDEPENDENS: A gene involved in the control of flowering time in Arabidopsis. Plant Cell 6, 75–83.

Lee, I., Michaels, S.D., Masshardt, A.S., and Amasino, R.M. (1994a). The late-flowering phenotype of FRIGIDA and mutations in LUMINIDEPENDENS is suppressed in the Landsberg erecta strain of Arabidopsis. Plant J. 6, 903–909.

Lee, I.A.B., and Amasino, R. (1993). Analysis of naturally occurring late flowering in Arabidopsis thaliana. Mol. Gen. Genet. 237, 171–176.

Lee, Y., Lloyd, A.M., and Roux, S.J. (1999). Antisense expression of the CK2 alpha-subunit gene in Arabidopsis. Effects on light-regulated gene expression and plant growth. Plant Physiol. 119, 989–1000.

Lin, C. (2002). Blue light receptors and signal transduction. Plant Cell 14 (suppl.), S207–S225.

Lin, H.X., Yamamoto, T., Sasaki, T., and Yano, M. (2000). Characterization and detection of epistatic interactions of 3 QTLs, Hd1, Hd2, and Hd3, controlling heading date in rice using nearly isogenic lines. Theor. Appl. Genet. 101, 1021–1028.

Liu, J.Y., Yu, J.P., McIntosh, L., Kende, H., and Zeavaart, J.A.D. (2001a). Isolation of a CONSTANS ortholog from Pharbitis nil and its role in flowering. Plant Physiol. 125, 1821–1830.

Liu, X.L., Covington, M.F., Fankhauser, C., Chory, J., and Wanger, D.R. (2001b). ELF3 encodes a circadian clock-regulated nuclear protein that functions in an Arabidopsis PHYB signal transduction pathway. Plant Cell 13, 1293–1304.

Luo, M., Bilodeau, P., Koltunow, A., Dennis, E.S., Peacock, W.J., and Chaudhury, A.M. (1999). Genes controlling fertilization-independent seed development in Arabidopsis thaliana. Proc. Natl. Acad. Sci. USA 96, 296–301.

Macknight, R., Bancroft, I., Page, T., Lister, C., Schmidt, R., Love, K., Westphal, L., Murphy, G., Sherson, S., Cobbett, C., and Dean, C. (1997). FCA, a gene controlling flowering time in Arabidopsis, encodes a protein containing RNA-binding domains. Cell 89, 737–745.

Makino, S., Kiba, T., Imamura, A., Hanaki, N., Nakamura, A., Suzuki, T., Taniguchi, M., Ueguchi, C., Sugiyama, T., and Mizuno, T. (2000). Genes encoding pseudo-response regulators: Insight into His-to-Asp phosphorelay and circadian rhythm in Arabidopsis thaliana. Plant Cell Physiol. 41, 791–803.

Martinez-Garcia, J.F., Huq, E., and Quail, P.H. (2000). Direct targeting of light signals to a promoter element-bound transcription factor. Science 288, 859–863.

Martinez-Zapater, J.M., and Somerville, C.R. (1990). Effect of light quality and vernalization on late-flowering mutants of Arabidopsis thaliana. Plant Physiol. 92, 770–776.

McClung, C.R. (2001). Circadian rhythms in plants. Annu. Rev. Plant Physiol. Plant Mol. Biol. 52, 139–162.

McWatters, H.G., Bastow, R.M., Hall, A., and Millar, A.J. (2000). The ELF3 zeitnehmer regulates light signalling to the circadian clock. Nature 408, 716–720.

Meier, C., Bouquin, T., Nielsen, M.E., Raventos, D., Mattsson, O., Rocher, A., Schomburg, F., Amasino, R.M., and Mundy, J. (2001). Gibberellin response mutants identified by luciferase imaging. Plant J. 25, 509–519.

Melzer, S., Kampmann, G., Chandler, J., and Apel, K. (1999). FPF1 modulates the competence to flowering in Arabidopsis. Plant J. 18, 395–405.

Michaels, S.D., and Amasino, R.M. (1999a). The gibberellic acid biosynthesis mutant ga1–3 of Arabidopsis thaliana is responsive to vernalization. Dev. Genet. 25, 194–198.

Michaels, S.D., and Amasino, R.M. (1999b). FLOWERING LOCUS C encodes a novel MADs domain protein that acts as a repressor of flowering. Plant Cell 11, 949–956.

Michaels, S.D., and Amasino, R.M. (2000). Memories of winter: Vernalization and the competence to flower. Plant Cell Environ. 23, 1145–1153.

Michaels, S.D., and Amasino, R.M. (2001). Loss of FLOWERING LOCUS C activity eliminates the late-flowering phenotype of FRIGIDA and autonomous pathway mutations but not responsiveness to vernalization. Plant Cell 13, 935–941.

Millar, A.J., Carre, I.A., Strayer, C.A., Chua, N.H., and Kay, S.A. (1995a). Circadian clock mutants in Arabidopsis identified by luciferase imaging. Science 267, 1161–1163.

Millar, A.J., Straume, M., Chory, J., Chua, N.H., and Kay, S.A. (1995b). The regulation of circadian period by phototransduction pathways in Arabidopsis. Science 267, 1163–1166.

Mizoguchi, T., Wheatley, K., Hanzawa, Y., Wright, L., Mizoguchi, M., Song, H.-R., Carré, I.A., and Coupland, G. (2002). LHY and CCA1 are partially redundant genes required to maintain circadian rhythms in Arabidopsis. Dev. Cell, doi/10.1016/S1534–5807(02) 00170-3.

Mozley, D., and Thomas, B. (1995). Developmental and photobiological factors affecting photoperiodic induction in Arabidopsis thaliana Heynh. Landsberg erecta. J. Exp. Bot. 46, 173–179.

Muramoto, T., Kohchi, T., Yokota, A., Hwang, I.H., and Goodman, H.M. (1999). The Arabidopsis photomorphogenic mutant hy1 is deficient in phytochrome chromophore biosynthesis as a result of a mutation in a plastid heme oxygenase. Plant Cell 11, 335–347.

Nelson, D.C., Lasswell, J., Rogg, L.E., Cohen, M.A., and Bartel, B. (2000). FKF1, a clock-controlled gene that regulates the transition to flowering in Arabidopsis. Cell 101, 331–340.

Nilsson, O., Lee, I., Blazquez, M.A., and Weigel, D. (1998). Flowering-time genes modulate the response of LEAFY activity. Genetics 150, 403–410.

Ohad, N., Yadegari, R., Margossian, L., Hannon, M., Michaeli, D., Harada, J.J., Goldberg, R.B., and Fischer, R.L. (1999). Mutations in FIE, a WD polycomb group gene, allow endosperm development without fertilization. Plant Cell 11, 407–415.

Ohto, M., Onai, K., Furukawa, Y., Aoki, E., Araki, T., and Nakamura, K. (2001). Effects of sugar on vegetative development and floral transition in Arabidopsis. Plant Physiol. 127, 252–261.

Olszewski, N., Sun, T., and Gubler, F. (2002). Gibberellin signaling: Biosynthesis, catabolism and response pathways. Plant Cell 14 (suppl.), S61–S80.

Onouchi, H., Igeno, M.I., Perilleux, C., Graves, K., and Coupland, G. (2000). Mutagenesis of plants overexpressing CONSTANS demonstrates novel interactions among Arabidopsis flowering-time genes. Plant Cell 12, 885–900.

Park, D.H., Somers, D.E., Kim, Y.S., Choy, Y.H., Lim, H.K., Soh, M.S., Kim, H.J., Kay, S.A., and Nam, H.G. (1999). Control of circadian rhythms and photoperiodic flowering by the Arabidopsis GIANTEA gene. Science 285, 1579–1582.

Patton, E.E., Willems, A.R., and Tyers, M. (1998). Combinatorial control in ubiquitin-dependent proteolysis: Don’t Skp the F-box hypothesis. Trends Genet. 14, 236–243.

Pena, L., Martin-Trillo, M., Juarez, J., Pina, J.A., Navarro, L., and Martinez-Zapater, J.M. (2001). Constitutive expression of Arabidopsis LEAFY or APETALA1 genes in citrus reduces their generation time. Nat. Biotechnol. 19, 263–267.

Peng, J.R., Carol, P., Richards, D.E., King, K.E., Cowling, R.J., Murphy, G.P., and Harberd, N.P. (1997). The Arabidopsis GAI gene defines a signaling pathway that negatively regulates gibberellin responses. Genes Dev. 11, 3194–3205.

Pineiro, M., and Coupland, G. (1998). The control of flowering time and floral identity in Arabidopsis. Plant Physiol. 117, 1–8.

Putterill, J., Robson, F., Lee, K., Simon, R., and Coupland, G. (1995). The CONSTANS gene of Arabidopsis promotes flowering and encodes a protein showing similarities to zinc finger transcription factors. Cell 80, 847–857.

Pysh, L.D., Wysocka-Diller, J.W., Camilleri, C., Bouchez, D., and Benfey, P.N. (1999). The GRAS gene family in Arabidopsis: sequence characterization and basic expression analysis of the SCARECROW-LIKE genes. Plant J. 18, 111–119.

Ratcliffe, O.J., Nadzan, G.C., Reuber, T.L., and Riechmann, J.L. (2001). Regulation of flowering in Arabidopsis by an FLC homologue. Plant Physiol. 126, 122–132.

Reed, J.W., Nagpal, P., Bastow, R.M., Solomon, K.S., Dowson-Day, M.J., Elumalai, R.P., and Millar, A.J. (2000). Independent action of elf3 and phyB to control hypocotyl elongation and flowering time. Plant Physiol. 122, 1149–1160.

Reeves, P.H., and Coupland, G. (2000). Response of plant development to environment: Control of flowering by daylength and temperature. Curr. Opin. Plant Biol. 3, 37–42.

Reeves, P.H., and Coupland, G. (2001). Analysis of flowering time control in Arabidopsis by comparison of double and triple mutants. Plant Physiol. 126, 1085–1091.

Robertson, M., Swain, S.M., Chandler, P.M., and Olszewski, N.E. (1998). Identification of a negative regulator of gibberellin action, HvSPY, in barley. Plant Cell 10, 995–1007.

Robson, F., Costa, M.M.R., Hepworth, S., Vizir, I., Pineiro, M., Reeves, P.H., Putterill, J., and Coupland, G. (2001). Functional importance of conserved domains in the flowering-time gene CONSTANS demonstrated by analysis of mutant alleles and transgenic plants. Plant J. 28, 619–631.


Roenneberg, T., and Merrow, M. (2000). Circadian clocks: Omnes viae Romam ducunt. Curr. Biol. 10, R742–R745.

Ruiz-Garcia, L., Madueno, F., Wilkinson, M., Haughn, G., Salinas, J., and Martinez-Zapater, J.M. (1997). Different roles of flowering-time genes in the activation of floral initiation genes in Arabidopsis. Plant Cell 9, 1921–1934.

Samach, A., and Coupland, G. (2000). Time measurement and the control of flowering in plants. Bioessays 22, 38–47.

Samach, A., and Gover, A. (2001). Photoperiodism: The consistent use of CONSTANS. Curr. Biol. 11, R651–R654.

Samach, A., Onouchi, H., Gold, S.E., Ditta, G.S., Schwarz-Sommer, Z., Yanofsky, M.F., and Coupland, G. (2000). Distinct roles of CONSTANS target genes in reproductive development of Arabidopsis. Science 288, 1613–1616.

Sanchez-Fernandez, R., Ardiles-Diaz, W., Van Montagu, M., Inze, D., and May, M.J. (1998). Cloning of a novel Arabidopsis thaliana RGA-like gene, a putative member of the VHIID-domain transcription factor family. J. Exp. Bot. 49, 1609–1610.

Schaffer, R., Landgraf, J., Accerbi, M., Simon, V., Larson, M., and Wisman, E. (2001). Microarray analysis of diurnal and circadian-regulated genes in Arabidopsis. Plant Cell 13, 113–123.

Schaffer, R., Ramsay, N., Samach, A., Corden, S., Putterill, J., Carre, I.A., and Coupland, G. (1998). The late elongated hypocotyl mutation of Arabidopsis disrupts circadian rhythms and the photoperiodic control of flowering. Cell 93, 1219–1229.

Schomburg, F.M., Patton, D.A., Meinke, D.W., and Amasino, R.M. (2001). FPA, a gene involved in floral induction in Arabidopsis, encodes a protein containing RNA-recognition motifs. Plant Cell 13, 1427–1436.

Scortecci, K.C., Michaels, S.D., and Amasino, R.M. (2001). Identification of a MADS-box gene, FLOWERING LOCUS M, that represses flowering. Plant J. 26, 229–236.

Sheldon, C.C., Burn, J.E., Perez, P.P., Metzger, J., Edwards, J.A., Peacock, W.J., and Dennis, E.S. (1999). The FLF MADS box gene: A repressor of flowering in Arabidopsis regulated by vernalization and methylation. Plant Cell 11, 445–458.

Sheldon, C.C., Rouse, D.T., Finnegan, E.J., Peacock, W.J., and Dennis, E.S. (2000). The molecular basis of vernalization: The central role of FLOWERING LOCUS C (FLC). Proc. Natl. Acad. Sci. USA 97, 3753–3758.

Silverstone, A.L., Ciampaglio, C.N., and Sun, T.P. (1998). The Arabidopsis RGA gene encodes a transcriptional regulator repressing the gibberellin signal transduction pathway. Plant Cell 10, 155–169.

Silverstone, A.L., Jung, H.S., Dill, A., Kawaide, H., Kamiya, Y., and Sun, T.P. (2001). Repressing a repressor: Gibberellin-induced rapid reduction of the RGA protein in Arabidopsis. Plant Cell 13, 1555–1565.

Simon, R., Igeno, M.I., and Coupland, G. (1996). Activation of floral meristem identity genes in Arabidopsis. Nature 384, 59–62.

Simpson, G.G., Gendall, A.R., and Dean, C. (1999). When to switch to flowering. Annu. Rev. Cell Dev. Biol. 15, 519–550.

Somers, D.E., Devlin, P.F., and Kay, S.A. (1998a). Phytochromes and cryptochromes in the entrainment of the Arabidopsis circadian clock. Science 282, 1488–1494.

Somers, D.E., Schultz, T.F., Milnamow, M., and Kay, S.A. (2000). ZEITLUPE encodes a novel clock-associated PAS protein from Arabidopsis. Cell 101, 319–329.

Somers, D.E., Webb, A.A.R., Pearson, M., and Kay, S.A. (1998b). The short-period mutant, toc1–1, alters circadian clock regulation of multiple outputs throughout development in Arabidopsis thaliana. Development 125, 485–494.

Soppe, W.J.J., Jacobsen, S.E., Alonso-Blanco, C., Jackson, J.P., Kakutani, T., Koornneef, M., and Peeters, A.J.M. (2000). The late flowering phenotype of fwa mutants is caused by gain-of-function epigenetic alleles of a homeodomain gene. Mol. Cell 6, 791–802.

Spillane, C., MacDougall, C., Stock, C., Kohler, C., Vielle-Calzada, J.P., Nunes, S.M., Grossniklaus, U., and Goodrich, J. (2000). Interaction of the Arabidopsis Polycomb group proteins FIE and MEA mediates their common phenotypes. Curr. Biol. 10, 1535–1538.

Strayer, C., Oyama, T., Schultz, T.F., Raman, R., Somers, D.E., Mas, P., Panda, S., Kreps, J.A., and Kay, S.A. (2000). Cloning of the Arabidopsis clock gene TOC1, an autoregulatory response regulator homolog. Science 289, 768–771.

Suarez-Lopez, P., Wheatley, K., Robson, F., Onouchi, H., Valverde, F., and Coupland, G. (2001). CONSTANS mediates between the circadian clock and the control of flowering in Arabidopsis. Nature 410, 1116–1120.

Sugano, S., Andronis, C., Green, R.M., Wang, Z.-Y., and Tobin, E.M. (1998). Protein kinase CK2 interacts with and phosphorylates the Arabidopsis circadian clock-associated 1 protein. Proc. Natl. Acad. Sci. USA 95, 11020–11025.

Sugano, S., Andronis, C., Ong, M.S., Green, R.M., and Tobin, E.M. (1999). The protein kinase CK2 is involved in regulation of circadian rhythms in Arabidopsis. Proc. Natl. Acad. Sci. USA 96, 12362–12366.

Sun, T.P., and Kamiya, Y. (1994). The Arabidopsis ga1 locus encodes the cyclase ent-kaurane synthetase-A of gibberellin biosynthesis. Plant Cell 6, 1509–1518.

Sung, Z.R., Belachew, A., Shunong, B., and Bertrand-Garcia, R. (1992). EMF, an Arabidopsis gene required for vegetative shoot development. Science 258, 1645–1647.

Swarup, K., Alonso-Blanco, C., Lynn, J.R., Michaels, S.D., Amasino, R.M., Koornneef, M., and Millar, A.J. (1999). Natural allelic variation identifies new genes in the Arabidopsis circadian system. Plant J. 20, 67–77.

Takahashi, Y., Shomura, A., Sasaki, T., and Yano, M. (2001). Hd6, a rice quantitative trait locus involved in photoperiod sensitivity, encodes the alpha subunit of protein kinase CK2. Proc. Natl. Acad. Sci. USA 98, 7922–7927.

Takeba, G., and Takimoto, A. (1966). Translocation of floral stimulus in *Pharbitis nil*. Bot. Mag. 79, 811–814.

Talon, M., Koornneef, M., and Zeevaart, J.A.D. (1990). Accumulation of C19-gibberellins in the gibberellin-insensitive dwarf mutant gai of Arabidopsis thaliana (L.) Heynh. Planta 182, 501–505.

Thomas, B., and Vince-Prue, B. (1997). Photoperiodism in Plants, 2nd ed. (San Diego, CA: Academic Press).

Vince-Prue, D., and Gressel, J. (1985). Pharbitis nil. In Handbook of flowering, A. Halevy, ed (Boca Raton, FL: CRC Press), pp. 47–81.

Wagner, D., Sablowski, R.W.M., and Meyerowitz, E.M. (1999).

Transcriptional activation of APETALA1 by LEAFY. Science 285, 582–584.

Wang, Z.-Y., and Tobin, E.M. (1998). Constitutive expression of the CIRCADIAN CLOCK ASSOCIATED 1 (CCA1) gene disrupts circadian rhythms and suppresses its own expression. Cell 93, 1207–1217.

Weigel, D., and Nilsson, O. (1995). A developmental switch sufficient for flower initiation in diverse plants. Nature 377, 495–500.

Wen, C.-K., and Chang, C. (2002). Arabidopsis RGL1 encodes a negative regulator of gibberellin responses. Plant Cell 14, 87–100.

Wilson, R.N., Heckman, J.W., and Somerville, C.R. (1992). Gibberellin is required for flowering in Arabidopsis thaliana under short days. Plant Physiol. 100, 403–408.

Xu, Y.L., Gage, D.A., and Zeevaart, J.A.D. (1997). Gibberellins and stem growth in Arabidopsis thaliana—effects of photoperiod on expression of the GA4 and GA5 loci. Plant Physiol. 114, 1471–1476.

Xu, Y.L., Li, L., Wu, K.Q., Peeters, A.J.M., Gage, D.A., and Zeevaart, J.A.D. (1995). The ga5 locus of Arabidopsis thaliana encodes a multifunctional gibberellin 20-oxidase—molecular cloning and functional expression. Proc. Natl. Acad. Sci. USA 92, 6640–6644.

Yadegari, R., Kinoshita, T., Lotan, O., Cohen, G., Katz, A., Choi, Y., Nakashima, K., Harada, J.J., Goldberg, R.B., Fischer, R.L., and Ohad, N. (2000). Mutations in the FIE and MEA genes that encode interacting polycomb proteins cause parent-of-origin effects on seed development by distinct mechanisms. Plant Cell 12, 2367–2381.

Yamamoto, T., Lin, H.X., Sasaki, T., and Yano, M. (2000). Identification of heading date quantitative trait locus Hd6 and characterization of its epistatic interactions with Hd2 in rice using advanced backcross progeny. Genetics 154, 885–891.

Yang, C.-H., Chen, L.-J., and Sung, Z.R. (1995). Genetic regulation of shoot development in Arabidopsis: Role of the EMF genes. Dev. Biol. 169, 421–435.

Yano, M., Harushima, Y., Nagamura, Y., Kurata, N., Minobe, Y., and Sasaki, T. (1997). Identification of quantitative trait loci controlling heading date in rice using a high-density linkage map. Theor. Appl. Genet. 95, 1025–1032.

Yano, M., Katayose, Y., Ashikari, M., Yamanouchi, U., Monna, L., Fuse, T., Baba, T., Yamamoto, K., Umehara, Y., Nagamura, Y., and Sasaki, T. (2000). Hd1, a major photoperiod sensitivity quantitative trait locus in rice, is closely related to the Arabidopsis flowering time gene CONSTANS. Plant Cell 12, 2473–2483.

Yanovsky, M.J., Mazzella, M.A., and Casal, J.J. (2000). A quadruple photoreceptor mutant still keeps track of time. Curr. Biol. 10, 1013–1015.

Yoshida, N., Yanai, Y., Chen, L., Kato, Y., Hiratsuka, J., Miwa, T., Sung, Z.R., and Takahashi, S. (2001). EMBRYONIC FLOWER2, a novel polycomb group protein homolog, mediates shoot development and flowering in Arabidopsis. Plant Cell 13, 2471–2481.

Zagotta, M.T., Shannon, S., Jacobs, C., and Meekswagner, D.R. (1992). Early-flowering mutants of Arabidopsis thaliana. Aust. J. Plant Physiol. 19, 411–418.

Control of Flowering Time: Interacting Pathways as a Basis for Diversity  
Aidyn Mouradov, Frédéric Cremer and George Coupland  
Plant Cell 2002;14;S111-S130  
DOI 10.1105/tpc.001362  

This information is current as of November 28, 2012  

| References | This article cites 167 articles, 95 of which can be accessed free at: http://www.plantcell.org/content/14/suppl_1/S111.full.html#ref-list-1 |
| Permissions | https://www.copyright.com/ccc/openurl.do?sid=pd_hw1532298X&issn=1532298X&WT.mc_id=pd_hw1532298X |
| eTOCs | Sign up for eTOCs at: http://www.plantcell.org/cgi/alerts/ctmain |
| CiteTrack Alerts | Sign up for CiteTrack Alerts at: http://www.plantcell.org/cgi/alerts/ctmain |
| Subscription Information | Subscription Information for The Plant Cell and Plant Physiology is available at: http://www.aspb.org/publications/subscriptions.cfm |

© American Society of Plant Biologists  
ADVANCING THE SCIENCE OF PLANT BIOLOGY
