
Microbiology (2002), 148, 1845–1853

**Isolation and characterization of 14 additional genes specifying the anaerobic biosynthesis of cobalamin (vitamin B₁₂) in Propionibacterium freudenreichii (P. shermanii)**

Charles A. Roessner,$^{1}$ Ke-xue Huang,$^{1\dagger}$ Martin J. Warren,$^{2}$ Evelyne Raux$^{2}$ and A. Ian Scott$^{1}$

Author for correspondence: Charles A. Roessner. Tel: +1 979 845 3243. Fax: +1 979 845 5992.  
e-mail: c-roessner@tamu.edu

---

**A search for genes encoding enzymes involved in cobalamin (vitamin B₁₂) production in the commercially important organism Propionibacterium freudenreichii (P. shermanii) has resulted in the isolation of an additional 14 genes encoding enzymes responsible for 17 steps of the anaerobic B₁₂ pathway in this organism. All of the genes believed to be necessary for the biosynthesis of adenosylcobinamide from uroporphyrinogen III have now been isolated except two (cbiA and an as yet unidentified gene encoding cobalt reductase). Most of the genes are contained in two divergent operons, one of which, in turn, is closely linked to the operon encoding the B₁₂-dependent enzyme methylmalonyl-CoA mutase. The close linkage of the three genes encoding the subunits of transcarboxylase to the hemYHBXRL gene cluster is reported. The functions of the P. freudenreichii B₁₂ pathway genes are discussed, and a mechanism for the regulation of cobalamin and propionic acid production by oxygen in this organism is proposed.**

---

**Keywords:** hem operon, methylmalonyl-CoA mutase, transcarboxylase, metabolic regulation, genetic linkage

---

**INTRODUCTION**

Vitamin B₁₂ is produced commercially by two organisms, *Pseudomonas denitrificans* and *Propionibacterium freudenreichii* (*P. shermanii*) (Vorobjeva, 2000). The genes that determine the aerobic biosynthetic route to B₁₂ in *Ps. denitrificans* were isolated by complementation of mutants, and most of the encoded enzymes have been assigned functions (Table 1) (for reviews see Blanche *et al.*, 1995; Roessner *et al.*, 2001). Genes for the biosynthesis of B₁₂ via an alternative anaerobic pathway have been isolated and sequenced from other bacteria having systems amenable to genetic manipulation such as *Salmonella typhimurium* (Roth *et al.*, 1993) and *Bacillus megaterium* (Raux *et al.*, 1998). Recently, B₁₂ biosynthesis genes of both the aerobic and anaerobic pathways have been revealed in several other Eubacteria.

and Archaea as the result of genomic sequencing projects and have been annotated on the basis of sequence similarities.

Prior to the age of recombinant DNA technology, the majority of research on cobalamin biosynthesis was undertaken with extracts of *P. freudenreichii* (see, for example, Battersby, 1979; Scott, 1979) with little attention paid to molecular genetics or control and regulation of the pathway. Even two of the more recent studies (Scott *et al.*, 1996; Wang *et al.*, 1996) on the biosynthesis of the vitamin in *P. freudenreichii* were performed with little knowledge of the genetics of this organism. Indeed, only a few genes have so far been isolated from *P. freudenreichii*, including four, *hemL, hemB, cobA* and *cbiO*, from the early part of the B₁₂ pathway (Hashimoto *et al.*, 1996; Sattler *et al.*, 1995). We report here the isolation and sequencing of an additional 14 *P. freudenreichii* genes homologous to 17 genes encoding proteins involved in vitamin B₁₂ biosynthesis in *S. typhimurium* and *B. megaterium*, describe their positions relative to other known *P. freudenreichii* genes, and demonstrate the activity of two of them

---

**Present address:** Protein Engineering, logen Corp., 400 Hunt Club Road, Ontario, Ottawa, Canada K1V 1C1.

The GenBank accession numbers for the sequences reported in this paper are AY033235, AY033236, U13043 and U51164.

C. A. Roessner and others

Table 1. Reactions and genes required for the transformation of 5-aminolaevulinic acid (ALA) into adenosylcobalamin

The known differences between the aerobic pathway (exemplified by *P. denitrificans*) and the anaerobic pathway (exemplified by *S. typhimurium*) with respect to gene nomenclature are shown. Differences in *P. freudenreichii* gene nomenclature are shown in parentheses after the *S. typhimurium* gene. See the text and the note on nomenclature in the Introduction for additional details.

| Aerobic pathway (*P. denitrificans*) | Gene | Anaerobic pathway (*S. typhimurium, P. shermanii*) | Gene |
|--------------------------------------|------|----------------------------------------------------|------|
| Aminolaevulinic acid                 |      | Aminolaevulinic acid                             |      |
| ↓ condensation of 2 ALA              | 1. hemB | ↓ condensation of 2 ALA                          | hemB |
| Porphobilinogen                      |      | Porphobilinogen                                  |      |
| ↓ tetrapyrrole formation             | 2. hemC | ↓ tetrapyrrole formation                         | hemC |
| Hydroxymethylbilane                  |      | Hydroxymethylbilane                              |      |
| ↓ cyclization and ring D inv.        | 3. hemD | ↓ cyclization and ring D inv.                    | hemD |
| Uroporphyrinogen III                 |      | Uroporphyrinogen III                             |      |
| ↓ methylation at C-2 and C-7         | 4. cobA | ↓ methylation at C-2 and C-7                     | cysG (cobA) |
| Precorrin-2                          |      | Precorrin-2                                      |      |
| ↓ methylation at C-20               | 5. cobI | ↓ cobalt insertion                               | cysG (cbiX-cysGB) |
| Precorrin-3                          |      | Cobalt-Precorrin-2                               |      |
| ↓ C-20 hydroxylation,                | 6. cobG | ↓ methylation at C-20                            | cbiL |
| ↓ γ-lactone formation                |      |                                                  |      |
| Precorrin-3 hydroxylactone           |      | Cobalt-Precorrin-3                               |      |
| ↓ C-17 methylation                   | 7. cobJ | ↓ C-17 methylation                              | cbiH (cbiEGH) |
| ↓ ring contraction                   |      | ↓ ring contraction                               |      |
| ↓ δ-lactone formation                |      |                                                  |      |
| Precorrin-4                          |      | Cobalt-Precorrin-4                               |      |
| ↓ C-11 methylation                   | 8. cobM | ↓ C-11 methylation                              | cbiF |
| Precorrin-5                          |      | Cobalt-Precorrin-5                               |      |
| ↓ C-1 methylation                    | 9. cobF | ↓ C-1 methylation                               | not known |
| ↓ acetic acid extrusion              |      | ↓ acetaldehyde extrusion                         |      |
| Precorrin-6                          |      | Cobalt-Precorrin-6                               |      |
| ↓ C-18–19 reduction                  | 10. cobK | ↓ C-18–19 reduction                            | cbiJ |
| Dihydro-Precorrin-6                  |      | Cobalt-Dihydro-Precorrin-6                       |      |
| ↓ C-5, 15 methylation                | 11a. cobL | ↓ C-5, 15 methylation                          | cbiE (cbiEGH) |
| ↓ decarboxylation                    | 11b cobL | ↓ decarboxylation                              | cbiT |
| Precorrin-8                          |      | Cobalt-Precorrin-8                               |      |
| ↓ methyl rearrangement               | 12. cobH | ↓ methyl rearrangement                          | cbiC |
| Hydrogenobyrinic acid                |      | Cobyric acid                                     |      |
| ↓ a,c amidation                      | 13. cobB | ↓ a,c amidation                                | cbiA |
| Hydrogenobyrinic acid a,c-diamide    |      | Cob(II)yric acid a,c-diamide                     |      |
| ↓ cobalt insertion                   | 14. cobNST |                                              |      |
| Cob(II)yric acid a,c-diamide         |      |                                                  |      |
| ↓ cobalt reduction                   | 15. not known | ↓ cobalt reduction                           | not known |
| Cob(I)yric acid a,c-diamide          |      | Cob(I)yric acid a,c-diamide                      |      |
| ↓ adenosylation                      | 16. cobO | ↓ adenosylation                                | cobA (cobA^adoT) |
| ado-Cob(I)yric acid a,c-diamide       |      | ado-Cob(I)yric acid a,c-diamide                  |      |
| ↓ b,d,e,g amidation                  | 17. cobQ | ↓ b,d,e,g amidation                            | cbiP |
| ado-Cobyric acid                     |      | ado-Cobyric acid                                 |      |
| ↓ aminopropanol attachment           | 18. cobC,D | ↓ aminopropanol attachment                    | cbiB |
| ado-Cobinamide                       |      | ado-Cobinamide                                   |      |
| ↓ phosphorylation                    | 19a. cobP | ↓ phosphorylation                              | cobU |
| ↓ GMP addition                       | 19b. cobP | ↓ GMP addition                                 | cobU |
| ado-GDP-Cobinamide                   |      | ado-GDP-Cobinamide                               |      |
| ↓ α-ribazole addition                | 20. cobV | ↓ α-ribazole addition                           | cobS |
| Ado-Cobalamin                        |      | Ado-Cobalamin                                    |      |
| α-ribazole synthesis                 | cobU | α-ribazole synthesis                             | cobT |
| from DBI and NaMN                    |      | from DBI and NaMN                                |      |

through complementation of mutants. This information will not only allow engineered overproduction of the cobalamin biosynthetic enzymes and the deduction of pathway intermediates, but will also permit the design of experiments to monitor the function of the pathway enzymes under a range of different physiological conditions.

Note on nomenclature. Since it has been demonstrated that *Ps. denitrificans* requires O₂ for the synthesis of B₁₂ and that *S. typhimurium* synthesizes B₁₂ only under strict anaerobic conditions, the nomenclatures assigned to the *Ps. denitrificans* genes (*cobA–V*) and the *S. typhimurium* genes (*cysG*, *cbiA–Q* and *cobSTU*) have become associated with the ‘aerobic’ and ‘anaerobic’ pathways to B₁₂, respectively (see Table 1). *P. freudenreichii* synthesizes B₁₂ anaerobically; therefore the *S. typhimurium* nomenclature will be used herein as much as possible. Unfortunately, no convention was followed when naming the genes from different organisms, with result that many of the genes encoding the same enzyme have different names whereas some of the genes with the same name encode different enzymes in the two organisms (e.g. *cobA*, *cobB*, *cobC*, *cobD*, *cobS*, *cobT*, *cobU*). Since two *P. freudenreichii cobA* genes are described in this manuscript, they will be distinguished by using *cobA* for the gene encoding uroporphyrinogen III methyltransferase, based on prior usage (Sattler et al., 1995), and *cobA*<sup>adoT</sup> for the gene encoding ATP: corrinoid adenosyltransferase.


---

### METHODS

#### Strains and plasmids. These are listed in Table 2.

#### Isolation, sequencing and subcloning of B₁₂ biosynthesis genes. The cobalamin biosynthetic genes were isolated from a genomic *P. freudenreichii* library contained within bacteriophage lambda EMBL 4 previously used for isolation of the gene encoding pyrophosphate-dependent phosphofructokinase (Ladror et al., 1991). EMBL 4 clones containing the cobalamin biosynthetic genes were identified by screening the library with two labelled probes (*cbiT* probe and *cbiP* probe) that were generated by PCR amplification of portions of the *P. freudenreichii cbiT* and *cbiP* genes using *P. freudenreichii* genomic DNA as template and primers derived from conserved DNA sequences found within the similar genes of *S. typhimurium* and *Ps. denitrificans*. An additional probe was generated by PCR amplification of a portion of the *P. freudenreichii cbiO* gene using primers derived from the *P. freudenreichii cbiO* gene sequence, which was previously reported to be contiguous with *cobA* (Sattler et al., 1995). The PCR products were labelled with the Genius I nonradioactive labelling kit from Boehringer Mannheim. Screening and subcloning of restriction fragments of the phage inserts into pBluescript for sequencing were performed using standard techniques (Maniatis et al., 1982). Sequences were determined using an ABI model 373A sequencer and the dideoxy termination method (Sanger et al., 1977) with fluorescent dye detection. Sequence analysis was performed with software provided by the Genetics Computer Group, and protein similarities were determined using the BLAST program provided by the National Center for Biotechnology Information.


---

### Table 2. Plasmids and strains used in this study

| Strain or plasmid | Description                                                                 | Source or reference                     |
|-------------------|-----------------------------------------------------------------------------|-----------------------------------------|
| **Plasmids**      |                                                                                 |                                         |
| pUC19             | ColE1, *plac*, Ap<sup>R</sup>                                              | Yanisch-Perron et al. (1985)           |
| pET23a(+)         | ColE1, *pT7*, Ap<sup>R</sup>                                               | Novagen                                |
| pKK223.3          | ColE1, *ptac*, Ap<sup>R</sup>                                              | Pharmacia                              |
| pBluescript       | ColE1, *plac*, Ap<sup>R</sup>                                              | Stratagene                             |
| pISA417           | pUC19:*cobA*<sup>*</sup>                                                   | Sattler et al. (1995)                  |
| pCR487            | pUC19:*cbiX-cysG*<sup>B+</sup>                                             | This study                             |
| pCR488            | pUC19:*cbiX-cysG*<sup>B:</sup>*cobA*<sup>*</sup>                           | This study                             |
| pCR571            | pET23a(+):*cbiEGH*<sup>+</sup>                                            | This study                             |
| pER*cbiL*         | pKK223.3:*cbiL*<sup>+</sup>                                                | This study                             |
| **E. coli strains** |                                                                                 |                                         |
| TB1               | General cloning strain                                                     | Baldwin et al. (1984)                  |
| BL21DE3           | Provides T7 polymerase for expression from pET vectors                      | Novagen                                |
| AR3711            | *cbiL* mutant used for complementation studies                               | Raux et al. (1996)                     |
| AR3717            | *cbiL* mutant used for complementation studies                               | Raux et al. (1996)                     |
| CB302Δa           | *cysG* deletion mutant used for complementation studies                      | Griffiths & Cole (1987)                |
| CR487             | TB1(pCR487) used for analysis of the CbiX-CysGB protein                      | This study                             |
| CR571             | BL21DE3(pCR571) used for analysis of the CbiEGH protein                       | This study                             |

* *P. freudenreichii cobA* gene encoding uroporphyrinogen III methyltransferase.
+ *P. freudenreichii* genes described in this paper.

C. A. Roessner and others

(a)

![Diagram](attachment:diagram_a.png)

(b)

Methylmalonyl-
CoA mutase
mutB     mutA    orf1   cbiL    cbiF    cbiE + cbiG + cbiH    cbiX + cysG^B    cbiD    cbiC    cbiT    cbiJ    araD    bcp

![Diagram](attachment:diagram_b.png)

(c)

cbiB     cbiP    cobA^adoT

![Diagram](attachment:diagram_c.png)

(d)

hemY     hemH    hemB    antibiotic^r    reg.    hemL    5S    12S    1·3S

![Diagram](attachment:diagram_d.png)

Fig. 1. The structure of four fragments of *P. freudenreichii* DNA containing vitamin B₁₂ biosynthetic genes. The arrows indicate regions of previously unreported sequence which have been assigned the following GenBank accession numbers: (a) U13043; (b) AY033236; (c) AY033235; (d) derived from U51164 and D85417 (Hashimoto et al., 1996).

For SDS-PAGE analysis of the gene products of the *cbiX-cysG^B* and *cbiEGH* genes described herein (Fig. 1), the genes were amplified by PCR providing optimal *Escherichia coli* translational signals and inserted into expression vectors as previously described (Roessner et al., 1995). The sequence of the *cbiX-cysG^B* forward primer was: CCCGGGGGTGCGAC-AAGCTTAGGAATTTAAAATGACTGATCTCGTCCC-ACTGGTCATTGCC, providing a 5′ *HindIII* site (underlined), a ribosome-binding site (italics) and codons for the first 10 amino acids of the protein including the ATG start codon. The sequence of the *cbiX-cysG^B* reverse primer was CCCGG-GTCTAGAGGATCCTTAGACCTGGAGCGCC-TTGAG-GAGCTCGTCGCG, providing a 3′ *BamHI* site (underlined), a stop anticodon (TTA) and anticodons for the last 10 amino acids of the protein. The PCR product was cut with *HindIII* and *BamHI* and ligated into pUC19 to give pCR487. The sequence of the *cbiEGH* forward primer was GGTACCC-GGGGATCCAGGAGGAATTTAAAATGATTCGCGTA-CATGGTTTCCTCGGCGGC, providing a 5′ *BamHI* site (underlined), a ribosome-binding site (italics) and codons for the first 10 amino acids of the protein including the ATG start codon. The sequence of the *cbiEGH* reverse primer was CGCGCAAGCTTTTAGTCATGGGAATCCTCCTGGG-TGGGGACATCGGAGCC, providing a 3′ *HindIII* site (underlined), a stop anticodon (TTA) and anticodons for the last 12 amino acids of the protein. The PCR product was cut with *BamHI* and *HindIII* and ligated into pET23a(+) to form pCR571.

For complementation studies, the *cbiX-cysG^B* PCR product was also inserted in front of the *P. freudenreichii cobA* gene in pISA417 (Sattler et al., 1995) to give pCR488. Similarly, the *P. freudenreichii cbiL* gene was inserted into pKK223.3 for complementation studies of the *cbiL* mutants AR3711 and AR3717 as previously described (Raux et al., 1996). Plasmids

pCR487 and pCR571 were transformed into *E. coli* strains TB1 and BL21DE3, respectively, to allow protein overproduction of CbiX–CysGB and CbiEGH. The cells were grown at 37 °C in LB medium containing 50 μg ampicillin ml⁻¹ to OD₆₀₀ 0·8, induced by the addition of IPTG to 1·0 mM, and incubated at 37 °C for an additional 16 h. SDS-PAGE analysis was performed as described previously (Roessner et al., 1995).

### RESULTS AND DISCUSSION

#### Isolation and sequence of *P. freudenreichii* vitamin B₁₂ biosynthetic genes

Screening of a *P. freudenreichii* genomic library in bacteriophage lambda resulted in isolation of four phages: one that hybridized with the *cbiO* probe, one that hybridized with the *cbiP* probe, and two that hybridized with the *cbiT* probe. Sequencing of the inserts of these phages has revealed a number of ORFs with a high degree of similarity to known *cbi* genes. These genes appear to be clustered into three discrete regions of the genome as described below and shown in Fig. 1(a–c). In total we have now identified 16 genes including *cobA*, *cbiB*, *C*, *D*, *E–G–H*, *F*, *J*, *L*, *M*, *N*, *O*, *P*, *Q*, *T*, *cbiX–cysG^B* and *cobA^adoT*. With the exception that *cbiA* (encoding cobyrinic acid α,β-diamide synthase) and an unidentified gene encoding a cobalt reductase are yet to be found, these genes encode all of the enzymes believed to be responsible for the transformation of uroporphyrinogen III into adenosylcobinamide in *P. freudenreichii* (Fig. 2).

Vitamin B₁₂ biosynthesis in *P. freudenreichii*

(a)

![Diagram](attachment:diagram_a.png)

Uroporphyrinogen III

Co-Precorrin-2

Co-Precorrin-3

Co-Precorrin-4

Co-Precorrin-5

Co-Precorrin-6

Co-dihydro-Precorrin-6

Co-Precorrin-8

Cobyrinic Acid

A = CH₂COOH, P = CH₂CH₂COOH

(b)

![Diagram](attachment:diagram_b.png)

Cob(II)yric acid

Cob(I)yric acid-a,c-diamide

Adenosylcob(I)yric acid a,c-diamide

Adenosylcobyric acid

Adenosylcobinamide

Adenosylcobalamin

cyanide

Vitamin B₁₂

Fig. 2. Proposed pathway for the conversion of uroporphyrinogen III to adenosylcobinamide in *P. freudenreichii*. (a) The pathway from uroporphyrinogen III to cobyrinic acid. (b) The pathway from cobyrinic acid to adenosylcobinamide.

C. A. Roessner and others

# P. freudenreichii has a cobalt transport system similar to that of S. typhimurium

We previously reported the isolation and sequence of the *P. freudenreichii* *cobA* gene (Sattler *et al.*, 1995) and the presence of the *cbiO* gene at its 5′ end. Extension of this sequence 5′ to *cbiO* has now revealed the presence of the *cbiM*, *cbiN* and *cbiQ* genes (Fig. 1a). The homologues of these genes in *S. typhimurium* are found in a B₁₂ operon and, as discussed by Roth *et al.* (1993), are believed to encode a cobalt transport system. The association of these genes with *cobA*, encoding uroporphyrin III methyltransferase, an enzyme essential for the biosynthesis of both sirohaem (see below) and B₁₂ in *P. freudenreichii*, suggests that this operon may be specific for B₁₂ biosynthesis and that another *cobA* or *cysG* locus may exist for sirohaem synthesis.

## The genes necessary for the conversion of precorrin 2 to cobyrinic acid in *P. freudenreichii* are contained in two convergent operons

In *Ps. denitrificans* the eight genes required for the aerobic conversion of precorrin 2 to hydrogenobyrinic acid (*cobF–M*) are contiguous (Crouzet *et al.*, 1990). Similarly, the genes of *S. typhimurium* and *B. megaterium* believed to be necessary for the anaerobic conversion of cobalt-precorrin 2 to cobyrinic acid are found in large B₁₂ operons (Roth *et al.*, 1993; Raux *et al.*, 1998). Within an 11·6 kb fragment of *P. freudenreichii* DNA (Fig. 1b), we found genes homologous to 10 different genes of *S. typhimurium* (*cysG*ᴮ, *cibC*, *cbiD*, *cbiE*, *cbiF*, *cbiG*, *cbiH*, *cbiJ*, *cbiL* and *cbiT*) and to one gene of *B. megaterium* (*cbiX*). However, since some of the genes are combined into a single larger gene in *P. freudenreichii*, the two operons encode only eight proteins (see Fig. 1b and discussed below).

CysG has been proposed to be a multifunctional enzyme for the synthesis of sirohaem in *E. coli* and *S. typhimurium* (Spencer *et al.*, 1993; Warren *et al.*, 1994). The C-terminal domain of CysG (*CysG*ᴬ) is a methyltransferase that catalyses the attachment of the first two methyl groups to uroporphyrinogen III to form precorrin 2, whereas the N-terminal domain (*CysG*ᴮ) is believed to catalyse the NADP-dependent dehydrogenation of the macrocycle and iron chelation to give sirohaem. Genetic evidence has been provided (Fazzio & Roth, 1996; Raux *et al.*, 1998) that, in *S. typhimurium*, CysG may also catalyse not only the bismethylation of uroporphyrinogen III but also the insertion of cobalt into precorrin 2 (Fig. 2a) to give cobalt-precorrin 2, an intermediate unique to the anaerobic vitamin B₁₂ pathway (cobalt insertion occurs at a much later stage in the aerobic pathway into hydrogenobyrinic acid *a,c*-diamide to form cobyrinic acid *a,c*-diamide, the structure of which is shown in Fig. 2b). However, it has also been demonstrated that *S. typhimurium* has an additional gene, *cbiK*, that can substitute for *cysG*ᴮ (Raux *et al.*, 1997) and may encode the actual cobaltochelatase. A gene encoding uroporphyrinogen III methyltransferase was previously isolated from *P. freudenreichii*

---

**Fig. 3.** SDS-PAGE analysis of the gene products of the *cbiX–cysG*ᴮ and *cbiEGH* genes of *P. freudenreichii*. Lane 1, TB1(pUC19). Lane 2, TB1(pUC19:*cbiX–cysGB*); the overexpressed protein is indicated by the arrow, predicted molecular mass 44·3 kDa. Lane 3, BL21DE3(pET23a); the arrow indicates endogenous β-galactosidase, molecular mass 107 kDa. Lane 4, BL21DE3(pET23a:*cbiEGH*); the overexpressed protein is indicated by the arrow, predicted molecular mass 91·4 kDa. Lane 5, molecular mass markers. From the top: 66, 45, 36, 29 and 24 kDa.

(Sattler *et al.*, 1995) which, in contrast to *cysG*, contained no dehydrogenase or chelatase (*cysG*ᴮ) encoding region. It was therefore called *cobA* rather than *cysG* after the similar gene in *Ps. denitrificans*. The *P. freudenreichii* homologue to *cysG*ᴮ is found within a gene whose product is similar not only to *cysG*ᴮ but also to *cbiX* found in *B. megaterium* (Raux *et al.*, 1998). The *P. freudenreichii* gene encodes a protein of 414 amino acids, the C-terminal 150 of which are similar (32% identity, 50% similarity) to CysGᴮ, and the N-terminal 264 of which are similar to CbiX. Further evidence that *cbiX–cysG*ᴮ is a single gene was provided by SDS-PAGE analysis of the overproduced protein which showed (Fig. 3) that its molecular mass is consistent with the value (44·3 kDa) calculated from the amino acid sequence. In an experiment similar to that reported previously for the analysis of *S. typhimurium* *cbiK* gene (Raux *et al.*, 1997), plasmids for the expression of the *P. freudenreichii* *cobA* and *cbiX–cysG*ᴮ genes, either by themselves or together, were constructed and tested for the ability to complement an *E. coli cysG* deletion mutant (CB302Δa). The mutant was complemented by a plasmid (pCR488) bearing both genes but not by a plasmid bearing either *cobA* (pISA417) or *cbiX–cysG*ᴮ (pCR487) by itself. Based on these results, we propose that *cbiX–cysG*ᴮ encodes dehydrogenase and/or chelatase activities necessary for cobalamin biosynthesis in *P. freudenreichii*, although the precise functions of the two domains have not yet been firmly established.

The functions of the enzymes encoded by the other seven genes have been deduced from homology with the corresponding enzymes from *Ps. denitrificans* and *S. typhimurium* (Table 1). Based on its similarity to CobI,
CbiL has been predicted to be the C-20 methylase in *S. typhimurium* and, thus, *P. freudenreichii*. The *S. typhimurium* enzyme has been shown to have methyltransferase activity (Roessner *et al.*, 1992) and is believed to methylate C-20 of cobalt-precorrin 2 to yield cobalt-precorrin 3 (Fig. 2a). A plasmid, pERcbiL, bearing the *P. freudenreichii cbiL* gene was able to complement two different *cbiL* mutants (AR3711 and AR3717) of an engineered *E. coli* strain that synthesizes cobinamide using the *S. typhimurium cbi* genes. The homologous gene, *cobI*, from the aerobic *Ps. denitrificans* pathway does not complement the same mutants (Raux *et al.*, 1996), which is most likely a reflection of the difference in the substrates for the two encoded enzymes (precorrin 2 for CobI and cobalt precorrin 2 for CbiL).

In *S. typhimurium*, *cbiE*, *cbiG* and *cbiH* are all individual genes, but in *P. freudenreichii* their homologues are found in a single ORF (Fig. 1b). Evidence that *cbiEGH* is a single gene was also provided by SDS-PAGE analysis of the overproduced protein, which showed (Fig. 3) that its molecular mass is consistent with the value (91·4 kDa) calculated from the amino acid sequence. In *Synechocystis* sp., *cbiG* and *cbiH* are also apparently found in one gene (Kaneko *et al.*, 1996). CbiH of *S. typhimurium* has recently been shown to be responsible for C-17 methylation of cobalt-precorrin 3 (Fig. 2a), leading to formation of the ring-contracted, lactonized intermediate, cobalt-precorrin 4 (Santander *et al.*, 1997). Although the function of CbiG is unknown, the occurrence of CbiG and CbiH as a single enzyme in two different organisms suggests that CbiG may operate either immediately before or immediately after CbiH. However, the association of CbiE with CbiG and CbiH in *P. freudenreichii* is not as easily rationalized since it is believed that, based on its similarity to CobL, CbiE catalyses the methylation of C-5 and C-15 but only after methylation of C-11 and C-1, extrusion of acetaldehyde, and reduction of the macrocycle (Fig. 2a). C-11 methylation is catalysed by CbiF (Roessner *et al.*, 1992) but the enzyme responsible for C-1 methylation and extrusion of acetaldehyde has yet to be identified. The corresponding process (C-1 methylation and extrusion of acetic acid) in *Ps. denitrificans* is catalysed by CobF, for which no homologous enzyme exists in *S. typhimurium*. No gene corresponding to CobF was found within the *P. freudenreichii* operons reported here. In the anaerobic pathway, these steps are presumably carried out by one of the other methyltransferases, perhaps CbiF, or by CbiD as suggested by Raux *et al.* (1998). On the basis of homology with the corresponding enzymes of *Ps. denitrificans*, CbiJ is responsible for the NADPH-dependent reduction of the C-18,19 double bond, CbiT catalyses decarboxylation of the C-12 acetate, and CbiC catalyses the final step in the biosynthesis of cobyrinic acid, migration of the methyl group at C-11 to C-12 (Fig. 2a).

### Other B₁₂ biosynthesis genes

In addition to the genes required for the conversion of uroporphyrinogen III to cobyrinic acid, we have isolated a fragment of *P. freudenreichii* DNA (Fig. 1c) bearing genes encoding enzymes similar to CbiB, CbiP, and CobA<sup>adot</sup> of *S. typhimurium* (Roth *et al.*, 1993; Suh & Escalante-Semerena, 1993) involved in the attachment of aminopropanol, amidation of the *b*, *d*, *e*, and *g* acetate and propionate side chains, and transfer of an adenosyl group to the cobalt metal ion (Fig. 2b). The CobA adenosyltransferase is also similar to BtuR of *E. coli* (Lundrigan & Kadner, 1989), responsible for adenylation of imported B₁₂. The *P. freudenreichii* genes for the biosynthesis of adenosylcobalamin from 5-aminolaevulinic acid that have not yet been isolated include the early-pathway genes *hemC*, *hemD* and *cbiA*, and the late-pathway genes *cobU*, *cobS* and *cobT* (see Table 1 for functions). In addition, the gene encoding the enzyme necessary for the reduction of cobalt(II) to cobalt(I) prior to adenylation has not yet been determined in any organism.

### Other *P. freudenreichii* genes (non-B₁₂)

During our search for B₁₂ biosynthesis genes, we rediscovered partial sequences for most of the *P. freudenreichii* genes previously isolated, which has allowed the positioning of these genes relative to the sequences described herein. Hashimoto *et al.* (1996) reported the isolation of an 8 kb fragment of DNA bearing the *hemY*, *hemH*, *hemB* (5-aminolaevulinic acid dehydratase) and *hemL* (glutamate-1-semialdehyde aminotransferase) genes and two additional, non-B₁₂ biosynthesis genes encoding a probable membrane-associated antibiotic-resistance protein and its regulator. We extended this sequence in the 3′ direction and discovered a portion of the gene encoding the 5S subunit of methylmalonyl-CoA transcarboxylase (Thornton *et al.*, 1993a). Since the genes encoding the 12S and 1·3S subunits of the transcarboxylase are contiguous with the 5S gene (Thornton *et al.*, 1993b), the structure of the 12 kb fragment of *P. freudenreichii* DNA bearing all of these genes may be deduced as shown in Fig. 1(d).

The 3′ end of the fragment bearing most of the B₁₂ biosynthesis genes (Fig. 1b) also harbours the *araD* and *bcp* genes encoding proteins similar to L-ribulose-5-phosphate 4-epimerase and bacterioferritin comigratory protein, respectively. Of particular interest, however, is the presence of another operon at the 5′ end of this fragment, divergently transcribed from the B₁₂ operon, containing the *mutA* and *mutB* genes that encode the two subunits of the adenosylcobalamin-dependent enzyme methylmalonyl-CoA mutase (Marsh *et al.*, 1989). A similar arrangement of operons occurs in *S. typhimurium* (Roth *et al.*, 1993), in which the major B₁₂ operon is divergently transcribed from the operon encoding propanediol dehydratase, encoded by the *pduC* and *pduD* genes (Walter *et al.*, 1997), which is also B₁₂ dependent. In *S. typhimurium*, B₁₂ is produced only anaerobically (Jeter *et al.*, 1984) and production of the vitamin is also inhibited by exogenous B₁₂. Both operons are positively regulated by the PocR protein (Bobik *et al.*, 1992; Rondon & Escalante-Semerena, 1992), the gene for which lies between the two operons, and whose
C. A. Roessner and others

GAGCCAGGAGACCTGCC Ps. denitrificans cobP gene
|||||| ||||||||||
AAGCCAGAAGACCTGCC S. typhimurium cob operon
|| ||| ||||||||||
AAGCCCAGAAGACCTGCC S. typhimurium btuB operon
|| |||| ||||||||||
386 GAGTCCGAAGACCTGCC 371 P. freudenreichii B12 operon

Fig. 4. Homology between a proposed vitamin B₁₂ binding consensus sequence and 17 bases found between the P. freudenreichii methylmalonyl-CoA mutase and vitamin B₁₂ operons.

level is highest under anaerobic conditions in the presence of propanediol. Upstream from the B₁₂ operon lies a 'B₁₂ box', a short conserved sequence also found before the Ps. denitrificans cobP gene and the E. coli btuB gene (Roth et al., 1993). B₁₂ has been shown to regulate the synthesis of BtuB by binding to the B₁₂ box in the mRNA, thus interfering with ribosomal binding (Nou & Kadner, 2000).

By analogy with the regulatory system in S. typhimurium, it is interesting to suggest that the region between the divergently transcribed methylmalonyl-CoA mutase and B₁₂ operons of P. freudenreichii described here (Fig. 1b) would be an ideal location for the observed inhibition of B₁₂ production by exogenous B₁₂ and of the inhibition of propionate and B₁₂ production by oxygen (Quesada-Chanto et al., 1998; Vorobjeva, 2000). Methylmalonyl-CoA mutase is a key enzyme in the production of propionic acid (see Vorobjena, 2000, for a review of metabolic pathways in propionibacteria), and a decrease in the enzyme level in the presence of oxygen, by inhibiting synthesis either of the cofactor or of the apoenzyme, would result in decreased propionate levels. In the P. freudenreichii sequence reported here, there is a small ORF (Fig. 1b, orf1) between the first known gene (cbiL) of the B₁₂ operon and the first gene of the methylmalonyl-CoA mutase operon (mutA). This ORF could encode a protein involved in the regulation of the two operons. The encoded protein, however, has no similarity to PocR or any other known protein. Within orf1, however, there is a 17 base sequence that is a close match to the 'B₁₂ box' sequence (Fig. 4) which may be responsible for regulation of cobalamin production by exogenous B₁₂ in P. freudenreichii.

### ACKNOWLEDGEMENTS

We thank R. G. Kemp for providing the P. freudenreichii genomic library in bacteriophage lambda EMBL 4, Chris Brooks for technical assistance, and Aventis (Romainville, France) and NIH for financial support through MERIT award no. 2 R01 DK32034-18 to A.I.S.

### REFERENCES

Baldwin, T. O., Berends, T., Bunch, T. A., Holtzman, T. F., Rausch, S. K., Shamansky, L. & Treat, M. L. (1984). Cloning of the luciferase structural genes from Vibrio harveyi and expression of bioluminescence in Escherichia coli. Biochemistry 23, 3663–3667.

Battersby, A. R. (1979). Recent biosynthetic researches on vitamin B₁₂. In Vitamin B12, pp. 217–246. Edited by B. Zagalak & W. Friedrich. New York: Walter de Gruyter.

Blanche, F., Cameron, B., Crouzet, J., Debussche, L., Thibaut, D., Vuilhorgne, M., Leeper, F. J. & Battersby, A. R. (1995). Vitamin B₁₂: how the problem of its biosynthesis was solved. Angew Chem Int Ed Engl 34, 383–411.

Bobik, T. A., Ailion, M. & Roth, J. R. (1992). A single regulatory gene integrates control of vitamin B₁₂ synthesis and propanediol degradation. J Bacteriol 174, 2253–2266.

Crouzet, J., Cameron, B., Cauchois, L., Blanche, F., Rigault, S., Rouyez, M.-C., Thibaut, D. & Debussche, L. (1990). Genetic and sequence analysis of an 8·7-kilobase Pseudomonas denitrificans fragment involved in transformation of precorrin-2 to cobyrinic acid. J Bacteriol 172, 5980–5990.

Fazzio, T. G. & Roth, J. R. (1996). Evidence that the CysG protein catalyzes the first reaction specific to B₁₂ synthesis in Salmonella typhimurium, insertion of cobalt. J Bacteriol 178, 6952–6959.

Griffiths, L. A. & Cole, J. A. (1987). Lack of redox control of the anaerobically-induced nirB⁺ gene of Escherichia coli K-12. Arch Microbiol 147, 364–369.

Hashimoto, Y., Yamashita, M. & Murooka, Y. (1996). The Propionibacterium freudenreichii hemYHBXRL gene cluster which encodes enzymes and a regulator involved in the biosynthetic pathway from glutamate to protoheme. Appl Microbiol Biotechnol 47, 385–392.

Jeter, R., Olivera, B. M. & Roth, J. R. (1984). Salmonella typhimurium synthesizes cobalamin (vitamin B₁₂) de novo under anaerobic growth conditions. J Bacteriol 159, 206–216.

Kaneko, T., Sato, S., Tanaka, A. & 21 other authors (1996). Sequence analysis of the genome of the unicellular cyanobacterium Synechocystis sp. strain PCC6803. II. Sequence determination of the entire genome and assignment of potential protein-coding regions. DNA Res 3, 109–136.

Ladror, U. S., Gollapudi, L., Tripathi, R. L., Latshaw, S. P. & Kemp, R. G. (1991). Cloning, sequencing and expression of pyrophosphate-dependent phosphofructokinase from Propionibacterium freudenreichii. J Biol Chem 266, 16550–16555.

Lundrigan, M. D. & Kadner, R. J. (1989). Altered cobalamin metabolism in Escherichia coli btuR mutants affects btuB gene regulation. J Bacteriol 171, 154–161.

Maniatis, T., Fritsch, E. F. & Sambrook, J. (1982). Molecular Cloning: a Laboratory Manual. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory.

Marsh, E. N., McKie, M., Davis, N. K. & Leadlay, P. F. (1989). Cloning and structural characterization of the genes coding for adenosylcobalamin-dependent methylmalonyl-CoA mutase from Propionibacterium shermanii. Biochem J 260, 345–352.

Nou, X. & Kadner, R. J. (2000). Adenosylcobalamin inhibits ribosome binding to btuB RNA. Proc Natl Acad Sci USA 97, 7190–7195.

Quesada-Chanto, A., Silveira, M. M., Schmid-Meyer, A. C., Schroeder, A. G., DaCosta, J. P. C. L., Lopez, J., Carvalho-Jonas, M. F., Artolozaga, M. J. & Jonas, R. (1998). Effect of the oxygen supply on pattern of growth and corrinoid and organic acid production of Propionibacterium shermanii. Appl Microbiol Biotechnol 49, 732–736.

Raux, E., Lanois, A., Levillayer, F., Warren, M. J., Brody, E., Rambach, A. & Thermes, C. (1996). Salmonella typhimurium cobalamin (vitamin B₁₂) biosynthetic genes: functional studies in S. typhimurium and Escherichia coli. J Bacteriol 178, 753–767.

Raux, E., Beck, R., Levillayer, F., Rambach, A., Thermes, C. & Warren, M. J. (1997). A role for Salmonella typhimurium cbiK in cobalamin (vitamin B₁₂) and siroheme biosynthesis. *J Bacteriol* 179, 3202–3212.

Raux, E., Lanois, A., Warren, M. J., Rambach, A. & Thermes, C. (1998). Cobalamin (vitamin B₁₂) biosynthesis: identification and characterization of a Bacillus megaterium cobI operon. *Biochem J* 335, 159–166.

Roessner, C. A., Warren, M. J., Santander, P. J., Atshaves, B. P., Ozaki, S., Stolowich, N. J., Iida, K. & Scott, A. I. (1992). Expression of 9 Salmonella typhimurium enzymes for cobinamide biosynthesis: identification of the 11-methyl and 20-methyl transferases of corrin biosynthesis. *FEBS Lett* 301, 73–78.

Roessner, C. A., Spencer, J. B., Ozaki, S. & 7 other authors (1995). Overexpression in Escherichia coli of twelve vitamin B₁₂ biosynthetic enzymes. *Protein Expr Purif* 6, 155–163.

Roessner, C. A., Santander, P. J. & Scott, A. I. (2001). Multiple biosynthetic pathways for vitamin B₁₂: variations on a central theme. In Vitamins and Hormones, vol. 61, pp. 267–297. Edited by G. Litwack & T. Begley. San Diego, CA: Academic Press.

Rondon, M. R. & Escalante-Semerena, J. C. (1992). The poc locus is required for 1,2-propanediol-dependent transcription of the cobalamin biosynthetic (cob) and propanediol utilization (pdu) genes of Salmonella typhimurium. *J Bacteriol* 174, 2267–2272.

Roth, J. R., Lawrence, G. J., Rubenfield, M., Kieffer-Higgins, S. & Church, G. M. (1993). Characterization of the cobalamin (vitamin B₁₂) biosynthetic genes of Salmonella typhimurium. *J Bacteriol* 175, 3303–3316.

Sanger, F., Nicklen, S. & Coulson, A. R. (1977). DNA sequencing with chain-terminating inhibitors. *Proc Natl Acad Sci USA* 74, 3303–3316.

Santander, P. J., Roessner, C. A., Stolowich, N. J., Holderman, M. T. & Scott, A. I. (1997). How corrinoids are synthesized without oxygen: Nature’s first pathway to vitamin B₁₂. *Chem Biol* 4, 659–666.

Sattler, I., Roessner, C. A., Stolowich, N. J., Hardin, S. H., Harris-Haller, L. W., Yokubaitis, N. T., Murooka, Y., Hashimoto, Y. & Scott, A. I. (1995). Cloning, sequencing and expression of the uroporphyrinogen III methyltransferase cobA gene of Propionibacterium freudenreichii (shermanii). *J Bacteriol* 177, 1564–1569.

Scott, A. I. (1979). Intermediary metabolism of cobyrinic acid. In

Vitamin B12, pp. 247–277. Edited by B. Zagalak & W. Friedrich. New York: Walter de Gruyter.

Scott, A. I., Stolowich, N. J., Wang, J., Gawatz, O., Fridrich, E. & Müller, G. (1996). Biosynthesis of vitamin B₁₂: factor IV, a new intermediate in the anaerobic pathway. *Proc Natl Acad Sci USA* 93, 14316–14319.

Spencer, J. B., Stolowich, N. J., Roessner, C. A. & Scott, A. I. (1993). The Escherichia coli cysG gene encodes the multifunctional enzyme, siroheme synthase. *FEBS Lett* 355, 57–60.

Suh, S. & Escalante-Semerena, J. C. (1993). Cloning, sequencing and overexpression of cobA which encodes ATP:corrinoid adenosyltransferase in Salmonella typhimurium. *Gene* 129, 93–97.

Thornton, C. G., Kumar, G. K., Shenoy, B. C. & 7 other authors (1993a). Primary structure of the 5S subunit of transcarboxylase as deduced from the genomic DNA structure. *FEBS Lett* 330, 191–196.

Thornton, C. G., Kumar, G. K., Haase, F. C. & 7 other authors (1993b). Primary structure of the monomer of the 12S subunit of transcarboxylase as deduced from DNA and characterization of the product expressed in Escherichia coli. *J Bacteriol* 175, 5301–5308.

Vorobjeva, L. I. (2000). *Propionibacteria*. Dordrecht: Kluwer.

Walter, D., Ailion, M. & Roth, J. (1997). Genetic characterization of the pdu operon: use of 1,2-propanediol in Salmonella typhimurium. *J Bacteriol* 179, 1013–1022.

Wang, J., Stolowich, N. J., Santander, P. J., Park, J.-H. & Scott, A. I. (1996). Biosynthesis of vitamin B12: concerning the identity of the two-carbon fragment eliminated during anaerobic formation of cobyrinic acid. *Proc Natl Acad Sci USA* 93, 14320–14322.

Warren, M. J., Bolt, E. L., Roessner, C. A., Scott, A. I., Spencer, J. B. & Woodcock, S. C. (1994). Gene dissection reveals that the E. coli cysG gene encodes a multifunctional enzyme. *Biochem J* 302, 837–844.

Yanisch-Perron, C., Vieira, J. & Messing, J. (1985). Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. *Gene* 33, 103–119.

Received 22 October 2001; revised 28 January 2002; accepted 27 February 2002.
