
Review

**Human cytomegalovirus immediate early proteins and cell growth control**

Jonathan P. Castillo${}^{a}$, Timothy F. Kowalik${}^{a,b,*}$

${}^{a}$Program in Immunology and Virology, University of Massachusetts Medical School, 55 Lake Avenue North, Worcester, MA 01655, USA  
${}^{b}$Department of Molecular Genetics and Microbiology, University of Massachusetts Medical School, 55 Lake Avenue North, Worcester, MA 01655, USA  

Received 26 October 2001; received in revised form 11 February 2002; accepted 13 March 2002  
Received by A.J. van Wijnen  

---

### Abstract

It is widely accepted that small DNA tumor viruses, such as adenovirus, simian virus 40 and papillomavirus, push infected cells into S-phase to facilitate the replication of their genome. Until recently, it was believed that the large DNA viruses (i.e. herpesviruses) functioned very differently in this regard by inducing a G₁ arrest in infected cells as part of their replication process. However, studies over the last 6–8 years have uncovered striking parallels (and differences) between the functions of the major immediate early (IE) proteins of at least one herpesvirus, human cytomegalovirus (HCMV) and IE equivalents encoded by small DNA tumor viruses, such as adenovirus. Similarities between the HCMV major IE proteins and adenovirus IE proteins include targeting of members of the RB and p53 families and an ability of these viral factors to induce S-phase in quiescent cells. However, unlike the small DNA tumor virus proteins, individual HCMV IE proteins target different RB family members. HCMV also encodes several other IE gene products as well as virion tegument proteins that act early during infection to prevent an infected cell from replicating its host genome and from undergoing apoptosis. Here, we review the specifics of several HCMV IE proteins, two virion components, and their functions in relation to cell growth control. © 2002 Elsevier Science B.V. All rights reserved.

#### Keywords: Human cytomegalovirus; Immediate early proteins; Cell cycle; Retinoblastoma; p53; Apoptosis

---

### 1. Introduction: HCMV

Human cytomegalovirus (HCMV) is a member of the Herpesviridae family of viruses. Much of the molecular biology of and diseases associated with HCMV infection have been reviewed in detail (Mocarski and Courcelle, 2001; Pass, 2001). We will summarize the biological and pathological features of HCMV and then focus on the associations among the HCMV immediate early (IE) gene products, contributions from two virion-associated factors and the mechanisms governing cell cycle control. HCMV is an enveloped beta-herpesvirus with an approximately 230 kb double-stranded DNA genome containing approximately 200 open reading frames (ORFs). The HCMV genome is divided into two segments, designated UL (unique long) and US (unique short), bounded by inverted repeats.

Although the virus is endemic within the human population, HCMV infection rarely causes symptomatic disease in healthy, immunocompetent individuals. HCMV, like other herpesviruses, can establish lifelong latency following primary infection. The reactivation of HCMV commonly occurs in immunocompromised and immunosuppressed individuals and has been shown to be the causative agent of a variety of disorders in these individuals. HCMV-associated pneumonia and retinitis are the most prevalent HCMV-associated problems detected following primary infection or reactivation of latent HCMV (Ho, 1982). In addition, reactivation of latent virus or transmission of the virus to organ transplant recipients may result in further complications such as disseminated viremia and in some instances, organ dysfunction (Pass, 2001). HCMV also poses a serious threat to the health of HIV-positive individuals because HCMV may accelerate the development of AIDS as well as contribute to the morbidity associated with increased immunodeficiency. Likewise, HCMV infection can be problematic for pregnant women and children, especially infants. HCMV is recognized as the most common

---

#### Abbreviations:
CDK, cyclin-dependent kinase; cIAPs, cellular inhibitors of apoptosis; crs, cis-repression sequence; HCMV, human cytomegalovirus; IE, immediate early; IRS, internal repeat sequence; ND10, nuclear domain 10; ORFs, open reading frames; PML, promyelocytic leukemia factor; PODs, PML oncogenic domains; RB, Retinoblastoma protein; SUMO, small ubiquitin-like modifier; TBP, TATA-binding protein; TRS, terminal repeat sequence; UL, unique long; US, unique short

* Corresponding author. Program in Immunology and Virology, University of Massachusetts Medical School, 55 Lake Avenue North, Worcester, MA 01655, USA.  
E-mail address: timothy.kowalik@umassmed.edu (T.F. Kowalik).

0378-1119/02/$ - see front matter © 2002 Elsevier Science B.V. All rights reserved.  
PII: S0378-1119(02)00566-8

congenital viral infection and is the leading cause of various neurological abnormalities associated with an infectious agent during early childhood (Ho, 1982). Infants congenitally infected with HCMV are more prone to hearing loss, chorioretinitis, and other disorders involving the perceptual organs (e.g. inner ears and eyes) and the central nervous system (Ho, 1982).

Recent evidence suggests a link between HCMV and restenosis, a coronary artery disorder. Several groups have demonstrated an association between HCMV infection and the overproliferation and migration of arterial smooth muscle cells along the vessel wall intima, both of which are characteristic of restenosis following balloon angioplasty (Speir et al., 1994, 1995; Zhou et al., 1996). In addition, it was shown that HCMV-seropositive patients are more likely to develop restenosis following balloon angioplasty as compared to HCMV-seronegative patients (Zhou et al., 1996). Moreover, HCMV DNA and protein can be detected in smooth muscle cells grown from restenotic lesions (Speir et al., 1994). The expression of the HCMV US28 gene product enhances the migration of smooth muscle cells, providing yet another link between HCMV and restenosis (Streblow et al., 1999). Taken together, these findings implicate HCMV as a potential contributor to the events leading to restenosis.

There have been a plethora of studies conducted to assess the relationship between HCMV infection and human cancer. Although HCMV does not appear to be oncogenic, numerous studies have demonstrated that HCMV can transform both human (Geder et al., 1976) and rodent embryo fibroblasts (Albrecht and Rapp, 1973; Boldogh et al., 1978) in vivo. Several regions of HCMV genome have transforming ability, including some of the IE genes. These IE genes and their oncoproteins are discussed in detail in the subsequent sections of this review. That HCMV DNA and antigens have both been detected in tumor tissues isolated from patient biopsies and elevated viral antibody titers within these patients suggest a potential link between HCMV and several malignancies including cervical carcinoma, prostate cancer, and adenocarcinoma of the colon (Doniger et al., 1999). Despite these observations, the etiological association between HCMV and these various forms of human cancer remains an enigma. Clearly, the ubiquitous nature of HCMV within the population would lead one to expect much higher incident rates for each of the malignancies linked to the virus. In light of this, it has been suggested that HCMV may act as a co-etiologic agent in the development of tumors under conditions not well understood at this point. Given the transforming potential of HCMV and only indirect evidence associating the virus with cancer, it has been postulated that HCMV may indirectly contribute to oncogenesis through a ‘hit-and-run’ mechanism (Shen et al., 1997; Lukac and Alwine, 1999). Proponents of this mechanism hypothesize that HCMV essentially acts as a tumor promoter, perhaps by causing genetic instability or by preventing cells from undergoing apoptosis, and that the

virus is subsequently dispensed of at some point during the oncogenic process. All told, it may be some time before a clear understanding of the relationship between HCMV and oncogenesis is apparent.

## 2. Background: IE proteins

### 2.1. General background

Expression of HCMV genes occurs in a temporal order analogous to the other members of the Herpesviridae family. The first set of viral gene products to be expressed are classified as IE, followed by the expression of early genes, and finally, the late gene products. The IE genes do not require de novo protein synthesis for their expression. The most abundantly transcribed and best-characterized IE gene products originate from sequences encoded in the major IE region located within the UL segment of the viral genome (Fig. 1). Specifically, the major IE genes are encoded by ORFs that are under the control of the major IE promoter. Transcription from the major IE promoter through this region gives rise to several spliced mRNA species. The initial and most abundant transcript originates from the UL123 region and gives rise to a spliced 1.95 kb mRNA composed of exons 1 through 4 and encodes a 491 aa (72 kDa) nuclear phosphoprotein referred to as IE72 (also known as IE1 or IE1-72). Transcription through the other IE gene, UL122, gives rise to two major transcripts, a 2.25 and a 1.7 kb mRNA, that have the same first three exons as in the IE72 mRNA but contain a novel exon, exon 5, in place of exon 4 as a result of alternative splicing (Fig. 1). The 2.25 kb mRNA encodes a 579 aa (82–86 kDa) nuclear protein, IE86 (also referred to as IE2 or IE2-86), and the 1.7 kb mRNA encodes for a 425 aa (55 kDa) protein, IE55 (also known as IE2-55). IE55 is identical to IE86 except for a 154 aa deletion between residues aa 365 and 519 resulting from a splicing event within exon 5. Transcription from a cryptic start site within exon 5 generates a transcript that encodes for a 338 aa (40 kDa) protein that is expressed as a late gene product. Because all three of the HCMV IE proteins contain the same first three exons, they all share the same 85 aa in their N-terminal sequence. However, the remaining sequences in each of the IE proteins differ and likely account for the divergent activities exhibited by each protein.

### 2.2. Similarities between HCMV IE proteins and the oncoproteins encoded by the small DNA tumor viruses

There is a wealth of evidence that suggests a direct relationship between the oncogenic capacity of certain small DNA tumor viruses such as adenovirus, simian virus 40 (SV-40), and human papillomavirus (HPV), and their ability to modulate the cell cycle of an infected host cell. The capacity of these viruses to transform cells is dependent on the interaction of their viral oncoproteins with key

J.P. Castillo, T.F. Kowalik / Gene 290 (2002) 19–34

![Diagram](attachment:diagram.png)

Fig. 1. HCMV IE proteins and their location within the genome. Schematic diagram representing the organization of the HCMV DNA genome and the relative position of the ORFs encoding the IE gene products described in this review. Green-colored objects represent transcripts originating from the UL36-38 ORF and their respective gene products. Black-colored objects represent the major IE gene transcripts and their respective gene products. Blue-colored objects represent transcripts originating from the repeat sequences and their respective gene products. U<sub>L</sub>, unique long; U<sub>S</sub>, unique short; major immediate early promoter, major immediate early promoter (shown in box with vertical lines). Arrows represent transcripts and broken lines depict splicing events/sites. Colored bars represent translated regions.

tumor suppressor proteins including the Retinoblastoma (RB) family of proteins and p53. The RB protein family consists of three members – pRb, p107, and p130 – whose function in part is to maintain cells in a quiescent state as well as regulate the transition from G<sub>0</sub>/G<sub>1</sub>- to S-phase by modulating the activity of the E2F family of transcription factors. The E2F proteins play an essential role in controlling the expression of genes required for DNA replication. Active hypophosphorylated RB proteins repress both E2F activity and the expression of E2F target genes by binding to E2F proteins at their C-terminal transactivation domain. When RB proteins become hyperphosphorylated by cyclin/cyclin-dependent kinase (CDK) complexes activated during mid-late G<sub>1</sub>, the RB proteins can no longer bind E2F proteins leading to derepression of E2F target genes (reviewed in Nevins et al., 1997; Nevins, 1998).

Each of the small DNA tumor viruses encode proteins that can bind to the RB proteins, displace their interaction with E2F, and alleviate the repression of the E2F proteins. The E1A protein of adenovirus, large T-antigen of SV-40, and the E7 protein of HPV each contain an LxCxE motif that facilitates their binding to the RB ‘pocket domain’ thereby displacing the E2F proteins from their interaction with the RB proteins in this region (reviewed in Trimarchi and Lees, 2002). The E2F proteins can then transactivate the promoters of their target genes, including genes required for S-phase. Since each of the small DNA tumor viruses infect quiescent cells and lack certain components needed to replicate their viral DNA, it is thought that targeting the RB proteins allows for the induction of S-phase genes and generates an environment conducive to viral DNA replication.

In response to aberrant growth signals or cellular stresses, such as viral infection, the p53 protein becomes activated. The p53 protein functions primarily as a transcriptional activator with target genes including the CDK inhibitor, p21, as well as Mdm2, a negative regulator of p53. Activation of p53 induces an arrest of the cell cycle, which is mainly due to the induction of p21 and/or apoptosis. To prevent the infected cell from undergoing p53-mediated growth arrest or apoptosis, the small DNA viruses express proteins that form a complex with and inactivate p53. The adenovirus E1B 55 kDa protein and the large T-antigen of SV-40 bind p53 and inhibit its function (Sarnow et al., 1982; Bargonetti et al., 1992; Yew and Berk, 1992; Lowe and Ruley, 1993; McCarthy et al., 1994). In contrast, the HPV E6 protein induces the degradation of p53 through the ubiquitin-dependent proteolytic system (Scheffner et al., 1990). In all of these instances, expression of the viral oncoproteins alters p53 function and/or levels within infected cells to override the potential host response against the viruses. Thus, the ability to disrupt RB-mediated repression of E2F and abrogate p53 activity facilitates the DNA replication of each of the small DNA tumor viruses.

2.2.1. Targeting of p107 by HCMV IE72 and its effect on the cell cycle

Two studies demonstrate that one of the IE gene products, IE72, interacts and modulates p107, a member of the RB protein family (Poma et al., 1996; Johnson et al., 1999) (Table 1). p107 protein levels increase following HCMV infection and complexes between IE72 and p107 are detected in extracts from HCMV-infected fibroblasts. It has also been demonstrated that IE72 interacts with p107 but not pRb and that expression of IE72 was sufficient to overcome p107-mediated repression of an E2F-responsive promoter (Poma et al., 1996). A subsequent study revealed that expression of the IE72 protein is sufficient to overcome p107-mediated growth suppression and that the ability of IE72 to alleviate p107 function is dependent on its interaction with p107 (Johnson et al., 1999).

IE72 appears to exhibit kinase activity (Margolis et al., 1995; Pajovic et al., 1997). In vitro kinase assays demonstrated that both p107 and p130 can serve as substrates for IE72 and that phosphorylation of these two RB proteins by IE72 is sufficient to disrupt their interaction with E2F4 (Pajovic et al., 1997). In addition to p107 and p130, E2F proteins—E2F-1, E2F-2, and E2F-3—were also shown to be phosphorylated by IE72 in vitro. The significance of these events to the cell cycle is not known. However, these findings imply that IE72 specifically targets at least one, and possibly two, of the RB proteins for inactivation, which causes the derepression of E2F-responsive promoters.

Studies to elucidate the effect of IE72 on the host cell cycle have shown that IE72 can modulate the cell cycle under certain conditions. Early studies showed that transient expression of IE72 cDNA in quiescent COS-1 cells induces a modest increase in the number of cells in S-phase (Poma et al., 1996). Subsequently, it was reported that IE72 expression is unable to induce S-phase entry in primary fibroblasts containing wild-type p53 but could induce S-phase in p53-deficient fibroblasts (Castillo et al., 2000). The ability of p53 to block an IE72-mediated induction of S-phase is dependent on the p21 CDK inhibitor (Castillo and Kowalik, 2001). Although IE72 does not appear to interact with p53, expression of IE72 in cells induces p53 protein accumulation, which may account for the observed growth arrest (Castillo and Kowalik, 2001).

2.2.2. Targeting of pRb and p53 by HCMV IE86 and its effect on the cell cycle

Analogous to IE72, numerous studies have shown that IE86 interacts with a specific member of the RB family of proteins, in this case, pRb (Hagemeier et al., 1994; Sommer et al., 1994; Choi et al., 1995; Fortunato et al., 1997) (Table 1). Although IE72 and IE86 can interact with RB family members, p107 and pRB respectively, their inability to bind

Table 1  
Cell cycle-associated proteins found to interact with HCMV IE72 and IE86  

| Proteins interacting with IE72 | Significance          | Reference               |
|--------------------------------|-----------------------|-------------------------|
| p107                          | Cell cycle modulation | Poma et al., 1996       |
| PML                           | Not known             | Ahn et al., 1998        |
| E2F-1                         | Transactivation       | Margolis et al., 1995   |
| E2Fs-2-5                      | Transactivation       | Azizkhan et al., 2000   |
| Sp-1                          | Cell cycle modulation? | Yurochko et al., 1997   |
| CTF-1                         | Cell cycle modulation? | Hayhurst et al., 1995   |
| SUMO-1                        | Not known             | Spengler et al., 2000   |
| HDaxx                         | Disruption of ND10/PODs | Woodhall et al., 2001   |

| Proteins interacting with IE86 | Significance          | Reference                               |
|--------------------------------|-----------------------|-----------------------------------------|
| pRb                           | Cell cycle modulation | Hagemeier et al., 1994; Sommer et al., 1994 |
| p53                           | Cell cycle modulation | Speir et al., 1994; Tsai et al., 1996    |
| TBP                           | Transactivation       | Caswell et al., 1993; Jupp et al., 1993b  |
| TFII B                        | Transactivation       | Caswell et al., 1993                     |
| TFII D                        | Transactivation       | Hagemeier et al., 1992                   |
| TAFII 110                     | Transactivation       | Lukac et al., 1997                       |
| TAFII 130                     | Transactivation       | Kim et al., 2000                         |
| CREB                          | Transactivation       | Lang et al., 1995                        |
| SUMO-1/hSMT3b                 | Transactivation?/unknown | Hofmann et al., 2000a                    |
| Sp-1/PU-1                     | Transactivation       | Lukac et al., 1994                       |
| Egr-1                         | Transactivation       | Yoo et al., 1996                         |
| Tef-1                         | Transactivation       | Lukac et al., 1994                       |
| c-jun                         | Transactivation       | Scully et al., 1995                      |
| Jun B                         | Transactivation       | Scully et al., 1995                      |
| CBP                           | Transactivation       | Schwartz et al., 1996                    |
| P/CAF                         | Transactivation       | Bryant et al., 2000                      |
| MCM3, MCM7                    | DNA synthesis         | Murphy and Stinski, 1999                 |
to the RB family members at their pocket domains is most likely attributed to the absence of the consensus pocket domain-binding motif, LxCxE. But analogous to the onco-proteins encoded by the small DNA tumor viruses, expression of IE86 was sufficient to alleviate pRb repression of E2F-responsive promoters (Fortunato et al., 1997).

In addition to IE86 binding to pRb, several groups have demonstrated that IE86 can also interact with p53 (Speir et al., 1994; Tsai et al., 1996; Bonin and McDougall, 1997). IE86 binds p53 via its N-terminus and this interaction inhibits the transactivation function of p53 (Tsai et al., 1996). IE86 has also been shown to co-immunoprecipitate with p53 in cells co-expressing both proteins, and expression of IE86 was found to inhibit the ability of p53 to transactivate both a p53-responsive promoter as well as a p53-specific target gene in cultured smooth muscle cells (Speir et al., 1994; Tanaka et al., 1999). In contrast to the observations suggesting that IE86 downregulates p53 activity, one study reported that a cell line expressing an inducible IE86 fails to repress the transactivation of several p53 target genes, including p21 and Mdm2 (Bonin and McDougall, 1997). However, it should be noted that subsequent analysis of the IE86 cDNA in this cell line revealed that it contains mutations within exon 5, which translates to the C-terminal portion of IE86 (Murphy et al., 2000). Therefore, the report that IE86 does not repress p53 activity may be attributed to the mutations present in the IE86 protein expressed by the cells.

HCMV appears to mirror the small DNA tumor viruses by targeting key cell cycle regulatory proteins. One may predict that expression of IE86 should be sufficient to overcome the cell cycle regulatory functions of the pRb and p53 proteins in a manner similar to the small DNA tumor virus oncproteins. In fact, the expression of the IE86 protein in a human cell line, U-373, blocked cell cycle progression in G1-phase similar to the phenotype observed when permissive cells are infected with HCMV (Wiebusch and Hagemeier, 1999). However, other studies have shown that IE86 can induce entry into S-phase in human and rodent fibroblasts (Castillo et al., 2000; Murphy et al., 2000). It has also been observed that IE86 promotes the transition from G1- to S-phase through transactivation of cyclin E promoter and that human cells accumulate in what appears to be early S-phase (Wiebusch and Hagemeier, 2001). Moreover, a recent study using DNA microarrays to analyze the effect of IE86 protein on cellular gene expression revealed that IE86 induces numerous factors associated with cell cycle regulation and the bioenzymatic machinery necessary for DNA replication. For example, IE86 induces an increase in the mRNA levels of B-myb, cyclin E, cdk-2, E2F-1, ribonucleotide reductase 1 and 2, thymidine synthetase, MCM3, and MCM7 (Song and Stinski, 2001). This study demonstrates that the vast majority of genes induced by IE86, including the ones listed above, are E2F targets. These observations are consistent with the expected outcome(s) following the disruption of pRb-E2F complexes and the subsequent transactivation of E2F-responsive promoters on numerous S-phase genes.

### 2.2.3. Disruption of nuclear domain 10 sites/promyelocytic leukemia protein domains affects cell growth and differentiation

The structures referred to as nuclear domain 10 (ND10) are sites in the nucleus at which DNA replication, transcription, pre-mRNA splicing, and ribosome assembly initiate (reviewed in Maul et al., 2000). These structures, also referred to as PODs for promyelocytic leukemia protein (PML) oncogenic domains, appear as punctate bodies dispersed throughout the nucleus that are present through most of the cell cycle except during mitosis. In addition to the PML protein, the ND10/PODs associate with other nuclear proteins including Sp100 and PIC1/SUMO-1. Based on sequence comparison with other proteins, Sp100 appears to be a transcription factor, while PIC1/SUMO-1 mediates the ubiquitination of certain target proteins. Though the precise functions of the ND10/PODs are unclear, it appears that a correlation exists between the nuclear distribution of ND10/PODs and normal cell growth and differentiation. This relationship is exemplified in individuals with acute promyelocytic leukemia where the disruption of ND10 correlates with the loss of cell growth control, and treatment with retinoic acid and arsenic leads to ND10 structure reformation, thus restoring growth control (Ishov et al., 1999). Evidence suggests that disruption of ND10/PODs is necessary for efficient viral DNA transcription and replication. Among the viruses encoding proteins that associate with ND10/PODs are adenovirus, SV-40, herpes simplex virus-1 (HSV-1), and Epstein-Barr Virus (EBV). In the case of adenovirus, SV-40, and HSV-1, their viral DNA is deposited near the periphery of the ND10/PODs following entry into the infected host cell nucleus, and transcription of viral DNA initiates at these sites (reviewed in Maul, 1998). The adenovirus and HSV-1 proteins, E4-ORF3 and IE110 respectively, promote the disruption of ND10/PODs within the nucleus (Maul et al., 1993; Carvalho et al., 1995; Doucas et al., 1996). However, it should be noted that disruption of the ND10/PODs by E4-ORF3 and IE110 is not absolutely essential for viral replication since loss of these proteins only diminishes the productive infectivities of adenovirus and HSV-1 at low MOIs.

Likewise, HCMV encodes proteins that also appear to associate with ND10/PODs during infection. As a result of the interaction with these nuclear bodies, HCMV induces the redistribution of both the PML and Sp100 from the ND10/PODs to a more diffuse pattern within the infected cell nucleus (Ahn and Hayward, 1997). Both IE72 and IE86 co-localize to ND10/PODs in permissively infected cells (i.e. cells undergoing productive infection) at early times post-infection. Initially, immunofluorescent staining reveals that IE72 appears as a mix of nuclear diffuse and punctate patterns. At later times of infection, IE72 diffuses throughout the nucleus. There are two reports that state IE72 disrupts the ND10/PODs causing an increasingly diffuse pattern throughout the nucleus post-infection (Korioth et al., 1996; Ahn and Hayward, 1997). Although the signifi-
cance of the ND10/PODs disruption by IE72 is not yet clear, this event may have implications for the host cell cycle since ND10/PODs are sites where transcription occurs and their disruption occurs normally during cell cycle progression.

It has been demonstrated that PML can co-localize with IE72 in punctate bodies but a direct protein-protein interaction has not been observed (Korioth et al., 1996; Ahn and Hayward, 1997). IE72 has also been shown to interact with hDaxx, another major component of the ND10/PODs, and this interaction has a significant effect on the relocalization of PML in the ND10/PODs in IE72-expressing cells (pers. commun.; Woodhall et al., 2001). Given that PML overexpression can reduce growth rates in cells, and the fact that its distribution in ND10/PODs is cell cycle-dependent, the dispersal of PML from the ND10/PODs by IE72 may advocate a growth-promoting environment.

The IE86 protein has been shown to localize to nuclear punctate structures during HCMV infection (Ahn and Hayward, 1997; Ahn et al., 1999). Expression of IE86 in human fibroblasts did not disrupt the ND10/PODs or cause the redistribution of PML protein within the nucleus but it did induce the co-localization of IE86 with PML protein in the ND10/PODs (Ahn and Hayward, 1997). Moreover, IE86 has been shown to undergo sumoylation, which appears to hinder its ability to localize in the ND10/PODs (Hofmann et al., 2000a, 2001). Like IE72, the ramifications of the IE86 association with ND10/PODs for the cell cycle remain unclear. Perhaps the disruption of ND10/PODs by IE72 and redistribution of factors normally associated with these structures by both HCMV IE proteins function primarily towards creating optimal conditions for viral DNA replication.

that occur during early infection. However, it is apparent that IE72 is not absolutely required for HCMV infection and replication of its viral DNA. Clearly, other proteins compensate for the loss of IE72 activity in cells infected with a high MOI of the CR208 mutant virus.

### 3.2. Functional domains of IE72

Studies to determine the functions of the IE72 protein have revealed several crucial domains that allow it to bind to cellular factors as well as perform certain tasks necessary for viral replication (Table 1, Fig. 2). Due to the fact that exons 2 and 3 encode the first 85 aa of both IE72 and IE86 proteins, one would expect similar peptide binding patterns to this sequence for both IE factors. In contrast to this supposition, it has been determined that sequences encoded by exons 2 and 3 on IE72 are required for its specific binding to p107 (Johnson et al., 1999) whereas this domain is part of the pRb-binding region of IE86. IE72 mutants lacking exons 2 and 3 are also unable to alleviate the growth arrest mediated by the overexpression of p107 in cells implying that this region is responsible for the ability of IE72 to alleviate p107 activity. In addition to p107 binding, sequences within exons 2 and 3 of IE72 are also required for its transactivation function. This region, along with its leucine zipper domain and zinc finger motif, is necessary for the efficient activation of the DNA polymerase alpha promoter (Hayhurst et al., 1995). This effect may be indirect, since the DNA polymerase alpha promoter is regulated in part by E2F and sequences encoded by exons 2 and 3 may likely contribute to the induction of E2F activity by inhibiting p107 repressor functions. Moreover, the regulation of the DNA polymerase promoter by IE72 is complex and is further discussed below (see Section 3.3).

The unique region of IE72, namely the sequence encoded by exon 4, contributes most of the amino acid residues contained in this protein (Fig. 2). Protein kinase activity has been mapped to aa 173–197 (Pajovic et al., 1997) which bears homology to the ATP-binding sites found on many other kinases. Sequences contained within the C-terminal acidic domain of IE72 contribute towards the ability of IE72 to co-localize and disrupt ND10/POD complexes (Ahn and Hayward, 1997; Ahn et al., 1998). Furthermore, aa 105–139, which encode a leucine-rich domain, and aa 267–286, which encode a zinc finger domain, have been identified as regions that bind to CTF-1 (Hayhurst et al., 1995). The yeast homolog of CTF-1 has been shown to be important for normal cell cycle progression in G₂/M (Gerring et al., 1990). However, it remains unclear whether IE72 affects cell cycle control by binding to CTF-1.

### 3.3. Other potential functions

The IE72 protein appears to exhibit multiple functions. First and foremost, it plays an indirect role in promoting the replication of viral DNA. Although previous data suggested it is nonessential (Greaves and Mocarski, 1998),

J.P. Castillo, T.F. Kowalik / Gene 290 (2002) 19–34

HCMV IE72


| 1 | LR | Zn | LZ | AD | 491 |
| --- | --- | --- | --- | --- | --- |


REGION SHARED WITH IE86
p107-BINDING
TRANSACTION
CTF-1 BINDING
KINASE ACTIVITY
ND10/PODS-TARGETING
PML-DISRUPTION


Fig. 2. The HCMV IE72 protein and its regulatory domains. Diagram of HCMV IE72 and location of the functional and protein binding domains. Note that many of the functional domains have only been grossly mapped. Mapping of some functional domains are still unclear. Motif abbreviations: Zn, zinc finger; LZ, leucine zipper; LR, leucine-rich; AD, acidic domain.

IE72 likely contributes to viral replication by transactivating the expression of various genes (i.e. early genes) that facilitate the replication process. Support of its importance as a co-activator stems from studies demonstrating that IE72 can activate some viral, as well as cellular, promoters when assayed in reporter assays. Although IE72 does not bind DNA directly, transient expression of IE72 cDNA is sufficient to stimulate transcription from certain TATA-less promoters including the cellular DHFR and DNA polymerase alpha gene promoters (Hayhurst et al., 1995; Margolis et al., 1995). The ability of IE72 to activate the DHFR and DNA polymerase alpha promoters is dependent upon the presence of an E2F binding site in the DHFR promoter (Margolis et al., 1995) and a CCAAT-box and E2F binding sites in the DNA polymerase alpha promoter (Hayhurst et al., 1995).

IE72 can also co-activate promoters containing TATA-box elements. Specifically, IE72 is capable of co-activating the HIV-1 long-terminal repeat (LTR) and the required regions are the TATA-box and a 10 bp element upstream of the TATA-box (Walker et al., 1992). IE72 also enhances AP-1 activity as well as induces AP-1-driven transcription in reporter constructs (Kim et al., 1999). It is suggested in this study that induction of AP-1 by IE72 is kinase-dependent because it fails to induce AP-1 in the presence of kinase inhibitors. It is unclear whether the kinase activity associated with IE72 plays a direct role in its ability to activate AP-1. In addition, IE72 is capable of transactivating its own promoter through NF-κB elements present within the major IE promoter (Cherrington and Mocarski, 1989).

IE72 contributes to the induction of Rel/NF-κB transcription by transactivating the p65 (Rel-A) promoter via two mechanisms: (i) interaction with Sp-1, and (ii) cooperation with Sp-1 to enhance p65 promoter transactivation (Yurochko et al., 1995, 1997). NF-κB transcription is induced in G1-phase as part of normal cell cycle progression from G0-to S-phase. Additionally, NF-κB can protect cells from apoptosis by inducing cellular inhibitors of apoptosis (cIAPs). Given this association with the cell cycle, it is feasible that the induction of NF-κB following IE72 expression may act to protect cells from undergoing apoptosis as they progress from G0, the normal cell cycle state of cells targeted by HCMV in vivo, towards the S-phase-like environment induced during virus replication.

4. IE86

4.1. Importance for viral infection

The significance of the IE86 protein to HCMV replication
is exemplified by the inability to generate infectious HCMV mutants lacking IE86. Despite the lack of success, alternate methods have been developed to examine the role of IE86 in viral replication. One method has been to generate a bacterial artificial chromosome (BAC) clone containing the full-length HCMV genome but lacking the majority of the UL122 gene, which encodes IE86 (Marchini et al., 2001). Transfection of a BAC containing a complete HCMV genome resulted in virus production in permissive cells. However, transfection of the BAC containing a UL122-deficient HCMV genome fails to generate plaques and is unable to induce the expression of several viral early genes (Marchini et al., 2001). Similar outcomes were observed when a HCMV recombinant virus expressing an IE86 exon 5-estrogen receptor fusion protein was generated and used to infect cells. Replication of the recombinant virus, CR372, is hormone-dependent after low multiplicity inoculation indicating the dependence on IE86 for virus production. Furthermore, infection with CR372 in the absence of hormone treatment results in decreased accumulation of several viral early proteins and the absence of detectable infectious virus (pers. commun.; Greaves, 2001). Additionally, a study in which a temperature-sensitive form of IE86 was incorporated into the parental strain of HCMV also demonstrated the importance of the protein to HCMV replication as demonstrated by replication at the permissive temperature but not at the nonpermissive temperature (pers. commun.; Heider and Shenk, 2001).

The failure of IE86-deficient strains of HCMV to replicate properly is consistent with its role as a potent transcriptional activator of numerous HCMV early genes, as well as several host cell genes. Transactivation of promoters by IE86 can occur through a TATA-box-dependent mechanism as observed by its ability to activate the expression of cellular promoters that are dependent on the interaction of IE86 with components of the basal transcription complex including TATA-binding protein (TBP) (Caswell et al., 1993; Jupp et al., 1993a,b), TFIIIB (Caswell et al., 1993), and TFIID (Hagemeier et al., 1992) (Table 1). IE86 also rescues a defect in ts13 cells, which exhibit a transcriptional defect due to a temperature-sensitive mutation in TAFII250, a TATA-binding protein associated factor (TAF), suggesting that IE86 exhibits a TAF-like function in infected cells (Lukac and Alwine, 1999). Moreover, several groups have shown that IE86 can transactivate the promoters of genes from other viruses. Studies show that HCMV can complement an E1A-deficient adenovirus resulting in the expression of early adenovirus genes and for lytic growth in HeLa cells (Spector and Tevethia, 1994). IE86 has also been shown to transactivate promoters in the LTR sequences of various retroviruses/lentiviruses such as HIV-1 (Walker et al., 1992; Yeung et al., 1993). Furthermore, analysis of the transformation capacity of IE86 has shown that both IE86 and IE72 can cooperate with adenovirus E1A to transform primary rodent cells in culture (Shen et al., 1997). However, unlike the E1A protein, the ability of the HCMV IE proteins

to transform cells in vitro is not believed to be through a direct mechanism. Evidence supporting this comes from the fact that both proteins fail to transform cells in vitro in the absence of E1A and that expression of both IE proteins is transient during the transformation process (Shen et al., 1997). Instead, both IE72 and IE86 proteins are believed to promote the accumulation of mutations within cells thereby contributing to the transformation process by inducing genetic instability. These observations suggest that IE86 and IE72 are potentially mutagenic proteins, and that under the appropriate conditions, these proteins may contribute to cellular transformation.

In addition to its function as a strong transactivator, IE86 also functions as a repressor that downregulates expression from the major IE promoter. IE86 binds to the cis-repression sequence (crs) that lies within the major IE promoter (Pizzorno and Hayward, 1990). These findings imply a dual role for IE86 in which the protein induces the expression of several viral and cellular genes while auto-regulating its own expression.

The IE86 protein localizes to ND10/PODs and unlike IE72 does not promote the disruption of these nuclear structures or the dispersal of PML protein within the nucleus. Rather, IE86 co-localizes with PML in ND10/PODs following HCMV infection or transient transfection of an IE86 encoding cDNA (Ahn and Hayward, 1997). Since the precise role of PML in cell cycle regulation is not known, the significance of IE86 and PML co-localization in the ND10/PODs to the cell cycle remains unclear. A recent report demonstrates that IE86 binding to other ND10/POD constituents results in the covalent modification of IE86 (Hofmann et al., 2000a). Specifically, IE86 interacts with several SUMO family members, SUMO-1, SUMO-2, and SUMO-3, as well as the SUMO-conjugating enzyme, Ubc9, and consequently, IE86 becomes modified as a result of the sumoylation process (pers. commun.; Ahn et al., 2001; Hofmann et al., 2000a, 2001). Although these modifications are not required for IE86 localization to the ND10/POD domains, they are essential for IE86 transactivation capacity.

There are several lines of evidence supporting the notion that sumoylation of some cellular proteins, such as PML and p53, is essential for their proper function. Modification of PML by SUMO is important for its localization to the ND10/PODs, as well as for the recruitment of other ND10/PODs proteins, such as Daxx, Sp100, and CBP to these domains (Muller et al., 1998; Ishov et al., 1999; Zhong et al., 2000). Although IE86 does not disrupt PML localization to the ND10/PODs, the effects of IE86 on PML sumoylation are not known. Additionally, the sumoylation of p53 enhances its transactivation ability (Gostissa et al., 1999; Rodriguez et al., 1999; Muller et al., 2000). Since IE86 has been reported to suppress p53 transactivation function, perhaps IE86 mediates this effect by interfering with the sumoylation of p53. The interaction between IE86 and the SUMO proteins may alter the modification of PML or
p53 by competing for SUMO binding or by sequestering available SUMO in the ND10/PODs. As a result, the amount of available SUMO within these domains could be limited, thereby decreasing sumoylation of both of these cellular factors. Thus, the association of IE86 with the SUMO proteins in the ND10/PODs may alter the transactivation function of p53 as well as the localization of PML to the ND10/PODs.

### 4.2. Functional domains of IE86

The IE86 protein and its functional domains are shown in Fig. 3. IE86 can transactivate a wide variety of viral as well as cellular promoters and this ability is attributed to the two acidic transactivation domains present at the N-terminus (aa 1–85) and another mapped to a sizeable portion of the C-terminus (aa 195–579) (Malone et al., 1990; Pizzorno et al., 1991). In addition, aa 388–542 contain the dimerization domain. This region appears to play a significant role in IE86 transactivation function since IE86 homo-dimerization

was found to positively correlate with transactivation function (Chiou et al., 1993). Furthermore, overlap exists between the C-terminal transactivation domain of IE86 and the regions associated with DNA binding and autorepression. Deletion of aa 346–579 abolishes the ability of IE86 to bind to the major IE promoter (Schwartz et al., 1994). The ability of IE86 to downregulate its own expression through the so-called crs element maps to aa 290–579 and includes the region involved with DNA binding as well as the C-terminal acidic activation and zinc finger domains (Stenberg et al., 1984; Malone et al., 1990; Pizzorno and Hayward, 1990; Jupp et al., 1993a).

IE86 has been shown to bind pRb through several sets of adjacent amino acids. At least three regions on IE86 have been identified that can interact with pRb: aa 85–135, aa 136–290, and aa 291–364 (Sommer et al., 1994). One of these regions, aa 136–290, was recently shown to be dispensable for IE86 transactivation and HCMV viability (pers. commun.; Clark et al., 2001). Of the three pRb-interacting domains, only aa 291–364 is able to interact with pRb in

HCMV IE86

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
| 1 | LR | Zn | HLH | AD 579 |

**REGION SHARED WITH IE72**

**TRANSACTION**

**AUTOREPRESSION**

**DNA-BINDING**

**TBP-BINDING**

**TFIIB-BINDING**

**DIMERIZATION**

**p53-REPRESION**

**p53-BINDING**

**pRb-BINDING**

Fig. 3. The HCMV IE86 protein and its regulatory domains. Diagram of HCMV IE86 and location of the functional and protein binding domains. Note that many of the functional domains have only been grossly mapped. Mapping of some functional domains is still unclear (e.g. pRb binding). HLH, helix-loop-helix domain.
reverse binding experiments. This observation is consistent with another report that identified aa 290–390 as the region of IE86 where pRb binding occurs (Hagemeier et al., 1994). Subsequently, another study demonstrated that two independent regions of IE86 bind to and independently affect pRb function. The two regions, aa 241–369 and aa 1–85, were identified as the pRb-binding domains of IE86 using functional assays that measured the ability of wild-type and mutant forms of IE86 to counteract pRb activity in transfected SAOS-2 cells (Fortunato et al., 1997).

The regions of IE86 that are responsible for p53 binding are contained within the N-terminus. Specifically, in vitro binding assays revealed that IE86 interacts with p53 through a region encompassing aa 1–135 (Tsai et al., 1996). In addition, these same studies demonstrated that a portion of this region (aa 45–135) along with C-terminal residues of IE86 (aa 290–579) confer a repressive effect on p53 transactivation function (Tsai et al., 1996).

### 4.3. Other potential functions

IE86 not only functions as a transcriptional activator of various cellular and viral promoters, but it has been recently shown that IE86 positively influences the expression of some genes through the recruitment of several transcriptional co-activator proteins. In addition to binding CBP (Schwartz et al., 1996), the CBP-associated factor P/CAF also interacts with IE86 (Bryant et al., 2000). Both CBP and P/CAF are histone acetyltransferase proteins that facilitate gene expression through the modification of histone-DNA interactions involved in chromatin remodeling. Binding to IE86 does not appear to compromise the histone acetyltransferase activity of P/CAF, but it remains unclear whether this association hinders the interaction between CBF and P/CAF. The interaction between IE86 and P/CAF results in the synergistic activation of target promoters and may account for the ability of IE86 to promiscuously activate a number of cellular promoters. While the influence of this interaction on the cell cycle has not been determined, it is feasible that the binding of IE86 to both CBP and P/CAF contributes to the activation of S-phase genes.

Prevention of apoptosis of infected cells can be viewed as a necessary mechanism for maintaining an appropriate environment for virus replication. The small DNA tumor viruses encode oncoproteins that block p53-mediated apoptosis thereby contributing to an environment suitable for viral DNA replication. It has been proposed that IE86 blocks the induction of apoptosis, in part, by binding to and repressing the transactivation function of p53 (Speir et al., 1994; Tsai et al., 1996). In this regard, IE86 protein appears to act in a manner comparable to the small DNA tumor virus oncoproteins to promote virus replication as well as preventing the premature death of the infected cell.

Evidence suggests that IE86 expression can affect both the p53-dependent and independent apoptotic pathways. IE86 inhibits apoptosis of cells following treatment with doxorubicin, indicating that IE86 can suppress p53-mediated apoptosis following DNA damage (Tanaka et al., 1999). Furthermore, ts13 cells, which are temperature-sensitive TAFII250 mutant cells, do not undergo p53-dependent apoptosis when IE86 is expressed and the cells are grown at the nonpermissive temperature (Lukac and Alwine, 1999). Despite these findings, IE86 fails to protect cells from apoptosis following UV irradiation (Lukac and Alwine, 1999). IE86 also blocks p53-independent apoptotic pathways such as the TNF-mediated death receptor-signaling pathway (Zhu et al., 1995). IE86 can induce NF-κB transcriptional activity and this may have an anti-apoptotic effect on the infected cell since NF-κB can protect cells from apoptosis by inducing the cIAPs under certain conditions. These findings support the notion that IE86 may affect certain components of the apoptotic process.

## 5. Other IE proteins

### 5.1. IE55

The IE55 protein is a splice variant of the IE86 gene product that has a conserved N-terminus and a 155 aa deletion at aa 365–519 in the C-terminus. The region that is deleted has been shown to be required for many of the functions that have been attributed to IE86 including transcriptional activation and DNA binding (Malone et al., 1990; Pizzorno et al., 1991; Chiou et al., 1993; Schwartz et al., 1994). Consequently, unlike IE86, IE55 fails to transactivate HCMV early promoters and to repress the major IE promoter (Klucher et al., 1993). Additionally, IE55 lacks the putative zinc finger and helix-turn-helix motifs, which may account for the inability of IE55 to dimerize with IE86. The significance of this observation is not yet clear, but the inability of IE55 to dimerize and/or form complexes with IE86 may account for the inability of IE55 to transactivate the promoters of HCMV early genes or downregulate the major IE promoter. This hypothesis is consistent with the observation that a positive correlation exists between IE86 transactivation and its ability to form dimers (Chiou et al., 1993).

The absence of dimerization, DNA binding, and transactivation domains in IE55 will obviously result in a protein that is functionally different from IE86. However, it should be noted that the majority of the putative pRb-binding sites are intact in IE55. Although not yet tested, one might predict that IE55 mimics IE86 by binding to pRb and consequently exhibiting similar effects on the cell cycle as IE86. There is an additional line of evidence that suggests that IE55 may have the capacity to affect similar aspects of the cell cycle. It has been shown that transient expression of IE55 induces the levels of NF-κB by somehow transactivating the p105/p50 promoter and enhancing the transactivation of the p65 promoter via Sp-1 (Yurochko et al., 1995, 1997). In contrast to IE86, the expression of IE55 fails to enhance the DNA
binding activity of Sp-1, suggesting that IE55 cooperates with Sp-1 to transactivate the NF-kB promoter by a different mechanism. The induction of NF-kB by IE55 should increase protection from apoptosis and therefore contribute to a more favorable environment for virus replications. However, this has not yet been determined.

### 5.2. IRS1/TRS1

The IRS1/TRS1 gene products are additional HCMV IE gene products required for viral DNA replication. Gene sequences for both IRS1 and TRS1 are contained within the internal and terminal repeat sequences of the short segment of the HCMV genome. The IRS1 ORF encodes an estimated 91 kDa protein while the TRS1 ORF encodes a protein with an estimated mass of 84 kDa (Pari et al., 1993). Transcripts from both IRS1 and TRS1 are expressed very early during infection (2–4 h) and can be detected during all phases of the HCMV replication cycle (Romanowski and Shenk, 1997). The pIRS1 and pTRS1 proteins are highly homologous with their N-terminal 549 aa being identical except for a single amino acid difference. The two proteins contain hydrophobic domains and are present within the viral particle (Romanowski et al., 1997). Localization of the two proteins within the HCMV-infected cell varies during the course of the infection. During early times post-infection, the pIRS1 and pTRS1 proteins localize to both the nucleus and cytoplasm. Despite their classification as IE proteins, pIRS1 and pTRS1 protein levels peak late in infection when localization of the proteins is primarily cytoplasmic.

pIRS1 and pTRS1 are capable of transactivating the promoters of numerous viral as well as cellular promoters in conjunction with other HCMV gene products, including the major IE proteins, IE72 and IE86. pIRS1 and pTRS1 can also cooperate with pUL69 to enhance the activation of the major IE promoter (Romanowski et al., 1997). Despite their function as transcriptional co-activators, the precise role of pIRS1 and pTRS1 in regulating cell cycle control has yet to be determined.

### 5.3. UL36-38

The UL36-38 loci of HCMV gives rise to several distinct IE transcripts through the activation of different promoters and alternative splicing events (Tenney and Colberg-Poley, 1991). The gene products expressed from these particular loci are considered to be one of the 11 HCMV loci essential for viral DNA replication (Pari and Anders, 1993). Additionally, the three IE proteins encoded at this loci have been reported to exhibit anti-apoptotic activity. The product of the UL36 ORF, vICA, exerts an anti-apoptotic effect on cells through its ability to bind procaspase-8 and inhibit its activity (Skaletskaya et al., 2001). Similarly, the UL37 ORF gene products, pUL37 and pUL37 exon 1 (pUL37x1), exhibit similar anti-apoptotic functions that are dependent upon the hydrophobic leader sequences in the first 22 aa of

their respective N-termini (Goldmacher et al., 1999; Hahn et al., 2001).

The UL37 protein, also referred to as vMIA, is a type-1, heavily-glycosylated transmembrane protein that localizes primarily to the mitochondria following HCMV infection (pers. commun.; Goldmacher et al., 1999). Although pUL37 does not directly interact with any components of the cell cycle machinery, the protein negatively regulates components of the death receptor-mediated apoptotic pathway and can suppress cell death mediated by various stimuli (pers. commun.; Goldmacher et al., 1999). Specifically, pUL37 blocks Fas-mediated apoptosis by interfering with the pathways downstream of procaspase-8 activation but upstream of cytochrome c release in a manner similar to Bcl-2 (Goldmacher et al., 1999).

Recent evidence has also demonstrated the importance of the pUL37x1 protein to cell survival and virus growth. The inability to generate infectious virus from BAC-derived mutants lacking pUL37x1 hints at the importance of this protein to HCMV replication. In fact, infection with the pUL37x1 mutants results in extensive cell death (pers. commun.; Hahn et al., 2001). Infectious virus can be generated when the pUL37x1 BAC-derived mutants are grown in cell lines expressing death antagonists. Analogous to vMIA, pUL37x1 is also a mitochondrial transmembrane protein that does not appear to directly influence components of the cell cycle machinery. However, pUL37x1 can prolong the life of the infected cell by binding to the adenosine nucleotide transporter on the mitochondria. This blocks the release of cytochrome C thereby preventing the activation of the apoptotic pathway (Goldmacher et al., 1999; Colberg-Poley et al., 2000). Taken together, the combined activities of the three IE proteins expressed from the UL36-38 loci appear to alter the activation and progression of the apoptotic pathway.

### 6. Virion (tegument) proteins

This review primarily focuses on the HCMV IE proteins and their effect on the host cell cycle. However, other HCMV proteins affect the host cell cycle and its regulation. Although these proteins are not categorized as IE gene products per se, they still mediate an effect immediately following infection. At least two constituents of the virion-associated tegument, pUL69 and pp71, have been reported to mediate effects on the cell cycle.

The product of the UL69 ORF, pUL69, functions as a tegument-associated transactivator and has sequence similarity to the HSV ICP27 gene product (Winkler et al., 1994; Winkler and Stamminger, 1996). The ability of pUL69 to function as a viral transactivator may be attributed to its interaction with and disruption of hSPT6, a protein involved in the regulation of chromatin structure (Winkler et al., 2000). Additionally, transient expression of pUL69 in human fibroblasts results in the accumulation of cells in

the G₁-phase of the cell cycle indicating that pUL69 inhibits
cell cycle progression (Lu and Shenk, 1999). A subsequent
study utilizing a HCMV mutant lacking the UL69 coding
region also demonstrated that pUL69 induces the accumulation
of cells in G₁-phase (Hayashi et al., 2000). Although the
mechanism by which pUL69 mediates the accumulation at
G₁-phase is not known, these findings demonstrate that a
component of the HCMV tegument can mediate an immediate
effect on the cell cycle. Furthermore, UV-inactivated
HCMV can also mediate an inhibitory effect on the host
cell cycle indicating that the presence of the UL69 protein
in virions may contribute to the growth arrest phenotype
observed in cells following HCMV infection (Jault et al.,
1995; Bresnahan et al., 1996; Lu and Shenk, 1996; Dittmer
and Mocarski, 1997).

The pp71 phosphoprotein, expressed from the UL82
ORF, localizes to the nucleus immediately after virion
entry (Roby and Gibson, 1986; Ruger et al., 1987; Hensel
et al., 1996). Functionally, pp71 is a transcription factor that
is packaged within the viral tegument and is essential for the
adequate accumulation of IE72 and IE86 by transactivating
the major IE promoter (pers. commun.; Bentham and
Greaves, 2000). Several reports have demonstrated that
pp71 co-localizes with PML and Sp100 at the ND10/POD
domains within the nuclei of infected cells (pers. commun.;
Hofmann et al., 2000b). Because pp71 functions as a trans-
activator and localizes to ND10/PODs, it has been
compared to HSV VP16. However, one notable difference
between the two viral proteins is that, unlike VP16, the pp71
protein does not disrupt these domains. Moreover, pp71
binds hDaxx and this interaction enhances the transactiva-
tion of the major IE promoter as well as the co-localization
and accumulation of pp71 at the ND10/PODs (pers.
commun.; Hofmann et al., 2000b; Ishov et al., 2001).

The pp71 protein can have a dramatic impact on cell cycle
control. pp71 protein accelerates the infection cycle of
HCMV (Baldick et al., 1997) as well as the transition from
G₁-phase to S-phase of the cell cycle such that cells expres-
sing pp71 are able to commit to S-phase and commence
cellular DNA replication faster than cells that do not express
pp71 (pers. commun.; Kalejta et al., 2000). Additionally,
pp71 induces quiescent (G₀) cells to enter S-phase (pers.
commun.; Kalejta et al., 2000). Studies examining the rela-
tionship between pp71 and the cell cycle reveal that pp71
contains a sequence (LACSD) that is similar to the RB-bind-
ing motif (LxCxE) present in viral oncproteins encoded by
the small DNA tumor viruses (pers. commun.; Kalejta et al.,
2000; Kalejta and Shenk, 2001). pp71 binds to all three Rb
family members in vitro (pers. commun.; Kalejta and Shenk,
2001). The introduction of mutations within the LACSD
motif abrogates the ability of pp71 to induce DNA synthesis
in quiescent cells but does not affect its ability to accelerate
cells through G₁- into S-phase (pers. commun.; Kalejta and
Shenk, 2001). Taken together, the ability of pp71 to induce
DNA synthesis in G₀ cells may be attributed to its ability to
target each of the RB protein family members.

7. Possible model and relationship to HCMV
pathogenesis

From the standpoint of HCMV infecting a cell, it is
imperative to generate and maintain an environment conducive
for viral DNA replication. HCMV accomplishes both
of these feats through the expression of numerous IE gene
products along with two virion-associated factors present
within its tegument (Fig. 4). Stimulation of the cellular
replication machinery by HCMV is due in part to the IE72
and IE86 proteins as well as the tegument protein, pp71,
inactivating the functions of the RB tumor suppressor
family. This results in the activation of numerous host
factors that appear to facilitate the replication of the viral
genome. In contrast to these activities, pUL69 induces cells
to growth arrest in late G₁ or alternately, specifically inhibits
cellular DNA replication. Through this strategy, replication
of viral DNA rather than cellular DNA is assured within the
infected cell.

In response to the initial insult stemming from virus binding
and entry, along with the inactivation of several cell
cycle regulatory proteins, the host cell responds by inducing
multiple defense pathways that are intended to result in
cellular growth arrest and/or apoptosis. HCMV is able to
overcome these blocks through the expression of proteins
that can inactivate or antagonize the cellular factors
involved in these host defense responses. One of the
major IE proteins, IE86, represses the growth arrest and
apoptosis response mediated by p53. Several of the gene

![Diagram](attachment:diagram.png)

Fig. 4. The effects of HCMV IE gene products and tegument proteins on the
cell cycle and growth arrest/apoptotic pathways. Arrows depicted in green
represent activities that promote proliferative signals or protect the cells
from apoptosis. Arrows depicted in red represent activities that can negatively
affect proliferative signals within the cell.

products expressed from the HCMV UL36-38 loci function to prevent apoptosis signaling from the mitochondria and activated caspases. In this manner, HCMV can manipulate the host cell to promote a proliferative environment that maximizes virus replication while preventing the infected cell from shutting down or killing itself.

The cell cycle promoting effects of the HCMV IE proteins hint at a potential role for HCMV in cancer. Indeed, occasional studies suggest that HCMV infection may contribute to some cancers such as cervical carcinoma (Shen et al., 1993), but at this juncture, no conclusive evidence links HCMV to cancer. However, HCMV is associated with at least one disorder in which cell growth control has been abrogated—restenosis. Restenosis is a commonly observed complication following balloon angioplasty of coronary arteries. It is characterized by the overproliferation and accumulation of arterial smooth muscle cells along the vessel wall intima. HCMV may contribute to this phenotype by inactivating RB family members, thereby over-stimulating the natural response of smooth muscle cells to undergo a limited proliferation necessary to repair vessel damage that can result from the angioplasty procedure. By inactivating the growth suppressing functions of p53 and RB proteins, the virus could also prevent infected smooth muscle cells from exiting the cell cycle. Finally, HCMV may contribute to the continued presence of excess cells at the site of injury by blocking cell death signals via the anti-apoptotic activities encoded by HCMV.

In a more general sense, the ability of the HCMV IE proteins to significantly alter the intracellular environment likely contributes to its pathogenicity in vivo. For example, HCMV infection results in a robust and constitutive induction of NF-κB activity. Part of the repertoire of genes induced by NF-κB includes those that regulate the expression of cytokines and interferon. Secretion of these factors would then induce an inflammatory response leading to the recruitment of cells involved in the host anti-viral defense. Among these are the dendritic cells and macrophages, cells normally targeted by HCMV for replication, latency, and/or dissemination. Thus, an added consequence of HCMV IE gene expression is the indirect activation and migration of cells for continued HCMV infection and spread throughout the host.

## Acknowledgements

We wish to thank the many researchers who made available the unpublished data presented in this review. We would also like to acknowledge Mary Pickering, Madelyn Schmidt, Fiona Frame, Anita Ballisteros, Michelle Debatis, and Rachel Gerstein for their critical review of the manuscript. This work was supported by a NIH National Research Service Award (HL 10334-01) to J.P.C. and grants from the American Heart Association (9830085N) and the NIH (CA86038) to T.F.K.

### References

Ahn, J.H., Hayward, G.S., 1997. The major immediate-early proteins IE1 and IE2 of human cytomegalovirus colocalize with and disrupt PML-associated nuclear bodies at very early times in infected permissive cells. J. Virol. 71, 4599–4613.

Ahn, J.H., Brignole 3rd, E.J., Hayward, G.S., 1998. Disruption of PML subnuclear domains by the acidic IE1 protein of human cytomegalovirus is mediated through interaction with PML and may modulate a RING finger-dependent cryptic transactivator function of PML. Mol. Cell. Biol. 18, 4899–4913.

Ahn, J.H., Jang, W.J., Hayward, G.S., 1999. The human cytomegalovirus IE2 and UL112-113 proteins accumulate in viral DNA replication compartments that initiate from the periphery of promyelocytic leukemia protein-associated nuclear bodies (PODs or ND10). J. Virol. 73, 10458–10471.

Ahn, J.H., Xu, Y., Jang, W.J., Matunis, M.J., Hayward, G.S., 2001. Evaluation of interactions of human cytomegalovirus immediate-early IE2 regulatory protein with small ubiquitin-like modifiers and their conjugation enzyme Ubc9. J. Virol. 75, 3859–3872.

Albrecht, T., Rapp, F., 1973. Malignant transformation of hamster embryo fibroblasts following exposure to ultraviolet-irradiated human cytomegalovirus. Virology 55, 53–61.

Azizkhan, J., Pajovic, S., Wong, S.E., Kuropatwinski, K., Ippolito, A., Berkowitz, J., Greaves, R., Black, A.R., 2000. HCMV IE1 (IE72) targets E2Fs to stimulate cell cycle. Abstract presented at the 25th International Herpesvirus Workshop, Portland, OR.

Baldick Jr., C.J., Marchini, A., Patterson, C.E., Shenk, T., 1997. Human cytomegalovirus tegument protein pp71 (ppUL82) enhances the infectivity of viral DNA and accelerates the infectious cycle. J. Virol. 71, 4400–4408.

Bargoni, J., Reynisdottir, I., Friedman, P.N., Prives, C., 1992. Site-specific binding of wild-type p53 to cellular DNA is inhibited by SV40 T antigen and mutant p53. Genes Dev. 6, 1886–1898.

Bentham, M.J., Greaves, R.F., 2000. Infection of fibroblast lines expressing functional immediate early proteins by HCMV pp71 mutant CR223. Abstract presented at the 25th International Herpesvirus Workshop, Portland, OR.

Boldogh, I., Gonczol, E., Vaczi, L., 1978. Transformation of hamster embryonic fibroblast cells by UV-irradiated human cytomegalovirus. Acta Microbiol. Acad. Sci. Hung. 25, 269–275.

Bonin, L.R., McDougall, J.K., 1997. Human cytomegalovirus IE2 86-kilodalton protein binds p53 but does not abrogate G1 checkpoint function. J. Virol. 71, 5861–5870.

Bresnahan, W.A., Boldogh, I., Thompson, E.A., Albrecht, T., 1996. Human cytomegalovirus inhibits cellular DNA synthesis and arrests productively infected cells in late G1. Virology 224, 150–160.

Bryant, L.A., Mixon, P., Davidson, M., Bannister, A.J., Kouzarides, T., Sinclair, J.H., 2000. The human cytomegalovirus 86-kilodalton major immediate-early protein interacts physically and functionally with histone acetyltransferase P/CAF. J. Virol. 74, 7230–7237.

Carvalho, T., Seeler, J.S., Ohman, K., Jordan, P., Pettersson, U., Akusjarvi, G., Carmo-Fonseca, M., Dejean, A., 1995. Targeting of adenovirus E1A and E4-ORF3 proteins to nuclear matrix-associated PML bodies. J. Cell Biol. 131, 45–56.

Castillo, J.P., Kowalik, T.F., 2001. HCMV IE1-72 activates p53 through multiple pathways to induce a host growth arrest response. Abstract presented at the 26th International Herpesvirus Workshop, Regensburg, Germany.

Castillo, J.P., Yurochko, A.D., Kowalik, T.F., 2000. Role of human cytomegalovirus immediate-early proteins in cell growth control. J. Virol. 74, 8028–8037.

Caswell, R., Hagemeier, C., Chiou, C.J., Hayward, G., Kouzarides, T., Sinclair, J., 1993. The human cytomegalovirus 86 K immediate early (IE) 2 protein requires the basic region of the TATA-box binding protein (TBP) for binding, and interacts with TBP and transcription

factor TFIIB via regions of IE2 required for transcriptional regulation. J. Gen. Virol. 74, 2691–2698.

Cherrington, J.M., Mocarski, E.S., 1989. Human cytomegalovirus iel trans-activates the alpha promoter-enhancer via an 18-base-pair repeat element. J. Virol. 63, 1435–1440.

Chiou, C.J., Zong, J., Waheed, I., Hayward, G.S., 1993. Identification and mapping of dimerization and DNA-binding domains in the C terminus of the IE2 regulatory protein of human cytomegalovirus. J. Virol. 67, 6201–6214.

Choi, K.S., Kim, S.J., Kim, S., 1995. The retinoblastoma gene product negatively regulates transcriptional activation mediated by the human cytomegalovirus IE2 protein. Virology 208, 450–456.

Clark, C.L., Sanchez, V., Dwarakanath, R., Spector, D.H., 2001. Characterization of a viable HCMV recombinant virus with an internal deletion of the IE2 86 gene corresponding to amino acids 136–290. Abstract presented at the 8th International Cytomegalovirus Conference, Monterey, CA.

Colberg-Poley, A.M., Patel, M.B., Erezo, D.P., Slater, J.E., 2000. Human cytomegalovirus UL37 immediate-early regulatory proteins traffic through the secretory apparatus and to mitochondria. J. Gen. Virol. 81 (Pt 7), 1779–1789.

Dittmer, D., Mocarski, E.S., 1997. Human cytomegalovirus infection inhibits G1/S transition. J. Virol. 71, 1629–1634.

Doniger, J., Muralidhar, S., Rosenthal, L.J., 1999. Human cytomegalovirus and human herpesvirus 6 genes that transform and transactivate. Clin. Microbiol. Rev. 12, 367–382.

Doucas, V., Ishov, A.M., Romo, A., Juguilon, H., Weitzman, M.D., Evans, R.M., Maul, G.G., 1996. Adenovirus replication is coupled with the dynamic properties of the PML nuclear structure. Genes Dev. 10, 196–207.

Fortunato, E.A., Sommer, M.H., Yoder, K., Spector, D.H., 1997. Identification of domains within the human cytomegalovirus major immediate-early 86-kilodalton protein and the retinoblastoma protein required for physical and functional interaction with each other. J. Virol. 71, 8176–8185.

Geder, K.M., Lausch, R., O’Neill, F., Rapp, F., 1976. Oncogenic transformation of human embryo lung cells by human cytomegalovirus. Science 192, 1134–1137.

Gerring, S.L., Spencer, F., Hieter, P., 1990. The CHL 1 (CTF 1) gene product of Saccharomyces cerevisiae is important for chromosome transmission and normal cell cycle progression in G2/M. EMBO J. 9, 4347–4358.

Goldmacher, V.S., Bartle, L.M., Skaletskaya, A., Dionne, C.A., Kedersha, N.L., Vater, C.A., Han, J.W., Lutz, R.J., Watanabe, S., Cairh McFarland, E.D., Kieff, E.D., Mocarski, E.S., Chittenden, T., 1999. A cytomegalovirus-encoded mitochondria-localized inhibitor of apoptosis structurally unrelated to Bcl-2. Proc. Natl. Acad. Sci. USA 96, 12536–12541.

Gostissa, M., Hengstermann, A., Fogal, V., Sandy, P., Schwarz, S.E., Scheffner, M., Del Sal, G., 1999. Activation of p53 by conjugation to the ubiquitin-like protein SUMO-1. EMBO J. 18, 6462–6471.

Greaves, R.F., 2001. Hormone dependent replication of a human cytomegalovirus recombinant encoding an IE2-estrogen receptor fusion protein. Abstract presented at the 8th International Cytomegalovirus Conference, Monterey, CA.

Greaves, R.F., Mocarski, E.S., 1998. Defective growth correlates with reduced accumulation of a viral DNA replication protein after low-multiplicity infection by a human cytomegalovirus iel mutant. J. Virol. 72, 366–379.

Hagemeier, C., Walker, S., Caswell, R., Kourzarides, T., Sinclair, J., 1992. The human cytomegalovirus 80-kilodalton but not the 72-kilodalton immediate early gene transactivates heterologous promoters in a TATA box independent mechanism and interacts directly with TFIID. J. Virol. 66, 4452–4456.

Hagemeier, C., Caswell, R., Hayhurst, G., Sinclair, J., Kourzarides, T., 1994. Functional interaction between the HCMV IE2 transactivator and the retinoblastoma protein. EMBO J. 13, 2897–2903.

Hahn, G., Eichhorst, S.T., Korn, B., Krammer, P.H., Greaves, R., 2001. An anti-apoptotic protein of human cytomegalovirus enables virus replication. Abstract presented at the 26th International Herpesvirus Workshop, Regensburg, Germany.

Hayashi, M.L., Blankenship, C., Shenk, T., 2000. Human cytomegalovirus UL69 protein is required for efficient accumulation of infected cells in the G1 phase of the cell cycle. Proc. Natl. Acad. Sci. USA 97, 2692–2696.

Hayhurst, G.P., Bryant, L.A., Caswell, R.C., Walker, S.M., Sinclair, J.H., 1995. CCAAT box-dependent activation of the TATA-less human DNA polymerase alpha promoter by the human cytomegalovirus 72-kilodalton major immediate-early protein. J. Virol. 69, 182–188.

Heider, J.A., Shenk, T., 2001. Identification of an HCMV IE2 temperature-sensitive allele. Abstract presented at the 26th International Herpesvirus Workshop, Regensburg, Germany.

Hensel, G.M., Meyer, H.H., Buchmann, I., Pommerehne, D., Schmolke, S., Plachter, B., Radsak, K., Kern, H.F., 1996. Intracellular localization and expression of the human cytomegalovirus matrix phosphoprotein pp71 (ppUL82): evidence for its translocation into the nucleus. J. Gen. Virol. 77, 3087–3097.

Ho, M., 1982. Cytomegalovirus: Biology & Infection, Plenum Medical Book, New York.

Hofmann, H., Floss, S., Stamminger, T., 2000a. Covalent modification of the transactivator protein IE2-p86 of human cytomegalovirus by conjugation to the ubiquitin-homologous proteins SUMO-1 and hSMT3b. J. Virol. 74, 2510–2524.

Hofmann, H., Sindre, H., Stamminger, T., 2000. Functional interaction between the pp71 protein of human cytomegalovirus and the PML-interacting protein hDaxx. Abstract presented at the 25th International Herpesvirus Workshop, Portland, OR.

Hofmann, H., Hahn, G., Stamminger, T., 2001. SUMO-1 modification of the IE2-86 transactivator is required for efficient replication of human cytomegalovirus. Abstract presented at the 26th International Herpesvirus Workshop, Regensburg, Germany.

Ishov, A.M., Sotnikov, A.G., Negorev, D., Vladimirova, O.V., Neff, N., Kamitani, T., Yeh, E.T., Strauss 3rd, J.F., Maul, G.G., 1999. PML is critical for ND10 formation and recruits the PML-interacting protein daxx to this nuclear structure when modified by SUMO-1. J. Cell Biol. 147, 221–234.

Ishov, A.M., Vladimirova, O.V., Maul, G.G., 2001. Inter- and intracellular traffic of HCMV tegument protein pp71. Abstract presented at the 26th International Herpesvirus Workshop, Regensburg, Germany.

Jault, F.M., Jault, J.M., Ruchti, F., Fortunato, E.A., Clark, C., Corbeil, J., Richman, D.D., Spector, D.H., 1995. Cytomegalovirus infection induces high levels of cyclins, phosphorylated Rb, and p53, leading to cell cycle arrest. J. Virol. 69, 6697–6704.

Johnson, R.A., Yurochko, A.D., Poma, E.E., Zhu, L., Huang, E.S., 1999. Domain mapping of the human cytomegalovirus IE1-72 and cellular p107 protein-protein interaction and the possible functional consequences. J. Gen. Virol. 80, 1293–1303.

Jupp, R., Hoffman, S., Depto, A., Stenberg, R.M., Ghazal, P., Nelson, J.A., 1993a. Direct interaction of the human cytomegalovirus IE86 protein with the cis repression signal does not preclude TBP from binding to the TATA box. J. Virol. 67, 5595–5604.

Jupp, R., Hoffman, S., Stenberg, R.M., Nelson, J.A., Ghazal, P., 1993b. Human cytomegalovirus IE86 protein interacts with promoter-bound TATA- binding protein via a specific region distinct from the autorepression domain. J. Virol. 67, 7539–7546.

Kalejta, R.F., Shenk, T., 2001. The HCMV protein pp71 induces DNA synthesis in quiescent cells by targeting members of the RB family of tumor suppressor for degradation. Abstract presented at the 26th International Herpesvirus Workshop, Regensburg, Germany.

Kalejta, R.F., Bechtel, J.T., Shenk, T., 2000. Mutations that abolish the ability of HCMV pp71 protein to induce DNA synthesis in quiescent cells do not affect its ability to accelerate G1 phase cell cycle progression. Abstract presented at the 25th International Herpesvirus Workshop, Portland, OR.

Kim, J.M., Hong, Y., Jeang, K.T., Kim, S., 2000. Transactivation activity of the human cytomegalovirus IE2 protein occurs at steps subsequent to TATA box-binding protein recruitment. J. Gen. Virol. 81 (Pt 1), 37–46.

Kim, S., Yu, S.S., Lee, I.S., Ohno, S., Yim, J., Kang, H.S., 1999. Human cytomegalovirus IE1 protein activates AP-1 through a cellular protein kinase(s). J. Gen. Virol. 80, 961–969.

Klucher, K.M., Sommer, M., Kadonaga, J.T., Spector, D.H., 1993. In vivo and in vitro analysis of transcriptional activation mediated by the human cytomegalovirus major immediate-early proteins. Mol. Cell. Biol. 13, 1238–1250.

Korioth, F., Maul, G.G., Plachter, B., Stamminger, T., Frey, J., 1996. The nuclear domain 10 (ND10) is disrupted by the human cytomegalovirus gene product IE1. Exp. Cell Res. 229, 155–158.

Lang, D., Gebert, S., Arlt, H., Stamminger, T., 1995. Functional interaction between the human cytomegalovirus 86-kilodalton IE2 protein and the cellular transcription factor CREB. J. Virol. 69, 6030–6037.

Lowe, S.W., Ruley, H.E., 1993. Stabilization of the p53 tumor suppressor is induced by adenovirus 5 E1A and accompanies apoptosis. Genes Dev. 7, 535–545.

Lu, M., Shenk, T., 1996. Human cytomegalovirus infection inhibits cell cycle progression at multiple points, including the transition from G1 to S. J. Virol. 70, 8850–8857.

Lu, M., Shenk, T., 1999. Human cytomegalovirus UL69 protein induces cells to accumulate in G1 phase of the cell cycle. J. Virol. 73, 676–683.

Lukac, D.M., Alwine, J.C., 1999. Effects of human cytomegalovirus major immediate-early proteins in controlling the cell cycle and inhibiting apoptosis: studies with ts13 cells. J. Virol. 73, 2825–2831.

Lukac, D.M., Manuppello, J.R., Alwine, J.C., 1994. Transcriptional activation by the human cytomegalovirus immediate-early proteins: requirements for simple promoter structures and interactions with multiple components of the transcription complex. J. Virol. 68, 5184–5193.

Lukac, D.M., Harel, N.Y., Tanese, N., Alwine, J.C., 1997. TAF-like functions of human cytomegalovirus immediate-early proteins. J. Virol. 71, 7227–7239.

Malone, C.L., Vesole, D.H., Stinski, M.F., 1990. Transactivation of a human cytomegalovirus early promoter by gene products from the immediate-early gene IE2 and augmentation by IE1: mutational analysis of the viral proteins. J. Virol. 64, 1498–1506.

Marchini, A., Liu, H., Zhu, H., 2001. Human cytomegalovirus with IE-2 (UL122) deleted fails to express early lytic genes. J. Virol. 75, 1870–1878.

Margolis, M.J., Pajovic, S., Wong, E.L., Wade, M., Jupp, R., Nelson, J.A., Azizkhan, J.C., 1995. Interaction of the 72-kilodalton human cytomegalovirus IE1 gene product with E2F1 coincides with E2F-dependent activation of dihydrofolate reductase transcription. J. Virol. 69, 7759–7767.

Maul, G.G., 1998. Nuclear domain 10, the site of DNA virus transcription and replication. Bioessays 20, 660–667.

Maul, G.G., Guldner, H.H., Spivack, J.G., 1993. Modification of discrete nuclear domains induced by herpes simplex virus type 1 immediate early gene 1 product (ICP0). J. Gen. Virol. 74, 2679–2690.

Maul, G.G., Negorev, D., Bell, P., Ishov, A.M., 2000. Review: properties and assembly mechanisms of ND10, PML bodies, or PODs. J. Struct. Biol. 129, 278–287.

McCarthy, S.A., Symonds, H.S., Van Dyke, T., 1994. Regulation of apoptosis in transgenic mice by simian virus 40 T antigen-mediated inactivation of p53. Proc. Natl. Acad. Sci. USA 91, 3979–3983.

Mocarski, E.S., Courcelle, C.T., 2001. Cytomegaloviruses and their replication. In: Knipe, D., Howley, P., Griffin, D., Lamb, R., Martin, M., Straus, S. (Eds.). Fields Virology, Lippincott Williams & Wilkins, Philadelphia, PA, pp. 2629–2673.

Muller, S., Matunis, M.J., Dejean, A., 1998. Conjugation with the ubiquitin-related modifier SUMO-1 regulates the partitioning of PML within the nucleus. EMBO J. 17, 61–70.

Muller, S., Berger, M., Lehembre, F., Seeler, J.S., Haupt, Y., Dejean, A., 2000. c-Jun and p53 activity is modulated by SUMO-1 modification. J. Biol. Chem. 275, 13321–13329.

Murphy, E.A., Stinski, M.F., 1999. Human cytomegalovirus IE86 protein induction of a cell cycle halt and interaction with members of the MCM protein family. Abstract presented at the 24th International Herpesvirus Workshop, Boston, MA.

Murphy, E.A., Streblow, D.N., Nelson, J.A., Stinski, M.F., 2000. The human cytomegalovirus IE86 protein can block cell cycle progression after inducing transition into the S phase of permissive cells. J. Virol. 74, 7108–7118.

Nevins, J.R., 1998. Toward an understanding of the functional complexity of the E2F and retinoblastoma families. Cell Growth Differ. 9, 585–593.

Nevins, J.R., DeGregori, J., Jakoi, L., Leone, G., 1997. Functional analysis of E2F transcription factor. Methods Enzymol. 283, 205–219.

Pajovic, S., Wong, E.L., Black, A.R., Azizkhan, J.C., 1997. Identification of a viral kinase that phosphorylates specific E2Fs and pocket proteins. Mol. Cell. Biol. 17, 6459–6464.

Pari, G.S., Anders, D.G., 1993. Eleven loci encoding trans-acting factors are required for transient complementation of human cytomegalovirus oriLyt-dependent DNA replication. J. Virol. 67, 6979–6988.

Pari, G.S., Kacica, M.A., Anders, D.G., 1993. Open reading frames UL44, IRS1/TRS1, and UL36-38 are required for transient complementation of human cytomegalovirus oriLyt-dependent DNA synthesis. J. Virol. 67, 2575–2582.

Pass, R.F., 2001. Cytomegalovirus. In: Knipe, D., Howley, P., Griffin, D., Lamb, R., Martin, M., Straus, S. (Eds.). Fields Virology, Lippincott Williams & Wilkins, Philadelphia, PA, pp. 2675–2705.

Pizzorno, M.C., Hayward, G.S., 1990. The IE2 gene products of human cytomegalovirus specifically down-regulate expression from the major immediate-early promoter through a target sequence located near the cap site. J. Virol. 64, 6154–6165.

Pizzorno, M.C., Mullen, M.A., Chang, Y.N., Hayward, G.S., 1991. The functionally active IE2 immediate-early regulatory protein of human cytomegalovirus is an 80-kilodalton polypeptide that contains two distinct activator domains and a duplicated nuclear localization signal. J. Virol. 65, 3839–3852.

Poma, E.E., Kowalik, T.F., Zhu, L., Sinclair, J.H., Huang, E.S., 1996. The human cytomegalovirus IE1-72 protein interacts with the cellular p107 protein and relieves p107-mediated transcriptional repression of an E2F-responsive promoter. J. Virol. 70, 7867–7877.

Roby, C., Gibson, W., 1986. Characterization of phosphoproteins and protein kinase activity of virions, noninfectious enveloped particles, and dense bodies of human cytomegalovirus. J. Virol. 59, 714–727.

Rodriguez, M.S., Desterro, J.M., Lain, S., Midgley, C.A., Lane, D.P., Hay, R.T., 1999. SUMO-1 modification activates the transcriptional response of p53. EMBO J. 18, 6455–6461.

Romanowski, M.J., Shenk, T., 1997. Characterization of the human cytomegalovirus irsl and trsl genes: a second immediate-early transcription unit within irsl whose product antagonizes transcriptional activation. J. Virol. 71, 1485–1496.

Romanowski, M.J., Garrido-Guerrero, E., Shenk, T., 1997. pIRS1 and pTRS1 are present in human cytomegalovirus virions. J. Virol. 71, 5703–5705.

Ruger, B., Klages, S., Walla, B., Albrecht, J., Fleckenstein, B., Tomlinson, P., Barrell, B., 1987. Primary structure and transcription of the genes coding for the two virion phosphoproteins pp65 and pp71 of human cytomegalovirus. J. Virol. 61, 446–453.

Sarnow, P., Ho, Y.S., Williams, J., Levine, A.J., 1982. Adenovirus Elb-58kd tumor antigen and SV40 large tumor antigen are physically associated with the same 54 kd cellular protein in transformed cells. Cell 28, 387–394.

Scheffner, M., Werness, B.A., Huibregtse, J.M., Levine, A.J., Howley, P.M., 1990. The E6 oncoprotein encoded by human papillomavirus types 16 and 18 promotes the degradation of p53. Cell 63, 1129–1136.

Schwartz, R., Sommer, M.H., Scully, A., Spector, D.H., 1994. Site-specific binding of the human cytomegalovirus IE2 86-kilodalton protein to an early gene promoter. J. Virol. 68, 5613–5622.

Schwartz, R., Helmich, B., Spector, D.H., 1996. CREB and CREB-binding proteins play an important role in the IE2 86-kilodalton protein-

mediated transactivation of the human cytomegalovirus 2.2-kilobase RNA promoter. J. Virol. 70, 6955–6966.

Scully, A.L., Sommer, M.H., Schwartz, R., Spector, D.H., 1995. The human cytomegalovirus IE2 86-kilodalton protein interacts with an early gene promoter via site-specific DNA binding and protein-protein associations. J. Virol. 69, 6533–6540.

Shen, C.Y., Ho, M.S., Chang, S.F., Yen, M.S., Ng, H.T., Huang, E.S., Wu, C.W., 1993. High rate of concurrent genital infections with human cytomegalovirus and human papillomaviruses in cervical cancer patients. J. Infect. Dis. 168, 449–452.

Shen, Y., Zhu, H., Shenk, T., 1997. Human cytomegalovirus IE1 and IE2 proteins are mutagenic and mediate “hit-and-run” oncogenic transformation in cooperation with the adenovirus E1A proteins. Proc. Natl. Acad. Sci. USA 94, 3341–3345.

Skaletskaya, A., Bartle, L.M., Chittenden, T., McCormick, A.L., Mocarski, E.S., Goldmacher, V.S., 2001. A cytomegalovirus-encoded inhibitor of apoptosis that suppresses caspase-8 activation. Proc. Natl. Acad. Sci. USA 98, 7829–7834.

Sommer, M.H., Scully, A.L., Spector, D.H., 1994. Transactivation by the human cytomegalovirus IE2 86-kilodalton protein requires a domain that binds to both the TATA box-binding protein and the retinoblastoma protein. J. Virol. 68, 6223–6231.

Song, Y., Stinski, M.F., 2001. Effect of the human cytomegalovirus IE86 protein on expression of E2F responsive genes: a DNA microarray analysis, submitted for publication.

Spector, D.J., Tevethia, M.J., 1994. Protein-protein interactions between human cytomegalovirus IE2-580aa and pUL84 in lytically infected cells. J. Virol. 68, 7549–7553.

Speir, E., Modali, R., Huang, E.S., Leon, M.B., Shawl, F., Finkel, T., Epstein, S.E., 1994. Potential role of human cytomegalovirus and p53 interaction in coronary restenosis. Science 265, 391–394.

Speir, E., Huang, E.S., Modali, R., Leon, M.B., Shawl, F., Finkel, T., Epstein, S.E., 1995. Interaction of human cytomegalovirus with p53: possible role in coronary restenosis. Scand. J. Infect. Dis. Suppl. 99, 78–81.

Spengler, M.L., Black, A., Zhu, X., Azizkhan-Clifford, J., 2000. Characterization of SUMO-1 modified HCMV IE72. Abstract presented at the 25th International Herpesvirus Workshop, Portland, OR.

Stenberg, R.M., Thomsen, D.R., Stinski, M.F., 1984. Structural analysis of the major immediate early gene of human cytomegalovirus. J. Virol. 49, 190–199.

Streblow, D.N., Soderberg-Naucler, C., Vieira, J., Smith, P., Wakabayashi, E., Ruchti, F., Mattison, K., Altschuler, Y., Nelson, J.A., 1999. The human cytomegalovirus chemokine receptor US28 mediates vascular smooth muscle cell migration. Cell 99, 511–520.

Tanaka, K., Zou, J.P., Takeda, K., Ferrans, V.J., Sandford, G.R., Johnson, T.M., Finkel, T., Epstein, S.E., 1999. Effects of human cytomegalovirus immediate-early proteins on p53-mediated apoptosis in coronary artery smooth muscle cells. Circulation 99, 1656–1659.

Tenney, D.J., Colberg-Poley, A.M., 1991. Expression of the human cytomegalovirus UL36-38 immediate early region during permissive infection. Virology 182, 199–210.

Trimarchi, J.M., Lees, J.A., 2002. Sibling rivalry in the E2F family. Nat. Rev. Mol. Cell Biol. 3, 11–20.

Tsai, H.L., Kou, G.H., Chen, S.C., Wu, C.W., Lin, Y.S., 1996. Human

cytomegalovirus immediate-early protein IE2 tethers a transcriptional repression domain to p53. J. Biol. Chem. 271, 3534–3540.

Walker, S., Hagemeier, C., Sissons, J.G., Sinclair, J.H., 1992. A 10-base-pair element of the human immunodeficiency virus type 1 long terminal repeat (LTR) is an absolute requirement for transactivation by the human cytomegalovirus 72-kilodalton IE1 protein but can be compensated for by other LTR regions in transactivation by the 80-kilodalton IE2 protein. J. Virol. 66, 1543–1550.

Wiebusch, L., Hagemeier, C., 1999. Human cytomegalovirus 86-kilodalton IE2 protein blocks cell cycle progression in G(1). J. Virol. 73, 9274–9283.

Wiebusch, L., Hagemeier, C., 2001. The human cytomegalovirus immediate early 2 protein dissociates cellular DNA synthesis from cyclin-dependent kinase activation. EMBO J. 20, 1086–1098.

Winkler, M., Stamminger, T., 1996. A specific subform of the human cytomegalovirus transactivator protein pUL69 is contained within the tegument of virus particles. J. Virol. 70, 8984–8987.

Winkler, M., Rice, S.A., Stamminger, T., 1994. UL69 of human cytomegalovirus, an open reading frame with homology to ICP27 of herpes simplex virus, encodes a transactivator of gene expression. J. Virol. 68, 3943–3954.

Winkler, M., aus Dem Siepen, T., Stamminger, T., 2000. Functional interaction between pleiotropic transactivator pUL69 of human cytomegalovirus and the human homolog of yeast chromatin regulatory protein SPT6. J. Virol. 74, 8053–8064.

Woodhall, D., Bryant, L., Wilkinson, G., Sinclair, J., 2001. The HCMV 72-KDa major immediate early protein interacts physically and functionally with a constituent of ND10 bodies, hDAXX. Abstract presented at the 26th International Herpesvirus Workshop, Regensburg, Germany.

Yeung, K.C., Stoltzfus, C.M., Stinski, M.F., 1993. Mutations of the human cytomegalovirus immediate-early 2 protein defines regions and amino acid motifs important in transactivation of transcription from the HIV-1 LTR promoter. Virology 195, 786–792.

Yew, P.R., Berk, A.J., 1992. Inhibition of p53 transactivation required for transformation by adenovirus early 1B protein. Nature 357, 82–85.

Yoo, Y.D., Chiou, C.J., Choi, K.S., Yi, Y., Michelson, S., Kim, S., Hayward, G.S., Kim, S.J., 1996. The IE2 regulatory protein of human cytomegalovirus induces expression of the human transforming growth factor beta1 gene through an Egr-1 binding site. J. Virol. 70, 7062–7070.

Yurochko, A.D., Kowalik, T.F., Huong, S.M., Huang, E.S., 1995. Human cytomegalovirus upregulates NF-kappa B activity by transactivating the NF-kappa B p105/p50 and p65 promoters. J. Virol. 69, 5391–5400.

Yurochko, A.D., Mayo, M.W., Poma, E.E., Baldwin Jr., A.S., Huang, E.S., 1997. Induction of the transcription factor Sp1 during human cytomegalovirus infection mediates upregulation of the p65 and p105/p50 NF-kappaB promoters. J. Virol. 71, 4638–4648.

Zhong, S., Muller, S., Ronchetti, S., Freemont, P.S., Dejean, A., Pandolfi, P.P., 2000. Role of SUMO-1-modified PML in nuclear body formation. Blood 95, 2748–2752.

Zhou, Y.F., Leon, M.B., Waclawiw, M.A., Popma, J.J., Yu, Z.X., Finkel, T., Epstein, S.E., 1996. Association between prior cytomegalovirus infection and the risk of restenosis after coronary atherectomy. N. Engl. J. Med. 335, 624–630.

Zhu, H., Shen, Y., Shenk, T., 1995. Human cytomegalovirus IE1 and IE2 proteins block apoptosis. J. Virol. 69, 7960–7970.
