
Arch Microbiol (2002) 178: 13–25  
DOI 10.1007/s00203-002-0417-3  

**ORIGINAL PAPER**

Caroline M. Laemmlli · René Schönenberger · Marc Suter  
Alexander J. B. Zehnder · Jan R. van der Meer  

**TfdD<sub>II</sub>, one of the two chloromuconate cycloisomerases  
of *Ralstonia eutropha* JMP134 (pJP4),  
cannot efficiently convert 2-chloro-cis,cis-muconate  
to trans-dienelactone to allow growth on 3-chlorobenzoate**

Received: 16 October 2001 / Revised: 7 February 2002 / Accepted: 22 February 2002 / Published online: 13 April 2002  
© Springer-Verlag 2002  

---

**Abstract** *Ralstonia eutropha* JMP134 (pJP4) harbors two functional gene clusters for the degradation of chlorocatechols, i.e. *tfdCDEF* (in short: *tfd<sub>I</sub>*) and *tfdD<sub>II</sub>C<sub>II</sub>E<sub>II</sub>F<sub>II</sub>* (in short: *tfd<sub>II</sub>*), which are both present on the catabolic plasmid pJP4. In this study, we compared the function of both gene clusters for degradation of chlorocatechols by constructing isolated and hybrid *tfd<sub>I</sub>-tfd<sub>II</sub>* clusters on plasmids in *R*. *eutropha*, by activity assays of Tfd enzymes, and by HPLC/MS of individual enzymatic catalytic steps in chlorocatechol conversion. *R*. *eutropha* containing the *tfd<sub>II</sub>* cluster alone or hybrid *tfd*-clusters with *tfdD<sub>II</sub>* as sole gene for chloromuconate cycloisomerase were impaired in growth on 3-chlorobenzoate, in contrast to *R*. *eutropha* harboring the complete *tfd<sub>I</sub>* cluster. Enzyme activities for TfdD<sub>II</sub> and for TfdE<sub>II</sub> were very low in *R*. *eutropha* when induced with 3-chlorobenzoate. By contrast, a relatively high enzyme activity was found for TfdF<sub>II</sub>. Spectral conversion assays with extracts from *R*. *eutropha* strains expressing *tfdD<sub>II</sub>* all showed accumulation of a compound with a similar UV spectrum as 2-chloro-cis,cis-muconate from 3-chlorocatechol. HPLC analysis of in vitro assays in which each individual step in 3-chlorocatechol conversion was reproduced by sequentially adding cell extracts of an *Escherichia coli* expressing one Tfd enzyme only demonstrated that TfdD<sub>II</sub> was unable to cause conversion of 2-chloro-cis,cis-muconate. No accumulation of intermediates was observed with 4-chlorocatechol. From these results, we conclude that at least TfdD<sub>II</sub> is a bottleneck in conversion of 3-chlorocatechol and, therefore, in efficient metabolism of 3-chlorobenzoate. This study showed the subtle functional and expression differences between similar enzymes of the *tfd*-encoded pathway and demonstrated that extreme care has to be taken when inferring functionality from sequence data alone.

**Keywords** *Ralstonia eutropha* JMP134 (pJP4) · 2-Chloro-cis,cis-muconate · Modified *ortho* cleavage pathway · Catabolic pathway · Chloromuconate cycloisomerase · *tfd* genes

---

**Introduction**

*Ralstonia eutropha* JMP134 (pJP4) was originally isolated in Australia from an unspecified soil sample and can use 2,4-dichlorophenoxyacetic acid (2,4-D) and 3-chlorobenzoate as sole carbon and energy sources (Don and Pemberton 1981; Don et al. 1985). Fifteen genes are involved in 2,4-D degradation (*tfd* genes), which are located within a 22-kb DNA fragment on plasmid pJP4. Among these are *tfdA*, an α-ketoglutarate-dependent dioxygenase (Streber et al. 1987; Fukumori and Hausinger 1993) encoding the first enzyme to attack 2,4-D, and *tfdB*, encoding 2,4-dichlorophenol hydroxylase catalyzing the conversion of 2,4-dichlorophenol to 3,5-dichlorocatechol. A cluster of four genes (*tfdCDEF*) was shown to code for enzymes of a so-called modified *ortho* ring cleavage pathway, catalyzing the conversion of 3,5-dichlorocatechol to 3-oxoadipate (Perkins et al. 1990). Degradation of 3-chlorobenzoate proceeds similarly, although the first two steps are not encoded by genes on plasmid pJP4. 3-Chlorobenzoate is first oxidized at the 1,2 or 1,6 positions by a benzoate dioxygenase (Pieper et al. 1993). Secondly, a dehydrogenase oxidizes the dihydriodiols to form 3-chloro- and 4-chlorocatechol. Approximately two-thirds of 3-chloro- and one-third of 4-chlorocatechol are formed from 3-chlorobenzoate (Pieper et al. 1993). Further metabolism of the generated chlorocatechols is then thought to proceed similarly as for 2,4-D through a modified *ortho* ring cleavage pathway encoded on plasmid pJP4.

Expression of all *tfd* genes is regulated by the two identical LysR-type regulatory proteins, TfdR and TfdS (Harker et al. 1989; Kaphammer et al. 1990; Matru-
butham and Harker 1994; You and Ghosal 1995; Leveau and van der Meer 1996). Another gene, called *tfdK*, codes for a 2,4-D transporter active at low-micromolar concentrations of 2,4-D (Leveau et al. 1998). The *tfdR*, *tfdS* and *tfdK* genes lie within a 10.5-kb region flanked by one complete and one partial copy of the IS element ISJP4 (Leveau and van der Meer 1997). Recently, five other genes, named *tfdDII*, *tfdCII*, *tfdEII*, *tfdFII*, were discovered between *tfdR* and *tfdK* (Leveau et al. 1999; Laemmlie et al. 2000). Expression studies in *Escherichia coli* showed that *tfdDII* codes for a chloromuconate cycloisomerase, *tfdCII* for a chlorocatechol 1,2-dioxygenase, *tfdEII* for a dienelactone hydrolase and *tfdFII* for a maleylacetate reductase (Laemmlie et al. 2000). This indicated that two equivalent sets of genes encoding enzymes for the degradation of chlorocatechols are present on pJP4. Sequence comparisons suggested that the two gene clusters had been acquired from different origins, because the actual percentage of identity among the counterparts was rather low (15–62% on amino acid level) and the G+C content of the *tfdDII*, *tfdCII*, *tfdEII*, *tfdFII* genes was significantly higher than that of the *tfdCDEF* genes (Laemmlie et al. 2000). This raised some questions as to what molecular events led to the current situation of two paralogous gene clusters and which advantages – if any – are presented by the current gene configuration. Previously, we presented evidence suggesting that two different gene clusters for the 2,4-D pathway on pJP4 had been recombined by the action of the ISJP4 insertion element (Leveau and van der Meer 1997). It was suspected that a previously complete regulatory gene, *tfdT*, necessary for activation of transcription from the *tfdCDEF* genes, had been inactivated by the insertion of ISJP4 (Leveau and van der Meer 1996). Therefore, there seems to be a selective advantage for keeping the current situation, because deletion of the region flanked by the ISJP4 copies would lead to loss of *tfdR* and *tfdS*, the present regulatory genes for the *tfd* pathway genes (Laemmlie et al. 2000). We also suspected that there might be subtle functional differences between the *tfdCDEF* (in short *tfdI*) and *tfdDII*, *tfdCII*, *tfdEII*, *tfdFII* (in short *tfdII*) counterparts.

The objective of this study was to compare the function of both *tfdI* and *tfdII* gene clusters with respect to chlorocatechol degradation in *R. eutropha*. For this purpose, the *tfdCDEF* genes, the *tfdDII*, *tfdCII*, *tfdEII*, *tfdFII* genes, and three hybrid gene clusters were cloned separately into *R. eutropha* JMP289. With the help of in vivo growth experiments as well as in vitro enzyme reactions, we obtained evidence that TfdDII is unable to catalyze the conversion of 2-chloro-*cis*,*cis*-muconate to *trans*-dienelactone, in contrast to TfdD. As a consequence, four different phenotypes with respect to growth on 3-chlorobenzoate were observed among the *R. eutropha* JMP289 derivatives.

---

**Materials and methods**

Bacterial strains and growth conditions

*R. eutropha* JMP134 (pJP4) can use 2,4-D and 3-chlorobenzoate as sole carbon and energy source (Don and Pemberton 1981). *R. eutropha* JMP289 (Don et al. 1985) is a rifampin-resistant derivative of *R. eutropha* JMP134 (pJP4) which was cured of plasmid pJP4. It was used as recipient strain in filter matings with *E. coli* DH5α (Sambrook et al. 1989) harboring pKT230-derived plasmids and *E. coli* HB101 (pRK2013) as helper strain (Ditta et al. 1980). *E. coli* BL21 (DE3) (pLysS) (Studier and Moffatt 1986), which carries the T7 RNA polymerase gene under control of the *lacUV5* promoter, was used for the T7-directed expression of pRSET6a-derived plasmids (Schoepfer 1993). *E. coli* cultures were grown at 37°C in Luria-Bertani (LB) medium supplemented with the appropriate antibiotic (at 100 μg/ml). *R. eutropha* cultures were grown at 30°C in nutrient broth (Biolife, Milan, Italy) or in *Pseudomonas* mineral medium 21A (MM) (Gerhardt et al. 1981) supplemented with 10 mM fructose, 1, 3 or 5 mM 3-chlorobenzoate plus the appropriate antibiotic (at 100 μg/ml). Growth was followed by measuring the optical density at 546 nm (*OD*~546~). In order to correct for the dark color produced by some strains during growth on 3-chlorobenzoate, culture samples were centrifuged for 1 min at 15,000×g and the *OD*~546~ of the supernatant was subtracted from the total *OD*~546~ value.

DNA manipulations

Plasmid DNA isolations, transformations and other DNA manipulations were carried out according to established procedures (Sambrook et al. 1989). Restriction enzymes and other DNA-modifying enzymes were obtained from Amersham Life Science or GIBCO/BRL and used according to the specifications of the manufacturer. Oligonucleotides for the polymerase chain reaction (PCR) were obtained from Microsynth (Balgach, Switzerland). The PCR mixtures contained 200 pmol of each primer per ml, 20 mM Tris-HCl (pH 8.4), 50 mM KCl, 0.05% (v/v) of W-1 reagent (Gibco/BRL), 2 mM MgCl₂, 0.25 mM of each deoxynucleotide triphosphate, and 30 U of *Taq* DNA polymerase (GIBCO/BRL) per ml. DNA was sequenced on a LiCOR IR² automated sequence reader as described previously (Ravatn et al. 1998). DNA sequences were analyzed with the DNASTAR software (DNASTAR, Madison, Wis., USA).

Plasmids

The most relevant plasmids constructed in this study are depicted in Fig. 1. Plasmids pUC18, pUC19 (Yanisch-Perron et al. 1985), pUC28 (Benes et al. 1993), pGEM-T Easy, pGEM-7Zf(+) (Promega, Madison, Wis., USA) and the expression vector pRSET6a (Schoepfer 1993) were used as general cloning vectors. Plasmid pKT230 (Bagdasarian et al. 1981) is a mobilizable, broad-host-range vector. All fragments amplified by using the PCR were sequenced to exclude undesired mutations.

For in vivo expression studies, several pKT230-derived plasmids with different combinations of the *tfd* genes from pJP4 were constructed. Plasmid pCBA59 contains the hybrid regulon *tfdR-CDEF* as described previously (Leveau and van der Meer 1996). In order to clone the cluster *tfdR-DII*, *tfdCII*, *tfdEII*, *tfdFII* into pKT230, a 6.9-kb *SacI* fragment of pJP4 was first cloned into pUC, which resulted in plasmid pCBA89. A 6.5-kb *HindIII-BglII* fragment was retrieved from pCBA89 and cloned into pKT230 cut with *HindIII* and *BamHI*, yielding plasmid pCBA93. The *tfdR-DII*, *tfdCII*, *tfdEII*, *tfdFII* cluster was extended with *tfdD* as follows. The *tfdD* gene was amplified by using the PCR with the forward primer (980104, 5′-CGGTACCCAGGCGGGGGGCAAAACC-3′, *KpnI* site underlined) and the reverse primer (980103, 5′-TTGGCTAGCTGACGCGT- GCGTATTGG-3′, *NheI* site underlined) and subsequently cloned into pGEM-T Easy. This resulted in plasmid pCBA127. The *KpnI-NheI* fragment of pCBA127 (containing *tfdD*) was retrieved and cloned into *KpnI-NheI*-digested pCBA93. This yielded plasmid pCBA129, containing the *tfdR-DII*, *tfdCII*, *tfdEII*, *tfdFII* regulon extended with the *tfdD* gene. The *tfdDII* gene on pCBA129 was knocked out by digesting pCBA118 (Fig. 1) with *SacII*. Religation resulted in plasmid pCBA217 in which the 230-bp *SacII* fragment at the end of *tfdDII* was missing. The *XbaI-BglII* fragment of pCBA129 was
(BamHI/BglII)  XbaI  SacII  BglII  EcoRI  SacI  KpnI  HindIII  NheI  pCBA93
|---|---|---|---|---|---|---|---|---|
| tfdR | tfdDII | tfdCII | tfdEII | tfdFII | (tfdBII) | pKT230 |

HindIII  SacI  Scal  ClaI  (BglII/BamHI)  pCBA59
|---|---|---|---|---|---|
| tfdR | tfdC | tfdD | tfdE | tfdF | (tfdB) | pKT230 |

KpnI  ClaI  NheI  pCBA127
|---|---|---|
| tfdD | pGEM®-T Easy |

(BamHI/BglII)  XbaI  Ncol  SacII  BglII  EcoRI  KpnI  NheI  pCBA129
|---|---|---|---|---|---|---|---|---|
| tfdR | tfdDII | tfdCII | tfdEII | tfdFII | (tfdBII) | tfdD | pKT230 |

XbaI  Ncol  SacII  BglII  EcoRI  pUC19  pCBA118
|---|---|---|---|---|
| tfdDII | tfdCII | tfdEII |

XbaI  Ncol  SacII  BglII  EcoRI  pUC19  pCBA217
|---|---|---|---|---|
| (tfdDII) | tfdCII | tfdEII |

(BamHI/BglII)  XbaI  Ncol  SacII  BglII  EcoRI  KpnI  NheI  pCBA220
|---|---|---|---|---|---|---|---|---|
| tfdR | (tfdDII) | tfdCII | tfdEII | tfdFII | (tfdBII) | tfdD | pKT230 |

EcoRI  Scal  Ndel  BamHI  pUC28  pCBA133
|---|---|---|---|
| tfdDII |

EcoRI  Scal  Ndel  Ncol  SacII  BamHI  XbaI  MluI  HindIII  pUC28  pCBA134
|---|---|---|---|---|---|---|---|---|
| tfdDII |

EcoRI  Scal  Ndel  Ncol  SacII  BamHI  XbaI  MluI  pCBA135
|---|---|---|---|---|---|---|---|
| tfdDII | tfdE | tfdF | (tfdB) | pUC28 |

EcoRI  Scal  Ndel  Ncol  SacII  BamHI  XbaI  pGEM7Zf(+)  pCBA141
|---|---|---|---|---|---|---|
| tfdE | tfdF | (tfdB) | pUC19 |

SacI  HindIII  SacI  NdeI  Ncol  SacII  BamHI  XbaI  pGEM7Zf(+)  pCBA147
|---|---|---|---|---|---|---|---|
| tfdDII |

SacI  NdeI  Ncol  SacII  BamHI  XbaI  SacI  pKT230  pCBA150
|---|---|---|---|---|---|---|---|
| tfdDII | tfdE | tfdF | (tfdB) |

Fig.1 Maps of the most relevant plasmid constructs used in this study. All plasmids depicted here except pCBA59 were constructed as described in Materials and methods. pCBA59 was constructed previously (Leveau and van der Meer 1996). For each plasmid, the vector and the restriction sites relevant to the construction are indicated. Boxes Genes, dashed boxes non-coding pJP4 DNA. A gene name in parentheses indicates that the gene is incomplete. Restriction sites in parentheses indicate sites destroyed during cloning.

then replaced with the XbaI-BglIII fragment of pCBA217, which resulted in plasmid pCBA220.

In order to be able to clone $t f d D_{II}$ gene fragments behind the $t f d C$ reading frame (Figs. 1 and 2), an NdeI site was created at its 3' end. For this purpose, the 3' end of $t f d C$ was custom amplified with primers 980501 (5'-GCGGAATTCCGT TGACCACGCAGTACTCTCG-3', EcoRI site underlined) and 980502 (5'-GCGCATATGCACGGGTTTGCCCCCGCCGTC-3', NdeI site underlined) using plasmid pCBA4 (Leveau and van der Meer 1996) as template, cut with EcoRI and NdeI, and ligated with pUC28 cut with the same enzymes (yielding pCBA133). Secondly, an NdeI site was created at the start of the $t f d D_{II}$ ORF and a BamHI site at the end of the $t f d D_{II}$ fragment. However, since the actual start of the $t f d D_{II}$ gene is not known, two possibilities were chosen (Fig. 2). Furthermore, in two fragments the $t f d D_{II}$ reading frame was extended by introducing a frame-shift at the 3' end (Fig. 2). This could produce a peptide with a more similar length and an extended number of identical amino acid residues in comparison to TfdD. Several intermediate cloning steps were necessary to place the $t f d D_{II}$ gene fragments behind $t f d C$, as outlined below.

The first $t f d D_{II}$ fragment (fragment A) included the reading frame from the most 5'-located start site until the normal stop codon (Fig. 2). This fragment was amplified from pCBA89 by using primers 971202 (5' TTCTAGACTACCATATGCTCACA 3', NdeI site underlined) and 971201 (5' GGACTTCGCGCACGGA TCCTCCTCGA 3', BamHI site underlined). A fragment with a more downstream located start site (fragment B) was encompassed between primers 971203 (5' CGCAAGCATATGCAGATCGA AGCGA 3') and 980707 (5' CGGGTGGATCCGGCGCTCAGGC 3'). The longest $t f d D_{II}$ fragment (fragment C) was encompassed by primers 971202 and 980707. Fragments B and C contained a frame-shift mutation due to an additional cytosine (Fig. 2B) and were obtained by PCR in two steps. First, primers 971202 or 971203 were used in combination with 981004 (5' TCGCCCCCGTA cGGC AGCGA 3') (Fig. 2B). The resulting two PCR fragments were purified and used as 'primers' in a new subsequent reaction with primer 980707. This resulted in the proper length $t f d D_{II}$ fragments. A fourth $t f d D_{II}$ fragment (fragment D) was amplified by using primers 971202 and 980707 directly. This $t f d D_{II}$ fragment therefore did not contain the extra frame-shift mutation. The fragments A, B, C, and D were then cloned into pGEM-T Easy. From here they were digested with NdeI and BamHI and cloned into NdeI-BamHI of pCBA133. This resulted in pCBA134 ($t f d D_{II}$ fragment A), pCBA149 ($t f d D_{II}$ fragment B), pCBA164 ($t f d D_{II}$ fragment C), and pCBA145 (fragment D). In addition, the same NdeI-BamHI fragments were cloned into pRSET6a cut with the same enzymes.

One complete hybrid regulon was then assembled from pCBA134 by first cutting with MluI and HindIII and then ligating this part with the MluI-HindIII fragment containing $t f d E F$ and part of $t f d B$ from pCBA4. This yielded pCBA135. A three-point ligation including the HindIII-ScaI fragment of pCBA59 ($t f d R-t f d C$), the ScaI-SacI fragment of pCBA135 ($t f d D_{II} E F$ and part of $t f d B$), and pUC19 cut with HindIII and SacI resulted in pCBA136, in which $t f d R-t f d C D_{II} E F$ were combined (Fig. 1).

In order to be able to create the hybrid regulons with the different $t f d D_{II}$-fragments on pKT230, first pCBA136 was digested with XbaI and SacI, and the resulting $t f d R-C D_{II}$ fragment was recovered. This fragment was ligated with pGEM7Z(+), cut with the same enzymes (yielding pCBA141). The NdeI-XbaI $t f d D_{II}$ fragment from pCBA145 (fragment D) was then used to replace the NdeI-XbaI fragment of pCBA141 (yielding pCBA147). Similarly, the NdeI-XbaI fragments of pCBA149 and pCBA164 were used to replace $t f d D_{II}$ fragment A of pCBA141 (not shown). Finally, the HindIII-XbaI $t f d R-C D_{II}$ fragment from pCBA136 was replaced with the HindIII-XbaI fragment from pCBA147 resulting in pCBA146. From pCBA146 the $t f d R-C D_{II} E F$ SacI fragment was introduced in the SacI site of vector pKT230 (yielding pCBA150). Plasmids pCBA161 (with $t f d D_{II}$ fragment B) and pCBA168 (with $t f d D_{II}$ fragment C), were constructed similarly as pCBA150.

pKT230-derived plasmids were then transferred to *R. eutropha* JMP289 by triparental mating as described elsewhere (Leveau and van der Meer 1996). *R. eutropha* transconjugants were selected on MM-agar plates supplemented with 10 mM fructose, rifampin (25 μg/ml) and either streptomycin or kanamycin (at 100 μg/ml). Colonies of potential transconjugants were inoculated in nutrient broth plus antibiotics and subsequently purified by streaking on selective agar plates. Plasmid DNA was isolated and analyzed by gel electrophoresis.

### $t f d$ Gene expression in *R. eutropha*

*R. eutropha* JMP289 strains harboring pKT230-derived plasmids were grown in 50 ml MM supplemented with 10 mM fructose to an OD$_{546}$ of between 0.8 and 0.9. To induce expression of the $t f d$ genes, 3-chlorobenzoate was added to a final concentration of 1 mM and strains were incubated for an additional 2 h. RNA was then isolated from 1.5-ml samples as described previously (Bau-mann et al. 1996). DNaseI-treated RNA samples were spotted on Hybond N+ membranes and hybridized with biotin-labeled antisense RNAs for the $t f d C D$, $t f d F$, $t f d C_{II}$ and $t f d F_{II}$ genes, as described elsewhere (Leveau et al. 1999). Spot densities were measured by using the program METAVIEW (version 4.1, Universal Imaging, Visitron Systems, Germany), subtracted with the background density obtained in hybridizations with RNA from *R. eutropha* JMP289, normalized for differences in total amount of RNA, and expressed relative to a dilution series of plasmid DNA (plasmid pCBA59 for antisense probes $t f d C D$ and $t f d F$; plasmid pCBA93 for antisense probes $t f d C_{II}$ and $t f d F_{II}$).

From the same cultures, 50 ml was centrifuged for 15 min at 6,000×g at 4°C, washed with 50 ml 20 mM Tris-HCl (pH 7), and resuspended in 1 ml 20 mM Tris-HCl (pH 7). The cell suspensions were disrupted by sonification on ice during five periods of 15 s each at an output of 30–40 W (Branson Sonifier 450, SCAN AG, Basel, Switzerland), with at least 1 min pause between each repetition. Subsequently, the suspensions were centrifuged at 4°C for 30 min at 16,300×g. The resulting supernatants, referred to as cell extracts, were used in enzyme assays and in conversion experiments (see below). Enzyme assays for chlorocatechol 1,2-dioxygenase, chloromuconate cycloisomerase, dienelactone hydrolase, and maleylacetate reductase were performed by spectrophotometric methods as described previously (Laemml et al. 2000). Extinction coefficients were taken from Dorn and Knackmuss (1978). 3-Chlorocatechol, 3-chloromuconate, *cis*-dienelactone and maleylacetate served, respectively, as substrates. No additional dienelactone hydrolase was added to assay cycloisomerization of muconates.

Conversion of 3-chloro- and 4-chlorocatechol by cell extracts of *R. eutropha* JMP289 strains harboring pKT230-derived plasmids was followed spectrophotometrically in 0.5-ml quartz cuvettes. Reaction mixtures contained 20 mM Tris-HCl (pH 7), 1 mM MnSO₄ and 0.1 mM substrate. The conversion reaction was started by the addition of cell extract (containing 0.05–0.2 mg protein). Spectral changes were monitored by scanning the absorption of the reactions between 200 and 350 nm at different time intervals.

### Expression in *E. coli* and preparation of samples for HPLC/MS analysis

The preparation of cell extracts of *E. coli* BL21 (DE3) (pLysS) strains harboring pRSET6a-derived plasmids and enzyme assays were carried out as described above for *R. eutropha* and previously (Laemml et al. 2000). The sole difference was that the washing buffer contained 20 mM ammonium acetate and 1 mM MnCl₂. Protein concentrations were determined as described by Bradford (1976) using bovine serum albumin as a standard.

In order to study the stepwise conversion of 3-chloro- and 4-chlorocatechol by the Tfd and Tfd$_{II}$ enzymes, the following reactions were set up. Different combinations of cell extracts of *E. coli* BL21 (DE3) (pLysS) strains harboring pRSET6a-derived plasmids were added to reaction mixtures containing 20 mM ammonium acetate (pH 7), 1 mM MnCl₂ and 1 mM of substrate (3-chloro- or 4-chlorocatechol) in a final volume of 0.5 ml. An amount of 50 μl of each cell extract was added, which corresponded to about 0.1 to 0.7 mg of protein. Subsequently, the samples were incubated while

Fig. 2 A DNA sequences of the links between the ORFs of \( t f d C \) and \( t f d D_{II} \) in plasmids pCBA136 (\( t f d D_{II} \) fragment A), pCBA150 (\( t f d D_{II} \) fragment D), pCBA168 (\( t f d D_{II} \) fragment C) (panel a) and plasmid pCBA161 (\( t f d D_{II} \) fragment B, panel b). B Part of the sequence of \( t f d D_{II} \) showing the location of the primers used to construct the different \( t f d D_{II} \) fragments with alternative start codon and frame-shift. Amino acid translations are given below the DNA sequence for wild-type TfdDII (no. 1), for TfdDII with frame-shift mutation at position 2629 (no. 2) and for wild-type TfdD (no. 3). Boxes point to identical amino acid residues. C Location and size of the four PCR-derived \( t f d D_{II} \) fragments. Depicted sequences were assessed from GenBank accession numbers: U16782 (\( t f d D_{II} \)) and M35097 (\( t f d D \))

A

a

Ndel

\( 5^{\prime}-\text { CCCGCCGCGTGCAGGGGGGCAAAACCCGTGACATATGCTCACAGAAAAAGCCATTGCCGACAGCCCC-3 } \)

P A R A Q A G A N P . M L T E K A I A D S P

\( t f d C \)

\( t f d D_{II-A / C / D} \)

b

Ndel

\( 5^{\prime}-\text { CCCGCCGCGTGCAGGGGGGCAAAACCCGTGACATATGCAGATCGAAGCGATCGAAACGGTGATCGTC-3 } \)

P A R A Q A G A N P . M Q I E A I E T V I V

\( t f d C \)

\( t f d D_{II-B} \)

B

971202 CAT

\( \text{GGCATTTCTAGACTACC GCCATGCTCACAGAAAAAGCCATTGCCGACAGCCCCAACGGCGGCGACGCCGAT} \)

CCGTAAAGATCTGATGGCGGTACGAGTGTCTTTTTCGGTAACGGCTGTCGGGGTTGCCC GCTGCGGCTA

M L T E K A I A D S P N G G D A D 1

M L T E K A I A D S P N G G D A D 2

971203 CATAT

\( \text{CGCAAAGGCGGCGCAGATCGAAAGCGATCGA} \)

\( \text{TGCATCGATCGAGTCGCGCTG CCTACGGGGGGCGA} \)

\( \text{ACGTAGCTAGCTCAGGACGGATGCC CCCCGCT} \)

R K A A Q I E A I E A S I E S L P T G A 1

R K A A Q I E A I E A S I E S L P T [G] G [E] 2

V K I D A I E S T V P S L P F [G] C [E] 3

A-G 971201

\( \text{AGTCATCGGGCCCTTGCTGATCGAGGAGGACCTGTGCGAA GTCCCCGCGGTCTACAAGGAACATGCCCTGT} \)

\( \text{TCAGTAGCCCCGGGAACGACTAGCTCCTCCTGGACACGCTTCAGGGGCGCCAGATGTTCCTTGTACGGGACA} \)

K S S G P C .

V [I G P] L L I E E D [L] C E V A V Y K E H A [L] W 1

L [I G P] F V L A D T [L] S H E [P] L E I R D Y E [L] Q 2

3

\( \text{GGCTCCCCGAAAGGCCCCGGGCTCGGAATCCGACTCGAGAGAACCGGTGCGGCGCTTCGCGCGTGCTTCG} \)

\( \text{CCGAGGGGCTTCCGGGCCCCGGAGCCTTAGGCTGAGCTGCTTTGGTCCA GCCGCGCGAAGCGCGCACGAAGC} \)

L [P] E [G] P [G] L [G] I R [L D E] N Q [V R] R F [A R] A [S] .

V [P] T [G] V [G] H [G] M T [L D E] D K [V R] Q Y [A R] V [S] .

CC 980707

\( \text{TCCCA GCGCATCGATCGTCACAGCGCCTGAGCGCCACATCCACCCGATTCTTTTTGAAGAGAAAAGCA} \)

\( \text{AGGGTCGCGTAGCTAGCAGTGT CGCGGACTCGCGGTGTAGGTGGGCTAAGTAGAAAAAACTTCTTTTCGT} \)

C

Xbal Ncol Sacll Sacll Apal

\( t f d D_{II} \)

PCR fragment

A

B

C

D

shaking at 200 rpm for 2 h at \( 30^{\circ} \mathrm{C} \). The reactions were stopped by the addition of \( 30 \mu \mathrm{l} \) hydrochloric acid (37%), which led to precipitation of all proteins, and centrifuged at \( 16,300 \times g \) for 15 min at \( 4^{\circ} \mathrm{C} \). The supernatants were transferred into clean vials and analyzed immediately by HPLC/UV/MS.

HPLC/UV/MS

As standards 1 mM 3-chloro-, 1 mM 4-chlorocatechol, 1 mM 2-chloro-cis,cis- and 1 mM 3-chloro-cis,cis-muconate, acidified by the addition of \( 30 \mu \mathrm{l} \) hydrochloric acid (37%) to \( 0.5 \mathrm{ml}\) substrate,

were used. All solvents were HPLC-grade (Scharlau, Barcelona, Spain) and filtered before use (0.2 μm). The HPLC used for LC/MS was a Hewlett Packard Series 1100 (Hewlett Packard Schweiz AG, Urdorf, Switzerland) with a variable wavelength detector set to 268 nm. The HPLC column (125×2 mm) used was a Nucleosil 120–3, with 3-μm particles (Macherey Nagel, Oensingen, Switzerland). Eluent A was 12 mM phosphate buffer (pH 2.3), and eluent B methanol. The eluents were degassed using an on-line degasser DG4 from Henggeler Analytic Instruments (Riehen, Switzerland). The column was conditioned for 5 min with eluent of initial composition (80% A, 20% B). The initial conditions were kept for the first 15 min of the run and then changed to 100% B in

5 min. These conditions were kept for 10 min. The initial conditions were then reestablished within 5 min and the column reequilibrated for another 5 min, giving a total running time of 40 min. The flow rate through the column was 150 μl/min. The column was kept at 32°C and 50-μl samples were injected.

All mass spectra were acquired on a Platform LC single quadrupole mass spectrometer using electrospray ionization (Micromass UK, Manchester, UK). Full-scan spectra were acquired in negative ion mode, scanning from \( m/z \) 50–400 at 1 s/scan. The mass range was calibrated and the sensitivity of the instrument tested using 2.5 mM NaNO₃ infused at a flow rate of 70 μl/min. The electrospray interface temperature was set to 150°C and the nitrogen gas flow to 500 l/h. The needle and cone voltages were optimized to give maximum signal intensity for the target analyte signal and were –5 kV and –10 V, respectively. The so-called pepper pot was used as counter electrode for all experiments. Low and high mass resolution of the quadrupole was set to 12.0 and 16.5, respectively, and the ion energy to 1, giving roughly mass resolution over the whole mass range. The multiplier was operated at 750 V.

### Chemicals

3-Chloro- (purity 99%) and 4-chlorocatechol (purity 95–99%) were purchased from Promochem (Wesel, Germany). 2-Chloro-cis,cis-muconate, 3-chloro-cis,cis-muconate, cis- and trans-dienelactone (purity 95%) were a kind gift of Dr. Walter Reineke (Bergische Universität-Gesamthochschule Wuppertal, Wuppertal, Germany). Maleylacetate was prepared by alkaline hydrolysis of cis-dienelactone (Evans et al. 1971) by mixing 1 ml of 5 mM cis-dienelactone with 7.5 μl of 2 N NaOH and incubating for 15 min at room temperature.

---

### Results

TfdD<sub>II</sub>, but not TfdD, limits growth on 3-chlorobenzoate by *R. eutropha* JMP289 *tfd* derivatives

Independent expression of the paralogous gene clusters *tfd<sub>I</sub>* and *tfd<sub>II</sub>* in *R. eutropha* JMP289 revealed two opposite phenotypes with respect to growth on 3-chlorobenzoate. Whereas introduction of plasmid pCBA59 (*tfd<sub>I</sub>*) into *R. eutropha* JMP289 allowed growth on up to 5 mM 3-chlorobenzoate (Fig. 3A), *R. eutropha* JMP289 expressing the *tfd<sub>II</sub>* gene cluster from plasmid pCBA93 was unable to use 1 mM 3-chlorobenzoate as sole carbon and energy source for growth (Fig. 3B). This phenotype was partially overcome by adding *tfdD* at the end of the *tfd<sub>II</sub>* gene cluster.

![Figure 3A-E](attachment:fig_3a-e.png)

**Fig. 3A–E** Growth of *Ralstonia eutropha* JMP289 *tfd* derivatives on 3-chlorobenzoate. A *R. eutropha* JMP289 with plasmid pCBA59, B pCBA93, C pCBA150, D pCBA129 and E pCBA220. The data points represent the average of at least three independent experiments (except for pCBA59, duplicate, and pCBA129, one representative curve). Error bars on the *x*-axis represent the time-range differences, whereas those on the *y*-axis show the minimum and maximum OD<sub>546</sub> measured within the given time range. ▲ 5 mM, ● 3 mM, ■ 1 mM 3-chlorobenzoate, □ no 3-chlorobenzoate

Fig. 4 A Specific enzyme activity encoded by the *tfd* genes as measured in total cell extracts of *R*. *eutropha* JMP289 *tfd* derivatives. 1 U corresponds to 1 mmol of substrate disappearance or product formation per min. Open bars Chlorocatechol 1,2-dioxygenase activity, stippled bars chloromuconate cycloisomerase, hatched bars dienelactone hydrolase, black bars maleylacetate reductase activity. B Transcription of the *tfd* genes of the different *R*. *eutropha* strains after induction with 3-chlorobenzoate. Antisense mRNA probes used for hybridization are indicated on the left. Note that the *tfdC<sub>II</sub>* probe partially overlaps with the *tfdD<sub>II</sub>* gene. C Average expression levels normalized to a plasmid DNA standard and to differences in total RNA amounts. Relative levels of *tfdCD* to *tfdF* and of *tfdC<sub>II</sub>* and *tfdF<sub>II</sub>* can be compared between the different strains. Open bars, *tfdCD* probe; black bars, *tfdF* probe; gray bars, *tfdC<sub>II</sub>* probe; black and white striped bars, *tfdF<sub>II</sub>* probe. Error bars in A and C indicate the average deviation from the average in three independent experiments

was exchanged for *tfdD<sub>II</sub>* [i.e. *tfdR-CD<sub>II</sub>EF(B)*] *R*. *eutropha* JMP289 (pCBA150) could still grow with 1 mM (Fig. 3C), but not with 3 or 5 mM 3-chlorobenzoate. This suggested that the chloromuconate cycloisomerase encoded by *tfdD<sub>II</sub>* was not as efficient as that encoded by *tfdD* in converting 2-chloro- or 3-chloro-cis,cis-muconate, the intermediates arising from 3-chlorobenzoate. On the other hand, knocking out *tfdD<sub>II</sub>* on plasmid pCBA129 [i.e., *tfdR-C<sub>II</sub>E<sub>II</sub>F<sub>II</sub>(B<sub>II</sub>)D*], as in plasmid pCBA220, allowed growth of *R*. *eutropha* JMP289 on 1 mM 3-chlorobenzoate, but no longer on 3 mM (Fig. 3E). This difference suggested at least some supportive role of TfdD<sub>II</sub>.

Interestingly, all *R*. *eutropha* JMP289 *tfd* derivatives except for *R*. *eutropha* JMP289 (pCBA59) and *R*. *eutropha* JMP289(pCBA150) accumulated a brownish compound in the medium during incubation with 3-chlorobenzoate. This coloring may be due to polymerization of accumulating chlorocatechols and is known to have a toxic effect on cell metabolism (Fritz et al. 1991).

Expression patterns of the *tfd* genes in the *R eutropha* JMP289 *tfd* derivatives

In order to determine whether any of the enzymes involved in chlorocatechol metabolism was impaired, cell extracts from all *R*. *eutropha* JMP289 *tfd* derivatives were assayed for chlorocatechol 1,2-dioxygenase, chloromuconate cycloisomerase, dienelactone hydrolase and maleylacetate reductase activity. Cell extracts of *R*. *eutropha* JMP289 and *R*. *eutropha* JMP134 induced with 3-chlorobenzoate were used as negative and positive controls, respectively. The specific enzyme activities, after induction with 3-chlorobenzoate, revealed two distinct enzyme patterns for the expression of the *tfd<sub>I</sub>* cluster and the *tfd<sub>II</sub>* cluster (Fig. 4A). Strains expressing only the *tfd<sub>I</sub>* cluster generally displayed relatively high levels of chlorocatechol 1,2-dioxygenase, chloromuconate cycloisomerase, and dienelactone hydrolase compared to maleylacetate reductase. For example, *R*. *eutropha* JMP289 (pCBA59) expressed 38 mU·(mg protein)<sup>-1</sup> chlorocatechol 1,2-dioxygenase, 27 mU·(mg protein)<sup>-1</sup> chloromuconate cycloisomerase, and 112 mU·(mg protein)<sup>-1</sup> dienelactone hydro-

lase but only 1 mU·(mg protein)⁻¹ maleylacetate reductase activity. On the other hand, strains expressing the *tfd*~II~ cluster displayed low activities of chlorocatechol 1,2-dioxygenase, chloromuconate cycloisomerase, and dienelactone hydrolase compared to relatively high maleylacetate reductase activities. For example, only 10 mU·(mg protein)⁻¹ chlorocatechol 1,2-dioxygenase, 6.4 mU·(mg protein)⁻¹ chloromuconate cycloisomerase, and 6 mU·(mg protein)⁻¹ dienelactone hydrolase activity were measured in *R. eutropha* JMP289 (pCBA93), whereas maleylacetate reductase activity was 285 mU·(mg protein)⁻¹ and was thereby higher than that encoded by the *tfd*~I~ cluster. Clearly, all measured enzyme activities were absent in strain JMP289 itself and the activity pattern measured in cell extracts of *R. eutropha* JMP134 was more or less a combination of both *tfd*~I~- and *tfd*~II~-type patterns (Fig. 4A). Transcription analysis confirmed that the *tfd* genes were transcribed from the hybrid constructs. *R. eutropha* JMP289 (pCBA129) clearly contained *tfdD* mRNA and *R. eutropha* JMP289 (pCBA150) contained *tfdD*~II~ mRNA after induction with 3-chlorobenzoate (Fig. 4B, C).

The expression patterns of the *tfd*~I~ and the *tfd*~II~ hybrid clusters showed only few apparent changes compared to the wild-type clusters, thereby not directly revealing the reasons for the observed growth differences. For example, strains JMP289 (pCBA129) and JMP289 (pCBA220) showed the same enzyme activity pattern as *R. eutropha* JMP289 (pCBA93) (Fig. 4), although their phenotypes with respect to 3-chlorobenzoate differed considerably (Fig. 3B, E). The activity pattern of strain JMP289 (pCBA150) strongly resembled that of JMP289 (pCBA59) (i.e., the *tfd*~I~ cluster), except for its 4.4-fold higher chlorocatechol 1,2-dioxygenase activity. With 166 mU·(mg protein)⁻¹, JMP289 (pCBA150) had the highest chlorocatechol 1,2-dioxygenase activity of all strains. We presume that this high activity is the result of an accumulation of the inducer 2-chloro-*cis*,*cis*-muconate, leading to overexpression of the *tfdCD*~II~ genes, although not so obviously of the *tfdE* and *tfdF* genes (Fig. 4B, C). Accumulation of 2-chloro-*cis*,*cis*-muconate probably takes place, because it is not converted by TfdD~II~ at measurable rates (see below). The high chlorocatechol 1,2-dioxygenase activities in strains *R. eutropha* JMP289 (pCBA59) and (pCBA150) explain why these strains were the only ones that did not accumulate chlorocatechols and became brown.

Conversion of 3-chloro- and 4-chlorocatechol by cell extracts of *R. eutropha* JMP289 derivatives

Since metabolism of 3-chlorobenzoate by *R. eutropha* JMP134 and JMP289 leads to a mixture of 3-chloro- and 4-chlorocatechol (Pieper et al. 1993), it was necessary to consider both degradation pathways individually when analyzing the function of the Tfd~I~ and Tfd~II~ enzymes during 3-chlorobenzoate degradation. Therefore, in vitro assays were set up in which the turnover of 3-chloro- and 4-chlorocatechol by total cell extracts of the *R. eutropha* JMP289 *tfd* derivatives was followed spectrophotometrically over time. Incubation of 3-chlorocatechol with cell extracts of *R. eutropha* JMP289 (pCBA59) or *R. eutropha* JMP134 led to the appearance of a peak at 260 nm in both strains which later disappeared (Fig. 5 A, D), although the spectra were not identical. These changes were indicative of complete disappearance of 3-chlorocatechol. Incubation of *R. eutropha* JMP289 (pCBA150) with 3-chlorocatechol produced a peak with a maximum at 260 nm within 4 min, which remained constant overnight (Fig. 5C). This

**Fig. 5 Turnover of 3-chlorocatechol by total cell extracts of *R. eutropha* JMP289 *tfd* derivatives, as observed from UV spectra recorded after different incubation times. The characteristic absorption maxima (*λ*~max~) of some possible intermediates are as follows:**

- 2-chloro-*cis*,*cis*-muconate, 267 nm (Dorn and Knackmuss 1978)
- 3-chloro-*cis*,*cis*-muconate, 259 nm (Dorn and Knackmuss 1978)
- *trans*-diene lactone, 274 nm (Blasco et al. 1995)
- Protoanemonin, 260 nm (Blasco et al. 1995)
- Maleylacetate, 243 nm (Blasco et al. 1995)

**A** *R. eutropha* JMP289 with plasmid pCBA59, **B** pCBA93, **C** pCBA150, **D** *R. eutropha* JMP134 (pJP4), **E** pCBA129, and **F** pCBA220. Incubation times are indicated for each spectrum; *o/n* overnight incubation.

Fig.6A–H Product analysis of 3-chlorocatechol transformations by cell extracts of E. coli BL21 expressing the *tfdI* and *tfdII* genes individually analyzed by HPLC/MS. Shown are the reconstructed ion chromatograms for a molecular mass of 175 Da, except for A, where a molecular mass of 241 Da is displayed. 100% peak height is a relative value different for each spectrum and should not be compared between spectra. The peak at 4.8 min is caused by the HPLC/MS matrix. A 3-Chlorocatechol, B 2-chloro-*cis,cis*-mu- conate. C Incubation of 3-chlorocatechol with cell extracts of E. coliBL21 expressing TfdC (chromatogram *a*), TfdC plus TfdD (chromatogram *b*), TfdC, TfdD plus TfdE (*c*). D Incubation with TfdC<sub>II</sub> alone (*a*), TfdC<sub>II</sub> plus TfdD<sub>II</sub> (*b*), TfdC<sub>II</sub>, TfdD<sub>II</sub> plus TfdE<sub>II</sub> (*c*). E Incubation with TfdC<sub>II</sub> plus TfdD; F incubation with TfdC plus TfdD<sub>II</sub> (*a*), TfdC, TfdD<sub>II</sub> plus TfdE (*b*); G incubation with TfdC<sub>II</sub>, TfdD<sub>II</sub> plus TfdD (*a*), TfdC<sub>II</sub>, TfdD<sub>II</sub>, TfdD plus TfdE (*b*). H Incubation with TfdC, TfdD plus TfdE<sub>II</sub>
spectrum resembled strongly that of 2-chloro-cis, cis-mu-  
conate. Similarly, but at a slower rate, a peak appeared at  
260 nm during incubation of cell extracts of R. eutropha  
JMP 289 (pCBA93), R. eutropha JMP 289 (pCBA129),  
and R. eutropha JMP 289 (pCBA220) with 3-chlorocate-  
chol (Fig. 5B, E, and F, respectively). Although this peak  
disappeared slowly it was still visible in the spectra taken  
after overnight incubation. A compound with a maximum  
absorption wavelength at 260 nm was accumulated in  
R. eutropha JMP 289 (pCBA150), R. eutropha JMP 289  
(pCBA93), R. eutropha JMP 289 (pCBA129) and R. eu-  
tropha JMP 289 (pCBA220). We propose that this com-  
pound was 2-chloro-cis, cis-muconate, the only compound  
in the 3-chlorocatechol degradation pathway with a maxi-  
mum at 260 nm. All four strains expressed TfdD$_{II}$ as  
chloromuconate cycloisomerase, except for R. eutropha  
JMP 289 (pCBA220). Therefore, it seemed that TfdD$_{II}$  
was unable or very inefficient in catalyzing conversion of  
2-chloro-cis, cis-muconate. No such difference was ob-  
served between the R. eutropha JMP 289 tfd derivatives  
with respect to 4-chlorocatechol turnover (data not shown).

TfdD$_{II}$ is the bottleneck during conversion  
of 3-chlorocatechol

As shown above, the conversion of 3-chlorocatechol by  
all R. eutropha JMP 289 tfd derivatives except JMP 289  
(pCBA59) led to the accumulation of a compound with a  
UV spectrum resembling that of 2-chloro-cis, cis-mu-  
conate, but did not allow identification of this compound.  
We therefore relied on HPLC/UV/MS for step-by-step  
analysis of the products formed during degradation of  
3-chloro- and 4-chlorocatechol by cell extracts of E. coli  
BL21 expressing each of the tfd$_I$ and tfd$_{II}$ genes individu-  
ally. The standard 3-chlorocatechol was detected by  
HPLC as a peak eluting at 18.72 min (Fig. 6A). Strangely,  
the 2-chloro-cis, cis-muconate preparation resulted in two  
peaks (9.33 min and 15.67 min), both with a molecular  
mass of 175 Da (Fig. 6B) and an isotope pattern clearly in-  
dicating the presence of one chlorine atom (not shown).  
One of these two peaks may correspond to 2-chloro-  
cis, trans-muconate, which is formed spontaneously from  
2-chloro-cis, cis-muconate under acidic conditions (pH be-  
low 1) and elutes later from reversed-phase columns  
(Schmidt et al. 1980). After incubation of cell extracts of  
E. coli BL21 expressing either tfdC or tfdC$_{II}$ with 3-chloro-  
catechol, two peaks with a molecular mass of 175 Da  
were detected similar to those obtained by the standard  
2-chloro-cis, cis-muconate (Fig. 6C and 6D, chromato-  
grams a). Further addition of cell extracts of E. coli BL21  
expressing tfdD led in both cases to disappearance of the  
two peaks and appearance of a peak eluting around 6 min  
(Fig. 6C, chromatogram b, and Fig. 6E). Unfortunately,  
this peak could not be identified by HPLC due to unavail-  
ability of an appropriate standard. Its molecular mass of  
175 Da and its isotope pattern suggest that this peak cor-  
responded to chloromuconolactone or a combination of  
chloromuconolactone and dienelactone, both of which  

---

were not resolved under the conditions used for HPLC. On the other hand, after incubation of 3-chlorocatechol  
with cell extracts of E. coli BL21 expressing tfdC or tfdC$_{II}$  
and cell extracts of E. coli BL21 expressing tfdD$_{II}$, the two  
peaks of 2-chloro-cis, cis-muconate remained (Fig. 6D,  
chromatogram b and Fig. 6F, chromatogram a). When  
E. coli cell extract containing TfdD was added to this mix-  
ture, the two peaks indicative of 2-chloro-cis, cis-mu-  
conate disappeared and again only the peak at 6.18 min  
was detected (Fig. 6G, chromatogram a). Both dienelac-  
tone hydrolases, TfdE and TfdE$_{II}$, were able to catalyze a  
transformation leading to disappearance of this “6 min”  
peak (Fig. 6C, chromatogram c, Fig. 6F, G, chromatograms  
b, and Fig. 6H). However, adding TfdE or TfdE$_{II}$ to the  
preparation with TfdC and TfdD$_{II}$ or TfdC$_{II}$ and TfdD$_{II}$,  
respectively, did not result in any change of the two peaks  
indicative of 2-chloro-cis, cis-muconate (Fig. 6F, chro-  
matogram b and Fig. 6D, chromatogram c). In contrast to  
3-chlorocatechol, no accumulation was observed for the  
transformation of 4-chlorocatechol by the Tfd$_I$ and Tfd$_{II}$  
enzymes (data not shown). These results confirmed the  
spectral observations with R. eutropha cell extracts, that  
TfdD$_{II}$ was not able to efficiently catalyze transformation of  
2-chloro-cis, cis-muconate, and confirmed our hypothesis that  
TfdD$_{II}$ was a bottleneck in 3-chlorocatechol degradation.

TfdD$_{II}$ is comparably shorter than TfdD

Interestingly, close inspection of both ORFs of $tfdD_{II}$ and  
$tfdD$, and comparison of the encoded peptides indicated that  
$tfdD_{II}$ has a relatively poor ribosome-binding site com-  
pared to $tfdD$ as well as an internal frame-shift (Fig. 2B).  
To determine if changing the start site and correcting the  
frame-shift would produce chloromuconate cycloiso-  
merases more active with 2-chloro-cis, cis-muconate, plas-  
mids pCBA161 (fragment B, with the more downstream  
located start codon of $tfdD_{II}$ and the extended reading  
frame) and pCBA168 (fragment C, with the most 5’ lo-  
cated start codon and the extended reading frame) were  
constructed and introduced into R. eutropha JMP 289.  
However, no differences were observed between R. eu-  
tropha JMP 289 (pCBA150) and R. eutropha harboring  
pCBA161 or pCBA168 with respect to growth on 5 mM  
3-chlorobenzoate and enzyme activity (data not shown).  
Similarly, overexpression of both TfdD$_{II}$ variants (frag-  
ments B and C) from the T7 promoter in E. coli BL21 did  
not result in increased chloromuconate cycloisomerase  
activity on 2-chloro-cis, cis-muconate (data not shown).  
Therefore, we concluded that the internal frame-shift in  
$tfdD_{II}$ or a more downstream-located start site could not  
be the reason for the observed specificity differences  
compared to $tfdD$.

---

Discussion

The finding that the second set of chlorocatechol-degrad-  
ing genes ($tfd_{II}$ genes) is transcribed in R. eutropha  
JMP134 during growth with 2,4-D (Leveau et al. 1999) and encodes functional chlorocatechol-degrading enzymes (Laemmlie et al. 2000; Pérez-Pantoja et al. 2000) demonstrated that two functional pathways for degradation of chlorophenols and chlorocatechols exist in *R. eutropha* JMP134. At this point the question arose as to what the specific nature and use of each of the *tfd* genes for growth on chlorinated substrates could be. Previous studies (Leveau et al. 1999; Laemmlie et al. 2000; Pérez-Pantoja et al. 2000) led us to believe that the *tfdI* and *tfdII* clusters are fully isofunctional, even though the sequence differences are substantial (Laemmlie et al. 2000). However, the results of this work indicate that this is not completely correct. As expected, expression of the *tfdI* cluster in *R. eutropha* JMP289 allows growth on 3-chlorobenzoate, whereas cells with the *tfdII* cluster cannot grow. These results differ from those obtained by Pérez-Pantoja et al. (2000), who described that both clusters allowed growth of *R. eutropha* on 3-chlorobenzoate, although it was also observed that expression of the *tfdI* cluster resulted in more efficient growth on 3-chlorobenzoate. Furthermore, they observed that plasmids with the *tfdII* cluster did not allow growth on 3-chlorobenzoate in *Pseudomonas putida*. The authors proposed that the TfdDII and/or TfdEII enzyme were responsible for the rate-limiting steps during degradation of 3-chlorobenzoate by the *tfdII* cluster, due to their low activity with 2-chloro-cis,cis-muconate and cis-dienelactone, respectively. Our results confirm the hypothesis that TfdDII is less efficient than TfdDI, or perhaps even completely unable to catalyze the transformation of 2-chloro-cis,cis-muconate, and is therefore a rate-limiting factor during degradation of 3-chlorobenzoate.

This conclusion was based on the substrate incubations with cell extracts of *E. coli* and subsequent HPLC/UV/MS analysis, which clearly demonstrated that 2-chloro-cis,cis-muconate is accumulated during the conversion of 3-chlorocatechol in the presence of TfdCII or TfdC and TfdDII. The two distinct peaks which were detected in the reactions corresponded to the two peaks obtained with our standard 2-chloro-cis,cis-muconate preparation. Since both compounds have a molecular mass of 175 Da as determined by mass spectroscopy and 2-chloro-cis,cis-muconate is known to spontaneously form 2-chloro-cis,trans-muconate under moderate acidic conditions (Schmidt et al. 1980), we assume that one peak corresponds to 2-chloro-cis,cis-muconate and the other to 2-chloro-cis,trans-muconate. No such accumulation was seen with 4-chlorocatechol and 3-chloro-cis,cis-muconate. This suggests that TfdDII is unable to dehalogenate 2-chloro-cis,cis-muconate to trans-dienelactone (Fig. 7), but can convert 3-chloro-cis,cis-muconate to cis-dienelactone. However, our analyses could not discriminate the pathway intermediates below the level of chloromuconates. Other chloromuconate cycloisomerases like TfdD (Schmidt et al. 1980; Kuhm et al. 1990) and ClcB of *P. putida* can dehalogenate both 2-chloro- and 3-chloro-cis,cis-muconate (Vollmer et al. 1994; Solyanikova et al. 1995). In contrast to chloromuconate cycloisomerases, muconate cycloisomerases, such as those of *Acinetobacter calcoaceticus* ADP1, *P. putida*

**Fig. 7** Proposed pathway for the metabolism of 3-chlorobenzoate by *R. eutropha* JMP134 and *R. eutropha* JMP289 *tfd* derivatives. The reactions are catalyzed by the following enzymes: benzoate dioxygenase and dihydrodiol dehydrogenase (reaction A), chlorocatechol 1,2-dioxygenase (*B*), chloromuconate cycloisomerase (*C*), dienelactone hydrolase (*D*) and maleylacetate reductase (*E*). Dashed arrows possible reactions catalyzed by muconolactone isomerase (*F*) (Prucha et al. 1996a) and muconate cycloisomerase (*G*) (Bruckmann et al. 1998; Vollmer et al. 1998), which are expressed in the presence of benzoate

PRS2000 and *Pseudomonas* sp. strain B13, convert 2-chloro-cis,cis-muconate to a mixture of (+)-2-chloro and (+)-5-chloromuconolactone (Vollmer et al. 1994) and 3-chloro-cis,cis-muconate to protoanemonin (Blasco et al. 1995), but cannot dehalogenate the chloromuconolactones. The chloromuconate cycloisomerase (ClcB) from *Rhodococcus erythropolis* 1CP, on the other hand, converts 2-chloro-cis,cis-muconate to (+)-5-chloromuconolactone only and 3-chloro- and 2,4-dichloro-cis,cis-muconate correctly to their corresponding dienelactones (Solyanikova et al. 1995). From this we conclude that TfdDII is similar in function to ClcB from *R. erythropolis* 1CP, although its sequence similarity is much higher to the group of normal muconate cycloisomerases (Solyanikova et al. 1995; Eulberg et al. 1998).

As a consequence of the inefficiency of TfdDII to catalyze the conversion of 2-chloro-cis,cis-muconate to

trans-dienelactone, different phenotypes with respect to growth on 3-chlorobenzoate were observed among the *R. eutropha* JMP289 *tfd* derivatives. The phenotypes could be divided into four categories: (1) no growth, (2) growth with up to 1 mM 3-chlorobenzoate, (3) growth with up to 3 mM 3-chlorobenzoate, and (4) growth with 5 mM 3-chlorobenzoate. We assume that the no-growth phenotype observed for *R. eutropha* JMP289 (pCBA93), expressing the *tfdII* cluster, can only be partially due to the inefficiency of TfdDII in catalyzing the conversion of chloromuconates, since strain *R. eutropha* JMP289 (pCBA150), expressing *tfdR-CDIIEF*, was able to grow with 3-chlorobenzoate, although *tfdD* was not present here. Furthermore, the better 3-chlorobenzoate phenotype of *R. eutropha* JMP289 (pCBA129), harboring *tfdR-DIICEIID*, compared to *R. eutropha* JMP289 (pCBA220), expressing *tfdR-CIIEIID*, indicated that expression of *tfdDII* had a positive contribution on growth. This could mean that also TfdEII is a limiting factor in *R. eutropha*, as was previously suggested by Pérez-Pantoja et al. (2000), since activities of dienelactone hydrolase measured in *R. eutropha* carrying *tfdII* plasmids were around our detection limit. Interestingly, also dienelactone hydrolase activity was clearly measurable from *tfdEII* in *E. coli* (Laemmlie et al. 2000), suggesting that this gene is very poorly expressed in its native configuration in *R. eutropha*.

We assume that: (1) the level of Tfd enzyme activity, and (2) the ratio between the individual Tfd enzyme activities are the major factors affecting growth on 3-chlorobenzoate. We could not obtain solid evidence that toxic compounds were formed in those strains expressing TfdDII, although our results suggest that perhaps toxicity occurs when 2-chloro-cis,cis-muconate is accumulated intracellularly. No evidence was found for accumulation of protoanemonin from 3-chlorobenzoate via 4-chlorocatechol and 3-chloro-cis,cis-muconate (Blasco et al. 1995; Kaulmann et al. 2001) in clones expressing TfdDII, which would have been detectable on HPLC chromatograms by its molecular mass. The inability to grow on 5 mM 3-chlorobenzoate could be overcome by lowering the initial 3-chlorobenzoate concentration. A similar observation was made previously for *R. eutropha* JMP134 (Pieper et al. 1993), in which accumulation of 2-chloro-cis,cis-muconate was overcome by adding 3-chlorobenzoate stepwise to a continuously growing *R. eutropha* JMP134 culture. Perhaps slowing down 2-chloro-cis,cis-muconate formation allows the cells to metabolize the chlorocatechols via 4-chlorocatechol before 2-chloro-cis,cis-muconate accumulates. Alternatively, there is some evidence for dechlorination of (+)-5-chloromuconolactone to cis-dienelactone, which is catalyzed by a muconolactone isomerase in benzoate grown cells of *R. eutropha* JMP134 (Prucha et al. 1996a, b). However, it is not known if muconolactone isomerase is expressed during growth on 3-chlorobenzoate.

In summary, we can conclude that there are no obvious enzymatic reasons as to the necessity for *R. eutropha* JMP134 (pJP4) to harbor two sets of chlorocatechol-degrading enzymes, at least during growth on 3-chlorobenzoate. It has been suggested previously that TfdFII might play an essential role, even in 3-chloro- and 4-chlorocatechol conversion (Pérez-Pantoja et al. 2000). Indeed, the maleylacetate reductase enzyme activity detected for TfdFII largely exceeds that of TfdF; however, this did not influence the observed growth behavior on 3-chlorobenzoate. The differences in substrate specificity, exemplified between TfdD and TfdDII, may indicate that the presence of both clusters is essential to allow metabolism of a broader variety of chlorinated compounds.

Acknowledgements We would like to thank R.P. Schaffner for excellent computer support. The work of C.M.L. was supported by grant 31–49222.96 from the Swiss National Science Foundation.

---

**References**

Bagdasarian M, Lurz R, Ruckert B, Franklin FC, Frey J, Timmis KN (1981) Specific-purpose plasmid cloning vectors. II. Broad host range, high copy number, RSF1010-derived vectors, and a host-vector system for gene cloning in *Pseudomonas*. Gene 16:237–247

Baumann B, Snozzi M, Zehnder AJB, van der Meer JR (1996) Dynamics of denitrification activity of *Paracoccus denitrificans* in continuous culture during aerobic-anaerobic changes. J Bacteriol 178:4367–4374

Benes V, Hostomsky Z, Arnold L, Paces V (1993) M13 and pUC vectors with new unique restriction sites for cloning. Gene 130:151–152

Blasco R, Wittich RM, Mallavarapu M, Timmis KN, Pieper DH (1995) From xenobiotic to antibiotic, formation of protoanemonin from 4-chlorocatechol by enzymes of the 3-oxoadipate pathway. J Biol Chem 270:29229–29235

Bradford MM (1976) A rapid and sensitive method for the quantification of microgram quantities of protein utilizing the principle of protein-dye binding. Anal Biochem 72:248–254

Bruckmann M, Blasco R, Timmis KN, Pieper DH (1998) Detoxification of protoanemonin by dienelactone hydrolase. J Bacteriol 180:400–402

Ditta G, Stanfield S, Corbin D, Helinski DR (1980) Broad host range DNA cloning system for Gram-negative bacteria: construction of a gene bank of Rhizobium meliloti. Proc Natl Acad Sci USA 77:7347–7351

Don RH, Pemberton JM (1981) Properties of six pesticide degradation plasmids isolated from *Alcaligenes paradoxus* and *Alcaligenes eutrophus*. J Bacteriol 145:681–686

Don RH, Weightman AJ, Knackmuss H-J, Timmis KN (1985) Transposon mutagenesis and cloning analysis of the pathways for degradation of 2,4-dichlorophenoxyacetic acid and 3-chlorobenzoate in *Alcaligenes eutrophus* JMP134(pJP4). J Bacteriol 161:85–90

Dorn E, Knackmuss H-J (1978) Chemical structure and biodegradability of halogenated aromatic compounds. Two catechol 1,2-dioxygenases from a 3-chlorobenzoate-grown pseudomonad. Biochem J 174:73–84

Eulberg D, Kourbatova EM, Golovleva LA, Schlömann M (1998) Evolutionary relationship between chlorocatechol catabolic enzymes from *Rhodococcus opacus* 1CP and their counterparts in proteobacteria: sequence divergence and functional convergence. J Bacteriol 180:1082–1094

Evans WC, Smith BSW, Fernley HN, Davies JI (1971) Bacterial metabolism of 2,4-dichlorophenoxyacetate. Biochem J 122:543–551

Fritz H, Reineke W, Schmidt E (1991) Toxicity of chlorobenzene on *Pseudomonas* sp. strain RHO1, a chlorobenzene-degrading strain. Biodegradation 2:165–170

Fukumori F, Hausinger RP (1993) *Alcaligenes eutrophus* JMP134 “2,4-dichlorophenoxyacetate monooxygenase” is an α-ketoglutarate-dependent dioxygenase. J Bacteriol 175:2083–2086

Gerhardt P, Murray RGE, Costilow RN, et al (eds) (1981) Manual of methods for general bacteriology. American Society for Microbiology, Washington, D.C.

Harker AR, Olsen RH, Seidler RJ (1989) Phenoxyacetic acid degradation by the 2,4-dichlorophenoxyacetic acid (TFD) pathway of plasmid pJP4: mapping and characterization of the TFD regulatory gene, *tfdR*. J Bacteriol 171:314–320

Kaphammer B, Kukor JJ, Olsen RH (1990) Regulation of *tfdCDEF* by *tfdR* of the 2,4-dichlorophenoxyacetic acid degradation plasmid pJP4. J Bacteriol 172:2280–2286

Kaulmann E, Kaschabek SR, Schlömann M (2001) Mechanism of chloride elimination from 3-chloro- and 2,4-dichloro-cis,cis-muconate: new insight obtained from analysis of muconate cycloisomerase variant *catB*-k169a. J Bacteriol 183:4551–4561

Kuhm AE, Schlömann M, Knackmuss H-J, Pieper DH (1990) Purification and characterization of dichloromuconate cycloisomerase from *Alcaligenes eutrophus* JMP134. Biochem J 266: 877–883

LaemmlI CM, Leveau JHJ, Zehnder AJB, van der Meer JR (2000) Characterization of a second *tfd* gene cluster for chlorophenol and chlorocatechol metabolism on plasmid pJP4 in *Ralstonia eutropha* JMP134(pJP4). J Bacteriol 182:4165–4172

Leveau JHJ, van der Meer JR (1996) The *tfdR* gene product can successfully take over the role of the insertion element-inactivated TfdT protein as a transcriptional activator of the *tfdCDEF* gene cluster, which encodes chlorocatechol degradation in *Ralstonia eutropha* JMP134(pJP4). J Bacteriol 178:6824–6832

Leveau JHJ, van der Meer JR (1997) Genetic characterization of insertion sequence ISJP4 on plasmid pJP4 from *Ralstonia eutropha* JMP134. Gene 202:103–114

Leveau JH, Zehnder AJ, van der Meer JR (1998) The *tfdK* gene product facilitates uptake of 2,4-dichlorophenoxyacetate by *Ralstonia eutropha* JMP134(pJP4). J Bacteriol 180:2237–2243

Leveau JHJ, König F, Füchslin H-P, Werlen C, van der Meer JR (1999) Dynamics of multigene expression during catabolic adaptation of *Ralstonia eutropha* JMP134 (pJP4) to the herbicide 2,4-dichlorophenoxyacetate. Mol Microbiol 33:396–406

Matrubutham U, Harker AR (1994) Analysis of duplicated gene sequences associated with TfdR and TfdS in *Alcaligenes eutrophus* JMP134. J Bacteriol 176:2348–2353

Pérez-Pantoja D, Guzman L, Manzano M, Pieper DH, Gonzalez B (2000) Role of *tfdC<sub>I</sub>D<sub>I</sub>E<sub>I</sub>F<sub>I</sub>* and *tfdD<sub>II</sub>C<sub>II</sub>E<sub>II</sub>F<sub>II</sub>* gene modules in catabolism of 3-chlorobenzoate by *Ralstonia eutropha* JMP134(pJP4). Appl Environ Microbiol 66:1602–1608

Perkins EJ, Gordon MP, Caceres O, Lurquin PF (1990) Organization and sequence analysis of the 2,4-dichlorophenol hydroxylase and dichlorocatechol oxidative operons of plasmid pJP4. J Bacteriol 172:2351–2359

Pieper DH, Knackmuss H-J, Timmis KN (1993) Accumulation of 2-chloromuconate during metabolism of 3-chlorobenzoate by *Alcaligenes eutrophus* JMP134. Appl Microbiol Biotechnol 39: 563–567

Prucha M, Petersheim A, Timmis KN, Pieper DH (1996a) Muconolactone isomerase of the 3-oxoadipate pathway catalyzes dechlorination of 5-chloro-substituted muconolactones. Eur J Biochem 237:350–356

Prucha M, Wray V, Pieper DH (1996b) Metabolism of 5-chloro-substituted muconolactones. Eur J Biochem 237:357–366

Ravatn R, Studer S, Zehnder AJB, van der Meer JR (1998) IntB13, an unusual site-specific recombinase of the bacteriophage P4 integrase family, is responsible for chromosomal insertion of the 105-kilobase *clc* element of *Pseudomonas sp.* strain B13. J Bacteriol 180:5505–5514

Sambrook J, Frisch EF, Maniatis T (1989) Molecular cloning: a laboratory manual, 2nd edn. Cold Spring Harbor Laboratory, Cold Spring Harbor, New York

Schmidt E, Remberg G, Knackmuss H-J (1980) Chemical structure and biodegradability of halogenated aromatic compounds. Halogenated muconic acids as intermediates. Biochem J 192: 331–337

Schoepfer R (1993) The pRSET family of T7 promoter expression vectors for *Escherichia coli*. Gene 124:83–85

Solyanikova IP, Maltseva OV, Vollmer MD, Golovleva LA, Schlömann M (1995) Characterization of muconate and chloromuconate cycloisomerase from *Rhodococcus erythropolis* 1CP: indications for functionally convergent evolution among bacterial cycloisomerases. J Bacteriol 177:2821–2826

Streber WR, Timmis KN, Zenk MH (1987) Analysis, cloning, and high-level expression of 2,4-dichlorophenoxyacetate monooxygenase gene *tfdA* of *Alcaligenes eutrophus*. J Bacteriol 169: 2950–2955

Studier FW, Moffatt BA (1986) Use of bacteriophage T7 RNA polymerase to direct selective high-level expression of cloned genes. J Mol Biol 189:113–120

Vollmer MD, Fischer P, Knackmuss HJ, Schlömann M (1994) Inability of muconate cycloisomerases to cause dehalogenation during conversion of 2-chloro-cis,cis-muconate. J Bacteriol 176: 4366–4375

Vollmer MD, Hoier H, Hecht HJ, Schell U, Gröning J, Goldman A, Schlömann M (1998) Substrate specificity of and product formation by muconate cycloisomerases: an analysis of wild-type enzymes and engineered variants. Appl Environ Microbiol 64:3290–3299

Yanisch-Perron C, Vieira J, Messing J (1985) Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. Gene 33:103–119

You I-S, Ghosal D (1995) Genetic and molecular analysis of a regulatory region of the herbicide 2,4-dichlorophenoxyacetate catabolic plasmid pJP4. Mol Microbiol 16:321–331
