
Plant Cell Physiol. 43(6): 675–683 (2002)
JSPP © 2002

# The APRR3 Component of the Clock-Associated APRR1/TOC1 Quintet is Phosphorylated by a Novel Protein Kinase Belonging to the WNK Family, the Gene for which is also Transcribed Rhythmically in Arabidopsis thaliana

Masaya Murakami-Kojima, Norihito Nakamichi, Takafumi Yamashino and Takeshi Mizuno¹  
Laboratory of Molecular Microbiology, School of Agriculture, Nagoya University, Chikusa-ku, Nagoya, 464-8601 Japan

In higher plants, clock-controlled circadian rhythms are a longstanding issue in physiology, and a newly emerging paradigm of molecular biology. In the model higher plant *Arabidopsis thaliana*, several genes have been proposed to encode potential clock-associated components, including a member (APRR1/TOC1) of the pseudo-response regulator family. We previously showed that transcripts of the APRR1/TOC1 family start accumulating after dawn rhythmically and sequentially at approximately 2h intervals in the order of APRR9 → APRR7 → APRR5 → APRR3 → APRR1/TOC1. This and other results led us to propose that this APRR1/TOC1 quintet might play coordinate roles in the mechanism underlying circadian rhythms in higher plants. To gain further insight as to such an idea, we here attempt to identify proteins that interact with one of the quintet members, APRR3. The identified component is a novel protein kinase, named WNK1, which is considerably similar to, but clearly distinct from, mitogen-activated protein kinases (MAPKs). It was found that APRR3 is a substrate of this novel protein kinase, the gene for which also shows a rhythmic transcription profile that is well coincident with the APRR3 rhythm. These findings give new insight into the mechanisms underlying the circadian rhythm in *A. thaliana*, providing a molecular link between the putative clock component, APRR3, and WNK1, a novel protein kinase that might be implicated as a signal transducer.

Keywords: *Arabidopsis* — Circadian rhythm — Clock component — Protein kinase.

---

## Introduction

In the model higher plant *Arabidopsis thaliana*, biological and genetic studies have begun to shed light on the molecular bases of circadian clocks in higher plants (for recent reviews see Barak et al. 2000, Murtas and Millar 2000, Samach and Coupland 2000, Carre 2001). In fact, a number of circadian-regulated genes have been identified through extensive analyses with *Arabidopsis* DNA microarrays (Harmer et al. 2000, Schaffer et al. 2001). In general, a biological clock presumably consists of input pathways, central oscillators, and output pathways, of which the central oscillators are thought to autonomously generate rhythms through transcription/translation and positive/negative feedback loops (for a review, see Dunlap 1999). Molecular studies on such central oscillators in higher plants have just begun (see the reviews cited above).

With regard to such *Arabidopsis* circadian oscillators, several genes have been proposed to encode potential clock-associated components, although none of them has yet been fully established to be a bona fide clock component. Best characterized are the CIRCADIAN CLOCK ASSOCIATED 1 (CCA1) and LATE ELONGATED HYPOCOTYL (LHY) genes, which encode putative Myb-related transcription factors similar to each other (Wang and Tobin 1998, Schaffer et al. 1998, Green and Tobin 1999). Constitutive expression of *CCA1* in plants stops overt rhythmicity, including the expression of genes that are under the control of the circadian rhythm. Another gene, TIMING OF CAB1 (TOC1), is also a putative clock component, because a semi-dominant (*toc1-1*) mutant exhibits shortened periods of certain rhythmic gene-markers, and also of the leaf movement and stomatal conductance rhythms (Somers et al. 1998). The *TOC1* gene was recently found to encode a nuclear protein with a motif similar to that found in the CONSTANS family of transcription factors (Strayer et al. 2000). Furthermore, recent results suggested that there are intimate links between these two types of circadian-associated components (*CCA1/LHY* and *APRR1/TOC1*) (Alabadi et al. 2001, Makino et al. 2002, Matsushika et al. 2002). In any case, molecular mechanisms underlying biological clocks are one of the major and current problems in higher plants.

Recent reverse-genomic studies have provided further insights into such *Arabidopsis* circadian clocks. In this regard, we have been characterizing a novel family of genes encoding *Arabidopsis* pseudo-response regulators (designated as APRR1, APRR3, APRR5, APRR7 and APRR9) (Makino et al. 2000, Matsushika et al. 2000, Makino et al. 2001). Importantly, APRR1 turned out to be identical to TOC1 (Matsushika et al. 2000). Similarly to TOC1, each of these APRR proteins contains not only a CONSTANS-motif at its C-terminal end, but also an N-terminal domain similar to the receivers of response regulators that are involved in widespread signal transduction systems (so-called two-component systems). More interestingly, not only the *APRR1/TOC1* gene, but also other *APRR* genes are all subjected to circadian rhythms in a coordinate

¹ Corresponding author: E-mail, tmizuno@agr.nagoya-u.ac.jp; Fax, +81-52-789-4091.

Circadian rhythms in *Arabidopsis*

**Fig. 1** APRR3 associates physically with WNK1. (A) By employing APRR3 as bait, yeast two-hybrid assays were carried out, as schematically illustrated in the upper part. The resulting prey was found to be an *Arabidopsis* hypothetical protein, designated hereafter as WNK1, as also illustrated. As shown in the lower part, the experimental evidence for such a two-hybrid interaction was represented by histidine-autotrophy and β-galactosidase activity. (B) In vitro pull-down assays were carried out with the APRR3 and GST-WNK1-C polypeptides, both of which were expressed in *E. coli*. Note that the WNK1-C polypeptide corresponds to the region carried on clone-45 (see panel A). Glutathione-Sephrose beads carrying GST-WNK-C or GST were respectively mixed with *E. coli* cellular proteins containing APRR3, as schematically shown. The beads were recovered by centrifugation and then washed extensively. The resulting precipitates were analyzed by SDS-PAGE, followed by immunoblotting with anti-APRR3 antiserum.

fashion at the transcription level. Each transcript of *APRR* starts accumulating after dawn rhythmically and sequentially at approximately 2 h intervals in the order *APRR9 → APRR7 → APRR5 → APRR3 → APRR1/TOC1* in such a manner that the wave of the *APRR9*-transcript is first seen immediately after dawn (Matsushika et al. 2000, Makino et al. 2001). Based on these findings, we previously speculated that such sequential and rhythmic events, termed ‘circadian waves of the *APRR1/TOC1* quintet’, might be the molecular basis of a biological clock.

To gain further insight into such circadian-related properties of APRRs, we attempted to identify proteins that physically associate with one of the *APRR1/TOC1* quintet, namely APRR3. We here report that the component thus identified is a novel protein kinase (named WNK1), demonstrating that the APRR3 component is a substrate of this kinase, the gene for which is also under the control of circadian rhythm.

---

### Results

#### APRR3 physically associates with a novel *Arabidopsis* protein

To gain an insight into the molecular function of APRR3, we carried out yeast two-hybrid screening in a search for *Arabidopsis* proteins that interact with APRR3. Among \(10^7\) yeast transformants thus screened, after excluding several false-positive clones, two positive ones were obtained (named clone-45 and cloned-62) (Fig. 1A). Analyses involving restriction endonuclease digestion and sequencing of these cloned cDNAs revealed that they were derived from the same *Arabidopsis* gene, which had been registered in the current GenBank database (accession no. AC009465_16, chromosome III). In the database, the gene product is hypothesized to consist of 700 amino acids (MIPS protein code, At3g04910). Accordingly, clone-45 contained a coding region corresponding to an N-terminally truncated polypeptide of 409 amino acids, whereas clone-62 contained a shorter coding sequence corresponding to a C-terminal portion of 287 amino acids. Thus, APRR3 might be able to interact with this particular gene product, which is

tentatively designated hereafter as *A. thaliana* WNK1 (the reason for this naming will be rationalized later). We isolated the entire cDNA of this particular gene, and then confirmed that it does indeed encode the protein of 700 amino acids, as has been noted in the database. These results of two-hybrid screening for APRR3 are schematically explained in Fig. 1A, along with the two-hybrid assay data (i.e. histidine-autotrophy and β-galactosidase activity).

To verify the above notion, an in vitro protein-protein interaction assay was carried out (Fig. 1B). In *E. coli*, the entire amino acid sequence of APRR3 was overexpressed, and then the product was detected with an antiserum that specifically reacts with APRR3. On the other hand, the C-terminal portion of WNK1 (from Lys-241 to the C-terminal Asn-700), fused to glutathione S-transferase (GST), was also overexpressed in *E. coli*. This GST-WNK1-C polypeptide was purified with glutathione-Sepharose. With this immobilized probe, an in vitro pull-down assay was carried out with the *E. coli* cellular proteins containing the APRR3 polypeptide in a soluble form. The APRR3 polypeptide was co-precipitated with the GST-WNK1-C polypeptide (Fig. 1B). The in vitro results supported the view that APRR3 physically associates with WNK1.

### WNK1 is a novel type of serine/threonine kinase

Then, an extensive homology search was performed for the amino acid sequence of WNK1 (Fig. 2). The results revealed that WNK1 contains a common eukaryotic serine/threonine (Ser/Thr) kinase domain in its N-terminal portion (Fig. 2A). Inspection of the recently completed *Arabidopsis* genome sequence showed that this higher plant has a number of protein kinases, which include those belonging to the MAPK family (The *Arabidopsis* Genome Initiative 2000; see also PlantsP, http://plantsp.sdsc.edu/). The kinase domain of WNK1 is significantly similar to those of MAPKs (Fig. 2A). When the amino acid sequence of the WNK1 kinase domain (about 400 amino acids) was aligned with that of the *Arabidopsis* leucine-rich transmembrane kinase 1 (non-MAPK kinase, MIPS protein code, At1g53730), 9.6% amino acids were identical between them, whereas when a similar comparison was done between WNK1 and the *Arabidopsis* mitogen-activated protein kinase 3 (MAPK3, MIPS protein code, At3g45640), 18.2% amino acids were identical between them. The MAPK family members are further classified into three sub-types according to their biochemical functions, such as MAP-kinases (MAPKs),

MAP-kinase kinases (MAPKKs), and MAP-kinase kinases (MAPKKKs) (Fig. 2B) (for an overview as to plant MAPKs, see PlantsP, http://plantsp.sdsc.edu/). As shown in Fig. 2A, the kinase domain of WNK1 is highly similar to those of the CTR1/EDR1 (Raf)-family of MAPKKKs (see also the phylogenetic tree in Fig. 2B). However, we needed to classify WNK1 into a novel family of Ser/Thr kinases, because of the reason given below.

The amino acid sequence of the WNK1 kinase domain is similar to those of MAPKs, but it is atypical. It is known that the kinase domains of common MAPKs consist of certain sub-domains. Among them, sub-domain II is crucial as a nucleotide (or ATP) recognition site, and it contains an invariant amino acid residue, lysine (K) (see the highlighted amino acid sequence of AVK in Fig. 2A). This K is missing in WNK1, which instead has an asparagine residue (N). However, this unusual feature is not unique to the *Arabidopsis* WNK1 protein kinase, because this particular feature has also been observed for members of the recently reported WNK-family of mammalian protein kinases (Xu et al. 2000, Wilson et al. 2001). Such mammalian protein kinases were recently discovered and named WNKs [with no lysine (K)] (Xu et al. 2000). As shown in Fig. 2A, the amino acid sequences of the plant WNK1 and human WNK3 kinase domains can be well aligned with each other, and both lack the critical K residue at the corresponding position in sub-domain II. This is the reason why we designated the novel APRR3-associated protein kinase as WNK1. Although it has previously been pointed out that the WNK family members occur in not only mammals, but also higher plants (Verissimo and Jordan 2001), this is the first experimental detection of the occurrence of this particular class of protein kinases for *Arabidopsis*, to our knowledge.

### WNK1 catalyzes protein phosphorylation

The fact that WNK1 exhibits a similarity in its amino acid sequence to certain Ser/Thr kinases does not necessarily prove that it acts as a protein kinase. We first needed to address this issue. It was then suggested that WNK1 acts as a protein kinase (Fig. 3). A GST-WNK1 fusion polypeptide containing the entire WNK1 amino acid sequence was expressed in *E. coli*, and then isolated with a glutathione-Sepharose column (Fig. 3A). When the isolated samples were analyzed by sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE), they contained many protein bands, among which

Fig. 2 WNK1 contains a protein kinase domain that is highly similar to those of MAPKs. (A) The amino acid sequence of the WNK1 kinase domain (see upper illustration) was aligned with a set of *Arabidopsis* protein kinases, each of which belongs to the MAPKKK-family, i.e. ANP2, AtMEKK1, and EDR1 (GenBank accession nos. AB000798, D50465 and AF305913, respectively). Identical amino acids are shaded. This WNK1 sequence is also aligned with that of the recently discovered human protein kinase named HsWNK3 (accession no. AJ409088). In the human sequence, identical amino acids are highlighted. Note that these two kinases lack the critical lysine (K) residue in their sub-domain II, as indicated (see the highlighted amino acid sequence, AVK). (B) The amino acid sequence of WNA1 was analyzed by the neighbor-joining method to examine the phylogenetic relationship (tree) among the members of the *Arabidopsis* MAPK-related protein kinase family. To construct this phylogenetic tree, the amino acid sequences that correspond to the kinase domains were used. The branch lengths are proportional to the sequence divergence. For more detailed information, such as sequence references and criteria for classification of MAPK, MAPKK, and MAPKKK, see the comprehensive and recent review for plant MAP kinases by Tena et al. (2001) (also see PlantsP, http://plantsp.sdsc.edu/).

A

Ser/Thr kinase domain

AtWNK1

sub-domain II

ANP2    WRKGQLIGRGAFGTVYMGMNLDSGELLAVKQVLLTSNCASKEKTQAHIQELEEVKLLEN
AtMEKK1 WQKGOLLGRGSFGSVYEGISGD-GDFFAVKEVSILD---QGSQAQECIQTEGEIKLESQ
EDR1    LVIAERIGLGSYGEVYHADWHG--TEYAVKKFLDDQ----FSGAALAERFRSEVRIMRR
AtWNK1  GRYNEVLGKGASKTYRAFDEYEGLAWNQVKLYD----FLQSPEDLERLYCEIHLLKT
HsWNK3  --FDIELGRGAFKTVYKGLDTETWVEVAWC--ELQD----RKLTKAEQQRFKEEAEMLKQ

ANP2    LSHPNIVRYLGTV--REDE--TLNILLEFVPGGSISSLEKFGAFP-ESVVRTYTNOLELLGI
AtMEKK1 IQHONTVRYRGA--KIDGS--NIYTFLEVTQGSLKELYQRYQLR--DSVVSLYTRQILDG-
EDR1    LRHPNVVFFLGAY--TRPP--NLSIYTEFLPRGSFYRIJHRPKSHIDERRRIKMALDVAMGM
AtWNK1  LKHKNIMKFYTSW--VQTANRNINFVTELFTSGLRQYRLRHKRVN-IRAMKHWCRQIERGL
HsWNK3  LQHPNIVRFYDSWE SILKGKKCIVLVTELMYSGTLKTYLKRFKVMK-PKVLRSWCRQILKGL

ANP2    EYLHNHA--IMHRDIKGANILVD-NQGCIKLADFQASKQVAELATISGAKSMKGTPYHMA
AtMEKK1 KYLHDKG--FIHRDIKGANILVD-ANGAVKLADF-----LAKVSKFNDIKSCKGTPFWMA
EDR1    NCLHTSTPTIVHRDLKTPNLLVD-NNWNVKVGDGF--LSRLKHNTFLSSKSTAGTPEWMA
AtWNK1  HYLIHSDPPVIHRDLKCDFNVNGNQGEVKIGDLG----LAAILRKSHAACVGTPPEFMA
HsWNK3  QFLHTRTPPIHRDLKCDNIFITGPTGSVKIGDLG----LATLMRTSFAKSVIGTPEFMA

ANP2    PEVILQT---GHSFSADIWSVGC TVIEMTGKAPWSQQYKEIAAIFHIGTTKSHPPIPDN
AtMEKK1 PEVINRKDSDGYGSPADIWSLGCTVLEMC TGQIPYS-DLEPVOALFRIG-RGTLPEVPDT
EDR1    PEVLRNE--PSN-EKCDVYSFGVILWELATLRLPWR-GMNPMQVVGAVGFQNRRLEIPKE
AtWNK1  PEMYEEA---YN-ELVDIYSEGMCIEMVTFDYFSECTHPAQIYKKVMSGKKPDALYKV
HsWNK3  PEMYEEH---YD-ESVDVYAFGMCLEMATSEYPYSECQNAAQIYRKVTSGIKPASFNKV

ANP2    ISSDANDFLLEKCL-QQEPNLRFTASELLKVKWST
AtMEKK1 ISLDARLFTILKCL-KVNPEERPTAAELLNHPFVR
EDR1    LDPVVGRILLECW-QTDPNLRPSFAQLTEVLKPL
AtWNK1  KDPEVKCFIEKCL-ATVS-LRVSARELLDDPFLR
HsWNK3  TDPEVKEIIEGGIRQNKSERLSIRDLLNHAF


B



**Fig. 2**

major bands seemed to correspond to degraded GST-moieties. However, a large (or slow migrating) discrete band with an apparent molecular mass of about 110 kDa was detected, the value of which was in good agreement with that calculated for GST-WNK1. A polyclonal antiserum was raised against the purified GST-WNK-C polypeptide lacking the putative kinase domain (see Fig. 1B). The band in question indeed cross-reacted with this antiserum on an immunoblotting assay, indicating that the band is the GST-WNK1 polypeptide (data not shown). Other intermediate bands appear to be degradation

products of GST-WNK1. Similarly, a GST-WNK1-KD polypeptide containing only the kinase domain was also isolated (Fig. 3B). Together with these, purified GST and MBP (myelin basic protein from bovine brain) polypeptides were also used for the following in vitro phosphorylation assays. First, GST-WNK1 alone was incubated with [γ-³²P]ATP in an appropriate buffer

A

![Image A]

containing MgCl₂ and MnCl₂. Samples were separated by means of SDS-PAGE, followed by autoradiography (Fig. 3B). The GST-WNK1 polypeptide acquired a radioactive phosphoryl group, suggesting that WNK1 underwent autophosphorylation. A more important observation was that when this sample was incubated with MBP in the presence of ATP, the conventional substrate for eukaryotic protein kinases was efficiently radiolabeled (Fig. 3B). These results, together with those for appropriate references, were compatible with the view that WNK1 acts as a protein kinase that can phosphorylate its own polypeptide as well as another common substrate. This is consistent with the recent notion that human WNKs act as Ser/Thr kinases (Xu et al. 2000).

APRR3 is a specific substrate of WNK1

It was then tempting to envisage that APRR3 might be a substrate of WNK1. This was indeed the case (Fig. 3C). The entire APRR3 polypeptide containing a histidine-tag was highly purified with E. coli (Fig. 3A). Using this purified APRR3 polypeptide as a substrate, a series of in vitro phosphorylation assays was carried out with the partially purified WNK1 and WNK1-KD enzymes. Note that the latter enzyme encompasses only the kinase domain of WNK1. The in vitro results were best interpreted by assuming that APRR3 is a substrate of WNK1 (Fig. 3, lane 2), and it is phosphorylated as efficiently as MBP (lane 4). Interestingly, however, APRR3 was not phosphorylated by WNK1-KD (lanes 13 and 15), whereas this truncated form of enzyme phosphorylated MBP efficiently (lanes 14 and 15). Since these observations were critical, we reproducibly confirmed such in vitro events with a set of separately isolated samples (Fig. 3D). These results were then explained by assuming that, although APRR3 is a substrate of WNK1, its specificity is dependent on the presence of the C-terminal (or non-kinase) domain of WNK1. The results in Fig. 1 should be recalled here, which showed that the physical interaction between APRR3 and WNK1 is also dependent

B

![Image B]

C

![Image C]

D

![Image D]

Fig. 3 WNK1 exhibits protein kinase activity, and APRR3 is its substrate. (A) The indicated polypeptides were isolated and analyzed by SDS-PAGE, following by Coomassie brilliant blue staining, as described under Materials and Methods. The polypeptides used were GST-WNK1 (lane 2), GST-WNK-KD (lane 3), APRR3-Hisx6 (lane 4), GST (lane 5), and MBP (lane 6). Molecular markers were loaded on lane 1. (B) These polypeptides were mixed in a phosphorylation buffer in the indicated combinations. The protein amounts used were; APRR3, 0.20 μg; MBP, 0.25 μg. They were subjected to in vitro phosphorylation assays in the presence of [γ-³²P]ATP at 22°C for 30 min. The reaction mixtures were analyzed by SDS-PAGE, followed by autoradiography. (C) Further in vitro phosphorylation assays were carried out under essentially the same conditions as in (B). The asterisks in B and C indicate a radiolabeled unknown band, intensity of which varied from one experiment to another. (D) To reproducibly confirm the above results in C, the GST-WNK1 and APRR3 polypeptides were newly (or separately) prepared. Then, essentially the same experiments as in C were carried out as indicated.

$\beta$-Galactosidase Activity (Miller's Unit)

| Bait | Prey |
|------|------|
| Vector |  |
| APRR1-R |  |
| APRR3-R |  |
| APRR3 |  |
| APRR5 |  |
| APRR7 |  |
| APRR9 |  |

Vector WNK1

Interaction of WNK1 with other members of the APRR1/TOC1 quintet

As mentioned above, APRR3 is one of the five members of the APRR1/TOC1 quintet, whose structural designs are very similar to each other (Matsushika et al. 2000). In this context, further two-hybrid analyses showed that WNK1 interacts with APRR3 through the pseudo-receiver domain of APRR3, as shown in Fig. 4. The bait designated as APRR3-R, containing only the receiver domain (Met-1 to Thr-209), showed the ability to interact with the WNK1 prey. It should be noted that the five members of the APRR1/TOC1 quintet all contain a highly homologous pseudo-receiver domain. This raised the question as to whether or not other APRRs interact with WNK1. To address this issue, extensive two-hybrid analyses were carried out with a set of appropriate bait vectors for APRR1-R, APRR5, APRR7, and APRR9, respectively. While the APRR1-R bait contains only the pseudo-receiver domain of APRR1 (Met-1 to Leu-146), other APRR baits otherwise contain each corresponding entire amino acid sequence. The results of such two-hybrid analyses suggested that WNK1 is capable of interacting with APRR5 as well as APRR3, but not with APRR1-R, APRR7, or APRR9 (Fig. 4). These data cannot be evaluated (or compared with each other) quantitatively, because the expression levels of each bait protein in yeast cells were not determined quantitatively. Nevertheless, the results suggested that WNK1 does not necessarily interact with all other members of the APRR1/TOC1 quintet. Rather, WNK1 appears to preferen-

Fig. 4 Yeast two-hybrid assays for examination of interaction between WNK1 and the APRR1/TOC1 quintet. The interaction between WNK1 and the APRR1/TOC1 quintet members was carried out with the indicated bait-prey combinations, by measuring $\beta$-galactosidase activities.

on the C-terminal domain of WNK1. We thus propose that APRR3 is a specific substrate of WNK1 in the sense that its phosphorylation appears to occur through the specific and physical interaction between them.

![Graph](https://i.imgur.com/yourimageurl.png)

APRR3  
WNK1  
UBQ10  
(APRR3)  
UBQ10  
(WNK1)

Fig. 5 The gene for $WNK1$ is under the control of free-running circadian rhythm in a similar manner to the $APRR3$ gene. To examine the circadian rhythms of both the $WNK1$ and $APRR3$ gene, wild-type plants were grown under 12 h light/12 h dark (LD) conditions for 20 d. They were further grown for 72 h under continuous light (LL) conditions, as schematically shown (shaded bars indicate the subjective night). RNA samples were prepared from leaves at 3 h intervals. They were analyzed by Northern hybridization with each specific probe, as indicated ($WNK1$, $APRR3$, and $UBQ10$). Note that $UBQ10$ was analyzed as an internal and loading reference in these experiments. The intensities of hybridized bands (lower panels) were then each quantified (by normalization as to the $UBQ10$-value). These quantitative data are shown schematically as the relative amounts of transcripts (upper panel), the maximum levels of $APRR3$ and $WNK1$ being taken as 10 arbitrarily, in order to clarify the profiles.

Circadian rhythms in Arabidopsis

tially interact with two of the five members (APRR3 and APRR5). We do not know at present whether or not APRR5 is an alternative substrate of the WNK1 kinase.

The transcription of WNK1 is under the control of free-running circadian rhythm

Although we provided evidence that APRR3 is a substrate of WNK1, the critical question was whether or not this in vitro finding has any in vivo relevance as to the circadian rhythm. In this context, one of the most crucial features of APRR3 is that the transcription profile of APRR3 is under the control of free-running circadian rhythm, as reproducibly shown in Fig. 5. The expression profile of APRR3 showed a robust rhythm when RNA samples were prepared from wild-type plants grown under continuous light (LL) conditions. The same set of RNA samples was analyzed by Northern hybridization with a probe specific for the WNK1 coding sequence. It was found that the transcription of WNK1 is also under the control of circadian rhythm, although the oscillation amplitude was less robust than that of the APRR3 rhythm (Fig. 5). More importantly, the phase of the WNK1 rhythm was well coincident with that of APRR3, both of them peaking at the subjective evening.

From the results of this study, we showed that the APRR3 component of the clock-associated APRR1/TOC1 quintet is a substrate of the novel WNK1 protein kinase, the gene for which shows a rhythmic transcription profile that is well coincident with the APRR3 rhythm.

Discussion

The significance and implication of this study is twofold. First, we identified a component that might act as a signal transducer in concert with the APRR1/TOC quintet in the mechanism underlying the circadian rhythm. Interestingly, the identified component (WNK1) is a protein kinase that phosphorylates the putative clock-associated protein, APRR3. Second, we identified WNK1 as a novel protein kinase that has not previously been characterized in higher plants. Although the inferred WNK1 protein is classified simply as a member of the MAPK family in the current Arabidopsis database, it should be considered as a novel type of Ser/Thr kinase, homologs of which have recently been found in mammals. This suggests that such a novel type of protein kinase occurs widely in both higher vertebrates and plants.

As mentioned above, so far, there are two distinct types of candidate components of the Arabidopsis clock oscillators. They are the CCA1/LHY and APRR1/TOC1 families of proteins. With regard to their biochemical natures, the members of the CCA1/LHY family most likely function as transcription factors, because each contains a Myb-related DNA binding domain. However, we do not know whether or not the members of the APRR1/TOC1 family act as DNA-binding transcription factors, although the protein products have been demonstrated

to be in the nucleus (Strayer et al. 2000, Makino et al. 2000). In the hope of clarifying their biochemical natures, we are currently conducting extensive yeast two-hybrid screening for each of the members of the APRR1/TOC1 quintet. We already succeeded in identifying an APRR1/TOC1-associated protein, named PIL1 (PIF3-LIKE1) (Makino et al. 2002). This protein is significantly similar to the well-characterized Arabidopsis bHLH protein named PIF3 (PHYTOCHROME INTERACTING FACTOR 3) (Ni et al. 1998, Martinez-Garcia et al. 2000). PIF3 was identified as the protein that specifically associates with the red light-activated PHYB photoreceptor, and thus it was proposed to function as the downstream component of a PHYB-dependent light-signaling pathway. Interestingly, the results of yeast two-hybrid assays suggested that PIL1 and PIF3 together form a heterodimer (our unpublished results). However, the second member, APRR3, showed no ability to interact with PIL1 (data not shown). Instead, it was found in this study that APRR3, but not APRR1, interacts with the WNK1 kinase in such a manner that this novel Ser/Thr kinase phosphorylates APRR3. This event seems to be relevant to the circadian rhythm, because the transcription of WNK1 oscillates in a very similar manner to that of the APRR3 rhythm.

The findings of this study provided us with new insight into the mechanisms underlying the circadian rhythm in *Arabidopsis thaliana*, suggesting a molecular link between the putative clock component, APRR3, and WNK1, a novel protein kinase. In general, as well as identifying putative oscillator components, it is also important to know how clock components might be regulated. In *Synechococcus*, *Drosophila*, *Neurospora* and hamsters, it is known that phosphorylation of oscillator proteins is a crucial event for clock function. In plants too, such phosphorylation appears to play a regulatory role in the clock mechanism. In this respect, it would be worth mentioning that a regulatory subunit (named CKB3) of casein kinase II (CK2) has been identified as a CCA1-interacting protein, and CK2 can phosphorylate CCA1 (and LHY) in vitro (Sugano et al. 1998, Sugano et al. 1999). In vivo, constitutive expression of CKB3 shortens the period of expression of CCA1 and LHY. The novel WNK1 protein kinase might also be implicated as a component of a signal transduction cascade, which modulates the biological activity (or stability/cellular location) of APRR3 (or APRR5) through phosphorylation at a proper timing. It is thus tempting to speculate that phosphorylation is a crucial and regulatory event underlying the circadian waves of APRR1/TOC1 quintet. In any case, clarification of in vivo relevance of the APRR3 phosphorylation by WNK1 must await further experiment with special reference to the circadian rhythm.

Inspection of the Arabidopsis genome sequence revealed that this plant has many MAPK-related proteins kinases, in a considerably higher number than any other eukaryotes (see PlantsP, http://plantsp.sdsc.edu/). With a few notable exceptions (Innes 2001, Asai et al. 2002), nevertheless, little is known about the biological functions of MAPK-related protein

kinases. Classification of MAPK-related protein kinases and characterization of the underlying signal transduction networks remain as enormous future challenges for plant biologists. In this context, the WNK family of protein kinases in *Arabidopsis* is intriguing, because this type of protein kinase is also common in both mammals and plants, as demonstrated in this study. Also, *Arabidopsis* has more protein kinases homologous to WNK1 (eight or more, data not shown). Among the WNK family members, however, their amino acid sequences of the C-terminal non-kinase domain are highly variable. Our preliminary results of two-hybrid analyses suggested that other WNK1 family members do not interact with APRR3 (our unpublished data). In any case, such members of the WNK-family of protein kinases might play as yet unknown roles in certain signal transduction pathways in plants. This is reminiscent of the intriguing case of mammalian WNK kinases that are somehow implicated as genetic determinants of a type of human hypertension, as recently reported (Wilson et al. 2001). Based on the findings of this study, it would thus be of interest to examine the WNK family of protein kinases, not only with special reference to the sophisticated mechanism underlying the biological clock, but also in terms of general components of intracellular signal transduction mechanisms in higher plants.

### Materials and Methods

#### Plant growth conditions
*Arabidopsis thaliana* (Columbia ecotype) was used as wild-type plants. Seeds were imbibed and cold-treated at 4°C for 2d before germination and growth at 22°C. Plants were grown in a chamber under fluorescent light conditions (150–200 μmol m⁻²s⁻¹) on soil, or on agar-plates containing MS salts and 2% sucrose, as described previously (Makino et al. 2000). To examine free-running circadian rhythms, plants were grown for 20d under 12h light/12h dark, and then transferred to continuous light. RNA samples were prepared from these plants at appropriate intervals, and then Northern hybridization analyses were carried out, as described below. The details were given previously (Makino et al. 2002).

#### Yeast two-hybrid screening
A kit for two-hybrid analysis (MATCHMAKER™) was obtained (Clontech). This kit contained all tools essential for two-hybrid screening, including vectors: pGBT9 providing the GAL4 DNA-binding domain, and pGAD424 providing the GAL4 activation domain. Two-hybrid screening was carried out essentially as recommended in the manual. An *Arabidopsis* cDNA-expression library was obtained from Clontech (*A. thaliana* MATCHMAKER cDNA Library). The following plasmids were constructed in this study from pGBT9 for use as bait, pGBT9-APRR1-R, pGBT9-APRR3, pGBT9-APRR3-R pGBT9-APRR5, pGBT9-APRR7, and pGBT9-APRR9. In each, the corresponding coding sequence was appropriately connected to the GAL4 DNA-binding domain of pGBT9.

#### Preparation of polypeptides
The following polypeptides were isolated, mainly using appropriate *E. coli* expression systems. The entire *APRR3* coding sequence was cloned onto pET22b(+) (Novagen), so as to be expressed as a histidine-tagged (Hisx6) polypeptide in *E. coli* BL21. The resulting APRR3-Hisx6 polypeptide was isolated by means of Ni-beads column chromatography, as described previously (Imamura et al. 1998). The entire *WNK1* coding sequence was also cloned onto pGEX5X1 (Toyobo), so as to be expressed as a glutathione S-transferase (GST) fusion polypeptide in *E. coli* AM93. The resulting GST-WNK1 polypeptide was isolated by means of glutathione-Sepharose 4B chromatography (Amersham Pharmacia), as described previously (Makino et al. 2002). Using the same procedures, a GST-WNK1-C polypeptide lacking the kinase domain and a GST-WNK1-KD polypeptide containing only the WNK1 kinase domain were also isolated (see Fig. 3).

#### In vitro phosphorylation experiment
The GST-WNK1 and GST-WNK1-KD polypeptides, isolated as described above, were incubated in a buffer comprising 50 mM HEPES, 5 mM MgCl₂, 10 mM MnCl₂, 75 mM KCl, 1 mM DTT, and pH 7.5 in the presence of [γ-³²P]ATP (10,000 cpm pmol⁻¹) at 22°C for 30 min. To this reaction mixture, when necessary, appropriate substrates were also added, which included APRR3-Hisx6, GST, and MBP. The samples were analyzed by SDS-PAGE, followed by autoradiography, as described previously (Imamura et al. 1998).

#### RNA preparation and Northern hybridization
Total RNA was isolated from appropriate organs (mainly leaves) of *Arabidopsis* by the phenol-SDS method (Makino et al. 2002). For Northern hybridization, RNA was separated in agarose gels (14%) containing 2.2 M formaldehyde, and then transferred to Hybond-N+ membranes. Other procedures for Northern hybridization were described in detail previously (Makino et al. 2000, Makino et al. 2002). The hybridized filters were analyzed with a phosphoimage analyzer (BAS-2500II) (Fujiﬁlm). A set of double-stranded ³²P-labeled DNA probes was prepared by the combination of PCR-ampliﬁcation and random-labeling (Takara Shuzo) with appropriate primers (APRR3, 5′-TCACATGATTCTATGGTTCTGGTCC and 5′-TTGTGCCCATCGGTCACTTAA; WNK1, 5′-GACCCTTTTCTCCGTATAGACG and 5′-CCCGGTCTCCAAC-TAGGTAC).

### Acknowledgments
This study was supported by a Grant-in-Aid for scientiﬁc research on a priority area (12142201 and COE to TM) from the Ministry of Education, Science, Sports and Culture of Japan.

### References
Alabadi, D., Oyama, T., Yanovsky, M.J., Harmon, F.G., Mas, P. and Kay, S.A. (2001) Reciprocal regulation between TOC1 and LHY/CCA1 within the *Arabidopsis* circadian clock. *Science* 293: 880–883.

Arabidopsis Genome Initiative, The (2000) Analysis of the genome sequence of the flowering plant *Arabidopsis thaliana*. *Nature* 408: 796–815.

Asai, T., Tena, G., Plotnikova, J., Willmann, M.R., Chiu, W.-L., Gomez-Gomez, L., Boller, J., Ausubel, F.M. and Sheen, J. (2002) MAP kinase signaling cascade in *Arabidopsis* innate immunity. *Nature* 415: 977–983.

Barak, S., Tobin, E.M., Andronis, C., Sugano, S. and Green, R.M. (2000) All in good time: the *Arabidopsis* circadian clock. *Trends Plant Sci.* 5: 517–522.

Carre, I.A. (2001) Day-length perception and the photoperiodic regulation of ﬂowering in *Arabidopsis*. *J. Biol. Rhythms* 1: 415–423.

Dunlap, J.C. (1999) Molecular bases for circadian clocks. *Cell* 96: 271–290.

Green, R.M. and Tobin, E.M. (1999) Loss of the circadian clock-associated protein 1 in *Arabidopsis* results in altered clock-regulated gene expression. *Proc. Natl. Acad. Sci. USA* 96: 4176–4179.

Harmer, S.L., Hogenesch, J.B., Straume, M., Chang, H.-S., Han, B., Zhu, T., Wang, X., Kreps, J.A. and Kay, S.A. (2000) Orchestrated transcription of key pathways in *Arabidopsis* by the circadian clock. *Science* 290: 2110–2113.

Imamura, A., Hanaki, N., Umeda, H., Nakamura, A., Suzuki, T., Ueguchi, C. and Mizuno, T. (1998) Response regulators implicated in His-to-Asp phos-

phototransfer signaling in *Arabidopsis*. Proc. Natl. Acad. Sci. USA 95: 2691–2696.

Innes, R.W. (2001) Mapping out the roles of MAP kinases in plant defense. Trends Plant Sci. 6: 392–394.

Makino, S., Kiba, T., Imamura, A., Hanaki, N., Nakamura, A., Suzuki, T., Taniguchi, M., Ueguchi, C., Sugiyama, T. and Mizuno, T. (2000) Genes encoding pseudo-response regulators: Insight into His-to-Asp phosphorelay and circadian rhythm in *Arabidopsis thaliana*. Plant Cell Physiol. 41: 791–803.

Makino, S., Matsushika, A., Kojima, M., Oda, Y. and Mizuno, T. (2001) Light response of the circadian waves of the APRR1/TOC1 quintet: When does the quintet start singing rhythmically in *Arabidopsis*. Plant Cell Physiol. 42: 334–339.

Makino, S., Matsushika, A., Kojima, M., Yamashino, T. and Mizuno, T. (2002) The APRR1/TOC1 quintet implicated in circadian rhythms of *Arabidopsis thaliana*: I. Characterization with APRR1-overexpressing plants. Plant Cell Physiol. 43: 58–69.

Matsushika, A., Makino, S., Kojima, M. and Mizuno, T. (2000) Circadian waves of expression of the APRR1/TOC1 family of pseudo-response regulators: Insight into the plant circadian clock. Plant Cell Physiol. 41: 1002–1012.

Matsushika, A., Makino, S., Kojima, M., Yamashino, T. and Mizuno, T. (2002) The APRR1/TOC1 quintet implicated in circadian rhythms of *Arabidopsis thaliana*: II. Characterization with CCA1-overexpressing plants. Plant Cell Physiol. 43: 118–122.

Martinez-Garcia, J.F., Huq, E. and Quail, P.H. (2000) Direct targeting of light signals to a promoter element-bound transcription factor. Science 288: 859–863.

Murtais, G. and Millar, A.J. (2000) How plants tell the time. Curr. Opin. Plant Biol. 3: 43–46.

Ni, M., Tepperman, J.M. and Quail, P.H. (1998) PIF3, a phytochrome-interacting factor necessary for normal photoinduced signal transduction, is a novel basic helix-loop-helix protein. Cell 95: 657–667.

Samach, A. and Coupland, G. (2000) Time measurement and the control of flowering in plants. Bioessays 22: 38–47.

Schaffer, R., Landgraf, J., Accerbi, M., Simon, V.V., Larson, M. and Wisman, E. (2001) Microarray analysis of diurnal and circadian-regulated gene in Arabi-

dopsis. Plant Cell 13: 113–123.

Schaffer, R., Ramsay, N., Samach, A., Corden, S., Putterill, J., Carre, I.A. and Coupland, G. (1998) The late elongated hypocotyls mutation of *Arabidopsis* disrupts circadian rhythms and the photoperiodic control of flowering. Cell 93: 1219–1229.

Somers, D.E., Webb, A.A.R., Pearson, M. and Kay, S.A. (1998) The short-period mutant, *toc1-1*, alters circadian clock regulation of multiple outputs throughout development in *Arabidopsis thaliana*. Development 125: 485–494.

Strayer, C., Oyama, T., Schultz, T.F., Raman, R., Somers, D.E., Mas, P., Panda, S., Kreps, J.A. and Kay, S.A. (2000) Cloning of the *Arabidopsis* clock gene *TOC1*, an autoregulatory response regulator homolog. Science 289: 768–771.

Sugano, S., Andronis, C., Green, R.M., Wang, Z.-Y. and Tobin, E.M. (1998) Protein kinase CK2 interacts with and phosphorylates the *Arabidopsis* circadian clock-associated 1 protein. Proc. Natl. Acad. Sci. USA 95: 11020–11025.

Sugano, S., Andronis, C., Org, M.S., Green, R.M. and Tobin, E.M. (1999) The protein kinase CK2 in involved in regulation of circadian rhythms in *Arabidopsis*. Proc. Natl. Acad. Sci. USA 96: 12362–12366.

Tena, G., Asai, T., Chiu, W.-L. and Sheen, J. (2001) Plant mitogen-activated protein kinase signaling cascades. Curr. Opin. Plant Biol. 4: 392–400.

Verissimo, F. and Jordan, P. (2001) WNK kinases, a novel protein kinase subfamily in multi-cellular organisms. Oncogene 20: 5562–5569.

Wang, Z.-Y. and Tobin, E.M. (1998) Constitutive expression of the CIRCADIAN CLOCK ASSOCIATED 1 (*CCA1*) gene disrupts circadian rhythms and suppresses its own expression. Cell 93: 1207–1217.

Wilson, F.H., Disse-Nicodeme, S., Choate, K.A., Ishikawa, K., Nelson-Williams, C., Desitter, I., Gunel, M., Milford, D.V., Lipkin, G.W., Achard, J.M., Feely, M.P., Dussol, B., Berland, Y., Unwin, R.J., Mayan, H., Simon, D.B., Farfel, Z., Jeunemaitre, X. and Lifton, R.P. (2001) Human hypertension caused by mutations in WNK kinases. Science 293: 1107–1112.

Xu, B., English, J.M., Wilsbacher, J.L., Stippec, S., Goldsmith, E.J. and Cobb, M.H. (2000) WNK1, a novel mammalian serine/threonine protein kinase lacking the catalytic lysine in subdomain II. J. Biol. Chem. 275: 16795–16801.

(Received February 22, 2002; Accepted April 12, 2002)
