
JOURNAL OF BACTERIOLOGY, Aug. 2002, p. 4219–4232
0021-9193/02/$04.00+0 DOI: 10.1128/JB.184.15.4219–4232.2002
Copyright © 2002, American Society for Microbiology. All Rights Reserved.

# Origins of the 2,4-Dinitrotoluene Pathway

Glenn R. Johnson, Rakesh K. Jain,† and Jim C. Spain*

Air Force Research Laboratory, U.S. Air Force, Tyndall Air Force Base, Florida 32403

Received 11 February 2002/Accepted 6 May 2002

The degradation of synthetic compounds requires bacteria to recruit and adapt enzymes from pathways for naturally occurring compounds. Previous work defined the steps in 2,4-dinitrotoluene (2,4-DNT) metabolism through the ring fission reaction. The results presented here characterize subsequent steps in the pathway that yield the central metabolic intermediates pyruvate and propionyl coenzyme A (CoA). The genes encoding the degradative pathway were identified within a 27-kb region of DNA cloned from *Burkholderia cepacia* R34, a strain that grows using 2,4-DNT as a sole carbon, energy, and nitrogen source. Genes for the lower pathway in 2,4-DNT degradation were found downstream from *dntD*, the gene encoding the extradiol ring fission enzyme of the pathway. The region includes genes encoding a CoA-dependent methylmalonate semialdehyde dehydrogenase (*dntE*), a putative NADH-dependent dehydrogenase (ORF13), and a bifunctional isomerase/hydrolase (*dntG*). Results from analysis of the gene sequence, reverse transcriptase PCR, and enzyme assays indicated that *dntD dntE* ORF13 *dntG* composes an operon that encodes the lower pathway. Additional genes that were uncovered encode the 2,4-DNT dioxygenase (*dntAaAbAcAd*), methylnitrocatechol monooxygenase (*dntB*), a putative LysR-type transcriptional (ORF12) regulator, an intradiol ring cleavage enzyme (ORF3), a maleylacetate reductase (ORF10), a complete ABC transport complex (ORF5 to ORF8), a putative methyl-accepting chemoreceptor protein (ORF11), and remnants from two transposable elements. Some of the additional gene products might play as-yet-undefined roles in 2,4-DNT degradation; others appear to remain from recruitment of the neighboring genes. The presence of the transposon remnants and vestigial genes suggests that the pathway for 2,4-DNT degradation evolved relatively recently because the extraneous elements have not been eliminated from the region.

---

Dinitrotoluenes are intermediates in the synthesis of explosives and expanded polyurethane foam (53). Bacteria degrade 2,4-dinitrotoluene (2,4-DNT) by a pathway involving two oxygenase reactions that lead to the removal of the nitro substituents (65) (Fig. 1). The resulting methylhydroxyquinone is reduced to yield 2,4,5-trihydroxytoluene, which subsequently undergoes *meta* cleavage. The reactions following ring cleavage have not been previously characterized.

There are few naturally occurring nitroaromatic compounds so environmental exposure to nitroarenes has been limited until the development of modern synthetic chemistry (44). De novo evolution of genes for nitrotoluene degradation during the short period seems unlikely. Instead, it is more plausible that the pathways evolve by recruiting genes that encode degradative enzymes for other compounds to assemble a functional pathway (47). Cloning and analysis of the genes encoding the nitroarene pathways makes it possible to distinguish between the two adaptation mechanisms.

*Burkholderia cepacia* R34 was isolated from surface water collected at the Radford Army Ammunition plant in West Virginia (43). Strain R34 grows using 2,4-DNT as a sole carbon and nitrogen source using a pathway like that from *Burkholderia* sp. strain DNT (43, 65) (Fig. 1). Previous molecular genetic studies of the enzymes for the catabolism of nitrotoluenes were limited to genes encoding individual steps of the degradative pathways (21, 22, 28, 48, 68). The studies provided clues to the relationships among the nitrotoluene pathway enzymes and other degradative enzymes but less insight about the evolution of the entire pathway. We reported that the *meta*-ring cleavage enzyme from strain R34 shares limited identity with its counterpart in strain DNT (28). The present paper provides the nucleotide sequence of a contiguous region encoding the pathway for 2,4-DNT degradation and additional elements proposed to contribute to regulation and evolution of the pathway. Inferences from molecular genetic data and results from enzyme assays allowed identification of the metabolic steps following ring cleavage. In addition, the analysis provided insight into the mechanisms involved in recruitment of the genes, assembly of the pathway, and progenitors of the enzymes used in nitroarene degradation.

(Preliminary reports of this work have been presented [B. E. Haigler, C. C. Somerville, J. C. Spain, and R. K. Jain, Abstr. 97th Gen. Meet. Am. Soc. Microbiol., abstr. Q-343, p. 512, 1997; G. R. Johnson, R. K. Jain, B. E. Haigler, and J. C. Spain, Abstr. 98th Gen. Meet. Am. Soc. Microbiol., abstr. Q-83, p. 435, 1998].)

## MATERIALS AND METHODS

### Bacterial strains and plasmids
Bacterial strains and plasmids used in the study are listed in Table 1.

### Media and growth conditions
Strain R34 was grown at 30°C using Trypticase soy medium (Difco Inc., Detroit, Mich.) or modified Stanier's base medium (MSB) (65) supplemented with succinate (0.5%) and other carbon sources as noted in the text. *Escherichia coli* strains were grown in Luria-Bertani medium (Difco, Inc.) or Terrific broth (73). Recombinant *E. coli* strains were tested for growth on 2,4,5-trihydroxytoluene by auxanography as described by Parke and Ornston (50). *E. coli* cultures were incubated at 37°C for molecular genetic procedures and at 30°C for use in enzyme and auxanography assays. Ampicillin

* Corresponding author. Mailing address: Air Force Research Laboratory/MLQ, 139 Barnes Dr., Tyndall Air Force Base, FL 32403. Phone: (850) 283-6058. Fax: (850) 283 6090. E-mail: jim.spain@tyndall.af.mil.
† Present address: Institute of Microbial Technology, Sector 39-A, Chandigarh 160036, India.

Plasmid DNA was transferred into competent *E. coli* strains using a chemical transformation method (24) or electroporation using the general method described previously (3) and parameters recommended by the manufacturer (Hoefer Scientific, San Francisco, Calif.).

Cloning and localization of 2,4-DNT pathway genes. Total plasmid DNA was isolated from strain R34 by using the method described by Kado and Liu (29), partially digested with restriction enzyme EcoRI, and then ligated with cosmid vector pCP13. The ligated vector mix was packaged into lambda phage using the Gigapak II Plus system from Stratagene Inc. (La Jolla, Calif.). The resulting phage mix was used to transfect *E. coli* HB101 cells. Recombinant strains were screened for DNT-dioxygenase, methylnitrocatechol (MNC) monooxygenase, and trihydroxytoluene (THT) oxygenase activity by using methods described by Suen and Spain (69). Subclones were derived from cosmids that appeared to carry genes for 2,4-DNT degradation enzymes, transferred to *E. coli* JM109 cells, and then tested for enzyme activity as described before in order to identify coding regions (69).

DNA sequencing and sequence analysis. Plasmid DNA for sequencing reactions was isolated using P100 midi-prep tips (Qiagen, Inc., Valencia, Calif.). Plasmid subclones, restriction endonuclease-based deletion derivatives, and unidirectional nested deletion derivatives of plasmids pJS324, pJS325, and pJS326 were used as sequencing templates. Primers complementary to vector and internal sequences were used in dideoxy-chain termination sequencing reactions (60). Sequencing reactions were carried out using the AutoRead sequencing reagents according to the manufacturer’s suggested methods, and the products were analyzed using an ALFexpress DNA sequencing system (Amersham Pharmacia Biotech, Piscataway, N.J.). The nucleotide (nt) sequence of the regions encoding the 2,4-DNT dioxygenase and methylnitrocatechol monooxygenase (nt 4341 to 14868) was determined by the Genetic Engineering Core Facility, University of Illinois.

The Lasergene software package (DNASTar Inc., Madison, Wis.) was used for initial alignment of DNA sequences and assisting primer design for PCR and sequencing reactions. Sequence databases were searched using the BLASTX, BLASTP, and BLASTN programs (1) via the National Center for Biotechnology Information website (Bethesda, Md.). Multiple-sequence alignments were done using the ClustalX program (75).

Enzyme assays. *E. coli* strains used in enzyme assays were grown to mid-log phase in Luria-Bertani broth containing appropriate antibiotics, and then the synthesis of recombinant proteins was induced by the addition of IPTG (1 mM). The optimal induction time was found by determining the specific activity of the enzyme of interest or the relative synthesis of recombinant protein by using sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE). Strain R34 was grown in MSB with 2,4-DNT (3 mM) as a sole carbon and energy source or in MSB with 2,4-DNT and 0.3% succinate to supplement growth.

Cell extracts were used in enzyme assays except where noted. The cells were harvested by centrifugation, washed with cold potassium phosphate buffer (20 mM, pH 7.5), suspended in a minimal volume of cold buffer, and then passed twice through a French pressure cell (1,360 Pa) for cell lysis. The lysate was clarified by centrifugation (21,800 × g, 30 min, 4°C) before use in enzyme assays. Protein concentration was measured by using the bicinchoninic acid protein reagent (Pierce Inc., Rockford, Ill.).

Salicylate hydroxylase activity was assayed using whole cells as described by Fuenmayor et al. (17). Reverse-phase high-performance liquid chromatography (HPLC) was used to measure salicylate and gentisate concentrations after incubation of cells with salicylate (1 mM). HPLC was done using a model 1050 chromatography system (Agilent, Santa Clarita, Calif.) with an Allsphere octyl C-8 column (250 by 4.6 mm) (Alltech Associates Inc., Deerfield, Ill.). The mobile phase was 0.1% trifluoroacetic acid-acetonitrile (60:40), 1.0 ml/min. The eluent was monitored at 210, 290, and 325 nm.

Benzenetriol oxygenase activity was measured spectrophotometrically (decrease \(A_{290}\), increase \(A_{345}\)) (8). The benzenetriol oxygenase was partially purified from *E. coli* JM109 (pJS326) cells by using a series of ammonium sulfate precipitation steps. Cell extracts were obtained as above except that the initial cell lysate was subjected to high-speed centrifugation (100,000 × g, 1.5 h, 4°C) before ammonium sulfate treatment. Benzenetriol oxygenase activity precipitated between 45 and 55% ammonium sulfate.

2,4-Dihydroxy-5-methyl-6-oxo-2,4-hexadienoate (DMOH) isomerase/4-hydroxy-2-keto-5-methyl-6-oxo-3-hexenoate (HKMOH) hydrolase activity was initially detected by measuring the decrease in absorbance at 350 nm due to transformation of DMOH in phosphate buffer (50 mM, pH 7.5). Cell extracts were dialyzed for 12 h against 2 liters of cold phosphate buffer to remove diffusible cofactors. Qualitative assays of substrate transformation were monitored by scanning the UV absorbance in reaction mixtures from 220 to 425 nm. Time course assays of DMOH isomerase/HKMOH hydrolase activity were done

(100 μg/ml), tetracycline (15 μg/ml), or kanamycin (40 μg/ml) was included in media when needed to select recombinant strains.

Recombinant DNA techniques. Standard molecular genetic techniques were used for DNA isolation, cloning, and analysis of recombinant bacteria (59).

TABLE 1. Bacterial strains and plasmids

| Strain or plasmid | Relevant properties$^a$ | Reference or source |
|-------------------|------------------------|---------------------|
| **Strains**       |                        |                     |
| *Burkholderia cepacia* R34 | 2, 4-DNT degrading strain | 43 |
| *Escherichia coli* HB101 | Cloning host | 36 |
| *Escherichia coli* JM109 | Cloning host, *lacI*$^q$ | 82 |
| *Escherichia coli* DH5$\alpha$ | Cloning host | 23 |
| *Escherichia coli* Top10F' | Cloning host, *lacI*$^q$ | Invitrogen (Carlsbad, Calif.) |

| **Plasmids**      |                        |                     |
| pCP13             | *E. coli* cloning vector (cosmid), Tet$^r$ | 9 |
| PGEM7F'           | *E. coli* cloning vector, Amp$^r$ | Promega (Madison, Wis.) |
| pBluescript II SK(+) | *E. coli* cloning vector, Amp$^r$ | Stratagene (La Jolla, Calif.) |
| pK18              | *E. coli* cloning vector, Kan$^r$ | 54 |
| pK19              | *E. coli* cloning vector, Kan$^r$ | 54 |
| pUC18             | *E. coli* cloning vector, Amp$^r$ | 82 |
| pJS311            | 216-kb plasmid from strain R34, *dnt* genes | 28 |
| pJS312            | 50-kb plasmid from strain R34 | 28 |
| pJS314            | pCP13 containing 27-kb DNA fragment from pJS312 R34 | This work |
| pJS324            | ORF12, *dntAaAbAcAd*, *dntB*; pGEM7f containing 14.8-kb EcoRI fragment from pJS314 | This work |
| pJS325            | *dntD*, *dntE*, ORF13, *dntG*; 12.2-kb EcoRI fragment from pJS314 | 28 |
| pJS326            | *dntB*, ORF3; 8.6-kb HindIII fragment from pJS314 | This work |
| pJS329            | *dntAaAbAcAd*, *dntB*; pGEM7f containing 7.7-kb NsiI fragment from pJS324 | This work |
| pJS332            | *dntAaAbAcAd*; pGEM7f containing 5.1-kb NsiI-EcoRV fragment from pJS329 | This work |
| pJS333            | *dntD*, pGEM7f containing 1.5-kb XhoI::ClaI fragment from pJS328. | 28 |
| pJS336            | *dntD*, *dntE*, ORF13, *dntG*, ORF9, ORF10; pGEM7f containing 8.2-kb EcoRI::ClaI fragment from pJS325 | This work |
| pJS336$\Delta$BamHI | ORF13, *dntG*, ORF9, ORF10; deletion derivative of pJS336 | This work |
| pJS338            | *dntB*; pGEM7f containing 3.2-kb BamHI fragment from pJS324 | This work |
| pJS1362           | *dntE*, pK18 containing 1.8-kb region from pJS328 that encodes methylmalonic acid semialdehyde dehydrogenase | This work |
| pJS1363           | ORF13, *dntG*, pK18 containing 3.0-kb BamHI::PstI fragment from pJS336 | This work |
| pJS1363$\Delta$XmaI | *dntG*, deletion derivative of pJS1363 | This work |
| pJS1364           | ORF9, ORF10; pK19 containing 3.4-kb PstI::EcoRI fragment from pJS336 | This work |
| pJS1364$\Delta$HindIII | ORF10; deletion derivative of pJS1364 | This work |

$a$ Tet, tetracycline; Amp, ampicillin; Kan, kanamycin.

in 5-ml reaction mixtures containing DMOH and cell extract from *E. coli* JM109 (pJS1363$\Delta$XmaI) or strain R34. Samples were collected from the reaction mixture and treated with H$_2$SO$_4$ (final concentration, 0.05 N) to stop the reaction. Pyruvate and methylmalonic acid semialdehyde production were measured by HPLC using a model 1050 chromatography system (Agilent) with an Alltech model OA-1000 column (300 by 6.5 mm) (Alltech Associates Inc.). The mobile phase was 0.01 NH$_2$SO$_4$, 0.7 ml/min. The eluent was monitored at 210 and 240 nm. Pyruvate concentrations were determined by reference to a standard (Chem Service, West Chester, Pa.). The methylmalonic acid semialdehyde used as a standard was produced by hydrolysis of sodium $\alpha$-formylpropionate diethyl acetal (34). In all assays, DMOH was produced from 2,4,5-trihydroxytoluene (THT) using excess THT oxygenase (DntD) supplied in cell extracts of *E. coli* (pJS333).

Analysis of mRNA by RT-PCR. Cultures of *B. cepacia* R34 were grown in MSB containing succinate and 2,4-DNT to mid-log phase. Total RNA was isolated using the RNeasy system from Qiagen. The purified RNA was treated with RNase-free DNase to eliminate any contaminating DNA prior to the reverse transcriptase (RT) reaction. RT-PCR was done using the Access RT-PCR system from Promega (Madison, Wis.) according to the manufacturer’s instructions. Primers for RT-PCR were designed to amplify the following: a 501-bp region within *dntAc* (*dntAc*_up, 5'-TCT TGC GGA AAA CTT TGT AGG TGA-3'; *dntAc*_down, 5'-TCG TTG TCG TCG CTT TCC CAG TAT-3'), a 383-bp region from *dntAd* to *dntB* (*dntAdB*_up, 5'-CTA TGC AAC GCG AGA AGA CAA ATG-3'; *dntAdB*_down, 5'-CAA GCT ACC CCC GAC GAT GAG AAC-3'), a 1,391-bp region from ORF8 to *dntD* (ORF8*dntD*_up, 5'-TCCGGC GGC CAG CAG CAG ATG-3'; ORF8*dntD*_down, 5'-TCC GGG AAA AGG GCA AGC GAG TGA-3'), a 533-bp region within *dntD* (*dntD*_up, 5'-GCT CTG GGG CGT GGT TAC TGC TAC-3'; *dntD*_down, 5'-GTG GTG GAC GCG GGG ACT C-3'), a 511-bp region from *dntD* to *dntE* (*dntDE*_up, 5'-GTC CCC GCG TCC ACC ACT TC-3'; *dntDE*_down, 5'-CGG CGG CGT TCA CCT CTT-3'), and an 845-bp region from *dntE* to *dntG* (*dntEG*_up 5' AGC TGG GCG ATC TCG GTC CTT CTT ACG 3'; *dntEG*_down 5' GCG ACG CTT CCC ATC CAA C 3'). Products from RT-PCR were analyzed by electrophoresis in a 1.5% agarose gel.

Chemicals. 2,4,5-Trihydroxytoluene was provided by R. Spanggord (SRI International, Menlo Park, Calif.). Sodium $\alpha$-formylpropionate diethyl acetal was kindly provided by R. Harris (Indiana University School of Medicine, Indianapolis).

Nucleotide sequence accession number. The nucleotide sequence described here is available in GenBank under accession no. AF169302.

**RESULTS**

Cloning 2,4-DNT catabolic genes. Strain R34 carries two large plasmids, pJS311 (216 kb) and pJS312 (50 kb) (28). The region encoding the 2,4-DNT pathway was localized to a 50-kb region of pJS311 by analyzing plasmid DNA from strains that had lost the ability to grow with 2,4-DNT (28). No DNT dioxygenase, MNC monooxygenase, or THT dioxygenase activity was detectable in the mutant strain, indicating that the genes encoding the three enzymes are within the deleted region. The deleted region was isolated on a cosmid subclone as described in Materials and Methods. Three recombinant strains showed activity for all three enzymes, and the cosmids carried by the strains were designated pJS314, pJS315, and pJS316.

Restriction endonuclease digests of the three cosmids showed identical EcoRI restriction patterns, indicating that each held the same region of DNA from strain R34. The fragments were subcloned from pJS314 into vector pGEM7f and transferred to *E. coli* JM109 in order to identify the loci encoding the individual enzymes. The THT oxygenase gene was located within the fragment cloned in pJS325 (28). A second large restriction fragment (14.8 kb) was subcloned in

4222 JOHNSON ET AL.
J. BACTERIOL.

![Diagram](#)

pJS324; DNT-dioxygenase, MNC-monooxygenase, Benzenetriol oxygenase

pJS332; DNT-dioxygenase

pJS325; THT-oxygenase, DMOH isomerase/HKMOH hydrolase

pJS338; MNC-monooxygenase

pJS326; MNC-monooxygenase, Benzenetriol oxygenase

pJS333; THT-oxygenase

FIG. 2. Physical map of region encoding 2,4-DNT pathway genes. DNA inserts from subclones used to localize structural genes and synthesis of 2,4-dihydroxy-5-methyl-6-oxo-2,4-hexadienoic acid for enzyme assays are shown below the map.

pGEM7f and designated pJS324 (Fig. 2). Results of the enzyme assays showed that pJS324 included genes for the DNT dioxygenase and MNC monooxygenase. Subclones were derived from pJS324, transferred to *E. coli* JM109, and then tested for enzyme activity. The results indicated that the DNT dioxygenase genes were within the 5.6-kb DNA insert in pJS332. The MNC monooxygenase was encoded within the 3.2-kb *Bam*HI restriction fragment subcloned in pJS338 (Fig. 2). Nucleotide sequence analysis allowed identification of other structural genes within the regions.

DNT dioxygenase genes. The genes encoding the DNT dioxygenase (*dntAaAbAcAd*) were contained within a 5.8-kb region of pJS324 (Fig. 2). The DNT dioxygenase is related to the three-component Rieske non-heme iron oxygenases (25). The hydroxylating dioxygenase comprises an oxidoreductase (DntAa), an iron-sulfur ferredoxin protein (DntAb), and a terminal oxygenase center, DntAc (*α*-subunit) and DntAd (*β*-subunit), containing a Rieske-type iron-binding site in the *α*-subunit. The deduced amino acid sequences of the individual polypeptides were closely related to those of other nitroarene dioxygenases (48, 68) and the naphthalene dioxygenase from *Pseudomonas* sp. strain U2 (17) (Table 2), and the genes for the enzymes also have the same organization in the respective operons. Significant identity was also shared with other naphthalene and PAH dioxygenase genes (6, 12, 35, 64, 71) (Table 2). However, analysis of multiple-sequence alignments revealed that the oxygenase subunits from archetypal naphthalene dioxygenases form a separate phylogenetic group from the nitroarene dioxygenases and the naphthalene dioxygenase from strain U2.

Additional terminal oxygenase genes. Within the operon encoding the initial dioxygenase, three additional open reading frames (ORFs) were identified and designated ORF1a, ORF1b, and ORF2. Significant identity was shared between

ORF1a/ORF1b and a group of terminal oxygenase *α*-subunit proteins (Table 2), in particular the salicylate hydroxylase from strain U2 (17). Alignment of the ORF1a and ORF1b amino acid sequences with their homologs revealed that the two peptides were similar to the N- and C-terminal portions of the oxygenase *α*-subunits, respectively. The result suggested that a frameshift mutation had disrupted the ORF following amino acid 295 and led to a truncated peptide at amino acid 302 (Table 2). The next ORF (ORF1b) begins just after the ORF1a stop codon and encodes a 120-amino-acid peptide. Again, the alignment indicated strong conservation of sequence (Table 2); however, it is questionable whether the peptide is synthesized. No putative ribosome-binding site is present upstream of the ATG start codon, though translational coupling could allow synthesis of the second peptide. The deduced amino acid sequence of ORF2 was similar to those of terminal oxygenase *β*-subunits. The relative identity shared with counterparts from other operons was like that of the ORF1a/ORF1b sequence (Table 2).

Assays were done with extracts from *E. coli* JM109(pJS332) to find whether ORF1a, ORF1b, and ORF2 encode a functional salicylate hydroxylase (Fig. 2). No transformation of salicylate to gentisate or reduction in the initial concentration of salicylate was found during the assays. The result is consistent with the frameshift mutation eliminating the function of the putative salicylate hydroxylase. Cells from the same cultures exhibited DNT dioxygenase activity, which indicated that the other gene products in the region were synthesized to yield a functional hydroxylating dioxygenase.

MNC monooxygenase and benzenetriol dioxygenase. Functional mapping of the *dnt* gene cluster showed that the MNC monooxygenase gene (*dntB*) was downstream of the *dntA* gene cluster (Fig. 2). Analysis of the nucleotide sequence for this region revealed a 1,683-bp ORF that was designated *dntB*. The

Gene product    Source                              Function                          Length (no. of aa)  % Identity  Alignment          Source or reference
DntAa           Burkholderia cepacia R34            2,4-DNT dioxygenase               328              -           -                 -
NtdAa           Pseudomonas sp. strain JS42        2-Nitrotoluene dioxygenase       328              99          1-328—1-328      48
NagAa           Ralstonia sp. strain U2            Naphthalene dioxygenase          328              99          1-328—1-328      17
DntAa           Burkholderia sp. strain DNT        2,4-DNT dioxygenase               346              98          1-293—1-293      68
NahAa           Pseudomonas putida 9816            Naphthalene dioxygenase          328              66          1-328—1-293      64
ORF1a           Burkholderia cepacia R34           Salicylate hydroxylase           302              -           -                 -
NagG            Ralstonia sp. strain U2            Salicylate hydroxylase           423              99          1-295—1-295      17
ORF2            Pseudomonas sp. strain JS42        Salicylate hydroxylase           186              99          1-179—1-179      48
ORF2            Burkholderia sp. strain DNT        Salicylate hydroxylase           423              93          1-295—1-295      68
OhbB            Pseudomonas aeruginosa JB2         o-Halobenzoate dioxygenase      421              79          10-292—7-290     Accession no.
                                                                           AAC 69484
OhbB            Pseudomonas aeruginosa I42         o-Halobenzoate dioxygenase      428              51          20-257—12-245    77
IpbAa           Pseudomonas putida RE204           Isopropylbenzene dioxygenase    459              37          30-226—41-236    14
ORF1b           Burkholderia cepacia R34           Salicylate hydroxylase           120              -           -                 -
NagG            Ralstonia sp. strain U2            Salicylate hydroxylase           423              100         1-120—304-423    17
ORF2            Burkholderia sp. strain DNT        Salicylate hydroxylase           423              100         1-120—304-423    68
OhbB            Pseudomonas aeruginosa JB2         o-Halobenzoate dioxygenase      421              81          1-118—304-423    Accession no.
                                                                           AAC 69484
OhbB            Pseudomonas aeruginosa I42         o-Halobenzoate dioxygenase      428              36          3-119—294-410    77
IsbAa           Pseudomonas putida RE204           Isopropylbenzene dioxygenase    459              25          3-120—325-451    14
ORF2            Burkholderia cepacia R34           Salicylate hydroxylase           161              -           -                 -
ORFX            Burkholderia sp. strain DNT        Salicylate hydroxylase           161              100         1-161—1-161      68
NagH            Ralstonia sp. strain U2            Salicylate hydroxylase           161              98          1-161—1-161      17
OhbC            Pseudomonas aeruginosa JB2         o-Halobenzoate dioxygenase      157              57          1-161—1-157      Accession no.
                                                                           AAC 69484
OhbA            Pseudomonas aeruginosa I42         o-Halobenzoate dioxygenase      176              26          8-161—27-176     77
DntAb           Burkholderia cepacia R34           2,4-DNT dioxygenase               104             -           -                 -
DntAb           Burkholderia sp. strain DNT        2,4-DNT dioxygenase               104             86          1-104—1-104      68
NtdAb           Pseudomonas sp. strain JS42        2-Nitrotoluene dioxygenase       104             86          1-104—1-104      48
NagAb           Ralstonia sp. strain U2            Naphthalene dioxygenase          104             84          1-104—1-104      17
NahAb           Pseudomonas putida 9816            Naphthalene dioxygenase          104             74          1-103—1-103      64
DntAc           Burkholderia cepacia R34           2,4-DNT dioxygenase               447             -           -                 -
DntAc           Burkholderia sp. strain DNT        2,4-DNT dioxygenase               451             93          1-447—6-451      68
NagAc           Ralstonia sp. strain U2            Naphthalene dioxygenase          447             91          1-447—1-447      17
NtdAc           Pseudomonas sp. strain JS42        2-Nitrotoluene dioxygenase       447             90          1-447—1-447      48
NahAc           Pseudomonas putida 9816            Naphthalene dioxygenase          449             83          1-447—1-449      64
DntAd           Burkholderia cepacia R34           2,4-DNT dioxygenase               194             -           -                 -
DntAd           Burkholderia sp. strain DNT        2,4-DNT dioxygenase               194             98          1-194—1-194      68
NagAd           Ralstonia sp. strain U2            Naphthalene dioxygenase          194             92          1-194—1-194      17
NtdAd           Pseudomonas sp. strain JS42        2-Nitrotoluene dioxygenase       194             92          1-194—1-194      48
NahAd           Pseudomonas putida 9816            Naphthalene dioxygenase          194             78          1-194—1-194      64
DntB            Burkholderia cepacia R34           4-Methyl-5-nitrocatechol oxygenase 560             -           -                 -
DntB            Burkholderia sp. strain DNT        4-Methyl-5-nitrocatechol oxygenase 548             53          15-547—12-547    22
PheA            Plasmid pEST126                    Phenol monooxygenase             610             31          5-546—24-598     46
TfdB            Pseudomonas putida (pEST401)       2,4-dichlorophenol hydroxylase   585             31          15-527—9-563     32
TfdBII          Ralstonia eutropha JMP134          Chlorophenol monooxygenase       586             30          13-547—7-584     39
ORF3            Burkholderia cepacia R34           Intradiol dioxygenase            278             -           -                 -
HadC            Ralstonia pickettii DTP0602        Hydroxyquinol-1,2-dioxygenase   315             65          1-264—1-264      72
TfdC            Alcaligenes eutrophus              Chlorocatechol 1,2-dioxygenase  255             29          22-278—2-250     18
DntD            Burkholderia cepacia R34           Trihydroxytoluene oxygenase     314             -           -                 -
DntD            Burkholderia sp. strain DNT        Trihydroxytoluene oxygenase     315             60          1-313—1-314      21
BphC            Pseudomonas stutzeri A401          2,3-Dihydroxybiphenyl dioxygenase 363             38          6-308—28-329     70
HpcA            Brucella melitensis                Homoprotocatechuate dioxygenase  331             25          14-308—30-309    11
XylE            Pseudomonas putida pWWO            Catechol dioxygenase             307             21          23-279—20-285    41
DntE            Burkholderia cepacia R34           Methylmalonate semialdehyde dehydrogenase 508         -           -                 -
MmsA            Rattus norvegicus                  Methylmalonate semialdehyde dehydrogenase 535         60          9-494—34-519     31
MmsA            Pseudomonas aeruginosa             Methylmalonate semialdehyde dehydrogenase 497         55          15-497—5-487     66
CmtC            Pseudomonas putida F1              p-Cumic aldehyde dehydrogenase   494             34          18-489—19-490    13
StyD            Pseudomonas sp. strain Y2          Phenylacetaldehyde dehydrogenase 502             31          16-487—22-498    78
ORF13           Burkholderia cepacia R34           Putative NAD(P) H-dependent reductase 188           -           -                 -
ORF9            Burkholderia cepacia R34           Putative NAD(P) H-dependent reductase 189           62          5-184—7-186      62
LanZ4           Streptomyces cyanogenus            Putative reductase               200             38          42-156—55-163    80
SsuE            Pseudomonas putida S-313           NADH-dependent FMN reductase     200             29          5-135—3-133      Accession no.
                                                                           O85762


TABLE 2—Continued

| Gene product | Source | Function | Length (no. of aa) | % Identity | Alignment | Source or reference |
|--------------|--------|----------|-------------------|------------|-----------|--------------------|
| DntG         | Burkholderia cepacia R34 | 2,4-Dihydroxy-5-methyl-6-oxo-2,4-hexadienoate isomerase/4-hydroxy-2-keto-5-methyl-6-oxo-3-hexenoate hydrolase | 281 | — | — | — |
| HpcE         | Methanococcus jannaschii | 2-Hydroxyhepta-2,4-diene-1,7-dioate (HHDD) isomerase | 237 | 45 | 45–256–36–236 | 7 |
| HpcE         | Brucella melitensis | HHDD isomerase/5-carboxymethyl-2-oxo-3-hexene-1,7-dioate decarboxylase | 311 | 38 | 1–279–25–309 | 11 |
| DitD         | Pseudomonas abietaniphila | Putative isomerase | 293 | 35 | 40–255–60–274 | 40 |
| ORF 10      | Burkholderia cepacia R34 | Putative maleylacetate reductase | 379 | — | — | — |
| TftE        | Burkholderia cepacia AC1100 | Maleylacetate reductase | 352 | 65 | 8–338–4–332 | 10 |
| ClcE        | Pseudomonas sp. strain B13 | Maleylacetate reductase | 352 | 53 | 7–336–2–329 | 30 |
| TfdF        | Alcaligenes eutrophus (pJP4) | Maleylacetate reductase | 354 | 52 | 16–338–12–332 | 52 |
| ORF 12      | Burkholderia cepacia R34 | Putative regulatory protein | 301 | — | — | — |
| NahR        | Pseudomonas putida G7 | Regulatory protein | 300 | 61 | 1–300–1–300 | 63 |
| OhbR        | Pseudomonas putida JB2 | Putative regulatory protein | 310 | 57 | 1–301–1–301 | Accession no. AAC69484 |

* Deduced amino acid sequences of gene products from R34 were compared to previously described peptides using the BlastP search algorithm. Length indicates size of peptide (in amino acids). % Identity, percent of identical amino acids over the aligned region. Alignment, first range indicates portion of the R34-derived peptide and second range indicates portion of homolog used in alignment.

VOL. 184, 2002                                                                 BACTERIAL 2,4-DINITROTOLUENE DEGRADATION 4225

Transformation of  
Ring Fission Product  

pJS336  
Cial  
PstI  
BamHI  
HindIII  
Xmal  
HindIII  
PstI  
ClaI  
Xmal  
Xmal  
HindIII  
EcoRI  

dntD  
dntE  
ORF13  
dntG  
ORF9  
ORF10  

pJS333  
Cial  
Xhol  

pJS1362  
BglII  
BglII  
BamHI  
HindIII  
Xmal  
HindIII  
PstI  
ClaI  
Xmal  
Xmal  
HindIII  
EcoRI  

pJS336ΔBamHI  

pJS1363  
BamHI  
HindIII  
Xmal  
HindIII  
PstI  
PstI  
ClaI  
Xmal  
Xmal  
HindIII  
EcoRI  

pJS1363ΔXmal  

pJS1364  
HindIII  
EcoRI  

pJS1364ΔHindIII  

FIG. 3. Localization of gene product that catalyzes transformation of 2,4-dihydroxy-5-methyl-6-oxo-2,4-hexadienoic acid. Physical map of the DNA insert in pJS336 and subclones derived from the region is shown. The column on the right side indicates whether cell extracts from the corresponding recombinant strain consumed the 2,4,5-THT ring fission product.

and cysteine residues (Glu-274 and Cys-292) (26) lie within the region that matches the consensus sequence of the dehydrogenase active site (31).

The next ORF, ORF13, did not share significant identity with any previously identified proteins but appears to be distantly related to putative NADH-dependent reductases (Table 2). The biochemical role of the ORF13 homologs in Table 2 is not well defined. Their reported function appears to be based upon their sequence similarity to prototypical NADH-dependent reductases.

The 774-bp ORF that follows ORF13 was designated dntG. The deduced amino acid sequence of DntG shared significant identity with the class of fumarylacetoacetate (FAA) hydrolases from the phenylalanine/tyrosine degradation pathway and cycloisomerases (tautomerases) from various lower pathways (Table 2). The crystal structure and reaction mechanism of the FAA hydrolase is known (76). Interpretation of the consensus sequence for the DntG sequence aligned with several FAA hydrolase sequences revealed catalytically and structurally significant residues of DntG. The carboxyl oxygens from Asp56, Glu123, Glu125, and Glu154, should coordinately bind the calcium ion cofactor that is involved in substrate binding and catalysis. The spacing of the four acidic amino acids in DntG is perfectly conserved with the other hydrolases, although aspartate is usually found in the final position rather than glutamate, like in DntG. The FAA hydrolase class of enzymes uses a Glu-His-water triad in the reaction mechanism (76). The histidine serves as the general base to form the nucleophilic water molecule that is used in the reaction, and the carboxy group from glutamate stabilizes the histidine in its basic form. In the DntG sequence, His-82 lies within the active site predicted from sequence comparisons and should act as the general base in catalysis. The glutamine residue at position 161 of DntG is also expected to participate in stabilization of the nucleophilic water. The stabilizing glutamate residue of the triad is not obvious from sequence comparisons alone, but four conserved acidic residues in the C-terminal portion of the peptide, Asp236, Asp260, Glu265, and Glu267, are the best candidates for the role based on the crystal structure of FAA hydrolase.

The remaining ORFs in pJS336 lie 1.8 kb downstream from dntG (Fig. 2). ORF9 appears to be homologous with ORF13, sharing 69% identity over a 133-amino-acid region of the two peptides, greater than any other gene product in the public databases. ORF10 is 68 bp downstream from ORF9. ORF10 encodes a 452-amino-acid polypeptide that is similar to several maleylacetate reductases (Table 2). Maleylacetate reductases are typically found in ortho-ring fission pathways, such as those for degradation of chloroaromatic compounds and the proposed dioxin pathway (2). The anomalous association with the meta-ring fission enzyme and significant distance from dntG suggest that ORF9 and ORF10 are not associated with the 2,4-DNT pathway and instead are artifacts remaining from assembly of the region.

Subclones and deletion derivatives of pJS336 were made and used as sources for cell extracts in enzyme assays in order to find which of the gene products downstream from dntD catalyze the step following ring fission (Fig. 3). Cell extracts from strains that carried dntG catalyzed rapid disappearance of the 2,4,5-THT ring fission product in the absence of added cofactors. The putative aldehyde dehydrogenase encoded by dntE did not transform the intermediate despite addition of NAD or an NAD-regenerating system. Thus, oxidation of the aldehyde moiety to yield the corresponding dicarboxylic acid does not

Further experiments were done to define the reaction catalyzed by DntG. The identity that DntG shares with isomerases and hydrolases suggested that DntG might display dual catalytic abilities like one of its homologs, the bifunctional HHDD isomerase/OPET decarboxylase from the homoprotocatechuate degradation pathway (58). The spectrophotometric scans of the reaction mixture containing the ring fission product and DntG support the dual catalysis hypothesis. There was an initial shift in λ_max of 288 to 283 nm, and then as the reaction progressed to completion, the compound at 283 nm was transformed to compounds with little UV absorbance (Fig. 4). The stepwise change and absence of an isosbestic point in the spectra indicated that the ring fission product is not directly converted to the final compounds.

The products of the DntG-catalyzed reaction were analyzed using HPLC. Pyruvate (retention time, 6.1 min) was produced during the reaction in stoichiometric amounts from the ring fission product (Fig. 5). The results are consistent with previous work using strain DNT (21). The HPLC analysis revealed a second product that eluted as a broad peak at 8 min. We hypothesized that the second product was methylmalonic acid semialdehyde (2-methyl-3-oxo-propionic acid) (Fig. 6) by analogy to reactions catalyzed by the fumarylacetoacetate hydrolase of the tyrosine pathway and fumarylpyruvate hydrolase from the gentisate degradation pathway (83). Because hydrolysis of the ring fission product yields pyruvate, the remainder of the molecule should yield methylmalonic acid semialdehyde. The second compound coeluted with a methylmalonic acid semialdehyde standard and provided an identical spectrum (not shown). The rate at which the second compound accumulated was parallel to that of pyruvate (Fig. 5), consistent with it being a product of the hydrolysis reaction along with pyruvate. Similar results were found with cell extracts from strain R34 grown with 2,4-DNT.

The methylmalonic acid semialdehyde that results from hy-

FIG. 4. Conversion of 2,4-dihydroxy-5-methyl-6-oxo-2,4-hexadienoic acid to pyruvate and methylmalonate semialdehyde. The sample and reference cuvettes contained 2 μl of cell extract from E. coli JM109(pJS333) (120 μg of protein) and 1 μl of cell extract from E. coli JM109(pJS1363ΔXmaI) (56 μg of protein). The sample cuvette contained 200 nmol of 2,4,5-THT, which was converted to 2,4-dihydroxy-5-methyl-6-oxo-2,4-hexadienoic acid by the THT oxygenase from E. coli JM109(pJS333) extract. Spectra were recorded prior to and 20, 40, 60, 80, 100, 120, 140, 160, 180, and 200 s after addition of E. coli JM109-(pJS1363ΔXmaI) cell extract. The arrow indicates the first spectrum that was collected.

appear to follow ring fission as it does in some meta-ring fission pathways (19). The two putative NADH-dependent reductases were not involved in the step following ring fission, since the deletion of ORF13 or ORF9 (Fig. 3, pJS1363ΔXmaI and pJS1364ΔHindIII) had no apparent effect in the reaction. Similarly, no change was seen at A_350 with additions of cell extract from E. coli(pJS1364) or E. coli(pJS1364ΔHindIII), so the ORF10 gene product had no measurable activity against the THT ring fission product.

FIG. 5. Production of pyruvate and methylmalonate semialdehyde from 2,4-dihydroxy-5-methyl-6-oxo-2,4-hexadienoic acid by cell extracts of E. coli JM109(pJS1363ΔXmaI). The reaction was initiated with addition of cell extract (17 μg of protein/ml) to 200 μM substrate. The substrate was produced enzymatically from 2,4,5-THT with cell extracts from E. coli JM109(pJS333) (70 μg of protein/ml). Time course reactions were done in triplicate; the error bars show standard deviations for each time point. The methylmalate semialdehyde quantitation is the peak area for the chromatogram showing absorbance at 240 nm (▲). ■, pyruvate concentration (μM).

DNT (accession number AF076848) contains an incomplete ORF similar to *dntE*. The identical organization suggests that the dehydrogenase step is conserved in the pathways from both 2,4-DNT-degrading strains. The metabolism of methylmalonic acid semialdehyde in the 2,4-DNT pathway has not been confirmed experimentally.

Identification of ancillary ORFs. The nucleotide sequence of the region lying between *dntD* and the boundary of pJS325 (Fig. 2) contains four ORFs (ORF5 to ORF8) encoding peptides that shared significant similarity to ATP-binding cassette (ABC) transport proteins (16). ORF5 and ORF6 appear to be the channel-forming polypeptides based on their similarity to proteins from ABC transport systems and hydropathy plots of the amino acid sequences. ORF7 and ORF8 resemble ATPase components of ABC transport systems, and the amino acid sequence of each contains the well-conserved ATP-binding motif (79) and significant identity to the hydrophilic region of ATPase proteins from other ABC transport systems. No sequence encoding a potential periplasmic protein for solute binding, another gene product often associated with operons encoding ABC transport systems (42), was present upstream of ORF5.

A divergently transcribed ORF lies 120-bp upstream from *dntAa* (Fig. 2) that is similar to members of the LysR family of transcriptional regulators (61). The location and sequence similarity suggests that the gene product acts as a transcriptional regulator of the *dntA* genes, but adequate characterization has not been done. The ORF was designated ORF12. ORF12 and NahR from different naphthalene-degrading strains (6, 62) share about 60% identity (Table 2). Similar LysR-type regulators are found adjacent to the dioxygenase operons of strain DNT (accession number AF076848), U2 (83), JS42 (48), and JS765 (D. J. Lessner, R. E. Parales, and D. T. Gibson, Abstr. Am. Soc. Microbiol. Conf. Biodegradation Biotransformation Biocatalysis, 2001, abstr. 53, p. 38). The 2,4-DNT dioxygenase specific activity increases in strain R34 when 2,4-DNT or salicylate is present in minimal succinate media. The induced activity level suggests that transcription of the dioxygenase genes is positively regulated, but the ORF12 gene product has not been directly implicated in the regulation.

A 1,422-bp ORF adjacent to ORF12 was designated ORF11. The deduced amino acid sequence of ORF11 shared identity with methyl-accepting chemotaxis proteins, such as CheC, the aspartate chemoreceptor from *E. coli*. The highest degree of identity is in the peptide sequence that contains the conserved signaling domain (amino acids [aa] 288 to 332), which interacts with other proteins of the chemotaxis complex (5, 67). Four potential methylation and demethylation sites were revealed (aa 223, 228-229, 235-236, and 417-418) from interpretation of multi-sequence alignments (74). The region predicted to contain the sensor domain of the naphthalene chemoreceptor (NahY) from *Pseudomonas putida* G7 (20) shares no significant similarity with the putative receptor encoded by ORF11.

Two regions of DNA sequence within the cloned fragment are related to insertion sequence elements. ORF4, downstream from the benzenetriol oxygenase gene (Fig. 2), shares limited identity with transposases from various bacteria (Table 2) (4, 84). The region including nucleotides 1 to 2160 is related to the IS21 family of insertion sequences (55). Two imperfect direct repeats (22 out of 25 identical) lie in the 5′ end of the

FIG. 6. Proposed pathway for degradation of 2,4,5-THT in the 2,4-DNT pathway of *B. cepacia* R34. Compounds in brackets are proposed and have not been confirmed experimentally.

drolysis of 4-hydroxy-2-keto-5-methyl-6-oxo-3-hexenoic acid (Fig. 6) would be the expected substrate for the methylmalonic acid semialdehyde dehydrogenase homolog encoded by *dntE*. Oxidative decarboxylation of methylmalonic acid semialdehyde yields propionyl-CoA in the valine degradation pathway. The nucleotide sequence downstream of *dntD* from strain

FIG. 7. Summary of RT-PCR experiments. Gray bars below map represent potential amplicons resulting from the RT-PCRs. X's crossed through gray boxes indicate primer pairs that did not provide products in RT-PCRs. Bold arrows indicate transcripts deduced from sequence analysis, protein assays, and RT-PCR results.

homologous region (nucleotides 2133 to 2156 and 2131 to 2107). The region does not encode a complete transposase; however, the deduced amino acid sequence from nucleotides 1802 to 447 shared 63% identity to the central portion of the transposase (IstA) from IS1631 (27). The deduced amino acid sequence for nucleotides 414 to 2 is similar to that of the N-terminal portion of the ATP-dependent binding protein (IstB) from IS1631 (71% identity) (27), but a frameshift mutation seems to have introduced a stop codon at nt 321 to 319.

Analysis of operon organization by RT-PCR. RT-PCR was used to determine which of the colinear genes found in the region were transcribed as operons (Fig. 7). The results showed that the genes for the initial hydroxylating dioxygenase ($dntAa$-$dntAd$) and the methyl nitrocatechol monooxygenase ($dntB$) were not cotranscribed. No product was detected using the $dntAdB_{\text{up}}$ and $dntAdB_{\text{down}}$ primers. The control reaction using primers within $dntAc$ provided the expected 501-bp product. In the lower pathway, the genes encoding the meta-ring cleavage dioxygenase ($dntD$) and methylmuconic acid semialdehyde dehydrogenase ($dntE$) were cotranscribed and the primers corresponding to the two regions allowed detection of the anticipated cDNA. Likewise, the $dntE$-$dntG$ primers allowed detection of the corresponding cDNA. The results indicate that $dntD$ is the initial gene of the lower pathway operon that comprises four identified genes. ORF8 and $dntD$ were not cotranscribed, so the $dntDEFG$ transcript must originate from a promoter lying between $dntD$ and ORF8 (Fig. 2).

DISCUSSION

Oxidative growth with 2,4-DNT requires two novel oxygenases to catalyze the hydroxylation reactions that lead to release of nitro substituents from the aromatic ring. The oxygenolytic removal of the nitro substituent is electrochemically similar to oxygenolytic dehalogenation. However, the 2,4-DNT dioxygenase is closely related to the naphthalene dioxygenases, enzymes that have not demonstrated dehalogenation or denitration activity in previous investigations (56). The second step of the 2,4-DNT pathway also involves an oxygenolytic denitration reaction when methyl nitrocatechol is transformed to 2-hydroxy-5-methylquinone. The second reaction is analogous to those catalyzed by the similar chlorophenol hydroxylases that remove the electron-withdrawing halogen during hydroxylation. The flavin-containing pentachlorophenol monooxygenase (PcbB) from strain UG30 catalyzes oxygenolytic denitration of 4-nitrocatechol to yield 1,2,4-benzenetriol (38). The similarities between the monooxygenase-catalyzed reactions are intriguing considering the presence of the benzenetriol oxygen-

VOL. 184, 2002                                                                 BACTERIAL 2,4-DINITROTOLUENE DEGRADATION  4229

![Diagram](https://i.imgur.com/yourimage.png)

FIG. 8. Genetic organization of various multicomponent dioxygenases related to the 2,4-DNT dioxygenase from strain R34.

strain JS42 is also followed by an ORF similar to the genes encoding dihydrodiol dehydrogenases, but it does not yield a functional enzyme (48). A putative transposase borders the 3′ end of the 2,4-DNT dioxygenase from strain DNT (Fig. 8). Strain R34 appears to have a more efficient organization than the other characterized nitroarene-degrading strains. The gene adjacent to the dioxygenase operon encodes the enzyme used in the next step of the pathway rather than an unnecessary dehydrogenase.

Although the 2,4-DNT degradation pathways of strains R34 and DNT are identical, the sequence and organization of genes encoding the enzymes that catalyze the reactions following initial hydroxylation differ considerably. The disparity in sequence and G+C content of the two *dntB* genes opposes the hypothesis that recent horizontal gene transfer from a common ancestor contributed to the ability to degrade 4-methyl-5-nitrocatechol. In strain R34, the monooxygenase is linked to the initial dioxygenase although the genes for the two enzymes are not cotranscribed. For strain DNT, the relative locations of the *dntA* and *dntB* are not known but it is clear that the two clusters are separated by more than 8 kb (69); furthermore, the regions downstream of the *dntB* genes do not share significant similarity. Likewise, the sequence, G+C content, and organization of the extradiol ring cleavage dioxygenases (DntD) from strain R34 and strain DNT are very different. One gap remains in the genetics of 2,4-DNT degradation. The gene encoding the methylhydroxyquinone reductase for the 2,4-DNT pathway has not been identified for either strain.

The two regions resembling portions of insertion sequences might be associated with initial positioning and assembly of the *dnt* structural genes, subsequent failed transposition events, or remnants from insertion into another transposon. The transposase sequences found in the region (Fig. 2) shared no significant identity, indicating that the elements represent independent transposition events since it is rare that individual arms of transposable elements carry nonhomologous transposase genes. Preliminary characterization showed that the 2,4-DNT pathway was spontaneously lost from strain R34 under nonselective conditions (28). Only a portion of the machinery necessary for transposition was identified, so the instability and deletion of the phenotype in strain R34 was not likely to be mediated by either of the described regions but by elements outside of the sequenced DNA fragment.

Parke et al. (49) suggest that genetic characterization of catabolic pathways will uncover associations between catabolic gene clusters and transport systems that play a role in degradative pathways. The suggestion was supported in the characterization of the catabolic plasmid, pNL1, from *Sphingomonas aromaticus* F199 (57). In pNL1, several putative membrane proteins homologous to transport proteins were identified and proposed to play a role in aromatic compound catabolism. The putative ABC transport system formed from the ORF5 to ORF8 gene products might be related to nitroarene catabolism. Alternatively, the transport system could have been recruited with the lower pathway operon, since each gene cluster has homologs in amino acid transport or degradation. However, since genes encoding ABC transport proteins compose as much as 5% of a prokaryotic genome (51), it is also reasonable that the close association is a coincidence and unrelated to 2,4-DNT transport and degradation.

The dissimilarity in the receptor regions (periplasmic domains) of ORF11 and the naphthalene chemoreceptor is incongruous with the high identity shared by the naphthalene and DNT dioxygenases. The difference suggests that ORF11 evolved for chemotaxis to another compound and is not a remnant from an earlier naphthalene-sensing cascade. There are two possibilities: the ORF11 gene product has evolved to mediate chemotaxis toward nitroarenes and enhance their biodegradation or, once again, its association with the *dnt* genes is coincidental.

Regulation of gene expression might be an important adaptation for bacteria to grow with 2,4-DNT, but the mechanism that strain R34 uses to control expression of the multiple transcripts is not known. The juxtaposition of the LysR-type regulatory protein (ORF12) found upstream from the *dntA* genes suggests that it might regulate their transcription but has

not been characterized adequately. No additional regulatory genes were found in analysis of the regions around the methyl nitrocatechol monooxygenase or lower pathway genes. Transcription of the other operons could be unregulated, controlled by other global regulators, or dependent on the promoter/operator region of each operon.

Inferences from the comparison of the structural genes of the 2,4-DNT pathway suggest that the pathway came together from three sources. The initial dioxygenase appears to have originated from a naphthalene degradation pathway like that of strain U2 (17). A large portion of the salicylate hydroxylase oxygenase component is retained but is not functional. The MNC monooxygenase was probably derived from a pathway for degradation of chloroaromatic compounds. The presence of the vestigial (with respect to 2,4-DNT degradation) ortho-ring fission dioxygenase is consistent with its recruitment from a pathway for chloroaromatic compounds. The true ring fission enzyme for 2,4-DNT degradation has a different origin. The sequence of DntD is quite dissimilar to all other described meta-ring fission enzymes, including those from naphthalene and chloroarene degradative pathways. The distinctive sequence of the ring cleavage enzyme reflects the substrate specificity observed for the THT oxygenase (28). The distant relationship between homogentisate dioxygenase and DntD and the association with homologs from amino acid metabolism (*dntE* and *dntG*) indicate that the lower pathway operon arose from a gene cluster for amino acid degradation.

The disparate origins of the various *dnt* and associated genes described in this study are consistent with the difficulties that bacteria face to achieve efficient metabolism of synthetic compounds like 2,4-DNT. The organization of the pathway genes suggests there is a progression towards a compact region encoding the entire pathway. In that progression, remnants from assembly persist, such as the benzenetriol oxygenase (ORF3), putative maleylacetate reductase (ORF10), and putative transposase (ORF4). No role in nitroarene degradation is apparent for the remnants; their presence might indicate an intermediate point in the evolution of an optimal system or perhaps some of the proteins could be used in other pathways when another substrate is available.

### ACKNOWLEDGMENTS

We thank Billy Haigler for his work with the benzenetriol oxygenase, Chuck Sommerville for compilation of sequence data provided by the University of Illinois core facility, George Paoli for many helpful discussions, Angela Day for technical support, and John Sokatch for assistance in locating the methylmalonic acid semialdehyde precursor. This work was supported in part by the Air Force Office of Scientific Research and the Strategic Environmental Research and Development Program. R.K.J. acknowledges the Air Force Office of Scientific Research and the International Window on Science Program for support.

### REFERENCES

1. Altschul, S. F., T. L. Madden, A. A. Schäfer, J. Zhang, Z. Zhang, W. Miller, and D. J. Lipman. 1997. Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res. **25**:3389–3402.
2. Armengaud, J., K. N. Timmis, and R. M. Wittich. 1999. A functional 4-hydroxy salicylate/hydroxyquinol degradative pathway gene cluster is linked to the initial dibenzo-*p*-dioxin pathway genes in *Sphingomonas* sp. strain RW1. J. Bacteriol. **181**:3452–3461.
3. Ausubel, F. M., R. Brent, R. E. Kingston, D. D. Moore, J. G. Seidman, J. A. Smith, and K. Struhl (ed.). 1998. Current protocols in molecular biology. John Wiley and Sons, Inc., New York, N.Y.
4. Berge, A., M. Rasmussen, and L. Björck. 1998. Identification of an insertion sequence located in a region encoding virulence factors of *Streptococcus pyogenes*. Infect. Immun. **66**:3449–3453.
5. Blair, D. F. 1995. How bacteria sense and swim. Annu. Rev. Microbiol. **49**:489–522.
6. Bosch, R., E. GarciaValdes, and E. R. B. Moore. 1999. Genetic characterization and evolutionary implications of a chromosomally encoded naphthalene-degradation upper pathway from *Pseudomonas stutzeri* AN10. Gene **236**:149–157.
7. Bult, C. J., O. White, G. J. Olsen, L. Zhou, R. D. Fleischmann, G. G. Sutton, J. A. Blake, L. M. FitzGerald, R. A. Clayton, J. D. Gocayne, A. R. Kerlavage, B. A. Dougherty, J. Tomb, M. D. Adams, C. I. Reich, R. Overbeek, E. F. Kirkness, K. G. Weinstock, J. M. Merrick, A. Glodek, J. D. Scott, N. S. Geoghagen, J. F. Weidman, J. L. Fuhrmann, D. T. Nguyen, T. Utterback, J. M. Kelley, J. D. Peterson, P. W. Sadow, M. C. Hanna, M. D. Cotton, M. A. Hurst, K. M. Roberts, B. B. Kaine, M. Borodovsky, H. P. Klénk, C. M. Fraser, H. O. Smith, C. R. Woese, and J. C. Venter. 1986. Complete genome sequence of the methanogenic archaeon *Methanococcus jannaschii*. Science **273**:1058–1073.
8. Chapman, P. J., and D. W. Ribbons. 1976. Metabolism of resorcinolic compounds by bacteria: alternative pathways for resorcinol catabolism in *Pseudomonas putida*. J. Bacteriol. **125**:985–998.
9. Darzins, A., and A. M. Chakrabarty. 1984. Cloning of genes controlling alginate biosynthesis from a mucoid cystic fibrosis isolate of *Pseudomonas aeruginosa*. J. Bacteriol. **159**:9–18.
10. Daubaras, D. L., C. D. Hershberger, K. Kitano, and A. M. Chakrabarty. 1995. Sequence analysis of a gene cluster involved in metabolism of 2,4,5-trichlorophenoxyacetic acid by *Burkholderia cepacia* AC1100. Appl. Environ. Microbiol. **61**:1279–1289.
11. Del Vecchio, V. G., V. Kapral, R. J. Redkar, G. Patra, C. Mujer, T. Los, N. Ivanova, I. Anderson, A. Bhattacharyya, A. Lykidis, G. Reznik, L. Jablonski, N. Larsen, M. D’Souza, A. Bernal, M. Mazur, E. Goltsman, E. Selkov, P. H. Elzer, S. Hagiuss, D. O’Callaghan, J.-J. Letesson, R. Haselkorn, N. Kyrpides, and R. Overbeek. 2002. The genome sequence of the facultative intracellular pathogen *Brucella melitensis*. Proc. Natl. Acad. Sci. USA **99**:443–448.
12. Denome, S. A., D. C. Stanley, E. S. Olson, and K. D. Young. 1993. Metabolism of dibenzothiophene and naphthalene in *Pseudomonas* strains: complete DNA sequence of an upper naphthalene catabolic pathway. J. Bacteriol. **175**:6890–6901.
13. Eaton, R. W. 1996. *p*-Cumate catabolic pathway in *Pseudomonas putida* F1: cloning and characterization of DNA carrying the *cmt* operon. J. Bacteriol. **178**:1351–1362.
14. Eaton, R. W., O. V. Selifonova, and R. M. Gedney. 1998. Isopropylbenzene catabolic pathway in *Pseudomonas putida* RE204: nucleotide sequence analysis of the *ibp* operon and neighboring DNA from pRE4. Biodegradation **9**:119–132.
15. Eggink, G., H. Engel, G. Vriend, P. Terpestra, and B. Witholt. 1990. Rubredoxin reductase of *Pseudomonas oleovorans*: structure relationship to other flavoprotein oxidoreductases based on one NAD and two FAD fingerprints. J. Mol. Biol. **212**:135–142.
16. Fath, M., and R. Kolter. 1993. ABC transporters: bacterial exporters. Microbiol. Rev. **57**:995–1017.
17. Fuenmayor, S. L., M. Wild, A. L. Boyes, and P. A. Williams. 1998. A gene cluster encoding steps in conversion of naphthalene to gentisate in *Pseudomonas* sp. strain U2. J. Bacteriol. **180**:2522–2530.
18. Ghosal, D., and I.-S. You. 1988. Nucleotide homology and organization of chlorocatechol oxidation genes of plasmids pJP4 and pAC27. Mol. Gen. Genet. **211**:113–120.
19. Gibson, D. T., and V. Subramanian. 1984. Microbial degradation of aromatic hydrocarbons, p. 181–252. In D. T. Gibson (ed.), Microbial degradation of organic compounds. Marcel Dekker, Inc., New York, N.Y.
20. Grimm, A. C., and C. S. Harwood. 1999. NahY, a catabolic plasmid-encoded receptor required for chemotaxis of *Pseudomonas putida* to the aromatic hydrocarbon naphthalene. J. Bacteriol. **181**:3310–3316.
21. Haigler, B. E., G. R. Johnson, W.-C. Suen, and J. C. Spain. 1999. Biochemical and genetic evidence for meta-ring cleavage of 2,4,5-trihydroxytoluene in *Burkholderia* sp. strain DNT. J. Bacteriol. **181**:965–972.
22. Haigler, B. H., W.-C. Suen, and J. C. Spain. 1996. Purification and sequence analysis of 4-methyl-5-nitrocatechol oxygenase from *Burkholderia* sp. strain DNT. J. Bacteriol. **178**:6019–6024.
23. Hanahan, D. 1983. Studies on transformation of *E. coli* with plasmids. J. Mol. Biol. **166**:557–580.
24. Hanahan, D. 1985. Techniques for transformation of *E. coli*, p. 109–136. In D. M. Glover (ed.), The practical approach, vol. 1. DNA cloning. IRL Press, Ltd., Oxford, United Kingdom.
25. Harayama, S., M. Kok, and E. L. Neidle. 1992. Functional and evolutionary relationships among diverse oxygenases. Annu. Rev. Microbiol. **46**:565–601.
26. Hempel, J., H. Nicholas, and R. Lindahl. 1993. Aldehyde dehydrogenases: widespread structural and functional diversity within a shared framework. Protein Sci. **2**:1890–1900.
27. Isawa, T., R. Sameshima, H. Mitsui, and K. Minamisawa. 1999. IS1631 occurrence in *Bradyrhizobium japonicum* highly reiterated sequence-

ing strains with high copy numbers of repeated sequences RSα and RSβ. Appl. Environ. Microbiol. 65:3493–3501.

28. Johnson, G. R., R. K. Jain, and J. C. Spain. 2000. Properties of the trihydroxytoluene oxygenase from *Burkholderia cepacia* R34; an extradiol dioxygenase from the 2,4-dinitrotoluene pathway. Arch. Microbiol. 173:86–90.

29. Kado, C. I., and S.-T. Liu. 1981. Rapid procedure for detection and isolation of large and small plasmids. J. Bacteriol. 145:1365–1373.

30. Kasberg, T., V. Seibert, M. Schlomann, and W. Reineke. 1997. Cloning, characterization, and sequence analysis of the *clcE* gene encoding the maleylacetate reductase of *Pseudomonas* sp. strain B13. J. Bacteriol. 179:3801–3803.

31. Kedishvili, N. Y., K. M. Popov, P. M. Rougraff, Y. Zhao, D. W. Crabb, and R. A. Harris. 1992. CoA-dependent methylmalonate-semialdehyde dehydrogenase, a unique member of the aldehyde dehydrogenase superfamily: cDNA cloning, evolutionary relationships, and tissue distribution. J. Biol. Chem. 267:19724–19729.

32. Kõiv, V., R. Marits, and A. Heinaru. 1996. Sequence analysis of the 2,4-dichlorophenol hydroxylase gene *tfdB* and 3,5-dichlorocatechol 1,2-dioxygenase gene *tfdC* of 2,4-dichlorophenoxyacetic acid degrading plasmid pEST4011. Gene 174:293–297.

33. Kukor, J. J., and R. H. Olsen. 1992. Complete nucleotide sequence of *tbuD*, the gene encoding phenol/cresol hydroxylase from *Pseudomonas pickettii* PKO1, and functional analysis of the encoded enzyme. J. Bacteriol. 174:6518–6526.

34. Kupiecki, F. P., and M. J. Coon. 1960. Methylmalonic semialdehyde. Biochem. Prep. 7:69–71.

35. Kurkela, S., H. Lehvaslaiho, E. T. Palva, and T. H. Teeri. 1988. Cloning, nucleotide sequence and characterization of genes encoding naphthalene dioxygenase of *Pseudomonas putida* NCIB9816. Gene 73:355–362.

36. Lacks, S., and J. R. Greenberg. 1977. Complementary specificity of restriction endonucleases of *Diplococcus pneumoniae* with respect to DNA methylation. J. Mol. Biol. 114:153–168.

37. Lessner, D. J., G. R. Johnson, R. E. Parales, J. C. Spain, and D. T. Gibson. 2002. Molecular characterization and substrate specificity of nitrobenzene dioxygenase from *Comamonas* sp. strain JS765. Appl. Environ. Microbiol. 68:634–641.

38. Leung, K. T., S. Campbell, Y. Gan, D. C. White, H. Lee, and J. T. Trevors. 1999. The role of the *Sphingomonas* species UG30 pentachlorophenol-4-monooxygenase in *p*-nitrophenol degradation. FEMS Microbiol. Lett. 173:247–253.

39. Leveau, J. H. J., and J. R. van der Meer. 1996. The *tfdR* gene product can successfully take over the role of the insertion element-inactivated TfdT protein as a transcriptional activator of the *tfdCDEF* gene cluster, which encodes chlorocatechol degradation in *Ralstonia eutropha* JMP134 (pJP4). J. Bacteriol. 178:6824–6832.

40. Martin, V. J. J., and W. W. Mohn. 1999. A novel aromatic-ring-hydroxylating dioxygenase from the diterpenoid-degrading bacterium *Pseudomonas abietaniphila* BKME-9. J. Bacteriol. 181:2675–2682.

41. Nakai, C., H. Kagamiyama, M. Nozaki, T. Nakazawa, S. Inouye, Y. Ebina, and A. Nakazawa. 1983. Complete nucleotide sequence of the metapyrocatechase gene on the TOL plasmid of *Pseudomonas putida* mt-2. J. Biol. Chem. 258:2923–2928.

42. Nikaido, H., and J. A. Hall. 1998. Overview of bacterial ABC transporters, p. 3–20. *In* S. V. Ambudkar and M. M. Gottesman (ed.), ABC transporters: biochemical, cellular, and molecular aspects. Academic Press Inc., San Diego, Calif.

43. Nishino, S. F., G. Paoli, and J. C. Spain. 2000. Aerobic biodegradation of 2,6-dinitrotoluene. Appl. Environ. Microbiol. 66:2139–2147.

44. Nishino, S. F., and J. C. Spain. 1996. Biodegradation and transformation of nitroaromatic compounds, p. 776–783. *In* C. J. Hurst, G. R. Knudsen, M. J. McInerney, L. D. Stetzenbach, and M. V. Walter (ed.), Manual of environmental microbiology. ASM Press, Washington, D.C.

45. Nishino, S. F., J. C. Spain, and Z. He. 2000. Strategies for aerobic degradation of nitroaromatic compounds by bacteria: process discovery to field application, p. 7–62. *In* J. C. Spain, J. B. Hughes, and H.-J. Knackmuss (ed.), Biodegradation of nitroaromatic compounds and explosives. Lewis Publishers, Boca Raton, Fla.

46. Nurk, A., L. Kasak, and M. Kivisaar. 1991. Sequence of the gene (*pheA*) encoding phenol monooxygenase from *Pseudomonas* sp. EST1001: expression in *Escherichia coli* and *Pseudomonas putida*. Gene 102:13–18.

47. Ochman, H., J. G. Lawrence, and E. A. Groisman. 2000. Lateral gene transfer and the nature of bacterial innovation. Nature 405:299–304.

48. Parales, J. V., A. Kumar, R. E. Parales, and D. T. Gibson. 1996. Cloning and sequence of the genes encoding 2-nitrotoluene dioxygenase from *Pseudomonas* sp. JS42. Gene 181:57–61.

49. Parke, D., D. A. D’argenio, and L. N. Ornston. 2000. Bacteria are not what they eat: that is why they are so diverse. J. Bacteriol. 182:257–263.

50. Parke, D., and L. N. Ornston. 1984. Nutritional diversity of *Rhizobiaceae* revealed by auxanography. J. Gen. Microbiol. 130:1743–1750.

51. Paulsen, I. T., M. K. Sliwinski, and M. H. Saier. 1998. Microbial genome analyses: global comparisons of transport capabilities based on phylogenies, bioenergetics and substrate specificities. J. Mol. Biol. 277:573–592.

52. Perkins, E. J., M. P. Gordon, O. Caceres, and P. F. Lurquin. 1990. Organization and sequence analysis of the 2,4-dichlorophenol hydroxylase and dichlorocatechol oxidative operons of plasmid pJP4. J. Bacteriol. 172:2351–2359.

53. Popp, J. A., and T. B. Leonard. 1985. The hepatocarcinogenicity of dinitrotoluenes, p. 53–60. *In* D. E. Rickert (ed.), Toxicity of nitroaromatic compounds. Hemisphere Publishing Corporation, Washington, D.C.

54. Pridmore, R. D. 1987. New and versatile cloning vectors with a kanamycin-resistance marker. Gene 56:309–312.

55. Reimann, C., and D. Haas. 1987. Mode of replication fusion mediated by the duplicated insertion sequence IS21 in *Escherichia coli*. Genetics 155:619–625.

56. Resnick, S. M., K. Lee, and D. T. Gibson. 1996. Diverse reactions catalyzed by naphthalene dioxygenase from *Pseudomonas* sp. strain NCIB 9816. J. Ind. Microbiol. 17:438–457.

57. Romine, M. F., L. C. Stillwell, K. K. Wong, S. J. Thurston, E. C. Sisk, C. Sensen, T. Gaasterland, J. K. Fredrickson, and J. D. Saffer. 1999. Complete sequence of a 184-kilobase catabolic plasmid from *Sphingomonas aromaticrovirans* F199. J. Bacteriol. 181:1585–1602.

58. Roper, D. I., and R. A. Cooper. 1993. Purification, nucleotide sequence and some properties of a bifunctional isomerase/decarboxylase from the homoprotocatechuate degradative pathway of *Escherichia coli* C. Eur. J. Biochem. 217:575–580.

59. Sambrook, J., E. F. Fritsch, and T. Maniatis. 1989. Molecular cloning: a laboratory manual, 2nd ed. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.

60. Sanger, F., S. Nicklen, and A. R. Coulson. 1977. DNA sequencing with chain-terminating inhibitors. Proc. Natl. Acad. Sci. USA 74:5463–5467.

61. Schell, M. A. 1993. Molecular biology of the LysR family of transcriptional regulators. Annu. Rev. Microbiol. 47:597–626.

62. Schell, M. A. 1990. Regulation of the naphthalene degradation genes of plasmid NAH7: example of a generalized positive control system in *Pseudomonas* and related bacteria, p. 165–176. *In* S. Silver, A. M. Chakrabarty, B. Iglewski, and S. Kaplan (ed.), *Pseudomonas*: biotransformations, pathogenesis, and evolving biotechnology. American Society for Microbiology, Washington, D.C.

63. Schell, M. A., and M. Sukordhaman. 1989. Evidence that the transcription activator encoded by the *Pseudomonas putida nahR* gene is evolutionarily related to the transcription activators encoded by the *Rhizobium nodD* genes. J. Bacteriol. 171:1952–1959.

64. Simon, M. J., T. D. Osslund, R. Saunders, B. D. Ensley, S. Suggs, A. A. Harcourt, W.-C. Suen, D. L. Cruden, D. T. Gibson, and G. J. Zylstra. 1993. Sequence of genes encoding naphthalene dioxygenase in *Pseudomonas putida* strain G7 and NCIB 9816–4. Gene 127:31–37.

65. Spanggord, R. J., J. C. Spain, S. F. Nishino, and K. E. Mortelmans. 1991. Biodegradation of 2,4-dinitrotoluene by a *Pseudomonas* sp. Appl. Environ. Microbiol. 57:3200–3205.

66. Steele, M. I., D. Lorenz, K. Hatter, A. Park, and J. R. Sokatch. 1992. Characterization of the *mmsAB* operon of *Pseudomonas aeruginosa* PAO encoding methylmalonate semialdehyde dehydrogenase and 3-hydroxyisobutyrate dehydrogenase. J. Biol. Chem. 267:13585–13592.

67. Stock, J. B., G. S. Lukat, and A. M. Stock. 1991. Bacterial chemotaxis and the molecular logic of intracellular signal transduction networks. Annu. Rev. Biophys. Chem. 20:109–136.

68. Suen, W. C., B. E. Haigler, and J. C. Spain. 1996. 2,4-Dinitrotoluene dioxygenase from *Burkholderia* sp. strain DNT: similarity to naphthalene dioxygenase. J. Bacteriol. 178:4926–4934.

69. Suen, W. C., and J. C. Spain. 1993. Cloning and characterization of *Pseudomonas* sp. strain DNT genes for 2,4-dinitrotoluene degradation. J. Bacteriol. 175:1831–1837.

70. Takami, H., T. Kudo, and K. Horikoshi. 1997. Isolation of extradiol dioxygenase genes that are phylogenetically distant from other *meta*-cleavage dioxygenase genes. Biosci. Biotechnol. Biochem. 61:530–532.

71. Takizawa, N., N. Kaida, S. Torigoe, T. Moritani, T. Sawada, S. Satoh, and H. Kiyohara. 1994. Identification and characterization of genes encoding polycyclic aromatic hydrocarbon dioxygenase and polycyclic aromatic hydrocarbon dihydrol dehydrogenase in *Pseudomonas putida* OUS82. J. Bacteriol. 176:2444–2449.

72. Takizawa, N., H. Yokoyama, K. Yanagihara, T. Hatta, and H. Kiyohara. 1995. A locus of *Pseudomonas pickettii* DTP0602, *had*, that encodes 2,4,6-trichlorophenol-4-dechlorinase with hydroxylase activity, and hydroxylation of various chlorophenols by the enzyme. J. Ferment. Bioeng. 80:318–326.

73. Tartof, K. D., and C. A. Hobbs. 1987. Improved media for growing plasmid and cosmid clones. Bethesda Res. Lab. Focus 9:12.

74. Terwilliger, T. C., J. Y. Wang, and D. E. Koshland. 1986. Surface structure recognized for covalent modification of the aspartate receptor in chemotaxis. Proc. Natl. Acad. Sci. USA 83:6707–6710.

75. Thompson, J. D., D. G. Higgins, and T. J. Gibson. 1994. CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, positions-specific gap penalties and weight matrix choice. Nucleic Acids Res. 22:4673–4680.

76. Timm, D. E., H. A. Mueller, P. Bhanumoorthy, J. M. Harp, and G. J. Bunick.

4232 JOHNSON ET AL.

1999. Crystal structure and mechanism of a carbon-carbon bond hydrolase. Struct. Des. 7:1023–1033.

77. Tsoi, T. V., E. G. Plotnikova, J. R. Cole, W. F. Guerin, M. Bagdasarian, and J. M. Tiedje. 1999. Cloning, expression, and nucleotide sequence of the *Pseudomonas aeruginosa* 142 *ohb* genes coding for oxygenolytic ortho dehalogenation of halobenzoates. Appl. Environ. Microbiol. 65:2151–2162.

78. Velasco, A., S. Alonso, J. L. Garcia, J. Perera, and E. Diaz. 1998. Genetic and functional analysis of the styrene catabolic cluster of *Pseudomonas* sp. strain Y2. J. Bacteriol. 180:1063–1071.

79. Walker, J. E., M. Saraste, and N. J. Gay. 1984. The unc operon. Nucleotide sequence, regulation and structure of ATP-synthase. Biochim. Biophys. Acta 768:164–200.

80. Westrich, L., S. Domann, B. Faust, D. Bedford, D. A. Hopwood, and A. Bechthold. 1999. Cloning and characterization of a gene cluster from *Strep-*

tomyces cyanogenus* S136 probably involved in landomycin biosynthesis. FEMS Microbiol. Lett. 170:381–387.

81. Wierenga, R. K., P. Terpstra, and W. G. J. Hol. 1986. Prediction of the occurrence of the ADP-binding βαβ-fold in proteins, using an amino acid sequence fingerprint. J. Mol. Biol. 187:101–107.

82. Yanisch-Perron, C., J. Vierra, and J. Messing. 1985. Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. Gene 33:103–119.

83. Zhou, N.-Y., S. L. Fuenmayor, and P. A. Williams. 2001. *nag* genes of *Ralstonia* (formerly *Pseudomonas*) sp. strain U2 encoding enzymes for genistein catabolism. J. Bacteriol. 183:700–708.

84. Zhu, P., G. Morelli, and M. Achtman. 1999. The *opcA* and pseudo-*opcB* regions in *Neisseria*: genes, pseudogenes, deletions, insertion elements and DNA islands. Mol. Microbiol. 33:635–650.

JOURNAL OF BACTERIOLOGY, Nov. 2002, p. 6084
0021-9193/02/$04.00+0 DOI: 10.1128/JB.184.21.6084.2002

Vol. 184, No. 21

ERRATA

Origins of the 2,4-Dinitrotoluene Pathway

Glenn R. Johnson, Rakesh K. Jain, and Jim C. Spain

Air Force Research Laboratory, U.S. Air Force, Tyndall Air Force Base, Florida 32403

Volume 184, no. 15, p. 4219–4232, 2002. Page 4222: In Fig. 2, the genes designated *dntR* and *dntF* should be labeled ORF 12 and ORF 13, respectively.

---

Processing Endoribonucleases and mRNA Degradation in Bacteria

David Kennell

Department of Molecular Microbiology, Washington University School of Medicine, St. Louis, Missouri 63110

Volume 184, no. 17, p. 4645–4657, 2002. Page 4647, column 1, lines 42–45: The sentence should read “For example, one message in the T7 early polycistronic mRNA, as well as a few late messages, yields increased protein from RNase III cleavage while the others are not affected by such cleavages (41).”
