
JOURNAL OF BACTERIOLOGY, Aug. 2002, p. 4161–4167
0021-9193/02/$04.00+0 DOI: 10.1128/JB.184.15.4161–4167.2002
Copyright © 2002, American Society for Microbiology. All Rights Reserved.

# The Putative Response Regulator BaeR Stimulates Multidrug Resistance of *Escherichia coli* via a Novel Multidrug Exporter System, MdtABC

Satoshi Nagakubo${}^{1,2}$ Kunihiko Nishino${}^{1,2,3}$ Takahiro Hirata${}^{1,2,3}$ and Akihito Yamaguchi${}^{1,2,3\ast}$

**Department of Cell Membrane Biology, Institute of Scientific and Industrial Research, Osaka University, Ibaraki, ${}^1$ and CREST, Japan Science and Technology Corporation, Osaka 567-0047, ${}^3$ and Faculty of Pharmaceutical Science, Osaka University, Suita, Osaka 565-0871, ${}^2$ Japan**

Received 19 February 2002/Accepted 29 April 2002

Overproduction of the response regulator BaeR confers resistance to novobiocin and bile salts in a $\Delta acrAB$ mutant by stimulating drug exporter gene expression. The *mdtABC* (multidrug transporter ABC, formerly known as *yegMNO*) genes, which encode a resistance-nodulation-cell division (RND) drug efflux system, are responsible for resistance. The MdtABC system comprises the transmembrane MdtB/MdtC heteromultimer and MdtA membrane fusion protein. MdtAC also confers bile salt, but not novobiocin, resistance. This indicates that the evolution from an MdtC homomultimer to an MdtBC heteromultimer contributed to extend the drug resistance spectrum. A BLAST search suggested that such a heteromultimer-type RND exporter constitutes a unique family among gram-negative organisms.

The emergence of bacterial multidrug resistance has become an increasing problem in the treatment of infectious diseases. Multidrug resistance often results from the overexpression of multidrug efflux transporters (19). Recent genome sequence analysis revealed that bacteria have a lot of intrinsic drug exporter genes (20). We previously cloned all of the open reading frame (ORF) clusters encoding putative drug exporter genes in *Escherichia coli* and revealed that 20 genes encode exporters of known drugs and/or toxic compounds (16). During the course of these comprehensive studies, we found that the overexpression of a response regulator of a bacterial two-component signal transduction system confers multidrug resistance by stimulating the expression of multidrug exporter genes (15, 17).

Two-component signal transduction systems are major environmental sensing mechanisms of bacteria and are a component of sensor kinases and response regulators (2, 11). The *vanS* and *vanR* genes encode an *Enterococcus* two-component system which confers vancomycin resistance via upregulation of *vanA* and *vanH*, which make an altered peptidoglycan precursor to which vancomycin does not bind (3, 6). Similar two-component-system-mediated vancomycin tolerance was reported to occur in *Streptococcus pneumoniae* (18). Recently, it was found that some two-component systems confer drug resistance by upregulating drug exporter genes (4, 9, 15, 17). Since two-component systems are a mechanism for bacterial environmental adaptation and intrinsic drug exporters are a bacterial self-defense mechanism (23, 24), it is reasonable that some two-component systems could control drug exporter genes.

In previous papers, Nishino and Yamaguchi reported a mechanism for *E. coli* to express multidrug resistance by overexpression of the response regulator EvgA, which stimulates the expression of novel multidrug exporters (15, 17). EvgA regulates the expression of *emrKY* (7, 15), which encodes a major facilitator superfamily (MFS)-type bile salt-specific exporter, and *yhiUV* (17), which encodes a resistance-nodulation-cell division (RND)-type multidrug exporter. In this study, we characterize a novel multiple-membrane component RND-type drug transporter system, MdtABC, which comprises a transmembrane MdtB/MdtC heteromultimer and the membrane fusion protein MdtA. This system is also stimulated by overproduction of the response regulator BaeR.

## MATERIALS AND METHODS

### Bacterial strains and plasmids.
The bacterial strains and plasmids used in this study are listed in Table 1. The chromosomal DNA of *E. coli* W3104 (25) was used as a template for PCR cloning of the *mdt-bae* ORF clusters. *E. coli* DH5$\alpha$ (Takara Shuzo Co., Kyoto, Japan) was used as a cloning host. *E. coli* KAM3 (12), a derivative of *E. coli* TG1 that lacks *acrAB*, and *E. coli* TG1$\Delta$tolC, which lacks *tolC*, were used for drug susceptibility testing. *E. coli* KO6494 was constructed for this study from *E. coli* KAM3 by means of random knockout of the genes by Mu d1 phage integration. Plasmids pUC119, pHSG398, pQE30, pTrc99A, and pACYC177 were purchased from commercial sources. Other plasmids were constructed in this study.

### Subcloning and expression of individual ORFs in the *mdt-bae* ORF cluster.
The *mdt-bae* ORF cluster was previously cloned from the chromosomal DNA of *E. coli* W3104 in our laboratory (16). Each ORF or ORF pair was amplified from pUCyeq-mdt-bae (16) by PCR with a pair of primers containing a restriction enzyme site that exists in the multicloning sites of the pUC119 and pQE30 vectors. The DNA fragments were digested with restriction enzymes and then ligated into the multicloning sites of pUC119 and pQE30. As for *mdtABCD*, the chromosomal DNA of *E. coli* W3104 was digested by restriction enzymes *PstI* and *SmaI* followed by separation by agarose gel electrophoresis. The DNA fragments of around 9.5 kb were extracted from the gel and ligated into the multicloning site of pUC119 followed by transformation of *E. coli* KAM3. The colonies that carried the *mdtABCD* genes were detected by colony hybridization, with the PCR fragment of *mdtC* used as a probe. The PCR fragment was obtained by amplification of the *mdtC* gene from the *E. coli* W3104 chromosome.

* Corresponding author. Mailing address: Institute of Scientific and Industrial Research, Osaka University, 8-1 Mihogaoka, Ibaraki-shi, Osaka 567-0047, Japan. Phone: 81-6-6879-8545. Fax: 81-6-6879-8549. E-mail: akihito@sanken.osaka-u.ac.jp.

TABLE 1. Bacterial strains and plasmids used in this study

| Strain or plasmid | Relevant phenotype or genotype | Reference or origin |
|-------------------|---------------------------------|---------------------|
| **E. coli strains** |  |  |
| W3104 | Wild-type strain | 25 |
| DH5α | supE44 ΔlacU169 (φ80lacZΔM15) hsdR17 recA1 gyrA96 thi-1 recA1 | Takara Shuzo Co. |
| TG1 | supE hsdΔ5 thiΔ(lac-proAB) F' [traD36 proAB⁺ lacIq lacZΔM15] | Takara Shuzo Co. |
| KAM3 | Derivative of TG1 that lacks a restriction system and acrAB | 12 |
| KO6494 | Derivative of KAM3 that lacks mdtB (formerly yegN) | This study |
| TG1ΔtolC | Derivative of TG1 that lacks tolC | This study |
| **Plasmids** |  |  |
| pUC119 | Vector; Ap<sup>r</sup>; multiple cloning site in lacZ | Takara Shuzo Co. |
| pQE30 | His expression vector; Ap<sup>r</sup>; multiple cloning site downstream of T5 promoter | Qiagen |
| pHSG398 | Derivative of pUC18 containing Cm<sup>r</sup> in place of Ap<sup>r</sup>. | Takara Shuzo Co. |
| pUCyeg-mdt-bae | 14.2-kb SphI-EcoRI fragment containing yegKL (putative regulatory system), mdtABC (former name, yegMNO; putative MFP, putative RND transporter, putative RND transporter), mdtD (putative MF transporter), and baeSR (putative two-component system) genes cloned into pUC119, Ap<sup>r</sup> | 16 |
| pUCmdtABCD | 9.5-kb PstI-SmaI fragment containing mdtABCD genes into pUC119, Ap<sup>r</sup> | This study |
| pUCmdtABC | BglII-SmaI fragment was removed from pUCmdtABCD | This study |
| pUCyegLK | yegLK genes with the promoter region were amplified from pUCyeg-mdt-bae by PCR and cloned into pUC119 | This study |
| pUCmdtA | mdtA gene with the native promoter was cloned into pUC119 as above | This study |
| pUCmdtAB | mdtAB genes with the native promoter were cloned into pUC119 as above | This study |
| pUCbaeSR | baeSR genes with the native promoter were cloned into pUC119 as above | This study |
| pHSGbaeSR | SphI-EcoRI fragment of pUCbaeSR was subcloned into the multicloning site of pHSG398 | This study |
| pQEbaeS | baeS gene was amplified by PCR from pUCbaeSR and cloned into pQE30 under the control of T5 promoter | This study |
| pQEbaeR | baeR gene was cloned into pQE30 as above | This study |
| pTrc99A | Expression vector; multiple cloning site downstream of trc promoter, Ap<sup>r</sup> | Amersham Pharmacia Biotech |
| pTrcmdtC | 3.1-kb EcoRI-PstI fragment containing mdtC gene cloned into pTrc99A, Ap<sup>r</sup> | This study |
| pTrcmdtBC | 6.2-kb EcoRI-PstI fragment containing mdtB and mdtC genes cloned into pTrc99A, Ap<sup>r</sup> | This study |
| pACYC177 | Vector; Ap<sup>r</sup>, Km<sup>r</sup> | MBI Fermentas |
| pACYCmdtA | 2.9-kb PstI-BamHI fragment containing mdtA gene cloned into pACYC177, Km<sup>r</sup> | This study |

with primers mdtC-F (CCGAATTCAAGTTTTTGGCCCTCTTCATTT) and mdtC-R (GGCTGCAGCTCGGTTACC GTTTGT TTAGGT). After purification, the PCR product was labeled with a digoxigenin labeling kit (Takara Shuzo Co.). The resulting cells carried plasmids encoding mdtABCD and the putative promoter region. pUCmdtABC was constructed from pUCmdtABCD by deletion of the BglII-SmaI fragment. The resulting plasmid lacked the mdtD gene.

Random knockout and selection of strains lacking the BaeR-responsive gene. Random knockout of the E. coli KAM3 chromosomal genes was performed by the method of Mud1 (Ap<sup>r</sup>) phage integration as described previously (22). In brief, KAM3 cells were infected by Mud1 phage followed by selection with 50 μg of ampicillin/ml. The phage-integrated KAM3 cells were then transformed with pHSGbaeSR (Cm<sup>r</sup>). The ampicillin- and chloramphenicol-resistant clones were isolated on agar plates containing 150 μg of ampicillin/ml and 10 μg of chloramphenicol/ml. Of these clones, the strains that were sensitive to 4 μg of novobiocin/ml were screened. The Mud1 phage integration site was then determined as follows. In order to construct the plasmid library of this strain, the genomic DNA of E. coli KO6494 was subjected to partial Sau3AI digestion and then ligated into the pUC118 vector that had been digested with BamHI and treated with alkaline phosphatase. From this plasmid library, the plasmids containing Mud1 DNA fragments were selected by colony hybridization with a PCR fragment of Mud1 DNA as a probe. This PCR fragment was obtained from Mud1 DNA by using primers MuF (GGTTGTGGTTAATTGT TTTATCA) and MuR (GG TAAATTCCTTTGATTACTGAT). The PCR product was labeled with a digoxigenin labeling kit (Takara Shuzo Co.). The plasmid obtained, which contained the Mud1 fragment, was sequenced by using a BigDye DNA sequencing kit (PE Applied Biosystems) with M13, M4, and M13RV sequencing primers (Takara Shuzo Co.).

Construction of tolC in-frame deletion mutants. To construct the tolC deletion mutant from E. coli TG1 cells, the precise in-frame deletions were generated by crossover PCR. The following oligonucleotide primers were used: tolC-No (CGCGGATCCTCATCCC GGCAACCATCTC), tolC-Ni (CACGCAATAAC CTTCACACTCCA AATTTATAACC ATTCCTGTGGTGAA GCA GTAT), tolC-Co (CGCGGATCCGCTGGATTGCTGGGCC), and tolC-Ci, (GTTATA AATTTGGAGTGTGAAGGTTATTGCGTGTGATGACGACGACGGGGG). The fragment containing the deletion was then cloned into the BamHI site

(underlined in the primer sequences) of the pKO3 vector (10), which is a gene replacement vector that contains a temperature-sensitive origin of replication and markers for positive and negative selection for chromosomal integration and excision. The deletion was introduced into the chromosome by use of the pKO3 gene replacement protocol as described previously (10). The plasmid obtained was then electroporated into TG1. Cells were then recovered in 1 ml of SOC (2% Bacto Tryptone, 0.5% yeast extract, 10 mM NaCl, 2.5 mM KCl, 10 mM MgCl₂, 20 mM glucose) for 1 h at 30°C. The cells were then plated on prewarmed chloramphenicol (20 μg/ml)-containing Luria broth (LB) plates and incubated at 43°C. From these plates, five colonies were picked up and inoculated into 1 ml of LB. After that, the cells were plated at 30°C on 5% (wt/vol) sucrose plates. The outgrowing bacteria were plated on LB plates with or without chloramphenicol at 30°C. Chloramphenicol-sensitive mutants were selected. Chromosomal insertions and deletions were confirmed by PCR.

Drug resistance determination. The MICs of drugs and toxic compounds were determined as concentrations that largely prevented bacterial growth on L-agar (1% tryptone, 0.5% yeast extract, 0.5% NaCl) plates by sequential twofold dilutions as described previously (16).

Determination of the amount of mRNA by quantitative real-time PCR. Total RNA was purified from KAM3 cells harboring pUC119 and pUCbaeSR by using RNA Protect bacterial reagent (Qiagen) and an SV total RNA isolation system (Promega). cDNA samples were synthesized from total RNA by using TaqMan reverse transcription reagents (PE Applied Biosystems) and random nucleotide hexamers. Then, specific primer pairs were designed with ABI PRISM Primer Express software (PE Applied Biosystems), and a real-time PCR was performed with each specific primer pair by using SYBR Green PCR Master Mix and run on an ABI PRISM 7000 sequence detection system.

---

RESULTS

ORF clusters responsible for the drug resistance phenotype of the genomic mdt and bae region. The genomic yeg, mdt, and bae region, which is located at 46.5 to 46.6 min in the E. coli

VOL. 184, 2002                                                                 BaeR STIMULATES E. COLI MULTIDRUG RESISTANCE 4163

![Diagram](https://i.imgur.com/yourimage.png)

FIG. 1. Physical map of the yeg-mdt-bae region located at 46.5 to 46.6 min in the *E. coli* chromosome (20a) and the regions cloned into plasmids. The drug resistance of cells carrying these plasmids is shown at the right. The Mu d1 phage insertion position of *E. coli* KO6494 is depicted.

chromosome and contains eight ORFs (Fig. 1) (20a), when cloned into a multicopy vector, gave *E. coli* KAM3 (∆acrAB) cells a drug resistance phenotype against bile salt derivatives, sodium dodecyl sulfate (SDS), and novobiocin. The MICs of deoxycholate, cholate, taurocholate, SDS, and novobiocin were increased by factors of 64, 8, >4, 4, and 8, respectively (Table 2). The MICs of 17 other drugs and toxic compounds (tetracycline, chloramphenicol, minocycline, erythromycin, enoxacin, kanamycin, vancomycin, doxorubicin, rifampin, trimethoprim, acriflavine, crystal violet, ethidium bromide, rhodamine 6G, methylviologen, tetraphenylphosphonium bromide, and carbonyl cyanide m-chlorophenylhydrazone) were unaffected. This region contains some putative drug exporter genes (16). According to the sequence similarities, *mdtB* and *mdtC* were suggested to encode RND-type drug exporters and *mdtA* was postulated to be a membrane fusion protein gene. In addition, *mdtD* is a putative MFS-type drug exporter gene. On the other hand, *yegK* and *yegL* are putative regulator genes, and *baeS* and *baeR* encode a putative sensor kinase and a response regulator, respectively, in a two-component signal transduction system (13). In order to determine which ORFs are responsible for the drug resistance phenotype, we subcloned the ORF clusters into the multicopy plasmid pUC119. As shown in Table 2, a plasmid carrying *mdtABCD* conferred the same drug resistance phenotype as a plasmid carrying the entire region. The genes *mdtABC* also showed the same resistance phenotype, while *mdtAB* and *mdtA* did not. The putative regulator genes, *yegLK*, conferred no resistance. Surprisingly, the *baeSR* two-component regulator genes gave the same resistance phenotype as *mdtABC*. The *baeR* gene alone also conferred the same resistance, while *baeS* alone conferred no resistance, indicating that overexpression of the response regulator is enough for the resistance phenotype. This result is similar to that obtained with *evgA* reported previously (15).

Genes responsible for *baeR*-modulated drug resistance. In order to determine the genes modulated by *baeR* and responsible for drug resistance, the chromosomal genes of *E. coli* KAM3 were randomly knocked out by transposition of Mu d1.

TABLE 2. Drug resistance of *E. coli* KAM3 (∆acrAB) and KO6494 [∆acrAB *mdtB*::Ap<sup>r</sup> (Mu d1)] cells harboring a pUC119 or pHSG398 plasmid carrying an ORF cluster(s) in the genomic yeg, *mdt*, and *bae* region<sup>a</sup>

| Drug                | None     | yeg/mdt/bae<sup>c</sup> | yegLK   | *mdtA*    | *mdtAB*   | *mdtABC*  | *mdtABCD* | *baeSR*   | *baeS*    | *baeR*    | None      | *baeSR*    |
|---------------------|----------|-------------------------|---------|-----------|-----------|-----------|-----------|-----------|-----------|-----------|------------|------------|
| Deoxycholate        | 1,000    | 64,000                 | 1,000   | 1,000     | 1,000     | 64,000    | 64,000    | 64,000    | 1,000     | >64,000   | 1,000      | 1,000      |
| Cholate             | 8,000    | 64,000                 | 8,000   | 8,000     | 8,000     | 64,000    | 64,000    | 64,000    | 8,000     | 64,000    | 8,000      | 8,000      |
| Taurocholate        | 16,000   | >64,000                | 16,000  | 16,000    | 16,000    | >64,000   | >64,000   | >64,000   | 16,000    | 64,000    | 16,000     | 16,000     |
| SDS                 | 64       | 256                    | 64      | 64        | 64        | 128       | 256       | 256       | 64        | 256       | 64         | 64         |
| Novobiocin          | 1        | 8                      | 1       | 1         | 1         | 8         | 8         | 8         | 1         | 8         | 1          | 1          |

<sup>a</sup> MICs more than fourfold higher than those against host KAM3 cells are indicated by boldface. MICs of 17 different drugs and toxic compounds other than those listed here (tetracycline, chloramphenicol, minocycline, erythromycin, enoxacin, kanamycin, vancomycin, doxorubicin, rifampin, trimethoprim, acriflavine, crystal violet, ethidium bromide, rhodamine 6G, methylviologen, tetraphenylphosphonium bromide, and carbonyl cyanide m-chlorophenylhydrazone) were measured. However, no change in MIC was observed when the cells were transformed with any plasmid carrying ORFs listed above.

<sup>b</sup> KO6494 is a Mu d1 phage-integrated *mdtB*-disrupted derivative of KAM3.

<sup>c</sup> yeg/mdt/bae contains *yegKL*, *mdtABCD* (*yegMNOB*), and *baeSR* genes as shown in Fig. 1.

loss of the drug resistance phenotype, even when the strain carried the multicopy *baeSR* genes (pHSGbaeSR).

Expression control of *mdtABCD* genes by BaeR. In order to determine whether the expression of the *mdtABC* genes is controlled by BaeR, we performed Northern blot analysis with a DNA fragment containing the *mdtC* region as a probe. Total cellular RNA isolated from KAM3 cells carrying pUCbaeSR showed a radioactive band corresponding to the *mdtC* mRNA, whereas KAM3 cells showed no such band (data not shown). The size of the radioactive band was 13 kb, which covers the entire *mdt-bae* locus, suggesting that these six genes (*mdtABCD* and *baeSR*) are transcribed as one operon (Fig. 1).

The amounts of mRNAs of individual genes of the *mdtABCD* region and of other multidrug exporter genes were determined by quantitative reverse transcription-PCR assay. As shown in Fig. 2, BaeR overexpression increased the amounts of *mdtA*, *mdtB*, *mdtC*, and *mdtD* mRNA by factors of about 7.4, 6.6, 4.5, and 2.8, respectively. The order of these values is consistent with the transcription of *mdtABCD* from the same promoter of the *mdt-bae* operon under the control of BaeR. On the other hand, the expression of the *yhiUV*, *emrA*, *emrE*, *macA*, and *ydhE* genes was not affected by BaeR overexpression.

MdtABC is a novel RND-type drug exporter complex with unique subunit composition. To determine whether any single transporter gene can confer drug resistance, the *mdtA*, *mdtB*, and *mdtC* genes were separately cloned into pACYC177 and pTrc99A. None of these plasmids gave any drug resistance to KAM3 cells, except for the very-low-level deoxycholate and SDS resistance of pTrcmdtC (Table 3). The effect of combinations of the transporter genes, *mdtB* and *mdtC*, and/or the membrane fusion protein gene, *mdtA*, on drug resistance was checked (Table 3). The pair of pACYCmdtA and pTrcmdtBC conferred the same resistance pattern as pUCmdtABC, except that the deoxycholate resistance was slightly low. On the other hand, the drug resistance pattern of the combination of pACYCmdtA and pTrcmdtC was unique; that is, the combination conferred resistance only to deoxycholate and its derivatives and not to novobiocin or SDS at all (Table 3). Since pUCmdtAB conferred no resistance (Table 2), there is no possibility that MdtABC is a mixture of independent drug exporters, i.e., bile salt-specific MdtAC and novobiocin-specific MdtAB. In light of the high (45%) amino acid sequence similarity of MdtB and MdtC, a complex of MdtB and MdtC is likely to be replaced with an MdtC homomultimer in the absence of MdtB.

FIG. 2. Fold induction of transcripts attributed to *baeR* amplification as determined by amplification of cDNA samples from *E. coli* KAM3/pUCbaeSR (for details, see Materials and Methods).

phage, which carries an ampicillin resistance gene as a marker. A mixture of cells with random insertions was transformed with pHSGbaeSR, which is a pHSG398 plasmid carrying *baeSR* genes and a chloramphenicol resistance gene as a marker. We obtained 7,200 clones of phage-integrated-plasmid-carrying strains by selection for ampicillin and chloramphenicol resistance. These clones were transplanted on the agar plate containing 4 μg of novobiocin/ml, and we obtained one novobiocin-sensitive strain, KO6494. The phage integration site, which was determined as described in Materials and Methods, was in the *mdtB* gene (data not shown). The MICs of bile salt derivatives, SDS, and novobiocin against strain KO6494 [Δ*acrAB mdtB::Ap<sup>r</sup>(Mu d1)] were the same as those against the host KAM3 cells (Table 2). Thus, phage Mu d1 insertion in the *mdtB* gene of strain KO6494 resulted in the

TABLE 3. Drug resistance of *E. coli* KAM3 cells harboring combinations of plasmids carrying *mdtA* and *mdtBC*

| Drug           | pTrcmdtC       | pACYCmdtA and pTrcmdtC      | pTrcmdtBC        | pACYCmdtA and pTrcmdtBC     |
|----------------|----------------|-----------------------------|------------------|-----------------------------|
|                | -IPTG          | +IPTG                       | -IPTG            | +IPTG                       | -IPTG            | +IPTG                       |
| Deoxycholate   | 2,000          | 2,000                       | 8,000            | 8,000                       | 1,000            | 2,000                       | 4,000            | 32,000                      |
| Cholate        | 16,000         | 16,000                      | 32,000           | 64,000                      | 8,000            | 16,000                      | 16,000           | 64,000                      |
| Taurocholate   | 32,000         | 32,000                      | 32,000           | 64,000                      | 32,000           | 32,000                      | 64,000           | >64,000                     |
| SDS            | 64             | 64                          | 64               | 64                          | 64               | 64                          | 64               | 128                         |
| Novobiocin     | 1              | 1                           | 1                | 1                           | 1                | 2                           | 1                | 8                           |

<sup>a</sup> MICs more than fourfold higher than those against host KAM3 cells are indicated by boldface. The gene expression from pTrc plasmids was induced with 10 μM IPTG (isopropyl-β-D-thiogalactopyranoside).

(A)

| TM1 | 2 3 4 | 5 6 | TM7 | TM | 8 9 10 | 11 12 |
| --- | --- | --- | --- | --- | --- | --- |
| hydrophathy |  |  |  |  |  |  |

number of amino acids

(B)

| TM1 | 2 3 4 | 5 6 | TM7 | TM | 8 9 10 | 11 12 |
| --- | --- | --- | --- | --- | --- | --- |
| hydrophathy |  |  |  |  |  |  |

number of amino acids

(C)

Deoxycholate  
Cholate  
Taurocholate  
SDS  
Novobiocin  

TolC  

Outer  
Membrane  

MdtA  

BaeS  

H₂N HOOC NH₂ COOH  
MdtB MdtC  

Inner  
Membrane  

BaeR  
Activation  

mdtA mdtB mdtC mdtD  

Promoter of mdt operon  

FIG. 3. Hydropathy profiles of MdtB (A) and MdtC (B). Putative transmembrane segments are indicated. (C) Schematic model of the molecular construction of MdtABC-TolC transporter complex and the expression regulation.

sence of MdtB. Thus, it seems that the complex of MdtC homomultimer and MdtA may have narrower drug specificity than the complex of MdtB/MdtC heteromultimer and MdtA. The very low resistance shown by pTrcmdtC and pTrcmdtBC alone may be caused by a trace amount of MdtA from leaky expression of the chromosomal gene in E. coli KAM3. Finally, we found that pUCmdtABC did not confer any resistance to E. coli TG1 cells from which the tolC gene was

deleted (data not shown), indicating that TolC is required for the resistance phenotype of MdtABC (Fig. 3).

# DISCUSSION

In this study, we showed that overexpression of the response regulator BaeR stimulates the drug resistance phenotype by upregulating the expression of a novel RND-type drug exporter, MdtABC. This is the second case in *E. coli* of response regulator overexpression-stimulated drug resistance. The first one involves EvgA, which regulates drug exporter genes (7, 15). The *evgAS* operon is upstream of the *evgA*-controlled *emrKY* operon, which encodes an MFS-type bile salt-specific drug exporter. These operons are divergently transcribed (15). *EvgAS* also controls the remote genes *yhiUV*, which encode an RND-type multidrug exporter conferring the multidrug phenotype of *evgA*-overexpressing strains (17). On the other hand, the *baeSR* genes are downstream of the *mdtABC* genes and are transcribed in the same direction. *baeSR* was first reported by Nagasawa et al. (13) as a novel two-component signal transduction system, but neither BaeR-controlled genes nor signals sensed by BaeS had been known. We found in this study that overexpression of BaeR upregulates the expression of MdtABC. The *baeSR*-mediated resistance can be assigned to *mdtABC*. Recently, Li et al. (9) reported that the two-component signal transduction system SmeSR in *Stenotrophomonas maltophilia* controls the *smeABC* genes, which encode a putative RND-type transporter system, and confers multidrug resistance. However, the gene responsible for *smeSR*-mediated resistance is *smeC* and not *smeAB*. Since SmeC is an outer membrane channel protein like *E. coli* TolC, some other genes encoding an inner membrane transporter may cooperate with SmeC.

Neither MdtA, MdtB, nor MdtC provided drug resistance when overexpressed individually. MdtAB also did not confer drug resistance, but MdtAC conferred limited resistance against bile salt derivatives. MdtABC conferred resistance against novobiocin and bile salt derivatives. These results strongly suggest that the transmembrane functional unit is an MdtB/MdtC heteromultimer and that MdtB contributes to extend the substrate specificity of the transporter to novobiocin. From an evolutionary point of view, MdtB might be derived from MdtC and the transporter might gain the extension of the substrate range by the replacement of the ancestral MdtC homomultimer with the MdtB/MdtC heteromultimer. Such transporters with heteromultimer-type transmembrane subunits are unique as RND-type drug exporters, but they are common in other bacterial multicomponent substrate transporters, such as maltose transporter (14) and phosphotransferase systems (21). We found that MdtABC homologues are present in *Pseudomonas aeruginosa* and *Xylella fastidiosa*. PA2526, PA2527, and PA2528 of *P. aeruginosa* showed sequence identities with MdtC, MdtB, and MdtA of about 56, 62, and 45%, respectively. XF2384, XF2385, and XF2386 of *X. fastidiosa* showed identities with MdtA, MdtC, and MdtB of about 37, 46, and 40%, respectively. Although none of these putative RND-type transporters has been analyzed yet, they may constitute a new type subfamily of RND-type drug efflux transporters.

MdtABC requires a multifunctional outer membrane channel TolC for its function, much like YhiUV (17), EmrKY (17), MacAB (8), and AcrAB (5). Of these, MdtABC, YhiUV, and AcrAB have the RND-type inner membrane transporters MdtB/MdtC, YhiV, and AcrB, respectively. On the other hand, EmrKY and MacAB have different types of inner membrane subunit, EmrY and MacB, which are MFS- and ABC (ATP binding cassette family)-type transporters, respectively. The common features of these transport systems are the presence of their own membrane fusion proteins. Thus, it seems that the membrane fusion protein-dependent drug exporter systems in *E. coli* generally require TolC as an outer membrane subunit, irrespective of the types of their inner membrane proteins. These results suggest that the TolC dependence may be determined by a membrane fusion protein.

It is likely that the two-component signal transduction system-mediated upregulation of the expression of intrinsic drug exporter genes will become a source of multidrug resistance in pathogens.

## ACKNOWLEDGMENTS

Satoshi Nagakubo and Kunihiro Nishino contributed equally to this work.

We thank Tomofusa Tsuchiya for strain KAM3 and George M. Church for plasmid pKO3. We thank Hiroshi Nikaido for helpful discussions. We thank Mary Berlyn and Kenneth Rudd for suggestions on genetic nomenclature.

K. Nishino is supported by a research fellowship from the Japan Society for the Promotion of Science for Young Scientists. This work was supported by grants-in-aid from the Ministry of Education, Culture, Sports, Science and Technology of Japan.

## ADDENDUM

After submitting this report, we became aware that the report of Baranova and Nikaido (1), which deals with the same subject, was submitted to this journal at almost the same time. The results of their study are consistent with ours, except for the limited resistance of MdtAC against deoxycholate and SDS. The lack of resistance of MdtAC-carrying cells against deoxycholate reported in their study may be due to the absence of the translation products. Their results showing BaeR binding to the *mdtABCD* promoter region also support our conclusion that BaeR is a transcriptional regulator of the *mdtABCD* operon.

## REFERENCES

1. Baranova, N., and H. Nikaido. 2002. The BaeSR two-component regulatory system activates transcription of the *yegMNOB (mdtABCD)* transporter gene cluster in *Escherichia coli* and increases its resistance to novobiocin and deoxycholate. J. Bacteriol. **184**:4168–4176.
2. Coote, J. G. 2001. Environmental sensing mechanisms in *Bordetella*. Adv. Microb. Physiol. **44**:141–181.
3. Evers, S., and P. Courvalin. 1996. Regulation of VanB-type vancomycin resistance gene expression by the VanS<sub>B</sub>-VanR<sub>B</sub> two-component regulatory system in *Enterococcus faecalis* V583. J. Bacteriol. **178**:1302–1309.
4. Fournier, B., R. Aras, and D. C. Hooper. 2000. Expression of the multidrug resistance transporter NorA from *Staphylococcus aureus* is modified by a two-component regulatory system. J. Bacteriol. **182**:664–671.
5. Fralick, J. A. 1996. Evidence that TolC is required for functioning of the Mar/AcrAB efflux pump of *Escherichia coli*. J. Bacteriol. **178**:5803–5805.
6. Haldimann, A., S. L. Fisher, L. L. Daniels, C. T. Walsh, and B. L. Wanner. 1997. Transcriptional regulation of the *Enterococcus faecium* BM4147 vancomycin resistance gene cluster by the VanS-VanR two-component regulatory system in *Escherichia coli* K-12. J. Bacteriol. **179**:5903–5913.
7. Kato, A., H. Ohnishi, K. Yamamoto, E. Furuta, H. Tanabe, and R. Utsumi. 2000. Transcription of *emrKY* is regulated by the EvgA-EvgS two-component system in *Escherichia coli* K-12. Biosci. Biotechnol. Biochem. **64**:1203–1209.

8. Kobayashi, N., K. Nishino, and A. Yamaguchi. 2001. Novel macrolide-specific ABC-type efflux transporter in *Escherichia coli*. J. Bacteriol. 183:5639–5644.

9. Li, X.-Z., L. Zhang, and K. Poole. 2002. SmeC, an outer membrane multidrug efflux protein of *Stenotrophomonas maltophilia*. Antimicrob. Agents Chemother. 46:333–343.

10. Link, A. J., D. Phillips, and G. M. Church. 1997. Methods for generating precise deletions and insertions in the genome of wild-type *Escherichia coli*: application to open reading frame characterization. J. Bacteriol. 179:6228–6237.

11. Mizuno, T. 1998. His-Asp phosphotransfer signal transduction. J. Biochem. (Tokyo) 123:555–563.

12. Morita, Y., K. Kodama, S. Shiota, T. Mine, A. Kataoka, T. Mizushima, and T. Tsuchiya. 1998. NorM, a putative multidrug efflux protein, of *Vibrio parahaemolyticus* and its homolog in *Escherichia coli*. Antimicrob. Agents Chemother. 42:1778–1782.

13. Nagasawa, S., K. Ishige, and T. Mizuno. 1993. Novel members of the two-component signal transduction genes in *Escherichia coli*. J. Biochem. 114:350–357.

14. Nikaido, H. 1994. Maltose transport system of *Escherichia coli*: an ABC-type transporter. FEBS Lett. 346:55–58.

15. Nishino, K., and A. Yamaguchi. 2001. Overexpression of the response regulator *evgA* of the two-component signal transduction system modulates multidrug resistance conferred by multidrug resistance transporters. J. Bacteriol. 183:1455–1458.

16. Nishino, K., and A. Yamaguchi. 2001. Analysis of a complete library of putative drug transporter genes in *Escherichia coli*. J. Bacteriol. 183:5803–5812.

17. Nishino, K., and A. Yamaguchi. 2002. EvgA of the two-component signal transduction system modulates production of the YhiUV multidrug transporter in *Escherichia coli*. J. Bacteriol. 184:2319–2323.

18. Novak, R., B. Henriques, E. Charpentier, S. Normark, and E. Tuomansen. 1999. Emergence of vancomycin tolerance in *Streptococcus pneumoniae*. Nature 399:590–593.

19. Okusu, H., D. Ma, and H. Nikaido. 1996. AcrAB efflux pump plays a major role in the antibiotic resistance phenotype of *Escherichia coli* multiple-antibiotic-resistance (Mar) mutants. J. Bacteriol. 178:306–308.

20. Paulsen, I. T., L. Nguyen, M. K. Sliwinski, R. Rabus, and M. H. Saier. 2000. Microbial genome analyses: comparative transport capabilities in eighteen prokaryotes. J. Mol. Biol. 301:75–100.

20a. Rudd, K. E. 1998. Linkage map of *Escherichia coli* K-12, edition 10: the physical map. Microbiol. Mol. Biol. Rev. 62:985–1019.

21. Siebold, C., K. Flukiger, R. Beutler, and B. Erni. 2001. Carbohydrate transporters of the bacterial phosphoenolpyruvate: sugar phosphotransferase system (PTS). FEBS Lett. 504:104–111.

22. Silhavy, T. J., M. L. Berman, and L. W. Enquist. 1984. Experiments with gene fusions. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.

23. Thanassi, D. G., L. W. Cheng, and H. Nikaido. 1997. Active efflux of bile salts by *Escherichia coli*. J. Bacteriol. 179:2512–2518.

24. White, D. G., J. D. Goldman, B. Demple, and S. B. Levy. 1997. Role of the *acrAB* locus in organic solvent tolerance mediated by expression of *marA*, *soxS*, or *robA* in *Escherichia coli*. J. Bacteriol. 179:6122–6126.

25. Yamamoto, T., M. Tanaka, C. Nohara, Y. Fukunaga, and S. Yamagishi. 1981. Transposition of the oxacillin-hydrolyzing penicillinase gene. J. Bacteriol. 145:808–813.
