
THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2002 by The American Society for Biochemistry and Molecular Biology, Inc.

**A Single Enzyme Catalyses Formation of Trypanothione from Glutathione and Spermidine in *Trypanosoma cruzi***  

Received for publication, May 6, 2002, and in revised form, July 9, 2002  
Published, JBC Papers in Press, July 16, 2002, DOI 10.1074/jbc.M204403200

Sandra L. Oza‡, Emmanuel Tetaud‡§, Mark R. Ariyanayagam, Stéphanie S. Warnon¶,  
and Alan H. Fairlamb||  

*From the School of Life Sciences, The Wellcome Trust Biocentre, University of Dundee, Dundee DD1 5EH, Scotland, United Kingdom*

Protozoa of the order Kinetoplastida differ from other organisms in their ability to conjugate glutathione (L-γ-glutamyl-cysteinyl-glycine) and spermidine to form trypanothione [N¹,N⁸-bis(glutathionyl)spermidine], a metabolite involved in defense against chemical and oxidant stress and other biosynthetic functions. In *Crithidia fasciculata*, trypanothione is synthesized from GSH and spermidine via the intermediate glutathionylspermidine in two distinct ATP-dependent reactions catalyzed by glutathionylspermidine synthetase (GspS; EC 6.3.1.8) and trypanothione synthetase (TryS; EC 6.3.1.9), respectively. Here we have cloned a single copy gene (*TcTryS*) from *Trypanosoma cruzi* encoding a protein with 61% sequence identity with *CfTryS* but only 31% with *CfGspS*. *Saccharomyces cerevisiae* transformed with *TcTryS* were able to synthesize glutathionylspermidine and trypanothione, suggesting that this enzyme is able to catalyze both biosynthetic steps, unlike *CfTryS*. When cultures were supplemented with aminopropylcadaverine, yeast transformants contained glutathionylaminopropylcadaverine and homotrypanothione [N¹,N⁹-bis(glutathionyl)aminopropylcadaverine], metabolites that have been previously identified in *T. cruzi*, but not in *C. fasciculata*. Kinetic studies on recombinant *TcTryS* purified from *Escherichia coli* revealed that the enzyme displays high substrate inhibition with glutathione (*K*~m~ and *K*~i~ of 0.57 and 1.2 mM, respectively, and *k*~cat~ of 3.4 s⁻¹), but obeys Michaelis-Menten kinetics with spermidine, aminopropylcadaverine, glutathionylspermidine, and MgATP as variable substrate. The recombinant enzyme possesses weak amidase activity and can hydrolyze trypanothione, homotrypanothione, or glutathionylspermidine to glutathione and the corresponding polyamine.

Polyamines and glutathione (L-γ-glutamyl-cysteinyl-glycine, GSH) are found in millimolar concentrations in most biological systems. Polyamines are thought to be essential for cell proliferation and differentiation in most organisms (1–3). GSH has a role in several important biochemical processes, including regulation of intracellular thiol-redox balance and defense against damage induced by oxidants or other chemicals (4–6). Unlike most prokaryotic and eukaryotic cells, in which the major thiol component is GSH, members of the Kinetoplastida rely on a glutathione-spermidine conjugate named trypanothione [N¹,N⁸-bis(glutathionyl)spermidine] (7, 8). Trypanothione, together with its associated enzyme, trypanothione reductase, appears to have replaced many of the antioxidant and metabolic functions of the glutathione/glutathione reductase and thioredoxin/thioredoxin reductase systems in other organisms (9, 10). The essential requirement for trypanothione reductase in these parasites (11–14), and the pronounced difference in substrate specificity to human glutathione reductase (15) makes this flavoenzyme an attractive target for drug design (10, 16–18). Since trypanothione reductase is essential for parasite survival, inhibition of the biosynthesis of its substrate offers an additional target for therapeutic intervention.

The biosynthesis of trypanothione involves two metabolic pathways that are distinct in other organisms: i.e. spermidine and GSH biosynthesis (19). In the African trypanosome, exemplified by *Trypanosoma brucei*, spermidine is synthesized from ornithine via putrescine and subsequently conjugated with GSH to produce glutathionylspermidine and then trypanothione (20). In contrast, in the South American trypanosome *Trypanosoma cruzi*, polyamines cannot be synthesized de novo from their precursor amino acids (ornithine, arginine or lysine) (21). Instead, exogenous putrescine and cadaverine can be readily taken up and converted via spermidine or aminopropylcadaverine into trypanothione or homotrypanothione [N¹,N⁹-bis(glutathionyl)aminopropylcadaverine], respectively (21) (Fig. 1). The biosynthesis of homotrypanothione seems to be unique to *T. cruzi*, a parasite that has evolved a number of inducible high-affinity transporters for putrescine, cadaverine, and spermidine that are not found in *T. brucei* (22).

Biosynthesis of trypanothione involves ATP-dependent conjugation of GSH to both primary amines of spermidine (19). The original report (23) of a single enzyme from *Crithidia fasciculata* converting GSH and spermidine into trypanothione (trypanothione synthetase) was superseded by the identification of two distinct ATP-dependent enzymes involved in the synthesis of trypanothione (24, 25): a glutathionylspermidine synthetase (GspS,¹ EC 6.3.1.8) that forms both N¹- and N⁸-

---

*This study was supported by the Wellcome Trust with additional funding from the European Commission (ERBIC18 CT98 0372). The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.*

*The nucleotide sequence(s) reported in this paper has been submitted to the GenBank™/EBI Data Bank with accession number(s) AF311782.*

‡ These authors contributed equally to this work and both should be considered as primary authors.

§ Present address: Laboratoire de Parasitologie Moléculaire, Université Victor-Segalen Bordeaux 2, CNRS UMR-5016, 146 Rue Léo Saignat, 33076 Bordeaux Cedex, France.

¶ Present address: Unité de Biologie et Biochimie Cellulaire, Facultés Universitaires Notre-Dame de la Paix, 61 rue de Bruxelles, B-5000 Namur, Belgium.

|| To whom correspondence should be addressed. Tel.: 44-1382-345155; Fax: 44-1382-345542; E-mail: a.h.fairlamb@dundee.ac.uk.

---

¹ The abbreviations used are: GspS, glutathionylspermidine synthetase; TryS, trypanothione synthetase; HEPPS, 4-(2-hydroxyethyl)-1-piperazinepropanesulfonic acid; MES, 4-morpholineethanesulfonic acid; CHES, 2-(cyclohexylamino)ethanesulfonic acid; HPLC, high performance liquid chromatography; MALDI, matrix-assisted laser desorption/ionization.

Trypanothione Biosynthesis in Trypanosoma cruzi

EXTRACELLULAR
INTRACELLULAR

Putrescine
Spermidine
Cadaverine

Putrescine
↓
Spermidine ←
GSH
Glutathionylspermidine
↓ GSH
Trypanothione

Aminopropylcadaverine (APC)
↓ GSH
Glutathionyl-APC
↓ GSH
Homotrypanothione

FIG. 1. Biosynthetic pathways to trypanothione and homotrypanothione in T. cruzi epimastigotes. Transport pathways are shown as dashed lines. T. cruzi can also convert spermidine to spermine and aminopropylcadaverine to bis(aminopropylcadaverine) (not shown).

glutathionylspermidine and a trypanothione synthetase (TryS, EC 6.3.1.9) that catalyzes addition of a second GSH to either of the preceding intermediates (23). Recently, genes for both GspS and TryS have been cloned from C. fasciculata and expressed in Saccharomyces cerevisiae (26). Although expression levels were too low to detect enzyme activity in cell extracts, thiol analysis of yeast transformed with CfGspS showed the presence of glutathionylspermidine and those containing both CfGspS and CfTryS trypanothione (26). The fact that no glutathionylspermidine or trypanothione was found in yeast transformed with CfTryS alone is consistent with previous reports that CfTryS is unable to synthesize glutathionylspermidine (24, 26). The physical and kinetic properties of CfGspS have been confirmed by subsequent studies on recombinant enzyme purified from E. coli (27). Like the E. coli enzyme (EcGspS), CfGspS also has an amidase activity associated with the N-terminal domain of the protein that can hydrolyze glutathionylspermidine to GSH and spermidine (26). CfGspS can also hydrolyze trypanothione at 1.5% of the rate with glutathionylspermidine (27). Sequence alignments indicate that CfTryS has a similar domain, although amidase activity with either glutathionylspermidine or trypanothione has not been demonstrated.

Here, we report the cloning and characterization of TryS from T. cruzi and describe some of the physical and kinetic properties of the enzyme. In contrast to C. fasciculata, we demonstrate that a single enzyme is capable of making both glutathionylspermidine and trypanothione from spermidine, as well as glutathionylaminopropylcadaverine and homotrypanothione from aminopropylcadaverine. We also show that the enzyme is a bifunctional synthetase/aminase capable of hydrolyzing trypanothione, homotrypanothione, and glutathionyl-

spermidine. The implications of CfGspS as a model for drug development against Chagas disease is discussed.

EXPERIMENTAL PROCEDURES

Organisms and Reagents—Trypanosoma cruzi Silvio X10 clone 7 was used as source for DNA. Routine DNA manipulations were performed in E. coli strain JM109 and XL1-Blue. Expression of recombinant proteins employed E. coli strains BL21(DE3) and BL21(DE3)pLysS or S. cerevisiae ABYS106. All chemicals were of the highest grade available from Sigma, Merck, or CALBIOCHEM. Restriction enzymes and DNA-modifying enzymes were from Promega.

Cloning and Characterization of a Trypanothione Synthetase Gene—Two degenerate oligonucleotides were designed based on an alignment of EcGspS (28), CfGspS (26), and CfTryS (26) (KYQCVEF, DTC5: 5′-AAR TAY CAR TGY GTN GAR TT-3′ and EPLWK, DTC2: 5′-YTT CCA NAD NGG YTC-3′). A 1344-bp PCR product was identified following agarose gel electrophoresis and cloned into the SmaI site of pUC18 (SureClone kit, Amersham Biosciences) giving the plasmid pUC18-TcTryS and sequenced (29).

Genomic DNA was isolated from T. cruzi (30), digested with PstI or SalI or KpnI and separated by agarose gel electrophoresis. DNA in the size range 3–4 kb (PstI), 1.6–2 kb (KpnI), and 1–1.6 kb (SalI) were isolated (QIAEX Extraction Kit, Qiagen), ligated to pUC18 vector (digested with PstI or SalI or KpnI and dephosphorylated, Appligene) and transformed in E. coli XL1-Blue. The PstI, KpnI, and SalI sublibraries each contained about 20,000 recombinant plasmids.

Following digestion of pUC18-TcTryS with EcoRI-HindIII, the cloned DTC5/DTC2 PCR fragment was isolated, labeled with [32P]dCTP by the random primer method (Ready Prime, Amersham Biosciences) and used to screen ~10,000 clones from each of the sublibraries (26). Southern blot analysis of T. cruzi genomic DNA (2 μg) was performed using the 32P-labeled DTC5/DTC2 PCR probe (26).

Expression of Trypanothione Synthetases in S. cerevisiae—Yeast strain ABYS106 expressing TcTryS was obtained by transformation with the gene cloned in the expression vector pRB1438 under the control of the GAL10 promoter. The coding sequence was amplified by

Trypanothione Biosynthesis in Trypanosoma cruzi

FIG. 2. Southern blot analysis of *T. cruzi* genomic DNA. A, restriction endonuclease digestion products of *T. cruzi* genomic DNA (2 μg) were separated, blotted, and hybridized as described under “Experimental Procedures.” The arrow indicates the position of the 1.3-kb *SalI* and 0.4-kb *KpnI* fragments. B, physical map of the *TcTryS* gene. The black box indicates the coding sequence of the *TcTryS* gene. The hatched box represents the DTC5/DTC2 PCR product used as a probe. *KpnI*, *SalI*, and *PstI* clones are indicated as a black line. Restriction sites used in this study are marked. E, *EcoRI*; H, *HindIII*; P, *PstI*; B, *BamHI*; K, *KpnI*; S, *SalI*.

PCR using oligonucleotide TCTRS5′ 5′-GG GAA TTC ATG ACG ACT CTA AAG TCT TTG-3′ for the sense strand containing an *EcoRI* cloning site (underlined), and an antisense strand oligonucleotide TCTRS3′: 5′-CC GAA TTC TCA CGT TTT CAA GCC ACC-3′, which includes a stop codon with a *EcoRI* restriction site (underlined). The PCR product was subsequently cloned in pUC18 and then transferred into pRB1438 creating plasmid pRB1438-*TcTryS*. The orientation of the gene was determined using the *KpnI* restriction site.

Yeast co-expressing both *CfGspS* and *TcTryS* were obtained by transformation with the genes cloned in the expression vector pRB1438 under the control of the *GAL1* and/or *GAL10* promoters. The *T. cruzi* coding sequence was removed from the pUC18 plasmid by digestion with *EcoRI* and ligated into the pRB1438-*CfGspS* vector (26) creating the plasmid pRB1438-*CfGspS-TcTryS*. The orientation of the gene was determined using *KpnI*. Yeast transformants were cultured and induced with galactose and lysed exactly as described previously (26).

Thiol and Polyamine Content of Yeast Transformants—Thiols were determined by ion-paired, reverse-phase HPLC following derivatization of 2 × 10⁷ cells with monobromobimane (31). Polyamines were determined by HPLC after derivatization of 2 × 10⁷ cells with dansyl chloride (32, 33).

Expression and Purification of Trypanothione Synthetase in *E. coli*—The complete open reading frame of *TcTryS* was amplified by PCR from *T.cruzi* genomic DNA using forward and reverse primers containing *NdeI* restriction sites. The PCR product was blunt-end ligated into the *SmaI* site of pUC18 (SureClone kit from Amersham Biosciences) and sequenced to ensure that the correct product had been obtained. The insert was removed by digestion with *NdeI* and ligated into pET15b (Novagen) creating plasmid pET15b-*TcTryS*. The correct orientation of the gene was confirmed using the *BamHI* and *XhoI* restriction sites. BL21(DE3)pLysS [pET15b-*TcTryS*] was grown at 37°C with vigorous agitation (220 rpm) in Terrific Broth containing 50 μg ml⁻¹ carbenicillin and 12.5 μg ml⁻¹ chloramphenicol. When the culture reached an OD₆₀₀ of ~0.6, it was cooled to 22–25°C, induced with 0.5 mM isopropyl-β-D-thiogalactopyranoside, and harvested after 16 h. Recombinant

*TcTryS* was purified from cell extracts by chromatography on nickel-chelated Sepharose, digested with thrombin to remove the His₆ tag and purified to homogeneity by anion exchange chromatography on ResourceQ (Amersham Biosciences) (27). The concentrated sample was stored at –20°C in 40% glycerol, 0.01% sodium azide in 0.1-ml aliquots. Under these conditions the enzyme is completely stable for at least two months.

Physical Properties of *TcTryS*—Subunit size and purity of *TcTryS* was determined by SDS-PAGE on a 4–12% gradient gel. Samples were transferred onto polyvinylidene difluoride membrane for N-terminal sequencing by pulsed liquid Edman sequencing on a Procise 494 protein sequencer (Applied Biosystems). MALDI-TOF protein was analyzed in linear mode using sinapinic acid as a matrix on a Voyager-DE STR mass spectrometer (PerSeptive Biosystems). The relative molecular mass of the native protein was determined by chromatography on a Superdex 200 HR column (Amersham Biosciences) as previously described (27).

Kinetic Analysis of *TcTryS* Activity—Kinetic analysis (*K*ₘ and *k*ₐₜ values) and pH optimization studies of synthetase activity were performed by spectrophotometric assay at 340 nm, in which hydrolysis of ATP is coupled through pyruvate kinase and lactate dehydrogenase to oxidation of NADH. Each assay contained 100 mM (K⁺) HEPPS, pH 8.0, 0.2 mM NADH, 1 mM phosphoenolpyruvate, 5 mM dithiothreitol, 0.5 mM EDTA, 10 mM MgSO₄, 2 units ml⁻¹ L-lactate dehydrogenase, 2 units ml⁻¹ pyruvate kinase, 0.8 μM *TcTryS*, and varying amounts of substrates. Initial rates were determined at 9 different substrate concentrations with fixed (saturating) concentrations of the co-substrates (*i.e.* 2 mM MgATP, 1 mM GSH, and either 10 mM spermidine or 1 mM glutathionylspermidine). The pH optimum of the enzyme was determined as above in a mixed buffer system containing 16.6 mM each of MES, CHES, and HEPPS (pH adjusted using KOH) with 20 units ml⁻¹ of each coupling enzyme. The effect of ionic strength was determined as above in 50 mM HEPPS buffer, pH 8 plus 0–500 mM KCl or (NH₄)₂SO₄. One unit of enzyme activity is defined as 1 μmol of NADH oxidized per min. Kinetic parameters (± S.E. of the mean) were determined using

35856

Trypanothione Biosynthesis in Trypanosoma cruzi

CfGspS MSSLPHNHHYETHHRGTAEVFDELIGVTBDGVPIISNGNEAHFSNLESITAACLPLSSFERKAPCKQPYRKMCGVKWQCV 80  
EcGspS MSKGTTSQDAPFGTLLIGYAPGGVAIYSSDYSSLDPQEYEDDAVFRSYIDDEYM-----------------CHKWQCV 60  
CfTryS MASAERVPSFNNKPRVPFGEVQGYAPGHIIPAYSNKHDFSGERSIDDNVFC--------------------GBKYQCV 60  
TcTryS MTTLKSLAVPFGCVQGYAPGGIPAYSNKHEYSFGERSIDGNLFC------------------------GBKYQCV 52  

CfGspS EFVRRYLASRKAVWMTSCLTAEEVWREENLFDVDGRPVVEVRTPNKSTGPAAVADIVWWGE-GPETPFGHVAIVTEV 159  
EcGspS EFARRFLFLNYGVFTDVGMAMEIFSLR-FLREVVNDNILPLQAFPNGSPRA-PVAGALLIWDKGGEFKDTGHVAIITQL 138  
CfTryS EFARRWLLERKGLVLPDVNWACHIFKLK-NVKDAATAEEVPVIAVRNGTEAK-FEPDTLIITYPS-SDVNTVGHVGAITEV 137  
TcTryS EFARRWLFERKSLVLVPDVDWAVHIFNLK-EVSDARTGQPVRCAIRNGTAAK-PVVDSELLIYPS-DDYSPIGHVAAITEV 129  

CfGspS CASCVRVAEQN--QGFEEKWPEDVPFSREIAMRTTESGEVELLDE---DPLLGGWVTQAPFYNFDD---GDLADSFRLLVG 231  
EcGspS HGNKVRIAEEQNVIHSPLPQQQWTRELEMVVENGCYTLKDTFDD---TTILGWMIQTEDTEYSLP---QPEIAGELLKISG 213  
CfTryS GDDYVCIAADQN--YRFHKWEASYSYKLKLQHDGVWTIIDIDIPNDVEIPLGWVTFPGYENRPEGAAFPALHPSLHFQPP 215  
TcTryS GDKWVRIADQN--HRFHKWDANYAAELPLIHEKGVWTILDPLED-EVLKPLIGWVTFPETPDRNPNEPLVLHESLHFKRG 205  

CfGspS QGQILLRQPF---PKHVDVPWLNTGECDTILKHSLVVFGNMGEGAHAEEGDVPGAFFYFLDYDMFCRLGRAASSLHRIAMAA 309  
EcGspS --ARLENKG----QFDGKWLDEKDPLQNAYVQANGQVINQ---------DPYHYYTITESAEQELIKATNELHLMYLHA 277  
CfTryS EEFYLVRKTYEPTETKANWLDINDPAEKLFVEEGMDVSR--------SRLEETTVNYYECDHEFHLCIAYGTQLHDYFMEA 290  
TcTryS EVPTLRRLTFTPNSREKDWLDLTNEAEAYFAGVCGIDVKN--------PKVEKASYYQMNRELYLDCVKYGNQLHQMFEA 278  

CfGspS TAKVLEDFESTHLLEHYFGVPPBIQPLLRRSWEMT--PPMGRFDFGYDGKN--VVMILEYNCDSGALLECCNTQEKMAR 385  
EcGspS TDKVLKDDNLL----ALFDIPKILWPRLLRSWQRRHHMITGRMDFCMDBRG--LKVVYEYNADSASCHTEAGLILERWAE 351  
CfTryS TAQVINDERL----RIFKIPEELWPRMRHSKYQ-QTYISGRFDFAYNNETHMKCFEYNADSASTLLCEGRIQQKWAEE 365  
TcTryS TKFVIGSDEQL----RLFHIPEEYWPRLLRYLWETQ-PHAITGRFDFAFDEDTQQFKCFEYNADSASTLLECGVIQQKWAR 353  

CfGspS FYGVSQGSTSTGSFLGAKCVTYFQRLLTNEKCPQHRLIHFMIDEDDEERYTARCMMGFAEQAGFRTKLCKLVNFRYRDG 465  
EcGspS QG--YKGNGFNPAE-----GLINELAGAWKHSRARPFVHIMQDKDIENYHAQFMEQALHQAGFETRILRGLDELGWDA 424  
CfTryS SVGLDKEGTRGSGW-----AVERNRTAWATCGATGRVHFLVDDEKEEQYTALYCLAEEAVGLEGKLCVMYDEFRFNEE 440  
TcTryS SVGLDDGTTYSSGS-----LISSRLQLAWEMAGVTGRVHFLIDKDDEEYYTALYVMHASAAGLETKLCVLFDEFHFDE 428  

CfGspS PPSNAAPLATPCDHPTIVDGEDEEVLMWKKTWSWDTVLHQYHSQRSDDAV-------NTPTLS------DILINN--NIRVLE 534  
EcGspS G-----------QLIDGEGRIVNCWKTTWAWEATAFDQIREVSDREFAAVPIRTGHPQNEVRLIDVLLRP--EVLVBE 488  
CfTryS G-----------YVWDSDGVRVRNIWKTWMWESAISDYFAAQAEARGAD------WKATPADKVRLCDMLLGKDWDILYE 503  
TcTryS G-----------VVVSDGVAVTTVWKTTWMWETAIADHQKARVQRGND------WRPTPKDEVRLCDILLGPNWDMRVFE 491  

CfGspS PLWKAVTGSKAILPFMHALAPDHEMLAAASFLPTREIISRHYISKPVNGRAGQNIMMYDPVTSPTELEGAPQQDICEALS 614  
EcGspS PLWTVPGNKAILPILWSLFPHHRYLLDTDFTVNDDELVKITGYAVKPIAGRCGSNIDLVSFFE---------- 550  
CfTryS PMWKILPSNKAILPIIYHNHPDHPAIRAEYELTDELRAVGYARKPIVGRVGRNVTTIDGTG------------- 565  
TcTryS PMWKISSNKAILPIIYNKHPDHPALLRASEYLTIELQRTGYVKKPIVGRVGRNVTVTEASG------------- 553  

CfGspS QNASARSLLNGSPLPLSQSVDQTNECSPGKFDSVLVYQRLFLLKKFDGKYFPIFCGWMVGDEFGGVVREDTSKITKLSE 694  
EcGspS -----------------EVLDKTSGKEAEQKNYYQOLWCLPKVDGKY-IQVCTFTVGNYGGTCLRGDESLVIKKE 608  
CfTryS -----------------EVHAESEGGNFERDMIYQELFSLTKQDGGY-AIIGGMLGDAFSGTGIERDKSIITGLD 623  
TcTryS -----------------DIAAKSDGNFLDRDMVYQELFRLPERDGGY-AILGGWVIGDVYC GTGVREDTTIITGLE 611  

CfGspS SMVVPARVVRDNVPLGVSYDEGET 719  
EcGspS SDIEPLIVVK 619  
CfTryS SPFAAIRIKINAI PRPLTHKDLDKLAEDE 652  
TcTryS SPFRALRVYQGAQRRPLTHELDNAEAAAVGGGLKT 647  

FIG. 3. Alignment of the predicted amino acid sequences of trypanothione synthetases and glutathionylspermidine synthetases. Gray boxes show identities with the *T.* cruzi TryS sequence. Gaps introduced into sequences to optimize alignments are represented by a dash. Sequences shown white on black indicate the two conserved sequences between the proteins used to design the degenerate primers. Overlined sequences indicate the six major insertions. The cysteine residue implicated in amidase activity is marked with an arrow. The GenBank™/EMBL/DDBJ accession numbers used in the alignment are: glutathionylspermidine synthetases CfGspS (U66520) and EcGspS (U23148) and trypanothione synthetases CfTryS (AF006615) and TcTryS (AF311782, this study). Two additional sequences (AF283000 and AF283001) showing 96% identity to TcTryS have also been deposited by the *T.* cruzi genome project (not shown).

the Michaelis-Menten equation by non-linear fitting with the program GraFit.

Analysis of Product Formation—Recombinant TcTryS (7.6 μg, 1.025 μM final concentration) was added to 100 μl of 50 mM bis-Tris propane, 50 mM Tris, pH 7.4, 1 mM EDTA, 5 mM MgSO₄, 2 mM ATP, 0.5 mM NADPH, 0.5 units of trypanothione reductase, 0.8 mM GSH, and either 0.8 mM spermidine or 0.8 mM GspdSH. Control samples contained heat-inactivated TcTryS (100 °C, 5 min). After 1 h at 37 °C, 10-μl aliquots were derivatized with monobromobimane and analyzed by HPLC (31).

Analysis of Amidase Activity—The pH optimum for the amidase activity was determined using the mixed buffer system described above in a final volume of 20 μl containing 0.4 mM glutathionylspermidine (0.2 mM glutathionylspermidine disulfide reduced with 0.5 mM dithiothreitol) and 20 μM Tc-TryS. Samples were incubated for 4 h at 37 °C and analyzed by HPLC (31). In subsequent experiments, amidase activity of TcTryS was determined under linear conditions in 20 μl of 0.1 M HEPPS, pH 8.0 containing 18.4 μM enzyme and either 0.5 mM trypanothione disulfide, 0.5 mM homotrypanothione disulfide, or 0.25 mM glutathionylspermidine disulfide reduced with 1 mM tris(2-carboxyethyl)-phosphine. After incubation for 10 min at 37 °C, 10-μl aliquots were derivatized and analyzed by HPLC (31). CfGspS amidase activity was determined under the same assay conditions in a total volume of 50 μl containing 2 μM enzyme with trypanothione or homotrypanothione as substrate and 0.0025–0.1 μM enzyme for glutathionylspermidine.

RESULTS

Isolation and Characterization of the Trypanothione Synthetase Gene from *T.* cruzi—Using the degenerate oligonucleotide primers DTC5/DTC2 described under “Experimental Procedures,” a 1344-bp fragment was amplified by PCR from *T.* cruzi genomic DNA and cloned into pUC18 giving the plasmid pUC18-TcTryS. Sequence analysis verified that this DNA fragment encoded a protein with sequence similarity to TryS from *C.* fasciculata (26) and GspS from *E.* coli (28) or *C.* fasciculata (26).

*T.* cruzi genomic DNA was digested with several restriction enzymes and probed with the DTC5/DTC2 PCR fragment labeled with [³²P]dCTP (Fig. 2A). The probe hybridized to a single ~4-kb *Pst*I fragment, two *Kpn*I fragments (~1.7 kb and ~0.4 kb, arrow) and two *Sal*I fragments (~10 kb and ~1.3 kb, arrow). The labeled probe was then used to screen the *Pst*I, *Kpn*I, and *Sal*I sublibraries (see “Experimental Procedures”). Seven clones were isolated from the *Pst*I sublibrary, all of which contained a DNA fragment of ~4 kb; 1 clone was isolated from the *Kpn*I sublibrary, which contained a DNA fragment of ~1.7 kb; and 3 clones were isolated from the *Sal*I sublibrary,

which contained a DNA fragment of ~1.3 kb. Preliminary sequence analysis of the *PstI*, *KpnI*, and *SalI* clones revealed partial open reading frames with similarity to the *C. fasciculata* synthetases. Using the *SalI* and *KpnI* clones, which encode the N and C-terminal regions of the protein, respectively, we were able to reconstitute a full-length clone (Fig. 2B). A PCR product overlapping the *PstI*, *KpnI*, and *SalI* restriction sites was cloned and sequenced to confirm there were no errors at these restriction sites. No sequence variations were found in any of the cloned fragments, suggesting that we have cloned one allelic copy. Moreover, the restriction patterns (Fig. 2, A and B) suggest that *TcTryS* is single copy per haploid genome of *T. cruzi* since the probe hybridized to a single band when DNA was digested with enzymes that do not cut within the coding sequence (*EcoRI* and *HindIII*), whereas the probe hybridized to two fragments when the DNA was digested with enzymes that cut at a single site within the coding sequence (*BamHI*, *SalI*, and *KpnI*).

**Amino Acid Sequence Analysis—Translation of TryS from *T. cruzi*** predicts a protein of 647 amino acids with a molecular mass of 73,560 Da. *TcTryS* shows strongest homology to *CfTryS* (75% similarity and 61% identity in sequence alignments), followed by *EcGspS* (51% similarity and 34% identity) and *CfGspS* (45% similarity and 31% identity) (Fig. 3). Like *CfTryS*, *TcTryS* lacks six major insertions found in *CfGspS* (residues 57–73, 270–276, 321–324, 400–404, 467–480, and 597–635, overlined in Fig. 3). Significant homology was found between *TcTryS* and *EcGspS* in the N-terminal region, which is responsible for the glutathionylspermidine amidase activity of the *E. coli* protein (28, 34). Moreover, a cysteine residue, corresponding to Cys-59 of *EcGspS* and Cys-79 *CfGspS* that have been shown by site-directed mutagenesis to be essential for amidase activity (27, 35), is present in *TcTryS*, suggesting that this enzyme may also possess amidase activity (see below).

Having completed our sequencing study, two similar sequences from *T. cruzi* CL Brener (43-CL F11F5) were deposited by the *T. cruzi* genome project annotated as putative trypanothione synthetases (accession numbers AF283000 and AF283001). These sequences show about 96% identity to each other and also to Silvio X10 clone 7 used in this study. This is consistent with the recent report that CL-Brener is a hybrid of clades B and C belonging to lineage II and distinct from X10 clone 7, which belongs to clade A of lineage I (36).

**Expression of TcTryS in S. cerevisiae—TcTryS** was cloned into pRB1438 for inducible expression in *S. cerevisiae* ABYS106 as previously described (26). Galactose-induced transformants containing pRB1438-*TcTryS* or pRB1438-*CfGspS-TcTryS* showed no overexpression of protein (by SDS-PAGE), and no detectable enzymatic activity above endogenous ATPase activity (using the coupled assay method), as previously described for the *Crithidia* enzymes (26).

However, evidence for enzyme activity could be demonstrated by HPLC analysis of thiols in yeast transformants (Fig. 4). In control cells, the principal thiol is GSH (38 nmol [10^8 cells]^-1) (26, 37), together with trace amounts of three other thiols (U1, U2, and U3), which we previously ascribed to L-cysteine, L-γ-glutamyl-L-cysteine, and L-cysteinylglycine, respectively (26). No glutathionylspermidine or trypanothione bimane adducts are evident in the control cells (compare Fig. 4A with standards in Fig. 4D). Since we had previously demonstrated that cells transformed with pRB1438-*CfTryS* do not form trypanothione, unless co-expressed with *CfGspS* (26), we were somewhat surprised to observe glutathionylspermidine (1.7 nmol [10^8 cells]^-1) and trypanothione (5.5 nmol [10^8 cells]^-1) in galactose-induced cells transformed with pRB1438-*TcTryS* alone (Fig. 4B). Transformants cultured with amino-

**FIG. 4. Analysis of thiols in transformed yeast.** Thiols were determined by HPLC after derivatization with monobromobimane; equivalents of 1 × 10^6 cells were injected for each sample. Trace A, uninduced yeast cells (ABYS106); trace B, yeast cells (ABYS106/pRB1438-*TcTryS*) induced with galactose; trace C, yeast cells (ABYS106/pRB1438-*TcTryS*) induced with galactose and grown in the presence of 0.1 mM aminopropylcadaverine; trace D, mixture of purified standard bimane-derivatives containing 125 pmol of GSH (glutathione), 100 pmol of T(SH)_2 (trypanothione), 75 pmol each of GspdSH (glutathionylspermidine), and hT(SH)_2 (homotrypanothione), 25 pmol of GapcSH (glutathionylaminopropylcadaverine). U1–U3 represent peaks that elute at the retention times of authentic L-cysteine, L-γ-glutamyl-L-cysteine, and L-cysteinylglycine, respectively. R, reagent peak.

propylcadaverine also contained glutathionylaminopropylcadaverine (1.8 nmol [10^8 cells]^-1) and homotrypanothione (3.8 nmol [10^8 cells]^-1) as well as glutathionylspermidine (1.2 nmol [10^8 cells]^-1), and trypanothione (3.7 nmol [10^8 cells]^-1) (Fig. 4C). These data strongly suggest that *TcTryS* encodes an enzyme with both glutathionylspermidine synthetase and trypanothione synthetase activity.

**Overexpression of TcTryS in E. coli—E. coli** expressing *TcTryS* in pET-3a produced large amounts of enzymatically inactive protein that was found exclusively in the insoluble fraction, as reported previously for the *Crithidia* enzymes (26). However, soluble and enzymatically active protein was obtained following subcloning into pET-15b and expression in *E. coli* BL21(DE3)pLysS. Induction at 25 °C rather than 37 °C was found to be critical for soluble expression. Pure *TcTryS* was obtained following two chromatographic steps (Ni^2+ affinity and anion exchange) and thrombin cleavage to remove the His_6 tag (Fig. 5). Typical yields were 0.5–1 mg per liter of culture, which is 10-fold less than that for *CfGspS* (27).

**Physical and Kinetic Properties of the Enzyme—N-terminal sequencing** gave the amino acid sequence GSHMTTLKS (the initiating methionine in the native protein is underlined) as

35858 Trypanothione Biosynthesis in Trypanosoma cruzi

![Figure 5](#fig5)

FIG. 5. Purification of recombinant TcTryS from *E. coli*. A, SDS-PAGE analysis: Lane 1, soluble fraction of BL21(DE3)pLysS [pET15b-TcTryS]; lane 2, pooled fractions after chromatography on nickel-chelating Sepharose High Performance column; lane 3, pooled fractions after removal of His₆ tag with thrombin protease and chromatography on Resource-Q. Approximately 2 μg of protein was loaded in each lane. B, gel filtration profile of the native protein after removal of the His₆ tag. The inset shows a plot of \( V_e / V_0 \) versus log molecular mass of a standard protein mixture (open symbols), where \( V_e \) is the elution volume and \( V_0 \) is the void volume of the column. The closed symbol refers to the elution volume of TcTryS.

predicted for His₆-tagged protein after cleavage at the thrombin site. MALDI analysis on the recombinant protein revealed a nominal molecular mass of 73,744 Da, which correlates well with the predicted nominal molecular mass (73,842 Da, after thrombin cleavage). The protein migrates on SDS-PAGE as a ~74.5-kDa protein (Fig. 5A) and elutes as a single symmetrical 64-kDa peak on gel filtration indicating that the native protein is a monomer (Fig. 5B).

The reactions catalyzed by TcTryS were examined by incubating recombinant enzyme with GSH and MgATP, and either spermidine, glutathionylspermidine, or aminopropylcadaverine, followed by HPLC analysis of the fluorescent bimane derivatives (Fig. 6). Spermidine was converted to trypanothione, with no apparent accumulation of glutathionylspermidine as an intermediate (trace A). Glutathionylspermidine was also efficiently converted into trypanothione (trace B). Aminopropylcadaverine was converted into both glutathionylaminopropylcadaverine and homotrypanothione (trace C). No glutathionylspermidine or trypanothione-bimane adducts were found in the heat-inactivated sample (trace D).

Kinetic Characterization of TcTryS Activity—Using the coupled assay system the specific activity of TcTryS with 1 mM GSH/10 mM spermidine or 1 mM GSH/0.2 mM GspdSH was determined to be 0.77 and 0.75 units mg⁻¹, respectively. The pH profile was determined in a mixed buffer system over the pH range 5.5–10. Addition of ADP to control incubation mixtures confirmed that the coupling enzymes were not rate-limiting. The synthetase activity follows a symmetrical bell-shaped curve with a pH optimum of 8.1 ± 0.1 with pKₐ values of 6.9 ± 0.1 and 9.4 ± 0.1 with GSH and spermidine (Fig. 7A, open circles). A similar pH profile was observed with glutathionylspermidine in place of spermidine, with a pH optimum of 8.5 ± 0.1 and pKₐ values of 7.5 ± 0.1 and 9.5 ± 0.1 with GSH and glutathionylspermidine (Fig. 7A, closed circles). Due to the susceptibility of thiols to oxidation at elevated pH, all subsequent assays were performed at pH 8.0. Activity is optimal in the range 0.05–0.1 M HEPES buffer (Fig. 7B, inset). Higher concentrations of HEPES are inhibitory, as are KCl or \((\text{NH}_4)_2\text{SO}_4\) (Fig. 7B, inset). When activity is expressed as a function of total ionic strength due to buffer and salts, the data fit a single curve with a narrow optimum ionic strength (\(I\)) of

![Figure 6](#fig6)

FIG. 6. Product formation by TcTryS with various polyamine substrates. All samples were incubated at 37 °C for 60 min with native TcTryS (traces A–C) or denatured TcTryS (trace D) prior to derivatization with monobromobimane and thiol analysis by HPLC. Trace A, GSH and spermidine; trace B, GSH and glutathionylspermidine; trace C, GSH and aminopropylcadaverine; trace D, GSH and spermidine; trace E, synthetic standard mixture containing 100 pmol each of GSH, glutathionylspermidine (GspdSH), trypanothione (T[SH]₂), and homotrypanothione (hT[SH]₂). Other abbreviations: GapcSH, glutathionylaminopropylcadaverine; R, reagent peaks.

Trypanothione Biosynthesis in Trypanosoma cruzi

![Figure 7](#fig7)

FIG. 7. Effect of pH and ionic strength on TcTryS. A, determination of optimum pH. Assays were carried out in triplicate at the indicated pH as described under "Experimental Procedures" in the presence of GSH and spermidine (open circles) or GSH and glutathionylspermidine (closed circles). Activity is expressed relative to the maximum activity obtained for both GSH/spermidine and GSH/glutathionylspermidine. B, effect of buffer concentration and ionic strength. The assay mixtures contained either 50 mM (K⁺) HEPPS, pH 8.0, plus salt (KCl, ○ or (NH₄)₂SO₄, ●) or varying amounts of HEPPS buffer (□). Activity is expressed as a percentage of the activity determined in 50 mM HEPPS buffer. The inset shows the effect on activity as a function of buffer or salt concentration.

0.03–0.06 mol liter⁻¹ (Fig. 7B). Accordingly, 0.1 M (K⁺) HEPPS, pH 8.0 was used for kinetic analyses.

The enzyme displays simple hyperbolic kinetics with spermidine, aminopropylcadaverine, glutathionylspermidine, or MgATP as variable substrate, when GSH is fixed at 1 mM and with a fixed (saturating) concentration of the third substrate (see Table I for details and a comparison with the equivalent enzymes from *C. fasciculata* and *E. coli*). However, high-substrate inhibition is observed with GSH as variable substrate in the presence of fixed concentrations (~10 × *K*ₘ) of the other co-substrates, MgATP plus either spermidine (Fig. 8, A and C) or glutathionylspermidine (Fig. 8, B and D). Since the two other co-substrates were held constant, these data can be fitted to the high-substrate inhibition Equation 1,

$$
v = \frac{V}{1 + \frac{K_{m}^{app}}{s} + \frac{s}{K_{i}^{s}}}
$$

where *K*ₘ⁶pp represents the apparent *K*ₘ and *K*ᵢˢ the apparent *K*ᵢ for the variable substrate, GSH. Excellent fits were obtained with both data sets yielding apparent *K*ₘ values of 570 ± 50 and 190 ± 30 μM, *K*ᵢˢ values of 1200 ± 100 and 600 ± 100 μM, and *k*ₐₜ values (dashed lines in Fig. 8, A and B) of 3.4 ± 0.2 and 2.8 ± 0.3 s⁻¹, for GSH with spermidine and glutathionylspermidine, respectively. Although the apparent *K*ₘ and *K*ᵢˢ values for GSH differ somewhat depending on the polyamine co-substrate, the relative inhibition constants *K*ᵢˢ/*K*ₘ are similar (2.1 and 3.2 for spermidine and glutathionylspermidine, respectively). Likewise the *k*ₐₜ/*K*ₘ ratios are broadly in agreement (6 × 10³ and 15 × 10³ M⁻¹ s⁻¹ for spermidine and glutathionylspermidine, respectively).

Demonstration of Amidase Activity—Recombinant TcTryS was examined for amidase activity by incubation with glutathionylspermidine in the absence of other substrates, derivatized with monobromobimane and analyzed by HPLC. The pH profile for the hydrolysis of glutathionylspermidine was also bell-shaped (not shown) with apparent p*K*ₐ values of 6.6 ± 0.2 and 9.2 ± 0.2 and a pH optimum of 7.9 ± 0.1, similar to the forward reaction (8.1 ± 0.1). A time course experiment revealed that glutathionylspermidine was stoichiometrically converted to the expected product, GSH, with linear product formation up to 15 min at pH 8.0 (not shown). TcTryS can also hydrolyze trypanothione and homotrypanothione to GSH with formation of appreciable amounts of the intermediates, glutathionylspermidine and glutathionylaminopropylcadaverine, respectively, (not shown). Due to the low amidase activity of TcTryS it was not possible to carry out a full kinetic analysis by this method. However, a comparative study of the substrate specificities of TcTryS and CfGspS assayed under identical conditions (0.1 M HEPPS, pH 8.0) is summarized in Table II. Since the bis-polyamine conjugates formed a mixture of products, all activities are expressed as nmol of amide bond hydrolyzed min⁻¹ mg⁻¹. The specific activities of TcTryS with the mono- and bis-polyamine conjugates were similar, whereas CfGspS shows a pronounced preference for glutathionylspermidine as a substrate.

DISCUSSION

All trypanosomatids possess a unique dithiol, trypanothione (7, 8), which together with its cognate flavoenzyme, trypanothione reductase, replaces the classical glutathione/glutathione reductase system of mammalian cells. Since trypanothione reductase is essential for parasite survival (11–14), the enzymes involved in trypanothione biosynthesis are also likely to represent important targets for the development of novel anti-parasitic drugs.

Our current findings suggest that the biosynthetic enzymes from *C. fasciculata* may not be ideal models for drug development against *T. cruzi*. Although TcTryS shows greatest sequence similarity to CfTryS, the enzyme from *T. cruzi* is clearly capable of catalyzing all biosynthetic steps involved in the formation of both the mono- and bis-glutathionyl conjugates with spermidine and aminopropylcadaverine. Although we were unable to identify a second gene with greater similarity.

Trypanothione Biosynthesis in Trypanosoma cruzi

**TABLE I**

Physical and kinetic properties of T. cruzi trypanothione synthetase compared with trypanothione and glutathionylspermidine synthetases from C. fasciculata and E. coli

| Property | $TcTryS$ (recombinant) | $CfTryS$ (native) | $CfGspS$ (recombinant)$^a$ | $EcGspS$ (recombinant)$^b$ |
|----------|-------------------------|--------------------|---------------------------|---------------------------|
| $M_r$ from sequence | 73,842 | 74,516$^c$ | 80,602 | 70,539 |
| Oligomeric structure | Monomer | Hetero-dimer with GspS$^d$ | Monomer | Dimer/trimer |
| Synthetase activity |  |  |  |  |
| $K_m^{app}$ GSH ($\mu$M)-with Spermidine | $570 \pm 50$ | Not active$^d$ | 242 | 800 |
| -with Glutathionylspermidine | $190 \pm 30$ | $910^e$ | Not active$^f$ | Not active |
| $K_m^{app}$ MgATP ($\mu$M) | $53 \pm 3$ | $400^e$ | 114 | 100 |
| $K_m^{app}$ Spermidine ($\mu$M) | $625 \pm 39$ | Not active$^d$ | 59 | 60 |
| $K_m^{app}$ Aminopropylcadaverine ($\mu$M) | $662 \pm 112$ | N.D.$^g$ | N.D. | N.D. |
| $K_m^{app}$ Glutathionylspermidine ($\mu$M) | $66 \pm 3$ | $20^e$ | Not active$^f$ | Not active |
| $k_{cat}$ (s$^{-1}$)-with Spermidine | $3.4 \pm 0.2$ | Not active$^d$ | 15.5 | 7 |
| -with Glutathionylspermidine | $2.8 \pm 0.3$ | $1.25^d$ | Not active$^f$ | Not active |
| Amidase activity |  |  |  |  |
| Glutathionylspermidine ($\mu$M) | N.D. | - | 500 | 900 |
| $k_{cat}$ (s$^{-1}$) | N.D. | - | 0.38 | 2.1 |

$a$ Oza et al. (27).  
$b$ Bollinger et al. (28).  
$c$ Tetaud et al. (26).  
$d$ Smith et al. (24).  
$e$ Henderson et al. (23).  
$f$ Activity is <0.1% when $CfGspS_{C79A}$ is incubated with 1 mM glutathionylspermidine instead of 10 mM spermidine.  
$g$ N.D.-not determined.

**FIG. 8. Kinetic analysis of $TcTryS$.** Coupled assays were carried out in 100 mM (K$^+$) HEPPS, pH 8.0 as described under “Experimental Procedures.” $TcTryS$ was incubated at 25°C with 9 varying concentrations of GSH in the presence of 2 mM MgATP and either 10 mM spermidine (panels A and C) or 1 mM glutathionylspermidine (panels B and D).

---

**TABLE II**

Relative amidase activities of $TcTryS$ and $CfGspS$

Enzymes were incubated with 0.5 mM of the reduced glutathionylpolyamine conjugates in 0.1 M HEPPS, pH 8.0 and analysed by HPLC as described under “Experimental Procedures.” Activity is expressed as nmol of amide bond hydrolysed per min per mg protein. Values are the mean ($\pm$ S.E.) of 3 or 6 determinations for $TcTryS$ and $CfGspS$, respectively.

| Substrate | Specific activity | Ratio |
|-----------|-------------------|-------|
|           | $TcTryS$          | $CfGspS$ | GspS/TryS |
|           | $nmol \, min^{-1} \, mg^{-1}$ |  |  |
| Glutathionylspermidine | $5.4 \pm 0.4$ | $1390 \pm 110$ | 257 |
| Trypanothione | $3.1 \pm 0.8$ | $74 \pm 13$ | 24 |
| Homotrypanothione | $3.7 \pm 1.0$ | $126 \pm 26$ | 34 |

With respect to its kinetic properties, $TcTryS$ displays a blend of features characteristic of $CfTryS$, $CfGspS$ and $EcGspS$ (Table I). Although previous studies have indicated that $CfGspS$ is unable to synthesize trypanothione from glutathionylspermidine (24, 27), we have reinvestigated this using much higher enzyme concentrations. Since $CfGspS$ also possesses amidase activity, which would interfere with the forward assay to trypanothione by forming glutathione and spermidine, these assays were carried out with the C79A mutant that is unable to hydrolyze glutathionylspermidine (27). Under these conditions we were unable to detect any activity whatsoever with glutathionylspermidine and GSH as substrates, indicating that $CfGspS$ has less than 0.1% activity as a trypanothione synthetase. Depending on the availability of polyamines in the growth medium and the growth phase of $T$. cruzi epimastigotes, the intracellular concentration of substrates for $TcTryS$ are ~5 mM for ATP (38), 200–1100 $\mu$M for GSH, <9–150 $\mu$M for glutathionylspermidine.

thionylspermidine, and 7–640 $\mu$M for spermidine (estimates based on a cell volume of 55 fl, Ref. 21 and metabolite determinations from Refs. 32 and 39). Thus the kinetic parameters determined in this study (Table I) suggest that the enzyme would function efficiently *in vivo* although the substrate inhibition by GSH observed at >1 mM (Fig. 7) may not be physiologically relevant.

TcTryS also has an opposing amidase activity, which is about 1% of the synthetase activity when assayed at pH 8.0 near the optima for the forward and reverse reactions. This is consistent with our finding that TcTryS has an N-terminal domain containing a conserved Cys (at position 51) previously shown to be essential for amidase activity in both *E. coli* (35) and *C. fasciculata* (27). However, the amidase activity of TcTryS with glutathionylspermidine is two orders of magnitude lower than that observed for GspS from *C. fasciculata*, when assayed under identical conditions (Table II). In addition, TcTryS shows equivalent activity with either mono- or bis-glutathionyl conjugates, unlike CfGspS, which shows a marked preference for glutathionylspermidine. Under certain conditions, *C. fasciculata* (31) (and *E. coli*, Ref. 40) can accumulate high concentrations of glutathionylspermidine in stationary phase, which can be rapidly hydrolyzed to release spermidine for promoting macromolecular biosynthesis when favorable growth conditions are restored. A similar pronounced accumulation of glutathionylspermidine has never been observed for *T. cruzi* in stationary phase (32). Nonetheless, when grown under limiting amounts of polyamines, trypanothione, and glutathionylspermidine content reaches a maximum in late exponential phase, and then drops by more than 10-fold as cells enter stationary phase suggesting that an amidase function is operative *in vivo* (32). Since *T. cruzi* behaves as a diamine/polyamine auxotroph, the opposing synthetic and hydrolytic functions of TcTryS may serve to regulate polyamine levels in response to polyamine availability and growth phase. Trypanothione is also required for regulation of thiol redox and other antioxidant functions via cytosolic and mitochondrial tryparedoxin peroxidases (41, 42) and a glutathione-dependent peroxidase (43). Thus, it is noteworthy that *T. cruzi* has an essential trypanothione-glutathione thiol transferase to catalyze thiol-disulfide exchange between dihydrotrypanothione and glutathione disulfide (44, 45). A similar activity has not been reported for other trypanosomatids. The mechanism by which the opposing synthetase and amidase functions are regulated merits further investigation, particularly whether binding of substrate(s) of the synthetase domain enhances amidase activity as reported for *E. coli* GspS (35).

In conclusion, with the isolation of TryS from *T. cruzi* and soluble expression in *E. coli* we now have a readily available source of active TryS for further detailed kinetic, mechanistic and structural studies. The identification of a single enzyme in *T. cruzi* capable of trypanothione biosynthesis indicates that the enzymes from *C. fasciculata* may not be ideal models for drug design, although renewed efforts to express active recombinant TryS from *C. fasciculata* are ongoing within this laboratory.

Acknowledgments—We would like to thank Dr. C. MacKintosh for useful advice and discussion on the kinetics of high substrate inhibition.

---

**REFERENCES**

1. Marton, L. J., and Pegg, A. E. (1995) *Annu. Rev. Pharmacol. Toxicol.* **35**, 55–91
2. Tabor, C. W., and Tabor, H. (1984) *Annu. Rev. Biochem.* **53**, 749–790
3. Cohen, S. S. (1998) *A Guide to the Polyamines*, Oxford University Press, New York
4. Meister, A., and Anderson, M. E. (1983) *Annu. Rev. Biochem.* **52**, 711–760
5. Meister, A. (1989) in *Metabolism and Functions of Glutathione* (Dolphin, D., Poulson, R., and Avramovic, O., eds) pp. 367–474, John Wiley & Sons, New York
6. Carmel-Harel, O., and Storz, G. (2000) *Annu. Rev. Microbiol.* **54**, 439–461
7. Fairlamb, A. H., and Cerami, A. (1985) *Mol. Biochem. Parasitol.* **14**, 187–198
8. Fairlamb, A. H., Blackburn, P., Ulrich, P., Chait, B. T., and Cerami, A. (1985) *Science* **227**, 1485–1487
9. Fairlamb, A. H., and Cerami, A. (1992) *Annu. Rev. Microbiol.* **46**, 695–729
10. Flohé, L., Hecht, H. J., and Steinert, P. (1999) *Free Radic. Biol. Med.* **27**, 966–984
11. Dumas, C., Ouellette, M., Tovar, J., Cunningham, M. L., Fairlamb, A. H., Tamar, S., Olivier, M., and Papadopoulou, B. (1997) *EMBO J.* **16**, 2590–2598
12. Tovar, J., Cunningham, M. L., Smith, A. C., Croft, S. L., and Fairlamb, A. H. (1998) *Proc. Natl. Acad. Sci. U. S. A.* **95**, 5311–5316
13. Tovar, J., Wilkinson, S., Mottram, J. C., and Fairlamb, A. H. (1998) *Mol. Microbiol.* **29**, 653–660
14. Krieger, S., Schwarz, W., Ariyanayagam, M. R., Fairlamb, A. H., Krauth-Siegel, R. L., and Clayton, C. (2000) *Mol. Microbiol.* **35**, 542–552
15. Shames, S. L., Fairlamb, A. H., Cerami, A., and Walsh, C. T. (1986) *Biochemistry* **25**, 3519–3526
16. Krauth-Siegel, R. L., and Schöneck, R. (1995) *FASEB J.* **9**, 1138–1146
17. Fairlamb, A. H. (1999) *Medicina* **59**, 179–187
18. Augustyns, K., Amssoms, K., Yamani, A., Rajan, P. K., and Haemers, A. (2001) *Curr. Pharm. Design* **7**, 1117–1141
19. Fairlamb, A. H., Henderson, G. B., and Cerami, A. (1986) *Mol. Biochem. Parasitol.* **21**, 247–257
20. Fairlamb, A. H., Henderson, G. B., Bacchi, C. J., and Cerami, A. (1987) *Mol. Biochem. Parasitol.* **24**, 185–191
21. Hunter, K. J., Le Quesne, S. A., and Fairlamb, A. H. (1994) *Eur. J. Biochem.* **226**, 1019–1027
22. Le Quesne, S. A., and Fairlamb, A. H. (1996) *Biochem. J.* **316**, 481–486
23. Henderson, G. B., Yamaguchi, M., Novoa, L., Fairlamb, A. H., and Cerami, A. (1990) *Biochemistry* **29**, 3924–3929
24. Smith, K., Nadeau, K., Walsh, C., and Fairlamb, A. H. (1992) *Protein Science* **1**, 874–883
25. Koenig, K., Menge, U., Kiess, M., Wray, V., and Flohé, L. (1997) *J. Biol. Chem.* **272**, 11908–11915
26. Tetaud, E., Manai, F., Barrett, M. P., Nadeau, K., Walsh, C. T., and Fairlamb, A. H. (1998) *J. Biol. Chem.* **273**, 19383–19390
27. Oza, S. L., Ariyanayagam, M. R., and Fairlamb, A. H. (2002) *Biochem. J.* **364**, 679–686
28. Bollinger, J. M., Jr., Kwon, D. S., Huisman, G. W., Kolter, R., and Walsh, C. T. (1995) *J. Biol. Chem.* **270**, 14031–14041
29. Sanger, F., Nicklen, S., and Coulson, A. R. (1977) *Proc. Natl. Acad. Sci. U. S. A.* **74**, 5463–5467
30. Bernards, A., Van der Ploeg, L. H. T., Frasch, A. C. C., Borst, P., Boothroyd, J. C., Coleman, S., and Cross, G. A. M. (1981) *Cell* **27**, 497–505
31. Shim, H., and Fairlamb, A. H. (1988) *J. Gen. Microbiol.* **134**, 807–817
32. Ariyanayagam, M. R., and Fairlamb, A. H. (1997) *Mol. Biochem. Parasitol.* **84**, 111–121
33. Kabra, P. M., Lee, H. K., Lubich, W. P., and Marton, L. J. (1986) *J. Chromatogr.* **380**, 19–32
34. Kwon, D. S., Lin, C. H., Chen, S. J., Coward, J. K., Walsh, C. T., and Bollinger, J. M., Jr. (1997) *J. Biol. Chem.* **272**, 2429–2436
35. Lin, C.-H., Kwon, D. S., Bollinger, J. M., Jr., and Walsh, C. T. (1997) *Biochemistry* **36**, 14930–14938
36. Machado, C. A., and Ayala, F. J. (2001) *Proc. Natl. Acad. Sci. U. S. A.* **98**, 7396–7401
37. Elskens, M. T., Jaspers, C. J., and Penninckx, M. J. (1991) *J. Gen. Microbiol.* **137**, 637–644
38. Thomson, L., Gadelha, F. R., Peluffo, G., Vercesi, A. E., and Radi, R. (1999) *Mol. Biochem. Parasitol.* **98**, 81–91
39. Ariyanayagam, M. R., and Fairlamb, A. H. (2001) *Mol. Biochem. Parasitol.* **115**, 189–198
40. Tabor, H., and Tabor, C. W. (1975) *J. Biol. Chem.* **250**, 2648–2654
41. Wilkinson, S. R., Temperton, N. J., Mondragon, A., and Kelly, J. M. (2000) *J. Biol. Chem.* **275**, 8220–8225
42. Lopez, J. A., Carvalho, T. U., De Souza, W., Flohe, L., Guerrero, S. A., Montemartini, M., Kalisz, H. M., Nogoceke, E., Singh, M., Alves, M. J. M., and Colli, W. (2000) *Free Radic. Biol. Med.* **28**, 767–772
43. Wilkinson, S. R., Meyer, D. J., and Kelly, J. M. (2000) *Biochem. J.* **352**, 755–761
44. Moutiez, M., Quemeneur, E., Sergheraert, C., Lucas, V., Tartar, A., and Davioud-Charvet, E. (1997) *Biochem. J.* **322**, 43–48
45. Allaoui, A., Francois, C., Zemzoumi, K., Guilvard, E., and Ouassi, A. (1999) *Mol. Microbiol.* **32**, 1273–1286

ENZYME CATALYSIS AND REGULATION:

A Single Enzyme Catalyses Formation of Trypanothione from Glutathione and Spermidine in *Trypanosoma cruzi*

Sandra L. Oza, Emmanuel Tetaud, Mark R. Ariyanayagam, Stéphanie S. Warnon and Alan H. Fairlamb

J. Biol. Chem. 2002, 277:35853-35861. doi: 10.1074/jbc.M204403200 originally published online July 16, 2002

---

Access the most updated version of this article at doi: [10.1074/jbc.M204403200](http://dx.doi.org/10.1074/jbc.M204403200)

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinity-sites).

Alerts:
- [When this article is cited](#)
- [When a correction for this article is posted](#)

Click [here](#) to choose from all of JBC's e-mail alerts

This article cites 43 references, 13 of which can be accessed free at [http://www.jbc.org/content/277/39/35853.full.html#ref-list-1](http://www.jbc.org/content/277/39/35853.full.html#ref-list-1)
