
FEBS 26324

A novel p53-inducible apoptogenic gene, PRG3, encodes a homologue of the apoptosis-inducing factor (AIF)

Yoichi Ohiro${}^{a,1,2}$, Igor Garkavtsev${}^{b,2}$, Shinichiro Kobayashi${}^{f}$, Kodangattil R. Sreekumar${}^{c}$, Regan Nantz${}^{f}$, Bryan T. Higashikubo${}^{f}$, Shannon L. Duffy${}^{a,3}$, Ryuji Higashikubo${}^{f}$, Anny Usheva${}^{a}$, David Gius${}^{d}$, Nikolai Kley${}^{e}$, Nobuo Horikoshi${}^{f,*}$

${}^{a}$Department of Medicine, Beth Israel Deaconess Medical Center, Harvard Medical School, Boston, MA 02215, USA  
${}^{b}$Department of Radiation Oncology, Massachusetts General Hospital, Harvard Medical School, Boston, MA 02129, USA  
${}^{c}$National Center for Biotechnology Information, National Institute of Health, Bethesda, MD 20894, USA  
${}^{d}$Radiation Oncology Sciences Program, Radiation Oncology Branch, National Cancer Institute, National Institute of Health, Bethesda, MD 20892, USA  
${}^{e}$GPC Biotech Inc., Waltham, MA 02451, USA  
${}^{f}$Department of Radiation Oncology, Washington University School of Medicine, St. Louis, MO 63108, USA  

Received 21 May 2002; accepted 11 June 2002  
First published online 8 July 2002  
Edited by Veli-Pekka Lehto

---

**Abstract** The p53 tumor suppressor protein induces cell cycle arrest or apoptosis in response to cellular stresses. We have identified PRG3 (p53-responsive gene 3), which is induced specifically under p53-dependent apoptotic conditions in human colon cancer cells, and encodes a novel polypeptide of 373 amino acids with a predicted molecular mass of 40.5 kDa. PRG3 has significant homology to bacterial oxidoreductases and the apoptosis-inducing factor, AIF, and the gene was assigned to chromosome 10q21.3–q22.1. Expression of PRG3 was induced by the activation of endogenous p53 and it contains a p53-responsive element. Unlike AIF, PRG3 localizes in the cytoplasm and its ectopic expression induces apoptosis. An amino-terminal deletion mutant of PRG3 that lacks a putative oxidoreductase activity retains its apoptotic activity, suggesting that the oxidoreductase activity is dispensable for the apoptotic function of PRG3. The PRG3 gene is thus a novel p53 target gene in a p53-dependent apoptosis pathway. © 2002 Federation of European Biochemical Societies. Published by Elsevier Science B.V. All rights reserved.

**Key words:** p53; Target gene; Cloning; Apoptosis; Oxidoreductase; Transcription

---

### 1. Introduction

The p53 gene is one of the most frequently altered genes in a wide variety of tumor types, indicating its importance in cell growth control and tumorigenesis [1,2]. The loss of wild-type p53 activity removes important controls on cell cycle regulation, apoptosis, and maintenance of genomic integrity in cultured cells [3], and contributes to tumor development [4]. The limited developmental abnormalities associated with the majority of p53 (−/−) mice indicate that p53 may be dispensable, to a large extent, for the apoptotic processes occurring during normal embryo development. However, under conditions of cellular stress that endanger the integrity of the genome, p53 becomes crucial in restricting inappropriate cell proliferation [3,5–7]. Cellular stresses that can induce p53-mediated apoptosis include DNA damage [8,9], deregulated oncogene expression [10–14], hypoxia [15], ribonucleotide depletion [16], and oxidative stress [17]. In normal cells, under physiological conditions the p53 tumor suppressor protein is present at a low level because of a short half-life due to rapid turnover mediated by ubiquitination and proteolysis [18,19]. After DNA damage, wild-type p53 accumulates and becomes an active transcription factor that binds a specific DNA sequence in the promoter of specific genes to activate transcription [20–23].

The p53-responsive genes (PRGs) so far identified are those involved in cell cycle control, such as p21 [21,24], GADD45 [25], B99 [26] and 14-3-3σ [27]. In addition, an increasing number of PRGs associated with the apoptotic process have been isolated [28]. These include Bax [29], Fas [30–32], IGF-BP3 [20], Dr5/KILLER [33], PAG608 [34], PRGs [35], PIDD [36], DRAL [37], p53AIP1 [1], PERP [2], Noxa [38], and p53-inducible genes (PIGs) [39]. One of the PIGs, PIG3, encodes a quinone oxidoreductase, suggesting that PIG3 is possibly involved in intracellular redox regulation and the induced stimulation of reactive oxygen species would result in disruption of mitochondrial integrity and apoptosis. Alterations in mitochondrial membrane structure and function have been shown to play a crucial role in caspase-9-dependent apoptosis [40] by releasing apoptotic factors from mitochondria, including cytochrome $c$. Recently, mitochondria have been found to release a new class of proapoptotic factors, apoptosis-inducing factor (AIF), in response to various stimuli. AIF is a ubiquitously expressed flavoprotein with significant homology to bacterial oxidoreductases [41] and has NADH oxidase activity [42]. At the induction of apoptosis AIF translocates through the outer mitochondrial membrane to the cytosol and to the nucleus, resulting in the induction of nuclear chromatin condensation and large DNA fragmentation in a caspase-independent manner [41,43,44]. This AIF-dependent, caspase-in-

*Corresponding author. Fax: (1)-314-362 9790.  
E-mail address: horikoshi@radonc.wustl.edu (N. Horikoshi).

1 Present address: Hokkaido University Graduate School of Dental Medicine, Sapporo 060-8586, Japan.  
2 These authors contributed equally.  
3 Present address: University of Connecticut, School of Dental Medicine, Farmington, CT 06030, USA.

0014-5793/02/$22.00 © 2002 Federation of European Biochemical Societies. Published by Elsevier Science B.V. All rights reserved.  
PII: S0014-5793(02)03049-1

1/1 AGC AGT GCC TGA TTT GAG ATG GGG TCC CAG GTC TCG GGA TCG GGA GCT CTG CAC GTG Met gly ser glu val ser val glu ser gly ala leu his val

61/15 GTG ATT GTG GGT GGC TTT GGC GGC ATC GCA GCA GCC AGC CAG CTG CAG CTC AAC val ile val gly gly phe gly gly ile ala ala ser gin leu gin ala leu asn

121/35 GTC CCC TTC ATG CGG GAC ATG AAG GAC TCC TTC TCC CAC CAC AAT GTG GCT CTC CGA val pro phe met leu val asp met lys asp ser Phe his his asn val ala ala leu arg

181/55 GCC TCC GGC GAG ACA GGG TTC GCC AAA AAG ACA TTC TCT TAC TCG TAC GTG ACT TTC AAG ala ser val glu thr gly phe ala lys lys thr Phe ile ser tyr ser val thr Phe lys

241/75 GAC BAC TTC CGG CAG GGG CTA GTA GTG GGG ATA GAC CTG AAG AAC CAG ATG GTG CTG CTG met val arg glx val leu val gly ile asp leu lys asn glx met val leu leu

301/95 CAG GTG GCC GAG GCC CTG CCC TTC TCT CAT CTT ATC ATC CTG GCC GCC ACC GCC ACT GGG CCC gln gly gly glu ala leu pro phe ser his leu ile leu ala thr gly ser thr gly pro

361/115 TTC CGG GAA TTT AAT GAG GFT TCC AGC CAG GAC GCC GCT ATC TAT TAC TCG TAC GTG ACT TTC AAG Phe Pro gly lys phe asn glu val ser ser glx glx ala ile glx ala tyr glu asp

421/135 ATG GTG AGG CAG GTC CGC TCA CGG TTC ATC GNG GTG GTG GGA GGC TCG GCT GGA met val arg glx val glx arg ser arg phe ile val val gly gly gly ser ala gly

481/155 GTG GAG ATG GCA GCA GAG ATM AAA ACA GAA TAT CCT GAG AAA GAG GTC ACT CTC ATT CAC val glu met ala ala glu ile lys thr glu tyr pro glu lys glu val thr leu ile his

541/175 TCC CAA GTG GCC CTG CTC CTC CCC TCC CTG CTC CCC TCC GTC GTC GAG GAA GTG GAG GAG ser glx val ala leu ala asp lys glu leu leu pro ser val arg glx glu val lys glu

601/195 ATC CTC CTC CGG AAG GGC GTG CAG CTG CTG AGT GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG GAG G
Y. Ohiro et al./FEBS Letters 524 (2002) 163–171

C

![Image Description](image.png)

Fig. 1. Identification of the PRG3 gene. A: DNA and predicted amino acid sequence of PRG3. DNA sequence revealed that PRG3 consists of 373 amino acids. The in-frame stop codon in the 5′-non-coding region is underlined at position 12. The previously identified cDNA fragment located in the 3′UTR at position 1742–2008 is indicated with an underline. B: Alignment of amino acids sequences of PRG3 and oxidoreductases. Identical amino acids are highlighted in white against a black background, and similar amino acids are shown against a gray background. rubB: *E. coli* rubredoxin NAD⁺ reductase, GenBank accession number C65051; CAMA: *P. putida* putidaredoxin reductase, GenBank accession number P16640; slr1743: *Synechocystis* sp. NADH dehydrogenase, GenBank accession number S74822; AIF: *Homo sapiens* AIF, GenBank accession number AAD16436. C: Chromosomal localization of PRG3 gene. The genomic DNA fragment containing the PRG3 gene was labeled with digoxigenin-dUTP and hybridized to synchronized human lymphocyte metaphase spreads. The probe was shown to localize to 10q21.3-q22.1 (arrows) as judged by counterstaining with DAPI and actinomycin D (for a DA-DAPI banding pattern) or PI (for an R banding pattern, shown).

dependent apoptosis has been shown to be essential to the cavitation of embryoid bodies [45]. Here we report that PRG3 encodes an apoptotic cytoplasmic protein with significant homology to bacterial oxidoreductase and to AIF released from mitochondria. Thus, PRG3 may provide a novel mechanism for p53-dependent apoptosis.

## 2. Materials and methods

### 2.1. Cell culture and cloning

Saos2, H1299, and HeLa cells were maintained at 5% CO₂ in Dulbecco's modified Eagle's medium (Life Technologies, Rockville MD, USA) supplemented with 10% fetal calf serum (FCS). EB1 cells were maintained at 5% CO₂ in McCoy's 5A medium (Life Technologies) supplemented with 10% FCS. TK6 cells were maintained at 5% CO₂ in RPMI 1640 medium (Life Technologies) supplemented with 10% FCS. A cDNA library, prepared from poly(A)⁺ RNA extracted from EB1 cells treated with 0.1 mM ZnCl₂ for 9 h was screened with the original PRG3 cDNA fragment as a probe. This screen yielded a few clones containing the entire open reading frame of PRG3 (GenBank accession number AF337957). The NH₂-terminus (1–37) deletion mutant PRG3 was constructed by polymerase chain reaction using the forward primer 5′-GGGGAATTCATGCTGGTGGACATGAAG-GACTCC-3′ and the reverse primer 5′-GGGCTCGAGTCAAGGA-GACTGCCTCATGGT-3′. The coding region was subcloned into mammalian expression vectors pcDNA3.1 or pCMV-FLAG2. A luciferase reporter plasmid was constructed with pTI-Luc [46] by inserting a p53-binding site element in the PRG3 gene: 5′-AGA-CATGCCTGGAAGCAGTAATTTTAAACAAGCTT-3′.

### 2.2. Fluorescence in situ hybridization (FISH)

A genomic clone of the PRG3 gene was isolated from a bacterial artificial chromosome (BAC) human genomic library with a cDNA probe under high stringency conditions (65°C 0.1× saline sodium citrate (SSC), 0.1% sodium dodecyl sulfate (SDS)). The identity of a positive BAC clone was confirmed by partial sequence analysis (data not shown). FISH was performed using established methods on methotrexate/thymidine or thymidine/bromodeoxyuridine-synchronized,

phytohemagglutinin-stimulated, normal peripheral blood lymphocytes [47]. The probe was preincubated for 30 min with a mixture of sonicated human DNA (Sigma, St. Louis, MO, USA) and Cot1 DNA (Life Technologies) to reduce the background. The stained slides were counterstained with 4,6-diamidino-2-phenylindole (DAPI) and actinomycin D (for a DA-DAPI banding pattern) or propidium iodide (PI; for an R banding pattern), mounted in an antifade medium and analyzed under a Zeiss Axioplan 2TM microscope. Approximately 50 metaphase spreads were examined. At least one specific probe signal was present in more than 90% of the mitoses examined. Images were captured using a cooled CCD camera (Photometrics PXL 1400, Photometrics, Tucson, AZ, USA). Digital alignment of the images from each fluorescence was applied after registration calibration through a triple band pass filter (FITC/Texas Red/DAPI) using commercial software (Electronic Photography, Biological Detection Inc., Pittsburgh, PA, USA) to minimize the registration error.

### 2.3. Northern blotting

Total RNA was extracted with Trizol (Life Technologies). TK6 cells were treated with 0.2 μg/ml doxorubicin for 15 h, and total RNA was isolated. 20 μg of total RNA was separated on a 1% agarose/1× MOPS/2% formaldehyde gel and transferred to nylon membranes (Hybond-N, Amersham, Piscataway, NJ, USA) in 10× SSPE and UV cross-linked to the membrane before pre-hybridizing. Filters were hybridized with random-primed ³²P-labeled probes in QuickHybri Solution (Stratagene, La Jolla, CA, USA) at 68°C for 1 h. Hybridized blots were washed twice at high stringency in a solution of 0.1× SSPE/0.1% SDS at 55°C and exposed to a Kodak X-Omat AR film (Fisher Scientific, Pittsburgh, PA, USA). Tissue-specific expression of the human PRG3 gene was evaluated using human multiple tissue Northern blots (Clontech, Palo Alto, CA, USA). Membranes were hybridized with radiolabeled human PRG3 cDNA probes in ExpressHyb Solution (Clontech) at 68°C for 2 h. Hybridized blots were washed twice at high stringency in a solution of 0.1× SSC/0.1% SDS at 50°C. Autoradiography was performed for 1–3 days using Kodak X-Omat AR film.

### 2.4. Functional assay for PRG3

The cellular localization of the PRG3 protein was determined by transient transfection of a FLAG-tagged PRG3 expression plasmid

into HeLa cells. Twenty-four hours post-transfection, cells were fixed and the localization of the expressed FLAG-PRG3 protein was identified by immunostaining with an anti-FLAG antibody (M2, Sigma) followed by detection with a FITC-conjugated anti-mouse IgG antibody. Mitochondria were visualized by transfecting a pEYFP-Mito plasmid (Clontech). PRG3-dependent apoptosis was examined in HeLa cells transfected with 2 μg of a PRG3 or PRG3ΔN expression plasmids together with 0.4 μg of pmemGFPcon, an expression plasmid for membrane-anchored green fluorescent protein (GFP). Three days after transfection, cells were fixed with methanol and stained with DAPI to observe nuclear morphology using a fluorescent microscope. The percentage of apoptotic cells was determined by dividing the number of GFP-positive cells with apoptotic nuclear morphology by the total number of GFP-positive cells. At least 400 cells from four randomly selected fields were counted in each experiment. Sub-G1 fractions were analyzed by FACS as follows. H1299 cells (2 × 10⁵ cells/6 cm dish) were co-transfected with PRG3 or PRG3ΔN expression plasmids and pmemGFPcon. Three days later, cells were recovered and fixed with 70% EtOH followed by PI stain (0.5 μg/ml). DNA contents in memGFP-positive cells were analyzed by FACS (FACS 440, Beckton Dickinson, Mountain View, CA, USA).

### 2.5. Reporter gene assay

H1299 cells (1 × 10⁵/well) were transfected with pTI-Luc plasmid containing a p53-responsive element from PRG3 by Lipofectamine (Life Technology). Two days after transfection, cells were lysed in Reporter Lysis Buffer (Promega, Madison, WI, USA) and luciferase activities were measured by a luminometer using an Enhanced Luciferase Assay kit (BD Biosciences, San Diego, CA, USA).

### 2.6. Sequence analysis

Searches of the protein sequence database were performed using the PSI-BLAST program and COGnitor. Sequence alignment was performed by ClustalW (version 1.7), a multiple sequence alignment program, and displayed in BEAUTY. The genomic sequence of PRG3 in the human chromosome 10 (GI: 15294724) was analyzed for the p53-responsive element using the FindPatterns program (GCG Wisconsin Package: Accelrys, Princeton, NJ, USA).

## 3. Results

### 3.1. Cloning and structural organization of the PRG3 gene

The PRG3 transcript has been identified as one of the p53 target genes whose expression is induced in EB1 human colon cancer cells that undergo p53-dependent apoptosis [35]. EB1 cells were derived from EB cells (p53−/−) by stable transfection of a wild-type p53-expressing gene under the control of the rat metallothionein MT-1 promoter. EB1 cells undergo massive apoptosis upon induction of wild-type p53 by exposure to 0.1 mM ZnCl₂ for 2–3 days [48]. To further characterize the PRG3 transcript, a cDNA library was constructed from EB1 cells treated with 0.1 mM ZnCl₂ for 9 h and used for the cDNA cloning. Half a million bacterial colonies were screened by hybridization using the previously identified PRG3 cDNA fragment as a probe. Positive clones were analyzed and sequenced.

The predicted amino acid sequence revealed that PRG3 encodes a novel protein containing 374 amino acids with an approximate molecular mass of 40.5 kDa (Fig. 1A). The NH₂-terminus of PRG3 (amino acid positions 13–35) contains a signature motif for the bacterial aromatic ring hydroxylases (flavoprotein monooxygenases): a conserved dinucleotide-binding motif that is found associated with a 'βαβ' fold [49]. The amino acid sequence of PRG3 shows 24% identity and 45% similarity with *Escherichia coli* rubredoxin NAD⁺ reductase, 26% identity and 41% similarity with *Pseudomonas putida* putidaredoxin reductase, and 25% identity and 40% similarity with *Synechocystis* NADH dehydrogenase (Fig. 1B). PRG3 does not share any homology with the known components of the respiratory chain. Instead, it displays a homology with the recently identified human AIF [41], with 22% identity and 44% similarity. AIF is a flavoprotein and an active quinone oxidoreductase with a relative molecular mass

---

**Fig. 2. p53-dependent expression of the PRG3 gene.**  
**A:** Expression of PRG3 mRNAs in EB1 cells. Total RNA was isolated from EB1 cells at 0, 4, 9, and 13 h after the treatment with 0.1 mM ZnCl₂ and analyzed by Northern blotting for the expression of PRG3, p21, and β-actin.  
**B:** PRG3 mRNA is induced by the activation of endogenous p53. Human lymphoblast TK6 (p53+/+) cells were exposed to 0.2 μg/ml doxorubicin for 15 h. Total RNA was isolated from cells and analyzed for the expression of the PRG3 (top) and p21/Waf1 (middle) genes by Northern blotting. The staining pattern of RNA by methylene blue is shown (bottom).

of 57 kDa. AIF is normally confined to mitochondria, but translocates to the nucleus after being released from mitochondria upon induction of apoptosis. Unlike AIF, the predicted amino acid sequence of PRG3 does not contain an obvious mitochondrial or nuclear localization sequence.

### 3.2. Chromosomal localization of the PRG3 gene

A genomic DNA fragment containing the PRG3 gene was isolated from a human BAC genomic DNA library by hybridization screening using PRG3 cDNA as a probe. The identity of the genomic DNA fragment was confirmed by partial sequencing (data not shown). The fragment was labeled with digoxigenin-dUTP and hybridized to synchronized human lymphocyte metaphase spreads. The probe was shown to localize to 10q21.3–q22.1 as judged by both DA-DAPI and PI banded metaphase chromosomes (Fig. 1C). These results also indicate that the PRG3 gene is a unique gene. This region is mutated in a variety of cancers, including thyroid, leukemia, uterus, lung, ovary, large intestine, breast, brain, and bladder (Recurrent Chromosome Aberrations in Cancer, Mitelman Database of Chromosome Aberrations in Cancer (2001) Mitelman, F., Johansson, B. and Mertens, F. (Eds.), [http://cgap.nci.nih.gov/Chromosomes/Mitelman](http://cgap.nci.nih.gov/Chromosomes/Mitelman)). Thus, it is possible that mutations to the PRG3 gene could contribute to human tumorigenesis.

#### Fig. 4. Expression profile of PRG3 gene in human tissues. A human multiple tissue Northern blot was probed with a cloned PRG3 cDNA. Lanes: 1, heart; 2, brain; 3, placenta; 4, lung; 5, liver; 6, skeletal muscle; 7, kidney; 8, pancreas.

### 3.3. Expression of the PRG3 gene induced by p53

The expression profile of the PRG3 gene in EB1 cells was studied by Northern blotting. Total RNA was prepared from EB1 cells treated with 0.1 mM ZnCl₂ for 0, 4, 9, and 13 h. Hybridization was performed using a cDNA fragment of the PRG3-coding region as a probe (Fig. 2A). Northern blotting revealed two transcripts expressed from the PRG3 gene following p53 induction. The larger mRNA, that has already been reported [35], is about 4.0 kb in length, and the shorter mRNA is approximately 1.8 kb. This shorter mRNA seems to be the major PRG3 transcript in EB1 cells. The induction of both PRG3 mRNAs was detectable at 4 h after the induction of p53. The maximum level of PRG3 mRNAs was reached in 9–13 h, which is slower than p21/Waf1 gene where the maximum level of expression was reached 4 h after the induction of p53. Only the larger mRNA (4.0 kb) was detectable with the probe containing the 3′-untranslated region (UTR) of the PRG3 cDNA [48], indicating that the two PRG3 mRNAs share a common coding sequence, but differ in their 3′ UTR.

The induction of PRG3 gene expression by p53 was examined in a TK6 (p53 +/+) human lymphoblast cell line. These cells retain wild-type p53 genes and the activation of p53 by ionizing radiation or chemotherapy drugs, such as doxorubicin, induces p53-dependent apoptosis. Cells were exposed to doxorubicin (0.2 μg/ml) for 15 h and total RNA was extracted from the cells. The expression of PRG3 was analyzed by Northern blotting (Fig. 2B). The PRG3 mRNA (4.0 kb transcript) was significantly induced by the induction of p53-dependent apoptosis induced by doxorubicin. The expression of Waf1/p21 was also induced under these experimental conditions. Thus, the expression of PRG3 gene is induced by the activation of authentic p53 after DNA is damaged, and the PRG3 gene is a potential PRG in p53-dependent apoptosis.

### 3.4. Identification of a p53-responsive element in the PRG3 gene

To determine whether p53 directly regulates PRG3 gene expression, we analyzed the entire genomic sequence for the presence of a p53 consensus sequence. This gene contains nine exons and the entire genomic sequence was subjected to FindPatterns analysis with the p53 consensus sequence (RRRCWWGYYYN(1–10)RRRCWWGYYY). We identified a 35 bp element containing two overlapped p53-DNA binding sites (Fig. 3A). Next, we analyzed whether this 35 bp element

#### Fig. 3. Identification of p53-responsive element in PRG3 gene. A: Genomic structure of PRG3 gene. PRG3 gene contains nine exons as indicated with black boxes. An identified p53-responsive element was shown with the alignment of p53-binding site consensus sequence. Asterisks and upper cases are matched sequence and lower cases indicate unmatched sequence with the consensus. B: Reporter gene assay for the p53-responsive element in PRG3 gene. Luciferase reporter plasmid containing a 25 bp p53-responsive element identified in PRG3 was co-transfected into H1299 (p53 –/–) cells with an empty vector, wild-type p53 expression plasmid, or R175H mutant p53 expression plasmid. Luciferase activities were measured after 2 days of transfection. Mean values of the results of three experiments are shown with error bars of standard deviations.

tochondrial localization signal peptide. The localization of the PRG3 protein in the cell was therefore studied. A full-length FLAG-tagged PRG3 protein was expressed in HeLa cells by transient transfection. Twenty-four hours post-transfection, cells were fixed and the over-expressed FLAG-PRG3 protein was identified by immunofluorescent staining with an anti-FLAG antibody. The pEYFP-Mito plasmid, which expresses the yellow fluorescent protein (YFP) in mitochondria, was used to visualize them. As shown in Fig. 5, FLAG-PRG3 clearly localized to the cytoplasm and its distribution pattern is different from YFP-Mito in the mitochondria. These results demonstrate that, in contrast to AIF, PRG3 is a cytoplasmic and not a mitochondrial protein.

Fig. 5. PRG3 is a cytoplasmic protein. FLAG-tagged PRG3 protein was expressed in HeLa cells by transient transfection. Twenty-four hours after transfection, cells were fixed and the localization of the expressed FLAG-PRG3 protein was identified by immunostaining with anti-FLAG antibody. For comparison, mitochondria were visualized by transfecting the pEYFP-Mito plasmid that expresses the YFP tagged with a mitochondria-targeting sequence adopted from subunit VIII of cytochrome c oxidase. YFP fluorescence was observed through the FITC fluorescence-filter set.

is transcriptionally responsive to p53 in a reporter gene assay. This fragment was cloned into the promoter region of the luciferase expression plasmid pTI-Luc. The luciferase activity was measured 2 days after transfection of pTI-Luc alone and in combinations with wild-type p53 or with R175H transactivation-dead mutant p53 expression plasmids. The luciferase activity was induced five-fold by co-transfection with the wild-type p53 expression plasmid. In the control co-transfection experiments with a R175H transactivation-dead mutant p53 expression plasmid, the luciferase activity was not induced (Fig. 3B). Collectively, these results support the hypothesis that the expression of the PRG3 gene in cells undergoing p53-dependent apoptosis involves direct activation of its promoter by p53.

3.5. Expression of PRG3 mRNA in human tissues

The expression pattern of the PRG3 gene in normal human tissues was studied using human multiple tissue Northern blots prepared from a variety of human tissues (Clontech). Both 4.0 kb and 1.8 kb mRNAs were detected in most tissues (Fig. 4). The highest level of PRG3 mRNA was detected in the heart; moderate levels were observed in the liver and skeletal muscles, and low levels of expression were observed in the placenta, lung, kidney, and pancreas. The expression of PRG3 mRNA was barely detectable in the brain. These results demonstrate that the relative expression levels of PRG3 mRNA vary in human tissues: a clear contrast to that of AIF where the expression is ubiquitous.

3.6. PRG3 is a cytoplasmic protein

The predicted amino acid sequence of the PRG3 protein has a homology to human and mouse AIF, but lacks a mi-

3.7. Ectopic expression of PRG3 induces apoptosis and its conserved ADP-binding motif is dispensable for its apoptotic function

The PRG3 gene is a potential p53 target gene in a p53-dependent apoptosis pathway. The PRG3 protein shares a significant sequence homology with the apoptogenic protein AIF. Thus, we tested if PRG3 protein induces apoptosis in cells. In a set of transient transfection experiments, we quantified the effect of ectopic over-expression of PRG3 on apoptosis in HeLa cells. Cells received the PRG3 expression plasmid together with a membrane-anchored GFP (memGFP) expression plasmid as a transfection marker. Three days after transfection, cells were fixed and stained with DAPI to gauge for apoptosis by monitoring nuclear morphology changes in the GFP-positive cells using a fluorescent microscope. As shown (Figs. 6A,B), over-expression of the PRG3 protein induced apoptotic nuclear morphology in 30% of the HeLa cells (Fig. 6A, panel 2, arrows, and Fig. 6B, lane 2). Transfection with a mixture of empty expression vector and memGFP expression vector (negative control) caused nuclear fragmentation in approximately 10% of the cells (Fig. 6A, panel 1 and Fig. 6B, lane 1). We observed apoptotic nuclear morphology in more than 50% of the p53-transfected cells (Fig. 6A, panel 4 and Fig. 6B, lane 4) under these conditions.

The predicted PRG3 protein sequence revealed a highly conserved NH₂-terminal domain found in oxidoreductases from various organisms (Fig. 1B). This region of PRG3 (amino acids 13–35) contains a signature motif for the conserved ADP-binding motif, which is essential for the enzymatic activity. We examined the contribution of the oxidoreductase activity of PRG3 to its apoptotic function. We constructed an NH₂-terminal (1–37) deletion mutant (PRG3ΔN), which lacks the putative oxidoreductase activity of PRG3. The PRG3ΔN expression plasmid was co-transfected with the memGFP-expressing plasmid into HeLa cells as described above, and the nuclear morphologies of transfected cells were analyzed. The ectopic expression of PRG3ΔN protein induced apoptosis in HeLa cells at almost the same level as

Fig. 6. PRG3 and its NH₂-terminal deletion mutant induce apoptosis in cells. A: Induction of apoptosis by ectopic expression of PRG3. Plasmids expressing PRG3, NH₂-terminal deletion mutant PRG3 (PRG3ΔN), or p53 were co-transfected with one-fifth the amount of a membrane-anchored GFP-expressing plasmid into HeLa cells. Three days after transfection, the nuclear morphology of GFP-positive cells was visualized by DAPI staining. Apoptotic cells are indicated by arrowheads. B: The percentage of apoptotic cells with S.D. was calculated from the results of A (lane numbers correspond to A). GFP: 10.6 ± 4.2%, PRG3: 29.3 ± 3.5%, PRG3ΔN: 25.2 ± 1.6%, p53: 52.5 ± 1.6%. C: Ectopic expression of PRG3 and the NH₂-terminal deletion mutant PRG3 induce apoptosis in cells. PRG3 or PRG3ΔN expression plasmids were transfected into H1299 cells together with memGFP expression plasmid. Cells were recovered 3 days after transfection and stained with PI (0.5 μg/ml). The DNA content of each cell was analyzed by FACS. The percentile of sub-G1 fraction is shown in each panel. Panel 1: memGFP alone, panel 2: PRG3, panel 3: PRG3ΔN, panel 4: p53.

Y. Ohiro et al./FEBS Letters 524 (2002) 163–171

A

|   |   |
|---|---|
| **GFP** | **PRG3** |
| 1 | 2 |

|   |   |
|---|---|
| **PRG3ΔN** | **p53** |
| 3 | 4 |

B

![Bar Chart](https://i.imgur.com/placeholder.png)

C

|   |   |
|---|---|
| **1** | **2** |
| ![Histogram 1](https://i.imgur.com/placeholder.png) | ![Histogram 2](https://i.imgur.com/placeholder.png) |
| **Counts** | **Counts** |
| **2N** **4N** | **2N** **4N** |

|   |   |
|---|---|
| **3** | **4** |
| ![Histogram 3](https://i.imgur.com/placeholder.png) | ![Histogram 4](https://i.imgur.com/placeholder.png) |
| **Counts** | **Counts** |
| **2N** **4N** | **2N** **4N** |
| **DNA Content** | **DNA Content** |

FEBS 26324 19-7-02
that induced by wild-type PRG3 (Fig. 6A, panel 3 and Fig. 6B, lane 3).

We further confirmed apoptosis in cells induced by the expression of the PRG3 protein by FACS analyses measuring sub-G1 fractions. Expression plasmids for PRG3 or PRG3ΔN were transfected into H1299 (p53 −/−) cells together with the memGFP-expressing plasmid. Three days after transfection, cells were fixed and stained with PI. The DNA content of memGFP-positive cells was analyzed by FACS and a sub-G1 fraction was quantified. The expression of both PRG3 and PRG3ΔN increased the sub-G1 fraction to 28.1% and 35.2%, respectively (Fig. 6C, panels 2 and 3), whereas the expression of memGFP showed 17.1% of sub-G1 fraction (Fig. 6C, panel 1). The expression of p53 showed 53.5% of sub-G1 fraction under the same conditions (Fig. 6C, panel 4). The proportions of apoptotic cells judged by the sub-G1 population correlated well with those measured by counting apoptotic nucleus in HeLa cells (Fig. 6A,B). These results clearly indicate that the ectopic expression of PRG3 induces apoptosis and that the NH₂-terminal domain of PRG3 does not contribute to the apoptotic function of PRG3. Therefore, the oxidoreductase activity of PRG3 is most likely dispensable for its apoptotic function.

### 4. Discussion

Activation of a cell death cascade is one of the key mechanisms by which p53 functions as a tumor suppressor. The effects of p53 are largely due to its transcription activation function because no mutant p53 so far isolated functions as a transcription activator as does wild-type p53. However, the downstream events of p53 in the process of apoptosis are still not completely understood.

We have identified a novel p53 target gene, PRG3. The expression of this gene was induced by endogenous p53, activated by DNA damage caused by doxorubicin. Furthermore, the PRG3 gene contains a putative p53-responsive element in its intron, which is a similar feature found in several other p53 target genes including Mdm2 and p53AIP. Therefore, it is likely that the induction of PRG3 expression is directly activated by p53 (Fig. 3). The activation of PRG3 gene in EB1 cells after the initiation of p53-dependent apoptosis is slower than that of p21/Waf1 (Fig. 2). This might mean that the binding affinity of p53 to the p53-responsive element in the PRG3 gene is lower than that of the p21/Waf1 gene in cells [50].

The mRNA for PRG3 is highly expressed in heart, liver, and skeletal muscles where the level of p53 expression is almost undetectable. It is unlikely that the highly expressed PRG3 mRNA in a normal human heart induces apoptosis (Fig. 4). PRG3 expressed in the heart, as well as expressed in other organs at a low level, may function as an oxidoreductase in their metabolic systems, or there may be a regulatory factor(s) or both, to suppress the apoptotic function of the protein. Recently, it has been shown that an apoptotic function of AIF is inhibited by heat shock protein 70 (HSP70) [51]. That PRG3 over-expression does not induce apoptosis in 293 cells (Ohiro et al., unpublished observation), where HSP70 protein is constitutively and highly expressed [52], raises the possibility that the apoptotic function of PRG3 is also inhibited by HSP70. Alternatively, there may be organ-specific PRG3 inhibitors expressed. For example,

many small heat shock proteins are predominantly expressed in heart and skeletal muscle [53], including recently identified HspB8/H11, a serine-threonine protein kinase whose expression is associated with cell growth [54,55]. Nevertheless, it is likely that the high level of expression of the PRG3 gene in the heart is achieved through a molecule other than p53. This suggests that there may be additional regulatory mechanisms that lead to high levels of PRG3 expression in the heart.

PRG3 protein shares homology with oxidoreductases, including the proapoptotic protein AIF (Fig. 1B); however, it does not show homology with PIG3, another p53 target gene sharing homology with oxidoreductases from several species [39]. The over-expression of the PRG3 protein was found to induce apoptosis in cells (Fig. 6). Various possibilities could be envisioned to explain how PRG3 initiates cell death. It has been reported that the expression of NADH quinone oxidoreductase 1 stabilizes p53 by inhibiting its degradation and therefore accelerates p53-dependent apoptosis in the cell [56]. This stabilization of p53 requires its oxidoreductase activity, and thus PRG3 may also stabilize p53 to support p53-dependent apoptosis. At present, it is not clear whether PRG3 is a flavoprotein; however, the putative oxidoreductase function of PRG3 protein is dispensable to the activity of PRG3 to induce apoptosis. The PRG3 NH₂-terminal deletion mutant lacking the putative ADP-binding domain effectively induced apoptosis, indicating that the ADP-binding domain, which is essential for the oxidoreductase activity, is not important for its apoptotic function (Fig. 6). Thus the domain responsible for the induction of apoptosis in PRG3 should be located within the rest of the molecule. Similar results have been reported for AIF. The AIF protein, lacking NAD oxidase activity after removing the FAD cofactor from its molecule, retains full apoptotic function [41]. Deletion mutant analysis of AIF suggests that the structure—or at least a part of the oxidoreductase protein structure—but not the enzyme activity is important for its apoptotic function [44].

The process of p53-dependent apoptosis requires mitochondria-dependent apoptotic machinery, including Apaf-1 and caspase-9, as well as the release of cytochrome c from the mitochondria [57], and is therefore a caspase-dependent mechanism. A p53 homologue recently identified in *Caenorhabditis elegans* revealed the existence of p53-dependent apoptosis without caspase activation [58]. Moreover, p53 has been shown to translocate into mitochondria upon the induction of apoptosis, and is directly involved in releasing cytochrome c from mitochondria [59]. Although it is not presently clear that translocation of p53 in mitochondria also releases AIF from mitochondria, it is possible that caspase-independent apoptosis will be initiated by p53 if AIF is released from mitochondria by such a mechanism. AIF is a proapoptotic mitochondrial protein that translocates into the nucleus after being released from mitochondria to initiate caspase-independent apoptosis by inducing chromatin condensation or DNA fragmentation in nuclei [43,44]. It is unclear how PRG3 induces apoptosis in cells. Like AIF or cytochrome c, PRG3 seems to be a phylogenetically old, bi-functional protein with an electron acceptor/donor (oxidoreductase) function and an independent apoptotic function. PRG3 shares homology with the mature form of AIF that is processed from its proapoptotic form by removing 102 NH₂-terminal residues responsible for mitochondrial localization (Fig. 1B). Lack of the mitochondrial localization signal in the PRG3 sequence
prevents the protein from entering mitochondria (Fig. 5). Similarly, it is not clear whether PRG3 translocates into the nucleus, as does AIF, to induce apoptosis since PRG3 does not have an obvious nuclear localization signal sequence. The structural similarity between mature AIF and PRG3 raises the possibility that PRG3 may be involved in a p53-dependent and caspase-independent apoptosis mechanism.

Cell death activated by p53 is stimulated by cellular stresses such as DNA damage and hypoxia. Moreover, the activation of p53-dependent apoptosis by stresses is thought to be a mechanism by which many chemotherapeutic agents act on human tumors. Therefore, understanding the p53-mediated apoptosis pathway is crucial, not only for understanding its mode of tumor suppression, but also for devising new cancer therapies aimed at reconstituting the apoptotic response in many tumors. As more than 50% of human cancers lack wild-type p53, PRG3 may be an attractive new candidate for study as a gene mediating a p53-dependent apoptotic response.

Acknowledgements: The authors thank M.J. Romanowski for his critical comments on the manuscript. The authors also thank Q. Qingqing for technical assistance. This work was partly supported by Grants from Edward Mallinckrodt Jr. Foundation (N.H.), American Heart Association (9930031, A.U.) and National Institute of Health (HL62458, A.U., CA75556, N.H.).

References

[1] Oda, K. et al. (2000) Cell 102, 849–862.
[2] Attardi, L.D., Reczek, E.E., Cosmas, C., Demicco, E.G., McCurrach, M.E., Lowe, S.W. and Jacks, T. (2000) Genes Dev. 14, 704–718.
[3] Ko, L.J. and Prives, C. (1996) Genes Dev. 10, 1054–1072.
[4] Levine, A.J. (1993) Annu. Rev. Biochem. 62, 623–651.
[5] Gottlieb, T.M. and Oren, M. (1996) Biochim. Biophys. Acta 1287, 77–102.
[6] Hansen, R. and Oren, M. (1997) Curr. Opin. Genet. Dev. 7, 46–51.
[7] Levine, A.J. (1997) Cell 88, 323–331.
[8] Clarke, A.R., Purdie, C.A., Harrison, D.J., Morris, R.G., Bird, C.C., Hooper, M.L. and Wyllie, A.H. (1993) Nature 362, 849–852.
[9] Lowe, S.W., Schmitt, E.M., Smith, S.W., Osborne, B.A. and Jacks, T. (1993) Nature 362, 847–849.
[10] Debbas, M. and White, E. (1993) Genes Dev. 7, 546–554.
[11] Hermeking, H. and Eick, D. (1994) Science 265, 2091–2093.
[12] Qin, X.Q., Livingston, D.M., Kaelin Jr., W.G. and Adams, P.D. (1994) Proc. Natl. Acad. Sci. USA 91, 10918–10922.
[13] Wagner, A.J., Kokontis, J.M. and Hay, N. (1994) Genes Dev. 8, 2817–2830.
[14] Wu, X. and Levine, A.J. (1994) Proc. Natl. Acad. Sci. USA 91, 3602–3606.
[15] Graeber, T.G., Osmanian, C., Jacks, T., Housman, D.E., Koch, C.J., Lowe, S.W. and Giaccia, A.J. (1996) Nature 379, 88–91.
[16] Linke, S.P., Clarkin, K.C., Di Leonardo, A., Tsou, A. and Wahl, G.M. (1996) Genes Dev. 10, 934–947.
[17] Yin, Y., Terauchi, Y., Solomon, G.G., Aizawa, S., Rangarajan, P.N., Yazaki, Y., Kadowaki, T. and Barrett, J.C. (1998) Nature 391, 707–710.
[18] Honda, R. and Yasuda, H. (2000) Oncogene 19, 1473–1476.
[19] Fang, S., Jensen, J.P., Ludwig, R.L., Vousden, K.H. and Weissman, A.M. (2000) J. Biol. Chem. 275, 8945–8951.
[20] Buckbinder, L., Talbott, R., Velasco-Miguel, S., Takenaka, I., Faha, B., Seizinger, B.R. and Kley, N. (1995) Nature 377, 646–649.
[21] El-Deiry, W.S. et al. (1993) Cell 75, 817–825.
[22] Wu, X., Bayle, J.H., Olson, D. and Levine, A.J. (1993) Genes Dev. 7, 1126–1132.
[23] Prives, C. (1998) Cell 95, 5–8.
[24] Harper, J.W., Adami, G.R., Wei, N., Keyomarsi, K. and Elledge, S.J. (1993) Cell 75, 805–816.
[25] Zhan, Q., Antinore, M.J., Wang, X.W., Carrier, F., Smith, M.L., Harris, C.C. and Fornace Jr., A.J. (1999) Oncogene 18, 2892–2900.
[26] Utrera, R., Collavin, L., Lazarevic, D., Delia, D. and Schneider, C. (1998) EMBO J. 17, 5015–5025.
[27] Hermeking, H., Lengauer, C., Polyak, K., He, T.C., Zhang, L., Thiagalingam, S., Kinzler, K.W. and Vogelstein, B. (1997) Mol. Cell 1, 3–11.
[28] El-Deiry, W.S. (1998) Semin. Cancer Biol. 8, 345–357.
[29] Miyashita, T. and Reed, J.C. (1995) Cell 80, 293–299.
[30] Muller, M. et al. (1998) J. Exp. Med. 188, 2033–2045.
[31] Owen-Schaub, L.B. et al. (1995) Mol. Cell Biol. 15, 3032–3040.
[32] Reinke, V. and Lozano, G. (1997) Oncogene 15, 1527–1534.
[33] Wu, G.S. et al. (1997) Nat. Genet. 17, 141–143.
[34] Israeli, D., Tessler, E., Haupt, Y., Elkeles, A., Wilder, S., Amson, R., Telerman, A. and Oren, M. (1997) EMBO J. 16, 4384–4392.
[35] Horikoshi, N., Cong, J., Kley, N. and Shenk, T. (1999) Biochem. Biophys. Res. Commun. 261, 864–869.
[36] Lin, Y., Ma, W. and Benchimol, S. (2000) Nat. Genet. 26, 122–127.
[37] Scholl, F.A., McLoughlin, P., Ehler, E., de Giovanni, C. and Schafer, B.W. (2000) J. Cell Biol. 151, 495–506.
[38] Oda, E. et al. (2000) Science 288, 1053–1058.
[39] Polyak, K., Xia, Y., Zweier, J.L., Kinzler, K.W. and Vogelstein, B. (1997) Nature 389, 300–305.
[40] Green, D. and Kroemer, G. (1998) Trends Cell Biol. 8, 267–271.
[41] Susin, S.A. et al. (1999) Nature 397, 441–446.
[42] Miramar, M.D. et al. (2001) J. Biol. Chem. 276, 16391–16398.
[43] Daugas, E. et al. (2000) FASEB J. 14, 729–739.
[44] Daugas, E., Nochy, D., Ravagnan, L., Loeffler, M., Susin, S.A., Zamzami, N. and Kroemer, G. (2000) FEBS Lett. 476, 118–123.
[45] Joza, N. et al. (2001) Nature 410, 549–554.
[46] Horikoshi, N., Usheva, A., Chen, J., Levine, A.J., Weinmann, R. and Shenk, T. (1995) Mol. Cell Biol. 15, 227–234.
[47] Demetrick, D.J., Matsumoto, S., Hannon, G.J., Okamoto, K., Xiong, Y., Zhang, H. and Beach, D.H. (1995) Cytogenet. Cell Genet. 69, 190–192.
[48] Shaw, P., Bovey, R., Tardy, S., Sahli, R., Sordat, B. and Costa, J. (1992) Proc. Natl. Acad. Sci. USA 89, 4495–4499.
[49] Vallon, O. (2000) Proteins 38, 95–114.
[50] Szak, S.T., Mays, D. and Pietenpol, J.A. (2001) Mol. Cell Biol. 21, 3375–3386.
[51] Ravagnan, L. et al. (2001) Nat. Cell Biol. 3, 839–843.
[52] Kondo, T., Matsuda, T., Tashima, M., Umehara, H., Domae, N., Yokoyama, K., Uchiyama, T. and Okazaki, T. (2000) J. Biol. Chem. 275, 8872–8879.
[53] Sugiyama, Y. et al. (2000) J. Biol. Chem. 275, 1095–1104.
[54] Kappe, G., Verschuure, P., Philipsen, R.L., Staalduinen, A.A., Van de Boogaart, P., Boelens, W.C. and De Jong, W.W. (2001) Biochim. Biophys. Acta 1520, 1–6.
[55] Smith, C.C., Yu, Y.X., Kulka, M. and Aurelian, L. (2000) J. Biol. Chem. 275, 25690–25699.
[56] Asher, G., Lotem, J., Cohen, B., Sachs, L. and Shaul, Y. (2001) Proc. Natl. Acad. Sci. USA 98, 1188–1193.
[57] Soengas, M.S., Alarcon, R.M., Yoshida, H., Giaccia, A.J., Hakem, R., Mak, T.W. and Lowe, S.W. (1999) Science 284, 156–159.
[58] Derry, W.B., Putzke, A.P. and Rothman, J.H. (2001) Science 13, 13.
[59] Moll, U.M. and Zaika, A. (2001) FEBS Lett. 493, 65–69.
