
The biosynthesis of adenosylcobalamin (vitamin B₁₂)

Martin J. Warren, ${}^{a}$ Evelyne Raux, ${}^{a}$ Heidi L. Schubert ${}^{b}$ and Jorge C. Escalante-Semerena ${}^{c}$

${}^{a}$ School of Biological Sciences, Queen Mary, University of London, Mile End Road, London, UK E1 4NS. E-mail: m.j.warren@qmul.ac.uk; Fax: +44 20 7882 7609; Tel: +44 20 7882 7718  
${}^{b}$ Department of Biochemistry, University of Utah, Salt Lake City, Utah 84132, USA  
${}^{c}$ Department of Bacteriology, University of Wisconsin, Madison, Wisconsin 53706-1521, USA  

Received (in Cambridge) 1st May 2002  
First published as an Advance Article on the web 17th June 2002  

Vitamin B₁₂, or cobalamin, is one of the most structurally complex small molecules made in Nature. Major progress has been made over the past decade in understanding how this synthesis is accomplished. This review covers some of the most important findings that have been made and provides the reader with a complete description of the transformation of uroporphyrinogen III into adenosylcobalamin (AdoCbl). 183 references are cited.

| 1.0 | Introduction and background† |
| --- | --- |
| 1.1 | Early studies on corrin ring synthesis |
| 2.0 | The start of the cobalamin pathway—the synthesis of precorrin-2 |
| 2.1 | Uroporphyrinogen III methyltransferase |
| 2.2 | The role of CysG |
| 3.0 | Horses for courses—different corrin biosynthetic pathways |
| 3.1 | Precorrin-3 to adenosylcobyrinic acid—the aerobic pathway |
| 3.1.1 | Precorrin-3A and B |
| 3.1.2 | Precorrin-4 |
| 3.1.3 | Precorrin-5 |
| 3.1.4 | Precorrin-6A and B |
| 3.1.5 | Precorrin-8X and hydrogenobyrinic acid |
| 3.1.6 | Amidations, cobalt chelation and adenosylation |
| 3.2 | The anaerobic biosynthesis of adenosylcobyrinic acid |
| 3.2.1 | Early cobalt chelation |

| 3.2.2 | Cobalt-dependent ring contraction |
| --- | --- |
| 3.2.3 | Methyltransferase structure |
| 3.2.4 | Missing intermediates |
| 4 | Biosynthesis of adenosylcobinamide (phosphate) |
| 5.0 | Topping and tailing: the upper and lower axial ligands |
| 6.0 | Attachment of the 5-deoxyadenosine upper (CoB) ligand to the corrin ring |
| 6.1 | Importance of the corrinoid adenosylation pathway |
| 6.2 | Reduction of the cobalt ion of the corrin ring from Co(III) to Co(I) |
| 6.3 | Formation of the Co–C bond |
| 6.4 | Structural features of the ATP:co(I) rrinoid adenosyltransferase (CobA) enzyme of S. enterica |
| 6.5 | Point of adenylation during de novo corrin ring biosynthesis |
| 6.6 | Other adenosyltransferase enzymes |
| 7.0 | The nucleotide loop assembly (NLA) pathway |

Martin Warren is Professor of Biochemistry at Queen Mary, University of London. For the last 15 years he has worked on the biosynthesis of tetrapyrroles, in particular concentrating on the biosynthesis of the tetrapyrrole macrocycle and its transformation into the corrin component of vitamin B₁₂.

Evelyne Raux is a senior research associate in the School of Biological Sciences, Queen Mary, University of London. She first started to work on cobalamin biosynthesis with a small French biotech company in 1988 before moving to London in 1995.

Heidi Schubert is a Research Assistant Professor of Biochemistry at the University of Utah. She began structural studies on tetrapyrrole biosynthetic enzymes in 1996 focusing on metal ion chelation, including the anaerobic cobalt chelatase involved in cobalamin formation and the ferrochelatase involved in sirohaem formation.

Jorge C. Escalante-Semerena is the Ira Baldwin Professor of Bacteriology at the University of Wisconsin-Madison. His work on coenzyme B₁₂ synthesis has focused on the analysis of the late steps of the pathway, the synthesis of the lower ligand base, and the attachment of the upper ligand.

390 Nat. Prod. Rep., 2002, 19, 390–412  
DOI: 10.1039/b108967f  

This journal is © The Royal Society of Chemistry 2002

8.0 Activation of adenosylcobinamide (AdoCbi)
8.1 Reactions catalyzed by the CobU enzyme
8.2 Structural features of the CobU enzyme
8.3 The adenosylcobinamide kinase activity of the S. enterica CobU enzyme
8.4 The guanylyltransferase activity of CobU
8.5 Cobamide-producing archaea possess a nonorthologous, functionally simpler cobU replacement
8.6 The kinase and guanylyltransferase activities of CobU play different physiological roles
9.0 Synthesis of the lower (Coa) ligand, 5,6-dimethylbenzimidazole
9.1 Activation of the lower ligand base
9.2 General features of the PRTase enzyme
9.3 Three-dimensional structure of the S. enterica NaMN:DMB PRTase enzyme
9.4 Specificity of the NaMN:DMB PRTase enzyme for its base substrate
9.5 The phosphatase reaction
9.6 Phenotypes of a S. enterica cobC mutant strain
9.7 Timing of phosphate removal by CobC
9.8 The last step of the NLA pathway: joining of AdoCbi-GDP and α-ribazole
10 Summary
11 Acknowledgements
12 References

1.0 Introduction and background†

It is 15 years since *Natural Product Reports* last published a review on cobalamin biosynthesis and during the intervening period giant steps have been taken towards understanding how this most complex of small molecules is made. Vitamin B₁₂ came to prominence in the 1920s after Minot and Murphy first described how they could cure the symptoms of pernicious anaemia by feeding patients with crude liver extract.¹ A race to discover the form of this liver (or extrinsic) factor resulted in its isolation and subsequent crystallisation in 1948.²³ The factor was given the name vitamin B₁₂ and, as it was shown to contain a cobalt ion, it was also referred to as cobalamin. The structure of cyanocobalamin was eventually deduced by Dorothy Hodgkin using X-ray crystallography⁴ and revealed the vitamin

to be a molecule of enormous complexity, containing a ring-contracted porphinoid with the cobalt ion ligated at the centre of the tetrapyrrole-derived macrocycle (Fig. 1). The cobalt ion is further held in place by a lower axial base (a dimethyl-benzimidazole) and an upper cyano group. The cyano group is an artificial ligand, there as a result of the extraction procedure. In biological systems, two different upper axial ligands are found, either an adenosyl group or a methyl group, giving either AdoCbl or methylcobalamin (Fig. 1). These are used in Nature to catalyse either rearrangement/reductase reactions (*e.g.* meth- ylmalonyl CoA mutase and type II ribonucleotide reductases) or methyl transfer reaction such as those found in the metabolism of methanogenic bacteria or in cobalamin-dependent methionine synthesis (see for example in ref. 5).

With the structure and function of biological forms of cobalamin now established, intense interest focused on how Nature was able to construct such a complex molecule. A superb account detailing the history and involvement of many leading chemists in the early days of tetrapyrrole and vitamin B₁₂ research has recently been published in *Natural Product Reports* by Sir Alan Battersby,⁶ who has himself played a pivotal role in helping to elucidate one of the pathways used by bacteria to make cobalamin.⁷ Research on cobalamin biosynthesis up to 1985 has also been covered in a series of reviews by Finian Leeper in *Natural Product Reports*.⁸⁻¹⁰ These accounts dealt with tetrapyrrole biosynthesis as a whole. However, in terms of cobalamin biosynthesis, it is interesting to note that up until 1985 no gene sequences for any cobalamin-specific enzyme had been identified, no individual enzyme purified to homogeneity and no protein structure deduced.

Vitamin B₁₂ is exceptional in comparison to other vitamins and coenzymes for several reasons. Firstly, as alluded to earlier, there is its structural complexity, which is also reflected in its biosynthetic requirements such that somewhere around thirty genes are necessary for its complete *de novo* synthesis. Secondly, B₁₂ is unique amongst the vitamins in that its synthesis is restricted to certain microorganisms. There is no genetic evidence that any eukaryote is able to make cobalamin. In contrast, in the prokaryotic world genome sequencing studies have revealed that the archaea and certain eubacteria possess the genetic software that encode the cobalamin biosynthetic enzymes (Fig. 2).

1.1 Early studies on corrin ring synthesis

Early studies on cobalamin biosynthesis centred on the bacterium *Propionibacterium freudenreichii* (formerly *P. shermanii*),

† Abbreviations: adenosylcobalamin, AdoCbl; adenosylcobinamide, AdoCbi; S-adenosyl-L-methionine, SAM; S-adenosyl-L-homocysteine, SAH; 5-aminolaevulinic acid, ALA.

![Figure 1](https://i.imgur.com/yourimage.png)

Fig. 1 The structure of vitamin B₁₂ and some of its derivatives. The numbering and labelling of the molecule are shown together with the names used to refer to incomplete derivatives of the compound.

| 'AEROBIC' or 'LATE COBALT INSERTION' PATHWAY | 'ANAEROBIC' or 'EARLY COBALT INSERTION' PATHWAY |
| --- | --- |
| uroporphyrinogen III<br> cobA<br> ↓<br> precorrin-2<br> -<br> cobI<br> ↓<br> precorrin-3A<br> cobG<br> ↓<br> precorrin-3B<br> cobJ<br> ↓<br> precorrin-4<br> cobM<br> ↓<br> precorrin-5<br> cobF<br> ↓<br> precorrin-6A<br> cobK<br> ↓<br> precorrin-6B<br> cobL<br> ↓<br> precorrin-8x<br> cobH<br> ↓<br> hydrogenobyrinic acid<br> cobB<br> ↓<br> hydrogenobyrinic acid a,c-diamide<br> cobN,<br> cobS &<br> cobT<br> ↓<br> cob(II) yrinic a,c-diamide<br> ?<br> ↓<br> cob(I) yrinic a,c-diamide<br> cobO<br> ↓<br> ado-cob(I) yrinic a,c-diamide<br> cobQ<br> ↓<br> ado-cobyric acid<br> cobD, cobC<br> & protein α<br> ↓<br> ado-cobinamide<br> cobP<br> ↓<br> ado-GDP-cobinamide<br> cobU<br> ↓<br> ado-cobalamin<br> cobV | uroporphyrinogen III<br> methylation at C2 and C7<br> ↓<br> precorrin-2<br> cobalt insertion<br> cysG<sup>A</sup><br> cobA<br> ↓<br> precorrin-2<br> cysG<sup>B</sup><br> cbiK<br> ↓<br> cbiX<br> cobalt-precorrin-2<br> ↓<br> cobalt-precorrin-3<br> cbiL<br> -<br> ring contraction/<br> δ-lactone formation<br> ↓<br> cobalt-precorrin-4<br> cbiH<br> cbiG?<br> methylation at C11<br> ↓<br> cobalt-precorrin-5<br> cbiF<br> methylation at C1<br> acetic acid extrusion<br> acetaldehyde extrusion<br> ↓<br> cobalt-precorrin-6A<br> cbiJ<br> C18-C19 reduction<br> ↓<br> cobalt-precorrin-6B<br> methylation at C5 and C15<br> decarboxylation<br> ↓<br> cobalt-precorrin-8x<br> cbiE<br> cbiT<br> methyl rearrangement C11 to C12<br> ↓<br> cobyric acid<br> cbiC<br> a, c-amidation<br> ↓<br> cobalt insertion<br> cbiA<br> -<br> cob(II) yrinic a,c-diamide<br> ↓<br> cobalt reduction<br> cob(I) yrinic a,c-diamide<br> adenosylation<br> ↓<br> ado-cob(I) yrinic a,c-diamide<br> b, d, e, g- amidation<br> ↓<br> ado-cobyric acid<br> aminopropanol attachment at C17<br> ↓<br> ado-cobinamide (phosphate)<br> phosphorylation &<br> GMP addition<br> ↓<br> ado-GDP-cobinamide<br> α-ribazole 5' phosphate synthesis &<br> dephosphorylation<br> α-ribazole attachment<br> ↓<br> ado-cobalamin |

Fig. 2 Pathway comparisons. A comparison of the genes required for cobalamin biosynthesis between the “aerobic” and “anaerobic” pathways, which should more accurately be referred to as “late cobalt insertion” and “early cobalt insertion” pathways. The gene names and enzyme functions are given. Major genetic differences are highlighted in grey.

since it is a relatively good producer of vitamin B₁₂. By the mid-1980s, such studies had established that vitamin B₁₂'s synthesis was based on the transformation of the common tetrapyrrole primogenitor, uroporphyrinogen III. The overall transformation of uroporphyrinogen III into vitamin B₁₂ requires a number of peripheral C-methylations, ring contraction with the loss of the C-20 meso position, cobalt chelation, amidation of the majority of the carboxylic acid side chains, decarboxylation

of the acetic acid side chain on ring C, aminopropanol attachment, adenosylation and attachment of the lower base in the from of α-ribazole.

Investigations into the methylations that took place during cobalamin biosynthesis revealed that eight S-adenosyl-L-methionine (SAM) derived methyl groups are transferred to the uroporphyrinogen framework, although only seven are observed in the final product. The order of the methyl group

392 Nat. Prod. Rep., 2002, 19, 390–412

Fig. 3 Early view of the biosynthesis of cobalamin. Factor II and factor III were two of the first dedicated intermediates of corrin ring synthesis to be isolated, although it was thought likely that these were oxidized versions of the true intermediates, precorrin-2 and precorrin-3. Factor II (also known as sirohydrochlorin) is the penultimate intermediate in the biosynthesis of sirohaem. For many years, researchers tried to identify intermediates that existed in the large black box between precorrin-3 and cobyrinic acid, the next characterized intermediate.

additions had been deduced from pulse-labelling experiments, occurring in the sequence C-2, C-7, C-20, C-17, C-11 and C-1 followed by C-5 and C-15. Based on this order of methyl group attachment, a nomenclature for the corrin biosynthetic intermediates was devised whereby the intermediates are called precorrin-$n$, where $n$ refers to the number of methyl groups that have been added to the macrocyclic template. Thus, addition of the first two methyl groups to uroporphyrinogen III generates precorrin-2 and the addition of the next methyl group would generate precorrin-3.

Some of the first cobalamin biosynthetic intermediates to be isolated included factor II and factor III, isobacteriochlorins with methyl groups at C-2 and C-7, and C-2, C-7 and C-20. It was thought that these represented oxidised versions of the true intermediates, precorrin-2 and precorrin-3 since reduction of factor II back to the level of a hexahydroporphyrin resulted in improved incorporation of this compound into the then next known intermediate in the pathway, cobyrinic acid, a cobalt containing corrin (Fig. 3). Furthermore, in a surprising apparent *fait accompli*, the methyl group added to C-20 was shown to be lost during the ring contraction process as acetic acid. However, no intermediates between precorrin-3 and cobyrinic acid could be isolated and there thus existed a large black box in the understanding of how bacteria were able to mediate corrin ring synthesis (Fig. 3). This situation was to change drastically during the early 1990s. The major breakthrough in cobalamin biosynthesis research was the application of molecular genetics to the problem, a technique that complemented the chemical procedures that had been developed previously for isotopic enrichment of the pathway precursors. A team of scientists at Rhone-Poulenc (now Aventis), one of the largest commercial producers of vitamin B₁₂, set about trying to understand how cobalamin was made with a view to improving their production strain. They developed strategies to isolate and express genes in their B₁₂ producer, a bacterium which, although called *Pseudomonas denitrificans*, probably belongs to the α-proteobacteria rather than to the Pseudomonads. Using microbiological skills and recombinant DNA technology, they were able to isolate the majority of the cobalamin biosynthetic genes in this organism and subsequently to characterise the function of the encoded proteins (see Fig. 2). The cobalamin biosynthetic genes were given the prefix *cob* and the letter following the prefix refers to the order that the gene is found in the operon, rather than

relating to any particular function (Fig. 2). Shortly after the isolation of the genes for cobalamin biosynthesis from *P. denitrificans*, cobinamide biosynthetic genes were isolated from *Salmonella enterica* serovar Typhimurium and *Bacillus megaterium* (Fig. 2). These genes were given the prefix *cbi*, which, although confusing, is also in recognition that these two latter organisms operate a different, “anaerobic”, biosynthetic pathway in comparison to the “aerobic” pathway found in *P. denitrificans*. Things are further complicated in organisms that operate the “anaerobic” cobalamin biosynthetic pathway since the genes encoding enzymes for the transformation of cobinamide to AdoCbl are also given the prefix *cob*. Thus the *cobA* gene refers to two quite distinct and unrelated genes depending on whether the organism operates an “aerobic” or “anaerobic” pathway. A complete list of genes involved in cobalamin biosynthesis and the reactions the encoded enzymes catalyse is given in Fig. 2.

### 2.0 The start of the cobalamin pathway—the synthesis of precorrin-2

Vitamin B₁₂ synthesis is initiated along a branched biosynthetic pathway, since the corrin ring is based upon the same porphinoid template as the haems, chlorophylls, sirohaem, haem $d_1$ and coenzyme F₄₃₀. The template, uroporphyrinogen III, is an unsymmetrical hexahydroporphyrin isomer that represents the point at which the pathways for vitamin B₁₂ and haem/chlorophyll divide (see Fig. 4). This review will concentrate on how uroporphyrinogen III is transformed into AdoCbl. The first step in this arduous process is the synthesis of precorrin-2.

### 2.1 Uroporphyrinogen III methyltransferase

Uroporphyrinogen III methyltransferases (uro'gen III methylase) catalyse the transformation of uroporphyrinogen III into precorrin-2 (dihydrosirohydrochlorin). This compound is also thought to be the last common intermediate for the synthesis of cobalamin, sirohaem, coenzyme F₄₃₀ and haem $d_1$ (Fig. 4).

The first uroporphyrinogen III methyltransferase to be isolated and characterised was that from *Pseudomonas denitrificans*. The enzyme is encoded by the *cobA* gene and the enzyme is sometimes referred to as CobA although the precise acronym is SUMT ($S$-adenosyl-L-methionine:uroporphyrinogen III methyltransferase). The enzyme is a homodimer with a

A = CH₂CO₂H  
P = CH₂CH₂CO₂H  

![Chemical Structure](https://i.imgur.com/1234567.png)  

HAEM $d_{1}$  

COENZYME F₄₃₀  

SIROHAEM  

5-AMINOLAEVULINIC ACID → PORPHOBILINOGEN  

UROPORPHIN III  

PRECORRIN-2  

PROTOPORPHYRIN IX  

CHLOROPHYLL a  

HAEM  

Fig. 4 Branched pathway synthesis of modified tetrapyrroles. All modified tetrapyrroles, such as haem, chlorophyll, sirohaem, haem $d_{1}$, coenzyme F₄₃₀ and vitamin B₁₂ are derived from 5-aminolaevulinic acid (ALA). Two molecules of ALA are condensed to give the pyrrole porphobilinogen which is then transformed into the unsymmetrical type III isomer of uroporphyrinogen. The incorporation of ALA into both porphobilinogen and uroporphyrinogen III is shown in colour. Uroporphyrinogen III can undergo decarboxylation and oxidation to give protoporphyrin IX, the precursor for haem and chlorophyll, or can undergo methylation to produce precorrin-2, the progenitor for sirohaem, coenzyme F₄₃₀, haem $d_{1}$ and vitamin B₁₂.

subunit molecular mass of 27 kDa. In vitro studies revealed that a monomethylated compound (precorrin-1, methylated at C-2) is initially synthesised and released from the protein and subsequently used as the substrate for the second (C-7) methylation¹⁸ (Fig. 5). As with many tetrapyrrole biosynthesis-associated enzymes, CobA was found to have a very low catalytic efficiency with a turnover of 38 h⁻¹ although as an important branchpoint enzyme in the pathway it was found to be inhibited when the concentration of the substrate, uroporphyrinogen III, reached 2 μM. Such substrate inhibition is quite rare but can be understood in terms of restricting cobalamin biosynthesis. CobA is also sensitive to S-adenosyl-L-homocysteine (SAH) inhibition but not to precorrin-2 or any B₁₂ intermediates. In the presence of 5 to 100 μM of cobalt, the enzyme activity decreased from 30 to 50% respectively and cobalt complexes of factor I and II (which are the oxidised forms of precorrin-1 and precorrin-2) were isolated.¹⁸ Other cobalamin biosynthetic eubacteria such as *Bacillus megaterium* also harbour a CobA that is inhibited by the tetrapyrrole substrate, even at a concentration as low as 0.5 μM.⁴⁸ From a commercial point of view it was important to find a CobA protein that did not show substrate inhibition. Such an enzyme was isolated from the methanogenic bacterium, *Methanobacterium ivanovii*, since methanogens require such high levels of F₄₃₀ that the substrate inhibition property does not occur. The *corA* gene for the uroporphyrinogen methyltransferase from *M. ivanovii*

was therefore isolated and cloned to allow overproduction of the CorA protein, which was found not to be inhibited by concentrations of uroporphyrinogen III up to 20 μM.²² However, as no three-dimensional structure for any uroporphyrinogen III methyltransferase has yet been deduced, it is not known what residues within the protein are responsible for mediating the inhibitory properties.

### 2.2 The role of CysG

In *Escherichia coli*, which does not make cobinamide de novo, the uroporphyrinogen III methyltransferase was isolated as a much larger protein that was referred to as CysG and which was known to be involved in sirohaem synthesis.⁴⁹,⁵⁰ The 57 kDa CysG protein is in fact a multifunctional enzyme, harbouring uroporphyrinogen methyltransferase, precorrin-2 dehydrogenase and factor II (sirohydrochlorin) ferrochelatase activities.⁵¹,⁵² The N-terminus of CysG has a predicted βαβ fold motif consistent with a NAD⁺ binding site. A mutation at position G21D in the βαβ fold leads to greatly reduced dehydrogenase activity.⁵³ The C-terminus of the protein is similar to the CobA proteins and is responsible for the methyltransferase activity associated with precorrin-2 synthesis. It is likely that *cysG* has arisen from a gene fusion between a dehydrogenase/chelatase and a methyltransferase.⁵² In strict anaerobic conditions and in the absence of NAD⁺/metals, CysG converts

394 Nat. Prod. Rep., 2002, 19, 390–412

Fig. 5 Biosynthesis of precorrin-2. Precorrin-2 is synthesized from uroporphyrinogen III by two SAM-dependent methyltransferase reactions. The synthesis proceeds via a monomethylated intermediate, precorrin-1. The reaction is catalysed by uroporphyrinogen methyltransferases such as CobA or CysG.

UROPORPHYRINOGEN III → PRECORRIN-1 → PRECORRIN-2

uroporphyrinogen III into precorrin-2 by adding two methyl groups from SAM at position 2 and 7 respectively and then can add a further methyl group at C-12 leading to a trimethylated pyrrocorphin.⁵⁴ This compound has no known physiological use and is thought to result from the abnormally high enzyme concentrations used during the in vitro studies.⁵⁴ The multifunctional CysG protein is also found in the B₁₂ producer, S. enterica. It was initially shown that all cysG mutants were defective in both sirohaem and cobalamin biosynthesis⁵⁵,⁵⁶ and it was suggested that CysG is also responsible for cobalt chelation (see section 3.2.1). The N-terminal region of CysG is homologous to Met8p, an enzyme found in Saccharomyces cerevisiae that catalyses both the dehydrogenation of precorrin-2 to give factor-II (sirohydrochlorin) and also inserts ferrous iron to give sirohaem (Fig. 3). The crystal structure of this protein has recently been deduced to give an insight into how a single protein can catalyse both dehydrogenation and ferrochelation reactions.⁵⁷

### 3.0 Horses for courses—different corrin biosynthetic pathways

In bacteria there are at least two routes for the synthesis of the corrin ring component of vitamin B₁₂ (see for example refs. 45, 58, 59). These routes differ in the timing of cobalt insertion and on the method used for ring contraction, differences that are reflected at the genetic level by the presence or absence of certain unique genes. Despite these differences there are also many similarities between the pathways such that the order of methylation and amidation would appear to be the same. These similarities are also reflected at the genetic level where homologous enzymes are present in bacteria that operate the different pathways (see Fig. 2). In this section we will first review the aerobic pathway, the more fully characterised pathway that is found to operate in bacteria such as P. denitrificans.

### 3.1 Precorrin-3 to adenosylcobyrinic acid—the aerobic pathway

The first committed step in the biosynthesis of vitamin B₁₂ along the so called “aerobic” pathway involves the addition of a methyl group to C-20 of the macrocycle to give precorrin-3 (Fig. 6). The first biosynthesis of precorrin-3 was reported in 1990³⁸ with the cloning, overproduction and isolation of S-adenosyl-L-methionine:precorrin-2 methyltransferase, from a recombinant strain of Pseudomonas denitrificans. The protein was purified over 100 fold from a strain that overexpressed the cobI gene. The purified enzyme was confirmed as the cobI gene product through N-terminal sequencing. Characterisation of the enzyme revealed it to be a homodimer with a native molecular mass of 53 kDa and a subunit molecular mass of 23 kDa. An assay was devised using precorrin-2, which had been generated using a coupled enzyme system starting with 5-amino laevulinic acid (ALA). The C-20 methyltransferase activity was demonstrated by radioactive isotope incorporation, mass spectroscopy and NMR on the oxidised product of the reaction, factor III. The P. denitrificans cobI was also recombinantly produced in E. coli, and the isolated enzyme allowed the structure of precorrin-3 to be conclusively deduced by NMR.⁶⁰ This was achieved by producing precorrin-3 from a range of enriched ¹³C-labelled isotopomers of ALA. The deduction was that the structure of precorrin-3 was similar to that of precorrin-2 but that there were there were a few differences in ¹³C chemical shifts which reflect the presence of the new C-methyl group at C-20 and its influence on the electronic distribution in the dipyrrorcorphin chromophore.

#### 3.1.1 Precorrin-3A and B

One of the major surprises in cobalamin biosynthesis was discovered with the next enzyme in the pathway, which was found to convert precorrin-3 into an oxidized form of precorrin-3, i.e. maintaining the intermediate as a trimethylated intermediate.³⁵ In order to avoid confusion, the product of the previous (CobI) reaction was termed precorrin-3A whilst the new intermediate was termed precorrin-3B. The enzyme that transforms precorrin-3A into precorrin-3B was identified as CobG, a 46 kDa putative iron-sulfur containing protein.³⁵,³⁶,⁶¹ Purified CobG was found to have a green/brown colour with a maximum absorption at 400 nm. Moreover, atomic absorption analysis revealed that it contained 4.6 moles of Fe per mole of CobG. With these data and also data from protein sequence similarity, CobG is thought most likely to contain a 4Fe–4S centre although it has not been characterised by EPR. The mass of precorrin-3B was determined by mass spectrometry to be 16 mass units higher than that for precorrin-3A, corresponding to the incorporation of an oxygen atom. The structure of precorrin-3A was deduced from multiple ¹³C-labelling studies coupled with FTIR³⁶ which revealed that it contains a γ-lactone attached to ring A and a hydroxy group at C-20 (Fig. 6). CobG requires molecular oxygen for activity and the oxygen is incorporated into the macrocycle at C-20.⁶¹ How the oxygen is activated or bound to the protein is not understood. CobG therefore produces an intermediate that is set up or “spring loaded” for ring contraction whilst the actual ring contraction process is catalysed by the C-17 methyltransferase CobJ. Several mechanisms for the activity of CobG have been proposed involving either a Fe(III)-O⁺ species or the formation of a 1,20-epoxide intermediate.⁵⁸

#### 3.1.2 Precorrin-4

Precorrin-3B acts as the substrate for CobJ, which transforms it into precorrin-4 (Fig. 6), and was initially isolated in its oxidized form, factor IV, from a recombinant P. denitrificans strain deleted in cobM.⁴¹ The structure of factor IV was determined rigorously by NMR and shown to have undergone ring contraction. When the compound was reduced back to precorrin-4 it was found to be converted into a later pathway intermediate (precorrin-6A) by cell extracts in high yield. Although it has been suggested that the ring contraction step occurs before methylation at C-17,²⁶ nonetheless in the absence of SAM no

Uroporphyrinogen III → CobA methyltransferase → Precurrin-2 → Cobl methyltransferase → Precurrin-3A → CobG monooxygenase → Precurrin-3B → CobJ methyltransferase

CobM methyltransferase → Precurrin-4 → CobF methyltransferase → Precurrin-5 → CobK reductase → Precurrin-6A → CobL methyltransferase & decarboxylase → Precurrin-6B

CobH isomerase → Precurrin-8x → CobB synthase → Hydrogenobyrinic acid → CobNST cobalt chelatase (complex) → Hydrogenobyrinic acid a,c-diamide → Cob(II)yric acid a,c-diamide → Cob? reductase

Cob(l)yric acid a,c-diamide → CobO transferase → Ado-Cobyric acid a,c-diamide → CobQ synthase → Ado-Cobyric acid

Fig. 6 The “aerobic” pathway for adenosylcobyrinic acid synthesis. In the bacterium *P. denitrificans*, the complete step-by-step synthesis of adenosylcobyrinic acid has been elucidated from uroporphyrinogen III. The fourteen enzyme catalysed steps shown in this reaction scheme were determined largely by the isolation of isotopically enriched \[^{13}C\] preparations of the intermediates which were then analysed by NMR.

reaction takes place. It is thus more likely that the pinacol-type rearrangement that affords the ring contraction occurs after C-17 methylation.

### 3.1.3 Precorrin-5

An implication of the preceding isolation of factor IV was that CobM had to be responsible for the next reaction and since CobM had sequence identity with the other methyltransferases,

it suggested that CobM was responsible for the synthesis of precurrin-5 (Fig. 6),\(^{35}\) which would have been predicted to occur via methylation at position C-12. However, what was found was even more surprising in that when precurrin-4 was incubated with CobM, the methyl group was found to be added to position C-11, prior to any decarboxylation of the acetate side chain attached to C-12. This implies that another enzyme must be required to catalyse the rearrangement or migration of the methyl group from C-11 to C-12 at a later stage in the synthesis.
The acetyl group attached to C-1 of precorrin-5 was found to be extremely labile and this kinetic instability led to some confusion as to whether it was removed by CobM or by the subsequent enzyme in the pathway CobF. However, incubation of precorrin-4 with CobM firmly established that the structure of precorrin-5 contained the acetyl group still attached at C-1 (Fig. 6).²

### 3.1.4 Precorrin-6A and B

The next step in the pathway is the addition of a methyl group to C-1 (Fig. 6). This reaction is catalysed by CobF and is concomitant with the loss of the acetyl group as acetic acid. Thus CobF is both a deacetylase and a methyltransferase. It was pointed out that methylation at C-11 produces a conjugated system to hasten the elimination of the acetyl group by a reverse Claisen condensation.²⁶ This, in turn, generates an extended enamine, the prerequisite form for methylation at C-1.²⁶ CobF, however, requires SAM for the reaction to take place so that presumably the binding of SAM to the enzyme allows it to take up an active conformation that allows it to first promote the deacetylation and subsequently catalyse the methylation.⁵⁸,⁶³ The final product of this reaction is precorrin-6A (Fig. 6). This compound represents an important landmark in the elucidation of the cobalamin biosynthetic pathway as, in chronological terms, it was the first new intermediate in the corrin pathway to be discovered in a decade,³⁷,³⁹ since the discovery of factor III. Indeed, the isolation and identification of precorrin-6A revolutionised the thinking on the biosynthesis of vitamin B₁₂.⁷,²⁶ Precorrin-6A was identified from *in vitro* experiments that were developed from a *P. denitrificans* strain overexpressing a number of the cobalamin biosynthetic genes. The Rhone-Poulenc scientists, who initiated the research on *P. denitrificans*, had found that incubating precorrin-3A with a crude cell extract from this strain together with SAM and NADPH gave high yields of hydrogenobyrinic acid. However, when NADPH was omitted from the incubation a new yellow compound accumulated and labelling studies soon revealed this compound to be precorrin-6A.³⁹ Since the accumulation of precorrin-6A was dependent on the omission of NADPH, the researchers realised that the next step in the pathway was probably a reduction, to get the molecule back to the level of oxidation of a hexahydroporphyrin. The enzyme that undertook this reaction was soon discovered and shown to be encoded by *cobK*, and reaction with CobK transformed precorrin-6A into precorrin-6B by reduction of the double bond between C-18 and C-19 in the presence of NADPH²⁵ (Fig. 6). The regio- and stereospecificity of this reaction were investigated and it was shown that the hydride (from the Hₐ position on the cofactor) was delivered to C-19 of the tetrapyrrole framework, with the hydrogen at C-18 coming from water.⁶⁴,⁶⁵

### 3.1.5 Precorrin-8X and hydrogenobyrinic acid

Precorrin-6B was shown to be the substrate for CobL, a homo-octameric multifunctional enzyme with a subunit molecular mass of 43 kDa, since it not only catalysed the addition of two methyl groups to the northern and southern methylene positions at C-5 and C-15, but also catalysed the decarboxylation of the acetate side chain on C-12²⁴ (Fig. 6). It is not known in which order the reactions really take place but as no reaction is observed in the absence of SAM, it is likely that methylation takes place first. Furthermore, in some organisms, the homologous gene to *cobL* is found as two separate genes, clearly indicating that CobL has arisen as a result of a gene fusion.⁴² Based on sequence comparisons, the N-terminal region of the protein is likely to house the methyltransferase activity whilst the C-terminal region is likely to participate in the decarboxylation reaction. With the decarboxylation now complete, the scene is set to allow the migration of the methyl group, previously added to position C-11, to C-12. This

suprafacial 1,5-sigmatropic rearrangement reaction was found to be catalysed by CobH, precorrin-8X methylmutase,⁴⁰ generating hydrogenobyrinic acid (Fig. 6). The precorrin-8 intermediate is still referred to as -8X since there may exist an additional precorrin-8 intermediate that has undergone methylation at C-5 and C-15 but that has not undergone decarboxylation. Until the existence of this intermediate has been proved or disproved, the designation of -8X will remain.

The CobH enzyme has been recently crystallised and its structure determined by X-ray crystallography to 2.1 Å resolution.⁶⁶ This comparatively small homodimeric enzyme, with a subunit molecular mass of 22 kDa, contains a highly conserved histidine residue at the active site. CobH has a high affinity for its product and the structure of the enzyme bound with hydrogenobyrinic acid reveals that this histidine is in close proximity to the site of methyl migration (Fig. 7), suggesting that it may

![Structure of CobH](image.png)

Fig. 7 The structure of CobH. The precorrin-8X methyl mutase structure was determined from crystals of recombinantly produced enzyme. The enzyme has a high affinity for its product and the structure shows CobH bound with hydrogenobyrinic acid. An active site histidine, highlighted, is in close proximity to the pyrrole nitrogen on ring C and is thought to play a key role in protonation during the migration of the methyl group from C-11 to C-12.

play a pivotal role in protonation of the ring C nitrogen during the rearrangement process.

### 3.1.6 Amidations, cobalt chelation and adenosylation

With the synthesis of the corrin ring now complete, the next steps see attention paid to amidation of the side chains and the insertion of the central cobalt ion. The first of the amidations is catalysed by CobB, a homodimeric enzyme with a subunit molecular mass of 45 kDa that amidates the *a* and *c* side chains found attached to carbons C-2 and C-7.³² The enzyme requires Mg²⁺/ATP and glutamine for activity and generates hydrogenobyrinic acid *a,c*-diamide (Fig. 6) by amidating first the *c* and then the *a* side chain. This compound acts as the substrate for the cobalt chelatase, which is a two component system made from the gene products of CobN, S and T.³⁴ The first component is made from CobN, a 140 kDa protein, whilst the second is made from CobS and T, a 450 kDa complex. Cobalt chelatase activity is observed only in the presence of the two components. What is particularly interesting is that the chelation reaction requires ATP, although the stoichiometry of ATP hydrolysis to metal insertion has not been accurately determined. It is likely that the corrin substrate binds to CobN while the ATP is bound via CobST. There is also an interesting parallel here with the insertion of Mg into chlorophyll, since the magnesium

Nat. Prod. Rep., 2002, 19, 390–412   397
chelatase is also made from three separate subunits (ChlH, I and D) and requires ATP for metal insertion.⁶⁷ Indeed, there is sequence similarity between CobN and ChlH, and between CobS and ChlI, suggesting that the enzymes may also share a common structure. Why ATP is required for the metal ion insertion process is not clear although once cobalt is inserted it is very difficult to remove. Insertion of Co²⁺ generates cob(II) yrinic acid *a,c*-diamide (Fig. 6), indicating that there has been no redox change associated with the chelation process. With the metal inserted, the pathway turns its attention to ensuring the cobalt is ligated securely. To do this the cobalt ion is reduced by a reductase. This enzyme has not been fully characterised and the gene sequence is not known. However, an enzyme capable of reducing Co(II) to Co(I) has been isolated and the first five amino acids of its N-terminus reported.²³ The amino acid sequence did not correspond to any of the genes isolated in the course of the vitamin B₁₂ studies, but was found to be a flavoprotein with specificity for FAD. More discussion of the reduction of the central cobalt ion is given in section 6.3. With the generation of the Co(I) species (Fig. 6), the corrin can now be adenosylated, this step being accomplished with the assistance of the CobO enzyme, which exists as a homodimer with a subunit molecular mass of 27 kDa.³³ The enzyme has a high affinity for cob(I) yrinic acid *a,c*-diamide (Fig. 6) but is strongly inhibited by the cobalt-free substrate equivalent, hydrogenobyrinic acid *a,c*-diamide. More detail on the adenylation of the corrin ring is given in section 6. The final amidase, CobQ, amidates the side chains *b, d, e* and *g* and generates adenosylcobyrinic acid²¹ (Fig. 6). This reaction is similar to that catalysed by CobB and requires ATP and glutamine as the amide donor. The enzyme is a homodimer with a subunit molecular mass of 57 kDa. It is at this point that the two pathways, the aerobic and anaerobic routes, for cobalamin biosynthesis rejoin.

### 3.2 The anaerobic biosynthesis of adenosylcobyrinic acid

With the discovery that the cobalamin biosynthetic pathway in *P. denitrificans* required molecular oxygen, researchers immediately realised that an alternative pathway must operate in bacteria that exist in anaerobic environments such as the methanogens. At around the same time, the main cobalamin biosynthetic operon from *S. enterica* was sequenced and published⁴² and the function of the *cbi* genes characterised.⁴⁶ Although *S. enterica* can grow both aerobically and anaerobically, it only makes cobalamin under anaerobic conditions. A comparison of the cobalamin genes in *S. enterica*, which were given the prefix *cbi* (cobyric acid biosynthesis), with those from *P. denitrificans* revealed the absence of *cobG*, the gene encoding the mono-oxygenase (Fig. 2). Moreover, other genes including *cobF, cobW* and *cobE* were also absent (Fig. 2). Similarly there were genes present in the *S. enterica* operon that were absent in *P. denitrificans* such as *cbiD, G* and *K* (Fig. 2) and which were shown to be essential for cobalamin biosynthesis. This suggested that some bacteria were able to make cobalamin by a different route. Separate research also demonstrated that cobalt-containing versions of factor II and III were incorporated with greater efficiency than cobalt-free equivalents into cobyric acid by cell free extracts of *P. freudenreichii*.⁶⁸ It had also been known for some time that when *P. freudenreichii* is grown in a cobalt-free medium factors II and III accumulate, whereas in the presence of cobalt synthesis proceeds via cobyric acid (Fig. 8). This in itself suggests that cobalt insertion occurs at an early stage of the pathway, most likely at the stage of precorrin-2 or 3A. A similar conclusion was also reached from a series of pulse labelling experiments using ⁶⁰Co²⁺ and [methyl-¹³C, ¹⁴C]-SAM.⁶⁹ Together, this evidence suggested that in some bacteria early cobalt insertion is a prerequisite for corrin synthesis. Moreover, it was also shown by ¹⁸O labelling that during the biosynthesis of cobalamin in

---

*P. freudenreichii* near complete ¹⁸O–¹⁶O exchange of the acetamido group attached to C-2 took place⁷⁰ and that this exchange occurred somewhere between the biosynthesis of precorrin-2 and cobyric acid.⁷¹ This was further refined to show that the C-2 acetate had lost one oxygen atom by the time that cobyric acid had been synthesised.⁷² Such an exchange of an oxygen would be consistent with the formation of a δ-lactone intermediate during the biosynthesis of the vitamin B₁₂ in this organism. No exchange of the carboxylic acid side chain oxygens would be expected in the “aerobic” biosynthesis of vitamin B₁₂ via the γ-lactone structure described in *P. denitrificans* (Fig. 6), although, significantly, such ¹⁸O isotope exchange experiments have not yet been carried out with this bacterium.

#### 3.2.1 Early cobalt chelation

The foregoing experiments all suggested that the anaerobic pathway is likely to be initiated by the insertion of cobalt at the level of precorrin-2 (Fig. 8). The enzyme that performs this task in *S. enterica* was thought to be CysG, the enzyme responsible for the synthesis of sirohaem. Indeed, genetic evidence concurred with the idea that CysG was the chelatase and could act both as the ferrochelatase for sirohaem synthesis and the cobaltochelatase for cobalamin biosynthesis. This was based on the isolation of a number of *cysG* mutants that were cobalamin defective but were still able to make sirohaem. Some of these mutants regained the ability to produce B₁₂ in the presence of additional cobalt to the minimal medium, although this was subsequently inhibited by adding an excess of iron. The results were interpreted as proving that CysG was involved in cobalt chelation.⁷³ However, these conclusions are more difficult to understand in view of the fact that *S. enterica* has an independent precorrin-2 cobalt chelatase CbiK, which can also insert both cobalt and iron into the modified tetrapyrrole.⁷⁴ CbiK was subsequently overproduced as a recombinant protein, purified, crystallised and its structure determined to 2.4 Å resolution.⁷⁵ This revealed the enzyme to have a similar topology to protoporphyrin ferrochelatase, an example of a shared enzyme design within a branched biosynthetic pathway (Fig. 9). The enzyme contains two histidines at the active site and it is thought that these may be involved in the deprotonation of the tetrapyrrole substrate as well as in metal binding. On the basis of in vitro assays and genetic complementation experiments it has been shown that CbiK inserts cobalt into precorrin-2 (Fig. 8). The cobaltochelatase in *B. megaterium* is CbiX, a protein with an unusual polyhistidine region.⁴³ In *B. megaterium* it is thought that cobalt is inserted at the level of factor II rather than that of precorrin-2.⁷⁶ The subsequent enzyme, the C-20 methyltransferase encoded by *cbiL* is then thought to generate cobalt-precorrin-3 (Fig. 8). Indeed, CbiL from *S. enterica* has been reported to convert not only cobalt-precorrin-2 into cobalt-precorrin-3, but also cobalt-factor II into cobalt-factor III.⁷⁷ It did not discriminate between Co(II) or Co(III) redox states and the enzyme did not recognise the metal-free intermediates as substrates.

#### 3.2.2 Cobalt-dependent ring contraction

The major question about the anaerobic pathway was how it mediated the ring contraction process. A major breakthrough on how this process occurred was reported with the isolation of cobalt-factor IV (Fig. 8) from cell-free extracts of *P. freudenreichii*.⁷⁸ The structure was confirmed by NMR and radioactively-labelled cobalt-factor IV was shown to be incorporated into cobyric acid by cell-free extracts of *P. freudenreichii* (Fig. 8). However, the true biosynthetic intermediate is likely to be the dihydro-version of this compound, cobalt-precorrin-4 (Fig. 8). Cobalt-factor IV is a metal containing tetramethylated intermediate that has undergone ring contraction. It contains a δ-lactone formed from the C-2 acetic acid side chain via the

398 Nat. Prod. Rep., 2002, 19, 390–412

![image](https://i.imgur.com/your_image_url.png)

Fig. 8 The “anaerobic” pathway for corrin ring synthesis. During the anaerobic synthesis of cobalamin, which operates in bacteria such as *S. enterica*, cobalt is thought to be inserted at an early stage, chelated by CbiK into precorrin-2 to give cobalt-precorrin-2. This is methylated at C-20 by CbiL to give cobalt-precorrin-3, and is then subject to ring contraction by CbiH, yielding cobalt-precorrin-4. However, only the oxidized version of cobalt-precorrin-4 has been isolated (cobalt-factor IV). The steps between this intermediate and cobyric acid have not been elucidated and neither has it been demonstrated how cobyric acid is converted into adenosylcobyric acid.

methylated C-20 position, which is now linked only to C-1. As the C-20 position also contains a proton it was highlighted that any subsequent hydrolysis of the δ-lactone would result in the ejection of the two-carbon unit (the methylated C-20) as acetaldehyde, rather than as acetic acid, which is how the fragment is released during the “aerobic” pathway. This was subsequently proved to be the case as cell-free incubations using labelled <sup>13</sup>C ALA and [methyl-<sup>13</sup>C]-SAM allowed the released acetaldehyde to be trapped as an acetaldehyde-dimedone derivative.<sup>79</sup> Significantly, the identification of cobalt-factor IV as a δ-lactone also provides a mechanistic rationale as to why exchange of one of the oxygens on the acetate side chain of C-2 is observed. *In vitro* synthesis of cobalt-factor IV was also achieved by incubating cobalt-precorrin-3 with an *E. coli* cell lysate containing recombinant *S. enterica* CbiH, the methyltransferase equivalent to CobJ that methylates at C-17.<sup>80</sup> In the absence of any other cobalamin biosynthetic enzyme, CbiH mediates the formation

of a ring-contracted molecule that, after oxidative esterification and extraction, was isolated and identified as cobalt-factor IV (Fig. 4), again the oxidised version of the likely biosynthetic intermediate, cobalt-precorrin-4. It is thought that the ring contraction process is mediated by the prior insertion of cobalt, which may participate in the process by one or two electron chemistry.<sup>58</sup> The *in vitro* synthesised cobalt-precorrin-4 was subsequently shown to be converted into cobyric acid by cell free extracts of *P. shermanii*, although the overall yield was low.

### 3.2.3 Methyltransferase structure

However, further intermediates between cobalt-precorrin-4 and adenosylcobyric acid have not been isolated (Fig. 8). The next step in the anaerobic pathway beyond cobalt-precorrin-4 would be expected to be methylation at C-11, which should be undertaken by the next methyltransferase, CbiF. However, attempts at

Nat. Prod. Rep., 2002, 19, 390–412 399

Fig. 9 Structure of CbiK. The cobaltochelatase found in *S. enterica* has a very similar overall topology to protoporphyrin ferrochelatase, the terminal enzyme of the haem branch of the pathway, despite any primary sequence similarity. The enzyme consists of two domains and the active site is formed between their junction. The precorrin-2 substrate is thought to stack against the highlighted phenylalanine side chain in a distorted conformation to allow two protons to be removed from two of the pyrrole nitrogens by the two highlighted histidine side chains. The metal ion is thought also to be co-ordinated by these two histidines and then to be transferred to the anionic macrocyclic cavity.

incubating cobalt-precorrin-4 with CbiF have not yielded anything significant. Nonetheless, incubation of CbiF with cobalt-precorrin-3 has yielded a novel mis methylated, and presumably non-physiological, tetramethylated compound, with methyl groups at C-2, C-7, C-20 and C-11. This is similar to the cobalt-free tetramethylated compound isolated previously, which was obtained by incubating precorrin-3 with CbiF. Thus the *S. enterica* CbiF would appear to be capable of methylating both precorrin-3A and cobalt-precorrin-3 at C-11.

CbiF is of interest for another reason since it is the only cobalamin biosynthetic methyltransferase to have had its structure determined (Fig. 10). The *B. megaterium* CbiF was overproduced as a His-tagged protein purified and crystallised. The subsequent X-ray derived structure at 2.4 Å resolution reveals a bilobal enzyme with SAH bound at the active site (Fig. 10). Since all the methyltransferases are homologous, that is their sequences suggest that they have arisen from a common ancestor, it is likely that they will all adopt a similar fold. Mechanistically, the enzymes catalyse the reactions through proximity, binding their substrates in the correct orientation to allow interaction between the nucleophilicity of the porphinoid and the electrophilicity of SAM. In the aerobic pathway the 6 methyltransferases that catalyse the addition of 8 methyl groups are known (CobA, C-2 and C-7; CobI, C-20; CobM, C-11; CobF, C-1; CobL, C-5 and C-15). In the anaerobic pathway homologues are present for all the specific methylases except

Fig. 10 Structure of CbiF. This methyltransferase in the only one of the cobalamin biosynthetic methyltransferases to have had its structure determined. The crystals of CbiF form in the presence of exogenous SAM, which is found located between the two domains of the polypeptide chain as SAH. The SAH is marked in the diagram as AdoHyc. Cobalt-precorrin-4 would have to bind in the active site cavity such that the C-11 position of the tetapyrrole is in relative close proximity to the green sulfur atom of SAH.

that no equivalent is found for CobF (see Fig. 1). It has been suggested that this methylation could be catalysed by CbiD, one of the enzymes found in the anaerobic pathway for which no function has been assigned.

### 3.2.4 Missing intermediates

From cobalt-precorrin-4 to cobyric acid no further intermediates have been identified (Fig. 8). Analogies with the “aerobic” pathway can be drawn but a degree of caution is required when so doing, since the history of cobalamin biosynthesis has always been full of surprises. No role for CbiG has been put forward so it is possible that this protein may play a key role in association with the ring contraction process. As with CbiD, mutations in CbiG inhibit cobalamin biosynthesis somewhere between the conversion of precorrin-2 into cobyric acid.

Finding the enzymes/genes for cobalt reduction has proved a difficult process, and as with the aerobic pathway no enzyme dedicated to cobalt reduction has been identified (see later) and the point of adenosylation has not been clarified (see also section 6). If parallels were to be drawn with the aerobic route one might predict that adenosylation would occur shortly after cobalt insertion, but this has not yet been demonstrated. From cobyric acid to adenosylcobyric acid we can assume that the amidations occur in the same order as that described for the aerobic route (Fig. 8). Thus, there still exist significant gaps in

AMINOPROPANOL

![Chemical Structure](chemical_structure_image)

ADENOSYLCOBYRIC ACID

ATP/Mg²⁺

ADENOSYLCOBINAMIDE (PHOSPHATE)

AMINOPROPANOL-O-2-PHOSPHATE

KINASE/ATP

L-THREONINE-O-3-PHOSPHATE

L-THREONINE

Fig. 11 Biosynthesis of AdoCbi and AdoCbi-phosphate. The synthesis of AdoCbi requires the attachment of aminopropanol to the *f* propionic acid side chain. The aminopropanol is derived from threonine. Threonine is first phosphorylated to give threonine phosphate and then undergoes a decarboxylation to give aminopropanol phosphate, in a reaction that in *S.* enterica is catalysed by CobD. It is thought that free aminopropanol can also be phosphorylated directly. Aminopropanol, or aminopropanol phosphate, is then incorporated into adenosylcobyrinic acid by the action of an enzyme complex, which in *P.* denitrificans consists of two components termed α and β. In *S.* enterica, this amide bond formation is thought to be catalysed by CbiB.

our knowledge concerning the “anaerobic” biosynthesis of adenosylcobyrinic acid. Moreover, it is a bit of a misnomer to refer to the two pathways for corrin synthesis as being “anaerobic” or “aerobic”, since the “anaerobic” pathway can operate aerobically and a version of the “aerobic” pathway in *Rhodobacter capsulatus* has evolved to operate in the absence of molecular oxygen. It is perhaps more accurate to discriminate between the pathways by referring to them as the “cobalt early” and “cobalt late” routes for cobalamin synthesis.⁴⁷

### 4 Biosynthesis of adenosylcobinamide (phosphate)

The biosynthesis of adenosylcobinamide (AdoCbi) from adenosylcobyrinic acid requires the attachment of an aminopropanol group, derived from threonine, to the *f* (propionic acid) side chain of the corrin ring (Fig. 11). In *P.* denitrificans, this process is accomplished by a two-component system, designated α and β.²⁶ For full activity, α and β require adenosylcobyrinic acid, ATP/Mg and (*R*)-1-aminopropan-2-ol, although the *K*ₘ value for the latter was found to be very high at 20 mM. The α component was purified to homogeneity and was found to comprise a 38 kDa protein that was not encoded by any known gene of cobalamin biosynthesis. The β component was also purified but found to be a large multiprotein complex with a mass of greater than 1000 kDa. A similar complex was made by expressing the *P.* denitrificans *cobC* and *D* genes in *E. coli* and thus it is believed that the β component is made from the CobC and D polypeptides.²⁶

The biosynthesis of AdoCbi in *S.* enterica is thought to be mediated by CbiB, which shares sequence similarity with the *P.* denitrificans CobD, since mutations in *cbiB* lead to the accumulation of cobyrinic acid,⁴⁶ although *in vitro* activity has not been demonstrated. However, it has been proposed that the substrate for CbiB is not aminopropanol but its phosphate and that the product of the reaction is AdoCbi phosphate⁸⁶ (Fig. 11). This is based on the observation that the *S.* enterica CobD is a novel enzyme with L-threonine *O*-3-phosphate decarboxylase activity that generates (*R*)-1-aminopropan-2-ol *O*-2-phosphate. Bearing in mind the high *K*ₘ value reported for the incorporation of aminopropanol into cobinamide in *P.* denitrificans, it seems likely that this is not the true physiological substrate for the reaction in this organism either. A structure for the *S.* enterica CobD has recently been published as the 40 kDa protein has been overproduced, purified and crystallised.⁸⁷ The native protein exists as dimer, where each subunit consists of a large and small domain, but is overall very similar to members of the family of aspartate aminotransferases. In particular, the active site is most related to that observed in histidinol phosphate aminotransferase, suggesting that they be evolutionarily related.

Cobalamin biosynthesis in *S.* enterica *cobD* mutants can be restored by the addition of exogenous (*R*)-aminopropanol. It is likely therefore that a kinase is able to phosphorylate the molecule prior to its incorporation into adenosylcobyrinic acid although there is always the possibility that it may be

Nat. Prod. Rep., 2002, 19, 390–412

incorporated without the phosphate group.86 As will be highlighted later (section 8), an enzyme exists in *S. enterica* that is capable of phosphorylating AdoCbi (CobU).

### 5.0 Topping and tailing: the upper and lower axial ligands

In comparison to the other cyclic tetrapyrroles, cobamides are unique in that their structures include an upper and a lower metal ligand. This section of the review focuses on the biochemistry and genetics of upper and lower ligand attachment to the corrin ring. By necessity, we will have to revisit some of the steps covered above since there are separate, though equivalent, steps required in the salvage system employed by some bacteria to reuse the corrin ring component of the macrocycle.

### 6.0 Attachment of the 5-deoxyadenosine upper (Coβ) ligand to the corrin ring

The biologically active form of Cbl contains a 5′-deoxyadenosyl group as the upper ligand (Fig. 1). Formation of the Co–C bond between the corrin ring and the upper ligand requires the Co ion of the ring to be in its Co(I) oxidation state before the 5′-deoxyadenosyl moiety of ATP can be enzymatically transferred to it. This set of reactions is known as the corrinoid adenosylation pathway and evidence for the existence of it was first described using crude cell-free extracts of *P. freundreichii*88 and *Clostridium tetanomorphum*.89

#### 6.1 Importance of the corrinoid adenosylation pathway

Genetic evidence obtained in *S. enterica* showed that inactivation of the gene encoding the ATP:co(I)rrinoid adenosyltransferase (CobA) enzyme blocks de novo synthesis of the corrin ring, indicating that this pathway proceeds via adenosylated intermediates.90 Salvaging of incomplete corrinoids such as cobyric acid and cobinamide is also blocked in *cobA* mutant strains *S. enterica*, suggesting the corrinoid-binding enzymes involved in nucleotide loop assembly steps also require adenosylated cobinamide as substrate. In addition, in *S. enterica* the inability to adenosylate cobalamin prevents the expression of the ethanolamine utilization (*eut*) genes, thus blocking growth on ethanolamine as carbon and energy source.91 In *E. coli*, inactivation of the *btuR* gene (encoding an *S. enterica* CobA homologue) blocks the synthesis of AdoCbl, resulting in the unregulated, constitutive expression of the *btuB* gene which directs the synthesis of the outer membrane protein responsible for translocating exogenous corrinoids into the periplasmic space of the cell.92–94 The latter results point at a clear interaction between endogenous AdoCbl biosynthesis and the exogenous corrinoid transport system.

#### 6.2 Reduction of the cobalt ion of the corrin ring from Co(III) to Co(I)

Reduction of Co(III) to Co(I) requires consecutive one-electron reductive steps catalyzed by the cob(III) alamin reductase and the cob(II) alamin reductase enzymes.95–97 Researchers working with *P. freundreichii*98 and *C. tetanomorphum*95,99 partially purified and characterized the reducing systems and the ATP:co(I)rrinoid adenosyltransferase enzymes responsible for the conversion of the vitamin to the coenzymic form of Cbl. However, it was not until recently that these enzymes were isolated to homogeneity from *S. enterica* and the genes encoding them were identified.100–103 Recently published work indicated that this bacterium likely lacks a dedicated cob(III) alamin reductase for the conversion of co(III) balamin to cob(II) alamin. This conclusion was reached on the basis of *in vitro* results which showed spontaneous enzyme-independent, dihydroflavin-dependent reduction of cob(III) alamin to cob(II) alamin, on the other hand, is thermodynamically very unfavorable, and dihydroflavins are unable to drive this reduction in

the absence of an enzyme.100 An *in vitro* reducing system for the enzymatic conversion of vitamin B₁₂ to coenzyme B₁₂ was recently reported. This reducing system is comprised of ferredoxin (flavodoxin):NADP⁺ reductase (Fpr) and flavodoxin A (FldA) proteins that transfer electrons from NADPH + H⁺ onto complete or incomplete co(III)rrinoids to generate co(I)rrinoids. The latter serve as substrates for the ATP:co(I)rrinoid adenosyltransferase CobA enzyme that converts them into their corresponding adenosylated forms.101 The Fpr/FldA system is the first enzymic system to be coupled to the CobA adenosyltransferase enzyme to convert cobalamin to its coenzymic form. The current model for the corrinoid adenosylation pathway is shown in Fig. 12.

![Diagram](https://i.imgur.com/yourimage.png)

**Fig. 12** The corrinoid adenosylation pathway in *S. enterica*. In this model, reducing power for the reduction of co(III)rrinoids to co(I)rrinoids is derived from NADPH + H⁺ by the action of the ferredoxin (flavodoxin):NADP⁺ reductase (Fpr) and flavodoxin A (FldA) proteins. Fpr-FAD, oxidized form of Fpr; Fpr-FADH₂, hydroquinone form of Fpr; FldA-FMNH₂, oxidized form of FldA; FldA-FMN, semiquinone form of FldA; PPPᵢ, triphosphate.

Although this was an important finding from the standpoint of advancing our understanding of upper ligand attachment, the involvement of the Fpr/FldA reducing system in cobalamin reduction is not unprecedented, since it had been established by elegant studies of the activation/reactivation reactions of the cobalamin-dependent methionine synthase in *E. coli*.105–109 Use of the Fpr/FldA system for corrinoid reduction appears to be widely spread in nature since human cells also use it to maintain function of the methionine synthase enzyme.110 As described in section 3.1, an NADH-dependent flavoenzyme with cob(II)-yrinic acid *a,c*-diamide reductase activity was isolated to homogeneity from cell-free extracts of *P. denitrificans*, and the first six residues (MEKTRL) of the polypeptide were identified by N-terminal sequencing. The protein displayed a molecular weight of 16,000 by SDS-PAGE and 30,000 by HPLC suggesting a homodimeric structure. No further analysis of the enzyme has been reported and the gene encoding this activity was not identified.23

### 6.3 Formation of the Co–C bond

Early studies of the conversion of cobalamin to AdoCbl showed that ATP was the donor of the 5′-deoxyadenosyl group.111,112 A more detailed biochemical analysis of the reaction, however, was not performed until the 1990s when the gene encoding the ATP:co(I)rrinoid adenosyltransferase enzyme responsible for catalyzing the reaction was identified in *P. denitrificans* (*cobO*) and *S. enterica* (*cobA*).33,90,102,103

### 6.4 Structural features of the ATP:co(I)rrinoid adenosyltransferase (CobA) enzyme of *S. enterica*

It was not until very recently that the three-dimensional structure of an ATP:co(I)rrinoid adenosyltransferase enzyme was solved. The X-ray structure of the CobA enzyme of *S. enterica* in its apo form, complexed with MgATP, and complexed with Cbl and MgATP (Fig. 13) was solved to a 2.1 Å, 1.8 Å, and 2.1 Å resolution, respectively.113 The CobA enzyme does not have the cobalamin binding motif found in enzymes that use cobalamin as cofactor such as methionine synthase, methylmalonyl-CoA mutase, diol dehydratase, or glutamate mutase.114–119 Analysis of the structure of the CobA-HOCbl-MgATP ternary complex revealed a new mode of protein-

Fig. 13 Stereo view of the ternary complex between ATP:co(I)rrinoid adenosyltransferase (CobA) enzyme of S. enterica, HOCbl and ATP. Shown is the ribbon representation of the CobA enzyme complexed with its corrinoid and nucleotide substrates. In this structure only one corrinoid binding site is occupied by the substrate. The N-terminal α-helix of subunit B is shown interacting with the nucleotide loop of the HOCbl molecule bound to subunit A. The lower ligand base DMB is shown coordinated to the cobalt ion of the corrin ring. Reproduced from ref. 101 with permission.

corrinoid interactions, where the ring is held *via* a limited number of hydrophobic interactions. This is likely the reason why CobA can bind several different corrinoids as substrates. The structure of the CobA–Cbl–MgATP complex also shows that ATP is bound to a P-loop that is one residue shorter than consensus, and that the nucleotide binds to the P-loop in opposite orientation to that of ATP hydrolases, *i.e.*, in CobA the γ-phosphate of ATP binds to the location occupied by the α-phosphate in nucleotide hydrolases. The structure of the CobA–Cbl–MgATP complex shows the cobalt ion of cob(III)-alamin to be too far away from the C5′ of the ribose (~6.1 Å), suggesting that the protein is likely to undergo a conformational change upon reduction of the cobalt ion to Co(II) or Co(I), bringing the Co(I) and C5′ target to sufficient proximity for the nucleophilic attack to take place.

### 6.5 Point of adenosylation during *de novo* corrin ring biosynthesis

The point of adenosylation during *de novo* synthesis of cobinamide has been clearly established in *P. denitrificans* but not in any other prokaryote. Results from the analysis of corrinoid substrates that can be used by the CobO enzyme showed that cobyrinic acid is not a substrate for the enzyme, but cobyrinic acid *a,c*-diamide is, indicating a key role for the amide groups in substrate recognition.³³ The CobO (*P. denitrificans*) and CobA (*S. enterica*) enzymes can adenosylate cobyrinic acid, cobinamide and cobalamin.³³,¹⁰³ The point of adenosylation during *de novo* cobinamide synthesis is likely to be different in *S. enterica*. As mentioned above, prokaryotes that use the anaerobic pathway of corrin ring biosynthesis (*e.g.*, *S. enterica*, *P. freudenreichii, B. megaterium*) insert the cobalt ion into the ring very early in the pathway,⁴⁴,⁵²,⁶⁸,⁷³,⁷⁵,⁸⁰ and there are no data to suggest that conversion of cobalt-precorrin-2 ring to cobyrinic acid *a,c*,-diamide proceeds *via* non-adenosylated intermediates. The point of adenosylation in these prokaryotes remains to be established.

### 6.6 Other adenosyltransferase enzymes

Homologues of the *S. enterica* CobA enzyme can be found in a host of cobamide-producing prokaryotes. Of these only the CobO enzyme from *P. denitrificans* has been isolated to homogeneity and partially characterized.³³ Like CobA, CobO adenosylates complete and incomplete corrinoids. Genetic and biochemical evidence obtained in *S. enterica* indicates that at least two other adenosyltransferase activities exist in this bacterium. In both cases, the enzymatic activity is associated with proteins that do not share any homology with CobA or its homologues. The genes encoding additional adenosyltransferase activities in *S. enterica* are the *eutT* gene¹²⁰ and the *pduO* gene.¹²¹,¹²² The three-dimensional structures of these proteins would reveal whether different folds could provide adenosyltransferase activity.

### 7.0 The nucleotide loop assembly (NLA) pathway

The lower ligand base of cobamides is tethered to the corrin ring *via* a structure known as the nucleotide loop. Two important features of the nucleotide loop of cobamides are the *N*-glycosidic bond linking the base to the ribose (in the α rather than the usual β configuration), and the participation of the 3′-hydroxy group (rather than the 5′-OH) of the ribosyl moiety in the phosphodiester bond linking the nucleotide to the (*R*)-1-aminopropan-2-ol (AP) moiety of Cbi (Fig. 1). AdoCbl is one of only three coenzymes whose structures contain a phosphodiester bond. The other two coenzymes sharing this feature are coenzyme F₄₂₀¹²³ and methanopterin,¹²⁴ but unlike AdoCbl, the *N*-glycosidic bond in these coenzymes displays a β configuration. It should be noted that elegant studies by Eschenmoser and coworkers showed that the formation of the amide bond between the *f* carboxy function and the amino group of (*R*)-1-aminopropan-2-ol already attached to *N*¹-(3-phospho-α-D-ribosyl)-5,6-dimethylbenzimidazole nucleotide (aka α-ribazole-5-P) is, under the correct conditions, a non-enzymatic, regioselective, self-assembling process.¹²⁵ From a biochemical standpoint, however, the NLA pathway consists of three steps: i) activation of the lower ligand base; ii) activation of the precursor cobinamide; and iii) joining of the activated precursors to yield AdoCbl (Fig. 14). Although the general features of the biochemistry underpinning this branch of the AdoCbl biosynthetic pathway have been known since the late 1960s, the breakthrough that accelerated the progress of the studies of AdoCbl biosynthesis was reported in 1984 by Jeter *et al.*, who identified a super operon encoding 20 of the cobalamin biosynthetic (*cob*) genes in *S. enterica*.¹²⁶ Subsequent work showed that the last three genes of this operon (*i.e.*, *cobUST*) encode functions required for the assembly of the nucleotide loop,¹²⁷ with one additional function encoded outside of the *cob* operon.¹²⁸ The last 10 years of research have generated a

Cobinamide Activation

DMB Activation

![Diagram](https://i.imgur.com/1234567.png)

Fig. 14 Nucleotide loop assembly pathway in *S. enterica*.

great deal of structure/function information regarding the enzymes involved in this branch of the AdoCbl biosynthetic pathway.

### 8.0 Activation of adenosylcobinamide (AdoCbi)

In the late 1960s crude cell-free extracts of *P. freudenreichii* were used to study the activation of AdoCbi *in vitro*. From this work it was learned that AdoCbi is activated to AdoCbi-GDP via an AdoCbi-phosphate (Cbi-P) intermediate. It is now known that the true intermediate of the *de novo* biosynthetic pathway is AdoCbi-P not AdoCbi, and that the phosphorylation of AdoCbi is important for salvaging Cbi from the environment. The identification of the nucleotide sequence of the gene encoding the enzyme responsible for these activities in *P. denitrificans* and *S. enterica* allowed access to sufficient amounts of enzyme for more in-depth biochemical and structure/function analyses.

The enzyme responsible for the conversion of AdoCbi to AdoCbi-GDP in *S. enterica* is encoded by the *cobU* gene, and in *P. denitrificans* by the *cobP* gene. Computer comparison of the primary amino acid sequences of the CobU and CobP proteins shows that they are 42.7% identical. Interestingly, homologues of these genes are found only in bacteria. To date, none of the archaeal genome sequences contain a *cobU* homologue, indicating that the lineages of the bacterial and archaeal enzymes are distinct. The possible significance of this difference amongst cobamide producing prokaryotes is discussed below. The following discussion of the reactions that activate AdoCbi to AdoCbi-GDP is based on data obtained for the *S. enterica* CobU enzyme, which is the only enzyme that has been most extensively studied from both a biochemical and structural point of view. Given the degree of identity amongst CobU homologues, it is assumed that what has been learned about the mechanism of function of CobU is likely to hold true for all of its homologues.

#### 8.1 Reactions catalyzed by the CobU enzyme

Fig. 15 summarizes the reactions catalyzed by the CobU enzyme. Briefly, the CobU enzyme uses either ATP or GTP (it is unclear what the *in vivo* substrate is) to phosphorylate AdoCbi to yield AdoCbi-P. The enzyme uses GTP to guanylylate itself and then transfers this GMP group to AdoCbi-P to yield AdoCbi-GDP, the final product of the reaction. Most likely, *in vivo* the CobU enzyme is always guanylylated, since the large conformational change of the protein induced upon guanylylation is necessary for both kinase and guanylyltransferase activity.

#### 8.2 Structural features of the CobU enzyme

CobU is a remarkable enzyme for it has multiple biochemical activities associated with a small <20 kDa polypeptide (180 amino acids). The enzyme has AdoCbi kinase and AdoCbi-P guanylyltransferase activities that work in concert to convert AdoCbi to AdoCbi-GDP. In addition, CobU has a auto-guanylylating activity that allows it to undergo a large conformational change needed for function of both activities. The 2.3 Å resolution X-ray structure of CobU shows the quaternary structure of the enzyme to be trimeric, looking somewhat like a propeller (Fig. 16). Each subunit is comprised of a single domain with seven β-sheets, six of which run parallel to each other. The β-strands are flanked on either side by a total of five α-helices and one helical turn. The C-terminal β-strand runs antiparallel to all other β-strands. Each subunit contains a P-loop motif located at the base of a cleft formed between two subunits. In addition, each subunit of CobU contains a rare non-prolyl *cis* peptide bond between residues Glu 80 and Cys81. Residue Glu 80 faces the P-loop, and is thought to coordinate the magnesium ion of the triphosphate substrate. The apo CobU structure shows residue His 46 (the site of guanylylation) exposed to the solvent and ~21 Å away from the P-loop. The structure of CobU complexed with GMP shows that upon guanylylation CobU undergoes a substantial conformational change (Fig. 16). The helix containing His 46 rotates 30° and translates 11 Å with a compensatory unwinding and rewinding at the helix ends to allow the formation of a guanosine binding pocket between β-strand 2 and α-helix 2. This conformational change brings the Cα of His 46 ~10 Å closer to the P-loop positioning a phosphate ion in the P-loop only 6 Å away from the α-phosphate of GMP (Fig. 17). These data support the conclusion that the P-loop is used by the enzyme to coordinate the terminal phosphates in both the kinase and transferase reactions, implying that the active sites overlap. An intriguing lesson learned from the crystallographic studies of apo CobU was the remarkable topological similarities of CobU with the RecA protein. This striking homology of CobU with RecA is difficult to rationalize since it would imply that CobU probably evolved from a kinase enzyme which acquired guanylyltransferase activity, yet the guanylyltransferase activity is the one activity of the enzyme that is required for both *de novo* synthesis of AdoCbl and for salvaging exogenous Cbi, hence one would expect CobU to be a guanylyltransferase that acquired kinase activity in response to the presence of Cbi in the cell’s environment. These interesting questions remain to be answered.

#### 8.3 The adenosylcobinamide kinase activity of the *S. enterica* CobU enzyme

The kinase activity of CobU converts AdoCbi to AdoCbi-P using either ATP or GTP as γ-phosphate donor. The steady state kinetic analysis of the kinase reaction has not been performed because of a burst of activity in the first few seconds of

404 Nat. Prod. Rep., 2002, 19, 390–412

Fig. 15 Reactions catalyzed by the CobU enzyme. The CobU enzyme is thought to be always guanylylated *in vivo* due to its auto-guanylylating activity. This activity is needed to induce the catalytically active conformation of the protein. In its active conformation the enzyme uses either ATP or GTP (shown as NTP) to phosphorylate the hydroxy group of the aminopropanol moiety of AdoCbi. The phosphate group of AdoCbi-P is retained in AdoCbi-GDP, which is generated by the transfer of the GMP moiety attached to CobU onto AdoCbi-P.

the reaction, and have not been performed.¹³¹ Pre-steady state kinetic studies are required to establish the kinetics of the reaction. In the absence of this information, the reported steady state kinetic data for the *S.* enterica and the *P.* denitrificans enzymes²⁰,¹³⁴ should be taken with caution.

The CobU enzyme uses either GTP or ATP nucleotides as γ-phosphate donor substrates for the kinase reaction, and although the enzyme can bind CTP, UTP, and dCTP, product formation is not observed, suggesting the P-loop can bind other nucleotides but only GTP or ATP generate a catalytically competent conformation of the enzyme. Also important for the kinase reaction is the 2′-OH group of the ribosyl moiety of the nucleotide.¹³¹ The precise interactions of the 2′-OH group with the CobU polypeptide are yet to be established.

The upper ligand of the corrinoid substrate is required for kinase activity *in vitro*, i.e., no activity is observed if (CN)₂Cbi is used as substrate. Although highest kinase activity of CobU is observed with AdoCbi (5′-deoxyadenosylcobinamide) or dAdoCbi (2′,5′-dideoxyadenosylcobinamide), a substantial amount of activity (i.e., 65% of the activity measured with AdoCbi) is retained when GuoCbi (5′deoxyguanosylcobinamide), InoCbi (5′-deoxyinosylcobinamide), or CyoCbi (5′-deoxycytosylcobinamide) is used as substrate.¹³¹ The same observation was reported by Cameron *et al.* about the CobP enzyme of *P.* denitrificans.²⁰

It is important to note that to date activity assays for CobU have only been performed in the presence of air. This point is relevant because genetic evidence suggests that the under anaerobic growth conditions the CobU enzyme does not require an adenosylated corrinoid substrate.¹³⁶ *cobU* residues S30 and G155 are important for the use of non-adenosylated cobinamide as substrate. Mutations of these residues result in a CobU enzyme that requires AdoCbi for function under aerobic and anaerobic conditions.¹³⁴ Further studies are required to understand the reason(s) why under anoxic conditions the CobU enzyme does not require adenosylated cobinamide as substrate.

### 8.4 The guanylyltransferase activity of CobU

The conformational change that occurs upon self-guanylylation of CobU is essential for both kinase and guanylyltransferase activities of the enzyme.¹³¹,¹³⁴ As stated above, the GMP group is covalently attached to residue His46 of CobU via a phosphoramidate bond, and mutation of this residue leads to loss of both activities of the enzyme.¹²⁷,¹³¹,¹³⁷ It has been shown that the guanylyl group of the CobU-GMP intermediate is incorporated into AdoCbi-GDP.¹³¹ Because of the substantial conformational changes undergone by the enzyme during the process of self-guanylylation-transfer-self-guanylylation, it is likely this process is the rate-limiting step of AdoCbi-GDP formation *in vivo.*

It should be noted that CobU can use ATP to adenylate itself, however, unlike the CobU-GMP intermediate, the CobU-AMP complex promotes a non-productive conformational state of the enzyme that blocks kinase and transferase activities. If CobU is exposed to equimolar concentrations of ATP and GTP, GTP prevents inactivation of CobU by ATP, suggesting a higher affinity of the enzyme for GTP. It is clear from *in vitro* studies that the CobU-AMP intermediate is stable and the enzyme cannot be reactivated by GTP.¹³¹ Conformational differences between the CobU-AMP and CobU-GMP derivatives of the enzyme were demonstrated by nucleotide exchange experiments. From these experiments it was learned that the half-life of the CobU-GMP complex is much shorter (~23 s) than the half-life of the CobU-AMP complex (>4 min).¹³¹

CobU is a unique guanylyltransferase. Of the four enzymes reported to date to have guanylyltransferase activity (i.e., mRNA capping enzyme,¹³⁸⁻¹⁴¹ mannose-1-phosphate guanylyltransferase,¹⁴² and GTP:GTP guanylyltransferase,¹⁴³ three (including CobU) proceed via a covalent phosphoramidate-linked enzyme-GMP intermediate. Of these three enzymes a histidinyl-phosphoramidate intermediate has only been demonstrated in CobU, while the other two reactions proceed via an enzyme-lysyl-phosphoramidate intermediate.⁵⁵,¹⁴⁴

### 8.5 Cobamide-producing archaea possess a nonorthologous, functionally simpler *cobU* replacement

None of the archaeal genome sequences reported to date contain an orthologue of *cobU*. Since many archaea synthesize copious amounts of cobamides,¹⁴⁵ it is predicted that an alternative function to CobU exists in these prokaryotes. Indeed, the new cobamide biosynthetic *cobY* gene of *Methanobacterium thermoautotrophicum* strain ΔH was shown *in vitro* and *in vivo* to encode a protein with GTP:AdoCbi-P guanylyltransferase activity but no NTP:AdoCbi kinase activity.¹⁴⁶ Although AdoCbi-P kinase activity was not detected in crude cell-free extracts of *M.* thermoautotrophicum, it remains unclear whether this activity exists in this archaeon. Results from computer

Nat. Prod. Rep., 2002, 19, 390–412

a

Subunit C

Subunit B

Subunit A

Guanylylation

b

Subunit C

Subunit B

Subunit A

Fig. 16 Ribbon diagram of the AdoCbi kinase/AdoCbi-P guanylyltransferase (CobU) trimer isolated from *S. enterica*. (a) The apo form of the enzyme and (b) GMP bound to the enzyme. Subunits are shown in different colours with the P-loop shown in yellow, and residue His 46 (the site of guanylylation) shown as a ball-and-stick representation. Arrows in panel a show the direction of the large conformational change the enzyme undergoes upon guanylylation. Reproduced from ref. 133 with permission.

analyses of microbial genome databases show that the *cobY* gene is unique to archaea, since none of the bacterial genome sequences reported to date contains a *cobY* orthologue.

8.6 The kinase and guanylyltransferase activities of CobU play different physiological roles

The existence of CobY helps framing the activities of CobU

a)

-GMP

α-helix 2

Arg 9

Pα

15.8 Å

His 46

Ser 10

6.0 Å

Lys 12

b)

+GMP

α-helix 2

His 46

Arg 9

5.4 Å

6.0 Å

GMP

Ser 10

Lys 12

Fig. 17 Ribbon representation of the helix shift associated with binding GMP. Panel a shows the spatial relationship between the P-loop (in yellow) and the guanylylation site. Panel b shows the same region after the enzyme has undergone guanylylation. The position of the guanosine P atom was generated by superposition of the guanylylated structure with the apo structure. Reproduced from ref. 133 with permission.

within a physiological context. Previous to the identification of *cobY* it was suggested that the product of *de novo* corrin ring biosynthesis in *S. enterica* was AdoCbi-P, not AdoCbi as previously thought. The most important implication from these studies was that the kinase activity of CobU was not required for *de novo* corrin ring biosynthesis. Strong support for this hypothesis was obtained from *in vivo* complementation studies of mutant strains of *S. enterica* carrying a deletion of the chromosomal copy of *cobU* and a plasmid-encoded *cobY* gene. Results from these studies clearly showed that under growth conditions where *de novo* corrin ring biosynthesis was essential for growth the above-mentioned strains grew, whilst under conditions where assimilation of exogenous Cbi was required for growth, CobY failed to compensate for the lack of CobU in the strains. The explanation for this observation is the following. Under anaerobic conditions, *S. enterica* synthesizes the corrin ring *de novo*, i.e., it makes AdoCbi-P, which can be converted to AdoCbi-GDP by CobY. Under aerobic conditions, where *de novo* corrin ring biosynthesis is blocked and the cell depends on exogenous Cbi to grow, phosphorylation of AdoCbi is required to generate the substrate for CobY. Since CobY lacks

406 Nat. Prod. Rep., 2002, 19, 390–412

Fig. 18 Stereo ribbon representation of the NaMN:DMB PRTase (CobT) dimer isolated from *S. enterica*. The large domains are shown in blue and pink and the small domains are shown in magenta and green. Reproduced from ref. 167 with permission.

the kinase activity of CobU, the tester strains could not make AdoCbi-GDP under aerobic conditions, thus failed to grow. From this work it is concluded that in *S. enterica* the guanylyl-transferase activity of CobU is required *de novo* corrin ring biosynthesis and the kinase activity of the enzyme is dedicated to salvaging exogenous Cbi.

Because CobU homologues are only found in bacteria, does it mean that only bacteria learned how to salvage Cbi from their environment? Or is it possible that archaea use a different mechanism to salvage Cbi?

### 9.0 Synthesis of the lower (Coa) ligand, 5,6-dimethylbenzimidazole

Studies of the synthesis of the lower ligand base 5,6-dimethylbenzimidazole (DMB) have been performed in strict anaerobes such as *Eubacterium limosum* and in aerotolerant bacteria such as *P. freudenreichii.*¹²² These prokaryotes take two completely different approaches to the synthesis of DMB, the chief difference being that in *E. limosum* DMB is put together from glycine, formate, SAM, glutamine, and erythrose-4-phosphate,¹⁴⁷⁻¹⁵¹ whereas in *P. freudenreichii* DMB is derived from FMN *via* an oxygen-dependent pathway.¹⁵²⁻¹⁵⁶ *S. enterica* appears to be able to synthesize DMB aerobically from FMN;¹⁵⁷ however, under anaerobic growth conditions this bacterium synthesizes adenyl and 2-methyladenylcobamide instead of 5,6-dimethylbenzimidazolylcobamide (aka Cbl).¹⁵⁸ Two reviews on this topic have been recently published.¹⁵⁹,¹⁶⁰

#### 9.1 Activation of the lower ligand base

Even though the lower ligand base of cobamides varies considerably,¹⁶⁰ *in vitro* studies of lower ligand base activation have only been reported when DMB was used as substrate.²⁷, ¹⁶¹⁻¹⁶⁵ Thus, what has been learned about lower ligand activation applies only to one ligand, *i.e.*, DMB. The activation of DMB occurs in two steps catalyzed by a phosphoribosyltransferase (PRTase) and a phosphatase enzyme.

#### 9.2 General features of the PRTase enzyme

The nicotinate mononucleotide (NaMN):DMB phosphoribosyltransferase (PRTase) enzyme catalyzes the transfer of the phosphoribosyl moiety of NaMN onto DMB to yield N¹-(5-phospho-α-D-ribosyl)-DMB nucleotide (aka α-ribazole-5-P). The PRTase enzyme has been partially purified from

Propionibacterium freudenreichii,¹⁶¹ Clostridium sticklandii,¹⁶³ and Propionibacterium arabinosum,¹⁶⁶ and to homogeneity from *P. denitrificans* (encoded by the *cobU* gene)²⁷ and *Salmonella enterica* (encoded by the *cobT* gene).¹⁶⁵ NaMN:DMB PRTase enzymes isolated from different sources appear to have different affinities for NaMN. For example, the affinity of the *P. denitrificans* CobU enzyme for NaMN was higher (*K*ₘ = 83 μM)²⁷ than the *S. enterica* CobT enzyme (*K*ₘ = 684 μM),¹⁶⁵ a significant difference that could be attributed to structural differences between the two enzymes. Support for this explanation is suggested by the results of computer analyses, which indicate that the *P. denitrificans* CobU and *S. enterica* CobT enzymes are only 30% identical at the amino acid sequence level.¹⁶⁴ The affinity for DMB has only been reported for the *P. denitrificans* CobU enzyme, and is considerably higher than the affinity for NaMN (*K*ₘ = 16 nM),²⁷ probably reflecting the low intracellular level of DMB in this bacterium.

#### 9.3 Three-dimensional structure of the *S. enterica* NaMN:DMB PRTase enzyme

The X-ray structure of the CobT enzyme from *S. enterica* complexed with DMB was solved at 1.9 Å resolution.¹⁶⁷ The quaternary structure of native CobT enzyme is a dimer with 2-fold symmetry, and each subunit has two domains (Fig. 18). A substantial section of the extensive interface between the two subunits is contributed by the small domain and by the final helix of the large domain. The large domains consist of six-stranded β-sheets with connecting α-helices exhibiting Rossmann fold topology. The small domain is comprised of components of the N- and C-termini of the polypeptide chain and contains a three-helix bundle. The active site of the enzyme is formed by the loops at the C-terminal end of the β-strands and the small domain of the small subunit. The structure of the CobT enzyme suggests that residue Glu³¹⁷ is the catalytic base for the reaction. The proposed mechanism for how the CobT reaction may proceed is schematized in Fig. 19. However, experimental support for this hypothesis has not been reported. Transfer of the phosphoribosyl moiety of NaMN onto DMB was accomplished in the lattice of crystals of the enzyme complexed with DMB (Fig. 18). Both products of the reaction, *i.e.*, nicotinate and α-ribazole-5′-P, were unequivocally resolved; the resolution of this complex was also at 1.9 Å (Fig. 20). The fold of the CobT enzyme is very different from that of type I or type II

NaMN

![Chemical Structure](chemical_structure.png)

Fig. 19 Proposed mechanism for the synthesis of α-ribazole-5′-P synthesis by the CobT PRTase. Residue Glu<sup>317</sup> is proposed to extract a proton from DMB leading to generate the nucleophile that displaces nicotinate from nicotinate mononucleotide (NaMN).

![Stereo View](stereo_view.png)

Fig. 20 Stereo view of the difference electron density for α-ribazole-5′-phosphate and nicotinic acid in NaMN:DMB PRTase (CobT)-products complexes. Reproduced from ref. 167 with permission.

phosphoribosylpyrophosphate (PRPP)-dependent PRTases, and the orientation of substrates and products is opposite to that expected for a Rossman fold.<sup>167</sup> In fact, the structural fold of CobT is not related to any other fold found in protein databases.

### 9.4 Specificity of the NaMN:DMB PRTase enzyme for its base substrate

The chemical nature of the base in the nucleotide of cobamides varies depending on their source. The list bases includes DMB, 5-methylbenzimidazole, 5-methoxybenzimidazole, 5-hydroxybenzimidazole, 5-methoxy-6-methylbenzimidazole, adenine, 2-methylsulfinyladenine, 2-methylsulfonlyadenine, *p*-cresol, and phenol.<sup>168–175</sup> Although the *S*. enterica CobT enzyme displays high specificity for NaMN, its specificity for the base substrate is very poor. This conclusion is based on results from *in vitro* studies performed with homogeneous *S*. enterica CobT enzyme, NaMN and benzimidazole, dimethylphenylenediamine, imidazole, histidine, adenine or guanine as substrates.<sup>165</sup> In all cases, the enzyme catalyzed the formation of the corresponding α-nucleotide, and in all cases the product of the reaction was incorporated into cobamides that were active *in vivo*.<sup>165</sup>

To understand the molecular basis for the lack of specificity of the CobT enzyme of *S*. enterica for its base substrates, X-ray structures of the enzyme complexed with adenine, 5-methylbenzimidazole, 2-hydroxypurine, *p*-cresol or phenol were solved.<sup>176</sup> Adenine, 5-methylbenz-

imidazole, 5-methoxybenzimidazole, 2-hydroxypurine reacted with NaMN in the crystal lattice to form the corresponding α-nucleotide. The crystal structures of these complexes revealed that only minor conformational changes in the side chains that form the DMB binding site are needed to accommodate different base substrates. No product was formed in the crystal containing *p*-cresol or phenol even though *p*-cresol and phenol bound to CobT in approximately the same location as DMB. Analysis of the crystal structures showed that in both cases the active site of CobT was too large for the reaction to occur, suggesting that the CobT homologue in *Sporomusa ovata* (a strict anaerobic bacterium that synthesizes phenyl-cobamides) must have evolved an active site that brings the two substrates closer together than the *S*. *enterica* enzyme. From the mechanistic standpoint, it is of interest to learn how the *S*. *ovata* activates the substrate in the absence of an aromatic nitrogenous base.

### 9.5 The phosphatase reaction

Fig. 17 shows the second step in the activation of DMB being catalyzed by a phosphatase enzyme. In this reaction, α-ribazole-5′-P (the product of the NaMN:DMB PRTase enzyme) is converted to α-ribazole. This enzymatic activity was reported present in cell-free extracts of *P*. *freudenreichii*<sup>162</sup> and *P*. *denitrificans*,<sup>27</sup> but neither the enzyme nor the gene encoding it were identified in these bacteria. In *S*. *enterica*, the α-ribazole-5′-P phosphatase activity is encoded by the *cobC* gene, and com-
puter analysis of the predicted primary amino acid sequence of  
the CobC protein shows striking homology to phosphoglycer-  
ate mutases, acid phosphatases and to the biphosphatase  
domain of eukaryotic 6-phosphofructo-2-kinase/fructose-2,6-  
biphosphatase enzymes.128 The characteristic Arg-His-Gly  
motif found in these enzymes is conserved in CobC as is the  
second active-site histidine and other residues involved in sub-  
strate binding.177 Several studies of the mechanism of action of  
the biphosphatase activity of the rat liver 6-phosphofructo-2-  
kinase/fructose-2,6-biphosphatase enzyme showed the reaction  
proceeding via a transient phosphoenzyme intermediate involv-  
ing a phosphoramidate bond with a conserved histidine.178–181  
Mechanistic studies of the S. enterica CobC enzyme have not  
been reported. In S. enterica, the cobC gene is divergently tran-  
scribed from the cobD gene, an AdoCbl biosynthetic gene  
encoding L-threonine O-3-phosphate decarboxylase.86 Com-  
puter analysis of the cobC and cobD sequences predicts the  
regulatory regions of these genes to be embedded in the neigh-  
bor’s coding sequence with only one base pair separating the  
cobD and cobC coding sequences.86 In E. coli K12, the cobC  
homologue is referred to as phpB, however, no information is  
available regarding the involvement of the putative PhpB  
protein in AdoCbl biosynthesis in this bacterium. It should  
be noted that in E. coli, the cobD gene is missing, thus this  
bacterium is unable to convert cobyric acid into cobalamin.  

### 9.6 Phenotypes of a S. enterica cobC mutant strain  

In S. enterica, the effect of the lack of CobC enzyme on  
AdoCbl synthesis is most evident when the demand for AdoCbl  
is high, e.g., during growth on ethanolamine as carbon and  
energy source.128 However, when the demand for AdoCbl is low  
(e.g., synthesis of methionine via the Cbl-dependent methionine  
synthase), there is no detectable effect on Cbl-dependent  
growth unless the strain also harbors a null allele of the  
cobT gene encoding the NaMN:DMB phosphoribosyl-  
transferase enzyme. Explanations for these phenotypes are  
not obvious. It is possible that under conditions where low  
levels of AdoCbl are sufficient to support growth, the activity  
of a non-specific phosphatase generates α-ribazole bypassing  
the need for CobC activity. Alternatively, under these condi-  
tions the Cbl-dependent enzymes in this bacterium may be able  
to use Cbl-5-P as coenzyme.128 An intriguing, and as yet  
unexplained, phenotype of cobC mutant strains is the DMB  
auxotrophy observed when the strains are grown under high  
aeration.128 How the lack of CobC function affects DMB syn-  
thesis when the level of environmental oxygen is elevated  
remains unclear. A physiological explanation for the observed  
DMB auxotrophy of cobC mutant strains will likely shed valu-  
able insights into the pathway of DMB synthesis in S. enterica.  

### 9.7 Timing of phosphate removal by CobC  

In S. enterica, it is clear that the 5′-phosphate group of  
α-ribazole-5′-P is absent in AdoCbl, the final product of the  
pathway175 (Fig. 1). It is unclear, however, whether α-ribazole-  
5′-P or AdoCbl-5-P is the substrate for the CobC phosphatase  
enzyme. In vitro evidence that CobC can use either α-ribazole-  
5′-P or AdoCbl-5-P as substrate has been reported.128,182  
Insights into the timing of phosphate removal may be obtained  
through the kinetic analysis of the CobC phosphatase  
reaction when α-ribazole-5′-P or AdoCbl-5-P is the substrate  
for the enzyme. In P. denitrificans, it was concluded that the  
5′-phosphate group is removed from α-ribazole-5-P. This  
conclusion was reached exclusively on the basis of the high  
intracellular level of α-ribazole over α-ribazole-5′-P.31  

### 9.8 The last step of the NLA pathway: joining of AdoCbi-GDP  
and α-ribazole  

The enzyme that catalyzes the last step of the NLA pathway,  
and for that matter the last step of AdoCbl biosynthesis,  
is cobalamin (5′-phosphate) synthase. In S. enterica this  
enzyme is encoded by the cobS gene, and in P. denitrificans it is  
encoded by the cobV gene.31,42 The cobalamin (5′-phosphate)  
synthase enzyme has not been isolated to homogeneity, and in  
P. denitrificans, it appears to form complexes with a number of  
unidentified proteins.31 Computer analyses of predicted CobS  
orthologues primary amino acid sequences strongly suggest  
that this enzyme is likely to be a membrane protein.183 Purified  
preparations of CobS enzyme were obtained to reconstitute  
the entire NLA pathway in vitro.182 The in vitro data showed  
that the S. enterica CobS enzyme, like the P. denitrificans  
CobV enzyme, can use either α-ribazole or α-ribazole-5′-P as  
substrate, bringing into question the timing of the dephosphor-  
ylation step. It is clear that the CobC enzyme can dephosphor-  
ylate either α-ribazole-5′-P or AdoCbl-P,128,182 and kinetic  
analysis of the CobS reaction with each one of these substrates  
may provide further insights into which route is likely to be the  
one occurring in vivo.  

## 10 Summary  

We have reviewed the biochemical steps required for the trans-  
formation of uroporphyrinogen III into AdoCbl, one of the  
most mesmerizing and at times bewildering pathways operated  
in nature. It is outside the scope of this paper to cover aspects  
of control and regulation or to delve too deeply into how this  
intricate network of enzymes may have arisen. Similarly, we  
have not dealt with the biochemistry of the end product, the  
role that vitamin B₁₂ plays in biological systems, the beguiling  
chemical transformations it is able to mediate. These aspects,  
harnessed with the many unanswered questions concerning its  
biosynthesis, can be addressed in future reviews on Nature's  
most interesting and charismatic vitamin.  

## 11 Acknowledgements  

Work in the M.J.W. laboratory is supported by grants from  
the BBSRC and The Wellcome Trust, whilst research in the  
J.C.E.-S. laboratory is supported by grants GM40313 and  
GM62203 from the General Medical Sciences Institute of the  
National Institutes of Health.  

## 12 References  

1 G. R. Minot and W. P. Murphy, JAMA, J. Am. Med. Assoc., 1926,  
87, 470.  
2 E. L. Rickes, N. G. Brink, F. R. Koniuszy, T. R. Wood and  
K. Folkers, Science, 1948, 107, 396.  
3 E. L. Smith, Nature, 1948, 161, 638.  
4 D. C. Hodgkin, J. Pickworth, J. H. Robertson, K. N. Trueblood,  
R. J. Prosen and J. White, Nature, 1955.  
5 R. Banerjee, Chemistry and Biochemistry of B12, John Wiley and  
Sons, New York, 1999.  
6 A. R. Battersby, Nat. Prod. Rep., 2000, 17, 507.  
7 A. R. Battersby, Science, 1994, 264, 1551.  
8 F. J. Leeper, Nat. Prod. Rep., 1985, 2, 19.  
9 F. J. Leeper, Nat. Prod. Rep., 1985, 2, 561.  
10 F. J. Leeper, Nat. Prod. Rep., 1987, 441.  
11 H. C. Uzar, A. R. Battersby, T. A. Carpenter and F. J. Leeper,  
Chem. Soc. Perkin. Trans. I, 1987, 1689.  
12 A. I. Scott, H. J. Williams, N. J. Stolowich, P. Karuso, M. D.  
Gonzalez, G. Müller, K. Hlineny, E. Savvidis, E. Schneider,  
U. Traub-Eberhard and G. Wirth, J. Am. Chem. Soc., 1989, 111,  
1897.  
13 A. R. Battersby, K. Frobel, F. Hemmerschimdt and C. Jones,  
J. Chem. Soc., Chem. Commun., 1982, 1393.  
14 A. R. Battersby, M. J. Bushell, C. Jones, N. G. Lewis and A.  
Pfenninger, Proc. Natl. Acad. Sci. U. S. A., 1981, 78, 13.  
15 C. Nussbaumer, M. Imfeld, G. Worner, G. Muller and D. Arigoni,  
Proc. Natl. Acad. Sci. U. S. A., 1981, 78, 9.  
16 L. Mombelli, C. Nussbaumer, H. Weber, G. Muller and D. Arigoni,  
Proc. Natl. Acad. Sci. U. S. A., 1981, 78, 11.  
17 B. Cameron, K. Briggs, S. Pridmore, G. Brefort and J. Crouzet,  
J. Bacteriol., 1989, 171, 547.  
18 F. Blanche, L. Debussche, D. Thibaut, J. Crouzet and B. Cameron, *J. Bacteriol.*, 1989, **171**, 4222.

19 F. Blanche, D. Thibaut, M. Couder and J. C. Muller, *Anal. Biochem.*, 1990, **189**, 24.

20 F. Blanche, L. Debussche, A. Famechon, D. Thibaut, B. Cameron and J. Crouzet, *J. Bacteriol.*, 1991, **173**, 6052.

21 F. Blanche, M. Couder, L. Debussche, D. Thibaut, B. Cameron and J. Crouzet, *J. Bacteriol.*, 1991, **173**, 6046.

22 F. Blanche, C. Robin, M. Couder, D. Faucher, L. Cauchois, B. Cameron and J. Crouzet, *J. Bacteriol.*, 1991, **173**, 4637.

23 F. Blanche, L. Maton, L. Debussche and D. Thibaut, *J. Bacteriol.*, 1992, **174**, 7452.

24 F. Blanche, A. Famechon, D. Thibaut, L. Debussche, B. Cameron and J. Crouzet, *J. Bacteriol.*, 1992, **174**, 1050.

25 F. Blanche, D. Thibaut, A. Famechon, L. Debussche, B. Cameron and J. Crouzet, *J. Bacteriol.*, 1992, **174**, 1036.

26 F. Blanche, B. Cameron, J. Crouzet, L. Debussche, D. Thibaut, M. Vuilhorgne, F. J. Leeper and A. R. Battersby, *Angew. Chem., Int. Ed. Engl.*, 1995, **34**, 383.

27 B. Cameron, F. Blanche, M.-C. Rouyez, D. Bisch, A. Famechon, M. Couder, L. Cauchois, D. Thibaut, L. Debussche, B. Cameron and J. Crouzet, *J. Bacteriol.*, 1991, **173**, 6066.

28 B. Cameron, C. Guilhot, F. Blanche, L. Cauchois, M. C. Rouyez, S. Rigault, S. Levy-Schil and J. Crouzet, *J. Bacteriol.*, 1991, **173**, 6058.

29 J. Crouzet, B. Cameron, L. Cauchois, S. Rigault, M. C. Rouyez, F. Blanche, D. Thibaut and L. Debussche, *J. Bacteriol.*, 1990, **172**, 5980.

30 J. Crouzet, L. Cauchois, F. Blanche, L. Debussche, D. Thibaut, M. C. Rouyez, S. Rigault, J. F. Mayaux and B. Cameron, *J. Bacteriol.*, 1990, **172**, 5968.

31 J. Crouzet, S. Levy-Schil, B. Cameron, L. Cauchois, S. Rigault, M.-C. Rouyez, F. Blanche, L. Debussche and D. Thibaut, *J. Bacteriol.*, 1991, **173**, 6074.

32 L. Debussche, D. Thibaut, B. Cameron, J. Crouzet and F. Blanche, *J. Bacteriol.*, 1990, **172**, 6239.

33 L. Debussche, M. Couder, D. Thibaut, B. Cameron, J. Crouzet and F. Blanche, *J. Bacteriol.*, 1991, **173**, 6300.

34 L. Debussche, M. Couder, D. Thibaut, B. Cameron, J. Crouzet and F. Blanche, *J. Bacteriol.*, 1992, **174**, 7445.

35 L. Debussche, D. Thibaut, B. Cameron, J. Crouzet and F. Blanche, *J. Bacteriol.*, 1993, **175**, 7430.

36 L. Debussche, D. Thibaut, M. Danzer, F. Debu, D. Frechet, F. Herman, F. Blanche and M. Vuilhorgne, *J. Chem. Soc., Chem. Commun.*, 1993, 1100.

37 D. Thibaut, F. Blanche, L. Debussche, F. J. Leeper and A. R. Battersby, *Proc. Natl. Acad. Sci. U. S. A.*, 1990, **87**, 8800.

38 D. Thibaut, M. Couder, J. Crouzet, L. Debussche, B. Cameron and F. Blanche, *J. Bacteriol.*, 1990, **172**, 6245.

39 D. Thibaut, L. Debussche and F. Blanche, *Proc. Natl. Acad. Sci. U. S. A.*, 1990, **87**, 8795.

40 D. Thibaut, M. Couder, A. Famechon, L. Debussche, B. Cameron, J. Crouzet and F. Blanche, *J. Bacteriol.*, 1992, **174**, 1043.

41 D. Thibaut, L. Debussche, D. Frechet, F. Herman, M. Vuilhorgne and F. Blanche, *J. Chem. Soc., Chem. Commun.*, 1993, 513.

42 J. R. Roth, J. G. Lawrence, M. Rubenfield, S. Kieffer-Higgins and G. M. Church, *J. Bacteriol.*, 1993, **175**, 3303.

43 E. Raux, A. Lanois, M. J. Warren, A. Rambach and C. Thermes, *Biochem. J.*, 1998, **335**, 159.

44 E. Raux, A. Lanois, A. Rambach, M. J. Warren and C. Thermes, *Biochem. J.*, 1998, **335**, 167.

45 F. Blanche, D. Thibaut, L. Debussche, R. Hertle, F. Zipfel and G. Müller, *Angew. Chem., Int. Ed. Engl.*, 1993, **32**, 1651.

46 E. Raux, A. Lanois, F. Levillayer, M. J. Warren, E. Brody, A. Rambach and C. Thermes, *J. Bacteriol.*, 1996, **178**, 753.

47 E. Raux, H. L. Schubert and M. J. Warren, *Cell. Mol. Life Sci.*, 2000, **57**, 1880.

48 C. Robin, F. Blanche, L. Cauchois, B. Cameron, M. Couder and J. Crouzet, *J. Bacteriol.*, 1991, **173**, 4893.

49 T. Peakman, J. Crouzet, J. F. Mayaux, S. Busby, S. Mohan, N. Harborne, J. Wootton, R. Nicolson and J. Cole, *Eur. J. Biochem.*, 1990, **191**, 315.

50 M. J. Warren, C. A. Roessner, P. J. Santander and A. I. Scott, *Biochem. J.*, 1990, **265**, 725.

51 J. B. Spencer, N. J. Stolowich, C. A. Roessner and A. I. Scott, *FEBS Lett.*, 1993, **335**, 57.

52 M. J. Warren, E. L. Bolt, C. A. Roessner, A. I. Scott, J. B. Spencer and S. C. Woodcock, *Biochem. J.*, 1994, **302**, 837.

53 S. C. Woodcock, E. Raux, F. Levillayer, C. Thermes, A. Rambach and M. J. Warren, *Biochem. J.*, 1998, **330**, 121.

54 M. J. Warren, N. J. Stolowich, P. J. Santander, C. A. Roessner, B. A. Sowa and A. I. Scott, *FEBS Lett.*, 1990, **261**, 76.

55 R. M. Jeter, B. M. Olivera and J. R. Roth, *J. Bacteriol.*, 1984, **159**, 206.

56 B. S. Goldman and J. R. Roth, *J. Bacteriol.*, 1993, **175**, 1457.

57 H. L. Schubert, E. Raux, A. A. Brindley, H. K. Leech, K. S. Wilson, C. P. Hill and M. J. Warren, *EMBO J.*, 2002, **21**, 1.

58 C. A. Roessner, P. J. Santander and A. I. Scott, *Vitam. Horm. (San Diego)*, 2001, **61**, 267.

59 E. Raux, H. L. Schubert, J. M. Roper, S. W. Wilson and M. J. Warren, *Bioorg. Chem.*, 1999, **27**, 100.

60 M. J. Warren, C. A. Roessner, S. Ozaki, N. J. Stolowich, P. J. Santander and A. I. Scott, *Biochemistry*, 1992, **31**, 603.

61 A. I. Scott, C. A. Roessner, N. J. Stolowich, J. B. Spencer, C. Min and S. I. Ozaki, *FEBS Lett.*, 1993, **331**, 105.

62 C. Min, B. A. Atshaves, C. A. Roessner, N. J. Stolowich, J. B. Spencer and A. I. Scott, *J. Am. Chem. Soc.*, 1993, **115**, 10380.

63 C. A. Roessner, J. B. Spencer, N. J. Stolowich, J. Wang, G. P. Nayar, P. J. Santander, C. Pichon, C. Min, M. T. Holderman and A. I. Scott, *Chem. Biol.*, 1994, **1**, 119.

64 F. Kiuchi, D. Thibaut, L. Debussche, F. J. Leeper, F. Blanche and A. R. Battersby, *J. Chem. Soc., Chem. Commun.*, 1992, 306.

65 G. W. Weaver, F. J. Leeper, A. R. Battersby, F. Blanche, D. Thibaut and L. Debussche, *J. Chem. Soc., Chem. Commun.*, 1991, 976.

66 L. W. Shipman, D. Li, C. A. Roessner, A. I. Scott and J. C. Sacchettini, *Structure (London)*, 2001, **9**, 587.

67 C. J. Walker and R. D. Willows, *Biochem. J.*, 1997, **327**, 321.

68 G. Muller, F. Zipfel, K. Hlineny, H. Zipfel, A. I. Scott, H. J. Williams, N. J. Stolowich, P. J. Santander, M. J. Warren, F. Blanche and D. Thibaut, *J. Am. Chem. Soc.*, 1991, **113**, 9891.

69 S. Balachandran, R. A. Vishwakarma, S. M. Monaghan, A. Pelle, P. J. Stamford, F. J. Leeper and A. R. Battersby, *J. Chem. Soc., Perkin Trans. 1*, 1994, 487.

70 K. Kurumaya, T. Okazaki and M. Kajiwara, *Chem. Pharm. Bull.*, 1989, **37**, 1151.

71 A. I. Scott, N. J. Stolowich, B. P. Atshaves, P. Karuso, M. J. Warren, H. J. Williams, M. Kajiwara, K. Katsuyuki and T. Okazaki, *J. Am. Chem. Soc.*, 1991, **113**, 9891.

72 R. A. Vishwakarma, S. Balachandran, A. I. D. Alanine, P. J. Stamford, F. Kiuchi, F. J. Leeper and A. R. Battersby, *J. Chem. Soc., Perkin Trans. 1*, 1993, 2893.

73 T. G. Fazzio and J. R. Roth, *J. Bacteriol.*, 1996, **178**, 6952.

74 E. Raux, C. Thermes, P. Heathcote, A. Rambach and M. J. Warren, *J. Bacteriol.*, 1997, **179**, 3202.

75 H. L. Schubert, E. Raux, K. S. Wilson and M. J. Warren, *Biochemistry*, 1999, **38**, 10660.

76 E. Raux, T. McVeigh, S. E. Peters, T. Leustek and M. J. Warren, *Biochem. J.*, 1999, **338**, 701.

77 P. Spencer, N. J. Stolowich, L. W. Sumner and A. I. Scott, *Biochemistry*, 1998, **37**, 14917.

78 A. I. Scott, N. J. Stolowich, J. Wang, O. Gawatz, E. Fridich and G. Müller, *Proc. Natl. Acad. Sci. U. S. A.*, 1996, **93**, 14316.

79 J. Wang, N. J. Stolowich, P. J. Santander, J. H. Park and A. I. Scott, *Proc. Natl. Acad. Sci. U. S. A.*, 1996, **93**, 14320.

80 P. J. Santander, C. A. Roessner, N. J. Stolowich, M. T. Holderman and A. I. Scott, *Chem. Biol.*, 1997, **4**, 659.

81 P. J. Santander, N. J. Stolowich and A. I. Scott, *Bioorg. Med. Chem.*, 1999, **7**, 789.

82 C. A. Roessner, M. J. Warren, P. J. Santander, B. P. Atshaves, S. Ozaki, N. J. Stolowich, K. Ida and A. I. Scott, *FEBS Lett.*, 1992, **301**, 73.

83 E. Raux, H. L. Schubert, S. C. Woodcock, K. S. Wilson and M. J. Warren, *Eur. J. Biochem.*, 1998, **254**, 341.

84 H. L. Schubert, K. S. Wilson, E. Raux, S. C. Woodcock and M. J. Warren, *Nat. Struct. Biol.*, 1998, **5**, 585.

85 J. M. Roper, E. Raux, A. A. Brindley, H. L. Schubert, S. E. Gharbia, H. N. Shah and M. J. Warren, *J. Biol. Chem.*, 2000, **275**, 40316.

86 K. R. Brushaber, G. A. O’Toole and J. C. Escalante-Semerena, *J. Biol. Chem.*, 1998, **273**, 2684.

87 C. G. Cheong, C. B. Bauer, K. R. Brushaber, J. C. Escalante-Semerena and I. Rayment, *Biochemistry*, 2002, **41**, 4798.

88 R. O. Brady and H. A. Barker, *Biochem. Biophys. Res. Commun.*, 1961, **4**, 373.

89 H. Weissbach, B. Redfield and A. Peterkofsky, *J. Biol. Chem.*, 1961, **236**, PC40.

90 J. C. Escalante-Semerena, S.-J. Suh and J. R. Roth, *J. Bacteriol.*, 1990, **172**, 273.

91 PhD thesis: S.-J. Suh, *The role of CobA in adenosylation of corrinoids in Salmonella typhimurium*, University of Wisconsin-Madison, 1994.

92 M. D. Lundrigan, L. C. DeVeaux, B. J. Mann and R. J. Kadner, *Mol. Gen. Genet.*, 1987, **206**, 401.

93 C. V. Franklund and R. J. Kadner, *J. Bacteriol.*, 1997, **179**, 4039.

94 X. Nou and R. J. Kadner, *Proc. Natl. Acad. Sci. U. S. A.*, 2000, **97**, 7190.

95 G. A. Walker, S. Murphy and F. M. Huennekens, *Arch. Biochem. Biophys.*, 1969, **134**, 95.

96 K. Fujii and F. M. Huennekens, *J. Biol. Chem.*, 1974, **249**, 6745.

97 K. Fujii, J. H. Galivan and F. M. Huennekens, *Arch. Biochem. Biophys.*, 1977, **178**, 662.

98 R. O. Brady, E. G. Castanera and H. A. Barker, *J. Biol. Chem.*, 1962, **223**, 2325.

99 E. Vitols, G. A. Walker and F. M. Huennekens, *J. Biol. Chem.*, 1966, **241**, 1455.

100 M. V. Fonseca and J. C. Escalante-Semerena, *J. Bacteriol.*, 2000, **182**, 4304.

101 M. V. Fonseca and J. C. Escalante-Semerena, *J. Biol. Chem.*, 2001, **276**, 32101.

102 S. J. Suh and J. C. Escalante-Semerena, *Gene*, 1993, **129**, 93.

103 S.-J. Suh and J. C. Escalante-Semerena, *J. Bacteriol.*, 1995, **177**, 921.

104 D. Lexa and J.-M. Saveant, *Acc. Chem. Res.*, 1983, **16**, 235.

105 D. M. Hoover, J. T. Jarrett, R. H. Sands, W. R. Dunham, M. L. Ludwig and R. G. Matthews, *Biochemistry*, 1997, **36**, 127.

106 D. M. Hoover and M. L. Ludwig, *Protein Sci.*, 1997, **6**, 2525.

107 J. T. Jarrett, D. M. Hoover, M. L. Ludwig and R. G. Matthews, *Biochemistry*, 1998, **37**, 12649.

108 D. A. Hall, T. C. Jordan-Starck, R. O. Loo, M. L. Ludwig and R. G. Matthews, *Biochemistry*, 2000, **39**, 10711.

109 D. A. Hall, C. W. Vander Kooi, C. N. Stasik, S. Y. Stevens, E. R. Zuiderweg and R. G. Matthews, *Proc. Natl. Acad. Sci. U. S. A.*, 2001, **98**, 9521.

110 H. Olteanu and R. Banerjee, *J. Biol. Chem.*, 2001, **276**, 35558.

111 A. Peterkofsky, B. Redfied and H. Weissbach, *Biochem. Biophys. Res. Commun.*, 1961, **5**, 213.

112 A. Peterkofsky and H. Weissbach, *J. Biol. Chem.*, 1963, **238**, 1491.

113 C. B. Bauer, M. V. Fonseca, H. M. Holden, J. B. Thoden, T. B. Thompson, J. C. Escalante-Semerena and I. Rayment, *Biochemistry*, 2001, **40**, 361.

114 C. L. Drennan, S. Huang, J. T. Drummond, R. G. Matthews and M. L. Ludwig, *Science*, 1994, **266**, 1660.

115 F. Mancia, N. H. Keep, A. Nakagawa, P. F. Leadlay, S. McSweeney, B. Rasmussen, P. Bsecke, O. Diat and P. R. Evans, *Structure (London)*, 1996, **4**, 339.

116 F. Mancia, G. A. Smith and P. R. Evans, *Biochemistry*, 1999, **38**, 7999.

117 N. Shibata, J. Masuda, T. Tobimatsu, T. Toraya, K. Suto, Y. Morimoto and N. Yasuoka, *Structure with Folding with Design*, 1999, **7**, 997.

118 B. Hoffmann, R. Konrat, H. Bothe, W. Buckel and B. Kräutler, *Eur. J. Biochem.*, 1999, **263**, 178.

119 R. Reitzer, K. Gruber, G. Jogl, U. G. Wagner, H. Bothe, W. Buckel and C. Kratky, *Structure with Folding with Design*, 1999, **7**, 891.

120 E. Kofoid, C. Rappleye, I. Stojiljkovic and J. Roth, *J. Bacteriol.*, 1999, **181**, 5317.

121 D. Walter, M. Ailion and J. Roth, *J. Bacteriol.*, 1997, **179**, 1013.

122 C. L. Johnson, E. Pechonick, S. D. Park, G. D. Havemann, N. A. Leal and T. A. Bobik, *J. Bacteriol.*, 2001, **183**, 1577.

123 L. D. Eirich, G. D. Vogels and R. S. Wolfe, *Biochemistry*, 1978, **17**, 4583.

124 J. T. Keltjens, L. Daniels, H. G. Jannsen, P. J. Borm and G. D. Vogels, *Eur. J. Biochem.*, 1983, **130**, 545.

125 A. Eschenmoser and E. Loewenthal, *Chem. Soc. Rev.*, 1992, **1992**, 1.

126 R. M. Jeter, *J. Gen. Microbiol.*, 1990, **136**, 887.

127 G. A. O’Toole, M. R. Rondon and J. C. Escalante-Semerena, *J. Bacteriol.*, 1993, **175**, 3317.

128 G. A. O’Toole, J. R. Trzebiatowski and J. C. Escalante-Semerena, *J. Biol. Chem.*, 1994, **269**, 26503.

129 K. Bernhauer, F. Wagner, H. Michna, P. Rapp and H. Vogelmann, *Hoppe-Seyler’s Z. Physiol. Chem.*, 1968, **349**, 1297.

130 R. A. Ronzio and H. A. Barker, *Biochemistry*, 1967, **6**, 2344.

131 M. G. Thomas, T. B. Thompson, I. Rayment and J. C. Escalante-Semerena, *J. Biol. Chem.*, 2000, **275**, 27376.

132 T. B. Thompson, M. G. Thomas, J. C. Escalante-Semerena and I. Rayment, *Biochemistry*, 1998, **37**, 7686.

133 T. Thompson, M. G. Thomas, J. C. Escalante-Semerena and I. Rayment, *Biochemistry*, 1999, **38**, 12995.

134 G. A. O’Toole and J. C. Escalante-Semerena, *J. Biol. Chem.*, 1995, **270**, 23560.

135 D. Pal and P. Chakrabarti, *J. Mol. Biol.*, 1999, **294**, 271.

136 G. A. O’Toole and J. C. Escalante-Semerena, *J. Bacteriol.*, 1993, **175**, 6328.

137 PhD thesis: G. A. O’Toole, *Biochemistry and genetics of cobalamin nucleotide loop assembly in Salmonella typhimurium.*, University of Wisconsin-Madison, 1994.

138 S. A. Martin and B. Moss, *J. Biol. Chem.*, 1975, **250**, 9330.

139 S. Shuman and J. Hurwitz, *Proc. Natl. Acad. Sci. USA*, 1981, **78**, 187.

140 S. Shuman and B. Schwer, *Mol. Microbiol.*, 1995, **17**, 405.

141 K. Hakansson, A. J. Doherty, S. Shuman and D. B. Wigley, *Cell*, 1997, **89**, 545.

142 T. Szumilo, R. R. Drake, J. L. York and A. D. Elbein, *J. Biol. Chem.*, 1993, **268**, 17943.

143 J. L. Cartwright and A. G. McLennan, *Arch. Biochem. Biophys.*, 1999, **361**, 101.

144 J. J. Liu and A. G. McLennan, *J. Biol. Chem.*, 1994, **269**, 11787.

145 A. A. DiMarco, T. A. Bobik and R. S. Wolfe, *Annu. Rev. Biochem.*, 1990, **59**, 355.

146 M. G. Thomas and J. C. Escalante-Semerena, *J. Bacteriol.*, 2000, **182**, 4227.

147 L. Lamm, J. A. Hrig, P. Renz and G. Heckmann, *Eur. J. Biochem.*, 1980, **109**, 115.

148 V. Höllriegl, L. Lamm, J. Rowold, J. Hrig and P. Renz, *Arch. Microbiol.*, 1982, **132**, 155.

149 L. Lamm, G. Heckmann and P. Renz, *Eur. J. Biochem.*, 1982, **122**, 569.

150 J. R. A. Vogt, L. Lamm-Kolonko and P. Renz, *Eur. J. Biochem.*, 1988, **174**, 637.

151 M. Munder, J. R. A. Vogt, B. Volger and P. Renz, *Eur. J. Biochem.*, 1992, **204**, 679.

152 P. Renz, *FEBS Lett.*, 1970, **6**, 187.

153 P. Renz and R. Weyhenmeyer, *FEBS Lett.*, 1972, **22**, 124.

154 J. A. Hörig, P. Renz and G. Heckmann, *J. Biol. Chem.*, 1978, **253**, 7410.

155 B. Lingens, T. A. Schild, B. Volger and P. Renz, *Eur. J. Biochem.*, 1992, **207**, 981.

156 B. Kolonko, J. A. Hörig and P. Renz, *Z. Naturforsch., C: Biosci.*, 1992, **47**, 171.

157 B. Keck, M. Munder and P. Renz, *Arch. Microbiol.*, 1998, **171**, 66.

158 B. Keck and P. Renz, *Arch. Microbiol.*, 2000, **173**, 76.

159 P. Renz, Investigations in the biosynthesis of the 5,6-dimethylbenzimidazole moiety of vitamin B₁₂, in *Vitamin B₁₂ and B₁₂ Proteins*, ed. B. Kräutler, D. Arigoni and B. T. Golding, Wiley-VCH, Weinheim, 1998, p. 119.

160 P. Renz, Biosynthesis of the 5,6-dimethylbenzimidazole moiety of cobalamin and of other bases found in natural corrinoids, in *Chemistry and Biochemistry of B₁₂*, ed. R. Banerjee, John Wiley & Sons, Inc., New York, 1999, p. 557.

161 H. C. Friedmann, *J. Biol. Chem.*, 1965, **240**, 413.

162 H. C. Friedmann and D. L. Harris, *J. Biol. Chem.*, 1965, **240**, 406.

163 J. A. Fyfe and H. C. Friedmann, *J. Biol. Chem.*, 1969, **244**, 1659.

164 J. R. Trzebiatowski, G. A. O’Toole and J. C. Escalante-Semerena, *J. Bacteriol.*, 1994, **176**, 3568.

165 J. R. Trzebiatowski and J. C. Escalante-Semerena, *J. Biol. Chem.*, 1997, **272**, 17662.

166 Z. Schneider and A. Stroinski, The occurrence and distribution of corrinoids: microorganisms, in *Comprehensive B₁₂*, Walter de Gruyter, Berlin, 1987, p. 157.

167 C. G. Cheong, J. C. Escalante-Semerena and I. Rayment, *Biochemistry*, 1999, **38**, 16125.

168 E. Stupperich, H. J. Eisinger and B. Kräutler, *Eur. J. Biochem.*, 1988, **172**, 459.

169 E. Stupperich, H. J. Eisinger and B. Kräutler, *Eur. J. Biochem.*, 1989, **186**, 657.

170 E. Stupperich and B. Kräutler, *Arch. Microbiol.*, 1988, **149**, 213.

171 B. Kräutler, H. P. E. Kohler and E. Stupperich, *Eur. J. Biochem.*, 1988, **176**, 461.

172 P. Renz, S. Blickle and W. Friedrich, *Eur. J. Biochem.*, 1987, **163**, 175.

173 D. C. Hodgkin, J. Pickworth, J. H. Robertson, K. N. Trueblood, R. J. Prosen and J. G. White, *Nature*, 1956, **176**, 325.

174 D. C. Hodgkin, J. Kamper, J. Lindsey, M. MacKay, J. Pickworth, J. H. Robertson, C. B. Shoemaker, J. G. White, R. J. Prosen and K. N. Trueblood, *Proc. R. Soc.*, 1957, **A242**, 228.

175 M. G. Johnson and J. C. Escalante-Semerena, *J. Biol. Chem.*, 1992, **267**, 13302.

176 C. G. Cheong, J. C. Escalante-Semerena and I. Rayment, *J. Biol. Chem.*, 2001, **276**, 37612.

177 S. J. Pilakis, I. J. Claus and A. J. Lange, *Annu. Rev. Biochem.*, 1995, **64**, 799.

178 D. A. Okar, L. T. Kakalis, S. S. Narula, I. M. Armitage and S. J. Pilkis, *Biochem. J.*, 1995, **308**(Pt 1), 189.

179 D. A. Okar, D. H. Live, T. L. Kirby, E. J. Karschnia, L. B. von Weymarn, I. M. Armitage and A. J. Lange, *Biochemistry*, 1999, **38**, 4471.

180 D. A. Okar, D. H. Live, M. H. Devany and A. J. Lange, *Biochemistry*, 2000, **39**, 9754.

181 H. Mizuguchi, P. F. Cook, C. H. Tai, C. A. Hasemann and K. Uyeda, *J. Biol. Chem.*, 1999, **274**, 2166.

182 L. A. Maggio-Hall and J. C. Escalante-Semerena, *Proc. Natl. Acad. Sci. USA*, 1999, **96**, 11798.

183 PhD thesis: L. A. Maggio-Hall, *Synthesis and Incorporation of the Lower Ligand Base of Cobalamin.*, Department of Bacteriology, University of Wisconsin, Madison, 2001, p. 155.
