
The Plant Journal (2002) 31(5), 555–564

# Key amino acid residues required for aryl migration catalysed by the cytochrome P450 2-hydroxyisoflavanone synthase

Yuji Sawada¹, Kengo Kinoshita²,³,†, Tomoyoshi Akashi¹, Toshio Aoki¹ and Shin-ichi Ayabe¹*

¹Department of Applied Biological Sciences, Nihon University, Fujisawa, Kanagawa 252-8510,  
²Institute for Protein Research, Osaka University, Yamadaoka 3-2, Suita, Osaka, 565-0871, and  
³RIKEN Genomic Sciences Center, Yokohama, Kanagawa 230-0045, Japan

Received 21 November 2001; revised 16 April 2002; accepted 14 May 2002.  
*For correspondence (fax +81 466 80 1141; e-mail ayabe@brs.nihon-u.ac.jp).  
†Present address: Division of Structural Bioinformatics, Graduate School of Integrated Science, Yokohama City University, Yokohama, Kanagawa 230-0045, Japan.

## Summary

Isoflavonoids are distributed predominantly in leguminous plants, and play pivotal roles in the interaction of host plants with biological environments. Isoflavones in the diet also have beneficial effects on human health as phytoestrogens. The isoflavonoid skeleton is constructed by the CYP93C subfamily of cytochrome P450s in plant cells. The reaction consists of hydroxylation of the flavanone molecule at C-2 and an intramolecular 1,2-aryl migration from C-2 to C-3 to yield 2-hydroxyisoflavanone. In this study, with the aid of alignment of amino acid sequences of CYP93 family P450s and a computer-generated putative stereo structure of the protein, candidates for key amino acid residues in CYP93C2 responsible for the unique aryl migration in 2-hydroxyisoflavanone synthase reaction were identified. Microsomes of recombinant yeast cells expressing mutant proteins of CYP93C2 were prepared, and their catalytic activities tested. The reaction with the mutant in which Ser 310 in the centre of the α-helix was converted to Thr yielded increased formation of 3-hydroxyflavanone, a by-product of the 2-hydroxyisoflavanone synthase reaction, in addition to the major isoflavonoid product. More dramatically, the mutant in which Lys 375 in the end of β-sheet 1–4 was replaced with Thr produced only 3-hydroxyflavanone and did not yield the isoflavonoid any longer. The roles of these amino acid residues in the catalysis and evolution of isoflavonoid biosynthesis are discussed.

Keywords: CYP93C subfamily, evolution, homology modelling, intramolecular aryl migration, isoflavonoid, site-directed mutagenesis.

## Introduction

Flavonoids are very widely distributed in vascular plants as anthocyanin pigments and as other polyphenols with physiological roles essential for plants. The structures of flavonoids are based upon a phenylpropanoid part (C₆–C₃) combined with an acetate-derived aromatic ring (C₆) usually possessing a 2-arylchroman skeleton. Isoflavonoids are a subclass of flavonoid in which the phenylalanine-derived ring B is rearranged to C-3, resulting in the overall structure of a 3-arylchroman skeleton (Figure 1). The distribution of isoflavonoids is very restricted: more than 95% of isoflavonoids are present in the leguminous plants (Fabaceae) (Hegnauer and Grayer-

556 Yuji Sawada et al.

![Diagram](attachment:diagram.png)

Figure 1. The reactions catalysed by 2-hydroxyisoflavanone synthase (CYP93C2), (2S)-flavanone 2-hydroxylase (CYP93B1), and flavone synthase II (CYP93B2-4).

The names of both flavonoid skeletons and individual compounds (in parentheses) are shown under the structures. 2-Hydroxyisoflavanone and 2-hydroxyflavanone are converted to isoflavone and flavone, respectively, by acid treatment *in vitro* or by putative specific dehydratases *in vivo*. 3-Hydroxyflavanone is the by-product of the IFS reaction. Flavone synthase II directly converts flavanone to flavone. F2H, (2S)-flavanone 2-hydroxylase; IFS, 2-hydroxyisoflavanone synthase; FNSII, flavone synthase II.

cytochrome P450 in the microsomes of leguminous plant cell cultures (Hashim *et al.*, 1990; Kochs and Grisebach, 1986), and isoflavone is the dehydration product of the highly unstable intermediate 2-hydroxyisoflavanone. The P450 enzyme that carries out 2-hydroxylation and aryl migration of a flavanone is called 2-hydroxyisoflavanone synthase (IFS; Heller and Forkmann, 1994). The IFS reaction was previously examined by Hashim *et al.* (1990) using carefully prepared microsomes of elicitor-treated *Pueraria lobata* cells. Based on the observation of the formation of a by-product, 3-hydroxyflavanone, in addition to the major 2-hydroxyisoflavanone and other findings, a mechanism involving abstraction of the hydrogen from C-3 by an activated oxygen intermediate bound to heme iron (Fe = O), followed by migration of the aryl group from C-2 to C-3 and rebinding of the hydroxyl radical to C-2, has been proposed. However, further detailed analysis of the mechanism of the IFS reaction has been hampered due to the lack of purified proteins and amino acid sequence information, as is true of many other plant P450s. This unique reaction of the P450 occurring in legumes is of interest from the mechanistic as well as the evolutionary viewpoint. Furthermore, biotechnological application through the artificial modification of IFS is expected to improve the plant properties and to enable the mass production of health-promoting isoflavonoids. Thus understanding the protein structure required for the IFS reaction should be of fundamental significance.

Since 1990, plant P450 cDNAs have been isolated, mainly using the nucleotide sequence information without prior protein purification. At present about 1000 P450 sequences comprising 52 families are known, and the catalytic functions of several P450-dependent monooxygenases have been identified (reviewed by Chapple, 1998; Kahn and Durst, 2000; Werck-Reichhart *et al.*, 2002; http://drnelson.utmem.edu/CytochromeP450.html). Recently, P450s belonging to CYP93C subfamilies – CYP93C2 of licorice (*Glycyrrhiza echinata*; Fabaceae: Akashi *et al.*, 1999a) and CYP93C1 of soybean (Jung *et al.*, 2000; Steele *et al.*, 1999), were identified as IFS using heterologous yeast and insect expression systems. In the reaction with recombinant licorice CYP93C2 protein, the formation of the by-product 3-hydroxyflavanone (3,7,4′-trihydroxyflavanone) from a flavanone (7,4′-dihydroxyflavanone) together with the major 2-hydroxyisoflavanone product (2,7,4′-trihydroxyisoflavanone) was observed (Akashi *et al.*, 1999a).

The P450s are very widely distributed in biological organisms, and a great diversity of oxidative reactions are catalysed by P450s. X-ray crystal structures of P450s of

© Blackwell Science Ltd, *The Plant Journal*, (2002), **31**, 555–564

bacterial (Cupp-Vickery and Poulos, 1995; Hasemann *et al.*, 1994; Park *et al.*, 1997; Podust *et al.*, 2001; Poulos *et al.*, 1985; Poulos *et al.*, 1987; Ravichandran *et al.*, 1993; Yano *et al.*, 2000) and vertebrate (Williams *et al.*, 2000a) origins are now available. The P450 domains composed of α-helices and β-sheets are fundamentally conserved in these structures (Graham and Peterson, 1999; Williams *et al.*, 2000b), but individual isoforms usually exhibit narrow substrate specificities and strict regiospecificities of the oxidation site in the substrate (Mansuy, 1998). Thus the relationship between protein structure and reaction catalysed has attracted the attention of many researchers: for example, substrate recognition by P450<sub>cam</sub> (CYP101) has been studied using a combination of site-directed mutagenesis and substrate analogues (Müller *et al.*, 1995). Six putative substrate-recognition sites (SRS 1–6) for P450s have been proposed in the regions of primary sequences based on the alignment of the CYP2 family with P450<sub>cam</sub> (Gotoh, 1992), and a number of key amino acid residues that affect the substrate specificity have been reported within the SRSs (Domanski and Halpert, 2001; Wachenfeldt and Johnson, 1995). Computer-generated homology models of P450s can also be constructed based on their amino acid sequences and the P450 crystal structures (Szklarz and Halpert, 1997), providing an important method for predicting key residues for reactions.

The purpose of the present study is to identify the amino acid residues responsible for the unique aryl migration which represents a new category of P450 reactions catalysed by the CYP93C proteins. Based on the alignment of CYP93 family members and construction of the three-dimensional structures of CYP93C2 using homology modelling, candidates for the important functional residues in the active site of CYP93C2 were estimated. These residues were converted to others by site-directed mutagenesis, and the catalytic functions of the mutants were analysed. The results are discussed in relation to the putative three-dimensional structure of the active site and the proposed sequence of the IFS reaction.

### Results

#### Identification of 3β-hydroxyflavanone as the by-product of the IFS reaction

The formation of 3-hydroxyflavanone as the by-product of an IFS reaction by CYP93C protein provides an important clue to the mechanism of the unusual aryl migration catalysed by a P450 (Figure 1). The occurrence of this by-product has been reported in previous biochemical studies by Kochs and Grisebach (1986) and Hashim *et al.* (1990), but the phenomenon has not been analysed in detail in the recent molecular cloning work on IFS. We have reported that the by-product of the IFS reaction with heterologously expressed CYP93C2 from liquiritigenin (7,4′-dihydroxyflavanone) had a UV spectrum identical to 3,7,4′-trihydroxyflavanone (Akashi *et al.*, 1999a). Now the <sup>1</sup>H-NMR spectrum of the HPLC-purified sample has confirmed the structure of 3,7,4′-trihydroxyflavanone (see Experimental procedures). It further showed the dihedral angle between 2-H and 3-H about 180° from the spin-coupling constant of vicinal protons (*J* = 12 Hz), indicating that the configuration of the hydroxyl at C-3 is β-equatorial.

In interpreting the roles of amino acid residues at the active site of the enzyme, it is essential to establish whether the apparent by-product, 3-hydroxyflavanone, is the intermediate in the aryl migration. Previously, the intermediacies of 3,7,4′-trihydroxyflavanone and 3,5,7,4′-tetrahydroxyflavanone in biosynthesis of the isoflavonoid skeleton have been reported to be unlikely on the basis of *in vivo* incorporation experiments with intact plants (Wong, 1965), but no *in vitro* experiments have been conducted. Thus 3,7,4′-trihydroxyflavanone recovered from the CYP93C2 reaction with liquiritigenin as the substrate was reincubated with the yeast microsome expressing CYP93C2 protein, to examine if this putative intermediate could be converted to 2-hydroxyisoflavanone. However, no 2,7,4′-trihydroxyisoflavanone or other reaction product(s) was detected.

#### Site-directed mutagenesis of CYP93C2 and catalytic activities of the mutants

CYP93 proteins catalyse multiple reactions in flavonoid biosynthesis: CYP93A1 (pterocarpan 6a-hydroxylase; Schopfer *et al.*, 1998) and CYP93B1 [(2*S*)-flavanone 2-hydroxylase; Akashi *et al.*, 1998] catalyse the typical mono-oxygenation, and CYP93B2-4 (flavone synthase II) catalyses the formation of a double bond (Figure 1; Akashi *et al.*, 1999b; Martens and Forkmann, 1999). CYP93C also catalyses 2-hydroxylation, but specifically accompanied by the aryl migration of a flavanone molecule (Akashi *et al.*, 1999a; Jung *et al.*, 2000; Shimada *et al.*, 2000; Steele *et al.*, 1999). Our initial trial for the elucidation of amino acid residues and/or sequence motifs for this unique reaction employed domain-swapping between CYP93C2 and CYP93B1. Three segments derived from each cDNA were combined to produce chimeric proteins in recombinant yeast, and activities toward the flavanone substrate were assayed. However, the chimeras did not show significant catalytic activities, although a faint activity was observed in some combinations (data not shown). The loss of major activity [either IFS or (2*S*)-flavanone 2-hydroxylase] was considered to be due to the too-low sequence identity between the two CYP93 family proteins (54% at the amino acid level) to form chimeric proteins with a catalytically active conformation.

558 Yuji Sawada et al.

Amino acid number

N- MBR
100 200 300 400 -C
1 2 3 4 5 6 HBR
CYP93C2 ★Arg 104 ★Ser 310 ★Lys 375
CYP93C1v2 RF-QTSA AGTDS PVVKRK
CYP93C17 RF-QTSA AGTDS PVVKRK
CYP93B1 RI-ESTA AGTDT PMVTRR
CYP93B2 RK-HSLA AGTDT PMLIRK
CYP93B4 RK-HSTA AGTDT PMLSRE
CYP93A1 RPGQNV A AGTDT PLVVRE
B'-helix I-helix β-sheet 1-4

Figure 2. Putative positions of the substrate recognition sites (1–6) of CYP93C2 (accession number AB023636) and the alignment of partial amino acid sequences of CYP93 family. The identical amino acid residues in at least four of the seven sequences are shown in reverse type. The residues converted to other amino acids by site-directed mutagenesis in this work are indicated by stars. The secondary structural elements predicted by the homology model of CYP93C2 are indicated below the alignment. The functions of CYP93 family members have been identified as follows: CYP93C, IFS; CYP93B1, F2H; CYP93B2–4, FNSII; CYP93A1, pterocarpan 6a-hydroxylase. MBR, membrane-binding region; HBR, heme-binding region.

Comparison of amino acid sequences of CYP93 proteins provided interesting information (Figure 2). All CYP93 proteins, except for CYP93C proteins, have a conserved AGTD T in the centre of the I-helix, the longest α-helix in P450 structures (Wachenfeldt and Johnson, 1995). The terminal Thr in this conserved motif in SRS4 is generally thought to promote O–O bond scission in the P450 reaction cycle (Aikens and Sligar, 1994). CYP93C proteins lack this Thr residue and, instead, have Ser at the corresponding position. Also, the computer-generated three-dimensional structure of CYP93C2 by homology modelling based on the X-ray structure of P450BM3 (Ravichandran et al., 1993) indicated several amino acid residues located at the active site of the enzyme. Among those, a Lys residue at the end of β-sheet 1–4 (Wachenfeldt and Johnson, 1995) located in the SRS5 (Lys 375 of CYP93C2) was present in all CYP93C, but was not found in other CYP93 (Figure 2). In CYP93B1, which catalyses hydroxylation of flavanone at C-2 but not aryl migration, this residue is substituted to Thr. In addition, Arg 104 within the SRS1 of CYP93C2 at the B′-helix in the homology model was conspicuous at the putative active site. It must be noted that the structures around this region, including the B-C loop of CYP2C5 (a eukaryotic P450), are very variable among the known P450 crystals, and thus a potential problem may exist in predicting the structure of this region of a eukaryotic P450 using the P450BM3 structure (Ridderström et al., 2000; Williams et al., 2000b). However, as the importance of amino acid residues at SRS1 of mammalian P450s in substrate recognition and oxidation has been demonstrated (Domanski and Halpert, 2001), it would be interesting to

CYP93C2 S310T K375T R104A
0.02
Absorbance
0
-0.02
400 450 500 400 450 500 400 450 500 400 450 500
Wavelength (nm)

Figure 3. Reduced carbon monoxide difference spectra of CYP93C2 constructs in yeast microsomal fractions. The microsomes (~0.5 mg protein ml⁻¹) expressing CYP93C2 and the mutant proteins reduced with sodium dithionite were bubbled with carbon monoxide, and the absorbance spectra from 400 to 500 nm were recorded.

test the activity of CYP93C2 mutagenized at this Arg. Therefore we prepared point mutants of CYP93C2 to examine the roles of Ser 310, Lys 375 and Arg 104. The wild-type and mutant proteins were expressed in heterologous yeast systems, and the reduced carbon monoxide difference spectra of the microsomes were measured. As shown in Figure 3, absorbance at 450 nm was observed in wild-type and mutant proteins S310T and K375T, but R104A showed only 420 nm absorption, indicating that this mutant cannot yield an active conformation of P450 (Martinis et al., 1996). The P450 level in the microsomes of recombinant yeast expressing K375T was reproducibly higher than those expressing the wild-type and S310T proteins (Table 1).

The microsomes were tested for activity toward the flavanone substrate. Figure 4 shows a typical HPLC of the products from liquiritigenin with the microsome of yeast expressing wild-type and mutant CYP93C2 proteins. The product ratio calculated using known absorption constants, and the HPLC peak areas of both direct products and those following conversion of 2-hydroxyisoflavone into isoflavone, are listed in Table 1. The reaction with the wild-type CYP93C2 protein yielded the main (92% at pH 7.5) 2-hydroxyisoflavone (2,7,4′-trihydroxyisoflavanone) and a minor (8%) by-product, 3-hydroxyflavanone (3,7,4′-trihydroxyflavanone) (Figure 4a, upper panel). Acid treatment converted 2,7,4′-trihydroxyisoflavone to 7,4′-dihydroxyisoflavone (daidzein), but 3,7,4′-trihydroxyflavanone remained unchanged (Figure 4a, lower panel). As shown in Figure 4(b), the mutant S310T also catalysed the aryl migration from liquiritigenin to yield 2-hydroxyiso-

© Blackwell Science Ltd, *The Plant Journal*, (2002), 31, 555–564

(a) CYP93C2                      (b) S310T                       (c) K375T
2HI                              2HI                            3HF
Liq                              Liq                            Liq
3HF                              3HF                            DHF
(Acid-treated)                   (Acid-treated)                 (Acid-treated)
Liq                              Liq                            3HF
Dai                              Dai                            Liq
3HF                              DHF                            3HF

Retention time (min)

Figure 4. Reversed-phase HPLC profiles of the products from liquiritigenin in the reactions with recombinant yeast microsomes expressing CYP93C2 (a) and the mutants S310T (b) and K375T (c). Chromatograms of the direct products (upper panel) and those after acid-treatment (lower panel) are shown. Solvent, 40% methanol in water; detection, UV 285 nm. Liq, liquiritigenin; 2HI, 2,7,4'-trihydroxyisoflavanone; 3HF, 3,7,4'-trihydroxyflavanone; Dai, daidzein; DHF, 7,4'-dihydroxyflavone. For structures of the compounds see Figure 1.

Table 1. Expression levels and catalytic activities of CYP93C2 and the mutants S310T and K375T in recombinant yeast microsomes

| Enzyme | P 450 level<sup>a</sup> [nmol (g protein)<sup>-1</sup>] | Product<sup>b</sup> | Product ratio<sup>c</sup> (%) | $K_{\mathrm{m}}^{\mathrm{d}, \mathrm{e}}$ ($\mu$M) | $V_{\max }^{\mathrm{e}}$ [(product mol) min<sup>-1</sup>] (P450mol)<sup>-1</sup> |
|--------|-------------------------------------------------------|---------------------|-------------------------------|---------------------------------------------|--------------------------------------------------------------------|
| CYP93C2 | 117 ± 8                                              | 2HI                 | 92                          | 8                                           | 9                                                                  |
|         |                                                       | 3HF                 | 8                           |                                             | 4                                                                  |
| S310T   | 176 ± 17                                             | 2HI                 | 57                          | 6                                           | 0.6                                                                |
|         |                                                       | 3HF                 | 36                          |                                             | 0.5                                                                |
|         |                                                       | DHF                 | 7                           |                                             | 0.03                                                               |
| K375T   | 408 ± 6                                              | 3HF                 | 100                         | 11                                          | 0.1                                                                |

<sup>a</sup> Mean ± SD from three independent experiments.  
<sup>b</sup> Abbreviations: 2HI, 2,7,4'-trihydroxyisoflavanone; 3HF, 3,7,4'-trihydroxyflavanone; DHF, 7,4'-dihydroxyflavone.  
<sup>c</sup> Average from two independent experiments (maximum deviation ~2%).  
<sup>d</sup> Calculated from total production of main and minor products.  
<sup>e</sup> Average of two independent experiments (maximum deviation ~25%).

flavanone, but the ratio of the by-product (3-hydroxyflavanone) formation was increased to 36% (pH 7.5), and a small amount (7%) of a new product, 7,4'-dihydroxyflavone, was detected. It appears that the substitution of Ser for the Thr residue in CYP93C suppressed aryl migration. A more dramatic change in the product ratio was observed in the reaction with the mutant K375T (Figure 4c). This mutant produced only 3-hydroxyflavanone, whereas aryl migration product was hardly detected. Clearly, Lys 375 of CYP93C2 is essential for the aryl migration of a flavanone molecule to produce the isoflavonoid skeleton.

The rates and kinetic parameters for the reactions were calculated from data from experiments using <sup>14</sup>C-labelled liquiritigenin as the substrate (Table 1). The incorporation of radioactivity into the products recovered from the TLC of the reaction mixture was regarded as the enzyme activity, and the Michaelis constant ($K_{\mathrm{m}}$) was based on the total production of main and minor products in each reaction. There were no remarkable differences in $K_{\mathrm{m}}$ values between CYP93C2 and the mutants. In contrast, maximum velocity ($V_{\max }$) values showed marked differences: these values indicate that the overall rates for S310T

© Blackwell Science Ltd, The Plant Journal, (2002), 31, 555–564

560 Yuji Sawada et al.

and K375T reactions were about 1/10 and 1/100 of the wild type, respectively. The product percentages and \( V_{\text{max}} \) values shown in Table 1, deduced from different experiments with cold and radiolabelled experiments, are roughly correlated.

As expected, the microsomes of yeast transformed with the mutant CYP93C2R104A, which showed only a 420 nm band in carbon monoxide-difference spectra, had no activity toward the flavanone substrate. We also produced other CYP93C2 mutants in which the amino acid residues supposed to be located at the active site were replaced by others, and examined their catalytic activities. As a result, the mutants in which Gln 106, Val 119, and Ser 305 were replaced with corresponding Glu, Leu, and Thr in CYP93B1, respectively, did not show significant changes in catalysis compared to the wild-type enzyme (data not shown).

### pH-Dependence of the main and by-product formation by wild-type and mutant CYP93C2 proteins

The pH-dependence of catalytic activities on CYP93C2 and mutant proteins S310T and K375T was analysed. As shown in Figure 5(a), the reaction by the wild type proceeded in a wide range of pHs, but the overall reaction rate and 2,7,4'-trihydroxyisoflavanone formation were highest between pH 7.5 and 8.5. The yield of 3,7,4'-trihydroxyflavanone was more-or-less constant, despite a tendency toward higher production at the lower pH. The pH profile of formation of the aryl migration product by the mutant S310T was similar to that by the wild type (Figure 5b), and the formation of non-migrated products (3,7,4'-trihydroxyflavanone and 7,4'-dihydroxyflavone) was also observed in a wide pH range above 7.0. The mutant K375T displayed the sole enzyme activity of producing 3,7,4'-trihydroxyflavanone in a wide pH range, with the highest activity between pH 6 and 7 (Figure 5c). Its pH profile was clearly different from those of the IFS activity by wild-type and the mutant S310T.

### Discussion

In this study we identified the amino acid residues essential for the oxidative aryl migration catalysed by a CYP93C P450 protein. The domain swapping between CYP93C2 and CYP93B1 was not successful. We then took the approach of site-directed mutagenesis of the CYP93C2 protein considering the sequences of other CYP93s that are also involved in flavonoid metabolism. Computer-generated homology modelling of the stereo structure of CYP93C2 was particularly useful in selecting the amino acid residues to be converted to others, and Ser 310 and Lys 375 were shown to be key amino acid residues by the assays with recombinant yeast microsomes. A possible CYP93C2 structure in which the substrate [(2S)-liquiritigenin] is accommodated at the active site is shown in Figure 6. In this illustration, the ε-amino group of Lys 375 is proximal to (2S)-liquiritigenin and can interact with the hydroxyl group at C-7, probably acting as an anchor for the substrate. Also, the migration of the B-ring from C-2 to C-3 of the (2S)-flavanone molecule can be facilitated by the presence of Ser instead of Thr in the centre of the I-helix, providing a wide space for the migration. Initial abstraction of the pro-R hydrogen from C-3 of (2S)-flavanone by the activated oxygen intermediate would be followed by aryl migration to give isoflavonoid skeleton or re-binding of the hydroxyl at C-3 to yield the by-product 3-hydroxyflavanone. The orientation of the hydrogen abstraction/hydroxyl re-binding at C-3 is supported from the identification of the 3β-hydroxyl configuration of the by-product. However, Ser 310 and Lys 375 do not seem to be the sole residues responsible for the aryl migration because the

© Blackwell Science Ltd, *The Plant Journal*, (2002), 31, 555–564

Mutagenesis of P450 in isoflavonoid pathway 561

I-helix  
Ser 310  

Arg 104  
Gln 106  

B'-helix  
Lys 375  

β-sheet 1-4  
(2S)-Liquiritigenin  

C-helix  
Val 119  

Heme  

Figure 6. Computer-generated three-dimensional structure of the active site of CYP93C2. Amino acid residues (2S)-liquiritigenin and heme are shown as balls and sticks (carbon, black; nitrogen, blue; oxygen, red; iron, purple). Hydrogens at β-(pro-R) and α-(pro-S) positions at C-3 of (2S)-liquiritigenin are displayed as white balls.

point mutant of CYP93B1 (flavanone 2-hydroxylase) in which the Thr in the centre of the I-helix or in β-sheet 1–4 was converted into the corresponding Ser or Lys of CYP93C2 did not show aryl migration or hydroxylation at C-3 of the flavanone substrate (data not shown). Further amino acid residues in CYP93C proteins critical for aryl migration may be present, and/or the overall protein conformation of CYP93C can be an important factor determining the whole catalytic activity. The production of 7,4′-dihydroxyflavone by the mutant S310T could be the result of an initial abstraction of hydrogen from C-2 instead of C-3, as demonstrated in CYP93B2-4 reactions (Akashi et al., 1999b; Martens and Forkmann, 1999), indicating loose binding of the substrate at the active site of the mutant.

3-Hydroxyflavanone was not shown to be the intermediate of aryl migration in the IFS reaction. A reduction of aryl migrating activity, instead of a relative increase of flavanone 3β-hydroxylase activity partially compensating for the overall decrease of P450 activity, was observed in CYP93C2 mutants S310T and K375T. Also, the pH optima for IFS and flavanone 3β-hydroxylase activities in reactions with the mutants were different. Taken together, some different amino acid residues should be involved in the production of 2-hydroxyisoflavanone and 3-hydroxyflavanone in the CYP93C reaction, and the 3-hydroxyflavanone formation should be caused by an inefficiency of the aryl migrating apparatus of the enzyme protein. Flavanone 3β-hydroxylase usually participates in the biosynthesis of anthocyanins and flavonols, and it has been assigned to the soluble 2-oxoglutarate-dependent dioxygenases rather than to P450s (Britsch et al., 1992; Heller and Forkmann, 1994). The findings reported here suggest that flavanone

3β-hydroxylase of the P450 type was present in past plant species, and could be the evolutionary origin of IFS. An example of involvement of P450 and 2-oxoglutarate-dependent dioxygenase in the same reaction of natural product biosynthesis can be found in flavone formation from a flavanone: a P450 flavone synthase II is involved in flavone biosynthesis in snapdragon, torenia (Akashi et al., 1999b), and gerbera (Martens and Forkmann, 1999), while a soluble flavone synthase I is involved in parsley (Martens et al., 2001). Also, hydroxylation at C-3 of the gibberellin molecule is carried out by a P450 in plants and by a dioxygenase in a fungus (Yamaguchi and Kamiya, 2000). The aryl migration activity of P450 IFS must have emerged through a rare gain-of-function event by mutations from ancestral CYP93 genes, and have survived and become dominant in present-day legumes, because the enzyme reaction products have been of great ecophysiological importance (Stafford, 2000).

Finally, several oxidative rearrangements of carbon skeletons and C–C bond fissions in natural product biosynthesis, in addition to the isoflavonoid skeleton construction, have been predicted to be P450-dependent (Hakamatsuka et al., 1991). These include the ring contraction of ent-7-hydroxykaurenoic acid producing GA<sub>12</sub>-aldehyde in gibberellin biosynthesis, and another ring cleavage of a monoterpenoid loganin to form secologanin (an essential precursor of a huge family of monoterpenoid indole alkaloids). Recent cloning, and the identification of cDNAs of enzymes responsible for these reactions, clearly showed that these are in fact P450s: CYP88A of maize and Arabidopsis (Hellwiel et al., 2001); CYP68 of *Gibberella fujikuroi* possessing multifunctional enzyme activity including GA<sub>12</sub>-forming activity (Rojas et al., 2001); and

© Blackwell Science Ltd, The Plant Journal, (2002), 31, 555–564

CYP72A of *Catharanthus roseus* attributed to secologanin synthase (Irmler et al., 2000). Future investigations of these P450s with the aid of homology modelling and site-directed mutagenesis, like those reported here, may open up a new area of P450 research which will clarify the mechanism and evolution of the construction of carbon skeletons of natural products possessing significant physiological roles in both plants and other organisms.

### Experimental procedures

#### Construction of expression vectors
*KpnI* and *NotI* sites were introduced into upstream and downstream regions of initiation and stop codons, respectively, of CYP93C2 sequence by PCR using Ex Taq DNA polymerase (Takara, Tokyo, Japan) with the primers 93C2-S1 (5′-ACAGG-TACCATGTTGGTGGAACTTGC-3′; *KpnI* site shown in bold) and 93C2-AS1 (5′-GGTGTCGCGCGGCCGCTTTACGACGAAAAG-3′; *NotI* site shown in bold), and CYP93C2 cDNA in pBluescript SK(−) as the template (Akashi et al., 1999a). The PCR product was cloned into pT7Blue(R) vector (Novagen, Madison, WI) to produce plasmid pT7-93C2. The *KpnI–NotI* fragment of pT7-93C2 was inserted into the corresponding sites of the yeast expression vector pYES2 (Invitrogen, Carlsbad, CA).

#### PCR-based site-directed mutagenesis
Three partial CYP93C2 sequences containing specific restriction sites and point mutations were amplified by PCR using Ex Taq DNA polymerase with pT7-93C2 as template and the following sets of primers (mutated sites are underlined in each primer): S310T primer (5′-GTTGTTGATGAGCTCTGACAGAGCCCAGTCTG-TTGCCACC GCCGTGGTATCCGTCCCTGC-3′; *SacI* site shown in bold) and U-19 primer (5′-ATGACAAAAGGACGGCGAA CG-3′); K375T primer (5′-TTCCG CATGCA CCCACC ACT ACCC GTGG TCA C-AAGAAAGT GCGTG-3′; *SphI* site shown in bold) and R-20 primer (5′-CAGCTATGACC ATGATT ACG-3′); and R104A primer (5′-TA-GGCGCCTAATGGCAGAGGTTTGGAAACGCTGTGTT-3′; *BbeI* site shown in bold) and U-19 primer. The CYP93C2 fragments containing the mutations S310T, K375T and R104A were excised from the PCR products by treatment with *SacI* [the *SacI* site existed in the pT7Blue(R) sequence]; *SphI* (the *SphI* site also existed in the vector sequence); and the combination of *BbeI* and *KpnI* (the *KpnI* site existed in the 5′-adaptor), respectively. The corresponding regions of pT7-93C2 were removed by digestion with restriction enzymes (*SacI*, *SphI*, or combined *BbeI* and *KpnI*) and replaced with the mutated fragments. The sequences of mutant cDNAs were verified by DNA sequence analysis, and *KpnI–NotI* fragments were subcloned into pYES2.

#### Expression of mutant proteins in yeast cells and preparation of microsomes
Transformation of the yeast expression vector harbouring wild-type or mutant CYP93C2 cDNA into *Saccharomyces cerevisiae* BJ2168 strain (Nippon Gene, Tokyo, Japan), selection of the transformants, and induction of P450 proteins in yeast cells were performed as described previously (Akashi et al., 1999a). The yeast microsomes prepared by ultracentrifugation at 160 000 g for 90 min (Akashi et al., 1999a) were suspended in 0.1 M potassium phosphate buffer (pH 7.5) supplemented with 10% (w/v) sucrose and 14 mM 2-mercaptoethanol. The P450 content in the microsome was determined based on the method of Omura and Sato (1964). The microsome (~0.5 mg protein ml⁻¹) reduced with sodium dithionite was bubbled with carbon monoxide for 30 sec, and *A*₄₅₀ and *A*₄₉₀ were measured with a non-bubbled aliquot as the blank. The P450 concentration (mM) was calculated according to the formula (*A*₄₅₀ – *A*₄₉₀)/91.

#### Enzyme assay
(2RS)-Liquiritigenin (7,4′-dihydroxyflavanone, 100 nmol) dissolved in 20 ml 2-methoxyethanol was incubated with 1 ml of the recombinant yeast microsome (~1.0 mg protein ml⁻¹) prepared as above in the presence of 1 mM NADPH. The reaction mixture was extracted with ethyl acetate after incubation at 30°C for 1 h, and the extract was evaporated to dryness *in vacuo*. The products were dissolved in methanol, and an aliquot was analysed by reversed-phase HPLC using a CLC-ODS column (Shim-Pack 6.0 × 150 mm; Shimadzu, Kyoto, Japan) with 40% methanol in water (flow rate 1 ml min⁻¹) monitored at 285 nm. Acid treatment of the reaction mixture was performed by stirring the dried extract in 10% (v/v) aqueous hydrochloric acid (35–37%) in methanol (200 ml) at room temperature for 1 h and subsequently 55°C for 10 min. The ethyl acetate extract of the reaction mixture was analysed by reversed-phase HPLC as above. The concentration of each product was calculated from the HPLC peak area, referring to the peak area of known concentrations of standard samples. The concentration of 2,7,4′-trihydroxyisoflavnone was determined after conversion to daidzein by acid treatment.

The pH-dependency of the reaction was measured in 0.1 M potassium phosphate buffer (pH 5.5–8.5) supplemented with 10% (w/v) sucrose and 14 mM 2-mercaptoethanol. Recombinant microsomes (50 μl) containing 75 nM, 0.6 μM and 1.5 μM equivalent of P450 each for CYP93C2, S310T and K375T, respectively, were suspended in 420 μl of the buffer and incubated with (2RS)-liquiritigenin (100 nmol) in the presence of 1 mM NADPH in a total volume of 500 μl at 30°C for 40 min. The enzyme activities were estimated from peak areas of reversed-phase HPLC by the same procedure as above.

For the measurement of ¹H-NMR spectra, a large-scale incubation of liquiritigenin (3 mg) with 0.5 mM NADPH and the recombinant yeast microsome (~30 mg protein) expressing CYP93C2 was carried out. After incubation at 25°C for 2 h, the ethyl acetate extract of the reaction mixture was concentrated and subjected to silica-gel TLC [Kieselgel F254 (Merck, Darmstadt, Germany); solvent, toluene : ethyl acetate : methanol : light petroleum = 6 : 4 : 1 : 3], and 2,7,4′-trihydroxyisoflavnone (*R*~f~ 0.20) and 3,7,4′-trihydroxyflavanone (*R*~f~ 0.27) were recovered. Assay and separation from TLC were performed 10 times, and the resultant products were combined and further purified by reversed-phase HPLC with 35% methanol in water (2,7,4′-trihydroxyisoflavnone, *R*~t~ 11.5 min; 3,7,4′-trihydroxyflavanone, *R*~t~ 26.5 min). ¹H-NMR spectra were recorded with a Bruker ARX400 spectrometer at 27°C. The spectrum of 2,7,4′-trihydroxyisoflavnone in *d*₆-acetone was identical with the reported spectrum (Hashim et al., 1990). 3,7,4′-Trihydroxyflavanone: δ (p.p.m.) 4.50 d (1H, *J* = 11.7 Hz, 3ax-H); 5.30 d (1H, *J* = 11.9 Hz, 2ax-H); 6.39 d (1H, *J* = 2.2 Hz, 8-H); 6.63 br. d (1H, *J* = 8.6 Hz, 6-H); 6.89 d (2H, *J* = 8.6 Hz, 3′, 5′-H); 7.43 d (2H, *J* = 8.6 Hz, 2′, 6′-H); 7.71 d (1H, *J* = 8.7 Hz, 5-H).

Radiolabelled experiments and measurements of kinetic parameters

(2S)-[^14^C]Liquiritigenin (5.88 GBq mol⁻¹) was prepared by the incubation of 4-coumaroyl-CoA and [^14^C]malonyl-CoA with the cell-free extract of cultured *Glycyrrhiza echinata* cells, as described previously (Otani *et al.*, 1994). Cold (2S)-liquiritigenin was enzymatically synthesized by the incubation of isoliquiritigenin with recombinant chalcone isomerase of *Lotus japonicus* (Shimada *et al.*, unpublished results). To determine the $K_{\mathrm{m}}$ values, the combined substrate of 0.05 μM (2S)-[^14^C]liquiritigenin and 0.2–100 μM cold (2S)-liquiritigenin in 10 μl 2-methoxyethanol was incubated with 180 μl of recombinant yeast microsome (75 nM, 0.6 μM and 1.5 μM P450 for CYP93C2, S310T and K375T, respectively) in the presence of 1 mM NADPH in a total volume of 200 μl. After incubation at 30°C for 10 min, the reaction was stopped by adding 10 μl acetic acid. The ethyl acetate extract of the reaction mixture was subjected to cellulose TLC [Funacel SF, Funakoshi, Tokyo, Japan; solvent, 15% (v/v) acetic acid in water] and analysed by a Typhoon 8600 (Amersham Pharmacia Biotech, Buckinghamshire, UK). The products (2,7,4′-trihydroxyisoflavone, $R_{f}$ 0.76; 3,7,4′-trihydroxyflavanone, $R_{f}$ 0.64; 7,4′-dihydroxyflavone, $R_{f}$ 0.20) were scraped from the TLC plate, and radioactivity was measured by a liquid scintillation counter. The kinetic constants were calculated using the Lineweaver-Burk equation.

Homology modelling of the CYP93C2 protein structure

The amino acid sequence of CYP93C2 was aligned with the reported stereostructures of P450BM3 (accession numbers 2HPDA, 2BMHA and 1BU7A) available in the ExNRL-3D database (http://swissmodel.ncifcrf.gov/SM_Blast.html) using the SWISS-MODEL BLAST program. Three-dimensional homology models of CYP93C2 were constructed from the alignment data using the program MODELLER-4 (Šali and Blundell, 1993). From these, we selected one model with the best value of object function (14403.7139) for the analyses. The overall quality of the CYP93C2 model was checked by PROCHECK 3.0 (Laskowski et al., 1993). Forty-eight out of 490 residues were in disallowed regions on the Ramachandran plot, but these were on the surface of the molecule and/or distant from the active site. The illustration (Figure 6) for the substrate-protein complex was generated by manual docking of individual structures using INSIGHT II (Accelrys, San Diego, CA), and drawn with MOLSCRIPT 2.12 (Krulis, 1991) and RASTER3D 2.5 (Merritt and Bacon, 1997) programs.

Acknowledgements

We thank Dr Hiroshi Hirota (RIKEN Genomic Sciences Center) for the measurement of NMR spectra. We are grateful to Yutaka Nakatani and Nobutada Nisougi (Nihon University) for experimental assistance. We also thank the anonymous reviewers for generous comments and useful information regarding P450 structure and function. This work was supported by a Grant-in-Aid for Encouragement of Young Scientists (A) (No. 13780474) from the Japan Society for the Promotion of Science, and by a Grant-in-Aid for Scientific Research on Priority Areas (A) (No. 1302470) from the Ministry of Education, Culture, Sports, Science, and Technology of Japan.

References

Aikens, J. and Sligar, S.G. (1994) Kinetic solvent isotope effects during oxygen activation by cytochrome P-450cam. *J. Am. Chem. Soc.* **116**, 1143–1144.

Akashi, T., Aoki, T. and Ayabe, S. (1998) Identification of a cytochrome P450 cDNA encoding (2S)-flavanone 2-hydroxylase of licorice (*Glycyrrhiza echinata* L. Fabaceae) which represents licodione synthase and flavone synthase II. *FEBS Lett.* **431**, 287–290.

Akashi, T., Aoki, T. and Ayabe, S. (1999a) Cloning and functional expression of a cytochrome P450 cDNA encoding 2-hydroxyisoflavanone synthase involved in biosynthesis of the isoflavonoid skeleton in licorice. *Plant Physiol.* **121**, 821–828.

Akashi, T., Fukuchi-Mizutani, M., Aoki, T., Ueyama, Y., Yonekura-Sakakibara, K., Tanaka, Y., Kusumi, T. and Ayabe, S. (1999b) Molecular cloning and biochemical characterization of a novel cytochrome 450, flavone synthase II, that catalyzes direct conversion of flavanones to flavones. *Plant Cell Physiol.* **40**, 1182–1186.

Britsch, L., Ruhnau-Brich, B. and Forkmann, G. (1992) Molecular cloning, sequence analysis, and in vitro expression of flavanone 3β-hydroxylase from *Petunia hybrida*. *J. Biol. Chem.* **267**, 5380–5387.

Chapple, C. (1998) Molecular-genetic analysis of plant cytochrome P450-dependent monooxygenases. *Annu. Rev. Plant Physiol. Plant Mol. Biol.* **49**, 311–343.

Cupp-Vickery, J.R. and Poulos, T.L. (1995) Structure of cytochrome P450eryF involved in erythromycin biosynthesis. *Nature Struct. Biol.* **2**, 144–153.

Dixon, R.A. and Steele, C.L. (1999) Flavonoids and isoflavonoids – a gold mine for metabolic engineering. *Trends Plant Sci.* **4**, 394–400.

Dixon, R.A., Canovas, P., Guo, Z.J., He, X.Z., Lamb, C. and McAlister, F. (1999) Molecular controls for isoflavonoid biosynthesis in relation to plant and human health. In *Recent Advances in Phytochemistry*, Vol. 33. *Phytochemicals in Human Health Protection, Nutrition, and Plant Defense* (Romeo, J.T., ed.). New York, NY: Kluwer Academic, pp. 133–159.

Domanski, T.L. and Halpert, J.R. (2001) Analysis of mammalian cytochrome P450 structure and function by site-directed mutagenesis. *Curr. Drug Metab.* **2**, 117–137.

Gotoh, O. (1992) Substrate recognition sites in cytochrome P450 family 2 (CYP2) proteins inferred from comparative analyses of amino acid and coding nucleotide sequences. *J. Biol. Chem.* **267**, 83–90.

Graham, S.E. and Peterson, J.A. (1999) How similar are P450s and what can their differences teach us? *Arch. Biochem. Biophys.* **369**, 24–29.

Hakamatsuka, T., Hashim, M.F., Ebizuka, Y. and Sankawa, U. (1991) P-450-dependent oxidative rearrangement in isoflavone biosynthesis: reconstitution of P-450 and NADPH: P-450 reductase. *Tetrahedron,* **47**, 5969–5978.

Hasemann, C.A., Ravichandran, K.G., Peterson, J.A. and Deisenhofer, J. (1994) Crystal structure and refinement of cytochrome P450terp at 2.3 Å resolution. *J. Mol. Biol.* **236**, 1169–1185.

Hashim, M.F., Hakamatsuka, T., Ebizuka, Y. and Sankawa, U. (1990) Reaction mechanism of oxidative rearrangement of flavanone in isoflavone biosynthesis. *FEBS Lett.* **271**, 219–222.

Hegnauer, R. and Grayer-Barkmeijer, R.J. (1993) Relevance of seed polysaccharides and flavonoids for the classification of the Leguminosae: a chemotaxonomic approach. *Phytochemistry,* **34**, 3–16.

Heller, W. and Forkmann, G. (1994) Biosynthesis of flavonoids. In *The Flavonoids. Advances in Research since 1986* (Harborne, J.B., ed.). London, UK: Chapman & Hall, pp. 499–535.

Helliwell, C.A., Chandler, P.M., Poole, A., Dennis, E.S. and Peacock, W.J. (2001) The CYP88A cytochrome P450, ent-kaurenoic acid oxidase, catalyzes three steps of the gibberellin biosynthesis pathway. *Proc. Natl Acad. Sci. USA*, **98**, 2065–2070.

Irmler, S., Schröder, G., St-Pierre, B., Crouch, N.P., Hotze, M., Schmidt, J., Strack, D., Matern, U. and Schröder, J. (2000) Indole alkaloid biosynthesis in Catharanthus roseus: new enzyme activities and identification of cytochrome P450 CYP72A1 as secologanin synthase. *Plant. J.*, **24**, 797–804.

Jung, W., YuO., Lau, S.M., O’Keefe, D.P., Odell, J., Fader, G. and McGonigle, B. (2000) Identification and expression of isoflavone synthase, the key enzyme for biosynthesis of isoflavones in legumes. *Nature Biotechnol.*, **18**, 208–212.

Kahn, R.A. and Durst, F. (2000) Function and evolution of plant cytochrome P450. In *Recent Advances in Phytochemistry*, Vol. 34. *Evolution of Metabolic Pathways* (Romeo, J.T., Ibrahim, R., Varin, L. and DeLuca, V., eds). Amsterdam, the Netherlands: Pergamon, pp. 151–189.

Kochs, G. and Grisebach, H. (1986) Enzymic synthesis of isoflavones. *Eur. J. Biochem.*, **155**, 311–318.

Kraulis, P.J. (1991) MOLSCRIPT: a program to produce both detailed and schematic plots of protein structure. *J. Appl. Crystallogr.*, **24**, 946–950.

Laskowski, R.A., MacArthur, M.W., Moss, D.S. and Thornton, J.M. (1993) PROCHECK: a program to check the stereochemical quality of protein structures. *J. Appl. Crystallogr.*, **26**, 283–291.

Mansuy, D. (1998) The great diversity of reactions catalyzed by cytochromes P450. *Comp. Biochem. Physiol. C*, **121**, 5–14.

Martens, S. and Forkmann, G. (1999) Cloning and expression of flavone synthase II from Gerbera hybrids. *Plant J.*, **20**, 611–618.

Martens, S., Forkmann, G., Matern, U. and Lukacin, R. (2001) Cloning of parsley flavone synthase I. *Phytochemistry*, **58**, 43–46.

Martinis, S.A., Blanke, S.R., Hager, L.P., Sligar, S.G., Hoa, G.H., Rux, J.J. and Dawson, J.H. (1996) Probing the heme iron coordination structure of pressure-induced cytochrome P420<sub>cam</sub>. *Biochemistry*, **35**, 14530–14536.

Merritt, E.A. and Bacon, D.J. (1997) RASTER3D: photorealistic molecular graphics. *Meth. Enzymol.*, **227**, 505–524.

Müller, E.J., Loida, P.J. and Sligar, S.G. (1995) Twenty-five years of P450<sub>cam</sub> research. In *Cytochrome P450: Structure, Mechanism, and Biochemistry*, 2nd edn (Ortiz de Montellano, P.R., ed.). New York, NY: Plenum Press, pp. 83–124.

Omura, T. and Sato, R. (1964) The carbon monoxide-binding pigment of liver microsomes II. Solubilization, purification, and properties. *J. Biol. Chem.*, **239**, 2379–2385.

Otani, K., Takahashi, T., Furuya, T. and Ayabe, S. (1994) Licodione synthase, a cytochrome P450 monooxygenase catalyzing 2-hydroxylation of 5-deoxyflavanone, in cultured Glycyrrhiza echinata L. cells. *Plant Physiol.*, **105**, 1427–1432.

Park, S.-Y., Shimizu, H., Adachi, S., Shiro, Y., Iizuka, T., Nakagawa, A., Tanaka, I., Shoun, H. and Hori, H. (1997) Crystallization, preliminary diffraction and electron paramagnetic resonance studies of a single crystal of cytochrome P450<sub>nor</sub>. *FEBS Lett.*, **412**, 346–350.

Podust, L.M., Poulos, T.L. and Waterman, M.R. (2001) Crystal structure of cytochrome P450 14α-sterol demethylase (CYP51) from Mycobacterium tuberculosis in complex with azole inhibitors. *Proc. Natl Acad. Sci. USA*, **98**, 3068–3073.

Poulos, T.L., Finzel, B.C., Gunsalus, I.C., Wagner, G.C. and Kraut,

J. (1985) The 2.6-Å crystal structure of Pseudomonas putida cytochrome P-450. *J. Biol. Chem.*, **260**, 16122–16130.

Poulos, T.L., Finzel, B.C. and Howard, A.J. (1987) High-resolution crystal structure of cytochrome P450<sub>cam</sub>. *J. Mol. Biol.*, **234**, 687–700.

Ravichandran, K.G., Boddupalli, S.S., Hasemann, C.A., Peterson, J.A. and Deisenhofer, J. (1993) Crystal structure of hemoprotein domain of P450BM-3, a prototype for microsomal P450s. *Science*, **261**, 731–736.

Ridderström, M., Masimirembwa, C., Trump-Kallmeyer, S., Ahlefelt, M. Otter, C. and Andersson, T.B. (2000) Arginines 97 and 108 in CYP2C9 are important determinants of the catalytic function. *Biochem. Biophys. Res. Commun.*, **270**, 983–987.

Rojas, M.C., Hedden, P., Gaskin, P. and Tudzynski, B. (2001) The P450-1 gene of Gibberella fujikuroi encodes a multifunctional enzyme in gibberellin biosynthesis. *Proc. Natl Acad. Sci. USA*, **98**, 5838–5843.

Šali, A. and Blundell, T.J. (1993) Comparative protein modelling by satisfaction of spatial restraints. *J. Mol. Biol.*, **234**, 779–815.

Schopfer, C.R., Kochs, G., Lottspeich, F. and Ebel, J. (1998) Molecular characterization and functional expression of dihydroxypterocarpan 6a-hydroxylase, an enzyme specific for pterocarpanoid phytoalexin biosynthesis in soybean (Glycine max L.). *FEBS Lett.*, **432**, 182–186.

Shimada, N., Akashi, T., Aoki, T. and Ayabe, S. (2000) Induction of isoflavonoid pathway in the model legume Lotus japonicus: molecular characterization of enzymes involved in phytoalexin biosynthesis. *Plant Sci.*, **160**, 37–47.

Stafford, H. (2000) The evolution of phenolics in plants. In *Recent Advances in Phytochemistry*, Vol. 34. *Evolution of Metabolic Pathways* (Romeo, J.T., Ibrahim, R., Varin, L. and DeLuca, V., eds). Amsterdam, the Netherlands: Pergamon, pp. 25–54.

Steele, C.L., Gijzen, M., Qutob, D. and Dixon, R.A. (1999) Molecular characterization of the enzyme catalyzing the aryl migration reaction of isoflavonoid biosynthesis in soybean. *Arch. Biochem. Biophys.*, **367**, 146–150.

Szklarz, G.D. and Halpert, J.R. (1997) Use of homology modeling in conjunction with site-directed mutagenesis for analysis of structure–function relationship of mammalian cytochrome P450. *Life Sci.*, **61**, 2507–2520.

Wachenfeldt, C.V. and Johnson, E.F. (1995) Structures of eukaryotic cytochrome P450 enzyme. In *Cytochrome P450: Structure, Mechanism, and Biochemistry*, 2nd edn (Ortiz de Montellano, P.R., ed.). New York, NY: Plenum Press, pp. 183–223.

Werck-Reichhart, D., Bak, S. and Paquette, S. (2002) Cytochromes P450. In *The Arabidopsis Book* (Somerville, C. and Meyerowitz, E., eds). Rockville, MD: American Society of Plant Biologists (http://www.aspb.org/publications/arabidopsis/).

Williams, P.A., Cosme, J., Sridhar, V., Johnson, E.F. and McRee, D.E. (2000a) Mammalian microsomal cytochrome P450 monooxygenase: structural adaptations for membrane binding and functional diversity. *Mol. Cell*, **5**, 121–131.

Williams, P.A., Cosme, J., Sridhar, V., Johnson, E.F. and McRee, D.E. (2000b) Microsomal cytochrome P450 2C5: comparison to microbial P450s and unique features. *J. Inorg. Biochem.*, **81**, 183–190.

Wong, E. (1965) Flavonoid biosynthesis in Cicer arietinum. *Biochim. Biophys. Acta.*, **111**, 358–363.

Yamaguchi, S. and Kamiya, Y. (2000) Gibberellin biosynthesis: its regulation by endogenous and environmental signals. *Plant Cell Physiol.*, **41**, 251–257.

Yano, J.K., Koo, L.S., Schuller, D.J., Li, H., Ortiz de Montellano, P.R. and Poulos, T.L. (2000) Crystal structure of a thermophilic cytochrome P450 from the archaeon Sulfolobus solfataricus. *J. Biol. Chem.*, **275**, 31086–31092.
