
# Common Origin and Evolution of Glycosyltransferases Using Dol-P-monosaccharides as Donor Substrate

**Rafael Oriol, Ivan Martinez-Duncker, Isabelle Chantret, Rosella Mollicone, and Patrice Codogno**

INSERM U504, University of Paris Sud XI, Villejuif, France

On the basis of the analysis of 64 glycosyltransferases from 14 species we propose that several successive duplications of a common ancestral gene, followed by divergent evolution, have generated the mannosyltransferases and the glucosyltransferases involved in asparagine-linked glycosylation (ALG) and phosphatidylinositol glycan anchor (PIG or GPI), which use lipid-related donor and acceptor substrates. Long and short conserved peptide motifs were found in all enzymes. Conserved and identical amino acid positions were found for the α2/6- and the α3/4-mannosyltransferases and for the α2/3-glucosyltransferases, suggesting unique ancestors for these three superfamilies. The three members of the α2-mannosyltransferase family (ALG9, PIG-B, and SMP3) and the two members of the α3-glucosyltransferase family (ALG6 and ALG8) shared 11 and 30 identical amino acid positions, respectively, suggesting that these enzymes have also originated by duplication and divergent evolution. This model predicts a common genetic origin for ALG and PIG enzymes using dolichyl-phospho-monosaccharide (Dol-P-monosaccharide) donors, which might be related to similar spatial orientation of the hydroxyl acceptors. On the basis of the multiple sequence analysis and the prediction of transmembrane topology we propose that the endoplasmic reticulum glycosyltransferases using Dol-P-monosaccharides as donor substrate have a multispan transmembrane topology with a first large luminal conserved loop containing the long motif and a small cytosolic conserved loop containing the short motif, different from the classical type II glycosyltransferases, which are anchored in the Golgi by a single transmembrane domain.

## Introduction

Asparagine-linked glycans (*N*-glycans) are engaged in a large panel of cellular and tissular functions (Varki 1993). They are involved in (1) quality control of proteins synthesized in the endoplasmic reticulum (ER) (Helenius and Aebi 2001), (2) intracellular trafficking of some glycoproteins along the secretory pathway (Hauri et al. 2000), (3) lysosomal delivery of acid hydrolases (Kornfeld 1990), (4) membrane targeting of some proteins in polarized cells (Scheiffele, Peränen, and Simons 1995), and (5) they act as recognition signals at the cell surface and participate in morphogenesis and development (Takahashi, Honda, and Nishikawa 2000).

The process of *N*-glycosylation starts in the cytosolic face of the ER by the successive assembly of two *N*-acetylglucosamines and five mannoses on the lipid carrier dolichylpyrophosphate. Then the lipid-linked heptasaccharide is translocated across the membrane to the lumen of the ER by the RFT1 flippase (Helenius et al. 2002), where consecutive additions of four residues of mannose and three glucose residues complete the oligosaccharide, which is in turn transferred en bloc from the dolichol carrier to asparagine, in the sequence Asn-X-Ser/Thr of the nascent protein chain. The *N*-glycosylation process is completed by a series of trimming and processing reactions ending in the late Golgi compartment (Kornfeld and Kornfeld 1985; Burda and Aebi 1999).

The assembly of the initial heptasaccharide on the cytosolic face of the ER is carried out by glycosyltransferases using UDP-GlcNAc or GDP-Man as donor substrates, whereas the sequential addition in the ER lumen of the following four mannoses and three glucoses is carried out by glycosyltransferases using Dol-P-Man and Dol-P-Glc as donor substrates (fig. 1). These last two donor substrates are synthesized in the cytosol from GDP-Man and UDP-Glc. The existence of specific flippases able to transport the dolichol-based donor substrates from the cytosol to the luminal side of the membrane has been proposed but they have not yet been identified (Schenk, Fernandez, and Waechter 2001), and it is possible that more than one protein contributes to these flippase activities (Anand et al. 2001).

The phosphatidylinositol glycan anchor (PIG or GPI) is a posttranslational modification frequently observed in proteins expressed in the plasma membrane (Cole and Hart 1997). GPI is synthesized in the ER membrane, where it substitutes the C-terminus of some newly synthesized proteins. The four mannoses of the PIG anchor are also added in the ER (Maeda et al. 2001). The first mannose is added by PIG-M which is an α4-mannosyltransferase, the second mannose is added by an α6-mannosyltransferase which has not yet been cloned, and the third and fourth mannoses are added by the PIG-B and SMP3 α2-mannosyltransferases, respectively. These four enzymes use the same Dol-P-Man donor substrate and make linkages similar to those of the corresponding asparagine-linked glycosylation (ALG) mannosyltransferases of the *N*-glycan series (fig. 1). In spite of these functional similarities, the ALG and PIG mannosyltransferases have always been considered separate.

---

**Abbreviations:** ALG, Asparagine-Linked Glycosylation; CAZy, Carbohydrate Active enZyme database; CDG, Congenital Disorder of Glycosylation; Dol, dolichol; ER, endoplasmic reticulum; Man, mannose; Glc, glucose; GlcNAc, *N*-acetylglucosamine; PIG, Phosphatidylinositol Glycan anchor (GPI-anchor); TMD, transmembrane domain.

**Key words:** glycosyltransferase, GPI-anchor, mannosyltransferase, *N*-glycan, transmembrane topology, phylogeny.

**Address for correspondence and reprints:** Rafael Oriol, INSERM U504, University of Paris Sud XI, 16 Avenue Paul Vaillant-Couturier, 94807 Villejuif Cedex, France. E-mail: oriol@infobiogen.fr.

*Mol. Biol. Evol.* 19(9):1451–1463. 2002

© 2002 by the Society for Molecular Biology and Evolution. ISSN: 0737-4038

1452 Oriol et al.

![Diagram](#)

FIG. 1.—Comparative structures of the oligosaccharides of ALG and PIG series. Empty symbols of ALG represent the initial heptasaccharide (GlcNAc₂Man₅) assembled in the cytosol by the glycosyltransferases using nucleotide-sugar donor substrates. Solid symbols of the ALG and PIG series are Man and Glc units added by the glycosyltransferases using Dol-P-monosaccharides as donor substrates in the lumen of the ER. The numbers inside the symbols indicate the order of sequential addition of the cytosolic GlcNAc (1–2), the cytosolic Man (1–5), the luminal Man (6–9) and the luminal Glc (1–3) of ALG, and the luminal Man (1–4) of PIG. The dashed line represents the α-linkage made by PIG-M, between the OH of C1 in the first mannose of PIG and the OH of C4 in glucosamine. This linkage has an equatorial orientation similar to that of the α1→3 linkage made by ALG3 (fig. 9). Enzymes not cloned are represented by a question mark. The same ALG9 enzyme is supposed to make both branches of N-glycans, but a different ALG9-like enzyme may exist.

arately, and no attempts had been made, in the past, to find a common evolutionary path for these two series of enzymes.

The aims of this work were (1) to find conserved peptide motifs, potential candidates for interaction with acceptor or donor substrates, (2) to find eventual relations between ALG and PIG mannosyltransferases using the same donor substrate, and (3) to see if some common characters of these enzymes could be related to structural features of the linkage to the acceptor substrates and if these could help to understand their evolutionary path.

### Materials and Methods

#### Nomenclature

We consider that the glycosyltransferases which use the same donor and acceptor substrates and make the same sugar linkage (i.e., α2-, α3-, α4- or α6-mannosyltransferases and α2- or α3-glucosyltransferases) belong to the same family and that families sharing common peptide motifs, identical amino acid positions, and similar transmembrane domain (TMD) topology can be clustered in superfamilies (i.e., α2/6-mannosyltransferases, α3/4-mannosyltransferases, or α2/3-glucosyltransferases).

#### Sequence Retrieval

* **Saccharomyces cerevisiae** or human (or both) DNA and protein glycosyltransferase sequences were retrieved from the literature. Orthologous protein sequences from other species were first searched with gapped-BLAST. The cloning of human ALG12 (AJ303120) has been reported by Chantret et al. (2002). The human EST databank was searched with rat *alg10* and with *S. cerevisiae rft1* (U15087). Contigs were built for both series of human ESTs by alignment with HUCAP (Huang 1992). Primers were designed in the EST contigs (sense 5′-CAGGAGTAGGGTTCTTG-GGCAGTGGC-3′ and antisense 5′-AATGTAATTTGAAGACCACCTGCACC-3′ for ALG10, and sense 5′-GGCATTTCCCTGGGTGTCTGAGCCTG-3′ and antisense 5′-CACAGAACTACCCATAGCTGGTCC-3′ for RFT1). The corresponding human 1,582-bp ALG10 (AJ312278) and 1,761-bp RFT1 (AJ318099) cDNA sequences were amplified from an expression library (Cailleau-Thomas et al. 2000), and they were cloned in a PCR3.1 expression vector (Eukaryotic TA cloning kit from Invitrogen), and sequenced in both strands by the dideoxy chain termination method with the T7 DNA polymerase (kit Amersham-Pharmacia-Biotech) and submitted to EMBL. The cDNA sequences of the coding sequences of *Mus musculus alg12*, *Drosophila melanogaster alg10* and *Caenorhabditis elegans pig-B* were deduced from HUCAP alignments made with more than two concordant EST overlaps per position.

#### Conserved Peptide Motifs

After retrieval of all the sequences, the best-conserved peptide patterns for each family were determined with CLUSTALW 1.8 (Thomson, Higgins, and Gibson 1994) and used in a final search with the corresponding human sequence in an iterative PHI-BLAST search with default parameters, until convergence (Altschul et al. 1997). The seed expression patterns for the PHI-BLAST were: H[KQ]EXRF[ILMV][IYFLSV][YPLV] for the α2/6-mannosyltransferase superfamily; [VL]X[YF]T[DEK][IV]D[YW]X[VIAT] for the α3/4-mannosyltransferase superfamily; W[GT]LDYPP[LF][TF]A[FYW] for the α3-glucosyltransferase family; and LADNRH[YF][TL]FY for the α2-glucosyltransferase family. A combined PHI-PSI-BLAST was also used for the α2/3-glucosyltrans-

| ENZYME (transfer) | DNA GENBANK/EBI | PROTEIN SWISSPROT ALL (SWALL) | SIZE Amino Acid | TMD | ER RETENTION SIGNAL |
|-------------------|----------------|----------------------------|---------------|-----|----------------------|
| ALG 3 (α3-mannosyltransferase) |  |  |  |  |  |
| Neurospora crassa | AF309686 | Q9C1K8 | 442 | 10 |  |
| Saccharomyces cerevisiae | X79489 | P38179 | 458 | 11 | KKLN |
| Schizosaccharomyces pombe | Z99532 | Q9Y714 | 406 | 14 |  |
| Arabidopsis thaliana | AC005309 | O82244 | 438 | 12 | KKA |
| Caenorhabditis elegans | Z83234 | Q9NAP5 | 604 | 8 | KK |
| Drosophila melanogaster | X95241 | Q27333 | 510 | 11 | KK LQ |
| Drosophila virilis | X77635 | Q24332 | 526 | 12 | KKSK |
| Homo sapiens | Y09022 | Q92685 | 438 | 11 | KKHA |
| ALG 6 (α3-glucosyltransferase) |  |  |  |  |  |
| Saccharomyces cerevisiae | U43491 | Q12001 | 544 | 13 |  |
| Schizosaccharomyces pombe | AL022019 | O43053 | 506 | 13 | KKEA |
| Arabidopsis thaliana | AB005248 | BAB09358 | 533 | 9 | KKQ1 |
| Caenorhabditis elegans | Z46676 | Q09226 | 503 | 12 | KKE |
| Drosophila melanogaster | AE003628 | Q9VKX7 | 461 | 13 |  |
| Homo sapiens | AF102851 | Q9Y672 | 507 | 10 | KKIS |
| ALG 8 (α3-glucosyltransferase) |  |  |  |  |  |
| Saccharomyces cerevisiae | X75929 | P40351 | 577 | 12 |  |
| Schizosaccharomyces pombe | Z73099 | Q10479 | 501 | 13 |  |
| Arabidopsis thaliana | AC003672 | O80505 | 383 | 11 |  |
| Caenorhabditis elegans | Z54342 | P52887 | 682 | 9 |  |
| Drosophila melanogaster | AE003438 | Q9W3V8 | 511 | 10 | KRKT K |
| Homo sapiens | AJ224875 | O60860 | 526 | 12 | KTKKQ |
| ALG 9 (α2-mannosyltransferase) |  |  |  |  |  |
| Saccharomyces cerevisiae | X96417 | P53868 | 555 | 10 |  |
| Schizosaccharomyces pombe | AL157734 | Q9P7Q9 | 577 | 9 |  |
| Arabidopsis thaliana | AC051629 | Q9FZ49 | 570 | 12 |  |
| Caenorhabditis elegans | Z49909 | P54002 | 653 | 10 | KTKKK |
| Drosophila melanogaster | AE003751 | Q9VBV8 | 623 | 10 | KTK |
| Leishmania major | AL390114 | Q9GY50 | 822 | 10 |  |
| Mus musculus | BC021791 | AAH21791 | 611 | 11 |  |
| Homo sapiens | AK025498 | Q9H6U8 | 618 | 10 | KKSGG |
| ALG 10 (α2-glucosyltransferase) |  |  |  |  |  |
| Saccharomyces cerevisiae | X87941 | P50076 | 525 | 12 |  |
| Schizosaccharomyces pombe | Z69728 | Q10254 | 445 | 12 |  |
| Arabidopsis thaliana | AL162874 | Q9LZ85 | 498 | 12 |  |
| Caenorhabditis elegans | Z81131 | O02332 | 438 | 12 |  |
| Drosophila melanogaster | AJ431376 | CAD24126 | 499 | 10 |  |
| Ratus norvegicus | U78090 | O88788 | 474 | 13 |  |
| Homo sapiens | AJ312278 | CAC41349 | 473 | 13 |  |
| ALG 12 (α6-mannosyltransferase) |  |  |  |  |  |
| Saccharomyces cerevisiae | Z71645 | P53730 | 551 | 9 |  |
| Schizosaccharomyces pombe | AL031856 | O74753 | 547 | 9 |  |
| Caenorhabditis elegans | U53155 | Q23361 | 492 | 10 |  |
| Drosophila melanogaster | AE003684 | Q9VH78 | 678 | 10 |  |
| Mus musculus | AJ429133 | CAD22101 | 483 | 12 |  |
| Homo sapiens | AJ303120 | Q9BV10 | 488 | 12 |  |
| PIG-B (α2-mannosyltransferase) |  |  |  |  |  |
| Zymomonas mobilis | AF179611 | Q9RNP1 | 509 | 10 |  |
| Saccharomyces cerevisiae | X99960 | P30777 | 616 | 10 |  |
| Schizosaccharomyces pombe | AL109957 | Q9USN0 | 506 | 9 | KKL |
| Arabidopsis thaliana | AL391149 | Q9LEQ5 | 498 | 12 |  |
| Oryza sativa | AP002908 | Q9AWW0 | 506 | 8 |  |
| Caenorhabditis elegans | AJ431373 | CAD24083 | 496 | 8 |  |
| Drosophila melanogaster | AE003479 | Q9VZM5 | 512 | 9 | KK |
| Trypanosoma brucei | AB033824 | Q9NKZ7 | 558 | 12 |  |
| Mus musculus | D84436 | Q9JJQ0 | 542 | 10 |  |
| Homo sapiens | D42138 | Q92521 | 554 | 8 | KMKF |

Table 1  
Continued  

| ENZYME (transfer) | DNA GENBANK/EBI | PROTEIN SWISSPROT ALL (SWALL) | SIZE Amino Acid | TMD | ER RETENTION SIGNAL |
|-------------------|-----------------|-------------------------------|----------------|-----|----------------------|
| PIG-M (α4-mannosyltransferase) |  |  |  |  |  |
| Saccharomyces cerevisiae....... | Z49513 | P47088 | 403 | 11 | KKNN |
| Schizosaccharomyces pombe...... | AL354632 | Q9P6R5 | 411 | 12 |  |
| Arabidopsis thaliana........... | AL589883 | Q9C575 | 417 | 11 |  |
| Caenorhabditis elegans......... | Z49907 | Q17515 | 417 | 11 | KPKTD |
| Drosophila melanogaster........ | AE003454 | Q9W2E4 | 466 | 9 |  |
| Trypanosoma brucei............. | AB050105 | BAB20994 | 431 | 11 |  |
| Mus musculus................... | AK018560 | Q9D315 | 423 | 11 | RIKYD |
| Ratus norvegicus............... | AB028126 | Q9EQY6 | 423 | 10 | RIKYD |
| Homo sapiens................... | AB028127 | Q9H3S5 | 423 | 12 | RIKYD |
| SMP3 (α2-mannosyltransferase) |  |  |  |  |  |
| Saccharomyces cerevisiae....... | X58121 | Q04174 | 516 | 8 |  |
| Schizosaccharomyces pombe...... | Z56276 | Q09837 | 533 | 10 |  |
| Drosophila melanogaster........ | AE003464 | Q9W176 | 830 | 12 |  |
| Homo sapiens................... | AK022830 | Q9H9G6 | 579 | 10 |  |

ferase superfamily with human ALG10 and the initial PHI-BLAST seed pattern: [WP][X][LI][DT][YT][PFL] PX[TFIL][AY]. Only complete sequences comprising both the long and the short conserved peptide motifs were used for this study, but the presence of numerous EST and partial sequences in other species suggests that these enzymes are widely distributed among other animals and plants and even some bacteria.

**Transmembrane Domains**

Helical TMDs were predicted by PHD-htm for each enzyme (table 1) (Rost, Fariselli, and Casadio 1996). This analysis gave a similar, but not identical multispan TMD structure for all individual sequences (ranging from 8–14 TMDs, table 1). Then, because the best computer-assisted TMD prediction methods have about 86% accuracy (Rost, Fariselli, and Casadio 1996), we searched for consensus TMD topologies. Individual TMDs predicted by PHD-htm were added to each species sequence on the CLUSTALW alignments for each enzyme, and TMDs detected at marginal significance were added or deleted, when needed, to get the best overall consensus TMD fit with the other species. Helical segments of 15–24 amino acids were considered as single helical TMDs, and longer hydrophobic helical domains of more than 25 amino acids were considered as double TMDs entering and leaving the membrane from the same side. This artificial consensus TMD prediction of all species for each enzyme was drawn, and the loop invariant amino acids were added to the figures. The high stringent criteria of total identity for each individual amino acid position was selected on purpose for visualization, but the less stringent analysis based on conserved amino acid positions at 50% leads to similar conclusions. The analysis of conserved amino acid positions and the phylogeny were derived independently from the transmembrane topology.

**Phylogeny**

Peptide sequences of mannosyltransferases and glucosyltransferases were aligned with CLUSTALW. Two strategies for selection of informative positions were used to make the trees: (1) selection of the two conserved peptide motifs of each sequence (fig. 2), followed by CLUSTALW alignment and global gap removal, which gave blocks of 43 of the original 53 sites (81%) for the 45 species of mannosyltransferases and 48 of the 57 original sites (84%) for the 17 species of glucosyltransferases; and (2) computer selection of conserved blocks with GBLOCKS (Castresana 2000) with parameters adjusted for maximum block retrieval in the CLUSTALW alignments of each of the three superfamilies, which gave 106 sites (9% of the original 1,132) for the 28 species of the α2/6-mannosyltransferase superfamily, 150 sites (25% of the original 616) for the 17 species of the α3/4-mannosyltransferase superfamily, and 137 sites (20% of the original 668) for the 17 species of the α2/3-glucosyltransferase superfamily. Both selection methods gave trees with similar topology, showing that the additional sites recruited by GBLOCKS do not bias the phylogeny. Distance matrices were generated with the observed difference of the neighbor-joining method from the PHYLO_WIN package (Galtier, Gouy, and Gautier 1996). The sequence of alg8 of C. elegans, which has a frameshift and lacks the short motif, and the sequence of alg10 of A. thaliana, which lacks half of the short motif, were excluded from the phylogeny study. Five hundred sets of data were used for bootstrap calculations.

**Results and Discussion**

**Conserved Peptide Motifs**

We found two highly conserved peptide motifs in each of the mannosyltransferases and glucosyltransferases, by protein sequence alignment with CLUSTALW.

A

| ALG12 α6-mannosyltransferase | long | short |
| --- | --- | --- |
| S.cerevisiae | 22 | PFTKVEESFNQATHDLLTYSVFDIS-QYDHLKFPGVVPRTEVG <262> | HKEWRFIHY | 334 |
| S.pombe | 28 | PFTKVEESFAMQALHDTQTYRWDLSKVYDHLEFPGAVKRSEIP <231> | HKEWRFIHY | 308 |
| C.elegans | 20 | PGTKVEESFNVOATHDEMFLPTNLT-QYDHSQFBGVVPRTEIG <240> | HKEERFIHY | 311 |
| D.melanogaster | 17 | PFTKVEESFNLOAMHDILYYLRNNFT-QYDHHDYBGVVPRTEIG <238> | HKEERFIHY | 303 |
| M.musculus | 30 | PYTKVEESFNLOATHDLL-YHQLDID-KYDHEEFPGVVPRTELG <236> | HKEERFIHY | 316 |
| H.sapiens | 29 | PYTKVEESFNLOATHDLL-YHWQDLE-QYDHLLEFPGVVPRTELG <236> | HKEERFIHY | 315 |

| ALG9 α2-mannosyltransferase | long | short |
| --- | --- | --- |
| S.cerevisiae | 27 | LITSDCDETFNYWEBPLNLLVRGFGKT-WEYSPEYSIRSWAFLLP <247> | HKEERFLYP | 320 |
| S.pombe | 33 | VITDDCDEVYNYWEBLYLLYCYGLQT-WEYSPEYAIRSWFYAL <257> | HKEERFMYP | 337 |
| A.thaliana | 65 | ITHDCDEVFNYWEBLEYILYKSGFQT-WEYSSNFALRSYLILF <236> | HKEERFLYP | 352 |
| C.elegans | 72 | IINDCDEVYNWEBLLHLFLYGEGFOT-WEYSPVYAIRSYFYTL | HKEERFLFP | 374 |
| D.melanogaster | 91 | YHADCDETFNYWEBLMINGHGLQT-WEYSQFGLRSYTLL | HKEERFLFP | 389 |
| L.major | 30 | PISDQDETFNIEPIHYLLYGSGKT-WEVCSRALRSWIFWI | HKEERFMSIP | 462 |
| M.musculus | 77 | NISDQDETFNYWEBPTHVLIIYGKGFT-WEYSPVYAIRSYAYLL | HKEERFLFP | 374 |
| H.sapiens | 77 | NISDQDETFNYWEBPTHYLIIYGEGFT-WEYSPAYAIRSYAYLL | HKEERFLFP | 374 |

| PIG-B α2-mannosyltransferase | long | short |
| --- | --- | --- |
| S.cerevisiae | 34 | TFEOADEFWQALEPAHKAFKYGELT-NEWKFGE-VRSYLFPMI <317> | HKEERFIFP | 399 |
| S.pombe | 22 | TFEPDEFYQSLEVARRHFIRXGFLT-NEWTSA--IRSALHPLI <263> | HKEIRFVYP | 333 |
| A.thaliana | 58 | TYENPDEHWQSLEVAERTLFGGYMT-NEWKRG--IRSYLHPML <216> | HKEERFVLP | 322 |
| O.sativa | 65 | TYPEPDEHWQCLEVAERRIVFGMGHLT-NEWKRG--LRSYLHPFI <228> | HKEERFIFL | 345 |
| C.elegans | 24 | SWBVPDEVYQSAEVAHHLVYGIG HLS-NEWRHS--LRSFFHPAL <240> | HKEORFLIP | 313 |
| D.melanogaster | 23 | TYWVPDEYWQSLEVAHKLTFGCYLT-NEWVOC--IRSYVYP LL <215> | HKEFRFVSP | 286 |
| T.brucei | 22 | TVEAPDEWWQSTEVAYNMVFGKHLP-NEWRYG--LRSVLFP AV <281> | HKEMRFVFV | 351 |
| M.musculus | 69 | TSFVPDEYWQSLEVAHRMVSFGYLT-NEWTER--LRCYTYPLI <237> | HKEERFYP | 354 |
| H.sapiens | 80 | TSFVPDEYWQSLEVSHHMVFNMGYLT-NEWTER--LRSYTYPLI <227> | HKEERFIYP | 365 |
| Z.mobilis | 42 | GYVHPDETFQYLEQAHLRAFGSVIP-WEYRYG--MRGGLVPVL <229> | HKEERFVLP | 320 |

| SMP3 α2-mannosyltransferase | long | short |
| --- | --- | --- |
| S.cerevisiae | 23 | SYIHDPDEHFQCIEILAMQFMKVKG TIPWEFKSKFAAARSYGPLL <254> | HOELRFLVP | 321 |
| S.pombe | 26 | SYIHDPDEHLQSFEIFANKLFGWKVELPWEFTKKPIRSVVPLNV | HOEPRFLLP | 322 |
| D.melanogaster | 37 | GYVHPDEFFQSVEVMTGBHFRLEHTRTWEFNNSLPVRSLVLP EA <328> | HOEARFLLP | 409 |
| H.sapiens | 55 | GYVHPDEFFQSPEVMAEDILGVOAARPWEFYPSSSCRSVLF PLL <287> | HOEARFLLP | 394 |

B

| PIG-M α4-mannosyltransferase | long | short |
| --- | --- | --- |
| S.cerevisiae | 27 | DANFKVRYTDIDYFVBHDAAKYYE-GKSPLYARDTYRYTPLLSW <239> | NKVCTSQYF | 313 |
| S.pombe | 53 | DRRSALKFTDDIDYFVSDASKYYSI-GMSPYMRTYRYTPMLAI <237> | NKVCTSQYF | 336 |
| A.thaliana | 40 | DAHMEVRYTDVDYIVRSDAASLMAS-GESPYKRTTYRYSPLLAL <228> | NKVHTAQYF | 319 |
| C.elegans | 37 | DYLFKVNFTDIDYHVESDAAKHVSN-GGSPFDRATYRYTPLAW <249> | NKVCTSQYF | 333 |
| D.melanogaster | 45 | DSQSAVPYTDIDYKVVLDGARQVLA-GDJPFARHTYRYSPTMAV <286> | NSVVTSQYF | 377 |
| T.brucei | 34 | DQWFRVKYTDIDYMITVDGARHMWN-GGSPEFDRTTERYTPLLA A <244> | NKVCTVQYF | 324 |
| M.musculus | 40 | DRTLLVRYTDIDYHVETDAAKFTE-GRSPYLRAITYRYTPLLS W <245> | NKVCTSQYF | 336 |
| R.norvegicus | 40 | DRTLLVRYTDIDYHVRIDAARETE-GRSPYLRAITYRYTPLLS W <250> | NKVCTSQYF | 336 |
| H.sapiens | 40 | DRTLHVRYTDIDYQVRIDAARETE-GRSEPYLRAITYRYTPLLG W <250> | NKVCTSQYF | 336 |

| ALG3 α3-mannosyltransferase | long | short |
| --- | --- | --- |
| N.crassa | 50 | LTIWKVYTEIDWAAYMEQVSQILS-GERDYTKVRGGTGPLVYP <261> | ARSLHYQFY | 362 |
| S.cerevisiae | 60 | HTKKVAYTEIDYKAYMEQLEMIO LDGMLEDYSOVSGGTGPLVYP <263> | SRSIHYQFL | 375 |
| S.pombe | 40 | ATSKVPYTEIDWIAYMEQVNSKLL-GERBYKSLVGCTGPLVYP | ARSLHYQFY | 342 |
| A.thaliana | 49 | LTEAYVPYTKIDWDAYMSQVSGLG-GERBYGNLKGDTCPLVYP | ARSLHYQFY | 359 |
| C.elegans | 37 | LVTLKVNYTEIDWS TYMQOECYVNKGILNYTOIGGDTGPCVYP <256> | SRSIHYQFY | 345 |
| D.melanogaster | 62 | VVIQRVPYTEIDWVA YMOCECGLEN-GTTNYSLLRGDTGPLVYP <315> | SRSIHYQFY | 428 |
| D.virilis | 61 | LVIORVPYTEIDWKAYMOCECGLEN-GTTNYSLLRGDTGPLVYP <321> | ARSLHYQFY | 433 |
| H.sapiens | 59 | WVTHRVAYTEIDWKA YMAREVCGVIN-GTYDYTQLQCDTGPLVYP <254> | SRSIHYQFY | 361 |

C

| ALG6 α3-glucosyltransferase | long | short |
| --- | --- | --- |
| S.cerevisiae | 67 | YGDFEAQRHWMMEITOHLPISKWY---WYDLGWCLDYPPFAYFSY <261> | FLFSFQVHEK | 380 |
| S.pombe | 56 | YGDFEAQRHWMELTHTPVSCQWY---FRDLWWCLDYPPFAYVSW | FLFSFQVHEK | 365 |
| A.thaliana | 70 | RGDFEAQRHWMETINLPVIDWY-RNGTYNDLTYWCLDYPPFAYQSY | YLFSFQVHEK | 382 |
| C.elegans | 77 | YGDYEAOQRHWMETYNLPIEQWY-LNGTHNDLLYWCLDYPPFAYHY | FLFSFHVHEK | 381 |
| D.melanogaster | 33 | HGDYEAQRHWQEITVLAGEWY-TNSSNNDLQYWCLDYPPFAYS Y | FLFSFQVHEK | 331 |
| H.sapiens | 35 | FGDYEAQRHWQEITNLPVKQWY-FNSSDNLQYWCLDYPPFAYHS L | FLFSFQVHEK | 346 |

| ALG8 α3-glucosyltransferase | long | short |
| --- | --- | --- |
| S.cerevisiae | 64 | STDFFHRNWLAITNKLPISEWY---YEHTSQWCLDYPPFAYFEW <313> | FLFCWHVHEK | 429 |
| S.pombe | 27 | STDFEVRNWLAITHSLPISEWY---KSSISEWCLDYPPFAYMEC | FLFCWHVHEK | 364 |
| A.thaliana | 32 | STDFEVRNWLAITNSLPITKWY---FDETSQWCLDYPPFAYFER | FLRCWHVHEK | 327 |
| C.elegans | 25 | STDFEVRNWMAWTWORPLCEWY---TEATS EWCLDYPPFAYREL | frameshift | ? |
| D.melanogaster | 25 | STDFEVRNWLAITHSLPINQWY---VDATS EWCLDYPPFAYFEW | FVFCWHVHEK | 355 |
| H.sapiens | 34 | STDFEVRNWLAITHSLPISQWY---YEATS EWCLDYPPFAYWEEY | FMFCWHVHEK | 383 |

| ALG10 α2-glucosyltransferase | long | short |
| --- | --- | --- |
| S.cerevisiae | 64 | TTNIYPYEFIDEKEHVGQTLTYL--KGKWTQDPKITTPPGIYELGL <248> | LADNRHYTFY | 364 |
| S.pombe | 25 | IQYYVPNPLYDEIEHTAGAORFC---RKDWBDPAITTTPPGLYLV SV | LADNRHYLFY | 328 |
| A.thaliana | 23 | VNHIYPEPYMDLEEHVPOAQOYC---NGNFRSWDPMITPPGLYYS L | ----HYPFY | 379 |
| C.elegans | 41 | VYHYYPEPYMDLEFHQTORSY---SGNYSWELITTPPAIYYSM | LADNRHETFY | 324 |
| D.melanogaster | 23 | RVNGTSDYVDEEHTHPQGLAC---RKEFDWDPKITTPPGCLYLA | LADNRHYTFY | 332 |
| R.norvegicus | 29 | FSRALEPYMDEIFHLPQAORYCEGRF SLSQWDPMITTLPGLYLVS | LADNRHYTFY | 358 |
| H.sapiens | 29 | FSRALEPYMDEIFHLPQAORYCEGHF SLSQWDPMITTLPGLYLVS | LADNRHYTFY | 357 |

FIG. 2.—Long and short conserved peptide motifs of the α2/6-mannosyltransferase superfamily (A) the α3/4-mannosyltransferase superfamily (B), and the α2/3-glucosyltransferase superfamily (C). Black letters on a gray background correspond to amino acid positions conserved at 50% in one enzyme, and white letters on a black background are positions conserved at 50% in two or more enzymes. The size, the relative positions of the long and the short conserved motifs, and the inter motif distance <> are well conserved. For the evaluation of conserved amino acid positions, ST, KR, DENQ, VILM, and WFY were considered equivalent.

The long motif (41–44 amino acids) is located close to the amino terminus and the short motif (9–10 amino acids) is located in the carboxy terminal side of the enzymes. The distance between these two motifs is relatively well conserved in the three superfamilies (median 252, range 215–381) (fig. 2A–C).

### The α2- and α6-Mannosyltransferase Superfamily Using Dol-P-Man as Donor Substrate

The α6-mannosyltransferases (ALG12) (Burda et al. 1999) are clustered in the same glycosyltransferase-22 family of the Carbohydrate Active enZyme database (CAZy) (http://afmb.cnrs-mrs.fr/~pedro/CAZY/db.html) together with three α2-mannosyltransferases, one belonging to the same asparagine-linked glycosylation series (ALG9) and two enzymes involved in the synthesis of the PIG anchor (PIG-B and SMP3) (Ferguson 1992). These four enzymes were known to have a short common peptide pattern: H[KQ]EXRF (Canivenc-Gansel et al. 1998), which is part of the short conserved peptide motif shown in figure 2A. After five iterative runs with the human ALG12 protein sequence, the PHI-BLAST converges and retrieves the 28 enzymes of the α2/6-mannosyltransferase superfamily, including all the ALG12, ALG9, PIG-B, and SMP3 enzymes (table 1).

The alignment of the sequences of the homologous ALG12 enzymes from four different species (*S. cerevisiae*, *Schizosaccharomyces pombe*, *D. melanogaster*, and *C. elegans*) showed another highly conserved peptide pattern (TKVEESF) in the long conserved peptide motif of ALG12 (fig. 2A), which helped us to clone the mouse and the human *ALG12* genes. Unlike the short motif, which is highly conserved in the four enzymes of this superfamily, the long motif has few positions conserved in all four families, but it has a large proportion of amino acid positions conserved among the α6-mannosyltransferases on one side and among the three α2-mannosyltransferase families on the other side, suggesting the existence of two conserved long peptide motifs, one specific for the α6-mannosyltransferases (ALG12) and another specific for the α2-mannosyltransferases (ALG9, PIG-B and SMP3) (fig. 2A). The phylogenetic tree of all mannosyltransferases supports the idea that the three α2-mannosyltransferases have a common genetic origin and are more closely related among themselves than to the α6-mannosyltransferase family (fig. 3).

The enzymes of this α2/6-mannosyltransferase superfamily have a consensus multispan TMD structure with an average of 12 spans. The long motif is flanked by TMD-1 and TMD-2, whereas the short motif is flanked by TMD-10 and TMD-11 (fig. 4). PIG-B (Takahashi et al. 1996) and ALG9 (Burda et al. 1996) glycosyltransferases add an α2-mannose to a sterically similar mannose acceptor (fig. 1), and they share eleven identical amino acid positions (solid and gray circles in fig. 4). Three of the eight ALG9 and three of the ten PIG-B enzymes have an ER retention signal (table 1) (Teasdale and Jackson 1996), suggesting that their COOH terminus is in the cytosol. The SMP3 enzymes

FIG. 3.—Neighbor-joining phylogenetic tree of mannosyltransferases (based on the 43 selected sites of the long and the short motifs). ALG12 are α6-mannosyltransferases. ALG9, PIG-B and SMP3 are α2-mannosyltransferases (α2-mt). ALG3 are α3-mannosyltransferases and PIG-M are α4-mannosyltransferases. Bootstrap values from 500 data sets are indicated at the divergence points. The scale bar represents the number of substitutions per site for a unit branch length.

make a further elongation of a fourth mannose (Grimme et al. 2001) also linked in α1→2 in PIG (fig. 1). They share the same eleven identical amino acids with the two other α2-mannosyltransferases (fig. 4), but none of the four known SMP3 enzymes has ER retention signals (table 1).

It is interesting to note that a large proportion of the conserved amino acids in the α2-mannosyltransferases are located in the long loop between the first two TMDs, where we found the ALG12-specific highly conserved pattern (TKVEESF). In addition, the four identical amino acids, which are specific of the α2-mannosyltransferase activity (gray circles in fig. 4), are also in this first loop, suggesting that it might be involved in α2- and α6-mannosyltransferase active sites.

Seven of the eleven amino acids, identical among α2-mannosyltransferases, are also shared by the α6-mannosyltransferases (solid circles, fig. 4). This might be related to the fact that the hydroxyl groups on C2 and C6 of the α-D-mannose acceptor are on the same side of the molecule, and they may offer some similarities of acceptor surface to the corresponding α2- and α6-mannosyltransferases (fig. 5), although they are not very close in space. We have previously found a similar phenomenon between α2-fucosyltransferases and α6-fucosyltransferases which share three conserved peptide motifs (Breton, Oriol, and Imberty 1998; Chazalet et al. 2001). In addition, conserved peptide motifs were also found between α3-fucosyltransferases and α4-fucosyltransferases (Oriol et al. 1999). This last observation

Phylogeny of ALG and PIG Glycosyltransferases 1457

ALG12  
α6-mannosyltransferase  

ALG9  
α2-mannosyltransferase  

SMP3  
α2-mannosyltransferase  

PIG-B  
α2-mannosyltransferase  

FIG. 4.—Consensus TMD topology of the α6-mannosyltransferases (six ALG12) and the α2-mannosyltransferases (eighth ALG9, ten PIG- B, and four SMP3). Vertical empty rectangles are helical TMDs across the ER membrane bilayer (gray horizontal band). Empty circles with black letters are identical amino acid positions in all members within each of the four series of enzymes. Gray solid circles with black letters represent the four identical amino acids in the three enzymes of the α2-mannosyltransferase family, and solid circles with white letters are the seven amino acids identical in the four series of enzymes constituting the α2/6-mannosyltransferase superfamily. The boxed KK represents a putative ER retention signal (table 1). The solid star in TMD-4 of ALG12 identifies the inactivating mutation F142→V (AJ290427) of human ALG12, inducing the CDG of type Ig (Chantret et al. 2002). C, cytosol; L, lumen of the ER.

prompted us to ascertain whether the α3- and the α4- mannosyltransferases using Dol-P-Man as donor substrate have also conserved common peptide motifs.

The α3- and α4-Mannosyltransferase Superfamily Using Dol-P-Man as Donor Substrate

After three iterative runs with the corresponding pattern and either of the human ALG3 (CAZy glycosyltransferase-58 family) or PIG-M (CAZy glycosyltransferase-50 family), the PHI-BLAST reaches convergence and retrieves the 17 enzymes of the α3/4-mannosyltransferase superfamily (table 1). Several conserved amino acid positions were found for both the long and the short motifs of the α3/4-mannosyltransferase superfamily (fig. 2B). The phylogenetic tree of all mannosyltransferases show that the ALG3 family and the PIG-M family constitute separate clusters (fig. 3) and are different enough from the cluster of α2/6-mannosyltransferases to be considered as an outgroup defining the position of a possible root for the tree between the two superfamilies of α2/6- and α3/4-mannosyltransferases.

The TMD prediction also suggests a multispan structure with an average of 12 spans, with the long motif in the first loop and the short motif between TMD-10 and TMD-11 (fig. 6). Seven identical amino acid positions are shared by the α3- and the α4-mannosyltransferases (solid circles in fig. 6). Five of these seven iden-

1458 Oriol et al.

PIG-M  
α4-mannosyltransferase

ALG3  
α3-mannosyltransferase

FIG. 7.—Neighbor-joining phylogenetic tree of glucosyltransferases (based on the 160 computer selected sites with GBLOCKS). ALG6 and ALG8 are α3-glucosyltransferases and ALG10 are α2-glucosyltransferases (α2-glu-tr). Bootstrap values from 500 data sets are indicated at the divergence points. The scale bar represents the number of substitutions per site for a unit branch length.

FIG. 6.—Consensus TMD topology of the α4-mannosyltransferases (nine PIG-M, top) and the α3-mannosyltransferases (eight ALG3, bottom). Empty circles with black letters indicate the identical amino acids among all members of each of the two families of enzymes. Solid circles with white letters represent the seven amino acids identical in the two enzymes of the α3/4-mannosyltransferase superfamily. The boxed KK indicates a putative ER retention signal (table 1). The solid star in the second loop of ALG3 represents the inactivating mutation G118→N of the human ALG3 enzyme inducing the CDG of type Id (Körner et al. 1999; Sharma, Knauer, and Lehle 2001). C, cytosol; L, lumen of the ER.

torial orientation and offer a similar equatorial approach to the corresponding α3-mannosyltransferases (ALG3) and α4-mannosyltransferases (PIG-M), especially if the glucosamine is rotated 180° around its horizontal axis (fig. 5). Consequently, we expected more common features between α3- and α4-mannosyltransferases, which share the same equatorial approach to the acceptor and the same donor and less common features between the two superfamilies of α2/6- and α3/4-mannosyltransferases because they only share the donor substrate (Dol-P-Man).

The α2 and α3-Glucosyltransferase Superfamily Using Dol-P-Glc as Donor Substrate

After two iterative runs, the PHI-BLAST with either of human ALG6 or ALG8 converges and retrieves the 12 sequences of the α3-glucosyltransferase family, comprising the ALG6 and the ALG8 enzymes. These two enzymes are closely related as shown in the phylogenetic tree of figure 7. Their close similarity is also illustrated by 30 identical amino acid positions among ALG6 and ALG8 (fig. 8). Four out of the six ALG6 and two out of the six ALG8 enzymes (Stagljar, Hessen, and Aebi 1994; Stanchi et al. 2001) have a putative ER retention signal (table 1), suggesting that their COOH-end is in the cytosol, as predicted by the consensus TMD topology (fig. 8).

Either of PHI-BLAST or PSI-BLAST alone with ALG10 retrieved only the ALG10 enzymes of the different species, but the combined PHI-PSI-BLAST with human ALG10 retrieves ALG10, ALG8, and ALG6 enzymes of the α2/3-glucosyltransferase superfamily. Few

Phylogeny of ALG and PIG Glycosyltransferases 1459

![Diagram](#)

FIG. 8.—Consensus TMD topology of the α3-glucosyltransferases (six ALG6 and five ALG8) and the α2-glucosyltransferases (six ALG10), constituting the α2/3-glucosyltransferase superfamily. Empty circles with black letters indicate identical amino acid positions among all members within each of the three enzymes. Gray solid circles with black letters represent 27 amino acids identical in both α3-glucosyltransferases (ALG6 and ALG8), and solid circles with white letters are the five ALG10 identical amino acids with ALG6 or ALG8 (or both). The solid stars identify inactivating mutations in human ALG6: S170→I, TMD-4 (de Lonlay et al. 2001); deletion of I 299, TMD-7 (Hanefeld et al. 2000); F304→S, TMD-7 (Imbach et al. 2000); A333→V, TMD-8 (Imbach et al. 1999); deletion of L 444, TMD-11 (de Lonlay et al. 2001); and S478→P, TMD-12 (Imbach et al. 2000), which can induce the CDG of type Ic (Grünewald et al. 2000). C, cytosol; L, lumen of the ER.

similarities exist between ALG10 and the α3-glucosyltransferases, but the short and long motifs are well identified, among all the different species for ALG10 (gray and black areas of the ALG10 family in fig. 2C). The phylogenetic tree confirms this impression, showing a cluster of ALG10 enzymes at high distance from the two α3-glucosyltransferases, ALG6 and ALG8 (fig. 7). The *alg10* gene was first described in yeast (Burda and Aebi 1998), then in rat (Hoshi et al. 1998), and we have cloned the *Drosophila* (AJ431376) and the human (AJ312278) ALG10 orthologous genes. None of the ALG10 enzymes has ER retention signals, but all of them have three identical amino acid positions with ALG6 and ALG8, plus a glutamic acid (E) identical with ALG6 in the long motif, and a histidine (H) identical with ALG8 in the short motif (solid circles fig. 8).

The transmembrane topology of the last three glucosyltransferases suggests a consensus TMD organization similar to the aforementioned TMD structure of mannosyltransferases with about 12 TMD. A large conserved luminal loop between TMD-1 and TMD-2 and a small conserved loop, in the cytosolic side of the membrane, between TMD-8 and TMD-9 are present (fig. 8).

The scheme of figure 9 illustrates identical equatorial orientations of the hydroxyls on C3 of the last α-D-mannose and the first α-D-glucose, which are acceptors for ALG6 and ALG8, respectively. Unlike this, a different spatial orientation can be seen for the hydroxyl on C2 of the second α-D-glucose, which is the acceptor

![Diagram](#)

FIG. 9.—Structure of the α-D-mannose (top), acceptor for the first glucose (added by ALG6). The first α-D-glucose (bottom), acceptor for the second glucose (added by ALG8) or the second α-D-glucose, acceptor for the third glucose (added by ALG10). The shaded oval areas correspond to the OH on C3 of the mannose and the first glucose. The empty oval area shows the OH on C2 of the second glucose. Identical equatorial orientation is illustrated by the two shaded areas, in spite of being on different monosaccharides (mannose and glucose), whereas different orientations are shown by the two oval shaded areas on the left side and the oval empty area below the glucose.

substrate for ALG10. Therefore, we expect more common features between ALG6 and ALG8, which share both acceptor and donor substrates, than between the α2-glucosyltransferase (ALG10) and the two α3-glucosyltransferases (ALG6 and ALG8) because they have different acceptor substrates and they only share the donor substrate (Dol-P-Glc). Comparison of the OH on C3 (acceptors for ALG6 and ALG8) and the OH on C2 (acceptor for ALG10, fig. 9) with the OH on C3, C4, C2, and C6 (acceptors for mannosyltransferases, fig. 5), suggests that the acceptor OH of the two families of glucosyltransferases are as different from each other as the corresponding acceptors for the two mannosyltransferase superfamilies.

### Potential Functions of TMD and Conserved Loops

The transmembrane topology of the glycosyltransferases, using Dol-P-monosaccharides as donor substrate, suggests a common consensus multispan transmembrane topology. The enzymes have an average of 12 putative helical TMD, and both the NH₂ and COOH-ends are in the cytosolic side of the membrane (figs. 4, 6, and 8). But they still are a statistical prediction, and we cannot be certain that all the putative TMDs span the membrane in vivo, for example only 7 of the 11 TMDs predicted for the protein O-mannosyltransferase (Pmt1p) were shown to span the membrane (Straahl-Bolsinger and Scheinost 1999). However, if some of the putative α-helical TMDs do not span the membrane, the second most probable topology with both extremes in the cytosol would be a 10 TMD, and if there is an odd number of TMD, the NH₂ and the COOH terminus would be on different sides of the membrane. None of these possibilities can be formally ruled out for individual sequences because the consensus topology is only an average artificial prediction. Nevertheless, we can say that all these enzymes have a multispan TMD topology, similar to the general structure of sugar transport proteins such as glucose-6-P-transporter (G6PT) (Pan, Lin, and Chou 1999) or glucose transporter 1 (Glut1) (Sato and Mueckler 1999). In fact, the great majority of sugar transporters have a multispan structure with 10 or 12 TMDs (Maiden et al. 1987; Marger and Saier 1993). It is possible that each of the enzymes under study contributes also to the translocation across the membrane of the Dol-P-monosaccharide, which is synthesized in the cytosol and has to reach the acceptor substrate oligosaccharides in the lumen of the ER (Rush and Wachter 1998; Rush et al. 1998).

A similar distribution of the putative TMD along the peptide chain is an additional structural feature in favor of a common genetic origin for a series of related enzymes. Our consensus TMD topology suggests that the first, large and highly conserved loop, is on the luminal side of the membrane and contains identical amino acid positions specific for each linkage. These properties are compatible with the contribution of this first loop to the glycosyltransferase active site.

For the sake of clarity, only identical amino acid positions located in the extramembrane loops are shown in figures 4, 6, and 8, but between 30% and 42% of the total amino acids of these enzymes is located inside the putative helical TMD. The comparative analysis of identical amino acid positions in the loops and in the helical TMD shows that they are not randomly distributed. In general, there are fewer identical amino acid positions inside the TMD (8%–35%) than in the loops (65%–92%), and the proportion of identical amino acids in TMDs are characteristic and different for the α2/6-mannosyltransferase (8%–19%), the α3/4-mannosyltransferase (21%–27%) and the α2/3-glucosyltransferase (35%), in favor of a common origin for each of these three superfamilies of glycosyltransferases.

The mutations of these ER glycosyltransferases, known to be responsible of congenital disorders of glycosylation (CDG), are also not randomly distributed. Only one out of eight mutations is in a short loop and seven are in TMDs (solid stars in figs. 4, 6, and 8). A similar difference in favor of a TMD location has been reported for the mutations inactivating the Golgi transporter of CMP-sialic acid (Eckhardt, Gotza, and Gerardy-Schahn 1999), and this protein is also a candidate for establishing a new CDG of type II (Mollicone et al., personal communication).

### Divergent Evolutionary Model

All together the common structural features of the enzymes using Dol-P-monosaccharides as donor substrate suggest that they might have followed a common evolutionary path. The duplication of a hypothetical ancestral gene, encoding for a protein with the long and short conserved peptide motifs and the consensus multispan TMD topology, followed by divergent evolution, might have generated the two main mannosyltransferase and glucosyltransferase genes (fig. 10). A second duplication within the mannosyltransferases might have originated the α2/6- and the α3/4-mannosyltransferase superfamilies. A duplication of the α2/6-mannosyltransferase gene may be at the origin of the α6-mannosyltransferase genes (ALG12) and the ancestor of the α2-mannosyltransferase genes. Two further duplications of the α2-mannosyltransferase branch may be at the origin of SMP3, PIG-B, and ALG9 genes. A duplication of the gene of the α3/4-mannosyltransferase superfamily may have originated ALG3 and PIG-M genes. Similar duplication events may have originated the α2-glucosyltransferase genes (ALG10) and the ancestor of α3-glucosyltransferase genes in the glucosyltransferase branch. Finally, a last and more recent duplication must be at the origin of the present two enzymes with α3-glucosyltransferase activity (ALG6 and ALG8) because they display the highest score of sequence identity (30 amino acids, fig. 8).

The compilation of the data reported previously and the identical and conserved amino acid positions in each of the superfamilies, families, and enzymes, constitute indirect evidence for a common genetic origin followed by divergent evolution of these enzymes. The existence of the more ancestral precursor genes (the two empty circles in fig. 10) is supported by the relative

Phylogeny of ALG and PIG Glycosyltransferases 1461

![Diagram](attachment:diagram.png)

FIG. 10.—Divergent evolutionary model, based on the consensus TMD topology (figs. 4, 6, and 8), phylogeny (figs. 3 and 7), and conserved peptide motifs (fig. 2) of all the glycosyltransferases using Dol-P-monosaccharides as donor substrate. The model indicates the duplication events needed to obtain the six α-mannosyltransferase genes (ALG12, ALG9, PIG-B, SMP3, ALG3, and PIG-M) and the three α-glucosyltransferase genes (ALG6, ALG8, and ALG10) from a hypothetical common ancestral gene having the consensus multispan TMD topology. The three superfamilies of α2/6-mannosyltransferases (solid circles), α3/4-mannosyltransferases (dark gray circles), and α2/3-glycosyltransferase (light gray circles) are supported by the existence of an increasing number of identical and conserved amino acids, at each divergence point, from left to right, and a conserved proportion of identical amino acids in TMD, within each of the three enzyme superfamilies. The empty circles represent hypothetical ancestors based on the similar size and location of the conserved peptide motifs, the multispan TMD topology, and their common use of Dol-P-monosaccharides as donor substrates. This model suggests that the glycosyltransferases of the $N$-glycan series (ALG) and of the PIG anchor series (PIG-B, PIG-M, and SMP3), using Dol-P-monosaccharides as donor substrates, have a common genetic origin.

positions and sizes of the long and the short conserved peptide motifs, but no identical or conserved amino acid positions were found between mannosyltransferases and glucosyltransferases, nor between the α2/6 and the α3/4-superfamilies of mannosyltransferases. The relative positions of the duplications reflect the increasing proportion of identical and conserved amino acids from left to right. All the enzymes on the right side of the tree have been cloned and experimentally tested, whereas the existence of a single common ancestor on the left is only a working hypothesis. However, if the single common ancestor hypothesis is not confirmed, a very similar evolutionary model with two non-related ancestors, one for the α-mannosyltransferases and another for the α-glucosyltransferases, or three ancestors, one for each superfamily, can be imagined. Irrespective of considering one, two, or three ancestors, the tree depicted in figure 10 must be quite ancient because these nine glycosyltransferases have been found in plants, yeasts, worms, insects, and vertebrates.

### Acknowledgments

The research was partially supported by the Association for Research on Cancer (ARC) grant 5348, the French network for Recombinant Glycosyltransferases (GT-rec), and the French INSERM/AFM network 4MR29F for CDG. We are grateful to Stuart E. H. Moore for critical reading of the manuscript and helpful discussions.

#### LITERATURE CITED

- ALTSCHUL, S., F. STEPHEN, T. L. MADDEN, A. A. SCHAFFER, J. ZHANG, Z. ZHANG, W. MILLER, and D. J. LIPMAN. 1997.
- Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acid Res. 25:3389–3402.
- ANAND, M., J. S. RUSH, S. RAY, M. A. DOUCEY, J. WEIK, F. E. WARE, J. HOFSTEENGE, C. J. WAECHTER, and M. A. LEHRMAN. 2001. Requirement of the Lec35 gene for all known classes of monosaccharide-P-Dolichol-dependent glycosyltransferase reactions in mammals. Mol. Biol. Cell 12:487–501.
- BRETON, C., R. ORIOL, and A. IMBERTY. 1998. Conserved structural features in eukaryotic and prokaryotic fucosyltransferases. Glycobiology 8:87–94.
- BURDA, P., and M. AEBI. 1998. The ALG10 locus of Saccharomyces cerevisiae encodes the α-1,2 glucosyltransferase of the endoplasmic reticulum: the terminal glucose of the lipid-linked oligosaccharide is required for efficient $N$-linked glycosylation. Glycobiology 8:455–462.
- _______. 1999. The dolichol pathway of $N$-linked glycosylation. Biochim. Biophys. Acta 1426:239–257.
- BURDA, P., S. T. HEESSEN, A. BRACHAT, A. WACH, A. DUSTERHOFT, and M. AEBI. 1996. Stepwise assembly of the lipid-linked oligosaccharide in the endoplasmic reticulum of Saccharomyces cerevisiae: identification of the ALG9 gene encoding a putative mannosyl transferase. Proc. Natl. Acad. Sci. USA 93:7160–7165.
- BURDA, P., C. A. JAKOB, J. BEINHAUER, J. H. HEGEMANN, and M. AEBI. 1999. Orderly assembly of the asymmetrically branched lipid-linked oligosaccharide in the endoplasmic reticulum is ensured by the substrate specificity of the individual glycosyltransferases. Glycobiology 9:617–625.
- CAILLEAU-THOMAS, A., P. COULLIN, J. J. CANDELIER, L. BALANZINO, B. MENNESSON, R. ORIOL, and R. MOLLI CONE. 2000. FUT4 and FUT9 genes are expressed early in human embryogenesis. Glycobiology 10:789–802.
- CANIVENC-GANSEL, E., I. IMHOF, F. REGGIORI, P. BURDA, A. CONZELMANN, and A. BENACHOUR. 1998. GPI anchor bio-

1462 Oriol et al.

synthesis in yeast: phosphoethanolamine is attached to the α1,4-linked mannose of the complete precursor glycosphingolipid. Glycobiology 8:761–770.

CASTRESANA, J. 2000. Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Mol. Biol. Evol. 17:540–552.

CHANTRETT, I., T. DUPRE, C. DELEND A et al. (15 co-authors). 2002. Congenital disorder of glycosylation type Ig is defined by a deficiency in dolichyl-P-mannose: Man7GlcNAc2-PP-dolichyl mannosyltransferase. J. Biol. Chem. 277: 25815–25822.

CHAZALET, V., K. UEHARA, R. A. GEREMIA, and C. BRETON. 2001. Identification of essential amino acids in the Azorhizobium caulinodans fucosyltransferase NodZ. J. Bacteriol. 183:7067–7075.

COLE R. N., and G. W. HART. 1997. Glycosyl-phosphatidylinositol anchors: structure, biosynthesis and function. Pp. 69–88 in J. MONTREUIL et al., ed. Glycoproteins II. Elsevier, Amsterdam.

DE LONLAY, P., N. SETA, S. BARROT et al. (22 co-authors). 2001. A broad spectrum of clinical presentations in congenital disorders of glycosylation I: a series of 26 cases. J. Med. Genet. 38:14–19.

ECKHARDT, M., B. GOTZA, and R. GERARDY-SCHAHN. 1999. Membrane topology of the mammalian CMP-sialic acid transporter. J. Biol. Chem. 274:8779–8787.

FERGUSON, M. A. J. 1992. Glycosyl-phosphatidylinositol membrane anchors: the tale of a tail. Biochem. Soc. Trans. 20: 243–256.

GALTIER, N., M. GOUY, and C. GAUTIER. 1996. Two graphic tools for sequence alignment and molecular phylogeny. Comput. Appl. Biosci. 12:543–548.

GRIMME, S. J., B. A. WESTFALL, J. M. WIEDMAN, C. H. TARON, and P. ORLEAN. 2001. The essential smp3 protein is required for addition of the side branching fourth mannose during assembly of yeast glycosylphosphatidylinositols. J. Biol. Chem. 276:27731–27739.

GRÜNEWALD, S., T. IMBACH, K. HUIJBE et al. (14 co-authors). 2000. Clinical and biochemical characteristics of congenital disorder of glycosylation type Ic, the first recognized endoplasmic reticulum defect in N-glycan synthesis. Ann. Neurol. 47:776–781.

HANEFELD, F., C. KÖRNER, U. HOLZBACH-ERBERLE, and K. VON FIGURA. 2000. Congenital disorder of glycosylation-Ic: case report and genetic defect. Neuropediatrics 31:57–59.

HAURI, H. P., C. APPENZELLER, F. KUHN, and O. NUFER. 2000. Lectins and traffic in the secretory pathway. FEBS Lett. 476:32–37.

HELENIUS, A., and M. AEBI. 2001. Intracellular functions of N-linked glycans. Science 291:2364–2369.

HELENIUS, J., D. T. W. NG, C. L. MAROLDA, P. WALTER, M. A. VALVANO, and M. AEBI. 2002. Translocation of lipid-linked oligosaccharides across the ER membrane requires Rft1 protein. Nature 415:447–450.

HOSHI, N., H. TAKAHASHI, M. SHAHIDULLAH, S. YOKOYAMA, and H. HIGASHIDA. 1998. KCR1, a membrane protein that facilitates functional expression of non-inactivating K⁺ currents associates with rat EAG voltage-dependent K⁺ channels. J. Biol. Chem. 273:23080–23085.

HUANG, X. 1992. A contig assembly program based on sensitive detection of fragment overlaps. Genomics 14:18–25.

IMBACH, T., P. BURDA, P. KUHNER, R. A. WEVERS, M. AEBI, and E. G. BERGER. 1999. A mutation in the human ortholog of the Saccharomyces cerevisiae ALG6 gene causes carbohydrate-deficient glycoprotein syndrome type-Ic. Proc. Natl. Acad. Sci. USA 96:6982–6987.

IMBACH, T., S. GRÜNEWALD, B. SCHENK et al. (12 co-authors). 2000. Multi-allelic origin of congenital disorder of glycosylation (CDG)-Ic. Hum. Genet. 106:538–545.

KÖRNER, C., R. KNAUER, U. STEPHANI, T. MARQUARD, L. LEHLE, and K. VON FIGURA. 1999. Carbohydrate deficient glycoprotein syndrome type IV: deficiency of dolichol-P-Man: Man₅GlcNAc₂-PP-dolichyl mannosyltransferase. EMBO J. 18:6816–6822.

KORNFELD, S. 1990. Lysosomal enzyme targeting. Biochem. Soc. Trans. 18:367–374.

KORNFELD, R., and S. KORNFELD. 1985. Assembly of asparagine-linked oligosaccharides. Annu. Rev. Biochem. 54:631–664.

MAEDA, Y., R. WATANABE, C. L. HARRIS, Y. HONG, K. OHISHI, K. KINOSHITA, and T. KINOSHITA. 2001. PIG-M transfers the first mannose to glycosylphosphatidylinositol on the luminal side of the ER. EMBO J. 20:250–261.

MAIDEN, M. C., E. O. DAVIS, S. A. BALDWIN, D. C. M. N. MOORE, and P. J. HENDERSON. 1987. Mammalian and bacterial sugar transport proteins are homologous. Nature 325: 641–643.

MARGER, M. D., and M. H. SAIER. 1993. A major superfamily of transmembrane facilitators that catalyse uniport, symport and antiport. Trends Biochem. Sci. 18:248–249.

ORIOL, R., R. MOLLICONE, A. CAILLEAU, L. BALANZINO, and C. BRETON. 1999. Divergent evolution of fucosyltransferase genes from vertebrates, invertebrates, and bacteria. Glycobiology 9:323–334.

PAN, C. J., B. LIN, and J. Y. CHOU. 1999. Transmembrane topology of human glucose-6-phosphate transporter. J. Biol. Chem. 274:13865–13869.

ROST, B., P. FARISELLI, and R. CASADIO. 1996. Topology prediction for helical transmembrane proteins at 86% accuracy. Protein Sci. 7:1704–1718.

RUSH, J. S., K. VAN LEYEN, O. OUERFELLI, B. WOLUCKA, and C. J. WAECHTER. 1998. Transbilayer movement of Glc-P-dolichol and its function as a glucosyl donor: protein-mediated transport of a water-soluble analog into sealed ER vesicles from pig brain. Glycobiology 8:1195–1205.

RUSH, J. S., and C. J. WAECHTER. 1998. Topological studies on the enzymes catalyzing the biosynthesis of Glc-P-dolichol and the triglucosyl cap of Glc3Man9GlcNAc2-P-P-dolichol in microsomal vesicles from pig brain: use of the processing glucosidase I/II as latency markers. Glycobiology 8:1207–1213.

SATO, M., and M. MUECKLER. 1999. A conserved amino acid motif (R-X-G-R-R) in the Glut1 glucose transporter is an important determinant of membrane topology. J. Biol. Chem. 274:24721–24725.

SCHEIFFELE, P., J. PERÄNEN, and K. SIMONS. 1995. N-glycans as apical sorting signals in epithelial cells. Nature 378:96–98.

SCHENK, B., F. FERNANDEZ, and C. WAECHTER. 2001. The ins(ide) and outs(ide) of dolichyl phosphate biosynthesis and recycling in the endoplasmic reticulum. Glycobiology 11:61R–70R.

SHARMA, C. B., R. KNAUER, and L. LEHLE. 2001. Biosynthesis of lipid-linked oligosaccharides in yeast: the ALG3 gene encodes the Dol-P-Man:Man5GlcNAc2-PP-Dol mannosyltransferase. Biol. Chem. 382:321–328.

STAGLJAR, I., S. T. HESSEN, and M. AEBI. 1994. New phenotype of mutations deficient in glycosylation of the lipid-linked oligosaccharide: cloning of the ALG8 locus. Proc. Natl. Acad. Sci. USA 91:5977–5981.

STANCHI, F., E. BERTOCCO, S. TOPPO, R. DIOGUARDI, B. SIMIONATI, N. CANNATA, R. ZIMBELLO, G. LANFRANCHI, and G. VALLE. 2001. Characterization of 16 novel human genes

showing high similarity to yeast sequences. Yeast 18:69–80.

STRAAHL-BOLSINGER, S., and A. SCHEINOST. 1999. Transmembrane topology of Pmt1p, a member of an evolutionarily conserved family of protein O-mannosyltransferases. J. Biol. Chem. 274:9068–9075.

TAKAHASHI, M., N. INOUE, K. OHISHI, Y. MAEDA, N. NAKAMURA, Y. ENDO, T. FUJITA, J. TAKEDA, and T. KINOSHITA. 1996. PIG-B, a membrane protein of the endoplasmic reticulum with a large lumenal domain is involved in transferring the third mannose of the GPI anchor. EMBO J. 15:4254–4261.

TAKAHASHI, T., R. HONDA, and Y. NISHIKAWA. 2000. Cloning of the human cDNA which can complement the defect of the yeast mannosyltransferase I-deficient mutant alg1. Glycobiology 10:321–327.

TEASDALE, R. D., and M. R. JACKSON. 1996. Signal-mediated sorting of membrane proteins between the endoplasmic reticulum and the Golgi apparatus. Annu. Rev. Cell Develop. Biol. 12:27–54.

THOMSON, J. D., D. G. HIGGINS, and T. J. GIBSON. 1994. CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, positions-specific gap penalties and weight matrix choice. Nucleic Acid Res. 22:4673–4680.

VARKI, A. 1993. Biological roles of oligosaccharides, all of the theories are correct. Glycobiology 3:97–130.

PIERRE CAPY, reviewing editor

Accepted April 17, 2002
