
CMLS, Cell. Mol. Life Sci. 59 (2002) 1223–1232  
1420-682X/02/071223-10 $ 1.50 + 0.20/0  
© Birkhäuser Verlag, Basel, 2002  

---

**Research Article**

**The 2.4-Å crystal structure of the penicillin-resistant penicillin-binding protein PBP5fm from *Enterococcus faecium* in complex with benzylpenicillin**

E. Sauvage${}^{a,*}$, F. Kerff${}^{a}$, E. Fonzé${}^{a}$, R. Herman${}^{a}$, B. Schoot${}^{b,d}$, J.-P. Marquette${}^{b}$, Y. Taburet${}^{b}$, D. Prevost${}^{b}$, J. Dumas${}^{b}$, G. Leonard${}^{c}$, P. Stefanic${}^{a}$, J. Coyette${}^{a}$ and P. Charlier${}^{a}$

${}^{a}$ Centre d’Ingénierie des Protéines, Université de Liège, Institut de Physique B5 and Institut de Chimie B6, Sart Tilman, 4000 Liège (Belgium), Fax +32 4 366 3667, e-mail: eric.sauvage@ulg.ac.be  
${}^{b}$ Aventis Pharma, 102, route de Noisy, 93235 Romainville Cedex (France)  
${}^{c}$ Macromolecular Crystallography, European Synchrotron Radiation Facility, BP 220, 38043 Grenoble Cedex (France)  
${}^{d}$ Present address: Analyse et Biologie Structurale, Galderma Research and Development, 635, route des Lucioles, 06902 Sophia Antipolis (France)

Reveived 30 April 2002; received after revision 6 June 2002; accepted 6 June 2002

---

**Abstract.** Penicillin-binding proteins (PBPs) are membrane proteins involved in the final stages of peptidoglycan synthesis and represent the targets of β-lactam antibiotics. Enterococci are naturally resistant to these antibiotics because they produce a PBP, named PBP5fm in *Enterococcus faecium*, with low-level affinity for β-lactams. We report here the crystal structure of the acyl-enzyme complex of PBP5fm with benzylpenicillin at a resolution of 2.4 Å. A characteristic of the active site, which distinguishes PBP5fm from other PBPs of known structure, is the topology of the loop 451–465 defining the left edge of the cavity. The residue Arg464, involved in a salt bridge with the residue Asp481, confers a greater rigidity to the PBP5fm active site. In addition, the presence of the Val465 residue, which points into the active site, reducing its accessibility, could account for the low affinity of PBP5fm for β-lactam. This loop is common to PBPs of low affinity, such as PBP2a from *Staphylococcus aureus* and PBP3 from *Bacillus subtilis*. Moreover, the insertion of a serine after residue 466 in the most resistant strains underlines even more the determining role of this loop in the recognition of the substrates.

**Key words.** Penicillin-binding protein; PBP5fm; *Enterococcus faecium*; resistance; benzylpenicillin; peptidoglycan synthesis.

Penicillin-binding proteins (PBPs) are essential membrane enzymes functioning at the late stages of peptidoglycan assembly. They are members of the penicilloyl serine transferase family, a family of enzymes that catalyze the transfer of the penicilloyl moiety of penicillin to their active site serine [1]. PBPs represent the lethal targets of penicillins and more generally of β-lactam antibiotics in susceptible bacteria. β-Lactams exhibit structural analogy with the PBP transpeptidation substrate, the D-alanyl-D-alanine carboxy terminus of peptidoglycan peptides. They acylate the penicillin-binding domain of PBPs, forming a rather stable acyl-enzyme devoid of transpeptidase activity.

PBPs fall into two major groups: the low-molecular-mass (LMM) PBPs are monofunctional enzymes acting mainly as DD-peptidases [2], while the high-molecular-mass (HMM) PBPs are multimodular enzymes also displaying a DD-peptidase activity sometimes associated with a

* Corresponding author.

transglycosylase activity. HMM-PBPs are further divided into classes A and B according to their function and the sequence similarities found in their non-penicillin-binding module [3, 4]. The penicillin-binding module of the multimodular PBPs is associated with at least one other domain which is assumed to have a transglycosylase activity in class A HMM-PBPs. The role of the non-penicillin-binding domain of class B HMM-PBPs is not yet understood. Transglycosylase activity has not been detected in this class of PBPs and the non-penicillin-binding module has been proposed to interact with other proteins to form multiprotein complexes [5–7]. Enterococci, which are involved in many nosocomial infections, are naturally resistant to β-lactam antibiotics. Furthermore, they have become increasingly resistant to many antibiotics and are thus responsible for an increasing number of therapeutic failures [8–10]. *Enterococcus faecium* is the most penicillin-resistant enterococcal species, with MICs ranging from 0.5 to more than 64 μg/ml for clinical isolates. MICs as high as 256 and 512 μg/ml have been found in very resistant strains [11, 12]. Some highly β-lactam-resistant *E. faecium* strains are also vancomycin-resistant [12–14], although vancomycin and β-lactam antibiotics can act in a synergistic way against vancomycin-resistant enterococci [15]. Since ampicillin resistance and vancomycin-vanB-type resistance have recently been shown to be cotransferable within a large mobile element [14–16], this double resistance has become a serious public health problem.

The natural resistance of enterococci to β-lactam antibiotics is due to the presence of one PBP (named PBP5fm in *E. faecium*) with low-level affinity for penicillin. Resistance to β-lactam in *E. faecium* may be enhanced by higher levels of PBP5fm production. However, overproduction is not the only path to resistance within resistant strains of *E. faecium*. Point mutations in the PBP5fm further reduce the affinity of the protein for β-lactams, leading to very high levels of resistance [13, 17]. PBP5fm is thought to be able to take over the role of the other HMM-PBPs in peptidoglycan synthesis when these proteins are inhibited by an antibiotic [10, 11, 18, 19]. In *E. faecalis*, the peptidoglycan synthesized by PBP5 in the presence of benzylpenicillin is slightly different from that produced by the usual PBPs in the absence of benzylpenicillin. The main differences are the lack of oligomers higher than tetramers and a reduction in the number of tetramers in the muropeptide composition of the peptidoglycan, suggesting that PBP5 is unable to use trimeric peptides as donors or acceptors in transpeptidase reactions to form oligomeric cross-linked muropeptides. Nevertheless, under these conditions, PBP5 still cross-links the unmodified peptidoglycan pentapeptide precursors [20]. In contrast, strains showing vancomycin resistance are now assumed to synthesize a pentapeptide with an abnormal carboxyterminal unable to bind vancomycin. The synergic effect between vancomycin and β-lactam can be explained by the inability of the PBP5fm to cross-link the modified pentapeptide [15, 21].

The PBP5fm crystal structure is the first structure of a low-level-affinity PBP. PBP5fm serves as a paradigm of the B1 subgroup of class B HMM-PBPs (B1-PBPs) that reassembles PBPs from enterococci (*E. hirae* PBP5, *E. faecalis* PBP5 or PBP4), bacilli (*Bacillus subtilis* PBP3, *B. halodurans* PBP3, *B. cereus* PBP3), staphylococci (*Staphylococcus aureus* PBP2a, *S. sciuri* PBP2, *S. epidermis* PBP2) and *Clostridium acetobutylicum* PBP2. PBP5fm also shares sequence similarities with a protein from the *Listeria innocua* genome (comparison computation with PBP5fm sequence was performed at the SIB using the BLAST network service [22]). The B1 subgroup enzymes are assumed to be responsible for the β-lactam resistance of their respective bacterial species. Structural information on class B HMM-PBPs has been made available through the crystallographic structure of PBP2x from *Streptococcus pneumoniae* [23]. PBP2x belongs to the B4 subgroup of class B HMM-PBPs and is sensitive to benzylpenicillin and other β-lactam antibiotics. Typical MICs of benzylpenicillin for non-resistant *S. pneumoniae* strains are 0.02 μg/ml (strain R6) whereas values of 4 μg/ml (strain BM4107) are characteristic of *E. faecium* strains. While high resistance observed in some *S. pneumoniae* clinical isolates rests on a PBP2x harboring many mutations [24], resistance to β-lactams is a constitutive element of *E. faecium* and a few point mutations in PBP5fm can lead to very high levels of resistance.

## Materials and methods

### Construction and expression of recombinant PBP5fm molecules

DNA encoding PBP5fm was amplified by PCR using pDML517 [12] as template and primers designed to introduce *NcoI* and *NdeI* cloning sites, to replace the hydrophobic anchoring peptide and putative polar region by a Met-Gly peptide in positions 32 and 33 and to introduce a Leu-Glu dipeptide at the C terminal end of the protein. The PCR product was first cloned into a pUC18 derivative generating pDML536 and subcloned in the pET28a expression vector generating pDML537. The recombinant protein was overproduced at 25°C in the cytoplasm of *Escherichia coli* BL21 (DE3) cells that were grown in medium containing kanamycin (50 mg/l) by a fed-batch process [25]. To obtain a selenium-labeled PBP5fm derivative, the pDML537 insert was subcloned into *NcoI* and *EcoRI* restriction sites of the pTRC99a expression vector. The labeled recombinant protein was overproduced at 33°C in the cytoplasm of methionine-auxotrophic *E. coli* β180 cells that were grown in a medium containing Se-methionine (200 mg/l) [26].

Purification and crystallization

IPTG-induced cells were sonicated (15 min⁻¹) and centrifuged to discard cell debris. Both PBP5fm forms were purified by anion exchange (Q-sepharose HP and Q-sepharose Hitrap), hydroxyapatite (ceramic-hydroxyapatite type I) and gel filtration (Superdex 200 pf) chromatography to reach an approximate 95% purity. Several crystals were obtained from the purified native protein and its selenium-labeled derivative, at various protein concentrations ranging from 5 to 90 mg/ml. All crystals were grown by the sitting-drop vapor diffusion method. For the crystal leading to the structure determination, the purified seleno-methionine-labeled PBP5fm protein was concentrated and cocrystallized with benzylpenicillin by mixing at the ratio 1:5. The crystal was grown at 20°C by mixing equal volumes of PBP5fm (15 mg/ml), 10% PEG8000 (w/v), 0.4 M Li₂SO₄ and 100 mM Na acetate at pH 4.5. The crystal appeared within 1 week and reached its maximum size (0.300 × 0.15 × 0.10 mm) in about 2 weeks.

Data collection, phase determination and structure refinement

The crystal was cryoprotected in the same buffer containing 20% (v/v) glycerol and flash-cooled at 100 K for data collection at the ID14-eh4 beamline (European Synchrotron Radiation Facility-ESRF, Grenoble) using an ADSC Quantum 4RCCD detector. MAD data were collected at three different wavelengths (Se-peak, inflection, remote). At each wavelength, 540 frames were collected with an exposure time of 0.5 s and an oscillation of one-third of a degree. The three data sets were complete up to 2.4 Å (statistics are given in table 1). The

Table 1. Data collection and refinement statistics.

| Data set       | Peak     | Inflection | Remote   |
|----------------|----------|------------|----------|
| Wavelength (Å) | 0.9789   | 0.9793     | 0.9393   |
| Highest resolution (Å) | 2.4        | 2.4      | 2.4     |
| Unique reflections (total) | 89,322    | 89,274   | 91,206  |
|                 | (323,708)| (321,438) | (334,158)|
| Completeness (%) * | 96.6 (82.2) | 98.4 (81.9) | 97.4 (95.2) |
| R<sub>merge</sub> (%) * | 5.6 (18.6) | 5.3 (16.9) | 5.8 (15.1) |
| I/σ(I)         | 9.2      | 8.8        | 7.6      |

Refinement

| Resolution range (Å) | 20–2.4 |
|----------------------|--------|
| Number of protein residues | 1710  |
| Number of water molecules | 1164  |
| R<sub>cryst</sub>/R<sub>free</sub> (%) | 22.7/28.6 |
| R.m.s. deviations |
| Bond lengths (Å) | 0.008  |
| Bond angles (°)  | 1.50   |

* Statistics for the highest-resolution shell (2.55–2.4 Å) are given in parentheses.

---

crystal belongs to the space group C2 (a = 79.3 Å, b = 128.8 Å, c = 236.1 Å and β = 93.92°) with three molecules in the asymmetric unit. Integration was performed with MOSFLM [27] and scaling with the program SCALA from the CCP4 program suite [28]. PBP5fm is a 648-amino-acid protein with 13 methionines. Thirty out of the 39 possible selenium sites were localized using SHAKE-AND-BAKE [29] and coordinates further refined by use of the MLPHARE program from the CCP4 suite. The multiwavelength anomalous diffraction method (MAD) map calculated after density modification with DM [30] was suitable for initiating the protein construction. The major part of the C-terminal and the N-terminal domain (without residues 193–256) were built in the MAD map using TURBO-FRODO [31]. Simulating annealing with CNS [32] and manual rebuilding allowed completion of the model except residues 32–38, 193–256 and 623–632. The three monomers were refined individually. The Ramachandran plot analysis revealed that 84.4% of the non-proline, non-glycine residues are in the most favored region, and 14.7% are in the additionally allowed region.

Results

Structure of PBP5fm

The crystal structure of PBP5fm was solved at 2.4 Å by MAD using data collected from one crystal of the Se-Met-substituted protein cocrystallized in the presence of benzylpenicillin. The space group is C2 with three molecules in the asymmetric unit. The structure was refined against the data set collected at the remote wavelength with an R-factor of 22.7% and an R<sub>free</sub> of 28.6%. The final model contains 570 residues per monomer and one benzylpenicillin molecule bound to the active-site serine of each monomer. PBP5fm presents two clear distinct domains (fig. 1), the N- and the C-terminal domain, the latter being the transpeptidase domain that shows weak penicillin-binding activity.

The N-terminal domain exhibits a new type of folding. The N-terminal helix begins at residue 39, residues 33–38 being invisible in the electron density map. The region up to residue 162 is globular and composed of two helices (α<sub>N1</sub> and α<sub>N2</sub>) and a three-stranded antiparallel β sheet (β<sub>N1</sub> to β<sub>N3</sub>). A long β sheet connects this globular domain to the C-terminal domain. It is comprised of the strands β<sub>N5</sub>, β<sub>N10</sub> and β<sub>N11</sub> that form a three-stranded antiparallel β sheet on the N-terminal side and becomes a two-stranded parallel β sheet (segments 170–177 and 337–344) on the C-terminal side. In the central region (residues 190–260), the electron density maps only suggest some secondary-structure elements that were not included in the final model. Its conformation must differ from the equivalent PBP2x region since there would be a

situation that also prevails in *E. coli* PBP3 [5] and *S. aureus* PBP2a [34]. Transglycosylase activity has not been demonstrated in PBP5fm and no clear biological function could be attributed to the N-terminal domain, which could, according to the most widely accepted assumptions, interact with other proteins to form a multiprotein complex responsible for the synthesis of the cell wall peptidoglycan.

The C-terminal domain shows structural similarities with the transpeptidase domain of the penicillin-sensitive PBP2x and more generally with the penicillin-recognizing enzymes (β-lactamases of classes A, C or D [35], *E. coli* PBP5 [36], and the *Streptomyces* R61 [37] and K15 [38] DD-peptidases). The active-site serine 422 is located between two subdomains, one a five-stranded β sheet covered by helices, the other being all α helices. The Ser422–Lys425 segment at the beginning of helix α<sub>C2</sub> (the secondary-structural elements defined for class A β-lactamases will be used with subscript C to describe the transpeptidase domain) and the Ser480-Asp-Asn and Lys618-Thr-Gly-Thr motifs define the active-site cleft as in most active serine PBPs. The loop at the bottom of the cavity contains residues Gly537-Tyr-Gly-Gln-Gly and adopts the same conformation as in PBP2x. Downstream, the β<sub>C3</sub> strand that bears the Lys618-Thr-Gly motif, the β<sub>C3</sub>-β<sub>C4</sub> connecting loop (residues 623–632), is completely invisible in the electron density map of the three monomers. In PBP2x, this loop has three additional residues and defines one side of a long groove that stretches on both parts of the active site. The other side of the groove is delimited by residues 461–465 which cover the Ser480-Asp-Asn motif. In PBP5fm, the groove is not as deep as in PBP2x and the residues bordering the active-site entry are different in both structures, maybe reflecting the specificity of the natural substrate of each enzyme.

The main structural difference between PBP5fm and the β-lactamases or the LMM-PBPs is the insertion of three loops extending in the direction of the N-terminal domain. One of these loops (388–410) is close to the poorly defined region 190–260 and a double hydrogen bond is formed between the Asn388 side chain and the Tyr265 backbone.

Figure 1. Overall structure of PBP5fm. (A) Ribbon representation of the tertiary structure of PBP5fm. The C-terminal domain is the transpeptidase domain, while the function of the N-terminal domain is unknown. (B) Ribbon representation of the central region of PBP5fm. The backbone trace of residues 192–257 has been tentatively determined in the monomer A. (C) Stereoview of the Cα trace of the C-terminal domain (346–680). (D) Stereoview of the Cα trace of the N-terminal domain (39–346).

steric clash between this region and a symmetrical protein in the crystal packing. An attempt to determine the backbone trace in monomer A is shown in figure 1B. The N-terminal domain has been shown to be important for the folding of the PBP5 enzyme from *E. hirae*, the sequence of which is nearly identical to that of PBP5fm [33]. The deletion of some segments in the N-terminal domain resulted in proteins unable to bind penicillin,

### Structure of class B HMM-PBPs

Both PBP5fm and PBP2x belong to the class B HMM-PBPs. All the members of this class share five common sequence motifs not directly involved in the enzymatic activity (motifs I–V: Arg173-Gly-Xaa-Xaa-Asp-Arg-Asn-Gly, Arg263-Xaa-Tyr-Pro-Xaa-Gly, Gly299-Xaa-Xaa-Gly-Xaa-Glu, Gly340-Xaa-Asp-Xaa-Xaa-Xaa-Thr-Xaa-Asp-Xaa-Xaa-Xaa-Glu and Thr373-Gly-Asp (Glu)-Xaa-Leu-Ala-Xaa-Xaa-Xaa-Xaa-Pro-Ser-Xaa-Asp) [3]. These motifs are located at the interface between the N- and C-terminal domains (fig. 2). Motifs I and IV con-

CMLS, Cell. Mol. Life Sci. Vol. 59, 2002 Research Article 1227

E.faecium PBP5
MKASDKHGKNRTGAYI AG- -AVL L VAASGGYFYYRHYQETQAVEAGEKTVEQFVQAL NKGDYNKAAAGMASKKAANKS

E.faecalis PBP5
MERSNARNKKSSKKPLI LGVSALVL AAAGGGYYAYSQWAKOELAEAKKTATTFLNVL SKQEFDKLPSVVQEASLKKN

L.innocua PBP
MANLNGKGNNKKAII I G-SI VAVVVLAGI AI YFFIQNQHKDE-KNALAAAETFTSNIAKEKYDKLGNSYSSESLKKV

B.subtilis PBP3
- - - - - -MLKKCI LLVFLCVGLI GLI G- - - - -CSKTDSPEDRMEAFVKQWNDQQFDDMYQSLTKD- - VKK

S.aureus PBP2a
- - - - - -MKKI KIVPLTLI VVVVGFGL - - - - - -IYFYASKDKEINNTIDAI EDKNFKQVYK- - DSSYISKS

S.pneumoniae PBP2x

αN2
βN1
βN2
βN3
βN4
80
100
110
130
140
150
160
ALSEKEILEKYONI YGAADVKGLEI SNLK-VDKKDDST--YSFSYKAKMNTSLGELKDL S--YKGTLDANDGKTTI NWOPNLVFPEMEGNDK
GYDTKSVVEKYQAI YSGIQAEGVKASDVQ-VKKAKDNO--YFTTYKLSMSTPLGEMKDLS--YQSSI AKKGDTYQI AWKPSLIFPOMSGNDK
EFTKKEEMEEKYQAVYDGIGAKDIKVKNLKSVYDDKENK--FNLTYELEMATSLGKLATQK--YKTTISKQDDDWKI DWKPAIIFPGMVKTDK
ESKKDFVRNYKAI YEQAGVKNLKVTADEVDKDDQDNKTMKHIPYKVSMNTNAGKVSFKNTAVLKLEKTDDDEESWNIDWDPSFI FKQLADDKT
DNGEVEMTERPI KI YNSLGVKDI NI QDAKKKVSKNKK--RVDAQYKIKTNYGNIDANV---QNFVKEDEGMWKLDWDHSVIIPGMQKDQS

βN5
βN6
170
180
190
200
210
220
230
240
250
VSLTTQEAIRGNI LDRANGEPLATTGKLKQLGVVP5KLGDGDEKTANI KAI ASAFDLTEDAIN---QAI5SQWWPDYFVPLKIIDGATPELP-
ISIQVDNAKRGEI VDRANGSLAI NKVFDEVGVVPGKLGSGAEKTANI KAFSDKFGVSDEIN---QKLSQGWQADSFVPI TVASEPVTELP-
VRMTEDDAERRGOI VDRANGKPLATGQFAEAGVPSKLGEDEKVKNIA DI SKKLEVESTDYI N---KQLDQKWQADSFVPLVTLNKD KLPEA-
VQIMSERPKRRGQI YDKNGKGLAVNTDVPEIGIVPEGELG--DKKEKVI KELAKKLDLTEDDI K---KKLDQGWKDDS FVPLKKVKPQQEKLV S
IHIE NLKSERGKILDANNVELANTGTHMRLGI VPKNVS----KKDYKAI AKELSISEDYI NNKWI KIQYRMI PSHFKTVKKMDEYLSDFAK
TVPAAKRGTIYDRANGVPI AEDAT

βN7
αN3
βN8
αN4
βN9
αN5
βN10
260
270
280
290
300
310
320
330
--- ---AGATI QEVDGRYYPLGEAAAAQLI GYVGDI TAEDI DKNEP--LSSNGKI GRSGLE MAFDKDLAGTTGGKLSI T---DTDGVEKKVLI EH
--- ---TQAAATKDTES RYYPLGEAAAAQLI GYTGTI TAEDI EKNEP--LSTGVI GTKGLE RAFDKELRGQDGGS LVIL---DDKENVKKALQTK
EATSLQGVTRTNVSS RYYPYGEKTAHLTGYVRAI TAEELKKKKEQTYS DTSNI GIAGLENVYEDKL RGTTGWKI YVP---QTQ---EVIAEK
K--- ---FHLLTTNETES RNYPLGKATS HLLGYVGPI NSEELKQKEYKQYKDDAIV GKKGLELYDKKLQHEDGYRVTI VRVDDNSNTI AHTLI EK
NRSYPNGQFASSFI GLAQLH KSLL GTSQMES SSLNSI LAQT DG

βN11
αC1
βC1
βC2
αC1a
αC1b
αC1c
340
350
360
370
380
390
400
410
420
EVQNGKDI KLTIDAKAQKTAFDSLGGKAGSTVATTPKTGDLLALA SS PSYDPNKMTNGI SOEDYKAYEENPEORBI SRFATGYAPGSTFKMIT
EKKDGQTI KLTIDS GVQQQAAFI FDKRPGSAVITDPQKGDLLATVSSPSYDPNKMANGI SQEKYDAYNNNKDLPF TARFATGYAPGSTFKTIT
DKKDGEEI KLTIDADVQKKAFDSLGS ETGAVTMINPTNGELLALVSTPSYDANOMVLGI TSEDYAKYNDDKRLPFLARYANRAPGSTFKTIT
KAKDGEDL HLTIDI KTOMKLYDELKDDSGAAVALOPKTGETLALVSAPSYDPNGFI FGWSDKEWKCLKNDKNNFPSAKFNKTYAPGSTIKPIA
KKKGKDI QLTIDA KVOKSI YNNMKNDYGSGTAI HPQTELLALVSTPSYDVYPFMYGMSNEEYNKL TEDKKEPLLNKFQITTSPGSTQKILLT
RTMDGKDVYTTISSPLQSFMETQM MTATLVSAKTGEILLATTQRPTFDA D

αC2
αC4
αC5
430
450
460
470
480
490
500
510
520
AAI GLDNGTI DPNEVLTI NGLKWQKDS SWGSYQVTRV SD- VS HVDLKTALI YSDI YMAQETLKMGEEKNFRAGLDKF FGEDLDLPISMNP AQ
GAI GLDAGTL KPDEELEI NGLKWQKDS WGGYFATRV KE- ASPVNLRTALVN SDNI YFAOQTLRMGEDKFRA GLNKFI FGEELDLP IAMTPAQ
ATI GLDTGI TKPDKVREI SGLQWQKDS WGKYFVTRV HD- VPKVNMTOALVH SDNI YFAQEGL EIGDKKLTAGLK KFD FDK EYNLPTMKPAQ
AAI GLNGTL KADEKKTI KGKEWQKDS SWGYSVTRV SERLQOQVLENALI SDNI YFAQNAL DMGADTFTKGLKTFGFSEDVPYEFFI OKSS
AMIGLNNKTL DDKTSYKI DGKGWQKDS WGGYNVTRV EVVNGNI DLKQAI ES SDNI FFARVALELGSKKFEKG MKKLG VGEDI PSDYPFYNAQ
LAAAAIDNNTFP GGEVFNS GRMMTFSQGFAHSS VGMTLLEEQKMGAATWL DYLNRA

αC6
αC8
αC9
αC10
530
540
550
560
570
580
590
600
610
SNEESFNSDILLADTG YGOGE LLI NPIOAAMS VFANNGT L VYPKLA-DKE TKD- -KKNVIGETA VQTIVPDLREVVOVNGTAHSL SAL
SNEODKFNSFILLADT GYGOQ EL LI SPIOQATMSV FQNNGTLVYPKLVL-DKE TTK--KDNVILSANAA NTI ATDLLGSVEDPSGYVYNMYNP
SN-DGLNSFILLADT SYGOQ ELLMSP IQQAIAYSAI ANDGKMPYPKLDT-KEKAGK--ETOATEAASANQVKAALVKTVSDPAGTAHALQI O
AN-DKLDSIILLADT GYGOQ QMQMSPLHLATA YTPFVDNGDLVKPTLK KDOSQTADVWHKQVVTKEGAA DI TKGLKG VVEDE RGSA YQP VVK
SN-KNLDNEILLADS GYGOQE ILINPVQILSI YSALENNGNINA PHLLK-DTKNK-VWK KNISKE NI NLLNDGMQQVVN--KTHKEDI YRS
NIVNI AQSSFGOQ-1.SVTQTQMI RAFTA IANDGVMLEPKFIS OKEI --VGNPVSKDAA SLTRTNMVLVGTDP

βC3
βC4
βC5
αC11
620
630
640
650
660
670
GI PLAAKTGTAEI KE-KQDEKGKENSFLFAFNPDNQGYMMVSMLENKE DDSATKRAPELLQYL NQNYQ- - - - - - - -
NFS LAAKTGTAEI KD-KQDTDGKENSFLLTLDRSNNKFLTMI MVENGENGSA TDISKPLI DYLEATIK- - - - - - - -
GHNI AAKTGAELKE-KOGEDGLENGFVYA FDADNPNYLMVOMI ENVKGRGGSGL VI DKLKPVIESMYK- - - - - - - -
GI TVAGKTGTAELKTSDKDKGTENGWFVGYDYENKDLLVAMMI QNVQDRGGSHYVEAKKQFQSN- - - - - - - -
YANLI GKSGTAELKM-KQGETGROI GWFI SYDKDNPNMMM IVNKDVQDKGMASYNAKI SGKVYDEL YENG NKKYDI DE
VALKSGTA YIFS AVSMP PDFILYVTVQOPE FANPILERAS AMK


Figure 2. Structure of class B high-molecular mass PBPs. (A) Sequence alignment of five representatives of subgroup B1 of class B HMM-PBPs and partial alignment with Streptococcus pneumoniae PBP2x that belongs to subgroup B4. The following sequences are available from GenBank (accession numbers in parentheses): Enterococcus faecium PBP5 (CAA59287), Enterococcus faecalis PBP5 (CAB89865), Listeria innocua PBP (CAC95693), Bacillus subtilis PBP3 (CAB12221), Staphylococcus aureus PBP2a (CAA68684), Streptococcus pneumoniae PBP2x (CAA34412). PBP2x alignment is based on a structural comparison with PBP5fm. Only PBP2x residues with structural analogs in PBP5fm are shown. Active-site catalytic residues of PBPs are colored red, conserved structural motifs of class B-PBPs are colored yellow and B1-PBPs conserved boxes near the active site are colored green. The loops in magenta define a groove surrounding the conserved structural motifs of class B-PBPs. The secondary structures of the transpeptidase domain are labeled following their usual name in class A b-lactamases. Color scheme is conserved for figures 2 and 4. (B) Conserved structural motifs in the class B HMM-PBPs. PBP5fm is shown in gray and PBP2x is shown in blue. The conserved structural motifs of class B HMM-PBPs are colored yellow and the loops defining the groove around the conserved motifs are colored magenta.

stitute the two parallel β strands of the ‘degenerated’ β sheet described above, while motif V is the β<sub>2C</sub> strand defining the C-terminal domain interface. Motifs II and III are small loops connecting strand β<sub>N7</sub> to helix α<sub>N3</sub> and strand β<sub>N9</sub> to strand β<sub>N10</sub>, respectively. The N-terminal globular region (39–162) is connected to motif I through strand β<sub>N5</sub>. The region between motifs I and II (190–260) is not seen in the density map. The region between motifs II and III (313–332) defines the central β sheet and a little loop constituted by residues 279–297 is connected to motifs III and IV. These four regions are very different in sequence and in length in PBP2x, whereas all these elements are of similar length and exhibit partially conserved amino acid sequences in all B1-PBPs.

Motifs I–IV may be schematically represented as the four fingers of a right hand pointing into the palm of a left hand. The thumb of this left hand is the fifth motif and the palm is defined by the three loops (loop A: 388–410, loop B: 505–519 and loop C: 561–581) belonging to the C-terminal transpeptidase domain. The three loops do not exist in class A β-lactamases or in LMM-PBPs. They are characteristic of class B HMM-PBPs even if they differ in length and amino acid sequences. The 561–581 loop C is more elongated in PBP2x than in PBP5fm, while the 388–410 loop A is shorter. Some strong interactions maintain these fingers together (Arg179-Asp375, Arg263-Glu304) and anchor them into the transpeptidase module (Arg173-Glu304). Whatever the structures of the N-non-penicillin-binding and C-transpeptidase-penicillin-binding terminal domains of class B HMM-PBPs, they are held together by a tight interface structure of about 150 Å<sup>2</sup>, defined by the conserved motifs I–V.

**Interaction with penicillin**

The electron density of the PBP5fm active site unambiguously reveals a covalent acyl-enzyme with benzylpenicillin (fig. 3A). The thiazolidine ring and the carboxylate are well defined and the electron density is continuous between the active site serine hydroxyl group and the β-lactam carbonyl group. The side chain is easily traced in the electron density of monomer A but is less well defined in the other monomers. The benzylpenicillin conformation (fig. 3B) is similar to the one observed in the acyl-enzyme formed by benzylpenicillin and the Glu166-Asn inactive mutant of TEM1 class A β-lactamase [39]. The carbonyl oxygen lies in the oxyanion hole defined by the Ser422 and Thr618 main chain amine groups and the carboxylate is at hydrogen bond distance from Thr618 Oγ and Thr620 Oγ. The amide group of the benzylpenicillin side chain is wedged between Asn482 and the β<sub>C3</sub> strand. In class A and class D β-lactamases, the carboxylate interacts with the guanidinium group of an arginine that can occupy different positions in the primary structure of these enzymes. The presence of such a

Figure 3. Benzylpenicillin binding in PBP5fm. (A) Stereo view of an F<sub>obs</sub>−F<sub>calc</sub> electron density omit map calculated without the benzylpenicillin molecule, contoured at 2.5 σ. (B) Stereoview of the interactions between the benzylpenicillin and the PBP5fm active-site residues. Benzylpenicillin is colored magenta. Hydrogen-bonded interactions are shown as thin black lines.

guanidinium group in β-lactamases is favorable to acylation by β-lactams [40].

Acylation of PBP5fm is very slow (second-order rate constant k<sub>2</sub>/K = 20 M<sup>−1</sup> s<sup>−1</sup>) [12]. It can proceed according to a two-step model where Lys425 abstracts a proton from the active-site serine hydroxyl group to promote the nucleophilic attack of the β-lactam carbonyl by the activated serine Oγ [39, 41]. The proton is subsequently back donated by Lys425 to the β-lactam nitrogen via Ser480. An alternative interpretation of the acylation mechanism relies upon a concerted one-step process where covalent bond formation between penicillin and the active-site serine Ser422 is concomitant with the transfer of the Ser422 proton directly to Ser480 and to the transfer of the Ser480 hydroxyl hydrogen to the β-lactam nitrogen [42]. Both charged Lys425 and Lys617 support the correct orientation of the Ser480 hydroxyl group. This explanation could prevail for all penicilloyl serine transferases including enzymes possessing a tyrosine residue instead of a serine in the second conserved active-site motif, such as class C β-lactamases. Both interpretations are in accordance with the distances between the active-site residues observed in PBP5fm.

As for most PBPs, the deacylation step is extremely slow compared to the efficiency of the deacylation step in class A β-lactamases. In the latter, a glutamic acid residue in

position 166 (bottom of the active site) is responsible for the high deacylation rate, by activating a conserved water molecule that attacks the acyl-enzyme carbonyl [43]. Though a water molecule is present in the PBP5fm active site at a position equivalent to the hydrolytic water molecule of class A β-lactamases, there is no counterpart to Glu166 that could be relevant to activate this molecule.

### Low-level affinity

A low-level affinity for β-lactams is the main characteristic of the B1-PBPs. The second-order rate constant k₂/K for the acylation of PBP2a from *S. aureus* by penicillin V is about 16 M⁻¹ s⁻¹ [44]. The corresponding values for PBP5fm from *E. faecium* strains vary from 15 to 24 M⁻¹ s⁻¹ and they are at least ten times lower for PBP5fm from highly resistant strains [12]. In comparison, a k₂/K value of 58,000 M⁻¹ s⁻¹ has been reported for the acylation of PBP2x from a sensitive *S. pneumoniae* strain by benzylpenicillin [45].

The common motifs required to allow efficient acylation by penicillin are present in PBP5fm, with the exception of the guanidinium group found in β-lactamases that increases the efficacy of the acylation process, but this group is not essential for the acylation step [46]. The active-site cleft is delimited by the β₃ strand on one side, by residues 461–465 on the opposite side, and the bottom of the cavity is covered with residues 537–541. Among the residues surrounding the active-site groove, only Glu622 on the β₃ strand and Val465 on the opposite side

![Figure 4](attachment:Figure_4.png)

**Figure 4.** Ribbon representation of the C-terminal domain of PBP5fm. Color scheme as in figure 2. The conserved loops of subgroup B1 HMM-PBPs are colored green. Active-site catalytic residues are colored red, benzylpenicillin is colored black and the residues Val465 and Met485 are colored blue. On the left of the figure, Arg464, Asp481 and the backbone carbonyl of residues 446, 466 and 468 are shown in yellow with oxygen atoms in red and nitrogen atoms in blue. Arg464 is involved in a salt bridge with Asp481 and is at hydrogen bond distance from the carbonyl oxygen of residues 446, 466 and 468. See text for details.

can hinder the incoming antibiotic. In PBP2x, the mutation of Gln552 (equivalent to Glu622 in PBP5fm) into a glutamic acid results in a decreased affinity for benzylpenicillin by a factor of three [47]. Thus Glu622 could partially contribute to the low-level affinity of PBP5fm. On the other side of the groove, residues 461–465 are part of a loop (451–465) very well conserved in B1-PBPs, whereas the primary structure of this loop is different in the other classes of PBPs (figs 2A, 4). Arg464 might play a critical structural role. It is within hydrogen-bonding distance from the backbone carbonyl oxygen of residues 446, 466 and 468, and is involved in a salt bridge with Asp481. Val465 points into the active site and its side chain is close to the benzylpenicillin thiazolidine ring. The distance between the valine side chain Cγ2 and the sulfur atom of the benzylpenicillin is about 3.5 Å. A backward movement of the valine and of the loop bearing this residue is perhaps necessary to allow access of benzylpenicillin into the active site. Comparison between the PBP2x structure and its adduct with cefuroxime (pdb codes 1QME and 1QMF [48]) shows that a tryptophan, lying at a position equivalent to Val465, is slightly pushed away from the active site by cefuroxime and that the helix bearing the tryptophan is also somewhat displaced. In PBP5fm, the loop movement is accompanied by a shift of the Ser480-Asp-Asn loop via the Arg464-Asp481 salt bridge. Ser480 would then be too far away from the active-site serine to allow the proton shuttle between the active-site serine and the β-lactam nitrogen. According to this interpretation, the interaction between the 451–465 loop and the second active-site motif, and more precisely the constraints induced by the Arg464 interactions, confer a great rigidity to this side of the groove that would be responsible for the low-level affinity of PBP5fm. Among the penicilloyl transferase enzymes, only B1-PBPs and class A β-lactamases have an aspartic acid in the second active-site element (Ser480-Asp-Asn). Nevertheless, the aspartic acid interactions as well as the enzyme structure around this residue are completely different in both types of enzyme. The strong interaction between Arg464 and Asp481 is thus typical of B1-PBPs.

The bottom of the cavity is covered with residues 537–541. These residues and the 530–536 helix are well conserved in B1-PBPs whereas the whole 530–541 sequence is different in other class B PBPs (fig. 4). Nevertheless, no side chain position or particular interaction occurring in this region can easily be related to the low-level affinity of PBP5fm.

### Highly resistant mutants

The protein described above is responsible for the high resistance to β-lactam antibiotics of the *E. faecium* D63r strain (MIC of penicillin = 70 μg/ml). The mechanism involved in the resistance is the overproduction of the protein [12]. The past two decades have seen the emergence
of some clinical isolates highly resistant to ampicillin, not by means of PBP5fm overproduction but by a reduction of the protein affinity for the antibiotic. Following an analysis by Rybkine et al. [13] who studied a series of clinical isolates with various levels of resistance to ampicillin (well correlated with the resistance to benzylpenicillin), a first level of resistance is associated with PBP5fm overproduction but higher levels of resistance can be achieved by mutations of Met485 (fig. 4) or the insertion of an additional serine in position 466'.

MICs of ampicillin for strains retaining Met485 are not higher than 16 μg/ml (observed in four strains). Replacement of the methionine by threonine results in an MIC increase to values as high as 64 μg/ml (seven strains) while its mutation into alanine gives MICs ranging from 128 to 256 μg/ml (five strains). Met485 belongs to helix αC5 (three residues after the Ser480-Asp-Asn triad) and its side chain is located behind the Lys425 side chain. The removal of the bulky methionine side chain would set some free space, giving the lysine more conformational freedom and resulting in a less efficient acylation process. In a similar way, the substitution of Met426 by an isoleucine, as observed in the ampicillin-resistant strain 9439 [49], could be responsible for the resistance of this strain to ampicillin. As Met426 is near Lys425, modification of its side chain could slightly destabilize the active site. Nevertheless, changing a methionine into an isoleucine only leads to minor steric modifications and can hardly account for an important reduction in the acylation of PBP5fm by ampicillin.

Insertion of a serine in position 466' (residue 466 is also a serine) in the most resistant strains emphasizes the importance of the 451–465 loop on the left-hand side of the active site. This insertion is associated with a twofold-increased MIC for penicillin (strains AR9, 6885 and H80721) and seems to be independent of Met485 mutations [13]. As mentioned above, Val465 is close to the thiazolidine ring of the penicilloyl moiety and the structure rigidity of the left side of the active-site cleft rests upon the Arg464–Asp481 salt bridge. The insertion of a residue in position 466' may slightly displace Val465 inside the active site, reducing its accessibility for penicillin. The serine insertion would therefore affect substrate recognition.

### Discussion

PBP5fm is a multimodular protein with two main domains. The C-terminal one, commonly named the penicillin-binding domain, is expected to be responsible for the DD-transpeptidase activity and is inhibited by β-lactam antibiotics. The function of the N-terminal domain is still unclear. However, the two domains seem to be interdependent. As the PBP5fm N-terminal domain is far from the penicillin-binding module active site (about 50 Å), the N-terminal module has been hypothesized to be necessary for correct folding of the C-terminal one. This assumption remains to be demonstrated but, presumably, a misfolded or unfolded truncated N-terminal domain may somehow modify the transpeptidase domain structure or interfere with its penicillin-binding ability. An interaction between the N-terminal domain and one or several other proteins remains the most attractive proposal. Since PBP5fm is deprived of the sequence motifs correlated with the transglycosylase activity of class A HMM-PBPs and seems, accordingly, to be unable to carry out this activity, the final step of the peptidoglycan synthesis may require its transpeptidase module to be near the transglycosylase domain of another protein. The recent finding that the transglycosylase domain of *S. aureus* PBP2, together with the PBP2a transpeptidase domain, is needed for the optimal expression of methicillin resistance [50] is supporting this hypothesis.

The N-terminal extensions of the three representative B1-PBPs (PBP5fm, *S. aureus* PBP2a and *B. subtilis* PBP3) are of similar length and share common motifs which are missing in other subgroups of class B HMM-PBPs. Nevertheless, the homology between the primary structures of the three enzymes is lower in the globular region of the N-terminal module (from the anchor peptide to the first class B conserved motif, residues 39–162) than in the rest of the molecule. This lends further support to the idea of an N-terminal module interacting with other proteins, since protein-protein interactions occurring in the supramolecular assembly involved in the late steps of peptidoglycan synthesis can be specific for each species. Yet, biological function of the N-terminal domain remains unclear and the validity of a model associating a normal PBP transglycosylase activity and a rescue PBP transpeptidase activity to express β-lactam resistance remains to be demonstrated in enterococci.

The reason why PBP5fm is naturally resistant to penicillin is not evident at first sight. Sequence alignments give supportive evidence that all B1-PBPs are structurally alike and probably play a similar functional role in peptidoglycan synthesis. The active-site region is surrounded by sequence elements (451–465, 530–540) that are strongly conserved in B1-PBPs, while the sequence homology is lower in other regions. These sequence elements are not conserved in other classes of PBPs, in their primary structure and probably also in their tertiary structure, as shown by the comparison with the PBP2x three-dimensional structure. Accordingly, the presence of Val465 near the binding site and the salt bridge Arg464-Asp481, characteristic of B1-PBPs, can be invoked as structural elements responsible for the low-level affinity for penicillin of these enzymes. PBP5fm mutations associated with highly resistant clinical isolates are the replacement of Met485 by threonine or alanine, potentially
leading to a decrease of k₂, and the insertion of a serine in position 466', which might in turn affect β-lactam recognition. The latter modification is found in the sequence of the most highly ampicillin resistant strains and seems to be an essential determinant in PBP5fm-mediated β-lactam resistance.

**Acknowledgements.** This work was supported in part by the Belgian Program on Interuniversity Poles of Attraction initiated by the Belgian State, Prime Minister’s Office, Science Policy Programming (PAI P4/03).

1. Ghuysen J. M. (1994) Molecular structure of penicillin-binding proteins and β-lactamases. Trends Microbiol. **2**: 372–380.
2. Frère J. M., Nguyen-Distèche M., Coyette J. and Joris B. (1992). Mode of action: interaction with the penicillin-binding proteins. In: The Chemistry of β-Lactams, pp. 148-197, Page M. I. (ed.), Blackie, London
3. Goffin C. and Ghuysen J. M. (1998) Multimodular penicillin-binding proteins: an enigmatic family of orthologs and paralogs. Microbiol. Mol. Biol. Rev. **62**: 1079–1093.
4. Massova I. and Mobashery S. (1998) Kinship and diversification of bacterial penicillin-binding proteins and β-lactamases. Antimicrob. Agents Chemother. **42**: 1–17
5. Nguyen-Distèche M., Fraipont C., Buddelmeijer N. and Nanninga N. (1998) The structure and function of Escherichia coli penicillin-binding protein 3. Cell. Mol. Life Sci. **54**: 309–316
6. Vicente M. and Errington J. (1996) Structure, function and controls in microbial division. Mol. Microbiol. **20**: 1–7
7. Ghuysen J. M. (1997) Penicillin-binding proteins: wall peptidoglycan assembly and resistance to penicillin: facts, doubts and hopes. Int. J. Antimicrob. Agents **8**: 45–60
8. Williamson R., LeBouguenec C., Gutmann L. and Horaud T. (1985) One or two low affinity penicillin-binding proteins may be responsible for the range of susceptibility of Enterococcus faecium to benzylpenicillin. J. Gen. Microbiol. **131**: 1933–1940
9. Hakenbeck R. and Coyette J. (1998) Resistant penicillin-binding proteins. Cell. Mol. Life Sci. **54**: 332–340
10. Murray B. E. (1990) The life and times of the Enterococcus. Clin. Microbiol. Rev. **3**: 46–65
11. Fontana R., Aldegheri M., Ligozzi M., Lopez H., Sucari A. and Satta G. (1994) Overproduction of a low-affinity penicillin-binding protein and high-level resistance in Enterococcus faecium. Antimicrob. Agents Chemother. **38**: 1980–1983
12. Zorzi W., Zhou X. Y., Dardenne O., Lamotte J., Raze D., Pierre J. et al. (1996) Structure of the low-affinity penicillin-binding protein 5 PBP5fm in wild-type and highly penicillin-resistant strains of Enterococcus faecium. J. Bacteriol. **178**: 4948–4957
13. Rybkine T., Mainardi J. L., Sougakoff W., Collatz E. and Gutmann L. (1998) Penicillin-binding protein 5 sequence alteration in clinical isolates of Enterococcus faecium with different levels of β-lactam resistance. J. Infect. Dis. **178**: 159–163
14. Hanrahan J., Hoyen C. and Rice L. B. (2000) Geographic distribution of a large mobile element that transfers ampicillin and vancomycin resistance between Enterococcus faecium strains. Antimicrob. Agents Chemother. **44**: 1349–1351
15. Al-Obeid S., Billot-Klein D., Heijenoort J. van, Collatz E. and Gutmann L. (1992) Replacement of the essential penicillin-binding protein 5 by high-molecular mass PBPs may explain vancomycin-β-lactam synergy in low-level vancomycin-resistant Enterococcus faecium D366. FEMS Microbiol. Lett. **91**: 79–84
16. Carias L. L., Rudin S. D., Donskey C. J. and Rice L. B. (1998) Genetic linkage and cotransfer of a novel, vanB-containing transposon (Tn5382) and a low-affinity penicillin-binding pro-
tein 5 gene in a clinical vancomycin-resistant *Enterococcus faecium* isolate. J. Bacteriol. **180**: 4426–4434
17. Klare I., Rodloff A. C., Wagner J., Witte W. and Hakenbeck R. (1992) Overproduction of a penicillin-binding protein is not the only mechanism of penicillin resistance in *Enterococcus faecium*. Antimicrob. Agents Chemother. **36**: 783–787
18. Rice L. B., Carias L. L., Hutton-Thomas R., Sifaoui F., Gutmann L. and Rudin S. D. (2001) Penicillin-binding protein 5 and expression of ampicillin resistance in *Enterococcus faecium*. Antimicrob. Agents Chemother. **45**: 1480–1486
19. Sifaoui F., Arthur M., Rice L. and Gutmann L. (2001) Role of penicillin-binding protein 5 in expression of ampicillin resistance and peptidoglycan structure in *Enterococcus faecium*. Antimicrob. Agents Chemother. **45**: 2594–2597
20. Signoretto C., Boaretti M. and Canepari P. (1998) Peptidoglycan synthesis by *Enterococcus faecalis* penicillin binding protein 5. Arch. Microbiol. **170**: 185–190
21. Bugg T. D. H., Wright G. D., Dutka-Malen S., Arthur M., Courvalin P. and Walsh C. T. (1991) Molecular basis for vancomycin resistance in *Enterococcus faecium* BM4147: biosynthesis of a depsipeptide peptidoglycan precursor by vancomycin resistance proteins VanH and VanA. Biochemistry **30**: 2017–2021
22. Altschul S. F., Madden T. L., Schaffer A. A., Zhang J., Zhang Z., Miller W. et al. (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res. **25**: 3389–3402
23. Pares S., Mouz N., Petillot Y., Hakenbeck R. and Dideberg O. (1996) X-ray structure of *Streptococcus pneumoniae* PBP2x, a primary penicillin target enzyme. Nat. Struct. Biol. **3**: 284–289
24. Dessen A., Mouz N., Hopkins J. and Dideberg O. (2001) Crystal structure of PBP2x from a highly penicillin-resistant *Streptococcus pneumoniae* clinical isolate: a mosaic framework containing 83 mutations. J. Biol. Chem. **30**: 45106–45112
25. Korz D. J., Rinas U., Hellmuth K., Sanders E. A. and Deckwer W. D. (1995) Simple fed-batch technique for high cell density cultivation of *Escherichia coli*. J. Biotechnol. **39**: 59–65
26. LeMaster D. M. and Richards F. M. (1985) 1H-15N heteronuclear NMR studies of *Escherichia coli* thioredoxin in samples isotopically labeled by residue type. Biochemistry **24**: 7263–7268
27. Leslie A. G. R. W. (1996) MOSFLM Version 5.40 Mosflm Users Guide
28. CCP4 (1994) Collaborative computing project. Acta Crystallogr. **D56**: 791–794
29. Miller R., Gallo S. M., Khalak H. G and Weeks C. M. (1994) *SnB*: crystal structure determination via Shake-and-Bake. J. Appl. Cryst. **27**: 613–621
30. Cowtan K. and Main P. (1998) Miscellaneous algorithms for density modification. Acta Crystallogr. **D54**: 487–493
31. Roussel A. and Cambillau C. (1989). Silicon Graphics Geometry Partner Directory, pp. 77–78, Silicon Graphics, Mountain View, Calif.
32. Brünger A. T., Adams P. D., Clore G. M., DeLano W. L., Gros P., Grosse-Kunstleve R. W. et al. (1998) Crystallography & NMR system: a new software suite for macromolecular structure determination. Acta Crystallogr. **D54**: 905–921
33. Möllerach M. E., Partoune P., Coyette J. and Ghuysen J. M. (1996) Importance of the E46–D160 polypeptide segment of the non-penicillin-binding module for the folding of the low-affinity, multimodular class B penicillin-binding protein 5 of *Enterococcus hirae*. J. Bacteriol. **178**: 1774–1775
34. Wu C. Y. E., Alborn W. E., Flokowitsch J. E., Hoskins J., Ünal S., Blaszczak L. C. et al. (1994) Site-directed mutagenesis of the *mecA* gene from a methicillin-resistant strain of *Staphylococcus aureus*. J. Bacteriol. **176**: 443–449
35. Maveyraud L., Golemi D., Kotra L. P., Tranier S., Valulenko S., Mobashery S. et al. (2000) Insights into class D β-lactamases

are revealed by the crystal structure of the OXA10 enzyme from *Pseudomonas aeruginosa*. Structure **8**: 1289–1298

36 Davies C., White S. W. and Nicholas R. A. (2001) Crystal structure of a deacylation-defective mutant of penicillin-binding protein 5 at 2.3 Å resolution. J. Biol. Chem. **276**: 616–623

37 Kelly J. A., Knox J. R., Moews P. C., Hite G. J., Bartolone J. B., Zhao H. et al. (1985) 2.8-Å structure of penicillin-sensitive D-alanyl carboxypeptidase-transpeptidase from *Streptomyces* R61 and complexes with beta-lactams. J. Biol. Chem. **260**: 6449–6458

38 Fonzi E., Vermeire M., Nguyen-Distèche M., Brasseur R. and Charlier P. (1999) The crystal structure of a penicilloyl-serine transferase of intermediate penicillin sensitivity: the DD-transpeptidase of *Streptomyces* K15. J. Biol. Chem. **274**: 21853–21860

39 Strynadka N. C. J., Adachi H., Jensen S. E., Johns K., Sielecki A., Betzel C. et al. (1992) Molecular structure of the acyl-enzyme intermediate in β-lactam hydrolysis at 1.7 Å resolution. Nature **359**: 700–705

40 Zafaralla G., Manavathu E. K., Lerner S. A. and Mobashery S. (1992) Elucidation of the role of arginine-244 in the turnover processes of class A beta-lactamases. Biochemistry **31**: 3847–3852

41 Wladkowski B. D., Chenoweth S. A., Sanders J. N., Krauss M. and Stevens W. J. (1997) Acylation of β-lactams by class A β-lactamase: an ab initio theoretical study on the effects of the oxy-anion hole. J. Am. Chem. Soc. **119**: 6923–6431

42 Dive G. and Dehareng D. (1999) Serine peptidase catalytic machinery: cooperative one-step mechanism. Int. J. Quant. Chem. **73**: 161–174

43 Adachi H., Ohta T. and Matsuzawa H. (1991) Site-directed mutants, at position 166, of RTEM-1 β-lactamase that form a stable acyl-enzyme intermediate with penicillin. J. Biol. Chem. **266**: 3186–3191

44 Lu W. P., Sun Y., Bauer M. D., Paule S., Koenigs P. M. and Kraft W. G. (1999) Penicillin-binding protein 2a from methicillin-resistant *Staphylococcus aureus*: kinetic characterization of its interactions with β-lactams using electrospray mass spectrometry. Biochemistry **38**: 6537–6546

45 Jamin M., Damblon C., Millier S., Hakenbeck R. and Frère J. M. (1993) Penicillin-binding protein 2x of *Streptococcus pneumoniae*: enzymic activities and interactions with beta-lactams. Biochem. J. **292**: 735–741

46 Delaire M., Labia R., Samama J. P. and Masson J. M. (1992) Site-directed mutagenesis at the active site of *Escherichia coli* TEM-1 beta-lactamase: suicide inhibitor-resistant mutants reveal the role of arginine 244 and methionine 69 in catalysis. J. Biol. Chem. **267**: 20600–20606

47 Mouz N., Di Guilmi A. M., Gordon E., Hakenbeck R., Dideberg O. and Vernet T. (1999) Mutations in the active site of penicillin-binding protein PBP2x from *Streptococcus pneumoniae*. J. Biol. Chem. **274**: 19175–19180

48 Gordon E., Mouz N., Duée E. and Dideberg O. (2000) The crystal structure of the penicillin-binding protein 2x from *Streptococcus pneumoniae* and its acyl-enzyme form: implication in drug resistance. J. Mol. Biol. **299**: 477–485

49 Ligozzi M., Pittaluga F. and Fontana R. (1996) Modification of penicillin-binding protein 5 associated with high-level ampicillin-resistance in *Enterococcus faecium*. Antimicrob. Agents Chemother. **40**: 354–357

50 Pinho M. G, de Lencastre H. and Tomasz A. (2001) An acquired and a native penicillin-binding protein cooperate in building the cell wall of drug-resistant staphylococci. Proc. Natl. Acad. Sci. USA **98**: 10886–10891

To access this journal online:
http://www.birkhauser.ch
