
Biochemistry 2002, 41, 11649–11657

Multistate Binding in Pyridoxine 5′-Phosphate Synthase: 1.96 Å Crystal Structure in Complex with 1-Deoxy-D-xylulose Phosphate†

Joanne I. Yeh,*,$^{\ddagger}$,$^{§}$ Shoucheng Du,$^{§,\|}$ Ehmke Pohl,$^{\bot}$ and David E. Cane$^{§}$

Molecular Biology, Cell Biology, and Biochemistry Department, Brown University, Box G-J2, Providence, Rhode Island 02912-9108, Department of Chemistry, Brown University, Box H, Providence, Rhode Island 02912-9108, and Outstation Hamburg, European Molecular Biology Laboratory, c/o Desy, Notkestrasse 85, D-22603 Hamburg, Germany

Received June 13, 2002; Revised Manuscript Received July 29, 2002

ABSTRACT: We report the 1.96 Å crystal structure of pyridoxine 5′-phosphate synthase (PdxJ) in complex with 1-deoxy-D-xylulose phosphate (dXP). The octameric enzyme possesses eight distinct binding sites, and three different binding states are observed. The observation of these three states supports a mechanism in which precise conformational changes of a peptide loop and groups of active site residues modulate binding and specificity. The differences in protein conformation when one or two substrates are bound can be correlated with a condensation mechanism that leads productively to the formation of pyridoxine 5′-phosphate (PNP). “Snapshots” of the progression from the apo form to a singly occupied “transitional binding” state and, subsequently, to a fully occupied, reactive state are revealed and indicate how the enzyme structure can be related to a plausible catalytic mechanism and, moreover, to favorable energetics of reaction.

Vitamin B₆, consisting of three closely related pyridoxine derivatives and their phosphates, pyridoxol phosphate (PNP),$^{1}$ pyridoxal phosphate (PLP), and pyridoxamine phosphate (PMP), is an essential cofactor in important metabolic pathways, including amino acid metabolism and glycolysis. The biosynthesis of PNP in *Escherichia coli* and its ubiquitous essential coenzyme derivatives, pyridoxal 5′-phosphate and pyridoxamine 5′-phosphate, is dependent on a small number of enzymes. In *E. coli*, two gene products, PdxA and PdxJ, have been shown to mediate the oxidative conversion of 4-hydroxy-L-threonine 4-phosphate (4-HTP) and 1-deoxy-D-xylulose 5-phosphate (dXP) to form pyridoxol phosphate (vitamin B₆) (Scheme 1) ($I$–4). PdxA is a dehydrogenase, requiring NAD⁺ as a cofactor, and is responsible for the oxidation of 4-HTP to a product tentatively identified as 1-amino-3-hydroxyacetone 3-phosphate (AHP) (3). Incubation of 1-deoxy-D-xylulose 5-phosphate (dXP) with PdxJ in the presence of PdxA, 4-HTP, and NAD⁺ results in the formation of pyridoxol phosphate (PNP) and 1 equiv of inorganic phosphate, with a $K_{\text{m}}$(app) for dXP

---

† This work was supported by NIH Grant GM22172 to D.E.C.  
* To whom correspondence should be addressed. E-mail: Joanne_Yeh@brown.edu.  
$^{\ddagger}$ Molecular Biology, Cell Biology, and Biochemistry Department, Brown University.  
$^{§}$ Department of Chemistry, Brown University.  
$\|$ Current address: Sention Inc., 4 Richmond Square, 4th Floor, Providence, RI 02906.  
$\bot$ European Molecular Biology Laboratory.  

$^{1}$ Abbreviations: PdxJ, pyridoxine 5′-phosphate synthase; dXP, 1-deoxy-D-xylulose phosphate; PNP, pyridoxol phosphate; PLP, pyridoxal phosphate; PMP, pyridoxamine phosphate; 4-HTP, 4-hydroxy-L-threonine 4-phosphate; AHP, 1-amino-3-hydroxyacetone 3-phosphate; SIR, single isomorphous replacement; MR, molecular replacement; $P_{i}$, inorganic phosphate; dX, 1-deoxy-D-xylulose; ITC, isothermal titration calorimetry.

scaffold to support the biosynthesis of PNP. Earlier structural studies had reported the structures of apo PdxJ and PdxJ bound with PNP soaked into the crystals, both with and without the bound coproduct inorganic phosphate (P<sub>i</sub>) (6). Although these structures provided important insights into the biochemical function of PdxJ, several questions remained unanswered: (1) How is the conformation of a disordered loop region correlated with the occupancy of the active site? (2) Which active site residues are involved in mediating the binding and reaction of the substrates, dXP and AHP? (3) What is the actual mechanism of catalysis? We report here the structure of PdxJ cocrystallized in complex with both dXP and inorganic P<sub>i</sub>, the latter of which acts as an AHP “surrogate”. On the basis of this structure, we also propose a mechanism by which the substrates themselves organize the conformation of the active site peptide loop to modulate access to the active site, and we explain how precise rearrangements of the active site residues result in catalysis and formation of the product PNP.

# EXPERIMENTAL PROCEDURES

## Molecular Cloning, Overexpression, Purification, and dXP Preparation. Details on the cloning and overexpression of PdxJ have been reported elsewhere (3). Briefly, purification of PdxJ was achieved by the use of successive hydroxyapatite, Q-Sepharose, and Superdex HiLoad 16/60 columns. For the last column, the protein was eluted with 50 mM K<sup>+</sup> phosphate (pH 7.0) and 0.15 M NaCl. The major fractions containing purified PdxJ were combined; buffer was exchanged and concentrated with 10 mM Tris-HCl (pH 7.5) four times, resulting in a calculated residual inorganic phosphate concentration of 0.4 mM P<sub>i</sub>. The final components in the PdxJ preparation were therefore 10 mM Tris-HCl (pH 7.5), 0.40 mM K<sup>+</sup> phosphate, and 1.21 mM NaCl. PdxJ was concentrated up to 21 mg/mL, as determined by Bradford analysis; this concentrated enzyme was used as stock for subsequent crystallization setups.

The substrate, dXP, was synthesized from pyruvate and glyceraldehyde 3-phosphate in the presence of thiamin diphosphate using recombinant *E. coli* deoxyxylulose 5-phosphate synthase, as described previously (3). The purified dXP was dissolved in water and its concentration determined by conversion to PNP by incubation with 4-HTP and NAD<sup>+</sup> in the presence of PdxJ and PdxA. The concentration of the resultant PNP was determined by UV, based on the extinction coefficient of 7330 M<sup>−1</sup> cm<sup>−1</sup> (4).

## Crystallization. Crystals of PdxJ were obtained from an in-house screening array at room temperature; small crystals were grown from a solution initially containing 10% PEG 8000 and 10% PEG 1000. PdxJ at 12 mg/mL was incubated with 4 mM dXP for at least 30 min prior to crystallization setups. Nucleation and growth of small crystals were observed after 3 weeks, and the crystals continued to grow over 2 months using the hanging drop method. Optimization of the initial crystallization conditions utilizing additives and modifying the concentration of organic precipitants resulted in the growth of crystals that were suitable for data collection. The largest of these crystals were approximately 0.6 mm × 0.2 mm × 0.2 mm in dimension, rod-shaped, and birefringent, in space group *P2<sub>1</sub>2<sub>1</sub>2<sub>1</sub>*. These crystals were difficult to reproduce and exhibited inconsistent diffraction characteristics in that only two crystals in a drop of seven diffracted to high resolution, while other crystals from the same well diffracted to lower resolution and with higher mosaicity. The final crystallization solution contained 8% PEG 8000, 9% PEG 1000, and 10% glycerol.

## Data Collection and Processing. A high-resolution data set for structure determination and refinement was collected at EMBL-Hamburg at wiggler beamline BW12 of DESY in Hamburg, Germany, equipped with MAR CCD. This native data set was collected with a λ of 1.05 Å to a resolution of 1.96 Å. This data set was indexed and integrated using DENZO and scaled with the program SCALEPACK (7).

Although selenomethionine-substituted protein was produced for phasing, crystallization yielded only small crystals that were unsuitable for data collection, even after extensive screening efforts. To aid in structure determination, a gold derivative was obtained by soaking a native crystal in 0.1 mM KAu(CN)<sub>2</sub> for 18 h and collecting data using the in-house rotating anode X-ray source equipped with Yale mirrors on an R-Axis II image plate system (Rigaku/MSC, The Woodlands, TX). These crystals initially exhibited high mosaicity (>1.3°); application of the flash-annealing approach (8) to the crystal resulted in a mosaicity decreased to 0.7°. The flash-annealed derivative crystal diffracted to 3.2 Å with an *R*<sub>iso</sub> of 19.7% when scaled to the native data set. This data set was processed as described for the native data set. Data statistics and parameters are listed in Table 1.

## Structure Determination and Refinement. A combination of single isomorphous replacement (SIR; 9, 10) and molecular replacement (MR; 11) approaches using the published structure of PdxJ in complex with product PNP (6) was used to obtain the structure of PdxJ in complex with dXP. Attempts to obtain molecular replacement solutions using monomer and tetramer models did not yield promising results with the program CNS or AmoRe (12, 13). Using the Au derivative data set, Patterson maps were generated in CCP4

Table 1: Data Collection and Refinement Statistics

| Parameter | PdxJ, dXP, and P<sub>i</sub> | Au derivative<sup>d</sup> |
|-----------|-----------------------------|--------------------------|
| resolution range (Å) | 20–1.96 | 20–3.2 |
| resolution range in refinement (Å) | 6.0–1.96 | 6.0–3.2 |
| total no. of reflections | 494561 | 239354 |
| no. of independent reflections | 157249 | 37688 |
| completeness (%) | 96.0 | 97.5 |
| completeness at the highest-resolution shell (%) | 84.4 (2.03–1.96 Å) | 89.9 (3.3–3.2 Å) |
| no. of unique reflections > 3σ | 104455 | 41358 |
| cell parameters (Å) | *a* = 100.007, *b* = 129.404, *c* = 176.088 | *a* = 100.147, *b* = 129.608, *c* = 176.377 |
| *R*<sub>merge</sub> | 4.5 | 5.9 |
| *R*<sub>cryst</sub><sup>a</sup> | 19.5 | — |
| *R*<sub>free</sub><sup>a,b</sup> | 24.9 | — |
| average *B* | 31.3 | — |
| rms for bond lengths (Å) | 0.007 | — |
| rms for bond angles (deg) | 1.21 | — |
| rms coordinate error<sup>c</sup> (Å) | 0.11 | — |
| no. of protein and cofactor atoms | 15737 (six dXPs, two P<sub>i</sub>s) | — |
| no. of water molecules | 959 | — |

<sup>a</sup> All *R*-factors are defined as *R* = (∑|*F*<sub>o</sub> − *F*<sub>c</sub>|)/∑*F*<sub>o</sub>, where *F*<sub>o</sub> is the observed structure factor amplitude and *F*<sub>c</sub> is the calculated structure factor amplitude. <sup>b</sup> *R*<sub>free</sub> is calculated with 10% of the data that are not included in the refinement. <sup>c</sup> Coordinate errors were estimated through σ<sub>A</sub> plots (14). <sup>d</sup> *R*<sub>iso</sub> = 19.7. FOM = 0.35. Phasing power = 1.51. *R*<sub>cullis</sub> = 71%.

Structure of the Pdxj Complex

Biochemistry, Vol. 41, No. 39, 2002    11651

![Image Description](image1.png)

FIGURE 1: (A) Stereoview of the $\sigma_{A}$-weighted $2|F_{o}-F_{c}|$ electron density contoured at $3.5\sigma$ at the doubly occupied, with dXP and inorganic $P_{i}$ site. The presence of both dXP and $P_{i}$ at the active site results in loop closure. This concomitantly locks down the loop conformation and positions residues Thr102 and Thr103, located on the loop, to interact with the dXP substrate. (B) Stereoview of the $\sigma_{A}$-weighted $2|F_{o}-F_{c}|$ electron density contoured at $3.5\sigma$ at the singly occupied, with dXP, site. In the absence of AHP, dXP spans the positions eventually occupied by inorganic phosphate and the $5^{\prime}$-phosphate position of the product PNP, presumably corresponding to the location of the 3-phosphate of AHP. When only one of the two substrate molecules binds, the loop is in a conformation that is more closed than in the apo form but still open such that access to the active site is facile. This figure was displayed with the program O (23).

(14) and the positions of eight gold sites over $5\sigma$ were found in the difference Patterson map. SIR phases calculated from this derivative exhibited a figure of merit of 0.37, a phasing power of 1.51, and an $R_{\text{cullis}}$ of 71%. Starting from the experimental SIR phases, we carried out a round of density modification, solvent flattening, and histogram matching in DM which yielded initial phasing information. At 3.2 Å, the $F_{o}e^{i\alpha_{\text{calc}}}$ maps calculated using SIR phases were not optimal and discontinuous at many places; however, it did allow a model of PdxJ to be generated. This model was made using the apo tetramer (6) (PDB entry 1HO1) where one subunit of the apo form was held constant while a rigid-body rotation of approximately $1.2^{\circ}$ was applied to each of the three monomer subunits related by 222 symmetry about the eventual 4-fold axis of the octamer to effectively generate a “tighter” model. A final octamer search model was generated by applying a 2-fold rotation to this tetramer; this model also had a loop region (residues 95–105) omitted, using the stronger and more continuous densities to guide the model building. This new search model yielded a molecular replacement solution through CNS, where the highest peak in the cross rotation search was >3σ above the next peak and after a cycle of rigid-body refinement resulted in an $R$-factor of 34%. Once the solution for MR was found, a difference Fourier map was calculated to verify the solution using the positions of the heavy atoms obtained from the

Au derivative. After model building of regions of the N-terminus using the program O, refinement steps consisting of conjugated gradient minimization, simulated annealing, and individual $B$-factor refinement were carried out in CNS using the 1.96 Å native data (Table 1). Initially, noncrystallographic symmetry averaging was attempted on residues 21–220, omitting the loop region of residues 95–105, as ncs 222 symmetry was indicated to be present; however, the low correlation coefficient obtained from DM and the lack of map improvement suggested that symmetry averaging was not beneficial. Confirmation of differences at various regions between the structures was carried out by calculating local density correlation maps (15, 16). After a cycle of refinement, the loop regions were built in, and the differences in the loop conformations of the subunits indicated that some of the monomers of the octamer had rmsds for the Cα positions significantly greater than 1 Å, and up to 4 Å over Cα coordinates. Utilizing $\sigma_{A}$-weighted $2F_{o}-F_{c}$ and $F_{o}-F_{c}$ maps (17), substrates were built into the strongest densities. Of the eight binding sites, initial model building located four of the dXPs and two $P_{i}$ molecules (Figure 1A,B). After subsequent cycles of refinement, two additional dXPs were built into two of the remaining sites, while two of the active sites remain empty in the final, refined model. We designate each site by the conformation of the loop encompassing residues 95–105, hereafter called the “active site loop”: fully

11652 Biochemistry, Vol. 41, No. 39, 2002

open (apo, unbound form), partially open (single-dXP-bound form), and closed (dXP- and P<sub>i</sub>-bound form) binding states. The nonequivalence of the binding sites results in localized changes in the interaction of some of the residues at the active site as well as more substantial differences in the conformation of the active site loop. The conformation of the loop plays an important role in moderating accessibility of the active site, as discussed below. Hence, from the structure, we in effect have snapshots of a plausible sequence of events encompassing three distinct catalytic states leading to the formation of the product. The final model has been refined at 1.96 Å to an *R*-factor of 19.7% and an *R*<sub>free</sub> of 25.1% (Table 1).

**Isothermal Titration Calorimetry.** Isothermal titration calorimetry (ITC) experiments were carried out to assess the binding of dXP to PdxJ in the absence of AHP, utilizing the same buffer conditions that were used to obtain crystals of PdxJ. ITC was performed using a MicroCal titration microcalorimeter (Northampton, MA) (18). Solutions were prepared in 10 mM Tris (pH 7.5) and degassed under vacuum prior to use. The reference offset was 25% and the reaction temperature 25 °C with a stirring speed of 300 rpm. The reaction and reference cells had a volume of 1.374 mL. The concentration of the enzyme, PdxJ, was 100 μM in the cell and of the titrant, dXP, was 2 mM with an injection volume of 5 μL, an injection duration of 10 s, and 120 s between injections. A total of 25 injections resulted in a 1.8-fold molar excess of dXP to PdxJ at the end of the titration. Heats of dilution of ligand were obtained by titrating dXP into buffer in the absence of macromolecules, and all runs were performed in duplicate. Background reaction heats were subtracted from the data before isotherms were evaluated. Integration of data was done through the Origin software supplied by MicroCal, Inc.

## RESULTS

The structure of PdxJ in complex with its substrate, dXP, and inorganic phosphate, P<sub>i</sub>, displaying three different binding states, has revealed unexpected new details which shed light on the mode of action of pyridoxine synthase.

### Quaternary Structure. PdxJ is an octameric enzyme, with a monomer fold comprised of a TIM barrel motif (*β/α*) and substrate-binding sites at the interface of two subunits (Figure 2A,B). Although the free protein is a dimer of tetramers (6), asymmetry is induced in the structure of PdxJ with bound dXP and P<sub>i</sub>. The previously reported structures were of both the apo and PNP-bound forms, in which PNP was soaked into pre-existing apo crystals; both of these protein structures contained a tetramer in the asymmetric unit (6). In contrast, the substrate-complexed form, in which dXP has been cocrystallized with PdxJ in the presence of P<sub>i</sub>, has as its asymmetric unit a complete octamer, comprised of eight unique protein molecules, labeled A–H, according to the nomenclature of Franco et al. (6). Differences exist between the individual monomer subunits of the octamer, particularly at the loop regions and in the conformation of specific active site residues. When compared to the previously described apo and PNP-bound structures, the structure with bound substrate displays a tighter, more closed state. Superpositioning analysis (19, 20) allows a quantitative comparison

FIGURE 2: (A) View down the 4-fold axis of the octamer, where each monomer subunit is in a different color. The substrates, shown in ball-and-stick form, bind at the interface of two monomer subunits. The conformation of the active site loop, in purple, is correlated to the occupancy of the active site and can be in one of three states: open, partially open, and closed. (B) Dimer interface of the A and E subunits. Subunit A on top is the ternary complex, bound with both dXP and P<sub>i</sub>, while subunit E is of the binary complex of a singly bound dXP. Residues at the active site are depicted in ball-and-stick form; one residue, Arg20, from one subunit interacts with the substrate at the second subunit. The active site loop, colored in purple and shown as a coil, closes over the active site once both substrates are bound. This figure was generated with MOLSCRIPT and Raster3D (24, 25).

of the apo, product-bound, and substrate-bound forms (Table 2). Overall, the apo structure is >2° more “open”, as viewed along a reference 2-fold rotation axis that relates the A

Structure of the Pdxj Complex

Table 2: rmsd Analysis$^a$

| subunit | $\langle \text{rmsd} \rangle$ (Å) | subunit | $\langle \text{rmsd} \rangle$ (Å) |
|---------|---------------------------------|---------|---------------------------------|
| (1) double (dXP and $P_i$) | 0.35 | (4) double:single ratio | 1.99 |
| (2) single (dXP) | 0.93 | (5) double:empty ratio | 2.00 |
| (3) empty | 0.46 | (6) single:empty ratio | 1.21 |

$^a$ Cα superposition of the monomer subunits of the octamer; of the eight substrate-binding sites, two are empty, four sites contain a single dXP substrate (single), and two sites contain both dXP and $P_i$ (double). The presence or absence of substrate(s) essentially establishes the conformational state of the monomers. This is indicated by the low rmsd values when subunits of similar binding states are superimposed (values 1–3), while larger conformational differences exist in the enzyme when different substrate-bound forms are compared (values 4–6).

subunits to subunit H. When subunit A of the apo octamer is superimposed with subunit A of the dXP substrate-bound form, a rigid-body rotation of 2.6° is required to superimpose the H subunits. When this same analysis is done using the PNP-complexed form, the magnitude of rotation for superpositioning is 1.5°. These values illustrate that the structure in complex with the substrate and $P_i$, the presumed AHP surrogate, results in a more tightly associated state. One can speculate that the open apo form of PdxJ allows access of the substrates to the active site; once substrates are bound, the entire octameric structure is in a more closed conformation which then relaxes once the product is formed, allowing the product to diffuse out of the active site.

**Three Binding States Revealed.** As described in Experimental Procedures, phosphate buffer was used during protein purification, and although extensive washing was done during buffer exchange to Tris-HCl (pH 7.5), inorganic phosphate was carried through. At the calculated residual phosphate concentration of 0.4 mM, there would be ~0.6 equiv of $P_i$ for each PdxJ subunit under the crystallization conditions. Solely on the basis of stoichiometry, there would thus be sufficient $P_i$ to occupy more than half of the active sites of PdxJ. In the octameric enzyme, however, only two of eight sites were found to have bound $P_i$ and then only when a molecule of dXP was also present. Although the enzyme was incubated with a >8-fold excess of dXP, two empty sites were still found in the octameric complex structure. It is unknown whether the nonequivalence of $P_i$ binding exhibited by the crystals is possibly due to a lowered affinity for dXP and $P_i$ under the crystallization conditions or whether this reflects the intrinsic binding affinities of the enzyme since saturation of sites would have been expected from a simple binding model at the concentrations used in the crystallization setups (PdxJ at 0.46 mM and dXP at 4 mM). The fact that differential binding is exhibited by the enzyme, despite the presence of excess dXP and more than half-equivalence of $P_i$, suggests that the enzyme inherently possesses nonequivalence of binding states. The binding of the first substrate, which transforms the initially similar binding sites into three different forms, could induce this nonequivalence.

The correlation of the enzyme structure to binding site occupancy is mediated through the conformation of the active site loop comprised of amino acids 95–105. In the absence of substrate, the loop is fully folded back so that the empty active site is completely accessible. When only dXP is bound, the loop changes conformation and partially closes the active

FIGURE 3: Superposition of the singly occupied dXP site with the PNP- and $P_i$ product-bound form, in green. The phosphate moiety of dXP bridges the position eventually occupied by the 5′-phosphate of PNP and the product inorganic phosphate ($P_i$), derived from the 5-phosphate group of dXP. The 5′-phosphate of the product PNP is derived from the 3-phosphate ester of the second substrate AHP (Scheme 2; see the text). This figure was displayed with the program O (23).

site while still allowing access of the second substrate. The closed loop conformation results when both dXP and $P_i$, which in our structure is thought to be a surrogate for AHP, are bound. The latter conformation is very similar to the loop conformation observed in one subunit of the tetramer which had the product PNP and $P_i$ soaked in (6). Significantly, binding of both substrates locks down the active site loop, limiting further entry to the active site.

**Partially Open Conformation: Single-dXP-Bound State.** Four of the eight active sites have a single bound dXP. This substrate is held in a conformation such that the 5-phosphate moiety bridges the sites that are to be occupied by inorganic phosphate (ultimately derived from C-5 of dXP) and the 5′-phosphate of PNP, as found in the previously described closed PNP-$P_i$ product structure (Figure 3). The pentulose sugar backbone, however, is twisted away from the site that is eventually occupied by the resultant PNP product. The bridging of the presumed binding sites for both substrates, dXP and AHP (Figure 4), therefore corresponds to a transitional binding state, in which the dXP molecule is bound such that both substrate binding sites are partially occupied. Thus, one end of the pentulose is held close to the position it will occupy in the dXP- and $P_i$-bound state, but the sugar backbone occupies the position where AHP must eventually bind. When the singly bound dXP structure (Figure 5) is compared with the structure of the doubly bound state, in which both dXP and $P_i$ are present in the pocket (Figure 6), the dXP molecule must flip over to coordinate with Glu72 and simultaneously interact with residues Thr102 and Thr103 of the active site loop. This reorganization brings the loop into a closed conformation, in which the ternary complex is assembled to allow formation of PNP. One could speculate that in the absence of substrates or when only one of the two substrates is present, the loop is flexible and open,

11654  Biochemistry, Vol. 41, No. 39, 2002

FIGURE 4: Superposition of the doubly occupied dXP- and P<sub>i</sub>-bound site with the PNP- and P<sub>i</sub> product-bound form, in green. The presence of P<sub>i</sub>, a surrogate of AHP, flips the dXP molecule which then coordinates with Glu72. Additional coordination with groups located on the active site loop, Thr102 and Thr103, maintains the loop in a closed conformation. This figure was displayed with the program O (23).

FIGURE 6: Doubly occupied active site, showing the interaction of dXP, in orange in ball-and-stick form, and P<sub>i</sub>, in violet in ball-and-stick form, with active site residues. In addition to the interactions of the phosphate ester moiety of dXP, additional interactions are formed with Glu72, Arg20, Glu72, and His193. The active site loop, in red, closes over the active site, precluding further access. The loop is held in this position by interactions between Thr102 and Thr103 of PdxJ with dXP in this new conformation. This figure was generated with MOLSCRIPT and Raster3D (24, 25).

and Arg20 NH1 (3.5 Å) of the second subunit. This correlates well with solution data that show that deoxyxylulose does not bind (4). This is also energetically favorable for the subsequent step, formation of the ternary complex once AHP is present (described below). The observation of singly bound dXP does not necessarily imply that there is an intrinsically preferred order to substrate binding.

Closed Conformation: dXP- and P<sub>i</sub>-Bound State. Two of the eight binding sites of dXP have both dXP and P<sub>i</sub> bound (Figure 6). The productive ternary complex will be formed when the second substrate, normally AHP, binds, thereby shifting the dXP to a conformation in which it is poised correctly for PNP formation while locking down the active site loop such that further access to the active site during catalysis is prevented. Superposition of the structure of the closed ternary substrate complex with the structure of PdxJ with bound PNP and P<sub>i</sub> (6) provides important clues about the roles played by several key active site residues in catalyzing the formation of PNP (Figure 4 and Scheme 2). In the closed, ternary complex, the C═O bond of dXP is oriented roughly orthogonal to the plane of the eventually formed PNP, as required for formation of Schiff base 1 by nucleophilic attack of the amino group of the AHP. Glu72 is suitably positioned to catalyze formation of this Schiff base by protonating the carbonyl oxygen atom of dXP. The resulting Glu72 carboxylate might then be correctly positioned to remove the H-3 proton of the dXP–AHP imine 1. Alternatively, Asp11 might act as the base that removes H-3. The nearby His193, on the other hand, is clearly unable to carry out this deprotonation, since it is positioned on the opposite face of the bound dXP. The protonated imidazolium form of His193 could serve as a Lewis acid that promotes

allowing facile access to the active site. The major interactions are through the phosphate ester group of dXP, where the phosphate oxygen atoms interact with residues Asp11, His45, and His52 of the enzyme as follows: O4P—His45 Nδ1 (3.1 Å), O5P—Asp11 Oδ2 (2.7 Å), and O5P—His52 Nε2 (2.9 Å). An additional interaction is between dXP O2

FIGURE 5: Single dXP-bound state, showing the interaction of dXP, colored orange in ball-and-stick form, with active site residues. The major binding interactions are through the phosphate ester moiety of dXP, with residues Asp11, His45, and His52. This correlates well with the solution data which show that deoxyxylulose (without the phosphate ester group) does not bind to PdxJ (see the text). The conformation of the active site loop (residues 95–105), in red, is partially open, allowing facile access of the second substrate, AHP, to the active site. This figure was generated with MOLSCRIPT and Raster3D (24, 25).

Structure of the Pdxj Complex

Biochemistry, Vol. 41, No. 39, 2002    11655

Scheme 2

the elimination of the original C-4 hydroxyl group, thereby generating the transient enol species 2. Generation of enol 2 would promote elimination of the 5-phosphate group, which in turn should be facilitated by ionic interactions with the guanidinium groups of Arg47 (not shown) and Arg20 from the adjacent subunit, as well as hydrogen bonds to the side chain hydroxyls of Thr102 and Thr103 in the active site loop. In principle, His193, once again in its basic form, would be properly positioned to remove the enol proton from 2. Addition of water to the resulting enone 3, coupled with nucleophilic attack on the carbonyl derived from C-2 of AHP leads to ring closure, whereupon successive dehydration and tautomerization will yield pyridoxol phosphate, PNP.

The interaction between PdxJ and dXP through the phosphate ester group of dXP is similar to the singly bound state but tighter in that additional coordination occurs. The phosphate oxygen atoms interact with residues His45 and His52 as in the singly bound form, but an additional interaction with Arg47 forms O4P—His45 Nε2 (3.5 Å), O5P—His45 Nδ1 (3.1 Å), O5P—His52 Nε2 (3.6 Å), and O5P—Arg47 ND1 (2.75 Å) interactions. The interaction between dXP O2 and Arg20 NH2 of the second subunit is strengthened by a tighter bond distance of 3.2 Å. Further interactions formed with the new conformation of dXP involve dXP O2 and Glu72 Oε1 (2.6 Å) and Glu72 Oε2 (3.0 Å) and dXP O4 and His193 Nδ1 (2.3 Å). Coordination of dXP with loop residues Thr102 and Thr103 is only possible when the active site loop is in the closed conformation, with both dXP and AHP, or its presumed surrogate, Pi, is bound. It is tempting to speculate that the formation of the product PNP decreases the extent of interaction with loop residues Thr102 and Thr103. This is indicated by a change in the favorable distance of 2.6 Å and bond angle of 168° between Thr102 Oγ1 and dXP O4 to 3.4 Å between Thr102 Oγ1 and O4A in the PNP structure, loosening the interaction with the active site loop. The interaction between dXP O3 and Thr103 Oγ1 is maintained in the PNP structure, but the geometry becomes less favorable. Together, these subtle shifts of interactions with the active site loop should allow greater mobility of the loop itself, thereby allowing the product to diffuse out of the active site. Indeed, release of the actual product inorganic phosphate, originally derived from C-5 of dXP, gives a complex in which PNP is still bound but the loop is in the open conformation (6).

Open Conformation: Apo State. In the substrate-free, apo state, the active site residues coordinate five molecules of water which are found close to the dXP binding site, with none of the waters actually fitting into the positions of the substrate or product. In addition to the lack of equivalent water coordination, the apo subunit is in a more open conformation overall. Notably, there is looser packing between the apo subunit and its neighboring subunit, and the active site loop is fully open and crystallographically well defined.

Active Site Loop. The structure of PdxJ crystallized in the presence of dXP reveals several distinct loop conformations, most of which are clearly defined, as indicated from the electron density maps and refined B-factors of the residues at each loop region. The active site loop [“mobile loop 4” in the nomenclature of Franco et al. (6)] plays an important role in mediating solvent accessibility depending on its conformation. Correlating with, and perhaps even providing, the mechanism for the opening or closure of this loop is the occupancy of the active site. In the absence of substrates, the loop is seen to be flexible or open, allowing free access to the active site. Once one of the two substrates binds, the loop changes to an intermediate, partially closed conformation. The B-factors of the loop residues in this latter conformation are higher than the average B-factor for the rest of the protein, with an average of 45 over residues 95–105 versus 27.3 over the remainder of the 238 amino acids of this subunit, but are still within the acceptable range for structural characterization. The conformation of the mobile active site loop appears to be dependent solely on its interaction with the substrate, dXP, and largely independent of crystal packing effects; hence, the structure is likely to reflect its conformation in solution.

Isothermal Titration Calorimetry. Isothermal titration calorimetry (ITC) experiments were carried out to confirm that the dXP can bind to PdxJ in the absence of AHP, utilizing the same buffer conditions that were used to obtain crystals of PdxJ. The titration profile for addition of dXP to PdxJ resulted in a complex binding isotherm with more than one phase (data not shown). Fitting the isotherms by a least-squares analysis to a simple model consistent with interacting sites of binding gave a calculated Ka of (1.85 ± 0.13) × 10⁴ M⁻¹, corresponding to a Kd of 54 μM, which compares very favorably with the previously determined Km for dXP of 27 μM (4). It is possible that with increased saturation and optimization of the titration, e.g., longer equilibration times between injections, an additional binding phase(s) would be detected. The intricate nature of the data from these

11656  *Biochemistry*, Vol. 41, No. 39, 2002

preliminary titration experiments does suggest that the binding of dXP is too complex to be described by a single-binding site model and further experiments are needed to define the exact nature of binding.

Currently, no data are available to elucidate whether PdxJ exhibits a preferential sequence of substrate binding. However, the ITC results presented here confirm that PdxJ can bind dXP in the absence of AHP with an affinity on the order of the Michaelis constant for the biochemical reaction in the presence of AHP. The ITC data therefore indicate that the crystallographic results showing bound dXP do indeed reflect the true catalytically relevant states of pyridoxol phosphate synthase.

## CONCLUSION

The 1.96 Å structure of PdxJ cocrystallized with one of its substrates, dXP, and with Pi as a surrogate of the second substrate, reveals nonequivalency of active sites, with three distinct binding states being observed. Each subunit of the octamer has a unique conformation that can be correlated with the occupancy of the active site. In this new structure of PdxJ, two of the eight active sites are empty and the corresponding subunit is in the “open” state. Another four active sites have bound dXP and are in a partially open state. In these four subunits, the 5-phosphate group of the bound dXP bridges the sites that are occupied by the 5′-phosphate of PNP and inorganic phosphate (derived from the 5-phosphate of dXP) as revealed in the previously described closed PNP–Pi product structure (6). In the singly occupied sites, however, the pentulose sugar backbone of dXP is twisted away from the positions occupied by the corresponding carbons, C-2′, C-2, C-3, C-4, and C-4′ of the product PNP (Figure 3). The two remaining active sites of PdxJ have both dXP and inorganic Pi bound, and the relevant subunits are now in the closed conformation. In these subunits, the bound Pi occupies the site that corresponds to the 5′-OP of PNP in the “product” structure and therefore appears to be acting as a surrogate for the phosphate moiety of the natural cosubstrate, 1-amino-3-hydroxyacetone phosphate (AHP) (Figure 4). Binding of this Pi forces the phosphate group of dXP to slide laterally and the pentulose backbone to flip over, thereby adopting a conformation in which C-2, C-3, and C-4 more or less occupy the positions corresponding to the derived carbon atoms, C-2, C-3, and C-4 of PNP, respectively. The carbonyl group of dXP, however, is tipped up out of the plane of the incipient pyridoxine ring, which appropriately positions it for nucleophilic attack by the amino group of AHP on one face of the C-2 ketone carbonyl, leading to formation of the Schiff base. The C-3 hydroxyl of the dXP forms an H-bond with the side chain hydroxyl groups of two threonines, Thr102 and Thr103, both of which are located in the mobile active site protein loop, thereby locking down the loop itself. These interactions are reinforced by hydrogen bonding to an arginine, Arg20, which is itself brought into place by the bound Pi.

The binding of the first substrate, dXP, transforms the initially similar binding sites into three different, nonequivalent forms. Consequently, the model of catalysis and turnover that may be operative in PdxJ is analogous to that proposed for ATP synthase (21, 22). The F1 complex of ATP synthase has three nonequivalent adenine nucleotide binding sites, one

FIGURE 7: Occupancy of the active sites, at the interface of subunits A and E, can be correlated to the configuration of the active site loop, in red, the conformation of which controls access to the binding sites. Both the singly and doubly bound states are shown; the mechanism for formation of these different states may be through the binding of the substrate itself in that binding induces nonequivalency between the individual active sites. This figure was generated with MOLSCRIPT and Raster3D (24, 25).

for each pair of α and β subunits. At any given moment, one of these sites is in the β-ATP conformation, which binds ATP tightly. A second is found in the β-ADP conformation, which binds the nucleotide loosely, and the third is in the β-empty conformation. Similarly, in pyridoxol phosphate synthase, there are apparently three structurally and functionally distinct states. The simultaneous presence of all three states despite a saturating concentration of the dXP substrate suggests that PdxJ utilizes a cycle of substrate binding akin to the “binding change mechanism” proposed for ATP synthase. Consequently, the occupancies observed in the crystal may reflect catalytically relevant binding and modulation of substrate affinities once the first substrate is bound.

In PdxJ, binding of a ligand to one site appears to be influenced by whether ligands are bound to any of the other sites. Prior to any binding events, all sites on PdxJ appear to be open and equivalent; presumably, the octameric structure is already present. Once the first substrate binds, the remaining sites become nonequivalent, resulting in three states which can be related to the occupancy of the active site and the conformation of the enzyme loop that modulates or controls access to the active site (Figure 7). In summary, this newly reported structure of PdxJ provides, in effect, snapshots of the enzyme in three stages of the catalytic mechanism: (1) a resting, open state, (2) a partially open state with bound dXP, and (3) a closed state with bound dXP, positioned to react, and bound inorganic phosphate, the latter acting as a surrogate for the cosubstrate aminohydroxyacetone phosphate.

## ACKNOWLEDGMENT

J.I.Y. thanks Dr. Mary Walsh for access to instrumentation and technical advice for the isothermal titration calorimetry experiments.

Structure of the Pdxj Complex

Biochemistry, Vol. 41, No. 39, 2002    11657

SUPPORTING INFORMATION AVAILABLE

Movie showing a plausible PdxJ reaction cycle, modeled from structural data (can be viewed using any browser software). This material is available free of charge via the Internet at http://pubs.acs.org.

REFERENCES

1. Roa, B. B., Connolly, D. M., and Winkler, M. E. (1989) Overlap between pdxA and ksgA in the complex pdxA-ksgA-apaG-apaH operon of *Escherichia coli* K-12. *J. Bacteriol.* 171, 4767–4777.
2. Lam, H.-M., Tancula, E., Dempsey, W. B., and Winkler, M. E. (1992) Suppression of insertions in the complex pdxJ operon of *Escherichia coli* K-12 by lon and other mutations. *J. Bacteriol.* 174, 1554–1567.
3. Cane, D. E., Hsiung, Y., Cornish, J. A., Robinson, J. K., and Spenser, I. D. (1998) Biosynthesis of Vitamin B6: The Oxidation of 4-(Phosphohydroxy)-L-threonine by PdxA. *J. Am. Chem. Soc.* 120, 1936–1937.
4. Cane, D. E., Du, S., Robinson, J. K., Hsiung, Y., and Spenser, I. D. (1999) Biosynthesis of Vitamin B6: Enzymatic Conversion of 1-Deoxy-D-xylulose-5-phosphate to Pyridoxal Phosphate. *J. Am. Chem. Soc.* 121, 7722–7723.
5. Cane, D. E., Du, S., and Spenser, I. D. (2000) Biosynthesis of Vitamin B6: Origin of the Oxygen Atoms of Pyridoxol Phosphate. *J. Am. Chem. Soc.* 122, 4213–4214.
6. Franco, M. G., Laber, B., Huber, R., and Clausen, T. (2001) Structural Basis for the Function of Pyridoxine 5′-Phosphate Synthase. *Structure* 9, 245–253.
7. Otwinowski, Z., and Minor, W. (1997) Processing of X-ray diffraction data collected in oscillation mode. *Methods Enzymol.* 276, 307–326.
8. Yeh, J. I., and Hol, W. G. J. (1998) A flash-annealing technique to improve diffraction limits and lower mosaicity in crystals of glycerol kinase. *Acta Crystallogr. D58*, 479–480.
9. Wang, B. C. (1985) Resolution of Phase Ambiguity in Macromolecular Crystallography. *Methods Enzymol.* 115, 90–112.
10. Liu, Y.-D., Gu, Y.-X., Zheng, C.-D., Hao, Q., and Fan, H.-F. (1999) Combining direct methods with isomorphous replacement or anomalous scattering data. VIII. Phasing experimental SIR data with the replacing atoms in a centrosymmetric arrangement. *Acta*

Crystallogr. D55, 846–848.
11. Read, R. J. (2001) Pushing the boundaries of molecular replacement with maximum likelihood. *Acta Crystallogr. D57*, 1373–1382.
12. Brünger, A. T. (1992) *XPLORE (Version 3.1) Manual*, Yale University Press, New Haven, CT.
13. Navaza, J. (2001) Implementation of molecular replacement in AMoRe. *Acta Crystallogr. D57*, 1367–1372.
14. Collaborative Computational Project Number 4 (1994) The CCP4 suite: programs for protein crystallography. *Acta Crystallogr. D50*, 760–763.
15. Kleywegt, G. J. (1999) Experimental assessment of differences between related protein crystal structures. *Acta Crystallogr. D55*, 1878–1884.
16. Kleywegt, G. J. (2000) Validation of protein crystal structures. *Acta Crystallogr. D56*, 249–265.
17. Read, R. J. (1986) Improved Fourier coefficients for maps using phases from partial structures with errors. *Acta Crystallogr. A42*, 140–149.
18. Wiseman, T., Williston, S., Brandts, J. F., and Lin, L. N. (1989) Rapid measurement of binding constants and heats of binding using a new titration calorimeter. *Anal. Biochem.* 179, 131–137.
19. Jones, T. A., Zou, J. Y., Cowan, S. W., and Kjeldgaard, M. (1991) Improved methods for building protein models in electron density maps and the location of errors in these models. *Acta Crystallogr. A47*, 110–119.
20. Schneider, T. R. (2000) Objective comparison of protein structures: error-scaled difference distance matrices. *Acta Crystallogr. D56*, 714–721.
21. Stock, D., Leslie, A. G. W., and Walker, J. E. (1999) Molecular Architecture of the Rotary Motor in ATP Synthase. *Science* 286, 1700–1705.
22. Boyer, P. D. (1997) The ATP Synthase: A Splendid Molecular Machine. *Annu. Rev. Biochem.* 66, 717–749.
23. Jones, T. A. (1978) A graphics model building and refinement system for macromolecules. *J. Appl. Crystallogr.* 11, 268–272.
24. Kraulis, P. J. (1991) MOLSCRIPT: a program to produce both detailed and schematic plots of protein structures. *J. Appl. Crystallogr. A47*, 392–400.
25. Merritt, E. A., and Bacon, D. J. (1997) Raster3D: Photorealistic Molecular Graphics. *Methods Enzymol.* 277, 505–524.
BI026292T
