
THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2002 by The American Society for Biochemistry and Molecular Biology, Inc.

**Biochemical Evidence for an Editing Role of Thioesterase II in the Biosynthesis of the Polyketide Pikromycin**

Received for publication, July 31, 2002, and in revised form, October 2, 2002  
Published, JBC Papers in Press, October 3, 2002, DOI 10.1074/jbc.M207770200

Beom Seok Kim‡§, T. Ashton Cropp‡, Brian J. Beck¶, David H. Sherman¶, and Kevin A. Reynolds‡||

From the ‡Department of Medicinal Chemistry and Institute for Structural Biology and Drug Discovery, Virginia Commonwealth University, Richmond, Virginia 23219 and ¶Department of Microbiology and Biotechnology Institute, University of Minnesota, Minneapolis, Minnesota 55455

The pikromycin biosynthetic gene cluster contains the *pikAV* gene encoding a type II thioesterase (TEII). TEII is not responsible for polyketide termination and cyclization, and its biosynthetic role has been unclear. During polyketide biosynthesis, extender units such as methylmalonyl acyl carrier protein (ACP) may prematurely decarboxylate to generate the corresponding acyl-ACP, which cannot be used as a substrate in the condensing reaction by the corresponding ketosynthase domain, rendering the polyketide synthase module inactive. It has been proposed that TEII may serve as an “editing” enzyme and reactivate these modules by removing acyl moieties attached to ACP domains. Using a purified recombinant TEII we have tested this hypothesis by using *in vitro* enzyme assays and a range of acyl-ACP, malonyl-ACP, and methylmalonyl-ACP substrates derived from either PikAIII or the loading domain of DEBS1 (6-deoxyerythronolide B synthase; AT<sub>L</sub>-ACP<sub>L</sub>). The pikromycin TEII exhibited high *K<sub>m</sub>* values (>100 μM) with all substrates and no apparent ACP specificity, catalyzing cleavage of methylmalonyl-ACP from both AT<sub>L</sub>-ACP<sub>L</sub> (*k<sub>cat</sub>/*K<sub>m</sub> 3.3 ± 1.1 M<sup>−1</sup> s<sup>−1</sup>) and PikAIII (*k<sub>cat</sub>/*K<sub>m</sub> 2.9 ± 0.9 M<sup>−1</sup> s<sup>−1</sup>). The TEII exhibited some acyl-group specificity, catalyzing hydrolysis of propionyl (*k<sub>cat</sub>/*K<sub>m</sub> 15.8 ± 1.8 M<sup>−1</sup> s<sup>−1</sup>) and butyryl (*k<sub>cat</sub>/*K<sub>m</sub> 17.5 ± 2.1 M<sup>−1</sup> s<sup>−1</sup>) derivatives of AT<sub>L</sub>-ACP<sub>L</sub> faster than acetyl (*k<sub>cat</sub>/*K<sub>m</sub> 4.9 ± 0.7 M<sup>−1</sup> s<sup>−1</sup>), malonyl (*k<sub>cat</sub>/*K<sub>m</sub> 3.9 ± 0.5 M<sup>−1</sup> s<sup>−1</sup>), or methylmalonyl derivatives. PikAIV containing a TEI domain catalyzed cleavage of propionyl derivative of AT<sub>L</sub>-ACP<sub>L</sub> at a dramatically lower rate than TEII. These results provide the first unequivocal *in vitro* evidence that TEII can hydrolyze acyl-ACP thioesters and a model for the action of TEII in which the enzyme remains primarily dissociated from the polyketide synthase, preferentially removing aberrant acyl-ACP species with long half-lives. The lack of rigorous substrate specificity for TEII may explain the surprising observation that high level expression of the protein in *Streptomyces venezuelae* leads to significant (>50%) titer decreases.

---

Polyketides are a large and structurally diverse class of natural products that possess a wide range of biological activities (1). These compounds are used throughout medicinal and agricultural fields as antimicrobials, immunosuppressants, antiparasitics, and anticancer agents. Despite their structural diversity, polyketides are assembled by a common mechanism of decarboxylative condensations of simple malonate derivatives by polyketide synthases (PKSs)<sup>1</sup> in a manner very similar to fatty acid biosynthesis (2, 3). Type I PKSs are a family of PKSs that are analogous to vertebrate fatty acid synthase that catalyze the biosynthesis of the polyketide moieties of various secondary metabolites in *Streptomyces* (4–6). They are gigantic multifunctional modular proteins. Each module is responsible for one cycle of polyketide chain elongation and contains a set of discrete catalytic domains of ketosynthase (KS), acyltransferase (AT), and acyl carrier protein (ACP). Ketoreductase, dehydratase (DH), and enoyl reductase domains may also be present, allowing structural variation in the level of processing of the β-ketoacyl chain (4, 6). The fully extended polyketide chain bound to the PKS as an acyl-ACP thioester is often released and cyclized by a thioesterase domain (TEI) covalently linked to the last extending module of the PKS (7).

In many cases, additional genes encoding a TE have been found within a polyketide biosynthetic gene cluster, for example, the tylosin PKS of *Streptomyces fradiae* (8), pikromycin PKS of *Streptomyces venezuelae* (5), rifamycin PKS of *Amycolatopsis mediterranei* (9), and the erythromycin PKS (DEBS) of *Saccharopolyspora erythraea* (10). These genes encoding a discrete protein were named as TEII to differentiate from the chain releasing TEI domains in modular polyketide synthases. Discrete TEII enzymes are also associated with bacterial nonribosomal peptide synthases, responsible for the production of macrocyclic peptide compounds (11), and animal fatty acid synthases (12). Sequence analysis has revealed that these thioesterases are probably structurally and evolutionarily related (13). They have a common serine esterase motif, GXSXG, ~100 residues from the N terminus and another conserved downstream motif, GXH, of which histidine might accept a proton from the hydroxyl group of the active site serine, thereby increasing its nucleophile character (13). Because TEI domains have been shown to be a necessary and sufficient factor for the release and cyclization of the polyketide chain *in vivo* and *in vitro* (7, 14–16), the role of the TEII enzyme encoded within many PKS gene clusters has presented an intriguing question.

---

* This research was supported by National Institute of Health Grant GM48562. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
* § Recipient of a Korean Science and Engineering Foundation (KOSEF) Postdoctoral Research Fellowship.
* || To whom correspondence should be addressed: Institute for Structural Biology and Drug Discovery, 800 East Leigh St., Richmond, VA 23219. Tel.: 804-828-5679; Fax: 804-827-3664; E-mail: kareynol@hsc.vcu.edu.

---

<sup>1</sup> The abbreviations used are: PKS, polyketide synthase; PikPKS, pikromycin (Pik) polyketide synthase; DEBS, 6-deoxyerythronolide B synthase; KS, ketosynthase; AT, acyltransferase; ACP, acyl carrier protein; TE, thioesterase; DTT, dithiothreitol; HPLC, high performance liquid chromatography.

48028 This paper is available on line at http://www.jbc.org

normal condensation reaction. The polyketide biosynthetic process would thus be derailed, resulting in low yields of the fully extended polyketide product (24). TEII-catalyzed cleavage of these ACP-bound acyl residues would allow the activated dicarboxylic acid extender units to be loaded onto the PKS, restoring polyketide production. Consistent with such a role is the observation that the homologous TEII and TEI from an animal fatty acid synthases have substrate specificity for medium length fatty acid chains (C8, C10, and C12) and longer chain fatty acids (C14, C16, and C18), respectively (25). Recent studies on TEII (*tylO*) of the TylPKS show that it could hydrolyze synthetic acyl (acetyl-, propionyl-, and butyryl-)-*p*-nitrophenyl esters and *N*-acetylcysteamine thioesters, providing additional preliminary support for the proposed editing role of TEII (24).

In this study, we have generated recombinant TEII (*PikAV*) from *S. venezuelae* and performed the first catalytic study of such an enzyme with physiologically relevant substrates, various acyl- and carboxylated acyl-ACP thioesters. The pikromycin TEII exhibits a very high $K_m$ for all ACP substrates and likely is predominantly dissociated from the PKS, minimizing its potential to block the normal biosynthetic process. TEII is active with a wide range of ACP-bound thioesters, consistent with an editing role for all of the modules within a PKS. However, this lack of rigorous substrate specificity allows the TEII to remove acyl-ACP thioesters in the PKS-loading module required to initiate the process or the activated carboxylated acyl-ACP thioesters in the extension modules. Thus, either an excess or lack of TEII may lead to decreases in polyketide production. *In vivo* experiments with overexpression of *pikAV* in pikromycin-producing cultures of *S. venezuelae* are shown to be consistent with this observation.

EXPERIMENTAL PROCEDURES

Materials and General Methods—[1-¹⁴C]Acetyl-CoA (53 mCi/mmol, 50 μCi/ml), [1-¹⁴C]propionyl-CoA (55 mCi/mmol, 50 μCi/ml), [1-¹⁴C]butyryl-CoA (53 mCi/mmol, 100 μCi/ml), and [2-¹⁴C]malonyl-CoA (55 mCi/mmol, 20 μCi/ml) were purchased from Movarek Biochemicals. DL-2-[2-¹⁴C]methylmalonyl-CoA (60 mCi/mmol, 10 μCi/ml) was purchased from American Radiolabeled Chemicals, Inc. All other chemicals were purchased from Sigma and were of the highest available grade. Standard molecular cloning techniques were employed (26). PCR amplifications were performed using standard PCR strategies with *Pfu* Turbo DNA polymerase (Stratagene) (27).

Protein Expression and Purification—TEII was obtained by expression of *pikAV* of *S. venezuelae* in *Escherichia coli*. The *pikAV* gene (GenBank™ accession number AF79138: 36989–37834) was amplified from a genomic library of *S. venezuelae* by PCR using the following primers designed with *Nde*I and *Hind*III sites at the 5′ and 3′ ends, respectively (5′-AAGCGGGCATATGACCGACAGACCTCTGAA-3′ and 5′-TCGTAAGCTTCCGTGGGGTTCTGCCATCT-3′). The PCR product was ligated into the pET21c expression vector to give pBK18, which was maintained in *E. coli* TG2. *E. coli* BL21-CodonPlus-RP was used as a host for the expression of PikAV. Overexpression (2–4 mg/liter) was accomplished by induction using 0.8 mM isopropyl-1-thio-β-*d*-galactopyranoside at $A_{600} = 0.5–0.6$ and incubation of the culture at 28°C overnight. Cells were harvested by centrifugation at 3,000 × *g* and resuspended with buffer A (50 mM Tris (pH 8.0), 2.5 mM dithiothreitol (DTT), 10% glycerol). After disruption of the cells using French press at 10,000 p.s.i., the lysate was clarified by centrifugation at 10,000 × *g*. The supernatant was loaded onto nickel nitrilotriacetic acid column previously equilibrated with 50 mM Tris (pH 8.0), and washed with 10 mM imidazole in the same buffer. The pikromycin TEII was eluted with buffer A containing 100 mM imidazole. Fractions containing the protein were pooled and concentrated with a Centricon™ (Amicon, Inc.) concentrator and subsequently diluted with buffer A containing 5 mM imidazole. The concentrated protein was further purified using a HiTrap chelating high performance column (5 ml, Amersham Biosciences) with a gradient elution from 5 to 250 mM imidazole in buffer A. The purification was monitored spectrophotometrically by absorbance at 280 nm. The pooled peak fractions were concentrated, and buffer was subsequently changed to buffer B (100 mM Na₂HPO₄ (pH 7.2), 2.5 mM

48030

**Biochemical Evidence for an Editing Role of Thioesterase II**

DTT, 1 mM EDTA, 20% glycerol). The concentration of the resulting pure TEII solution was determined using the dotMETRIC™ assay kit from Bioworld.

The last module of PikPKS, PikAIV, containing a TEI domain was expressed in *E. coli*. After several steps of PCR and subcloning, the pikAIV gene (GenBank™ accession number AF79138: 32952–36992) was cloned into pET24b as an *NdeI* and *HindIII* fragment to give pDHS4188, which expresses *pikAIV* as a C-terminal His-tagged fusion protein. The plasmid pDHS4188 was transferred into BL21(DE3) strain harboring the *sfp* gene (pRSG56) (28) and induced by 0.1 mM isopropyl-1-thio-β-D-galactopyranoside at 0.6 absorbance. After overnight culture at 25 °C, the cells were harvested and disrupted using a French press at 10,000 p.s.i. The PikAIV-TEI was expressed at level of 8–12 mg/liter and purified as described above for TEII. The protein (2.5 mg/ml) was kept in 100 mM NaH₂PO₄ (pH 7.3), 1 mM EDTA, 0.2 mM DTT, 10% glycerol after dialysis in the same buffer. The activity of TEII and TEI was confirmed by hydrolysis assay using the *p*-nitrophenyl esters of acetate, propionate, and butyrate in sodium phosphate buffer (pH 7.5), 100 μM EDTA as described previously (24). The substrates were dissolved into assay buffer using dimethyl sulfoxide.

To obtain the loading domain (AT_L-ACP_L) of DEBS1 with a C-terminal hexahistidine tag, the corresponding gene (GenBank™ accession number 63677) was cloned as an *XbaI-HindIII* fragment into pET21c (Novagen) overexpression vector to give pBK12. The PCR amplification was from a DEBS1 construct (pBK3) (29) using the primers 5′-GAATTCGAGCATATGGACGCGTGCGGACC-3′ and 5′-TTCGTTAAGCTT-GCGGGTTTCCCGTTG-3′. For expression of the holo-AT_L-ACP_L didomain, pBK12 was introduced into *E. coli* BL21-CodonPlus-RP (Stratagene) containing the plasmid pRSG56, which carries a kanamycin gene and the *sfp* gene. BL21-CodonPlus-RP/pBK12 was cultured in LB supplemented with 50 mg/ml kanamycin and 100 mg/ml ampicillin at 37 °C. When the growth reached A₆₀₀ = 0.5–0.6, expression of the didomain (16–18 mg/liter) was induced by 0.8 mM isopropyl-1-thio-β-D-galactopyranoside followed by 12 h of incubation at 30 °C. Cells were harvested by centrifugation at 3000 × g and resuspended in 50 mM Tris (pH 8.0). The resuspended cells were lysed by a French press at 10,000 p.s.i. and pelleted by centrifugation at 10,000 × g. The published purification procedure was then used to obtain 8.6 mg/ml concentration of holo-AT_L-ACP_L (30).

The *pikAIll* gene (GenBank™ accession number AF79138: 28161–32849) was cloned into pET28b as an *NdeI-HindIII* fragment after multi-step PCR reactions and subcloning steps. The resulting construct (pDHS4405) was used to express PikAIll (8–12 mg/liter) as a fusion protein with an N-terminal His tag in *E. coli*. Expression and purification were analogous to that described for PikAIV.

**Autoradiography Demonstrating Deacylation of Propionyl AT_L-ACP_L by TEII—** To generate a radiolabeled acyl-AT_L-ACP_L, holo-AT_L-ACP_L (50 μM) was incubated at room temperature in a reaction mixture containing 180 μM [1-¹⁴C]propionyl-CoA, 100 mM NaH₂PO₄ (pH 7.2), 2.5 mM DTT, 1 mM EDTA, and 20% glycerol. After a 45-min incubation, the reaction mixture was concentrated, and the excess propionyl-CoA was removed by exchanging buffer using a Centriprep™ concentrator (Amicon). The radiolabeled propionyl-AT_L-ACP_L (20 μM) was incubated with 5 μM TEII in 50 mM NaH₂PO₄ (pH 8.0) at room temperature for 6 h. A control incubation in the absence of TEII was carried out simultaneously. The reaction was quenched by the addition of 10% trichloroacetic acid and incubated for 30 min in ice. The proteins were pelleted and rinsed with acetone. The pellet was redissolved into 20 μl of protein gel-loading buffer. The mixture was loaded onto a 1.5-mm 12% polyacrylamide gel and run at 70 V. When the marker dye reached the bottom of the gel, it was stained, destained, and dried on filter paper. The dried gel was subjected to PhosphorImager analysis (Molecular Dynamics) for 48 h.

**Enzyme Assays—** The enzyme activity of TEII was measured on several radiolabeled acyl-ACP substrates. Typical reactions to make radiolabeled acyl-ACPs contained 50 μM either AT_L-ACP_L or PikAIll in a mixture of 1:9 or 1:5 radiolabeled and nonradiolabeled acyl-CoA, malonyl-CoA, or methylmalonyl-CoA (1 mM), 100 mM NaH₂PO₄ (pH 7.2), 2.5 mM DTT, 1 mM EDTA, and 20% glycerol. After a 1–3-h incubation, the reaction mixture was concentrated, and the buffer was exchanged to 50 mM phosphate buffer (pH 8.0) using a Centriprep™ concentrator. A time course experiment was performed using a reaction mixture (25 μl) containing 5.7 μM TEII or PikAIV (containing TEI), 100 μM propionyl-AT_L-ACP_L, 50 mM phosphate (pH 8.0). For kinetic analyses, a range of concentrations (5–150 μM) of each acyl-ACP and carboxylated acyl-ACP substrate was incubated with 1.4 μM TEII or PikAIV-TEI in 50 mM phosphate buffer (pH 8.0, final volume 25 μl) at room temperature for 40 min (the reaction rate was linear over 40 min; see Fig. 4A). Triplicate

FIG. 2. SDS-PAGE analyses of TEII (PikAV) and PikAIV-TEI. Samples from the expression and purification procedures were analyzed by SDS-PAGE and visualized with Coomassie Brilliant Blue. Experiments were performed as described under “Experimental Procedures.” A, recombinant TEII expressed from *pikAV*. First lane, soluble fraction from noninduced cells; second lane, soluble fraction from induced cells; third lane, purified TEII protein. B, recombinant PikIV-TEI protein expressed from *pikIV*. First lane, soluble fraction from noninduced cells; second lane, soluble fraction from induced cells; third lane, purified PikIV-TEI protein.

analyses were carried out for each concentration. The reactions were quenched by 100 μl of 10% trichloroacetic acid, and the proteins were pelleted by centrifugation at 10,000 × g. Each pellet was rinsed with an excess of 10% trichloroacetic acid and dissolved into 200 μl of SDS solution (2% SDS, 20 mM NaOH). The protein suspension was mixed with 5 ml of liquid scintillation fluid, and the radioactivity was quantified by liquid scintillation counting. The loss of radioactivity compared with the control reaction without TEII/PikAIV-TEI was used to calculate the amount of hydrolyzed acyl chain from the ACP domain.

**Effect of High Level Expression of TEII on Aglycone Production in S. venezuelae—** The TEII gene (*pikAV*) was PCR-amplified as a *XbaI* and *HindIII* fragment and cloned into a high copy plasmid pSE34, a derivative of pWHM3 with the *ermE* promoter (31), to give pSC70. The *pikAV* gene was also cloned into a derivative of pDHS702 (16) (a low copy number plasmid based on SCP2* origin of replication) for expression of TEII under the control of the *pikAI* promoter. Expression of the TEII from the resulting pSC38 plasmid resembles a natural expression pattern of *pikA* cluster in *S. venezuelae*. A site-directed mutation was introduced to pSC70 to produce an expression plasmid of inactive TEII protein (pBK56). Ser-99 was changed to Ala using QuikChange™ XL mutagenesis kit (Stratagene) and primers 5′-CGGGCACGCCCTCGGCGCTAGCGTCGCCTTCGAGACG-3′ and 5′-CGTCTCGAAAGGCGACGCCTAGCGCCGGAGGGCGT GCCCG-3′ (the alanine codon is underlined). The TEII expression plasmids are shuttle vectors having a ColE *E. coli* origin of replication. They were introduced to *S. venezuelae* SC1022 strain, which has an in-frame deletion of 687 bp of *pikAV* (32). Each of the transformants was cultured in 50 ml of SCM medium (20) supplemented with thiostrepton and kanamycin at 25 μg/ml in a 250 ml flask. After 3 days of incubation at 30 °C, the culture broth was clarified by centrifugation at 4000 × g, and the supernatant was extracted with chloroform twice. The aglycone production was quantified by HPLC using a C18 column with monitoring at 230 nm as described previously (20).

---

**RESULTS**

**Generation of Recombinant TEII and Acyl-ACP Substrates—** The TEII protein encoded by *pikAIV* was expressed from plasmid pBK18 in *E. coli* as a fusion protein tagged with a C-terminal hexahistidine and purified to homogeneity using Ni²⁺ affinity column chromatography (Fig. 2). The recombinant protein was stable in 50 mM phosphate buffer with 1 mM EDTA during the purification procedures. Overnight dialysis or high salt concentration buffer, however, resulted in partial precipitation of the TEII. The esterase activity of the TEII was confirmed by hydrolysis assay using *p*-nitrophenyl ester derivatives (data not shown), which are well known substrates for serine proteases and have been used for thioesterase studies in the animal fatty acid synthase (33) and the TEII associated with the tylosin PKS (24). The TEII showed significant hydrolytic rates with acetyl, propionyl, and butyryl esters. However, the substrate specificity of TEII with these model systems could not readily be addressed due to high background hydrolysis rates and limited solubility of the substrates under the

Biochemical Evidence for an Editing Role of Thioesterase II

**FIG. 3. Hydrolysis of AT<sub>L</sub>-ACP<sub>L</sub> thioester of \[^{14}C\]propionate by the TEII.** The thioesterase activity of the TEII was confirmed by autoradiography using a radiolabeled acyl-ACP substrate. The radiolabeled propionyl-AT<sub>L</sub>-ACP<sub>L</sub> (20 μM) was incubated with 5 μM TEII in 50 mM NaH<sub>2</sub>PO<sub>4</sub> (pH 8.0) at room temperature for 6 h. After quenching the reaction, the pelleted proteins were redissolved into 20 μl of protein gel loading buffer and loaded onto a 1.5-mm 12% polyacrylamide gel and electrophoresed at 70 V. A, SDS-PAGE analysis visualized by Coomassie staining shows 59-kDa AT<sub>L</sub>-ACP<sub>L</sub> and 32-kDa TEII. First lane, reaction mixture without TEII; second lane, the reaction mixture with TEII. B, autoradiography after the exposure of the SDS-PAGE gel to PhosphorImager for 48 h. First lane, reaction mixture without TEII; second lane, the reaction mixture with TEII.

assay conditions. Similar observations have been made with these substrates using the TEII (TylO) encoded within the tylosin biosynthetic gene cluster (24).

To further probe the role of TEII in polyketide biosynthesis, we examined catalytic activity with a more physiologically relevant substrate, a PKS ACP thioester of monocarboxylic acid, as might be generated by aberrant decarboxylation of malonate or malonate derivatives bound to ACP domain. We selected the loading domain of DEBS (AT<sub>L</sub>-ACP<sub>L</sub>) because this is known to have relaxed substrate specificity accepting acetyl-, propionyl-, butyryl-, malonyl-, and methylmalonyl-CoA (30), allowing a clear comparison of TEII hydrolysis of thioesters of these groups on the same ACP.

The gene encoding the AT<sub>L</sub>-ACP<sub>L</sub> didomain region of DEBS1 was PCR amplified as an NdeI and HindIII fragment and cloned into an expression vector pET21c to give pBK12. The didomain fragment includes the N-terminal 107 extra amino acids that have been shown recently to be essential for activity of this didomain (30). The gene encoding the Sfp (a phosphopantetheine transferase from *Bacillus subtilis*) was co-expressed to ensure posttranslational modification of the apo-AT<sub>L</sub>-ACP<sub>L</sub> to the holo form (28). The holo-AT<sub>L</sub>-ACP<sub>L</sub> was expressed as a C-terminal hexahistidine-tagged fusion protein and purified by nickel affinity chromatography to 99% purity.

The AT<sub>L</sub>-ACP<sub>L</sub> was incubated with \[^{14}C\]propionyl-CoA, leading to rapid acylation. Other acyl-AT<sub>L</sub>-ACP<sub>L</sub> derivatives were also generated in the same reaction mixture using different radiolabeled CoA derivatives (acetyl-, butyryl-, malonyl-, and methylmalonyl-CoA). Prolonged incubations (up to 3 h) ensured that greater than 95% of the protein was converted to the thioester, even with substrates such as malonyl-CoA.

**Thioesterase Activity of TEII on Propionyl-AT<sub>L</sub>-ACP<sub>L</sub>—** The radiolabeled propionyl-AT<sub>L</sub>-ACP<sub>L</sub> was incubated at room temperature with or without TEII for 6 h. After trichloroacetic acid precipitation, the reaction mixture was analyzed by SDS-PAGE and autoradiography. The reaction without TEII showed a clear radioactive protein band at the position of AT<sub>L</sub>-ACP<sub>L</sub> protein, whereas the corresponding band was barely detected in the reaction containing TEII protein (Fig. 3), indicating hydrolysis of the radioactive propionyl residue from the protein. A kinetic analysis of this phenomenon using liquid scintillation counting showed that the loss of radioactivity of propionyl-AT<sub>L</sub>-ACP<sub>L</sub> was proportionally correlated with the TEII concentration, indicating the deacylation is the result of enzyme reaction by TEII. These results provide the first clear evidence of thioesterase activity of TEII on the acyl thioester of a PKS ACP domain, as might be generated by aberrant decarboxylation of extending units during polyketide biosynthesis.

A comparison of the thioesterase activity of Pikromycin TEII was made with that of TEI at the C terminus of PikAV. The pikromycin PikAV-TEI was overexpressed in *E. coli* and purified as a C-terminal His-tag fusion protein (Fig. 2B). Numerous in vivo and in vitro studies have previously shown that an equivalent TEI domain at the end of the erythromycin PKS can catalyze hydrolysis and cyclization reactions with shorter chain acyl-ACPs than the natural length of acyl-ACP (34, 35). A time course experiment comparing catalytic activity of TEII and the PikAV-TEI multifunctional protein revealed that the propionyl group was removed from the ACP domain more rapidly by the TEII (Fig. 4A). The concentration dependence of the propionyl-AT<sub>L</sub>-ACP<sub>L</sub> hydrolysis by TEII and PikAV-TEI (Fig. 4B) was determined. TEII removed propionyl groups from the ACP domain about 8 times more efficiently than the TEI.

**Substrate Specificity of TEII—** More quantitative information.

48032

**Biochemical Evidence for an Editing Role of Thioesterase II**

![Velocity (nM min⁻¹)](https://i.imgur.com/1.png)
Substrate concentration (µM)

FIG. 5. Thioesterase activity of TEII on short-chain acyl-ATₗ-ACPₗ thioesters. TEII-catalyzed hydrolysis rates were measured at increasing concentrations of various short chain acyl-ACP and carboxylated acyl-ACP groups. A series of concentrations of the radiolabeled acetyl (open circles)-, propionyl (closed circles)-, butyryl (open triangles)-, and malonyl (closed triangles)-ATₗ-ACPₗ were incubated with 1.4 µM TEII in 50 mM phosphate buffer (pH 8.0, final volume 25 µl) respectively. The reactions were measured for loss of radioactivity by liquid scintillation counting.

Information concerning the substrate specificity of TEII was obtained by calculation of $k_{cat}/K_m$ for several acyl-ACP substrates such as propionyl-, acetyl- and butyryl-ATₗ-ACPₗ (potentially generated by decarboxylation of the malonate-derived extender units such as methyl malonate, malonate, or ethyl malonate bound to an ACP domain). All three acyl-ACP substrates tested were hydrolyzed by the TEII at a significant rate (Fig. 5). In all cases the high $K_m$ for the substrates (all reaction rates exhibited a linear dependence on substrate concentration to 150 µM) coupled with their limited solubility precluded us from obtaining true saturation kinetics. Instead, the apparent $k_{cat}/K_m$ was used for the comparison of enzyme efficiency and specificity and calculated on the assumption that at substrate concentrations significantly below the $K_m$ most of the enzyme is free ($[E]_t$ is approximately $[E]$). Under these circumstances, the ratio $k_{cat}/K_m$ behaves as a second-order rate constant for the reaction between substrate and free enzyme ($k_{cat}/K_m = V/[E][S]$). The catalytic efficiency for butyryl- and propionyl-ACP was 17.5 ± 2.1 and 15.8 ± 1.8 M⁻¹s⁻¹, respectively. The acetyl-ACP was a poorer substrate under these conditions (4.9 ± 0.7 M⁻¹s⁻¹). Carboxylated acyl substrates were also examined for the first time with a TEII. Because these are the correct substrate for PKS elongation modules, we expected to see significant substrate discrimination by TEII. The malonyl and methylmalonyl derivatives of the ATₗ-ACPₗ (3.9 ± 0.5 and 3.3 ± 1.1 M⁻¹ s⁻¹, respectively) were hydrolyzed less efficiently than the propionyl and butyryl derivatives but were substrates nonetheless.

**ACP Specificity of TEII—**The use of the DEBS PKS ATₗ-ACPₗ-loading domain and its established broad substrate specificity allowed comparison of the relative activities of pikromycin TEII of a range of acyl and carboxylated substrates bound to the same ACP. To investigate the ACP specificity of the pikromycin TEII, the hydrolysis rate of the methylmalonyl thioesters from DEBS ATₗ-ACPₗ-loading domain and PikAIII (a single modular protein of PikPKSs and, thus, a potential physiological substrate for TEII) were compared (Fig. 6). The TEII showed significant and indistinguishable $k_{cat}/K_m$ (3.3 ± 1.1 and 2.9 ± 0.9 M⁻¹s⁻¹ for methylmalonyl thioesters of ATₗ-ACPₗ and PikAIII, respectively).

![Velocity (nM min⁻¹)](https://i.imgur.com/2.png)
Substrate concentration (µM)

FIG. 6. ACP specificity of the pikromycin TEII. Hydrolysis rates of methylmalonyl-ACP thioesters by TEII were determined using the loading domain ACP of DEBS1 (open circles) and the extension module ACP of PikAIII (closed circles). Reaction conditions were as described for Fig. 5.

**Effect of TEII Expression on Aglycone Production in Pikromycin PKS—**We carried out a complementation study of TEII in *S. venezuelae* strain SC1022, which has an in-frame deletion of pikAV. In previous studies, this strain has been shown to produce only aglycones, as the deleted region of *pikAV* contains a transcription unit essential for expression of the *des* genes responsible for the production of the final glycosylated products (32). The titer of the polyketide products was apparently not affected by deletion of the TEII gene (yields were comparable with that observed for a *desVI* mutant) (20). Complementation of strain SC1022 with the TEII gene expressed from the pikAI promoter in a low copy number (pSC38) plasmid had no effect upon aglycone levels (10-deoxymethynolide and narbonolide levels of 9 ± 2 and 49 ± 4 mg/liter), comparable with that observed with SC1022. Complementation of SC1022 with pSC70 (a derivative of the high copy number plasmid pSE34, which expresses TEII constitutively from the *ermE* promoter) led to a consistent and dramatic 50–70% decrease in aglycone production (4 ± 3 mg/liter 10-deoxymethynolide and 12 ± 8 mg/liter narbonolide, respectively) (Fig. 7). No suppression of aglycone production was observed with SC1022 carrying either pBK56, a pSE34 derivative expressing the S99A TEII mutant, or pSE34.

---

**DISCUSSION**

KS enzymes catalyze the elongation step in the biosynthesis of fatty acids and polyketides in a multi-step process involving decarboxylation of the elongation unit (typically malonyl- or methylmalonyl-ACP) and attack of the resulting carbanion on an acyl group covalently bound to the active site cysteine residue (36). Evidence indicates that the decarboxylation reaction is most efficient when its active site cysteine is thus modified, thereby minimizing nonproductive decarboxylation of the activated extender units (36). Nonetheless, studies have shown that decarboxylation of malonyl-ACP without prior acylation of the active site, so-called aberrant decarboxylation, could take place, albeit at a lower rate. The KS domain of the yeast fatty acid synthase and the KS components of type II fatty acid synthase system can catalyze malonyl-ACP decarboxylation in the absence of an acyl-enzyme intermediate (37, 38). Examples of extender unit decarboxylation in the absence of the acylated KS have also been seen in modular PKSs. Under normal conditions, DEBS3 from *S. erythraea* can catalyze the last two extending reactions in 6-deoxyerythronolide B biosynthesis us-

Biochemical Evidence for an Editing Role of Thioesterase II 48033

than observed for intermolecular acyl-ACP cleavage by PikAIV-TE in the current study (7).

As an editing enzyme, TEII would need to scan all of the ACP domains within a given PKS and would not be expected to exhibit significant ACP specificity, accounting for the observation that methyl malonate units were removed from ACP domains of DEBS loading domain and PikAIII at nearly identical rates. Such low ACP specificity for TEII may also explain why heterologous expression of the TEII gene ($nbmB$) from $S$. narbonensis in $S$. fradiae increased tylactone production in a TEII ($tylO$) mutant from 15% to that observed in the wild type strain (18). A similar observation involving complementation of the $tylO$ mutant strain with a TEII ($scoT$) obtained from a putative type I modular PKS gene cluster of $S$. coelicolor A3 (2) has also been reported (15, 19). In fact it appears that the ACP group may not contribute significantly to substrate binding and catalysis with TEII. The hydrolysis rates reported for TylO with propionyl- and butyryl-N-acetylcysteamine thioesters (12.9 and 6.5 M⁻¹s⁻¹, respectively) (24) are comparable with our observations of the PikAV (TEII) with the corresponding thioesters of the DEBS ATₗ-ACPₗ loading domain (17.5 ± 2.1 and 15.8 ± 1.8 M⁻¹s⁻¹).

Although TEII is not predicted to have ACP specificity, it might be expected to hydrolyze acyl-ACP units formed by aberrant decarboxylation more efficiently than with the corresponding carboxylated acyl-ACP substrates (required for the extension steps in polyketide biosynthesis). Consistent with this prediction, the pikromycin TEII hydrolyzed propionyl- and butyryl-ACP at a rate higher than for malonyl- and methylmalonyl-ACP derivatives. At odds with the prediction was the observation that acetyl-ACP was cleaved at a rate comparable with malonyl-ACP (the correct substrate for the PikAII-catalyzed second extension step in pikromycin biosynthesis) and significantly slower than butyryl-ACP (the ethylmalonyl-ACP, which would decarboxylate to generate this substrate, is not used in pikromycin biosynthesis). Preferential hydrolysis of propionate- and butyrate-$p$-nitrophenol esters (data not shown) was also observed with the TEII, suggesting this observation was not an artifact associated with use of the DEBS ATₗ-ACPₗ. In a previous observation, the tylosin TEII was shown to prefer the hydrolysis of an $N$-acetyl cysteamine thioesters over that observed for either propionate and acetate thioesters despite tylosin being generated using methylmalonyl-, ethylmalonyl-, and malonyl-ACP extender units (24). Thus, there are some small differences in the acyl group specificities of the tylosin and pikromycin TEII, which do not readily correlate with the substrate specificities of the extension modules within the two corresponding PKSs.

The pikromycin TEII exhibited a very high $K_m$ (>100 μM) with all thioesters of PikAIII and DEBS ATₗ-ACPₗ. High $K_m$ values have previously been reported for the tylosin TEII using propionyl (37.9 mM)- and butyryl (28.2 mM)-N-acetyl cysteamine thioesters (24). Because concentrations of each of the PKS polypeptides within a cell are unlikely to even approach 100 μM, the pikromycin TEII would appear to operate under non-saturating conditions. The pikromycin TEII would, thus, be predominantly dissociated from the PKS and, as such, more able to scan all of the ACP domains within a PKS. A greater capacity of TEII to bind ACP would presumably give rise to a higher affinity for acyl-ACP and a more efficient enzyme. However, a concomitant increase in affinity for the carboxylated acyl-ACP derivatives or even holo-ACP domains would be predicted. Not only might this interfere with an ability to scan all of the PKS ACP domains, but a tightly bound TEII could also interfere with the normal processing of the PKS. A low affinity for the PKS and a relatively slow rate of reaction may thus

FIG. 7. Effect of TEII overexpression on polyketide production in $S$. venezuelae. The aglycones (10-deoxymethynolide and narbonolide) produced by pSC38/SC1022 (solid line) and pSC70/SC1022 (dashed line) were quantified by HPLC analysis. pSC70 is a high copy number TEII (PikAV) expression plasmid derived from pWHM3 containing the $ermE^*$ promoter (pSE34). The pSC38 plasmid derived from pDHS702 is based on SCP2* origin of replication with TEII expression under control of the $pikAI$ promoter, which allows regulated expression in $S$. venezuelae. Fermentations of SC1022 carrying pBK56 (expressing an inactive TEII), pSE34 (an empty plasmid control), or pSC38 were indistinguishable. The fermentation and analysis conditions were as described under "Experimental Procedures."

ing methylmalonyl-CoA as an extending unit. In a cell-free extract, DEBS3 is able to synthesize triketide lactone (without a loading domain) from just methylmalonyl-CoA, indicating that the KS domain was primed with a propionyl starting unit through an aberrant decarboxylation of methylmalonyl-ACP (39). Similarly, decarboxylation of methylmalonyl-ACP within the first extension module may explain why a site-specific mutant of the loading domain ACP in DEBS1-TE (resulting in the absence of 4′-phosphopantetheine prosthetic group) can still produce 2–5% of wild type levels of triketide lactone (22). These results suggest that as a result of this activity of the KS domain, a low rate of aberrant decarboxylation is inevitable during polyketide synthesis. Although such aberrant decarboxylation may occur at low frequency, it inevitably may lead over a multi-day fermentation to significant decreased efficiency of a PKS (24). Hydrolysis of the acyl-ACP groups by a thioesterase (TEII) would restore the PKS efficiency and maximize polyketide production.

For such a role TEII would need to be an individual protein (not covalent linked to module) able to hydrolyze short-chain acyl-ACP thioesters. Previous studies on the tylosin TEII have demonstrated using $N$-acetylcysteamine thioesters a preference for short-chain acyl groups over longer chain acyl groups or diketide derivatives. In this study we have demonstrated for the first time the ability of a PKS TEII to hydrolyze acyl groups attached to PKS ACP domains. The observation that the TEII enzyme catalyzed propionyl-ACP hydrolysis for DEBS1 ATₗ-ACPₗ significantly faster than PikAIV containing TEI is also consistent with the differing roles of the enzymes. The PikAIV-TEI is covalently linked to the last extension module of the PKS and is, thus, presented with substrate for efficient cyclization via an intramolecular process. In addition, recent crystallographic studies of the homologous TEI domain from the erythromycin PKS have revealed how these domains are ideally designed for catalyzing the cyclization process with longer polyketide chains rather than thioester hydrolysis (40). Indeed, the TEI domain attached to DEBS1-TE catalyzes cyclization reactions several orders of magnitude faster ($k_{cat}$ of >5 min⁻¹)

ensure that TEII only becomes involved once an aberrant decarboxylation has occurred and the biosynthetic process has stalled (providing acyl-ACP substrates with significant half-lives). As such, carboxylated acyl-ACP units, although substrates for TEII, should be processed readily by the PKS and not readily hydrolyzed *in vivo*. The $k_{\text{cat}}$ of $5 \pm 1 \, \text{min}^{-1}$ for triketide production by DEBS1-TE (7) is much faster than the observed rate of $0.15 \, \text{min}^{-1}$ for propionyl-ACP ($150 \, \mu \text{M}$) hydrolysis observed for TEII under nonsaturating conditions. The editing function of TEII might, thus, be dependent not only upon its substrate specificity but also a low affinity for ACP thioester substrates and a significant difference in the reaction it catalyzes with that of the normal functioning PKS.

An editing role for TEII is consistent with previous observations that deletion of this gene generally leads to loss of production of polyketide or polypeptide. This is not the case for the pikromycin PKS, where deletion of the gene in strain SC1022 has previously been reported to have no affect upon polyketide yields. No increase in aglycone production in SC1022 was observed with the TEII complementation plasmid, consistent with this previous observation. The reasons why the TEII encoded by *pikAV* are not required for maximal polyketide production in SC1022 are unclear but may be due to the presence of another enzyme within *S. venezuelae* with similar activity. Such a possibility is quite reasonable given that putative TEII genes are often associated with modular PKS gene clusters, and many streptomycetes (41) including *S. venezuelae* contain several distinct clusters.

An editing role for TEII suggests that manipulation of its expression level or properties might have significant benefits for commercially important fermentation processes (24). It has recently been shown that expression of the TEII for *S. erythraea* leads to an 80% increase in production of the erythromycin aglycone, 6-deoxyerythronolide B, in cultures of recombinant *E. coli* (42). It has also been suggested that in genetically engineered PKSs, aberrant decarboxylation may be more prevalent, and TEII may help boost polyketide production (although levels of polyketides produced by hybrid pikromycin PKSs in *S. venezuelae* are very low despite the presence of the pikromycin TEII and possibly an additional TEII (29, 43)). However, the observation that TEII removes an acyl group from a loading domain ACP (the DEBS AT<sub>L</sub>-ACP<sub>L</sub>) and indications that it is unable to rigorously differentiate between an acyl-ACP substrate and its carboxylated derivative on an extension module suggest high level expression of TEII could suppress, not enhance, polyketide production *in vivo*. The decreased yields of aglycone production with plasmid-based high level expression of the pikromycin TEII in strain SC1022 are consistent with this prediction. The exact reasons for this decrease are unknown. However, the observation that high level expression of an inactive TEII does not suppress titers suggests that it is a result of the catalytic activity of TEII rather than simply binding to the PKS. Thus, it would appear that in addition to substrate specificity and kinetic properties, control of the timing and level of TEII expression are important for the enzyme to function appropriately as an editing enzyme. Within this context we note that recent S1 nuclease mapping studies of the TEII (*scoT*) promoter region in *S. coelicolor* have shown that transcription is dependent on growth phase and is only detectable during the late transition period. A regulatory system presumably prevents expression of the gene throughout most of the growth cycle and coordinates expression with the associated modular PKS gene cluster (19). Indeed, expression profiling of the entire *pik* gene cluster will determine the coordinated regulation of all *pikA* biosynthetic genes, including *pikAV*.

---

Note Added in Proof—An elegant biochemical characterization of two TEIIs associated with nonribosomal peptide syntheses was published (44) after acceptance of this manuscript.

---

REFERENCES

1. Hopwood, D. A. (1997) *Chem. Rev.* **97**, 2465–2497
2. Hopwood, D. A., and Sherman, D. H. (1990) *Annu. Rev. Genet.* **24**, 37–66
3. Katz, L., and Donadio, S. (1993) *Annu. Rev. Microbiol.* **47**, 875–912
4. Donadio, S., Staver, M. J., McAlpine, J. B., Swanson, S. J., and Katz, L. (1991) *Science* **252**, 675–679
5. Xue, Y., Zhao, L., Liu, H. W., and Sherman, D. H. (1998) *Proc. Natl. Acad. Sci. U. S. A.* **95**, 12111–12116
6. Schwecke, T., Aparico, J. F., Molner, I., Konig, A., Khaw, L. E., Haydock, S. F., Oliynyk, M., Caffrey, P., Cortes, J., Lester, J. B., Bohm, G. A., Staunton, J., and Leadlay, P. F. (1995) *Proc. Natl. Acad. Sci. U. S. A.* **92**, 7839–7843
7. Gokhale, R. S., Hunziker, D., Cane, D. E., and Khosla, C. (1999) *Chem. Biol. (Lond.)* **6**, 117–125
8. Merson-Davies, L. A., and Cundliffe, E. (1994) *Mol. Microbiol.* **13**, 349–355
9. August, P. R., Tang, L., Yoon, Y. J., Ning, S., Muller, R., Yu, T. W., Taylor, M., Hoffmann, D., Kim, C. G., Zhang, X., Hutchinson, C. R., and Floss, H. G. (1998) *Chem. Biol. (Lond.)* **5**, 69–79
10. Haydock, S. F., Dowson, J. A., Dhillon, N., Roberts, G. A., Cortes, J., and Leadlay, P. F. (1991) *Mol. Gen. Genet.* **230**, 120–128
11. Schneider, A., and Marahiel, M. A. (1998) *Arch. Microbiol.* **169**, 404–410
12. Witkowski, A., Naggert, J., Wessa, B., and Smith, S. (1991) *J. Biol. Chem.* **266**, 18514–18519
13. Witkowski, A., Naggert, J., Witkowska, H. E., Randhawa, Z. I., and Smith, S. (1992) *J. Biol. Chem.* **267**, 18488–18492
14. Aggarwal, R., Caffrey, P., Leadlay, P. F., Smith, C. J., and Staunton, J. (1995) *J. Chem. Soc. Chem. Commun.* **15**, 1519–1520
15. Pieper, R., Luo, G., Cane, D. E., and Khosla, C. (1995) *Nature* **378**, 263–266
16. Xue, Y., and Sherman, D. H. (2000) *Nature* **403**, 571–575
17. Doi-Katayama, Y., Yoon, Y. J., Choi, C. Y., Yu, T. W., Floss, H. G., and Hutchinson, C. R. (2000) *J. Antibiot. (Tokyo)* **53**, 484–495
18. Butler, A. R., Bate, N., and Cundliffe, E. (1999) *Chem. Biol. (Lond.)* **6**, 287–292
19. Kotowska, M., Pawlik, K., Butler, A. R., Cundliffe, E., Takano, E., and Kuczek, K. (2002) *Microbiology* **148**, 1777–1783
20. Chen, S., Roberts, J. B., Xue, Y., Sherman, D. H., and Reynolds, K. A. (2001) *Gene* **262**, 255–264
21. Cosmina, P., Rodriguez, F., Ferra, F. d., Grandi, G., Perego, M., Venema, G., and Sinderen, D. v. (1993) *Mol. Microbiol.* **8**, 821–831
22. Long, P. F., Wilkinson, C. J., Bisang, C. P., Cortes, J., Dunster, N., Oliynyk, M., McCormick, E., McArthur, H., Mendez, C., Salas, J. A., Staunton, J., and Leadlay, P. F. (2002) *Mol. Microbiol.* **43**, 1215–1225
23. Bisang, C., Long, P. F., Cortes, J., Westcott, J., Crosby, J., Matharu, A.-L., Simpson, T. J., and Leadlay, P. F. (1999) *Nature* **401**, 502–505
24. Heathcote, M. L., Staunton, J., and Leadlay, P. F. (2001) *Chem. Biol. (Lond.)* **8**, 207–220
25. Naggert, J., Witkowski, A., Wessa, B., and Smith, S. (1991) *Biochem. J.* **273**, 787–790
26. Sambrook, J., Fritsch, E. F., and Maniatis, T. (1989) *Molecular Cloning: A Laboratory Manual*, 2nd Ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY
27. Ausubel, F. M., Brent, R., Kingston, R. E., Moore, D. D., Seidman, J. G., Smith, J. A., and Struhl, K. (1991) *Current Protocols in Molecular Biology*, Vol. 2, Greene and Wiley, New York
28. Lambalot, R. H., Gehring, A. M., Flugel, R. S., Zuber, P., LaCelle, M., Marahiel, M. A., Reid, R., Khosla, C., and Walsh, C. T. (1996) *Chem. Biol. (Lond.)* **3**, 923–936
29. Kim, B. S., Cropp, T. A., Florova, G., Lindsay, Y., Sherman, D. H., and Reynolds, K. A. (2002) *Biochemistry* **41**, 10827–10833
30. Lau, J., Cane, D. E., and Khosla, C. (2000) *Biochemistry* **39**, 10514–10520
31. Vara, J., Lewandowska-Skarbek, M., Wang, Y.-G., Donadio, S., and Hutchison, C. R. (1989) *J. Bacteriol.* **171**, 5872–5881
32. Chen, S., Xue, Y., Sherman, D. H., and Reynolds, K. A. (2000) *Chem. Biol. (Lond.)* **7**, 907–918
33. Witkowski, A., Witkowska, H. E., and Smith, S. (1994) *J. Biol. Chem.* **269**, 379–383
34. Kao, C. M., Luo, G., Katz, L., Cane, D. E., and Khosla, C. (1995) *J. Am. Chem. Soc.* **117**, 9105–9106
35. Cortes, J., Wiesmann, K. E. H., Roberts, G. A., Brown, M. J., Staunton, J., and Leadlay, P. F. (1995) *Science* **268**, 1487–1489
36. Witkowski, A., Joshi, A. K., Lindqvist, Y., and Smith, S. (1999) *Biochemistry* **38**, 11643–11650
37. Kresze, G. B., Steber, L., Oesterhelt, D., and Lynen, F. (1977) *Eur. J. Biochem.* **79**, 191–199
38. Smirnova, N., and Reynolds, K. A. (2001) *J. Bacteriol.* **183**, 2335–2342
39. Jacobson, J. R., Cane, D. E., and Khosla, C. (1998) *Biochemistry* **37**, 4928–4934
40. Tsai, S. C., Miercke, L. J., Krucinski, J., Gokhale, R., Chen, J. C., Foster, P. G., Cane, D. E., Khosla, C., and Stroud, R. M. (2001) *Proc. Natl. Acad. Sci. U. S. A.* **98**, 14808–14813
41. Omura, S., Ikeda, H., Ishikawa, J., Hanamoto, A., Takahashi, C., Shinose, M., Takahashi, Y., Horikawa, H., Nakazawa, H., Osonoe, T., Kikuchi, H., Shiba, T., Sakaki, Y., and Hattori, M. (2001) *Proc. Natl. Acad. Sci. U. S. A.* **98**, 12215–12220
42. Pfeifer, B., Hu, Z., Licari, P., and Khosla, C. (2002) *Appl. Environ. Microbiol.* **68**, 3287–3292
43. Yoon, Y. J., Beck, B. J., Kim, B. S., Kang, H.-Y., Reynolds, K. A., and Sherman, D. H. (2002) *Chem. Biol. (Lond.)* **9**, 203–214
44. Schwarzer, D., Mootz, H. D., Linne, U., and Marahiel, M. A. (2002) *Proc. Natl. Acad. Sci. U. S. A.* **99**, 14083–14088
