
Downloaded from genome.cshlp.org on March 25, 2015 - Published by Cold Spring Harbor Laboratory Press

---

**Letter**

# Conservation of the Biotin Regulon and the BirA Regulatory Signal in Eubacteria and Archaea

Dmitry A. Rodionov,$^{1,3}$ Andrei A. Mironov,$^{2}$ and Mikhail S. Gelfand$^{1,2}$

$^{1}$State Scientific Center GosNIIGenetika, Moscow 113545, Russia; $^{2}$Integrated Genomics–Moscow, Moscow 117333, Russia

Biotin is a necessary cofactor of numerous biotin-dependent carboxylases in a variety of microorganisms. The strict control of biotin biosynthesis in *Escherichia coli* is mediated by the bifunctional BirA protein, which acts both as a biotin-protein ligase and as a transcriptional repressor of the biotin operon. Little is known about regulation of biotin biosynthesis in other bacteria. Using comparative genomics and phylogenetic analysis, we describe the biotin biosynthetic pathway and the BirA regulon in most available bacterial genomes. Existence of an N-terminal DNA-binding domain in BirA strictly correlates with the presence of putative BirA-binding sites upstream of biotin operons. The predicted BirA-binding sites are well conserved among various eubacterial and archaeal genomes. The possible role of the hypothetical genes *bioY* and *yhfS-yhfT*, newly identified members of the BirA regulon, in the biotin metabolism is discussed. Based on analysis of co-occurrence of the biotin biosynthetic genes and *bioY* in complete genomes, we predict involvement of the transmembrane protein BioY in biotin transport. Various nonorthologous substitutes of the *bioC*-coupled gene *bioH* from *E. coli*, observed in several genomes, possibly represent the existence of different pathways for pimeloyl-CoA biosynthesis. Another interesting result of analysis of operon structures and BirA sites is that some biotin-dependent carboxylases from *Rhodobacter capsulatus*, actinomycetes, and archaea are possibly coregulated with BirA. BirA is the first example of a transcriptional regulator with a conserved binding signal in eubacteria and archaea.

Biotin (vitamin H) is an essential cofactor for a class of important metabolic enzymes, biotin carboxylases and decarboxylases (Perkins and Pero 2001). The biotin biosynthetic pathway is widespread among microorganisms. The well-studied systems of biotin biosynthesis from *Escherichia coli*, *Bacillus subtilis*, and *Bacillus sphaericus* differ in the first step of biosynthesis. *B. subtilis* and *B. sphaericus* use pimeloyl-CoA synthase encoded by the *bioW* gene to synthesize pimeloyl-CoA from pimelic acid. In addition, pimelic acid formation in *B. subtilis* has been proposed to use cytochrome P450 encoded by *bioI* (Stok and De Voss 2000). In *E. coli*, pimeloyl-CoA is synthesized from L-alanine and/or acetate via acetyl-CoA, instead of pimelic acid (Ifuku et al. 1994), and products of the *bioC* and *bioH* genes are required for pimeloyl-CoA synthesis in *E. coli*. The pathway from pimeloyl-CoA to biotin is similar in *E. coli* and bacilli and uses products of the *bioF*, *bioD*, *bioA*, and *bioB* genes (Fig. 1). Genes encoding biotin transporters have not been identified in bacteria until now, but *E. coli* can uptake biotin by active transport (Piffeteau and Gaudry 1985), and a gene for biotin transport, *bioP*, has been mapped on the *E. coli* chromosome (Eisenberg 1985).

The operon organization of the biotin biosynthetic genes differs between *E. coli* and bacilli. *E. coli* has *bioBFCD* operon located divergently with the *bioA* gene and single *bioH* gene (DeMoll 1994). In contrast, *B. subtilis* has the single *bioWAFDBI* operon (Perkins et al. 1996). Two unlinked biotin biosynthetic operons, *bioDAYB* and *bioXWF*, were described in *B. sphaericus* (Gloeckler et al. 1990). The functions of two new biotin-related genes, *bioX* and *bioY*, are presently unknown; however, it has been proposed that BioX of *B. sphaericus* and BioC of *E. coli* may function as acyl carrier proteins

$^{3}$Corresponding author.  
E-MAIL rodionov@genetika.ru; FAX 7-095-3150501.  
Article and publication are at http://www.genome.org/cgi/doi/10.1101/gr.314502.

12:1507–1516 ©2002 by Cold Spring Harbor Laboratory Press ISSN 1088-9051/02 $5.00; www.genome.org Genome Research 1507 www.genome.org

Downloaded from genome.cshlp.org on March 25, 2015 - Published by Cold Spring Harbor Laboratory Press

Rodionov et al.

BBS genes are less widespread than *birA*: among all complete genomes, *Sinorhizobium meliloti*, *Rickettsia prowazekii*, *Deinococcus radiodurans*, *Thermotoga maritima*, *Treponema pallidum*, most archaea, and Gram-positive pathogens from the Bacillus/Clostridium group lack the BBS genes, but have *birA*. Among archaeal genomes, only *Methanococcus jannaschii* has a cluster of the BBS genes. Phylogenetic analysis of the BBS proteins shows that this archaeal BBS gene cluster may be the result of possible horizontal gene transfer from bacilli. The detailed phylogenetic and positional analysis of the BBS genes is given below.

### BirA Regulon

To analyze possible transcriptional regulation of the BBS genes, we started with identification of the N-terminal regulatory domains in the detected BirA proteins. Using multiple alignment, we compiled the list of 46 sequences of the BirA N-terminal domains that have the same length as the known regulatory domain of *E. coli* BirA. To determine the significance of the possible helix-turn-helix (HTH) regulatory motif in each of the collected sequences, the HTH motif prediction program (Dodd and Egan 1990) was used (Fig. 2). After that, eight sequences without HTH motifs were removed, and 38 BirA proteins with the predicted DNA-binding regulatory domains (D-b-BirA) were retained (Table 1). We also retained the BirA protein from *Bacillus cereus*, although it was predicted to contain no HTH motif. This looks like a false-negative prediction. Indeed, not only is BirA highly conserved among bacilli, but the *B. cereus* genome has several strong BirA sites upstream of biotin-related operons. To support the selection of D-b-BirA, the phylogenetic tree of 50 BirA N-terminal domains was constructed (Fig. 3). It shows that each sequence without a potential HTH motif is highly diverged from the D-b-BirA sequences and looks like an outgroup in this tree.

D-b-BirA is widely distributed in the Bacillus/Clostridium group, gamma-proteobacteria, and archaea. In addition, it was found in *Nitrosomonas europaea*, *Methylobacillus flagellatus*, *Magnetococcus* sp., and *Thermus thermophilus*. The N-terminal domains of BirA from the Pasteurellaceae family of gamma-proteobacteria possibly have lost their regulatory function. The genomes of *Clostridium acetobutylicum*, *Lactococcus lactis*, *Halobacterium* sp., *Pyrococcus abyssi*, and *Pyrococcus furiosus* have two BirA paralogs, with and without the N-terminal regulatory domain. The phylogenetic analysis of the catalytic BirA domains shows that paralogous BirA in the first three genomes could result from a recent duplication. In *P. abyssi* and *P. furiosus*, BirA without the N-terminal regulatory domain is close to the other archaeal BirA, whereas the second BirA (D-b-BirA) has a weakly conserved catalytic domain and a well-conserved N-terminal regulatory domain.

Based on the phylogenetic analysis of the D-b domains, all D-b-BirAs were divided into two major groups, proteobacterial and nonproteobacterial (Fig. 3). Consistent with this, two different recognition rules (profiles) for the BirA sites were constructed using the sets of upstream regions of the BBS genes from various genomes. The BirA profile for proteobacteria (with consensus 5′-tGTGaAACc-N14 ... 16-GGTtACAa-3′, where strongly conserved positions are shown in capitals) is more strict than that for other bacteria (5′-wwTGTtAAC-N14 ... 16-GTTaACAwW-3′, where ‘w’ stands for A or T). The constructed profiles were used to detect new candidate members of the BirA regulons in the genomes containing D-b-BirA. Proteobacteria possess only one strong BirA site per genome occurring upstream of the BBS operon. How-

### RESULTS AND DISCUSSION

Orthologs of *birA* and biotin biosynthetic genes (BBS) from *E. coli* and *B. subtilis* were identified in all available bacterial genomes by similarity search (Table 1). The biotin-protein ligase BirA is widely distributed in eubacteria and archaea. Only *Buchnera* sp., *Borrelia burgdorferi*, *Aeropyrum pernix*, thermoplasmas, and mycoplasmas have neither the BBS genes nor *birA*, which is consistent with the lack of biotin-dependent carboxylases in the genomes of these microorganisms.

Table 1. Operon Structure and Predicted BirA Sites for the Biotin Biosynthetic Genes in Prokaryotes

| Genome | BirA | AB | D-b | BPL | Biotin biosynthetic genes | Biotin-dependent carboxylases | BirA sites | Pos | Score |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| alpha-Proteobacteria |  |  |  |  |  |  |  |  |  |
| Caulobacter crescentus | CO | 0 | + | + | bioB/bioA <> bioF-bioD/bioC | cbiO-cbiQ-bioY-yhfT-yhfs |  |  |  |
| Sinorhizobium meliloti | SM | 0 | + | + | bioB-bioF-bioD-bioA-bioZ/bioC | bioY1/bioY2-X |  |  |  |
| Mesorhizobium loti | MLO | 0 | + | + | bioB-bioF-bioD-bioA-bioZ/bioC | cbiO-cbiQ-bioY |  |  |  |
| Agrobacterium tumefaciens | AT | 0 | + | + | bioB/bioF-bioD-bioA/bioC | bioY-X-X |  |  |  |
| Rhodopseudomonas palustris | RPA | 0 | + | + | bioB/bioF-bioD-bioA/bioC | bioY-X-X |  |  |  |
| Bradyrhizobium japonicum | BJA | 0 | + | + | bioB-bioF-bioD-bioA-bioG/bioC | cbiO1-cbiQ1-bioY1 | modYZGB-birA- |  |  |
| Rhodobacter capsulatus# | RS | 0 | + | + |  | cbiO2-cbiQ2-bioY2 | madAECDHKELM |  |  |
| M.magnetotacticum# | MMA | 0 | + | + | [bioB bioF]/bioD-bioA/bioC | bioY1/bioY2-X |  |  |  |
| Brucella melitensis | BME | 0 | + | + | bioB-bioF-bioD-bioA-bioZ/bioC | bioY |  |  |  |
| Rickettsia prowazekii | RP | 0 | + | + | none |  |  |  |  |
| beta-Proteobacteria |  |  |  |  |  |  |  |  |  |
| Bordetella pertussis# | BP | 0 | + | + | bioA <> bioF/bioB | cbiO-cbiQ-bioY |  |  |  |
| Burkholderia pseudomallei# | BU | 0 | + | + | bioA-bioF-bioD-bioB/bioC |  |  |  |  |
| Nitrosomonas europaea | NE | + | - | + | bioB-bioF-bioH-bioC-bioD-$bioA | CTGTcttgC-(15)-GcTTgACAA | -246 | 5.99 |  |
| Neisseria meningitidis | NM | + | - | + | bioB/bioH-bioC2/bioF-bioG-bioC1/bioA-bioD | aTGTA AAAtg-(15)-GcTTgACAA | -68 | 7.10 |  |
| Methylobacillus flagellatus# | MFL | + | - | + | $bioB-bioF-bioH-bioC-bioD/bioA-X |  |  |  |  |
| Ralstonia solanacearum | RSO | 0 | + | + | bioA-bioF-bioD/X-X-bioB/bioC |  |  |  |  |
| Ralstonia eutropha# | REU | 0 | + | + | bioA-bioF-bioD-bioB/bioC |  |  |  |  |
| gamma-Proteobacteria |  |  |  |  |  |  |  |  |  |
| Escherichia coli | EC | + | + | + | bioA <> $> bioB-bioF-bioC-bioD-bioH/bioH | TTG TAAAACC-(16)-GGTTTACAA | -80 | 9.10 |  |
| Salmonella typhi | TY | + | + | + | bioA <> $> bioB-bioF-bioC-bioD-bioH/bioH | TTG TAAAACC-(16)-GGTTTACGA | -80 | 8.49 |  |
| Klebsiella pneumoniae# | KP | + | + | + | bioA <> $> bioB-bioF-bioC-bioD-bioH/bioH | TTG TAAAACC-(16)-GGTTTACAA | -207 | 9.10 |  |
| Yersinia pestis | VP | + | + | + | bioA <> $> bioB-bioF-bioC-bioD/bioH | TTG TAAAACC-(16)-GGTTgACAg | -88 | 8.87 |  |
| Vibrio cholerae | VC | + | + | + | bioA <> $> bioB-bioF-bioC-bioD | aTG TAAAACC-(15)-tGTTgACAg | -94 | 8.12 |  |
| Francisella tularensis# | FT | + | + | + | [bioA/[bioB-bioF-bioH-bioD/bioC | TTG TAAAACC-(15)-aGTTgACAT | -90 | 8.44 |  |
| Legionella pneumophila# | LP | + | + | + | bioA-bioF-bioC-bioC-bioD-bioD/bioC |  |  |  |  |
| Haemophilus influenzae | HI | + | + | + | bioA-bioF-bioC-bioC-bioD-bioD/bioB |  |  |  |  |
| Haemophilus ducreyi# | DU | + | + | + | bioA-bioF-bioG-bioC-bioD/bioD/bioB |  |  |  |  |
| Pasteurella multocida | VK | + | + | + | bioA-bioF-bioG-bioC-bioD/bioD/bioB |  |  |  |  |
| A.actinomy cetecommitans# | AB | + | + | + | $bioB-bioF-bioH-bioC-bioD/bioA | aTG TAGtCC-(14)-GGTTgACAg | -130 | 7.43 |  |
| Pseudomonas aeruginosa | PA | + | + | + | $bioB-bioF-bioH-bioC-bioD/bioA | TTG TAAAACC-(15)-aGTTgACAg | -125 | 8.47 |  |
| Pseudomonas putida | Ppu | + | + | + | $bioB-bioF-bioH-bioC-bioD/bioA | aTG TAAAACC-(15)-GGTTgACAg | -128 | 8.71 |  |
| Pseudomonas fluorescens | PU | + | + | + | $bioB-bioF-bioH-bioC-bioD/bioA | TG TAAAACC-(15)-cGTTgACAg | -90 | 7.78 |  |
| Shewanella putrefaciens# | SH | + | + | + | bioA <> $> bioB-bioF-bioC-bioD/bioH | TTG TAAAACC-(15)-aGTTgACAA | -112 | 8.60 |  |
| Thermochromatium tepidum# | CTE | + | + | + | $bioB-bioF-bioH-bioCJ/[X-bioA |  |  |  |  |
| Xylella fastidiosa | XFA | + | + | + | bioB/bioF-bioH/bioD/bioC/bioA |  |  |  |  |
| Acinetobacter calcoaceticus# | AC | 0 | + | + | bioB/bioH-bioA-bioF-bioC-bioD |  |  |  |  |
| Buchnera sp. | BUC | 0 | + | + |  |  |  |  |  |
| epsilon-Proteobacteria |  |  |  |  |  |  |  |  |  |
| Helicobacter pylori | HX | 0 | + | + | bioA/bioD/X-bioF/bioC/bioB-X | ATT GTTIAAC-(15)-GGTTAACAT | -127 | 8.84 |  |
| Campylobacter jejuni | CJ | 0 | + | + | bioA <> bioF-bioG-bioC/X-bioD/X-bioB-X-bioD/bioA/bioC2 | tTT GTTIAAC-(15)-GGTTAACAT | -89 | 8.54 |  |
| Magnetococcus# | MCO | + | + | + | $bioW-bioA-bioF-bioD-bioB-biol | tAT GTAAC-(15)-GGTTgACAT | -88 | 8.32 |  |
| Bacillus/Clostridium group |  |  |  |  |  |  |  |  |  |
| Bacillus subtilis | BS | + | + | + | $bioW-bioA-bioF-bioD-bioB-biol |  | -46 | 7.44 |  |

(Continued on next page)

| Bacillus sphaericus # | BW | ? | $bioD-bioA-bioY-bioB | tgGTGTTAAC-(16)-GTTTACtAA | 7.86 |
| --- | --- | --- | --- | --- | --- |
| Bacillus halodurans | HD | + | $bioX-bioW-bioF | tgGTGTTAAC-(15)-GTTTACtca | 7.49 |
|  |  |  | $bioB | ATTGGTTAAC-(15)-GTTTACAAT | 8.72 |
|  |  |  | $bioD-bioA | TATGTTAAC-(15)-GTTTACATA | 8.64 |
|  |  |  | $bioF-bioH-bioC | tATGTTAAC-(15)-GTTTACATT | 8.74 |
|  |  |  |  | tATGTcAAC-(15)-GTTqACAAA | 8.24 |
|  |  |  |  | tATGTTAAC-(15)-GTaACACAT | 8.08 |
|  |  |  |  | AATGTTAAC-(15)-GTTgACAAA | 8.42 |
|  |  |  |  | AATGTTAAC-(16)-GTTTACATA | 8.50 |
|  |  |  |  | TTGTGTTAAC-(15)-GTTTACATA | 8.52 |
|  |  |  |  | AATGTTAAC-(15)-GTTAACATT | 8.84 |
|  |  |  |  | AATGTTAAC-(16)-GTTAACATT | 8.32 |
|  |  |  |  | ATTGTGTTAAC-(15)-GTTgACAAA | 8.84 |
|  |  |  |  | ATTGTGTTAAC-(16)-GTTAACATT | 8.84 |
|  |  |  |  | ATTGTaAAC-(16)-GTTTACAA | 8.60 |
|  |  |  |  | ATTGTaAAC-(16)-GTTTACAA | 8.60 |
|  |  |  |  | ATTGTaAAC-(16)-GTTTACAA | 8.64 |
|  |  |  |  | tAaGTaAAC-(16)-GTTTACAA | 7.91 |
|  |  |  |  | ATA GTaAAC-(16)-GTTTACAA | 8.84 |
|  |  |  |  | AATGTaAAC-(16)-GTTTACAA | 8.42 |
|  |  |  |  | ATTGTaAAC-(16)-GTTTACAA | 8.52 |
|  |  |  |  | ATTGTagAC-(16)-GTTgCAA | 6.82 |
|  |  |  |  | ATTGTTAAC-(16)-GTTTACAA | 8.84 |
|  |  |  |  | ATTGTcAAC-(16)-GTTTACAA | 8.44 |
|  |  |  |  | tATGTTAAC-(14)-GTTTACAA | 8.44 |
|  |  |  |  | tATGTTAAC-(14)-GTTTACAA | 8.64 |
|  |  |  |  | AATGTTAAC-(15)-GTTTACAA | 8.72 |
|  |  |  |  | AcaGTTAAC-(16)-GgTTActgF | 6.48 |
|  |  |  |  | AATGTTAAC-(15)-GTTTACAA | 8.60 |
|  |  |  |  | ATTGTTAAC-(15)-GTTTACAA | 8.60 |
|  |  |  |  | ATTGTTAAC-(15)-GTTTACAA | 8.72 |
|  |  |  |  | tTTGTTAAC-(16)-GTTgACATc | 7.41 |
|  |  |  |  | ACA GTTAcC-(16)-GTTgACAAA | 7.14 |
|  |  |  |  | AAA GTcAcC-(16)-GgTTACAgT | 6.56 |
|  |  |  |  | gTTGTcAAC-(15)-GgTTACAA | 6.66 |
|  |  |  |  | AAA GTTAAAC-(16)-GgTTACAA | 7.75 |
| Bacillus stearothermophilus # | BE | + | $bioY1-bioD-bioA/[bioB | $bioY |  |
|  |  |  | $bioF | $bioY |  |
| Bacillus cereus | ZC | + | $bioA-bioD-bioF-bioH-bioC-bioB | $bioY2-yhFT-yhFS |  |
| Clostridium acetobutylicum | CA | 0 | $bioY1-bioD-bioA | $bioY2-X |  |
|  |  |  | (D-b-birA)-$>bioY-bioB | $bioY |  |
| Clostridium botulinum # | CB | + | [bioY-bioB-bioD | $bioY-yhS-yhT |  |
| Clostridium difficile # | DF | + | [bioD/[bioA | $bioY |  |
| Helicobacter pylori # | EF | + | $bioY-bioD-bioA | $bioY |  |
| Kurthia sp. # | HMO | + | none | $bioY |  |
| Listeria innocua | Kur | ? | $orf2-bioF-bioB | $bioY |  |
| Lactococcus lactis | LI | + | $bioF-bioH-bioC/bioH | $bioY |  |
| Staphylococcus aureus | LL | + | none | $bioY |  |
| Streptococcus pneumoniae | SAX | + | $bioD-bioA-bioB-bioF-bioW-bioX | $bioY |  |
| Streptococcus pyogenes | PN | + | none | $yhT-yhFS |  |
| Streptomyces coelicolor # | ST | + | none | $bioY |  |
| Thermomonospora fusca # | SX | + | none? | $yhS-yhT |  |
|  | TFU | 0 | none | $bioY |  |
| Actinobacteria |  |  |  | bioY-dbiO-obiQ | birA<>pccB1-pccB2 |  |
| Corynebacterium glutamicum # | CGL | 0 | bioB/bioA-bioD/bioF-bioW-bioF/bioB2 | bioY-dbiO-obiQ | birA<>pccB |  |
| Corynebacterium diphtheriae # | DI | 0 | bioB/bioA-bioF-bioD | bioY | birA<>pccB-X-X-pccA |  |
| Mycobacterium tuberculosis | MT | 0 | bioB/bioA-bioF-bioD | bioY | birA<>pccB-X-X-pccA |  |
| Streptomyces coelicolor # | SX | 0 | bioF<>bioB-bioA-bioD | bioY_dbiO-obiQ | birA_ppc<>pccB_pccA |  |
| Thermomonospora fusca # | TFU | 0 | none? | bioY_dbiO-obiQ |  |  |
| CFB/Green sulfur bacteria group |  |  |  | bioA-bioF-bio(GC)-bioD |  |  |
| Bacteroides fragilis # | BX | 0 | bioB/bioF-bioD-bioA | bioA-bioF-bioD-bioA |  |  |
| Cytophaga hutchinsonii # | CHU | 0 | bioB/bioD-bioA | bioB-bioA/X-bioD/bioG-bioC/bioF |  |  |
| Porphyromonas gingivalis # | PG | 0 |  |  |  |  |

(Continued on next page)

Table 1. (Continued)

| BirA | Biotin biosynthetic genes | Biotin-dependent carboxylases | Biotin transporters | BirA sites | Score | Pos |
|------|---------------------------|-------------------------------|--------------------|------------|-------|-----|
| AB   | D-b                      |                              |                    |            |       |     |

**Cyanobacteria**
- Nostoc sp.
- Synechocystis sp.
- Prochlorococcus marinus
- Synechococcus sp.

| NPU | 0 | bioB/bioD/bioF/bioA | bioY-jspA | + | + | + |
| CY  | 0 | bioB-bioY-jspA/bioD/bioF/bioA | bioY-jspA | + | + | + |
| CK  | 0 | X-X-bioB/bioF-X-bioc-bioC-bioD-bioA | bioY-jspA | + | + | + |
| SN  | 0 | X-X-bioB/bioF-X-bioc-bioC-bioD-bioA |          | + | + | + |

**Others**
- Aquifex aeolicus
- Chlamydia trachomatis
- Chloroflexus aurantiacus #
- Chlorobium tepidum
- Deinococcus radiodurans
- Fusobacterium nucleatum #
- Thermotoga maritima
- Thermus thermophilus #
- Treponema pallidum

| AA  | 0 | X-X-bioB/bioW-X-X/X-X-bioD/bioA/bioC | TTGTCACC-(14)-GGTTTACAA | -143 | 9.00 |      |
| QT  | 0 | bioB-bioF-bioD-bioA-bioW             | bioY                  | +    |      |      |
| CL  | + | $birA-bioB-bioF-bioG-bioC-bioD-bioA-fadD | bioY                | +    |      |      |
| CAU | + | none                                 | bioY-cbiO-cbiQ        | +    |      |      |
| DR  | + | none                                 | fahH-fabZ-fabK-bioY-fabD | +    |      |      |
| FN  | 0 | bioB-bioD-bioA/bioF-bioG-bioC         | $bioY                | 21   | 7.48 |      |
| TM  | 0 | none                                 | bioY-cbiO-HTP1       | 615  | 7.52 |      |
| TQ  | + | $bioB                                | TCCTAAAAC-(15)-GGTTTACgA | +    |      |      |
| TP  | 0 | none                                 | acGTcAACc-(15)-GGTTgACgA | +    |      |      |

**Archaea**
- Archaeoglobus fulgidus
- Halobacterium sp.
- M. thermoautotrophicum
- Methanosarcina barkeri #
- Methanosarcina mazei
- Pyrococcus abyssii
- Pyrococcus furiosus
- Pyrococcus horikoshii
- Sulfolobus solfataricus

| AG  | + | none | CTcGTTAAC-(15)-GTTAACGAT | -22 | 6.39 |      |
| HSL | + | none | gCTGTaAAc-(16)-GaTACAACAT | -173 | 6.03 |      |
| TH  | 0 | none | gTcCTaAAC-(16)-GTTGACgAc | -50 | 5.70 |      |
| MJ  | 0 | bioB1/bioB2 <> bioW-bioF-bioD-bioA | tAaaTaAAC-(14)-GTTgAgTTa | -118 | 5.80 |      |
| MBA | + | none? | AATGTaAAC-(16)-GTTAACAAAT | -275 | 8.72 |      |
| MMZ | + | none | gTaGTTCAC-(16)-GTTAACAGg | -365 | 5.81 |      |
| PO  | + | none | AACGlcAGc-(15)-GTTAACAAAT | -269 | 8.72 |      |
| PF  | + | none | tTcGTTAAC-(16)-GTTAACcAa | -349 | 6.11 |      |
| PH  | 0 | none | gTgGTTAAC-(16)-GTTACgAa | -43 | 6.96 |      |
| STO | 0 | none | bioY | -55 | 6.49 |      |

$bioY-bioO-HTP2
$pycA
$bioY-bioO-HTP3
$pccB-pccA-X-$(D-b-birA)
$pycA-birA
$bioY-bioO-cbiO-cbiQ
$pycB-pycA-(D-b-birA)
$bioY-bioO-cbiO-cbiQ
$pycB-pycA-(D-b-birA)
bioY <$(D-b-birA)
bioY <>(D-b-birA)
bioY

The genome abbreviations are given in column AB. Unfinished genomes are marked by #. The names of taxonomic groups are given in bold. The signs + and 0 in the columns "BirA D-b" and "BirA BPL" denote the existence or absence of the N-terminal regulatory domain (D-b) and C-terminal catalytic domain (BPL) of BirA, respectively; – denotes N-terminal BirA domain not similar to the known regulatory BirA domain. Other columns show the operon structure and regulation of the biotin-related genes. Genes forming one candidate operon (with spacer <100 bp) are separated by dashes. Different loci are separated by slashes. The direction of transcription in divergons is shown by angle brackets. Predicted BirA sites are denoted by $. The contig ends are shown by square brackets. Bio(GC) is the fusion of the bioG and bioC genes. HTP1, HTP2, and HTP3 are nonhomologous hypothetical transmembrane proteins clustered with bioY-cbiO. The other genes of unknown function are denoted by X. The birA genes are shown only if they are colocalized with other biotin-related genes. The positions of the site are given relative to annotated translation starts. The site scores are computed using positional nucleotide weight matrices of two types, proteobacterial and nonproteobacterial, as described in Methods. The BirA sites of the proteobacterial type are given in bold.

ever, most Gram-positive bacteria and some archaea have multiple BirA sites located upstream of BBS genes and new genes of the BirA regulon (Table 1). For a control, we checked the genomes without D-b-BirA for the existence of BirA sites upstream of the BBS operons, and found none.

After comparison of the BirA regulons from numerous bacteria, we predicted several new biotin-regulated genes. A gene of unknown function, *bioY* (so named by Gloeckler et al. 1990), is widely distributed in bacteria and often clusters with genes of biotin metabolism. The homologs of BioY form a unique protein family (InterPro entry IPR003784), and have no significant similarity to any gene of known function. Analysis of the BirA sites showed that *bioY* is always under regulation of the biotin repressor in genomes containing regulatory D-b-BirA. The existence of the BirA-regulated *bioY* in several complete genomes that have no BBS genes indicates that *bioY* is probably not involved in biotin biosynthesis. On the other hand, proteins of the BioY family have six candidate transmembrane segments, an arrangement typical for prokaryotic transporters. The phylogenetic tree of the BioY protein family consists of several branches, and within each branch most members are positionally linked to BBS genes, or have upstream candidate BirA-binding sites, or both (Fig. 4A). Taken together, these observations strongly imply that all BioY paralogs are transporters of biotin or some biotin precursor.

Another gene pair of unknown function, *yhfS-yhfT*, has been detected in several bacteria from the *Bacillus/Clostridium* group and in *S. meliloti*. Except for the latter genome, the *yhfS-yhfT* genes are always under predicted regulation by BirA. YhfT and YhfS are homologous to numerous long-chain fatty acid-CoA ligases and acetyl-CoA acetyltransferases, respectively. Each of them forms a separate branch on the phylogenetic tree for the corresponding protein family (Fig. 4B, C). One of the *bioY* paralogs from *B. subtilis*, *yhfU*, belongs to the *yhfUST* operon, and transcription of this operon is repressed by BirA (Lee et al. 2001). In addition, *yhfU* and *yhfS-yhfT* are clustered in the genomes of *B. cereus*, *Lactococcus lactis*, *Clostridium difficile*, and *S. meliloti*; whereas *Streptococcus pyogenes*, *Streptococcus equi*, and *Staphylococcus aureus* have separate BirA-regulated *yhfST* and *yhfU* operons. Surprisingly, all YhfU paralogs except one from *C. difficile* form a separate branch in the phylogenetic tree of the BioY family (Fig. 4A). Again, occurrence of the positionally linked *yhfU-yhfS-yhfT* genes in complete genomes without BBS genes rules out their involvement in the first steps of biotin biosynthesis. A plausible hypothesis is that the YhfS-YhfT proteins are involved in fatty acid metabolism,

the pathway that requires biotin at one of the early steps (cf. clustering of *bioY* with fatty acid biosynthetic genes in *T. maritima*; see below).

### Positional Analysis of Biotin Genes

To reveal new biotin-related genes, we analyzed putative operon structures and chromosomal clustering of the BBS, *birA*, and *bioY* genes. In some eubacterial and archael genomes, *bioY* is clustered with a hypothetical two-component ABC cassette that encodes ATPase and permease components from the CbiO and CbiQ families, respectively (Table 1; Fig. 4A). The *cbiN-cbiO-cbiQ* operon of *Salmonella typhimurium* encodes the permease, ATPase, and the second permease components, respectively, of a putative cobalt transporter (Roth et al. 1993). Analysis of the phylogenetic trees for the CbiO and CbiQ protein families shows the existence of separate tree branches for the *bioY*-linked CbiO and CbiQ components of putative ABC transporters from *S. meliloti*, *R. capsulatus*, *Agrobacterium tumefaciens*, *Bordetella pertussis*, *Thermomonospora fusca*, two corynebacteria, and *D. radiodurans* (data not shown). The *bioY* genes from *T. pallidum*, *Halobacterium sp.*, and *Archaeoglobus fulgidus* form possible operons with *cbiO* homologs and hypothetical transmembrane proteins (with six predicted TMS) that are not similar to any known protein. Both *Methanosarcina* genomes have BirA-regulated *bioY-cbiO1-cbiO2-cbiQ* operons encoding two paralogous ATPase

| AB | Score | Probability | α1 | α2 | α3 | β1 | β2 |
| --- | --- | --- | --- | --- | --- | --- | --- |
| EC | 5.07 | 100% | PLKLLALLAN..GEFHSGEQLGETLGM | SRAATNKHIQTTLR..DWGVDDFTVPGKGYSL |
| TY | 5.67 | 100% | PLTLSILLAD..GEFHSGEQLGERILGM | SRAAINKHIIQTTLR..DWGVDDFTVPGKGYSL |
| KP | 5.68 | 100% | PLTLSIILAD..GEFHSGEQLGEQLGM | SRAAINKHIIQTTLR..DWGVDDFTVPGKGYSL |
| YP | 4.40 | 90% | PLRLVSILSD..GFFHSGEQLGETLGM | SRAATNKHMQTIR..DWGLDVFTVPGKGYSL |
| VC | 4.92 | 100% | KLAILKQLAD..GDFHSGEVLGAGLGI | SRAAISKHIQGIR..DWGVDDFRVQGKGYQL |
| SH | 5.11 | 100% | KRQILGLLSN..EHFVSGEELATQLGI | SRAAVSKHVDTLEDY..GVAIFSVKGRGYKL |
| MFL | 5.11 | 100% | VFPILRLLAD..GRFFSGEDTARRESVTRSSWNMAIQAAEAL..GVEVFSVRGRGYRL |
| CTE | 4.70 | 100% | QLVLLRRLLAD..GRLHSGESTACELGM | TRAAVKILRKTTSET.LGLAVIAEPGRGYRL |
| PU | 4.93 | 100% | MLTLLNLLKD..GRFFSGQALGAAALGI | SRSAVWKOLQHELEA.ELGLSIHKVRGRGYQL |
| Ppu | 4.97 | 100% | MLKLENLLKD..GRFFSGEALGAAALGV | SRSAVWKOQLQHLES.ELNLTIHKVRGRGYQL |
| PA | 4.38 | 90% | MQTLLKKLQD..GRFFSGEELGAVLGI | SRSAVWKRLOHLEA.EHGLQFKVRGRGYRL |
| NE | 3.84 | 71% | TFAILRMMSD..GNYHSGTTLGQALKV | SRSSISNTLRDLSES..YGLTIHKIPGRGYRW |
| LP | 4.37 | 90% | QHTLMQILGD..GACHSGSELGNALKI | SRSAVWKQINQN..DLGIPITRIPHQGYQL |
| TQ | 4.36 | 90% | PG.LDLLLTE...SCQSGEALAERLGV | SRAAVWSKEARRAE..GYPVEVS.RRGYRI |
| FT | 5.77 | 100% | QLQILTFLLDD..EKYVSGEDIAQKLGI | SRAAISKNIKALKDS..KITVSSNSRVGYRL |
| MBA | 3.96 | 71% | ..RIIKALKDAQKSPVSGEELLGLKLGI | SRTMVWKYIKSLQAD..GYEIESSPKRGYVL |
| MMZ | 3.96 | 71% | ..RIIKALKDAQKT'PVSGEELLGLKLGI | SRTMVWKYIKSLQAD..GYEIESSPKRGYVL |
| HSL | 3.85 | 71% | RRAVLDALADG...PTBGPALADKLGV | SRAAVWHVDAALRDA..GFTIDSG.HDGYT'L |
| PF | 3.48 | 50% | KRKILEELRK..GETVSGDYLASKLGV | SRVAIWKHIRELKEL..GYGIIDAD.KKGYKL |
| PO | 3.88 | 71% | .FRILEMLKA..GKRVSGELIARELGI | SRIAVWKHVKMKSL..GYEIDAR.RNGYRL |
| BE | 4.28 | 90% | ..KLELEFAEANGEFLSGQKISEQLGC | SRAAVWKHIEELRKE..GFLEEAVRRLGYRI |
| ZC | 2.33 | - | ..QLLQVFSEADGEFVSGQTISDKLGC | SRTAVWKHMEDLRSE..GYELEAVRRLGYRI |
| BS | 4.06 | 90% | ..DLIELFSQAGNEFISGQKISDALGO | SRTAVWKHIEELRKE..GYEVEAVRRLGYRL |
| HD | 3.23 | 50% | ..TLLKMILLTAGDDFVSGEKISQAIGQ | SRTAVWKHIEELRKS..GYEVEAQRKGGYRI |
| LI | 3.62 | 71% | ..KLLALETESDGAYLSGQQETADSLOG | SRTAVWQMEAELRKE..GFEIEAVRNRGYRL |
| MCO | 3.30 | 50% | ..KLVALLVVAHGEIPCSGAAIGAGLGI | SRAAVWHIQCLRQR..GYRIEALHRRGYRL |
| HMO | 4.39 | 90% | ..AILAALKHEADG.YLSGEDLSRTLGM | SRSAVWRHIVALRQE..GYEIQAHTRLGYKL |
| CB | 4.55 | 100% | .AIVNILKNSSHTFISGQYISDKLGV | SRTAIWKYINGKKE..GYNIESASKKGYRL |
| DF | 5.02 | 100% | .KIIEILLNKSEFISGEEELSKQLGV | SRAAIWHMKAIKEE..GYNIESVNKKGYRL |
| CA | 3.66 | 71% | .RVLEILKKNSNDFVSGGQRIISDELGI | TRTAIWKNIKSILKDN..GYTILSJTNKGYKL |
| CP | 5.16 | 100% | KESVLKILLENRERSIGEELAKHLSV | SRAAIWKAIRSLRGE..GYNISAVTNRGYQL |
| ST | 2.83 | 25% | .KIYQLLSQ.TDDFVSGEYLADQLSI | SRTSVWSIKSLLENQ..GIQIDSILKHKGYRM |
| SEQ | 2.95 | 25% | .KKIYQLLAC.SDDYVSCEYLAEQKLK | SRTSVWKSIKSLESQ..GLHJESILKHKGYRL |
| PN | 4.65 | 100% | ..AVYQILLSK.ETDYISGEKIAEKLSL | SRTAIWKAIRLEQE..GIEIDSIKNRGYKL |
| SAX | 3.75 | 71% | ..DVLQLLYKNKPNIISGQSIAESLNI | SRTAVWKVIDQKLE..GCKIDSVNHKGHLIL |
| EF | 4.21 | 90% | ..QVLLMKQTPAFLSGEEMAQRLSL | SRTAIWKAINELKKD..GYQITSVQNKGYRL |
| LL | 4.36 | 90% | ..FVLLKLLQQKGNWVSGBDELAEEKLKL | SRESIWKAINTLKRK..GNQIESRKNLGYKY |
| AG | 5.26 | 100% | DYRVYRILSE...NPLSGEKLAKKLGI | SRTAVWKAIVQKLEKC..GVYEESNAS.GYTI |
| DR | 1.28 | - | PARLLPLLTD...RPQSGDDLGRALDL | GRVTVNTLARLQED..GVPVLISS.RAGYAL |
| XF | 2.36 | - | ERVLLTCIGC..S.YVSGNVLASMTGM | TEATIAACIQALR..AVGIPIKSKAEQGYAL |
| HI | -1.37 | - | ..MNFTLLTY...LADCQPKVRSELEKF | SKNLEEDIQOLR..EIGLD.LVDGQDYRL |
| DU | -1.93 | - | ...VDKLLSL....LNKFVVVFG.INL | MLSSILKRFVSL.....GKQVKSJVKKLHVNL |
| AB | -0.43 | - | MAKLELLVS..GVENSLENLTALLGC | DAQQLOQEILOLEE..QQOGIC.FEVENGHLYL |
| VK | 2.16 | - | MQNVLLAS..YQQLSLAELTALLGY | SEQDILHNIIEKLK..QQGIQ.IEQQVQHFRL |
| NM | 1.70 | - | HWRVLABELAD..GLPQHVSQJARMADM | KPQQLINGFWQOMPA...HIRGLLNQHDGYWR |
| BPS | -0.79 | - | .RREHAAMMG..ASPTEPRAAIRPV | KR.RARRTLRWQP....GISSATIRAMNVATT |

Figure 2 Multiple alignment of the BirA N-terminal domains and identification of the HTH motif. The known secondary structure of the *Escherichia coli* BirA is shown in the first row. The α2 and α3 helices form the helix-turn-helix (HTH) structure. The score and the probability of the candidate HTH motif are given. A score of <2.5 is not significant. Non-HTH proteins are boxed, except BirA from *Bacillus cereus*, which is a false-negative prediction (see text). The genome abbreviations are listed in Table 1.

components from the CbiO family. Computational approaches alone cannot explain the possible functional link between the predicted biotin transporter BioY and the putative ABC transporter CbiO–CbiQ, but the obtained data seem to be sufficiently strong to warrant experimental analysis.

Another interesting finding is that bioY from *T. maritima* was found in one operon with genes involved in fatty acid biosynthesis (Table 1). One logical explanation of this linkage is that fatty acid biosynthesis requires biotin as a coenzyme for a hypothetical biotin carboxylase. In addition, positional linkage of the bioY gene with a hypothetical signal peptidase lspA was observed in all cyanobacteria; the functional meaning of this observation is unclear.

Some differences in the gene organization and BirA-mediated regulation of the bioY genes were observed in three Pyrococcus genomes. Strong BirA sites in the common regulatory regions of divergently transcribed bioY and birA genes were predicted in the genomes of P. abyssii and P. furiosus. Besides the regulatory birA gene, these two genomes also contain the second birA gene, encoding BirA without the regulatory domain. In contrast, Pyrococcus horikoshii has no regulatory birA gene, and BirA sites were not found in this genome.

We predicted possible coregulation of various biotin-dependent carboxylases and BirA in some genomes (Table 1). The pycA and pycB genes encoding the biotin-dependent pyruvate carboxylase were found in one candidate operon with birA in two Methanosarcina genomes. These Methanosarcina operons and the single pycA gene from A. fulgidus are preceded by weak BirA sites. The genes encoding subunits of putative propionyl-CoA carboxylase (pccA and pccB) are clustered on the chromosome with the birA gene in all actinobacteria and Halobacterium sp. Finally, in R. capsulatus, birA is located within a long gene cluster encoding components of the malonate decarboxylase Na+ pump. The BirA-regulated gene clusters from C. acetobutylicum, L. lactis, and some archaea contain the birA gene itself; therefore, the biotin repressors from these bacteria can be autoregulated.

The bioC–bioH gene pair is required for the synthesis of pimeloyl-CoA in E. coli. The bioC gene is widely distributed in bacteria, whereas bioH was not found in many bioC-containing bacterial genomes. Instead, we predict several nonorthologous gene displacements of bioH in some of these genomes. It was recently shown that the bioZ gene from the bioABFDZ operon of Mesorhizobium loti can complement bioH of E. coli (Sullivan et al. 2001). The orthologs of bioZ with the same gene organization were found in A. tumefaciens and Brucella melitensis.

Using comparative analysis, we have detected displacement of bioH by another gene, named here bioG, in some proteobacteria (including all Pasteurellaceae), the CFB group of bacteria, and Fusobacterium nucleatum (Table 1). The bioG gene always forms an operon with bioC and other BBS genes in these genomes; furthermore, in Bacteroides fragilis there is a single gene encoding a fused protein BioC–BioG. Interestingly, all gamma-proteobacteria except Pasteurellaceae possess the bioC–bioH gene pair, whereas all Pasteurellaceae have bioC–bioG. Neisseria meningitidis has both bioC–bioH and bioC–bioG gene pairs, and the latter likely has been acquired from Haemophilus influenzae or a closely related bacterium, as the respective genes are highly similar. The phylogenetic tree of the BioC family has a separate branch for the proteins associated with BioG (Fig. 5).

Another bioC-linked gene, named bioK, was found in two cyanobacteria, Synechococcus sp. and Prochlorococcus marinus. The genomes of these bacteria contain the bioFKCDA operon and the bioB gene. Two other cyanobacteria, Synechocystis sp. and Nostoc sp., have all biotin biosynthetic genes except bioC and bioK. Therefore, they possibly use a different pathway for pimeloyl-CoA synthesis.

Using similarity search, we detected that BioC possesses an S-adenosylmethionine binding motif (InterPro entry IPR000379) and belongs to the methyltransferase superfamily. BioK and BioG are not similar to any known protein. The BioZ protein is similar to the 3-oxoacyl-[acyl-carrier-protein] synthase FabH involved in fatty acid biosynthesis in bacteria. Another BioC-linked protein, BioH, possesses the active-site serine of a wide variety of enzymes including esterases, lipases, and peptidases (InterPro entry IPR000379) and is similar to arylesterase EstE from Pseudomonas fluorescens (26% identity). All bioK and bioG genes, as well as most bioH genes, are located immediately upstream of the bioC gene in the biotin operon.

The observed diversity of enzymes for the first step of biotin biosynthesis can reflect either frequent nonorthologous gene displacements, or possible use of different substrates for biotin biosynthesis. In contrast, B. subtilis, S. aureus, Corynebacterium diphtheriae, Aquifex aeolicus, and M. jannaschii possess pimeloyl-CoA synthase encoded by the bioW gene and can use pimelate as a biotin precursor (Table 1).

It remains unclear why the

Figure 3 Maximum likelihood tree of the N-terminal domains of BirA. Domains containing the regulatory HTH motif are shown in solid lines. Other N-terminal domains of BirA (without HTH) are shown as outgroups by broken lines. The proteobacterial and nonproteobacterial subtrees are separated by arrowtail signs. The genome abbreviations are listed in Table 1.

Downloaded from genome.cshlp.org on March 25, 2015 - Published by Cold Spring Harbor Laboratory Press

Rodionov et al.

---

### Conclusions

The biotin-protein ligase BirA is a ubiquitous enzyme in bacteria. In addition, BirA can act as a repressor of transcription when it has the N-terminal DNA-binding domain. Using a global analysis of BirA proteins and DNA-binding sites in available bacterial genomes, we have found that the BirA regulon is widely distributed in eubacteria and archaea. A correlation exists between the presence of D-b-BirA and finding of the BirA sites in bacterial genomes. Conservation of the BirA binding sites across large phylogenetic distances allows us to suggest that D-b-BirA is the first example of an ancient DNA-binding transcriptional factor common to eubacteria and archaea. It is unlikely that numerous BirA regulons in various archaea result from mass gene transfer from bacteria, as this scenario would involve many similar, but independent events (although some cases of horizontal transfer are very clear). In contrast, analysis of regulatory systems for biosynthesis of riboflavin and thiamin showed that they are operated by conserved RNA elements, the RFN element (Vitreschak et al. 2002) and the Thi-box (Miranda-Rios et al. 2001), respectively. These unique regulatory elements are widely distributed in eubacteria and, in addition, several Thi-boxes have been found in archaeal genomes (Vitreschak et al. 2002). Thus, it seems very likely that, in general, the regulatory systems for vitamin biosynthesis are ancient.

Comparative analysis of the biotin regulon in complete genomes resulted in new functional assignments for the bioY, yhfS, and yhfT genes. The first of them, bioY, widely distributed in eubacteria and archaea, is a member of the BirA regulon in all genomes containing D-b-BirA, and it has been predicted to encode a transporter for biotin or biotin-related compounds. Proteins YhfS and YhfT, associated with BioY, can be involved in the metabolic pathway that requires biotin as a coenzyme. The systematic comparison of putative operon structures revealed the conserved gene string bioY-cbiO–cbiQ in some bacterial genomes. Such functional linkage between the putative ABC transporter CbiO–CbiQ and the biotin transporter BioY is enigmatic.

Positional analysis resulted in dissection of novel interesting examples of coregulation of biotin-related genes.

---

### Figure 4 Maximum likelihood trees of the predicted biotin-related transporter BioY (A), the hypothetical long-chain-fatty acid-CoA ligase Yhft (B), and the hypothetical acetyl-CoA-acetyltransferase YhfS (C). Genes predicted to be regulated by BirA are boxed and shown in bold. The co-occurrence of the bioY, yhfS, and yhfT genes in one genome is shown by thick lines. Background colors signify: (black) single bioY gene; (blue) bioY from the biotin biosynthetic operon; (red) bioY in one operon with cbiO–cbiQ; (yellow) bioY in one operon with lspA; (magenta) bioY in the fadH–fabZ–fabK–bioY–fabD operon; (green) bioY positionally linked to the yhfS-yhfT gene pair. The bioY genes positionally linked to birA are shown by broken lines. The genome abbreviations are listed in Table 1.

Comparative analysis of regulation and operon structures failed to identify missing BBS genes in the complete genomes of Clostridium perfringens and C. acetobutylicum. The former has no the bioF and bioA counterparts, whereas the latter lacks only bioF. However, these bacteria possess the predicted biotin transporter BioY. It would be interesting to check if these bacteria can synthesize biotin de novo, and if they can, to search for genes missing in their incomplete BBS pathways.

Downloaded from genome.cshlp.org on March 25, 2015 - Published by Cold Spring Harbor Laboratory Press

Biotin Regulon in Eubacteria and Archaea

thiamin (A. Vitreschak, D. Rodionov, A. Mironov, and M. Gelfand, in prep.) regulons.

## METHODS

Complete and partial bacterial genomes were downloaded from GenBank (Benson et al. 2000). Preliminary sequence data were also obtained from the Web sites of the Institute for Genomic Research (http://www.tigr.org), the University of Oklahoma's Advanced Center for Genome Technology (http://www.genome.ou.edu/), the Wellcome Trust Sanger Institute (http://www.sanger.ac.uk/), the DOE Joint Genome Institute (http://jgi.doe.gov), and the ERGO database (Overbeek et al. 2000; http://ergo.integratedgenomics.com/ERGO/). The gene identifiers from the ERGO database and GenBank are used throughout.

The existence of BirA with an N-terminal DNA-binding domain (D-b-BirA) is a prerequisite to the comparative analysis of the BirA regulons in bacteria. Therefore, the bacterial genomes containing D-b-BirA were selected and divided into two major groups, proteobacterial and nonproteobacterial including archaeal, according to the phylogenetic tree of the DNA-binding domains of D-b-BirA (Fig. 3). Two training sets were composed; each of them included the upstream regions of the biotin biosynthetic genes (operons) from one of the above genomic groups.

For construction of the BirA profiles, we used the "inverted repeat" option in the SignalX program (Mironov et al. 2000) with a 14–16-bp spacer between two 9-bp units of the inverted repeat. The positional nucleotide weights in the profile were defined as

\[
W(b,k) = \log[N(b,k) + 0.5] - 0.25 \sum_{i=A,C,G,T} \log[N(i,k) + 0.5],
\]

where \(N(b,k)\) is the count of nucleotide \(b\) in position \(k\) (Mironov et al. 1999). The score of a candidate site was calculated as the sum of the respective positional nucleotide weights:

\[
Z(b_1 \ldots b_L) = \sum_{k=1}^{L} W(b_k,k),
\]

where \(L\) is the length of the site. All genomes containing D-b-BirA were scanned using the constructed profiles, and the genes with candidate regulatory sites in the upstream regions were selected.

Protein alignment was performed using the Smith-Waterman algorithm implemented in the GenomeExplorer program (Mironov et al. 2000). Orthologous proteins were defined by the best-bidirectional-hits criterion (Tatusov et al. 2000). Distant homologs were identified using PSI-BLAST (Altschul et al. 1997). Multiple sequence alignments were constructed using CLUSTALX (Thompson et al. 1997). Phylogenetic trees were created by the maximum likelihood method implemented in PHYLIP (Felsenstein 1981) and drawn using the GeneMaster program (A.A. Mironov, unpubl.). Prediction of potential transmembrane segments in

Figure 5 Maximum likelihood tree of BioC. The proteins predicted to be associated with (blue) BioH, (red) BioG, (yellow) BioZ, and (green) BioK. The genome abbreviations are listed in Table 1.

Positional linkage between *birA* and genes encoding biotin-dependent carboxylases was found in Actinobacteria and some archaea, and a fraction of these genes were predicted to be regulated by the biotin repressor. Several genomes have divergently transcribed *birA* and *bioY* genes with predicted BirA sites in their common regulatory region. Another example of coregulation of *bioY* with genes of fatty acid biosynthesis in *T. maritima* can be easily explained, as biotin is a required cofactor of carboxylase, the latter being involved in the first step of fatty acid biosynthesis.

The enzymes mediating the first step of the biotin biosynthetic pathway are diverse. BioW and BioC represent two major types of enzymes involved in the synthesis of pimeloyl-CoA, a biotin precursor. Moreover, another type of pimeloyl-CoA synthetase, namely, PauA, was found recently in *Pseudomonas mendocina* (Binieda et al. 1999). In contrast to BioW, PauA belongs to the newly recognized superfamily of acyl-CoA synthetases (Sanchez et al. 2000) and is involved in catabolism rather than biosynthesis. The most interesting observation is that various bacteria have different BioC-associated proteins (BioH, BioG, BioK, or BioZ). It can be explained either by utilization of different sources for biotin biosynthesis or by nonorthologous displacements of the BioC-linked proteins.

This report once again shows the power of comparative genomics for prediction of regulatory sites and functional annotation of genomes, especially when experimental data are limited. In particular, this approach is a powerful tool for prediction of missing transport genes, shown by this study and in the analysis of riboflavin (Vitreschak et al. 2002) and

Genome Research 1515  
www.genome.org

protein sequences was done using TMpred (http://www.ch.embnet.org/software/TMPRED_form.html). Helix-turn-helix (HTH) DNA-binding motifs were analyzed using the weight matrix method (Dodd and Egan 1990; http://npsa-pbil.ibcp.fr/). The significance of a candidate HTH motif in a given sequence was estimated using the HTH score and probability reported by the above program. In addition, the InterPro database (Apweiler et al. 2000; http://www.ebi.ac.uk/interpro/) was used to verify the protein functional and structural annotation.

# ACKNOWLEDGMENTS

The authors are grateful to Andrei Osterman, Olga Vassieva, Sveta Gerdes, and Alexandra Rachmaninova for helpful discussions. This study was partially supported by grants from INTAS (99-1476) and HHMI (55000309). It is a part of the “missing genes” project of Integrated Genomics.

The publication costs of this article were defrayed in part by payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 USC section 1734 solely to indicate this fact.

# REFERENCES

Altschul, S., Madden, T., Schaffer, A., Zhang, J., Zhang, Z., Miller, W., and Lipman, D. 1997. Gapped BLAST and PSI-BLAST: A new generation of protein database search programs. *Nucleic Acids Res.* **25**: 3389–3402.

Apweiler, R., Attwood, T.K., Bairoch, A., Bateman, A., Birney, E., Biswas, M., Bucher, P., Cerutti, L., Corpet, F., Croning, M.D., et al. 2000. InterPro—An integrated documentation resource for protein families, domains and functional sites. *Bioinformatics* **16**: 1145–1150.

Benson, D.A., Karsch-Mizrachi, I., Lipman, D.J., Ostell, J., Rapp, B.A., and Wheeler, D.L. 2000. GenBank. *Nucleic Acids Res.* **28**: 15–18.

Binieda, A., Fuhrmann, M., Lehner, B., Rey-Berthod, C., Frutiger-Hughes, S., Hughes, G., and Shaw, N.M. 1999. Purification, characterization, DNA sequence and cloning of a pimeloyl-CoA synthetase from *Pseudomonas mendocina* 35. *Biochem. J.* **340**: 793–801.

Bower, S., Perkins, J.B., Yocum, R.R., Howitt, C.L., Rahaim, P., and Pero, J. 1996. Cloning, sequencing, and characterization of the *Bacillus subtilis* biotin biosynthetic operon. *J. Bacteriol.* **178**: 4122–4130.

DeMoll, E. 1994. Biosynthesis of biotin and lipoic acid. In *Escherichia coli and Salmonella*. *Cellular and molecular biology* (ed. F.C. Neidhardt), pp. 704–709. American Society for Microbiology, Washington, DC.

Dodd, I.B. and Egan, J.B. 1990. Improved detection of helix-turn-helix DNA-binding motifs in protein sequences. *Nucleic Acids Res.* **18**: 5019–5026.

Eisenberg, M.A. 1985. Regulation of the biotin operon in *E. coli*. *Ann. N.Y. Acad. Sci.* **447**: 335–349.

Felsenstein, J. 1981. Evolutionary trees from DNA sequences: A maximum likelihood approach. *J. Mol. Evol.* **17**: 368–376.

Gelfand, M.S. 1999. Recognition of regulatory sites by genomic comparison. *Res. Microbiol.* **150**: 755–771.

Gloeckler, R., Ohsawa, I., Speck, D., Ledoux, C., Bernard, S., Zinsius, M., Villeval, D., Kisou, T., Kamogawa, K., and Lemoine, Y. 1990. Cloning and characterization of the *Bacillus sphaericus* genes controlling the bioconversion of pimelate into dethiobiotin. *Gene* **87**: 63–70.

Ifuku, O., Miyaoaka, H., Koga, N., Kishimoto, J., Haze, S., Wachi, Y., and Kajiwara, M. 1994. Origin of carbon atoms of biotin. 13C-NMR studies on biotin biosynthesis in *Escherichia coli*. *Eur. J. Biochem.* **220**: 585–591.

Kiyasu, T., Nagahashi, Y., and Hoshino, T. 2001. Cloning and characterization of biotin biosynthetic genes of *Kurthia* sp. *Gene* **265**: 103–113.

Kwon, K., Streaker, E.D., Ruparelia, S., and Beckett, D. 2000. Multiple disordered loops function in corepressor-induced dimerization of the biotin repressor. *J. Mol. Biol.* **304**: 821–833.

Lee, J.M, Zhang, S., Saha, S., Santa Anna, S., Jiang, C., and Perkins, J. 2001. RNA expression analysis using an antisense *Bacillus subtilis* genome array. *J. Bacteriol.* **183**: 7371–7380.

Lemoine, Y., Wach, A., and Jeltsch, J.M. 1996. To be free or not: The fate of pimelate in *Bacillus sphaericus* and in *Escherichia coli*. *Mol. Microbiol.* **19**: 645–647.

Miranda-Rios, J., Navarro, M., and Soberon, M. 2001. A conserved RNA structure (Thi box) is involved in regulation of thiamin biosynthetic gene expression in bacteria. *Proc. Natl. Acad. Sci.* **98**: 9736–9741.

Mironov, A.A., Koonin, E.V., Roytberg, M.A., and Gelfand, M.S. 1999. Computer analysis of transcription regulatory patterns in completely sequenced bacterial genomes. *Nucleic Acids Res.* **27**: 2981–2989.

Mironov, A.A., Vinokurova, N.P., and Gelfand, M.S. 2000. GenomeExplorer: Software for analysis of complete bacterial genomes. *Mol. Biol.* **34**: 222–231.

Overbeek, R., Fonstein, M., D’Souza, M., Pusch, G.D., and Maltsev, N. 1999. The use of gene clusters to infer functional coupling. *Proc. Natl. Acad. Sci.* **96**: 2896–2901.

Overbeek, R., Larsen, N., Pusch, G.D., D’Souza, M., Selkov Jr., E., Kyrpides, N., Fonstein, M., Maltsev, N., and Selkov, E. 2000. WIT: Integrated system for high-throughput genome sequence analysis and metabolic reconstruction. *Nucleic Acids Res.* **28**: 123–125.

Perkins, J.B. and Pero, J.G. 2001. Vitamin biosynthesis. In *Bacillus subtilis and its relatives: From genes to cells* (eds. A.L. Sonenshein et al.), pp. 279–293. American Society for Microbiology, Washington, DC.

Perkins, J.B., Bower, S., Howitt, C.L., Yocum, R.R., and Pero, J. 1996. Identification and characterization of transcripts from the biotin biosynthetic operon of *Bacillus subtilis*. *J. Bacteriol.* **178**: 6361–6365.

Piffeteau, A. and Gaudry, M. 1985. Biotin uptake: Influx, efflux and countertransport in *Escherichia coli* K12. *Biochim. Biophys. Acta* **816**: 77–82.

Roth, J.R., Lawrence, J.G., Rubenfield, M., Kieffer-Higgins, S., and Church, G.M. 1993. Characterization of the cobalamin (vitamin B12) biosynthetic genes of *Salmonella typhimurium*. *J. Bacteriol.* **175**: 3303–3316.

Sanchez, L.B., Galperin, M.Y., and Muller, M. 2000. Acetyl-CoA synthetase from the amitochondriate eukaryote *Giardia lamblia* belongs to the newly recognized superfamily of acyl-CoA synthetases (nucleoside diphosphate-forming). *J. Biol. Chem.* **275**: 5794–5803.

Stok, J.E. and De Voss, J. 2000. Expression, purification, and characterization of BioI: A carbon-carbon bond cleaving cytochrome P450 involved in biotin biosynthesis in *Bacillus subtilis*. *Arch. Biochem. Biophys.* **384**: 351–360.

Sullivan, J.T., Brown, S.D., Yocum, R.R., and Ronson, C.W. 2001. The *bio* operon on the acquired symbiosis island of *Mesorhizobium* sp. strain R7A includes a novel gene involved in pimeloyl-CoA synthesis. *Microbiology* **147**: 1315–1322.

Tatusov, R.L., Galperin, M.Y., Natale, D.A., and Koonin, E.V. 2000. The COG database: A tool for genome-scale analysis of protein functions and evolution. *Nucleic Acids Res.* **28**: 33–36.

Thompson, J.D., Gibson, T.J., Plewniak, F., Jeanmougin, F., and Higgins, D.G. 1997. The CLUSTAL-X windows interface: Flexible strategies for multiple sequence alignment aided by quality analysis tools. *Nucleic Acids Res.* **25**: 4876–4882.

Vitreschak, A.G., Rodionov, D.A., Mironov, A.A., and Gelfand, M.S. 2002. Regulation of riboflavin biosynthesis and transport genes in bacteria by transcriptional and translational attenuation. *Nucleic Acids Res.* **30**: 3141–3151.

# WEB SITE REFERENCES

http://ergo.integratedgenomics.com/ERGO/; ERGO database.

http://jgi.doe.gov; DOE Joint Genome Institute.

http://npsa-pbil.ibcp.fr; Network Protein Sequence Analysis server.

http://www.ch.embnet.org/software/TMPRED_form.html; TMpred Server.

http://www.ebi.ac.uk/interpro/; InterPro database.

http://www.genome.ou.edu; University of Oklahoma’s Advanced Center for Genome Technology.

http://www.sanger.ac.uk; Wellcome Trust Sanger Institute.

http://www.tigr.org/; Institute for Genomic Research.

Received March 27, 2002; accepted in revised form August 9, 2002.

Downloaded from genome.cshlp.org on March 25, 2015 - Published by Cold Spring Harbor Laboratory Press

![CSH PRESS Logo](https://example.com/csh_press_logo.png) GENOME  
RESEARCH

**Conservation of the Biotin Regulon and the BirA Regulatory Signal in Eubacteria and Archaea**

Dmitry A. Rodionov, Andrei A. Mironov and Mikhail S. Gelfand

Genome Res. 2002 12: 1507-1516  
Access the most recent version at doi:10.1101/gr.314502

---

**References**  
This article cites 28 articles, 17 of which can be accessed free at:  
[http://genome.cshlp.org/content/12/10/1507.full.html#ref-list-1](http://genome.cshlp.org/content/12/10/1507.full.html#ref-list-1)

**Creative Commons License**  
This article is distributed exclusively by Cold Spring Harbor Laboratory Press for the first six months after the full-issue publication date (see [http://genome.cshlp.org/site/misc/terms.xhtml](http://genome.cshlp.org/site/misc/terms.xhtml)). After six months, it is available under a Creative Commons License (Attribution-NonCommercial 3.0 Unported License), as described at [http://creativecommons.org/licenses/by-nc/3.0/](http://creativecommons.org/licenses/by-nc/3.0/).

**Email Alerting Service**  
Receive free email alerts when new articles cite this article - sign up in the box at the top right corner of the article or [click here](http://www.example.com).

---

**Gene Link™**  
All Modifications and Oligo Types Synthesized  
Long Oligos • Fluorescent • Chimeric • DNA • RNA • Antisense  

Your wish is our command.

---

To subscribe to Genome Research go to:  
[http://genome.cshlp.org/subscriptions](http://genome.cshlp.org/subscriptions)

---

Cold Spring Harbor Laboratory Press
