
Gene 297 (2002) 123–127

**Sequence of the Escherichia coli O26 O antigen gene cluster and identification of O26 specific genes**

Jocelyne M. D'Souza, Lei Wang${}^{1}$, Peter Reeves*

Department Of Microbiology, School of Molecular and Microbial Biosciences (G08), University Of Sydney, Sydney, NSW 2006, Australia

Received 13 February 2002; received in revised form 19 July 2002; accepted 1 August 2002  
Received by A.M. Campbell

---

### Abstract

Escherichia coli associated with outbreaks of gastroenteritis and hemolytic uremic syndrome include clones with O antigens O157 and O111. However, O26 has emerged as an O antigen present in pathogenic strains, particularly those implicated in cases of infantile gastroenteritis worldwide. The O26 O antigen gene cluster was sequenced. It was found to contain the genes expected for biosynthesis of nucleotide sugars L-rhamnose, N-acetyl-L-fucosamine and N-acetyl-glucosamine, as well genes for O unit flippase, O antigen polymerase and potential transferase genes. By polymerase chain reaction testing against representative strains for the 166 Escherichia coli O serogroups and some randomly selected Gram-negative bacteria, we identified three O antigen genes that are highly specific to O26. This work provides the basis for a sensitive test for the rapid detection of pathogenic clones with the O26 antigen, which has implications for public health, especially in the control of food-borne outbreaks. © 2002 Elsevier Science B.V. All rights reserved.

Keywords: Polymerase chain reaction; Escherichia coli O26 gene cluster; Molecular typing

---

### 1. Introduction

Escherichia coli is a clonal species with clones normally identified by their combination of O and H (and sometimes K) antigens. O antigens such as O157, O111 and O26 associated with EHEC (enterohaemorrhagic) clones are of particular significance in human disease. A clonal population structure exists for these pathogenic *E. coli* strains and studies have indicated that *E. coli* O26 strains are diverse with unique virulence profiles (Zhang et al., 2000). *E. coli* O26 strains have been classified as Shiga toxin-producing *Escherichia coli* (STEC) (Paton and Paton, 1998) distinguished from EHEC on the basis of toxin production. This highly potent Shiga toxin and the capacity to form attaching and effacing lesions play important roles in pathogenesis.

The isolation of *E. coli* O26 strains from infantile diarrhoea was reported as early as 1951, but it was only recently that they were implicated in causing serious enteric disorders such as hemolytic uraemic syndrome (Sramkova et al., 1990). Infection by O26 strains has been reported worldwide, for example, in the United Kingdom (Scotland et al., 1993), Spain (Blanco et al., 1996) and Finland (Keskimaki et al., 1998). Cattle are the main reservoir reported for *E. coli* O26 strains, although other animals such as pigs have also been implicated (Rios et al., 1999). Attempts to detect *E. coli* O26 have had varied success and thus problems arise with it entering the human food chain due to its low infective dose and the ability to survive long term in faeces (Fukushima et al., 1999).

The O antigen is part of the lipopolysaccharide (LPS) present in the outer membrane of Gram-negative bacteria and consists of many repeats of an oligosaccharide unit (O unit). The O antigen contributes to major antigenic variability on the cell surface and on the basis of this variation, 166 forms have been recognized in typical *E. coli* (not including *Shigella* strains). The genes involved in the biosynthesis of O antigens in *E. coli* are generally clustered and flanked by the *galF* and *gnd* genes at the 5′ and 3′ ends, respectively. Several O antigen gene clusters have been cloned and sequenced including those for the *E. coli* O157 (Wang and Reeves, 1998) and O111 (Wang et al., 1998) antigens. Genes specific to these clusters have been identified which may be potentially useful in PCR diagnostic work for the identification of the strain.

In this paper, we report the sequence of the O26 O antigen

---

Abbreviations: EHEC, enterohaemorrhagic *Escherichia coli*; GlcNAc, N-acetyl-D-glucosamine; L-FucNAc, N-acetyl-L-fucosamine; LPS, lipopolysaccharide; PCR, polymerase chain reaction; STEC, Shiga toxin-producing *E. coli*

* Corresponding author. Tel.: +61-2-9351-2536; fax: +61-2-9351-4571.  
E-mail address: reeves@angis.org.au (P. Reeves).

${}^{1}$ Present address: College of Life Science, Nankai University, Tianjin 300071, PR China.

0378-1119/02/$ - see front matter © 2002 Elsevier Science B.V. All rights reserved.  
PII: S0378-1119(02)00876-4

gene cluster. Analysis of the sequence revealed four genes for L-rhamnose biosynthesis, three genes for N-acetyl-L-fucosamine biosynthesis, two putative transferases, the O-unit flippase gene (wzx) and the polymerase gene (wzy). PCR testing against E. coli representing the 16O antigens and a variety of other bacterial strains was undertaken and three sequences for genes (the O unit flippase and polymer-ase and a putative transferase gene) were found to be specific to the O26 antigen.

(O26:K60:H311b) was obtained from the Institute of Medical and Veterinary Science (IMVS), Adelaide, Australia. Five other E. coli O26 strains that were used in this study included O26:K60 strain RVC95 (Central Veterinary Laboratories, Surrey, UK); O26:K-:H11 strains RDZ42 and RDZ114, O26:K-:H-strains RDZ102 and RDZ49 (Departamento de Microbioloxia e Parasitoloxia, Facultade de Veterinaria, Universidade de Santiago, Spain). All other E. coli serotypes and bacterial species used are listed in Table 1.

2. Materials and methods

2.1. Bacterial strains and plasmids

All plasmids used in this study were maintained in E. coli K-12 strain JM109. The E. coli O26 type strain

Table 1  
Bacterial strains and PCR pools used for testing of E. coli O26 primers

| Pool no. | Strains from which chromosomal DNA is included in the pool                                                                                          | Source |
|----------|--------------------------------------------------------------------------------------------------------------------------------------------------|--------|
| 1        | E. coli type strains for O serotypes 1, 2, 3, 4, 10, 16, 18 and 39                                                                                 | IMVS    |
| 2        | E. coli type strains for O serotypes 40, 41, 43, 49, 71, 73, 88 and 100                                                                           | IMVS    |
| 3        | E. coli type strains for O serotypes 102, 109, 119, 120, 121, 125, 126 and 137                                                                    | IMVS    |
| 4        | E. coli type strains for O serotypes 138, 139, 149, 7, 5, 6, 11 and 12                                                                            | IMVS    |
| 5        | E. coli type strains for O serotypes 13, 14, 15, 17, 19, 20, 21 and 22                                                                           | IMVS    |
| 6        | E. coli type strains for O serotypes 23, 24, 25, 27, 28, 29, 30 and 32                                                                           | IMVS    |
| 7        | E. coli type strains for O serotypes 33, 34, 35, 36, 37, 38, 42 and 43                                                                           | IMVS    |
| 8        | E. coli type strains for O serotypes 44, 45, 46, 50, 51, 52, 53 and 54                                                                           | IMVS    |
| 9        | E. coli type strains for O serotypes 55, 56, 57, 58, 59, 60, 61 and 62                                                                           | IMVS    |
| 10       | E. coli type strains for O serotypes 63, 64, 65, 66, 68, 69, 70 and 74                                                                           | IMVS    |
| 11       | E. coli type strains for O serotypes 75, 76, 77, 78, 79, 80, 81 and 82                                                                           | IMVS    |
| 12       | E. coli type strains for O serotypes 83, 84, 85, 86, 87, 89, 90 and 91                                                                           | IMVS    |
| 13       | E. coli type strains for O serotypes 92, 95, 96, 97, 98, 99, 101 and 103                                                                         | IMVS    |
| 14       | E. coli type strains for O serotypes 104, 105, 106, 107, 108, 110, 111 and 112a                                                                  | IMVS    |
| 15       | E. coli type strains for O serotypes 112b, 113, 114, 115, 116, 117, 118 and 123                                                                  | IMVS    |
| 16       | E. coli type strains for O serotypes 165, 166, 167, 168, 169, 170, 171 and 172                                                                   | IMVS    |
| 17       | E. coli type strains for O serotypes 173, 127, 128, 129, 130, 131, 132 and 133                                                                   | IMVS    |
| 18       | E. coli type strains for O serotypes 134, 135, 136, 140, 142, 143 and 144                                                                        | IMVS    |
| 19       | E. coli type strains for O serotypes 145, 146, 147, 148, 150, 151, 152 and 153                                                                   | IMVS    |
| 20       | E. coli type strains for O serotypes 154, 155, 156, 157, 158, 159, 160 and 161                                                                   | IMVS    |
| 21       | E. coli type strains for O serotypes 163, 164, 162, 8, 9 and 124                                                                                 | IMVS    |
| 22       | As for pool 2, plus E. coli O26:K60:H-311B Orskov                                                                                                 | IMVS    |
| 23       | As for pool 2, plus E. coli O26:K60 (RVC95)                                                                                                      | IMVS    |
| 24       | As for pool 2, plus E. coli O26:K-:H11 (RDZ42)                                                                                                   | IMVS    |
| 25       | As for pool 2, plus E. coli O26:K-:H11 (RDZ114)                                                                                                  | IMVS    |
| 26       | As for pool 2, plus E. coli O26:K-:H- (RDZ102)                                                                                                   | IMVS    |
| 27       | As for pool 2, plus E. coli O26:K-:H- (RDZ49)                                                                                                    | IMVS    |
| 28       | Streptococcus pneumoniae strains type 4, 6A, 6B, 7, 12F, 18C and 19F                                                                             | IMVS    |
| 29       | Shigella strains: S. flexneri type 2b 3019, S. sonnei 3354g, S. boydii type 1h and 17h, S. dysenteriae type 2h, 12h and 13h                           | IMVS    |
| 30       | Vibrio cholerae strains with O-groups O1, O2, O5, O8, O11, O28 and O31                                                                          | IMVS    |

a Institute of Medical and Veterinary Science, Adelaide, Australia.  
b 123 from IMVS; the rest from Statens Serum Institut, Copenhagen, Denmark.  
c 172 and 173 from Statens Serum Institut, Copenhagen, Denmark; the rest from IMVS.  
d Central Veterinary Laboratories, Surrey, UK.  
e J. Blanco, Departamento de Microbioloxia e Parasitoloxia, Facultade de Veterinaria, Universidade de Santiago, Spain.  
f James Paton, Molecular Microbiology Unit Women's and Children's Hospital, North Adelaide, Australia.  
g Swiss Serum and Vaccine Institute, Berne, Switzerland.  
h Canadian National Laboratory for Enteric Pathogens (NLEP).  
i Toshio Shimada, National Institute of Health, Tokyo, Japan.

amplify the O antigen gene cluster by long-range PCR, using the Expand Long Template PCR system from Boehringer. The PCR cycles were as follows: denaturation at 94 °C for 10 s, annealing at 58 °C for 30 s and extension at 68 °C for 15 min. The long-range PCR products were subjected to DNaseI digestion, cloned into the plasmid pGEM-TE to produce a bank using the method previously described (Wang and Reeves, 1998), and the inserts were sequenced.

2.3. Sequencing and analysis

DNA templates for sequencing were prepared with a 96-well-format plasmid DNA miniprep kit from Advanced Genetic Technologies Corp. by the procedure developed by The Institute for Genomic Research (Utterback et al., 1995). Sequencing was carried out by SUPAMAC (Sydney University and Prince Alfred Macromolecular Analysis Centre) and AGRF (Australian Genomic Research Facility) using Applied Biosystems 377 automated DNA sequencers. Sequence data was assembled using the PHRAP/PHRED package of the University of Washington Genome Center, and the program CONSED (Gordon et al., 1998). The sequencing data was analysed by using the Australian National Genomic Information Service (ANGIS). The BLAST database was used to search for sequence matches and the algorithm previously described (Eisenberg et al., 1984) to identify potential transmembrane segments from the amino acid sequence.

2.4. Specificity assay by PCR

Chromosomal DNA was isolated using the Promega Genomic isolation kit and checked by electrophoresis. The quality of chromosomal DNA from each *E. coli* strain was checked by PCR amplification of the *mdh* gene (coding for malate dehydrogenase) using primers as described in Wang and Reeves (Wang and Reeves, 1998). A total of 30 pools (Table 1) were used (each pool containing DNA from eight different strains). Pools were screened using primers based on the O26 O unit processing genes and transferase genes. PCR was carried out in a volume of 25 μl of which 10 μl was run on a 1% agarose gel to check for amplification. For pools that were positive, the eight individual strains were screened to determine which yielded the positive result.

2.5. Nucleotide sequence accession number

The DNA sequence of the *E. coli* O26 O antigen gene cluster has been deposited in GenBank under the accession number AF529080.

3. Results and discussion

3.1. Sequencing

Escherichia coli O antigen gene clusters are generally located between the *galF* and *gnd* genes, two housekeeping genes that do not participate in O antigen synthesis. Primers based on these genes were used to amplify the gene cluster by long-range PCR. A sequence of 13,270 bases was obtained that covered the DNA from *galF* (positions 1–1054) to *gnd* (positions 13,051–13,270) of the O26 type strain. The O26 O antigen gene cluster contains 12 open reading frames (Fig. 1) all in the same transcriptional direction from *galF* to *gnd*. The respective nucleotide and deduced amino acid sequences were used to search databases for indication of possible functions.

3.2. O26 O antigen genes

The O26 O antigen contains the sugars L-rhamnose (L-Rha), N-acetyl-L-fucosamine (L-FucNAc) and N-acetyl-D-glucosamine (GlcNAc) (Manca et al., 1996) (Fig. 2). It was anticipated that the cluster would contain genes responsible for the synthesis of nucleotide precursors for the first two sugars and also genes for sugar transferases of these sugars to make the single O unit, the O unit flippase and polymerase.

The first four genes were identified as *rmlB* (dTDP-glucose 4,6-dehydratase), *rmlD* (dTDP-4-dehydrorhamnose reductase), *rmlA* (dTDP-glucose synthase) and *rmlC* (dTDP-4-dehydrorhamnose 3,5-epimerase) respectively, based on their high level of similarity to other dTDP-L-rhamnose biosynthesis genes.

*fnl1*, *fnl2* and *fnl3* shared 63, 40 and 49% identity with *orf13*, *orf14* and *orf15*, respectively, of the capsular polysaccharide of *Streptococcus pneumoniae* type 4 (Jiang et al., 2001). These three genes are also present in the same order in the five reported gene clusters coding for L-FucNAc containing structures (see also www.microbio.usyd.edu.au/BPGD/default.htm for details). These are clearly involved in biosynthesis of the sugar L-FucNAc (Jiang et al., 2001), but until there is direct confirmation of a pathway, these genes will not be allocated final gene names (*fnlA*, *fnlB* and *fnlC* in pathway order) but have been renamed *fnl1*, *fnl2* and *fnl3* in map order.

3.3. Transferase genes

Although three sugars are present in the O26 O antigen, only two transferases were expected (for L-rhamnose and L-FucNAc). The *wecA* gene, known to encode the transferase for adding UDP-GlcNAc as the first sugar, resides outside the gene cluster (Alexander and Valvano, 1994).

The deduced product of *orf7* shares 19% identity to a putative rhamnosyl transferase (*cps19bH* gene) in the capsule 19B gene cluster of *Streptococcus pneumoniae*. We suggest that *orf7* encodes a rhamnosyl transferase and it has been named *wbuA*.

*Orf11* shares 19% identity to a putative L-FucNAc transferase (WciJ) from the capsule 4 cluster of *S. pneumoniae* (Jiang et al., 2001). It also shares sequence similarity to other transferases such as a putative glycosyl transferase from *Leptospira borgpetersenii* (Kalambaheti et al., 1999). It has been named WbuB and we suggest that it is respon-

J.M. D'Souza et al. / Gene 297 (2002) 123–127

| $galF$ | $mlB$ | $mlD$ | $mlA$ | $mlC$ | $wzx$ | $wzy$ | $wbuA$ | $fnl1$ | $fnl2$ | $fnl3$ | $wbuB$ | $wbuC$ | $gnd$ |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| kb | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 |
| orf# | 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12 |
| G+C content | 22% | 23% | 22% | 19% | 16% | 16% | 16% | 20% | 19% | 21% | 20% | 21% |

Fig. 1. O antigen gene cluster of *E. coli* O26.

sible for the linkage α-L-FucNAc-(1–3)-α-GlcNAc: this assignment is strongly supported by data reported below.

Most *E. coli* O antigens are synthesized by the Wzx/Wzy-dependent system, whereby the repeat units are constructed on the inner face of the cytoplasmic membrane and transported across the membrane by a flippase (Wzx) before being polymerized by the Wzy polymerase (Reeves, 1994).

Orf5 is predicted to encode an integral inner membrane protein with 12 transmembrane segments. This protein was grouped with direct homologues being Wzx proteins of O antigen gene clusters of *E. coli* K-12 (Stevenson et al., 1994), *E. coli* Dysenteriae (Klena and Schnaitman, 1993) and *Yersinia enterocolitica* O3 (Skurnik et al., 1995). Analysis using the BLOCKMAKER program (Henikoff et al., 1995) revealed four motifs (10, 14, 28 and 30 amino acid residues) which are conserved within this group of proteins. The program PSI-BLAST (Altschul et al., 1997) was used to compare these motifs and search the Genpept database: the three input Wzx proteins and other distantly related Wzx proteins were retrieved (E-value ≤ 1e-20), confirming the designation of *orf5* as *wzx*.

A presumptive *wzy* gene (*orf6*) was identified as encoding an integral inner membrane protein with nine predicted transmembrane segments. Based on sequence comparisons this putative protein was grouped with the Wzy protein of the O antigen gene cluster of *E. coli* K12 (Stevenson et al., 1994). Analysis using the BLOCKMAKER program revealed 5 conserved motifs (3 × 16, 28 and 42 amino acid residues) which were then used to run PSI-BLAST to search the Genpept database: the *E. coli* O157 Wzy protein (Wang and Reeves, 1998) and other distantly related Wzy proteins were retrieved (E-value ≤ 1e-174), confirming *orf6* as the *wzy* gene.

The deduced amino acid sequence of the segment from positions 12,533 to 12,940 (*wbuC*) shows 49% similarity to the C-terminal half of WaaK of *E. coli* K-12. WaaK is a GlcNAc transferase. However, all the necessary genes required for the transfer and the biosynthesis of nucleotide

![Diagram](attachment:diagram.png)

Fig. 2. Structure of the O26 O antigen (Manca et al., 1996).

Table 2  
PCR testing of the specificity of O26 genes  

| Gene   | Base positions of gene$^{a}$ | Base positions of forward/reverse primer |
|--------|-----------------------------|-----------------------------------------|
| wzx    | 4893–6155                  | 4922–4942/6069–6089                    |
| wzy    | 6221–7243                  | 6267–6287/7223–7243                    |
| wbuA   | 7259–8053                  | 7291–7309/8017–8036                    |
| wbuB   | 11334–12542                | 11370–11388/12498–12516                |

$^{a}$ These nucleotide regions are based on the sequence of accession number AF529080.

### 3.5. General conclusions

The *E. coli* O26 O antigen gene cluster has been sequenced and all expected genes identified. Consistent with other sequenced *E. coli* O antigen gene clusters, all the genes within the O26 gene cluster have a low G + C content, indicating transfer from another species.

Currently, methods for the identification of *E. coli* O26 strains have proven difficult and most are based on assays involving toxin production. We have identified three genes (wzx, wzy, and wbuA) with sequences highly specific for *E. coli* O26.

Diagnostic screening for O26 strains is important for public health especially in the food industry. We believe that these sequences would be useful in PCR-based methods for the identification of *E. coli* O26, for a faster response in the control of foodborne outbreaks.

### Acknowledgements

We thank all the people and institutes listed in Table 1 for kindly supplying strains. This investigation was supported by the Australian Research Council. J.M.D. is a recipient of a Faculty of Agriculture Scholarship of the University of Sydney.

### References

Alexander, D.C., Valvano, M.A., 1994. Role of the *rfb* gene in the biosynthesis of the *Escherichia coli* O7-specific lipopolysaccharide and other O-specific polysaccharides containing N-acetylglucosamine. J. Bacteriol. 176, 7079–7084.

Altschul, S.F., Madden, T.L., Schaffer, A.A., Zhang, J., Zhang, Z., Miller, W., Lipman, D.J., 1997. Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res. 25, 3398–3402.

Bastin, D.A., Reeves, P.R., 1995. Sequence and analysis of the O antigen gene (*rfb*) cluster of *Escherichia coli* O111. Gene 164, 17–23.

Blanco, M., Blanco, J.E., Blanco, J., Gonzalez, E.A., Alonso, M.P., Maas, H., Jansen, W.H., 1996. Prevalence and characteristics of human and bovine verotoxigenic *Escherichia coli* strains in Galicia (north-western Spain). Eur. J. Epidemiol. 12, 13–19.

Eisenberg, D., Schwarz, E., Komaromy, M., Wall, R., 1984. Analysis of membrane and surface protein sequences with the hydrophobic moment plot. J. Mol. Biol. 179, 125–142.

Fukushima, H., Hoshina, K., Gomyoda, M., 1999. Long-term survival of

Shiga toxin-producing *Escherichia coli* O26, O111 and O157 in bovine feces. Appl. Environ. Microbiol. 65, 5177–5181.

Gordon, D., Abajian, C., Green, P., 1998. CONSED – A graphical tool for sequence finishing. PCR Methods Appl. 8, 195–202.

Henikoff, S., Henikoff, J.G., Alford, W.J., Pietrokovski, S., 1995. Automated construction and graphical presentation of protein blocks from unaligned sequences. Gene 163, GC17–GC26.

Jiang, S., Wang, L., Reeves, P.R., 2001. Molecular characterization of *S. pneumoniae* types 4, 6B, 8, and 18C capsular polysaccharide gene clusters. Infect. Immun. 69, 1244–1255.

Kalambaheti, T., Bulach, D.M., Rajakumar, K., Adler, B., 1999. Genetic organization of the lipopolysaccharide O-antigen biosynthetic locus of *Leptospira borgpetersenii* serovar Hardjobovis. Microb. Pathog. 27, 105–117.

Keskimaki, M., Saari, M., Heiskanen, T., Shitonen, A., 1998. Shiga-toxin producing *Escherichia coli* in Finland from 1990 through 1997: Prevalence and characteristics of isolates. J. Clin. Microbiol. 36, 3641–3646.

Klena, J.D., Schnaitman, C.A., 1993. Function of the *rfb* gene cluster and the *rfe* gene in the synthesis of O antigen by *Shigella dysenteriae* 1. Mol. Microbiol. 9, 393–402.

Kogan, G., Haraguchi, G., Hull, S.I., Hull, R.A., Shashkov, A.S., Jann, B., Jann, K., 1993. Structural analysis of O4-reactive polysaccharides from recombinant *Escherichia coli*: changes in the O-specific polysaccharide induced by cloning of the *rfb* genes. Eur. J. Biochem. 214, 259–265.

Manca, M.C., Weintraub, A., Widmalm, G., 1996. Structural studies of the *Escherichia coli* O26 O-antigen polysaccharide. Carbohydr. Res. 281, 155–160.

Paton, J.C., Paton, A.W., 1998. Pathogenesis and diagnosis of Shiga toxin-producing *Escherichia coli* infections. Clin. Microbiol. Rev. 11, 450–479.

Reeves, R.R., 1994. Biosynthesis and assembly of lipopolysaccharide. In: Neuberger, A., van Deenen, L.L.M. (Eds.). Bacterial Cell Wall, Elsevier Science, Amsterdam, pp. 281–314.

Rios, M., Prado, V., Trucksis, M., Arellano, C., Borie, C., Alexandre, M., Fica, A., Levine, M., 1999. Clonal diversity of Chilean isolates of enterohemorrhagic *Escherichia coli* from patients with hemolytic-uremic syndrome, asymptomatic subjects, animal reservoirs and food products. J. Clin. Microbiol. 37, 778–781.

Scotland, S., Willshaw, G., Smith, H., Said, B., Stokes, N., Rowe, B., 1993. Virulence properties of *Escherichia coli* strains belonging to serogroups O26, O111 and O128 isolated in the United Kingdom in 1991, from patients with diarrhea. Epidemiol. Infect. 111, 429–438.

Skurnik, M., Venho, R., Toivanen, P., Alhendy, A., 1995. A novel locus of *Yersinia enterocolitica* serotype O-3 involved in lipopolysaccharide outer core biosynthesis. Mol. Microbiol. 17, 575–594.

Sramkova, L., Bielaszewska, M., Janda, J., Blahova, K., Hausner, O., 1990. Vero cytotoxin-producing strains of *Escherichia coli* in children with haemolytic uraemic syndrome and diarrhoea in Czechoslovakia. Infect. 18, 204–209.

Stevenson, G., Neal, B., Liu, D., Hobbs, M., Packer, N.H., Batley, M., Redmond, J.W., Lindquist, L., Reeves, P.R., 1994. Structure of the O-antigen of *E. coli* K-12 and the sequence of its *rfb* gene cluster. J. Bacteriol. 176, 4144–4156.

Utterback, T.R., McDonald, L.A., Fuldner, R.A., 1995. A reliable, efficient protocol for 96-well plasmid DNA miniprep with rapid DNA quantification for high-throughput automated DNA sequencing. Genome Sci. Technol. 1, 1–8.

Wang, L., Reeves, P.R., 1998. Organization of *Escherichia coli* O157 O antigen gene cluster and identification of its specific genes. Infect. Immun. 66, 3545–3551.

Wang, L., Curd, H., Qu, W., Reeves, P.R., 1998. Sequencing of *Escherichia coli* O111 O antigen gene cluster and identification of O111 specific genes. J. Clin. Microbiol. 36, 3182–3187.

Zhang, W.L., Bielaszewska, M., Liesegang, A., Tschape, H., Schmidt, H., Bitzan, M., Karch, H., 2000. Molecular characteristics and epidemiological significance of Shiga toxin-producing *Escherichia coli* O26 strains. J. Clin. Microbiol. 38, 2134–2140.
