
Eur. J. Biochem. 269, 5659–5667 (2002) © FEBS 2002 doi:10.1046/j.1432-1033.2002.03275.x

# Molecular cloning of columbamine O-methyltransferase from cultured *Coptis japonica* cells

Takashi Morishige${}^{1}$, Emilyn Dubouzet${}^{2}$, Kum-Boo Choi${}^{2}$, Kazufumi Yazaki${}^{1,2}$ and Fumihiko Sato${}^{1,2}$

${}^{1}$Division of Applied Life Sciences, Graduate School of Agriculture, Kyoto University, Japan; ${}^{2}$Division of Integrated Life Science, Graduate School of Biostudies, Kyoto University, Japan

To identify all of the $O$-methyltransferase genes involved in isoquinoline alkaloid biosynthesis in *Coptis japonica* cells, we sequenced 1014 cDNA clones isolated from high-alkaloid-producing cultured cells of *C. japonica*. Among them, we found all three reported $O$-methyltransferases and an $O$-methyltransferase-like cDNA clone (CJEST64). This cDNA was quite similar to $S$-adenosyl-$L$-methionine:coclaurine $6-O$-methyltransferase and $S$-adenosyl-$L$-methionine:iso-flavone $7-O$-methyltransferase. As $S$-adenosyl-$L$-methionine:columbamine $O$-methyltransferase, which catalyzes the conversion of columbamine to palmatine, is one of the remaining unelucidated components in isoquinoline alkaloid biosynthesis in *C. japonica*, we heterologously expressed the protein in *Escherichia coli* and examined the activity of columbamine $O$-methyltransferase. The recombinant protein clearly showed $O$-methylation activity using columbamine, as well as $(S)$-tetrahydrocolumbamine, $(S)$-, $(R,S)$-scoulerine and $(R,S)$-2,3,9,10-tetrahydroprotoberine as substrates. This result clearly indicated that EST analysis was useful for isolating the candidate gene in a relatively well-characterized biosynthetic pathway. The relationship between the structure and substrate recognition of the $O$-methyltransferases involved in isoquinoline alkaloid biosynthesis, and a reconsideration of the biosynthetic pathway to palmatine are discussed.

**Keywords:** alkaloid biosynthesis; methyltransferase; *Coptis japonica*; palmatine; columbamine.

Higher plants produce a wide variety of chemicals, including more than 25 000 terpenoids, approximately 12 000 alkaloids and approximately 8000 phenolic compounds [1]. Thus, secondary metabolism in plant cells could be a useful source of these chemicals, especially alkaloids, many of which have high biological activities and are used as medicines.

Isoquinoline alkaloid biosynthesis is the most well-characterized pathway in plant cells and some biosynthetic genes have been isolated [2,3 and references cited therein]. However, the conventional method for isolating the corresponding biosynthetic genes based on the purified enzymes is a rate-limiting step for characterization, even though this approach can be very useful when we have no information about the desired enzyme(s). Although a useful approach is to isolate a group of genes based on their structural similarity, such as $O$-methyltransferase (OMT) or P450 motifs, this approach faces the problem of identifying biological activity due to the high redundancy of related genes.

On the other hand, recent scientific and technological advances now enable the high-through-put sequencing of cDNA clones, and we can analyze these expressed sequence tag (EST) using on-line databases and search tools. These ESTs are also useful for evaluating mRNA transcripts to estimate their biological function.

To evaluate the effectiveness of this technology, we sequenced the cDNA clones isolated from high-alkaloid-producing cultured cells of *Coptis japonica*. As we previously characterized some of the OMTs in isoquinoline alkaloid biosynthesis in *Coptis* cells, we focused on the isolation of the another unelucidated OMT in this biosynthetic pathway; i.e. $S$-adenosyl-$L$-methionine:columbamine $O$-methyltransferase (CoOMT), which catalyzes the transfer of the $S$-methyl group of $S$-adenosyl-$L$-methionine (AdoMet) to the 2-hydroxyl group of columbamine to form palmatine [4] (Fig. 1).

Methyltransferases are essential for directing intermediates to specific biosynthetic pathways [5]. Each methyltransferase in the biosynthesis of palmatine (i.e. $S$-adenosyl-$L$-methionine:norcoclaurine $6-O$-methyltransferase (6OMT) [6–10]; $S$-adenosyl-$L$-methionine:coclaurine $N$-methyltransferase (CNMT) [11–14]; $S$-adenosyl-$L$-methionine:$3'$-hydroxy-$N$-methylcoclaurine $4'$-$O$-methyltransferase (4'OMT) [10,15]; $S$-adenosyl-$L$-methionine:scoulerine $9-O$-methyltransferase (SMT) [16–18]; and CoOMT [4]) has strict substrate specificity, despite the structural

Correspondence to F. Sato, Division of Integrated Life Sciences, Graduate School of Biostudies, Kyoto University, Kyoto, 606-8502, Japan. Fax: + 81 75 753 6398, Tel.: + 81 75 753 6381, E-mail: fumihiko@kais.kyoto-u.ac.jp

**Abbreviations:** OMT, $O$-methyltransferase; EST, expressed sequence tag; CoOMT, $S$-adenosyl-$L$-methionine:columbamine $O$-methyltransferase; AdoMet, $S$-adenosyl-$L$-methionine; AdoHcy, $S$-adenosyl-$L$-homocysteine; 6OMT, $S$-adenosyl-$L$-methionine:norcoclaurine $6-O$-methyltransferase; CNMT, $S$-adenosyl-$L$-methionine:coclaurine $N$-methyltransferase; 4'OMT, $S$-adenosyl-$L$-methionine:$3'$-hydroxy-$N$-methylcoclaurine $4'$-$O$-methyltransferase; SMT, $S$-adenosyl-$L$-methionine:scoulerine $9-O$-methyltransferase; IOMT, $S$-adenosyl-$L$-methionine:isoflavone $7-O$-methyltransferase; LC-MS, liquid chromatography-mass spectrometry.

**Note:** The nucleotide sequence for columbamine $O$-methyltransferase has been deposited in the GenBank database under GenBank Accession Number AB073908.

(Received 13 March 2002, revised 2 July 2002, accepted 20 September 2002)

5660 T. Morishige et al. (Eur. J. Biochem. 269) © FEBS 2002

L-Tyrosine → (S)-Norcoclaurine → (S)-Coclaurine  
AdoMet AdoHcy  

(S)-Reticuline ← (S)-3'-Hydroxy-N-methylcoclaurine → (S)-N-Methylcoclaurine  
4'OMT AdoHcy AdoMet  

(S)-Scoulerine → (S)-Tetrahydrocolumbamine → Berberine  
SMT AdoMet AdoHcy  

(S)-Tetrahydropalmatine → Columbamine → Palmitine  
CoOMT? AdoMet AdoHcy CoOMT AdoMet AdoHcy  

Fig. 1. Schematic biosynthetic pathway of palmitine. Dotted lines and question marks indicate the possible pathway based on the present findings.

similarities of the various substrates. Due to their importance in the production of pharmaceutically important alkaloids and their strict substrate recognition, these methyltransferases, especially OMTs, have been well characterized [6–17] and their cDNA has been isolated [10,14,18]. However, CoOMT was characterized using only partially purified enzymes [4] and its cDNA has not yet been isolated.

We report here the first isolation of CoOMT from *C. japonica* ESTs, the expression of functionally active recombinant enzyme, and its characterization. Characterization of its specificity showed that *Coptis* CoOMT could methylate tetrahydrocolumbamine as well as columbamine, and suggests that the biosynthetic pathway to palmitine should be reconsidered.

## EXPERIMENTAL PROCEDURES

### Cultured cells

The original cultured cells were induced from rootlets of *C. japonica* Makino var. dissecta (Yatabe) Nakai. A cell line (156-1) that produces large amounts of alkaloids was subcultured as described elsewhere [19]. Ten-day-old-cultured cells were harvested and used for the extraction of mRNA.

### Chemicals

(R,S)-Norlaudanosoline and (R,S)-laudanosoline were purchased from Aldrich, and berberine was from Wako Pure Chemical Industries, Ltd. (R)- and (S)-coclaurine were the gift of Dr Nagakura of Kobe Women's College of Pharmacy, and (S)-reticuline was a gift from Dr Facchini of the University of Calgary. The other alkaloids were generous gifts from Mitsui Petrochemical Industries, Ltd.

### Chemical synthesis of columbamine

Columbamine was synthesized and purified from berberine by two-step reactions according to the method of Cava and Reed [20,21].

### Construction and sequencing of cDNA library of *C. japonica*

Poly(A)<sup>+</sup> RNA was isolated, and a cDNA library was constructed as described elsewhere [14]. The cDNA

fragments were ligated into pDR196 vector [22]. Sequencing of the cDNA library was performed using a MegaBACE 1000 DNA Sequencing System (Amersham Pharmacia Biotech) in accordance with the manufacturer's instructions. Overall, 1014 ESTs were obtained in sufficient length and these sequences were annotated using a BLAST search (http://www.ncbi.nlm.nih.gov/BLAST/) (E. Dubouzet *et al.* in preparation). During the BLAST search, we found an unidentified OMT cDNA (CJEST64) in addition to the sequences of reported *Coptis* OMTs. To obtain more information about the sequence of this CJEST64, the complete cDNA sequence was determined with deletion clones of CJEST64 and a DSQ-2000 L DNA sequencer (Shimadzu) with fluorescein isothiocyanate-labelled primers.

### Isolation of full-length cDNA

To isolate full-length cDNA of the unknown OMT (CJEST64), rapid amplification of the 5′ end of cDNA (5′RACE) was carried out using a Marathon cDNA Amplification Kit (Clontech). Gene-specific antisense primer (5′-CTCCAAAAC TGAGAACTCTTCCG-3′) was designed based on the nucleotide sequence of CJEST64. PCR fragments were isolated and subcloned into pT7Blue vector (Novagen), and their nucleotide sequences were determined.

### Construction of an expression vector for full-length CJEST64

An expression vector was constructed for full-length CJEST64 cDNA without the fused peptide derived from the vector sequence in pET-21d vector (Novagen). The full-length cDNA was first prepared by PCR with a Marathon adaptor primer (Clontech) and a forward primer 5′-TTGT TCTAAGGCCATGGTATCTCCG-3′ to introduce a *NcoI* site (CCATGG) at the start codon. A restriction site was introduced by changing the underlined base. pET-21d was digested with *NcoI* and *XhoI*, and the PCR product was ligated into the vector. This construct was completely sequenced to confirm that no changes were introduced by the subcloning process.

#### Heterologous expression of full-length CJEST64 in *Escherichia coli*

The expression vector for full-length CJEST64 cDNA was introduced into *Escherichia coli* BL21 (DE3). After induction with 1 mM isopropylthiogalactoside, *E. coli* cells were incubated at 25 °C for 5 h, and then harvested and extracted in extraction buffer (0.1 M Tris/HCl, pH 8.0, containing 10% glycerol, 5 mM EDTA and 10 mM 2-mercaptoethanol). After centrifugation at 12,000 g for 10 min, the supernatant was desalted through an NAP-5 column (Amersham Pharmacia Biotech) and used to measure CoOMT activity.

#### Measurement of O-methylation activity

The standard CoOMT reaction mixture (50 μL) consisted of 100 mM potassium phosphate (pH 7.8), 25 mM sodium ascorbate, 1 mM columbamine, 3 mM AdoMet and the enzyme preparation (*c*. 100 μg protein). The assay mixture was incubated at 30 °C for 1 h, after which the reaction was terminated by the addition of methanol. After protein precipitation, the reaction product was detected by reversed-phase HPLC (mobile phase, 35% acetonitrile/H₂O containing 1% acetic acid; column, TSKgel ODS-80Tm (4.6 × 250 mm; TOSOH); flow rate, 0.5 mL·min⁻¹; detection, absorbance measurement at 280 nm. Mass spectra were obtained with an LCMS-2010 (Shimadzu).

Tetrahydrocolumbamine was prepared from scoulerine via a scoulerine 9-O-methyltransferase (SMT) reaction. The SMT reaction was carried out as described previously [18] with a modification. In brief, the expression of SMT in *E. coli* was induced using a pET expression system and the *E. coli*

---

**Fig. 2. Amino acid sequence alignment of the O-methyltransferases in palmatine biosynthesis. Motifs A, B, C; conserved sequence motifs in plant S-adenosyl-L-methionine-dependent methyltransferase. Amino acids conserved in all of the sequences are boxed and similar ones are shaded.**

| CJEST64 | 6OMT | 4′OMT | SMT |
|---------|------|-------|-----|
| DTPNTFQNDD EIKAQAVWK-HMFGFAEBTIMRST | EVKKDNLSSQ-AKLWN-FIYGFAESLVKSA | AFHGKDDVLDIKAQ-AHVWK-IIYGFADSLVRCA | CTSLSELKCPVPSTKRKLLLLBFALRTSVDMAAQEGVNYLSGLGLSRLICLPMA-RRAN |
| VSI GIPDIHHNGPVT-SQLVTHLELKSTSIDRFHHF-MRYLVHMQLFTISTDQITK | VQILDLANIHNSGTSMTL-SBLSSRLPSQPVNEDALYRV-MRYLVHMKLPTKASIDGBL | VELGIVDIIDNNQPMAL-ADLASKLEVS DVNC D NLYRI-LRYLVKMEILRVEBKSDDGQ | IELNVFEISQAGRDAQLSPSDIVAKIETKNPSAAISLDRILRMIGASSILSVSTTKSGR |
| EDKYELTPASKLIVGHGQKSLA-PYVMLQTHPREEFS-VWSHVINVLDGKKPY-WB-SNDT | -R-YQAPPAKYVKGWDKCMV-GSILAITDKD-FMAPWHYLKDGLSGBSGTAFPKALGT | KK-YALEPIATLLSRNAKRSMV-PMILGMTQKD-FMTPWHSMKDGGLSDNGT-AFEKAMGM | V--WGLNEESRCIVASEDDKVSVVPMLLLTDKAVVESFY-NIKDVVLEEGVIPPDRTHG- |
| SM-YEKTEGDPE-IN EILNDAMTSHSTFMLPALVSGLMKENVL DGVA SIVDGGNSGVVA | NI-WGYMAHBPE-KNQLFNEAMANDSRLIMSALVKE-CGNIFNGITTLVDGGGTATV | TI-WEYLEGHPD-QSQLFNEGAGETRLLTSSLISG-SRDMFQGIDSIVDGGGNETTV | MDPPFQYAGKE-ERVNKSFNQAMGAGSTIAFDBVPK--VYKGFDNLKELVDUGGIGTSL |
| KGJVDAFPHVKCSVMDLNHVVERVIKNPKLDYVAGDMETSIPNADAILLKSTLHNYEODD | RNIANAFPNIKCTVYDLPHVIA DSPG YSEVHCVA GDMEBKFIPKADAIMMKCILHDWDCKE | KAISDAFPHIKCTLFDLPHVIANSYDLPNIERIGGDMEBKSVDSAQAIIKLILHDWNED | SNIVAKHPHIRGINFELPHVIDGADAPDYRGVEHVPGDMEEGVMNAONILLKWVLHDWDCR |
| CIKINIAKERLBSTGKVILVEIVVDTENLPLFTSARLSMGMDMM-L-MSGKER TKKEW | CIELLKRCKEAVVKGGKVIIVDIVLNVQSEHPYTKMRITLDLDMML-LNTGKERTEE BW | SIKIKQCRNAVHKDG GKVIIVDVALDEESDHELSTRILLDIDML-VNTGKER TKE VW | SIKIKNCWKALHENETVIVIEFVLPVLGNNAESFNAITPDLLMMALNDPGKER TTI EF |
| BDLLRKANPTSHQVIPIMAI BSI-IVAYS- | KKL IHDAGYKGHKITQI-TAVQSV-IEAYPY | EKIVKSAFPSGCKIRHI-AAIQSV-IEVFP- | DGLAKAAGFAETKFFPISQGLHVMEFKINC |

Motif A  
Motif B  
Motif C

5662 T. Morishige et al. (Eur. J. Biochem. 269) © FEBS 2002


CoOMT (AB073908)
(this work)

flavonoid 7OMT
(barley) (X77467)

0.5789
0.5789
0.3395
0.3395
0.3336
0.3336
0.5000
0.5000
0.1296
0.1296
0.1403
0.1665
0.2042
0.2133
0.2825
0.3741
0.3925
0.4849
0.6291
1.2799

6a-hydroxymaackia 3'OMT
(pea) (U69554)

isoflavone 7OMT
(alfalfa) (AF000976)

4OMT
(Coptis) (D29812)

6OMT
(Coptis) (D29811)

SMT
(Coptis) (D29809)

AEOMT
(loblolly pine) (U39301)

caffeic acid OMT
(alfalfa) (M63853)

O-diphenol OMT
(hot pepper) (U83789)

caffeic acid OMT
(Arabidopsis) (U70424)

catechol OMT (Thatu; 2)
(Thalictrum) (AF064694)

3'flavonoid OMT
(Chrysosplenium) (U16794)

(iso)eugenol OMT
(clarkia) (U86760)

caffeic acid OMT
(sugarcane) (AJ231133)

O-diphenol OMT
(tobacco) (X71430)

myo-inositol OMT
(ice plant) (M87340)

isoliquiritigenin 2'OMT
(alfalfa) (L10211)

caffeic acid OMT
(barley) (U54767)

caffeoyl CoA 3OMT
(tobacco) (AF022775)


Fig. 3. Phylogenetic tree of plant S-adenosyl-L-methionine-dependent O-methyltransferase sequences. OMT protein sequences obtained from GenBank™ were used to build the tree. Accession numbers are indicated in parenthesis. Overall, 20 sequences were aligned by the multiple sequence alignment program in GENETYX-MAC ver.1.1 (Software Development, Inc., Tokyo, Japan) using the UPGMA (unweighted pair group maximum average) method. AEOMT, hydroxycinnamic acids/hydroxycinnamoyl CoA esters O-methyltransferase.

lysate was prepared. The SMT reaction mixture consisted of 100 mM Tris/HCl (pH 8.0), 25 mM sodium ascorbate, 1 mM (S)-scoulerine, 5 mM AdoMet and the enzyme preparation. The reaction mixture was incubated at 30°C for 3 h. In this reaction, scoulerine was converted into tetrahydrocolumbamine. After the enzyme reaction was terminated by the addition of 0.5 M sodium carbonate, the reaction product was extracted twice with ethyl acetate. The extracted product was dried and used for further characterization. HPLC analysis showed that tetrahydrocolumbamine prepared was 80% pure and contained unconverted scoulerine. Because scoulerine was a poor substrate for CoOMT, we analyzed substrate affinity of CoOMT for tetrahydrocolumbamine using this 80% pure tetrahydrocolumbamine.

To quantify the enzymatic activity of CoOMT, the transfer of the ³H-labeled methyl group of S-adenosyl-L-[methyl-³H]methionine (0.5 MBq/µmol⁻¹) (PerkinElmer Life Sciences) to the product was measured as described elsewhere [7]. The kinetic constants of the crude enzyme were determined by varying the concentration of alkaloid substrates, but keeping the concentration of ³H-labeled AdoMet fixed at 1 mM then apparent \( K_m \) and \( V_{\text{max}} \) values were calculated.

Other methods

The subunit molecular mass of the enzyme was determined by SDS/PAGE (12.5% polyacrylamide). Protein

© FEBS 2002 Coptis columbamine O-methyltransferase (Eur. J. Biochem. 269) 5663

concentration was determined according to Bradford [23] with bovine serum albumin as a standard.

# RESULTS

## Sequence analysis of the OMT-like clone

A cDNA library was prepared from *C. japonica* cells that produce large amounts of alkaloids, and its 1014 ESTs were obtained (E. Dubouzet *et al.* in preparation). A BLAST search (http://www.ncbi.nlm.nih.gov/blast/) showed that these sequenced clones included 4 clones of 4′OMT, 2 clones of 6OMT and 4 clones of SMT. This result indicated that this cDNA library was highly enriched with biosynthetic genes involved in isoquinoline alkaloid biosynthesis. Thus, we speculated that this library also included the gene for the additional unelucidated OMT in isoquinoline alkaloid biosynthesis in *C. japonica*, i.e. CoOMT. In fact, a BLAST search identified a clone (CJEST64) which was quite similar to isoflavone 7-O-methyltransferase (IOMT) and 6OMT.

Whereas CoOMT is found in the last step of palmatine biosynthesis and 6OMT is found at an early step, we examined the possibility that this CJEST64 was CoOMT. As only a partial sequence was determined for the EST project, full-length of CJEST64 was determined. The sequence obtained further confirmed that CJEST64 was similar to 6OMT, 4′OMT and IOMT (data not shown), although the sequence was not full-length based on sequence alignment. Thus, the 5′-fragment of CJEST64 was obtained by 5′RACE and sequenced. Finally, full-length cDNA was re-cloned from the cDNA library and the expression vector was constructed as described below.

This full-length cDNA carried 1378 nucleotides, with an open reading frame that encoded 351 amino acids (Fig. 2). The deduced amino acid sequences had conserved putative AdoMet binding domains at the C-terminal end (Fig. 2, motifs A–C), like other OMTs [24]. This polypeptide had a high degree of similarity (approximately 40% identity) to *Coptis* 6OMT, *Coptis* 4′OMT, IOMT of alfalfa [25], and 6a-hydroxymaackiain 3-O-methyltransferase of pea [26]. The deduced polypeptide showed a rather low identity (21%) to SMT. Phylogenetic analysis clearly indicated that this polypeptide belongs to the same branch as 6OMT and 4′OMT (Fig. 3).

## Expression of the full-length cDNA in *E. coli*

We constructed the expression vector to produce recombinant proteins in *E. coli* to examine the activity of CoOMT. We introduced an *NcoI* site into the cDNA to fit the initiation codon in the *E. coli* expression vector pET-21d to produce nontagged polypeptide. This construct was then introduced into *E. coli* cells, and production of the recombinant protein was induced. The crude *E. coli* lysate was used to detect CoOMT activity. SDS/PAGE analysis clearly showed that transgene expression was successfully induced and the subunit molecular mass was estimated to be approximately 40 kDa (Fig. 4).

HPLC analysis of the reaction mixture clearly showed that recombinant *E. coli* lysate had CoOMT activity (data not shown). The control *E. coli* lysate with the pET-21d vector showed no enzymatic activity. LC-MS analysis

**Fig. 4.** SDS/PAGE of *E. coli* crude extract expressing CJEST64. Crude extracts of *E. coli* transformed with expression vector for CJEST64 or control vector were separated by SDS/PAGE and stained with Coomassie Brilliant Blue G-250. Lane 1, molecular mass markers; lane 2, pET-CJEST64 (10 μg); lane 3, control vector (pET-21d) (10 μg). Arrow indicates production of the expected recombinant CoOMT.

**Fig. 5.** LC-MS analysis of the reaction product (A), authentic columbamine (B), authentic palmatine (C) and the control reaction (pET-21d). TIC, total ion chromatogram.

5664 T. Morishige *et al.* (Eur. J. Biochem. 269) © FEBS 2002

confirmed that palmatine was produced from columbamine by recombinant *E. coli* lysate (Fig. 5), and that the CJEST64 product was CoOMT.

### Characterization of recombinant CoOMT

As CoOMT activity was identified, we further characterized the enzyme properties of CoOMT using this crude recombinant enzyme. First, we optimized the pH conditions for the CoOMT reaction using columbamine as a substrate. Enzyme assays at various pH values indicated that the optimum pH for the methylation of columbamine was approximately 8.4. Whereas product inhibition of 6OMT and SMT has been reported [8,17], berberine, one of the end-products of isoquinoline alkaloid biosynthesis in *Coptis* cells, did not inhibit CoOMT activity in the assay mixture at 2.5 mM.

Next, we determined the substrate specificity of CoOMT using the incorporation of radioactivity from S-adenosyl-L-[methyl-³H]methionine into the products as an index (Table 1). When columbamine was used as the control substrate (i.e., relative incorporation 100%), the respective relative activities with (S)- and (R,S)-scoulerine and 2,3,9,10-tetrahydroxyprotoberberine were 31, 22 and 14%, whereas no significant methylation was found for other substrates. The methylation of scoulerine and 2,3,9,10-tetrahydroxyprotoberberine by CoOMT was further confirmed by LC-MS (Fig. 6). Interestingly, mono- and di-methylated products were formed from 2,3,9,10-tetrahydroxyprotoberberine by reacting with CoOMT. This result indicated that the regio-specificity of CoOMT was rather low.

Above result suggested that tetrahydroprotoberberine should also be a good substrate for CoOMT. To confirm this idea, tetrahydrocolumbamine was produced from scoulerine by an SMT reaction, and the reaction product, tetrahydrocolumbamine, was used in the successive CoOMT reaction. Figure 6E,F show that a new reaction product with a 14-*m/z* greater mass was detected.

To compare the substrate specificity, we determined the kinetic constants of CoOMT for columbamine, tetrahydrocolumbamine and (S)-scoulerine. Substrate-saturation kinetics of the recombinant CoOMT prepared from *E. coli* for these alkaloids were the typical Michaelis-Menten type. Then, we calculated apparent *K*~m~ and *V*~max~ values by varying the concentration of these alkaloids in the presence of 1 mM AdoMet. The respective *K*~m~ values of CoOMT for columbamine, tetrahydrocolumbamine and (S)-scoulerine were 66 ± 18, 35 ± 18 and 173 ± 51 μM, and respective *V*~max~ were 125 ± 12, 21 ± 7, and 6.9 ± 1.3 pkat·mg protein⁻¹.

---

### Table 1. Transfer of [³H]-methyl group of S-adenosyl-L-methionine into different substrates by columbamine OMT. Values indicate the incorporation of radioactivity from S-adenosyl-L-[methyl-³H]methionine into the product relative to columbamine. Other assay conditions are indicated in Materials and methods. ND, not detected.

| Substrate                                                                 | R1    | R2   | R3   | R4   | R5   | Relative activity |
|--------------------------------------------------------------------------|-------|------|------|------|------|-------------------|
| ![Structure 1](structure_1.png)                                          | Columbamine                                                                 | OCH₃ | OH   | OCH₃ | OCH₃ | -     | 100%               |
| 2,3,9,10-Tetrahydroxyprotoberberine                                      | OH    | OH   | OH   | OH   | -     | 14%               |
| ![Structure 2](structure_2.png)                                          | (S)-Scoulerine                                                             | OCH₃ | OH   | OH   | OCH₃ | -     | 31%                |
| (R,S)-Scoulerine                                                         | OCH₃ | OH   | OH   | OCH₃ | -     | 22%               |
| (R,S)-2,3,9,10-Tetrahydroxytetrahydroprotoberberine                      | OH    | OH   | OH   | OH   | -     | ND                |
| ![Structure 3](structure_3.png)                                          | (R,S)-6-O-Methylnorlaudanosoline                                           | OCH₃ | OH   | OH   | OH   | H     | ND                 |
| (R,S)-Laudanosine                                                        | OH    | OH   | OH   | OH   | CH₃  | ND                |
| (R,S)-Norlaudanosoline                                                   | OH    | OH   | OH   | OH   | H     | ND                |
| (R,S)-Reticuline                                                         | OCH₃ | OH   | OH   | OCH₃ | CH₃  | ND                |
| (S)-Reticuline                                                           | OCH₃ | OH   | OH   | OCH₃ | CH₃  | ND                |
| (R,S)-Norreticuline                                                      | OCH₃ | OH   | OH   | OCH₃ | H     | ND                |
| (S)-Coclaurine                                                           | OCH₃ | OH   | H    | OH   | H     | ND                |
| (R)-Coclaurine                                                           | OCH₃ | OH   | H    | OH   | H     | ND                |
| ![Structure 4](structure_4.png)                                          | Caffeic acid                                                              | OH    | -    | -    | -    | -     | ND                 |
| 4-Hydroxy-3-methoxycinnamic acid                                         | OCH₃ | -    | -    | -    | -     | ND                |

---

© FEBS 2002

**Coptis columbamine O-methyltransferase (Eur. J. Biochem. 269) 5665**

Fig. 6. LC-MS analysis of the CoOMT reaction products for (S)-scoulerine (A), 2,3,9,10-tetrahydroxyprotoberberine (C) and tetrahydrocolumbamine (E), respectively. (B) (D) and (F) show the respective control reactions (pET-21d) for (S)-scoulerine, 2,3,9,10-tetrahydroxyprotoberberine and tetrahydrocolumbamine. Products with a m/z increase of 14 mass units were observed only in the CoOMT reaction. Tetrahydrocolumbamine was prepared by the SMT reaction and used in a successive CoOMT reaction without further purification. TIC, total ion chromatogram.

## DISCUSSION

The present results indicate that high-throughput sequencing of a cDNA library of high-metabolite-producing cells and computer-assisted sequence analysis can be useful for isolating a desired but not yet identified gene(s). Using this strategy, we successfully identified cDNA of CoOMT without purification of the enzyme from plant material. Similar approaches have been reported to isolate enzyme in peppermint oil gland cells [27]. Heterologously expressed enzyme clearly showed CoOMT activity and provided a sufficient amount of material for characterization, whereas such characterization was only partial due to substrate limitations. Further computer-assisted analyses for the hydrophathy and localization of CoOMT using the SIGNAL P (http://www.cbs.dtu.dk/services/SignalP/) and PSORT (http://psort.nibb.ac.jp/) programs indicated that CoOMT was a cytosolic enzyme, whereas Berberis CoOMT has been reported to be vesicle-bound [4].

This CoOMT is the fourth OMT isolated from *C. japonica* cells to play a role in isoquinoline alkaloid biosynthesis. As each enzyme shows distinct substrate specificity, we compared the sequences of these OMTs. Surprisingly, CoOMT was much more similar to 6OMT and 4′OMT than SMT, even though SMT catalyzes the O-methylation of a protoberberine alkaloid, like CoOMT. CoOMT showed 40% identity to 4′OMT. However, we did not detect recombinant CoOMT in the crude extract of *E. coli* by an immunoblot analysis with anti4′OMT polyclonal antibodies. Multiple

sequence alignment of CoOMT and the biosynthetic enzymes involved in benzylisoquinoline alkaloid biosynthesis, including berberine bridge enzyme [28–30] and (S)-N-methylcoclaurine 3′-hydroxylase (CYP80B1) [31], also failed to indicate any sequence homology (data not shown). A comparison of CoOMT and CNMT only showed limited sequence similarity in motif A, as seen between CNMT and other OMTs [14]. It is not clear why the sequence of CoOMT is so different from that of SMT.

The three-dimensional structures of IOMT and chalcone OMT in flavonoid biosynthesis have recently been characterized [32]. Multiple sequence alignment of *Coptis* OMTs, including CoOMT, with OMTs in flavonoid biosynthesis showed that the residues involved in AdoMet binding and the catalytic residues were highly conserved, and the methionine residues that interact with methyl group acceptor were also conserved. However, other substrate-binding residues were not conserved (data not shown). The sequence diversity in the substrate-binding site, which is expected to be located in the N-terminal end of *Coptis* OMTs (T. Morishige & F. Sato, unpublished data), is considerably high. The sequence of CoOMT may be useful for understanding substrate binding and the enzyme structures in isoquinoline alkaloid biosynthesis.

Our recombinant CoOMT also provides additional information about the branching point in the late steps in palmatine biosynthesis. Columbamine was once thought to be the main substrate for palmatine biosynthesis in *Berberis wilsoniae* and *Berberis aggregata* using partially purified

enzyme, as *Berberis* CoOMT could not methylate tetrahydrocolumbamine [4]. However, our findings clearly indicate that tetrahydrocolumbamine can be a substrate for the formation of tetrahydropalmatine (Fig. 6), which can be easily converted to palmatine by (*S*)-tetrahydroprotoberine oxidase [33]. Our preliminary analyses also showed that CoOMT had rather smaller *K*<sub>m</sub> value for tetrahydrocolumbamine than columbamine, while the *V*<sub>max</sub> value for tetrahydrocolumbamine was also smaller than that for columbamine. These data suggested that both pathways might operate in *Coptis* cells. Differences in substrate specificity have also been reported for the CNMT of *Coptis* and *Berberis*, i.e. *Coptis* enzyme could *N*-methylate norlaudanosoline, whereas *Berberis* enzyme could not [11,13]. These results suggest that the late biosynthetic pathway should be re-examined in various plant species, and that the pathway in secondary metabolism may vary depending on the enzyme(s) that each plant has acquired during its evolution.

### ACKNOWLEDGEMENTS

We thank Dr N. Nagakura, Dr P. Facchini and Mitsui Petrochemical Industries Ltd. for their generous gifts of the alkaloids. We also thank Ms. L. Huang for her technical assistance with preparing columbamine. We thank Dr W. Frommer of the University of Tuebingen for the gift of pDR196 vector.

This research was supported in part by a Research for the Future Program Grant (JSPS-RFTF00L01607) from the Japan Society for the Promotion of Science (to F. S.), and a fellowship from the Japan Society for the Promotion of Science (to T. M).

### REFERENCES

1. Croteau, R., Kutchan, T.M. & Lewis, N. (2000) *Biochemistry and Molecular Biology of Plants*. (Buchanan, B., Gruissem, W. & Jones, R., eds), pp. 1250–1318. American Society of Plant Physiologists, Rockville, Maryland.
2. Facchini, P.J. (2001) Alkaloid biosynthesis in plants: biochemistry, cell biology, molecular regulation, and metabolic engineering application. *Annu. Rev. Plant Physiol. Plant Mol. Biol.* **52**, 29–66.
3. DeLuca, V. & Laflamme, P. (2001) The expanding universe of alkaloid biosynthesis. *Curr. Opin. Plant Biol.* **4**, 225–233.
4. Rueffer, M., Amann, M. & Zenk, M.H. (1986) *S*-adenosyl-L-methionine: columbamine-*O*-methyltransferase, a compartmentalized enzyme in protoberberine biosynthesis. *Plant Cell Reports* **3**, 182–185.
5. Poulton, J.E. (1981) The Biochemistry of Plants. (Conn, E.E., ed.), Vol. 7, pp. 667–723. Academic Press, New York.
6. Rueffer, M., Nagakura, N. & Zenk, M.H. (1983) Partial purification and properties of *S*-adenosylmethionine:(R), (*S*)-norlau-danosolin-6-*O*-methyltransferase from *Argemone platyceras* cell cultures. *Planta Med.* **49**, 131–137.
7. Müller, M.J. & Zenk, M.H. (1992) The norcoclaurine pathway is operative in berberine biosynthesis in *Coptis japonica*. *Planta Med.* **58**, 524–527.
8. Sato, F., Tsujita, T., Katagiri, Y., Yoshida, S. & Yamada, Y. (1994) Purification and characterization of *S*-adenosyl-L-methio-nine: norcoclaurine 6-*O*-methyltransferase from cultured *Coptis japonica* cells. *Eur. J. Biochem.* **225**, 125–131.
9. Hara, M., Yazaki, K., Tanaka, S. & Tabata, M. (1995) *S*-adenosyl-L-methionine: norcoclaurine 6-*O*-methyltransferase from *Thalictrum minus* cell cultures. *Phytochemistry* **38**, 1131–1135.
10. Morishige, T., Tsujita, T., Yamada, Y. & Sato, F. (2000) Molecular characterization of the *S*-adenosyl-L-methionine: 3′-hydroxy-N-methylclaurine 4′-*O*-methyltransferase involved in isoquinoline alkaloid biosynthesis in *Coptis japonica*. *J. Biol. Chem.* **275**, 23398–23405.
11. Frenzel, T. & Zenk, M.H. (1990) *S*-adenosyl-L-methionine: 3′-hydroxy-N-methyl-(S)-coclaurine-4′-*O*-methyltransferase, a regio- and stereoselective enzyme of the (*S*)-reticuline pathway. *Phytochemistry* **29**, 3505–3511.
12. Wat, C.K., Steffens, P. & Zenk, M.H. (1986) Partial purification and characterization of *S*-adenosyl-L-methionine: norreticuline *N*-methyltransferases from *Berberis* cell suspension cultures. *Z. Naturforsch.* **41c**, 126–134.
13. Choi, K.B., Morishige, T. & Sato, F. (2001) Purification and characterization of coclaurine *N*-methyltransferase from cultured *Coptis japonica* cells. *Phytochemistry* **56**, 649–655.
14. Choi, K.B., Morishige, T., Shitan, N., Yazaki, K. & Sato, F. (2002) Molecular cloning and characterization of coclaurine *N*-methyltransferase from cultured cells of *Coptis japonica*. *J. Biol. Chem.* **277**, 830–835.
15. Frenzel, T. & Zenk, M.H. (1990) Purification and characterization of three isoforms of *S*-adenosyl-L-methionine: (*R*,*S*)-tetra-hydrobenzylisoquinoline-*N*-methyltransferase from *Berberis koetineana* cell cultures. *Phytochemistry* **29**, 3491–3497.
16. Muemmler, S., Rueffer, M. & Zenk, M.H. (1985) *S*-adenosyl-L-methionine: (*S*)-scoulerine 9-*O*-methyltransferase, a highly stereo- and regio-specific enzyme in tetrahydroprotoberine biosynthesis. *Plant Cell Reports* **4**, 36–39.
17. Sato, F., Takeshita, N., Fitchen, J.H., Fujiwara, H. & Yamada, Y. (1993) *S*-adenosyl-L-methionine: scoulerine-9-*O*-methyltransferase from cultured *Coptis japonica* cells. *Phytochemistry* **32**, 659–664.
18. Takeshita, N., Fujiwara, H., Mimura, H., Fitchen, J.H., Yamada, Y. & Sato, F. (1995) Molecular cloning and characterization of *S*-adenosyl-L-methionine: scoulerine-9-*O*-methyltransferase from cultured cells of *Coptis japonica*. *Plant Cell Physiol.* **36**, 29–36.
19. Sato, F. & Yamada, Y. (1984) High berberine-producing cultures of *Coptis japonica* cells. *Phytochemistry* **23**, 281–285.
20. Cava, M.P. & Reed, T.A. (1967) A synthesis of columbamine from berberine. *J. Org. Chem.* **32**, 1640–1641.
21. Cava, M.P., Reed, T.A. & Beal, J.L. (1965) An efficient separation of the common alkaloids of the berberine group; the isolation and characterization of columbamine. *Lloydia* **28**, 73–83.
22. Rentsch, D., Laloi, M., Rouhara, I., Schmelzer, E., Delrot, S. & Frommer, W.B. (1995) *NTRI* encodes a high affinity oligopeptide transporter in *Arabidopsis*. *FEBS Lett.* **370**, 264–268.
23. Bradford, M.M. (1976) A rapid and sensitive method for the quantitation of protein utilizing the protein-dye binding. *Anal. Biochem.* **72**, 248–254.
24. Joshi, C.P. & Chiang, V.L. (1998) Conserved sequence motifs in plant *S*-adenosyl-L-methionine-dependent methyltransferases. *Plant Mol. Biol.* **37**, 663–674.
25. He, X.Z., Reddy, J.T. & Dixon, R.A. (1998) Stress responses in alfalfa (*Medicago sativa* L.). XXII. cDNA cloning and characterization of an elicitor-inducible isoflavone 7-*O*-methyltransferase. *Plant Mol. Biol.* **36**, 43–54.
26. Wu, Q., Perisig, C.L. & VanEtten, H.D. (1997) Isolation of the cDNAs encoding (+) 6a-hydroxymaackiain 3-*O*-methyltransferase, the terminal step for the synthesis of the phytoalexin pisatin in *Pisum sativum*. *Plant Mol. Biol.* **35**, 551–560.
27. Crock, J., Wildung, M. & Croteau, R. (1997) Isolation and bacterial expression of a sesquiterpene synthase cDNA clone from peppermint (*Mentha x piperita*, L.) that produces the aphid alarm pheromone (*E*)-β-farnesene. *Proc. Natl Acad. Sci. USA* **94**, 12833–12838.
28. Dittrich, H. & Kutchan, T.M. (1991) Molecular cloning, expression, and induction of berberine bridge enzyme, an enzyme essential to the formation of benzophenanthridine alkaloids in the response of plants to pathogenic attack. *Proc. Natl Acad. Sci. USA* **88**, 9969–9973.

© FEBS 2002

Coptis columbamine O-methyltransferase (Eur. J. Biochem. 269) 5667

29. Facchini, P.J., Penzes, C., Johnson, A.G. & Bull., D. (1996) Molecular characterization of berberine bridge enzyme genes from opium poppy. *Plant Physiol.* **112**, 1669–1677.

30. Hauschild, K., Pauli, H.H. & Kutchan, T.M. (1998) Isolation and analysis of a gene *bbe1* encoding the berberine bridge enzyme from the California poppy *Eschscholzia californica*. *Plant Mol. Biol.* **36**, 473–478.

31. Pauli, H.H. & Kutchan, T.M. (1998) Molecular cloning and functional heterologous expression of two alleles encoding (*S*)-*N*-methylcoclaurine 3′-hydroxylase (CYP80B1), a new methyl jasmonate-inducible cytochrome P-450-dependent mono-oxgenase of benzylisoquinoline alkaloid biosynthesis. *Plant J.* **13**, 793–801.

32. Zubieta, C., He, X.Z., Dixon, R.A. & Noel, J.P. (2001) Structures of two natural product methyltransferases reveal the basis for substrate specificity in plant *O*-methyltransferases. *Nat. Struct. Biol.* **8**, 271–279.

33. Amman, M., Nagakura, N. & Zenk, M.H. (1988) Purification and properties of (*S*)-tetrahydroprotoberine oxidase from suspension-cultured cells of *Berberis wilsoniae*. *Eur. J. Biochem.* **175**, 17–25.
