
LIPIDS AND LIPOPROTEINS:
Membrane Topography of Human
Phosphatidylethanolamine $N$
-Methyltransferase

David J. Shields, Richard Lehner, Luis B.
Agellon and Dennis E. Vance
J. Biol. Chem. 2003, 278:2956-2962.
doi: 10.1074/jbc.M210904200 originally published online November 12, 2002

Access the most updated version of this article at doi: 10.1074/jbc.M210904200

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/affinity-sites).

Alerts:
- [When this article is cited](https://www.jbc.org/alerts)
- [When a correction for this article is posted](https://www.jbc.org/alerts)

[Click here](https://www.jbc.org/alerts) to choose from all of JBC's e-mail alerts

This article cites 40 references, 11 of which can be accessed free at [http://www.jbc.org/content/278/5/2956.full.html#ref-list-1](http://www.jbc.org/content/278/5/2956.full.html#ref-list-1)

THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2003 by The American Society for Biochemistry and Molecular Biology, Inc.

Membrane Topography of Human Phosphatidylethanolamine N-Methyltransferase*

Received for publication, October 24, 2002, and in revised form, November 10, 2002
Published, JBC Papers in Press, November 12, 2002, DOI 10.1074/jbc.M210904200

David J. Shields‡, Richard Lehner§, Luis B. Agellon¶, and Dennis E. Vance∥

From the Departments of Biochemistry, Pediatrics and Cell Biology and Canadian Institutes of Health Research Group on Molecular and Cell Biology of Lipids, University of Alberta, Edmonton, Alberta T6G 2S2, Canada

In liver, phosphatidylethanolamine is converted to phosphatidylcholine through a series of three sequential methylation reactions. Phosphatidylethanolamine N-methyltransferase (PEMT) catalyzes each transmethylation reaction, and S-adenosylmethionine is the methyl group donor. Biochemical analysis of human liver revealed that the methyltransferase activity is primarily localized to the endoplasmic reticulum and mitochondria-associated membranes. Bioinformatic analysis of the predicted amino acid sequence suggested that the enzyme adopts a polytopic conformation in those membranes. To elucidate the precise membrane topography of PEMT and thereby provide the basis for in-depth functional characterization of the enzyme, we performed endoproteinase-protection analysis of epitope-tagged, recombinant protein. Our data suggest a topographical model of PEMT in which four transmembrane regions span the membrane such that both the N and C termini of the enzyme are localized external to the ER. Two hydrophilic connecting loops protrude into the lumenal space of the microsomes whereas a corresponding loop on the cytosolic side remains proximate to the membrane. Further support for this model was obtained following endoproteinase-protection analysis of mutant recombinant PEMT derivatives in which specific protease cleavage sites had been genetically engineered or ablated.

substantial demand is imposed on the PC pool by the liver-specific functions of bile and very low density lipoprotein (VLDL) particle production and secretion (3, 4). The phosphatidylethanolamine N-methyltransferase (PEMT) and CDP-choline biosynthetic pathways mediate continual replenishment of the hepatic PC pools (5). The liver is the primary site of PEMT activity whereas the enzymes of the CDP-choline pathway are active in all nucleated cells (2, 6).

PC biosynthesis is clearly essential to liver function, but why that synthesis must be conducted through two distinct pathways is less evident. Recent studies investigating the proportion of hepatic PC that is derived from each pathway defined the PEMT-controlled pathway as the source of 30% of hepatic PC with the CDP-choline pathway accounting for 70% (7–9). Significantly perhaps, data from one group also revealed that the PEMT pathway is a metabolically channeled process suggesting that PEMT-derived PC may be destined for a specific function (9).

Given that PEMT is primarily expressed in liver, PEMT-derived PC might be targeted to a liver-specific fate such as VLDL particles or bile (10–13). In efforts to address these hypotheses, studies were recently conducted using hepatocytes from mice homozygous for a disrupted PEMT allele. The data revealed a defect in the secretion of triacylglycerol and apoB100, key components of VLDL particles, suggesting that PEMT is required for optimal VLDL assembly and/or secretion (14). Additional studies investigating a role for PEMT in bile production or secretion are currently in progress.

Metabolic channeling is central to several metabolic processes including glycolysis and glycogenolysis and involves the retention of metabolites in a specific microenvironment to promote consecutive enzymatic reactions and hence efficient energy utilization (15). For metabolic channeling to be effective, however, spatial organization is requisite. Thus, not only should enzymes and substrates be localized in the same cellular subcompartment, but enzymes must also be topographically organized such that key catalytic residues or motifs are correctly oriented.

Liver is the primary site of human PEMT expression, with extra-hepatic PEMT accounting for a mere fraction of corporeal expression (6, 16–20). Herein, biochemical analysis of human liver reveals that PEMT is primarily localized to the endoplasmic reticulum (ER) and a subfraction of ER membranes that co-fractionate with mitochondria, mitochondria-associated membranes (MAM). However, the exact topography of the enzyme within those membranes has not been determined. Resolution of the topographical orientation of PEMT will permit further analysis of the role of metabolic partitioning in PC biosynthesis. Moreover, it will provide the basis for in depth exploration of the mechanism by which PEMT becomes rate-limiting in the secretion of VLDL particles.

Bioinformatic analysis predicts that PEMT is a polytopic

* This research was supported by a grant from the Canadian Institutes for Health Research. Ethics approval for work on human tissues was obtained from the Health Research Ethics board at the University of Alberta. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
‡ Supported by a Studentship from the Alberta Heritage Foundation for Medical Research.
§ Scholar of the Alberta Heritage Foundation for Medical Research.
¶ Senior Scholar of the Alberta Heritage Foundation for Medical Research.
∥ Canada Research Chair in Molecular and Cell Biology of Lipids and Heritage Medical Scientist of the Alberta Heritage Foundation for Medical Research. To whom correspondence should be addressed: 328 HMRC, Dept. of Biochemistry, University of Alberta, Edmonton, Alberta T6G 2S2, Canada. Tel.: 780-492-8286; Fax: 780-492-3383; Email: Dennis.Vance@ualberta.ca.
¹ The abbreviations used are: PC, phosphatidylcholine; AdoMet, S-adenosylmethionine; ER, endoplasmic reticulum; MAM, mitochondria-associated membranes; PDI, protein disulfide isomerase; PEMT, phosphatidylethanolamine N-methyltransferase; h, human; VLDL, very low density lipoprotein; HA, hemagglutinin; PVDF, polyvinylidene difluoride.

membrane protein with four transmembrane domains and thus yields a model that positions the N and C termini of the enzyme in the same intracellular compartment (21). However, *in silico* analysis cannot resolve whether the end termini of PEMT reside in the cytosol or in the microsomal lumen.

Here, we detail the biochemical validation of a topographical model of PEMT, using endoprotease protection analysis of functional epitope-tagged derivatives of the enzyme. These studies will provide the basis for detailed structural and functional characterization of the human enzyme.

### EXPERIMENTAL PROCEDURES

#### Materials—Dulbecco’s modified Eagle’s medium, fetal bovine serum, restriction endonucleases, and Platinum Pfx DNA polymerase were from Invitrogen. Oligonucleotides for mutagenesis and epitope tagging were synthesized at the DNA core facility in the Department of Biochemistry, University of Alberta. FuGENE transfection reagent was from Roche Molecular Biochemicals. S-Adenosyl-L-[methyl-³H]methionine (15 Ci/mmol) was obtained from Amersham Biosciences. Nonradiolabeled S-adenosyl-L-methionine, anti-HA monoclonal antibody (clone HA-7), and endoprotease Lys-C were from Sigma. Rabbit polyclonal anti-protein disulfide isomerase (PDI) antibody was from Stressgen Biotech. Goat anti-rabbit and goat anti-mouse secondary antibodies were purchased from Pierce. All other reagents were of the highest standard commercially available.

#### Subcellular Fractionation—Adult human liver samples were obtained from the Department of Surgery at the University Of Alberta Hospital and were snap-frozen in liquid nitrogen at resection. Differential subcellular fractionation was performed according to the procedure of Croze and Morre (22) as modified by Vance (23), yielding fractions corresponding to ER, nucleus, plasma membrane, mitochondria, MAM, and Golgi apparatus. Protein concentrations of individual fractions were measured by the Bradford method, using albumin as standard.

Microsomes for protease protection analysis were prepared by a modified version of the method of Graham (24). Briefly, 24 h post-transfection with the various human PEMT (hPEMT) recombinant plasmids, COS-7 cells were washed, harvested into phosphate-buffered saline, and pelleted at 1000 × g. The pellet was resuspended in Buffer A (50 mM Tris, pH 7.4, 250 mM sucrose, 1 mM EDTA), sonicated for 5 s, and centrifuged at 6000 × g for 15 min to pellet nuclei, heavy mitochondria, plasma membrane, Golgi, and cell debris. The resulting supernatant was then centrifuged for 45 min, at 99,000 × rpm at 4 °C. The pellet was resuspended in 75 μl of Tris-buffered saline by pipetting gently 20 times. Protein concentrations of the prepared microsomes were determined by the Bradford method. Integrity of the microsomes was verified by immunoblotting with a polyclonal antibody against the ER luminal marker, PDI.

#### Bioinformatic Analysis—Hydropathy analysis, based on the method of Kyte and Doolittle (25) was performed on the predicted human PEMT amino acid sequence (GenBank™ accession number NP_009100), using the Grease program of the San Diego Supercomputer Center Biology Workbench (workbench.sdsc.edu). The TMAP program in the Biology Workbench was employed to predict the position and length of individual transmembrane domains (26).

#### Recombinant Plasmid Construction—All plasmids were constructed using the wild-type hPEMT-pCI plasmid as template (21). This plasmid consists of the human PEMT open reading frame cloned 5′ to 3′ into the XhoI and XbaI sites, respectively, of the pCI mammalian expression vector polylinker (Promega). Transcription is under the control of a cytomegalovirus promoter. Using PCR, an oligonucleotide encoding an HA (YPYDVPDYA)-tagged epitope was appended to the 5′ end of hPEMT to generate the plasmid, HA-hPEMT. PCR products were blunt-end ligated into *Sma*I-cut pBluescript II (KS) (Stratagene) and recloned to pCI using XhoI and *Xba*I restriction sites. Mutant PEMT derivatives for protease protection analysis were generated by the “splice by overlap extension” PCR mutagenesis method, using the HA-hPEMT plasmid as template (27). Full-length mutant products were subcloned into the pCI expression vector as detailed above. All constructs were sequenced to confirm fidelity of PCR and orientation of the insert at the Molecular Biology Services Unit, Department of Biological Sciences, University of Alberta. To mutagenize the two lysines at positions 38 and 41 in loop A to arginine residues, generating the HA-tagged double mutant HA-AK2R2, PCR A was performed with oligonucleotides 1 (5′-CTCGAGATGTATCCATATGATGTTCCAGATTATGCTACCCGGCTGCTGGGCTACGTGGACCCCCCTG-3′) and 2 (5′-TGTTCC-).

CATCGTGCAACCACATTCAG-3′), PCR B was performed with oligonucleotides 3 (5′-GTGGTTGCACGGATGGGAACACAGGACCCGCAG-GCTGAGCAGGGGCCCTCG-3′) and 4 (5′-TCTAGATCAGCTCCTCTTG-TGGGACCCGGAGGCT-3′), and PCR C, to generate the full-length mutant product, was performed with oligonucleotides 1 and 4, using amplicons from PCR A and B as templates. To mutate the two C-terminal lysines at positions 191 and 197 to arginine residues and generate the HA-tagged double mutant HA-CK2R2, PCR D was performed with oligonucleotides 1 and 5 (5′-TCTAGATCAGCTCCTCC-TGTGGGACCCGGAGGCTCTCTGCCG-3′). To insert a novel endoprotease Lys-C cleavage site into loop B, the arginine residue at position 80 was mutated to a lysine, generating the plasmid HA-R80K. This was achieved as follows: PCR E was performed with oligonucleotides 1 and 6 (5′-GGCTGGCTCAGCATGGCCTG-3′), PCR F was performed with oligonucleotides 7 (5′-CAGGCCATGCTGAGCCAGCCCAAAGATGGAG-AGCCTGGAC-3′) and 4, and the full-length product, using PCR products E and F as template, was generated using oligonucleotides 1 and 4. Each full-length mutant amplicon was recloned into pCI as described.

#### Cell Culture and Transfections—COS-7 cells, obtained from the American Type Culture Collection repository, were maintained in Dulbecco’s modified Eagle’s medium, 10% fetal bovine serum, 100 units/ml penicillin, and 100 μg/ml streptomycin sulfate at 37 °C, 5% CO₂. On day 0, cells were plated at a confluency of 1.75 × 10⁶ cells/60-mm dish. After an overnight incubation, cells were transfected with test plasmids or mock transfected with empty pCI expression vector using FuGENE transfection reagent as per the manufacturer’s protocol. Specifically, we used 5 μl FuGENE/3 μg DNA plasmid per 60-mm dish. Cells were harvested 24 h later and treated as described in the figure legends.

#### Phosphatidylethanolamine N-Methyltransferase Activity Assays—PEMT activity assays were performed as described previously (28). Briefly, 24 h after transfection, COS-7 cells were washed with and harvested into phosphate-buffered saline, pelleted at 1000 × g, and resuspended in Buffer B (10 mM Tris, pH 7.4, 150 mM NaCl, 1 mM EDTA). Following homogenization by sonication, protein homogenates (50 μg) were assayed for PEMT activity using phosphatidylmonomethylethanolamine (Avanti Polar Lipids, Alabaster, AL) as a methyl acceptor and S-adenosyl-L-[methyl-³H]methionine as the methyl group donor.

#### Immunoblot Analysis—Cell homogenate proteins (25 μg) were separated by Tris/glycine SDS-polyacrylamide electrophoresis, on 12.5% polyacrylamide gels calibrated with prestained molecular weight standards (Bio-Rad). Following electrophoresis, proteins were transferred to PVDF membranes and immunoblotted with primary antibodies at the indicated concentration. Protein-antibody complexes were detected by enhanced chemiluminescence with horseradish peroxidase-conjugated secondary antibody using the ECL reagent (Amersham Biosciences) as directed. Membranes were exposed to Biomax MR film (Eastman Kodak Co.) for the indicated time at room temperature.

#### Endoproteinase Lys-C Protection Assays—Endoprotease protection analyses were performed on microsomes prepared from transfected COS-7 cells as described above. Briefly, microsomal proteins (50 μg) were incubated with 1% Triton X-100 or 100 mM Tris-HCl, pH 8.5, on ice for 30 min. Endoprotease Lys-C (0, 0.1, or 1 μg) was added to the mixture (final reaction volume, 20 μl) and incubated at 37 °C for 3 h. Each reaction was stopped by the addition of Buffer C (5× 60 mM Tris-HCl, pH 6.8, 25% glycerol, 2% SDS, 715 mM β-mercaptoethanol, 0.1% bromphenol blue) and boiled for 10 min. Endoprotease cleavage products were separated by Tris/glycine SDS-polyacrylamide gel electrophoresis on 15% polyacrylamide gels and immunoblotted as described above. Integrity of the microsomes was validated by immunoblotting with a monoclonal anti-PDI antibody.

### RESULTS

#### Subcellular Localization of PEMT—Early studies on human PEMT identified the liver as the primary site of expression, which parallels the expression pattern of PEMT in rodents (6, 29). Although the rat PEMT activity is distributed between ER and MAM, only the isoform designated PEMT2, in the MAM fraction, is immunoreactive with an antibody raised against a C-terminal rat PEMT peptide (16). To determine whether the human enzyme displays similar disparity in the localization of enzymatic activity and immunoreactivity, subcellular fractionation of human liver was performed. Similar to rodents, the human PEMT activity is primarily localized to ER and MAM (Fig. 1A), but unlike rodents the human PEMT enzyme is also immunoreactive to the anti-PEMT peptide antibody in both ER

Membrane Topography of PEMT

A.
![Graph](https://i.imgur.com/1234567.png)

B.
![Image](https://i.imgur.com/8901234.png)

C.
![Image](https://i.imgur.com/5678901.png)

FIG. 1. Subcellular localization of PEMT in human liver. Human liver samples, snap-frozen at resection, were subjected to differential subcellular fractionation (22, 23). A, PEMT specific activity in individual fractions. Homogenates of each fraction, 50-μg protein, were assayed for PEMT activity. PM, plasma membrane; Mito, mitochondria. B, immunoblot with anti-PEMT antibody using 25 μg of protein homogenates for each fraction. C, immunoblot with anti-PDI antibody using 25 μg of protein homogenates for each fraction.

and MAM (Fig. 1B). Immunoblotting with anti-PDI confirmed that the fractions representing ER and MAM (which is a subfraction of ER) were of ER origin (Fig. 1C). As the activity and immunoreactivity of human PEMT are superimposable, it appears that the differential subcellular localization of PEMT isoforms, as observed in the rat, has not been conserved in evolution.

**Predicted Membrane Topography of PEMT—**Purification of PEMT revealed the enzyme to be an integral membrane protein (30). To gain insight into the topography of PEMT in the membranes of ER/MAM, the deduced amino acid sequence was examined *in silico* using the method of Kyte and Doolittle (25). The hydropathy profile of PEMT shown in Fig. 2A predicts the presence of four hydrophobic regions. Each hydrophobic region exceeds 20 amino acids in length and registers a value of >2 units on the hydropathy plot, two properties strongly indicative of a transmembrane domain. A polytopic model based on four transmembrane α-helical domains colocalizes the N and C termini on one side of the membrane plane, suggesting that PEMT adopts one of two opposing topographical orientations (both termini in the lumen or both termini in the cytosol).

To investigate which of the two possible topographical models is valid, intact microsomes were prepared from transfected cells and subjected to endoprotease digestion in the absence or presence of detergent. In the absence of detergent, proteolysis is expected to occur only at exposed cleavage sites on the microsomal exterior, and the luminal oriented sites should remain protected. Cleavage of protected luminal oriented sites should occur only in the presence of detergent. The protease utilized in these studies, endoprotease Lys-C, specifically cleaves at the C terminus of lysine residues, and, given the position of the lysine residues within the PEMT amino acid sequence, was expected to yield an informative proteolytic pattern (Fig. 2, B and C).

**Characterization of HA-tagged PEMT Protein—**To perform the topographical analyses, an antibody capable of detecting proteolytic cleavage products was required. An antibody against a peptide corresponding to an epitope at the extreme C terminus of PEMT was raised previously, but this epitope contains two lysine residues (endoprotease Lys-C cleavage sites) with the consequence that proteolysis would result in

C.
![Sequence](https://i.imgur.com/abcdefg.png)

FIG. 2. Hydropathy plot and predicted membrane topography of PEMT. A, hydropathy plot of the human PEMT amino acid sequence, as determined by the Grease program (based on the method of Kyte and Doolittle) (25), at the San Diego Supercomputer Center Biology Workbench. B, working model for PEMT topography. Hydrophilic connecting loops are labeled A, B, and C. Arrows indicate endoproteinase Lys-C cleavage sites. C, shaded and unshaded regions in the PEMT amino acid sequence denote predicted transmembrane α-helices and hydrophilic connecting loops, respectively. Asterisks indicate the position of lysine residues (endoproteinase Lys-C cleavage sites).

cleavage at these residues and destruction of the epitope. Therefore, although this antibody is informative for the localization of the C terminus, it would not be expected to yield interpretable results for the remainder of the protein.

To circumvent this problem a HA tag, which does not contain lysine residues, was appended to the N terminus of PEMT. Hydropathy analysis of the epitope-tagged protein sequence did not predict changes in the number or length of the predicted transmembrane domains. Furthermore, the HA-tagged PEMT expressed in COS-7 cells is enzymatically active (Fig. 3A). Immunoblotting verified the production of recombinant proteins (Fig. 3B) and faithful recognition of the HA antigen tag by the anti-HA antibody (Fig. 3C).

**Protease Protection Analysis of Epitope-tagged PEMT—**To analyze which one of the two possible membrane topographic models of PEMT is valid, a plasmid encoding HA-tagged PEMT was transfected into COS-7 cells. Subsequently, microsomes were prepared and incubated with various concentrations of endoprotease Lys-C in the absence or presence of Triton X-100, and the resultant proteolytic products were separated by SDS-polyacrylamide gel electrophoreses and analyzed by immunoblotting (Fig. 4A). In microsomes incubated without protease, an immunoreactive band corresponding to the epitope-tagged PEMT was detectable at ~22 kDa in the absence or presence of Triton X-100 (Fig. 4A, lanes 1 and 2).

In the presence of endoprotease, but in the absence of Triton X-100, the ~22-kDa band was replaced by one of increased electrophoretic mobility (Fig. 4A, lanes 3 and 5). This is

Membrane Topography of PEMT

A.

| % Specific Activity relative |
|----------------------------|
| to Wild Type PEMT            |

B.
α-PEMT

C.
α-HA

FIG. 3. Epitope-tagged PEMT protein is enzymatically active in transfected COS-7 cells. COS-7 cells were transiently transfected with 3 μg of plasmids containing wild-type PEMT or epitope-tagged PEMT derivatives, or they were mock transfected with empty pCI vector. A, cellular homogenates, 50-μg protein, were assayed for PEMT activity. The results are expressed as the mean of three separate experiments, each performed in duplicate, ± S.E., relative to the values obtained for similar assays on cells transfected with wild-type PEMT. B, immunoblot with anti-PEMT antibody using 25-μg protein of transfected cellular homogenates. C, immunoblot with anti-HA antibody using 25-μg protein of transfected cellular homogenates.

indicative of cleavage at the C-terminal lysine residues, which generates a truncation product lacking the final eight residues of the epitope-tagged protein. As proteolysis occurred in the absence of detergent, this result suggests that the C terminus of PEMT is localized external to the microsomes.

To confirm that the endoprotease was functional in the absence of Triton X-100, and hence that the electrophoretic shift observed in Fig. 4A (lanes 3–6) was because of proteolytic cleavage, duplicate proteolytic products were immunoblotted with a rabbit polyclonal anti-PEMT antibody. Protease treatment, in the absence or presence of Triton X-100, resulted in destruction of the C-terminal PEMT epitope and consequently, a loss of immunoreactivity, confirming that the protease remained active (Fig. 4B, lanes 3–6).

In the presence of Triton X-100, the C-terminal truncation product was again evident, but a reduction in intensity resulted as protease concentrations increased (Fig. 4A, lanes 4 and 6). This was because of proteolysis at the previously inaccessible luminal cleavage sites. Digestion in the presence of Triton X-100 also resulted in the appearance of a fast migrating band of ~5.2 kDa (lanes 4 and 6). This band corresponds to the expected proteolytic product resulting from cleavage at the lysine residues in loop A. The diffuse appearance of the 5.2-kDa immunoreactive band is probably because of the high concentration of proteolytic fragments in this region of the gel. Given the appearance of this immunoreactive band only in the presence of detergent, loop A appears to reside in the ER lumen.

To verify the integrity of the prepared microsomes, duplicate proteolysis products were immunoblotted with an antibody against the ER luminal marker, PDI (Fig. 4C). In the absence of detergent, a 57-kDa immunoreactive band was detectable, indicating protection of the epitope and thus demonstrating the integrity of the microsomes (Fig. 4C, lanes 1, 3, and 5). In the presence of detergent, proteolysis abolished the immunoreac-

Lane | Endoprot (µg) | Tx-100 | α-HA
---- | ------------- | ------ | -----
1    | —             | —      | 
2    | —             | +      | 
3    | +             | —      | 
4    | +             | +      | 
5    | ++            | —      | 
6    | ++            | +      | 

kDa
-22
-5.2

B.
α-PEMT

C.
α-PDI

D.

cytosol
22

α-HA
5.2
15
lumen

FIG. 4. Luminal orientation of loop A and cytosolic orientation of the C terminus as determined by protease protection analysis. A, microsomes were prepared from transfected cells as described under “Experimental Procedures.” Aliquots, 50-µg protein, were incubated with various concentrations of endoproteinase at 37 °C for 3 h, in the absence or presence of 1% Triton X-100. Reactions were stopped by the addition of electrophoretic loading buffer and boiling at 100 °C for 10 min. Samples were separated by SDS-polyacrylamide gel electrophoresis, transferred to PVDF membranes, and immunoblotted with anti-HA antibody. The film was exposed at room temperature for 30 s. B, replicate membranes of protease protection products, generated as described above, were immunoblotted with an anti-PEMT antibody. C, replicate membranes of protease protection products, generated as described above, were immunoblotted with an anti-PDI antibody to confirm the integrity of the microsomes. Representative immunoblots are shown. Each protease protection experiment was repeated at least three times with similar results. D, predicted membrane topography model of PEMT. Endoproteinase Lys-C cleavage sites are denoted by arrows, and the length of cleavage fragments generated from proteolysis at each site, as measured from the N-terminal HA-tagged epitope, are indicated in kDa.

tivity of PDI, demonstrating that the detergent permeabilized the microsomes and that the protease was active (Fig. 4C, lanes 4 and 6). These data support the validity of a topographical model that localizes both termini external to the microsomes (Fig. 4D).

Evaluation of HA-tagged PEMT Mutants—Further analysis of the proposed topography of PEMT required the design of three novel HA-tagged PEMT derivatives. To confirm the specificity of cleavage at the lysine residues in the C terminus (Fig. 4A), both residues were mutated to arginine residues to generate the plasmid HA-CK2R2. To evaluate the proposed cytosolic localization of loop B, a mutant version of PEMT was generated in which a lysine residue and hence endoprotease site was engineered into loop B, resulting in the plasmid HA-R80K. To confirm the specificity of cleavage at the two lysine residues in loop A, and to investigate the orientation of loop C, a third

2960

Membrane Topography of PEMT

A.

![Diagram](https://i.imgur.com/1234567.png)

B.

![Image](https://i.imgur.com/8901234.png)

FIG. 5. Epitope-tagged mutant PEMT derivatives retain enzymatic activity in transfected COS-7 cells. COS-7 cells, set up as described under “Experimental Procedures,” were transiently transfected with 3 μg of plasmids containing epitope-tagged PEMT or epitope-tagged mutant PEMT derivatives, or they were mock transfected with empty pCI vector. A, cellular homogenates, 50-μg protein, were assayed for PEMT activity. The results are expressed as the mean of three separate experiments, each performed in duplicate, ± S.E., relative to the values obtained for similar assays on cells transfected with HA-tagged PEMT. B, cellular homogenates, 25-μg protein, were immunoblotted with anti-HA antibody.

plasmid, HA-AK2R2, was generated in which both lysine residues in loop A were mutated to arginine residues. To ensure that the mutant constructs retained PEMT activity, each construct was transfected into COS-7 cells, and activity assays were performed. Cells transfected with each mutant construct displayed equal (HA-AK2R2) or greater (HA-CK2R2, HA-R80K) PEMT activity compared with cells expressing the unmodified PEMT enzyme (Fig. 5A), signifying that the structure and topology required for enzymatic activity are retained. Immunoblots demonstrated similar levels of expression of the recombinant proteins (Fig. 5B).

Protease Protection Analysis of HA-tagged PEMT Mutants—In the next set of experiments, the plasmid HA-CK2R2, encoding hPEMT, which lacks the C-terminal proteolysis sites (Fig. 4A), was transfected into COS-7 cells to evaluate the specificity of endoprotease cleavage at the C-terminal lysine residues. Microsomes were prepared from the transfected cells, and protease protection experiments were conducted as before. Digestion of the microsomes with various concentrations of the protease, in the absence of Triton X-100, did not change the electrophoretic mobility of the ~22-kDa band that corresponds to the full-length tagged mutant protein (Fig. 6B, lanes 1–6). This contrasts with data obtained from the HA-hPEMT proteolysis experiments (Fig. 4A, lanes 3–6), in which the C-terminal lysine residues are intact, and cleavage results. This result supports the notion that the C terminus of PEMT resides in the cytosol, validating our earlier findings and one portion of our predicted model (Fig. 4D). As anticipated, proteolytic products from microsomes treated with protease in the presence of detergent (lanes 4 and 6) were similar to those generated from protection experiments on HA-hPEMT (Fig. 4A, lanes 4 and 6). Reprobing of the membranes with a polyclonal anti-PDI antibody verified the integrity of the microsomal membranes (Fig. 6C).

The topographical model in Fig. 4D postulated that loop B is exposed to the cytosol. To examine this hypothesis, COS-7 cells were transfected with the plasmid HA-R80K, which contains an engineered endoprotease site in loop B. Proteolysis in the absence of detergent was predicted to yield a novel immunoreactive proteolytic fragment reflecting cleavage at the exposed

FIG. 6. Protease protection analysis of the PEMT mutant CK2R2 confirms the cytosolic orientation of the C terminus. A, predicted membrane topography model of PEMT. Endoproteinase Lys-C cleavage sites are denoted by arrows, and the length of cleavage fragments generated from proteolysis at each site, as measured from the N-terminal HA-tagged epitope, are indicated in kDa. Ablation of the C-terminal endoprotease cleavage sites is indicated. B, microsomes were prepared from transfected cells as described under “Experimental Procedures.” Aliquots, 50-μg protein, were incubated with various concentrations of endoproteinase at 37 °C for 3 h, in the absence or presence of 1% Triton X-100. Reactions were stopped by the addition of electrophoresis loading buffer and boiling at 100 °C for 10 min. Samples were separated by SDS-polyacrylamide gel electrophoresis, transferred to PVDF membranes, and immunoblotted with anti-HA antibody. The film was exposed at room temperature for 30 s. C, duplicate membranes of protease protection products, generated as described above, were immunoblotted with an anti-PDI antibody to confirm the integrity of the microsomes. Representative immunoblots are shown. Each protease protection experiment was repeated at least three times with similar results.

loop B site. However, cleavage did not occur in the absence of detergent (results not shown), suggesting that the engineered cleavage site is protected and that loop B is localized proximate to the membrane. Given the length and hydrophobicity of each predicted transmembrane domain, a bitopic model based on two transmembrane domains that would orient loop B into the ER lumen is unlikely. Thus, although the topography of loop B remains indeterminate, a model positioning the hydrophilic connecting loop contiguous with the external leaflet of the membrane bilayer is favored.

In the final set of experiments, the plasmid HA-AK2R2, in which the cleavage sites in loop A are abolished (Fig. 7A), was transfected into COS-7 cells, and protease protection analysis was performed. As anticipated, results from the protease protection experiments conducted in the absence of Triton X-100 (Fig. 7B, lanes 1, 3, and 5) were similar to those from similar experiments on HA-hPEMT (Fig. 4A, lanes 1, 3, and 5). However, in the presence of detergent, addition of protease failed to yield a proteolytic fragment of ~5.2 kDa (Fig. 7B, lanes 4 and 6), confirming that the 5.2-kDa fragment generated following

Membrane Topography of PEMT

exist that are distinguishable on the basis of immunoreactivity with an antibody raised against a rat PEMT C-terminal peptide; PEMT1 is localized to the ER whereas PEMT2 is confined to the MAM (16). However, in humans, PEMT activity and immunoreactivity are detectable in both the ER and MAM suggesting that the differential subcellular localization of PEMT isoforms may be confined to rodents (Fig. 1).

Although the localization of PEMT within the human hepatic ultrastructure has now been revealed, factors that direct PEMT to the specific subcellular compartment remain to be identified. Targeting of ER membrane proteins is a well defined process that is modulated by specific retention or retrieval signals (31, 32). Whereas the first transmembrane segment of some polytopic proteins serves to “retain” the protein in the ER membrane, a C-terminal dilysine motif (KKXX or KXKXX) can similarly confer ER localization, albeit through retrieval from an intermediate compartment (31, 32). A C-terminal dilysine motif is present in the yeast PEM2 amino acid sequence, but this motif is not conserved in the higher eukaryotes. However, a hybrid XHKRX motif is conserved in the rat, mouse, and human amino acid sequences. Moreover, in certain instances, it has been demonstrated that mutagenesis of one lysine in the dilysine motif to an arginine or a histidine residue can occur without detriment to ER targeting (33). Because histidine and arginine residues flank the C-terminal lysine residue, one of several amino acid combinations could potentially mediate ER targeting. For a dilysine motif to be functional as an ER targeting signal, a cytosolic orientation is requisite. Our proposed topographical model for PEMT in which the C terminus is localized to the cytosol conforms to this requirement. Further analysis would be required to determine the relative contribution of the XHKRX motif to the subcellular distribution of PEMT.

Elucidation of the subcellular distribution of the integral membrane protein, PEMT, prompted an investigation of the topographical orientation of PEMT in the microsomal membranes. Although early trypsin-proteolysis studies suggested that certain domains of PEMT were localized external to the microsomal membranes, the specific membrane topography of PEMT had, until now, remained elusive (34). Here, we present data that are consistent with the tetra-span membrane topology model of PEMT shown in Fig. 2B.

Bioinformatic analysis of the PEMT amino acid sequence revealed the presence of four regions of hydrophobicity that varied in length between 23 and 29 amino acids. Separating the putative transmembrane α-helical regions are short hydrophilic loops (A, B, and C) that range from 8 to 29 residues in length. Although the exact functional significance of the length of the short loops remains undefined, this structural organization may facilitate juxtaposition of distinct functional domains from the adjoining transmembrane α-helices. Such an alignment is not without precedent, as the topography of two enzymes central to cellular cholesterol homeostasis (*i.e.* sterol regulatory element-binding protein cleavage-activating protein and 3-hydroxy-3-methylglutaryl CoA reductase) features a series of five closely aligned transmembrane domains that together constitute a conserved sterol-sensing domain (35, 36).

Because each hydrophobic region of the PEMT protein exceeds the minimum length considered necessary for the formation of a transmembrane segment (20 amino acids), and because of the relative hydrophobicity (>2 units) of each segment, a membrane topography based on four transmembrane domains is proposed (Fig. 2B) (37). In contrast, whereas the yeast ortholog (PEM2) is proposed to be similarly polytopic, one portion of the yeast protein contains a hydrophobic stretch of 31 amino acids, which, intriguingly, is the minimum length re-

FIG. 7. Luminal orientation of loops A and C demonstrated by protease protection analysis of PEMT mutant, AK2R2. A, predicted membrane topography model of PEMT. Endoprotease Lys-C cleavage sites are denoted by arrows, and the length of cleavage fragments generated from proteolysis at each site, as measured from the N-terminal HA-tagged epitope, are indicated in kDa. Ablation of the loop A endoprotease cleavage sites is indicated. B, microsomes were prepared from transfected cells as described under “Experimental Procedures.” Aliquots, 50-μg protein, were incubated with various concentrations of endoprotease at 37 °C for 3 h, in the absence or presence of 1% Triton X-100. Reactions were stopped by the addition of electrophoresis loading buffer and boiling at 100 °C for 10 min. Samples were separated by SDS-polyacrylamide gel electrophoresis, transferred to PVDF membranes, and immunoblotted with anti-HA antibody. The film was exposed at room temperature for 30 s. C, duplicate membranes of protease protection products, generated as described above, were immunoblotted with an anti-PDI antibody to confirm the integrity of the microsomes. Representative immunoblots are shown. Each protease protection experiment was repeated at least three times with similar results.

cleavage of HA-hPEMT was a result of specific proteolysis at the lysine residues in loop A (Fig. 5B, lanes 4 and 6). Hence, the predicted luminal localization of loop A is supported.

Furthermore, proteolysis in the presence of Triton X-100 yielded a fragment of ~15 kDa as postulated. Previously, this product was not generated because of the presence of the loop A cleavage sites within the 15-kDa fragment. However, following ablation of the loop A sites, the 15-kDa proteolytic product was generated following cleavage at the lysine residue in loop C. Given that the appearance of this product occurs only in the presence of detergent, our notion of a luminal orientation for loop C is supported. Immunoblotting with a polyclonal anti-PDI antibody confirmed the integrity of the microsomes and thus the interpretation of our results.

DISCUSSION

Expression of the human *PEMT* gene is greatest in the liver, and here we demonstrate that the encoded PEMT protein is enriched subcellularly in both the ER and MAM (Fig. 1). This contrasts with findings in rats where two isoforms of PEMT

2962

**Membrane Topography of PEMT**

required for the formation of a helical hairpin (helix-turn-helix) in the membrane (38). Moreover, a pair of residues with helix turn-inducing propensity (lysine-proline) is centrally located in the 31-residue hydrophobic segment (39, 40). However, as each putative transmembrane domain of the human protein ranges in length from 23 to 29 amino acids and is thus below the minimum requirement for the formation of a helical hairpin, the four human transmembrane α-helical regions are not predicted to reorient within the membrane plane.

Protease protection analysis of epitope-tagged PEMT in intact microsomes revealed that the C terminus is sensitive to proteolytic digestion and hence is exposed to the cytosol, whereas both hydrophilic loops A and C are protease-resistant and are thus predicted to reside in the lumen (see Fig. 4A, Fig. 6B, and Fig. 7B). Although the orientation of loop B and the N terminus of PEMT were not resolved unequivocally in the present studies, a hydropathy profile that is strongly indicative of a tetra-spanning topography, combined with the orientation of loops A and C and the C terminus, suggests that both loop B and the N-terminal domain are cytosolically oriented. Furthermore, as the hydropathy profile of PEMT is highly conserved in species from *Rattus norvegicus* to *Homo sapiens*, the elucidated membrane topography of the human enzyme should prove representative of the higher eukaryotic PEMT family.

Recent data from experiments utilizing isotopic labeling and NMR spectroscopy suggest that channeling of metabolites occurs in the PEMT pathway (9). Identification of residues essential for binding of the methyl group donor, AdoMet, combined with the current data on the subcellular localization and topographical orientation of PEMT, should provide the clearest insight yet into the specific role of metabolic channeling in this pathway.

Approximately 85% of methylation reactions occur in the liver, and AdoMet is the primary methyl group donor (41, 42). Although several consensus AdoMet binding motifs have been identified that are conserved in the majority of AdoMet-dependent methyltransferases, a small fraction of AdoMet-dependent methyltransferases, including the eukaryotic PEMT family of enzymes, lack these motifs (43). Cellular AdoMet is concentrated predominantly in the cytosol, with a smaller fraction present in mitochondria (44). Thus, we posit that residues essential for binding of the AdoMet moiety are localized in the cytosolically disposed hydrophilic loop (B) or at the cytosolic face of the transmembrane α-helices. Elucidation of the topographical organization of PEMT should therefore accelerate the identification of residues that are important for binding AdoMet.

In summary, we describe the first experimental resolution of the topography of an enzyme that catalyzes the synthesis of PC. Data from the current studies should provide the impetus for detailed structural analysis of PEMT, which, in turn, should yield evidence for a definitive topographical model of this AdoMet-dependent methyltransferase. Elucidation of the topographical organization of PEMT will enable detailed analysis of the spatio-temporal organization of residues essential for the binding of AdoMet and hence promote a mechanistic understanding of the methylation-dependent biosynthesis of PC.

---

**Acknowledgments**—We thank Susanne Lingrell for invaluable technical assistance. We acknowledge Drs. Belinda Hsi and Norm Kneteman for human liver samples and Wen-Hui Gao for microsome samples during initial topography studies. We thank Jenny Altarejos for production of the graphical topography models.

---

**REFERENCES**

1. Exton, J. H. (1994) *Biochim. Biophys. Acta* **1212**, 26–42
2. Kent, C. (1997) *Biochim. Biophys. Acta* **1348**, 79–90
3. Agellon, L. B. (2002) in *Biochemistry of Lipids, Lipoproteins and Membranes* (Vance, D. E. and Vance, J. E., eds) 4th Ed., pp. 433–448, Elsevier Science Publishers B.V., Amsterdam
4. Mathur, S. N., Born, E., Murthy, S., and Field, F. J. (1996) *Biochem. J.* **314**, 569–575
5. Vance, D. E. (2002) in *Biochemistry of Lipids, Lipoproteins and Membranes* (Vance, D. E. and Vance, J. E., eds) 4th Ed., pp. 205–232, Elsevier Science Publishers B.V., Amsterdam
6. Vance, D. E., and Ridgway, N. D. (1988) *Prog. Lipid. Res.* **27**, 61–79
7. DeLong, C. J., Shen, Y. J., Thomas, M. J., and Cui, Z. (1999) *J. Biol. Chem.* **274**, 29683–29688
8. Reo, N. V., and Adinehzadeh, M. (2000) *Toxicol. Appl. Pharmacol.* **164**, 113–126
9. Reo, N. V., Adinehzadeh, M., and Foy, B. D. (2002) *Biochim. Biophys. Acta* **1580**, 171–188
10. Vance, J. E., and Vance, D. E. (1986) *J. Biol. Chem.* **261**, 4486–4491
11. Nishimaki-Mogami, T., Suzuki, K., and Takahashi, A. (1996) *Biochim. Biophys. Acta* **1304**, 21–31
12. Nishimaki-Mogami, T., Suzuki, K., Okochi, E., and Takahashi, A. (1996) *Biochim. Biophys. Acta* **1304**, 11–20
13. Agellon, L. B., Walkey, C. J., Vance, D. E., Kuipers, F., and Verkade, H. J. (1999) *Hepatology* **30**, 725–729
14. Noga, A. A., Zhao, Y., and Vance, D. E. (2002) *J. Biol. Chem.* **277**, 42358–42365
15. Al-Habori, M. (1994) *Int. J. Biochem. Cell Biol.* **27**, 123–132
16. Cui, Z., Vance, J. E., Chen, M. H., Voelker, D. R., and Vance, D. E. (1993) *J. Biol. Chem.* **268**, 16655–16663
17. Panagia, V., Ganguly, P. K., and Dhalla, N. S. (1984) *Biochim. Biophys. Acta* **792**, 245–253
18. Prasad, C., and Edwards, R. M. (1981) *J. Biol. Chem.* **256**, 1300–1303
19. Nieto, A., and Catt, K. J. (1983) *Endocrinology* **113**, 758–762
20. Sarzale, M. G., and Pilarska, M. (1976) *Biochim. Biophys. Acta* **441**, 81–92
21. Walkey, C. J., Shields, D. J., and Vance, D. E. (1999) *Biochim. Biophys. Acta* **1436**, 405–412
22. Croze, E., and Morre, D. J. (1984) *J. Cell. Physiol.* **119**, 46–57
23. Vance, J. E. (1990) *J. Biol. Chem.* **265**, 7248–7256
24. Graham, J. M. (1997) in *Subcellular Fractionation: A Practical Approach* (Graham, J. M. and Rickwood, D., eds) pp. 205–242, Oxford University Press, Oxford
25. Kyte, J., and Doolittle, R. F. (1982) *J. Mol. Biol.* **157**, 105–132
26. Persson, B., and Argos, P. (1994) *J. Mol. Biol.* **237**, 182–192
27. Ho, S. N., Hunt, H. D., Horton, R. M., Pullen, J. K., and Pease, L. R. (1989) *Gene* **77**, 51–59
28. Ridgway, N. D., and Vance, D. E. (1992) *Methods Enzymol.* **209**, 366–374
29. Shields, D. J., Agellon, L. B., and Vance, D. E. (2001) *Biochim. Biophys. Acta* **1532**, 105–114
30. Ridgway, N. D., and Vance, D. E. (1987) *J. Biol. Chem.* **262**, 17231–17239
31. Teasdale, R. D., and Jackson, M. R. (1996) *Annu. Rev. Cell Dev. Biol.* **12**, 27–54
32. Nilsson, T., and Warren, G. (1994) *Curr. Opin. Cell Biol.* **6**, 517–521
33. Hardt, B., and Bause, E. (2002) *Biochem. Biophys. Res. Commun.* **291**, 751–757
34. Audubert, F., and Vance, D. E. (1984) *Biochim. Biophys. Acta* **792**, 359–362
35. Nohturfft, A., Brown, M. S., and Goldstein, J. L. (1998) *J. Biol. Chem.* **273**, 17243–17250
36. Olender, E. H., and Simoni, R. D. (1992) *J. Biol. Chem.* **267**, 4223–4235
37. Van Geest, M., and Lolkema, J. S. (2000) *Microbiol. Mol. Biol. Rev.* **64**, 13–33
38. Monne, M., Nilsson, I., Elofsson, A., and von Heijne, G. (1999) *J. Mol. Biol.* **293**, 807–814
39. Romano, J. D., and Michaelis, S. (2001) *Mol. Biol. Cell* **12**, 1957–1971
40. Monne, M., Hermansson, M., and von Heijne, G. (1999) *J. Mol. Biol.* **288**, 141–145
41. Finkelstein, J. D., and Martin, J. J. (1986) *J. Biol. Chem.* **261**, 1582–1587
42. Fauman, E. B., Blumenthal, R. M., and Cheng, X. (1999) in *S-Adenosylmethionine-Dependent Methyltransferases: Structures and Functions* (Cheng, X. and Blumenthal, R. M., eds) pp. 1–32, World Scientific Publishing, Singapore
43. Kagan, R. M., and Clarke, S. (1994) *Arch. Biochem. Biophys.* **310**, 417–427
44. Farooqui, J. Z., Lee, H. W., Kim, S., and Paik, W. K. (1983) *Biochim. Biophys. Acta* **757**, 342–351
