
This article was downloaded by: [McMaster University]
On: 19 November 2014, At: 09:15
Publisher: Taylor & Francis
Informa Ltd Registered in England and Wales Registered Number: 1072954 Registered office: Mortimer House, 37-41 Mortimer Street, London W1T 3JH, UK

---

**Bioscience, Biotechnology, and Biochemistry**

Publication details, including instructions for authors and subscription information:
[http://www.tandfonline.com/loi/tbbb20](http://www.tandfonline.com/loi/tbbb20)

**Molecular Bases of Aerobic Bacterial Degradation of Dioxi ns: Involvement of Angular Dioxygenation**

Hideaki NOJIRI<sup>a</sup> & Toshio OMORI<sup>a</sup>

<sup>a</sup> Biotechnology Research Center, The University of Tokyo 1-1-1 Yayoi, Bunkyo-ku, Tokyo 113-8657, Japan

Published online: 22 May 2014.

---

To cite this article: Hideaki NOJIRI & Toshio OMORI (2002) Molecular Bases of Aerobic Bacterial Degradation of Dioxins: Involvement of Angular Dioxygenation, Bioscience, Biotechnology, and Biochemistry, 66:10, 2001-2016, DOI: [10.1271/bbb.66.2001](https://doi.org/10.1271/bbb.66.2001)

To link to this article: [http://dx.doi.org/10.1271/bbb.66.2001](http://dx.doi.org/10.1271/bbb.66.2001)

---

PLEASE SCROLL DOWN FOR ARTICLE

Taylor & Francis makes every effort to ensure the accuracy of all the information (the “Content”) contained in the publications on our platform. However, Taylor & Francis, our agents, and our licensors make no representations or warranties whatsoever as to the accuracy, completeness, or suitability for any purpose of the Content. Any opinions and views expressed in this publication are the opinions and views of the authors, and are not the views of or endorsed by Taylor & Francis. The accuracy of the Content should not be relied upon and should be independently verified with primary sources of information. Taylor and Francis shall not be liable for any losses, actions, claims, proceedings, demands, costs, expenses, damages, and other liabilities whatsoever or howsoever caused arising directly or indirectly in connection with, in relation to or arising out of the use of the Content.

This article may be used for research, teaching, and private study purposes. Any substantial or systematic reproduction, redistribution, reselling, loan, sub-licensing, systematic supply, or distribution in any form to anyone is expressly forbidden. Terms & Conditions of access and use can be found at [http://www.tandfonline.com/page/terms-and-conditions](http://www.tandfonline.com/page/terms-and-conditions)

Biosci. Biotechnol. Biochem., 66 (10), 2001–2016, 2002

**Review**

**Molecular Bases of Aerobic Bacterial Degradation of Dioxins: Involvement of Angular Dioxygenation**

Hideaki NOJIRI and Toshio OMORI†

Biotechnology Research Center, The University of Tokyo, 1-1-1 Yayoi, Bunkyo-ku, Tokyo 113-8657, Japan

In the last decade, extensive investigation has been done on the bacterial degradation of dioxins and its related compounds, because this class of chemicals is highly toxic and has been widely distributed in the environment. These studies have revealed the primary importance of a novel dioxygenation reaction, called angular dioxygenation, in the aerobic bacterial degradation pathway of dioxin. Accompanied by the electron transport proteins, Rieske nonheme iron oxygenase catalyzes the incorporation of oxygen atoms to the ether bond-carrying carbon (the angular position) and an adjacent carbon, resulting in the irreversible cleavage of the recalcitrant aryl ether bond. The 2,2′,3-trihydroxybiphenyl or 2,2′,3-trihydroxydiphenyl ether derivatives formed are degraded through *meta* cleavage. In addition to the degradation system of dibenzofuran and dibenzo-*p*-dioxin (the nonchlorinated model compounds of dioxin), those of fluorene and carbazole were shown to function in dioxin degradation. Some dioxin degradation pathways have been studied biochemically and genetically. In addition, feasibility studies have shown that some dioxin-degrading strains can function in actual dioxin-contaminated soil. These studies provide useful information for the establishment of a bioremediation method for dioxin contamination. This review summarizes recent progress on molecular and biochemical bases of the bacterial aerobic degradation of dioxin and related compounds.

Key words: angular dioxygenation; bacterial degradation; carbazole; dioxin; fluorene

Polychlorinated dibenzo-*p*-dioxins (PCDDs) and polychlorinated dibenzofurans (PCDFs), called dioxins, are produced by incineration processes and, unintentionally, as by-products during the synthesis of pesticides and herbicides. Many of their congeners are highly toxic compounds with undesirable effects on the environment and living beings, but have been widely distributed in the environment. The dioxin-like compounds should not only comprise the so-called “dirty dozen,” those PCDDs and PCDFs having a 2,3,7,8-substitution pattern, but also low-chlorinated and unchlorinated dibenzo-*p*-dioxins (DD) and dibenzofuran (DF), as well as their brominated analogues. Figure 1 illustrates the chemical structure and numbering system of the dioxins and its related compounds cited in this review. Complex mixtures of congeners of PCDDs and PCDFs are frequently detected in environmental samples such as sediments.¹⁻³⁾ The dioxins are recalcitrant molecules that are extremely persistent in soil sediments. For example, the half-life of 2,3,7,8-tetrachloro-DD in soils and sediments is 1 to 10 years.⁴⁻⁵⁾

Recently, bioremediation, the exploitation of the biodegradative activities of microorganisms to

![Chemical Structure of Polychlorinated Dibenzo-p-dioxins (PCDDs) and Polychlorinated Dibenzofurans (PCDFs) and Their Structural Analogues, and Their Numbering Conventions.](chemical_structure.png)

Fig. 1. Chemical Structure of Polychlorinated Dibenzo-*p*-dioxins (PCDDs) and Polychlorinated Dibenzofurans (PCDFs) and Their Structural Analogues, and Their Numbering Conventions.

---

† To whom correspondence should be addressed. Fax: +81-3-5841-8030; E-mail: aseigyo@mail.ecc.u-tokyo.ac.jp

Abbreviations: CAR, carbazole; DD, dibenzo-*p*-dioxin; DF, dibenzofuran; FN, fluorene; IS, insertion sequence; ORF, open reading frame; PAH, polycyclic aromatic hydrocarbon; PCDD, polychlorinated dibenzo-*p*-dioxin; PCDF, polychlorinated dibenzofuran; TEQ, toxicity equivalency quantity; Tn, transposon

involved in the degradation pathways for dioxins (DF and DD), and its structural analogues, fluorene (FN), carbazole (CAR), diphenyl ether and its derivatives, and dibenzothiophene. Other biodegradation pathways for these compounds via lateral dioxygenation or monooxygenation has been well investigated and reviewed. ${ }^{7-10}$ Diphenyl ether and dibenzothiophene degradation pathways containing angular dioxygenation have not been thoroughly described, but the FN- and CAR-degradation systems function in the degradation of dioxins. As summarized below, the degradation pathways of FN and CAR are similar to those of DF and DD, and the FN- and CAR-degrading bacteria can decompose dioxins through an identical pathway by DF (DD)-mineralizing bacteria. Among the degradation steps of dioxins, angular dioxygenation is the most important, because this single-step oxygenation destroys the planar structure of dioxin from which the dioxin toxicity derives.

In this paper, to outline the progress in the biochemical and molecular studies on the bacterial degradation of dioxins, we describe the degradation pathway of dioxin and related compounds via angular dioxygenation, and the features of their degradative genes and enzymes.

## Degradation of the Dioxins, FN and CAR via Angular Dioxygenation

### (1) Microbial degradation of DF and DD

Over the last decade, as listed in Table 1, a number of bacteria capable of mineralizing DF, DD, or both have been isolated by enrichment culture methods using a carbon-free mineral medium with DF or DD added as the sole source of carbon and energy. Fortnagel et al. ${ }^{16}$ and Engesser et al. ${ }^{11}$ established that initial angular attack could occur on DF at the 4 and 4a carbon atoms (Fig. 3). This angular dioxygenation for DF produces highly unstable hemiacetal products, which have not been observed directly. A compound analogous to DF, 9-fluorenone, was incubated with DF-grown bacteria, and the chemical structure of a dead-end product accumulated in the reaction mixture was identified. Based on the mass spectrum and nuclear magnetic resonance analytical data, the resultant product was identified as 1-hydro-1,1a-dihydroxyfluoren-9-one ${ }^{11,16,22}$ (the chemical structure is shown in Fig. 3). Thus, formation of hemiacetals from DF/DD was inferred.

Although, in early works using various DF-degrading bacteria, the formation of a trihydroxyl compound was not observed in the culture supernatant, Fortnagel et al. ${ }^{17}$ reported that a UV-derived mutant strain (designated HH69-II), which could no longer grow with DF as the carbon source, accumulates 2,2',3-trihydroxybiphenyl as a metabolic intermediate of DF (Fig. 3). From the culture supernatant of

Involvement of Angular Dioxygenation in Dioxin Degradation

Table 1. Aerobic Bacteria Able to Degrade or Co-Oxidize Dioxins, 9-Fluorenone, or CAR via Angular Dioxygenation

| Bacteria | Initial dioxygenase | Substrate$^a$ | Reference |
|----------|---------------------|---------------|-----------|
| Brevibacterium sp. strain DPO 220 | DF 4,4a-dioxygenase | DF, FN | Engesser et al., $^{11}$ Strubel et al.$^{12}$ |
| Terrabacter sp. strain DPO 1361 | DF 4,4a-dioxygenase | DF, FN | Engesser et al., $^{11}$ Strubel et al., $^{12,13}$ Trenz et al., $^{14}$ Schmid et al.$^{15}$ |
| Terrabacter sp. strain DPO 360 | DF 4,4a-dioxygenase | DF | Schmid et al.$^{15}$ |
| Sphingomonas sp. strain HH 69 | DF 4,4a-dioxygenase | DF, DD | Fortnagel et al., $^{16,17}$ Harms et al., $^{18-20}$ Schreiner et al.$^{21}$ |
| S. wittichii strain RW 1 | DF 4,4a-dioxygenase (Dioxin dioxygenase) | DF, DD, 9-Fluorenone | Wittich et al., $^{22}$ Bünz et al., $^{23}$ Bertini et al., $^{26}$ Wilkes et al., $^{27}$ Megharaj et al., $^{28}$ Arfmann et al., $^{29}$ Aermengaud and Timmis, $^{30,31}$ Aermengaud et al., $^{32,33,36}$ Halden et al., $^{34}$ Keim et al., $^{35}$ Yabuuchi et al.$^{37}$ |
| Terrabacter sp. strain DBF 63 | DF 4,4a-dioxygenase | DF, FN | Monna et al., $^{38}$ Kasuga et al., $^{39,40}$ Habe et al., $^{41,42}$ Nojiri et al.$^{43}$ |
| Sphingomonas sp. strain HH19k | DF 4,4a-dioxygenase | DF | Harms and Zehnder $^{44,45}$ |
| Sphingomonas sp. strain RW 16 | DF 4,4a-dioxygenase | DF | Wittich et al.$^{46}$ |
| Pseudomonas sp. strain F 274 | 9-Fluorenone 1,1a-dioxygenase | FN, DF | Selifonov et al., $^{47}$ Grifoll et al.$^{48,49}$ |
| P. mendocina strain MC 2 | 9-Fluorenone 1,1a-dioxygenase | 9-Fluorenone | Casellas et al.$^{50}$ |
| Ralstonia sp. strain RJGII.123 | CAR 1,9a-dioxygenase | CAR | Grosser et al., $^{51}$ Schneider et al.$^{52}$ |
| Pseudomonas sp. strain CA 06 | CAR 1,9a-dioxygenase | CAR | Ouchiyama et al.$^{53}$ |
| P. resinovorans strain CA 10 | CAR 1,9a-dioxygenase | CAR, DF, DD | Ouchiyama et al., $^{53}$ Kimura et al., $^{54}$ Sato et al., $^{55,56}$ Nojiri et al., $^{57,58,62}$ Nam et al., $^{59}$ Habe et al., $^{41,60}$ Widada et al.$^{61}$ |
| P. stutzeri strain ATCC 31258 | CAR 1,9a-dioxygenase | CAR | Hisatsuka and Sato $^{63}$ |
| Pseudomonas sp. strain LD 2 | CAR 1,9a-dioxygenase | CAR | Gieg et al.$^{64}$ |
| P. stutzeri strain OM 1 | CAR 1,9a-dioxygenase | CAR | Ouchiyama et al.$^{65}$ |
| Sphingomonas sp. strain CDH-7 | CAR 1,9a-dioxygenase | CAR | Kirimura et al.$^{66}$ |
| Burkholderia sp. strain JB 1 | Not identified | 2-chlorinated DF, 2-chlorinated DD$^b$ | Parsons et al.$^{67}$ |
| Burkholderia sp. strain LB 400 | Biphenyl 2,3-dioxygenase | DF, DD | Seeger et al.$^{68}$ |
| P. pseudoalcaligenes strain KF 707 | Biphenyl 2,3-dioxygenase$^c$ | DF, DD | Suenaga et al.$^{69}$ |

$a$ Only the parental compounds which were found to be metabolized *via* angular dioxygenation are shown, if not otherwise noted.

$b$ Angular attack was proposed by the transformation activities of 2-chlorinated DD and DF to 4-chlorocatechol and 5-chlorosalicylic acid, respectively.

$c$ The large ($\alpha$) subunit of the terminal oxygenase component was mutated by random priming recombination.

DF-using bacteria, salicylic acid was identified.$^{12,17,38}$ This indicates that 2,2',3-trihydroxybiphenyl is meta cleaved, and the resultant meta cleavage compound is hydrolyzed to salicylic acid and 2-hydroxypenta-2,4-dienoic acid.$^{13,23}$ In the DF-degradation pathway reported in *Pseudomonas* sp. strain HH 69,$^{17}$ which was reassigned to the genus *Sphingomonas*, $^{20}$ *S. wittichii* strain RW 1,$^{22}$ and *Staphylococcus auriculans* strain DBF 63,$^{38}$ which was reassigned to the genus *Terrabacter*, $^{39}$ the metabolic intermediate, salicylic acid, was branched into catechol and gentisic acid degradation pathways.

Initial angular dioxygenation for DD followed by spontaneous ring cleavage resulted in the formation of 2,2',3-trihydroxydiphenyl ether (Fig. 3). $^{22}$ Catechol has also been identified by GC-MS analysis as a metabolite of DD. Because the meta cleavage activity for 2,3-dihydroxydiphenyl ether could be detected, Wittich and coworkers proposed that catechol was formed from 2,2',3-trihydroxydiphenyl ether via meta cleavage and subsequent hydrolysis.$^{22}$ However, the possible product formed by the hydrolysis, 2-hydroxymuconic acid, has not been identified as a metabolite of DD. Pfeifer et al.$^{70}$ reported that

the meta cleavage of 2,3-dihydroxydiphenyl ether by 2,3-dihydroxybiphenyl 1,2-dioxygenase leads to the formation of phenol and 2-pyrone-6-carboxylate as products of ring fission and ether cleavage without participation of free intermediates. Thus, it is likely that only a meta cleavage enzyme is necessary for the production of catechol from 2,2',3-trihydroxydiphenyl ether as shown in Fig. 3. Similar to the catechol metabolism in the DF degradation pathway, catechol formed from DD was mineralized through meta, ortho, or both cleavage pathways.$^{22}$

(2) Microbial degradation of FN

FN is an abundant constituent of coal tar and creosote, and one of the simplest polycyclic aromatic hydrocarbons (PAHs). FN is one of the 16 PAHs in the list of priority pollutants compiled by the U.S. Environmental Protection Agency.$^{71}$ FN-using bacteria, *Pseudomonas* sp. strain F 274, $^{48}$ and 9-fluorenone-using bacteria, *P. mendocina* strain MC 2, $^{50}$ were isolated from soil severely contaminated with creosote and PAHs, respectively. Several DF-using strains can also grow on FN.$^{14,38}$ Based on the identification of 9-fluorenol and

H. NOJIRI and T. OMORI

![Chemical Structures and Reactions](chemical_structures.png)

Fig. 3. Overview of Bacterial Degradative Pathways for Dibenzofuran (DF), Dibenzo-$p$-dioxin (DD), Fluorene (FN), and Carbazole (CAR) via Angular Dioxygenation.

The structures shown in brackets have not been characterized. The red, blue, and green arrows show angular dioxygenation, $meta$ cleavage, and hydrolysis, respectively. The production of catechol and 2-pyrone-6-carboxylic acid from the $meta$ cleavage compound of 2,2',3-trihydroxydiphenyl ether in the DD degradation pathway was proposed by the results reported by Pfeifer et al. ${}^{70)}$ 2-Pyrone-6-carboxylic acid is a by-product. The arrows with solid and broken lines indicate the enzymatic and spontaneous conversions, respectively.

9-fluorenone as metabolic intermediates of FN, monooxygenation at the C9 position to give 9-fluorenol followed by the dehydrogenation to 9-fluorenone was shown to be the initial conversion before angular dioxygenation of the aromatic ring (Fig. 3). ${}^{14,38,48}$ Because 1-hydro-1,1a-dihydroxy-9-fluorenone was identified as the degradation product of FN or 9-fluorenone, angular dioxygenation has been established as the crucial step in FN degradation (Fig. 3). ${}^{11,47,50}$ Unlike the hemiacetal intermediates produced by angular dioxygenation of DF and DD, the angular dioxygenation product of 9-fluorenone is chemically stable. Dehydrogenation of the 1-sec-hydroxyl group of 1-hydro-1,1a-dihydroxy-9-fluorenone would yield a $\beta$-diketone-type compound, hydrolysis of which at the C9–C1a bond would give 2'-carboxy-2,3-dihydroxybiphenyl. 8-Hydroxy-3,4-benzocoumarin was identified as a metabolite of FN, ${}^{48,50}$ but this compound is in fact the $\delta$-lactone of

2'-carboxy-2,3-dihydroxybiphenyl and could be formed as a result of a reversible dehydration reaction. 2'-Carboxy-2,3-dihydroxybiphenyl was $meta$ cleaved at the C1–C2 bond, and subsequent hydrolysis resulted in the formation of phthalic acid and 2-hydroxypenta-2,4-dienoic acid, as shown in Fig. 3. ${}^{48}$ Phthalic acid is further degraded by *Pseudomonas* sp. strain F274 via protocatechuic acid. ${}^{48}$

(3) Microbial degradation of CAR

CAR is a nitrogen heterocycle released into the environment with fossil fuels or their products such as creosote. ${}^{72}$ CAR is used as a chemical feedstock for the production of dyes, medicines, and plastics, and is known to possess mutagenic and toxic activities ${}^{73}$ and also to be a recalcitrant molecule. As shown in Table 1, several CAR-using bacteria were isolated by enrichment culture, with CAR as a sole source of carbon and energy, or of carbon, nitrogen, and energy.

Ouchiyama *et al.*⁵³ isolated *Pseudomonas* sp. strain CA 06 and *P. resinovorans* strain CA 10 from soil and activated sludge samples, respectively, which use CAR as the sole source of carbon, nitrogen, and energy. Anthranilic acid and catechol were identified as the main metabolites of CAR by HPLC and GC-MS. In addition, *meta* cleavage activity for 2,3-dihydroxybiphenyl was strongly induced in strain CA 10 cell grown with CAR. These results suggested that strain CA 10 degrades CAR to anthranilic acid through angular dioxygenation, *meta* cleavage, and hydrolysis. This degradation pathway of CAR by strain CA 10 is homologous to that of DF degradation as shown in Fig. 3. From the extract of culture medium of various CAR-using bacteria, anthranilic acid was identified as a main metabolite,⁵²,⁶³⁻⁶⁶ suggesting that these CAR-using bacteria degrade CAR through a pathway similar to that of strain CA 10.

When anthranilic acid was used for a growth substrate by strain CA 10, catechol and a small amount of cis, cis-muconic acid were detected.⁵³ These results indicate the possibility that anthranilic acid is converted to catechol by anthranilate 1,2-dioxygenase, and that the catechol formed is mineralized via ortho cleavage by strain CA 10. On the other hand, *P. stutzeri* strain OM1 was found to metabolize catechol via *meta* cleavage.⁶⁵

### Biochemical and Genetic Characterization of the Dioxin Degradation Pathway

The genes for the *meta* cleavage enzyme involved in DF and DD degradations were isolated by shotgun cloning with the enzyme activity used to screen for the positive clone.¹⁵,²⁵,³⁹ This method has been used to clone the aromatic compound degradation genes, because a colony of *Escherichia coli* cells having the *meta* cleavage enzyme gene has a bright yellow color from the *meta* cleavage compound when an appropriate substrate having the catechol moiety is sprayed on. In addition, angular dioxygenase, *meta* cleavage enzyme, and hydrolase were purified from the DF-degrading bacteria.¹⁵,²³,²⁴ The degradative genes involved in the degradation pathways of easily degradable aromatic compounds, such as naphthalene, biphenyl, toluene, and phenol, often have a well-clustered gene organization. Thus, based on the genetic information on the *meta* cleavage enzyme and purified enzyme, attempts to clone the other degradative genes from DF- and DD-degrading bacteria by the gene-walking method have been done. However, these attempts almost completely failed, because the DF- and DD-degradative genes reported so far are dispersed in the genome and do not constitute a simple gene cluster.³²,⁴⁰,⁴³ This is why the dioxin degradation pathway has not been well characterized. In contrast, CAR and dioxin degradation pathway of *P.*

resinovorans strain CA 10 has been characterized reasonably well at the genetic level.⁵⁵,⁵⁶,⁵⁸,⁶²

#### (1) Dioxin degradation pathway of *S. wittichii* strain RW1

*S. wittichii* strain RW1 is the most well-characterized DF/DD-degrading bacterium. Since Wittich *et al.*²² reported the isolation of the DF and DD-degrader, strain RW1, there have been intensive attempts to isolate the enzymes and genes involved in the DF and DD degradation. Two isofunctional *meta* cleavage compound hydrolases (hydrolase H1 and H2), and respective components of the initial angular dioxygenase system, DF 4,4a-dioxygenase (dioxin dioxygenase) were purified from strain RW1 cells by Cook and coworkers.²³,²⁴ The hydrolases H1 and H2 were presumed to be monomeric. There was a 50% identity between the N-terminal amino acid sequences of the two hydrolases, but no similar protein sequences were observed by database search analysis.²³ The DF 4,4a-dioxygenase system was found to consist of a terminal oxygenase that has a heterotetrameric structure (*α₂β₂*) and an electron-transfer system.²⁴ An atypical electron transfer chain interacting with the terminal oxygenase component of the dioxin dioxygenase system, consisting of a 12-kDa putidaredoxin-type [2Fe–2S] ferredoxin and two isofunctional monomeric flavoproteins (ferredoxin reductase A1 and A2), has been purified, and the sequences of the N-termini of the polypeptides have been analyzed.²⁴ The characteristics of the electron transport chain indicate that the dioxin dioxygenase system of strain RW1 is a class IIA dioxygenase system according to the classification scheme of Batie *et al.*⁷⁴

As described above, the *dbfB* gene encoding 2,2′,3-trihydroxybiphenyl 1,2-dioxygenase, the *meta* cleavage enzyme involved in DF degradation, was cloned from the genomic cosmid library of strain RW1.²⁵ Comparison of the amino acid sequence of DbfB showed significant similarities to the *meta* cleavage enzyme involved in the biphenyl/polychlorinated biphenyl degradation pathway. Unlike most *meta* cleavage enzymes, which have an oligomeric quaternary structure, the DbfB of strain RW1 is a monomeric protein. ¹H NMR analysis of purified DbfB showed that the high spin Fe(II) iron present in the active form of the enzyme is coordinated by at least two His residues.²⁶

Using the N-terminal sequence of purified putidaredoxin-type [2Fe–2S] ferredoxin and the conserved stretch of only four amino acids at the C-terminal of the ferredoxin, Armengaud and Timmis³⁰ designed a nested-PCR strategy to obtain a specific probe for the ferredoxin gene. With this probe, the authors identified the ferredoxin gene, *fdxI*. Ferredoxin FdxI of strain RW1 shares about 40% identity with several [2Fe–2S] ferredoxins involved in elec-

2006

H. NOJIRI and T. OMORI

**dxn locus**


dbfB —— dxnA1 A2 dxnB dxnC fdx3 dxnD dxnE F G H I transposase


**redA2 locus**


ORF R1 redA2


**fdx1 locus**


ORF E1 fdx1 ORF E2 ORF E3 ORF E4


2 kb

---

| Symbol | Gene product or putative gene product |
|--------|---------------------------------------|
| ▲      | Component of dioxin dioxygenase       |
| ▶      | meta Cleavage enzyme                 |
| ▷      | meta Cleavage compound hydrolase     |
| ▷      | 4-Hydroxysalicylic acid/hydroxyquinol degradative enzyme |
| ▶      | Other enzyme or unknown function     |

---

Fig. 4. Genetic Organization of Loci Encompassing the Genes Involved in DF and DD Degradation by *S. wittichii* Strain RW1.

This figure is adapted from Armengaud et al.³²,³³ The pentagons and triangles in the physical map indicate the size, location, and the direction of transcription of the ORFs derived from the DNA sequence analysis.

tron transfer to bacterial monooxygenase systems. UV and visible spectrophotometry and electron paramagnetic resonance spectroscopy using the hyperexpressed Fdx1, it was found that Fdx1 contains a putidaredoxin-type [2Fe–2S] cluster. Fdx1 is thus related to the group of ferredoxins that typically donate electrons to monooxygenases, named cytochrome $P_{450}$, and that are an atypical electron transfer system for the Rieske nonheme iron oxygenase system. The redA2 gene encoding the ferredoxin reductase A2²⁴) was also cloned by PCR.³¹) From the nucleotide and deduced amino acid sequences, as well as from the biochemical data obtained from a recombinant form of this reductase, it is evident that this flavoprotein is similar to class-I cytochrome $P_{450}$-type reductase.⁷⁵) Indeed, the two proteins of the electron transport system, RedA2 and Fdx1, are similar to their counterparts supplying electrons to the well-characterized cytochrome $P_{450cam}$,⁷⁶) a monooxygenase from *P. putida*, which is able to oxidize camphor. The genes encoding the terminal oxygenase component of dioxin dioxygenase were also isolated by a PCR-based technique.³²) The dxnA1 and dxnA2 cistrons, encoding the large (α) and small (β) subunits of the terminal oxygenase component, were just upstream of the hydrolase gene dxnB encoding hydrolase H1²³) (Fig. 4). In the 4.5 kb upstream region of the dxnA1 gene, the dbfB gene was located in the opposite direction (Fig. 4). Recently, Armengaud et al.³³) reported that there are nine additional genes, dxnC to dxnI, in the downstream region of dxnB, constituting the same operon with dxnA1A2B (Fig. 4). DxnDFEGHI were found to form a complete 4-hydroxysalicylic acid/hydroxyquinol-degradative pathway. A homology search analysis found an additional putidaredoxin-type

ferredoxin gene (fdx3) in this operon (Fig. 4). Fdx3 functions as an electron transport protein from RedA2 to DxnA1A2 in the dioxin dioxygenase system.³⁶) Thus, *S. wittichii* degrades DF and DD by the enzymes as shown in the Fig. 5, and the respective enzymes was encoded in the physically distinct loci, dxnA1A2Bfdx3, fdx1, redA2, and dbfB in the RW1 genome. Aermengaud et al.³²) reported that, while the expression of the dbfB gene was constitutive, that of dbfA1A2 is modulated according to the available carbon source. However, the detailed transcriptional control of dioxin degradation genes of strain RW1 has not been reported.

(2) **Dioxin degradation pathway by Terrabacter sp. strain DBF63**

Two genes encoding meta cleavage enzymes [dbfB and open reading frame K1 (ORFK1)] were cloned from *Terrabacter* sp. strain DBF63 by shotgun cloning on the basis of the expressed activity of meta cleavage enzymes.³⁹) Based on the comparison of the substrate specificity, it was suggested that, at least DbfB is involved in the DF degradation by strain DBF63. Nucleotide sequence analysis showed dbfB and dbfC cistrons encoding a meta cleavage enzyme and a meta cleavage compound hydrolase, respectively.

Recently, Kasuga et al.⁴⁰) succeeded in cloning the dbfA1 and dbfA2 genes, which encode the large (α) and small (β) subunits of the terminal oxygenase component of the DF 4,4a-dioxygenase system, respectively, from strain DBF63 genome by a PCR-based method. DbfA1 and DbfA2 showed moderate homology to the respective subunits of other ring-hydroxylating dioxygenases (less than 40%), and some motifs such as the Fe(II) binding site and the

Involvement of Angular Dioxygenation in Dioxin Degradation

NAD⁺
Red.
Ferredoxin
reductase
Ox.
NADH
Ox.
Red.
Ferredoxin
Red.
Iron sulfur
protein
(terminal
oxygenase)
Ox.

I

OH
II
OH
OH
meta Cleavage
enzyme
OH
III
HO₂C
OH
Hydrolase
IV
OH
CO₂H

| S. wittichii | RedA2 | Fdx1 | DxnA1A2 | DbfB | DxnB |
| --- | --- | --- | --- | --- | --- |
| strain RW1 | Reductase A1 | Fdx3 |  |  | Hydrolase H2 |

| Terrabacter sp. | ? | ? | DbfA1A2 | DbfB | DbfC |
| --- | --- | --- | --- | --- | --- |
| strain DBF63 |  |  |  |  |  |

| P. resinovorans | CarAd | CarAc | CarAa | CarBaBb | CarC |
| --- | --- | --- | --- | --- | --- |
| strain CA10 |  |  |  |  |  |

Fig. 5. DF Degradation by the Enzyme Systems Harbored by S. wittichii Strain RW1, Terrabacter sp. Strain DBF63, or P. resinovorans Strain CA10.

The reactions done by the multi-component ring-hydroxylating dioxygenase (iron-sulfur protein; terminal oxygenase component) and their electron transfer system, meta cleavage enzyme, and hydrolase are shown. Chemical designations: I, dibenzofuran (DF); II, 2,2′,3-trihydroxybiphenyl; III, 2-hydroxy-6-(2-hydroxyphenyl)-6-oxo-2,4-hexadienoic acid; IV, salicylic acid.

Rieske-type [2Fe–2S] cluster ligands were conserved in DbfA1. DF 4,4a-dioxygenase activity was confirmed by the resting cell reaction of E. coli cells containing the cloned *dbfA1* and *dbfA2* genes with the complementation of nonspecific ferredoxin and ferredoxin reductase components of E. coli. It was also found that DbfA1A2 can catalyze the angular dioxygenation for 9-fluorenone (Habe et al., unpublished results) and 9-hydroxylation for FN. Phylogenetic analysis showed that DbfA1 formed a branch with a recently reported large (α) subunit of PAH dioxygenase from Gram-positive bacteria. The cistron *dbfA1A2* is a part of an about 7.3-kb-long operon consisting of 8 ORFs. Although a gene encoding a putative [3Fe–4S] ferredoxin was observed in this operon, the ferredoxin reductase gene has not been identified (Habe et al., unpublished results). The enzymes involved in the dioxin degradation by strain DBF63 are summarized in Fig. 5.

By the pulsed field gel electrophoresis and Southern hybridization, two linear plasmids, pDBF1 and pDBF2, about 160 kb and 190 kb long, respectively, were detected in strain DBF63, and the *dbfA1A2* cistron was shown to be located on these two plasmids. On the other hand, *dbfBC* genes were located on the chromosome. Although the regulator protein and effector molecule have not been identified, reverse transcription-PCR analysis showed that *dbfA1A2* and *dbfBC* genes are strongly expressed in the DF-grown strain DBF63 cells. These results suggest that *dbfA1A2* and *dbfBC* genes are involved in the DF degradation, and that these degradative genes are dispersed on the strain DBF63 genome.

(3) Degradation pathway for CAR and dioxin of P. resinovorans strain CA10

The transposon mutagenesis of *P. resinovorans* strain CA10 indicated a putative gene cluster involved in the CAR degradation including the meta cleavage enzyme gene. On the basis of these results, we cloned the intact CAR-degrading *car* gene cluster from the wild-type strain CA10 by shotgun cloning, using the meta cleavage activity for 2,3-dihydroxybiphenyl. As a result of the nucleotide sequence analysis of the cloned DNA fragment, homology search analyses, and the measurement of the enzyme activities, the genes encoding angular dioxygenase for CAR (CAR 1,9a-dioxygenase; CarAaAcAd), meta cleavage enzyme (CarBaBb), and meta cleavage compound hydrolase (CarC) were identified, as shown in Fig. 6(A). It was found that the 1,263-bp DNA region containing the entire *carAa* gene was tandemly duplicated except for one base, and that there were only three bases (GGC) between these tandemly-linked DNA regions. To our knowledge, such gene structure has never been observed in other degradative gene clusters. The significance of this duplication of *carAa* genes in strain CA10 cells is unknown. Recently, we succeeded in isolating several CAR- and dioxin-degrading bacteria, some of which have a homologous *car* gene cluster containing a single copy of the *carAa* gene (Widada et al., unpublished results). Comparison of the CAR- and dioxin-degrading capacity between bacteria having different copy numbers of the *carAa* gene might provide us insight on the significance of this gene duplication. In addition, the genes encoding the terminal oxygenase component (CarAa) of CAR 1,9a-dioxygenase are separated from the genes encoding the ferredoxin component (CarAc) and ferredoxin reductase com-

2008
H. NOJIRI and T. OMORI

**A**
5 kb

Symbols
- car gene cluster
  - involved in the conversion of CAR to anthranilic acid (carAaAcAdBaBbC)
  - and 2-hydroxypenta-2,4-dienoic acid to TCA cycle intermediates (carDEF)
- Other genes
- Insertion sequence
- ant gene cluster
  - encoding anthranilate 1,2-dioxygenase
- cat gene cluster
  - involved in the metabolism of catechol

**Fig. 6. Physical Maps of the 44,266-base Pair EcoRV-EcoRI DNA Region Containing car and ant Genes**<sup>55,56,58</sup> (A) **and 8,918-base Pair EcoRI-NotI DNA Region Containing catRBCA Genes**<sup>62</sup> (B), **That are Involved in the Carbazole (CAR) Degradation by P. resinovorans Strain CA10.**

The pentagons and triangles in the physical map indicate the size, location, and the direction of transcription of the ORFs derived from the DNA sequence analysis. This figure is adapted from Nojiri et al.<sup>58,62</sup> Restriction site designations are: A, ApaI; C, ClaI; EI, EcoRI; EV, EcoRV; H, HindIII; K, KpnI; N, NotI; P, PstI; Sl, SalI; Xh, XhoI.

component (CarAd) by a DNA fragment of about 2-kb encoding carBaBbC (Fig. 6(A)). These first three enzymes in CAR degradation have been found to function in dioxin degradation as the counterparts of the corresponding enzymes in DF (DD)-degrading bacteria.<sup>39,41,56,57</sup> The functions of the Car enzymes of strain CA10 in DF degradation may be as shown in Fig. 5.

On the basis of the identification of the oxygenation product after a biotransformation experiment, CAR 1,9a-dioxygenase was shown to catalyze the angular dioxygenation for CAR, DF, DD, xanthene, and phenoxathiin, but not for 9-fluorenone or dibenzothiophene,<sup>56,57</sup> indicating that the angular dioxygenation occurs at the angular position adjacent to an oxygen or nitrogen atom more preferably than that adjacent to a sulfur or carbon atom. The three components of the CAR 1,9a-dioxygenase system have been purified (Nam et al., in preparation). Based on gel-filtration analyses, CarAa, CarAc, and CarAd were trimer, monomer, and monomer proteins, respectively. In the purified His-tagged CarAd, 0.6 mole FAD per mole of protein was detected. UV and visible absorption spectroscopy indicated that CarAa, CarAc, and CarAd proteins have the Rieske-type,<sup>78</sup> Rieske-type,<sup>78</sup> and plant-type<sup>79</sup> [2Fe–2S] clusters, respectively. These facts are in agreement with the functions suggested from the deduced amino acid sequences.<sup>56</sup>

The Rieske nonheme iron oxygenase systems are classified into three classes, classes I, II, and III, in terms of the number of constituent components and the nature of the redox centers.<sup>74,80</sup> Further subdivision is based on the type of flavin in the reductase (Ia and IB), or the coordination of the [2Fe–2S] cluster in the ferredoxin (IIa and IIb). Based on this classification system, CAR 1,9a-dioxygenase seems to have a class III electron transport chain like a well-known naphthalene dioxygenase. On the other hand, Nam et al.<sup>59</sup> proposed a new classification scheme for the oxygenase components based on their amino acid sequence similarity in the α subunits of terminal oxygenase component, because the Batie classification system could not cover all of the dioxygenase systems. On the basis of Nam’s scheme, terminal oxygenases were classified into four groups, I–IV. The oxygenase components included by group I in Nam’s classification have only the identical subunit, and most of the terminal oxygenase components of class IA oxygenases in the Batie classification belong to this group I. The terminal oxygenase components of class IB, class II, and class III oxygenase systems in the Batie classification are mainly classified into group II, group IV, and group III, respectively, in Nam’s classification system. The phylogenetic study found that CarAa has a closer relation to the terminal oxygenase contained by class IA dioxygenase systems, and CarAa was classified into group I in Nam’s classification.<sup>59</sup>

Recently, Schmidt and Shaw<sup>81</sup> suggested that the terminal oxygenase and ferredoxin components containing a Rieske-motif form a superfamily with one ancestor. To investigate the evolutionary relationships of CarAa and CarAc with other Rieske-type

Involvement of Angular Dioxygenation in Dioxin Degradation

Fig. 7. Phylogenetic Tree of Ferredoxin and Terminal Oxygenase Components of Oxygenase Systems.

The scale bar denotes 0.1 substitution per site. The tree was constructed with the Clustal W package. The name of each bacterial strain and its DDBJ/EMBL/GenBank or SWISSPROT accession number are shown after the enzyme name in parentheses. Grouping into Groups I–IV of oxygenase proteins is based on the scheme of Nam et al.⁵⁹ This figure is adapted from Nam et al. (in preparation). The terminal oxygenase (CarAa) and ferredoxin (CarAc) of CAR 1,9a-dioxygenase from *P. resinovorans* strain CA10 are shaded.

proteins, a phylogenetic tree was established (Fig. 7, Nam et al., in preparation). As shown in Fig. 7, the ferredoxins involved in the monooxygenase system and dioxygenase system constitute the distinctive clusters, and CarAc was contained in the ferredoxin cluster of dioxygenase system. As for the oxygenase components, the distant affiliations of the oxygenases were observed between group I and groups II, III, and IV in Nam’s classification. These phylogenies indicate that group I oxygenases are evolutionarily different from group II, III, and IV oxygenases. In addition, among the oxygenases classified into group I, CarAa of strain CA10 and OxoO of strain 86 were phylogenetically distant from other group I oxygenases (Fig. 7). On the other hand, the phylogenetic tree of ferredoxin-NADP⁺ reductase-type reductases showed that CarAd of strain CA10 is more closely related to the reductase components of some monooxygenase systems, such as toluene 4-monooxygenase, toluene/*o*-xylene monooxygenase, and alkene monooxygenase, than those of class IB and III dioxygenase systems (Fig. 8) (Nam et al., in preparation). The distant phylogenetic affiliations of respective components of CAR 1,9a-dioxygenase indicate the possibility that some of these three components have been recruited from different oxygenase systems.

We have analyzed the nucleotide sequence of the 27,939-bp-long upstream and 9,448-bp-long downstream regions of *carAaAaBaBbCAc(ORF7)Ad*

Fig. 8. Phylogenetic Tree of the Reductase Components of Oxygenase Systems.

The scale bar denotes 0.2 substitution per site. The tree was constructed by the neighbor-joining method (Phylip package version 3.573c). The name of each bacterial strain and its DDBJ/EMBL/GenBank or SWISSPROT accession number are shown after the enzyme name in parentheses. Grouping into Classes IB and III is based on the scheme of Batie et al. This figure is adapted from Nam et al. (in preparation). The ferredoxin reductase (CarAd) of CAR 1,9a-dioxygenase from *P. resinovorans* strain CA10 is shaded.

operon (Fig. 6(B)), 62) it can be considered that strain CA10 had acquired a metabolic capacity for dioxin and CAR by recruitment of pCAR1. In other bacteria having the ability to degrade both dioxin and CAR, we have also detected the pCAR1-like plasmid containing the car gene cluster homologue, although chromosomally encoded car gene clusters were found as well (Widada et al., unpublished results). This phenomenon suggests that pCAR1 is important in the distribution of the degradation capacity for dioxin and CAR in the environment. Very recently, the entire nucleotide sequence (199,035 bp) of pCAR1 was analyzed (Maeda et al., unpublished results). The presence of the homologous genes to the trh and tra genes involved in the conjugal transfer of the plasmids Rts1 from Proteus vulgaris 83) and R27 from Salmonella strains 84) suggested the possibility that pCAR1 is a self-transmissible plasmid. Interestingly, it was also shown that the entire car and ant loci are contained in a 73-kb-long transposon, Tn4676. The transposase and cointegrate resolution proteins are closely related to those of the toluene- and xylene-degrading transposon, Tn4651, 85) identified from the TOL plasmid pWW0, although the replication and maintenance systems of pCAR1 has no relationship to those of pWW0. Because several bacterial strains have Tn4676 inserted on their chromosome (Shintani et al., unpublished results), both the plasmid pCAR1 and Tn4676 are important in the distribution of the degrading capacity for dioxin and CAR in the environment.

### Microbial Degradation of Chlorinated DF and DD

#### (I) Aerobic degradation of chlorinated DFs and DDs

Although the nonhalogenated basic structure DF and DD is effectively mineralized by appropriate bacterial strains as mentioned above, their polychlorinated derivatives are not. The abilities of DF (DD)-or CAR-mineralizing bacteria to oxidize mono-to trichlorinated congeners of DD and DF were analyzed. 19,27,35,41) Harms et al. 19) reported that a mutant strain Pseudomonas (Sphingomonas) sp. strain HH69-II, which is defective in 2,3-dihydroxybiphenyl 1,2-dioxygenase, led to the formation of an equal amount of 4′-chloro-2,2′,3-trihydroxybiphenyl and 4-chloro-2,2′,3-trihydroxybiphenyl from 3-chloro-DF. Wilkes et al. 27) demonstrated that most mono- and dichlorinated DFs and DDs tested were degraded to the corresponding mono-and dichlorinated salicylic acids and catechols, respectively, together with salicylic acid and catechol. Wittich et al. 46) also reported that the conversion of 3-chloro-DF in the presence of an inhibitor of meta cleavage by Sphingomonas sp. strain RW16 led to the accumulation of two metabolites that are likely to be 4′-chloro-2,2′,3-trihydroxybiphenyl and 4-chloro-2,2′,3-trihydroxybiphenyl. These results indicated that angular dioxygenase from these bacteria attacked both the substituted and the nonsubstituted aromatic nuclei of monochlorinated DFs and DDs nonspecifically. However, in these studies, the concentrations of the target substrates were from 100 to 1,000 ppm. On the other hand, Parsons et al. 67) proposed that the initial dioxygenation took place on the nonchlorinated ring during the degradation of a low concentration (0.3 ppm) of 2-chloro-DF and 2-chloro-DD by the biphenyl-using Burkholderia sp. strain JB1, since 5-chlorosalicylic acid and 4-chlorocatechol as well as only one isomer each of chlorinated 2,2′,3-trihydroxybiphenyl and 2,2′,3-trihydroxydiphenyl ether were detected in respective biotransformation experiments. In our experiments on dioxin degradability by the CAR 1,9a-dioxygenase from strain CA10 and the DF 4,4a-dioxygenase from strain DBF63, we found that these two angular dioxygenases act mainly on the nonsubstituted aromatic nucleus of chlorinated DFs and DDs (10 ppm), although both enzymes were able to act on the chlorine-substituted aromatic rings. 41) Wilkes et al. 27) also suggested that at least the initial conversion rates for the chlorinated DFs and DDs decreased with increasing chlorine substitution. In addition, based on a biotransformation experiment by using parental bacterial strain, strains CA10 and DBF63, and E. coli transformants expressing the meta cleavage enzyme and hydrolase in addition to the initial angular dioxygenase, Habe et al. 41) suggested that the influence of the chlorine substitution pattern of mono- to trichlorinated DFs and DDs on the formation of corresponding chlorocatechol and chlorosalicylic acid may not only depend on the substrate specificity of the initial angular dioxygenases but also on those of subsequent acting enzymes, especially hydrolase. Two DF (DD)-degrading bacteria, S. wit-tichii strain RW1 and Sphingomonas sp. strain RW16, can transform monochlorinated DF by angular dioxygenation, meta cleavage, and hydrolysis, but the resultant products, corresponding monochlorinated salicylic acids, were accumulated in the culture. 27,29,46) Wittich and co-workers reported the complete mineralization of monochlorinated DF by the consortiums consisting of a DF-degrader and a chlorosalicylic acid-degrading bacteria. 29,46)

An extensive study of the influence of the substituted pattern on the depletion of many of the 210 congeners of polychlorinated DFs and DDs by several bacterial strains was reported by Schreiner et al. 21) In the study, the DF-degrader Sphingomonas (formerly Pseudomonas 17) sp. strain HH69 degraded 31% of 2,3,7,8-tetrachloro-DF and 15% of 2,3,7,8-tetrachloro-DD within 84 days. Surprisingly, 1,2,4,5-tetrachlorobenzene-mineralizing Burkholderia

(2012) H. NOJIRI and T. OMORI

(formerly *Pseudomonas*<sup>86</sup>) sp. strain PS12 degraded 64% of 2,3,7,8-tetrachloro-DF in the same time, and 100% of 2,3,7,8-tetrachloro-DD within 25 days. Chlorine substitution in the 1-, 4-, 6-, or 9-position generally retarded the biodegradation.<sup>21)</sup>

### (2) Bacterial biodegradation of dioxins in soils and sediments

Polychlorinated DDs/DFs are often bound to soil organic matter and surfaces of particles. The reduced bioavailability of halogenated DD/DF, therefore, is of interest and was examined by Harms and Zehnder.<sup>44,45)</sup> Megharaj *et al.*<sup>28)</sup> reported that culturing the *S. wittichii* strain RW1 first in soil extract medium prolonged the survival of this strain in soil. Halden *et al.*<sup>34)</sup> reported the removal of DF, DD, and 2-chloro-DD (10 ppm each) from a soil microcosm by the addition of strain RW1. The density of strain RW1 cells and the contents of soil organic matter influenced the rate and extent of removal. Recently, we tried experimentally to bioremediate an actual dioxin-contaminated soil that was contaminated mainly by tetra-to octachlorinated dioxins, by using the soil slurry system and *P. resinovorans* strain CA10 cells.<sup>60)</sup> The soil slurry (ratio of soil:water, 1:5, wt/vol) prepared with dioxin-contaminated soil collected at an incinerator site was incubated with CAR-grown strain CA10 cells. During the 7-day incubation, the total amount of chlorinated DD and DF congeners and toxicity equivalency quantity (TEQ) decreased from 725 to 665 ng/g soil and 11 to 9.4 ng TEQ/g soil, respectively, by a single inoculation of the CAR-grown strain CA10 cells (10<sup>11</sup> CFU/g dry soil). Although the degradation rate of total dioxin was 8.3%, strain CA10 had a potential to transform tetra-to heptachlorinated congeners including the most toxic compound, 2,3,7,8-tetrachlorinated DD. With a similar slurry system, a preliminary bioremediation experiment with *Terrabacter* sp. strain DBF63 cells was done.<sup>42)</sup> From this experiment, we detected approximately 10% depletion of tetra-to hexachlorinated congeners by 7 days of incubation with DF-grown strain DBF63 cells (10<sup>8</sup> CFU/g dry soil).

dioxygenase with point mutation(s) introduced of *P. pseudoalcaligenes* strain KF707 catalyzed the angular dioxygenation for DD and DF. These results indicate that this novel dioxygenation is one of the reactions originating from the “relaxed” substrate specificity of the Rieske nonheme iron oxygenase super family. Recently, the three-dimensional structure of the terminal oxygenase component of naphthalene 1,2-dioxygenase has been reported.<sup>77,87)</sup> The structure of the angular dioxygenase will provide useful information on the “relaxed” substrate recognition and activation of the reaction.

On the basis of biochemical and genetic information, the extension of the substrate range of dioxin-degrading bacteria, higher expression of dioxin degrading enzyme(s), and the construction of the hybrid metabolic pathway allowing the improved mineralizing capacity for dioxins will probably be achieved in the near future. Although, to optimize the biodegradation conditions, a laboratory scale-microcosm study, a feasibility study with a pilot plant, and a field trial on a long-term basis are still necessary, bacteria harboring the dioxin-degrading capacity are promising for the bioremediation of dioxin contamination.

The well-clustered genetic organization of entire pathways or independently functioning pathway segments developed through exposure to corresponding compounds for a long time. On the other hand, the degradative genes involved in the degradation of higher recalcitrant compounds seemed to be disordered or separated. In fact, dioxin-degrading genes were separated from each other on the genomes of *S. wittichii* strain RW1<sup>32)</sup> and *Terrabacter* sp. strain DBF63.<sup>43)</sup> Dioxin- and CAR-degrading *car* genes of *P. resinovorans* strain CA10 are disordered and have a novel mosaic structure.<sup>58)</sup> Perhaps the different levels of maturation in genetic structures resulted from the differences in the length of exposure to xenobiotics or rarity of the gene(s) indispensable for degradation. That is, the genetic analysis of the degradative genes for dioxin and CAR will provide useful information on the maturation (evolution) of xenobiotic-degrading gene clusters.

## Conclusions and Prospects

Because angular dioxygenation for dioxin results in the spontaneous cleavage of the three-ring structure, the single-step oxidation destroys the planar structure from which the toxicity of dioxin originated. This fact indicates the primary importance of initial angular dioxygenation in decontamination of dioxin. Recently, biphenyl 2,3-dioxygenase, the initial dioxygenase in the biphenyl degradation pathway by *Burkholderia* sp. strain LB400, has been reported to catalyze the angular dioxygenation for DD and DF.<sup>68)</sup> Suenaga *et al.*<sup>69)</sup> also reported that the biphenyl 2,3-

## Acknowledgments

A part of this work was supported by the Program for Promotion of Basic Research Activities for Innovative Biosciences (PROBRAIN) in Japan.

## References

1. Schramm, K. W., Henkelmann, B., and Kettrup, A., PCDD/F sources and levels in river Elbe sediments. *Wat. Res.*, **29**, 2160–2166 (1995).
2. Engwall, M., Broman, D., Näf, C., Zebühr, Y., and Brunström, B., Dioxin-like compounds in HPLC-

fractionated extracts of marine samples from the east
and west coast of Sweden: bioassay- and instrumentally-derived TCDD equivalents. Mar. Poll. Bull., 34,
1032–1040 (1997).

3) Götz, R., Steiner, B., Sievers, S., Friesel, P., Roch,
K., Schwörer, R., and Haag, F., Dioxin, dioxin-like
PCBs and organotin compounds in the river Elbe and
the Hamburg harbour: identification of sources. Wat.
Sci. Tech., 37, 207–215 (1998).

4) Philippi, M., Krasnobajew, V., Zeyer, J., and
Hütter, R., Fate of TCDD in microbial cultures and
in soil under laboratory conditions. In “Microbial
Degradation of Xenobiotics and Recalcitrant Compounds”, eds. Leisinger, T., Hütter, R., Cook, A.
M., and Nüesch, J., Academic Press, New York, pp.
221–233 (1981).

5) Freeman, R. M., and Schroy, J. M., Environmental
mobility of 2,3,7,8-TCDD. Chemosphere, 14,
873–876 (1985).

6) Vogel, T. M., Bioaugmentation as a soil bioremediation approach. Curr. Opin. Biotechnol., 7, 311–316
(1996).

7) Wittich, R.-M., Degradation of dioxin-like compounds by microorganisms. Appl. Microbiol.
Biotechnol., 49, 489–499 (1998).

8) Wittich, R.-M., Aerobic degradation by bacteria of
dibenzo-p-dioxin, dibenzofurans, diphenyl ethers and
their halogenated derivatives. In “Biodegradation of
Dioxins and Furans”, ed. Wittich, R.-M., Springer-
Verlag, Heidelberg, pp. 1–28 (1998).

9) Ohshiro, T., and Izumi, Y., Microbial desulfurization
of organic sulfur compounds in petroleum. Biosci.
Biotechnol. Biochem., 63, 1–9 (1999).

10) Bressler, D. C., and Fedorak, P. M., Bacterial
metabolism of fluorene, dibenzofuran, dibenzothiophene, and carbazole. Can. J. Microbiol., 46,
397–409 (2000).

11) Engesser, K. H., Strubel, V., Christglou, K., Fischer,
P., and Rast, H. G., Dioxygenolytic cleavage of aryl
ether bonds: 1,10-dihydro-1,10-dihydroxyfluoren-9-one, a novel arene dihydrodiol as evidence for angular dioxygenation of dibenzofuran. FEMS Microbiol.
Lett., 65, 205–210 (1989).

12) Strubel, V., Rast, H. G., Fietz, W., Knackmuss,
H.-J., and Engesser, K. H., Enrichment of dibenzofuran utilizing bacteria with high co-metabolic
potential towards dibenzodioxin and other anellated
aromatics. FEMS Microbiol. Lett., 58, 233–238
(1989).

13) Strubel, V., Engesser, K. H., Fischer, P., and
Knackmuss, H.-J., 3-(2-Hydroxyphenyl)catechol as
substrate for proximal meta ring cleavage in dibenzofuran degradation by Brevibacterium sp. strain
DPO 1361. J. Bacteriol., 173, 1932–1937 (1991).

14) Trenz, S. P., Engesser, K. H., Fischer, P., and
Knackmuss, H.-J., Degradation of fluorene by
Brevibacterium sp. strain DPO 1361: a novel C-C
bond cleavage mechanism via 1,10-dihydro-1,10-dihydroxyfluorene-9-one. J. Bacteriol., 176, 789–795
(1994).

15) Schmid, A., Rothe, B., Altenbuchner, J., Ludwig,
W., and Engesser, K. H., Characterization of three
distinct extradiol dioxygenases involved in mineralization of dibenzofuran by Terrabacter sp. strain
DPO360. J. Bacteriol., 179, 53–62 (1997).

16) Fortnagel, P., Wittich, R.-M., Harms, H., Schmidt,
S., Francke, S., Sinnwell, V., Wilkes, H., and
Francke, W., New bacterial degradation of the biaryl
ether structure. Naturwissenschaften, 76, 523–524
(1989).

17) Fortnagel, P., Harms, H., Wittich, R.-M., Krohn,
S., Meyer, H., Sinnwell, V., Wilkes, H., and
Francke, W., Metabolism of dibenzofuran by Pseudomonas sp. strain HH69 and the mixed culture
HH27. Appl. Environ. Microbiol., 56, 1148–1156
(1990).

18) Harms, H., Wittich, R.-M., Sinnwell, V., Meyer, H.,
Fortnagel, P., and Francke, W., Transformation of
dibenzo-p-dioxin by Pseudomonas sp. strain HH69.
Appl. Environ. Microbiol., 56, 1157–1159 (1990).

19) Harms, H., Wilkes, H., Sinnwell, V., Wittich,
R.-M., Figge, K., Francke, W., and Fortnagel, P.,
Transformation of 3-chlorodibenzofuran by Pseudomonas sp. strain HH69. FEMS Microbiol. Lett., 81,
25–30 (1991).

20) Harms, H., Wilkes, H., Wittich, R.-M., and
Fortnagel, P., Metabolism of hydroxydibenzofurans,
methoxydibenzofurans, acetoxydibenzofurans, and
nitrodibenzofurans by Sphingomonas sp. strain
HH69. Appl. Environ. Microbiol., 61, 2499–2505
(1995).

21) Schreiner, G., Wiedmann, T., Schimmel, H., and
Ballschmiter, K., Influence of the substitution pattern
on the microbial degradation of mono- to tetrachlorinated dibenzo-p-dioxins and dibenzofurans.
Chemosphere, 34, 1315–1331 (1997).

22) Wittich, R.-M., Wilkes, H., Sinnwell, V., Francke,
W., and Fortnagel, P., Metabolism of dibenzo-p-dioxin by Sphingomonas sp. strain RW1. Appl. Environ. Microbiol., 58, 1005–1010 (1992).

23) Bünz, P. V., Falchetto, R., and Cook, A. M., Purification of two isofunctional hydrolases (EC 3.7.1.8) in
the degradative pathway for dibenzofuran in Sphingomonas sp. strain RW1. Biodegradation, 4, 171–178
(1993).

24) Bünz, P. V., and Cook, A. M., Dibenzofuran 4,4a-dioxygenase from Sphingomonas sp. strain RW1:
angular dioxygenation by three-component enzyme
system. J. Bacteriol., 175, 6467–6475 (1993).

25) Happe, B., Eltis, L. D., Poth, H., Hedderich, R., and Timmis, K. N., Characterization of 2,2′,3-trihydroxybiphenyl dioxygenase, an extradiol dioxygenase from the dibenzofuran- and dibenzo-p-dioxin-degrading bacterium Sphingomonas sp. strain
RW1. J. Bacteriol., 175, 7313–7320 (1993).

26) Bertini, I., Capozzi, F., Dikiy, A., Happe, B., Luchinat, C., and Timmis, K. N., Evidence of histidine coordination to the catalytic ferrous ion in the ring-cleaving 2,2′,3-trihydroxybiphenyl dioxygenase from the dibenzofuran-degrading bacterium Sphingomonas sp. strain RW1. Biochem. Biophys. Res. Commun., 215, 855–860 (1995).

27) Wilkes, H., Wittich, R.-M., Timmis, K. N., Fortnagel, P., and Franke, W., Degradation of chlorinated dibenzofurans and dibenzo-p-dioxins by Sphingomonas sp. strain RW1. Appl. Environ. Microbiol.,

Involvement of Angular Dioxygenation in Dioxin Degradation
2013

28) Megharaj, M., Wittich, R.-M., Blasco, R., Pieper, D. H., and Timmis, K. N., Superior survival and degradation of dibenzo- *p*-dioxin and dibenzofuran in soil by soil-adapted *Sphingomonas* sp. strain RW1. *Appl. Microbiol. Biotechnol.*, **48**, 109–114 (1997).

29) Arfmann, H.-A., Timmis, K. N., and Wittich, R.-M., Mineralization of 4-chlorodibenzofuran by a consortium consisting of *Sphingomonas* sp. strain RW1 and *Burkholderia* sp. strain JWS. *Appl. Environ. Microbiol.*, **63**, 3458–3462 (1997).

30) Armengaud, J., and Timmis, K. N., Molecular characterization of Fdx1, a putidaredoxin-type [2Fe-2S] ferredoxin able to transfer electrons to the dioxin dioxygenase of *Sphingomonas* sp. RW1. *Eur. J. Biochem.*, **247**, 833–842 (1997).

31) Armengaud, J., and Timmis, K. N., The reductase RedA2 of the multi-component dioxin dioxygenase system of *Sphingomonas* sp. RW1 is related to class-I cytochrome *P*~450~-type reductases. *Eur. J. Biochem.*, **253**, 437–444 (1998).

32) Armengaud, J., Happe, B., and Timmis, K. N., Genetic analysis of dioxin dioxygenase of *Sphingomonas* sp. strain RW1: catabolic genes dispersed on the genome. *J. Bacteriol.*, **180**, 3954–3966 (1998).

33) Armengaud, J., Timmis, K. N., and Wittich, R.-M., A functional 4-hydroxysalicylate/hydroquinol degradative pathway gene cluster is linked to the initial dibenzo-*p*-dioxin pathway genes in *Sphingomonas* sp. strain RW1. *J. Bacteriol.*, **181**, 3452–3461 (1999).

34) Halden, R. U., Halden, B. G., and Dwyer, D. F., Removal of dibenzofuran, dibenzo-*p*-dioxin, and 2-chlorodibenzo-*p*-dioxin from soils inoculated with *Sphingomonas* sp. strain RW1. *Appl. Environ. Microbiol.*, **65**, 2246–2249 (1999).

35) Keim, T., Franke, W., Schmidt, S., and Fortnagel, P., Catabolism of 2,7-dichloro- and 2,4,8-trichlorodibenzofuran by *Sphingomonas* sp. strain RW1. *J. Ind. Microbiol. Biotechnol.*, **23**, 359–363 (1999).

36) Armengaud, J., Gaillard, J., and Timmis, K. N., A second [2Fe-2S] ferredoxin from *Sphingomonas* sp. strain RW1 can function as an electron donor for the dioxin dioxygenase. *J. Bacteriol.*, **182**, 2238–2244 (2000).

37) Yabuuchi, E., Yamamoto, H., Terakubo, S., Okamura, N., Naka, T., Fujiwara, N., Kobayashi, K., Kosako, Y., and Hiraishi, H., Proposal of *Sphingomonas wittichii* sp. nov. for strain RW1<sup>T</sup>, known as a dibenzo-*p*-dioxin metabolizer. *Int. J. Syst. Evol. Microbiol.*, **51**, 281–292 (2001).

38) Monna, L., Omori, T., and Kodama, T., Microbial degradation of dibenzofuran, fluorene, and dibenzo-*p*-dioxin by *Staphylococcus auriculans* DBF63. *Appl. Environ. Microbiol.*, **59**, 285–289 (1993).

39) Kasuga, K., Nojiri, H., Yamane, H., Kodama, T., and Omori, T., Cloning and characterization of the genes involved in the degradation of dibenzofuran by *Terrabacter* sp. strain DBF63. *J. Ferment. Bioeng.*, **84**, 387–399 (1997).

40) Kasuga, K., Habe, H., Chung, J.-S., Yoshida, T., Nojiri, H., Yamane, H., and Omori, T., Isolation and characterization of the genes encoding a novel oxygenase component of angular dioxygenase from the Gram-positive dibenzofuran-degrader *Terrabacter* sp. strain DBF63. *Biochem. Biophys. Res. Commun.*, **283**, 195–204 (2001).

41) Habe, H., Chung, J.-S., Lee, J.-H., Kasuga, K., Yoshida, T., Nojiri, H., and Omori, T., Degradation of chlorinated dibenzofurans and dibenzo-*p*-dioxins by two types of bacteria having angular dioxygenase with different features. *Appl. Environ. Microbiol.*, **67**, 3610–3617 (2001).

42) Habe, H., Ide, K., Yotsumoto, M., Tsuji, H., Yoshida, T., Nojiri, H., and Omori, T., Degradation characteristics of a dibenzofuran-degrader *Terrabacter* sp. strain DBF63 toward chlorinated dioxins in soil. *Chemosphere*, **48**, 201–207 (2002).

43) Nojiri, H., Kamakura, M., Urata, M., Tanaka, T., Chung, J.-S., Takemura, T., Yoshida, T., Habe, H., and Omori, T., Dioxin catabolic genes are dispersed on the *Terrabacter* sp. DBF63 genome. *Biochem. Biophys. Res. Commun.*, **296**, 233–240 (2002).

44) Harms, H., and Zehnder, A. J. B., Influence of substrate diffusion on degradation of dibenzofuran and 3-chlorodibenzofuran by attached and suspended bacteria. *Appl. Environ. Microbiol.*, **60**, 2736–2745 (1994).

45) Harms, H., and Zehnder, A. J. B., Bioavailability of sorbed 3-chlorodibenzofuran. *Appl. Environ. Microbiol.*, **61**, 27–33 (1995).

46) Wittich, R.-M., Strömpl, C., Moore, E. R. B., Blasco, R., and Timmis, K. N., Interaction of *Sphingomonas* and *Pseudomonas* strains in the degradation of chlorinated dibenzofurans. *J. Ind. Microbiol. Biotechnol.*, **23**, 353–358 (1999).

47) Selifonov, S. A., Grifoll, M., Gurst, J. E., and Chapman, P. J., Isolation and characterization of (+)-1,1a-dihydroxy-1-hydrofluoren-9-one formed by angular dioxygenation in the bacterial catabolism of fluorene. *Biochem. Biophys. Res. Commun.*, **193**, 67–76 (1993).

48) Grifoll, M., Selifonov, S. A., and Chapman, P. J., Evidence for a novel pathway in the degradation of fluorene by *Pseudomonas* sp. strain F274. *Appl. Environ. Microbiol.*, **60**, 2438–2449 (1994).

49) Grifoll, M., Selifonov, S. A., and Chapman, P. J., Transformation of substituted fluorenes and fluorene analogs by *Pseudomonas* sp. strain F274. *Appl. Environ. Microbiol.*, **61**, 3490–3493 (1995).

50) Casellas, M., Grifoll, M., Sabaté, J., and Solanas, A. M., Isolation and characterization of a 9-fluorenone-degrading bacterial strain and its role in synergistic degradation of fluorene by a consortium. *Can. J. Microbiol.*, **44**, 734–742 (1998).

51) Grosser, R. J., Warshawsky, D., and Vestal, J. R., Indigenous and enhanced mineralization of pyrene, benzo[ *a*]pyrene, and carbazole in soils. *Appl. Environ. Microbiol.*, **57**, 3462–3469 (1991).

52) Schneider, J., Grosser, R. J., Jayasimhulu, K., Xue, W., Kinkle, B., and Warshawsky, D., Biodegradation of carbazole by *Ralstonia* sp. RJGII.123 isolated from a hydrocarbon contaminated soil. *Can. J. Microbiol.*, **46**, 269–277 (2000).

53) Ouchiyama, N., Zhang, Y., Omori, T., and Kodama, T., Biodegradation of carbazole by *Pseudomonas*

spp. CA06 and CA10. *Biosci. Biotechnol. Biochem.*, 57, 455–460 (1993).

54) Kimura, T., Zhang, Y., Kodama, T., and Omori, T., Isolation and characterization of Tn5-induced mutants deficient in carbazole catabolism. *FEMS Microbiol. Lett.*, 135, 65–70 (1996).

55) Sato, S., Ouchiyama, N., Kimura, T., Nojiri, H., Yamane, H., and Omori, T., Cloning of genes involved in carbazole degradation of *Pseudomonas* sp. strain CA10: nucleotide sequences of genes and characterization of meta-cleavage enzymes and hydrolase. *J. Bacteriol.*, 179, 4841–4849 (1997).

56) Sato, S., Nam, J.-W., Kasuga, K., Nojiri, H., Yamane, H., and Omori, T., Identification and characterization of genes encoding carbazole 1,9a-dioxygenase in *Pseudomonas* sp. strain CA10. *J. Bacteriol.*, 179, 4850–4858 (1997).

57) Nojiri, H., Nam, J.-W., Kosaka, M., Morii, K., Takemura, T., Furihata, K., Yamane, H., and Omori, T., Diverse oxygenations catalyzed by carbazole 1,9a-dioxygenase from *Pseudomonas* sp. strain CA10. *J. Bacteriol.*, 181, 3105–3113 (1999).

58) Nojiri, H., Sekiguchi, H., Maeda, K., Urata, M., Nakai, S., Yoshida, T., Habe, H., and Omori, T., Genetic characterization and evolutionary implications of a car gene cluster in the carbazole degrader *Pseudomonas* sp. strain CA10. *J. Bacteriol.*, 183, 3663–3679 (2001).

59) Nam, J.-W., Nojiri, H., Yoshida, T., Habe, H., Yamane, H., and Omori, T., New classification system for oxygenase components involved in ring-hydroxylating oxygenations. *Biosci. Biotechnol. Biochem.*, 65, 254–263 (2001).

60) Habe, H., Ide, K., Yotsumoto, M., Tsuji, H., Hirano, H., Widada, J., Yoshida, T., Nojiri, H., and Omori, T., Preliminary examinations for applying a carbazole-degrader, *Pseudomonas* sp. strain CA10, to dioxin-contaminated soil remediation. *Appl. Microbiol. Biotechnol.*, 56, 788–795 (2001).

61) Widada, J., Nojiri, H., Kasuga, K., Yoshida, T., Habe, H., and Omori, T., Quantification of carbazole 1,9a-dioxygenase gene by real-time competitive PCR combined with coextraction of internal standards. *FEMS Microbiol. Lett.*, 202, 51–57 (2001).

62) Nojiri, H., Maeda, K., Sekiguchi, H., Urata, M., Shintani, M., Yoshida, T., Habe, H., and Omori, T., Organization and transcriptional characterization of catechol degradation genes involved in carbazole degradation by *Pseudomonas resinovorans* strain CA10. *Biosci. Biotechnol. Biochem.*, 66, 897–901 (2002).

63) Hisatsuka, K., and Sato, M., Microbial transformation of carbazole to anthranilic acid by *Pseudomonas stutzeri*. *Biosci. Biotechnol. Biochem.*, 58, 213–214 (1994).

64) Gieg, L. M., Otter, A., and Fedorak, P. M., Carbazole degradation by *Pseudomonas* sp. LD2: metabolic characteristics and the identification of some metabolites. *Environ. Sci. Technol.*, 30, 575–585 (1996).

65) Ouchiyama, N., Miyachi, S., and Omori, T., Cloning and nucleotide sequence of carbazole catabolic genes from *Pseudomonas stutzeri* strain OM1, isolated

from activated sludge. *J. Gen. Appl. Microbiol.*, 44, 57–63 (1998).

66) Kirimura, K., Nakagawa, H., Tsuji, K., Matsuda, K., Kurane, R., and Usami, S., Selective and continuous degradation of carbazole contained in petroleum oil by resting cells of *Sphingomonas* sp. CDH-7. *Biosci. Biotechnol. Biochem.*, 63, 1563–1568 (1999).

67) Parsons, J. R., de Bruijne, J. A., and Weiland, A. R., Biodegradation pathway of 2-chlorodibenz-p-dioxin and 2-chlorodibenzofuran in the biphenyl-utilizing strain JB1. *Chemosphere*, 37, 1915–1922 (1998).

68) Seeger, M., Cámara, B., and Hofer, B., Dehalogenation, denitrification, dehydroxylation, and angular attack on substituted biphenyls and related compounds by a biphenyl dioxygenase. *J. Bacteriol.*, 183, 3548–3555 (2001).

69) Suenaga, H., Goto, M., and Furukawa, K., Emergence of multifunctional oxygenase activities by random priming recombination. *J. Biol. Chem.*, 276, 22500–22506 (2001).

70) Pfeifer, F., Trüper, H. G., Klein, J., and Schacht, S., Degradation of diphenylether by *Pseudomonas cepacia* Et4: enzymatic release of phenol from 2,3-dihydroxydiphenylether. *Arch. Microbiol.*, 159, 323–329 (1993).

71) Kieth, L. H., and Teillard, W. A., Priority pollutants I—a perspective view. *Environ. Sci. Technol.*, 13, 416–423 (1979).

72) Santodonato, J., and Howard, P. H., Azaarenes: sources, distribution, environmental impact, and health effects. In “Hazard Assessment of Chemicals: Current Developments”, eds. Saxena, J., and Fisher, F., Academic Press, New York, pp. 421–440 (1981).

73) Arcos, J. C., and Argus, M. F., Molecular geometry and carcinogenic activity of aromatic compounds. New perspectives. *Adv. Cancer Res.*, 11, 305–471 (1968).

74) Batie, C. J., Ballow, D. P., and Correll, C. C., Phthalate dioxygenase reductase and related flavin-iron-sulfur containing electron transferases. In “Chemistry and Biochemistry of Flavoenzymes”, ed. Muller, F., CRC Press, Boca Raton, pp. 543–556 (1991).

75) Munro, A. W., and Lindsay, J. G., Bacterial cytochrome P-450. *Mol. Microbiol.*, 20, 1115–1125 (1996).

76) Peterson, J. A., Lorence, M. C., and Amarneh, B., Putidaredoxin reductase and putidaredoxin. Cloning, sequence determination and heterologous expression of the proteins. *J. Biol. Chem.*, 265, 6066–6073 (1990).

77) Kauppi, B., Lee, K., Carredano, E., Parales, R. E., Gibson, D. T., Eklund, H., and Ramaswamy, S., Structure of an aromatic ring-hydroxylating dioxygenase - naphthalene 1,2-dioxygenase. *Structure*, 6, 571–586 (1998).

78) Rieske, J. S., MacLennan, D. H., and Coleman, R., Isolation and properties of an iron-protein from the (reduced coenzyme Q)-cytochrome c reductase complex of the respiratory chain. *Biochem. Biophys. Res. Commun.*, 15, 338–344 (1964).

79) Otaka, E., and Ooi, T., Examination of protein se-

sequence homologies: V. New perspectives on evolution between bacterial and chloroplast-type ferredoxins inferred from sequence evidence. *J. Mol. Evol.*, 29, 246–254 (1989).

80) Mason, J. R., and Cammack, R., The electron-transport proteins of hydroxylating bacterial dioxygenases. *Annu. Rev. Microbiol.*, 46, 277–305 (1992).

81) Schmidt, C. L., and Shaw, L., A comprehensive phylogenetic analysis of Rieske and Rieske-type iron-sulfur proteins. *J. Bioenerg. Biomembr.*, 33, 9–26 (2001).

82) Bosch, R., García-Valdés, E., and Moore, E. R. B., Genetic characterization and evolutionary implications of a chromosomally encoded naphthalene-degradation upper pathway from *Pseudomonas stutzeri* AN10. *Gene*, 236, 149–157 (1999).

83) Murata, T., Ohnishi, M., Ara, T., Kaneko, J., Han, C.-G., Li, Y. F., Takashima, K., Nojima, H., Nakayama, K., Kaji, A., Kamio, Y., Miki, T., Mori, H., Ohtsubo, E., Terawaki, Y., and Hayashi, T., Complete nucleotide sequence of plasmid Rts1: implications for evolution of large plasmid genomes. *J.

84) Sherburne, C. K., Lawley, T. D., Gilmour, M. W., Blattner, F. R., Burland, V., Grotbeck, E., Rose, D. J., and Taylor, D. E., The complete DNA sequence and analysis of R27, a large IncHI plasmid from *Salmonella typhi* that is temperature sensitive for transfer. *Nucleic Acids Res.*, 28, 2177–2186 (2000).

85) Tsuda, M., and Iino, T., Genetic analysis of a transposon carrying toluene degrading genes on a TOL plasmid pWW0. *Mol. Gen. Genet.*, 210, 270–276 (1987).

86) Sander, P., Wittich, R.-M., Fortnagel, P., Wilkes, H., and Francke, W., Degradation of 1,2,4-trichloro-and 1,2,4,5-tetrachlorobenzene by *Pseudomonas* strains. *Appl. Environ. Microbiol.*, 57, 1430–1440 (1991).

87) Carredano, E., Karlsson, A., Kauppi, B., Choudhury, D., Parales, R. E., Parales, J. V., Lee, K., Gibson, D. T., Eklund, H., and Ramaswamy, S., Substrate binding site of naphthalene 1,2-diox-ygenase: functional implications of indole binding. *J. Mol. Biol.*, 296, 701–712 (2000).
