
JOURNAL OF BACTERIOLOGY, Jan. 2003, p. 41–50
0021-9193/03/$08.00+0 DOI: 10.1128/JB.185.1.41–50.2003
Copyright © 2003, American Society for Microbiology. All Rights Reserved.

**Characterization of the 4-Carboxy-4-Hydroxy-2-Oxoadipate Aldolase Gene and Operon Structure of the Protocatechuate 4,5-Cleavage Pathway Genes in *Sphingomonas paucimobilis* SYK-6**

Hirofumi Hara,$^{1}$ Eiji Masai,$^{1\ast}$ Keisuke Miyauchi,$^{1}$ Yoshihiro Katayama,$^{2}$ and Masao Fukuda$^{1}$

*Department of Bioengineering, Nagaoka University of Technology, Nagaoka, Niigata 940-2188,$^{1}$ and Graduate School of Bio-Applications and Systems Engineering, Tokyo University of Agriculture and Technology, Koganei, Tokyo 184-8588,$^{2}$ Japan*

Received 29 July 2002/Accepted 4 October 2002

The protocatechuate (PCA) 4,5-cleavage pathway is the essential metabolic route for degradation of low-molecular-weight products derived from lignin by *Sphingomonas paucimobilis* SYK-6. In the 10.5-kb EcoRI fragment carrying the genes for PCA 4,5-dioxygenase (*ligAB*), 2-pyrone-4,6-dicarboxylate hydrolase (*ligI*), 4-oxalomesaconate hydratase (*ligJ*), and a part of 4-carboxy-2-hydroxymuconate-6-semialdehyde dehydrogenase (*ligC*), we found the *ligK* gene, which encodes 4-carboxy-4-hydroxy-2-oxoadipate (CHA) aldolase. The *ligK* gene was located 1,183 bp upstream of *ligI* and transcribed in the same direction as *ligI*. We also found the *ligR* gene encoding a LysR-type transcriptional activator, which was located 174 bp upstream of *ligK*. The *ligK* gene consists of a 684-bp open reading frame encoding a polypeptide with a molecular mass of 24,131 Da. The deduced amino acid sequence of *ligK* showed 57 to 88% identity with those of the corresponding genes recently reported in *Sphingomonas* sp. strain LB126, *Comamonas testosteroni* BR6020, *Arthrobacter keyseri* 12B, and *Pseudomonas ochraceae* NGJ1. The *ligK* gene was expressed in *Escherichia coli*, and the gene product (LigK) was purified to near homogeneity. Electrospray-ionization mass spectrometry indicated that LigK catalyzes not only the conversion of CHA to pyruvate and oxaloacetate but also that of oxaloacetate to pyruvate and CO₂. LigK is a hexamer, and its isoelectric point is 5.1. The *Kₘ* for CHA and oxaloacetate are 11.2 and 136 μM, respectively. Inactivation of *ligK* in *S.* *paucimobilis* SYK-6 resulted in the growth deficiency of vanillate and syringate, indicating that *ligK* encodes the essential CHA aldolase for catabolism of these compounds. Reverse transcription-PCR analysis revealed that the PCA 4,5-cleavage pathway genes of *S.* *paucimobilis* SYK-6 consisted of four transcriptional units, including the *ligK-orf1-ligI-lsdA* cluster, the *ligJAB* cluster, and the monocistronic *ligR* and *ligC* genes.

Protocatechuate (PCA) is one of the most important intermediate metabolites in the bacterial degradation pathways for various aromatic compounds, including low-molecular-weight products derived from lignin. *Sphingomonas paucimobilis* SYK-6 is able to degrade a wide variety of dimeric lignin compounds, including β-aryl ether (24, 25), biphenyl (33, 34), pinoresinol, phenylcoumarane, and diarylpropane. In *S.* *paucimobilis* SYK-6, dimeric lignin compounds with guaiacyl (4-hydroxy-3-methoxyphenyl) and syringyl (4-hydroxy-3,5-dimethoxyphenyl) moieties are thought to be converted to vanillate and syringate, respectively (26). Vanillate and syringate are converted into PCA and 3-O-methylgallate (3MGA), respectively. It is known that the aromatic ring opening of PCA is catalyzed by one of the three dioxygenase species: PCA 3,4-dioxygenase (3,4-PCD) (4, 9, 12, 51), 4,5-PCD (31, 44), and 2,3-PCD (49). The 3,4-PCD is the most extensively characterized enzyme, and the metabolic pathway for the 3,4-PCD product, β-carboxy-cis,cis-muconate into succinyl coenzyme A and acetyl coenzyme A (the β-ketoadipate pathway) has been well characterized (7, 13, 14, 32). On the other hand, the PCA 4,5- and PCA 2,3-cleavage pathways are poorly understood.

In the case of *S.* *paucimobilis* SYK-6, PCA is degraded via the PCA 4,5-cleavage pathway (Fig. 1). This pathway was enzymatically characterized by Kersten et al. (16) and Maruyama and coworkers (18–22). In this pathway, PCA is initially transformed to 4-carboxy-2-hydroxymuconate-6-semialdehyde (CHMS) by 4,5-PCD (LigAB). CHMS is nonenzymatically converted to an intramolecular hemiacetal form and then oxidized by CHMS dehydrogenase. The resulting intermediate, 2-pyrone-4,6-dicarboxylate (PDC), is hydrolyzed by PDC hydrolase to yield the keto form and enol form of 4-oxalomesaconate (OMA), which are in equilibrium. OMA is converted to 4-carboxy-4-hydroxy-2-oxoadipate (CHA) by OMA hydratase. Finally, CHA is cleaved by CHA aldolase to produce pyruvate and oxaloacetate. Recently, we have identified and characterized all of the gene products and genes except the CHA aldolase gene in the SYK-6 PCA 4,5-cleavage pathway (11, 27, 28, 31, 44). These genes are essential to PCA degradation, while the 3MGA degradation is suggested to go through both the PCA 4,5-cleavage pathway and an alternative ring-cleavage pathway. In this alternative pathway, 3MGA was finally converted to OMA and then entered the PCA 4,5-cleavage pathway. Thus, the PCA 4,5-cleavage pathway genes play a key role in PCA and 3MGA degradation (11).

Recently, cloning of the PCA 4,5-cleavage pathway genes has been reported in *Sphingomonas* sp. strain LB126 (48), *Arthrobacter keyseri* 12B (6), *Comamonas testosteroni* BR6020

* Corresponding author. Mailing address: Department of Bioengineering, Nagaoka University of Technology, Kamitomioka, Nagaoka, Niigata 940-2188, Japan. Phone: 81-258-47-9428. Fax: 81-258-47-9450. E-mail: emasai@vos.nagaokaut.ac.jp.

(A)

(B)

FIG. 1. Catabolic pathway of vanillate and syringate by *S. paucimobilis* SYK-6 (A) and organization of the PCA 4,5-cleavage pathway genes (B). (A) LigA and LigB, the small and large subunits of 4,5-PCD (31, 44); LigH, an essential gene product for vanillate and syringate O demethylations (30); LigC, CHMS dehydrogenase (27); LigI, PDC hydrolase (28); LigJ, OMA hydratase (11); LigK, CHA aldolase/oxaloacetate (OA) decarboxylase (in this study). The degradation pathway for syringate indicated by a dashed line was suggested in our previous study (11, 27, 28). (B) *orf1*, *orf2*, *ligI*, *ligJ*, *ligK*, *ligA*, *ligB*, *ligC*, *ligR*, and lignostilbene α,β-dioxygenase homolog (*lsdA*) genes are demonstrated by the filled arrows. Vertical bars above the restriction map indicate the positions of the Km<sup>r</sup> gene insertion of *orf1* mutant (DF1), *ligK* mutant (DLK), *ligR* mutant (DLR), and *orf2* mutant (DF2). Double-headed arrows indicate locations of amplified RT-PCR products shown in Fig. 5. Abbreviations for restriction enzymes: C, ClaI; E, EcoRI; Ec, Eco47III; P, PstI; Pu, PpuMI; Sh, SphI; SI, SalI; Sm, SmaI; X, XhoI; Xb, XbaI.

TABLE 1. Strains and plasmids used in this study

| Strain or plasmid | Relevant characteristic(s) $^{a}$ | Reference or source |
|------------------|---------------------------------|---------------------|
| **Strains**      |                                 |                     |
| S. paucimobilis   |                                 |                     |
| SYK-6            | Wild type; Nal<sup>r</sup> Sm<sup>r</sup> | 15                  |
| DLK              | Mutant derivative of SYK-6; Km<sup>r</sup> gene insertion mutant of *ligK*; Nal<sup>r</sup> Sm<sup>r</sup> Km<sup>r</sup> | This study          |
| DLR              | Mutant derivative of SYK-6; Km<sup>r</sup> gene insertion mutant of *ligR*; Nal<sup>r</sup> Sm<sup>r</sup> Km<sup>r</sup> | This study          |
| DF1              | Mutant derivative of SYK-6; Km<sup>r</sup> gene insertion mutant of *orf1*; Nal<sup>r</sup> Sm<sup>r</sup> Km<sup>r</sup> | This study          |
| DF2              | Mutant derivative of SYK-6; Km<sup>r</sup> gene insertion mutant of *orf2*; Nal<sup>r</sup> Sm<sup>r</sup> Km<sup>r</sup> | This study          |
| E. coli          |                                 |                     |
| JM109            | *recA1 supE44 endA1 hsdR17 gyrA96 relA1 thi Δ(lac-proAB) F'[traD36 proAB<sup>+</sup> lacI<sup>q</sup> lacZΔM15]* | 50                  |
| BL21(DE3)        | *hsdS gal(λcIts857 ind1 Sam7 nin5 lacUV5-T7 geneI)* | 43                  |
| **Plasmids**     |                                 |                     |
| pUC18 and pUC19  | Cloning vectors; Ap<sup>r</sup> | 50                  |
| pBluescript II SK(+) | Cloning vector; Ap<sup>r</sup> | 42                  |
| pT7 Blue         | Cloning vector; Ap<sup>r</sup> T7 promoter | Novagen             |
| pET21(+)         | Expression vector; Ap<sup>r</sup> T7 promoter | Novagen             |
| pTS1210          | Broad-host-range vector, pSa *ori* pBR *ori*, Km<sup>r</sup> Ap<sup>r</sup> | T. Nakazawa         |
| pUC4K            | Ap<sup>r</sup> Km<sup>r</sup> | 47                  |
| pK19mobsacB      | *oriT sacB* Km<sup>r</sup> | 40                  |
| pHN139F          | pUC18 with a 10.5-kb EcoRI fragment carrying *ligI, ligJAB*, and a part of *ligC* | 28                  |
| pHN139R          | pUC18 carrying the same fragment as pHN139F in opposite direction | 28                  |
| pETK             | pET21(+) with a 1.0-kb *SphI-SalI* fragment carrying *ligK* | This study          |
| pXS4             | pBluescript II SK(+) with a 4.0-kb *XhoI-SmaI* fragment carrying *ligK* and *orf1* | This study          |
| pXS4K            | pXS4 with an insertion of Km<sup>r</sup> gene of pUC4K into PpuMI site | This study          |
| pLKD             | pK19mobsacB with a 5.2-kb *BamHI-KpnI* fragment of pXS4K | This study          |
| pXS4K2           | pXS4 with an insertion of Km<sup>r</sup> gene of pUC4K replacing a 300-bp *SalI* fragment | This study          |
| pF1D             | pK19mobsacB with a 4.9-kb *BamHI-KpnI* fragment of pXS4K2 | This study          |
| pCS18            | pUC19 with a 1.8-kb *ClaI-SmaI* fragment carrying *ligR* | This study          |
| pCS18K           | pCS18 with an insertion of Km<sup>r</sup> gene of pUC4K into *Eco47III* site | This study          |
| pLRD             | pK19mobsacB with a 3.0-kb *KpnI-SacI* fragment of pCS18K | This study          |
| pPS17            | pUC19 with a 1.7-kb *PstI* fragment carrying *orf2* | This study          |
| pPS17K           | pPS17 with an insertion of Km<sup>r</sup> gene of pUC4K into *SmaI* site | This study          |
| pF2D             | pK19mobsacB with a 2.9-kb *BamHI-KpnI* fragment of pPS17K | This study          |
| pUCF1            | pUC19 with a 2.0-kb *XhoI-PpuMI* fragment carrying *orf1* | This study          |
| pTSF1            | pTS1210 with a 2.4-kb *PvuII* fragment of pUCF1 carrying the *lac* promoter and *orf1* | This study          |

$^{a}$ Abbreviations: Nal<sup>r</sup>, Sm<sup>r</sup>, Km<sup>r</sup>, and Ap<sup>r</sup>, resistance to nalidixic acid, streptomycin, kanamycin, and ampicillin, respectively.

(i) Preparation of cell extract. *Escherichia coli* BL21(DE3) harboring pETK was grown in 100 ml of LB medium containing 100 mg of ampicillin/liter. Expression of *ligK* was induced for 4 h at 37°C by the addition of isopropyl-β-d-thiogalactopyranoside (final concentration, 1 mM) when the turbidity of the culture at 660 nm reached 0.5. Cells were harvested by centrifugation and resuspended in 20 mM Tris-HCl buffer (pH 8.0) (buffer A). The cells were broken by two passages through a French pressure cell. The cell lysate was centrifuged at 15,000 × g for 15 min. Streptomycin (final concentration, 1% [wt/vol]) was added to the supernatant, which was centrifuged again at 15,000 × g for 15 min to remove nucleic acids. The supernatant was recovered and then centrifuged again at 170,000 × g for 60 min at 4°C. The crude extract was obtained after concentration by ultrafiltration using a minicon B15 (Amicon, Beverly, Mass.).

(ii) POROS PI anion-exchange chromatography. The crude extract was applied to a POROS polyethyleneimine (PI) column (7.5 by 100 mm) (PerSeptive Biosystems) previously equilibrated with buffer A. The enzyme was eluted with 88 ml of linear gradient of 0 to 0.5 M NaCl. The CHA aldolase was eluted at approximately 0.20 M.

(iii) POROS HQ anion-exchange chromatography. The fractions containing CHA aldolase activity eluted from a PI column were pooled, desalted, and concentrated by ultrafiltration using a minicon B15. The resulting solution was applied to a POROS quaternized polyethyleneimine (HQ) column (4.6 by 100 mm; PerSeptive Biosystems) previously equilibrated with buffer A. The enzyme was eluted with 33 ml of a linear gradient of 0 to 0.5 M NaCl. The fractions containing CHA aldolase activity that eluted at approximately 0.30 M were pooled.

(iv) POROS PE hydrophobic-interaction chromatography. The fractions containing CHA aldolase activity eluted from an HQ column were pooled, desalted, and concentrated. Ammonium sulfate was added to the enzyme solution to a final concentration of 2 M. After centrifugation at 15,000 × g for 10 min, the supernatant was recovered and applied to a POROS phenylether (PE) column (4.6 by 100 mm) (PerSeptive Biosystems) equilibrated with buffer B (buffer A containing 2 M ammonium sulfate). The enzyme was eluted with 25 ml of a linear gradient of 2.0 to 0 M ammonium sulfate. The fractions containing CHA aldolase activity that eluted at approximately 1.3 M were pooled, desalted, and concentrated as described above. Glycerol was added to a final concentration of 10%, and the purified enzyme was stored at −80°C until use.

Analytical method. The protein concentration was determined by the method of Bradford (2). The purity of the enzyme preparation was examined by sodium dodecyl sulfate–15% polyacrylamide gel electrophoresis (SDS–15% PAGE) (17). The molecular mass of the native enzyme was estimated by Superdex 200 HR10/30 (Pharmacia Biotech) gel filtration column chromatography using a BioCAD700E apparatus. Elution was performed with 50 mM potassium phosphate buffer (pH 7.0) containing 0.15 M NaCl at a flow rate of 0.8 ml/min. The molecular weight was estimated on the basis of calibration curve of reference proteins.

To determine the N-terminal amino acid sequence, the cell extract of *E. coli* BL21(DE3) harboring pETK was subjected to SDS–15% PAGE and electroblotted onto a polyvinylidene difluoride membrane (Bio-Rad, Hercules, Calif.). The area at 27 kDa was cut out and analyzed on a PPSQ-21 protein sequencer (SHIMADZU, Kyoto, Japan). The isoelectric point of LigK was determined by isoelectric focusing on an Ampholine PAG plate (pH 3.5 to 9.5; Pharmacia Biotech) using a model Multiphor II electrophoresis system (Pharmacia Biotech).

The substrate and the reaction products were detected and identified by gas chromatography (GC)-MS using model 5971A with an Ultra-2 capillary column (50 m by 0.2 mm; Agilent technologies, Palo Alto, Calif.) and ESI-MS using HP1100 series LC-MSD (Agilent technologies). The analytical conditions for GC-MS were the same as described previously (28). In ESI-MS analysis, mass spectra were obtained by negative-mode ESI, with a needle voltage of −3.5 kV and a source temperature at 350°C. The sample was injected directly into the mass spectrometer; the water/methanol ratio was 90:10 (vol/vol), and the flow rate was 0.2 ml/min.

Identification of the reaction product. 200 μM CHA was incubated with purified LigK (0.5 μg) in 0.1 M Tris-acetate buffer (pH 8.0) containing 1 mM MgCl₂ for 1 min or 5 min, the reaction mixture was diluted to 1/10 with 10 mM Tris-acetate buffer (pH 8.0), and the portion of mixture (5 μl) was injected into the ESI-mass spectrometer.

In the case of GC-MS analysis, the reaction product was acidified and extracted with ethylacetate, and then the extract was trimethylsilylated. The resultant trimethylsilylated derivatives were analyzed.

The metabolites of vanillate and syringate by the *ligK* insertion mutant (DLK) were analyzed. DLK cells grown in 10 ml of LB medium were washed with 0.1 M Tris-acetate buffer (pH 8.0). The cells were resuspended in the same buffer and incubated with 10 mM vanillate and 10 mM syringate for 12 h at 30°C. After centrifugation, the supernatant was diluted 20-fold with 10 mM Tris-acetate

buffer (pH 8.0) and analyzed by ESI-MS as described above. On the other hand, the metabolites were extracted by ethylacetate, trimethylsilylated, and analyzed by GC-MS.

Disruption of *orfI*, *ligK*, *ligR*, and *orf2*. The 4.0-kb *XhoI-SmaI* fragment carrying *ligK* and *orfI* was cloned into pBluescript II SK(+) to generate pXS4, and it was digested with *PpuMI* for *ligK* disruption or with *SalI* for *orfI* disruption. The 1.2-kb *PstI* fragment containing the kanamycin resistance gene from pUC4K (47) was inserted into the *PpuMI* or *SalI* site of the 4.0-kb *XhoI-SmaI* fragment to construct pXS4K and pXS4K2, respectively. pXS4K and pXS4K2 were digested with *BamHI* and *KpnI*, and their inserts were cloned into pK19mobsacB (40) to generate pLKD and pF1D, respectively. The 1.8-kb *ClaI-SmaI* fragment carrying *ligR* was cloned into pUC19 to generate pCS18, and it was digested with *Eco47III*. The kanamycin resistance gene was inserted into this *Eco47III* site. The resultant plasmid, pCS18K, was digested with *KpnI* and *SacI*, and the insert containing the inactivated *ligR* gene was cloned into pK19mobSacB to generate pLRD. The 1.7-kb *PstI* fragment carrying *orf2* was cloned into pUC19 to generate pPS17, and it was digested with *SmaI*. The kanamycin resistance gene was inserted into the *SmaI* site. The resultant plasmid, pPS17K, was digested with *BamHI* and *KpnI*, and the insert containing the inactivated *orf2* gene was cloned into pK19mobsacB to generate pF2D.

Each of plasmids, pLKD, pF1D, pLRD, and pF2D was introduced into SYK-6 cells by electroporation, and the candidates for mutants were isolated as described previously (28). To examine the disruption of each gene, Southern hybridization analysis was carried out. The total DNA of the candidates for *ligK*, *ligR*, and *orf2* mutants were digested with *PstI*, and those for *orfI* were digested with *SmaI*. The 1.2-kb *PstI* fragment carrying the kanamycin resistance gene, the 2.3-kb *PstI* fragment carrying *ligR* and *ligK*, the 4.0-kb *XhoI-SmaI* fragment carrying *orfI*, and the 1.7-kb *PstI* fragment carrying *orf2* were labeled with the DIG system (Roche Diagnostics, Indianapolis, Ind.) and used as probes.

Reverse transcription (RT)-PCR. Cells of *S. paucimobilis* SYK-6 were grown in W minimal salt medium containing 10 mM vanillate until they reached the turbidity at 660 nm of 0.5. Total RNA was prepared from 10 ml of culture by using RNeasy Mini columns (Qiagen Inc, Chatsworth, Calif.). To remove any contaminating genomic DNA, the RNA samples were incubated with 1 U of RNase-free DNase (Takara Shuzo Co., Ltd.) in 40 mM Tris-HCl (pH 7.9) containing 1 U of RNase inhibitor (Takara Shuzo Co., Ltd.), 10 mM NaCl, 10 mM CaCl₂, and 6 mM MgSO₄ for 30 min at 37°C. RT-PCR was carried out with a *BcaBEST* RNA PCR kit (Takara Shuzo Co., Ltd.). A cDNA library was obtained by an RT reaction using a hexanucleotide random priming mix. The cDNA was used as a template for subsequent PCRs with specific primers, which amplify the boundaries of *ligK-orfI-ligI-lsdA* and *ligR-orf2-ligJ-ligA-ligB-ligC*. The forward and reverse primers used were as follows: *lsdA*-forward (nucleotide positions from 1,363 to 1,383 in the 10.5-kb *EcoRI* fragment) and *ligI*-reverse (positions 1,924 to 1,944); *ligI*-forward (positions 2,528 to 2,548) and *orfI*-reverse (positions 2,999 to 3,019); *orfI*-forward (positions 3,489 to 3,509) and *ligK*-reverse (positions 3,740 to 3,760); internal *ligR*-forward (positions 4,613 to 4,533) and internal *ligR*-reverse (positions 5,215 to 5,235); *ligR*-forward (positions 5,770 to 5,790) and *orf2*-reverse (positions 6,476 to 6,496); internal *orf2*-forward (positions 5,843 to 5,863) and *orf2*-reverse; *orf2*-forward (positions 6,536 to 6,556) and *ligJ*-reverse (positions 6,943 to 6,963); *ligJ*-forward (positions 7,662 to 7,682) and *ligA*-reverse (positions 8,162 to 8,182) and *ligB*-reverse (positions 8,706 to 8,726); *ligB*-forward (positions 9,119 to 9,139) and *ligC*-reverse (positions 9,598 to 9,608); internal *ligC*-forward (positions 9,609 to 9,629) and internal *ligC*-reverse (positions 10,098 to 10,118). Control samples in which reverse transcriptase was omitted in RT-PCR and in which genomic DNA was used as a template in PCRs were run in parallel with RT-PCRs.

Nucleotide sequence accession number. The nucleotide sequence reported in this paper was deposited in the DDBJ, EMBL, and GenBank nucleotide sequence databases under accession no. AB073227.

# RESULTS AND DISCUSSION

Nucleotide sequence analysis. We detected the CHA aldolase activity in *E. coli* JM109 harboring pHN139F, which contained the 10.5-kb *EcoRI* fragment carrying *ligAB* (31), *ligI* (28), *ligJ* (11), and a part of *ligC* (27) (Fig. 1B). In the deletion analysis, the DNA region that conferred CHA aldolase activity to *E. coli* was limited to the 1.0-kb *SalI-SphI* fragment. The nucleotide sequences of the 5.0-kb *SmaI* fragment and the overlapping 1.7-kb *PstI* fragment were determined, and an

VOL. 185, 2003 ALDOLASE FOR PROTOCATECHUATE 4,5-CLEAVAGE PATHWAY 45

(A)

(B)

(C)

(D)

FIG. 2. Identification of the reaction product from CHA catalyzed by LigK. CHA (200 μM) was incubated with 0.5 μg of LigK in 0.1 M Tris-acetate buffer (pH 8.0) containing 1 mM MgCl₂. (A) Negative-ion ESI-MS spectrum of the reaction mixture without CHA. (B to D) Negative-ion ESI-MS spectrum of the reaction mixture at 0, 1, and 5 min of incubation with LigK, respectively.

ORF of 684 bp was revealed in the 1.0-kb SalI-SphI fragment. This ORF encodes 228 amino acid residues with a molecular mass of 24,131 Da and was designated *ligK*. The deduced amino acid sequence of *ligK* shared the highest degree of identity (88%) with that of *fldZ*, which is located in the putative PCA 4,5-cleavage pathway genes of the fluorene degrader Sphingomonas sp. strain LB126 (48). The deduced amino acid sequence of *ligK* also showed 66, 66, and 57% identity with those of the CHA aldolase genes recently identified in *C. testosteroni* BR6020 (36), *P. ochraceae* NGJ1 (23), and *A. keyseri* 12B (6), respectively. Three ORFs were also found adjacent to *ligK* (Fig. 1B). *orf1* was located downstream of *ligK* with a direction of transcription identical to that of *ligK*. On the other hand, *ligR* and *orf2* were located upstream of *ligK* with a transcription orientation opposite that of *ligK*. The *orf1* product showed 78 and 44% identity to FldA of LB126 and a putative transmembrane protein of YbhH in *E. coli* (GenBank accession no. D90715), whose functions are unknown. This similarity may suggest that *orf1* encodes a transporter for substrates such as PCA, vanillate, and/or syringate. *LigR* has 28% identity with various LysR-type transcriptional regulators, including PcaQ of α proteobacterium Y3F (3), and 22% identity with SdsB, which positively regulates alkyl sulfatase (SdsA) of *Pseudomonas* sp. strain ATCC 19151 (5). *LigR* might be involved in the transcriptional control of *ligK-orf1-ligI-lsdA*, because LysR-type transcriptional regulators generally control the divergently transcribed operon (41). We are unable to conjecture about the function of *orf2*, although its deduced amino acid sequence showed 50% identity with FldX from *Sphingomonas* sp. strain LB126 (48) and 39% identity with a conserved hypothetical protein from *Sinorhizobium meliloti* (8), whose functions are unknown.

The PCA 4,5-cleavage pathway genes of *S.* paucimobilis SYK-6 consisted of the two divergently transcribed gene clusters of *ligK-orf1-ligI-lsdA* and *ligR-orf2-ligJ-ligA-ligB-ligC* (Fig. 1B). Recently, all or partial sequences of the PCA 4,5-cleavage pathway genes have been reported in *Sphingomonas* sp. strain LB126 (48), *A. keyseri* 12B (6), *C. testosteroni* BR6020 (36), and *P. ochraceae* NGJ1 (23). Two types of gene clusters can be clearly seen. Interestingly, the gene organization of the fluorene degrader, *Sphingomonas* sp. strain LB126, is essentially the same as that of SYK-6, although the identities of the corresponding genes between SYK-6 and LB126 vary from 50 to 88%. On the other hand, *C. testosteroni* BR6020 and *A. keyseri* 12B have a packed single gene cluster that seems to be an operon.

Purification of CHA aldolase. The 1.0-kb SalI-SphI fragment carrying *ligK* was cloned in pET21(+) to construct pETK, and *ligK* was expressed in *E. coli* BL21(DE3) under the control of the T7 promoter. Production of the 27-kDa protein was observed by SDS-PAGE (data not shown). The size of the product is close to the molecular mass calculated from the deduced amino acid sequence of *ligK*. *LigK* was purified from a cell extract of *E. coli* BL21(DE3) harboring pETK by a series of

(A)

| Abundance (10⁴) |
|------------------|
| 20              |
| 16              |
| 12              |
| 8               |
| 4               |
| 0               |

PDC

(II)
(I)

OMA  
(enol form)

Time (min)
10.00 15.00 20.00 25.00 30.00 35.00

(B)

| Abundance (10⁴) |
|------------------|
| 14              |
| 12              |
| 10              |
| 8               |
| 6               |
| 4               |
| 2               |
| 0               |

PDC

(II)
(I)

OMA  
(enol form)

Time (min)
10.00 15.00 20.00 25.00 30.00 35.00

(C)

| Abundance (10⁴) |
|------------------|
| 4               |
| 3               |
| 2               |
| 1               |
| 0               |

m/z
180 190 200 210 220 230 240

(D)

| Abundance (10⁴) |
|------------------|
| 4               |
| 3               |
| 2               |
| 1               |
| 0               |

m/z
180 190 200 210 220 230 240

FIG. 3. Identification of accumulated products from vanillate and syringate by DLK. (A and B) Gas chromatograms of trimethylsilylated derivatives of the accumulated products from vanillate and syringate, respectively. In both cultures, PDC, the enol form of OMA, product I, and unidentified product II were observed. (C and D) Negative-ion ESI-MS spectrum of the same reaction mixture used in panels A and B, respectively.

column chromatography procedures with PI, HQ, and PE. LigK was purified approximately 52-fold to near homogeneity (>99%) with a recovery of 20%. N-terminal amino acid sequencing revealed that the first 15 residues, with the exception of the first methionine, Arg-Gly-Ala-Ala-Met-Gly-Val-Val-Gln-Asn-Ile-Glu-Arg-Ala, corresponded to the deduced amino acid sequence of *ligK*.

Identification of the reaction product. To identify the reaction product of CHA catalyzed by the purified LigK, the reaction mixture was analyzed by ESI-MS. The fragment at *m/z* 219 in Fig. 2B was estimated to be the deprotonated molecular ion ([M-H]⁻) of CHA (where M is a molecular ion). The other peaks in the spectrum of CHA were originated from components of the reaction buffer containing LigK enzyme (Fig. 2A). After 1 min of reaction, the intensity of the fragment at *m/z* 219 of CHA decreased to 42% of its initial intensity, and the generation of two fragments at *m/z* 87 and at *m/z* 131 corresponding to [M-H]⁻ of pyruvate and [M-H]⁻ of oxaloacetate, respectively, was observed (Fig. 2C). This result indicated that LigK catalyzes the conversion of CHA to pyruvate and oxaloacetate. After 5 min of reaction, the intensity of the fragment at *m/z* 87 increased to 144% of that of the corresponding fragment in the 1-min reaction mixture, whereas the intensity of the fragment at *m/z* 131 decreased to 38% of that of the corresponding fragment in the 1-min reaction mixture (Fig. 2D). These results strongly suggested that oxaloacetate was converted into pyruvate by LigK. The activity for β-decarboxylation of oxaloacetate has been reported in CHA aldolase of *P. ochraceae* (21), 4-hydroxy-4-methyl-2-oxoglutarate aldolase of *P. putida* (45), and 2-keto-4-hydroxyglutarate aldolase of *E. coli* (29). To examine whether LigK has this activity, oxaloacetate was incubated with LigK in the presence of lactate dehydrogenase and NADH. A decrease in absorbance at 340 nm derived from NADH was observed. It is concluded that LigK is able to decarboxylate oxaloacetate to generate pyruvate.

Enzyme properties. In accord with the previous study by Maruyama (21), the CHA aldolase activity was observed only when a divalent cation such as Mg²⁺ was present in the reaction mixture. We examined the effect of the various divalent cations for the enzyme activities of LigK. Addition of 1 mM Co²⁺, Zn²⁺, Ca²⁺, or Mn²⁺ resulted in 85, 65, 20, or 0% of the activity resulting from addition of 1 mM Mg²⁺, respectively. A similar metal dependency was observed in the decarboxylation of oxaloacetate by LigK. When 1 mM EDTA was added to the reaction mixture, both enzyme activities were completely lost in the presence of 1 mM metal ion. Aldolases are categorized as class I or class II based on their metal dependency. LigK was suggested to be one of the class II aldolases, which require the metal ion. Most class II aldolases show a significant rate enhancement in the presence of phosphate ion (37). Addition of 0.5 mM phosphate ion in the LigK reaction mixture caused 3.0- and 1.7-fold activation of CHA aldolase and oxaloacetate decarboxylase activity, respectively.

Gel filtration column chromatography using Superdex200 indicated that the molecular mass of the native LigK was 160 kDa. This result suggested that LigK is a homohexamer. The isoelectric point of LigK was determined to be 5.1 by isoelectric focusing gel electrophoresis. The optimal tem-

temperature of LigK for aldolase activity on CHA, and the decarboxylase activity on oxaloacetate were both determined to be 25°C. The optimal pH for aldolase activity and decarboxylase activity were estimated to be 8.0 and 7.0, respectively. The \( K_m \) for oxaloacetate (136 μM) is 12 times higher than that for CHA (11.2 μM). The \( V_{\text{max}} \) for CHA aldol cleavage (265 U/mg) is 20 times higher than that for oxaloacetate decarboxylation (13.2 U/mg).

We also examined the influence of sulfhydryl reagents on LigK. One microgram of purified LigK was preincubated with 1 mM sulfhydryl reagents for 10 min. HgCl₂ and N-ethylmaleimide inhibited 60 and 62% of the CHA aldolase and 92 and 88% of the oxaloacetate decarboxylase activities, respectively. These results suggested that some cysteine residues might be involved in the enzyme reaction. CHA aldolase activity was inhibited by oxaloacetate with a \( K_i \) value of 23 μM. As suggested by Maruyama (21), the amount of oxaloacetate in the cells might control the production of oxaloacetate from CHA.

CHA aldolase has been biochemically characterized only in *P. ochraceae* (21) and *P. putida* (45). The molecular mass, subunit structure, and pI of LigK are very similar to those of aldolases of *P. ochraceae* and *P. putida*. In the *P. ochraceae* enzyme, the kinetics parameters are measured using the substrates *d*-CHA and *l*-CHA. The \( K_m \) and \( V_{\text{max}} \) values of the *P. ochraceae* enzyme for *l*-CHA were similar to those for LigK. In our experiment, CHA was prepared from OMA by using the purified OMA hydratase from *E. coli* carrying the SYK-6 *ligJ* gene. The physiological substrate for LigK might be an *l*-isomer. The \( K_m \) for oxaloacetate of the *P. ochraceae* enzyme was twofold higher than that of LigK, although the \( V_{\text{max}} \) values of these strains are similar. LigK has a significantly higher affinity for oxaloacetate than did the *P. ochraceae* enzyme.

Disruption of *ligK* in *S. paucimobilis* SYK-6. The *ligK* gene was disrupted to clarify the actual role of *ligK* in the catabolism of vanillate and syringate by SYK-6. Gene inactivation was carried out using the *ligK* disruption plasmid, pLKD. The *ligK* insertional mutation was confirmed by Southern hybridization analysis using the 2.3-kb *PstI* fragment carrying *ligK* and the 1.2-kb *PstI* fragment carrying the kanamycin resistance gene as probes (data not shown). The *ligK* and kanamycin resistance gene probes revealed that the *ligK* gene was inactivated by homologous recombination through the double crossover. This mutant strain was designated DLK and used for the following experiments. The obtained mutant strain DLK completely lost the ability to grow on both vanillate and syringate. This result is compatible with the deduced catabolic pathways of vanillate and syringate by SYK-6 shown in Fig. 1A.

To determine the accumulated products from vanillate and syringate incubated with DLK, 10 mM concentrations of vanillate and syringate were independently incubated with the LB-grown whole cells of DLK in the W minimal medium, and the metabolites were identified by GC-MS and ESI-MS. As shown in the gas chromatogram (Fig. 3A and B), vanillate and syringate detected with retention times of 21.2 and 25.2 min, respectively, disappeared completely, and the accumulation of PDC, the enol form of OMA, product I with a retention time of 30.5 min, and product II with a retention time of 26.1 min was observed in both cultures. In a previous study, we identified product I as the compound generated from OMA by addition of two atoms of hydrogen by NADPH-dependent

(A) vanillate

![Graph A](#)

(B) syringate

![Graph B](#)

FIG. 4. Growth of SYK-6 and *ligR* insertion mutant (DLR) on vanillate and syringate. Growth of SYK-6 (open circles) and DLR (filled circles) on 10 mM vanillate (A) and 10 mM syringate (B). The results are the means of the representative of three independent experiments.

reductase in the *ligJ* insertion mutant of SYK-6 (11). The mass spectrum of product II accumulated in both cultures are identical but could not be assigned (data not shown). On the other hand, ESI-MS analysis indicated accumulation of the products whose deprotonated molecular ions appeared at *m/z* 203 and at *m/z* 221 in the metabolite from vanillate and syringate (Fig. 3C and D). In this analytical condition, neither PDC nor the enol form of OMA could be detected. We previously suggested that the ion at *m/z* 203 was a deprotonated molecular ion of product I. We therefore estimated that the ion at *m/z* 221 was generated from CHA accumulated by addition of two hydrogen atoms catalyzed by unidentified reductase(s) in DLK. To examine this hypothesis, CHA was incubated with the DLK crude extract prepared from cells grown in LB. ESI-MS of the reaction product after 10 min incubation showed that the peak at *m/z* 219 derived from CHA was converted to that at *m/z* 221 only in the presence of NADPH (data not shown). These

FIG. 5. Agarose gel electrophoresis of RT-PCR products amplified from SYK-6 cells grown on vanillate. The sizes of molecular weight markers in lane M are indicated on the left side of the gels. Odd-numbered lanes are controls without reverse transcriptase. The primers for spanning each ORF intergenic region or ORF internal region are shown in the Materials and Methods. Lanes: 1 and 2, *lsdA-ligI* intergenic region (expected size, 581 bp); 3 and 4, *ligI-orfI* intergenic region (expected size, 491 bp); 5 and 6, *orfI-ligK* intergenic region (expected size, 271 bp); 7 and 8, *ligR-orf2* intergenic region (expected size, 726 bp); 9 and 10, *orf2-ligJ* intergenic region (expected size, 427 bp); 11 and 12, *ligJ-ligA* intergenic region (expected size, 520 bp); 13 and 14, *ligA-ligB* intergenic region (expected size, 564 bp); 15 and 16, *ligB-ligC* intergenic region (expected size, 489 bp); 17 and 18, *ligC* internal region (expected size, 509 bp); 19 and 20, *ligR* internal region (expected size, 622 bp); and 21 and 22, *orf2* internal region (expected size, 653 bp).

results strongly suggested that product II was produced from accumulated CHA. Our preliminary experiment indicated that LigJ activity was not inhibited by the presence of CHA, and thus the reason why a large amount of OMA (product I) and PDC were also accumulated from vanillate and syringate in DLK is unknown.

Disruption of *ligR*, *orfI*, and *orf2* in *S*. *paucimobilis* SYK-6. To investigate whether *ligR*, *orfI*, and *orf2* are involved in the catabolism of vanillate and syringate, each of these genes in SYK-6 was disrupted. Gene inactivation was carried out using the *ligR*, *orfI*, and *orf2* disruption plasmids, pLRD, pF1D, and pF2D, respectively. The growth rates of the *ligR* disruption mutant, DLR, on both vanillate and syringate were decreased compared with those of SYK-6 (Fig. 4). Based on this result and the fact that LigR has similarity with LysR-type transcriptional regulator, LigR may positively regulate the expression of the PCA 4,5-cleavage pathway genes, although it is not essential to growth of SYK-6 on vanillate and syringate. The *orfI* insertion mutant, DF1, completely lost the ability to grow on both vanillate and syringate. This growth deficiency of DF1 on vanillate and syringate was complemented by introduction of pTS1210 carrying *orfI* (pTSF1), indicating that *orfI* is necessary for growth of SYK-6 on these compounds. Considering that *orfI* has similarity with a putative transmembrane protein YbhH of *E*. *coli*, it is likely that *orfI* encodes a transporter of vanillate and syringate. However, the actual role of *orfI* is remained to clarify together with that of *ligR*. On the other hand, the disruption of *orf2* did not affect the growth of SYK-6 on both vanillate and syringate.

RT-PCR analysis of PCA 4,5-cleavage pathway genes. To determine the operon structure of the genes included in the 10.5-kb *EcoRI* fragment, RT-PCR experiments were performed with total RNA isolated from SYK-6 grown on vanillate and primers complementary to neighboring ORFs. The amplification products of *lsdA-ligI* (581 bp), *ligI-orfI* (491 bp), *orfI-ligK* (271 bp), *ligJ-ligA* (520 bp), and *ligA-ligB* (564 bp) were obtained. However, RT-PCR products using primer which span the *ligR-orf2*, *orf2-ligJ*, and *ligB-ligC* regions were not obtained (Fig. 5), while PCR using their primers with SYK-6 total DNA as a template gave the expected PCR products (data not shown). In order to confirm the presence of the *ligR*, *orf2*, and *ligC* transcripts in the RNA samples, RT-PCR was carried out using primers to amplify inside of each ORF. RT-PCR products of *ligR* (622 bp) and *ligC* (509 bp) with the

expected sizes were obtained (Fig. 5). On the other hand, the RT-PCR product of *orf2* did not appear, indicating the DNA region of *orf2* was not transcribed in SYK-6 cells grown on vanillate.

In conclusion, the PCA 4,5-cleavage pathway genes of *S.* *paucimobilis* SYK-6 consist of four transcriptional units, including the *ligK-orf1-ligI-lsdA* cluster, the *ligJAB* cluster, and the monocistronic *ligR* and *ligC* genes (Fig. 1B). In the case of the PCA 3,4-cleavage pathway genes, the diversity in gene organization and transcriptional regulation has been found (7, 10, 13, 14, 32, 35, 46). In *Acinetobacter* sp. strain ADP1, all of the enzyme genes involved in this pathway constitute a single operon, *pcaIJFBDKCHG*, and their expression is regulated by the IclR-type transcriptional activator PcaU in concert with inducer PCA (10, 35, 46). On the other hand, the enzyme genes of *P.* *putida* PRS2000 (13) and *Agrobacterium tumefaciens* A348 (32) consist of several transcriptional units. In A348, the *pcaDCHGB* and *pcaIJ* clusters are independently regulated by the LysR-type transcriptional activator PcaQ, which responds to both β-carboxy-cis,*cis*-muconate and γ-carboxymuconolactone, and the IclR-type transcriptional activator PcaR, which responds to β-ketoadipate, respectively (32). To gain better understanding of the PCA 4,5-cleavage pathway genes, it is essential to address their transcriptional regulation. Determination of the promoter regions and the actual role of *ligR* are currently under way in our laboratory.

### ACKNOWLEDGMENTS

We thank T. Nakazawa for providing pTS1210. This work was supported in part by Grant-in-Aid for Encouragement of Young Scientists 11760057 from the Ministry of Education, Science, Sports, and Culture of Japan to E.M. H.H. was financially supported by research fellowship 2068 from the Japan Society for the Promotion of Science for Young Scientists.

### REFERENCES

1. Ausubel, F. M., R. Brent, R. E. Kingston, D. D. Moore, J. G. Seidman, J. A. Smith, and K. Struhl (ed.). 1990. Current protocols in molecular biology. John Wiley & Sons, Inc., New York, N.Y.
2. Bradford, M. M. 1976. A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal. Biochem. **72**:248–254.
3. Buchan, A., L. S. Collier, E. L. Neidle, and M. A. Moran. 2000. Key aromatic-ring-cleaving enzyme, protocatechuate 3,4-dioxygenase, in the ecologically important marine *Roseobacter* lineage. Appl. Environ. Microbiol. **66**:4662–4672.
4. Contzen, M., and A. Stolz. 2000. Characterization of the genes for two protocatechuate 3,4-dioxygenases from the 4-sulfocatechol-degrading bacterium *Agrobacterium radiobacter* strain S2. J. Bacteriol. **182**:6123–6129.
5. Davison, J., F. Brunel, A. Phanopoulos, D. Prozzi, and P. Terpstra. 1992. Cloning and sequencing of *Pseudomonas* genes determining sodium dodecyl sulfate biodegradation. Gene **114**:19–24.
6. Eaton, R. W. 2001. Plasmid-encoded phthalate catabolic pathway in *Arthrobacter keyseri* 12B. J. Bacteriol. **183**:3689–3703.
7. Eulberg, D., S. Lakner, L. A. Golovleva, and M. Schlömann. 1998. Characterization of a protocatechuate catabolic gene cluster from *Rhodococcus opacus* 1CP: evidence for a merged enzyme with 4-carboxymuconolactone-decarboxylating and 3-oxoadipate enol-lactone-hydrolyzing activity. J. Bacteriol. **180**:1072–1081.
8. Finan, T. M., S. Weidner, K. Wong, J. Buhrmester, P. Chain, F. J. Vorhölter, I. Hernandez-Lucas, A. Becker, A. Cowie, J. Gouzy, B. Golding, and A. Pühler. 2001. The complete sequence of the 1,683-kb pSymB megaplasmid from the N₂-fixing endosymbiont *Sinorhizobium meliloti*. Proc. Natl. Acad. Sci. USA **98**:9889–9894.
9. Frazee, R. W., D. M. Livingston, D. C. LaPorte, and J. D. Lipscomb. 1993. Cloning, sequencing, and expression of the *Pseudomonas putida* protocatechuate 3,4-dioxygenase genes. J. Bacteriol. **175**:6194–6202.
10. Gerischer, U., A. Segura, and L. N. Ornston. 1998. PcaU, a transcriptional activator of genes for protocatechuate utilization in *Acinetobacter*. J. Bacteriol. **180**:1512–1524.
11. Hara, H., E. Masai, Y. Katayama, and M. Fukuda. 2000. The 4-oxalo-mesaconate hydratase gene, involved in the protocatechuate 4,5-cleavage pathway, is essential to vanillate and syringate degradation in *Sphingomonas paucimobilis* SYK-6. J. Bacteriol. **182**:6950–6957.
12. Hartnett, C., E. L. Neidle, K. L. Ngai, and L. N. Ornston. 1990. DNA sequences of genes encoding *Acinetobacter calcoaceticus* protocatechuate 3,4-dioxygenase: evidence indicating shuffling of genes and of DNA sequences within genes during their evolutionary divergence. J. Bacteriol. **172**:956–966.
13. Harwood, C. S., and R. E. Parales. 1996. The β-ketoadipate pathway and the biology of self-identity. Annu. Rev. Microbiol. **50**:553–590.
14. Iwagami, S. G., K. Yang, and J. Davies. 2000. Characterization of the protocatechuic acid catabolic gene cluster from *Streptomyces* sp. strain 2065. Appl. Environ. Microbiol. **66**:1499–1508.
15. Katayama, Y., S. Nishikawa, M. Nakamura, K. Yano, M. Yamasaki, N. Morohoshi, and T. Haraguchi. 1987. Cloning and expression of *Pseudomonas paucimobilis* SYK-6 genes involved in the degradation of vanillate and protocatechuate in *P. putida*. Mokuzai Gakkaishi **33**:77–79.
16. Kersten, P. J., S. Dagley, J. W. Whittaker, D. M. Arciero, and J. D. Lipscomb. 1982. 2-pyrone-4,6-dicarboxylic acid, a catabolite of gallic acids in *Pseudomonas* species. J. Bacteriol. **152**:1154–1162.
17. Laemmli, U. K. 1970. Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature (London) **227**:680–685.
18. Maruyama, K. 1979. Isolation and identification of the reaction product of α-hydroxy-γ-carboxymuconic-ε-semialdehyde dehydrogenase. J. Biochem. **86**:1671–1677.
19. Maruyama, K. 1983. Purification and properties of 2-pyrone-4,6-dicarboxylate hydrolase. J. Biochem. **93**:557–565.
20. Maruyama, K. 1985. Purification and properties of γ-oxalomesaconate hydratase from *Pseudomonas ochraceae* grown with phthalate. Biochem. Biophys. Res. Commun. **128**:271–277.
21. Maruyama, K. 1990. Purification and properties of 4-hydroxy-4-methyl-2-oxoglutarate aldolase from *Pseudomonas ochraceae* grown on phthalate. J. Biochem. **108**:327–333.
22. Maruyama, K., N. Ariga, M. Tsuda, and K. Deguchi. 1978. Purification and properties of α-hydroxy-γ-carboxymuconic-ε-semialdehyde dehydrogenase. J. Biochem. **83**:1125–1134.
23. Maruyama, K., M. Miwa, N. Tsujii, T. Nagai, N. Tomita, T. Harada, H. Sobajima, and H. Sugisaki. 2001. Cloning, sequencing, and expression of the gene encoding 4-hydroxy-4-methyl-2-oxoglutarate aldolase from *Pseudomonas ochraceae* NGJ1. Biosci. Biotechnol. Biochem. **65**:2701–2709.
24. Masai, E., Y. Katayama, S. Kawai, S. Nishikawa, M. Yamasaki, and N. Morohoshi. 1991. Cloning and sequencing of the gene for a *Pseudomonas paucimobilis* enzyme that cleaves β-aryl ether. J. Bacteriol. **173**:7950–7955.
25. Masai, E., Y. Katayama, S. Kubota, S. Kawai, M. Yamasaki, and N. Morohoshi. 1993. A bacterial enzyme degrading the model lignin compound β-etherase is a member of the glutathione-S-transferase superfamily. FEBS Lett. **323**:135–140.
26. Masai, E., Y. Katayama, S. Nishikawa, and M. Fukuda. 1999. Characterization of *Sphingomonas paucimobilis* SYK-6 genes involved in degradation of lignin-related compounds. J. Ind. Microbiol. Biotechnol. **23**:364–373.
27. Masai, E., K. Momose, H. Hara, S. Nishikawa, Y. Katayama, and M. Fukuda. 2000. Genetic and biochemical characterization of 4-carboxy-2-hydroxymuconate-6-semialdehyde dehydrogenase and its role in the protocatechuate 4,5-cleavage pathway in *Sphingomonas paucimobilis* SYK-6. J. Bacteriol. **182**:6651–6658.
28. Masai, E., S. Shinohara, H. Hara, S. Nishikawa, Y. Katayama, and M. Fukuda. 1999. Genetic and biochemical characterization of a 2-pyrone-4,6-dicarboxylic acid hydrolase involved in the protocatechuate 4,5-cleavage pathway of *Sphingomonas paucimobilis* SYK-6. J. Bacteriol. **181**:55–62.
29. Nishihara, H., and E. E. Dekker. 1972. Purification, substrate specificity and binding, β-decarboxylase activity, and other properties of *Escherichia coli* 2-keto-4-hydroxyglutarate aldolase. J. Biol. Chem. **247**:5079–5087.
30. Nishikawa, S., T. Sonoki, T. Kasahara, T. Obi, S. Kubota, S. Kawai, N. Morohoshi, and Y. Katayama. 1998. Cloning and sequencing of the *Sphingomonas (Pseudomonas)* *paucimobilis* gene essential for the O demethylation of vanillate and syringate. Appl. Environ. Microbiol. **64**:836–842.
31. Noda, Y., S. Nishikawa, K. Shiozuka, H. Kadokura, H. Nakajima, K. Yoda, Y. Katayama, N. Morohoshi, T. Haraguchi, and M. Yamasaki. 1990. Molecular cloning of the protocatechuate 4,5-dioxygenase genes of *Pseudomonas paucimobilis*. J. Bacteriol. **172**:2704–2709.
32. Parke, D. 1997. Acquisition, reorganization, and merger of genes: novel management of the β-ketoadipate pathway in *Agrobacterium tumefaciens*. FEMS Microbiol. Lett. **146**:3–12.
33. Peng, X., T. Egashira, K. Hanashiro, E. Masai, S. Nishikawa, Y. Katayama, K. Kimbara, and M. Fukuda. 1998. Cloning of a *Sphingomonas paucimobilis* SYK-6 gene encoding a novel oxygenase that cleaves lignin-related biphenyl and characterization of the enzyme. Appl. Environ. Microbiol. **64**:2520–2527.
34. Peng, X., E. Masai, Y. Katayama, and M. Fukuda. 1999. Characterization of the meta-cleavage compound hydrolase gene involved in degradation of the lignin-related biphenyl structure by *Sphingomonas paucimobilis* SYK-6. Appl. Environ. Microbiol. **65**:2789–2793.

35. Popp, R., T. Kohl, P. Patz, G. Trautwein, and U. Gerischer. 2002. Differential DNA binding of transcriptional regulator PcaU from *Acinetobacter* sp. strain ADP1. J. Bacteriol. 184:1988–1997.

36. Providenti, M. A., J. Mampel, S. MacSween, A. M. Cook, and R. C. Wyndham. 2001. *Comamonas testosteronei* BR6020 possesses a single genetic locus for extradiol cleavage of protocatechuate. Microbiology 147:2157–2167.

37. Richards, O. C., and W. J. Rutter. 1961. Preparation and properties of yeast aldolase. J. Biol. Chem. 236:3177–3184.

38. Sambrook, J., E. F. Fritsch, and T. Maniatis. 1989. Molecular cloning: a laboratory manual, 2nd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.

39. Sanger, F., S. Nicklen, and A. R. Coulson. 1977. DNA sequencing with chain-terminating inhibitors. Proc. Natl. Acad. Sci. USA 74:5463–5467.

40. Schäfer, A., A. Tauch, W. Jäger, J. Kalinowski, G. Thierbach, and A. Pühler. 1994. Small mobilizable multi-purpose cloning vectors derived from the *Escherichia coli* plasmids pK18 and pK19: selection of defined deletions in the chromosome of *Corynebacterium glutamicum*. Gene 145:69–73.

41. Schell, M. A. 1993. Molecular biology of the LysR family of transcriptional regulators. Annu. Rev. Microbiol. 47:597–626.

42. Short, J. M., J. M. Fernandez, J. A. Sorge, and W. Huse. 1988. AZAP: a bacteriophage λ expression vector with in vivo excision properties. Nucleic Acids Res. 16:7583–7600.

43. Studier, F. W., and B. A. Moffatt. 1986. Use of bacteriophage T7 RNA polymerase to direct selective high-level expression of cloned genes. J. Mol. Biol. 189:113–130.

44. Sugimoto, K., T. Senda, H. Aoshima, E. Masai, M. Fukuda, and Y. Mitsui.

45. Tack, B. F., P. J. Chapman, and S. Dagley. 1972. Purification and properties of 4-hydroxy-4-methyl-2-oxoglutarate aldolase. J. Biol. Chem. 247:6444–6449.

46. Trautwein, G., and U. Gerischer. 2001. Effects exerted by transcriptional regulator PcaU from *Acinetobacter* sp. strain ADP1. J. Bacteriol. 183:873–881.

47. Vieira, J., and J. Messing. 1982. The pUC plasmids, an M13mp7-derived system for insertion mutagenesis and sequencing with synthetic universal primers. Gene 19:259–268.

48. Wattiau, P., L. Bastiaens, R. van Herwijnen, L. Daal, J. R. Parsons, M-E. Renard, D. Springael, and G. R. Cornelis. 2001. Fluorene degradation by *Sphingomonas* sp. LB126 proceeds through protocatechuic acid: a genetic analysis. Res. Microbiol. 152:861–872.

49. Wolgel, S. A., J. E. Dege, P. E. Perkins-Olson, C. H. Jaurez-Garcia, R. L. Crawford, E. Munck, and J. D. Lipscomb. 1993. Purification and characterization of protocatechuate 2,3-dioxygenase from *Bacillus macerans*: a new extradiol catecholic dioxygenase. J. Bacteriol. 175:4414–4426.

50. Yanisch-Perron, C., J. Vieira, and J. Messing. 1985. Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. Gene 33:103–119.

51. Zylstra, G. J., R. H. Olsen, and D. P. Ballou. 1989. Genetic organization and sequence of the *Pseudomonas cepacia* genes for the alpha and beta subunits of protocatechuate 3,4-dioxygenase. J. Bacteriol. 171:5915–5921.
