
APPLIED AND ENVIRONMENTAL MICROBIOLOGY, Jan. 2003, p. 524–532  
0099-2240/03/$08.00+0 DOI: 10.1128/AEM.69.1.524–532.2003  
Copyright © 2003, American Society for Microbiology. All Rights Reserved.

**Genes for Chlorogenate and Hydroxycinnamate Catabolism (hca) Are Linked to Functionally Related Genes in the dca-pca-qui-pob-hca Chromosomal Cluster of Acinetobacter sp. Strain ADP1**

Michael A. Smith,† Valerie B. Weaver,‡ David M. Young,‡ and L. Nicholas Ornston*

Department of Molecular, Cellular and Developmental Biology, Yale University,  
New Haven, Connecticut 06520-8103

Received 23 July 2002/Accepted 21 October 2002

Hydroxycinnamates are ubiquitous in the environment because of their contributions to the structure and defense mechanisms of plants. Additional plant products, many of which are formed in response to stress, support the growth of *Acinetobacter* sp. strain ADP1 through pathways encoded by genes in the *dca-pca-qui-pob* chromosomal cluster. In an appropriate genetic background, it was possible to select for an *Acinetobacter* strain that had lost the ability to grow with caffeate, a commonly occurring hydroxycinnamate. The newly identified mutation was shown to be a deletion in a gene designated *hcaC* and encoding a ligase required for conversion of commonly occurring hydroxycinnamates (caffeate, ferulate, coumarate, and 3,4-dihydroxyphenylpropionate) to thioesters. Linkage analysis showed that *hcaC* is linked to *pobA*. Downstream from *hcaC* and transcribed in the direction opposite the direction of *pobA* transcription are open reading frames designated *hcaDEFG*. Functions of these genes were inferred from sequence comparisons and from the properties of knockout mutants. HcaD corresponded to an acyl coenzyme A (acyl-CoA) dehydrogenase required for conversion of 3,4-dihydroxyphenylpropionyl-CoA to caffeoyl-CoA. HcaE appears to encode a member of a family of outer membrane proteins known as porins. Knockout mutations in *hcaF* confer no discernible phenotype. Knockout mutations in *hcaG* indicate that this gene encodes a membrane-associated esterase that hydrolyzes chlorogenate to quinate, which is metabolized in the periplasm, and caffeate, which is metabolized by intracellular enzymes. The chromosomal location of *hcaG*, between *hcaC* (required for growth with caffeate) and *quiA* (required for growth with quinate), provided the essential clue that led to the genetic test of HcaG as the esterase that produces caffeate and quinate from chlorogenate. Thus, in this study, organization within what is now established as the *dca-pca-qui-pob-hca* chromosomal cluster provided essential information about the function of genes in the environment.

Hydroxycinnamates are abundant in plants and are used in both structural (3, 4, 61) and chemical (6) plant defense strategies. These compounds occur freely or as components of plant polymers, such as suberin (3, 4, 23, 38, 61). Highly significant among the hydroxycinnamates is chlorogenate, which is the most abundant phenolic compound in potato tubers (37) and accounts for 1.5% of the dry weight of defatted sunflower oil (22). Chlorogenate has numerous functions, including effects on disease resistance (39) and on the palatability of leaves to insects (27). The production of animal feeds is hindered by the presence of chlorogenic acid, which binds to proteins and keeps them from being fully metabolized (22).

Chlorogenate is an ester, and hydrolysis of its most common form (5-O-caffeoylequinic acid) produces quinate and caffeate, a free hydroxycinnamate (Fig. 1). Included in the chemical family of chlorogenates are additional esters of caffeate, and these compounds are hydrolyzed with various degrees of efficiency by an extracellular chlorogenate esterase (EC 3.1.1.43)

from *Aspergillus niger* (54, 55) or *Aspergillus japonicus* (45, 46). A preparation of the latter enzyme is commercially available (http://www.kikkoman.co.jp/bio/j/rinsyou/enzymes/) and is used to control bitterness and to prevent enzymatic browning in the preparation of juice, wine, and coffee. Chlorogenate esterase has been identified as an intracellular enzyme in gut bacteria (7), but little is known about genes encoding this enzyme or how their expression is controlled.

Here we describe bacterial *hca* genes for catabolism of the ester chlorogenate and the free hydroxycinnamates caffeate, ferulate, coumarate, and 3,4-dihydroxypropionyl coenzyme A (3,4-dihydroxypropionyl-CoA) (Fig. 1). The genetic analysis was conducted with *Acinetobacter* sp. strain ADP1 (also known as strain BD413 [31]) because of this organism’s extraordinary competence for natural transformation (30). An additional convenience for study of catabolic genes was afforded by the toxicity of carboxymuconate, the product of protocatechuate 3,4-dioxygenase. As summarized in Fig. 1, strains unable to metabolize carboxymuconate can be used to select mutants defective in protocatechuate 3,4-dioxygenase and in metabolic pathways that converge upon protocatechuate (26). The initial evidence (2) for clustering of *Acinetobacter* genes for metabolic pathways converging through protocatechuate emerged from characterization of mutant strains blocked in expression of *pcaHG*, which are structural genes for protocatechuate oxygenase, and *pobA*, which is the structural gene for *p*-hydroxy-

VOL. 69, 2003 GENES FOR CHLOROGENATE AND HYDROXYCINNAMATE CATABOLISM 525

![Diagram]

FIG. 1. Metabolic pathways converging upon protocatechuate in *Acinetobacter* sp. strain ADP1. The Δ*pcaBDK1* deletion blocks metabolism of carboxymuconate (dashed box). Accumulation of this compound prevents growth in the presence of protocatechuate (box) or its metabolic precursors, and this allows selection of strains with secondary mutations that prevent metabolism of these compounds (20). Replacement of Δ*pcaBDK1* with wild-type DNA produces strains in which the secondary mutation is the only genetic barrier to catabolism. In previous investigations researchers reported properties of mutants blocked in expression of *pcaHG* (10, 20), *pobA* (14, 26), and *vanAB* (43, 56). In this report we describe a spontaneous *hcaC* (box) mutant unable to metabolize caffeate. As shown here, the Δ*hcaC1* mutation appears to inactivate a CoA ligase required for growth with coumarate, ferulate, and 3,4-dihydroxyphenylpropionate, as well as caffeate. Between *hcaC* and *pobA* in the *Acinetobacter* chromosome are the structural genes *hcaD* (box), encoding an enzyme that oxidizes 3,4-dihydroxyphenylpropionyl-CoA to caffeoyl-CoA, and *hcaG* (box), encoding an enzyme that hydrolyzes chlorogenate to quinate and caffeate. Wild-type *Acinetobacter* cells exhibit doubling times ranging from 40 min to 1 h when they are grown at the expense of 3,4-dihydroxyphenylpropionate, coumarate, ferulate, quinate, caffeate, or chlorogenate.

benzoate hydroxylase (Fig. 1). Chromosomal fragments containing *pobA* and surrounding DNA were cloned (2, 13). Analysis of open reading frames between *pcaHG* and *pobA* revealed that the *quiBCXA* cluster is required for conversion of quinate to protocatechuate (17, 18).

Spontaneous mutants lacking *pcaHG* often have large deletions (20), and some of these deletions prevent growth with straight-chain dicarboxylic acids (8). This finding led to elucidation of *dca* genes required for growth with the acids having chain lengths ranging from 6 to 16 carbon atoms (48). The dicarboxylic acids and hydroxycinnamates are major components of suberin (3, 4, 23, 38, 61). Thus, the overall *dca-pca-qui-pob* cluster was known to contain independently transcribed groups of genes that allow growth with plant products, including dicarboxylic acids (*dca*), protocatechuate (*pca*), quinate (*qui*), and *p*-hydroxybenzoate (*pob*). An exception to this pattern of clustering was provided by the unlinked *vanAB* genes encoding the demethylase that converts vanillate to protocatechuate (56).

Vanillate is an intermediate in catabolism of ferulate (11, 19, 44, 51, 58). The location of *Acinetobacter* genes for metabolism of ferulate and other hydroxycinnamates was unknown. As described here, a strain containing a spontaneous mutation blocking catabolism of a hydroxycinnamate, caffeate (Fig. 1), was isolated, and a cloned restriction fragment containing *pobA* was shown to restore wild-type function to the mutant

strain. Designed mutations altering DNA within the fragment allowed identification of open reading frames required for metabolism of hydroxycinnamates, including chlorogenate (5-*O*-caffeoylquinate). The newly discovered genes expand our knowledge of the *dca-pca-qui-pob-hca* supraoperonic cluster, which makes a major contribution to the metabolic versatility of *Acinetobacter*. The phenotypic characterization of clustered genes provides clues about biological sources of the mixtures of nutrients used by bacteria in the natural environment.

MATERIALS AND METHODS

Strains and culture conditions. Plasmids and relevant mutants of *Acinetobacter* sp. strain ADP1 are listed in Table 1. *Acinetobacter* strains were routinely grown at 37°C in mineral medium supplemented with 10 mM succinate, 5 mM *p*-hydroxybenzoate, or one of the following compounds at a concentration of 3 mM: caffeate, quinate, coumarate, 3,4 dihydroxyphenylpropionate, protocatechuate, chlorogenate (5-*O*-caffeoylquinate), or ferulate. These chemicals were supplied by Sigma Chemical Company. The chemical instability of caffeate, protocatechuate, and chlorogenate demanded that plates containing these carbon sources be prepared within 24 h of use. *Escherichia coli* strain DH5α was obtained as a frozen suspension from Gibco BRL. *E. coli* cells were grown in Luria-Bertani (53) broth at 37°C. Growth media were supplemented as required with ampicillin (100 μg/ml) or kanamycin(100 μg/ml).

DNA manipulation. General previously described procedures (53) were used for manipulation of DNA. Instructions from Bio-Rad were followed when InstaGene matrix was used to isolate chromosomal DNA and for PCR. PCR mixtures (50 μl) used to amplify DNA for sequencing contained 2 μl of supernatant liquid separated from the InstaGene matrix as a template, 2 U of *Pfu*

TABLE 1. Bacterial strains and plasmids

| Strain or plasmid | Relevant characteristics | Reference or source |
|-------------------|-------------------------|---------------------|
| Acinetobacter strains |  |  |
| ADP1 | Wild type (also known as *Acinetobacter* sp. strain BD413) | 31 |
| ADP230 | $\Delta pcaBDK1$ | 10 |
| ADP1027 | $\Delta hcaC1$ (spontaneous 100-bp deletion in $hcaC$) | This study |
| ADP9500 | $\Delta pobA-hcaG1$ (1,126-bp deletion between most distant MfeI sites of $pobA$ and $hcaG$) | This study |
| ADP9501 | $\Delta hcaG2$ (518-bp designed chromosomal deletion in $hcaG$) | This study |
| ADP9502 | $hcaF1::Km^r$ ($Km^r$ from pRME1 in EcoRV site of $hcaF$) | This study |
| ADP9503 | $\Delta hcaF2$ (155-bp chromosomal deletion) | This study |
| ADP9504 | $hcaD1::Km^r$ ($Km^r$ from pRME1 in MfeI site of $hcaD$) | This study |
| ADP9510 | $hcaG3::T7\cdot TAG$ ($T7\cdot TAG$ fused in frame to 3′ end of $hcaG$) | This study |
| Plasmids |  |  |
| pGEM-3Zf(+) | $Amp^r$ | Promega |
| pRME1 | $Km^r$ | 21 |
| pZR3 | 2,754-bp HindIII fragment containing wild-type $pcaBDK$ | 16 |
| pZR400 | 13.5-Kb insert in pUC18 containing $pobAR$ and DNA downstream from $pobA$ | 13 |
| pZR9500 | $PstI-XhoI$ fragment from pZR400 in pGEM-3Zf(+) | This study |
| pZR9501 | 3,135-bp $XhoI-HindIII$ fragment from pZR9500 in pGEM-3Zf(+) | This study |
| pZR9502 | 3,045-bp HindIII fragment from pZR9500 in pGEM-3Zf(+) | This study |
| pZR9503 | 1,714-bp HindIII fragment from pZR9500 in pGEM-3Zf(+) | This study |
| pZR9507 | pZR9501 containing $\Delta pobA-hcaG1$ | This study |
| pZR9508 | pZR9502 containing $hcaD::Km^r$ ($Km^r$ in MfeI site in $hcaA$) | This study |
| pZR9509 | pZR9501 containing $hcaF1::Km^r$ | This study |

VOL. 69, 2003 GENES FOR CHLOROGENATE AND HYDROXYCINNAMATE CATABOLISM 527

![Diagram]

FIG. 2. Genetic characterization of *Acinetobacter hca* genes associated with hydroxycinnamate catabolism. The newly discovered *hca* genes extend downstream from *pobA* in the *dca-pca-qui-pob-hca* cluster. As described here, HcaG hydrolyzes chlorogenate to quinate and caffeate. Metabolism of quinate to protocatechuate requires the *qui* genes, and HcaC initiates caffeate catabolism by converting the compound to its thioester. Upstream from *hcaC* and translated in the same direction is a 486-bp segment of an open reading frame encoding an aldehyde dehydrogenase. The complete open reading frame (*hcaB*) and an open reading frame designated *hcaA* have been shown in a separate study (Parke, unpublished) to encode genes for functions that complete the conversion of caffeate to protocatechuate. The locations of mutations used to identify functions of *hca* genes are indicated, as are the locations of plasmid inserts and restriction sites that were used for genetic construction.

primers B and H were used to prepared the amplicon BH (Table 2). After restriction with EcoRI, the amplicons were ligated to form amplicon AGHB, which contained T7·TAG ligated to the carboxy terminus of HcaG (Table 2). After amplification with primers A and B, the AGHB DNA was used to transform strain ADP9500. A resulting transformant, strain ADP9510, was selected on p-hydroxybenzoate. This organism grew on chlorogenate and expressed *hcaG3::T7·TAG* inducibly. Sequencing demonstrated that the T7·TAG was in the predicted chromosomal location.

SDS-PAGE and immunodetection of HcaG::T7·TAG. Strain ADP9510 (*hcaG3::T7·TAG*) was grown to a concentration of about \(5 \times 10^8\) cells per ml and harvested by centrifugation at \(5,000 \times g\) for 5 min. Centrifuged cells were suspended in 10 mM Na₂KPO₄ buffer (pH 7.0) and sonicated. Cellular debris was removed by centrifugation of the sonicated material at \(5,000 \times g\) for 15 min, and the supernatant liquid was filtered through a 2-μm-pore-size membrane. Protein concentrations were measured by the method of Bradford by using bovine serum albumin as the standard. Sodium dodecyl sulfate (SDS)-polyacrylamide gel electrophoresis (PAGE) was performed as described by Laemmli (33) with a Bio-Rad mini-slab gel apparatus. Proteins from the SDS-PAGE gels were transferred to nitrocellulose by capillary action in blotting buffer (10 mM Tris, 150 mM glycine, 20% methanol; pH 8.3) for no less than 24 h. Proteins transferred to the nitrocellulose were immunostained by using horseradish peroxidase-conjugated T7·TAG antibody (Novagen) and the SuperSignal substrate (Novagen) and following the instructions of the manufacturer, which also supplied Perfect Protein Western Markers as molecular weight markers.

Separation of soluble and membrane-associated fractions. Chlorogenate-grown cultures of ADP9510 were centrifuged, resuspended in 10 mM Na₂KPO₄ buffer (pH 7.0), and sonicated. Whole cells were removed by centrifugation for 20 min at \(5,000 \times g\), followed by filtration through a 2-μm-pore-size filter. The filtrate was centrifuged at \(110,000 \times g\) for 1 h at \(4^\circ \mathrm{C}\). The supernatant liquid was removed, and the pellet was suspended in a volume equal to that of the removed supernatant liquid.

Nucleotide sequence accession number. The DNA sequence of the *hca* genes from *Acinetobacter* sp. strain ADP1 have been deposited in the GenBank database under accession no. L05770. It should be noted that the designation *ech* has been assigned to the structural gene for enoyl-CoA hydratase lyase and the designation *fcs* has been assigned to the feruloyl-CoA ligase gene (1). As shown here and elsewhere (Parke, unpublished), *Acinetobacter* genes with these functions are closely clustered in what may prove to be a single transcript generally associated with hydroxycinnamate catabolism. The designation *hca* has been assigned to this transcript; *hcaA* is the designation for the structural gene for

TABLE 2. Primers and amplicons

| Primer or amplicon | Strand | Position$^a$ | Sequence or modification$^b$ |
|--------------------|--------|-------------|-----------------------------|
| **Primers**        |        |             |                             |
| A                  | +      | 400–418     | $5'$-GCCGATGTGCCGCCTGTGG-$3'$ |
| B                  | –      | 1743–1760   | $5'$-CCGGGATTACTTGGCAGC-$3'$ |
| C                  | +      | 2279–2291   | $5'$-CATAACCATAGTTGATGCGAG-$3'$ |
| D                  | –      | 2840–2857   | $5'$-CATAAATTCAGGCATAAGG-$3'$ |
| E                  | +      | 3013–3034   | $5'$-GGTCAGATTAATCGTCGTGGG-$3'$ |
| F                  | –      | 3630–3648   | $5'$-GGTGGTGAAACAGGACTGC-$3'$ |
| G                  | –      | 1069–1089   | $5'$-AAAAAAGAATTCTAACACAGCCGAGATAAAAAAG-$3'$$^b$ |
| H                  | +      | 1092–1110   | $5'$-TTTTTGAATTCTAACCCATTTGCTCCACCAGTCATGCTAGCCATTTTACAAGTGAAATTCTCG-$3'$$^b$ |
| **Amplicons**      |        |             |                             |
| AB                 |        | 400–1760    |                             |
| CD                 |        | 2279–2857   |                             |
| ABCD               |        | 400–2857    | 518 bp deleted between positions 1760 and 2279 (residues 956 to 1475 in $hcaG$) |
| EF                 |        | 3013–3630   |                             |
| CDEF               |        | 2279–3630   | 115 bp deleted between positions 2857 and 3013 (residues 230 to 386 in $hcaF$) |
| AG                 |        | 400–1089    | Contains poly(A) leader and EcoRI site |
| HB                 |        | 1092–2297   | Contains poly(T) leader and EcoRI site |
| AGHB               |        | 400–2297    | Encodes T7·TAG fused to C terminus of $hcaG$ |

$a$ Position relative to XhoI site in $pobA$.  
$b$ The chromosomal sequence is indicated by boldface type, the EcoRI sequence is italicized, and the T7·TAG sequence is underlined.

enoyl-CoA hydratase lyase (Parke, unpublished), and $hcaC$ is the designation for the structural gene for hydroxycinnamate-CoA ligase.

---

## RESULTS

Selection and properties of mutant strain ADP1027 ($\Delta hcaCl$) lacking a CoA ligase required for growth with hydroxycinamic acids. Exposure of $10^8$ cells from an overnight culture of strain ADP230 ($\Delta pcaBDK1$) to 3 mM caffeate in plates containing 10 mM succinate produced about 100 colonies on each plate. After purification by streaking on succinate plates, some of the mutants grew with succinate in the presence of caffeate but not in the presence of protocatechuate. It seemed likely that these strains were blocked in the conversion of caffeate to protocatechuate, and this possibility was tested by replacing $\Delta pcaBDK1$ in one of the strains with wild-type DNA and examining the growth properties of the resulting organism, strain ADP1027 (Table 1). This strain used $p$-hydroxybenzoate as a growth substrate and failed to grow at the expense of caffeate, $p$-coumarate, ferulate, or 3,4-dihydroxyphenylpropionate (Fig. 1).

Plasmid pZR400 is known to contain $pobA$ and a segment of DNA downstream from this gene (13). The subclone pZR9500 (Table 1) contains a portion of $pobA$ and the previously uncharacterized downstream DNA. Transformation of strain ADP1027 with linearized pZR9500 produced recombinants that grew with caffeate. Therefore, it was evident that the gene inactivated in strain ADP1027 is part of the insert in pZR9500 and is linked to $pobA$. Sequencing of DNA downstream from $pobA$ in pZR9500 revealed five complete open reading frames designated $hcaC$, $hcaD$, $hcaE$, $hcaF$, and $hcaG$ (Fig. 2; Table 3). Transformation of strain ADP1027 with subclone pZR9503 (Fig. 2) showed that the mutation is within an open reading frame designated $hcaC$. Sequencing the corresponding portion of the mutant strain ADP1027 chromosome revealed that the selected spontaneous $hcaCl$ mutation is a 100-bp deletion extending from the sequence ACATT at position 702 to ACATT beginning at position 802 in the gene.

The deduced primary structure of HcaC corresponds to

---

## TABLE 3. Genes in the sequenced $hca$ region

| Gene  | Strand | Position     | Deduced function                     |
|-------|--------|--------------|--------------------------------------|
| $hcaC$| –      | 5830–7710    | Hydroxycinnamate lyase               |
| $hcaD$| –      | 4621–5760    | Acyl-CoA dehydrogenase              |
| $hcaE$| –      | 3295–4530    | Porin                                |
| $hcaF$| –      | 2854–3243    | Unknown                              |
| $hcaG$| –      | 1089–2831    | Chlorogenate esterase                |
| $pobA$| +      | 1–1041       | $p$-Hydroxybenzoate hydroxylase     |

VOL. 69, 2003 GENES FOR CHLOROGENATE AND HYDROXYCINNAMATE CATABOLISM 529

Marker Proteins  
Quinate-grown (60 μg)  
Chlorogenate-grown (60 μg)  
Chlorogenate-grown (20 μg)  
Caffeate-grown (20 μg)

Molecular Weight  
150-kDa  
75-kDa  
50-kDa  
35-kDa  
25-kDa  
15-kDa  

Marker Proteins  
Caffeate-grown (20 μg)  
Ferulate-grown (20 μg)  
Coumarate-grown (20 μg)  
Chlorogenate-grown (20 μg)

Marker Proteins  
Chlorogenate-grown (20 μg of membrane fraction)  
Chlorogenate-grown (20 μg of soluble fraction)

HcaG

A.  
B.  
C.

FIG. 3. Induction and cellular location of chlorogenate esterase detected as HcaG3::T7·Tag in cell extracts (A and B) and in a membrane preparation (C). The amount of protein separated by SDS-PAGE in each lane is indicated. Immunoblots show the presence of the esterase in extracts of cells grown with chlorogenate, caffeate, ferulate, and coumarate (A and B) but not in extracts of cells grown with quinate (A). The esterase was associated with the membrane fraction of cell extracts (C).

sion of vanillate, a product of ferulate, to protocatechuate (Fig. 1). The *van* genes are exceptional in that they occupy a genetically unstable locus far from all of the other genes required for complete catabolism of ferulate (56). In light of the structural similarity of ferulate and vanillate, it seemed likely that multiple knockout mutations would be required in order to observe the possible contributions of *vanP* and *hcaE* to ferulate metabolism, so genetic analysis of *hcaE* was not included as part of this study.

A knockout mutation in *hcaF* confers no discernible phenotype. Sequence comparisons revealed few matches for the 129-amino-acid protein predicted to be encoded by *hcaF*. The closest matches are with proteins with unknown functions from *Brucella* and *Caulobacter*. Knockouts caused by *hcaF1::Km*<sup>r</sup> and Δ*hcaF2* had no discernible effect on catabolism of hydroxycinnamates, so the function of *hcaF* remains unknown.

Properties of strain ADP9501 (Δ*hcaG2*) lacking an esterase required for hydrolysis of chlorogenate to quinate and caffeate. The amino acid sequence of HcaG exhibits 29 and 27% identity to the deduced primary structures of a tannase from *Agrobacterium tumefaciens* (60) and a feruloyl esterase from *A. niger* (12), respectively. A likely function for HcaG was inferred from the location of its structural gene between other *hca* genes required for caffeate catabolism and the *qui* genes associated with quinate catabolism in *Acinetobacter* (Fig. 2). Since hydrolysis of chlorogenate produces caffeate and quinate (Fig. 2), it seemed likely that HcaG is chlorogenate esterase. This supposition was confirmed by demonstration that the knockout mutation Δ*hcaG1* eliminated the ability of *Acinetobacter* to grow with chlorogenate while it left the ability to grow with caffeate or quinate intact. Mutations blocking *hcaF* (Table 1) did not prevent growth with chlorogenate.

Regulation of chlorogenate esterase (HcaG) synthesis by intracellular catabolites and evidence that the enzyme is membrane associated. An interesting feature of the chlorogenate esterase reaction is that one of its products, quinate, is metabolized outside the inner cell membrane (9), whereas the other product, caffeate, presumably is metabolized intracellularly through the thioester intermediate. This raised the possibility that the esterase acts upon chlorogenate in the inner cell membrane and that chlorogenate does not enter cells. This consideration, coupled with the observation that *hcaG* is in a position where it may be cotranscribed with other *hca* genes, suggested that the inducer of chlorogenate esterase might be either caffeate or an intracellular catabolite formed from chlorogenate. The protein was determined by using mutant strain ADP9510 (*hcaG3::T7·TAG*), which produces HcaG with an epitope tag fused to its carboxyl terminus. As judged by growth properties, the in vivo activity of the mutant was unimpaired by the fusion, and the esterase protein was detected immunochemically in cell extracts. As shown in Fig. 3, HcaG3::T7·TAG was not induced by growth with quinate. Growth with chlorogenate, caffeate, ferulate, or coumarate induced the enzyme (Fig. 3). An indication that metabolism of caffeate may be necessary for induction of HcaG emerged from the observation that mutant strain ADP1027 (Δ*hcaCI*), which was unable to form caffeoyl-CoA from caffeate, did not grow with chlorogenate.

The inference that chlorogenate esterase (HcaG) is membrane associated was explored by testing for the presence of HcaG3::T7·TAG in soluble and membrane fractions separated from extracts of chlorogenate-grown cultures of strain ADP9510. As shown in Fig. 3C, HcaG protein was strongly represented in the membrane fraction and undetectable in the soluble fraction.

530 SMITH ET AL.

APPL. ENVIRON. MICROBIOL.

![Diagram](attachment:diagram.png)

FIG. 4. Cellular location of chlorogenate esterase. The products of the enzyme are metabolized on different sides of the intracellular membrane. QuiA initiates a sequence of reactions that convert quinate to protocatechuate on the outer surface of the membrane. Specific transport systems transfer protocatechuate to the cytosol (9). Metabolism of caffeate in the cytosol is initiated by HcaC, which converts the compound to a thioester of CoA.

---

### DISCUSSION

Gene organization and the natural environment. The nutritional versatility of microorganisms has been examined most frequently and most conveniently by individual presentation of potential growth substrates. Such studies have resulted in valuable conclusions about how shared sets of common nutrients help to define microbial taxa. Left unanswered are questions about where and how the nutrients are encountered by the microorganisms in their natural environments. Curiously, some clues in this regard have emerged from studies of chromosomal gene organization. A tendency towards clustering of bacterial genes for related catabolic functions has been known for some time (35, 52), and two large clusters have been identified in the chromosome of *Acinetobacter* sp. strain ADP1. One of these, the *sal-are-ben-cat* cluster (28, 29), encodes enzymes that metabolize a range of aromatic acids, alcohols, and esters through catechol and the β-ketoadipate pathway.

Our knowledge concerning the other large cluster, containing the *dca-pca-qui-pob* genes, was expanded to include *hca* genes through this investigation. Most of the *dca-pca-qui-pob-hca* genes contribute to metabolism of aromatic (2, 15) and hydroaromatic (17) compounds through protocatechuate, but the

*dca* genes (48) encode enzymes that act upon straight-chain dicarboxylic acids by independent metabolic pathways (Fig. 2). A shared characteristic of all the genes is that they encode pathways for catabolism of compounds that are produced by plants in response to stress. In particular, the dicarboxylic acids and hydroxyaromatic acids are components of the protective plant polymer suberin (4, 23, 38, 61). Hydroxycinnamate compounds serve additional protective roles, such as the contribution of ferulate in forming a “molecular spot weld” (32) in response to physical damage. This grouping of functions warranted the initial examination of cloned DNA extending downstream from *pobA* for the gene inactivated by the spontaneous Δ*hcaC1* mutation blocking conversion of a hydroxycinnamate, caffeate, to protocatechuate. The requirement for *hcaC* for metabolism of ferulate, coumarate, and 3,4-dihydroxypropionate shows that the CoA ligase encoded by the gene has specificity broad enough to encompass common hydroxycinnamates. Breadth of specificity also is exhibited by plant hydroxycinnamate-CoA ligases, which are commonly designated coumarate CoA ligases (25, 34, 36, 57). Interest in the production of vanillin from ferulate has directed attention to microbial ferulate CoA ligases (1, 19, 40, 47, 58), although these enzymes act

upon a range of hydroxycinnamates (40). Where breadth of specificity has been established and is known to contribute to biological function, it might be preferable to refer to such enzymes as hydroxycinnamate CoA ligases.

Downstream from *hcaC* is *hcaD* encoding a protein corresponding to a member of an acyl-CoA dehydrogenase family of enzymes. Inactivation of *hcaD* prevents growth with 3,4-dihydroxyphenylpropionate (Fig. 1). Since *hcaC* also is required for metabolism of 3,4-dihydroxyphenylpropionate, the overall findings are consistent with the hypothesis that HcaC has a specificity broad enough to allow formation of 3,4-dihydroxyphenylpropionyl-CoA, the substrate for the acyl-CoA dehydrogenase (Fig. 1). The genetic location of the HcaD gene suggests that hydroxycinnamates and their derivatives with saturated side chains may serve as companion nutrients for organisms related to *Acinetobacter* sp. strain ADP1 in their natural environments.

The chromosomal location of *hcaG* between the *qui* (quinate) and *hca* (hydroxycinnamate) genes for catabolism of its products (quinate and caffeate) provided an essential clue in determining the function of HcaG, which proved to be chlorogenate esterase. Had *hcaG* been located elsewhere on the chromosome, the effect of a knockout *hcaG* mutation on chlorogenate metabolism would not have been tested. Does chlorogenate accompany the hydroxycinnamates that combine to provide nutrition for *Acinetobacter* strain sp. strain ADP1 and related organisms in the natural environment? This possibility is heightened by evidence that hydroxycinnamate metabolism induces chlorogenate esterase. Growth with hydroxycinnamates alone would cause gratuitous synthesis of the esterase.

Left unanswered by this investigation is the range of chlorogenate substrates that are hydrolyzed by HcaG. This information, coupled with information about the distribution of chlorogenic acids in different plant sources (6, 42, 49, 59), might provide greater insight into the niche occupied by *Acinetobacter* sp. strain ADP1. The inference that hydroxycinnamates help to define the niche of the organism comes from the observation that the organism does not grow with tyrosine, a common amino acid that could be converted by a single enzyme (tyrosine ammonia lyase) to coumarate, a hydroxycinnamate that supports robust growth of the strain. In this sense, the organism is a nutritional specialist that is dependent upon plant products and seemingly eschews the opportunity to be a generalist which is capable of growth with a compound found in all living things.

Another question, not addressed here, is the mechanism by which the two-carbon fragment is removed from the side chain of hydroxycinnamyl thioesters during their metabolism. In another investigation, Parke (unpublished) has shown that, as reported in studies with other bacteria (1, 19, 41, 50), a hydratase/lyase cleaves the thioesters, forming the corresponding aldehydes and acetyl-CoA in *Acinetobacter* sp. strain ADP1. Ferulate is catabolized through vanillate in this organism (56).

Cellular location of chlorogenate esterase. Chlorogenate esterase synthesis may or may not be physiologically burdensome, but its demands certainly are complex. One product of the enzyme, quinate, is metabolized outside the inner cell membrane (9), whereas the other product, caffeate, is metabolized at an intracellular location, as indicated by the requirement for CoA (Fig. 4). The simplest interpretation of this finding is that chlorogenate esterase is located within the inner cell membrane so that quinate is produced on one side and caffeate is produced on the other (Fig. 4). Alternatively, quinate and caffeate may be produced in the periplasm, followed by transport of the caffeate across the inner cell membrane. Consistent with both interpretations is the apparent location of the epitope-tagged enzyme (HcaG3::T7·TAG) in the membrane fraction of cell extracts (Fig. 3).

### ACKNOWLEDGMENTS

This research was funded by grant DAAG55-98-1-0232 from the Army Research Office and by grant MCB-9603980 from the National Science Foundation to L.N.O. M.A.S. received support from the Yale Institute for Biospheric Studies.

We thank Donna Parke for providing essential information about *hca* genes that were inaccessible in the present investigation.

### REFERENCES

1. Achterholt, S., H. Priefert, and A. Steinbüchel. 2000. Identification of *Amycolatopsis* sp. strain HR167 genes, involved in the bioconversion of ferulic acid to vanillin. Appl. Microbiol. Biotechnol. **54**:799–807.
2. Averhoff, B., L. Gregg-Jolly, D. Elsemore, and L. N. Ornston. 1992. Genetic analysis of supraoperonic clustering by use of natural transformation in *Acinetobacter calcoaceticus*. J. Bacteriol. **174**:200–204.
3. Bernards, M. A., and N. Lewis, G. 1998. The macromolecular aromatic domain in suberized tissue: a changing paradigm. Phytochemistry **47**:915–933.
4. Bernards, M. A., and F. A. Razem. 2001. The poly(phenolic) domain of potato suberin: a non-lignin cell wall bio-polymer. Phytochemistry **57**:1115–1122.
5. Birnboim, H. C., and J. Doly. 1979. A rapid alkaline extraction procedure for screening recombinant plasmid DNA. Nucleic Acids Res. **7**:1513–1523.
6. Clifford, M. N. 2000. Chlorogenic acids and other cinnamates—nature, occurrence, dietary burden, absorption and metabolism. J. Sci. Food Agric. **80**:1033–1043.
7. Couteau, D., A. L. McCartney, G. R. Gibson, G. Williamson, and C. B. Faulds. 2001. Isolation and characterization of human colonic bacteria able to hydrolyse chlorogenic acid. J. Appl. Microbiol. **90**:873–881.
8. D'Argenio, D. A., A. Segura, P. V. Bunz, and L. N. Ornston. 2001. Spontaneous mutations affecting transcriptional regulation by protocatechuate in *Acinetobacter*. FEMS Microbiol. Lett. **201**:15–19.
9. D'Argenio, D. A., A. Segura, W. M. Coco, P. V. Bunz, and L. N. Ornston. 1999. The physiological contribution of *Acinetobacter* PcaK, a transport system that acts upon protocatechuate, can be masked by the overlapping specificity of VanK. J. Bacteriol. **181**:3505–3515.
10. D'Argenio, D. A., M. W. Vetting, D. H. Ohlendorf, and L. N. Ornston. 1999. Substitution, insertion, deletion, suppression, and altered substrate specificity in functional protocatechuate 3,4-dioxygenases. J. Bacteriol. **181**:6478–6487.
11. Delneri, D., G. Degrassi, R. Rizzo, and C. V. Bruschi. 1995. Degradation of trans-ferulic and *p*-coumaric acid by *Acinetobacter calcoaceticus* DSM 586. Biochim. Biophys. Acta **1244**:363–367.
12. de Vries, R. P., P. A. VanKuyk, H. C. M. Kester, and J. Visser. 2002. The *Aspergillus niger faeB* gene encodes a second feruloyl esterase involved in pectin and xylan degradation and is specifically induced in the presence of aromatic compounds. Biochem. J. **363**:377–386.
13. DiMarco, A. A., B. Averhoff, and L. N. Ornston. 1993. Identification of the transcriptional activator *pobR* and characterization of its role in the expression of *pobA*, the structural gene for *p*-hydroxybenzoate hydroxylase in *Acinetobacter calcoaceticus*. J. Bacteriol. **175**:4499–4506.
14. DiMarco, A. A., B. A. Averhoff, E. E. Kim, and L. N. Ornston. 1993. Evolutionary divergence of *pobA*, the structural gene encoding *p*-hydroxybenzoate hydroxylase in an *Acinetobacter calcoaceticus* strain well-suited for genetic analysis. Gene **125**:25–33.
15. DiMarco, A. A., and L. N. Ornston. 1994. Regulation of *p*-hydroxybenzoate hydroxylase synthesis by PobR bound to an operator in *Acinetobacter calcoaceticus*. J. Bacteriol. **176**:4277–4284.
16. Doten, R. C., K. L. Ngai, D. J. Mitchell, and L. N. Ornston. 1987. Cloning and genetic organization of the *pca* gene cluster from *Acinetobacter calcoaceticus*. J. Bacteriol. **169**:3168–3174.
17. Elsemore, D. A., and L. N. Ornston. 1994. The *pca-pob* supraoperonic cluster of *Acinetobacter calcoaceticus* contains *quiA*, the structural gene for quinate shikimate dehydrogenase. J. Bacteriol. **176**:7659–7666.
18. Elsemore, D. A., and L. N. Ornston. 1995. Unusual ancestry of dehydratases associated with quinate catabolism in *Acinetobacter calcoaceticus*. J. Bacteriol. **177**:5971–5978.
19. Gasson, M. J., Y. Kitamura, W. R. McLauchlan, A. Narbad, A. J. Parr,

E. L. H. Parsons, J. Payne, M. J. C. Rhodes, and N. J. Walton. 1998. Metabolism of ferulic acid to vanillin. A bacterial gene of the enoyl-SCoA hydratase/isomerase superfamily encodes an enzyme for the hydration and cleavage of a hydroxycinnamic acid SCoA thioester. J. Biol. Chem. 273:4163–4170.

20. Gerischer, U., and L. N. Ornston. 1995. Spontaneous mutations in *pcaH* and *-G*, structural genes for protocatechuate 3,4-dioxygenase in *Acinetobacter calcoaceticus*. J. Bacteriol. 177:1336–1347.

21. Gielow, A., L. Diederich, and W. Messer. 1991. Characterization of a phage-plasmid hybrid (phasyl) with two independent origins of replication isolated from *Escherichia coli*. J. Bacteriol. 173:73–79.

22. Gonzalez-Perez, S., K. B. Merck, J. M. Vereijken, G. A. van Koningsveld, H. Gruppen, and A. G. J. Voragen. 2002. Isolation and characterization of undenatured chlorogenic acid free sunflower (*Helianthus annuus*) proteins. J. Agric. Food Chem. 50:1713–1719.

23. Graca, J., and H. Pereira. 2000. Suberin structure in potato periderm: glycerol, long-chain monomers, and glyceryl and feruloyl dimers. J. Agric. Food Chem. 48:5476–5483.

24. Hancock, R. E. W., and F. S. L. Brinkman. 2002. Function of *Pseudomonas* porins in uptake and efflux. Annu. Rev. Microbiol. 56:17–38.

25. Harding, S. A., J. Leshkevich, V. L. Chiang, and C. J. Tsai. 2002. Differential substrate inhibition couples kinetically distinct 4-coumarate:coenzyme A ligases with spatially distinct metabolic roles in quaking aspen. Plant Physiol. 128:428–438.

26. Hartnett, G. B., B. Averhoff, and L. N. Ornston. 1990. Selection of *Acinetobacter calcoaceticus* mutants deficient in the *p*-hydroxybenzoate hydroxylase gene (*pobA*), a member of a supraoperonic cluster. J. Bacteriol. 172:6160–6161.

27. Ikonen, A., J. Tahvanainen, and H. Roininen. 2002. Phenolic secondary compounds as determinants of the host plant preferences of the leaf beetle, *Agelastica alni*. Chemecology 12:125–131.

28. Jones, R. M., L. S. Collier, E. L. Neidle, and P. A. Williams. 1999. *areABC* genes determine the catabolism of aryl esters in *Acinetobacter* sp. strain ADP1. J. Bacteriol. 181:4568–4575.

29. Jones, R. M., V. Pagmantidis, and P. A. Williams. 2000. *sal* genes determining the catabolism of salicylate esters are part of a supraoperonic cluster of catabolic genes in *Acinetobacter* sp. strain ADP1. J. Bacteriol. 182:2018–2025.

30. Juni, E. 1972. Interspecies transformation of *Acinetobacter*: genetic evidence for a ubiquitous genus. J. Bacteriol. 112:917–931.

31. Juni, E., and A. Janik. 1969. Transformation of *Acinetobacter calcoaceticus* (*Bacterium anitratum*). J. Bacteriol. 98:281–288.

32. Kroon, P. A., M. T. Garcia-Conesa, I. J. Fillingham, G. P. Hazlewood, and G. Williamson. 1999. Release of ferulic acid dehydrodimers from plant cell walls by feruloyl esterases. J. Sci. Food Agric. 79:428–434.

33. Laemmli, U. K. 1970. Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature 227:680–685.

34. Lee, D., and C. J. Douglas. 1996. Two divergent members of a tobacco 4-coumarate:coenzyme A ligase (4CL) gene family. cDNA structure, gene inheritance and expression, and properties of recombinant proteins. Plant Physiol. 112:193–205.

35. Leidigh, B. J., and M. L. Wheelis. 1973. The clustering on the *Pseudomonas putida* chromosome of genes specifying dissimilatory functions. J. Mol. Evol. 2:235–242.

36. Lindermayr, C., B. Mollers, J. Fliegmann, A. Uhlmann, F. Lottspeich, H. Meimberg, and J. Ebel. 2002. Divergent members of a soybean (*Glycine max* L.) 4-coumarate:coenzyme A ligase gene family. Eur. J. Biochem. 269:1304–1315.

37. Lukaszewicz, M., I. Matysiak-Kata, A. Aksamit, J. Oszmianski, and J. Szopa. 2002. 14–3–3 protein regulation of the antioxidant capacity of transgenic potato tubers. Plant Sci. 163:125–130.

38. Lulai, E. C., and D. L. Corsini. 1998. Differential deposition of suberin phenolic and aliphatic domains and their roles in resistance to infection during potato tuber (*Solanum tuberosum* L.) wound-healing. Physiol. Mol. Plant Pathol. 53:209–222.

39. Maher, E. A., N. J. Bate, W. T. Ni, Y. Elkind, R. A. Dixon, and C. J. Lamb. 1994. Increased disease susceptibility of transgenic tobacco plants with suppressed levels of preformed phenylpropanoid products. Proc. Natl. Acad. Sci. USA 91:7802–7806.

40. Masai, E., Y. Harada, and X. Peng. 2002. Cloning and characterization of the ferulic acid catabolic genes of *Sphingomonas paucimobilis* SYK-6. Appl. Environ. Microbiol. 68:4416–4424.

41. Mitra, A., Y. Kitamura, M. J. Gasson, A. Narbad, A. J. Parr, J. Payne, M. J. Rhodes, C. Sewter, and N. J. Walton. 1999. 4-Hydroxycinnamoyl-CoA hydratase/lyase (HCHL)—an enzyme of phenylpropanoid chain cleavage from *Pseudomonas*. Arch. Biochem. Biophys. 365:10–16.

42. Moelgaard, P., and H. Ravn. 1988. Evolutionary aspects of caffeoyl ester distribution in dicotolydens. Phytochemistry 27:2411–2421.

43. Morawski, B., A. Segura, and L. N. Ornston. 2000. Substrate range and genetic analysis of *Acinetobacter* vanillate demethylase. J. Bacteriol. 182:1383–1389.

44. Narbad, A., and M. J. Gasson. 1998. Metabolism of ferulic acid via vanillin using a novel CoA-dependent pathway in a newly-isolated strain of *Pseudomonas fluorescens*. Microbiology 144:1397–1405.

45. Okamura, S., and M. Watanabe. 1982. Measurement of hydroxycinnamate acid ester hydrolase by the change of UV absorption. Agric. Biol. Chem. 46:297–299.

46. Okamura, S., and M. Watanabe. 1982. Purification and properties of hydroxycinnamic acid hydrolase from *Aspergillus japonicum*. Agric. Biol. Chem. 46:1839–1848.

47. Overhage, J., H. Priefert, and A. Steinbüchel. 1999. Biochemical and genetic analyses of ferulic acid catabolism in *Pseudomonas* sp. strain HR199. Appl. Environ. Microbiol. 65:4837–4847.

48. Parke, D., M. A. Garcia, and L. N. Ornston. 2001. Cloning and genetic characterization of *dca* genes required for β-oxidation of straight-chain dicarboxylic acids in *Acinetobacter* sp. strain ADP1. Appl. Environ. Microbiol. 67:4817–4827.

49. Parr, A. J., and G. P. Bolwell. 2000. Phenols in the plant and in man. The potential for possible nutritional enhancement of the diet by modifying the phenols content or profile. J. Sci. Food Agric. 80:985–1012.

50. Plaggenborg, R., A. Steinbüchel, and H. Priefert. 2001. The coenzyme A-dependent, non-β-oxidation pathway and not direct deacetylation is the major route for ferulic acid degradation in *Delftia acidovorans*. FEMS Microbiol. Lett. 205:9–16.

51. Rosazza, J. P., Z. Huang, L. Dostal, T. Volm, and B. Rousseau. 1995. Review: biocatalytic transformations of ferulic acid: an abundant aromatic natural product. J. Ind. Microbiol. 15:457–471.

52. Rosenberg, S. L., and G. D. Hegeman. 1969. Clustering of functionally related genes in *Pseudomonas aeruginosa*. J. Bacteriol. 99:353–355.

53. Sambrook, J., and D. Russell. 2001. Molecular cloning, a laboratory manual, 3rd ed. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.

54. Schöbel, B., and W. Pollmann. 1980. Further characterization of a chlorogenic acid hydrolase from *Aspergillus niger*. Z. Naturforsch. Sect. C J. 35:699–701.

55. Schöbel, B., and W. Pollmann. 1980. Isolation and characterization of a chlorogenic acid esterase from *Aspergillus niger*. Z. Naturforsch. Sect. C 35:209–212.

56. Segura, A., P. V. Bunz, D. A. D'Argenio, and L. N. Ornston. 1999. Genetic analysis of a chromosomal region containing *vanA* and *vanB*, genes required for conversion of either ferulate or vanillate to protocatechuate in *Acinetobacter*. J. Bacteriol. 181:3494–3504.

57. Stuible, H. P., and E. Kombrink. 2001. Identification of the substrate specificity-conferring amino acid residues of 4-coumarate:coenzyme A ligase allows the rational design of mutant enzymes with new catalytic properties. J. Biol. Chem. 276:26893–26897.

58. Venturi, V., F. Zennaro, G. Degrassi, B. C. Okeke, and C. V. Bruschi. 1998. Genetics of ferulic acid bioconversion to protocatechuic acid in plant-growth-promoting *Pseudomonas putida* WCS358. Microbiology 144:965–973.

59. Winter, M., and K. Hermann. 1986. Esters and glucosides of hydroxycinnamic acids in vegetables. J. Agric. Food Chem. 34:616–620.

60. Wood, D. W., J. C. Setubal, R. Kaul, D. E. Monks, J. P. Kitajima, V. K. Okura, Y. Zhou, L. Chen, G. E. Wood, N. F. Almeida, L. Woo, Y. C. Chen, I. T. Paulsen, J. A. Eisen, P. D. Karp, D. Bovee, P. Chapman, J. Clendenning, G. Deatherage, W. Gillet, C. Grant, T. Kutyavin, R. Levy, M. J. Li, E. McClelland, and E. W. Nester. 2001. The genome of the natural genetic engineer *Agrobacterium tumefaciens* C58. Science 294:2317–2323.

61. Yan, B., and R. E. Stark. 2000. Biosynthesis, molecular structure, and domain architecture of potato suberin: a ¹³C NMR study using isotopically labeled precursors. J. Agric. Food Chem. 48:3298–3304.
