
# REVIEWS

## DISRUPTION OF DIFFERENTIATION IN HUMAN CANCER: AML SHOWS THE WAY

**Daniel G. Tenen**

Although much is understood about the ways in which transcription factors regulate various differentiation systems, and one of the hallmarks of many human cancers is a lack of cellular differentiation, relatively few reports have linked these two processes. Recent studies of acute myeloid leukaemia (AML), however, have indicated how disruption of transcription-factor function can disrupt normal cellular differentiation and lead to cancer. This model involves lineage-specific transcription factors, which are involved in normal haematopoietic differentiation. These factors are often targeted in AML — either by direct mutation or by interference from translocation proteins. Uncovering these underlying pathways will improve the diagnosis and treatment of AML, and provide a working model for other types of human cancer, including solid tumours.

The focus in cancer research has traditionally been on the identification of oncogenes and tumour suppressors that regulate cellular proliferation and cell death. It is also often mentioned that disruption of normal differentiation is an important component of tumorigenesis, but little is understood about the genetic mechanisms of this process. One possibility is that disruption of cell-cycle control can itself block differentiation. A second alternative is that differentiation mechanisms are affected as important secondary events in carcinogenesis. A third possibility is that disruption of certain gene products affects both the cell cycle and differentiation, and a fourth is that both are affected independently. Although there is, at present, no definitive answer as to whether one or more of these mechanisms causes the block in differentiation that is observed in many neoplastic processes, recent evidence supports the third and fourth models as being important in ACUTE MYELOID LEUKAEMIA (AML).

AML is a disease that is characterized by an accumulation of granulocyte or monocyte precursors in the bone marrow and blood. It is also an excellent model for studying genetic regulation of differentiation and cancer progression. Researchers who study normal blood development and neoplasms have access to large numbers of malignant cells, powerful tools to isolate and characterize cellular subpopulations, and a number of cell-line models of different haematopoietic lineages. Early studies focused on the role of cytokines and growth factors in normal differentiation. Studies of AML — facilitated by the powerful and well-developed techniques of cytogenetics and FLUORESCENCE IN SITU HYBRIDIZATION (FISH) — have focused on consistent chromosomal translocations and the fusion proteins that are encoded by these factors.

In the past few years, a number of studies have pointed to the dominant role of lineage-specific transcription factors in normal haematopoietic differentiation. These studies predicted that the function of these transcription-factor pathways would be disrupted in AML. Recent studies have confirmed this hypothesis, showing that a number of cases of AML are not associated with a consistent chromosome translocation, but rather with small mutations in the coding region of these lineage-specific transcription factors. In addition, in many of those cases of AML that are associated with a common chromosome translocation, the resulting translocation product disrupts the expression and/or function of the same lineage-specific factors. In fact, the most common genetic mechanism that is associated with AML, as well as ACUTE LYMPHOCYTIC LEUKAEMIA (ALL),

**Harvard Institutes of Medicine, 77 Avenue Louis Pasteur, Boston, Massachusetts 02115, USA.**
e-mail: dtenen@caregroup.harvard.edu
doi:10.1038/nrc989

NATURE REVIEWS | CANCER
VOLUME 3 | FEBRUARY 2003 | 89

© 2003 Nature Publishing Group

Summary

- Loss of differentiation is an important component in the pathogenesis of many cancers.
- Acute myeloid leukaemia (AML) represents a salient example of a cancer that is characterized by a differentiation block.
- Specific haematopoietic transcription factors are crucial for differentiation to particular lineages during normal differentiation, and are controlled by specific patterns of expression and protein interactions.
- These same transcription factors are frequently disrupted in AML.
- Some mechanisms of disruption involve the effect of fusion proteins that are generated by chromosomal translocations on haematopoietic transcription factors.
- In other cases, in the absence of common translocations, the transcription factors themselves are mutated.
- Characterizing these transcription-factor abnormalities has already affected classification schemes based on patient outcome.
- These transcription-factor pathways represent important targets for therapeutic intervention.

ACUTE LYMPHOCYTIC LEUKAEAMIA (ALL). Involves early lymphoid progenitors, and, along with acute myeloid leukaemia, comprises the acute leukaemias. This cancer is more common in children than adults.

MYELOID In the haematopoietic system, myeloid refers to granulocytes and monocytes/macrophages and their precursors.

is the aberrant expression of a transcription factor or the production of an abnormal hybrid transcription factor¹. These results support the view that disruption of normal differentiation is a key component in the development of some types of cancer.

Haematopoietic differentiation Pluripotential haematopoietic stem cells — an extremely rare population (<0.1%) of nucleated bone-marrow cells—undergo a decision to either self-renew and remain pluripotent or to differentiate into immature but committed progenitor cells. These

Lymphoid pathway
![Diagram](#)
Myeloid pathway

Figure 1 | Role of transcription factors in haematopoietic development Long-term and short-term haematopoietic stem cells (HSCs) provide long-term (more than 3 months) and short-term reconstitution in lethally irradiated mice. The common lymphoid progenitor (CLP) gives rise to T and B cells, whereas the common myeloid progenitor (CMP) gives rise to granulocyte/macrophage progenitors (GMPs) and megakaryocyte/erythroid progenitors (MEPs). These progenitor populations are defined and can be isolated by multicolour fluorescence-activated cell sorting (FACS)⁵. Upregulation of the transcription factor PU.1 is essential for the transition from HSC to CLP, whereas downregulation of PU.1 is required for the differentiation of CMP to MEP. CCAAT/enhancer binding protein-α (C/EBPα) upregulation initiates the transition from CMP to GMP. One hypothesis is that the ‘default’ pathway involves differentiation of GMPs to monocytes and macrophages, which depends on PU.1 activity. In this model, C/EBPα subverts this default monocytic pathway and promotes differentiation into granulocytes — perhaps through inhibition of PU.1 function³¹.

Table 1 | Transcription factors involved in normal haematopoiesis

| Factor    | Expression                                                                 | Gene-target products                                                                                   | Comments                                                                                                                                                                                                 |
|-----------|--------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| AML1      | HSCs and most others                                                      | M-CSF receptor; T-cell receptor enhancer                                                              | Knockouts lack all definitive haematopoiesis; conditional knockouts develop moderate thrombocytosis                                                                                                      |
| GATA1     | HSCs, CMPs, MEPs, not in GMPs or lymphoid                                 | Erythropoietin receptor, and many others                                                              | Knockouts lack definitive erythroid cells                                                                                                                                                              |
| PU.1      | All progenitors; downregulated in erythroid and T cells                    | Receptors for GM-CSF, G-CSF and M-CSF, and many others                                                | Knockout leads to complete loss of macrophages and B cells, delayed development of T cells and granulocytes; the block occurs at the HSC to CLP transition and at the CMP stage                                    |
| C/EBPα    | HSCs, CMPs, GMPs, not in MEPs or lymphoid and cells                        | Receptors for G-CSF, IL-6, E2F c-MYC; primary granule proteins                                        | Knockout results in complete loss of granulocytic maturation, block at the CMP to GMP stage, and can induce granulocytic and block monocyctic differentiation of multipotent cell lines                                |
| C/EBPβ    | Most                                                                     | G-CSF, and others                                                                                    | Knockout shows not required for myeloid development but has role in macrophage activation; knock-in into the C/EBPα locus rescues granulopoiesis                                                             |
| C/EBPε    | Granulocytic and some lymphoid cells                                      | Secondary granule proteins                                                                           | Knockout blocks terminal granulocyte maturation and function                                                                                                                                          |

C/EBPα/β/ε, CCAAT/enhancer binding protein-α/β/ε; CLP, common lymphoid progenitor; CMPs, common myeloid progenitors; GM-CSF, granulocyte-monocyte colony-stimulating factor; GMPs, granulocyte/macrophage progenitors; HSCs, haematopoietic stem cells; IL-6, interleukin-6; M-CSF, macrophage colony-stimulating factor; MEPs, megakaryocyte/erythrocyte progenitors.

and will not be discussed here in detail, other than to compare it to the 'lineage-specific' transcription factors GATA1, PU.1 and C/EBPα<sup>13,14</sup>. One piece of data that implicates AML1 in myeloid leukaemias is that its DNA-binding activity is greatly enhanced by core binding factor-β (CBFβ) — the heterodimeric partner of AML1. Recently, a second mechanism has been described, in which CBFβ enhances the stability of the AML1 protein<sup>15</sup>. CBFβ is itself commonly involved in a different translocation that involves the inversion of chromosome 16.

GATA1 was the first 'lineage-specific' transcription factor to be described in detail, and its role in the development of erythroid and megakaryocytic lineages has been elucidated in a number of studies<sup>8,16</sup>. GATA1 was previously considered to be more of an erythroid than myeloid transcription factor, and as noted below inhibits PU.1 function to direct early progenitor cells towards erythroid and megakaryocytic differentiation<sup>17–20</sup> (FIG. 2). However, more recent models of haematopoiesis that use more precisely defined progenitor populations (FIG. 1) indicate that GATA1 participates in the differentiation of CMPs to megakaryocyte/erythroid progenitors (MEPs) and not GMPs. This role is supported by studies involving targeted disruption of regulatory elements that resulted in selective loss of erythroid development<sup>21</sup>. The relative expression levels of GATA1 and its co-activator(s), such as friend of GATA (FOG), are crucial to regulation of the differentiation of erythroid versus megakaryocytic cells<sup>22</sup>.

In terms of myeloid development, studies undertaken over the past decade have shown the importance of two transcription factors, PU.1 and C/EBPα. The biology of the PU.1 transcription factor places it slightly downstream of AML1 in the process of differentiation<sup>8–10,23,24</sup>. Whereas AML1 falls into the category of transcription factors that are involved in the development of

Figure 2 | PU.1 and GATA reciprocal antagonism during haematopoietic stem-cell lineage differentiation. In this simple binary model, a signalling event, perhaps mediated through cytokines and/or stromal interactions, changes the ratio of PU.1 and GATA proteins in a stem cell. These two proteins inhibit each other's function through different mechanisms. Increases in GATA protein inhibit PU.1 function by blocking the interaction of PU.1 with its co-activator c-JUN (see FIG. 3a). Increases in PU.1 protein lead to inhibition of GATA function through inhibition of GATA DNA binding. These transcription factors also autoregulate their own expression (dashed lines). In this model, the activity of one of these two regulatory molecules inhibits the ability of the other to activate the transcription of growth-factor receptors, such as those for granulocyte-monocyte colony-stimulating factor (GM-CSF) and erythropoietin (EPO), which facilitate differentiation along either the myeloid or erythroid pathways, respectively.

FLUORESCENCE-ACTIVATED CELL SORTING (FACS). A method that is used to separate and classify blood cells and to assess the expression of surface molecules on single cells.

granulocytic differentiation³². In summary, in normal myelopoiesis, PU.1 seems to have two well-defined functions. The first is to mediate an early role in the development of a multipotent myeloid precursor. The second is a later role in the development of monocytes/macrophages. Recent studies that involved conditional Pu.1-knockout mice have shown the block to be at the HSC to common lymphoid progenitor (CLP), and HSC to CMP transitions³³ (FIG. 1). PU.1 regulates almost every myeloid gene, including the receptors for granulocyte-macrophage colony-stimulating factor (GM-CSF), macrophage colony-stimulating factor (M-CSF) and granulocyte colony-stimulating factor (G-CSF)³⁴⁻³⁷. These are growth and survival factors, the receptors of which are specifically expressed on myeloid cells³⁴⁻³⁷.

In contrast to PU.1, C/EBPα has a more specific function in granulopoiesis. The main role of this LEUCINE ZIPPER transcription factor — the founding member of this family — in haematopoiesis is in the development of granulocytes. Non-conditional targeted disruption of C/EBPα results in a selective early block in granulocyte maturation, without affecting other haematopoietic lineages, including monocytes³⁸. Other studies have shown that upregulation of C/EBPα function results in induction of granulocytic development and inhibition of monocytic development from bipotential myeloid progenitors, and that C/EBPα can inhibit PU.1 function³¹,³². These results are consistent with a model in which PU.1 is required for the development of multipotent myeloid progenitors from HSCs, and in which the role of C/EBPα is to channel the PU.1-directed ‘default’ monocytic pathway to the alternative granulocytic direction (FIG. 1). Consistent with this model is the finding that analysis of adult haematopoiesis in conditional knockout models shows a block at the earlier HSC to CMP transition following disruption of PU.1, but a block at the later CMP to GMP transition in conditional C/Ebpα knockouts (K. Akashi and D.G.T., unpublished observations). However, as C/Ebpa-knockout mice do have monocytes³⁸, it is likely that the actual role of PU.1 and C/EBPα in progenitor-cell differentiation is more complicated, and that both have important functions in several steps during early myeloid differentiation³⁹. In contrast to PU.1, loss of C/EBPα affects expression of the G-CSF receptor, but not the receptors for GM-CSF or M-CSF³⁷.

How does C/EBPα function differently from PU.1? One mechanism is a clear difference in the cell-type-specific and differentiation-stage-specific expression patterns of the two transcription factors⁵,⁶,⁹,¹⁰. A second is in the different protein-protein interactions (FIGS 2,3), which are important not only in activation of gene targets, but also in the inhibition of alternative haematopoietic lineages¹⁸,²⁰. Finally, although both transcription factors activate many important myeloid-specific gene targets, including myeloid-specific receptors for myeloid CSF growth factors³⁷, C/EBPα, but not PU.1, has been shown to inhibit cell proliferation through several mechanisms. These include downregulation of p21 (REF. 40) and inhibition

Figure 3 | PU.1 and C/EBPα interacting proteins. **a** | The PU.1 amino-terminal transactivation domain (TAD) can interact with the basal transcription factors TFIIID, TATAA binding protein (TBP), GATA1 and GATA2, glucocorticoid receptor (GR) and heat-shock protein 90 (HSP90). The PEST (potential protein degradation) domain interacts with interferon regulatory factors (IRFs), including PIP/NF-EM5 and interferon consensus sequence binding protein (ICSBP). The carboxy-terminal DNA binding (ETS) domain interacts with c-JUN, GATA1, GATA2, CCAAT/enhancer binding protein-α (C/EBPα) and C/EBPβ, AML1 and herpes virus intermediate early protein 2 (IE2). These interactions can result in either positive (c-JUN) or negative (GATA1, C/EBPα) effects on PU.1 function. The site of interaction with the transcriptional co-activator CBP has not been mapped. **b** | The diagram shows the C/EBPα transactivation domains, which were defined by deletion studies of Nerlov *et al.* (TE1, TE2 and TE3)¹²⁴ and Friedman *et al.* (TAD1 and TAD2)¹²⁵. The basic zipper (bZip) domain mediates DNA binding as well as homo- and heterodimerization with other C/EBP proteins. The mRNA translated from the ATG at amino acid (AA) 1 encodes the functional 42-kDa form of the protein. The mRNA translated from the ATG at AA 120 encodes the 30-kDa dominant-negative form, which is found in some patients with AML. Listed below the diagram are the regions of protein interaction, which are thought to regulate the differentiation of granulocytes or other lineages. These include E2F (which mediates expression of c-MYC), the chromatin remodelling proteins SWI/SNF, and cyclin-dependent kinases CDK2 and CDK4.

haematopoietic cells from haemangioblasts (SCL is another), and disruption of AML1 results in loss of all definitive haematopoiesis²⁵,²⁶, most published studies are consistent with the concept that PU.1 is important for haematopoietic stem-cell differentiation. In mice, *Pu.1* is absolutely required for the development of macrophages and B cells, and disruption of Pu.1 also leads to delayed development of granulocytes and T cells²⁷,²⁸. With respect to the myeloid lineages, PU.1 seems to be required for the development of early multipotent myeloid precursors²⁹,³⁰.

Although the precise mechanisms that are involved in further maturation of these myeloid precursors into either monocyte/macrophage or granulocytic progenitors is not fully understood, an attractive model is that continued PU.1 expression leads to development of the ‘default’ pathway — that of monocytes/macrophages. Perturbation of this default pathway by C/EBP factors, which themselves can inhibit PU.1 function³¹, leads to

LEUCINE ZIPPER  
A protein-protein interaction domain that was first described in C/EBPα, but was used by a number of transcription-factor families to mediate protein-protein interactions.
of cyclin-dependent kinases${}^{41}$. The most important function of C/EBP$\alpha$ in granulocytic differentiation seems to be the inhibition of E2F pathways, resulting in downregulation of c-MYC and proliferation arrest${}^{42,43}$.

Results from studies with knockout mice are consistent with those of expression studies, which show low levels of expression of GATA1, PU.1 and C/EBP$\alpha$ in HSCs, and upregulation of PU.1 at the CMP and pro-B-cell stage, whereas C/EBP$\alpha$ is upregulated at the GMP stage. C/EBP$\alpha$ is expressed at the highest levels in GMPs, and, in contrast to PU.1, is not expressed by CLPs (the precursors of lymphoid cells). C/EBP$\alpha$ is downregulated as CMPs differentiate to MEPs${}^{5,6}$. GATA1, in contrast to the myeloid factors, is upregulated at the MEP stage${}^{5}$. In general, the results of both knockout and expression studies are consistent with a role for GATA1 in erythroid and megakaryocytic differentiation, for PU.1 in the development of B cells, monocytes and macrophages, and for C/EBP$\alpha$ in granulopoiesis.

**Induction of differentiation**

Recent studies have revealed a number of mechanisms by which transcription factors regulate differentiation, and these mechanisms provide a useful framework to discuss how haematopoietic differentiation proceeds in a largely irreversible fashion. These mechanisms include autoregulation, inhibition of alternative pathways, activation of lineage-specific genes, inhibition of proliferation and induction of apoptosis.

Although *in vivo* autoregulation remains to be shown, tissue-culture studies indicate that most, if not all, of these factors — including GATA1 (REF. 44), C/EBP$\alpha{}^{45}$ and PU.1 (REF. 46) — positively regulate their own expression. The proximal promoter of PU.1, although functional in tissue-culture studies, does not induce *in vivo* expression. However, a far-upstream element can induce expression of the promoter in transgenic studies. Interestingly, the crucial functional domain of this upstream enhancer contains a site that binds PU.1 *in vivo*, underscoring the importance of autoregulation of these factors${}^{47}$(Okuno Y. and D. G. T., unpublished observations).

A second important function of these lineage-specific transcription factors is inhibition of alternative pathways, and this is usually achieved through protein–protein interactions with other lineage-specific factors (*FIGS* 2,3). The best example of this type of mechanism was recently described by several laboratories, showing that GATA can inhibit PU.1, and PU.1 can inhibit GATA${}^{17,18,20}$ (*FIG*. 2). Therefore, in addition to promoting its own transcription, PU.1 prevents GATA from activating genes that are involved in alternative pathways (that is, erythropoiesis). Recent studies have shown antagonism between the myeloid factors PU.1, which is essential for monocyte/macrophage differentiation, and C/EBP$\alpha$, which induces granulocytic and blocks monocytic development${}^{31,32}$.

Of course, one of the most important functions of transcriptional regulators is to activate the expression of large sets of lineage-specific genes. There are two main

aspects involved in the activation of gene-expression programmes. First, many lineage-specific genes contain functional binding sites for lineage-specific transcription factors. For example, there are very few myeloid promoters that do not have a functional PU.1 site. Second, these factors might serve to maintain an open chromatin structure through the chromatin remodelling apparatus. So far, this has been most convincingly shown for C/EBP$\alpha{}^{48}$. Some lineage-specific target genes include those for growth-factor receptors. GATA has been shown to activate the transcription of genes encoding the erythropoietin receptor gene, whereas PU.1 has been shown to activate transcription of all three growth-factor receptors that are expressed by myeloid cells (GM-CSF, G-CSF and M-CSF). C/EBP$\alpha$ has been shown to activate the transcription of receptors for the granulocytic factors G-CSF and interleukin-6 (*IL-6*)${}^{36-38,49}$. These factors convey proliferation and anti-apoptotic signals to lineage-specific progenitors${}^{10}$.

An important function of one of these lineage-specific factors — C/EBP$\alpha$ — is inhibition of proliferation. In cells that are terminally differentiated and do not undergo proliferation, we would predict that differentiation factors might directly inhibit the cell-cycle apparatus. This is particularly true for C/EBP$\alpha$, which works through multiple mechanisms to activate p21, inhibit E2F and inhibit CDK function${}^{40,41,43,50}$. In fact, several studies indicate that inhibition of E2F might be the most crucial target of C/EBP$\alpha$ in the induction of granulocytic differentiation${}^{42,43,51}$, and that induction of other target genes could contribute to, but not be crucial for, granulocytic differentiation. However, inhibition of cell proliferation and induction of differentiation genes are not always correlated, at least in some granulocytic cell lines; it is possible to inhibit C/Eb$p\alpha$-induced differentiation of mouse 32D granulocytic cells without inhibiting C/Ebp$\alpha$-mediated inhibition of cell proliferation${}^{39,52}$. An important question is whether this will be a general characteristic of these lineage transcription factors, as there is no such data yet for factors such as GATA and PU.1 — the latter might only affect differentiation and not proliferation. Therefore, PU.1 might be an example of a factor that affects differentiation without affecting cell-cycle control or cell-cycle checkpoints. If mutation of PU.1 does indeed cooperate with other genetic events, resulting in loss of cell-cycle control in AML, it will be of interest to determine whether disruption of PU.1 is an early or late mutation. Some data indicate that GATA1 might function in a cell-cycle-dependent manner${}^{53}$. AML1, by contrast, has been reported to shorten the G1 phase of the cell cycle and to stimulate G1- to S-phase progression${}^{11,54,55}$.

Finally, an important question is whether these regulators induce apoptosis of granulocytic cells, which is the final ‘differentiation’ step. This is largely unknown, but is a very important question in terms of relevance to leukaemia. It might also be important in normal haematopoiesis, as some studies have estimated that most granulocytic cells fail to leave the bone marrow,
Table 2 | French-American-British (FAB) classification of AML

| FAB subtype | Description | Comments |
|-------------|-------------|----------|
| M0          | Undifferentiated | Myeloperoxidase negative; myeloid markers positive |
| M1          | Myeloblastic without maturation | Some evidence of granulocytic differentiation |
| M2          | Myeloblastic with maturation | Maturation at or beyond the promyelocytic stage of differentiation; can be divided into those with t(8;21) AML1-ETO fusion and those without |
| M3          | Promyelocytic | APL; most cases have t(15;17) PML-RARα or another translocation involving RARα |
| M4          | Myelomonocytic |  |
| M4<sub>Eo</sub> | Myelomonocytic with bone-marrow eosinophilia | Characterized by inversion of chromosome 16 involving CBFβ, which normally forms a heterodimer with AML1 |
| M5          | Monocytic |  |
| M6          | Erythroleukaemia |  |
| M7          | Megakaryoblastic | GATA1 mutations in those associated with Down's syndrome |

AML1, acute myeloid leukaemia 1; APL, acute promyelocytic leukaemia; PML, promyelocytic leukaemia; RARα, retinoic-acid receptor-α. Modified from REF. 65.

but instead die before entering the circulation<sup>56</sup>. So far, studies in cell lines and *in vivo* fail to show any important effect of expression or disruption of C/EBPα on apoptosis. An interesting possibility is that *C/EBPε*, which is downstream of C/EBPα in the granulocytic pathway, serves to induce apoptosis of granulocytes<sup>57</sup>.

A model of haematopoietic differentiation

It is important to consider models of haematopoietic differentiation to understand the role of transcription factors in haematopoiesis and leukaemia. Previous models proposed that haematopoietic lineage determination is driven extrinsically (through growth factors, stroma or other external influences)<sup>58,59</sup>, intrinsically (as described in 'stochastic' models)<sup>60</sup>, or both<sup>61</sup>. Our working hypothesis is that regardless of whether the environmental or stochastic models are invoked, transcription factors are the final common pathway that drives differentiation, and that haematopoietic commitment to different lineages is driven by alternative expression of specific combinations of transcription factors.

One recent model of haematopoietic lineage commitment proposes that stem cells or early multipotent progenitor cells can co-express low levels of lineage-specific transcription factors, such as GATA1 and PU.1 (REFS 9,62,63). Some event — and this might represent a stochastic aspect of differentiation — leads to an increase in expression and/or activity of one or more of these transcription factors. Examples of possible initiating (or destabilizing) events include interactions with stroma, or perhaps local differences in the concentrations of a growth factor that lead to activation of specific transcription-factor pathways<sup>59</sup> (FIG. 2). This relative increase in expression or function of one of these lineage-specific transcription factors then directs differentiation along a specific lineage by means of the mechanisms described above<sup>16–18,20,64</sup>.

The differentiation block

In AML, the accumulation of blast cells probably arises from a failure of myeloid blasts to mature, and therefore traditional classification systems have used cell morphology to define different subtypes of AML. These subtypes are categorized based on the stage at which normal differentiation blocked in the leukaemic blasts<sup>65</sup> (TABLE 2). More recently, enzyme activities, surface markers and specific chromosome translocations — detected by either polymerase chain reaction (PCR) and/or FISH — have improved classification schemes. However, it should be emphasized that none of these characteristics are based on the assessment of the function of normal differentiation pathways. A more simplistic model would show that the type of AML — and differentiation block manifested by the leukaemia — would be determined by the mechanism and stage at which the normal differentiation pathway was disrupted, and these mechanisms should eventually be incorporated into classification schemes. Such schemes seem warranted, as a number of studies have indicated that the overall proliferation rate of the leukaemic stem cell in AML is not necessarily greater than that of normal haematopoietic stem cells and/or myeloid blasts<sup>66,67</sup>, although it is likely that some abnormalities in cell-cycle control are present in leukaemic cells. There are proliferative diseases that occur without a differentiation block, such as chronic myelogenous leukaemia (*CML*). CML, however, eventually transforms into an acute leukaemia, as factors that regulate differentiation, such as C/EBPα, eventually become disrupted<sup>68</sup>.

Our main working hypothesis is that these transcription-factor pathways will be disrupted in most, if not all, myeloid leukaemias. However, several alternative mechanisms can be imagined, and these differences might be associated with different AML phenotypes. Such mechanisms include mutation, downregulation of expression and functional inhibition by other oncogenes. Recently, all of these mechanisms have been described in largely non-overlapping subtypes of AML. Consistent with this model, disruption of specific transcription-factor pathways in mice is associated with specific stages of differentiation blockade.

Table 3 | Medical Research Council (MRC) classification of AML

| Subtype       | MRC                                                                 | MRC and C/EBPα mutation status                                                                 |
|---------------|---------------------------------------------------------------------|---------------------------------------------------------------------------------------------|
| Favourable    | t(15;17), Inv(16), t(8;21)                                          | t(15;17), Inv(16) and t(8;21), or C/EBPα mutation without FLT3-ITD or adverse cytogenetics |
| Intermediate  | All others                                                         | C/EBPα mutation with FLT3-ITD or all others                                                 |
| Unfavourable  | del(5q)/–5, –7, 3q; complex karyotypic abnormalities                | del(5q)/–5, –7, 3q; complex karyotypic abnormalities                                         |

AML, acute myeloid leukaemia; C/EBPα, CCAAT/enhancer binding protein-α; ITD, internal tandem duplication. Modified from REF. 69.

---

### AML-associated mutations

With the exception of *AML1*, the genes that encode transcription factors that are involved in myeloid differentiation are generally not the sites of common chromosome translocations. Therefore, one prediction would be that genes encoding these factors might be mutated in a number of patients with AML. Furthermore, if common chromosome translocations inactivate myeloid transcription factors by other mechanisms, it might be predicted that transcription-factor mutations would be found in patients without translocations. This is precisely what has been observed in a limited set of studies. In fact, the pattern is already consistent enough such that certain mutations could be used in the classification of AML subtypes.

A different type of classification scheme has divided AML patients into 'favourable', 'intermediate' and 'unfavourable' subtypes based on survival<sup>69</sup> (TABLE 3). The favourable category consists of patients that have the common chromosome translocations t(15;17), t(8;21) and Inv16, or patients without these translocations that have specific C/EBPα mutations<sup>70</sup>. The newer classification categories are associated with specific effects on myeloid transcription factors, supporting the hypothesis that this will be a useful way to classify AML in the future.

Initial attention on *AML1* was focused on the nature of its translocation product, and the role of this protein in both myeloid (t(8;21) and AML1/Evi1) and lymphoid (t(12;21)) leukaemias<sup>13,14,71</sup> (TABLE 4). The

---

### Table 4 | Haematopoietic transcription factors associated with AML

| Factor     | Subtype                                      | Mutation                                                                                          |
|------------|----------------------------------------------|---------------------------------------------------------------------------------------------------|
| AML1       | FAB M0                                       | Often biallelic mutations                                                                         |
| AML1       | FAB M2                                       | t(8;21) AML1-ETO and AML1/Evi1                                                                   |
| GATA1      | FAB M7 associated with Down's syndrome       | Amino-terminal mutations similar to those seen with C/EBPα                                        |
| PU.1       | M0, M4, M5, M6                               | Not associated with common chromosome translocations; mostly M0 and M4 without Inv16               |
| C/EBPα     | M1, M2, few M4                               | Mutation not associated with t(8;21); amino-terminal dominant negative                            |
| C/EBPα     | M2 with t(8;21)                              | No mutation; downregulation at RNA level                                                          |
| C/EBPα     | CML myeloid-blast crisis                     | No mutation; downregulation at protein level                                                      |
| C/EBPα     | APL                                          | No downregulation; loss of DNA binding in cell lines; C/EBPβ and C/EBPε mediate ATRA response in APL |

FRENCH-AMERICAN-BRITISH (FAB). Refers to a classification system of acute myeloid leukaemia that is based on assessment of the stage of differentiation of blasts.

BIALLELIC MUTATIONS Mutations of both alleles of a gene, often resulting in no normal function.

AML, acute myelocytic leukaemia; APL, acute promyelocytic leukaemia; ATRA, all-trans retinoic acid; C/EBPα, CCAAT/enhancer binding protein-α; CML, chronic myelogenous leukaemia.

characterized as a specific subtype of myelomonocytic leukaemia (FAB M4<sub>Eo</sub>). Therefore, disruption of function of a single haematopoietic transcription factor (AML1–CBFβ) can lead to three different subtypes of AML — one that is characterized by the earliest differentiation block, and two by later-stage blocks.

Based on the biology of PU.1 in normal haematopoiesis, a prediction would be that disruption of *PU.1* in AML would contribute to a differentiation block at either a very early stage (FAB subtype M0), or to monocytic leukaemias. Adult mice with a conditional disruption of Pu.1 do develop a very early differentiation block — similar to that observed in patients with M0 AML (K. Akashi and D. G. T., unpublished observations). Although an earlier study that involved PCR-SINGLE-STRAND CONFORMATION POLYMORPHISM (PCR-SSCP) screening failed to detect *PU.1* mutations in 60 AML samples<sup>80</sup>, a more recent study that used direct sequencing detected *PU.1* mutations in 7% of 126 AML patients<sup>81</sup>. In general, the mutations had the following properties: with only a single exception, mutations were found in either the most immature FAB subtype (M0), myelomonocytic or monocytic (M4 or M5), or erythroleukaemia (M6) — consistent with the normal role of PU.1 in haematopoiesis; mutations were found in different domains of the *PU.1* gene; and, except for two patients, the mutations were heterozygous, resulting in loss of function in one allele, but not the creation of a dominant-negative protein. As *Pu.1*<sup>+/−</sup> mice<sup>82</sup> hardly have any phenotypic abnormalities, it is difficult to understand how loss of one allele results in a differentiation block.

Therefore, although it is likely that mutation of *PU.1* contributes to the differentiation block that is found in AML, more work is required to understand the mechanisms that are involved. As for *C/EBPα* mutations (see below), *PU.1* mutations were not found in patients with translocations that are usually associated with AML, such as t(15;17), t(8;21) or Inv16. This indicates that, in these leukaemias, the fusion proteins themselves might disrupt PU.1 function. Indeed, it seems that the t(8;21) fusion protein AML1-ETO can inhibit C/EBPα<sup>83</sup> and PU.1 (REF. 84), and therefore we would not expect to observe selection for mutations of these factors in this specific subtype of AML.

Three different groups have now reported the presence of mutations in *C/EBPα* in AML, with the frequency approximating 7–11% of all AML patients<sup>70,85,86</sup>. As predicted from the biology of *C/EBPα*, these mutations have mainly been detected in patients with ‘granulocytic’ forms of AML (M1 and M2). These patients do not have common translocations, such as t(8;21). The mutations can be largely divided into two common types.

In the first type, carboxy-terminal mutations disrupt the basic zipper region, which mediates DNA binding as well as homo- and heterodimerization with other *C/EBP* family members. In many cases, these mutations are associated with a second mutation in the other allele, which usually leads to loss of *C/EBPα* function. In the second type, amino-terminal frameshift mutations generally result in premature termination of the normal 42-kDa form of the *C/EBPα* protein. These mutations are found in approximately 50% of patients with a *C/EBPα* mutation<sup>70,85</sup>. Although different mutations are also found in the other *C/EBPα* allele, the premature termination mutations also seem to inhibit the remaining normal *C/EBPα* 42-kDa protein in a dominant-negative fashion<sup>85</sup>. Therefore, it is expected that even though patients might have a normal wild-type allele that accompanies this type of mutation, they would still lose *C/EBPα* function<sup>85</sup>.

As was the case with *PU.1*, mutations in *C/EBPα* are almost always found in patients whose leukaemic cells lack common translocations, such as t(8;21). These patients fall into the ‘intermediate’-risk AML subgroup (TABLE 3). However, within this subgroup, the presence of a *C/EBPα* mutation is a relatively good prognostic indicator<sup>70</sup>. In fact, those AML patients with a common translocation, such as t(15;17), t(8;21) or Inv16, or alternatively those with a *C/EBPα* mutation, represent patients with an AML resulting from a relatively simple set of genetic abnormalities, and perhaps for this reason represent a good prognostic group. Additional studies with a larger series of patients will be required to confirm these initial findings. Of course, studies of prognostic significance depend on the therapies available, and until specific therapies are available for patients with specific translocations and/or mutations, the value of detecting such mutations is limited.

The lack of evidence for a role of GATA1 in promoting myeloid differentiation and its role in megakaryocytic development indicates that if GATA1 were to be involved in AML, it would probably be involved in the FAB M7 (megakaryoblastic) subtype. A recent study reported that every paediatric patient that was analysed — with acute megakaryoblastic leukaemia associated with Down’s syndrome — harboured mutations in *GATA1*, whereas other M7 AML samples did not<sup>87</sup>. Interestingly, the mutations were similar to the amino-terminal mutations in *C/EBPα* that introduced a stop codon in the amino-terminal transactivation region. These mutations led to a loss of the functional 50-kDa GATA1 protein and synthesis of a shorter 40-kDa protein, which was initiated from a downstream in-frame ATG. This shorter protein binds DNA and interacts with the FOG1 co-activator but has decreased transactivation potential. It had previously been thought that the amino-terminal transactivation domain was not essential for haematopoiesis<sup>88,89</sup>, but these findings are compatible with a recent report that showed a role for the amino-terminal region in transgenic rescue studies *in vivo*<sup>90</sup>. In summary, these amino-terminal GATA1 mutations seem to define another subtype of FAB M7 AML.

---

**Fusion proteins**

If myeloid differentiation is mediated by myeloid transcription factors such as *PU.1* and *C/EBPα*, then, in most cases of AML, *PU.1* and/or *C/EBPα* function

---

SINGLE-STRAND  
CONFORMATION  
POLYMORPHISM  
(SSCP). A technique that is used  
to screen for mutations. It  
involves polymerase chain  
reaction, followed by resolution  
of amplified DNA by gel or  
capillary electrophoresis. This is  
quicker, but not as definitive as  
direct sequencing of genes of  

AML1-ETO  
The fusion protein that is  
generated by the t(8;21)  
translocation found in FAB M2  
acute myeloid leukaemia.

a

b

Figure 4 | The t(8;21) AML1-ETO fusion protein can disrupt normal C/EBPα function by two mechanisms.

a | In the first mechanism, C/EBPα can cooperate with AML1 in normal cells to activate the transcription of granulocytic-specific genes such as *defensin*. In addition, the human C/EBPα promoter demonstrates autoregulation.

b | The AML1-ETO fusion protein can bind to the AML1 site and can interact with and inhibit C/EBPα activation from such promoters — presumably by recruitment of corepressors. In addition, C/EBPα RNA is selectively decreased in t(8;21) patients, presumably through inhibition of positive autoregulation.

---

should be disrupted. As described above, most patients with AML that have mutations in PU.1 or C/EBPα do not have chromosome translocations such as t(8;21), t(15;17) or Inv16. So, do the fusion proteins that are generated by these translocations disrupt the function of PU.1 and/or C/EBPα?

The product of the translocation t(8;21) — the AML1–ETO fusion protein — has been shown to disrupt C/EBPα function<sup>83</sup>. Recently, downregulation of C/EBPα RNA has been shown to occur selectively in AML patients that carry the t(8;21) chromosome translocation<sup>91</sup> (FIG.4). The two mechanisms might be related, in that AML1-ETO could potentially inhibit C/EBPα expression through inhibition of autoregulation<sup>35,91</sup>. Furthermore, the AML1-ETO protein might also inhibit PU.1 (REF. 84).

A second example of a way in which a fusion protein could inactivate a myeloid transcription factor is mediated by the BCR-ABL fusion protein during the myeloid-blast crisis phase of CML. Chronic-phase CML is characterized by hyperproliferation but full maturation of functional granulocytes. After several years, chronic phase accelerates and then transforms into an acute phase, which can either be lymphoid or myeloid, and is characterized by a differentiation block and resistance to therapy. A screen of more than 60 CML patients, including those in myeloid-blast crisis, failed to uncover any mutations in C/EBPα (T. Pabst and D. G. T., unpublished observations). However, a recent study reported that the transition from chronic phase to myeloid-blast phase of CML was accompanied by downregulation of the C/EBPα protein, but not RNA, through a post-transcriptional mechanism<sup>68</sup>. These findings are consistent with the hypothesis that the block in differentiation that was found in de novo AML, or in syndromes resembling AML, such as CML in myeloid-blast crisis, involve disruption of normal myeloid transcription-factor pathways. In summary, the disruption of myeloid transcription-factor pathways in AML with common translocations or CML in myeloid-blast crisis indicates that these are shared biological pathways.

THERAPY-RELATED AML

In contrast to *de novo* AML, this leukaemia arises following (and presumably due to) chemotherapy and/or radiation therapy for a previous cancer.

DIFFERENTIATION THERAPY

In contrast to cytotoxic therapy, differentiation therapy is directed towards inducing normal differentiation of cancer rather than killing cancer cells.

ACUTE PROMYELOCYTIC LEUKAEAMIA (APL)

A subtype of acute myeloid leukaemia that is characterized by translocations involving the retinoic-acid receptor-α (RARα), usually t(15;17), resulting in production of the PML-RARα oncoprotein.

PML-RARα

The fusion protein that is formed by an in-frame fusion of the PML gene on chromosome 15 and the retinoic-acid receptor-α on chromosome 17. Associated with t(15;17) acute promyelocytic leukaemia.

Therapeutic implications

The information we have recently learned about the molecular basis of AML could have several important therapeutic implications. First, these findings have already led to the development of molecularly based classification schemes. Expression-profile analysis methods are also being developed to classify AML subtypes that have not been associated with a common translocation<sup>92</sup>. An alternative to classify these subtypes would be to associate them with defects in pathways that are associated with certain transcription factors. For example, FAB M0 patients have mutations in *AML1* and/or *PU.1*, M1 and M2 patients have dominant-negative C/EBPα mutations, M4 and M5 patients have *PU.1* mutations, and M7 patients have *GATA1* mutations. Further classifications could involve combinations with other genetic mechanisms, such as FLT3-ITD (internal tandem duplication).

It is not clear which method will be more biologically meaningful, or, alternatively, whether these classification schemes could be used together. Certain primary AMLs might have a relatively good prognosis because they are simple diseases with relatively few genetic abnormalities. Examples include those AMLs that are associated with t(15;17), t(8;21), Inv16 and C/EBPα mutations. Indeed, the presence of C/EBPα mutations has already been used to classify patients with a good prognosis<sup>70</sup>. At the opposite end of the spectrum are THERAPY-RELATED AMLS, which have many genetic abnormalities. Although there will be numerous attempts to relate the presence of specific genetic abnormalities, such as C/EBPα mutations, with prognosis<sup>70</sup>, such studies will have limited usefulness until we have specific therapies aimed to target each specific genetic pathway.

These types of studies will, however, improve DIFFERENTIATION-THERAPY approaches. There are several examples of therapeutics that already target myeloid transcription factors that are involved in differentiation. For example, the ability of all-*trans* retinoic acid (ATRA) to induce differentiation of ACUTE PROMYELOCYTIC LEUKAEAMIA (APL) cells might be mediated by C/EBP factors — most notably, C/EBPβ and C/EBPε<sup>93-95</sup>. Histone deacetylase (HDAC) inhibitors, alternatively, have been proposed to induce differentiation by inhibiting corepressors that are recruited by AML fusion proteins, such as PML-RARα and AML1-ETO. A recent report, however, has shown that these agents might mediate their differentiation effects through several mechanisms, including the relative levels of transcription factors, such as PU.1 (REF. 96).

Differentiation therapies can also be based on the restoration of normal or elevated levels of wild-type transcription factors. For example, induction of wild-type C/EBPα has been shown to induce differentiation in human AML cell line models, in mouse *in vivo* models and in lung cancer cell lines<sup>32,39,94,97</sup>. The development of newer, more effective gene-therapy vectors, such as lentiviral vectors, might facilitate replacement therapy in humans<sup>98</sup>. Alternatively, another method could use fusion peptides, such as VP22 or TAT, to facilitate entry into cells<sup>99,100</sup>. This could be applied either in treating bone marrow

ex vivo with differentiation agents, followed by bone-
marrow transplantation for re-introduction into the
patient, or by local therapies, such as instillation into
the lung by bronchoscopy<sup>101</sup>.

The identification of specific disruptions in trans-
scription-factor function supports efforts to carry out
drug screening to identify small-molecule modulators
of transcription factors. Although the requirement for
nuclear location makes targeting transcription factors
more difficult, a number of drugs have already been
identified that affect myeloid transcription factors,
such as ATRA and HDAC inhibitors. Luciferase-based
assays are being developed to screen for compounds
that increase factor expression and/or function. Of
course, drugs that can be used to treat AML, such as
those that target C/EBPα, might also be useful in treat-
ing other types of cancer, such as lung cancer. So, our
improved understanding of the mechanisms of differ-
entiation block in AML might well translate to
advances in therapies of other cancers.

**Future directions**

**Lineage-specific transcription factors.** What is the role
of lineage-specific transcription factors on stem cells,
and how does this relate to AML? Although most of
the blast cells from patients with AML clearly show a
lineage- and stage-specific differentiation block, sev-
eral studies support the idea that it is a disease that
also involves the stem-cell compartment<sup>66,102</sup>. So far,
the role of these transcription factors in normal
stem-cell biology has not been fully elucidated, and
even less information is available regarding the effects
of transcription-factor disruption on AML stem cells.
Future studies using conditional transcription-factor
knockouts, as well as involving HSC and progenitor
analysis in human leukaemias<sup>103</sup>, will help resolve
these questions.

**Other types of AML.** So far, mutations in these tran-
scription factors, or disruption of other types (such as
t(8;21) or CML myeloid-blast crisis), represent a minor-
ity of all AMLs. What about all the other cases? One
possibility is that myeloid transcription-factor function
is disrupted, but these defects have not yet been
described. It is relatively difficult to define the disrup-
tion of transcription-factor function in AML patient
samples in the absence of mutations. Even investigation
of downregulation is problematic in the absence of a
good control-cell population; establishing downregula-
tion in t(8;21) was greatly facilitated by the availability
of a ‘control’ AML population — that is, FAB M2
patients without the translocation. A second possibility
is that there are alternative pathways that do not depend
on PU.1 or C/EBPα, and therefore in these other AML
cases other genetic alterations block these alternative
pathways. Evidence for such alternative pathways has
been shown *in vitro*<sup>49,104,105</sup> but not *in vivo*.

**Other genetic abnormalities.** One simple model is
that induction of a differentiation block alone will be
sufficient to induce AML, without requiring other

genetic abnormalities. However, so far, several studies
indicate that although AML, in many cases, might be
a simpler disease than solid tumours, such as colon
cancer<sup>106</sup>, in terms of genetic abnormalities, it still
requires several genetic pathways to be disrupted. For
example, analysis of mouse models of Aml1-Eto indi-
cates that a second genetic pathway, in addition to the
fusion protein alone, is required<sup>7,12,107</sup>. Adult mice
with a conditional C/EBPα gene disruption develop a
profound early block in myeloid differentiation, but
lack many of the characteristics of AML (P. Zhang
and D. G. T., unpublished observations). One candi-
date cooperating mutation is ITD and activating
mutations in FLT3, which are associated with a high
percentage of human AML. Interestingly, FLT3-ITDs
occur in a significant percentage of patients that have
C/EBPα mutations, and the combination of these
mutations results in a worse prognosis for the
patient<sup>70</sup>. Another possibility is that such cooperating
mutations disrupt cell-death induction pathways,
which are also important in AML<sup>108,109–111</sup>.

**Loss of heterozygosity (LOH).** As noted above, in many
cases mutations in myeloid transcription factors are
found in one allele in the presence of a normal allele,
and many of these mutations do not act as dominant-
negative phenotypes. In the case of AML1, it is clear
that loss of one allele can contribute to the leukaemic
phenotype, as shown in patients with familial throm-
bocytopaenia that have the propensity to develop
AML<sup>72</sup>. However, in the case of PU.1 and C/EBPα,
there is no evidence (in humans) to show an effect of
LOH, and Pu.1 and C/Ebpα<sup>+/−</sup> mice have a minimal or
no demonstrable phenotype. Therefore, a challenge
will be to show the biological significance of heterozy-
gous mutations of PU.1 and C/EBPα. One approach is
to pursue family studies, which are difficult because of
the rarity of familial AML pedigrees. Alternative
approaches include investigating the effects of ‘knock-
ing-in’ a mutant form of these genes into mice and
investigating the haematopoietic consequences —
either as a single genetic abnormality or together with
other alterations, such as a FLT3-ITD.

**Protein-protein interactions.** As noted above and
described in several recent publications and
reviews<sup>16,20</sup>, interactions between transcription fac-
tors are important for their normal function. Mutations
in transcription factors that abrogate these
interactions have been implicated in human dis-
ease<sup>112</sup>. It is possible to propose several mechanisms
by which abnormalities in protein-protein interac-
tions could contribute to the block in differentiation
found in AML. For example, mutations of these fac-
tors could lead to a loss of interactions that con-
tribute positively to differentiation, such as mutations
in PU.1 that result in a loss of interaction with the
PU.1 co-activator c-JUN<sup>81,113</sup>. Alternatively, expres-
sion of an abnormal fusion protein — AML1-ETO
— inhibits PU.1 function by displacing the c-JUN
co-activator<sup>84</sup>.

Mouse models. Although this review is focused on human AML, it is clear that mouse models, including transgenic studies, knockouts and knock-ins, are crucial tools for understanding the pathogenesis of human diseases. Although inbred mice have been incredibly useful in studying shared aspects of human and mouse haematopoiesis, it should be emphasized that there are some subtle but significant differences between human and mouse haematopoietic gene regulation<sup>114</sup>. Of crucial relevance is the fact that AML1 binding sites are not conserved between homologous human and mouse genes (D. E. Zhang, unpublished observations), that the regulation of human and mouse C/EBPα are different<sup>45,115,116</sup>, and that the effect of AML1-ETO on C/EBPα expression could differ between human AML and mouse models<sup>117</sup>. Of particular interest is the finding that introduction of the C/EBPα 30-kDA peptide, which is generated as a dominant-negative mutation in AML<sup>85</sup>, has directly opposite effects when expressed in human versus mouse progenitors. In human CD34<sup>+</sup> cells, this peptide blocks granulopoiesis as predicted, but when expressed in mouse-lineage-negative cells, it promotes granulopoiesis (C. Stocking, manuscript in preparation). These differences should all be kept in mind when interpreting the results from mouse models of human AML, but often they are ignored.

Myeloid transcription factors in other human cancers. The same principles that have led to the discovery that these factors are involved in AML could apply to other human cancers. PU.1 is also a good candidate for involvement in B-cell leukaemias. Screening of more than 46 patients with pre-B-cell ALL failed to detect PU.1 mutations (B.U. Mueller and D. G. T., manuscript in preparation), but it is possible that other mechanisms lead to loss of PU.1 function in specific B-cell leukaemias.

So far, AML1 has been associated with the AML FAB subtype M0, the t(8;21) subtype of AML M2 and with TEL-AML1 fusions found in a significant percentage of pre-B-cell ALL<sup>118,119</sup>. A PCR-SSCP screen of cells from patients with other types of leukaemias or solid tumours did not reveal AML1 mutations<sup>80</sup>. However, a recent study has implicated another AML1 family member, RUNX3, in gastric carcinomas<sup>120</sup>.

C/EBPα is a good candidate for involvement in other types of human malignancies, as it is expressed in and inhibits the proliferation of several different tissue types, including hepatocytes, adipocytes and lung<sup>50</sup>. Both hepatocytes and type II alveolar lung cells undergo hyperproliferation in C/Ebpα-knockout animals<sup>121</sup>. Although PCR-SSCP did not identify C/EBPα mutations in 36 primary non-small-cell lung cancer samples, two C/EBPα mutations were detected in a screen of human lung cancer cell lines — including one dominant-negative form that was similar to that observed in AML patients (B. Halmos and D. G. T., unpublished observations). In addition, C/EBPα seems to be downregulated in a significant percentage of non-small-cell lung cancers<sup>97</sup>. It is also possible that C/EBPα function could be affected in other lung cancers by mechanisms that have been commonly observed with other genes that are involved in cell proliferation, including LOH and promoter methylation.

In a similar manner, although C/EBPβ was not found to be mutated in 20 breast cancer specimens<sup>80</sup>, other studies have implicated the role of C/EBPβ in normal breast development<sup>122</sup>. In addition, overexpression of a shorter, dominant-negative form of C/EBPβ — analogous to the shorter dominant-negative form of C/EBPα found in AML cells — has been reported in human breast cancer specimens<sup>123</sup>. Therefore, it is likely that disruption of C/EBPα and other C/EBP family members will be implicated in the pathogenesis of several human neoplasms through multiple mechanisms.


1. Nichols, J. & Nimer, S. D. Transcription factors, translocations, and leukemia. *Blood* **80**, 2953–2963 (1992).
2. Till, J. E. & McCulloch, E. A. Hemopoietic stem cell differentiation. *Biochim. Biophys. Acta* **605**, 431–459 (1980).
3. Ogawa, M. Differentiation and proliferation of hematopoietic stem cells. *Blood* **81**, 2844–2853 (1993).
4. Spangrude, G. J., Heimfeld, S. & Weissman, I. L. Purification and characterization of mouse hematopoietic stem cells. *Science* **241**, 58–62 (1988).
5. Akashi, K., Traver, D., Miyamoto, T. & Weissman, I. L. A clonogenic common myeloid progenitor that gives rise to all myeloid lineages. *Nature* **404**, 193–197 (2000). Describes the methodology for isolation and analysis of progenitors at precisely defined functional stages of differentiation.
6. Miyamoto, T. *et al.* Myeloid or lymphoid promiscuity as a critical step in hematopoietic lineage commitment. *Dev. Cell* **3**, 137–147 (2002).
7. Yuan, Y. *et al.* AML1-ETO expression is directly involved in the development of acute myeloid leukemia in the presence of additional mutations. *Proc. Natl Acad. Sci. USA* **98**, 10398–10403 (2001).
8. Shivdasani, R. A. & Orkin, S. H. The transcriptional control of hematopoiesis. *Blood* **87**, 4025–4039 (1996).
9. Tenen, D. G., Hromas, R., Licht, J. D. & Zhang, D.-E. Transcription factors, normal myeloid development, and leukemia. *Blood* **90**, 489–519 (1997).
10. Tenen, D. G. *Hematopoiesis* (ed. Zon, L. I.) 417–428 (Oxford University Press, New York, 2001).
11. Burel, S. A. *et al.* Dichotomy of AML1-ETO functions: growth arrest versus block of differentiation. *Mol. Cell. Biol.* **21**, 5577–5590 (2001).
12. De Guzman, C. G. *et al.* Hematopoietic stem cell expansion and distinct myeloid developmental abnormalities in a murine model of the AML1-ETO translocation. *Mol. Cell. Biol.* **22**, 5506–5517 (2002).
13. Speck, N. A. & Gilliland, D. G. Core-binding factors in haematopoiesis and leukaemia. *Nature Rev. Cancer* **2**, 502–513 (2002).
14. Nucifora, G. & Rowley, J. D. AML1 and the 8;21 and 3;21 translocations in acute and chronic myeloid leukemia. *Blood* **86**, 1–14 (1995).
15. Huang, G. *et al.* Dimerization with PEBP2β protects RUNX1/AML1 from ubiquitin-proteasome-mediated degradation. *EMBO J.* **20**, 723–733 (2001).
16. Orkin, S. H. Diversification of haematopoietic stem cells to specific lineages. *Nature Rev. Genet.* **1**, 57–64 (2000).
17. Rekhtman, N., Radparvar, F., Evans, T. & Skoultchi, A. I. Direct interaction of hematopoietic transcription factors PU.1 and GATA-1: functional antagonism in erythroid cells. *Genes Dev.* **13**, 1398–1411 (1999).
18. Zhang, P. *et al.* Negative cross-talk between hematopoietic regulators: GATA proteins repress PU.1. *Proc. Natl Acad. Sci. USA* **96**, 8705–8710 (1999).
19. Nerlov, C., Querfurth, E., Kulessa, H. & Graf, T. GATA-1 interacts with the myeloid PU.1 transcription factor and represses PU.1 dependent transcription. *Blood* **95**, 2543–2551 (2000).
20. Nerlov, C., Tenen, D. G. & Graf, T. *Hematopoiesis: A Developmental Approach* (ed. Zon, L. I.) 363–367 (Oxford University Press, New York, 2001).
21. Mcdevitt, M. A., Shivdasani, R. A., Fujiwara, Y., Yang, H. D. & Orkin, S. H. A ‘knockdown’ mutation created by cis-element gene targeting reveals the dependence of erythroid cell maturation on the level of transcription factor GATA-1. *Proc. Natl Acad. Sci. USA* **94**, 6781–6785 (1997).
22. Cantor, A. B., Katz, S. G. & Orkin, S. H. Distinct domains of the GATA-1 cofactor FOG-1 differentially influence erythroid versus megakaryocytic maturation. *Mol. Cell. Biol.* **22**, 4268–4279 (2002).
23. Zhang, D.-E. *et al.* *Molecular Aspects of Myeloid Stem Cell Development* (eds Wolff, L. & Perkins, A. S.) 137–147 (Springer, Heidelberg, 1996).
24. Tenen, D. G. *Transcription Factors: Normal and Malignant Development of Blood Cells* (eds Ravid, K. & Licht, J.) 103–115 (Wiley & Sons, New York, 2001).
25. Okuda, T., van Deursen, J., Hiebert, S. W., Grosveld, G. & Downing, J. R. AML1, the target of multiple chromosomal translocations in human leukemia, is essential for normal fetal liver hematopoiesis. *Cell* **84**, 321–330 (1996).
26. Wang, Q. *et al.* Disruption of the *Cbfa2* gene causes necrosis and hemorrhaging in the central nervous system and blocks definitive hematopoiesis. *Proc. Natl Acad. Sci. USA* **93**, 3444–3449 (1996).
27. Scott, E. W., Simon, M. C., Anastasi, J. & Singh, H. The transcription factor PU.1 is required for the development of multiple hematopoietic lineages. *Science* **265**, 1573–1577 (1994).
28. McKercher, S. R. *et al.* Targeted disruption of the *PU.1* gene results in multiple hematopoietic abnormalities. *EMBO J.* **15**, 5647–5658 (1996).

NATURE REVIEWS | CANCER

VOLUME 3 | FEBRUARY 2003 | 99

© 2003 Nature Publishing Group

29. DeKoter, R. P., Walsh, J. C. & Singh, H. PU.1 regulates both cytokine dependent proliferation and differentiation of granulocyte/macrophage progenitors. *EMBO J.* 17, 4456–4468 (1998). Shows that PU.1-deficient cells are blocked at an early stage of myeloid maturation, and that introduction of PU.1 (but not cytokine signalling) restores differentiation.

30. Anderson, K. L. *et al.* PU.1 and the granulocyte- and macrophage colony-stimulating factor receptors play distinct roles in late-stage myeloid cell differentiation. *Blood* 94, 2310–2318 (1999).

31. Reddy, V. *et al.* The granulocytic inducer C/EBPα inactivates the myeloid master gene PU.1: possible role in lineage commitment decisions. *Blood* 100, 483–490 (2002).

32. Radomska, H. S. *et al.* CCAAT/enhancer binding protein-α is a regulatory switch sufficient for induction of granulocytic development from bipotential myeloid progenitors. *Mol. Cell. Biol.* 18, 4301–4314 (1998).

33. Zhang, P. *et al.* C/EBPα deficiency blocks granulocytic differentiation at the common myeloid progenitor stage in both adult and fetal liver myelopoiesis. *Blood* 98, 792a (2001).

34. Hohaus, S. *et al.* PU.1 (Spi-1) and C/EBPa regulate the expression of the granulocyte-macrophage colony-stimulating factor receptor-α gene. *Mol. Cell. Biol.* 15, 5830–5845 (1995).

35. Zhang, D.-E., Hetherington, C. J., Chen, H. M. & Tenen, D. G. The macrophage transcription factor PU.1 directs tissue specific expression of the macrophage colony stimulating factor receptor. *Mol. Cell. Biol.* 14, 373–381 (1994).

36. Smith, L. T., Hohaus, S., Gonzalez, D. A., Dziennis, S. E. & Tenen, D. G. PU.1 (Spi-1) and C/EBPα regulate the granulocyte colony-stimulating factor receptor promoter in myeloid cells. *Blood* 88, 1234–1247 (1996).

37. Iwama, A. *et al.* Use of RDA analysis of knockout mice to identify myeloid genes regulated *in vivo* by PU.1 and C/EBPα. *Nucl. Acids Res.* 26, 3034–3043 (1998).

38. Zhang, D.-E. *et al.* Absence of granulocyte colony-stimulating factor signaling and neutrophil development in CCAAT enhancer binding protein α-deficient mice. *Proc. Natl Acad. Sci. USA* 94, 569–574 (1997). First demonstration that targeted disruption of C/EBPα results in an early granulocytic differentiation block.

39. Wang, X., Scott, E. W., Sawyers, C. L. & Friedman, A. D. C/EBPα bypasses granulocyte colony-stimulating factor signals to rapidly induce PU.1 gene expression, stimulate granulocytic differentiation, and limit proliferation in 32D cl3 myeloblasts. *Blood* 94, 560–571 (1999).

40. Timchenko, N. A., Wilde, M., Nakanishi, M., Smith, J. R. & Darlington, G. J. CCAAT/enhancer-binding protein α (C/EBPα) inhibits cell proliferation though the p21 (WAF-1/CIP-1/SDI-1) protein. *Genes Dev.* 10, 804–815 (1996).

41. Wang, H. *et al.* C/EBPα arrests cell proliferation through direct inhibition of Cdk2 and Cdk4. *Mol. Cell* 8, 817–828 (2001).

42. Johansen, L. M. *et al.* C-myc is a critical target for C/EBPα in granulopoiesis. *Mol. Cell. Biol.* 21, 3789–3806 (2001).

43. Porse, B. T. *et al.* E2F repression by C/EBPα is required for adipogenesis and granulopoiesis *in vivo*. *Cell* 107, 247–258 (2001). Shows the important link between the ability of C/EBPα to inhibit the cell-cycle-promoting gene E2F and the ability to induce granulocytic differentiation.

44. Zon, L. I., Youssoufian, H., Mather, C., Lodish, H. F. & Orkin, S. H. Activation of the erythropoietin receptor promoter by transcription factor GATA-1. *Proc. Natl Acad. Sci. USA* 88, 10638–10641 (1991).

45. Timchenko, N. *et al.* Autoregulation of the human C/EBPα gene by stimulation of upstream stimulatory factor binding. *Mol. Cell. Biol.* 15, 1192–1202 (1995).

46. Chen, H. M. *et al.* PU.1 (Spi-1) autoregulates its expression in myeloid cells. *Oncogene* 11, 1549–1560 (1995).

47. Li, Y. *et al.* Regulation of the PU.1 gene by distal elements. *Blood* 98, 2958–2965 (2001).

48. Pedersen, T. A., Kowenz-Leutz, E., Leutz, A. & Nerlov, C. Cooperation between C/EBPα TBP/TFIIB and SWI/SNF recruiting domains is required for adipocyte differentiation. *Genes Dev.* 15, 3208–3216 (2001).

49. Zhang, P. *et al.* Upregulation of interleukin 6 and granulocyte colony-stimulating factor receptors by transcription factor CCAAT enhancer binding protein α (C/EBPα) is critical for granulopoiesis. *J. Exp. Med.* 188, 1173–1184 (1998).

50. McKnight, S. L. McBindall: a better name for CCAAT/enhancer binding proteins? *Cell* 107, 259–261 (2001).

51. Felsher, D. W. & Bishop, J. M. Reversible tumorigenesis by MYC in hematopoietic lineages. *Mol. Cell* 4, 199–207 (1999).

52. Schuster, C. *et al.* The effects of Bcr-Abl on C/EBP transcription-factor regulation and neutrophilic differentiation are reversed by the Abl kinase inhibitor imatinib mesylate. *Blood* 101, 655–663 (2003).

53. Sposi, N. M. *et al.* Cell cycle-dependent initiation and lineage-dependent abrogation of GATA-1 expression in pure differentiating hematopoietic progenitors. *Proc. Natl Acad. Sci. USA* 89, 6353–6357 (1992).

54. Strom, D. K. *et al.* Expression of the AML-1 oncogene shortens the G(1) phase of the cell cycle. *J. Biol. Chem.* 275, 3438–3445 (2000).

55. Bernardin, F. & Friedman, A. D. AML1 stimulates G1 to S progression via its transactivation domain. *Oncogene* 21, 3247–3252 (2002).

56. Metcalf, D., Lindeman, G. J. & Nicola, N. A. Analysis of hematopoiesis in max 41 transgenic mice that exhibit sustained elevations of blood granulocytes and monocytes. *Blood* 85, 2364–2370 (1995).

57. Verbeek, W., Wächter, M., Lekstrom-Himes, J. & Koeffler, H. P. C/EBPε⁻/⁻ mice: increased rate of myeloid proliferation and apoptosis. *Leukemia* 15, 103–111 (2001).

58. Metcalf, D. Hematopoietic regulators: redundancy or subtlety. *Blood* 82, 3515–3523 (1993).

59. Roberts, R. *et al.* Heparan sulphate bound growth factors: a mechanism for stromal cell mediated haemopoiesis. *Nature* 332, 376–378 (1988).

60. Fairbairn, L. J., Cowling, G. J., Reipert, B. M. & Dexter, T. M. Suppression of apoptosis allows differentiation and development of a multipotent hemopoietic cell line in the absence of added growth factors. *Cell* 74, 823–832 (1993).

61. Just, U. *et al.* Upregulation of lineage specific receptors and ligands in multipotential progenitor cells is part of an endogenous program of differentiation. *Growth Factors* 9, 291–300 (1993).

62. Voso, M. T. *et al.* Inhibition of hematopoiesis by competitive binding of the transcription factor PU.1. *Proc. Natl Acad. Sci. USA* 91, 7932–7936 (1994).

63. Hu, M. *et al.* Multilineage gene expression precedes commitment in the hematopoietic system. *Genes Dev.* 11, 774–785 (1997).

64. Zhang, P. *et al.* PU.1 inhibits GATA-1 function and erythroid differentiation by blocking GATA-1 DNA binding. *Blood* 96, 2641–2648 (2000).

65. Bennett, J. M. *et al.* Proposals for the classification of the acute leukaemias. French-American-British (FAB) cooperative group. *Br. J. Haematol.* 33, 451–458 (1976). A landmark paper that established a classification system of AML on the basis of differentiation status.

66. Jordan, C. T. Unique molecular and cellular features of acute myelogenous leukemia stem cells. *Leukemia* 16, 559–562 (2002).

67. Guan, Y. & Hogge, D. E. Proliferative status of primitive hematopoietic progenitors from patients with acute myelogenous leukemia (AML). *Leukemia* 14, 2135–2141 (2000).

68. Perrotti, D. *et al.* BCR–ABL suppresses C/EBPα expression through inhibitory activity of hnRNP E2. *Nature Genet.* 30, 48–58 (2001). Describes yet another mechanism of inactivation of C/EBPα in AML — that involving the downregulation of C/EBPα through post-transcriptional mechanisms as the chronic (proliferative) phase transforms to acute leukaemia.

69. Grimwade, D. *et al.* The importance of diagnostic cytogenetics on outcome in AML: analysis of 1,612 patients entered into the MRC AML 10 trial. The Medical Research Council Adult and Children's Leukaemia Working Parties. *Blood* 92, 2322–2333 (1998).

70. Preudhomme, C. *et al.* Favorable prognostic significance of CEBPA mutations in patients with de novo acute myeloid leukemia: a study from the Acute Leukemia French Association (ALFA). *Blood* 100, 2717–2723 (2002). Relates the status of C/EBPα mutations, with and without FLT3-ITD, to prognosis, and proposes a new classification of AML based on C/EBPα mutation status.

71. Miyoshi, H. *et al.* t(8;21) breakpoints on chromosome 21 in acute myeloid leukemia are clustered within a limited region of a single gene, AML1. *Proc. Natl Acad. Sci. USA* 88, 10431–10434 (1991).

72. Song, W. J. *et al.* Haploinsufficiency of CBFA2 causes familial thrombocytopenia with propensity to develop acute myelogenous leukaemia. *Nature Genet.* 23, 166–175 (1999). Reports the haploinsufficiency of an important haematopoietic transcription factor that leads to a propensity to develop AML.

73. Osato, M. *et al.* Biallelic and heterozygous point mutations in the runt domain of the AML1/PEBP2αB gene associated with myeloblastic leukemias. *Blood* 93, 1817–1824 (1999). This study, along with reference 74, shows that AML1 could be involved in the pathogenesis of AML through a mechanism other than translocation, and that AML1 mutations were associated with a distinct differentiation subtype of AML.

74. Preudhomme, C. *et al.* High incidence of biallelic point mutations in the Runt domain of the AML1/PEP2aB gene in

75. Michaud, J. *et al.* In vitro analyses of known and novel RUNX1/AML1 mutations in dominant familial platelet disorder with predisposition to acute myelogenous leukemia (FPD/AML): implications for mechanisms of pathogenesis. *Blood* 99, 1364–1372 (2002).

76. Ichikawa, H. *et al.* AML1 is required for megakaryocytic maturation in adult hematopoiesis. *Blood* 100, 130a (2003).

77. Leroy, H. *et al.* Unlike AML1, CBFβ is not deregulated by point mutations in acute myeloid leukemia and in myelodysplastic syndromes. *Blood* 99, 3848–3850 (2002).

78. Wang, Q. *et al.* The CBFβ subunit is essential for CBFα2 (AML1) function *in vivo*. *Cell* 87, 697–708 (1996).

79. Sasaki, K. *et al.* Absence of fetal liver hematopoiesis in mice deficient in transcriptional coactivator core binding factor beta. *Proc. Natl Acad. Sci. USA* 93, 12359–12363 (1996).

80. Vegesna, V. *et al.* C/EBP-β, C/EBP-δ, PU.1, AML1 genes: mutational analysis in 381 samples of hematopoietic and solid malignancies. *Leuk. Res.* 26, 451
100. Schwarze, S. R., Ho, A., Vocero-Akbani, A. & Dowdy, S. F. *In vivo* protein transduction: delivery of a biologically active protein into the mouse. *Science* **285**, 1569–1572 (1999).

101. Jackson, E. L. *et al.* Analysis of lung tumor initiation and progression using conditional expression of oncogenic K-Ras. *Genes Dev.* **15**, 3243–3248 (2001).

102. Bonnet, D. & Dick, J. E. Human acute myeloid leukemia is organized as a hierarchy that originates from a primitive hematopoietic cell. *Nature Med.* **3**, 730–737 (1997).

103. Miyamoto, T., Weissman, I. L. & Akashi, K. AML1-ETO-expressing nonleukemic stem cells in acute myelogenous leukemia with 8;21 chromosomal translocation. *Proc. Natl Acad. Sci. USA* **97**, 7521–7526 (2000).

104. Collins, S. J., Ulmer, J., Purton, L. E. & Darlington, G. Multipotent hematopoietic cell lines derived from C/EBPα (-/-) knock out mice display macrophage-colony-stimulating factor, granulocyte-colony-stimulating factor, and retinoic acid-induced granulocytic differentiation. *Blood* **98**, 2382–2388 (2001).

105. Zhang, P. *et al.* Induction of granulocytic differentiation by two pathways. *Blood* **99**, 4406–4412 (2002).

106. Kinzler, K. W. & Vogelstein, B. Lessons from hereditary colorectal cancer. *Cell* **87**, 159–170 (1996).

107. Higuchi, M. *et al.* Expression of a conditional AML1-ETO oncogene bypasses embryonic lethality and establishes a murine model of human t(8;21) acute myeloid leukemia. *Cancer Cell* **1**, 75–87 (2002).

108. Traver, D., Akashi, K., Weissman, I. L. & Lagasse, E. Mice defective in two apoptosis pathways in the myeloid lineage develop acute myeloblastic leukemia. *Immunity* **9**, 47–57 (1998).

109. Guzman, M. L. *et al.* Nuclear factor-κB is constitutively activated in primitive human acute myelogenous leukemia cells. *Blood* **98**, 2301–2307 (2001).

110. Kogan, S. C. *et al.* BCL-2 cooperates with promyelocytic leukemia retinoic acid receptor alpha chimeric protein (PML-RARα) to block neutrophil differentiation and initiate acute leukemia. *J. Exp. Med.* **193**, 531–543 (2001).

111. Schimmer, A. D., Hedley, D. W., Penn, L. Z. & Minden, M. D. Receptor- and mitochondrial-mediated apoptosis in acute leukemia: a translational view. *Blood* **98**, 3541–3553 (2001).

112. Nichols, K. E. *et al.* Familial dyserythropoietic anaemia and thrombocytopenia due to an inherited mutation in GATA1. *Nature Genet.* **24**, 266–270 (2000).

113. Behre, G. *et al.* C-Jun is a JNK-independent coactivator of the PU.1 transcription factor. *J. Biol. Chem.* **274**, 4939–4946 (1999).

114. Okuno, Y. *et al.* Differential regulation of the human and murine CD34 genes in hematopoietic stem cells. *Proc. Natl Acad. Sci. USA* **99**, 6246–6251 (2002).

115. Christy, R. J., Kaestner, K. H., Geiman, D. E. & Lane, M. D. CCAAT/enhancer binding protein gene promoter: binding of nuclear factors during differentiation of 3T3-L1 preadipocytes. *Proc. Natl Acad. Sci. USA* **88**, 2593–2597 (1991).

116. Legraverend, C., Antonson, P., Flodby, P. & Xanthopoulos, K. G. High level activity of the mouse CCAAT/enhancer binding protein (C/EBPα) gene promoter involves autoregulation and several ubiquitous transcription factors. *Nucl. Acids Res.* **21**, 1735–1742 (1993).

117. Zhang, D.-E., Burel, S. A., Zhou, L., Hetherington, C. J. & Yuan, Y. AML1 and AML1 fusion protein AML1-ETO in myeloid gene regulation and leukemogenesis. *Blood Cells Mol. Dis.* **27**, 368–376 (2001).

118. Romana, S. P. *et al.* The t(12;21) of acute lymphoblastic leukemia results in a TEL-AML1 gene fusion. *Blood* **85**, 3662–3670 (1995).

119. Golub, T. R. *et al.* Fusion of the TEL gene on 12p13 to the AML1 gene on 21q22 in acute lymphoblastic leukemia. *Proc. Natl Acad. Sci. USA* **92**, 4917–4921 (1995).

120. Li, Q. L. *et al.* Causal relationship between the loss of RUNX3 expression and gastric cancer. *Cell* **109**, 113–124 (2002).

121. Flodby, P., Barlow, C., Kylefjord, H., Ahrlund-Richter, L. & Xanthopoulos, K. G. Increased hepatic cell proliferation and lung abnormalities in mice deficient in CCAAT/enhancer binding protein-α. *J. Biol. Chem.* **271**, 24753–24760 (1996).

122. Seagroves, T. N. *et al.* C/EBPβ, but not C/EBPα, is essential for ductal morphogenesis, lobuloalveolar proliferation, and functional differentiation in the mouse mammary gland. *Genes Dev.* **12**, 1917–1928 (1998).

123. Zahnow, C. A., Younes, P., Laucirica, R. & Rosen, J. M. Overexpression of C/EBPβ-LIP, a naturally occurring, dominant-negative transcription factor, in human breast cancer. *J. Natl Cancer Inst.* **89**, 1887–1891 (1997).

124. Nerlov, C. & Ziff, E. B. CCAAT/enhancer binding protein-alpha amino acid motifs with dual TBP and TFIIB binding ability co-operate to activate transcription in both yeast and mammalian cells. *EMBO J.* **14**, 4318–4328 (1995).

125. Friedman, A. D. & McKnight, S. L. Identification of two polypeptide segments of CCAAT/enhancer binding protein required for transcriptional activation of the serum albumin gene. *Genes Dev.* **4**, 1416–1426 (1990).

Acknowledgements

I wish to acknowledge the many colleagues who have contributed to these ideas, both by supporting them and by constructive disagreements, with particular thanks to G. Gilliland for telling me on a weekly basis why I am wrong. I also thank K. Akashi and C. Stocking for providing unpublished material. I apologize in advance to those investigators whose work or contributions were not cited due to space limitations or oversight on my part. This work was supported by National Institutes of Health grants.

### Online links

#### DATABASES

- Cancer.gov: http://www.cancer.gov/cancer_information/acute_lymphocytic_leukaemia | acute_myeloid_leukaemia | chronic_myelogenous_leukaemia
- LocusLink: http://www.ncbi.nih.gov/LocusLink/
  - *Aml1* | *AML1* | *C/EBPα* | *C/EBPε* | *Cbfb* | *CBFβ* | *ETO* | *FLT3* | *FOG* | *GATA1* | *G-CSF* | *GM-CSF* | *IL-6* | *JUN* | *M-CSF* | *Pu.1* | *PU.1* | *SCL*

#### FURTHER INFORMATION

- CancerCare information on AML:
  - http://www.cancercare.org/types/leukemia/index.asp
- MedLine information on AML:
  - http://www.nlm.nih.gov/medlineplus/leukemiaadultacute.html

Access to this interactive links box is free online.
