
Cell, Vol. 112, 329–341, February 7, 2003, Copyright ©2003 by Cell Press

# vriille, Pdp1, and dClock Form a Second Feedback Loop in the Drosophila Circadian Clock

Shawn A. Cyran,$^{1}$ Anna M. Buchsbaum,$^{1}$  
Karen L. Reddy,$^{2,5}$ Meng-Chi Lin,$^{1}$  
Nicholas R.J. Glossop,$^{3}$ Paul E. Hardin,$^{3}$  
Michael W. Young,$^{4}$ Robert V. Storti,$^{2}$  
and Justin Blau$^{1,*}$  

$^{1}$Department of Biology  
New York University  
100 Washington Square East  
New York, New York 10003  

$^{2}$Department of Biochemistry and Molecular Biology  
University of Illinois at Chicago  
1819 West Polk Street  
Chicago, Illinois 60612  

$^{3}$Department of Biology and Biochemistry  
University of Houston  
369 Science and Research Building 2  
Houston, Texas 77204  

$^{4}$Rockefeller University  
1230 York Avenue  
New York, New York 10021  

---

## Summary

The *Drosophila* circadian clock consists of two interlocked transcriptional feedback loops. In one loop, dCLOCK/CYCLE activates *period* expression, and PERIOD protein then inhibits dCLOCK/CYCLE activity. *dClock* is also rhythmically transcribed, but its regulators are unknown. *vriille* (*vri*) and *Par Domain Protein 1* (*Pdp1*) encode related transcription factors whose expression is directly activated by dCLOCK/CYCLE. We show here that VRI and PDP1 proteins feed back and directly regulate *dClock* expression. Repression of *dClock* by VRI is separated from activation by PDP1 since VRI levels peak 3-6 hours before PDP1. Rhythmic *vri* transcription is required for molecular rhythms, and here we show that the clock stops in a *Pdp1* null mutant, identifying *Pdp1* as an essential clock gene. Thus, VRI and PDP1, together with *dClock* itself, comprise a second feedback loop in the *Drosophila* clock that gives rhythmic expression of *dClock*, and probably of other genes, to generate accurate circadian rhythms.

---

## Introduction

Clock genes function together in molecular clocks that regulate circadian rhythms of behavior and physiology (reviewed by Harmer et al., 2001). All of the molecular clocks so far described are based on negative feedback loops in which the protein products of one or more clock genes inhibit transcription of their own gene(s).

---

gives an approximately 24 hr (circadian) rhythm in RNA and protein levels of some clock components, and these rhythms continue even in a constant environment.

The first description of a molecular clock feedback loop involved rhythmic transcription of *Drosophila period* (*per*, reviewed by Allada et al., 2001). PER protein binds the protein product of another rhythmically transcribed gene, *timeless* (*tim*), and the PER/TIM complex enters the nucleus where TIM is later degraded. PER protein then represses *per* and *tim* gene transcription by inhibiting the transcriptional activity of the dCLOCK/CYCLE heterodimer (dCLK/CYC). CYC is also known as dBMAL1. Thus, the initial production of *per* and *tim* RNAs is tied to inhibition of further RNA production ~12 hr later.

A second feedback loop interlocked to the first loop exists in circadian clocks as diverse as *Neurospora* and mouse (reviewed by Harmer et al., 2001). The second loop is a transcriptional loop in *Drosophila* and mammals and involves oscillations in RNA levels of one or both of the activators of the first loop: *dClk* in *Drosophila*, and both *Clock* and *Bmal1* in mammals (Glossop et al., 1999; Shearman et al., 2000; Lee et al., 2001). Recent studies have identified elements in the *Bmal1* promoter that are sufficient for cyclic transcription in vitro (Ueda et al., 2002a). These elements are bound by the transcriptional repressor REV-ERBα, which is required for rhythmic *Bmal1* transcription in vivo (Preitner et al., 2002). This second transcriptional clock loop adds precision to the circadian clock in mice (Preitner et al., 2002) and also offers a molecular mechanism for rhythmic expression of clock output genes in antiphase to CLK/BMAL1 (or dCLK/CYC) activated genes. It is not clear what proteins activate *Bmal1* transcription in mammals or which factors regulate rhythmic *dClk* expression.

In the *Drosophila* clock, *dClk* RNA and protein levels peak shortly after dawn in antiphase to maximal *per/tim* RNA levels shortly after dusk (Bae et al., 1998), suggesting different transcriptional regulation. Indeed, null mutations in clock genes have opposite effects on *per/tim* and *dClk* RNA levels. *per* and *tim* RNAs are constitutively high in *per⁰* and *tim⁰¹* mutant flies, and constitutively low in *Clk⁰rk* and *cyc⁰* mutant flies, while *dClk* RNA levels are constitutively low in *per⁰* and *tim⁰¹*, and high in *Clk⁰rk* and *cyc⁰* mutants (Glossop et al., 1999 and references therein). Arguably the simplest model to explain antiphase RNA peaks is that dCLK/CYC directly activates transcription of a rhythmically expressed *dClk* repressor. This repressor would be at low levels in *Clk⁰rk* and *cyc⁰* mutants, relieving repression of *dClk* and leading to high *dClk* RNA levels.

One candidate *dClk* repressor is *vrille* (*vri*), which encodes a basic leucine zipper (bZip) transcription factor (George and Terracol, 1997). *vri* is a direct target of dCLK/CYC (Blau and Young, 1999; McDonald and Rosbash, 2001) and is expressed in the lateral neuron pacemaker cells in the central brain that regulate rhythmic locomotor activity (Blau and Young, 1999). Flies with

---

*Correspondence: justin.blau@nyu.edu  
$^{5}$ Present address: Howard Hughes Medical Institute, University of Chicago, 5841 South Maryland Avenue, Chicago, Illinois 60637.
only one functional copy of *vri* have a short period locomotor rhythm while constitutive expression of *vri* causes either long period rhythms or arrhythmicity (Blau and Young, 1999). Pacemaker cells constitutively expressing *vri* have undetectable levels of *tim* RNA, which is consistent with VRI repression of *dClk*. In this study, we show that VRI protein levels accumulate with the expected phase of a *dClk* repressor, that overexpression of *vri* reduces *dClk* RNA levels in vivo independently of nuclear PER and TIM and that VRI directly binds and represses *dClk* promoter activity in vitro.

How then is *dClk* transcription activated? The DNA binding domain of VRI is almost identical to four mammalian bZip transcription factors expressed with a circadian rhythm (Lopez-Molina et al., 1997; Mitsui et al., 2001). Three of these—DBP, HLF, and TEF—are transcriptional activators that contain a PAR (proline and acidic rich) domain, while the fourth, E4BP4, has no PAR domain and is a transcriptional repressor (Cowell et al., 1992). We searched for *Drosophila* genes with homology to VRI and found only one—PAR domain protein 1 (*Pdp1*)—that was expressed in adult heads (McDonald and Rosbash, 2001; Ueda et al., 2002b, this study, and J.B., unpublished data). Indeed, *Pdp1* is a good candidate *dClk* activator for two reasons: (1) *Pdp1* is a direct target of dCLK/CYC and is expressed rhythmically in adult fly heads (McDonald and Rosbash, 2001; Ueda et al., 2002b); and (2) all *Pdp1* isoforms possess a transcriptional activation domain (Lin et al., 1997; Reddy et al., 2000). *Pdp1* was originally cloned by its ability to bind a regulatory site in the Tropomyosin I enhancer (Lin et al., 1997), but is widely expressed during development from at least four differently regulated promoters (Reddy et al., 2000).

We show here that although expression of *vri* and the *Pdp1ε* isoform is directly regulated by dCLK/CYC, *vri* and *Pdp1ε* RNA and proteins accumulate with different phases in vivo. Flies heterozygous for a *Pdp1* null mutation have a long period behavioral rhythm in contrast to the short period rhythms of *vri* heterozygotes. This led to the idea that VRI and PDP1 have opposite functions in the *Drosophila* clock. Indeed, overexpression of *vri* combined with reduced *Pdp1* levels synergistically increased period length. We also found that the clock stops in homozygous *Pdp1* null mutants with very similar molecular phenotypes to a gain-of-function *vri* mutation, including greatly reduced *dClk* expression. Finally, we used in vitro assays and found that VRI and PDP1ε compete for binding to the same site in the *dClk* promoter. On the basis of these findings, we propose that *vri*, *Pdp1ε*, and *dClk* are three essential components of the second feedback loop in the *Drosophila* clock that generates rhythmic *dClk* transcription. A unique feature of this loop is that dCLK/CYC activates transcription of its own repressor and activator simultaneously, but different phases of *vri* and *Pdp1ε* RNA and protein accumulation separate the times at which *dClk* expression is repressed and activated.

### Results

#### *Pdp1ε* Is a Clock-Controlled Gene
*vri* and *Pdp1* encode basic zipper transcription factors with highly conserved basic DNA binding domains (Figure 1A), suggesting they bind the same set of target genes. *vri* and *Pdp1* are both direct targets of dCLK/CYC (Blau and Young, 1999; McDonald and Rosbash, 2001). We first tested which *Pdp1* isoform(s) are clock-controlled since four alternative promoters and alternative splicing generate six *Pdp1* isoforms in vivo (Reddy et al., 2000). RNase protection probes specific for the different isoforms revealed that only *Pdp1ε* RNA levels oscillated in adult fly heads (data not shown).

Taking time points every three hours during a light-dark (LD) cycle revealed that *vri* and *Pdp1ε* RNA levels oscillated with similar phases to one another, but peak levels of *Pdp1ε* are not reached until 3–6 hr after the peak of *vri* RNA levels (Figure 1B). Oscillating *Pdp1ε* RNA levels were also seen in constant darkness (data not shown). Figure 1B shows that *Pdp1ε* RNA levels were high at both ZT2 and ZT14 in *per⁰* and *tim⁰¹* mutants. *Pdp1ε* RNA was low at both ZT2 and ZT14 in *Clk⁰rk* and *cyc⁰* mutants at levels close to the *Pdp1ε* RNA levels at ZT2 in wild-type flies (Figure 1C and data not shown). The phase of *Pdp1ε* RNA expression in wild-type flies, and the loss of rhythms in clock mutants, are consistent with *Pdp1ε* transcription being regulated in a similar manner to *per*, *tim*, and *vri* transcription (Allada et al., 1998; Rutila et al., 1998; Blau and Young, 1999). Indeed, analysis of the first 4 kb of sequence upstream of the start site of *Pdp1ε* transcription revealed six perfect CACGTG E boxes (data not shown), which are potential dCLK/CYC binding sites (Darlington et al., 1998). This is similar to the *vri* promoter, which has 4 E boxes (Blau and Young, 1999) in 2.4 kb. Thus, *Pdp1ε* is the clock-regulated *Pdp1* transcript.

The different phases of *vri* and *Pdp1ε* RNAs (Figure 1B) may reflect subtly different transcriptional activities of their promoters and/or different mRNA half-lives. Thus, the *vri* promoter could be stronger than the *Pdp1ε* promoter, and *vri* RNA may have a shorter half-life than *Pdp1ε* RNA. Indeed, the *vri* 3′ UTR contains seven copies of an AATAA element, likely to be associated with mRNA instability (Chen and Shyu, 1995).

### VRI and PDP1ε Proteins Oscillate with Different Phases
Next, we tested for oscillations in VRI and PDP1ε protein levels. The Western blot in Figure 1D was probed sequentially with antibodies to VRI, PDP1, and heat shock protein 70 (HSP70). Four blots from different extracts were quantitated in the graph in Figure 1D. VRI and PDP1ε protein levels oscillated robustly compared to constant levels of HSP70. Levels of VRI and PDP1ε proteins are detected with phases that largely reflect their RNA levels with VRI protein levels peaking at ~ZT15, and PDP1ε at ~ZT18. It may be even more significant that VRI was detectable at ZT9 and ZT12, when little or no PDP1ε was present, while PDP1ε was still detectable at ZT21 and ZT24, when there was little or no VRI protein. As expected, VRI and PDP1ε protein levels were constitutively high and low in head extracts isolated from *per⁰* and *Clk⁰rk* mutant flies, respectively (Figure 1E). The broad band associated with VRI was resolved to a tight band by treating the extract with phosphatase, indicating that VRI is phosphorylated in vivo (data not shown; Glossop et al., 2003), and this may influence VRI protein stability.
A Second Feedback Loop in the Drosophila Clock
331

**Figure 1. Pdp1ε Expression Is Clock-Controlled**

(A) Models of VRI and PDP1ε proteins show highly conserved basic DNA binding domains (b, light blue, and right image). Both proteins contain leucine zippers (Zip, dark blue). VRI has a glycine-serine rich domain (GS-rich, purple). PDP1ε has glutamine rich (Q-rich, red), alanine rich (A-rich, orange), and proline and acidic rich domains (PAR domain, green). TADα is the likely transactivation domain in PDP1α, a smaller PDP1 isoform (Lin et al., 1997).

(B) Pdp1ε and vri RNA levels oscillate in wild-type flies. y w flies were entrained in light-dark cycles and collected at zeitgeber times (ZT) shown. ZT is time in light-dark cycles with ZT0 lights on, ZT12 lights off. Levels of Pdp1ε (light blue) and vri (dark blue) RNA relative to non-cycling levels of n-synaptobrevin (n-syb) were assayed by Real Time PCR and quantitated as described in Experimental Procedures. Results are an average of two independent experiments (except one sample at ZT21 and ZT24 for Pdp1ε), and error bars depict standard error of the mean (SEM). The Pdp1ε RNA peak is significantly later than the vri RNA peak (p < 0.05, unpaired t test), and we have seen oscillations with the phases shown here in four other RNA series.

(C) Oscillations in Pdp1ε and vri RNA levels are blocked by mutations in per, tim, and dClk. The numbers 2 and 14 indicate ZT2 and ZT14, respectively. The experiment was conducted as in (B). Data are an average of two independent experiments.

(D) Western blot of protein extracts of fly heads collected at times shown during a light-dark cycle. A total 50 μg of protein extract were run in each lane, and the blot was sequentially probed with antibodies to VRI, PDP1ε, and HSP70. * denotes a non-specific band recognized by anti-VRI. Four sets of extracts were assayed and the relative levels of VRI (dark blue) and PDP1ε (light blue) at different times quantitated using NIH Image Software and shown in the graph below the blots. Overlaid is quantitative RT-PCR data for dClk RNA relative to n-syb (black) as in (B) for two experiments. PDP1ε protein levels peak significantly later than VRI (p < 0.02, unpaired t test).

(E) Normal oscillations of VRI and PDP1ε proteins are blocked in per⁰ and ClkJrk mutants. Representative blots are shown, with the same results seen in an additional blot for each protein. * denotes non-specific bands.

Figure 2. Clock-Dependent PDP1 Protein Oscillations in Pacemaker Cells

(A) Third instar larval brain pacemaker cells were identified by antibodies to PDF (green). Cells were costained for PDP1 (red), which is nuclear at ZT21 and undetectable at ZT10. Images were taken by confocal microscopy. Essentially identical results have been seen in more than 50 brains.

(B) PDP1 protein continues to oscillate in wild-type (WT) pacemaker cells of third instar larvae in constant darkness (black arrowhead, left images), but is constitutively high in $tim^{01}$ (middle images) and constitutively low in $cyc^0$ mutants (right images). White arrowhead indicates clock cells that oscillate in antiphase to pacemaker cells (Kaneko et al., 1997). These are also presumably present in $tim^{01}$ mutants, but their location cannot be unambiguously assigned in these images. CT indicates circadian time and reflects zeitgeber time from previous light-dark cycles. Results were consistent in at least 20 brain hemispheres analyzed for each genotype.

(C) Cryo-sections of adult fly heads frozen at ZT9 or ZT18 and stained for PDP1 (red) and ELAV (green). ELAV marks the nuclei of all neurons. Arrowheads indicate representative outer photoreceptor cell nuclei at the same position in top and bottom images. Clear changes in PDP1 levels in these nuclei can be seen between ZT9 and ZT18. Results were consistent in at least 16 fly heads analyzed at each time point.

PDP1 Is a Nuclear Protein in Pacemaker and Photoreceptor Cells

Direct regulation of $Pdp1\epsilon$ expression by dCLK/CYC made it likely that PDP1$\epsilon$ protein would be found in clock cells as previously shown for $vri$ (Blau and Young, 1999). We detected PDP1 protein at night (ZT21) but not by day (ZT10) in larval pacemaker cells, marked by the neuropeptide pigment dispersing factor (PDF, Figure

2A). Oscillation of PDP1 protein continued in constant darkness in wild-type pacemaker cells (Figure 2B, left images) but was blocked by null or dominant-negative mutations in the $per$, $tim$, $dClk$, and $cyc$ clock genes (Figure 2B, data not shown). These images also revealed that high levels of PDP1 are restricted to clock cells at this stage in the development of the fly brain. A robust oscillation in PDP1 levels was also visible

A Second Feedback Loop in the Drosophila Clock
333

in photoreceptor cells of the adult eye, which contain functional clocks. Figure 2C shows low PDP1 levels during the day at ZT9, and high levels in the middle of the night at ZT18. The oscillation is especially clear in the outer photoreceptor cell nuclei (see arrowheads, Figure 2C, top images). PDP1 at ZT18 colocalized with ELAV, which marks the nuclei of neurons (Robinow and White, 1991). Although our antibodies to PDP1 do not distinguish between the different PDP1 isoforms, RNase protection data and Western blots (Figure 1, data not shown) detected rhythmic expression of only *Pdp1ε* in fly heads—thus, mostly PDP1ε protein is detected in Figure 2C. PDP1 protein is thus rhythmically detectable in both central and peripheral clock cells and it is a nuclear protein, as predicted by its ability to activate transcription (Lin et al., 1997).

### A *Pdp1* Mutant Lengthens the Period of Behavioral Rhythms

Loss of one copy of *vri* shortens the behavioral period, while constitutive overexpression of *vri* causes either a long period or arrhythmicity (Blau and Young, 1999). We tested if *Pdp1* also regulates behavioral rhythmicity using a *Pdp1* mutant, *Pdp1<sup>P205</sup>*, which specifically deletes the entire *Pdp1* locus (see Experimental Procedures). *Pdp1<sup>P205</sup>* homozygotes are developmentally delayed, and are often normal size third instar larvae 14–21 days after egg laying, in contrast to their heterozygous siblings which are adults by this time. Some homozygous *Pdp1<sup>P205</sup>* mutants pupate, but only a very small proportion eclose, and these adult flies die within a day, preventing our testing the behavioral rhythms of flies lacking *Pdp1*. Full details of the *Pdp1<sup>P205</sup>* mutant will be published elsewhere (K.L.R. and R.V.S., unpublished data).

The rhythms of locomotor activity in constant darkness of adult flies heterozygous for *Pdp1<sup>P205</sup>* were compared to wild-type siblings, and the distribution of period lengths is shown in Figure 3A. All of the flies were rhythmic, and *Pdp1<sup>P205</sup>* heterozygotes showed an average period lengthening of ~0.5 hr. Representative actograms are shown in the top two images in Figure 3B. The altered period in *Pdp1<sup>P205</sup>* heterozygotes is similar in magnitude to that seen for *per* and *vri* heterozygotes (Baylies et al., 1987; Blau and Young, 1999). The opposite effects on period length of deleting one copy of *vri* and *Pdp1* indicate that VRI and PDP1 have opposite effects on the clock and suggest that the clock is sensitive to the ratio of VRI:PDP1.

We tested this idea by simultaneously overexpressing *vri* and reducing *Pdp1* dosage. *vri* overexpression in clock cells with the V1 UAS-*vri* transgene expressed from the *tim(UAS)-gal4* driver lengthens the circadian clock to ~25 hr (Figure 3B and Blau and Young, 1999). Although the *tim* promoter is cyclically activated and repressed over a 24 hr cycle, we have previously shown that the UAS elements included in the *tim* promoter and the stability of GAL4 protein give constitutively high

---

Figure 3. Mutation of *Pdp1* Lengthens the Behavioral Period

(A) Distribution of behavioral periods from locomotor assays in constant darkness for 10 days of *Pdp1<sup>P205</sup>* heterozygotes (bottom) and wild-type siblings (top). The average periods (*τ*) ± SEM for wild-type and *Pdp1<sup>P205</sup>* heterozygotes were 23.1 ± 0.1 and 23.6 ± 0.1, respectively. All flies assayed were rhythmic. The two groups are significantly different (*p* < 0.0001, two-sample unpaired t test).

(B) Representative actograms of *tim(UAS)-gal4/+* flies assayed in constant darkness with no additional mutations (top left), heterozygous for *Pdp1<sup>P205</sup>* (top right), with the UAS-*vri1* transgene (V1-bottom left) or V1 and heterozygous for *Pdp1<sup>P205</sup>* (bottom right). Average period (*τ*) ± SEM and number of flies assayed (*n*) is shown below each actogram. The expected period if V1 and *Pdp1<sup>P205</sup>* acted additively is 25.6. The observed results are significantly different from this expected result (*p* < 0.0001, one sample t test).

(C) Distribution of behavioral periods for flies described in (B) with the V1 transgene.

Cell
334

**A tim RNA**

|  | CT3 | CT9 | CT15 | CT21 |
| --- | --- | --- | --- | --- |
| WT | ![Image](image1) | ![Image](image2) | ![Image](image3) | ![Image](image4) |
| $Pdp1^{P205}$ | ![Image](image5) | ![Image](image6) | ![Image](image7) | ![Image](image8) |

**B PER protein**  
**C pdf RNA**

|  | WT | $Pdp1^{P205}$ | WT | $Pdp1^{P205}$ | $Clk^{Jrk}$ |
| --- | --- | --- | --- | --- | --- |
| CT13 | ![Image](image9) | ![Image](image10) | ![Image](image11) | ![Image](image12) | ![Image](image13) |
| CT1 | ![Image](image14) | ![Image](image15) |  |  |  |

**Figure 4. Pdp1 Is Required for a Functional Molecular Clock**

(A) Larvae were entrained in light-dark cycles for at least 2 days, and then shifted to constant darkness (DD). Brains of third instar larvae were dissected at the times shown on the first day in DD and processed for in situ hybridization with an antisense *tim* RNA probe. *tim* RNA cycles strongly in pacemaker cells of wild-type brains (arrowheads), but is either absent (26/52 brain hemispheres, three separate experiments) or only weakly detectable (26/52 brain hemispheres) in $Pdp1^{P205}$ mutants (arrowhead at CT15). The results are representative of at least 22 brain hemispheres assayed for each time point, with the exception of $Pdp1^{P205}$ at CT21 (12 hemispheres).

(B) Larvae were treated as in (A) except they were processed to detect PER protein in pacemaker cells at either CT13 on day 1 of DD or CT1 on day 2. Arrowhead shows PER immunoreactivity at CT1 in wild-type, but not in $Pdp1^{P205}$ mutants (0/16 brain hemispheres had detectable levels of PER).

(C) *pdf* RNA is detected in pacemaker cells in wild-type brains (white arrowhead, 20/20 brain hemispheres) but not in $Pdp1^{P205}$ (0/26 brain hemispheres) or $Clk^{Jrk}$ (0/30 brain hemispheres) mutants. Black arrowheads show *pdf* RNA in 4 cells at posterior of the larval CNS detected in all genotypes.

expression of a UAS transgene activated by *tim*(UAS)-  
*gal4* (Blau and Young, 1999). Removing one copy of  
*Pdp1* in a V1 background gave an average period of 27  
hr, which is significantly greater than the multiplicative  
increase in period length typically seen in flies with muta-  
tions in two different clock genes (Rothenfluh et al., 2000  
and references therein). Representative actograms are  
shown in Figure 3B and the distribution of periods in  
Figure 3C. Thus, wild-type rhythms are dependent on  
the correct ratio of VRI:PDP1.

**Pdp1 Encodes an Essential Clock Gene**

Although we could not test the behavioral rhythms of  
$Pdp1^{P205}$ homozygous adult flies, we tested the function  
of the molecular clock in pacemaker cells of second  

and third instar $Pdp1^{P205}$ larvae in constant darkness.  
Wild-type and $Pdp1^{P205}$ mutant larvae were entrained in  
light-dark cycles, shifted into constant darkness, and  
*tim* RNA levels in pacemaker cells were assayed by in  
situ hybridization. *tim* RNA levels cycled over the first  
day of constant darkness in wild-type larval brain pace-  
maker cells (Figure 4A, top row), with a peak at CT15,  
and were still weakly detectable at CT21, consistent  
with previous descriptions of the larval pacemaker clock  
(Kaneko et al., 1997). In contrast, we saw only a very  
weak *tim* RNA rhythm in $Pdp1^{P205}$ mutant larval pace-  
maker cells (Figure 4A, bottom row). We selected  
$Pdp1^{P205}$ brains with the highest levels of *tim* RNA for  
this figure. The brains shown have very low *tim* RNA  
levels at CT15 and undetectable levels at the other three

times. However, half of $Pdp1^{P205}$ mutant brain hemispheres (26/52) had no detectable $tim$ RNA in pacemaker cells at CT15. As an independent measure of the clock, we assayed for PER protein in $Pdp1^{P205}$ mutants. No PER protein was detected in $Pdp1^{P205}$ mutants at either the peak (CT1) or the trough (CT13) of the wild-type PER protein oscillation (Figure 4B). Since the free-running molecular clock stops in the absence of $Pdp1$, we conclude that $Pdp1$ encodes an essential clock gene.

The presence of low levels of $tim$ RNA at CT15 in 50% of brain hemispheres and of low levels of TIM protein in light-dark cycles (data not shown) indicated that the $Pdp1^{P205}$ mutation does not affect the viability of pacemaker cells. We also tested $pdf$ expression to attempt to verify the presence of pacemaker cells. $pdf$ RNA is detected in the 4 pacemaker neurons in each brain lobe and in 4 cells at the posterior extremity of the CNS in wild-type brains, which are not clock cells (Blau and Young, 1999; Park et al., 2000; and Figure 4C, left image). However, $pdf$ RNA was only detected in the posterior cells in $Pdp1^{P205}$ mutants (Figure 4C, middle image). This phenotype had only previously been seen with loss of dCLK function in $Clk^{Jrk}$ mutants (Blau and Young, 1999; Park et al., 2000; and Figure 4C, right image) and suggested that dCLK activity is reduced in $Pdp1^{P205}$ mutants.

VRI and PDP1 Regulate $dClk$ Expression In Vivo

Overexpression of $vri$ and loss of $Pdp1$ produce almost identical molecular clock phenotypes to one another in larval pacemaker cells (Blau and Young, 1999 and Figure 4), and these phenotypes are also similar to loss of dCLK function in $Clk^{Jrk}$ mutants (Allada et al., 1998). Figure 1D shows that $dClk$ RNA levels are at their lowest when VRI protein levels are highest and start to rise as VRI levels fall and PDP1ε levels rise. Therefore, we tested whether $vri$ and $Pdp1$ regulate $dClk$ expression in vivo.

First, $vri$ was expressed in clock cells via the $tim(UAS)$-gal4 driver and the strongest UAS-$vri$ transgene, V3 (Blau and Young, 1999). We previously showed that this causes constitutively high $vri$ expression with RNA levels between 1 and 2.5 times wild-type peak $vri$ RNA levels (Blau and Young, 1999). The results in Figure 5A show that $dClk$ RNA levels oscillated with an ~3-fold amplitude in flies with the $tim(UAS)$-gal4 driver in constant darkness as they do in wild-type flies (Bae et al., 1998). However, constitutive expression of $vri$ reduced the amplitude of the $dClk$ RNA oscillation in adult head RNA.

(D) Real-time PCR assays to measure $dClk$, $Pdp1\epsilon$, and $n$-syb RNA levels in third instar larvae of wild-type and $Pdp1^{P205}$ homozygous mutants. y axis is fluorescence, and x axis is cycle number. The top three traces in $dClk$ are from wild-type larvae (+) and show that a $dClk$ amplicon becomes detectable at ~cycle 25. In contrast, the $dClk$ amplicon never moves into the exponential phase in $Pdp1^{P205}$ homozygous mutants ($P$). Very similar results were seen when these samples were re-assayed for $Pdp1\epsilon$ RNA. RNA levels were controlled by amplification of $n$-syb, which moved into the exponential phase at cycle 25.5 ± 0.2 for wild-type ($n = 3$) and cycle 26.7 ± 0.2 for $Pdp1^{P205}$ homozygous mutants ($n = 3$). These differences correspond to an ~2-fold difference in neuronal RNA. Essentially identical results to those shown here for CT15 were seen in three samples at CT3; in no case did the $dClk$ or $Pdp1\epsilon$ RNA amplicon enter the exponential phase in $Pdp1^{P205}$ homozygotes.

dClk RNA levels in V3 flies were less than half the normal peak levels of dClk RNA at CT2, and at a similar level as wild-type flies at CT14 when dClk RNA levels normally reach their minimum.

vri-mediated repression of dClk could occur directly by VRI repressing dClk promoter activity or indirectly via decreased per and tim expression seen when vri is overexpressed (Blau and Young, 1999). We thus tested whether VRI can repress dClk independently of nuclear PER and TIM by overexpressing vri in a per0 background, in which TIM is cytoplasmic and PER is absent (see Allada et al., 2001). vri RNA levels were increased ~3-fold by tim-gal4 driven expression of V3 in a per0 background (Figure 5B, left graph). This resulted in a >2-fold decrease in dClk RNA (Figure 5B, right graph), which is already at low levels in per0 (Bae et al., 1998). Thus, overexpression of vri can repress dClk independently of nuclear PER and TIM.

We also tested whether removing one copy of Pdp1 affects the dClk RNA oscillation. Figure 5C shows that dClk RNA levels in Pdp1P205 heterozygotes were reduced ~2-fold compared to wild-type flies at each time point. Thus, the simplest explanation for the residual (< 2-fold) oscillation of dClk RNA levels in V3 flies is due to competition between VRI and PDP1ε. dClk RNA levels are also higher in flies heterozygous for a loss-of-function vri mutation than in wild-type flies (Glossop et al., 2003). Thus, altering the ratio of vri:Pdp1 affects dClk expression in otherwise wild-type flies as shown earlier for behavioral rhythms.

dClk expression was also tested in RNA isolated from Pdp1P205 homozygous third instar larvae. The results in Figure 5D show that dClk was barely detectable in Pdp1P205 mutants compared to wild-type third instar larvae. RNA was assayed by quantitative real-time PCR, and fluorescence (y axis) versus cycle number (x axis) is shown for three samples for each genotype in Figure 5D. We also confirmed that Pdp1P205 larvae do not express detectable Pdp1ε RNA (Figure 5D).

VRI and PDP1 Compete for Access to the dClk Promoter

The sequence upstream of the major start site of dClk transcription (Experimental Procedures) contains an average of three or four experiments each performed in duplicate or triplicate. PDP1ε and VRI-VP16 significantly activated dClk-luc compared to tk-luc (p < 0.02 and p < 0.01, respectively, two-sample unpaired t test).

(C) Assays were conducted as in (B) except that reporters had three copies of wild-type or mutant C3 sequences (C3 or C3m) fused to tk-luc. Titration were performed with CMV-PDP1ε (0, 50, and 200 ng) and CMV-VRI-VP16 (0, 2.5, and 10 ng). Results are an average of three experiments each performed in duplicate. PDP1ε and VRI-VP16 significantly activated C3-tk-luc compared to C3m-tk-luc (p < 0.005 and p < 0.02, respectively, two-sample unpaired t test).

(D) Direct binding of in vitro translated VRI and PDP1ε to radiolabeled C3 oligonucleotide in the absence (-), or presence of 100-fold excess unlabeled C3 or C3m competitor. Binding of VRI and PDP1ε to C3 was seen four times.

(E) Cells were transfected as in (C) with 100 ng CMV-PDP1ε where indicated and a titration of CMV-VRI (0, 250, 500, and 1000 ng). Data are an average of three experiments each performed in duplicate. VRI repressed PDP1ε-dependent activation in a dose-dependent manner (p < 0.02 for 250 ng and p < 0.005 for 500 and 1000 ng, two-sample unpaired t test).

A Second Feedback Loop in the Drosophila Clock
337

![](image.png)

Figure 7. A Two-Loop Model for the *Drosophila* Clock

Two interconnected transcription feedback loops lie at the core of the *Drosophila* molecular clock. In one loop, dCLK and CYC directly activate transcription of *per* and *tim* by binding their promoters. Inhibition of dCLK/CYC activity is mediated by TIM transporting PER into the nucleus. dCLK/CYC also directly activate *vri* and *Pdp1ε* transcription. *dClk* transcription is first repressed by VRI, and then activated by PDP1ε. Repression and activation of *dClk* are separated by the different phases of VRI and PDP1ε proteins. Removal of PER in the early morning frees dCLK/CYC to resume transcription of *per*, *tim*, *vri*, and *Pdp1ε*, thus restarting both loops simultaneously.

number of potential VRI and PDP1 binding sites—C1, C2, and C3 in Figure 6A—based on binding sites identified for this protein family (Cowell et al., 1992; Falvey et al., 1996; Lin et al., 1997). This suggested that the effects on *dClk* expression in *vri* and *Pdp1* mutants could be explained by VRI and PDP1ε directly regulating *dClk* transcription. This hypothesis was tested in vitro.

A *dClk*-luciferase reporter gene (*dClk*-luc) was constructed by fusing a firefly luciferase reporter gene to 3.2 kb of *dClk* genomic DNA containing 3 kb directly upstream of the major transcription start site and 200 bp downstream. *dClk*-luc was transfected into mammalian HEK-293 cells with an expression vector for *Pdp1ε*. The results on the left in Figure 6B show that PDP1ε activated *dClk*-luc to a maximum of ~9-fold, but only very weakly affected a control luciferase reporter controlled by a minimal herpes simplex virus thymidine kinase promoter, tk-luc (1.6-fold). tk-luc was used as a negative control because of a similar basal activity as *dClk*-luc, and it is unlikely that the viral tk promoter is regulated by clock proteins in vivo.

It is often difficult to detect repression before a promoter is activated, and this was the case for VRI (data not shown). To counter this, VRI was converted into a transcriptional activator by fusing the viral VP16 activation domain to either the N- or C-terminal of VRI. VRI-VP16 (right graph, Figure 6B) and VP16-VRI (data not shown) strongly activated *dClk*-luc but not tk-luc. Thus, both PDP1ε and VRI can bind the *dClk* promoter in vitro.

Next, we tested the ability of PDP1ε and VRI-VP16 to activate transcription from reporter plasmids containing three copies of the C1, C2, or C3 sites inserted into tk-luc. Only the C3-tk-luc reporter was strongly activated by PDP1ε and VRI-VP16 (Figure 6C and data not shown). This activation is specific since mutating four of the ten bases in C3 to C3m (Figure 6A) rendered the reporter non-responsive to PDP1ε and VRI-VP16 (Figure 6C).

Gel shift analysis detected direct binding of in vitro translated VRI and PDP1ε to C3, which was specific since it was competed by an excess of unlabeled C3 oligo, but not by excess C3m (Figure 6D). The VRI gel shift was not seen in a VRI mutant lacking its DNA binding domain (data not shown). Two independent in vitro assays lead to the conclusion that PDP1ε and VRI can both bind the C3 site in the *dClk* promoter.

The behavioral data in Figure 3 indicated that the

correct ratio of VRI:PDP1 is essential for wild-type rhythms and led to the suggestion that VRI and PDP1 compete to regulate the same step in the molecular clock. The finding that VRI and PDP1ε can bind exactly the same site in the *dClk* promoter suggested a molecular mechanism for how the proteins might compete. We tested this idea by transfecting cells with a moderate dose of PDP1ε, which activated C3-tk-luc ~30-fold (Figure 6E). Cotransfected VRI repressed PDP1ε-dependent transactivation. This conclusion is supported by the lack of an effect with the C3m-tk-luc reporter. Thus, VRI and PDP1ε compete for binding to the same site in the *dClk* promoter. The simplest model arising from the data in this figure is that mutations in *vri* and *Pdp1* affect *dClk* RNA levels in vivo because VRI and PDP1ε proteins directly regulate *dClk* transcription in opposite ways: VRI represses and PDP1ε activates the *dClk* promoter.

Discussion

Current models of the *Drosophila* circadian oscillator are based upon rhythmic activation of *per/tim* transcription by cycling levels of dCLK/CYC, and rhythmic repression of *per/tim* transcription by cycling levels of PER/TIM (Allada et al., 2001). While these models explain PER and TIM oscillations, the molecular mechanisms underlying cycling of dCLK/CYC were unknown. Here, we identify VRI as a rhythmically expressed *dClk* repressor and PDP1ε as a rhythmically expressed *dClk* activator. We show that VRI and PDP1ε directly regulate *dClk* transcription by binding the same site in the *dClk* promoter. We also demonstrate that *Pdp1* is required for circadian clock oscillation and for *dClk* expression, thus establishing it as a novel and essential clock gene. VRI and PDP1ε proteins accumulate with a phase delay that presumably underlies sequential repression and activation of *dClk* transcription. Thus, VRI, PDP1ε, and dCLK form a second feedback loop in the circadian oscillator responsible for regulating rhythms in dCLK/CYC levels.

A Two-Loop Model

A second feedback loop in the *Drosophila* clock, interlocked to the first feedback loop, was predicted by Glossop et al. (1999) to explain antiphase rhythms of *dClk* and *per* expression. Direct regulation of *vri* and *Pdp1ε* transcription by dCLK/CYC, and direct regulation
of $dClk$ expression by VRI and PDP1ε proteins establishes the existence of this second loop and identifies its components (see Figure 7 for the two-loop model). The first loop of this model starts with activation of *per* and *tim* expression by dCLK/CYC at about noon. PER/TIM then feeds back to inhibit dCLK/CYC activity during the second half of the night (reviewed by Allada et al., 2001). In the second loop, dCLK/CYC also activates *vri* and *Pdp1ε* transcription at about noon. *vri* and *Pdp1ε* RNAs and proteins accumulate with different kinetics such that VRI protein accumulates first and represses $dClk$ expression. PDP1ε protein then accumulates and activates $dClk$ transcription after VRI-mediated repression ends in the middle of the night. However, newly produced dCLK protein is inactive due to the presence of PER repressor. Once PER is degraded, dCLK/CYC reactivates *per/tim* and *vri/Pdp1ε* transcription to start a new cycle. The two loops are linked together by dCLK/CYC and restart simultaneously.

Conceptually, a molecular clock must separate the phases of clock gene transcription and repression otherwise clock components reach a stable steady state. The delay separating active transcription and repression of *per/tim* is controlled by the double-time and shaggy/GSK3 protein kinases that regulate PER/TIM accumulation and nuclear transport (Price et al., 1998; Suri et al., 2000; Martinek et al., 2001). The phases of $dClk$ transcription and repression are separated by two mechanisms: (1) accumulation of VRI protein before PDP1ε, which ensures that repression of $dClk$ precedes activation; and (2) PER inhibition of dCLK/CYC activity in the early morning which prevents reactivation of *vri* and *Pdp1ε* transcription even when dCLK levels are high.

### Does the Model Fit the Data?

The model in Figure 7 explains our observation that VRI represses $dClk$ independently of nuclear PER/TIM. It also suggests that in a *per<sup>0</sup>* background, $dClk$ expression is repressed because of high VRI protein levels. High levels of VRI must therefore dominate over high PDP1ε levels and suppress $dClk$ expression in *per<sup>0</sup>* flies. Indeed, overexpression of *vri* is dominant and stops the clock in an otherwise wild-type background with constantly low $dClk$ expression (Blau and Young, 1999, and Figure 4A).

However, this model does not immediately explain why $dClk$ RNA levels are high in *Clk<sup>Jrk</sup>* and *cyc<sup>0</sup>* mutants (Glossop et al., 1999). In the absence of dCLK/CYC function, *vri* RNA levels are low (Blau and Young, 1999), and the consequently low levels of VRI protein (Figure 1E) would not be sufficient to repress $dClk$ expression. But how can expression of $dClk$ RNA be maximal with very low PDP1ε levels in *Clk<sup>Jrk</sup>* and *cyc<sup>0</sup>* mutants? This question is especially relevant given the very low levels of $dClk$ in *Pdp1<sup>P205</sup>* homozygous mutant larvae in constant darkness, which makes the existence of additional factors that positively regulate $dClk$ expression in constant conditions unlikely. The simplest explanation is that the very low levels of *Pdp1ε* RNA present in *Clk<sup>Jrk</sup>* and *cyc<sup>0</sup>* mutants are still sufficient to give enough PDP1ε protein to activate $dClk$ when competition from VRI is minimal due to very low VRI protein levels. Indeed, $dClk$ RNA

levels are close to their peak at ZT3 and ZT6 in wild-type flies when both VRI and PDP1ε levels are very low (Figure 1D). In contrast, PDP1ε protein is totally absent in *Pdp1<sup>P205</sup>* null mutants because the *Pdp1* gene is deleted and thus, $dClk$ is at low levels. However, further work is required to test this hypothesis.

TIM protein can be detected in *Pdp1<sup>P205</sup>* pacemaker cells in LD cycles (data not shown), indicating the existence of additional controls on the clock in LD. Light-driven molecular cycles in pacemaker cells that do not persist in constant conditions have previously been observed in *dbt<sup>P</sup>* mutants (Price et al., 1998) and in electrically silenced pacemaker cells (Nitabach et al., 2002).

### How Many Phases of Expression of Clock-Controlled Genes?

The model in Figure 7 can also be used to explain how clock-controlled genes are expressed with different phases. Genes activated by dCLK/CYC will reach maximum RNA levels at ~ZT14 and these include *per*, *tim*, *vri*, and *Pdp1ε*. Genes regulated by VRI and PDP1ε will peak at ~ZT2 and $dClk$ is one example. Another candidate VRI/PDP1ε regulated gene is *cryptochrome* (*cry*), whose RNA levels oscillate in phase with $dClk$ RNA and follow the same pattern as $dClk$ in clock mutants (Emery et al., 1998). Indeed, overexpression of VRI represses *cry* expression, and the *cry* promoter contains functional VRI (and therefore probably also PDP1ε) binding sites (Glossop et al., 2003).

It is also conceivable that certain DNA sequences bind VRI with higher affinity than PDP1ε or vice versa. One could then imagine two promoters, one with 5 optimal VRI and another with 5 optimal PDP1ε binding sites, that would give RNA expression profiles differing by ~2–4 hr. Such a mechanism may help to explain the multiple peaks of rhythmically expressed genes found in *Drosophila* (e.g., Claridge-Chang et al., 2001; McDonald and Rosbash, 2001).

### Comparisons with the Mammalian Clock

Most clock genes are conserved between *Drosophila* and mammals, and they function in a broadly similar mechanism (Allada et al., 2001). For example, peak levels of *Bmal1* and *Clock* RNAs are antiphase to *mPer1* and *mPer2* in mice (Lee et al., 2001) just as $dClk$ RNA peaks in antiphase to *Drosophila per*. A recent study identified the clock-controlled *Bmal1* repressor (Preitner et al., 2002), which parallels the VRI repression of $dClk$ data presented here. Our data extend the similarities of the *Drosophila* and mammalian clocks and suggest the existence of a rhythmically transcribed *Bmal1* transcriptional activator that plays an analogous role to *Drosophila Pdp1* in the second mammalian feedback loop. A group of potential activators are suggested from the studies of Ueda et al., (2002a).

However, the *Bmal1* repressor is REV-ERBα, an orphan nuclear receptor, which is unrelated to VRI. Perhaps even more surprising is that REV-ERBα is dispensable for rhythmicity in mice, although it adds robustness and precision to the circadian clock (Preitner et al., 2002). Posttranscriptional regulation of clock proteins in the first loop presumably compensates for the loss of rhythmic *Bmal1* expression in the second loop in
rev-erb${}_{\alpha}^{-/-}$ mice. Posttranscriptional regulation of dCLK protein also plays an important part in the dCLK protein cycle (Kim et al., 2002). However, the magnitude of the period alterations in $vri$ and $Pdp1$ heterozygous flies are comparable to those seen in mice homozygous for a rev-erb${}_{\alpha}$ knockout. Therefore, the *Drosophila* clock may rely more heavily on transcriptional control than the mammalian clock, especially in the second loop.

Homologs of VRI and PDP1 do exist in mammals and are even expressed with a circadian rhythm in pacemaker cells (Lopez-Molina et al., 1997; Mitsui et al., 2001). However, genetic loss-of-function experiments suggest that none of the three mammalian $Pdp1$ homologs, either alone or in combination, affects the period length of circadian locomotor activity by more than 30 min (Lopez-Molina et al., 1997, and F. Gachon, F. Damiola, P. Fonjallaz, P. Gos, and U. Schibler, personal communication). Similar loss-of-function experiments have yet to be performed for E4BP4, the mammalian homolog of VRI. The mammalian homologs of $vri$ and $Pdp1$ may thus play only an ancillary regulatory role in the mammalian central clock (Yamaguchi et al., 2000; Mitsui et al., 2001), with their primary role being the regulation of rhythmic clock outputs as suggested by Fonjallaz et al. (1996) and Franken et al. (2000).

Tightly regulated and interconnected feedback loops are conserved in the circadian clocks of all the model organisms so far studied (Harmer et al., 2001). A second interconnected loop adds robustness to oscillators (Cheng et al., 2001; Preitner et al., 2002). Two transcription loops also provide the potential for multiple inputs to the clock such as light (see Allada et al., 2001), temperature (Majercak et al., 1999), membrane potential (Nitabach et al., 2002), and redox state (Rutter et al., 2001). Additionally, a second transcriptional loop provides a mechanism to regulate a novel phase of rhythmic expression of clock output genes. Such downstream genes presumably allow an organism to anticipate a constantly changing, but relatively predictable, environmental cycle, and adjust its behavior and physiology accordingly. The identification of downstream genes that link the molecular ticking of a central clock to changes in whole animal behavior and physiology is clearly the next major challenge in circadian biology.

### Experimental Procedures

#### RNA Analysis
Quantitative Real Time RT-PCR was used to assay RNA levels in all figures except 4A (see below). RNA was purified from adult heads or third instar larvae with RNAsolB (Tel-Test). Two to three $\mu$g total RNA was reverse transcribed with random hexamers using ThermoScript (Invitrogen) and reverse transcription products were amplified in a Roche LightCycler. Hybridization probes (P1 and P2) and 5′ and 3′ primers for quantitative PCR were designed so that one primer or probe spanned an exon/intron boundary to prevent DNA amplification. Primers and probes used were as follows: $Pdp1{\epsilon}$: 5′-GCGGCAACTGGGTAATG; 3′-ATTT CCTGCCTGAGCT; P1 GCAGTGATCGCCAATGAGC-Fluorescein (FI); P2 LC Red640-CACAACCATTGAACAGCTT GAAAG; $vri$: 5′-AGGCAAAGAGGAGAAGC; 3′-CGGATGCAAGTTAGAAGC; P1 CCGAGGCGTCTGTGTCC-FI; P2 LC Red640-AGCAATGCCGAGGTCC; $dClk$: 5′-GTCAGTT CGCAAAGCCA; 3′-CGGCTCAAGAAAATGTCG; P1 GCAACATACAGTGGGTACTCCG-FI; P2 LC Red640-AATGGTGCCTCTCCTGGCC; $n$-syb: 5′G-GCTTCAGA CT TAAAAGATGA; 3′-CAC TAA TCG AGA AAC TTT CGT; P1 CATCATGGGCGTGATGGGC-FI; P2 LC Red640-GGTTGTCGTGGGCAT TATTGC. Levels of RNA in the original sample were determined by comparing the time at which the reaction moved into detectable exponential phase to standard curves for each primer set constructed by reamplifying known quantities of PCR products. For each time series, the maximum value was set to 100%, and other values are expressed as a percentage of the maximum. RNase protection experiments in Figure 4A were as previously described (Blau and Young, 1999) using antisense probes to $dClk$ (Darlington et al., 1998) and $rp49$ (Blau and Young, 1999). In situ hybridization of third instar larval brains for $tim$ and $pdf$ was as described (Blau and Young, 1999).

#### Protein Analysis
Anti-PDP1 antibodies used on Western blots were described previously (Reddy et al., 2000). Anti-VRI and anti-PDP1${}_{\alpha}$ antibodies were generated against bacterially purified GST-VRI 171–729 and GST-PDP1${}_{\alpha}$ respectively by Covance Research Products and used for VRI Western blots and PDP1 immunostaining. We know that VRI and PDP1 antibodies recognize the appropriate proteins for the following reasons: (1) in vitro translated VRI and PDP1${}_{\epsilon}$ proteins are recognized by their appropriate antibodies (data not shown); (2) three different VRI and PDP1 antibodies recognize an ${\sim}75$ kDa protein for VRI and an ${\sim}80$ kDa protein for PDP1${}_{\epsilon}$ on Western blots (data not shown); (3) overexpression of $vri$ in clock cells using the $tim(UAS)$-$gal4$ driver causes a peak of $vri$ RNA at ZT2 (Blau and Young, 1999), and this correlates with high levels of anti-VRI reactivity at ZT3 in these flies, but not in wild-type flies (data not shown); (4) weak anti-PDP1${}_{\epsilon}$ reactivity in pupal head extracts is not seen in extracts made from homozygous $Pdp1^{P205}$ pupae (data not shown); and (5) no PDP1 immunoreactivity was seen in homozygous $Pdp1^{P205}$ larval pacemaker cells (data not shown). Antibodies to PER and HSP70 were obtained from Jeff Hall and Sigma respectively. The 9F8A9 ELAV antibody developed by Gerry Rubin was obtained from the Developmental Studies Hybridoma Bank developed under the auspices of the National Institute of Child Health and Human Development and maintained by University of Iowa. Western blots were performed according to standard procedures. Immunodetection of whole-mount larval brains and adult head sections was as previously described (Price et al., 1998; Kloss et al., 2001).

Gel shift conditions for reticulocyte-translated PDP1${}_{\epsilon}$ and VRI proteins were as described (Reddy et al., 2000) except that the VRI binding buffer was 10 mM HEPES [pH 7.5], 75 mM KCl, 2.5 mM MgCl${}_{2}$, 0.1 mM EDTA, 1 mM DTT, 2% Ficoll, and 100 $\mu$g/ml salmon sperm DNA. The sequence of the wild-type and mutant C3 oligos were AGCTTATTCAATTACATAAACCTGGGCGATAA (VRI/PDP1${}_{\epsilon}$ binding site underlined) and AGCTTATTCAAggACATggCCTGGGCGATAA (changes from wild-type in lower case) respectively. Complementary oligonucleotides were synthesized and wild-type probes were labeled with [$\gamma$-${}^{32}$P]ATP.

#### Cell Culture Experiments
Two potential $dClk$ transcriptional start sites 5 kb apart were predicted based on reported cDNAs (Allada et al., 1998; Bae et al., 1998). An RNase protection probe spanning the downstream start site gave no evidence for use of the upstream site in adult fly heads (data not shown). All reporter luciferase constructs were generated in pGL3 (Promega) whose coding region was modified to remove a potential VRI/PDP1 binding site (Yamaguchi et al., 2000). $dClk$-luc contains a 2.8 kb BglII-Eco0109I fragment of the $dClk$ promoter ligated to a PCR-amplified 400 bp fragment from Eco0109I to 200 bp downstream of the transcription start site. tk-luc contains a minimal thymidine kinase promoter from EcoRI ($-91$) to HindIII ($+20$) of the tk promoter from pRL-tk (Promega). C3 and C3m-tk-luc contain three copies of the sequences in Figure 6A upstream of tk-luc.

A $Pdp1{\epsilon}$ cDNA was amplified by RT-PCR from adult head RNA and its sequence verified before subcloning into pcDNA3 zeo (Invitrogen) to make CMV-PDP1${}_{\epsilon}$. CMV-VRI was constructed by subcloning the full $vri$ open reading frame into pcDNA3 (Invitrogen). The stop codon of VRI was changed into an Xhol site and a 240 bp fragment encoding the VP16 activation domain was added, and cDNA encoding the fusion protein subcloned into pcDNA3 (Invitrogen) to construct CMV-VRI-VP16.

Human embryonic kidney (HEK-293) cells were grown in DMEM

supplemented with 10% fetal bovine serum. Cells were transfected using FuGENE 6 (Roche) in six-well plates with 50 ng firefly luciferase reporter plasmid and varying amounts of expression vector (see Figure Legends). Empty pcDNA3 expression vector was added as appropriate to keep the total amount of DNA constant between plates. Twenty-four hours after transfection, cells were washed and harvested in reporter lysis buffer. Luciferase activity was determined using the Luciferase Assay Reagent (Promega) in a Beckman scintillation counter and was normalized to protein concentration.

Fly Culture and Behavioral Analysis

The *Pdp1*<sup>P205</sup> deletion mutant was generated by imprecise excision of the S071411 P element and verified by inverse PCR. The deletion removes the entire *Pdp1* genomic locus but the genes immediately to the left (CG13684) and right (CG8294) are intact. Full details will be described elsewhere (K.L.R. and R.V.S., unpublished data). Since *dClk* is close to *Pdp1*, we verified that *dClk* locus was unaffected in *Pdp1*<sup>P205</sup> mutants by crossing *Pdp1*<sup>P205</sup> and *Clk*<sup>Irk</sup> mutants. The majority of the *Pdp1*<sup>P205</sup>/*Clk*<sup>Irk</sup> flies were rhythmic, which would not be expected if *dClk* had been inactivated (Allada et al., 1998).

In Figure 3A, w; *Pdp1*<sup>P205</sup>/TM6eSb flies were outcrossed to y w flies for one generation, and F1 females crossed to Canton S males. Locomotor activity of adult F2 male flies was assayed as described (Blau and Young, 1999) with *Pdp1*<sup>P205</sup> mutants identified by eye color. In Figure 3B, w; *tim(UAS)-gal4*; *Pdp1*<sup>P205</sup>/TM6 flies were crossed to either V1 (Blau and Young, 1999) or y w flies, and the progeny assayed in locomotor assays to enable comparison of siblings. For Figure 4, eggs laid by either y w or w; *Pdp1*<sup>P205</sup>/TM6eSb flies were collected on apple juice plates supplemented with yeast for 12–24 hr. The y w embryos were immediately entrained in light-dark cycles. w; *Pdp1*<sup>P205</sup>/TM6eSb progeny were allowed to develop for 3–5 days, and homozygous (non-Tb) *Pdp1*<sup>P205</sup> larvae selected and plated on apple juice plates supplemented with yeast, and entrained in light-dark cycles.

Acknowledgments

We are very grateful for the contributions of the following: Qian Wang for isolating genomic *dClk*; Mike Nitabach and Todd Holmes for advice on statistics, cell culture, and for cells; Laurence Lejay and Karen Thum for advice on real-time PCR; Esteban Mazzoni for help with confocal microscopy; and Eugenia Olesnicky and especially Meg Younger for sorting through thousands of larvae. David Bentley, Jeff Hall, Sebastian Martinek, Hitoshi Okamura, Jae Park, Michael Rosbash, Ralf Stanewsky, and Karen Wager-Smith generously provided antibodies, plasmids, and flies. Florence Blanchard, Nikolaus Rajewsky, Adrian Rothenfluh, and especially Claude Desplan, Mike Nitabach, Ueli Schibler, and four anonymous reviewers made excellent comments in the writing of this manuscript. We also thank Ueli Schibler for sharing unpublished results. This work was supported by an NIH grant to R.V.S.; by NIH grant MH61423 to P.E.H., NIH grant GM54339 to M.W.Y.; and by the NYU Lab Start-Up Program, the Whitehead Fellowship for Junior Faculty, and NIH grant GM063911 to J.B.

Received: July 26, 2002  
Revised: January 11, 2003  

References

Allada, R., Emery, P., Takahashi, J.S., and Rosbash, M. (2001). Stopping time: the genetics of fly and mouse circadian clocks. Annu. Rev. Neurosci. 24, 1091–1119.

Allada, R., White, N.E., So, W.V., Hall, J.C., and Rosbash, M. (1998). A mutant *Drosophila* homolog of mammalian *Clock* disrupts circadian rhythms and transcription of *period* and *timeless*. Cell 93, 791–804.

Bae, K., Lee, C., Sidote, D., Chuang, K.-Y., and Edery, I. (1998). Circadian regulation of a *Drosophila* homolog of the mammalian *Clock* gene: PER and TIM function as positive regulators. Mol. Cell. Biol. 18, 6142–6151.

Baylies, M.K., Bargiello, T.A., Jackson, F.R., and Young, M.W. (1987). Changes in abundance or structure of the per gene product can alter periodicity of the *Drosophila* clock. Nature 326, 390–392.

Blau, J., and Young, M.W. (1999). Cycling *vrille* expression is required for a functional *Drosophila* clock. Cell 99, 661–671.

Chen, C.Y., and Shyu, A.B. (1995). AU-rich elements: characterization and importance in mRNA degradation. Trends Biochem. Sci. 20, 465–470.

Cheng, P., Yang, Y., and Liu, Y. (2001). Interlocked feedback loops contribute to the robustness of the *Neurospora* circadian clock. Proc. Natl. Acad. Sci. USA 98, 7408–7413.

Claridge-Chang, A., Wijnen, H., Naef, F., Boothroyd, C., Rajewsky, N., and Young, M.W. (2001). Circadian regulation of gene expression systems in the *Drosophila* head. Neuron 32, 657–671.

Cowell, I.G., Skinner, A., and Hurst, H.C. (1992). Transcriptional repression by a novel member of the bZIP family of transcription factors. Mol. Cell. Biol. 12, 3070–3077.

Darlington, T.K., Wager-Smith, K., Ceriani, M.F., Staknis, D., Gekakis, N., Steeves, T.D.L., Weitz, C.J., Takahashi, J.S., and Kay, S.A. (1998). Closing the circadian loop: CLOCK-induced transcription of its own inhibitors *per* and *tim*. Science 280, 1599–1603.

Emery, P., So, W.V., Kaneko, M., Hall, J.C., and Rosbash, M. (1998). CRY, a *Drosophila* clock and light-regulated cryptochrome, is a major contributor to circadian rhythm resetting and photosensitivity. Cell 95, 669–679.

Falvey, E., Marcacci, L., and Schibler, U. (1996). DNA-binding specificity of PAR and C/EBP leucine zipper proteins: a single amino acid substitution in the C/EBP DNA-binding domain confers PAR-like specificity to C/EBP. J. Biol. Chem. 377, 797–809.

Fonjallaz, P., Ossipow, V., Wanner, G., and Schibler, U. (1996). The two PAR leucine zipper proteins, TEF and DBP, display similar circadian and tissue-specific expression, but have different target promoter preferences. EMBO J. 15, 351–362.

Franken, P., Lopez-Molina, L., Marcacci, L., Schibler, U., and Tafti, M. (2000). The transcription factor DBP affects circadian sleep consolidation and rhythmic EEG activity. J. Neurosci. 20, 617–625.

George, H., and Terracol, R. (1997). The *vrille* gene of *Drosophila* is a maternal enhancer of decapentaplegic and encodes a new member of the bZIP family of transcription factors. Genetics 146, 1345–1363.

Glossop, N.R.J., Lyons, L.C., and Hardin, P.E. (1999). Interlocked feedback loops within the *Drosophila* circadian oscillator. Science 286, 766–768.

Glossop, N.R.J., Houl, J.H., Zheng, H., Ng, F.S., Dudek, S.M., and Hardin, P.E. (2003). VRILLE feeds back to control circadian transcription of *Clock* in the *Drosophila* circadian oscillator. Neuron 37, 249–261.

Harmer, S.L., Panda, S., and Kay, S.A. (2001). Molecular bases of circadian rhythms. Annu. Rev. Cell Dev. Biol. 17, 215–253.

Kaneko, M., Helfrich-Forster, C., and Hall, J.C. (1997). Spatial and temporal expression of the *period* and *timeless* genes in the developing nervous system of *Drosophila*: newly identified pacemaker candidates and novel features of clock gene product cycling. J. Neurosci. 17, 6745–6760.

Kim, E.Y., Bae, K., Ng, F.S., Glossop, N.R., Hardin, P.E., and Edery, I. (2002). *Drosophila* CLOCK protein is under posttranscriptional control and influences light-induced activity. Neuron 34, 69–81.

Kloss, B., Rothenfluh, A., Young, M.W., and Saez, L. (2001). Phosphorylation of PERIOD is influenced by cycling physical associations of DOUBLE-TIME, PERIOD, and TIMELESS in the *Drosophila* clock. Neuron 30, 699–706.

Lee, C., Etchegaray, J.P., Cagampang, F.R., Loudon, A.S., and Reppert, S.M. (2001). Posttranslational mechanisms regulate the mammalian circadian clock. Cell 107, 855–867.

Lin, S.C., Lin, M.H., Horvath, P., Reddy, K.L., and Storti, R.V. (1997). PDP1, a novel *Drosophila* PAR domain bZIP transcription factor expressed in developing mesoderm, endoderm and ectoderm, is a transcriptional regulator of somatic muscle genes. Development 124, 4685–4696.

Lopez-Molina, L., Conquet, F., Dubois-Dauphin, M., and Schibler, U. (1997). The DBP gene is expressed according to a circadian

rhythm in the suprachiasmatic nucleus and influences circadian
behavior. EMBO J. 16, 6762–6771.
Majercak, J., Sidote, D., Hardin, P.E., and Edery, I. (1999). How a
circadian clock adapts to seasonal decreases in temperature and
day-length. Neuron 24, 219–230.
Martinek, S., Inonog, S., Manoukian, A.S., and Young, M.W. (2001).
A role for the segment polarity gene shaggy/GSK-3 in the *Drosophila*
circadian clock. Cell 105, 769–779.
McDonald, M.J., and Rosbash, M. (2001). Microarray analysis and
organization of circadian gene expression in *Drosophila*. Cell 107,
567–578.
Mitsui, S., Yamaguchi, S., Matsuo, T., Ishida, Y., and Okamura, H.
(2001). Antagonistic role of E4BP4 and PAR proteins in the circadian
oscillatory mechanism. Genes Dev. 15, 995–1006.
Nitabach, M.N., Blau, J., and Holmes, T.C. (2002). Electrical silencing
of *Drosophila* pacemaker neurons stops the free-running circadian
clock. Cell 109, 485–495.
Park, J.H., Helfrich-Forster, C., Lee, G., Liu, L., Rosbash, M., and Hall,
J.C. (2000). Differential regulation of circadian pacemaker output by
separate clock genes in *Drosophila*. Proc. Natl. Acad. Sci. USA 97,
3608–3613.
Preitner, N., Damiola, F., Lopez-Molina, L., Zakany, J., Duboule, D.,
Albrecht, U., and Schibler, U. (2002). The orphan nuclear receptor
REV-ERBα controls circadian transcription within the positive limb
of the mammalian circadian oscillator. Cell 110, 251–260.
Price, J.L., Blau, J., Rothenfluh, A., Abodeely, M., Kloss, B., and
Young, M.W. (1998). *double-time* is a novel *Drosophila* clock gene
that regulates PERIOD protein accumulation. Cell 94, 83–95.
Reddy, K.L., Wohlwill, A., Dzitoeva, S., Lin, M.H., Holbrook, S., and
Storti, R.V. (2000). The *Drosophila* PAR domain protein 1 (*Pdp1*)
gene encodes multiple differentially expressed mRNAs and proteins
through the use of multiple enhancers and promoters. Dev. Biol.
224, 401–414.
Robinow, S., and White, K. (1991). Characterization and spatial dis-
tribution of the ELAV protein during *Drosophila melanogaster* devel-
opment. J. Neurobiol. 22, 443–461.
Rothenfluh, A., Abodeely, M., and Young, M.W. (2000). Short-period
mutations of *per* affect a *double-time*-dependent step in the Dro-
sophila circadian clock. Curr. Biol. 10, 1399–1402.
Rutila, J.E., Suri, V., Le, M., So, W.V., Rosbash, M., and Hall, J.C.
(1998). CYCLE is a second bHLH-PAS clock protein essential for
circadian rhythmicity and transcription of *Drosophila period* and
*timeless*. Cell 93, 805–814.
Rutter, J., Reick, M., Wu, L.C., and McKnight, S.L. (2001). Regulation
of clock and NPAS2 DNA binding by the redox state of NAD cofac-
tors. Science 293, 510–514.
Shearman, L.P., Sriram, S., Weaver, D.R., Maywood, E.S., Chaves,
I., Zheng, B., Kume, K., Lee, C.C., van der Horst, G.T., Hastings,
M.H., and Reppert, S.M. (2000). Interacting molecular loops in the
mammalian circadian clock. Science 288, 1013–1019.
Suri, V., Hall, J.C., and Rosbash, M. (2000). Two novel *double-time*
mutants alter circadian properties and eliminate the delay between
RNA and protein in *Drosophila*. J. Neurosci. 20, 7547–7555.
Ueda, H.R., Chen, W., Adachi, A., Wakamatsu, H., Hayashi, S., Taka-
sugi, T., Nagano, M., Nakahama, K., Suzuki, Y., Sugano, S., et al.
(2002a). A transcription factor response element for gene expression
during circadian night. Nature 418, 534–539.
Ueda, H.R., Matsumoto, A., Kawamura, M., lino, M., Tanimura, T.,
and Hashimoto, S. (2002b). Genome-wide transcriptional orchestra-
tion of circadian rhythms in *Drosophila*. J. Biol. Chem. 277, 14048–
14052.
Yamaguchi, S., Mitsui, S., Yan, L., Yagita, K., Miyake, S., and Oka-
mura, H. (2000). Role of DBP in the circadian oscillatory mechanism.
Mol. Cell. Biol. 20, 4773–4781.
