
Available online at www.sciencedirect.com

SCIENCE DIRECT®

Biochimica et Biophysica Acta 1557 (2003) 67–76

BIOCHIMICA ET BIOPHYSICA ACTA
BBA
www.bba-direct.com

---

**The menD and menE homologs code for 2-succinyl-6-hydroxyl-2,4-cyclohexadiene-1-carboxylate synthase and O-succinylbenzoic acid–CoA synthase in the phylloquinone biosynthetic pathway of Synechocystis sp. PCC 6803**

T. Wade Johnson${}^{a,*}$, Sushma Naithani${}^{a}$, Charles Stewart Jr.${}^{a}$, Boris Zybailov${}^{b}$, A. Daniel Jones${}^{c}$, John H. Golbeck${}^{b}$, Parag R. Chitnis${}^{a}$

${}^{a}$Department of Biochemistry, Biophysics and Molecular Biology, Iowa State University, Ames, IA 50011, USA  
${}^{b}$Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, PA 16802, USA  
${}^{c}$Department of Chemistry, The Pennsylvania State University, University Park, PA 16802, USA  

Received 17 September 2002; received in revised form 26 November 2002; accepted 3 December 2002

---

### Abstract

The genome of the cyanobacterium *Synechocystis* sp. PCC 6803 contains genes identified as *menD* and *menE*, homologs of *Escherichia coli* genes that code for 2-succinyl-6-hydroxyl-2,4-cyclohexadiene-1-carboxylate (SHCHC) synthase and O-succinylbenzoic acid–CoA ligase in the menaquinone biosynthetic pathway. In cyanobacteria, the product of this pathway is 2-methyl-3-phytyl-1,4-naphthoquinone (phylloquinone), a molecule used exclusively as an electron transfer cofactor in Photosystem (PS) I. The *menD*${}^{-}$ and *menE*${}^{-}$ strains were generated, and both were found to lack phylloquinone. Hence, no alternative pathways exist in cyanobacteria to produce O-succinylbenzoyl-CoA. Q-band EPR studies of photoaccumulated quinone anion radical and optical kinetic studies of the P700${}^{+}$[F${}_{\mathrm{A}}$/F${}_{\mathrm{B}}$]${}^{-}$ backreaction indicate that in the mutant strains, plastoquinone-9 functions as the electron transfer cofactor in the A${}_{1}$ site of PS I. At a light intensity of 40 μE m${}^{-2}$ s${}^{-1}$, the *menD*${}^{-}$ and *menE*${}^{-}$ mutant strains grew photoautotrophically and photoheterotrophically, but with doubling times slower than the wild type. Both of which are sensitive to high light intensities. Low-temperature fluorescence studies show that in the *menD*${}^{-}$ and *menE*${}^{-}$ mutants, the ratio of PS I to PS II is reduced relative to the wild type. Whole-chain electron transfer rates in the *menD*${}^{-}$ and *menE*${}^{-}$ mutant cells are correspondingly higher on a chlorophyll basis. The slower growth rate and high-light sensitivity of the *menD*${}^{-}$ and *menE*${}^{-}$ mutants are therefore attributed to a lower content of PS I per cell.

© 2002 Elsevier Science B.V. All rights reserved.

Keywords: Phylloquinone; Photosystem I; *Synechocystis*; Plastoquinone; *menD*; *menE*

---

### 1. Introduction

Photosystem (PS) I is a light-driven, multisubunit enzyme that catalyzes the oxidation of plastocyanin (or cytochrome $c_{6}$) in the thylakoid lumen and the reduction of ferredoxin (or flavodoxin) in the cyanobacterial cytoplasm [1] or chloroplast stroma [2]. Cyanobacterial PS I consists of a hetero-

dimeric core composed of the PsaA and PsaB proteins surrounded by six small polypeptides labeled PsaF, PsaI, PsaJ, PsaK, PsaL, PsaM and PsaX [3]. Charge separation is carried out between a chlorophyll $a$ dimer (P700) that serves as the primary electron donor, and a chlorophyll $a$ monomer ($A_{0}$) that serves as the primary electron acceptor. The secondary acceptor, phylloquinone ($A_{1}$), functions to transfer an electron from $A_{0}$ to $F_{\mathrm{X}}$, a [4Fe–4S] cluster ligated by cysteine residues contributed by the PsaA and PsaB heterodimer. Two molecules of phylloquinone exist per molecule of P700 [4–7], although it is uncertain whether one or both are active in forward electron transfer. A peripheral protein labeled PsaC contains two [4Fe–4S] iron–sulfur clusters ($F_{\mathrm{A}}$ and $F_{\mathrm{B}}$), and two additional proteins PsaD and PsaE act as a

* This paper is a fifth in the series, “Recruitment of a Foreign Quinone into the A${}_{1}$ Site of Photosystem I”.
* Corresponding author. Present address: Department of Biochemistry and Molecular Biology, The Pennsylvania State University, University Park, PA 16802, USA. Tel.: +1-814-865-1162; fax: +1-814-863-7024.
* E-mail address: twj2@psu.edu (T. Wade Johnson).

0005-2728/02/$ - see front matter © 2002 Elsevier Science B.V. All rights reserved.

doi:10.1016/S0005-2728(02)00396-1

ferredoxin docking site. The cofactors function to transfer an electron from P700 on the lumenal side of the membrane to the terminal iron–sulfur clusters F<sub>A</sub> and F<sub>B</sub> on cytoplasmic side of the membrane.

The only known function of 2-methyl-3-phytyl-1,4-naphthoquinone (phylloquinone) in cyanobacteria and plants is to act as the electron transfer cofactor A<sub>1</sub> in PS I. In spite of its importance, the biosynthetic route of phylloquinone has not yet been elucidated in cyanobacteria. Many prokaryotes contain the metabolic pathway for biosynthesis of menaquinone. In certain bacteria, menaquinone (vitamin K<sub>2</sub>), is used during fumarate reduction in anaerobic respiration [8,9]. In green sulfur bacteria and heliobacteria, menaquinones are thought to be used as a loosely bound secondary electron acceptor in the photosynthetic reaction center [10]. The genes encoding enzymes involved in the conversion of the chorismate to menaquinone have been cloned in *Escherichia coli* [11–16], *Bacillus subtilis* [17–20], and *Bacillus stearothermophilus* [21]. Menaquinone differs from phylloquinone in having an unsaturated C-40 side chain rather than a mostly saturated C-20 phytyl side chain. We expect the synthesis of the naphthalene ring to involve similar steps in the two pathways. The genome of *Synechocystis* sp. PCC 6803 [22] contains homologs for several genes that encode enzymes in menaquinone biosynthesis: *menF* (isochorismate synthase), *menD* (2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate (SHCHC) synthase), *menE* (*O*-succinylbenzoic acid–CoA synthase), *menB* (DHNA synthase), *menA* (phytyl transferase), and *menG* (methyl transferase). Previously, we have shown phylloquinone is not found in cells after insertional inactivation of the *menA* and *menB* homologs [23]; therefore, these genes are proven to be involved in phylloquinone biosynthesis. More recently, the inactivation of *menG* has been performed in which the cells generate demethylphylloquinone [24]. This gene therefore codes for the methyl transferase that specifically methylates 2-phytyl-1,4-naphthoquinone.

Two open reading frames in the *Synechocystis* sp. PCC 6803 genome, slr0603 and slr0492, code for proteins that have 28% and 29% sequence identity to the deduced amino acid sequences of SHCHC synthase (*menD* gene) and *O*-succinylbenzoic acid–CoA ligase (*menE* gene) in *Arabidopsis thaliana*. Therefore, we suggest that the *menD* and *menE* genes code for the enzymes involved in the phylloquinone biosynthetic pathway in *Synechocystis* sp. PCC 6803 (Fig. 1). In this paper, we describe the generation, physiology and spectroscopic characteristics of the *menE* and *menD* gene inactivated mutant strains.


![Diagram](attachment:diagram.png)

Fig. 1. Proposed biosynthetic pathway of phylloquinone biosynthesis in *Synechocystis* sp. PCC 6803. The gene responsible for the biosynthesis of menaquinone was initially described in *E. coli* [15]. The homologs of these genes have been identified in the genome sequence of *Synechocystis* sp. PCC 6803 and *menA*, *menB* [23], and *menG* were confirmed [24].

T. Wade Johnson et al. / Biochimica et Biophysica Acta 1557 (2003) 67–76

2. Materials and methods

2.1. Generation of the menD and menE mutant strains of *Synechocystis* sp. PCC 6803

To generate a recombinant DNA construction for inactivation of the menD gene, a 1.8-kb DNA fragment from *Synechocystis* sp. PCC 6803 genomic DNA was amplified by polymerase chain reaction (PCR) using primer menDSN1 (5′-GGATCAAGCTTTTCAGCACAGC-3′) and menDSN2 (5′-ACCTCAGCCTAGGTAATGGC-3′). The 1.8-kb fragment containing the full menD gene and flanking sequences was cloned at the EcoRV blunt end in pBluescript II KS(+) and the validity of the insert was confirmed by sequencing. The chloramphenicol resistance gene (CAT) from pUC4C was cloned at a unique EcoRV site within the menD coding region. The resultant plasmid was designated menDC and was used for transformation of the wild-type strain of *Synechocystis* sp. PCC 6803. Recombinants were selected on BG11 plates containing chloramphenicol (20 μg/ml), and isolation of segregated mutants was performed according to previously published methods [25]. Recombinant clones were confirmed for insertional inactivation of the menD gene by PCR and sequencing and separately by Southern blot. For PCR confirmation primers, menDSN3 (5′-ACCGCCGTTATTGTTGAGC-3′) and menDSN4 (5′-GGTGGTGCATCTTAATTGTC-3′) were used.

To generate a recombinant DNA construction for inactivation of the menE gene, a 996-bp DNA fragment from *Synechocystis* sp. PCC 6803 genomic DNA was amplified by PCR using primer menESN3 (5′-TAATGTTGGCAACAGCAGTGG-3′) and menESN4 (5′-AGG-GTGGAACATCATCCCC-3′) and was cloned at the EcoRV blunt end in pBluescript II KS(+). This clone, designated as menEC, was digested with the MscI enzyme, which has two sites within the menE coding region. A 260-bp region was replaced by inserting the chloramphenicol resistance gene at the blunt end. The chloramphenicol resistance gene (CAT) from pUC4C was cloned at MscI and SmaI. The resultant plasmid was designated menEC3 and used for transformation of the wild-type strain of *Synechocystis* sp. PCC 6803. Recombinants were selected on BG11 plates containing chloramphenicol (20 μg/ml), and isolation of segregated mutants was performed as described above. Recombinant clones were confirmed for insertional inactivation of the menE gene by PCR and sequencing using menSN3 and menSN4 primers.

2.2. DNA isolation, PCR, and Southern blotting

Genomic DNA from *Synechocystis* sp. PCC 6803 was prepared as described in Ref. [26]. Hybridization probes were generated with the DIG High-Prime DNA labeling system (Roche Molecular Biochemicals). Hybridization and detection were performed according to the manufacturer's protocols.

2.3. Cyanobacterial strains and growth

A glucose-tolerant strain of *Synechocystis* sp. PCC 6803 was used as the wild-type strain. The wild-type, menD and menE mutant cells were grown in the BG-11 medium, with 5 mM glucose and chloramphenicol antibiotic for the mutants [27]. Agar plates for the growth of stock cells were kept at a low light intensity (2–10 μE m⁻² s⁻¹). Liquid cultures of the wild-type and mutant strain were grown heterotrophically under normal light conditions (40–60 μE m⁻² s⁻¹), and were bubbled with sterile filtered air. Cell growth was monitored by measuring the optical density at 730 nm (A₇₃₀) with a Shimadzu spectrophotometer. Cells from liquid cultures in the late exponential phase of growth (0.8–1.2 A₇₃₀) were harvested by centrifugation at 5000×g for 15 min. Growth on a large scale was achieved by starting a 3-l culture at 0.050 A₇₃₀ and adding 2.5 mM glucose (final concentration) in addition to chloramphenicol for the mutants.

2.4. Growth rates of the wild-type and mutant cells

For estimating growth rates, the cells were grown in six-well culture plates with 8 ml liquid medium in each well. Cyanobacterial cultures in the late exponential phase were pelleted by centrifugation, washed twice with BG11 medium, and suspended in BG11 and chloramphenicol at concentration of ca. 10 A₇₃₀ units. All cultures were adjusted to the same initial cell density (0.08–0.10 A₇₃₀ units). The final concentrations of glucose and 3-(3,4-dichlorophenyl)-1,1-dimethylurea (DCMU) were 5 mM and 10 μM, respectively. The cells were shaken on an orbital shaker at 110 rpm under a bank of fluorescent lights at a light intensity of 40–60 μE m⁻² s⁻¹.

2.5. Isolation of thylakoid membranes and PS I complexes

Thylakoid membranes were prepared from cells as described [28]. The thylakoid membranes were pelleted by centrifugation at 50,000×g for 90 min and were resuspended in SMN buffer (0.4 M sucrose, 10 mM MOPS, 10 mM NaCl) for storage. Chlorophyll was extracted from thylakoid membranes and PS I trimers with 80% acetone and determined according to Ref. [29]. For the isolation of PS I complexes, thylakoid membranes were incubated in SMN buffer with 20 mM CaCl₂ for 0.5–1.0 h at room temperature in the dark to enhance trimerization of PS I. To the mixture, n-dodecyl-β-D-maltoside (DM) was added to a final concentration of 1.5% (w/v) of the chlorophyll content and incubated in the dark on ice with occasional gentle mixing for 0.5–1.5 h. The non-solubilized material was removed by centrifugation at 10,000×g for 15 min. The trimeric PS I complexes were separated from PS I mono-

mers and PS II by centrifugation in 10–30% (w/v) sucrose gradients with 0.04% DM in 10 mM MOPS (pH 7.0).

### 2.6. Chlorophyll analysis and oxygen evolution measurements

Chlorophyll was extracted from whole cells and thylakoids with 100% methanol. Chlorophyll concentrations were determined according to Ref. [30]. Oxygen evolution measurements were performed using a Clark-type electrode as described in Ref. [31]. Cells were prepared by starting cultures at the same time and grown photomixotrophically with 5 mM glucose for two subcultures. The final liquid culture was made without addition of glucose and grown until mid-log growth phase (0.4–0.6 \(A_{730}\) units). The temperature of the electrode chamber was maintained at 25 °C by a circulating water bath. Cells were pelleted and washed twice with BG11 and resuspended in 40 mM HEPES/NaOH (pH 7.0) buffer to a concentration of 8–10 \(A_{730} \, \text{ml}^{-1}\). Measurements were done on 5 \(A_{730}\) units of cells. Whole-chain electron transport (\(H_2O\) to \(CO_2\)) measurements were determined after the addition of 5 mM NaHCO₃; oxygen evolution mediated by PS II only was determined after addition of 4 mM *p*-benzoquinone (BQ). The light intensity was 1840 μE m⁻² s⁻¹.

### 2.7. Seventy-seven kelvin fluorescence emission spectra

The low-temperature fluorescence emission spectra were measured using a SLM 8000C spectrofluorometer as described [32]. Cells from the exponential phase of growth were harvested by pelleting and resuspended in 25 mM HEPES/NaOH (pH 7.0) buffer. Cells (5 μg chlorophyll) were diluted in 25 mM HEPES/NaOH (pH 7.0) to a final volume of 30 μl and dark adapted for 30 min on ice. To the solution, 70 μl of neat glycerol was added and mixed before quickly freezing in liquid nitrogen. The excitation wavelength was 435 nm. The excitation slit width was set at 4 nm and the emission slit width was set at 2 nm. The emission was scanned from 600 to 800 nm twice and averaged.

### 2.8. Analysis of phylloquinone using HPLC and mass spectrometry

The samples were prepared and quinones were analyzed on an equal chlorophyll basis as described previously [23] but with the following modifications. Membranes containing 0.025 mg chlorophyll were centrifuged at 10,000×g for 60 min and the supernatant was removed. PS I trimers were concentrated by ultrafiltration and lyophilized to dryness. The pigments were extracted sequentially with 1 ml methanol, 1 ml 1:1 (v/v) methanol/acetone, and 1 ml acetone, and the three extracts were combined. The resulting solution was concentrated by vacuum at 4 °C in the dark to dryness. Dry pigment samples were stored at –80 °C until used. The pigments were resuspended in a mixture of 1:1 methanol/

isopropanol at ca. 0.8 mg Chl ml⁻¹. HPLC separations were monitored with photodiode array UV/Vis detection using a Hewlett Packard (Agilent Technologies, Palo Alto, CA) model 1100 quaternary pump and model G1316A photodiode array detector. Sample injections (35 μl) were made on a 4.6 mm × 250 mm Ultrasphere C₁₈ column with 5 μm packing (Beckman Instruments, Palo Alto, CA) using a gradient elution (solvent A = methanol; solvent B = isopropanol; 100% A from 0 to 10 min to 3% A/97% B at 30 min, hold until 40 min) at 1.0 ml min⁻¹. The column was washed after each run (from 100% B to 100% A for 10 min at 0.5 ml min⁻¹, 100% A for 5 min at 1.0 ml min⁻¹.). A solution of phylloquinone (40 μM) was prepared in absolute ethanol and kept at –20 °C as a standard for calibration. Extracts were also analyzed by LC/MS using a Perceptive Biosystems Mariner time-of-flight mass spectrometer using electrospray ionization in negative mode with a needle potential of –3500 V and a nozzle potential of –80 V. A post-column flow splitter delivered column eluent to the electrospray ion source at 10 μl min⁻¹.

### 2.9. Spectroscopic characterization of PS I complexes

Time-resolved optical spectroscopy of the P700⁺ [Fₐ/Fₔ]⁻ charge recombination was performed on purified PS I trimers as described in Ref. [33]. Q-band CW EPR spectroscopy of the quinone anion radical in the A₁ site of PS I was measured as described in Ref. [34].

## 3. Results

### 3.1. Genotype of the menD⁻ and menE⁻ mutant strains

The genotype of the menD⁻ mutant strain was confirmed by PCR amplification of the appropriate genomic loci and sequenced. In Fig. 2A (left panel), the genomic region of menD is shown for the wild-type and mutant strains. A 1.1-kb chloramphenicol resistance cartridge was inserted in the EcoRV site of the menD gene. Using primers within the menD coding sequence containing the EcoRV site, PCR amplification of the menD locus of the wild type produced the expected 990-bp fragment (Fig. 2A, right panel). PCR amplification of the menD mutant locus produced the 2.1-kb fragment, accounting for the integration of the chloramphenicol-resistance cartridge. There was no amplification of the 990-bp fragments in the mutant strain, indicating complete segregation of the mutant allele. Southern blot hybridization analysis of the menD⁻ mutant confirmed complete segregation in the mutant strain (data not shown).

Insertional inactivation of the menE gene was verified by PCR amplification of the menE locus and by DNA sequencing. The left panel of Fig. 2B shows the restriction map of the wild-type and mutant menE gene. Using primers within the menE coding sequence, PCR amplification of the menE locus of the wild type produced the expected 980-bp frag-

T. Wade Johnson et al. / Biochimica et Biophysica Acta 1557 (2003) 67–76

**Fig. 2.** (A) Left panel: The restriction map of the *Synechocystis* sp. PCC 6803 genomic region around the *menD* wild-type (top) and disrupted gene (below). Right panel: Electrophoretic analysis of the DNA fragments amplified by PCR from genomic DNA of the wild-type and mutant gene. Molecular weight marker left lane. (B) Left panel: The restriction map of the *Synechocystis* sp. PCC 6803 genomic region around the *menE* wild-type (above) and disrupted gene (bottom). Right panel: Electrophoretic analysis of the DNA fragments amplified by PCR from genomic DNA of the wild-type and mutant gene. Molecular weight marker left lane. Each PCR fragment was sequenced to confirm its identity.

ment. A 260-bp fragment of the *menE* gene was deleted and replaced with a 1.2-kb chloramphenicol cassette. PCR amplification of the mutant strain produced the expected 1.9-kb fragment. The wild-type 980-bp fragment was not observed in the mutant. These results indicate that segregation of the alleles had occurred completely in the *menE* mutant.

### 3.2. Absence of phylloquinone in the *menD*⁻ and *menE*⁻ mutant strains

In the wild-type strain, phylloquinone is present in PS I and plastoquinone-9 is present in PS II and in the membrane as an electron and proton transporter that functions between PS II and the cyt *b*₆*f* complex [35]. The phylloquinone content of PS I trimers and thylakoid membranes from the *menD*⁻ and *menE*⁻ mutant strains was determined using HPLC with a photodiode array UV/Vis detector. By co-injecting standards and by interpreting the UV/Vis spectra, chlorophyll *a* was identified at 19.0 min, β-carotene was identified at 28.5 min, and phylloquinone was identified at 20.7 min, showing the distinctive doublet peak in the near-UV at 248 and 269 nm. Plastoquinone-9 standard was found to elute at 29 min, showing a singlet peak in the near-UV at 256 nm. The chromatogram of solvent-extracted pigments from PS I trimers of *menD*⁻ and *menE*⁻ mutant strains shows a peak at 256 nm with an elution time of 29.7 min characteristic of plastoquinone-9 (Fig. 3). In contrast, solvent-extracted pigments from PS I trimers of the wild type contain only phylloquinone. To determine if phylloquinone was present elsewhere in the membrane, thylakoid membrane pigments were solvent extracted and analyzed. The *menD*⁻ and *menE*⁻ mutant thylakoid membranes were found to be devoid of phylloquinone (Fig. 4 inset and Fig. 4, respectively). In contrast, wild-type thylakoid membranes contained both phylloquinone and plastoquinone-9 (inset of Fig. 4).

The wild-type and mutant PS I complexes yield a ratio of chlorophyll *a* to quinone similar to the published value of 50:1 Chl/Q [4,36,37]. The wild type has a ratio of 42:1 ± 3.6 Chl/PhQ while *menD*⁻ and *menE*⁻ mutants have a slightly higher ratio of 51:1 ± 2.1 and 54:1 ± 4.1 Chl/PQ-9, respectively. The wild-type PS I trimers contained a minute amount (<2%) of plastoquinone-9. Wild-type PS I trimers that are washed with 3–10 volumes of 10 mM MOPS and 0.04% DM and concentrated do not contain detectable plastoquinone-9, yet have stoichiometric quantities of phylloquinone (data not shown). Therefore, the small content of

T. Wade Johnson et al. / Biochimica et Biophysica Acta 1557 (2003) 67–76

Fig. 3. HPLC profiles of pigment extracts from lyophilized PS I complexes of the $menE^{-}$ mutant strain of *Synechocystis* sp. PCC 6803. The pigments were separated on a 5.0-μm ultrasphere C₁₈ reverse phase column and detected from 190 to 800 nm by a photodiode array. The detection wavelength shown was 254 nm. The inset spectrum from the mutant shows a peak that co-elutes with plastoquinone at 29.7 min.

plastoquinone-9 is probably captured in the lipid membranes and detergent during PS I purification from the plastoquinone-rich thylakoid membranes.

The minimum detection limit, by optical absorption, of phylloquinone in the mutants is approximately 300–350 Chl/Q [23]. LC-MS has the potential to afford several orders of magnitude greater sensitivity [38]. The molecular ions at $m/z=450$ and $m/z=748$, which represent phylloquinone and plastoquinone, respectively, eluted at the same retention times as with the UV/Vis detector. Pigments extracted from both thylakoid membranes and from purified PS I trimers of $menD^{-}$ and $menE^{-}$ mutants contain only plastoquinone-9.

Fig. 4. HPLC profiles of pigment extracts from lyophilized thylakoid membranes of wild-type, $menD^{-}$ and $menE^{-}$ mutant strain of *Synechocystis* sp. PCC 6803. The pigments were separated on a 5.0-μm ultrasphere C₁₈ reverse phase column and detected from 190 to 800 nm by a photodiode array. The detection wavelength shown was 254 nm. The inset chromatograms compare wild type and $menD^{-}$ to the full $menE^{-}$ chromatogram in the region of the PhQ peak (20.5 min).

T. Wade Johnson et al. / Biochimica et Biophysica Acta 1557 (2003) 67–76

Phylloquinone was not detected in either thylakoid membranes or PSI trimers (data not shown).

### 3.3. EPR spectroscopy of photoaccumulated \( A_1 \)

CW EPR spectra at 34 GHz (Q-band) of photoaccumulated PSI complexes from the mutants show significant differences from the wild type at 34 GHz (Q-band). In \( menD^- \) and \( menE^- \) mutants, the low-field g-value is 2.0065, the g-anisotropy is considerably broader than the wild type, and the four prominent methyl hyperfine lines derived from the 2-methyl group of phylloquinone are missing (data not shown). This spectrum is similar to that described in the \( menA \) and \( menB \) mutants, and is consistent with the presence of a plastoquinone-9 radical anion in the \( A_1 \) site [34].

### 3.4. \( P700^+ \) optical recombination kinetics

In PSI trimers isolated from the wild type strain, the charge recombination between \( P700^+ \) and \([F_A/F_B]^- \) is multiphasic after a saturating flash [33,39]. When measured in the absence of external electron acceptors, the \( P700^+ \) is reduced with lifetimes of 10 and 100 ms. There is also a long-lived (>1200 ms) kinetic phase of \( P700^+ \) reduction from reduced DCPIP that contributes 10% of the total absorbance change. In PSI trimers isolated from \( menD^- \) and \( menE^- \) mutant strains, the reduction of \( P700^+ \) is also multiphasic (data not shown). When \( menD^- \) is measured in the absence of external electron acceptor, \( P700^+ \) is reduced with lifetimes of approximately 3.7 and 10 ms. When \( menE^- \) is measured in the absence of external electron acceptor, \( P700^+ \) is reduced with lifetimes of approximately 2.6 and 10 ms in a similar ratio to the wild type. The ratio of the 3- to 10-ms phases is approximately 3:1 in both mutant strains. There is also a minor, long-lived kinetic phase of \( P700^+ \) reduction from reduced DCPIP. These kinetic phases are similar to the kinetic phases in the \( menA \) and \( menB \) mutants, and are consistent with the presence of a functional plastoquinone-9 in the \( A_1 \) site [33].

### 3.5. Growth of the mutant strains

Photoautotrophic growth rates were measured in wild-type and mutant cells grown in BG11 medium, which contains minerals and bicarbonate as the sole carbon source. Under normal light intensity (40 μE m⁻² s⁻¹), the 80-h doubling times of \( menD^- \) and \( menE^- \) mutant strains were significantly slower than the 26-h doubling time of the wild-type strain (Table 1). At higher light intensities (>120 μE m⁻² s⁻¹), both \( menD^- \) and \( menE^- \) strains did not grow. Photomixotrophic growth rates of the mutants were determined in the presence of 5 mM glucose, which allows both respiration and photosynthesis to provide energy for growth. The 14-h doubling time of the wild-type strain was less than one-half the 41- and 42-h doubling times of the \( menD^- \) and \( menE^- \) mutant strains. The doubling times of the \( menD^- \)

---

### Table 1  
Physiological characteristics of the *Synechocystis* sp. PCC 6803 wild-type and \( menD^- \) and \( menE^- \) mutant strains

|                      | Wild type       | \( menD^- \)    | \( menE^- \)    |
|----------------------|-----------------|-----------------|-----------------|
| **Doubling time (h)** |                 |                 |                 |
| Photoautotrophic growth | 26 ± 3.0       | 80 ± 8.0        | 80 ± 8.2        |
| Photoheterotrophic growth | 14 ± 1.5      | 41 ± 5.5        | 42 ± 1.2        |
| Photoheterotrophic growth + DCMU | 20 ± 2.9      | 21 ± 4.1        | 22 ± 1.2        |
| **Photosynthetic rates** |               |                 |                 |
| (\( \mu \)mol O₂/(5OD₇₃₀ h)) |             |                 |                 |
| Whole chain          | 2420 ± 180     | 2300 ± 164     | 1860 ± 190     |
| PS II-mediated       | 7110 ± 560     | 6350 ± 580     | 5790 ± 450     |
| (\( \mu \)mol O₂/(Chl h)) |             |                 |                 |
| Whole chain          | 708 ± 62       | 836 ± 30       | 815 ± 83       |
| PS II-mediated       | 2070 ± 130     | 2269 ± 210     | 2450 ± 190     |

<sup>a</sup> Experimental conditions described in Materials and methods.

and \( menE^- \) mutant strains were similar to the wild type under photoheterotrophic growth conditions conducted in the presence of 10 μM DCMU to inhibit PS II. These studies indicate that the \( menD^- \) and \( menE^- \) mutant strains were capable of photoautotrophic growth (except at high light intensities) and that respiration functions normally.

### 3.6. Electron transfer rates in whole cells

To determine whether differences of the mutant strains are due to differences in photosynthetic electron transfer, we measured whole-chain and partial-chain electron transfer rates of PSI and PS II in the \( menD^- \) and \( menE^- \) strains. Whole-chain electron transfer from water to bicarbonate was measured in cells grown under normal light conditions. On an equivalent cell number basis, the rates of whole-chain oxygen evolution are 95% and 77%, for the \( menD^- \) and \( menE^- \) mutants, respectively, compared to the wild type (Table 1). PS II activity was measured by adding a soluble quinone electron acceptor, 1,4-benzoquinone, to intercept the electron at the level of the quinone pool. The PS II partial electron transfer activities of \( menD^- \) and \( menE^- \) are 89% and 81% of the wild type, respectively.

When equalized on a chlorophyll basis, a more consistent trend begins to emerge. The chlorophyll levels per cell of the \( menD^- \) and \( menE^- \) mutants are 89% and 65% of the wild type, respectively. On a per chlorophyll basis, the light-sensitive \( menD^- \) and \( menE^- \) mutants show an apparent increased whole chain and PS II activity over the wild type. The relatively high activity of PS II in the \( menD^- \) and \( menE^- \) mutants appears to stress the cells as indicated by the slower growth rates and the high-light sensitivity.

### 3.7. Relative content of PS I per cell

The intensity of the PS II chlorophyll fluorescence emission at 685 and 695 nm is similar to the intensities in the wild-type and the mutant strains, with changes occurring primarily in the 721-nm fluorescence emission due to PS I

T. Wade Johnson et al. / Biochimica et Biophysica Acta 1557 (2003) 67–76

Fig. 5. Fluorescence emission spectra at 77 K of whole cells from *Synechocystis* sp. PCC 6803 wild-type, $menD^{-}$ and $menE^{-}$ mutant strains. Spectra were recorded at the same cell density and normalized relative to PS II. Each spectrum was the average of the four measurements performed in duplicate. The excitation wavelength was 435 nm, which excites mostly chlorophyll. PS II and its accessory pigments exhibit emission maxima at 685 and 695 nm; PS I has a maximum emission at 721 nm.

(Fig. 5). The PS I to PS II ratio is reduced to 2.7 in $menD^{-}$ and to 2.2 in $menE^{-}$ mutant strain compared to 3.9 in the wild-type strain. The relative increase of PS II in the these mutant strains suggests that excess reductant is generated that overwhelms the ability of PS I to utilize this reductant. At higher light intensities, this proves toxic to the cells [23].

## 4. Discussion

The $menA$, $menB$, and $menG$ genes were initially identified in *Synechocystis* sp. PCC 6803 by comparing homolog genes that code for enzymes involved in menaquinone biosynthesis in other bacteria [23,24]. The $menB$ gene codes for naphthoate synthase, which is involved in ring closure, and $menA$ gene codes for phytyl transferase, which attaches the phytyl tail to the naphthoquinone ring. Insertional inactivation of either gene results in a complete loss of phylloquinone biosynthesis and in the incorporation of plastoquinone-9 in the A₁ site of PS I [23]. The $menG$ gene methylates 2-phytyl-1,4-naphthoquinone at the ring position ortho to the phytyl tail. Insertional inactivation of the $menG$ gene results in the incorporation of demethylphylloquinone in the A₁ site [24].

This study focuses on $menD$ and $menE$ genes that code for earlier enzymatic steps in the biosynthetic pathway of phylloquinone (Fig. 1). These genes were first identified in *E. coli* [11,15] and homologs were identified in the genome of *Synechocystis* sp. PCC 6803. The $menD$ gene codes for the enzyme 2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate (SHCHC) synthase which catalyzes isochorismic acid to 2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate while concurrently releasing pyruvate and carbon dioxide. The precursor to the naphthoate synthase enzyme, the $menE$ gene, codes for the enzyme $O$-succinylbenzoyl-CoA synthetase, which attaches a CoA moiety to $O$-succinyl benzoic acid.

To determine whether the genes identified as $menD$ and $menE$ code for SHCHC synthase and $O$-succinylbenzoyl-CoA synthetase in *Synechocystis* sp. PCC 6803, we engineered deletion mutants by targeted inactivation. The $menD^{-}$ and $menE^{-}$ genes were PCR amplified and sequenced, with Southern blot analysis additionally performed on $menD^{-}$, to confirm the complete absence of the functional gene in the mutants (Fig. 2). HPLC-MS and HPLC-UV/Vis showed that neither thylakoid membranes (Fig. 4) nor purified PS I trimers (Fig. 3) of the $menD^{-}$ and $menE^{-}$ deletion strains contained detectable levels of phylloquinone. Thus, the $menD$ and $menE$ genes in *Synechocystis* sp. PCC 6803 genome are shown to code for those essential enzymes in phylloquinone biosynthesis. This result also shows that there

are no other biosynthetic routes to phylloquinone that circumvent SHCHC synthase or $O$-succinylbenzoyl–CoA synthetase in *Synechocystis* sp. PCC 6803. We therefore propose that SHCHC synthase is the first enzyme dedicated to phylloquinone biosynthesis. This, however, needs to be confirmed by disabling all of the genes in the biosynthetic pathway (i.e. $menF$ and earlier; see Fig. 1). Precursors to the products of SHCHC synthase, chorismate and isochorismate, are common biomolecules employed in not only phylloquinone biosynthesis, but in the production of ubiquinone, aromatic amino acids, and serine. Therefore, a toxic buildup of these two phylloquinone precursors in the cells should not occur in *Synechocystis* sp. PCC 6803.

Pigment analysis of the $menD^{-}$ and $menE^{-}$ mutants show a chlorophyll to plastoquinone-9 ratio similar to the wild type [36]. Photoaccumulation protocols in PS I trimers of the $menD^{-}$ and $menE^{-}$ mutants show a CW EPR spectrum of Q at 34 GHz (Q-band) [34]. Thus, plastoquinone-9 is present in the A$_{1}$ site and is competent in accepting electrons from A$_{0}^{-}$. To determine whether the electron is transferred from reduced phylloquinone to the iron–sulfur clusters ($F_{X}$, $F_{A}$, $F_{B}$), we monitored the backreaction between P700$^{+}$ and [$F_{A}/F_{B}$]$^{-}$. When observed at 810 nm, the reduction of P700$^{+}$ in the $menD^{-}$ and $menE^{-}$ mutant strains shows a multiphasic backreaction consistent with an electron originating from [$F_{A}/F_{B}$]$^{-}$ and passing backward through PQ to P700$^{+}$ [33]. Thus, these assays indicate that, similar to the $menA^{-}$ and $menB^{-}$ mutants, PS I is functional in the $menD^{-}$ and $menE^{-}$ mutants with plastoquinone-9 rather than phylloquinone in the A$_{1}$ site.

There are physiological consequences to the absence phylloquinone in PS I. The photoautotrophic and photoheterotrophic growth rates of the $menD^{-}$ and $menE^{-}$ strains are significantly slower than the wild type when grown at normal light intensities (40 $\mu$E m$^{-2}$ s$^{-1}$). Furthermore, the $menD^{-}$ and $menE^{-}$ strains are incapable of growing at high light intensities (>120 $\mu$E m$^{-2}$ s$^{-1}$). All mutant strains have doubling times similar to the wild type when grown photoheterotrophically with glucose and DCMU to block PS II function. This suggests that the slower growth rate and high-light phototoxicity of the $menD^{-}$ and $menE^{-}$ strains is an indirect effect caused by an imbalance in the rates of electron transfer between PS I and PS II.

The imbalance in two photosystems was reflected in the markedly different 77-K fluorescence spectra. The PS I to PS II ratio in the wild type was 3.9, whereas the PS I to PS II ratio in the $menD^{-}$ and $menE^{-}$ mutants was 2.7 and 2.1, respectively (Fig. 5). The characteristics of the mutant strains were similar to that of the $menA^{-}$ and $menB^{-}$ mutants, both in an altered PS I to PS II ratio and in high-light sensitivity [23]. The resulting physiological changes in growth rates and oxygen evolution are also similar to the $menA^{-}$ and $menB^{-}$ mutants.

Through gene inactivation, we have demonstrated that the $menD$ and $menE$ genes in *Synechocystis* sp. PCC 6803 are confirmed to be components of the early steps in the

biosynthetic pathway that generates phylloquinone. Also in the absence of phylloquinone of the mutant strains, PS I utilizes plastoquinone as a suitable functional replacement.

### Acknowledgements

This work was supported by the National Science Foundation Grants (MCB-0078264 to P.R.C., and MCB-9723661 and MCB-0117079 to J.H.G.).

### References

[1] R. Bittl, S.G. Zech, P. Fromme, H.T. Witt, W. Lubitz, Biochemistry 36 (1997) 12001–12004.

[2] R. Malkin, in: D. Ort, C.F. Yocum (Eds.), Oxygenic Photosynthesis: The Light Reactions, Kluwer Academic Publishers, Dordrecht, 1996, pp. 313–329.

[3] P.R. Chitnis, Plant Physiol. 111 (1996) 661–669.

[4] R. Malkin, FEBS Lett. 208 (1986) 343–346.

[5] H.U. Schoeder, W. Lockau, FEBS Lett. 199 (1986) 23–27.

[6] Y. Takahashi, K. Hirota, S. Katoh, Photosynth. Res. 6 (1985) 183–192.

[7] J. Biggins, P. Mathis, Biochemistry 27 (1988) 1494–1500.

[8] T.M. Iverson, C. Luna-Chavez, G. Cecchini, D.C. Rees, Science 284 (1999) 1961–1966.

[9] C. Luna-Chavez, T.M. Iverson, D.C. Rees, G. Cecchini, Protein Expr. Purif. 19 (2000) 188–196.

[10] G. Hauska, T. Schoedl, H. Remigy, G. Tsiotis, Biochim. Biophys. Acta 1507 (2001) 260–277.

[11] C. Palaniappan, V. Sharma, M.E. Hudspeth, R. Meganathan, J. Bacteriol. 174 (1992) 8111–8118.

[12] V. Sharma, K. Suvarna, R. Meganathan, M.E. Hudspeth, J. Bacteriol. 174 (1992) 5057–5062.

[13] V. Sharma, R. Meganathan, M.E. Hudspeth, J. Bacteriol. 175 (1993) 4917–4921.

[14] R. Daruwala, O. Kwon, R. Meganathan, M.E. Hudspeth, FEMS Microbiol. Lett. 140 (1996) 159–163.

[15] V. Sharma, M.E. Hudspeth, R. Meganathan, Gene 168 (1996) 43–48.

[16] K. Suvarna, D. Stevenson, R. Meganathan, M.E. Hudspeth, J. Bacteriol. 180 (1998) 2782–2787.

[17] J.R. Driscoll, H.W. Taber, J. Bacteriol. 174 (1992) 5063–5071.

[18] K.F. Hill, J.P. Mueller, H.W. Taber, Arch. Microbiol. 153 (1990) 355–359.

[19] C. Palaniappan, H. Taber, R. Meganathan, J. Bacteriol. 176 (1994) 2648–2653.

[20] H.W. Taber, E.A. Dellers, L.R. Lombardo, J. Bacteriol. 145 (1981) 321–327.

[21] A. Koike-Takeshita, T. Koyama, K. Ogura, J. Biol. Chem. 272 (1997) 12380–12383.

[22] T. Kaneko, S. Sato, H. Kotani, A. Tanaka, E. Asamizu, Y. Nakamura, N. Miyajima, M. Hirosawa, M. Sugiura, S. Sasamoto, T. Kimura, T. Hosouchi, A. Matsuno, A. Muraki, N. Nakazaki, K. Naruo, S. Okumura, S. Shimpo, C. Takeuchi, T. Wada, A. Watanabe, M. Yamada, M. Yasuda, S. Tabata, DNA Res. 3 (1996) 109–136.

[23] T.W. Johnson, G. Shen, B. Zybailov, D. Kolling, R. Reategui, S. Beauparlant, I.R. Vassiliev, D.A. Bryant, A.D. Jones, J.H. Golbeck, P.R. Chitnis, J. Biol. Chem. 275 (2000) 8523–8530.

[24] Y. Sakuragi, B. Zybailov, G. Shen, A.D. Jones, P.R. Chitnis, A. van der Est, R. Bittl, S. Zech, D. Stehlik, J.H. Golbeck, D.A. Bryant, Biochemistry 41 (2002) 394–405.

[25] V.P. Chitnis, Q. Xu, L. Yu, J.H. Golbeck, H. Nakamoto, D.L. Xie, P.R. Chitnis, J. Biol. Chem. 268 (1993) 11678–11684.

[26] L.B. Smart, S.L. Anderson, L. McIntosh, EMBO J. 10 (1991) 3289–3296.

[27] R. Rippka, J. Deruelles, J.B. Waterbury, M. Herdman, R.Y. Stanier, J. Gen. Microbiol. 111 (1979) 1–61.

[28] J. Sun, A. Ke, P. Jin, V.P. Chitnis, P.R. Chitnis, Methods Enzymol. 297 (1998) 124–139.

[29] D. Arnon, Plant Physiol. 24 (1949) 1–14.

[30] G. MacKinney, J. Biol. Chem. 268 (1941) 11678–11684.

[31] G. Shen, J.J. Eaton-Rye, W.F. Vermaas, Biochemistry 32 (1993) 5109–5115.

[32] G.Z. Shen, D.A. Bryant, Photosynth. Res. 44 (1995) 41–53.

[33] A.Y. Semenov, I.R. Vassiliev, A. van der Est, M.D. Mamedov, B. Zybailov, G. Shen, D. Stehlik, B.A. Diner, P.R. Chitnis, J.H. Golbeck, J. Biol. Chem. 275 (2000) 23429–23438.

[34] B. Zybailov, A. van der Est, S.G. Zech, C. Teutloff, T.W. Johnson, G. Shen, R. Bittl, D. Stehlik, P.R. Chitnis, J.H. Golbeck, J. Biol. Chem. 275 (2000) 8531–8539.

[35] A.B. Hope, Biochim. Biophys. Acta 1456 (2000) 5–26.

[36] W.D. Schubert, O. Klukas, N. Krauss, W. Saenger, P. Fromme, H.T. Witt, J. Mol. Biol. 272 (1997) 741–769.

[37] J. Biggins, P. Mathis, Biochemistry 27 (1988) 1494–1500.

[38] M. Careri, A. Mangia, P. Manini, N. Taboni, Fresenius’ J. Anal. Chem. 355 (1996) 48–56.

[39] I.R. Vassiliev, Y.S. Jung, M.D. Mamedov, A. Semenov, J.H. Golbeck, Biophys. J. 72 (1997) 301–315.
