
Available online at www.sciencedirect.com

SCIENCE DIRECT®

Archives of Biochemistry and Biophysics 411 (2003) 277–288

Isolation and characterization of 27-O-demethylrifamycin SV methyltransferase provides new insights into the post-PKS modification steps during the biosynthesis of the antitubercular drug rifamycin B by *Amycolatopsis mediterranei* S699

Jun Xu, Taifo Mahmud,* and Heinz G. Floss

Department of Chemistry, University of Washington, Box 351700, Seattle, WA 98195-1700, USA

Received 12 November 2002, and in revised form 2 January 2003

---

### Abstract

The gene *riforf14* in the rifamycin biosynthetic gene cluster of *Amycolatopsis mediterranei* S699, producer of the antitubercular drug rifamycin B, encodes a protein of 272 amino acids identified as an AdoMet: 27-O-demethylrifamycin SV methyltransferase. Frameshift inactivation of *riforf14* generated a mutant of *A. mediterranei* S699 that produces no rifamycin B, but accumulates 27-O-demethylrifamycin SV (DMRSV) as the major new metabolite, together with a small quantity of 27-O-demethyl-25-O-desacyl-rifamycin SV (DMDARSV). Heterologous expression of *riforf14* in *Escherichia coli* yielded a 33.8-kDa polyhistidine-tagged polypeptide, which efficiently catalyzes the methylation of DMRSV to rifamycin SV, but not that of DMDARSV or rifamycin W. 27-O-Demethylrifamycin S was methylated poorly, if at all, by the enzyme to produce rifamycin S. The purified enzyme does not require a divalent cation for catalytic activity. While Ca²⁺ or Mg²⁺ inhibits the enzyme activity slightly, Zn²⁺, Ni²⁺, and Co²⁺ are strongly inhibitory. The *Kₘ* values for DMRSV and S-adenosyl-L-methionine (AdoMet) are 18.0 and 19.3 μM, respectively, and the *Kcat* is 87 s⁻¹. The results indicate that DMRSV is a direct precursor of rifamycin SV and that acetylation of the C-25 hydroxyl group must precede the methylation reaction. They also suggest that rifamycin S is not the precursor of rifamycin SV in rifamycin B biosynthesis, but rather an oxidative shunt-product.

© 2003 Elsevier Science (USA). All rights reserved.

Keywords: AdoMet; 27-O-demethylrifamycin SV methyltransferase; 27-O-demethylrifamycin SV; 27-O-demethyl-25-O-desacylrifamycin SV; *Amycolatopsis mediterranei*; *riforf14*

---

Rifamycin B, a prominent member of the ansamycin family of antibiotics, is produced by the actinomycete *Amycolatopsis mediterranei* [1,2]. Its synthetically modified derivatives, such as rifampicin, are used clinically in the treatment of tuberculosis, leprosy, and AIDS-related mycobacterial infections [3–5]. The potent antibacterial activity of the rifamycins is due to their specific inhibition of bacterial DNA-dependent RNA polymerases [6,7], enzymes that are responsible for mRNA biosynthesis in the cell. However, they have relatively low if any activity against eukaryotic RNA polymerases [8,9]. The biosynthesis of the rifamycins has been studied extensively by feeding experiments, classical mutagenesis experiments, and, most recently, genetic and biochemical analysis of the biosynthetic gene cluster [10,11]. The polyketide framework of rifamycin B is assembled from 3-amino-5-hydroxybenzoic acid (AHBA)¹ as the starter molecule

---

¹ Abbreviations used: AHBA, 3-amino-5-hydroxybenzoic acid; orf, open reading frames; AdoMet, S-adenosyl-L-methionine; LBBS, LB medium containing 1 M betaine and 2.5 M sorbitol; SSC, standard saline citrate; ESI-MS, electrospray ionization mass spectrometry; DMRSV, 27-O-demethylrifamycin SV; DMDARSV, 27-O-demethyl-25-O-desacylrifamycin SV; DMRS, 27-O-demethylrifamycin S; IPTG, isopropyl-β-D-thiogalactopyranoside; PKS, polyketide synthase.

---

This work was supported by Research Grant AI 20264 from the National Institutes of Health.

Corresponding author. Fax: 1-206-543-8318.

E-mail address: mahmud@chem.washington.edu (T. Mahmud).

0003-9861/03/$ - see front matter © 2003 Elsevier Science (USA). All rights reserved.

doi:10.1016/S0003-9861(03)00004-3

[12,13], which arises from a new metabolic route, the aminoshikimate pathway [14], by chain extension with 2 acetate and 8 propionate units [15]. This is followed by a series of reactions modifying the initially formed polyketide to the final product, rifamycin B. A 96-kb cluster of genes (*rif*), including a set of five large open reading frames (orfs) encoding a type I PKS, responsible for the biosynthesis of rifamycin B has been cloned, sequenced, and analyzed [16–18]. Progress has been made in the past few years toward the understanding of the biosynthesis of AHBA [19–24] and of the assembly of the ansa chain, the long aliphatic bridge, of rifamycin B [25–30]. However, little that is new has so far been learned, beyond sequences and putative functions deduced from sequence homologies, about the post-PKS modification genes and the details of the tailoring processes leading from the initial PKS product to rifamycin B.

Based on predictions from the domain structure of the *rif* PKS [16–18] and on the results of feeding experiments with \[^{7-13}C\], \[^{18}O_2\]\]AHBA, which showed that one \[^{18}O\) is retained at C-8 of rifamycin B [12,26], it has been proposed [16,26] that the earliest macrocyclic intermediate in rifamycin B biosynthesis is proansamycin X (Fig. 1). Its dehydrated analog, proansamycin B, had also been postulated [27,31] as the earliest intermediate, based on the structures of the linear polyketides accumulated by a mutant lacking the downloading enzyme

RifF [26,27] and on the fact that its C-34a hydroxy analog, protorifamycin I, was the least functionalized macrocyclic rifamycin isolated [31]. However, proansamycin B has recently been isolated and it as well as protorifamycin I has been shown to be an intermediate not in rifamycin B formation but in a shunt pathway leading to 5-deoxyrifamycin B [30]. From proansamycin X, further tailoring reactions consecutively give rifamycin W, rifamycin S (SV), and rifamycin B. In 1969, Lancini et al. [32] proposed a pathway for the transformation of rifamycin S into rifamycins B and L via rifamycin SV and rifamycin O. The latter has been postulated as a common progenitor of rifamycins B and L. However, Gisalba et al. [33] later concluded, based on transformation experiments with rifamycin S in permeabilized mycelia of *A. mediterranei*, that rifamycins B and L are derived from rifamycin S or SV by different pathways using different three-carbon precursors for their glycolic acid moieties. While the bioconversion from rifamycin S or SV to rifamycin B had been investigated to some extent, although the origin of the attached two-carbon chain has remained unclear, details of the biosynthetic pathway prior to rifamycin SV are not well understood. Conversion of rifamycin W to rifamycin S or SV must involve a number of oxidation steps, as well as acetylation and methylation. Analysis of the rifamycin biosynthetic gene cluster [16] revealed a


![image](https://i.imgur.com/your_image_url.png)


Fig. 1. Originally proposed biosynthetic pathway to rifamycin B. Rifamycin S was assumed to be an intermediate in rifamycin B biosynthesis by *A. mediterranei* S699, and rifamycin SV was postulated to be converted to rifamycins L and B by independent routes.
series of putative post-PKS modifying genes, including  
several which encode proteins highly homologous to  
P450-dependent monooxygenases, some of which may  
be involved in the conversion of rifamycin W to rifa-  
mycin SV. A gene, *rif orf14*, encoding a protein ho-  
mologous to methyltransferases, was also found, but  
surprisingly, no obvious candidate gene to encode an  
acetyltransferase could be located in the 96-kb rifamycin  
biosynthesis gene cluster.  

Here we report the identification and characteriza-  
tion of RifOrf14 by gene inactivation and heterologous  
expression as an S-adenosyl-L-methionine (AdoMet)-  
dependent methyltransferase responsible for the O-  
methylation at C-27 of the ansa chain during rifamycin  
B biosynthesis.  

### Experimental procedures  

#### Microorganisms, culture conditions, and vectors  

The rifamycin B-producing strain, *A. mediterranei*  
S699, was obtained from Professor Giancarlo Lancini at  
the former Lepetit Laboratories (Geranzano, Italy). For  
sporulation it was grown at 30°C on agar plates con-  
taining YMG medium [20]. YMG liquid medium was  
used for both seed and production cultures. *Escherichia*  
*coli* XL1-Blue (Stratagene) was used as host for subcl-  
oning and *E. coli* BL21(DE3)pLysS (Stratagene) for  
gene expression. pBluescript II KS(−) was used as vector  
for subcloning and gene inactivation, whereas pRSETB  
(Invitrogen) was used for gene expression. *A. mediter-  
ranei* transformants were selected with hygromycin B at  
70 μg/ml in both solid and liquid media. *E. coli* strains  
were grown in LB [34] or LB containing 1 M betaine and  
2.5 M sorbitol (LBBS) [20] media supplemented with  
carbenicillin (100 μg/ml), hygromycin B (70 μg/ml), or  
carbenicillin (100 μg/ml)/chloramphenicol (25 μg/ml) for  
selection of plasmids.  

#### DNA manipulation  

Routine genetic procedures such as genomic and  
plasmid DNA isolations, restriction endonuclease di-  
gestions, alkaline phosphatase treatments, DNA liga-  
tions, and other DNA manipulations were performed  
according to standard techniques [34]. DNA fragments  
were excised from agarose gels and residual agarose  
was removed with the QiaQuick Gel Extraction Kit  
(Qiagen). For Southern hybridizations, the genomic  
DNA was immobilized on a Hybond N+ membrane  
(Amersham Pharmacia Biotech). Hybridization was  
performed at 68°C for 5 h using nick-translated  
[³²P]dCTP (Perkin-Elmer Life Sciences)-labeled DNA  
probes. Stringency washes were done with 0.1× SSC  
(34) at 68°C.  

---

### Instrumentation  

Electrospray ionization mass spectrometry (ESI-MS)  
was carried out on a Bruker-Esquire ion trap mass  
spectrometer with electrospray, APCI, or nanospray  
ionization sources, and high resolution ESI-MS was  
carried out on a Fison VG Quattro II electrospray  
ionization mass spectrometer. Proton NMR spectra  
were recorded on a Bruker DRX-499 spectrometer with  
an SGI O2 computer. An ISF-4-V culture shaker (Adolf  
Kuhner AG, Birsfelden, Switzerland) was used for the  
fermentation of *A. mediterranei* S699. Ultraviolet spec-  
troscopy was carried out on a Hewlett Packard 8452A  
diode array spectrophotometer. Radioactive samples  
were counted in Bio-Safe II biodegradable scintillation  
cocktail (Research Products International Corp.) in a  
Beckman LS 1801 scintillation counter. For HPLC, a  
Beckman System Gold programmable solvent module  
was used with a Beckman System Gold diode array  
detector module. FPLC was carried out on a Pharmacia  
Biotech Pump P-500 with a controller LCC-501 Plus  
and a Pharmacia Monitor UV-M.  

---

### Gene inactivation  

A 5.87-kb *StuI–EcoRI* DNA fragment containing  
*orf14* from plasmid pTM1303, generated from cosmid  
clone pFKN108 [16], was subcloned into the *EcoRV–  
EcoRI* site of pBluescript II KS(−) to give pTM1401  
(Fig. 2). Frameshift inactivation was carried out by  
partial digestion of pTM1401 with *XhoI* endonuclease,  
followed by treatment with DNA polymerase I, Klenow  
fragment (GIBCO), and 1 μmol of each deoxynucleotide  
triphosphate (GIBCO), to generate blunt ends at the cut  
site. Ligation of the product with T4 DNA ligase (NEB)  
yielded pTM1401X. Cloning of a 1.7-kb *KpnI* fragment  
from pIJ5607 into the *KpnI* site of pTM1401X to gen-  
erate pTM1401XH introduced the hygromycin resis-  
tance gene (*hygR*) [21] in the same orientation as *orf14*.  
Transformation of heat-denatured pTM1401XH into *A.*  
*mediterranei* S699 was carried out by electroporation at  
12.5 kV/cm, and the transformants were immediately  
transferred to a 500-ml Erlenmeyer flask containing  
50 ml YMG medium and incubated at 200 rpm and  
28°C for 8–24 h. Single-crossover mutants were selected  
by plating out the cells onto YMG plates containing  
hygromycin B, which were incubated at 30°C for 7–14  
days. The genotypes of mutant candidates were con-  
firmed by Southern hybridization. Double-crossover  
mutants were selected after three rounds of cultivation  
of a single-crossover mutant in YMG liquid medium by  
replica plating on YMG agar plates with and without  
hygromycin B. Colonies that lost the antibiotic resis-  
tance were selected and genotypically verified by  
Southern hybridization as either mutants or revertants  
to wild type. The expected mutant strain, MT1401XH,  
A. mediterranei S699
Chromosomal DNA

KpnI
Apal
DraII
EcoO109I
XhoI
SalI
AccI
HindIII
ClaI
Bsp106I
HindIII

StuI
20
12
13
14
Xhol
15
EcoRI
16
ScaI

hyg<sup>R</sup>

pTM1401XH
8.8 kb

PstI
Smal
BamHI
SpeI
Xbal
NotI
EagI
BsiXI
SacI

3259 bp
5870 bp

ScaI
StuI
BamHI
BamHI
XhoI
EcoRI

pTM1303
C
C

pFKN108
part of Rif
PKS genes
AHBA Biosynthesis genes
regulatory and post-PKS modification genes

Fig. 2. Construction of the *orf14* inactivated mutant MT1401XH. The region of the post-PKS genes in the *rif* gene cluster of *A. mediterranei* S699 undergoing double homologous recombination with pT1401XH is shown at the top. pTM1401XH was constructed from pTM1303, a 9129-bp *ScaI/EcoRI* fragment of pFKN108 ligated into pBlueScript KS(−), with an additional *hyg<sup>R</sup>* resistance marker. *C* indicates that pTM1303 is a circular plasmid.

in which the original *orf14* has been replaced by the disrupted gene from pTM1401XH, was used for further analysis.

**Isolation and structure elucidation of rifamycin intermediates**

Spores of the mutant strain MT1401XH were initially grown in YMG medium for 3 days at 28°C and 200 rpm. This seed culture was used to inoculate (10%, v/v) 20 500-ml Erlenmeyer flasks containing 50 ml of YMG medium. After incubation for 10 days under the same conditions, the cultures were centrifuged, the pooled supernatants acidified to pH 3 with 1 N HCl, and the metabolites extracted with 3 × 100 ml of ethyl acetate. Rifamycin-related compounds were purified by repeated SiO<sub>2</sub> gravity column chromatography (step gradient ethyl acetate to ethyl acetate:acetone 9:1) to give 27-O-demethylrifamycin SV (DMRSV, 163 mg) and 27-O-demethyl-25-O-desacylrifamycin SV (DMDARSV, 5 mg).

DMRSV, yellow amorphous solid. Low-resolution ESI-MS *m/z*: 706.3 (M + Na)<sup>+</sup>. High-resolution ESI-MS: calcd for C<sub>36</sub>H<sub>45</sub>NO<sub>12</sub>Na [M + Na]: 706.2839; found, 706.2828. <sup>1</sup>H NMR (D<sub>2</sub>O, 499 MHz) δ (ppm)

0.35 (3H, d, *J* = 6.8 Hz, H-34), 0.53 (3H, d, *J* = 6.8 Hz, H-33), 0.76 (6H, br d, *J* = ca. 5 Hz, H-31, H-32), 1.07 (1H, m, H-26), 1.23 (1H, m, H-24), 1.61 (3H, s, H-13), 1.76 (1H, m, H-22), 1.86 (3H, s, H-30), <sup>2</sup> 1.88 (3H, s, H-14), <sup>2</sup> 1.91 (3H, s, H-36), 2.21 (1H, m, H-20), 2.98 (1H, d, *J* = 10.3 Hz, H-23), 3.68 (1H, d, *J* = 9.7 Hz, H-21), 3.79 (1H, d, *J* = 8.0 Hz, H-27), 4.85 (1H, d, *J* = 10.9 Hz, H-25), 5.17 (1H, dd, *J* = 8.0, 12.7 Hz, H-28), 6.03 (1H, dd, *J* = 7.8, 15.7 Hz, H-19), 6.16 (1H, d, *J* = 12.7 Hz, H-29), 6.19 (1H, d, *J* = 10.7 Hz, H-17), 6.35 (1H, dd, *J* = 10.7, 15.7 Hz, H-18), 7.04 (s, H-3).

DMDARSV, yellow amorphous solid. ESI-MS *m/z*: 664 (M + Na)<sup>+</sup>. <sup>1</sup>H NMR (D<sub>2</sub>O, 499 MHz) δ 0.17 (3H, d, *J* = 6.9 Hz, H-34), 0.70 (3H, d, *J* = 6.8 Hz, H-33), 1.01 (3H, d, *J* = 6.7 Hz, H-32), <sup>2</sup> 1.07 (3H, d, *J* = 7.0 Hz, H-31), <sup>2</sup> 1.37 (1H, m, H-26), 1.53 (1H, m, H-24), 1.80 (3H, s, H-13), 1.99 (1H, m, H-22), 2.13 (3H, s, H-14), <sup>3</sup> 2.16 (3H, s, H-30), <sup>3</sup> 2.42 (1H, m, H-20), 3.54 (1H, d, *J* = 10.6 Hz, H-23), 3.86 (1H, d, *J* = 10.7 Hz, H-21), 3.95 (1H, d, *J* = 8.6 Hz, H-27), 4.50 (1H, d, *J* = 5.8 Hz, H-25), 5.40 (1H, dd, *J* = 5.75, 12.6 Hz, H-28), 6.19 (1H, dd, *J* = 7.1, 15.2 Hz, H-19), 6.30 (1H, d, *J* = 12.6 Hz,

<sup>2</sup> These assignments may be interchanged.
<sup>3</sup> These assignments may be interchanged.
H-29), 6.17 (1H, d, $J = 10.5$ Hz, H-17), 6.30 (1H, dd, $J = 10.5$, 15.2 Hz, H-18), 7.59 (s, H-3).

**Oxidation of DMRSV to DMRS using divalent metal ions as catalysts**

One millimeter of a solution of DMRSV in 50 mM Tris–HCl buffer, pH 7.0, was placed in a UV spectrophotometer cuvette and the absorption at 445 nm was measured every 2 s for the initial 180 s. The reaction was started by quickly adding 0.1 mM respective divalent cation ($\mathrm{Zn}^{2+}$, $\mathrm{Ni}^{2+}$, $\mathrm{Mn}^{2+}$, $\mathrm{Mg}^{2+}$, $\mathrm{Fe}^{2+}$, $\mathrm{Cu}^{2+}$, $\mathrm{Co}^{2+}$, $\mathrm{Ca}^{2+}$) and the measurement was continued for a total of 1500 s. The oxidation of DMRSV and the formation of 27-$O$-demethylrifamycin S (DMRS) was followed by a decrease in the absorbance at 445 nm. The product was then verified by ESI-MS.

**Gene expression**

orf14 was PCR amplified from pTM1401 as template using the primers 5′-AAGGATCCATATGTGACGAA GCCGACT-3′ (forward primer) and 5′-ATTAAGCTT CAGTCGGGGCTTGCGCGC-3′ (reverse primer), introducing *Bam*HI and *Nde*I restriction sites in the forward primer and a *Hind*III site in the reverse primer to facilitate subcloning. PCRs were performed in a thermocycler (MJ Research, PTC-200) under the following conditions: 60 cycles of 30 s at 95 °C, 2 min at 58 °C, and 3 min at 72 °C. The PCR product was purified, subcloned into pBlueScript II KS(−) as a 0.8-kb *Hind*III/*Bam*HI fragment, and amplified in *E. coli* XL1Blue. For expression as a His$_6$ fusion protein, the 0.8-kb *Hind*III/*Bam*HI fragment of the correct clone (pMT1) was moved into the *E. coli* expression vector pRSETB, generating pMT2. The construct was introduced into *E. coli* BL21(DE3)pLysS by heat-pulse transformation, and carbenicillin/chloramphenicol-resistant transformants were selected. *E. coli* BL21(DE3) pLysS/pMT2 was grown in 20 ml LB medium containing carbenicillin and chloramphenicol at 37 °C and 300 rpm for 16 h. The culture was then transferred to 1 L LBBS medium and incubated at 37 °C and 300 rpm. At an OD$_{600}$ of 0.5, isopropyl-$\beta$-d-thiogalactopyranoside (IPTG) was added to a final concentration of 0.2 mM and the incubation was continued at 30 °C and 200 rpm for 24 h. The cells were harvested by centrifugation at 6500 rpm for 5 min and stored frozen at −80 °C until further use.

**Purification of His$_6$-tagged Orf14 protein**

To the thawed cell pellet was added 0.5 ml of lysis buffer (50 mM NaH$_2$PO$_4$, 300 mM NaCl, 10 mM imidazole, pH 8) containing 1 mg/ml lysozyme. The mixture was agitated in a Vortex mixer and incubated on ice for 30 min. The suspension was then sonicated three times

for 15 s each and cell debris was removed by centrifugation at 8000 rpm for 15 min. The protein solution was loaded onto a Ni-NTA spin column (Qiagen) and centrifuged at 2000 rpm for 4 min. The column was washed twice with washing buffer (50 mM NaH$_2$PO$_4$, 300 mM NaCl, 20 mM imidazole, pH 8.0), then the His$_6$-tagged protein was eluted twice with elution buffer (50 mM NaH$_2$PO$_4$, 300 mM NaCl, 250 mM imidazole, pH 8.0). The protein solution was desalted on a PD-10 column (Pharmacia) and concentrated in a Centricon-10 device (Millipore). Protein concentration was measured by the Bradford protein microassay with bovine serum albumin as standard.

**Enzyme assays**

Incubations to determine the substrate specificity contained 50 μM substrate, 5 μg of purified protein, 50 mM Tris–HCl buffer, pH 8.0, and 0.1 mM AdoMet in a volume of 50 μl. The mixture was incubated at 30 °C for 20 min, acidified with 50 μl of 0.1 N HCl, and extracted with 50 μl of ethyl acetate. The ethyl acetate solution was used directly for mass spectral analysis. To determine the optimal buffer and pH, 50 μM DMRSV was incubated with 5 μg of purified protein, 40 μM S-adenosyl-L-[methyl-$^{14}$C]methionine (sp radioact 59 μCi/ml), 1 mM dithiothreitol, and 1 mM EDTA in the appropriate buffer and at the desired pH (total volume 50 μl) for 20 min at 30 °C. After work up as before, 25 μl of the ethyl acetate extract was used for radioactivity determination. Enzyme kinetics were determined in the same way, using 50 mM Hepes buffer, pH 8.0, and different concentrations of the variable substrate.

**Results**

**Inactivation of rif orf14**

The gene *rif orf14* is located in the rifamycin biosynthetic gene cluster downstream of the AHBA biosynthesis genes in a region of presumed post-PKS modification genes. It is predicted to encode a protein of 272 aa with three highly conserved S-adenosylmethionine binding motifs in its structure (Fig. 3) and strong homology to streptomycete methyltransferases, such as AveD from *Streptomyces avermitilis* [35] (40% identity, 54% similarity) and MitM from *Streptomyces lavendulae* [36,37] (41% identity, 51% similarity). It also shows homology (32% identity, 48% similarity) to the Erg6 protein of the cyanobacterium *Synechocystis* sp. [38]. To determine its role in rifamycin B biosynthesis, we constructed a mutant of *A. mediterranei* carrying an inactivated *orf14*, in which four additional nucleotides had been inserted at the *XhoI* site, yielding a frameshift mutation in *orf14*. The mutant, MT1401XH, grows
Orf14 32 DD--ESA-TLEEAT-TRLTDR AGMLP--LRAGDRLD GCCNGEPAIRMATANDVMV
Erg6 61 GPHGTYRIDRRAQIDLIKE LAWAVPQNSAKPRKLDL DGCGTGSSLYAQHQAEV
AveD 38 PQPGDGS-SPGKAA-DRLTDL IGKLR--GITGRRLDV GCCSGKPAVRALSAPVDV
MitM 42 DSPDSQV-PLAEAT-DRLTDM AERLR--IGAGSRVLDD GCCVGTPGVRA RLSGAHV
UbiG 19 DRESE-FKPLHDIN-PLRVNW DER A---GLAGKELD GCCGGILSEAQAQ-RGASV
consensus 61 . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .NADH or NADPH [33,40]. While NADH or NADPH alone cannot reduce rifamycin S, ascorbate, dithiothreitol, and 2-mercaptoethanol are good reducing agents for the quinone moiety [40]. Studies on the divalent metal ion-catalyzed oxidation of rifamycin SV, including its dependence on pH and the involvement of superoxide, have shown Mn²⁺ to be the most effective catalyst for this reaction [41,42].

To investigate the corresponding oxidation of DMRSV to DMRS, a series of divalent metal ions was tested at 0.1 mM concentration in Tris buffer for their catalytic effect on the oxidation rate, following the reaction by the change in OD₄₄₅ (Fig. 4). It was found that at pH 7.0 Cu²⁺ was by far the most effective catalyst for the oxidation of DMRSV to DMRS (Fig. 4A), whereas at pH 9.0 Mn²⁺ catalyzed the oxidation somewhat more efficiently than Cu²⁺ (Fig. 4B). The latter result is comparable to those reported by Kono on the oxidation of rifamycin SV at pH 9.2 [42].

As reported previously for rifamycin S [33], DMRS could also be reduced to DMRSV by dithiothreitol (data not shown). This reaction goes to completion instantaneously upon the addition of 1 mM reducing agent. The same reduction was also observed when 50 μl of cell-free extract of *A. mediterranei* mutant 0031ABH, in which a 21,053-bp DNA fragment containing most of the putative post-PKS modification genes had been deleted from the chromosome (T. Mahmud, J. Xu, H.G.

Floss, unpublished data), was added to the solution of DMRS.

### Overexpression of the orf14 gene

In order to confirm the function of its gene product as a DMRSV methyltransferase, *orf14* was overexpressed in *E. coli*. A 0.8-kb DNA fragment encoding the methyltransferase was amplified from pTM1401 by polymerase chain reaction, introducing *BamHI* and *NdeI* restriction sites with the forward primer and a *HindIII* site with the reverse primer. The PCR product was subcloned into pBluescript II KS(−) as a *HindIII/BamHI* fragment, amplified in *E. coli* XL1Blue, and then moved into the *E. coli* expression vector pRSETB. Expression in *E. coli* BL21Gold(DE3)pLysS under the control of the T7 promoter was induced by IPTG and gave Orf14 as a soluble His₆-tagged protein. Affinity purification on a Ni-NTA column increased the specific activity of the enzyme 4.4-fold to 4.3 mU/mg, giving a protein that was >95% pure as judged by SDS-polyacrylamide gel electrophoresis (Fig. 5).

### Properties of Orf14

The subunit molecular mass of the His-tagged protein of 33.8 kDa was confirmed by SDS–PAGE. Native PAGE indicated a molecular mass of ~100 kDa, whereas FPLC on a size-fractionation column [Superdex 200 HR 10/30 column (Pharmacia), 0.1 M sodium phosphate buffer–0.3 M sodium chloride 1:1, flow rate 0.4 ml/min] gave a somewhat smaller value of 87 kDa. To determine the optimal buffer and pH conditions for enzyme catalysis, four different buffers (Tris–HCl buffer, pH 7.5, potassium phosphate buffer, pH 6.5 and 7.5; and Hepes buffer, pH 7.5) were used in incubations of the presumed substrate, DMRSV, with Orf14. The results (Fig. 6A) indicate that Orf14 is most active with Hepes

![Figure 4](https://i.imgur.com/yourimageurl.png)

**Fig. 4.** Effect of divalent cations on the oxidation of 27-O-demethylrifamycin SV to 27-O-demethylrifamycin S. The ion to be tested (0.1 mM) was added at 180 s (arrow) and the reaction was monitored by measuring the decrease in the absorbance at 445 nm. (A) Oxidation of 27-O-demethylrifamycin SV to 27-O-demethylrifamycin S catalyzed by different divalent cations in Tris–HCl buffer, pH 7.0. (B) Comparison of the catalytic activity of Cu²⁺ and Mn²⁺ in Tris buffer, pH 9.0.

![Figure 5](https://i.imgur.com/yourimageurl.png)

**Fig. 5.** SDS- and Native PAGE of Orf14. M, marker protein. TP, total protein of cell-free extract of *E. coli* BL21Gold(DE3)pLys/pMT2 after induction with IPTG. SP, soluble protein fraction of TP. PP, purified Rif Orf14 protein after elution from nickel–NTA column. PD, partially denatured (98 °C for 5 min) PP protein.
J. Xu et al. / Archives of Biochemistry and Biophysics 411 (2003) 277–288

**Fig. 6. Catalytic properties of Orf14.** (A) Effect of buffers. (B) Effect of pH. (C) Effect of divalent cations. Reactions contained 50 μM DMRSV, 5 μg of purified protein, 50 mM buffer, 40 μM S-adenosyl-L-[methyl-<sup>14</sup>C]methionine (sp radioact 59 μCi/ml), 1 mM dithiothreitol, and 1 mM cation or 1 mM EDTA in a volume of 50 μl.

addition of EDTA to the reaction mixture did not inhibit the reaction (Fig. 6C); whereas Ca<sup>2+</sup> or Mg<sup>2+</sup> were weakly inhibitory, Zn<sup>2+</sup>, Ni<sup>2+</sup>, and Co<sup>2+</sup> strongly inhibited the methylation of DMRSV catalyzed by Orf14.

### Substrate specificity of Orf14

The isolation of DMRSV as the major new compound accumulated by the mutant MT1401XH lacking a functional *orf14* suggested that DMRSV is a substrate for the methyltransferase encoded by *orf14*. The assays of the enzyme with DMRSV as substrate confirmed this expectation. However, the additional isolation of DMDARSV and the detection of rifamycin W at early stages in the fermentation of this mutant raised the question whether DMRSV is the only or even the natural substrate of the enzyme. Furthermore, since rifamycin S, the quinone corresponding to rifamycin SV, had been postulated as a biosynthetic intermediate [32,33], it was important to determine whether DMRS is also a substrate for methylation and, if it is, whether DMRSV is methylated only after oxidation to DMRS or whether both compounds can be methylated independently. To clarify these questions and to identify the step in the natural biosynthetic pathway to rifamycin B at which 27-O-methylation takes place, we examined the substrate specificity of the purified Orf14 protein. The enzyme was incubated in Tris-HCl buffer, pH 8.0, with AdoMet and DMRSV, DMDARSV, rifamycin W, or DMRS, which had been generated from DMRSV by passing it through an HPLC column (YMC-Pack ODS-AQ, 250 × 10 mm, methanol:water 78:22). No other cofactors and no dithiothreitol were added to the reaction mixtures to minimize potential oxidation/reduction reactions of the substrates during the incubations. Following incubation, the reaction mixtures were analyzed by electrospray mass spectrometry. The results show that DMRSV serves as a good substrate for Orf14 to give exclusively rifamycin SV as product (Fig. 7A), whereas the oxidized form, DMRS, was methylated to only a very minor degree and gave both rifamycin S and rifamycin SV as products (Fig. 7B). No methylation products could be detected in the incubations with DMDARSV and with rifamycin W (Figs. 7C and D), indicating that neither compound is a substrate for the 27-O-methyltransferase.

as the reaction buffer. The methylation of DMRSV by Orf14 was found to have two pH optima at 6.5 and 7.5–8.0 (Fig. 6B). The enzyme reaction was linear with time for up to 30 min and linear with protein for enzyme concentrations up to 5.8 nM. For standard assays, a protein concentration of 2.9 nM, a pH of 8.0, and a reaction time of 20 min were chosen. The kinetic parameters, obtained from Lineweaver-Burk double-reciprocal plots, were *K*<sub>m</sub> values for DMRSV and AdoMet of 18.0 and 19.3 μM, respectively, and a *K*<sub>cat</sub> of 87 s<sup>−1</sup>. Although most AdoMet-dependent methyltransferases require a divalent metal ion for catalytic activity [43–46], this is not the case for the Orf14 protein. Ad-

### Discussion

Like many other biosyntheses of biologically active secondary metabolites, the biosynthetic pathway to rifamycin B in *A. mediterranei* S699 includes a methylation reaction which in this case modifies the ansa-bridge architecture by methylation of the 27-OH group. Based on sequence comparisons a candidate gene for a

J. Xu et al. / Archives of Biochemistry and Biophysics 411 (2003) 277–288

Fig. 7. Electrospray mass spectra of incubation mixtures of Orf14 with different substrates. (A) 27-O-Demethylrifamycin SV. (B) 27-O-Demethylrifamycin S. (C) Rifamycin W. (D) 27-O-Demethyl-25-O-desacylrifamycin SV.

methyltransferase, *orf14*, had been identified in the *rif* biosynthetic gene cluster [16]. AdoMet-dependent methyltransferases share a highly conserved AdoMet binding motif and comparable protein folds within their catalytic domains [47]. This motif is clearly present in Orf14 (Fig. 3). The deduced amino acid sequence of Orf14 has a high homology to MitM, an aziridine *N*-methyltransferase catalyzing the methylation of 9a-demethylmitomycin A to 9-epi-mitomycin B during mitomycin biosynthesis in *S. lavendulae* [37]. It also is homologous to Erg6 from the cyanobacterium *Synechocystis* sp., a putative δ(24)-sterol C-methyltransferase, which is similar to enzymes with the same function present in plants, fungi, and protozoa [48]. As has been pointed out [47], the high degree of conservation of certain motifs suggests that many AdoMet-dependent methyltransferases have probably evolved from a common ancestor. On the other hand, members of this family of enzymes utilize an enormously broad range of substrates, indicating that a diversity of active sites has

evolved among these enzymes to bind and methylate broadly different substrates while maintaining the conserved regions of equivalent structure and function. Sometimes, minor structural differences between proteins can drastically alter substrate recognition. For example, studies on the substrate specificity of plant small-molecule *O*-methyltransferases have revealed that in the extreme, a single amino acid difference can be responsible for different substrate recognition [49,50].

Inactivation of the *orf14* gene in the *A. mediterranei* S699 genome caused the bacteria to lose their ability to produce rifamycin B and led instead to the accumulation of unmethylated rifamycins, DMRSV as well as traces of DMDARSV and, occasionally, rifamycin W. Two of these compounds, DMRSV and DMDARSV, together with 27-demethylrifamycin B, had previously been isolated by Lancini et al. from a mutant of *S. mediterranei* strain A791 generated by random mutagenesis [51]. Although their mutant may well also carry an inactive methyltransferase, like our mutant

MT1401XH, we have never detected 27-demethylrifamycin B in fermentations of MT1401XH.

Heterologous expression of *orf14* gave the methyltransferase as a soluble His₆-tagged protein, which represented over 20% of the soluble *E. coli* protein and was purified 4.4-fold to near homogeneity. Molecular mass estimates for the native protein by native PAGE and FPLC chromatography on a size-fractionation column gave values which suggest that the enzyme exists as a trimer. This is rather unusual, since most methyltransferases exist as monomers, dimers, or tetramers, although there is at least one previous report of a methyltransferase existing as a trimer [43]. The migration of the native protein as an apparent trimer could, however, also be due to a rapid equilibrium between a dimer and a tetramer. A similar phenomenon has been observed in some other AdoMet-dependent methyltransferases [52,53]. It is not known whether the quaternary structure of the enzyme is important for activity or whether the monomeric subunits are catalytically active [52]. The kinetic properties of the enzyme are within the normal range for methyltransferases, with AdoMet and the methyl acceptor binding equally tightly. An unusual feature of the AdoMet:rifamycin SV methyltransferase is the observation that the enzyme does not require Mg²⁺ for catalytic activity, but is actually inhibited by divalent cations, although the inhibitory effects of Mg²⁺ or Ca²⁺ are much less pronounced than those of Fe²⁺, Mn²⁺, Cu²⁺, Zn²⁺, Ni²⁺, and Co²⁺. This contrasts with the fact that most O-methyltransferases from actinomycetes, as well as from other organisms, are dependent on Mg²⁺ for optimal activity. Examples from related organisms are the demethylmacrocin O-methyltransferase and macrocin O-methyltransferase involved in tylosin biosynthesis [43,44], the catechol O-methyltransferase from *Streptomyces griseus* [45], and the 31-O-desmethyl-FK506 O-methyltransferase from *Streptomyces* sp. MA6858 [46].

The substrate specificity studies with the purified recombinant enzyme revealed that DMRSV is indeed the substrate of Orf14, furnishing rifamycin SV, which directly confirms the function of Orf14 as a methyltransferase that catalyzes the O-methylation at C-27. Since DMDARSV is not methylated by the enzyme, it follows that O-methylation at C-27 of rifamycins can take place only after the acetylation of the 25-OH group. Surprisingly, no candidate gene for an acetyltransferase has so far been identified in the *rif* cluster, which led us to consider the possibility that this reaction might be catalyzed nonspecifically by a gene unrelated to the *rif* biosynthetic gene cluster. However, the finding that acetylation seems to occupy such a specific position in the biosynthetic pathway has rendered this notion less plausible and has caused us to resume the search for a gene responsible for this transformation. The observation that rifamycin W is not a substrate for the 27-O-methyltransferase confirms some earlier notions about the biosynthetic pathway leading from an initial polyketide synthase product, now thought to be the hypothetical proansamycin X [16,26,30], to rifamycin B. Based on studies which demonstrated that rifamycin W can be converted into rifamycin B by mycelia of *A. mediterranei* it was suggested [54] that rifamycin W is a

![Chemical Structures](attachment:chemical_structures.png)

Fig. 8. Revised biosynthetic pathway from rifamycin W to rifamycin B.
normal intermediate in the formation of the other rifamycins and that acetylation and methylation of the hydroxyl groups at C-25 and C-27 must be late events in the biosynthetic reaction sequence. The observed substrate specificity of Orf14 confirms this hypothesis, at least as far as the methylation step is concerned. Finally, the finding that DMRS is poorly or not at all methylated by Orf14 sheds new light on the position of rifamycin S in the biosynthetic pathway.

Because it has the same quinone structure as rifamycin W, rifamycin S has generally been considered an intermediate in the biosynthesis of rifamycin B and the immediate precursor of rifamycin SV, which in turn had been proven to be a precursor of rifamycin B [55]. The quinone/hydroquinone pair rifamycin S/SV is readily interconverted, and Ghisalba et al. [33] observed that permeabilized cells of *N. mediterranei* and *E. coli* W71 ETH in the presence of NADH efficiently reduced rifamycin S to rifamycin SV. They concluded that this interconversion is catalyzed by a nonspecific enzyme and that the quinone/hydroquinone equilibrium in the cell depends on the physiological redox state of the cell. They also noted that all mutants blocked in the last step of rifamycin B formation, the attachment of the glycolate side chain, isolated by them and by Lancini and Hengeler [55] accumulated mostly rifamycin SV and only traces of rifamycin S. Although these observations are compatible with rifamycin S being an intermediate in the biosynthesis of rifamycin SV and B, they certainly do not prove such a relationship. Our results show that the redox pair DMRS/DMRSV behaves in the same way as the rifamycin S/SV pair, i.e., the hydroquinone is air-oxidized to the quinone in the presence of cations such as Mn²⁺ or Cu²⁺ and the quinone is reduced to the hydroquinone by reducing agents such as dithiothreitol. Importantly, cell-free extracts of *A. mediterranei* reduce DMRS efficiently to DMRSV, even in the absence of NADH, suggesting that DMRSV is probably the predominant form of this pair present in the cell. This together with the observation that DMRS is a poor substrate, if any, for the 27-O-methyltransferase argues against the intermediacy of rifamycin S in the formation of rifamycin SV. Rather, the reduction of the quinone to the hydroquinone seems to occur prior to the 27-O-methylation, either at the stage of DMRS or earlier in the pathway. The small degree of formation of rifamycin S and SV in the methylation reaction with DMRS as substrate may be due to a direct methylation of DMRS, although at a probably physiologically insignificant rate. More likely, however, it is due to reduction to DMRSV, methylation, and subsequent partial reoxidation to rifamycin S due to the presence of a large pool of the quinone, DMRS. Either way, it is clear from the results that the normal biosynthetic pathway involves methylation of DMRSV to rifamycin SV and that rifamycin S is not a precursor of rifamycin SV but an oxidative shunt product in the pathway of rifamycin B biosynthesis.

Many steps in the biosynthesis of rifamycin B are still unclear, such as the loss of a methyl group from C-28 of the original polyketide product, the mechanism of the cleavage of and oxygen insertion into the ansa chain, and the origin and mode of attachment of the glycolate moiety of rifamycin B. However, based on the present results we can at least refine the biosynthetic pathway to this compound, as shown in Fig. 8, to define the position of the methylation step and to redefine the role assigned to rifamycin S.

## Acknowledgments

The authors thank Dr. Ross F. Lawrence, University of Washington Mass Spectrometry Center, for his help in carrying out the high-resolution ESI-MS, as well as Mr. Po-Chieh Chang and Ms. Hien Pham for their technical assistance. Acquisition of the Bruker Esquire ion trap mass spectrometer was supported by the National Science Foundation through Grant 9807748.

## References

[1] W. Oppolzer, V. Prelog, P. Sensi, Experientia 20 (1964) 336–339.

[2] W. Oppolzer, V. Prelog, Helv. Chim. Acta 56 (1973) 2287–2314.

[3] K.A. Sepkowitz, J. Raffalli, L. Riley, T.E. Kiehn, D. Armstrong, Clin. Microbiol. Rev. 8 (1995) 180–199.

[4] C.A. Peloquin, Clin. Pharmacokinet. 32 (1997) 132–144.

[5] M. Ramos-e-Silva, P.F. Rebello, Am. J. Clin. Dermatol. 2 (2001) 203–211.

[6] A. Sippel, G. Hartmann, Biochim. Biophys. Acta 157 (1968) 218–219.

[7] E.A. Campbell, N. Korzheva, A. Mustaev, K. Murakami, S. Nair, A. Goldfarb, S.A. Darst, Cell 104 (2001) 901–912.

[8] W. Wehrli, M. Staehelin, Biochim. Biophys. Acta 182 (1969) 24–29.

[9] W. Wehrli, Top. Curr. Chem. 72 (1977) 21–49.

[10] G. Lancini, B. Cavalleri, in: W.R. Strohl (Ed.), Biotechnology of Antibiotics, second ed., Dekker, New York, 1997, pp. 521–549.

[11] H.G. Floss, T.-W. Yu, Curr. Opin. Chem. Biol. 3 (1999) 592–597.

[12] J.J. Kibby, I.A. McDonald, R.W. Rickards, J. Chem. Soc. Chem. Commun. (1980) 768–769.

[13] O. Ghisalba, J. Nüesch, J. Antibiot. 34 (1981) 64–71.

[14] H.G. Floss, Nat. Prod. Rep. 14 (1997) 433–452.

[15] M. Brufani, D. Kluepfel, G.C. Lancini, J. Leitich, A.S. Mesentsev, V. Prelog, F.P. Schmook, P. Sensi, Helv. Chim. Acta 56 (1973) 2315–2323.

[16] P.R. August, L. Tang, Y.J. Yoon, S. Ning, R. Müller, T.-W. Yu, M. Taylor, D. Hoffmann, C.-G. Kim, X. Zhang, C.R. Hutchinson, H.G. Floss, Chem. Biol. 5 (1998) 69–79.

[17] T. Schupp, C. Toupet, N. Engel, S. Goff, FEMS Microbiol. Lett. 159 (1998) 201–207.

[18] L. Tang, Y.J. Yoon, C.Y. Choi, C.R. Hutchinson, Gene 216 (1998) 255–265.

[19] C.-G. Kim, A. Kirschning, P. Bergon, P. Zhou, E. Su, B. Sauerbrei, S. Ning, Y. Ahn, M. Breuer, E. Leistner, H.G. Floss, J. Am. Chem. Soc. 118 (1996) 7486–7491.
[20] C.-G. Kim, T.-W. Yu, C.B. Fryhle, S. Handa, H.G. Floss, J. Biol. Chem. 273 (1998) 6030–6040.
[21] T.-W. Yu, R. Müller, M. Müller, X. Zhang, G. Draeger, C.-G. Kim, E. Leistner, H.G. Floss, J. Biol. Chem. 276 (2001) 12546–12555.
[22] J. Guo, J.W. Frost, J. Am. Chem. Soc. 124 (2002) 528–529.
[23] J. Guo, J.W. Frost, J. Am. Chem. Soc. 124 (2002) 10642–10643.
[24] K. Arakawa, R. Müller, T. Mahmud, T.-W. Yu, H.G. Floss, J. Am. Chem. Soc. 124 (2002) 10644–10645.
[25] D. Hunziker, T.-W. Yu, C.R. Hutchinson, H.G. Floss, C. Khosla, J. Am. Chem. Soc. 120 (1998) 1092–1093.
[26] T.-W. Yu, Y. Shen, Y. Doi-Katayama, L. Tang, C. Park, B.S. Moore, C.R. Hutchinson, H.G. Floss, Proc. Natl. Acad. Sci. USA 96 (1999) 9051–9056.
[27] A. Stratmann, C. Toupet, W. Schilling, R. Traber, L. Oberer, T. Schupp, Microbiology 145 (1999) 3365–3375.
[28] Y. Doi-Katayama, Y.J. Yoon, C.-Y. Choi, T.-W. Yu, H.G. Floss, C.R. Hutchinson, J. Antibiot. 53 (2000) 484–495.
[29] S. Admiraal, C.T. Walsh, C. Khosla, Biochemistry 40 (2001) 6116–6123.
[30] A. Stratmann, T. Schupp, C. Toupet, W. Schilling, L. Oberer, R. Traber, J. Antibiot. 55 (2002) 396–406.
[31] O. Ghisalba, P. Traxler, J. Nüesch, J. Antibiot. 31 (1978) 1124–1131.
[32] G.C. Lancini, G.G. Gallo, G. Sartori, P. Sensi, J. Antibiot. 22 (1969) 369–377.
[33] O. Ghisalba, R. Roos, T. Schupp, J. Nüesch, J. Antibiot. 35 (1982) 74–80.
[34] J. Sambrook, E.F. Fritsch, T. Maniatis, Molecular Cloning: A Laboratory Manual, second ed., Cold Spring Harbor Laboratory, Cold Spring Harbor, NY, 1989.
[35] H. Ikeda, T. Nonomiya, M. Usami, T. Ohta, S. Omura, Proc. Natl. Acad. Sci. USA 96 (1999) 9509–9514.
[36] Y. Mao, M. Varoglu, D.H. Sherman, Chem. Biol. 6 (1999) 251–263.
[37] M. Varoglu, Y. Mao, D.H. Sherman, J. Am. Chem. Soc. 123 (2001) 6712–6713.
[38] T. Kaneko, S. Sato, H. Kotani, A. Tanaka, E. Asamizu, Y. Nakamura, N. Miyajima, M. Hirosawa, M. Sugiura, S. Sasamoto, T. Kimura, T. Hosouchi, A. Matsuno, A. Muraki, N. Nakazaki, K. Naruo, S. Okumura, S. Shimpo, C. Takeuchi, T. Wada, A.

Watanabe, M. Yamada, M. Yasuda, S. Tabata, DNA Res. 3 (1996) 109–136.
[39] E. Martinelli, G.G. Gallo, P. Antonini, R.J. White, Tetrahedron 30 (1974) 3087–3091.
[40] Y. Kono, J. Biochem. 91 (1982) 381–395.
[41] M.C. Scrutton, FEBS Lett. 78 (1977) 216–220.
[42] Y. Kono, J. Biochem. 91 (1982) 1789–1794.
[43] A.J. Kreuzman, J.R. Turner, W.-K. Yeh, J. Biol. Chem. 263 (1988) 15626–15633.
[44] N.J. Bauer, A.J. Kreuzman, J.E. Dotzlaf, W.-K. Yeh, J. Biol. Chem. 263 (1988) 15619–15625.
[45] K. Dhar, J.P.N. Rosazza, Appl. Environ. Microbiol. 66 (2000) 4877–4882.
[46] A. Shafiee, H. Motomedi, T. Chen, Eur. J. Biochem. 225 (1994) 755–764.
[47] G. Schluckebier, M. O’Gara, W. Saenger, X. Cheng, J. Mol. Biol. 247 (1995) 16–20.
[48] W.D. Nes, Biochim. Biophys. Acta 1529 (2000) 63–88.
[49] C. Zubieta, X.-Z. He, R.A. Dixon, J.P. Noel, Nat. Struct. Biol. 8 (2001) 271–279.
[50] D.R. Gang, N. Lavid, C. Zubieta, F. Chen, T. Beuerle, E. Lewinsohn, J.P. Noel, E. Pichersky, Plant Cell 14 (2002) 505–519.
[51] G.C. Lancini, C. Hengeller, P. Sensi, in: Progress in Antimicrobial and Anticancer Chemotherapy, Proceedings of the 6th International Congress of Chemotherapy, vol. II, University of Tokyo Press, Tokyo, 1970, pp. 1166–1173.
[52] X. Zhang, L. Zhou, X. Cheng, EMBO J. 19 (2000) 3509–3519.
[53] J. Tang, J.D. Gary, S. Clarke, H.R. Herschman, J. Biol. Chem. 273 (1998) 16935–16945.
[54] R.J. White, E. Martinelli, G. Lancini, Proc. Natl. Acad. Sci. USA 71 (1974) 3260–3264.
[55] G. Lancini, C. Hengeller, J. Antibiot. 22 (1969) 637–638.
[56] C.K. Stover, X.Q. Pham, A.L. Erwin, S.D. Mizoguchi, P. Warrener, M.J. Hickey, F.S.L. Brinkman, W.O. Hufnagle, D.J. Kowalik, M. Lagrou, R.L. Garber, L. Goltry, E. Tolentino, S. Westbrock-Wadman, Y. Yuan, L.L. Brody, S.N. Coulter, K.R. Folger, A. Kas, K. Larbig, R. Lim, K. Smith, D. Spencer, G.K.-S. Wong, Z. Wu, I.T. Paulsen, J. Reizer, M.H. Saier, R.E.W. Hancock, S. Lory, M.V. Olson, Nature 406 (2000) 959–964.
