
MICROBIOLOGY AND MOLECULAR BIOLOGY REVIEWS, Mar. 2003, p. 86–156
1092-2172/03/$08.00+0 DOI: 10.1128/MMBR.67.1.86–156.2003
Copyright © 2003, American Society for Microbiology. All Rights Reserved.

# Bacteriophage T4 Genome†

Eric S. Miller,${}^{1*}$ Elizabeth Kutter,${}^{2}$ Gisela Mosig,${}^{3‡}$ Fumio Arisaka,${}^{4}$ Takashi Kunisawa,${}^{5}$ and Wolfgang Rüger${}^{6}$

Department of Microbiology, North Carolina State University, Raleigh, North Carolina 27695-7615${}^{1}$; The Evergreen State College, Olympia, Washington 98505${}^{2}$; Department of Biological Sciences, Vanderbilt University, Nashville, Tennessee 37232${}^{3}$; Department of Molecular and Cellular Assembly, Tokyo Institute of Technology, Yokohama 226-8501${}^{4}$ and Department of Applied Biological Sciences, Science University of Tokyo, Noda 278-8510${}^{5}$ Japan; and Faculty for Biology, Ruhr-University-Bochum, 44780 Bochum, Germany${}^{6}$

## T4 GENES TO GENOME
87

### NUCLEOTIDE SKEW IN THE T4 GENOME
88

### IDENTIFYING T4 GENES
88

#### Computational Strategies for Gene Assignment
88

#### Characterized T4 Genes and the Early Genetics
104

#### ORFs of Unknown Function and Host Lethality
106

### PROMOTERS AND TRANSCRIPTION FUNCTIONS
107

#### Early Transcription
107

#### Middle Transcription
109

#### Late Transcription
110

#### Microarray Analysis of T4 Transcription
110

#### Transcription Termination and Predicted RNA Structures
110

##### Intrinsic transcription terminators
110

##### Rho-dependent transcription terminators
112

### TRANSLATION AND POSTTRANSCRIPTIONAL CONTROL
112

#### Ribosome-Binding Sites
112

#### RNA Structure at Ribosome Binding Sites
113

#### Internal Initiation Sites
113

#### Translational Coupling
114

#### Translational Repressor Proteins
114

#### Codon Usage
114

#### tRNAs
115

#### Introns
115

#### mRNA and tRNA Turnover
116

#### Proteolysis
116

### DNA METABOLISM, REPLICATION, RECOMBINATION, AND REPAIR
117

#### Enzymes of Nucleotide Metabolism
117

#### DNA Replication Proteins
117

#### Initiation of DNA Replication
118

#### Recombination and Recombination-Dependent DNA Replication
119

#### DNA Repair
119

### MOBILE ENDONUCLEASES, GENE TRANSFER, AND GENE EXCLUSION
120

### T4 PARTICLE, INFECTION, AND LYSIS
120

#### Heads
121

#### DNA Packaging
121

#### Baseplate and Tails
122

#### Infection and Superinfection Exclusion
124

#### Lysis and Lysis Inhibition
124

### RESTRICTION-MODIFICATION SYSTEMS AND PHAGE EXCLUSION
125

### PREDICTED INTEGRAL MEMBRANE PROTEINS
125

#### Integral Membrane Proteins of Known Function
126

#### Hypothetical Proteins with Predicted Cell Membrane Associations
127

#### Missing Membrane-Associated Proteins
127

### EVOLUTIONARY PERSPECTIVES: T4 PROTEINS AND THE GENOME
128

---

* Corresponding author. Mailing address: Department of Microbiology, North Carolina State University, Raleigh, NC 27696-7615. Phone: (919) 515-7922. Fax: (919) 515-7867. E-mail: eric_miller@ncsu.edu.
† Dedicated to the memory of Gisela Mosig, our friend, colleague, and mentor.
‡ Deceased.

T4 Protein Structures...................................................128  
Orthologous T4 Proteins.................................................128  
Paralogous Genes in the T4 Genome.....................................130  
A Glimpse at Genome Diversity and Evolution in T4-Type Phages..........130  
OUTLOOK..................................................................131  
ACKNOWLEDGMENTS.......................................................132  
REFERENCES..............................................................132  

---

**T4 GENES TO GENOME**

T-even phages (Fig. 1) have been major model systems in the development of modern genetics and molecular biology since the 1940s; many investigators have taken advantage of their useful degree of complexity and the ability to derive detailed genetic and physiological information with relatively simple experiments. Bacteriophages T2 and T4 were instrumental in the first formulations of many fundamental biological concepts. These include the unambiguous recognition of nucleic acids as the genetic material; the definition of the gene by fine-structure mutational, recombinational, and functional analyses; the demonstration that the genetic code is triplet; the discovery of mRNA; the importance of recombination in DNA replication; light-dependent and light-independent DNA repair mechanisms; restriction and modification of DNA; self-splicing introns in prokaryotes; translational bypassing; and others (506, 697). The advantages of T4 as a model system stemmed in part from the virus's total inhibition of host gene expression, which allows investigators to differentiate between host and phage macromolecular syntheses. Analysis of the assembly of the intricate T4 capsid and of the functioning of its nucleotide-synthesizing complex, its replisome, and its recombination complexes has led to important insights into macromolecular interactions, substrate channeling, and cooperation between phage and host proteins within such complexes. Indeed, the current view of biological "molecular machines" (15, 16) has its beginnings in T4 biology; the T4 replisome, late gene transcription complex and capsid assembly are paradigms of molecular machines.

The redundancies of protein functions and of pathways of DNA transactions probably allow T-even phages to exploit a broad range of potential hosts and environments while conferring substantial resistance against a wide range of antiviral mechanisms imposed by the host (4a, 599, 599a, 601, 786). T4 also produces several enzymes with widespread commercial applications, including its DNA and RNA ligase, polynucleotide kinase, and DNA polymerase. Many would argue that to know T4 is to know the foundations of molecular biology and the essential paradigms of genetics and gene expression.

There was a price to pay for all of the benefits provided by this highly tractable genetic system. Early efforts to clone T4 genes were largely thwarted by the glucosylated hydroxymethyl cytosine (HMC) DNA (which is central to the high expression and replication of the phage genome, the concurrent total inhibition of host transcription, and the eventual degradation of the host DNA). Most of the available restriction endonucleases failed to digest T4 DNA, delaying the gene-by-gene cloning analysis that rapidly advanced in other model organisms. Eventually, multiply mutant T4 strains defective in the nucleases that cleave unmodified DNA, in the enzymes leading to the synthesis of HMC-DNA, and in the protein blocking tran-

scription of cytosine-containing DNA were constructed (1020). These T4dC (or T4C) strains permitted the construction of detailed restriction maps of T4 (137a, 139, 600, 814, 833a, 1214) and rapidly accelerated cloning and sequence analysis of T4 gene clusters. By the early 1990s, much of the genome had been sequenced, but extensive regions remained intractable. The uncloned DNA appeared to largely encode proteins involved in the transition from host to phage metabolism, nucleases, and other proteins toxic to the *Escherichia coli* cloning host. These regions were sequenced by different members of the T4 community, who closed the gaps by using PCR to carry out direct sequencing without cloning. Regions that have not otherwise been published include the *nrdC-tk* region (laboratory of E. Kutter), the *e-tRNA* region (laboratories of V. Mesyanzhinov and E. Kutter), the 34–35 region (laboratory of E. Goldberg), the *t-asiA.5* region (laboratory of J. Drake) and the *ndd-rIIB* region (laboratories of K. Kreuzer and M. Uzan). The complete 168,903-bp sequence of the T4 genome is available as GenBank accession no. AF158101 and as entry NC_000866 at the NCBI Entrez Genome site (http://www.ncbi.nlm.nih.gov/Entrez). Among sequenced viruses in the database, only *Pseudomonas* phage φKZ (727), the African swine fever virus, herpesviruses, chlorella virus, and vaccinia virus have larger genomes.

The T4 genome is a rich arena for evaluating complete genomes in the context of a well-characterized biological system. Here, we demonstrate the use of some of the computational tools currently available for complete genome sequence

---

**FIG. 1.** Electron micrographs of bacteriophage T4. The well-recognized T4 morphology was nature’s prototype of the NASA lunar excursion module. (A) Extended tail fibers recognize the bacterial envelope, and its prolate icosahedral head contains the 168,903-bp dsDNA genome. Reprinted with permission of M. Wurtz, Biozentrum, Basel, Switzerland. (B) The DNA genome is delivered into the host through the internal tail tube, which is visible protruding from the end of the contracted tail sheath. Courtesy of W. Rüger.

analysis and discuss the new insights gained from this analysis of the T4 genome and its nearly 300 genes.

# NUCLEOTIDE SKEW IN THE T4 GENOME

T4 DNA has only 34.5% G+C, while its *E. coli* host has about 50% G+C. If T4 were assembled from “modules” of other genomes, as has been suggested for many phages (discussed below), different regions might be expected to have quite different G+C contents, particularly if they were recently acquired. However, only 18 of the known or predicted genes have less than 60% A+T and only 4 have less than 58%. Therefore, while some genes may have been more recently acquired, most of the T4 genome appears to have a lengthy, common history. Interestingly, it is the capsid proteins that have the lowest A+T contents, and these are the most widely conserved in the T4-related phages (701, 748, 919, 1069) and presumably among the earliest to have arisen. Gene 23, encoding the major head protein, is the lowest, at 55% A+T. It also uses the highest proportion of codons that are translationally optimal for the host (65%), in keeping with its very high level of expression; about 1,000 copies of the protein are needed per phage particle synthesized.

A substantial skew toward G and against C in the coding strand is observed in translated regions. Only four genes have more than 20% C in the coding strand, while about 130 have more than 20% G and 37 have more than 22% G. A and T are more equitably divided between the strands. However, the AT bias is strong in the third position of codons, as expected with high-A+T genomes, and reflection points in the bias (Fig. 2) do correlate with changes in the direction of T4 transcription (499). Whether these biases are coupled to effects of transcription or replication on directional mutation pressure, as suggested previously (499), remains to be demonstrated. Variably used multiple origins of T4 DNA replication (see below) presumably preclude the use of nucleotide skew analysis to identify the origin of replication, as it is often used for microbial chromosomes (352). Overall, AT skew is a strong predictor of T4 coding regions and the transcribed strand, although in a few regions both strands are transcribed and, in at least one region, both are translated.

A genome of AT compositional bias presents issues of DNA structure that are worthy of brief consideration. Starting with a balanced 50% A+T genome, each GC replaced by an AT base pair eliminates one Watson-Crick hydrogen bond. This suggests that the evolution of HMC and glucosylation conferred a secondary selective advantage: it not only protects the DNA against degrading endonucleases but also improves double-strand stability. The OH and H side groups of the added glucose are able to form hydrogen bonds when in proximity with neighboring bases (456, 457). With only one hydrogen bond formed per glucose residue, the approximately 16% glucosylated HMC in T4 DNA could compensate for the 14% A+T bias above average in the genome.

The AT-rich T4 genome may also present features advantageous for a virus: a DNA structure different from the B-DNA of its host (809). On a local scale, the structure would approach D-form DNA: a polymer consisting of poly(dA-dT) double strands, overwound with only 8 bp per turn, a wider and shallower major groove, and a deeper and narrower minor groove

(126, 127, 636). Close contacts of the glucosyl residues with side groups of neighboring bases could alter the preferred values of roll, slide, and twist angles of base pairs (258). Such forces and structural features can influence the outward appearance of the DNA in a way that may be recognized by proteins. Enzymes that melt DNA as part of their action (such as RNA polymerase and DNA polymerase) might transcribe and replicate AT-rich DNA faster than they would transcribe and replicate DNA with a balanced GC and AT content or might attract RNA polymerase and other host proteins in a competitive manner.

## IDENTIFYING T4 GENES

On the basis of all available criteria, we conclude that T4 has about 300 probable genes packed into its 168,903-bp genome. The nucleotide positions of all probable genes, promoters, terminators, and the best characterized origins of replication are given in Table 1, along with several calculated properties for the genes and their encoded proteins. T4 has a total of 289 probable protein-encoding genes, 8 tRNA genes, and at least 2 other genes that encode small, stable RNAs of unknown function. Table 2 summarizes and references the functions and properties of the approximately 156 genes that have been characterized by mutation and/or by the properties of cloned gene products. Imprecision in the number of “genes” reflects ambiguities of genetic nomenclature, when some genes contain multiple coding regions (for instance, genes 16, 17, and 49 encode more than one protein).

### Computational Strategies for Gene Assignment

The probability that an open reading frame (ORF) encodes a protein can be estimated by various computational methods that depend on observed patterns in the distribution of bases in known genes, along with such criteria as the presence of apparent translation initiation regions and the relationship to promoters and other genes. In the assembly and annotation of the T4 genome, the main tools used were the correlation coefficient, which compares the fractional use of each base at each of the three codon positions to those of a set of known T4 genes (971; T. Stidham, S. Peterson, and E. Kutler, Abstr. Evergreen Int. Phage Biol. Meet. p. 51, 1993), and the linguistics-based analysis, GenMark (99, 671). These methods were supplemented by identification of likely Shine-Dalgarno (SD) sequences for ribosome binding. As discussed below, such analyses indicate that virtually all the uncharacterized ORFs of T4 probably do encode proteins. Most known T4 genes have correlation coefficients above 0.85, as do most of the unsigned ORFs (Table 1). However, there appear to be constraints on the composition of some specific proteins that result in far lower values. This is seen for a few of the well-characterized but very small T4 genes, such as *stp* (−0.14), and for those that are predicted to encode integral membrane proteins, such as *imm* (0.31) and *ac* (0.51). Negative values are generally seen where a short but definitely expressed reading frame is superimposed on a different reading frame of another gene, such as 30.3′, or in the complementary strand, as in *repEA* and *repEB*. Therefore, while a high correlation coefficient makes it very likely that an ORF does indeed encode a

VOL. 67, 2003                                                                 BACTERIOPHAGE T4 GENOME 89

![Graph Image]

FIG. 2. Intrastrand biases (nucleotide skew) in the T4 genome. (A) Cumulative values of the number of T's minus the number of A's in a contiguous strand of the T4 genome for the first (●), second (□), and third (○) codon positions and for the intergenic regions (+), plotted against the genome position. The plus strand was used (5′ to 3′), from position 0 clockwise through the genome map, for the calculation. (B) Cumulative values of C's minus G's plotted as described for panel A. (C) Vertical lines show the distribution of genes in each strand, where "Direct" is the plus strand for which the analysis was performed and "Complementary" is the minus strand. Reprinted from reference 499, with permission from the publisher.

protein product, a low correlation coefficient cannot be used to exclude that possibility.

Work with T4 makes it clear that precisely identifying protein-coding regions can be complex, even in prokaryotes. (i) Five known T4 genes and several other ORFs have functional internal starts, with good experimental evidence for genes 17 and 49 that the shorter proteins have distinct functional roles (39, 286, 784, 788). In these two cases, separate but related gene names have been assigned (e.g., 17, 17′, and 17″) to indicate this complex relationship. We expect that other examples of internal translational start sites will be identified.

(ii) Five other genes and ORFs have two closely spaced start codons with similarly strong values for the sequence information content (defined below) at their translation initiation sites

| Gene$^a$ | Strand$^b$ | Start$^c$ | Stop$^c$ | Length (bp)$^d$ | Length (aa)$^d$ | Correlation$^e$ | pI$^f$ | $M_r$$^f$ |
|-----------|------------|-----------|----------|-----------------|---------------|------------------|--------|---------|
| Start     | –          | 1         | –        | –               | –             | –                | –      | –       |
| Pm        | –          | 123       | –        | –               | –             | –                | –      | –       |
| Pm        | –          | 377       | –        | –               | –             | –                | –      | –       |
| rIIA      | –          | 2189      | 12       | 2,178           | 725           | 0.99             | 6.131  | 82,903  |
| Pm        | –          | 2263      | –        | –               | –             | –                | –      | –       |
| rIIA.1    | –          | 2403      | 2200     | 204             | 67            | 0.89             | 7.156  | 8,129   |
| Pe(1.4–3.9) | –          | 2422      | –        | –               | –             | –                | –      | –       |
| 60+       | –          | 2802      | 2458     | 345             | 160           | 0.93             | 8.738  | 18,600  |
| 60+       | –          | 2990      | 2853     | 138             | –             | –                | –      | –       |
| 60.1      | –          | 3351      | 2971     | 381             | 126           | 0.96             | 9.364  | 14,651  |
| mobA      | –          | 3767      | 3654     | 114             | 37            | 0.78             | 9.54   | 4,192   |
| Pe(3.6)   | –          | 3847      | –        | –               | –             | –                | –      | –       |
| 39        | –          | 5328      | 3778     | 1,551           | 516           | 0.96             | 7.103  | 57,978  |
| Pm        | –          | 5349      | –        | –               | –             | –                | –      | –       |
| Term      | –          | 5384      | –        | –               | –             | –                | –      | –       |
| 39.1      | –          | 5595      | 5398     | 198             | 65            | 0.79             | 7.997  | 7,156   |
| 39.2      | –          | 5839      | 5702     | 138             | 45            | 0.86             | 8.497  | 5,107   |
| goF=comCa | –          | 6267      | 5842     | 426             | 141           | 0.91             | 4.462  | 16,682  |
| cef=mb    | –          | 6482      | 6267     | 216             | 71            | 0.95             | 5.203  | 8,464   |
| motB      | –          | 7141      | 6653     | 489             | 162           | 0.98             | 9.205  | 18,219  |
| Pe(5.9–7.3) | –          | 7179      | –        | –               | –             | –                | –      | –       |
| motB.1    | –          | 7650      | 7291     | 360             | 119           | 0.84             | 4.993  | 13,804  |
| motB.2    | –          | 8161      | 7661     | 501             | 166           | 0.93             | 6.45   | 19,736  |
| Pe(8.1)   | –          | 8182      | –        | –               | –             | –                | –      | –       |
| dexA      | –          | 8908      | 8225     | 684             | 227           | 0.98             | 4.763  | 25,966  |
| dexA.1    | –          | 9150      | 8908     | 243             | 80            | 0.90             | 5.31   | 9,392   |
| dexA.2    | –          | 9388      | 9143     | 246             | 81            | 0.93             | 4.184  | 9,518   |
| dda       | –          | 10729     | 9410     | 1,320           | 439           | 0.99             | 7.982  | 49,903  |
| dda.1     | –          | 11037     | 10726    | 312             | 103           | 0.84             | 9.776  | 12,104  |
| srd       | –          | 11785     | 11039    | 747             | 248           | 0.89             | 9.999  | 29,044  |
| Pe(11.5)  | –          | 11815     | –        | –               | –             | –                | –      | –       |
| modA      | –          | 12510     | 11908    | 603             | 200           | 0.94             | 6.035  | 23,350  |
| modB      | –          | 13130     | 12507    | 624             | 207           | 0.91             | 5.306  | 24,244  |
| Pe(12.8)  | –          | 13150     | –        | –               | –             | –                | –      | –       |
| modA.2    | –          | 13380     | 13198    | 183             | 60            | 0.83             | 4.11   | 7,024   |
| modA.3    | –          | 13859     | 13389    | 471             | 156           | 0.94             | 6.628  | 18,333  |
| modA.4    | –          | 14016     | 13852    | 165             | 54            | 0.91             | 5.852  | 6,162   |
| srh       | –          | 14216     | 14013    | 204             | 67            | 0.66             | 6.7    | 8,104   |
| mrh       | –          | 14676     | 14191    | 486             | 161           | 0.92             | 4.33   | 18,494  |
| mrh.1     | –          | 15026     | 14685    | 342             | 113           | 0.77             | 3.649  | 12,621  |
| mrh.2     | –          | 15232     | 15026    | 207             | 68            | 0.82             | 5.687  | 8,257   |
| Pe(15.0)  | –          | 15252     | –        | –               | –             | –                | –      | –       |
| Term      | –          | 15305     | –        | –               | –             | –                | –      | –       |
| soc       | –          | 15573     | 15331    | 243             | 80            | 0.95             | 6.361  | 9,117   |
| Pl        | –          | 15597     | –        | –               | –             | –                | –      | –       |
| Pl        | –          | 16162     | –        | –               | –             | –                | –      | –       |
| segF=69   | –          | 16280     | 15606    | 675             | 224           | 0.78             | 10.148 | 26,218  |
| Pl        | –          | 16359     | –        | –               | –             | –                | –      | –       |
| 56        | –          | 16785     | 16270    | 516             | 171           | 0.78             | 4.734  | 20,425  |
| oriA      | –          | 16763     | –        | –               | –             | –                | –      | –       |
| Pm        | –          | 16813     | –        | –               | –             | –                | –      | –       |
| dam       | –          | 17625     | 16846    | 780             | 259           | 0.93             | 8.846  | 30,420  |
| 61=58     | –          | 18963     | 17935    | 1,029           | 342           | 0.96             | 9.174  | 39,782  |
| Pm        | –          | 19122     | –        | –               | –             | –                | –      | –       |
| 61.1      | –          | 19130     | 18966    | 165             | 54            | 0.92             | 5.333  | 5,896   |
| 61.2      | –          | 19758     | 19132    | 627             | 208           | 0.91             | 6.323  | 24,334  |
| sp=rV     | –          | 20051     | 19758    | 294             | 97            | 0.93             | 4.769  | 10,994  |
| Pe(19.8)  | –          | 20073     | –        | –               | –             | –                | –      | –       |
| 61.4      | –          | 20369     | 20112    | 258             | 85            | 0.75             | 9.828  | 10,187  |
| dmd=61.5  | –          | 20553     | 20371    | 183             | 60            | 0.75             | 5.138  | 7,027   |
| Pe(20.3)  | –          | 20576     | –        | –               | –             | –                | –      | –       |
| 41        | –          | 22039     | 20612    | 1,428           | 475           | 0.98             | 5.439  | 53,602  |
| Term      | –          | 22347     | –        | –               | –             | –                | –      | –       |
| 40        | –          | 22393     | 22049    | 345             | 114           | 0.91             | 4.793  | 13,291  |
| uvsX      | –          | 23561     | 22386    | 1,176           | 391           | 0.98             | 5.310  | 43,999  |
| Pm        | –          | 23752     | –        | –               | –             | –                | –      | –       |
| segA      | –          | 24235     | 23570    | 666             | 221           | 0.92             | 9.933  | 25,342  |


Continued on following page

| Gene$^a$ | Strand$^b$ | Start$^c$ | Stop$^c$ | Length (bp)$^d$ | Length (aa)$^d$ | Correlation$^e$ | pI$^f$ | $M_r$$^f$ |
|-----------|------------|-----------|----------|-----------------|----------------|------------------|--------|---------|
| Pm        | –          | 24460     | –        | –               | –              | –                | –      | –       |
| β-gt      | –          | 25455     | 24400    | 1,056           | 351            | 0.97             | 9.079  | 40,670  |
| 42        | –          | 26219     | 25479    | 741             | 246            | 0.98             | 5.933  | 28,492  |
| Pm        | –          | 26317     | –        | –               | –              | –                | –      | –       |
| imm       | –          | 26624     | 26373    | 252             | 83             | 0.31             | 9.436  | 9,343   |
| imm.1     | –          | 27013     | 26636    | 378             | 125            | 0.96             | 6.486  | 14,074  |
| Pe(26.4)  | –          | 27044     | –        | –               | –              | –                | –      | –       |
| Term      | –          | 27183     | –        | –               | –              | –                | –      | –       |
| 43        | –          | 29893     | 27197    | 2,697           | 898            | 0.97             | 6.087  | 103,622 |
| Pm        | –          | 29931     | –        | –               | –              | –                | –      | –       |
| Term      | –          | 29967     | –        | –               | –              | –                | –      | –       |
| regA      | –          | 30340     | 29972    | 369             | 122            | 0.77             | 8.939  | 14,620  |
| 62        | –          | 30905     | 30342    | 564             | 187            | 0.93             | 8.592  | 21,364  |
| 44        | –          | 31866     | 30907    | 960             | 319            | 0.98             | 7.016  | 35,790  |
| Term      | –          | 31912     | –        | –               | –              | –                | –      | –       |
| 45        | –          | 32603     | 31917    | 687             | 228            | 0.98             | 4.759  | 24,861  |
| Pm        | –          | 32626     | –        | –               | –              | –                | –      | –       |
| rpbA      | –          | 33048     | 32659    | 390             | 129            | 0.98             | 7.283  | 14,712  |
| 45.2      | –          | 33246     | 33058    | 189             | 62             | 0.85             | 5.671  | 7,477   |
| Pm        | –          | 33257     | –        | –               | –              | –                | –      | –       |
| 46        | –          | 34984     | 33302    | 1,683           | 560            | 0.98             | 8.263  | 63,588  |
| Pm        | –          | 35014     | –        | –               | –              | –                | –      | –       |
| 46.1      | –          | 35187     | 34981    | 207             | 68             | 0.73             | 4.068  | 8,153   |
| 46.2      | –          | 35431     | 35168    | 264             | 87             | 0.85             | 4.288  | 10,268  |
| Pe(35.3)  | –          | 35662     | –        | –               | –              | –                | –      | –       |
| 47        | –          | 36447     | 35428    | 1,020           | 339            | 0.98             | 4.981  | 39,170  |
| Pm        | –          | 36576     | –        | –               | –              | –                | –      | –       |
| 47.1      | –          | 36584     | 36444    | 141             | 46             | 0.25             | 4.210  | 5,321   |
| Term      | –          | 36622     | –        | –               | –              | –                | –      | –       |
| α-gt      | –          | 37826     | 36624    | 1,203           | 400            | 0.94             | 6.358  | 46,709  |
| mobB      | –          | 38679     | 37885    | 795             | 264            | 0.78             | 9.737  | 30,367  |
| Pm        | –          | 38681     | –        | –               | –              | –                | –      | –       |
| Term      | –          | 38731     | –        | –               | –              | –                | –      | –       |
| α-gt.2    | –          | 38922     | 38731    | 192             | 63             | 0.91             | 9.135  | 7,322   |
| α-gt.3    | –          | 39110     | 38907    | 204             | 67             | 0.91             | 9.609  | 7,931   |
| α-gt.4    | –          | 39396     | 39079    | 318             | 105            | 0.87             | 8.849  | 12,445  |
| α-gt.5    | –          | 39616     | 39398    | 217             | 72             | 0.95             | 4.22   | 8,548   |
| 55        | –          | 40157     | 39600    | 558             | 185            | 0.94             | 5.45   | 21,537  |
| Pm        | –          | 40180     | –        | –               | –              | –                | –      | –       |
| 55.1      | –          | 40456     | 40193    | 264             | 87             | 0.79             | 4.114  | 9,846   |
| 55.2      | –          | 40785     | 40459    | 327             | 108            | 0.88             | 9.717  | 12,727  |
| Term      | –          | 40836     | –        | –               | –              | –                | –      | –       |
| 55.3      | –          | 41038     | 40799    | 240             | 79             | 0.52             | 8.751  | 9,153   |
| 55.4      | –          | 41170     | 41039    | 132             | 43             | 0.52             | 8.575  | 5,145   |
| Pe(40.4)  | –          | 41225     | –        | –               | –              | –                | –      | –       |
| 55.5      | –          | 41471     | 41178    | 294             | 97             | 0.75             | 9.821  | 11,809  |
| 55.6      | –          | 41646     | 41464    | 183             | 60             | 0.71             | 9.581  | 6,962   |
| Pe(41.0)  | –          | 41670     | –        | –               | –              | –                | –      | –       |
| Term      | –          | 41800     | –        | –               | –              | –                | –      | –       |
| nrdH      | –          | 42113     | 41805    | 309             | 102            | 0.93             | 9.075  | 11,720  |
| 55.8      | –          | 42328     | 42116    | 213             | 70             | 0.80             | 9.26   | 7,913   |
| Pm        | –          | 42805     | –        | –               | –              | –                | –      | –       |
| nrdG      | –          | 42916     | 42446    | 471             | 156            | 0.78             | 8.312  | 18,248  |
| Pm        | –          | 43023     | –        | –               | –              | –                | –      | –       |
| mobC      | –          | 43538     | 42906    | 633             | 210            | 0.88             | 9.852  | 23,978  |
| nrdD+     | –          | 44814     | 43535    | 1,280           | 605            | 0.99             | 6.889  | 67,964  |
| I-TevII   | –          | 45612     | 44836    | 777             | 258            | 0.98             | 9.808  | 30,371  |
| Pl        | –          | 45625     | –        | –               | –              | –                | –      | –       |
| nrdD+     | –          | 46385     | 45848    | 538             | –              | –                | –      | –       |
| Pm        | –          | 46441     | –        | –               | –              | –                | –      | –       |
| 49'       | –          | 46699     | 46382    | 318             | 105            | –                | –      | 11,888  |
| 49        | –          | 46855     | 46382    | 474             | 157            | 0.79             | 8.618  | 18,145  |
| Pl        | –          | 46879     | –        | –               | –              | –                | –      | –       |
| Term      | –          | 46884     | –        | –               | –              | –                | –      | –       |
| pin       | –          | 47382     | 46897    | 486             | 161            | 0.98             | 4.369  | 18,817  |
| Pe(46.7)  | –          | 47416     | –        | –               | –              | –                | –      | –       |
| 49.1      | –          | 47521     | 47366    | 156             | 51             | 0.80             | 3.780  | 6,163   |

| Gene$^a$ | Strand$^b$ | Start$^c$ | Stop$^c$ | Length (bp)$^d$ | Length (aa)$^d$ | Correlation$^e$ | p$^f$ | $M_r$$^f$ |
| --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 49.2 | – | 47826 | 47506 | 321 | 106 | 0.89 | 4.352 | 12,579 |
| 49.3 | – | 48131 | 47823 | 309 | 102 | 0.50 | 3.895 | 11,938 |
| nrdC | – | 48391 | 48128 | 264 | 87 | 0.80 | 7.183 | 10,050 |
| nrdC.1 | – | 48635 | 48393 | 243 | 80 | 0.97 | 8.342 | 9,444 |
| nrdC.2 | – | 48936 | 48622 | 315 | 104 | 0.81 | 6.586 | 12,159 |
| nrdC.3 | – | 49859 | 48933 | 927 | 308 | 0.92 | 9.01 | 36,297 |
| nrdC.4 | – | 50915 | 49914 | 1,002 | 333 | 0.95 | 7.63 | 38,996 |
| Pe(50.0) | – | 50937 | – | – | – | – | – | – |
| nrdC.5 | – | 51995 | 50973 | 1,023 | 340 | 0.92 | 9.327 | 39,670 |
| nrdC.6 | – | 52893 | 52066 | 828 | 275 | 0.96 | 9.20 | 31,725 |
| nrdC.7 | – | 53302 | 52901 | 402 | 133 | 0.84 | 6.41 | 15,309 |
| nrdC.8 | – | 53885 | 53358 | 528 | 175 | 0.88 | 6.11 | 20,758 |
| Pe(54.0) | – | 53907 | – | – | – | – | – | – |
| nrdC.9 | – | 54248 | 53946 | 303 | 100 | 0.75 | 9.68 | 11,979 |
| nrdC.10 | – | 55320 | 54343 | 978 | 325 | 0.93 | 4.85 | 36,691 |
| Pe(54.4) | – | 55348 | – | – | – | – | – | – |
| Term | – | 55432 | – | – | – | – | – | – |
| nrdC.11 | – | 56445 | 55435 | 1,011 | 336 | 0.92 | 6.89 | 38,899 |
| mobD | – | 57208 | 56429 | 780 | 259 | 0.79 | 9.957 | 30,456 |
| mobD.1 | – | 57828 | 57283 | 546 | 181 | 0.95 | 6.01 | 21,179 |
| mobD.2 | – | 57932 | 57828 | 105 | 34 | –0.05 | 9.49 | 4,205 |
| Pe | – | 57954 | – | – | – | – | – | – |
| mobD.2a | – | 58165 | 58049 | 117 | 38 | 0.68 | 8.85 | 4,516 |
| mobD.3 | – | 58349 | 58155 | 195 | 64 | 0.91 | 4.95 | 7,605 |
| mobD.4 | – | 58534 | 58352 | 183 | 60 | 0.94 | 4.41 | 6,858 |
| mobD.5 | – | 58722 | 58534 | 189 | 62 | 0.82 | 4.061 | 7,122 |
| Pe(57.9) | – | 58744 | – | – | – | – | – | – |
| Term | – | 58813 | – | – | – | – | – | – |
| rI.-1 | – | 59205 | 58819 | 387 | 128 | 0.97 | 5.61 | 14,649 |
| rI | – | 59495 | 59202 | 294 | 97 | 0.76 | 4.83 | 11,125 |
| rI.1 | – | 59720 | 59508 | 213 | 70 | 0.73 | 10.225 | 8,273 |
| Pl | – | 59740 | – | – | – | – | – | – |
| tk | – | 60344 | 59763 | 582 | 193 | 0.95 | 6.49 | 21,624 |
| tk.1 | – | 60534 | 60346 | 189 | 62 | 0.77 | 3.989 | 7,238 |
| tk.2 | – | 60716 | 60531 | 186 | 61 | 0.67 | 4.194 | 7,134 |
| tk.3 | – | 60925 | 60713 | 213 | 70 | 0.78 | 8.628 | 8,507 |
| Term | – | 61369 | – | – | – | – | – | – |
| tk.4 | – | 61389 | 60922 | 468 | 155 | 0.94 | 6.124 | 17,491 |
| vs | – | 61733 | 61386 | 348 | 115 | 0.67 | 8.744 | 13,057 |
| vs.1 | – | 62271 | 61726 | 546 | 181 | 0.95 | 9.798 | 20,683 |
| regB | – | 62740 | 62279 | 462 | 153 | 0.85 | 8.444 | 17,978 |
| Pe(62.2) | – | 62761 | – | – | – | – | – | – |
| vs.3 | – | 63078 | 62800 | 279 | 92 | 0.87 | 5.378 | 10,904 |
| vs.4 | – | 63344 | 63078 | 267 | 88 | 0.95 | 4.555 | 10,211 |
| vs.5 | – | 63557 | 63381 | 177 | 58 | 0.17 | 9.52 | 6,611 |
| vs.6 | – | 63919 | 63557 | 363 | 120 | 0.93 | 5.902 | 13,814 |
| vs.7 | – | 64256 | 63927 | 330 | 109 | 0.75 | 9.031 | 12,836 |
| vs.8 | – | 64912 | 64253 | 660 | 219 | 0.87 | 9.21 | 25,029 |
| denV | – | 65355 | 64939 | 417 | 138 | 0.94 | 9.393 | 16,080 |
| Pe(64.6) | – | 65378 | – | – | – | – | – | – |
| ipII | – | 65718 | 65416 | 303 | 100 | 0.94 | 9.349 | 11,086 |
| Pe(65.0) | – | 65763 | – | – | – | – | – | – |
| ipIII | – | 66415 | 65834 | 582 | 194 | 0.89 | 9.557 | 21,689 |
| Pe | – | 66462 | – | – | – | – | – | – |
| e | – | 66997 | 66503 | 495 | 164 | 0.97 | 9.599 | 18,693 |
| Pl | – | 67005 | – | – | – | – | – | – |
| Pl | – | 67018 | – | – | – | – | – | – |
| Pl | – | 67234 | – | – | – | – | – | – |
| nudE=e.1 | – | 67490 | 67035 | 456 | 151 | 0.95 | 5.141 | 17,025 |
| e.2 | – | 67960 | 67652 | 309 | 102 | 0.96 | 6.485 | 12,156 |
| e.3 | – | 68319 | 67957 | 363 | 120 | 0.85 | 8.784 | 14,156 |
| e.4 | – | 68693 | 68301 | 393 | 130 | 0.60 | 9.647 | 15,139 |
| e.5 | – | 69270 | 68662 | 609 | 202 | 0.97 | 5.72 | 23,816 |
| Term | – | 69306 | – | – | – | – | – | – |
| e.6 | – | 69905 | 69312 | 594 | 197 | 0.99 | 6.162 | 22,045 |
| Pe(69.9) | – | 69931 | – | – | – | – | – | – |
| e.7 | – | 70303 | 69968 | 336 | 111 | 0.83 | 4.214 | 13,070 |

| Gene$^a$ | Strand$^b$ | Start$^c$ | Stop$^c$ | Length (bp)$^d$ | Length (aa)$^d$ | Correlation$^e$ | pI$^f$ | $M_r$$^f$ |
|-----------|------------|-----------|----------|-----------------|-----------------|------------------|--------|-------------|
| Pe(69.4)  | –          | 70323     |          |                 |                 |                  |        |             |
| e.8       | –          |           | 70623    | 264             | 87              | 0.74             | 4.324  | 10,199      |
| Pe(69.8)  | –          |           | 70660    |                 |                 |                  |        |             |
| Term      | –          |           | 70856    |                 |                 |                  |        |             |
| rnaC      | –          | 71046     | 70908    | 139             |                 |                  |        |             |
| rnaD      | –          | 71171     | 71053    | 119             |                 |                  |        |             |
| tRNAR     | –          | 71247     | 71173    | 75              |                 |                  |        |             |
| segB      | –          | 71918     | 71253    | 666             | 221             | 0.96             | 9.320  | 26,146      |
| tRNAI     | –          | 72033     | 71960    | 74              |                 |                  |        |             |
| tRNAT     | –          | 72110     | 72035    | 76              |                 |                  |        |             |
| tRNAS     | –          | 72204     | 72118    | 87              |                 |                  |        |             |
| tRNAP     | –          | 72280     | 72203    | 78              |                 |                  |        |             |
| tRNAG     | –          | 72364     | 72291    | 74              |                 |                  |        |             |
| tRNAL     | –          | 72456     | 72369    | 88              |                 |                  |        |             |
| tRNAE     | –          | 72530     | 72456    | 75              |                 |                  |        |             |
| Pm        | –          |           | 72593    |                 |                 |                  |        |             |
| Pl        | –          |           | 72863    |                 |                 |                  |        |             |
| tRNA.2    | –          | 72915     | 72628    | 288             | 95              | 0.96             | 5.031  | 11,285      |
| tRNA.3    | –          | 73328     | 72918    | 411             | 136             | 0.93             | 4.736  | 16,035      |
| tRNA.4    | –          | 73514     | 73329    | 186             | 61              | 0.69             | 7.996  | 6,558       |
| Pe(72.6)  | –          |           | 73536    |                 |                 |                  |        |             |
| ipI       | –          | 73878     | 73591    | 288             | 95              | 0.90             | 8.973  | 10,177      |
| Pe(73.0)  | –          |           | 73903    |                 |                 |                  |        |             |
| 57B       | –          | 74410     | 73952    | 459             | 152             | 0.83             | 5.089  | 17,246      |
| 57A       | –          | 74649     | 74407    | 243             | 80              | 0.97             | 4.248  | 8,731       |
| Pm        | –          |           | 74877    |                 |                 |                  |        |             |
| Pl        | –          |           | 74999    |                 |                 |                  |        |             |
| I         | –          | 75374     | 74649    | 726             | 242             | 1.00             | 4.947  | 27,332      |
| Pm        | –          |           | 75393    |                 |                 |                  |        |             |
| 3         | –          | 75954     | 75424    | 531             | 176             | 0.93             | 4.267  | 19,713      |
| 2=64      | –          | 76885     | 76061    | 825             | 274             | 0.87             | 10.144 | 31,613      |
| 4=50=65   | –          | 77337     | 76885    | 453             | 150             | 0.77             | 9.793  | 17,629      |
| Pl        | –          |           | 77358    |                 |                 |                  |        |             |
| Pl        | +          |           | 77362    |                 |                 |                  |        |             |
| Pl        | +          |           | 77381    |                 |                 |                  |        |             |
| 53        | +          | 77385     | 77975    | 591             | 196             | 0.96             | 6.005  | 22,968      |
| Pl        | +          |           | 77491    |                 |                 |                  |        |             |
| 5         | +          | 77959     | 79686    | 1,728           | 575             | 0.95             | 5.235  | 63,121      |
| repEB     | –          | 78118     | 77981    | 138             | 45              | 0.35             | 5.19   | 5,483       |
| repEA     | –          | 79237     | 79085    | 153             | 50              | 0.24             | 8.52   | 6,130       |
| Pe        | –          |           | 79405    |                 |                 |                  |        |             |
| 5.1       | +          | 79721     | 80215    | 495             | 164             | 0.86             | 4.593  | 18,499      |
| Pl        | +          |           | 79799    |                 |                 |                  |        |             |
| segC      | +          | 80196     | 80618    | 423             | 140             | 0.90             | 9.74   | 15,945      |
| 5.3       | +          | 80621     | 80791    | 171             | 56              | 0.75             | 9.960  | 6,089       |
| 5.4       | +          | 80779     | 81072    | 294             | 97              | 0.89             | 8.462  | 10,221      |
| 6         | +          | 81081     | 83063    | 1,983           | 660             | 0.97             | 4.508  | 74,436      |
| 7         | +          | 83060     | 86158    | 3,099           | 1,032           | 0.96             | 4.953  | 119,226     |
| Pl        | +          |           | 85812    |                 |                 |                  |        |             |
| 8         | +          | 86151     | 87155    | 1,005           | 334             | 0.91             | 4.453  | 38,011      |
| Term      | +          |           | 87161    |                 |                 |                  |        |             |
| Pl        | +          |           | 87200    |                 |                 |                  |        |             |
| 9         | +          | 87219     | 88085    | 867             | 288             | 0.91             | 4.929  | 31,000      |
| Pl        | +          |           | 87885    |                 |                 |                  |        |             |
| 10        | +          | 88085     | 89893    | 1,809           | 602             | 0.94             | 4.275  | 66,238      |
| 11        | +          | 89893     | 90552    | 660             | 219             | 0.91             | 5.066  | 23,708      |
| 12        | +          | 90549     | 92132    | 1,584           | 527             | 0.91             | 6.072  | 56,220      |
| wac       | +          | 92129     | 93592    | 1,464           | 487             | 0.95             | 4.445  | 51,876      |
| 13        | +          | 93624     | 94553    | 930             | 309             | 0.92             | 4.917  | 34,745      |
| 14        | +          | 94555     | 95325    | 771             | 256             | 0.92             | 4.468  | 29,575      |
| Pl        | +          |           | 95337    |                 |                 |                  |        |             |
| 15        | +          | 95367     | 96185    | 819             | 272             | 0.94             | 4.772  | 31,558      |
| Pl        | +          |           | 96153    |                 |                 |                  |        |             |
| 16        | +          | 96194     | 96688    | 495             | 164             | 0.90             | 4.423  | 18,388      |
| Term      | +          |           | 93596    |                 |                 |                  |        |             |
| 17        | +          | 96672     | 98504    | 1,833           | 610             | 0.98             | 5.638  | 69,764      |
| Pl        | +          |           | 96913    |                 |                 |                  |        |             |

| Gene$^a$ | Strand$^b$ | Start$^c$ | Stop$^c$ | Length (bp)$^d$ | Length (aa)$^d$ | Correlation$^e$ | $p^f$ | $M_f$ |
|-----------|------------|-----------|----------|-----------------|-----------------|------------------|-------|--------|
| 17'A      | +          | 96933     | 98504    | 1,572           | 523             |                  | 5.16  | 59,245 |
| 17'B      | +          | 96987     | 98504    | 1,518           | 505             |                  | 5.02  | 57,108 |
| 17'       | +          | 97254     | 98504    | 1,251           | 416             |                  | 4.67  | 46,841 |
| Pl        | +          | 98513     |          |                 |                 |                  |       |        |
| 18        | +          | 98536     | 100515   | 1,980           | 659             | 0.95             | 4.795 | 71,338 |
| Pl        | +          | 100564    |          |                 |                 |                  |       |        |
| Pl        | +          | 100623    |          |                 |                 |                  |       |        |
| 19        | +          | 100632    | 101123   | 492             | 163             | 0.99             | 4.546 | 18,462 |
| Term      | +          | 101131    |          |                 |                 |                  |       |        |
| Pl        | +          | 101184    |          |                 |                 |                  |       |        |
| 20        | +          | 101207    | 102781   | 1,575           | 524             | 0.99             | 5.341 | 61,037 |
| Pl        | +          | 102539    |          |                 |                 |                  |       |        |
| 67        | +          | 102781    | 103023   | 243             | 80              | 0.77             | 3.662 | 9,106  |
| 68        | +          | 103023    | 103448   | 426             | 141             | 0.98             | 10.108 | 15,874 |
| Pl        | +          | 103146    |          |                 |                 |                  |       |        |
| 21        | +          | 103448    | 104086   | 639             | 212             | 0.95             | 4.725 | 23,253 |
| 21'       | +          | 103514    | 104086   | 573             | 190             |                  | 5.134 | 20,834 |
| Pl        | +          | 104095    |          |                 |                 |                  |       |        |
| 22        | +          | 104117    | 104926   | 810             | 269             | 0.92             | 4.498 | 29,906 |
| Pl        | +          | 104787    |          |                 |                 |                  |       |        |
| Pl        | +          | 104816    |          |                 |                 |                  |       |        |
| 23        | +          | 104945    | 106510   | 1,566           | 521             | 0.97             | 5.29  | 56,023 |
| Term      | +          | 106537    |          |                 |                 |                  |       |        |
| segD      | –          | 107232    | 106561   | 672             | 223             | 0.99             | 9.740 | 25,619 |
| Pl        | +          | 107301    |          |                 |                 |                  |       |        |
| 24        | +          | 107323    | 108606   | 1,284           | 427             | 0.92             | 4.618 | 46,998 |
| Term      | +          | 108613    |          |                 |                 |                  |       |        |
| Term      | +          | 108668    |          |                 |                 |                  |       |        |
| $rn l B = 24.1$ | – | 109640    | 108636   | 1,005           | 334             | 0.99             | 5.666 | 37,631 |
| 24.2      | –          | 109928    | 109650   | 279             | 92              | 0.87             | 4.923 | 11,003 |
| 24.3      | –          | 110085    | 109915   | 171             | 56              | 0.85             | 10.22 | 6,550  |
| Term      | –          | 110180    |          |                 |                 |                  |       |        |
| hoc       | –          | 111317    | 110187   | 1,317           | 376             | 0.93             | 4.626 | 40,388 |
| inh       | –          | 112007    | 111327   | 681             | 226             | 0.97             | 4.304 | 25,570 |
| Pl        | –          | 112029    |          |                 |                 |                  |       |        |
| Pl        | +          | 112034    |          |                 |                 |                  |       |        |
| segE      | +          | 112057    | 112674   | 618             | 205             | 0.92             | 4.559 | 22,896 |
| Pl        | +          | 112588    |          |                 |                 |                  |       |        |
| uvsW      | +          | 112677    | 114440   | 1,764           | 587             | 0.93             | 10.304 | 67,526 |
| Term      | –          | 114472    |          |                 |                 |                  |       |        |
| uvsY.-2   | –          | 114663    | 114496   | 168             | 55              | 0.91             | 4.323 | 6,062  |
| Pl        | –          | 114681    |          |                 |                 |                  |       |        |
| uvsY.-1   | –          | 114914    | 114690   | 225             | 74              | 0.74             | 4.939 | 8,963  |
| uvsY      | –          | 115327    | 114914   | 414             | 137             | 0.88             | 8.53  | 15,840 |
| Pm        | –          | 115371    |          |                 |                 |                  |       |        |
| 25        | –          | 115802    | 115404   | 399             | 132             | 0.95             | 4.49  | 15,096 |
| 26'       | –          | 116089    | 115802   | 288             | 95              |                  | 5.27  | 10,856 |
| Pl        | –          | 116412    |          |                 |                 |                  |       |        |
| 26        | –          | 116428    | 115802   | 627             | 208             | 0.86             | 5.748 | 23,883 |
| Pl        | –          | 116436    |          |                 |                 |                  |       |        |
| Pl        | –          | 116444    |          |                 |                 |                  |       |        |
| Pl        | +          | 116467    |          |                 |                 |                  |       |        |
| 51        | +          | 116479    | 117228   | 750             | 249             | 0.91             | 6.229 | 29,340 |
| 27        | +          | 117228    | 118403   | 1,176           | 391             | 0.97             | 5.24  | 44,462 |
| 28        | +          | 118348    | 118881   | 534             | 177             | 0.94             | 5.75  | 20,122 |
| 29        | +          | 118878    | 120650   | 1,773           | 590             | 0.95             | 4.931 | 64,416 |
| 48        | +          | 120659    | 121753   | 1,095           | 364             | 0.76             | 8.715 | 39,738 |
| 54        | +          | 121753    | 122715   | 963             | 320             | 0.87             | 5.383 | 34,981 |
| Term      | +          | 122720    |          |                 |                 |                  |       |        |
| alt.-3    | –          | 123032    | 122742   | 291             | 96              | 0.93             | 4.58  | 10,704 |
| Pe        | –          | 123057    |          |                 |                 |                  |       |        |
| alt.-2    | –          | 123268    | 123065   | 204             | 67              | 0.70             | 9.81  | 7,382  |
| alt.-1    | –          | 123450    | 123265   | 186             | 61              | 0.95             | 5.71  | 6,622  |
| alt       | –          | 125502    | 123454   | 2,049           | 682             | 0.95             | 6.158 | 75,819 |
| Pl        | –          | 125525    |          |                 |                 |                  |       |        |
| Term      | –          | 125558    |          |                 |                 |                  |       |        |
| alt.1     | –          | 125748    | 125560   | 189             | 62              | 0.89             | 4.371 | 7,153  |

| Gene$^a$ | Strand$^b$ | Start$^c$ | Stop$^c$ | Length (bp)$^d$ | Length (aa)$^d$ | Correlation$^e$ | pI$^f$ | $M_r$$^f$ |
|-----------|------------|-----------|----------|-----------------|-----------------|------------------|---------|-------------|
| 30        | –          | 127208    | 125745   | 1,464           | 487             | 0.98             | 6.315   | 55,299      |
| Pm        |            | 127234    |          |                 |                 |                  |         |             |
| 30.1      | –          | 127474    | 127205   | 270             | 89              | 0.82             | 8       | 10,833      |
| 30.2      | –          | 128310    | 127474   | 837             | 278             | 0.94             | 6.241   | 32,433      |
| Pm        |            | 128355    |          |                 |                 |                  |         |             |
| 30.3'     | –          | 128629    | 128402   | 228             | 75              | –0.17            | 10.4    | 8,945       |
| 30.3      | –          | 128765    | 128307   | 459             | 152             | 0.85             | 8.849   | 17,088      |
| 30.4      | –          | 128964    | 128758   | 207             | 68              | 0.95             | 4.535   | 8,064       |
| 30.5      | –          | 129158    | 128961   | 198             | 65              | 0.82             | 5.091   | 7,252       |
| 30.6      | –          | 129445    | 129158   | 288             | 95              | 0.88             | 7.19    | 10,814      |
| 30.7      | –          | 129852    | 129487   | 366             | 121             | 0.82             | 7.305   | 14,131      |
| Pe(128.2) | –          | 129883    |          |                 |                 |                  |         |             |
| 30.8      | –          | 130253    | 129921   | 333             | 110             | 0.94             | 6.612   | 12,893      |
| Pe(128.6) | –          | 130274    |          |                 |                 |                  |         |             |
| Term      | –          | 130358    |          |                 |                 |                  |         |             |
| Term      | –          | 130402    |          |                 |                 |                  |         |             |
| 30.9      | –          | 130540    | 130364   | 177             | 58              | 0.92             | 11.44   | 6,519       |
| rIII      | –          | 131033    | 130785   | 249             | 82              | 0.90             | 8.479   | 9,325       |
| Pl        | –          | 131167    |          |                 |                 |                  |         |             |
| 31        | –          | 131516    | 131181   | 336             | 111             | 0.89             | 5.315   | 12,079      |
| Pm        |            | 131540    |          |                 |                 |                  |         |             |
| 31.1      | –          | 131881    | 131573   | 309             | 102             | 0.84             | 9.113   | 11,520      |
| 31.2      | –          | 132118    | 131882   | 237             | 78              | 0.71             | 9.806   | 9,397       |
| cd        | –          | 132699    | 132118   | 582             | 193             | 0.90             | 7.833   | 21,200      |
| cd.1      | –          | 133034    | 132696   | 339             | 112             | 0.84             | 8.138   | 12,814      |
| cd.2      | –          | 133261    | 133031   | 231             | 76              | 0.69             | 4.823   | 10,131      |
| Pe(131.7) | –          | 133295    |          |                 |                 |                  |         |             |
| Term      | –          | 133376    |          |                 |                 |                  |         |             |
| cd.3      | –          | 133609    | 133334   | 276             | 91              | 0.88             | 4.82    | 10,131      |
| cd.4      | –          | 133812    | 133612   | 201             | 66              | 0.83             | 4.19    | 7,918       |
| cd.5      | –          | 134032    | 133805   | 228             | 75              | 0.81             | 8.521   | 8,738       |
| pseT      | –          | 134907    | 134002   | 906             | 301             | 0.96             | 8.671   | 34,622      |
| pseT.1    | –          | 135135    | 134908   | 228             | 75              | 0.74             | 8.455   | 8,833       |
| pseT.2    | –          | 135431    | 135132   | 300             | 99              | 0.75             | 8.577   | 11,645      |
| pseT.3    | –          | 135781    | 135428   | 354             | 117             | 0.90             | 8.947   | 13,136      |
| alc       | –          | 136275    | 135772   | 504             | 167             | 0.94             | 7.23    | 18,962      |
| Pe(134.4) | –          | 136300    |          |                 |                 |                  |         |             |
| Pl        | +          | 136889    |          |                 |                 |                  |         |             |
| rnlA = 63 | –          | 137464    | 136340   | 1,125           | 374             | 0.96             | 4.885   | 43,514      |
| denA      | –          | 137951    | 137517   | 435             | 144             | 0.94             | 9.442   | 16,744      |
| Term      | –          | 137950    |          |                 |                 |                  |         |             |
| nrdB+     | –          | 138457    | 137955   | 503             | 388             | 0.92             | 4.924   | 45,357      |
| I-TevIII  | –          | 138886    | 38593    | 294             | 97              | 0.78             | 9.011   | 11,331      |
| Pl        | –          | 138933    |          |                 |                 |                  |         |             |
| Pm        | –          | 138939    |          |                 |                 |                  |         |             |
| nrdB+     | –          | 139719    | 139056   | 664             |                 |                  |         |             |
| Pm        | –          | 139878    |          |                 |                 |                  |         |             |
| nrdB.1    | –          | 139967    | 139716   | 252             | 83              | 0.83             | 9.874   | 9,409       |
| Term      | –          | 140384    |          |                 |                 |                  |         |             |
| mobE      | –          | 140416    | 139991   | 426             | 141             | 0.94             | 10.102  | 16,448      |
| nrdA      | –          | 142680    | 140416   | 2,265           | 754             | 0.98             | 6.117   | 85,982      |
| Pm        | –          | 142725    |          |                 |                 |                  |         |             |
| nrdA.1    | –          | 142997    | 142671   | 327             | 108             | 0.70             | 9.035   | 12,362      |
| nrdA.2    | –          | 143214    | 142951   | 264             | 87              | 0.96             | 5.233   | 10,065      |
| td+       | –          | 143546    | 143235   | 312             | 286             | 0.89             | 8.617   | 33,077      |
| I-TevI    | –          | 144431    | 143694   | 738             | 245             | 0.83             | 9.625   | 28,175      |
| Pl        | –          | 144449    |          |                 |                 |                  |         |             |
| td+       | –          | 145112    | 144564   | 549             |                 |                  |         |             |
| Pm        | –          | 145142    |          |                 |                 |                  |         |             |
| frd       | –          | 145690    | 145109   | 582             | 193             | 0.98             | 6.35    | 21,714      |
| frd.1     | –          | 146004    | 145762   | 243             | 80              | 0.75             | 4.843   | 9,471       |
| Term      | –          | 146051    |          |                 |                 |                  |         |             |
| frd.2     | –          | 146529    | 146143   | 387             | 128             | 0.73             | 4.281   | 14,742      |
| frd.3     | –          | 146802    | 146575   | 228             | 75              | 0.85             | 3.699   | 8,820       |
| Pe(144.6) | –          | 146839    | 146833   |                 |                 |                  |         |             |
| Term      | –          | 146925    |          |                 |                 |                  |         |             |
| 32        | –          | 147853    | 146948   | 906             | 301             | 0.96             | 4.681   | 33,509      |
| Pl        | –          | 147998    |          |                 |                 |                  |         |             |
| Pm        | –          | 148057    |          |                 |                 |                  |         |             |
| segG = 32.1 | –          | 148541    | 147909   | 633             | 210             | 0.95             | 7.194   | 24,564      |

| Gene$^a$ | Strand$^b$ | Start$^c$ | Stop$^c$ | Length (bp)$^d$ | Length (aa)$^d$ | Correlation$^e$ | pI$^f$ | $M_r$$^f$ |
|-----------|------------|-----------|----------|-----------------|----------------|------------------|--------|---------|
| 59        | –          | 149196    | 148543   | 654             | 217            | 0.89             | 9.387  | 26,000  |
| 33        | –          | 149531    | 149193   | 339             | 112            | 0.93             | 4.38   | 12,831  |
| dsbA      | –          | 149778    | 149509   | 270             | 89             | 0.96             | 4.969  | 10,379  |
| Pm        | –          | 149873    |          |                 |                |                  |        |         |
| rnh       | –          | 150704    | 149787   | 918             | 305            | 0.97             | 8.535  | 35,562  |
| Pe(148.6) | –          | 150727    |          |                 |                |                  |        |         |
| Pl        | +          | 150780    |          |                 |                |                  |        |         |
| 34        | +          | 150809    | 154678   | 3870            | 1289           | 0.94             | 5.206  | 140,416 |
| Pm        | –          | 153011    |          |                 |                |                  |        |         |
| 35        | +          | 154687    | 155805   | 1,119           | 372            | 0.84             | 4.96   | 40,123  |
| Term      | +          | 155811    |          |                 |                |                  |        |         |
| Pl        | +          | 155850    |          |                 |                |                  |        |         |
| 36        | +          | 155868    | 156533   | 666             | 221            | 0.92             | 8.072  | 23,343  |
| Pl        | +          | 156369    |          |                 |                |                  |        |         |
| 37        | +          | 156542    | 159622   | 3,081           | 1,026          | 0.87             | 8.537  | 109,226 |
| Term      | +          | 159628    |          |                 |                |                  |        |         |
| 38        | +          | 159649    | 160200   | 552             | 183            | 0.88             | 6.633  | 22,311  |
| Pl        | +          | 160209    |          |                 |                |                  |        |         |
| t         | +          | 160221    | 160877   | 657             | 218            | 0.94             | 7.921  | 25,178  |
| Term      | +          | 160924    |          |                 |                |                  |        |         |
| asiA      | –          | 161150    | 160878   | 273             | 90             | 0.95             | 5.395  | 10,590  |
| Pe(158.7) | –          | 161175    |          |                 |                |                  |        |         |
| asiA.1    | –          | 161315    | 161163   | 153             | 50             | 0.89             | 4.51   | 5,935   |
| arn       | –          | 161590    | 161312   | 279             | 92             | 0.81             | 4.351  | 10,903  |
| arn.1     | –          | 161805    | 161674   | 132             | 43             | 0.54             | 8.334  | 5,173   |
| arn.2     | –          | 162172    | 161876   | 297             | 98             | 0.59             | 5.243  | 12,402  |
| arn.3     | –          | 162630    | 162172   | 459             | 152            | 0.90             | 5.078  | 17,837  |
| arn.4     | –          | 162833    | 162627   | 207             | 68             | 0.99             | 9.24   | 12,802  |
| motA      | –          | 163602    | 162967   | 636             | 211            | 0.96             | 8.772  | 23,577  |
| Pe(161.1) | –          | 163637    |          |                 |                |                  |        |         |
| Term      | –          | 163724    |          |                 |                |                  |        |         |
| motA.1    | –          | 163879    | 163730   | 150             | 49             | 0.49             | 10.036 | 4,842   |
| 52        | –          | 165204    | 163876   | 1,329           | 442            | 0.99             | 8.799  | 50,582  |
| 52.1      | –          | 165349    | 165209   | 141             | 46             | 0.42             | 8.649  | 5,098   |
| Term      | –          | 165332    |          |                 |                |                  |        |         |
| ac        | –          | 165497    | 165342   | 156             | 51             | 0.51             | 3.916  | 5,472   |
| stp       | –          | 165585    | 165505   | 81              | 26             | –0.14            | 10.28  | 3,184   |
| ndd       | –          | 166040    | 165585   | 456             | 151            | 0.85             | 9.524  | 16,935  |
| ndd.1     | –          | 166316    | 166101   | 216             | 71             | 0.86             | 4.111  | 8,143   |
| ndd.2     | –          | 166435    | 166325   | 111             | 36             | 0.09             | 5.818  | 4,354   |
| ndd.2a    | –          | 166554    | 166432   | 123             | 40             | 0.73             | 7.14   | 4,303   |
| ndd.3     | –          | 166628    | 166548   | 81              | 26             | –0.49            | 8.25   | 3,019   |
| ndd.4     | –          | 166764    | 166636   | 129             | 42             | 0.11             | 8.622  | 4,954   |
| Pe(164.2) | –          | 166771    |          |                 |                |                  |        |         |
| ndd.5     | –          | 166913    | 166815   | 99              | 32             | 0.81             | 7.082  | 3,687   |
| ndd.6     | –          | 166996    | 166910   | 87              | 28             | –0.05            | 8.014  | 3,406   |
| Pe(164.5) | –          | 167050    |          |                 |                |                  |        |         |
| denB      | –          | 167660    | 167103   | 558             | 185            | 0.91             | 7.232  | 21,162  |
| Term      | –          | 167736    |          |                 |                |                  |        |         |
| denB.1    | –          | 167937    | 167743   | 195             | 64             | 0.52             | 8.134  | 7,452   |
| rIIB      | –          | 168903    | 167965   | 939             | 312            | 0.96             | 6.24   | 35,544  |
| End       |            | 168903    |          |                 |                |                  |        |         |

$a$ Genes are listed sequentially as they appear in the GenBank file (accession no. AF158101), clockwise on the circular map (by convention) starting with the first base 5' of $rIIB$. Recently renamed genes, or those with multiple names, are labeled with =. Intron-containing or translational bypass genes ($nrdB^+$, $60^+$) are noted with a $^+$ for each reading frame. Genes marked with a prime ('') are overlapping with, or internal to, the designated gene. Transcription signals listed are Pe, Pm, and Pl for early, middle and late promoters, respectively, and Ter for terminator. Pe entries in parentheses are promoter designations used in earlier literature.

$b$ The coding strand is noted as either the GenBank deposited (+) sequence or the complement (–).

$c$ Start and stop coordinates denote the first base of the coding region (usually the A of the initiator ATG) and the last base of the stop codon. Promoter coordinates given are either the mapped or predicted transcript start sites (the “+1” position), and terminator coordinates are the first 5' base of the hairpin.

$d$ The length (bp) entry includes the stop codon of each coding sequence. Only the mature protein length (aa) is given for those proteins that arise from spliced or bypassed genes.

$e$ The correlation coefficient given for each gene is the probability of an ORF being a T4 gene based on the codon usage in characterized T4 genes. The program was written by Gary Stormo and is available at the web site: http://www.lecb.ncifcrf.gov/~toms/delila/frame.html.

$f$ pI and $M_r$ are calculated values.

| Gene$^a$ | Function of gene product$^b$ | Size (kDa)$^b$ | Mutant phenotype | Restrictive host or condition$^c$ | Reference(s) |
| --- | --- | --- | --- | --- | --- |
| $rIIA$ | Membrane-associated protein; affect host membrane ATPase | 82.9 | Rapid lysis; suppress T4 30 and some 32 mutations | Auxiliary; $rex^{+}\lambda$ lysogens; P2-like HK239 lysogen; $tabR$ | 2-4, 56, 59, 95, 96, 106, 121, 159, 181, 184, 191, 198, 216, 224, 263, 292, 293, 365, 375, 441, 430, 431, 451, 504, 582, 768, 769, 774, 793, 810, 811, 834, 851, 874, 940, 1007, 1021, 1059, 1114b, 1159, |
| $mobA$ | DNA topoisomerase subunit | 18.6 | DNA delay; $rc=$ acriflavine resistance | Essential; 25°C or below | 447, 450, 451, 452, 653, 654, 681, 801, 968, 1037 |
| $mobA$ | Pseudogene of Mob site-specific DNA endonuclease | 4.2 | Nonessential | E. Thomas, F. Zucker, and E. Kutter, unpublished data |
| 39 | DNA topoisomerase subunit; DNA-dependent ATPase; membrane-associated protein | 58.0 | DNA delay; $rc=$ acriflavine resistance | Essential; 25°C or below; synthetic lethal with T4 49 and 17 mutations, or when host topoisomerase IV is poisoned with novobiocin | 452, 454, 571, 589, 653, 654, 708, 789, 768, 769, 801, 834, 853, 1006, 1037, 1047, 1059, 1216, 1236 |
| $gOF = comC-a = go9H$ | Affects mRNA metabolism | 16.7 | Allows T4 growth in $rho$ ($nusD$) hosts | Auxiliary | 144, 431, 474, 879, 925, 956, 1028, 1044, 1045, 1062, 1154, 1241 |
| $cef = mb = M1 = motC$ | Processing of T4 tRNAs | 8.5 | Auxiliary; CT439; $roc^{-}$ hosts | Auxiliary; restricted on $optA$ hosts | 431, 869, 780, 878, 937, 956 |
| $pseF = plaCTr5x?$ | 5' phosphatase | Auxiliary | 956 |
| $dexA$ | Exonuclease A | 26.0 | Affects middle transcription | Auxiliary | 308, 355, 431, 604, 737, 780, 956, 1152 |
| $dla = sud$ | DNA helicase; DNA-dependent ATPase | 49.9 | Suppress certain T4 32 mutations | Auxiliary; synthetic lethal with T4 59 mutations | 45, 309, 369, 431, 481, 546, 547, 587, 588, 649, 680, 769, 780, 783, 956, 970; P. Gauss, personal communication |
| $srd = dda.2$ | Postulated decoy of host $\sigma^{70}$ or $\sigma$s | 29.1 | $\alpha$ subunits of host RNA polymerase are incompletely modified | Auxiliary | 780 |
| $modA$ | Adenylylribosylating enzyme | 23.4 | Auxiliary | 324, 431, 435, 780, 1011, 1077 |
| $modB$ | Postulated decoy of host $\sigma^{32}$ | 24.2 | Delays early T4 gene expression at high temperatures | Auxiliary | 780, 1077 |
| $syc = modA.5$ | Affects phosphorylation of host $\sigma^{32}$ | 18.5 | Allows T4 growth in a $\sigma^{32}$ host | Auxiliary | 290, 780 |
| $mrh$ | Small outer capsid protein | 9.1 | Unstable T4 capsids | Auxiliary | 77, 89, 167, 431, 461, 462, 466, 675, 780, 916, 918 |
| $soc$ | Intron-like endonuclease. A probable fusion protein, generated from 56 and 69 by hopping of ribosomes across a pseudoknot, is larger | Nonessential | 51, 305, 677, 769, 780, 790, 783 |
| $segF = 69$ | DNA replication origin; cis-acting sequences in 56, 69, and soc; primer transcript same as transcript for these genes | 26.2 | Little DNA synthesis; unstable DNA No DNA synthesis from $oriA$ | Essential Auxiliary | 305, 347, 602, 605, 696, 769, 781, 783, 839, 1162 160, 674, 678, 691, 791, 1215 |
| 56 | dCTPase; dUTPase; dCDPase; dUDPase | 20.4 | Little DNA synthesis; unstable DNA No DNA synthesis from $oriA$ | Essential Auxiliary | 305, 347, 602, 605, 696, 769, 781, 783, 839, 1162 160, 674, 678, 691, 791, 1215 |
| $oriA$ | DNA replication origin; cis-acting sequences in 56, 69, and soc; primer transcript same as transcript for these genes | 20.4 | Little DNA synthesis; unstable DNA No DNA synthesis from $oriA$ | Essential Auxiliary | 305, 347, 602, 605, 696, 769, 781, 783, 839, 1162 160, 674, 678, 691, 791, 1215 |
| $dam$ | DNA adenine methylase | 30.4 | No DNA adenine methylation | Auxiliary | 112, 139, 395, 676, 677, 683, 742, 743, 921, 960, 961, 1072 |
| $6l = 58$ | Primase; requires interaction with gp41 helicase for priming at unique sequence | 39.8 | DNA delay | Auxiliary; 25°C or below; synthetic lethal with T4 49 or 17 mutations | 17, 47, 60, 123, 154, 380, 415, 416, 421, 422, 652, 653, 667, 761, 768, 769, 783, 788, 801, 829, 826, 831, 970, 996, 997, 998, 1216 |
| $sp = 6l.3 = rIV$ | Periplasmic protein | 11.0 | Rapid lysis; suppresses $e$ lysozyme mutations | Auxiliary | 2, 261, 492, 585, 851, 971, 1208 |
| $dnd = 6l.5$ | Discriminator of mRNA degradation | 7.0 | Excessive mRNA degradation | Nonessential; suppressed by $motA$ mutations | 491, 493, 971, 1102 |

| Replicative and recombination DNA | 53.6 | DNA arrest; little DNA displacement synthesis | Essential | 17, 54, 60, 154, 155, 184, 197, 227, 228, 264, 309, 424, 415, 416, 421, 451, 478, 479, 554, 593, 653, 652, 651, 761, 768, 769, 826, 831, 838, 930, 931, 950, 970, 1029, 1064, 1122, 1220 |
| --- | --- | --- | --- | --- |
| helicase; GTPase; ATPase; dGTPase; dATPase |  |  |  |  |
| Membrane-associated protein initiator of head vertex | 13.3 | Polyheads | Auxiliary; high temperatures | 89, 115, 116, 301, 416, 443, 500, 608, 693, 729 |
| RecA-like recombination protein; DNA-ATPase | 44.0 | UV- and X-ray sensitive; recombination deficient; suppress 49 mutations | Auxiliary | 26, 80, 82, 165, 182, 213, 254, 282, 283, 281, 301, 351, 384, 386, 392, 416, 423, 549, 572, 589, 686, 723, 739, 762, 768, 769, 938, 949, 950, 970, 1047, 1088, 1138, 1222–1225 |
| segA | 25.3 | No β-glucosylation of HMC DNA | Nonessential | 986, 988 |
| B-gt | 40.7 | No β-glucosylation of HMC DNA | Auxiliary; Shigella | 139, 316, 451, 615a, 757, 924, 1075, 1084, 1134 |
| Site-specific intron-like DNA endonuclease | 42 | Little or no DNA synthesis | Essential | 60, 68, 124, 184, 264, 320, 348, 383, 451, 476, 475, 477, 598, 611, 612, 695, 698, 834, 1027, 1076, 1114a, 114a, 1165, 1192 |
| dCMP hydroxymethylase | 28.5 | No immunity to superinfection No DNA synthesis; mutator or antimutator activities of conditional lethals under semipermissive conditions | Auxiliary Essential; nonessential dsd mutants do not grow in optA hosts | 2, 3, 4, 166, 188, 665, 666, 836, 1113, 1232 60, 68, 94, 130, 212a, 229, 230, 231, 237, 264, 259, 289, 298, 334, 343, 394, 393, 451, 488, 495, 506, 507, 527, 529, 555, 581, 617, 645, 646, 653, 689, 768, 769, 826, 827, 828, 831, 834, 856, 857, 903, 904, 905, 906, 907, 908, 909, 910, 911, 912, 913, 914, 970, 978, 983, 1029, 1030, 1033, 1046, 1088, 1128, 1142, 1143, 1147, 1150, 1165, 1207 |
| imm | 9.3 |  |  |  |
| Inner membrane protein DNA polymerase; 3′-to-5′ exonuclease | 103.6 |  |  |  |
| 43 |  |  |  |  |
| Translational repressor of several early genes | 14.6 | Extended synthesis of several early proteins | Auxiliary; restricted in E. coli rpoB5081 at 42°C | 9, 10, 19, 131, 320, 338, 484, 485, 503, 505, 637, 735–738, 835, 860, 951, 952, 953, 975, 1095, 1167, 1182 |
| Clamp-loader subunit | 21.4 | No DNA synthesis | Essential | 17, 20, 57, 60, 264, 310, 311, 312, 320, 451, 469, 470, 486, 487, 488, 616, 619, 653, 679, 826, 831, 834, 865, 901, 902, 953, 1089, 1217, 1227 |
| 62 |  |  |  |  |
| Clamp-loader subunit | 35.8 | No DNA synthesis | Essential | 20, 57, 310, 311, 312, 320, 469, 470, 486, 487, 488, 616, 618, 619, 679, 826, 831, 864, 865, 901, 902, 970, 1032, 1089, 1217, 1227 |
| 44 |  |  |  |  |
| Processivity enhancing sliding clamp of DNA polymerase; and mobile enhancer of late promoters | 24.9 | No DNA synthesis; no late transcription | Essential | 17, 20, 22, 264, 299, 310, 311, 312, 320, 334, 404, 405, 451, 469, 552, 552a, 616, 617, 619, 618, 653, 673, 679, 747, 786, 826, 831, 834, 865, 901, 902, 951, 952, 953, 1082, 1091, 1092, 1186, 1217, 1227 |
| subunit |  |  |  |  |
| RNAP-binding protein | 14.7 | Essential in B strains; mutants are “leaky” in some K strains | Auxiliary | 444, 552, 786, 1171, 1172, 1174 |
| Recombination protein and nuclease | 63.6 | Recombination deficient; DNA arrest; no host DNA degradation | Essential in B strains; mutants are “leaky” in some K strains | 60, 68, 93, 111, 184, 195, 264, 345, 380, 403, 451, 605, 627, 668, 731, 740, 744, 768, 769, 784, 1036, 1047, 1138, 1163, 1164 |
| subunit |  |  |  |  |
| α-Glucosyltransferase | 46.7 | No α-glucosylation of HMC | Auxiliary | 140, 316, 345, 437, 924, 1072, 1084, 1191 |
| Putative site-specific intron-like DNA endonuclease | 30.4 |  | Nonessential | 48, 1072 |
| mobB |  |  |  |  |
| σ factor recognizing late T4 promoters | 21.5 | No late transcription | Essential | 97, 98, 109, 310, 313, 311, 345, 404, 552, 635, 834, 895, 1038, 1082, 1171, 1173, 1174, 1186 |
| 55 |  |  |  |  |

| Gene | Description | Function | Essentiality |
| --- | --- | --- | --- |
| **nrdH = 55.7** | Anaerobic nucleotide reductase subunit | 11.7 | Auxiliary |
| **nrdG = 55.9** | Anaerobic nucleotide reductase subunit | 18.2 | Auxiliary |
| **mobC = 55.10** | Putative intron-like DNA endonuclease | 24.0 | Anaerobic growth |
| **nrdD = sunY** | Anaerobic ribonucleotide reductase subunit; RNA contains a self-splicing intron | 68.0 | M. Ohman-Heden, personal communication |
| **I-TevII** | Endonuclease for *nrdD*-intron homing | 30.4 | Nonessential |
| **49'** | Internal translation initiation product | 11.9 | No resolution of recombination junctions; incomplete packaging of DNA; reduced heteroduplex repair, reduced DNA synthesis | Essential |
| **pin** | Inhibitor of host Lon protease | 18.8 | Degradation of amber peptides | Auxiliary |
| **ndc** | Thioredoxin, glutaredoxin | 10.1 | Nonessential | Auxiliary |
| **mobD** | Putative site-specific DNA endonuclease | 30.5 | Nonessential | Auxiliary |
| **rl = tk-2** | Membrane protein | 11.1 | No lysis inhibition | Auxiliary |
| **tk** | Thymidine kinase | 21.6 | Auxiliary |
| **vsr** | Modifier of valyl-tRNA synthetase | 13.1 | Misregulation of early genes; specific mRNAs stabilized | Auxiliary |
| **regB** | Site-specific RNase | 18.0 | Essential, except when suppressed by sp and 5 mutations | Auxiliary |
| **denV** | Endonuclease V; N-glycosidase | 16.1 | UV sensitive | Auxiliary |
| **ipII** | Internal protein II | 11.1*9.9 | Allow T4 growth in *nusD rho* hosts | Essential, except when suppressed by sp and 5 mutations |
| **ipIII** | Internal protein III | 21.7*20.4 | Nonessential | Auxiliary |
| **e** | Soluble lysozyme; endolysin | 18.7 | No cell lysis | Nonessential |
| **nudE = e.1** | Nudix hydrolase | 17.0 | Nonessential | Auxiliary |
| **gofF3** | Stable RNA |  | Allow T4 growth in *nusD rho* hosts | Auxiliary |
| **rnaC = species 1** | Stable RNA |  | Nonessential | Auxiliary |
| **rnaD = species 2** | Stable RNA |  | Nonessential | Auxiliary |
| **tRNAArg** | psu4 opal suppressor |  | Auxiliary | CT439 |
| **segB** | Probable site-specific intron-like DNA endonuclease | 26.2 | Nonessential | Auxiliary |
| **tRNAIle** | psu<sub>a</sub>; psu<sub>b</sub>; psu<sub>c</sub>; amber suppressors |  | Auxiliary | CT439 |
| **tRNAThr** |  |  | Auxiliary | CT439 |
| **tRNASer** |  |  | Auxiliary | CT439 |
| **tRNAPro** |  |  | Auxiliary | CT439 |
| **tRNAGly** |  |  | Auxiliary | CT439 |
| **tRNALeu** | psu<sub>3</sub>; SB |  | Auxiliary | CT439 |
| **tRNAGln** | psu<sub>2</sub> |  | Auxiliary | CT596 |
| **ipl** | Internal protein 1 | 10.2*8.5 | ? | Essential; bypassed by certain host mutations |
| **57B** | Chaperone of long and short tail fiber assembly | 17.3 | Defective tail fiber assembly | Essential |
| **57A** |  | 8.7 | No DNA synthesis | Essential |
| **1** | dNMP kinase | 27.3 | No DNA synthesis | Essential |

References:
- 39, 70, 79, 81, 82, 172, 213, 249, 250, 278, 300, 285, 330, 331, 332, 333, 350, 522, 523, 524, 525, 526, 541, 559, 669, 670, 740, 768, 769, 783, 788, 792, 793, 817, 883, 1025, 1030, 1029, 1047, 1083, 1085, 1226; G. Mosig and D. Powell, Abstr.
- Annu. Meet. ASM, p. 209, 1985
- 39, 784, 788
- 1005, 1012, 1085
- 64, 257, 341, 460, 632, 815, 816, 1066, 1085
- 1072
- 3, 56, 224, 236, 851
- 620, 621, 622, 623, 656, 657, 685, 714,
- 715, 716, 718, 758, 806, 812, 813, 832,
- 862, 884, 885, 899, 969, 1110, 1111,
- 1112, 1117, 1120, 1151, 1212, 1213
- 84, 88, 89, 442, 595, 604, 1093, 1112
- 803, 804, 1093, 1112, 1114a
- 704, 720, 787, 850, 871, 872, 948, 1050,
- 1180
- 1204
- 720, 1045
- 110, 302, 707, 870, 962
- 110, 302, 707, 870, 962
- 5, 110, 284, 302, 328, 361, 366, 367, 501,
- 707, 710–712, 870, 962, 964, 1178, 1179,
- 1112
- 110, 302, 604, 772, 988
- 302, 707, 962
- 302, 707, 962
- 302, 707, 962
- 302, 707, 962
- 302, 707, 962
- 302, 707, 962
- 6, 84, 87, 88, 89, 110, 442, 543, 595, 1093,
- 110, 280, 409, 410, 922
- 110, 122, 186, 319, 391, 401, 409, 410, 699,
- 922
- 110, 184, 241, 264, 348, 543, 696, 834

| 3 | Head-proximal tip of tail tube | 19.7 | Unstable tails | Essential | 7, 186, 264, 535, 536, 543, 648, 1124 |
| --- | --- | --- | --- | --- | --- |
| 2 = 64 | Protein protecting DNA ends | 31.6 | Noninfectious particles with filled heads | Essential, except in recBCD hosts | 28, 89, 186, 249, 250, 264, 543, 647, 1000 |
| 4 = 50 = 65 | Head completion protein | 17.6 | Noninfectious particles with filled heads but tails attached at wrong angles | Essential | 1001, 1074, 1144 |
| 53 | Base plate wedge component | 23.0 | Defective tails | Essential | 186, 249, 250, 530, 531, 532, 787, 1155, 1157 |
| 5 | Base plate lysozyme; hub component | 63.1* 44* 19 | Defective tails | Essential | 2, 186, 249, 264, 497, 500, 530, 531, 532, 787, 807, 1063, 1119, 1157 |
| oriE | DNA replication origin; cis-acting sequences in genes 4, 53, 5; primer transcript in opposite orientation of gene 5 transcripts | No DNA synthesis from oriE | Auxiliary | G. Mosig, unpublished data |
| repEB | Protein required for initiation from oriE | 5.48 | No DNA replication from oriE | Auxiliary; synthetic lethal | 1109 |
| repEA | Protein auxiliary for initiation from oriE | 6.13 | Anomalous DNA replication from oriE | with motA mutation | 1109 |
| segC | Site-specific intron-like DNA endonuclease | 16.0 | Nonessential | 490, 641, 988, 989a; Lin and Mosig, unpublished |
| 6 | Base plate wedge component | 74.4 | Defective tails; permit plating of fiberless phage | Essential | 186, 193, 249, 253, 264, 537, 1119, 1157; R. Marsh, personal communication |
| 7 | Base plate wedge component | 119.2 | Defective tails; permit plating of fiberless phage | Essential | 186, 193, 249, 250, 253, 264, 537, 1119, 1156, 1157; Marsh, personal communication |
| 8 | Base plate wedge component | 38.0 | Defective tails | Essential | 186, 250, 249, 253, 264, 537, 1119, 1156, 1157 |
| 9 | Base plate wedge component, tail fiber socket, trigger for tail sheath contraction | 31.0 | No attachment of tail fibers | Essential | 186, 250, 264, 535, 537, 562, 876, 1114a, 1155, 1157 |
| 10 | Base plate wedge component, tail pin | 66.2 | Defective tails | Essential | 186, 249, 250, 264, 272, 537, 726, 867, 868, 876, 1119, 1155, 1156, 1157, 1239 |
| 11 | Base plate wedge component, tail pin, interface with short tail fibers, gp12 | 23.7 | Defective tails | Essential | 186, 249, 250, 264, 535, 537, 633, 868, 867, 876, 1119, 1155, 1157, 1239 |
| 12 | Short tail fibers | 56.2 | Defective tails | Essential | 122, 186, 391, 521, 535, 537, 745, 972, 1114a, 1155, 1157 |
| wac | Whiskers, facilitate long tail fiber attachment | 51.9 | No whiskers | Auxiliary | 186, 214, 745, 877, 1024, 1065, 1114a, 1188 |
| 13 | Head completion | 34.7 | Inactive, but filled heads | Essential | 89, 249, 250, 264, 973, 1114a |
| 14 | Head completion | 29.6 | Inactive, but filled heads | Essential | 89, 250, 249, 264, 973, 1114a |
| 15 | Proximal tail sheath stabilizer, connector to gp3 and/or gp19 | 31.6 | Defective tails | Essential | 249, 250, 264, 272, 535, 537, 973, 1114a |
| 16 | Terminase subunit, binds dsDNA | 18.4 | Empty heads | Nearly essential | 85, 86, 89, 90, 249, 250, 264, 286, 287, 643, 644, 669, 802, 803, 873, 891, 1194 |
| 17 | Truncated C-terminal end | 69.8 | Empty heads | Essential | 75, 85, 86, 89, 90, 249, 250, 264, 286, 287, 288, 433, 586, 631, 642, 643, 669, 746, 769, 784, 785, 873, 891, 892, 893, 1194, 1195, 1196 |
| 17' | Terminase subunit with nuclease and ATPase activity; internal transcription and translation in frame; does not bind ssDNA | 59.2 | ? | ? | 286, 287, 288 |
| 17" | Terminase subunit with nuclease and ATPase activity (transcript processing and internal initiation of translation in frame); does not bind ssDNA; several additional proteins most likely initiated from internal ribosome binding sites of the 17 transcripts | 57.1 | ? | ? | 286, 287, 288, 333, 784 |

| 18 | Tail sheath monomer | 71.3 | Defective tails | Essential | 29, 31, 186, 249, 250, 264, 272, 535, 537, 1096, 1119, 1157 |
| --- | --- | --- | --- | --- | --- |
| 19 | Tail tube monomer | 18.5 | Defective tails | Essential | 30, 186, 249, 250, 264, 272, 535, 536, 537, 1119, 1157, 1194 |
| 20 | Portal vertex protein of the head | 61.0 | Polyheads | Essential | 86, 89, 90, 238, 250, 264, 333, 608, 642, 643, 694, 990, 1114a |
| pip = 67 | Prohead core protein; precursor to internal peptides | 9.1* small peptides | Defective heads | Essential | 89, 519, 1130, 1131 |
| 68 | Prohead core protein and protease | 15.9 | Isometric heads | Essential | 89, 516, 518, 520 |
| 21 | Prohead core protein and protease (internal initiation of translation) | 23.3* small peptides | No or defective heads | Essential | 89, 250, 264, 329, 414, 516, 517, 606, 608, 844, 845, 990, 1116 |
| 22 | Precursor of major head subunit | 29.9* small peptides | No or faulty heads | Essential | 89, 250, 264, 270, 518, 595, 608, 728, 805, 844, 845, 990, 1094, 1093, 1114a |
| 23 | Precursor of head vertex subunit | 56.0* 48.7* 43 | No or faulty heads; gol mutations in gene 23 allow growth in lit hosts (CTR5x) | Essential; Gol peptide together with E. coli Lit, cleaves host EF-Tu | 256, 260, 264, 270, 315, 396, 461, 466, 606, 608, 684, 719, 754, 825, 841, 854, 936, 1021, 1093, 1094, 1119, 1231 |
| segD | Probable site-specific intron-like DNA endonuclease | 25.6 | Nonessential | Nonessential | 490, 988 |
| 24 = os | rnlB = 24.1 | 47.0* 46 | No or faulty heads, osmotic shock resistance | Essential; bypassed by certain gene 23 mutations | 8, 76, 89, 250, 262, 264, 396, 461, 606, 608, 634, 719, 1114a; G. Yasuda, G. A. Churchill, M. Parker, and D. Moorey, personal communication |
| hoc = eph | inh = lip | 37.6 | Unstable capsids | Auxiliary | 426 |
| segE | uvsW = dar | 40.4 | ? | Auxiliary | 89, 167, 164, 168, 461, 462, 496, 916, 917, 1205 |
|  | Second RNA ligase |  |  |  | 496 |
|  | Large outer capsid protein |  |  |  |  |
|  | Minor capsid protein; inhibitor of gp21 protease |  |  |  |  |
|  | Probable site-specific intron-like DNA endonuclease | 22.9 | Nonessential |  | 489, 490, 988 |
|  | RNA-DNA- and DNA-helicase; DNA-dependent ATPase | 67.5 | UV sensitive; fail to unwind R-loops; suppress T4 59 uvsX, uvsY, and 46 mutations | Auxiliary | 132, 195, 196, 207, 208, 212, 244, 722, 737, 768, 769, 1061, 1191, 1197, 1199, 1222, 1240 |
|  | ssDNA binding, recombination and repair protein; helper of UvsX, inhibitor of endoVII | 15.8 | UV sensitive; recombination-deficient; repair-deficient, DNA arrest; suppress T4 49 mutations | Auxiliary | 25, 44, 82, 182, 183, 195, 212, 213, 232, 357, 358, 380, 387, 392, 522, 542, 548, 589, 723, 724, 739, 768, 769, 1013, 1047, 1054, 1055, 1060, 1061, 1138, 1191, 1199, 1210, 1225, 1222, 1223, 1240 |
| oriF = oriuvSY | uvsY = fdsB |  | No DNA synthesis from oriF | Auxiliary | 46, 133, 357, 378, 563, 574, 573, 576, 577, 678, 724, 779, 830, 1109, 1215 |
|  | DNA replication origin; cis-acting sequences in genes uvsY, uvsY.-J and uvsY.-2; primer transcript same as uvsY, uvsY.-1 and uvsY.-2 transcript |  |  |  |  |
| 25 | Base plate wedge subunit | 15.1 | Defective tails | Essential | 186, 249, 250, 264, 356, 358, 357, 530, 531, 532, 540, 822, 819, 1057, 1155, 1157; B. Szewczyk and J. Nieradko, personal communication; E. Tourkin and B. Poglozov, 186, 249, 250, 264, 357, 540, 567, 821, 958, 1108, 1157, 1240 357, 823 |
| 26 | Base plate hub subunit | 23.9 | Defective tails | Essential | 186, 249, 250, 264, 357, 540, 567, 820, 958, 1108 |
| 26' | Internal in-frame translation initiation | 12 | ? | Essential | 104, 186, 193, 249, 250, 264, 532, 1157, 1240 |
| 26" | Internal out-of-frame translation initiation | 10.9 | ? | Essential |  |
| 5I | Base plate hub assembly catalyst? | 29.3 | Defective tails | Defective tails |  |
| 27 | Base plate hub subunit | 44.5 | Defective tails; permit plating of fiberless phage | Essential |  |

| Base plate distal hub subunit | 20.1 | Defective tails | Essential |
| --- | --- | --- | --- |
| 186, 249, 250, 264, 532, 565, 566, 568, 157, 1240 |
| Base plate hub; determinant of tail length | 64.4 | Defective tails | Essential |
| 7, 186, 242, 243, 249, 250, 264, 532, 463, 464, 1157, 1240 |
| Base plate; tail tube associated | 39.7 | Defective tails | Essential |
| 61, 186, 242, 243, 249, 250, 264, 463, 464, 530, 535, 536, 1114a, 1157 |
| Base plate-tail tube initiator | 35.0 | Defective tails | Essential |
| 61, 186, 243, 250, 249, 264, 463, 464, 530, 531, 535, 536, 1155, 1157 |
| Adenosylribosyltransferase (packaged and injected with DNA) | 75.8 | Synthetic defective with *modA* and *modB* deletions | Auxiliary |
| 323, 324, 413, 435, 544, 545, 552, 937a, 1170 |
| DNA ligase | 55.3 | DNA arrest; hyperrecombination | Essential; can be bypassed by functioning host ligase, when T4 *II* is defective |
| 32, 68, 184, 269, 439, 504, 580, 734, 776, 769, 821, 826, 1177, 1235; Thomas et al., unpublished |
| Unknown | 9.3 | Rapid lysis | Auxiliary |
| 2, 3, 4, 224, 851, 875, 896, 897, 898, 923, 1235 |
| Cochaperonin for GroEL | 12.1 | Head assembly; gp23 forms lumps; T4 topoisomerase is defective | Essential |
| 27, 89, 226, 250, 264, 318, 319, 528, 607, 613, 693, 818, 824, 875, 896, 897, 898, 926, 927, 928, 1004, 1093, 1094, 1104, 1115 |
| dCMP deaminase | 21.2 | Auxiliary | Auxiliary; CTr5x (*lit*) |
| 170, 347, 376, 377, 682, 696, 717, 755, 756 |
| Deoxyribonucleotide 3′ phosphatase, 5′ poly nucleotide kinase | 34.6 | Auxiliary; *coli B rpoB5087* | 129, 185, 203, 512, 513, 514, 732, 734, 1008, 1021; Thomas et al., unpublished |
| RNA polymerase- and DNA-binding protein; transcription terminator on dC-containing DNA | 19.0 | Allow transcript elongation on C-DNA; no unfolding of host nucleoid | E. coli (pR386) |
| 239, 406, 407, 511, 550, 597, 603, 786, 976, 1009, 1019, 1020, 1022 |
| RNA ligase; catalyst of tail fiber attachment | 43.5 | Defective tail fiber attachment | Auxiliary |
| 319, 362, 374, 467, 721, 734, 889, 944, 1014, 1114a |
| Endonuclease II that restricts dC-containing DNA | 16.7 | Defective in host DNA degradation | Auxiliary; restricted in *E.* coli B rpoB5087 |
| 114, 162, 173, 179, 255, 274, 341, 342, 348, 349, 373, 377, 397, 431, 604, 695, 698, 1149, 1153 |
| Ribonucleotide reductase β subunit (contains intron) | 45.4 | Reduced DNA synthesis | Auxiliary; *nrd*-defective hosts |
| 63, 169, 173, 247, 341, 342, 348, 349, 382, 398, 696, 734, 1010, 1097, 1191, 1218, 1219 |
| *rmlA = 63* | 11.3 | Nonessential | Nonessential |
| 178, 247, 342, 609, 882, 982, 991, 993, 1003, 1010 |
| *denA* | 16.5 | Nonessential | Auxiliary; *nrd*-defective hosts |
| Thomas et al., unpublished |
| *mobE* | 86.0 | Reduced DNA synthesis | 63, 169, 348, 349, 382, 398, 604, 696, 734, 1098, 1097, 1218, 1219 |
| *nrdA* | 33.1 | Reduced DNA synthesis | Auxiliary; *td*-defective hosts |
| 717, 798, 881, 882, 991, 1002, 1157 |
| *td* | 28.2 | Nonessential | 52, 114, 119, 120, 174, 175, 178, 206, 251, 252, 342, 348, 349, 453, 661, 662, 796, 797, 798, 882, 991, 993 |
| I-TevIII | 11.3 | Reduced DNA synthesis | Auxiliary |
| 162, 173, 347, 349, 377, 381, 431, 604, 734, 763, 880, 881, 882 |
| Dihydrofolate reductase | 21.7 | Reduced DNA synthesis | Essential; Tab32 for *ts* mutants; 32 *am* mutations in ochre-suppressor-containing hosts are suppressed by *ddla* mutations |
| 18, 60, 107, 108, 142, 145–148, 154, 155, 264, 307, 392, 438, 440, 451, 465, 510, 549, 554, 564, 570, 579, 581, 629, 630, 660, 739, 760, 764, 768, 776, 774, 785, 794, 784, 811, 826, 831, 945, 982, 984, 985, 1034, 1055, 1047, 1054, 1064, 1071, 1087, 1128, 1132, 1161, 1173, 1176, 1198, 458, 473, 508, 509, 562, 663, 980, 1125–1127, 1133, 1136, 1200, 1223 |
| ssDNA-binding protein, scaffold of DNA replication, recombination and DNA precursor-synthesizing protein machines | 33.5 | DNA arrest, UV sensitive, recombination and excision repair deficient | DNA arrest; fail to load gp41 helicase onto recombination intermediates, or ssDNA covered with gp32 orUvsX |
| 38, 142, 195, 309, 371, 465, 478, 577, 629, 630, 760, 761, 768, 769, 799, 826, 830, 831, 890, 979, 1030, 1064, 1197, 1199, 1201, 1202, 1221 |
| *fdt* | 32 | Site-specific DNA endonuclease; Loader of gene 41 DNA helicase, ssDNA-binding protein | Almost essential |
| 48, 655 |
| *segG = 32.1* | 59 | Site-specific DNA endonuclease; Loader of gene 41 DNA helicase, ssDNA-binding protein | Nonessential |


Protein connecting gp45 and gp55, to allow transcription by RNA polymerase from late promoters

| Gene | Function | Essential/Auxiliary |
|------|----------|---------------------|
| **dsbA** | RNase H; 5' to 3' DNase; yeast FEN homologue | Essential |
| **rht = das** | Facilitates some late RNA synthesis Defective processing of Okazaki fragments; *das* mutations suppress T4 46, 47 and *uvsX* mutations | Auxiliary |
| **Proximal tail fiber subunit** | No DNA synthesis from *oriG* | Essential |
| **oriG = ori34** | DNA replication origin; primer transcript in opposite orientation of 34 transcript | Auxiliary |
| **Tail fiber hinge** | Fiberless particles | Essential |
| **Small distal tail fiber subunit** | Fiberless particles | Essential |
| **Large distal tail fiber subunit** | Fiberless particles, host range | Essential |
| **Assembly catalyst of distal tail fiber** | Fiberless particles | Essential |
| **Holin, inner membrane pore protein,** | Affect lysis by *e* lysozyme; suppress T4 *rII* and 63 mutations | Essential |
| **affects lysis timing and inhibition** | Defective middle mode, and (indirectly) late transcription | Almost essential |
| **Protein that binds to host σ70**, inhibits interaction with –35 regions of classical promoters, and facilitates interaction with T4 MotA protein | Inhibitor of MrBC restriction nuclease | Auxiliary |
| **DNA topoisomerase subunit;** | DNA delay | Essential; temperatures below 25°C; inhibition of host topoisomerase IV with novobiocin |
| **membrane-associated protein** | Unknown | Auxiliary; CT447 |
| **Membrane protein** | Activator of middle mode binding protein specific for mot boxes | Almost essential |
| **Peptide modulating host restriction system** | Defective middle mode transcription; suppress *rII*-defects in λ lysogens; affects interaction with σ70 and AsiA | Auxiliary |
| **Protein that disrupts host nucleoid; binds to host HU** | Nucleoid disruption defective with DC-containing DNA | Auxiliary |
| **pla262** | Allow progeny production of T4 | Auxiliary |
| **denB** | Endonuclease IV, single-strand-specific endonuclease | Auxiliary |
| **Membrane-associated protein; affects host membrane ATPase** | Rapid lysis; suppresses T4 30 and some 32 mutations | Auxiliary; *rev+λ* lysogens; P2-like HK239 lysogen; *tabR* |

---

* **Genes are listed by the currently used names, followed by alternative designations in the literature.**
* **Gene products processed into smaller peptides are indicated (\*) with the sizes or size range following the principal product.**
* **Because the distinction between “essential” and “nonessential” is not always obvious, when mutants have not been tested under all possible growth conditions or in all possible hosts, some “nonessential” genes are noted as “auxiliary.” Where known, restrictive hosts or plating conditions for mutant genes are noted.**


(or ribosome binding sites [RBS]). These include *alc*, *vs.4*, *e.5*, *tRNA.2*, and *57B*. Until further evidence is available, we have listed these genes as simply starting from the first of the two possible sites. It will be interesting to determine if both starts are used in any or all of these cases and if there are special functions for two nearly identical proteins. In bacteriophage lambda, for example, two nested proteins, differing in start sites by only two amino acids, have important complementary functions: one makes the pore to permit access by lysozyme to the peptidoglycan layer, and the other delays formation of the pore (91). The regulation of the balance between these two genes is not understood but is crucial in determining the timing of lysis.

(iii) It is clear that there can be genes within genes in different reading frames. These can be read in the same direction, as seen for gene 30.3′ (1234). They can also be in the opposite orientation, as seen for genes *repEA* and *repEB*, which are associated with initiation from origin E and are located opposite gene 5 (1109).

(iv) Introns that are later spliced out of the transcripts occur in at least three T4 genes: the thymidylate synthase gene (*td*), the gene encoding a subunit of the aerobic ribonucleotide reductase (*nrdB*), and the anaerobic ribonucleotide reductase gene (*nrdD*) (615, 991, 1229).

(v) As first demonstrated in T4 gene 60, an unusual relationship between nucleic acid and protein sequence can also occur through translational bypassing. A 50-base mRNA segment in the coding region is not translated in gene 60 by a mechanism that depends on *cis*-acting signals in the mRNA, ribosomal protein L9, a pair of GGA codons 47 bases apart, and the structure of the cognate glycyl tRNA (408, 450). This is the only known high-efficiency bypass site; to date, the phenomenon is unique to T4. Bypass with much lower efficiency appears to occur at the junction of genes 56 and 69 (*segF*) (160, G. Mosig, unpublished data).

Programmed frameshifting, which shifts translation by 1 base into the +1 or −1 reading frame, can expand the coding capacity of a genome (13). To date, no instance of programmed frameshifting has been identified in T4, although many other viral DNA and RNA genomes use this approach to “recode” (322).

T4 shows nearly four times the gene density predicted for herpesviruses and yeast and twice that for *E. coli* (92, 556, 557). The high gene density reflects both the small size of many T4 genes and the fact that there are very few noncoding regions (about 9 kb, 5.3% of the genome). Furthermore, regulatory regions are compact, occasionally overlapping coding regions. In many cases, the termination codon of one gene overlaps the start codon of the next gene (see “Translation and posttranscriptional control” below). In addition, T4 has several groups of nested genes as mentioned above. Clearly, computational and bioinformatic tools do not yet identify all the genes and complex coding arrangements in a genome perceived by many to be “simple,” like that of T4.

Table 3 summarizes the functional assignments of T4 genes, referring to the color codes used in the functional genome map of Fig. 3. Some T4 proteins have multiple activities and are listed in more than one group. For example, T4 RNA ligase A (*rnlA* or 63) is also a catalyst for attaching tail fibers. Alternatively, a single activity can be viewed as being involved in

---

**TABLE 3. Functional categories of T4 genes**  
**a**

| A. Transcription [red] |
| --- |
| *asiA*, *dsbA*, *goF*, *modA*, *motA*, *motB*, *mrh*, *rpbA*, *srd*, *srh*, 33, 55 (*alc*, *alt*, 45) |

| B. Translation [brown] |
| --- |
| *cef*, *dmd*, *modB*, *regA*, *regB*, *vs.*, *rnlA*, *rnlB* (*modA*), tRNAR, tRNAl, tRNAT, tRNAS, tRNAP, tRNAG, tRNAL, tRNAE; *rnaC*, *rnaD* |

| C. Nucleotide metabolism [orange] |
| --- |
| *cd*, *denA*, *denB*, *frd*, *nrdA,B,C,D,G,H*, *nudE*, *pseT*, *td*, *tk*, 1, 42, 56 |

| D. DNA replication, recombination, repair, packaging, and processing [yellow] |
| --- |
| *dda*, *denV*, *dexA*, *repEA*, *repEB*, *rnH*, *uvsW*, *uvsX*, *uvsY*, 16, 17, 30, 32, 39, 41, 43, 44, 45, 46, 47, 49, 52, 60, 61/58, 62 |

| E. Virion proteins [blue] |
| --- |
| Head: *soc*, *hoc*, *inh*, *ipl*, *ipII*, *ipIII*, 2, 4, 20, 23, 24, 67, 68, (22, 21) [dark blue] Neck: 13, 14 [medium blue] Tail: 3, 5, 6, 7, 8, 9, 10, 11, 12, 15, 18, 19, 25, 26, 27, 28, 29, 48, 53, 54 [light blue] Tail fiber: *wac*, 34, 35, 36, 37 (*rnlA*) [pale blue] |

| F. Chaperonins/assembly catalysts [stippled blue] |
| --- |
| 21, 22, 31, 38, 40, 51, 57A, *rnlA* |

| G. Lysis [green] |
| --- |
| *e*, *rI*, *rIII*, *sp*, *t* (*rIIA*, *rIIB*) |

| H. Host or phage interactions [purple] |
| --- |
| *ac*, *arn*, α-*gt*, β-*gt*, *dam*, *imm*, *pin*, *rIIA*, *rIIB*, *stp*, (*gol*, *pseT*, *rnlA*) |

| I. Host alteration/shutoff [pink] |
| --- |
| *alc*, *alt*, *gol*, *ndd* (*denA*, *denB*, *modA*, *modB*) |

| J. Homing endonucleases and homologs [peach] |
| --- |
| I-Tevl-III, *mobA-mobE*, *segA-segG* |

| K. Predicted integral membrane or periplasmic proteins [squiggle] |
| --- |
| *denB.-1*, *e.2*, *e.3*, *e.4*, *ndd.3*, *ndd.4*, *ndd.5*, *nrdC.7*, *pseT.3*, *tRNA.4*, 47.1, 52.1, 55.8 (*ac*, *imm*, *rI*, *t*, 7, 29) |

| L. Unknown function [white] |
| --- |
|  |

*"a" Genes in parentheses appear in another, primary category. Primary functional assignments and the corresponding colors are used in Fig. 3.*

---

Multiple processes. For example, the nucleases EndoII and EndoIV (encoded by *denA* and *denB*) are responsible primarily for initiating degradation of cytosine-containing host DNA. They are included in the “nucleotide precursor” category because one important function of these proteins is the timely provision of nucleotide precursors. They are also included among the host alteration/shutoff genes.

---

### Characterized T4 Genes and the Early Genetics

Only 62 of the T4 genes are “essential” under standard laboratory conditions (rich medium, aeration, 30 to 37°C); mutants altered in a few other genes produce very small plaques under standard conditions. Many of these key genes are much larger than the average T4 gene; together, they occupy almost half of the genome. They include genes that encode proteins of the replisome and of the nucleotide-precursor complex, several transcriptional regulatory factors, and most of the structural and assembly proteins of the phage particle. Most of these genes were first identified by the isolation of amber or temperature-sensitive conditional-lethal mutations and were assigned numbers (Table 2) before their functions were determined (264).

rllA 1200
rlIA.1 2400
60 3600
mobA 4800
39 6000
goFcefmotB 7200
motB.1 8400
dexA 9600
dda 10800
dda.1 12000
modAmodB 13200
modA.2
modA.3
modA.4
modA.5

mrh 14400
mrh.1 15600
soc segF 16800
oriA 18000
dam 19200
61 20400
sp 21600
dmd 22800
uvsX 24000
segA 25200
B-gt 26400
immimm.1 27600

43 28800
regA 30000
62 31200
44 32400
45 rpbA45.2 33600
46 34800
47 36000
a-gt 37200
mobB 38400
a-gt.2 39600
a-gt.3 40800
a-gt.4 42000
a-gt.5 45.1
nrdH 45.6
49' 45.8

nrdGmobC 43200
nrdD 44400
I-Tevl 45600
nrdD 46800
pin 48000
nrdC.1 49200
nrdC.2 50400
nrdC.3 51600
nrdC.4 52800
nrdC.5 54000
nrdC.6 55200
nrdC.7 56400
nrdC.8 56400
nrdC.9 56400
nrdC.10 56400
nrdC.11 56400

mobD.1 57600
mobD.2 58800
mobD.3 60000
mobD.4 61200
mobD.5 62400
rl.1 63600
tk.1 64800
tk.2 66000
tk.3 67200
vs vs.1 regB 68400
vs.3 69600
vs.4 69600
vs.5 69600
vs.6 69600
vs.7 69600
e 69600
nudE 69600
iplll 69600
e.2 69600
e.3 69600
e.4 69600
e.5 69600
e.6 69600
e.7 69600
e.8 69600

tRNA(1) 70800
RNA-C, D 72000
tRNAs(7) 73200
segB 74400
tRNA.2 75600
ipl 76800
57B57A 78000
1 79200
repEB 80400
oriE 81600
repEA 82800
5.1segC 84000
5.3 85200
5.4 86400
6 87600
7 88800
8 90000
9 91200
10 92400
wac 93600
11 94800
12 96000
13 97200
14 98400
15 99600
16 100800
17 102000
18 103200
19 104400
20 105600
21 106800
22 108000
23 109200
segD 110400
rnIB 111600
hoc 112800
inh 114000
segE 115200
uvsW 116400
uvSY-2 117600
uvSY-1 118800
oriF 120000
26' 121200
25 122400
26 123600
51 124800
27 126000
28 127200
29 128400
48 129600
54 130800
alt 132000
alt.3 133200
alt.2 134400
alt.-1 135600
30 136800
30.1 138000
30.2 139200
30.3 140400
30.4 141600
30.5 142800
30.6 144000
30.7 145200
30.8 146400
30.9 147600
rlll 148800
31 150000
cd 151200
pseT.1 152400
pseT.2 153600
pseT.3 154800
alc 156000
rnIA 157200
denAnrdB 158400
I-Tevl 159600
nrdB 160800
mobE 162000
nrdA 163200
nrdA.1 164400
nrdA.2 165600
td 166800
I-Tevl 168000
td 170400
frd 172800
frd.1 175200
frd.2 177600
frd.3 180000
32 182400
segG 184800
59 187200
33dsbArn 189600
oriG 192000
34 194400
35 196800
36 199200
37 201600
38 204000
t asiA 206400
arm.1 208800
arm.2 211200
arm.3 213600
arm.4 216000
motA 218400
motA.1 220800
52 223200
stp 225600
ndd.1 228000
ndd.2 230400
ndd.3 232800
ndd.4 235200
ndd.5 237600
ndd.6 240000
denB 242400
rlIB 244800


FIG. 3. Functional genome map of bacteriophage T4. The coding capacity of the T4 genome is shown for both characterized and hypothetical ORFs. The color scheme (by gene function) is as defined in Table 3. Origins of DNA replication (ori) indicated are those that are best characterized. Locations of the multiple promoters and terminators can be determined from Table 1.
Nonessential genes were typically assigned letter designations, reflecting the phenotype associated with the mutation (Table 2) or the host function that the gene duplicated (*nrd*, *frd*, *td*, etc.). They encode such products as enzymes for nucleotide biosynthesis, recombination, and DNA repair; nucleases to degrade cytosine-containing DNA; proteins responsible for exclusion of superinfecting phage, for lysis inhibition under conditions of high phage/host ratios, and for other membrane changes; and inhibitors of host replication, transcription, and protease activity. Unfortunately, the designation by letters versus numbers does not automatically identify a gene as essential. For example, the products of genes *t*, *motA* and *asiA* are essential under standard conditions, while that of 69 (*segF*) is not. Mutations in genes 46 and 47 still permit the synthesis of a few phage per cell, but too few are produced to reliably produce plaques under most conditions; a burst size of about 10 is generally required for plaque formation. Primase (gene 61) and topoisomerase (genes 39, 52, and 60) mutants produce plaques at temperatures above 25°C because they can use a recombinational bypass mechanism to prime lagging-strand DNA synthesis (784, 788). In several cases, mutations initially assigned to different genes by spot-test complementation ultimately proved to reside within the same gene; thus, genes 58 and 61 are identical, as are genes 2 and 64 and genes 4, 50, and 65.

Most genes first identified by mutation have now been located in the DNA sequence. However, no genes have yet been identified for any of the reported ribosome-binding proteins or other proteins that might be involved in the shutoff of host translation (reviewed in reference 1166). Mutations *ama*, *stI*, *stIII*, *rs*, *goFB*, and *goFC* have not been assigned to a sequence; the original mutants identifying most of these genes have been lost.

### ORFs of Unknown Function and Host Lethality

As noted above, the T4 genome is tightly packed with probable genes. Almost half of these still do not have an assigned function, but most have some or many of the characteristics of true T4 genes that encode known proteins. By convention among T4 researchers, each hypothetical or uncharacterized ORF is named sequentially in the clockwise direction by reference to the preceding known gene, as in “*xxxY.n*”. Therefore, *dexA.1* and *dexA.2* are the two ORFs following *dexA* on the map. This convention immediately locates each such ORF on the T4 map but implies nothing about its function. The only exceptions to this convention are in positions where an ORF follows a gene transcribed in the opposite direction or with very different timing. In those cases, the ORF may be rooted to the following gene on the map, but a minus sign is used (e.g., *uvsY.-1*, *rI.-1*).

Most of the 127 uncharacterized ORFs lie in regions transcribed counterclockwise from strong early promoters. Only 16 of the uncharacterized ORFs would be expressed late in the T4 infection cycle. These are (i) ORFs under control of a late promoter in the clockwise direction, where almost exclusively late genes are found (5.1, 5.3, 5.4); (ii) ORFs following late promoters (some of which also may still be expressed from upstream early and/or middle promoters) in the counterclockwise direction (*rI.1* and *rI.-1*; 24.2 and 24.3; *uvsY.-1* and *uvsY.-2*; *alt.-1* to *alt.-3*; and 30.9); and (iii) ORFs following middle promoters and without late promoters (*denB.1*)

Because they are likely to be expressed immediately after infection, some of the 127 uncharacterized T4 ORFs may be involved in the transition from host to phage metabolism or in resistance to plasmid- or prophage-encoded toxic proteins. Many of these genes (shown in white in Fig. 3) are in regions that can be deleted without seriously affecting phage production under usual laboratory conditions. However, at the same time, they have largely been retained in T4-related phages (534, 596, 919; E. Kutter et al., unpublished data about the *nrdC-tRNA* region). Most of the T4 early promoters are in these widely conserved yet deletable regions, which are densely packed with the predicted ORFs. Many of the hypothetical ORF proteins—at least those over about 9 kDa—have been identified on two-dimensional gels by comparing labeled proteins produced by wild-type and the T4 deletion strains (604). These proteins are often produced in large quantities just after infection. Those that have been tested are generally lethal or very deleterious to the growth of *E. coli*.

Together, these findings suggest that the host-lethal, immediate-early proteins confer selective advantage for the phage but that they are necessary only under certain environmental conditions, for infecting other hosts, or that there is redundancy in their functions. Some of the proteins are quite large, but most are smaller than 15 kDa. In general, work with T-even phages emphasizes that small hypothetical ORF-encoded proteins should not be overlooked. The smallest characterized T4 protein, Stp, consists of only 29 amino acids; 62 predicted T4 proteins have fewer than 100 amino acids.

Most of the unidentified ORFs show very little homology to non-phage genes in the databases. That many of these ORFs are deleterious to *E. coli* when cloned reinforces the notion that their products inhibit or redirect important host proteins and that they may be useful in studying cellular proteins in their active, functional state. One example, the Alc protein, specifically terminates the elongation of transcription on cytosine-containing DNA (599, 601). Alc appears to uniquely recognize the rapidly elongating form of the RNA polymerase (RNAP) complex. It would be a valuable tool for studying the dynamic structural changes that occur in the polymerase during transcription; all other current approaches only examine the polymerase paused at particular sites and infer its behavior from the resultant static state.

Some of the host-lethal proteins may also suggest new targets for antibiotics. They should also aid in studies of evolutionary relationships and protein-protein interactions.

Another interesting set of proteins involved in the transition from host to phage gene expression involves three different ADP-ribosyltransferases. These include Alt, which is packaged in the phage particle and carried into the cell with the DNA, ModA, and ModB. The role of these ADP-ribosylation activities in the T4 transcription cycle is detailed below.

To fully understand the takeover of host metabolism by T4-like phages, it will be necessary to identify the ORFs that indeed encode proteins in vivo and to determine their biological functions and the conditions under which they exert their effects. The sequences of some of the small proteins that have been studied are highly conserved among the T-even phages,

Transcription

T4
Promoters:
Early
Middle
Late

Replication & Recombination

origin initiation
single-strand invasion
Join-Copy
Pathway II
cut & paste
or cut & replicate
or cut & package
Single-Strand-Annealing
Pathway I
Join-Cut-Copy
Pathway III
replicate
cut & package
Packaging

FIG. 4. Diagram of the relationship between the T4 transcriptional pattern and the different mechanisms of DNA replication and recombination. The top panel shows the transcripts initiated from early, middle, and late promoters by sequentially modified host RNA polymerase. Hairpins in several early and middle transcripts inhibit the translation of the late genes present on these mRNAs. The bottom panel depicts the pathways of DNA replication and recombination detailed later in this review. Hatched lines represent strands of homologous regions of DNA, and arrows point to positions of endonuclease cuts. Reprinted from reference 769 with permission from the publisher.

presumably reflecting their complex interactions with multiple cell components.

PROMOTERS AND TRANSCRIPTION FUNCTIONS

T4 transcription uses three major classes of promoters—early (Pe), middle (Pm), and late (Pl)—which broadly define the developmental stages of the T4 infection cycle (Fig. 4). The genomic positions of these promoters and of rho-independent terminators are indicated in Table 1. The overall temporal pattern of transcription through the T4 genome is quite complex. Many genes are served by multiple classes of promoters, and so a number of promoters may precede genes and a terminator in a transcription unit. Furthermore, protein-dependent or cotranslation-dependent antitermination contributes to the pattern of active T4 transcripts. Some RNA processing and superimposed translational controls (discussed later) also complicate the interpretation of data.

T-even phages rely entirely on the host core RNAP throughout infection. It is therefore not surprising that T4 promoter specificity and transcription are affected by the multiple interactions of the bacterial RNAP α subunits, β/β′ subunits, and σ⁷⁰ promoter recognition subunit. Most studies with T4 have been done in cells growing exponentially under high aeration, where the host σ⁷⁰ is present throughout infection. Under these conditions, the temporal transition through the different classes of promoters is accompanied by covalent modifications of RNAP and the appearance of new protein transcription factors that act in various ways. All of these functions serve to enhance phage promoter recognition and transcription; no DNA-binding transcriptional repressor protein has been identified in the T4 developmental cycle.

To date, little is known about T4 infection under stationary-phase or anaerobic conditions (such as the phage would encounter in nature [599a]). Preliminary evidence shows that the patterns of infection under these conditions are often very different and that the status of *rpoS* clearly makes a difference in the outcome of aerobic infection in stationary-phase cells (E. Kutter, unpublished data). Corbin et al. (187a) have recently shown that T4 infection affects the morphology of *E. coli* biofilms and that glucose-limited biofilm cells can be a reservoir for phage. Additional study of T4 gene expression under different environmental conditions is warranted.

Early Transcription

At the onset of infection, 39 T4 early promoters (plus a few host-like promoters [see below]) compete with about 650 σ⁷⁰-dependent bacterial promoters for approximately 2,000 RNAP holoenzymes in the commonly studied, rapidly growing exponential cells; the polymerase number is smaller under more limiting growth conditions. T4 redirects the transcriptional machinery to T4 promoters with high efficiency, as reflected by the appearance of phage-specific proteins soon after infection, the rapid shutoff of host gene expression (reviewed in reference 599), and, ultimately, the virulence of the phage. That T4 early promoters are stronger than *E. coli* promoters presumably plays a major role, since most promoters can be cloned only on

A. Early
-35

B. Middle
-30

C. Late
-10

FIG. 5. Logo of T4 promoters. Nearly all the sequences in each alignment have promoter activity, as demonstrated by primer extension, transcription from cloned DNA fragments, or RNA hybridization assays. The promoters included whose start sites have not been mapped all precede a corresponding early, middle, or late gene and show significant similarity to the relevant promoter class. Sequences were independently aligned in the –10, –30, or –35 region. The information content (R<sub>s</sub>) is calculated in “bits” and is the sum of the R<sub>s</sub> for each region (except for the late logo, which was calculated from the single alignment at –10). Alignments, logos and R<sub>s</sub> values were obtained as described previously (966; E. Miller, T. Dean, and T. Schneider, unpublished data). The triangle marks the +1 transcription start site. (A) 39 early promoters, R<sub>s</sub> = 38.3 bits; (B) 30 middle promoters, R<sub>s</sub> = 21.1 bits; (C) 50 late promoters, R<sub>s</sub> = 16.2 bits.

plasmids designed to attenuate their transcriptional activity. Transcription start sites of many of the early promoters have been mapped by primer extension off of mRNA from T4-infected cells and/or from promoter-cloning vectors (reviewed in reference 1169).

The 39 characterized Pe sequences (1168, 1169) are noted in Table 1 and have been analyzed using the information content software developed by Schneider and Stephens (966). The sequence logos, maximizing the alignment at the –10 region and, independently, at the –35 region, are shown in Fig. 5A (E. Miller, T. Dean, and T. Schneider, unpublished data). The analyses show that there is high conservation at the –12, –11, and –7 positions similar to that in the E. coli Eσ<sup>70</sup> promoters. However, T4 Pe sequences have more extended –10 regions, with sequence conservation extending through the G predominating at –14 to –18. In one group of early promoters, significant conservation extends on both sides of the –10 region [5′-GTGG(TAT/CT/AAT)ACAAC-3′] up to the T at posi-

for the $-10$, $-35$, and A-tract regions displayed for the early promoters in Fig. 5A is 38.3 bits, which is substantially greater than the 17.6 bits (as calculated by Liebig and Rüger [639]) required to select the Pe promoters from a genome of known length and base composition ($R_{\text{frequency}}$ [see reference 966 for a thorough description of logos and information theory as applied to DNA-binding proteins]). The $R_s/R_f$ ratio of >2 for these values suggests that the twofold excess information in the aligned T4 early promoters is due to both unmodified host RNA polymerase and its ADP-ribosylated counterpart (see below) binding and initiating transcription in these regions. The refinement of the analysis of the T4 early promoters by means of information theory is in progress (Miller et al., unpublished). Together, features of T4 early promoters allow them to be distinguished from the host promoters and elevate their transcriptional activity to a level that often exceeds that of the strongest *E. coli* promoters.

In addition to these early T4 promoters, there are some promoters that more closely resemble *E. coli* promoters. P bac (639, 1169) has been identified by mapping transcripts from cells carrying plasmid-borne T4 genes. It directs the synthesis of transcripts that are complementary to gene 3 mRNA. P repE(coordinate 79405 [Table 1]) has been identified in T4-infected cells (1109). It directs the synthesis of RepEA and RepEB proteins and an RNA primer for oriE-initiated replication. This RNA would be complementary to late-gene 5 transcripts but is undetectable by the time when these transcripts are made. Transcripts preceding gene 32 (142) have been detected that also map to $\sigma^{70}$-like promoters. While the later are active on supercoiled plasmids, little to no transcription was observed in T4-infected cells. A similar promoter preceding gene 57A was inferred to be active on plasmids (409). These host-like promoters as a group may be of limited significance, when host transcription in general is turned off and RNAP is modified early during T4 infection.

T4 modifies the host RNAP in several ways after infection. However, most of these modifications are not essential to the infection process. A 70-kDa protein, gpAlt, enters the host with the infecting DNA. Alt is a mono-ADP-ribosyltransferase that targets arginine residues. It efficiently ADP-ribosylates one of the $\alpha$ subunits of RNAP in the carboxy-terminal domain at position Arg265 (323, 324, 435, 459, 937a, 1011) and ADP-ribosylates the three other polymerase subunits to a lesser extent, along with a number of other uncharacterized polypeptides. ADP-ribosylation of RNAP by cloned Alt protein leads to enhanced transcription from cloned T4 early promoters (544). Mutation analyses reveal that T4 early promoters interact strongly with unmodified RNAP and even better, in most cases, with RNAP in which only one of the $\alpha$ subunits is ADP-ribosylated. In particular, base position $-33$ of the T4 promoter and the A-rich UP element at position $-42$ contribute to the strong interactions with ADP-ribosylated RNAP of T4-infected cells (1026). Therefore, Alt presumably contributes to the preferential transcription from T4 promoters after infection (1168, 1170).

Shortly after infection, two new ADP-ribosyltransferases are expressed, ModA (23 kDa) and ModB (24 kDa) (780, 1077). ModA, first observed by Skorko et al. (1011), ADP-ribosylates the $\alpha$ subunits of host RNAP but shows no activity toward the $\beta$, $\beta'$, and $\sigma$ subunits. Like Alt, ModA ADP-ribosylates Arg265

on the $\alpha$ subunit; unlike Alt, it targets both $\alpha$ subunits, not just one. ADP-ribosylation replaces the positive charge of the Arg residue by two negative charges carried by the two phosphate groups and affects DNA-protein as well as protein-protein interactions. This second ADP-ribosylation inhibits transcription from promoters with the UP element; expression of cloned $modA$ is highly lethal to the host. (The action of ModB [205] is summarized below.)

### Middle Transcription

Thirty T4 middle promoters (Pm) were compiled and are presented in the sequence logo of Fig. 5B. All of the middle promoters used in the logo have been mapped with respect to the $5'$ end of the transcript and shown to be dependent on the transcriptional activator protein MotA (reviewed in references 692, 987, and 1043). There appears to be little dependence on an A at the +1 start site of the middle transcripts (coordinate 0 in Fig. 5B). The conserved $-10$ region resembles that of the Pe sequences ($5'$-TATAAT-$3'$ is most common), with $-12$, $-11$, $-8$, and $-7$ having nearly the same base composition. As seen for T4 early promoters, sequence conservation extends into the traditional spacer region of E$\sigma^{70}$ promoters, up to position $-16$. Significantly though, T4 Pm sequences have neither the well-characterized E$\sigma^{70}$ $-35$ region nor the Pe $-35$ region. Middle promoters are characterized by a specific $-30$ sequence called the Mot box, which extends between $-32$ and $-27$, with GCTT being the most highly conserved. The information content ($R_s$) calculated for the optimally aligned regions from $-60$ to $+10$ of the logo in Fig. 5B is 21.1 bits, with 13.1 bits of the information being associated with the $-10$ alignment. This is considerably less than the 38-bit $R_s$ value of the Pe promoters, implying that there is less competition with host promoters for RNAP, perhaps because host DNA is already being degraded and ADP-ribosylation of RNAP is completed. Approximately 8 bits of $R_s$ information are required for MotA to recognize the MotA box sequence. T4 middle promoters are all located on the minus strand (Table 1) relative to the GenBank genome entry. Fourteen new middle promoters have been recently described (1095a; R. Nivinskas, personal communication).

T4 gene products AsiA and MotA are required for middle-mode transcription. AsiA is an anti-$\sigma$ factor protein (see reference 454a for a review of anti-$\sigma$ proteins) that coactivates RNAP for middle-mode transcription initiation by the formation of AsiA-$\sigma^{70}$ heterodimers (12, 180, 1104). This interaction interferes with the recognition of $-35$ promoter sequences and at the same time stimulates T4 middle-mode transcription (180, 425, 1103, 1104). The AsiA-$\sigma^{70}$ interaction is regarded as the pivotal event in the transition between T4 early and middle transcription: in vitro it both inhibits the recognition of most host promoters and early T4 promoters and stimulates T4 middle-mode transcription (180, 425, 848, 849, 1104). However, in vivo, defective $asiA$ mutants do not prolong early transcription (858), suggesting that other proteins (i.e., ModA and ModB) turn off most early T4 promoters. MotA is a DNA-binding transcriptional activator protein that binds to the MotA box sequence (Fig. 5B) through its C-terminal domain, facilitating Pm promoter recognition and transcriptional activation (see the model proposed in reference 987 and in Fig. 4

of that reference). MotA and AsiA together increase the initial recruitment of RNA polymerase to T4 middle promoters and facilitate the clearance of RNAP from the promoter and into the elongation mode (419).

# Late Transcription

Late transcription is responsible for the synthesis of T4 head, tail, and fiber proteins, in addition to the several virion assembly factors (1173) and recombination genes required for T4 late recombination/replication (784) (see below). Fifty late promoters (Pl) have been compiled and aligned for the Pl sequence logo shown in Fig. 5C. There is only a slight bias toward purines at the +1 transcription start site, while there is extensive conservation of the –10 sequence TATAAAATA from –13 to –6. This sequence alone contributes the major information content for late promoters, which have an \(R_s\) value (see the definition above) of 16.2 bits. There is no –35 or MotA-like –30 sequence in T4 late promoters. T4 encodes one of the smallest known sigma factors, gp55 or \(\sigma^{55}\), for RNAP recognition of late promoters (1173). It specifically recognizes the –10 region sequence. Although \(\sigma^{55}\) is required to selectively initiate transcription at T4 late promoters, it is not sufficient. AsiA does not appear to be a major determinant of middle- versus late-promoter competition (552). Instead, another phage-encoded protein, gp33, acts as a coactivator of late transcription, mediating interactions between \(\sigma^{55}\) and the sliding clamp encoded by T4 gene 45. The trimeric gp45 protein is a key component in the processivity of the DNA replication complex and is also essential for late transcription (a “mobile enhancer” [405, 1186]). Primer-template junctions and single-stranded DNA (ssDNA) nicks are the most efficient loading sites for gp45, which is loaded by the clamp-loader proteins gp44 and gp62; gp45 slides on the DNA, enhancing the opening of late promoters more than 1,000 bp away from the loading site. Activated late promoters outcompete middle promoters on the same plasmid in vitro, especially at higher ionic strengths. This advantage is enhanced by ADP-ribosylation of RNAP \(\alpha\) subunits and by binding of the phage-encoded RpbA protein to the RNAP core (552, 1082, 1173). DsbA protein is thought to also affect transcription from some late promoters (995), although it is not essential (1114).

At least three T4 proteins—Mrh, Srd, and Srh—are implicated in the interactions of different host sigma factors with core RNAP (781). Under heat shock conditions, the host \(\sigma^{32}\) (RpoH) competes with other sigma factors for host core RNAP (354, 482). The products of the two nonessential genes *mrh* and *srh* together modulate the phosphorylation of \(\sigma^{32}\) using ATP (781; Mosig, unpublished). Presumably, this would be most important for T4 late transcription, since T4 \(\sigma^{55}\) is one of the weakest known sigma factors. Consistent with this idea, infection with wild-type T4 of one specific host *rpoH* mutant (but not others) is aborted at the onset of late transcription, unless the T4 *mrh* gene is deleted (290). Srh protein resembles a segment of \(\sigma^{32}\) that interacts with RNAP, suggesting that it acts as a decoy. Similarly, T4 Srd protein resembles an RNAP-interacting segment of \(\sigma^{70}\) and \(\sigma^{38}\) (RpoS; stationary-phase and oxidative stress sigma factor) and would also decoy RNAP from the host promoters. Expression of *srd* from a clone is lethal to *E. coli*.

## Microarray Analysis of T4 Transcription

In a recent report (672), the expression profile of the entire T4 genome was evaluated by mRNA hybridization microarray analysis. RNA samples were obtained from 0 to 25 min during a T4 infection cycle at 30°C. Gene expression patterns were then evaluated by cluster analysis. Early-, middle-, and late-gene clusters were clearly identified and were in striking agreement with the extensive literature for individual T4 genes. Exceptions were in regions yielding overlapping transcripts from different promoters, where temporal assignments would be more difficult. Of particular note was the complete absence of late-gene expression prior to 15 min, with the near cessation of all early- and middle-gene transcription following onset of the late period. The analysis, as stated by the authors, not only confirms the extensive literature on T4 but also suggests that microarray-based expression profiling will be a valuable tool in determining the transcription pattern, and ultimately the function, of the hypothetical and uncharacterized T4 genes. Similar strategies will be invaluable for future studies of other phage and viral genomes.

## Transcription Termination and Predicted RNA Structures

### Intrinsic transcription terminators. Intrinsic, Rho-independent transcription termination sites are characterized by an intramolecular RNA helix (stem-loop or hairpin) in the mRNA, followed by a U-rich sequence (33, 364, 929, 1209). These features were used in the computer programs TransTerm, GCG Terminator, and FindPatterns (211, 265) to predict probable Rho-independent terminators in the T4 genome (E. Miller, unpublished data). About 15 years ago, 4-nucleotide UUCG loop sequences were characterized in T4 as conferring exceptional stability to RNA secondary structures (1100). Following that initial report, other stabilizing RNA tetraloop sequences were described (412, 1183), and their prevalences in *E. coli* Rho-independent terminators were later compiled (201). Identification of T4 transcription terminators was enhanced using pattern searches for the prominent tetraloop sequences (e.g., UUCG and GNRA), which to date are not included in the TransTerm or Terminator search parameters. In some cases, the predicted RNA structures may act to stabilize mRNA against degradation rather than functioning directly in termination (142, 340).

Features of the predicted intrinsic transcription terminators in the T4 genome are summarized in Table 4, and their genome positions are noted in Table 1. Overall, 34 terminators were located between genes or at the 3′ end of an ORF; 24 of these are predicted to be on early transcripts (therefore, their sequence corresponds to the minus strand of the T4 GenBank entry), while 10 are on late transcripts. The predominant tetraloop sequence is UUCG, found in 18 of these terminators, while 3 are GAAA and 3 are GCAA. All are about equally present on early and late transcripts. The remaining 10 transcription terminators have noncanonical 4-nucleotide loop sequences or have 3-, 5- or 6-base loop regions. Their features and locations suggest that they, too, are probably functional.

Many of the probable terminators are located at the ends of long early or middle transcripts, preceding a downstream

TABLE 4. Intrinsic terminators mapped or predicted on the T4 genome

| 5'gene       | 3'gene   | Strand | Start$^a$ | Stem-loop-stem-poly(U)$^b$                                                                                   | Tetraloop | Identifier$^b$ |
|--------------|----------|--------|-----------|----------------------------------------------------------------------------------------------------------------|-----------|----------------|
| Intergenic locations |        |        |           |                                                                                                                |           |                |
| 39.1         | 39       | –      | 5384      | GGCCUUCGGGCC TTTAGCTTTAT                                                                                     | UUCG      | TT, Term       |
| soc          | mrh.2    | –      | 15305     | GGACTCCTTCG GGAGTCC TTTTTTCATTT                                                                               | UUCG      | TT, Term       |
| 43           | imm.1    | –      | 27183     | GGACCTCCA GGTCC CTTTTT                                                                                       | UCCA      | Term           |
| regA         | 43       | –      | 29967     | GGGGCTTCG GCCCC TTATTT                                                                                       | UUCG      | TT, FP         |
| 45           | 44       | –      | 31912     | GGGCTTCG GCCC TTTATAATTT                                                                                      | UUCG      | TT, FP         |
| a-gt         | 47       | –      | 36622     | GGGCTTCG GCCC TTTAGCTTT                                                                                       | UUCG      | TT, FP         |
| a-gt.2       | mobB     | –      | 38731     | GGAGCTTCG GCTCC TATATTGCTTTATAAATTTTT                                                                          | UUCG      | FP             |
| 55.3         | 55.2     | –      | 40836     | GGGCTTCG GCCT TTTT                                                                                           | UUCG      | FP             |
| nrdH         | 55.6     | –      | 41800     | GGCCCCAGA GGGCC CGTCTTAATCTTCT                                                                                | None      | TT             |
| pin          | 49       | –      | 46884     | CCCTTACCT TAAAT AGATAAGGG TATTTATTATTTTT                                                                       | None      | TT             |
| nrdC.11      | nrdC.10  | –      | 55432     | GGGAGCCTTCG GGCTCCC TTTTTTATTT                                                                                | UUCG      | TT, Term, FP    |
| rI.-1        | mobD.5   | –      | 58813     | GTCTCCTTCG GGAGAC TTTTTTTCATTT                                                                                | UUCG      | TT, Term, FP    |
| vs           | tk.4     | –      | 61369     | GGGCGATATG CGCCC TTTT                                                                                         | None      | TT, Term       |
| e.6          | e.5      | –      | 69306     | GGGGCTTCG GCCCC TATTACTT                                                                                      | UUCG      | Term, FP       |
| RNAC         | e.8      | –      | 70856     | GCCCCCCGACC GAAA GGTTGGGGC TTTTT                                                                              | GAAA      | TT, Term       |
| 8            | 9        | +      | 87161     | GGGAGCCCATG GGCTCCC TTTTTCTTT                                                                                 | CAUG      | TT, Term       |
| wac          | 13       | +      | 93596     | GGGGCCGCAA GGCCCC AAAGGATTTT                                                                                  | GCAA      | FP             |
| 19           | 20       | +      | 101131    | GGGGA GAAA TCCCC ATCCTGCTT                                                                                    | GAAA      | Term           |
| 23           | segD     | +      | 106537    | GGGAACCTTCG GGTTCCC TTTTTTCTATTTTT                                                                            | UUCG      | Term, FP       |
| 24           | rnlB     | +      | 108613    | GGGACCTTTC GGTCCTT TTTTTATTT                                                                                  | UUUC      | TT, Term       |
| 24           | rnlB     | +      | 108668    | GTACATCTTGTAC CATTTT                                                                                          | None      | TT             |
| hoc          | 24.3     | –      | 110180    | GGGGCTTCG GCCCC TTTCTTCATTTTT                                                                                 | UUCG      | TT, Term       |
| uvsY.-2      | uvsWend  | –      | 114472    | GGCCCTTCC TTTTT GGTTGGGCT TTTTAAT                                                                             | None      | TT, Term       |
| 54           | alt.-3end| +      | 122720    | TGGGGACC GAAA GGTCCATA TTTTTATTT                                                                              | GAAA      | Term           |
| alt.1        | alt      | –      | 125558    | GGCCUUCG GGCC TTTAATTTTAT                                                                                     | UUCG      | TT, Term       |
| 30.9         | 30.8     | –      | 130358    | GGACTCCTTCG GGAGTCC TTTTTTATTTTT                                                                              | UUCG      | TT, Term       |
| 30.9         | 30.8     | –      | 130402    | CGAGATGATG CTTCTCG TTTT                                                                                        | None      | TT             |
| nrdB         | denA     | –      | 137950    | TGGGCCGCAA GGCCCA TTTTATTAT                                                                                   | GCAA      | FP             |
| nrdA         | mobE     | –      | 140384    | TTCCCGAGCTCAG GCTCGGGAA CCTTTAT                                                                               | UCAG      | Term           |
| 32           | frd.3    | –      | 146925    | GGGACCTAGA GGTCCC TTTTTTATTTTT                                                                                | None      | TT, Term       |
| 35           | 36       | +      | 155811    | GGGACCC TTCG GGTTCCC TTTTTCTTT                                                                                | UUCG      | TT, Term, FP    |
| 37           | 38       | +      | 159628    | GGGGCTTCG GCCC TTCT                                                                                           | UUCG      | FP             |
| t            | asiAend  | +      | 160924    | CCCTCGTTGAA TTCG TCGATGAGGG TTTTCTTTATCTTCTTT                                                                  | UUCG      | FP, Term       |
| motA.1       | motA     | –      | 163724    | GGGAGAGCCGAG GCTCTCCC TTTTTTATTTTT                                                                            | CGAG      | TT, Term       |
| ac           | 52.1     | –      | 165332    | GGGCTATTCAT TAGCCC TTGCTGCTTTTATT                                                                             | None      | Term           |
| denB.1       | denB     | –      | 167736    | GGGCUUCG GCCT TTTGTTTT                                                                                       | UUCG      | Term, FP       |

Peculiar locations or nonpoly(U) 3' ends

| 5'gene       | 3'gene   | Strand | Start$^a$ | Stem-loop-stem-poly(U)$^b$                                                                                   | Tetraloop | Identifier$^b$ |
|--------------|----------|--------|-----------|----------------------------------------------------------------------------------------------------------------|-----------|----------------|
| 56/segF      | segF     | –      | 16233     | GACGCC GAAA GGCGTC TCTTTT                                                                                     | GAAA      | FP             |
| uvsX-40      | 40       | –      | 22347     | CCCCCTC TTCG GAGGGGG AAGAAGAAAAGAAAAAGAA                                                                      | UUCG      | FP             |
| in5.4        | ntr.$^c$ | –      | 80949     | GCCATGTG TTT CATACGGC TTTTTAATTT                                                                              | None      | Term           |
| 5.4/6'       | 6        | +      | 81769     | TTGATG GAAA CACTGAA TTCTATTTTT                                                                                | GAAA      | FP             |
| 34/35'       | 35       | +      | 154701    | GGCCGAGTTTGGACA AGGATA TGTCCAAAACGCC ATTITT                                                                    | None      | Term           |
| instp        | ntr.     | +      | 165510    | GGCGTC CGAA GACGCC TTTAGTTTT                                                                                   | CGAA      | Term           |
| rIIB         | denB.1   | –      | 167967    | TAAGGC TTCG GCCCTTA ACTAAGGAAAAATTATGTT                                                                        | UUCG      | FP             |

$a$ Numbered 5'start is the first base of the RNA helix, and underlined characters are nonpaired bases.

$b$ Programs that identify the terminator: FP, GCG FindPattern; Term, GCG Terminator; TT = TIGR TransTerm.

$c$ ntr., nontranscribed strand.

Seven regions were identified by the programs described in the preceding section as possible transcription termination sites, although they showed unusual attributes with respect to their location and the 3′ U-rich region. Some are located wholly within coding regions (e.g., position 81769).

Overall, the predicted T4 intrinsic terminators generally appear to both define the 3′ ends of multicistronic mRNAs and affect the dynamics of transcription complexes advancing on opposing DNA strands.

Rho-dependent transcription terminators. In enteric bacteria, the RNA-binding protein Rho modulates transcription termination at sites that are distinguished from intrinsic terminators by the absence of both the stable RNA hairpin and the 3′ U-rich region. Rho utilization sequences (*rut*) in RNA generally are C-rich, have small amounts of G, and can be as long as 85 nucleotides (929). In addition, *rut* sites can be 150 to 200 bp 5′ of the actual transcription termination site and therefore appear to function as locations for entry of Rho on transcribed RNA. Some of the better studied Rho-dependent termination sites (i.e., lambda *tR1* and *E. coli tnp*) are regulated by antitermination which also involves host Nus proteins, lambda N protein, and the RNA sequence of the *boxA* and *boxB* regions (344, 553, 929). Together, these complex features have made computational methods for identifying Rho-dependent termination sites problematic relative to the easily defined intrinsic terminators.

Rho-dependent transcription termination sites in T4 have not been extensively characterized; little additional work has been done since the review by Stitt and Hinton (1043). One of the better candidate Rho terminators, or a 3′ end of the RNA that is indirectly influenced by a *rho* mutation, lies between genes *uvsX* and *40* (416). Readthrough transcription from *uvsX* into *40* (and on through the helicase gene *41*) is diminished by the Rho mutant *rho026* (1044). In addition, the low level of readthrough transcripts is elevated in *goF* (*comCa*) mutants, probably by better protection against RNases (416, 1043). The Rop protein of ColE1-derived plasmids has a stabilizing effect similar to that of *goF* mutations (1028). As mentioned above, the *uvsX-40* site (position 22347) is characterized by a stable tetraloop hairpin that is not followed by the typical U-rich sequence (Table 4). However, the *rut*-like C-rich region is part of a hairpin, which is not characteristic of other *rut* sites, and there is not an apparent nearby *boxA* sequence. Nonetheless, the available evidence points to this region as a likely Rho-dependent termination region. Similar properties are predicted for the putative *rIIb-denB.l* terminator at position 167967. These RNA structures may help direct Rho-dependent termination.

Other sites in the T4 genome that have *rut-* and *boxA*-like sequences, and that therefore may be affected by Rho, occur at the end of the *tRNA* cluster (after RNA C at position 70742), in the region between genes *repEB* and *repEA* (position 78810), and between the late promoter at position 77490 and gene 5. The last two potential sites are near the *oriE* origin of DNA replication (1109; A. Harvey, R. Vaiskunaite, and G. Mosis, unpublished data) (see below). Other *rut-* and *boxA*-like sequences can be identified in the T4 genome, but the significance of these, as well as the entire aspect of Rho-dependent termination in the T4 developmental cycle, requires further study. Mutations in the gene *goF* (*comCa*) have been repeatedly isolated as suppressors of host mutations that affect T4 transcription termination; the GoF protein, which stabilizes residual long transcripts produced in the Rho026 mutant host, does not show overall similarity to other proteins in the genome databases (171, 956, 1043). However, the short acidic region between residues 87 and 111 is similar to amino acids in other RNA-binding proteins and ATP-dependent RNA helicases (Miller, unpublished).

# TRANSLATION AND POSTTRANSCRIPTIONAL CONTROL

The transition from host to phage protein synthesis is a rapid and efficient process (601); virtually no host proteins are observed on two-dimensional gels of proteins labeled after 1 min of T4 infection (189). Intrinsic properties of T4 mRNAs, such as the strength of SD sequences, several T4-induced modifications to the translation initiation apparatus, and the translational coupling arrangement seen for many phage genes may play key roles in the shift of host ribosomes to translation of T4 mRNAs.

## Ribosome-Binding Sites

In general, T4 RBS have properties that are nearly identical to those of its *E. coli* host (reviewed in reference 736). mRNA sequences 5′ of the initiation codon (the SD sequence) show a variable extent of complementarity to the 3′ end of 16S rRNA, followed by a spacing of 6 to 10 nucleotides and then the initiation codon. Furthermore, there is a modest bias in favor of certain codons for the second amino acid. Many T4 proteins have been purified for biochemical or structural characterization, so that their N-terminal residue and hence their translational start codon are definitively known. Where the N-terminal amino acid has not been experimentally determined, the translation initiation sites were assigned to each gene and ORF (Table 1) using predictions based on the correlation coefficient (described above), the T4 hidden Markov model (671), and the presence of an SD sequence in an appropriate position. Most of the translation start codons of T4 genes are AUG. GUG as initiator occurs at eight T4 ORFs which, at 3%, is similar to the frequency of GUG starts occurring in *E. coli* genes (92). T4 genes and ORFs using GUG initiation codons include *mobB*, *dexA.2, 46, 46.1, cd.1, 55.7, 41*, and *49′*. One occurrence of an AUU initiation codon has been documented; it is an internal start site within gene 26 (823) (see below).

Aligned T4 RBS sequences can be collectively viewed in a sequence logo (966), although the variable spacing between the SD sequence and the AUG initiation codon presents a particular challenge. Figure 6A shows the logo aligned at the AUG. Due to the variable spacing between the SD sequence and initiation codon, only a minor peak for the SD is observed, in the −8 to −9 region. Alignment of the SD sequence alone, independent of the AUG (Fig. 6B), clearly illustrates the importance of the SD sequence. The *R<sub>s</sub>* (defined above) of T4 RBS sequences, using the optimally aligned regions from −15 to +14 (Fig. 6) is 14.3 bits, which is higher than the calculated *R<sub>s</sub>* for *E. coli* RBS sequences (8.9 bits [994]). However, a refined “flexible” model of *E. coli* RBS appears to more accurately account for the variable spacing between the SD se-

VOL. 67, 2003                                                                 BACTERIOPHAGE T4 GENOME 113

A. Aligned at Shine-Dalgarno
![Image A](image_a.png)

B. Aligned at initiation codon
![Image B](image_b.png)

FIG. 6. Logo of T4 RBS. Translation initiation regions of the annotated T4 GenBank file AF158101 were used; genes 25 and 38, which have extended spacing and RNA hairpins between the AUG and SD region, and gene \(26^{\prime}\) were excluded. (A) Genes aligned at the initiator AUG or GUG codon. Information content analysis \((R_{s}\), in “bits”), from positions 0 to +14, yields an \(R_{s}=7.5\) bits. The variable spacing between the AUG and the SD region yields a reduced contribution of the SD region to the total \(R_{s}\) in the logo. This is seen by the low shoulder of purine-rich nucleotides in the logo from −11 to −6. (B) Genes aligned at the SD region. The region from −20 to −1 (relative to the 0 position in panel A) was independently aligned to achieve the highest \(R_{s}\) value in the SD region. In the region from −15 to −1, \(R_{s}=6.8\) bits. Over the entire RBS, spanning −15 to +14, the sum of \(R_{s}=14.3\) bits. Shultzaberger et al. (994) describe an alternative approach to modeling RBS \(R_{s}\) values that accounts for the variable spacing between the SD and initiator codon. Logos were created (Miller et al., unpublished) and alignments and \(R_{s}\) values were calculated as described previously (965, 966, 994).

sequence and AUG (994). In effect, subtracting the uncertainty of the variable SD-AUG spacing lowers the total \(R_{s}\); thus, the 14.3-bit \(R_{s}\) value currently calculated for T4 ribosome binding sites is likely to be slightly lower (Miller et al., unpublished). Overall, the strength of the T4 RBS would in part account for the observed redirection of ribosomes from host to phage mRNAs.

A few prokaryotic “leaderless” mRNAs have been identified that lack the SD sequence and have the initiator AUG positioned right at the \(5^{\prime}\) end of the transcript; the best-studied phage leaderless mRNA is that for the lambda cI repressor protein (833). Some leaderless mRNAs are highly expressed (e.g., *aph* [353, 480]). To date, no leaderless mRNAs have been characterized from T4.

The transition from host to phage protein synthesis may also involve changes that T4 reportedly makes in proteins of the translation apparatus, including IF3 alteration, release of S1 from ribosomes, and synthesis of new ribosome-binding proteins (601, 1166). These modifications to the translation initiation apparatus potentially could have major effects on the initiation efficiency of either phage or host mRNAs. Unfortunately, most of the genes responsible for these changes have not been identified. ModB ADP-ribosylates the S1 protein, elongation factor EF-TU, and the chaperone “trigger factor” (205), and thus these changes may be important for diminished translation of host mRNAs or may have a direct impact on the translation of phage mRNAs.

RNA Structure at Ribosome Binding Sites

Two T4 RBS have unusually long spacing between the SD sequence and the initiation codon, with an additional RNA helix stacked into the RNA structure of the initiating ribosome-mRNA complex. For gene 38 mRNA, an RNA helix (hairpin) in the variable SD-to-AUG spacing region brings the SD sequence from 22 bases away to within 5 bases of the AUG, which is in the range of spacing observed for other T4 genes (326, 388). Gene 25 has an SD-to-AUG spacing of 27 bases, but an intervening RNA structure reduces that to 11 bases

(819). These compact intramolecular mRNA and intermolecular rRNA-mRNA helices at the RBS are reminiscent of RNA pseudoknots (428) (the regulatory RNA pseudoknot preceding the RBS of gene 32 mRNA is discussed below). T4 gene 38 and gene 25 mRNAs are good examples of how RNA structure can enhance translation initiation efficiency.

RNA structures can also have the opposite effect. Several T4 mRNAs fold into intramolecular RNA helices that inhibit ribosome binding and translation (736). Usually this is observed with mRNAs that are transcribed from early promoters and extend downstream into a late gene. The longer early transcript forms an RNA helix that sequesters the late-gene RBS (such as in the mRNAs for genes *e*, *soc*, I-TevI, and 49). Late promoters, located immediately upstream of the late-gene RBS, lack the \(5^{\prime}\) region of the helix and present RBS sequences that are accessible for translation initiation. As mentioned below, for gene 49, the intramolecular helix at the first RBS promotes use of the internal RBS for gp49'.

Internal Initiation Sites

A few overlapping or internal reading frames have been identified in the T4 genome. In each case, the internal translation initiation sites yield proteins shortened from the amino-terminal end. T4 EndoVII (the Holliday junction resolvase), encoded by gene 49, is 157 amino acids (aa) long. An internal initiation site, utilizing a GUG start codon, yields a protein of 105 aa (39). The shorter protein is synthesized predominantly from a long early transcript in which the first RBS is sequestered in a hairpin. The larger protein is synthesized from a shorter late transcript, in which the RBS is free. The full-length T4 gene 17 product (terminase/DNA-binding protein) is 610 aa. Internal initiation sites on two shorter gene 17 mRNAs (one is initiated from an internal promoter, and the other is cleaved) yield smaller proteins of 523, 505, and 416 residues (286). Because only the largest one contains a single-stranded DNA-binding domain and the second largest one suffices to package DNA of mature size, it has been proposed that the

different-sized proteins recognize different substrate DNA for recombination (288) and for packaging (784) (see below).

The rare AUU initiation codon used by gene 26′ yields a protein, initiated at codon 114, that is only 95 residues long compared to the full-length gp26, which is 208 residues long (823). The function of gp26′ is unknown.

ORF 30.3′ is the one example in T4 of a coding region that is translated in the +1 reading frame entirely within another gene (30.3). Translation of the two overlapping ORFs has been confirmed, with the internal RBS of 30.3′ resembling other T4 RBS sequences (1234).

## Translational Coupling

In translational coupling, the translation initiation of a distal gene is dependent on the translation of the gene immediately upstream. The process, which has been appreciated for many years (709, 808, 846), facilitates the coordinate expression of proteins that are involved in the same metabolic pathway or that assemble into multimeric complexes. In compact, densely coding phage genomes, translationally coupled gene arrangements are commonplace, although few have been explicitly studied. Translational coupling has been examined in RNA phages (638) and ssDNA Ff phage (1230). The very first intimations of translational coupling in T4 were observed by Stahl et al. (1035). It has been specifically studied in the T4 DNA polymerase clamp loader proteins encoded by genes 44 and 62 (502, 1089, 1095); in this complex, the 44 and 62 proteins occur in a 4:1 ratio. It appears that translational coupling helps determine the relative levels of each subunit, since the frequency of translation initiation of gene 62, transmitted from the upstream translation of gene 44, was measured to be about 25% (1089). These and other genes inferred to be translationally coupled have the stop codon of the upstream reading frame close to, or even overlapping, the downstream initiation codon. In the T4 genome, there are 52 clusters of genes arranged in this fashion. Thirty-five involve only two genes. Groups with the largest number of such genes are wac-9 (five genes), cd.2-31.1 (five genes), vs-tk (six genes), and the 30.6-alt.1 region (eight genes). Many of these include ORFs of unknown function, although the translational configuration would suggest a functional relationship to the adjacent, often characterized, gene. The extent, mechanisms and significance of translational coupling in phage T4 clearly deserve further attention.

specificity is determined by a smaller helical hairpin upstream of the RBS; binding does extend to the RBS and thereby represses translation initiation (857). T4 gp43 was the first protein used in developing the in vitro selection method (SELEX) for identifying high-affinity RNA-binding sites (1101). RegA binds and translationally represses more than a dozen T4 early mRNAs, but it does so with weaker affinity than that observed for gp32 and gp43, and the binding site is not well defined (reviewed in reference 736). One of the better T4 RegA-binding sites (\(K_d = 0.2 \, \mu M\)) overlaps the clamp loader gp44 RBS (338). This site is at an upstream RBS, with regA positioned as the most distal gene on the transcript. The configuration suggests that repression is transmitted through translational coupling to gene 62 and regA itself. SELEX-isolated binding sites did not precisely match any specific T4 site, but the consensus sequence (5′-AAAAUUUGUUAAUGUAA-3′) resembles many of the RegA-sensitive RBS (113). Interestingly, RegA is conserved in all T-even-related phages examined, although it is nonessential under laboratory growth conditions. Its RNA-binding domain appears to be a unique helix-loop groove structure (338, 484, 975). Structural studies of complexes bound to RNA will have to be done for all three translational repressor proteins before we can fully appreciate the details of the RNA-protein interactions.

## Codon Usage

In the 275 T4 protein coding sequences, all the standard codons are used. Kunisawa (590, 591) has compared the synonymous codon usage patterns of T4 with *E. coli* and has found that, as expected, T4 makes greater use of codons with A and U in the third position whereas *E. coli* uses G or C (Table 5). The overall codon usage in T4 genes reflects the 65.5% A+T content of all coding regions and both the general base position preferences and codon biases typically observed in AT-rich genomes. *E. coli* tRNAs can read all T4 codons because of the wobble in third-position recognition of most codons. While T4 encodes eight tRNAs of its own (discussed below), mutants from which they are deleted grow normally in most bacterial strains and under standard laboratory conditions (5, 962, 1179).

Unrestricted use of all codon triplets requires 50% G+C and 50% A+T, whereas an AT-rich genome has an overall reduced codon capacity. This is evident from codon usage patterns. From the T4 codon usage table (Table 5), it can be calculated that T4 uses 64.7% A+T in its codons and only 35.3% G+C. These values are close to the approximate theoretical edge of 66.6% A+T to allow, by probability, for amino acids encoded predominantly by GC-rich triplets (e.g., Arg, Ala, Gly, and Pro) to be encoded only rarely. An analysis of the intrastrand bias of bases in the first, second, and third positions of codons in T4 genes has been presented by Kano-Sueoka et al. (499) and is summarized above (see “Nucleotide skew in the T4 genome”). The correlation coefficient for T4 genes (Table 1) in part utilizes the skew to predict probable coding regions for most genes—for example, over half of the G’s in coding regions are found in the first position of the codon.

## Translational Repressor Proteins

Autogenous translational repression by the T4 ssDNA-binding protein gp32 played a significant role in establishing the importance of posttranscriptional gene regulation (reviewed in references 325 and 736). T4 has three well-characterized translational repressors, gp32, gp43, and RegA. The first two proteins have high-affinity binding sites only on their own mRNAs, whereas RegA binds to several other separate mRNAs in addition to its own (736). gp32 binds to an RNA pseudoknot upstream of the RBS, which then promotes cooperative loading in the 3′ direction to block the translation initiation site (240, 428, 984). The protein is a metalloprotein that utilizes a retrovirus-like Zn(II) domain for RNA-binding specificity (363, 985). With the DNA polymerase, gp43, the repression

TABLE 5. T4 and E. coli codon usage and tRNA availability

| Amino acid | Codon | Cumulative frequency (per $10^{3}$) | Anticodon | tRNA availability, gene copy no. [relative content] |
|------------|-------|-------------------------------------|-----------|----------------------------------------------------|
|            |       | $E. coli$                          |           | $E. coli$                                      |
| Arg        | CGU   | 20.9                               | ACG       | 4 [0.9]                                        |
|            | CGC   | 22.0                               |           |                                                |
|            | CGA   | 3.6                                | CCG       | 1 [minor]                                       |
|            | CGG   | 5.4                                |           |                                                |
|            | AGA   | 2.1                                | UCU       | 1 [minor]                                       |
|            | AGG   | 1.2                                | CCU       | 1 [minor]                                       |
| Leu        | UUA   | 13.9                               | UAA       | 1 [0.25]                                       |
|            | UUG   | 13.7                               | CAA       | 1 [0.2]                                         |
|            | CUU   | 11.0                               | GAG       | 1 [0.3]                                         |
|            | CUC   | 11.1                               |           |                                                |
|            | CUA   | 3.9                                | UAG       | 1 [minor]                                       |
|            | CUG   | 52.6                               | CAG       | 4 [1.0]                                         |
| Ser        | UCU   | 8.5                                | GGA       | 2 [0.25]                                       |
|            | UCC   | 8.6                                | UGA       | 1 [0.25]                                       |
|            | UCA   | 7.2                                | CGA       | 1 [0.05]                                       |
|            | UCG   | 8.9                                |           |                                                |
|            | AGU   | 8.8                                | GCU       | 1 [0.25]                                       |
| Ala        | GCU   | 15.3                               | GGC       | 2 [0.3]                                         |
|            | GCC   | 25.5                               | UGC       | 3 [1.0]                                         |
|            | GCA   | 20.1                               |           |                                                |
|            | GCG   | 33.6                               |           |                                                |
| Gly        | GGU   | 24.7                               | GCC       | 4 [1.1]                                         |
|            | GGC   | 29.6                               | UCC       | 1 [0.15]                                       |
|            | GGA   | 8.0                                | CCC       | 1 [0.1]                                         |
|            | GGG   | 11.1                               |           |                                                |
| Pro        | CCU   | 7.0                                | GGG       | 1 [minor]                                       |
|            | CCC   | 5.5                                | UGG       | 1 [0.3]                                         |
|            | CCA   | 8.4                                | CGG       | 1 [0.3]                                         |
|            | CCG   | 23.2                               |           |                                                |
| Thr        | ACU   | 9.0                                | GGU       | 2 [0.8]                                         |
|            | ACC   | 23.4                               | UGU       | 1 [0.1]                                         |
|            | ACA   | 7.1                                | CGU       | 2 [0.1]                                         |
| Val        | GUU   | 18.3                               | GAC       | 2 [0.4]                                         |
|            | GUC   | 15.3                               | UAC       | 5 [1.05]                                       |
|            | GUA   | 10.9                               |           |                                                |
|            | GUG   | 26.4                               |           |                                                |
| Ile        | AUU   | 30.3                               | GAU       | 3 [1.0]                                         |
|            | AUC   | 25.1                               | CAU       | 2 [0.05]                                       |
| Asn        | AAU   | 17.7                               | GUU       | 4 [0.6]                                         |
|            | AAC   | 21.7                               |           |                                                |
| Asp        | GAU   | 32.1                               | GUC       | 3 [0.8]                                         |
|            | GAC   | 19.1                               |           |                                                |
| Cys        | UGU   | 5.2                                | GCA       | 1 [minor]                                       |
|            | UGC   | 6.5                                |           |                                                |
| Gln        | CAA   | 15.3                               | UUG       | 2 [0.3]                                         |
|            | CAG   | 28.8                               | CUG       | 2 [0.4]                                         |
| Glu        | GAA   | 39.4                               | UUC       | 4 [0.9]                                         |
|            | GAG   | 17.8                               |           |                                                |
| His        | CAU   | 12.9                               | GUG       | 1 [0.4]                                         |
|            | CAC   | 9.7                                |           |                                                |
| Lys        | AAA   | 33.6                               | UUU       | 6 [1.0]                                         |
|            | AAG   | 10.3                               |           |                                                |
| Phe        | UUU   | 22.3                               | GAA       | 2 [0.35]                                       |
|            | UUC   | 16.6                               |           |                                                |
| Tyr        | UAU   | 16.2                               | GUA       | 3 [0.5]                                         |
|            | UAC   | 12.2                               | CAU       | 6 [0.8]                                         |
| Met        | AUG   | 27.9                               | CCA       | 1 [0.3]                                         |
| Trp        | UGG   | 15.2                               |           |                                                |

$tRNAs$

As indicated (Tables 1, 2, and 5), T4 encodes eight tRNAs, with the following specificities: Ile (AU A), Thr (ACA), Ser (UCA), Pro (CCA), Gly (GGA), Leu (UUA), Gln (CAA), and Arg (AGA). A prominent late transcript contains all eight tRNAs, although early and middle promoters direct transcription into the tRNA cluster as well (772). Maturation from the primary transcript occurs through the activity of host-encoded processing enzymes (962) and autocatalysis (reviewed in reference 772). In each case, the T4 tRNA recognizes a codon that is relatively minor in E. coli but more frequent in T4 genes. There is no positive correlation between the most abundant amino acids in the T4 proteome and the tRNAs encoded by the phage (591). E. coli-optimal codons are in fact used more frequently for T4 proteins present in large numbers per phage particle (such as in gp23, the major capsid protein), while T4-optimal codons, defined as those recognized by the phage tRNAs, are used more frequently for T4 proteins present in small numbers per phage particle (and probably in weakly expressed genes). This may serve to enhance the expression of low-abundance T4 late proteins, whose products are required in larger amounts than the typical low-abundance E. coli protein (590–592).

The T4 tRNAs may have been acquired more recently in the evolutionary history of the phage, possibly through the activity of the segB-encoded endonuclease located in the T4 tRNA gene cluster. Schmidt and Apirion (962) and Mosig (772) discuss how the T4 tRNAs are required in certain hosts and may help increase fitness in some environments. Phages related to T4 that have been examined also encode some tRNA genes, yet there is a surprising variation in the specific tRNAs represented (919). Additional work on the importance of the T4 tRNAs under different growth conditions, and in different hosts, would be of interest.

The program tRNAscan-SE (664) combines up to three algorithms to examine genomic sequence for putative tRNAs and pseudo-tRNAs, identifying the specific anticodon of each tRNA (http://www.genetics.wustl.edu/eddy/tRNAscan-SE/). tRNAscan-SE was used to analyze the complete T4 genome for previously undetected tRNAs, and none were found. T4 also encodes two small RNAs (RNAD and RNAC) that immediately follow the tRNA genes and are cotranscribed with them. The functions of these RNAs remain unknown.

**Introns**

The first intron identified in the prokaryotic world was found in the thymidylate synthase gene ($td$) of T4 (174). T4 has three self-splicing group I introns, one each in $td$, $nrdB$, and $nrdD$ ($sunY$) (reviewed in reference 992); the last two genes encode subunits of the aerobic and anaerobic ribonucleotide reductases, respectively. All three introns are structurally similar, and all use guanosine nucleotide (GTP) in transesterification reactions that lead to ligation of the flanking mRNA exons. T4 introns are clear structural members of group IA2, which are prevalent in Eucarya (993). Each intron contains an ORF that is located on the periphery of the intron structure and does not interfere with the catalytic activity of the RNA. The ORF products, designated I-TevI, I-TevII, and I-TevIII, are DNA endonucleases involved in the “homing” or dissemination of

$a$ The total number of codons is 1,363,498 for $E. coli$ (4,290 protein-coding genes) and 54,574 for T4 (274 genes). Optimal codons of $E. coli$ are underlined.

$b$ The cellular tRNA contents relative to Leu-tRNA with anticodon CAG are shown in brackets. Low-abundance tRNAs that are difficult to quantify by the two-dimensional gel analysis are shown as [minor].

the intron DNA into intronless sites of homologous sequence. Homing enzymes are also commonly encoded by group IA introns of *Eucarya*. The T4 I-Tev enzymes are related to the non-intron-encoded Seg and Mob endonucleases distributed throughout the T4 genome (see below) (988).

T4 introns are similar to other group I introns in that they fold into a “core” secondary structure involving the paired regions designated P3, P4, P6, and P7 (151). The catalytic RNA center has binding sites for GTP and for Mg²⁺, which is important for proper folding of the RNA. An internal RNA guide sequence (usually located in P1) is important for properly positioning the 5′ exon with the 3′ exon splice site. Folding of the T4 *td* intron into the catalytically active RNA is affected by host RNA chaperones, such as the *E. coli* RNA-binding proteins StpA, ribosomal protein S12, and Cyt-18 (179, 1140, 1238). Splicing in vitro can occur without these chaperones. Most of what is known about RNA catalysis by group I introns, including those of T4, derives from experiments conducted on the *Tetrahymena* introns (150, 152). The similarity between T4 introns and group I introns of *Eucarya* is thought to reflect a common ancestry (993).

Although the *td* intron was identified by the sequence disparity between the DNA, RNA, and protein, the *nrdB* and *nrdD* introns were detected by in vitro labeling of the excised intron with [α-³²P]GTP (341). This approach has emerged as a general method of identifying group I introns in mRNAs (915), showing that they are variably distributed among different groups of phage and bacteria. Other T-even phages (including the recently sequenced RB69 genome [J. Karam et al., personal communication]) lack one or more of the introns (247, 882). In phage RB3, for example, the *nrdB* intron and its homing endonuclease, I-TevII, are intact and functional whereas the T4 I-TevII is partially deleted and inactive (247).

More recent work on phage group I introns has focused on phage of gram-positive bacteria. Such introns are present in lysin genes of lambdoid phages infecting *Streptococcus thermophilis* (279), thymidylate synthase (*thy*) of *Bacillus subtilis* phage B22 (42), and the DNA polymerase genes of *B. subtilis* phages SPO1, SP82, 2C, and phi e (336). Three introns disrupt *orf142*, and two introns disrupt the large subunit of ribonucleotide reductase2 (*nrdE*) of *Staphylococcus aureus* phage Twort (614, 615); this phage, the first ever identified and described, therefore has at least five functional group I introns. As in T4, phage and bacterial group I introns are usually located in important genes for enzymes of DNA metabolism and usually are inserted in or adjacent to codons for conserved amino acids (614). The distribution and ancestry of group I introns in phage populations have been discussed (251); expression of the homing endonuclease, the cleavage specificity of these enzymes, and the likelihood of a phage infecting a cell where introns are present can all impact their distribution.

Although inteins (intervening sequences excised at the protein level) have been identified in other bacteriophages (863), none have yet been observed in T4 proteins or proteins of T4-like phages.

An RNA helix at the 5′ end of a transcript stabilizes the mRNA against degradation; the T4 gene 32 5′ hairpin is well documented to confer stability on its mRNA (340, 658). Host-encoded enzymes (such as RNase E), comprising an RNA “degradosome,” directly participate in the decay of T4 mRNAs (142, 795; reviewed in reference 359). However, the phage does not appear to modify the RNA degradosome or to encode any accessory proteins.

T4 does encode a riboendonuclease, RegB, that inactivates numerous early mRNAs by cleaving them at the SD sequence, GGAG (942, 943, 954). RegB also decreases the chemical half-life of early mRNAs, whereas middle and late mRNAs are neither cleaved nor destabilized. It appears that RegB recognizes a structured conformation of the GGAG sequence that is presented or stabilized by the 30S ribosomal protein S1 (628). It is not clear how RegB-mediated cleavage at these sites is affected (or not) by the covalent modification of S1 directed by the T4 ModB ADP-ribosylation enzyme. Kai and Yonesaki (494) described effects of mutations in *dmd* that lead to the accumulation of discretely cleaved mRNAs of middle and late mRNAs. The presence of the wild-type *dmd* gene, directly or indirectly, stabilizes these T4 mRNAs; RNase I (also called RNase M [1052]), among others, was implicated in the mRNA cleavage. As mentioned in the discussion of transcription termination, the goF product is also implicated in mRNA degradation.

On infection of certain *E. coli* strains, the 26-aa polypeptide encoded by the T4 *stp* gene activates the latent DNA and RNA restriction system of the host *prr* locus. The *prrC* anti-codon nuclease cleaves Lys tRNA, the most frequently used tRNA for T4 protein synthesis. While *stp* would serve to self-destruct the mRNA translation of the infecting phage, the T4 genes *pnk* and *nlA* encode the requisite 3′-phosphatase-polynucleotide kinase and RNA ligase, respectively, to restore the functional tRNA Lys (859). Quite possibly *stp* and *prrC* represent components of an evolutionarily important RNA restriction system, and the T-even phages, in their various hosts, employ tRNA cleavage reactions to exclude the propagation of related viruses (see below) (859).

### Proteolysis

Another interesting degradative “restriction system” operating on the translational apparatus during T4 infection is the Gol-Lit interaction (1021). The defective prophage e14 present in certain *E. coli* strains encodes a latent metalloprotease (Lit, for “late inhibition of T4”). During the late stages of the T4 infection cycle, Lit is active in cleaving host translation elongation factor EF-Tu between amino acids Gly59 and Ile60. These residues are central to the RGITI motif of the Mg-GTP-binding domain (1231). Consequently, all translation is inhibited, albeit at a time shortly before lysis of the T4-infected cells. Activation or binding of the Lit protease is promoted by T4 Gol (for “grows on Lit-producing bacteria”), a 29-residue peptide (AVMGMVRRAIPNLIAFDICGVQPMNSPTG) corresponding to residues 93 to 122 of the gp23 protein (78). Lit associates with the EF-Tu–GDP open complex, which appears to be stabilized by the Gol peptide, resulting in EF-Tu cleavage. Because the Gol peptide is a segment of the N-terminal proximal region of the phage gp23 head protein, it has been

### mRNA and tRNA Turnover

The early T4 literature on RNA processing and mRNA decay has been reviewed (962, 1166; also see reference 359).

proposed that binding of EF-Tu to the Gol domain may also assist in the assembly of phage capsids during synthesis of gp23 (78). The Gol (gp23)–EF-Tu interaction is just one of several associations between viral proteins and cellular translation factors that suggest ubiquitous strategies for viral development and maturation (78).

The T4 Pin protein (1012) inhibits the host Lon protease. As one consequence, truncated peptides of T4 nonsense mutants are more stable than those of *E. coli*.

# DNA METABOLISM, REPLICATION, RECOMBINATION, AND REPAIR

A large fraction of the T4-encoded enzymes with known metabolic functions are devoted to DNA metabolism. T4 not only encodes all of the components of its own replisome and recombination systems but also makes most of the enzymes involved in preparing nucleotides for incorporation into DNA. A number of these duplicate host enzymes (aerobic and anaerobic ribonucleotide reductases, thymidylate synthase, and thymidine kinase). Others are uniquely related to the utilization of hydroxymethylcytosine rather than cytosine in T4 DNA (dCTPase, dCMP hydroxymethylase, dHMP/dTMP/dCMP kinase, and DNA glucosyl transferases that sugar coat the HMC-containing DNA to protect it from attack by certain host nucleases).

Many known T4 proteins function only as parts of macromolecular complexes (14). This is true not only for the assembly of the complex phage particle (see below) but also for most of the T4 enzymes of nucleotide biosynthesis, DNA replication, recombination, and transcription. Understanding these complex protein machines requires not only work with purified proteins but also analysis of the effects of mutations in these proteins in vivo and examination of the conformational changes that occur in the proteins while they interact with different components of the complexes.

## Enzymes of Nucleotide Metabolism

Among the best understood of these machines is the T4 nucleotide precursor complex (reviewed in references 348 and 695). It takes both cellular nucleoside diphosphates (NDPs) and the deoxynucleoside monophosphates (dNMPs) from host-DNA breakdown and converts them into deoxynucleoside triphosphates (dNTPs), in the proper ratios for T4 DNA (two-thirds A+T, in contrast to the 50% A+T found in the host). The precursor synthesis occurs at the appropriate rate for normal T4 DNA replication even when DNA synthesis is otherwise blocked, implying that the regulation is not via feedback inhibition. Proteins of the nucleotide-precursor complex, which includes two host proteins, are thought to interact with the replisome as they funnel nucleotides directly into the DNA replication complex. T4 ssb (gp32) is thought to be an essential coupler of the precursor and replication complex (1161). The interactions at all these levels have been documented by such methods as in vivo substrate channeling, intergenic complementation, cross-linking, immunoprecipitation, and affinity chromatography, as well as by kinetic studies of substrates moving through the purified precursor complex (1161). One consequence of the tight coupling is that dNTPs entering per-

them) (799, 800, 982–983) show homologies to replication proteins of many other prokaryotic and eukaryotic organisms. The two most striking examples are the DNA polymerase, gp43, and the sliding clamp of T4, gp45. Like the eukaryotic sliding clamp, T4 gp45 forms a pseudo-hexameric protein which is trimer of a protein with two similarly folded domains. In contrast, the *E. coli* sliding clamp is a dimer of trimers. Moreover, the clamp loader of T4 can partially substitute for a eukaryotic clamp loader in an *in vitro* replication system (1135).

Mutations in many other genes affect DNA replication in vivo (Table 2), because these genes are important for recombination-dependent DNA replication and repair of broken forks (see below). Since DNA is constantly assaulted by intrinsic and extrinsic damaging agents, DNA replication and recombination are tightly correlated and should not be considered in isolation. Studies of T4 have paved the way to understanding these relationships (200, 572, 575, 577, 668, 771).

DNA replication genes are scattered throughout the genome, with a major cluster that includes the DNA polymerase (gene 43); most of these genes are transcribed from both early and middle promoters. Coordination of the T4 DNA replication functions is achieved by the assembly and disassembly of protein complexes that appear to use stretches of DNA covered with ssDNA-binding protein, gp32, as scaffolds (283, 765, 766, 774, 793, 1161). In wild-type T4, the syntheses of the leading and lagging strands are coupled, but both in vitro and in vivo leading-strand synthesis can be uncoupled to proceed on single-stranded templates, or by displacement of one parental DNA strand on double-stranded templates (488, 788).

In contrast to the *E. coli* replisome, the T4 replisome has not been isolated as a functional complex. Possibly, the weak interactions of the core replisome components facilitate interactions of DNA polymerase with different accessory replication and recombination proteins, as discussed below. These various interactions can be correlated with the three-dimensional structures of the DNA polymerase, the sliding clamp and the core of the ssb, gp32, from T4 and the related phage RB69 (983, 1141, 1146). Extensive analysis of mutator and antimutator strains has also facilitated our understanding of these interactions (827, 904, 906, 909).

Replication and initiation of late transcription are strongly coupled by the requirement for the sliding clamp of DNA polymerase, gp45, to transform closed to open RNA polymerase-promoter complexes, and the requirement for ssDNA interruptions to load the clamp (552a, 951, 1082, 1173). The replication complex moves along the DNA at 10 times the speed of the transcription complex. Because both processes proceed in both directions, frequent collisions must occur. In vitro evidence shows that T4 transcription and replication complexes can pass each other when they meet head-on, with the RNA polymerase changing templates without interfering with the total processivity of transcription (650). It now appears that this surprising and very useful ability is not universal. For *E. coli* (294), evidence indicates that head-on collisions between DNA and RNA polymerases retard the movement of both.

---

### Initiation of DNA Replication

The first round of T4 DNA replication is initiated from one of several different origins (Table 1; Fig. 3). Replication is mediated by T4-encoded enzymes, with the exception that the host RNA polymerase synthesizes the primers for leading-strand initiation at origins (577, 668) and that host DNA polymerase I can remove RNA primers of Okazaki fragments (427). T4 primase then synthesizes primers for Okazaki fragments. In its absence, unidirectional displacement synthesis occurs (46, 47, 778, 779, 788). The displaced strand is later copied by a recombination-dependent “join-cut-copy” mechanism (discussed below). Combinations of density shift with electron microscopy analyses (199, 200) have shown that in most cases only one origin is used and is used only once, probably because at the time of origin initiation there are limited supplies of DNA polymerase and other replisome components. When the timing of early-gene expression is distorted by mutation, origins can be used repeatedly (for a review, see reference 563). As soon as the first replication forks have reached an end, recombination intermediates compete effectively for assembly of replisomes, and because T4 chromosome ends are circularly permuted, less dependence on specific origins is observed (200).

Four major DNA replication origins have been mapped to specific sequences, which are different in each case. The transition from RNA primers to leading-strand DNA synthesis in vivo occurs at several sites located over 1 kb downstream of the promoter that initiates primer transcripts (778, 779, 1109). At *oriA*, *oriF*, and *oriG*, the priming transcripts are initiated from MotA-dependent middle promoters (577). Priming of DNA synthesis by a transcript requires that the transcript reinvade the DNA, forming an R-loop. This reinvagination can be facilitated by global torsional stress in the DNA. Indeed, in vitro initiation from *oriF* has been achieved with an RNA primer that had been taken up by a supercoiled plasmid (830). The end of the R-loop used in these experiments was designed to be the same as in R-loops isolated from replicated DNA in vivo and probed for single strandedness of the displaced DNA strand. By analogy to *E. coli* (551a), this RNA end was postulated to be processed from a longer transcript by RNase H (133). However, this site is different from the major in vivo priming sites detected by primer extensions on nascent DNA (779). These occur in the terminator region between genes *wsY.-2* and *wsW* and require little or no processing. Possibly, both priming mechanisms are used in the *oriF* region.

In contrast, *oriE* can still function when torsional stress is reduced (for instance, by mutation in the host gyrase [789]). At *oriE*, the priming transcript is initiated from an early promoter. *oriE* function depends, in addition, on a small protein, RepEB, and the auxiliary protein RepEA, both encoded by transcripts related to the primer. In contrast to the other origins, *oriE* contains repeat sequences, the so-called iterons (779, 1109). The binding of RepEB to one or more of these iterons is required for *oriE* to function (Harvey et al., unpublished). It is postulated that binding of RepEB to the iterons facilitates the loading of a DNA helicase and that unwinding by the helicase can compensate for the lack of global torsional stress in *oriE*.

The different structures and functional requirements of different T4 origins may be considered as models for poorly

understood complexities of other multiorigin chromosomes. For example, in the major origin of *E. coli*, *oriC*, leading-strand synthesis can be primed by primase-dependent or RNA polymerase-dependent RNAs (558); *E. coli* can use other origins, *oriK*, which, like T4 origins, depend entirely on transcripts for priming (34). Similarly, different yeast origins are detected in different labs using different methods (1042).

### Recombination and Recombination-Dependent DNA Replication

The discovery of recombination provided a powerful argument that phages can be used as model systems (411), and a strong connection between replication and recombination was suspected early on (1129). Our current view of these tight interconnections and the interactions with the transcriptional and posttranscriptional developmental program of T4 are summarized in Fig. 4.

Although the onset of DNA replication is largely dependent on replication origins, most T4 replication forks are initiated by using intermediates of recombination as DNA primers at more or less random positions throughout the genome. Because origin-dependent DNA replication is inactivated during the developmental program, recombination-deficient T4 mutants arrest DNA replication prematurely.

There are several different recombination-dependent replication modes, which require different recombination proteins; these have been thoroughly reviewed (768, 769, 784). In undamaged chromosomes, early T4 join-copy recombination depends on origin-dependent replication; it is initiated from the ssDNA at the unreplicated end of the template for lagging-strand synthesis. In damaged T4 chromosomes, similar mechanisms can be used to repair broken replication forks (575) (see below). When origin-initiated DNA replication is inhibited, recombination can occur, but it begins later and requires additional nuclease activities (111). Electron micrographs of such T4 DNA intermediates provided the first compelling evidence for annealing of single strands as one way to initiate recombination (pathway I in Fig. 4) and for the importance of branch migration in homologous recombination (111). Under replication-deficient conditions, no viable progeny particles are produced. There is no late join-cut-copy recombination, no packageable DNA concatemers are formed, there is little late transcription (since that is dependent on DNA replication), and no heads are formed that can be filled.

The main genes required for join-copy recombination-replication (Fig. 4) (in addition to the SSB gene 32 that is required for most aspects of DNA replication, recombination and repair) are *uvsX*, *uvsY*, 46, and 47 (Table 2). The 46 and 47 proteins, acting in a complex, have similarity to the *E. coli* SbcBC and RecBC proteins and the eukaryotic Rad50/Mre11 complex. The strand invasion UvsX protein is a structural and functional homologue of the *E. coli* RecA protein and the eukaryotic Rad51, Dmc1, and Rad54 proteins (69, 192, 739). As expected, if this is the major pathway to initiate T4 replication forks, defective mutants arrest DNA replication after limited origin replication has occurred. The corresponding genes are expressed from early and middle promoters, concomitantly with the replication proteins acting at the fork. Thus, this pathway can operate early.

In contrast, the endoVII (49) and terminase (17) genes required for join-cut-copy recombination-replication are predominantly expressed late and therefore are exclusively part of a late recombination pathway that has been called the “join-cut-copy” pathway (768, 769). The late *uvsW* gene, encoding an RNA-DNA helicase and a functional homologue of the *E. coli* RecG protein (132), is probably also specifically important in this late recombination pathway (722). This pathway is also implicated in horizontal transfer between different phages of DNA segments with limited homology (777, 784).

Together, these mechanisms generate the branched, concatemeric intracellular T4 DNA. This DNA is debranched in vivo by T4 endonuclease VII (gp49), which can cut Holliday junctions, Y-junctions, and mismatched base pairs in heteroduplex DNA (reviewed in reference 522), and by the largest (70-kDa) subunit of the heteromeric terminase (288). As mentioned above, genes 49 and 17 both code for several nested proteins by initiation from internal ribosome binding sites, and the function of EndoVII (gp49) in vivo depends on the regulated timed synthesis of the two nested proteins (reviewed in reference 784). This can now be correlated with the three-dimensional structure of EndoVII (883).

### DNA Repair

As in other organisms, damage and mismatches in T4 DNA can be repaired in several different ways, and repair defects result in increased sensitivities to such damage or increased mutation rates. The consequences of such mutations have been summarized by Bernstein and Wallace (67).

The first mechanism shown to repair UV-damaged T-even DNA was photoreactivation (245). It is now known that the host enzyme responsible for this repair causes the reversion exclusively of pyrimidine dimers to the monomeric state (reviewed in references 233 and 234).

Harm (384) first isolated DNA repair mutants with mutations in T4 genes that are now called *denV* and *uvsX*. EndoV, the product of *denV*, is the prototype of a base excision repair protein. It has both *N*-glycosylase and abasic lyase activities (222, 656), which incise the DNA (forming a covalently linked protein-DNA intermediate). Together, these activities remove the pyrimidine dimers to allow resynthesis by DNA polymerases, notably including *E. coli* PolI (1056) and the T4 ssDNA-binding protein, gp32 (764). The profound difference in UV sensitivities of T4 and T2 is due to the presence of *denV* in T4 but not in T2 (222, 384).

The T4 *uvsX* gene encodes a RecA homologue, the major protein required for invasion of ssDNA into homologous double-stranded DNA (dsDNA) to form D-loops, which can be extended to form two heteroduplexes joined by Holliday junctions. The radiation sensitivity of *uvsX* mutants is now understood to be a consequence of defective recombination-dependent DNA replication that can repair or bypass DNA lesions. Therefore, all recombination-deficient mutants (Table 2) are also defective in DNA repair (60, 68, 1056; reviewed in references 67, 572, and 575).

Heteroduplex (or mismatch) repair of T4 DNA in vivo and in vitro is mediated by EndoVII, the enzyme that cuts Holliday junctions (522, 526, 793), not by MutHLS-like enzymes. The specificity of this activity is thought to contribute to the exclusion.

sion of viable recombinants between different T-even phages whose sequences have diverged (305, 784).

# MOBILE ENDONUCLEASES, GENE TRANSFER, AND GENE EXCLUSION

Following discovery of group I introns in T4 (see above) (991), site-specific endonucleolytic DNA cleavage by the protein of the intron ORF was demonstrated (882); reviewed in reference (178). The three intron ORFs of T-even-like phages are I-TevI (*td* intron), I-TevII (*nrdD* intron), and I-TevIII (*nrdB* intron). In T4, I-TevIII is partially deleted so that the endonuclease activity is absent; the T4-related phage RB3 has an intact ORF, and endonucleolytic activity of the I-TevIII enzyme is detected (247). Each endonuclease recognizes a so-called “homing” site that is cleaved by the enzyme, with the double-strand break (DSB) serving as a recombination site for insertion, or conversion, of the intron. Thus, intronless sites, when present in mixed infections with intron-containing phage or other DNAs containing the intron and its ORF, are efficiently converted to contain the intron. The apparent DSB recombination process involves the I-Tev nuclease only for the generation of the DSB. The specificity of these nucleases involves a site on the DNA for binding and a separate cleavage site, which for I-TevI is 23 to 25 bp upstream of the insertion site (202). Subdomains for DNA binding by the I-TevI nucleases include a zinc finger, an α-helix, and a helix-turn-helix. Homing endonucleases of introns can be viewed as systems that target localized gene conversion events, mobilize specific DNAs from one chromosome to another, or, as for SegF (see below), effectively exclude genes or larger regions from “invading” a related chromosome.

At least 15 T4 genes (including the I-Tev genes) belong to two of the four structural families of homing endonucleases (163, 992) defined by the conserved sequences GIY-YIG, LAGLIDADG, H-N-H, and His-Cys (49). Sharma et al. (988) recognized and described the *segA-segE* set of T4 proteins, which are not associated with an intron, and provided convincing evidence for the endonuclease activity of SegA. Kadyrov et al. (489, 490) later demonstrated that *segE* promotes its own transfer to the related phage RB30, which does not carry this gene. They reported that efficient transfer requires bases upstream and downstream of the *segE* cleavage site, and they cited unpublished data that SegB and SegD can initiate similar nonreciprocal genetic exchange. The location of SegB in the tRNA gene cluster suggests that its cleavage site is in or near these genes and that it could be responsible for maintaining (recombinational exclusion) or transferring the tRNAs. Errors near the putative start codons in the initially published sequences of the three *seg* genes (which were sequenced from clones) have been corrected (489, 490). This is consistent with the suggestion that they are expressed and lethal to the host and that mutated versions were selected during cloning. Indeed, the recombination-dependent gene exclusion in the T4 56-69 region (305) is due to a DSB generated by the product of the gene 69 ORF, prompting the renaming of the gene as *segF* (51). A similar localized gene conversion activity in the gene 32 (encoding the essential T4 SSB protein) region has been recently demonstrated for the product of ORF 32.1, which has

been renamed *segG* (48; Q. Liu, A. Belle, D. A. Shub, M. Belfort, and D. R. Edgell, submitted for publication).

I-TevI, I-TevII, and SegA through SegG all appear to have the GIY-YIG family signature (cited in references 51 and 202; also see the pfam alignment at http://www.sanger.ac.uk/cgi-bin/Pfam/getacc?PF01541). SegF aligns in the N-terminal half with the GIY-YIG domain, but in the C-terminal half it aligns with MobD, a member of the H-N-H family (51, 305).

I-TevIII and T4 Mob proteins (MobA through MobE) group with the H-N-H family of DNA endonucleases, which are rooted by the sequence and structure of the DNase colicin e7 (microcin e7). The H-N-H signature sequence also includes the mobility endonucleases of group II introns and the *E. coli* restriction endonuclease McrA (337, 992). Reiterative PSI-BLAST analyses of the uncharacterized T4 ORFs may yet yield more members of these DNA endonuclease, which appear to be surprisingly abundant in the T4 genome.

It is not yet clear how many of the *mob* genes are actually expressed during T4 infection. Two of them—*mobA*, between genes 39 and 60, and the nuclease gene in the T4 *nrdB* intron—seem clearly to be pseudogenes and are nonfunctional (247). *segD* is situated in reverse orientation with respect to adjacent genes, with antisense RNA from gene 24 occurring between *segD* and the nearest promoter from which it could be expressed. No functional or homing studies have yet been carried out for any of the *mob* genes, although *mobD* has been successfully cloned and overexpressed (Kutter, unpublished). The role that these enzymes play in gene mobility and transfer and/or in gene exclusion and recombination processes merits further analysis.

## T4 PARTICLE, INFECTION, AND LYSIS

T-even phages build some of the most complex virus particles known (Fig. 1 and 8). They devote more than 40% of their genetic information to the synthesis and assembly of the prolate icosahedral heads, tails with contractile sheaths, and six tail fibers that contribute to their very high efficiency of infection. Most of the genes for the structural proteins are transcribed in the clockwise direction on the standard genomic map. The genes responsible for each substructure are largely clustered, with these clusters distributed over more than half of the genome (Table 1; Fig. 3). There are some interesting exceptions to the clustering. For example, the tail is built on a baseplate made of a hub and six wedge components, and each of the substructures is encoded by a gene cluster. However, gene 5, one of the five genes involved in assembling the hub, is actually located with the genes for the wedge, while a wedge gene, gene 25, is located with hub genes; these two clusters are separated by the head and other tail proteins. Interestingly, each cluster also contains a replication origin and the direction of transcription switches within each cluster.

Exquisite genetic and biochemical analyses revealed the complex assembly pathways of the T4 particle (89, 186, 272). Twenty-four proteins are involved in head morphogenesis, and there are 22 for tail morphogenesis and 7 for tail fibers, including one for tail fiber attachment (Tables 2 and 3). As described below, 5 of the 54 proteins are catalysts for assembly rather than components of the final virion. In the assembly pathway, the head, tail and tail fibers are assembled indepen-

Head
23, 24
hoc, soc
IP I, II, III
alt
DNA

Tail
3, 15,
18, 19

Whiskers / Neck
wac, 20, 13, 14

Tail fibers
34, 35, 36, 37

Baseplate
7, 8, 9, 10, 11, 12, 6, 25
53, 5, 27, 29 ,26, 28 ,48, 54

FIG. 8. Structural components of the T4 particle. Features of the particle have been resolved to about 3 nm. The positions of several head, tail, baseplate, and tail fiber proteins are indicated (see the text for details and references). Adapted and reprinted from reference 506 with permission from the American Society for Microbiology, with baseplate modifications introduced by Petr Leiman (M. Rossmann laboratory, Purdue University).

dently. A head and tail are associated, and then the six tail fibers attach to the baseplate (250).

Heads

Of the 24 proteins assigned to head morphogenesis, 16 are involved in prohead formation and maturation, 5 are responsible for DNA packaging, and 3 complete and stabilize the head. Only 10 of the 16 genes for prohead formation are essential; these include GroEL, the one host-encoded protein involved in head assembly. This contrasts with phage lambda, where GroES, DnaK, DnaJ, GrpE, and GroEL are all essential (187, 317, 318, 1078). van der Vies et al. (1115) showed that T4 gp31 can substitute for the function of GroES. Since then, crystallographic analysis revealed that gp31 forms a heptameric ring that is quite similar to the GroES structure. However, T4 gp31 has an extra loop that makes the inner cavity of the GroEL-gp31 complex larger so that it can accommodate the folding intermediate of gp23, the major capsid protein (455). The head is assembled on the initiator complex, which is a 12-mer of gp20 arranged in a ring. The scaffold, made of gp22-gp21 (stoichiometry = 576:72) and the capsid protein, gp23, are assembled onto the initiator, which eventually becomes the portal vertex. Pentamers of gp24, which is 28% identical in amino acids residues to gp23, are placed at the

other 11 vertices. After the scaffold is completely surrounded by gp23 and gp24, the T4 prohead protease, gp21, degrades the scaffold and cleaves most of the other head proteins, including gp23 and gp24. This creates space in the cavity of the prohead. The prohead thus formed is then detached from the membrane (ESP = empty small particle). Jardine and Coombs (471, 472) demonstrated by pulse-chase experiment that ESP then initiates DNA packaging and forms the ISP (initiated small particle), which contains about 10 kb of DNA. The ISP is expanded about 15%, resulting in a twofold (1.15³ = 1.98) increase of the inner volume (ILP = initiated large particle). The resulting mature head is much more resistant to any kind of perturbation.

DNA Packaging

Packaging of DNA is initiated from double-stranded ends. For intracellular concatemers, the terminase complex initiates packaging by first generating ends. This complex contains a small subunit, gp16, and large subunits, gp17, gp17′ and gp17″ (286, 288). The nuclease activity of terminase resides in the products of gene 17 (74, 75, 288). The terminase-DNA complex is translocated from the cytosol to the portal protein gp20 at the vertex of the head to form the “packasome” (893), which uses the energy of ATP hydrolysis to translocate DNA into the

head. Expansion of the head is coupled to entry of DNA (471, 472). There is a symmetry mismatch between the neck initiator complex, which has 12-fold symmetry, and the head, which has 5-fold symmetry. It has been suggested (399) that the neck rotates during DNA packaging. The packaging mechanism cuts the DNA when the head is filled, and it appears that EndoVII trims branches of DNA even after packaging has been initiated (523). The head full of DNA is about 3% longer than the genome size, accounting for the circular permutation of T4 genomes, with terminal redundancy of each genome; this circular permutation is the basis for the circular T4 genetic map (1048, 1049, 1073). Shorter or longer phage heads are occasionally formed, due to assembly errors that are increased by specific mutations in some head genes (256) or by incorporation of arginine analogues (194). The amount of DNA packaged into the head is altered accordingly. While short-headed phages cannot infect singly, they can complement each other to give a normal infection (767, 770, 782). After packaging, gp13, gp14, and six trimers of gp wac (whisker or fibritin) bind to the portal vertex to complete the head, which then binds to the tail.

Although the complex head assembly pathway has resisted full in vitro reconstitution, either T4 or foreign DNA can be packaged in vitro into empty heads. These can then be assembled in vitro with tails and tail fibers to form infectious or transducing particles (434, 802, 893).

Nearly all the genes for virion structural proteins, the assembly catalysts, and the scaffold appear to be present in the genomes of T4-like phages examined to date (701, 1069), although some exceptions have been noted for the *Vibrio* phage KVP40 (E. S. Miller, J. F. Heidelberg, J. A. Eisen, W. C. Nelson, A. S. Durkin, A. Ciecko, T. V. Feldblyum, J. Lee, B. Szczypinski, O. White, I. T. Paulsen, W. C. Nierman, and C. M. Fraser, submitted for publication). Two T4 head proteins—encoded by *soc* and *hoc*—are nonessential. The unusual locations of their genes, their absence in some T4-related phages, and the fact that they are added only after head expansion during assembly are consistent with their being a later acquisition. They are possibly retained because they enhance particle stabilization. The dispensable nature of Soc and Hoc has provided a rationale for a T4 phage display system capable of presenting large polypeptides on the capsid surface (916, 918).

### Baseplate and Tails

The tail and tail fibers are responsible for the high efficiency of T4 infection. The tail is made of a baseplate and two slender cocylinders. The inner cylinder, called the tail tube or simply tube, consists of 144 subunits of gp19 arranged in 24 stacked hexameric rings. The inner space of the tail tube allows for the passage of phage DNA. The same number of gp18 molecules form the outer tail sheath, with the subunits arranged in the same manner as gp19. Each stacked sheath ring is offset 17 degrees to the right of the one below it, which gives an apparent right-handed helix (753). While the noncontracted tail sheath is 98.4 nm long, the contracted tail sheath is only 36 nm long and the offset (or twist angle) of sheath proteins is increased to 32°. The assembly and three-dimensional reconstruction of the tail and tail sheath have been reviewed (186).

The baseplate consists of a hub surrounded by six wedges, which are assembled independently. Hub assembly is fairly complex. The six products of genes 5, 27, 29, 26, 28, and 51 have been reported to be involved in the assembly. gp51 is a catalytic protein, while gp26 and gp28 have not been rigorously proven to be components of the hub or baseplate. gp5 and gp27 associate first. The hub is completed by binding of gp29 to the gp5–gp27 complex. It appears that some structural modification of gp29 is necessary before associating with the gp5–gp27 complex. Wedge assembly is initiated by association of gp10 and gp11, followed by addition of gp7, gp8, gp6, gp53 and gp25, in that order (except that gp11 can be added later, along with gp12). In the absence of any of the other components, the assembly stops at that point and the remaining components are left free in “assembly-naive states” (532). After six wedge segments bind to the hub, gp9, gp12, gp48, and gp54 are added.

The reported stoichiometry of the subunits has been summarized (186), but some corrections have since been made. There are 18 molecules of gp9 instead of 24 (560), 18 each of gp10 and gp11 instead of 12 (1239), 3 each of gp5 and gp27 instead of 6 (498), and 12 of gp8 (P. G. Leiman, unpublished data). The stoichiometry of gp3 was not known but has now been determined to be 6 (L. Zhao, unpublished data). Three-dimensional structures of gp11 (633), gp9 (560), and the gp5–gp27 complex (498) have been determined by X-ray crystallography. Interestingly, all these structures (except gp8) involve trimers of each component.

Baseplate morphogenesis appears to occur in association with the cell membrane. The baseplates remain attached to the membrane by 300-Å fibers from the six corners of the baseplate during the remainder of phage assembly until the time of cell lysis, as shown by electron microscopy (1003). This is seen even in mutants lacking gp12, which encodes the short tail fiber involved in irreversible phage binding during infection (see below). The finding that gp7 has a predicted membrane-spanning domain near its C-terminus (see below) suggests a possible mechanism for this attachment.

The presumed localization of the tail lysozyme gp5–gp27 complex in the baseplate is shown in Fig. 9. The baseplate protein gp5 is a natural chimera. A lysozyme domain, a paralog of the soluble lysozyme of T4 (787), is inserted into the center of a structural baseplate component. The overall shape of the trimeric gp5 resembles a torch, where the N-terminal domains, together with the trimeric gp27, form a cup, the lysozyme domains form the rim, and the C termini of gp5 form the handle. This “handle” is the most conspicuous structure—a three-stranded beta-helix 110 Å long and 28 Å wide. The primary structure of the β-helix region has a peculiar motif of VXGXXXXX (8 residues) repeated 12 times. The cross-section of the cylinder is not a circle but, rather, a triangle, with the glycine at position 3 located at the edges to form a kink. Among some T4-related phages (e.g., RB69, RB49, KVP40, and S-PM2) the beta-helix is well preserved, but there is some indication that its length can vary (F. Arisaka, unpublished data).

The gp5–gp27 heterohexameric complex is attached at the tip of the tube. When the tail sheath contracts and the tail tube protrudes from the bottom of the baseplate, the triple-stranded β-helix is considered to play a role like that of a needle to puncture the cell. gp5 is the only protein in the tail that experiences processing; the peptide bond between Ser351 and Ala352 is cleaved during assembly, but the C-terminal

FIG. 9. Three-dimensional image reconstruction of the T4 tube-baseplate from cryoelectron microscopy. (A) Stereo image view of the baseplate and part of the tube at 17 Å resolution. The top quarter of the baseplate has been removed to show the internal features. Note the presence of the needle-like stick at the center of the baseplate beneath the tube. The arrangement of the six short tail fibers is also clearly visible. (B) Cross-section of the reconstituted baseplate into which the atomic structure of the (gp27-gp5*-gp5C)₃ complex solved by X-ray crystallography at 2.9 Å resolution is fitted. The conspicuous three-stranded β-helix, the C-terminal domain of gp5 or gp5C, with a length of 110 Å, precisely fits into the needle-like stick. gp27 constitutes the “cup” on top of the needle. The three gp5 monomers are colored red, green, and blue. The contour map of the baseplate is in purple. Reprinted from reference 498 with permission from the publisher.

domain stays associated with the other part of the molecule in the mature virion. The lysozyme activity is activated only when the C-terminal domain is detached from the other part of gp5. It has been proposed (498) that the C-terminal domain of gp5 is detached from the lysozyme when the needle penetrates into the outer membrane upon infection (see “Infection and superinfection exclusion” below). The activated gp5 lysozyme in vitro is monomeric (497).

The complete assembly of the tail requires two chaperones, namely, gp51 and gp57A (1015). In the established model (531), gp51 is involved in the hub assembly while gp57A is involved in assembly of both the short and long tail fibers. gp57A consists of 79 aa, and more than 90% is α-helix (391, 699). Although the molecular nature of the protein has been worked out, the interactions with the short and long tail fibers are unknown.

The short tail fiber is a trimer of gp12; its partial three-dimensional structure has been reported (1118). It consists of

three domains called the pin (N terminus), shaft, and head (C terminus). The shaft is mainly β-helix and β-spiral. gp11 is located at the tip of the tail pin and bound to the middle part of the P12 trimer, at a site where the P12 shaft is bent about 94°.

gp9 forms the socket of a long tail fiber (1105) consisting of four gene products, gp34, gp35, gp36, and gp37, where gp34 and gp37 are the proximal and distal long tail fibers, respectively. gp35 and gp36 attach to the distal fiber, forming the junction between the half-fibers. Presumably one or both interact(s) with the tip of the whiskers (fibritin; wac). Structural analysis of the C-terminal portion of the whisker (1051) revealed a three-stranded coiled-coil structure with a beta structure “propeller” at the C terminus. This beta structure is thought to bind the bend or “knee” in long tail fibers to facilitate tail attachment to the baseplate. The assembly of the tail fibers requires two molecular chaperone-like proteins, gp57A and gp38. In a major difference from the T4 system, T2 gp38

(which is unrelated to T4 gp38) is a structural component of the tail fibers; it, rather than the C terminus of gp37, recognizes the host bacterium (750, 751).

# Infection and Superinfection Exclusion

The initial energy for infection is provided by the baseplate, which is built like an exquisite, “cocked” mechanical device. gp9 is the stabilizer between the long tail fibers and the baseplate. After at least three of the six long tail fibers bind to a glucose residue of the outer core of the lipopolysaccharide on the bacterial surface, a structural rearrangement of the baseplate from “hexagon” to “star” is triggered (190). This deploys the six short tail pins, after which the short tail fibers (gp12) previously in the baseplate firmly bind to the bacterial outer membrane at the heptose residue of the lipopolysaccharide inner core (751, 933). This is referred to as the second receptor of the phage. The conformational change of the baseplate simultaneously triggers contraction of the tail sheath, pushing the inner tube through the outer membrane. Contraction of the tail sheath appears to advance as a wave of compression transmitted through the helix-like arrangement of tail sheath annuli. The tail lysozyme (gp5), after detachment of the C-terminal “needle,” helps to digest the peptidoglycan layer and reach the inner membrane. The beautiful baseplate/needle structure (498) is reprinted in Fig. 9. Phosphatidylglycerol in the inner membrane appears to play a role in release of the DNA through the tip of the tail tube, but the electrochemical potential across the inner membrane is necessary for the DNA to be pulled into the cytoplasm (reviewed in reference 327). In the absence of the electrochemical potential, DNA remains in the periplasm and is eventually degraded.

Host range is determined primarily by distal sites on the long tail fibers. A C-terminal region of gp37 is hypervariable in different T-even phages (390, 750), allowing for adaptation to different host receptors. The system has been referred to as a primitive prokaryotic immunity system. As shown by Wais and Goldberg (1137), T4 can grow well in a number of other gram-negative bacteria if it can gain entry, such as when the bacteria are converted to spheroplasts and the phage are treated with urea. The urea leads to contraction of the tail sheath and formation of activated phage particles that, on contact with bacterial inner membranes, can release their DNA into a spheroplast.

The initial infection leads to poorly understood membrane alterations involving the product of the T4 *imm* (immunity) gene. One consequence of these changes is that by about 4 min after infection, the DNA from T4 or related phages attempting to penetrate the cell envelope is released into the periplasm, where it is degraded by nucleases, resulting in a phenomenon called superinfection exclusion (665, 666). Other genes that appear to be involved in this process include *sp*, *rIIA*, *rIIB*, and *rI*, but their precise functions are not well understood.

## Lysis and Lysis Inhibition

Normally, 100 to 150 phage particles have accumulated in the cell by the time lysis occurs. As in many other tailed phages, two proteins are involved in the lysis process: gpe and gpt. gpe is the so-called T4 lysozyme (1050), whose structure has been extensively studied by Mathews and colleagues (1193, 1206). Under special conditions, the gp5 lysozyme discussed above can substitute for gpe (500). gpt is the T4 holin, which, by analogy to other well-studied phagelysis systems, creates a hole in the inner membrane so that lysozyme can reach the peptidoglycan layer from the cytosol; the timing of holin assembly thus determines the timing of lysis (360, 887). In the absence of either lysozyme or the holin, lysis does not occur.

The T-even phages display a unique phenomenon, lysis inhibition, which allows them to sense when there are numerous phage around and respond appropriately to delay lysis, maximize the use of their current host, and potentially await the accumulation of additional bacterial hosts (41a, 851). Lysis is extensively delayed if more phage attack the infected cell at any time after 5 min into infection. This signal is somehow mediated by the rI protein, which regulates assembly of the holin (851). Recently, Ramanculov and Young substituted the T4 *t* gene for the holin of bacteriophage lambda and showed that these lambda phage could now produce lysis inhibition if they infected *E. coli* that carried a cloned T4 *rI* gene (886, 887). T4 gprIII further extended lysis inhibition, but no other or additional T4 proteins were required. We still do not know the specific signal that gprI senses to establish lysis inhibition; possibilities include the phage DNA or the internal proteins that have been injected into the periplasm rather than the cytoplasm under these circumstances.

The classic *rII* genes were involved in defining the phenomenon of lysis inhibition when T4 was propagated on *E. coli* B strains (224). The T4 *rII* mutants provided an example of phage exclusion by genes of resident prophages. They are completely excluded by the lambda *rexA* and *rexB* genes expressed in lysogens. This phenomenon was elegantly used in Benzer’s fine-structure analysis of the gene (56). Exclusion occurs at the time of transition from join-copy to join-cut-copy recombination and involves several enzymes important in the latter process (769, 793). The molecular mechanism of this exclusion is still poorly characterized. It has been proposed that RexB forms an ion channel that is opened after infection with T4 *rII* mutants or various other phages, leading to loss of ions and cellular energy (855). However, the way in which the *rII* proteins bypass this process is still unclear (reviewed in reference 1021). More recently (851), it was demonstrated that gprIIA and gprIIB are also primarily responsible for protecting T4-infected *E. coli* B cells against the attack of a P2-related resident prophage, with less severe consequences than when T4 *rII* mutants infect K-12 strains lysogenic for lambda. A similar phenomenon appears to be responsible for the large size of *rII* plaques on the lysogenic *E. coli* B strains (851). In that case, DNA replication is not affected and lysis does not occur until about 25 min after infection, so that a reasonable burst is produced. If the host B cell has been cured of the P2-related prophage, *rII* mutants show normal lysis inhibition. As first shown by Rutberg and Rutberg (947), many *E. coli* B strains carry a defective prophage related to P2. The primary role of the *rII* genes seems to be related to cellular energetics. The apparent “lysis inhibition” phenomenon seen on lysogenic B strains, rather than the phage death seen on K-12 lambda lysogens, appears to be due to the breakdown of cell energetics occurring near the normal lysis time on B cells, rather than at 12 min after infection of K-12 (lambda) cells.

VOL. 67, 2003

# RESTRICTION-MODIFICATION SYSTEMS AND PHAGE EXCLUSION

As mentioned above, the T4 nucleotide metabolism enzymes result in HMC-containing DNA that is protected from T4-encoded nucleases that degrade host DNA. These enzymes (encoded by *denA*, *denB*, *dexA*, and others genes) can be viewed as DNA restriction systems and also as mechanisms for generating nucleotide precursors for T4 DNA replication (139). Indeed, the DenA (EndoII) recognition sequence is the C-rich sequence 5′-CCGC-3′, which more frequently nicks the complementary strand of the sequence shown but does generate double-strand breaks (135). The modified T4 HMC DNA is resistant to cleavage, so that EndoII serves to restrict “foreign” phage or host DNA. Properties of EndoII have prompted the suggestion that the free DNA ends it generates would be recombinogenic for acquisition of DNA into the T4 genome (134). DenB (EndoIV) also cleaves cytosine-containing DNA and not the phage HMC DNA, but there is still only limited information available on its specificity (140).

In addition to the protection afforded by the HMC in T4 DNA, the phage encodes other nucleotide modification enzymes that act postreplicatively and protect the DNA. The *dam*-encoded DNA-(N⁶-adenine)methyltransferase of T4 belongs to the α group of GATC family of DNA methyltransferases. Using S-adenosylmethionine as the methyl donor, it modifies adenine in GATC and in the T4 sequence GAT HMC (140, 267). This modification protects otherwise unmodified T4 DNA from a restriction system of P1 phages.

The α-*gt*- and β-*gt*-encoded enzymes (α- and β-glucosyltransferases, respectively) modify the T4 HMC residues to the extent that there is ca. 70% in α-glucose linkage and 30% in β-glucose (reviewed in reference 139). Restriction of nonglucosylated T4 DNA (strains defective in *gt* activity) led to the discovery of the host restriction enzymes RglA and RglB, which were then recognized as broader systems for modified cytosine restriction and hence were renamed McrA and McrBC, respectively (139). The enzymes do not distinguish between mC and hmC modified DNA. *mcrA* of *E. coli* K-12 resides in a cryptic prophage-like element, e14, that is not present in all *E. coli* strains (37), whereas *mcrBC* resides at an unlinked location adjacent to the *mrr* and *hsd* restriction systems. T4 provided an important entry into unraveling the genetic organization and specificities of these enzymes. The presence of Mcr enzymes in *E. coli* or other bacterial hosts may have provided selective pressure to maintain the different DNA-glycosylating enzymes of T-even phages. In addition, the T4 *arn* gene encodes an antirestriction endonuclease that inhibits the host McrBC (RglB) enzyme (533).

Exclusion of T4 rII mutants by *E. coli* lambda lysogens is discussed above (see “Lysis and lysis” inhibition).

As detailed above (see “mRNA and tRNA turnover”), another cryptic DNA element of certain *E. coli* strains, *prr*, encodes the PrrC protein that excludes mutants deficient in T4 RNA ligase/polynucleotide kinase. The PrrC RNA endonuclease is activated by the small T4 Stp protein to cleave the anticodon loop of an essential host lysine tRNA (515, 1021). T4 RNA ligase (*rnlA* or gene 63) and polynucleotide kinase (*pseT*) can repair this damage, but in the absence of RNA ligase the cleavage of the tRNA is lethal to T4 protein synthesis. Intriguingly, the *prrC* gene is located between three genes of a type IC restriction cassette. The corresponding proteins are thought to inhibit PrrC RNase activity in uninfected cells.

Two additional exclusion mechanisms involving T4 can be cited. Phage P2 lysogens exclude T4 by two mechanisms: the Tin protein poisons gp32, which is essential for all T4 DNA replication and recombination (794), and the P2 Old protein degrades T4 and lambda DNA from ends, nicks, and gaps unless they are protected by specific proteins (125). It seems more than likely that many other cell-, phage-, and plasmid-encoded mechanisms of T4 exclusion remain to be discovered.

## PREDICTED INTEGRAL MEMBRANE PROTEINS

One approach to exploring the function of the many uncharacterized T4 ORFs is to determine the cellular localization of their products. A number of early T4 studies used cell fractionation and gel electrophoresis to identify membrane-associated phage proteins during infection (reviewed by Harper et al. [385]). There is evidence that the cell envelope continues to be synthesized after infection. The optical density of the infected culture continues to increase substantially under a variety of growth conditions. Freedman and Krisch (291) demonstrated ongoing cell enlargement coupled with gradual arrest of cell division after infection of *E. coli* B in M9 by using a combination of Coulter Counter and dry-weight measurements. The rate of incorporation of diaminopimelic acid remained equal to that of uninfected cells for at least 50 min after infection, supporting the presence of ongoing growth and repair of the peptidylglycan layer. The total phospholipid content continues to rise as rapidly in T4-infected cells as in the uninfected control (as reviewed in reference 385). The rate at which phosphatidylglycerol is synthesized actually increases markedly after infection, while that of phosphatidylethanolamine diminishes, suggesting some changes after infection in basic membrane properties. Synthesis of host membrane proteins appears to be shut off along with that of other host proteins, as observed on two-dimensional gels (189), implying that most new protein in this expanded membrane is either phage encoded or the result of continued processing of already-synthesized host membrane proteins (385).

In recent years, an alternative (bioinformatic) approach to exploring protein function has been developed on the basis of primary sequence data. Computer programs are now available that use various algorithms to predict localization of proteins to the membrane or periplasm. Boyd et al. (105) optimized a statistical approach to determine the probability of an individual protein being integrated into the bacterial inner cell membrane. The calculation of these so-called MaxH values gives two very distinct peaks for proteins from a variety of organisms. These calculations were carried out for the complete set of T4 proteins and ORFs (D. Boyd, E. Thomas, and E. Kutter, Evergreen Int. Phage Biol. Meet., abstr. 11, 1998), using normalization constants determined with the training set of known *E. coli* proteins (105). MaxH values above 1.505 are considered to have a >50% probability of being integral inner membrane proteins. Using this approach, two very distinct peaks are also obtained for T4, with 15 T4 proteins predicted to be integrated into the inner membrane with a probability of over 99%. Three proteins are given a probability of 45 to 95%, three are given

TABLE 6. Potential T4 membrane proteins and their cellular locations

| Protein | Inner membrane | Periplasm | Outer membrane | Membrane domain$^b$ | Size (aa) | Cut site$^c$ | MaxH (%) | Comments$^d$ |
|---------|----------------|-----------|----------------|--------------------|------------|-------------|-----------|--------------|
| t       | 0.516          | 0.000     | 0.000          | 30–46              | 218        |             | 100.0     |              |
| rI      | 0.210          | 0.000     | 0.000          | 2–18               | 97         |             |           | Not M        |
| 7       | 0.297          | 0.000     | 0.000          | 890–906            | 1,032      |             | 99.6      |              |
| 29      | 0.425          | 0.000     | 0.000          | 237–253            | 590        |             | 96.0      |              |
|         |                |           |                | 327–343            |            |             |           |              |
| Ac      | 0.700          | 0.000     | 0.790          | 5–27               | 51         | 32, 19      | 100.0     | M, SOSUI: IM |
|         |                |           |                | 29–48              |            |             |           | lpp          |
| Imm     | 0.000          | 0.926     | 0.181          | 7–29               | 83         | 54          | 100.0     | M, SOSUI: IM |
|         |                |           |                | 41–63              |            |             |           |              |
| 52.1    | 0.359          | 0.000     | 0.000          | 23–39              | 46         | 15          | 1.3       |              |
| Ndd.3   | 0.196          | 0.000     | 0.000          | 3–19               | 26         |             | 100.0     |              |
| Ndd.4   | 0.750          | 0.000     | 0.000          | 16–32              | 42         |             | 100.0     |              |
| e.2     | 0.219          | 0.000     | 0.000          | 82–98              | 102        |             | 100.0     |              |
| e.3     | 0.230          | 0.000     | 0.000          | 32–48              | 120        | 25          | 100.0     |              |
|         |                |           |                | 57–73              |            |             |           |              |
| e.4     | 0.650          | 0.000     | 0.000          | 26–42              | 130        |             | 100.0     |              |
|         |                |           |                | 57–73              |            |             |           |              |
| tRNA.4  | 0.624          | 0.000     | 0.000          | 38–54              | 61         | 22          | 100.0     |              |
|         |                |           |                | 9–31               |            |             |           |              |
| 55.8    | 0.561          | 0.000     | 0.000          | 26–42              | 70         | 24          | 100.0     |              |
| PseT.3  | 0.387          | 0.000     | 0.000          | 8–24               | 117        |             | 100.0     |              |
| PseT.2  | 0.000          | 0.000     | 0.000          | 1–21               | 98         |             |           | Not M, P     |
| 47.1    | 0.338          | 0.000     | 0.000          | 30–46              | 46         | 19          | 100.0     | Not S        |
| NrdC.7  | 0.200          | 0.000     | 0.000          | 110–126            | 133        | 23          | 16.0      |              |
| Imm.1   | 0.700          | 0.000     | 0.790          | 125                | 19, 14     |             | 1.0       | lpp          |

| Protein | Inner membrane | Periplasm | Outer membrane | Membrane domain$^b$ | Size (aa) | Cut site$^c$ | MaxH (%) | Comments$^d$ |
|---------|----------------|-----------|----------------|--------------------|------------|-------------|-----------|--------------|
| sp=rIV  | 0.000          | 0.935     | 0.274          |                    | 97         | 22          |           | Not M        |
| MotA.1  | 0.000          | 0.765     | 0.191          |                    | 49         | 31          | 45.0      |              |
| Ndd.6   | 0.000          | 0.926     | 0.174          |                    | 28         | 23          |           |              |
| Ndd.2a  | 0.000          | 0.753     | 0.146          |                    | 40         | 17          |           |              |
| Ndd.2   | 0.000          | 0.266     | 0.934          |                    | 36         | 32          |           | Not M        |
| Ndd.5   | 0.000          | 0.132     | 0.922          |                    | 32         | 25          | 93.0      |              |

$a$ The PSORT predictions for each site are given in a decimal form, 0 to 1.0, that is related to the strength of the prediction but is not an actual probability; for a given protein, they never add up to 1 for the sum of the different sites. All of these proteins show a “0” prediction for the cytoplasm.

$b$ Amino acids in the predicted membrane domain of each protein are listed.

$c$ Cut site refers to the position(s) at which the predicted leader peptide would be cleaved.

$d$ not M, P, S: not predicted by MaxH, PSORT or SOSUI, respectively. M, SOSUI, inner membrane location predicted by these programs. lpp, lipoprotein. See the text for the URL and description of the output for each program.

other known phage holins, as extensively reviewed (886, 1145). Analysis of *t* mutants with a rapid-lysis phenotype (*rV* mutants) indicates that various segments throughout much of gpt are involved in the phenomenon of lysis inhibition (discussed above) (236). Biochemical confirmation has been provided that gpt has a single N-terminal transmembrane domain and that the bulk of the protein lies in the periplasm (886, 887). Thus positioned, T4 holin apparently receives a signal that additional phage are trying to enter the already-infected cell; lysis inhibition is thus an effective strategy to promote exclusion of other phages and optimal accumulation of T4. This extra role for gpt in receiving the signal for lysis inhibition may explain why it is larger than other holins.

gp rI (97 aa), the other T4 product required for lysis inhibition, is the apparent signal transducer (851, 888). Cloned *rI* can produce lysis inhibition in lambda-infected cells if the lambda holin gene has been replaced by the T4 *t* holin gene. It is classified by both PSORT and TMpred as an inner membrane protein with one TM domain at residues 1 to 22, but its MaxH value of 1.40 puts rI slightly below the threshold of membrane proteins, with a probability of \(10^{-7}\) of being in the inner membrane. The SignalP program was used to predict that gprI is a periplasmic protein (851), and SOSUI assigns the one potential TM domain as a signal peptide. The uncertainty revolves around whether or not this signal peptide is cleaved. Unpublished experiments (E. Ramanculov and R. Young, personal communication) were inconclusive in determining whether it is in the membrane or in the periplasmic space, due to the extreme instability of the rI protein; clearly, more experimental work is needed.

Ac (51 aa) confers resistance to acridine dyes, and its gene served as an important genetic marker in early phage crosses (894). It is a clear, if not typical, membrane protein according to MaxH and SOSUI analysis. However, PSORT suggests that the potential signal peptide is cleaved so as to add an N-terminal lipid and classifies it as a lipoprotein, giving it equally high probabilities of being in the inner or outer membrane. SignalP suggests it may be cleaved between residues 18 and 19.

Two baseplate proteins, gp7 (1032 aa) and gp29 (590 aa), have MaxH values predicting a 99.6% and 96% probability, respectively, of being integral membrane proteins. PSORT, SOSUI and TMPred all predict that gp7 has a single C-terminal TM domain and that gp29 has a pair of TM domains in the middle (Table 6). As mentioned above, the baseplate is assembled on the membrane. gp7 and gp29 are involved in initiating wedge and hub assembly, respectively, and gp29 later becomes the tail-length calibrator. The baseplate remains attached to the inner cell surface (via 300Å fibers from the six corners) throughout tail morphogenesis and the start of lysis (1003), and antibody studies have localized gp7 to the outer corners of the baseplate. This location is consistent with the C-terminal region of gp7 being the observed fibrous attachment structures; at 1,032 aa, gp7 is the second largest T4 protein, comparable in size to the two main tail fiber proteins.

### Hypothetical Proteins with Predicted Cell Membrane Associations

The T4 genome has three apparent gene clusters that encode predicted envelope proteins (Table 6). One of these lies between *rII B* and 52, a region of very short genes and ORFs, including *ac*, where it was difficult to determine gene and ORF assignments using standard computational methods. Other proteins encoded by this region and predicted by MaxH to be membrane proteins include 52.1 (51 aa, N inside the cell), Ndd.2a (40 aa, C inside but atypical in its structure), Ndd.4 (42 aa, N inside), and possibly MotA.1, Ndd3 (26 aa), Ndd.5 (32 aa, N inside), and Ndd.6 (28 aa, C inside). Ndd.2a, Ndd.6, and MotA.1 are characterized by PSORT as probable periplasmic/possible outer membrane proteins, while PSORT gives Ndd.2 and Ndd.5 over a 90% chance of being in the outer membrane. The experimental work of Harper et al. (385) suggested that at least two genes in the *ac* region are required for the stimulation of phosphatidylglycerol biosynthesis following T4 infection; one of them is required for ongoing phospholipid synthesis after infection. Their identity has yet to be determined, but some of these predicted membrane proteins may well be involved. Ndd (for “nuclear disruption deficient”) has often been assumed to be a membrane protein, due to its observed role in binding the bacterial DNA to the membrane. However, its MaxH score is only 1.22, and PSORT and SOSUI localize it to the cytoplasm. Nucleoid binding may actually involve the combined function of multiple proteins from this transcriptional unit, which extends from *ndd.4* through *motA.1*.

The second cluster consists of significantly larger predicted membrane proteins and is located in the *e* (lysozyme)-*tRNA* region, the general region where the still-unpositioned “star” plaque mutants *stI* and *stII*, affecting lysis timing, were mapped (583, 1208). gpe.3 (120 aa) and gpe.4 (130 aa) both clearly have two potential TM domains, with N and C inside the cell; gpe.4 is one of the few T4 proteins that looks like a typical membrane protein. PSORT calls the N-terminal helical domain of gpe.3 a cleavable signal peptide, which would leave only one transmembrane domain in the final protein. gpe.2 (102 aa) is predicted to be an integral inner membrane protein, although it is otherwise very hydrophilic. gptRNA.4 (61 aa) is also predicted by all four programs to have two transmembrane regions.

The third possible cluster is a compact eight-gene operon that starts with *cd.3*. PseT.3 (117 aa, with an apparently uncleavable signal peptide TM) is encoded in this cluster and is predicted by all programs to be an integral membrane protein. The adjacent ORF, *pseT.2* (99 aa), is also predicted by SOSUI (but not the other programs) to encode a membrane-associated protein, with a potential signal peptide at residues 1 to 21. PSORT, SOSUI, and TMPred all suggest that the Cd.3 protein may also be in the inner membrane, although the scores are barely above the cutoff in each case.

Three other isolated ORFs are also generally predicted to encode integral membrane proteins. These are 47.1 (46 aa), 55.8 (70 aa, very hydrophobic) and NrdC.7 (133 aa, with a cleaved signal peptide as well as one C-terminal TM domain). ORF 47.1 overlays a middle promoter and encodes a protein of only 46 aa. Because gp47.1 has a correlation coefficient of only 0.30, it was earlier dropped from the list of probable T4 proteins. It has been reinserted on the basis of this analysis. It could function as the membrane anchor for the gp46/47 nuclease earlier reported to be a membrane protein (345, 731).

### Missing Membrane-Associated Proteins

Notable by their absence from this list of predicted integral membrane proteins are several proteins that were classified as

membrane associated on the basis of early experimental work (reviewed in reference 385). Huang (446) performed the most extensive gel analysis to date of proteins that are enriched in the membrane fraction following differential centrifugation. Most that were observed have not yet been identified genetically. The MaxH and PSORT scores and other predicting programs all give zero integral membrane probabilities for most known proteins that were identified as membrane associated in those studies, including gprIIA, gprIIB, Ndd, and gp46. gp52 has a slight potential for one TM domain near the center but is still classified by all of the programs as cytoplasmic. It is important to test more carefully the connection that these proteins have to the membrane, a relationship that was observed by several groups. The current analysis would predict that they are likely to be bound peripherally, perhaps through other proteins, lipid, or DNA. Some of the small membrane proteins cotranscribed with 46, ndd, and 52 could be involved in such attachments, as suggested above.

The use of multiple programs provides complementary insights when using predictive algorithms. Experimental work is required to assess the significance of the predictions by the various programs for TM domains and other membrane associations for these T4 proteins. However, the analysis suggests a starting point for determining the functions of a number of otherwise uncharacterized ORFs and should aid the study of cell expansion and membrane changes during T4 infection. The clustering of many predicted membrane-associated proteins is consistent with the organization of other functional groups of T4 genes.

## EVOLUTIONARY PERSPECTIVES: T4 PROTEINS AND THE GENOME

Extensive functional, mutational, and structural data on a number of the T4 proteins provide an excellent framework for advancing the study of protein evolution. Many of the DNA metabolism and replication enzymes of T4 have orthologous proteins represented in all domains of life, which is why the biochemical and structural studies of T4 proteins have been so broadly relevant. Growing knowledge about specific genes, complete genomes, and the proteomes of at least a few T4-related bacteriophages are beginning to make possible comparative genomics studies that impact our understanding of the well-studied T4 systems and broaden our perspectives for other organisms. In the sections below, we briefly summarize the reported structural and evolutionary relationships of T4 proteins and provide some evolutionary reflections on the T4 genome and that of its relatives. A more thorough discussion of the evolutionary relationships among T4-related phages will appear elsewhere (E. Thomas, F. Zucker, and E. Kutter, unpublished data).

### T4 Protein Structures

Structural studies of T4 proteins began with the crystallization and three-dimensional structure determination of gpe (lysozyme) (705, 1158). T4 lysozyme is an excellent example of structural analysis and targeted amino acid replacements used hand-in-hand to unravel an enzyme's catalytic properties and protein conformation (594, 1193). Solving the structure of the

TABLE 7. T4 proteins in the structure database

| Protein       | Description                                      | Protein Data Bank entry |
|---------------|--------------------------------------------------|-------------------------|
| AsiA          | Anti-σ⁷⁰ regulatory protein                      | 1JR5, 1KA3              |
| β-gt          | β-Glucosyltransferase                           | 1QKJ                   |
| DenV          | Pyrimidine-dimer excisionase                    | 2END, 1VAS              |
| gpe           | Lysozyme                                         | 1LYD                   |
| I-TevI        | Intron-homing endonuclease                      | 1I3J                   |
| MotA          | Transcription regulatory factor                  | 1BJA, 1I1S             |
| NrdC          | Glutaredoxin, thioredoxin                       | 1ABA, 1DE1             |
| NrdD          | Anaerobic NTP reductase, large chain            | 1H77                   |
| RegA          | Translation regulatory protein                   | 1REG                   |
| Rnh           | RNase H                                          | 1TFR                   |
| TS            | Thymidylate synthase                            | 1TIS                   |
| Wac           | Fibritin deletions E and M                      | 1AAO, 1AVY             |
| gp1           | dNMP kinase                                     | 1DEK                   |
| gp5/27        | Tail-associated lysozyme                        | 1K28                   |
| gp9           | Long-tail fiber connector                       | 1QEX                   |
| gp11          | Baseplate-short-fiber connector                 | 1EL6                   |
| gp12          | Short tail fiber                                | 1H6W                   |
| gp31          | Cochaperone                                     | 1G31                   |
| gp32          | ssDNA-binding protein                           | 1GPC                   |
| gp42          | dCMP-hydroxymethylase                           | 1B5D                   |
| gp43          | T4 DNA polymerase fragment, RB69                | 1NOY, 1WAF             |
|               | DNA polymerase                                  |                         |
| gp45          | Processivity clamp                              | 1CZD                   |
| gp49          | EndoVII                                         | 1E7D                   |
| gp59          | Helicase assembly protein                       | 1C1K                   |
| nrdD intron   | Group IA intron RNA/ribozyme                    | 1SUN                   |

T4-related RB69 phage DNA polymerase (1146), when the T4 enzyme has been refractory to crystallization, has permitted a full appreciation of the structural and catalytic effects of the numerous available mutations in T4 gene 43 (DNA polymerase) (40, 507, 827, 906). Currently, the structures of 23 T4 proteins, protein domains, and protein complexes are deposited in the Protein Data Bank (Table 7; http://www.rcsb.org/pdb/). In addition to protein structure studies, an RNA-folding model (pdb entry 1SUN) predicts how the 3'-terminal domain of the RNA stabilizes the intron core (468, 730). Each of these structures provides a framework for functional and evolutionary analysis of the respective protein or molecular machine in which it participates. Evolutionary relationships between macromolecules, whether phage or cellular, will be fully appreciated only in the context of their structures, so that the yield from structural studies of additional T4 proteins would appear to be high.

#### Orthologous T4 Proteins

T4 proteins involved in nucleotide and nucleic acid metabolism typically show sequence similarity to functionally related enzymes of other organisms (69). Proteins that have orthologs in the database are often members of multientry clusters of orthologous groups (COGs) curated at the National Center for Biotechnology Information (NCBI) (http://www.ncbi.nlm.nih.gov/COG). The T4 genome page (NC_000866) provides access to each protein and its respective COG. Proteins involved in DNA transactions (UvsX, Topo II, NrdD, Td, and many others) are present in organisms across the phylogenetic tree and have dozens if not hundreds of entries. However, only 28 of 274 T4 proteins can be grouped into clusters of related

unique to archaeal thymidylate synthases, which are sufficiently different from those of bacteria and eukaryotes that they are more difficult to align unequivocally. Figure 11 also shows the distant relationship between thymidylate synthases and T4 HMase (gp42; also see pdb 1B5D). This is not too surprising since both enzymes catalyze the transfer of methyl (or hydroxymethyl) to the same position on a pyrimidine monophosphate (dUMP and dCMP, respectively).

The T4 dihydrofolate reductase and the three topoisomerase II components (gp39, gp52, and gp60) also appear to have diverged before the separation of prokaryotes and eukaryotes. This is seen in both the clade patterns and the clear interspersion of sequences uniquely conserved among eukaryotes between ones that are characteristic of prokaryotes.

Other T4 proteins are more closely related to bacterial proteins. These include thymidine kinase (tk), DNA adenine methylase (dam), and the ribonucleotide reductases. The T4 anaerobic NTP reductase (NrdD) and its copeptide, NrdG, are most closely related to the *E. coli* proteins; however, even these two enzymes appear to have diverged from their host counterparts well before the separation between the *Haemophilus influenzae* and *E. coli* enzymes.

T4 DNA polymerase aligns with the B family of DNA polymerases, which includes archaeal, eucaryal, bacterial, and viral enzymes (273). Included in this group are pol II enzymes of γ-proteobacteria (such as *E. coli*), involved in DNA repair, and DNA polymerases of *Saccharomyces*, herpesvirus, and chlorella virus. Interestingly, the T4 DNA polymerase is most closely related to polymerases of the archaeal halophile *Halobacterium* sp. and two of its viruses, HF1 and HF2 (273). These relationships also emerged in the crystal structure of the DNA polymerase of the T4-related phage RB69 DNA (507, 1146) and have been functionally confirmed. A mutation introduced into yeast DNA polymerase (POL3) on the basis of the mutator properties of an altered T4 DNA polymerase gave a yeast mutator phenotype (370). Moreover, gp44, a subunit of the DNA polymerase clamp loader, is orthologous to eukaryotic replication factor C, as discussed above (see “DNA metabolism, replication, recombination, and repair”). Its highest-scoring homology is 29% identity, over its entire 319-residue length, to the *Archaeoglobus fulgidus* replication factor.

Primary sequence homologies to eukaryotic viruses are also observed. For example, T4 DNA ligase (gp30) is most homologous to the DNA ligase of the African swine fever virus (25% identity over a conserved region of 229 of its 487 aa). It also shares smaller conserved regions with archaeal DNA ligases (such as 27% identity over 154 aa for *Methanobacterium thermoformicum*). Two T4 proteins have a particularly interesting homology to an insect viral protein. PseT (the 5′ polynucleotide kinase-3′ phosphatase) and RnlA (gp63 RNA ligase) are similar in amino acid sequence to the two halves of ORF86 in the *Autographa californica* nuclear polyhedrosis virus (246). Together, *rnlA* and *pseT* look somewhat like the tRNA splicing machinery in eukaryotes. The *A. californica* nuclear polyhedrosis virus ORF86 protein has been named Pnk/Pnl to reflect its relationship to the T4 enzymes and its motifs that are characteristic of polynucleotide kinase and RNA ligase. There is generally no need for tRNA splicing machinery in T4, since its tRNA genes contain no introns. However, the ligase and

FIG. 10. Structure of T4 thymidylate synthase. The T4 sequence was aligned with other available thymidylate synthases, with the invariant regions colored in red and the regions in which the T4 enzyme is different from all others colored in yellow. The latter regions are largely hydrophilic for most thymidylate synthases but are hydrophobic for the T4 enzyme (which may facilitate its incorporation into the nucleotide-synthesizing complex). These regions were not included for the predicted evolutionary tree in Fig. 11. Structural coordinates are from reference 274 and were used to create this figure. Also see reference 143.

phage proteins. These relationships among viral and phage proteins, including proteins of T4, are now cataloged by NCBI at a dedicated website (http://www.ncbi.nlm.nih.gov/PMGifs/Genomes/crp_start.html).

One of the most interesting and potentially instructive examples of an orthologous protein is thymidylate synthase (td or TS). A number of stretches of amino acids are highly conserved between *Bacteria*, *Eucarya* and T4, facilitating precise alignment and analysis; these are indicated in red on the crystal structure of the T4 enzyme shown in Fig. 10 (also see pdb 1TIS). The two stretches indicated in yellow are totally different between the T4 enzyme and all other thymidylate synthases; these regions are largely hydrophobic in T4 but hydrophilic in other members of the family. They lie on the surface of the enzyme, where presumably they are involved in the interaction between thymidylate synthase and other enzymes of the nucleotide synthesis complex described above. When these two segments are excluded and the core regions are used for alignment, the phylogenetic relationship shown in Fig. 11 is obtained. The tree suggests that the T4 enzyme branched off somewhere before the split between *Eucarya* and *Bacteria*. The apparently ancient branch point is not just due to faster evolution of viral proteins than of proteins of their hosts, since, for example, herpesviruses appear to branch off much later in the *Eucarya* lineage, just before the separation between the human and rat-mouse lines. Also, T4 TS has several regions that are characteristic of the eukaryotic enzymes, intermixed with others that seem to be unique to the bacterial sequences. T4 TS also has one sequence near the N terminus that is otherwise

MILLER ET AL.
MICROBIOL. MOL. BIOL. REV.

Methanobacterium TS  
Methanobacterium HMase dUMP  
Methanococcus TS  
Methanococcus HMase dUMP  
T2 HMase dCMP  
T6 HMase dCMP  
T4 HMase dCMP  

E. coli TS  
Shigella TS  
Mycobacterium TS  
Neisseria TS  
B. subtilis TS B  
Staphylococcus TS  
Mycoplasma TS  
Mouse TS  
Rat TS  
Human TS  
Herpes TS  
Arabidopsis TS  
Soybean TS  
Yeast TS  
Trypanosome TS  
Pneumocystis TS  
T4 TS  
B. subtilis TS A  
Phage phi-3T TS  
Phage SPO1 HMase dUMP  

FIG. 11. Phylogenetic tree of thymidylate synthases and deoxynucleotide hydroxymethylases. All protein sequences were obtained from the public databases. Alignment and tree construction were done by the methods of Feng and Doolittle (271).

kinase activities are both required in host strains with the restriction system carried by the cryptic DNA element, *prr* (see above) (515).

### Paralogous Genes in the T4 Genome

The T4 genome contains a few apparent duplications that have evolved into separate functional genes. These include *modA-modB* (781) and genes 9–10 and 23–24 (225), as well as a duplication of lysozyme (gene e) found inserted into the baseplate protein gp5 (787). This lysozyme insertion, although present in gp5 of coliphages RB49 and RB69, is not seen in gp5 of the *Vibrio* phage KVP40 (Miller et al., submitted). Duplications in the T4 genome have also been explored by evaluating structural similarities in proteins with low-level sequence homologies (T. Kawabata, K. Nishikawa, F. Arisaka, and E. Kutter, unpublished data). Potential duplications were identified in the pairs of proteins encoded by the adjacent genes 26–51, *rnH-34*, 49-*nrdD*, *dexA.1-dexA.2*, and *tk.1-tk.2*. FASTA alone indicated relationships between gpe.3 and gpe.4 and between gpAlt.-1 and Alt.-2 and the N-terminal region of Alt, the latter apparently reflecting an ancient duplication followed by insertion of a new internal start codon to generate this pair of ORFs. Noted similarities were also seen between gp34, gp37, and gp12, all of which form trimeric β-helix fibers. Internal repeats were seen in gp34 (40 aa, 7 X), gp12 (25 aa, 5 X), hoc (94 aa, 3 X) and gp46 (100 aa, 2 X), where X indicates the number of times each motif is repeated.

#### A Glimpse at Genome Diversity and Evolution in T4-Type Phages

Bacteriophages may well be the most numerous living entities on Earth; about 10 times as many phages as bacteria have been seen in ocean samples, leading to a total estimate of about \(10^{32}\) phages on Earth (62, 1184). There is interest in how viruses arose, how they acquire their special properties and genes, and how they relate to each other and to cellular genomes.

Botstein (100) presented evidence that lambdoid phages are a mosaic of ordered sets of modules, each of which may have come from a particular host, plasmid, or other phage. This concept has since been extended to other temperate phages, and a lambdoid “supergroup” seems to extend even into gram-positive bacteria (128, 149, 209, 400). For example, L5- and ψM1-like phages, which infect mycobacteria and *Archaea*, respectively, and are members of the Siphoviridae family, show a distant but still detectable similarity in their genome organization with lambdoid phages (209). Another genus of Siphoviridae, the c2-like *Lactococcus* phages, differs a good deal from the lambda pattern of structural gene organization but could still be aligned with the lambda-like *Lactococcus* phage sk1 when allowing for two genome rearrangement events (117, 118).

Nonetheless, “modular mosaicism” does not appear to be an appropriate characterization for the genomes of T4-like phage. Genome sequence surveys of T4-like phages (210, 700, 702, 919, 1067, 1069) suggest that T4 and its relatives are largely in a group by themselves, undergoing few exchanges with other phage families. Sequence homology is observed between the

receptor-binding region of the tail fibers of the T-even group and other phages, apparently reflecting the selection for new receptor-binding capabilities and the special recombinogenic properties of this region (1067). BLAST analysis between T4 and P1 proteins (M. Lobocka and M. Yarmolinsky, personal communication) reveals some homologies in the tail tube, baseplate, and two DNA metabolism enzymes. Nonetheless, there is no evidence for ongoing exchange events between T-even phages and other phages that involve entire gene cassettes or modules; exchanged segments are usually the size of single genes or smaller. The high frequency of recombination observed for T4, its lytic developmental program, and the presence of multiple promoters throughout the genome allow for many independent exchanges of smaller genome segments, apparently precluding extensive selection for large modular genetic units.

The DNA endonucleases or “homing enzymes” in group I introns of T-even phage could afford one mechanism for gene dissemination. However, Edgell (251) discusses reasons why this appears not to occur. The related T4 Seg and Mob endonucleases may also either disseminate or exclude targeted regions of the phage genome (51, 335). The presence in phages of gram-positive bacteria of ribonucleotide reductase genes (*bnrdE* and *bnrdF*) with T4-like group I introns (626) may suggest that introns and genes with introns (see the discussions above on homing endonucleases) can be transferred between phage of major phylogenetic divisions.

Phages with T4 morphology have been isolated from a variety of sources throughout the world: sewage plants, coastal and offshore seawater, zoos, and diarrhea patients in the former Soviet Union and the United States (8, 247, 379, 596, 703, 946, 1233). Members of the family have been found infecting several different gram-negative bacteria (8). The T4-like phages infecting a range of host genera generally show conservation in gene order and protein sequence for the essential units—capsid and tail structures and DNA replication proteins. Since T4 has until very recently been the only fully sequenced member of its myovirid subgroup, with just short genomic regions of sequence data obtained from other phage by PCR analysis or localized cloning (485, 748, 919, 1217), the evolutionary relationships of only a few specific genes (e.g., 56 and the head and tail genes) have been studied in detail (777, 1069).

During the completion of this review, progress on genomic sequencing of T4-type phages has been achieved. The genomes of phage that infect γ-proteobacteria (*E. coli* phage RB69 and RB49; *Aeromonas* phages Aeh1, 44RR2, and 65; *Vibrio* phage KVP40; *Acinetobacter johnsonii* phage 133 β-proteobacteria), (*Burkholderia cepacia* phage 42), and cyanobacteria (*Synechococcus* phage S-PM2) (210, 379, 700, 1069; H. Krisch, J. Karam, et al., personal communication; Miller et al., submitted) are all under study. Many have dsDNA genomes similar in size to T4, while some, such as KVP40 (Miller et al., submitted) have longer heads, filled by substantially larger genomes (e.g., 245 kbp). For these different T4-like phages, ClustalW and neighbor-joining bootstrap analysis of capsid and tail tube proteins led to similar phylogenetic trees with relationships paralleling those of the host bacteria (379, 1069).

Current sequencing projects reveal that the genomes of some T4-type phage infecting *E. coli* are arranged much like

that of T4 (such as RB69; see http://phage.bioc.tulane.edu), although group I introns, *seg* genes, and other characterized nucleotide metabolism genes are absent. For other T4-like coliphages (such as RB49), there are yet fewer of the “nonessential” T4 genes and more hypothetical, uncharacterized ORFs (210). For RB49, conservation of the DNA replication and virion structural proteins is most evident. A related situation is seen for the *Vibrio* phage KVP40. The DNA replication and structural proteins most closely align with those of T4 (ca. 35 to 70% similar), but this represents only about 20% of the ORFs, with more than 60% still having no characterized function (Miller et al., submitted). The same pattern was observed in the genome sequence of *Bacillus* phage SPβc2; 75% of its predicted ORF products have no significant homologies to proteins in the databases (625). One of the more extreme examples to date is the 280,334-bp genome of *Pseudomonas aeruginosa* myoviridae phage phiKZ, with only 59 of its 306 ORF products aligning with proteins of known function in the database (727); fully 80% of the proteins it encodes appear not to have been characterized in any organism. It seems likely that the more conserved DNA replication and virion genes of T4-like phages are the ancient genes. The few complete phage genome sequences that are available, which can have 50 to 80% of the ORFs as unique, suggest the presence of a separate, more variable group of genes in each genome. However, our overall view of phage genomes is still very limited, where the small number of BLAST hits with phage proteins in part reflects the paucity of complete phage genome sequences in the databases.

No phages morphologically identical to T4 have been identified as infecting gram-positive bacteria. *Bacillus subtilis* phage SP01, containing dsDNA (ca. 140 kbp) with HMU in place of thymine, somewhat resembles T4 (47-nm-diameter head and 142-nm-long sheathed, contractile tail) and has several DNA metabolism and replication enzymes similar to those of T4. The sequenced dsDNA genome of the temperate *Bacillus* phage SPβc2 (accession no. NC_001884 and AF020713; 134,416 bp) (625) has a few DNA metabolism enzymes that are present in T4, and its ribonucleotide reductase genes harbor the T4-like group IA2 introns (624). Clearly, genomic sequences of large lytic phages infecting gram-positive bacteria and others from across the phylogenetic tree are of great interest. The return on phage genome sequencing would appear to be highest for comparative functional and structural biochemistry on the individual gene products and for new protein and RNA resources for biotechnology, such as novel antimicrobials, therapeutics, and diagnostic reagents.

---

### OUTLOOK

T4, with its legions of investigative disciples over the last 50 years, has provided us with a beautifully integrated system of biological machines and networks (506, 697). The otherwise elaborate biochemical perspective on the fully sequenced T4 genome provides a vast resource for phage genomics and the future of phage biology. Among the large, ubiquitous group of tailed, T4-like phages found on Earth (8), phage T4 has been the most extensively studied. T4 biology will “bootstrap” us to the recognition of similar biochemical processes in related phage while identifying genes and proteins that are novel to

each. It is already clear from emerging genome sequences that although some of the uncharacterized ORFs in T4 are shared in other phages (i.e., RB49, RB69, and KVP40), each has a relatively large proportion of individually unique ORFs. Just in the past year, the identities of uncharacterized T4 ORFs have been revealed: gene 69 (*segF*) and ORF 32.1 (*segG*) encode DNA endonucleases (51, 655), ORF e.1 (*nudE*) encodes a Nudix hydrolase (1204), and ORF 24.1 (*rnlB*) encodes a second T4 RNA ligase related to the RNA-editing enzymes of trypanosomes and those present in sequenced *Archaea* (426). T4 will be a handy genetic and biochemical tool for detailed studies of their respective enzymatic activities. As with any fully sequenced genome, strategies (beyond BLAST alignments) are needed to explore the function of the novel phage ORFs. T4 should be a sound model for dissecting the emerging genome sequences of related phages while itself continuing to provide new insights into gene function and phage metabolism with relevance across phylogenetic domains.

Many of the unresolved problems in T4 biology reflect the subtlety of the process or the demands on the researcher for sample preparation, timing, or control of growth conditions. We still do not fully understand the process of DNA entry during infection, the very early shutoff of host gene expression, and the relevance or mechanism of T4-directed disruption of the host nucleoid. Moreover, the effects of hydroxymethylated and glucosylated cytosines in T4 DNA on DNA-protein interactions important for transcription, DNA replication recombination, repair, and restriction remain to be determined. To achieve this, the availability of T4-like modified phosphoramidites and of α and β glucosyltransferases for synthesis of modified DNA would be a major asset in addressing the recognition of early, middle, and late promoters by variously modified RNAP and the relevant activator proteins. Similarly, the potential roles of ModB modifications of the host translation proteins and the genetically still unidentified ribosomal alterations and the role of the membrane in transcription, replication, and capsid assembly are all in need of additional biochemical studies.

One of the more elegant aspects of T4 biology has been the elucidation of the assembly mechanism of its supramolecular structure. To construct such a huge, complicated and intricate structure, a number of intriguing molecular tricks have evolved, such as a scaffold, DNA-packaging apparatus, a ruler molecule, and phage-encoded molecular chaperones (e.g., gp31). Details of this process are still probably hidden in the genome. Additional study on the high-resolution structure of the particle will eventually elucidate how a series of structural changes (conformational change of the baseplate, contraction of the tail sheath, and DNA ejection) take place at atomic resolution.

Practical applications of phage and phage gene products should continue to emerge. In an era of increasing bacterial antibiotic resistance, there is renewed interest in the therapeutic applications of phage in the treatment of infectious disease (37a, 83, 141, 725, 866, 1052a; www.evergreen.edu/phage). It was Delbrück and the initial American phage group who selected three of the seven virulent coliphages—T2, T4, and T6—from among early, largely “therapeutic” isolates. Even today, one of the best sources of T-even-like phages is the stools of patients recovering from dysentery (L. Gachechiladze

and H. Brussow, personal communication). Specific proteins, phage lysins in particular, have been proposed as useful enzymes for killing troublesome bacteria (66, 759). Recently, purified PlyG lysin (an *N*-acetylmuramoyl-ʟ-alanine amidase), produced by gamma phage of *Bacillus anthracis*, was shown to effectively kill the bacterium (967). T4 and its relatives will probably yield novel products that target various cellular processes, inhibiting or killing their host bacteria.

Many of the major enzymes of molecular biology came onto the scene with T4, yet there are few laboratory reagents that derive from other phages beyond the well-studied isolates. There is every reason to expect that enzymes with unique catalytic parameters will emerge from genome sequences of other phages.

Phage are also an excellent teaching tool. They are easy to work with, so students can learn the simple methods required and get meaningful results quite easily. Phage research also calls for integrating broad areas of microbial physiology, biochemistry, biophysics, genetics, and molecular biology. The analysis presented here of the T4 genome and its relationship to phage biochemical processes, ecology, and evolution is based on the work of many students of many ages and countries. One can only hope that the scientific community will continue to take advantage of the historically large investment of intellectual and fiscal resources committed to T4 and will continue to explore the vast, wonderful world of phage biology.

### ACKNOWLEDGMENTS

We express our appreciation to the entire T4 community for their help through the years in the assembly and analysis of the T4 genome sequence. Special thanks go to Burt Guttman, Tsotne Djavachishvili, Nino Mzhavia, Elena Marusich, Tom Stidham, Elizabeth Thomas, Raul Raya, and Judy Cushing at Evergreen; to Tim Dean at NC State; to Vadim Mesyanzhinov and his students in Moscow; and to Tim Hunkapillar, Frank Zucker, Mark Borodovsky, Tom Schneider, Fred Blattner, and Gary Stormo for various computational collaborations.

Work at Evergreen was supported by a series of NSF grants from the Microbial Genetics, Biological Database and Collaborative Research at Undergraduate Institutions (CRUI) programs. G.M. thanks the National Science Foundation and the Vanderbilt Ingram Cancer Center for support. W.R. thanks the DFG and the MBF NRW for many years of financial support, past group members of Molecular Genetics at the Ruhr-University, and the groups of R. S. Nivinskas (Institute of Biochemistry, Vilnius, Lithuania), P. S. Freemont (Imperial Cancer Research Fund, London, United Kingdom), S. Moréra (Laboratoire d’Enzymologie et Biochimie Structurale, Gif-sur-Yvette, France), and R. R. Schmidt (Fachbereich Biochemie, University of Konstanz, Konstanz, Germany) for their cooperation in unveiling the secrets of T4.

### REFERENCES

1. Abdus Sattar, A. K., T. C. Lin, C. Jones, and W. H. Konigsberg. 1996. Functional consequences and exonuclease kinetic parameters of point mutations in bacteriophage T4 DNA polymerase. Biochemistry **35**:16621–16629.
2. Abedon, S. T. 1999. Bacteriophage T4 resistance to lysis-inhibition collapse. Genet. Res. **74**:1–11.
3. Abedon, S. T. 1994. Lysis and the interaction between free phages and infected cells., p. 397–405. *In* J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.
4. Abedon, S. T. 1990. Selection for lysis inhibition in bacteriophage. J. Theor. Biol. **146**:501–511.
4a. Abedon, S. T., T. D. Herschler, and D. Stopar. 2001. Bacteriophage latent-period evolution as a response to resource availability. Appl. Environ. Microbiol. **67**:4233–4241.
5. Abelson, J., K. Fukada, P. Johnson, H. Lamfrom, D. P. Nierlich, A. Otsuka, G. V. Paddock, T. C. Pinkerton, A. Sarabhai, S. Stahl, J. H.

Wilson, and H. Yesian. 1975. Bacteriophage T4 tRNAs: structure, genetics, and biosynthesis. Brookhaven Symp. Biol. 1975:77–88.

6. Abremski, K., and L. W. Black. 1979. The function of bacteriophage T4 internal protein I in a restrictive strain of *Escherichia coli*. Virology 97: 439–447.

7. Abuladze, N. K., M. Gingery, J. Tsai, and F. A. Eiserling. 1994. Tail length determination in bacteriophage T4. Virology 199:301–310.

8. Ackermann, H. W., and H. M. Krisch. 1997. A catalogue of T4-type bacteriophages. Arch. Virol. 142:2329–2345.

9. Adari, H. Y., K. Rose, K. R. Williams, W. H. Konigsberg, T. C. Lin, and E. K. Spicer. 1985. Cloning, nucleotide sequence, and overexpression of the bacteriophage T4 *regA* gene. Proc. Natl. Acad. Sci. USA 82:1901–1905.

10. Adari, H. Y., and E. K. Spicer. 1986. Translational repression *in vitro* by the bacteriophage T4 *regA* protein. Proteins 1:116–124.

11. Adelman, K., E. N. Brody, and M. Buckle. 1998. Stimulation of bacteriophage T4 middle transcription by the T4 proteins MotA and AsiA occurs at two distinct steps in the transcription cycle. Proc. Natl. Acad. Sci. USA 95:15247–15252.

12. Adelman, K., G. Orsini, A. Kolb, L. Graciani, and E. N. Brody. 1997. The interaction between the AsiA protein of bacteriophage T4 and the σ⁷⁰ subunit of *Escherichia coli* RNA polymerase. J. Biol. Chem. 272:27435–27443.

13. Alam, S. L., J. F. Atkins, and R. F. Gesteland. 1999. Programmed ribosomal frameshifting: much ado about knotting. Proc. Natl. Acad. Sci. USA 96:14177–14179.

14. Alberts, B. 1998. The cell as a collection of protein machines: preparing the next generation of molecular biologists. Cell. 92:291–294.

15. Alberts, B., and R. Miake-Lye. 1992. Unscrambling the puzzle of biological machines: the importance of the details. Cell 68:415–420.

16. Alberts, B. M. 1987. Prokaryotic DNA replication mechanisms. Philos. Trans. R. Soc. London Ser. B 317:395–420.

17. Alberts, B. M., J. Barry, B. P. Bedinger, R. L. Burke, U. Hibner, C. C. Liu, and R. Sheridan. 1980. Studies of replication mechanisms with the T4 bacteriophage *in vitro* system. Mechanistic studies of DNA replication and genetic recombination. UCLA Symp. Mol. Biol., 19:449–473.

18. Alberts, B. M., and L. Frey. 1970. T4 bacteriophage gene 32: a structural protein in the replication and recombination of DNA. Nature 227:1313–1318.

19. Allen, S. V., and E. S. Miller. 1999. RNA-binding properties of *in vitro* expressed histidine-tagged RB69 translational repressor protein. Anal. Biochem. 269:32–37.

20. Alley, S. C., E. Abel-Santos, and S. J. Benkovic. 2000. Tracking sliding clamp opening and closing during bacteriophage T4 DNA polymerase holoenzyme assembly. Biochemistry 39:3076–3090.

21. Alley, S. C., A. D. Jones, P. Soumillion, and S. J. Benkovic. 1999. The carboxyl terminus of the bacteriophage T4 DNA polymerase contacts its sliding clamp at the subunit interface. J. Biol. Chem. 274:24485–24489.

22. Alley, S. C., V. K. Shier, E. Abel-Santos, D. J. Sexton, P. Soumillion, and S. J. Benkovic. 1999. Sliding clamp of the bacteriophage T4 polymerase has open and closed subunit interfaces in solution. Biochemistry 38:7696–7709.

23. Alley, S. C., M. A. Trakselis, M. U. Mayer, F. T. Ishmael, A. D. Jones, and S. J. Benkovic. 2001. Building a replisome solution structure by elucidation of protein-protein interactions in the bacteriophage T4 DNA polymerase holoenzyme. J. Biol. Chem. 276:39340–39349.

24. Alley, S. C., et al. 2000. Mapping protein-protein interactions in the bacteriophage T4 DNA polymerase holoenzyme using a novel trifunctional photocrosslinking and affinity reagent. J. Am. Chem. Soc. 122:6126–6127.

25. Ando, R. A., and S. W. Morrical. 1999. Relationship between hexamerization and ssDNA binding affinity in the *uvsY* recombination protein of bacteriophage T4. Biochemistry 38:16589–16598.

26. Ando, R. A., and S. W. Morrical. 1998. Single-stranded DNA binding properties of the *uvsX* recombinase of bacteriophage T4: binding parameters and effects of nucleotides. J. Mol. Biol. 283:785–796.

27. Andreadis, J. D., and L. W. Black. 1998. Substrate mutations that bypass a specific Cpn10 chaperonin requirement for protein folding. J. Biol. Chem. 273:34075–34086.

28. Appasani, K., D. S. Thaler, and E. B. Goldberg. 1999. Bacteriophage T4 gp2 interferes with cell viability and with bacteriophage λ Red recombination. J. Bacteriol. 181:1352–1355.

29. Arisaka, F. 1994. Use of chemical modification and limited proteolysis to study structural changes in the tail sheath protein gp18, p. 329–331. *In* J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

30. Arisaka, F., L. Ishimoto, G. Kassavetis, T. Kumazaki, and S. Ishii. 1988. Nucleotide sequence of the tail tube structural gene of bacteriophage T4. J. Virol. 62:882–886.

31. Arisaka, F., T. Nakako, H. Takahashi, and S. Ishii. 1988. Nucleotide sequence of the tail sheath gene of bacteriophage T4 and amino acid sequence of its product. J. Virol. 62:1186–1193.

32. Armstrong, J., R. S. Brown, and A. Tsugita. 1983. Primary structure and genetic organization of phage T4 DNA ligase. Nucleic Acids Res. 11:7145–7156.

33. Artsimovitch, I., and R. Landick. 2000. Pausing by bacterial RNA polymerase is mediated by mechanistically distinct classes of signals. Proc. Natl. Acad. Sci. USA 97:7090–7095.

34. Asai, T., and T. Kogoma. 1994. D-loops and R-loops: alternative mechanisms for the initiation of chromosome replication in *Escherichia coli*. J. Bacteriol. 176:1807–1812.

35. Augustine, M. L., R. W. Hamilton, M. L. Dodson, and R. S. Lloyd. 1991. Oligonucleotide site directed mutagenesis of all histidine residues within the T4 endonuclease-V gene—role in enzyme nontarget DNA binding. Biochemistry 30:8052–8059.

36. Baker, R. P., and L. J. Reha-Krantz. 1998. Identification of a transient excision intermediate at the crossroads between DNA polymerase extension and proofreading pathways. Proc. Natl. Acad. Sci. USA 95:3507–3512.

37. Barcus, V. A., A. J. B. Titheradge, and N. E. Murray. 1995. The diversity of alleles at the *hsd* locus in natural populations of *Escherichia coli*. Genetics 140:1187–1197.

37a. Barrow, P. A., and J. S. Soothill. 1997. Bacteriophage therapy and prophylaxis: rediscovery and renewed assessment of potential. Trends Microbiol. 5:268–271.

38. Barry, J., and B. Alberts. 1994. Purification and characterization of bacteriophage T4 gene 59 protein. J. Biol. Chem. 269:33049–33062.

39. Barth, K. A., D. Powell, M. Trupin, and G. Mosig. 1988. Regulation of two nested proteins from gene 49 (recombination endonuclease VII) and of a lambda RexA-like protein of bacteriophage T4. Genetics 120:329–343.

40. Bebenek, A., H. K. Dressman, G. T. Carver, S. Ng, V. Petrov, G. Yang, W. H. Konigsberg, J. D. Karam, and J. W. Drake. 2001. Interacting fidelity defects in the replicative DNA polymerase of bacteriophage RB69. J. Biol. Chem. 276:10387–10397.

41. Bebenek, A., L. A. Smith, and J. W. Drake. 1999. Bacteriophage T4 *mh* (RNase H) null mutations: effects on spontaneous mutation and epistatic interaction with rII mutations. J. Bacteriol. 181:3123–3128.

42. Bechhofer, D. H., K. K. Hue, and D. A. Shub. 1994. An intron in the thymidylate synthase gene of *Bacillus* bacteriophage β22: evidence for independent evolution of a gene, its group I intron, and the intron open reading frame. Proc. Natl. Acad. Sci. USA 91:11669–11673.

43. Beechem, J. M., M. R. Otto, L. B. Bloom, R. Eritja, L. J. Reha-Krantz, and M. F. Goodman. 1998. Exonuclease-polymerase active site partitioning of primer-template DNA strands and equilibrium Mg²⁺ binding properties of bacteriophage T4 DNA polymerase. Biochemistry 37:10144–10155.

44. Beernink, H. T., and S. W. Morrical. 1998. The *uvsY* recombination protein of bacteriophage T4 forms hexamers in the presence and absence of single-stranded DNA. Biochemistry 37:5673–5681.

45. Behme, M. T., and K. Ebisuzaki. 1975. Characterization of a bacteriophage T4 mutant lacking DNA-dependent ATPase. J. Virol. 15:50–54.

46. Belanger, K. G., and K. N. Kreuzer. 1998. Bacteriophage T4 initiates bidirectional DNA replication through a two-step process. Mol. Cell 2:693–701.

47. Belanger, K. G., C. Mirzayan, H. E. Kreuzer, B. M. Alberts, and K. Kreuzer. 1996. Two-dimensional gel analysis of rolling circle replication in the presence and absence of bacteriophage T4 primase. Nucleic Acids Res. 24:2166–2175.

48. Belfort, M., V. Derbyshire, M. Parker, B. Cousineau, and A. M. Lambowitz. 2002. Mobile introns: pathways and proteins, p. 761–783. *In* N. L. Craig, R. Craigie, M. Gellert, and A. M. Lambowitz (ed.), Mobile dNA II. ASM Press, Washington, D.C.

49. Belfort, M., and R. J. Roberts. 1997. Homing endonucleases: keeping the house in order. Nucleic Acids Res. 25:3379–3388.

50. Bell, J. A., K. P. Wilson, X. J. Zhang, H. R. Faber, H. Nicholson, and B. W. Matthews. 1991. Comparison of the crystal structure of bacteriophage T4-lysozyme at low, medium, and high ionic strengths. Proteins Struct. Funct. Genet. 10:10–21.

51. Belle, A., M. Landthaler, and D. A. Shub. 2002. Intronless homing: site-specific endonuclease SegF of bacteriophage T4 mediates localized marker exclusion analogous to homing endonucleases of group I introns. Genes Dev. 16:351–362.

52. Bell-Pedersen, D., S. M. Quirk, M. Bryk, and M. Belfort. 1991. I-TevI, the endonuclease encoded by the mobile td intron, recognizes binding and cleavage domains on its DNA target. Proc. Natl. Acad. Sci. USA 88:7719–7723.

53. Benkovic, S. J., A. M. Valentine, and F. Salinas. 2001. Replisome-mediated DNA replication. Annu. Rev. Biochem. 70:181–208.

54. Benson, K. H., and K. N. Kreuzer. 1992. Plasmid models for bacteriophage T4 DNA replication: requirements for fork proteins. J. Virol. 66:6960–6968.

55. Benson, K. H., and K. N. Kreuzer. 1992. Role of MotA transcription factor in bacteriophage T4 DNA replication. J. Mol. Biol. 228:88–100.

56. Benzer, S. 1957. The elementary units of heredity, p. 70–93. *In* W. D.

McElroy and B. Glass (ed.), The chemical basis of heredity. The Johns Hopkins Press, Baltimore, Md.

57. Berdis, A. J., and S. J. Benkovic. 1996. Role of adenosine 5′-triphosphate hydrolysis in the assembly of the bacteriophage T4 DNA replication holoenzyme complex. Biochemistry 35:9253–9265.

58. Berdis, A. J., P. Soumillion, and S. J. Benkovic. 1996. The carboxyl terminus of the bacteriophage T4 DNA polymerase is required for holoenzyme complex formation. Proc. Natl. Acad. Sci. USA 93:12822–12827.

59. Berger, H., and A. W. Kozinski. 1969. Suppression of T4D ligase mutations by rIIA and rIIB mutations. Proc. Natl. Acad. Sci. USA 64:897–904.

60. Berger, H., A. J. Warren, and K. E. Fry. 1969. Variations in genetic recombination due to amber mutations in T4D bacteriophage. J. Virol. 3:171–175.

61. Berget, P. B., and H. R. Warner. 1975. Identification of P48 and P54 as components of bacteriophage T4 baseplates. J. Virol. 16:1669–1677.

62. Bergh, O., K. Y. Borsheim, G. Bratbak, and M. Heldal. 1989. High abundance of viruses found in aquatic environments. Nature 340:467–468.

63. Berglund, O. 1972. Ribonucleoside diphosphate reductase induced by bacteriophage T4. I. Purification and characterization. J. Biol. Chem. 247:7270–7275.

64. Berglund, O., and B. M. Sjoberg. 1970. A thioredoxin induced by bacteriophage T4. II. Purification and characterization. J. Biol. Chem. 245:6030–6035.

65. Bergsland, K. J., C. Kao, Y. T. Yu, R. Gulati, and L. Snyder. 1990. A site in the T4 bacteriophage major head protein gene that can promote the inhibition of all translation in Escherichia coli. J. Mol. Biol. 213:477–494.

66. Bernhardt, T. G., I. N. Wang, D. K. Struck, and R. Young. 2001. A protein antibiotic in the phage Qbeta virion: diversity in lysis targets. Science 292:2326–2329.

67. Bernstein, C., and S. S. Wallace. 1983. DNA repair, p. 138–151. In C. K. Mathews, E. Kutter, G. Mosig, and P. B. Berget (ed.), Bacteriophage T4. American Society for Microbiology, Washington, D.C.

68. Bernstein, H. 1968. Repair and recombination in phage T4. I. Genes affecting recombination. Cold Spring Harbor Symp. Quant. Biol. 33:325–331.

69. Bernstein, H., and C. Bernstein. 1989. Bacteriophage T4 genetic homologies with bacteria and eucaryotes. J. Bacteriol. 171:2265–2270.

70. Bertrand-Burggraf, E., B. Kemper, and R. P. Fuchs. 1994. Endonuclease VII of phage T4 nicks N-2-acetylaminofluorene-induced DNA structures in vitro. Mutat. Res. 314:287–295.

71. Bhagwat, M., L. J. Hobbs, and N. G. Nossal. 1997. The 5′-exonuclease activity of bacteriophage T4 RNase H is stimulated by the T4 gene 32 single-stranded DNA-binding protein, but its flap endonuclease is inhibited. J. Biol. Chem. 272:28523–28530.

72. Bhagwat, M., D. Meara, and N. G. Nossal. 1997. Identification of residues of T4 RNase H required for catalysis and DNA binding. J. Biol. Chem. 272:28531–28538.

73. Bhagwat, M., and N. G. Nossal. 2001. Bacteriophage T4 RNase H removes both RNA primers and adjacent DNA from the 5′ end of lagging strand fragments. J. Biol. Chem. 276:28516–28524.

74. Bhattacharyya, S. P., and V. B. Rao. 1993. A novel terminase activity associated with the DNA packaging protein gp17 of bacteriophage T4. Virology 196:34–44.

75. Bhattacharyya, S. P., and V. B. Rao. 1994. Structural analysis of DNA cleaved in vivo by bacteriophage T4 terminase. Gene 146:67–72.

76. Bijlenga, R. K., U. Aebi, and E. Kellenberger. 1976. Properties and structure of a gene 24-controlled T4 giant phage. J. Mol. Biol. 103:469–498.

77. Bijlenga, R. K., T. Ishii, and A. Tsugita. 1978. Complete primary structure of the small outer capsid (soc) protein of bacteriophage T4. J. Mol. Biol. 120:249–263.

78. Bingham, R., S. I. Ekunwe, S. Falk, L. Snyder, and C. Kleanthous. 2000. The major head protein of bacteriophage T4 binds specifically to elongation factor Tu. J. Biol. Chem. 275:23219–23226.

79. Birkenbihl, R. P., and B. Kemper. 1998. Endonuclease VII has two DNA-binding sites each composed from one N- and one C-terminus provided by different subunits of the protein dimer. EMBO J. 17:4527–4534.

80. Birkenkamp, K., and B. Kemper. 1996. Bacteriophage T4 strand transfer protein UvsX tolerates symmetric and asymmetric heterologies in short double-stranded oligonucleotides. J. Mol. Biol. 259:622–631.

81. Birkenkamp, K., and B. Kemper. 1995. In vitro processing of heteroduplex loops and mismatches by endonuclease VII. DNA Res. 2:9–14.

82. Birkenkamp-Demtröder, K., S. Golz, and B. Kemper. 1997. Inhibition of Holliday-structures resolving endonuclease VII of bacteriophage T4 by recombination enzymes UvsX and UvsY. J. Mol. Biol. 267:150–162.

83. Biswas, B., S. Adhya, P. Washart, B. Paul, A. N. Trostel, B. Powell, R. Carlton, and C. R. Merril. 2002. Bacteriophage therapy rescues mice bacteremic from a clinical isolate of vancomycin-resistant Enterococcus faecium. Infect. Immun. 70:204–210.

84. Black, L. W. 1974. Bacteriophage T4 internal protein mutants: isolation and properties. Virology 60:166–179.

85. Black, L. W. 1995. DNA packaging and cutting by phage terminases: control in phage T4 by a synaptic mechanism. Bioessays 17:1025–1030.

86. Black, L. W. 1989. DNA packaging in dsDNA bacteriophages. Annu. Rev. Microbiol. 43:267–292.

87. Black, L. W., and K. Abremski. 1974. Restriction of phage T4 internal protein I mutants by a strain of Escherichia coli. Virology 60:180–191.

88. Black, L. W., and Ahmad-Zadeh. 1971. Internal proteins of bacteriophage T4D: their characterization and relation to head structure and assembly. J. Mol. Biol. 57:71–92.

89. Black, L. W., M. K. Showe, and A. C. Steven. 1994. Morphogenesis of the T4 head, p. 218–258. In J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

90. Black, L. W., A. L. Zachary, and V. Manne. 1981. Studies of the mechanism of bacteriophage T4 DNA encapsidation. Prog. Clin. Biol. Res. 64:111–126.

91. Bläsi, U., and R. Young. 1996. Two beginnings for a single purpose: the dual-start holins in the regulation of phage lysis. Mol. Microbiol. 21:675–682.

92. Blattner, F. R., G. Plunkett III, C. A. Bloch, N. T. Perna, V. Burland, M. Riley, J. Collado-Vides, J. D. Glasner, C. K. Rode, G. F. Mayhew, J. Gregor, N. W. Davis, H. A. Kirkpatrick, M. A. Goeden, D. J. Rose, B. Mau, and Y. Shao. 1997. The complete genome sequence of Escherichia coli K-12. Science 277:1453–1462.

93. Bleuit, J. S., H. Xu, Y. Ma, T. Wang, J. Liu, and S. W. Morrical. 2001. Mediator proteins orchestrate enzyme-ssDNA assembly during T4 recombination-dependent DNA replication and repair. Proc. Natl. Acad. Sci. USA 98:8298–8305.

94. Bloom, L. B., M. R. Otto, R. Eritja, L. J. Reha-Krantz, M. F. Goodman, and J. M. Beechem. 1994. Pre-steady-state kinetic analysis of sequence-dependent nucleotide excision by the 3′-exonuclease activity of bacteriophage T4 DNA polymerase. Biochemistry 33:7576–7586.

95. Boikov, P., and L. L. Gumanov. 1972. Modification of the membrane structure of Escherichia coli B cells infected with different mutants of rII phase T4. Biokhimia 37:142–145. (In Russian.)

96. Boikov, P. Y., and L. L. Gumanov. 1974. Functions of rII genes of bacteriophage T4. II. Prereplicative structure of bacteriophage T4r+ and T4r 1231 DNA in Escherichia coli K12 (lambda) cells. Mol. Biol. 7:458–463.

97. Bolle, A., R. H. Epstein, W. Salser, and E. P. Geiduschek. 1968. Transcription during bacteriophage T4 development: requirements for late messenger synthesis. J. Mol. Biol. 33:339–362.

98. Bolle, A., R. H. Epstein, W. Salser, and E. P. Geiduschek. 1968. Transcription during bacteriophage T4 development: synthesis and relative stability of early and late RNA. J. Mol. Biol. 31:325–348.

99. Borodovsky, M., and J. McIninch. 1993. GeneMark: parallel gene recognition for both DNA strands. Comput. Chem. 17:123–133.

100. Botstein, D. 1980. A theory of modular evolution for bacteriophages. Ann. N. Y. Acad. Sci. 354:484–491.

101. Bouet, J.-Y., H. M. Krisch, and J.-M. Louarn. 1998. Ndd, the bacteriophage T4 protein that disrupts the Escherichia coli nucleoid, has a DNA binding activity. J. Bacteriol. 180:5227–5230.

102. Bouet, J. Y., N. J. Campo, H. M. Krisch, and J. M. Louarn. 1996. The effects on Escherichia coli of expression of the cloned bacteriophage T4 nucleoid disruption (ndd) gene. Mol. Microbiol. 20:519–528.

103. Bouet, J. Y., J. Woszczyk, F. Repoila, V. Francois, J. M. Louarn, and H. M. Krisch. 1994. Direct PCR sequencing of the ndd gene of bacteriophage T4: identification of a product involved in bacterial nucleoid disruption. Gene 141:9–16.

104. Bova, R., A. Cascino, M. Cipollaro, O. Grau, M. R. Micheli, M. Santoro, A. Storlazzi, V. Scarlato, and S. Gargano. 1990. Bacteriophage T4 gene 27. Nucleic Acids Res. 18:3046.

105. Boyd, N., N. Schierle, and J. Beckwith. 1998. How many membrane proteins are there? Protein Sci. 7:201–205.

106. Brenner, S., L. Barnett, F. H. C. Crick, and A. Orgel. 1961. The theory of mutagenesis. J. Mol. Biol. 3:121–124.

107. Breschkin, A., and G. Mosig. 1977. Multiple interactions of a DNA-binding protein in vivo. I. Gene 32 mutations of phage T4 inactivate different steps in DNA replication and recombination. J. Mol. Biol. 112:279–294.

108. Breschkin, A., and G. Mosig. 1977. Multiple interactions of a DNA-binding protein in vivo. II. Effects of host mutations on DNA replication of phage T4 gene-32 mutants. J. Mol. Biol. 112:295–308.

109. Brody, E. N., G. A. Kassavetis, M. Ouhammouch, G. M. Sanders, R. L. Tinker, and E. P. Geiduschek. 1995. Old phage, new insights: two recently recognized mechanisms of transcriptional regulation in bacteriophage T4 development. FEMS Microbiol. Lett. 128:1–8.

110. Broida, J., and J. Abelson. 1985. Sequence organization and control of transcription in the bacteriophage T4 tRNA region. J. Mol. Biol. 185:545–563.

111. Broker, T. R. 1973. An electron microscopic analysis of pathways for bacteriophage T4 DNA recombination. J. Mol. Biol. 81:1–16.

112. Brooks, J., and S. Hattman. 1973. Location of the DNA-adenine methylase gene on the genetic map of phage T2. Virology 55:285–288.

113. Brown, D., J. Brown, C. Kang, L. Gold, and P. Allen. 1997. Single-stranded RNA recognition by the bacteriophage T4 translational repressor, *reg4*. J. Biol. Chem. 272:14969–14974.

114. Brown, M. D., L. S. Ripley, and D. H. Hall. 1993. A proflavin-induced frameshift hotspot in the thymidylate synthase gene of bacteriophage T4. Mutat. Res. 286:189–197.

115. Brown, S. M., and F. A. Eiserling. 1979. T4 gene 40 mutants. I. Isolation of new mutants. Virology 97:68–76.

116. Brown, S. M., and F. A. Eiserling. 1979. T4 gene 40 mutants. II. Phenotypic properties. Virology 97:77–89.

117. Brussow, H., and F. Desiere. 2001. Comparative phage genomics and the evolution of Siphoviridae: insights from dairy phages. Mol. Microbiol. 39:213–222.

118. Brussow, H., and R. W. Hendrix. 2002. Phage genomics: small is beautiful. Cell 108:13–16.

119. Bryk, M., M. Belisle, J. E. Mueller, and M. Belfort. 1995. Selection of a remote cleavage site by 1-TevI, the *td* intron-encoded endonuclease. J. Mol. Biol. 247:197–210.

120. Bryk, M., S. M. Quirk, J. E. Mueller, N. Loizos, C. Lawrence, and M. Belfort. 1993. The *td* intron endonuclease I-TevI makes extensive sequence-tolerant contacts across the minor groove of its DNA target. EMBO J. 12:2141–2149. (Erratum, 12:4040–4041.)

121. Bujard, H., A. J. Mazaitis, and E. K. Bautz. 1970. The size of the rII region of bacteriophage T4. Virology 42:717–723.

122. Burda, M. R., and S. Miller. 1999. Folding of coliphage T4 short tail fiber *in vitro*. Analysing the role of a bacteriophage-encoded chaperone. Eur. J. Biochem. 265:771–778.

123. Burke, R. L., M. Munn, J. Barry, and B. M. Alberts. 1985. Purification and properties of the bacteriophage T4 gene 61 RNA priming protein. J. Biol. Chem. 260:1711–1722.

124. Butler, M. M., K. L. Graves, and L. W. Hardy. 1994. Evidence from <sup>18</sup>O exchange studies for an exocyclic methylene intermediate in the reaction catalyzed by T4 deoxycytidylate hydroxymethylase. Biochemistry 33:10521–10526.

125. Calendar, R., S. Yu, H. Myung, V. Barreiro, R. Odegrip, K. Carlson, L. Davenport, G. Mosig, G. E. Christie, and E. Haggård-Ljundquist. 1998. The lysogenic conversion genes of coliphage P2 have unusually high AT content, p. 241–252. In M. Syvanen (ed.), Horizontal gene transfer. Chapman & Hall, New York, N.Y.

126. Calladine, C. R., and H. R. Drew. 1997. Understanding DNA: the molecule & how it works, 2nd ed. Academic Press, Inc., San Diego, Calif.

127. Calladine, C. R., and H. R. Drew. 1996. A useful role for “static” models in elucidating the behaviour of DNA in solution. J. Mol. Biol. 257:479–485.

128. Campbell, A., and D. Botstein. 1983. Evolution of the lambdoid phages, p. 365–380. In R. Hendrix, J. Roberts, F. Stahl, and R. Weisberg (ed.), Lambda II. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.

129. Campos, M., M. Ortega, G. Padron, M. P. Estrada, J. Delafuente, and L. Herrera. 1991. Cloning of coliphage-T4 gene *pseT* and high-level synthesis of polynucleotide kinase in *Escherichia coli*. Gene 101:127–131.

130. Capson, T. L., J. A. Peliska, B. F. Kaboord, M. W. Frey, C. Lively, M. Dahlberg, and S. J. Benkovic. 1992. Kinetic characterization of the polymerase and exonuclease activities of the gene 43 protein of bacteriophage T4. Biochemistry 31:10984–10994.

131. Cardillo, T. S., E. F. Landry, and J. S. Wiberg. 1979. RegA protein of bacteriophage T4D: identification, schedule of synthesis, and autogenous regulation. J. Virol. 32:905–916.

132. Carles-Kinch, K., J. W. George, and K. N. Kreuzer. 1997. Bacteriophage T4 UvsW protein is a helicase involved in recombination, repair and the regulation of DNA replication origins. EMBO J. 16:4142–4151.

133. Carles-Kinch, K., and K. N. Kreuzer. 1997. RNA-DNA hybrid formation at a bacteriophage T4 replication origin. J. Mol. Biol. 266:915–926.

134. Carlson, K., and L. D. Kosturko. 1998. Endonuclease II of coliphage T4: a recombinase disguised as a restriction endonuclease? Mol. Microbiol. 27:671–676.

135. Carlson, K., L. D. Kosturko, and A.-C. Nyström. 1999. Sequence-specific cleavage by bacteriophage T4 endonuclease II *in vitro*. Mol. Microbiol. 31:1395–1405.

136. Carlson, K., L. D. Kosturko, and A.-C. Nyström. 1996. Short-range and long-range context effects on coliphage T4 endonuclease II-dependent restriction. J. Bacteriol. 178:6419–6426.

137. Carlson, K., M. Krabbe, A. C. Nystrom, and L. D. Kosturko. 1993. DNA determinants of restriction. Bacteriophage T4 endonuclease II-dependent cleavage of plasmid DNA *in vivo*. J. Biol. Chem. 268:8908–8918.

137a. Carlson, K., and B. Nicolaisen. 1979. Cleavage map of bacteriophage T4 cytosine-containing DNA by sequence-specific endonucleases *SalI* and *KpnI*. J. Virol. 31:112–123.

138. Carlson, K., and A. Øvervatn. 1986. Bacteriophage T4 endonucleases II and IV, oppositely affected by dCMP hydroxymethylation activity, have different roles in the degradation and in the RNA polymerase-dependent replication of T4 cytosine-containing DNA. Genetics 114:669–685.

139. Carlson, K., E. Raleigh, A., and S. Hattman. 1994. Restriction and mod-

ification, p. 369–381. In J. D. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. ASM Press, Washington, D.C.

140. Reference deleted.

141. Carlton, R. M. 1999. Phage therapy: past history and future prospects. Arch. Immunol. Ther. Exp. 47:267–274.

142. Carpousis, A. J., E. A. Mudd, and H. M. Krisch. 1989. Transcription and messenger RNA processing upstream of bacteriophage T4 gene 32. Mol. Gen. Genet. 219:39–48.

143. Carreras, C. W., and D. V. Santi. 1995. The catalytic mechanism and structure of thymidylate synthase. Annu. Rev. Biochem. 64:721–762.

144. Caruso, M., A. Coppo, A. Manzi, and J. F. Pulitzer. 1979. Host-virus interactions in the control of T4 prereplicative transcription. I. *tabC* (*rho*) mutants. J. Mol. Biol. 135:959–977.

145. Casas-Finet, J. R. 1989. Binding properties of T4 gene 32 protein fragments carrying partially cleaved terminal domains. FEBS Lett. 249:396–400.

146. Reference deleted.

147. Casas-Finet, J. R., K. R. Fischer, and R. L. Karpel. 1992. Structural basis for the nucleic acid binding cooperativity of bacteriophage T4 gene 32 protein: the (Lys/Arg)3(Ser/Thr)2 (LAST) motif. Proc. Natl. Acad. Sci. USA 89:1050–1054. (Erratum, 89:5201.)

148. Casas-Finet, J. R., and R. L. Karpel. 1993. Bacteriophage T4 gene 32 protein: modulation of protein-nucleic acid and protein-protein association by structural domains. Biochemistry 32:9735–9744.

149. Casjens, S., G. Hatfull, and R. Hendrix. 1992. Evolution of dsDNA tailed-bacteriophage genomes. Semin. Virol. 3:383–397.

150. Cech, T. R. 1993. Structure and mechanism of the large catalytic RNAs: group I and group II introns and ribonuclease P, p. 239–269. In R. F. Gesteland and J. F. Atkins (ed.), The RNA world. Cold Spring Harbor Laboratory Press, Press, Cold Spring Harbor, N.Y.

151. Cech, T. R., S. H. Damberger, and R. R. Gutell. 1994. Representation of the secondary and tertiary structure of group I introns. Nat. Struct. Biol. 1:273–280.

152. Cech, T. R., and B. L. Golden. 1999. Building a catalytic active site using only RNA, p. 321–349. In R. F. Gesteland, T. R. Cech, and J. F. Atkins (ed.), The RNA world, 2nd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.

153. Cerritelli, M. E., J. S. Wall, M. N. Simon, J. F. Conway, and A. C. Steven. 1996. Stoichiometry and domainal organization of the long tail-fiber of bacteriophage T4: a hinged viral adhesin. J. Mol. Biol. 260:767–780.

154. Cha, T.-A., and B. M. Alberts. 1986. Studies of the DNA helicase-RNA primase unit from bacteriophage T4. J. Biol. Chem. 261:7001–7010.

155. Cha, T. A., and B. M. Alberts. 1990. Effects of the bacteriophage T4 gene 41 and gene 32 proteins on RNA primer synthesis: coupling of leading- and lagging-strand DNA synthesis at a replication fork. Biochemistry 29:1791–1798.

156. Chace, K. V., and D. H. Hall. 1975. Characterization of new regulatory mutants of bacteriophage T4. II. New class of mutants. J. Virol. 15:929–945.

157. Chace, K. V., and D. H. Hall. 1973. Isolation of mutants of bacteriophage T4 unable to induce thymidine kinase activity. J. Virol. 12:343–348.

158. Champness, W. C., and L. Snyder. 1984. Bacteriophage T4 *gol* site: sequence analysis and effects of the site on plasmid transformation. J. Virol. 50:555–562.

159. Chan, V. L., and K. Ebisuzaki. 1973. Intergenic suppression of amber polynucleotide ligase mutation in bacteriophage T4. II. Virology 53:60–74.

160. Chang, A. 1992. Ph.D. dissertation. Vanderbilt University, Nashville, Tenn.

161. Chapman, D., I. Morad, G. Kaufmann, M. J. Gait, L. Jorissen, and L. Snyder. 1988. Nucleotide and deduced amino acid sequence of *stp*: the bacteriophage T4 anticodon nuclease gene. J. Mol. Biol. 199:373–377.

162. Chen, X., C. K. Mathews, L. J. Wheeler, G. Maley, F. Maley, and D. H. Coombs. 1995. An immunoblot assay reveals that bacteriophage T4 thymidylate synthase and dihydrofolate reductase are not virion proteins. J. Virol. 69:2119–2125.

163. Chevalier, B. S., and B. L. Stoddard. 2001. Homing endonucleases: structural and functional insight into the catalysts of intron/intein mobility. Nucleic Acids Res. 29:3757–3774.

164. Childs, J. D. 1980. Effect of hoc protein on the electrophoretic mobility of intact bacteriophage T4D particles in polyacrylamide gel electrophoresis. J. Mol. Biol. 141:163–173.

165. Childs, J. D. 1980. Isolation and genetic properties of a bacteriophage T4 *uvsX* mutant. Mutat. Res. 71:1–14.

166. Childs, J. D. 1973. Superinfection exclusion by incomplete genomes of bacteriophage T4. J. Virol. 11:1–8.

167. Childs, J. D., and H. C. Birnboim. 1975. Polyacrylamide gel electrophoresis of intact bacteriophage T4D particles. J. Virol. 16:652–661.

168. Childs, J. D., and R. Pilon. 1983. Evidence that bacteriophage T4 *eph1* is a missense *hoc* mutation. J. Virol. 46:629–631.

169. Chiu, C. S., S. M. Cox, and G. R. Greenberg. 1980. Effect of bacteriophage

T4 nrd mutants on deoxyribonucleotide synthesis *in vivo*. J. Biol. Chem. 255:2747–2751.

170. Chiu, C. S., T. Ruettinger, J. B. Flanegan, and G. R. Greenberg. 1977. Role of deoxycytidylate deaminase in deoxyribonucleotide synthesis in bacteriophage T4 DNA replication. J. Biol. Chem. 252:8603–8608.

171. Chiurazzi, M., and J. F. Pulitzer. 1998. Characterisation of the bacteriophage T4 *comC* alpha 55.6 and *comJ* mutants. A possible role in an antitermination process. FEMS Microbiol. Lett. 166:187–195.

172. Christoph, A., G. von Heesberg, and B. Kemper. 1998. Epitope mapping of T4 endonuclease VII with monoclonal antibodies reveals importance of both ends of the protein for target binding. J. Mol. Biol. 277:529–540.

173. Chu, F. K., G. F. Maley, and F. Maley. 1988. RNA splicing in the T-even bacteriophage. FASEB J. 2:216–223.

174. Chu, F. K., G. F. Maley, F. Maley, and M. Belfort. 1984. Intervening sequence in the thymidylate synthase gene of bacteriophage T4. Proc. Natl. Acad. Sci. USA. 81:3049–3053.

175. Chu, F. K., G. F. Maley, D. K. West, M. Belfort, and F. Maley. 1986. Characterization of the intron in the phage T4 thymidylate synthase gene and evidence for its self-excision from the primary transcript. Cell 45:157–166.

176. Cicero, M. P., K. A. Alexander, and K. N. Kreuzer. 1998. The MotA transcriptional activator of bacteriophage T4 binds to its specific DNA site as a monomer. Biochemistry 37:4977–4984.

177. Cicero, M. P., M. M. Sharp, C. A. Gross, and K. N. Kreuzer. 2001. Substitutions in bacteriophage T4 AsiA and *Escherichia coli* sigma(70) that suppress T4 motA activation mutations. J. Bacteriol. 183:2289–2297.

178. Clyman, J., S. Quirk, and M. Belfort. 1994. Mobile introns in the T-even phages, p. 83–88. In J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

179. Coetzee, T., D. Herschlag, and M. Belfort. 1994. *Escherichia coli* proteins, including ribosomal protein S12, facilitate *in vitro* splicing of phage T4 introns by acting as RNA chaperones. Genes Dev. 8:1575–1588.

180. Colland, F., G. Orsini, E. N. Brody, H. Buc, and A. Kolb. 1998. The bacteriophage T4 AsiA protein: a molecular switch for σ<sup>70</sup>-dependent promoters. Mol. Microbiol. 27:819–829.

181. Colowick, M. S., and S. P. Colowick. 1983. Membrane ATPase activation on infection of *E. coli* K (λ) cells with phage *T4rII* mutants. Trans. N. Y. Acad. Sci. 41:35–40.

182. Conkling, M. A., and J. W. Drake. 1984. Isolation and characterization of conditional alleles of bacteriophage T4 genes *uvsX* and *uvsY*. Genetics 107:505–523.

183. Conkling, M. A., and J. W. Drake. 1984. Thermal rescue of UV-irradiated bacteriophage T4 and biphasic mode of action of the WXY system. Genetics 107:525–536.

184. Cook, K. S., and A. F. Seasholtz. 1982. Identification of some bacteriophage T4 prereplicative proteins on two-dimensional gel proteins. J. Virol. 42:767–772.

185. Cooley, W., K. Sirotkin, R. Green, and L. Snyder. 1979. A new gene of *Escherichia coli* K-12 whose product participates in T4 bacteriophage late gene expression: interaction of lit with the T4-induced polynucleotide 5′-kinase 3′-phosphatase. J. Bacteriol. 140:83–91.

186. Coombs, D. H., and F. Arisaka. 1994. T4 tail structure and function, p. 259–281. In J. D. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

187. Coppo, A., A. Manzi, J. F. Pulitzer, and H. Takahashi. 1973. Abortive bacteriophage T4 head assembly in mutants of *Escherichia coli*. J. Mol. Biol. 76:61–87.

187a. Corbin, B. D., R. J. McLean, and G. M. Aron. 2001. Bacteriophage T4 multiplication in a glucose-limited *Escherichia coli* biofilm. Can. J. Microbiol. 7:680–684.

188. Cornett, J. B., and M. Valleé. 1973. The map position of the immunity (*imm*) gene of bacteriophage T4. Virology 51:506–508.

189. Cowan, J., K. d’Acci, B. Guttmann, and E. Kutter. 1994. Gel analysis of T4 prereplicative proteins., p. 520–527. In J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

190. Crawford, J. T., and E. B. Goldberg. 1980. The function of tail fibers in triggering baseplate expansion of bacteriophage T4. J. Mol. Biol. 139:679–690.

191. Crick, F. H. C., L. Barnett, S. Brenner, and R. J. Watts-Tobin. 1961. General nature of the genetic code for proteins. Nature 192:1227–1232.

192. Cromie, G. A., J. C. Connelly, and D. R. Leach. 2001. Recombination at double-strand breaks and DNA ends: conserved mechanisms from phage to humans. Mol. Cell 8:1163–1174.

193. Crowther, R. A. 1980. Mutants of bacteriophage T4 that produce infective fibreless particles. J. Mol. Biol. 137:159–174.

194. Cummings, D. J., and R. W. Bolin. 1976. Head length control in T4

bacteriophage morphogenesis: effect of canavanine on assembly. Bacteriol. Rev. 40:314–339.

195. Cunningham, R. P., and H. Berger. 1977. Mutations affecting genetic recombination in bacteriophage T4D. I. Pathway analysis. Virology 80:67–82.

196. Cunningham, R. P., and H. Berger. 1977. A new DNA-delay mutation in bacteriophage T4D. Virology 79:320–329.

197. Cupido, M., J. Grimbergen, and B. De Groot. 1980. Participation of bacteriophage T4 gene 41 in replication repair. Mutat. Res. 70:131–138.

198. Daegelen, P., and E. Brody. 1990. The *rIIA* gene of bacteriophage T4. II. Regulation of its messenger RNA synthesis. Genetics 125:249–260.

199. Dannenberg, R., and G. Mosig. 1981. Semiconservative DNA replication is initiated at a single site in recombination-deficient gene 32 mutants of bacteriophage T4. J. Virol. 40:890–900.

200. Dannenberg, R., and G. Mosig. 1983. Early intermediates in bacteriophage T4 DNA replication and recombination. J. Virol. 45:813–831.

201. d’Aubenton Carafa, Y., E. Brody, and C. Thermes. 1990. Prediction of rho-independent *Escherichia coli* transcription terminators. A statistical analysis of their RNA stem-loop structures. J. Mol. Biol. 216:835–858.

202. Dean, A. B., M. J. Stanger, J. T. Dansereau, P. Van Roey, V. Derbyshire, and M. Belfort. 2002. Inaugural article. Zinc finger as distance determinant in the flexible linker of intron endonuclease I-TevI. Proc. Natl. Acad. Sci. USA 99:8554–8561.

203. Depew, R. E., and N. R. Cozzarelli. 1974. Genetics and physiology of bacteriophage T4 3′-phosphatase: evidence for involvement of the enzyme in T4 DNA metabolism. J. Virol. 13:888–897.

204. Depew, R. E., T. J. Snopek, and N. R. Cozzarelli. 1975. Characterization of a new class of deletions of the D region of the bacteriophage T4 genome. Virology 64:144–155.

205. Depping, R. 2002. Doctoral thesis. Ruhr-University Bochum, Bochum, Germany.

206. Derbyshire, V., J. C. Kowalski, J. T. Dansereau, C. R. Hauer, and M. Belfort. 1997. Two-domain structure of the *td* intron-encoded endonuclease I-TevI correlates with the two-domain configuration of the homing site. J. Mol. Biol. 265:494–506.

207. Derr, L. K., and J. W. Drake. 1990. Isolation and genetic characterization of new *uvsW* alleles of bacteriophage T4. Mol. Genet. 222:257–264.

208. Derr, L. K., and K. N. Kreuzer. 1990. Expression and function of the *uvsW* gene of bacteriophage T4. J. Mol. Biol. 214:643–656.

209. Desiere, F., C. Mahanivong, A. J. Hillier, P. S. Chandry, B. E. Davidson, and H. Brussow. 2001. Comparative genomics of lactococcal phages: insight from the complete genome sequence of *Lactococcus lactis* phage BK5-T. Virology 283:240–252.

210. Desplats, C., C. Dez, F. Tetart, H. Eleaume, and H. M. Krisch. 2002. Snapshot of the genome of the pseudo-T-even bacteriophage RB49. J. Bacteriol. 184:2789–2804.

211. Devereux, J., P. Haeberli, and O. Smithies. 1984. A comprehensive set of sequence analysis programs for the VAX. Nucleic Acids Res. 12:387–395.

212. DeVries, J. K., and S. S. Wallace. 1983. Expression of cloned bacteriophage T4 *uvsW* and *uvsY* genes in *rec*<sup>+</sup> and *rec*<sup>−</sup> *Escherichia coli*. J. Virol. 47:406–412.

212a. de Waard, A., A. V. Paul, and I. R. Lehman. 1965. The structural gene for deoxyribonucleic acid polymerase in bacteriophage T4 and T5. Proc. Natl. Acad. Sci. USA 54:1241–1248.

213. Dewey, M. J., and F. R. Frankel. 1975. Two suppressor loci for gene 49 mutations of bacteriophage T4. I. Genetic properties and DNA synthesis. Virology 68:387–401.

214. Dewey, M. J., J. S. Wiberg, and F. R. Frankel. 1974. Genetic control of whisker antigen of bacteriophage T4D. J. Mol. Biol. 84:625–634.

215. Dharmalingam, K., H. R. Revel, and E. B. Goldberg. 1982. Physical mapping and cloning of bacteriophage T4 anti-restriction endonuclease gene. J. Bacteriol. 149:694–699.

216. Dhillon, E. K. S., and T. S. Dhillon. 1973. HK239: a P2-related temperate phage which excludes rII mutants of T4. Virology 55:136–142.

217. Dickson, R. C. 1973. Assembly of bacteriophage T4 tail fibers. IV. Subunit composition of tail fibers and fiber precursors. J. Mol. Biol. 79:633–647.

218. Dickson, R. C., S. L. Barnes, and F. A. Eiserling. 1970. Structural proteins of bacteriophage T4. J. Mol. Biol. 53:461–474.

219. Dizdaroglu, M., T. H. Zastawny, J. R. Carmical, and R. S. Lloyd. 1996. A novel DNA N-glycosylase activity of *E. coli* T4 endonuclease V that excises 4,6-diamino-5-formamidopyrimidine from DNA, a UV-radiation- and radical-induced product of adenine. Mutat. Res. 362:1–8.

220. Reference deleted.

221. Doan, P. L., K. G. Belanger, and K. N. Kreuzer. 2001. Two types of recombination hotspots in bacteriophage T4: One requires DNA damage and a replication origin and the other does not. Genetics 157:1077–1087.

222. Dodson, M. L., and R. S. Lloyd. 1994. Structure-function analysis of T4 endonuclease V., p. 318–321. In J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

223. Dodson, M. L., M. A. Prince, W. F. Anderson, and R. S. Lloyd. 1991.

Site-directed deletion mutagenesis within the T4 endonuclease-V gene—dispensable sequences within putative loop regions. Mutat. Res. 255:19–29.

224. Doermann, A. H. 1948. Lysis and lysis inhibition with *Escherichia coli* bacteriophage. J. Bacteriol. 55:257–275.
225. Doermann, A. H., F. A. Eiserling, and L. Boehner. 1973. Genetic control of capsid length in bacteriophage T4. I. Isolation and preliminary description of four new mutants. J. Virol. 12:374–385.
226. Doermann, A. H., and L. D. Simon. 1984. Bacteriophage T4 bypass31 mutations that make gene 31 nonessential for bacteriophage T4 replication: mapping bypass31 mutations by UV rescue experiments. J. Virol. 51:315–320.
227. Dong, F., E. P. Gogol, and P. H. von Hippel. 1995. The phage T4-coded DNA replication helicase (gp41) forms a hexamer upon activation by nucleoside triphosphate. J. Biol. Chem. 270:7462–7473.
228. Dong, F., S. E. Weitzel, and P. H. von Hippel. 1996. A coupled complex of T4 DNA replication helicase (gp41) and polymerase (gp43) can perform rapid and processive DNA strand-displacement synthesis. Proc. Natl. Acad. Sci. USA 93:14456–14461.
229. Doublie, S., M. R. Sawaya, and T. Ellenberger. 1999. An open and closed case for all polymerases. Struct. Fold Des. 7:R31–R35.
230. Drake, J. W. 1993. General antimutators are improbable. J. Mol. Biol. 229:8–13.
231. Drake, J. W. 1973. The genetic control of spontaneous and induced mutation rates in bacteriophage T4. Genetics 73(Suppl.):45–64.
232. Drake, J. W. 1985. Photodynamic inactivation and mutagenesis by angelicin (isopsoralen) or thiopyronin (methylene red) in wild-type and repair-deficient strains of bacteriophage T4. J. Bacteriol. 162:1311–1313.
233. Drake, J. W., and K. N. Kreuzer. 1994. DNA transactions in T4-infected *Escherichia coli*, p. 11–13. *In* J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.
234. Drake, J. W., and L. S. Ripley. 1994. Mutagenesis, p. 98–124. *In* J. D. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.
235. Reference deleted.
236. Dressman, H. K., and J. W. Drake. 1999. Lysis and lysis inhibition in bacteriophage T4: *rV* mutations reside in the *holin t* gene. J. Bacteriol. 181:4391–4396.
237. Dressman, H. K., C.-C. Wang, J. D. Karam, and J. W. Drake. 1997. Retention of replication fidelity by a DNA polymerase functioning in a distantly related environment. Proc. Natl. Acad. Sci. USA 94:8042–8046.
238. Driedons, R. A. 1981. The quaternary structure of the T4 gene product 20 oligomer. Prog. Clin. Biol. Res. 64:315–323.
239. Drivdahl, R. H., and E. M. Kutter. 1990. Inhibition of transcription of cytosine-containing DNA *in vitro* by the *alc* gene product of bacteriophage T4. J. Bacteriol. 172:2716–2727.
240. Du, Z., and D. W. Hoffman. 1997. An NMR and mutational study of the pseudoknot within the gene 32 mRNA of bacteriophage T2: insights into a family of structurally related RNA pseudoknots. Nucleic Acids Res. 25:1130–1135.
241. Duckworth, D. H., and M. J. Bessman. 1967. The enzymology of virus-infected bacteria. X. A biochemical-genetic study of the deoxynucleotide kinase induced by wild type and amber mutants of phage T4. J. Biol. Chem. 242:2877–2885.
242. Duda, R. L., M. Gingery, and F. A. Eiserling. 1986. Potential length determiner and DNA injection protein is extruded from bacteriophage T4 tail tubes in vitro. Virology 151:296–314.
243. Duda, R. L., M. Gingery, L. K. Ishimoto, and F. A. Eiserling. 1990. Expression of plasmid-encoded structural proteins permits engineering of bacteriophage T4 assembly. Virology 179:728–737.
244. Dudas, K. C., and K. N. Kreuzer. 2001. UvsW protein regulates bacteriophage T4 origin-dependent replication by unwinding R-loops. Mol. Cell. Biol. 21:2706–2715.
245. Dulbecco, R. 1949. Reactivation of ultraviolet-inactivated bacteriophage by visible light. Nature 163:949.
246. Durantel, D., L. Croizier, M. D. Ayres, G. Croizier, R. D. Possee, and M. Lopez-Ferber. 1998. The *pnk/pnl* gene (ORF 86) of Autographa californica nucleopolyhedrovirus is a non-essential, immediate early gene. J. Gen. Virol. 79:629–637.
247. Eddy, S. R., and L. Gold. 1991. The phage T4 *nrdB* intron: a deletion mutant of a version found in the wild. Genes Dev. 5:1032–1041.
248. Edgar, R. S., and I. Lielausis. 1965. Serological studies with mutants of phage T4D defective in genes determining tail fiber structure. Genetics 52:1187–1200.
249. Edgar, R. S., and I. Lielausis. 1968. Some steps in the assembly of bacteriophage T4. J. Mol. Biol. 32:263–276.
250. Edgar, R. S., and W. B. Wood. 1966. Morphogenesis of bacteriophage T4

in extracts of mutant-infected cells. Proc. Natl. Acad. Sci. USA 55:498–505.
251. Edgell, D. R., M. Belfort, and D. A. Shub. 2000. Barriers to intron promiscuity in bacteria. J. Bacteriol. 182:5281–5289.
252. Edgell, D. R., and D. A. Shub. 2001. Related homing endonucleases I-BmoI and I-TevI use different strategies to cleave homologous recognition sites. Proc. Natl. Acad. Sci USA 98:7898–7903.
253. Efimov, V. P., A. G. Prilipov, and V. V. Mesyazhinov. 1990. Nucleotide sequences of bacteriophage-T4 gene-6, 7 and 8. Nucleic Acids Res. 18:5313.
254. Eggleson, A. K., and S. C. Kowalczykowski. 1991. An overview of homologous pairing and DNA strand exchange proteins. Biochimie 73:163–176.
255. Ehrenman, K., J. Pedersen-Lane, D. West, R. Herman, F. Maley, and M. Belfort. 1986. Processing of phage T4 td-encoded RNA is analogous to the eukaryotic group I splicing pathway. Proc. Natl. Acad. Sci. USA 83:5875–5879.
256. Eiserling, F. A., E. P. Geiduschek, R. H. Epstein, and E. J. Metter. 1970. Capsid size and deoxyribonucleic acid length: the petite variant of bacteriophage T4. J. Virol. 6:865–876.
257. Eklund, H. 1994. Structure and function of T4 glutaredoxin (thioredoxin), p. 326–328. *In* J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.
258. el Hassan, M. A., and C. R. Calladine. 1996. Propeller-twisting of base-pairs and the conformational mobility of dinucleotide steps in DNA. J. Mol. Biol. 259:95–103.
259. Elisseeva, E., S. S. Mandal, and L. J. Reha-Krantz. 1999. Mutational and pH studies of the 3′ → 5′ exonuclease activity of bacteriophage T4 DNA polymerase. J. Biol. Chem. 274:25151–25158.
260. Elliott, T., and E. P. Geiduschek. 1984. Defining a bacteriophage T4 late promoter: absence of a “−35” region. Cell 36:211–219.
261. Emrich, J. 1968. Lysis of T4-infected bacteria in the absence of lysozyme. Virology 35:158–165.
262. Engman, H. W., and K. N. Kreuzer. 1993. Deletion of the essential gene 24 from the bacteriophage T4 genome. Gene 123:69–74.
263. Ennis, H. L., and K. D. Kievitt. 1973. Association of the rIIA protein with the bacterial membrane. Proc. Natl. Acad. Sci. USA 70:1468–1472.
264. Epstein, R. H., A. Bolle, C. Steinberg, E. Kellenberger, E. Boy de la Tour, R. Chevalley, R. Edgar, M. Susman, C. Denhardt, and I. Lielausis. 1964. Physiological studies of conditional lethal mutants of bacteriophage T4D. Cold Spring Harbor Symp. Quant. Biol. 28:375–392.
265. Ermolaeva, M. D., H. G. Khalak, O. White, H. O. Smith, and S. L. Salzberg. 2000. Prediction of transcription terminators in bacterial genomes. J. Mol. Biol. 301:27–33.
266. Estrem, S. T., W. Ross, T. Gaal, Z. W. S. Chen, W. Niu, R. H. Ebright, and R. L. Gourse. 1999. Bacterial promoter architecture: subsite structure of UP elements and interactions with the carboxy-terminal domain of the RNA polymerase α subunit. Genes Dev. 13:2134–2147.
267. Evdokimov, A. A., V. V. Zinoviev, E. G. Malygin, S. L. Schlagman, and S. Hattman. 2002. Bacteriophage T4 Dam DNA-[N6-adenine]methyltransferase. Kinetic evidence for a catalytically essential conformational change in the ternary complex. J. Biol. Chem. 277:279–86.
268. Faber, H. R., and B. W. Matthews. 1990. A mutant T4 lysozyme displays 5 different crystal conformations. Nature 348:263–266.
269. Fareed, G. C., and C. C. Richardson. 1967. Enzymatic breakage and joining of deoxyribonucleic acid. II. The structural gene for polynucleotide ligase in bacteriophage T4. Proc. Natl. Acad. Sci. USA 58:665–672.
270. Favre, R., E. Boy de la Tour, N. Segre, and E. Kellenberger. 1965. Studies on the morphopoiesis of the head of phage T-even. I. Morphological, immunological, and genetic characterization of polyheads. J. Ultrastruct. Res. 13:318–342.
271. Feng, D. F., and R. F. Doolittle. 1996. Progressive alignment of amino acid sequences and construction of phylogenetic trees from them. Methods Enzymol. 266:368–382.
272. Ferguson, P. L., and D. H. Coombs. 2000. Pulse-chase analysis of the *in vivo* assembly of the bacteriophage T4 tail. J. Mol. Biol. 297:99–117.
273. Filee, J., P. Forterre, T. Sen-Lin, and J. Laurent. 2002. Evolution of DNA polymerase families: evidences for multiple gene exchange between cellular and viral proteins. J. Mol. Evol. 54:763–773.
274. Finer-Moore, J. S., G. F. Maley, F. Maley, W. R. Montfort, and R. M. Stroud. 1994. Crystal structure of thymidylate synthase from T4 phage: component of a deoxynucleotide triphosphate-synthesizing complex. Biochemistry 33:15459–15468.
275. Finnin, M. S., M. P. Cicero, C. Davies, S. J. Porter, S. W. White, and K. N. Kreuzer. 1997. The activation domain of the MotA transcription factor from bacteriophage T4. EMBO J. 16:1992–2003.
276. Finnin, M. S., D. W. Hoffman, K. N. Kreuzer, S. J. Porter, R. P. Schmidt, and S. W. White. 1993. The MotA protein from bacteriophage T4 contains two domains. Preliminary structural analysis by X-ray diffraction and nuclear magnetic resonance. J. Mol. Biol. 232:301–304.
277. Finnin, M. S., D. W. Hoffman, and S. W. White. 1994. The DNA-binding

domain of the MotA transcription factor from bacteriophage T4 shows structural similarity to the TATA-binding protein. Proc. Natl. Acad. Sci. USA 91:10972–10976.

278. Flemming, M., B. Deumling, and B. Kemper. 1993. Function of gene 49 of bacteriophage T4 III. Isolation of Holliday structures from very fast-sedimenting DNA. Virology 196:910–913.

279. Foley, S., A. Bruttin, and H. Brussow. 2000. Widespread distribution of a group I intron and its three deletion derivatives in the lysin gene of Streptococcus thermophilus bacteriophages. J. Virol. 74:611–618.

280. Follansbee, S. E., R. W. Vanderslice, L. G. Chavez, and C. D. Yegian. 1974. A new set of adsorption mutants of bacteriophage T4D: identification of a new gene. Virology 58:180–199.

281. Formosa, T., and B. M. Alberts. 1986. Purification and characterization of the T4 bacteriophage UvsX protein. J. Biol. Chem. 261:6107–6118.

282. Formosa, T., and B. M. Alberts. 1984. The use of affinity chromatography to study proteins involved in bacteriophage T4 genetic recombination. Cold Spring Harbor Symp. Quant. Biol. 49:363–370.

283. Formosa, T., R. L. Burke, and B. M. Alberts. 1983. Affinity purification of bacteriophage T4 proteins essential for DNA replication and genetic recombination. Proc. Natl. Acad. Sci. USA 80:2442–2446.

284. Foss, K., S. Kao, and W. H. McClain. 1979. Three suppressor forms of bacteriophage T4 leucine transfer RNA. J. Mol. Biol. 135:1013–1021.

285. Frankel, F. R., M. L. Batcheler, and C. K. Clark. 1971. The role of gene 49 in DNA replication and head morphogenesis in bacteriophage T4. J. Mol. Biol. 62:439–463.

286. Franklin, J. G., and G. Mosig. 1996. Expression of the bacteriophage T4 DNA terminase genes 16 and 17 yields multiple proteins. Gene 177:179–189.

287. Franklin, J. L. 1992. Ph.D. dissertation. Vanderbilt University, Nashville, Tenn.

288. Franklin, J. L., D. Haseltine, L. Davenport, and G. Mosig. 1998. The largest (70kDa) product of the bacteriophage T4 terminase gene 17 binds to single-stranded DNA segments and digests them towards junctions with double-stranded DNA. J. Mol. Biol. 277:541–557.

289. Franklin, M. C., J. Wang, and T. A. Steitz. 2001. Structure of the replicating complex of a Pol alpha family DNA polymerase. Cell 105:657–667.

290. Frazier, M. W., and G. Mosig. 1990. The bacteriophage T4 gene *mrh* whose product inhibits late T4 gene expression in an *Escherichia coli rpoH* (*σ**32*) mutant. Gene 88:7–14.

291. Freedman, M. L., and R. E. Krisch. 1971. Enlargement of *Escherichia coli* after bacteriophage infection. II. Proposed mechanism. J. Virol. 8:95–102.

292. Freedman, R., and S. Brenner. 1972. Anomalously revertible *rII* mutants of phage T4. Genet. Res. 19:165–171.

293. Freedman, R., and S. Brenner. 1972. Hybrid T4 r II B cistrons created by genetic duplications. J. Mol. Biol. 69:409–419.

294. French, S. 1993. Replication meets transcription: who’s got the right of way. ASM News 59:437–443.

295. Freudenreich, C. H., C. Chang, and K. N. Kreuzer. 1998. Mutations of the bacteriophage T4 type II DNA topoisomerase that alter sensitivity to antitumor agent 4′-(9-acridinylamino) methanesulfon-m-aniside and an antibacterial quinolone. Cancer Res. 58:1260–1267.

296. Freudenreich, C. H., and K. N. Kreuzer. 1994. Localization of an amino-acridine antitumor agent in a type II topoisomerase-DNA complex. Proc. Natl. Acad. Sci. USA 91:11007–11011.

297. Freudenreich, C. H., and K. N. Kreuzer. 1993. Mutational analysis of a type II topoisomerase cleavage site: distinct requirements for enzyme and inhibitors. EMBO J. 12:2085–2097.

298. Frey, M. W., N. G. Nossal, T. L. Capson, and S. J. Benkovic. 1993. Construction and characterization of a bacteriophage T4 DNA polymerase deficient in 3′ → 5′ exonuclease activity. Proc. Natl. Acad. Sci. USA 90:2579–2583.

299. Fu, T.-J., G. M. Sanders, M. O’Donnell, and E. P. Geiduschek. 1996. Dynamics of DNA-tracking by two sliding-clamp proteins. EMBO J. 15: 4414–4422.

300. Fu, T. J., B. Kemper, and N. C. Seeman. 1994. Cleavage of double-crossover molecules by T4 endonuclease VII. Biochemistry 33:3896–3905.

301. Fujisawa, H., T. Yonesaki, and T. Minagawa. 1985. Sequence of the T4 recombination gene, *uvsX*, and its comparison with that of the *recA* gene of *Escherichia coli*. Nucleic Acids Res. 13:7473–7481.

302. Fukada, K., and J. Abelson. 1980. DNA sequence of a T4 transfer RNA gene cluster. J. Mol. Biol. 139:377–391.

303. Ganz, A., U. Kruse, and W. Ruger. 1991. Gene product *dsbA* of bacteriophage T4 binds to late promoters and enhances late transcription. Mol. Gen. Genet. 225:427–434.

304. Garvish, J. F., and R. S. Lloyd. 2000. Active-site determination of a pyrimidine dimer glycosylase. J. Mol. Biol. 295:479–488.

305. Gary, T. P., N. E. Colowick, and G. Mosig. 1998. A species barrier between bacteriophages T2 and T4: exclusion, join-copy and join-cut-copy recombination and mutagenesis in the dCTPase genes. Genetics 148:1461–1473.

306. Reference deleted.

307. Gauss, P., D. H. Doherty, and L. Gold. 1983. Bacterial and phage mutations that reveal helix-unwinding activities required for bacteriophage T4 DNA replication. Proc. Natl. Acad. Sci. USA 80:1669–1673.

308. Gauss, P., M. Gayle, R. B. Winter, and L. Gold. 1987. The bacteriophage T4 *dexA* gene: sequence and analysis of a gene conditionally required for DNA replication. Mol. Genet. 206:24–34.

309. Gauss, P., K. Park, T. E. Spencer, and K. J. Hacker. 1994. DNA helicase requirements for DNA replication during bacteriophage T4 infection. J. Bacteriol. 176:1667–1672.

310. Geiduschek, E. P. 1995. Connecting a viral DNA replication apparatus with gene expression. Semin. Virol. 6:25–33.

311. Geiduschek, E. P. 1997. Paths to activation of transcription. Science 275: 1614–1616.

312. Geiduschek, E. P. 1995. Riding the (mono)rails: the structure of catenated DNA-tracking proteins. Chem. Biol. 2:123–125.

313. Geiduschek, E. P. 1992. Two prokaryotic transcriptional enhancer systems. Prog. Nucleic Acid Res. Mol. Biol. 43:109–133.

314. Gentz, R., and H. Bujard. 1985. Promoters recognized by *Escherichia coli* RNA polymerase selected by function: highly efficient promoters from bacteriophage T5. J. Bacteriol. 164:70–77.

315. Georgiou, T., Y.-T. N. Yu, S. Ekunwe, M. J. Buttner, A.-M. Zuurmond, B. Kraal, C. Kleanthous, and L. Snyder. 1998. Specific peptide-activated proteolytic cleavage of *Escherichia coli* elongation factor Tu. Proc. Natl. Acad. Sci. USA 95:2891–2895.

316. Georgopoulos, C. P. 1968. Location of glucosyl transferase genes on the genetic map of phage T4. Virology 34:364–366.

317. Georgopoulos, C. P., R. W. Hendrix, S. R. Casjens, and A. D. Kaiser. 1973. Host participation in bacteriophage lambda head assembly. J. Mol. Biol. 76:45–60.

318. Georgopoulos, C. P., R. W. Hendrix, A. D. Kaiser, and W. B. Wood. 1972. Role of the host cell in bacteriophage morphogenesis: effects of a bacterial mutation on T4 head assembly. Nature New Biol. 239:38–41.

319. Georgopoulos, C. P., and C. H. Linder. 1994. Molecular chaperones in T4 assembly, p. 213–217. In J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

320. Gerald, W. L., and J. D. Karam. 1984. Expression of a DNA replication gene cluster in bacteriophage T4: genetic linkage and the control of gene product interactions. Genetics 107:537–549.

321. Gerber, J. S., and D. M. Hinton. 1996. An N-terminal mutation in the bacteriophage T4 *motA* gene yields a protein that binds DNA but is defective for activation of transcription. J. Bacteriol. 178:6133–6139.

322. Gesteland, R. F., and J. F. Atkins. 1996. RECODING: dynamic reprogramming of translation. Annu. Rev. Biochem. 65:741–768.

323. Goff, C. G. 1979. Bacteriophage T4 *alt* gene maps between genes 30 and 54. J. Virol. 29:1232–1234.

324. Goff, C. G., and J. Setzer. 1980. ADP ribosylation of *Escherichia coli* RNA polymerase is nonessential for bacteriophage T4 development. J. Virol. 33:547–549.

325. Gold, L. 1988. Posttranscriptional regulatory mechanisms in *Escherichia coli*. Annu. Rev. Biochem. 57:199–233.

326. Gold, L., D. Pribnow, T. Schneider, S. Shinedling, B. S. Singer, and G. Stormo. 1981. Translational initiation in prokaryotes. Annu. Rev. Microbiol. 35:365–403.

327. Goldberg, E., L. Grinius, and L. Letellier. 1994. Recognition, attachment, and injection, p. 347–356. In J. D. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

328. Goldfarb, A., and V. Daniel. 1981. Mapping of transcription units in the bacteriophage T4 tRNA gene cluster. J. Mol. Biol. 146:393–412.

329. Goldstein, J., and S. P. Champe. 1974. T4-induced activity required for specific cleavage of a bacteriophage protein in vitro. J. Virol. 13:419–427.

330. Golz, S., R. P. Birkenbihl, and B. Kemper. 1995. Improved large-scale preparation of phage T4 endonuclease VII overexpressed in *E. coli*. DNA Res. 2:277–284.

331. Golz, S., K. Birkenkamp-Demtröder, and B. Kemper. 1998. Enzymatic mutation detection. Procedure for screening and mapping of mutations by immobilised endonuclease VII. Nucleic Acids Res. 26:1132–1133.

332. Golz, S., A. Christoph, K. Birkenkamp-Demtroder, and B. Kemper. 1997. Identification of amino acids of endonuclease VII essential for binding and cleavage of cruciform DNA. Eur. J. Biochem. 245:573–580.

333. Golz, S., and B. Kemper. 1999. Association of Holliday-structure resolving endonuclease VII with gp20 from the packaging machine of phage T4. J. Mol. Biol. 285:1131–1144.

334. Goodrich, L. D., T. C. Lin, E. K. Spicer, C. Jones, and W. H. Konigsberg. 1997. Residues at the carboxy terminus of T4 DNA polymerase are important determinants for interaction with the polymerase accessory proteins. Biochemistry 36:10474–10481.

335. Goodrich-Blair, H., and D. A. Shub. 1996. Beyond homing: competition between intron endonucleases confers a selective advantage on flanking genetic markers. Cell 84:211–221.

336. Goodrich-Blair, H., and D. A. Shub. 1994. The DNA polymerase genes of several HMU-bacteriophages have similar group I introns with highly divergent open reading frames. Nucleic Acids Res. 22:3715–3721.

337. Gorbelenya, A. E. 1994. Self-splicing group I and group II introns encode homologous (putative) DNA endonucleases of a new family. Protein Sci. 3:1117–1120.

338. Gordon, J., T. K. Sengupta, C. A. Phillips, S. M. O’Malley, K. R. Williams, and E. K. Spicer. 1999. Identification of the RNA binding domain of T4 RegA protein by structure-based mutagenesis. J. Biol. Chem. 274:32265–32273.

339. Gordon, L. K., and W. A. Haseltine. 1980. Comparison of the cleavage of pyrimidine dimers by the bacteriophage T4 and *Micrococcus luteus* UV-specific endonucleases. J. Biol. Chem. 255:12047–12050.

340. Gorski, K., J. M. Roch, P. Prentki, and H. M. Krisch. 1985. The stability of bacteriophage T4 gene 32 mRNA: a 5′ leader sequence that can stabilize mRNA transcripts. Cell 43:461–469.

341. Gott, J. M., D. A. Shub, and M. Belfort. 1986. Multiple self-splicing introns in bacteriophage T4: evidence from autocatalytic GTP labeling of RNA *in vitro*. Cell 47:81–87.

342. Gott, J. M., A. Zeh, D. Bell-Pedersen, K. Ehrenman, M. Belfort, and D. A. Shub. 1988. Genes within genes: independent expression of phage T4 intron open reading frames and the genes in which they reside. Genes Dev. 2:1791–1799.

343. Goulian, M., Z. J. Lucas, and A. Kornberg. 1968. Enzymatic synthesis of deoxyribonucleic acid. XXV. Purification and properties of deoxyribonucleic acid polymerase induced by infection with phage T4. J. Biol. Chem. 243:627–638.

344. Graham, J. E., and J. P. Richardson. 1998. rut Sites in the nascent transcript mediate Rho-dependent transcription termination in vivo. J. Biol. Chem. 273:20764–20769.

345. Gram, H., and W. Ruger. 1985. Genes 55, *alpha gt*, 47 and 46 of bacteriophage T4: the genomic organization as deduced by sequence analysis. EMBO J. 4:257–264.

346. Gray, T. M., and B. W. Matthews. 1987. Structural analysis of the temperature-sensitive mutant of bacteriophage T4 lysozyme, glycine 156→aspartic acid. J. Biol. Chem. 262:16858–16864.

347. Reference deleted.

348. Greenberg, G. R., P. He, J. Hilfinger, and M.-J. Tseng. 1994. Deoxyribonucleotide triphosphate synthesis and T4 DNA replication, p. 14–27. *In* J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

349. Greenberg, G. R., and J. M. Hilfinger. 1996. Regulation of synthesis of ribonucleotide reductase and relationship to DNA replication in various systems. Prog. Nucleic Acid Res. Mol. Biol. 53:345–395.

350. Greger, B., and B. Kemper. 1998. An apyrimidinic site kinks DNA and triggers incision by endonuclease VII of phage T4. Nucleic Acids Res. 26:4432–4438.

351. Griffith, J., and T. Formosa. 1985. The uvsX protein of bacteriophage T4 arranges single-stranded and double-stranded DNA into similar helical nucleoprotein filaments. J. Biol. Chem. 260:4484–4491.

352. Grigoriev, A. 1998. Analyzing genomes with cumulative skew diagrams. Nucleic Acids Res. 26:2286–2290.

353. Grill, S., C. O. Gualerzi, P. Londei, and U. Blasi. 2000. Selective stimulation of translation of leaderless mRNA by initiation factor 2: evolutionary implications for translation. EMBO J. 19:4101–4110.

354. Gross, C. A. 1996. Function and regulation of heat shock proteins, p. 1382–1399. *In* F. C. Neidhardt, R. Curtiss III, J. L. Ingraham, E. C. C. Lin, K. B. Low, B. Magasanik, W. S. Reznikoff, M. Riley, M. Schaechter, and H. E. Umbarger (ed.), *Escherichia coli* and *Salmonella*: cellular and molecular biology, 2nd ed., vol. 1. American Society for Microbiology, Washington, D.C.

355. Gruber, H., G. Kern, P. Gauss, and L. Gold. 1988. Effect of DNA sequence and structure on nuclease activity of the DexA protein of bacteriophage T4. J. Bacteriol. 170:5830–5836.

356. Gruidl, M. E., N. C. Canan, and G. Mosig. 1988. Bacteriophage T4 gene 25. Nucleic Acids Res. 16:9862.

357. Gruidl, M. E., T. C. Chen, S. Gargano, A. Storlazzi, A. Cascino, and G. Mosig. 1991. Two bacteriophage-T4 base plate genes (25 and 26) and the DNA repair gene *uvsY* belong to spatially and temporally overlapping transcription units. Virology 184:359–369.

358. Gruidl, M. E., and G. Mosig. 1986. Sequence and transcripts of the bacteriophage T4 DNA repair gene *uvsY*. Genetics 114:1061–1079.

359. Grunberg-Manago, M. 1999. Messenger RNA stability and its role in control of gene expression in bacteria and phages. Annu. Rev. Genet. 33:193–227.

360. Gründling, A., M. D. Manson, and R. Young. 2001. Holins kill without warning. Proc. Natl. Acad. Sci. USA 98:9348–9352.

361. Guerrier-Takada, C., W. H. McClain, and S. Altman. 1984. Cleavage of tRNA precursors by the RNA subunit of *E. coli* ribonuclease P (M1 RNA) is influenced by 3′-proximal CCA in the substrates. Cell 38:219–224.

362. Gumpert, R. I., D. M. Hinton, V. S. Pyle, and R. W. Richardson. 1980. T4 RNA ligase as a nucleic acid synthesis and modification reagent. Nucleic Acids Symp. Ser. 7:167–171.

363. Guo, J., S. Wang, J. Dong, H. Qui, R. A. Scott, and D. P. Giedroc. 1995. X-ray and visible absorption spectroscopy of wild-type and mutant T4 gene 32 proteins: His<sup>64</sup>, not His<sup>81</sup>, is the non-thiolate zinc ligand. J. Am. Chem. Soc. 117:9437–9440.

364. Gusarov, I., and E. Nudler. 1999. The mechanism of intrinsic transcription termination. Mol. Cell 3:495–504.

365. Gussin, G. N., and V. Peterson. 1972. Isolation and properties of rex-mutants of bacteriophage lambda. J. Virol. 10:760–765.

366. Guthrie, C., and W. H. McClain. 1979. Rare transfer ribonucleic acid essential for phage growth. Nucleotide sequence comparison of normal and mutant T4 isoleucine-accepting transfer ribonucleic acid. Biochemistry 18:3786–3795.

367. Guthrie, C., J. G. Seidman, M. M. Comer, R. M. Bock, F. J. Schmidt, B. G. Barrell, and W. H. McClain. 1975. The biology of bacteriophage T4 transfer RNAs. Brookhaven Symp. Biol. 1975:106–123.

368. Gvakaria, B. O., E. Hanson, E. K. Koonin, and C. K. Mathews. 1996. Identification of a second functional glutaredoxin encoded by the bacteriophage T4 genome. J. Biol. Chem. 271:15307–15310.

369. Hacker, K., and B. Alberts. 1992. Overexpression, purification, sequence analysis, and characterization of the T4 bacteriophage *dda* helicase. J. Biol. Chem. 267:20674–20681.

370. Hadjimarcou, M. I., R. J. Kokoska, T. D. Petes, and L. J. Reha-Krantz. 2001. Identification of a mutant DNA polymerase delta in *Saccharomyces cerevisiae* with an antimutator phenotype for frameshift mutations. Genetics 158:177–186.

371. Hahn, S., U. Kruse, and W. Ruger. 1986. The region of phage T4 genes 34, 33 and 59: primary structures and organization on the genome. Nucleic Acids Res. 14:9311–9327.

372. Hahn, S., and W. Ruger. 1989. Organization of the bacteriophage T4 genome between map positions 150.745 and 145.824. Nucleic Acids Res. 17:6729.

373. Hall, D. H., C. M. Povinelli, K. Ehrenman, J. Pedersen-Lane, F. Chu, and M. Belfort. 1987. Two domains for splicing in the intron of the phage T4 thymidylate synthase (*td*) gene established by nondirected mutagenesis. Cell 48:63–71.

374. Hall, D. H., R. G. Sargent, K. F. Trofatter, and D. L. Russell. 1980. Suppressors of mutations in the bacteriophage T4 gene coding for both RNA ligase and tail fiber attachment activities. J. Virol. 36:103–108.

375. Hall, D. H., and R. D. Snyder. 1981. Suppressors of mutations in the *rII* gene of bacteriophage T4 affect promoter utilization. Genetics 97:1–9.

376. Hall, D. H., and I. Tessman. 1966. T4 mutants unable to induce deoxycytidylate deaminase activity. Virology 29:339–345.

377. Hall, D. H., I. Tessman, and O. Karlstrom. 1967. Linkage of T4 genes controlling a series of steps in pyrimidine biosynthesis. Virology 31:442–448.

378. Halpern, M. E., T. Mattson, and A. W. Kozinski. 1979. Origins of phage T4 DNA replication as revealed by hybridization to cloned genes. Proc. Natl. Acad. Sci. USA 76:6137–6141.

379. Hamblin, E., F. Tétart, C. Desplats, W. H. Wilson, H. M. Krisch, and N. H. Mann. 2001. A conserved genetic module that encodes the major virion components in both the coliphage T4 and the marine cyanophage S-PM2. Proc. Natl. Acad. Sci. USA 98:11411–11416.

380. Hamlett, N. V., and H. Berger. 1975. Mutations altering genetic recombination and repair of DNA in bacteriophage T4. Virology 63:539–567.

381. Hanggi, U. J., and H. G. Zachau. 1980. Isolation and characterization of DNA fragments containing the dihydrofolate-reductase gene of coliphage T4. Gene 9:271–285.

382. Hanson, E., and C. K. Mathews. 1994. Allosteric effectors are required for subunit association in T4 phage ribonucleotide reductase. J. Biol. Chem. 269:30999–31005.

383. Hardy, L. W., K. L. Graves, and E. Nalivaika. 1995. Electrostatic guidance of catalysis by a conserved glutamic acid in *Escherichia coli* dTMP synthase and bacteriophage T4 dCMP hydroxymethylase. Biochemistry 34:8422–8432.

384. Harm, W. 1963. Mutants of phage T4 with increased sensitivity to ultraviolet. Virology 19:66–71.

385. Harper, D., V. Eryomin, T. White, and E. Kutter. 1994. Effects of T4 infection on membrane lipid synthesis, p. 385–390. *In* J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

386. Harris, L. D., and J. Griffith. 1987. Visualization of the homologous pairing of DNA catalyzed by the bacteriophage T4 UvsX protein. J. Biol. Chem. 262:9285–9292.

387. Harris, L. D., and J. D. Griffith. 1989. UvsY protein of bacteriophage T4 is an accessory protein for in vitro catalysis of strand exchange. J. Mol. Biol. 206:19–27.

388. Hartz, D., D. S. McPheeters, and L. Gold. 1991. Influence of messenger-

RNA determinants on translation initiation in *Escherichia coli*. J Mol Biol. 218:83–97.

389. Reference deleted.

390. Hashemolhosseini, S., D. Montag, L. Kramer, and U. Henning. 1994. Determinants of receptor specificity of coliphages of the T4 family. A chaperone alters the host range. J. Mol. Biol. 241:524–533.

391. Hashemolhosseini, S., Y.-D. Stierhof, I. Hindennach, and U. Henning. 1996. Characterization of the helper proteins for the assembly of tail fibers of coliphages T4 and lambda. J. Bacteriol. 178:6258–6265.

392. Hashimoto, K., and T. Yonesaki. 1991. The characterization of a complex of three bacteriophage-T4 recombination proteins, UvsX protein, UvsY protein, and gene-32 protein, on single-stranded DNA. J. Biol. Chem. 266:4883–4888.

393. Hatahet, Z., M. Zhou, L. J. Reha-Krantz, H. Ide, S. W. Morrical, and S. S. Wallace. 1999. *In vitro* selection of sequence contexts which enhance bypass of abasic sites and tetrahydrofuran by T4 DNA polymerase holoenzyme. J. Mol. Biol. 286:1045–1057.

394. Hatahet, Z., M. Zhou, L. J. Reha-Krantz, S. W. Morrical, and S. S. Wallace. 1998. In search of a mutational hotspot. Proc. Natl. Acad. Sci. USA 95:8556–8561.

395. Hattman, S., J. Wilkinson, D. Swinton, S. Schlagman, P. M. Macdonald, and G. Mosig. 1985. Common evolutionary origin of the phage T4 *dam* and host *Escherichia coli dam* DNA-adenine methyltransferase genes. J. Bacteriol. 164:932–937.

396. Haynes, J. A., and F. A. Eiserling. 1996. Modulation of bacteriophage T4 capsid size. Virology 221:67–77.

397. Hazebrouck, S., F. Maley, V. Machtelinckx, P. Sonigo, and J. J. Kupiec. 1999. Structural and functional analysis of surface domains unique to bacteriophage T4 thymidylate synthase. Biochemistry 38:2094–2101.

398. Hendricks, S. P., and C. K. Mathews. 1997. Regulation of T4 phage aerobic ribonucleotide reductase. Simultaneous assay of the four activities. J. Biol. Chem. 272:2861–2865.

399. Hendrix, R. W., and R. L. Garcea. 1994. Capsid assembly of dsDNA viruses. Virology 5:15–26.

400. Hendrix, R. W., M. C. Smith, R. N. Burns, M. E. Ford, and G. F. Hatfull. 1999. Evolutionary relationships among diverse bacteriophages and prophages: all the world’s a phage. Proc. Natl. Acad. Sci. USA 96:2192–2197.

401. Henning, U., and S. Hashemol-Hosseini. 1994. Receptor recognition by T-even-type coliphages, p. 291–298. *In* J. D. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

402. Hercules, K., J. L. Munro, S. Mendelsohn, and J. S. Wiberg. 1971. Mutants in a nonessential gene of bacteriophage T4 which are defective in the degradation of *Escherichia coli* deoxyribonucleic acid. J. Virol. 7:95–105.

403. Hercules, K., and J. S. Wiberg. 1971. Specific suppression of mutations in genes 46 and 47 by das, a new class of mutations in bacteriophage T4D. J. Virol. 8:603–612.

404. Herendeen, D. R., G. A. Kassavetis, J. Barry, B. M. Alberts, and E. P. Geiduschek. 1989. Enhancement of bacteriophage T4 late transcription by components of the T4 DNA replication apparatus. Science 245:952–958.

405. Herendeen, D. R., G. A. Kassavetis, and E. P. Geiduschek. 1992. A transcriptional enhancer whose function imposes a requirement that proteins track along DNA. Science 256:1298–1303.

406. Herman, R. E., N. Haas, and D. P. Snustad. 1984. Identification of the bacteriophage T4 *unf* (= *alc*) gene product, a protein involved in the shutoff of host transcription. Genetics 108:305–317.

407. Herman, R. E., and D. P. Snustad. 1985. Bacteriophage T4 *unf* (=*alc*) gene function is required for late replication in the presence of plasmid pR386. J. Virol. 53:430–439.

408. Herr, A. J., C. C. Nelson, N. M. Wills, R. F. Gesteland, and J. F. Atkins. 2001. Analysis of the roles of tRNA structure, ribosomal protein L9, and the bacteriophage T4 gene 60 bypassing signals during ribosome slippage on mRNA. J. Mol. Biol. 309:1029–1048.

409. Herrmann, R. 1982. Nucleotide sequence of the bacteriophage T4 gene 57 and a deduced amino acid sequence. Nucleic Acids Res. 10:1105–1112.

410. Herrmann, R., and W. B. Wood. 1981. Assembly of bacteriophage T4 tail fibers: identification and characterization of the nonstructural protein gp57. Mol. Gen. Genet. 184:125–132.

411. Hershey, A. D., and R. Rotman. 1948. Linkage among genes controlling inhibition of lysis in a bacterial virus. Proc. Natl. Acad. Sci. USA 34:89–96.

412. Heus, H. A., and A. Pardi. 1991. Structural features that give rise to the unusual stability of RNA hairpins containing GNRA loops. Science 253: 191–194.

413. Hilse, D., T. Koch, and W. Ruger. 1989. Nucleotide sequence of the alt gene of bacteriophage T4. Nucleic Acids Res. 17:6731.

414. Hintermann, E., and A. Kuhn. 1992. Bacteriophage T4 gene 21 encodes two proteins essential for phage maturation. Virology 189:474–482.

415. Hinton, D. M. 1989. Altered expression of the bacteriophage T4 gene 41 (primase-helicase) in an *Escherichia coli rho* mutant. J. Biol. Chem. 264: 14440–14446.

416. Hinton, D. M. 1989. Transcript analyses of the *uvsX-40–41* region of

bacteriophage T4. Changes in the RNA as infection proceeds. J. Biol. Chem. 264:14432–14439.

417. Hinton, D. M. 1991. Transcription from a bacteriophage T4 middle promoter using T4 MotA protein and phage-modified RNA polymerase. J. Biol. Chem. 266:18034–18044.

418. Reference deleted.

419. Hinton, D. M., R. March-Amegadzie, J. S. Gerber, and M. Sharma. 1996. Bacteriophage T4 middle transcription system: T4-modified RNA polymerase; AsiA, a σ⁷⁰ binding protein; and transcriptional activator MotA. Methods Enzymol. 274:43–57.

420. Hinton, D. M., R. March-Amegadzie, J. S. Gerber, and M. Sharma. 1996. Characterization of pre-transcription complexes made at a bacteriophage T4 middle promoter: Involvement of the T4 MotA activator and the T4 AsiA protein, a sigma-70 binding protein, in the formation of the open complex. J. Mol. Biol. 256:235–248.

421. Hinton, D. M., and N. G. Nossal. 1987. Bacteriophage T4 DNA primase-helicase. Characterization of oligomer synthesis by T4 61 protein alone and in conjunction with T4 41 protein. J. Biol. Chem. 262:10873–10878.

422. Hinton, D. M., and N. G. Nossal. 1985. Bacteriophage T4 DNA replication protein 61. Cloning of the gene and purification of the expressed protein. J. Biol. Chem. 260:12858–12865.

423. Hinton, D. M., and N. G. Nossal. 1986. Cloning of the bacteriophage T4 *uvsX* gene and purification and characterization of the T4 UvsX recombination protein. J. Biol. Chem. 261:5663–5673.

424. Hinton, D. M., L. L. Silver, and N. G. Nossal. 1985. Bacteriophage T4 DNA replication protein 41. Cloning of the gene and purification of the expressed protein. J. Biol. Chem. 260:12851–12857.

425. Hinton, D. M., and S. Vuthoori. 2000. Efficient inhibition of *Escherichia coli* RNA polymerase by the bacteriophage T4 AsiA protein requires that AsiA binds first to free σ⁷⁰. J. Mol. Biol. 304:731–739.

426. Ho, C. K., and S. Shuman. 2002. Bacteriophage T4 RNA ligase 2 (gp24.1) exemplifies a family of RNA ligases found in all phylogenetic domains. Proc. Natl. Acad. Sci. USA 99:12709–12714.

427. Hobbs, L. J., and N. G. Nossal. 1996. Either bacteriophage T4 RNase H or *Escherichia coli* DNA polymerase I is essential for phage replication. J. Bacteriol. 178:6772–6777.

428. Holland, J. A., M. R. Hansen, Z. Du, and D. W. Hoffman. 1999. An examination of coaxial stacking of helical stems in a pseudoknot motif: the gene 32 messenger RNA pseudoknot of bacteriophage T2. RNA 5:257–271.

429. Hollingsworth, H. C., and N. G. Nossal. 1991. Bacteriophage T4 encodes an RNase H which removes RNA primers made by the T4 DNA replication system in vitro. J. Biol. Chem. 266:1888–1897.

430. Homyk, T., A. Rodriguez, and J. Weil. 1976. Characterization of T4 mutants that partially suppress the inability of T4 *rII* to grow in lambda lysogens. Genetics 83:477–487.

431. Homyk, T., and J. Weil. 1974. Deletion analysis of two nonessential regions of the T4 genome. Virology 61:505–523.

432. Hong, G., and K. N. Kreuzer. 2000. An antitumor drug-induced topoisomerase cleavage complex blocks a bacteriophage T4 replication fork in vivo. Mol. Cell. Biol. 20:594–603.

433. Hong, Y. R., and L. W. Black. 1993. An expression-packaging-processing vector which selects and maintains 7-kb DNA inserts in the blue T4 phage genome. Gene 136:193–198.

434. Hong, Y. R., J. M. Mullaney, and L. W. Black. 1995. Protection from proteolysis using a T4::T7-RNAP phage expression-packaging-processing system. Gene 162:5–11.

435. Horvitz, H. R. 1974. Bacteriophage T4 mutants deficient in alteration and modification of the *Escherichia coli* RNA polymerase. J. Mol. Biol. 90: 739–750.

436. Horvitz, H. R. 1973. Polypeptide bound to the host RNA polymerase is specified by T4 control gene 33. Nature New Biol. 244:137–140.

437. Hosoda, J. 1967. A mutant of bacteriophage T4 defective in alpha-glucosyl transferase. Biochem. Biophys. Res. Commun. 27:294–298.

438. Hosoda, J., L. Burke, H. Moise, I. Kubota, and A. Tsugita. 1980. The control of T4 gene 32 helix-destabilizing protein activity in a DNA replication complex. ICN-UCLA Symp. Mol. Cell. Biol. 19:505–514.

439. Hosoda, J., and E. Mathews. 1971. DNA replication in vivo by polynucleotide-ligase defective mutants of T4. II. Effect of chloramphenicol and mutations in other genes. J. Mol. Biol. 55:155–179.

440. Hosoda, J., and H. Moise. 1978. Purification and physicochemical properties of limited proteolysis products of T4 helix destabilizing protein (gene 32 protein). J. Biol. Chem. 253:7547–7555.

441. Howard, B. D. 1967. Phage lambda mutants deficient in *r-II* exclusion. Science 158:1588–1589.

442. Howard, G. W., Jr., M. L. Wolin, and S. P. Champe. 1972. Diversity of phage internal components among members of the T-even group. Trans. N. Y. Acad. Sci. 34:36–51.

443. Hsiao, C. L., and L. W. Black. 1978. Head morphogenesis of bacteriophage T4. I. Isolation and characterization of gene 40 mutants. Virology 91:1–14.

444. Hsu, T., R. X. Wei, M. Dawson, and J. D. Karam. 1987. Identification of

two new bacteriophage T4 genes that may have roles in transcription and DNA replication. J. Virol. 61:366–374.

445. Huang, W. M. 1986. The 52-protein subunit of T4 DNA topoisomerase is homologous to the gyrA-protein of gyrase. Nucleic Acids Res. 14:7379–7390.

446. Huang, W. M. 1975. Membrane-associated proteins of T4-infected Escherichia coli. Virology 66:508–521.

447. Huang, W. M. 1992. Multiple DNA gyrase-like genes in eubacteria., p. 39–48. In T. Andoh, H. Ikeda, and M. Oguro (ed.), Molecular biology of DNA topoisomerases. CRC Press, Inc., Boca Raton, Fla.

448. Huang, W. M. 1986. Nucleotide sequence of a type II DNA topoisomerase gene. Bacteriophage T4 gene 39. Nucleic Acids Res. 14:7751–7765.

449. Huang, W. M. 1979. Positive regulation of T-even-phage DNA replication by the DNA-delay protein of gene 39. Cold Spring Harbor Symp. Quant. Biol. 43:495–499.

450. Huang, W. M., S. Z. Ao, S. Casjens, R. Orlandi, R. Zeikus, R. Weiss, D. Winge, and M. Fang. 1988. A persistent untranslated sequence within bacteriophage T4 DNA topoisomerase gene 60. Science 239:1005–1012.

451. Huang, W. M., and J. M. Buchanan. 1974. Synergistic interactions of T4 early proteins concerned with their binding to DNA. Proc. Natl. Acad. Sci. USA 71:2226–2230.

452. Huang, W. M., L. S. Wei, and S. Casjens. 1985. Relationship between bacteriophage T4 and T6 DNA topoisomerases. T6 39-protein subunit is equivalent to the combined T4 39- and 60-protein subunits. J. Biol. Chem. 260:8973–8977.

453. Huang, Y. J., M. M. Parker, and M. Belfort. 1999. Role of exonucleolytic degradation in Group I intron homing in phage T4. Genetics 153:1501–1512.

454. Huff, A. C., J. K. Leatherwood, and K. N. Kreuzer. 1989. Bacteriophage T4 DNA topoisomerase is the target of antitumor agent 4′-(9-acridinylamino)methanesulfon-m-anisidine (m-AMSA) in T4-infected Escherichia coli. Proc. Natl. Acad. Sci. USA 86:1307–1311.

454a. Hughes, K. T., and K. Mathee. 1998. The anti-sigma factors. Annu. Rev. Microbiol. 52:231–286.

455. Hunt, J. F., S. M. van der Vries, L. Henry, and J. Deisenhofer. 1997. Structural adaptations in the specialized bacteriophage T4 co-chaperonin Gp31 expand the size of the Anfinsen cage. Cell 90:361–371.

456. Hunter, C. A. 1996. Sequence-dependent DNA structure. Bioessays. 18:157–162.

457. Hunter, C. A. 1993. Sequence-dependent DNA structure. The role of base stacking interactions. J. Mol. Biol. 230:1025–1054.

458. Hurley, J. M., S. A. Chervitz, T. C. Jarvis, B. S. Singer, and L. Gold. 1993. Assembly of the bacteriophage T4 replication machine requires the acidic carboxy terminus of gene 32 protein. J. Mol. Biol. 229:398–418.

459. Igarashi, K., N. Fujita, and A. Ishihama. 1991. Identification of a subunit assembly domain in the alpha subunit of Escherichia coli RNA polymerase. J. Mol. Biol. 218:1–6.

460. Ingelman, M., P. Nordlund, and H. Eklund. 1995. The structure of a reduced mutant T4 glutaredoxin. FEBS Lett. 370:209–211.

461. Ishii, T., and M. Yanagida. 1975. Molecular organization of the shell of the T-even bacteriophage head. J. Mol. Biol. 97:655–660.

462. Ishii, T., and M. Yanagida. 1977. The two dispensable structural proteins (soc and hoc) of the T4 phage capsid; their purification and properties, isolation and characterization of the defective mutants, and their binding with the defective heads in vitro. J. Mol. Biol. 109:487–514.

463. Ishimoto, L. K., J. Elisha, and F. A. Eiserling. 1990. Expression and regulation of genes coding for three bacteriophage T4 tail tube-associated proteins. Virology 175:586–590.

464. Ishimoto, L. K., K. S. Ishimoto, A. Cascino, M. Cipollaro, and F. A. Eiserling. 1988. The structure of three bacteriophage T4 genes required for tail-tube assembly. Virology 164:81–90.

465. Ishmael, F. T., S. C. Alley, and S. J. Benkovic. 2001. Identification and mapping of protein-protein interactions between gp32 and gp59 by cross-linking. J. Biol. Chem. 276:25236–25242.

466. Iwasaki, K., B. L. Trus, P. T. Wingfield, N. Cheng, G. Campusano, V. B. Rao, and A. C. Steven. 2000. Molecular architecture of bacteriophage T4 capsid: vertex structure and bimodal binding of the stabilizing accessory protein. Soc. Virol. 271:321–333.

467. Jabbar, M. A., and L. Snyder. 1984. Genetic and physiological studies of an Escherichia coli locus that restricts polynucleotide kinase- and RNA ligase-deficient mutants of bacteriophage T4. J. Virol. 51:522–529.

468. Jaeger, L., E. Westhof, and F. Michel. 1993. Monitoring of the cooperative unfolding of the sunY group I intron of bacteriophage T4. The active form of the sunY ribozyme is stabilized by multiple interactions with 3′ terminal intron components. J. Mol. Biol. 234:331–346.

469. Janzen, D. M., M. Y. Torgov, S. N. Abbott, and M. K. Reddy. 1999. Comparison of the assembly of the bacteriophage T4 clamp loader complex (gp44/62) expressed in a cis versus trans genomic configuration. Virology 260:64–73.

470. Janzen, D. M., M. Y. Torgov, and M. K. Reddy. 1999. In vitro reconstitution of the bacteriophage T4 clamp loader complex (gp44/62). J. Biol. Chem. 274:35938–35943.


471. Jardine, P. J., and D. H. Coombs. 1998. Capsid expansion follows the initiation of DNA packaging in bacteriophage T4. J. Mol. Biol. 284:661–672.

472. Jardine, P. J., M. C. McCormick, C. Lutze-Wallace, and D. H. Coombs. 1998. The bacteriophage T4 DNA packaging apparatus targets the unexpanded prohead. J. Mol. Biol. 284:647–659.

473. Jensen, D. E., R. C. Kelly, and P. H. von Hippel. 1976. DNA “melting” proteins. II. Effects of bacteriophage T4 gene 32-protein binding on the conformation and stability of nucleic acid structures. J. Biol. Chem. 251:7215–7228.

474. Jensen, J. L., and M. Susman. 1980. A mutant of *E. coli* that restricts growth of bacteriophage T4 at elevated temperatures. Genetics 94:301–325.

475. Ji, J., and C. K. Mathews. 1993. A forward mutation assay in phage T4: application to gene 42 mutator mutations. Mutat. Res. 294:247–254.

476. Ji, J. P., and C. K. Mathews. 1991. Analysis of mutagenesis induced by a thermolabile-T4 phage deoxycytidylate hydroxymethylase suggests localized deoxyribonucleotide pool imbalance. Mol. Genet. 226:257–264.

477. Johnson, J. R., and D. H. Hall. 1973. Isolation and characterization of mutants of bacteriophage T4 resistant to folate analogs. Virology 53:413–426.

478. Jones, C. E., T. C. Mueser, K. C. Dudas, K. N. Kreuzer, and N. G. Nossal. 2001. Bacteriophage T4 gene 41 helicase and gene 59 helicase-loading protein: a versatile couple with roles in replication and recombination. Proc. Natl. Acad. Sci. USA 98:8312–8318.

479. Jones, C. E., T. C. Mueser, and N. G. Nossal. 2000. Interaction of the bacteriophage T4 gene 59 helicase loading protein and gene 41 helicase with each other and with fork, flap, and cruciform DNA. J. Biol. Chem. 275:27145–27154.

480. Jones, R. L., III, J. C. Jaskula, and G. R. Janssen. 1992. In vivo translational start site selection on leaderless mRNA transcribed from the *Streptomyces fradiae aph* gene. J. Bacteriol. 174:4753–4760.

481. Jongeneel, C. V., T. Formosa, and B. M. Alberts. 1984. Purification and characterization of the bacteriophage T4 dda protein. A DNA helicase that associates with the viral helix-destabilizing protein. J. Biol. Chem. 259:12925–12932.

482. Joo, D. M., A. Nolte, R. Calendar, Y. N. Zhou, and D. J. Jin. 1998. Multiple regions on the *Escherichia coli* heat shock transcription factor σ³² determine core RNA polymerase binding specificity. J. Bacteriol. 180:1095–1102.

483. Josslin, R. 1970. The lysis mechanism of phage T4: mutants affecting lysis. Virology 40:719–726.

484. Jozwik, C. E., and E. S. Miller. 1992. Regions of bacteriophage T4 and RB69 RegA translational repressor proteins that determine RNA-binding specificity. Proc. Natl. Acad. Sci. USA 89:5053–5057.

485. Jozwik, C. E., and E. S. Miller. 1995. RNA-protein interactions of the bacteriophage RB69 RegA translational repressor protein. Nucleic Acids Symp. Ser. 33:256–257.

486. Kaboord, B. F., and S. J. Benkovic. 1995. Accessory proteins function as matchmakers in the assembly of the T4 DNA polymerase holoenzyme. Curr. Biol. 5:149–157.

487. Kaboord, B. F., and S. J. Benkovic. 1996. Dual role of the 44/62 protein as a matchmaker protein and DNA polymerase chaperone during assembly of the bacteriophage T4 holoenzyme complex. Biochemistry 35:1084–1092.

488. Kadyrov, F. A., and J. W. Drake. 2001. Conditional coupling of leading-strand and lagging-strand DNA synthesis at bacteriophage T4 replication forks. J. Biol. Chem. 276:29559–29566.

489. Kadyrov, F. A., and V. M. Kriukov. 1996. Phage T4 *segE* gene “mobility”: high frequency of *segE* gene transfer from the plasmid into the phage genome depends on the intactness of this gene. Dokl. Akad. Nauk 346:700–704.

490. Kadyrov, F. A., M. G. Shlyapnikov, and V. M. Kryukov. 1997. A phage T4 site-specific endonuclease, SegE, is responsible for a non-reciprocal genetic exchange between T-even-related phages. FEBS Lett. 415:75–80.

491. Kai, T., H. E. Selick, and T. Yonesaki. 1996. Destabilization of bacteriophage T4 mRNAs by a mutation of gene 61.5. Genetics 144:7–14.

492. Kai, T., H. Ueno, Y. Otsuka, W. Morimoto, and T. Yonesaki. 1999. Gene 61.3 of bacteriophage T4 is the spackle gene. Virology 260:254–259.

493. Kai, T., H. Ueno, and T. Yonesaki. 1998. Involvement of other bacteriophage T4 genes in the blockade of protein synthesis and mRNA destabilization by a mutation of gene 61.5. Virology 248:148–155.

494. Kai, T., and T. Yonesaki. 2002. Multiple mechanisms for degradation of bacteriophage T4 *soc* mRNA. Genetics 160:5–12.

495. Kaiser, V. L., and L. S. Ripley. 1995. DNA nick processing by exonuclease and polymerase activities of bacteriophage T4 DNA polymerase accounts for acridine-induced mutation specificities in T4. Proc. Natl. Acad. Sci. USA 92:2234–2238.

496. Kaliman, A. V., M. A. Khasanova, V. M. Kryukov, V. I. Tanyashin, and A. A. Bayev. 1990. The nucleotide sequence of the region of bacteriophage T4 *inh(lip)-hoc* genes. Nucleic Acids Res. 18:4277.

497. Kanamaru, S., N. C. Gassner, N. Ye, S. Takeda, and F. Arisaka. 1999. The

C-terminal fragment of the precursor tail lysozyme of bacteriophage T4 stays as a structural component of the baseplate after cleavage. J. Bacteriol. 181:2739–2744.

498. Kanamaru, S., P. G. Leiman, V. A. Kostyuchenko, P. R. Chipman, V. V. Mesyanzhinov, F. Arisaka, and M. G. Rossmann. 2002. Structure of the cell-puncturing device of bacteriophage T4. Nature 415:553–557.

499. Kano-Sueoka, T., J. R. Lobry, and N. Sueoka. 1999. Intra-strand biases in bacteriophage T4 genome. Gene 238:59–64.

500. Kao, S. H., and W. H. McClain. 1980. Roles of bacteriophage T4 gene 5 and gene s products in cell lysis. J. Virol. 34:104–107.

501. Kao, S. H., and W. H. McClain. 1977. U-G-A suppressor of bacteriophage T4 associated with arginine transfer RNA. J. Biol. Chem. 252:8254–8257.

502. Karam, J., M. Bowles, and M. Leach. 1979. Expression of bacteriophage T4 genes 45, 44, and 62. I. Discoordinate synthesis of the T4 45- and 44-proteins. Virology 94:192–203.

503. Karam, J., L. Gold, B. S. Singer, and M. Dawson. 1981. Translational regulation: identification of the site on bacteriophage T4 rIIB mRNA recognized by the reg4 gene function. Proc. Natl. Acad. Sci. USA 78:4669–4673.

504. Karam, J. D., and B. Barker. 1971. Properties of bacteriophage T4 mutants defective in gene 30 (deoxyribonucleic acid ligase) and the rII gene. J. Virol. 7:260–266.

505. Karam, J. D., and M. G. Bowles. 1974. Mutation to overproduction of bacteriophage T4 gene products. J. Virol. 13:428–438.

506. Karam, J. D., J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, C. Carlson, and E. S. Miller (ed.). 1994. Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

507. Karam, J. D., and W. H. Konigsberg. 2000. DNA polymerase of the T4-related bacteriophages. Prog. Nucleic Acid Res. Mol. Biol. 64:65–96.

508. Karpel, R. L. 2002. LAST motifs and SMART domains in gene 32 protein: an unfolding story of autoregulation? IUBMB Life 53:161–166.

509. Karpel, R. L. 2001. Prokaryotic DNA-binding proteins, vol. 2001. Encyclopedia of life sciences. Nature Publishing Group, London, United Kingdom.

510. Karpel, R. L. 1990. T4 bacteriophage gene 32 protein, p. 103–130. In A. Revzin (ed.), The biology of nonspecific DNA protein interactions. CRC Press, Inc., Boca Raton, Fla.

511. Kashlev, M., E. Nudler, A. Goldfarb, T. White, and E. Kutter. 1993. Bacteriophage T4 Alc protein: a transcription termination factor sensing local modification of DNA. Cell 75:147–154.

512. Kaufman, B. A., S. M. Newman, R. L. Hallberg, C. A. Slaughter, P. S. Perlman, and R. A. Butow. 2000. In organello formaldehyde crosslinking of proteins to mtDNA: identification of bifunctional proteins. Proc. Natl. Acad. Sci. USA 97:7772–7777.

513. Kaufmann, G., and M. Amitsur. 1985. Host transfer RNA cleavage and reunion in T4-infected Escherichia coli CTr5x. Nucleic Acids Res. 13:4333–4341.

514. Kaufmann, G., M. David, G. D. Borasio, A. Teichmann, A. Paz, and M. Amitsur. 1986. Phage and host genetic determinants of the specific anticodon loop cleavages in bacteriophage T4-infected Escherichia coli CTr5X. J. Mol. Biol. 188:15–22.

515. Kaufmann, I. 2000. Anticodon nucleases. Trends Biochem. Sci. 25:70–74.

516. Keller, B. 1985. Determination of the cleavage site of the phage T4 prohead protease in gene product 68. J. Mol. Biol. 186:665–667.

517. Keller, B., and T. A. Bickle. 1986. The nucleotide sequence of gene 21 of bacteriophage T4 coding for the prohead protease. Gene 49:245–251.

518. Keller, B., J. Dubochet, M. Adrian, M. Maeder, M. Wurtz, and E. Kellenberger. 1988. Length and shape variants of the bacteriophage T4 head: mutations in the scaffolding core genes 68 and 22. J. Virol. 62:2960–2969.

519. Keller, B., M. Maeder, C. Becker-Laburte, E. Kellenberger, and T. A. Bickle. 1986. Amber mutants in gene 67 of phage T4. Effects on formation and shape determination of the head. J. Mol. Biol. 190:83–95.

520. Keller, B., C. Sengstag, E. Kellenberger, and T. A. Bickle. 1984. Gene 68, a new bacteriophage T4 gene which codes for the 17K prohead core protein is involved in head size determination. J. Mol. Biol. 179:415–430.

521. Kells, S. S., and R. Haselkorn. 1974. Bacteriophage T4 short tail fibers are the product of gene 12. J. Mol. Biol. 83:473–485.

522. Kemper, B. 1998. Branched DNA resolving enzymes (X-solvases), p. 179–204. In J. A. Nickoloff and M. Hoekstra (ed.), DNA damage and repair, vol. 1. DNA repair in prokaryotes and lower eukaryotes. Humana Press, Totowa, N.J.

523. Kemper, B., and D. T. Brown. 1976. Function of gene 49 of bacteriophage T4. II. Analysis of intracellular development and the structure of very fast-sedimenting DNA. J. Virol. 18:1000–1015.

524. Kemper, B., and M. Garabett. 1981. Studies on T4-head maturation. 1. Purification and characterization of gene-49-controlled endonuclease. Eur. J. Biochem. 115:123–131.

525. Kemper, B., M. Garabett, and U. Courage. 1981. Studies on T4-head maturation. 2. Substrate specificity of gene-49-controlled endonuclease. Eur. J. Biochem. 115:133–141.

526. Kemper, B., F. Jensch, M. U. Depka-Prondzynski, H. J. Fritz, R. U.

Borgmeyer, and M. Mizuuchi. 1984. Resolution of Holliday structures by endonuclease VII as observed in interactions with cruciform DNA. Cold Spring Harbor Symp. Quant. Biol. 49:815–825.

527. Keohavong, P., R. Shukla, A. Melacrinos, B. W. Day, and L. Reha-Krantz. 1998. Effects of bulky polycyclic aromatic hydrocarbon adducts on DNA replication by exonuclease-deficient T7 and T4 DNA polymerases. DNA Cell Biol. 17:541–549.

528. Keppel, F., B. Lipinska, D. Ang, and C. Georgopoulos. 1990. Mutational analysis of the phage T4 morphogenetic 31 gene, whose product interacts with the Escherichia coli GroEL protein. Gene 86:19–25.

529. Khan, N. N., L. J. Reha-Krantz, and G. E. Wright. 1994. Analysis of inhibitors of bacteriophage T4 DNA polymerase. Nucleic Acids Res. 22:232–237.

530. Kikuchi, Y., and J. King. 1975. Genetic control of bacteriophage T4 baseplate morphogenesis. I. Sequential assembly of the major precursor, in vivo and in vitro. J. Mol. Biol. 99:645–672.

531. Kikuchi, Y., and J. King. 1975. Genetic control of bacteriophage T4 baseplate morphogenesis. II. Mutants unable to form the central part of the baseplate. J. Mol. Biol. 99:673–694.

532. Kikuchi, Y., and J. King. 1975. Genetic control of bacteriophage T4 baseplate morphogenesis. III. Formation of the central plug and overall assembly pathway. J. Mol. Biol. 99:695–716.

533. Kim, B. C., K. Kim, E. H. Park, and C. J. Lim. 1997. Nucleotide sequence and revised map location of the arn gene from bacteriophage T4. Mol. Cell 7:694–696.

534. Kim, J.-S., and N. Davidson. 1974. Electron microscope heteroduplex study of sequence relations of T2, T4, and T6 bacteriophage DNAs. Virology 57:93–111.

535. King, J. 1968. Assembly of the tail of bacteriophage T4. J. Mol. Biol. 32:231–262.

536. King, J. 1971. Bacteriophage T4 tail assembly: four steps in core formation. J. Mol. Biol. 58:693–709.

537. King, J., and U. K. Laemmli. 1973. Bacteriophage T4 tail assembly: structural proteins and their genetic identification. J. Mol. Biol. 75:315–337.

538. King, J., and U. K. Laemmlli. 1971. Polypeptides of the tail fibres of bacteriophage T4. J. Mol. Biol. 62:465–477.

539. King, J., and W. B. Wood. 1969. Assembly of bacteriophage T4 tail fibers: the sequence of gene product interaction. J. Mol. Biol. 39:583–601.

540. Klausa, V. I., and R. G. Nivinskas. 1988. Determination of the direction of the transcription of bacteriophage T4 genes by heat induction of the transcription from recombinant plasmid P1-promoter: 2 directions in the region of genes 25–29. Genetika 24:42–52. (In Russian.)

541. Kleff, S., and B. Kemper. 1988. Initiation of heteroduplex-loop repair by T4-encoded endonuclease VII in vitro. EMBO J. 7:1527–1535.

542. Kobayashi, M., H. Saito, and H. Takahashi. 1988. Confirmation of the reading frame of bacteriophage T4 uvsY gene. Nucleic Acids Res. 16:7729.

543. Koch, T., N. Lamm, and W. Ruger. 1989. Sequencing, cloning and overexpression of genes of bacteriophage T4 between map positions 74.325 and 77.184. Nucleic Acids Res. 17:4392.

544. Koch, T., A. Raudonikiene, K. Wilkens, and W. Rüger. 1995. Overexpression, purification, and characterization of the ADP-ribosyltransferase (gpAlt) of bacteriophage T4: ADP-ribosylation of E. coli RNA polymerase modulates T4 “early” transcription. Gene Expression 4:253–264.

545. Koch, T., and W. Rüger. 1994. The ADP-ribosyltransferases (gpAlt) of bacteriophages T2, T4, and T6: sequencing of the genes and comparison of their products. Virology 203:294–298.

546. Kodadek, T. 1991. Inhibition of protein-mediated homologous pairing by a DNA helicase. J. Biol. Chem. 266:9712–9718.

547. Kodadek, T., and B. M. Alberts. 1987. Stimulation of protein-directed strand exchange by a DNA helicase. Nature 310:312–314.

548. Kodadek, T., D.-C. Gan, and K. Stemke-Hale. 1989. The phage T4 uvsY recombination protein stabilizes presynaptic filaments. J. Biol. Chem. 264:16451–16457.

549. Kodadek, T., M. L. Wong, and B. M. Alberts. 1988. The mechanism of homologous DNA strand exchange catalyzed by the bacteriophage T4 uvsX and gene 32 proteins. J. Biol. Chem. 263:9427–9436.

550. Koerner, J. F., and D. P. Snustad. 1979. Shutoff of host macromolecular synthesis after T-even bacteriophage infection. Microbiol. Rev. 43:199–223.

551. Koerner, J. F., S. K. Thies, and D. P. Snustad. 1979. Protein induced by bacteriophage T4 which is absent in Escherichia coli infected with nuclear disruption-deficient phage mutants. J. Virol. 31:506–513.

551a. Kogoma, T. 1997. Stable DNA replication: interplay between DNA replication, homologous recombination, and transcription. Microbiol. Mol. Biol. Rev. 61:212–238.

552. Kolesky, S., M. Ouhammouch, E. N. Brody, and E. P. Geiduschek. 1999. Sigma competition: the contest between bacteriophage T4 middle and late transcription. J. Mol. Biol. 291:267–281.

552a. Kolesky, S. E., M. Ouhammouch, and E. P. Geiduschek. 2002. The mechanism of transcriptional activation by the topologically DNA-linked sliding clamp of bacteriophage T4. J. Mol. Biol. 321:767–784.

553. Konan, K. V., and C. Yanofsky. 2000. Rho-dependent transcription ter-

mination in the tna operon of *Escherichia coli*: roles of the *boxA* sequence and the rut site. J. Bacteriol. 182:3981–3988.

554. Kong, D., and C. C. Richardson. 1996. Single-stranded DNA binding protein and DNA helicase of bacteriophage T7 mediate homologous DNA strand exchange. EMBO J. 15:2010–2019.

555. Konigsberg, W. H. 1995. Limited proteolysis of DNA polymerases as probe of functional domains. Methods Enzymol. 262:331–346.

556. Koonin, E., P. Bork, and C. Sander. 1994. Yeast chromosome III: new gene functions. EMBO J. 13:493–503.

557. Koonin, E. V., L. Aravind, and A. S. Kondrashov. 2000. The impact of comparative genomics on our understanding of evolution. Cell 101:573–576.

558. Kornberg, A., and T. A. Baker. 1992. DNA replication, 2nd ed. W. H. Freeman & Co., New York, N.Y.

559. Kosak, H. G., and B. W. Kemper. 1990. Large-scale preparation of T4 endonuclease-VII from over-expressing bacteria. Eur. J. Biochem. 194:779–784.

560. Kostyuchenko, V. A., G. A. Navruzbekov, L. P. Kurochkina, S. V. Strelkov, V. V. Mesyanzhinov, and M. G. Rossmann. 1999. The structure of bacteriophage T4 gene product 9: the trigger for tail contraction. Struct. Fold Des. 7:1213–22.

561. Reference deleted.

562. Kowalczykowski, S. C., N. Lonberg, J. W. Newport, and P. H. von Hippel. 1981. Interactions of bacteriophage T4-coded gene 32 protein with nucleic acids. I. Characterization of the binding interactions. J. Mol. Biol. 145:75–104.

563. Kozinski, A. W. 1983. Origins of T4 DNA replication, p. 111–119. In C. K. Mathews, E. M. Kutter, G. Mosig, and P. B. Berget (ed.), Bacteriophage T4. American Society for Microbiology, Washington, D.C.

564. Kozinski, A. W., and Z. Z. Felgenhauer. 1967. Molecular recombination in T4 bacteriophage deoxyribonucleic acid. II. Single-strand breaks and exposure of uncomplemented areas as a prerequisite for recombination. J. Virol. 1:1193–202.

565. Kozloff, L. M., and M. Lute. 1973. Bacteriophage tail components. IV. Pteroyl polyglutamate synthesis in T4D-infected *Escherichia coli*. B. J. Virol. 11:630–636.

566. Kozloff, L. M., and M. Lute. 1981. Dual functions of bacteriophage T4D gene 28 product: structural component of the viral tail baseplate central plug and cleavage enzyme for foly polyglutamates. II. Folate metabolism and polyglutamate cleavage activity of uninfected and infected *Escherichia coli* cells and bacteriophage. J. Virol. 40:645–656.

567. Kozloff, L. M., and M. Lute. 1984. Identification of bacteriophage T4D gene products 26 and 51 as baseplate hub structural components. J. Virol. 52:344–349.

568. Kozloff, L. M., and J. Zorzopulos. 1981. Dual functions of bacteriophage T4D gene 28 product: structural component of the viral tail baseplate central plug and cleavage enzyme for foly polyglutamates. I. Identification of T4D gene 28 product in the tail plug. J. Virol. 40:635–644.

569. Krabbe, M., and K. Carlson. 1991. *In vivo* restriction: sequence and structure of endonuclease II-dependent cleavage sites in bacteriophage T4 DNA. J. Biol. Chem. 266:23407–23415.

570. Krassa, K. B., L. S. Green, and L. Gold. 1991. Protein-protein interactions with the acidic COOH-terminus of the single-stranded DNA-binding protein of the bacteriophage T4. Proc. Natl. Acad. Sci. USA 88:4010–4014.

571. Kreuzer, K. N. 1998. Bacteriophage T4, a model system for understanding the mechanism of type II topoisomerase inhibitors. Biochim. Biophys. Acta 1400:339–347.

572. Kreuzer, K. N. 2000. Recombination-dependent DNA replication in phage T4. Trends Biochem. Sci. 25:165–173.

573. Kreuzer, K. N., and B. M. Alberts. 1986. Characterization of a defective phage system for the analysis of bacteriophage T4 replication origins. J. Mol. Biol. 188:185–198.

574. Kreuzer, K. N., and B. M. Alberts. 1985. A defective phage system reveals bacteriophage T4 replication origins that coincide with recombination hot spots. Proc. Natl. Acad. Sci. USA 82:3345–3349.

575. Kreuzer, K. N., and J. W. Drake. 1994. Repair of lethal DNA damage, p. 89–97. In J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

576. Kreuzer, K. N., H. W. Engman, and W. Y. Yap. 1988. Tertiary initiation of replication in bacteriophage T4. Deletion of the overlapping *uvsY* promoter/replication origin from the phage genome. J. Biol. Chem. 263:11366–11373.

577. Kreuzer, K. N., and S. W. Morrical. 1994. Initiation of DNA replication, p. 28–42. In J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

578. Reference deleted.

579. Krisch, H. M., and B. Allet. 1982. Nucleotide sequences involved in bac-

teriophage T4 gene 32 translational self-regulation. Proc. Natl. Acad. Sci. USA 79:4937–4941.

580. Krisch, H. M., D. B. Shah, and H. Berger. 1971. Replication and recombination in ligase-deficient *rII* bacteriophage T4D. J. Virol. 7:491–498.

581. Krisch, H. M., G. Van Houwe, D. Belin, W. Gibbs, and R. H. Epstein. 1977. Regulation of the expression of bacteriophage T4 genes 32 and 43. Virology 78:87–98.

582. Krylov, V. N. 1972. A mutation of T4B phage, which enhances suppression of ligase mutants with *rII* mutations. Virology 50:291–293.

583. Krylov, V. N., and T. C. Plotnikova. 1972. Genetic and physiological study of amber mutants in gene *stII* of T4B phage. Genetika 8:85–95.

584. Krylov, V. N., and T. G. Plotnikova. 1972. Effect of gene-specified suppressor *suα* on the frequency of genetic recombination of T4B phage. Genetika 8:60–64.

585. Krylov, V. N., and A. Zapadnaya. 1965. Bacteriophage T4B r mutations sensitive to temperature (*rts*). Genetika 1:7–11.

586. Kuebler, D., and V. B. Rao. 1998. Functional analysis of the DNA-packaging/terminase protein gp17 from bacteriophage T4. J. Mol. Biol. 281:803–814.

587. Kuhn, B., M. Abdel-Monem, and H. Hoffmann-Berling. 1979. DNA helicases. Cold Spring Harbor Symp. Quant. Biol. 43:63–67.

588. Kuhn, B., M. Abdel-Monem, H. Krell, and H. Hoffmann-Berling. 1979. Evidence for two mechanisms for DNA unwinding catalyzed by DNA helicases. J. Biol. Chem. 254:11343–11350.

589. Kumagai, M., T. Yamashita, M. Honda, and H. Ikeda. 1993. Effects of *uvsX*, *uvsY* and DNA topoisomerase on the formation of tandem duplications of the *rII* gene in bacteriophage T4. Genetics 135:255–264.

590. Kunisawa, T. 1999. Bacteriophage transfer RNA’s and their significance in enhancing translational efficiency. Res. Commun. Biochem. Cell Mol. Biol. 3:147–155.

591. Kunisawa, T. 1992. Synonymous codon preferences in bacteriophage T4: a distinctive use of transfer RNAs from T4 and from its host *Escherichia coli*. J. Theor. Biol. 159:287–298.

592. Kunisawa, T., S. Kanaya, and E. Kutter. 1998. Comparison of synonymous codon distribution patterns of bacteriophage and host genomes. DNA Res. 5:319–326.

593. Kuroki, E., and T. Yonesaki. 1999. DNA helicase mutants of bacteriophage T4 that are defective in DNA recombination. Mol. Gen. Genet. 262:525–533.

594. Kuroki, R., L. H. Weaver, and B. W. Matthews. 1999. Structural basis of the conversion of T4 lysozyme into a transglycosidase by reengineering the active site. Proc. Natl. Acad. Sci. USA 96:8949–8954.

595. Kurtz, M. B., and S. P. Champe. 1977. Precursors of the T4 internal peptides. J. Virol. 22:412–419.

596. Kutter, E. 1996. Analysis of bacteriophage T4 based on the completed sequence data, p. 13–28. In J. Collado-Vides, B. Magasanik, and T. Smith (ed.), Integrative approaches to molecular biology. MIT Press, Cambridge, Mass.

597. Kutter, E., R. Drivdahl, and K. Rand. 1984. Identification and characterization of the *alc* gene product of bacteriophage T4. Genetics 108:291–304.

598. Kutter, E., K. Gachechiladze, A. Poglazov, E. Marusich, M. Schneider, P. Aronsson, A. Napuli, D. Porter, and V. Mesyanzhinov. 1995. Evolution of T4-related phages. Virus Genes 11:285–297.

599. Kutter, E., B. Guttman, and K. Carlson. 1994. The transition from host to phage metabolism after T4 infection, p. 343–346. In J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

599a. Kutter, E., E. Kellenberger, K. Carlson, S. Eddy, J. Neitzel, L. Messinger, J. North, and B. Guttman. 1994. Effects of bacterial growth conditions and physiology on T4 infection, p. 406–418. In J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

600. Kutter, E., T. Stidham, B. Guttman, E. Kutter, D. Batts, S. Peterson, T. Djavakhishvili, F. Arisaka, V. Mesyanzhinov, W. Rüger, and G. Mosig. 1994. Genomic map of bacteriophage T4, p. 491–519. In J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

601. Kutter, E., T. White, M. Kashlev, M. Uzan, J. McKinney, and B. Guttman. 1994. Effects on host genome structure and expression, p. 357–368. In J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

602. Kutter, E., and J. S. Wiberg. 1969. Biological effects of substituting cytosine for 5-hydroxymethylcytosine in the deoxyribonucleic acid of bacteriophage T4. J. Virol. 4:439–453.

603. Kutter, E. M., D. Bradley, R. Schenck, B. S. Guttman, and R. Laiken.

144 MILLER ET AL.

1981. Bacteriophage T4 *alc* gene product: general inhibitor of transcription from cytosine-containing DNA. J. Virol. 40:822–829.

604. Kutter, E. M., K. d’Acci, R. H. Drivdahl, J. Gleckler, J. C. McKinney, S. Peterson, and B. S. Guttman. 1994. Identification of bacteriophage T4 prereplicative proteins on two-dimensional polyacrylamide gels. J. Bacteriol. 176:1647–1654.

605. Kutter, E. M., and J. S. Wiberg. 1968. Degradation of cytosine-containing bacterial and bacteriophage DNA after infection of *Escherichia coli* B with bacteriophage T4D wild type and with mutants defective in genes 46, 47 and 56. J. Mol. Biol. 38:395–411.

606. Laemmli, U. K. 1970. Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature 227:680–685.

607. Laemmli, U. K., F. Beguin, and G. Gujer-Kellenberger. 1970. A factor preventing the major head protein of bacteriophage T4 from random aggregation. J. Mol. Biol. 47:69–85.

608. Laemmli, U. K., E. Molbert, M. Showe, and E. Kellenberger. 1970. Form determining function of the genes required for the assembly of the head of bacteriophage T4. J. Mol. Biol. 49:99–113.

609. Lal, S. K., and D. H. Hall. 1993. A novel approach for isolation and mapping of intron mutations in a ribonucleotide reductase encoding gene (*nrdB*) of bacteriophage T4 using the white halo plaque phenotype. Biochem. Biophys. Res. Commun. 196:943–949.

610. Lambert, L. J., V. Schirf, B. Demeler, M. Cadene, and M. H. Werner. 2001. Flipping a genetic switch by subunit exchange. EMBO J. 20:7149–7159.

611. Lamm, N., J. Tomaschewski, and W. Ruger. 1987. Nucleotide sequence of the deoxycytidylate hydroxymethylase gene of bacteriophage T4 (*g42*) and the homology of its gene product with thymidylate synthase of *E. coli*. Nucleic Acids Res. 15:3920.

612. Lamm, N., Y. Wang, C. K. Mathews, and W. Ruger. 1988. Deoxycytidylate hydroxymethylase gene of bacteriophage T4. Nucleotide sequence determination and over-expression of the gene. Eur. J. Biochem. 172:553–563.

613. Landry, S. J., A. Taher, C. Georgopoulos, and S. M. van der Vies. 1996. Interplay of structure and disorder in cochaperonin mobile loops. Proc. Natl. Acad. Sci. USA 93:11622–11627.

614. Landthaler, M., U. Begley, N. C. Lau, and D. A. Shub. 2002. Two self-splicing group I introns in the ribonucleotide reductase large subunit gene of *Staphylococcus aureus* phage Twort. Nucleic Acids Res. 30:1935–1943.

615. Landthaler, M., and D. A. Shub. 1999. Unexpected abundance of self-splicing introns in the genome of bacteriophage Twort: introns in multiple genes, a single gene with three introns, and exon skipping by group I ribozymes. Proc. Natl. Acad. Sci. USA 96:7005–7010.

615a. Larivière, L., and S. Moréra. 2002. A base-flipping mechanism for the T4 phage beta-glucosyltransferase and identification of a transition-state analog. J. Mol. Biol. 29:483–490.

616. Latham, G. J., D. J. Bacheller, P. Pietroni, and P. H. von Hippel. 1997. Structural analyses of gp45 sliding clamp interactions during assembly of the bacteriophage T4 DNA polymerase holoenzyme. II. The Gp44/62 clamp loader interacts with a single defined face of the sliding clamp ring. J. Biol. Chem. 272:31677–31684.

617. Latham, G. J., D. J. Bacheller, P. Pietroni, and P. H. von Hippel. 1997. Structural analyses of gp45 sliding clamp interactions during assembly of the bacteriophage T4 DNA polymerase holoenzyme. III. The Gp43 DNA polymerase binds to the same face of the sliding clamp as the clamp loader. J. Biol. Chem. 272:31685–31692.

618. Latham, G. J., F. Dong, P. Pietroni, J. M. Dozono, D. J. Bacheller, and P. H. von Hippel. 1999. Opening of a monomer-monomer interface of the trimeric bacteriophage T4-coded GP45 sliding clamp is required for clamp loading onto DNA. Proc. Natl. Acad. Sci. USA 96:12448–12453.

619. Latham, G. J., P. Pietroni, F. Dong, M. C. Young, and P. H. von Hippel. 1996. Fluorescence monitoring of T4 polymerase holoenzyme accessory protein interactions during loading of the sliding clamp onto the template-primer junction. J. Mol. Biol. 264:426–439.

620. Latham, K. A., and R. S. Lloyd. 1995. Delta-elimination by T4 endonuclease V at a thymine dimer site requires a secondary binding event and amino acid Glu-23. Biochemistry 34:8796–8803.

621. Latham, K. A., R. C. Manuel, and R. S. Lloyd. 1995. The interaction of T4 endonuclease V E23Q mutant with thymine dimer- and tetrahydrofuran-containing DNA. J. Bacteriol. 177:5166–5168.

622. Latham, K. A., S. Rajendran, J. R. Carmical, J. C. Lee, and R. S. Lloyd. 1996. T4 endonuclease V exists in solution as a monomer and binds to target sites as a monomer. Biochim. Biophys. Acta 1292:324–334.

623. Latham, K. A., J. S. Taylor, and R. S. Lloyd. 1995. T4 endonuclease V protects the DNA strand opposite a thymine dimer from cleavage by the footprinting reagents DNase I and 1,10-phenanthroline-copper. J. Biol. Chem. 270:3765–3771.

624. Lazarevic, V. 2001. Ribonucleotide reductase genes of *Bacillus* prophages: a refuge to introns and intein coding sequences. Nucleic Acids Res. 29: 3212–3218.

625. Lazarevic, V., A. Dusterhoft, B. Soldo, H. Hilbert, C. Mauel, and D. Karamata. 1999. Nucleotide sequence of the *Bacillus subtilis* temperate bacteriophage SPβc2. Microbiology 145:1055–1067.

626. Lazarevic, V., B. Soldo, A. Düsterhöft, H. Hilbert, C. Mauël, and D.


Karamata. 1998. Introns and intein coding sequence in the ribonucleotide reductase genes of *Bacillus subtilis* temperate bacteriophage SPβ. Proc. Natl. Acad. Sci. USA 95:1692–1697.

627. Leach, D. R., R. G. Lloyd, and A. F. Coulson. 1992. The SbcCD protein of *Escherichia coli* is related to two putative nucleases in the UvrA superfamily of nucleotide-binding proteins. Genetica 87:95–100.

628. Lebars, I., R. M. Hu, J. Y. Lallemand, M. Uzan, and F. Bontems. 2001. Role of the substrate conformation and of the S1 protein in the cleavage efficiency of the T4 endoribonuclease RegB. J. Biol. Chem. 276:13264–13272.

629. Lefebvre, S. D., and S. W. Morrical. 1997. Interactions of the bacteriophage T4 gene 59 protein with single-stranded polynucleotides: binding parameters and ion effects. J. Mol. Biol. 272:312–326.

630. Lefebvre, S. D., M. L. Wong, and S. W. Morrical. 1999. Simultaneous interactions of bacteriophage T4 DNA replication proteins gp59 and gp32 with single-stranded (ss) DNA. Co-modulation of ssDNA binding activities in a DNA helicase assembly intermediate. J. Biol. Chem. 274:22830–22838.

631. Leffers, G., and V. B. Rao. 2000. Biochemical characterization of an ATPase activity associated with the large packaging subunit gp17 from bacteriophage T4. J. Biol. Chem. 275:37127–37136.

632. Leibo, S. P., E. Kellenberger, C. Kellenberger-van der Kamp, T. G. Frey, and C. M. Steinberg. 1979. Gene 24-controlled osmotic shock resistance in bacteriophage T4: probable multiple gene functions. J. Virol. 30:327–338.

633. Leiman, P. G., V. A. Kostyuchenko, M. M. Shneider, L. P. Kurochkina, V. V. Mesyanzhinov, and M. G. Rossmann. 2000. Structure of bacteriophage T4 gene product 11, the interface between the baseplate and short tail fibers. J. Mol. Biol. 301:975–985.

634. LeMaster, D. M. 1986. Nucleotide sequence and protein overproduction of bacteriophage T4 thioredoxin. J. Virol. 59:759–760.

635. Leonetti, J. P., K. Wong, and E. P. Geiduschek. 1998. Core-sigma interaction: probing the interaction of the bacteriophage T4 gene 55 promoter recognition protein with *E. coli* RNA polymerase core. EMBO J. 17:1467–1475.

636. Leslie, A. G., S. Arnott, R. Chandrasekaran, and R. L. Ratliff. 1980. Polymorphism of DNA double helices. J. Mol. Biol. 143:49–72.

637. Liang, Y. M., R. X. Wei, T. Hsu, C. Alford, M. Dawson, and J. Karam. 1988. Autogenous regulation of the *regA* gene of bacteriophage T4: derepression of translation. Genetics 119:743–749.

638. Licis, N., J. van Duin, Z. Balklava, and V. Berzins. 1998. Long-range translational coupling in single-stranded RNA bacteriophages: an evolutionary analysis. Nucleic Acids Res. 26:3242–3246.

639. Liebig, H.-D., and W. Rüger. 1989. Bacteriophage T4 early promoter regions. Consensus sequences of promoters and ribosome-binding sites. J. Mol. Biol. 208:517–536.

640. Reference deleted.

641. Lin, G. W. 1988. Ph.D. dissertation. Vanderbilt University, Nashville, Tenn.

642. Lin, H., and L. W. Black, 1998. DNA requirements in vivo for phage T4 packaging. Virology 242:118–127.

643. Lin, H., V. B. Rao, and L. W. Black. 1999. Analysis of capsid portal protein and terminase functional domains: interaction sites required for DNA packaging in bacteriophage T4. J. Mol. Biol. 289:249–260.

644. Lin, H., M. N. Simon, and L. W. Black. 1997. Purification and characterization of the small subunit of phage T4 terminase, gp16, required for DNA packaging. J. Biol. Chem. 272:3495–3501.

645. Lin, T. C., G. Karam, and W. H. Konigsberg. 1994. Isolation, characterization, and kinetic properties of truncated forms of T4 DNA polymerase that exhibit 3′-5′ exonuclease activity. J. Biol. Chem. 269:19286–19294.

646. Lin, T. C., J. Rush, E. K. Spicer, and W. H. Konigsberg. 1987. Cloning and expression of T4 DNA polymerase. Proc. Natl. Acad. Sci. USA 84:7000–7004.

647. Lipinska, B., A. S. Rao, B. M. Bolten, R. Balakrishnan, and E. B. Goldberg. 1989. Cloning and identification of bacteriophage T4 gene 2 product gp2 and action of gp2 on infecting DNA in vivo. J. Bacteriol. 171:488–497.

648. Lipinska, B., A. S. M. K. Rao, B. M. Bolten, R. Balakrishnan, and E. B. Goldberg. 1989. Cloning and identification of bacteriophage T4 gene 2 product gp2 and action of gp2 on infecting DNA in vivo. J. Bacteriol. 171:488–497.

649. Little, J. W. 1973. Mutants of bacteriophage T4 which allow amber mutants of gene 32 to grow in ochre-suppressing hosts. Virology 53:47–59.

650. Liu, B., and B. M. Alberts. 1995. Head-on collision between a DNA replication apparatus and RNA polymerase transcription complex. Science 267:1131–1137.

651. Liu, C. C., and B. M. Alberts. 1981. Characterization of RNA primer synthesis in the T4 bacteriophage in vitro DNA replication system. J. Biol. Chem. 256:2821–2829.

652. Liu, C. C., and B. M. Alberts. 1981. Characterization of the DNA-dependent GTPase activity of T4 gene 41 protein, an essential component of the T4 bacteriophage DNA replication apparatus. J. Biol. Chem. 256:2813–2820.

653. Liu, C. C., R. L. Burke, U. Hibner, J. Barry, and B. Alberts. 1979. Probing

DNA replication mechanisms with the T4 bacteriophage in vitro system. Cold Spring Harbor Symp. Quant. Biol. 43:469–487.

654. Liu, L. F., C. C. Liu, and B. M. Alberts. 1979. T4 DNA topoisomerase: a new ATP-dependent enzyme essential for initiation of T4 bacteriophage DNA replication. Nature 281:456–461.

655. Reference deleted.

656. Lloyd, R. S. 1999. The initiation of DNA base excision repair of dipyrimidine photoproducts. Prog. Nucleic Acid Res. Mol. Biol. 62:155–175.

657. Lloyd, R. S., and P. C. Hanawalt. 1981. Expression of the denV gene of bacteriophage T4 cloned in *Escherichia coli*. Proc. Natl. Acad. Sci. USA 78:2796–2800.

658. Loayza, D., A. J. Carpousis, and H. M. Krisch. 1991. Gene 32 transcription and mRNA processing in T4-related bacteriophages. Mol. Microbiol. 5:715–725.

659. Reference deleted.

660. Logan, D. T., J. Andersson, B. M. Sjöberg, and P. Nordlund. 1999. A glycy radical site in the crystal structure of a class III ribonucleotide reductase. Science 283:1499–1504.

661. Loizos, N., G. H. Silva, and M. Belfort. 1996. Intron-encoded endonuclease I-TevII binds across the minor groove and induces two distinct conformational changes in its DNA substrate. J. Mol. Biol. 255:412–424.

662. Loizos, N., E. R. M. Tillier, and M. Belfort. 1994. Evolution of mobile group I introns: recognition of intron sequences by an intron-encoded endonuclease. Proc. Natl. Acad. Sci. USA 91:11983–11987.

663. Lonberg, N., S. C. Kowalczykowski, L. S. Paul, and P. H. von Hippel. 1981. Interactions of bacteriophage T4-coded gene 32 protein with nucleic acids. III. Binding properties of two specific proteolytic digestion products of the protein (G32P*I and G32P*III). J Mol Biol. 145:123–138.

664. Lowe, T. M., and S. R. Eddy. 1997. tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence. Nucleic Acids Res. 25:955–964.

665. Lu, M. J., and U. Henning. 1989. The immunity (*imm*) gene of *Escherichia coli* bacteriophage T4. J. Virol. 63:3472–3478.

666. Lu, M. J., Y. D. Stierhof, and U. Henning. 1993. Location and unusual membrane topology of the immunity protein of the *Escherichia coli* phage T4. J. Virol. 67:4905–4913.

667. Luder, A. 1981. Ph.D. thesis. Vanderbilt University, Nashville, Tenn.

668. Luder, A., and G. Mosig. 1982. Two alternative mechanisms for initiation of DNA replication forks in bacteriophage T4: priming by RNA polymerase and by recombination. Proc. Natl. Acad. Sci. USA 79:1101–1105.

669. Luftig, R. B., and C. Ganz. 1972. Bacteriophage T4 head morphogenesis. IV. Comparison of gene 16-, 17-, and 49-defective head structures. J. Virol. 10:545–554.

670. Luftig, R. B., W. B. Wood, and R. Okinaka. 1971. Bacteriophage T4 head morphogenesis. On the nature of gene 49-defective heads and their role as intermediates. J. Mol. Biol. 57:555–573.

671. Lukashin, A. V., and M. Borodovsky. 1998. GeneMark.hmm: new solutions for gene finding. Nucleic Acids Res. 26:1107–1115.

672. Luke, K., A. Radek, X. Liu, J. Campbell, M. Uzan, R. Haselkorn, and Y. Kogan. 2002. Microarray analysis of gene expression during bacteriophage T4 infection. Virology 299:182–191.

673. Macchiato, M. F., G. F. Grossi, and A. Cascino. 1979. Roles of gene 45 product into T4 DNA replication and late gene expression of: temperature reversibility effect. FEBS Lett. 104:187–192.

674. Macdonald, P. M. 1983. Ph.D. dissertation, Vanderbilt University, Nashville, Tenn.

675. Macdonald, P. M., E. Kutter, and G. Mosig. 1984. Regulation of a bacteriophage T4 late gene, *soc*, which maps in an early region. Genetics 106:17–27.

676. Macdonald, P. M., and G. Mosig. 1984. Cloning and physical mapping of an early region of the bacteriophage T4 genome. Genetics 106:1–16.

677. Macdonald, P. M., and G. Mosig. 1984. Regulation of a new bacteriophage T4 gene, 69, that spans an origin of DNA replication. EMBO J. 3:2863–2871.

678. Macdonald, P. M., R. M. Seaby, W. Brown, and G. Mosig (ed.). 1983. Initiator DNA from a primary origin and induction of a secondary origin of bacteriophage T4 DNA replication. American Society for Microbiology, Washington, D.C.

679. Mace, D. C., and B. M. Alberts. 1984. Characterization of the stimulatory effect of T4 gene 45 protein and the gene 44/62 protein complex on DNA synthesis by T4 DNA polymerase. J. Mol. Biol. 177:313–327.

680. Maine, I. P., and T. Kodadek. 1994. Inhibition of the DNA unwinding and ATP hydrolysis activities of the bacteriophage T4 Dda helicase by a sequence specific DNA-protein complex. Biochem. Biophys. Res. Commun. 198:1070–1077.

681. Maldonado, R., and A. J. Herr. 1998. Efficiency of T4 gene 60 translational bypassing. J. Bacteriol. 180:1822–1830.

682. Maley, G. F., B. W. Duceman, A. M. Wang, J. Martinez, and F. Maley. 1990. Cloning, sequence analysis, and expression of the bacteriophage T4 cd gene. J. Biol. Chem. 265:47–51.

683. Malygin, E. G., N. A. Petrov, Y. A. Gorbunov, V. G. Kossykh, and S. Hattman. 1997. Interaction of the phage T4 Dam DNA-[N⁶-adenine] methyltransferase with oligonucleotides containing native or modified (defective) recognition sites. Nucleic Acids Res. 25:4393–4399.

684. Manne, V., V. B. Rao, and L. W. Black. 1982. A bacteriophage T4 DNA packaging related DNA-dependent ATPase-endonuclease. J. Biol. Chem. 257:13223–13232.

685. Manuel, R. C., K. A. Latham, M. L. Dodson, and R. S. Lloyd. 1995. Involvement of glutamic acid 23 in the catalytic mechanism of T4 endonuclease V. J. Biol. Chem. 270:2652–2661.

686. March-Amegadzie, R., and D. M. Hinton. 1995. The bacteriophage T4 middle promoter P<sub>uvsx</sub>: analysis of regions important for binding of the T4 transcriptional activator MotA and for activation of transcription. Mol. Microbiol. 15:649–660.

687. Reference deleted.

688. Marchin, G. L. 1980. Mutations in a nonessential viral gene permit bacteriophage T4 to form plaques on *Escherichia coli valS ts relA*. Science 209:294–295.

689. Marquez, L. A., and L. J. Reha-Krantz. 1996. Using 2-aminopurine fluorescence and mutational analysis to demonstrate an active role of bacteriophage T4 DNA polymerase in strand separation required for 3′ → 5′-exonuclease activity. J. Biol. Chem. 271:28903–28911.

690. Reference deleted.

691. Marsh, R. C., A. M. Breschkin, and G. Mosig. 1971. Origin and direction of bacteriophage T4 DNA replication. II. A gradient of marker frequencies in partially replicated T4 DNA as assayed by transformation. J. Mol. Biol. 60:213–233.

692. Marshall, P., M. Sharma, and D. M. Hinton. 1999. The bacteriophage T4 transcriptional activator MotA accepts various base-pair changes within its binding sequence. J Mol Biol. 285:931–944.

693. Marusich, E. I., L. P. Kurochkina, and V. V. Mesyanzhinov. 1998. Chaperones in bacteriophage T4 assembly. Biochemistry (Moscow) 63:399–406.

694. Marusich, E. I., and V. V. Mesyanzhinov. 1989. Nucleotide and deduced amino acid sequences of bacteriophage T4 gene 20. Nucleic Acids Res. 17:7514.

695. Mathews, C. K. 1993. The cell—bag of enzymes or network of channels? J. Bacteriol. 175:6377–6381.

696. Mathews, C. K. 1993. Enzyme organization in DNA precursor biosynthesis. Prog. Nucleic Acid Res. Mol Biol. 44:167–203.

697. Mathews, C. K., E. Kutter, G. Mosig, and P. B. Berget (ed.). 1983. Bacteriophage T4. American Society for Microbiology, Washington, D.C.

698. Mathews, C. K., L. J. Wheeler, C. Ungermann, J. P. Young, and N. B. Ray. 1993. Enzyme interactions involving T4 phage-coded thymidylate synthase and deoxycytidylate hydroxymethylase. Adv. Exp. Med. Biol. 338:563–570.

699. Matsui, T., B. Griniuviene, E. Goldberg, A. Tsugita, N. Tanaka, and F. Arisaka. 1997. Isolation and characterization of a molecular chaperone, gp57A, of bacteriophage T4. J. Bacteriol. 179:1846–1851.

700. Matsuzaki, S., T. Inoue, M. Kuroda, S. Kimura, and S. Tanaka. 1998. Cloning and sequencing of major capsid protein (mcp) gene of a vibriophage, KVP20, possibly related to T-even coliphages. Gene 222:25–30.

701. Matsuzaki, S., T. Inoue, and S. Tanaka. 1998. A vibriophage, KVP40 with major capsid protein homologous to gp23 of coliphage T4. Virology 242:314–318.

702. Matsuzaki, S., M. Kuroda, S. Kimura, and S. Tanaka. 1999. Vibriophage KVP40 and coliphage T4 genomes share a homologous 7-kbp region immediately upstream of the gene encoding the major capsid protein. Arch. Virol. 144:2007–2012.

703. Matsuzaki, S., S. Tanaka, T. Koga, and T. Kawata. 1992. A broad-host-range vibriophage, KVP40, isolated from sea water. Microbiol. Immunol. 36:93–97.

704. Matthews, B. W. 1996. Structural and genetic analysis of the folding and function of T4 lysozyme. FASEB J. 10:35–41.

705. Matthews, B. W., and S. J. Remington. 1974. The three-dimensional structure of the lysozyme from bacteriophage T4. Proc. Natl. Acad. Sci. USA 71:4178–4182.

706. Mattson, T., J. Richardson, and D. Goodin. 1974. Mutant of bacteriophage T4D affecting expression of many early genes. Nature 250:48–50.

707. Mazzara, G. P., G. D. Plunkett, and W. H. McClain. 1981. DNA sequence of the transfer RNA region of bacteriophage T4: implications for transfer RNA synthesis. Proc. Natl. Acad. Sci. USA 78:889–892.

708. McCarthy, D. 1979. Gyrase-dependent initiation of bacteriophage T4 DNA replication: interactions of *Escherichia coli* gyrase with novobiocin, coumermycin and phage DNA-delay gene products. J. Mol. Biol. 127:265–283.

709. McCarthy, J. E., and C. Gualerzi. 1990. Translational control of prokaryotic gene expression. Trends Genet. 6:78–85.

710. McClain, W. H. 1988. Specific duplications fostered by a DNA structure containing adjacent inverted repeat sequences. J. Mol. Biol. 204:27–40.

711. McClain, W. H. 1970. UAG suppressor coded by bacteriophage T4. FEBS Lett. 6:99–101.

712. McClain, W. H., and K. Foss. 1984. Hybrid transfer RNA genes in phage T4. Cell 38:225–231.

713. McClain, W. H., G. L. Marchin, and F. C. Neidhardt. 1975. A gene of

bacteriophage T4 controlling the modification of host valy-tRNA synthetase. Virology 67:385–394.

714. McCullough, A. K., M. L. Dodson, O. D. Scharer, and R. S. Lloyd. 1997. The role of base flipping in damage recognition and catalysis by T4 endonuclease V. J. Biol. Chem. 272:27210–27217.

715. McCullough, A. K., M. T. Romberg, S. Nyaga, Y. Wei, T. G. Wood, J. S. Taylor, J. L. Van Etten, M. L. Dodson, and R. S. Lloyd. 1998. Characterization of a novel cis-syn and trans-syn-II pyrimidine dimer glycosylase/AP lyase from a eukaryotic algal virus, *Paramecium bursaria* chlorella virus-1. J. Biol. Chem. 273:13136–13142.

716. McCullough, A. K., O. Scharer, G. L. Verdine, and R. S. Lloyd. 1996. Structural determinants for specific recognition by T4 endonuclease V. J. Biol. Chem. 271:32147–32152.

717. McGaughey, K. M., L. J. Wheeler, J. T. Moore, G. F. Maley, F. Maley, and C. K. Mathews. 1996. Protein-protein interactions involving T4 phage-coded deoxycytidylate deaminase and thymidylate synthase. J. Biol. Chem. 271:23037–23042.

718. McMillan, S., H. J. Edenberg, E. H. Radany, R. C. Friedberg, and E. C. Friedberg. 1981. *DenV* gene of bacteriophage T4 codes for both pyrimidine dimer-DNA glycosylase and apyrimidinic endonuclease activities. J. Virol. 40:211–223.

719. McNicol, L. A., and L. E. Simon. 1977. A mutation which bypasses the requirement for p24 in bacteriophage T4 capsid morphogenesis. J. Mol. Biol. 116:261–283.

720. McPheeters, D. S., A. Christensen, E. T. Young, G. Stormo, and L. Gold. 1986. Translational regulation of expression of the bacteriophage T4 lysozyme gene. Nucleic Acids Res. 14:5813–5826.

721. Mei-hao, H., W. Ai, and H. Hui-fen. 1982. Purification of T4 RNA ligase by dextran blue-Sepharose 4B affinity chromatography. Anal. Biochem. 125:1–5.

722. Melamede, R. J., and S. S. Wallace. 1980. Properties of the nonlethal recombinational repair deficient mutants of bacteriophage T4. III. DNA replicative intermediates and T4w. Mol. Genet. 177:501–509.

723. Melamede, R. J., and S. S. Wallace. 1977. Properties of the nonlethal recombinational repair x and y mutants of bacteriophage T4. II. DNA synthesis. J. Virol. 24:28–40.

724. Menken, A. E., and K. N. Kreuzer. 1988. Deletion analysis of bacteriophage T4 tertiary origins. A promoter sequence is required for a rifampicin-resistant replication origin. J. Biol. Chem. 263:11358–11365.

725. Merrill, C. R., B. Biswas, R. Carlton, N. C. Jensen, G. J. Creed, S. Zullo, and S. Adhya. 1996. Long-circulating bacteriophage as antibacterial agents. Proc. Natl. Acad. Sci. USA 93:3188–3192.

726. Reference deleted.

727. Mesyanzhinov, V. V., J. Robben, B. Grymonprez, V. A. Kostyuchenko, M. V. Bourkaltseva, N. N. Sykilinda, V. N. Krylov, and G. Volckaert. 2002. The genome of bacteriophage phiKZ of *Pseudomonas aeruginosa*. J. Mol. Biol. 317:1–19.

728. Mesyanzhinov, V. V., B. N. Sobolev, E. I. Marusich, A. G. Prilipov, and V. P. Efimov. 1990. A proposed structure of bacteriophage T4 gene product 22—a major prohead scaffolding core protein. J. Struct. Biol. 104:24–31.

729. Michaud, G., A. Zachary, V. B. Rao, and L. W. Black. 1989. Membrane-associated assembly of a phage T4 DNA entrance vertex structure studied with expression vectors. J. Mol. Biol. 209:667–681.

730. Michel, F., M. Costa, C. Massire, and E. Westhof. 2000. Modeling RNA tertiary structure from patterns of sequence variation. Methods Enzymol. 317:491–510.

731. Mickelson, C., and J. S. Wiberg. 1981. Membrane-associated DNase activity controlled by genes 46 and 47 of bacteriophage T4 D and elevated DNase activity associated with the T4 *das* mutation. J. Virol. 40:65–77.

732. Midgley, C. A., and N. E. Murray. 1985. T4 polynucleotide kinase; cloning of the gene (*pseT*) and amplification of its product. EMBO J. 4:2695–2703.

733. Mileham, A. J., N. E. Murray, and H. R. Revel. 1984. Gamma-T4 hybrid bacteriophage carrying the thymidine kinase gene of bacteriophage T4. J. Virol. 50:619–622.

734. Mileham, A. J., H. R. Revel, and N. E. Murray. 1980. Molecular cloning of the T4 genome; organization and expression of the frd—DNA ligase region. Mol Gen Genet. 179:227–239.

734a. Reference deleted.

735. Miller, E. S., and C. E. Jozwik. 1990. Sequence analysis of conserved *regA* and variable Orf43.1 genes in T4-like bacteriophages. J. Bacteriol. 172:5180–5186.

736. Miller, E. S., J. D. Karam, and E. Spicer. 1994. Control of translational initiation: mRNA structure and protein repressors, p. 193–208. In J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

737. Miller, E. S., G. C. Shih, S. K. Chang, and D. N. Ballard. 1997. An *E. coli* B mutation, *rpoB5081*, that prevents growth of phage T4 strains defective in host DNA degradation. FEMS Microbiol Lett. 157:109–116.

738. Miller, E. S., R. B. Winter, K. M. Campbell, S. D. Power, and L. Gold.

1985. Bacteriophage T4 regA protein. Purification of a translational repressor. J. Biol. Chem. 260:13053–13059.

739. Minagawa, T., H. Fujisawa, T. Yonesaki, and Y. Ryo. 1988. Function of cloned T4 recombination genes, *uvsX* and *uvsY*, in cells of *Escherichia coli*. Mol. Gen. Genet. 211:350–356.

740. Minagawa, T., and Y. Ryo. 1979. Genetic control of formation of very fast sedimenting DNA of bacteriophage T4. Mol. Gen. Genet. 170:113–115.

741. Minakhin, J., J. A. Camarero, M. Holford, C. Parker, T. W. Muir, and K. Severinov. 2001. Mapping the molecular interface between s70 subunit of E. coli RNA polymerase and T4 AsiA. J. Mol. Biol. 306:631–642.

742. Miner, Z., and S. Hattman. 1988. Molecular cloning, sequencing, and mapping of the bacteriophage T2 *dam* gene. J. Bacteriol. 170:5177–5184.

743. Miner, Z., S. L. Schlagman, and S. Hattman. 1989. Single amino acid changes that alter the DNA sequence specificity of the DNA-[N6-adenine] methyltransferase (Dam) of bacteriophage T4. Nucleic Acids Res. 17:8149–8157.

744. Minner, C. A., and H. Bernstein. 1976. Genes 46 and 47 of phage T4: possible compensation for loss of their function. I. Gen. Virol. 31:277–280.

745. Miroshnikov, K. A., E. I. Marusich, M. E. Cerritelli, N. Cheng, C. C. Hyde, A. C. Steven, and V. V. Mesyanzhinov. 1998. Engineering trimeric fibrous proteins based on bacteriophage T4 adhesins. Protein Eng. 11:329–332.

746. Mitchell, M. S., S. Matsuzaki, S. Imai, and V. B. Rao. 2002. Sequence analysis of bacteriophage T4 DNA packaging/terminase genes 16 and 17 reveals a common ATPase center in the large subunit of viral terminases. Nucleic Acids Res. 30:4009–4021.

747. Moarefi, I., D. Jeruzalmi, J. Turner, M. O'Donnell, and J. Kuriyan. 2000. Crystal structure of the DNA polymerase processivity factor of T4 bacteriophage. J. Mol. Biol. 296:1215–1223.

748. Monod, C., F. Repoila, M. Kutateladze, F. Tétart, and H. M. Krisch. 1997. The genome of the pseudo T-even bacteriophages, a diverse group that resembles T4. J. Mol. Biol. 267:237–249.

749. Montag, D., M. Degen, and U. Henning. 1987. Nucleotide sequence of gene *t* (lysis gene) of the *E. coli* phage T4. Nucleic Acids Res. 15:6736.

750. Montag, D., S. Hashemolhosseini, and U. Henning. 1990. Receptor-recognizing proteins of T-even type bacteriophages—the receptor-recognizing area of proteins-37 of phages-T4 Tula and TuIb. J. Mol. Biol. 216:327–334.

751. Montag, D., I. Riede, M. L. Eschbach, M. Degen, and U. Henning. 1987. Receptor-recognizing proteins of T-even type bacteriophages. Constant and hypervariable regions and an unusual case of evolution. J. Mol. Biol. 196:165–174.

752. Montag, D., H. Schwarz, and U. Henning. 1989. A component of the side tail fiber of *Escherichia coli* bacteriophage λ can functionally replace the receptor-recognizing part of a long tail fiber protein of the unrelated bacteriophage T4. J. Bacteriol. 171:4378–4384.

753. Moody, M. F. 1971. Application of optical diffraction to helical structures in the bacteriophage tail. Philos. Trans. R. Soc. London Ser. B 261:181–195.

754. Mooney, D. T., J. Stockard, M. L. Parker, and A. H. Doermann. 1987. Genetic control of capsid length in bacteriophage T4: DNA sequence analysis of petite and petite/giant mutants. J. Virol. 61:2828–2834.

755. Moore, J. T., J. M. Ciesla, L. M. Changchien, G. F. Maley, and F. Maley. 1994. Identification of a site necessary for allosteric regulation in T4-phage deoxycytidylate deaminase. Biochemistry 33:2104–2112.

756. Moore, J. T., R. E. Silversmith, G. F. Maley, and F. Maley. 1993. T4-phage deoxycytidylate deaminase is a metalloprotein containing two zinc atoms per subunit. J. Biol. Chem. 268:2288–2291.

757. Morera, S., A. Imberty, U. Aschke-Sonnenborn, W. Ruger, and P. S. Freemont. 1999. T4 phage beta-glucosyltransferase: substrate binding and proposed catalytic mechanism. J. Mol. Biol. 292:717–730.

758. Morikawa, K., O. Matsumoto, M. Tsujimoto, K. Katauanagi, M. Ariyoshi, T. Doi, M. Ikehara, T. Inaoka, and E. Ohtsuka. 1992. X-ray structure of T4 endonuclease V: an excision repair enzyme specific for a pyrimidine dimer. Science 256:523–526.

759. Morita, M., Y. Tanji, Y. Orito, K. Mizoguchi, A. Soejima, and H. Unno. 2001. Functional analysis of antibacterial activity of *Bacillus amyloliquefaciens* phage endolysin against Gram-negative bacteria. FEBS Lett. 500:56–59.

760. Morrical, S. W., H. T. H. Beernink, A. Dash, and K. Hempstead. 1996. The gene 59 protein of bacteriophage T4. Characterization of protein-protein interactions with gene 32 protein, the T4 single-stranded DNA binding protein. J. Biol. Chem. 271:20198–20207.

761. Morrical, S. W., K. Hempstead, and M. D. Morrical. 1994. The gene 59 protein of bacteriophage T4 modulates the intrinsic and single-stranded DNA-stimulated ATPase activities of gene 41 protein, the T4 replicative DNA helicase. J. Biol. Chem. 269:33069–33081.

762. Morrical, S. W., M. L. Wong, and B. M. Alberts. 1991. Amplification of snap-back DNA synthesis reactions by the *uvsX* recombinase of bacteriophage-T4. J. Biol. Chem. 266:14031–14038.

763. Mosher, R. A., and C. K. Mathews. 1979. Bacteriophage T4-coded dihydrofolate reductase: synthesis, turnover, and location of the virion protein. J. Virol. 31:94–103.

764. Mosig, G. 1985. Bacteriophage T4 gene 32 participates in excision repair as well as recombinational repair of UV damages. Genetics 110:159–171.
765. Mosig, G. 1987. The essential role of recombination in phage T4 growth. Annu. Rev. Genet. 21:347–371.
766. Mosig, G. (ed.). 1978. Evidence for a complex that coordinates different steps in general genetic recombination. Springer-Verlag KG, Berlin, Germany.
767. Mosig, G. 1963. Genetic recombination in bacteriophage T4 during replication of DNA fragments. Cold Spring Harbor Symp. Quant. Biol. 28: 35–42.
768. Mosig, G. 1994. Homologous recombination, p. 54–82. In J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.
769. Mosig, G. 1998. Recombination and recombination-dependent DNA replication in bacteriophage T4. Annu. Rev. Genet. 32:379–413.
770. Mosig, G. 1970. Recombination in bacteriophage T4. Adv. Genet. 15:1–53.
771. Mosig, G. 1983. Relationship of T4 DNA replication and recombination, p. 120–130. In C. K. Mathews, E. M. Kutter, G. Mosig, and P. B. Berget (ed.), Bacteriophage T4. American Society for Microbiology, Washington, D.C.
772. Mosig, G. 1994. Synthesis and maturation of T4-encoded tRNAs, p. 182–185. In J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.
773. Mosig, G. 1994. T4 bacteriophage and related bacteriophages, p. 1376–1383. In R. G. Webster and A. Granoff (ed.), Encyclopedia of virology, vol. 3. Academic Press, Inc., San Diego, Calif.
774. Mosig, G., W. Berquist, and S. Bock. 1977. Multiple interactions of a DNA-binding protein *in vivo*. III. Phage T4 gene-32 mutations differentially affect insertion-type recombination and membrane properties. Genetics 86:5–23.
775. Mosig, G., and S. Bock. 1976. Gene 32 protein of bacteriophage T4 moderates the activities of the T4 gene 46/47-controlled nuclease and of the Escherichia coli RecBC nuclease in vivo. J. Virol. 17:756–761.
776. Mosig, G., and A. M. Breschkin. 1975. Genetic evidence for an additional function of phage T4 gene 32 protein: interaction with ligase. Proc. Natl. Acad. Sci. USA 72:1226–1230.
777. Mosig, G., and R. Calendar. 2002. Horizontal gene transfer in bacteriophages, p. 141–146. In M. Syvanen and C. I. Kado (ed.), Horizontal gene transfer, 2nd ed. San Diego, Calif.
778. Mosig, G., and N. Colowick. 1995. DNA replication of bacteriophage T4 *in vivo*. Methods Enzymol. 262:587–604.
779. Mosig, G., N. Colowick, M. E. Gruidl, A. Chang, and A. J. Harvey. 1995. Multiple initiation mechanisms adapt phage T4 DNA replication to physiological changes during T4’s development. FEMS Microbiol. Rev. 17:83–98.
780. Mosig, G., N. E. Colowick, and B. C. Pietz. 1998. Several new bacteriophage T4 genes, mapped by sequencing deletion endpoints between genes 56 (dCTPase) and *dda* (a DNA-dependent ATPase-helicase) modulate transcription. Gene 223:143–155.
781. Mosig, G., N. E. Colowick, and B. C. Pietz. 1998. Several new bacteriophage T4 genes, mapped by sequencing deletion endpoints between genes 56 (dCTPase) and *dda* (a DNA-dependent ATPase-helicase) modulate transcription. Gene 223:143–55.
782. Mosig, G., R. Ehring, W. Schliewen, and S. Bock. 1971. The patterns of recombination and segregation in terminal regions of T4 DNA molecules. Mol. Gen. Genet. 113:51–91.
783. Reference deleted.
784. Mosig, G., J. Gewin, A. Luder, N. Colowick, and D. Vo. 2001. Two recombination-dependent DNA replication pathways of bacteriophage T4, and their roles in mutagenesis and horizontal gene transfer. Proc. Natl. Acad. Sci. USA 98:8306–8311.
785. Mosig, G., D. Ghosal, and S. Bock. 1981. Interactions between the maturation protein gp17 and the single-stranded DNA binding protein gp32 initiate DNA packaging and compete with initiation of secondary replication forks in phage T4, p. 139–150. In M. DuBow (ed.), Bacteriophage assembly. Alan R. Liss, Inc., New York, N.Y.
786. Mosig, G., and D. H. Hall. 1994. Gene expression: a paradigm of integrated circuits, p. 127–131. In J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.
787. Mosig, G., G. W. Lin, J. Franklin, and W. H. Fan. 1989. Functional relationships and structural determinants of two bacteriophage T4 lysozymes: a soluble (gene *e*) and a baseplate-associated (gene 5) protein. New Biol. 1:171–179.
788. Mosig, G., A. Luder, A. Ernst, and N. Canan. 1991. Bypass of a primase requirement for bacteriophage T4 DNA replication *in vivo* by a recombination enzyme, endonuclease VII. New Biol. 3:1195–1205.

789. Mosig, G., P. Macdonald, G. Lin, M. Levin, and R. Seaby. 1983. Gene expression and initiation of DNA replication of bacteriophage T4 in phage and host topoisomerase mutants, p. 173–186. In N. R. Cozzarelli (ed.), Mechanisms of DNA replication and recombination. Alan R. Liss, Inc., New York, N.Y.
790. Mosig, G., and P. M. Macdonald. 1986. A new membrane-associated DNA replication protein, the gene 69 product of bacteriophage T4, shares a patch of homology with the *Escherichia coli* DnaA protein. J. Mol. Biol. 189:243–248.
791. Mosig, G., P. M. Macdonald, D. Powell, M. Trupin, and T. Gary. 1987. A membrane protein involved in initiation of DNA replication from the *oriA* region of phage T4, p. 403–414. In T. Kelly and R. McMacken (ed.), DNA replication and recombination.
792. Reference deleted.
793. Mosig, G., M. Shaw, and G. M. Garcia. 1984. On the role of DNA replication, endonuclease VII, and rII proteins in processing of recombination intermediates in phage T4. Cold Spring Harbor Symp. Quant. Biol. 49:371–382.
794. Mosig, G., S. Yu, H. Myung, E. Haggård-Ljungquist, L. Davenport, K. Carlson, and R. Calendar. 1997. A novel mechanism of virus-virus interactions: bacteriophage P2 Tin protein inhibits phage T4 DNA synthesis by poisoning the T4 single-stranded DNA binding protein, gp32. Virology 230:72–81.
795. Mudd, E. A., H. M. Krisch, and C. F. Higgins. 1990. RNase-E, an endoribonuclease, has a general role in the chemical decay of *Escherichia coli* messenger RNA—evidence that *me* and *ams* are the same genetic locus. Mol. Microbiol. 4:2127–2135.
796. Mueller, J. E., J. Clyman, Y.-J. Huang, M. M. Parker, and M. Belfort. 1996. Intron mobility in phage T4 occurs in the context of recombination-dependent DNA replication by way of multiple pathways. Genes Dev. 10:351–364.
797. Mueller, J. E., D. Smith, and M. Belfort. 1996. Exon coconversion biases accompanying intron homing: battle of the nucleases. Genes Dev. 10:2158–2166.
798. Mueller, J. E., D. Smith, M. Bryk, and M. Belfort. 1995. Intron-encoded endonuclease I-TevI binds as a monomer to effect sequential cleavage via conformational changes in the *td* homing site. EMBO J. 14:5724–5735.
799. Mueser, T. C., C. E. Jones, N. G. Nossal, and C. C. Hyde. 2000. Bacteriophage T4 gene 59 helicase assembly protein binds replication fork DNA. The 1.45 Å resolution crystal structure reveals a novel α-helical two-domain fold. J. Mol. Biol. 296:597–612.
800. Mueser, T. C., N. G. Nossal, and C. C. Hyde. 1996. Structure of Bacteriophage T4 RNase H, a 5′ to 3′ RNA-DNA and DNA-DNA exonuclease with sequence similarity to the RAD2 family of eukaryotic proteins. Cell 85:1101–1112.
801. Mufti, S., and H. Bernstein. 1974. The DNA-delay mutants of bacteriophage T4. J. Virol. 14:860–871.
802. Mullaney, J. M., and L. W. Black. 1998. Activity of foreign proteins targeted within the bacteriophage T4 head and prohead: implications for packaged DNA structure. J. Mol. Biol. 283:913–929.
803. Mullaney, J. M., and L. W. Black. 1996. Capsid targeting sequence targets foreign proteins into bacteriophage T4 and permits proteolytic processing. J. Mol. Biol. 261:372–385.
804. Mullaney, J. M., R. B. Thompson, Z. Gryczynski, and L. W. Black. 2000. Green fluorescent protein as a probe of rotational mobility within bacteriophage T4. J. Virol. Methods 88:35–40.
805. Muller-Salamin, L., L. Onorato, and M. K. Showe. 1977. Localization of minor protein components of the head of bacteriophage T4. J. Virol. 24:121–134.
806. Nakabeppu, Y., and M. Sekiguchi. 1981. Physical association of pyrimidine dimer DNA glycosylase and apurinic/apyrimidinic DNA endonuclease essential for repair of ultraviolet-damaged DNA. Proc. Natl. Acad. Sci. USA 78:2742–2746.
807. Nakagawa, H., F. Arisaka, and S. Ishii. 1985. Isolation and characterization of the bacteriophage T4 tail-associated lysozyme. J. Virol. 54:460–466.
808. Napoli, C., L. Gold, and B. S. Singer. 1981. Translational reinitiation in the rIIB cistron of bacteriophage T4. J. Mol. Biol. 149:433–449.
809. Nelson, H. C., J. T. Finch, B. F. Luisi, and A. Klug. 1987. The structure of an oligo(dA).oligo(dT) tract and its biological implications. Nature 330:221–226.
810. Nelson, M. A., M. Ericson, L. Gold, and J. F. Pulitzer. 1982. The isolation and characterization of TabR bacteria: hosts that restrict bacteriophage T4 *rII* mutants. Mol. Gen. Genet. 188:60–68.
811. Nelson, M. A., and L. Gold. 1982. The isolation and characterization of bacterial strains (ab 32) that restrict bacteriophage T4 gene 32 mutants. Mol. Gen. Genet. 188:69–76.
812. Nickell, C., W. F. Anderson, and R. S. Lloyd. 1991. Substitution of basic amino acids within endonuclease-V enhances nontarget DNA binding. J. Biol. Chem. 266: 5634–5642.
813. Nickell, C., and R. S. Lloyd. 1991. Mutations in endonuclease-V that affect both protein-protein association and target site location. Biochemistry 30:8638–8648.

814. Niggemann, E., I. Green, H. P. Meyer, and W. Ruger. 1981. Physical mapping of bacteriophage T4. Mol. Gen. Genet. 184:289–299.
815. Nikkola, M., A. Engstrom, M. Saarinen, M. Ingelman, T. Joelson, and H. Eklund. 1993. An elongated form of T4 glutaredoxin with four extra residues. Biochemistry 32:7133–7135.
816. Nikkola, M., F. K. Gleason, and H. Eklund. 1993. Reduction of mutant phage T4 glutaredoxins by *Escherichia coli* thioredoxin reductase. J. Biol. Chem. 268:3845–3849.
817. Nishimoto, H., M. Takayama, and T. Minagawa. 1979. Purification and some properties of deoxyribonuclease whose synthesis is controlled by gene 49 of bacteriophage T4. Eur. J. Biochem. 100:433–440.
818. Nivinskas, R., and L. W. Black. 1988. Cloning, sequence, and expression of the temperature-dependent phage T4 capsid assembly gene 31. Gene 73:251–257.
819. Nivinskas, R., N. Malys, V. Klausa, R. Vaiskunaite, and E. Gineikiene. 1999. Post-transcriptional control of bacteriophage T4 gene 25 expression: mRNA secondary structure that enhances translational initiation. J. Mol. Biol. 288:291–304.
820. Nivinskas, R., A. Raudonikiene, and R. Vaiskunaite. 1990. Bacteriophage T4 gene 26. Nucleic Acids Res. 18:1913.
821. Nivinskas, R., R. Vaiskunaite, R. Dagyte, A. Raudonikiene, and V. Klausa. 1992. Cloning, sequence, and overexpression of bacteriophage T4 gene 51. Virology 188:887–889.
822. Nivinskas, R., R. Vaiskunaite, and A. Raudonikiene. 1993. Expression of bacteriophage T4 gene 25 is regulated via RNA secondary structure in the translational initiation region. J. Mol. Biol. 230:717–721.
823. Nivinskas, R., R. Vaiskunaite, and A. Raudonikiene. 1992. An internal AUU codon initiates a small peptide encoded by bacteriophage T4 baseplate gene 26. Mol. Gen. Genet. 232:257–261.
824. Nivinskas, R. G., and L. W. Black. 1988. Nucleotide sequence of gene 31 of the T4 bacteriophage. Mol. Biol. (Moscow). 22:1507–1516.
825. Niyogi, K. K., O. Björkman, and A. R. Grossman. 1997. *Chlamydomonas* xanthophyll cycle mutants identified by video imaging of chlorophyll fluorescence quenching. Plant Cell 9:1369–1380.
826. Nossal, N. G. 1994. The bacteriophage T4 replication fork, p. 43–53. *In* J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.
827. Nossal, N. G. 1998. A new look at old mutants of T4 DNA polymerase. Genetics 148:1535–1538.
828. Nossal, N. G. 1992. Protein-protein interactions at a DNA replication fork: bacteriophage T4 as a model. FASEB J. 6:871–878.
829. Nossal, N. G. 1980. RNA priming of DNA replication by bacteriophage T4 proteins. J. Biol. Chem. 255:2176–2182.
830. Nossal, N. G., K. C. Dudas, and K. N. Kreuzer. 2001. Bacteriophage T4 proteins replicate plasmids with a preformed R loop at the T4 *ori(uvsY)* replication origin in vitro. Mol. Cell 7:31–41.
831. Nossal, N. G., D. M. Hinton, L. J. Hobbs, and P. Spacciapoli. 1995. Purification of bacteriophage T4 DNA replication proteins. Methods Enzymol. 262:560–584.
832. Nyaga, S. G., M. L. Dodson, and R. S. Lloyd. 1997. Role of specific amino acid residues in T4 endonuclease V that alter nontarget DNA binding. Biochemistry 36:4080–4088.
833. O'Donnell, S. M., and G. R. Janssen. 2001. The initiation codon affects ribosome binding and translational efficiency in *Escherichia coli* of cI mRNA with or without the 5′ untranslated leader. J. Bacteriol. 183:1277–1283.
833a. O'Farrell, P. H., E. Kutter E, and M. Nakanishi. 1980. A restriction map of the bacteriophage T4 genome. Mol. Gen. Genet. 179:421–435.
834. O'Farrell, P. Z., L. M. Gold, and W. M. Huang. 1973. The identification of prereplicative bacteriophage T4 proteins. J. Biol. Chem. 248:5499–5501.
835. O'Malley, S. M., A. K. Sattar, K. R. Williams, and E. K. Spicer. 1995. Mutagenesis of the COOH-terminal region of bacteriophage T4 RegA protein. J. Biol. Chem. 270:5107–5114.
836. Obringer, J. W. 1988. The functions of the phage T4 immunity and spackle genes in genetic exclusion. Genet. Res. 52:81–90.
837. Reference deleted
838. Oishi, M. 1968. Studies of DNA replication in vivo. 3. Accumulation of a single-stranded isolation product of DNA replication by conditional mutant strains of T4. Proc. Natl. Acad. Sci. USA 60:1000–1006.
839. Okker, R. J. H., E. Pees, and V. Bom. 1981. Partial exclusion of bacteriophage T2 by bacteriophage T4: an exclusion resistant mutation in gene 56 of T2. J. Gen. Virol. 53:13–19.
840. Oliver, D. B., and R. A. Crowther. 1981. DNA sequence of the tail fibre genes 36 and 37 of bacteriophage T4. J. Mol. Biol. 153:545–568.
841. Olson, N. H., M. Gingery, F. A. Eiserling, and T. S. Baker. 2001. The structure of isometric capsids of bacteriophage T4. Virology 279:385–391.
842. Olson, N. J., and G. L. Marchin. 1985. Response of a phage modification factor to enhanced production of its target molecule. J. Virol. 53:702–704.
843. Olson, N. J., and G. L. Marchin. 1984. Valyl-tRNA synthetase modification-dependent restriction of bacteriophage T4. J. Virol. 51:42–46.

844. Onorato, L., and M. K. Showe. 1975. Gene 21 protein-dependent proteolysis *in vitro* of purified gene 22 product of bacteriophage T4. J. Mol. Biol. 92:395–412.
845. Onorato, L., B. Stirmer, and M. K. Showe. 1978. Isolation and characterization of bacteriophage T4 mutant preheads. J. Virol. 27:409–426.
846. Oppenheim, D. S., and C. Yanofsky. 1980. Translational coupling during expression of the tryptophan operon of *Escherichia coli*. Genetics 95:785–795.
847. Orsini, G., M. Ouhammouch, J. P. Le Caer, and E. N. Brody. 1993. The *asiA* gene of bacteriophage T4 codes for the anti-sigma 70 protein. J. Bacteriol. 175:85–93.
848. Ouhammouch, M., K. Adelman, S. R. Harvey, G. Orsini, and E. N. Brody. 1995. Bacteriophage T4 MotA and AsiA proteins suffice to direct *Escherichia coli* RNA polymerase to initiate transcription at T4 middle promoters. Proc. Natl. Acad. Sci. USA 92:1451–1455.
849. Ouhammouch, M., G. Orsini, and E. N. Brody. 1994. The *asiA* gene product of bacteriophage T4 is required for middle mode RNA synthesis. J. Bacteriol. 176:3956–3965.
850. Owen, J. E., D. W. Schultz, A. Taylor, and G. R. Smith. 1983. Nucleotide sequence of the lysozyme gene of bacteriophage T4. Analysis of mutations involving repeated sequences. J. Mol. Biol. 165:229–248.
851. Paddison, P., S. T. Abedon, H. K. Dressman, K. Gailbreath, J. Tracy, E. Mosser, J. Neitzel, B. Guttman, and E. Kutter. 1998. The roles of the bacteriophage T4 r genes in lysis inhibition and fine-structure genetics: a new perspective. Genetics 148:1539–1550.
852. Pahari, S., and D. Chatterji. 1997. Interaction of bacteriophage T4 AsiA protein with *Escherichia coli σ⁷⁰* and its variant. FEBS Lett. 411:60–62.
853. Panuska, J. R., and D. A. Goldthwait. 1980. A DNA-dependent ATPase from T4-infected *Escherichia coli*. Purification and properties of a 63,000-dalton enzyme and its conversion to a 22,000-dalton form. J. Biol. Chem. 255:5208–5214.
854. Parker, M. L., A. C. Christensen, A. Boosman, J. Stockard, E. T. Young, and A. H. Doermann. 1984. Nucleotide sequence of bacteriophage T4 gene 23 and the amino acid sequence of its product. J. Mol. Biol. 180:399–416.
855. Parma, D. H., M. Snyder, S. Sobolevski, M. Nawroz, E. Brody, and L. Gold. 1992. The Rex system of bacteriophage lambda: tolerance and altruistic cell death. Genes Dev. 6:497–510.
856. Pavlov, A. R., and J. D. Karam. 1994. Binding specificity of T4 DNA polymerase to RNA. J. Biol. Chem. 269:12968–12972.
857. Pavlov, A. R., and J. D. Karam. 2000. Nucleotide-sequence-specific and non-specific interactions of T4 DNA polymerase with its own mRNA. Nucleic Acids Res. 28:4657–4664.
858. Pene, C., and M. Uzan. 2000. The bacteriophage T4 anti-sigma factor AsiA is not necessary for the inhibition of early promoters in vivo. Mol. Microbiol. 35:1180–1191.
859. Penner, M., I. Morad, L. Snyder, and G. Kaufmann. 1995. Phage T4-coded Stp: double-edged effector of coupled DNA and tRNA-restriction systems. J. Mol. Biol. 249:857–868.
860. Phillips, C. A., J. Gordon, and E. K. Spicer. 1996. Bacteriophage T4 regA protein binds RNA as a monomer, overcoming dimer interactions. Nucleic Acids Res. 24:4319–4326.
861. Piechowski, M. M., and M. Susman. 1967. Acridine resistance in phage T4D. Genetics 56:133–148.
862. Piersen, C. E., A. K. McCullough, and R. S. Lloyd. 2000. AP lyases and dRPases: commonality of mechanism. Mutat. Res. 459:43–53.
863. Pietrokovski, S. 2001. Intein spread and extinction in evolution. Trends Genet. 17:465–472.
864. Pietroni, P., M. C. Young, G. J. Latham, and P. H. von Hippel. 2001. Dissection of the ATP-driven reaction cycle of the bacteriophage T4 DNA replication processivity clamp loading system. J. Mol. Biol. 309:869–891.
865. Pietroni, P., M. C. Young, G. J. Latham, and P. H. von Hippel. 1997. Structural analyses of gp45 sliding clamp interactions during assembly of the bacteriophage T4 DNA polymerase holoenzyme. I. Conformational changes within the gp44/62-gp45-ATP complex during clamp loading. J. Biol. Chem. 272:31666–31676.
866. Pirisi, A. 2000. Phage therapy—advantages over antibiotics? Lancet 356:1418.
867. Plishker, M. F., and P. B. Berget. 1984. Isolation and characterization of precursors in bacteriophage T4 baseplate assembly. III. The carboxyl termini of protein P11 are required for assembly activity. J. Mol. Biol. 178:699–709.
868. Plishker, M. F., M. Chidambaram, and P. B. Berget. 1983. Isolation and characterization of precursors in bacteriophage T4 baseplate assembly. II. Purification of the protein products of genes 10 and 11 and the in vitro formation of the P(10/11) complex. J. Mol. Biol. 170:119–135.
869. Plunkett, G. 1988. Ph.D. dissertation. University of Wisconsin, Madison.
870. Plunkett, G., G. P. Mazzara, and W. H. McClain. 1981. Characterization of bacteriophage T4 and D RNA, a low-molecular-weight RNA of unknown function. Arch. Biochem. Biophys. 210:298–306.
871. Poteete, A. R., and L. W. Hardy. 1994. Genetic analysis of bacteriophage T4 lysozyme structure and function. J. Bacteriol. 176:6783–6788.

872. Poteete, A. R., D. P. Sun, H. Nicholson, and B. W. Matthews. 1991. Second-site revertants of an inactive T4 lysozyme mutant restore activity by restructuring the active site cleft. Biochemistry 30:1425–1432.

873. Powell, D., J. Franklin, F. Arisaka, and G. Mosig. 1990. Bacteriophage T4 DNA packaging genes 16 and 17. Nucleic Acids Res. 18:4005.

874. Pribnow, D., D. C. Sigurdson, L. Gold, B. S. Singer, C. Napoli, J. Brosius, T. J. Dull, and H. F. Noller. 1981. *rII* cistrons of bacteriophage T4. DNA sequence around the intercistronic divide and positions of genetic landmarks. J. Mol. Biol. 149:337–376.

875. Prilipov, A. G., V. V. Mesyanzhinov, U. Aebi, and E. Kellenberger. 1990. Cloning and sequencing of bacteriophage T4 genes between map positions 128.3–130.3. Nucleic Acids Res. 18:3635.

876. Prilipov, A. G., N. A. Selivanov, V. P. Efimov, E. I. Marusich, and V. V. Mesyanzhinov. 1989. Nucleotide sequences of bacteriophage T4 genes 9, 10 and 11. Nucleic Acids Res. 17:3303.

877. Prilipov, A. G., N. A. Selivanov, L. I. Nikolaeva, and V. V. Mesyanzhinov. 1988. Nucleotide and deduced amino acid sequence of bacteriophage T4 gene *wac*. Nucleic Acids Res. 16:10361.

878. Pulitzer, J. F., M. Colombo, and M. Ciaramella. 1985. New control elements of bacteriophage T4 pre-replicative transcription. J. Mol. Biol. 182:249–263.

879. Pulitzer, J. F., A. Coppo, and M. Caruso. 1979. Host-virus interactions in the control of T4 prereplicative transcription. II. Interaction between *tabC* (*ρ*) mutants and T4 *mot* mutants. J. Mol. Biol. 135:979–997.

880. Purohit, S., R. K. Bestwick, G. W. Lasser, C. M. Rogers, and C. K. Mathews. 1981. T4 phage-coded dihydrofolate reductase. Subunit composition and cloning of its structural gene. J. Biol. Chem. 256:9121–9125.

881. Purohit, S., and C. K. Mathews. 1984. Nucleotide sequence reveals overlap between T4 phage genes encoding dihydrofolate reductase and thymidylate synthase. J. Biol. Chem. 259:6261–6266.

882. Quirk, S. M., D. Bell-Pedersen, J. Tomaschewski, W. Ruger, and M. Belfort. 1989. The inconsistent distribution of introns in the T-even phages indicates recent genetic exchanges. Nucleic Acids Res. 17:301–315.

883. Raijmakers, H., O. Vix, I. Törö, S. Golz, B. Kemper, and D. Suck. 1999. X-ray structure of T4 endonuclease VII: a DNA junction resolvase with a novel fold and unusual domain-swapped dimer architecture. EMBO J. 18:1447–1458.

884. Radany, E. H., and E. C. Friedberg. 1980. A pyrimidine dimer-DNA glycosylase activity associated with the v gene product of bacteriophage T4. Nature 286:182–185.

885. Radany, E. H., L. Naumovski, J. D. Love, K. A. Gutekunst, D. H. Hall, and E. C. Friedberg. 1984. Physical mapping and complete nucleotide sequence of the *denV* gene of bacteriophage T4. J. Virol. 52:846–856.

886. Ramanculov, E., and R. Young. 2001. An ancient player unmasked: T4 rI encodes a t-specific antiholin. Mol. Microbiol. 41:575–783.

887. Ramanculov, E., and R. Young. 2001. Functional analysis of the phage T4 holin in a λ context. Mol. Genet. Genomics 265:345–353.

888. Ramanculov, E., and R. Young. 2001. Genetic analysis of the T4 holin: timing and topology. Gene 265:25–36.

889. Rand, K. N., and M. J. Gait. 1984. Sequence and cloning of bacteriophage T4 gene 63 encoding RNA ligase and tail fibre attachment activities. EMBO J. 3:397–402.

890. Raney, K. D., T. E. Carver, and S. J. Benkovic. 1996. Stoichiometry and DNA unwinding by the bacteriophage T4 41:59 helicase. J. Biol. Chem. 271:14074–14081.

891. Rao, V. B., and L. W. Black. 1988. Cloning, overexpression and purification of the terminase proteins gp16 and gp17 of bacteriophage T4. J. Mol. Biol. 200:475–488.

892. Rao, V. B., and M. S. Mitchell. 2001. The N-terminal ATPase site in the large terminase protein gp17 is critically required for DNA packaging in bacteriophage T4. J. Mol. Biol. 314:401–411.

893. Rao, V. B., V. Thaker, and L. W. Black. 1992. A phage T4 in vitro packaging system for cloning long DNA molecules. Gene 113:25–33.

894. Rappaport, H., M. Russel, and M. Susman. 1974. Some acridine-resistant mutations of bacteriophage T4D. Genetics 78:579–592.

895. Ratner, D. 1974. The interaction of bacterial and phage proteins with immobilized *Escherichia coli* RNA polymerase. J. Mol. Biol. 88:373–383.

896. Raudonikiene, A., and R. Nivinskas. 1992. Gene *rIII* is the nearest downstream neighbour of bacteriophage T4 gene 31. Gene 114:85–90.

897. Raudonikiene, A., and R. Nivinskas. 1990. Nucleotide sequence of bacteriophage T4 gene 31 region. Nucleic Acids Res. 18:4280.

898. Raudonikiene, A., and R. Nivinskas. 1993. The sequences of gene *rIII* of bacteriophage T4 and its mutants. Gene 134:135–136.

899. Recinos, A. D., M. L. Augustine, K. M. Higgins, and R. S. Lloyd. 1986. Expression of the bacteriophage T4 *denV* structural gene in *Escherichia coli*. J. Bacteriol. 168:1014–1018.

900. Reddy, G. P., and C. K. Mathews. 1978. Functional compartmentation of DNA precursors in T4 phage-infected bacteria. J. Biol. Chem. 253:3461–3467.

901. Reddy, M. K., S. E. Weitzel, S. S. Daube, T. C. Jarvis, and P. H. von Hippel. 1995. Using macromolecular crowding agents to identify weak interactions within DNA replication complexes. Methods Enzymol. 262: 466–476.

902. Reddy, M. K., S. E. Weitzel, and P. H. vonHippel. 1993. Assembly of a functional replication complex without ATP hydrolysis: a direct interaction of bacteriophage T4 gp45 with T4 DNA polymerase. Proc. Natl. Acad. Sci. USA 90:3211–3215.

903. Reha-Krantz, L. J. 1988. Amino acid changes coded by bacteriophage T4 DNA polymerase mutator mutants. J. Mol. Biol. 202:711–724.

904. Reha-Krantz, L. J. 1994. Genetic dissection of T4 DNA polymerase structure-function relationships, p. 307–312. In J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

905. Reha-Krantz, L. J. 1990. Genetic evidence for two protein domains and a potential new activity in bacteriophage T4 DNA polymerase. Genetics 124:213–220.

906. Reha-Krantz, L. J. 1998. Regulation of DNA polymerase exonucleolytic proofreading activity: studies of bacteriophage T4 “antimutator” DNA polymerases. Genetics 148:1551–1557.

907. Reha-Krantz, L. J. 1995. Use of genetic analyses to probe structure, function, and dynamics of bacteriophage T4 DNA polymerase. Methods Enzymol. 262:323–331.

908. Reha-Krantz, L. J., and J. K. J. Lambert. 1985. Structure-function studies of the bacteriophage T4 DNA polymerase. J. Mol. Biol. 186:505–514.

909. Reha-Krantz, L. J., L. A. Marquez, E. Elisseeva, R. P. Baker, L. B. Bloom, H. B. Dunford, and M. F. Goodman. 1998. The proofreading pathway of bacteriophage T4 DNA polymerase. J. Biol. Chem. 273:22969–22976.

910. Reha-Krantz, L. J., and R. L. Nonay. 1993. Genetic and biochemical studies of bacteriophage T4 DNA polymerase 3′ → 5′-exonuclease activity. J. Biol. Chem. 268:27100–27108.

911. Reha-Krantz, L. J., and R. L. Nonay. 1994. Motif A of bacteriophage T4 DNA polymerase: role in primer extension and DNA replication fidelity. Isolation of new antimutator and mutator DNA polymerases. J. Biol. Chem. 269:5635–5643.

912. Reha-Krantz, L. J., R. L. Nonay, R. S. Day, and S. H. Wilson. 1996. Replication of O6-methylguanine-containing DNA by repair and replicative DNA polymerases. J. Biol. Chem. 271:20088–20095.

913. Reha-Krantz, L. J., R. L. Nonay, and S. Stocki. 1993. Bacteriophage T4 DNA polymerase mutations that confer sensitivity to the PPi analog phosphonoacetic acid. J. Virol. 67:60–66.

914. Reha-Krantz, L. J., S. Stocki, R. L. Nonay, E. Dimayuga, L. D. Goodrich, W. H. Konigsberg, and E. K. Spicer. 1991. DNA polymerization in the absence of exonucleolytic proofreading—in vivo and in vitro studies. Proc. Natl. Acad. Sci. USA 88:2417–2421.

915. Reinhold-Hurek, B., and D. A. Shub. 1993. Experimental approaches for detecting self-splicing group I introns. Methods Enzymol. 224:491–502.

916. Ren, Z., and L. W. Black. 1998. Phage T4 SOC and HOC display of biologically active, full-length proteins on the viral capsid. Gene 215:439–444.

917. Ren, Z. J., R. G. Baumann, and L. W. Black. 1997. Cloning of linear DNAs in vivo by overexpressed T4 DNA ligase: construction of a T4 phage hoc gene display vector. Gene 195:303–311.

918. Ren, Z. J., G. K. Lewis, P. T. Wingfield, E. G. Locke, A. C. Steven, and L. W. Black. 1996. Phage display of intact domains at high copy number: a system based on SOC, the small outer capsid protein of bacteriophage T4. Protein Sci. 5:1833–1843.

919. Repoila, F., F. Tétart, J.-Y. Bouet, and H. M. Krisch. 1994. Genomic polymorphism in the T-even bacteriophages. EMBO J. 13:4181–4192.

920. Revel, H. R. 1981. Organization of the bacteriophage T4 tail fiber gene cluster 34–38. Prog. Clin. Biol. Res. 64:353–364.

921. Revel, H. R., and S. M. Hattman. 1971. Mutants of T2 *gt* with altered DNA methylase activity: relation to restriction by prophage P1. Virology 45:484–495.

922. Revel, H. R., R. Herrmann, and R. J. Bishop. 1976. Genetic analysis of T4 tail fiber assembly. II. Bacterial host mutants that allow bypass of T4 gene 57 function. Virology 72:255–265.

923. Revel, H. R., and I. Lielausis. 1978. Revised location of the *rIII* gene on the genetic map of bacteriophage T4. J. Virol. 25:439–441.

924. Revel, H. R., and S. E. Luria. 1970. DNA-glucosylation in T-even phage: genetic determination and role in phage-host interaction. Annu. Rev. Genet. 4:177–192.

925. Revel, H. R., B. L. Stitt, I. Lielausis, and W. B. Wood. 1980. Role of the host cell in bacteriophage T4 development. J. Virol. 33:366–376.

926. Richardson, A., and C. Georgopoulos. 1999. Genetic analysis of the bacteriophage T4-encoded chaperonin Gp31. Genetics 152:1449–1457.

927. Richardson, A., S. J. Landry, and C. Georgopoulos. 1998. The ins and outs of a molecular chaperone machine. Trends Biochem. Sci. 23:138–143.

928. Richardson, A., S. M. van der Vies, F. Keppel, A. Taher, S. J. Landry, and C. Georgopoulos. 1999. Compensatory changes in GroEL/Gp31 affinity as a mechanism for allele-specific genetic interaction. J. Biol. Chem. 274:52–58.

929. Richardson, J. P., and J. Greenblatt. 1996. Control of RNA chain elon-

gation and termination, p. 822–848. *In* C. F. Neidhardt, R. Curtiss, J. L. Ingraham, E. C. C. Lin, K. B. Low, B. Magasanik, W. S. Reznikoff, M. Riley, M. Schaechter, and H. E. Umbarger (ed.), *Escherichia coli* and *Salmonella*: cellular and molecular biology, vol. 1. American Society for Microbiology, Washington, D.C.

930. Richardson, R. W., and N. G. Nossal. 1989. Characterization of the bacteriophage T4 gene 41 DNA helicase. J. Biol. Chem. 264:4725–4731.

931. Richardson, R. W., and N. G. Nossal. 1989. Trypsin cleavage in the COOH terminus of the bacteriophage T4 gene 41 DNA helicase alters the primase-helicase activities of the T4 replication complex in vitro. J. Biol. Chem. 264:4732–4739.

932. Riede, I. 1987. Lysis gene *t* of T-even bacteriophages: evidence that colicins and bacteriophage genes have common ancestors. J. Bacteriol. 169: 2956–2961.

933. Riede, I., M. Degen, and U. Henning. 1985. The receptor specificity of bacteriophages can be determined by a tail fiber modifying protein. EMBO J. 4:2343–2346.

934. Riede, I., K. Drexler, M. L. Eschbach, and U. Henning. 1986. DNA sequence of the tail fiber genes 37, encoding the receptor recognizing part of the fiber, of bacteriophages T2 and K3. J. Mol. Biol. 191:255–266.

935. Ripley, L. S. 1999. Predictability of mutant sequences. Relationships between mutational mechanisms and mutant specificity. Ann. N. Y. Acad. Sci. 870:159–172.

936. Robinson, D. R., N. R. Watts, and D. H. Coombs. 1988. Heat cleavage of bacteriophage T4 gene 23 product produces two peptides previously identified as head proteins. J. Virol. 62:1723–1729.

937. Rodriguez-Prieto, A. 1976. Ph.D. dissertation. Vanderbilt University, Nashville, Tenn.

937a.Rohrer, H., W. Zillig, and R. Mailhammer. 1975. ADP-ribosylation of DNA-dependent RNA polymerase of *Escherichia coli* by an NAD<sup>+</sup>: protein ADP-ribosyltransferase from bacteriophage T4. Eur J Biochem. 60: 227–238.

938. Rosario, M. O., and J. W. Drake. 1990. Frameshift and double-amber mutations in the bacteriophage T4 *uvsX* gene: analysis of mutant UvsX proteins from infected cells. Mol. Genet. 222:112–119.

939. Ross, W., S. E. Aiyar, J. Solomon, and R. L. Gourse. 1998. *Escherichia coli* promoters with UP elements of different strengths: modular structure of bacterial promoters. J. Bacteriol. 180:5375–5383.

940. Rothman, F. G., A. R. Robbins, and D. Lackey. 1975. Novel rII duplications in bacteriophage T4. J. Virol. 15:1024–1008.

941. Rowe, T. C., K. M. Tewey, and L. F. Liu. 1984. Identification of the breakage-reunion subunit of T4 DNA topoisomerase. J. Biol. Chem. 259: 9177–9181.

942. Ruckman, J., D. Parma, C. Tuerk, D. H. Hall, and L. Gold. 1989. Identification of a T4 gene required for bacteriophage mRNA processing. New Biol. 1:54–65.

943. Ruckman, J., S. Ringquist, E. Brody, and L. Gold. 1994. The bacteriophage T4 *regB* ribonuclease. J. Biol. Chem. 269:26655–26662.

944. Runnels, J. M., D. Soltis, T. Hey, and L. Snyder. 1982. Genetic and physiological studies of the role of the RNA ligase of bacteriophage T4. J. Mol. Biol. 154:273–286.

945. Russel, M., L. Gold, H. Morissett, and P. Z. O'Farrell. 1976. Translational, autogenous regulation of gene 32 expression during bacteriophage T4 infection. J. Biol. Chem. 251:7263–7270.

946. Russell, R. L. 1974. Comparative genetics of the T-even bacteriophages. Genetics 78:967–988.

947. Rutberg, B., and L. Rutberg. 1965. Role of superinfecting phage in lysis inhibition with phage T4 in *Escherichia coli*. J. Bacteriol. 90:891–894.

948. Sagermann, M., W. A. Baase, and B. W. Matthews. 1999. Structural characterization of an engineered tandem repeat contrasts the importance of context and sequence in protein folding. Proc. Natl. Acad. Sci. USA 96:6078–6083.

949. Salinas, F., H. Jiang, and T. Kodadek. 1995. Homology dependence of UvsX protein-catalyzed joint molecule formation. J. Biol. Chem. 270: 5181–5186.

950. Salinas, F., and T. Kodadek. 1995. Phage T4 homologous strand exchange: A DNA helicase, not the strand transferase, drives polar branch migration. Cell 82:111–119.

951. Sanders, G. M., G. A. Kassavetis, and E. P. Geiduschek. 1997. Dual targets of a transcriptional activator that tracks on DNA. EMBO J. 16: 3124–3132.

952. Sanders, G. M., G. A. Kassavetis, and E. P. Geiduschek. 1995. Rules governing the efficiency and polarity of loading a tracking clamp protein onto DNA: determinants of enhancement in bacteriophage T4 late transcription. EMBO J. 14:3966–3976.

953. Sanders, G. M., G. A. Kassavetis, and E. P. Geiduschek. 1994. Use of a macromolecular crowding agent to dissect interactions and define functions in transcriptional activation by a DNA-tracking protein: bacteriophage T4 gene 45 protein and late transcription. Proc. Natl. Acad. Sci. USA 91:7703–7707.

954. Sanson, B., R. M. Hu, E. Troitskayadagger, N. Mathy, and M. Uzan. 2000. Endoribonuclease RegB from bacteriophage T4 is necessary for the deg radiation of early but not middle or late mRNAs. J. Mol. Biol. 297:1063– 1074.

955. Sanson, B., and M. Uzan. 1995. Post-transcriptional controls in bacteriophage T4: roles of the sequence-specific endoribonuclease RegB. FEMS Microbiol. Rev. 17:141–150.

956. Sanson, B., and M. Uzan. 1992. Sequence and characterization of the bacteriophage T4 *comC* alpha gene product, a possible transcription antitermination factor. J. Bacteriol. 174:6539–6547.

957. Reference deleted.

958. Santoro, M., V. Scarlato, A. Franze, O. Grau, M. Cipollaro, S. Gargano, R. Bova, M. R. Micheli, A. Storlazzi, and A. Cascino. 1988. Symmetric transcription of bacteriophage T4 base plate genes. Gene. 72:241–245.

959. Reference deleted.

960. Schlagman, S. L., and S. Hattman. 1983. Molecular cloning of a functional *dam+* gene coding for phage T4 DNA adenine methylase. Gene 22:139– 156.

961. Schlagman, S. L., Z. Miner, Z. Feher, and S. Hattman. 1988. The DNA [adenine-N6]methyltransferase (Dam) of bacteriophage T4. Gene 73:517– 530.

962. Schmidt, F. J., and D. Apirion. 1983. T4 transfer RNAs: paradigmatic system for the study of RNA processing, p. 208–217. *In* C. K. Mathews, E. M. Kutter, G. Mosig, and P. B. Berget (ed.), Bacteriophage T4. American Society for Microbiology, Washington, D.C.

963. Schmidt, R. P., and K. N. Kreuzer. 1992. Purified MotA protein binds the –30 region of a bacteriophage T4 middle promoter and activates transcription in vitro. J. Biol. Chem. 267:11399–11407.

964. Schneider, D., C. Tuerk, and L. Gold. 1992. Selection of high affinity RNA ligands to the bacteriophage R17 coat protein. J. Mol. Biol. 228:862–869.

965. Schneider, T. D. 1996. Reading of DNA sequence logos: prediction of major groove binding by information theory. Methods Enzymol. 274:445– 455.

966. Schneider, T. D., and R. M. Stephens. 1990. Sequence logos: a new way to display consensus sequences. Nucleic Acids Res. 18:6097–6100.

967. Schuch, R., D. Nelson, and V. A. Fischetti. 2002. A bacteriolytic agent that detects and kills *Bacillus anthracis*. Nature 418:884–889.

968. Seasholtz, A. F., and G. R. Greenberg. 1983. Identification of bacteriophage T4 gene 60 product and a role for this protein in DNA topoisomerase. J. Biol. Chem. 258:1221–1226.

969. Seawell, P. C., C. A. Smith, and A. K. Ganesan. 1980. *denV* gene of bacteriophage T4 determines a DNA glycosylase specific for pyrimidine dimers in DNA. J. Virol. 35:790–796.

970. Selick, H. E., J. Barry, T.-A. Cha, M. Munn, M. Nakanishi, M. L. Wong, and B. M. Alberts. 1987. Studies on the T4 bacteriophage DNA replication system. DNA replication and recombination. UCLA Symp. Mol. Cell. Biol. 47:183–214.

971. Selick, H. E., G. D. Stormo, R. L. Dyson, and B. M. Alberts. 1993. Analysis of five presumptive protein-coding sequences clustered between the primosome genes, 41 and 61, of bacteriophages T4, T2, and T6. J. Virol. 67:2305–2316.

972. Selivanov, N. A., A. G. Prilipov, and V. V. Mesyanzhinov. 1988. Nucleotide and deduced amino acid sequence of bacteriophage T4 gene 12. Nucleic Acids Res. 16:2334.

973. Selivanov, N. A., A. G. Prilipov, and V. V. Mesyanzhinov. 1989. Nucleotide sequences of bacteriophage T4 genes 13, 14 and 15. Nucleic Acids Res. 17:3583.

974. Reference deleted.

975. Sengupta, T. K., J. Gordon, and E. K. Spicer. 2001. RegA proteins from phage T4 and RB69 have conserved helix-loop groove RNA binding motifs but different RNA binding specificities. Nucleic Acids Res. 29:1175– 1184.

976. Severinov, K., M. Kashlev, E. Severinova, I. Bass, K. McWilliams, E. Kutter, V. Nikiforov, L. Snyder, and A. Goldfarb. 1994. A non-essential domain of *Escherichia coli* RNA polymerase required for the action of the termination factor Alc. J. Biol. Chem. 269:14254–14259.

977. Severinova, E., K. Severinov, and S. A. Darst. 1998. Inhibition of *Escherichia coli* RNA polymerase by bacteriophage T4 AsiA. J. Mol. Biol. 279:9–18.

978. Sexton, D. J., T. E. Carver, A. J. Berdis, and S. J. Benkovic. 1996. Protein-protein and protein-DNA interactions at the bacteriophage T4 DNA replication fork: characterization of a fluorescently labeled DNA polymerase sliding clamp. J. Biol. Chem. 271:28045–28051.

979. Shah, D. B. 1976. Replication and recombination of gene 59 mutant of bacteriophage T4D. J. Virol. 17:175–182.

980. Shamoo, Y., H. Adari, W. H. Konigsberg, K. R. Williams, and J. W. Chase. 1986. Cloning of T4 gene 32 and expression of the wild-type protein under lambda promoter PL regulation in *Escherichia coli*. Proc. Natl. Acad. Sci. USA 83:8844–8848.

981. Reference deleted.

982. Shamoo, Y., A. M. Friedman, M. R. Parsons, W. H. Konigsberg, and T. A. Steitz. 1995. Crystal structure of a replication fork single-stranded DNA binding protein (T4 gp32) complexed to DNA. Nature 376:362–366. (Erratum, 376:616.)

983. Shamoo, Y., and T. A. Steitz. 1999. Building a replisome from interacting pieces: sliding clamp complexed to a peptide from DNA polymerase and a polymerase editing complex. Cell 99:155–166.

984. Shamoo, Y., A. Tam, W. H. Konigsberg, and K. R. Williams. 1993. Translational repression by the bacteriophage T4 gene 32 protein involves specific recognition of an RNA pseudoknot structure. J. Mol. Biol. 232: 89–104.

985. Shamoo, Y., K. R. Williams, and W. H. Konigsberg. 1994. The function of zinc(II) in gene 32 protein (gp32). *In* J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

986. Sharma, M., and D. M. Hinton. 1994. Purification and characterization of the SegA protein of bacteriophage T4, an endonuclease related to proteins encoded by group I introns. J. Bacteriol. 176:6439–6448.

987. Sharma, M., P. Marshall, and D. M. Hinton. 1999. Binding of the bacteriophage T4 transcriptional activator, MotA, to T4 middle promoter DNA: evidence for both major and minor groove contacts. J. Mol. Biol. 290:905–915.

988. Sharma, M. S., R. L. Ellis, and D. M. Hinton. 1992. Identification of a family of bacteriophage T4 genes encoding proteins similar to those present in group I introns of fungi and phage. Proc. Natl. Acad. Sci. USA 89:6658–6662.

989. Sharma, U. K., S. Ravishankar, R. K. Shandil, P. V. Praveen, and T. S. Balganesh. 1999. Study of the interaction between bacteriophage T4 *asiA* and *Escherichia coli σ⁷⁰*, using the yeast two-hybrid system: neutralization of *asiA* toxicity to *E. coli* cells by coexpression of a truncated *σ⁷⁰* fragment. J. Bacteriol. 181:5855–5859.

989a. Shcherbakov, V., I. Granovsky, L. Plugina, T. Shcherbakova, S. Sizova, K. Pyatkov, M. Shlyapnikov, and O. Shubina. 2002. Focused genetic recombination of bacteriophage T4 initiated by double-strand breaks. Genetics 162:543–546.

990. Showe, M. K., and L. W. Black. 1973. Assembly core of bacteriophage T4: an intermediate in head formation. Nature New Biol. 242:70–75.

991. Shub, D. A., T. Coetzee, D. W. Hall, and M. Belfort. 1994. The self-splicing introns of bacteriophage T4, p. 186–192. *In* J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

992. Shub, D. A., H. Goodrich-Blair, and S. R. Eddy. 1994. Amino acid sequence motif of group I intron endonucleases is conserved in open reading frames of group II introns. Trends Biochem. Sci. 19:402–404.

993. Shub, D. A., J. M. Gott, M. Q. Xu, B. F. Lang, F. Michel, J. Tomaschewski, J. Pedersen-Lane, and M. Belfort. 1988. Structural conservation among three homologous introns of bacteriophage T4 and the group I introns of eukaryotes. Proc. Natl. Acad. Sci. USA 85:1151–1155.

994. Shultzaberger, R. K., R. E. Bucheimer, K. E. Rudd, and T. D. Schneider. 2001. Anatomy of *Escherichia coli* ribosome binding sites. J. Mol. Biol. 313:215–228.

995. Sieber, P., A. Lindemann, M. Boehm, G. Seidel, U. Herzing, P. van der Heusen, R. Muller, W. Ruger, R. Jaenicke, and P. Rosch. 1998. Overexpression and structural characterization of the phage T4 protein DsbA. Biol. Chem. 379:51–58.

996. Silver, L. L., and N. G. Nossal. 1979. DNA replication by bacteriophage T4 proteins: role of the DNA-delay gene 61 in the chain-initiation reaction. Cold Spring Harbor Symp. Quant. Biol. 43:489–494.

997. Silver, L. L., and N. G. Nossal. 1982. Purification of bacteriophage T4 gene 61 protein. A protein essential for synthesis of RNA primers in the T4 in vitro DNA replication system. J. Biol. Chem. 257:11696–11705.

998. Silver, L. L., M. Venkatesan, and N. G. Nossal. 1980. RNA priming and DNA synthesis by bacteriophage T4 proteins. *In* B. Alberts (ed.), Mechanistic studies of DNA replication and genetic recombination. ICN-UCLA Symp. Mol. Biol. 19:475–484.

999. Silver, S. 1965. Acriflavine resistance: a bacteriophage mutation affecting the uptake of dye by the infected bacterial cells. Proc. Natl. Acad. Sci. USA 53:24–30.

1000. Silverstein, J. L., and E. B. Goldberg. 1976. T4 DNA injection. I. Growth cycle of a gene 2 mutant. Virology 72:195–211.

1001. Silverstein, J. L., and E. B. Goldberg. 1976. T4 DNA injection. II. Protection of entering DNA from host exonuclease V. Virology 72:212–223.

1002. Simon, E. H., and I. Tessman. 1963. Thymidine-requiring mutants of phage T4. Proc. Natl. Acad. Sci. USA 50:526–532.

1003. Simon, L. D. 1969. The infection of *Escherichia coli* by T2 and T4 bacteriophages as seen in the electron microscope. 3. Membrane-associated intracellular bacteriophages. Virology 38:285–296.

1004. Simon, L. D., and B. Randolph. 1984. Bacteriophage T4 bypass31 mutations that make gene 31 nonessential for bacteriophage T4 replication: isolation and characterization. J. Virol. 51:321–328.

1005. Simon, L. D., B. Randolph, N. Irwin, and G. Binkowski. 1983. Stabilization of proteins by a bacteriophage T4 gene cloned in *Escherichia coli*. Proc. Natl. Acad. Sci. USA 80:2059–2062.

1006. Simon, L. D., D. Snover, and A. H. Doermann. 1974. Bacterial mutation

affecting T4 phage DNA synthesis and tail production. Nature 252:451–455.

1007. Singer, B. S., S. T. Shinedling, and L. Gold. 1983. The *rII* genes: a history and a prospectus, p. 327–333. *In* C. K. Mathews, E. M. Kutter, G. Mosig, and P. B. Berget (ed.), Bacteriophage T4. American Society for Microbiology, Washington, D.C.

1008. Sirotkin, K., W. Cooley, J. Runnels, and L. R. Snyder. 1978. A role in true-late gene expression for the T4 bacteriophage 5′ polynucleotide kinase 3′ phosphatase. J. Mol. Biol. 123:221–233.

1009. Sirotkin, K., J. Wei, and L. Snyder. 1977. T4 Bacteriophage-coded RNA polymerase subunit blocks host transcription and unfolds the host chromosome. Nature 265:28–32.

1010. Sjoberg, B. M., S. Hahne, C. Z. Mathews, C. K. Mathews, K. N. Rand, and M. J. Gait. 1986. The bacteriophage T4 gene for the small subunit of ribonucleotide reductase contains an intron. EMBO J. 5:2031–2036.

1011. Skorko, R., W. Zillig, H. Rohrer, H. Fujiki, and R. Mailhammer. 1977. Purification and properties of the NAD⁺:protein ADP-ribosyltransferase responsible for the T4 phage-induced modification of the subunit of DNA-dependent RNA polymerase of *Escherichia coli*. Eur. J. Biochem. 79:55–66.

1012. Skorupski, K., J. Tomaschewski, W. Ruger, and L. D. Simon. 1988. A bacteriophage T4 gene which functions to inhibit *Escherichia coli* Lon protease. J. Bacteriol. 170:3016–3024.

1013. Smith, S. M., and N. Symonds. 1973. The unexpected location of a gene conferring abnormal radiation sensitivity on phage T4. Nature 241:395–396.

1014. Snopek, T. J., W. B. Wood, M. P. Conley, P. Chen, and N. R. Cozzarelli. 1977. Bacteriophage T4 RNA ligase is gene 63 product, the protein that promotes tail fiber attachment to the baseplate. Proc. Natl. Acad. Sci. USA 74:3355–3359.

1015. Snustad, D. P. 1968. Dominance interactions in *Escherichia coli* cells mixedly infected with bacteriophage T4D wild-type and amber mutants and their possible implications as to type of gene-product function: catalytic vs. stoichiometric. Virology 35:550–63.

1016. Snustad, D. P., C. J. Bursch, K. A. Parson, and S. H. Hefeneider. 1976. Mutants of bacteriophage T4 deficient in the ability to induce nuclear disruption: shutoff of host DNA and protein synthesis gene dosage experiments, identification of a restrictive host, and possible biological significance. J. Virol. 18:268–288.

1017. Snustad, D. P., A. C. Casey, and R. E. Herman. 1985. Plasmid-dependent inhibition of growth of bacteriophage T4 ndd mutants. J. Bacteriol. 163: 1290–1292.

1018. Snustad, D. P., and L. M. Conroy. 1974. Mutants of bacteriophage T4 deficient in the ability to induce nuclear disruption. I. Isolation and genetic characterization. J. Mol. Biol. 89:663–673.

1019. Snustad, D. P., M. A. Tigges, K. A. Parson, C. J. Bursch, F. M. Caron, J. F. Koerner, and D. J. Tutas. 1976. Identification and preliminary characterization of a mutant defective in the bacteriophage T4-induced unfolding of the *Escherichia coli* nucleoid. J. Virol. 17:622–641.

1020. Snyder, L., L. Gold, and E. Kutter. 1976. A gene of bacteriophage T4 whose product prevents true late transcription on cytosine-containing T4 DNA. Proc. Natl. Acad. Sci. USA 73:3098–3102.

1021. Snyder, L., and G. Kaufmann. 1994. T4 phage exclusion mechanisms, p. 391–396. *In* J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

1022. Snyder, L. R., and D. L. Montgomery. 1974. Inhibition of T4 growth by an RNA polymerase mutation of *Escherichia coli*: physiological and genetic analysis of the effects during phage development. Virology 62:184–196.

1023. Snyder, M., and W. B. Wood. 1989. Genetic definition of two functional elements in a bacteriophage T4 host-range “cassette.” Genetics 122:471–479.

1024. Sobolev, B. N., and V. V. Mesyanzhinov. 1991. The wac gene product of bacteriophage-T4 contains coiled-coil structural patterns. J. Biomol. Struct. Dyn. 8:953–965.

1025. Solaro, P. C., K. Birkenkamp, P. Pfeiffer, and B. Kemper. 1993. Endonuclease VII of phage T4 triggers mismatch correction in vitro. J. Mol. Biol. 230:868–877.

1026. Sommer, N., V. Salniene, E. Gineikiene, R. Nivinskas, and W. Ruger. 2000. T4 early promoter strength probed in vivo with unribosylated and ADP-ribosylated *Escherichia coli* RNA polymerase: a mutation analysis. Microbiology 146:2643–2653.

1027. Song, H. K., S. H. Sohn, and S. W. Suh. 1999. Crystal structure of deoxycytidylate hydroxymethylase from bacteriophage T4, a component of the deoxyribonucleotide triphosphate-synthesizing complex. EMBO J. 18:1104–1113.

1028. Sozhamannan, S., and B. L. Stitt. 1997. Effects on mRNA degradation by *Escherichia coli* transcription termination factor Rho and pBR322 copy number control protein Rop. J. Mol. Biol. 268:689–703.

1029. Spacciapoli, P., and N. G. Nossal. 1994. Interaction of DNA polymerase

and DNA helicase within the bacteriophage T4 DNA replication complex. J. Biol. Chem. 269:447–455.

1030. Spacciapoli, P., and N. G. Nossal. 1994. A single mutation in bacteriophage T4 DNA polymerase (A737V, tsL141) decreases its processivity as a polymerase and increases its processivity as a 3′ → 5′ exonuclease. J. Biol. Chem. 269:438–446.

1031. Spicer, E. K., J. A. Noble, N. G. Nossal, W. H. Konigsberg, and K. R. Williams. 1982. Bacteriophage T4 gene 45. Sequences of the structural gene and its protein product. J. Biol. Chem. 257:8972–8979.

1032. Spicer, E. K., N. G. Nossal, and K. R. Williams. 1984. Bacteriophage T4 gene 44 DNA polymerase accessory protein. Sequences of gene 44 and its protein product. J. Biol. Chem. 259:15425–15432.

1033. Spicer, E. K., J. Rush, C. Fung, L. J. Reha-Krantz, J. D. Karam, and W. H. Konigsberg. 1988. Primary structure of T4 DNA polymerase. Evolutionary relatedness to eucaryotic and other procaryotic DNA polymerases. J. Biol. Chem. 263:7478–7486.

1034. Spicer, E. K., K. R. Williams, and W. H. Konigsberg. 1979. T4 gene 32 protein trypsin-generated fragments: fluorescence measurement of DNA-binding parameters. J. Biol. Chem. 254:6433–6436.

1035. Stahl, F. W., J. M. Crasemann, C. Yegian, M. M. Stahl, and A. Nakata. 1970. Co-transcribed cistrons in bacteriophage T4. Genetics 54:539–552.

1036. Stemke-Hale, K., and T. Kodadek. 1992. Cloning of 46 and 47, two genes implicated in initiation of homologous recombination in bacteriophage T4. J. Cell. Biochem. Suppl. 16B:98.

1037. Stetler, G. L., G. J. King, and W. M. Huang. 1979. T4 DNA-delay proteins, required for specific DNA replication, form a complex that has ATP-dependent DNA topoisomerase activity. Proc. Natl. Acad. Sci. USA 76: 3737–3741.

1038. Stevens, A. 1972. New small polypeptides associated with DNA-dependent RNA polymerase of *Escherichia coli* after infection with bacteriophage T4. Proc. Natl. Acad. Sci. USA 69:603–607.

1039. Stevens, A. 1976. A salt-promoted inhibitor of RNA polymerase isolated from T4 phage-infected *E. coli*, p. 617–627. In R. Losick and M. Cham- berlin (ed.), RNA polymerase. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.

1040. Stevens, A., and J. C. Rhoton. 1975. Characterization of an inhibitor causing potassium chloride sensitivity of an RNA polymerase from T4 phage-infected *Escherichic coli*. Biochemistry 14:5074–5079.

1041. Reference deleted.

1042. Stillman, B. 2001. Genomic views of genome duplication. Science 294: 2301–2304.

1043. Stitt, B., and D. Hinton. 1994. Regulation of middle-mode transcription, p. 142–160. In J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, Kutter, E., K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4, vol. American Society for Microbiology, Washington, D.C.

1044. Stitt, B. L., and G. Mosig. 1989. Impaired expression of certain prereplicative bacteriophage T4 genes explains impaired T4 DNA synthesis in *Escherichia coli rho* (*nusD*) mutants. J. Bacteriol. 171:3872–3880.

1045. Stitt, B. L., H. R. Revel, I. Lielausis, and W. B. Wood. 1980. Role of the host cell in bacteriophage T4 development. II. Characterization of host mutants that have pleiotropic effects on T4 growth. J. Virol. 35:775–789.

1046. Stocki, S. A., R. L. Nonay, and L. J. Reha-Krantz. 1995. Dynamics of bacteriophage T4 DNA polymerase function: identification of amino acid residues that affect switching between polymerase and 3′–5′ exonuclease activities. J. Mol. Biol. 254:15–28.

1047. Stohr, B. A., and K. N. Kreuzer. 2001. Repair of topoisomerase-mediated DNA damage in bacteriophage T4. Genetics 158:19–28.

1048. Streisinger, G. 1966. Terminal redundancy, or all’s well that ends well, p. 335–340. In J. Cairns, G. S. Stent, and J. D. Watson (ed.), Phage and the origins of molecular biology. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.

1049. Streisinger, G., R. S. Edgar, and G. H. Denhardt. 1964. Chromosome structure in phage T4. I. Circularity of the linkage map. Proc. Natl. Acad. Sci. USA 51:775–779.

1050. Streisinger, G., F. Mukai, W. J. Dreyer, B. Miller, and S. Horiuchi. 1964. Mutations affecting the lysozyme of phage T4. Cold Spring Harbor Symp. Quant. Biol. 26:25–30.

1051. Strelkov, S. V., Y. Tao, M. G. Rossmann, L. P. Kurochkina, M. M. Shneider, and V. V. Mesyanzhinov. 1996. Preliminary crystallographic studies of bacteriophage T4 fibritin confirm a trimeric coiled-coil structure. Virology 219:190–194.

1052. Subbarayan, P. R., and M. P. Deutscher. 2001. *Escherichia coli* RNase M is a multiply altered form of RNase I. RNA 7:1702–1707.

1052a. Sulakvelidze, A., Z. Alavidze, and J. G. Morris, Jr. 2001. Bacteriophage therapy. Antimicrob. Agents Chemother. 45:649–659

1053. Sun, X., J. Harder, M. Krook, H. Jornvall, B. M. Sjoberg, and P. Reichard. 1993. A possible glycine radical in anaerobic ribonucleotide reductase from *Escherichia coli*: nucleotide sequence of the cloned *nrdD* gene. Proc. Natl. Acad. Sci. USA 90:577–581.

1054. Sweezy, M. A., and S. W. Morrical. 1999. Biochemical interactions within a ternary complex of the bacteriophage T4 recombination proteins uvsY and gp32 bound to single-stranded DNA. Biochemistry 38:936–944.

1055. Sweezy, M. A., and S. W. Morrical. 1997. Single-stranded DNA binding properties of the uvsY recombination protein of bacteriophage T4. J. Mol. Biol. 266:927–938.

1056. Symonds, N., H. Heindl, and P. White. 1973. Radiation sensitive mutants of phage T4: a comparative study. Mol. Genet. 120:253–259.

1057. Szewczyk, B., K. Bienkowska-Szewczyk, and L. M. Kozloff. 1986. Identification of T4 gene 25 product, a component of the tail baseplate, as a 15K lysozyme. Mol. Genet. 202:363–367.

1058. Reference deleted.

1059. Takacs, B. J., and J. P. Rosenbusch. 1975. Modification of *Escherichia coli* membranes in the prereplicative phase of phage T4 infection. Specificity of association and quantitation of bound phage proteins. J. Biol. Chem. 250:2339–2250.

1060. Takahashi, H., M. Kobayashi, T. Noguchi, and H. Saito. 1985. Nucleotide sequence of bacteriophage T4 *uvsY* gene. Virology 147:349–353.

1061. Takahashi, H., and H. Saito. 1982. Cloning of *uvsW* and *uvsY* genes of bacteriophage T4. Virology 120:122–129.

1062. Takahashi, H., and H. Yoshikawa. 1979. Genetic study of a new early gene, *comC-α*, of bacteriophage T4. Virology 95:215–217.

1063. Takeda, S., K. Hoshida, and F. Arisaka. 1998. Mapping of functional sites on the primary structure of the tail lysozyme of bacteriophage T4 by mutational analysis. Biochim. Biophys. Acta 1384:243–252.

1064. Tarumi, K., and T. Yonesaki. 1995. Functional interactions of gene 32, 41, and 59 proteins of bacteriophage T4. J. Biol. Chem. 270:2614–2619.

1065. Terzaghi, B. E., E. Terzaghi, and D. Coombs. 1979. The role of the collar/whisker complex in bacteriophage T4D tail fiber attachment. J. Mol. Biol. 127:1–14.

1066. Tessman, I., and D. B. Greenberg. 1972. Ribonucleotide reductase genes of phage T4: map location of the thioredoxin gene *nrdC*. Virology 49:337–338.

1067. Tetart, F., C. Desplats, and H. M. Krisch. 1998. Genome plasticity in the distal tail fiber locus of the T-even bacteriophage: recombination between conserved motifs swaps adhesin specificity. J. Mol. Biol. 282:543–556.

1068. Reference deleted.

1069. Tetart, F., C. Desplats, M. Kutateladze, C. Monod, H. W. Ackermann, and H. M. Krisch. 2001. Phylogeny of the major head and tail genes of the wide-ranging T4-type bacteriophages. J. Bacteriol. 183:358–366.

1070. Tetart, F., C. Monod, and H. M. Krisch. 1996. Bacteriophage T4 host range is expanded by duplications of a small domain of the tail fiber adhesin. J. Mol. Biol. 258:726–731.

1071. Theimer, C. A., Y. Wang, D. W. Hoffman, H. M. Krisch, and D. P. Giedroc. 1998. Non-nearest neighbor effects on the thermodynamics of unfolding of a model mRNA pseudoknot. J. Mol. Biol. 279:545–564.

1072. Reference deleted.

1073. Thomas, J., C. A., and L. A. MacHattie. 1967. The anatomy of viral DNA molecules. Annu. Rev. Biochem. 36:485–518.

1074. Thoms, B., and W. Wackernagel. 1998. Interaction of RecBCD enzyme with DNA at double-strand breaks produced in UV-irradiated escherichia coli: requirement for DNA end processing. J. Bacteriol. 180:5639–5645.

1075. Thylen, C. 1987. Controlling mechanisms for expression of the bacteriophage T4 beta-glucosyltransferase gene. J. Gen. Virol. 68:253–262.

1076. Thylen, C. 1988. Expression and DNA sequence of the cloned bacteriophage T4 dCMP hydroxymethylase gene. J. Bacteriol. 170:1994–1998.

1077. Tiemann, B., R. Depping, and W. Ruger. 1999. Overexpression, purification, and partial characterization of ADP-ribosyltransferases modA and modB of bacteriophage T4. Gene Expression 8:187–196.

1078. Tilly, K., H. Murialdo, and C. Georgopoulos. 1981. Identification of a second *Escherichia coli groE* gene whose product is necessary for bacteriophage morphogenesis. Proc. Natl. Acad. Sci. USA 78:1629–1633.

1079. Tinker, R. L., G. A. Kassavetis, and E. P. Geiduschek. 1994. Detecting the ability of viral, bacterial and eukaryotic replication proteins to track along DNA. EMBO J. 13:5330–5337.

1080. Tinker, R. L., K. P. Williams, G. A. Kassavetis, and E. P. Geiduschek. 1994. The COOH-terminal domain of the RNA polymerase alpha subunit in transcriptional enhancement and deactivation at the bacteriophage T4 late promoter. J. Biol. Chem. 270:15899–15907.

1081. Tinker, R. L., K. P. Williams, G. A. Kassavetis, and E. P. Geiduschek. 1994. Transcriptional activation by a DNA-tracking protein: structural consequences of enhancement at the T4 late promoter. Cell 77:225–237.

1082. Tinker-Kulberg, R. L., T. J. Fu, E. P. Geiduschek, and G. A. Kassavetis. 1996. A direct interaction between a DNA-tracking protein and a promoter recognition protein: implications for searching DNA sequence. EMBO J. 15:5032–5039.

1083. Tomaschewski, J. 1987. Doctoral dissertation. Ruhr Universität, Bochum, Germany.

1084. Tomaschewski, J., H. Gram, J. W. Crabb, and W. Ruger. 1985. T4-induced α- and β-glucosyltransferase: cloning of the genes and a comparison of their products based on sequencing data. Nucleic Acids Res. 13:7551–7568.

1085. Tomaschewski, J., and W. Ruger. 1987. Nucleotide sequence and primary

structures of gene products coded for by the T4 genome between map positions 48.266 kb and 39.166 kb. Nucleic Acids Res. **15**:3632–3633.

1086. Reference deleted.

1087. Tomizawa, J. I., N. Anraku, and Y. Iwama. 1966. Molecular mechanisms of genetic recombination in bacteriophage. VI. A mutant defective in the joining of DNA molecules. J. Mol. Biol. **21**:247–253.

1088. Tomso, D. J., and K. N. Kreuzer. 2000. Double-strand break repair in tandem repeats during bacteriophage T4 infection. Genetics **155**:1493–1504.

1089. Torgov, M. Y., D. M. Janzen, and M. K. Reddy. 1998. Efficiency and frequency of translational coupling between the bacteriophage T4 clamp loader genes. J. Bacteriol. **180**:4339–4343.

1090. Reference deleted.

1091. Trakselis, M. A., S. C. Alley, E. Abel-Santos, and S. J. Benkovic. 2001. Creating a dynamic picture of the sliding clamp during T4 DNA polymerase holoenzyme assembly by using fluorescence resonance energy transfer. Proc. Natl. Acad. Sci. USA **98**:8368–8375.

1092. Trakselis, M. A., M. U. Mayer, F. T. Ishmael, R. M. Roccaseca, and S. J. Benkovic. 2001. Dynamic protein interactions in the bacteriophage T4 replisome. Trends Biochem. Sci. **26**:566–572.

1093. Traub, F., B. Keller, A. Kuhn, and M. Maeder. 1984. Isolation of the prohead core of bacteriophage T4 after cross-linking and determination of protein composition. J. Virol. **49**:902–908.

1094. Traub, F., M. Maeder, and E. Kellenberger. 1981. Bacteriophage T4 head assembly. *In vivo* characterisation of the morphopoietic core. Prog. Clin. Biol. Res. **64**:127–137.

1095. Trojanowska, M., E. S. Miller, J. Karam, G. Stormo, and L. Gold. 1984. The bacteriophage T4 *regA* gene: primary sequence of a translational repressor. Nucleic Acids Res. **12**:5979–5993.

1096. Tschopp, J., F. Arisaka, R. van Driel, and J. Engel. 1979. Purification, characterization and reassembly of the bacteriophage T4D tail sheath protein P18. J. Mol. Biol. **128**:247–258.

1096a. Truncaite, L., A. Zajanckauskaité, and R. Nivinskas. 2002. Identification of two middle promoters upstream of DNA ligase gene 30 of bacteriophage T4. J. Mol. Biol. **317**:179–190.

1097. Tseng, M.-J., J. M. Hilfinger, P. He, and G. R. Greenberg. 1992. Tandem cloning of bacteriophage T4 *nrdA* and *nrdB* genes and overproduction of ribonucleotide reductase (*α₂β₂*) and a mutationally altered form (*α₂β₂*⁹³). J. Bacteriol. **174**:5740–5744.

1098. Tseng, M. J., J. M. Hilfinger, A. Walsh, and G. R. Greenberg. 1988. Total sequence, flanking regions, and transcripts of bacteriophage T4 *nrdA* gene, coding for alpha chain of ribonucleotide reductase. J. Biol. Chem. **263**:16242–16251.

1099. Tsugita, A., and M. Inouye. 1968. Purification of bacteriophage T4 lysozyme. J. Biol. Chem. **243**:391–397.

1100. Tuerk, C., P. Gauss, C. Thermes, D. R. Groebe, M. Gayle, N. Guild, G. Stormo, Y. d’Aubenton-Carafa, O. C. Uhlenbeck, I. Tinoco, Jr., et al. 1988. CUUCGG hairpins: extraordinarily stable RNA secondary structures associated with various biochemical processes. Proc. Natl. Acad. Sci. USA **85**:1364–1368.

1101. Tuerk, C., and L. Gold. 1990. Systematic evolution of ligands by exponential enrichment: RNA ligands to bacteriophage T4 DNA polymerase. Science **249**:505–510.

1102. Ueno, H., and T. Yonesaki. 2001. Recognition and specific degradation of bacteriophage T4 mRNAs. Genetics **158**:7–17.

1103. Urbauer, J. L., K. Adelman, R. J. Urbauer, M. F. Simeonov, J. M. Gilmore, M. Zolkiewski and E. N. Brody EN. 2001. Conserved regions 4.1 and 4.2 of sigma(70) constitute the recognition sites for the anti-sigma factor AsiA, and AsiA is a dimer free in solution. J. Biol. Chem. **276**:41128–41132.

1104. Urbauer, J. L., M. F. Simeonov, R. J. Urbauer, K. Adelman, J. M. Gilmore, and E. N. Brody. 2002. Solution structure and stability of the anti-sigma factor AsiA: implications for novel functions. Proc. Natl. Acad. Sci. USA **99**:1831–1835.

1105. Urig, M. A., S. M. Brown, P. Tedesco, and W. B. Wood. 1983. Attachment of tail fibers in bacteriophage T4 assembly. Identification of the baseplate protein to which tail fibers attach. J. Mol. Biol. **169**:427–437.

1106. Uzan, M., E. Brody, and R. Favre. 1990. Nucleotide sequence and control of transcription of the bacteriophage T4 *motA* regulatory gene. Mol. Microbiol. **4**:1487–1496.

1107. Uzan, M., J. Leautey, Y. d’Aubenton-Carafa, and E. Brody. 1983. Identification and biosynthesis of the bacteriophage T4 mot regulatory protein. EMBO J. **2**:1207–1212.

1108. Vaishkunaite, R. I., and R. G. Nivinskas. 1990. Gene 26 of bacteriophage T4 basal plate. I. Two expression products of the cloned gene. Mol. Biol. (Moscow) **24**:379–390. (In Russian.)

1109. Vaiskunaite, R., A. Miller, L. Davenport, and G. Mosig. 1999. Two new early bacteriophage T4 genes, *repEA* and *repEB*, that are important for DNA replication initiated from origin E. J. Bacteriol. **181**:7115–7125.

1110. Valerie, K., E. E. Henderson, and J. K. de Riel. 1985. Expression of a cloned *denV* gene of bacteriophage T4 in *Escherichia coli*. Proc. Natl. Acad. Sci. USA **82**:4763–4767.


1111. Valerie, K., E. E. Henderson, and J. K. deRiel. 1984. Identification, physical map location and sequence of the *denV* gene from bacteriophage T4. Nucleic Acids Res. **12**:8085–8096.

1112. Valerie, K., J. Stevens, M. Lynch, E. E. Henderson, and J. K. de Riel. 1986. Nucleotide sequence and analysis of the 58.3 to 65.5-kb early region of bacteriophage T4. Nucleic Acids Res. **14**:8637–8654.

1113. Valleé, M., and J. B. Cornett. 1972. A new gene of bacteriophage T4 determining immunity against superinfecting ghosts and phage in T4-infected *Escherichia coli*. Virology **48**:777–784.

1114. van der Heusen, P. 1997. Doctoral dissertation. Ruhr-Universität Bochum, Germany.

1114a. Vanderslice, R. W., and C. D. Yegian. 1974. The identification of late bacteriophage T4 proteins on sodium dodecyl sulfate polyacrylamide gels. Virology **60**:265–275.

1114b. van der Vate, C., P. van der Ende, and N. Symonds. 1974. A stable duplication of the rII region of bacteriophage T4. Genet. Res. **23**:107–113.

1115. van der Vies, S. M., A. A. Gatenby, and C. Georgopoulos. 1994. Bacteriophage T4 encodes a co-chaperonin that can substitute for *Escherichia coli* GroES in protein folding. Nature **368**:654–656.

1116. van Driel, R., F. Traub, and M. K. Showe. 1980. Probable localization of the bacteriophage T4 prehead proteinase zymogen in the center of the prehead core. J. Virol. **36**:220–223.

1117. van Minderhout, L., J. Grimbergen, and B. de Groot. 1979. Bacteriophage T4 v mutants with a slow rate of thymine-dimer excision and a particular reactivation phenotype. Mutat. Res. **60**:253–262.

1118. van Raaij, M. J., G. Schoehn, M. R. Burda, and S. Miller. 2001. Crystal structure of a heat and protease-stable part of the bacteriophage T4 short tail fiber. J. Mol. Biol. **314**:1137–1146.

1119. Reference deleted.

1120. Vassylyev, D. G., T. Kashiyagi, Y. Mikami, M. Ariyoshi, S. Iwai, E. Ohtsuka, and K. Morikawa. 1995. Atomic model of a pyrimidine dimer excision repair enzyme complexed with a DNA substrate: structural basis for damaged DNA recognition. Cell **83**:773–782.

1121. Reference deleted.

1122. Venkatesan, M., L. L. Silver, and N. G. Nossal. 1982. Bacteriophage T4 gene 41 protein, required for the synthesis of RNA primers, is also a DNA helicase. J. Biol. Chem. **257**:12426–12434.

1123. Vetter, D., and P. D. Sadowski. 1974. Point mutants in the D2a region of bacteriophage T4 fail to induce T4 endonuclease IV. J. Virol. **14**:207–213.

1124. Vianelli, A., G. R. Wang, M. Gingery, R. L. Duda, F. A. Eiserling, and E. B. Goldberg. 2000. Bacteriophage T4 self-assembly: localization of gp3 and its role in determining tail length. J. Bacteriol. **182**:680–688.

1125. Villemain, J. L., and D. P. Giedroc. 1996. Characterization of a cooperativity domain mutant Lys3 → Ala (K3A) T4 gene 32 protein. J. Biol. Chem. **271**:27623–27629.

1126. Villemain, J. L., and D. P. Giedroc. 1993. Energetics of arginine-4 substitution mutants in the N-terminal cooperativity domain of T4 gene 32 protein. Biochemistry **32**:11235–11246.

1127. Villemain, J. L., and D. P. Giedroc. 1996. The N-terminal B-domain of T4 gene 32 protein modulates the lifetime of cooperatively bound Gp32-ss nucleic acid complexes. Biochemistry **35**:14395–14404.

1128. Villemain, J. L., Y. Ma, D. P. Giedroc, and S. W. Morrical. 2000. Mutations in the N-terminal cooperativity domain of gene 32 protein alter properties of the T4 DNA replication and recombination systems. J. Biol. Chem. **275**:31496–31504.

1129. Visconti, N., and M. Delbrück. 1953. The mechanism of genetic recombination in phage. Genetics **38**:5–33.

1130. Volker, T. A., J. Gafner, T. A. Bickle, and M. K. Showe. 1982. Gene 67, a new, essential bacteriophage T4 head gene codes for a prehead core component, PIP. I. Genetic mapping and DNA sequence. J. Mol. Biol. **161**:479–489.

1131. Volker, T. A., A. Kuhn, M. K. Showe, and T. A. Bickle. 1982. Gene 67, a new, essential bacteriophage T4 head gene codes for a prehead core component, PIP. II. The construction in vitro of unconditionally lethal mutants and their maintenance. J. Mol. Biol. **161**:491–504.

1132. von Hippel, P. H., S. C. Kowalczykowski, N. Lonberg, J. W. Newport, L. S. Paul, G. D. Stormo, and L. Gold. 1983. Autoregulation of expression of T4 gene 32: a quantitative analysis, p. 202–207. *In* C. K. Mathews, E. M. Kutter, G. Mosig, and P. B. Berget (ed.), Bacteriophage T4. American Society for Microbiology, Washington, D.C.

1133. von Hippel, P. H., S. C. Kowalczykowski, N. Lonberg, J. W. Newport, L. S. Paul, G. D. Stormo, and L. Gold. 1982. Autoregulation of gene expression. Quantitative evaluation of the expression and function of the bacteriophage T4 gene 32 (single-stranded DNA binding) protein system. J. Mol. Biol. **162**:795–818.

1134. Vrielink, A., W. Rüger, H. P. C. Driessen, and P. S. Freemont. 1994. Crystal structure of the DNA modifying enzyme β-glucosyltransferase in the presence and absence of the substrate uridine diphosphoglucose. EMBO J. **13**:3413–3422.

1135. Waga, S., and B. Stillman. 1994. Anatomy of a DNA replication fork revealed by reconstitution of SV40 DNA replication in vitro. Nature **369**:207–212.

1136. Waidner, L. A., E. K. Flynn, M. Wu, X. Li, and R. L. Karpel. 2001. Domain effects on the DNA-interactive properties of bacteriophage T4 gene 32 protein. J. Biol. Chem. 276:2509–2516.

1137. Wais, A. C., and E. B. Goldberg. 1969. Growth and transformation of phage T4 in Escherichia coli B-4, Salmonella, Aerobacter, Proteus, and Serratia. Virology 39:153–161.

1138. Wakem, L. P., and K. Ebisuzaki. 1981. DNA repair-recombination functions in the DNA processing pathway of bacteriophage T4. Virology 112: 472–479.

1139. Wakem, L. P., and K. Ebisuzaki. 1984. A new suppressor of mutations in the DNA repair-recombination genes of bacteriophage T4: sur. Virology 137:331–337.

1140. Waldsich, C., R. Grossberger, and R. Schroeder. 2002. RNA chaperone StpA loosens interactions of the tertiary structure in the td group I intron in vivo. Genes Dev. 16:2300–2312.

1141. Wang, C.-C., A. Pavlov, and J. D. Karam. 1997. Evolution of RNA-binding specificity in T4 DNA polymerase. J. Biol. Chem. 272:17703–17710.

1142. Wang, C.-C., L.-S. Yeh, and J. D. Karam. 1995. Modular organization of T4 DNA polymerase. J. Biol. Chem. 270:26558–26564.

1143. Wang, F. J., and L. S. Ripley. 1998. The spectrum of acridine resistant mutants of bacteriophage T4 reveals cryptic effects of the tsL141 DNA polymerase allele on spontaneous mutagenesis. Genetics 148:1655–1665.

1144. Wang, G. R., A. Vianelli, and E. B. Goldberg. 2000. Bacteriophage T4 self-assembly: in vitro reconstitution of recombinant gp2 into infectious phage. J. Bacteriol. 182:672–679.

1145. Wang, I. N., D. L. Smith, and R. Young. 2000. Holins: the protein clocks of bacteriophage infections. Annu. Rev. Microbiol. 54:799–825.

1146. Wang, J., A. K. M. A. Sattar, C. C. Wang, J. D. Karam, W. H. Konigsberg, and T. A. Steitz. 1997. Crystal structure of a pol α family replication DNA polymerase from bacteriophage RB69. Cell 89:1087–1099.

1147. Wang, J., P. Yu, T. C. Lin, W. H. Konigsberg, and T. A. Steitz. 1996. Crystal structures of an NH2-terminal fragment of T4 DNA polymerase and its complexes with single-stranded DNA and with divalent metal ions. Biochemistry 35:8110–8119.

1148. Ward, S., and R. C. Dickson. 1971. Assembly of bacteriophage T4 tail fibers. 3. Genetic control of the major tail fiber polypeptides. J. Mol. Biol. 62:479–492.

1149. Warner, H. R. 1971. Partial suppression of bacteriophage T4 ligase mutations by T4 endonuclease II deficiency: role of host ligase. J. Virol. 7:534–536.

1150. Warner, H. R., and J. E. Barnes. 1966. Evidence for a dual role for the bacteriophage T4-induced deoxycytidine triphosphate nucleotidohydro- lase. Proc. Natl. Acad. Sci. USA 56:1233–1240.

1151. Warner, H. R., L. M. Christensen, and M. L. Persson. 1981. Evidence that the UV endonuclease activity induced by bacteriophage T4 contains both pyrimidine dimer-DNA glycosylase and apyrimidinic/apurinic endonucle- ase activities in the enzyme molecule. J. Virol. 40:204–210.

1152. Warner, H. R., D. P. Snustad, J. F. Koerner, and J. D. Childs. 1972. Identification and genetic characterization of mutants of bacteriophage T4 defective in the ability to induce exonuclease A. J. Virol. 9:399–407.

1153. Warner, H. R., P. Snustad, S. E. Jorgensen, and J. F. Koerner. 1970. Isolation of bacteriophage T4 mutants defective in the ability to degrade host deoxyribonucleic acid. J. Virol. 5:700–708.

1154. Washburn, R. S., and B. L. Stitt. 1996. In vitro characterization of transcription termination factor Rho from Escherichia coli rho(nusD) mutants. J. Mol. Biol. 260:332–346.

1155. Watts, N. R., and D. H. Coombs. 1989. Analysis of near-neighbor contacts in bacteriophage T4 wedges and hubless baseplates by using a cleavable chemical cross-linker. J. Virol. 63:2427–2436.

1156. Watts, N. R., J. Hainfeld, and D. H. Coombs. 1990. Localization of the proteins gp7, proteins gp8 and proteins gp10 in the bacteriophage T4 baseplate with colloidal gold: F(ab)₂ and Undecagold: Fab’ conjugates. J. Mol. Biol. 216:315–325.

1157. Watts, N. R. M., and D. H. Coombs. 1990. Structure of the bacteriophage T4 baseplate as determined by chemical cross-linking. J. Virol. 64:143– 154.

1158. Weaver, L. H., and B. W. Matthews. 1987. Structure of bacteriophage T4 lysozyme refined at 1.7 Å resolution. J. Mol. Biol. 193:189–199.

1159. Weil, J., and B. Terzaghi. 1970. The correlated occurrence of duplications and deletions in phage T4. Virology 42:234–237.

1160. Weintraub, S. B., and F. R. Frankel. 1972. Identification of the T4rIIB gene product as a membrane protein. J. Mol. Biol. 70:589–615.

1161. Wheeler, L. J., N. B. Ray, C. Ungermann, S. P. Hendricks, M. A. Bernard, E. S. Hanson, and C. K. Mathews. 1996. T4 phage gene 32 protein as a candidate organizing factor for the deoxyribonucleoside triphosphate synthetase complex. J. Biol. Chem. 271:11156–11162.

1162. Wiberg, J. S. 1967. Amber mutants of bacteriophage T4 defective in deoxycytidine diphosphatase and deoxycytidine triphosphatase. On the role of 5-hydroxymethylcytosine in bacteriophage deoxyribonucleic acid. J. Biol. Chem. 242:5842–5829.

1163. Wiberg, J. S. 1966. Mutants of bacteriophage T4 unable to cause break- down of host DNA. Proc. Natl. Acad. Sci. USA 55:614–621.

1164. Wiberg, J. S., T. S. Cardillo, and C. Mickelson. 1981. Genetic and amber fragment maps of genes 46 and 47 of bacteriophage T4D. J. Virol. 40: 309–313.

1165. Wiberg, J. S., M.-L. Dirksen, R. H. Epstein, S. E. Luria, and J. M. Buchanan. 1962. Early enzyme synthesis and its control in E. coli infected with some amber mutants of bacteriophage T4. Proc. Natl. Acad. Sci. USA 48:293–302.

1166. Wiberg, J. S., and J. D. Karam. 1983. Translational regulation in T4 phage development, p. 193–201. In C. K. Mathews, E. Kutter, G. Mosig, and P. B. Berget (ed.), Bacteriophage T4. American Society for Microbiology, Washington, D.C.

1167. Wiberg, J. S., S. Mendelsohn, V. Warner, K. Hercules, C. Aldrich, and J. L. Munro. 1973. SP62, a viable mutant of bacteriophage T4D defective in regulation of phage enzyme synthesis. J. Virol. 12:775–792.

1168. Wilkens, K., and W. Rüger. 1996. Characterization of bacteriophage T4 early promoters in vivo with a new promoter probe vector. Plasmid 35: 108–120.

1169. Wilkens, K., and W. Rüger. 1994. Transcription from early promoters, p. 132–141. In J. D. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

1170. Wilkens, K., B. Tiemann, F. Bazan, and W. Rüger. 1997. ADP-ribosylation and early transcription regulation by bacteriophage T4, p. 71–82. In F. Haag and F. Koch-Nolte (ed.), ADP-ribosylation in animal tissue. Plenum Press, New York, N.Y.

1171. Williams, K. P., G. A. Kassavetis, F. S. Esch, and E. P. Geiduschek. 1987. Identification of the gene encoding an RNA polymerase-binding protein of bacteriophage T4. J. Virol. 61:597–599.

1172. Williams, K. P., G. A. Kassavetis, and E. P. Geiduschek. 1987. Interactions of the bacteriophage T4 gene 55 product with Escherichia coli RNA polymerase. Competition with Escherichia coli σ⁷⁰ and release from late T4 transcription complexes following initiation. J. Biol. Chem. 262:12365– 12371.

1173. Williams, K. P., G. A. Kassavetis, D. R. Herendeen, and E. P. Geiduschek. 1994. Regulation of late-gene expression, p. 161–175. In J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Wash- ington, D.C.

1174. Reference deleted.

1175. Williams, K. P., R. Muller, W. Ruger, and E. P. Geiduschek. 1989. Over- produced bacteriophage T4 gene 33 protein binds RNA polymerase. J. Bacteriol. 171:3579–3582.

1176. Williams, K. R., M. B. LoPresti, and M. Setoguchi. 1981. Primary struc- ture of the bacteriophage T4 DNA helix-destabilizing protein. J. Biol. Chem. 256:1754–1762.

1177. Wilson, G. G., and N. E. Murray. 1979. Molecular cloning of the DNA ligase gene from bacteriophage T4. I. Characterisation of the recombi- nants. J. Mol. Biol. 132:471–491.

1178. Wilson, J. H. 1973. Function of the bacteriophage T4 transfer RNA’s. J. Mol. Biol. 74:753–757.

1179. Wilson, J. H., and J. N. Abelson. 1972. Bacteriophage T4 transfer RNA. II. Mutants of T4 defective in the formation of functional suppressor transfer RNA. J. Mol. Biol. 69:57–73.

1180. Wilson, J. H., J. S. Kim, and J. N. Abelson. 1972. Bacteriophage T4 transfer RNA. 3. Clustering of the genes for the T4 transfer RNA’s. J. Mol. Biol. 71:547–556.

1181. Winkelman, J. W., G. A. Kassavetis, and E. P. Geiduschek. 1994. Molec- ular genetic analysis of a prokaryotic transcriptional coactivator: func- tional domains of the bacteriophage T4 gene 33 protein. J. Bacteriol. 176:1164–1171.

1182. Winter, R. B., L. Morrissey, P. Gauss, L. Gold, T. Hsu, and J. Karam. 1987. Bacteriophage T4 regA protein binds to mRNAs and prevents trans- lation initiation. Proc. Natl. Acad. Sci. USA 84:7822–7826.

1183. Woese, C. R., S. Winker, and R. R. Gutell. 1990. Architecture of ribosomal RNA—constraints on the sequence of tetra-loops. Proc. Natl. Acad. Sci. USA 87:8467–8471.

1184. Wommack, K. E., and R. R. Colwell. 2000. Virioplankton: viruses in aquatic ecosystems. Microbiol. Mol. Biol. Rev. 64:69–114.

1185. Wong, D. L., J. G. Pavlovich, and N. O. Reich. 1998. Electrospray ionization mass spectrometric characterization of photocrosslinked DNA-EcoRI DNA methyltransferase complexes. Nucleic Acids Res. 26:645–649.

1186. Wong, K., and E. P. Geiduschek. 1998. Activator-sigma interaction: A hydrophobic segment mediates the interaction of a sigma family promoter recognition protein with a sliding clamp transcription activator. J. Mol. Biol. 284:195–203.

1187. Wood, W. B., and J. R. Bishop. 1973. Bacteriophage T4 tail fibers: struc- ture and assembly of a viral organelle, p. 303–304. In C. F. Fox and W. F. Robinson (ed.), Virus research. Academic Press, Inc., New York, N.Y.

1188. Wood, W. B., and M. P. Conley. 1979. Attachment of tail fibers in bacte- riophage T4 assembly: role of the phage whiskers. J. Mol. Biol. 127:15–29.

1189. Wood, W. B., F. A. Eiserling, and R. A. Crowther. 1994. Long tail fibers: genes, proteins, structure and assembly, p. 282–290. *In* J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

1190. Wood, W. B., and M. Henninger. 1969. Attachment of tail fibers in bacteriophage T4 assembly: some properties of the reaction in vitro and its genetic control. J. Mol. Biol. **39**:603–618.

1191. Woodworth, D. L., and K. N. Kreuzer. 1996. Bacteriophage T4 mutants hypersensitive to an antitumor agent that induces topoisomerase-DNA cleavage complexes. Genetics **143**:1081–1090.

1192. Wovcha, M. G., P. K. Tomich, C. S. Chiu, and G. R. Greenberg. 1973. Direct participation of dCMP hydroxymethylase in synthesis of bacteriophage T4 DNA. Proc. Natl. Acad. Sci. USA **70**:2196–2200.

1193. Wozniak, J. A., X.-J. Zhang, L. H. Weaver, and B. W. Matthews. 1994. Structural and genetic analysis of the stability and function of T4 lysozyme, p. 332–339. *In* J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

1194. Wu, C. H., and L. W. Black. 1995. Mutational analysis of the sequence-specific recombination box for amplification of gene 17 of bacteriophage T4. J. Mol. Biol. **247**:604–617.

1195. Wu, C. H., H. Lin, and L. W. Black. 1995. Bacteriophage T4 gene 17 amplification mutants: evidence for initiation by the T4 terminase subunit gp16. J. Mol. Biol. **247**:523–528.

1196. Wu, D. G., C. H. Wu, and L. W. Black. 1991. Reiterated gene amplifications at specific short homology sequences in phage T4 produce *hp17* mutants. J. Mol. Biol. **218**:705–721.

1197. Wu, J.-R., and Y.-C. Yeh. 1978. New late gene, *dar*, involved in the replication of bacteriophage T4 DNA. III. DNA replicative intermediates of T4 *dar* and a gene 59 mutant suppressed by *dar*. J. Virol. **27**:90–102.

1198. Wu, J. R., and Y. C. Yeh. 1973. Requirement of a functional gene 32 product of bacteriophage T4 in UV repair. J. Virol. **12**:758–765.

1199. Wu, J. R., Y. C. Yeh, and K. Ebisuzaki. 1984. Genetic analysis of *dar*, *uvsW*, and *uvsY* in bacteriophage T4: *dar* and *uvsW* are alleles. J. Virol. **52**:1028–1031.

1200. Wu, M., E. K. Flynn, and R. L. Karpel. 1999. Details of the nucleic acid binding site of T4 gene 32 protein revealed by proteolysis and DNA Tm depression methods. J. Mol. Biol. **286**:1107–1121.

1201. Wu, R., F. Ma, and Y.-C. Yeh. 1972. Suppression of DNA-arrested synthesis in mutants defective in gene 59 of bacteriophage T4. Virology **47**:147–156.

1202. Wu, R., and Y. C. Yeh. 1974. DNA arrested mutants of gene 59 of bacteriophage T4. II. Replicative intermediates. Virology **59**:108–122.

1203. Xu, M. Q., and D. A. Shub. 1989. The catalytic core of the *sunY* intron of bacteriophage T4. Gene **82**:77–82.

1204. Xu, W., P. Gauss, J. Shen, C. A. Dunn, and M. J. Bessman. 2002. The gene *e.I* (*nudE.I*) of T4 bacteriophage designates a new member of the Nudix hydrolase superfamily active on flavin adenine dinucleotide, adenosine (5′) triphospho(5′)adenosine, and ADP-ribose. J. Biol. Chem. **277**:23181–23185.

1205. Yamaguchi, Y., and M. Yanagida. 1980. Head shell protein hoc alters the surface charge of bacteriophage T4. Composite slab gel electrophoresis of phage T4 and related particles. J. Mol. Biol. **141**:175–193.

1206. Yang, G., C. Cecconi, W. A. Baase, I. R. Vetter, W. A. Breyer, J. A. Haack, B. W. Matthews, F. W. Dahlquist, and C. Bustamante. 2000. Solid-state synthesis and mechanical unfolding of polymers of T4 lysozyme. Proc. Natl. Acad. Sci. USA **97**:139–144.

1207. Yang, G., T. Lin, J. Karam, and W. H. Konigsberg. 1999. Steady-state kinetic characterization of RB69 DNA polymerase mutants that affect dNTP incorporation. Biochemistry **38**:8094–8101.

1208. Yankovsky, N. K., and V. N. Krylov. 1975. A genetic and physiologic study of mutations of T4 phage suppressing the lysis defect of gene stII mutants. Genetika **11**:51–60. (In Russian.)

1209. Yarnell, W. S., and J. W. Roberts. 1999. Mechanism of intrinsic transcription termination and antitermination. Science **284**:611–675.

1210. Yassa, D. S., K. M. Chou, and S. W. Morrical. 1997. Characterization of an amino-terminal fragment of the bacteriophage T4 *uvsY* recombination protein. Biochimie **79**:275–285.

1211. Reference deleted.

1212. Yasuda, S., and M. Sekiguchi. 1970. Mechanism of repair of DNA in bacteriophage. II. Inability of ultraviolet-sensitive strains of bacteriophage in inducing an enzyme activity to excise pyrimidine dimers. J. Mol. Biol. **47**:243–255.

1213. Yasuda, S., and M. Sekiguchi. 1970. T4 endonuclease involved in repair of DNA. Proc. Natl. Acad. Sci. USA **67**:1839–1845.

1214. Yee, J. K., and R. C. Marsh. 1981. Alignment of a restriction map with the genetic map of bacteriophage T4. J. Virol. **38**:115–124.

1215. Yee, J. K., and R. C. Marsh. 1985. Locations of bacteriophage T4 origins of replication. J. Virol. **54**:271–277.

1216. Yegian, C. D., M. Mueller, G. Selzer, V. Russo, and F. W. Stahl. 1971. Properties of DNA-delay mutants of bacteriophage T4. Virology **46**:900–919.

1217. Yeh, L.-S., T. Hsu, and J. D. Karam. 1998. Divergence of a DNA replication gene cluster in the T4-related bacteriophage RB69. J. Bacteriol. **180**:2005–2013.

1218. Yeh, Y. C., E. J. Dubovi, and I. Tessman. 1969. Control of pyrimidine biosynthesis by phage T4: mutants unable to catalyze the reduction of cytidine diphosphate. Virology **37**:615–623.

1219. Yeh, Y. C., and I. Tessman. 1972. Control of pyrimidine biosynthesis by phage T4. II. In vitro complementation between ribonucleotide reductase mutants. Virology **47**:767–772.

1220. Yonesaki, T. 1994. Involvement of a replicative DNA helicase of bacteriophage T4 in DNA recombination. Genetics **138**:247–252.

1221. Yonesaki, T. 1994. The purification and characterization of gene 59 protein from bacteriophage T4. J. Biol. Chem. **269**:1284–1289.

1222. Yonesaki, T., and T. Minagawa. 1987. Studies on the recombination genes of bacteriophage T4: suppression of *uvsX* and *uvsY* mutations by *uvsW* mutations. Genetics **115**:219–227.

1223. Yonesaki, T., and T. Minagawa. 1989. Synergistic action of three recombination gene products of bacteriophage T4, *uvsX*, *uvsY*, and gene 32 proteins. J. Biol. Chem. **264**:7814–7820.

1224. Yonesaki, T., and T. Minagawa. 1985. T4 phage gene *uvsX* product catalyzes homologous DNA pairing. EMBO J. **4**:3321–3327.

1225. Yonesaki, T., Y. Ryo, T. Minagawa, and H. Takahashi. 1985. Purification and some of the functions of the products of bacteriophage T4 recombination genes, *uvsX* and *uvsY*. Eur. J. Biochem. **148**:127–134.

1226. Youil, R., B. W. Kemper, and R. G. H. Cotton. 1995. Screening for mutations by enzyme mismatch cleavage with T4 endonuclease VII. Proc. Natl. Acad. Sci. USA **92**:87–91.

1227. Young, M. C., M. K. Reddy, T. C. Jarvis, E. P. Gogol, M. K. Dolejsi, and P. von Hippel. 1994. Protein-protein and protein-DNA interactions in the T4 DNA polymerase accessory protein complex, p. 313–317. *In* J. Karam, J. W. Drake, K. N. Kreuzer, G. Mosig, D. H. Hall, F. A. Eiserling, L. W. Black, E. K. Spicer, E. Kutter, K. Carlson, and E. S. Miller (ed.), Molecular biology of bacteriophage T4. American Society for Microbiology, Washington, D.C.

1228. Young, P., M. Ohman, and B. M. Sjoberg. 1994. Bacteriophage T4 gene 55.9 encodes an activity required for anaerobic ribonucleotide reduction. J. Biol. Chem. **269**:27815–27818.

1229. Young, P., M. Ohman, M. Q. Xu, D. A. Shub, and B. M. Sjoberg. 1994. Intron-containing T4 bacteriophage gene *sunY* encodes an anaerobic ribonucleotide reductase. J. Biol. Chem. **269**:20229–20232.

1230. Yu, J. S., S. Madison-Antenucci, and D. A. Steege. 2001. Translation at higher than an optimal level interferes with coupling at an intercistronic junction. Mol. Microbiol. **42**:821–834.

1231. Yu, Y.-T. N., and L. Snyder. 1994. Translational elongation factor Tu cleaved by a phage-exclusion system. Proc. Natl. Acad. Sci. USA **91**:802–806.

1232. Yutsudo, M. 1979. Regulation of *imm* gene expression in bacteriophage T4-infected cells. J. Gen. Virol. **45**:351–359.

1233. Zachary, A. 1978. An ecological study of bacteriophages of Vibrio natriegens. Can. J. Microbiol. **24**:321–324.

1234. Zajanckauskaitė, A., A. Malys, and R. Nivinskas. 1997. A rare type of overlapping genes in bacteriophage T4: gene 30.3′ is completely embedded within gene 30.3 by one position downstream. Gene **194**:157–162.

1235. Zajanckauskaitė, A., A. Raudonikiene, and R. Nivinskas. 1994. Cloning and expression of genes from the genomic region between genes *cd* and 30 of bacteriophage T4. Gene **147**:71–76.

1236. Zechiedrich, E. L., A. B. Khodursky, S. Bachellier, R. Schneider, D. Chen, D. M. Lilley, and N. R. Cozzarelli. 2000. Roles of topoisomerases in maintaining steady-state DNA supercoiling in *Escherichia coli*. J. Biol. Chem. **275**:8103–8113.

1237. Zeeh, A., and D. A. Shub. 1991. The product of the split *sunY* gene of

bacteriophage T4 is a processed protein. J. Bacteriol. 173:6980–6985.

1238. Zhang, A., V. Derbyshire, J. L. Salvo, and M. Belfort. 1995. *Escherichia coli* protein StpA stimulates self-splicing by promoting RNA assembly in vitro. RNA 1:783–793.

1239. Zhao, L., S. Takeda, P. G. Leiman, and F. Arisaka. 2000. Stoichiometry and inter-subunit interaction of the wedge initiation complex, gp10-gp11,

of bacteriophage T4. Biochim. Biophys. Acta 1479:286–292.

1240. Zograff, Y. N., V. V. Ogryz'ko, I. A. Bass, and D. I. Chernyi. 1985. The region of phage T4 W-29 genes: cloning and expression. Mol. Biol. (Moscow) 19:818–832. (In Russian.)

1241. Zograff, Y. N., and A. L. Gintsburg. 1980. Transcription termination factor rho and T-even phage development. Mol. Gen. Genet. 177:699–705.
