
Metabolism and Bioenergetics:  
The Saccharomyces cerevisiae COQ6 Gene  
Encodes a Mitochondrial Flavin-dependent  
Monooxygenase Required for Coenzyme Q  
Biosynthesis  

Peter Gin, Adam Y. Hsu, Steven C. Rothman,  
Tanya Jonassen, Peter T. Lee, Alexander  
Tzagoloff and Catherine F. Clarke  
*J. Biol. Chem.* 2003, 278:25308-25316.  
doi: 10.1074/jbc.M303234200 originally published online April 29, 2003  

Access the most updated version of this article at doi: [10.1074/jbc.M303234200](http://dx.doi.org/10.1074/jbc.M303234200)  

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinity-sites).

Alerts:
- [When this article is cited](#)
- [When a correction for this article is posted](#)

[Click here](#) to choose from all of JBC's e-mail alerts

This article cites 60 references, 29 of which can be accessed free at  
[http://www.jbc.org/content/278/28/25308.full.html#ref-list-1](http://www.jbc.org/content/278/28/25308.full.html#ref-list-1)

THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2003 by The American Society for Biochemistry and Molecular Biology, Inc.
Vol. 278, No. 28, Issue of July 11, pp. 25308–25316, 2003
Printed in U.S.A.

# The Saccharomyces cerevisiae COQ6 Gene Encodes a Mitochondrial Flavin-dependent Monooxygenase Required for Coenzyme Q Biosynthesis*

Received for publication, March 28, 2003
Published, JBC Papers in Press, April 29, 2003, DOI 10.1074/jbc.M303234200

Peter Gin‡, Adam Y. Hsu‡, Steven C. Rothman‡§, Tanya Jonassen‡, Peter T. Lee‡,
Alexander Tzagoloff¶, and Catherine F. Clarke‡

From the ‡Department of Chemistry and Biochemistry and the Molecular Biology Institute, University of California, Los Angeles, California 90095 and the ¶Department of Biological Sciences, Columbia University, New York, New York 10027

Coenzyme Q (Q) is a lipid that functions as an electron carrier in the mitochondrial respiratory chain in eukaryotes. There are eight complementation groups of Q-deficient Saccharomyces cerevisiae mutants, designated *coq1-coq8*. Here we have isolated the *COQ6* gene by functional complementation and, in contrast to a previous report, find it is not an essential gene. *coq6* mutants are unable to grow on nonfermentable carbon sources and do not synthesize Q but instead accumulate the Q biosynthetic intermediate 3-hexaprenyl-4-hydroxybenzoic acid. The Coq6 polypeptide is imported into the mitochondria in a membrane potential-dependent manner. Coq6p is a peripheral membrane protein that localizes to the matrix side of the inner mitochondrial membrane. Based on sequence homology to known proteins, we suggest that *COQ6* encodes a flavin-dependent monooxygenase required for one or more steps in Q biosynthesis.

---

Coenzyme Q (ubiquinone or Q)¹ is a prenylated benzoquinone found in cell membranes that functions in redox chemistry as both an oxidant (Q) and reductant (*QH₂*) (1). Q is most commonly associated with its role in the respiratory chain where it transports electrons from either Complex I or Complex II to Complex III (2); however, Q serves multiple functions. Q has been demonstrated to play a role in stabilizing the *bc₁* complex (3). Additionally, Q functions in the electron transport chains of lysosomal and plasma membranes (4, 5), and *QH₂* acts as a chain breaking antioxidant of lipid peroxyl radicals (6). In *Escherichia coli*, a high *QH₂*:Q ratio is sensed by ArcB, a transmembrane sensor kinase, that phosphorylates ArcA, activating operons involved in fermentation and repressing those involved in respiration (7). In *Caenorhabditis elegans*, dietary Q produces a shortened life span (8). This phenomenon has been attributed to the generation of superoxide by *Q⁻* (the Q semiquinone radical) generated during respiratory electron transport (9). In humans, Q supplementation has been shown to be effective in treating patients with specific respiratory chain defects (10) and to slow the progression of Parkinson's disease symptoms (11).

Cells normally acquire Q through *de novo* synthesis, and the length of the prenyl tail varies among different organisms (12). *Saccharomyces cerevisiae* produce Q₆, which has six isoprene units, whereas *E. coli* synthesizes Q₈, and humans synthesize Q₁₀. Eight *COQ* genes have been identified to be required for Q synthesis in *S. cerevisiae* (13, 14). Fig. 1 shows the pathway for Q biosynthesis in both prokaryotes and eukaryotes. In yeast, mutation in any of the eight *COQ* genes results in cells that cannot synthesize Q and fail to grow on nonfermentable carbon sources. Yeast *coq3-coq8* mutants each accumulate the same early intermediate in Q biosynthesis, 3-hexaprenyl-4-hydroxybenzoic acid (Fig. 1, HHB or compound 1) (15).

In this work the yeast *COQ6* gene has been isolated by functional complementation of a mutant from the G63 (*coq6*) complementation group. Transformation of *coq6* mutant strains with a plasmid bearing wild type *COQ6* restores Q biosynthesis and growth on nonfermentable carbon sources. In contrast to a previous report (16), we find *COQ6* to be a nonessential gene. Here we show the Coq6 polypeptide is imported into mitochondria and is peripherally associated with the inner membrane on the matrix side. Based on sequence analysis and alignment with other known hydroxylases, the proposed function of Coq6 polypeptide is that of a flavin-dependent monooxygenase required for one or more steps in Q biosynthesis.

## EXPERIMENTAL PROCEDURES

### Strains and Growth Media—The strains used in this study are listed in Table I. Growth media for yeast were prepared as described (17) and included YPD (1% yeast extract, 2% peptone, 2% dextrose), YPG (1% yeast extract, 2% peptone, 3% glycerol), YPGal (1% yeast extract, 2% peptone, 2% galactose, 0.1% dextrose), SDC (0.18% yeast nitrogen base without amino acids, 2% dextrose, 0.14% NaH₂PO₄, 0.5% (NH₄)₂SO₄, and complete supplement of amino acids), SD-Leu (SDC minus leucine), and SD-Ura (SDC minus uracil). The complete supplement was modified as described (18). Semisynthetic lactate medium was prepared as described (19). Media for sporulation and tetrad analysis were prepared as described (17). All components of growth medium were purchased from Difco, Fisher, or Sigma. 2% agar was added for solid medium.

### Cloning of the *COQ6* Gene—The haploid strain SR128-3C containing the *coq6-1* allele was obtained from mating C128 and FY250 (Table I). SR128-3C yeast were grown to early-log phase in YPD medium and transformed (20) with the YCp50 centromeric plasmid library of *S. cerevisiae* genomic DNA (21) containing the *URA3* gene as a selectable marker. Transformants were selected on SD-Ura plates and after a 3-day incubation at 30 °C were replica-plated onto YPG plates to test for respiratory growth. Putative Q prototrophic transformants, able to grow on medium containing a nonfermentable carbon source, were further purified and tested for co-segregation of uracil prototrophy and

* This work was supported in part by National Institutes of Health Grants GM45952 (to C. F. C.) and HL2274 (to A. T.). The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
§ Present address: Dept. of Chemistry, University of Utah, 315 South 1400 E, Salt Lake City, UT 84112-0850.
∥ To whom correspondence should be addressed: Dept. of Chemistry and Biochemistry, University of California, Los Angeles, 607 Charles E. Young Dr. East, Los Angeles, CA 90095-1569. Tel.: 310-825-0771; Fax: 310-206-5213; E-mail: cathy@mbi.ucla.edu.
¹ The abbreviations used are: Q, ubiquinone or coenzyme Q; HHB, 3-hexaprenyl-4-hydroxybenzoic acid; ORF, open reading frame; HPLC, high performance liquid chromatography; HA, hemagglutinin.

Coq6 Is a Flavin-dependent Monooxygenase in Q Biosynthesis

![Diagram](attachment:diagram.png)

FIG. 1. The Q biosynthetic pathway in eukaryotes and prokaryotes. Coq1p (*S. cerevisiae*) or IspB (*E. coli*) assemble the polyisoprenyl diphosphate tail with 6 and 8 isoprene units, respectively. After formation of 3-polyprenyl-4-hydroxybenzoic acid (compound 1) by the 4-hydroxybenzoic acid:polyprenyltransferase (Coq2p or UbiA), the proposed biosynthetic pathways for Q in eukaryotes and in prokaryotes is thought to diverge as shown. In yeast, $n = 6$, and compound 1 is HHB. *E. coli* gene products are identified as *Ubi* (and also include IspB); *S. cerevisiae* gene products are identified as *Coq*.

TABLE I  
Genotypes and sources of *S. cerevisiae* strains

| Strain          | Genotype                                                                                   | Source or reference                     |
|-----------------|-------------------------------------------------------------------------------------------|-----------------------------------------|
| D273-10B/A1     | $\alpha$, met6                                                                            | Ref. 55                                |
| W303-1A         | a, ade2-2, his3-11, 15, leu2-3-115, trp1-1, ura3-1                                       | R. Rothstein$^a$                       |
| W303-1B         | $\alpha$, ade2-2, his3-11, 15, leu2-3-115, trp1-1, ura3-1                                | R. Rothstein$^a$                       |
| W303-1AB        | a/$\alpha$, ade2-2/ade2-2, his3-11, 15/his3-11, 15, leu2-3-115/leu2-3-115, trp1-1/trp1-1, ura3-1/ura3-1 | This study                             |
| CEN.PK2-1C      | a, ura3, his3, leu2, trp1                                                                  | Ref. 56                                |
| FY251           | a, ura3-52, his3$\Delta$200, leu2$\Delta$1, trp1$\Delta$63                               | F. Winston$^b$                         |
| JM6             | a, his4, $\rho^o$                                                                         | Ref. 57                                |
| JM8             | $\alpha$, ade1, $\rho^o$                                                                  | Ref. 57                                |
| C128            | $\alpha$, coq6-1, met                                                                     | Ref. 13                                |
| SR128-1A        | a, coq6-1, leu2$\Delta$1                                                                  | C128 × FY251                           |
| SR128-3C        | $\alpha$, coq6-1, his3$\Delta$200, trp1$\Delta$63, ura3-52                               | C128 × FY251                           |
| W303ACOQ6-1     | W303-1A, coq6::HIS3                                                                       | This study                             |
| $\alpha$W303$\Delta$COQ6-1 | W303-1B, coq6::HIS3                                                              | This study                             |
| SR$\Delta$COQ6-1 | SR128-3C, coq6::HIS3                                                                      | This study                             |
| W303$\Delta$COQ6-2 | W303-1A, coq6::LEU2                                                                     | This study                             |
| $\alpha$W303$\Delta$COQ6-2 | W303-1B, coq6::LEU2                                                              | This study                             |
| W303-1AB$\Delta$COQ6-2 | W303, coq6::LEU2                                                                     | This study                             |
| CEN$\Delta$COQ6-2 | CEN.PK2-1C, coq6::LEU2                                                                   | This study                             |
| W303$\Delta$COQ6–1:pSR1-1 | W303-1A, coq6::HIS3:pSR1-1                                                       | This study                             |

$^a$ Dr. Rodney Rothstein, Department of Human Genetics, Columbia University.  
$^b$ Dr. Fred Winston, C. Dollard, and S. Ricapero-Hovasse, Department of Genetics, Harvard University.

respiration competency following vegetative growth in rich medium. Such co-segregation was observed in two transformants, indicating that these traits were plasmid-linked. Yeast plasmid DNA was recovered from one transformant (494SR) and was amplified in DH5$\alpha$ *E. coli* (Invitrogen). The plasmid p494SR contained an insert of 4.1-kb and transformation of SR128-3C with p494SR restored growth on YPG medium. A similar cloning procedure was also performed with a multicopy expression library prepared from yeast genomic DNA in the vector YEp24 (22) and resulted in the isolation of pG63/T1, which was found to contain a 2.8-kb segment of DNA that overlapped with the insert present in p494SR (Fig. 2). Seven other rescuing yeast genomic

DNA clones were similarly isolated from a recombinant pUV1-based plasmid library (generous gift of Junichi Nikawa and Michael Wigler, Cold Spring Harbor Laboratory). Southern analysis showed that all of the clones contained overlapping DNA fragments (data not shown).

Subcloning and Disruption of the COQ6 Gene—A 3.8-kb HindIII fragment was isolated from p494SR and was ligated to the HindIII site of pRS316 (23) to generate pSR1-1. The insert of pSR1-1 was sequenced and found to contain two ORFs located on a segment of chromosome VII. Only one ORF, YGR255C, was also present on pG63/T1, and was designated COQ6 (deposited in GenBank™ with accession number AF003698).

To construct a disruption allele of COQ6, a 2.9-kb HindIII fragment,

25310

**Coq6 Is a Flavin-dependent Monooxygenase in Q Biosynthesis**

![Diagram](https://i.imgur.com/yourimageurl.png)

FIG. 2. Restriction map of isolated yeast genomic DNA containing the COQ6 gene and strategy of COQ6 deletion. The gray bars indicate the positions of the GND2 and COQ6 genes present on p494SR. Each of the plasmids depicted was able to restore growth of coq6 yeast mutant strains on YPG plate medium as identified by the + sign. A deletion construct was prepared by replacing a 414-bp BglII fragment (black bar) with the HIS3 gene on a BamHI fragment, as described under “Experimental Procedures.” Restriction sites are symbolized by either one or two letters and correspond to the following DNA restriction enzymes: B, BglII; H, HindIII; RI, EcoRI; RV, EcoRV.

containing about 350 bp of the YEp24 sequence and 2.55 kb of yeast genomic DNA, was excised from pG63/T1 and inserted into the HindIII site of YEp352 to generate the plasmid pG63/ST2. As shown in Fig. 2, the coq6 deletion plasmid was constructed by replacing a 414-bp BglII fragment in pG63/ST2 with a 1.7-kb BamHI fragment containing HIS3 (24). The resulting plasmid, pG63/ST3 was used to obtain a 4.0-kb EcoRI fragment with the disrupted gene. W303-1A, W303-1B, and SR128-3C were each transformed with 1 μg of DNA (25). Most of the histidine prototrophic clones issued from the transformations were respiration-defective and were complemented by the ρ° test strains JM6 or JM8 but not by the coq6-1 mutant strains, implying a genetic linkage between the coq6Δ::HIS3 and the coq6-1 alleles. Nuclear DNA from three independent transformants (one from each of the three separate parental strains) was digested with EcoRI and EcoRV, separated on 1% agarose, and transferred to a nitrocellulose membrane. The probe, a 1.1-kb EcoRI/EcoRV fragment within the COQ6 ORF, recognizes a 1.1-kb fragment in the genomic DNA of the parental strain. The genomic DNA of the mutant strains (W303ΔCOQ6-1, αW303ΔCOQ6-1, and SRΔCOQ6-1) contained a larger hybridizing species at ~3.15 kb, as expected for the disrupted allele. To verify the allelism between coq6Δ and coq6-1 mutations, SRΔCOQ6-1 was mated to wild type strain W303-1A to form diploid cells, which were then sporulated to produce meiotic progeny. Spores from 14 tetrads showed 2:2 segregation for respiration competence and histidine auxotrophy. In all cases, the spores competent to grow on glycerol medium were histidine auxotrophs, whereas the spores unable to grow on glycerol were histidine prototrophs, confirming the allelism between the cloned COQ6 gene and the original coq6-1 mutation.

**Complete Deletion of the COQ6 ORF—**A complete deletion of the COQ6 open reading frame was performed with a PCR-targeting strategy (26). A 1.53-kb fragment containing LEU2 was generated from the YEp13 vector (GenBank™ accession number U03498) using forward primer pPG6DLF 5′-ATAATTCTTAAAAAGTGAGCTAGTCTATTTCT-ATTTACATACCTCATTTTGTAATTTCGTGTCG-3′ and reverse primer pPG6DLR 5′-TCAAATTGGTCTTTTCAGTGAACCTTGTATCGATTGA-CACAGAGGCAGAGGTCGCCTGACGCATA-3′. The 5′ end of the forward primer contained 45 nucleotides corresponding to −45 to −1 upstream of the COQ6 ORF and at the 3′ end 19 nucleotides corresponding to nucleotides 5579–5597 of YEp13. The reverse primer similarly corresponded to 45 nucleotides from +1485 to +1441 of the reverse strand of the COQ6 ORF followed by 19 residues from 7018 to 7000 of the reverse strand of YEp13. A W303-1AB diploid was created by mating W3031A with W3031B. W3031A, W3031B, CEN.PK2-1C, and W303-1AB cells were transformed with 1 μg of the fragment using the lithium-acetate method as described (27). Transformed cells were grown on SD−Leu plates at 30 °C for 2 days. Gene disruption was verified by PCR using forward primer Coq6SeqF1 5′-ACCTTTGCAT-TACAAGTGCAACGCCTCTACC-3′ and reverse primer Coq6SeqR1 5′-GGTGACGCGGTATCCGCCCCGCTTTTGG-3′ and produced a product of 1.84-kb for both the wild type and disrupted strains. The PCR product was then digested using the restriction enzyme EcoRI, which digests the wild type product and produces two products of 0.44 and 1.4 kb, whereas the EcoRI digestion fragments of the disrupted product are 1.2 and 0.64 kb. Tetrad analysis was performed as described above.

**In Vivo Radiolabeling of Q<sub>6</sub> Intermediates, Lipid Extraction, and Analysis by HPLC—**Yeast strains SR128-3C and W303ΔCOQ6-1 were grown in 1 liter of SDC medium supplemented with 0.65 μCi of 4-[U-<sup>14</sup>C]hydroxybenzoic acid (365 Ci/mol), synthesized from L-[U-<sup>14</sup>C]tyrosine by alkali heat fusion (28). Cells the were harvested after 3 days of incubation at 30 °C with shaking (200 rpm, A<sub>600 nm</sub> = 10), and the lipids were extracted as described (29). The lipid extracts were concentrated, the volume was adjusted to 1 ml with hexane, and 0.10-ml aliquots (5.0–12.0 × 10<sup>3</sup> cpm) were analyzed by normal phase HPLC employing a cyanopropyl column (Zorbax, 5 μm, 4.6 × 250 mm; Mac-Mod Analytical, Chadds Ford, PA) as described (30). The column was equilibrated with a mobile phase composed of 98% solvent A (hexane) and 2% solvent B (isopropanol:hexane:methylene chloride, 52:41:5:2) at a flow rate of 1 ml/min. Ten minutes after sample injection, the percentage of solvent B was increased linearly from 2 to 27% in 25 min (35 min from the start) and then from 27 to 100% in 20 min (55 min after the start). In the next 5 min, the percentage of solvent B was decreased linearly from 100 to 2% and remained at 2% for 30 min to equilibrate the column before next sample injection. The radioactivity of 1-ml fractions was measured by scintillation counting in 10 ml of BIOSafe nonaqueous scintillation mixture (Research Products International) with a Beckman scintillation counter (model LS-3133P). The efficiency of <sup>14</sup>C detection was 90%.

**Determination of Q<sub>6</sub> Content in Lipid Extracts by HPLC and Electrochemical Detection—**Lipid extraction and analysis were performed as described previously (31). Yeast strains W303-1A, SR128-3C, W303ΔCOQ6-1, and W303ΔCOQ6-1:pSR1-1 (the coq6-null strain harboring a COQ6 containing plasmid) were grown at 30 °C with shaking (200 rpm) in 50 ml of YPGal to an A<sub>600 nm</sub> of about 4 or alternatively were grown to saturation (2 days). 750 ng of Q<sub>9</sub> was added as an internal standard to a yeast cell pellet (100 mg of wet weight), and the cells were lysed by vortexing with 1 g of glass beads in 0.35 ml of water.

**Mitochondrial Import Assay—**An in vitro transcription template plasmid was constructed by inserting the COQ6 ORF into the pRS426 vector downstream of the T7 promoter (32). The COQ6 ORF with SalI and NotI linkers at 5′ and 3′ ends, respectively, was PCR-amplified with the template pSR1-1, 5′ primer JF3 (5′-ACGCACGCGTGCACAT-GTTCTTTTCAAAAAGTTATGC-3′), and 3′ primer JF4–1 (5′-ATAAGA-ATGCGGCCGCTCTCATTTCTCATTTCCTCC-3′) using Vent DNA polymerase (New England Biolabs). The PCR product was digested with SalI and NotI and inserted into the corresponding SalI and NotI sites in pRS426. The resulting product, pT7Q6, was linearized with XhoI and provided template DNA for in vitro transcription (Promega Ribomax large scale RNA production system). The resulting mRNA was then translated with Promega Flexi rabbit reticulocyte lysate system in the presence of [35S]methionine from Amersham Biosciences (1000 Ci/mmol, at a final concentration of 0.75 μM). Both the mRNA and the <sup>35</sup>S-labeled polypeptides were stored at –80 °C. The isolation of mito-

chondria (from D273-10B/A1) and import reaction conditions were performed according to Yaffe (33) as described previously (34). The cell cultures (1 liter) were grown in semisynthetic lactate medium to saturation density. The spheroplasts were prepared and lysed by Dounce homogenization with a tight fitting pestle as described (19). Purified mitochondria were isolated from a linear Nycodenz gradient as described (19). Each import reaction contained 6 μl of radiolabeled *in vitro* translated product and isolated mitochondria (200 μg of protein). Following the 30-min incubation at 30 °C, the mitochondria were reisolated and washed once as described (33). Proteinase K treatment after the import was performed by adding proteinase K at final concentration of 50 μg/ml to resuspended mitochondria. The proteolytic digest was allowed to proceed at 0 °C for 20 min and was terminated by the addition of phenylmethylsulfonyl fluoride to a final concentration of 1 mM.

**Plasmid Construction of HA-tagged COQ6**—Two yeast expression plasmids, one single copy and one multicopy, were constructed to express the Coq6 polypeptide containing a carboxyl-terminal peptide (MYPYDVDPDYASLDGPMST) corresponding to the carboxyl terminus of the influenza HA viral protein, an epitope for the 12CA5 monoclonal antibody (35). The COQ6 ORF region without the stop codon was PCR-amplified using 5′ primer JF3 (5′-ACGCACGCGTGCACATGT-TCTTTCAAAGTTATGC-3′) and 3′ primer JF4 (5′-ATAAGAATGCCGGCACGTTTCTCATTTCCCTCCTAATGTG-3′). The PCR product was directionally cloned into the multicopy pADCL vector (*SalI* and *NotI* at 5′ and 3′, respectively) (36), to generate pHA6. pSHA6, a single copy version of COQ6-HA construct, was generated by removing the 3.6-kb transcriptional cassette from pHA6 by *BamHI* partial digestion and insertion into *BamHI* site of the vector pRS316 (23). The yeast cells were transformed with pHA6, pSHA6, or pSR1-1 (25). Transformants were selected for the presence of either the *LEU2* (pHA6) or *URA3* (pSHA6) gene on SD-Leu or SD-Ura plate media. Colonies obtained on the respective plate medium were subsequently replica-plated to YPG plate medium.

**Generation of Antisera against Coq6p**—The 1.44-kb COQ6 ORF was PCR-amplified with forward primer pPG6F 5′-GCGGATCCGATGT-TCTTTCAAAGTTATGCTT-3′ and a reverse primer pPG6R 5′-GCG-GATCCTCATTTCTCATTTCCCTCCTAATGTG-3′ and Vent DNA polymerase. The product was then digested with *BamHI* and inserted into PET15b (Novagen) at the *BamHI* site to generate a fusion protein containing a His₆ tag at the amino terminus. The fusion protein was overexpressed in the *E. coli* BL21(DE3) under induction by 1 mM isopropyl-β-D-thiogalactopyranoside and was purified over His-Bind resin (Novagen) and used to generate antiserum in rabbits (Cocalico).

**Mitochondrial Localization of Coq6p**—Yeast cultures (W303-1A) were grown in YPGal to an A₆₀₀ₙₘ between 2 and 4, and mitochondria were isolated and purified as described above. Mitoplasts were generated by hypotonic treatment of mitochondria (19). Mitochondria (1 mg of protein) were suspended in five volumes of 20 mM HEPES-KOH, pH 7.4, and incubated on ice for 20 min. The mixture was then sedimented in a microcentrifuge for 10 min at 4 °C to separate the intermembrane space components (supernatant) and the mitoplasts (pellet). The mitoplasts were then sonicated and centrifuged at 150,000 × g for 60 min at 4 °C to generate matrix (supernatant) and membrane (pellet) fractions. Alternatively, the mitoplasts were alkaline-extracted by incubating with 0.1 M Na₂CO₃, pH 11.5, for 30 min on ice, followed by centrifugation at 150,000 × g for 60 min at 4 °C to separate the integral membrane components (pellet) from the peripheral membrane and matrix components (supernatant) (37). Proteinase K protection experiments were carried out as described (38). The samples were analyzed by immunoblot.

**Immunoblot Analysis**—The fractions were assayed for protein concentration by the bicinchoninic acid assay (Pierce). Equal amounts of protein from the mitochondrial fractions of cells were analyzed by electrophoresis on 12% Tris/glycine gels and were subsequently transferred to Hybond ECL Nitrocellulose (Amersham Biosciences). Immunoblot analysis and treatment of membranes for reuse with another antisera were performed as described by Amersham Biosciences. An exception to the stated protocol was the use of washing buffer: 1× phosphate-buffered saline, 0.1% Tween 20. Primary antibodies were used at the following concentrations: anti-Coq6p, 1:500; anti-Coq3p, 1:1000; anti-Coq4p, 1:1000; anti-cytochrome *b*₂, 1:1000; anti-Hsp60p, 1:10,000; anti-β subunit of F₁-ATPase, 1:10,000; anti-cytochrome *c*₁, 1:1000; anti-OM45p, 1:1000; and anti-Mas2p, 1:1000. Goat anti-rabbit secondary antibodies conjugated to horseradish peroxidase (Calbiochem) were used at a 1:10,000 dilution.

25312 Coq6 Is a Flavin-dependent Monooxygenase in Q Biosynthesis


|  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |  |
Coq6 Is a Flavin-dependent Monooxygenase in Q Biosynthesis 25313

Region 1: ADP Binding Fingerprint

| 30 | V | L | I | V | G | G | P | A | G | L | T | L | A | A | S | I | K | N | S | P | Coq6 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 3 | V | I | I | V | G | G | G | M | A | G | A | T | L | A | L | A | I | S | R | L | S | UbiH |
| 5 | V | A | I | I | G | S | G | P | S | G | L | L | L | G | Q | L | L | T | E | A | G | PobARh |
| 10 | V | L | I | V | G | A | G | P | A | G | L | M | A | A | R | V | L | S | E | Y | V | PhyATc |
| 8 | V | A | V | I | G | G | G | I | A | G | T | A | L | A | L | G | L | S | K | S | S | ShPs |
| 17 | V | L | I | V | G | G | G | P | T | G | L | I | A | A | N | E | L | L | R | R | G | PcpB |
| 5 | V | A | I | I | I | G | A | G | P | S | G | L | L | L | G | Q | L | L | H | K | A | G | PobAPs |
| 8 | V | A | I | I | G | S | G | P | A | G | L | L | L | G | Q | L | L | Y | K | A | G | PobAAc |
| 10 | V | L | V | V | G | G | G | L | G | G | L | S | T | A | L | F | L | A | R | R | G | DnrF |
| 38 | V | L | I | V | G | S | G | P | A | G | S | S | A | A | M | F | L | S | T | Q | G | PheA |

Δ Δ Δ Δ

Region 2: NAD(P)H / FAD Recognition

| 195 | R | L | L | V | G | A | D | G | F | N | S | P | T | R | Coq6 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 156 | R | V | L | V | A | A | D | G | T | H | S | A | L | A | UbiH |
| 153 | D | F | I | A | G | C | D | G | F | H | G | A | S | R | PobARh |
| 219 | K | Y | V | I | G | C | D | G | G | H | S | W | V | R | PhyATc |
| 154 | D | V | A | I | A | A | D | G | I | K | S | S | M | R | ShPs |
| 167 | R | W | V | I | G | A | D | G | V | R | S | R | V | R | PcpB |
| 153 | D | Y | I | A | G | C | D | G | F | H | G | I | S | R | PobAPs |
| 155 | D | F | I | A | G | C | D | G | Y | H | G | V | C | R | PobAAc |
| 173 | R | Y | L | V | A | A | D | G | P | R | S | A | I | R | DnrF |
| 201 | K | Y | L | I | G | A | D | G | A | R | S | K | V | A | PheA |

Region 3: Ribityl Binding Motif

| 367 | D | R | V | A | L | V | G | D | A | A | H | T | T | H | P | L | A | G | Q | G | L | N | M | G | Q | T | D | V | H | G | L | V | Y | A | L | Coq6 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| 279 | H | R | T | V | L | V | G | N | A | A | Q | T | L | H | P | I | A | G | Q | G | F | N | L | G | M | R | D | V | M | S | L | A | E | T | L | UbiH |
| 279 | G | R | L | F | L | V | G | D | A | A | H | I | V | P | P | T | G | A | K | G | L | N | L | A | A | S | D | V | H | Y | L | F | S | G | L | PobARh |
| 350 | E | R | V | F | I | A | G | D | A | C | H | T | H | S | P | K | A | G | Q | G | M | N | T | S | M | M | D | T | Y | N | L | G | W | K | L | PhyATc |
| 305 | G | R | V | A | L | I | G | D | A | A | H | A | M | L | P | H | Q | G | A | G | A | G | Q | G | L | E | D | A | Y | F | M | A | E | L | L | ShPs |
| 291 | G | N | V | F | L | A | G | D | A | A | H | C | H | S | P | S | G | G | S | G | M | N | V | G | M | Q | D | A | F | N | L | G | W | K | I | PcpB |
| 279 | G | R | L | F | L | A | G | D | A | A | H | I | V | P | P | T | G | A | K | G | L | N | L | A | A | S | D | V | S | T | L | Y | R | L | L | PobAPs |
| 281 | G | K | L | F | L | A | G | D | A | A | H | I | V | P | P | T | G | A | K | G | L | N | L | A | A | S | D | I | A | Y | L | S | S | A | L | PobAAc |
| 301 | G | P | V | L | L | V | G | D | A | A | K | V | T | P | P | T | G | G | M | G | G | N | T | A | I | G | H | G | F | D | V | A | W | K | L | DnrF |
| 322 | G | R | V | C | C | A | G | D | A | I | H | K | H | P | P | S | H | G | L | G | S | N | T | S | I | Q | D | S | Y | N | L | C | W | K | L | PheA |

FIG. 5. Comparison of regions of the predicted Coq6 amino acid sequence with conserved sequences found in FAD binding aromatic hydroxylases. Alignments were created on DNASTAR™ Megalign with the Clustal method and the PAM 250 residue weight table. The Δ symbol represents amino acid residues Ala, Ile, Leu, Val, Met, or Cys that occur as part of the ADP binding fingerprint of Region 1 (47). Region 2 is implicated in the recognition of NADH or NADPH and is also involved indirectly in binding the pyrophosphate moiety of FAD (48). Region 3 contains a consensus for binding to the ribityl moiety of FAD (49). The aromatic hydroxylases are designated as UbiH, 2-octaprenyl-6-methoxyphenol hydroxylase from *E. coli* (GenBank™ accession number P25534) (43); PobARh, 4-hydroxybenzoate hydroxylase (PobA) from *Rhizobium leguminosarum* (GenBank™ accession number AAA73519) (58); PhyATc, phenol hydroxylase from *Trichosporon cutaneum* (GenBank™ accession number AAA34202) (59); ShPs, salicylate hydroxylase from *Pseudomonas putida* (GenBank™ accession number d1011754) (60); PcpB, pentachlorophenol 4-monooxygenase (pcpB) from *Flavobacterium sp.* (GenBank™ accession number AAF15368) (61); PobAPS, p-hydroxybenzoate hydroxylase from *Pseudomonas fluorescens* (GenBank™ accession number WHPSBF) (62); PobAAc, p-hydroxybenzoate hydroxylase (pobA) from *Acinetobacter calcoaceticus* (GenBank™ accession number AAC37163) (63); DnrF, 11-aklavinone hydroxylase from *Streptomyces peucetius* (GenBank™ accession number AAC43342) (64); and PheA, phenol monooxygenase from *Pseudomonas sp.* (GenBank™ accession number AAC64901) (65).

of yeast, respectively, when grown to log phase, and were 189 and 169 pmol/mg of wet weight of yeast, respectively, when grown to the stationary phase (data not shown).

**Coq6p May Act as a Flavin-dependent Monooxygenase to Catalyze Quinone Formation—**The predicted amino acid sequence of the COQ6 ORF revealed 21 and 24% sequence identity with the respective *E. coli* UbiH and UbiF polypeptides (Fig. 4). *E. coli ubiH* mutants lack Q₈ and accumulate compound 6, indicating that UbiH is required for the monooxygenase step that catalyzes quinone formation (Fig. 1 and Ref. 42). The *ubiH* gene product was identified as a flavin-dependent monooxygenase (43). The *E. coli ubiF* gene product has also been identified as a flavin-dependent monooxygenase, with 31% amino acid sequence identity to UbiH (44). *E. coli ubiH* and *ubiF* mutants lack Q₈, accumulate compound 8, and both *ubiH* and *ubiF* mutants fail to grow on media containing succinate (Fig. 1 and Ref. 45). Coq6p, UbiH, UbiF, and other eukaryotic homologs of Coq6 each contain three regions with amino acid

sequence identities that are present in a large family of flavin-dependent monooxygenases (Fig. 5 and Ref. 46). Region 1 contains an ADP-binding fingerprint (47), Region 2 is implicated in the recognition of NAD(P)H and may also be involved indirectly in binding the pyrophosphate moiety of FAD (48), whereas Region 3 contains a consensus sequence for binding to the ribityl moiety of FAD (49). Based on the homology with UbiH and UbiF and the presence of the conserved motifs found in other aromatic flavin-dependent monooxygenases, it seems likely that Coq6p functions in one or more hydroxylation steps in Q biosynthesis.

**Mitochondrial Import of Coq6p—**The amino-terminal sequence predicted for Coq6p showed characteristics of mitochondrial targeting sequences (14). The first 28 amino acid residues are abundant in positively charged amino acid residues and devoid of acidic residues. Arrangement of these 28 residues in a helical wheel shows the positively charged residues are located along one side of the α-helix. The amino-terminal region

25314

**Coq6 Is a Flavin-dependent Monooxygenase in Q Biosynthesis**

|  | 1 | 2 | 3 | 4 | 5 | 6 |
| --- | --- | --- | --- | --- | --- | --- |
| Mitochondria | - | + | + | + | + | + |
| Proteinase K | - | - | + | + | + | - |
| Triton X-100 | - | - | - | + | - | - |
| Valinomycin | - | - | - | - | + | + |

P M

---

FIG. 6. *In vitro* mitochondrial import and proteolytic processing of Coq6. Lane 1, 10% of the radiolabeled *in vitro* translation product used in each import reaction. Lane 2, *in vitro* translation product incubated with isolated mitochondria. Lane 3, same as lane 2 but including a post-import proteinase K treatment. Lane 4, same as lane 3 except Triton X-100 was added to disrupt the mitochondria before proteinase K treatment. Lanes 5 and 6, *in vitro* translation product was incubated with mitochondria in the presence of the uncoupler valinomycin (final concentration, 10 μg/ml). After the incubation, the reisolated mitochondria were either treated with proteinase K as in lane 5 or were directly analyzed as in lane 6. The samples were mixed with sample buffer and heated at 100 °C for 5 min prior to analysis by SDS-polyacrylamide electrophoresis (12% polyacrylamide) and autoradiography. $P$ and $M$ indicate the positions of the precursor form and the mature form of Coq6p, respectively.

contains the sequence motif common in many mitochondrial matrix proteins that are proteolytically cleaved twice once imported into matrix, as characterized by an arginine at –10, a hydrophobic residue (phenylalanine, leucine, or isoleucine) at –8, and serine, threonine, or glycine at –5 relative to the amino-terminal residue of the proteolytically processed protein. To determine whether Coq6p is a mitochondrial protein, an *in vitro* mitochondrial import assay of the Coq6 polypeptide was performed. Upon incubation with mitochondria prepared from wild type yeast, the *in vitro* translated Coq6p was cleaved, and the resulting mature form of the protein was observed to have a mass of about 51 kDa, consistent with lysine 21 as the putative cleavage site (Fig. 6). This mature protein was protected and resistant to exogenous proteinase K treatment (Fig. 6, lane 3). Disruption of mitochondrial membrane with detergent exposed the imported and processed Coq6p, which became accessible to the proteinase K (Fig. 6, lane 4). The import of Coq6p also required a mitochondrial membrane potential, because the addition of valinomycin inhibited import of Coq6p (Fig. 6, lanes 5 and 6).

**Mitochondrial Localization of Coq6p—**Our initial investigation of the subcellular localization of Coq6p made use of a carboxyl-terminal fusion protein between Coq6p and the HA epitope peptide (see “Experimental Procedures”). The Coq6-HA epitope fusion protein retained activity as assayed by the ability of either the single- or multi-copy plasmid construct to rescue *coq6*-null mutant yeast strains for growth on medium containing a nonfermentable carbon source (YPG plates; data not shown). However, subsequent subcellular and submitochondrial fractionation analysis revealed that the Coq6-HA tagged construct was present in mitochondria as an insoluble aggregate because it remained in the pellet following treatment with 1% Triton X-100 and 1 M NaCl. For this reason, antibodies were generated to the Coq6 polypeptide.

A polyclonal antibody was generated in rabbit against a Coq6-His-tagged fusion protein. This antibody recognized a polypeptide of 51 kDa in wild type yeast cell extracts and also recognized the Coq6 HA-tagged polypeptide that migrates at about 57 kDa (Fig. 7a). No polypeptide at these molecular weights was detected in cell extracts of the *coq6* deletion strain, indicating that the antibody is specific for Coq6p. Western blot analysis of subcellular yeast fractions revealed that Coq6p co-fractionated with the mitochondria along with Coq4p,

FIG. 7. Coq6p is peripherally associated with the inner membrane of mitochondria. $a$, the antiserum generated against Coq6p and its specificity was tested against the wild type strain W303-1A (WT), the *coq6* deletion strain W303-1AΔCOQ6-1 (Δ), and the multicopy HA-tagged rescued strain W303-1AΔCOQ6-1:pHA6-1 (MC). $b$, whole cell yeast was homogenized and separated by differential centrifugation into nuclear ($P1$), crude mitochondrial ($CM$), Nycodenz gradient-purified mitochondrial ($NM$), and post-mitochondrial supernatant fractions ($PS$). $c$, Nycodenz-purified mitochondria ($NM$) were subjected to hypotonic swelling and centrifuged to separate intermembrane space proteins in the supernatant ($IMS$). The pellet was then sonicated and centrifuged to release a soluble matrix protein fraction in the supernatant ($S$) and membrane protein fraction pellet ($P$). $d$, intact mitochondria or mitoplasts generated by hypotonic swelling were treated with proteinase K (100 μg/ml) for 30 min with or without 1% Triton X-100. $e$, mitoplasts were incubated with 0.1 M $Na_2CO_3$, pH 11.5, on ice 30 min. Centrifugation produced soluble ($S$) and insoluble ($P$) fractions, which were compared against Nycodenz-purified mitochondria ($NM$). All of the samples were separated by SDS-PAGE and analyzed by immunoblotting as described under “Experimental Procedures.” The antiserum was used against Coq6p and compared with results of antisera against outer membrane protein OM45p, intermembrane space protein cytochrome $b_2$ ($Cyt b_2$), integral inner membrane protein cytochrome $c_1$ ($Cyt c_1$), peripheral inner membrane proteins Coq3p, Coq4p, and the $\beta$ subunit of $F_1$-ATPase ($F_1\beta$), and matrix proteins Hsp60 and Mas2.

To determine whether Coq6p is a membrane-bound or soluble protein, purified mitochondria were osmotically shocked to disrupt the outer membrane and release components of the intermembrane space. Mitoplasts were then further disrupted by sonication to separate the remaining soluble and membrane components. Fig. 7c shows that Coq6p localizes to the membrane (pellet) fraction. To determine whether Coq6p resides in the inner or outer membrane, mitochondria were treated with combinations of proteinase K, Triton X-100, and hypotonic swelling. Fig. 7d shows that Coq6p is degraded only when the detergent is present, as is the case for two matrix proteins, Hsp60p and Mas2p. Finally, to determine whether Coq6p is a peripheral or integral protein in the inner mitochondrial membrane, mitoplasts were treated with 0.1 M sodium carbonate buffer, pH 11.5. Fig. 7e shows that following this treatment Coq6p is released into the supernatant, as are the $\beta$ subunit of $F_1$-ATPase and Coq4p (two peripheral membrane proteins), whereas cytochrome $c_1$ (an integral membrane protein) is not.

DISCUSSION

This work characterizes yeast *coq6* mutants and the isolation of the *COQ6* gene. *COQ6* is necessary for Q biosynthesis. In the original *coq6* mutant and in both *coq6* partial and complete deletion mutants, $Q_{6}$ is undetectable, and as a result, these cells are rendered respiratory incompetent. Based on the data presented here, *COQ6* encodes a mitochondrial protein necessary for Q biosynthesis. Similar to Coq2p, Coq3p, Coq4p, Coq5p, Coq7p, and Coq8p/Abc1p, the Coq6 polypeptide is imported into mitochondria (14). In agreement with submitochondrial localization studies of Coq1p, Coq3p, Coq4p, Coq5p, and Coq7p, Coq6p is localized to the matrix side of the inner membrane (18, 40, 50).²

*S. cerevisiae* is a facultative anaerobe capable of fermentation and aerobic respiration, and none of the other *COQ* genes have been identified as being essential. Complete deletions of the *COQ6* gene were introduced into both haploid and diploid strains of W303 and also into CEN.PK2-1C, a haploid strain similar in genetic background to the diploid used by Fiori *et al.* (16). In contrast to their results, all of the yeast strains were viable when *COQ6* was deleted, indicating that none of the eight *COQ* genes is essential. Q itself is nonessential for growth in this organism.

The function of Coq6p is still unknown, as are the functions of Coq4p and Coq8p. Based on sequence homology to known flavin-dependent monooxygenases, it is likely that Coq6p acts similarly. Although Coq6p and UbiF share 24% sequence identity, it seems unlikely that Coq6p functions like UbiF in the hydroxylation of compound 8 in yeast, because previous studies have shown that yeast Coq7p is required for this hydroxylation step (29). The Coq7 polypeptide has been identified as a di-iron carboxylate protein, a member of a monooxygenase family distinct from UbiF (51). Although *E. coli* lack a homolog of Coq7p, homologs of yeast Coq7p from other prokaryotic species rescue the *E. coli ubiF* hydroxylase mutants, indicating that these two distinct types of monooxygenases each catalyze the same reaction in Q biosynthesis (51). Currently, two of the uncharacterized steps in Q biosynthesis are hydroxylations, and Coq6p may be responsible for either or both of these steps. Coq6p has 21% identity to the *E. coli* UbiH, which converts compound 6 to compound 7. Of course, other hydroxylation substrates for Coq6p are also possible, including the hydroxylation of HHB to compound 4 (Fig. 1). Indeed, because *coq6-1* and *coq6-null* mutants both accumulate HHB (Fig. 3), it seems attractive to postulate its action at this step. However, the accumulation of HHB is not diagnostic of the blocked step, because so many other *coq* mutants also accumulate HHB (15). Another possibility is that Coq6p catalyzes the oxidative decarboxylation of compound 5 to compound 7. It has been demonstrated that 4-hydroxybenzoate 1-hydroxylase oxidatively decarboxylates 4-hydroxybenzoic acid with the formation of hydroquinone (52). Salicylate hydroxylase similarly oxidatively decarboxylates salicylate to catechol (53). However, in yeast this is less likely given that previous work showed a Q-deficient yeast mutant accumulated compound 6 (54). That mutant was never sequenced, nor is it available. Identification of the role of Coq6p in the hydroxylation step(s) of Q biosynthesis will require development of *in vitro* assays for these hydroxylation steps.

Blast analysis of Coq6p against other higher eukaryotic genomes revealed that only one homolog existed for *Homo sapiens*, *Mus musculus*, and *C. elegans*, indicating a conserved evolution for this protein (Fig. 4). For *Drosophila melanogaster*, two matches were found, but only one had a high score and spanned the entire sequence. The homologs in each case retained the three conserved motifs depicted in Fig. 5.

Genetic data have suggested that a putative complex of these Coq polypeptides is responsible for Q biosynthesis because a deletion in any one of the eight *COQ* genes results in a profound decrease in steady state protein levels for Coq3p, Coq4p, and Coq7p (34).² Furthermore, null mutations in *coq3*, *coq4*, *coq5*, *coq7*, or *coq8/abc1* result in the accumulation of HHB (15). Similarly, analyses of lipid extracts of *coq6* mutants in the studies presented here show that these mutants also accumulate HHB. It has been observed that an E226K *coq4* point mutant did allow for the steady state expression of Coq3p and Coq7p (40). In addition, a G104D *coq7* point mutant allowed for the accumulation of 2-hexaprenyl-3-methyl-6-methoxy-1,4-benzoquinone (Fig. 1, demethoxy-Q or compound 8) (29). It would appear that both the *coq4* and *coq7* amino acid substitution mutants allow for the expression of correctly folded but inactive Coq4p and Coq7p, respectively, and each may be interacting and stabilizing the steady state expression of the other Coq polypeptides in a complex. The accumulation of HHB in both the *coq6* mutants further suggests that Coq6p similarly is required for stabilization of a complex. Additionally, the *coq6-1* mutant has been found to be a nonsense mutation of the tyrosine at position 218.³ This mutant would be unable to produce a stable polypeptide at full length and would explain why we see an accumulation of HHB.

REFERENCES

1. Brandt, U., and Trumpower, B. (1994) *Crit. Rev. Biochem. Mol. Biol.* **29**, 165–197
2. Dutton, P. L., Ohnishi, T., Darrouzet, E., Leonard, M. A., Sharp, R. E., Gibney, B. R., Daldal, F., and Moser, C. C. (2000) in *Coenzyme Q: Molecular Mechanisms in Health and Disease* (Kagan, V. E., and Quinn, P. J., eds) pp. 65–82, CRC Press, Boca Raton, FL
3. Santos-Ocaña, C., Do, T. Q., Padilla, S., Navas, P., and Clarke, C. F. (2002) *J. Biol. Chem.* **277**, 10973–10981
4. Gille, L., and Nohl, H. (2000) *Arch. Biochem. Biophys.* **375**, 347–354
5. Santos-Ocana, C., Cordoba, F., Crane, F. L., Clarke, C. F., and Navas, P. (1998) *J. Biol. Chem.* **273**, 8099–8105
6. Kagan, V. E., Nohl, H., and Quinn, P. J. (1996) in *Handbook of Antioxidants* (Cadenas, E., and Packer, L., eds) pp. 157–201, Dekker, New York
7. Georgellis, D., Kwon, O., and Lin, E. C. (2001) *Science* **292**, 2314–2316
8. Larsen, P. L., and Clarke, C. F. (2002) *Science* **295**, 120–123
9. Jonassen, T., Marbois, B. N., Faull, K. F., Clarke, C. F., and Larsen, P. L. (2002) *J. Biol. Chem.* **277**, 45020–45027
10. Geromel, V., Darin, N., Chretien, D., Benit, P., DeLonlay, P., Rotig, A., Munnich, A., and Rustin, P. (2002) *Mol. Genet. Metab.* **77**, 21–30
11. Shults, C. W., Oakes, D., Kieburtz, K., Beal, M. F., Haas, R., Plumb, S., Juncos, J. L., Nutt, J., Shoulson, I., Carter, J., Kompoliti, K., Perlmutter, J. S., Reich, S., Stern, M., Watts, R. L., Kurlan, R., Molho, E., Harrison, M., and Lew, M. (2002) *Arch. Neurol.* **59**, 1541–1550
12. Meganathan, R. (2001) *FEMS Microbiol. Lett.* **203**, 131–139
13. Tzagoloff, A., and Dieckmann, C. L. (1990) *Microbiol. Rev.* **54**, 211–225
14. Jonassen, T., and Clarke, C. F. (2001) in *Coenzyme Q: Molecular Mechanisms in Health and Disease* (Kagan, V. E., and Quinn, P. J., eds) pp. 185–208, CRC Press, Boca Raton, FL
15. Poon, W. W., Do, T. Q., Marbois, B. N., and Clarke, C. F. (1997) *Mol. Aspects Med.* **18**, s121–s127
16. Fiori, A., Bianchi, M. M., Fabiani, L., Falcone, C., Francisci, S., Palleschi, C., Solimando, N., Uccelletti, D., and Frontali, L. (2000) *Yeast* **16**, 377–386
17. Adams, A., Gottschling, D. E., Kaiser, C. A., and Stearns, T. (1997) *Methods in Yeast Genetics*, pp. 145–154, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY
18. Poon, W. W., Barkovich, R. J., Hsu, A. Y., Frankel, A., Lee, P. T., Shepherd, J. N., Myles, D. C., and Clarke, C. F. (1999) *J. Biol. Chem.* **274**, 21665–21672
19. Glick, B. S., and Pon, L. A. (1995) *Methods Enzymol.* **260**, 213–223
20. Marbois, B. N., Hsu, A., Pillai, R., Colicelli, J., and Clarke, C. F. (1994) *Gene (Amst.)* **138**, 213–217
21. Rose, M. D., Novick, P., Thomas, J. H., Botstein, D., and Fink, G. R. (1987) *Gene (Amst.)* **60**, 237–243
22. Carlson, M., and Botstein, D. (1982) *Cell* **28**, 145–154
23. Sikorski, R. S., and Hieter, P. (1989) *Genetics* **122**, 19–27
24. Rothstein, R. J. (1983) *Methods Enzymol.* **101**, 202–211
25. Elble, R. (1992) *BioTechniques* **13**, 18–20
26. Baudin, A, Ozier-Kalogeropoulos, O., Denouel, A., Lacroute, F., and Cullin, C. (1993) *Nucleic Acids Res.* **21**, 3329–3330
27. Gietz, R., Schiestl, R., Willems, A., and Woods, R. (1995) *Yeast* **11**, 355–360
28. Clarke, C. F., Williams, W., and Teruya, J. H. (1991) *J. Biol. Chem.* **266**, 16636–16644
29. Marbois, B. N., and Clarke, C. F. (1996) *J. Biol. Chem.* **271**, 2995–3004

² P. Gin and C. F. Clarke, unpublished data.
³ E. Hsieh and C. F. Clarke, unpublished data.

25316 Coq6 Is a Flavin-dependent Monooxygenase in Q Biosynthesis

30. Poon, W. W., Marbois, B. N., Faull, K. F., and Clarke, C. F. (1995) *Arch. Biochem. Biophys.* **320**, 305–314

31. Jonassen, T., and Clarke, C. F. (2000) *J. Biol. Chem.* **275**, 12381–12387

32. Christianson, T. W., Sikorski, R. S., Dante, M., Shero, J. H., and Hieter, P. (1992) *Gene (Amst.)* **110**, 119–122

33. Yaffe, M. P. (1991) *Methods Enzymol.* **194**, 627–643

34. Hsu, A. Y., Poon, W. W., Shepherd, J. A., Myles, D. A., and Clarke, C. F. (1996) *Biochemistry* **35**, 9797–9806

35. Niman, H. L., Houghten, R. A., Walker, L. E., Reisfeld, R. A., Wilson, I. A., Hogle, J. M., and Lerner, R. A. (1983) *Proc. Natl. Acad. Sci. U. S. A.* **80**, 4949–4953

36. Spain, B. H., Koo, D., Ramakrishnan, M., Dzudzor, B., and Colicelli, J. (1995) *J. Biol. Chem.* **270**, 25435–25444

37. Jarosch, E., Tuller, G., Daum, G., Waldherr, M., Voskova, A., and Schwien, R. J. (1996) *J. Biol. Chem.* **271**, 17219–17225

38. Glick, B. S., Brandt, A., Cunningham, K., Muller, S., Hallberg, R. L., and Schatz, G. (1992) *Cell* **69**, 809–822

39. Tzagoloff, A., Akai, A., and Needleman, R. B. (1975) *J. Biol. Chem.* **250**, 8228–8235

40. Belogrudov, G. I., Lee, P. T., Jonassen, T., Hsu, A. Y., Gin, P., and Clarke, C. F. (2001) *Arch. Biochem. Biophys.* **392**, 48–58

41. Do, T. Q., Hsu, A. Y., Jonassen, T., Lee, P. T., and Clarke, C. F. (2001) *J. Biol. Chem.* **276**, 18161–18168

42. Young, I. G., Stroobant, P., MacDonald, C. G., and Gibson, F. (1973) *J. Bacteriol.* **114**, 42–52

43. Nakahigashi, K., Miyamoto, K., Nishimura, K., and Inokuchi, H. (1992) *J. Bacteriol.* **174**, 7352–7359

44. Kwon, O., Kotsakis, A., and Meganathan, R. (2000) *FEMS Microbiol. Lett.* **186**, 157–161

45. Young, I. G., McCann, L. M., Stroobant, P., and Gibson, F. (1971) *J. Bacteriol.* **105**, 769–778

46. Palfey, B. A., Ballou, D. P., and Massey, V. (1995) in *Active Oxygen in Biochemistry* (Valentine, J. S., Foote, C. S., Greenberg, A., and Liebman, J. F., eds) pp. 37–83, Blackie Academic & Professional Press, Glasgow, UK

47. Wierenga, R. K., Terpstra, P., and Hol, W. G. J. (1986) *J. Mol. Biol.* **187**, 101–107

48. Eppink, M. H. M., Schreuder, H. A., and Van Berkel, W. J. H. (1997) *Protein Sci.* **6**, 2454–2458

49. Eggink, G., Engel, H., Vriend, G., Terpstra, P., and Witholt, B. (1990) *J. Mol. Biol.* **212**, 135–142

50. Jonassen, T., Proft, M., Randez-Gil, F., Schultz, J. R., Marbois, B. N., Entian, K. D., and Clarke, C. F. (1998) *J. Biol. Chem.* **273**, 3351–3357

51. Stenmark, P., Grunler, J., Mattsson, J., Sindelar, P. J., Nordlund, P., and Berthold, D. A. (2001) *J. Biol. Chem.* **276**, 33297–33300

52. Eppink, M. H. M., Boeren, S. A., Vervoort, J., and Van Berkel, W. J. H. (1997) *J. Bacteriol.* **179**, 6680–6687

53. Katagiri, M., Takemori, S., Suzuki, K., and Yasuda, H. (1966) *J. Biol. Chem.* **241**, 5675–5677

54. Goewert, R. R., Sippel, C. J., Grimm, M. F., and Olson, R. E. (1978) *FEBS Lett.* **87**, 219–221

55. Tzagoloff, A., Akai, A., and Foury, F. (1976) *FEBS Lett.* **65**, 391–395

56. Proft, M., Kotter, P., Hedges, D., Bojunga, N., and Entian, K. D. (1995) *EMBO J.* **14**, 6116–6126

57. McEwen, J. E., Ko, C., Kloeckner-Gruissem, B., and Poyton, R. O. (1986) *J. Biol. Chem.* **261**, 11872–11879

58. Wong, C. M., Dilworth, M. J., and Glenn, A. R. (1994) *Microbiology* **140**, 2775–2786

59. Kalin, M., Neujahr, H. Y., Weissmahr, R. N., Sejlitz, T., Johl, R., Fiechter, A., and Reiser, J. (1992) *J. Bacteriol.* **174**, 7112–7120

60. Suzuki, K., Mizuguchi, M., Ohnishi, K., and Itagaki, E. (1996) *Biochim. Bio- phys. Acta* **1275**, 154–156

61. Orser, C. S., Lange, C. C., Xun, L., Zahrt, T. C., and Schneider, B. J. (1993) *J. Bacteriol.* **175**, 411–416

62. Weijer, W. J., Hofsteenge, J., Beintema, J. J., Wierenga, R. K., and Drenth, J. (1983) *Eur. J. Biochem.* **133**, 109–118

63. DiMarco, A. A., Averhoff, B. A., Kim, E. E., and Ornston, L. N. (1993) *Gene (Amst.)* **125**, 25–33

64. Filippini, S., Solinas, M. M., Breme, U., Schluter, M. B., Gabellini, D., Biamonti, G., Colombo, A. L., and Garofano, L. (1995) *Microbiology* **141**, 1007–1016

65. Nurk, A., Kasak, L., and Kivisaar, M. (1991) *Gene (Amst.)* **102**, 13–18
