
Chemistry & Biology, Vol. 10, 301–311, April, 2003, ©2003 Elsevier Science Ltd. All rights reserved. DOI 10.1016/S1074-5521(03)00069-3

# The Biosynthetic Gene Cluster for the β-Lactam Carbapenem Thienamycin in Streptomyces cattleya

Luz Elena Núñez, Carmen Méndez, Alfredo F. Braña, Gloria Blanco, and José A. Salas*

Departamento de Biología Funcional and Instituto Universitario de Oncología del Principado de Asturias Universidad de Oviedo 33006 Oviedo Spain

## Summary

β-lactam ring formation in carbapenem and clavam biosynthesis proceeds through an alternative mechanism to the biosynthetic pathway of classic β-lactam antibiotics. This involves the participation of a β-lactam synthetase. Using available information from β-lactam synthetases, we generated a probe for the isolation of the thienamycin cluster from *Streptomyces cattleya*. Genes homologous to carbapenem and clavulanic acid biosynthetic genes have been identified. They would participate in early steps of thienamycin biosynthesis leading to the formation of the β-lactam ring. Other genes necessary for the biosynthesis of thienamycin have also been identified in the cluster (methyltransferases, cysteinyI transferases, oxidoreductases, hydroxylase, etc.) together with two regulatory genes, genes involved in exportation and/or resistance, and a quorum sensing system. Involvement of the cluster in thienamycin biosynthesis was demonstrated by insertional inactivation of several genes generating thienamycin nonproducing mutants.

## Introduction

Carbapenems are a class of β-lactam antibiotics with important application as antimicrobial agents, particularly in infections mediated by multidrug-resistant bacteria. They show a broad spectrum of activity and are relatively resistant to most of the clinically encountered bacterial β-lactamases. They are produced by *Streptomyces* species, although they have also been identified to be produced by gram-negative bacteria. Thienamycin (Figure 1) has been the first carbapenem isolated [1]. It is produced by *Streptomyces cattleya* NRRL 8057, which also synthesizes N-acetyl thienamycin and cephamycin C and penicillin N, and by *Streptomyces penemifaciens*. Thienamycin has been reported as the most potent of the naturally produced antibiotics, showing an extraordinary broad spectrum of activity, being highly resistant to bacterial β-lactamases [2], and playing an important clinical role in the treatment of nosocomial infectious diseases. However, thienamycin is highly unstable, and a most stable derivative produced by chemical synthesis, named imipenem, is the antibiotic of choice for clinical use.

Biosynthesis of classic β-lactam antibiotics (penicillins, cephalosporins, and cephamycins) is a well-known process that takes place by condensation of L-α-amino adipic acid, L-cysteine, and L-valine to form the tripeptide δ-(L-α-amino adipyl)-L-cysteinyl-D-valine (ACV) by a nonribosomal peptide synthetase, named ACVS (encoded by *pcbAB* genes). Cyclization of this tripeptide is then carried out by isopenicillin N synthase or IPNS (encoded by *pcbC* gene). These two steps are common to all producers of traditional β-lactam antibiotics, both in bacteria and in fungi [3]. However, in the biosynthesis of the nonclassic β-lactam compounds (carbapenems and clavams), a novel biosynthetic mechanism has been reported for the β-lactam ring formation that involves the biosynthetic enzyme, β-lactam synthetase (β-LS). This enzyme was first reported to participate in the biosynthesis of the simplest carbapenem antibiotic, car bapen-2-em-3-carboxylic acid, produced by the phytopathogen *Erwinia carotovora* (now *Pectobacterium car otovorum*), from which the gene cluster has been cloned [4]. An equivalent mechanism, involving a β-LS, was shown to participate in the biosynthesis of the clavulanic acid, being its coding gene located within the gene cluster for this β-lactamase inhibitor in *Streptomyces clavuligerus* [5]. Both findings revealed the existence of an alternative pathway for the biosynthesis of carbapenems and clavams.

Studies on the biochemistry of thienamycin biosynthesis have been addressed by Williamson et al. [6]. These authors provide biochemical evidence supporting that thienamycin biosynthesis in *S.* cattleya could proceed through an alternative pathway to that of cephamycin C. According to these authors, the bicyclic ring in thienamycin would derive from acetate (β-lactam carbons) and glutamate (pyrroline ring), and they proposed that it would be formed by condensation of acetyl-S-CoA with γ-glutamylphosphate instead of the ACV tripeptide formation as occurs in penicillins, cephalosporins, and cephamycins biosynthesis [6]. Li and coworkers [7, 8] reported the cloning of one gene (*tcs*) that was proposed to be involved in thienamycin biosynthesis. This gene product is a protein highly homologous to the C terminus of ACVS, and they named it thienamycin cyclase. Immediately downstream of the *tcs* gene they also reported a *pcbC*-like gene encoding a protein resembling an IPNS.

Here we report the cloning of the thienamycin gene cluster from *S.* cattleya NRRL8057. We also report the sequencing and insertional inactivation of several genes of the cluster and the generation of different thienamycin nonproducing mutants.

## Results

### Cloning of the Thienamycin Biosynthetic Gene Cluster from *Streptomyces cattleya* NRRL8057

To clone the gene cluster for thienamycin biosynthesis from *S.* cattleya NRRL8057 and based on the report of

Chemistry & Biology
302

![OH](https://i.imgur.com/1234567.png)

Figure 1. Chemical Structure of Thienamycin

the cloning of a thienamycin cyclase gene *tcs* [7], we used a cloning strategy by genetic homology. Two synthetic oligonucleotides were synthesized: LE1, including the start codon of the *tcs* coding region, and LE4, corresponding to a sequence within the downstream gene proposed to encode an IPNS by Li and coworkers [8]. Using these oligonucleotides, we amplified a 559 bp fragment from *S.* cattleya genomic DNA that was cloned and sequenced. A BLAST search of the deduced products from the amplified fragment showed similarities in its 5′ end with the *tcs* product, but with a degree of identity (59% identity) much lower than expected. It also showed similarity to the *pcbAB* gene product, an ACVS involved in cephamycin C biosynthesis (63% identity) [9]. At its 3′ end, the amplified fragment showed similarities with the first 28 amino acids of the *pcbC*-like gene product reported by Li et al. [8] and with the IPNS involved in cephamycin C biosynthesis (81% identity) [10]. The PCR product was used as a probe against BamHI-digested DNA from *S.* cattleya, and a 5 kb positively hybridizing band was identified and cloned for further analysis. To verify whether the cloned DNA region was involved in thienamycin biosynthesis, we carried out a gene replacement experiment. An apramycin resistance cassette containing the *aac(3)/IV* gene was cloned into the unique Ncol site located in the middle of the 5 kb BamHI fragment. Then, the in vitro mutagenized fragment was subcloned into pHZ1358. Through intergeneric conjugation from *E.* coli ET12567 to *S.* cattleya and after a double crossover, the wild-type region was replaced by the in vitro mutated one. Transconjugants were selected for apramycin resistance and thiostrepton sensitivity and verified by Southern hybridization, and they were tested for β-lactam production by HPLC and by bioassay against *Staphylococcus aureus* ATCC 6538P (cephamycin C-resistant, thienamycin-sensitive strain) and *Escherichia coli* ESS (strain supersensitive to β-lactams). All of them were found to still produce thienamycin, but they did not produce cephamycin C (data not shown). The conclusion from these experiments was that the use of the *tcs* gene as probe resulted in the identification of the cephamycin C gene cluster and not in the cloning of the thienamycin gene cluster.

Bearing in mind the report of a novel biosynthetic mechanism involving the enzyme β-lactam synthetase (β-LS) for the biosynthesis of a carbapenem [4] and clavulanic acid [5] as an alternative to classic β-lactam antibiotic biosynthesis, we searched for the possible existence of an equivalent gene in the *S.* cattleya genome that could be involved in the biosynthesis of thienamycin. Available sequence information for β-LSs was used together with information from the related family of class B asparagine synthetases (AS-Bs). These AS-Bs show an overall sequence similarity to β-LSs, despite the fact that they catalyze very different reactions. To construct the DNA probe for the screening, degenerate oligonucleotides were designed according to conserved amino acid regions deduced from proteins alignment between the two known β-LSs: β-LS from *S.* clavuligerus [5] and CarA from *P.* carotovorum [4]. Amino acid regions conserved in β-LSs but not present in asparagine synthetases were clear candidates for the design of the oligonucleotides. Total genomic DNA from *S.* cattleya was used as a template using the designed BLS1 and BLS4 oligonucleotides, and after PCR amplification, a 233 bp DNA fragment was obtained that was cloned and sequenced. Analysis of the sequence revealed that the PCR product encoded an amino acid region similar to known β-LS: 39% identity with β-LS and 28% identity with CarA. We used this fragment as a probe for the screening of a genomic library of *S.* cattleya. Three hybridizing cosmid clones, cosCAT25, cosCAT22, and cosCAT14, showing overlapping restriction maps were isolated and one of them, cosCAT25, selected for further analysis. Southern analysis (using the same probe) showed that at least two of the cosmids shared a hybridizing BamHI band of approximately 8.4 kb. To demonstrate the involvement of the cloned region in thienamycin biosynthesis, two constructs were generated in a conjugative vector in which the *aac(3)/IV* gene was inserted, in both orientations, in the unique BglII site located in the middle of the 8.4 kb BamHI fragment. After intergeneric conjugation, several mutants (apramycin resistant, thiostrepton sensitive) were obtained and analyzed by Southern hybridization and for thienamycin and cephamycin C production by bioassay and HPLC. All these mutants were found to be nonproducers of thienamycin but still produced cephamycin C. Southern analysis also confirmed the replacement of the 8.4 kb BamHI fragment of the wild-type strain by a 9.9 kb BamHI fragment in all the mutants. The results obtained with one of these mutants, T25N1, are shown in Figures 2 and 3. These experiments confirmed the involvement of this region in thienamycin biosynthesis.

Sequence Analysis of the Thienamycin Biosynthetic Gene Cluster

The insert in cosCAT25, and two overlapping BamHI fragments subcloned in pUC18 (pLE14 and pLE22) from cosCAT22 and cosCAT14, respectively, were sequenced. The complete overlapping nucleotide sequence comprising 32,329 bp was deposited at the EMBL Nucleotide Sequence Database with accession number AJ421798. Computer-assisted analysis of the DNA sequence revealed the presence of 28 complete ORFs and two incomplete ones (Figure 4). Functions for 22 gene products in the biosynthesis of thienamycin were proposed after comparison of the deduced amino acid sequences with other sequences in databases (Table 1). The products of the other eight ORFs (designated as *orf1–orf8*) did not show significant similarities with proteins in databases, with the exception of *orf5* that showed similarity with a predicted phosphoesterase. Taking into consideration

Thienamycin Biosynthetic Gene Cluster
303

**Figure 2. Insertional Inactivation in the Thienamycin Gene Cluster**

(A) Scheme representing the replacement in the chromosome of the wild-type *thn* region by the in vitro mutagenized. “B,” BamHI; “G,” BglII; “S,” Smal. *aac(3)IV*, Am resistance gene.

(B) Southern hybridization using the 8.4 kb BamHI fragment as probe. Chromosomal DNA from the wild-type strain and mutant T25N1 were digested with BamHI and analyzed by Southern hybridization.

---

ation these data, we cannot propose a role for any of these genes in thienamycin biosynthesis, although we cannot exclude it. Some of the genes could be translationally coupled with the stop codon of the first one overlapping with the start codon of the second one.

### Genes Probably Involved in Structural Functions in Thienamycin Biosynthesis

It has been reported that precursors of thienamycin and carbapen-2-em-3-carboxylic acid derive from L-glutamate (pyrroline ring) and acetate (β-lactam carbons) [6, 11]. In *S. cattleya*, it has been postulated that the initial reaction in thienamycin biosynthesis would be the condensation of acetyl-S-CoA with γ-glutamylphosphate (or some other γ-activated form of glutamate) with formation of the pyrroline ring [6]. In *P. carotovorum* it has been shown that CarB would be responsible for the first step in carbapenem biosynthesis [12, 13]. A BLAST search with the *thnE*-deduced product showed significant similarities with two enzymes catalyzing the first step of carbapenem biosynthesis: CarB (37% identity) from *P. carotovorum* [12] and the recently reported CpmB (33% identity) from *Photorhabdus luminescens* [14]. ThnE shows a conserved region, which is common to the enoyl-CoA hydratase/isomerase (ECH) family of enzymes which interact with acylCoA derivatives (Pfam00378). We propose that ThnE could have an analogous function to CarB and CpmB, being responsible for the condensation of glutamate and acetate as the first step in thienamycin biosynthesis in *S. cattleya* leading to the formation of the pyrroline ring.

---

**Figure 3. HPLC Analysis of Cultures of Wild-Type Strain and Mutant T25N1**

HPLC analysis for cephamycin C (A and C) and for thienamycin (B and D) production in the wild-type strain (A and B) and in mutant T25N1 (C and D). Detection was carried out at 264 nm for cephamycin C and at 309 nm for thienamycin. The arrows indicate the mobility of cephamycin C (*cef*) and thienamycin (*thn*).

Chemistry & Biology
304

cosCAT 25

pLE14
B P BB B
5000 10000
/1 2 A B C D E F G H I J K L M N O P Q R S T U V 3 4 5 6 7 8 thn cluster

pLE22
B S B G P B B B A A A A B
15000 20000 25000 30000

Figure 4. Genetic Organization of the thn Gene Cluster in S. cattleya

Proposed functions for the different genes are summarized in Table 1. The thn gene cluster is in blue. Sites for restriction enzymes are abbreviated as follows: “B,” BamHI (all sites indicated); and “P,” PshAI; “S,” SnaBI; “G,” BgIII; and “A,” BsaAI (only relevant sites for gene replacement experiments are indicated for these four enzymes). Cosmid and plasmids from which the nucleotide sequence was obtained are also shown.

ing to the formation of a biosynthetic intermediate containing the pyrroline ring.
The deduced product of thnM showed significant similarities to β-LSs involved in β-lactam ring formation.
The closest homologs to ThnM were: β-LS from S. cla-

vuligerus involved in clavulanic acid biosynthesis, 29% identity [5], and CarA, involved in carbapenem biosynthesis in P. carotovorum, 25% identity [12]. All three proteins, ThnM, CarA, and β-LS, are related to members of the AS-Bs, ATP/Mg²⁺-dependent amidotransferases

Table 1. Deduced Functions for the Genes in the Thienamycin Gene Cluster

| Gene | Amino Acids | Similar Protein (% Identity/Similarity), Accession Number | Proposed Function |
|------|-------------|---------------------------------------------------------|-------------------|
| thnA | 259         | Putative oxidoreductase from Streptomyces coelicolor     | Reductase         |
|      |             | (44/64), CAB 58301                                      |                   |
| thnB | 360         | Putative regulator MmyR from Streptomyces coelicolor(29/52), | Lactone-dependent  |
|      |             | CAC 36772                                               | transcriptional regulator |
| thnC | 212         | Hypothetical efflux protein from Mesorhizobium loti    | Lactone efflux    |
|      |             | (32/54), Np_104349                                      | transmembrane protein |
| thnD | 363         | Probable zinc-binding dehydrogenase Yersinia pestis     | Oxidoreductase    |
|      |             | (32/53), CAC 90475                                      |                   |
| thnE | 294         | CarB from Erwinia carotovora (37/60), AAD 38230          | Condensation of acetyl-S-CoA with γ-glutamyl phosphate |
| thnF | 327         | N-acetyltransferase like protein Myxococcus xanthus    | N-acetyltransferase |
|      |             | (30/52), AAL 56600                                      |                   |
| thnG | 263         | L-proline 4-hydroxylase from Dactylosporangium sp       | Hydroxylase       |
|      |             | (29/49), BAA 20094                                      |                   |
| thnH | 224         | Hypothetical protein from Thermobifida fusca           | Unknown           |
|      |             | (42/59), ZP_00058519                                    |                   |
| thnI | 476         | claR protein from Streptomyces clavuligerus            | Transcriptional activator |
|      |             | (32/52), AAC 38197                                      |                   |
| thnJ | 483         | Putative membrane transport protein from Streptomyces   | Transport protein |
|      |             | coelicolor (24/53), CAB 92892                           |                   |
| thnK | 681         | Fortimycin KL 1 methyltransferase from                  | Methyltransferase |
|      |             | Micromonospora olivasterospora (24/47),                 |                   |
|      |             | BAA 08420                                              |                   |
| thnL | 474         | Fortimycin KL 1 methyltransferase from Micromonospora   | Methyltransferase |
|      |             | olivasterospora (26/52), BAA 08420                      |                   |
| thnM | 458         | β-Is from Streptomyces clavuligerus (29/53), AAC 31901   | β-lactam synthetase |
| thnN | 367         | ORF 2 from Streptomyces griseus (35/55), BAA 37082       | Unknown           |
| thnO | 472         | ORF 3 from Streptomyces griseus (30/50), BAA 37083       | Oxidoreductase    |
| thnP | 484         | Phosphonoacetaldehyde methylase FOM 3 from              | Methyltransferase |
|      |             | Streptomyces wedmoriensis (28/52), BAA 32490            |                   |
| thnQ | 259         | Putative oxygenase AviO1 from Streptomyces viridochromogenes | Oxygenase        |
|      |             | (34/54), AAK 83181                                      |                   |
| thnR | 240         | Hypothetical protein from Streptomyces antibioticus    | Unknown           |
|      |             | (43/65), AAL 15612                                      |                   |
| thnS | 329         | ORF 12 from Streptomyces clavuligerus                   | β-lactamase      |
|      |             | (43/68), CAB 03624                                      |                   |
| thnT | 399         | Hypothetical protein SCE 22.10 from Streptomyces       | Cysteine transferase |
|      |             | coelicolor (71/81), CAB 90977                           |                   |
| thnU | 268         | Positive regulator CcaR from Streptomyces clavuligerus | Transcriptional activator |
|      |             | (43/60), AAC 32494                                      |                   |
| thnV | 137         | Glutathione transferase from Escherichia coli           | Cysteine transferase |
|      |             | (34/54), A60635                                         |                   |

Thienamycin Biosynthetic Gene Cluster
305

| β-Ls | 1   | MGAPVLPAAFGFLASARTGGGRAPGPVFA TRGSHTDIDTPQGE.RSLAATLVHAPSVA PDRAVARSLTGAPTAVLAGEI |
|------|-----|-------------------------------------------------------------|
| ThnM | 1   | . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . MAGFLLRCR...PEGAVPOHLGACTG...LATEV |
| CarA | 1   | . . . . . . . . . . . . . . MSNSFCVVYKGSDTDINNIQRDFDGKEALSNGYLFEQNGHYQKCEMERGTAYLIGSL |

| β-Ls | 80  | YNRDELLSV..LPAGR.APEGDAELVLRILERYDLHAFRILVNGRFATVV.RTCDPVILLATDHAGSVPL.YTCVAPGEV |
|------|-----|---------------------------------------------------------------------------|
| ThnM | 29  | TRDGADVIV..VHGDPITPGGQAPHAPLDELIAWTVRAY...SRFCAVL.VRDDPAVLWSDNGATCPLHYARGADDTL |
| CarA | 60  | YNRTFLIGAGVWEGEAYLANDAELLALL..FTRLGANALALAEGDFCFFIDEPNGELTVITESRGFSPVH..VVQGKKA |

| β-Ls | 153 | RASTEAKAL....AAHRDPKG....FPLADARRVAGLTGVYQVPAGAVMD.DLGSG..JAVTRTWTPGLSRRJ..LP |
|------|-----|---------------------------------------------------------------------------|
| ThnM | 102 | IVATSAGALLPLLGTAPVLGED....RAVTPGGRFAGVTAVPAGATAVLAVAGFDTE..PLITRRYHRLPARPTE..TD |
| CarA | 136 | WMINSKLVTAAEGEGALWFEEBALVCQSILMRADTYTPVKNAQRJKPGAHVITHDSEGYSFVESRTLTTPASNQLLALP |

| β-Ls | 219 | EGBAVAAVRAALEKAVAQRVTPGDTPILVLSGGIDS S GVAACAHRAAGELDTVSMGTDTSNEFREARAWVDHLRTRHREL |
|------|-----|---------------------------------------------------------------------------|
| ThnM | 174 | PERAVTAVRDALTHAVGLATGLTEAGVMLSGGVDS SSVAALAAREVTSLSTYTVGTPFGDEFFAQAWFAERLGSKHHEL |
| CarA | 216 | REPLIALTDRYLNAPREDLAPRFDTVGTIPLSGGL DSSLVTALASRHFKKLNNTYSIGTELSNEFEFSQQVADALGTHHQMK |

| β-Ls | 299 | TIPPTELLAQLPYAWASESVDPIDIEYLLPI TALYRAELDGPERRI.LTGYGADIPLGGM.HREDRLPALDTVLAHDMAT |
|------|-----|---------------------------------------------------------------------------|
| ThnM | 254 | VFEPEQLTALLPEMIRSLETWDLLTLQIAAPAC FLLGHLATGARQVMSGYGADLIIFAG.LGGTTETRIERSVAAQTA |
| CarA | 296 | ILESETEVINGTIESIYYNBIFEDGLSAEIQS GLFNVYRQ.AOQVSCMLTGYGSDLLFGGILKPGAQYDNPNOLAEOMYR |

| β-Ls | 377 | FDGLNEMSPVLSTLACHWTTTHPYWDREVLD LIVSLEAGLKRPHGGRDKWVLR..AAMADALPAETVNRPKLGVHEGSTTS |
|------|-----|---------------------------------------------------------------------------|
| ThnM | 333 | TAVSNEFPAYADARDVVVRYPYTFREVMSTAL GIRTGRLKVRPDTAKWVLR..SAVTGILPDEVAWRPKRGIHEGTA... |
| CarA | 375 | TRWTEFATHGASCYGIDIRHPFWSHSLISLCH ALHPDYKITFDNEVKNTILREYADSLQLLPKDIVWRKKIGIHEGS.... |

| β-Ls | 455 | SFSQLLLDHGVAEDRVHEAKRQVVR ELFDLTVGGGPHPSEVDTDVVRSVADR TARCAA |
|------|-----|---------------------------------------------------------------------------|
| ThnM | 408 | .MSPMFAAALGSDDPH TQARR.....LYEMATEVFRDTAA.GTDELEGSD EGLAGVAS* |
| CarA | 451 | SVNQAFAANVLGSTVDNYQTKSRFTYRY YQAFTRGRLSI TDVTPSQLKDLIKKD...... |

Figure 5. Alignment of the Deduced Amino Acid Sequences of β-LSs: β-LS from *S. clavuligerus*, CarB from *P. carotovorum*, and ThnM from *S. cattleya*

The arrows indicate the conserved regions selected for the oligonucleotides design for PCR amplification.

which catalyze the conversion of aspartic acid to asparagine. An alignment showing the conserved motifs shared by the three β-LS is shown in Figure 5. We propose that ThnM is a β-LS with a role in β-lactam ring formation in thienamycin biosynthesis in *S. cattleya*.

Thienamycin contains a hydroxyethyl side chain that derives from methionine through two methylations steps with retention of configuration as has been shown by Floss and coworkers [15]. In the thienamycin gene cluster, three genes (*thnK*, *thnL*, and *thnP*) encode proteins with significant similarities to methyltransferases from different secondary metabolite gene clusters: 24%, 26%, and 25% identity for ThnK, ThnL, and ThnP, respectively, with a methyltransferase involved in the biosynthesis of fortimicin [16] and with a methyltransferase involved in fosfomycin biosynthesis (20%, 24%, and 28% identity, respectively) [17]. Insertional inactivation of *thnL* showed its involvement in thienamycin biosynthesis (see below). We propose that some of these methyltransferases could be responsible for the methylation steps involved in the formation of the hydroxyethyl side chain in thienamycin biosynthesis.

The *thnV* product showed significant similarity (34% identity) with a glutathione transferase from the fosfomycin gene cluster [18]. In thienamycin biosynthesis, cysteine has been proposed as a precursor of the cysteamyl side chain [6], and this transfer could be mediated by the ThnV protein. The product of another gene (*thnT*) showed 71% identity with the hypothetical protein

SCE22.1 from *Streptomyces coelicolor* A3(2) (accession number CAB90977), and it also shares a conserved domain in the amino acid region 80–124 and 297–383 characteristic of the T4 peptidase family that include L- and D-aminopeptidases, that can also act as transferases (Pfam03576). ThnT could also play a role in the transfer of the cysteine residue.

Several oxidoreduction steps are required for thienamycin biosynthesis. Several genes identified in the cluster could encode oxidoreductases. The *thnA*-deduced product showed significant similarities (44% identity, accession number CAB58301) with a putative oxidoreductase from *S. coelicolor*. At the very beginning of its amino-terminal region, ThnA contains a very well-conserved motif of the short chain dehydrogenases/reductases (SDR) family. In the thienamycin biosynthetic pathway, ThnA could catalyze a reduction step between C6 and C8 that would occur after the second methylation step. The deduced product of *thnD* resembles a putative zinc binding dehydrogenase (32% identity; accession number CAC90475). ThnD shows a well-conserved domain of zinc binding dehydrogenases. The gene product of *thnO* displays a significant similarity with Orf3 from *Streptomyces griseus* (30% identity; accession number BAA37083) and with a probable aldehyde dehydrogenase from *Acinetobacter sp.* (25% identity) [19]. Gene replacement experiments showed the involvement of ThnO in thienamycin biosynthesis (see below). We propose that ThnD and ThnO could catalyze two oxidation

steps in thienamycin biosynthesis, occurring before and after the action of ThnA.

The closest similar protein to the *thnG* gene product is a L-proline 4-hydroxylase (29% identity) [20]. In the thienamycin biosynthetic pathway, a hydroxylation reaction could be one of the last steps in the biosynthesis and ThnG could be responsible for this hydroxylation reaction.

### Genes Probably Involved in Exportation and/or Resistance

Several genes in the thienamycin cluster could be involved in secretion of the antibiotic and/or in conferring resistance to the producer organism. The *thnJ* gene product resembles the membrane transport protein (32% identity; accession number CAB92892) from *S. coelicolor* A3(2) and the multidrug resistance protein from *Sulfolobus tokodaii* (24% identity; accession number BAB66030). ThnJ showed a well-conserved N-terminal region which is also found in the metabolite export pump, TcmA, involved in tetracenomycin C resistance [21]. We propose for ThnJ a role in thienamycin secretion through the cell membrane in *S. cattleya*.

The *thnF*-deduced product showed a conserved domain present in the family of proteins with N-acetyltransferase (GNAT) functions, a family which includes acetyltransferases involved in antibiotic resistance (Pfam00583). The closest homolog is a N-acetyltransferase that acetylates a ribosomal protein (30% identity; accession number AAL56600) and a putative acetyltransferase from *S. coelicolor* (26% identity; accession number CAB59671). N-acetyl thienamycin has been detected in culture broths of *S. cattleya* together with thienamycin, and a mutant strain of *S. cattleya* accumulating N-acetyl-deshydroxythienamycin has also been isolated [22]. The presence of an acetyltransferase in the clavulanic acid biosynthetic gene cluster with a putative role in antibiotic resistance [23] has been recently reported. The closest homolog of the *thnS* gene product is ORF12 from *S. clavuligerus*, (43% identity) that encodes a putative class A β-lactamase [23, 24], and it also shows similarity to a putative secreted protein from *S. coelicolor* (38% identity; accession number CAB52901). ThnS shares a β-lactamase-B motif which is conserved in the metallo-β-lactamase superfamily (Pfam00753). Whether ThnS (and ThnF) are involved or not in thienamycin resistance has to be clarified.

### Genes Probably Involved in Regulation of Thienamycin Biosynthesis

The *thnl* gene would encode a protein which shows the highest similarity (32% identity) with the transcriptional activator ClaR [25]. ClaR acts as an activator of clavulanic acid biosynthesis. A disrupted mutant in *claR* accumulated clavaminic acid, suggesting for ClaR a role in regulating late steps in clavulanic acid biosynthesis [26]. Thnl and ClaR show two well-conserved domains: a LysR-substrate binding domain and a helix-turn-helix motif of the LysR family. The latter motif is located at the C terminus of the protein (like in the case of ClaR), which is in contrast with most of the LysR regulatory proteins in which resides at the N terminus [26]. Based on the strong similarity of Thnl to ClaR, we propose for Thnl a role as a regulatory protein, probably acting as a transcriptional activator of some thienamycin biosynthetic genes in a similar way to ClaR in clavulanic acid biosynthesis.

The deduced product of *thnU* has been found to be significantly similar (43% identity) to CcaR. CcaR is a positive regulator which activates transcription of both clavulanic acid and cephamycin C biosynthetic genes [27] and is related to the SARP family of regulatory proteins [28]. ThnU shows a well-conserved bacterial transcriptional activator domain (BAD), found in the SARP-like family of regulators. We propose for ThnU a function in regulating early steps in thienamycin biosynthesis and probably cephamycin biosynthesis.

Two other genes in the cluster (*thnB* and *thnC*) are probably involved in quorum-sensing regulation. ThnB shows in its N-terminal region a well-conserved domain of bacterial regulatory proteins (Pfam00440) and displays similarity to lactone-dependent transcriptional regulators such as MmyR (29%; accession number CAC36772) and MmfR (28%; accession number CAC36768), two putative lactone-dependent transcriptional regulators (TetR family) from *S. coelicolor* [29]. ThnC showed significant similarities with many putative homoserine/homoserine lactone efflux transmembrane proteins, i.e., a protein from *Mesorhizobium loti* (32% identity; accession number NP_104349). ThnC shares a well-conserved domain with the LysE type translocator family (Pfam01810). It has been reported that synthesis of carbapenem-2-em-3-carboxilic acid is under quorum sensing control [30]. In this bacterium, it has been shown that the product of the first gene in the carbapenem cluster, named *carR*, is a member of the LuxR family of transcriptional regulators which activates transcription of the remaining genes in the cluster in response to a quorum-sensing signaling molecule, N-acyl homoserine lactone [31]. This homoserine lactone-dependent transcriptional activation allows the cells to coordinate expression of carbapenem with cell density. In *S. clavuligerus*, it has also been reported that clavulanic acid and cephamycin C production are under cell-density-dependent control [32].

### Genes with No Assigned Function

There are few other genes (*thnN, thnQ, thnH,* and *thnR*) in the cluster for which a role in thienamycin biosynthesis cannot be proposed based on the available information. The closest homolog to the *thnN*-deduced product is ORF2 from *S. griseus* (35% identity; accession number BAA37082) of unknown function. ThnN, shows in its C terminus, significant similarity (26% identity) to RedM, a putative peptide synthetase (accession number CAA16182) and contains a conserved domain of the AMP binding enzymes. This family includes a number of enzymes that catalyze reactions via an ATP-dependent covalent binding of AMP to their substrates. Insertional inactivation of *thnN* has shown its involvement in thienamycin biosynthesis (see below). The deduced product of *thnQ* showed significant similarities with two proteins involved in antibiotic biosynthesis: Avi01, an oxygenase from the avilamycin cluster, 34% identity [33], and SnoK

from the nogalamycin biosynthesis cluster, 30% identity [34]. The deduced products of *thnH* and *thnR* showed significant similarities with enzymes belonging to two different hydrolases families. The closest homologs to ThnH is a hypothetical protein from the actinomycete *Thermobifida fusca* (42% identity; accession number ZP_00058519) of unknown function and a hydrolase from *Caulobacter crescentus* (30% identity; accession number AAK22231). In its C-terminal region, ThnH shows part of a conserved domain of the hydrolase, haloacid dehalogenase-like family. The deduced product of *thnR* is related to two proteins of unknown function from different *Streptomyces* species: 43% identity for accession number AAL15612 and 42% identity for accession number CAB45548. ThnR shows a conserved domain characteristic of the NUDIX hydrolases family.

### Insertional Inactivation in the Thienamycin Gene Cluster

To demonstrate the involvement of different identified genes of the sequenced region in thienamycin biosynthesis, several mutants were generated by gene replacement. In all cases, the *aac(3)IV* gene was inserted within the coding region in the same direction of transcription (and in the opposite one). After introduction of the construct into *S. cattleya* by intergeneric conjugation, transconjugants in which a double crossover occurred were identified by their resistance to apramycin and susceptibility to thiostrepton and further verified by Southern analysis. The following genes were targeted: *thnL*, *thnN*, and *thnO*. In all three cases, the replacement gave rise to thienamycin nonproducing mutants, thus demonstrating their involvement in thienamycin biosynthesis.

Two other mutants were also generated at both ends of the sequenced region in order to define the possible boundaries of the thienamycin cluster. In one of the mutants, a single gene (*orf2*) located at the left end of the sequenced region was mutated by inserting the *aac3(IV)* gene. In the second mutant, a gene replacement was carried out causing the simultaneous deletion of four *orfs* (*orf4–orf7*) and partial deletion of other two (*orf3* and *orf8*) located at the right end of the sequenced region. The deduced products of all these *orfs* did not resemble any protein in databases that could indicate a possible role in thienamycin biosynthesis, although we cannot exclude it. Both mutants still retained the ability to produce thienamycin at similar yields than the wild-type strain.

### Discussion

Gene clusters for the biosynthesis of classic β-lactam antibiotics have been isolated from different producers, both bacteria and fungi. In all the cases, the mechanism for β-lactam ring formation involves two conserved steps: condensation of three aminoacids (L-α-amino-adipic acid, L-cysteine, and L-valine) by a nonribosomal peptide synthetase (ACVS) and cyclization of the formed tripeptide by an isopenicillin N synthase (IPNS). However, not much genetic information exists on the biosynthesis of the carbapenem-type β-lactam thienamycin, despite its significance and clinical importance in the treatment of nosocomial infectious diseases. At the time we began our research on this cluster, there were only two reports in the literature about the cloning of thienamycin biosynthetic genes [7, 8]. These authors reported the cloning of the *tcs* gene, supposed to encode a cyclase involved in thienamycin biosynthesis. However, as will be discussed below, the involvement of this gene in thienamycin biosynthesis is questionable and requires reinterpretation.

Antibiotic biosynthetic gene clusters have been cloned using different approaches. Two of the strategies most commonly used are the cloning of resistance genes and the cloning by genetic homology. We could not use the cloning of the resistance determinant and the search for biosynthetic genes in the adjacent DNA due to the natural resistance of streptomycetes to β-lactams. Cloning by genetic homology using information of previously characterized related pathways was hampered in the case of thienamycin by the lack of appropriate probes. In fact, the use of the only available probe, *tcs* gene, for the screening of genomic DNA from *S. cattleya*, led us to the isolation of a chromosomal region that was found to be involved in cephamycin C biosynthesis as demonstrated by insertional inactivation. Possibly these authors cloned cephamycin C biosynthetic genes instead of thienamycin genes. In the last years, a second major mechanism for β-lactam ring formation has been reported for carbapenem and clavam biosynthesis involving a β-LS [4, 5]. Given the carbapenem nature of thienamycin and assuming that a similar process could be responsible for β-lactam ring formation in thienamycin biosynthesis, we used conserved regions between two known β-LSs in the design of degenerated oligonucleotides for PCR amplification of part of a β-LS-encoding gene from the *S. cattleya* genome. This approach by genetic homology resulted in the successful isolation of the thienamycin gene cluster.

The gene cluster for thienamycin biosynthesis was located in a region of approximately 26.5 kb in which 22 genes were identified putatively involved in thienamycin biosynthesis. Gene replacement experiments of some of the genes in the cluster generated thienamycin non-producing mutants as an unequivocal proof of the involvement of the cloned cluster in the biosynthesis of this carbapenem.

Studies on the biochemistry of the biosynthetic pathway for thienamycin have been reported [6, 15]. We have correlated the biochemical information from these authors with our genetic data in an attempt to assign functions to the gene products corresponding to particular biosynthetic steps. A tentative pathway for thienamycin biosynthesis, essentially based on that proposed by Williamson et al. is shown in Figure 6. Earlier reactions in thienamycin biosynthesis are involved in formation of the bicyclic ring system. The initial reaction in thienamycin biosynthesis would be the pyrroline ring formation after condensation of acetyl-S-CoA with an activated form of glutamate. This reaction could be carried out by then, since it shows similarity to CarB, the first enzyme in the biosynthesis of the simplest carbapenem molecule in *P. carotovorum*, which is involved in condensation of the precursors (acetate and glutamate).

Concerning the origins of the hydroxyethyl side chain

One of the main features in thienamycin biosynthesis, that occurs immediately after the first methylation step, concerns the mechanism for β-lactam ring formation which involves a β-LS that originates the typical bicyclic ring structure of thienamycin. ThnM would be involved in this step since it shows convincing similarity to β-LSs known to participate in carbapenem and clavulanic acid biosynthesis [4, 5]. Once the bicyclic ring is formed, the following step would be the generation of the cysteiny1 side chain through the transfer of a cysteine residue, which could be carried out by the action of ThnV and/or ThnT. Other gene products, ThnD and ThnO, could be responsible for the oxidation reactions that occur between carbons C6 and C8 before the second methylation step or later on between carbons C2 and C3. This later oxidation step (C2, C3) takes place after a reduction reaction, between carbons C6 and C8, probably catalyzed by the action of ThnA, and it is not clear if it would occur before or after a hydroxylation reaction, putatively carried out by ThnG, and would render thienamycin as a final product.

Genes encoding proteins probably involved in exportation and/or resistance have been also found in the cluster. ThnJ, resembles membrane transport proteins, and we propose that it would play a role in thienamycin secretion through the cell membrane. In addition, ThnF, a putative N-acetyltransferase highly related to acetyltransferases involved in antibiotic resistance, would be responsible for the formation of N-acetyl thienamycin that has been also found in culture broths of S. cattleya. It has been suggested that N-acetyl thienamycin could be the last intermediate of the thienamycin pathway in S. cattleya, and this would be consistent with a possible role in resistance. Another protein, ThnS, could represent a type A β-lactamase, with a role not yet clarified in thienamycin biosynthesis and/or resistance. These type of enzymes have been identified in β-lactam producers (mainly cephamycin producers) and also in non-producing actinomycetes. They are cell wall-bound extracellular enzymes, and it is discussed whether they are involved in antibiotic resistance, in modulation of antibiotic biosynthesis, or in cell-wall formation [3].

Many pathway-specific transcriptional regulators involved in the production of different antibiotics have been found among antibiotic-producing actinomycetes, and in most cases they regulate expression of all biosynthetic and resistance genes in the cluster. Two exceptions are ClaR, which only regulates the expression of genes involved in late steps of the biosynthesis (after clavaminic acid) of clavulanic acid [26], and CcaR, involved in regulation of the early steps (up to clavaminic acid) of the same cluster [27]. In addition, CcaR is involved in regulation of a second gene cluster, cephamycin C [27]. Two genes of the thienamycin cluster, *thnl* and *thnU*, encode transcriptional activators similar to ClaR and CcaR, and possibly would act in a similar manner to these two activators. Thus, some of the gene products of the thienamycin cluster have their homologous counterparts in the carbapenem or clavam clusters, which is consistent with the relatedness found between the biosynthesis of these two β-lactam groups.

Two other gene products found in the thienamycin cluster could belong to a quorum-sensing regulation

Figure 6. Proposed Biosynthetic Pathway for Thienamycin Biosynthesis in S. cattleya and Assignment of Functions to the Different Genes

The biosynthetic pathway is based on that proposed by Williamson et al. [6].

of thienamycin, it has been shown that the hydroxyethyl side chain derives from methionine through two single carbon transfers with two methyl group incorporations involving S-adenosylmethionine [15]. In accordance with that, it has been reported that, in addition to thienamycin, S. cattleya also produces northienamycin, which contains a single carbon side chain at C-6 instead of the two carbons side chain of thienamycin [35]. According to Williamson et al. [6], the first methylation step in thienamycin biosynthesis would take place immediately after the condensation of precursors and before β-lactam ring formation, but the second methylation step would occur later in the pathway once the bicyclic ring was formed (Figure 6). Three genes in the thienamycin cluster, *thnK*, *thnL*, and *thnP*, would encode proteins with significant similarities to methyltransferases from different secondary metabolite gene clusters and could catalyze these methylation reactions.

Thienamycin Biosynthetic Gene Cluster

309

system. ThnB is a putative lactone-dependent transcriptional regulator, and ThnC shows similarity to putative homoserine lactone efflux transmembrane proteins. In S. cattleya, some mutants deficient in the synthesis of a diffusible factor required for thienamycin biosynthesis have been isolated, and thienamycin production was restored in one of these mutants after feeding with a diffusible product from another mutant [36]. These authors suggested a role for this diffusible product in regulating expression of the thienamycin cluster in a similar way to homoserine lactone regulating carbapenem production in the nondifferentiating bacterium *P. carotovorum*. In thienamycin biosynthesis, ThnB and ThnC could play a similar role in an analogous quorum sensing regulation system.

The availability of the genes for thienamycin biosynthesis provides the potentiality of its use for increasing thienamycin yields through heterologous expression of the cluster in another suitable host or through the use of the regulatory and/or structural genes. Furthermore, it opens up the possibility of generating by combinatorial biosynthesis novel derivatives with improved therapeutic properties, biosynthetic intermediates for further chemical modification, or derivatives with better stability properties.

### Significance

The biosynthetic gene cluster for the therapeutically important β-lactam antibiotic thienamycin has been cloned and characterized from the producer organism *Streptomyces cattleya*. According to the deduced functions of the different gene products, biosynthesis of thienamycin should proceed through an alternative mechanism to that of classic β-lactam antibiotics (pencillins, cephalosporins, and cephamycins). This mechanism involves the participation of a β-lactam synthetase as occurs in the biosynthesis of a simple carbapenem by *Pectobacterium carotovorum* and clavulanic acid by *Streptomyces clavuligerus*. Knowledge of the thienamycin gene cluster will allow the establishment and understanding of the biosynthesis of this clinically important antibiotic. It will also provide a great potential for increasing thienamycin production through the use of pathway-specific regulatory genes in the producer strain or through the heterologous expression of the cluster in another host. Genetic manipulation also opens up the possibility of generating novel thienamycin derivatives with improved desirable properties for clinical use or for further chemical modifications. This could be achieved either by inactivating specific genes of the cluster in the producer organism or by combinatorial biosynthesis with genes from other related β-lactam antibiotic producers.

### Experimental Procedures

#### Bacterial Strains, Culture Conditions, and Vectors

*Streptomyces cattleya* NRRL8057 was used in this study. *E. coli* DH10B (GIBCO) was used as a cloning host, *E. coli* ED8767 for constructing the cosmid library, and *E. coli* ET12567 (pUB307) for intergeneric conjugation. *Staphylococcus aureus* ATCC 6538P, tiamycin-sensitive-cephamycin C-resistant strain, and *E. coli* ESS, a supersensitive strain to all β-lactams, were used for bioassays.

For sporulation, *S. cattleya* was cultured on Bennet medium and for antibiotic production in R5A medium [37] using an inoculum previously grown in TSB medium (Merck). When needed, antibiotics were added to a final concentration of 25 μg/ml for thiostrepton, apramycin, kanamycin, chloramphenicol, or nalidixic acid; 100 μg/ml for ampicillin; and 20 μg/ml for tobramycin. pUC18 (Pharmacia) and the pUC derivative plJ2925 [38] were used as *E. coli* cloning vectors. Cosmid pKC505 [39] was used for construction of the genomic library and cosmid pHZ1358 [40] for intergeneric conjugation.

#### DNA Manipulation

Total DNA isolation, plasmid DNA preparations, restriction endonuclease digestions, ligations, and other DNA manipulations were performed according to standard procedures for *E. coli* [41] and for *Streptomyces* [42]. *S. cattleya* genomic library was constructed in cosmid pKC505. Intergeneric conjugation from *E. coli* ET12567 (pUB307) into *S. cattleya* was done using procedures previously described [43]. Total genomic DNA from *S. cattleya* was used as template for polymerase chain reaction (PCR) to obtain two independent probes, *tcs* and *β-Is*, that were used for the screening of either the chromosomal DNA or the cosmid library, respectively. For *tcs* amplification the following oligoprimers were used: LE1 (5′-ATCGCTAGAATGCGGCTGGCCCCGAGGGGGAGAAAGATCGACAA-3′, XbaI underlined) and LE4 (5′-ATCGAAGCTTCGCCTGGGCGTCTGGGTG GTGCCGAACAACTG-3′, HindIII underlined). For *β-Is*-like amplification, the following degenerated oligoprimers were used: BLS1 (5′-ATCGTCTAG ACG/C GAG/C ACG/C TCG/C AAC GAG TTG/C-3′, XbaI underlined) and BLS4 (5′-ATCGAAGCTT G/CGA G/CCC CTC GTG GAC GCC-3′, HindIII underlined). PCR reactions were carried out in a total volume of 50 μl, and the mixture contained 0.1 μg of *S. cattleya* total DNA, 30 pmol of each oligonucleotide primer (for *tcs* amplification), and 200 pmol of each degenerated oligonucleotide primer (for *β-Is* amplification), dNTPs (final concentration of 200 μM), 1×PCR buffer from Taq DNA polymerase, and 5 U of Taq DNA polymerase (GIBCO-BRL). Reactions were performed on a MJ Research MiniCycler with the following program: 1 cycle of denaturation at 98°C (5 min), 30 cycles of denaturation/annealing/synthesis at 94°C (1 min)/65°C (1 min)/72°C (1 min), and 1 cycle of final extension at 72°C (5 min). DNA fragments obtained after PCR amplification were cloned in the *E. coli* vector pUC18 and subjected to DNA sequencing. Labeling of DNA probes were performed with α-<sup>32</sup>P-dCTP or with digoxigenin, and Southern analysis, hybridization, and detection were performed according to literature procedures [41] and manufacturer recommendations (Amersham, Boehringer Mannheim).

#### DNA Sequencing

DNA sequencing was performed on double-stranded DNA templates by using the dideoxynucleotide chain termination method and the Cy5 Autocycle Sequencing Kit (Pharmacia Biotech). An ALF-express automatic DNA sequencer (Pharmacia) was used to sequence both DNA strands with primers supplied in the kit or with internal oligoprimers (17-mer). Computer-aided database searching and sequence analyses were made using the GCG sequence analysis software package of the University of Wisconsin Genetics Computer group and the BLAST program.

#### Gene Replacement Experiments

As a general rule, gene replacement experiments were carried out by insertion of the *aac(3)IV* gene. Introduction of plasmid DNA into *S. cattleya* was achieved through intergeneric conjugation from *Escherichia coli* ET12567 harboring pUB307 as has been described [43].

For insertional inactivation of the *tcs*-like region, a 1.5 kb BamHI (blunt-ended)-Smal containing the *aac(3)IV* gene was cloned in both orientations into the unique Ncol (blunt-ended) site located in the middle of the 5 kb BamHI fragment previously cloned in pUC18 to generate pLE5AF and pLE5AR. Then, the in vitro mutagenized BamHI fragments were subcloned into pHZ1358 generating plasmids pHZLE5F and pHZLE5R (depending on the fragment orientation).

For generation of a mutant in the *thnN* coding region, a 8.4 kb DNA BamHI fragment cosCAT25 was subcloned into pUC18, resulting in pLESC6. The *aac(3)IV* gene was then inserted as a BamHI (blunt-ended)-Smal fragment in the BglII site (blunt-ended) located within

thnN, generating pLE6F and pLE6R. The resultant 9.9 kb BamHI inserts were excised and subcloned into pHZ1358 generating pHZLE6F and pHZLE6R.

For insertional inactivation of *thnL*, a construct was generated by subcloning a 8.5 kb BglII fragment into plJ2925, resulting in pLESC4. The *aac(3)IV* gene was then inserted as a BamHI (blunt-ended)-Smal fragment in the SnaBI site located within *thnL* generating pLE4F and pLE4R. The resultant 10 kb BgIII inserts were excised and subcloned into pHZ1358 generating pHZLE4F and pHZLE4R.

To obtain a mutant in *thnO*, we used the previously generated pLESC6 (see above) in which the *aac(3)IV* gene was inserted as a BamHI (blunt-ended)-Smal fragment in the PshAI site located within *thnO*, generating pLE7F and pLE7R. The resultant 9.9 kb BgIII inserts were excised and subcloned into pHZ1358 generating pHZLE7F and pHZLE7R.

For insertional inactivation of ORF2, the corresponding 4.8 kb BamHI fragment was subcloned in pUC18 resulting in pLE14. The *aac(3)IV* was then inserted as a BamHI (blunt-ended)-Smal fragment in the PshAI site generating pLE2F and pLE2R. The 6.3 kb BamHI inserts were subcloned in pHZ1358 giving rise to pHZLE2F and pHZLE2R.

For the generation of a deletion mutant lacking ORFs 4–7 and part of ORFs 3 and 8, the corresponding 6.1 kb BamHI fragment was subcloned into pUC18 resulting in pLE22. Then, the four internal BsaAI fragments were replaced by the *aac(3)IV* gene causing a deletion in ORFs 3 to 8. The *aac(3)IV* gene was inserted in this construct as a BamHI-HindIII blunt-ended fragment in the BsaAI generating pLEΔ3-8F and pLEΔ3-8R, from which the resultant 4.2 kb inserts were obtained and subcloned into pHZ1358 generating pHZLEΔ3-8F and pHZLEΔ3-8R.

### Analysis of β-Lactam Production

Thienamycin and Cephamycin C production was qualitatively assayed by bioassay using the thienamycin-sensitive strain *Staphylococcus aureus* ATCC 6538P (cephamycin C resistant) and *E. coli* ESS (β-lactam supersensitive strain) as indicator strains. Identification and quantitative analysis of thienamycin and cephamycin C production was performed by HPLC using a reverse phase column (Symmetry C18, 4.6 × 250 mm; Waters) with acetonitrile and 0.1% trifluoroacetic acid in water as the mobile phase (5:95), at a flow rate of 1 ml/min. Detection and spectral characterization of peaks were made with a photodiode array detector and Millennium software (Waters), and quantification was done after signal integration at 309 nm for thienamycin and 264 nm for cephamycin C.

### Acknowledgments

This research has been supported by a grant of the Ministry of Science and Technology (1FD97-0799) and grants of the Plan Regional de Investigación del Principado de Asturias (GE-MEDO1-05 and FC-02-PC-REC01-20). L.E.N. was the recipient of a MIT grant from the Spanish Ministry of Science and Technology. We wish to thank Asturpharma, SA for helpful discussions.

Received: January 24, 2003  
Revised: February 28, 2003  
Accepted: March 7, 2003  
Published: April 21, 2003  

### References

1. Kahan J.S., Kahan, F.M., Geogelman, R., Currie, S.A., Jackson, M., Stapley, E.O., Miller, T.W., Miller, A., Henduin, D., Mochales, et al. (1979). Thienamycin, a new β-lactam antibiotic. I. Discovery, taxonomy, isolation and physical properties. J. Antibiot. 23, 1255–1265.
2. Coulton, S., and Hunt, E. (1996). Recent advances in the chemistry and biology of carbapenem antibiotics. Prog. Med. Chem. 33, 99–145.
3. Martín, J.F. (1998). New aspects of genes and enzymes β-lactam antibiotic biosynthesis. Appl. Microbiol. Biotechnol. 50, 1–15.
4. McGowan, S.J., Sebaihia, M., Porter, L.E., Stewart, G.S.A.B., Williams, P., Bycroft, B.W., and Salmond, G.P. (1996). Analysis of bacterial carbapenem antibiotic production genes reveals a novel beta-lactam biosynthesis pathway. Mol. Microbiol. 22, 415–426.
5. Bachmann, B.O., Li, R., and Townsend, C.A. (1998). β-Lactam synthetase: a new biosynthetic enzyme. Proc. Natl. Acad. Sci. USA 95, 9082–9086.
6. Williamson, J.M., Inamine, E., Wilson, K.E., Douglas, A.W., Liesch, J.M., and Albers-Schönberg, G. (1985). Biosynthesis of the β-lactam antibiotic, thienamycin, by *Streptomyces cattleya*. J. Biol. Chem. 260, 4637–4647.
7. Li, R., Wang, Y., and Zeng, Y. (1993). Cloning of thienamycin biosynthetic genes from *Streptomyces cattleya*. Chin. J. Biotech. 9, 1–7.
8. Li, R., and Wang, Y. (1996). Gene localization and expression of thienamycin cyclase gene in *Streptomyces lividans* TK24. Chin. J. Biotech. 12, 1–7.
9. Coque, J.J., Martin, J.F., Calzada, J.G., and Liras, P. (1991). The cephamycin biosynthetic genes *pcbAB*, encoding a large multidomain peptide synthetase, and *pcbC* of *Nocardia lactamdurans* are clustered together in an organization different from the same genes in *Acremonium chrysogenum* and *Penicillium chrysogenum*. Mol. Microbiol. 5, 1125–1133.
10. Locke, P., Ng, C.P., and Sim, T.S. (2000). PCR cloning, heterologous expression, and characterization of isopenicillin N synthase from *Streptomyces lipmanii* NRRL 3584. Can. J. Microbiol. 46, 166–170.
11. Bycroft, B.W., and Maslen, C. (1988). The biosynthetic implications of acetate and glutamate incorporation into (3R,5R)-carbapenem-3-carboxylic acid and (5R)-carbapen-2-em-3-carboxylic acid by *Serratia* sp. J. Antibiot. 4, 1231–1242.
12. McGowan, S.J., Sebaihia, M., O’Leary, S., Hardie, K.R., Williams, P., Stewart, G.S.A.B., Bycroft, B.W., and Salmond, G.P. (1997). Analysis of the carbapenem gene cluster of *Erwinia carotovora*: definition of the antibiotic biosynthetic genes and evidence for a novel beta-lactam resistance mechanism. Mol. Microbiol. 26, 545–556.
13. Li, R., Stapon, A., Blanchfield, J.T., and Townsend, C.A. (2000). Three unusual reactions mediate carbapenem and carbapenam biosynthesis. J. Am. Chem. Soc. 122, 9296–9297.
14. Derzelle, S., Duchaud, E., Kunst, F., Danchin, A., and Bertin, P. (2002). Identification, characterization, and regulation of a cluster of genes involved in carbapenem biosynthesis in *Photorhabdus luminescens*. Appl. Environ. Microbiol. 68, 3780–3789.
15. Houck, D.R., Kobayashi, K., Williamson, J.M., and Floss, H.G. (1986). Stereochemistry of methylation in thienamycin biosynthesis: example of a methyl transfer from methionine with retention of configuration. J. Am. Chem. Soc. 108, 5365–5366.
16. Kuzuyama, T., Seki, T., Dairi, T., Hidaka, T., and Seto, H. (1995). Nucleotide sequence of fortimicin KL1 methyltransferase gene isolated from *Micromonospora olivasterospora*, and comparison of its deduced amino acid sequence with those of methyltransferases involved in the biosynthesis of bialaphos and fosfomycin. J. Antibiot. 48, 1191–1193.
17. Hidaka, T., Goda, M., Kuzuyama, T., Takei, N., Hidaka, M., and Seto, H. (1995). Cloning and nucleotide sequence of fosfomycin biosynthetic genes of *Streptomyces wedmorensis*. Mol. Gen. Genet. 249, 274–280.
18. Navas, J., Leon, J., Arroyo, M., and Garcia Lobo, J.M. (1990). Nucleotide sequence and intracellular location of the product of the fosfomycin resistance gene from transposon Tn2921. Antimicrob. Agents Chemother. 34, 2016–2018.
19. Fujii, T., Takeo, M., and Maeda, Y. (1997). Plasmid-encoded genes specifying aniline oxidation from *Acinetobacter* sp. strain YAA. Microbiology 143, 93–99.
20. Shibasaki, T., Mori, H., Chiba, S., and Ozaki, A. (1999). Microbial proline 4-hydroxylase screening and gene cloning. Appl. Environ. Microbiol. 65, 4028–4031.
21. Guilfoile, P.G., and Hutchinson, C.R. (1992). Sequence and transcriptional analysis of the *Streptomyces glaucescens tcmAR* tetracenomycin C resistance and repressor gene loci. J. Bacteriol. 174, 3651–3658.
22. Rosi, D., Drozd, M.L., Kuhrt, M.F., Terminello, L., Came, P.E., and Daum, S.J. (1981). Mutants of *Streptomyces cattleya* producing

N-acetyl and deshydroxy carbapenems related to thienamycin. J. Antibiot. 34, 341–343.

23. Mellado, E., Lorenzana, L.M., Rodriguez-Sáiz, M., Díez, B., Liras, P., and Barredo, J.L. (2002). The clavulanic acid biosynthetic cluster of Streptomyces clavuligerus: genetic organization of the region upstream of the car gene. Microbiology 148, 1427–1438.

24. Perez-Llarena, F., Martín, J.F., Galleni, M., Coque, J.J.R., de la Fuente, J.L., Frere, J.M., and Liras, P. (1997). The bla gene of the cephamycin cluster of Streptomyces clavuligerus encodes a class A β-lactamase of low enzymatic activity. J. Bacteriol. 179, 6035–6040.

25. Aidoo, K.A., Wong, A., Alexander, D.C., Rittamner, R.A., and Jensen, S.E. (1994). Cloning, sequencing and disruption of a gene from Streptomyces clavuligerus involved in clavulanic acid biosynthesis. Gene 147, 41–46.

26. Paradkar, A.S., Aidoo, K.A., and Jensen, S.E. (1988). A pathway-specific transcriptional activator regulates late steps of clavulanic acid biosynthesis in Streptomyces clavuligerus. Mol. Microbiol. 27, 831–843.

27. Perez-Llarena, F.J., Liras, P., Rodriguez-Garcia, A., and Martin, J.F. (1997b). A regulatory gene (ccaR) required for cephamycin and clavulanic acid production in Streptomyces clavuligerus: amplification results in overproduction of both beta-lactam compounds. J. Bacteriol. 179, 2053–2059.

28. Wietzorrek, A., and Bibb, M.J. (1997). A novel family of proteins that regulates antibiotic production in streptomycetes appears to contain an OmpR-like DNA-binding fold. Mol. Microbiol. 25, 1181–1184.

29. Redenbach, M., Ikeda, K., Yamasaki, M., and Kinashi, H. (1998). Cloning and physical mapping of the EcoRI fragments of the giant linear plasmid SCP1. J. Bacteriol. 180, 2796–2799.

30. Bainton, N.J., Stead, P., Chabra, S.R., Bycroft, B.W., Salmond, G.P.C., Stewart, G.S.A.B., and Williams, P. (1992). N-(3-Oxohez- anoyl)-L-homoserine lactone regulates carbapenem antibiotic production in Erwinia carotovora. Biochem. J. 288, 997–1004.

31. McGowan, S.J., Sebaihia, M., Jones, S., Yu, B., Bainton, N., Chan, P.F., Bycroft, B., Stewart, G.S.A.B., Williams, P., and Salmond, G.P. (1995). Carbapenem antibiotic production in Erwinia carotovora is regulated by CarR, a homologue of the LuxR transcriptional activator. Microbiology 141, 541–550.

32. Sánchez, L., and Braña, A.F. (1996). Cell density influences antibiotic biosynthesis in Streptomyces clavuligerus. Microbiology 142, 1209–1220.

33. Weitnauer, G., Muhlenweg, A., Trefzer, A., Hoffmeister, D., Suss- muth, R.D., Jung, G., Welzel, K., Vente, A., Girreser, U., and Bechthold, A. (2001). Biosynthesis of the orthosomycin antibiotic avilamycin A: deductions from the molecular analysis of the avi biosynthetic gene cluster of Streptomyces viridochro- mogenes Tü57 and production of new antibiotics. Chem. Biol. 8, 569–581.

34. Torkkell, S., Ylihonko, K., Hakala, J., Skurnik, M., and Mantsala, P. (1997). Characterization of Streptomyces nogalater genes encoding enzymes involved in glycosylation steps in nogala- mycin biosynthesis. Mol. Gen. Genet. 256, 203–209.

35. Wilson, K.E., Kempf, A.J., Liesch, J.M., and Arison, B.H. (1983). Northienamycin and 8-epi-Thienamycin, new carbapenems from Streptomyces cattleya. J. Antibiot. 36, 1109–1117.

36. Buchan, T., Roach, C., Ruby, C., and Taylor, D. (1994). Mutants of Streptomyces cattleya defective in the synthesis of a factor required for thienamycin production. J. Antibiot. 49, 992–1000.

37. Fernández, E., Weissbach, U., Sánchez Reillo, C., Braña, A.F., Méndez, C., Rohr, J., and Salas, J.A. (1998). Identification of two genes from Streptomyces argillaceus encoding two glyco- syltransferases involved in the transfer of a disaccharide during the biosynthesis of the antitumor drug mithramycin. J. Bacteriol. 18, 4929–4937.

38. Janssen, G.R., and Bibb, M.J. (1993). Derivatives of pUC18 that have Bg/II sites flanking a modified multiple cloning site and that retain the ability to identify recombinant clones by visual screening of Escherichia coli colonies. Gene 124, 133–134.

39. Richardson, M.A., Kuhstoss, S., Solenberg, P., Schauss, N.A., and Nagaraja, R.N. (1987). A new shuttle cosmid vector, pKC505, for streptomycetes: its use in the cloning of three differ-

ent spiramycin-resistance genes from a Streptomyces ambofa- ciens library. Gene 61, 231–241.

40. Sun, Y., Zhou, X., Liu, J., Bao, K., Zhang, G., Tu, G., Kieser, T., and Deng, Z. (2002). “Streptomyces nanchangensis,” a pro- ducer of the insecticidal polyether antibiotic nanchangmycin and the antiparasitic macrolide meilingmycin, contains multiple polyketide gene clusters. Microbiology 148, 361–371.

41. Sambrook, J., Fritsch, E.F., and Maniatis, T. (1989). Molecular Cloning: A Laboratory Manual. (Cold Spring Harbor, NY: Cold Spring Harbor laboratory Press).

42. Kieser, T., Bibb, M.J., Buttner, M.J., Chater, K.F., and Hopwood, D.A. (2000). Practical Streptomyces Genetics (Norwich, UK: The John Innes Foundation).

43. Mazodier, P., Petter, R., and Thompson, C.J. (1989). Intergeneric conjugation between Escherichia coli and Streptomyces spe- cies. J. Bacteriol. 171, 3583–3585.

Accession Numbers

The complete DNA sequence of the sequenced region has been deposited at the EMBL Nucleotide Sequence Database under ac- cession number AJ421798.
