
CORRECTED 27 JUNE 2003; SEE LAST PAGE

REPORTS

8. M. Krssak et al., Diabetologia 42, 113 (1999).
9. G. Perseghin et al., Diabetes 48, 1600 (1999).
10. L. S. Szczepaniak et al., Am. J. Physiol. 276, E977 (1999).
11. K. F. Petersen et al., J. Clin. Invest. 109, 1345 (2002).
12. E. W. Kraegen et al., Diabetes 40, 1397 (1991).
13. A. Seppala-Lindroos et al., J. Clin. Endocrinol. Metab. 87, 3023 (2002).
14. M. E. Griffin et al., Diabetes 48, 1270 (1999).
15. A. Dresner et al., J. Clin. Invest. 103, 253 (1999).
16. C. Yu et al., J. Biol. Chem. 277, 50230 (2002).
17. S. I. Itani, N. B. Ruderman, F. Schmieder, G. Boden, Diabetes 51, 2005 (2002).
18. J. K. Kim et al., Proc. Natl. Acad. Sci. U.S.A. 98, 7522 (2001).
19. G. Boden, X. Chen, J. Ruiz, J. V. White, L. Rossetti, J. Clin. Invest. 93, 2438 (1994).
20. M. Roden et al., J. Clin. Invest. 97, 2859 (1996).
21. K. F. Petersen et al., Diabetes 47, 381 (1998).

22. J. A. Baecke, J. Burema, J. E. Frijters, Am. J. Clin. Nutr. 36, 936 (1982).
23. Written consent was obtained from each participant after the purpose, nature, and potential complications of the studies were explained. The protocol was approved by the Yale University Human Investigation Committee.
24. Materials and methods are available as supporting material on Science Online.
25. D. E. Kelley, J. He, E. V. Menshikova, V. B. Ritov, Diabetes 51, 2944 (2002).
26. G. I. Shulman, J. Clin. Invest. 106, 171 (2000).
27. V. Lebon et al., J. Clin. Invest. 108, 733 (2001).
28. I. Trounce, E. Byrne, S. Marzuki, Lancet 1, 637 (1989).
29. Y. Michikawa, F. Mazzucchelli, N. Bresolin, G. Scarlato, G. Attardi, Science 286, 774 (1999).
30. D. Harman, J. Am. Geriatr. Soc. 20, 145 (1972).
31. A. W. Linnane, S. Marzuki, T. Ozawa, M. Tanaka, Lancet 1, 642 (1989).

32. R. Luft, H. Luthman, Lakartidningen 90, 2770 (1993).
33. We thank Y. Kossover, M. Smolgovsky, A. Romanelli, and the staff of the Yale/New Haven Hospital General Clinical Research Center for expert technical assistance and the volunteers for participating in this study. Supported by grants from the U.S. Public Health Service (K-23 DK-02347, R01 AG-09872, P60 AG-10469, P30 DK-45735, M01 RR-00125, and R01 DK-49230).

Supporting Online Material  
www.sciencemag.org/cgi/content/full/300/5622/1140/DC1  
Materials and Methods  
SOM Text  
Figs. S1 and S2  
References  

29 January 2003; accepted 14 April 2003

---

**Regulation of Aging and Age-Related Disease by DAF-16 and Heat-Shock Factor**

Ao-Lin Hsu, Coleen T. Murphy, Cynthia Kenyon*

The Caenorhabditis elegans transcription factor HSF-1, which regulates the heat-shock response, also influences aging. Reducing *hsf-1* activity accelerates tissue aging and shortens life-span, and we show that *hsf-1* overexpression extends life-span. We find that HSF-1, like the transcription factor DAF-16, is required for *daf-2*-insulin/IGF-1 receptor mutations to extend life-span. Our findings suggest this is because HSF-1 and DAF-16 together activate expression of specific genes, including genes encoding small heat-shock proteins, which in turn promote longevity. The small heat-shock proteins also delay the onset of polyglutamine-expansion protein aggregation, suggesting that these proteins couple the normal aging process to this type of age-related disease.

Heat-shock factor activates transcription of heat-shock genes, which encode chaperones and proteases, in response to heat and other forms of stress. Previous studies have implicated heat-shock proteins (HSPs) in aging. For example, mild heat stress can cause a period of decreased mortality rate in *Drosophila*, and *hsp70* has been implicated in this effect (1). In addition, expression of genes encoding small heat-shock proteins (sHSPs) is increased in *Drosophila* lines selected for increased life-span (2), and overexpression of *hsp70F* increases the life-span of *C. elegans* (3).

Previously, we showed that reducing the activity of *C. elegans* HSF-1 causes a rapid-aging phenotype and shortens life-span (4). Conversely, we found that animals carrying additional *hsf-1* gene copies (5), which were resistant to heat and oxidative stress (fig. S1, A and B), lived approximately 40% longer than normal (Fig. 1A). Thus HSF-1 activity promotes longevity.

The FOXO transcription factor DAF-16, which functions in the *C. elegans* insulin/

Department of Biochemistry and Biophysics, University of California, San Francisco, CA 94143–2200, USA.

*To whom correspondence should be addressed. Email: ckenyon@biochem.ucsf.edu

---

**Fig. 1. HSF-1 promotes longevity.** (A) Blue, survival of wild-type animals grown on control bacteria containing vector alone; green, animals grown on bacteria expressing *hsf-1* dsRNA; red, animals carrying additional copies of *hsf-1* (CF1824). Two additional HSF-1 overexpressing lines were obtained and were found to extend life-span (table S1). (B) *hsf-1* overexpression extends life-span in a *daf-16*-dependent manner. Adult life-spans of wild-type and *hsf-1*-overexpressing (CF1824) animals grown on *daf-16* RNAi bacteria (green and blue lines). In addition, *hsf-1* overexpression extends the life-span of animals treated with *daf-2* RNAi (red and orange lines). (C and D) RNAi of *hsf-1* completely prevents the *daf-2(e1370)* mutation, but not the *isp-1(qm150)* or *eat-2(ad1116)* mutations, from extending life-span. Animals were grown on *hsf-1* RNAi bacteria from the time of hatching. Adult life-spans of wild-type (N2) (blue), *daf-2(e1370)* (red), *eat-2(ad1116)* (orange), and *isp-1(qm150)* (green) animals grown on (C) control bacteria or (D) *hsf-1* RNAi bacteria. All experiments have been repeated more than once with similar effects. For statistical data, see table S1.
IGF-1 signaling pathway (6, 7), also promotes longevity (8–10). As with *hsf-1*, inhibiting *daf-16* activity shortens life-span, and elevating *daf-16* activity increases life-span.

Fig. 2. HSF-1 is required for the expression of a subset of DAF-16 targets in *daf-2* mutants. Likewise, DAF-16 is required for the expression of a subset of heat-shock response genes after heat shock. (A) RT-PCR analysis of *hsf-1*, *mtl-1*, *sod-3*, and four *shsp*s in wild-type (N2) (lanes 1 to 3 and 10 to 12), *daf-16(mu86)* (lanes 4 to 6) and *daf-2(e1370)* (lanes 7 to 9 and 13 to 15) animals grown on either control or *hsf-1* RNAi bacteria. Shown are RT-PCR products from serial dilutions of total RNA isolated from the animals after RNAi treatment. *act-1* (β-actin) served as an internal control. (B) RT-PCR analysis of *hsf-1*, *hsp-70*, *aip-1*, *unc-33*, and four *shsp*s before (lanes 1 to 3 and 7 to 9) or after (lanes 4 to 6, 10 to 12, and 13 to 15) heat shock. RNA was harvested on day 1 of adulthood from wild-type (N2) animals grown on control bacteria (lanes 1 to 6) or bacteria expressing *hsf-1* dsRNA (lanes 13 to 15) as well as *daf-16(mu86)* animals grown on control bacteria (lanes 7 to 12). The heat-shock treatment was carried out less than 5 min before the RNA was harvested. Only slight variation was observed among replicates.

We found that the longevity phenotypes produced by *hsf-1(RNAi)* were similar to those produced by *daf-16(−)* mutations. DAF-16 is required for *daf-2*-insulin/IGF-1 receptor mutations to increase life-span; in the short-lived *daf-16* mutants, *daf-2* mutations are unable to increase life-span (6, 7, 11). HSF-1 appeared to be required as well, because neither of two *daf-2* mutations we examined, *e1370* and *mu150*, was able to extend the short life-spans of *hsf-1* RNAi-treated animals (Fig. 1, C and D; table S1). In contrast, the life-spans of both *daf-16(−)* mutants (12, 13) and *hsf-1* RNAi-treated animals (Fig. 1, C and D) can be extended by *eat-2(ad1116)* mutations [which inhibit pharyngeal pumping and may cause caloric restriction (12)] and by *isp-1(qm150)* mutations [which inhibit mitochondrial respiration (13)]. [Control experiments indicated that *hsf-1* RNAi sharply decreased *hsf-1* mRNA levels in all of these strains (fig. S1C).] Together these similarities suggested the hypothesis that HSF-1, like DAF-16, might function in the insulin/IGF-1 system. We also found that *daf-16* was required for *hsf-1* overexpression to extend lifespan (Fig. 1B). This, too, suggested that DAF-16 and HSF-1 might act together to promote longevity.

To investigate this hypothesis, we first asked whether DAF-16's activities required HSF-1. We found that *hsf-1(RNAi)* did not prevent DAF-16 from accumulating in the nuclei of *daf-2* mutants (9–11) (fig. S2) or activating expression of two known downstream genes, the metallothionein gene *mtl-1* (14) and the superoxide dismutase gene *sod-3* (15) (Fig. 2A). This indicates that DAF-16 can function independently of HSF-1. In addition, it implies that, without adequate HSF-1 activity, the increased expression of *sod-3* and *mtl-1* activity are not sufficient to increase the life-span of *daf-2* mutants [though these and other *hsf-1*-independent genes could certainly contribute to longevity (16)]. Next, we asked whether DAF-16 was required for HSF-1 to activate gene expression after heat-shock. This seemed plausible, because *daf-2* mutants are thermotolerant (17, 18), and we found that their thermotolerance required *daf-16* (table S2). However, the heat-inducibility of the genes *aip-1*, *unc-33*, and the HSP-70 homolog F44E5.4 (19) was not diminished by *daf-16(−)* mutations (Fig. 2B). Thus, HSF-1 can function independently of DAF-16.

The finding that neither DAF-16 nor HSF-1 was absolutely required for the other’s activity suggested that the two proteins might function together to turn on a specific subset of genes. Using DNA microarrays (16) we asked whether any known heat-shock genes (19) were regulated by the DAF-2 pathway. Others have found (20), and we confirmed (16), that expression of *hsp-70* and *hsp-90* did not change in *daf-2(−)* adults. In addition, *hsf-1* mRNA levels were not affected by *daf-2* mutations (16) (Fig. 2A). However, the expression of several heat-inducible genes, including four small heat-shock protein (*shsp*) genes, *hsp-16.1*, *hsp-16.49*, *hsp-12.6*, and *sip-1*, was sharply increased in animals with

Fig. 3. (A) Model. In DAF-2 pathway mutants and in normal animals subjected to heat shock, DAF-16 and HSF-1 bind the regulatory regions of common target genes, including *shsp* (and possibly other) genes and activate their expression. It is possible that DAF-2 pathway mutations or heat shock somehow increase the ability of HSF-1 or DAF-16 (respectively) to activate, in a specific fashion, the common target genes (dashed lines), although this need not be the case. Only a subset of DAF-16’s or HSF-1’s downstream targets require both proteins for their expression. These common target genes are required for the increased longevity of *daf-2* mutants and *hsf-1*-overexpressing animals. Other genes (such as *sod-3* or *mtl-1*) may also contribute to longevity; however, our findings suggest that without sufficient *hsf-1* activity, their increased expression is not sufficient to extend life-span. (B) Potential DAF-16 (blue) and HSF-1 (red) binding sites, GTAAAc/tA (16, 21) and TTCTA/cGAA (19), respectively, located upstream of the *shsp* genes.
reduced *daf-2* activity and decreased in animals with reduced *daf-16* activity (16).

We investigated the *shsp* genes in more detail. First, we confirmed the microarray data using reverse transcription polymerase chain reaction (RT-PCR) (Fig. 2A). We then asked whether HSF-1 was required for increased *shsp* gene expression in *daf-2* mutants. We found that it was (Fig. 2A). Thus,

HSF-1 functions in the insulin/IGF-1 system. In addition, DAF-16 was required along with HSF-1 to activate *shsp* expression after heat shock (Fig. 2B). Thus, DAF-16 functions in the heat-shock response.

A simple model to explain these findings is that in *daf-2* mutants and in normal animals subjected to heat shock, DAF-16 and HSF-1 both bind directly to regulatory sequences in the

*shsp* (and possibly other) genes (Fig. 3A). We favor this model because we found sequences identical to consensus DAF-16 (16, 21) and HSF-1 (19) binding sites upstream of all of these *shsp* genes (Fig. 3B). Heat shock triggers DAF-16 nuclear localization (9, 11), suggesting that it may increase the ability of DAF-16 to activate *shsp* expression. Thus it is possible that DAF-16 functions as a heat-shock factor to regulate part of the heat-shock response. Likewise, DAF-2 pathway mutations could potentially increase the ability of HSF-1 to activate *shsp* expression (Fig. 3A).

Because HSF-1 and DAF-16 are both required for the longevity of *daf-2* mutants, we next asked whether their common targets, the *shsp* genes, influenced life-span. RNAi of each gene shortened the life-span of *daf-2(e1370)* mutants by approximately 25% (Fig. 4B) and that of *daf-2(mu150)* mutants by a similar extent (16). *shsp* RNAi also decreased the longevity of animals overexpressing HSF-1 (Fig. 4C). In addition, like *hsf-1* RNAi, *shsp* RNAi decreased the life-span of wild type, though to a lesser extent than for *daf-2* mutants (Fig. 4A). Together these findings suggest that DAF-16 and HSF-1 increase longevity, at least in part, by increasing *shsp* expression. Moreover, because at least some of these sHSPs are likely to be functionally redundant, together they may make a substantial contribution to longevity.

The failure of previous workers to observe increased *shsp-gfp* expression in DAF-2 pathway mutants (22) was probably due to low GFP expression levels. However, this group found that when such mutants are subjected to transient heat shock, both *shsp* expression later in life and life-span increase substantially (22). This has suggested the hypothesis that *shsp* expression extends life-span (22). Curiously, treating wild-type animals in the same way does not increase *shsp* expression later in life, and produces only small increases in life-span (22). One possible explanation is that DAF-16 activity, which is required for *shsp* expression, is elevated in *daf-2* mutants. Heat-shocking these animals provides additional HSF-1 activity, which then further increases *shsp* expression and life-span. Consistent with this, we found that overexpression of HSF-1 further increased the life-spans of *daf-2(-)* animals (Fig. 1B).

How might sHSPs protect cells against aging? The sHSPs are known to form large oligomers that bind to unfolded proteins and prevent them from aggregating (23–25). In aging animals, this activity may prevent oxidized or otherwise damaged proteins from aggregating before they can be refolded or degraded (although sHSPs could conceivably influence longevity in a different way).

A fundamental mystery in biology is how the normal aging process is coupled to the diseases of aging. At least part of the answer appears to involve the insulin/IGF-1 pathway. For example, Huntington’s-like polyglutamine-

repeat proteins expressed in *C. elegans* form aggregates as the animals age, and this aggregation is delayed in long-lived insulin/IGF-1 pathway mutants (26). Because sHSPs are known to inhibit protein aggregation (27), we asked whether *shsp* RNAi might accelerate the onset of polyglutamine aggregates in *C. elegans*. We found that it did (Fig. 4, D and E), whereas RNAi of the other stress-response genes we tested did not (Fig. 4). As predicted, *daf-16* or *hsf-1* RNAi accelerated aggregation formation to an even greater extent (Fig. 4E) (28). Thus, possibly by functioning as molecular chaperones, sHSPs may influence the rates of aging and polyglutamine aggregation coordinately. In this model, mutations in the DAF-2 pathway delay both aging and susceptibility to this age-related disease, at least in part, by increasing *shsp* gene expression.

**References and Notes**

1. M. Tatar, A. A. Khazaeli, J. W. Curtsinger, *Nature* 390, 30 (1997).
2. R. Kurapati, H. B. Passananti, M. R. Rose, J. Tower, *J. Gerontol. A* 55, B552 (2000).
3. K. Yokoyama et al., *FEBS Lett.* 516, 53 (2002).
4. D. Garigan et al., *Genetics* 161, 1101 (2002).
5. Materials and methods are available as supporting online material at Science Online.
6. D. Gems, L. Partridge, *Curr. Opin. Genet. Dev.* 11, 287 (2001).
7. L. Guarente, C. Kenyon, *Nature* 408, 255 (2000).
8. C. Kenyon, J. Chang, E. Gensch, A. Rudner, R. Tabtiang, *Nature* 366, 461 (1993).
9. S. T. Henderson, T. E. Johnson, *Curr. Biol.* 11, 1975 (2001).
10. R. Y. Lee, J. Hench, G. Ruvkun, *Curr. Biol.* 11, 1950 (2001).
11. K. Lin, H. Hsin, N. Libina, C. Kenyon, *Nature Genet.* 28, 139 (2001).
12. B. Lakowski, S. Hekimi, *Proc. Natl. Acad. Sci. U.S.A.* 95, 13091 (1998).
13. J. Feng, F. Bussiere, S. Hekimi, *Dev. Cell.* 1, 633 (2001).
14. D. Barsyte, D. A. Lovejoy, G. J. Lithgow, *Faseb J* 15, 627-34 (Mar, 2001).
15. Y. Honda, S. Honda, *Faseb J.* 13, 1385 (1999).
16. C. T. Murphy et al., *Nature*, in press.
17. G. J. Lithgow, T. M. White, S. Melov, T. E. Johnson, *Proc. Natl. Acad. Sci. U.S.A.* 92, 7540 (1995).
18. D. Gems et al., *Genetics* 150, 129 (1998).
19. D. GuhaThakurta et al., *Genome Res.* 12, 701 (2002).
20. H. Yu, P. L. Larsen, *J. Mol. Biol.* 314, 1017 (2001).
21. T. Furuyama, T. Nakazawa, I. Nakano, N. Mori, *Biochem. J.* 349, 629 (2000).
22. G. A. Walker et al., *J. Gerontol. A* 56, B281(2001).
23. J. Horwitz, *Proc. Natl. Acad. Sci. U.S.A.* 89, 10449 (1992).
24. R. Van Montfort, C. Slingsby, E. Vierling, *Adv. Protein Chem.* 59, 105 (2001).
25. K. C. Giese, E. Vierling, *J. Biol. Chem.* 277, 46310 (2002).
26. J. F. Morley, H. R. Brignull, J. J. Weyers, R. I. Morimoto, *Proc. Natl. Acad. Sci. U.S.A.* 99, 10417 (2002).
27. J. I. Clark, P. J. Muchowski, *Curr. Opin. Struct. Biol.* 10, 52 (2000).
28. R. Morimoto's lab has observed that *hsf-1* RNAi causes short polyQ repeats to aggregate in *C. elegans* (personal communication).
29. P. K. Sorger, *Cell* 65, 363 (1991).
30. P. Verbeke, J. Fonager, B. F. Clark, S. I. Rattan, *Cell Biol. Int.* 25, 845 (2001).
31. We thank C.K. lab members for discussions and advice, J. Morley and R. Morimoto for the Q-40::YFP lines, and J. Ahringer for the bacterial RNAi clones. D. Lee and C.K. carried out the life-span analysis of *daf-2(mu150)*; *hsf-1(RNAi)*. A-L.H. was supported by the Canadian Institutes of Health Research. C.T.M.


was supported by Bristol-Myers Squibb Fellowship of the Life Sciences Research Foundation. This work was supported by grants from the Ellison Foundation and the NIH to C.K.

Supporting Online Material  
www.sciencemag.org/cgi/content/full/300/5622/1142/ DC1

Materials and Methods  
Figs. S1 to S3  
Tables S1 to S3  
References  

**Regulation of Interferon Regulatory Factor–3 by the Hepatitis C Virus Serine Protease**

Eileen Foy,$^{1}$ Kui Li,$^{2}$ Chunfu Wang,$^{1}$ Rhea Sumpter Jr.,$^{1}$ Masanori Ikeda,$^{2}$ Stanley M. Lemon,$^{2}$ Michael Gale Jr.$^{1\ast}$

Persistent infections with hepatitis C virus (HCV) are likely to depend on viral inhibition of host defenses. We show that the HCV NS3/4A serine protease blocks the phosphorylation and effector action of interferon regulatory factor–3 (IRF-3), a key cellular antiviral signaling molecule. Disruption of NS3/4A protease function by mutation or a ketoamide peptidomimetic inhibitor relieved this blockade and restored IRF-3 phosphorylation after cellular challenge with an unrelated virus. Furthermore, dominant-negative or constitutively active IRF-3 mutants, respectively, enhanced or suppressed HCV RNA replication in hepatoma cells. Thus, the NS3/4A protease represents a dual therapeutic target, the inhibition of which may both block viral replication and restore IRF-3 control of HCV infection.

Persistent HCV infection is a leading cause of liver disease worldwide and is frequently refractory to current interferon (IFN)-based therapies (1, 2). HCV persistence is facilitated by the ability of the virus to incorporate adaptive mutations and to replicate as a population of genetically distinct quasispecies (3, 4), but is likely to result from specific disruption of host immune responses by HCV proteins (5–7). The HCV genome, a 9.6-kb single-stranded RNA of

![Figure 1](#fig1)

**Fig. 1.** HCV regulation of the IRF-3 pathway. (A) Organization of the genotype 1b HCV RNA replicating in Huh7 C5B2-3 cells (referred to as Huh7 2-3 cells) (19) and of the genotype 1a polyprotein expressed conditionally in UHCV11 cells. The NS3/4A coding region is shaded. (B) Immunostaining for IRF-3 in SenV-infected (SenV<sup>+</sup>) or mock-infected (SenV<sup>-</sup>) cells. From top to bottom, IRF-3 subcellular localization in control Huh7 cells, 2-3 cells, or interferon-cured 2-3c cells is shown (20), as are UHCV11 cells with (+HCV 1a) or without polyprotein expression (-HCV 1a). On the right are immunoblots for NS3 and actin in corresponding cell extracts. (C) Huh7 cells, Huh7 2-3 cells, or 2-3c cells were transfected with the indicated promoter-luciferase reporter constructs and then infected with SenV (black bars) or mock-infected (gray bars). Luciferase activities were determined in cell extracts (mean ± S.D. from three experiments) (20).

www.sciencemag.org SCIENCE VOL 300 16 MAY 2003 1145

CORRECTIONS AND CLARIFICATIONS

ERRATUM

post date 27 June 2003

REPORTS: "Regulation of aging and age-related disease by DAF-16 and heat-shock factor" by A.-L. Hsu *et al.* (16 May 2003, p. 1142). The authors stated incorrectly that Lithgow's lab had previously used an *shsp::gfp* fusion to assay *shsp* gene expression. Instead, this group used an *shsp::lacZ* fusion and anti-SHSP antisera [G. Walker *et al.*, *J. Geron-tol*. 56A, B281 (2001)]. In addition, as the Hsu *et al.* paper was going to press, Walker and Lithgow reported that overexpression of *hsp-16* can extend *C. elegans'* life-span [G. A. Walker, G. Lithgow, *Aging Cell* 2, 131 (2003)], and, using microarrays, McElwee *et al.* observed an increase in *shsp* expression in insulin/IGF-1 pathway mutants [J. McElwee *et al.*, *Aging Cell* 2, 111 (2003)], as did Hsu *et al.*
