
Microbiology (2003), 149, 1447–1460

The THI5 gene family of Saccharomyces cerevisiae: distribution of homologues among the hemiascomycetes and functional redundancy in the aerobic biosynthesis of thiamin from pyridoxine

Raymond Wightman and Peter A. Meacock

Department of Genetics, University of Leicester, University Road, Leicester LE1 7RH, UK

Correspondence  
Peter A. Meacock  
mea@le.ac.uk  

Received 19 December 2002  
Revised 4 February 2003  
Accepted 6 March 2003  

The THI5 gene family of Saccharomyces cerevisiae comprises four highly conserved members named THI5 (YFL058w), THI11 (YJR156c), THI12 (YNL332w) and THI13 (YDL244w). Each gene copy is located within the subtelomeric region of a different chromosome and all are homologues of the Schizosaccharomyces pombe nmt1 gene which is thought to function in the biosynthesis of hydroxymethylpyrimidine (HMP), a precursor of vitamin B₁, thiamin. A comprehensive phylogenetic study has shown that the existence of THI5 as a gene family is exclusive to those yeasts of the Saccharomyces sensu stricto subgroup. To determine the function and redundancy of each of the S. cerevisiae homologues, all combinations of the single, double, triple and quadruple deletion mutants were constructed using a PCR-mediated gene-disruption strategy. Phenotypic analyses of these mutant strains have shown the four genes to be functionally redundant in terms of HMP formation for thiamin biosynthesis; each promotes synthesis of HMP from the pyridoxine (vitamin B₆) biosynthetic pathway. Furthermore, growth studies with the quadruple mutant strain support a previous proposal of an alternative HMP biosynthetic pathway that operates in yeast under anaerobic growth conditions. Comparative analysis of mRNA levels has revealed subtle differences in the regulation of the four genes, suggesting that they respond differently to nutrient limitation.

INTRODUCTION

The completion of the Saccharomyces cerevisiae genome-sequencing project revealed the presence of a large number of gene families concentrated within the subtelomeric regions of the chromosomes (Goffeau *et al.*, 1996). In some cases, these families are involved in the utilization of alternative carbon sources such as sucrose (SUC), maltose (MAL) and melibiose (MEL) (Carlson & Botstein, 1983; Chow *et al.*, 1989; Naumov *et al.*, 1995). However, other gene families have an uncertain or no function assigned to them; examples include the COS and PAU genes that form the largest families in S. cerevisiae, possessing 22 and 23 members, respectively.

We are interested in looking at the kind of cellular processes conferred by such subtelomeric gene families and in determining their redundancy. In this study, we report on a highly conserved gene family of four members, which, based upon homology to a gene present in Schizosaccharomyces pombe, have a putative role in biosynthesis of the enzyme cofactor thiamin diphosphate (ThdP). This gene family, here collectively termed the ‘THI5 gene family’, comprises the subtelomeric open reading frames (ORFs) designated YFL058w (THI5), YJR156c (THI11), YNL332w (THI12) and YDL244w (THI13).

The THI5 gene was originally isolated as a cDNA clone obtained through a screen for genes expressed during entry into stationary phase, when Saccharomyces cerevisiae was grown on industrial molasses medium (Praekelt & Meacock, 1992). Several genes were identified, one of which was originally named MOL1 (molasses inducible). MOL1 was later found to be expressed as a result of the depletion of exogenous thiamin (vitamin B₁) and was renamed THI4. Disruption of THI4 resulted in thiamin auxotrophy that was rescuable by supplementation of the growth medium with one of the thiamin precursors, hydroxyethylthiazole (HET) (Praekelt *et al.*, 1994). A second cDNA identified from that screen, MOL2 (THI5), represented another gene with a putative role in the biosynthesis of the other thiamin precursor, hydroxymethylpyrimidine diphosphate (HMP-PP). This prediction was based upon its homology to the single copy gene of Schizosaccharomyces pombe, termed nmt1 (Maundrell, 1990). An nmt1-negative mutant strain

Abbreviations: HET, hydroxyethylthiazole; HMP, hydroxymethylpyrimidine; ThdP, thiamin diphosphate.
R. Wightman and P. A. Meacock

was reported to be a thiamin auxotroph (Schweingruber *et al.*, 1991) that could be rescued by the addition of thiamin or hydroxymethylpyrimidine (HMP), but not of HET, to the growth medium.

The biosynthesis of thiamin, vitamin B₁, in *S. cerevisiae* has been reviewed elsewhere (Hohmann & Meacock, 1998). Thiamin is formed by the condensation of its phosphorylated precursors hydroxyethylthiazole phosphate and HMP-PP. HMP-PP is itself derived from pyridoxine (vitamin B₆) (Tazuya *et al.*, 1993). There has been a report of an alternative but unrelated HMP biosynthetic pathway that functions when *S. cerevisiae* is grown under anaerobic conditions (Tanaka *et al.*, 2000). This pathway has been shown not to proceed through the pyridoxine intermediate.

Most of the known thiamin biosynthetic genes are under feedback regulation from the active cofactor ThdP. So far, only positive regulators of this thiamin-dependent gene expression have been identified, encoded by *THI2*, *THI3* and *PDC2* (Hohmann & Meacock, 1998; Muller *et al.*, 1999; Nishimura *et al.*, 1992a, b), although several so-called *det* (derepressed expression on thiamin) mutants have been isolated which could represent negatively acting regulators (Burrows *et al.*, 2000). In *Schizosaccharomyces pombe*, *nmt1* gene expression is totally repressed in the presence of thiamin at concentrations of 0·5 μM or greater (Schweingruber *et al.*, 1991). In fact, *nmt1* was the first fully repressible gene discovered in *Schizosaccharomyces pombe* and its promoter is extensively used as a tool for the regulated expression of cloned genes (Maundrell, 1993).

In this study, we examined the distribution of the *THI5* multicopy state among the hemiascomycetes in order to establish how widespread this situation is, and to help formulate a hypothesis about the selective advantage it might confer; so far, only the sequenced genome of *Saccharomyces cerevisiae* S288C has been shown to possess *THI5* in more than one copy. We describe the construction of all combinations of deletion mutant strains covering the *THI5* gene family in this strain and we report their phenotypes with respect to HMP biosynthesis. The position of the putative isozymes relative to pyridoxine is also reported, an issue not yet addressed for the *Schizosaccharomyces pombe* homologue. Further analysis looks at the regulation of each of the four genes (*THI5*, *THI11*, *THI12* and *THI13*) and the roles of their products in the anaerobic formation of the precursor.

### METHODS

#### Strains and media.
Yeast strains used and created within this study are listed in Tables 1 and 2. Yeast media used were YP [1% (w/v) yeast extract and 2% (w/v) bacto-peptone], SD [0·67% (w/v) Difco yeast nitrogen base without amino acids] and Wickerham's (Wickerham, 1951). Where indicated, thiamin, HMP or HET was added to a final concentration of 2 μM, and pyridoxine was added to a final concentration of 9 μM. SD medium was supplemented with a complete synthetic mix containing all the amino acids and bases as listed by Rose *et al.* (1990). For prototrophic selection of yeast, the powder mix was made up as before, omitting the relevant nutrient(s). For selection of yeast strains containing the *kanMX4* module, YPD medium contained 300 mg G418 l⁻¹.

Solid media contained 2% (w/v) Difco purified agar. Glucose was added as carbon source to a final concentration of 2% (w/v). All

---

### Table 1. *Saccharomyces cerevisiae* strains used in this study and their genotypes

| Strain       | Genotype                                                                                          |
|--------------|---------------------------------------------------------------------------------------------------|
| S288Ca*      | *MATα mal gal2(cir⁺)*                                                                             |
| BY4705†      | *MATα ade2Δ::hisG his3Δ200 leu2Δ0 lys2Δ0 met15Δ0 trp1Δ63 ura3Δ0*                                     |
| RWY1         | *MATα ade2Δ::hisG his3Δ200 leu2Δ0 lys2Δ0 met15Δ0 trp1Δ63 thi5::TRP1 ura3Δ0*                         |
| RWY2         | *MATα ade2Δ::hisG his3Δ200 leu2Δ0 lys2Δ0 met15Δ0 trp1Δ63 thi11::TRP1 ura3Δ0*                        |
| RWY3         | *MATα ade2Δ::hisG his3Δ200 leu2Δ0 lys2Δ0 met15Δ0 trp1Δ63 thi12::TRP1 ura3Δ0*                        |
| RWY4         | *MATα ade2Δ::hisG his3Δ200 leu2Δ0 lys2Δ0 met15Δ0 trp1Δ63 thi13::TRP1 ura3Δ0*                        |
| RWY5         | *MATα ade2Δ::hisG his3Δ200 thi12::HIS3 leu2Δ0 lys2Δ0 met15Δ0 trp1Δ63 thi5::TRP1 ura3Δ0*             |
| RWY6         | *MATα ade2Δ::hisG his3Δ200 thi11::HIS3 leu2Δ0 lys2Δ0 met15Δ0 trp1Δ63 thi5::TRP1 ura3Δ0*             |
| RWY7         | *MATα ade2Δ::hisG his3Δ200 thi13::HIS3 leu2Δ0 lys2Δ0 met15Δ0 trp1Δ63 thi5::TRP1 ura3Δ0*             |
| RWY8         | *MATα ade2Δ::hisG his3Δ200 thi13::HIS3 leu2Δ0 lys2Δ0 met15Δ0 trp1Δ63 thi11::TRP1 ura3Δ0*            |
| RWY9         | *MATα ade2Δ::hisG his3Δ200 thi12::HIS3 leu2Δ0 lys2Δ0 met15Δ0 trp1Δ63 thi11::TRP1 ura3Δ0*            |
| RWY10        | *MATα ade2Δ::hisG his3Δ200 thi13::HIS3 leu2Δ0 lys2Δ0 met15Δ0 trp1Δ63 thi12::TRP1 ura3Δ0*            |
| RWY11        | *MATα ade2Δ::hisG his3Δ200 thi11::HIS3 leu2Δ0 thi12::LEU2 lys2Δ0 met15Δ0 trp1Δ63 thi5::TRP1 ura3Δ0* |
| RWY12        | *MATα ade2Δ::hisG his3Δ200 thi13::HIS3 leu2Δ0 thi12::LEU2 lys2Δ0 met15Δ0 trp1Δ63 thi5::TRP1 ura3Δ0* |
| RWY13        | *MATα ade2Δ::hisG his3Δ200 thi13::HIS3 leu2Δ0 thi12::LEU2 lys2Δ0 met15Δ0 trp1Δ63 thi11::TRP1 ura3Δ0* |
| RWY14        | *MATα ade2Δ::hisG his3Δ200 thi13::HIS3 leu2Δ0 thi5::LEU2 lys2Δ0 met15Δ0 trp1Δ63 thi11::TRP1 ura3Δ0*  |
| RWY15        | *MATα ade2Δ::hisG his3Δ200 thi13::HIS3 leu2Δ0 thi12::LEU2 lys2Δ0 met15Δ0 trp1Δ63 thi11::TRP1 ura3Δ0 thi5::URA3* |
| RWY16        | *MATα ade2Δ::hisG his3Δ200 thi13::HIS3 leu2Δ0 thi12::LEU2 lys2Δ0 met15Δ0 trp1Δ63 thi11::TRP1 ura3Δ0 thi5::KANMX4* |

* Johnston & Mortimer (1984).
† Brachmann *et al.* (1998).

1448 Microbiology 149
Table 2. Saccharomyces and Kluyveromyces strains used for copy number survey

| Strain | Reference/source |
| --- | --- |
| Z1278b | M. Werner-Washburne, University of New Mexico, USA |
| S. cerevisiae Type | CBS 1171 |
| S. cerevisiae Lager (formerly S. uvarum) | NCYC 1324 |
| S. cerevisiae Ale (formerly S. uvarum) | NCYC 1681 |
| S. cerevisiae EM 93 | NCYC 1324 |
| S. cerevisiae American Yeast Foam | NCYC 232 |
| S. bayanus | CBS 380 |
| S. pastorianus (formerly S. carlsbergensis) | CBS 1538 |
| S. paradoxus N12 | E. Louis, University of Leicester, UK |
| S. paradoxus N17 | E. Louis, University of Leicester, UK |
| S. cariocanus | Naumov et al. (2000) |
| S. kudriavzevii | Naumov et al. (2000) |
| S. mikatae | Naumov et al. (2000) |
| S. servazzii | CBS 4311 |
| S. castelli | CBS 4309 |
| S. unisporus | CBS 398 |
| S. kluyveri | CBS 3082 |
| S. exiguus | CBS 379 |
| K. lactis | CBS 2359 |
| K. drosophilaeum | CBS 2896 |
| K. polysporus | CBS 2163 |
| K. thermotolerans | CBS 6924 |

growth media were sterilized by autoclaving at 121°C, 15 p.s.i. (0·10Pa) for 20min. Where necessary, the additions of defined amounts of vitamins were added to the media after sterilization. For studies requiring anaerobic conditions, all media were further supplemented with 0·9mlTween80l-1 and 30mg ergosterol l-1 (C.Grant, personal communication). Anaerobic incubation of yeast on solid media was achieved by placing the agar plates into a BBL GasPak chamber containing an Oxoid gas-generating sachet (Unipath). Confirmation of anaerobic conditions was determined by the absence of growth of Saccharomyces cerevisiaeS288ConYP plates with glycerol (3%, v/v) as sole carbon source.

Reaction conditions for degenerate PCR analysis. The primers SNZfwd, THI5back, THI5fwd and AADfwd (Table 3) were designed to anneal to conserved regions of the SNZ, THI5 and AAD genes in S. cerevisiae. As the SNZ gene family has two almost identical subtelomeric members, the primer was designed to anneal to portions conserved between S. cerevisiae SNZ and Saccharomyces mikatae SNZ (sequence kindly supplied by P. Clifton, University of Washington, USA). Reaction conditions were 94°C for 2min, followed by 30 cycles at 94°C for 1min, 52°C for 1min and 72°C for 55s, with a final incubation step at 72°C for 10min.

Construction of targeting vectors for disruption of THI5, THI11, THI12 and THI13. A single set of two oligonucleotide primers was used to amplify a nutritional marker or kanMX4 from the pRS4XX series of plasmids (Brachmann et al., 1998). This created a targeting cassette consisting of the marker flanked by 40bp of

Table 3. Sequences of oligonucleotide primers

| Name | Sequence (5'→3') | Target sequence |
| --- | --- | --- |
| THI5UP | ATGTCTACAGACAAGATCACATTTTGTGTTGAACTGG CAACCAACCCCAGATTGTACTGAGAGTGCCAC | Marker gene flanking sequence of pRS41X series of plasmids |
| THI5DN | AGCTGGAAGAGCCAATCTTGTGAAAGTACC TTCC TG TCTGCATTTTCTGTGCGGTATTCACACCCG | Marker gene flanking sequence of pRS41X series of plasmids |
| UPSTHI5 | GGCGCGTTA(AG)TG CATAAAAGACAGC | THI5/11/12/13 promoter |
| TRP1 | CAATGCCCTCCCTCTTGGCC | TRP1 disruption cassette |
| HIS3 | GCTTCTCTTATGGCAACCGC | HIS3 disruption cassette |
| LEU2 | CAGATGAGGCGCTGGAAGCC | LEU2 disruption cassette |
| URA3 | TACCTGGGCCCAACCACACCG | URA3 disruption cassette |
| THI5fwd | ACCAAAGATGTTACGCTTAC | THI5/11/12/13 reading frames |
| AADfwd | GGAACATGGCACTGARTCTG | Conserved AAD gene sequence |
| SNZfwd | GATGGGTGGGTATYTCCATCC | Conserved SNZ gene sequence |
| THI5back | AAACCCATGT CGACCTTACC | THI5/11/12/13 reading frames |

sequence that was homologous to any of the four loci. The primers used were THI5UP and THI5DN (Table 3). After purification by gel electrophoresis, cassette DNA was transformed into yeast using the lithium acetate method of Geitz *et al.* (1995).

**Identification of THI5, THI11, THI12 and THI13 gene disruptions.** Identification of disrupted loci was carried out using Southern blot hybridization analysis of yeast genomic DNA digested with XhoI (Gene Images; Amersham Pharmacia Biotech). A ClaI–XhoI THI5 DNA fragment from plasmid pRH4 was used as a probe. For strains RWY15 and RWY16, disrupted loci were examined by PCR as described in the text using the oligonucleotide primers listed in Table 3.

**THI5 copy number survey.** Yeast genomic DNAs were digested with XhoI, PstI and EcoRI. Southern blot analysis used as probes the *S. cerevisiae* THI5 sequence, obtained on a ClaI–XhoI fragment from pRH4, and the *Kluyveromyces lactis* THI5 sequence, obtained on HindIII fragment from pDW5a.

**Slot-blot hybridization of yeast RNA.** All strains were inoculated into Wickerham's glucose medium without thiamin. After overnight incubation, each yeast culture was then inoculated to a final cell density of \(3 \times 10^5\) cells ml\(^{-1}\), into the same medium supplemented as indicated with thiamin, HMP or HET. Samples for RNA preparations were harvested at a cell density of \(2-5 \times 10^7\) cells ml\(^{-1}\).

Total yeast RNA was isolated using the rapid phenol/SDS extraction protocol described by Schmitt *et al.* (1990). Approximately 2 μg RNA was applied to a Hybond-N\(^+\) membrane using a Hybri-Slot filtration manifold apparatus (Gibco-BRL). The membranes were then hybridized as described by Church & Gilbert (1984) using either the pRH4 THI5 probe or a 342 bp ACT1 EcoRI–HindIII fragment of pBS-Actin (gift of T. Pillar, Universitäts Krankenhaus-Eppendorf, Hamburg, Germany). Both probes were labelled by the incorporation of

---

[α-32P]CTP. The relative emitted radioactivity of positively hybridizing bands was measured using a Molecular Dynamics PhosphorImager. Analysis of the intensities was carried out using IMAGEQUANT software.

---

**RESULTS**

**THI5 gene family members are subtelomerically located with other gene families in conserved arrangements**

In *S. cerevisiae*, the four members of the THI5 gene family are found within the subtelomeric regions of chromosomes IV-L (THI13), VI-L (THI5), X-R (THI11) and XIV-L (THI12) along with members of other gene families. The DNA sequences both upstream and downstream of each of the four genes were compared by examining computer alignments of sequential 2–3 kb blocks of DNA. Significant ORFs were identified and annotated with their gene names or systematic codes from the Saccharomyces Genome Database. Further functional assignment of each ORF was attempted by using both bibliographic searches and BLASTP analyses of the predicted proteins to identify any previously characterized homologues.

Fig. 1 shows the results of these analyses in the form of a simplified physical map. Centromere proximal to all four THI5 genes is a member of the AAD gene family (Delneri *et al.*, 1999a), originally postulated to encode an aryl-alcohol dehydrogenase. Two of these genes, AAD4 and AAD6, are expressed in response to oxidative stress in a Yap1p-dependent manner (Delneri *et al.*, 1999b). The

---

![Diagram](attachment:diagram.png)

---

**Fig. 1. Chromosomal environments of members of the THI5 gene family.** The locations and orientations of identified genes (ORFs) in regions of the yeast genome containing the THI5 gene family members THI5, THI11, THI12 and THI13, which are denoted as solid arrows, are shown. Vertical arrows point to proposed 'divergence points' where one of two blocks of genes were inserted between the THI5 member and the COS gene. Chromosome numbers and arm are indicated for each locus. In all cases, the chromosome telomere is on the left of the figure as drawn.

chromosomal sequences that are centromere proximal to the AAD genes and adjacent to the four THI5 genes diverge extensively, showing no conservation and the ORFs are not repeated. These AAD genes, therefore, appear to represent the boundary of the subtelomeric repeated DNA.

The sequences of the THI5 gene family members are highly conserved with only 27 nt differences between the four ORFs, of which 24 are codon third base changes and only three result in amino acid substitutions. This conservation extends into the promoter regions for approximately 400 bp upstream of the ORF start codon, beyond which sequence identity is only maintained between chromosomes IV-L and X-R, and between chromosomes VI-L and XIV-L. The point at which the two sets of chromosomes diverge can be assigned to base residue –399 of the THI5 sequence. The resumption of sequence conservation between all four chromosomes occurs at the final leucine codon (TTA) of each of the COS genes. Between these ‘divergence points’ exists one of two duplicated blocks of genes (Fig. 1, shaded areas). Each block, which includes intergenic regions as well as ORFs, is very highly conserved with its partner.

The block adjacent to THI11 and THI13 on chromosomes X-R and IV-L contains genes that encode enzymes involved in carbohydrate metabolism: HXT15 and HXT16 encode proteins that resemble hexose transporters and both function in the uptake of mannose, fructose and glucose, but not galactose (Wieczorke et al., 1999); SOR1 and SOR2 appear to encode a putative zinc-containing alcohol dehydrogenase that is homologous to sorbitol dehydrogenase (Gonzalez et al., 2000); both MPH2 and MPH3 encode high-affinity α-glucoside permeases capable of transporting maltose, maltotriose, methyl α-glucoside and turanose (Day et al., 2002); the function of YJR157w and its homologous pseudogene sequence on chromosome IV-L is unresolved.

Located telomere proximal to THI5 and THI12 on chromosomes VI-L and XIV-L is a block that contains members of the SNO and SNZ gene families. Both SNO and SNZ were first identified as genes expressed at the entry to stationary phase (Padilla et al., 1998). Homologues of SNZ from other fungi have since been found to encode pyridoxine biosynthetic enzymes (Ehrenshaft et al., 1999; Osmani et al., 1999). Work done in this laboratory has shown that the subtelomeric copies of S. cerevisiae SNO and SNZ exist to provide pyridoxine (or a pyridoxal intermediate) as a precursor for the HMP pathway (L. Marsh, R. Wightman & P. A. Meacock, unpublished data); this has recently been confirmed by other workers (Rodriguez-Navarro et al., 2002). The determination of a role for the homologues YFL061w and YNL335w awaits the construction of a mutant S. cerevisiae strain containing deletions of both genes.

The COS genes and adjoining ORFs, which are found distally upstream of all four THI5 genes on the other side of the duplicated gene blocks, are members of much larger gene families of unknown function.


Distribution of THI5, as a gene family, among the hemiascomycetes

To date, the existence of THI5 as a gene family has only been observed in the sequenced genome of S. cerevisiae S288C. The homologous gene nmt1 of Schizosaccharomyces pombe and those of filamentous fungi appear to be present only as a single copy per genome. Therefore, to gain information about when and why amplification of the THI5 gene occurred we decided to investigate other related hemiascomycete yeasts. A survey of THI5 copy number was carried out by Southern blot analysis of digested yeast genomic DNAs using a probe that cross-hybridized with all four Saccharomyces cerevisiae genes. Restriction digests using the enzymes XhoI, PstI and EcoRI individually can distinguish each of the genes THI5, THI11, THI12 and THI13 in strain S288C (Fig. 2). Therefore, equivalent hybridizations were carried out on genomic DNAs isolated from a sample of hemiascomycetes classified as Saccharomyces or Kluyveromyces, so allowing an estimate of THI5 copy number based upon the number of positively hybridizing fragments.

Initially, two of the parental strains of S288C, EM93 and American Yeast Foam, were examined (Fig. 2). EM93 is thought to be the principal progenitor, donating up to 90 % of the S288C genome (Mortimer & Johnston, 1986). Our analysis showed that this yeast contains four THI5 family genes on restriction fragments of similar sizes to those seen in S288C. However, for each of the three restriction digests a fifth band was also detected from EM93, suggesting that this strain actually possesses a five-member THI5 family.

| kb | S288C | EM93 | Yeast Foam |
|----|-------|------|------------|
|    | 1     | 2    | 3          | 1      | 2     | 3    | 1       | 2     | 3    |

Fig. 2. THI5 copy number survey of S288C and its two progenitor strains, EM93 and American Yeast Foam. Genomic DNA from each strain was digested with restriction endonucleases Xhol (lane 1), Pstl (lane 2) and EcoRI (lane 3) individually and subjected to electrophoresis. Southern blot hybridization was carried out using the pRH4 THI5 probe (see Methods). The full ancestry of S. cerevisiae strain S288C is described by Mortimer & Johnston (1986).

R. Wightman and P. A. Meacock

contrast, similar restriction digests of DNA from American Yeast Foam displayed just three THI5-hybridizing fragments, but because of restriction fragment length polymorphisms the bands could not be attributed to particular loci. Therefore, to determine the genotype of the Yeast Foam strain, we used a PCR-based assay that we had developed to detect the individual members of the THI5 gene family. This involved DNA amplification between a primer that anneals within a conserved region of the reading frames and one of four primers that anneals to a short unique sequence within the variable downstream non-coding region of a particular gene family member. This PCR analysis showed that only the THI11 gene of chromosome X was not amplified and so suggests that this particular gene may be absent from this strain (data not shown).

Other laboratory strains of S. cerevisiae were also surveyed for THI5 copy number (data not shown). Both BY 4705 and W303a, which are descended from S288C, showed the same four hybridizing fragments as S288C, whilst a pseudohyphal strain, Σ1278b, gave no signal corresponding to the THI5 gene on chromosome VI. This same strain had previously been shown to lack the adjacent SNZ3 gene (Padilla et al., 1998), suggesting that the whole of the left subtelomeric portion of this chromosome is absent. A survey of three commercial strains of S. cerevisiae, the CBS 1171 'type' strain and two strains (NCYC 1681 and NCYC 1324) formerly designated as Saccharomyces uvarum used in the production of ales and lagers (Goodey & Tubb, 1982), revealed THI5 to be present as a gene family but with fewer copies than S288C (data not shown); from the number of positively hybridizing fragments we deduce that it is likely that these yeasts possess only two copies of THI5.

Other species of yeasts from the Saccharomyces and Kluyveromyces genera were surveyed using the same S. cerevisiae-derived THI5 hybridization probe (Fig. 3). Within the Saccharomyces collection were a group of strains very closely related to S. cerevisiae and termed the 'Saccharomyces sensu stricto complex', which included

| Species | THI5 copy number |
|---------|-------------------|
| S. unisporus | 0 |
| S. servazzi | 0 |
| S. paradoxus | N17: 5 |
|           | N12: 5 |
| S. cariocanus | 3+ |
| S. cerevisiae CBS 1171 | 2 |
| EM93 | 5 |
| S288C | 4 |
| Yeast Foam | 3 |
| S. kudriavzevii | 3+ |
| S. mikatae | 3+ |
| S. pastorianus | 2+ |
| S. bayanus | 2+ |
| K. lactis | 1 |
| K. polysporus | 0 |
| S. exiguus | 1 |
| K. thermotolerans | 1 |
| S. kluyveri | 1 |
| S. castelli | 0 |

Fig. 3. Phylogenetic tree of yeasts surveyed for THI5 copy number. The tree was created from a CLUSTAL W alignment of ITS1 and ITS2 sequence data. The shaded region shows those yeasts that belong to the Saccharomyces sensu stricto complex. Copy numbers with a plus symbol denote that these yeast species contain at least the number of copies of THI5 family genes indicated, possibly more. The full ancestry of S. cerevisiae strain S288C, denoted by an asterisk, is described by Mortimer & Johnston (1986).

Analysis of a thiamin gene family in yeast

Saccharomyces bayanus, Saccharomyces paradoxus, Saccharomyces pastorianus and three newly identified species, S. mikatae, Saccharomyces kudriavzevii and Saccharomyces cariocanus (Naumov et al., 2000). Southern analysis showed that all these yeasts possess multiple copies of THI5. Yeasts classified outside of this subgroup, including the Saccharomyces sensu lato species and several Kluyveromyces species, possess either one or no copies of a THI5 homologue. To eliminate the possibility that our failure to detect positively hybridizing signals in some species was because of sequence divergence between the S. cerevisiae-derived probe and the Kluyveromyces genomic DNAs, we also used as probe the THI5 gene of K. lactis; the results obtained with the original S. cerevisiae DNA probe were confirmed. Therefore, we conclude that the existence of THI5 as a gene family is exclusive to those yeasts of the Saccharomyces sensu stricto complex, and some yeasts of both genera contain no THI5 gene at all.

### Gene order conservation in the Saccharomyces sensu stricto complex

Although THI5 is present as a gene family throughout the Saccharomyces sensu stricto group, the absolute copy number varies, even between different strains of the same species. However, we decided it would be interesting to ask whether the chromosomal environments of THI5 homologues in the sensu stricto yeasts are the same as those of S. cerevisiae S288C.

By using a PCR approach based on degenerate primers, gene order conservation between THI5 and its adjacent genes was investigated in the sensu stricto strains S. cerevisiae CBS 1171, S. bayanus, S. pastorianus, S. paradoxus and S. mikatae in comparison to S. cerevisiae S288C. The primers were designed to amplify the intergenic regions between SNZ-THI5/12 and THI5/12-AAD by annealing to conserved sequences within the two genes. All strains gave SNZ-THI5/12 products equal in size to that derived from S288C (Fig. 4), implying a conservation of gene order. In the case of the THI5/12-AAD analysis, all except S. bayanus gave products of similar size. The S. bayanus exception might be a consequence of DNA polymorphisms within the primer-binding region or it could be due to this particular yeast having no AAD gene downstream of THI5/12. Even so, the detection of amplified DNA from these two gene intervals in these strains implies synteny between the chromosomes of the sensu stricto group. Further evidence of a common genome environment around the THI5/12 genes of the sensu stricto group is documented within the 'Synteny Viewer' on the Saccharomyces Genome Database web site (http://genome-www.stanford.edu/Saccharomyces/), part of which has been published as a survey of several Saccharomyces species belonging to this complex (Cliften et al., 2001).

Naturally occurring thiamin auxotrophy may be due solely to the absence of a THI5 homologue

A number of the non-sensu stricto yeasts that were studied were found not to contain a THI5 homologue (Fig. 3).

---

**Fig. 4.** Degenerate PCR analysis to determine SNZ-THI5-AAD gene order conservation in yeasts of the Saccharomyces sensu stricto complex. (Top) Map, drawn to scale, showing the relative locations and orientations of the SNZ, THI5 and AAD genes in the sequenced genome of S. cerevisiae S288c, positions of PCR primers used to investigate conservation of gene order and the sizes of these products. (Bottom) Agarose gel of the PCR products, using genomic DNAs as templates, from members of the Saccharomyces sensu stricto complex. Lanes: 1, S288C-a; 2, CBS 1171; 3, S. bayanus; 4, S. paradoxus; 5, S. cariocanus; 6, S. pastorianus; 7, S. mikatae. For each of the two PCRs, a Saccharomyces sensu lato member, Saccharomyces exiguus, was used as a negative control. No bands were visualized for this yeast (data not shown). The strategy for primer selection is described in Methods.

Based upon the phenotype of a Schizosaccharomyces pombe nmt1-negative mutant strain, the absence of a THI5 homologue should result in thiamin auxotrophy. Each of the yeasts Saccharomyces servazzii, Saccharomyces castelli, Saccharomyces unisporus and Kluyveromyces polysporus was tested for growth on medium lacking thiamin (Table 4). All

**Table 4.** Precursor requirements of yeasts that exhibit thiamin auxotrophy

Growth was scored on Wickerham's medium lacking thiamin (no addition), or on the same medium supplemented with thiamin (+Thi) or one of the precursors HMP (+HMP) or HET (+HET). +, Growth after 5 days incubation; –, no growth after 5 days incubation.

| Species           | No addition | +Thi | +HET | +HMP |
|-------------------|-------------|------|------|------|
| S. cerevisiae S288C | +          | +    | +    | +    |
| S. servazzii      | –          | +    | –    | +    |
| S. castelli       | –          | +    | –    | +    |
| S. unisporus      | –          | +    | –    | +    |
| K. polysporus     | –          | +    | –    | +    |

http://mic.sgmjournals.org

R. Wightman and P. A. Meacock

exhibited thiamin auxotrophy rescuable by supplementation of the medium with thiamin or the precursor HMP. The presence or absence of the other precursor, HET, had no effect upon this phenotype. Therefore, these yeasts are strictly HMP auxotrophs and they must contain all other enzymes and genes of the thiamin biosynthetic pathway. Consistent with these conclusions, we have found a gene encoding a HET biosynthetic enzyme, THI4, to be present in these species (R. Wightman & P. A. Meacock, unpublished data). Therefore, the HMP auxotrophy of these natural isolates could be solely due to the lack of a THI5 homologue.

### Construction of all combinations of null alleles of the S. cerevisiae THI5 gene family

To learn more about the function and redundancy of the THI5 gene family, we carried out a systematic deletion of the gene set coupled with a phenotypic analysis of the resulting mutant strains. The deletion of the four members of the THI5 gene family in all 15 possible single, double, triple and quadruple combinations was performed in the S. cerevisiae laboratory strain BY4705, which contains completely deleted alleles of seven of the most commonly used nutritional marker genes (Brachmann et al., 1998). This makes BY4705 a good choice for the deletion of members of gene families using different selectable markers. Furthermore, because these mutant alleles are total deletions there is no undesirable marker gene conversion on transformation with gene disruption constructs targeted to other loci. The complete replacement of each THI5 family member with a different selectable nutritional marker gene used a PCR-generated targeting cassette, as described in Methods. The outcome of this replacement was a total gene deletion with the exception of 40 bp at each terminus. As the THI5 gene family sequences are highly conserved at the DNA level, it was not possible to target a particular member specifically, thus the cassette could integrate at any of the four loci.

Mutants with deleted alleles were detected by Southern blot hybridization using the THI5 cDNA as probe. Strain BY4705 gives rise to four positively hybridizing XhoI fragments of different sizes; these are 14·0 kb for THI11, 10·2 kb for THI12, 7·0 kb for THI13 and 3·8 kb for THI5. Thus, after each round of gene disruption, transformants were scored for the presence/absence of each of the four genes (data not shown). A genealogical tree (Fig. 5) describes the routes to the construction of all the mutant strains. The genotype of the RWY15 strain containing deletions of all four members was confirmed by a PCR approach. This made use of primers, four marker-gene-specific and one common, that amplified the DNA between each of the individual integrated disruption cassettes and a conserved region of the four promoters (Fig. 6). A second quadruple mutant strain, RWY16, was constructed using the kanMX4 cassette, in order to allow for the stable maintenance of URA3+ vectors.

---

**Fig. 5. Genealogy of the RWY mutant strains. Each solid line indicates the replacement of the specified gene. Asterisk, the full ancestry of BY4705 is described by Brachmann et al. (1998).**

![Genealogy Diagram](https://example.com/genealogy-diagram.png)

---

**Fig. 6. A PCR genealogy of strain RWY15. Four separate PCRs were performed for each mutant strain and the products were pooled and electrophoresed together. Lanes: 1, RWY2; 2, RWY8; 3, RWY13; 4, RWY14; 5, λ HindIII size marker. Each mutant allele can be detected as a differently sized fragment. UPSTHI5 is the upstream primer common to all four promoters. The second primer is unique to each marker gene.**

![PCR Genealogy](https://example.com/pcr-genealogy.png)

Analysis of a thiamin gene family in yeast

---

### Phenotypic analysis of the S. cerevisiae mutant strains

The growth of the triple mutant strains (RWY11–RWY14) and the quadruple deletion of RWY15 were compared on Wickerham's glucose medium supplemented with the vitamins thiamin and pyridoxine singly or together, or with the HMP precursor. On medium containing thiamin and pyridoxine, all the triple mutants and RWY15 showed strong growth equal to that of the BY4705 parental strain (Fig. 7a). On medium supplemented with pyridoxine alone (Fig. 7b), the triple mutants RWY11 (which contains just *THI13*), RWY12 (*THI11*), RWY13 (*THI5*) and RWY14 (*THI12*) all exhibited identical growth to BY4705. However, the quadruple mutant RWY15 showed minimal growth indicative of thiamin auxotrophy. Therefore, we conclude that each of the four genes can function in thiamin biosynthesis. The addition of HMP to this medium (Fig. 7c) resulted in growth of RWY15 comparable to that of the triple mutants and BY4705, indicating that the four gene products all function in the biosynthesis of the HMP moiety. Growth, albeit somewhat reduced, was detected for all strains on Wickerham's medium without added pyridoxine but supplemented with thiamin (Fig. 7d). On this medium growth of strains RWY11 and RWY12, which contain only functional copies of *THI13* and *THI11*, respectively, was slightly poorer than that of RWY13 and RWY14, in which the *THI5* and *THI12* genes were intact.

Minimal growth was exhibited by RWY15 in the absence of both pyridoxine and thiamin, with normal growth on medium supplemented only with HMP (data not shown). Taken together, these data show that Thi5p, Thil1p, Thil2p and Thil3p are essentially redundant in terms of HMP biosynthesis and that these enzymes function in the formation of this precursor from pyridoxine. The reduced growth of all strains, including the parental strain BY4705, on medium supplemented with thiamin but lacking pyridoxine suggests that under these conditions production of

---

**Fig. 7.** Phenotypic analysis of the triple mutants RWY11 to RWY14 and the quadruple *THI5* mutant RWY15. Cells were grown overnight in liquid CSD medium and harvested at approximately \(1 \times 10^8\) cells ml\(^{-1}\). Aliquots (5 μl) of 1/10 serial dilutions beginning with \(1 \times 10^8\) cells ml\(^{-1}\) were spotted onto Wickerham's SCM agar medium supplemented as indicated with the vitamins pyridoxine (Pyr) and thiamin (Thi) and the HMP precursor. All agar plates were incubated at 28°C for 2 days. (a) Medium + Pyr + Thi; (b) medium + Pyr – Thi; (c) medium + Pyr – Thi + HMP; (d) medium – Pyr + Thi. The arrow on (b) indicates the positions of the RWY15 inoculations.

R. Wightman and P.A. Meacock

vitamin B6 may be limited. This growth inhibition by thiamin in pyridoxine-free medium is a phenomenon shared by S. cerevisiae and other sensu stricto yeasts (Kamihara & Nakamura, 1982) and probably arises from the repression of the SNO2/3 and SNZ2/3 genes involved in pyridoxine biosynthesis by thiamin (L. Marsh, R. Wightman & P.A. Meacock, unpublished data; Rodriguez-Navarro et al., 2002). The slight growth differences between RWY11 and RWY12 versus RWY13 and RWY14 might reflect subtle differences in gene expression levels or efficiencies of the four iso-enzymes.

Regulation of THI5, THI11, THI12 and THI13 by thiamin and its precursors

Slot-blot RNA hybridization was used to investigate the expression of each of the THI5 genes in response to exogenous thiamin and its precursors. Total RNA was isolated from BY4705 and each of the triple mutants (RWY11–RWY14) after growth to late exponential phase in Wickerham's medium supplemented with thiamin, HET and/or HMP. Equal amounts of each RNA sample were applied to two membranes that were hybridized, one with a THI5 probe and the other with an ACT1 probe. The intensity of each THI5-hybridizing RNA signal was recorded after normalization for loading differences using the intensity levels of the ACT1 signals (Table 5). For each yeast strain, the intensities of the hybridization signals have been expressed as a percentage relative to the RNA sample isolated from medium without added thiamin. The actual hybridization signals, as measured on a PhosphorImager, from strains grown in thiamin-deficient medium are shown (Table 5).

As expected, strains containing only one of the THI5 family genes gave a much reduced hybridization signal indicating a lower mRNA content (Table 5, bold). For those samples isolated from cultures grown in thiamin-supplemented medium, basal expression levels did not exceed 7% of maximal expression. Similar low levels were also seen in cultures on medium containing both the HMP and HET precursors.

Differences in transcript levels between the four genes were seen for samples from medium supplemented with only HMP. The THI11 (RWY12) and THI13 (RWY11) genes both exhibited the low levels of transcription described above, indicating that these genes were almost fully repressed by the presence of HMP. In contrast, THI5 (RWY13) and THI12 (RWY14) yielded transcript levels that were approximately 20% of their maximum and so were not fully repressed by HMP (Table 5). These differences in patterns of gene expression mirror the type of promoters that are found upstream of these genes. Both THI11 and THI13 possess 100% DNA sequence identity for approximately 1 kb upstream of their ATG translation start site but only 60% identity to the equivalent region upstream of THI5 and THI12. Within the 1 kb sequence upstream of THI5 and THI12, there are differences at just eight nucleotide positions. As the Thi5p isozymes have been shown to be involved in HMP biosynthesis, it was surprising to see a noticeable effect of HET upon mRNA levels of members, most notably THI13. This suggests that the regulatory circuits between HET and HMP biosynthesis are linked.

HMP production during anaerobiosis

It has been suggested by Tanaka et al. (2000) that S. cerevisiae uses an alternative route for the formation of HMP when propagated under anaerobic conditions. Moreover, this anaerobic pathway does not proceed via pyridoxine. With this in mind, the effects of aerobic and anaerobic conditions

Table 5. Medium-dependent gene expression of the THI5 gene family

Data shown are from slot-blot hybridization of RNAs isolated from strains BY4705, RWY11, RWY12, RWY13 and RWY14 against a THI5 probe. The experiment was performed in duplicate. For each strain, the results are shown as percentages relative to RNA levels in cells grown in unsupplemented Wickerham's medium (-Thi). Data shown in bold are total mRNA transcript levels from cells grown in unsupplemented medium (-Thi) as quantified by PhosphorImager analysis.

| Medium supplement | Relative levels of THI5-hybridizing mRNAs |
|--------------------|-----------------------------------------|
|                    | BY4705                                  | RWY11 (THI13+)                          | RWY12 (THI11+)                          | RWY13 (THI5+)                           | RWY14 (THI12+)                          |
| +Thi              | 7                                       | 7                                      | 4                                      | 5                                      | 4                                      |
| –Thi              | 100                                     | 100                                    | 100                                    | 100                                    | 100                                    |
| +HMP              | 27                                      | 8                                      | 8                                      | 21                                     | 21                                     |
| +HET              | 60                                      | 40                                     | 82                                     | 98                                     | 82                                     |
| +HMP+HET          | 4                                       | 4                                      | 6                                      | 4                                      | 2                                      |
| RNA level         | 128538                                  | 59977                                  | 78082                                  | 62938                                  | 86064                                  |

1456                                                                                                          Microbiology 149

on the growth of S. cerevisiae strains S288C, BY4705 and RWY15 were investigated with regard to thiamin biosynthesis. The strains were inoculated, in duplicate, onto Wickerham's glucose agar medium supplemented with or without thiamin, or with HMP or HET replacing the vitamin. Both sets of plates were incubated at the same temperature with one set placed under anaerobic conditions and the other under aerobic conditions; growth was monitored after 7 days incubation (Fig. 8). On medium with added thiamin, all three strains displayed similar

**Aerobic**                                                                 **Anaerobic**

(a)

(b)

(c)

(d)

Fig. 8. Phenotypic investigation of the aerobic/anaerobic biosynthesis of thiamin and its precursors. Strains S288C, BY4705 and RWY15 were grown overnight in liquid YPD medium. Cells were washed and diluted to an OD₆₀₀ value of 0·25. Aliquots (5 μl) of 1/5 serial dilutions, made in sterile water, were spotted onto two sets of plates containing Wickerham's SCM agar glucose medium supplemented as indicated with thiamin (Thi), HET or HMP. (a) +Thi; (b) –Thi; (c) +HET; (d) +HMP. One set of plates was incubated under anaerobic conditions, the other was incubated under aerobic conditions.

growth in both aerobic and anaerobic conditions (Fig. 8a). Without added thiamin and in an aerobic environment, the quadruple mutant strain RWY15 exhibited the expected minimal growth due to the deficiencies in HMP formation described previously (Fig. 8b). Normal growth was seen for the other strains. However, when incubated on the same medium in an anaerobic environment, all three strains showed the same severely restricted growth as seen for aerobic RWY15. This phenotype was not rescued by supplementation of the medium with HET (Fig. 8c). Instead, normal anaerobic growth of all strains was restored by HMP (Fig. 8d). All the triple mutants (RWY11–RWY14) exhibited identical growth to each other and to BY4705 under the conditions tested (data not shown). Taken together, these data confirm that under anaerobic conditions even strains that contain functional THI5 family genes only use this alternative pathway to produce the HMP for thiamin, and imply that the Thi5p-catalysed pathway includes an obligatory oxidative step that cannot be accomplished under anaerobic conditions.

## DISCUSSION

In S. cerevisiae, the four members of the THI5 gene family are distributed among other gene families at the ends of different chromosomes. Our analyses have shown that this gene family state, as well as the conserved subtelomeric gene order, is unique to those yeasts classified within the Saccharomyces sensu stricto complex, even though the absolute gene copy number may vary between two and five, as in the strains tested here. Outside the sensu stricto complex, other yeasts of the genera Saccharomyces and Kluyveromyces possess just one or no copies of a THI5 homologue. Therefore, this multicopy state arose after divergence of the sensu stricto complex from the other hemiascomycetes, and has probably been maintained by the demands of metabolic activities unique to this group. The genetic basis of thiamin auxotrophy in the other yeasts surveyed has been found to be due solely to a defect in HMP precursor biosynthesis and may arise simply from lack of a Thi5 enzyme.

Through the application of a PCR-mediated disruption method, S. cerevisiae strains that cover all 15 combinations of deletions of the THI5 gene family have been constructed. Included in this set of deletants is a subset of strains that possess just one of the four members (RWY11–RWY14) plus strains that contain deletions of all the genes (RWY15, RWY16). Each subset of deletion mutants, although differing in genotype, has the same overall phenotype (e.g. triple mutants are all TRP⁺ HIS⁺ LEU⁺). This will be useful for carrying out effective competition experiments in a single culture.

Deletion of all four THI5 genes in a single strain resulted in severely retarded growth on medium lacking thiamin. Vigorous growth of this strain was dependent upon the presence of thiamin or its precursor HMP but not

R. Wightman and P. A. Meacock

pyridoxine, showing that all isozymes are involved in the production of HMP from pyridoxine. The presence of any one of the genes *THI5*, *THI11*, *THI12* or *THI13* conferred apparently normal thiamin prototrophic growth, showing that this gene family is functionally redundant for HMP formation.

Fig. 9 shows how these genes might feature in HMP formation from the metabolite 5-phosphoribosyl pyrophosphate. The only other known enzymes in this pathway are the hydroxymethylpyrimidine phosphate kinases Thi20p and Thi21p (Llorente *et al.*, 1999), and the glutamine amidotransferase activity catalysed by Ade4p (Mantsala & Zalkin, 1984). As Ade4p is feedback-inhibited by the products of purine biosynthesis, another glutamine amidotransferase must exist to produce phosphoribosylamine for pyridoxine and thiamin biosynthesis. The SNO and SNZ genes, which are also represented by multicopy gene families with some members positioned adjacent to *THI5* gene family members, are potential candidates for this role since multiple null mutants display pyridoxine auxotrophy (L. Marsh, R. Wightman & P. A. Meacock, unpublished data; Rodríguez-Navarro *et al.*, 2002).

The restricted growth on thiamin-depleted medium, exhibited by the quadruple mutant RWY15, is also shared by wild-type strains of *S. cerevisiae* when grown on the same medium during anaerobiosis. This limited growth must be due to the inefficient production of HMP since it is remedied by the addition of this compound. The alternative HMP biosynthetic pathway that operates during anaerobiosis is likely to be that described by Tanaka *et al.* (2000) which does not proceed through pyridoxine as an intermediate. Our results show that this pathway is inefficient and independent of any of the Thi5p isozymes. Moreover, the residual growth of strain RWY15 under aerobic conditions suggests that this less efficient pathway is constitutive

and operates alongside the main pyridoxine pathway even when oxygen is present.

The existence of two aerobic biosynthetic pathways is consistent with the observations of Grue-Sørensen *et al.* (1986), who examined the incorporation of \[^{14}C\]formate into the pyrimidine ring of thiamin and found the radiolabel to be present at two sites, C-2 and C-4, within the HMP moiety. Moreover, the radiolabel was not uniformly distributed between the two atoms; 70% was located at C-4 whilst only 20% was located at C-2. This observation can only be explained by the presence of two independent pathways where \[^{14}C\]formate enters C-4 by a more efficient route and C-2 by a less efficient route. The C-4 route represents the pathway that occurs via pyridoxal/THI5 as described in Fig. 9.

Why then should the *Saccharomyces sensu stricto* group require the *THI5* gene in multiple copies? Presumably, there must be either a strong metabolic need to produce large amounts of the pathway end-product, ThdP, or one of the intermediates, or the proteins themselves fulfil other functions needed when cells are depleted of thiamin; expression of these genes is repressed in thiamin-replete conditions. This group of yeasts are identified by their 'petite-positive' character and ability to ferment a variety of sugars, all of which are metabolized via glycolysis to pyruvate and thence either by anaerobic fermentation to ethanol or by aerobic metabolism to produce biomass.

It is well documented that multiple gene copies allow for rapid and abundant synthesis of their cognate RNAs and proteins; classic examples are the genes encoding rRNAs and proteins, and histone proteins in various organisms. Consistent with this, examination of *THI5* mRNA levels in the triple mutant *S. cerevisiae* strains showed that no single gene is able to generate the maximal level of expression seen


![Diagram](attachment:diagram.png)

Fig. 9. Proposed scheme of hydroxymethylpyrimidine diphosphate (HMP-PP) biosynthesis. The scheme is compiled from published evidence. Gene symbols are denoted in italics. References to pathway intermediates are: (1) Carter *et al.* (1997) and Hernando *et al.* (1998); (2) Mantsala & Zalkin (1984); (3) Tazuya *et al.* (1995); (4) Zeidler *et al.* (2002); (5) Tazuya *et al.* (1995); (6) Tazuya *et al.* (1989); (7) Llorente *et al.* (1999). PRPP, 5-phosphoribosyl pyrophosphate; PRA, phosphoribosylamine; HMP-P, hydroxymethylpyrimidine phosphate.
in the wild-type cell possessing all four genes. Additionally, multiple gene copies facilitate evolution through the functional divergence and differential gene expression of the individual members, in this case with respect to the production of the thiamin precursor HMP. The subtle differences that we have observed in the growth patterns of mutant strains (Fig. 7) and in expression levels of individual genes (Table 5) suggest that different family members might function with differing efficiencies under different growth conditions.

If the reason for multiple copies of THI5 is to respond to a high ThdP requirement then what is causing this in the sensu stricto yeasts? It is interesting to note that a great many of the amplified genes that are found near telomeres, including those in one of the duplicated THI5 blocks, encode proteins involved in sugar uptake and metabolism (e.g. HXT, SUC, MAL, MPH, ERR, FSP). These amplifications probably confer rapid uptake mechanisms leading to a greater glycolytic flux to pyruvate. The fate of pyruvate, towards either fermentation products or respiratory metabolism, requires ThdP-dependent reactions catalysed by pyruvate decarboxylase (Pdc) and pyruvate dehydrogenase (Pdh) respectively. It is perhaps relevant in this context to note that the commercial production of pyruvate makes use of thiamin auxotrophic yeasts (Li et al., 2001).

It is curious that the enzyme activities of thiamin biosynthesis encoded by multigene families are those of the HMP and pyridoxine branch (THI5, SNO, SNZ, THI20), whereas enzymes of the HET branch (THI4) and those involved in condensation of the precursors (THI6) and the downstream steps (THI80) are encoded by genes that are only present in single copy. *A priori* there should be no requirement for HMP in larger quantities than HET since thiamin is synthesized by the condensation of these precursors in equimolar quantities. If HMP is itself an intermediate in another metabolic pathway then this could explain the need for a greater HMP pool relative to HET. However, data from this study seem to contradict this model since the HMP auxotrophy of the quadruple mutant RWY15 can be rescued completely by the addition of just thiamin to the growth medium.

Alternatively, the selection pressure might be the intracellular pyridoxine pool. The depletion of pyridoxine or a derivative is known to occur by three processes: as an enzyme cofactor in amino acid biosynthesis, as a HMP and thiamin precursor, and as a free radical scavenging activity as shown by Ehrenshaft *et al.* (1999) and Osmani *et al.* (1999). The extra production of pyridoxine for HMP formation has been met through the amplification of the SNO and SNZ genes and by their regulation by ThdP. Co-amplification of THI5 in the same gene block would provide extra Thi5p to ensure that this extra pyridoxine (or intermediate) is channelled into thiamin biosynthesis for metabolism of sugar growth substrates, rather than being expended completely in free radical breakdown.

---

A solution to this somewhat paradoxical and enigmatic situation will only come from physiological studies of normal and mutant yeast cells growing in a carefully controlled situation, such as in chemostat cultures.

### ACKNOWLEDGEMENTS

We are grateful to Dr Kerry Byrne and Mr Leigh Marsh for encouragement and helpful discussions, and to Professor Ed Louis for the gift of yeast strains. This work was supported by BBSRC Research Grant 91/G08756 to P.A.M. and a BBSRC Research Committee Studentship to R.W.

### REFERENCES

Brachmann, C. B., Davies, A., Cost, G. J., Caputo, E., Li, J. C., Hieter, P. & Boeke, J. D. (1998). Designer deletion strains derived from Saccharomyces cerevisiae S288C: a useful set of strains and plasmids for PCR-mediated gene disruption and other applications. *Yeast* 14, 115–132.

Burrows, R. J., Byrne, K. L. & Meacock, P. A. (2000). Isolation and characterization of Saccharomyces cerevisiae mutants with derepressed thiamine gene expression. *Yeast* 16, 1497–1508.

Carlson, M. & Botstein, D. (1983). Organization of the SUC gene family in Saccharomyces. *Mol Cell Biol* 3, 351–359.

Carter, A. T., Beiche, F., Hove-Jensen, B., Narbad, A., Barker, P. J., Schweizer, L. M. & Schweizer, M. (1997). PRS1 is a key member of the gene family encoding phosphoribosylpyrophosphate synthetase in Saccharomyces cerevisiae. *Mol Gen Genet* 254, 148–156.

Chow, T. H., Sollitti, P. & Marmur, J. (1989). Structure of the multigene family of MAL loci in Saccharomyces. *Mol Gen Genet* 217, 60–69.

Church, G. M. & Gilbert, W. (1984). Genomic sequencing. *Proc Natl Acad Sci U S A* 81, 1991–1995.

Cliften, P. F., Hillier, L. W., Fulton, L., Graves, T., Miner, T., Gish, W. R., Waterston, R. H. & Johnston, M. (2001). Surveying Saccharomyces genomes to identify functional elements by comparative DNA sequence analysis. *Genome Res* 11, 1175–1186.

Day, R. E., Higgins, V. J., Rogers, P. J. & Dawes, I. W. (2002). Characterization of the putative maltose transporters encoded by YDL247w and YJR160c. *Yeast* 19, 1015–1027.

Delneri, D., Gardner, D. C., Bruschi, C. V. & Oliver, S. G. (1999a). Disruption of seven hypothetical aryl alcohol dehydrogenase genes from Saccharomyces cerevisiae and construction of a multiple knock-out strain. *Yeast* 15, 1681–1689.

Delneri, D., Gardner, D. C. & Oliver, S. G. (1999b). Analysis of the seven-member AAD gene set demonstrates that genetic redundancy in yeast may be more apparent than real. *Genetics* 153, 1591–1600.

Ehrenshaft, M., Bilski, P., Li, M. Y., Chignell, C. F. & Daub, M. E. (1999). A highly conserved sequence is a novel gene involved in de novo vitamin B6 biosynthesis. *Proc Natl Acad Sci U S A* 96, 9374–9378.

Geitz, R. D., Schiestl, R. H., Willems, A. R. & Woods, R. A. (1995). Studies on the transformation of intact yeast cells by the LiAc/ss-DNA/PEG procedure. *Yeast* 11, 355–360.

Goffeau, A., Barrell, B. G., Bussey, H. & 13 other authors (1996). Life with 6000 genes. *Science* 274, 546–552.

Gonzalez, E., Fernandez, M. R., Larroy, C., Sola, L., Pericas, M. A., Pares, X. & Biosca, J. A. (2000). Characterization
of a (2R,3R)-2,3-butanediol dehydrogenase as the *Saccharomyces cerevisiae* YAL060W gene product. Disruption and induction of the gene. *J Biol Chem* 275, 35876–35885.

Goodey, A. R. & Tubb, R. S. (1982). Genetic and biochemical analysis of the ability of *Saccharomyces cerevisiae* to decarboxylate cinnamic acids. *J Gen Microbiol* 128, 2615–2620.

Grue-Sorensen, G., White, R. L. & Spenser, I. D. (1986). Thiamin biosynthesis in *Saccharomyces cerevisiae* – origin of the pyrimidine unit. *J Am Chem Soc* 108, 146–158.

Hernando, Y., Parr, A. & Schweizer, M. (1998). *PRS5*, the fifth member of the phosphoribosyl pyrophosphate synthetase gene family in *Saccharomyces cerevisiae*, is essential for cell viability in the absence of either *PRS1* or *PRS3*. *J Bacteriol* 180, 6404–6407.

Hohmann, S. & Meacock, P. A. (1998). Thiamin metabolism and thiamine diphosphate-dependent enzymes in the yeast *Saccharomyces cerevisiae*: genetics and regulation. *Biochim Biophys Acta* 1385, 201–219.

Johnston, J. R. & Mortimer, R. K. (1984). Pedigree of the yeast genetics stock centre wild-type strains S288c and X2180 of *S. cerevisiae*. *Heredity* 52, 459–469.

Kamihara, T. & Nakamura, I. (1982). Regulation of respiration and its related metabolism by vitamin B1 and vitamin B6 in *Saccharomyces cerevisiae*. *Adv Biochem Eng Biotech* 29, 35–82.

Li, Y., Chen, J., Lun, S. Y. & Rui, X. S. (2001). Efficient pyruvate production by a multi-vitamin auxotroph of *Torulopsis glabrata*: key role and optimization of vitamin levels. *Appl Microbiol Biotechnol* 55, 680–685.

Llorente, B., Fairhead, C. & Dujon, B. (1999). Genetic redundancy and gene fusion in the genome of the baker’s yeast *Saccharomyces cerevisiae*: functional characterization of a three-member gene family involved in the thiamine biosynthetic pathway. *Mol Microbiol* 32, 1140–1152.

Mantsala, P. & Zalkin, H. (1984). Glutamine nucleotide sequence of *Saccharomyces cerevisiae* *ADE4* encoding phosphoribosylpyrophosphate amidotransferase. *J Biol Chem* 259, 8478–8484.

Maundrell, K. (1990). *nmt1* of fission yeast – a highly transcribed gene completely repressed by thiamine. *J Biol Chem* 265, 10857–10864.

Maundrell, K. (1993). Thiamine-repressible expression vectors pREP and pRIP for fission yeast. *Gene* 123, 127–130.

Mortimer, R. K. & Johnston, J. R. (1986). Genealogy of principal strains of the Yeast Genetic Stock Center. *Genetics* 113, 35–43.

Muller, E. H., Richards, E. J., Norbeck, J., Byrne, K. L., Karlsson, K.-A., Pretorius, G. H. J., Meacock, P. A., Blomberg, A. & Hohmann, S. (1999). Thiamine repression and pyruvate decarboxylase autoregulation independently control the expression of the *Saccharomyces cerevisiae* *PDC5* gene. *FEBS Lett* 449, 245–250.

Naumov, G. I., Naumova, E. S. & Louis, E. J. (1995). Genetic mapping of the alpha-galactosidase *MEL* gene family on right and left telomeres of *Saccharomyces cerevisiae*. *Yeast* 11, 481–483.

Naumov, G. I., James, S. A., Naumova, E. S., Louis, E. J. & Roberts, I. N. (2000). Three new species in the *Saccharomyces sensu stricto* complex: *Saccharomyces cariocanus*, *Saccharomyces kudriavzevii* and *Saccharomyces mikatae*. *Int J Syst Evol Microbiol* 50, 1931–1942.

Nishimura, H., Kawasaki, Y., Kaneko, Y., Nosaka, K. & Iwashima, A. (1992a). Cloning and characteristics of a positive regulatory gene, *THI2* (*PHO6*), of thiamin biosynthesis in *Saccharomyces cerevisiae*. *FEBS Lett* 297, 155–158.

Nishimura, H., Kawasaki, Y., Kaneko, Y., Nosaka, K. & Iwashima, A. (1992b). A positive regulatory gene, *THI3*, is required for thiamine metabolism in *Saccharomyces cerevisiae*. *J Bacteriol* 174, 4701–4706.

Osmani, A. H., May, G. S. & Osmani, S. A. (1999). The extremely conserved *pyroA* gene of *Aspergillus nidulans* is required for pyridoxine synthesis and is required indirectly for resistance to photosensitizers. *J Biol Chem* 274, 23565–23569.

Padilla, P. A., Fuge, E. K., Crawford, M. E., Errett, A. & Werner-Washburne, M. (1998). The highly conserved, coregulated SNO and SNZ gene families in *Saccharomyces cerevisiae* respond to nutrient limitation. *J Bacteriol* 180, 5718–5726.

Praekelt, U. M. & Meacock, P. A. (1992). *MOL1*, a *Saccharomyces cerevisiae* gene that is highly expressed in early stationary phase during growth on molasses. *Yeast* 8, 699–710.

Praekelt, U. M., Byrne, K. L. & Meacock, P. A. (1994). Regulation of *THI4* (*MOL1*), a thiamine biosynthetic gene of *Saccharomyces cerevisiae*. *Yeast* 10, 481–490.

Rodríguez-Navarro, S., Llorente, B., Rodríguez-Manzaneque, M. T. & 7 other authors (2002). Functional analysis of yeast gene families involved in metabolism of vitamins B₁ and B₆. *Yeast* 19, 1261–1276.

Rose, M. D., Winston, F. & Hieter, P. (1990). *Methods in Yeast Genetics*. CSHL Press.

Schmitt, M. E., Brown, T. A. & Trumpower, B. L. (1990). A rapid and simple method for preparation of RNA from *Saccharomyces cerevisiae*. *Nucleic Acids Res* 18, 3091–3092.

Schweingruber, A. M., Dlugonski, J., Edenharter, E. & Schweingruber, M. E. (1991). Thiamine in *Schizosaccharomyces pombe*: dephosphorylation, intracellular pool, biosynthesis and transport. *Curr Genet* 19, 249–254.

Tanaka, K., Tazuya, K., Yamada, K. & Kumaoka, H. (2000). Biosynthesis of thiamin under anaerobic conditions in *Saccharomyces cerevisiae*. *Biol Pharm Bull* 23, 108–111.

Tazuya, K., Yamada, K. & Kumaoka, H. (1989). Incorporation of histidine into the pyrimidine moiety of thiamin in *Saccharomyces cerevisiae*. *Biochim Biophys Acta* 990, 73–79.

Tazuya, K., Yamada, K. & Kumaoka, H. (1993). Pyridoxine is a precursor of the pyrimidine moiety of thiamin in *Saccharomyces cerevisiae*. *Biochem Mol Biol Int* 30, 893–899.

Tazuya, K., Adachi, Y., Masuda, K., Yamada, K. & Kumaoka, H. (1995). Origin of the nitrogen atom of pyridoxine in *Saccharomyces cerevisiae*. *Biochim Biophys Acta* 1244, 113–116.

Wickerham, L. J. (1951). Taxonomy of yeast. *U S Dept Agric Tech Bull* 1029, 11–56.

Wieczorke, R., Krampe, S., Weierstall, T., Freidel, K., Hollenberg, C. P. & Boles, E. (1999). Concurrent knock-out of at least 20 transporter genes is required to block uptake of hexoses in *Saccharomyces cerevisiae*. *FEBS Lett* 464, 123–128.

Zeidler, J., Ullah, N., Gupta, R. N., Pauloski, R. M., Sayer, B. G. & Spenser, I. D. (2002). 2′-Hydroxypyridoxol, a biosynthetic precursor of vitamins B₆ and B₁ in yeast. *J Am Chem Soc* 124, 4542–4543.
