
Plant Molecular Biology 51: 925–948, 2003.
© 2003 Kluwer Academic Publishers. Printed in the Netherlands.

# Genome organization in *Arabidopsis thaliana*: a survey for genes involved in isoprenoid and chlorophyll metabolism

**B. Markus Lange**\* and **Majid Ghassemanian**

Torrey Mesa Research Institute, Syngenta Research & Technology, 3115 Merryfield Row, San Diego, CA 92121, USA (*author for correspondence; e-mail mark.lange@syngenta.com*)

Received 30 November 2001; accepted in revised form 28 September 2002

**Key words:** Arabidopsis, chlorophyll, deoxyxylulose, isoprenoid, methylerythritol, mevalonate

## Abstract

The isoprenoid biosynthetic pathway provides intermediates for the synthesis of a multitude of natural products which serve numerous biochemical functions in plants: sterols (isoprenoids with a C30 backbone) are essential components of membranes; carotenoids (C40) and chlorophylls (which contain a C20 isoprenoid side-chain) act as photosynthetic pigments; plastoquinone, phylloquinone and ubiquinone (all of which contain long isoprenoid side-chains) participate in electron transport chains; gibberellins (C20), brassinosteroids (C30) and abscisic acid (C15) are phytohormones derived from isoprenoid intermediates; prenylation of proteins (with C15 or C20 isoprenoid moieties) may mediate subcellular targeting and regulation of activity; and several monoterpenes (C10), sesquiterpenes (C15) and diterpenes (C20) have been demonstrated to be involved in plant defense. Here we present a comprehensive analysis of genes coding for enzymes involved in the metabolism of isoprenoid-derived compounds in *Arabidopsis thaliana*. By combining homology and sequence motif searches with knowledge regarding the phylogenetic distribution of pathways of isoprenoid metabolism across species, candidate genes for these pathways in *A. thaliana* were obtained. A detailed analysis of the vicinity of chromosome loci for genes of isoprenoid metabolism in *A. thaliana* provided evidence for the clustering of genes involved in common pathways. Multiple sequence alignments were used to estimate the number of genes in gene families and sequence relationship trees were utilized to classify their individual members. The integration of all these datasets allows the generation of a knowledge-based metabolic map of isoprenoid metabolic pathways in *A. thaliana* and provides a substantial improvement of the currently available gene annotation.

**Abbreviations:** AACT, acetoacetyl-CoA thiolase; ABA, abscisic acid; CMK, 4-(cytidine 5′-diphospho)-2-C-methyl-D-erythritol kinase; DHNA, 1,4-dihydroxy-2-naphthoate; DMAPP, dimethylallyl diphosphate; DXPS, 1-deoxy-D-xylulose 5-phosphate synthase; DXR, 1-deoxy-D-xylulose 5-phosphate reductoisomerase; ER, endoplasmic reticulum; FCC, fluorescent chlorophyll catabolite; FPPS, farnesyl diphosphate synthase; FTase, protein farnesyltransferase; GGPPS, geranylgeranyl diphosphate synthase; GGR, geranylgeranyl reductase; GGTTase, protein geranylgeranyltransferase; GPPS, geranyl diphosphate synthase; HMG, 3-hydroxy-3-methylglutaryl; HPPD, p-hydroxyphenylpyruvate; IPP, isopentenyl diphosphate; IPPI, isopentenyl diphosphate:dimethylallyl diphosphate isomerase; MCT, 2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase; MECPS, 2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase; OSB, O-succinylbenzoate; PPPS, polyprenyl diphosphate synthase; RCC, red chlorophyll catabolite; SHCHC, 2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate

## Introduction

Conceptually, isoprenoid biosynthesis can be divided into four phases (Figure 1). Phase I includes the formation of isopentenyl diphosphate (IPP) and dimethylallyl diphosphate (DMAPP). In plants, two separate pathways are utilized for the synthesis of these universal C5 intermediates (Lange *et al.*, 2000): the

cytosolic mevalonate (MVA) pathway provides the precursors for the synthesis of certain sesquiterpenes, sterols (Nes and Venkatramesh, 1999) and for the side-chain of ubiquinone (Disch *et al.*, 1998), whereas the plastidial MVA-independent pathway is employed for the synthesis of precursors involved in the biosynthesis of isoprene, monoterpenes, certain sesquiterpenes, diterpenes, carotenoids, as well as the side-chains of tocopherols, phylloquinone and chlorophylls (Eisenreich *et al.*, 2001). A cooperation of both pathways has been reported for certain sesquiterpenes in chamomile (Adam *et al.*, 1999) and lima bean (Piel *et al.*, 1998). Phase II begins with the isomerization of IPP to DMAPP, the latter of which then serves as the reactive starter molecule for subsequent condensation reactions with IPP to form geranyl diphosphate (GPP, C10), farnesyl diphosphate (FPP, C15), geranylgeranyl diphosphate (GGPP, C20), squalene (C30), phytyl diphosphate (C40), phytoene (C40) and higher prenyl diphosphates (Ogura, 1999). In phase III, these prenyl diphosphates, or compounds directly derived from them, undergo a range of cyclization reactions to produce the parent skeletons of the different monoterpenes from GPP, sesquiterpenes from FPP, diterpenes from GGPP, triterpenes (sterols) from 2,3-epoxysqualene, and tetraterpenes (carotenoids) from lycopene. The cyclic parent compounds are then transformed in phase IV by redox, isomerization, substitution and conjugation reactions to yield the various isoprenoid end-products. With the recent completion of the *Arabidopsis thaliana* genome sequencing (Arabidopsis Genome Initiative, 2000), a comprehensive analysis of the structural organization of genes involved in the intertwined pathways leading to isoprenoids, chlorophylls, isoprenoid quinones and prenylated proteins in a model plant becomes feasible. In addition, extensive mutant collections (available at http://www.tmri.org/pages/collaborations/garlic_files/RequestInstructions.html and at http://www.jic.bbsrc.ac.uk/staff/michael-bevan/atis/Resources1.html) in combination with advances in transcription profiling of *A. thaliana* (Zhu and Wang, 2000) have set the stage for studying the function and the regulation of genes involved in isoprenoid metabolism.

**Materials and methods**

Keyword searches and sequence-based searches were used to generate a database representing all classes of genes and encoded enzymes known to be involved in isoprenoid metabolism.

**Figure 1.** Overview of isoprenoid biosynthesis in *A. thaliana*. The circled P denotes a phosphate moiety, dotted arrows indicate multiple steps, metabolic end products are underlined and the carbon number of the core structure of each metabolite class is given in parentheses. Known enzymes involved in isoprenoid metabolism are numbered according to the annotations given in Table 1.

L-Glutamate
↓ 32
L-Glutamyl-tRNA
↓ 33
Glutamate-1-semialdehyde
↓ 34
5-Aminolevulinate
↓ 35
Porphobilinogen
↓ 36
1-Hydroxymethylbilane
↓ 37
Uroporphyrinogen III
↓ 38
Coproporphyrinogen III
↓ 39
Protoporphyrinogen IX
↓ 40
Protoporphyrin IX
↓ 41
Mg-Protoporphyrin IX
↓ 42
Mg-Protoporphyrin IX monomethylester
↓ 43
Divinylprotochlorophyllide a
↓ 44
Chlorophyllide a
↓ 45
Chlorophyll a
↑ 48
↓ 46
Chlorophyll b
↑ 47

$p$-Hydroxyphenylpyruvate
↑ 69
Tyrosine
Nonaprenyl diphosphate
GGPP
FPP
GPP
Heme
IPP + DMAPP
O-Succinylbenzoate
O-Succinylbenzoyl-CoA
2-Succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate
1,4-Dihydroxy-2-naphthoate
Isochorismate
2-Phytol-1,4-naphthoquinol
Phylloquinol
Chorismate
Biliverdin IX
Phytochromobilin
Phytochrome
Pheophorbide a
RCC
Chlorophylls
R = H, chlorophyll a
R = CHO, chlorophyll b

2-Methyl-6-nonaprenylquinol
Plastoquinol
2-Methyl-6-phytylquinol
$\delta$-Tocopherol
2,3-Dimethyl-5-phytylquinol
$\gamma$-Tocopherol
$\beta$-Tocopherol
Carotenoids
7,9,9',7'-cis-Lycopene
all-trans-Lycopene
$\beta$-Carotene
$\alpha$-Carotene
Lutein
ent-Kaurene
Copalyl diphosphate
ent-Kaurenoic acid
GA₁₂-aldehyde
GA₁₂
GA₉
GA₂₀
Violaxanthin
Xanthoxin
ABA-aldehyde
Abscisic acid
Gibberellins
GA₁
GA₄
GA₈-catabolite
GA₃₄-catabolite

Figure 2. Metabolism of isoprenoids derived from plastidal pathways. The symbol for anabolic reactions is a solid arrow, whereas catabolic reactions are indicated by arrows with dotted lines. Known enzymes involved in isoprenoid metabolism are numbered according to the annotation given in Table 1 and question marks indicate steps for which an enzymatic activity has not yet been demonstrated.

| Gene product                                                                 | AGI number   | Full-length cDNA | Assignment     | Sub-cellular location |
|-----------------------------------------------------------------------------|--------------|-----------------|----------------|-----------------------|
| 1 l-deoxy-D-xylulose 5-phosphate synthase (3 sequences)                       |              |                 |                |                       |
| DXPS 1                                                                        | At3g21500    | Y14333          | by homology    | P                     |
| DXPS 2 (CLA 1)                                                                | At4g15560    | U27099          | functional     | P                     |
| DXPS 3                                                                        | At5g11380    |                 | by homology    | P                     |
| 2 l-deoxy-D-xylulose 5-phosphate reductoisomerase (DXR) (1 seq.)               | At5g62790    | Q9XFS9          | functional     | P                     |
| 3 2-C-methyl-D-erythritol 4-phosphate cytidyltransferase (MCT) (1 seq.)        | At2g02500    | AF230737        | functional     | P                     |
| 4 4-(cytidine 5'-diphospho) -2-C-methyl-D-erythritol kinase (CMK) (1 seq.)      | At2g26930    | AF288615        | functional     | P                     |
| 5 2-C-methyl-D-erythritol 2, 4-cyclodiphosphate synthase (MECPS) (1 seq.)       | At1g63970    | AF321531        | by homology    | P                     |
| 6 gcpE homologue (1 seq.)                                                    | At5g60600    |                 | by homology    | P                     |
| 7 lytB homologue (1 seq.)                                                    | At4g34350    | AY042877        | by homology    | P                     |
| 8 acetoacetyl-CoA thiolase (2 seq.)                                           |              |                 |                |                       |
| AACT 1                                                                        | At5g47720    |                 | by homology    | C                     |
| AACT 2                                                                        | At5g48230    | AY059736        | by homology    | C                     |
| 9 3-hydroxy-3-methylglutaryl-CoA synthase (HMGS) (1 seq.)                     | At4g11820    | P54873          | functional     | C                     |
| 10 3-hydroxy-3-methylglutaryl-CoA reductase (2 seq.)                          |              |                 |                |                       |
| HMGR 1                                                                        | At1g76490    | P14891          | functional     | C/ER                  |
| HMGR 2                                                                        | At2g17370    | P43256          | functional     | C/ER                  |
| 11 mevalonate kinase (MK) (1 seq.)                                            | At5g27450    | P46086          | functional     | C                     |
| 12 phosphomevalonate kinase (PMK) (1 seq.)                                    | At1g31910    |                 | by homology    | C                     |
| 13 mevalonate diphosphate decarboxylase (2 seq.)                              |              |                 |                |                       |
| MPDC 1                                                                        | At2g38700    | Y14325          | functional     | C                     |
| MPDC 2                                                                        | At3g54250    |                 | by homology    | C                     |
| 14 isopentenyl diphosphate: dimethylallyl diphosphate isomerase (2 seq.)       |              |                 |                |                       |
| IPP I                                                                         | At3g02780    | U49259          | functional     | P/C                   |
| IPP II                                                                        | At5g16440    | U47324          | functional     | P/C                   |
| 15 geranyl diphosphate synthase (GPPS) (1 seq.)                               | At2g34630    | Y17376          | functional     | P                     |
| 16 farnesyl diphosphate synthase (2 seq.)                                     |              |                 |                |                       |
| FPP S 1                                                                       | At4g17190    | Q43315          | functional     | C                     |
| FPP S 2                                                                       | At5g47770    | U80605          | functional     | C/M                   |
| 17 geranylgeranyl diphosphate synthase (12 seq.)                               |              |                 |                |                       |
| GGPPS 1 (GGPS 6)                                                              | At1g49530    | AB000835        | functional     | M                     |
| GGPPS 2                                                                       | At2g18620    |                 | by homology    | P                     |
| GGPPS 3 (GGPS 4)                                                              | At2g18640    | AAD12206        | functional     | C/ER                  |
| GGPPS 4 (GGPS 2)                                                              | At2g23800    | U44876          | functional     | C/ER                  |
| GGPPS 5                                                                       | At3g14510    |                 | by homology    | P                     |
| GGPPS 6                                                                       | At3g14530    |                 | by homology    | P                     |
| GGPPS 7 (GGPS 3)                                                              | At3g14550    | BAB02385        | functional     | P                     |
| GGPPS 8                                                                       | At3g20160    |                 | by homology    | P                     |
| GGPPS 9                                                                       | At3g29430    |                 | by homology    | P                     |
| GGPPS 10                                                                      | At3g32040    |                 | by homology    | P                     |
| GGPPS 11 (GGPPS 1)                                                            | At4g36810    | L25813          | functional     | P                     |
| GGPPS 12                                                                      | At4g38460    |                 | by homology    | P                     |

18 geranylgeranyl reductase (GGR) (1 seq.)          | At1g74470 | Y14044 | functional | P
19 phytoene synthase (PS) (1 seq.)                 | At5g17230 | L25812 | by homology | P
20 phytoene desaturase (PDS) (1 seq.)              | At4g14210 | Q07356 | by homology | P
21 ε-carotene desaturase (ZDS) (1 seq.)            | At3g04870 | Q38893 | by homology | P
22 carotenoid isomerase (2 seq.)                   |           |        |             |
CRT ISO 1                                         | At1g06820 |        | functional | P
CRT ISO 2                                         | At1g57770 |        | by homology | P
23 lycopene β-cyclase (LCYB) (1 seq.)              | At3g10230 | Q38933 | functional | P
24 lycopene ε-cyclase (LCYE) (1 seq.)              | At5g57030 | AF117257 | functional | P
25 carotenoid ε-hydroxylase                       |           | n.c.y. |             |
26 β-carotene hydroxylase (2 seq.)                |           |        |             |
CHY 1                                             | At4g25700 | AF125576 | functional | P
CHY 2                                             | At5g52570 |        | by homology | P
27 zeaxanthin epoxidase (ZEP) (1 seq.)             | At5g67030 | AF281655 | functional | P
28 violaxanthin de-epoxidase (VDE1) (1 seq.)       | At1g08550 | U44133 | functional | P
29 neoxanthin synthase existence in Arabidopsis questionable
30 epoxy carotenoid (neoxanthin) cleavage enzyme (7 seq.)
NC1                                                | At1g30100 |        | by homology | P
NC2                                                | At1g78390 |        | by homology | P
NC3                                                | At3g14440 | AY056255 | by homology | P
NC4                                                | At3g24220 |        | by homology | P
NC5                                                | At3g63520 | AJ005813 | by homology | P
NC6                                                | At4g18350 |        | by homology | P
NC7                                                | At4g19170 | AY056789 | by homology | P
31 abscisic aldehyde oxidase (1 seq.)              | At2g27150 | AC007154 | functional C | 
32 glutamyl tRNA synthetase (1 seq.)               | At5g26707 | AF067773 | functional P | 
33 glutamyl tRNA reductase (3 seq.)                |           |        |             |
HEMA 1                                            | At1g09940 | P49294 | functional | P
HEMA 2                                            | At1g58290 | P42804 | functional | P
HEMA 3                                            | At2g31250 |        | by homology | P
34 glutamate 1-semialdehyde aminotransferase (2 seq.)
GSA 1                                             | At3g48730 | Q42522 | by homology | P
GSA 2                                             | At5g63570 | P42799 | by homology | P
35 aminolevulinate dehydratase (2 seq.)
ALAD 1                                            | At1g44318 |        | by homology | P
ALAD 2                                            | At1g69740 | AF327428 | by homology | P
36 porphobilinogen deaminase (HEM3) (1 seq.)       | At5g08280 | Q43316 | functional | P
37 uroporphyrinogen synthase (1 seq.)              | At2g26540 | Ceres 33364 | by homology | P
38 uroporphyrinogen decarboxylase (2 seq.)
UPD 1                                             | At2g40490 | O22886 | by homology | P
UPD 2                                             | At3g14930 |        | by homology | P
39 coproporphyrinogen III oxidase (CPOX) (1 seq.)   | At1g03475 | AJ420796 | functional | P
40 protoporphyrinogen IX oxidase (2 seq.)
PPO 1                                             | At4g01690 | P55826 | functional | P
PPO 2                                             | At5g14220 |        | by homology | P

41Mg-protoporphyrinogen IX chelatase (3 seq.)
CLD 1                                      At1g08520 AF083555 functional P
CLD 2                                      At4g18480 P16127  functional P
CLD 3                                      At5g49530                      functional P
42Mg-protoporphyrinogen IX methyltransferase (PPMT) (1 seq.) At4g25080 AY034948 by homology P
43Mg-protoporphyrinogen IX monomethylester cyclase n.c.y.
44 protochlorophyllide reductase (EC 1.3.1.33) (3 seq.)
POR 1 (PORC; light-dependent)               At1g03630 O48741  functional P
POR 2 (PORB)                               At4g27440 P21218  functional P
POR 3 (PORA)                               At5g54190 Q42536  functional P
45 chlorophyll synthetase (1 seq.)          At3g51820 AY042821 functional P/En
46 chlorophyll a oxygenase (chlorophyll b synthase) (1 seq.) At1g44446 AF177200 functional P
47 chlorophyl b reductase                   n.c.y.
48 chlorophyllase (2 seq.)
CLH 1                                      At1g19670 AF021244 functional C/ER
CLH 2                                      At5g43860 AF134302 functional P/En
49Mg-dechelatase                           n.c.y.
50 pheophorbide a oxygenase                 n.c.y.
51 red chlorophyll catabolite reductase (1 seq.) At4g37000 AF326347 functional C
52 squalene synthase (2 seq.)
SQS 1                                      At4g34640 D29017  functional C
SQS 2                                      At4g34650 AF004396 by homology C
53 squalene monooxygenase (EC 1.14.99.7) (6 seq.)
SE1                                        At1g58440                      by homology C
SE2                                        At2g22830                      by homology C
SE3                                        At4g37760                      by homology C
SE4                                        At5g24140 O65403               by homology C
SE5                                        At5g24150 O65404               by homology C
SE6                                        At5g24160 O65402               by homology C
54 cycloartenol synthase (EC 5.4.99.8) (2 seq.)
CAS 1                                      At2g07050 P38605               functional C
CAS 2                                      At3g45130                      by homology C
55 cycloartenol C 24 methyltransferase (1 seq.) At5g13710 AF195648 by homology C
56 24-methylene cycloartenol C-4 methyl oxidase (1–2 sequences?)
candidate 1                                At4g12110                      by homology C
candidate 2                                At4g22750                      by homology C
57 cycloeucalenol cycloisomerase (1 seq.)   At5g50375 AF216756 functional C
58 sterol C 14 reductase (fackel) (1 seq.)  At3g50430 AF257178 functional C/ER
59 obtusifoliol C 14 methyl oxidase (CYP 51) (2 seq.)
CYP 51-1                                   At1g11680 AB014459 functional C
CYP 51-2                                   At2g17330                      by homology C
60C-8, 7 sterol isomerase (1 seq.)          At1g20050 AF030357 functional C
61 sterol C-methyltransferase (2 seq.)
SMT 2-1                                    At1g20330 X89867               functional C
SMT 2-2                                    At1g76090 U71400               functional C
62 sterol C-4 methyl oxidase (2 seq.)
S4MO-1                                     At1g07420 AF346734 functional C
S4MO-2                                     At2g29390 AF327853 functional C
63 sterol C 5-desaturase (2 seq.)
DWF 7-1                                    At3g02580 X90454               functional C
DWF 7-2                                    At3g02590                      by homology C

64 sterol Δ7 reductase (1 seq.)                      | At1g50430 | U49398  | functional | C
65 sterol C24-reductase (DWF1, diminuto) (1 seq.)    | At3g19820 | Q39085  | functional | C/ER
66 sterol C22-desaturase                           | n.c.y.
67 polyprenyl diphosphate synthase (2 seq.)         | At1g17050 |         | by homology | C/ER
PPDS 1                                            | At1g78510 |         | by homology | M
PPDS 2                                            |
68 dehydrolicol chol diphosphate synthase (3 seq.)  | At2g23410 | AJ277136 | functional | C/ER
DPPS 1                                            | At5g58770 |         | by homology | M
DPPS 2                                            | At5g58780 |         | by homology | C/ER
DPPS 3                                            |
69 p-hydroxyphenylpyruvate synthase                 | n.c.y.
70 p-hydroxyphenylpyruvate dioxygenase (HPPD) (1 seq.) | At1g06570 | P93836  | functional | C
71 homogentisate poly prenyltransferase (1 seq.)   | At3g11950 | BAB03104 | by homology | P/En(?)
72 C-methyltransferase (for candidates see also 79, additional candidates) | n.c.y.
73 isochorismate synthase (EC 5.4.99.6; entC) (2 seq.)
ICS 1                                             | At1g18870 |         | by homology | P(?)
ICS 2                                             | At1g74710 | AF344321 | functional | P
74 SHCHC synthase (menD) (1 seq.)                  | At1g68890 |         | by homology | P
75 OSB synthase (menC) (1 seq.)                    | At1g68900 |         | by homology | C
76 OSB-CoA ligase (menE) (1 seq.)                  | At3g48990 |         | by homology | P
77 DHNA synthase (1 seq.)                          | At1g60550 |         | by homology | C
78 DHNA phytyltransferase (menA) (1 seq.)          | At1g60600 |         | by homology | P/En(?)
79 C-methyltransferase (ubiE) (1–4 candidates)
very likely candidate                              | At1g23360 |         | by homology | P
additional candidate                               | At3g02770 | Q9M8R9  | by homology | C
additional candidate                               | At5g16450 | Q9FFE0  | by homology | C
additional candidate                               | At5g56260 | Q9FH13  | by homology | C
80 copalyl diphosphate synthase (GA1) (1 seq.)     | At4g02780 |         | functional | P
81 ent-kaurene synthase (GA2) (1 seq.)             | At1g79460 |         | functional | P
82 ent-kaurene oxidase (GA3) (1 seq.)              | At5g25900 |         | functional | C/ER
83 ent-kaurenoic acid oxidase (2 seq.)
KAO1                                              | At1g05160 | AF31850 | functional | C/ER
KAO2                                              | At2g32440 | AF318501| functional | C/ER
84 gibberellin 7-oxidase (2 seq.)
G7OX-1                                            | At3g11180 |         | by homology | C
G7OX2                                             | At5g05600 | Ceres 13012 | by homology | C
85 gibberellin 13-hydroxylase
86 gibberellin 20-oxidase (5 seq.)
GA5-1                                             | At1g44090 |         | by homology | C
GA5-2                                             | At1g60980 |         | by homology | C
GA5-3                                             | At4g25420 | X83379  | functional | C
GA5-4                                             | At5g07200 | X83381  | functional | C
GA5-5                                             | At5g51810 | X83380  | functional | C
87 gibberellin 3-hydroxylase (4 seq.)
GA4-1                                             | At1g15550 | L37126  | functional | C
GA4-2                                             | At1g80330 |         | by homology | C
GA4-3                                             | At1g80340 | AF070937| functional | C
GA4-4                                             | At4g25420 |         | by homology | C

88 gibberellin 2-oxidase (3 seq.)
2OX-1                      At1g30040    AJ132436    functional    C
2OX-2                      At1g78440    AJ132435    functional    C
2OX-3                      At2g34550    AJ132437    functional    C
89 gibberellin oxidation enzyme
n.c.y.
90 ferrochelatase (2 seq.)
FC1                        At2g30390    O04921      functional    P
FC2                        At5g26030    P42043      functional    P/M
91 heme oxygenase (4 seq.)
HO1                        At1g58300    AF320023    by homology   P/M
HO2                        At1g69720    Ceres 101223 by homology   P/M
HO3                        At2g26550    AF320022    functional    P
HO4                        At2g26670    AF132475    functional    P
92 phytochromobilin synthase (HY 2) (1 seq.)
At3g09150    AB045112    functional    P
93 monoterpene/sesquiterpene synthase-like genes (33 seq.)
TPS 1 (TPS 29)             At1g31950    by homology   C
TPS 2 (TPS 22)             At1g33750    by homology   C
TPS 3 (TPS 28)             At1g48800    by homology   C
TPS 4 (pseudo-F)           At1g48820    by homology   C
TPS 5 (TPS 4)              At1g61120    AY059759     by homology   –
TPS 6 (TPS 14)             At1g61680    by homology   –
TPS 7 (TPS 26)             At1g66020    by homology   C
TPS 8 (TPS 6)              At1g70080    by homology   –
TPS 9 (TPS 5)              At2g23230    by homology   ?
TPS 10 (TPS 10)            At2g24210    by homology   P
TPS 11 (short) (pseudo-C)  At2g37140    by homology   ?
TPS 12 (TPS 17)            At3g14490    by homology   C
TPS 13 (TPS 18)            At3g14520    by homology   C
TPS 14 (TPS 19)            At3g14540    by homology   C
TPS 15 (TPS 24)            At3g25810    by homology   P
TPS 16 (TPS 27)            At3g25823    by homology   ?
TPS 17 (TPS 23)            At3g25830    by homology   P
TPS 18 (TPS 16)            At3g29110    by homology   C
TPS 19 (TPS 15)            At3g29190    by homology   C
TPS 20 (TPS 25)            At3g29410    by homology   ?
TPS 21 (short) (pseudo-G)  At3g31415    by homology   ?
TPS 22 (TPS 30)            At3g32030    by homology   C
TPS 23 (TPS 12)            At4g13280    by homology   –
TPS 24 (TPS 13)            At4g13300    by homology   C
TPS 25 (TPS 1)             At4g15870    by homology   C
TPS 26 (TPS 2)             At4g16730    by homology   P
TPS 27 (TPS 3)             At4g16740    by homology   P
TPS 28 (TPS 7)             At4g20200    by homology   C
TPS 29 (TPS 8)             At4g20210    by homology   C
TPS 30 (TPS 9)             At4g20230    by homology   C
TPS 31 (TPS 21)            At5g23960    by homology   C
TPS 32 (TPS 11)            At5g44630    by homology   C
TPS 33 (TPS 20)            At5g48110    by homology   C

94 farnesyltransferase/geranylgeranyltransferase I, α-subunit (1 seq.) | At3g59380 | Q9LX33 | functional | C
95 farnesyltransferase, β-subunit (ERA1) (1 seq.) | At5g40280 | AF214106 | functional | C
96 geranylgeranyl transferase I, β-subunit (1 seq.) | At2g39550 | AF311225 | functional | C
97 (Rab) geranylgeranyl transferase II, α-subunit (2 seq.)
GGTaseIIα-1 | At4g24490 | by homology | C
GGTaseIIα-2 | At5g41820 | by homology | C
98 (Rab) geranylgeranyl transferase II, β-subunit (2 seq.)
GGTaseIIβ-1 | At3g12070 | by homology | C
GGTaseIIβ-2 | At5g12210 | by homology | C
99 dehydrogenase
100 sterol reductase (DET2) (1 seq.) | At2g38050 | U53860 | functional C/ER
101 | 3-keto-reductase n.c.y.
102 6-α-hydroxylase | n.c.y.
103 6-α-dehydrogenase | n.c.y.
104 steroid C22-hydroxylase (CYP90B1; DWF1) (1 seq.) | At3g50660 | AF044216 | functional | C/ER
105 steroid C23-hydroxylase (CYP90A1) (1 seq.) | At5g05690 | Q42569 | functional | C/ER
106 C3-dehydrogenase | n.c.y.
107 C3-reductase | n.c.y.
108 C2-alpha-hydroxylase | n.c.y.
109 brassinolide synthase | n.c.y.
110 p-hydroxybenzoate poly prenyltransferase (CoQ2) (1 seq.) | At4g23660 | AB052553 | functional (?) | M
111 O-methyltransferase (multifunctional) (CoQ3) | At2g30920 | by homology | M
112 decarboxylase | n.c.y.
113 monooxygenase (ubiF/ubiH-like) | At3g24200 | by homology | M
114 C-methyltransferase (CoQ5/ubiE) | At5g57300 | by homology | M
115 monooxygenase (ubiF/CoQ7-like) | no homologues in plants
116 triterpene synthases (not membrane sterol biosynthesis)
lupeol synthase-like (4 seq.)
LS1 | At1g66960 | by homology | C
LS2 | At1g78950 | by homology | C
LS3 (multifunctional) | At1g78960 | functional | C
LS4 (LUP1) | At1g78970 | functional | C
2, 3-oxidosqualene synthase-like (6 seq.)
TTPS1 | At1g78500 | by homology | C
TTPS2 | At4g15340 | by homology | C
TTPS3 | At4g15370 | by homology | C
TTPS4 | At5g36150 | by homology | C
TTPS5 | At5g42600 | by homology | C
TTPS6 | At5g48010 | by homology | C
117 cis-poly prenyl diphosphate synthase (3 seq.)
UPPS1 | At2g17570 | AF387010 | by homology | C/ER
UPPS-like2 | At5g60500 | by homology | C/ER
UPPS-like3 | At5g60510 | by homology | C/ER
118 homogentisate phytyltransferase (1 seq.) | At2g18950 | AF324344 | functional | P/En(?)
119 tocopherol cyclase (δ-tocopherol-forming) | n.c.y.
120 C-methyltransferase (β-tocopherol-forming) | n.c.y.
121 C-methyltransferase (2, 3-dimethyl-5-phytylquinol-forming) | n.c.y.
122 tocopherol cyclase (γ-tocopherol-forming) | n.c.y.
123 γ-tocopherol C-methyltransferase (TMT) (1 seq.) | At1g64970 | AF104220 | functional | P

124β-tocopherol C-methyltransferase

**Accessory proteins/regulatory proteins**

- plastid terminal oxidase (PTO) (immutans) (1 seq.) | At4g22260 | AJ004881 | functional | P/En(?)
- FLU (negative regulator of chlorophyll biosynthesis) (1 seq.) | At3g14110 | functional | P
- Rab escort protein-like (required for GGTase activity) (4 seq.)
  - RabEP1 | At2g44100 | Ceres 29536 | by homology | C
  - RabEP2 | At3g06540 | by homology | C
  - RabEP3 | At3g59920 | AB005560 | by homology | C
  - RabEP4 | At5g09550 | by homology | C
- prenylcysteine lyase-like (turnover of prenylated proteins) (1 seq.) | At5g63910 | by homology | C

**Additional genes (function not established yet)**

- CoQ 4 homologue (function unknown) | At2g03690 | AY065081 | by homology | M
- 3-oxo-5-a-sterol reductase (function?) | At5g16010 | by homology | C
- sterol glucosyltransferase (EC 2.4.1.173) (2 seq.)
  - SGT1 | At1g43620 | by homology | C
  - SGT2 | At3g07020 | Z83833 | functional | C
- steroid sulfotransferase-like (15 seq.)
  - SST1 | At1g13420 | by homology | C
  - SST2 | At1g13430 | by homology | C
  - SST3 | At1g28170 | by homology | C
  - SST4 | At2g03750 | by homology | C
  - SST5 | At2g03760 | by homology | C
  - SST6 | At2g03770 | by homology | C
  - SST7 | At2g14920 | by homology | C
  - SST8 | At2g27570 | by homology | C
  - SST9 | At2g43420 | by homology | C
  - SST10 | At3g45070 | by homology | C
  - SST11 | At3g45080 | by homology | C
  - SST12 | At4g26280 | by homology | C
  - SST13 | At5g07000 | by homology | C
  - SST14 | At5g07010 | by homology | C
  - SST15 | At5g43690 | by homology | C
- 3-β-hydroxysteroid dehydrogenase-like (3 seq.)
  - 3HDH1 | At1g47290 | by homology | C
  - 3HDH2 | At2g16160 | by homology | C
  - 3HDH3 | At2g33630 | AF428460 | by homology | C
- 11-β-hydroxysteroid dehydrogenase-like (7 seq.)
  - 11HDH1 | At3g47350 | by homology | C
  - 11HDH2 | At3g47360 | by homology | C
  - 11HDH3 | At4g10020 | by homology | C
  - 11HDH4 | At5g50590 | by homology | C
  - 11HDH5 | At5g50600 | by homology | C
  - 11HDH6 | At5g50770 | by homology | C
- 17-β-hydroxysteroid dehydrogenase-like (1 seq.) | At1g76150 | by homology | C

**Abbreviations:** ER, endoplasmatic reticulum; P/En, plastidal envelope membrane; C, cytosolic; M, mitochondrial; n.c.y., not cloned yet; P, plastidal.

L-Phenylalanine
↓
Cinnamate
↓
p-Coumarate
↓
p-Coumaroyl-CoA
↓
p-Hydroxybenzoate
↓
3-Polyprenyl-4-hydroxy-benzoate
↓
3,4-Dihydroxy-5-polyprenyl-benzoate
↓
3-Methoxy-4-hydroxy-5-polyprenylbenzoate
↓
2-Polyprenyl-6-methoxy-phenol
↓
2-Polyprenyl-6-methoxy-1,4-benzquinol
↓
2-Polyprenyl-3-methyl-6-methoxy-1,4-benzquinol
↓
2-Polyprenyl-3-methyl-5-hydroxy-6-methoxy-1,4-benzquinol
↓
Ubiquinol
↓
Ubiquinone

Polyterpenes
Sesquiterpenes
Dolichols (protein modification)
Dehydrolicolchol diphosphate
Polyprenyl diphosphate

IPP + DMAPP
FPP
Squalene

Prenylated proteins
RabEP
GGPP

Triterpenes
β-Amyrin
Lupeol
4α-Methyl-fecosterol
Citrostadienol
Avenasterol
Episterol
5-Dehydro-episterol
3-Dehydro-Δ⁴,5-campesterol
24-Methylene-cholesterol
Isofucosterol
5-Dehydro-avenasterol
Campesterol
6-Oxo-campestanol
6-α-Hydroxy-campestanol
Cathasterone
6-Deoxo-cathasterone
Teasterone
3-Deoxo-teasterone
3-Dehydro-teasterone
3-Dehydro-6-deoxo-teasterone
Typhasterol
6-Deoxo-typhasterol
Castasterone
6α-Hydroxy-castasterone
6-Deoxo-castasterone
Campesterol
Sitosterol
Stigmasterol
Brassinolide

Figure 3. Metabolism of isoprenoids derived from cytosolic and mitochondrial pathways. The symbol for anabolic reactions is a solid arrow, whereas catabolic reactions are indicated by arrows with dotted lines. Known enzymes involved in isoprenoid metabolism are numbered according to the annotation given in Table 1 and question marks indicate steps for which an enzymatic activity has not yet been demonstrated.

involved in (or presumed to be related to) isoprenoid and chlorophyll metabolism in plants, animals and microorganisms. Sequence similarity searches (BLASTX; Altschul et al., 1990) were performed in batches with FASTA data files at the TIGR database (A. thaliana annotation from January 2002; http://www.tigr.org/tdb/e2k1/ath1/) and at our in-house non-redundant database to reveal the A. thaliana homologues. The output of the BLASTX searches was processed with PERL scripts to extract the top 10 hits from each result file, to correlate the top hits with AGI numbers and to indicate if sequences of full-length cDNA clones were available in public databases. The sequences of predicted proteins encoded by gene fam-
ilies were aligned by means of the CLUSTALW algorithm (Higgins *et al.*, 1994), with homologues from other organisms for which a biochemical function had already been established. The subcellular localization of proteins was predicted using the PSORT (http://psort.nibb.ac.jp/form.html) and TargetP (http://www.cbs.dtu.dk/services/TargetP/) programs. All data sets were integrated in a database, the results were manually evaluated, and predicted proteins were categorized according to their putative biochemical function (Table 1). To identify gene clusters, a PERL script was devised that recognizes putative clusters by employing user-defined parameters. For the visualization of gene clusters, the exact chromosome position (resolution 1 kb) of each gene was entered in the index file for the *A. thaliana* genome, which is then used by the GeneSpring gene expression analysis tool (Silicon Genetics, Redwood City, CA) to generate a chromosome image with zooming and editing capabilities.

### Results

#### Metabolism of isoprenoids derived from plastidal pathways

##### IPP and DMAPP biosynthesis in plastids

An overview of the metabolism of plastid-derived isoprenoids and chlorophylls, featuring key intermediates and end-products, is shown in Figure 2. The enzymes responsible for the biosynthesis of IPP and DMAPP via the MVA-independent pathway (phase I) in plants are localized to plastids (Figure 1). Three copies of the first enzyme, 1-deoxy-D-xylulose 5-phosphate synthase (1) (DXPS), are present in the *A. thaliana* genome, and functional data has been obtained for one of them (At4g15560; Estevez *et al.*, 2001). The subsequent steps are catalyzed by 1-deoxy-D-xylulose 5-phosphate reductoisomerase (2) (DXR), 2-C-methyl-D-erythritol 4-phosphate cytidyltransferase (3) (MCT), 4-(cytidine 5′-diphospho)-2-C-methyl-D-erythritol kinase (4) (CMK) and 2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase (5) (MECPS). Functional data are available for *A. thaliana* orthologues of DXR (Schwender *et al.*, 1999) and MCT (Rohdich *et al.*, 2000a), whereas functional data for a plant CMK have been obtained only for tomato (Rohdich *et al.*, 2000b), and MECPS has not yet been functionally characterized from a plant source. Two additional open reading frames from

Escherichia coli, annotated as lytB (7) and gcpE (6), have been characterized recently to encode enzyme products capable of generating IPP and DMAPP from 2-C-methyl-D-erythritol 2,4-cyclodiphosphate (Hintz *et al.*, 2001; Hecht *et al.*, 2001; Rohdich *et al.*, 2002). Genes homologous to lytB, the biochemical function of which has yet to be determined, and gcpE, which has been characterized as encoding 1-hydroxy-2-methyl-2-(E)-butenyl 4-phosphate synthase (Querol *et al.*, 2002), occur as single copies in *A. thaliana*.

#### Short-chain prenyltransferases of the plastid

As part of the plastidal phase II reactions, *A. thaliana* genes encoding two isoenzymes of isopentenyl diphosphate isomerase (14) and a geranyl diphosphate synthase (15) (GPPS; single-copy gene in the *A. thaliana* genome) have been characterized (Campbell *et al.*, 1997; Bouvier *et al.*, 2000a). Geranylgeranyl diphosphate synthase (17) (GGPPS) is encoded by a multigene family with 12 members in *A. thaliana*; functional data have been obtained for five of these family members (Okada *et al.*, 2000), two of which have been shown to encode plastidal isoenzymes (At3g14530 and At4g36810). Based on the sequence characteristics of the remaining 7 members of the GGPPS gene family, all encoded proteins contain putative plastidal targeting sequences and are likely to be localized to plastids.

#### Gibberellin metabolism (plastidal and cytosolic)

Gibberellins are tetracyclic diterpenes with a role as hormones controlling stem elongation and affecting reproductive processes. Two diterpene synthases (phase III) are encoded in the *A. thaliana* genome which code for the two plastidal entry enzymes into the gibberellin biosynthetic pathway, copalyl diphosphate synthase (80) (Sun and Kamiya, 1994) and ent-kaurene synthase (81) (Yamaguchi *et al.*, 1998). The gibberellin end-products are generated by a series of extraplasmid oxidative modifications (phase IV), starting with the multifunctional cytochrome P450 monooxygenases ent-kaurene oxidase (82) (Hell iwell *et al.*, 1999; single gene in *A. thaliana*) and ent-kaurenoic acid oxidase (83) (Hell iwell *et al.*, 2001; 2 isogenes), which are localized to the endoplasmic reticulum. Further modifications are catalyzed by multifunctional oxoglutarate-dependent enzymes, and include gibberellin 20-oxidase (86) (Xu *et al.*, 1995; Phillips *et al.*, 1995; 5 isogenes in *A. thaliana*) and gibberellin 3β-oxidase (87) (Williams *et al.*, 1998), which are localized in the cytosol. Gibberellin 7-oxidase(84) was first cloned from pumpkin (Lange, 1997) and two *A. thaliana* homologues are detectable (both are annotated as leucoanthocyanidin dioxygenase-like proteins). Gibberellin deactivation reactions are initiated by gibberellin 2-oxidase (88) (Thomas *et al.*, 1999; 3 isogenes in *A. thaliana*), another multifunctional oxoglutarate-dependent dioxygenase with cytosolic localization. Additional steps in gibberellin catabolism have not yet been defined.

### Carotenoid and abscisic acid metabolism (plastidal and cytosolic)

The biosynthesis of carotenoids, which protect the photosynthetic apparatus from photo-oxidative damage and play a structural role in the assembly of light-harvesting complexes, is entirely localized to plastids (Figure 2). Phytoene synthase (19), which catalyzes the first committed step in this pathway (phase II), was first cloned from tomato (Bartley *et al.*, 1992), and is encoded by a single gene in *A. thaliana*. Further metabolic conversions to yield all-trans-lycopene are catalyzed by phytoene desaturase (20) (Bartley *et al.*, 1991; single-copy gene in *A. thaliana*), ζ-carotene desaturase (21) (Albrecht *et al.*, 1995; single-copy gene in *A. thaliana*) and carotenoid isomerase (22) (Park *et al.*, 2002; two homologues present in the *A. thaliana* genome). In addition, phytoene desaturation also requires plastoquinone (Norris *et al.*, 1995) and a plastidial terminal oxidase activity (Wu *et al.*, 1999). Lycopene is cyclized on both ends (phase III) to generate either β-carotene (catalyzed by lycopene β-cyclase (23), Pecker *et al.*, 1996) or α-carotene (combination of lycopene β-cyclase and lycopene ε-cyclase (24); Cunningham *et al.*, 1996). Hydroxylation of β-carotene (phase IV) by β-carotene hydroxylase (26) (Sun *et al.*, 1996; 2 isogenes present in the *A. thaliana* genome) results in the formation of zeaxanthin. Hydroxylation of α-carotene, possibly by the action of the same β-hydroxylase and an as yet unidentified ε-hydroxylase (25), produces lutein, the most abundant xanthophyll in plant plastids. Zeaxanthin epoxidase (27), which was first cloned from tobacco (Marin *et al.*, 1996), catalyzes the conversion of zeaxanthin to violaxanthin via antheraxanthin as an intermediate. A homologous sequence, represented by a single-copy gene, is detectable in the *A. thaliana* genome. Under high light intensity, violaxanthin is converted back to zeaxanthin (violaxanthin de-epoxidase (28); Búgos *et al.*, 1998), which participates in the thermal dissipation of excess absorbed light energy (xanthophyll cycle; Demming-Adams *et al.*, 1996). Both

violaxanthin and its allene derivative neoxanthin can be precursors for the carotenoid-derived plant hormone abscisic acid (ABA). In potato and tobacco, violaxanthin can be converted to neoxanthin by neoxanthin synthase (29), an enzyme with high homology to carotenoid cyclases and capsanthin/capsorubin synthase from pepper (Bouvier *et al.*, 2000b; Al-Babili *et al.*, 2000). However, in *A. thaliana*, no additional candidate genes, besides the already characterized carotenoid cyclases, are detectable with reasonable sequence homology, indicating either that a neoxanthin synthase activity may not exist in this organism, that it has evolved independently from a divergent enzyme, that *A. thaliana* expresses multifunctional carotenoid cyclases, or that it uses violaxanthin as the precursor for ABA biosynthesis. Thus far, two further enzymes involved in the breakdown of xanthophylls to ABA have been cloned: genes for epoxycarotenoid (neoxanthin) cleavage enzyme (30) (Schwartz *et al.*, 1997) occur as a family of 7 members in the *A. thaliana* genome, whereas abscisic aldehyde oxidase (31) (Seo *et al.*, 2000) is encoded by a single-copy gene. Except for ABA 8′-hydroxylase, a membrane-bound cytochrome P450 monooxygenase which has been characterized biochemically (Krochko *et al.*, 1998), none of the enzymes involved in ABA catabolism and conjugation have been characterized at the biochemical or the molecular levels.

### Tocopherol biosynthesis (plastidal and cytosolic)

Tocopherols, collectively known as vitamin E, are lipid-soluble antioxidants synthesized exclusively by photosynthetic organisms. The first committed step in tocopherol biosynthesis involves the condensation of homogentisate and phytyl diphosphate to yield 2-methyl-6-phytylquinol, a condensation reaction catalyzed by a membrane-bound plastid-localized homogentisate phytyltransferase (118) (Collakova and DellaPenna, 2001; single-copy gene in *A. thaliana*). Homogentisate itself derives from tyrosine (tyrosine aminotransferase; Lopukhina *et al.*, 2001; family of 7 genes in *A. thaliana*) via p-hydroxyphenylpyruvate (HPP dioxygenase, Norris *et al.*, 1998; single-copy gene in *A. thaliana*), whereas phytyl diphosphate is generated from GGPP by the action of geranylgeranyl reductase (18) (GGR; Keller *et al.*, 1998; single-copy gene in *A. thaliana*). Further conversions of 2-methyl-6-phytylquinol to γ-tocopherol involve a C-methyltransferase (121) and a tocopherol-specific cyclase (122), both of which have not yet been cloned. An alternative pathway from 2-methyl-
6-phytylquinol via δ-tocopherol or β-tocopherol to γ-tocopherol has been proposed (Schultz *et al.*, 1985), but no enzymes involved in these steps have as yet been identified. As the last step of the biosynthetic sequence, γ-tocopherol is converted to α-tocopherol by γ-tocopherol O-methyltransferase (123) (Shintani and DellaPenna, 1998), for which a single gene is present in the *A. thaliana* genome.

**Plastoquinone and phylloquinone biosynthesis (plastidal and cytosolic)**

Substituted quinones are employed as cofactors in electron transport chains of the two photosystems of plants and cyanobacteria. In the photosystem II reaction centers, plastoquinone-9 functions as a one-electron cofactor at the QA site and as a two-electron/two proton cycling cofactor at the QB site. Plastoquinone biosynthesis involves, as in the case of tocopherols, a condensation of homogentisate with a prenyl diphosphate. The localization of enzymes responsible for the biosynthesis of the polyprenyl side-chain of plastoquinone is still a matter of debate. Swiezewska *et al.* (1993) have presented biochemical evidence that microsomal preparations from spinach contain polyprenyl diphosphate synthase and prenyltransferase activity for plastoquinone biosynthesis, and they have hypothesized that a specific transport and targeting system is required for the transfer of intermediates of plastoquinone biosynthesis into plastids. In contrast, in feeding experiments with isotopically labeled precursors, Lichtenthaler *et al.* (1997) and Disch *et al.* (1998) have shown that the polyprenyl side-chain of plastoquinone is synthesized in plastids via the MVA-independent pathway. The *A. thaliana* genome contains two predicted proteins with homology to solanesyl diphosphate synthase (phase II) from *Rhodobacter capsulatus* (Okada *et al.*, 1997). One of these polyprenyl diphosphate synthase (67) isoenzymes appears to be, according to PSORT and TargetP analyses, localized to mitochondria (At1g78510). For the second isogene (At1g17050) a cytosolic localization is indicated by PSORT and TargetP analyses; however, an alternative gene model for the same sequence (AAD50025) results in a gene product that is predicted to be localized to mitochondria. A plastidial isoform has not yet been predicted nor been detected. The *A. thaliana* genome contains a small gene family of prenyltransferases of which one gene has been characterized as being the tocopherol biosynthetic enzyme homogentisate phytyltransferase (118) (Collakova and DellaPenna, 2001). A second prenyltransferase en-

codes chlorophyll synthetase (45) (Gaubier *et al.*, 1995), whereas a third homologue is predicted by PSORT and TargetP analyses to be localized to plastids (Table 1; At3g11950), which is consistent with a homogentisate polyprenyltransferase (71) for plastoquinone biosynthesis based on the isotope labeling data (Lichtenthaler *et al.*, 1997; Disch *et al.*, 1998).

Phylloquinone, a second plastidial quinone of plants, functions as a one-electron cofactor in the Al site of photosystem I. Although the biosynthetic route to phylloquinone has not been described in plants yet, the pathway is likely to be similar to the pathway of menaquinone biosynthesis in eubacteria. Menaquinone differs from phylloquinone by the presence of a partly unsaturated C-40 side-chain rather than a mostly saturated, C-20 phytyl side-chain. The genome database for the cyanobacterium *Synechocystis*, which also produces phylloquinone and not menaquinone, contains homologues for several genes that encode enzymes for menaquinone biosynthesis in *E. coli*, two of which, 1,4-dihydroxy-2-naphthoate (DHNA) synthase (menB) and DHNA phytyltransferase (menA) have been characterized by a mutant approach (Johnson *et al.*, 2000). Based on a homology search, the putative *A. thaliana* orthologues for these genes were identified (77, 78) (Table 1). In addition, homologues of the *E. coli* menaquinone biosynthetic genes *O*-succinylbenzoate (OSB) synthase (75) (menC), 2-succinyl-6-hydroxy-2,4-cyclohexadiene-1-carboxylate (SHCHC) synthase (74) (menD), and OSB-CoA ligase (76) (menE) are detectable in the *A. thaliana* genome (Table 1). It should be noted that the predicted subcellular distribution of these enzymes would require exchange of intermediates across the chloroplast envelope membrane, and may hence be an artifact of incorrect gene prediction. Additional functional data will be needed to resolve this uncertainty. Two genes encoding isochorismate synthase (73) are present in the *A. thaliana* genome (Table 1), one of which is involved in salicylic acid biosynthesis (At1g74710; Wildermuth *et al.*, 2001). The function of the second isogene (At1g18870) has not yet been established, and additional experiments will have to be conducted to elucidate which isogene is involved in the phylloquinone biosynthetic pathway. In *E. coli*, the menaquinone and the ubiquinone biosynthetic pathways use the same C-methyltransferase (ubiE); however, in plants the pathway to the menaquinone analogue phylloquinone is localized to plastids whereas ubiquinone biosynthesis is restricted to mitochondria. Interestingly, the

A. thaliana genome contains five genes with homology to ubiE. One ubiE homologue appears to specify an enzyme with a mitochondrial targeting sequence (thus a candidate gene for ubiquinone biosynthesis; At5g57300) and a second gene encodes, based on a TargetP analysis, a plastidial isoenzyme which may be involved phylloquinone biosynthesis (At1g23360). The three additional copies are likely to be cytosolic based on current gene predictions (At3g02770; At5g16450; At5g56260).

**Chlorophyll metabolism (plastidal and cytosolic)**

Cyanobacteria and chloroplasts of green algae and plants utilize chlorophyll *a* and chlorophyll *b* for light harvesting and photosynthetic charge separation. The Mg-tetrapyrrols of chlorophylls are derived from L-glutamate. Their biosynthesis starts with the conversion of L-glutamate to glutamate 1-semialdehyde, catalyzed by glutamyl-tRNA synthetase (32) (Day et al., 1998; single-copy gene in A. thaliana) and glutamyl-tRNA reductase (33) (Kumar and Soll, 1992; Kumar et al., 1996; 3 isogenes in A. thaliana). Genes encoding the subsequent enzymes, glutamate 1-semialdehyde aminotransferase (34), which was first cloned from barley (Grimm, 1990), and aminolevulinate dehydratase (35), which was first cloned from pea (Boese et al., 1991), are present with two copies each in the A. thaliana genome. Porphobilinogen deaminase (36) catalyzes the condensation of four molecules of porphobilinogen to form one molecule of 1-hydroxymethylbilane (Lim et al., 1994; single copy gene in A. thaliana). One gene homologous to uroporphyrinogen synthase (37), which has been cloned from bacteria, yeasts and animals, can be detected in A. thaliana. For uroporphyrinogen decarboxylase (38), which was first cloned from tobacco and barley (Mock et al., 1995), two copies occur in the A. thaliana genome. Catalysis by coproporphyrinogen III oxidase (39) (Ishikawa et al., 2001; single-copy gene in A. thaliana) and protoporphyrinogen IX oxidase (40) (Narita et al., 1996; 2 isogenes in the A. thaliana genome) leads to the formation of protoporphyrin IX. At this stage, the pathway branches to yield phytochromobilin, the chromophore of the plant photoreceptor phytochrome, via an iron-dependent pathway, or the chlorophylls via a Mg-dependent pathway. The synthesis of phytochromobilin requires three enzymes, ferrochelatase (90) (Chow et al., 1998; two isogenes in the A. thaliana genome), heme oxygenase (91) (Muramoto et al., 1999; Davis et al., 1999; Davis et al., 2001; 4 isogenes in the A. thaliana

genome), and phytochromobilin synthase (92) (Kohchi et al., 2001; single-copy gene in A. thaliana). Isoenzymes of ferrochelatase and heme oxygenase are also localized to mitochondria (Table 1). The Mg-dependent pathway to chlorophylls starts with Mg-protoporphyrinogen IX chelatase (41) (Papenbrock et al., 1997; 3 genes in A. thaliana). Subsequent reactions involve Mg-protoporphyrinogen IX methyltransferase (42), first cloned from Synechocystis (Smith et al., 1996; single-copy homologue in A. thaliana), and two additional enzymes, Mg-protoporphyrin IX monomethylester cyclase (43) and 8-vinyl reductase, which have not yet been cloned. The conversion to chlorophyllide *a* is catalyzed by protochlorophyllide oxidoreductase (44) (POR; Benli et al., 1991; Armstrong et al., 1995; Su et al., 2001), for which three copies are encoded in the A. thaliana genome. In spinach a dual targeting of one POR isoenzyme to mitochondria and plastids has been demonstrated (Watanabe et al., 2001); however, gene prediction and subcellular targeting algorithms do not indicate that this is the case in A. thaliana, where all isoenzymes are suggested to be localized to plastids. The synthesis of chlorophyll *a* involves the attachment of the phytyl side-chain by chlorophyll synthetase (45) (Gaubier et al., 1995; single-copy gene in A. thaliana). Chlorophyll *b* differs from chlorophyll *a* in the conversion of a methyl group to a formyl side group, a reaction catalyzed by chlorophyll *a* oxygenase (46) (Espineda et al., 1999; single-copy gene in A. thaliana).

Senescence and subsequent death are terminal phases in the development of all organs of a plant, including leaves, stems, roots, and flowers. During senescence, one of the clearest symptoms is the loss of green color due to chlorophyll breakdown. The first step in the catabolic cascade is the conversion of chlorophyll to chlorophyllide in a reaction catalyzed by chlorophyllase (48) (Benedetti et al., 1998; Tsuchiya et al., 1999; 2 isogenes in A. thaliana). The other product of chlorophyllase action is phytol, which usually accumulates in the lipid globules of gerontoplasts in the form of esters. Mg-dechelatase (49) is required to remove Mg from chlorophyllide, a reaction followed by a ring closure catalyzed by pheophorbide *a* oxygenase (50) (both genes not yet cloned). The resulting catabolite (red chlorophyll catabolite; RCC) is further metabolized by RCC reductase (51) (Mach et al., 2001; single-copy gene in A. thaliana), which catalyzes the ferredoxin-dependent reduction of a double bond in the pyrrol system of RCC to produce fluorescent chlorophyll catabolite (FCC). FCC is then

exported and further metabolized in the cytosolic and vacuolar compartments to yield the non-fluorescent chlorophyll catabolites (Hörtensteiner, 1998). The regulation of chlorophyll biosynthesis is complex; thus far, the characterized mediators include light (activation of protochlorophyllide oxidoreductase; Armstrong *et al.*, 1995; Su *et al.*, 2001), heme (negative regulator of glutamyl-tRNA reductase; Pontoppidan and Kannangara, 1995) and FLU (negative regulator of several enzymes of the Mg-dependent branch of the pathway; Meskauskiene *et al.*, 2001).

**Isoprenoid biosynthesis in the cytosol and in mitochondria**

**IPP and DMAPP biosynthesis in the cytosol**

Cytosolic and mitochondrial isoprenoids derive from acetyl-CoA (Figure 1), which itself is obtained from carbon fixation of CO2 through a sequence of glycolytic enzymes. The early steps of the biosynthetic pathway to yield mevalonate (MVA) include acetoacetyl-CoA thiolase (8) (AACT), which was cloned from radish (Vollack and Bach, 1996; two homologous genes are detectable in *A. thaliana*), 3-hydroxy-3-methylglutaryl (HMG)-CoA synthase (9) (Montamat *et al.*, 1995; single-copy gene in *A. thaliana*), and HMG-CoA reductase (10) (Caelles *et al.*, 1989; Enjuto *et al.*, 1994; two gene copies in *A. thaliana*). Two successive phosphorylations, catalyzed by mevalonate kinase (11) (Riou *et al.*, 1994; single-copy gene in *A. thaliana*) and phosphomevalonate kinase (12) (distant homologue of the yeast (Tsay and Robinson, 1991) gene detectable in the *A. thaliana* genome), followed by a decarboxylation (mevalonate diphosphate decarboxylase (13); Cordier *et al.*, 1999; 2 isogenes in *A. thaliana*) lead to the formation of IPP (phase I). Interestingly, the *A. thaliana* genome does not contain obvious additional homologues to the two plastidal IPPI (14) isoenzymes; however, it has been proposed that an alternative translation start site may allow the generation of cytosolic isoenzymes from the same genes (Cunningham and Gantt, 2000). *A. thaliana* contains two farnesyl diphosphate synthase (16) (FPPS) genes, coding for one cytosolic isoenzyme (At4g171890; Delourme *et al.*, 1994; likely involved in phytosterol and sesquiterpene biosynthesis) and one mitochondrial isoenzyme (At5g47770; Cunillera *et al.*, 1997; likely involved in ubiquinone biosynthesis).

**Biosynthesis of long-chain polyprenyl diphosphates in the cytosol**

The biosynthetic sequence to dolichols, which are long-chain polyprenols required for the synthesis of glycoproteins, starts with dehydrolicolyl diphosphate synthase (68) (Cunillera *et al.*, 2000). In the *A. thaliana* genome, three copies of this cis-prenyltransferase are detectable, two of which appear to encode cytosolic isoenzymes (ER-associated; At2g23410; At5g58780), whereas one isoenzyme, based on a TargetP analysis, appears to be localized to mitochondria (At5g58770). At present further steps towards dolichyl sugars are completely uncharacterized in plants. However, Oh *et al.* (2000) have reported on a cis-prenyltransferase, homologous to undecaprenyl diphosphate synthase from eubacteria, which catalyzes the formation of polyprenyl diphosphates ranging from C100 to C130. The *A. thaliana* genome also contains three copies of genes putatively encoding cis-prenyltransferases, which could produce cytosolic polyprenyl diphosphates, the metabolic fate of which is currently unknown.

**Phytosterol and brassinosteroid metabolism in the cytosol**

The biosynthesis of sterols, which are important in controlling membrane fluidity and are involved in plant embryogenesis, starts with the condensation of two molecules of farnesyl diphosphate (squalene synthase (52); Nakashima *et al.*, 1995; 2 gene copies detectable in *A. thaliana*) and a subsequent oxidation (squalene monooxygenase (53); Schäfer *et al.*, 1999; 6 isogenes in *A. thaliana*). The resulting 2,3-epoxysqualene serves as a substrate for triterpene synthases catalyzing cyclization reactions that ultimately lead to a vast array of triterpenoid compounds in addition to the typical membrane sterols. A sequence relationship tree of the *A. thaliana* triterpenoid synthases (54, 116) indicates the presence of three distinct subfamilies: one family codes for cycloartenol synthase (54) (Corey *et al.*, 1993; two copies in *A. thaliana*), a second family includes lupeol synthase-like sequences (Herrera *et al.*, 1998; 4 members in *A. thaliana*), and the function of a third family, which comprises six members, is yet to be characterized (Husselstein-Muller *et al.*, 2001). The further conversions of the lupeol and amyrin intermediates synthesized by the members of the second triterpene synthase family are unknown. The membrane sterol pathway involves methyltransferases acting on cycloartenol (sterol C24-methyltransferase 1 (55); first cloned

from soybean (Shi *et al.*, 1996); one homologue in *A. thaliana* genome) and 24-methylenelophenol (sterol C24-methyltransferase 2 (61); Husselstein *et al.*, 1996; Bouvier-Nave *et al.*, 1997; 2 isogenes in *A. thaliana*). The substrates for demethylases include obtusifoliol (obtusifoliol C14-methyl oxidase (59) (CYP51); Kushiro *et al.*, 2001; 2 isogenes in *A. thaliana*), 24-methylenelophenol/citrostadienol (C4-methyl oxidase (62); Darnet *et al.*, 2001; 2 isogenes in *A. thaliana*) and 24-methylenecycloartenol (24-methylenecycloartenol C4-methyl oxidase (56); not cloned yet; two candidate genes based on sequence relatedness to CYP51 in the *A. thaliana* genome; Table 1). Isomerases are involved in the conversion of cycloecalenol to obtusifoliol (57) (Lovato *et al.*, 2000) and of 4α-methylfecosterol to 24-methylenelophenol (60) (Grebenok *et al.*, 1998). Three sterol reductases have been cloned, comprising obtusifoliol C14-reductase (58) (Schrick *et al.*, 2000), sterol C7-reductase (64) (Lecain *et al.*, 1996) and a sterol C24-reductase (65) (Klahre *et al.*, 1998). Thus far, one desaturase has been cloned (sterol C5-desaturase (63); Gachotte *et al.*, 1996; 2 closely related genes in *A. thaliana*), with a sterol C22-desaturase (66), leading from β-sitosterol to stigmasterol, still undiscovered. Besides free sterols, steryl glycosides and acylated steryl glycosides are abundant in plant cell membranes. The first two enzymes involved in these conversions have been characterized as UDP-glucose:sterol glucosyltransferases (Warneke *et al.*, 1997; two gene copies detectable in the *A. thaliana* genome).

The biosynthetic steps leading from campesterol to the plant hormone brassinolide have been proposed, but few enzymes have been characterized. Thus far, a sterol 5α-reductase (100) (DET2; Li *et al.*, 1996), a C22α-hydroxylase (104) (Choe *et al.*, 1998) and a C23α-hydroxylase (105) (Szekeres *et al.*, 1996) have been cloned. The biosynthetic sequence from campesterol to brassinolide involves 4 steps for which catalysis by a 3β-hydroxy dehydrogenase or a 3-keto-reductase (101) is required. Interestingly, the *A. thaliana* genome contains three copies of a gene with homology to 3β-hydroxysteroid dehydrogenases; in animals, this bifunctional enzyme catalyzes multiple oxidative conversions of Δ⁵-ene-3β-hydroxysteroids and is also able to act as a 3-keto reductase (Rogerson *et al.*, 1995). In addition, a putative 17β-hydroxysteroid dehydrogenase and 8 copies of a putative 11β-hydroxysteroid dehydrogenase are present in the *A. thaliana* genome, but their function in plants has yet to be demonstrated. The closest characterized plant homologue of the *A. thaliana* 11β-hydroxysteroid dehydrogenase family is a steroleosin gene from *Sesamum indicum*, which encodes a steroid-binding dehydrogenase/reductase putatively involved in signal transduction (Lin *et al.*, 2002). Steroid sulfo-transferases are known to play an important role in the modulation of the biological activity of a number of metabolites in animals, including steroid hormones and neurotransmitters. Thus far, the only plant gene known is from *Brassica napus*, and its encoded enzyme has been demonstrated to catalyze the sulfonation of a brassinosteroid, which results in the loss of its biological activity (Rouleau *et al.*, 1999). The *A. thaliana* genome contains a family of 15 genes with homology to mammalian steroid sulfo-transferases the role of which is currently unknown.

**Ubiquinone biosynthesis in mitochondria**

The first committed step of the biosynthetic pathway to ubiquinone, which is an integral part of the mitochondrial electron transfer chain, is catalyzed by p-hydroxybenzoate polyprenyltransferase (110). A homologue of the eubacterial (ubiA) and the yeast (CoQ2) genes for this enzyme (Meganathan, 2001) is detectable in the *A. thaliana* genome and, according to TargetP analysis, specifies a protein with the expected N-terminal mitochondrial targeting sequence. Additional homologues to eubacterial/yeast genes involved in this pathway include a monooxygenase (115) (ubiF/CoQ7-like), a bifunctional O-methyltransferase (111) (CoQ3-like; Avelange and Joyard, 1998), two C-methyltransferases (114) (ubiH/CoQ6-like and ubiE/CoQ5-like), and a CoQ4-like sequence (function unknown). The C-methyltransferases are part of a larger gene family and their involvement in ubiquinone biosynthesis is suggested only by the presence of a predicted mitochondrial targeting sequence.

**Monoterpene and sesquiterpene biosynthesis**

*A. thaliana* has been reported to emit a complex mixture of monoterpene volatiles as a response to herbivory (Van Poeke *et al.*, 2001). The *A. thaliana* genome encodes a large family of enzymes with homology to monoterpene and sesquiterpene synthases, which catalyze the first committed step to terpenoids. Thus far, only one gene product, a myrcine/β-ocimene synthase, has been characterized from *A. thaliana* (Bohlmann *et al.*, 2000). A detailed analysis of terpene synthase-like genes in *A. thaliana* has been published

very recently (Aubourg *et al.*, 2002). The discovery of the function of these intriguing genes, and the conditions and cues that elicit their expression and the subsequent release of terpenoid end-products, will be of great value in improving our understanding of the thus far poorly studied metabolic diversity of *A. thaliana*.

### Protein prenylation

Protein prenylation, a post-translational modification which involves the covalent attachment of farnesyl (C15) or geranylgeranyl (C20) isoprenoid moieties to target proteins in the cytosol, plays important roles in signal transduction and intracellular trafficking pathways (Crowell, 2000). The transfer of the prenyl group is catalyzed by farnesyltransferase (FTase) or geranylgeranyltransferases I and II (GGTase I and GGTase II, respectively). FTase and GGTase I act as heterodimers with a common α-subunit (94) (Qian *et al.*, 1996; Yalovsky *et al.*, 1997) and a distinct β-subunit (95, 96) (Caldelari *et al.*, 2001). All of these subunits occur as single-copy genes in the *A. thaliana* genome. GGTase II was first cloned and characterized from rat (Armstrong *et al.*, 1993), and homologues for both the α-subunit (97) and the distinct β-subunit (98) are detectable in the *A. thaliana* genome (two copies each). In animals, GGTase requires the presence of a Rab escort protein activity (Andres *et al.*, 1993) for which 4 homologous genes are detectable in the *A. thaliana* genome. Geranylgeranyl diphosphate, the precursor for geranylgeranylations, most likely derives from a cytosolic pool provided by two cytosolic/ER GGPPS (17) isoenzymes (Okada *et al.*, 2000). A second protein prenylation pathway, involving a modification with phytol (C20), has been described in plant plastids (Parmryd *et al.*, 1999), but thus far the gene encoding the responsible prenyltransferase activity remains unidentified. The turnover of prenylated proteins in animals involves the disposal of prenlycysteines by prenylcysteine lyase (Zhang *et al.*, 1997), for which a homologous gene of unidentified function occurs in *A. thaliana*.

### Genomic organization of isoprenoid and chlorophyll biosynthesis

In some cases, functionally linked genes can occur in close proximity on plant chromosomes or may even be part of pathway gene clusters (Kanazin *et al.*, 1996; Frey *et al.*, 1997; Mayfield *et al.*, 2001).

Our analysis of the *A. thaliana* genome for the location of genes involved in isoprenoid and chlorophyll biosynthesis indicates that gene families often occur as clusters of 3 or more adjacent genes, in addition to the widespread existence of tandem repeats (Figure 4). For example, members of the GGPPS (17) family form 3-gene clusters on chromosome 3; the family of mono/sesquiterpene synthase-like genes (93) is present in larger clusters on chromosome 3 (3 clusters with 2, 3 and 4 genes each) and chromosome 4 (2 clusters with 2 genes, one cluster with 3 genes); the triterpene-synthase-like sequences (116) occur as a 3-gene-cluster on chromosome 1; the squalene monooxygenase-like genes (53) constitute a 3-gene cluster on chromosome 5; members of the steroid sulfotransferase gene family cluster on chromosome 2 (3 genes); and the 11β-hydroxysteroid dehydrogenase gene family is present as a single large cluster on chromosome 5 (5 genes). Pathway clustering is also apparent for genes putatively involved in phylloquinone biosynthesis: DHNA synthase (77) and DHNA phytyltransferase (78) are located three open reading frames apart on chromosome 1, and SHCHC synthase (74) and OSB synthase (75) are adjacent to one another further downstream on the same chromosome. Interestingly, a gene cluster for the genes putatively involved in menaquinone (a quinone analogue to phylloquinone synthesized in eubacteria) biosynthesis exists in *E. coli* but not in the cyanobacterium *Synechocystis*. Of the genes involved in chlorophyll metabolism, heme oxygenase (91) isogenes are clustered separately on chromosome 1, together with glutamyl-tRNA reductase (33) and with aminolevulinate dehydratase (35). Since heme is known to be a negative regulator of glutamyl-tRNA reductase (33), the co-regulation of these neighboring genes could be a key switch in controlling chlorophyll biosynthesis. On chromosome 4, a Mg-chelatase isogene is located immediately next to a gene which encodes an apparently essential chloroplast enzyme of as-yet-unknown function (Koncz *et al.*, 1990). The genes coding for the two isoenzymes of IPPI (14) are both clustered together with a C-methyltransferase likely involved in the biosynthesis of a quinone. On chromosome 5, AACT (8) is located close to FPPS (16) (4 ORFs downstream), cycloartenol synthase (54) (another 20 ORFs downstream), a putative sesquiterpene synthase (93) (another 9 ORFs downstream) and an additional AACT (8) gene (another 20 ORFs downstream). An intriguing feature of the *A. thaliana* genome is the occurrence of several clusters of GGPPS-like (17) and

Figure 4. Genomic organization of isoprenoid metabolism and related genes in *A. thaliana*. The positions of genes mentioned in this study are indicated by a vertical bar adjacent to their location on the chromosome map. Enzymes involved in isoprenoid metabolism are numbered according to the annotation given in Table 1 (for details see text).

terpene synthase-like (93) genes, and additional clusters with cytochrome P450 genes of as yet unknown function (Figure 2). Larger clusters of genes involved in isoprenoid biosynthesis are located on chromosome 1 (epoxycarotenoid cleavage enzyme (30), gibberellin 2-oxidase (88), a cytochrome P450, a putative triterpene synthase (116), a polyprenyl diphosphate synthase (67) and an oxoglutarate-dependent dioxygenase of unknown function) and on chromosome 3 (3 terpene synthase-like (93) genes, 3 GGPPS-like (17) genes, 5 cytochrome P450 genes and a gene of the epoxycarotenid cleavage enzyme (30) family).

### Conclusions

The current analysis of pathways involved in isoprenoid and chlorophyll metabolism represents a framework for future endeavors to characterize the missing links and to study the regulatory events controlling these pathways. The bioinformatic evaluation of the present knowledge and predictions as to which

additional genes are part of isoprenoid and chlorophyll metabolism, will have to be complemented by mutant and biochemical approaches to unravel the true function of the candidate genes mentioned in our survey. The diversity of genes encoded in the *A. thaliana* genome is intriguing and may indicate that the metabolic capabilities of this model plant have been vastly underestimated in the past. Multi-copy gene families encode several key metabolic enzymes, which may facilitate the tightly regulated expression of isoenzymes specific to certain tissues, developmental stages and/or environmental challenges.

### Acknowledgements

We would like to thank Dr P.B. Heifetz (Torrey Mesa Research Institute of Syngenta) for valuable discussions and comments.

Note added in proof

After the submission of the revised version of this manuscript, a report describing the cloning and characterization of a tocopherol cyclase from *A. thaliana* has been published (Porfirova, S., Bergmuller, E., Tropf, S., Lemke, R. and Dormann, P. 2002. Isolation of an *Arabidopsis* mutant lacking vitamin E and identification of a cyclase essential for all tocopherol biosynthesis. Proc. Natl. Acad. Sci. USA 99: 12495–12500).

References

Adam, K.P., Thiel, R. and Zapp, J. 1999. Incorporation of 1-[1-(13)C]deoxy-D-xylulose in chamomile sesquiterpenes. Arch. Biochem. Biophys. 369: 127–132.

Al-Babili, S., Hugueney, P., Schledz, M., Welsch, R., Frohnmeier, H., Laule, O. and Beyer, P. 2000. Identification of a novel gene coding for neoxanthin synthase from *Solanum tuberosum*. FEBS Lett. 485: 168–172.

Albrecht, M., Klein, A., Hugueney, P., Sandmann, G. and Kuntz, M. 1995. Molecular cloning and functional expression in *E. coli* of a novel plant enzyme mediating ζ-carotene desaturation. FEBS Lett. 372: 199–202.

Altschul, S.F., Gis, W., Miller, W., Myers, E.W. and Lipman, D.J. 1990. Basic local alignment search tool. J. Mol. Biol. 215: 403–410.

Andres, D.A., Deabra, M.C., Brown, M.S., Armstrong, S.A., Smeland, T.E., Cremers, F.P. and Goldstein, J.L. 1993. cDNA cloning of component A of Rab geranylgeranyl transferase and demonstration of its role as a Rab escort protein. Cell 73: 1091–1099.

Arabidopsis Genome Initiative 2000. Analysis of the genome sequence of the flowering plant *Arabidopsis thaliana*. Nature 408: 796–815.

Armstrong, G.A., Runge, S., Fric, G., Sperling, U. and Apel, K. 1995. Identification of NADPH:protochlorophyllide oxidoreductases A and B: a branched pathway for light-dependent chlorophyll biosynthesis in *Arabidopsis thaliana*. Plant Physiol. 108: 1505–1517.

Armstrong, S.A., Seabra, M.C., Sudhof, T.C., Goldstein, J.L. and Brown, M.S. 1993. cDNA cloning and expression of the alpha and beta subunits of rat Rab geranylgeranyl transferase. J. Biol. Chem. 268: 12221–12229.

Aubourg, S., Lecharny, A. and Bohlmann, J. 2002. Genomic analysis of the terpenoid synthase (AtTPS) gene family of *Arabidopsis thaliana*. Mol. Genet. Genomics 267: 730–745.

Avelange-Macharel, M.H. and Joyard, J. 1998. Cloning and functional expression of AtCOQ3, the *Arabidopsis* homologue of the yeast COQ3 gene, encoding a methyl-transferase from plant mitochondria involved in ubiquinone biosynthesis. Plant J. 14: 203–213.

Bartley, G.E., Viitanen, P.V., Pecker, I., Chamovitz, D., Hirschberg, J. and Scolnik, P.A. 1991. Molecular cloning and expression in photosynthetic bacteria of a soybean cDNA coding for phytoene desaturase, an enzyme of the carotenoid biosynthetic pathway. Proc. Natl. Acad. Sci. USA 88: 6532–6536.

Bartley, G.E., Viitanen, P.V., Bacot, K.O. and Scolnik, P.A. 1992. A tomato gene expressed during fruit ripening encodes an enzyme o the carotenoid biosynthetic pathway. J. Biol. Chem. 267: 5036–5039.

Benedetti, C.E., Costa, C.L., Turcinelli, S.R. and Arruda, P. 1998. Differential expression of a novel gene in response to coronatine, methyl jasmonate, and wounding in the Coil mutant of *Arabidopsis*. Plant Physiol. 116: 1037–1042.

Benli, M., Schulz, R. and Apel, K. 1991. Effect of light on the NADPH-protochlorophyllide oxidoreductase of *Arabidopsis thaliana*. Plant Mol. Biol. 16: 615–625.

Boese, Q.F., Spano, A.J., Li, J.M. and Timko, M.P. 1991. Aminolevulinic acid dehydratase in pea (*Pisum sativum* L.). Identification of an unusual metal-binding domain in the plant enzyme. J. Biol. Chem. 266: 17060–17066.

Bohlmann, J., Martin, D., Oldham, N.J. and Gershenzon, J. 2000. Terpenoid secondary metabolism in *Arabidopsis thaliana*: cDNA cloning, characterization, and functional expression of a myrcene/(E)-β-ocimene synthase. Arch. Biochem. Biophys. 375: 261–269.

Bouvier, R., Suire, C., d’Harlingue, A., Backhaus, R.A. and Camara, B. 2000a. Molecular cloning of geranyl diphosphate synthase and compartmentation of monoterpene synthesis in plant cells. Plant J. 24: 241–252.

Bouvier, F., d’Harlingue, A., Backhaus, R.A., Kumagai, M.H. and Camara, B. 2000b. Identification of neoxanthin synthase as a carotenoid cyclase paralog. Eur. J. Biochem. 267: 6346–6352.

Bouvier-Nave, P., Husselstein, T., Desprez, T. and Benveniste, P. 1997. Identification of cDNAs encoding sterol methyltransferases involved in the second methylation step of plant sterol biosynthesis. Eur. J. Biochem. 246: 518–529.

Bugos, R.C., Hieber, A.D. and Yamamoto, H.Y. 1998. Xanthophyll cycle enzymes are members of the lipocalin family, the first identified from plants. J. Biol. Chem. 273: 15321–15324.

Caelles, C., Ferrer, A., Balcells, L., Hegardt, F.G. and Boronat, A. 1989. Isolation and structural characterization of a cDNA encoding *Arabidopsis thaliana* 3-hydroxy-3-methylglutaryl coenzyme A reductase. Plant Mol. Biol. 13: 627–638.

Caldelari, D., Sternberg, H., Rodriguez-Concepcion, M., Grussem, W. and Yalovsky, S. 2001. Efficient prenylation by a plant geranylgeranyltransferase-I requires a functional CaaL box motif and a proximal polybasic domain. Plant Physiol. 126: 1416–1429.

Campbell, M., Hahn, F.M., Poulter, C.D. and Leustek, T. 1997. Analysis of the isopentenyl diphosphate isomerase gene family from *Arabidopsis thaliana*. Plant Mol. Biol. 36: 323–328.

Choe, S., Dilkes, B.P., Fujioka, S., Takatsuto, S., Sakurai, A. and Feldmann, K.A. 1998. The DWF4 gene of *Arabidopsis* encodes a cytochrome P450 that mediates multiple 22α-hydroxylation steps in brassinosteroid biosynthesis. Plant Cell 10: 231–243.

Chow, K.S., Singh, D.P., Walker, A.R. and Smith, A.G. 1998. Two different genes encode ferrochelatase in *Arabidopsis*: mapping, expression and subcellular targeting of the precursor proteins. Plant J. 15: 531–541.

Collakova, E. and DellaPenna, D. 2001. Isolation and functional analysis of homogentisate phytyltransferase from *Synechocystis* sp. PCC6803 and *Arabidopsis*. Plant Physiol. 127: 1113–1124.

Cordier, H., Karst, F. and Berges, T. 1999. Heterologous expression in *Saccharomyces cerevisiae* of an *Arabidopsis thaliana* cDNA encoding mevalonate diphosphate decarboxylase. Plant Mol. Biol. 39: 953–967.

Corey, E.J., Matsuda, S.P. and Bartel, B. 1993. Isolation of an *Arabidopsis thaliana* gene encoding cycloartenol synthase by functional expression in a yeast mutant lacking lanosterol synthase by the use of a chromatographic screen. Proc. Natl. Acad. Sci. USA 90: 11628–11632.

Crowell, D.N. 2000. Functional implications of protein isoprenylation in plants. Progr. Lipid Res. 39: 393–408.

Cunillera, N., Boronat, A. and Ferrer, A. 1997. The *Arabidopsis thaliana FPS1* gene generates a novel mRNA that encodes a mitochondrial farnesyl-diphosphate synthase isoform. J. Biol. Chem. 272: 15381–15388.

Cunillera, N., Arro, M., Fores, O., Manzano, D. and Ferrer, A. 2000. Characterization of dehydrolicolichol diphosphate synthase of *Arabidopsis thaliana*, a key enzyme in dolichol biosynthesis. FEBS Lett. 477: 170–174.

Cunningham, F.X. and Gantt, E. 2000. Identification of multi-gene families encoding isopentenyl diphosphate isomerase in plants by heterologous complementation in *Escherichia coli*. Plant Cell Physiol. 41: 119–123.

Cunningham, F.X., Pogson, B., Sun, Z., McDonald, K.A., Delapenna, D. and Gantt, E. 1996. Functional analysis of the β and ε lycopene cyclase enzymes of *Arabidopsis* reveals a mechanism for control of cyclic carotenoid formation. Plant Cell 8: 1613–1626.

Darnet, S., Bard, M. and Rahier, A. 2001. Functional identification of sterol-4α-methyl oxidase cDNAs from *Arabidopsis thaliana* by complementation of a yeast erg25 mutant lacking sterol-4-α-methyl oxidation. FEBS Lett. 508: 39–43.

Davis, S.J., Kurepa, J. and Viestra, R.D. 1999. The *Arabidopsis thaliana HY1* locus, required for phytochrome-chromophore biosynthesis, encodes a protein related to heme oxygenases. Proc. Natl. Acad. Sci. USA 96: 6541–6546.

Davis, S.J., Bhoo, S.H., Durski, A.M., Walker, J.M. and Vierstra, R.D. 2001. The heme-oxygenase family required for phytochrome chromophore biosynthesis is necessary for proper photomorphogenesis in higher plants. Plant Physiol. 126: 656–669.

Day, I.S., Golovkin, M. and Reddy, A.S. 1998. Cloning of the cDNA for glutamyl-tRNA synthetase from *Arabidopsis thaliana*. Biochim. Biophys. Acta 1399: 219–224.

Delourme, D., Lacroute, F. and Karst, F. 1994. Cloning of an *Arabidopsis thaliana* cDNA coding for farnesyl diphosphate synthase by functional complementation in yeast. Plant Mol. Biol. 26: 1867–1873.

Demming-Adams, B., Gilmore, A.M. and Adams, W.W. 1996. Carotenoids. 3. *In vivo* function of carotenoids in higher plants. FASEB J. 10: 403–412.

Disch, A., Hemmerlin, A., Bach, T.J. and Rohmer, M. 1998. Mevalonate-derived isopentenyl diphosphate is the biosynthetic precursor of ubiquinone prenyl side chain in tobacco BY-2 cells. Biochem. J. 15: 615–621.

Eisenreich, W., Rohdich, F. and Bacher, A. 2001. Deoxyxylulose phosphate pathway to terpenoids. Trends Plant Sci. 6: 78–84.

Enjuto, M., Balcells, L., Campos, N., Caelles, C., Arro, M. and Boronat, A. 1994. *Arabidopsis thaliana* contains two differentially expressed 3-hydroxy-3-methylglutaryl-CoA reductase genes, which encode microsomal forms of the enzyme. Proc. Natl. Acad. Sci. USA 91: 927–931.

Espineda, C.E., Linford, A.S., Devine, D. and Brusslan, J.A. 1999. The AtCAO gene, encoding chlorophyll *a* oxygenase, is required for chlorophyll *b* synthesis in *Arabidopsis thaliana*. Proc. Natl. Acad. Sci. USA 96: 10507–10511.

Estevez, J.M., Cantero, A., Reindl, A., Reichler, S. and Leon, P. 2001. 1-Deoxy-D-xylulose-5-phosphate synthase, a limiting enzyme for plastidic isoprenoid biosynthesis in plants. J. Biol. Chem. 276: 22901–22909.

Frey, M., Chomet, P., Glawischnig, E., Stettner, C., Grün, S., Winklmair, A., Eisenreich, W., Bacher, A., Meeley, R.B., Briggs, S.P., Simcox, K. and Gierl, A. 1997. Analysis of a chemical plant defense mechanism in grasses. Science 277: 696–699.

Gachotte, D., Husselstein, T., Bard, M., Lacroute, F. and Benveniste, P. 1996. Isolation and characterization of an *Arabidopsis thaliana* cDNA encoding a Δ⁷-sterol-C5-desaturase by functional complementation of a defective yeast mutant. Plant J. 9: 391–398.

Gaubier, P., Wu, H.J., Laudie, M., Delseny, M. and Grellet, F. 1995. A chlorophyll synthetase gene from *Arabidopsis thaliana*. Mol. Gen. Genet. 249: 58–64.

Grebenok, R.J., Ohnmeiss, T.E., Yamamoto, A., Huntley, E.D., Galbraith, D.W. and DellaPenna, D. 1998. Isolation and characterization of an *Arabidopsis thaliana* C-8,7 sterol isomerase: functional and structural similarities to mammalian C-8,7 sterol isomerase/emopamil-binding protein. Plant Mol. Biol. 38: 807–815.

Grimm, B. 1990. Primary structure of a key enzyme in plant tetrapyrrole synthesis: glutamate 1-semialdehyde aminotransferase. Proc. Natl. Acad. Sci. USA 87: 4169–4173.

Hecht, S., Eisenreich, W. Adam, P., Amslinger, S., Kis, K., Bacher, A., Arigoni, D. and Rohdich, F. 2001. Studies on the nonmevalonate terpene biosynthetic pathway: the role of the GcpE (IspG) protein. Proc. Natl. Acad. Sci. USA 98: 14837–14842.

Helliwell, C.A., Poole, A., Peacock, W.J. and Dennis, E.S. 1999. *Arabidopsis* ent-kaurene oxidase catalyzes three steps of gibberellin biosynthesis. Plant Physiol. 119: 507–510.

Helliwell, C.A., Chandler, P.M., Poole, A., Dennis, E.S. and Peacock, W.J. 2001. The CYP88A cytochrome P450, ent-kaurene acid oxidase, catalyzes three steps of the gibberellin biosynthesis pathway. Proc. Natl. Acad. Sci. USA 98: 2065–2070.

Herrera, J.B., Bartel, B., Wislon, W.K. and Matsuda, S.P. 1998. Cloning and characterization of the *Arabidopsis thaliana* lupeol synthase gene. Phytochemistry 49: 1905–1911.

Higgins D., Thompson J., Gibson T., Thompson J.D., Higgins D.G. and Gibson T.J. 1994. CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucl. Acids Res. 22: 4673–4680.

Hintz, M., Reichenberg, A., Altincicek, B., Bahr, U., Gschwind, R.M., Kollas, A.K., Beck, E., Wiesner, J., Eberl, M. and Jomaa, H. 2001. Identification of (E)-4-hydroxy-3-methyl-but-2-enyl pyrophosphate as a major activator for human γδ T cells in *Escherichia coli*. FEBS Lett. 509: 317–322.

Hörtensteiner S. 1998. NCC malonyltransferase catalyzes the final step of chlorophyll breakdown in rape (*Brassica napus*). Phytochemistry 49: 953–956.

Husselstein, T., Gachotte, D., Desprez, T., Bard, M. and Benveniste, P. 1996. Transformation of *Saccharomyces cerevisiae* with a cDNA encoding a sterol C-methyltransferase from *Arabidopsis thaliana* results in the synthesis of 24-ethyl sterols. FEBS Lett. 381: 87–92.

Husselstein-Muller, T., Schaller, H. and Benveniste, P. 2001. Molecular cloning and expression in yeast of 2,3-oxidosqualene triterpenoid cyclases from *Arabidopsis thaliana*. Plant Mol. Biol. 45: 75–92.

Ishikawa A., Okamoto, H., Iwasaki, Y. and Asahi, T. 2001. A deficiency of coproporphyrinogen III oxidase causes lesion formation in *Arabidopsis*. Plant J. 27: 89–99.

Johnson, T.W., Shen, G., Zybailov, B., Kolling, D., Raetegui, R., Beauparlant, S., Vassiliev, I.R., Bryant, D.A., Jones, D., Golbeck, J.H. and Chitnis, P. 2000. Recruitment of a foreign quinone into the A1 site of photosystem I. J. Biol. Chem. 275: 8523–8530.

Kanazin, V., Marek, L.F. and Shoemaker, R.C. 1996. Resistance gene analogs are conserved and clustered in soybean. Proc. Natl. Acad. Sci. USA 93: 11746–11750.

Keller, Y., Bouvier, F., d’Harlingue, A. and Camara, B. 1998. Metabolic compartmentation of plastid prenyllipid biosynthesis. Evidence for the involvement of a multifunctional geranylgeranyl reductase. Eur. J. Biochem. 251: 413–417.

Klahre, U., Nogushi T., Fujioka, S., Takatsuto, S., Yokota, T. Nomura, T., Yoshida, S. and Chua, N.H. 1998. The *Arabidopsis* DIMINUTO/DWARF1 gene encodes a protein involved in sterol biosynthesis. Plant Cell 10: 1677–1690.

Kohchi, T., Mukougawa, K., Frankenberg, N., Masuda, M., Yokota, A. and Lagarias, J.C. 2001. The *Arabidopsis hy2* gene encodes phytochromobilin synthase, a ferredoxin-dependent biliverdin reductase. Plant Cell 13: 425–436.

Koncz, C., Mayerhofer, R., Koncz-Kalman, Z., Nawrath, C., Reiss, B., Redei, G.P. and Schell, J. 1990. Isolation of a gene encoding a novel chloroplast protein by T-DNA tagging in *Arabidopsis thaliana*. EMBO J. 9: 1337–1346.

Krochko, J.E., Abrams, G.D., Loewen, M.K., Abrams, S.R. and Cutler, A.J. 1998. (+)-Abscisic acid 8′-hydroxylase is a cytochrome P450 monooxygenase. Plant Physiol. 118: 849–860.

Kumar, A.M. and Soll, D. 1992. *Arabidopsis* alternative oxidase sustains *Escherichia coli* respiration. Proc. Natl. Acad. Sci. USA 89: 10842–10846.

Kumar, A.M., Csankovszki, G. and Soll, D. 1996. A second and differentially expressed glutamyl-tRNA reductase gene from *Arabidopsis thaliana*. Plant Mol. Biol. 30: 419–426.

Kushiro, M., Nakano, T., Sato, K., Yamagishi, K., Asami, T., Nakano, A., Takatsuto, S., Fujioka, S., Ebizuka, Y. and Yoshida, S. 2001. Obtusifoliol 14α-demethylase (cyp51) antisense *Arabidopsis* shows slow growth and long life. Biochem. Biophys. Res. Commun. 285: 98–104.

Lange, B.M., Rujan, T., Martin, W. and Croteau, R. 2000. Iso-prenoid biosynthesis: the evolution of two ancient and distinct pathways across genomes. Proc. Natl. Acad. Sci. USA 97: 13172–13177.

Lange, T. 1997. Cloning gibberellin dioxygenase genes from pumpkin endosperm by heterologous expression of enzyme activities in *Escherichia coli*. Proc. Natl. Acad. Sci. USA 94: 6553–6558.

Lecain, E., Chenivesse, X., Spagnoli, R. and Pompon, D. 1996. Cloning by metabolic interference in yeast and enzymatic characterization of *Arabidopsis thaliana* sterol Δ⁷-reductase. J. Biol. Chem. 271: 10866–10873.

Li, J., Nagpal, P., Vitart, V., McMorris, T.C. and Chory, J. 1996. A role for brassinosteroids in light-dependent development of *Arabidopsis*. Science 272: 398–401.

Lichtenthaler, H.K., Schwender, J., Disch, A. and Rohmer, M. 1997. Biosynthesis of isoprenoids in higher plant chloroplasts proceeds via a mevalonate-independent pathway. FEBS Lett. 400: 271–274.

Lim, S.H., Witty, M., Wallace-Cook, A.D., Ilag, L.I. and Smith, A.G. 1994. Porphobilinogen deaminase is encoded by a single gene in *Arabidopsis thaliana* and is targeted to the chloroplasts. Plant Mol. Biol. 26: 863–872.

Lin, L.J., Tai, S.S.K., Peng, C.C. and Tzen, J.T.C. 2002. Steroleosin, a sterol-binding dehydrogenase in seed oil bodies. Plant Physiol. 128: 1200–1211.

Lopukhina, A., Dettenberg, W., Weiler, E.W. and Holländer-Czytko, H. 2001. Cloning and characterization of a coronatine-regulated tyrosine aminotransferase from *Arabidopsis*. Plant Physiol. 126: 1678–1687.

Lovato, M.A., Hart, E.A., Segura, M.J.R., Giner, J.L. and Matsu, S.P.T. 2000. Functional cloning of an *Arabidopsis thaliana* cDNA encoding cycloeucalenol cycloisomerase. J. Biol. Chem. 275: 13394–13397.

Mach, J.M., Castillo, A.R., Hoogstraten, R. and Greenberg, J.T. 2001. The *Arabidopsis* accelerated cell death gene ACD2 encodes red chlorophyll catabolite reductase and suppresses the spread of disease symptoms. Proc. Natl. Acad. Sci. USA 98: 771–776.

Marin, E., Nussaume, L., Quesada, A., Gonneau, M., Sotta, B., Hugueney, P., Frey, A. and Marion-Poll, A. 1996. Molecular identification of zeaxanthin epoxidase of *Nicotiana plumbaginifolia*, a gene involved in abscisic acid biosynthesis and corresponding to the ABA locus of *Arabidopsis thaliana*. EMBO J. 15: 2331–2342.

Mayfield, J.A., Fiebig, A., Johnstone, S.E. and Preuss, D. 2001. Gene families from the *Arabidopsis thaliana* pollen coat proteome. Science 292: 2482–2485.

Meganathan, R. 2001. Ubiquinone biosynthesis in microorganisms. FEMS Microbiol. Lett. 203: 131–139.

Meskauskiene, R., Nater, M., Goslings, D., Kessler, F., op den Camp, R. and Apel, K. 2001. FLU: a negative regulator of chlorophyll biosynthesis in *Arabidopsis thaliana*. Proc. Natl. Acad. Sci. USA 98: 12826–12831.

Mock, H.P., Trainotti, L., Kruse, E. and Grimm, B. 1995. Isolation, sequencing and expression of cDNA sequences encoding uroporphyrinogen decarboxylase from tobacco and barley. Plant Mol. Biol. 28: 245–256.

Montamat, F., Guilloton, M., Karst, F. and Delrot, S. 1995. Isolation and characterization of a cDNA encoding *Arabidopsis thaliana* 3-hydroxy-3-methylglutaryl-coenzyme A synthase. Gene 167: 197–201.

Muramoto, T., Kohchi, T., Yokota, A., Hwang, I. and Goodman, H.M. 1999. The *Arabidopsis* photomorphogenic mutant *hy1* is deficient in phytochrome chromophore biosynthesis as a result of a mutation in a plastid heme oxygenase. Plant Cell 11: 335–348.

Nakashima, T., Inoue, T., Oka, A., Nishino, T. Osumi, T. and Hata, S. 1995. Cloning, expression, and characterization of cDNAs encoding *Arabidopsis thaliana* squalene synthase. Proc. Natl. Acad. Sci. USA 92: 2328–2332.

Narita, S., Tanaka, R., Ito, T., Okada, K., Taketani, S. and Inokuchi, H. 1996. Molecular cloning and characterization of a cDNA that encodes protoporphyrinogen oxidase of *Arabidopsis thaliana*. Gene 182: 169–175.

Nes, W.D. and Venkatramesh, M. 1999. Enzymology of phytosterol transformations. Crit. Rev. Biochem. Mol. Biol. 34: 81–93.

Norris, S.R., Barrette, T.R. and DellaPenna, D. 1995. Genetic dissection of carotenoid synthesis in *Arabidopsis* defines plastoquinone as an essential component of phytoene desaturation. Plant Cell. 7: 2139–2149.

Norris, S.R., Shen, X. and DellaPenna, D. 1998. Complementation of the *Arabidopsis pds1* mutation with the gene encoding p-hydroxyphenylpyruvate dioxygenase. Plant Physiol. 117: 1317–1323.

Ogura, K. 1999. Isomerase and prenyl transferases. In: D.E. Cane (Ed.) Comprehensive Natural Products Chemistry, Vol. 2: Isoprenoids Including Carotenoids and Steroids, Pergamon Press, Oxford, pp. 69–96.

Okada, K., Kamiya, Y., Zhu, X., Suzuki, K., Tanaka, K., Nakagawa, T., Matsuda, H. and Kawamukai, M. 1997. Cloning of the *sdsA* gene encoding solanesyl diphosphate synthase from *Rhodobacter capsulatus* and its functional expression in *Escherichia coli* and *Saccharomyces cerevisiae*. J. Bact. 179: 5992–5998.

Okada, K., Saito, T., Nakagawa, T., Kawamukai, M. and Kamiya, Y. 2000. Five geranylgeranyl diphosphate synthases expressed in different organs are localized into three subcellular compartments in *Arabidopsis*. Plant Physiol. 122: 1045–1056.

Papenbrock, J., Gräfe, S., Kruse, E., Hänel, F. and Grimm, B. 1997. Mg-chelatase of tobacco: identification of a ChlD cDNA sequence encoding a third subunit, analysis of the interaction of the three subunits with the yeast two-hybrid system, and reconstruction of the enzyme activity by co-expression of recombinant ChlD, ChlH and ChlI. Plant J. 12: 981–990.

Park, H., Kreunen, S.S., Cuttries, A.J., DellaPenna, D. and Pogson, B.J. 2002. Identification of the carotenoid isomerase provides insight into carotenoid biosynthesis, prolamellar body formation, and photomorphogenesis. Plant Cell 14: 321–332.

Parmryd, I., Andersson, B. and Dallner, G. 1999. Protein prenylation in spinach chloroplasts. Proc. Natl. Acad. Sci. USA 96: 10074–10079.

Pecker, I., Gabbay, R., Cunningham, F.X. and Hirschberg, J. 1996. Cloning and characterization of the cDNA for lycopene β-cyclase from tomato reveals decrease in its expression during fruit ripening. Plant Mol. Biol. 30: 807–819.

Phillips, A.L., Ward, D.A., Uknes, A., Appleford, N.E., Lange, T., Hutty, A.K., Gaskin, P., Gräbe, J.E. and Hedden, P. 1995. Isolation and expression of three gibberellin 20-oxidase cDNA clones from *Arabidopsis*. Plant Physiol. 108: 1049–1057.

Piel J., Donath, J., Bandemer, K. and Boland, W. 1998. Mevalonate-independent biosynthesis of terpenoid volatiles in plants: induced and constitutive emission of volatiles. Angew. Chem. 37: 2478–2481.

Pontoppidan, B. and Kannangara, C.G. 1994. Purification and partial characterisation of barley glutamyl-tRNA(Glu) reductase, the enzyme that directs glutamate to chlorophyll biosynthesis. Eur. J. Biochem. 225: 529–537.

Qian, D., Zhou, D., Ju, R., Cramer, C.L. and Yang, Z. 1996. Protein farnesyltransferase in plants: molecular characterization and involvement in cell cycle control. Plant Cell 8: 2381–2394.

Querol, J., Campos, N., Imperial, S., Boronat, A. and Rodriguez-Concepcion, M. 2002. Functional analysis of the *Arabidopsis thaliana* gcpE protein involved in plastid isoprenoid biosynthesis. FEBS Lett. 514: 343–346.

Riou, C., Tourte, Y., Lacroute, F. and Karst, F. 1994. Isolation and characterization of a cDNA encoding *Arabidopsis thaliana* mevalonate kinase by genetic complementation in yeast. Gene 148: 293–297.

Rogerson, F.M., LeHoux, J.G. and Mason, J.I. 1995. Expression and characterization of isoforms of 3β-hydroxysteroid dehydrogenase/δ5→4-isomerase in the hamster. J. Steroid Biochem. Mol. Biol. 55: 481–487.

Rohdich, F., Wungsintaweekul, J., Eisenreich, W., Richter, G., Schuhr, C.A., Zenk, M.H. and Bacher, A. 2000a. Biosynthesis of terpenoids: 4-diphosphocytidyl-2-C-methyl-D-erythritol synthase of *Arabidopsis thaliana*. Proc. Natl. Acad. Sci. USA 97: 6451–6456.

Rohdich, F., Wungsintaweekul, J., Lüttgen, H., Fischer, M., Eisenreich, W., Schuhr, C.A., Fellermeier, M., Schramek, N., Zenk, M.H. and Bacher, A. 2000b. Biosynthesis of terpenoids: 4-diphosphocytidyl-2-C-methyl-D-erythritol kinase from tomato. Proc. Natl. Acad. Sci. USA 97: 8251–8256.

Rohdich, F., Hecht, S., Gärtner, K., Adam, P., Krieger, C., Am-slinger S., Arigoni, D., Bacher, A. and Eisenreich, W. 2002. Studies on the nonmevalonate terpene biosynthetic pathway: metabolic role of IspH (LytB) protein. Proc. Natl. Acad. Sci. USA 99: 1158–1163.

Rouleau, M., Marsolais, F., Richard, M., Nicolle, L., Voigt, B., Adam, G. and Varin, L. 1999. Inactivation of brassinosteroid biological activity by a salicylate-inducible steroid sulfotransferase from *Brassica napus*. J. Biol. Chem. 274: 20925–20930.

Schäfer, U.A., Reed, D.W., Hunter, D.G., Yao, K., Weninger, A.M., Tsang, E.W., Reaney, M.J., McKenzie, S.L. and Covello, P.S. 1999. An example of intron junctional sliding in the gene families encoding squalene monooxygenase homologues in *Arabidopsis thaliana* and *Brassica napus*. Plant Mol. Biol. 39: 721–728.

Schrick, K., Mayer, U., Horrichs, A., Kuhnt, C., Bellini, C., Dangl, J., Schmidt, J. and Jürgens, G. 2000. FACKEL is a sterol C-14 reductase required for organized cell division and expansion in *Arabidopsis* embryogenesis. Genes Dev. 14: 1471–1484.

Schultz, G., Soll, J., Fiedler, E. and Schulze-Siebert, D. 1985. Synthesis of prenylquinones in chloroplasts. Physiol. Plant. 64: 123–129.

Schwartz, S.H., Tan, B.C., Gage, D.A., Zeevaart, J.A. and McCarty, D.R. 1997. Specific oxidative cleavage of carotenoids by VP14 in maize. Science 276: 1872–1874.

Schwender, J., Müller, C., Zeidler, J. and Lichtenthaler, H.K. 1999. Cloning and heterologous expression of a cDNA encoding 1-deoxy-D-xylulose-5-phosphate reductoisomerase of *Arabidopsis thaliana*. FEBS Lett. 455: 140–144.

Seo, M., Peeters, A.J., Koiwai, H., Oritani, T., Marion-Poll, A., Ze-vart, J.A., Koornneef, M., Kamiya, Y. and Koshiba, T. 2000. The *Arabidopsis* aldehyde oxidase 3 (AAO3) gene product catalyzes the final step in abscisic acid biosynthesis in leaves. Proc. Natl. Acad. Sci. USA 97: 12908–12913.

Shi, J., Gonzales, R.A. and Bhattacharya, M.K. 1996. Identification and characterization of an S-adenosyl-L-methionine: Δ24-sterol-C-methyltransferase cDNA from soybean. J. Biol. Chem. 271: 9384–9389.

Shintani, D. and DellaPenna, D. 1998. Elevating the vitamin E content of plants through metabolic engineering. Science 282: 2098–2100.

Smith, C.A., Suzuki, J.Y. and Bauer, C.E. 1996. Cloning and characterization of the chlorophyll biosynthesis gene *chlM* from *Synechocystis* PCC 6803 by complementation of a bacteriochlorophyll biosynthesis mutant of *Rhodobacter capsulatus*. Plant Mol. Biol. 30: 1307–1314.

Su, Q., Frick, G., Armstrong, G. and Apel, K. 2001. POR C of *Arabidopsis thaliana*: a third light- and NADPH-dependent protochlorophyllide oxidoreductase that is differentially regulated by light. Plant Mol. Biol. 47: 805–813.

Sun, T.P. and Kamiya, Y. 1994. The *Arabidopsis GA1* locus encodes the cyclase ent-kaurene synthetase A of gibberellin biosynthesis. Plant Cell 6: 1509–1518.

Sun, Z., Gantt, E. and Cunningham, F.X. 1996. Cloning and functional analysis of the β-carotene hydroxylase of *Arabidopsis thaliana*. J. Biol. Chem. 271: 24349–24352.

Swiezewska, E., Dallner, G., Andersson, B. and Ernster, L. 1993. Biosynthesis of ubiquinone and plastoquinone in the endoplasmic reticulum-Golgi membranes of spinach leaves. J. Biol. Chem. 268: 1494–1499.

Szekeres, M., Németh, K., Koncz-Kálmán, Mathur, J., Kauschmann, A., Altmann, T., Rédei, G.P., Nagy, F., Schell, J. and Koncz, C. 1996. Brassinosteroids rescue the deficiency of CYP90, a cytochrome P450, controlling cell elongation and de-etiolation in *Arabidopsis*. Cell 85: 171–182.

Thomas, S.G., Phillips, A.L. and Hedden, P. 1999. Molecular cloning and functional expression of gibberellin 2-oxidases, multifunctional enzymes involved in gibberellin deactivation. Proc. Natl. Acad. Sci. USA 96: 4698–4703.

Tsay, Y.H. and Robinson, G.W. 1991. Cloning and characterization of ERG8, an essential gene of *Saccharomyces cerevisiae* that encodes phosphomevalonate kinase. Mol. Cell. Biol. 11: 620–631.

Tsuchiya, T., Ohta, H., Okawa, K., Iwamatsu, A., Shimada, H., Masuda, T. and Takamiya, K.I. 1999. Cloning of chlorophyllase, the key enzyme in chlorophyll degradation: finding of a lipase motif and the induction by methyl jasmonate. Proc. Natl. Acad. Sci. USA 96: 15362–15367.

Van Poecke, R.M.P., Posthumus, M.A. and Dicke, M. 2001. Herbivore-induced volatile production by *Arabidopsis thaliana* leads to attraction of the parasitoid *Cotesia rubecula*: chemical, behavioral, and gene-expression analysis. J. Chem. Ecol. 27: 1911–1928.

Vollack, K.U. and Bach, T.J. 1996. Cloning of a cDNA encoding cytosolic acetoacetyl-coenzyme A thiolase from radish by functional expression in *Saccharomyces cerevisiae*. Plant Physiol. 111: 1097–1107.

Warnecke, D.C., Baltrusch, M., Buck, F., Wolter, F.P. and Heinz, E. 1997. UDP-glucose:sterol glucosyltransferase: cloning and functional expression in *Escherichia coli*. Plant Mol. Biol. 35: 597–603.

Watanabe, N., Che, F.S., Iwano, M., Takayama, S., Yoshida, S. and Isogai, A. 2001. Dual targeting of spinach protoporphyrinogen oxidase II to mitochondria and chloroplasts by alternative use of two in-frame initiation codons. J. Biol. Chem. 276: 20474–20481.

Wildermuth, M.C., Dewdney, J., Wu, G. and Ausubel, F.M. 2001. Isochorismate synthase is required to synthesize salicylic acid for plant defence. Nature 414: 562–565.

Williams, J., Phillips, A.L., Gaskin, P. and Hedden, P. 1998. Function and substrate specificity of the gibberellin 3β-hydroxylase encoded by the *Arabidopsis GA4* gene. Plant Physiol. 117: 559–563.

Wu, D., Wright, D.A., Wetzel, C., Voytas, D.F. and Rodermel, S. 1999. The IM MUTANS variegation locus of *Arabidopsis* defines a mitochondrial alternative oxidase homolog that functions during early chloroplast biogenesis. Plant Cell 11: 43–55.

Xu, Y.L., Li, L., Peeters, A.J., Gage, D.A. and Zeevart, J.A. (1995) The *GA5* locus of *Arabidopsis thaliana* encodes a multifunctional gibberellin 20-oxidase: molecular cloning and functional expression. Proc. Natl. Acad. Sci. USA 92: 6640–6644.

Yalovsky, S., Trueblood, C.E., Callan, K.L., Narita, J.O., Jenkins, S.M., Rine, J. and Gruissem, W. 1997. Plant farnesyltransferase can restore yeast Ras signaling and mating. Mol. Cell. Biol. 17: 1986–1994.

Yamaguchi, S., Sun, T., Kawaide, H. and Kamiya, Y. 1998. The *GA2* locus of *Arabidopsis thaliana* encodes ent-kaurene synthase of gibberellin biosynthesis. Plant Physiol. 116: 1271–1278.

Zhang, L., Tschanz, W.R. and Casey, P.J. 1997. Isolation and characterization of a prenylcysteine lyase from bovine brain. J. Biol. Chem. 272: 23354–23359.

Zhu, T. and Wang, X. 2000. Large-scale profiling of the *Arabidopsis* transcriptome. Plant Physiol. 124: 1472–1476.
