
THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2003 by The American Society for Biochemistry and Molecular Biology, Inc.

**CloR, a Bifunctional Non-heme Iron Oxygenase Involved in**
**Clorobiocin Biosynthesis**

Received for publication, March 27, 2003, and in revised form, May 30, 2003  
Published, JBC Papers in Press, May 30, 2003, DOI 10.1074/jbc.M303190200

Florence Pojer‡, Rainer Kahlich§, Bernd Kammerer§, Shu-Ming Li‡, and Lutz Heide¶

From the ‡Universität Tübingen, Pharmazeutische Biologie, Auf der Morgenstelle 8, 72076 Tübingen, Germany  
and the §Universität Tübingen, Institut für Pharmakologie und Toxikologie, Abteilung Klinische Pharmakologie, Otfried-Müller-Str. 45, 72076 Tübingen, Germany

The aminocoumarin antibiotics novobiocin and clorobiocin contain a 3-dimethylallyl-4-hydroxybenzoate (3DMA-4HB) moiety. The biosynthesis of this moiety has now been identified by biochemical and molecular biological studies. CloQ from the clorobiocin biosynthetic gene cluster in *Streptomyces roseochromogenes* DS 12976 has recently been identified as a 4-hydroxyphenylpyruvate-3-dimethylallyltransferase. In the present study, the enzyme CloR was overexpressed in *Escherichia coli*, purified, and identified as a bifunctional non-heme iron oxygenase, which converts 3-dimethylallyl-4-hydroxyphenylpyruvate (3DMA-4HPP) via 3-dimethylallyl-4-hydroxymandelic acid (3DMA-4HMA) to 3DMA-4HB by two consecutive oxidative decarboxylation steps. In <sup>18</sup>O<sub>2</sub> labeling experiments we showed that two oxygen atoms are incorporated into the intermediate 3DMA-4HMA in the first reaction step, but only one further oxygen is incorporated into the final product 3DMA-4HB during the second reaction step. CloR does not show sequence similarity to known oxygenases. It apparently presents a novel member of the diverse family of the non-heme iron (II) and α-ketoacid-dependent oxygenases, with 3DMA-4HPP functioning both as an α-keto acid and as a hydroxylation substrate. The reaction catalyzed by CloR represents a new pathway for the formation of benzoic acids in nature.

Novobiocin and clorobiocin contain a prenylated 4-hydroxybenzoate moiety (called Ring A) (Fig. 1A). Recently, Lafitte *et al.* (9) demonstrated that Ring A plays a role in the binding affinity of these antibiotics for gyrase. Knowledge of the biosynthetic pathway of Ring A may facilitate future efforts to create structural modifications of this moiety by combinatorial biosynthesis.

The dimethylallyl moiety of Ring A is derived from the methyl erythritol-4-phosphate (MEP) pathway (10). The 4-hydroxybenzoic acid (4HB)<sup>1</sup> moiety can be formed in nature by at least three well-established mechanisms: (*a*) the direct conversion of chorismic acid to 4HB by chorismate pyruvate-lyase (11); (*b*) the removal of acetyl-CoA from the side chain of 4-coumaroyl-CoA by an oxidative reaction mechanism, analogous to the β-oxidation of fatty acids, resulting in 4-hydroxybenzoyl-CoA (12, 13); and (*c*) the removal of a C2 unit from the side chain of a β-hydroxy-β-phenylpropionate derivative via a retro-aldol reaction; the resulting benzaldehyde derivative can subsequently be oxidized to the corresponding acid (14, 15).

In novobiocin biosynthesis, feeding experiments with isotope-labeled precursors showed that 4-hydroxyphenylpyruvate and tyrosine are efficiently incorporated into Ring A (10, 16), ruling out the chorismate pyruvate-lyase reaction as the principal source of this compound. The aminocoumarin moiety of novobiocin (Ring B) is formed from tyrosine via the intermediate β-hydroxytyrosyl-S-NovH. Chen and Walsh (17) suggested that Ring A may also be formed from this intermediate, following the reaction mechanism shown in Fig. 2A, *i.e.* by a retro-aldol reaction to 4-hydroxybenzaldehyde, oxidation to the acid and prenylation. 3-Prenylation of 4HB has a well-established precedent in ubiquinone biosynthesis (18).

Cloning and sequencing of the biosynthetic gene clusters of novobiocin, clorobiocin, and coumermycin A<sub>1</sub> (Fig. 1B) (19) first appeared to support this hypothesis. Three genes were identified in the novobiocin and clorobiocin clusters for which no homologues existed in the coumermycin cluster. We speculated that these genes might be involved in the biosynthesis of Ring A (which is absent in coumermycin A<sub>1</sub>). These genes were: (*a*) *cloR* and *novR*, which showed sequence similarity to putative aldolases; CloR was indeed shown to be involved in Ring A biosynthesis by a gene inactivation experiment (19); (*b*) *cloF* and *novF*, which show sequence similarities to dehydrogenases; and (*c*) *cloQ* and *novQ*, which did not show sequence similarities to known genes in the data base. We recently

---

* This work was supported by a grant from the Deutsche Forschungsgemeinschaft (to L. H. and S.-M. L.), by the Fonds der Chemischen Industrie, and by the BMBF-Modellprojekt Klinische Pharmakologie (Grant FKZ-01EC0001). The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
* ¶ To whom correspondence should be addressed. Tel.: 49-7071-297-2460; Fax: 49-7071-295-250; E-mail: heide@uni-tuebingen.de.

This paper is available on line at http://www.jbc.org

---

<sup>1</sup> The abbreviations used are: 4HB, 4-hydroxybenzoic acid; 4HPP, 4-hydroxyphenylpyruvate; 3DMA-4HBAL, 3-dimethylallyl-4-hydroxybenzaldehyde; 3DMA-4HMA, 3-dimethylallyl-4-hydroxymandelic acid; 3DMA-4HPP, 3-dimethylallyl-4-hydroxyphenylpyruvate; IPTG, isopropyl-1-thio-β-D-galactopyranoside; GST, glutathione S-transferase; LC, liquid chromatography; MS, mass spectrometry; MS-MS, tandem mass spectrometry; HPLC, high performance liquid chromatography; HmaS, 4-hydroxymandelate synthase; DMAPP, dimethylallyl diphosphate.

CloR in Clorobiocin Biosynthesis

A

![Chemical Structure](https://i.imgur.com/chemical_structure.png)

Ring C Ring B Ring A

B

nov

![Gene Cluster Map](https://i.imgur.com/gene_cluster_map.png)

FIG. 1. A, structures of the aminocoumarin antibiotics. B, map of the biosynthetic gene cluster of the aminocoumarin antibiotics: novobiocin (top), clorobiocin (middle), and coumermycin A₁ (bottom).

A

![Reaction Pathway 1](https://i.imgur.com/reaction_pathway_1.png)

β-hydroxy-tyrosyl-S-NovH 4HBAL 4HB Ring A

B

![Reaction Pathway 2](https://i.imgur.com/reaction_pathway_2.png)

4HPP 3DMA-4HPP 3DMA-4HBAL Ring A

FIG. 2. Previous hypotheses for the formation of the 3-dimethylallyl-4-hydroxybenzoate moiety (Ring A) of novobiocin and clorobiocin.

confirmed that CloQ codes for a prenyltransferase (20). The substrate of this enzyme, however, was none of the intermediates discussed above but, rather, 4-hydroxyphenylpyruvate (4HPP, Fig. 2B). The product of the reaction, i.e. 3-dimethylallyl-4HPP, was easily degraded to 3-dimethylallyl-4-hydroxybenzaldehyde (3DMA-4HBAL), suggesting a biosynthesis of Ring A by the mechanism depicted in Fig. 2B, i.e. via oxidation of 3DMA-4HBAL to the corresponding acid.

In the present study, we investigated the conversion of 3DMA-4HPP to Ring A and the role of CloR in this reaction sequence. Unexpectedly, the results showed that the biosynthesis of Ring A follows neither of the two mechanisms depicted in Fig. 2. Rather, the conversion of 3DMA-4HPP to Ring A proceeds in two oxidative decarboxylation steps, via 3-dimethylallyl-4-hydroxymandelic acid (3DMA-4HMA) as intermediate, and is catalyzed by the bifunctional non-heme iron oxygenase CloR. This represents a new pathway for the formation of benzoic acids in nature.

### EXPERIMENTAL PROCEDURES

#### General Chemicals—4-Hydroxyphenylpyruvate (4HPP), D- and L-mandelic acid, DL-4-hydroxymandelic acid, and 4-hydroxyphenyllactic acid were purchased from Sigma. β-(R,S)-Hydroxy-L-tyrosine and

![Figure 3](#fig3)

**FIG. 3.** Purification of CloR after overexpression as a fusion protein with glutathione S-transferase. The 12% SDS-PAGE gel was stained with Coomassie Brilliant Blue. Lane 1, molecular mass standards; lane 2, total protein after IPTG induction; lane 3, soluble protein after induction; lane 4, eluate after thrombin treatment. The calculated molecular masses were 56.5 kDa (CloR-GST) and 30.5 kDa (CloR).

DMAPP were kindly provided by Dr. K.-H. van Pée (Dresden, Germany) and Dr. K. Yazaki (Kyoto, Japan), respectively. 3-Dimethylallyl-4-hydroxybenzaldehyde (3DMA-4HBAL) was synthesized as described by Glüsenkamp and Büchi (21). Ring A (3-dimethylallyl-4-hydroxybenzoic acid) was prepared as described previously (22). Oxygen (^{18}O₂; 99%) was purchased from Campro Scientific (Berlin, Germany).

DNA Manipulation—DNA manipulations and standard genetic techniques in Escherichia coli were carried out as described by Sambrook and Russell (23). DNA fragments were isolated from agarose gels using a NucleoSpin 2 in 1 extraction kit (Macherey-Nagel, Düren, Germany). Isolation of plasmids was carried out with ion-exchange columns (Nucleobond AX kit, Macherey-Nagel).

Cloning of cloR—For construction of the plasmid cloR-pGEX4T1, cloR was amplified using the primers cloR-Nterm-BamHI (5′-AGGT-GCCCCGGATCCAAGGCTTTGGCGGAAC-3′) and cloR-Cterm-XhoI (5′-TCGGCACTCGAGCCTTGCCAGGGCGGGCGTTTG-3′). Introduced restriction sites are underlined. The product was digested with BamHI and XhoI, gel purified and ligated into the same site of pGEX-4T-1 (Amersham Biosciences) to give cloR-pGEX4T1. The plasmid was transformed into E. coli strain BL21(DE3) for overexpression of CloR as GST (glutathione S-transferase)-tagged protein.

Overexpression and Purification of CloR Protein—E. coli BL21(DE3)/cloR-pGEX4T1 cells were grown in 100 ml of LB medium containing 50 μg/ml carbenicillin at 15 °C until an A₆₀₀ of 0.6 was reached (~20 h). IPTG was added to a final concentration of 250 μM. After 24 h at 15 °C, the cells were harvested by centrifugation (10 min at 5,000 × g) and resuspended in phosphate-buffered saline (140 mM NaCl, 2.7 mM KCl, 10 mM Na₂HPO₄, 1.8 mM KH₂PO₄, pH 7.3) containing 1 mg/ml lysozyme (76,344 units/mg, Fluka). Resuspended cells were broken by sonication (Branson sonifier, 10-min sonification). Cell debris was removed by centrifugation (30 min at 20,000 × g). Purification of CloR as a GST fusion protein and subsequent cleavage of the GST tag by thrombin treatment were carried out according to the manufacturer’s instructions (Amersham Biosciences) using glutathione-Sepharose 4B and the batch method.

Overexpression and Purification of CloQ—Construction of vector cloQ-pGEX4T1, overexpression in E. coli BL21(DE3), and purification of CloQ were carried out as described in Pojer et al. (20).

Protein Analysis—Standard protein techniques were used as described in previous studies (24, 25). The molecular mass of native CloR was determined by gel filtration on a HiLoad 26/60 Superdex 200 column that has been equilibrated with 50 mM Tris-HCl buffer (pH 8.0), containing 150 mM NaCl. The column was calibrated with dextran blue 2000 (2000 kDa), aldolase (158 kDa), albumin (67 kDa), ovalbumin (43 kDa), and ribonuclease A (13.7 kDa) (Amersham Biosciences).

Incubation of Holo-CloR with CloQ Reaction Products—3DMA-4HPP, the substrate for CloR reaction, was obtained by incubating CloQ with 4HPP and DMAPP as described in Pojer et al. (20). A reaction mixture (50 μl) containing 75 mM Tris-HCl (pH 7.5), 0.25 mM 4HPP, 0.5 mM DMAPP, 2.5 mM MgCl₂, and purified CloQ, was incubated at 30 °C for 1 h. This reaction produces about 7.5 nmol of 3DMA-4HPP.

![Figure 4](#fig4)

**FIG. 4.** HPLC analysis of the CloR reaction products. Detection was carried out at 277 nm. The identity of each product was confirmed by LC-MS-MS analysis, and the fragmentation pattern is indicated.

CloR in Clorobiocin Biosynthesis

A 3DMA-4HPP $\stackrel{18O_{2}}{\longrightarrow}$ $[^{18}O]$ 3DMA-4HMA $\stackrel{18O_{2}}{\longrightarrow}$ $[^{18}O]$ 3DMA-4HB

**Unenriched atmosphere**


235.13
[M-H-CO₂]⁻ = 191
[M-H-CO₂-2H]⁺ = 189
HO
    |
    O
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
    |
    OH
    |
    C
CloR in Clorobiocin Biosynthesis

$\begin{aligned}
& \text{C} \\
& \begin{array}{c}
\text{O=C-O}^{\ominus} \\
\text{C=O} \\
\text{CH}_{2} \\
\text{OH}
\end{array}
+ \quad
\begin{array}{c}
\text{O=C-O}^{\ominus} \\
\text{HO-C-H} \\
\text{OH}
\end{array}
\rightarrow \quad
\begin{array}{c}
\text{O=C=O} \\
+
\end{array}
\begin{array}{c}
\text{O=C-O}^{\ominus} \\
\text{OH}
\end{array}
+ \quad
\text{H}_{2}\text{O}
\end{aligned}$

3DMA-4HPP 3DMA-4HMA 3DMA-4HB = Ring A

FIG. 5—continued

(pH 7.5), 25 mM ascorbic acid, and 4 μg of holo-CloR. After incubation for 1 h at 30 °C, the reaction was stopped with 5 μl of formic acid. Product formation was analyzed by high performance liquid chromatography (HPLC) at 277 nm (absorption maximum for 4-hydroxymandelic acid) with a Multosphere RP18-5 column (250 × 4 mm, 5 μm) at a flow rate of 1 ml/min. A linear gradient of acetonitrile (25–75%) in 1% aqueous formic acid was used. A control incubation was carried out with denatured holo-CloR.

For incubation of holo-CloR with other substrates, 0.5 mM of the respective substrate (4-hydroxyphenylpyruvate, β-hydroxytyrosine, 4-hydroxymandelic acid, D-mandelate, L-mandelate, or 4-hydroxyphenyllactic acid) were used instead of 3DMA-4HPP, and analysis was carried out by HPLC at 254 nm with a linear gradient of acetonitrile (0–100%) in 1% aqueous formic acid at a flow rate of 1 ml/min. Control incubations were carried out with denatured holo-CloR. CloQ was also incubated with 4-hydroxymandelic acid or 4-hydroxybenzoic acid and DMAPP as described in Pojer et al. (20).

**Isolation of 3-Dimethylallyl-4-hydroxymandelic Acid**—The incubation mixture (1 ml) contained 500 μl of CloQ reaction product (~75 nmol of 3DMA-4HPP), 0.2 M Tris-HCl (pH 7.5), 80 mM potassium phosphate (pH 7.5), 25 mM ascorbic acid, and ~40 μg of holo-CloR and was incubated for 1 h at 30 °C. The reaction was stopped with 50 μl of formic acid and extracted twice with 1 ml of ethyl acetate. After evaporation, the residue was dissolved in 150 μl of ethanol. 3DMA-4HMA was isolated by HPLC as described above, and its identity was confirmed by mass spectrometry.

**Reactions in the Presence of ${}^{18}O_{2}$**—Two flasks, one containing enzymatically produced 3DMA-4HPP (see above) and one containing holo-CloR reaction mixture (see above) were degassed by application of a vacuum and flushed with argon for three times. The anaerobic holo-CloR solution was transferred to the substrate vial containing 3DMA-4HPP. The argon was removed by application of a vacuum, and finally ${}^{18}O_{2}$ was allowed to enter into the flask. After incubation for 1 h at 30 °C, the reaction was stopped with formic acid and analyzed by liquid chromatography (LC) coupled to an electrospray ionization mass spectrometer (TSQ Quantum, ThermoFinnigan, San Jose, CA) in negative ion mode. A linear gradient of acetonitrile (25–75%) in 0.1% aqueous formic acid was used. In a parallel ${}^{18}O_{2}$ labeling experiment, 3DMA-4HMA (isolated as described above) was used as substrate instead of 3DMA-4HPP.

---

**RESULTS**

**Sequence Analysis of CloR and NovR**—The gene *cloR* from the clorobiocin cluster and the corresponding gene *novR* from the novobiocin cluster (Fig. 1B) encode proteins of 277 and 270 amino acids, respectively, and show 95% identity between each other (GenBank™ accession numbers AAN65240 and AAF67511, respectively). A database search revealed sequence similarity to class II aldolases, which are represented, e.g., by L-fucose-1-phosphate aldolase (28) and L-rhamnose-1-phosphate aldolase (29), and to the structurally related L-ribulose-5-phosphate-4-epimerase (30). These enzymes contain a catalytic zinc residue in their active center and are involved in the catabolism of sugars in *E. coli*.

*cloR* is transcriptionally coupled to the gene *cloQ* (accession number: AF329398), which encodes the aromatic prenyltransferase involved in Ring A formation (20). The same situation is found in the novobiocin cluster for the corresponding genes *novQ* and *novR* (accession number: AF170880).

**Expression and Purification of CloR**—We decided to express CloR as a glutathione S-transferase (GST) fusion protein rather than as an His-tagged protein, because CloR showed sequence similarity to class II aldolases, and these enzymes require zinc for their activity (31). A purification using the His-tagged protein can lead to a complete loss of the aldolase activity due to interactions between the hexahistidyl tag and the metal ion ($Zn^{2+}$). *E. coli* cells harboring CloR expression constructs yielded only insoluble protein when grown at temperatures of 20 °C or higher. To obtain soluble CloR-GST fusion protein, cells were cultured at 15 °C and induced with 250 μM IPTG. After purification, GST was cleaved from CloR by thrombin treatment and removed. This procedure resulted in apparently homogenous CloR protein as judged by SDS-PAGE (Fig. 3). The molecular mass observed in SDS corresponded to the calculated mass of the protein (30.5 kDa). A protein yield of 1 mg of pure CloR per liter of culture was obtained. By using gel chromatography, the molecular mass of native CloR was determined as 124.5 kDa showing that the protein was tetrameric in solution.

**Characterization of the Reaction Products of the CloR Reaction**—To investigate the catalytic activity of CloR, we first produced 3DMA-4HPP (Fig. 2B), the putative substrate of CloR, by incubation of 4-hydroxyphenylpyruvate (4HPP) with DMAPP and the prenyltransferase CloQ. Subsequently, CloR and different cofactors, e.g. $Zn^{2+}$ and NADH (32), were added to the reaction mixture. However, no formation of 4-hydroxybenzaldehyde (3DMA-4HBAL) or Ring A could be detected by HPLC.

A more sensitive analysis using a radioactive assay with 4HPP and [1-${}^{14}$C]DMAPP as substrates, however, revealed the presence of a small amount of a new radioactive compound (termed product X), with a retention time of 11.1 min in HPLC (Ring A, 16.7 min; 3DMA-4HBAL, 20.7 min; and 3DMA-4HPP, 18.4 min) (data not shown). This product was absent if heat-denatured CloR was used, or if 4HPP or DMAPP were omitted from the prenylation assay. This indicated that the new metabolite X was derived enzymatically from 3DMA-4HPP.

In the biosynthesis of chloroeremomycin in *Amycolatopsis orientalis*, 4HPP is converted to 4-hydroxymandelic acid under catalysis of the non-heme iron dioxygenase HmaS (= ORF21) (26, 27). As described by Hubbard et al. (26), HmaS and similar enzymes need to be activated by preincubation with an excess of $Fe^{2+}$ immediately before incubation and $Fe^{3+}$, generated by oxidation, has to be reduced to $Fe^{2+}$ by ascorbic acid to maintain an active enzyme. Although CloR did not show sequence similarity to HmaS or other non-heme iron-dependent enzymes, we decided to test CloR under similar conditions. After preincubation of CloR for 20 min with 1 mM FeSO₄, ascorbic acid and enzymatically generated 3DMA-4HPP were added. After incubation for 1 h, the products of the reaction were analyzed by HPLC (Fig. 4). Under these conditions, formation of the new product X was ~15-fold higher than in the absence of $Fe^{2+}$ and ascorbate. Furthermore, an additional product was

CloR in Clorobiocin Biosynthesis

Primary metabolism → L-Tyrosine → Tyrosyl-S-CloH → β-hydroxy-tyrosyl-S-CloH → Ring B

Transamination

Prephenate → 4HPP → 3DMA-4HPP → 3DMA-4HMA → 3DMA-4HB (Ring A)

FIG. 6. Hypothetical pathways to the aminocoumarin moiety (Ring B) and the prenylated 4-hydroxybenzoate moiety (Ring A) of clorobiocin.

detected that showed the same retention time as an authentic Ring A standard. LC-MS-MS analysis in comparison with an authentic reference compound confirmed that this latter product was indeed Ring A (*m/z*: 205, 161, 150, 106).

When product X was isolated by HPLC and incubated with holo-CloR in the presence of ascorbic acid, again the formation of Ring A was clearly demonstrated by LC-MS-MS analysis (data not shown), proving that product X was an intermediate in the formation of Ring A.

LC-MS-MS analysis of product X revealed that this compound showed the calculated mass ([M-H]⁻ = 235) of 3-dimethylallyl-4-hydroxymandelic acid (3DMA-4HMA). Furthermore, it showed the characteristic fragmentation pattern of a 4-hydroxymandelic acid derivative (*m/z*: 235 [M-1]⁻, 191 [M-44]⁻, and 189 [M-46]⁻); the same pattern was observed from an authentic 4-hydroxymandelic acid (*m/z*: 167 [M-1]⁻, 123 [M-44]⁻, and 121 [M-46]⁻). Therefore, CloR catalyzes two consecutive reactions (Fig. 4): first, the conversion of 3DMA-4HPP to 3DMA-4-hydroxymandelic acid (3DMA-4HMA), and second, the conversion of 3DMA-4HMA to 3DMA-4-hydroxybenzoic acid, *i.e.* Ring A.

CloR was found to be specific for 3DMA-4HPP and 3DMA-4HMA as substrates. No product formation was observed with the non-prenylated substrates 4-hydroxyphenylpyruvate or DL-4-hydroxymandelic acid, nor with D-mandelic acid, L-mandelic acid, β-hydroxytyrosine, or 4-hydroxyphenyllactic acid. Additional experiments confirmed that the prenyltransferase CloQ (20) specifically prenylated 4HPP and was not able to react with DL-4-hydroxymandelic acid or 4-hydroxybenzoic acid (data not shown).

Replacement of Fe²⁺ with other monovalent or divalent metal ions (1 mM Cu⁺, Zn²⁺, Mg²⁺, or Mn²⁺) resulted in almost complete (95–98%) loss of enzymatic activity of CloR, as described previously for non-heme iron oxygenases (33). Many non-heme iron-dependent oxygenases require, besides Fe²⁺, an α-ketoacid as cofactor, e.g., α-ketoglutaric acid (27). However, the activity of CloR was not stimulated by addition of α-ketoglutarate.

Purified CloR was a colorless protein. UV-visible spectrometry showed an absorption maximum at 283 nm but no absorption in the visual range. Therefore, CloR is not a heme protein.

comparison to [M-H]⁻ = 205 for the unlabeled compound (Fig. 5A). As expected, both these labeled oxygens were located in the carboxyl group (Fig. 5A).

For a second labeling experiment, the intermediate 3DMA-4HMA was first produced in unlabeled form and isolated by HPLC. This compound was then incubated in an ${}^{18}$O₂ atmosphere with CloR (preincubated with Fe²⁺) and ascorbate. The resulting 3DMA-4HB was analyzed by LC-MS-MS (Fig. 5B), and this clearly showed the incorporation of one ${}^{18}$O atom into the carboxyl group of the product, as demonstrated by the molecular ion at [M-H]⁻ = 207, and the decarboxylation product at $m/z$ = 161 (Fig. 5B).

Fig. 5C summarizes the results of the ${}^{18}$O₂ incorporation experiments. Two ${}^{18}$O atoms are incorporated in the first reaction step, and one ${}^{18}$O atom is incorporated in the second reaction step. The other ${}^{18}$O atom involved in the second step is most likely converted to water (see “Discussion”).

## DISCUSSION

In the present study, CloR was identified as a bifunctional oxygenase that converts 3-dimethylallyl-4-hydroxypyruvate (3DMA-4HPP) in two consecutive reaction steps to 3-dimethylallyl-4-hydroxybenzoate (3DMA-4HB), *i.e.* to the Ring A moiety of clorobiocin. An ${}^{18}$O₂ labeling experiment unequivocally confirmed that molecular oxygen is used as substrate by CloR. The purified CloR protein does not contain a heme prosthetic group, and its activation by Fe²⁺ and ascorbate indicated that it belongs to the non-heme iron oxygenase.

The first reaction catalyzed by CloR, *i.e.* the conversion of 3DMA-4HPP to 3-dimethylallyl-4-hydroxymandelic acid (3DMA-4HMA), has a well-established precedent in the HmaS reaction in chloroeremomycin biosynthesis. HmaS belongs to the iron(II)- and α-ketoacid-dependent dioxygenases (reviewed in Refs. 33, 35, 36). These enzymes utilize O₂ and an α-ketoacid as cosubstrates. During the reaction, the α-ketoacid loses CO₂, and the keto function is oxidized to a carboxyl group by introduction of one of the oxygen atoms of O₂. The other oxygen may be used for a hydroxylation reaction, exemplified, *e.g.*, by the prolyl 3-hydroxylase reaction. However, iron(II)- and α-ketoacid-dependent oxygenases have been shown to catalyze not only hydroxylations but a wide range of diverse oxidative transformations, including epoxidations, desaturations, ring formation, and ring expansion reactions. Some of these enzymes are bifunctional (*e.g.* deacetoxy/deacetylcephalosporin synthase) or even trifunctional (*e.g.* clavaminic acid synthase or thymine hydroxylase), catalyzing several consecutive oxidative transformations within a single biosynthetic pathway.

In the conversion of 4-hydroxyphenylpyruvate (4HPP) to homogentisate by 4HPP dioxygenase, 4HPP serves as a substrate for hydroxylation and at the same time as α-ketoacid cosubstrate. CloR (in its first reaction step) and HmaS carry out a very similar reaction as 4HPP dioxygenase but hydroxylate the benzylic position of the substrate instead of the phenyl ring. However, although HmaS shows obvious sequence similarity to 4HPP dioxygenase, CloR does not.

The second reaction step catalyzed by CloR is the conversion of 3DMA-4HMA to 3DMA-4HB. It is tempting to speculate that this reaction may involve hydroxylation of the α-position of 3DMA-4HMA, resulting in a α,α-gem-diol, which eliminates water to give the corresponding α-ketoacid, *i.e.* 4-hydroxybenzoylformate. Oxidative decarboxylation of this compound would result in the final product, 3DMA-4HB. Thereby, the overall reaction mechanism would be similar to that of the first reaction and resemble that of other iron(II)- and α-ketoacid-dependent dioxygenases.

However, LC-MS-MS analysis did not confirm the presence

of 4-hydroxybenzoylformate in the incubation mixture, and the exact mechanism of the CloR reaction remains speculative at present. It is believed that the reactions catalyzed by iron(II)- and α-ketoacid-dependent oxygenases involve a reactive Feᴵⱽ=O species (35, 37). Whether this is true for CloR remains to be shown.

The different groups of iron(II)- and α-ketoacid-dependent oxygenases possess little overall sequence similarity to each other (35, 36), and it is therefore not surprising that CloR does not show sequence similarity to known members of this family. However, a database search reveals that CloR does show significant similarity to several proteins of so far unknown function, deduced from genome sequences of different microorganisms. These may possibly represent enzymes of similar function as CloR.

The common structural motif of iron(II)- and α-ketoacid-dependent enzymes is the so-called 2-His-1-carboxylate facial triad (36, 38). It consists of two histidyl groups and one glutaryl or asparagyl residue, which together anchor the iron atom in the active site of the enzyme. Comparison of the primary sequence of CloR with that of NovR as well as with the sequences of six of the data base entries of unknown function with high sequence similarity to CloR (data not shown) identifies His-161, His-176, His-178, His-241, and Asp-170 of CloR as strictly conserved residues. These amino acids may be candidates for a potential 2-His-1-carboxylate facial triad of this oxygenase.

The involvement of CloR in the biosynthesis of the prenylated 4-hydroxybenzoate moiety of clorobiocin was proven *in vivo* by a gene inactivation experiment (19). The present results now allow the formulation of a detailed hypothesis for the formation of this moiety of clorobiocin, as shown in Fig. 6: CloQ is a prenyltransferase, which converts 4HPP to 3DMA-4HPP (20). CloR converts the CloQ reaction product to Ring A, as demonstrated in the present study. CloF shows sequence similarity to prephenate dehydrogenases and is therefore likely to produce 4HPP as the substrate for CloQ, similar to ORF1 of the chloroeremomycin biosynthetic gene cluster (39). Notably, the coumermycin cluster does not contain a CloF homologue (Fig. 1B).

Radioactive feeding experiments with the novobiocin producer (40) showed that [U-${}^{14}$C]L-tyrosine was incorporated preferentially into Ring B, whereas [U-${}^{14}$C]4HPP was incorporated preferentially in Ring A. This may suggest that a primary metabolic prephenate dehydrogenase provides tyrosine for the formation of Ring B, whereas CloF supplies 4HPP for Ring A biosynthesis, and that cross-talk exists between both pathways probably via a transaminase reaction (Fig. 6).

Chen and Walsh (17) have previously speculated that Ring A and Ring B of novobiocin may both be produced via β-hydroxytyrosyl-S-NovH as a common precursor. Our studies on CloQ and CloR (Refs. 19 and 20 and the present study), however, now establish that these two aromatic moieties of clorobiocin (and very likely those of novobiocin as well) are produced by two independent pathways (Fig. 6). The discovery of CloR adds a new interesting member to the diverse family of the non-heme iron oxygenases and demonstrates the existence of a new pathway for the formation of benzoic acids in nature.

Acknowledgments—We thank Christopher T. Walsh for helpful suggestions, Emmanuel Wemakor for technical support, and T. Luft for the synthesis of 3DMA-4HBAL.

## REFERENCES

1. Lanoot, B., Vancanneyt, M., Cleenwerck, I., Wang, L., Li, W., Liu, Z., and Swings, J. (2002) *Int. J. Syst. Evol. Microbiol.* **52**, 823–829
2. Berger, J., and Batcho, A. D. (1978) *J. Chromatogr. Lib.* **15**, 101–159
3. Maxwell, A., and Lawson, D. M. (2003) *Curr. Top. Med. Chem.* **3**, 283–303
4. Kampranis, S. C., Gormley, N. A., Tranter, R., Orphanides, G., and Maxwell, A. (1999) *Biochemistry* **38**, 1967–1976
5. Montecalvo, M. A., Horowitz, H., Wormser, G. P., Seiter, K., and Carbonaro,

C. A. (1995) Antimicrob. Agents Chemother. 39, 794

6. Rappa, G., Shyam, K., Lorico, A., Fodstad, O., and Sartorelli, A. C. (2000) Oncol. Res. 12, 113–119

7. Maxwell, A. (1993) Mol. Microbiol. 9, 681–686

8. Walsh, C. T. (2002) Chembiochem 3, 124–134

9. Lafitte, D., Lamour, V., Tsvetkov, P. O., Makarov, A. A., Klich, M., Deprez, P., Moras, D., Briand, C., and Gilli, R. (2002) Biochemistry 41, 7217–7223

10. Li, S.-M., Hennig, S., and Heide, L. (1998) Tetrahedron Lett. 39, 2717–2720

11. Siebert, M., Severin, K., and Heide, L. (1994) Microbiology 140, 897–904

12. Löschner, R., and Heide, L. (1994) Plant Physiol. 106, 271–279

13. Hertweck, C., and Moore, B. S. (2000) Tetrahedron 56, 9115–9120

14. Gasson, M. J., Kitamura, Y., McLauchlan, W. R., Narbad, A., Parr, A. J., Parsons, E. L., Payne, J., Rhodes, M. J., and Walton, N. J. (1998) J. Biol. Chem. 273, 4163–4170

15. Ahmed, M. A., El-Mawla, A., and Beerhues, L. (2002) Planta 214, 727–733

16. Kominek, L. A., and Sebek, O. K. (1974) Dev. Ind. Microbiol. 15, 60–69

17. Chen, H., and Walsh, C. T. (2001) Chem. Biol. 8, 301–312

18. Melzer, M., and Heide, L. (1994) Biochim. Biophys. Acta. 1212, 93–102

19. Pojer, F., Li, S.-M., and Heide, L. (2002) Microbiology 148, 3901–3911

20. Pojer, F., Wemakor, E., Kammerer, B., Chen, H., Walsh, C. T., Li, S.-M., and Heide, L. (2003) Proc. Natl. Acad. Sci. U. S. A. 100, 2316–2321

21. Gluesenkamp, K. H., and Buechi, G. (1986) J. Org. Chem. 51, 4481–4483

22. Kominek, L. A., and Meyer, H. F. (1975) Methods Enzymol. 43, 502–508

23. Sambrook, J., and Russell, D. W. (2001) Molecular Cloning: A Laboratory Manual, 3rd Ed., Cold Spring Harbor Laboratory Press, New York

24. Bradford, M. M. (1976) Anal. Biochem. 72, 248–254

25. Laemmli, U. K. (1970) Nature 227, 680–685

26. Hubbard, B. K., Thomas, M. G., and Walsh, C. T. (2000) Chem. Biol. 7, 931–942

27. Choroba, O. W., Williams, D. H., and Spencer, J. B. (2000) J. Am. Chem. Soc. 122, 5389–5390

28. Dreyer, M. K., and Schulz, G. E. (1996) J. Mol. Biol. 259, 458–466

29. Kroemer, M., and Schulz, G. E. (2002) Acta Crystallogr. D. Biol. Crystallogr. 58, 824–832

30. Luo, Y., Samuel, J., Mosimann, S. C., Lee, J. E., Tanner, M. E., and Strynadka, N. C. (2001) Biochemistry 40, 14763–14771

31. Dreyer, M. K., and Schulz, G. E. (1993) J. Mol. Biol. 231, 549–553

32. Wehmeier, U. F. (2001) FEMS Microbiol. Lett. 197, 53–58

33. Que, L., Jr., and Ho, R. Y. (1996) Chem. Rev. 96, 2607–2624

34. Serre, L., Saillard, A., Sy, D., Boudec, P., Rolland, A., Pebay-Peyroula, E., and Cohen-Addad, C. (1999) Structure Fold. Des. 7, 977–988

35. Prescott, A. G., and Lloyd, M. D. (2000) Nat. Prod. Rep. 17, 367–383

36. Que, L., Jr. (2000) Nat. Struct. Biol. 7, 182–184

37. Rohde, J. U., In, J. H., Lim, M. H., Brennessel, W. W., Bukowski, M. R., Stubna, A., Munck, E., Nam, W., and Que, L., Jr. (2003) Science 299, 1037–1039

38. Hegg, E. L., and Que, L., Jr. (1997) Eur. J. Biochem. 250, 625–629

39. van Wageningen, A. M., Kirkpatrick, P. N., Williams, D. H., Harris, B. R., Kershaw, J. K., Lennard, N. J., Jones, M., Jones, S. J., and Solenberg, P. J. (1998) Chem. Biol. 5, 155–162

40. Calvert, R. T., Spring, M. S., and Stoker, J. R. (1972) J. Pharm. Pharmacol. 24, 972–978

Enzyme Catalysis and Regulation:  
CloR, a Bifunctional Non-heme Iron  
Oxygenase Involved in Clorobiocin  
Biosynthesis  

Florence Pojer, Rainer Kahlich, Bernd  
Kammerer, Shu-Ming Li and Lutz Heide  
J. Biol. Chem. 2003, 278:30661-30668.  
doi: 10.1074/jbc.M303190200 originally published online May 30, 2003  

Access the most updated version of this article at doi: [10.1074/jbc.M303190200](http://dx.doi.org/10.1074/jbc.M303190200)  

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinitysites).  

Alerts:  
- [When this article is cited](#)  
- [When a correction for this article is posted](#)  

[Click here](#) to choose from all of JBC's e-mail alerts  

This article cites 39 references, 9 of which can be accessed free at  
[http://www.jbc.org/content/278/33/30661.full.html#ref-list-1](http://www.jbc.org/content/278/33/30661.full.html#ref-list-1)
