
The herpes simplex virus VP16-induced complex: the makings of a regulatory switch

Joanna Wysocka and Winship Herr

Cold Spring Harbor Laboratory, Cold Spring Harbor, NY 11724, USA

When herpes simplex virus (HSV) infects human cells, it is able to enter two modes of infection: lytic and latent. A key activator of lytic infection is a virion protein called VP16, which, upon infection of a permissive cell, forms a transcriptional regulatory complex with two cellular proteins – the POU-domain transcription factor Oct-1 and the cell-proliferation factor HCF-1 – to activate transcription of the first set of expressed viral genes. This regulatory complex, called the VP16-induced complex, reveals mechanisms of combinatorial control of transcription. The activities of Oct-1 and HCF-1 – two important regulators of cellular gene expression and proliferation – illuminate strategies by which HSV might coexist with its host.

When the prevalent human pathogen herpes simplex virus (HSV) infects a cell, it can enter a lytic program in which, ultimately, the host cell is killed, or it can enter a latent state and reside silently for long periods – indeed a lifetime – in the cells of the infected individual. Little is known of how the choice between lytic and latent infection is made, but it clearly involves intimate interactions between the virus and the infected host cell. A key player in this decision is the HSV virion protein VP16, which is important in the initiation of the lytic program through the assembly of a multiprotein transcriptional regulatory complex – the VP16-induced complex – with two host-cell factors, Oct-1 and HCF-1, on the viral immediate-early (IE) promoters. This VP16-induced complex represents a regulatory switch: when it is ‘on’, it promotes IE-gene expression and thus lytic infection, and when it is ‘off’ limits IE-gene transcription favoring latent infection.

The VP16-induced complex has served as a model for combinatorial mechanisms of transcriptional regulation and also as a probe to uncover the functions of the two cellular components Oct-1 and HCF-1. This review describes the architecture and mechanics of the VP16-induced complex and what this teaches us about transcriptional regulation in human cells. We also discuss the natural functions of Oct-1 and HCF-1, and what makes these cellular proteins attractive targets for VP16 to regulate HSV infection. For more detail, the reader is encouraged to also refer to other reviews related to this topic [1–6].

---

Two modes of HSV infection

The lytic mode of HSV infection naturally occurs in epithelial cells, resulting in lesions such as cold sores, and the latent mode naturally occurs in sensory neurons [5]; thus, the nature of the infected cell dictates the course of infection. Upon primary infection, HSV first grows lytically in the epithelial cells, which leads to subsequent infection of neurons. In lytic infection, VP16 activation of transcription of the five HSV IE genes results in a cascade of gene transcription, in which early genes and late genes are transcribed in an ordered temporal pattern. HSV encodes its own DNA replication machinery and, thus, HSV infection is independent of cellular S-phase induction. This independence means that the initiation of viral gene expression by the VP16-induced complex is likely to be a key step in determining the lytic program of infection.

During latent infection, there is little viral gene transcription [5]. However, the virus can be reactivated from this stable silent state by stimuli such as stress, fever and UV exposure, resulting in renewed episodes of lytic infection at the site of primary infection. The mechanism of viral reactivation is poorly understood, but reactivation stimuli provoke the resumption of viral gene expression, which can occur independently of VP16-induced complex formation [5].

---

The VP16-induced complex

In lytic infection, ~500–1000 VP16 molecules within the tegument surrounding the core of the infecting virion are delivered to the host cell and the VP16-induced complex is formed soon afterwards [6]. VP16 first associates with HCF-1 and is primed for association with Oct-1 on 'TAATGARAT' regulatory elements present in each IE-gene promoter (Fig. 1a). A potent VP16 transcriptional activation domain then activates IE-gene transcription. VP16 itself is a late-gene product that is incorporated into the newly synthesized virions for the next round of infection [6]. Below, we describe the structure and function of VP16, Oct-1 and HCF-1, which are illustrated in Fig. 1b, and discuss their roles in the context of the VP16-induced complex.

---

VP16

VP16 is a 490-amino-acid protein that contains a core region and a C-terminal transcriptional activation

Review TREND$ in Biochemical Sciences Vol.28 No.6 June 2003 295

(a)

(i) HSV infection and  
VP16 delivery  

Latent  
?  

Lytic  

VP16  

(ii) VP16 association  
with HCF-1  

HCF-1  
VP16  

(iii) VP16-induced  
complex assembly  

Oct-1  
VP16  
'TAATGARAT'  

IE-gene  
transcription  

HSV IE  
promoter  

(iv) Cascade of gene expression,  
viral DNA replication, and  
virus production  

(b)  

VP16  
VIC | AD | 490  

Oct-1  
Q | POU | S/T | 766  

HCF-1  
Kelch | Basic | Acidic | 2035  

HCFPRO repeats  

TIBS  

Fig. 1. The makings of a regulatory switch. (a) Role of VP16-induced complex in the herpes simplex virus (HSV) lytic cycle. Upon HSV infection the virus can enter a lytic or latent phase (i). In the lytic mode, VP16 (yellow) is delivered to the host cell, where it associates with HCF-1 (red; ii), leading to VP16-induced complex assembly on HSV immediate-early (IE) promoters with Oct-1 (blue; iii) and initiation of a cascade of gene expression that results in virus production and lysis of the host cell (iv). Broken lines indicate the acidic VP16 transcriptional activation domain (AD). (b) VP16-induced complex components VP16, Oct-1 and HCF-1, with different regions of each protein indicated. Minimal domains required and sufficient for VP16-induced complex assembly are indicated in yellow (VP16 core), blue (Oct-1 POU domain) and red (HCF-1 Kelch domain). Abbreviations: Q, Oct-1 N-terminal glutamine-rich transcriptional activation domain; S/T, serine and threonine-rich transcriptional activation domain; VIC, VP16-induced complex forming domain. Adapted, with permission, from Ref. [2].

domain, which is enriched in acidic residues (Fig. 2a). The sequence of the transcriptional activation domain are not conserved although it interacts with many factors involved in transcriptional activation [7]. The VP16 core contains both structured and unstructured regions [8]. The structured region possesses a seat-like structure (Fig. 2b, left) in which the seat surface (Fig. 2c) recognizes and binds the TAATGARAT

http://tibs.trends.com

Fig. 2. VP16 (a) VP16 homologs from five α herpesviruses. Conserved VP16-induced complex forming (VIC) core regions (yellow) and HSV VP16 transcriptional activation domain (AD; pink) are indicated. (b) Ribbon diagrams of the left side and front views of the VP16 core. Left-side view shows the seat-like structure with the DNA-binding surface indicated. Broken lines denote disordered regions in the crystal structure [8]. (c) Molecular surface representation of the VP16 core with surfaces of amino acids whose replacement with alanine prevents DNA binding and VP16-induced complex formation shown in red. Abbreviations: BHV-1, bovine herpesvirus type 1; EHV-1, equine herpesvirus type 1; HSV-1, herpes simplex virus type 1; MDV, Marek's disease virus; VZV, varicella-zoster virus.

element [9]. On its own, however, VP16 does not bind to DNA well. Instead, VP16 is stabilized on the DNA by the unstructured region (Fig. 2b, right), which interacts with Oct-1, HCF-1 and the DNA in the complex [2,8]. Thus, VP16 is likely to have a bipartite DNA-binding structure as suggested by the bidentate VP16 shape that contacts the TAATGARAT element (Fig. 1a).

Role of VP16 in HSV infection

Elegant experiments with HSV-carrying mutant VP16 molecules incapacitated for VP16-induced complex formation (in1814) or transcriptional activation (RP5) but not virion assembly, have provided insights into VP16 function during HSV infection [10,11]. In mice, these viruses are essentially avirulent, whereas in tissue-culture cells, infection is debilitated but not abolished: IE-gene transcription is markedly reduced, establishing that there is VP16-dependent and -independent IE-gene transcription in cultured cells. These studies, however, probably underestimate the role of VP16 in HSV infection because they were performed in rodent cells in which VP16-induced complex formation is debilitated owing to a rodent-specific variation in Oct-1 [12,13] (A. Bubulya and W.H., unpublished results).

Consistent with latency involving a lack of IE-gene transactivation by VP16, the in1814 virus (albeit not the RP5 virus [11]) establishes latent infection in mice as well as wild-type virus [14]. Thus, establishment of latency could represent, in at least some instances, a default pathway resulting from the lack of VP16 function—specifically VP16-induced complex formation.

Oct-1

Oct-1 is a broadly expressed and versatile transcription factor, and a founding member of the POU family of homeo-domain proteins, which are characterized by a bipartite POU (for Pit-1, Oct-1, Oct-2 and Unc86) DNA-binding domain [3,4]. Oct-1 performs many divergent roles in cellular transcriptional regulation partly owing to its flexibility in DNA binding and ability to associate with multiple and varied co-regulators. Table 1 reviews known

functions and roles of Oct-1 in uninfected and infected cells, the central elements of which are discussed below.

Cellular functions of Oct-1

Oct-1 activates transcription of genes that are involved in basic cellular processes—chromatin structure, RNA processing and transcriptional elongation—which are shared by many (if not all) cells, particularly growing cells. Thus, Oct-1 activates small nuclear RNA (snRNA) and S-phase histone H2B gene transcription [15,16]. Oct-1 also participates in the activation of cell-specific promoters, particularly in the immune and nervous systems, usually in cooperation with cell-specific co-activators [1,4]. The best-characterized cell-specific promoters are those of the genes encoding immunoglobulin (Ig) heavy- and light-chains, in which the Oct-1 POU-domain associates with a B-cell specific co-regulator OCA-B (also known as OBF-1 and Bob-1). OCA-B stabilizes Oct-1 on DNA and provides a transcriptional activation domain. Although generally thought of as an activator of gene transcription, Oct-1 also represses transcription through a variety of mechanisms (Table 1).

In addition to cell-specific transcription, Oct-1 is important for inducible transcription. For example, independent of p53, the levels and intrinsic DNA-binding activity of Oct-1 are induced post-transcriptionally in response to DNA damage [17]. Subsequently, Oct-1 activates the growth-arrest and DNA-damage inducible GADD45 promoter [18] through association with the breast-cancer tumor suppressor BRCA-1 [19]. By participating in the DNA-damage response, Oct-1 might play a role in tumor suppression and perhaps HSV reactivation.

Oct-1 as a target for multiple viruses

Many viruses use Oct-1 to promote infection; in addition to the α herpesviruses, Oct-1 is involved in autoregulation of the Kaposi's sarcoma-associated herpesvirus (KSHV) ORF50/Lyta promoter [20] and activation of the mouse mammary tumor virus (MMTV) LTR promoter through interaction with the glucocorticoid nuclear-hormone receptor [21,22]. Oct-1 also participates in DNA replication

Review TRENDS in Biochemical Sciences Vol.28 No. 6 June 2003 297

Table 1. Oct-1 functions*

| Oct-1 function | Example | Protein partner | Refs |
| --- | --- | --- | --- |
| Transcriptional activation |  |  |  |
| Activation of ubiquitously expressed genes | Oct-1 activates transcription from histone H2B promoter<br>Oct-1 activates snRNA promoters in cooperation with SNAPc<br>Oct-1 and Sp1 bind cooperatively to the DSE on U2snRNA promoters | SNAPc<br>Sp1 | [16]<br>[15]<br>[69] |
| Activation of cell-specific genes | Oct-1 associates with B-cell specific co-regulator OCA-B and activates immunoglobulin gene transcription<br>Oct-1 associates with a cofactor distinct from OCA-B to activate the tie1-and tie2-gene promoters, which encode endothelial-cell specific tyrosine kinase receptors<br>Oct-1 interacts with enhancer and proximal promoter sequences of the GnRH gene and is crucial for episodic activation | OCA-B<br>Unidentified endothelial cell-specific cofactor | [4]<br>[70,71]<br>[1,72] |
| Inducible activation during stress response | Oct-1 levels and DNA-binding activity are post-transcriptionally induced in response to DNA damage and Oct-1 then activates the GADD45 promoter together with the coactivator BRCA1 | BRCA1 | [17–19] |
| Transcriptional repression |  |  |  |
| Through co-repressor recruitment | The Oct-1 POU domain interacts with SMRT leading to HDAC recruitment and repression<br>In addition to activation, Oct-1 also represses the GnRH promoter via recruitment of GR | SMRT<br>GR | [73]<br>[1] |
| Through activator sequestration | The Oct-1 POU domain interacts with the RXR DNA-binding domain preventing dimer formation with the thyroid hormone receptor thus repressing T3-dependent transcriptional activation | RXR | [74] |
| Through blocking of activator binding to an adjacent site | Octamer site binding by Oct-1 in the IL-4 promoter blocks adjacent NFAT binding and inhibits IL-4 gene transcription |  | [75] |
| Through localization to nuclear periphery structures | Oct-1 in the nuclear periphery represses collagenase gene transcription and Oct-1 dissociation from the periphery induces senescence-associated collagenase gene expression<br>Oct-1 binds the silencing MAR leading to repression of the human thyrotropin β gene |  | [76]<br>[77] |
| Virus infection |  |  |  |
| Activation of viral genes | Oct-1 activates HSV IE-gene transcription<br>Oct-1 activates VZV IE 62-gene expression<br>Oct-1 activates KSHV ORF 50/Lyta-dependent promoters<br>Oct-1 interacts synergistically with GR to activate MMTV transcription | VP 16/HCF-1<br>ORF 10/HCF-1<br>GR | [2]<br>[78]<br>[20]<br>[21,22] |
| Viral DNA replication | Oct-1 stimulates initiation of adenovirus DNA replication by viral pTP-DNA polymerase complex association with the POUH domain | pTP | [23] |

*Abbreviations: BRCA1, breast cancer 1; DSE, distal sequence element; GADD, growth-arrest and DNA-damage inducible; GnRH, gonadotropin-releasing hormone; GR, glucocorticoid receptor; HDAC, histone deacetylase; HSV, herpes simplex virus; IE, immediate-early; IL-4, interleukin 4; KSHV, Kaposi's sarcoma-associated herpesvirus; MAR, matrix attachment region; MMTV, mouse mammary tumor virus; NFAT, nuclear factor of activated T cells; OCA-B, Oct co-activator from B cells; POU, for Pit-1, Oct-1, Oct-2 and Unc86; POUH, POU-homeo domain; pTP, precursor terminal protein; RXR, retinoid X receptor; SMRT, silencing mediator for retinoic acid and thyroid hormone receptors; SNAPc, snRNA-activating protein complex; snRNA, small nuclear RNA; VZV, varicella-zoster virus

through POU-domain interaction with the adenovirus precursor terminal protein pTP [23]. Oct-1 is probably an attractive viral target because it is broadly expressed and possesses a versatile DNA-binding POU domain.

Versatility of the Oct-1 POU domain in protein-DNA and protein-protein interaction

The POU domain is unusual in that it consists of two independent DNA-binding modules – a POU-specific (POUS) domain and POU-homeo (POUH) domain – tethered by an unstructured linker [3,4]. On the natural high-affinity Oct-1 octamer (ATGCAAAT) binding site, the two Oct-1 POU-subdomains lie on opposite sides of the DNA (Fig.3a). The unstructured linker, however, permits flexible Oct-1 POU-subdomain positioning on the DNA and, thus, allows diversity in Oct-1 DNA-sequence recognition. For example, HSV TAATGARAT elements fall into two classes called (OCTA+) TAATGARAT and (OCTA−) TAATGARAT depending on whether or not they contain an overlapping octamer sequence. (OCTA−) TAATGARAT sites bear little resemblance to an octamer site, and here the POUS domain switches its position relative to the POUH domain (compare Fig.3b top and bottom). On an (OCTA+) TAATGARAT site, the conformation of Oct-1 is dynamic because it can bind in both octamer-like and (OCTA−) TAATGARAT-like conformations (Fig.3b, middle) [2]. Both conformations promote VP16-induced complex assembly, demonstrating freedom in Oct-1 DNA binding in the formation of this transcriptional regulatory complex.

The Oct-1 POU domain is also a versatile protein-protein interaction domain. Different POU-domain surfaces activate or repress transcription through protein-protein interactions. For example, in snRNA-gene transcriptional

http://tibs.trends.com

Fig. 3. Oct-1 (a) Schematic of the Oct-1 POU domain bound to the histone H2B octamer sequence. The POU-specific domain (POU<sub>S</sub>) is turquoise, POU-homeo domain (POU<sub>H</sub>) is blue and linker segment is yellow. Individual α helices of POU<sub>S</sub> and POU<sub>H</sub> domains are numbered. Adapted, with permission, from Ref. [3]. (b) Arrangements of the Oct-1 POU<sub>S</sub> and POU<sub>H</sub> domains on different DNA elements. Arrows indicate the relative orientation, where known, of the POU<sub>S</sub> and POU<sub>H</sub> domains on their DNA-binding sites (underlined and overlined, respectively).

activation, SNAP<sub>C</sub>/PTF – a basal machinery complex – contacts the Oct-1 POU<sub>S</sub> domain [15,24]. By contrast, during Ig-gene activation, OCA-B contacts both of the Oct-1 POU<sub>S</sub> and POU<sub>H</sub> domains [4,25]. In the VP16-induced complex, VP16 only contacts the POU<sub>H</sub> domain [4]. When binding to Oct-1, VP16 probably does not mimic a cellular factor because it binds to the top surface of the POU<sub>H</sub> domain [26], which is not conserved among mammals and, thus, unlikely to mediate an important cellular function. Although the Xenopus, chicken and human Oct-1 POU<sub>H</sub> domains are identical, the mouse Oct-1 POU<sub>H</sub> domain differs at four positions such that mouse Oct-1 fails to associate with VP16 effectively [12,13]. An interesting question is whether the changes in the mouse Oct-1 POU<sub>H</sub> domain were specifically selected in a rodent ancestor to avoid infection by an α herpesvirus.

HCF-1

The host-cell factor HCF-1 (also known as C1, VCAF and CFF) was first detected by virtue of its ability to stabilize the VP16-induced complex [27]. In cultured human cells, HCF-1 is an abundant chromatin-associated protein that is emerging as a key regulator of cell proliferation.

HCF-1 proteolytic processing and structural-element conservation

Human (Hs) HCF-1 is synthesized as a large 2035-amino-acid precursor protein [28] that is subsequently processed through proteolytic cleavage at a series of six centrally-located 26-amino-acid repeats called HCF-1<sub>PRO</sub> repeats [29,30] (Fig. 4a). Curiously, after cleavage the resulting N-terminal (HCF-1<sub>N</sub>) and C-terminal (HCF-1<sub>C</sub>) subunits remain noncovalently associated through the action of two pairs of ‘self-association sequences’ (SAS1 and SAS2; [31]; Fig. 4a).

HCF-1 contains several distinguishing regions (Fig. 4a) including (from N-to-C termini): (1) a Kelch domain, named because of its similarity to the Drosophila protein Kelch; (2) basic and acidic regions that are enriched in basic and acidic residues, respectively; (3) two fibronectin type 3 (Fn3) repeats that make up the SAS1C element; and (4) a C-terminal nuclear localization signal (NLS). The Kelch domain consists of six Kelch repeats (HCF-1<sub>Kel</sub>), which are predicted to form four-stranded β sheets that assemble into a six-bladed propeller-like structure [32]. It is this structure that binds VP16 and is sufficient to stabilize a VP16-induced complex [32,33]. HCF-1, however, does more than just stabilize the VP16-induced complex because the acidic region also contains a transcriptional activation domain, which cooperates with the activation domain of VP16 to activate transcription [34].

Figure 4a also shows HCF-1-related proteins from the fish *Fugu rubripes* (FrHCF-1) and two invertebrates: *Drosophila* (DmHCF; [35]) and *Caenorhabditis elegans* (CeHCF; [36]). All of the vertebrate HCF-1 proteins known – human, mouse [37], hamster [38] and fish – share the aforementioned HCF-1 structural features. The invertebrate proteins conserve the terminal Kelch and Fn3 repeat domains but not the intervening sequences. In the DmHCF protein, although not displaying sequence identity, there are corresponding basic- and acidic-residue enriched regions; the DmHCF protein undergoes proteolytic processing near the boundary between these DmHCF

Review TREND$S$ in Biochemical Sciences Vol.28 No.6 June 2003 299

(a)                                                                                             (b)

Interacting
chromatin
modifying
complexes
Interacting
proteins

Set1/Ash2 HMT                      Sin3 HDAC
VP16, LZIP,                        GABPβ
HPIP, Zhangfei                     Miz-1 Sp1
tsBN 67-rescue region

Function
Chromatin
association &
VIC formation
Structure
Kelch domain &
HCF-1_KEL repeats

Basic
HCF-1 processing
HCF-1_PRO repeats
Acidic
SAS2N                              SAS2C                               SAS1C
HCF-1_Fn3
repeats

HsHCF-1
FrHCF-1
DmHCF
CeHCF
HsHCF-2

Δ382–450
tsBN 67
P134S

PP1
PDCD2
Activation
domain

HCF-1_N
HCF-1_C
Kelch
NLS
2035
1745
1500
782
792

DNA-binding
protein or complex
VP16/Oct-1
'TAATGARAT'
??

TiBS

Fig.4.HCF-1 (a) Overall structure of HCF-1 and HCF-family members. The top diagram represents a schematic representation of human (Hs) HCF-1. Above this diagram structural or sequence elements, functional regions, interacting proteins and chromatin-modifying complexes of HCF-1 are shown. Below, the positions of the P134S mis-sense mutation that causes temperature-sensitive cell-proliferation arrest in tsBN 67 cells and positions of an alternatively spliced form that disrupts N (HCF-1_N)-terminal and C (HCF-1_C)-terminal association (Δ382–450) are indicated. Lower diagrams represent Fugu rubripes (Fr) HCF-1 (Ensembl gene ID SINFRUG 00000137201; SWISS-PROT gene ID Q61191), Drosophila melanogaster (Dm) HCF, Caenorhabditis elegans (Ce) HCF and human (Hs) HCF-2. Structural regions that are conserved between HCF proteins are indicated with the same color pattern as in HCF-1. (b) HCF-1 uses the same Kelch domain (green barrel) to indirectly bind to the HSV IE-gene TAATGARAT element (pale purple) via association with VP16/Oct-1 (dark red) and to associate with unknown chromosomal DNA sites (pale purple) in the uninfected cell probably via binding to an individual DNA-binding protein or protein complex (light red).

basic and acidic regions [35]. By contrast, the CeHCF protein lacks independent basic and acidic regions, and is not proteolytically cleaved [39]. These observations suggest that, although conserved, HCF-1 functions have diverged during animal evolution.

Cellular HCF-1 partners
The modular nature of HCF-1 provides many potential protein-interaction surfaces. Indeed, proteins that bind to different regions of HCF-1 have been discovered via indirect means (e.g. two-hybrid screens; Fig. 4a). Three of these proteins – LZIP/Luman, Zhangfei and HPIP – interact with the Kelch domain [40–44]. Three others – GABPβ, Sp1 and Miz-1 – are transcription factors that bind to the basic region [45–47]. Lastly, protein phosphatase 1 (PP1; [48]) and the programmed cell-death protein PDCD2 [49] bind the HCF-1_C subunit. The relevance of many of these and other unmapped interactions (e.g. PGC-1α and β [50]) to the cellular function of HCF-1 is poorly understood (see, for example, [51]).

Recent results indicate that HCF-1 also associates with two human protein complexes that are involved in histone modification (Fig. 4a, top): the basic region associates with the Sin3 histone deacetylase (HDAC) complex and the Kelch domain associates with the trithorax-related Set1/ Ash2 lysine 4 histone H3 methyltransferase (HMT) [52]. HCF-1 tethers the Sin3 and Set1/Ash2 transcriptional

regulatory complexes together even though they are generally associated with opposite transcriptional outcomes – repression and activation of transcription, respectively. Nevertheless, this tethering is context-dependent because, consistent with its role in activating transcription, VP16 selectively binds HCF-1 that is associated with the activating Set1/Ash2 HMT complex in the absence of the repressive Sin3 HDAC complex. Thus, HCF-1 might broadly regulate transcription, both positively and negatively, through selective modulation of chromatin structure.

Cellular roles of HCF-1
HCF-1 is emerging as a key regulator of both cell growth and division – the two essential steps in cell proliferation. HCF-1 is implicated in multiple aspects of cell proliferation because a proline-to-serine substitution at position 134 (P134S; Fig. 4a) of the HCF-1 Kelch domain, encoded by the single X chromosome-linked allele of the gene encoding HCF-1 in the hamster-cell line tsBN 67, causes a temperature-induced cell-growth arrest [38] and a defect in cytokinesis [53]. The P134S HCF-1-induced cell-growth arrest is unusual because tsBN 67 cells do not arrest immediately after temperature shift. Instead, they continue proliferating for 36–48 h during which HCF-1 dissociates from the chromatin [54] and, subsequently, they enter a stable and reversible cell-proliferation arrest [38]. The

http://tibs.trends.com

arrested cells contain active hypophosphorylated pRb [53] and the majority have a 2C-DNA content [38] consistent with a G1/G0-phase arrest; about a quarter of the cells, however, arrest with a 4C-DNA content [38] and are binucleated, suggesting a G1/G0 arrest after improper cytokinesis [53]. This prominent cytokinesis defect is due to loss of HCF-1 function, indicating that HCF-1 is involved in both cell growth and division [53].

Although the precise molecular roles of HCF-1 in the cell are unknown, HCF-1 probably promotes cell proliferation, at least in part, via regulation of gene transcription because it acts as a transcriptional co-activator for LZIP and GABP [34,45], a transcriptional co-repressor for Miz-1 [47], and associates with chromatin modifying activities that can repress (Sin3 HDAC) and activate (Set1/Ash2 HMT) transcription [52]. The role of HCF-1 in Miz-1 function is particularly interesting. Miz-1 can activate the promoter for the cyclin-dependent kinase inhibitor p15INK4b, which inhibits pRb inactivation by phosphorylation. HCF-1 can repress Miz-1 activation of p15INK4b-gene transcription, suggesting that HCF-1 indirectly promotes pRb inactivation by phosphorylation [47], which is disrupted upon loss of HCF-1 function in tsBN67 cells [53].

Consistent with these observations, Miz-1 overexpression leads to a G1 cell-cycle arrest similar to the HCF-1 loss-of-function tsBN67-cell proliferation arrest [55], and the pRb-inactivating function of viral oncoproteins (e.g. HPV E7 and SV40 T antigen) can prevent both Miz-1- [55] and tsBN67 temperature-induced arrest [56]. Thus, by antagonizing Miz-1 function, HCF-1 might promote cell proliferation. This model, however, probably does not explain all of the roles of HCF-1 in cell proliferation, for example, it does not explain the role of the Kelch domain in cell proliferation. Whichever the case, a role of HCF-1 in cellular transcription and proliferation might have made it an attractive partner for VP16 in controlling the course of HSV infection.

**Viral mimicry**

In contrast to its association with Oct-1, VP16 probably mimics an important cellular interaction in the way it associates with HCF-1. We conclude this because, whereas only vertebrate Oct-1-like proteins support VP16-induced complex formation, HCF-protein activity (i.e. the ability to stabilize a complex between VP16 and Oct-1 on a TAAT-GARAT site) is present in all metazoans tested [57,58], but not in the yeast Saccharomyces cerevisiae [58]. It is unlikely that HCF proteins have conserved the ability to stabilize a VP16-induced complex simply to aid vertebrate viral pathogens!

This mimicry concept is underscored by the similarity between the way in which VP16 proteins and the three known cellular HCF-1 Kelch-domain-binding proteins (LZIP/Luman, Zhangfei and HPIP) associate with HCF-1; these proteins all recognize the HCF-1 Kelch domain via a shared short tetrapeptide HCF-1-binding motif (HBM), [D/E]HxY [40,42–44]. Thus, the way in which HCF-1 associates with VP16 on viral DNA might be analogous to how HCF-1 associates with chromatin in uninfected cells (Fig. 4b). Indeed, as for VP16-induced complex formation,

the HCF-1 Kelch domain is necessary and sufficient for HCF-1 chromatin association [54]. HCF-1 is not known to bind DNA independently, instead the Kelch domain probably tethers HCF-1 to chromatin by contacting HBM-containing DNA-binding proteins. Consistent with this model, the tsBN67 P134S mutation – which interferes with HCF-1 association with chromatin at the nonpermissive temperature [54] – also disrupts association with both VP16 and the cellular HBM-containing proteins [32,40,42–44]. Interestingly, although the Kelch domain is sufficient for transcriptional activation by VP16 [32] and chromatin association [54], it is not sufficient to rescue the tsBN67-cell proliferation defect – the adjacent basic region is also required [32] (Fig. 4a). Thus, VP16 probably mimics only a subset of the cellular effectors that HCF-1 requires to promote cell proliferation.

**The VP16-induced complex: a model for combinatorial selectivity of transcription**

VP16 is the linchpin of the VP16-induced complex and each interaction between VP16 and Oct-1, HCF-1 and the DNA provides a mechanism for combinatorial selectivity of transcription (Fig. 5).

**Selective co-regulator function via differential cis-element recognition**

Figure 5a illustrates how the sequence of the TAATGARAT element can confer specificity for the formation of a VP16-induced complex by the VP16 co-regulator. Owing to differences in the structured region of the VP16 core [8], the human HSV and bovine BHV-1 VP16 molecules (H-VP16 and B-VP16, respectively) recognize different three-base-pair ‘D’ element sequences flanking the TAATGARAT core sequence of the full TAATGARAT VP16-response element (VRE). Through this difference, the H-VP16 protein forms an active VP16-induced complex on a cognate H-VRE but not on a bovine B-VRE, whereas the B-VP16 protein forms an active VP16-induced complex on a B-VRE but not on an H-VRE [59,60]. Thus, differential DNA-sequence recognition by these two co-regulators confers different promoter-specific transcriptional activities to the homeo-domain transcription factor Oct-1.

**Selective activator function via discriminating activator-co-regulator association**

VP16 interaction with the Oct-1 POUH domain provides a second mechanism for differential transcriptional regulation because VP16 proteins discriminate between the Oct-1 POUH domain and that of the closely related B-cell POU-domain transcription factor Oct-2. Oct-1 and Oct-2 possess identical DNA-contacting surfaces, but the solvent-exposed surfaces differ and one of the amino-acid differences controls the ability and inability of Oct-1 and Oct-2, respectively, to associate with VP16 [2]. Thus, because of this one amino acid difference, VP16 is able to activate transcription in the presence of Oct-1 but not Oct-2 [13] (Fig. 5b). These activities show how two proteins that display the same intrinsic DNA-binding specificities can have very different transcriptional

http://tibs.trends.com

Review
TRENDS in Biochemical Sciences Vol.28 No.6 June 2003

On (promoter active) Off (promoter inactive)

(a)
![Diagram](#)

(b)
![Diagram](#)

(c)
![Diagram](#)

Fig. 5. Mechanisms of combinatorial control of transcription by the VP16-induced complex. (a) Selective recognition of human (H) but not bovine (B) TAATGARAT core sequence VP16-responsive elements (VRE) by human VP16 (H-VP16), and B-VRE but not H-VRE recognition by bovine VP16 (B-VP16). The D element (D) is also shown. (b) Selective co-recruitment of Oct-1 (blue) but not Oct-2 (green) to the TAATGARAT VP16-responsive element (TAATGARAT core + D element) by the VP16/HCF-1 complex. (c) Both HCF-1 (red) and HCF-2 (orange) stabilize the VP16-induced complex on a TAATGARAT element, but only the HCF-1-containing complex activates transcription effectively.

activities owing to a single amino-acid difference that controls their association with a co-regulator.

Selective transcriptional activation by related VP16-induced complexes

A human relative of HCF-1, called HCF-2 [61] (Fig. 4a), provides a third mechanism for combinatorial control (Fig. 5c). Here, both HCF proteins stabilize the VP16-induced complex, but only the HCF-1-containing complex activates transcription effectively [62]. Thus, diverse transcriptional outcomes are achieved by differences, not in complex formation, but in how the assembled complex interacts with the transcriptional machinery.

Why a VP16-induced complex?

Although elegant in their versatility, these mechanisms of combinatorial selectivity of transcription do not themselves explain why HSV complicates the infection process by depending on cellular factors to activate its own lytic

http://tibs.trends.com

is because a single infecting HSV virion deposits only 500–1000 molecules of VP16 into the cell, which is likely to contain many more Oct-1 and certainly many more HCF-1 molecules (there are ~100 000–500 000 molecules of HCF-1 per HeLa cell! [54]). It is therefore likely that VP16 takes advantage of properties afforded by Oct-1 and HCF-1 to coordinate the course of viral infection with the host cell.

### A regulatory switch

We suggest that the VP16-induced complex acts as a regulatory switch that, when turned on in permissive cells by productive VP16 association with Oct-1 and HCF-1, activates the lytic program of infection as occurs in epithelial cells and, when turned off, latent infection ensues, as occurs in sensory neurons. By ensuring that turning the switch on requires the joining of viral and cellular regulators, viral infection is automatically integrated into the nature and status of the infected cell. The mechanisms for turning on the switch through association with Oct-1 and HCF-1, two factors involved in cell growth and proliferation, have been documented above. What causes the switch to be off during latency establishment is less well understood, but might involve (1) the absence of functional VP16, (2) an altered status of Oct-1 or HCF-1, or (3) the presence of related proteins (e.g. other POU-domain or HCF-like proteins) that can interfere with the activity of the VP16-induced complex.

One key difference between the two cell types that HSV naturally infects is that, in contrast to sensory neurons, epithelial cells are proliferation competent. Because HCF-1 promotes the proliferation process, it is likely that in non-proliferating cells the status of HCF-1 is changed to accommodate or perhaps even promote the arrest of cell proliferation. Indeed, in non-proliferating sensory neurons, HCF-1 is detected in the cytoplasm and, thus, is perhaps not able to transport VP16 to the nucleus and promote transcriptional activation [63,64]. Therefore, VP16 might target HCF-1 for its cell-proliferation regulatory properties to gauge the proliferative status of the infected cell and, as a consequence, its appropriateness for lytic or latent infection. Oct-1, with its involvement in transcription of growth-related genes, might play a similar role.

Proteins related to Oct-1, for example POU-domain proteins present in the nervous system (e.g. Oct-2, Brn-1, Brn-2, Brn-3a–c and Brn-4 [1]), could affect the status of the regulatory switch in neurons. Only the Oct-1 POU domain is known to associate with VP16, but other POU proteins can bind TAATGARAT sites effectively, thus perhaps interfering with VP16-induced complex formation [65]; indeed, Brn-2 (N-Oct3) can bind a TAATGARAT site more tightly than Oct-1 [66]. Similarly, if present, HCF-2 – which stabilizes the VP16-induced complex but fails to support transcriptional activation [62] – might inhibit IE-gene transcription.

An unanswered question is that of the role of the VP16-induced-complex components in viral reactivation from latency. Interestingly, IE promoters can be active in neurons in the absence of VP16 [67]. Consistent with VP16-independent activation of IE-gene transcription,

Oct-1 binds, albeit weakly, to VP16-responsive TAAT-GARAT sites [13] and HCF-1 can interact with Oct-1, Sp1 and GABP – all HSV IE-gene activators – in the absence of VP16 [45,46,51], raising the possibility of VP16-independent activation of IE-gene transcription during reactivation. This possibility is underscored by the observation that Oct-1 levels and DNA-binding activity are induced upon UV exposure [17], a stimulus that can cause HSV reactivation from latency.

### Concluding remarks

This review has focused on one small transcriptional regulatory complex composed of just one viral and two cellular proteins. The master regulator of this complex, VP16, uses the two cellular proteins – the DNA-binding transcriptional regulator Oct-1 and cell-proliferation regulator HCF-1 – to direct early events in HSV infection. When the VP16 protein was first shown to activate HSV IE-gene transcription in 1984 [68], Oct-1 and HCF-1 were unknown; indeed, HCF-1 was discovered owing to its role in VP16-induced complex assembly [27]. Since that time, often through studies with VP16, much has been learned about the actions of these two key cellular regulators of gene expression and cell proliferation.

Human viruses, by their intimate interactions with the cellular machinery, provide excellent tools to uncover mechanisms by which the cells in our bodies function. The studies of the VP16-induced complex serve to reinforce this view and reveal how much more we have yet to learn about how HSV so successfully coexists with humans.

### Note added in proof

Recent studies show that the HCF-1<sub>N</sub> subunit promotes passage through the G1 phase and the HCF-1<sub>C</sub> subunit promotes proper cytokinesis. Proteolytic processing separates these two HCF-1 activities and ensures proper cytokinesis [Julien, E. and Herr, W. Proteolytic processing is necessary to separate and ensure proper cell growth and cytokinesis functions of HCF-1. *EMBO J.* (in press)]

### Acknowledgements

We are indebted to the past and present members and collaborators of our laboratory for their contributions to ideas described here. We thank A. Wilson for communicating results before publication; T. Swigut for advice on sequence alignment; A. Bubulya, N. Hernandez, M. Myers, J. Skowronski, R-M. Xu and X. Zhao for critical readings of the article, and anonymous reviewers of the article for their helpful comments. We regret that, owing to space limitations, it is not possible to refer to all the primary studies covered by this review.

### References

1. Andersen, B. and Rosenfeld, M.G. (2001) POU domain factors in the neuroendocrine system: lessons from developmental biology provide insights into human disease. *Endocr. Rev.* 22, 2–35
2. Herr, W. (1998) The herpes simplex virus VP16-induced complex: mechanisms of combinatorial transcriptional regulation. *Cold Spring Harb. Symp. Quant. Biol.* 63, 599–607
3. Herr, W. and Cleary, M.A. (1995) The POU domain: versatility in transcriptional regulation by a flexible two-in-one DNA-binding domain. *Genes Dev.* 9, 1679–1693
4. Phillips, K. and Luisi, B. (2000) The virtuoso of versatility: POU proteins that flex to fit. *J. Mol. Biol.* 302, 1023–1039
5. Preston, C.M. (2000) Repression of viral transcription during herpes simplex virus latency. *J. Gen. Virol.* 81, 1–19

6 Roizman, B. and Knipe, D.M. (2001) Herpes simplex viruses and their replication. In Fields Virology (Vol. 2), 4th edn, (Knipe, D.M. *et al.*, eds), pp. 2399–2459, Lippincott, Williams & Wilkins

7 Hall, D.B. and Struhl, K. (2002) The VP16 activation domain interacts with multiple transcriptional components as determined by protein–protein cross-linking *in vivo*. *J. Biol. Chem.* 277, 46043–46050

8 Liu, Y. *et al.* (1999) Crystal structure of the conserved core of the herpes simplex virus transcriptional regulatory protein VP16. *Genes Dev.* 13, 1692–1703

9 Babb, R. *et al.* (2001) DNA recognition by the herpes simplex virus transactivator VP16: a novel DNA-binding structure. *Mol. Cell. Biol.* 21, 4700–4712

10 Ace, C.I. *et al.* (1989) Construction and characterization of a herpes simplex virus type 1 mutant unable to transduce immediate-early gene expression. *J. Virol.* 63, 2260–2269

11 Tal-Singer, R. *et al.* (1999) The transcriptional activation domain of VP16 is required for efficient infection and establishment of latency by HSV-1 in the murine peripheral and central nervous systems. *Virology* 259, 20–33

12 Suzuki, N. *et al.* (1993) Mouse Oct-1 contains a composite homeodomain of human Oct-1 and Oct-2. *Nucleic Acids Res.* 21, 245–252

13 Cleary, M.A. *et al.* (1993) Differential positive control by Oct-1 and Oct-2: activation of a transcriptionally silent motif through Oct-1 and VP16 corecruitment. *Genes Dev.* 7, 72–83

14 Steiner, I. *et al.* (1990) A herpes simplex virus type 1 mutant containing a nontransinducing Vmw65 protein establishes latent infection *in vivo* in the absence of viral replication and reactivates efficiently from explanted trigeminal ganglia. *J. Virol.* 64, 1630–1638

15 Schramm, L. and Hernandez, N. (2002) Recruitment of RNA polymerase III to its target promoters. *Genes Dev.* 16, 2593–2620

16 Fletcher, C. *et al.* (1987) Purification and characterization of OTF-1, a transcription factor regulating cell cycle expression of a human histone H2B gene. *Cell* 51, 773–781

17 Zhao, H. *et al.* (2000) Activation of the transcription factor Oct-1 in response to DNA damage. *Cancer Res.* 60, 6276–6280

18 Jin, S. *et al.* (2001) Transcription factors Oct-1 and NF-YA regulate the p53-independent induction of the GADD45 following DNA damage. *Oncogene* 20, 2683–2690

19 Fan, W. *et al.* (2002) BRCA1 regulates GADD45 through its interactions with the OCT-1 and CAAT motifs. *J. Biol. Chem.* 277, 8061–8067

20 Sakakibara, S. *et al.* (2001) Octamer-binding sequence is a key element for the autoregulation of Kaposi’s sarcoma-associated herpesvirus ORF 50/Lyta gene expression. *J. Virol.* 75, 6894–6900

21 Bruggemeier, U. *et al.* (1991) Ubiquitous transcription factor OTF-1 mediates induction of the MMTV promoter through synergistic interaction with hormone receptors. *Cell* 64, 565–572

22 Prefontaine, G.G. *et al.* (1998) Recruitment of octamer transcription factors to DNA by glucocorticoid receptor. *Mol. Cell. Biol.* 18, 3416–3430

23 de Jong, R.N. *et al.* (2002) Recruitment of the priming protein pTP and DNA binding occur by overlapping Oct-1 POU homeodomain surfaces. *EMBO J.* 21, 725–735

24 Hovde, S. *et al.* (2002) Activator recruitment by the general transcription machinery: X-ray structural analysis of the Oct-1 POU domain/human U1 octamer/SNAP190 peptide ternary complex. *Genes Dev.* 16, 2772–2777

25 Chasman, D. *et al.* (1999) Crystal structure of an OCA-B peptide bound to an Oct-1 POU domain/octamer DNA complex: specific recognition of a protein-DNA interface. *Genes Dev.* 13, 2650–2657

26 Pomerantz, J.L. *et al.* (1992) Recognition of the surface of a homeo domain protein. *Genes Dev.* 6, 2047–2057

27 Gerster, T. and Roeder, R.G. (1988) A herpesvirus trans-activating protein interacts with transcription factor OTF-1 and other cellular proteins. *Proc. Natl. Acad. Sci. U. S. A.* 85, 6347–6351

28 Wilson, A.C. *et al.* (1993) The VP16 accessory protein HCF-1 is a family of polypeptides processed from a large precursor protein. *Cell* 74, 115–125

29 Kristie, T.M. *et al.* (1995) The cellular C1 factor of the herpes simplex virus enhancer complex is a family of polypeptides. *J. Biol. Chem.* 270, 4387–4394

30 Wilson, A.C. *et al.* (1995) The HCF repeat is an unusual proteolytic cleavage signal. *Genes Dev.* 9, 2445–2458

31 Wilson, A.C. *et al.* (2000) HCF-1 amino- and carboxy-terminal subunit association through two separate sets of interaction modules: involvement of fibronectin type 3 repeats. *Mol. Cell. Biol.* 20, 6721–6730

32 Wilson, A.C. *et al.* (1997) VP16 targets an amino-terminal domain of HCF involved in cell cycle progression. *Mol. Cell. Biol.* 17, 6139–6146

33 LaBoissière, S. *et al.* (1997) Concerted activity of host cell factor subregions in promoting stable VP16 complex assembly and preventing interference by the acidic activation domain. *Mol. Cell. Biol.* 17, 7108–7118

34 Luciano, R.L. and Wilson, A.C. (2002) An activation domain in the C-terminal subunit of HCF-1 is important for transactivation by VP16 and LZIP. *Proc. Natl. Acad. Sci. U. S. A.* 99, 13403–13408

35 Mahajan, S.S. *et al.* (2003) Molecular cloning of *Drosophila* HCF reveals proteolytic processing and self-association of the encoded protein. *J. Cell. Physiol.* 194, 117–126

36 Liu, Y. *et al.* (1999) Selected elements of herpes simplex virus accessory factor HCF are highly conserved in *Caenorhabditis elegans*. *Mol. Cell. Biol.* 19, 909–915

37 Kristie, T.M. (1997) The mouse homologue of the human transcription factor C1 (host cell factor). Conservation of forms and function. *J. Biol. Chem.* 272, 26749–26755

38 Goto, H. *et al.* (1997) A single-point mutation in HCF causes temperature-sensitive cell-cycle arrest and disrupts VP16 function. *Genes Dev.* 11, 726–737

39 Wysocka, J. *et al.* (2001) Developmental and cell-cycle regulation of *Caenorhabditis elegans* HCF phosphorylation. *Biochemistry* 40, 5786–5794

40 Freiman, R.N. and Herr, W. (1997) Viral mimicry: common mode of association with HCF by VP16 and the cellular protein LZIP. *Genes Dev.* 11, 3122–3127

41 Lu, R. *et al.* (1997) Luman, a new member of the CREB/ATF family, binds to herpes simplex virus VP16-associated host cellular factor. *Mol. Cell. Biol.* 17, 5117–5126

42 Lu, R. *et al.* (1998) The herpesvirus transactivator VP16 mimics a human basic domain leucine zipper protein, Luman, in its interaction with HCF. *J. Virol.* 72, 6291–6297

43 Lu, R. and Misra, V. (2000) Zhangfei: a second cellular protein interacts with herpes simplex virus accessory factor HCF in a manner similar to Luman and VP16. *Nucleic Acids Res.* 28, 2446–2454

44 Mahajan, S.S. *et al.* (2002) Interaction of HCF-1 with a cellular nuclear export factor. *J. Biol. Chem.* 277, 44292–44299

45 Vogel, J.L. and Kristie, T.M. (2000) The novel coactivator C1 (HCF) coordinates multiprotein enhancer formation and mediates transcription activation by GABP. *EMBO J.* 19, 683–690

46 Gunther, M. *et al.* (2000) A set of proteins interacting with transcription factor Sp1 identified in a two-hybrid screening. *Mol. Cell. Biochem.* 210, 131–142

47 Piluso, D. *et al.* (2002) Host cell factor-1 interacts with and antagonizes transactivation by the cell cycle regulatory factor miz-1. *J. Biol. Chem.* 277, 46799–46808

48 Ajuh, P.M. *et al.* (2000) Association of a protein phosphatase 1 activity with the human factor C1 (HCF) complex. *Nucleic Acids Res.* 28, 678–686

49 Scarr, R.B. and Sharp, P.A. (2002) PDCD2 is a negative regulator of HCF-1 (C1). *Oncogene* 21, 5245–5254

50 Lin, J. *et al.* (2002) Peroxisome proliferator-activated receptor γ coactivator 1β (PGC-1β), a novel PGC-1-related transcription coactivator associated with host cell factor. *J. Biol. Chem.* 277, 1645–1648

51 Mahajan, S.S. and Wilson, A.C. (2000) Mutations in host cell factor 1 separate its role in cell proliferation from recruitment of VP16 and LZIP. *Mol. Cell. Biol.* 20, 919–928

52 Wysocka, J. *et al.* (2003) Human Sin3 deacetylase and trithorax-related Set1/Ash2 histone H3-K4 methyltransferase are tethered together selectively by the cell-proliferation factor HCF-1. *Genes Dev.* 17, 896–911

53 Reilly, P.T. and Herr, W. (2002) Spontaneous reversion of tsBN67 cell proliferation and cytokinesis defects in the absence of HCF-1 function. *Exp. Cell Res.* 277, 119–130

54 Wysocka, J. *et al.* (2001) Loss of HCF-1 chromatin association precedes temperature-induced growth arrest of tsBN67 cells. *Mol. Cell. Biol.* 21, 3820–3829

55 Orian, A. and Eisenman, R.N. (2001) TGF-β flips the Myc switch. *Sci. STKE* 88, PE1

56 Reilly, P.T. *et al.* (2002) Inactivation of the retinoblastoma protein family can bypass the HCF-1 defect in tsBN 67 cell proliferation and cytokinesis. *Mol. Cell. Biol.* 22, 6767–6778

57 Kristie, T.M. *et al.* (1989) The octamer-binding proteins form multi-protein-DNA complexes with the HSV α TIF regulatory protein. *EMBO J.* 8, 4229–4238

58 Wilson, A.C. *et al.* (1993) Combinatorial control of transcription: the herpes simplex virus VP16-induced complex. *Cold Spring Harb. Symp. Quant. Biol.* 58, 167–178

59 Huang, C.C. and Herr, W. (1996) Differential control of transcription by homologous homeodomain coregulators. *Mol. Cell. Biol.* 16, 2967–2976

60 Misra, V. *et al.* (1996) Conformational alteration of Oct-1 upon DNA binding dictates selectivity in differential interactions with related transcriptional coactivators. *Mol. Cell. Biol.* 16, 4404–4413

61 Johnson, K.M. *et al.* (1999) Herpes simplex virus transactivator VP16 discriminates between HCF-1 and a novel family member, HCF-2. *J. Virol.* 73, 3930–3940

62 Lee, S. and Herr, W. (2001) Stabilization but not the transcriptional activity of herpes simplex virus VP16-induced complexes is evolutionarily conserved among HCF family members. *J. Virol.* 75, 12402–12411

63 Kristie, T.M. *et al.* (1999) Nuclear localization of the C1 factor (host cell factor) in sensory neurons correlates with reactivation of herpes simplex virus from latency. *Proc. Natl. Acad. Sci. U. S. A.* 96, 1229–1233

64 LaBoissière, S. *et al.* (1999) HCF-dependent nuclear import of VP16. *EMBO J.* 18, 480–489

65 Latchman, D.S. (1999) Regulation of DNA virus transcription by cellular POU family transcription factors. *Rev. Med. Virol.* 9, 31–38

66 Hagmann, M. *et al.* (1995) Transcription factors interacting with herpes simplex virus alpha gene promoters in sensory neurons. *Nucleic Acids Res.* 23, 4978–4985

67 Loiacono, C.M. *et al.* (2002) Neurons differentially activate the herpes

simplex virus type 1 immediate-early gene ICP0 and ICP27 promoters in transgenic mice. *J. Virol.* 76, 2449–2459

68 Campbell, M.E. *et al.* (1984) Identification of herpes simplex virus DNA sequences which encode a trans-acting polypeptide responsible for stimulation of immediate early transcription. *J. Mol. Biol.* 180, 1–19

69 Strom, A.C. *et al.* (1996) The transcription factors Sp1 and Oct-1 interact physically to regulate human U2 snRNA gene expression. *Nucleic Acids Res.* 24, 1981–1986

70 Fadel, B.M. *et al.* (1999) Octamer-dependent in vivo expression of the endothelial cell-specific TIE2 gene. *J. Biol. Chem.* 274, 20376–20383

71 Boutet, S.C. *et al.* (2001) Identification of an octamer element required for in vivo expression of the TIE1 gene in endothelial cells. *Biochem. J.* 360, 23–29

72 Vazquez-Martinez, R. *et al.* (2002) Episodic activation of the rat GnRH promoter: role of the homeoprotein oct-1. *Mol. Endocrinol.* 16, 2093–2100

73 Kakizawa, T. *et al.* (2001) Silencing mediator for retinoid and thyroid hormone receptors interacts with octamer transcription factor-1 and acts as a transcriptional repressor. *J. Biol. Chem.* 276, 9720–9725

74 Kakizawa, T. *et al.* (1999) Functional interaction between Oct-1 and retinoid X receptor. *J. Biol. Chem.* 274, 19103–19108

75 Cron, R.Q. *et al.* (2001) Octamer proteins inhibit IL-4 gene transcription in normal human CD4T cells. *Genes Immun.* 2, 464–468

76 Imai, S. *et al.* (1997) Dissociation of Oct-1 from the nuclear peripheral structure induces the cellular aging-associated collagenase gene expression. *Mol. Biol. Cell* 8, 2407–2419

77 Kim, M.K. *et al.* (1996) A soluble transcription factor, Oct-1, is also found in the insoluble nuclear matrix and possesses silencing activity in its alanine-rich domain. *Mol. Cell. Biol.* 16, 4366–4377

78 Moriuchi, H. *et al.* (1995) Proteins and cis-acting elements associated with transactivation of the varicella-zoster virus (VZV) immediate-early gene 62 promoter by VZV open reading frame 10 protein. *J. Virol.* 69, 4693–4701

---

**TiBS COMPETITION**

Are there any talented poets or songwriters in your lab? Did you see 'Coomassie Blues' and feel inspired? TiBS is running a competition to find the best poem or song relating to TiBS, biochemical sciences or life in the lab. The prize will be a free subscription to TiBS for one year. The winning and runner-up poems/songs will feature in forthcoming issues of TiBS. So get writing!

Send your entry to:

Fiona G. Hutton, Editor  
Trends in Biochemical Sciences  
Elsevier Science  
84 Theobalds Road  
London, UK WC1X 8RR

http://tibs.trends.com
