
Genes that act downstream of DAF-16  
to influence the lifespan of  
Caenorhabditis elegans  

Coleen T. Murphy*, Steven A. McCarroll†, Cornelia I. Bargmann†, Andrew Fraser‡, Ravi S. Kamath‡, Julie Ahringer‡, Hao Li* & Cynthia Kenyon

* Department of Biochemistry and Biophysics, and † Department of Anatomy and Howard Hughes Medical Institute, University of California, San Francisco, California 94143-2200, USA  
‡ Wellcome CRC Institute and Department of Genetics, University of Cambridge, Tennis Court Road, Cambridge CB2 1QR, UK  

Ageing is a fundamental, unsolved mystery in biology. DAF-16, a FOXO-family transcription factor, influences the rate of ageing of *Caenorhabditis elegans* in response to insulin/insulin-like growth factor 1 (IGF-I) signalling. Using DNA microarray analysis, we have found that DAF-16 affects expression of a set of genes during early adulthood, the time at which this pathway is known to control ageing. Here we find that many of these genes influence the ageing process. The insulin/IGF-I pathway functions cell non-autonomously to regulate lifespan, and our findings suggest that it signals other cells, at least in part, by feedback regulation of an insulin/IGF-I homologue. Furthermore, our findings suggest that the insulin/IGF-I pathway ultimately exerts its effect on lifespan by upregulating a wide variety of genes, including cellular stress-response, antimicrobial and metabolic genes, and by downregulating specific life-shortening genes.

The recent discovery that the ageing process is regulated hormonally by an evolutionarily conserved insulin/IGF-I signalling pathway has provided a powerful entry point for understanding the causes of ageing at the molecular level. The nematode *C. elegans* lives for only a few weeks; however, animals carrying mutations that decrease insulin/IGF-I signalling, such as *daf-2* insulin/IGF-I receptor mutations, remain youthful and live more than twice as long as normal. The insulin/IGF-I system also regulates reproduction and lipid metabolism, as well as entry into a state of diapause called dauer. The dauer is an arrested, long-lived juvenile form normally induced by food limitation and also by strong *daf-2* mutations. The DAF-2 pathway regulates reproduction, lipid metabolism, dauer formation and ageing independently of one another. For example, whereas it acts during development to regulate dauer formation, it acts exclusively in the adult to influence ageing.

The DAF-2 pathway exerts its effects on the animal by influencing downstream gene expression, as the ability of *daf-2* mutations (*daf-2(−)*) to increase lifespan or produce other Daf-2(−) phenotypes depends on the activity of DAF-16 (refs 5–7,11,13), a FOXO-family transcription factor. In wild-type animals, the activity of DAF-16 is inhibited by a conserved phosphatidylinositol-3-OH kinase (PI(3)K)/protein kinase D (PDK)/Akt pathway in response to DAF-2 activity.

It should be possible to learn how insulin/IGF-I signalling influences ageing by identifying and characterizing the genes regulated by DAF-16. Some of these genes are predicted to encode or regulate downstream signals or hormones, because *daf-2* (and therefore presumably *daf-16*) functions cell non-autonomously: removing *daf-2* activity from subsets of cells can cause the entire animal to enter the dauer state, or to become a long-lived adult. In addition, DAF-16 is predicted to influence expression of genes whose activities influence the ageing process directly. Animals with reduced DAF-2 pathway activity are resistant to heat and oxidative stress, which has suggested that an increased ability to prevent or repair oxidative damage increases lifespan. Consistent with this idea, overexpressing superoxide dismutase can extend the lifespan of *Drosophila* and yeast. However, this hypothesis has never been tested directly; for example, by asking whether stress response genes are required for the longevity of *daf-2* mutants. (An influential report was retracted recently.)

In this study, we have identified genes that are regulated by DAF-16 and investigated their roles in the ageing process. To do this, we used microarray analysis to identify downstream genes, and then carried out a functional analysis of these genes using RNA interference (RNAi).

### DNA microarray analysis

We identified genes whose expression changed in insulin/IGF-I pathway mutants using DNA microarrays containing approximately 93% of the predicted *C. elegans* open reading frames. We did this in two ways. First, we compared the transcriptional profiles of multiple alleles of long-lived *daf-2* and *age-1*/PI(3)K mutants to profiles of wild-type animals and *daf-16; daf-2* double mutants on the first day of adulthood. We grouped genes with similar expression patterns by hierarchical clustering of those with at least fourfold expression changes. This allowed us to identify genes that were upregulated or downregulated across the set of arrays. We also identified genes that were regulated in a highly consistent fashion, regardless of the degree to which their expression was changed (Supplementary Table 1s).

In addition, we reduced *daf-2* and *daf-16* activity using RNAi, which phenocopies *daf-2* and *daf-16* mutants. This allowed us to analyse the transcriptional profiles of isogenic and developmentally synchronous animals. We grew a sterile strain (*fer-15(b26); fem-1(hc17)*) on bacteria expressing *daf-2* double-stranded (ds)RNA, both *daf-2* and *daf-16* dsRNA, or control bacteria, and collected the animals at intervals throughout adulthood. Because reducing the level of insulin/IGF-I signalling during early adulthood is sufficient to increase lifespan, we carried out an early adult time course (ten time points from 0–48 h of adulthood; Fig. 1a) to identify changes that occurred during this period. We also carried out a longer time course (ten time points from 0–192 h of adulthood) to identify changes that occurred as these animals began to age, but before a significant fraction died (Fig. 1a).

articles

**The early ageing transcriptome is largely unaffected**

Because mutations in the insulin/IGF-I pathway slow the rate of ageing ${ }^{5,25,26}$, we wondered whether reducing $daf-2$ activity would slow the rate of all age-associated changes in gene expression. To investigate this, we compared the whole-transcriptome profiles of RNAi-treated animals at different ages (Fig. 1b, c). In the early adult time course, before the different strains began to differ morphologically ${ }^{25}$ (0–48 h; Fig. 1b), we found that a subset of genes was expressed differently between animals exposed to $daf-2$ RNAi and animals exposed to both $daf-16$ and $daf-2$ RNAi—we refer to these animals, which do not have long lifespans (Fig. 1a), as $daf-16(RNAi)$; $daf-2(RNAi)$ animals. These differences in expression persisted during the longer time course (0–192 h). During this period, the expression of many other genes changed as well; however, most of these age-dependent changes were not different between the $daf-2(RNAi)$ and $daf-16(RNAi)$; $daf-2(RNAi)$ animals. This was surprising, as the tissue morphology of these animals differs significantly by the end of this period ${ }^{25}$. Together these findings raised the possibility that the insulin/IGF-I pathway might influence ageing through a relatively small set of physiologically important targets that were differentially expressed even in young adults.

---

**Two classes of downstream genes**

We combined the data from our 60 microarrays into a single set and performed hierarchical clustering ${ }^{23}$ (Fig. 2; see also Supplementary Information). We then focused on clusters that showed opposite expression profiles under $daf-2(-)$ and $daf-16(-)$ conditions. By examining a variety of mutants in multiple experiments and by performing two longitudinal studies, we were able to eliminate false positives caused by differences in developmental rates and by systematic errors. This approach revealed a relatively small number of differentially expressed $daf-2/daf-16$-dependent targets.

Two clusters were of particular interest. The first contained genes that were induced in DAF-2 pathway mutants and in $daf-2(RNAi)$ animals but repressed in $daf-16(RNAi)$; $daf-2(RNAi)$ animals (class 1). These were candidates for genes that extend lifespan (Fig. 2; see also Supplementary Table 1s). The second cluster contained genes that displayed the opposite profile (class 2, Fig. 2; see also Supplementary Table 1s), and are candidates for genes that shorten lifespan. This approach identified genes previously thought to be regulated by DAF-16, such as the metallothionein homologue $mtl-1$ (ref. 27) and the mitochondrial superoxide dismutase gene $sod-3$ (ref. 28). We carried out polymerase chain reaction with reverse transcription (RT–PCR) of several RNA samples with $sod-3$- and $mtl-1$-specific primers, and found that expression of both was increased in $daf-2(RNAi)$ animals (data not shown), confirming our microarray results.

---

**A positive feedback loop amplifies DAF-2 pathway activity**

In humans, reduced insulin receptor activity in the pancreas reduces insulin production. We found that gene expression of $ins-7$, which encodes an insulin/IGF-1-like peptide, was repressed in animals with reduced $daf-2$ activity and elevated in animals with reduced $daf-16$ activity. More than 35 insulin-like genes have been identified in the C. elegans genome ${ }^{29-31}$, and 23 of these insulin-like genes were represented on our microarrays. A number of insulin-like peptides have been implicated in DAF-2 regulation ${ }^{30-32}$. To investigate whether $ins-7$ might function as a DAF-2 agonist, we inhibited its activity using RNAi. We found that $ins-7$ RNAi increased the lifespan of $daf-2(+)$ animals significantly (Fig. 3a), but was unable to further extend the lifespan of long-lived $daf-2(mu150)$ animals (see Supplementary Table 2b). Furthermore, $ins-7$ RNAi increased the frequency of dauer formation (Fig. 3b). Thus INS-7 behaved as expected for a DAF-2 agonist.

The regulatory properties of $ins-7$ suggest that it might contribute to the non-autonomy of $daf-2$ function. In this model, if $daf-2$ gene activity is removed from cells that normally express $ins-7$, the level of $ins-7$ expression will fall, which in turn will lower the level of INS-7 available to activate the DAF-2 receptor present on wild-type cells (Fig. 3c). In addition, the regulatory properties of INS-7 might contribute to an interesting phenomenon that occurs in nature. When a population of wild-type juvenile animals is confronted with a diminishing food supply (or when temperature-sensitive $daf-2$ mutants are grown at a semi-permissive temperature), some but not all individuals enter the dauer state. It is interesting that under these threshold conditions, one does not observe animals containing random mixtures of dauer and non-dauer cells. It is possible that the INS-7 positive feedback loop contributes to this cellular conformity. In this model, a downward or upward fluctuation in the level of INS-7 would be amplified, which in turn would bias all of the cells in the animals towards dauer or adult development, respectively.

---

**Additional downstream signalling molecules**

In addition to $ins-7$, a number of other genes that encoded potential signalling molecules were regulated by DAF-2 and DAF-16. One was a known $daf-2/daf-16$-regulated gene, $scl-1$, which encodes a putative secreted protein that promotes longevity ${ }^{33}$. Furthermore, a large

number of class 1 (daf-2 (-)-induced) genes encoded proteins that might potentially participate in the synthesis of a steroid or lipid-soluble hormone, including four cytochrome P450s, two estradiol-17-β-dehydrogenases, two alcohol/short-chain dehydrogenases, several esterases, two UDP-glucuronosyltransferases, and several fat genes known to function in fatty acid desaturation (Supplementary Table 1s). We investigated the functions of many of these genes and found that, in most cases, reducing their activities with RNAi shortened lifespan up to 20% (Fig. 4a, b). Together these findings suggest that the DAF-2 pathway may regulate multiple downstream signalling molecules. We also found that gcy-6 and gcy-18, two receptor guanylate cyclases that are expressed in neurons³⁴, were repressed under daf-2 (-) conditions (class 2). Inhibiting their activities lengthened the lifespan of daf-2 (+) animals (Fig. 5a). Thus insulin/IGF-1 signaling may affect the animal's response to the environment.

**A broad-based stress response increases longevity**

A major goal of this study was to identify genes whose products directly influence ageing. We identified two prominent groups of functionally related genes that were candidates for direct effectors. The first group contained a wide variety of stress-response genes. In addition to mtl-1 and sod-3, we found that expression of the catalase genes ctl-1 and ctl-2, the glutathione-S-transferase gene gst-4, and the small heat-shock protein genes were all increased in animals with reduced daf-2 activity and decreased in animals with reduced daf-16 activity. We inhibited the activities of these genes with RNAi, and found that, in each case, the lifespans of daf-2 mutants were shortened, generally between 10–20% (Fig. 4c, d and Table 1). Because DAF-16 also functions in the wild type to extend lifespan, inhibiting these genes would be predicted to shorten wild-type lifespan as well. This was often the case, although the magnitude of the effect was smaller than in daf-2 (-) mutants (Supplementary Table 2s). Thus, each of these genes functions to promote longevity, probably by preventing or repairing oxidative and other forms of macromolecular damage.

---

An antimicrobial response lengthens lifespan

The second prominent set of potential lifespan effectors encoded antimicrobial proteins. Caenorhabditis elegans feeds on bacteria, and, at least under laboratory conditions, wild-type animals exhibit pharyngeal and intestinal bacterial packing as they age²⁵, and are ultimately killed by proliferating bacteria²⁵. daf-2 mutants display reduced bacterial packing when compared with wild-type nematodes of the same age²⁵. We found that several genes encoding antibacterial lysosomes were induced in daf-2 mutants, including two intestinally expressed genes, lys-7 (C02A12.4) and lys-8 (C17G10.5), which are also induced when C. elegans is infected with pathogenic Serratia marcescens³⁵. The saposin-like gene spp-1 (T07C4.4), which has demonstrated antibacterial activity³⁶, was also upregulated in daf-2 (-) animals. To test whether expression of these genes contributes to the longevity of daf-2 mutants, we reduced the activities of several using RNAi. We found that these treatments shortened lifespan of daf-2 mutants (Fig. 4e, f), indicating that these genes contribute to longevity.

Other daf-2/daf-16-regulated genes also influence lifespan

We found a number of other daf-2 / daf-16 -regulated genes with substantial effects on lifespan. For example, the vitellogenin (yolk protein/apolipoprotein-like) genes vit-2 and vit-5 were downregulated in daf-2 (-) animals and upregulated in daf-16 (-) animals, and we found that reducing their activities lengthened the lifespan of daf-2 (+) animals (Fig. 5b). Several proteases and metabolic genes were also class 2 genes, including an aminopeptidase, a carboxypeptidase, an amino-oxidase, an aminocyclase, and pep-2, an oligopeptide transporter, as well as several F-box/cullin/Skp proteins (including skr-8, skr-9 and pes-2) associated with ubiquitin-mediated protein degradation. Inhibition of several of these genes extended the lifespan of daf-2 (+) animals (Table 2). This suggests that daf-2 lifespan extension may involve turnover of specific proteins or metabolites. The glyoxylate cycle gene gei-7 encoding isocitrate lyase/malate synthase, which is upregulated in dauer³⁷ and hibernating mammals³⁸, was upregulated in daf-2 (-)

---

Figure 2 Class 1 genes are upregulated (red) with daf-2 RNAi treatment and in daf-2 pathway mutants, and downregulated (green) with daf-16 RNAi treatment, whereas class 2 genes are upregulated with daf-16 RNAi treatment and downregulated with daf-2 RNAi treatment and in daf-2 pathway mutants. Time points from the short and long time courses of daf-2 and daf-16 RNAi treatments and day 1 adult mutant comparisons are shown in hours along the top. (See Supplementary Materials for individual gene

NATURE | VOL 424 | 17 JULY 2003 | www.nature.com/nature © 2003 Nature Publishing Group 279

articles

animals. Inhibiting the function of this gene shortened the lifespan of *daf-2* (-) mutants substantially, while shortening wild type lifespan only slightly (D. Cristina and C.K., unpublished data). Thus this alternative metabolic pathway contributes to longevity. A large class of unknown genes containing a shared domain of unknown function (the DUF141 domain) was downregulated in

$daf-2$ mutants, and RNAi of these genes extended lifespan (Fig. 5c). One gene that is repressed in $daf-2$ mutants and induced in $daf-16$ mutants, C54G4.6, had a relatively large effect on lifespan (Fig. 5c). This gene shares homology with bacterial orfE/MAF inhibitor of septum formation proteins and with a human protein, ASMTL³⁹. Finally, several other class 2 genes that significantly extended lifespan shared no homology with known genes (Fig. 5d and Table 2; see also Supplementary Table 2s).

### A new potential regulatory sequence

To identify potential transcription-factor binding sites, we searched in an unbiased way for common sequence patterns in the upstream regulatory regions of genes in each cluster using two different algorithms. We used the ‘Mobydick’ algorithm⁴⁰ to identify short sequences (words) whose statistical distribution suggests that they are meaningful informational units. In this analysis we used sequences taken from a 1-kilobase (kb) region upstream of each gene in the cluster; words that are over-represented in the cluster are candidate transcription-factor binding sites. We also used another algorithm that searches exhaustively for oligonucleotides over-represented in each cluster⁴¹. We found that the sequence T(G/A)TTTAC, which has been shown to be bound by DAF-16 in vitro⁴², was over-represented, suggesting that our set of genes includes many direct DAF-16 targets. Notably, this canonical site was present not only in the promoters of class 1 (*daf-2*-induced) genes, but also in the promoters of class 2 (*daf-2*-downregulated) genes (Tables 1 and 2). Thus, DAF-16 may both directly repress and activate gene expression. We also found that a new sequence, CTTATCA, scored highly in both algorithms. Both sequences were present in various combinations in the promoters of both the class 1 and class 2 genes (Tables 1 and 2). The existence of this new site suggests that DAF-16 may regulate its target genes in combination with an additional, as yet unidentified, factor.

### Mechanisms that modulate the rate of ageing

Together these findings suggest that the regulation of ageing by the insulin/IGF-I pathway is achieved through a combination of global regulators, such as INS-7 and neuronal signalling components, and a wide variety of genes whose products may affect the ageing process directly. Several DAF-16 target genes that had significant effects on lifespan encoded new proteins, and it will be interesting to learn whether these genes act in unexpected ways to influence lifespan. In addition, many DAF-16 target genes encoded proteins predicted to protect cells from oxidative and other forms of stress. Thus our study provides strong support for the theory that genes that increase resistance to environmental stress contribute to longevity.

In addition, our findings revealed that the ability to ward off microbial infections contributes to the longevity of *C. elegans*, and that this ability is regulated by insulin/IGF-I signalling. Bacterial infections are a major cause of disease and death in elderly humans. Thus, it will be interesting to learn whether the human insulin or IGF-I systems regulate the susceptibility to bacterial infections by controlling the expression of antimicrobial genes.

It was particularly interesting to find that no single RNAi treatment, other than *daf-16* RNAi itself, completely suppressed the lifespan extension of *daf-2* mutants. This was true also when we used a mutant strain with increased RNAi sensitivity⁴³ (Tables 1 and 2; see also Supplementary Table 2s(m)). This result indicates that multiple effector genes, whose expression is coordinated by the DAF-2 pathway, probably act in a cumulative manner to influence ageing. Because by themselves most genes have a relatively small effect on lifespan, it would have been difficult to identify any particular one in a standard genetic screen. Thus this study demonstrates the power of functional microarray analysis for dissecting complex regulatory systems.

Longevity must have evolved not just once, but many times. Insect lifespans range from a few weeks to several years, and those of

Figure 3 INS-7 behaves as a DAF-2 agonist, and is part of a positive feedback loop predicted to amplify DAF-2 pathway activity. **a**, *ins-7* RNAi extends the lifespan of the *daf-2(+)* RNAi-sensitive⁴³ strain *rrf-3(pk1426)*. *ins-7* RNAi also extends the lifespan of *fer-15; fem-1* animals (Supplementary Table 2s). **b**, *ins-7* RNAi increases the fraction of *daf-2(e1370ts)* mutants that become dauer. Parents were fed *ins-7* RNAi bacteria at 20°C; their progeny were moved to 22.5°C as eggs and observed 72 h later. **c**, Model of *ins-7* feedback regulation: when DAF-2 is active, DAF-16 activity is inhibited and *ins-7* is expressed, allowing further DAF-2 activation. When DAF-2 activity is reduced, DAF-16 is activated and *ins-7* expression is inhibited. (The expression of three other insulin-like genes changed in our microarrays: *ins-18* and *ins-2* were upregulated and *ins-21* was slightly downregulated in *daf-2(-)* animals.)

Table 1 Class 1: Genes upregulated under daf-2(-) conditions

| Cosmid no. | Gene       | Brief description                                                                 | Per cent of vector control lifespan (experiment) | Canonical GTAAAt/cA | New CTTATCA |
|-----------|------------|----------------------------------------------------------------------------------|----------------------|--------------|---------------|
|           | daf-16     |                                                                                  | 43.3(b)*            |              |               |
| Y54G11A.5b| ctl-2      | Peroxisomal catalase                                                             | 54(a)*              | 1            | 3             |
| T10B9.1   | dod-1      | Cytochrome P450 family, low similarity to mouse cytochrome P450Cyp3a11                | 61.0(a)*            | 3            | 1             |
| T27E4.8   | hsp-16.1   | Member of the C.elegans hsp-16 family; identical hsp-16.11                           | 71.3(d)+            | 0            | 0             |
| C02A12.4  | lys-7      | Response to pathogenic bacteria; lysosome/similar to N-acetylmuramidase             | 72.6(a)*            | 2            | 1             |
| F28D1.3   | dod-2      | Thaumatin plant pathogenesis associated (PR) proteins, similar to F28D1.5          | 73.0(a)*            | 2            | 2             |
| F38E11.2  | hsp-12.6   | Hsp20/alpha crystalline family, similar to alpha-B crystalline                      | 75.8(a)+            | 3            | 2             |
| K11G9.6   | mtl-1      | Metallothionein-related cadmium-binding protein                                   | 75.8(a)+            | 2            | 3             |
| C05E4.9   | gei-7      | Malate synthase family/isocitrate lyase family                                    | 77.1(a)+            | 5            | 3             |
| C24B9.9   | dod-3      | Unknown protein                                                                  | 78.5(a)s            | 6            | 1             |
| F32A5.5   | dod-4      | Aquaporin AQP; major intrinsic protein (MIP) family of transmembrane channels       | 78.7(d)*            | 1            | 4             |
| T22G5.7   | dod-5      | Saposin type B                                                                   | 79.2(d)*            | 4            | 1             |
| F10D2.9   | fat-7      | Putative stearoyl-CoA delta-9 fatty acid desaturase/ polyunsaturated fatty acid biosynthesis | 79.8(a)s            | 3            | 1             |
| T20G5.7   | dod-6      | Meditrin-like ShK toxin                                                          | 80.0(a)+            | 1            | 2             |
| T27E4.9   | hsp-16.49  | Hsp20/alpha crystallin family, similar to alpha-B crystalline                       | 81.0(d)+            | 4            | 1             |
| C50B8.2   | bir-2      | Protein with two baculoviral inhibitor of apoptosis protein repeat (BIR) domains    | 81.1(c)*            | 5            | 1             |
| T27E4.2   | hsp-16.11  | Member of the C.elegans hsp-16 family; identical to hsp-16.11                       | 81.4(d)+            | 1            | 0             |
| T20G5.8   |            | Meditrin-like ShK toxin                                                          | 84.3(d)+            | 1            | 2             |
| T07C4.4   | spp-1      | Saposin; similar to bactericidal amoebapore, may act as an antibacterial agent       | 84.3(d)s            | 1            | 2             |
| K11D2.2   | dod-7      | ASAH acid ceramidase; choleylglycine hydrolase, cleaves C-N non-peptide bonds in linear amides | 85.4(c)+            | 1            | 3             |
| C06B3.4   | dod-8      | Estradiol 17b dh; short-chain dehydrogenase-reductase family oxidoreductases        | 87.6(c)s            | 4            | 1             |
| Y54G11A.6 | ctl-1      | Cytosolic catalase                                                                | 87.8(b)*            | 3            | 0             |
| C46F4.2   | dod-9      | Acyl-CoA synthetase; high similarity to long-chain fatty acid-CoA ligase 4          | 87.8(c)+            | 2            | 1             |
| F43D9.4   | sip-1      | Hsp20/alpha crystallin family, moderately similar to C.elegans HSO-16 involved in heat shock response | 88.4(c)s            | 3            | 1             |
| F11A5.12  | dod-10     | Short-chain dehydrogenase-reductase family, NAD-or NADP-dependent oxidoreductases   | 88.6(c)s            | 1            | 3             |
| C52E4.1   | gcp-1      | Cysteine protease expressed in the intestine                                      | 89.2(c)+            | 1            | 0             |
| K12G11.3  | dod-11     | High similarity to C.albicans Adh1p, an alcohol dehydrogenase                      | 89.6(a)s            | 3            | 2             |
| R12A1.4   | ges-1      | Carboxylesterase expressed in gut cells                                           | 89.6(c)s            | 1            | 3             |
| C55B7.4   | dod-12     | Short branched chain acyl-CoA dehydrogenase (human ACADSB)                        | 89.9(c)s            | 0            | 1             |
| H22K11.1  | asp-3      | Probable aspartyl protease and an orthologue of human cathepsin D                  | 90.3(c)s            | 3            | 0             |
| Y46H3A.3  | hsp-16.2   | Strong similarity to C.elegans HSP-16 heat shock protein, Hsp20/alpha crystallin family | 90.4(d)s            | 1            | 0             |
| K07C6.4   | dod-13     | Cytochrome P450 family, low similarity to cytochrome P450 subfamily 2C polypeptide 8 | 90.6(b)*            | 1            | 2             |
| R03E9.1   | mdl-1      | MAD family of putative transcription factors, interacts with C.elegans MAX-1       | 91.1(d)s            | 1            | 2             |
| C08A9.1   | sod-3      | Manganese superoxide dismutase                                                    | 92.6(b)s            | 6            | 2             |
| K10B3.8   | gpd-2      | Glyceraldehyde-3-phosphate dehydrogenase                                          | 92.9(c)s            | 4            | 0             |
| K07E3.3   | dao-3      | Tetrahydrofolate dehydrogenase/cyclohydrolase catalytic domain, NAD(P)-binding domain | 93.4(b)s            | 5            | 2             |
| T28B8.2   | ins-18     | Insulin-like protein of the type-beta subfamily; maybe a ligand for the DAF-2 receptor | 94.1(b)+            | 2            | 1             |
| K12G11.4  | dod-14     | High similarity to C.albicans Adh1p alcohol dehydrogenase; Zn alcohol dehydrogenase family | 95.0(b)s            | 4            | 1             |
| AC3.7     | dod-15     | UDP-glucoronosyl, UDP-glucosyl transferase domains                                | 95.1(c)s            | 4            | 2             |
| daf-2     |            |                                                                                  | 106.1(b)+           | -            | -             |
| B0213.15  | dod-16     | Cytochrome P450, oxidation of arachidonic acid to eicosanoids; (mouse Cyp2j5)         | 119.0(a)+           | 1            | 2             |

The table is a summary of data from selected class 1 genes. Animals were treated with RNAi of selected genes and lifespans were compared to those of animals treated with control vector RNAi; experiments are briefly described below. "dod" stands for “downstream of DAF-16”. (Detailed lifespan data are included in the Supplementary Information.) The number of canonical DAF-16 and new sequences in the 5kb upstream of each gene is also shown. All experiments were performed with $n \geq 60$ animals. (a), $daf-2(mu150)$, 25°C whole life; (b), $daf-2(mu150)$ shifted from 20°C to 25°C at L3; (c), $daf-2(mu150)$ shifted from 20°C to 25°C at L2; (d), $daf-2(mu150)$ shifted from 20°C to 25°C at L2; (e), $daf-2(e1370)$ shifted from 20°C to 25.5°C at L4; (j), $rrf-3(pk1426)$; $daf-2(e1370)$ at 20°C. *$P \leq 0.0001$; †$P \leq 0.001$; ‡$P \leq 0.005$; §$P \leq 0.01$; ¶$P \leq 0.05$; ||$P > 0.05$.

articles

![](image.png)

Figure 4 Lifespans of *daf-2* mutants fed dsRNA of class 1 genes. *daf-2(mu150)* mutants subjected to RNAi of metabolic and steroid and lipid synthesis genes (a, b) and oxidative stress genes (c, d). *daf-2(e1370)* (e) and *daf-2(mu150)* (f) mutants subjected to RNAi of antimicrobial genes. (Complete lifespan data are presented in Supplementary Table 2s.)

Table 2 Class 2: Genes downregulated under *daf-2(-)* conditions

| Cosmid no. | Gene     | Brief description                                                                 | Per cent of vector control lifespan | Canonical GTAAAt/cA | New CTTATCA |
|------------|----------|-----------------------------------------------------------------------------------|--------------------------------------|---------------------|--------------|
|            | *daf-2*  |                                                                                 | 130.5(a)†                           |                     |              |
| K10D11.1   | *dod-17* | DUF141 domain of unknown function, high similarity to uncharacterized *C. elegans* F55G11.8 | 133.7(c)*                           | 4                   | 3            |
| C07B5.5    | *nuc-1*  | Endonuclease with strong similarity to *H. sapiens* DNase II: DNA degradation during apoptosis | 130.4(c)*                           | 2                   | 0            |
| C54G4.6    | *dod-18* | Maf-like protein family, inhibitors of septum formation, low similarity to uncharacterized *S. pombe* Spac3g6.03cp | 129.2(a)*                           | 132.4(b)†           | 0            | 2            |
| ZK6.10     | *dod-19* | Protein of unknown function                                                       | 127.9(c)*                           | 1                   | 3            |
| B0024.6    | *gcy-6*  | Putative guanylyl cyclase expressed in the ASEL neuron                              | 126.1(c)*                           | 2                   | 2            |
| B0554.6    | *dod-20* | Protein of unknown function (DUF274) family, high similarity to uncharacterized *C. elegans* ZK6.11 | 123.0(c)*                           | 3                   | 5            |
| C32H11.10  | *dod-21* | DUF141 domain of unknown function, strong similarity to uncharacterized *C. elegans* C32H11.9 | 121.7(c)†                           | 2                   | 2            |
| C04F6.1    | *vit-5*  | Vitellogenin; 170 kDa yolk protein                                                | 121.5(a)†                           | 116.5(b)Ⅱ           | 109.7(c)§    | 1            | 1            |
| T08G5.10   | *mtl-2*  | Protein of unknown function, has high similarity to uncharacterized *C. elegans* MTL-1 | 120.2(c)†                           | 96.5(d)Ⅱ            | 1            | 1            |
| F55G11.5   | *dod-22* | DUF141 domain of unknown function, high similarity to uncharacterized *C. elegans* K10D11.2 | 118.1(c)¶                           | 3                   | 3            |
| F49E12.2   | *dod-23* | Protein of unknown function                                                       | 116.5(c)†                           | 101.1(d)Ⅱ           | 1            | 2            |
| T22G5.2    | *lbp-7*  | High similarity to *C. elegans* LBP-5 (locomotory behaviour) ipocalin and cytosolic fatty-acid binding | 114.4(c)¶                           | 101.4(d)Ⅱ           | 0            | 4            |
| K04E7.2    | *pep-2*  | Member of the proton-coupled oligopeptide transporter superfamily                    | 113.7(c)¶                           | 3                   | 0            |
| ZK1251.2   | *ins-7*  | Insulin-like protein of the type-beta subfamily                                    | 155.2(b)*                           | 133.3(c)*           | 0            | 2            |
| F56G4.2    | *pes-2*  | Unknown function, has very strong similarity to uncharacterized *C. elegans* F56G4.3 | 111.8(c)Ⅱ                           | 124.5(d)Ⅱ           | 3            | 3            |
| C08H9.5    | *old-1*  | Putative receptor tyrosine protein kinase; similar to human and *D. melanogaster* FGF receptor protein kinases | 111.6(c)¶                           | 109.6(d)Ⅱ           | 0            | 1            |
| C32H11.12  | *dod-24* | DUF141 domain of unknown function, high similarity to uncharacterized *C. elegans* C32H11.9 | 131.3(b)*                           | 124.4(c)*           | 2            | 2            |
| ZK896.8    | *gcy-18* | Guanylate cyclase catalytic domain; receptor family ligand binding and protein kinase domain | 125.5(b)§                           | 124.7(c)*           | 3            | 1            |
| C42D8.2    | *vit-2*  | Vitellogenin structural genes (yolk protein genes)                                 | 121.0(b)¶                           | 124.4(c)†           | 1            | 1            | 2            |
|            | *daf-16* |                                                                                 | 79.3(c)*                            |                     |              |

The Table is the same as Table 1, except for class 2 genes (see Table 1 legend). (f), *fer-15(b26)*; *fem-1(hcl7)* at 25°C whole life; (g), *rrf-3(pk1426)* at 20°C whole life; (h), *rrf-3(pk1426)* shifted to 25°C at L2-L4, back to 20°C; (i), *rrf-3(pk1426)* shifted to 25°C at L2-L4, back to 20°C.

*P ≤ 0.0001; †, P ≤ 0.001; ‡, P ≤ 0.005; §, P ≤ 0.01; ¶, P ≤ 0.05; ǁ, P > 0.05.

Microarray hybridizations

Standard techniques were used to obtain RNA (Trizol), messenger RNA (Oligotex, Qiagen), complementary DNA (reverse transcription) and Cy-dye-labelled cDNA⁴⁹; arrays were hybridized for 18 h at 63 °C, washed and scanned. One-half of each time course sample was added to a pool, and every Cy5-labelled sample was compared to this Cy3-labelled mixed reference. Mutant comparisons were done both directly and in a pooled comparison.

Significance analysis

After array normalization (see Supplementary Information), SAM analysis²⁴ was performed on data from nine mutant arrays (one-class response) to identify genes with small but consistent changes. In this set of arrays at a δ-value of 1.47, 70 upregulated and 100 downregulated genes were found to be significant (*q*-value = 0.0011194) with 0.6207 median false significant genes (Supplementary Table 1s).

Correlation coefficient analysis

We calculated a vector comprising the entirety of log ratio comparisons for all the genes with a valid signal at a single time point, to describe each array as a single value. We compared each array in the two time courses to all other arrays in that time course, and the Pearson correlation of the log base-two of these expression ratios was calculated. Five arrays from the set of 60 time points did not correlate with neighbouring time points, and were eliminated.

Cluster analysis

After normalization (see Supplementary Information), log transformation and quality confirmation through correlation coefficient analysis, data from 55 RNAi arrays and 5 mutant arrays were imported into Gene Cluster²³ for fold-cutoff analysis and hierarchical clustering. Genes were filtered to obtain only those that were present in 80% of the 60 arrays in the data set and which met a max–min of 4-fold, 8-fold or 16-fold criterion. A total of 7,380 genes met a 4-fold cutoff, 2,734 genes met an 8-fold cutoff and 1,280 genes met a 16-fold cutoff over the entire set of 55 RNAi arrays and five mutant arrays. The filtered set was hierarchically clustered, a self-organized map was constructed with 300,000 iterations, and the gene set was displayed in TreeView²³.

Upstream sequence analysis

The sequence 1 kb upstream of the translation start site of each open reading frame was assembled and subjected to two algorithms to search for potential transcription-factor binding sites. Exact repeats of length 14 or longer were removed before building the Mobydick⁴⁰ dictionary; words were screened by contrasting the frequency of occurrences in the cluster to that in the upstream regions of all the genes in the genome. We also searched for oligonucleotides over-represented in the cluster⁴¹. The occurrence of the identified sequences in the 5 kb upstream of each gene was then determined⁵⁰.

Survival analyses

Our lifespan analysis focused on a subset of genes whose expression profiles changed in opposite ways under *daf-2* (-) and *daf-16* (-) conditions. Genes were prioritized by fold expression change (Fig. 2) and by interesting gene function. The bacteria for 58 genes (Tables 1 and 2) were selected from the RNAi library⁴⁸. A total of 60–70 nematodes were used per experiment as described previously⁵,¹⁶. The first day of adulthood was used as *t* = 0, and the log-rank (Mantel–Cox) method was used to test the null hypothesis (StatView 5.01, SAS Software). *fer-15* (*b26*); *fem-1* (*hc17*) animals were grown at 25 °C on RNAi bacteria and lifespans were measured at this temperature unless otherwise indicated. *daf-2* (*mu150*) nematodes were measured at 25 °C in one trial, and in subsequent tests were raised at 20 °C then shifted to 25 °C at L3. *rrf-3* (*pk1426*)⁴³ mutants were treated at 20 °C in one experiment; in subsequent experiments, the nematodes were shifted to 25 °C at L2 through young adulthood to induce sterility, and adult lifespan was measured at 20 °C. *rrf-3* (*pk1426*); *daf-2* (*e1370*) lifespan tests were done at 20 °C.

Dauer tests

*daf-2* (*e1370*) nematodes were grown on RNAi bacteria at 20 °C, F₁ eggs were incubated at 22.5 °C, and animals were scored for dauer arrest 72 h later.

Received 17 December 2002; accepted 7 May 2003; doi:10.1038/nature01789.  
Published online 29 June 2003.

1. Tatar, M., Bartke, A. & Antebi, A. The endocrine regulation of aging by insulin-like signals. *Science* **299**, 1346–1351 (2003).
2. Holzenberger, M. *et al.* IGF-1 receptor regulates lifespan and resistance to oxidative stress in mice. *Nature* **421**, 182–187 (2003).
3. Bluher, M., Kahn, B. B. & Kahn, C. R. Extended longevity in mice lacking the insulin receptor in adipose tissue. *Science* **299**, 572–574 (2003).
4. Kimura, K. D., Tissenbaum, H. A., Liu, Y. & Ruvkun, G. *daf-2*, an insulin receptor-like gene that regulates longevity and diapause in *Caenorhabditis elegans*. *Science* **277**, 942–946 (1997).
5. Kenyon, C., Chang, J., Gensch, E., Rudner, A. & Tabtiang, R. A *C. elegans* mutant that lives twice as long as wild type. *Nature* **366**, 461–464 (1993).
6. Larsen, P. L., Albert, P. S. & Riddle, D. L. Genes that regulate both development and longevity in *Caenorhabditis elegans*. *Genetics* **139**, 1567–1583 (1995).
7. Gems, D. *et al.* Two pleiotropic classes of *daf-2* mutation affect larval arrest, adult behavior, reproduction and longevity in *Caenorhabditis elegans*. *Genetics* **150**, 129–155 (1998).
8. Riddle, D. L. & Albert, P. S. in *C. elegans II* (eds Riddle, D. L., Blumenthal, T., Meyer, B. J. & Priess, J. R.) 739–768 (Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, 1997).
9. Guarente, L. & Kenyon, C. Genetic pathways that regulate ageing in model organisms. *Nature* **408**, 255–262 (2000).

articles

10. Gems, D. & Partridge, L. Insulin/IGF signalling and ageing: seeing the bigger picture. *Curr. Opin. Genet. Dev.* **11**, 287–292 (2001).

11. Dillin, A., Crawford, D. K. & Kenyon, C. Timing requirements for insulin/IGF-1 signaling in *C. elegans*. *Science* **298**, 830–834 (2002).

12. Wolkow, C. A., Kimura, K. D., Lee, M. S. & Ruvkun, G. Regulation of *C. elegans* life-span by insulinlike signaling in the nervous system. *Science* **290**, 147–150 (2000).

13. Tissenbaum, H. A. & Ruvkun, G. An insulin-like signaling pathway affects both longevity and reproduction in *Caenorhabditis elegans*. *Genetics* **148**, 703–717 (1998).

14. Lin, K., Dorman, J. B., Rodan, A. & Kenyon C. *daf-16*: An HNF-3/forkhead family member that can function to double the life-span of *Caenorhabditis elegans*. *Science* **278**, 1319–1322 (1997).

15. Ogg, S. *et al.* The Fork head transcription factor DAF-16 transduces insulin-like metabolic and longevity signals in *C. elegans*. *Nature* **389**, 994–999 (1997).

16. Apfeld, J. & Kenyon, C. Cell nonautonomy of *C. elegans daf-2* function in the regulation of diapause and life span. *Cell* **95**, 199–210 (1998).

17. Larsen, P. L. Aging and resistance to oxidative damage in *Caenorhabditis elegans*. *Proc. Natl Acad. Sci. USA* **90**, 8905–8909 (1993).

18. Lithgow, G. J., White, T. M., Hinerfeld, D. A. & Johnson, T. E. Thermotolerance of a long-lived mutant of *Caenorhabditis elegans*. *J. Gerontol.* **49**, B270–B276 (1994).

19. Orr, W. C. & Sohal, R. S. Extension of life-span by overexpression of superoxide dismutase and catalase in *Drosophila melanogaster*. *Science* **263**, 1128–1130 (1994).

20. Sun, J., Folk, D., Bradley, T. J. & Tower, J. Induced overexpression of mitochondrial Mn-superoxide dismutase extends the life span of adult *Drosophila melanogaster*. *Genetics* **161**, 661–672 (2002).

21. Longo, V. D., Liou, L. L., Valentine, J. S. & Gralla, E. B. Mitochondrial superoxide decreases yeast survival in stationary phase. *Arch. Biochem. Biophys.* **365**, 131–142 (1999).

22. Taub, J. *et al.* retraction: A cytosolic catalase is needed to extend adult lifespan in *C. elegans daf-C* and *clk-1* mutants. *Nature* **421**, 764 (2003).

23. Eisen, M. B., Spellman, P. T., Brown, P. O. & Botstein, D. Cluster analysis and display of genome-wide expression patterns. *Proc. Natl Acad. Sci. USA* **95**, 14863–14868 (1998).

24. Tusher, V. G., Tibshirani, R. & Chu, G. Significance analysis of microarrays applied to the ionizing radiation response. *Proc. Natl Acad. Sci. USA* **98**, 5116–5121 (2001).

25. Garigan, D. *et al.* Genetic analysis of tissue aging in *Caenorhabditis elegans*. A role for heat-shock factor and bacterial proliferation. *Genetics* **161**, 1101–1112 (2002).

26. Herndon, L. A. & Driscoll, M. Contributions of cell death to aging in *C. elegans*. *Results Probl. Cell Differ.* **29**, 113–129 (2000).

27. Barsyte, D., Lovejoy, D. A. & Lithgow, G. J. Longevity and heavy metal resistance in *daf-2* and *age-1* long-lived mutants of *Caenorhabditis elegans*. *FASEB J.* **15**, 627–634 (2001).

28. Honda, Y. & Honda, S. The *daf-2* gene network for longevity regulates oxidative stress resistance and Mn-superoxide dismutase gene expression in *Caenorhabditis elegans*. *FASEB J.* **13**, 1385–1393 (1999).

29. Gregoire, F. M., Chomiki, N., Kachinskas, D. & Warden, C. H. Cloning and developmental regulation of a novel member of the insulin-like gene family in *Caenorhabditis elegans*. *Biochem. Biophys. Res. Commun.* **249**, 385–390 (1998).

30. Pierce, S. B. *et al.* Regulation of DAF-2 receptor signaling by human insulin and *ins-1*, a member of the unusually large and diverse *C. elegans* insulin gene family. *Genes Dev.* **15**, 672–686 (2001).

31. Kawano, T. *et al.* Molecular cloning and characterization of a new insulin/IGF-like peptide of the nematode *Caenorhabditis elegans*. *Biochem. Biophys. Res. Commun.* **273**, 431–436 (2000).

32. Li, W., Kennedy, S. G. & Ruvkun, G. *daf-28* encodes a *C. elegans* insulin superfamily member that is regulated by environmental cues and acts in the DAF-2 signaling pathway. *Genes Dev.* **17**, 844–858 (2003).

33. Ookuma, S., Fukuda, M. & Nishida, E. Identification of a DAF-16 transcriptional target gene, *scl-1*, that regulates longevity and stress resistance in *Caenorhabditis elegans*. *Curr. Biol.* **13**, 427–431 (2003).

34. Yu, S., Avery, L., Baude, E. & Garbers, D. L. Guanylyl cyclase expression in specific sensory neurons: a new family of chemosensory receptors. *Proc. Natl Acad. Sci. USA* **94**, 3384–3387 (1997).

35. Mallo, G. *et al.* Inducible antibacterial defense system in *C. elegans*. *Curr. Biol.* **12**, 1209–1214 (2002).

36. Banyai, L. & Patthy, L. Amoebapore homologs of *Caenorhabditis elegans*. *Biochim. Biophys. Acta* **1429**, 259–264 (1998).

37. Wang, J. & Kim, S. K. Global analysis of dauer gene expression in *Caenorhabditis elegans*. *Development* **130**, 1621–1634 (2003).

38. Davis, W. L., Goodman, D. B., Crawford, L. A., Cooper, O. J. & Matthews, J. L. Hibernation activates glyoxylate cycle and gluconeogenesis in black bear brown adipose tissue. *Biochim. Biophys. Acta* **1051**, 276–278 (1990).

39. Ried, K., Rao, E., Schiebel, K. & Rappold, G. A. Gene duplications as a recurrent theme in the evolution of the human pseudoautosomal region 1: isolation of the gene ASMTL. *Hum. Mol. Genet.* **7**, 1771–1778 (1998).

40. Bussemaker, H. J., Li, H. & Siggia, E. D. Building a dictionary for genomes: identification of presumptive regulatory sites by statistical analysis. *Proc. Natl Acad. Sci. USA* **97**, 10096–10100 (2000).

41. van Helden, J., Andre, B. & Collado-Vides, J. Extracting regulatory sites from the upstream region of yeast genes by computational analysis of oligonucleotide frequencies. *J. Mol. Biol.* **281**, 827–842 (1998).

42. Furuyama, T., Nakazawa, T., Nakano, I. & Mori, N. Identification of the differential distribution patterns of mRNAs and consensus binding sequences for mouse DAF-16 homologues. *Biochem. J.* **349**, 629–634 (2000).

43. Sijen, T. *et al.* On the role of RNA amplification in dsRNA-triggered gene silencing. *Cell* **107**, 465–476 (2001).

44. Kirkwood, T. B. & Austad, S. N. Why do we age? *Nature* **408**, 233–238 (2000).

45. Clancy, D. J. *et al.* Extension of life-span by loss of CHICO, a *Drosophila* insulin receptor substrate protein. *Science* **292**, 104–106 (2001).

46. DeRisi, J. L., Iyer, V. R. & Brown, P. O. Exploring the metabolic and genetic control of gene expression on a genomic scale. *Science* **278**, 680–686 (1997).

47. Lin, K., Hsin, H., Libina, N. & Kenyon, C. Regulation of the *Caenorhabditis elegans* longevity protein DAF-16 by insulin/IGF-1 and germline signaling. *Nature Genet.* **28**, 139–145 (2001).

48. Fraser, A. G. *et al.* Functional genomic analysis of *C. elegans* chromosome I by systematic RNA interference. *Nature* **408**, 325–330 (2000).

49. DeRisi, J. *et al.* Genome microarray analysis of transcriptional activation in multidrug resistance yeast mutants. *FEBS Lett.* **470**, 156–160 (2000).

50. van Helden, J., Andre, B. & Collado-Vides, J. A web site for the computational analysis of yeast regulatory sequences. *Yeast* **16**, 177–187 (2000).

51. Lee, S. S., Kennedy, S., Tolonen, A. C. & Ruvkun, G. DAF-16 target genes that control *C. elegans* life-span and metabolism. *Science* **300**, 644–647 (2003).

52. McElwee, J., Bubb, K. & Thomas, J. H. Transcriptional outputs of the *Caenorhabditis elegans* forkhead protein DAF-16. *Aging Cell* **1**, 111–121 (2003).

Supplementary Information accompanies the paper on [www.nature.com/nature](http://www.nature.com/nature).

**Acknowledgements** We thank all the members of the Kenyon laboratory, as well as Z. Gitai, for critical review and discussion of this work, and A. Dillin, J. Lehrer-Graiver, D. Cristina, B. Albinder, and V. Tenberg for assistance. We also thank J. DeRisi and the DeRisi laboratory for assistance and advice on the design and use of microarrays, as well as T. Kirkwood for discussions about evolution. S.A.M., from the laboratory of C.I.B., participated in the statistical analysis and the whole-transcriptome analysis; A.F., R.S.K. and J.A. contributed the RNAi clones; and H.L. participated in the promoter analysis. C.T.M. is a Bristol-Myers Squibb Fellow of the Life Sciences Research Foundation. This work was supported by grants from the NIA and the Ellison Foundation to C.K.

**Competing interests statement** The authors declare that they have no competing financial interests.

**Correspondence** and requests for materials should be addressed to C.K. (ckenyon@biochem.ucsf.edu). The data of the microarray experiments for *daf-2* mutants, *daf-2* RNAi (0–48 h) and *daf-2* RNAi (0–192 h) are deposited in ArrayExpress under accession numbers E-MEXP-7, E-MEXP-8 and E-MEXP-9, respectively.
