
Molecular Microbiology (2003) 49(3), 807–821

doi:10.1046/j.1365-2958.2003.03599.x

# Two-component system VraSR positively modulates the regulation of cell-wall biosynthesis pathway in Staphylococcus aureus

**Makoto Kuroda**,¹* Hiroko Kuroda,¹ Taku Oshima,² Fumihiko Takeuchi,³⁴ Hirotada Mori² and Keiichi Hiramatsu¹

¹ Bacteriology, Juntendo University, 2-1-1 Hongo, Bunkyo-ku, Tokyo 113–8421, Japan.

² Research and Education Center for Genetic Information, Nara Institute of Science and Technology, Ikoma 630–0101, Japan.

³ Research Institute, International Medical Center of Japan, 1-21-1 Toyama, Shinjuku-ku, Tokyo 162–8655, Japan.

⁴ The Organization for Pharmaceutical Safety and Research, 3-3-2 Kasumigaseki, Chiyoda-ku, Tokyo 100–0013, Japan.

## Summary

DNA microarray covering the whole genome of *Staphylococcus aureus* strain N315 was prepared to investigate transcription profiles. The microarray analyses revealed that vancomycin induces transcription of 139 genes. Forty-six genes among them failed to be induced in the *vraSR* null mutant KVR. Part of the genes regulated by VraSR system is associated with cell-wall biosynthesis, such as PBP2, SgtB and MurZ. Other cell-wall synthesis inhibitors also induced VraSR, suggesting that the sensor kinase VraS responds to the damage of cell-wall structure or inhibition of cell-wall biosynthesis. Additionally, the *vraSR* null mutants derived from hetero- and homo-methicillin-resistant *S.* aureus showed significant decrease of resistance against teicoplanin, β-lactam, bacitracin and fosfomycin but not of D-cycloserine and levofloxacin. The observation strongly indicates that VraSR constitutes a positive regulator of cell-wall peptidoglycan synthesis, and that is deeply involved in the expression of β-lactam and glycopeptide resistance in *S.* aureus.

## Introduction

We previously determined the whole genome sequence of *Staphylococcus aureus* strains N315 and Mu50 (Kuroda et al., 2001). The *S.* aureus genome contains 16 pairs of two-component system encoding histidine kinase sensor and response regulator, and additional one pair on a genomic island staphylococcal cassette chromosome *mec* (SCCmec) (Ito et al., 1999; Kuroda et al., 2001). Several two-component systems in *S.* aureus are well characterized as global regulatory systems involved in the induced production of toxins and exoproteins, regulation of biofilm formation, cell permeability and autolysis (Brunskill and Bayles, 1996; Giraudo et al., 1997; Martin et al., 1999; Novick and Muir, 1999; Fournier and Hooper, 2000; Yarwood et al., 2001). Accessory gene regulator (*agr*), *arlRS*, *saeRS* and *srrAB* two-component systems have been reported to regulate the expression of virulence factors in mutually different manners (Giraudo et al., 1997; Novick and Muir, 1999; Fournier and Hooper, 2000; Yarwood et al., 2001). The *agr* locus is well known as virulence-associated sensor-regulator system in which the *agrD* encodes autoinducing peptide pheromone (AIP) that activates AgrC-AgrA quorum-sensing sensor-regulator system (Novick and Muir, 1999). Dunman et al. (2001) identified a spectrum of *S.* aureus genes regulated by *agr* and *sarA* using the combination of *agr* and *sarA* mutant strains and DNA chip technology. The two-component system orthologous to *yycFG* (*Bacillus subtilis*) is reported as an essential system involved in the regulation of cell permeability. Another two-component system *lytSR* controls the rate of autolysis by regulating the intrinsic murein hydrolase activity (Brunskill and Bayles, 1996; Martin et al., 1999).

In contrast, 11 *S.* aureus two-component systems still remain to be explored for their physiological roles. The orthologues of these systems are widely found in Gram-positive bacteria, suggesting that they play important roles in the cell physiology.

Previously, we showed that the transcription of the two-component system *vraSR* (standing for vancomycin-resistance associated sensor/regulator) is upregulated in hetero-vancomycin resistant *S.* aureus (hetero-VRSA) strain Mu3 and VRSA strain Mu50 as compared to

Accepted 18 April, 2003. *For correspondence.* E-mail mkuroda@med.juntendo.ac.jp; Tel. (+81) 3 5802 1040; Fax (+81) 3 5684 7830.

© 2003 Blackwell Publishing Ltd

808 M. Kuroda et al.

vancomycin-susceptible S. aureus (VSSA) strain Mu50ω by using cDNA differential hybridization method (Kuroda et al., 2000). We showed as well that the experimental overexpression of VraR response regulator in S. aureus strain N315 reduces vancomycin susceptibility of the strain (Kuroda et al., 2000).

The vraS and vraR genes are homologous to yvqE and yvqC genes of B. subtilis respectively. Kobayashi et al. (2001) reported that YvqE-YvqC might be cross-talking with other two-component systems as shown by comprehensive DNA microarray analysis of B. subtilis gene expression under the overexpression of YvqC response regulator. However, the biological role of YvqE-YvqC system and the nature of its physiological signal input have remained unknown.

In this study, we demonstrate that the VraSR system is induced by exposure of S. aureus cell to the cell-wall-affecting antibiotics such as glycopeptides, β-lactams, bacitracin and D-cycloserine. In addition, the vraSR knockout leads to significant increase in the susceptibility of the S. aureus cell to the cell-wall-synthesis inhibitor antibiotics. We finally report the spectrum of S. aureus gene expression under the VraSR regulation.

---

### Results

#### VraSR two-component system responds to inhibition of cell-wall synthesis

Previously, we reported the upregulation of vraSR transcription in VRSA strain Mu50 as compared to VSSA strain Mu50ω (Kuroda et al., 2000). We explored the involvement of the VraSR-mediated regulation in expression of vancomycin resistance primarily by evaluating the effect of various antibiotics on the induction of vraSR, and complementary by constructing the vraSR null mutants with allelic replacement.

Northern blot analyses showed that the 3.0 kb of transcript was clearly induced after 10 min of the exposure to 10-fold minimum inhibitory concentration (MIC) of vancomycin (final concentration 10 μg ml⁻¹) with either vraS- or vraR-specific probes (Fig. 1A). The vraS and vraR genes would be co-transcribed, and the length of the transcript was about 3.0 kb corresponding to that encompassing the predicted open reading frame (ORF) (SA1703) and vraR (SA1700). Likewise, other inhibitors of cell-wall synthesis (teicoplanin, ceftizoxime, imipenem, bacitracin, D-cycloserine) induced the vraS transcription, whereas levofloxacin, a new quinolone antibiotic inhibiting the function of DNA gyrase did not (Fig. 1A). Opposing, general stress such as heat, high osmolarity and pH shift did not induce the transcription of vraS gene (Fig. 1B). We have observed that the transcript of vraS and vraR has reached its maximum expression more or less 10 min after the

---

**Fig. 1. Northern blot analysis. Five μg of total RNA was loaded per lane, and the transcripts were detected with the gene-specific DNA probe shown in the left side of Northern blot results. The size of detected bands is shown in the right side of them. Electrophoresis image of 16S rRNA was shown as an internal control.**

**A.** Induction specificity of the vraS, vraR, pbp2, sgtB, saeS and coa genes by the exposure to antibiotics. Mid-log phase culture of S. aureus N315 strain was exposed to 10-fold MIC of antibiotics for 10 min, and harvested to prepare the total RNA. MICs of individual antibiotics were determined on BHI-agar plates. Lanes: – (control before exposure to antibiotics), VCM (10 μg ml⁻¹ vancomycin), TEIC (10 μg ml⁻¹ teicoplanin), CZX (40 μg ml⁻¹ ceftizoxime), IPM (1 μg ml⁻¹ imipenem), BC (320 μg ml⁻¹ bacitracin), CS (640 μg ml⁻¹ D-cycloserine) and LVFX (2.5 μg ml⁻¹ levofloxacin).

**B.** Mid-log phase culture of S. aureus N315 strain was exposed to general stresses for 10 min. Lanes: – (control), VCM (10 μg ml⁻¹ vancomycin), NaCl (2 M sodium chloride), Heat (shifted from 37°C to 46°C), pH 5 (shifted from pH 7.4–5.0 by addition of 1 N hydrochloride) and pH 9 (shifted from pH 7.4–9.0 by addition of 1 N sodium hydroxide).

vancomycin treatment (data not shown). Therefore, we consider that VraSR probably induced as primary response by inhibition of cell-wall biosynthesis.

By using the penicillin-binding protein 2 (PBP2) as gene-specific probe, the induction of transcripts of 2.9 kb and 2.3 kb in size was detected by the exposure to inhibitors of cell-wall synthesis (Fig. 1A). It is known that the pbp2 gene is co-transcribed with recU located upstream of pbp2, and regulated by two vegetative sigmaA-type promoters upstream of recU and pbp2 respectively (Pinho et al., 1998). Both the transcripts were significantly

© 2003 Blackwell Publishing Ltd, *Molecular Microbiology*, **49**, 807–821

DNA microarray analyses reveal the regulatory network via VraSR two-component system 809

increased after the treatment with vancomycin and other cell-wall synthesis inhibitors but not with levofloxacin. The transcript of *sgtB* gene encoding putative monofunctional glycosyltransferase was 1.0 kb in size that corresponded to the length of the *sgtB* gene alone, which was induced together with those of *vraS* and *pbp2* (Fig. 1A).

At the same time the transcripts of two-component system *saeRS* and coagulase gene *coa* found to be induced by vancomycin with both microarray and Northern analysis as described below (Fig. 1A). They were also induced by teicoplanin, ceftizoxime and imipenem, though the exposure to bacitracin or D-cycloserine rather suppressed the transcripts than induced them (Fig. 1A).

### Characterization of the vraSR null mutants

The *vraSR* null mutant KVR derived from N315 strain was constructed by chloramphenicol-positive/tetracycline-negative selection with allelic replacement by using flanking sequence of the *vraSR* genes. The growth of KVR was slightly retarded after mid-exponential phase (*OD*<sub>660</sub> = 1.0), but otherwise we did not observe any remarkable difference from the parental strain N315 in the growth with brain heart infusion (BHI)-broth (data not shown).

Northern blot demonstrated that KVR lost the induction of *pbp2* and *sgtB* by vancomycin exposure, and the transcription of those genes were substantially decreased in KVR with drug-free condition (Fig. 2A: KVR). It seems that a basal level of uncharacterized signal input must exist through VraSR system during normal cell multiplication. The induction of *pbp2* and *sgtB* genes was completely recovered by introducing the *vraSR* genes on the plasmid (Fig. 2A: KVR-SR). Nevertheless, the introduction of the *vraS* gene alone did not complement the induction (Fig. 2A: KVR-S).

Besides, the *saeRS* null mutant KSA lost the transcription of coagulase gene *coa*, but not that of the *vraS*, *pbp2*, and *sgtB* genes, suggesting that those inductions under VraSR control were not affected by loss of SaeRS-mediated signal transduction (Fig. 2A: KSA). On the contrary, the *vraSR* mutant did not affect the induction of the *saeS* and *coa* genes (Fig. 2A: KVR).

Table 1 displays the antibiotic susceptibilities in MICs of tested *S.* aureus strains including series of parental strains and the null mutants in this study. Knock out of *vraSR* gene caused an increasing susceptibility to vancomycin, teicoplanin, oxacillin, ceftizoxime, imipenem, bacitracin and fosfomycin, but not to D-cycloserine and levofloxacin. In particular, MIC of teicoplanin in KVR drastically reduced from 0.5 to 0.0625 as compared to parental N315 than that of vancomycin from 0.5 to 0.25, although both are glycopeptide antibiotics. Similarly, the MIC among tested β-lactams in KVR also decreased from 4 to

© 2003 Blackwell Publishing Ltd, *Molecular Microbiology*, **49**, 807–821

![Figure 2](#fig-2)

**Fig. 2.** Northern blot analysis of the *vraSR* null mutants.

A. The loss of induction of the *pbp2* and *sgtB* genes in the *vraSR* null mutant KVR. Mid-log phase culture of strain was exposed to 10 μg ml<sup>-1</sup> of vancomycin (VCM) before (-) and after 10 min (+), and harvested to prepare total RNA. Strains: N315, pre-MRSA; KVR, the *vraSR* null mutant from N315; KVR-Y, KVR harbouring vector pYT3; KVR-S, KVR complemented with *vraS* gene; KVR-SR, KVR complemented with *vraS* and *vraR* genes; KSA, the *saeRS* null mutant from N315.

B. Constitutive gene expression of *vraS*, *pbp2* and *sgtB* in homo-MRSA ΔIPH14. Mid-log phase culture of strain was harvested to prepare total RNA. Strains: N315, pre-MRSA; KVR, the Δ*vraSR* from N315; ΔIP, hetero-MRSA; ΔIP-KV, Δ*vraSR* from ΔIP; ΔIPH5, homo-MRSA derived from ΔIP; H5-KV, Δ*vraSR* from ΔIPH5; ΔIPH14, homo-MRSA derived from ΔIP; H14-KV, Δ*vraSR* from ΔIPH14; KSA, Δ*saeRS* from N315.

16-fold. Interestingly, the MIC of ceftizoxime having more potential inhibition against PBP2 among tested cephalosporins (Chambers and Sachdeva, 1990; Okonogi, 1990) shows more susceptibility in KVR than N315ex that lacks *mecA* gene. The reduced-resistance of tested antibiotics against KVR was not complemented by introduction of the *vraS* gene on plasmid pNVS. However, introduction of both *vraS* and *vraR* could almost completely complement the resistance levels of N315.

ΔIP is a heterogeneous type β-lactam resistant strain resulted from overexpression of *mecA* gene encoding penicillin-binding protein 2′ (PBP2′) due to disruption of MecI transcription repressor (Fig. 2B: ΔIP). ΔIPH5 and ΔIPH14 strains were isolated with selection of 8 μg ml<sup>-1</sup> of

Table 1. Antibiotics susceptibility profiles of Staphylococcus aureus strains in this study.

| Strains   | VCM | TEIC    | MPIPC | CEX  | CZX   | IPM     | BC | CS | FOM | LVFX  |
|-----------|-----|---------|-------|------|-------|---------|----|----|------|-------|
| ATCC 25923 | 1   | 1       | 0.25  | 1    | 0.5   | <0.03125 | 16 | 32 | 16   | 0.125 |
| N315      | 0.5 | 0.5     | 8     | 64   | 4     | 0.5     | 16 | 32 | 8    | 0.125 |
| N315ex    | 0.5 | 0.5     | 0.25  | 2    | 0.5   | 0.0313  | 16 | 32 | 8    | 0.125 |
| KVR       | 0.25| 0.0625  | 1     | 16   | 0.25  | 0.125   | 2  | 32 | 2    | 0.125 |
| KVR-Y     | 0.25| 0.0625  | 1     | 16   | 0.25  | 0.125   | 2  | 32 | 2    | 0.125 |
| KVR-S     | 0.25| 0.0625  | 1     | 16   | 0.25  | 0.125   | 2  | 32 | 2    | 0.125 |
| KVR-SR    | 1   | 0.5     | 8     | 64   | 4     | 0.5     | 8  | 32 | 8    | 0.125 |
| ΔIP       | 0.5 | 0.5     | 32    | 256  | 1024  | 8       | 16 | 32 | 8    | 0.125 |
| ΔIP-KV    | 0.25| 0.0625  | 8     | 64   | 8     | 0.5     | 2  | 32 | 16   | 0.125 |
| ΔIPH5     | 0.5 | 0.5     | 512   | 512  | 2048  | 128     | 16 | 16 | 8    | 0.125 |
| H5-KV     | 0.25| 0.0625  | 128   | 256  | 2048  | 32      | 2  | 32 | 4    | 0.125 |
| ΔIPH14    | 1   | 4       | 512   | 512  | 2048  | 128     | 16 | 32 | 64   | 0.125 |
| H14-KV    | 0.25| 0.0625  | 8     | 64   | 8     | 0.5     | 1  | 16 | 2    | 0.125 |
| KSA       | 0.5 | 0.5     | 8     | 64   | 4     | 0.5     | 16 | 32 | 8    | 0.125 |

a. MIC, minimum inhibitory concentration.
Abbreviations: VCM, vancomycin; TEIC, teicoplanin; MPIPC, oxacillin; CEX, cefalexin; CZX, ceftizoxime; IPM, imipenem; BC, bacitacin; CS, D-cycloserine; FOM, fosfomycin; LVFX, levofloxacin

imipenem, and we regard those as homogenous type β-lactam resistant strains with considerable increasing MIC against oxacillin (512) and imipenem (128) (Table 1). We constructed the vraSR null mutants ΔIP-KV, H5-KV and H14-KV from ΔIP, ΔIPH5 and ΔIPH14 by phage transduction respectively. Those transductants still remain the expression of mecA gene, suggesting that VraSR might not be involved in regulation of the mecA gene (Fig. 2B). The ΔIP-KV and H14-KV showed significant reduced-resistance against teicoplanin as shown in KVR, and also against β-lactams as shown in N315 (Table 1). The transcripts of vraS, pbp2 and sgtB were overexpressed in ΔIPH14 in spite of the drug-free cultivation (Fig. 2B: ΔIPH14). Those constitutive expressions were reduced to the level of transcriptions shown in KVR (Fig. 2B: H14-KV). It suggests that highly activation of VraSR system might mainly contribute to the resistant phenotype of ΔIPH14. Interestingly, H5-KV still remains some degree of resistance against β-lactams even in loss of the VraSR function (Table 1). Unlike the ΔIPH14 strain, in ΔIPH5 we did not observe overexpression of vraS, pbp2, and sgtB (Fig. 2B: ΔIPH5).

Identification of the genes under VraSR-mediated transcription

To characterize the transcription control mediated by the VraSR two-component system, DNA microarray experiment was performed using the parental strain N315 and the vraSR null mutant KVR. Induction ratios of the vraS, pbp2, sgtB, saeS, and coa genes by exposure to vancomycin obtained by both Northern blot (Fig. 1A) and microarray experiment (Fig. 3C) were substantially well correlated with the impressive correlation coefficient of 0.988 (Fig. 3A). The former proves that our microarray experiments were precisely performed in order to determine the differentially expressed genes. In our comparative array experiments, we considered as significant ratios of induction and suppression by vancomycin treatment, the ones with greater than two and less than 0.5-fold respectively.

Comparison of transcription profile between N315 and KVR in the absence of vancomycin by microarray revealed that the transcription of only 13 genes including sgtB (0.4-fold increase) was altered less than 0.5-fold, which indicates significantly suppression due to the disruption of vraSR (Fig. 3B; Table 2). However, the transcription of pbp2 in KVR was not altered significantly (0.87-fold increase) as compared to N315 (Fig. 3B; Table 2). As shown in Fig. 2A, the induction of pbp2 was completely lost in KVR exposed to vancomycin. This finding suggests that the actual signal input appear inadequate to activate VraSR system under drug-free cultivation. Therefore, we considered that the stimulation of vancomycin could reveal the whole regulation network involving VraSR system. The above microarray analysis identified 139 vancomycin-inducible genes in N315 (Fig. 3C), 46 of which showed significant loss of induction in KVR but not in N315 (Table 2; Fig. 3D). It indicates that additional 33 genes found to be positively regulated by VraSR system with vancomycin treatment. Some of them are apparently associated with cell-wall peptidoglycan synthesis encoding such as PBP2, SgtB, UDP-N-acetylg glucosamine 1-carboxylvinyl transferase 2 (murZ), methicillin resistance-related protein (fmtA) and teicoplanin resistance-related proteins (tcaA/tcaB). Curiously, the expression of the genes encoding the four other penicillin-binding proteins (pbpA, pbp3, pbp4 and mecA) and other putative mono-

DNA microarray analyses reveal the regulatory network via VraSR two-component system 811

Fig. 3. Scatter diagrams of normalized spot intensities. Spot intensities have been normalized, and the spots drawn in grey dot showing low signal intensity (less than 25% population) were excluded for further analysis as described in Experimental procedures. r, correlation coefficient.

A. Well correlation of the induced ratio between the results of Northern blot and microarray. The ratio for the vraS, pbp2, sgtB, saeS, coa genes was obtained from Northern blot shown in Fig. 1A and microarray shown in Fig. 3C when parental strain N315 strain was exposed to vancomycin.

B. Spot intensities of array hybridization with comparison between N315 and the vraSR null mutant KVR.

C. Spot intensities of array hybridization obtained by N315 exposed to 10 μg ml⁻¹ of vancomycin (VCM) for 10 min.

D. Spot intensities of array hybridization obtained by KVR exposed to 10 μg ml⁻¹ of vancomycin (VCM) for 10 min.

functional glycosyltransferase (sgtA) were not affected by the exposure to vancomycin for 10 min

The other VraSR-mediated upregulated genes were identified as encoding peptidases (spsA, ctpA), stress response proteins (prsA), proline/glycine-betaine transporters (proP and opuD), the components of phosphotransferase system (PTS) specific to glucose (SA1255), drug-responsive protein (drp35) and unknown predicted ORFs. Among the upregulated genes, fmtA, drp35 and pbp2 have already been identified as β-lactam- or vancomycin-inducible genes, previously (Komatsuzawa et al., 1999; Murakami et al., 1999).

None of VraSR-mediated downregulated gene was observed reproducibly by performing the microarray analysis between N315 and KVR (Fig. 3B).

Identification of VraS-non-mediated regulated genes by vancomycin treatment

Inhibition of cell growth by vancomycin treatment seems to induce the transcription of the genes involved in amino acids biosynthesis pathways such as glutamate (gltD), cysteine (cysK, cysM and metE), histidine (hisB, hisF, hisG, hisH and hisI), lysine (asd, dapA, dapB, dapD, dhoM), threonine (thrC), serine (serA), and valine/isoleucine (ilvE), irrespective of the integrity of vraSR (Table 3). In addition, eight genes encoding the predicted amino acids and oligopeptide transporters also found to be induced.

Furthermore, the transcription of the genes involved in a part of tricarboxylic acid cycle (citB, citC and citZ) catalysing from oxaloacetate to 2-oxoglutarate were induced by exposure to vancomycin. 2-oxoglutarate, one of the resources for the biosynthesis of L-glutamate by NADH-glutamate synthetase small subunit GluD, was as well induced by vancomycin (Table 3).

Additionally the SaeRS system, known as a regulation system of virulence factors such as α-haemolysins, β-haemolysin and coagulase (Giraudo et al., 1997), was induced by vancomycin treatment (Table 3).

On the other hand, vancomycin treatment reduced the expression of the genes encoding cell-wall lytic enzymes, i.e. autolysin (N-acetylmuramyl-L-alanine amidase and endo-β-N-acetylglucosaminidase) and autolysin-like protein (SA0423), although partial contribution of vraSR in the downregulation of autolysin could not be excluded

© 2003 Blackwell Publishing Ltd, Molecular Microbiology, 49, 807–821

Table 2. The genes regulated by VraSR system.

| ORF ID | Gene    | Function                                                                 | KVR/N315 | N315 | KVR |
|--------|---------|--------------------------------------------------------------------------|----------|------|-----|
|        |         | Carbohydrate transport and metabolism                                    |          |      |     |
| SA0531 | proP    | Proline/betaine transporter                                              | 0.20     | 4.26 | 0.96|
| SA1255 | —       | PTS system, glucose-specific enzyme II, A component                      | 0.73     | 2.95 | 1.22|
| SA2145 | tcaB    | TcaB, permeases of the major facilitator superfamily                     | 0.79     | 3.81 | 0.82|
| SA2220 | —       | Glycerate kinase family                                                 | 0.41     | 3.33 | 0.85|
| SA2222 | —       | Permeases of the major facilitator superfamily                          | 0.65     | 2.52 | 0.84|
|        |         | Lipid metabolism                                                        |          |      |     |
| SA1548 | —       | Similar to 1-acylglycerol-3-phosphate O-acyltransferase                   | 0.67     | 2.28 | 1.01|
|        |         | Translation, ribosomal structure and biogenesis                          |          |      |     |
| SA1713 | —       | RNA methyltransferase homologue                                          | 0.93     | 2.29 | 0.89|
|        |         | Transcription                                                           |          |      |     |
| SA2103 | —       | Similar to lyt divergon expression attenuator LytR                       | 0.64     | 4.06 | 1.16|
| SA2296 | —       | Similar to transcriptional regulator, MerR family                       | 0.62     | 3.87 | 1.14|
|        |         | DNA replication, recombination and repair                                |          |      |     |
| SA1711 | —       | Similar to DNA-damage inducible protein P                               | 0.80     | 3.07 | 0.74|
|        |         | Cell envelope biogenesis, outer membrane                                |          |      |     |
| SA0592 | tagA    | Teichoic acid biosynthesis protein                                       | 0.74     | 2.08 | 1.13|
| SA0909 | fmtA    | Autolysis and methicillin resistant-related protein FmtA                 | 0.68     | 2.86 | 0.94|
| SA1183 | opuD    | Choline-glycine betaine transporter                                      | 0.81     | 1.92 | 0.93|
| SA1253 | ctpA    | Carboxy-terminal processing protease                                     | 1.16     | 2.64 | 1.49|
| SA1283 | pbp2    | Penicillin-binding protein 2                                             | 0.87     | 1.87 | 0.86|
| SA1691 | sgtB    | Monofunctional glycosyltransferase                                       | 0.40     | 5.64 | 0.99|
| SA1926 | murZ    | UDP-N-acetyglucosamine enolpyruvyl transferase                           | 0.42     | 2.52 | 0.84|
|        |         | Cell motility and secretion                                             |          |      |     |
| SA0825 | spsA    | Signal peptidase I                                                      | 0.69     | 2.22 | 0.94|
|        |         | Post-translational modification, protein turnover, chaperones            |          |      |     |
| SA1549 | —       | Similar to serine proteinase Do, heat-shock protein htrA                  | 0.61     | 3.62 | 1.43|
| SA1659 | prsA    | Parvulin-like peptidyl-prolyl isomerase                                  | 0.43     | 5.74 | 1.83|
|        |         | Inorganic ion transport and metabolism                                   |          |      |     |
| SA2344 | copA    | Copper-transporting ATPase                                               | 0.72     | 2.04 | 1.24|
| SA2413 | —       | Sulphite reductase flavoprotein subunit                                  | 0.92     | 2.01 | 1.03|
|        |         | Signal transduction mechanisms                                           |          |      |     |
| SA1700 | vraR    | Two-component response regulator                                         | 0.05     | 5.18 | 1.10|
| SA1701 | vraS    | Two-component sensor histidine kinase                                   | 0.20     | 7.41 | 1.23|
|        |         | Poorly characterized                                                    |          |      |     |
| SA0141 | —       | Hypothetical protein                                                    | 0.75     | 1.74 | 0.88|
| SA0536 | —       | Hypothetical protein                                                    | 0.49     | 6.84 | 1.07|
| SA0608 | —       | Hypothetical protein                                                    | 0.68     | 1.78 | 0.84|
| SA0674 | —       | Conserved hypothetical protein                                           | 0.66     | 1.69 | 1.18|
| SA0914 | —       | Similar to chitinase B                                                  | 0.59     | 1.94 | 1.15|
| SA0931 | —       | Hypothetical protein                                                    | 0.92     | 2.20 | 0.91|
| SA0961 | —       | Conserved hypothetical protein                                           | 0.75     | 1.71 | 0.90|
| SA1254 | —       | Conserved hypothetical protein                                           | 0.83     | 3.23 | 1.39|
| SA1282 | recU    | Recombination protein U homologue                                        | 0.87     | 1.62 | 1.04|
| SA1476 | —       | Hypothetical protein                                                    | 0.26     | 2.70 | 0.63|
| SA1657 | —       | Conserved hypothetical protein                                           | 0.50     | 4.12 | 1.25|
| SA1658 | —       | Conserved hypothetical protein                                           | 0.45     | 4.96 | 1.58|
| SA1702 | —       | Conserved hypothetical protein                                           | 0.49     | 7.27 | 1.15|
| SA1703 | —       | Hypothetical protein                                                    | 0.49     | 7.27 | 1.15|
| SA1712 | —       | Conserved hypothetical protein                                           | 0.87     | 4.73 | 0.79|
| SA2113 | —       | Hypothetical protein                                                    | 0.56     | 2.65 | 0.77|
| SA2146 | tcaA    | TcaA                                                                    | 0.92     | 2.40 | 0.96|
| SA2221 | —       | Hypothetical protein                                                    | 0.31     | 5.37 | 1.03|
| SA2297 | —       | Similar to GTP-pyrophosphokinase                                        | 0.65     | 3.85 | 1.13|
| SA2298 | —       | Conserved hypothetical protein                                           | 1.18     | 2.04 | 1.01|
| SA2343 | —       | Hypothetical protein                                                    | 0.26     | 6.55 | 1.48|
| SA2480 | drp35   | Drug responsive protein 35                                              | 0.55     | 2.94 | 1.02|

a. The cell grown to logarithmic phase (OD<sub>660</sub>=0.6) was exposed to 10 μg ml<sup>-1</sup> of vancomycin for 10 min.

DNA microarray analyses reveal the regulatory network via VraSR two-component system 813

Table 3. VraSR non-mediated up or downregulated genes by vancomycin treatment.

| ORF ID | Gene       | Function                                                                 | KVR/N 315 | N315 | KVR |
|--------|------------|-------------------------------------------------------------------------|-----------|------|-----|
| Upregulated by VCM treatment                                                                                          |
| Energy production and conversion                                                                                     |
| SA0171 | fdh        | NAD-dependent formate dehydrogenase                                | 1.54      | 2.63 | 2.91 |
| SA1184 | citB       | Aconitase A                                                           | 1.22      | 1.85 | 2.50 |
| SA1517 | citC       | Isocitrate dehydrogenase                                             | 0.98      | 1.59 | 2.30 |
| SA1518 | citZ       | Citrate synthase II                                                  | 1.17      | 1.88 | 2.62 |
| SA2346 | –          | Similar to D-specific D-2-hydroxyacid dehydrogenase Ddh             | 0.86      | 2.20 | 1.89 |
| Amino acid transport and metabolism                                                                                   |
| SA0344 | metE       | Methionine synthase II (cobalamin-independent)                      | 0.95      | 3.16 | 3.36 |
| SA0345 | –          | Similar to 5,10-methylenetetrahydrofolate reductase                  | 1.09      | 3.37 | 2.90 |
| SA0346 | –          | Similar to cystathionine β-lyase                                    | 1.30      | 3.55 | 2.98 |
| SA0418 | cysM       | Cysteine synthase                                                    | 0.78      | 1.79 | 1.93 |
| SA0419 | metB       | Cystathionine β-lyase/cystathionine γ-synthase                       | 0.67      | 1.76 | 2.11 |
| SA0431 | gltD       | NADH-glutamate synthase small subunit                               | 1.42      | 3.79 | 2.55 |
| SA0471 | cysK       | Cysteine synthase (o-acetylserine sulfhydrolase)                    | 0.73      | 1.82 | 2.29 |
| SA0512 | ilvE       | Branched-chain amino acid aminotransferase                          | 0.81      | 1.91 | 1.82 |
| SA0677 | –          | ABC-type proline/glycine betaine transport system                    | 1.20      | 2.14 | 2.29 |
| SA0845 | oppB       | ABC-type dipeptide/oligopeptide/nickel transport                     | 1.25      | 2.06 | 1.82 |
| SA0846 | –          | Similar to oligopeptide transport system permease protein OppC      | 1.25      | 2.06 | 1.82 |
| SA0847 | oppD       | Oligopeptide transport system ATP-binding protein OppD              | 1.60      | 2.47 | 2.12 |
| SA0848 | oppF       | Oligopeptide transport system ATP-binding protein OppF              | 1.60      | 2.47 | 2.12 |
| SA0849 | –          | Similar to peptide binding protein OppA                             | 1.68      | 2.45 | 2.04 |
| SA0859 | –          | Oligoendopeptidase F                                                 | 1.14      | 2.06 | 1.83 |
| SA1163 | –          | Similar to aspartokinase                                             | 1.39      | 2.09 | 1.84 |
| SA1164 | dhoM       | Homoserine dehydrogenase                                             | 1.00      | 2.46 | 2.39 |
| SA1165 | thrC       | Threonine synthase                                                   | 1.21      | 2.72 | 2.61 |
| SA1226 | asd        | Aspartate-semialdehyde dehydrogenase                                 | 0.74      | 3.03 | 3.45 |
| SA1227 | dapA       | Dihydromipicolinate synthase/N-acetylneuraminate lyase                | 0.76      | 2.65 | 3.22 |
| SA1228 | dapB       | Dihydromipicolinate reductase                                        | 0.76      | 2.65 | 3.22 |
| SA1229 | dapD       | Tetrahydromipicolinate N-succinyltransferase                         | 0.94      | 3.09 | 2.56 |
| SA1366 | –          | Glycine cleavage system protein P (pyridoxal-binding)               | 1.16      | 1.74 | 2.26 |
| SA1544 | –          | Similar to serine-pyruvate aminotransferase/archaeal aspartate      | 1.13      | 5.19 | 2.59 |
| SA1545 | serA       | D-3-phosphoglycerate dehydrogenase                                   | 1.73      | 3.71 | 2.61 |
| SA1558 | –          | Chorismate mutase homolog                                            | 1.13      | 2.10 | 1.88 |
| SA2347 | –          | Similar to aspartate aminotransferase                                | 0.88      | 2.44 | 2.04 |
| SA2396 | –          | Similar to amino acid transporter                                    | 1.00      | 3.44 | 2.13 |
| SA2397 | –          | Similar to pyridoxal-phosphate dependent aminotransferase           | 0.86      | 2.22 | 2.06 |
| SA2464 | hisI       | Phosphoribosyl-AMP cyclohydrolase                                   | 1.14      | 1.90 | 2.21 |
| SA2465 | hisF       | Imidazoleglycerol-phosphate synthase                                | 1.05      | 2.06 | 2.37 |
| SA2466 | –          | Similar to phosphoribosylformimino-5-aminoimidazole carboxamide     | 1.05      | 2.18 | 2.47 |
| SA2467 | hisH       | Glutamine amidotransferase                                           | 1.29      | 2.71 | 3.25 |
| SA2468 | hisB       | Imidazoleglycerol-phosphate dehydratase                             | 1.25      | 3.19 | 3.29 |
| SA2470 | –          | Similar to histidinol dehydrogenase                                 | 1.20      | 2.82 | 2.76 |
| SA2471 | hisG       | ATP phosphoribosyltransferase (histidine biosynthesis)              | 1.11      | 2.48 | 3.12 |
| Carbohydrate transport and metabolism                                                                                   |
| SA2204 | pgm        | Phosphoglycerate mutase 1                                            | 0.80      | 1.56 | 1.99 |
| Coenzyme metabolism                                                                                                   |
| SA1586 | ribH       | Riboflavin synthase β-chain                                         | 0.92      | 3.06 | 2.55 |
| SA1587 | ribA       | 3,4-dihydroxy-2-butanone 4-phosphate synthase                       | 0.96      | 2.93 | 2.64 |
| SA1588 | ribB       | Riboflavin synthase α-chain                                         | 1.03      | 3.10 | 2.77 |
| SA1589 | ribD       | Pyrimidine reductase, riboflavin biosynthesis                        | 0.83      | 3.00 | 2.53 |
| Lipid metabolism                                                                                                      |
| SA0169 | –          | Similar to acyl-CoA dehydrogenase                                   | 0.89      | 5.36 | 9.15 |
| SA1434 | –          | Acetyl-CoA carboxylase (biotin carboxylase subunit)                 | 1.51      | 2.73 | 3.16 |
| SA1435 | –          | Similar to acetyl-CoA carboxylase (biotin carboxyl carrier subunit) | 1.23      | 2.39 | 2.82 |
| Translation, ribosomal structure and biogenesis                                                                         |
| SA0707 | –          | Ribosome-associated protein Y                                       | 0.75      | 1.98 | 2.39 |
| Cell envelope biogenesis, outer membrane                                                                              
| SA0678 | –          | Similar to choline transporter                                      | 1.06      | 2.02 | 1.88 |

814 M. Kuroda et al.

Table 3. cont.

| ORF ID | Gene    | Function                                                                 | KVR/N 315 | N315   | KVR    |
|--------|---------|--------------------------------------------------------------------------|-----------|--------|--------|
|        |         | Post-translational modification, protein turnover, chaperones              |           |        |        |
| SA0483 | clpC    | ClpC chaperone homologue                                                 | 0.99      | 1.94   | 2.21   |
| SA0835 | clpB    | ClpB chaperone homologue                                                 | 0.79      | 1.89   | 2.24   |
| SA1849 | –       | Conserved hypothetical protein                                           | 1.21      | 2.26   | 4.85   |
|        |         | Inorganic ion transport and metabolism                                   |           |        |        |
| SA0166 | –       | Similar to ABC-type nitrate/sulphonate/taurine/bicarbonate transport       | 1.09      | 5.03   | 6.68   |
| SA0167 | –       | Similar to ABC-type nitrate/sulphonate/taurine/bicarbonate transport       | 1.09      | 5.03   | 6.68   |
| SA0168 | –       | Similar to ABC-type nitrate/sulphonate/taurine/bicarbonate transport       | 0.89      | 5.36   | 9.15   |
| SA2475 | –       | ABC-type cobalt transport system, permease component                      | 1.35      | 2.77   | 2.07   |
| SA2476 | –       | ABC-type cobalt transport system, ATPase component                        | 1.52      | 2.74   | 2.25   |
|        |         | Secondary metabolites biosynthesis, transport and catabolism               |           |        |        |
| SA0122 | butA    | Acetoin(diacetyl) reductase                                              | 1.44      | 2.10   | 1.84   |
|        |         | Signal transduction mechanisms                                           |           |        |        |
| SA0660 | saeS    | Two-component sensor histidine kinase                                    | 1.07      | 2.37   | 2.11   |
| SA0661 | saeR    | Two-component response regulator                                         | 1.20      | 4.23   | 3.79   |
|        |         | Virulence factors                                                       |           |        |        |
| SA0222 | coa     | Staphylocoagulase precursor                                              | 0.70      | 3.17   | 2.96   |
| SA0393 | set15   | Exotoxin 15                                                             | 0.78      | 1.91   | 2.05   |
| SA2206 | sbi     | IgG-binding protein SBI                                                 | 0.77      | 3.36   | 3.51   |
| SA2207 | hlgA    | γ-haemolysin component A                                                 | 1.06      | 2.77   | 3.56   |
| SA2208 | hlgC    | γ-haemolysin component C                                                 | 2.23      | 3.88   | 4.11   |
| SA2209 | hlgB    | γ-haemolysin component B                                                 | 2.90      | 5.72   | 5.41   |
|        |         | Poorly characterized                                                    |           |        |        |
| SA0164 | –       | Hypothetical protein                                                    | 0.94      | 3.67   | 5.56   |
| SA0165 | –       | Similar to alpha-helical coiled-coil protein SrpF                         | 0.94      | 3.67   | 5.56   |
| SA0170 | –       | Conserved hypothetical protein                                           | 1.54      | 2.63   | 2.91   |
| SA0343 | –       | Uncharacterized ACR, predicted metal-dependent hydrolase                   | 0.95      | 3.16   | 3.36   |
| SA0394 | –       | Hypothetical protein                                                     | 0.94      | 2.82   | 3.05   |
| SA0420 | –       | Uncharacterized ABC-type transport system ATPase                           | 1.02      | 2.77   | 3.08   |
| SA0421 | –       | Permease component of an uncharacterized ABC transporter                   | 0.99      | 2.84   | 3.40   |
| SA0481 | –       | Conserved hypothetical protein                                           | 0.75      | 1.84   | 2.41   |
| SA0482 | –       | Similar to creatine kinase                                               | 0.80      | 1.78   | 2.23   |
| SA0771 | –       | ABC-type uncharacterized transport system                                | 1.46      | 2.33   | 2.03   |
| SA1230 | –       | Metal-dependent amidase/aminoacylase/carboxypeptidase                     | 0.88      | 2.73   | 2.66   |
| SA1432 | –       | Conserved hypothetical protein                                           | 1.62      | 2.20   | 2.77   |
| SA1433 | –       | Uncharacterized proteins, homologues of lactam utilization                 | 1.65      | 2.46   | 2.99   |
| SA1436 | –       | Conserved hypothetical protein                                           | 1.28      | 2.23   | 2.83   |
| SA1437 | –       | Conserved hypothetical protein                                           | 1.42      | 2.33   | 2.72   |
| SA1532 | –       | Conserved hypothetical protein                                           | 0.99      | 1.89   | 1.98   |
| SA1546 | –       | Hypothetical protein                                                     | 1.18      | 3.09   | 1.99   |
| SA1850 | –       | Conserved hypothetical protein                                           | 1.06      | 2.26   | 5.37   |
| SA2062 | –       | Hypothetical protein                                                     | 0.83      | 3.48   | 3.45   |
| SA2357 | –       | Predicted membrane protein, putative toxin regulator                      | 0.82      | 2.32   | 1.94   |
| SA2398 | –       | Hypothetical protein                                                     | 1.16      | 2.01   | 2.14   |
| SA2431 | isaB    | Immunodominant antigen B                                                 | 1.36      | 1.95   | 2.17   |
| SA2474 | –       | Conserved hypothetical protein                                           | 1.43      | 2.19   | 1.91   |
| SA2492 | vraD    | ABC-type transport system                                                | 2.07      | 3.79   | 1.90   |
| SA2493 | vraE    | ABC-type transport system                                                | 3.95      | 9.32   | 2.15   |
|        |         | Downregulated by VCM treatment                                           |           |        |        |
|        |         | Cell division and chromosome partitioning                                 |           |        |        |
| SA2499 | gidB    | Glucose inhibited division protein B                                      | 0.94      | 0.48   | 0.59   |
| SA2500 | gidA    | Glucose inhibited division protein A                                      | 0.85      | 0.42   | 0.52   |
|        |         | Amino acid transport and metabolism                                       |           |        |        |
| SA1538 | –       | Similar to iron-sulphur cofactor synthesis protein nifZ                    | 1.04      | 0.37   | 0.43   |
|        |         | Nucleotide transport and metabolism                                       |           |        |        |
| SA0016 | purA    | Adenylosuccinate synthase                                                 | 0.83      | 0.27   | 0.37   |
|        |         | Carbohydrate transport and metabolism                                     |           |        |        |
| SA2142 | –       | Permeases of the major facilitator superfamily                            | 0.99      | 0.44   | 0.45   |
| SA2241 | –       | Similar to chloramphenicol resistance protein                            | 0.93      | 0.41   | 0.39   |

© 2003 Blackwell Publishing Ltd, *Molecular Microbiology*, 49, 807–821

DNA microarray analyses reveal the regulatory network via VraSR two-component system 815

Table 3. cont.

| ORF ID | Gene    | Function                                                                 | KVR/N315 | N315 | KVR |
|--------|---------|--------------------------------------------------------------------------|----------|------|-----|
| Coenzyme metabolism                                                                                          |          |      |     |
| SA0666 | –       | 6-pyruvoyl tetrahydrobiopterin synthase                                      | 0.88     | 0.42 | 0.52|
| SA1537 | –       | Similar to thiamine biosynthesis ATP pyrophosphatase                         | 1.04     | 0.49 | 0.49|
| Translation, ribosomal structure and biogenesis                                                   |          |      |     |
| SA1083 | trmD    | tRNA-(guanine-N1)-methyltransferase                                          | 0.78     | 0.40 | 0.47|
| DNA replication, recombination and repair                                                        |          |      |     |
| SA1188 | parE    | Topoisomerase IV subunit B                                                   | 1.06     | 0.40 | 0.46|
| SA1885 | –       | Similar to ATP-dependent RNA helicase                                        | 0.88     | 0.33 | 0.34|
| Cell envelope biogenesis, outer membrane                                                         |          |      |     |
| SA0423 | –       | similar to autolysin                                                        | 0.76     | 0.37 | 0.54|
| SA0905 | atl     | Autolysin                                                                  | 0.99     | 0.25 | 0.36|
| SA1959 | glmS    | Glucosamine 6-phosphate synthetase                                           | 0.91     | 0.45 | 0.41|
| Signal transduction mechanisms                                                                   |          |      |     |
| SA1666 | –       | Two-component response regulator                                             | 1.00     | 0.47 | 0.50|
| SA1667 | –       | Two-component sensor histidine kinase                                        | 0.86     | 0.48 | 0.49|
| Poorly characterized                                                                              |          |      |     |
| SA0085 | –       | Conserved hypothetical protein                                               | 0.92     | 0.44 | 0.41|
| SA0710 | –       | Conserved hypothetical protein                                               | 0.83     | 0.31 | 0.17|
| SA1062 | –       | Conserved hypothetical protein                                               | 0.98     | 0.46 | 0.56|
| SA1224 | –       | ATPase components of ABC transporters                                         | 1.02     | 0.35 | 0.60|
| SA1664 | –       | Conserved hypothetical protein                                               | 1.18     | 0.54 | 0.44|
| SA1665 | –       | Hypothetical protein                                                        | 1.08     | 0.42 | 0.43|
| SA2093 | ssaA    | Secretory antigen precursor SsaA homologue                                    | 0.42     | 0.10 | 0.09|
| SA2126 | –       | Hypothetical protein                                                        | 0.80     | 0.38 | 0.38|
| SA2193 | –       | Conserved hypothetical protein                                               | 1.26     | 0.39 | 0.67|
| SA2194 | –       | Similar to Zn-binding lipoprotein adcA                                       | 1.04     | 0.22 | 0.48|
| SA2355 | –       | Conserved hypothetical protein                                               | 0.93     | 0.33 | 0.33|
| SA2356 | isaA    | Immunodominant antigen A                                                     | 0.80     | 0.16 | 0.11|
| SA2447 | –       | Similar to streptococcal haemagglutinin protein                               | 0.61     | 0.28 | 0.27|
| SA2501 | thdF    | Possible thiophene and furan oxidation protein                                | 0.90     | 0.42 | 0.52|

a. The cell grown to logarithmic phase (OD<sub>660</sub> = 0.6) was exposed to 10 μg ml<sup>-1</sup> of vancomycin for 10 min.

based on this data set alone (Table 3). As a total effect of cell-wall synthesis inhibition mediated by vancomycin, it was reasonable that cell-wall lytic activity was negatively regulated by S. aureus regulation systems. Curiously, highly immunoreactive antigens (ssaA and isaA) and putative haemagglutinin proteins (SA2447) were significantly downregulated by vancomycin.

The other genes involved in cell division (gidA and gidB), DNA replication (parE), and tRNA modification were downregulated by the growth inhibition of vancomycin.

Discussion

We have previously observed upregulation of vraSR in vancomycin-resistant S. aureus strains Mu50 and Mu3 (Kuroda et al., 2000). In this study, we tried to identify certain effector enzymes responsible for vancomycin resistance through the characterization of the VraSR regulation network. In view of the demonstrated activation of cell-wall synthesis in vancomycin-resistant strains

(Hanaki et al., 1998; Cui et al., 2000), we suspected that VraSR might be involved in the activation of cell-wall synthesis pathway in S. aureus. Based on the observation in this study, VraSR system seems to co-ordinate the important steps of cell-wall biosynthesis: murein monomer precursor synthesis (MurZ) and polymerization of peptidoglycan (PBP2 and SgtB). In agreement with this, disruption of the vraSR system was accompanied by significant increase in the susceptibility against various cell wall synthesis inhibitors (Table 1). The vraSR knock-out strain KVR exhibited reduced transcription of mRNAs for UDP-N-acetylglucosamine enolpyruvyl transferase (MurZ) and PBP2, which well correlated with increased susceptibility of the strain towards cell-wall synthesis inhibitors such as fosfomycin, glycopeptides and β-lactam antibiotics.

Because mecA gene encoding PBP2' is required for β-lactam resistance in S. aureus, N315ex that lacks mecA exhibited β-lactam susceptibility indistinguishable from that of the laboratory MSSA strain ATCC 25923. However,

even in the presence of *mecA* gene, methicillin resistance is not expressed if an intact *mecI* gene encoding *mecA* gene repressor is co-existent as in strain N315 (Kuwahara *et al.*, 1996). Strain ΔIP is the N315-derived strain whose *mecI* gene is deleted and the transcription of *mecA* gene is de-repressed. ΔIP expresses heterogeneous type methicillin resistance (hetero-resistance). Hetero-resistant strain spontaneously generates highly methicillin-resistant (homo-resistance) mutants at high frequencies of one in 10<sup>−6</sup> (Finan *et al.*, 2002). The genetic event that underlies this hetero-to-homo phenotypic conversion of methicillin resistance is the one that has long been sought after by many researchers of staphylococcus (Ryffel *et al.*, 1994).

ΔIPH14 was an *in vitro* strain with homo methicillin resistance, which was derived from ΔIP by *in vitro* selection with imipenem, a potent β-lactam antibiotic. The *vraSR* knockout of ΔIPH14 resulted in remarkable reduction in β-lactam resistance to the susceptibility level equal to that of strain N315 (Table 1). Furthermore, the *vraS*, *pbp2* and *sgtB* genes were constantly overexpressed in ΔIPH14. These observations indicated the possibility that altered expression of VraSR system, which induces activation of cell-wall synthesis pathway, is responsible for the homo-type methicillin resistance of ΔIPH14. Though this hypothesis is now under investigation together with the reason why *vraS* was over-expressed in ΔIPH14, it is clear that the function of VraSR was required for the expression of methicillin resistance in ΔIPH14. In this regard, it is important to note that the VraSR inactivation did not annihilate the homo-type methicillin resistance of ΔIPH5. This is consistent with the observation that there is no alteration in the transcription of *vraSR*, *pbp2* or *sgtB* gene in ΔIPH5 (Fig. 2B). This shows that altered expression of VraSR is not involved in every genetic mechanism for homo-type methicillin resistance, and that there are multiple mechanisms for *S.* aureus to achieve homogeneously high methicillin resistance (Kondo *et al.*, 2001). On the other hand, this fact makes the VraSR system distinctive in comparison with many other genes previously identified in association with homo-type methicillin resistance expression. Most of the genes called *fem* or *aux* turned out to be the genes encoding enzymes directly involved in the cell-wall peptidoglycan synthesis pathway, and they are basically essential for the production of physically intact cell wall (Berger-Bächi and Rohrer, 2002). The fact that complete deletion of VraSR was possible without decreasing the β-lactam resistance of ΔIPH5 indicated that the VraSR system is not an essential system for *S.* aureus to maintain vegetative life without hazardous impingement from the environment. In this sense, VraSR system would be considered as the ‘accelerator system’ for the accidental breach of cell-wall peptidoglycan integrity as discussed below.

We have demonstrated that overproduction of PBP2 significantly increases resistance to teicoplanin (Hanaki *et al.*, 1998). Therefore, the significant reduction in teicoplanin resistance observed in *vraSR* null mutants agrees well with the loss of PBP2 induction. PBP2 has been shown to possess transglycosylase (TGase) activity that catalyses elongation of nascent peptidoglycan chains (Pinho *et al.*, 2001). However, elongation of the chains is not completely abolished after inactivation of TGase domain of PBP2, indicating that other TGases are also in charge of the elongation reaction. By whole genome analysis of *S.* aureus strains, we identified two putative monofunctional glycosyltransferase SgtA and SgtB whose predicted products show significant similarity to TGase domain (pfam00912) with 3e<sup>−38</sup> and 2e<sup>−39</sup> of *E*-value respectively (Kuroda *et al.*, 2001). Computer search for TGase domain over the all ORFs found in *S.* aureus chromosome hits only three TGase candidates: PBP2, SgtA and SgtB. Wang *et al.* (2001) demonstrated that SgtB (they call it Mgt) catalyses the elongation of peptidoglycan chains. Therefore, our observation suggests that VraSR system positively regulates the two glycosyltransferase genes, and as such, may be involved in glycopeptide resistance in *S.* aureus (Hiramatsu 2001). It is considered that increased TGase activity contributes to the resistance either by competing with glycopeptides for the capture of membrane-bound murein monomers or by increasing production of nascent peptidoglycan chains to provide more D-alanyl-D-alanine residues that serve as false targets for vancomycin (Hiramatsu, 1998; 2001).

Introduction of *vraSR* genes in a high-copy-number plasmid alone did not increase transcription of *pbp2* and *sgtB*. Induction requires exposure to the cell-wall synthesis inhibitors in the presence of VraSR (Fig. 2A: KVR-SR). This indicates that certain physiological signal is generated upon inhibition of cell-wall synthesis that in turn activates VraS sensor kinase. Although the transcription of *vraSR* was not affected by growth phase during aerobic cultivation, that of *saeRS* was slightly upregulated at late mid-exponential growth phase (data not shown). It appears therefore that the two systems are receptive of different physiological signals, though both were activated by exposure to vancomycin. It is probable that vancomycin treatment generates signals in diverse physiological pathways, which are recognized by multiple signal sensors that in turn activate multiple response regulators. Further investigation is needed to clarify what signal VraSR system recognizes and how it modifies physiological milieu of the cell.

Many researchers have identified the genes associated with β-lactam as well as glycopeptide resistance of *S.* aureus using such experimental techniques as transposon mutagenesis and differential expression screening. In this study, we found some of the previously identified

© 2003 Blackwell Publishing Ltd, *Molecular Microbiology*, **49**, 807–821

DNA microarray analyses reveal the regulatory network via VraSR two-component system 817

genes are under the control of VraSR system (Table 2). Among them, the disruption of *fmtA* or *vraDE* (RUSA221) by transposon mutagenesis reduces β-lactam resistance of a homogeneously methicillin-resistant strain COL (De Lencastre *et al.*, 1999; Komatsuzawa *et al.*, 1999). Inactivation of *tcaA* (or entire *tcaRAB* operon) leads to reduction in methicillin resistance and concomitant increase in teicoplanin resistance (Brandenberger *et al.*, 2000). The *drp35* gene is induced by β-lactam treatment, but its association with antibiotic resistance has not been reported yet (Murakami *et al.*, 1999). Besides the above, we newly identified nine *orfs* encoding hypothetical proteins and eight *orfs* encoding conserved hypothetical proteins regulated by VraSR system. Those may turn out to be important genes for further study of glycopeptide and β-lactam resistance in *S. aureus*.

Glutamine and lysine amino acids are essential constituents of murein monomer precursor. The part of activation of tricarboxylic acid cycle producing 2-oxoglutarate from oxaloacetate might be associated with maintaining enough amounts of glutamine pool. The *dap* operon, which consists of lysine biosynthesis genes, should also be an essential contributor to cell-wall peptidoglycan synthesis. In fact, transposon-mutagenesis of *dap* operon genes leads to the growth attenuation of the cell both *in vivo* and in serum (Mei *et al.*, 1997; Coulter *et al.*, 1998; Wiltshire and Foster, 2001). Furthermore, it has been reported that *lysA* disruption reduces oxacillin resistance of MRSA strain COL from 1600 to 25 µg ml⁻¹ in MIC (De Lencastre *et al.*, 1999). In addition, oligopeptide transporter homologues *oppD* and *oppF* are also identified as the genes affecting its growth *in vivo* (Mei *et al.*, 1997; Coulter *et al.*, 1998). This observation suggests that uptake of oligopeptide would support biosynthesis of the amino acid components of cell-wall peptidoglycan. Induction of other amino acid biosynthesis pathway may or may not directly contribute to the cell-wall biosynthesis (Table 3).

Besides the genes apparently involved in cell-wall synthesis, VraSR regulates transcription of the genes encoding stress-response proteins such as osmoprotectant transporters (*proP* and *opuD*). Such transporters contribute to the maintenance of osmolarity of the cytoplasm by taking up osmoprotectants such as proline and glycine-betaine from the environment (MacMillan *et al.*, 1999). Because of the high osmolarity of the bacterial cytoplasm (Whatmore and Reed, 1990), the cell becomes vulnerable to ruptured death when the peptidoglycan becomes fragile by the exposure to cell-wall biosynthesis inhibitors. It was also noted in this regard that the genes encoding cell-wall lytic enzymes such as autolysins (*N*-acetylmuramyl-L-alanine amidase and endo-β-*N*-acetylglucosaminidase) and autolysin-like protein (SA0423) were suppressed after exposure to vancomycin (Table 3). It would be reasonable to assume that the down regulation of autolysin production is in the same physiological context to avoid autolytic death of the cells with compromised cell-wall peptidoglycan integrity.

Aside from the VraSR-regulated genes, there were genes activated or suppressed by vancomycin treatment of the cell. From the point of view of virulence regulation, vancomycin might activate certain virulence factors through the activation of SaeRS system (Table 3). SaeRS is known as the sensor-regulator system for haemolysins and coagulase. Our results coincide with previously reported results except for α- and β-haemolysins (Giraudo *et al.*, 1997). Based on the report of Giraudo *et al.* (1997) the transcription of *hla* gene encoding α-haemolysin is positively regulated after exponential growth phase (OD₆₅₀ = 1.3). There was no upregulation of *hla* in our experiments. The reason for this discrepancy is unknown, but could be due to the difference in the growth phase of the culture at which we observed the transcription profile, which in our case was the mid-exponential growth phase (OD₆₆₀ = 0.6). The reason for the lack of positive regulation of *hlb* gene encoding β-haemolysin is apparent: the gene is disrupted in N315 by the integrated copy of bacteriophage φN315 (Kuroda *et al.*, 2001). Curiously, vancomycin treatment might also modulate the immune responsiveness of the infected host towards the bacterium, as the treatment suppressed the expression the genes encoding immuno-dominant antigens (Lang *et al.*, 2000; Lorenz *et al.*, 2000). That might make the bacterium less ‘visible’ from the immune system, though the significance of this phenomenon needs discreet clinical observation of the vancomycin-treated MRSA infection cases.

### Experimental procedures

#### Bacterial strains and plasmids

*Staphylococcus aureus* strains and constructed plasmids are listed in Table 4. For gene complementation, following polymerase chain reaction (PCR) amplified fragment was cloned into the *BamHI* restriction site of pYT3 shuttle vector: 1299 bp of *vraS* on pNVS (from 1948627 to 1947329 in the position of N315 genome sequence); 2013 bp of *vraSR* on pNVSR (from 1948627 to 1946615 in N315). Those cloned genes were oriented downstream of *tetL* gene on pYT3, and co-transcribed with the *tetL* gene by the *tetL* original promoter. Construction of targeting plasmids pVDK14 and pSDK5 is described below.

#### Antibiotic susceptibility tests

Minimum inhibitory concentration (MIC) was determined on Mueller-Hinton (MH) agar plates (DIFCO, MD, USA) with standard twofold increments of antibiotic concentration as described previously (Suzuki *et al.*, 1993).

Table 4. Bacterial strains and plasmids used in this study.

| Strain and plasmid | Description$^a$ | Reference |
|--------------------|-----------------|-----------|
| S. aureus strains  |                 |           |
| ATCC25923          | Wild-type strain, methicillin-susceptible | Boyle et al. (1973) |
| N315               | Standard strain for genome analysis, pre-MRSA | Kuroda et al. (2001) |
| N315ex             | N315 cured of SCCmec | Katayama et al. (2000) |
| KVR                | N315 ΔvraSR::cat | This study |
| KVR-Y              | KVR: pYT3 | This study |
| KVR-S              | KVR: vraS with pNVS | This study |
| KVR-SR             | KVR: vraS and vraR with pNVSR | This study |
| ΔIP                | N315 ΔmecI::tetL | Kuwahara-Arai et al. (1996) |
| ΔIPH5              | Homogenous MRSA obtained from ΔIP | This study |
| ΔIPH14             | Homogenous MRSA obtained from ΔIP | This study |
| ΔIP-KV             | ΔIP ΔvraSR::cat | This study |
| H5-KV              | ΔIP-H5 ΔvraSR::cat | This study |
| H14-KV             | ΔIP-H14 ΔvraSR::cat | This study |
| KSA                | N315 ΔsaeRS::cat | This study |
| Plasmids           |                 |           |
| pYT3               | Shuttle vector; E. coli, amp<sup>r</sup>; S. aureus, tet<sup>r</sup>, ts-replication | Hanaki et al. (1998) |
| pVDK14             | Targeting plasmid for vraSR | This study |
| pNVS               | 1299 bp of vraS fragment in pYT3 | This study |
| pNVSR              | 2013 bp of vraS and vraR fragment in pYT3 | This study |
| pSDK5              | Targeting plasmid for saeRS | This study |

a. MRSA, methicillin-resistant S. aureus; amp<sup>r</sup>, ampicillin resistance; tet<sup>r</sup>, tetracycline resistance.

**Construction of the vraSR null mutants and saeRS null mutant**

A DNA construct for allelic replacement of the vraSR locus were generated by three-piece PCR fragments including two separate long flanking sequences [1342 bp upstream (from 1949720 to 1948379 in the position of N315 genome sequence) and 807 bp downstream (from 1946733 to 1945927 in N315) of the vraSR locus] surrounding a central selectable cassette containing the 822 bp cat gene from pRIT5. The whole length DNA construct was digested and cloned into the BamHI site of pYT3. The resulting plasmid pVDK14 was introduced into S. aureus N315 by electroporation with tetracycline selection. The transformant was cultured with brain heart infusion (BHI) broth overnight, and spread 10<sup>6</sup> colony forming unit (CFU) onto BHI agar plate containing 10 μg ml<sup>-1</sup> of chloramphenicol, following incubation at 42°C overnight. Integration of pVDK14 into chromosome DNA was verified by PCR with primer sets located at pYT3 vector specific and outside of the fragments used for allelic replacement. The resulting single crossing-over mutant was cultured with drug free BHI broth at 30°C overnight, and the double-crossing over vraSR null mutant showing chloramphenicol resistance and tetracycline susceptible was selected by duplicating to BHI agar containing either 10 μg ml<sup>-1</sup> of chloramphenicol or 10 μg ml<sup>-1</sup> of tetracycline. The vraSR null mutant KVR was verified for allelic replacement by performing PCR with vraS and vraR gene-specific oligoprimers. The cat gene in KVR was transduced by phage MR83A to ΔIP (hetero-MRSA), ΔIPH5 (homo-MRSA), and ΔIPH14 (homo-MRSA, and reduced-susceptibility against teicoplanin). The obtained transductants ΔIP-KV, H5-KV, H14-KV with allelic replacement of vraSR by cat were verified by PCR as described above.

As is the case with construction of the vraSR null mutant, the saeRS null mutant KSA was also selected with allelic replacement by using pSDK5 including two separate long flanking sequences of saeRS locus [1028 bp upstream (from 758426 to 757399 in N315) and 1139 bp downstream (from 755258 to 754119 in N315) of saeRS locus] surrounding the central selectable cat gene.

**Exposure to antibiotics or stresses**

Ten ml of bacterial culture was grown under aeration at 37°C in BHI broth with bio-photorecorder TN2612 (ADVANTEC, Japan). Exponential-phase culture (optical density OD at 660 nm = 0.6) was exposed to antibiotics at 10-fold MIC (final concentration) for 10 min In case of heat stress, culture was shifted from 37°C to 46°C. In case of salt stress, cell was resuspended with BHI broth containing 2 M sodium chloride. In case of pH stress, pH of culture was shifted from usual cultivation (pH 7.4) to either acidic (pH 5.0) or alkali (pH 9.0) by addition of 1 N hydrochloride and sodium hydroxide respectively.

**Preparation of total RNA**

Exponential-phase culture (optical density OD at 660 nm = 0.6) was harvested and suspended with 6 ml of precooled T10E10 buffer [10 mM Tris-HCl (pH 8.0), 10 mM ethylenediamine-N,N',N'-tetraacetic acid] containing 10 μg recombinant-lysostaphin (WAKO, Japan). Immediately, the suspension was incubated at 37°C for 3 min until complete lysis was observed, and then added with 600 μl of 3 M sodium acetate (pH 5.2), 600 μl of 10% sodium dodecyl sulfate (SDS) and 7 ml of acidic-phenol (pH 5.2, equilibrated with 20 mM sodium acetate). Following phenol, chloroform extraction and ethanol precipitation, the pelleted total RNA was subjected to RNase free-DNase I (Roche, Mannheim, Germany) digestion at 37°C for 20 min. The RNA sample was purified again with phenol, chloroform extraction and ethanol

© 2003 Blackwell Publishing Ltd, *Molecular Microbiology*, 49, 807–821

DNA microarray analyses reveal the regulatory network via VraSR two-component system 819

precipitation. Quality and quantity of total RNA were confirmed by agarose electrophoresis and UV spectrophotometer respectively.

### Northern blot analysis

Five μg of total RNA and formamide-bromophenol dye were mixed, and incubated at 65°C for 15 min following quick chilled on ice. Denatured total RNA was separated with conventional formaldehyde-denatured agarose gel electrophoresis in 1× MOPS buffer [20 mM 3-(N-morpholino)propanesulphonic acid, 5 mM sodium acetate and 1 mM ethylenediamine-N,N′,N′,N′-tetraacetic acid, pH 7.0]. After checking quality of total RNA with ethidium bromide staining, the agarose gel was subjected to 20 × SSC (3 M NaCl and 0.3 M sodium citrate) transfer to GeneScreen Plus hybridization transfer membrane (NEN™ Life science, Boston, USA) following UV cross-linking. The transferred membrane was hybridized at 65°C for 18 h with PerfectHyb™ hybridization solution (TOYOBO, Japan) and [α-³²P]-dATP labelled probe using Strip-EZ™ PCR kit (Ambion, Texas, USA), antisense primer and gene specific PCR product as a template. The hybridized membrane was washed twice with 2 × SSC and 0.1% SDS at room temperature, following twice with 0.1 × SSC, 0.1% SDS at 65°C. Hybridization signal and intensity were analysed with BAS2500 imaging analyzer (FUJIFILM, Japan). The hybridized membrane was reused after probe degradation and stripping followed by manufacturer instruction.

### Construction of Staphylococcus aureus N315 genomic DNA microarray

To construct the DNA microarray covering whole genome region of S. aureus N315, 2166 clone were selected from about 35 000 shotgun clone of N315 genome. These clone (average insert size, 1.4 kb; average number of ORFs, 2.2) were used as a template for PCR amplification, following custom spotting to silanate-coated slide glass was requested to ASAHI TECHNO GLASS (ATG, Japan).

### Preparation of Cy3- and Cy5-dUTP labelled cDNA probe

Fluorescence-labelled single-stranded cDNA was prepared by reverse transcription (RT) of total RNA. Before RT reaction, following reaction mixture was incubated at 65°C for 5 min; total 40 μl of reaction, which was consisted of Tris-HCl (pH 8.3) (50 mM), KCl (100 mM), dithiothreitol (4 mM), MgCl₂ (10 mM), total RNA (30 μg), random hexamers (5.3 nmol) (TaKaRa, Japan), FluoroLink™ Cy3- or Cy5-dUTP (4 nmol) (Amersham Bioscience, NJ, USA), dATP, dGTP and dCTP (0.5 mM each), dTTP (0.2 mM). After incubation, 50 U of XLAMV reverse transcriptase (TaKaRa, Japan) was added to this mixture. Reverse transcription reaction was carried out at 42°C for 1 h, and then continued for another hour following the addition of 50 U of XLAMV reverse transcriptase. The reaction mixture was applied to a Centri-Sep spin column (Perkin-Elmer/Applied Biosystems, CA, USA) pre-equilibrated in 800 μl of 100 mM NaCl, according to the supplier’s instructions. The fluorescence-labelled cDNA was extracted with phenol/water/chloroform (Perkin-Elmer/

Applied Biosystems, CA, USA), and pelleted by ethanol precipitation. After washing with 70% ethanol, the pellet was air-dried and dissolved in 8 μl of RNase-free distilled water. The labelled cDNA probe was stored in –80°C until use.

#### Array hybridization and image analysis

Thirty μl of hybridization mixture, containing 4 μl of each Cy3- and Cy5-dUTP labeled cDNA probes, 4 × SSC, 0.2% SDS, 5 × Denhardt’s solution and 3 μg sonicated salmon sperm DNA, was denatured at 95°C for 2 min, and then applied onto microarray slide glass following incubation at 65°C overnight under humidified condition. Microarray was washed twice with 2 × SSC at room temperature for 5 min following twice with 0.1 × SSC, 0.1% SDS at 65°C for 5 min, three times with 0.1 × SSC at room temperature for 5 min, and finally desiccated.

To increase reproducibility, RNA extraction and array hybridization were performed in duplicate. The intensity of fluorescence of the microarray was scanned with GMS418 Array scanner (Affymetrix/Genetic MicroSystems, CA, USA), and the individual signal intensity was analysed with ImaGene™ software ver 4.2 (Biodiscovery, LA, USA). The actual signal intensity was calculated by subtracting mean value of local background intensity from mean value of signal intensity of individual spot. The subtracted signal (less than 100) was excluded as low for further accurate analysis of the data. The Cy3/Cy5 ratio of signal intensity of each spot was normalized by the median value of log₂(Cy3/Cy5) of all the spots to obtain the normalized ratio. The spots shown as grey dots on scattered plots in Fig. 3 indicate low value of either Cy3 or Cy5 signal less than 25% of all spots. Such spots were not significantly differentially expressed, thus were excluded in order to obtain reliable and reproducible results. From our microarray results we identified the differentially expressed genes as the ones with differences in their normalized ratio of the corresponding spots either greater or smaller than twofold. The genes under the VraSR-mediated regulation were selected by the comparison of microarray results between N315 exposed to VCM (Fig. 3C) and KVR exposed to VCM (Fig. 3D), if the gene is differentially expressed in N315 but not in KVR.

### Acknowledgements

This work was supported by Grant-in-Aid for Scientific Research on Priority Areas (13226114) from The Ministry of Education, Science, Sports, Culture and Technology of Japan and the Core University System Exchange Programme under the Japan Society for the Promotion of Science, co-ordinated by the University of Tokyo Graduate School of Medicine and Mahidol University. The study was also partly supported by the Research for the future Program of the Japan Society for the Promotion of Science. We are grateful to M. Kapi (Juntendo University) for critical reading.

### References

Berger-Bächi, B., and Rohrer, S. (2002) Factors influencing methicillin resistance in staphylococci. *Arch Microbiol* **178**: 165–171.

Boyle, V.J., Fancher, M.E., and Ross, R.W., Jr (1973) Rapid, modified Kirby-Bauer susceptibility test with single, high-concentration antimicrobial disks. Antimicrob Agents Chemother 3: 418–424.

Brandenberger, M., Tschierske, M., Giachino, P., Wada, A., and Berger-Bächi, B. (2000) Inactivation of a novel three-cistronic operon *tcaR-tcaA-tcaB* increases teicoplanin resistance in *Staphylococcus aureus*. Biochim Biophys Acta 1523: 135–139.

Brunskill, E.W., and Bayles, K.W. (1996) Identification and molecular characterization of a putative regulatory locus that affects autolysis in *Staphylococcus aureus*. J Bacteriol 178: 611–618.

Chambers, H.F., and Sachdeva, M. (1990) Binding of β-lactam antibiotics to penicillin-binding proteins in methicillin-resistant *Staphylococcus aureus*. J Infect Dis 161: 1170–1176.

Coulter, S.N., Schwan, W.R., Ng, E.Y., Langhorne, M.H., Ritchie, H.D., Westbrock-Wadman, S., et al. (1998) *Staphylococcus aureus* genetic loci impacting growth and survival in multiple infection environments. Mol Microbiol 30: 393–404.

Cui, L., Murakami, H., Kuwahara-Arai, K., Hanaki, H., and Hiramatsu, K. (2000) Contribution of a thickened cell wall and its glutamine nonamidated component to the vancomycin resistance expressed by *Staphylococcus aureus* Mu50. Antimicrob Agents Chemother 44: 2276–2285.

De Lencastre, H., Wu, S.W., Pinho, M.G., Ludovice, A.M., Filipe, S., Gardete, S., et al. (1999) Antibiotic resistance as a stress response: complete sequencing of a large number of chromosomal loci in *Staphylococcus aureus* strain COL that impact on the expression of resistance to methicillin. Microb Drug Resist 5: 163–175.

Dunman, P.M., Murphy, E., Haney, S., Palacios, D., Tucker-Kellogg, G., Wu, S., et al. (2001) Transcription profiling-based identification of *Staphylococcus aureus* genes regulated by the *agr* and/or *sarA* loci. J Bacteriol 183: 7341–7353.

Finan, J.E., Rosato, A.E., Dickinson, T.M., Ko, D., and Archer, G.L. (2002) Conversion of oxacillin-resistant staphylococci from heterotypic to homotypic resistance expression. Antimicrob Agents Chemother 46: 24–30.

Fournier, B., and Hooper, D.C. (2000) A new two-component regulatory system involved in adhesion, autolysis, and extracellular proteolytic activity of *Staphylococcus aureus*. J Bacteriol 182: 3955–3964.

Giraud, A.T., Cheung, A.L., and Nagel, R. (1997) The *sae* locus of *Staphylococcus aureus* controls exoprotein synthesis at the transcriptional level. Arch Microbiol 168: 53–58.

Hanaki, H., Kuwahara-Arai, K., Boyle-Vavra, S., Daum, R.S., Labischinski, H., and Hiramatsu, K. (1998) Activated cell-wall synthesis is associated with vancomycin resistance in methicillin-resistant *Staphylococcus aureus* clinical strains Mu3 and Mu50. J Antimicrob Chemother 42: 199–209.

Hiramatsu, K. (1998) Vancomycin resistance in staphylococci. Drug Resistance Updates 1: 135–150.

Hiramatsu, K. (2001) Vancomycin-resistant *Staphylococcus aureus*: a new model of antibiotic resistance. Lancet Infect Dis 1: 147–155.

Ito, T., Katayama, Y., and Hiramatsu, K. (1999) Cloning and nucleotide sequence determination of the entire *mec* DNA

of pre-methicillin-resistant *Staphylococcus aureus* N315. Antimicrob Agents Chemother 43: 1449–1458.

Katayama, Y., Ito, T., and Hiramatsu, K. (2000) A new class of genetic element, staphylococcus chromosome *mec*, encodes methicillin resistance in *Staphylococcus aureus*. Antimicrob Agents Chemother 44: 1549–1555.

Kobayashi, K., Ogura, M., Yamaguchi, H., Yoshida, K., Ogasawara, N., Tanaka, T., and Fujita, Y. (2001) Comprehensive DNA microarray analysis of *Bacillus subtilis* two-component regulatory systems. J Bacteriol 183: 7365–7370.

Komatsuzawa, H., Ohta, K., Labischinski, H., Sugai, M., and Suginaka, H. (1999) Characterization of *fmtA*, a gene that modulates the expression of methicillin resistance in *Staphylococcus aureus*. Antimicrob Agents Chemother 43: 2121–2125.

Kondo, N., Kuwahara-Arai, K., Kuroda-Murakami, H., Tateda-Suzuki, E., and Hiramatsu, K. (2001) Eagle-type methicillin resistance: new phenotype of high methicillin resistance under *mec* regulator gene control. Antimicrob Agents Chemother 45: 815–824.

Kuroda, M., Kuwahara-Arai, K., and Hiramatsu, K. (2000) Identification of the up- and down-regulated genes in vancomycin-resistant *Staphylococcus aureus* strains Mu3 and Mu50 by cDNA differential hybridization method. Biochem Biophys Res Commun 269: 485–490.

Kuroda, M., Ohta, T., Uchiyama, I., Baba, T., Yuzawa, H., Kobayashi, I., et al. (2001) Whole genome sequencing of meticillin-resistant *Staphylococcus aureus*. Lancet 357: 1225–1240.

Kuwahara-Arai, K., Kondo, N., Hori, S., Tateda-Suzuki, E., and Hiramatsu, K. (1996) Suppression of methicillin resistance in a *mecA*-containing pre-methicillin-resistant *Staphylococcus aureus* strain is caused by the *mecI*-mediated repression of PBP 2′ production. Antimicrob Agents Chemother 40: 2680–2685.

Lang, S., Livesley, M.A., Lambert, P.A., Littler, W.A., and Elliott, T.S. (2000) Identification of a novel antigen from *Staphylococcus epidermidis*. FEMS Immunol Med Microbiol 29: 213–220.

Lorenz, U., Ohlsen, K., Karch, H., Hecker, M., Thiede, A., and Hacker, J. (2000) Human antibody response during sepsis against targets expressed by methicillin resistant *Staphylococcus aureus*. FEMS Immunol Med Microbiol 29: 145–153.

MacMillan, S.V., Alexander, D.A., Culham, D.E., Kunte, H.J., Marshall, E.V., Rochon, D., and Wood, J.M. (1999) The ion coupling and organic substrate specificities of osmoregulatory transporter ProP in *Escherichia coli*. Biochim Biophys Acta 1420: 30–44.

Martin, P.K., Li, T., Sun, D., Biek, D.P., and Schmid, M.B. (1999) Role in cell permeability of an essential two-component system in *Staphylococcus aureus*. J Bacteriol 181: 3666–3673.

Mei, J.M., Nourbakhsh, F., Ford, C.W., and Holden, D.W. (1997) Identification of *Staphylococcus aureus* virulence genes in a murine model of bacteremia using signature-tagged mutagenesis. Mol Microbiol 26: 399–407.

Murakami, H., Matsumaru, H., Kanamori, M., Hayashi, H., and Ohta, T. (1999) Cell wall-affecting antibiotics induce

DNA microarray analyses reveal the regulatory network via VraSR two-component system 821

expression of a novel gene, drp35. Staphylococcus aureus. Biochem Biophys Res Commun 264: 348–351.

Novick, R.P., and Muir, T.W. (1999) Virulence gene regulation by peptides in staphylococci and other Gram-positive bacteria. Curr Opin Microbiol 2: 40–45.

Okonogi, K. (1990) [Mechanism of β-lactam-resistance in methicillin-resistant Staphylococcus aureus]. Rinsho Byori 38: 983–989.

Pinho, M.G., de Lencastre, H., and Tomasz, A. (1998) Transcriptional analysis of the Staphylococcus aureus penicillin binding protein 2 gene. J Bacteriol 180: 6077–6081.

Pinho, M.G., de Lencastre, H., and Tomasz, A. (2001) An acquired and a native penicillin-binding protein cooperate in building the cell wall of drug-resistant staphylococci. Proc Natl Acad Sci USA 98: 10886–10891.

Ryffel, C., Strassle, A., Kayser, F.H., and Berger-Bächi, B. (1994) Mechanisms of heteroresistance in methicillin-resistant Staphylococcus aureus. Antimicrob Agents Chemother 38: 724–728.

Suzuki, E., Kuwahara-Arai, K., Richardson, J.F., and Hiramatsu, K. (1993) Distribution of mec regulator genes in methicillin-resistant Staphylococcus clinical strains. Antimicrob Agents Chemother 37: 1219–1226.

Wang, Q.M., Peery, R.B., Johnson, R.B., Alborn, W.E., Yeh, W.K., and Skatrud, P.L. (2001) Identification and characterization of a monofunctional glycosyltransferase from Staphylococcus aureus. J Bacteriol 183: 4779–4785.

Whatmore, A.M., and Reed, R.H. (1990) Determination of turgor pressure in Bacillus subtilis: a possible role for K⁺ in turgor regulation. J Gen Microbiol 136: 2521–2526.

Wiltshire, M.D., and Foster, S.J. (2001) Identification and analysis of Staphylococcus aureus components expressed by a model system of growth in serum. Infect Immun 69: 5198–5202.

Yarwood, J.M., McCormick, J.K., Schlievert, P.M. (2001) Identification of a novel two-component regulatory system that acts in global regulation of virulence factors of Staphylococcus aureus. J Bacteriol 183: 1113–1123.
