
Molecular Microbiology (2003) 49(4), 1135–1144

doi:10.1046/j.1365-2958.2003.03653.x

# The BceRS two-component regulatory system induces expression of the bacitracin transporter, BceAB, in Bacillus subtilis

Reiko Ohki,$^{1*†}$Giyanto,$^{2†}$ Kozue Tateno,$^{1}$  
Waka Masuyama,$^{2}$ Shigeki Moriya,$^{2}$ Kazuo Kobayashi$^{2}$  
and Naotake Ogasawara$^{2}$  

$^{1}$ Department of Molecular Biology, School of Health Sciences, Kyorin University, 476 Miyashita, Hachiouji, Tokyo 192-0005, Japan.  
$^{2}$ Department of Bioinformatics and Genomics, Graduate School of Information Science, Nara Institute of Science and Technology, Ikoma, Nara 630-0101, Japan.

## Summary

BceA and bceB encode a nucleotide-binding domain (NBD) and membrane-spanning domain (MSD) subunit, respectively, of an ATP-binding cassette (ABC) transporter in *Bacillus subtilis*. Disruption of these genes resulted in hypersensitivity to bacitracin, a peptide antibiotic that is non-ribosomally synthesized in some strains of *Bacillus*. Northern hybridization analyses showed that expression of the bceAB operon is induced by bacitracin present in the growth medium. The bceRS genes encoding a two-component regulatory system are located immediately upstream of bceAB. Deletion analyses of the bceAB promoter together with DNase I footprinting experiments revealed that a sensor kinase, BceS, responds to extracellular bacitracin either directly or indirectly and transmits a signal to a cognate response regulator, BceR. The regulator binds directly to the upstream region of the bceAB promoter and upregulates the expression of bceAB genes. The bcrC gene product is additionally involved in bacitracin resistance. The expression of bcrC is dependent on the ECF σ factors, σ<sup>M</sup> and σ<sup>X</sup>, but not on the BceRS two-component system. In view of these results, possible roles of BceA, BceB and BcrC in bacitracin resistance of *B. subtilis* 168 are discussed.

## Introduction

Bacteria are equipped with devices to cope with external and internal stresses, including toxic compounds produced by other bacteria or themselves. One of the mechanisms of acquiring drug resistance involves the exclusion of compounds by membrane-bound transporter proteins. A number of drug efflux transporters are specific for a given drug or class of drugs, whereas others designated ‘multidrug resistance transporters’ (MDR) display specificity for various compounds with different chemical structures. Transporters are classified into several groups, depending on sequence similarity (Levy, 1992; Paulsen et al., 1996; Putman et al., 2000). The ATP-binding cassette (ABC) transporter family is one of the most abundant of these groups (Fath and Kolter, 1993). The LmrA protein of *Lactococcus lactis* is the earliest identified MDR efflux transporter of the ABC family in bacteria (van Veen et al., 1996). LmrA pumps out various drugs using a similar mechanism to that proposed for P-glycoprotein of mammalian MDR transporters (van Veen et al., 2000). However, few ABC-type MDR transporters in bacteria have been functionally characterized to date.

Antibiotic expulsion, mediated by a membrane-associated drug efflux transporter, is also a major self-protecting mechanism devised by antibiotic-producing bacteria (Cundliffe, 1989). Drug efflux transporters belonging to the ABC family are frequently involved in these systems (Méndez and Salas, 2001). In bacteria, the biosynthetic operons of peptide antibiotics, such as subtilin, nisin, mersacidin and bacitracin, are associated with genes encoding efflux transporters of the ABC family, which pump out these antibiotics for self-protection (Klein and Entian, 1994; Podlesek et al., 1995; De Ruyter et al., 1996; Altena et al., 2000). Furthermore, genes encoding a two-component signal transduction system, the major mechanism for sensing environmental signals in bacteria (Stock et al., 2000), are commonly located adjacent to the efflux transporter genes and regulate their expression (De Ruyter et al., 1996; Altena et al., 2000; Neumüller et al., 2001; Stein et al., 2002). In the Gram-positive bacterium *Bacillus subtilis*, 40 operons are predicted to encode efflux transporters of the ABC family (Quentin et al., 1999). Analysis of their genomic location revealed that eight of these operons are located adjacent to genes encoding the two-component regulatory system (Kunst et al., 1997). In view of this finding, we examined whether the ABC trans-

Accepted 3 June, 2003. *For correspondence. E-mail ohkir@kyorin-u.ac.jp; Tel. (+81) 426 91 0011; Fax (+81) 426 91 1094.  
†These authors contributed equally to this work.

© 2003 Blackwell Publishing Ltd

porters and two-component regulatory systems are functionally related. We compared the resistance of the wild-type strain and disruptants of ABC transporter genes to various compounds. Our data show that *bceA* and *bceB* (formerly denoted *ytsC* and *ytsD*), which encode the nucleotide-binding domain (NBD) and membrane spanning domain (MSD) subunit of the ABC transporter, respectively, are involved in resistance to bacitracin, a peptide antibiotic non-ribosomally synthesized in specific strains of *Bacillus* (Johnson *et al.*, 1945; Azevedo *et al.*, 1993). We additionally demonstrate that *bceRS* (formerly known as *ytsAB*) genes of the two-component regulatory system, located immediately upstream of *bceAB*, control expression of the *bceAB* operon by sensing extracellular bacitracin.

Recently, we (Ohki *et al.*, 2003) and Cao and Helmann (2002) showed that the *bcrC* gene (formerly designated *ywoA*), which encodes a homologue of the possible MSD subunit of the *Bacillus licheniformis* ABC transporter for bacitracin efflux (*bcrC*), is involved in bacitracin resistance in *B. subtilis* 168. Expression of *bcrC* is dependent on ECF σ factors, σ<sup>M</sup> and σ<sup>X</sup>, but not on the BceRS two-component system. Based on these results, possible roles of BceAB and BcrC in bacitracin resistance of *B. subtilis* 168 are discussed.

### Results

#### An ABC transporter, BceAB, and a two-component regulatory system, BceRS, are involved in bacitracin resistance in *B. subtilis*

Among 40 possible ABC efflux transporter operons in *B. subtilis*, eight (*ycbN*, *natAB*, *ycllH*, *yfiLMN*, *ytsCD*, *yvcRS*, *yvfRS* and *yxdLM*) are located next to genes encoding the two-component regulatory system (Kunst *et al.*, 1997). Seven of these transporters (except *natAB* encoding the Na<sup>+</sup> extrusion transporter) and the associated two-component regulatory systems have not yet been functionally characterized (Fabret *et al.*, 1999). To determine the functions of these genes, we compared the resistance of the wild-type strain and pMutin disruptants of ABC transporter genes to various drugs. Mutants of *ytsC* and *ytsD* encoding NBD and MSD subunits, respectively, displayed hypersensitivity to bacitracin, as shown in Table 1. The *bcrC* gene product is additionally involved in bacitracin resistance in *B. subtilis* 168 (Cao and Helmann, 2002; Ohki *et al.*, 2003). Disruptants of *ytsC* or *ytsD* were more sensitive to bacitracin, compared with a *bcrC* mutant.

The *ytsC* and *ytsD* genes constitute an operon. Therefore, it is necessary to confirm that the bacitracin-sensitive phenotype of YTSCd results from inactivation of the *ytsC* gene, and not the polar effect of the pMutin insertion on downstream *ytsD* expression. In pMutin dis-

**Table 1. Bacitracin sensitivity of various mutants.**

| Strain | Genotype | Relative resistance<sup>a</sup> |
|--------|----------|-----------------------------|
| 168    |          | 1                           |
| BCRCd  | *bcrC*::pMutin2 | 0.025                   |
| BCERd  | *bceR(ytsA)*<sup>b</sup>::pMutin2 | 0.013                   |
| BCESd  | *bceS(ytsB)*::pMutin2 | 0.009                   |
| BCEAd  | *bceA(ytsC)*::pMutin2 | 0.007                   |
| BCEBd  | *bceB(ytsD)*::pMutin2 | 0.007                   |

a. Relative resistance was determined by dividing the IC<sub>50</sub> for mutant strains by that for the wild-type strain 168. The IC<sub>50</sub> value for strain 168 is 39.9 U ml<sup>-1</sup>.

b. Names used formerly are shown in parenthesis.

ruptants, downstream genes are placed under the control of the *spac* promoter, which is induced by the addition of IPTG (Vagner *et al.*, 1998). Northern hybridization analyses using total RNA isolated from YTSCd grown in the presence or absence of IPTG demonstrated that a transcript detected with a probe specific for *ytsD* increased 25-fold with IPTG (*Supplementary material*, Fig. S1A). However, bacitracin resistance was not increased upon addition of IPTG, indicating that YtsC is required for bacitracin resistance (data not shown). Moreover, disruption of these genes had no effect on resistance to the other drugs examined, including vancomycin, tunicamycin, surfactin, iturin A and several cationic dyes (data not shown).

The *ytsA* and *ytsB* genes, located immediately upstream of *ytsCD*, encode a response regulator and a histidine sensor kinase, respectively, of the two-component regulatory system. Disruption of either *ytsA* or *ytsB*, which constitute an operon, caused a similar decrease in bacitracin resistance to that observed in *ytsC* and *ytsD* mutants (Table 1). In YTSAd, the addition of IPTG resulted in a more than 10-fold increase in the amount of 1.7 kb transcript detected with a probe specific for *ytsB* (*Supplementary material*, Fig. S1B), but had no effect on bacitracin resistance (data not shown).

The results collectively suggest that *ytsC* and *ytsD* constitute subunits of a bacitracin efflux transporter, whereas *ytsA* and *ytsB* are involved in the regulation of bacitracin resistance. Accordingly, we designated *ytsC* and *ytsD* as *bceA* and *bceB* (bacitracin efflux), and *ytsA* and *ytsB* as *bceR* and *bceS* respectively.

#### Identification of the *bceRS* and *bceAB* transcriptional units and induction of *bceAB* expression by external bacitracin

It is predicted that *bceRS* and *bceAB* genes are transcribed in the same direction, and the open reading frames (ORFs) of *bceR* and *bceS*, as well as of *bceA* and *bceB*, overlap by two and three codons, respectively, as shown schematically in Fig. 1E. Northern blot experiments using total RNA isolated from wild-type cells

© 2003 Blackwell Publishing Ltd, *Molecular Microbiology*, **49**, 1135–1144

Bacitracin-resistant genes in B. subtilis 1137

A
bacitracin:
- + 
1 2 3 4 5 6

probe:  
bceR

B
bacitracin:
- +
1 2 3 4 5 6

probe:  
bceS

C
bacitracin:
- +
1 2 3 4 5 6

2.7kb

probe:  
bceA

D
bacitracin:
- +
1 2 3 4 5 6 M

probe:  
bceB

E

bceR  
(231aa)

bceS  
(334aa)

bceA  
(253aa)

bceB  
(646aa)

Fig. 1. Induction of bceAB operon expression by preincubation with 10 U ml⁻¹ bacitracin. An overnight culture of *B. subtilis* 168 grown in LB medium was inoculated into LB medium (OD₅₃₀, 0.05) with or without 10 U ml⁻¹ bacitracin. Samples were withdrawn during early (OD₅₃₀, 0.5; lanes 1 and 4), middle (OD₅₃₀, 1.2; lanes 2 and 5) and late (OD₅₃₀, 2.0; lanes 3 and 6) log phase growth. Isolation of total RNA and Northern hybridization were performed as described in Experimental procedures. ³²P-labelled bceR, bceS, bceA and bceB probes were used as indicated. Arrows indicate 1.7 kb bceRS (A and B) and 2.7 kb bceAB transcripts (C and D). Lanes 1–3, LB medium; lanes 4–6, LB medium containing 10 U ml⁻¹ bacitracin; M, molecular size standard (Novagen Perfect RNA™ markers 0.1–1 kb, Invitrogen RNA ladder 0.24–9.5 kb). E. Thick arrows signify open reading frames (ORFs) of bceR, bceS, bceA and bceB. Thin arrows indicate promoters, and the circle on a stem represents a transcriptional termination signal. Boxes under ORFs specify the probes used for Northern hybridization.

revealed a 1.7 kb band with probes specific for bceR or bceS and a 2.7 kb band with probes specific for bceA or bceB (Fig. 1). Levels of both transcripts were low during all the growth phases examined.

The addition of 40 U ml⁻¹ bacitracin to LB inhibited the growth of wild-type cells by about 50%. An approximately twofold increase in bacitracin resistance was observed with wild-type cells grown in LB medium containing 10 U ml⁻¹ bacitracin without apparent growth inhibition (Ohki et al., 2003). As shown in Fig. 1C and D, bceAB mRNA levels increased by more than 50-fold upon addition of 10 U ml⁻¹ bacitracin to the growth medium. This finding suggests that the increase in bacitracin resistance is induced as a result of increased bceAB expression. Conversely, bceRS mRNA levels were decreased under the same growth conditions (Fig. 1A and B).

Overexpression of BceR in a mutant of the sensor kinase gene, bceS (BCESd), results in increased expression of the bceAB operon and restores bacitracin resistance.

Overproduction of response regulators in the absence of cognate sensor kinases may result in the constitutive expression of regulator target genes (Kobayashi et al., 2001; Ogura et al., 2001). We transformed wild-type and bceS-disrupted (BCESd) strains with a multicopy plasmid, pDG148bceR, expressing bceR from the IPTG-inducible spac promoter (Kobayashi et al., 2001). Northern hybridization analyses showed that, in the absence of IPTG, the cellular level of bceAB mRNA was low in both wild-type and BCESd strains harbouring pDG148bceR. Overexpression of bceR after IPTG addition led to a 50-fold increase in bceAB mRNA in both strains (Supplementary material, Fig. S2). In contrast, no changes were observed

© 2003 Blackwell Publishing Ltd, Molecular Microbiology, 49, 1135–1144

1138 R. Ohki et al.

Table 2. Bacitracin resistance of $bceS$-disrupted strains harbouring a plasmid containing $bceR$.

| Strain           | IPTG | Relative resistance${}^{a}$ |
|------------------|------|-----------------------------|
| 168              | –    | 1                           |
| BCESd/pDG148     | –    | 0.006                       |
|                  | +    | 0.012                       |
| BCESd/pDG148$bceR$ | –    | 0.007                       |
|                  | +    | 2.01                        |

a. Relative resistance was determined by dividing the IC${}_{50}$ for mutant strains by that for wild-type strain 168. The IC${}_{50}$ value for strain 168 is 39.9 U ml${}^{-1}$.

in the $bceRS$ mRNA level in wild-type cells harbouring pDG148$bceR$ after IPTG addition. This finding strongly suggests that the BceRS two-component system regulates BceAB transporter expression. Consistently, a bacitracin-hypersensitive phenotype of BCESd was suppressed upon overexpression of BceR in cells harbouring pDG148$bceR$ (Table 2).

The cis-acting sequence is required for $bceAB$ expression

To analyse further the regulation of $bceAB$ expression by the BceRS two-component system, we determined the 5' end of the $bceAB$ transcript by primer extension analysis. Using total RNA from $B$. subtilis 168 cells grown in the presence of 10 U ml${}^{-1}$ bacitracin and 168 cells harbouring pDG148$bceR$ grown in the presence of IPTG, we identified the transcription start site at 26 bp upstream of the initiation codon for $bceA$ (Fig. 2). Consistent with $bceAB$ transcript expression from early log phase growth (Fig. 1C and D), a –10 consensus sequence for $\sigma^A$, TATAAT, was detected 6 bp upstream of the transcription start site. On the other hand, no clear –35 sequence was observed at the appropriate position from the –10 sequence, which is typical of promoters regulated by transcriptional activators (see Fig. 4C).

Next, to determine the $cis$-acting region necessary for $bceAB$ transcription, we constructed a series of transcriptional fusion strains with progressively shorter $bceAB$ promoter sequences fused to the $lacZ$ reporter gene at the $amyE$ locus on the $B$. subtilis chromosome (Fig. 3). Strains were grown with or without 10 U ml${}^{-1}$ bacitracin to early log phase. The $\beta$-galactosidase (LacZ) activities of cells grown in the absence of bacitracin were at background levels for all the constructs. However, in cells containing constructs that included at least 81 bp of the upstream region of the $bceAB$ transcription start site, LacZ activity was strongly induced upon addition of bacitracin to the culture medium. Interestingly, deletion of a further 18 bp from the 81 bp sequence completely abolished induction. We also monitored the effect of BceR overexpression in these strains, after the transfection of the pDG148$bceR$ plasmid. Measurement of LacZ activities of

cells harbouring pDG148$bceR$ at exponential growth phase in the presence of IPTG revealed that BceR overproduction also induces $bceAB$ promoter activity if the 81 bp sequence is retained. These results indicate that a sequence between –81 and –63 bp upstream of the $bceAB$ transcription initiation site is essential for the induction of $bceAB$ transcription by bacitracin. Moreover, transcription is dependent on the binding of the BceR regulator to this site.

The BceR regulator binds directly upstream of the $bceAB$ promoter

To confirm that BceR indeed binds to the $cis$-acting sequence required for $bceAB$ expression, the protein was fused to glutathione S-transferase (GST) at the N-terminus, expressed in *Escherichia coli* and purified to homogeneity using glutathione Sepharose (data not shown). Gel shift experiments showed that the purified protein specifically binds to a DNA fragment containing a sequence from –373 bp to +78 bp relative to the $bceAB$ transcription start site (data not shown). We performed DNase I footprinting experiments to map the BceR binding

![Figure 2](https://i.imgur.com/yourimage.png)

Fig. 2. Determination of the $bceAB$ transcription start site by primer extension. Total RNA was prepared from $B$. subtilis 168 cells grown in LB medium ($OD_{600}$, 0.5) without (lane 1) and with (lane 2) bacitracin, and from 168 cells harbouring pDG148$bceR$ (lane 3) grown in LB medium with IPTG ($OD_{600}$, 0.5) and used for primer extension analyses. Sequencing ladders (lanes T, G, C and A) were generated with the primer used for the reverse transcriptase reaction. The sequence and transcriptional start site (indicated by an arrow) are shown.

site precisely. A 279 bp fragment from –200 to +78 bp relative to the bceA transcription start site was labelled radioactively at either end and incubated with increasing amounts of purified GST-BceR protein, followed by partial digestion with DNase I. The results indicate that the cis-regulatory region for bceAB induction identified by promoter deletion experiments is protected by GST-BceR in both coding and non-coding strands (Fig. 4A and B). Regulatory elements for the induction of bceAB transcription are summarized in Fig. 4C. Notably, an inverted repeat sequence, AAGCgTGTGACgaaatGTCACAtGCTT, overlapping the region protected by DNase I, may be a recognition sequence for BceR.

Expression of bcrC is not regulated by the BceRS two-component regulatory system, and bceA–bcrC, bceB–bcrC and bcrC–bceR double mutants are more sensitive to bacitracin than individual single mutants

Disruption of bcrC, which encodes a homologue of the presumed MSD subunit of the Bacillus licheniformis bacitracin transporter, BcrC, results in hypersensitivity to bacitracin. Expression of bcrC is dependent on σM and σX (Cao and Helmann, 2002; Ohki et al., 2003). If BcrC, BceA and BceB constitute the bacitracin efflux transporter, expression may additionally be modulated by the BceRS two-component regulatory system. However, Northern hybridization analyses revealed that bcrC mRNA expression in wild-type and BCESd strains harbouring pDG148bceR are not affected by IPTG (Fig. 5), indicating that the BceRS two-component regulatory system is not involved in this process.

Next, we examined whether the combined absence of BcrC, BceA and BceB has an additive effect on bacitracin resistance by generating double mutants of BCEAd/bcrC::tet and BCEBd/bcrC::tet. As shown in Table 3, bacitracin resistance was further decreased in double

© 2003 Blackwell Publishing Ltd, Molecular Microbiology, 49, 1135–1144

Fig. 3. Mapping of the cis-acting sequence required for the induction of bceAB expression. Transcriptional fusion strains corresponding to various bceAB promoter regions and lacZ at the amyE locus on the B. subtilis chromosome were constructed, as shown schematically. The pDG148bceR plasmid was introduced into these strains. β-Galactosidase activities are shown for strains without the plasmid and cultivated in the absence (−) or presence of bacitracin (+bacitracin). Activities of the plasmid-containing strains cultivated in the presence of IPTG (+pDG148bceR) are listed on the right. Cells without the plasmid were harvested at OD600 of 0.3 and plasmid-containing cells at OD600 of 0.5.

| Strain | β-galactosidase activity (Unit) |
| --- | --- |
|  | −bacitracin | +bacitracin | +pDG148bceR |
| BSGY001 | 25 | 25 | 123 |
| BSGY002 | 27 | 35 | 119 |
| BSGY003 | 22 | 38 | 129 |
| BSGY004 | 22 | 28 | 126 |
| BSGY005 | 14 | 4975 | 11123 |
| BSGY006 | 21 | 4808 | 10307 |
| BSGY007 | 21 | 4050 | 9619 |
| BSGY008 | 20 | 3225 | 10567 |

mutants, compared with single mutants. Moreover, bacitracin resistance in a double mutant, bcrC::tet/bceR::cat, decreased to one-tenth that in bceR::cat (data not shown). These results suggest that BcrC and BceAB are involved in different mechanisms of bacitracin resistance in B. subtilis.

Discussion

Bacitracin is non-ribosomally synthesized in some strains of Bacillus (Johnson et al., 1945; Azevedo et al., 1993) and has potent activity against Gram-positive bacteria (Toscano and Storm, 1982). In this study, we identified four B. subtilis genes involved in bacitracin resistance, specifically bceA, bceB, bceR and bceS. The amino acid sequences of these proteins suggest that BceA and BceB are NBD and MSD subunits, respectively, of an ABC transporter, whereas BceR is a response regulator, and BceS is a sensor histidine kinase of a two-component regulatory system. This study is the first report showing that, in antibiotic non-producing bacteria, expression of the ABC transporter involved in antibiotic resistance is regulated by a two-component transduction system that senses the presence of antibiotic in the extracellular medium.

Table 3. Bacitracin sensitivity of bceA/bcrC and bceB/bcrC double mutants.

| Strain | Relative resistance<sup>a</sup> |
| --- | --- |
| bcrC::tet | 1 |
| BCEAd | 0.28 |
| BCEBd | 0.28 |
| BCEAd/bcrC::tet | 0.04 |
| BCEBd/bcrC::tet | 0.05 |

a. Relative resistance was determined by dividing the IC50 for mutant strains by that for bcrC::tet. IC50 value for strain bcrC::tet is 0.68 U ml<sup>−1</sup>.

1140 R. Ohki et al.

bacitracin is 55C-isoprenyl pyrophosphate, a carrier of a peptidoglycan unit or a disaccharide with pentapeptide in the membrane. The *B. subtilis* efflux transporter might use a mechanism similar to the hydrophobic vacuum cleaner model in which the efflux transporter removes bacitracin directly from the lipid bilayer to the extracellular medium (Bolhuins *et al.*, 1996). However, the exact mechanism by which the BceAB efflux transporter removes bacitracin from the target site in the membrane remains to be elucidated.

We demonstrate that the sensor kinase, BceS, responds to extracellular bacitracin and activates the response regulator, BceR, which binds specifically upstream of the *bceAB* operon to induce transcription of *bceAB*. An unphosphorylated protein was used in DNase I footprinting experiments. Protection was apparent at high concentrations of protein (more than 500-fold excess with respect to DNA), suggesting that BceR phosphorylation increases affinity for DNA and is required for responses in wild-type cells. However, there is currently no direct evidence on the role of BceR phosphorylation. Recently, Joseph *et al.* (2002) reported that expression of the *B. subtilis ytsCD* (*bceAB*) transporter genes was activated by overexpression of *ytsA* (*bceR*), although the biological roles of these genes were not explored in detail.

Notably, no self-induction of *bceRS* expression was observed upon addition of bacitracin in growth medium or overexpression of BceR. However, a decrease in the level of *bceRS* mRNA was observed after incubation with bacitracin. The mechanism of this effect of bacitracin requires further clarification.

Our group (Ohki *et al.*, 2003) and that of Cao and Helmann (2002) reported that disruption of the *bcrC* gene in *B. subtilis* 168 results in hypersensitivity to bacitracin and that transcription of *bcrC* is dependent on the ECF σ factors, σ<sup>M</sup> and σ<sup>X</sup>. Here, we show that *bcrC* expression is independent of the BceRS two-component system. Cao and Helmann (2002) observed the σ<sup>M</sup>-dependent induction of *bcrC* expression and increased resistance to bacitracin between 2 and 30 min after addition. Significantly, no changes in *bcrC* mRNA levels were observed after longer incubation periods (more than 2 h) with bacitracin, although increased resistance to bacitracin was maintained (Ohki *et al.*, 2003). These results indicate that the σ<sup>M</sup>-dependent response of *bcrC* expression is transient. In contrast, the data presented here imply that induction of BceAB expression through the BceRS signal transduction system is consistent. Thus, increased resistance observed in our previous study may have resulted mainly from increased expression of *bceAB* genes.

The BcrC protein displays 28% identity to BcrC of *B. licheniformis* and contains seven transmembrane helices. It is proposed that the *bcrC* gene of *B. licheniformis* encodes an MSD subunit of the bacitracin efflux trans-

Fig. 4. DNase I footprinting assays of the GST-BceR binding to the *bceAB* promoter region. A 279 bp fragment encompassing –200 to +78 bp relative to the *bceA* transcription start site was labelled radioactively at either end and used as coding (A) and non-coding (B) strand probes. A constant amount of the probe (300 000 c.p.m., 9.25 fmol) was incubated with different amounts of GST-BceR. Protein amounts in lanes 1–4 are 0, 5, 10 and 15 pmol respectively. Lanes A, C, G and T contain sequencing ladders. Sequences of the maximum protected regions in the coding and non-coding strands are shown.

C. The nucleotide sequence of the promoter region for *bceAB* transcription is depicted. The bent arrow indicates the transcription start site detected using primer extension (Fig. 2). The initiation codon of *bceA* and termination codon of *bceS* are boxed. Coding regions for *bceS* and *bceA* are underlined. The ribosome binding sequence (RBS) for *bceA* and the possible –10 sequence of the *bceAB* promoter are double underlined. Numbers (relative to the *bceAB* transcription start site) at the top indicate the end-point of promoter sequences used in deletion analyses (Fig. 3). DNase I-protected areas (A and B) are specified by brackets. Thick arrows signify an inverted repeat sequence in the protected region.

Bacitracin-producing *B. licheniformis* contains an ABC transporter that is assumed to mediate bacitracin efflux for self-protection (Podlesek *et al.*, 1995). Furthermore, expression of these proteins is regulated by a two-component system consisting of BacS and BacR (Neumüller *et al.*, 2001). In contrast to BceR of *B. subtilis*, BacR negatively regulates the expression of *bcrAB* in *B. licheniformis*. Additionally, *B. subtilis* BceRS and BceAB are not closely related in structure to the bacitracin-sensing and -extruding proteins of *B. licheniformis*. The target site of

© 2003 Blackwell Publishing Ltd, *Molecular Microbiology*, **49**, 1135–1144

Fig. 5. Expression of *bcrC* in wild-type and *bceS*-disrupted mutant strains harbouring a plasmid containing *bceR*. *B. subtilis* 168 and BCESd strains harbouring pDG148*bceR* were grown in LB medium with or without 0.5 mM IPTG to early (*OD*~530~, 0.5; lanes 1 and 4), middle (*OD*~530~, 1.2; lanes 2 and 5) and late (*OD*~530~, 2.0; lanes 3 and 6) log phase. Total RNA (5 μg) was loaded per lane. A ^32^P-labelled *bcrC* probe was used. The arrow signifies a 0.6 kb band corresponding to *bcrC* transcripts.

porter, despite the low amino acid sequence homology to known MSD subunits of ABC transporters. This assumption is largely based on the fact that *bcrC* constitutes an operon with *bcrA* and *bcrB* genes encoding typical NBD and MSD subunits, respectively, of an ABC transporter (Podlesek *et al.*, 1995; Neumüller *et al.*, 2001). However, the distinct regulatory mechanisms for *bceAB* and *bcrC* expression in *B.* *subtilis* suggest that these proteins contribute independently to bacitracin resistance. In support of this hypothesis, *bceA/bcrC* and *bceB/bcrC* double mutants are more sensitive to bacitracin than individual single mutants. In this respect, it is interesting that BcrC belongs to the PAP2 superfamily of phosphatases in the Pfam database (Bateman *et al.*, 2002) and is classified as a membrane-associated phospholipid phosphatase in the COG database (http://www.ncbi.nlm.nih.gov/cgi-bin/COG/). Further studies are required to determine the precise role of BcrC in bacitracin resistance.

Tsuda *et al.* (2002) reported recently that disruption of the homologous genes in *Streptococcus mutans* resulted in a bacitracin-sensitive phenotype. Disruption of *mbrA* and *mbrB* genes, which encode the NBD and MSD subunits, respectively, of the ABC transporter, resulted in an approximately 100-fold increase in bacitracin sensitivity. The *mbrC* and *mbrD* genes located immediately downstream of *mbrAB* encode a response regulator and a sensor kinase, respectively, of a two-component regulatory system. Disruption of either *mbrC* or *mbrD* additionally causes a decrease in bacitracin resistance similar to that observed in *mbrA* and *mbrB* mutants.

Various peptide antibiotics are synthesized in different bacteria. These organisms need to evolve defence systems against antibiotics produced by themselves or others. Clusters of the two-component system and ABC transporter genes that are similar to the *B.* *subtilis bceRS*

and *bceAB*, respectively, are widespread in the *Bacillus/Clostridium* group (Joseph *et al.*, 2002). Many of these genes may be involved in defence against peptide antibiotics. Identification of the substrates sensed and excluded by these molecules is important for our understanding of the evolution of bacterial communities.

### Experimental procedures

#### Primers

The PCR primers used in this study are listed in Supplementary material (Table S1).

#### Construction of mutant strains and plasmids

The pMutin insertion mutants, BCESd, BCEAd, BCEBd and BCRCd, were constructed by members of the Japan and European Union consortia of *B.* *subtilis* functional genomics (Kobayashi *et al.*, 2003). The *bceR::pMutin2* (BCERd) and *bceR::cat* mutants were constructed in this study. To generate double mutants of *bceR* and *bcrC* (BCRCd/*bceR::cat*), the *bceR* gene was replaced with a chloramphenicol-resistant gene (*cat*). Next, BCRCd was transformed with chromosomal DNA prepared from *bceR::cat*, and transformants were selected for chloramphenicol and erythromycin resistance. To obtain double mutants BCEAd/*bcrC::tet* and BCEBd/*bcrC::tet*, an erythromycin-resistant gene of BCRCd was disrupted by the insertion of a tetracycline-resistant gene (*bcrC::tet*) and transformed with chromosomal DNA prepared from BCEAd and BCEBd with selection for erythromycin and tetracycline resistance. pDG148*bceR* (*ytsA*) was used to overexpress BceR in *B.* *subtilis* cells, as described previously (Kobayashi *et al.*, 2001).

To construct strains with transcriptional fusion of the gradually deleted *bceAB* promoter and the *lacZ* reporter gene at the *amyE* locus on the *B.* *subtilis* chromosome, pDL2 (Fukuchi *et al.*, 2000) was used. The eight polymerase chain reac-

tion (PCR) products obtained with a reverse primer, GYR11, and forward primers, GYF03, GYF04, GYF05, GYF06, GYF07, GYF08, GYF09 and GYF10, were digested at the EcoRI and BamHI sites incorporated within the primers and cloned between the corresponding sites of pDL2 to generate pGY001, pGY002, pGY003, pGY004, pGY005, pGY006, pGY007 and pGY008 respectively. The pDL2 derivatives were linearized by PstI digestion and used to transform *B. subtilis* 168 cells by double cross-over with selection for chloramphenicol resistance to obtain BSGY001, BSGY002, BSGY003, BSGY004, BSGY005, BSGY006, BSGY007 and BSGY008 respectively.

To express the BceR regulator (from the initiation to the termination codon) in *E. coli* as a fusion with the GST protein at the N-terminus, the entire coding sequence of *bceR* was amplified by PCR using GYF12 and GYR13 primers. The amplified fragment was digested at the EcoRI and SalI sites within the primers and inserted between the corresponding sites of the pGEX-4T-1 plasmid (Amersham Pharmacia Biotech) to obtain pGEX-*bceR*.

### Culture conditions

Bacterial strains were cultured in LB medium (10 g l⁻¹ polypeptone, 5 g l⁻¹ yeast extract, 5 g l⁻¹ NaCl, pH 7.2). Where necessary, IPTG (0.5 mM) and antibiotics were added to medium, specifically, ampicillin (100 μg ml⁻¹), chloramphenicol (20 μg ml⁻¹ for *E. coli*, 5 μg ml⁻¹ for *B. subtilis*), kanamycin (20 μg ml⁻¹ for *E. coli*, 10 μg ml⁻¹ for *B. subtilis*) and erythromycin (100 μg ml⁻¹ for *E. coli*, 1 μg ml⁻¹ for *B. subtilis*). Transformation of *B. subtilis* cells was performed as described previously (Moriya et al., 1998).

### Northern hybridization analysis

Total RNA was isolated from cells grown in LB medium to early (*OD*~530~, 0.5), middle (*OD*~530~, 1.2) or late (*OD*~530~, 2.0) log phase. Cells were resuspended in SET buffer (20% sucrose, 50 mM EDTA, 50 mM Tris-HCl, pH 7.6) containing 2 mg ml⁻¹ lysozyme and incubated on ice for 10 min. RNA was extracted with hot phenol according to the procedure of Aiba et al. (1981). Total RNA (5 μg) was electrophoresed on a 1.5% formaldehyde–agarose gel and blotted onto a nylon membrane. DNA fragments amplified by PCR using the following primers were used as probes: *bceS*, BceSF, BceSR; *bceR*, BceRF, BceRR; *bceA*, BceAF, BceAR; *bceB*, BceBF, BceBR; *bcrC*, BcrCF, BcrCR. Probes were labelled with [α-³²P]-dCTP using the Multiprime labelling system (Amersham Pharmacia Biotech). Hybridization was performed at 42°C as described previously (Ohki et al., 2003).

### Assay of the bacitracin-resistant phenotype

Bacitracin (Sigma; 70 U mg⁻¹) resistance was assayed in LB medium, as described in an earlier report (Ohki et al., 2003). To calculate the relative resistance for various mutants, the concentration of bacitracin leading to 50% inhibition of growth (*IC*~50~) was expressed as a ratio, relative to the *IC*~50~ value in the control strain.

### Primer extension analysis

Total RNA was extracted from *B. subtilis* 168 cells grown in LB medium (*OD*~600~, 0.5) with and without bacitracin, and from 168 cells harbouring pDG148*bceR* grown in LB medium with IPTG (*OD*~600~, 0.5), using the method of Igo and Losick (1986). The GYR02 primer was labelled with [γ-³²P]-ATP (Amersham Pharmacia Biotech) using T4 polynucleotide kinase (Takara Shuzo). RNA (10 μg) was incubated with the end-labelled primer (0.5 pmol) for 60 min at 60°C. Reverse transcription was performed for 60 min at 42°C using a Superscript reverse transcriptase, according to the manufacturer’s instructions (Gibco BRL). The cDNA products were electrophoresed through an 8% polyacrylamide–urea gel. Radioactive bands were detected with an imaging plate and a BAS2500 scanner (Fuji Photo Film). DNA ladders for use as size markers were created with the same end-labelled primer and a cycle sequencing kit (Takara Shuzo). Primers GYF01 and GYR02 were used to amplify template DNA for the sequencing reaction.

### β-Galactosidase assay

*Bacillus subtilis* cells in exponential phase of growth were collected by centrifugation, and β-galactosidase activity was assayed according to the method of Youngman et al. (1985). One unit was defined as 1 pmol of 4-methylumbelliferyl β-D-galactoside hydrolysed in 1 min per mg of protein. Protein concentration was determined using the Bio-Rad protein assay kit. All results are presented as mean values of three assays.

### Purification of GST-BceR

Escherichia coli BL21(DE3)pLysS cells (Novagen) harbouring pGEX-*bceR* were incubated at 30°C in 100 ml of LB medium containing ampicillin. At an *OD*~600~ of 0.7, IPTG was added to a final concentration of 1 mM. Cells were incubated for another 3 h and harvested by centrifugation. Collected cells were washed with phosphate-buffered saline (PBS; 140 mM NaCl, 2.7 mM KCl, 10.1 mM Na₂HPO₄, 1.8 mM KH₂PO₄, pH 7.3) and resuspended in 20 ml of the same buffer. Cells were sonicated, and the lysate was centrifuged at 8000 r.p.m. for 30 min at 4°C. The supernatant fraction was mixed with glutathione Sepharose beads (Amersham Pharmacia Biotech). Beads were washed three times with 10 ml of PBS. GST-BceR was eluted with 1 ml of 10 mM glutathione in 50 mM Tris-HCl (pH 8.0).

### DNase I footprinting assays

A 279 bp fragment encompassing –200 to +78 bp relative to the *bceA* transcription start site was amplified by PCR using GYF14 and GYR11 primers, and subsequently used as a probe for DNase I footprinting assays. To prepare one-strand labelled probes, one of the primers was initially end-labelled with [γ-³²P]-ATP. A constant amount of probe (300 000 c.p.m., 9.25 fmol) and 5 μg of poly-(dl–dC) as competitor DNA were incubated with 5, 10 or 15 pmol of GST-BceR (100, 200 and 300 nM respectively) in 50 μl of binding buffer (50 mM

PIPES, 200 mM NaCl, 4 mM MgCl₂, 4 mM dithiothreitol, 0.5% Tween 20 and 10% glycerol) at 25°C for 30 min. DNase I (0.14 U; Takara Shuzo) was added to the reaction mixture. After incubation for 1 min at 20°C, the reaction was stopped by adding 100 μl of 20 mM EDTA. The mixture was subjected to phenol–chloroform–isoamyl alcohol (24:24:1) extraction and ethanol precipitation. The pellet was dissolved in loading buffer (0.01% bromophenol blue, 0.01% xylene cyanol and 1 mM EDTA in 90% formamide) and separated on a denaturing 6% polyacrylamide gel. DNA ladders for use as size markers were created with the appropriate end-labelled primer and a cycle sequencing kit (Takara Shuzo). After electrophoresis, radioactive bands were detected with an imaging plate and a BAS2500 scanner.

### Acknowledgements

We are grateful to members of the Japan and EU consortia of *B. subtilis* functional genomics for providing the pMutin mutants used in this study. We also thank S. Dusko Ehrlich for valuable discussions. This work was supported by a Grant-in-Aid for Scientific Research on Priority Areas, Genome Biology, from the Ministry of Education, Culture, Sports, Science and Technology of Japan.

### Supplementary material

The following material is available at  
http://www.blackwellpublishing.com/products/journals/  
suppmat/mmi/mmi3653/mmi3653sm.htm  

**Fig. S1.** Northern hybridization analysis of *bceB* and *bceS* transcripts from the spac promoter in BCEAd and BCERd mutants respectively. Overnight cultures of BCEAd (A) and BCERd (B) mutants grown in LB medium were inoculated into LB medium (OD530, 0.05) with or without 0.5 mM IPTG. Samples were withdrawn during early (OD530, 0.5; lanes 1 and 3) and middle (OD530, 1.2; lanes 2 and 4) log phase growth. Total RNA (5 μg) was loaded per lane. 32P-labelled *bceB* (A) and *bceS* (B) were used as indicated.

**Fig. S2.** Northern hybridization analysis of RNA prepared from wild-type and *bceS*-disrupted mutant strains harbouring a plasmid containing *bceR*. A 32P-labelled *bceA* probe was used.

**Table S1.** Primers used in this study.

### References

Aiba, H., Adhya, S., and de Crombrugghe, B. (1981) Evidence for two functional *gal* promoters in intact *Escherichia coli* cells. *J Biol Chem* **256**: 11905–11910.

Altena, K., Guder, A., Cramer, C., and Bierbaum, G. (2000) Biosynthesis of the lantibiotic mersacidin: organization of a type B lantibiotic gene cluster. *Appl Environ Microbiol* **66**: 2565–2571.

Azevedo, E.C., Rios, E.M., Fukushima, K., and Campos-Takaki, G.M. (1993) Bacitracin production by a new strain of *Bacillus subtilis*. Extraction, purification and characterization. *Appl Biochem Biotechnol* **42**: 1–7.

Bateman, A., Birney, E., Cerruti, L., Durbin, R., Eddy, S.R., *et al.* (2002) The Pfam protein families database. *Nucleic Acids Res* **30**: 276–280.

Bolhuins, H., van Veen, H.W., Molenaar, D., Poolman, B., Driessen, A.J.M., and Konings, W.N. (1996) Multidrug resistance in *Lactococcus lactis*: evidence for ATP-dependent drug extrusion from the inner leaflet of the cytoplasmic membrane. *EMBO J* **15**: 4239–4245.

Cao, M., and Helmann, J.D. (2002) Regulation of the *Bacillus subtilis bcrC* bacitracin resistance gene by two extracytoplasmic function σ factors. *J Bacteriol* **184**: 6123–6129.

Cundliffe, E. (1989) How antibiotic-producing organisms avoid suicide. *Annu Rev Microbiol* **43**: 207–233.

De Ruyter, P.G.G.A., Kuipers, O.P., Beerthuyzen, M.M., Alen-Boerrigter, I.V., and De Vos, W.M. (1996) Functional analysis of promoters in the nisin gene cluster of *Lactococcus lactis*. *J Bacteriol* **178**: 3434–3439.

Fabret, C., Feher, V.A., and Hosh, J.A. (1999) Two-component signal transduction in *Bacillus subtilis*: how one organism sees its world. *J Bacteriol* **181**: 1975–1983.

Fath, M.J., and Kolter, R. (1993) ABC transporters: bacterial exporters. *Microbiol Rev* **57**: 995–1017.

Fukuchi, K., Kasahara, Y., Asai, K., Kobayashi, K., Moriya, S., and Ogasawara, N. (2000) The essential two-component regulatory system encoded by *ycF* and *ycG* modulates expression of the *ftsAZ* operon in *Bacillus subtilis*. *Microbiology* **146**: 1573–1583.

Igo, M., and Losick, R. (1986) Regulation of a promoter that is utilized by minor forms of RNA polymerase holoenzyme in *Bacillus subtilis*. *J Mol Biol* **191**: 615–624.

Johnson, B.A., Anker, H., and Meleney, F.L. (1945) Bacitracin: a new antibiotic produced by a member of the *B. subtilis* group. *Science* **102**: 376–377.

Joseph, P., Fichant, G., Quentin, Y., and Denizot, F. (2002) Regulatory relationship of two-component and ABC transport systems and clustering of their genes in the *Bacillus/Clostridium* group suggest a functional link between them. *J Mol Microbiol Biotechnol* **4**: 503–513.

Klein, C., and Entian, K.-D. (1994) Genes involved in self-protection against the lantibiotic subtilin produced by *Bacillus subtilis* ATCC 6633. *Appl Environ Microbiol* **60**: 2793–2801.

Kobayashi, K., Ogura, M., Yamaguchi, H., Yoshida, K., Ogasawara, N., Tanaka, T., and Fujita, Y. (2001) Comprehensive DNA microarray analysis of *Bacillus subtilis* two-component regulatory systems. *J Bacteriol* **183**: 7365–7370.

Kobayashi, K., Ehrlich, S.D., Albertini, A., Amati, G., Andersen, K.K., Arnaud, M., *et al.* (2003) Essential *Bacillus subtilis* genes. *Proc Natl Acad Sci USA* **100**: 4678–4683.

Kunst, F., Ogasawara, N., Moszer, I., Albertini, A.M., Alloni, G., Azevedo, V., *et al.* (1997) The complete genome sequence of the Gram-positive bacterium *Bacillus subtilis*. *Nature* **390**: 249–256.

Levy, S.B. (1992) Active efflux mechanisms for antimicrobial resistance. *Antimicrob Agents Chemother* **36**: 695–703.

Méndez, C., and Salas, J.A. (2001) The role of ABC transporters in antibiotic-producing organisms: drug secretion and resistance mechanisms. *Res Microbiol* **152**: 341–350.

Moriya, S., Tsujikawa, E., Hassan, A.K., Asai, K., Kodama, T., and Ogasawara, N. (1998) A *Bacillus subtilis* gene-

encoding protein homologous to eukaryotic SMC motor protein is necessary for chromosome partition. Mol Microbiol 29: 179–187.

Neumüller, A.M., Konz, D., and Marahiel, M.A. (2001) The two-component regulatory system BacRS is associated with bacitracin ‘self-resistance’ of *Bacillus licheniformis* ATCC 10716. Eur J Biochem 268: 3180–3189.

Ogura, M., Yamaguchi, H., Yoshida, K., Fujita, Y., and Tanaka, T. (2001) DNA microarray analysis of *Bacillus subtilis* DegU, ComA and PhoP regulons: an approach to comprehensive analysis of *B. subtilis* two-component regulatory systems. Nucleic Acids Res 29: 3804–3813.

Ohki, R., Tateno, K., Okada, Y., Okajima, H., Asai, K., Sadaie, Y., et al. (2003) A bacitracin-resistant *Bacillus subtilis* gene encodes a homologue of the membrane-spanning subunit of the *Bacillus licheniformis* ABC transporter. J Bacteriol 185: 51–59.

Paulsen, I.T., Brown, M.H., and Skurray, R.A. (1996) Proton-dependent multidrug efflux system. Microbiol Rev 60: 575–608.

Podlesek, Z., Comino, A., Herzog-Velikonja, B., Zgur-Bertok, D., Komel, R., and Grabnar, M. (1995) *Bacillus licheniformis* bacitracin-resistance ABC transporter: relationship to mammalian multidrug resistance. Mol Microbiol 16: 969–976.

Putman, M., van Veen, H.W., and Konings, W.N. (2000) Molecular properties of bacterial multidrug transporters. Microbiol Mol Biol Rev 64: 672–693.

Quentin, Y., Fichant, G., and Denizot, F. (1999) Inventory, assembly and analysis of *Bacillus subtilis* ABC transport systems. J Mol Biol 287: 467–484.

Stein, T., Borchert, S., Kiesau, P., Heinzmann, S., Klöss, S., Klein, C., et al. (2002) Dual control of subtilin biosynthesis and immunity in *Bacillus subtilis*. Mol Microbiol 44: 403–416.

Stock, A.M., Robinson, V.L., and Goudreau, P.N. (2000) Two-component signal transduction. Annu Rev Biochem 69: 183–215.

Toscano, W.A., Jr, and Storm, D.R. (1982) Bacitracin. Pharmacol Ther 16: 199–210.

Tsuda, H., Yamashita, Y., Shibata, Y., Nakano, Y., and Koga, T. (2002) Genes involved in bacitracin resistance in *Streptococcus mutans*. Antimicrob Agents Chemother 46: 3756–3764.

Vagner, V., Dervyn, E., and Ehrlich, S.D. (1998) A vector for synthetic gene inactivation in *Bacillus subtilis*. Microbiology 144: 3097–3104.

van Veen, H.W., Venema, K., Bolhuis, H., Oussenko, I., Kok, J., Poolman, B., et al. (1996) Multidrug resistance mediated by a bacterial homolog of the human multidrug transporter MDR1. Proc Natl Acad Sci USA 93: 10668–10672.

van Veen, H.W., Margolles, A., Müller, M., Higgins, C.F., and Konings, W.N. (2000) The homodimeric ATP-binding cassette transporter LmrA mediates multidrug transport by an alternating two-site (two-cylinder engine) mechanism. EMBO J 19: 2503–2514.

Youngman, P., Perkins, J., and Sandman, K. (1985) Use of Tn917-mediated transcriptional gene fusions to *lacZ* and *cat-86* for the identification and study of regulated genes in the *Bacillus subtilis* chromosome. In Molecular Biology of Microbial Differentiation. Hoch, J.A., and Setlow, P. (eds). Washington, DC: American Society for Microbiology Press, pp. 47–54.
