
CloN2, a novel acyltransferase involved in the attachment of the pyrrole-2-carboxyl moiety to the deoxysugar of clorobiocin

Hui Xu,$^{1}$ Rainer Kahlich,$^{2}$ Bernd Kammerer,$^{2}$ Lutz Heide$^{1}$ and Shu-Ming Li$^{1}$

$^{1}$Pharmazeutische Biologie, Pharmazeutisches Institut, Eberhard-Karls-Universität Tübingen, Auf der Morgenstelle 8, 72076 Tübingen, Germany  
$^{2}$Institut für Pharmakologie und Toxikologie, Abteilung Klinische Pharmakologie, Eberhard-Karls-Universität Tübingen, Otfried-Müller Str. 45, 72076 Tübingen, Germany  

The aminocoumarin antibiotic clorobiocin contains a 5-methylpyrrole-2-carboxylic acid unit, attached via an ester bond to the 3-OH group of the deoxysugar moiety. To investigate candidate genes responsible for the formation of this ester bond, a gene inactivation experiment was carried out in the clorobiocin producer Streptomyces roseochromogenes var. oscitans DS 12.976. An in-frame deletion was created in the coding sequence of the gene cloN2. The production of secondary metabolites in the wild-type and in the cloN2 mutant was analysed. The wild-type showed clorobiocin as the main product, whereas the cloN2 mutant accumulated a new aminocoumarin derivative, novclobiocin 104, lacking the pyrrole moiety at the 3-OH of the deoxysugar. In addition, free pyrrole-2-carboxylic acid accumulated in the culture extract of the cloN2 mutant. The structures of the metabolites were confirmed by NMR and LC-MS analysis. Clorobiocin production was successfully restored in the cloN2 mutant by introducing a replicative plasmid containing the cloN2 sequence. These results prove an involvement of cloN2 in the formation of the ester bond between the pyrrole moiety and the deoxysugar in clorobiocin biosynthesis. Furthermore, they indicate that the C-methylation at position 5 of the pyrrole moiety occurs after the attachment of pyrrole-2-carboxylic acid unit to the deoxysugar moiety.

---

**INTRODUCTION**

The structurally related aminocoumarin antibiotics clorobiocin, novobiocin and coumermycin A₁ (Fig. 1) are potent DNA gyrase B inhibitors (Maxwell & Lawson, 2003). Clorobiocin is more active against Gram-positive bacteria than novobiocin (Berger & Batcho, 1978) and shows much better solubility in water as well as in organic reagents than coumermycin A₁. This makes clorobiocin a very interesting starting material for the development of new antibacterial compounds.

Recently, our group has identified the biosynthetic gene clusters for novobiocin from Streptomyces spheroides NCIMB 11891 (Steffensky et al., 2000), for coumermycin A₁ from Streptomyces rishiriensis DSM 40489 (Wang et al., 2000) and for clorobiocin from Streptomyces roseochromogenes var. oscitans DS 12.976 (Pojer et al., 2002). A comparison of these gene clusters showed that the structural differences and similarities between the three antibiotics are reflected perfectly by the differences and similarities in the organization of the corresponding biosynthetic gene clusters (Pojer et al., 2002).

Structurally, both clorobiocin and coumermycin A₁ have a 5-methylpyrrole-2-carboxyl moiety attached to the 3-OH of the deoxysugar moiety, while novobiocin contains a carbamoyl group at the corresponding position. The novobiocin gene cluster contains a carbamoyltransferase gene, novN. At the same relative position, a group of seven genes, cloN1 to cloN7 and the homologous couN1 to couN7, are found in the gene clusters of clorobiocin and coumermycin A₁, respectively (Pojer et al., 2002). Our earlier study (Xu et al., 2002) showed that couN3 and couN4 are involved in the biosynthesis of the pyrrole-2-carboxylic acid in coumermycin A₁. Pyoluteorin and undecylprodigiosin also contain pyrrole-2-carboxylic acid moieties, and in their biosynthesis the intermediate pyrrole-2-carboxyl-S-[peptidyl carrier protein] is formed from L-proline by the enzymes PltE, PltF and PltL (pyoluteorin) and RedW, RedM and RedO (undecylprodigiosin), respectively (Cerdeno et al., 2001; Thomas et al., 2002; Xu et al., 2002). CloN3, CloN4 and CloN5, encoded by genes of the clorobiocin gene cluster, show high sequence similarity to PltE/F/L and RedW/M/O, respectively, and are therefore likely to convert L-proline to

Abbreviations: LC-MS, liquid chromatography-mass spectrometry; PCP, peptidyl carrier protein; SRM, selected reaction monitoring.

H. Xu and others

![image](https://i.imgur.com/1234567.png)

(A) $\mathbf{R}_{1}=\underset{\mathrm{H}}{\stackrel{\mathrm{H}_{3} \mathrm{C}}{\mathrm{N}}} \quad \mathbf{R}_{2}=\mathrm{CH}_{3}$ clorobiocin

(B) $\mathbf{R}_{1}=\mathrm{H} \quad \mathbf{R}_{2}=\mathrm{CH}_{3}$ novclobiocin 104

(C) $\mathbf{R}_{1}=\mathrm{H} \quad \mathbf{R}_{2}=\mathrm{H}$ novelobiocin 105

(D) novobiocin

(E) coumermycin $A_{1}$

Fig. 1. Structures of aminocoumarin antibiotics.

a pyrrole-2-carboxyl-S-PCP intermediate as well. A similar function is expected for their homologues in the coumermycin biosynthetic gene cluster, i.e. $couN3$, $couN4$ and $couN5$.

Little is known about the transfer of the pyrrole-2-carboxyl moiety to the deoxysugar of clorobiocin or coumermycin $A_{1}$. No genes with similarity to acyltransferases involved in the biosynthesis of acylated sugar moieties were found in the clorobiocin or coumermycin $A_{1}$ biosynthetic gene clusters.

The gene $cloN2$, immediately upstream of $cloN3$, shows homology to genes which have been proposed to be involved in orsellinic acid biosynthesis. We provide here evidence for the involvement of $cloN2$ in the transfer of pyrrole-2-carboxylic acid to the deoxysugar moiety in clorobiocin biosynthesis of $S$. roseochromogenes.

---

### METHODS

#### Bacterial strains and culture conditions
The composition of the media and the culture conditions used for $S$. roseochromogenes var. oscitans DS 12.976 (Aventis) were as described previously (Pojer et al., 2002). For isolation of chromosomal DNA and preparation of protoplasts, $S$. roseochromogenes was grown in CRM medium containing 0·75 % glycine. For the production of clorobiocin and other secondary metabolites, wild-type and mutant strains of $S$. roseochromogenes were cultured in production medium containing distiller solubles (Mancy et al., 1974; Pojer et al., 2002).

Escherichia coli XL-1 Blue MRF' (Stratagene) was used for the preparation of recombinant plasmids. The DNA-methylase-negative strain $E$. coli ET 12567 (MacNeil et al., 1992) was used to propagate plasmids for transformation in Streptomyces.

Carbenicillin (50 μg ml⁻¹ for $E$. coli) and thiostrepton (50 μg ml⁻¹ for $S$. roseochromogenes) were used for selection.

#### Genetic procedures
Standard methods for DNA isolation and manipulation were performed as described elsewhere (Kieser et al., 2000; Sambrook & Russell, 2001). DNA fragments were isolated from agarose gels using a NucleoSpin 2 in 1 extraction kit (Macherey-Nagel) according to the manufacturer's protocol. Plasmids were isolated from $E$. coli with ion-exchange columns (Nucleobond AX, Macherey-Nagel). Plasmids were isolated from Streptomyces as described by Kieser et al. (2000).

Southern blot analysis was performed on Hybond-N membranes (Amersham Biosciences) with a digoxigenin-labelled probe by using the DIG high prime DNA labelling and detection kit II (Roche Applied Science).

#### Construction of vector pN2 for gene inactivation
Vector pN2 was constructed for in-frame deletion of $cloN2$ as follows. Two fragments, cloN2-1 and cloN2-2, containing the flanking regions of $cloN2$, were generated by PCR amplification using cosmid K1F2 as template. Primer pairs used were cloN2-1/XbaI (5'-TTG ACC CGG TCT AGA TGT TCA GAC-3') and cloN2-1/NotI (5'-CTC CGA TCG CGG CCG CGT ATA TAC-3'); cloN2-2/NotI (5'-CAT ACA AGG CGG CCG CCA TCC GAA-3') and cloN2-2/HindIII (5'-GCA GTA GGA AAG CTT GGT TCA-3'). The letters shown in bold represent the mutations inserted in the original sequence to give desired restriction sites (underlined).

The PCR fragments of cloN2-1 and cloN2-2 were purified and ligated into the linearized vector pGEM-T (Promega), to give plasmids pN21
and pN22, respectively. The 1276 bp *NotI–HindIII* fragment (bp 22151–23426 in sequence AF329398) was excised from pN22 and cloned into the same sites of pcDNA 2.1 (Invitrogen) to give plasmid pN23. The *XbaI–NotI* fragment of 1308 bp (bp 19866–21173 in sequence AF329398) was isolated from plasmid pN21 and ligated into the same sites of pN23, resulting in plasmid pN24, which contained an in-frame deletion of 978 bp in the coding region of *cloN2*. The inactivation vector pN2 was obtained by releasing the *XbaI–HindIII* fragment of 2584 bp from pN24 and cloned into the same sites of pBSKT (Lombo *et al.*, 1997), a non-replicative vector containing a thiostrepton-resistance gene.

**Transformation of S. roseochromogenes and selection of recombinant mutant.** *S. roseochromogenes* protoplast preparation, transformation and regeneration were carried out by a modification of the method described by Pojer *et al.* (2002), using 40% polyethylene glycol (PEG) 1000 (Merck) instead of 50%. After 20–24 h of incubation at 30 °C, the plates were overlaid with 3 ml soft nutrient agar (Kieser *et al.*, 2000) containing a total of 0·5 mg thiostrepton for selection of integration mutants.

After transforming *S. roseochromogenes* protoplasts with plasmid pN2, thiostrepton-resistant colonies were obtained. The single-crossover mutant XHB1 was grown in the absence of thiostrepton, sporulated, and screened for loss of resistance as a consequence of double-crossover events. Three thiostrepton-sensitive mutants, named XHB11, XHB12 and XHB13, were further examined. Chromosomal DNA from *S. roseochromogenes* wild-type as well as from mutants XHB1, XHB11, XHB12 and XHB13 was digested by *PstI/PvuII* and hybridized with a probe of 880 bp containing sequences of *cloM* and *cloN1*. A band at 3·33 kb was detected in the *S. roseochromogenes* wild-type, while chromosomal DNA from strain XHB12 showed the expected band of 2·35 kb corresponding to the in-frame deletion of *cloN2*. Strains XHB11 and XHB13 gave the same band of 3·33 kb as the wild-type (Fig. 2).

**Complementation of the cloN2 mutant with the cloN2 gene.** For the complementation with *cloN2*, vector pN2C was constructed. A fragment of 1152 bp containing the whole sequence of *cloN2* (bp 21090–22242 in sequence AF329398) was obtained by PCR amplification with primer pair cloN2C/*BamHI* (5′-CTA TCG CGG ATC CTG TTC TGC GA-3′) and cloN2C/*PstI* (5′-TAG ACC TGC AGT TGC TGC TGT G-3′) using cosmid K1F2 as template. The purified PCR product was cloned into the vector pGEM-T, to give pN2C-T. The *BamHI–SpeI* fragment of 1·2 kb from pN2C-T was religated into the restriction sites of *BamHI* and *XbaI* of pUWL201 (Steffensky *et al.*, 2000). The resulting vector pN2C, containing a thiostrepton-resistance gene for selection in *Streptomyces*, was used for complementation of the *cloN2* mutant.

Vector pN2C was introduced into the *cloN2* mutant using PEG-mediated protoplast transformation, as described above. After transformation, thiostrepton-resistant clones were selected. To confirm the presence of the intact vector pN2C, plasmid was reisolated and controlled by restriction digestion and agarose gel electrophoresis.

**Analysis of secondary metabolites in S. roseochromogenes strains.** Wild-type and mutant strains of *S. roseochromogenes* were cultured in production medium (Mancy *et al.*, 1974; Pojer *et al.*, 2002) for 8–10 days at 33 °C, 210 r.p.m. A 20 ml sample of the whole culture was acidified with 1 M HCl to pH 2–4, and extracted twice with an equal volume of ethyl acetate. The solvent of the organic phase was removed and the residue was dissolved in 1 ml ethanol. This solution was used for HPLC and HPLC/MS analysis.

HPLC analysis was carried out on a Multosphere RP18-5 column (5 μm, 250 × 4 mm). For analysis of aminocoumarin derivatives, a linear gradient from 60 to 100 % methanol in 1 % aqueous formic acid over 20 min was used; UV detection was carried out at 340 nm. Authentic clorobiocin (Aventis) was used as standard. For analysis of pyrrole-2-carboxylic acid, an isocratic elution with 20 % methanol

![Figure 2](#fig2)

**Fig. 2.** Inactivation of *cloN2* in *S. roseochromogenes* var. *oscitans* DS 12.976. (A) Schematic representation of the *cloN2* gene inactivation. (B) Southern blot analysis of strains obtained in the *cloN2* gene inactivation experiment. Genomic DNA was restricted with *PstI/PvuII* from wild-type (lane 1), single-crossover mutant XHB1 (lane 2), reversion to wild-type XHB11 and XHB13 (lanes 3 and 5), and *cloN2* mutant XHB12 (lane 4). The indicated Scal–EcoRI fragment of 880 bp was used as a probe.

http://mic.sgmjournals.org
H. Xu and others

in 1% aqueous formic acid was used; UV detection was carried out at 262 nm. Pyrrole-2-carboxylic acid (Lancaster, Lancashire, UK) was used as standard.

For preparative isolation of the secondary metabolites, the fractions after HPLC analysis were collected and the solvent was evaporated under vacuum. The residues were subjected to MS and NMR (^1H and ^13C) analysis.

Negative fast-atom bombardment (FAB) mass spectra were recorded on a TSQ 70 spectrometer (Finnigan) using methanol as the solvent. The negative ion mass spectrum of clorobiocin is characterized by signals at *m/z* 695 ([M–H]⁻), 588, 507, 414 and 225. Novclobiocin 104 gave the following negative ions: *m/z* 588 ([M–H]⁻), 554, 400, 225 and 209. Novclobiocin 105 yielded the negative ions: *m/z* 574 ([M–H]⁻), 540, 339 and 209.

The NMR spectra were acquired on an AMX 400 spectrometer (Bruker). The NMR data of clorobiocin, novclobiocin 104 and novclobiocin 105 are shown in Table 1. The ^1H-NMR spectrum of pyrrole-2-carboxylic acid gave the following signals, δ (p.p.m.) (CD₃OD): 6·93 (1H, dd, 2·50 Hz, 1·35 Hz, H-5), 6·84 (1H, dd, 3·78 Hz, 1·35 Hz, H-3), 6·17 (1H, dd, 3·78 Hz, 2·50 Hz, H-4).

**LC-MS analysis and selected reaction monitoring.** LC-MS was used to identify the presence of pyrrolocarboxylic acids in the culture extracts of wild-type and *cloN2* mutant. Negative electrospray ionization (ESI) mass spectra were obtained from a ThermoFinnigan TSQ Quantum instrument (electrospray voltage 3 kV; heated capillary temperature 300 °C; sheath and auxiliary gas: nitrogen) equipped with a Nucleosil RP18-column (5 μm, 2 × 250 mm, Macherey-Nagel) under basic conditions achieved by adding ammonia (15 μl min⁻¹). For separation a gradient system of H₂O/CH₃CN (each containing 0·1% HCOOH) ranging from 95:5 to 40:60 over 40 min, then to 100% CH₃CN over 7 min, followed by elution with 100% CH₃CN for 5 min, was used; flow rate 0·2 ml min⁻¹. The collision-induced dissociation (CID) spectra and the selected reaction monitoring (SRM) during an HPLC run were recorded with collision energy +40 eV, collision gas argon, and collision pressure 1·0 × 10⁻³ torr (133 mPa).

Negative-ion ESI-CID mass spectrum of pyrrole-2-carboxylic acid (*m/z*): 110 ([M–H]⁻), 66 (M–CO₂).

Negative-ion ESI-CID mass spectrum of 5-methylpyrrole-2-carboxylic acid (*m/z*): 124 ([M–H]⁻), 80 (M–CO₂).

(45% identity) (Weitnauer *et al.*, 2001) and *evrI* of the evernimicin cluster in *Micromonospora carbonacea* var. *africana* ATCC 39149 (44% identity) (Hosted *et al.*, 2001). *cloN2* also shows 38% identity (amino acid level) to *dpsC* of the daunorubicin/doxorubicin cluster in *Streptomyces peuceticus* (Grimm *et al.*, 1994). DpsC was shown to catalyse the transfer of propionate to an acyl carrier protein in daunorubicin biosynthesis (Bao *et al.*, 1999a, b). CalO4, AviN and EvrI were proposed to be involved in orsellenic acid biosynthesis and to control the nature of the starter unit of the biosynthesis (Ahlert *et al.*, 2002; Weitnauer *et al.*, 2001), but no experimental evidence has been yet published for the function of these enzymes.

By comparison of the structures of calicheamicin, avilamycin A and evernimicin with those of clorobiocin and coumermycin A₁, it is notable that all these compounds contain one or two aromatic acyl moieties attached to a deoxysugar by an ester bond. In the cases of calicheamicin, avilamycin A and evernimicin, the acyl components are orsellenic acid derivatives and in the cases of clorobiocin and coumermycin A₁, the acyl component is 5-methyl-pyrrole-2-carboxylic acid. We speculated that CloN2, and possibly CalO4, AviN and EvrI, may be involved in the transfer of the acyl component to the sugar moiety. To provide evidence for our hypothesis, we decided to inactivate *cloN2* in the clorobiocin gene cluster.

### Inactivation of *cloN2* in *S. roseochromogenes* var. *oscitans* DS 12.976

For inactivation of *cloN2* by in-frame deletion, plasmid pN2 was constructed by cloning two PCR fragments into the vector pBSKT, a pBluescript derivative containing a thiostrepton-resistance marker. Thereby, 978 bp were deleted from the coding sequence of *cloN2*, shortening the predicted gene product from 355 to 29 amino acids (Fig. 2A).

pN2 was introduced into the clorobiocin producer *S. roseochromogenes* var. *oscitans* DS 12.976 by protoplast transformation. After appropriate selection procedures (see Methods), three antibiotic-sensitive strains resulting from double-crossover events were obtained. Southern blot examination (Fig. 2B) revealed that one of them (strain XHB12) represented the desired genotype with the inactivated *cloN2* gene, while the other two (strains XHB11 and XHB13) had reverted to the wild-type.

The absence of the intact *cloN2* gene and the presence of the disrupted gene were also confirmed by PCR, using primers *cloN2*-1/*XbaI* and *cloN2*-2/*HindIII* (see Methods; data not shown).

### RESULTS

#### Sequence analysis of *cloN2*

Upstream of the genes *cloN3*, *cloN4* and *cloN5*, which are expected to encode enzymes for the conversion of L-proline to pyrrole-2-carboxyl-S-PCP, the gene *cloN2* is located in the clorobiocin biosynthetic cluster. A homologous gene, *couN2*, is located at the same relative position in the coumermycin A₁ biosynthetic cluster (Pojer *et al.*, 2002).

Database searches revealed that *cloN2* shows sequence similarity to genes contained in the biosynthetic gene clusters of certain antibiotics and antitumour agents, e.g. *calO4* of the calicheamicin biosynthetic gene cluster in *Micromonospora echinospora* subsp. *calichensis* (47% identity at the amino acid level) (Ahlert *et al.*, 2002), *aviN* of the avilamycin A cluster in *Streptomyces viridochromogenes*

#### Analysis of secondary metabolites and identification of novclobiocin 104 and 105

For analysis of secondary metabolites, both wild-type and the *cloN2* mutant XHB12 were cultured in production medium as described previously (Mancy *et al.*, 1974; Pojer

et al., 2002). Ethyl acetate extracts of the cultures were analysed by HPLC with UV detection at 340 nm. The wild-type strain showed clorobiocin with a retention time of 20·9 min as the dominant product (Fig. 3A), which was identified by co-chromatography with an authentic reference substance as well as by LC-MS (*M*<sub>r</sub> 696).

By contrast, the *cloN2* mutant accumulated, instead of clorobiocin, two new products, with retention times of 15·5 min and 18·2 min (Fig. 3B), which were designated novclobiocin 105 and 104, respectively. To elucidate their structures, these two products were isolated on a preparative scale and analysed by MS, <sup>1</sup>H-NMR and <sup>13</sup>C-NMR. The NMR data are summarized in Table 1.

The mass spectrum of novclobiocin 104 showed a negative ion at *m/z* 588 [M–H]<sup>−</sup>, 107 mass units less than that of clorobiocin, corresponding to the lack of the methylpyrrole-carboxyl moiety. The <sup>1</sup>H-NMR spectrum of novclobiocin 104 is very similar to that of clorobiocin, with the exception that the signals for protons at H-3‴ (6·90 p.p.m.), H-4‴ (5·94 p.p.m.) and H-6‴ (2·29 p.p.m.) of the pyrrole unit

in clorobiocin were absent (Table 1). Furthermore, the absence of the signals for carbons C-7‴, C-2‴, C-3‴, C-4‴, C-5‴, C-6‴ in the <sup>13</sup>C-NMR spectrum of novclobiocin 104 also confirmed the lack of the pyrrole unit in its structure (Table 1). This proved that novclobiocin 104 is a clorobiocin derivative lacking the 5-methylpyrrole-2-carboxyl moiety at the 3‴ position of the deoxysugar (Fig. 1B).

The mass spectrum of novclobiocin 105 showed a negative ion at *m/z* 574 (M–H)<sup>−</sup>, 14 mass units less than that of novclobiocin 104, indicating the lack of a methyl group. The signals at 3·59 p.p.m. in the <sup>1</sup>H-NMR spectrum and at 62·3 p.p.m. in the <sup>13</sup>C-NMR spectrum of novclobiocin 104, corresponding to the methoxy group at 4″ of the deoxysugar, were not found in the corresponding spectra of novclobiocin 105. The other signals in the spectra of both compounds corresponded to each other very well (Table 1). Obviously, both the 3″ and 4″ of the deoxysugar in novclobiocin 105 were not substituted (Fig. 1C).

The amount of novclobiocin 104 and 105 produced by the *cloN2* mutant was, in total, about 100–180 mg l<sup>−1</sup> (corresponding to 172–309 μmol l<sup>−1</sup>), i.e. ten times higher than the clorobiocin content in the wild-type, which varied from 15 to 20 mg l<sup>−1</sup> (22–29 μmol l<sup>−1</sup>) under our culture conditions.

### Identification of pyrrole-2-carboxylic acid by <sup>1</sup>H-NMR and LC/MS

Culture extracts of both the wild-type and the *cloN2* mutant XHB12 of *S.* roseochromogenes were further analysed for free pyrrole-2-carboxylic acid by LC-MS using selected reaction monitoring (SRM). In SRM, authentic pyrrole-2-carboxylic acid gave the reaction of *m/z* 110→*m/z* 66, corresponding to the decarboxylation of [M–H]<sup>−</sup>. This reaction was observed in the wild-type as well as in the *cloN2* mutant XHB12 (Fig. 4), demonstrating the presence of pyrrole-2-carboxylic acid in both cultures. The biosynthesis of the pyrrole moiety in the *cloN2* mutant was therefore not blocked.

The presence of pyrrole-2-carboxylic acid in the *cloN2* mutant was further confirmed by <sup>1</sup>H-NMR analysis. The pyrrole-2-carboxylic acid (retention time 7·7 min) was isolated from the culture extract on a preparative scale and analysed by <sup>1</sup>H-NMR. The spectrum showed three doublet-doublet signals, at 6·93, 6·84 and 6·17 p.p.m., which were identical to those of authentic pyrrole-2-carboxylic acid and consistent with the literature data (Shimokawa et al., 1970). The amount of pyrrole-2-carboxylic acid accumulated in the *cloN2* mutant was about 1·2 mg l<sup>−1</sup> (corresponding to 11 μmol l<sup>−1</sup>). Comparable amounts of this substance were found in the wild-type.

The ethyl acetate extracts of the cultures were also used to investigate the presence of 5-methylpyrrole-2-carboxylic acid (*M*<sub>r</sub> 125) by LC-MS with SRM. Using SRM, the reaction of *m/z* 124→*m/z* 80, representing the decarboxylation

Fig. 3. HPLC analysis of the secondary metabolites in *S.* roseochromogenes strains. (A) DS 12.976 (wild-type); (B) XHB12 (*cloN2* mutant); (C) XHBC12 (*cloN2* mutant harbouring pN2C).

H. Xu and others

Table 1. ${}^{1}$H-and ${}^{13}$C-NMR data of clorobiocin, novclobiocin 104 and 105 (CD${}_{3}$OD)

See Fig. 1 for numbering of the structures. br, broad signal; ND, not detected.

| Position | ${}^{1}$H-NMR (400 MHz) [$\delta$ (p.p.m.), multi, ($J$/Hz)] | Position | ${}^{13}$C-NMR (100 MHz) [$\delta$ (p.p.m.)] |
| --- | --- | --- | --- |
|  | Clorobiocin | Novclobiocin 105 | Novclobiocin 104 |  | Clorobiocin | Novclobiocin 105 | Novclobiocin 104 |
| H-2 | 7·76, d(2·5) | 7·76, brs | 7·76, brs | C-1 | 124·2 | 124·1 | 124·5 |
| H-5 | 6·84, d(8·4) | 6·82, d(8·5) | 6·84, d(8·5) | C-2 | 130·9 | 130·8 | 130·9 |
| H-6 | 7·72, dd(8·4, 2·5) | 7·72, dd(8·5, 2·2) | 7·72, dd(8·5, 2·2) | C-3 | 129·9 | 129·6 | 129·6 |
| 2H-7 | 3·34, d(7·1) | 3·33, d(7·3) | 3·34, d(7·2) | C-4 | 161·0 | 160·5 | 160·4 |
| H-8 | 5·35, brt(7·1) | 5·35, brt(7·3) | 5·35, brt(7·2) | C-5 | 115·6 | 115·4 | 115·4 |
| 3H-10 | 1·74, s | 1·73, s | 1·75, s | C-6 | 128·5 | 128·4 | 128·4 |
| 3H-11 | 1·75, s | 1·73, s | 1·74, s | C-7 | 29·2 | 29·2 | 29·3 |
| H-5' | 7·90, d(9·2) | 7·88, d(8·8) | 7·88, d(9·0) | C-8 | 123·2 | 123·4 | 123·5 |
| H-6' | 7·33, d(9·2) | 7·24, d(8·8) | 7·29, d(9·0) | C-9 | 133·8 | 133·6 | 133·4 |
| H-1" | 5·73, d(1·8) | 5·66, brs | 5·65, d(1·6) | C-10 | 17·9 | 17·9 | 17·9 |
| H-2" | 4·34, t(2·8) | 4·13, brs† | 4·12, t(3·1) | C-11 | 26·0 | 26·0 | 26·0 |
| H-3" | 5·71, dd(10·3, 2·8) | 4·12, dd(10·0, 3·3) | 4·18, dd(9·9, 3·1) | C-12 | 170·0 | 169·9 | 170·0 |
| H-4" | 3·72, dd(10·3) | 3·76, d(10·0) | 3·40, d(9·9) | C-2' | 162·4‡ | 163·5‡ | ND |
| 3H-6" | 1·18, s | 1·13, s | 1·10, s | C-3' | 103·7 | 102·6 | 103·6 |
| 3H-7" | 1·35, s | 1·29, s | 1·31, s | C-4' | 156·5‡ | 156·4‡ | 156·2‡ |
| 3H-8" | 3·52, s | — | 3·59, s | C-5' | 123·9 | 124·9 | 125·5 |
| H-3'" | 6·90, d(3·6) | — | — | C-6' | 112·5 | 110·5 | 110·3 |
| H-4'" | 5·94, brd(3·6) | — | — | C-7' | 161·8‡ | 160·5‡ | 160·4‡ |
| 3H-6'" | 2·29, s | — | — | C-8' | 110·7 | ND | ND |
|  |  |  |  | C-9' | 150·0 | 150·1 | 150·3 |
|  |  |  |  | C-10' | 113·3 | 112·0 | 111·5 |
|  |  |  |  | C-1" | 100·4 | 100·6 | 100·5 |
|  |  |  |  | C-2" | 71·0 | 68·9 | 69·4 |
|  |  |  |  | C-3" | 71·6 | 72·4 | 72·7 |
|  |  |  |  | C-4" | 82·7 | 74·4 | 85·2 |
|  |  |  |  | C-5" | 80·5 | 80·6 | 80·1 |
|  |  |  |  | C-6" | 22·9 | 22·4 | 22·9 |
|  |  |  |  | C-7" | 29·3 | 29·3 | 29·6 |
|  |  |  |  | C-8" | 62·1 | — | 62·3 |
|  |  |  |  | C-2'" | 121·8 | — | — |
|  |  |  |  | C-3'" | 118·3 | — | — |
|  |  |  |  | C-4'" | 109·8 | — | — |
|  |  |  |  | C-5'" | 136·3 | — | — |
|  |  |  |  | C-6'" | 12·9 | — | — |
|  |  |  |  | C-7'" | 161·8‡ | — | — |

† Overlapped with the signal of H-3".

‡ The signals of C-2', 4', 7' and 7" showed similar chemical shift (156·2–163·5 p.p.m.), and their assignment may be interchanged.

of [M-H]${}^{-}$, was observed in the wild-type, but not in the cloN 2 mutant (Fig. 4), which indicated the complete absence of 5-methylpyrrole-2-carboxylic acid in the cloN 2 mutant.

Complementation of the cloN 2 mutant with pN2C

To complement the cloN 2 mutant XHB 12, a vector pN2C was constructed, containing the entire sequence of cloN 2 in pUWL 201 under the control of the constitutive ermE*p promoter. HPLC analysis showed that the cloN 2 mutant

harbouring plasmid pN2C (strain XHBC 12) produced clorobiocin in an amount similar to the wild-type (Fig. 3C). At the same time, this strain still produced considerable amounts of novclobiocin 104 and 105. No clorobiocin production was detected in transformants with empty vector pUWL 201 (data not shown).

DISCUSSION

In the clorobiocin producer S. roseochromogenes, inactivation of the gene cloN 2 by in-frame deletion resulted in the accumulation of two new aminocoumarin derivatives,

Novel acyltransferase in clorobiocin biosynthesis

$m/z$ 110 $\longrightarrow$ $m/z$ 66  
Wild-type  

$m/z$ 124 $\longrightarrow$ $m/z$ 80  
Wild-type  

$cloN2$ mutant  

$cloN2$ mutant  

Time (min)  

Fig. 4. Analysis of pyrrole-2-carboxylic acids in extracts of bacterial cultures by SRM. The reactions $m/z$ 110 $\rightarrow$ $m/z$ 66 and $m/z$ 124 $\rightarrow$ $m/z$ 80 were used to detect pyrrole-2-carboxylic acid and 5-methylpyrrole-2-carboxylic acid, respectively.

novclobiocin 104 and 105, both lacking the pyrrole moiety at C-3" of the deoxysugar. However, free pyrrole-2-carboxylic acid was still produced by the $cloN2$ mutant, proving that the mutation did not affect the biosynthesis of this moiety, but rather its transfer to the deoxysugar unit of clorobiocin. Clorobiocin production could be restored by expression of $cloN2$ in the $cloN2$ mutant, which demonstrates that only the $cloN2$ gene had been inactivated. Our results prove that CloN2 is involved in the formation of the ester bond between the pyrrole-2-carboxylic acid moiety and the deoxysugar. Most likely, CloN2 transfers the acyl moiety from a pyrrole-2-carboxyl-S-CloN5 intermediate to the 3-OH of the deoxysugar moiety of clorobiocin (see Fig. 5)

This indicates that the methylation of the pyrrole unit occurs only after its transfer to the deoxysugar moiety (Fig. 5). The wild-type contained a small quantity of free 5-methylpyrrole-2-carboxylic acid, possibly produced by hydrolysis of clorobiocin. This compound was not found in the $cloN2$ mutant, which is consistent with the above hypothesis.

A clorobiocin derivative lacking the methyl group at the pyrrole moiety (designated as antibiotic 2562 B) has been isolated previously from *Streptomyces griseovariabilis* (Lysenkova et al., 1980). Similarly, coumermycin derivatives lacking the methyl group at C-5 of one or both the pyrrole-2-carboxylic acid moieties have been identified in the coumermycin producer (Claridge et al., 1984).

The same function may be postulated for the corresponding gene of the coumermycin biosynthetic gene cluster, i.e. $couN2$. CouN2 shows 86% identity to CloN2, and both proteins comprise 355 amino acids.

Both in the clorobiocin and in the coumermycin A₁ gene cluster, a contiguous group of seven genes is found ($cloN1-cloN7$ and $couN1-couN7$, respectively), which are absent in the biosynthetic gene cluster of novobiocin (Pojer et al., 2002). These genes show high homology between the clorobiocin and the coumermycin cluster (average 87% identity at the amino acid level). CloN3, CloN4 and CloN5 and the corresponding CouN3, CouN4 and CouN5 are expected to catalyse the conversion of L-proline to pyrrole-2-carboxyl-S-PCP (see Fig. 5) (Thomas et al., 2002; Xu et al., 2002). CloN6/CouN6 may catalyse the C-methylation at position 5 of the pyrrole-2-carboxyl moieties (Pojer et al., 2002). CloN2 is involved in the acyl transfer to the deoxysugar moiety, as shown in this study. Therefore, only the roles of the small ORF $cloN1$ and the putative hydrolase gene $cloN7$ are as yet unclear. Inactivation of these genes may provide evidence of their importance for the biosynthesis of aminocoumarin antibiotics.

Genes with sequence similarity to $cloN2$, i.e. $calO4$, $aviN$ and $evrI$, are also found in the biosynthetic clusters of calicheamicin (Ahlert et al., 2002), avilamycin A (Weitnauer et al., 2001) and evernimicin (Hosted et al., 2001), respectively. It has been proposed that they may control the starter unit for orsellinic acid biosynthesis, based on their sequence similarity to $dpsC$ (Hosted et al., 2001; Weitnauer et al., 2001). However, their function has not been proven experimentally. All three compounds (calicheamicin, avilamycin and evernimicin) contain an orsellinic acid unit attached via an ester bond to a deoxysugar moiety. Since we have now provided evidence that $cloN2$ is involved in the acyl transfer of an aromatic carboxylic acid to the deoxysugar moiety of clorobiocin, it should be considered whether also CalO4, AviN and EvrI may be responsible for the acylation of the deoxysugar moieties of the respective antibiotics.

In our experiments, only pyrrole-2-carboxylic acid, but not its 5-methyl derivative, was found in the $cloN2$ mutant.

ACKNOWLEDGEMENTS

This work was supported by a grant from the Deutsche Forschungsgemeinschaft and from the BMBF ‘Klinische Pharmakologie’

http://mic.sgmjournals.org

H. Xu and others

![Chemical Reaction Diagram](attachment:chemical-reaction-diagram.png)

Fig. 5. Hypothetical scheme of the late steps of clorobiocin biosynthesis. The exact sequence of the CloP, CloN2 and CloN6 reactions has not yet been established.
*Pojer et al. (2003); F. Pojer, unpublished data.

---

01EC0001. H. Xu received a graduate student scholarship from Bundesland Baden-Wuerttemberg, Germany.

### REFERENCES

Ahlert, J., Shepard, E., Lomovskaya, N. & Zazopoulos, E. *et al.* (2002). The calicheamicin gene cluster and its iterative type I enediyne PKS. Science 297, 1173–1176.

Bao, W., Sheldon, P. J. & Hutchinson, C. R. (1999a). Purification and properties of the Streptomyces peucetius DpsC beta-ketoacyl acyl carrier protein synthase III that specifies the propionate-starter unit for type II polyketide biosynthesis. Biochemistry 38, 9752–9757.

Bao, W., Sheldon, P. J., Wendt-Pienkowski, E. & Hutchinson, C. R. (1999b). The Streptomyces peucetius dpsC gene determines the choice of starter unit in biosynthesis of the daunorubicin polyketide. J Bacteriol 181, 4690–4695.

Berger, J. & Batcho, A. D. (1978). Coumarin-glycoside antibiotics. J Chromatogr Libr 15, 101–158.

Cerdeno, A. M., Bibb, M. J. & Challis, G. L. (2001). Analysis of the prodigine biosynthesis gene cluster of Streptomyces coelicolor A3(2): new mechanisms for chain initiation and termination in modular multienzymes. Chem Biol 8, 817–829.

Claridge, C. A., Elander, R. P. & Price, K. E. (1984). The coumermycins: properties, biosynthesis, and fermentation. Drugs Pharm Sci 22, 413–425.

Grimm, A., Madduri, K., Ali, A. & Hutchinson, C. R. (1994). Characterization of the Streptomyces peucetius ATCC 29050 genes encoding doxorubicin polyketide synthase. Gene 151, 1–10.

Hosted, T. J., Wang, T. X., Alexander, D. C. & Horan, A. C. (2001). Characterization of the biosynthetic gene cluster for the oligosaccharide antibiotic, evernimicin, in Micromonospora carbonacea var. africana ATCC39149. J Ind Microbiol Biotechnol 27, 386–392.

Kieser, T., Bibb, M. J., Buttner, M. J., Chater, K. F. & Hopwood, D. A. (2000). *Practical Streptomyces Genetics*. Norwich: John Innes Foundation.

Lombo, F., Siems, K., Braña, A. F., Méndez, C., Bindseil, K. & Salas, J. A. (1997). Cloning and insertional inactivation of *Streptomyces argillaceus* genes involved in the earliest steps of biosynthesis of the sugar moieties of the antitumor polyketide mithramycin. *J Bacteriol* 179, 3354–3357.

Lysenkova, L. N., Brazhnikova, M. G., Borisova, V. N., Fedorova, G. B., Rubasheva, L. M., Potapova, N. P. & Rozynov, B. V. (1980). Production and physicochemical study of components of the new antibacterial antibiotic 2562. *Antibiotiki* 25, 483–488.

MacNeil, D. J., Gewain, K. M., Ruby, C. L., Dezeny, G., Gibbons, P. H. & MacNeil, T. (1992). Analysis of *Streptomyces avermitilis* genes required for avermectin biosynthesis utilizing a novel integration vector. *Gene* 111, 61–68.

Mancy, D., Ninet, L. & Preud'Homme, J. (1974). Antibiotic 18631 RP. (Rhone-Poulenc SA). US Patent 189 259[3 793 147].

Maxwell, A. & Lawson, D. M. (2003). The ATP-binding site of type II topoisomerases as a target for antibacterial drugs. *Curr Top Med Chem* 3, 283–303.

Pojer, F., Li, S.-M. & Heide, L. (2002). Molecular cloning and sequence analysis of the clorobiocin biosynthetic gene cluster: new insights into the biosynthesis of aminocoumarin antibiotics. *Microbiology* 148, 3901–3911.

Pojer, F., Wemakor, E., Kammerer, B., Chen, H., Walsh, C. T., Li, S.-M. & Heide, L. (2003). CloQ, a prenyltransferase involved

in clorobiocin biosynthesis. *Proc Natl Acad Sci U S A* 100, 2316–2321.

Sambrook, J. & Russell, D. W. (2001). *Molecular Cloning: a Laboratory Manual*, 3rd edn. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory.

Shimokawa, S., Fukui, H. & Sohma, J. (1970). Analyses of the N.M.R. spectra of pyrrole derivatives: pyrrole-2-carboxylic acid and pyrrole-2-aldehyde. *Mol Phys* 19, 695–702.

Steffensky, M., Mühlenweg, A., Wang, Z.-X., Li, S.-M. & Heide, L. (2000). Identification of the novobiocin biosynthetic gene cluster of *Streptomyces spheroides* NCIB 11891. *Antimicrob Agents Chemother* 44, 1214–1222.

Thomas, M. G., Burkart, M. D. & Walsh, C. T. (2002). Conversion of L-proline to pyrrolyl-2-carboxyl-S-PCP during undecylprodigiosin and pyoluteorin biosynthesis. *Chem Biol* 9, 171–184.

Wang, Z.-X., Li, S.-M. & Heide, L. (2000). Identification of the coumermycin A1 biosynthetic gene cluster of *Streptomyces rishiriensis* DSM 40489. *Antimicrob Agents Chemother* 44, 3040–3048.

Weitnauer, G., Mühlenweg, A., Trefzer, A. & 7 other authors (2001). Biosynthesis of the orthosomycin antibiotic avilamycin A: deductions from the molecular analysis of the avi biosynthetic gene cluster of *Streptomyces viridochromogenes* Tu57 and production of new antibiotics. *Chem Biol* 8, 569–581.

Xu, H., Wang, Z. X., Schmidt, J., Heide, L. & Li, S. M. (2002). Genetic analysis of the biosynthesis of the pyrrole and carbamoyl moieties of coumermycin A(1) and novobiocin. *Mol Genet Genomics* 268, 387–396.
