
Microbiology (2003), 149, 2317–2329

DOI 10.1099/mic.0.26257-0

---

**The Bacillus subtilis YufLM two-component system regulates the expression of the malate transporters MaeN (YufR) and YfIS, and is essential for utilization of malate in minimal medium**

Kousei Tanaka, Kazuo Kobayashi and Naotake Ogasawara

Department of Bioinformatics and Genomics, Graduate School of Information Science, Nara Institute of Science and Technology, 8916-5, Takayama, Ikoma, Nara 630-0101, Japan

---

The Gram-positive bacterium *Bacillus subtilis* has a complete set of enzymes for the tricarboxylic acid (TCA) cycle and can grow aerobically using most of the TCA cycle intermediates (malate, fumarate, succinate and citrate) as a sole carbon source. The *B. subtilis* genome sequence contains three paralogous two-component regulatory systems, CitST, DctSR and YufLM. CitST and DctSR activate the expression of a transporter of the Mg²⁺-citrate complex (CitM) and a fumarate and succinate transporter (DctP), respectively. These findings prompted an investigation of whether the YufL sensor and its cognate regulator, YufM, play a role in malate uptake. This paper reports that the YufM regulator shows *in vitro* binding to the promoter region of two malate transporter genes, *maeN* and *yfIS*, and is responsible for inducing their expression *in vivo*. It was also found that inactivation of the *yufM* or *maeN* genes resulted in bacteria that could not grow in a minimal salts medium containing malate as a sole carbon source, indicating that the induction of the MaeN transporter by the YufM regulator is essential for the utilization of malate as a carbon source. Inactivation of the *yufL* gene resulted in the constitutive expression of MaeN. This expression was suppressed by reintroduction of the kinase domain of YufL, indicating that the YufL sensor is required for proper signal detection and signalling specificity. The authors propose that a phosphatase activity of YufL plays an important role in the YufLM two-component regulatory system. The studies reported here have revealed that members of a set of paralogous two-component regulatory systems in *B. subtilis*, CitST, DctSR and YufLM, are involved in a related function – uptake (and metabolism) of the TCA cycle intermediates – but with distinct substrate specificities.

---

**INTRODUCTION**

C₄-dicarboxylates such as malate, fumarate and succinate are metabolized aerobically or anaerobically by bacteria, which contain transporters for their uptake, exchange or efflux. Studies in Gram-negative bacteria have revealed that many contain more than one C₄-dicarboxylate transporter, expression of which is induced only in the presence of the external substrate, usually through the action of a two-component regulatory system (Janausch *et al.*, 2002). The two-component regulatory system, composed of a sensor kinase and a response regulator, is a ubiquitous mechanism that is able to sense and respond to various environmental stimuli (Stock *et al.*, 2000). The sensor kinases monitor environmental signals and modulate functions of response regulators through phosphotransfer reactions. In many cases, the response regulators control gene expression directly as a transcriptional repressor or activator.

The Gram-positive bacterium *Bacillus subtilis* has a complete set of enzymes for the tricarboxylic acid (TCA) cycle and can grow aerobically on most of the TCA cycle intermediates (malate, fumarate, succinate and citrate; Fortnagel & Freese, 1968). The sequenced *B. subtilis* genome contains three paralogous sensor kinase genes, *citS*, *dctS* (*ydbF*) and *yufL*, which are highly similar to the *Klebsiella pneumoniae citA* and the *Escherichia coli dcuS* (CitA family; Fabret *et al.*, 1999; Kasper *et al.*, 1999). The CitA protein is a citrate sensor and controls the expression of genes for citrate fermentation in *K. pneumoniae* (Bott *et al.*, 1995). The *E. coli* DcuS sensor is responsible for the activation of the *dcuB* and *fumB* genes encoding the anaerobic fumarate transporter and fumarase (Zientz *et al.*, 1998; Golby *et al.*, 1999). In *B. subtilis*, the CitS sensor kinase and its cognate response regulator, CitT, have been shown to activate the expression of the secondary transporter of the Mg²⁺-citrate complex, CitM, in the presence of citrate (Boorsma *et al.*, 1996; Yamamoto *et al.*, 2000). Although *B. subtilis* has a second citrate transporter,

Abbreviation: TCA, tricarboxylic acid.
K. Tanaka, K. Kobayashi and N. Ogasawara

YxiQ (CitH), which transports the free citrate anion (Boorsma *et al.*, 1996), the *citSTM* genes are essential for *B. subtilis* growth on minimal plates including citrate as a sole carbon source (Yamamoto *et al.*, 2000). Furthermore, we have reported that the DctS sensor and the DctR (YdbG) regulator in *B. subtilis* regulate the expression of the DctP (YdbH) protein, a homologue of the C₄-dicarboxylate transporters found in Gram-negative bacteria (Asai *et al.*, 2000). Disruption of the *dctS*, *dctR* or *dctP* gene resulted in loss of the ability to utilize fumarate and succinate, but not malate, in the growth medium. These findings prompted us to study the role of the YufL sensor and its cognate regulator, YufM, in malate uptake.

Three *B. subtilis* secondary transporters having an ability to transport malate have been reported. The *B. subtilis* genome contains two genes, *yufR* and *yxkJ*, that code for membrane proteins belonging to the citrate–cation symporter family (Saier *et al.*, 2002). The YufR protein (renamed MaeN) was expressed in *E. coli* and shown to catalyse Na⁺-coupled malate transport, although its physiological role in *B. subtilis* has not been studied (Wei *et al.*, 2000). The YxkJ protein (renamed CimH) was also expressed in *E. coli* and found to translocate free citrate and malate anions (Krom *et al.*, 2001). In addition, Wei *et al.* (2000) demonstrated that the YqlI protein (renamed MleN), which belongs to the Na⁺–H⁺ antiporter family (Saier *et al.*, 2002), is an antiporter of H⁺-malate against Na⁺-lactate, and proposed that the transporter plays a role in malate uptake and Na⁺–H⁺ exchange in low protonmotive conditions. Furthermore, a search for possible malate transporter in *B. subtilis* revealed an additional candidate of the divalent anion–Na⁺ symporter family (Saier *et al.*, 2002), YflS, which is highly homologous (54% identity in 464 aa overlap) to the 2-oxoglutarate/ malate translocator of spinach (Weber *et al.*, 1995).

In this investigation, we examined whether expression of these malate transporter genes is induced by malate in the culture medium and is under the control of the YufLM two-component system. The genetic and biochemical studies described here demonstrated that YufLM induces the expression of the *maeN* and *yflS* genes in the presence of malate, and that the induction of the MaeN transporter by the YufM regulator is essential for the utilization of malate as a carbon source. The YufL sensor was required for proper signal detection and signalling specificity in this process.

### METHODS

#### Bacterial strains and growth media.
The bacterial strains used in this study are listed in Table 1. *E. coli* strains were cultured in LB medium (5 g yeast extract l⁻¹, 10 g polypeptone l⁻¹ and 5 g NaCl l⁻¹, pH 7·2) at 37°C. *B. subtilis* strains were grown in Spizizen’s minimal salts (SM) medium (2 g (NH₄)₂SO₄ l⁻¹, 14 g K₂HPO₄ l⁻¹, 6 g KH₂PO₄ l⁻¹, 1 g trisodium citrate.2H₂O l⁻¹; Anagnostopolos & Spizizen, 1961) containing 0·5% (w/v) malate, a minimal salts medium (2 g (NH₄)₂SO₄ l⁻¹, 14 g K₂HPO₄ l⁻¹, 6 g KH₂PO₄ l⁻¹) containing 0·05% (w/v) yeast extract (NYE medium; Willecke & Lange, 1974), and a minimal salts medium (2 g K₂SO₄ l⁻¹, 10·8 g K₂HPO₄ l⁻¹, 6 g KH₂PO₄ l⁻¹, 1 g trisodium citrate.2H₂O l⁻¹)

containing 0·5% (w/v) glucose and 0·2% (w/v) glutamine (GGM medium) with or without the addition of malate, fumarate or succinate at a final concentration of 2 mM. When required, ampicillin, erythromycin, kanamycin and chloramphenicol were added to a final concentration of 50, 0·5, 10 or 5 µg ml⁻¹, respectively. IPTG was added at 1 mM where indicated. All *B. subtilis* growth media were supplemented with L-tryptophan (required for the growth of the parent strain, 168 *trpC2*; 50 µg ml⁻¹).

#### Plasmids and construction of *B. subtilis* mutants.
The plasmids and primers used in this study are listed in Tables 2 and 3, respectively. Transformation of *B. subtilis* cells was performed as previously described (Moriya *et al.*, 1998).

To construct insertional inactivation mutants, internal segments of the *yufL*, *yufM*, *maeN* and *yflS* genes were amplified using the primers listed in Table 3, digested at the *BamHI* or *HindIII* sites within the primers, and inserted between the *BamHI* and *HindIII* sites of pMutinT3 (Vagner *et al.*, 1998). The resulting plasmids were used to transform 168 *trpC2* by a single crossover, with selection of the erythromycin-resistant transformants. pMutin-insertion mutants of *cimH* and *mleN* were constructed by members of the Japan and European Union consortia of *B. subtilis* functional genomics.

To create a deletion mutant of the *yufL* gene, the plasmid pIFDLC was constructed (Fig. 1). An 811 bp fragment encompassing the 7 N-terminal codons of *yufL* and the upstream region was amplified by PCR using the IFDL-BFF and IFDL-BFR primers, and a 1525 bp fragment encompassing the 5 C-terminal codons of *yufL*, including the termination codon, and the downstream region was obtained using the IFDL-BRF and IFDL-BRR primers. Then the two fragments were PCR ligated using an overlapping sequence contained within the primers. The resultant fragment (fragment A in Fig. 1) was restricted at the *KpnI* and *BamHI* sites in the primers and inserted into *KpnI-BamHI*-restricted pUC19. Next, a 2348 bp fragment covering the *pbpD* and *yuxK* region (fragment B in Fig. 1) was amplified with the IFDL-FF and IFDL-FR primers, digested at the *KpnI* and *XhoI* sites in the primers, and inserted between the *KpnI* and *XhoI* sites of pUC19 containing fragment A. Finally, a kanamycin-resistance gene of pDG780 (Anne-Marie *et al.*, 1995) was excised by *XhoI* digestion and inserted between fragments A and B on pUC19. The resulting plasmid, pIFDLC, was linearized by *BamHI* digestion and used to transform 168 *trpC2* cells by a double crossover with selection for kanamycin resistance.

To place the *yufL* gene under the control of the IPTG-inducible *spac* promoter at the *amyE* locus on the *B. subtilis* chromosome, a 1618 bp fragment containing the Shine–Dalgarno (SD) sequence and the entire coding region of *yufL* was PCR amplified using the UFLF-F and UFLF-R primers, and cloned into the *BamHI* site of plasmid pDLT3 (Morimoto *et al.*, 2002) to obtain pDLYufl. Plasmid pDLDSYL was constructed to express a fusion protein of the receiver domain of DctS and the kinase domain of YufL at the *amyE* locus. The C-terminal portion of *yufL* (from codon 295 to the termination codon) was amplified with the SLF-BF primer (which has a base substitution to create a *KpnI* restriction site without changing the coding amino acid) and the SLE-BR primer. The amplified fragment was *BamHI* restricted and inserted into a *BamHI*-restricted pDLT3. Next, the DctS receiver domain (codons 1–212) and the YufL kinase domain (codons 212–295) were amplified using primers SLF-MF (which contains the SD sequence of *yufL*) and SLF-MR, and SLF-BF and SLF-BR, respectively. The two fragments were then PCR ligated using overlapping sequences contained within the SLF-MR and SLF-BR primers. Finally, the resultant fragment was restricted at the *XhoI* and *KpnI* sites in the primers and inserted into *XhoI-KpnI*-restricted pDLT3 containing the C-terminal portion of *yufL*, to obtain pDLDSYL. Plasmids pDLYufl
Table 1. Bacterial strains

| Strain          | Genotype                                                                                   | Source                                      |
|-----------------|-------------------------------------------------------------------------------------------|---------------------------------------------|
| **B. subtilis** |                                                                                           |                                             |
| 168             | trpC2                                                                                     | S.D. Ehrlich, INRA, Jouy en Josas, France   |
| NT101           | trpC2 yufL::pMutinT3                                                                      | pMUFL→168                                   |
| NT102           | trpC2 yufM::pMutinT3                                                                      | pMUFM→168                                   |
| NT103           | trpC2 maeN::pMutinT3                                                                      | pMUFR→168                                   |
| NT104           | trpC2 yflS::pMutinT3                                                                      | pMFLS→168                                   |
| NT201           | trpC2 maeN::pMutinT3*                                                                     | pMUFRd→168                                  |
| NT202           | trpC2 ΔyufL                                                                               | pIFDLC→168                                  |
| NT203           | trpC2 amyE::(Pspac-yufL cat)                                                              | pDLYufL→168                                 |
| NT204           | trpC2 amyE::[Pspac-dctS'(1–212)-yufL'(212–533)cat]                                        | pDLDSYL→168                                 |
| NT205           | trpC2 maeN::pMutinT3*ΔyufL                                                                | NT201→NT202                                 |
| NT206           | trpC2 maeN::pMutinT3*ΔyufL amyE::(Pspac-yufL cat)                                         | NT203→NT205                                 |
| NT207           | trpC2 maeN::pMutinT3*ΔyufL amyE::(Pspac-dctS'-yufL'cat)                                   | NT204→NT205                                 |
| NT301           | trpC2 amyE::[PmaeN(-381 to +230)†-bgaBcat]                                                | pDR381→168                                  |
| NT302           | trpC2 amyE::[PmaeN(-381 to +230)†-bgaBcat] yufM::pMutinT3                                  | pMUFM→NT401                                 |
| NT303           | trpC2 amyE::[PmaeN(-267 to +230)†-bgaBcat]                                                | pDR267→168                                  |
| NT304           | trpC2 amyE::[PmaeN(-98 to +230)†-bgaBcat]                                                 | pDR98→168                                   |
| NT305           | trpC2 amyE::[PmaeN(-92 to +230)†-bgaBcat]                                                 | pDR92→168                                   |
| NT306           | trpC2 amyE::[PmaeN(-89 to +230)†-bgaBcat]                                                 | pDR89→168                                   |
| NT307           | trpC2 amyE::[PmaeN(-85 to +230)†-bgaBcat]                                                 | pDR85→168                                   |
| NT401           | trpC2 amyE::[PyflS(-339 to +196)‡-bgaBcat]                                                | pDS339→168                                  |
| NT402           | trpC2 amyE::[PyflS(-339 to +196)‡-bgaBcat] yufM::pMutinT3                                  | pMUFM→NT501                                 |
| NT403           | trpC2 amyE::[PyflS(-162 to +196)‡-bgaBcat]                                                | pDS162→168                                  |
| NT404           | trpC2 amyE::[PyflS(-98 to +196)‡-bgaBcat]                                                 | pDS98→168                                   |
| NT405           | trpC2 amyE::[PyflS(-92 to +196)‡-bgaBcat]                                                 | pDS92→168                                   |
| NT406           | trpC2 amyE::[PyflS(-85 to +196)‡-bgaBcat]                                                 | pDS85→168                                   |
| NT407           | trpC2 amyE::[PyflS(-73 to +196)‡-bgaBcat]                                                  | pDS73→168                                   |
| **E. coli**    |                                                                                           |                                             |
| C600            | supE44 hsdR17 thi-1 thr-1 leuB6 lacY1 tonA21                                               | Lab. stock                                  |
| BL21(DE3)pLysS  | F⁻ ompT hsdSB(r<sup>−</sup>m<sup>−</sup>) gal dcm Δ(srl-recA) 306::Tn10(DE3) pLysS          | Novagen                                     |

*A 367 bp HindIII-NruI fragment of pMutinT3 containing Pspac was deleted.

† Nucleotide numbers relative to the maeN initiation codon.

‡ Nucleotide numbers relative to the yflS initiation codon.

and pDLDSYL were linearized by ScaI digestion and used to transform 168 trpC2 cells by a double crossover with selection for chloramphenicol resistance.

To construct a series of strains containing transcriptional fusions of serially shortened maeN or yflS promoter regions and a promoter-less *Bacillus stearothermophilus bgaB* gene encoding a thermostable β-galactosidase (Hirata et al., 1986) at the amyE locus on the *B. subtilis* chromosome, plasmid pDLd (Nanamiya et al., 1998) was used. PCR products containing various lengths of the maeN or yflS promoter region were obtained with the primer sets listed in Table 2, digested at the EcoRI and BamHI sites in the primers, and cloned between the EcoRI and BamHI sites of pDLd. The resulting plasmids were transformed into *B. subtilis* cells by a double crossover with selection for chloramphenicol resistance.

To create *B. subtilis* cells having multiple mutations, chromosomal DNA of single mutants was used for transformation.

β-Galactosidase assay. *B. subtilis* cells (1 ml) were harvested by centrifugation when the OD<sub>600</sub> of cells reached 0·35, and β-galactosidase activity was measured as described by Youngman et al.

(1985). Incubation for the assay was at 28°C for LacZ and at 62°C for BgaB. Units are expressed as pmol 4-methylumbelliferyl β-D-galactoside hydrolysed min<sup>-1</sup>(mg protein)<sup>-1</sup>. Protein concentration was determined using the Bio-Rad Protein Assay Kit.

Northern blotting and primer-extension analysis. Total RNA was extracted from wild-type cells growing in GGM medium with and without malate at the exponential phase (OD<sub>600</sub> 0·35) by the method described by Igo & Losick (1986).

In the Northern analysis, aliquots containing 1 μg total RNA were electrophoresed and blotted onto a nylon membrane (positively charged; Roche). Hybridization and detection were performed using digoxigenin-labelled RNA probes (10 ng), according to the manufacturer's instructions (Roche). Inserts of the pMUFR and pMFLS plasmids containing an internal fragment of the yufR or yflS gene, respectively, were amplified with primers T3T7R and T3F, and used as templates for in vitro RNA synthesis using the DIG RNA Labelling Kit (Roche).

In the primer-extension analysis, 30 μg of the RNA sample was incubated with <sup>32</sup>P end-labelled primers (20 fmol) specific for the

K. Tanaka, K. Kobayashi and N. Ogasawara

Table 2. Plasmids constructed in this study

| Plasmid    | Vector     | Inserts                                                                 | Primers                                                                 |
|------------|------------|------------------------------------------------------------------------|-------------------------------------------------------------------------|
| pMUFL      | pMutinT3   | 272bp internal fragment of yufL (18 to 290)†                              | UFL-F, UFL-R                                                            |
| pMUFM      | pMutinT3   | 252bp internal fragment of yufM (18 to 270)†                              | UFM-F, UFM-R                                                            |
| pMUFR      | pMutinT3   | 200bp internal fragment of maeN (29 to 229)†                              | UFR-F, UFR-R                                                            |
| pMFLS      | pMutinT3   | 315bp internal fragment of yflS (213 to 528)†                             | FLS-F, FLS-R                                                            |
| pMUFRd     | pMutinT3*  | 200bp internal fragment of maeN (29 to 229)†                              | UFR-F, UFR-R                                                            |
| pIFDLC     | pUC19      | 6046bp fragment covering pbpD to yflN with deletion in yufL and insertion of kmR gene | IFDL-FF, IFDL-FR, IFDL-BFF, IFDL-BFR, IFDL-BRF, IFDL-BRR            |
| pDLYufL    | pDLT3      | amyE::(Pspac-yufL cat)                                                  | UFLF-F, UFLF-R                                                          |
| pDLDSYL    | pDLT3      | amyE::[Pspac-dctS'(1–212)-yufL'(212–533)cat]                             | SLF-FF, SLF-FR, SLF-MF, SLF-MR, SLF-BF, SLF-BR                         |
| pDR381     | pDLd       | PmaeN (-381 to 230)‡                                                    | EUFR-F1, UFR-R                                                          |
| pDR267     | pDLd       | PmaeN (-267 to 230)‡                                                    | EUFR-F2, UFR-R                                                          |
| pDR98      | pDLd       | PmaeN (-98 to 230)‡                                                     | EUFR-F3, UFR-R                                                          |
| pDR92      | pDLd       | PmaeN (-92 to 230)‡                                                     | EUFR-F4, UFR-R                                                          |
| pDR89      | pDLd       | PmaeN (-89 to 230)‡                                                     | EUFR-F5, UFR-R                                                          |
| pDR85      | pDLd       | PmaeN (-85 to 230)‡                                                     | EUFR-F6, UFR-R                                                          |
| pDS339     | pDLd       | PyflS (-339 to 196)$                                                    | EFLS-F1, EFLS-R                                                         |
| pDS162     | pDLd       | PyflS (162 to 196)$                                                     | EFLS-F2, EFLS-R                                                         |
| pDS98      | pDLd       | PyflS (98 to 196)$                                                      | EFLS-F3, EFLS-R                                                         |
| pDS92      | pDLd       | PyflS (-92 to 196)$                                                     | EFLS-F4, EFLS-R                                                         |
| pDS85      | pDLd       | PyflS (-85 to 196)$                                                     | EFLS-F5, EFLS-R                                                         |
| pDS73      | pDLd       | PyflS (-73 to 196)$                                                     | EFLS-F6, EFLS-R                                                         |
| pETYufM    | pET15b     | Entire coding region of yufM                                            | UFMF-F, UFMF-R                                                          |

*A 367bp HindIII-NruI fragment of pMutinT3 containing Pspac was deleted.

† Nucleotide numbers relative to the initiation codon of each gene.

‡ Nucleotide numbers relative to the maeN initiation codon.

$ Nucleotide numbers relative to the yflS initiation codon.

maeN or yflS genes (EX-R and EX-S, respectively), 250mM KCl, TE pH 7·4 (10mM Tris/HCl pH 7·4 and 1mM EDTA pH 8·0) and dNTPs (1·25mM each) for 60min at 65°C, and gradually cooled to ambient temperature over a period of 60min. After the addition of buffer (final concentration of 50mM Tris/HCl pH 8·3, 75mM KCl, 3mM MgCl₂, 10mM DTT) and reverse transcriptase (M-MLV Reverse Transcriptase, 400U; TaKaRa Biomedicals), the reaction mixture was incubated for 120min at 42°C. The cDNA products were then electrophoresed through a 6% polyacrylamide/urea gel, and radioactive bands were detected with Imaging Plates and a BAS 2500 scanner (Fuji Photo Film). The sequencing ladder used for size comparison was prepared with the same end-labelled primer and a cycle sequencing kit (TaKaRa Biomedicals) using B. subtilis chromosome DNA as template.

Purification of His₆-YufM protein. In order to express the YufM protein with a histidine tag (His₆) at the N-terminus, the entire coding sequence including the initiation codon was PCR amplified with primers UFMF-F and UFMF-R and cloned into plasmid pET15b (Novagen), to obtain pETYufM. Cultivation of E. coli BL21(DE3) pLysS cells containing pETYufM and purification of the His₆-YufM protein were performed as previously described (Morimoto et al., 2002).

DNase I footprinting experiments. As probes for DNase I footprinting assays, a 199bp fragment (-183 to +16bp relative to the maeN initiation codon) and a 212bp fragment (-162 to +50bp relative to the yflS initiation codon) were amplified by PCR using the RFOOT-F and RFOOT-R primers and the EFLS-F2 and SFOOT-R primers, respectively. To prepare single-stranded labelled probes,

one of the primers was end-labelled with [γ-³²P]ATP. A constant amount of the probes (100fmol) and 0·5µg poly(dI-dC)/poly(dI-dC) as a competitor DNA were incubated with increasing amounts of His₆-YufM (8·8, 17, 35, 70pmol) in 50µl binding buffer (50mM PIPES pH 6·25, 200mM NaCl, 4mM MgCl₂, 4mM DTT, 20mM imidazole, 0·5% Tween 20 and 10% glycerol) at 25°C for 30min. Then 1µl of a DNase I solution (0·1U µl⁻¹, TaKaRa Biomedicals) was added to the reaction mixture. After incubation for 1min at 25°C, the reaction was stopped by adding 100µl 20mM EDTA, and the mixture was subjected to phenol/chloroform/isoamyl alcohol (25:24:1) extraction and ethanol precipitation. The pellet was dissolved in a loading buffer (0·01% bromophenol blue, 0·01% xylene cyanol and 1mM EDTA in 90% formamide) and separated on a denaturing 6% polyacrylamide gel. After electrophoresis, radioactive bands were detected with Imaging Plates and a BAS 2500 scanner. DNA ladders used as size markers were created with the appropriate end-labelled primer and a cycle sequencing kit (TaKaRa Biomedicals).

---

RESULTS

The YufM regulator is essential for malate utilization in B. subtilis

The possible operonic structures of the genes investigated herein, together with positions of the pMutinT3 insertions, are shown schematically in Fig.2 (see also http://bacillus.genome.ad.jp).

Table 3. Primers used in this study

| Primer | Sequence (5'-3') * | Restriction site |
|--------|---------------------|------------------|
| UFL-F   | AAGAAGCTTCTGCAAACCAGACTCACC | HindIII |
| UFL-R   | GGAGGATCCGGGATACCGTTCATATCC | BamHI |
| UFM-F   | AAGAAGCTTGTTGAAGATGACCCCATGG | HindIII |
| UFM-R   | GGAGGATCCTGATCACGTCAAGCTCGC | BamHI |
| UFR-F   | AAGAAGCTTTTCACCTGAGCAAAAGAC | HindIII |
| UFR-R   | GGAGGATCCCCCTATGTCTCCGAGAAC | BamHI |
| FLS-F   | AAGAAGCTTGTTACTGCACTAACTAACTGG | HindIII |
| FLS-R   | GGAGGATCCTCTGTTCATTTCATTTGCC | BamHI |
| IFDL-FF | GGTGGTACCAGAGTGACCATGTTACGAAAAATAATC | KpnI |
| IFDL-FR | CTCCTCGAGTTCCTAGCCATTTTTATTTACGG | XhoI |
| IFDL-BFF | GGTGGTACCCATGCTCCTCGAGCAAATCTGACCACCAAACGC | KpnI, XhoI |
| IFDL-BFR | cttcggttctgttttttCATATAAAAACTTCCTTACGTTT |  |
| IFDL-BRF | aaaaaaacagaaccgaagGAGGAAAATCATG |  |
| IFDL-BRR | GGAGGATCCGTCAAATTCTCACGCGCC | BamHI |
| UFLF-F   | GGAGGATCCTTAATCATGATTTTCCCTCCTTCG | BamHI |
| UFLF-R   | GGAGGATCCGTAAGGAAGTTTTATATGAAAAAAC | BamHI |
| SLF-FF   | CTCCTCGAGCGCAGGAATCATAAAAACGTAAGGAAGTTTTATATGAAC- | XhoI |
|         | AAAAAGAAGCTCTCAATCC |  |
| SLF-FR   | agcatcgcgctccggtcttcaTACATTCGGACAATCTCATGG |  |
| SLF-MF   | ccatgagattgccgaatgtaTGAAGAACGGAGCGCGATGCT |  |
| SLF-MR   | ATAGGTACCTCATTAAACACAAGCTCC | KpnI |
| SLF-BF   | GGAGGATCCCTCGAGTGAGGTACCTATCCAGCTGAAAGG | BamHI, XhoI, KpnI |
| SLF-BR   | GGAGGATCCTTAATCATGATTTTCCCTCCTCGG | BamHI |
| EUFR-F1  | GAAGAATTCGTATTCGGAAGGACATCCG | EcoRI |
| EUFR-F2  | GAAGAATTCGTATATGCTGATGGCTCCTTATATC | EcoRI |
| EUFR-F3  | GAAGAATTCCTTAATTTAATTGTTTATTAGTTTTT | EcoRI |
| EUFR-F4  | GAAGAATCTTTAATTTGTTTATTAGTTTTTAACCT | EcoRI |
| EUFR-F5  | GAAGAATCAATTGTTTATTAGTTTTTAACCTAA | EcoRI |
| EUFR-F6  | GAAGAATTCGTTTATTAGTTTTTAACTTAAAAAAAAATAG | EcoRI |
| EFLS-F1  | GAAGAATTCCTGTCAAGCTGCTAAAACGATC | EcoRI |
| EFLS-F2  | GAAGAATTCGTTATAAATCAAGCGGGTGC | EcoRI |
| EFLS-F3  | GAAGAATTCTTTGTGTTTTTTAATTAATTAATGTTTATTAACCTAG | EcoRI |
| EFLS-F4  | GAAGAATTCGTTTTTTAATTAATTAATGTTTATTAACCTAGTTAAG | EcoRI |
| EFLS-F5  | GAAGAATCAATTAATTAATGTTTATTAACCTTAGTTAAGGAG | EcoRI |
| EFLS-F6  | GAAGAATTCGTTTATTAACCTTAGTTAAGGAGTAGAATGG | EcoRI |
| EFLS-R   | GGAGGATCCGCAATTCACCATTGGC | BamHI |
| EX-R     | CAAGACCAAATTTTTGAAACAGG |  |
| EX-S     | GGGAAATAAACAGATGATTAGTCCC |  |
| UF MF-F  | CATCATCATCATCATATGATTAATGTACTAATAGTTGAAGATG | NdeI |
| UF MF-R  | GGAGGATCCTTAGATATTGCTTTATCCGTTAATG | BamHI |
| R FOOT-F | GAAGAATTCGGCGTGCCCTATTTAAAGGAAAAAG | EcoRI |
| R FOOT-R | GGAGGATCCTTGGAAATGCTCCCATGTC | BamHI |
| S FOOT-R | GGAGGATCCGGAACAAGCTTTACTGCTGACTG | BamHI |
| T3F      | TTTATCTACAAGGTGTGG |  |
| T3RT7    | CAGGAAACAGCTATGACCTAATACGACTCACTATAGGGCGAAGTGTA- |  |
|         | TCAACAAGCTGG |  |

*Underlining indicates artificial recognition sites of restriction enzymes. Lower-case letters indicate sequences used for PCR ligation.

In order to examine the function of yufL and yufM, we inserted the pMutinT3 plasmid (Vagner et al., 1998) into each gene and monitored the growth of the mutants in Spizizen's minimal salts medium (SM medium) containing malate as a sole carbon source (Fig. 3). Growth of wild-type B.subtilis cells in the malate-only medium was comparable to that in the medium containing glucose as carbon source. In contrast, the yufL and yufM mutants were unable to grow

http://mic.sgmjournals.org

K. Tanaka, K. Kobayashi and N. Ogasawara

(A)
NT101 NT102 NT103
pMutinT3 pMutinT3 pMutinT3
yuxK yufK yufL yufM yufN yufO yufP yufQ maeN yufS

(B)
NT104
pMutinT3
pel yflS citS citT

(C)
pMutinT3
cydA cimH yxkl yxzE

(D)
pMutinT3
ansB mleN mleA yqkK
1.0 kbp

Fig. 2. Operon structure of genes investigated in this study. The location and direction of transcription of genes surrounding *yufL*, *yufM*, *maeN*, *yflS*, *cimH* and *mleN* on the *B.* subtilis chromosome are shown by arrows. The stem-loop structure indicates a possible transcriptional terminator. The insertion site of the pMutinT3 derivatives in each mutant is shown by inverted triangles. Small arrows indicate the direction of Pspac in integrated plasmids or that of the tet promoter.

unknown function. The *mleN* gene constitutes an operon with the downstream *mleA*, which encodes the malolactate enzyme that converts malate directly to lactate (Wei et al., 2000). In order to examine the expression of the *maeN*, *yflS*, *cimH* and *mleN* genes, we inserted pMutinT3 into each gene

in the malate-only medium, indicating that the YufLM two-component system indeed regulates malate uptake and/or metabolism.

To test whether the growth defect in the *yufL::pMutinT3* mutant could be due to a polar effect of the plasmid insertion on the expression of the downstream *yufM* gene, we examined the growth of the *yufL* mutant in the presence of IPTG to guarantee *yufM* expression from the *spac* promoter in the integrated plasmid. Unexpectedly, IPTG restored the growth of the mutant to wild-type levels (Fig. 3). This phenomenon will be discussed later.

The expression of the MaeN and YflS transporters is induced by malate in the growth medium

Next, we tested whether possible *B.* subtilis malate transporter genes are induced by malate in the growth medium. As shown in Fig. 2, the *maeN* and *yflS* genes seem to be transcribed as a monocistronic operon. The *cimH* gene is cotranscribed with the downstream *yxkI* and *yxzE* genes of

Fig. 3. Growth properties of the *yufLM* mutants in the SM medium containing malate as a carbon source. The OD~600~ of wild-type (168 trpC2, □), *yufL::pMutinT3* (NT101, △) and *yufM::pMutinT3* (NT102, ◇) cells was monitored. NT101 cells were also cultivated in the presence of 1 mM IPTG (▲).

and monitored the expression of the *lacZ* reporter gene on the integrated plasmid. We cultivated the mutants in a minimal salts medium containing yeast extract as carbon source (NYE medium; used to guarantee the growth of mutants) supplemented with glucose, malate, succinate or fumarate as additional carbon sources, as described previously (Asai et al., 2000). Measurements of the β-galactosidase activity of pMutinT3-inserted mutants in the exponential growth phase revealed that expression of the *maeN* and *yflS* genes is specifically induced by malate (data not shown).

![Figure](https://i.imgur.com/yourimageurl.png)

Fig. 4. Induction of *maeN* and *yflS* transcription by malate, and the growth properties of their inactivation mutants.  
(A) β-Galactosidase activity of *maeN::pMutinT3* (NT103, a) and *yflS::pMutinT3* (NT104, b) cells growing in GGM medium without (−) or with additional C₄-dicarboxylates: 2 mM malate (M), 2 mM fumarate (F) or 2 mM succinate (S).  
(B) Northern analysis using RNA probes specific to the *maeN* (a) and *yflS* (b) genes. Total RNA was prepared from *B. subtilis* 168 *trpC2* cells growing in GGM medium (*OD*₆₀₀ 0·35) with 2 mM malate (lane +) or without malate (lane −). The approximate positions of transcripts are indicated by triangles. Lane M contained an RNA molecular mass marker (Roche).  
(C) Determination of the *maeN* transcription-start site by primer extension experiment. RNA preparations from (B) were used for primer-extension analyses with the ³²P-labelled EX-R primer complementary to a sequence from 63 to 86 bp downstream from the *maeN* translation start site (lanes − and +). Lanes T, A, C and G contained sequencing ladders generated by using the primer used for the reverse-transcriptase reaction. The sequence and the transcriptional-start site (indicated by an arrow) are shown.  
(D) Growth properties of the *maeN* and *yflS* mutants in SM medium containing malate as carbon source: wild-type (168 *trpC2*, □), *maeN::pMutinT3* (NT103, △), and *yflS::pMutinT3* (NT104, ○) were monitored.
K. Tanaka, K. Kobayashi and N. Ogasawara

The expression of $yqkI$ was constitutive and no $cimH$ expression was detected under these growth conditions (data not shown). An operon encoding a possible ABC transport system, YufNOPQ, is located between $yufLM$ and $maeN$ (Fig. 2A). However, these genes were not induced by malate (data not shown).

Activation of the $maeN$ and $yflS$ promoters in the $maeN::pMutinT3$ and $yflS::pMutinT3$ cells by malate but not by fumarate or succinate was observed even in a minimal medium containing glucose (GGM medium, Fig. 4A), indicating that these genes are not under the control of carbon catabolite repression. Northern blot experiments using RNA from wild-type cells confirmed that 1·35 and 1·45 kb transcripts encompassing the $maeN$ and $yflS$ genes, respectively, are induced by external malate, although the induction of the latter was very weak (Fig. 4B). We performed primer extension experiments using the same RNA preparation, and the transcription initiation site was detected at 27 bp upstream of the initiation codon of $maeN$, accompanied by a possible –10 sequence of the promoter recognized by E$\sigma^{A}$, TGTAGA (Fig. 4C, see also Fig. 8B). However, no clear –35 sequence could be identified positioned at an appropriate distance from the –10 sequence, suggesting that the $maeN$ promoter is regulated by a transcriptional activator. However, we failed to determine the transcription initiation site of $yflS$, due to insufficient transcript levels and/or existence of a sequence that blocked reverse transcription. Growth of the $maeN::pMutinT3$ and $yflS::pMutinT3$ cells in SM medium containing malate as the carbon source (Fig. 4D) indicated that YflS apparently plays no role in this growth condition, although its expression is induced. By contrast, the $maeN$ mutant was unable to grow in the malate-only medium, demonstrating that the MaeN transporter is essential for the uptake of malate in the medium.

**The YufM regulator is essential for induction of $maeN$ and $yflS$ expression**

Next we tested whether the YufM regulator modifies the induction of $maeN$ and $yflS$. We transcriptionally fused the promoter sequence of each transporter gene, –381 to 230 bp (relative to the initiation codon) for $maeN$ and –339 to 196 bp for $yflS$, respectively, to a promoterless $B$. stearothermophilus $bgaB$ gene encoding a thermostable $\beta$-galactosidase (Hirata et al., 1986), and integrated the constructs into the $amyE$ locus of the $B$. subtilis genome. As shown in Fig. 5, the expression of $PmaeN::bgaB$ and $PyflS::bgaB$ in an otherwise wild-type background was induced by malate in a manner similar to that observed for the native promoters, while further introduction of the $yufM::pMutinT3$ mutation completely abolished their induction. These results demonstrated that, as expected, the YufM regulator activates the $maeN$ and $yflS$ promoters in response to malate in the growth medium.

### Role of the YufL sensor in the regulation of $maeN$ expression

The growth defect of the $yufL::pMutinT3$ mutant in SM medium with malate as carbon source was restored by the addition of IPTG to the medium (Fig. 3). Consistent with this observation, we found that the expression of $PmaeN::bgaB$ at the $amyE$ locus became constitutively active in $yufL::pMutinT3$ mutant cells cultivated in the presence of IPTG (data not shown). We questioned whether transcription by the $spac$ promoter resulted in an overproduction of the YufM regulator that led to constitutive activation of $maeN$ expression, although such a phenomenon has not been observed in similar mutants of $citS$ and $dctS$ (Yamamoto et al., 2000; Asai et al., 2000). Alternatively, the ectopic placement of the $maeN$ promoter might give unexpected results. To test these possibilities, we made a $yufL$ deletion mutant that allowed expression of YufM from its native promoter in the absence of YufL. We introduced this into the $maeN::pMutinT3$ cells to monitor the expression of the native $maeN$ promoter (NT205). In addition, we removed the $spac$ promoter in pMutinT3 in this strain, to avoid IPTG-induced side effects in subsequent experiments. The results shown in Fig. 6(a) indicate that $maeN$ expression became constitutively active in the absence of the YufL sensor. Then we ectopically expressed the YufL protein at the $amyE$ locus using the $spac$ promoter (NT206). $PmaeN$

---

**Fig. 5.** The YufM regulator regulates the induction of $maeN$ and $yflS$ transcription. Promoter sequences of each transporter gene, –381 to +230 bp (relative to the initiation codon) for $maeN$ and –339 to +196 bp for $yflS$, respectively, were transcriptionally fused to the $bgaB$ reporter gene and integrated into the $amyE$ locus. $\beta$-Galactosidase activities in otherwise wild-type background cells (NT301 and NT401, grey bars) and in $yufM$ inactivated cells (NT302 and NT402, white bars) were compared. Cells were grown to OD${}_{600}$ 0·35 in GGM medium without (–) or with additional C${}_{4}$-dicarboxylates: 2 mM malate (M), 2 mM fumarate (F) or 2 mM succinate (S).

(a) $PmaeN-bgaB$

(b) $PyflS-bgaB$

| $\beta$-Galactosidase activity (units) |
| --- |
| 2000 |
| 1500 |
| 1000 |
| 500 |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  | M | F | S |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  | M | F | S |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | --- |
|  |  |  |  |  |
|  |  |  |  |  |

|  |  |  |  |  |
| --- | --- | --- | --- | ---expression was still constitutive in NT206 cells grown in the absence of IPTG (Fig. 6b), but the addition of IPTG to the culture medium restored the malate-dependent expression (Fig. 6c). Sensor kinases of the CitA family consist of the N-terminal sensor domain flanked by two transmembrane helices (TMH) and the C-terminal intracellular autokinase domain that is connected to the second TMH with a linker region of about 100 aa in length (Kaspar et al., 1999). The kinase domain of the sensor is expected to interact with the cognate regulator. Therefore, we also expressed the linker and kinase domains of YufL (residues 212–533) as a fusion with the extracellular sensor domain including two TMHs of the fumarate/succinate sensor, DcuS (residues 1–212), in NT205 cells. Interestingly, the expression of the kinase and linker domains was sufficient for the repression of the constitutive activation of PmaeN, although the expression of PmaeN was not activated by the addition of malate, fumarate or succinate. These results demonstrate that the YufL sensor is required for proper signal detection and signalling specificity, but in a manner different from that of typical two-component system kinases, including B. subtilis CitS and DctS.

The sensor kinases usually activate the cognate response regulators by phosphorylating them in the presence of environmental signals. Besides directing the phosphorylation reaction, some sensor kinases have been reported to possess a phosphatase activity, enabling them to dephosphorylate their cognate response regulators (reviewed by Stock et al., 2000). Our observation would be explained if we assume that the YufM protein is nonspecifically phosphorylated independently of YufL, and that YufL specifically dephosphorylates the phospho-YufM in the absence of malate. When malate is present in the medium, the phosphatase activity of YufL would be suppressed and the specific phosphotransfer reaction to YufL activated.

The E. coli PhoB and the B. subtilis ComA regulators have been reported to be phosphorylated by acetyl phosphate in the absence of the cognate sensor kinase (Kim et al., 1996, 2001). Therefore we further disrupted the pta gene, encoding the phosphotransacetylase that is responsible for conversion of acetyl coenzyme A to acetyl phosphate, in NT205 cells. However, the constitutive maeN expression was not suppressed by the pta inactivation (data not shown).

### Binding of the YufM regulator to cis-acting sequences required for the induction of maeN or yflS expression

In order to determine the cis-acting region necessary for the maeN and yflS induction by malate and YufM, we performed promoter deletion experiments using the previously described amyE::PmaeN-bgaB and amyE::PyflS-bgaB cells. We found that maeN promoter activity was induced by malate at the same level as in the initial construct if 98 bp of sequence upstream of the initiation codon was retained, but a further deletion of 6 bp resulted in about 80 % reduction in the induction and the additional deletion of 3 bp completely abolished the promoter activity (Fig. 7a; see also Fig. 8B). Similarly, a full induction of the yflS expression was observed in cells having at least 92 bp of sequence upstream of the initiation codon, while removal of an additional 7 bp abolished the promoter activity (Fig. 7b, see also Fig. 8B).

Then we confirmed the binding of YufM to the identified cis-acting regulatory sequences of maeN and yflS. The YufM protein was fused to an N-terminal histidine tag (His₆), expressed in E. coli and purified to homogeneity by nickel-affinity chromatography (data not shown). Then, we performed DNase I footprinting experiments to precisely map the YufM binding sequence. Fig. 8(A) shows that YufM

![Diagram](https://i.imgur.com/placeholder.png)

**Fig. 6. Role of the YufL sensor in the regulation of maeN expression.** NT205 (∆yufL, a), NT206 (∆yufL amyE::Pspac-yufL, b and c) and NT207 [∆yufL Pspac-dctS′(sensor domain)-yufL′ (linker and kinase domains), d] cells were grown to OD~600~ 0·35 in GGM medium without (−) or with additional C₄-dicarboxylates – 2 mM malate (M), 2 mM fumarate (F) or 2 mM succinate (S) – and β-galactosidase activities were determined. IPTG was added at a final concentration of 1 mM to NT206 (c) and NT207 (d).
K. Tanaka, K. Kobayashi and N. Ogasawara

(a)
|  |  |  |  |  |
| --- | --- | --- | --- | --- |
| $a m y E$ | $y u f Q$ | $P m a e N m a e N^{\prime} b g a B$ | $a m y E$ | $\beta$-Galactosidase activity (Units) |
| back |  |  | front |  |
|  |  |  |  | malate |
|  |  |  |  | - |
|  |  |  |  | + |

| -85bp |  | NT307 | 22 | 33 |
| -89bp |  | NT306 | 20 | 37 |
| -92bp |  | NT305 | 18 | 342 |
| -98bp |  | NT304 | 21 | 1537 |
| -267bp |  | NT303 | 20 | 1355 |
| -381bp |  | NT301 | 16 | 1654 |

(b)
|  |  |  |  |  |
| --- | --- | --- | --- | --- |
| $a m y E$ | $p e l$ | $P y f l S \quad y f l S^{\prime} \quad b g a B$ | $a m y E$ | $\beta$-Galactosidase activity (Units) |
| back |  |  | front |  |
|  |  |  |  | malate |
|  |  |  |  | - |
|  |  |  |  | + |

| -73bp |  | NT407 | 17 | 15 |
| -85bp |  | NT406 | 19 | 18 |
| -92bp |  | NT405 | 19 | 211 |
| -98bp |  | NT404 | 23 | 244 |
| -162bp |  | NT403 | 14 | 174 |
| -339bp |  | NT401 | 16 | 175 |

Fig. 7. Mapping of the *cis*-acting sequence that is required for the induction of *maeN* and *yflS* expression. Transcriptional fusions of the gradually shortened *maeN* (a) and *yflS* (b) promoter regions and the *bgaB* reporter gene at the *amyE* locus were constructed as shown schematically in the figure. The $\beta$-galactosidase activities of these strains in GGM medium with or without malate are listed on the right. The lengths of the promoter regions (from the initiation codon) in each strain are listed on the left.

protected sequences of about 50 bp in both the coding and non-coding strands of the *maeN* and *yflS* promoters. These protected regions are consistent with the *cis*-regulatory region for the *maeN* and *yflS* induction identified by the promoter deletion experiments, as summarized in Fig. 8(B). However, the protection was only apparent at a high concentration of protein, 700-fold excess with respect to the DNA, suggesting that phosphorylation of YufM will increase the affinity to DNA and will be required for the *in vivo* action.

### DISCUSSION

Gram-negative bacteria such as *E. coli* and *K. pneumoniae* have been shown to monitor external citrate or fumarate and regulate the expression of genes involved in their anaerobic metabolism including transporters for their uptake (reviewed by Janausch *et al.*, 2002). The *B. subtilis* genome sequence contains three paralogous sensor kinase and response regulator pairs, CitST, DctSR and YufLM, consisting of the CitA and CitB families of sensor kinases and their cognate regulators (Fabret *et al.*, 1999; Kasper *et al.*, 1999). Our group and others have reported that the *B. subtilis* DctSR and CitST are essential for aerobic growth using citrate or fumarate/succinate as carbon sources, respectively, through the induction of expression of the transporter gene responsible for their uptake (Asai *et al.*, 2000; Yamamoto *et al.*, 2000). Here we report that the YufM regulator binds *in vitro* to the promoter region of two malate transporter genes, *maeN* and *yflS*, and is responsible for induction of *maeN* and

*yflS* expression by malate in the growth medium *in vivo*. We also demonstrated that nonfunctional mutants of *yufM* and *maeN* completely lost the ability to grow in the minimal salts medium (SM medium) containing malate as a sole carbon source, indicating that the induction of the MaeN transporter through the action of the YufM regulator is essential for the utilization of malate.

We have reported DNA microarray analyses of 24 two-component systems, including *yufLM*, to identify the target genes of each system (Kobayashi *et al.*, 2001). Each response regulator was overproduced in cells deficient for the cognate sensor kinase, and expression of 98 genes was found to be significantly induced or repressed by the YufM overproduction. However, *maeN* and *yflS* expression was not significantly affected. We found that expression of genes under the control of ComK was activated by YufM overproduction, for unknown reasons. This peculiar phenotype might be the reason why we did not detect *maeN* and *yflS* as target of YufLM.

It has been suggested that at least two transport systems for malate may be induced in *B. subtilis* (Willecke & Lange, 1974). The first system, required for the uptake of malate, fumarate and succinate, is induced in NYE medium. The second system is then induced by the addition of large amounts of malate to the growth medium. The Dct system (Asai *et al.*, 2000) would correspond to the first C₄-dicarboxylate transport system reported by Willecke & Lange (1974), although it has not yet been determined

Regulation of malate uptake in *B. subtilis*

A (a) *maeN* promoter region
coding strand noncoding strand
ATGC 123456 ATGC 123456

(b) *yfIS* promoter region
coding strand noncoding strand
ATGC 123456 ATGC 123456

B
(a)
-156 TAAAACATTCTCCCCCTGGGAAGACGATTCTC
-126 AGGGATTTTTTTCTAATCCAAAAAgacccccc
-96 ttaatttaattgttttattagtttttttaact
-66 taaaaaaatatgaagtgTTAACGCTTTCTT
-36 GTAGACTGTAACAAACAAATACATGGGGGTG
-6 TATGACATG

(b)
-129 TAAGAAAAGTGAAAAACACAAAAGGGTGCTaa
-99 ccttttgtgttttttaattaattaaaatgtt
-69 tattaacttagttaaggagtagaaTGGAAA
-39 AGGGGATCGGAAAAACAAGTATATAGGAGGA
-9 GACCTATTTTAG

Fig. 8. Binding of the YufM protein to the *cis*-acting regulatory sequences for *maeN* and *yfIS* expression. (A) DNase I footprinting assays of His₆-YufM binding to the *maeN* and *yfIS* promoter region. The YufM protein was fused to a histidine tag (His₆) at the N-terminus, expressed in *E. coli* and purified to homogeneity. A 199 bp fragment encompassing from –183 to +16 bp relative to the *maeN* initiation codon was radioactively labelled at either end and used as coding and non-coding strand probes for the *maeN* promoter sequence (a). A 212 bp fragment encompassing from –162 to +50 bp relative to the *yfIS* initiation codon was used as the probe for the *yfIS* promoter sequence (b). A constant amount of the probes (100 fmol) was incubated with different amounts of His₆-YufM. The amounts of proteins in lanes 1–6 were 0, 70, 35, 18, 8·8 and 0 pmol, respectively. Lanes A, C, G and T contained sequencing ladders. The maximum protected regions are shown by bars. The numbers are the distances with respect to the *maeN* and *yfIS* initiation codon. (B) Nucleotide sequence of the *maeN* (a) and *yfIS* (b) promoter regions. Lower-case letters indicate the His₆-YufM protected site. Italic letters indicate the start codon. The experimentally determined transcription-start site is labelled with an arrow. Underlining indicates the putative –10 sequence of the promoter recognized by Eσ⁷. Triangles indicate the end-point of the promoter sequences used in the promoter deletion experiments (Fig. 7). Black, grey and white triangles indicate full, partial and no induction, respectively, following addition of malate to the medium.
K. Tanaka, K. Kobayashi and N. Ogasawara

whether the DctP transporter uses malate as a substrate, and the second system will be the MaeN (and YflS) transporter characterized in this report.

Although its precise biochemical role is as yet unknown, the YflS protein seemed to act as a malate transporter in *B. subtilis*, in that it was induced by malate in a YufM-dependent manner. In addition to MaeN and YflS, CimH and YqkI have been reported to transport malate (Wei *et al.*, 2000; Krom *et al.*, 2001). However, we did not detect *cimH* expression in our cultivation conditions. We observed MleN expression independent of malate addition. Krom *et al.* (2000) reported that the growth of the *mleN* disruptant was indistinguishable from the wild-type cell in SM medium with either glucose or malate. Therefore, the physiological role of CimH and MleN is still unclear at this time.

The deletion analysis of *maeN* and *yflS* promoter regions and the DNase I footprinting experiments clearly demonstrated that the YufM regulator binds to the region essential for the activation of *maeN* and *yflS* transcription by external malate. Based on deletion analysis of the *dcuP* promoter, Asai *et al.* (2000) proposed that two tandem repeats of AGACCAAA compose the DcuR binding sequence. Yamamoto *et al.* (2000) identified the multiple repeats of (A/T)(A/T)-CAA in two CitT regulator binding regions in the *citM* promoter. However, we could not identify such clear repeat sequences in the YufM-binding regions of the *maeN* and *yflS* promoter sequences. Instead, we theorize that YufM might recognize some higher-order structure of the protected sequences.

The YufL sensor is required for proper signal detection and signalling specificity. Inactivation of the *yufL* gene resulted in the constitutive activation of YufM, which was suppressed by reintroduction of the kinase domain of YufL. Our observation suggests that YufM protein is phosphorylated independently of YufL and that YufL dephosphorylates the phospho-YufM in the absence of external signal, malate. Malate in the growth medium would suppress the phosphatase activity of YufL and activate the specific phosphotransfer reaction to YufL. It has been reported that the *E. coli* PhoB and the *B. subtilis* ComA regulators are phosphorylated by acetyl phosphate in the absence of the cognate sensor kinase, and that the inactivation of the *pta* gene abolishes the phosphorylation (Kim *et al.*, 1996, 2001). However, the constitutive activation of YufM was not suppressed by the *pta* inactivation. YufM might be phosphorylated nonspecifically by two-component kinases other than YufL. It is also possible that the YufM protein is activated by a small phosphodonor other than acetyl phosphate.

In Gram-negative bacteria, two-component regulatory systems regulating the uptake of C₄-dicarboxylates also regulate the expression of genes involved in their metabolism (Janausch *et al.*, 2002). A search for orthologues of the *yufLM* and *maeN* genes in complete genome sequences of bacteria using the MBGD database (http://mbgd.genome.ad.jp) revealed that the *yufNOPQ* operon found between the *yufLM* and

maeN genes in *B. subtilis* is missing in the *Bacillus halodurans* and *Oceanobacillus iheyensis* genomes and the *ytsJ* gene encoding a protein highly similar to malate oxidoreductase is located there in these organisms, suggesting that this additional malate metabolism gene is also under the control of YufLM. However, the expression of *B. subtilis ytsJ* was independent of malate in NYE medium (data not shown). Instead, another malate oxidoreductase gene in *B. subtilis*, *ywkA*, has recently been found to be regulated by YufM and induced by malate (Doan *et al.*, 2003). This finding suggests that the *B. subtilis* CitST and DctSR might also regulate the expression of genes for metabolism of citrate or C₄-dicarboxylates, although these have not yet been identified.

The studies reported here, together with previous reports (Asai *et al.*, 2000; Yamamoto *et al.*, 2000), have revealed that members of a set of paralogous two-component regulatory systems in *B. subtilis*, CitST, DctSR and YufLM, are involved in a related function, uptake (and metabolism) of the TCA cycle intermediates, but with distinct substrate specificities. In addition, different characteristics among them have been revealed. The DctS sensor requires an additional membrane protein, supposed to be C₄-dicarboxylate binding protein, DctB (YdbE), for recognition of fumarate/succinate in the growth medium (Asai *et al.*, 2000). Results presented in this report suggest that YufM protein would be nonspecifically phosphorylated by two-component kinases other than YufL, and the phosphatase activity of YufL would play an important role in proper signal detection and signalling specificity. The mode of recognition of and binding to DNA seem to be different between YufM and CitT/DctR. Further studies on the structure–function relationships of these homologous two-component sensors and kinases will be valuable in elucidating the evolution of the two-component regulatory systems.

## ACKNOWLEDGEMENTS

We are grateful to S. Aymerich and his colleagues for communicating unpublished results. This work was supported by a Grant-in-Aid for Scientific Research on Priority Areas, Genome Biology, from the Ministry of Education, Culture, Sports, Science and Technology of Japan.

## REFERENCES

Anagnostopolos, C. & Spizizen, J. (1961). Requirements for transformation in *Bacillus subtilis*. *J Bacteriol* 81, 741–746.

Anne-Marie, G., Kamran, S., Niels, F. & Patrick, S. (1995). Antibiotic-resistance cassettes for *Bacillus subtilis*. *Gene* 167, 335–336.

Asai, K., Baik, S. H., Kasahara, Y., Moriya, S. & Ogasawara, N. (2000). Regulation of the transport system for C₄-dicarboxylic acids in *Bacillus subtilis*. *Microbiology* 146, 263–271.

Boorsma, A., van der Rest, M. E., Lolkema, J. S. & Konings, W. N. (1996). Secondary transporters for citrate and the Mg²⁺-citrate complex in *Bacillus subtilis* are homologous proteins. *J Bacteriol* 178, 6216–6222.

Bott, M., Meyer, M. & Dimroth, P. (1995). Regulation of anaerobic citrate metabolism in *Klebsiella pneumoniae*. *Mol Microbiol* 18, 533–546.

Doan, T., Servant, P., Tojo, S., Yamaguchi, H., Lerondel, G., Yoshida, K., Fujita, Y. & Aymerich, S. (2003). The *Bacillus subtilis*
ywkA gene encodes a malic enzyme and its transcription is activated by the YufL/YufM two-component system in response to malate. Microbiology 149, 2331–2345.

Fabret, C., Feher, V. A. & Hoch, J. A. (1999). Two-component signal transduction in *Bacillus subtilis*: how one organism sees its world. *J Bacteriol* 181, 1975–1983.

Fortnagel, P. & Freese, E. (1968). Analysis of sporulation mutants. II. Mutants blocked in the citric acid cycle. *J Bacteriol* 95, 1431–1438.

Golby, P., Davies, S., Kelly, D. J., Guest, J. R. & Andrews, S. C. (1999). Identification and characterization of a two-component sensor-kinase and response-regulator system (DcuS-DcuR) controlling gene expression in response to $C_4$-dicarboxylates in *Escherichia coli*. *J Bacteriol* 181, 1238–1248.

Hirata, H., Fukazawa, T., Negoro, S. & Okada, H. (1986). Structure of a $\beta$-galactosidase gene of *Bacillus stearothermophilus*. *J Bacteriol* 166, 722–727.

Igo, M. & Losick, R. (1986). Regulation of a promoter that is utilized by minor forms of RNA polymerase holoenzyme in *Bacillus subtilis*. *J Mol Biol* 191, 615–624.

Janausch, I. G., Zientz, E., Tran, Q. H., Kroger, A. & Unden, G. (2002). $C_4$-dicarboxylate carriers and sensors in bacteria. *Biochim Biophys Acta* 1553, 39–56.

Kaspar, S., Perozzo, R., Reinelt, S., Meyer, M., Pfister, K., Scapozza, L. & Bott, M. (1999). The periplasmic domain of the histidine autokinase CitA functions as a highly specific citrate receptor. *Mol Microbiol* 33, 858–872.

Kim, S. B., Shin, B. S., Choi, S. K., Kim, C. K. & Park, S. H. (2001). Involvement of acetyl phosphate in the in vivo activation of the response regulator ComA in *Bacillus subtilis*. *FEMS Microbiol Lett* 195, 179–183.

Kim, S. K., Wilmes-Riesenberger, M. R. & Wanner, B. L. (1996). Involvement of the sensor kinase EnvZ in the in vivo activation of the response-regulator PhoB by acetyl phosphate. *Mol Microbiol* 22, 135–147.

Kobayashi, K., Ogura, M., Yamaguchi, H., Yoshida, K., Ogasawara, N., Tanaka, T. & Fujita, Y. (2001). Comprehensive DNA microarray analysis of *Bacillus subtilis* two-component regulatory systems. *J Bacteriol* 183, 7365–7370.

Krom, B. P., Warner, J. B., Konings, W. N. & Lolkema, J. S. (2000). Complementary metal ion specificity of the metal-citrate transporters CitM and CitH of *Bacillus subtilis*. *J Bacteriol* 182, 6374–6381.

Krom, B. P., Aardema, R. & Lolkema, J. S. (2001). *Bacillus subtilis* YxkJ is a secondary transporter of the 2-hydroxycarboxylate transporter family that transports L-malate and citrate. *J Bacteriol* 183, 5862–5869.

Morimoto, T., Loh, P. C., Hirai, T., Asai, K., Kobayashi, K., Moriya, S. & Ogasawara, N. (2002). Six GTP-binding proteins of the Era/Obg family are essential for cell growth in *Bacillus subtilis*. *Microbiology* 148, 3539–3552.

Moriya, S., Tsujikawa, E., Hassan, A. K., Asai, K., Kodama, T. & Ogasawara, N. (1998). A *Bacillus subtilis* gene-encoding protein homologous to eukaryotic SMC motor protein is necessary for chromosome partition. *Mol Microbiol* 29, 179–187.

Nanamiya, H., Ohashi, Y., Asai, K., Moriya, S., Ogasawara, N., Fujita, M., Sadaie, Y. & Kawamura, F. (1998). ClpC regulates the fate of a sporulation initiation sigma factor, $\sigma^{\text{H}}$ protein, in *Bacillus subtilis* at elevated temperatures. *Mol Microbiol* 29, 505–513.

Saier, M. H., Jr, Goldman, S. R., Maile, R. R., Moreno, M. S., Weyler, W., Yang, N. & Paulsen, I. T. (2002). Overview of transport in *Bacillus subtilis*. In *Bacillus subtilis and Its Closest Relatives: from Genes to Cells*, pp. 113–128. Edited by A. L. Sonenshein, J. A. Hoch & R. Losick, Washington, DC: American Society for Microbiology.

Stock, A. M., Robinson, V. L. & Goudreau, P. N. (2000). Two-component signal transduction. *Annu Rev Biochem* 69, 183–215.

Vagner, V., Dervyn, E. & Ehrlich, S. D. (1998). A vector for systematic gene inactivation in *Bacillus subtilis*. *Microbiology* 144, 3097–3104.

Weber, A., Menzlaff, E., Arbinger, B., Gutensohn, M., Eckerskorn, C. & Flugge, U. I. (1995). The 2-oxoglutarate/malate translocator of chloroplast envelope membranes: molecular cloning of a transporter containing a 12-helix motif and expression of the functional protein in yeast cells. *Biochemistry* 34, 2621–2627.

Wei, Y., Guffanti, A. A., Ito, M. & Krulwich, T. A. (2000). *Bacillus subtilis* YqkI is a novel malic/Na$^{+}$-lactate antiporter that enhances growth on malate at low protonmotive force. *J Biol Chem* 275, 30287–30292.

Willecke, K. & Lange, R. (1974). $C_4$-dicarboxylate transport in *Bacillus subtilis* studied with 3-fluoro-L-erythro-malate as a substrate. *J Bacteriol* 117, 373–378.

Yamamoto, H., Murata, M. & Sekiguchi, J. (2000). The CitST two-component system regulates the expression of the Mg-citrate transporter in *Bacillus subtilis*. *Mol Microbiol* 37, 898–912.

Youngman, P., Perkins, J. & Sandman, K. (1985). Use of Tn917-mediated transcriptional gene fusions to *lacZ* and *cat-86* for the identification and study of regulated genes in the *Bacillus subtilis* chromosome. In *Molecular Biology of Microbial Differentiation*, pp. 47–54. Edited by J. A. Hoch & P. Setlow. Washington, DC: American Society for Microbiology.

Zientz, E., Bongaerts, J. & Unden, G. (1998). Fumarate regulation of gene expression in *Escherichia coli* by the DcuSR (*dcuSR* genes) two-component regulatory system. *J Bacteriol* 180, 5421–5425.
