
Microbiology (2003), 149, 2331-2343

DOI 10.1099/mic.0.26256-0

# The Bacillus subtilis ywkA gene encodes a malic enzyme and its transcription is activated by the YufL/YufM two-component system in response to malate

Thierry Doan,$^{1}$ Pascale Servant,$^{1}$ t Shigeo Tojo,$^{2}$ Hirotake Yamaguchi,$^{2}$ Guillaume Lerondel,$^{1}$ Ken-Ichi Yoshida,$^{2}$ Yasutaro Fujita$^{2}$ and Stéphane Aymerich$^{1}$

**Correspondence**

Stéphane Aymerich  
stef@grignon.inra.fr

$^{1}$Génétique Moléculaire et Cellulaire, INRA (UMR 216) CNRS (URA 1925) and INAP-G, F-78850 Thiverval-Grignon, France  

$^{2}$Department of Biotechnology, Fukuyama University, 985 Sanzo, Higashimura, Fukuyama, Japan

---

A transcriptome comparison of a wild-type *Bacillus subtilis* strain growing under glycolytic or gluconeogenic conditions was performed. In particular, it revealed that the *ywkA* gene, one of the four paralogues putatively encoding a malic enzyme, was more transcribed during gluconeogenesis. Using a *lacZ* reporter fusion to the *ywkA* promoter, it was shown that *ywkA* was specifically induced by external malate and not subject to glucose catabolite repression. Northern analysis confirmed this expression pattern and demonstrated that *ywkA* is cotranscribed with the downstream *ywkB* gene. The *ywkA* gene product was purified and biochemical studies demonstrated its malic enzyme activity, which was 10-fold higher with NAD than with NADP ($k_{\text{cat}}/K_{\text{m}}$ 102 and $10 \, \text{s}^{-1} \, \text{mM}^{-1}$, respectively). However, physiological tests with single and multiple mutant strains affected in *ywkA* and/or in *ywkA* paralogues showed that *ywkA* does not contribute to efficient utilization of malate for growth. Transposon mutagenesis allowed the identification of the uncharacterized YufL/YufM two-component system as being responsible for the control of *ywkA* expression. Genetic analysis and *in vitro* studies with purified YufM protein showed that YufM binds just upstream of *ywkA* promoter and activates *ywkA* transcription in response to the presence of malate in the extracellular medium, transmitted by YufL. *ywkA* and *yufL/yufM* could thus be renamed *maeA* for **malic enzyme** and *malK/malR* for **malate kinase sensor/malate response regulator**, respectively.

---

## INTRODUCTION

Malic enzymes catalyse the reversible oxidative decarboxylation of malate into pyruvate with reduction of NAD or NADP. In eukaryotes, several isoforms of this enzyme have been characterized. Three classes of malic enzymes have been defined on the basis of their coenzyme specificity and their capability to also decarboxylate oxaloacetate (Kobayashi *et al.*, 1989). A first class of NADP-dependent malic enzymes (EC 1.1.1.40), decarboxylating malate and also oxaloacetate, are found in cytosol, chloroplasts and mitochondria; a second group of enzymes, preferentially utilizing NAD (EC 1.1.1.38), are also capable of decarboxylating oxaloacetate and are found in bacteria and insects; finally, a third class of NAD-dependent malic enzymes (EC 1.1.1.39), unable to use oxaloacetate, have been found only in mitochondrial matrix. In prokaryotes, malic enzymes are also widely distributed (Murai *et al.*, 1971; Diesterhaft & Freese, 1973; Lamed & Zeikus, 1981; K niche & Radler, 1982; Bartolucci *et al.*, 1987; Kobayashi *et al.*, 1989; Kawai *et al.*, 1996; Voegele *et al.*, 1999). However, few of them have been characterized so far, and in particular, few reports allowed distinction of whether the observed malic enzyme activity resulted from one or several isozymes. In *Escherichia coli*, both NAD- and NADP-dependent malic enzymes, named SfcA and B 2463 respectively, have been characterized (Murai *et al.*, 1971; Hansen & Juni, 1974, 1975; Stols & Donnelly, 1997) but their respective physiological functions have not been precisely documented. It has been suggested that SfcA is involved in gluconeogenesis by providing pyruvate, then converted into

---

Received 24 January 2003  
Revised 13 May 2003  
Accepted 1 July 2003

t Present address: Institut de Génétique et Microbiologie, Université Paris-Sud, Bât. 409, F-91405 Orsay, France.

Abbreviations: TCA, tricarboxylic acid; TCS, two-component system.

The online version of this paper (at http://mic.sgmjournals.org) contains a table listing the oligonucleotides used in this study and a figure showing the overproduction and purification of His$_6$-YufM.

T. Doan and others

phosphoenolpyruvate by a phosphoenolpyruvate synthase (Hansen & Juni, 1975), and that B2463 is involved in the supply of NADPH and acetyl-CoA, necessary for anabolic reactions during growth on C₄-dicarboxylates as sole carbon sources (Murai et al., 1971; Hansen & Juni, 1974, 1975). The synthesis of both enzymes is repressed in the presence of glucose while the expression of *sfcA*, encoding the NAD-dependent malic enzyme, is inducible by malate (Murai et al., 1971). The combined deletion of the two genes leads to a severe growth defect only on C₄-dicarboxylates, thus indicating that malic enzyme activity is necessary under such growth conditions in *E. coli* (van der Rest et al., 2000).

*Rhizobium meliloti*, a nitrogen-fixing Gram-negative symbiont of alfalfa, also synthesizes two malic enzymes named DME and TME, which are NAD- and NADP-dependent, respectively (Driscoll & Finan, 1997; Mitsch et al., 1998). The former has been shown to be involved in symbiotic nitrogen fixation as a result of its role for growth in the nodule, where C₄-dicarboxylates are the major carbon and energy source provided by the host (Driscoll & Finan, 1993; Dunn, 1998), while the latter has been proposed to function as a generator of NADPH necessary for biosynthesis reactions (Driscoll & Finan, 1996).

Interestingly, the analysis of the *Bacillus subtilis* complete genome sequence revealed the presence of four paralogous genes encoding putative malic enzymes: *ywkA*, *malS*, *ytsJ* and *mleA*. None of these genes has been previously characterized. Indeed, little information has been reported about malic enzyme activity in this bacterial species. Diesterhaft & Freese (1973) demonstrated that the presence of malate stimulates the synthesis of an NADP/NAD-dependent malic enzyme activity.

Amino acid sequence comparisons indicate that the *ywkA* and *malS* products, on the one hand, and the *ytsJ* and *mleA* products, on the other hand, belong to two distinct subgroups (60% identity between YwkA and MalS; 40% identity between YtsJ and MleA; but 28% identity between YtsJ and MalS). Furthermore, YwkA and MalS share 40% identity with SfcA, an NAD-dependent malic enzyme which was found to be more similar to malolactic enzymes than to other malic enzymes (Groysillier & Lonvaud-Funel, 1999). However, malic and malolactic enzymes are difficult to distinguish on the basis of the amino acid sequence because they are highly similar and harbour common conserved sequence motifs (Denayrolles et al., 1994; Kawai et al., 1996; Groysillier & Lonvaud-Funel, 1999) and because only very few malolactic enzymes have been enzymically characterized (Ansanay et al., 1993; Arthurs & Lloyd, 1999). Recently, a function for *mleA* in *B. subtilis* has been proposed by Wei et al. (2000). *mleA* is the second gene of a bicistronic operon including the *mleN* gene. *mleN* has been demonstrated to encode an antiporter that achieves simultaneous Na⁺/H⁺ and malate/lactate antiport. Therefore, MleA is strongly predicted to be a malolactic enzyme.

The examination of the sequence of the *B. subtilis* chromosome indicates that *ywkA* is probably cotranscribed with *ywkB*, which encodes a protein of unknown function. *malS* is linked to *ytnP*, an unknown gene, and *ytsJ* is linked to *dnaE*, which encodes the second essential DNA polymerase (Dervyn et al., 2001). Thus, the genetic organization of these three genes did not give any precise clue about their possible physiological roles.

Bacteria have to adapt to frequent changes in nutritional availability and, more generally, in environmental conditions. This is particularly true for free-living soil bacteria like *B. subtilis*. To accommodate gene expression to environmental conditions, two-component sensor-regulator systems are frequently used by bacteria. The two-component systems (TCSs) are composed of sensory protein kinases and of response regulator proteins. The protein kinases are typically located in the cytoplasmic membrane and comprise an N-terminal sensory and a conserved C-terminal transmitter domain. Binding of the signal ligand to the sensory domain induces ATP hydrolysis and autophosphorylation of the transmitter domain. The phosphoryl group is subsequently transferred to the cognate response regulator, leading to the activation of its regulatory property, most commonly transcriptional activation. More than 30 TCSs can be predicted from the complete *B. subtilis* genome sequence. However, a precise adaptive role has been identified for only a few of them (Perego & Hoch, 2002). Among them, two, YdbFG (DctSR) (Asai et al., 2000) and CitST (Yamamoto et al., 2000), appeared to be able to sense the presence of tricarboxylic acid (TCA) cycle intermediates in the medium and to activate the expression of genes encoding transporters for these compounds. Furthermore, a comprehensive DNA microarray analysis of the *B. subtilis* TCSs has suggested that *ywkA* could be a target of YufL/YufM (Kobayashi et al., 2001).

The starting point of the study reported here is a global transcriptome comparison of a wild-type *B. subtilis* strain growing in minimal medium containing glucose or malate as sole carbon source. Among the genes which appeared to be more transcribed on malate than on glucose was *ywkA*, but not *malS* or *ytsJ*. The regulation of the expression of the *ywkA* gene has therefore been further investigated, and the uncharacterized YufL/YufM TCS has been demonstrated to control directly *ywkA* transcription in response to the presence of malate. The enzymic activity encoded by *ywkA* has been characterized. Finally, single and multiple mutant strains for *ywkA* and the other putative malic enzyme genes have been constructed and studied to investigate the physiological role of YwkA.

---

### METHODS

**Bacterial strains and growth conditions.** *E. coli* TG1 [K12 Δ(*lac-pro*) *supE thi hsd-5/F' traD36 proA⁺B⁺ lacIq lacZΔM15]* was used as a general cloning host and *E. coli* strain M15 pREP4 (Qiagen) was used for overproduction of His₆-YufM and His₆-YwkA. *E. coli* strains were grown in Luria-Bertani broth (LB) supplemented with antibiotics when necessary (ampicillin, 100 mg l⁻¹;
kanamycin, 25 mg l⁻¹). Conventional calcium-shock or electro-
poration procedures were used for transformation (Sambrook
et al., 1989).

The B. subtilis strains used in this work are listed in Table 1. Standard
procedures were used to transform B. subtilis (Anagnostopoulos & Spizizen, 1961). B. subtilis strains were grown in LB, except when stated
otherwise. Antibiotics for selection were added at 5 mg l⁻¹ (chlor-
amphenicol), 0·4 mg l⁻¹ (erythromycin), 100 mg l⁻¹ (spectinomy-
cin), or 5 mg l⁻¹ (kanamycin). All cultures were performed at 37°C.
Growth tests were performed in MM minimal medium [K₂HPO₄
85 mM, KH₂PO₄ 40 mM, (NH₄)₂SO₄ 15 mM, trisodium citrate
6 mM, MgSO₄ 0·8 mM, pH 7] supplemented with tryptophan
(0·005%) and carbon sources (0·5%). Cultures for β-galactosidase
assays were performed in CQTHC minimal medium [C minimal
medium (Aymerich et al., 1986) supplemented with 0·005% try-
ptophan, 0·05% glutamine, 0·05% casein hydrolysate] supplemented
with 1% glucose, 1% glycerol, 1% fumarate, 1% malate, or 1%
succinate plus 1% glutamate.

DNA manipulation. PCR products were purified with the QIAquick
PCR purification kit (Qiagen). Plasmid DNA was extracted and
purified from E. coli with the QIAPrep spin miniprep kit (Qiagen).
Restriction enzymes and T4 DNA ligase were used as recommended
by the manufacturers. Pyrobest DNA polymerase (Takara) was used
for preparative polymerase chain reactions as recommended by the
manufacturer. DNA fragments were purified from agarose gels using
the Qiagen Gel Extraction Kit.

Oligonucleotides. The oligonucleotides used in this study are
listed in a table available as supplementary data with the online
version of this paper (http://mic.sgmjournals.org).

Construction of strains. A fusion of a ywkA promoter fragment
to a promoter-less lacZ gene was constructed using the vector
pDG1661 (Guérout-Fleury et al., 1996). The 277 bp promoter frag-
ment to be tested was obtained by PCR using primers PS 69 and
PS 70 (see supplementary table); this fragment was inserted between
the EcoRI and BamHI sites of pDG1661 to generate plasmid pPS 61,
then used to insert the reporter construct into the B. subtilis amyE
locus to generate strain PS 1716 (Table 1).

In vivo replacement by homologous recombination of a kanamycin-
resistance cassette for the lacZ ORF and erythromycin resistance gene
of pMUTIN was obtained using pEC 23, a plasmid specifically designed
for this use, constructed by P. Stragier, Institut de Biologie Physico-
Chimique, 75005 Paris, France.

Strain GTD 110 was constructed by replacing the malS ORF by the
spectinomycin-resistance cassette contained between the BamHI and
XbaI sites of plasmid pIC 333 (Steinmetz & Richter, 1994): two PCR
fragments, one corresponding to the 500 bp upstream of malS ORF
and one to the 500 bp downstream of the malS ORF, were generated
with primer pairs EagI-ytnP/XbaI-malS and BamHI-ytzB/EcoRI-ytzB
(see supplementary table), respectively. The two PCR fragments were
then ligated together with the spectinomycin cassette flanked by the
BamHI and XbaI restriction sites into vector pJH 101 (Ferrari et al.,
1982) to generate plasmid pTD 101, which was used to transform
B. subtilis.

RNA isolation and Northern blot analysis. For RNA preparation,
cell pellets were obtained from a mid-exponential-phase (OD₆₀₀ 0·8)
culture in S 6 medium (Fujita & Freese, 1981) containing 25 mM
glucose or 37·5 mM malate. Total RNAs were isolated from cells
essentially as previously described (Yoshida et al., 2001). For Northern
blot analysis, RNA samples (2 μg each) were electrophoresed in

Table 1. B. subtilis strains used in this study

| Strain     | Relevant genotype                                      | Source                          |
|------------|-------------------------------------------------------|----------------------------------|
| 168CA      | trpC 2                                                | Lab. stock                      |
| PS 1716    | amyE'::PywkA'::lacZ-cat                               | pPS 61→168CA                    |
| YWKAd      | ywkA'::pMUTINT 3(ery)                                | JFAN*                           |
| BFA 91     | ytsJ'::pMUTIN 2                                       | BSFA† collection                |
| GTD 102    | ywkA'::kan                                            | pEC 23→YWKAd                    |
| GTD 110    | ΔmalS::spc                                            | This study                      |
| GTD 111    | ywkA'::kanΔmalS::spc                                  | GTD 102DNA×GTD 110              |
| GTD 121    | ytsJ'::pMUTIN 2ywkA'::kan                             | BFA 91DNA×GTD 102               |
| GTD 122    | ytsJ'::pMUTIN 2ΔmalS::spc                             | BFA 91DNA×GTD 110               |
| GTD 123    | ytsJ'::pMUTIN 2ywkA'::kanΔmalS::spc                   | BFA 91DNA×GTD 111               |
| 1A 250     | alsR 1 ivlΔ1                                           | Bacillus Genetic Stock Center   |
| 1A 147     | ccpA 1 alsR 1 ivlΔ1                                    | Bacillus Genetic Stock Center   |
| C1         | yufL'::Tn10amyE'::PywkA'::lacZ-cat                     | This study                      |
| BFA 1431   | yufL'::pMUTIN 2                                        | BSFA collection                 |
| GTD 133    | yufL'::[pMUTIN 2Δ(lacZ-ery)::pEC 23(kan)]             | pEC 23→BFA 1431                 |
| GTD 134    | yufL'::[pMUTIN 2Δ(lacZ-ery)::pEC 23(kan)] amyE'::PywkA'::lacZ-cat | GTD 133DNA×PS 1716            |
| BFA 1432   | yufM'::pMUTIN 2                                        | This study                      |
| GTD 131    | yufM'::[pMUTIN 2Δ(lacZ-ery)::pEC 23(kan)]             | pEC 23→BFA 1432                 |
| GTD 132    | yufM'::[pMUTIN 2Δ(lacZ-ery)::pEC 23(kan)] amyE'::PywkA'::lacZ-cat | GTD 131DNA×PS 1716            |

* Japan Function Analysis Network for B. subtilis. Plasmid pMUTINT 3 used to construct strain YWKAd is a derivative of the pMUTIN plasmid
(Moriya et al., 1998).

† Bacillus Subtilis Functional Analysis programme.

http://mic.sgmjournals.org
glyoxal gels, transferred to a Hybond-N membrane (Amersham) and hybridized with a labelled probe as described previously (Yoshida *et al.*, 1997). To prepare the probes for *ywkA*, *ywkB* and *gapB*, parts of their coding regions were amplified by PCR using chromosomal DNA of *B. subtilis* 168 as a template and primer pairs *ywkA*-N1 and *ywkA*-N2, *ywkB*-N1 and *ywkB*-N2, and *gapB*-N1 and *gapB*-N2, respectively (see supplementary table). The PCR products were labelled radioactively by using the Bca BEST labelling kit (Takara Shuzo) and [α-³²P]dCTP (ICN Biomedicals).

**Preparation of fluorescently labelled cDNA, hybridization and microarray analysis.** The fluorescently labelled cDNA probes used for hybridization to DNA microarrays were prepared by a two-step procedure, as described previously (Ogura *et al.*, 2001).

The microarrays that we used in this study contained 4055 protein genes, 45 not being spotted due to a problem with DNA amplification by PCR, as well as 39 calf thymus DNA spots as negative controls. The hybridization and microarray analysis were performed as described previously (Ogura *et al.*, 2001; Yoshida *et al.*, 2001). The mean Cy3 and Cy5 fluorescence intensities for each spot were calculated, the background being taken as the mean of the intensities of the 39 calf thymus DNA spots. After subtracting the background from all the intensities of the *B. subtilis* gene spots, and their normalization using the total Cy3 and Cy5 intensities, we calculated the expression ratios. To get reliable ratios, we ignored the spots of the intensities used as numerators for this calculation which were less than the background, and replaced the intensities used as denominators with the standard deviation of the mean intensity of the negative controls if they were lower than them.

**Primer extension.** To map the 5′-end of the *ywkA* transcript by primer extension, 50 μg of each RNA was annealed to primer rev (see supplementary table) that had been labelled at its 5′-end by a MEGALABEL kit (Takara Shuzo) and [α-³²P]ATP (Amersham). Primer extension reactions were carried out as described previously (Yoshida *et al.*, 1997).

**Construction of the mini-Tn10 transposition library and genetic screening.** The mini-Tn10 delivery vector pIC333 (Steinmetz & Richter, 1994) was used for transposon mutagenesis. Outside the transposon, this plasmid carries a thermosensitive Gram-positive origin of replication, an erythromycin-resistance gene, and a gene encoding a modified transposase with relaxed target specificity. The mini-transposon itself confers spectinomycin resistance and contains the pUC origin of replication in *E. coli*. pIC333 was introduced into PS1716 at 25°C using erythromycin selection. Single transformant colonies were used to inoculate 12 independent cultures in 2 ml LB containing spectinomycin. At the beginning of the exponential growth phase, the temperature was shifted from 25 to 37°C and cells were grown for 4 h more at 37°C. Dilutions of the cultures were spread on plates of MM plus casein hydrolysate (0·05%) or LB medium, plus X-Gal supplemented with glucose (0·5%) or malate (0·5%) to screen for clones with deregulated *ywkA'::lacZ* expression.

**β-Galactosidase assays.** β-Galactosidase activities were measured using the method of Miller (1972) on cell extracts prepared by lysozyme treatment and centrifugation. Protein concentration was determined by the method of Bradford (1976) using the Bio-Rad protein assay solution. One unit of β-galactosidase activity is defined as the amount of enzyme that produces 1 nmol o-nitrophenol min⁻¹ at 20°C.

**Expression and purification of His₆-YufM.** His₆-YufM was overexpressed and purified using the QIAexpress kit (Qiagen). *yufM* coding sequence (705 bp) was integrated into *E. coli* expression vector pQE-30 using primers BamHI-YufM and PstI-YufM (see supplementary table) in order to produce a fusion protein with a hexahistidine tag at the N-terminus. The resulting plasmid, pTDp3,

was transformed into *E. coli* M15 pREP4 groESL to generate the strain used for production, GTDp3. All the subsequent procedures were performed as described in the QIAexpress manual. His₆-YufM was stored in a buffer containing 50 mM Tris pH 8, 150 mM NaCl, 1 mM DTT and 0·5 mM EDTA.

**Expression and purification of His₆-YwkA.** His₆-YwkA was overexpressed and purified as described for His₆-YufM. Thus, plasmid pTDp3 was built using primers BamHI-YwkA and SphI-YwkA (see supplementary table) and introduced into *E. coli* M15 pREP4 to generate strain GTDp4, used for production of His₆-YwkA. Storage buffer was the same as used for His₆-YufM.

**Determination of kinetic parameters of YwkA.** YwkA malic enzyme activity was tested at 37°C by monitoring spectrophotometrically at 340 nm the NAD(P)H formation during the reductive decarboxylation of malate [malate + NAD(P) → pyruvate + NAD(P)H]. The reaction mixture was composed of 25 nmol YwkA in a Tris buffer containing divalent cations (50 mM Tris/HCl pH 8, 10 mM MgCl₂, 10 mM MnCl₂, 50 mM KCl, 10 mM β-mercaptoethanol). For determination of the *K*ₘ for malate, 5 mM of both cofactors were added to the reaction. For determination of *K*ₘ and *V*ₘₐₓ for each cofactor, the malate concentration was raised to 40 mM. For both cofactors, a molar absorption coefficient of 6·22 × 10⁶ cm² mol⁻¹ was used for calculations. Demonstration of malic enzyme activity was assessed in a coupled reaction: at the end of the first reaction (end of NAD reduction), 40 units of strictly NAD-dependent lactate dehydrogenase from *Bacillus stearothermophilus* (Sigma) and 10 mM CaCl₂ were added to the reaction mixture. Then, oxidation of NADH, if produced by the first conversion of malate to pyruvate, was followed.

**DNA electrophoretic mobility shift assays.** The 277 bp DNA fragment of plasmid pPS61 containing the promoter region of *ywkA* was amplified by PCR with primers PS69 and PS70 (see supplementary table) and purified. Approximately 20 pmol of this fragment was end-labelled using T4 polynucleotide kinase (New England Biolabs) and 30 μCi (1·1 MBq) [γ-³²P]ATP (Isoblu, ICN). After precipitation, 10 000 c.p.m. of radiolabelled probe (≈0·05 pmol) was incubated for 5 min at room temperature in binding buffer (10 mM Tris/HCl pH 8·0, 50 mM NaCl, 1 mM EDTA, 1 mM DTT, 5%, w/v, glycerol) supplemented with 1 μg herring sperm DNA. Varying amounts of His₆-YufM were added for a total reaction volume of 20 μl. After incubating for 10 min at room temperature, samples were analysed by native 6% acrylamide gel electrophoresis, 1× Tris/glycine, run at 4°C and constant voltage (14 V cm⁻¹). After migration, gels were dried and the radiolabelled bands revealed by the PhosphoImager system.

**DNase I footprinting.** Primers PS69 and PS70 (see supplementary table), 30 pmol each, were both end-labelled using T4 polynucleotide kinase (Biolabs) and 30 μCi (1·1 MBq) [γ-³²P]ATP (Isoblu, ICN) and purified by precipitation. They were used to generate one-strand labelled PCR probes. The labelled DNA probes (60 000 c.p.m.; ≈0·25 pmol) were incubated for 15 min at room temperature with varying amounts of His₆-YufM (the amount of protein was increased fivefold compared to gel mobility shift experiments due to partial loss of YufM binding activity after freezing). DNase I (Amersham Pharmacia Biotech) reactions were then performed as previously described (Doan & Aymerich, 2003). All samples were analysed on a 6% acrylamide sequencing gel, which was dried and revealed by autoradiography.

Bacillus ywkA malic enzyme activated by YufLM TCS

# RESULTS

## Transcriptome analysis reveals ywkA induction in the presence of malate

A transcriptome analysis originally designed to identify the *B. subtilis* genes whose expression varies depending on whether the central carbon flux is glycolytic or gluconeogenic was performed. Cells of strain 168CA grew at almost the same rate on either glucose or malate as the sole carbon source; their doubling times in S6 medium containing glucose and malate were 1·37 and 1·25 h, respectively. Thus, these growth conditions are suitable to detect by DNA microarray analysis specific changes in global gene expression during glycolytic growth versus gluconeogenic growth.

changes of expression of numerous genes such as ribosome protein genes that are affected by growth rate would not interfere.

Fig. 1 shows the expression ratio of genes involved in the glycolytic pathway and the TCA cycle between the gluco-neogenic and the glycolytic growth conditions, which were inferred from our DNA microarray analysis (the complete microarray data are available on the KEGG website: http://www.genome.ad.jp/kegg/expression). Expression of genes boxed in the figure was changed at least twofold between glycolysis and gluconeogenesis. *ywkA* transcription appeared to be induced more than sevenfold in the presence of malate as compared to glucose. The transcription signal of *ywkB*, the gene located just downstream and most probably constituting a bicistronic operon with *ywkA*, was very weak. Therefore, the upregulation of *ywkB* transcription found under gluconeogenic growth conditions (ratio 1·96) cannot be considered as significant. The *gapB*, *pckA* and *pdhABCD* genes were also upregulated under gluconeogenic growth conditions. By contrast, the *gapA*, *pgk*, *tpi*, *eno* and *yjmC* genes were upregulated during glycolytic growth. The upregulation of *gapB* (Fillinger et al., 2000) and *pckA* (Yoshida et al., 2001) during gluconeogenesis was reported previously. It has also been reported that transcription of the *gapA*, *pgk*, *tpi* and *eno* genes, as well as that of the *pgm* gene, both belonging to the same *gapA-pgk-tpi-pgm-eno* operon, is stimulated during glycolysis (Fillinger et al., 2000; Ludwig et al., 2001, 2002; Doan & Aymerich, 2003).

### ywkA expression is induced only when malate is present in the medium

To confirm that *ywkA* transcription is induced in the presence of malate and to define more precisely the physiological conditions in which it is turned on, a reporter strain of the activity of the *ywkA* promoter region, PS1716, was constructed (Table 1). This strain harbours a *ywkA'::lacZ* fusion inserted into the chromosome at the ectopic *amyE* locus. The β-galactosidase activity synthesized in strain PS1716 grown in mineral medium supplemented with different carbon sources was measured (Table 2). The results

---

**Fig. 1. Expression change of the *B. subtilis* genes involved in glycolysis and gluconeogenesis.** Genes involved in the glycolytic pathway and the TCA cycle are followed by their expression ratios (malate/glucose) obtained by the DNA microarray analysis. The expression ratio for each gene was taken as the mean of two ratios obtained from the different hybridization profiles. Genes whose expression was changed at least twofold between glycolytic and gluconeogenic growths, are boxed (glycolytic genes) or boxed and stippled (gluconeogenic genes). The underlined *gapA*, *pgk*, *tpi*, *pgm* and *eno* genes, on the one hand, and *pdhABCD* genes, on the other, are organized in two operons. The complete microarray data are available on the KEGG website (http://www.genome.ad.jp/kegg/expression).

---

**Table 2. Expression of a *ywkA'::lacZ* reporter fusion in minimal medium supplemented with various carbon sources**

| Carbon source           | $\beta$-Galactosidase activity* [U (mg protein)$^{-1}$] |
|-------------------------|--------------------------------------------------------|
| Glucose                 | 7                                                      |
| Glycerol                | 9                                                      |
| Fumarate                | 11                                                     |
| Succinate + glutamate   | 17                                                     |
| Malate                  | 774                                                    |
| Malate + glucose        | 764                                                    |

*Activity was assayed at least three times from independent clones; representative results from one series are shown. Strains were cultivated in CQTHC medium supplemented with the carbon sources indicated.
indicated that ywkA promoter activity was very weak in the presence of succinate plus glutamate, or fumarate, and of glucose or glycerol, as sole gluconeogenic and glycolytic carbon source, respectively. Similarly, very weak basal β-galactosidase activity was detected in the presence of other gluconeogenic carbon sources, pyruvate or aspartate (data not shown). By contrast, the expression of the reporter construct was very strong in the presence of malate (and rather constant throughout the exponential growth phase). Moreover, addition of glucose to malate minimal medium did not affect ywkA expression. These results showed that ywkA transcription is specifically induced in the presence of malate in the culture medium and that it is not subject to carbon catabolite repression. Furthermore, because no expression was detected in the presence of fumarate, a C₄-dicarboxylic acid that enters the TCA cycle and is then converted in one step into intracellular malate, it could be assumed that only extracellular malate induces ywkA expression.

### ywkA does not contribute to the utilization of malate for growth

As ywkA is specifically expressed when malate is present in the culture medium, we wanted to find out whether this gene is involved in the utilization of malate as a carbon source for growth. A ywkA mutant strain, GTD102, was constructed and was tested for growth in minimal medium containing either malate, or succinate plus glutamate (entering the TCA cycle and subsequently providing malate), or a glycolytic carbon source, glucose. This growth test showed that strain GTD102 was able to use either malate (Fig. 2), or succinate plus glutamate or glucose (data not shown) as efficiently as the wild-type strain.

A ywkA inactivation might be functionally complemented by one or several other putative malic enzyme encoding genes. Therefore, we also constructed different strains harbouring different combinations of mutations of the ywkA, malS and ytsJ genes: single mutant strains BFA91 (ytsJ) and GTD110 (malS), double mutant strains GTD111 (ywkA, malS), GTD121 (ywkA, ytsJ) and GTD122 (ytsJ, malS), and a triple mutant strain GTD123 (ywkA, malS, ytsJ). All these strains were able to grow in glucose minimal medium as efficiently as the wild-type strain (data not shown). But when growing in the presence of malate as sole carbon source (Fig. 2), all the strains mutated for ytsJ, BFA91, GTD121, GTD122 (data not shown) and GTD123, exhibited the same lower growth rate than the wild-type strain. By contrast, all the other mutants, GTD102, GTD110 and GTD111 (data not shown) grew as well as the wild-type strain. The ywkA mutation is supposed to have a polar effect on ywkB, and the absence of phenotype could be due to the absence of expression of both ywkA and ywkB. Because the ywkA disruption was constructed using a pMUTIN derivative (Moriya et al., 1998; Vagner et al., 1998), the possible polar effect on ywkB can be suppressed by addition of IPTG. However, the growth pattern of the strains harbouring a ywkA mutation was identical in the presence and in the absence of IPTG (data not shown). These experiments confirmed that ywkA and malS are not necessary for growth under either of the conditions tested. By contrast, ytsJ appeared to play a major role in the utilization of malate for growth.

### YwkA exhibits a malic enzyme activity and preferentially utilizes NAD

Does ywkA encode a functional malic enzyme? To answer this question, a characterization of the enzymic properties of the YwkA protein was performed. First, a vector designed to overexpress a YwkA tagged with His₆ was constructed (pTDp3) and introduced into E. coli M15 pREP4. Overexpression was induced by cultivating the resulting strain in the presence of IPTG, and the His₆-YwkA protein was purified to apparent homogeneity. Then, the capacity of this protein to reduce NAD or NADP in the presence of malate was assayed. The results showed that YwkA has a strong activity of reduction of either NAD⁺ or NADP⁺ that depends on the presence of malate. To demonstrate that this reducing activity is associated with the conversion of malate into pyruvate, i.e. that YwkA exhibits a malic enzyme activity and not a malate dehydrogenase or a malolactic enzyme activity, the assay of the activity of an enzyme, lactate dehydrogenase, using specifically pyruvate as substrate, was coupled to the assay of YwkA. Indeed, addition of lactate dehydrogenase after the end of the first reaction (no further increase of A₄₃₀) led to rapid reoxidation of NADH (Fig. 3). This could result only from oxidative conversion of pyruvate into lactate and thus demonstrated that pyruvate is the product of the first reaction catalysed by YwkA. In a second series of experiments, we determined the kinetic parameters of YwkA activity. The Kₘ value for malate was

![Graph](attachment://graph.png)

**Fig. 2.** Comparative growth of strains 168CA (○), GTD102 (ywkA, ■), BFA91 (ytsJ, □), GTD110 (malS, △), GTD121 (ytsJ ywkA, ●) and GTD123 (ytsJ ywkA malS, ◇) in malate minimal medium (MM malate).
Bacillus ywkA malic enzyme activated by YufLM TCS

ywkJ gene as well as the downstream gene, ywkB. When the  
ywkJ probe was used, this 2·8 kb transcript was only slightly  
visible due to the presence of the faint broad bands from  
rRNAs (3·0 and 1·5 kb), only when cells of the three strains  
were grown on malate (lanes 2, 4 and 6). These results  
strongly suggested that the ywkAB genes are cotranscribed as  
a single transcription unit. As a control, a transcript (1·7 kb)  
of gapB was observed, as expected, only when cells were  
grown on malate (Fig. 4, gapB lanes 2, 4 and 6) (Fillinger  
et al., 2000). Furthermore, this analysis also confirmed that  
repression of ywkA-ywkB transcription as well as gapB  
transcription (Fillinger et al., 2000; Yoshida et al., 2001) is  
independent of CcpA (compare lanes 4 and 6).

Primer extension analysis mapped a 5'-end of the ywkAB  
transcript only with RNA prepared from cells of strain  
168CA grown on malate, not from those grown on glucose  
(Fig. 5, lane 1), and allowed us to find a corresponding  
promoter sequence consisting of -10 (TACAAT) and -35  
(TTGAAT) regions separated by a 17 bp spacer, which is  
probably recognized by σ^A RNA polymerase. No other clear  
5'-end of the ywkAB transcript was detected in the region  
(nucleotides about +120 to -140) covering the trans-  
lation initiation nucleotide of ywkA (+80) by our primer  
extension experiments, including ones using another more  
downstream primer (data not shown).

In conclusion, the ywkAB genes are cotranscribed to  
produce a 2·8 kb transcript from the transcription initiation  
site (Fig. 5); this transcription is induced during growth on  
malate and this regulation is independent of CcpA.

Genetic screening for regulator(s) involved in  
the control of ywkA transcription

The ywkAB genes are not physically linked to any gene  
encoding a putative transcriptional regulator. So, we decided  
to search for a regulator of ywkAB transcription by a genetic  
screening. A mini-Tn10 transposant library was obtained

Fig. 3. Characterization of the enzymic activity of YwkA.  
Addition of lactate dehydrogenase after the end of the first  
reaction (no further increase of A₄₃₀) results in reoxidation of  
NADH, thus showing conversion of pyruvate into lactate.

determined to be 6·6 mM. Kₘ, kcat and kcat/Kₘ values were  
1·1 mM, 112 s⁻¹ and 102 s⁻¹ mM⁻¹, respectively, for  
NAD, and 9·0 mM, 88 s⁻¹, and 10 s⁻¹ mM⁻¹, respectively,  
for NADP. Thus, we concluded that YwkA is a malic enzyme  
with a dual specificity for NAD and NADP as a cofactor but  
with a strong preference for NAD [10-fold higher catalytic  
efficiency (kcat/Kₘ) with NAD than with NADP].

Transcriptional characterization of ywkA

RNA samples prepared from cells of strains 168CA, 1A250  
and 1A147 (ccpA1) grown in malate or glucose minimal  
medium were subjected to Northern analysis using specific  
probes for ywkA, ywkB, and gapB, a key gluconeogenic  
gene. As shown in Fig. 4, a specific transcript (2·8 kb) was  
detected with the ywkA probe only when cells were grown on  
malate (lanes 2, 4 and 6). This transcript is able to cover the  

| ywkA | ywkB | gapB |
|------|------|------|
| 1    | 1    | 1    |
| 2    | 2    | 2    |
| 3    | 3    | 3    |
| 4    | 4    | 4    |
| 5    | 5    | 5    |
| 6    | 6    | 6    |

2.8 kb →  
← 1.7 kb

Fig. 4. Northern analysis of the ywkA, ywkB and gapB transcripts. The RNA samples from strains 168CA (lanes 1 and 2),  
1A250 (lanes 3 and 4) and 1A147 (lanes 5 and 6) grown on glucose (lanes 1, 3 and 5) or malate (lanes 2, 4 and 6) were  
subjected to Northern analyses using ywkA, ywkB and gapB probes. The specific bands for ywkA and ywkB were observed  
at 2·8 kb, while those for gapB were observed at 1·7 kb. Two faint bands indicated by arrowheads at about 1·5 and 3 kb are  
due to nonspecific hybridization to rRNAs.

T. Doan and others

![image](https://i.imgur.com/your_image_url.png)

GATC12

GTAATTTTCTTCTCTTGAAATTTTGATTTCGACAGGCATACAATAGCTTACGTAATGTAAA

-35 -10 +1

Fig. 5. Mapping of the 5′ end of the ywkAB transcript. Reverse transcriptase reaction was performed using total RNA from strain 168CA grown in malate (lane 1) or glucose (lane 2). Lanes G, A, T and C correspond to the dideoxy sequencing ladder obtained with the same primer as in the reverse transcriptase reaction. The transcriptional start site is indicated by an arrow on the right side of the panel. On the left side, the 5′ end position is marked by an arrow (+1) in the nucleotide sequence of the non-coding strand of the ywkA promoter region, where probable −10 and −35 regions are underlined.

from *B. subtilis* strain PS1716, which contains an ectopic ywkA′::lacZ reporter fusion. This library was then screened on plates for deregulation of ywkA expression (see Methods). Several clones from independent pools of transposants were isolated in which ywkA was no longer induced in the presence of malate. The gene targeted by the transposon in 18 of these clones was identified. YufL was the gene most frequently found to be interrupted (eight occurrences). Back-crossing experiments from these eight transposants indicated that the loss of ywkA inducibility was associated with the insertion of the transposon. As predicted on the basis of its nucleotide sequence, yufL would encode the membrane-bound sensor-kinase of a TCS, associated with the transcriptional response-regulator encoded by the putatively co-transcribed gene yufM. However, no clone harbouring a transposon inserted into the yufM ORF was picked up in this genetic screening.

The TCS YufL/YufM is required for ywkA induction by malate

To confirm the role of the YufL/YufM TCS in the regulation of ywkA transcription, we introduced a yufM disruption into strain PS1716 and analysed the expression of the ywkA′::lacZ reporter fusion in the resulting strain (GTD132). Strain GTD134, a PS1716-derived strain in which the yufL gene is disrupted by a plasmidic cassette that places the downstream gene yufM under the control of a Pspac promoter, and the parental strain PS1716 were also tested in parallel. The β-galactosidase activity synthesized by these strains grown in the presence of glucose or malate as sole carbon source was measured (Table 3). The results showed that ywkA expression was no longer induced by malate in a yufL or yufM mutant strain. In the yufL mutant strain (GTD134), expression of ywkA was restored after the artificial overexpression of yufM by addition of IPTG,

Table 3. Effect of yufL or yufM mutations on ywkA′::lacZ reporter fusion expression

| Strain | β-Galactosidase activity* [U (mg protein)^−1^] |
| --- | --- |
|  | Glucose | Malate |
| PS1716 [amyE′::PywkA:::lacZ] | 8 | 655 |
| GTD134 [amyE′::PywkA:::lacZ, yufL′::pEC23(kan)] | 3 | 14 |
| GTD134 [amyE′::PywkA:::lacZ, yufL′::pEC23(kan)] | 250 (+IPTG) | 368 (+IPTG) |
| GTD132 [amyE′::PywkA:::lacZ, yufM′::pEC23(kan)] | 5 | 7 |

*Activity was assayed at least three times from independent clones; representative results from one series are shown. Strains were cultivated in glucose or malate CQTHC media (+IPTG when indicated).

2338 Microbiology 149
independently of the carbon source, glucose or malate, present in the medium. These findings thus demonstrated that the YufL/YufM TCS is required for ywkA induction in response to the presence of malate.

The growth rate and the biomass yield of the yufM and of the yufL mutant strains in the malate medium was significantly lower than that of the parental strain (doubling time 1·6 h versus 1·2 h; final OD<sub>600</sub> 0·7 versus 1·5, respectively). This observation suggests that the YufL/YufM TCS positively controls the expression of other genes that, in contrast to ywkA, are involved in the utilization of malate as carbon source for growth.

### YufM is able to bind specifically to the regulatory region of ywkA

A His-tagged YufM protein was overexpressed in E. coli and purified to apparent homogeneity (a figure showing the purification is available as supplementary data with the online version of this paper at http://mic.sgmjournals.org). A gel mobility shift assay was then performed to test the capacity of YufM to bind to the ywkA regulatory region. The assay, which was carried out in the presence of a large excess of unspecific DNA competitor, showed that YufM can bind directly and specifically to the regulatory region of ywkA in a concentration-dependent manner (Fig. 6). The shift of the probe was complete only when the protein/DNA concentration ratio exceeded 150; this suggests that, as demonstrated for other TCS response regulators (Perego & Hoch, 2002), YufM requires to be phosphorylated to be fully active in DNA binding.

### YufM protects a 50 bp DNA sequence upstream of the ywkA promoter sequence

To define more precisely the YufM binding region, DNase I footprinting experiments were performed. The 277 bp fragment present upstream of the ywkA coding sequence was labelled on one strand or the other, incubated with an increasing amount of YufM, and then hydrolysed by DNase I. As revealed after electrophoresis (Fig. 7a), YufM protected a 50 bp sequence shifted three bases from one strand to the opposite one (Fig. 7b). This protected area is located just upstream of the –35 sequence of the ywkA promoter, in agreement with the transcriptional activator activity of YufM. Sequence examination of that protected region revealed two distinct perfect direct repeats (Fig. 7b): ATAAAAAAATTN<sub>11</sub>ATAAAAAAAATT

and TAAGTAN<sub>11</sub>TAAGTA. Searches for these motifs (with one mismatch allowed in each sequence) in the B. subtilis complete genome sequence revealed no other occurrences. In particular, no sequence similar to these direct repeats can be detected in the region protected by YufM upstream of two other target genes of YufM (Tanaka et al., 2003). This suggests that none of these DNA sequences is strictly required for efficient binding of YufM (see also Discussion).

---

## DISCUSSION

In this study, we have investigated the physiological function and the transcriptional regulation of ywkA, one of the four paralogous genes of B. subtilis that could be predicted to encode malic enzymes. On the basis of the results of a global transcriptomic analysis using DNA microarrays, we have further studied the expression pattern of the ywkA gene through lacZ reporter fusion, Northern and primer extension analyses. We have been able to demonstrate that ywkA expression is specifically induced by malate. A genetic study then identified the uncharacterized YufL/YufM TCS as the regulatory system necessary for the activation of ywkA transcription in response to the presence of malate in the culture medium. *In vitro* experiments demonstrated that YufM is able to bind directly to a region located just upstream of the ywkA promoter. Thus we propose to rename this TCS MalK/MalR for **malate sensor kinase/malate regulator** component. On the other hand, if our enzymic study clearly demonstrated that YwkA is a malic enzyme, our phenotypic analysis showed that ywkA is not necessary for utilization of malate as sole carbon source for growth.

The enzymic characterization reported here demonstrated that YwkA exhibits a malic enzyme activity and excludes the possibility that it is a malolactic enzyme or a malate dehydrogenase. Furthermore, it showed that YwkA has a 10-fold higher catalytic efficiency with NAD as a cofactor than with NADP. We therefore propose to class YwkA in the group of NAD-dependent malic enzymes, EC 1.1.1.38, and to rename its structural gene maeA for **malic enzyme A**.

Interestingly, our transcriptome analysis indicated that either malS or ytsJ, two paralogues of ywkA, were significantly and similarly transcribed in the presence of glucose or malate, suggesting that their expression is not regulated by malate nor by the glycolytic or gluconeogenic carbon flux

---

| 0 | 0.04 | 0.19 | 0.38 | 1.86 | 3.73 | 7.46 | 37.31 |
| --- | --- | --- | --- | --- | --- | --- | --- |
| pmol of YufM |
| free probe |

---

Fig. 6. Specific binding of His<sub>6</sub>-YufM to the promoter region of the ywkA operon. The 277 bp DNA fragment inserted into plasmid pPS61 (approx. 0·05 pmol) was labelled and then incubated with no or increasing amounts of His<sub>6</sub>-YufM in the presence of a large excess of unspecific DNA.

http://mic.sgmjournals.org   2339
T. Doan and others

(a)
coding strand
non-coding strand
[YufM]
[YufM]

1 2 3 4 5 6 7 8
1 2 3 4 5 6 7 8

(b)

AATA[TTCAATTTAAAAATTTAAATAAGTATATTAAAAAATTAAGTAATT]
+1
TTCTTCTC[TTGAATTTTGATTTTGACAGGCATACAATAGCTTACGT...ATG]
-35
-10

Fig. 7. (a) DNase I footprinting analysis of the interaction between YufM and the promoter region of ywkA examined on the coding and the non-coding strands. Approximately 0·25 pmol of labelled DNA fragment was incubated with no (lanes 2 and 7) or increasing amounts of His₆-YufM (lanes 3, 74·58 pmol; lanes 4, 37·29 pmol; lanes 5, 9·32 pmol; lanes 6, 1·86 pmol). Lanes 1 and 8, G+A sequencing ladder obtained from the same DNA fragment. (b) Nucleotide sequence of the ywkA promoter region. The protected region is boxed: the protected nucleotides on both strands are in normal bold letters; italic letters indicate nucleotides that are protected only on the coding (5′ side) or the non-coding (3′ side) strand. Two direct repeats are indicated by continued or dashed arrows.

orientation. By contrast, ywkA transcription was induced only in the presence of malate in the culture medium. Further addition of glucose or another glycolytic carbon source to the medium did not affect ywkA expression. Thus, ywkA is not subject to carbon catabolite repression. YwkA was not expressed during growth on fumarate or succinate plus glutamate, although these compounds enter the TCA cycle upstream of malate and thus very probably lead to the elevation of the intracellular malate concentration. It is thus very likely that extracellular malate is the signal for ywkA induction. Of course this does not exclude the possibility that the permeation of malate would be necessary for efficient transduction of the signal and thus full induction of ywkA transcription. The effects of the inactivation of the different putative malate transporters on the inducibility of ywkA need to be studied to test this possibility.

To try to get some clues concerning the biological role of ywkA, the regulatory mechanism of its expression has been investigated. Our analysis demonstrated that the YufL/YufM TCS is the positive regulatory system that mediates the induction of ywkA expression in response to the presence of malate in the culture medium, YufM being a direct transcriptional activator of ywkA transcription. This conclusion is in agreement with the results of a global transcriptome analysis performed by Kobayashi et al. (2001)
indicating that $ywkA$ expression is ninefold upregulated in a $yufL$ mutant strain overexpressing YufM, compared to a wild-type strain. The YufL/YufM TCS belongs to the CitA/ CitB family of TCSs. The members of this family that have already been characterized are CitA/CitB from *E. coli* (Kaspar & Bott, 2002) and *Klebsiella pneumoniae* (Bott et al., 1995; Kaspar et al., 1999), CitS/CitT from *B. subtilis* (Yamamoto et al., 2000) and *Streptomyces coelicolor*, YdbF/ YdbG from *B. subtilis* (Asai et al., 2000) and DcuS/DcuR from *E. coli* (Zientz et al., 1998; Golby et al., 1999). All these TCSs are involved in the control of genes required for TCA cycle intermediates import and utilization. Sequence analyses strongly suggest that YufL is a transmembrane protein containing an extracellular sensor domain and a cytoplasmic kinase domain, as shown for the highly similar *E. coli* DcuS protein (Golby et al., 1999). Thus, YufL would be able to modulate YufM DNA-binding activity by phosphorylation/dephosphorylation depending on the presence of extracellular malate. The transcriptome analysis performed by Kobayashi et al. (2001) has revealed 97 other differentially expressed genes in the $yufL$ mutant strain overexpressing $yufM$ compared to the wild-type strain. Because this list included several competence genes, and especially $comK$ and most of the members of the ComK regulon (Berka et al., 2002; Ogura et al., 2002), the authors suggested that the YufL/YufM TCS would be involved in competence development. However, we found no drastic transformability defect, if any, of $yufL$, $yufM$ or $ywkA$ mutant strains (T. Doan & S. Aymerich, unpublished data). The study of the genome-wide expression changes dependent on ComK, recently reported by Berka et al. (2002), led these authors to conclude that the ComK regulon defines a growth-arrested state, distinct from sporulation, of which competence for genetic transformation is but one notable feature. They suggest that this is a unique adaptation to stress and that it be termed the ‘K-state’. Thus the YufL/ YufM TCS could participate in the signalling of the ‘K-state’.

Interestingly, simultaneously with our work, the expression of $maeN$ ($yufR$), a gene located 5·3 kb downstream of $yufL$- $yufM$ on the chromosome, has been found to be activated by the YufL/YufM TCS in the presence of malate (Tanaka et al., 2003). Wei et al. (2000) had previously reported that MaeN exhibits a malate–Na⁺ symporter activity. Ogasawara and colleagues indeed found that MaeN, and thus YufL/YufM, are essential for utilization of malate as sole carbon source for growth (Tanaka et al., 2003). This is in agreement with the growth defect on malate associated with the $yufL$ and the $yufM$ mutations that we observed during the test of their effects on $ywkA$ expression; however, the defect was only partial in our experiments because the medium used contained not only malate but also glutamine and casein hydrolysate as carbon sources (Table 3). The coregulation of $ywkA$ and $maeN$ is of course physiologically relevant since MaeN could provide the YwkA malic enzyme with cytoplasmic malate. Furthermore, Krulwich and coworkers (Ito et al., 2000) have shown that, in some particular conditions, $maeN$ expression was coupled to the expression of the

downstream $mrp$ gene cluster involved in alkali resistance and pH homeostasis. These authors suggested a possible requirement of coordinated Na⁺–malate symport and Na⁺ re-extrusion in exchange for H⁺. The protonmotive force could be generated by secondary NADH dehydrogenases putatively encoded by some $mrp$ genes. Therefore, YwkA could be involved in the regeneration of NADH during this process.

Ogasawara and colleagues have also found that the expression of $yflS$, putatively encoding a 2-oxoglutarate/malate translocator, was under the control of the YufL/YufM TCS (Tanaka et al., 2003). Indeed, $maeN$ and $yflS$ have been detected as being up-regulated (ratios Mal/Glc 4·2 and 2·7, respectively) during growth on malate in our transcriptomic analysis. By contrast, they were not clearly identified (ratios 1·7 and 1·5) in the transcriptomic analysis of Kobayashi et al. (2001), which compared a $yufL$ mutant strain overexpressing YufM with the wild-type strain. We have compared the sequence of the region protected by YufM upstream of the $ywkA$, the $maeN$ and the $yflS$ promoters and found no significant similarities. The most conspicuous feature of these three regions is their very high A+T content (88, 83 and 82 mol%, upstream of $ywkA$, $maeN$ and $yflS$, respectively). On the other hand, no other occurrence of one or the other direct repeat included in the region protected by YufM upstream of $ywkA$ (Fig. 6) can be detected in the complete *B. subtilis* genome. Thus one should conclude that the YufM DNA-binding activator does not recognize a unique primary sequence, a property similar to that of several DNA-binding components of other well-characterized TCSs (Perego & Hoch, 2002).

YwkA does have malic enzyme activity but it does not contribute to the very efficient utilization of malate as sole carbon source for growth by *B. subtilis*. This is in agreement with the fact that $ywkA$ is not subject to carbon catabolite repression. Because neither a $malS$ nor a double $ywkA malS$ mutant strain showed any growth defect, it can also be concluded that, like $ywkA$, $malS$ is not necessary for growth on malate. By contrast, the $ytsJ$ gene is required for efficient growth on malate, as well as on other TCA cycle intermediates. $ytsJ$ is expressed constitutively during the exponential phase of growth and encodes an NADP- dependent malic enzyme (T. Doan, G. Lerondel & S. Aymerich, unpublished data). YtsJ could thus be required for its capacity to catalyse an anaplerotic reaction that allows the TCA cycle to function as a real cycle when the only carbon source is one intermediate of this cycle, or for its capacity to regenerate NADPH necessary for numerous anabolic reactions, or for both of these activities. We would propose that $ywkA$ is involved in a more specific biological process necessary only or principally when malate is present in the extracellular environment. The study of the $ywkB$ gene, of which no clear orthologue can be detected in the genome of other bacilli, and the identification of the activity of more genes belonging to the $yufL/YufM$ regulon, should help to develop this hypothesis.
T. Doan and others

# ACKNOWLEDGEMENTS

We thank Patrick Stragier for the generous gift of plasmid pEC23 (unpublished). We are very grateful to Naotake Ogasawara and Uwe Sauer for communicating results before publication and for very helpful discussions. We thank Claude Gaillardin and Josef Deutscher for valuable discussions and for their continuous encouragement, and Dominique Le Coq and Ivan Mijakovic for their comments on the manuscript.

This work was supported by EU Quality of Life and Management of Living Resources, programme grant QL GZ-CT-1999-O1455, and by a Grand-in-Aid for Scientific Research on Priority Areas from the Ministry of Education, Science, Sports and Culture of Japan. Stéphane Aymerich, Guillaume Lerondel and Thierry Doan are on the staff at INRA (France).

# REFERENCES

Anagnostopoulos, C. & Spizizen, J. (1961). Requirements for transformation in *Bacillus subtilis*. *J Bacteriol* 81, 741–746.

Ansanay, V., Dequin, S., Blondin, B. & Barre, P. (1993). Cloning, sequence and expression of the gene encoding the malolactic enzyme from *Lactococcus lactis*. *FEBS Lett* 332, 74–80.

Arthurs, C. E. & Lloyd, D. (1999). Kinetics, stereospecificity, and expression of the malolactic enzyme. *Appl Environ Microbiol* 65, 3360–3363.

Asai, K., Baik, S. H., Kasahara, Y., Moriya, S. & Ogasawara, N. (2000). Regulation of the transport system for $C_4$-dicarboxylic acids in *Bacillus subtilis*. *Microbiology* 146, 263–271.

Aymerich, S., Gonzy-Treboul, G. & Steinmetz, M. (1986). $5'$-noncoding region *sacR* is the target of all identified regulation affecting the levansucrase gene in *Bacillus subtilis*. *J Bacteriol* 166, 993–998.

Bartolucci, S., Rella, R., Guagliardi, A., Raia, C. A., Gambacorta, A., De Rosa, M. & Rossi, M. (1987). Malic enzyme from archaeabacterium *Sulfolobus solfataricus*. Purification, structure, and kinetic properties. *J Biol Chem* 262, 7725–7731.

Berka, R. M., Hahn, J., Albano, M., Draskovic, I., Persuh, M., Cui, X., Sloma, A., Widner, W. & Dubnau, D. (2002). Microarray analysis of the *Bacillus subtilis* K-state: genome-wide expression changes dependent on ComK. *Mol Microbiol* 43, 1331–1345.

Bott, M., Meyer, M. & Dimroth, P. (1995). Regulation of anaerobic citrate metabolism in *Klebsiella pneumoniae*. *Mol Microbiol* 18, 533–546.

Bradford, M. M. (1976). A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. *Anal Biochem* 72, 248–254.

Denayrolles, M., Aigle, M. & Lonvaud-Funel, A. (1994). Cloning and sequence analysis of the gene encoding *Lactococcus lactis* malolactic enzyme: relationships with malic enzymes. *FEMS Microbiol Lett* 116, 79–86.

Dervyn, E., Suski, C., Daniel, R., Bruand, C., Chapuis, J., Errington, J., Janniere, L. & Ehrlich, S. D. (2001). Two essential DNA polymerases at the bacterial replication fork. *Science* 294, 1716–1719.

Diesterhaft, M. D. & Freese, E. (1973). Role of pyruvate carboxylase, phosphoenolpyruvate carboxykinase, and malic enzyme during growth and sporulation of *Bacillus subtilis*. *J Biol Chem* 248, 6062–6070.

Doan, T. & Aymerich, S. (2003). Regulation of the central glycolytic genes in *Bacillus subtilis*: binding of the repressor CggR to its single DNA target sequence is modulated by fructose-1, 6-bisphosphate. *Mol Microbiol* 47, 1709–1721.

Driscoll, B. T. & Finan, T. M. (1993). NAD⁺-dependent malic enzyme of *Rhizobium meliloti* is required for symbiotic nitrogen fixation. *Mol Microbiol* 7, 865–873.

Driscoll, B. T. & Finan, T. M. (1996). NADP⁺-dependent malic enzyme of *Rhizobium meliloti*. *J Bacteriol* 178, 2224–2231.

Driscoll, B. T. & Finan, T. M. (1997). Properties of NAD⁺- and NADP⁺-dependent malic enzymes of *Rhizobium (Sinorhizobium) meliloti* and differential expression of their genes in nitrogen-fixing bacteroids. *Microbiology* 143, 489–498.

Dunn, M. F. (1998). Tricarboxylic acid cycle and anaplerotic enzymes in rhizobia. *FEMS Microbiol Rev* 22, 105–123.

Ferrari, F. A., Ferrari, E. & Hoch, J. A. (1982). Chromosomal location of a *Bacillus subtilis* DNA fragment uniquely transcribed by sigma-28-containing RNA polymerase. *J Bacteriol* 152, 780–785.

Fillinger, S., Boschi-Muller, S., Azza, S., Dervyn, E., Branlant, G. & Aymerich, S. (2000). Two glyceraldehyde-3-phosphate dehydrogenases with opposite physiological roles in a nonphotosynthetic bacterium. *J Biol Chem* 275, 14031–14037.

Fujita, Y. & Freese, E. (1981). Isolation and properties of a *Bacillus subtilis* mutant unable to produce fructose-bisphosphatase. *J Bacteriol* 145, 760–767.

Golby, P., Davies, S., Kelly, D. J., Guest, J. R. & Andrews, S. C. (1999). Identification and characterization of a two-component sensor-kinase and response-regulator system (DcuS-DcuR) controlling gene expression in response to C4-dicarboxylates in *Escherichia coli*. *J Bacteriol* 181, 1238–1248.

Groisillier, A. & Lonvaud-Funel, A. (1999). Comparison of partial malolactic enzyme gene sequences for phylogenetic analysis of some lactic acid bacteria species and relationships with the malic enzyme. *Int J Syst Bacteriol* 49, 1417–1428.

Guérout-Fleury, A. M., Frandsen, N. & Stragier, P. (1996). Plasmids for ectopic integration in *Bacillus subtilis*. *Gene* 180, 57–61.

Hansen, E. J. & Juni, E. (1974). Two routes for synthesis of phosphoenolpyruvate from C4-dicarboxylic acids in *Escherichia coli*. *Biochem Biophys Res Commun* 59, 1204–1210.

Hansen, E. J. & Juni, E. (1975). Isolation of mutants of *Escherichia coli* lacking NAD- and NADP-linked malic enzyme. *Biochem Biophys Res Commun* 65, 559–566.

Ito, M., Guffanti, A. A., Wang, W. & Krulwich, T. A. (2000). Effects of nonpolar mutations in each of the seven *Bacillus subtilis mrp* genes suggest complex interactions among the gene products in support of Na⁺ and alkali but not cholate resistance. *J Bacteriol* 182, 5663–5670.

Kaspar, S. & Bott, M. (2002). The sensor kinase CitA (DpiB) of *Escherichia coli* functions as a high-affinity citrate receptor. *Arch Microbiol* 177, 313–321.

Kaspar, S., Perozzo, R., Reinelt, S., Meyer, M., Pfister, K., Scapozza, L. & Bott, M. (1999). The periplasmic domain of the histidine autokinase CitA functions as a highly specific citrate receptor. *Mol Microbiol* 33, 858–872.

Kawai, S., Suzuki, H., Yamamoto, K., Inui, M., Yukawa, H. & Kumagai, H. (1996). Purification and characterization of a malic enzyme from the ruminal bacterium *Streptococcus bovis* ATCC 15352 and cloning and sequencing of its gene. *Appl Environ Microbiol* 62, 2692–2700.

K niche, W. & Radler, F. (1982). D-Malic enzyme of *Pseudomonas fluorescens*. *Eur J Biochem* 123, 547–552.

Kobayashi, K., Doi, S., Negoro, S., Urabe, I. & Okada, H. (1989). Structure and properties of malic enzyme from *Bacillus stearothermophilus*. *J Biol Chem* 264, 3200–3205.

Kobayashi, K., Ogura, M., Yamaguchi, H., Yoshida, K., Ogasawara, N., Tanaka, T. & Fujita, Y. (2001). Comprehensive DNA microarray analysis of *Bacillus subtilis* two-component regulatory systems. *J Bacteriol* 183, 7365–7370.
Lamed, R. & Zeikus, J. G. (1981). Thermostable, ammonium-activated malic enzyme of *Clostridium thermocellum*. Biochim Biophys Acta 660, 251–255.

Ludwig, H., Homuth, G., Schmalisch, M., Dyka, F. M., Hecker, M. & Stülke, J. (2001). Transcription of glycolytic genes and operons in *Bacillus subtilis*: evidence for the presence of multiple levels of control of the gapA operon. Mol Microbiol 41, 409–422.

Ludwig, H., Rebhan, N., Blencke, H. M., Merzbacher, M. & Stülke, J. (2002). Control of the glycolytic gapA operon by the catabolite control protein A in *Bacillus subtilis*: a novel mechanism of CcpA-mediated regulation. Mol Microbiol 45, 543–553.

Miller, J. H. (1972). *Experiments in Molecular Genetics*. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory.

Mitsch, M. J., Voegele, R. T., Cowie, A., Osteras, M. & Finan, T. M. (1998). Chimeric structure of the NAD(P)+- and NADP+-dependent malic enzymes of *Rhizobium (Sinorhizobium) meliloti*. J Biol Chem 273, 9330–9336.

Moriya, S., Tsujikawa, E., Hassan, A. K., Asai, K., Kodama, T. & Ogasawara, N. (1998). A *Bacillus subtilis* gene-encoding protein homologous to eukaryotic SMC motor protein is necessary for chromosome partition. Mol Microbiol 29, 179–187.

Murai, T., Tokushige, M., Nagai, J. & Katsuki, H. (1971). Physiological functions of NAD- and NADP-linked malic enzymes in *Escherichia coli*. Biochem Biophys Res Commun 43, 875–881.

Ogura, M., Yamaguchi, H., Yoshida, K., Fujita, Y. & Tanaka, T. (2001). DNA microarray analysis of *Bacillus subtilis* DegU, ComA and PhoP regulons: an approach to comprehensive analysis of *B.* subtilis two-component regulatory systems. Nucleic Acids Res 29, 3804–3813.

Ogura, M., Yamaguchi, H., Kobayashi, K., Ogasawara, N., Fujita, Y. & Tanaka, T. (2002). Whole-genome analysis of genes regulated by the *Bacillus subtilis* competence transcription factor ComK. J Bacteriol 184, 2344–2351.

Perego, M. & Hoch, J. A. (2002). Two-component systems, phosphorelays, and regulation of their activities by phosphatases. In *Bacillus subtilis and its Closest Relatives: from Genes to Cells*, pp. 423–481. Edited by A. L. Sonenshein, J. A. Hoch & R. Losick. Washington, DC: American Society for Microbiology.

Sambrook, J., Fritsch, E. F. & Maniatis, T. (1989). *Molecular Cloning: a Laboratory Manual*. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory.

Steinmetz, M. & Richter, R. (1994). Easy cloning of mini-Tn10 insertions from the *Bacillus subtilis* chromosome. J Bacteriol 176, 1761–1763.

Stols, L. & Donnelly, M. I. (1997). Production of succinic acid through overexpression of NAD+-dependent malic enzyme in an *Escherichia coli* mutant. Appl Environ Microbiol 63, 2695–2701.

Tanaka, K., Kobayashi, K. & Ogasawara, N. (2003). The *Bacillus subtilis* YufLM two-component system regulates the expression of the malate transporters MaeN (YufR) and YflS, and is essential for utilization of malate in minimal medium. Microbiology 149, 2317–2329.

Vagner, V., Dervyn, E. & Ehrlich, S. D. (1998). A vector for systematic gene inactivation in *Bacillus subtilis*. Microbiology 144, 3097–3104.

van der Rest, M. E., Frank, C. & Molenaar, D. (2000). Functions of the membrane-associated and cytoplasmic malate dehydrogenases in the citric acid cycle of *Escherichia coli*. J Bacteriol 182, 6892–6899.

Voegele, R. T., Mitsch, M. J. & Finan, T. M. (1999). Characterization of two members of a novel malic enzyme class. Biochim Biophys Acta 1432, 275–285.

Wei, Y., Guffanti, A. A., Ito, M. & Krulwich, T. A. (2000). *Bacillus subtilis* YqkI is a novel malic/Na+-lactate antiporter that enhances growth on malate at low protonmotive force. J Biol Chem 275, 30287–30292.

Yamamoto, H., Murata, M. & Sekiguchi, J. (2000). The CitST two-component system regulates the expression of the Mg-citrate transporter in *Bacillus subtilis*. Mol Microbiol 37, 898–912.

Yoshida, K., Aoyama, D., Ishio, I., Shibayama, T. & Fujita, Y. (1997). Organization and transcription of the myo-inositol operon, *iol*, of *Bacillus subtilis*. J Bacteriol 179, 4591–4598.

Yoshida, K., Kobayashi, K., Miwa, Y. & 9 other authors (2001). Combined transcriptome and proteome analysis as a powerful approach to study genes under glucose repression in *Bacillus subtilis*. Nucleic Acids Res 29, 683–692.

Zientz, E., Bongaerts, J. & Unden, G. (1998). Fumarate regulation of gene expression in *Escherichia coli* by the DcuSR (*dcuSR* genes) two-component regulatory system. J Bacteriol 180, 5421–5425.