
MOLECULAR AND CELLULAR BIOLOGY, Sept. 2003, p. 6373–6384
0270-7306/03/$08.00+0 DOI: 10.1128/MCB.23.18.6373–6384.2003
Copyright © 2003, American Society for Microbiology. All Rights Reserved.

# The Human I-mfa Domain-Containing Protein, HIC, Interacts with Cyclin T1 and Modulates P-TEFb-Dependent Transcription

Tara M. Young${}^{1,2}$ Qi Wang${}^{1,2}$ Tsafi Pe’ery${}^{1,2,3}$ and Michael B. Mathews${}^{1,2*}$

Departments of Biochemistry and Molecular Biology${}^{1}$ and Medicine,${}^{3}$ New Jersey Medical School, and Graduate School of Biomedical Sciences, University of Medicine and Dentistry of New Jersey,${}^{2}$ Newark, New Jersey 07013-2714

Received 22 November 2002/Returned for modification 21 January 2003/Accepted 27 May 2003

Positive transcription elongation factor b (P-TEFb) hyperphosphorylates the carboxy-terminal domain of RNA polymerase II, permitting productive transcriptional elongation. The cyclin T1 subunit of P-TEFb engages cellular transcription factors as well as the human immunodeficiency virus type 1 (HIV-1) transactivator Tat. To identify potential P-TEFb regulators, we conducted a yeast two-hybrid screen with cyclin T1 as bait. Among the proteins isolated was the human I-mfa domain-containing protein (HIC). HIC has been reported to modulate expression from both cellular and viral promoters via its C-terminal cysteine-rich domain, which is similar to the inhibitor of MyoD family a (I-mfa) protein. We show that HIC binds cyclin T1 in yeast and mammalian cells and that it interacts with intact P-TEFb in mammalian cell extracts. The interaction involves the I-mfa domain of HIC and the regulatory histidine-rich region of cyclin T1. HIC also binds Tat via its I-mfa domain, although the sequence requirements are different. HIC colocalizes with cyclin T1 in nuclear speckle regions and with Tat in the nucleolus. Expression of the HIC cDNA modulates Tat transactivation of the HIV-1 long terminal repeat (LTR) in a cell type-specific fashion. It is mildly inhibitory in CEM cells but stimulates gene expression in HeLa, COS, and NIH 3T3 cells. The isolated I-mfa domain acts as a dominant negative inhibitor. Activation of the HIV-1 LTR by HIC in NIH 3T3 cells occurs at the RNA level and is mediated by direct interactions with P-TEFb.

Cyclin T1 is a component of the essential cellular transcription elongation factor P-TEFb (positive transcription elongation factor b), which is required for the transcription of cellular genes in humans, insects, and nematodes (38). P-TEFb contains cyclin-dependent kinase 9 (CDK9) together with cyclin T1, T2a, T2b, or K. Initially discovered as a transcription factor in *Drosophila* extracts (29), it hyperphosphorylates the carboxy-terminal domain (CTD) of RNA polymerase II, resulting in the transition from abortive to productive elongation. P-TEFb complexes consisting of cyclin T1 and CDK9 serve as a human cellular cofactor for the human immunodeficiency virus type 1 (HIV-1) protein Tat (transactivator of transcription) (13, 16, 47, 49, 56). The Tat-P-TEFb complex stimulates HIV-1 transcriptional elongation via interactions with the TAR (transactivator response) RNA element located at the 5′ ends of nascent viral transcripts. Defects in the Tat-TAR-P-TEFb axis lead to the abortive termination of most viral transcription; in the presence of this complex, however, there is a dramatic increase in the generation of full-length viral transcripts.

In support of its role in transcription elongation, immunodepletion of P-TEFb from HeLa nuclear extracts eliminates both basal and Tat-activated HIV-1 transcription. Supplementing the depleted extract with partially purified wild-type human P-TEFb, but not a kinase-defective mutant, restores both activities (26, 56). Accumulating evidence suggests that P-TEFb also modulates gene expression by interacting, either directly or indirectly, with a number of cellular transcription factors.

Examples include the major histocompatibility complex class II transactivator CIITA (19), NF-κB (1), the p160 coactivator GRIP1 (21), c-Myc (6, 7), FBI-1 (35), Pur α (5), Tat-SF1 (9), the androgen receptor (23), granulin (17), and the CTD itself (45, 51).

Cyclin T1 is 726 amino acids (aa) in length, much larger than most other members of the cyclin family (see Fig. 1A). The protein is highly conserved at its N terminus, which contains the characteristic cyclin domain, whereas its C terminus is unique. Adjacent to the N-terminal cyclin domain is the Tat-TAR recognition motif (TRM). These elements are required for interactions with CDK9, Tat, and TAR. The C-terminal region contains a putative coiled-coil motif, a histidine-rich region, and a serine-rich region. The extreme C terminus is occupied by a PEST sequence which has been reported to destabilize CDK9 in human cells (20). The C-terminal half of cyclin T1 contributes to both basal and Tat-stimulated HIV-1 transcription (9, 36). We therefore postulated that this non-conserved region contains binding sites for regulatory proteins which function to control transcription of cellular and/or viral genes.

To identify such putative cellular regulatory proteins, we conducted a yeast two-hybrid screen with cyclin T1 as bait. Five cDNAs were isolated in this screen, one of which encodes part of the growth factor granulin. Granulin binds to the His-rich domain that was recently shown to be included in the region that interacts with the CTD (45), and it interferes with P-TEFb function (17). Another cDNA clone corresponds to the recently discovered transcriptional regulator known as HIC (human I-mfa domain-containing protein).

The major translation product of the HIC cDNA is a 246-aa protein with a C-terminal I-mfa domain (Fig. 1B) (46). This

* Corresponding author. Mailing address: UMDNJ-New Jersey Medical School, 185 South Orange Ave., P.O. Box 1709, Newark, NJ 07101-1709. Phone: (973) 972-4411. Fax: (973) 972-5594. E-mail: mathews@umdnj.edu.

6374 YOUNG ET AL.

A. Cyclin T1

|  | 254 | 272 | 384 | 425 | 517 | 526 | 560 | 570 | 708 | 726 |
| --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| HIC Binding |  |  |  |  |  |  |  |  |  |  |
| Cyclin Domain TRM | Coiled-Coil | His Rich | Ser Rich | PEST |

1 ………… 229  
1 ……………… 288  
1 ……………………… 418  
1 ……………………………………… 573  

229 ———————————————— 708  
364 —————————————————— 708  
501 —— 530  
480 —— 530  
480 —— 570  
573 —— 708  

B. HIC

Cyclin T1 & Tat Binding

1 ………… 164 ………… 246  

I-mfa Domain  

19 …………………… 186  
177 ———— 246  
115 ———— 246  

FIG. 1. Mapping of interaction in yeast. HIC interacts with the His-rich region of cyclin T1, while cyclin T1 and Tat interact with the C-terminal I-mfa domain of HIC. (A) Top, schematic representation of cyclin T1. Bottom, the indicated cyclin T1 truncations depicted were tested for their ability to interact with the HIC construct isolated from the library screen (aa 19 to 246; nucleotides 645 to 4152). Yeast cells were transformed with Gal4AD-HIC and plasmids expressing Gal4BD-cyclinT1ΔPEST or the truncations illustrated. (B) Top, schematic representation of HIC. Bottom, HIC (aa 19 to 246) truncations were tested for their ability to interact with cyclin T1 and Tat. Yeast cells were transformed with plasmids expressing Gal4AD-HIC or its truncations and with plasmids expressing Gal4BD-cyclin T1 or Gal4BD-Tat. Interactions were determined as described for Table 1. Solid lines, truncations that interacted; dotted lines, truncations that did not interact.

tropin-inducible transcription factor 1 (31). Thus, HIC has been implicated in several transcriptional regulatory pathways.

We show here that HIC interacts with cyclin T1 and also with HIV-1 Tat, in mammalian cells as well as in yeast. The interaction requires the His-rich domain of cyclin T1, which is emerging as a key regulatory region of the protein. HIC colocalizes with cyclin T1 in nuclear speckles and with Tat in the nucleolus. Furthermore, the CDK9 subunit of P-TEFb coimmunoprecipitates with HIC, implying that it binds to the intact elongation factor. All of these interactions are dependent upon the C-terminal I-mfa domain of HIC. Transactivation of the HIV-1 promoter is stimulated by expression of HIC cDNA in HeLa, COS, and NIH 3T3 cells but not in CEM cells. On its own, the I-mfa domain is a potent inhibitor of Tat transactivation. By tethering cyclin T1 to the HIV-1 promoter, we show that HIC regulates transcription via P-TEFb.

MATERIALS AND METHODS

Cell culture. CEM, HeLa, COS7, and NIH 3T3 cells were obtained from the American Type Culture Collection. CEM cells were maintained in RPMI 1640 medium (Invitrogen Life Technologies, Carlsbad, Calif.) supplemented with 10% heat-inactivated (at 50°C for 30 min) fetal bovine serum (Sigma-Aldrich, St. Louis, Mo.). Adherent cells (HeLa, COS7, and NIH 3T3) were maintained in Dulbecco’s modified Eagle’s medium (Invitrogen) supplemented with 10% heat-inactivated fetal bovine serum.

Plasmids and plasmid construction. The yeast two-hybrid plasmids pGBT9, pGADGH, and pACT2 and the yeast strain CG1945 were obtained from Clontech Laboratories Inc. (Palo Alto, Calif.). CyclinT1ΔPEST(1-708) was cloned into pGBT9, generating pGBT9-cyclinT1ΔPEST(1-708), and pACT2, generating pACT2-cyclinT1ΔPEST(1-708). Versions of pGBT9-cyclinT1ΔPEST(1-708) that express aa 1 to 229, 1 to 288, 1 to 418, 1 to 573, 229 to 708, 364 to 708, and 573 to 708 of cyclin T1 were generated. These truncations were generated by restriction enzyme digestion, blunt ending with Klenow fragment or T4 polymerase, and religation of the vector. The restriction enzymes used were StyI-XhoI, SacI-XhoI, NdeI-XhoI, PpuMI-XhoI, EcoRI-StyI, XmaI-PflMI, and XmaI-PpuMI, respectively. Shorter versions encompassing the His-rich region (aa 501 to 530, 480 to 530, and 480 to 570) were constructed by PCR amplification of the corresponding cyclin T1 sequences followed by ligation into pGBT9. CDK9 was subcloned into pACT2, generating pACT2-CDK9. HIV-1 Tat72 was cloned into pACT2 by PCR amplification, generating pACT2-Tat. pGADGH-HIC (aa 19 to 246) was purified from the library screen, and truncations of the vector expressing HIC aa 19 to 246, 177 to 246, and 115 to 246 were generated by restriction enzyme digestion, blunt-ending with Klenow fragment, and religation of the vector. The restriction enzymes used were PshAI-XhoI, EcoRI, and BamHI, respectively. HIC (aa 19 to 246) was also cloned into pGBT9, generating pGBT9-HIC. pGADGH-GrnCDE was isolated from the library screen. pVA3 expressing Gal4BD-p53, pTD1 expressing Gal4AD-simian virus 40 (SV40) T antigen, and pLAM5′ expressing Gal4BD-lamin C were obtained from Clontech Laboratories Inc.

Full-length HIC was generated by a reverse transcription-PCR (RT-PCR) procedure with the Advantage one-step RT-PCR kit from Clontech. The product was cloned into pcDNA3.1 (Invitrogen), generating full-length pHIC for expression in mammalian cells. The p32 isoform of HIC was constructed by PCR amplification of the open reading frame and subcloning into full-length pHIC, thereby deleting the 5′ untranslated region and p40 sequences, generating pHICp32. For fusions with enhanced green fluorescent protein (EGFP), full-length HIC was cloned into pBI-EGFP (Clontech), generating full-length pBI-HIC-EGFP. EGFP-HIC fusion vectors were constructed by inserting PCR-amplified EGFP from pBI-EGFP into pHICp32, generating pEGFP-HIC. The C-terminal fusion, pHIC-EGFP, was constructed by cloning EGFP at the C terminus of the p32 open reading frame. pEGFP-HICA1, pEGFP-HICA2, and pEGFP-HICA3 were generated by PCR amplification of the corresponding sequences and cloning into pEGFP-HIC. pRFP-Tat (Tat72) and pRFP-CyclinT1ΔPEST were generated by PCR amplification of the corresponding sequences into the pDSRed1-C1 vector (Clontech). All FLAG-tagged vectors (pFLAG, pFLAG-HIC, pFLAG-HICA1, pFLAG-HICA2, and pFLAG-HICA3) were constructed by chemically synthesizing complementary oligonucleotides corresponding to the FLAG epitope, annealing them, and cloning the duplex

cysteine-rich domain of HIC is 82 aa long and is 74% identical to the corresponding region of the cellular protein I-mfa (inhibitor of MyoD family a), but the N termini of these proteins exhibit a lesser homology. I-mfa binds to the muscle-specific transcription factor MyoD, leading to decreased expression of muscle-specific genes and resulting in inhibition of myogenesis (3). HIC was found to stimulate Tax-mediated transactivation of a reporter gene driven by the human T-cell leukemia virus type 1 long terminal repeat (LTR) and, conversely, to inhibit Tat-mediated transactivation of a reporter gene driven by the HIV-1 LTR. Both of these effects were dependent upon the C-terminal I-mfa domain of HIC (46). Recently, HIC was shown to bind the Axin complex through its C-terminal I-mfa domain and to regulate signaling via β-catenin and the c-Jun N-terminal kinase (JNK) pathway (22). Transcriptional functions that control cell fate and differentiation events via the Tcf enhancer binding protein and the Wnt signaling pathway have been inferred for XIC, the Xenopus orthologue of HIC (44). Its rat orthologue interacts with the zinc finger protein gonado-

into pcDNA3.1 or the pEGFP-HIC vectors in place of the EGFP tag. pFLAG-
HIC-N and pFLAG-HIC-Imfa were constructed by PCR amplification of the
corresponding sequences from pHICp32 and cloning into pQW3.1Flag, which
contains the sequence encoding the FLAG epitope inserted between the NheI
and HindIII sites of pcDNA3.1. Additional plasmids for expression in mamma-
lian cells were pcDNA3-HATatWT (expressing Tat72) from B. M. Peterlin (10),
pcDNA3.1-HAcyclinT1 from D. H. Price (37), and Gal4BD/Gal4BD-cyclin T1
from L. Lania (25). Reporter plasmids were pHIV1LTR-firefly luciferase and
pRSV-Renilla luciferase (constructed by S. Reza in our laboratory) (40), pCMV-
Renilla luciferase (Promega, Madison, Wis.), and G5-HIV-firefly luciferase
(from L. Lania) (25). The PCNA-firefly luciferase plasmid, containing the pro-
moter (-560 to +60) of the proliferating cell nuclear antigen gene (32), was
provided by T. W. Reichman in our laboratory. pSP-luc+ and pSP-rLuc were
purchased from Promega. Glutathione S-transferase-HIC (GST-HIC), GST-
HIC-N, and GST-HIC-Imfa were constructed through PCR amplification of the
corresponding sequences and cloning into pGEX-6P-1, purchased from Amer-
sham Biosciences (Piscataway, N.J.). All constructs generated for this study were
confirmed by sequencing.

Yeast two-hybrid experiments. Yeast two-hybrid protein interaction experi-
ments and library screening were performed according to the MATCHMAKER
Gal4 two-hybrid user manual (Clontech). pGBT9-cyclinT1ΔPEST was used as
bait to screen a MATCHMAKER HeLa cDNA library (Clontech) contained in
the Gal4 activation domain expression vector, pGADGH. Interactions were
screened by the ability of the cells to grow on selective medium (lacking leucine,
tryptophan, and histidine) and the expression of β-galactosidase, which was
measured by colony filter lift assays according to the manufacturer's instructions.
All putative interacting proteins were subjected to three rounds of screening. The
Gal4AD plasmid was purified from positive clones. Each plasmid carrying a
unique open reading frame was cotransformed with pGBT9-cyclinT1 and tested
for the ability of the encoded proteins to interact by using the above procedure.
Plasmids that gave positive interactions were identified by PCR sequencing.

Coimmunoprecipitation. COS cells were seeded at 10 × 10⁵ cells in a six-well
dish, transfected 12 h later with 4 μg of each plasmid (pFLAG, pFLAG-HIC,
pFLAG-HICA1, pFLAG-HICA2, and pFLAG-HICA3), and harvested at 48 h
posttransfection. Alternatively, 1 μg of each plasmid and 1 μg of pcDNA3.1-
HATat were used for transfections, and cells were harvested at 24 h posttrans-
fection. Transfections were performed with Lipofectamine 2000 (Invitrogen)
according to the manufacturer's instructions. Cells were lysed in 300 μl of lysis
buffer (50 mM Tris-Cl [pH 7.4], 150 mM NaCl, 1 mM EDTA, and 1% Triton
X-100) containing 0.1 mg of phenylmethylsulfonyl fluoride per ml, 1 μg of
pepstatin per ml, 1 μg of leupeptin per ml, 1 μg of aprotinin per ml, and 1 μg of
NaF per ml. Lysis was performed by rocking at 4°C for 30 min. Extracts were
cleared, and supernatants were collected after a 10-min centrifugation at 4°C
(Eppendorf centrifuge, 12,000 rpm). Cell extracts (300 μg of protein) were
incubated overnight at 4°C with anti-FLAG M2 monoclonal antibody (Sigma-
Aldrich) or for 2 h at 4°C with antihemagglutinin (anti-HA) (Y-11) rabbit
polyclonal antibody (Santa Cruz Biotechnology, Santa Cruz, CA). Immune com-
plexes were collected on protein A-Sepharose, washed extensively with lysis
buffer to remove unbound proteins, separated by denaturing polyacrylamide gel
electrophoresis, and subjected to immunoblotting as described previously (39).
Western blots were probed with anti-cyclin T1 (T-18) goat antibody or anti-
CDK9 rabbit antibody (Santa Cruz Biotechnology) or with the antibodies to HA
or FLAG mentioned above.

Confocal microscopy. COS cells were seeded at 2 × 10⁵ cells in six-well dishes
containing collagen-coated coverslips. Localization experiments were performed
by transfecting 1 μg of pEGFP-HIC, pEGFP-HICA1, pEGFP-HICA2, or
pEGFP-HICA3. Colocalization experiments were performed by transfecting 1
μg of an EGFP expression plasmid together with 1 μg of pRFP-Tat or pRFP-
cyclinT1ΔPEST, using polyethylenimine (Sigma). Live cells were viewed with a
Nikon PCM 2000 confocal microscope, and digital images were processed by
using Adobe Photoshop. For Fig. 3D, visualization was performed as described
above except that the cells were fixed on coverslips and viewed with a fluores-
cence microscope.

Gene expression assays. CEM cells (10⁷ cells in 250 μl of RPMI without fetal
bovine serum) were transfected by electroporation, in 1.4-mm cuvettes (USA
Scientific, Inc.), with a Bio-Rad Gene Pulser at 250 V, 960 μF, and 186 Ω. After
electroporation, the cells were transferred into 10-cm-diameter plates containing
10 ml of RPMI supplemented with 10% fetal bovine serum. Cells were harvested
24, 32, 48, and 72 h posttransfection, washed with phosphate-buffered saline, and
lysed in 50 μl of passive lysis buffer according to the instructions of the manu-
facturer (Promega). HeLa and COS cells (1.6 × 10⁵ cells) and NIH3T3 cells (1
× 10⁵ cells) were seeded in six-well dishes and transfected 24 h later by using
Lipofectamine 2000 (Invitrogen). Cells were harvested at 24 h posttransfection

TABLE 1. HIC interactions in the yeast two-hybrid system

| Sequence fused with: | Growth$^a$ | β-Gal$^b$ |
|----------------------|-------------|-----------|
| Gal4BD              | Gal4AD      |           |
| Cyclin T1           | CDK9        | +         |
|                     | CDK9        | -         |
| Cyclin T1           | —           | -         |
| Cyclin T1           | HIC         | +         |
|                     | HIC         | -         |
| Lamin C             | HIC         | -         |
| P53                 | HIC         | -         |
| HIC                 | Cyclin T1   | +         |
| HIC                 | Tat         | +         |
| HIC                 | CDK9        | -         |
| HIC                 | Granulin-CDE | -         |
| HIC                 | HIC         | +         |
| Lamin C             | Tat         | -         |

$a$ Growth on selective medium (lacking leucine, tryptophan, and histidine).

$b$ β-Galactosidase activity determined by using colony filter lift assays.

$c$ —, empty vector.

and lysed in 300 μl of passive lysis buffer (Promega). Luciferase assays were
performed with the Promega dual luciferase reporter system according to the
manufacturer’s instructions. Data are normalized to internal controls as specified
in the figure legends.

RNase protection assay. NIH 3T3 cells (6 × 10⁵ cells) were seeded in 10-cm-
diameter plates and transfected 24 h later by using Lipofectamine 2000. Cells
were harvested at 24 h posttransfection by lysis in 0.5% NP-40. Extracts were
centrifuged at 200 × g for 2 min, and then cytoplasmic RNA and nuclear RNA
were isolated from the supernatant and pellet, respectively, by using Trizol
(Invitrogen) according to the manufacturer’s instructions. The RNase protection
assay was performed with 10 μg of cytoplasmic RNA and 3 μg of nuclear RNA,
using the RPAIII kit from Ambion (Austin, Tex.) according to the manufactur-
er’s instructions. pSp-luc+ and pSP-rLuc were linearized with HincII and BsaAI,
respectively. Radiolabeled RNA was generated by transcription of the digested
plasmids with T7 RNA polymerase in the presence of [$\alpha$-³²P]UTP (ICN Phar-
maceuticals Inc., Costa Mesa, Calif.). Each reaction mixture was incubated for
1 h at 37°C, DNA templates were removed by DNase I digestion, and the RNA
probes were purified by gel electrophoresis. The resulting firefly and Renilla
probes were 390 and 245 nucleotides, respectively.

RESULTS

Cyclin T1 and Tat interact with HIC in yeast. We conducted
a yeast two-hybrid screen with cyclin T1 as bait to identify
cellular proteins that may regulate this transcription factor.
Cyclin T1 (aa 1 to 708), with a deletion of its C-terminal PEST
sequence, was cloned into a vector with the Gal4 DNA-binding
domain (BD), generating a plasmid expressing the Gal4BD-
cyclin T1ΔPEST fusion protein. The cyclin T1 PEST sequence
was removed to eliminate the risk of triggering proteolysis in
yeast cells. For a positive control, yeast cells were cotrans-
formed with this cyclin T1 derivative and a vector expressing
CDK9 fused to the GAL4 activation domain (AD). Growth
occurred on selective medium, as expected, and β-galactosi-
dase activity was detected by colony filter lift assays (Table 1).
Neither growth nor reporter gene expression was seen when
either the CDK9- or the cyclin T1-encoding sequence was
omitted.

Cyclin T1ΔPEST was then used to screen a HeLa cell cDNA
library. Five cDNAs were isolated in this screen, including the
cDNA encoding most of HIC (aa 19 to 246). The Gal4AD-
HIC plasmid was isolated from yeast cells, and the interaction
with cyclin T1 was verified by using purified plasmids (Table 1).
Negative controls included the Gal4BD alone, the “sticky”

fusion protein Gal4BD-lamin C, and Gal4BD-p53 (which was employed, together with Gal4AD-SV40 T antigen, as a standard yeast two-hybrid positive control). For further confirmation, the Gal4 fusion proteins were switched between cyclin T1 and HIC. When cotransformed into yeast, the interaction of Gal4BD-HIC with Gal4AD-cyclin T1ΔPEST was observed (Table 1).

Subsequently, we tested components that modulate HIV-1 transcription for their ability to interact with HIC in yeast. Surprisingly, the viral transactivator HIV-1 Tat was found to interact with HIC but not with lamin C (Table 1). HIC did not interact with CDK9 or with granulin-CDE, a second cyclin T1-interacting protein isolated from the library screen and containing the C-terminal portion of granulin (17). However, HIC appeared to be capable of dimerizing in this system.

Mapping of interaction sites in yeast. The human cyclin T1 cDNA used in the library screen contains several domains and motifs that could participate in the interaction with HIC (Fig. 1A). The cyclin domain is responsible for binding CDK9 and various other proteins, including NF-κB (1) CIITA (19), and GRIP1 (21); the TRM allows the cooperative binding of Tat and TAR to cyclin T1 (12); the His-rich region binds the CTD of RNA polymerase II (45) as well as granulin (17); and the PEST domain is required for the interaction with SCFSKP2 (20). Because cyclin T1 has a long C-terminal extension, we expected that this region would bind additional cellular regulatory proteins, possibly including HIC.

To map the sites of HIC interaction with cyclin T1, N- and C-terminal truncations of cyclin T1 (Fig. 1A) were tested in the yeast two-hybrid system. C-terminal truncations consisting of aa 1 to 229, 1 to 288, and 1 to 418 did not bind HIC, indicating that the N-terminal region of cyclin T1 containing the cyclin domain, TRM, and part of the coiled-coil motif is not sufficient for binding. However, the truncation consisting of aa 1 to 573 retained the ability to bind HIC, implying the existence of a binding site in the C-terminal region which contains the putative coiled-coil and His-rich motif. The corresponding N-terminal truncation, aa 573 to 708, did not bind, showing that the extreme C terminus is not sufficient. Additional N-terminal truncations consisting of aa 229 to 708 and aa 364 to 708 bound HIC, narrowing down the region responsible for binding to aa 364 to 573. Three internal fragments consisting of aa 501 to 530, 480 to 530, and 480 to 570 all interacted with HIC. These truncations defined aa 501 to 530 as a binding site for HIC: this region is almost precisely coterminous with the His-rich region of cyclin T1.

The region of HIC responsible for binding cyclin T1 and Tat was determined by using similar methods. Two N-terminal truncations and one C-terminal truncation of HIC were tested for their ability to bind cyclin T1 and HIV-1 Tat in the yeast two-hybrid assay (Fig. 1B). The N-terminal truncation (aa 115 to 246), which contains the I-mfa domain, interacted with both cyclin T1 and Tat. Neither the C-terminal truncation (aa 19 to 186), which lacks nearly all of the I-mfa domain, nor the N-terminal truncation (aa 177 to 246), which lacks its first 13 aa, was able to bind either cyclin T1 or Tat. These data imply that the I-mfa domain of HIC is required for the interaction with cyclin T1 and Tat and that its N-terminal 13 residues are indispensable.

P-TEFb and HIV-1 Tat bind HIC in vivo. All yeast experiments were performed with the cDNA, isolated from the library screen, expressing a protein composed of HIC aa 19 to 246. Using primers designed according to the published sequence (46), we cloned the full-length cDNA by RT-PCR. It was reported that the full-length sequence expresses a minor form, p40, from an unusual upstream GTG start codon, in addition to the predominant p32 form that begins at an ATG codon. However, only the p32 isoform was detected in vivo in Western blots with antibody directed against a polypeptide common to both forms (46). Correspondingly, we detected very little of the p40 form in cell-free translation experiments with the reticulocyte system and none in cells transfected with a HIC-EGFP fusion construct (data not shown). Therefore, we conducted all further experiments in mammalian cells with cDNAs expressing the p32 isoform of HIC or the truncations Δ1, Δ2, and Δ3 lacking portions of the C-terminal I-mfa domain (Fig. 2A).

To confirm the HIC-cyclin T1 interaction in mammalian cells, COS cells were transfected with vectors expressing FLAG-tagged versions of the proteins or the epitope alone (Fig. 2B). Extracts were subjected to immunoprecipitation with anti-FLAG antibody followed by Western blotting with antibody against cyclin T1. Endogenous cyclin T1 was present in complexes immunoprecipitated with FLAG-HIC (lane 2) but not in complexes immunoprecipitated with FLAGHICΔ1 (lane 3) or the FLAG epitope alone (lane 1), indicating a specific interaction between HIC and cyclin T1 that requires the C-terminal I-mfa domain.

Most of the cyclin T1 in HeLa cell nuclear extracts is bound to CDK9 in the P-TEFb complex (37). To determine whether HIC interacts with intact P-TEFb, a Western blot similar to that in Fig. 2B was probed with antibody against CDK9. Endogenous CDK9 was also present in complexes with FLAG-HIC (Fig. 2C, lane 2) but not with FLAG or FLAG-HICΔ1 (lanes 1 and 3, respectively). Since HIC did not interact directly with CDK9 in the yeast two-hybrid system (Table 1), this indicates that HIC interacts through its I-mfa domain with the cyclin T1 subunit of P-TEFb. The I-mfa domain contains 23 cysteine residues, including a stretch of five consecutive cysteines from position 200 to 204, which might contribute to the binding of cyclin T1 via interactions with its His-rich domain. To determine whether the five-cysteine run plays a part in cyclin T1 binding, we constructed two further C-terminal deletions, HICΔ2 and HICΔ3, which are truncated immediately before and after the five-cysteine stretch, respectively (Fig. 2A). They both failed to coprecipitate with cyclin T1 or CDK9 (Fig. 2B and C), implying that the extreme C-terminal region of the I-mfa domain contains sequences important for its interaction with cyclin T1.

Analogous experiments were performed to determine whether Tat and HIC interact in mammalian cells. COS cells were cotransfected with vectors expressing HA-tagged Tat and FLAG-tagged HIC. Complexes immunoprecipitated with anti-FLAG antibody were probed with anti-HA antibodies. Tat interacted with HIC (Fig. 2D, lane 2) but not with HICΔ1 (lane 3), indicating a specific interaction which is dependent upon the I-mfa domain. In contrast to cyclin T1, HICΔ2 and HICΔ3 both coprecipitated with Tat (lanes 4 and 5, respectively). Notably, the interaction with HICΔ2, which lacks the five-

VOL. 23, 2003                                                                 HIC INTERACTS WITH CYCLIN T1 AND STIMULATES P-TEFb                          6377

A.

HIC

1                       164                      246


HICΔ1

1                       164

HICΔ2

1                       199

HICΔ3

1                       204

I-mfa domain
165
dccvhcilacflfceltlcnivlgqasc
Δ2                     Δ3
gictsea/cccccd/gdemgddcncpcdm
246
dcgimdaccessdcleicmeccgicfps

B.
IP: anti-FLAG
WB: anti-T1

FLAG-HIC


FLAG-HICΔ1


FLAG-HICΔ2


FLAG-HICΔ3


T1


T1

C.
IP: anti-FLAG
WB: anti-CDK9

FLAG-HIC


FLAG-HICΔ1


FLAG-HICΔ2


FLAG-HICΔ3


CDK9


CDK9

D.
IP: anti-FLAG
WB: anti-HA

FLAG + HA-Tat


FLAG-HIC + HA-Tat


FLAG-HICΔ1 + HA-Tat


FLAG-HICΔ2 + HA-Tat


FLAG-HICΔ3 + HA-Tat


Tat


Tat

E.
IP: anti-HA
WB: anti-FLAG

FLAG + HA-Tat


FLAG-HIC + HA-Tat


FLAG-HICΔ1 + HA-Tat


FLAG-HICΔ2 + HA-Tat


FLAG-HICΔ3 + HA-Tat


HIC


HICΔ3


HICΔ2


HIC


HICΔ3


HICΔ2


HICΔ1

FIG. 2. HIC interacts with P-TEFb in vivo, and this interaction is dependent upon the I-mfa domain. (A) Schematic representation of HIC and its truncations, HICΔ1 to -Δ3. The sequence of the I-mfa region (aa 165 to 246) is shown in the box, and the C termini of the Δ2 and Δ3 proteins are marked (/). (B) HIC interacts with cyclin T1 in vivo. COS cells were transfected with plasmids expressing FLAG-HIC (lane 2), FLAG-HICΔ1 (lane 3), FLAG-HICΔ2 (lane 4), FLAG-HICΔ3 (lane 5), or the FLAG tag alone (lane 1), and cell extracts prepared at 24 h posttransfection were subjected to immunoprecipitation (IP) followed by Western blotting (WB). Top, complexes immunoprecipitated with anti-FLAG antibody were resolved in a sodium dodecyl sulfate-polyacrylamide gel, transferred to nitrocellulose, and probed with anti-cyclin T1 antibody. Bottom, as a control, 10% of the inputs used for the immunoprecipitation reactions were analyzed. (C) HIC coimmunoprecipitates with CDK9 in vivo. Transfected COS cell extracts were analyzed as for panel B except that the blot was probed with anti-CDK9 antibody. (D) HIC interacts with Tat in vivo. COS cells cotransfected with HA-Tat plasmid and the FLAG-HIC constructs were analyzed as for panel B except that the blot was probed for Tat with anti-HA antibody. (E) As for panel D, except that Tat-containing complexes were immunoprecipitated with anti-HA antibody and probed for HIC with anti-FLAG antibody.

cysteine run, was significantly weaker than that with full-length HIC and HICΔ3. The same result was obtained when the cell extracts were immunoprecipitated with anti-HA antibodies and Western blotting was performed with anti-FLAG antibodies to detect the HIC fusion (Fig. 2E). Thus, the five-cysteine stretch in the I-mfa domain increases the binding of HIC to Tat, whereas the extreme C-terminal portion is dispensable.

Cyclin T1 and HIV-1 Tat colocalize with HIC. We investigated the localization of HIC, cyclin T1, and Tat in living cells by confocal microscopy with fluorescently tagged versions of the proteins. COS cells were transfected singly with EGFP-tagged HIC or its C-terminally deleted derivatives and with red fluorescent protein (RFP)-tagged cyclin T1ΔPEST or Tat. As expected from previous reports (8, 15, 27, 33), RFP-cyclin T1 was distributed in a speckled pattern in the nucleus but was excluded from the nucleolus, and RFP-Tat localized in the nucleus concentrated in the nucleolus (Fig. 3A). Both RFP-tagged proteins were active in transactivation assays (data not shown). EGFP-HIC was distributed throughout the cell, in both the cytoplasm and nucleus, but was excluded from the nucleolus. The C-terminal fusion protein, HIC-EGFP, gave an identical distribution, indicating that the position of the fusion did not affect the localization of HIC (data not shown). Similar observations for wild-type HIC were reported by Thébault et al. (46). EGFP-HICΔ1 displayed a localization pattern similar to that of wild-type HIC (Fig. 3A), as did EGFP-HICΔ2 and -Δ3 (data not shown), indicating that the I-mfa domain does not influence the generalized intracellular distribution of HIC.

Since HIC and cyclin T1 interact in yeast and coimmunoprecipitate from cell extracts, we wanted to visualize their

6378 YOUNG ET AL. MOL. CELL. BIOL.

A.
![EGFP-HIC](image1.png) ![EGFP-HICΔ1](image2.png)
RFP-Tat RFP-T1

B.
![EGFP-HIC + RFP-T1](image3.png)
EGFP-HIC + RFP-T1
![EGFP-HICΔ1 + RFP-T1](image4.png)
EGFP-HICΔ1 + RFP-T1

C.
![EGFP-HIC + RFP-Tat](image5.png)
EGFP-HIC + RFP-Tat
![EGFP-HICΔ1 + RFP-Tat](image6.png)
EGFP-HICΔ1 + RFP-Tat

D.
![EGFP-HICΔ2 + RFP-Tat](image7.png)
EGFP-HICΔ2 + RFP-Tat
![EGFP-HICΔ3 + RFP-Tat](image8.png)
EGFP-HICΔ3 + RFP-Tat

FIG. 3. HIC colocalizes with Tat and cyclin T1. COS7 cells were transfected or cotransfected with plasmids expressing HIC proteins carrying the EGFP fluorescent tag (green) and cyclin T1 (T1) or Tat proteins carrying the RFP fluorescent tag (red). Live cells were viewed at 24 h posttransfection by confocal microscopy. (A) Cells were transfected singly with the plasmids indicated. Single fluorescence, green or red, is shown in each panel. (B to D) Cells were cotransfected with the plasmids indicated. The green, red, and dual fluorescences are shown from left to right. Yellow signifies colocalization of the two proteins.

interaction in living cells. COS cells were cotransfected with vectors expressing the two fusion proteins. As seen in Fig. 3B, the presence of RFP-cyclin T1 caused a change in the distribution of EGFP-HIC such that the two proteins colocalized in nuclear speckles. In contrast, EGFP-HICΔ1, which lacks the I-mfa region responsible for cyclin T1 binding, did not colocalize with cyclin T1 (Fig. 3B), indicating that the I-mfa domain of HIC is required for the proteins to colocalize as well as to interact biochemically. As expected, EGFP-HICΔ2 and EGFP-HICΔ3, which do not bind cyclin T1 (data not shown). Vectors expressing the EGFP and RFP fluorescent tags were used as negative controls; both proteins were uniformly distributed throughout the cell and did not colocalize with cyclin T1 or HIC (data not shown).

When cells were cotransfected with vectors expressing the HIC and Tat fusion proteins, EGFP-HIC redistributed into the nucleolus, where it colocalized with RFP-Tat (Fig. 3C). EGFP-HICΔ1 did not colocalize with Tat, as expected from the requirement for the I-mfa domain in the HIC-Tat interaction demonstrated in yeast and coimmunoprecipitation assays. Since HIC is excluded from the nucleolus in the absence of Tat, these observations indicate that HIC may be recruited through the interaction of its I-mfa domain with Tat. EGFP-HICΔ2 and EGFP-HICΔ3 were able to colocalize with Tat (Fig. 3C). With these two truncations the nuclear foci were generally slightly less intense than those seen with wild-type HIC, implying that additional factors are required for them to colocalize efficiently with Tat. Thus, a functional correlation exists between the abilities of the HIC deletion mutants to form complexes and colocalize with Tat.

Effects of HIC on Tat-mediated transactivation. For CEM cells, Thébault et al. (46) reported that HIC caused an approximately twofold reduction in the Tat-mediated expression of

FIG. 4. Effects of HIC on Tat transactivation. HIC inhibits Tat transactivation in CEM cells (A and B) but stimulates Tat transactivation in HeLa, COS, and 3T3 cells (C to F). (A and B) CEM cells were electroporated with 4 μg of HIV-1 LTR-firefly luciferase reporter plasmid, 1 μg of CMV-Renilla luciferase reporter plasmid, 0.5 μg of pcDNA3.1-HA-Tat or pcDNA3.1, and 10 μg of pcDNA3.1-HIC expression vector or pcDNA3.1 empty vector. Cells were harvested at the times indicated and assayed for luciferase by using the dual luciferase reporter assay. Cells used for electroporation either were grown at low density and were actively dividing (A) or were grown at a high density and were not actively dividing (B). Data represent Tat transactivation, calculated as the relative firefly/Renilla luciferase activity normalized to the value obtained without Tat, at each time point, in the absence and presence of HIC. Data for each time point are averages from two to four experiments with standard errors. (C) HeLa cells were cotransfected with 100 ng of HIV-1 LTR-firefly luciferase reporter plasmid, 20 ng of CMV-Renilla luciferase reporter plasmid, 5 ng of pcDNA3.1-HA-Tat expression vector or pcDNA3.1 empty vector, and 2 μg of pcDNA3.1-HIC expression vector or pcDNA3.1 empty vector. (D) COS cells were cotransfected as for panel C except that 20 ng of RSV-Renilla luciferase reporter plasmid was used to control for transfection efficiency. (E) NIH 3T3 cells were cotransfected with 100 ng of HIV-1 LTR-firefly luciferase reporter plasmid, 20 ng of CMV-Renilla luciferase reporter plasmid, 5 ng of pcDNA3.1-HA-Tat, 100 ng of pcDNA3.1-HA-cyclinT1 (T1), and 2 μg of pcDNA3.1-HIC expression vector. The total amount of DNA was kept constant in each sample by the addition of the pcDNA3.1 empty vector. In panels C to E, transactivation was measured at 24 h in the presence or absence of HIC, and the data represent averages from three experiments with standard errors. (F) NIH 3T3 cells were cotransfected as for panel E (shaded bars), except that 100 ng of PCNA-firefly luciferase was used in place of HIV-1 firefly luciferase (solid bars). The total amount of DNA was kept constant in each sample by the addition of the pcDNA3.1 empty vector. Transactivation was measured at 24 h and expressed as firefly/Renilla luciferase activity. Data represent the averages from two experiments.

luciferase driven by the HIV1 LTR. We were able to reproduce this result, but in our hands the maximal inhibition did not exceed 30%. This modest effect disappeared when the cells were growing rapidly (compare Fig. 4A and B). On the other hand, expression of HIC cDNA gave a robust stimulation in HeLa cells (Fig. 4C) and COS cells (Fig. 4D). HIC alone had no significant effect on basal gene expression. Tat stimulated HIV-1 transcription 35- to 60-fold, and cotransfection of Tat

and HIC resulted in a further 2.5- to 3-fold stimulation. In contrast, luciferase expression driven by the cytomegalovirus (CMV), Rous sarcoma virus (RSV), and SV40 promoters was either unaffected or only modestly affected (data not shown).

As seen in NIH 3T3 cells (Fig. 4E) transactivation of the HIV-1 LTR in mouse cells is enhanced by the introduction of human cyclin T1 (11, 44). In the absence of Tat, regardless of the presence or absence of cyclin T1, reporter gene expression

was low, and HIC did not show any significant effect on transcription. In the presence of Tat, HIC cDNA stimulated expression ~3-fold in the absence of cyclin T1 and 3.5- to 4-fold in its presence. Equivalent results were obtained by using chloramphenicol acetyltransferase instead of firefly luciferase as reporter (data not shown). HIC stimulation was most evident when the cells were at low confluency. Under these conditions, Tat activated expression from the HIV LTR by ~20-fold, compared to ~10-fold seen previously (12). Specificity was examined by comparison of effects on the HIV-1 and PCNA promoters in NIH 3T3 cells. The PCNA promoter is not affected by cyclin T1 expression (17). As seen in Fig. 4F, HIC stimulated the expression of luciferase from the viral promoter about threefold, whereas expression from the PCNA promoter increased by only ~50%. Therefore, HIC appears to have distinct cell type- and promoter-specific roles in gene expression.

The I-mfa domain inhibits gene expression. Two further HIC deletion mutants (Fig. 5A) were constructed to examine the activities of the I-mfa domain and the N-terminal region separately. HIC-Imfa consists of the FLAG-tagged I-mfa domain (aa 164 to 246), and HIC-N is the FLAG-tagged N terminus of the protein (aa 1 to 163) lacking the I-mfa domain. As expected, both GST–HIC-Imfa and GST-HIC efficiently pulled down cyclin T1 and CDK9 from HeLa cell extract, whereas no binding to GST or GST–HIC-N was detected (Fig. 5B). Correspondingly, cyclin T1 and CDK9 coprecipitated with FLAG–HIC-Imfa but not with FLAG–HIC-N (Fig. 5C) from COS cell extracts. The relatively weak signals obtained with FLAG–HIC-Imfa compared with FLAG–HIC correlate with the relatively low expression level of the isolated I-mfa domain (Fig. 5C, bottom panel). These results show that the isolated I-mfa domain is sufficient, as well as necessary, for the interaction of HIC with P-TEFb.

The HIC N-terminal region elicited a small but reproducible stimulation of gene expression driven by the HIV1 LTR (mean, 80%; range, 50 to 150%) when transfected into HeLa, COS, 293, or NIH 3T3 cells (Fig. 5D). By contrast, the isolated I-mfa domain caused a substantial reduction of reporter gene expression (mean, fivefold; range, three- to ninefold). This inhibitory response is profound in view of the relatively low accumulation of the I-mfa domain (Fig. 5C). Similar results were obtained in experiments using the RSV promoter to drive the Tat gene and the Renilla control construct (data not shown). We conclude that the isolated I-mfa domain is a potent inhibitor of transactivation, presumably because it is acting as a dominant negative repressor.

HIC acts directly on cyclin T1-mediated transcription. The promoter specificity of the HIC effect in transient-expression assays, as well as the interaction of HIC with cyclin T1, suggested that HIC’s action is likely to be at the transcriptional level. We carried out RNase protection assays to examine the effect of HIC on transcripts from the HIV-1–firefly luciferase construct in NIH 3T3 cells expressing human cyclin T1. As expected, Tat increased the abundance of luciferase transcripts in both the nuclear and cytoplasmic fractions (Fig. 6), whereas transcripts from the CMV-Renilla luciferase control vector were unaffected (data not shown). A further increase in firefly, but not Renilla, luciferase transcripts was observed with HIC,

FIG. 5. The I-mfa domain inhibits Tat transactivation. (A) Schematic representation of HIC and its truncations, HIC-N and HIC-Imfa. (B) The HIC I-mfa domain interacts with P-TEFb in vitro. Total HeLa cell extract was incubated with the indicated GST fusion proteins or with GST itself, as described by Hoque et al. (17). Bound proteins were examined by Western blotting (WB) with antibody directed against cyclin T1 (top panel) or CDK9 (bottom panel). (C) HIC I-mfa–P-TEFb complexes in cell extracts. COS cells were transfected with the indicated pcDNA3.1 expression vectors, and immunoprecipitation-Western analysis was conducted as described for Fig. 2B. Immunoprecipitates prepared with anti-Flag antibodies were probed with antibody against cyclin T1 (top panel), CDK9 (middle panel), or the Flag epitope (bottom panel). The second and fourth panels show Western blots of cell extract (without immunoprecipitation, equivalent to 10% of the input) probed with anti-cyclin T1 and anti-CDK 9 antibodies, respectively. (D) HeLa, COS, 293, and NIH 3T3 cells were transfected with 100 ng of HIV-1 LTR-firefly luciferase, 20 ng of CMV-Renilla luciferase, 5 ng of pcDNA3.1-HA-Tat, and 2 μg of pFLAG-HIC-N or pFLAG-HIC-Imfa. The total amount of DNA was kept constant in each sample by the addition of pQW3.1FLAG empty vector. Transactivation was measured at 24 h and expressed as firefly/Renilla luciferase activity normalized to the value obtained with Tat alone. Data represent the averages from three experiments with standard errors.

strongly supporting the conclusion that HIC acts at the level of transcription.

To determine whether Tat is essential for the activation by HIC, we used a version of the HIV promoter in which Gal4 binding sites have been inserted upstream of the promoter and human cyclin T1 is supplied as a fusion with the Gal4 binding domain (25) (Fig. 7A). In this system, Gal4-cyclin T1 activates gene expression in the absence of Tat (25) (Fig. 7B). Introduction of HIC stimulated luciferase expression almost twofold, showing that HIC can stimulate transcription in the absence of Tat. Cotransfection of a plasmid encoding Tat gave an additional ~2-fold activation, in both the absence and presence of HIC. This indicates that Tat can stimulate expression mediated by tethered cyclin T1 and that Tat is not absolutely necessary for the effect of HIC but acts additively with it to stimulate transcription from the HIV-1 LTR.

terminus of HIC is largely identical to the corresponding domain of I-mfa, a cellular factor known to inhibit the transcriptional activity of MyoD family members and Mash2 (3). Both HIC and I-mfa participate in a growing list of functions in transcription, development, and cell signaling. Here, we demonstrate interactions of HIC with cyclin T1/P-TEFb and Tat which are dependent upon the C-terminal I-mfa domain of HIC and can modulate the activity of the HIV promoter.

Interactions of HIC with cyclin T1. HIC binds cyclin T1, but not CDK9, in yeast and coimmunoprecipitates with both components of P-TEFb from mammalian cell extracts (Fig. 2B and C; Table 1). These observations indicate that HIC interacts directly with cyclin T1 present in free form or in the P-TEFb complex. While CDK9 binds to the N-terminal cyclin domain of cyclin T1, HIC interacts with the His-rich domain in the C terminus of cyclin T1. Other proteins that have recently been reported to interact with the C terminus of cyclin T1 include Tat-SF1 (9), the CTD (45), granulin (17), and PIE-1 (51). Both HIC and granulin specifically interact with a discrete 30-aa stretch containing the His-rich region of cyclin T1. The CTD and PIE-1 bind to sites encompassing this domain, although they have not been mapped so finely. All of these proteins have been reported to regulate Tat-mediated transactivation via P-TEFb. P-TEFb phosphorylates the CTD, thereby enabling progressive transcription (38); Tat-SF1 is a Tat-specific cofactor (54, 55); granulin inhibits Tat-mediated transactivation (17); and PIE-1 inhibits transcriptional elongation by P-TEFb (51). Thus, the C-terminal region of cyclin T1, especially its His-rich domain, may serve to bind cellular proteins which modulate P-TEFb's ability to regulate transcription of the HIV-1 LTR as well as cellular targets.

Colocalization of HIC with cyclin T1 in mammalian cells lends support to the idea that they interact functionally. Coexpression with cyclin T1 caused a redistribution of HIC such that it largely colocalized in a characteristic speckled pattern (Fig. 3B). Endogenous cyclin T1 localizes to specific foci in the nucleus. Cyclin T1 has been reported to localize in nuclear speckle regions and to colocalize with a number of splicing factors that also produce a speckled labeling pattern (15). Furthermore, cyclin T1 relocalized CDK9 and Tat to speckles, suggesting that it functions to recruit its binding partners to nuclear speckles and that nuclear speckles are a site of P-TEFb function or storage (33). On the other hand, it has been shown that cyclin T1 foci are juxtaposed with nuclear speckles but not exactly coincident with them and that overexpression of Tat determines recruitment of cyclin T1 outside these compartments (27). Additionally, cyclin T1 interacts via its C-terminal region with promyelocytic leukemia (PML) protein in vivo within specific subnuclear compartments that are coincident with PML nuclear bodies (28). Hence, HIC may be colocalizing with cyclin T1 in nuclear speckles or PML bodies, which contain several proteins known to participate in transcriptional regulation. The overexpression of HIC results in a diffuse nuclear and cytoplasmic staining in the absence of cyclin T1 overexpression but in a nuclear speckling pattern in its presence. This indicates that cyclin T1 is functioning to recruit HIC. The fact that endogenous, as well as overexpressed, cyclin T1 localizes to speckles indicates that its colocalization with HIC may be functionally significant.

Evidence that the HIC–P-TEFb interaction is physiologi-

and CTIIA (19), interact with the cyclin homology domain in the N terminus of cyclin T1. It has been suggested that these proteins serve to recruit P-TEFb to enhancer elements (45). In this scenario, P-TEFb could facilitate productive elongation by linking activators bound to enhancers with transcription complexes. We therefore speculate that one subset of proteins binds enhancer elements and interacts with P-TEFb through the N-terminal region of cyclin T1 and that another subset of proteins binds the C terminus of cyclin T1 and modulates transcription by affecting the interactions of P-TEFb with paused RNA polymerase II elongation complexes.

HIC and Tat transactivation. The HIC cDNA regulates gene expression in a cell-type-specific fashion (Fig. 4 and 5); it inhibits Tat transactivation to some extent in CEM cells but stimulates Tat transactivation in HeLa, COS, and NIH 3T3 cells. Whether this is due to intrinsic differences between cell types or to an imbalance in the levels of factors leading to squelching in CEM cells is presently unclear. Interestingly, the inhibitory and stimulatory effects were both sensitive to cell confluency and growth rate: the inhibition was reduced and the stimulation was increased when the cells were growing rapidly, suggesting that the two phenomena are linked.

Enhanced Tat transactivation could be due to the increased formation of Tat-TAR-P-TEFb complexes resulting from the binding of HIC to both Tat and cyclin T1. Since HIC binds to the His-rich domain of cyclin T1 while Tat binds to the TRM, in principle both proteins could interact with cyclin T1 simultaneously. On the other hand, HIC is capable of stimulating transcription via P-TEFb in the absence of Tat as evidenced in the cyclin T1 tethering experiments (Fig. 7). The observation that supplying Tat gave rise to a further increase in gene expression indicates that Tat is required for maximal activation by HIC and implies that HIC and Tat can activate transcription from the HIV-1 LTR through distinct mechanisms.

Role of the I-mfa domain. To date, the I-mfa domain has been found in two proteins, I-mfa itself and HIC, both of which play roles in transcription regulation and development. The I-mfa domain is necessary for the binding of these proteins to basic helix-loop-helix transcription factors such as MyoD (3), HMG box transcription factors such as Tcf3 and Lef (44), and the scaffolding protein Axin, which participates with β-catenin in the Wnt signaling pathway (22). Correspondingly, deletion of the I-mfa domain eliminated the functions of I-mfa, HIC, and XIC (3). Similarly, the I-mfa domain of HIC is required for its association with Tat and cyclin T1 (Fig. 1 and 2B, D, and E) and for its transcriptional effects (46). Although the I-mfa domain did not appear to influence the distribution of HIC between the nucleus and cytoplasm, its deletion eliminated HIC's colocalization with cyclin T1 and Tat (Fig. 3B).

Smaller deletions from the C terminus of HIC revealed differences between cyclin T1 and Tat in their interactions with the I-mfa domain. Specifically, the binding of Tat was reduced, but not abolished, by deletion of the C terminus of the I-mfa domain, whereas cyclin T1 interaction was eliminated (Fig. 2B, D, and E). Deleting the N-terminal 13 residues of the I-mfa domain prevented it from interacting with both Tat and cyclin T1 (Fig. 1). Remarkably, HIC and I-mfa have 23 conserved cysteine residues out of some 80 residues in the I-mfa domain. These cysteines presumably determine its structure and interactions with other proteins. In the case of cyclin T1, the inter-

action may be dependent on metal ions such as zinc, which could mediate cysteine-histidine interactions with the cyclin T1 His-rich domain. The interactions of the HIC I-mfa domain with Tat may also involve cysteine residues, since Tat is known to bind several proteins through its Cys-rich domain (18, 48).

Finally, it is notable that CDK9 is capable of phosphorylating MyoD in vitro (42) and that overexpression of CDK9 with cyclinT2a (but not cyclin T1) enhances MyoD function and promotes myogenic differentiation (42). Conversely, CDK2 and cdc2 can inactivate MyoD function and inhibit myogenic differentiation by phosphorylation-triggered degradation (14, 43, 52, 53). These findings raise the possibility that HIC participates in a network of controls over myogenic transcription that operate by regulating the phosphorylation status of MyoD.

# ACKNOWLEDGMENTS

We thank Luigi Lania, B. Matija Peterlin, and David H. Price for generously providing plasmids; Chee-Gun Lee for yeast two-hybrid materials and advice; and Paul Burnett and Andrew P. Thomas for advice and help with confocal microscopy.

This work was supported by grant AI31802 from the National Institutes of Health.

# REFERENCES

1. Barbioric, M., R. M. Nissen, S. Kanazawa, N. Jabrane-Ferrat, and B. M. Peterlin. 2001. NF-kappaB binds P-TEFb to stimulate transcriptional elongation by RNA polymerase II. Mol. Cell 8:327–337.
2. Bourgeois, C. F., Y. K. Kim, M. J. Churcher, M. J. West, and J. Karn. 2002. Spt5 cooperates with human immunodeficiency virus type 1 Tat by preventing premature RNA release at terminator sequences. Mol. Cell. Biol. 22: 1079–1093.
3. Chen, C. M., N. Kraut, M. Groudine, and H. Weintraub. 1996. I-mf, a novel myogenic repressor, interacts with members of the MyoD family. Cell 86: 731–741.
4. Chun, R. F., O. J. Semmes, C. Neuveut, and K. T. Jeang. 1998. Modulation of Sp1 phosphorylation by human immunodeficiency virus type 1 Tat. J. Virol. 72:2615–2629.
5. Darbinian, N., B. E. Sawaya, K. Khalili, N. Jaffe, B. Wortman, A. Giordano, and S. Amini. 2001. Functional interaction between cyclin T1/cdk9 and Puralpha determines the level of TNFalpha promoter activation by Tat in glial cells. J. Neuroimmunol. 121:3–11.
6. Eberhardy, S. R., and P. J. Farnham. 2001. c-Myc mediates activation of the cad promoter via a post-RNA polymerase II recruitment mechanism. J. Biol. Chem. 276:48562–48571.
7. Eberhardy, S. R., and P. J. Farnham. 2002. Myc recruits P-TEFb to mediate the final step in the transcriptional activation of the cad promoter. J. Biol. Chem. 277:40156–40162.
8. Efthymiadis, A., L. J. Briggs, and D. A. Jans. 1998. The HIV-1 Tat nuclear localization sequence confers novel nuclear import properties. J. Biol. Chem. 273:1623–1628.
9. Fong, Y. W., and Q. Zhou. 2000. Relief of two built-in autoinhibitory mechanisms in P-TEFb is required for assembly of a multicomponent transcription elongation complex at the human immunodeficiency virus type 1 promoter. Mol. Cell. Biol. 20:5897–5907.
10. Fujinaga, K., R. Taube, J. Wimmer, T. P. Cujec, and B. M. Peterlin. 1999. Interactions between human cyclin T, Tat, and the transactivation response element (TAR) are disrupted by a cysteine to tyrosine substitution found in mouse cyclin T. Proc. Natl. Acad. Sci. USA 96:1285–1290.
11. Garber, M. E., T. P. Mayall, E. M. Suess, J. Meisenhelder, N. E. Thompson, and K. A. Jones. 2000. CDK9 autophosphorylation regulates high-affinity binding of the human immunodeficiency virus type 1 tat-P-TEFb complex to TAR RNA. Mol. Cell. Biol. 20:6958–6969.
12. Garber, M. E., P. Wei, V. N. KewalRamani, T. P. Mayall, C. H. Herrmann, A. P. Rice, D. R. Littman, and K. A. Jones. 1998. The interaction between HIV-1 Tat and human cyclin T1 requires zinc and a critical cysteine residue that is not conserved in the murine CycT1 protein. Genes Dev. 12:3512–3527.
13. Gold, M. O., X. Yang, C. H. Herrmann, and A. P. Rice. 1998. PITALRE, the catalytic subunit of TAK, is required for human immunodeficiency virus Tat transactivation in vivo. J. Virol. 72:4448–4453.
14. Guo, K., and K. Walsh. 1997. Inhibition of myogenesis by multiple cyclin-Cdk complexes. Coordinate regulation of myogenesis and cell cycle activity at the level of E2F. J. Biol. Chem. 272:791–797.
15. Herrmann, C. H., and M. A. Mancini. 2001. The Cdk9 and cyclin T subunits of TAK/P-TEFb localize to splicing factor-rich nuclear speckle regions. J. Cell Sci. 114:1491–1503.
16. Herrmann, C. H., and A. P. Rice. 1995. Lentivirus Tat proteins specifically associate with a cellular protein kinase, TAK, that hyperphosphorylates the carboxyl-terminal domain of the large subunit of RNA polymerase II: candidate for a Tat cofactor. J. Virol. 69:1612–1620.
17. Hoque, M., T. M. Young, C. G. Lee, G. Serrero, M. B. Mathews, and T. Pe’ery. 2003. The growth factor granulin interacts with cyclin T1 and modulates P-TEFb-dependent transcription. Mol. Cell. Biol. 23:1688–1702.
18. Kamine, J., B. Elangovan, T. Subramanian, D. Coleman, and G. Chinnadurai. 1996. Identification of a cellular protein that specifically interacts with the essential cysteine region of the HIV-1 Tat transactivator. Virology 216: 357–366.
19. Kanazawa, S., T. Okamoto, and B. M. Peterlin. 2000. Tat competes with CIITA for the binding to P-TEFb and blocks the expression of MHC class II genes in HIV infection. Immunity 12:61–70.
20. Kiernan, R. E., S. Emiliani, K. Nakayama, A. Castro, J. C. Labbe, T. Lorca, K. Nakayama Ki, and M. Benkirane. 2001. Interaction between cyclin T1 and SCF (SKP2) targets CDK9 for ubiquitination and degradation by the proteasome. Mol. Cell. Biol. 21:7956–7970.
21. Kino, T., O. Slobodskaya, G. N. Pavlakis, and G. P. Chrousos. 2002. Nuclear receptor coactivator p160 proteins enhance the HIV-1 long terminal repeat promoter by bridging promoter-bound factors and the Tat-P-TEFb complex. J. Biol. Chem. 277:2396–2405.
22. Kusano, S., and N. Raab-Traub. 2002. I-mfa domain proteins interact with axin and affect its regulation of the Wnt and c-Jun N-terminal kinase signaling pathways. Mol. Cell. Biol. 22:6303–6405.
23. Lee, D. K., H. O. Duan, and C. Chang. 2001. Androgen receptor interacts with the positive elongation factor P-TEFb and enhances the efficiency of transcriptional elongation. J. Biol. Chem. 276:9978–9984.
24. Luznik, L., M. E. Martone, G. Kraus, Y. Zhang, Y. Xu, M. H. Ellisman, and F. Wong-Staal. 1995. Localization of human immunodeficiency virus Rev in transfected and virus-infected cells. AIDS Res. Hum. Retroviruses 11:795–804.
25. Majello, B., G. Napolitano, A. Giordano, and L. Lania. 1999. Transcriptional regulation by targeted recruitment of cyclin-dependent CDK9 kinase in vivo. Oncogene 18:4598–4605.
26. Mancebo, H. S., G. Lee, J. Flygare, J. Tomassini, P. Luu, Y. Zhu, J. Peng, C. Blau, D. Hazuda, D. Price, and O. Flores. 1997. P-TEFb kinase is required for HIV Tat transcriptional activation in vivo and in vitro. Genes Dev. 11:2633–2644.
27. Marcello, A., R. A. Cinelli, A. Ferrari, A. Signorelli, M. Tyagi, V. Pellegrini, F. Beltram, and M. Giacca. 2001. Visualization of in vivo direct interaction between HIV-1 TAT and human cyclin T1 in specific subcellular compartments by fluorescence resonance energy transfer. J. Biol. Chem. 276:39220–39225.
28. Marcello, A., A. Ferrari, V. Pellegrini, G. Pegoraro, M. Lusic, F. Beltram, and M. Giacca. 2003. Recruitment of human cyclin T1 to nuclear bodies through direct interaction with the PML protein. EMBO J. 22:2156–2166.
29. Marshall, N. F., and D. H. Price. 1995. Purification of P-TEFb, a transcription factor required for the transition into productive elongation. J. Biol. Chem. 270:12335–12338.
30. Michienzi, A., S. Li, J. A. Zaia, and J. J. Rossi. 2002. A nuclear TAR decoy inhibitor of HIV-1 replication. Proc. Natl. Acad. Sci. USA 99:14047–14052.
31. Mizutani, T., K. Yamada, T. Yazawa, T. Okada, T. Minegishi, and K. Miyamoto. 2001. Cloning and characterization of gonadotropin-inducible ovarian transcription factors (GIOT1 and -2) that are novel members of the (Cys)(2)-(His)(2)-type zinc finger protein family. Mol. Endocrinol. 15:1693–1705.
32. Morris, G. F., and M. B. Mathews. 1990. Analysis of the proliferating cell nuclear antigen promoter and its response to adenovirus early region 1. J. Biol. Chem. 265:16116–16125.
33. Napolitano, G., P. Licciardo, R. Carbone, B. Majello, and L. Lania. 2002. CDK9 has the intrinsic property to shuttle between nucleus and cytoplasm, and enhanced expression of cyclin T1 promotes its nuclear localization. J. Cell Physiol. 192:209–215.
34. Nguyen, V. T., T. Kiss, A. A. Michels, and O. Bensaude. 2001. 7SK small nuclear RNA binds to and inhibits the activity of CDK9/cyclin T complexes. Nature 414:322–325.
35. Pendergrast, P. S., C. Wang, N. Hernandez, and S. Huang. 2002. FBI-1 can stimulate HIV-1 Tat activity and is targeted to a novel subnuclear domain that includes the Tat-P-TEFb-containing nuclear speckles. Mol. Biol. Cell 13:915–929.
36. Peng, J., N. F. Marshall, and D. H. Price. 1998. Identification of a cyclin subunit required for the function of Drosophila P-TEFb. J. Biol. Chem. 273:13855–13860.
37. Peng, J., Y. Zhu, J. T. Milton, and D. H. Price. 1998. Identification of multiple cyclin subunits of human P-TEFb. Genes Dev. 12:755–762.
38. Price, D. H. 2000. P-TEFb, a cyclin-dependent kinase controlling elongation by RNA polymerase II. Mol. Cell. Biol. 20:2629–2634.
39. Ramanathan, Y., S. M. Reza, T. M. Young, M. B. Mathews, and T. Pe’ery. 1999. Human and rodent transcription elongation factor P-TEFb: interac-

tions with human immunodeficiency virus type 1 Tat and carboxy-terminal domain substrate. J. Virol. 73:5448–5458.

40. Reza, S. M., M. Rosetti, M. B. Mathews, and T. Pe’ery. 2003. Differential activation of Tat variants in mitogen-stimulated cells: implications for HIV-1 postintegration latency. Virology 310:141–156.

41. Sano, M., M. Abdellatif, H. Oh, M. Xie, L. Bagella, A. Giordano, L. H. Michael, F. J. DeMayo, and M. D. Schneider. 2002. Activation and function of cyclin T-Cdk9 (positive transcription elongation factor-b) in cardiac muscle-cell hypertrophy. Nat. Med. 8:1310–1317.

42. Simone, C., P. Stiegler, L. Bagella, B. Pucci, C. Bellan, G. De Falco, A. De Luca, G. Guanti, P. L. Puri, and A. Giordano. 2002. Activation of MyoD-dependent transcription by cdk9/cyclin T2. Oncogene 21:4137–4148.

43. Skapek, S. X., J. Rhee, P. S. Kim, B. G. Novitch, and A. B. Lassar. 1996. Cyclin-mediated inhibition of muscle gene expression via a mechanism that is independent of pRB hyperphosphorylation. Mol. Cell. Biol. 16:7043–7053.

44. Snider, L., H. Thirlwell, J. R. Miller, R. T. Moon, M. Groudine, and S. J. Tapscott. 2001. Inhibition of Tcf3 binding by I-mfa domain proteins. Mol. Cell. Biol. 21:1866–1873.

45. Taube, R., X. Lin, D. Irwin, K. Fujinaga, and B. M. Peterlin. 2002. Interaction between P-TEFb and the C-terminal domain of RNA polymerase II activates transcriptional elongation from sites upstream or downstream of target genes. Mol. Cell. Biol. 22:321–331.

46. Thébault, S., F. Gachon, I. Lemasson, C. Devaux, and J. M. Mesnard. 2000. Molecular cloning of a novel human I-mfa domain-containing protein that differently regulates human T-cell leukemia virus type I and HIV-1 expression. J. Biol. Chem. 275:4848–4857.

47. Wei, P., M. E. Garber, S. M. Fang, W. H. Fischer, and K. A. Jones. 1998. A novel CDK9-associated C-type cyclin interacts directly with HIV-1 Tat and

mediates its high-affinity, loop-specific binding to TAR RNA. Cell 92:451–462.

48. Weissman, J. D., J. R. Hwang, and D. S. Singer. 2001. Extensive interactions between HIV TAT and TAF(II)250. Biochim. Biophys. Acta 1546:156–163.

49. Yang, X., M. O. Gold, D. N. Tang, D. E. Lewis, E. Aguilar-Cordova, A. P. Rice, and C. H. Herrmann. 1997. TAK, an HIV Tat-associated kinase, is a member of the cyclin-dependent family of protein kinases and is induced by activation of peripheral blood lymphocytes and differentiation of promonocytic cell lines. Proc. Natl. Acad. Sci. USA 94:12331–12336.

50. Yang, Z., Q. Zhu, K. Luo, and Q. Zhou. 2001. The 7SK small nuclear RNA inhibits the CDK9/cyclin T1 kinase to control transcription. Nature 414:317–322.

51. Zhang, F., M. Barboric, T. K. Blackwell, and B. M. Peterlin. 2003. A model of repression: CTD analogs and PIE-1 inhibit transcriptional elongation by P-TEFb. Genes Dev. 17:748–758.

52. Zhang, J. M., Q. Wei, X. Zhao, and B. M. Paterson. 1999. Coupling of the cell cycle and myogenesis through the cyclin D1-dependent interaction of MyoD with cdk4. EMBO J. 18:926–933.

53. Zhang, J. M., X. Zhao, Q. Wei, and B. M. Paterson. 1999. Direct inhibition of G(1) cdk kinase activity by MyoD promotes myoblast cell cycle withdrawal and terminal differentiation. EMBO J. 18:6983–6993.

54. Zhou, Q., and P. A. Sharp. 1995. Novel mechanism and factor for regulation by HIV-1 Tat. EMBO J. 14:321–328.

55. Zhou, Q., and P. A. Sharp. 1996. Tat-SF1: cofactor for stimulation of transcriptional elongation by HIV-1 Tat. Science 274:605–610.

56. Zhu, Y., T. Pe’ery, J. Peng, Y. Ramanathan, N. Marshall, T. Marshall, B. Amendt, M. B. Mathews, and D. H. Price. 1997. Transcription elongation factor P-TEFb is required for HIV-1 tat transactivation in vitro. Genes Dev. 11:2622–2632.
