
JOURNAL OF BACTERIOLOGY, Oct. 2003, p. 5735–5746
0021-9193/03/$08.00+0 DOI: 10.1128/JB.185.19.5735–5746.2003
Copyright © 2003, American Society for Microbiology. All Rights Reserved.

# Genome-Wide Analyses Revealing a Signaling Network of the RcsC-YojN-RcsB Phosphorelay System in *Escherichia coli*

Daisuke Hagiwara,$^{1}$ Masahito Sugiura,$^{1}$ Taku Oshima,$^{2}$ Hirotada Mori,$^{2}$ Hirofumi Aiba,$^{1}$ Takafumi Yamashino,$^{1}$ and Takeshi Mizuno$^{1*}$

**Laboratory of Molecular Microbiology, School of Agriculture, Nagoya University, Chikusa-ku, Nagoya 464-8601,$^{1}$ and Research and Education Center for Genetic Information, Nara Institute of Science and Technology, Ikoma 630-0101,$^{2}$ Japan**

Received 12 May 2003/Accepted 10 July 2003

In *Escherichia coli*, capsular colanic acid polysaccharide synthesis is regulated through the multistep RcsC→YojN→RcsB phosphorelay. By monitoring a hallmark *cps::lacZ* reporter gene, we first searched for physiological stimuli that propagate the Rcs signaling system. The expression of *cps::lacZ* was activated when cells were grown at a low temperature (20°C) in the presence of glucose as a carbon source and in the presence of a relatively high concentration of external zinc (1 mM ZnCl₂). In this Rcs signaling system, the *rcsF* gene product (a putative outer membrane-located lipoprotein) was also an essential signaling component. Based on the defined signaling pathway and physiological stimuli for the Rcs signaling system, we conducted genome-wide analyses with microarrays to clarify the Rcs transcriptome (i.e., Rcs regulon). Thirty-two genes were identified as putative Rcs regulon members; these genes included 15 new genes in addition to 17 of the previously described *cps* genes. Using a set of 37 two-component system mutants, we performed alternative genome-wide analyses. The results showed that the propagation of the zinc-responsive Rcs signaling system was largely dependent on another two-component system, PhoQ/P. Considering the fact that the PhoQ/P signaling system responds to external magnesium, we obtained evidence which supports the view that there is a signaling network that connects the Rcs system with the PhoQ/P system, which coordinately regulates extracellular polysaccharide synthesis in response to the external concentrations of divalent cations.

---

*Escherichia coli* and other enteric bacteria are capable of synthesizing an extracellular polysaccharide capsule, called colanic acid. A large number of genes are involved in capsular polysaccharide synthesis (36). The major gene cluster involved in colanic acid synthesis is located at 45.8 min on the canonical *E. coli* K-12 genetic map (38). The 5′-proximal gene cluster consists of the *wza*, *wzb*, and *wzc* genes, which are followed by the *wca* operon consisting of about 20 genes (http://www.genolist.pasteur.fr/Colibri/). The locus containing these genes was previously referred to as the *cps* operons (the capsular polysaccharide synthesis genes are referred to as *cps* genes for clarity below). The expression of these *cps* genes is coordinately regulated in response to a variety of environmental cues. Nevertheless, the physiological stimuli that induce capsular polysaccharide synthesis are not fully understood.

Extensive genetic studies have revealed that in *E. coli* there is a complex signaling circuitry that modulates expression of the *cps* genes (for reviews, see references 13 and 39). Together with the RcsA transcription factor and the Lon protease (10, 44), the RcsC-mediated histidine (His)→aspartate (Asp) phosphorelay system appears to play a major role in transcriptional regulation of the *cps* genes. By analogy with other His→Asp phosphorelay systems (1, 25), a unique model has been formulated for the multistep Rcs signaling system, in which the RcsC hybrid sensor His kinase senses certain environmental stimuli (42). The downstream signaling component is YojN (a histidine-containing phosphotransfer HPt factor), which serves as an intermediate for the phosphorelay. Eventually, the RcsB response regulator acquires a phosphoryl group from YojN, and the phosphorylated RcsB functions as a DNA-binding transcriptional regulator, which together with RcsA transcribes the *cps* genes. It has been suggested previously that another factor (RcsF) also plays a role in the Rcs signaling system (11). However, the importance of this putative outer membrane lipoprotein remains to be determined.

Interestingly, the Rcs signaling system is common in the sense that there are homologous systems not only in other enteric bacteria but also in nonenteric bacteria, including *Salmonella enterica* serovar Typhi, *Vibrio cholerae*, *Klebsiella pneumoniae*, *Erwinia amylovora*, *Proteus mirabilis*, and *Pseudomonas aeruginosa*. In *S. enterica* serovar Typhi, the Rcs signaling system modulates differential expression of an invasion protein (designated Sip), flagellin, and Vi antigen (2). The synthesis of certain extracellular polysaccharides in *K. pneumoniae* and *E. amylovora* is also regulated through the Rcs signaling systems (4, 28). The dimorphic and motile gram-negative bacterium *P. mirabilis* also uses the Rcs signaling system to regulate its characteristic swarming behavior (3, 15), like *E. coli* (17, 42). The recently released genome sequences of both *V. cholerae* and *P. aeruginosa* revealed that each of these bacteria has a homologous Rcs signaling system (18, 41). These findings indicate that the multistep RcsC→YojN→RcsB phosphorelay system is evolutionarily conserved in a wide variety of bacteria, in which this common adaptive response system is often associated with bacterial virulence and/or pathogenesis.

Needless to say, *E. coli* is the organism of choice for gaining

* Corresponding author. Mailing address: Laboratory of Molecular Microbiology, School of Agriculture, Nagoya University, Chikusa-ku, Nagoya 464-8601, Japan. Phone: 81 52 7894089. Fax: 81 52 7894091. E-mail: tmizuno@agr.nagoya-u.ac.jp.

general insight into the molecular mechanism underlying the common Rcs signaling systems. A thorough understanding of the *E. coli* Rcs signaling mechanism would answer the general question of why many virulent and/or pathogenic bacteria have the common Rcs signaling systems. To investigate this, we performed genome-wide analyses of the *E. coli* Rcs signaling system by using microarrays. We also employed a set of 37 two-component system mutants, in which each of the *E. coli* two-component systems was knocked-out one by one (32). The results of this study revealed not only a new view of the Rcs regulon but also an intriguing signaling linkage between the Rcs and PhoQ/P two-component systems in response to external divalent cations (zinc and magnesium).

### MATERIALS AND METHODS

**Bacterial strains and media.** *E. coli* K-12 strain ST001 and derivatives of this strain were the main organisms used. The ST001 strain \[*cps::lacZ thi ara Δ(pro-lac)*\] was constructed from CSH26 (19), as described previously (42). Another CSH26 derivative, strain GY37 \[*katE::lacZ thi ara Δ(pro-lac)*\], was also used (49), as was the MG1301 strain carrying the *mgrB::lacZ* gene on the chromosome (21). A set of 37 two-component mutants was constructed previously in the *E. coli* K-12 strain BW25113 background (32) (see Fig. 6) by using the standard *E. coli* genetic procedures and/or the recently developed rapid procedures for constructing deletion mutants (7). The mutant alleles were each transferred into the ST001 background, one by one. In this study, a Δ*rcsF* derivative of ST001 was constructed by using the procedures used to construct deletion mutants (7). In this construct, the kanamycin resistance cassette replaced the entire *rcsF* gene. The primers used to construct an appropriate deletion cassette were 5′-CTCCTGATTCAATATTGACGTTTTGATCATACATTGAGGAAATACTATGGT GTAGGCTGGAGCTGCTTC and 5-GCCTATTTGCTCGAACTGGAAAAC TGCTCATTCGCGCTAATGTAAAGCGATTCCGGGGATCCGTCGACC. The *E. coli* cells were grown mainly at either 37 or 20°C in EB medium, which contains 1.8% (wt/vol) Eiken’s broth (Eiken Chemical Co. Ltd., Tokyo, Japan) and 0.4% glucose in 50 mM morpholinepropanesulfonic acid (MOPS) buffer (pH 7.0). When required, zinc, magnesium, and other divalent cations were added. Occasionally, Luria-Bertani medium was also used.

**Enzyme assay.** β-Galactosidase activity was measured by Miller’s method, with slight modifications (23). The cells were grown as described above (except as noted below). They were suspended in 250 mM sodium phosphate (pH 7.1) for accurate measurement of cell density. A portion of a cell suspension was used for a β-galactosidase assay after permeabilization with toluene.

**RNA isolation.** Cells were grown as described above or as described below. They were harvested at the mid-logarithmic growth phase. Total RNA was isolated with an RNeasy Maxi kit (Qiagen), as recommended by the supplier.

**Preparation of *E. coli* DNA microarrays.** Custom glass slide microarrays (from Takara Shuzo, Kyoto, Japan) were spotted with 4,095 PCR products corresponding to full-length *E. coli* open reading frames, together with human β-actin genes as a negative reference. PCR products were generated by using a clone bank containing *E. coli* genes in the Archive vector as the template, as described previously (26). Other details were the same as those described previously (31, 32).

**Preparation of labeled cDNA, hybridization, and data capture and analysis.** Appropriate pairs of RNA samples were prepared, and one sample was labeled with Cy3 and the other sample was labeled with Cy5. Each preparation was then tested twice by performing a microarray analysis. Thus, two values were obtained for each gene (or spot). The procedures used for hybridization, data capture, and data analysis were essentially the same as those described previously (31, 32). Briefly, by subtracting the local background value, we first corrected the intensity of each spot. In addition, a mean value for the intensity of the 24 negative control spots (human β-actin gene) was determined, together with the standard deviation (SD). Then spots were classified into three groups. For group 1 both the Cy3 signal intensity and the Cy5 signal intensity were greater than the mean plus one SD for the negative control. For group 2 either the Cy3 signal intensity or the Cy5 signal intensity was greater than the mean plus one SD for the negative control. For group 3 both the Cy3 signal intensity and the Cy5 signal intensity were less than the mean plus 1 SD for the negative control. Initially, we normalized spots classified in group 1 by defining the mean of the ratios (Cy5/Cy3) for all spots as 1.0. Group 2 spots with high Cy3 or Cy5 intensity values were selected. Group 3 spots were ignored as undetectable. Genes with significantly different expression

were then selected by using the following criteria, provided that similar values were obtained in the two independent hybridizations. We basically selected genes that showed relative Cy5/Cy3 ratios that were <0.5 and genes that showed relative Cy5/Cy3 ratios that were >2.0 in both hybridizations as being down and up, respectively.

---

### RESULTS

External stimuli that propagate the Rcs signaling system. Except for osmotic shock (37), physiological stimuli that activate the Rcs signaling system have not been fully defined yet. We attempted to identify such external stimuli for the Rcs signaling system. An approximately 800-bp nucleotide sequence that contained the promoter of the colanic acid synthesis *wza-wzb-wzc* gene cluster was fused to the *lacZ* gene (40, 46). Then the fusion gene was introduced into the *E. coli* chromosome at the λ *att* site; the resulting strain was designated ST001, and the reporter gene was designated *cps::lacZ* (42). We then searched for growth conditions under which the *cps::lacZ* reporter gene was induced in a manner dependent on the Rcs signaling system. We found that when ST001 cells were grown at a low temperature (<20°C) in the presence of 0.4% glucose, the reporter gene was activated significantly (Fig. 1A and B). The basal medium contained 1.8% nutrient broth in 50 mM MOPS buffer (pH 7.0) (designated EB medium). When the cells were incubated at 20°C in EB medium containing 0.4% glucose, the level of β-galactosidase activity was significantly increased, while such induction of *cps::lacZ* was not seen in the absence of glucose (Fig. 1A) or at 37°C (Fig. 1C and D). In these experiments, the cells were precultured at 37°C overnight. However, it should be noted that even if the cells were precultured at 20°C, essentially the same results were obtained. Thus, a cold shock appeared not to be implicated in this event.

The induction was dependent on the Rcs signaling components, RcsC and YojN (Fig. 1A). As shown in Fig. 1A, the colonies on EB agar plates containing 0.4% glucose produced large amounts of extracellular polysaccharides at 20°C and thus exhibited mucous and glittering morphologies on the solid medium. This suggested that not only expression of the *wza-wzb-wzc* cluster genes but also expression of the whole system of colanic acid production was activated under these growth conditions. We also found that glucose was the most effective carbon source, and the other carbohydrates tested were not as effective (Fig. 1E).

**External zinc is also an effective stimulus.** Although glucose serves as a stimulus for the Rcs signaling system, the observed β-galactosidase activity was only about 50 U, suggesting that the *cps::lacZ* reporter gene in ST001 may not be fully induced under these conditions. After extensively searching for other external stimuli, we found that addition of zinc (1 mM ZnCl₂) to EB medium resulted in marked induction of the *cps::lacZ* reporter gene at 20°C in a manner dependent on the RcsC function (Fig. 2A and B). A lower concentration of zinc (0.5 mM) was also effective (data not shown), and these concentrations of zinc in EB medium were not deleterious for growth of cells (Fig. 2B). In these experiments, the cells were precultured at 37°C overnight. However, it should be noted that even if the cells were precultured at 20°C, essentially the same results were obtained. Other divalent cations tested, including magnesium (30 mM MnCl₂) and nickel (1 mM NiCl₂), did not serve as stimuli (Fig. 2C and D). The effects of glucose and zinc

on the cps::lacZ reporter gene were additive (Fig. 2A). Thus, we found that the Rcs-mediated expression of cps::lacZ was fully induced under certain physiological growth conditions, namely, when cells were grown at a low temperature in the presence of glucose as a carbon source and in the presence of a relatively high concentration of external zinc. It should be noted that the levels of induction of cps::lacZ found in this study were much higher than the level of induction by an osmotic shock. As reported previously (42), the induction by an osmotic shock was less than 10 U under our experimental conditions.

RcsF is an essential component of the Rcs signaling system. It was previously suggested that the rcsF gene product might be a component of the Rcs signaling system (11). RcsF was inferred to be an outer membrane lipoprotein. It was reported that when the rcsF gene on a multicopy plasmid was introduced into E. coli cells, the cps genes were markedly induced in a manner dependent on the Rcs signaling system, even at 37°C. In this study we confirmed this with ST001 (Fig. 3A). Indeed, the expression of cps::lacZ was markedly elevated, provided that the rcsF gene on a multicopy plasmid was introduced into ST001. This event was completely dependent on each component of the Rcs signaling system (Fig. 3A). Nevertheless, our results do not necessarily mean that RcsF is an essential component of the Rcs signaling system, because the multicopy rcsF gene was used in this experiment. Since we succeeded in identifying certain physiological growth conditions under which the cps genes were reproducibly induced, we were able to critically examine the longstanding question of whether RcsF is an essential component of the Rcs signaling system. A ΔrcsF derivative of ST001 was constructed, in which the entire coding region on the chromosome was deleted (see Materials and Methods). This derivative was then examined with regard to expression of cps::lacZ in response to external zinc at 20°C (Fig. 3B). The results showed that the rcsF gene is an essential component of the Rcs signaling system, as far as signal transduction in response to glucose and zinc is concerned. Assuming that RcsF is an outer membrane protein, the following framework of the Rcs signaling system was formulated: RcsF (outer membrane protein)/RcsC (inner membrane protein) → YojN → RcsB/RcsA → Rcs regulon (target genes).

Genome-wide microarray analyses with special reference to the Rcs signaling system. Based on the framework and physiological stimuli that propagate the Rcs signaling system, we conducted genome-wide analyses with microarrays in order to characterize the Rcs regulon more extensively. Custom glass slide microarrays were spotted with PCR products corresponding

experiments whose results are shown in panel A. The symbols are the same as those used in panel A. OD<sub>660</sub>, optical density at 660 nm. (C) Culture incubated at 37°C as described above, as a control. (D) Growth curves for the cells which were examined in the experiment whose results are shown in panel C. The experiments were carried out at least three times, and the results of representative experiments are shown. (E) ST001 cells were grown at 20°C for both 8 and 12 h as described above in EB medium supplemented with either glucose (Glu), fructose (Fru), galactose (Gal), pyruvate (Pyr), or maltose (Mal) at a concentration of 0.4%. Then β-galactosidase activities were measured. Under both experimental conditions consistent results were obtained, and representative data (12 h) are shown.

FIG. 2. β-Galactosidase activity, determined by monitoring the cps::lacZ reporter gene. (A) ST001 (wild type [WT]) and a derivative of this strain (∆rcsC) were grown as described in the legend to Fig. 1. In these experiments, glucose and/or zinc (1 mM ZnCl₂) was added, as indicated. β-Galactosidase activities were measured during growth at 20°C. The experiments were carried out at least three times, and the results of representative experiments are shown. (B) Growth curves for the cells which were examined in the experiment whose results are shown in panel A. The symbols are the same as those used in panel A. OD₆₆₀, optical density at 660 nm. (C) ST001 cells were grown at 20°C for both 8 and 12 h (see panel D) in glucose-EB medium containing a divalent cation (1 mM ZnCl₂, 1 mM CuSO₄, 1 mM NiCl₂, 10 mM CaCl₂, or 30 mM MgCl₂). Then β-galactosidase activities were measured. Under both experimental conditions consistent results were obtained, and representative data (12 h) are shown. (D) Growth curves for the cells which were examined in the experiment whose results are shown in panel A. For an explanation of the symbols, see panel C.

FIG. 3. β-Galactosidase activity, determined by monitoring the cps::lacZ reporter gene. (A) ST001 (wild type [WT]) and mutant derivatives of this strain (∆rcsC, ∆yojN, and ∆rcsB) were transformed with a multicopy plasmid harboring the rcsF gene. The resulting cells (+plasmid), together with the control cells (WT), were grown in glucose-EB medium for 8 and 12 h at 37°C. Then β-galactosidase activities were measured. Under both experimental conditions, consistent results were obtained, and representative data (8 h) are shown. (B) ST001 (wild type) and mutant derivatives of this strain (∆rcsC and ∆rcsF) were grown for 12 h at 20°C in glucose-EB medium with and without zinc, as described in the legend to Fig. 1. Then β-galactosidase activities were measured. Under both experimental conditions consistent results were obtained, and representative data (12 h) are shown.

ing to 4,095 full-length E. coli open reading frames (Takara Shuzo). This E. coli whole-genome microarray has been successfully employed in previous studies (31, 32). With this microarray system, systematic profile analysis of transcriptomes was carried out after one RNA sample was labeled with Cy3 and the other RNA sample was labeled with Cy5 (Fig. 4). First, the labeled transcripts were prepared from ST001 cells grown in glucose-containing EB medium at 20°C in the presence of zinc (1 mM), and they were compared with the transcripts from cells grown in the absence of zinc. This microarray analysis revealed the genes whose expression was induced by zinc (zinc-up genes). Second, the transcripts of ST001 (wild-type) cells grown at 20°C in the presence of glucose and zinc were compared with those of a mutant derivative of ST001 (∆rcsC) grown under the same conditions. This microarray profile anal-

A
Zn²⁺- up
213 genes
ΔrcsC- down
58 genes

B
Zn²⁺- up
213 genes
ΔrcsF- down
120 genes

C
ΔrcsC- down
ΔrcsF- down
35
49
3
32
17
+ cps::lacZ

D
| CPS genes | New Rcs-regulon(?) | Characteristic Remarks |
|-----------|--------------------|------------------------|
| wzc       | bdm                | ↑ biofilm-dependent, unknown function |
| wcaA      | osmC               | ↓ osmo-inducible, (inner membrane protein) |
| wcaB      | katE               | RpoS-dependent, catalase HP || |
| wcaC      | osmB               | osmo-inducible, (lipoprotein) |
| wcaD      | osmY               | osmo-inducible, (periplasmic protein) |
| wcaE      | phnQ               | unknown function, (inner membrane protein) |
| wcaF      | spy                | ↓ spheroplast-inducible, (periplasmic protein) |
| gmd       | ydjR               | ↓ cold-inducible, unknown function |
| wcaG      | yajI               | unknown function, (lipoprotein) |
| wcaH      | ydcF               | unknown function |
| wcaI      | yggE               | autoinducer-responsive, (periplasmic protein) |
| cpsB      | yggG               | unknown function, (lipoprotein) |
| cpsG      | yghA               | unknown function |
| wcaJ      | yhbO               | unknown function |
| wcaK      | yjbJ               | unknown function |
| wcaL      |                    |                        |
| wcaM      |                    |                        |

FIG. 4. Schematic representation of microarray data obtained with mutants with mutations in the Rcs signaling system. Three types of microarray analyses were carried out. First, the transcriptome of ST001 (wild-type) cells grown for 8 h at 20°C in EB medium containing 0.4% glucose and 1 mM ZnCl₂ (labeled with Cy5) was compared with that of cells grown in the absence of zinc (labeled with Cy3). The data analyses revealed that 213 genes showed mean values for Cy5/Cy3 that were greater than 2. These genes were assumed to be the genes whose expression was up-regulated by zinc (designated zinc-up genes). Second, the transcriptome of ΔrcsC cells grown at 20°C in EB medium containing 0.4% glucose and 1 mM ZnCl₂ (labeled with Cy5) was compared with that of ST001 (wild-type) cells grown under the same conditions (labeled with Cy3). The data analyses revealed that 58 genes showed mean values for Cy5/Cy3 that were less than 0.5. These genes were assumed to be the genes whose expression was down-regulated in the ΔrcsC background (designated ΔrcsC-down genes). Similarly, 120 ΔrcsF-down genes were identified. Based on the microarray data, each overlapping set of genes was analyzed, as shown in panels A and B. These independent analyses revealed the putative Rcs regulon members, shown in panel C. (D) Putative Rcs regulon genes. The 17 cps genes are located together in cps operons on the E. coli chromosome. The 15 non-cps genes are located at different positions on the chromosome, although the bdm and osmC genes and the spy and ydjR genes are located next to each other, as indicated by arrows. The microarray data used in this study are available as supplemental data at the following web site: http://www.agr.nagoya-u.ac.jp/%7emicrobio/. Also, to gain additional ideas for each gene, see the web site of Colibri (http://www.genolist.pasteur.fr/Colibri/). CPS, capsular polysaccharide.

ysis revealed the genes whose induction was mediated by the RcsC function (ΔrcsC-down genes). To obtain reproducible data, a ΔrcsF mutant was also employed. The transcripts of ST001 (wild-type) cells grown at 20°C in the presence of glucose and zinc were compared with those of a mutant derivative of ST001 (ΔrcsF) grown under the same conditions. This microarray profile analysis revealed the genes whose induction was mediated by the RcsF function (ΔrcsF-down genes). When the data were considered together, we could systematically compile the genes whose expression was regulated through the RcsF/RcsC→YojN→RcsB signaling system in response to glucose and zinc.

The results of the intensive microarray analyses are schematically summarized in Fig. 4. First, 213 genes were identified as zinc-up genes. As expected, the cps::lacZ gene in ST001 was found in these genes (a critical internal reference for the reliability of the microarrays). Also, it was not surprising that such a large number of genes were affected by zinc, because when grown in the presence of external zinc, E. coli cells must change their physiologies in various ways. Second, 58 genes were identified as ΔrcsC-down genes, and 35 of these genes overlapped zinc-up genes (Fig. 4A). These zinc-induced genes were assumed to be the genes whose expression is mediated by the RcsC function. Similarly, 120 genes were identified as ΔrcsF-down genes, and 49 of these genes overlapped zinc-up genes (Fig. 4B). Then the 35 ΔrcsC-down and 49 ΔrcsF-down genes were compared with each other. Strikingly, 32 genes were found in both groups (Fig. 4C). In addition to these genes, the cps::lacZ gene was recovered in this group, as mentioned above. Thus, we assumed that these 32 genes are most likely

the genes whose expression is under control of the Rcs signaling system, directly or indirectly (these genes were tentatively referred to as Rcs regulon genes) (Fig. 4D).

It should be noted that 23 of the 58 ΔrcsC-down genes were not found in the zinc-up gene group. This fact may suggest that zinc and glucose regulate each set of genes through the Rcs signaling system. It may also be noteworthy that a much larger number of genes (120 genes) were affected in the ΔrcsF mutant than in the ΔrcsC mutant (58 genes). This fact may suggest that RcsF is also implicated in an as-yet-unknown regulatory system which is not dependent on the Rcs signaling system. In any event, these intriguing findings remain to be elucidated.

In addition to the cps genes, many other genes appear to be members of the Rcs regulon. To examine the putative Rcs regulon genes more closely, the 32 genes discovered are listed in Fig. 4D. First, most of the known cps genes were identified as putative Rcs regulon members, as expected. These genes include the wzc gene, as well as the wca operon genes (46). Altogether, 17 of the cps genes were successfully identified as Rcs regulon members. In addition to these cps genes, the same microarray analyses revealed that 15 non-cps genes were putative Rcs regulon members. Some of these genes (eight genes) have previously been characterized to some extent, as noted in Fig. 4D. They included three of the osm genes (8, 12, 20, 29, 50). The bdm gene was characterized as the gene relevant for biofilm formation (33), whereas spy was shown to be a spheroplast-inducible gene (16, 34). The katE gene encodes an RpoS-dependent catalase (designated HPII) (43, 49).

As noted above, the 15 non-cps genes, including 7 uncharacterized y genes, seem to be Rcs regulon members. It may be noted that two of these genes, the bdm and osmC genes, are located next to each other on the chromosome but are oriented in opposite directions (Fig. 4). Also, the spy and ydjR genes appear to constitute an operon. As mentioned above, RcsB (together with RcsA) is a crucial DNA-binding regulator for the Rcs regulon. Previously, a consensus sequence (TAAGA ATATTCCTA) was proposed as the RcsA/RcsB (heterodimer) recognition site (47). We searched for putative RcsA/ RcsB recognition sites in the presumed promoter regions of the newly identified target genes. Many of these genes, if not all of them, have a short nucleotide sequence similar to the RcsA/RcsB recognition site in the appropriate promoter region (Fig. 5A). However, some of the putative recognition sites are located far from the corresponding ATG initiation codons (e.g., katE, yajI, and yggE). Also, some of the promoters might also have an RcsB recognition site, which is independent of RcsA. Therefore, verification of RcsB and/or RcsA recognition sites in these putative Rcs regulon genes must await further inspection, which should include determination of each transcription start site. To address this issue further, an E. coli strain (designated GY37) carrying a katE::lacZ fusion gene on the chromosome was employed (49). The expression profiles of the katE::lacZ fusion gene were examined in terms of the Rcs signaling system (Fig. 5B). Indeed, expression of katE::lacZ was induced in response to external glucose and zinc at 20°C. The induction of katE::lacZ was severely impaired in a set of rcs mutants.

It should be emphasized that the proposed Rcs regulon genes are tentative, because in theory, the microarray analyses did not allow us to discriminate whether a given gene that was

---

**FIG. 5.** Putative RcsA/RcsB recognition sequences in the Rcs regulon genes and β-galactosidase activity as determined by monitoring the katE::lacZ reporter gene. (A) Using the previously proposed RcsA/ RcsB recognition consensus sequence (TAAGAATATTCCTA) (47), we searched for nucleotide sequences similar to the consensus sequence in each promoter region of the non-cps genes (see Fig. 4D). The numbers in parentheses indicate the position (last nucleotide) of the 14-nucleotide sequence; the first nucleotide of the inferred ATG initiation codon of the gene in question was defined as position 1. (B) Strain GY37 carries the katE::lacZ reporter gene (49). This wild-type strain and mutant derivatives (ΔrcsC, ΔyojN, and ΔrcsB) were grown in EB medium containing 0.4% glucose and 1 mM ZnCl₂ at 20°C, as described in the legend to Fig. 1. β-Galactosidase activities were measured during growth.

identified is indeed the direct target of the transcriptional regulator in question. In this strict sense, some of the proposed Rcs regulon genes may or may not be direct targets of the Rcs signaling system. For example, the Rcs signaling system might somehow affect the activity of RpoS at the level of transcription, translation, or protein stability, which in turn could affect expression of the katE gene indirectly. With such limitations in mind, however, the microarray analyses performed in this study consistently revealed the set of 32 genes whose expression is under control of the Rcs signaling system directly or indirectly. In any case, the Rcs regulon genes, proposed above, appeared to be positively regulated by the Rcs signaling system. Theoretically, one can search for genes whose expression is negatively regulated by the Rcs signaling system. Such genes should be classified as zinc-down, ΔrcsC-up, and ΔrcsF-up

reporter gene was examined after cells were grown for 12 h at 20°C in EB medium containing glucose and zinc (Fig. 6). The results showed that as in the *rcs* mutants, the levels of induction of *cps::lacZ* were considerably reduced in the following genetic backgrounds: Δ*arcA*, Δ(*envZ/ompR*), and Δ*phoQ/P*. Rcs-mediated *cps::lacZ* expression was not significantly affected in the other genetic backgrounds. The results suggested that the Rcs signaling system might have linkage to some other phosphorelay systems.

To critically examine this possibility, the time course of *cps::lacZ* induction was monitored in each genetic background in question (Fig. 7). In both the Δ*arcA* and Δ(*envZ/ompR*) backgrounds (Fig. 7C and D), a slightly different *cps::lacZ* expression profile was obtained compared with that of the wild type (Fig. 7A). In particular, the levels of expression at 12 h for Δ*arcA* (ca. 200 U) and Δ(*envZ/ompR*) (ca. 200 U) were significantly lower than the level of expression in the wild-type background (ca. 300 U). These observations were consistent with the findings shown in Fig. 6, and thus the ArcA and/or EnvZ signaling system might have a link to the Rcs signaling system. However, the apparent effects of these mutants were subtle; thus, they were not characterized any further in this study. In contrast, the induction of *cps::lacZ* was more severely impaired in the Δ*phoQ/P* background (Fig. 7E). Both the PhoQ sensor and PhoP regulator components were required for full induction of *cps::lacZ* in response to zinc (Fig. 7F and G). These results suggested that the PhoQ/P two-component genes might also be crucial determinants for regulating the *cps* genes in response to external zinc. In other words, there might be an intimate link between the Rcs and PhoQ/P signaling systems.

Evidence for close linkage between the Rcs and PhoQ/P signaling systems. We remembered the well-documented fact that the PhoQ/P signaling system functions in response to external magnesium (for a review, see reference 14). The phosphatase activity of PhoQ is modulated (or activated) through direct binding of magnesium ions to the PhoQ sensor, and thus the activity of the cognate PhoP response regulator is repressed at high concentrations of magnesium (45). The critical question then is, how do we explain the observed linkage between the PhoQ/P and Rcs signaling systems in response to zinc? We assumed that the PhoQ/P signaling system somehow functions upstream of the Rcs signaling system in response to zinc. If this is so, one can predict that the following events will occur: the *cps::lacZ* induction by zinc at 20°C in EB medium should be eliminated by the presence of a high concentration of magnesium in the medium, and the PhoQ/P signaling system should be attenuated under these conditions. The results of a critical examination of this hypothesis are shown in Fig. 8A. Indeed, induction of *cps::lacZ* by zinc (1 mM) was not observed, provided that magnesium (30 mM) was added concomitantly to the medium. This effect was dependent on the concentration of magnesium added (Fig. 8B).

This observation led us to envisage that the PhoQ sensor, not the RcsC sensor, might sense the external zinc signal. In other words, it was assumed that a high concentration of magnesium is a negative signal for the PhoQ sensor, whereas zinc might act as a positive signal. To test this idea, we examined expression of the *mgrB* gene, which is one of the hallmark targets of the PhoQ/P signaling system (21). When MG1301 cells carrying the *mgrB::lacZ* gene on the chromosome were

FIG. 6. β-Galactosidase activity, determined by monitoring the *cps::lacZ* reporter gene in a set of 37 two-component system mutants. ST001 (wild type) and 37 two-component system mutants (each designated by a gene designation[s]) were grown in EB medium containing 0.4% glucose and 1 mM ZnCl₂ at 20°C for 12 h, as described in the legend to Fig. 1. Then β-galactosidase activities were measured. The experiments were carried out three times, and the data shown are means. The *basS/R* mutant cells did not grow well in the medium used. ND, not determined.

5742 HAGIWARA ET AL

![Graph](#)

FIG. 7. β-Galactosidase activity, determined by monitoring the *cps::lacZ* reporter gene in two-component mutant backgrounds. ST001 (wild type) and mutant derivatives of this strain were grown in EB medium containing 0.4% glucose and 1 mM ZnCl₂ at 20°C, as described in the legend to Fig. 1. Then the β-galactosidase activities were measured at intervals. (A) ST001 (wild type); (B) ST001(Δ*rcsC*); (C) ST001(Δ*arcA*); (D) ST001[Δ(*envZ/ompR*)]; (E) ST001Δ*phoQ/P*; (F) Δ*phoQ*; (G) Δ*phoP*.

tion of external magnesium was relatively low. This event was confirmed by measuring expression of *mgrB::lacZ* with various concentrations of magnesium in the presence of zinc (Fig. 9B). The results were completely consistent with the view that the PhoQ sensor is capable of responding to both magnesium and zinc in such a way that a high concentration of zinc stimulates the PhoQ function, while a high concentration of magnesium inhibits the PhoQ sensor.

Critical genome-wide microarray analyses. The results described above supported the presence of the plausible zinc → PhoQ/P signaling → Rcs signaling → Rcs regulon pathway, through which zinc propagates the Rcs system in a PhoQ/P-dependent manner. To critically examine this possibility, we conducted the following microarray analysis. Transcripts in ST001 (wild-type) cells grown at 20°C in the presence of glucose and zinc were compared with transcripts of a mutant derivative of ST001 (Δ*phoQ/P*) grown under the same conditions. This microarray profile analysis should have revealed the genes whose expression was mediated by the PhoQ/P function (Δ*phoQ/P*-down genes). A total of 100 genes were identified as such genes, and 61 of these genes overlapped zinc-up genes (Fig. 10A; also see Fig. 4). These zinc-induced genes were assumed to be the genes whose expression is mediated by the PhoQ/P function. More importantly, it was found that 25 of the previously identified 32 Rcs regulon genes were included in the Δ*phoQ/P*-down group (Fig. 10B). In other words, only seven genes (e.g., *wcaF* and *wcaK*) had escaped the microarray analysis. When the intrinsic limitations of the microarray were considered, this result supported the presence of the zinc → PhoQ/P signaling → Rcs signaling → Rcs regulon signaling pathway.

DISCUSSION

In this study, the Rcs phosphorelay signaling system in *E. coli* was characterized with a genome-wide viewpoint. Several intriguing views emerged from this analysis, as summarized in Fig. 11 (1). Not only the three Rcs phosphorelay factors but also RcsF was demonstrated to be a crucial component of the Rcs signaling system (Fig. 3) (2). The resulting RcsF/RcsC → YojN → RcsB/RcsA signaling system was propagated under certain physiological growth conditions, namely, when cells were grown at a low temperature in the presence of glucose as a carbon source and in response to a relatively high concentration of external zinc (Fig. 1 and 2) (3). By using microarray analyses, a view of the Rcs regulon was clarified by identifying a number of putative new target genes whose expression is positively regulated by the Rcs signaling system (Fig. 4). In addition to the well-known *cps* genes, a number of apparently unrelated genes, including *osmB/C/Y*, were suggested to be the members of Rcs regulon (Fig. 4 and 5) (4). The results of another genome-wide examination revealed a close linkage between the Rcs and PhoQ/P signaling systems, the latter of which is known to respond to external magnesium (Fig. 6 and 7). The results of this study supported the occurrence of the zinc (magnesium) → PhoQ/P signaling → Rcs signaling → Rcs regulon pathway (Fig. 8 to 10). Collectively, the results obtained in this study provided deeper insight into the *E. coli* phosphorelay systems, with special reference to the Rcs signaling system (Fig. 11).

Microarray analysis is a powerful technique for systematically revealing the transcriptomes of organisms of interest.

FIG. 8. β-Galactosidase activity, determined by monitoring the *cps::lacZ* reporter gene with cells grown in the presence of both magnesium and zinc. (A) ST001 cells were grown at 20°C in EB containing 0.4% glucose and 1 mM ZnCl₂ with and without 30 mM MgCl₂ (also see the legend to Fig. 1). During growth, β-galactosidase activities were measured. (B) ST001 cells were grown at 20°C in EB containing 0.4% glucose and 1 mM ZnCl₂, as well as different concentrations of MgCl₂. After 8 and 12 h of incubation, β-galactosidase activities were measured. Under both experimental conditions consistent results were obtained, and representative data (12 h) are shown.

In *E. coli*, a large number of microarray data are accumulating rapidly (9, 30, 31, 32, 51). Previously, we have taken such genome-wide approaches to obtain a whole picture of the transcriptomes for the *E. coli* phosphorelay systems (32). To further extend our knowledge, in this study we used the same approach with special reference to the Rcs signaling system. The microarray analyses consistently revealed the members of the Rcs regulon. In addition to the known *cps* genes (17 genes), the results suggested that a number of new targets (15 genes) seemed to be activated through the Rcs signaling system. We need to confirm this assumption for each gene by another means, one by one. However, this view was based on the results of four independent microarray analyses (zinc-up, Δ*rcsC*-down, Δ*rcsF*-down, and Δ*phoQ/P*-down). Furthermore, we confirmed that the *katE* gene is regulated by the Rcs signaling system (Fig. 5), and the *osmC* gene has recently been reported to be under control of the Rcs signaling system (6, 8). In any case, the newly identified genes apparently are not relevant to colanic acid synthesis. Interestingly, however, many of them were predicted to encode cell envelope-associated proteins.

FIG. 9. β-Galactosidase activity, determined by monitoring the *mgrB::lacZ* reporter gene with cells grown in the presence of both magnesium and zinc. (A) MG1301 cells carrying the *mgrB::lacZ* reporter gene were grown at 20°C in EB medium containing 0.4% glucose with and without 1 mM ZnCl₂ and/or 30 mM MgCl₂. During growth, β-galactosidase activities were measured. (B) MG1301 cells carrying the *mgrB::lacZ* reporter gene were grown at 20°C in EB medium containing 0.4% glucose and 1 mM ZnCl₂ with different concentrations of MgCl₂. Then β-galactosidase activities were measured after 8 and 12 h of incubation. Under both experimental conditions consistent results were obtained, and representative data (12 h) are shown.

A  Zn²⁺-up  ΔphoQP-down
213 genes          100 genes

61  
+  
cps::lacZ  

B  
32   25   61  

bdm  
katE  
osmY  
wcaF  
wcaK  
yggG  
yhbO  

cpsB cpsG gmd  
osmB osmC phnQ  
spy wcaA wcaB  
wcaC wcaD wcaE  
wcaG wcaH wcaI  
wcaJ wcaL wcaM  
wzc yajl ydcF ydjR  
yggE yghA yjbJ  

Rcs-regulon vs ΔphoQP-down  

FIG. 10. Schematic representation of microarray data obtained with a mutant with a mutation in the PhoQ/P signaling system. Microarray analyses were carried out as indicated. The transcriptome of ΔphoQ/P cells grown at 20°C in EB medium containing 0.4% glucose and 1 mM ZnCl₂ (labeled with Cy5) was compared with that of ST001 (wild-type) cells grown under the same conditions (labeled with Cy3). The data analyses revealed that 100 genes showed mean values for Cy5/Cy3 that were less than 0.5. These genes were assumed to be the genes whose expression was down-regulated in the ΔphoQ/P background (designated ΔphoQ/P-down genes). (A) Uncovered genes were compared with zinc-up genes (see Fig. 4). This analysis allowed us to select 61 ΔphoQ/P-down genes, whose zinc-responsive expression was assumed to be mediated by the PhoQ/P signaling system. (B) Genes compared with genes belonging to the Rcs regulon (see Fig. 4). Note that most of the Rcs regulon members were also found to be ΔphoQ/P-down genes. The microarray data used in this study are available as supplemental data at the following web site: http://www.agr.nagoya-u.ac.jp/%7emicrobio/.

Glucose  
Zn²⁺  
Mg²⁺  

RcsF  
RcsA  
RcsC — YojN — RcsB  
Phosphorelay  
?  
PhoQ — PhoP  
Phosphorelay  

Putative new targets  
osmB/C/Y etc.  
CPS synthesis genes  
LPS modification genes  
Mg²⁺ transporter genes  

FIG. 11. Proposed organization of the PhoQ/P-Rcs signaling network. The magnesium-regulated PhoQ/P signaling system appears to function upstream of the Rcs signaling system. The results of this study suggested that the PhoQ/P signaling system is activated by an external zinc signal, resulting in activation of the Rcs signaling system. The function of RcsF is not clear in this model. However, RcsF is essential for the regulatory system (see Fig. 3), and the rcsF mutation is epistatic to the phoQ mutation (data not shown). We propose that such a sophisticated PhoQ/P-Rcs signaling network ensures that E. coli cells properly regulate the cell surface structures by modifying lipopolysaccharide (LPS) and/or capsular polysaccharide (CPS) in response to the external states of crucial divalent cations (magnesium and zinc). Details are discussed in the text.

14). In *S. enterica*, the PhoQ/P signaling system activates the PmrB/A signaling system during growth in the presence of a low magnesium concentration. In principle, the PmrB/A system is known to respond to external iron (5, 48). The unique gene product PmrD connects these two systems in such a way that the PhoQ/P signaling system regulates the production of PmrD, and the resulting protein activates the PmrB His kinase (22). Indeed, *E. coli* K-12 has an analogous set of genes, *pmrD*, *pmrB* (also called *basS*), and *pmrA* (or *basR*) (27). In the presumed zinc (stimulus) → PhoQ/P → X → RcsC/YojN/RcsB pathway, a hypothetical factor (X) might play a role similar to that of PmrD. Through both the PhoQ/P → Rcs and PhoQ/P → PmrB/A signaling networks, *E. coli* cells might be able to coordinately adapt to the states of external divalent cations, such as magnesium, iron, and zinc.

Many isolates of *E. coli* synthesize the slime polysaccharide colanic acid (references 35 and 36 and references therein). Small amounts of this capsule molecule are made by most *E. coli* strains, but certain growth conditions occasionally lead to dramatic increases in synthesis. Although the physiological role of colanic acid is still unclear, capsules are the outermost structures on an *E. coli* cell, and they may play a critical role in interactions between the bacterium and its environment by providing mechanisms for the cell to avoid host defenses in the intestinal tract. Capsules may also be important in the ability of *E. coli* to survive in extraorganismic environments. From the physiological point of view, the findings of this study showed that the environmental cation states, including the external levels of zinc and magnesium, appear to be critical environmental signals for capsular polysaccharide synthesis. The presumed framework shown in Fig. 11 provided several insights into the physiological role of the Rcs signaling system. It should be noted that the PhoQ/P signaling system regulates not only certain magnesium transporter genes but also genes involved in lipopolysaccharide modification (e.g., *pagP*) (14). Accordingly, the PhoQ/P → Rcs signaling network might allow *E. coli* cells to coordinately regulate cell surface polysaccharide (capsular polysaccharide and lipopolysaccharide) synthesis in response to the environmental states of divalent cations, such as magnesium and zinc. As elegantly proposed by Groisman (14) for the PhoQ/P → PmrB/A network in response to external magnesium and iron, the newly proposed PhoQ/P → Rcs network in response to external magnesium and zinc might also be important in natural habitats for maintenance of cell surface integrity and proper cell-cell interactions. These physiological processes might be crucial for swarming behavior on solid medium, quorum sensing, and/or biofilm formation, as mentioned above.

In summary, the genome-wide analyses performed in this study provided not only specific insight into the Rcs signaling system but also general insight into the *E. coli* phosphorelay systems. Since one can already see the impact of genome-wide analyses, such analyses for other phosphorelay systems should shed light on the global network of the His → Asp phosphorelay signal transduction systems in *E. coli*.

### ACKNOWLEDGMENTS

This study was supported by grants-in-aid from the Ministry of Education, Culture, Sports, Science and Technology of Japan (to T.M.) and from CREST of Japan Science and Technology (to H.M.).

We thank R. Utumi for providing an *E. coli* strain (*mgrB::lacZ*).

#### REFERENCES

1. Appleby, J. L., J. S. Parkinson, and R. B. Bourret. 1996. Signal transduction via the multi-step phosphorelay: not necessarily a road less traveled. Cell 86:845–848.
2. Arricau, N., D. Hermant, H. Waxin, C. Ecobichon, P. S. Duffey, and M. Y. Popoff. 1998. The RcsB-RcsC regulatory system of *Salmonella typhi* differentially modulates the expression of invasion proteins, flagellin and Vi antigen in response to osmolarity. Mol. Microbiol. 29:835–850.
3. Belas, R., R. Schneider, and M. Melch. 1998. Characterization of *Proteus mirabilis* precocious swarming mutants: identification of *rsbA*, encoding a regulator of swarming behavior. J. Bacteriol. 180:6126–6139.
4. Bereswill, S., and K. Geider. 1997. Characterization of the *rcsB* gene from *Erwinia amylovora* and its influence on exopolysaccharide synthesis and virulence of the fire blight pathogen. J. Bacteriol. 179:1354–1361.
5. Chamnongpol, S., W. Dodson, M. J. Cromie, Z. L. Harris, and E. A. Groisman. 2002. Fe(III)-mediated cellular toxicity. Mol. Microbiol. 45:711–719.
6. Conter, A., R. Surny, C. Gutierrez, and K. Cam. 2002. The RcsCB His-Asp phosphorelay system is essential to overcome chlorpromazine-induced stress in *Escherichia coli*. J. Bacteriol. 184:2850–2853.
7. Datsenko, K. A., and B. L. Wanner. 2000. One-step inactivation of chromosomal genes in *Escherichia coli* K-12 using PCR products. Proc. Natl. Acad. Sci. USA 97:6640–6645.
8. Davalos-Garcia, M., A. Conter, I. Toesca, C. Gutierrez, and K. Cam. 2001. Regulation of *osmC* gene expression by the two-component system *rcsB-rcsC* in *Escherichia coli*. J. Bacteriol. 183:5870–5876.
9. DeLisa, M. P., C. F. Wu, L. Wang, J. J. Valdes, and W. E. Bentley. 2001. DNA microarray-based identification of genes controlled by autoinducer 2-stimulated quorum sensing in *Escherichia coli*. J. Bacteriol. 183:5239–5247.
10. Ebel, W., and J. E. Trempy. 1999. *Escherichia coli* RcsA, a positive activator of colanic acid capsular polysaccharide synthesis, functions to activate its own expression. J. Bacteriol. 181:577–584.
11. Gervais, F. G., and G. R. Drapeau. 1992. Identification, cloning, and characterization of *rcsF*, a new regulator gene for exopolysaccharide synthesis that suppresses the division mutation *ftsZ84* in *Escherichia coli* K-12. J. Bacteriol. 174:8016–8022.
12. Gordia, S., and C. Gutierrez. 1996. Growth-phase-dependent expression of the osmotically inducible gene *osmC* of *Escherichia coli* K-12. Mol. Microbiol. 19:729–736.
13. Gottesman, S., and V. Stout. 1991. Regulation of capsular polysaccharide synthesis in *Escherichia coli* K-12. Mol. Microbiol. 5:1599–1606.
14. Groisman, E. A. 2001. The pleiotropic two-component regulatory system PhoP-PhoQ. J. Bacteriol. 183:1835–1842.
15. Gygi, G., M. Rahman, H. C. Lai, H. C., R. Carlson, J. Guard-Petter, and C. Hughes. 1995. A cell-surface polysaccharide that facilitates rapid population migration by differentiated swarm cells of *Proteus mirabilis*. Mol. Microbiol. 17:1167–1175.
16. Hagenmaier, S., Y. D. Stierhof, and U. Henning. 1997. A new periplasmic protein of *Escherichia coli* which is synthesized in spheroplasts but not in intact cells. J. Bacteriol. 179:2073–2076.
17. Harshey, R. M., and T. Matsuyama. 1994. Dimorphic transition in *Escherichia coli* and *Salmonella typhimurium*: surface-induced differentiation into hyperflagellate swarmer cells. Proc. Natl. Acad. Sci. USA 91:8631–8635.
18. Heidelberg, J. F., J. A. Eisen, W. C. Nelson, R. A. Clayton, M. L. Gwinn, R. J. Dodson, et al. 2000. DNA sequence of both chromosomes of the cholera pathogen *Vibrio cholerae*. Nature 406:477–483.
19. Ishige, K., S. Nagasawa, S. Tokishita, and T. Mizuno. 1994. A novel device of bacterial signal transducers. EMBO J. 13:5195–5202.
20. Jung, J. U., C. Gutierrez, F. Martin, M. Ardourel, and M. Villarejo. 1990. Transcription of *osmB*, a gene encoding an *Escherichia coli* lipoprotein, is regulated by dual signals. J. Biol. Chem. 265:10574–10581.
21. Kato, A., H. Tanabe, and R. Utumi. 1999. Molecular characterization of the PhoP-PhoQ two-component system in *Escherichia coli* K-12: identification of extracellular Mg²⁺-responsive promoters. J. Bacteriol. 181:5516–5520.
22. Kox, L. F., M. M. Wosten, and E. A. Groisman. 2000. A small protein that mediates the activation of a two-component system by another two-component system. EMBO J. 19:1861–1872.
23. Miller, J. H. 1972. Experiments in molecular genetics. Cold Spring Harbor Press, Cold Spring Harbor, N.Y.
24. Mizuno, T. 1997. Compilation of all genes encoding two-component phototransfer signal transducers in the genome of *Escherichia coli*. DNA Res. 4:161–168.
25. Mizuno, T. 1998. His-Asp phosphotranfer signal transduction. J. Biochem. (Tokyo) 123:555–563.
26. Mori, H., K. Isono, T. Horiuchi, and T. Miki. 2000. Functional genomics of *Escherichia coli*. Res. Microbiol. 151:121–128.
27. Nagasawa, S., K. Ishige, and T. Mizuno. 1993. Novel members of the two-component signal transduction genes in *Escherichia coli*. J. Biochem. (Tokyo) 114:350–357.
28. Nassif, X., N. Honore, T. Vasselon, S. T. Cole, and P. J. Sansonetti. 1989. Positive control of colanic acid synthesis in *Escherichia coli* by *rmpA* and

5746 HAGIWARA ET AL

$rmpB$, two virulence-plasmid genes of *Klebsiella pneumoniae*. Mol. Microbiol. 3:1349–1359.

29. Oh, J. T., Y. Cajal, M. E. Skowronska, S. Belkin, J. Chen, T. K. van Dyk, and M. K. Jain. 2000. Cationic peptide antimicrobials induce selective transcription of $micF$ and $osmY$ in *Escherichia coli*. Biochim. Biophys. Acta 15:43–54.

30. Oh, M. K., L. Rohlin, K. C. Kao, and J. C. Liao. 2001. Global expression profiling of acetate-grown *Escherichia coli*. J. Biol. Chem. 277:13175–13183.

31. Oshima, T., W. Wada, Y. Kawagoe, T. Ara, M. Maeda, Y. Masuda, et al. 2002. Genome-wide analysis of deoxy-adenosine methyltransferase-dimethylated control of gene expression in *Escherichia coli*. Mol. Microbiol. 45:673–695.

32. Oshima, T., H. Aiba, Y. Masuda, S. Kanaya, M. Sugiura, B. L. Wanner, et al. 2002. Transcriptome analysis of all two-component regulatory system mutants of *Escherichia coli* K-12. Mol. Microbiol. 46:281–291.

33. Prigent-Combaret, C., O. Vidal, C. Dorel, and P. Lejeune. 1999. Abiotic surface sensing and biofilm-dependent regulation of gene expression in *Escherichia coli*. J. Bacteriol. 181:5993–6002.

34. Raffa, R. G., and T. L. Raivio. 2002. A third envelope stress signal transduction pathway in *Escherichia coli*. Mol. Microbiol. 45:1599–1611.

35. Rick, P. D., and R. P. Silver. 1996. Enterobacterial common antigen and capsular polysaccharides: cellular and molecular biology, p. 104–122. In F. C. Neidhardt, R. Curtiss III, J. L. Ingraham, E. C. C. Lin, K. B. Low, B. Magasanik, W. S. Reznikoff, M. Riley, M. Schaechter, and H. E. Umbarger (ed.), *Escherichia coli* and *Salmonella*: cellular and molecular biology, 2nd ed. ASM Press, Washington, D.C.

36. Roberts, I. S. 1996. The biochemistry and genetics of capsular polysaccharide production in bacteria. Annu. Rev. Microbiol. 50:285–315.

37. Sledjeski, D. D., and S. Gottesman. 1996. Osmotic shock induction of capsule synthesis in *Escherichia coli* K-12. J. Bacteriol. 178:1204–1206.

38. Stevenson, G., K. Andrianopoulos, M. Hobbs, and P. R. Reeves. 1996. Organization of the *Escherichia coli* K-12 gene cluster responsible for production of the extracellular polysaccharide colanic acid. J. Bacteriol. 178:4885–4893.

39. Stout, V. 1994. Regulation of capsule synthesis includes interactions of the RcsC/RcsB regulatory pair. Res. Microbiol. 145:389–392.

40. Stout, V. 1996. Identification of the promoter region for the colanic acid

polysaccharide biosynthetic genes in *Escherichia coli* K-12. J. Bacteriol. 178: 4273–4280.

41. Stover, C. K., X. Q. Pham, A. L. Erwin, S. D. Mizoguchi, P. Warrener, M. J. Hickey, et al. 2000. Complete genome sequence of *Pseudomonas aeruginosa* PAO1, an opportunistic pathogen. Nature 406:959–964.

42. Takeda, S., Y. Fujisawa, M. Matsubara, H. Aiba, and T. Mizuno. 2001. A novel feature of the multistep phosphorelay in *Escherichia coli*: a revised model of the RcsC→YojN→RcsB signaling pathway implicated in capsular synthesis and swarming behavior. Mol. Microbiol. 40:440–450.

43. Tanaka, K., K. Handel, P. C. Loewen, and H. Takahashi. 1997. Identification and analysis of the $rpoS$-dependent promoter of $katE$, encoding catalase HPII in *Escherichia coli*. Biochim. Biophys. Acta 30:161–166.

44. Trisler, P., and S. Gottesman. 1984. $lon$ transcriptional regulation of genes necessary for capsular polysaccharide synthesis in *Escherichia coli* K-12. J. Bacteriol. 160:184–191.

45. Vescovi, E. G., F. C. Soncini, and E. A. Groisman. 1996. Mg²⁺ as an extracellular signal: environmental regulation of *Salmonella* virulence. Cell 84: 165–174.

46. Vincent, C., P. Doublet, C. Grangeasse, E. Vaganay, A. J. Cozzone, and B. Duclos. 1999. Cells of *Escherichia coli* contain a protein-tyrosine kinase, Wzc, and a phosphotyrosine-protein phosphatase, Wzb. J. Bacteriol. 181:3472–3477.

47. Wehland, M., and F. Bernhard. 2000. The RcsAB box. Characterization of a new operator essential for the regulation of exopolysaccharide biosynthesis in enteric bacteria. J. Biol. Chem. 275:7013–7020.

48. Wosten, M. M., L. F. Kox, S. Chamnongpol, F. C. Soncini, and E. A. Groisman. 2000. A signal transduction system that responds to extracellular iron. Cell 103:113–125.

49. Yamashino, T., M. Isomura, C. Ueguchi, and T. Mizuno. 1998. The $yhhP$ gene encoding a small ubiquitous protein is fundamental for normal cell growth of *Escherichia coli*. J. Bacteriol. 180:2257–2261.

50. Yim, H. H., and M. Villarejo. 1995. $osmY$, a new hyperosmotically inducible gene, encodes a periplasmic protein in *Escherichia coli*. J. Bacteriol. 174: 3637–3644.

51. Zheng, M., X. Wang, L. J. Templeton, D. R. Smulski, R. A. LaRossa, and G. Storz. 2001. DNA microarray-mediated transcriptional profiling of the *Escherichia coli* response to hydrogen peroxide. J. Bacteriol. 183:4562–4570.
