
Gene, 115 (1992) 105â€“111  
Â© 1992 Elsevier Science Publishers B.V. All rights reserved. 0378-1119/92/$05.00  

**GENE 06272**

# Streptomycin biosynthesis and its regulation in Streptomycetes*

*(Streptomyces griseus; Streptomyces glaucescens; 6-deoxy hexose; regulation; A-factor)*

JÃ¼rgen Distler, Kambiz Mansouri, Gerd Mayer, Michael Stockmann and Wolfgang Piepersberg

BUGH Wuppertal, FB9, Microbiology Department, W-5600 Wuppertal 1 (Germany)

Received by C.R. Hutchinson: 29 August 1991  
Accepted: 25 September 1991  
Received at publishers: 25 September 1991

---

## SUMMARY

New insights into the gene orders, structures, evolution, and functions of streptomycin (Sm) biosynthetic genes (*str*) were gained via hybridization studies, determination of nucleotide sequences, and measurement of expression in the *str* gene clusters of *Streptomyces griseus* and *S. glaucescens*. Both *str* clusters showed considerable divergence in macro and microstructure. Genes putatively involved in pathways leading to the (dihydro-)streptose and *N*-methyl-*L*-glucosamine moieties of Sm were identified. Additional regulatory elements, such as gene *strS* and conserved TTA codons in the N-terminal sections of reading frames, are reported. Evidences for the involvement of physiological state, signal transduction, and activators in the control of Sm production are presented.

---

## INTRODUCTION

The structural organization, functions, and regulation of the (hydroxy)-Sm production-encoding genes (*str*) have been investigated since 1984 by cloning, molecular analyses and expression studies in *S. griseus* and *S. glaucescens* (reviewed in Mansouri et al., 1989; Hunter and Baumberg, 1989; Distler et al., 1990). However, a complete picture cannot be given for any of the above aspects. A set of about 25 to 30 genes is expected to be directly involved, some 18 or 19 of which have been analyzed at least partially at present in the respective *str* clusters of both streptomycete species. This report summarizes part of the most recent progress on the structural and functional organization of *str*-encoding gene clusters and their regulation.

---

## RESULTS AND DISCUSSION

### (a) Comparison of two *str* gene clusters

A comparison of the currently known *str* genes from *S. griseus* N2-3-11 and *S. glaucescens* GLA.0 is given in Fig. 1. These clusters resemble each other to a large extent; but they also show some differences in organization. They include Sm modification (resistance) and de-modification (*strA*, *strK*), regulatory (e.g., *strR*, *strS*) and biosynthetic (e.g., *strB1*, *strDELM*) genes. When compared, the genes sequenced so far and their derived protein primary structures, from both organisms (Table I), vary in sequence identity between 85% (e.g., StrB1) and below 60% (e.g., StrB2), indicating that their development was subject to very different evolution though being components of a conserved gene cluster and that they did not diverge recently. Even less sequence similarity was retained in the comparable intercistronic regions, both in primary structures and lengths. For instance, the intercistronic *strA-strB1* (490 bp

---

**Correspondence to:** Dr. J. Distler, Bergische UniversitÃ¤t, FB9, Mikrobiologie, Gauss-Strasse 20, W-5600 Wuppertal (Germany).  
Tel. (49-202)439-2521; Fax (49-202)439-2698.

*Presented at the International Symposium on Biology of Actinomycetes, University of Wisconsin, Madison, WI (USA) 11â€“16 August 1991.*

**Abbreviations:** aa, amino acid(s); bp, base pair(s); CDP, cytidine diphosphate; 6DOH, 6-deoxy hexose(s); dTDP, deoxythymidine diphosphate; kb, kilobase(s) or 1000 bp; NMLG, *N*-methyl-*L*-glucosamine; nt, nucleotide(s); *P*, promoter; *S.*, Streptomyces; Sd, streptidine; SDS, sodium dodecyl sulfate; Sm, streptomycin; Str, products of the biosynthetic *str* gene; UDP, uridine diphosphate.
### Streptomyces griseus

#### Gene Cluster Map
- **Bgl** (BgII)
- **Bam** (BamHI)

| Position | Genes |
|----------|-------|
| 10       | strT  |
| 8        | strS  |
| 6        | strK  |
| 4        | strI  |
| 2        | strH  |
| 0        | strG, strF, strB1 |
| -2       | strA  |
| -4       | strR  |
| -6       | strD, strE, strL |
| -8       | strM, strB2, strN, strO |

---

#### Streptomyces glaucescens

#### Gene Cluster Map
- **Bgl** (BgII)
- **Bam** (BamHI)

| Position | Genes |
|----------|-------|
| 8        | strQ, strP, strH, strG, strF, strB1 |
| 6        | strA  |
| 4        | strK  |
| 2        | strS  |
| 0        | strT  |
| -2       | strR  |
| -4       | (strD, strE, strL) |
| -6       | strM, strB2, strN |
| -8       | strO  |
| -10      | strN  |
| -12      | strO  |

---

#### Legend for Symbols
- ðŸŸ©: regulation
- ðŸ”µ: phosphorylation/dephosphorylation of Sm
- â¬›: streptidine pathway
- ðŸŸ¨: streptose pathway
- ðŸŸ§: N-methyl-L-glucosamine pathway
- ðŸŸ¦: unknown function
- ðŸŸª: partial sequence only

---

### Figure 1. Comparison of the *str* gene clusters of S. griseus N2-3-11 and S. glaucescens GLA.0.
- Only genes identified by nt sequencing are indicated.
- Equivalent order and location of genes analogous to *strDEL* in S. glaucescens between the genes *strR* and *strM* was also suggested by hybridization data.
- Coordinates are given in kb referring to the start of the *strA* gene as position 1.

---

### Structural Elements
- Some significant structural elements (repeated 'box'-like elements, inverted repeats, etc.) identified in the S. griseus or S. glaucescens intercistronic regions were not conserved in the equivalent positions in both organisms, even if a clear function such as transcriptional terminators could be attributed (Distler et al., 1987; 1990; VÃ¶gtli and HÃ¼tter, 1987; Piepersberg et al., 1988).

---

### TABLE I: Percentage of aa identity among gene products from various *str* genes of Streptomyces griseus and Streptomyces glaucescens

| Gene Product<sup>a</sup> | Length (aa)<sup>b</sup> | % Identity (in shorter sequence)<sup>b,c</sup> |
|-------------------------|------------------------|---------------------------------------------|
| StrA, APH(6)            | 307                   | 73.3                                         |
| StrB1                   | 347                   | 85.0                                         |
| StrB2                   | 349                   | 59.5                                         |
| StrF                    | 281                   | 80.9                                         |
| StrG                    | 199                   | (71.7)                                       |
| StrH                    | 384                   | 79.3                                         |
| StrK                    | 449                   | (74.0)                                       |
| StrN                    | 319                   | 67.3                                         |
| StrR                    | 350                   | 62.5                                         |

#### Notes:
- <sup>a</sup> See Fig. 1.
- <sup>b</sup> Values in parentheses refer to incomplete sequences (StrG) or to the mature proteins without signal peptide (StrK).
- <sup>c</sup> The number of total aa in the shorter sequence is set to 100% in cases where differences in sequence length occur.

---

### (b) The *strDELM* genes of Streptomyces griseus N2-3-11
- Recently, information on the functions of the *strDELM* genes has accumulated, suggesting that they are responsible for the formation of dTDP-dihydrostreptose, a precursor of the streptose moiety of Sm, and a 6DOH derivative (Grisebach, 1978; Distler et al., 1987). The StrD enzyme was earlier postulated to be the dTDP-glucose synthetase (Distler et al., 1987).
- Interestingly, the gene products StrE and StrL were distantly related in their respective primary protein structures and to those of other enzymes (oxidoreductases/epimerases) of pyrimidine nucleotide-activated hexose metabolism; for example the UDP-glucose 4-epimerases from various organisms (e.g., GalE from S. lividans; Adams et al., 1988), and the 6DOH-synthases for (CDP-activated) tyvelose, abequose, and paratose, the group-specific sugars of some salmonellae (Verma and others).
Reeves, 1989; Wyk and Reeves, 1989). All these enzymes act mainly on the C-4 position of the sugar chain and seem to form a widespread and versatile family. StrE is likely to be the dTDP-glucose dehydratase because of its close sequence relationship to an enzyme of this activity from the erythromycin producer *Saccharopolyspora erythrea* (C.R. Hutchinson, personal commun.). Therefore, StrL should be the dTDP-dihydrostreptose synthase, which catalyses a reaction similar to the dTDP-L-rhamnose dehydrogenase (Grisebach, 1978). StrM, also involved in the oleandrose formation in *S. avermitilis* (see section b), is postulated to be the dTDP-4-keto-6-deoxyglucose 3,5-epimerase, which completes the requirements for the formation of dTDP-dihydrostreptose from glucose-1-phosphate (Fig. 2). It is interesting that genes closely related (hybridizing!) to the *strD*, *strE*, and *strM* genes could be identified in the production gene clusters of secondary metabolites, e.g., avermectin (D. MacNeill, personal communication), erythromycin, and daunorubicin (C.R. Hutchinson, personal communication). Since the biosynthetic pathways leading to all these compounds involve the formation and transfer of 6DOH constituents, we investigated the occurrence of *strD*-, *strE*-, and *strLM*-specific hybridization signals in the genomic DNA of more than 60 other actinomycete species, which were either known to produce 6DOH-containing metabolites or not. In more than two-thirds of these strains, positive signals were detected, at least with the *strD* and/or *strE* probes. Mostly these signals occurred on similar-sized fragments, indicating that 6DOH production genes are widespread, mostly clustered, and probably indicative of a large variety of primary and secondary metabolic gene clusters in actinomycetes. The scope of use of these probes could be even larger than that of the polyketide synthetase probes (Malpartida et al., 1987), because 6DOH moieties occur in more than 60 chemical variations, are constituents in a large number of natural products, and are found in all major chemical groups of aglycones (peptides, polyketides, nucleosides, aromatics, aminoglycosides).

(c) The *strPQ* genes of *Streptomyces glaucescens* GLA.0

The newly identified *strPQ* genes in the *str* cluster of *S. glaucescens* obviously encode enzymes also involved in the metabolism of nt-activated hexoses. The protein StrQ was distantly, but significantly related to the GlgC (ADP-glucose synthase) and StrD (dTDP-glucose synthase) gene products of *E. coli* and *S. griseus*, respectively. We postulate, therefore, that StrQ is the UDP-D-glucosamine synthase, the initial-step enzyme of the UDP-NMLG pathway. StrP is similarly related to the GalE (UDP-glucose 4-epimerase) gene products from other bacteria and yeasts (see above), and therefore a similar enzyme should be involved in NMLG formation, most probably a UDP-hexosamine (4-) epimerase. In view of the overall structure of the two gene clusters, this finding supports the idea that genes of the NMLG pathway are concentrated in the left part (cf. Fig. 1); evidence was presented for *strFG* to participate in this branch (Mansouri and Piepersberg, 1991). Since the streptose-encoding genes were located to the right (see section a) of the central resistance/regulatory subcluster (genes *strA* and *strR*), and since there is evidence that the cluster ends close to the left of the *strQ* gene in *S. glaucescens* (VÃ¶gtli, 1987), it seems likely that the still

---

**Figure 2. Routes of biosynthesis and modification of Sm. The involvement of *str* gene products is indicated (see Fig. 1). Abbreviations: G-6(1)-P, glucose-6(1)-phosphate; MI, myo-inositol; D-GNH2-1P, D-glucosamine-1-phosphate; APH(6), aminoglycoside 6-phosphotransferase; APH(3"), aminoglycoside 3"-phosphotransferase.**

![Diagram](attachment:diagram.png)

lacking genes (probably eight genes) for the streptidine pathway have to be searched for on the right side (Fig. 1).

(d) StrK, an extracellular Sm-phosphate phosphatase

The StrK products of *S. griseus* and *S. glaucescens* share 74% sequence identity in their mature proteins and show about 50% protein identity to the PhoA (alkaline phosphatase) gene product of *E. coli* (Mansouri and Piepersberg, 1991; and unpublished observations). The activity of the *S. griseus* StrK enzyme, as measured in culture supernatants, is that of an alkaline phosphatase with a preference of Sm-6-phosphate over Sm-3"-phosphate as a substrate. At least in *S. glaucescens* this gene could be under the control of the phosphate-regulated activator/repressor systems (Martin, 1989; Distler et al., 1990). The StrK genes were differently placed relative to upstream str genes in both organisms studied (Fig. 1). Nevertheless, they could be crucial for the expression of str genes in both clusters, since they are in close neighborhood to a gene, strS, which seems to encode another pleiotropic regulator (see section f).

(e) StrR regulator protein of Sm biosynthesis

StrR is a positive regulator of at least the strB1 gene of *S. griseus*. In an earlier analysis of the strB1 promoter in a promoter probe vector and of the strB1 expression in absence and presence of the strR gene, we postulated an antitermination mechanism for the regulation of StrB1 by StrR (Distler et al., 1987; 1990). However, more recent studies suggest a more complicated or even a completely different mode of regulation. The antitermination model was based on (i) the strong promoter activity of the PstI (838)-BamHI (1193) DNA fragment; (ii) the loss of promoter activity when DNA fragments extending the BamHI (1193) restriction site were analyzed (Distler et al., 1990; Fig. 3A) and (iii) the restoration of the promoter activity of these DNA fragments, when the strR gene was complemented in trans (Fig. 3A). By comparative S1-mapping experiments of the promoter PstrB1 in *S. lividans* pJDM401 (strR lacking; Distler et al., 1987) and *S. lividans* pJDM40 (strR positive; Distler et al., 1987), no transcripts reaching beyond the BamHI site could be detected in the StrR lacking *S. lividans* strain (Fig. 3B). At least one S1-resistant

![Diagram]

Fig. 3. Expression from the PstrB1 of *S. griseus* in presence and absence of the StrR activator protein in *S. lividans* TK23. The location of the PstrB1 promoter and the relevant restriction sites in the intercistronic region between strA and strB1 are shown (see Fig. 1). (A) The expression of the neomycin-resistance conferring phosphotransferase APH(3') encoded by the promoter probe vector plJ487 (Ward et al., 1986) under the control of the indicated DNA fragments (heavy black bars) was measured in vitro (Distler et al., 1987) in absence (- StrR) and presence (+ StrR) of the strR gene cloned on a second replicon (pGM8; Muth et al., 1989; Distler et al., 1990). (B) S1 mapping experiments (Favaloro et al., 1980) of the PstrB1 promoter were carried out with RNA prepared from *S. lividans* pJDM40 (+ StrR; Distler et al., 1987) and pJDM401 (- StrR; Distler et al., 1987) and the indicated 3' and 5'-labelled (Maniatis et al., 1982) DNA fragments (lines). The lengths of S1-resistant DNA fragments are given (-, no detectable DNA/RNA hybrid).

DNA fragment should have been detected in *S. lividans* pJDM401, because the *PstI-BamHI* DNA fragment, cloned in the promoter probe pIJ487, showed a strong promoter activity. Explanations for these contradictory results could be: (i) StrR neutralized a terminator between nt positions 1193 and 1306, and the small transcripts expected in the *strR* negative strain could be too unstable to be detected by the method used, or (ii) the strong promoter activity of the *PstI-BamHI* DNA fragment could be caused by the creation of an artificial promoter at the cloning site in pIJ487 (experiments are in progress to clarify this point). In this case, the putative site of antitermination would be located upstream from the *BamHI* (1193) restriction site, or StrR acts as a 'normal' activator and not as an antiterminator protein.

The StrR gene products of *S. glaucescens* and *S. griseus* have 62.5% identical aa and conserved helix-turn-helix motifs, which implies a similar mechanism and/or DNA binding site of the two positive regulator proteins. Experiments are in progress to verify this speculation. If the two StrR proteins have the same mode of action, the *strA-strB1* intercistronic DNA region in *S. griseus* and *S. glaucescens* sequences should share some functional and/or structural similarities. The overall identity was only 55%. However, several conserved sequence elements could be identified upstream and downstream from the promoter *PstrBl*, all within a region from nt position 1020 to the *BamHI* site at position 1193. These observations, in combination with the results of the promoter probe studies, the data of the S1-mapping experiments (cf. Fig. 3) and the repression of Sm production by the *BamHI* (517â€“1193) DNA fragment cloned on a multi-copy plasmid in *S. griseus* (Ohnuki et al., 1985) suggest that the main regulation probably occurs via activation at cis-acting elements within the *PstI* (839)-*BamHI* (1193) segment.

(f) Elements of pleiotropic regulation of Sm biosynthesis

Leskiw et al. (1991) had demonstrated that the *bldA* gene product, necessary to differentiation in *S. coelicolor*, codes for a Leu-specific tRNA recognizing the Leu codon UUA, which is rare in Streptomycetes. Genes containing the TTA codon are translationally regulated by the size of the tRNA<sup>Leu</sup> pool, as was recently shown for several genes, including the transcriptional activator in the actinorhodin gene cluster, ActIII (Leskiw et al., 1991). During the analysis of the *str* genes from *S. griseus* and *S. glaucescens* it became evident that a *bldA*-homologous gene in *S. griseus* and *S. glaucescens* may control Sm biosynthesis in a similar way. In both gene clusters the triplet TTA occurred in the activator gene *strR* (30th codon), in the *strN* gene (2nd codon) and in the *strA* gene (39th codon) of *S. glaucescens*. All these genes are the first ones in the respective transcription units (*strR-strA* in *S. griseus*, *strR* and *strA* in

*S. glaucescens*, and *strNB1M* in both strains; Fig. 1). BldA-negative mutants and homologous *bldA* gene have been identified in Sm-producing *S. griseus* (Kendrick et al., 1992). This could mean that all independently expressed key genes are regulated coordinately on the translational level via this mechanism.

With the *strS* gene a second putative regulatory gene was identified in the *str*-gene clusters of *S. griseus* and *S. glaucescens*. StrS resembles the in trans-acting regulatory proteins DegT (*Bacillus stearothermophilus*; Takagi et al., 1990), DnrJ (*S. peuceticus*; C.R. Hutchinson, personal communication) and EryCl (*Saccharopolyspora erythrae*; Dhillon et al., 1989). C.R. Hutchinson, S.L. Otten and K.J. Stutzmann-Engwall have speculated that these proteins function as the protein kinase sensor proteins in a two-component regulatory system. Although StrS and the three other proteins do not have the conserved ATP-binding motif typically found near the C terminus of the protein kinase components, they all contain two in the known protein kinase sensor protein's highly conserved aa: a His (site of autophosphorylation) and at a distance of approx. 100 aa a conserved Asp (Stock et al., 1989). Recently, we were able to determine the N-terminal aa sequences of several protein spots isolated from two-dimensional protein gels. One of these sequences was identical with the deduced N-terminal sequence of StrS.

![Figure 4](#fig4)

Fig. 4. In vitro phosphorylation of proteins in cell-free extracts of *S. griseus*. Cells of strains (lanes): a, N2-3-11 (wt); b, mutant M881 (A-factor and Sm non-producing; A-factor added at 20 Î¼g/ml; c, M881 (without A-factor); d, W67 (A-factor producing, Sm nonproducing) were grown in minimal medium (Distler et al., 1990) and harvested at different stages of growth phase. The in vitro phosphorylation of 25 Î¼g protein [Î³-<sup>32</sup>P]ATP was carried out as described elsewhere (Stowe et al., 1989); phosphorylated proteins were separated on 0.1% SDS-12.5% polyacrylamide gels and identified by autoradiography. The migration of molecular standard markers is indicated on margins.

The molecular size of the putative StrS protein was determined to be 43 kDa in SDS-polyacrylamide gels, which is in good correlation with the estimated size of 39.8 kDa deduced from the nt sequence. The protein StrS could be detected in cell extracts of *S. griseus* N2-3-11 grown for 22 h (the onset of Sm production) in minimal media (Distler et al., 1990), as a major protein spot. It was absent in extracts from the pleiotropic mutant M881 (A-factor nonproducer), but was induced by A-factor at a concentration of 2 Î¼g/ml.

To get a first insight into the postulated activation mechanism by StrS, we analyzed the pattern of phosphorylated proteins in *S. griseus* and two pleiotropic, Sm nonproducer mutants M881 (A-factor nonproducer, A-factor sensitive) and W67 (A-factor producer, A-factor insensitive) after 9 h, 22 h and 36 h growth in minimal media. In cell extracts of *S. griseus* five proteins (45, 43, 39, 29 and 26 kDa) were phosphorylated with the onset of Sm production at 22 h (Fig. 4). None of these proteins could be detected in the exponential or late stationary growth phase of *S. griseus*, nor in the mutants M881 and W67, but the phosphorylation or the expression of two proteins (43 and 29 kDa) was induced by A-factor (Fig. 4).

### (g) Conclusions

(1) The *str* gene cluster seems to be a very old and conserved genetic trait in a subfraction of all streptomycete species.

(2) Though the *str* genes are not strictly ordered according to function, it seems that the main genes involved in the three branches of the Sm pathway (formation of Sd, 6DOH and NMLG subunits) are assembled in subclusters.

(3) The *strDE* genes seem to be markers for similar subclusters in a majority of streptomycetes and can be used for the detection of 6DOH pathways.

(4) The alkaline Sm-phosphate phosphatase, StrK, and its gene could be involved in phosphate-regulation of Sm formation on both the gene and the enzyme levels.

(5) The regulation of the *str* gene cluster seems to be (i) very complex, (ii) under the control of both internal (e.g., StrR and StrS protein, tRNA<sup>Leu</sup><sub>UUA</sub>) and external (e.g., A-factor, phosphate, etc.) control, and (iii) to be partially different in *S. griseus* and *S. glaucescens*.

(6) A-factor seems to control the expression of all regulatory genes (*strR*, *strS*) controlling production of Sm in *S. griseus* and could be mediated by protein factors regulated via protein phosphorylation.

---

### ACKNOWLEDGEMENTS

The expert technical assistance of S. Kuberski, N. Ratka and P. VoigtlÃ¤nder is gratefully acknowledged. This work was supported by grants from the Deutsche Forschungsgemeinschaft (to J.D. and W.P.).

---

### REFERENCES

Adams, C.W., Fornwald, J.A., Schmidt, F.J., Rosenberg, M. and Brawner, M.: Gene organization and structure of the *Streptomyces lividans gal* operon. J. Bacteriol. 170 (1988) 203â€“212.

Dhillon, N., Hale, R.S., Cortes, J. and Leadlay, P.F.: Molecular characterization of a gene from *Saccharopolyspora erythraea* (*Streptomyces erythraeus*) which is involved in erythromycin biosynthesis. Mol. Microbiol. 3 (1989) 1405â€“1414.

Distler, J., Ebert, A., Mansouri, K., Pissowatzki, K., Stockmann, M. and Piepersberg, W.: Gene cluster for streptomycin biosynthesis in *Streptomyces griseus*: nucleotide sequence of three genes and analysis of transcriptional activity. Nucleic Acids Res. 15 (1987) 8041â€“8056.

Distler, J., Mansouri, K., Mayer, G. and Piepersberg, W.: Regulation of biosynthesis of streptomycin. In: Heslot, H., Davies, J., Florent, J., Bobichon, L., Durand, G. and Penasse, L. (Eds.), Proceedings of GIM'90, Vol. I. SociÃ©tÃ© FranÃ§aise de Microbiologie, Paris, 1990, pp. 379â€“392.

Favaloro, J., Treisman, R. and Kamen, R.: Transcription maps of polyoma virus-specific RNA: analysis by two-dimensional nuclease S1 gel mapping. Methods Enzymol. 65 (1980) 772â€“781.

Grisebach, H.: Biosynthesis of sugar components of antibiotic substances. Adv. Carbohydr. Chem. Biochem. 35 (1978) 81â€“126.

Hunter, I.S. and Baumberg, S.: Molecular genetics of antibiotic formation. In: Baumberg, S., Hunter, I.S. and Rhodes, P.M. (Eds.), Microbial Products: New Approaches. Cambridge University Press, Cambridge, MA, 1989, pp. 121â€“162.

Kendrick, K.: Genetic regulation of *Streptomyces griseus* morphogenesis. Genet. (1992)

Leskiw, B.K., Lawlor, E.J., Fernandez-Abalos, J.M. and Chater, K.F.: TTA codons in some genes prevent their expression in a class of developmental, antibiotic-negative, *Streptomyces* mutants. Proc. Natl. Acad. Sci. USA 88 (1991) 2461â€“2465.

Malpartida, F., Hallam, S.E., Kieser, H.M., Motamedi, H., Hutchinson, C.R., Butler, M.J., Sugden, D.A., Warren, M., McKillop, C., Bailey, C.R., Humphreys, G.O. and Hopwood, D.A.: Homology between *Streptomyces* genes coding for synthesis of different polyketides used to clone antibiotic biosynthetic genes. Nature 325 (1987) 818â€“821.

Maniatis, T., Fritsch, E.F. and Sambrook, J.: Molecular Cloning. A Laboratory Manual. Cold Spring Harbor Laboratory, Cold Spring Harbor, NY, 1982.

Mansouri, K. and Piepersberg, W.: Genetics of streptomycin production in *Streptomyces griseus*: nucleotide sequence of five genes, *strFGHIK*, including a phosphatase gene. Mol. Gen. Genet. 228 (1991) 459â€“469.

Mansouri, K., Pissowatzki, K., Distler, J., Mayer, G., Heinzel, P., Braun, C., Ebert, A. and Piepersberg, W.: Genetics of streptomycin production. In: Hershberger, C.L., Queener, S.W. and Hegeman, G. (Eds.), Genetics and Molecular Biology of Industrial Microorganisms. American Society for Microbiology, Washington, DC, 1989, pp. 61â€“67.

Martin, J.F.: Molecular mechanisms for the control by phosphate of the biosynthesis of antibiotics and other metabolites. In: Shapiro, S. (Ed.), Regulation of Secondary Metabolism in Actinomycetes. CRC Press, Boca Raton, FL, 1989, pp. 213â€“237.

Muth, G., Nussbaumer, W., Wohlleben, W. and PÃ¼hler, A.: A vector system with temperature-sensitive replication for gene disruption and mutational cloning in Streptomycetes. Mol. Gen. Genet. 219 (1989) 341â€“348.

Ohnuki, T., Imanaka, T. and Aiba, S.: Self-cloning in *Streptomyces griseus*

of an *str* gene cluster for streptomycin biosynthesis and streptomycin resistance. J. Bacteriol. 164 (1985) 85â€“94.

Piepersberg, W., Distler, J., Ebert, A., Heinzl, P., Mansouri, K., Mayer, G. and Pissowatzki, K.: Expression of genes for streptomycin biosynthesis. In: Okami, Y., Beppu, T. and Ogawara, H. (Eds.), Biology of Actinomycetes '88. Japan Scientific Societies Press, Tokyo, 1988, pp. 86â€“91.

Stock, J.B., Ninfa, A.J. and Stock, A.M.: Protein phosphorylation and regulation of adaptive responses in bacteria. Microbiol. Rev. 53 (1989) 450â€“490.

Stowe, D.J., Atkinson, T. and Mann, N.H.: Protein kinase activities in cell-free extracts of *Streptomyces coelicolor* A3(2). Biochimie 71 (1989) 1101â€“1105.

Takagi, M., Takada, H. and Imanaka, T.: Nucleotide sequence and cloning in *Bacillus subtilis* of the *Bacillus stearothermophilus* pleiotropic regulatory gene *degT*. J. Bacteriol. 172 (1990) 411â€“418.

Verma, N. and Reeves, P.: Identification and sequence of *rfbS* and *rfbE*, which determine antigenic specificity of group A and group D salmonellae. J. Bacteriol. 171 (1989) 5694â€“5701.

VÃ¶gli, M.: Structure and Expression of the *sph* Gene of *Streptomyces glaucescens* and its Genomic Environment. Dissertation, ETH, ZÃ¼rich, 1987.

VÃ¶gli, M. and HÃ¼tter, R.: Characterization of the hydroxystreptomycin phosphotransferase gene (*sph*) of *Streptomyces glaucescens*: nucleotide sequencing and promoter analysis. Mol. Gen. Genet. 208 (1987) 195â€“203.

Walker, J.B.: Pathways of the guanidinated inositol moieties of streptomycin and bluensomycin. Methods Enzymol. 43 (1975) 429â€“470.

Ward, J.M., Janssen, G.R., Kieser, T., Bibb, M.J. and Bibb, M.J.: Construction and characterization of a series of a multicopy promoter-probe plasmid vectors for *Streptomyces* using the aminoglycoside phosphotransferase gene from Tn5 as indicator. Mol. Gen. Genet. 203 (1986) 468â€“478.

Wyk, P. and Reeves, P.: Identification and sequence of the gene for abequose synthase, which confers antigenic specificity on group B salmonellae: homology with galactose epimerase. J. Bacteriol. 171 (1989) 5687â€“5693.
