
The Plant Cell, Vol. 15, 2343–2356, October 2003, www.plantcell.org © 2003 American Society of Plant Biologists

# Highly Divergent Methyltransferases Catalyze a Conserved Reaction in Tocopherol and Plastoquinone Synthesis in Cyanobacteria and Photosynthetic Eukaryotes

Zigang Cheng,$^{a}$ Scott Sattler,$^{a}$ Hiroshi Maeda,$^{a}$ Yumiko Sakuragi,$^{b}$ Donald A. Bryant,$^{b}$ and Dean DellaPenna$^{a,1}$

$a$ Department of Biochemistry and Molecular Biology, Michigan State University, East Lansing, Michigan 48824  
$b$ Department of Biochemistry and Molecular Biology, Pennsylvania State University, University Park, Pennsylvania 16802  

Tocopherols are lipid-soluble compounds synthesized only by photosynthetic eukaryotes and oxygenic cyanobacteria. The pathway and enzymes for tocopherol synthesis are homologous in cyanobacteria and plants except for 2-methyl-6-phytyl-1,4-benzoquinone/2-methyl-6-solanyl-1,4-benzoquinone methyltransferase (MPBQ/MSBQ MT), which catalyzes a key methylation step in both tocopherol and plastoquinone (PQ) synthesis. Using a combined genomic, genetic, and biochemical approach, we isolated and characterized the VTE3 (vitamin E defective) locus, which encodes MPBQ/MSBQ MT in Arabidopsis. The phenotypes of *vte3* mutants are consistent with the disruption of MPBQ/MSBQ MT activity to varying extents. The ethyl methanesulfonate-derived *vte3-1* allele alters tocopherol composition but has little impact on PQ levels, whereas the null *vte3-2* allele is deficient in PQ and $\alpha$- and $\gamma$-tocopherols. In vitro enzyme assays confirmed that VTE3 is the plant functional equivalent of the previously characterized MPBQ/MSBQ MT (Sll0418) from *Synechocystis* sp PCC6803, although the two proteins are highly divergent in primary sequence. Sll0418 orthologs are present in all fully sequenced cyanobacterial genomes, *Chlamydomonas reinhardtii*, and the diatom *Thalassiosira pseudonana* but absent from vascular and nonvascular plant databases. VTE3 orthologs are present in all vascular and nonvascular plant databases and in *C. reinhardtii* but absent from cyanobacterial genomes. Intriguingly, the only prokaryotic genomes that contain VTE3-like sequences are those of two species of archaea, suggesting that, in contrast to all other enzymes of the plant tocopherol pathway, the evolutionary origin of VTE3 may have been archael rather than cyanobacterial. In vivo analyses of *vte3* mutants and the corresponding homozygous *Synechocystis* sp PCC6803 *sll0418::aphII* mutant revealed important differences in enzyme redundancy, the regulation of tocopherol synthesis, and the integration of tocopherol and PQ biosynthesis in cyanobacteria and plants.

## INTRODUCTION

Tocopherols, collectively termed vitamin E, are a class of lipid-soluble compounds that are synthesized only by oxygenic photosynthetic organisms. All tocopherols are amphipathic molecules with polar head groups exposed to the membrane surface and hydrophobic tails that interact with the acyl groups of membrane lipids. Four types of tocopherols ($\alpha$-, $\beta$-, $\gamma$-, and $\delta$-tocopherols [$\alpha$-, $\beta$-, $\gamma$-, and $\delta$-T, respectively]) are synthesized naturally and differ only in the number and position of methyl substituents on the chromanol ring (Figure 1). Tocopherols are essential dietary components for humans and other mammals; as a result, most of our understanding of tocopherol function has been derived from studies in these systems (for reviews, see Hanck, 1985; Brigelius-Flohe and Traber, 1999; Valk and Hornstra, 2000; Brigelius-Flohe et al., 2002; Ricciarelli et al., 2002). Studies in mammals, animal cell cultures, and artificial membranes have shown that tocopherols help maintain membrane structure and integrity (Srivastava et al., 1989), act as antioxidants and free radical scavengers (Tappel, 1962; Jialal and Fuller, 1993; Jialal et al., 2001; Behl and Moosmann, 2002), and perform other nonantioxidant functions related to signaling and transcriptional regulation (Azzi et al., 1995; Grau and Ortiz, 1998; Ricciarelli et al., 2002). The functions of tocopherols in photosynthetic organisms have yet to be determined, but they are likely to include unique functions in addition to those reported in animals (Noctor and Foyer, 1998; Grasses et al., 2001; Reverberi et al., 2001). Mutant and transgenic approaches in Arabidopsis and *Synechocystis* sp PCC6803 that eliminate tocopherols (Collakova and DellaPenna, 2001; Schledz et al., 2001), replace tocopherols with biosynthetic intermediates (Porfirova et al., 2002; Sattler et al., 2003), or increase tocopherol levels (Collakova and DellaPenna, 2001; Savidge et al., 2002) are beginning to provide insight into tocopherol functions in photosynthetic organisms.

Tocopherols are synthesized by a pathway that is conserved between cyanobacteria and plants (Figure 1) (Soll et al., 1980, 1985; Lichtenthaler et al., 1981; Norris et al., 1995, 1998). The conversion of $p$-hydroxyphenylpyruvate (HPP) to homogentisic acid (HGA) by HPP dioxygenase (HPPD) yields the aromatic head group for both tocopherol and plastoquinone biosynthesis in plants (Norris et al., 1998). The committed step in tocopherol synthesis is the condensation of HGA and phytyldiphosphate by homogentisate phytyltransferase (HPT) to produce the first tocopherol intermediate, 2-methyl-6-phytylbenzoquinone (MPBQ) (Collakova and DellaPenna, 2001; Schledz et al., 2001; Savidge et al., 2002). Biochemical analyses have shown that the steps leading from MPBQ to $\alpha$-tocopherol are as follows: (1) ring

1 To whom correspondence should be addressed. E-mail dellapen@pilot.msu.edu; fax 517-353-9334.  
Article, publication date, and citation information can be found at www.plantcell.org/cgi/doi/10.1105/tpc.013656.

HPP

1

OH

HOOC₂H

OH

HGA

3

CO₂ + PPi

Phytyl-DP

2

CO₂ + PPi

PP [CH₂=CH]₉ H

Solanyl-DP

HO

HO

MPBQ

4

HO

C=C

OH

DMPBQ

SAM

HO

C=C

OH

MSBQ

5

HO

O

δ-tocopherol

6

SAM

HO

O

β-tocopherol

HO

O

γ-tocopherol

SAM

6

HO

O

α-tocopherol

Plastoquinone

Figure 1. The Tocopherol Biosynthetic Pathway in Plants and Cyanobacteria and the PQ Biosynthetic Pathway in Plants.

Boldface arrows represent the steps leading to α-tocopherol, the most abundant tocopherol produced in wild-type Arabidopsis leaves and Synechocystis sp PCC6803. DMPBQ, 2,3-dimethyl-5-phytyl-1,4-benzoquinone; HGA, homogentisic acid; HPP, p-hydroxyphenylpyruvate; MPBQ, 2-methyl-6-phytyl-1,4-benzoquinone; MSBQ, 2-methyl-6-solanyl-1,4-benzoquinone; phytyl-DP, phytyldiphosphate; SAM, S-adenosylmethionine; solanyl-DP, solanyldiphosphate. Enzymes are indicated by circled numbers: 1, HPP dioxygenase (HPPD); 2, homogentisate phytyltransferase (HPT); 3, homogentisate solanyltransferase (HST); 4, MPBQ/MSBQ methyltransferase; 5, tocopherol cyclase (TC); and 6, γ-tocopherol methyltransferase (γ-TMT).

metylation of MPBQ by 2-methyl-6-phytyl-1,4-benzoquinone/2-methyl-6-solanyl-1,4-benzoquinone methyltransferase (MPBQ/MSBQ MT) to yield 2,3-dimethyl-5-phytylbenzoquinone (DMPBQ) (Soll et al., 1985); (2) ring cyclization of DMPBQ by tocopherol cyclase (TC) to yield γ-T (Soll et al., 1985; Arango and Heise, 1998; Porfirova et al., 2002; Sattler et al., 2003); and (3) a second ring methylation by γ-tocopherol methyltransferase (γ-TMT) to yield α-T (Soll et al., 1980; d’Harlingue and Camara, 1985; Shintani and DellaPenna, 1998). Alternatively, MPBQ can be cyclized to form δ-T and then methylated by γ-TMT to yield β-T. All enzymatic activities for tocopherol synthesis in plants have been localized to the inner chloroplast envelope except HPPD, which is cytosolic (Soll et al., 1980, 1985).

In plants, the lipid-soluble, plastid-localized electron carrier plastoquinone (PQ) is synthesized by the pathway shown in Figure 1. As in tocopherol synthesis, the committed step in PQ synthesis is condensation of the aromatic compound HGA with a prenyldiphosphate, solanyldiphosphate, to yield 2-methyl-6-solanylbenzoquinone (MSBQ) by homogentisate solanyltransferase, an activity distinct from HPT (Norris et al., 1995; Collakova and DellaPenna, 2001). MSBQ then is methylated at the same ring position as MPBQ to yield PQ. The similarity of the MPBQ and MSBQ structures has led to the proposal that a single enzyme performs the methylation of both compounds. This has been demonstrated for MPBQ/MSBQ methyltransferase (MPBQ/MSBQ MT) from the cyanobacterium Synechocystis
PCC6803, which, when cloned and assayed in *Escherichia coli*, was capable of using both MPBQ and MSBQ as substrates (Shintani et al., 2002). Whether such a multifunctional activity exists in plants is unclear.

Although the plant tocopherol and PQ biosynthetic pathways were elucidated in labeling studies during the 1980s (d’Harlingue and Camara, 1985; Marshall et al., 1985; Soll et al., 1985), the membrane association and low specific activity of most pathway enzymes have hindered their isolation and characterization. Most tocopherol pathway enzymes have been isolated only recently by combining genetic and genomic approaches in *Synechocystis* sp PCC6803 and Arabidopsis, in which the isolation of an enzyme from one organism has facilitated the isolation of the respective ortholog from the other. Sequencing of the *Synechocystis* sp PCC6803 and Arabidopsis genomes has greatly facilitated this process, so that now, HPPD, HPT, TC, and γ-TMT have been cloned and characterized from both organisms (Norris et al., 1998; Shintani and DellaPenna, 1998; Collakova and DellaPenna, 2001; Shintani et al., 2002; Porfirova et al., 2002; Sattler et al., 2003). Although these studies have shown that these four pathway steps and enzymes are conserved between cyanobacteria and plants, other studies have provided evidence that portions of the tocopherol and PQ pathways are not identical in the two organisms.

One such example is the different requirement of HGA for PQ synthesis in cyanobacteria and higher plants. Although the *Synechocystis* sp PCC6803 and Arabidopsis genomes both encode HPPD enzymes, disruption of HPPD activity in the two organisms yields drastically different phenotypes. A null Arabidopsis HPPD mutant is deficient in both tocopherol and PQ and is seedling lethal (Norris et al., 1998), whereas the orthologous *Synechocystis* sp PCC6803 mutant is viable and lacks tocopherols only (Dahnhardt et al., 2002). This finding suggests that, unlike in plants, the aromatic head group for PQ synthesis in *Synechocystis* sp PCC6803 is not derived from HGA or that there is an alternative route for HGA synthesis in this organism. The different phenotypes of HPPD-deficient Arabidopsis and *Synechocystis* sp PCC6803 mutants indicate that the mere presence of functional orthologs in cyanobacteria and plants does not necessarily equal identical biosynthetic pathways.

The only tocopherol pathway enzyme that has not yet been cloned from plants is MPBQ/MSBQ MT. MPBQ/MSBQ MT activity has been demonstrated in spinach chloroplasts (Soll et al., 1985), and maize and sunflower mutants have been identified with phenotypes consistent with the disruption of MPBQ/MSBQ MT activity (Cook and Miles, 1992; Demurin et al., 1996). MPBQ/MSBQ MT has been cloned and characterized from *Synechocystis* sp PCC6803 (Shintani et al., 2002), but despite the high degree of evolutionary conservation between plants and cyanobacteria for other tocopherol pathway enzymes, no obvious orthologs could be identified in the completed Arabidopsis and rice genomes and numerous plant EST databases. This finding suggests that, analogous to the different routes to PQ head group synthesis in *Synechocystis* sp PCC6803 and Arabidopsis (Norris et al., 1998; Dahnhardt et al., 2002), MPBQ/MSBQ MT also may differ between cyanobacteria and plants. Here, we report the identification and characterization of a novel MPBQ/MSBQ MT from Arabidopsis that has

orthologs in all plant, but not cyanobacterial, databases. The low sequence identity between the cyanobacterial and plant MPBQ/MSBQ MTs suggests that they are nonorthologous, functionally equivalent enzymes that arose independently during the evolution of plants and cyanobacteria.

### RESULTS

#### Search for an Arabidopsis Homolog of *Synechocystis* sp PCC6803 MPBQ/MSBQ MT Using Genome-Based Approaches

To better understand tocopherol synthesis in plants, we attempted to identify an Arabidopsis ortholog of *Synechocystis* sp PCC6803 MPBQ/MSBQ MT, which had been identified previously as open reading frame (ORF) *slr0418* (Shintani et al., 2002). Slr0418 orthologs were identified using tBLASTn (Basic Local Alignment Search Tool; P < 10⁻⁸¹) in the fully sequenced cyanobacterial genomes of *Anabaena* sp PCC7120, *Thermosynechococcus elongatus*, *Prochlorococcus marinus* MIT9313 and MED4, and *Synechococcus* sp WH8102, in the partially sequenced cyanobacterial genomes of *Synechococcus* sp PCC7002 and *Trichodesmium erythraeum* IMS101, in the genome and EST databases of the green alga *Chlamydomonas reinhardtii*, and in the raw genome sequence data of the diatom *Thalassiosira pseudonana* (Figure 2). However, exhaustive searches of public DNA databases for vascular and nonvascular plants, including the complete Arabidopsis and rice genomes, failed to identify a convincing ortholog of *Synechocystis* sp PCC6803 *slr0418*. The two proteins in the Arabidopsis genome with the greatest similarity to Slr0418 are γ-TMT (P < 10⁻³⁷), which is not active toward MPBQ when assayed in vitro (data not shown), and SMT1 (P < 10⁻¹⁸), a sterol methyltransferase that is not targeted to the chloroplast (Diener et al., 2000).

In a parallel approach, 93 predicted proteins in the Arabidopsis genome were identified that contain motifs characteristic of conserved S-adenosylmethionine binding domains in methyltransferases (Kagan and Clarke, 1994). Eleven of these proteins, one of which was γ-TMT, also were predicted to be targeted to the chloroplast, the known subcellular location of plant MPBQ/MSBQ MT activity (Soll et al., 1985). The protein sequences of these putative chloroplast-targeted methyltransferases were aligned with methyltransferases of known functions, including previously characterized methyltransferases involved in tocopherol synthesis (Figure 2). The resulting phylogenetic tree indicated that none of the Arabidopsis sequences clustered with the *Synechocystis* sp PCC6803 MPBQ/MSBQ MT clade (Figure 2). Given that plant chloroplasts are known to contain MPBQ/MSBQ MT activity (Soll et al., 1985), these combined data suggest that, unlike the other enzymes of the tocopherol pathway, Arabidopsis and *Synechocystis* sp PCC6803 MPBQ/MSBQ MTs share little identity at the level of the primary amino acid sequence.

#### Identification of Arabidopsis MPBQ/MSBQ MT Mutants and Cloning of the VTE3 Locus

A genetic approach was used concurrently with genomic approaches to identify the MPBQ/MSBQ MT locus in Arabidopsis.
0.213 rat glycine N-MT

0.241 C. reinhardtii

0.169 O. sativa

0.124 At3g63410

0.129 S. oleracea

0.274 H. sp. NRC-1 "VTE 3"

0.292 A. fulgidus "VTE 3"

0.258 S. c. ERG6

0.107 Arabidopsis SMT 1

0.105 Z. mays C-24 SMT

0.276 Arabidopsis

0.175 S. sp. PCC 6803

0.16 N. punctiforme

0.139 T. elongatus BP-1

0.139 S. sp. PCC 6803

0.12 N. sp. PCC 7120

0.124 T. erythraem

0.227 P. m. MED 4

0.148 S. sp. WH8102

0.15 P. m. MIT 9313

0.305 C. reinhardtii

0.342 T. pseudonana

0.453 At5g53920

0.445 M. sativa

0.451 At3g20650

0.451 At3g56330

0.45 At5g17660

0.451 At3g05100

0.442 At1g01860

0.436 At3g13180

0.437 At5g49020

0.417 At1g23360

Figure 2. Phylogeny of Selected Methyltransferases from Photosynthetic Organisms.

Rat Gly N-methyltransferase (N-MT) is included as an outgroup. Boldface lettering indicates that the activity of the encoded protein has been demonstrated biochemically. Known or predicted functions for individual clades are indicated at right. A. fulgidus, Archaeoglobus fulgidus; At3g 63410, Arabidopsis VTE 3; H. sp NRC-1, Halobacterium sp NRC-1; M. sativa, alfalfa chalcone O-methyltransferase (O-MT); N. sp PCC 7120, Nostoc sp. PCC 7120; P. m. MED 4, Prochlorococcus marinus MED 4; P. m. MIT 9313, Prochlorococcus marinus MIT 9313; S. c., Saccharomyces cerevisiae; SMT, sterol methyltransferase; S. sp PCC 6803, Synechocystis sp PCC 6803; S. sp WH8102, Synechococcus sp WH8102; T. elongatus BP-1, Thermosynechococcus elongatus BP-1; T. erythraem, Trichodesmium erythraem IMS101; γ-TMT, γ-tocopherol methyltransferase; VTE3, plant-type MPBQ/MSBQ MT. All Arabidopsis gene numbers follow the standard genomic nomenclature: At#g#####.

A rapid HPLC-based screen of Arabidopsis leaf tissue was developed to identify mutants with altered leaf tocopherol profiles (Sattler et al., 2003). The screening of an ethyl methane-sulfonate-mutagenized Arabidopsis population (ecotype Columbia) yielded one mutant line that, compared with the wild type, had reduced levels of α-T and γ-T and greatly increased levels of β-T and δ-T (Table 1). This is a biochemical phenotype consistent with a reduction in MPBQ/MSBQ MT activity (Figure 1). The mutant was designated vte3-1 (vitamin E defective). A map-based cloning approach was used to isolate the VTE3 locus. vte3-1 was crossed to wild-type Landsberg erecta, and an F2 population was screened by HPLC for the vte3-1 phenotype. Seventy-two of 276 F2 plants were identified as vte3-1 homozygotes, indicating that vte3-1 is recessive (χ² = 0.168).

Table 1. Analysis of Prenyllipids in Wild-Type and vte3 Arabidopsis Leaf Tissue and Synechocystis Cell Culture

| Sample                     | Plastoquinone (pmole/mg) | Total Tocopherol (pmole/mg) | α-Tocopherol (pmole/mg) | β-Tocopherol (pmole/mg) | γ-Tocopherol (pmole/mg) | δ-Tocopherol (pmole/mg) |
|----------------------------|--------------------------|-----------------------------|--------------------------|--------------------------|--------------------------|--------------------------|
| **Arabidopsis plate grown** |                          |                             |                          |                          |                          |                          |
| VTE3/VTE3                  | 210 ± 14                 | 107 ± 6                     | 95 ± 4                   | 3 ± 1                    | 9 ± 1                    | 0                        |
|                            |                          |                             | (89%)                    | (3%)                     | (8%)                     | (0%)                     |
| vte3-1/vte3-1              | 173 ± 9*                 | 122 ± 8**                   | 40 ± 2**                 | 51 ± 2**                 | 3 ± 1**                  | 27 ± 3**                 |
|                            | (33%)                    | (42%)                       | (3%)                     | (22%)                    |                          |                          |
| vte3-2/vte3-2              | 0**                      | 48 ± 5**                    | 0**                      | 13 ± 3**                 | 0**                      | 35 ± 2**                 |
|                            |                          |                             | (0%)                     | (28%)                    |                          | (72%)                    |
| **Arabidopsis soil grown**  |                          |                             |                          |                          |                          |                          |
| VTE3/VTE3                  | 67.9 ± 9.9               | 31.5 ± 3.3                  | 29.1 ± 2.7               | 0.5 ± 0.1                | 1.9 ± 0.5                | 0                        |
|                            |                          |                             | (92%)                    | (2%)                     | (6%)                     | (0%)                     |
| vte3-1/vte3-1              | 55.3 ± 5.2*              | 21.5 ± 2.3*                 | 9.7 ± 1.1**              | 8.6 ± 0.1**              | 1.1 ± 0.5*               | 2.1 ± 0.6**              |
|                            |                          |                             | (45%)                    | (40%)                    | (5%)                     | (10%)                    |
| VTE3-2/vte3-2              | 72.7 ± 8.6*              | 28.0 ± 6                    | 24.5 ± 5.2               | 2.2 ± 0.4**              | 1.3 ± 0.4                | 0                        |
|                            |                          |                             | (88%)                    | (8%)                     | (5%)                     | (0%)                     |
| **Synechocystis**           |                          |                             |                          |                          |                          |                          |
| SLL0418                    | 233 ± 20                 | 93.3 ± 7.2                  | 87.5 ± 6                 | 0                        | 5.8 ± 1.2                | 0                        |
|                            |                          |                             | (94%)                    | (0%)                     | (6%)                     | (0%)                     |
| sll0418                    | 170 ± 1*                 | 28.7 ± 3.9**                | 27 ± 3.7**               | 0.7 ± 0.2**              | 1.0 ± 0.2*               | 0                        |
|                            |                          |                             | (90%)                    | (2%)                     | (8%)                     | (0%)                     |

Total lipids extracted from 3-week soil-grown or 18-day-old plate-grown Arabidopsis plants or log-phase *Synechocystis* cell cultures were resolved by normal-phase HPLC, and the quantities of individual tocopherols and PQ were determined relative to standards. Values are expressed as the average of quadruplicate analyses ± SD (in pmol/mg fresh weight). Student’s *t* test was performed, and statistically significant differences relative to respective controls are indicated by single (*P* ≤ 0.05) and double (*P* ≤ 0.01) asterisks. Numbers in parentheses indicate the percentage of an individual tocopherol in the total pool. Values for *Synechocystis* are expressed as pmol·OD${}_{730}^{-1}$·mL${}^{-1}$.

vte3-1 was mapped to an ~2-Mb interval at the bottom of chromosome III using simple sequence length polymorphism markers (Figure 3A). Two simple sequence length polymorphism markers on BACs T15C9 and T17J13 identified 13 and 2 recombination events for vte3-1, respectively, indicating that the VTE3 locus was located to the right of BAC T17J13 (Figure 3A). In the 398-kb interval from T17J13 to the telomere, there are two predicted methyltransferases, At3g63250 and At3g63410. At3g63250 encodes a previously cloned and characterized homocysteine S-methyltransferase (Ranocha et al., 2000). At3g63410 encodes a 338-amino acid (37.9 kD) protein of unknown function that contains conserved S-adenosylmethionine binding motifs and a predicted 58-amino acid chloroplast transit peptide.

To determine whether the At3g63410 locus encodes VTE3, the corresponding wild-type and vte3-1 genes were amplified and sequenced. At3g63410 in vte3-1 contains a C-to-T conversion at nucleotide 281 of the ORF that results in the mutation of Thr-94 to Ile-94 (Figure 3B). A second mutant allele, vte3-2, was identified from the Salk T-DNA insertion population. vte3-2 contains a T-DNA insertion in the first exon of At3g63410 at nucleotide 163 of the ORF (Figure 3B) and is predicted to result in a complete loss of enzyme activity.

Therefore, the vte3-2 allele was maintained as a heterozygote. Progeny from VTE3-2/vte3-2 germinated fully and segregated at a 3:1 ratio (wild type:mutant; χ² = 0.172). Seeds produced from vte3-1/vte3-1 and VTE3-2/vte3-2 were indistinguishable from wild-type seeds in size, shape, and germination rates (data not shown), suggesting that the partial or total loss of VTE3 activity does not affect embryo growth and development. vte3-2 seedlings grown on sterile medium supplemented with 1% sucrose were pale but, unlike soil-grown plants, survived for several weeks and produced sufficient tissue for biochemical analyses. The phenotype of plate-grown vte3-2 seedlings ranged from albino to those that had pale green newly emerging leaves with older leaves that were progressively bleached (Figure 3C). These observations suggest that, unlike the missense vte3-1 point mutation, a null mutation of the At3g63410 locus inhibits photosystem assembly/function and causes severe photobleaching.

### Whole-Plant Phenotypes of vte3-1 and vte3-2

Young vte3-1 seedlings were slightly smaller than wild-type seedlings (Figure 3C) but otherwise were healthy and indistinguishable from the wild type. By contrast, vte3-2 seedlings were pale green and did not survive beyond 7 days in soil.

### Analysis of Tocopherols in Leaves and Seeds of vte3 Mutants

Homozygous vte3-2 mutants are soil lethal. To directly compare the effects of vte3 mutations on tocopherol synthesis, wild-type and mutant plants were grown on media supplemented with 1% sucrose. The tocopherol profile of wild-type leaves consists of α-T, γ-T, and β-T in an ~90:8:2 ratio (Table 1). In vte3-1 leaves, total tocopherol levels were increased slightly but significantly relative to the wild type, and α-T and γ-T levels were decreased to 33 and 3%, whereas β-T and δ-T

In addition to leaves, seeds also accumulate tocopherols. The total tocopherol level of wild-type Arabidopsis seeds was >20-fold higher than that of unstressed soil-grown leaves on a fresh weight basis and was composed of 92% γ-T, 3% α-T, and 5% δ-T (Table 2). Seeds of *vte3-1* mutants showed a 15-fold increase in δ-T content, whereas γ-T levels decreased by 31% relative to the wild type (Table 2). These data indicate that, as in leaf tissue, the majority of MPBQ in *vte3-1* seeds is not methylated by MPBQ/MSBQ MT but instead undergoes cyclization to produce δ-T. β-T was not detected in the mutant because of the low γ-TMT activity in seeds (Shintani and DellaPenna, 1998).

Because homozygous *vte3-2* mutants are seedling lethal in soil, the effects of the mutation on seed tocopherol synthesis could be determined only in seeds from *VTE3-2/vte3-2* plants. Although only 25% of the seeds obtained from *VTE3-2/vte3-2* plants were homozygous for the *vte3-2* locus, there was an increase in δ-T similar to that seen in homozygous *vte3-1* seeds (Table 2). Because *vte3-2* is semidominant in leaf tissue (Table 1), we could not determine whether the δ-T produced is only from *vte3-2/vte3-2* seeds or whether *VTE3-2/vte3-2* seeds also contribute to the phenotype. It also is important that not only was seed tocopherol composition altered by the *vte3-1* and *vte3-2* mutations but the levels of total tocopherols in both mutant alleles were increased significantly (*P* ≤ 0.01), 38 and 30%, respectively, relative to those in the wild type (Table 2). This finding suggests that changes in tocopherol composition caused by altered MPBQ/MSBQ MT activity affect tocopherol levels either as a result of increased flux through the pathway or as decreased turnover of tocopherols in seeds.

**Figure 3. Positional Cloning of VTE3 and Phenotypes of the vte3 Mutants.**

(A) Diagram of chromosome III from ~76 centimorgan (cM) to the telomere (TEL). The number of recombination events is indicated under the markers on BACs T15C9 and T17J13. The *VTE3* locus was delineated further as At3g63410, as described in the text.

(B) Diagram of the *VTE3* genomic clone and the *vte3-1* and *vte3-2* mutations. Black bars represent exons. Thin lines represent untranslated regions and introns. The conversion of Thr-94 to Ile-94 in *vte3-1* is indicated.

(C) Whole-plant phenotypes of plate-grown wild-type and homozygous *vte3-1* and *vte3-2* mutant plants. Col, Columbia wild type. Bars = 2 mm.

were increased to 42 and 22% of total tocopherols, respectively (Table 1). These results indicate that the *vte3-1* mutation leads to a >60% reduction in the methylation of MPBQ, which instead is cyclized to δ-T, the majority of which then is methylated further by γ-TMT to yield β-T (Figure 1).

Leaf tissue from homozygous *vte3-2* mutants accumulated β-T and δ-T only (Table 1), a phenotype consistent with a complete lack of MPBQ/MSBQ MT activity. The absence of α-T and γ-T in *vte3-2* indicates that no redundant MPBQ/MSBQ MT activities exist in Arabidopsis and that the ethyl methane-sulfonate–derived *vte3-1* allele is leaky. The total tocopherol level in pale/bleached *vte3-2* leaf tissue was reduced by >50% relative to the wild type, most likely as a result of the reduced flux through the tocopherol pathway or accelerated tocopherol degradation as a result of the severe photobleaching in *vte3-2*. Unlike *vte3-1*, *vte3-2* was found to be semidominant, because leaves of soil-grown *VTE3-2/vte3-2* plants accumulated significantly increased levels of β-T compared with wild-type leaves (Table 1).

**PQ Synthesis Also Is Disrupted in Homozygous vte3-2 Plants**

The dramatic difference in the whole-plant phenotypes of *vte3-1* and *vte3-2* (Figure 3C), coupled with the fact that plant tocopherol and PQ synthesis are related biochemically (Figure 1), prompted us to determine whether PQ synthesis is affected differentially in these mutants. *vte3-1* leaf tissue showed a small (17%) but significant decrease in PQ relative to the wild type (Table 1) with a trace amount of MSBQ, the immediate biosynthetic precursor of PQ and substrate for MPBQ/MSBQ MT (Figure 4). Leaves of *vte3-2* plants completely lacked PQ and accumulated high levels MSBQ (Table 1, Figure 4). The small peak at 12.10 min in the *vte3-2* HPLC trace has a spectrum unlike that of any PQ pathway intermediates and is an unrelated compound of low abundance that migrates as a small shoulder of the large PQ peak in the wild type (Figure 4, inset).

The combined data from *vte3-1* and *vte3-2* provide genetic evidence that *VTE3* is involved in both tocopherol and PQ synthesis in vivo. The severe photobleaching phenotype of *vte3-2* is caused by the PQ deficiency and/or accumulation of MSBQ, rather than by altered tocopherol composition. This finding is in agreement with studies of tocopherol cyclase mutants of Arabidopsis that lack tocopherols and accumulate the biosynthetic intermediate DMPBQ but are phenotypically similar to wild-type plants (Porfirova et al., 2002; Sattler et al., 2003). Finally, the absence of PQ in *vte3-2* indicates that, as with tocopherol synthesis, the MPBQ/MSBQ MT reaction is required for PQ synthesis.

Table 2. Analysis of Seed Tocopherols in the Wild Type and vte3 Mutants

| Genotype       | Total Tocopherol | α-Tocopherol   | γ-Tocopherol    | δ-Tocopherol    |
|----------------|------------------|-----------------|------------------|------------------|
| VTE3/VTE3      | 853 ± 41         | 23 ± 1          | 789 ± 37         | 41 ± 3           |
|                |                  | (3%)            | (92%)            | (5%)             |
| vte3-1/vte3-1  | 1180 ± 12**      | 15 ± 1**        | 541 ± 4**        | 624 ± 7**        |
|                |                  | (1%)            | (46%)            | (53%)            |
| VTE3-2/vte3-2  | 1120 ± 22*       | 16 ± 4*         | 632 ± 6*         | 472 ± 12**       |
|                |                  | (2%)            | (56%)            | (42%)            |

Total lipids extracted from dry seeds were resolved by normal-phase HPLC, and the quantities of individual tocopherols were determined relative to standards. Values are expressed as the average of quadruplicate analyses ± SD (in pmol/mg dry weight). Student's t test was performed relative to the control, and statistically significant differences are indicated by single (P ≤ 0.01) and double (P ≤ 0.001) asterisks. Numbers in parentheses indicate the percentage of individual tocopherols in the total pool.

copherol synthesis, Arabidopsis does not contain other functionally redundant activities that catalyze the methylation of MSBQ to form PQ. VTE3-2/vte3-2 plants accumulated PQ at a level similar to wild-type plants (Table 1) with no detectable MSBQ (data not shown), indicating that, unlike the vte3-2 tocopherol phenotype, vte3-2 is recessive for the PQ phenotype. While this article was under review, a Ds-tagged VTE3 allele was reported from a large-scale screen for apg (albino or pale green) mutants in Arabidopsis (Motohashi et al., 2003). Although tocopherols were not analyzed in that study, the Ds-tagged VTE3 allele also was found to be deficient in PQ (Motohashi et al., 2003).

VTE3 Uses Both MPBQ and MSBQ as Substrates in Vitro

Genetic evidence strongly suggests that VTE3 encodes both MPBQ MT and MSBQ MT activities. To determine the activity of VTE3 against various potential substrates, the full-length VTE3 protein-coding region was amplified from an Arabidopsis EST clone and engineered for expression in E. coli. Activity assays against various substrates demonstrated that, like Synechocystis sp PCC6803 MPBQ/MSBQ MT, VTE3 uses both MPBQ and MSBQ, intermediates in tocopherol and PQ synthesis, respectively, as methylation substrates in vitro (Figure 5). Neither Synechocystis sp PCC6803 MPBQ/MSBQ MT (Shintani

![Figure 4](#)

Figure 4. HPLC Analysis of PQ and Its Precursor MSBQ in Leaves of the Wild Type and Homozygous vte3-1 and vte3-2 Mutants.

Total lipids extracted from leaves of plate-grown wild-type and vte3 mutant plants were subjected to normal-phase HPLC analysis. The inset shows spectra of the three indicated compounds in the wild type and vte3-2. Peak 1, plastoquinone-9 (PQ); peak 2, unknown peak at 12.10 min; peak 3, MSBQ.

and MSBQ is not detectable (data not shown). These observations are highly significant because they demonstrate the presence of one or more partially redundant MPBQ/MSBQ MT activities in *Synechocystis* sp PCC 6803 or the presence of an alternative biosynthetic route(s) to α-T and PQ. The homozygous *sll0418::aphlI* mutant phenotype is in sharp contrast to the corresponding homozygous Arabidopsis *vte3-2* mutant, in which both α-T and PQ synthesis are fully disrupted.

VTE3 and *Synechocystis* sp PCC 6803 MPBQ/MSBQ MT Are Highly Divergent in Primary Sequence

Although VTE3 and *Synechocystis* sp PCC 6803 MPBQ/MSBQ MT have conserved enzymatic activities (Figure 5), the two proteins are highly divergent in their primary amino acid sequences, with an overall identity of 18% (Table 3). The primary regions of significant similarity are in the S-adenosylmethionine binding motifs I, II, and III, domains involved in the binding of the methyl donor (Figure 6) (Kagan and Clarke, 1994; Joshi and Chiang, 1998). In addition to a low overall similarity, *Synechocystis* sp PCC 6803 MPBQ/MSBQ MT and VTE3 have numerous gaps in their alignment (Figure 6). Thus, the VTE3- and *Synechocystis* sp PCC 6803-type MPBQ/MSBQ MTs appear to represent nonorthologous, functionally equivalent activities that have evolved independently in the two organisms.

To gain insight into the prevalence of *Synechocystis* sp PCC 6803– and VTE3-type MPBQ/MSBQ MTs in nature and their possible evolutionary origins, we expanded our database searches beyond vascular plants and oxygenic cyanobacteria. Orthologs of *Synechocystis* sp PCC 6803 MPBQ/MSBQ MT are present in all oxygenic cyanobacterial genome databases but absent from databases representing nonoxygenic phototrophs, nonphotosynthetic eukaryotes, eubacteria, and most photosynthetic eukaryotes, with the exception of *C. reinhardtii* and *T. pseudonana*. The *C. reinhardtii* and *T. pseudonana* proteins

Table 3. Pair-Wise Comparison of Cyanobacteria- and VTE3-Type MPBQ/MSBQ MTs

| Species | S.sp | C.r.(I) | T.p. | H.sp | A.fu. | C.r.(II) | S.o. | O.s. |
|---------|------|---------|------|------|-------|----------|------|------|
| A.t.    | 18/27 | 18/28   | 18/28 | 39/55 | 40/56 | 70/79   | 85/93 | 81/89 |
| O.s.    | 17/27 | 17/29   | 18/28 | 40/56 | 43/60 | 65/75   | 74/83 |      |
| S.o.    | 18/27 | 16/28   | 17/28 | 41/56 | 41/58 |         | 62/71 |      |
| C.r.(II)| 16/31 | 16/28   | 16/27 | 42/58 | 40/60 |         |       |      |
| A.fu.   | 19/33 | 21/43   | 21/36 | 42/59 |       |         |       |      |
| H.sp.   | 20/31 | 22/36   | 20/33 |       |       |         |       |      |
| T.p.    | 48/66 | 54/67   |       |       |       |         |       |      |
| C.r.(I) | 45/62 |         |       |       |       |         |       |      |

Pair-wise comparisons of predicted mature proteins (as predicted by PSORT) were performed using CLUSTAL W. Values indicate the percentage identity/similarity in the shorter sequence of each pair. A.fu., *A. fulgidus* VTE3-like protein; A.t., Arabidopsis VTE3; C.r. (I), *C. reinhardtii* cyanobacteria-type ortholog; C.r. (II), *C. reinhardtii* VTE3-type ortholog; H.sp, *Halobacterium* sp NRC-1 VTE3-like protein; O.s., *Oryza sativa* VTE3; S.o., *Spinacia oleracea* VTE3; S.sp, *Synechocystis* sp PCC 6803; T.p., *T. pseudonana* Sll0418.

T.p. "SLL0418" 21 FMPTTTQPLSVASLKTNSFTSQQSNASLLPATSSSSTALS MGLKTSILTKLSTTTAKQHP-VATAVTTGAILTTATIKYDKPSRTYSEGSVAREYDAWTQDG 121
C.r. "SLL0418" 60 ARSAAREVYQLS------DAACCSTSAPTASTTFAGVDPTVLAAVAGLGLSLFALKRIDTTPSRKYDNN-VGQEYDAWTEBC 134
S.sp. SLL0418 18 IAAGLYLTARGYQSSDSVANAYDQWTEDC 47
A.t. VTE3 59 RPSAQPRFIQH 58
S.o. VTE3 18 KTPNRLRFGSDFTGSYKLPRLNLPNSR------NLRAKTETTVTKCTLSASERPASQPRFIQN 32
O.s. VTE3 42 PLVLTR----------RGGGGGNIS------VAREVRGVVVVGG-----GEADVAQPRFIQH 49
C.r. VTE3 22 HQRLLALPLSSRPLHRSSVFSSRRIPILQAGSTDTYSRPISAPRLVQH 68
A.fu. "VTE3" 1 MGILED 6
H.sp. "VTE3" 1 MRVQANSR-----------------------SMGVLED 15

T.p. "SLL0418" 122 ILEYYWGEHIHLCYNEEEMEAGYKKKDFIQAKYDFIDEMMKFGGIDATSD-AGAKVLDVGCGFGTTSRYLADKLGPKAEEVTGITISPNOVKRGTEIAMERN 224
C.r. "SLL0418" 135 VLEYYWGEHIHLCYSDEELARGYLKKDEKQAKEFDVDEMLRFSGAKNP------ATIILDVGCGFGTTSRHHLAKKFR-DANVTGITISPQQVKRATELAKEQG 232
S.sp. SLL0418 48 ILEYYWGDIHILCGNGD------PPVAKDFIQSKIDFVHAMAAQWGCGLDTLPP--GTTVIDVGCGTGGSRIILAKDYG--FNVTGITISPQQVKRATELTP--P 142
A.t. VTE3 59 KKEAYWFYRFLSIIVYDH---------VINPGHWTEDMRDDALEPADLSHPDMRVVDVCGGTGFTTLGIVKTVK-AKNVTILDSBHQLAKAKQKE---P 160
S.o. VTE3 33 KQEAFWFYRFLSIIVYDN---------VINPGHWTEDMRDVALEPADLNNRNMIYDVCGGTGFTTLGIIKHVD-PKNVTILDSBHQLAKAKAK----P 167
O.s. VTE3 50 KKEAAYWFYRFLSIIVYDH--------VINPGHWTEDMRDDALEPADLYSRKLRVVDVCGGTGFTTLGVVKTVK-PENVTILDSBHQLEKAREKE---A 173
C.r. VTE3 69 KSEAYWFYAGLSQVYDH---------VINPGHWTEDMRDDALAPAKLDDPNLKVDVCGGTGFTCLGVVKTVK-PENTIMDQSPBQLAКАКАКР----А 159
A.fu. "VTE3" 7 KKRARIFYRFSKIVD---------YTNIFYSDKMRKTVVDMAIDD-AESIVLEVGCGFTTEEIVRRIG-BERVVAVDITPEQMМКАРАК------- 94
H.sp. "VTE3" 16 KARARTFYKMLSKVYD---------RVNTFIWTEEMRAEALSMIDID-AEDRVILDVGCGFTGTEGLIKHT---EEVYGLDQSPBQLTKAWEBFG--K 102

SAM I

T.p. "SLL0418" 225 LPNAKFVVMNALEMDFPDNTFDIVWACESGEHMPDKAEYINEMMRRVLKPGEKFMATWCQRDDRVP----FD-KRDKRDLRFLEYEEWTHPYFISTIEAYKELI 306
C.r. "SLL0418" 233 VGNVKFVMDALAMEFPDNSFDLVWACESGEHMPDKRKYHEEMTRVLKPGETLVIACWCQREEGDKP----FT-PQDKEDLQFLYDEWAHPYFISTIAEFGRLM 269
S.sp. SLL0418 143 DVTAKRAVDDAMALSFPDGSFDVVWSVEAGBHNMPDKAVFAKELLRVVKPGEGLIVVADNQRDDRQVP----LN-FWBKPVNRQLLDQWSHPAFASIЕGFAENL 225
A.t. VTE3 161 LKECKIVEGDAEDLPFTDYADRYVSAGSIEYWPDPEQRGIREAYRVLKIGGKACLICP------VYPTFWLSRFFSDWMLEFPKEEEYEIEWFKNAGFKDVQL 199
S.o. VTE3 168 LKECRIIEGDAEDLPFTDYADRYVSAGSIEYWPDPEQRGIREAYRVLKIGGKACLICP------VYPTFWLSRFFADWMLEFPKEEEYEIEWFQKAGFKDVQL 247
O.s. VTE3 173 LKGVTIMEGDAEDLPFTDTFDARYVSAGSIEYWPDPEQRGIREAYRVLRLIGGVACMICP-----VHPTFWLSRFFADMMLFPKEEEYEIEWFKKAGFKDVKL 229
C.r. VTE3 150 LKGVTILEGDAEDLPFTDTFDARYVSAGSIEYWPDPEQRGIREAYRVVKEGGLACMICP-----VHPTHPVSRFFADAWMLFPTEEEYEIEWFTKAGFTDVKM 235
A.fu. "VTE3" 95 MGGVNYFLGDAENLPFKDNSFDAAILSAGSIEYWPNPQRGIEEMARVTKSGGKVYILAP------RKPDNFAVRKF AESMILFPS TQQCVYWFMKAGLEDIRF 190
H.sp. "VTE3" 103 HDPVARHFDAERLPLPKPDSF DVVWSSGIS EYWPHBVQGLRELRRAKPGSQVLVVCB------DYBPASTVFRKLADAIMLEFYDED EADRMFREACYBEIQH 200

SAM II

T.p. "SLL0418" 306 DATTLSMSN------VKTADWVTPTIASWRHSIWGVYDPMGWIFK-PTKYVKCARDAYCLERMHRAFKRGLMEYGMF AAVKKDDKSS 380
C.r. "SLL0418" 270 NGTCKLDG------VKLEDWNKNTISSWRHSIWGVFDPVVVFKGPRIWYKTVREIVTLERMHQAFEKGLMEYGMMTATKKLAPASESAANTQDRVPVGAKA 366
S.sp. SLL0418 226 EATCLVEGQ------VTTADWTVPTLLPAWLDTLWOCIIREQGWLQYGIRGFISVREVPTILLMRLAFGVGLCRFGMKFAVRKNATQA 301
A.t. VTE3 200 KRIGPKWYRGVRRHGLIMGCSVTVGVPASGDSPQLQGPKKEDEVEKPV-NNPFSLGRFLLGTLAAAWFVLIPIYMWIKDQIVPKDQPI 280
S.o. VTE3 250 KRIGPKWYRGVRRHGLIMGCSVTVGVPASGDSPQLQGPKVEDVQKPVH--PLVFLYRFLLGALASTYYVLPIYMWIKDKIFPKGMLP 327
O.s. VTE3 230 KRIGPKWYRGVRRHGLIMGCSVTVGVPASGDSPQLQGPKVEDVSKPVN--PITFLRFLMGTICAYYYVLPIYMWIKDQRLPKDQPI 309
C.r. VTE3 236 TRIGPKWYRGVRRHGLIMGCSVTVGVPKAGDSPLVMGPKAEVSGKMNTN-PLSFLNLNILGTAAGYYYFCLPIYMYYKNLIWPKNWEM 316
A.fu. "VTE3" 185 VETEP--YRFWSKLVVVISGTVIP 205
H.sp. "VTE3" 201 RVLRK------E---PNSPPAIVTVARAPAAD 218

SAM III

Figure 6. Alignment of Cyanobacteria- and VTE3-Type MPBQ/MSBQ MTs.

The transit peptide or signal sequence (as appropriate) was predicted by PSORT for each protein and removed before alignment using CLUSTAL W. The locations of three S-adenosylmethionine (SAM I to SAM III) binding domains are indicated by solid bars below the alignment. The asterisk above the alignment indicates the position of Thr-94 in Arabidopsis VTE3, which is mutated to Ile-94 in vte3-1. Dashed lines indicate gaps in the alignment. Arrowheads indicate the positions of conserved intron:exon junctions in the Arabidopsis and rice VTE3 genes. Background highlighting is as follows: black, identical in at least seven sequences; red, identical in both archael VTE3-like proteins and, where indicated, also in other sequences; blue, identical in all cyanobacteria-type MPBQ/MSBQ MT sequences and, where indicated, also in other sequences; yellow, identical in cyanobacteria-type and archael VTE3-like enzymes only. Red lettering indicates residues identical in at least three eukaryotic VTE3-type MPBQ/MSBQ MT orthologs and, where indicated, also in other sequences. A.fu. “VTE3,” A. fulgidus VTE3-like protein; A.t. VTE3, Arabidopsis VTE3; C.r. “Sll0418,” C. reinhardtii ortholog of Synechocystis sp PCC6803 Sll0418; C.r. VTE3, C. reinhardtii VTE3; H.sp. “VTE3,” Halobacterium sp NRC-1 VTE3-like protein; O.s. VTE3, Oryza sativa VTE3; S.o. VTE3, Spinacia oleracea VTE3; S.sp. Sll0418, Synechocystis sp PCC6803 MPBQ/MSBQ MT; T.p. “Sll0418,” Thalassiosira pseudonana ortholog of Synechocystis sp PCC6803 Sll0418.

share 45 and 48% identity, respectively, with Synechocystis sp PCC6803 Sll0418 (Table 3, Figure 6). The full-length C. reinhardtii MPBQ/MSBQ MT coding region was amplified from a cDNA library, sequenced, and engineered for the expression and assay of activity in E. coli. As with VTE3 and Synechocystis sp PCC6803 MPBQ/MSBQ MT, the C. reinhardtii cyanobacteria-type enzyme was found to use both MPBQ and MSBQ (Figure 5) as substrates in vitro but not β-T or δ-T (results not shown). The T. pseudonana protein could not be assayed, because cDNA libraries are not readily available for this organism.

Arabidopsis VTE3 appears to represent a highly conserved gene in vascular plants. The sequences shown in Table 3 and Figure 6 are representative of the numerous orthologs found in vascular plant databases. Furthermore, VTE3 orthologs also were identified in EST databases of two nonvascular plants,

Physcomitrella patens and Marchantia polymorpha, and in the genome and EST databases of the green alga C. reinhardtii, making the latter the only organism that encodes both types of MPBQ/MSBQ MTs. C. reinhardtii VTE3 is 70% identical to Arabidopsis VTE3 (Table 3). Attempts to engineer C. reinhardtii VTE3 for expression and assay of activity in E. coli have been unsuccessful to date. Expanding the search for VTE3 homology beyond the plant kingdom failed to identify VTE3 orthologs in mammalian, fungal, cyanobacterial, or other eubacterial databases. However, apparent orthologs to Arabidopsis VTE3 were identified in two archael species, Archaeoglobus fulgidus and Halobacterium sp NRC-1 (Table 3). These two archael “VTE3-like” proteins are ~40% identical to VTE3 proteins from throughout the plant kingdom (Table 3, Figure 6). However, although this similarity extends throughout the predicted archael protein sequences, both archael VTE3-like proteins lack an
$\sim$60-amino acid C-terminal extension that is highly conserved in plants and corresponds to the third exon of the Arabidopsis and rice VTE3 genes (Figure 6). BLAST searches using the protein sequence encoded by the third exon of Arabidopsis VTE3 as a query indicated that this protein domain is unique to eukaryotic VTE3 sequences.

### DISCUSSION

In this study, we have reported the identification and characterization of the Arabidopsis VTE3 locus, which encodes the nonorthologous functional equivalent of *Synechocystis* sp PCC6803 MPBQ/MSBQ MT, a key enzymatic activity in the synthesis of tocopherols and PQ in photosynthetic organisms. Both the Arabidopsis (VTE3-type) and *Synechocystis* sp PCC6803 (cyanobacterial-type) MPBQ/MSBQ MT enzymes have similar in vitro activities toward the tocopherol and PQ biosynthetic intermediates MPBQ and MSBQ, respectively. Despite the striking similarity in their activities, the corresponding proteins in the two organisms are highly divergent in primary sequence and appear to have arisen independently during the evolution of cyanobacteria and plants. Thus, unlike all other tocopherol pathway enzymes in plants, it was not possible to identify Arabidopsis MPBQ/MSBQ MT based on similarity with its cyanobacterial counterpart; instead, a genetic approach was required.

Two Arabidopsis mutants were identified that had phenotypes consistent with a partial or total disruption of MPBQ/MSBQ MT activity. The ethyl methanesulfonate–derived *vte3-1* allele was used to isolate the corresponding *VTE3* locus by positional cloning. The recessive *vte3-1* mutation substitutes Ile-94 for Thr-94, a residue conserved in all vascular and nonvascular plant VTE3 orthologs (Figure 6 and data not shown). *vte3-1* preferentially impairs the methylation of tocopherol substrates in planta and has little effect on the methylation of MSBQ to PQ. Despite the 58% reduction in $\alpha$-T in *vte3-1* leaves, no significant impact on plant growth and development was observed. *vte3-2* is a null, T-DNA insertion allele that, when homozygous, completely disrupts both MPBQ and MSBQ MT activity in vivo; this results in the absence of $\alpha$-T in leaves and $\gamma$-T in seeds and the accumulation of high levels of $\beta$-T and $\delta$-T in each tissue, respectively. Unlike *vte3-1*, *vte3-2* seedlings also lack PQ, accumulate the immediate precursor MSBQ, and are seedling lethal in soil as a result of severe photobleaching. Interestingly, in leaf tissue, *vte3-2* is semidominant for the tocopherol phenotype but recessive for the PQ phenotype.

The Arabidopsis *vte3* mutants do not appear to be the first such mutations identified in plants. Maize and sunflower mutants have been reported previously that, in retrospect, are consistent with the disruption of VTE3 activity in these organisms. Two allelic *Mutator* transposon-derived, high-chlorophyll fluorescence maize mutants have been reported that were pale green and seedling lethal (Cook and Miles, 1992). These mutants lacked PQ and $\alpha$-T in leaves, accumulated what was presumed to be biosynthetic precursors, and had a phenotype strikingly similar to Arabidopsis *vte3-2*. In sunflower, which predominantly accumulates $\alpha$-T in its seeds, an apparently viable mutant (*tph1*) was identified that accumulated $\beta$-T to as much as 40% of total seed tocopherols (Demurin et al., 1996). *tph1* likely contains a mutation in the sunflower VTE3 ortholog that preferentially affects tocopherol methylation in a manner similar to the Arabidopsis *vte3-1* allele.

Orthologs with high identity to Arabidopsis VTE3 are present in all plant genome and EST databases. Most notably, the spinach VTE3 ortholog had been cloned previously and characterized as an abundant 37-kD chloroplast inner envelope protein (named E37) of unknown function that contained three S-adenosylmethionine binding motifs (Teyssier et al., 1996). A substrate for E37 could not be determined, but the protein was immunologically detectable in both photosynthetic and nonphotosynthetic spinach tissues and in a number of other plant species (Teyssier et al., 1996). Given the facts that the inner envelope is the subcellular site of tocopherol biosynthesis in plants (Soll et al., 1980, 1985) and that spinach E37 is 84% identical to Arabidopsis VTE3 (Table 3), it now seems clear that spinach E37 encodes MPBQ/MSBQ MT. The widespread presence of VTE3 orthologs in plant databases and the isolation of maize and sunflower mutants with phenotypes similar to specific Arabidopsis *vte3* alleles suggest that this reaction in tocopherol and PQ synthesis is catalyzed by an enzyme that is highly conserved in both structure and function in dicots and monocots.

#### New Insights into the Synthesis and Regulation of Tocopherols and PQ

Previous studies have demonstrated that tocopherol biosynthetic enzymes are remarkably conserved in both primary sequence and activity in cyanobacteria and plants (Norris et al., 1998; Shintani and DellaPenna, 1998; Collakova and DellaPenna, 2001; Porfirova et al., 2002; Shintani et al., 2002; Sattler et al., 2003). In this regard, the low sequence identity of MPBQ/MSBQ MT in Arabidopsis and *Synechocystis* sp PCC6803 stands in stark contrast to other pathway enzymes and suggests that although similar, fundamental differences exist in tocopherol and PQ synthesis in cyanobacteria and plants. Several lines of evidence support this hypothesis. The null *vte3-2* mutant lacks $\alpha$-T and PQ and is seedling lethal, indicating that VTE3 is essential for the synthesis of both compounds in Arabidopsis. By contrast, the analogous *Synechocystis* sp PCC6803 *slr0418::aphll* mutant is viable and has only reduced levels of tocopherols and PQ (Table 1). This finding indicates that, unlike Arabidopsis, functional redundancy exists for MPBQ/MSBQ MT in *Synechocystis* sp PCC6803 and that this step in tocopherol and PQ synthesis has diverged significantly in cyanobacteria and plants.

Additional divergence of the two pathways is evident from the phenotypes of HPPD mutants in cyanobacteria and plants. As with VTE3, a null mutation of Arabidopsis HPPD disrupts both tocopherol and PQ synthesis and is seedling lethal (Norris et al., 1998), indicating that the synthesis of both compounds is HGA dependent in plants (Figure 1). By contrast, a null HPPD mutation in *Synechocystis* sp PCC6803 results in viable cells that are tocopherol deficient but contain wild-type levels of PQ, indicating that PQ synthesis is HGA independent in Syn-
echocystis sp PCC6803 (Dahnhardt et al., 2002). These data demonstrate that although the PQ produced in plants and cyanobacteria is chemically identical, the PQ biosynthetic pathways differ in the two organisms, at least at the steps of aromatic head group synthesis and ring methylation. Therefore, the PQ pathway shown in Figure 1 is valid only for photosynthetic eukaryotes, and PQ biosynthesis in cyanobacteria must be reassessed.

Enzymes of the tocopherol pathway often are grouped into two categories: those that contribute to tocopherol flux/accumulation (e.g., HPPD and HPT) and those that define the tocopherol composition of a tissue (e.g., γ-TMT and TC). Consistent with this classification scheme, overexpression of HPPD (Tsegaye et al., 2002) or HPT (Savidge et al., 2002; Collakova and DellaPenna, 2003) increased total seed tocopherol levels by up to 30 and 75%, respectively, whereas γ-TMT overexpression resulted in the conversion of the large pool of γ-T in seeds to α-T without affecting total seed tocopherol levels (Shintani and DellaPenna, 1998). From these previous studies, one would predict that altering VTE3 activity would affect tocopherol composition without positively affecting total tocopherol flux/accumulation. Therefore, it was quite unexpected to observe, in addition to altered tocopherol composition, a highly significant (P ≤ 0.01) 31 to 38% increase in total tocopherol levels in vte3 mutant seeds (Table 2). This phenotype suggests the presence of a previously unknown mechanism regulating tocopherol flux or accumulation in wild-type seeds that is disrupted by the vte3 mutations.

The nature of this regulatory mechanism is unclear and theoretically could occur at any time between gene expression and the modulation of enzyme activities in the pathway, but it likely involves the generation or removal of an effector by the vte3 mutations. Because MPBQ and DMPBQ are undetectable in both the wild type and vte3 mutants, these compounds can be excluded as possible effectors. Likewise, γ-T and α-T also can be excluded because altering the seed γ-T:α-T ratio by 80-fold as a result of γ-TMT overexpression did not affect total seed tocopherol levels (Shintani and DellaPenna, 1998). The most likely candidate effector molecule is δ-T, which is undetectable in wild-type seeds and increased in vte3 seeds. Whether δ-T mediates regulation by affecting the kinetic properties of other pathway enzymes or serves as a signal that results in the altered expression of pathway genes remains to be determined. What is clear from the phenotype of seeds containing vte3 mutations is that the tocopherol composition of a tissue plays a previously unsuspected role in regulating tocopherol flux and/or accumulation.

some 800 million years ago (O’Kelly, 1992; Lemieux et al., 2000). The absence of VTE3 orthologs from eubacteria (including cyanobacteria) and the presence of VTE3-like orthologs in two archeal species suggest that VTE3 was present in a common ancestor of archea and plants but subsequently was lost from most archeal lineages. Cyanobacteria-type MPBQ/MSBQ MT orthologs are present in all cyanobacterial genomes and in the genomes of two unicellular photosynthetic eukaryotes, *C. reinhardtii* and the diatom *T. pseudonana*. *C. reinhardtii* is unique in being the only organism currently known to encode both cyanobacteria- and VTE3-type MPBQ/MSBQ MTs, suggesting that the common ancestor of green algae and plants had both types of enzymes and that plants subsequently lost the cyanobacteria-type enzyme early in their evolution. *T. pseudonana* also is unique in being the only photosynthetic eukaryote known that does not encode a VTE3-type enzyme in its genome. Whether the common ancestor of plants, green algae, and diatoms contained both types of enzymes and VTE3 was lost in the diatom lineage will require the sequencing of additional genomes representing early branches of photosynthetic eukaryote evolution.

In summary, we have isolated and characterized an Arabidopsis MPBQ/MSBQ MT that is the nonorthologous functional equivalent of its cyanobacterial counterpart. The Arabidopsis VTE3 gene encodes a protein and activity in tocopherol and PQ biosynthesis that are highly conserved in photosynthetic eukaryotes. An intriguing question remains regarding when and how this plant enzyme evolved from archea, the third domain of life, whereas all other steps of the plant tocopherol biosynthetic pathway appear to have originated from an endosymbiotic event with a cyanobacterium. Understanding the origin of VTE3 during the evolution of plants will provide new insight into the transfer of genetic information among the three domains of life during the emergence and evolution of the fundamental biological process of photosynthesis.

### METHODS

#### Plant Materials and Growth Conditions

Ethyl methanesulfonate-mutagenized *Arabidopsis thaliana* seeds were obtained from Lehle Seeds (Round Rock, TX). The vte3-2 T-DNA insertion line was identified by searching the SIGNAL database World Wide Web site (http://signal.salk.edu/cgi-bin/tdnaexpress) at the Salk Institute for Biological Studies (La Jolla, CA). Seeds containing the vte3-2 mutation were obtained from the ABRC at Ohio State University (http://www.biosci.ohio-state.edu/plantbio/Facilities/abrc/abrchome.htm). Soil-grown plants were kept at 22°C under a 16-h photoperiod. Tissue culture-grown seedlings were grown at 22/19°C (day/night) with a 12-h photoperiod on 1× Murashige and Skoog (1962) salts (Gibco BRL), pH 5.7, with 1% (w/v) sucrose and 1% (w/v) phytagar.

#### Identification and Map-Based Cloning of vte3-1

Ethyl methanesulfonate-mutagenized M2 seeds were grown in soil in a 96-well format for 3 to 4 weeks. Total lipids were extracted from leaf tissues according to Collakova and DellaPenna (2001) and subjected to HPLC (1100 series; Agilent, Wilmington, DE) on a Spherisorb ODS-2 5-μm, 250- × 4.6-mm reverse-phase column (Column Engineering, On-

tario, CA) using the solvent system described by Sattler et al. (2003). The original *vte3-1* mutant line was backcrossed to wild-type Columbia two times. For mapping purposes, *vte3-1* (in the Columbia background) was crossed to wild-type Landsberg *erecta*, and the resulting F2 population was screened using reverse-phase HPLC for the *vte3-1* phenotype. Simple sequence length polymorphism markers for BAC clones T15C9 and T17J13 were described by Bell and Ecker (1994). DNA was isolated using Plant DnAZOL (Invitrogen, Carlsbad, CA) and used in a 20-μL PCR.

### Analysis of Prenylquinones

Quantitative analyses of tocopherols and plastoquinone (PQ) were performed using normal-phase HPLC. Total lipids were extracted from 65 to 75 mg of tissue culture–grown plants or from 20 mg of dry seeds according to Collakova and DellaPenna (2001). Total lipids were separated on a ReliaSil silica 250- × 4.6-mm normal-phase column (Column Engineering) at 30°C with 1 mL/min hexane and dioxane using the following gradient program: 0 to 15 min, 0.2 to 2% dioxane; 15 to 25 min, 2 to 4% dioxane; 25 to 35 min, dioxane held at 4%; 35 to 40 min, 4 to 0.2% dioxane; and 40 to 60 min, dioxane held at 0.2%. Absorption spectra (190 to 800 nm) and fluorescence signals (excitation at 290 nm, emission at 330 nm) were collected. PQ and tocopherol standard curves were prepared using the same method.

### Expression and Assay of 2-Methyl-6-Phytol-1,4-Benzoquinone/2-Methyl-6-Solanoyl-1,4-Benzoquinone Methyltransferase in *Escherichia coli*

*Synechocystis* sp. PCC6803 2-methyl-6-phytyl-1,4-benzoquinone/2-methyl-6-solanoyl-1,4-benzoquinone methyltransferase (MPBQ/MSBQ MT) was engineered and expressed in *E. coli* as described by Shintani et al. (2002). Arabidopsis VTE3 was amplified by PCR from an Arabidopsis leaf cDNA library using Pwo DNA polymerase (Roche Applied Science, Indianapolis, IN), a forward primer engineered with an Ndel site (underlined) to generate an in-frame ATG (5′-CGGCATATGGGCCTTTTG-ATGCTCAAC-3′), and a reverse primer (5′-CGGTCA-GATGGGTTG-GTCTTTGGG-3′). *Chlamydomonas reinhardtii* MPBQ/MSBQ MT (*Synechocystis* type) was amplified from a *C. reinhardtii* cDNA library (Davies et al., 1996) using Pwo DNA polymerase, a forward primer engineered with an Ndel site (underlined) to generate an in-frame ATG (5′-CATATGCTTGGGCAATCCCTGC-3′), and a reverse primer (5′-GCA-CCCCGCTCCTTACTTCA-3′). The amplified fragments were ligated to the EcoRV site of pBluescript KS(+) (Stratagene). Inserts were excised with Ndel and BamHI, inserted into the pET30A expression vector digested previously with the same enzymes (Novagen, Madison, WI), and transformed into BL21DE3 (Novagen).

Growth, induction, cell harvesting, and extraction of engineered proteins in *E. coli* were performed as described by Shintani et al. (2002). Cells carrying an empty pET30A vector also were induced as a negative control. To determine the optimal conditions for the solubilization and activity of each enzyme, each induced protein was solubilized in 50 mM Tris-Cl, pH 8.0, 5 mM DTT, and 1 mM phenylmethylsulfonyl fluoride containing 1% (v/v) of one of the following detergents: Triton X-100, Tween 20, 3-[(3-cholamidopropyl)dimethylammonio]-1-propanesulfonic acid (CHAPS), and β-dodecyl maltoside. After centrifugation at 12,000 rpm, the supernatant was collected, the protein concentration was measured with the Coomassie Protein Assay Reagent (Pierce, Rockford, IL), and aliquots were stored at −80°C until assay.

Enzyme assays were performed as described by Peddibhotla et al. (2002), except that the incubation time for Arabidopsis VTE3 was extended to 6 to 12 h. Enzymes were assayed at detergent concentrations ranging from 0.01 to 0.1% (v/v) final concentration, and the optimal type

and concentration of detergent for each solubilized protein were determined by comparing the yield of the radiolabeled products. The optimal detergents and final concentrations for the assays of *Synechocystis* sp. PCC6803 MPBQ/MSBQ MT, its ortholog from *C. reinhardtii*, and Arabidopsis VTE3 were 0.1% (v/v) Tween 20, 0.1% (v/v) β-D-dodecyl maltoside, and 0.1% (v/v) CHAPS, respectively. Chemical synthesis and purification of 2-methyl-6-phytylbenzoquinone substrate were as reported by Peddibhotla et al. (2002). 2-Methyl-6-solanoylbenzoquinone substrate was extracted and purified from iris bulbs as described by Henry et al. (1987).

### Phylogenetic Analysis

Sequence alignments were performed with CLUSTAL W (MacVector; Genetics Computer Group, Madison, WI) using the BLOSUM 30 protein matrix (for Figure 2), the identity protein matrix (for Figure 6), and default parameters for gap penalties. The phylogenetic tree was generated using the neighbor-joining method. Ties in the tree are treated randomly, with distances uncorrected and gaps distributed proportionally. Bootstrap measurements were conducted with 1000 iterations.

Upon request, materials integral to the findings presented in this publication will be made available in a timely manner to all investigators on similar terms for noncommercial research purposes. To obtain materials, please contact Dean DellaPenna, dellapen@msu.edu.

### Accession Numbers

Unless indicated otherwise, all gene accession numbers are from National Center for Biotechnology Information. Rat Gly N-MT, S00112; *C. reinhardtii* VTE3, AY333781; rice VTE3 (TIGR), TC105417; spinach VTE3, X56963; *Halobacterium* sp. NRC-1 VTE3-like protein, NP_280804; *A. fulgidus* VTE3-like protein, NP_069348; yeast ERG6, CAA89944; maize C-24 sterol methyltransferase, AAB70886; Arabidopsis γ-TMT (MIPS; www.mips.biochem.mpg.de), NM_105171; Arabidopsis SMT1, NM_121374; *Synechocystis* sp. PCC6803 γ-TMT, ORF Slr0089 (Cyano-base; www.kazusa.or.jp/cyano/cyano.html), BAA10562; *Nostoc punctiforme* γ-TMT, ZP_00110362; *C. reinhardtii* cyanobacteria-type MPBQ/MSBQ MT, AY293576; *Synechocystis* sp. PCC6803 MPBQ/MSBQ MT, ORF Sll0418 (CyanoBase), BAA18485; *T. elongatus*, BAC09278; *Nostoc* sp. PCC7120 cyanobacterial MPBQ/MSBQ MT (Joint Genomic Institute [JGI]; http://genome.jgi-psf.org/draft_microbes/nospu/nospu.home.html), contig651; *Trichodesmium erythraeum*, ZP_00074879; *P. marinus* MED4, ZP_00105453; *Synechococcus* sp. WH8102, ZP_00116290; *P. marinus* strain MIT9313, ZP_00114022; alfalfa COMT, 1FPQA; and *T. pseudonana* cyanobacteria-type MPBQ/MSBQ MT, assembled from JGI draft raw genome data (http://genome.jgi-psf.org/thaps0/thaps0.home.html) using the following sequences: PQL43478.x1, TEU52932.y1, SXZ28761.x1, TEU29767.x1, PQL75058.y1, PQL134681.x1, TEU44005.y1, PQL3077.x2, PQL20677.x1, PQL107604.y1, PQL127568.y1, PQL22068.y1, and PQL117660.y1. The coding region of the *T. pseudonana* consensus sequence was predicted by GENSCAN (http://genes.mit.edu/GENSCAN.html).

### ACKNOWLEDGMENTS

We thank Chris Cook, who assisted us with HPLC. This research was supported by funding to D.D.P. from the National Science Foundation (MCB-0077586) and the Center for New Plant Products at Michigan State University and by a National Institutes of Health (GM-31625) grant to D.A.B.

Received May 15, 2003; accepted July 22, 2003.

REFERENCES

Arango, Y., and Heise, K.P. (1998). Tocopherol synthesis from homogenitate in *Capsicum annuum* L. (yellow pepper) chromoplast membranes: Evidence for tocopherol cyclase. Biochem. J. **336**, 531–533.

Azzi, A., Boscoboinik, D., Marilley, D., Ozer, N.K., Stauble, B., and Tasinato, A. (1995). Vitamin E: A sensor and an information transducer of the cell oxidation state. Am. J. Clin. Nutr. **62**, 1337S–1346S.

Behl, C., and Moosmann, B. (2002). Oxidative nerve cell death in Alzheimer’s disease and stroke: Antioxidants as neuroprotective compounds. Biol. Chem. **383**, 521–536.

Bell, C.J., and Ecker, J.R. (1994). Assignment of 30 microsatellite loci to the linkage map of Arabidopsis. Genomics **19**, 137–144.

Brigelius-Flohe, R., Kelly, F.J., Salonen, J.T., Neuzil, J., Zingg, J.M., and Azzi, A. (2002). The European perspective on vitamin E: Current knowledge and future research. Am. J. Clin. Nutr. **76**, 703–716.

Brigelius-Flohe, R., and Traber, M.G. (1999). Vitamin E: Function and metabolism. FASEB J. **13**, 1145–1155.

Collakova, E., and DellaPenna, D. (2001). Isolation and functional analysis of homogentisate phytyltransferase from *Synechocystis* sp. PCC 6803 and Arabidopsis. Plant Physiol. **127**, 1113–1124.

Collakova, E., and DellaPenna, D. (2003). Homogentisate phytyltransferase activity is limiting for tocopherol biosynthesis in Arabidopsis. Plant Physiol. **131**, 632–642.

Cook, W., and Miles, D. (1992). Nuclear mutations affecting plastoquinone accumulation in maize. Photosynth. Res. **31**, 99–111.

Dahnhardt, D., Falk, J., Appel, J., van der Kooij, T.A., Schulz-Friedrich, R., and Krupinska, K. (2002). The hydroxyphenylpyruvate dioxygenase from *Synechocystis* sp. PCC 6803 is not required for plastoquinone biosynthesis. FEBS Lett. **523**, 177–181.

Davies, J.D., Yildiz, F.H., and Grossman, A. (1996). Sac1, a putative regulator that is critical for survival of *Chlamydomonas reinhardtii* during sulfur deprivation. EMBO J. **15**, 2150–2159.

Demurin, Y., Skoric, D., and Karlovic, D. (1996). Genetic variability of tocopherol composition in sunflower seeds as a basis of breeding for improved oil quality. Plant Breed. **115**, 33–36.

d’Harlingue, A., and Camara, B. (1985). Plastid enzymes of terpenoid biosynthesis: Purification and characterization of γ-tocopherol methyltransferase from *Capsicum* chromoplasts. J. Biol. Chem. **260**, 15200–15203.

Diener, A.C., Li, H., Zhou, W., Whoriskey, W.J., Nes, W.D., and Fink, G.R. (2000). Sterol methyltransferase 1 controls the level of cholesterol in plants. Plant Cell **12**, 853–870.

Grasses, T., Grimm, B., Koroleva, O., and Jahns, P. (2001). Loss of α-tocopherol in tobacco plants with decreased geranylgeranyl reductase activity does not modify photosynthesis in optimal growth conditions but increases sensitivity to high-light stress. Planta **213**, 620–628.

Grau, A., and Ortiz, A. (1998). Dissimilar protection of tocopherol isomers against membrane hydrolysis by phospholipase A2. Chem. Physiol. Lipids **91**, 109–118.

Hanck, A. (1985). The biochemical and physiological role of vitamins A and E and their interactions. Acta Vitaminol. Enzymol. **7**, 5–11.

Henry, A., Powls, R., and Pennock, J.F. (1987). Intermediates of tocopherol biosynthesis in the unicellular alga *Scenedesmus obliquus*: The presence of three isomeric methylphytylbenzoquinones. Biochem. J. **242**, 367–373.

Jialal, I., Devaraj, S., and Kaul, N. (2001). The effect of α-tocopherol on monocyte proatherogenic activity. J. Nutr. **131**, 389S–394S.

Jialal, I., and Fuller, C.J. (1993). Oxidized LDL and antioxidants. Clin. Cardiol. **16**, I6–I9.

Joshi, C., and Chiang, V. (1998). Conserved sequence motifs in plant S-adenosyl-L-methionine-dependent methyltransferases. Plant Mol. Biol. **37**, 663–674.


Kagan, R., and Clarke, S. (1994). Widespread occurrence of three sequence motifs in diverse S-adenosylmethionine-dependent methyltransferases suggests a common structure for these enzymes. Arch. Biochem. Biophys. **310**, 417–427.

Lemieux, C., Otis, C., and Turmel, M. (2000). Ancestral chloroplast genome in *Mesostigma viride* reveals an early branch of green plant evolution. Nature **403**, 649–652.

Lichtenthaler, H.K., Prenzel, U., Douce, R., and Joyard, J. (1981). Localization of prenylquinones in the envelope of spinach chloroplasts. Biochim. Biophys. Acta **641**, 99–105.

Marshall, P.S., Morris, S.R., and Threlfall, D.R. (1985). Biosynthesis of tocopherols: A re-examination of the biosynthesis and metabolism of 2-methyl-6-phytyl-1,4-benzoquinol. Phytochemistry **24**, 1705–1711.

Motohashi, R., Ito, T., Kobayashi, M., Taji, T., Nagata, N., Asami, T., Yoshida, S., Yamaguchi-Shinozaki, K., and Shinozaki, K. (2003). Functional analysis of the 37kDa inner envelope membrane polypeptide in chloroplast biogenesis using a Ds-tagged Arabidopsis pale-green mutant. Plant J. **34**, 719–731.

Murashige, T., and Skoog, F. (1962). A revised medium for rapid growth and bioassays with tobacco tissue culture. Physiol. Plant. **15**, 473–497.

Noctor, G., and Foyer, C. (1998). Ascorbate and glutathione: Keeping active oxygen under control. Annu. Rev. Plant Physiol. Plant Mol. Biol. **49**, 249–279.

Norris, S.R., Barrette, T.R., and DellaPenna, D. (1995). Genetic dissection of carotenoid synthesis in Arabidopsis defines plastoquinone as an essential component of phytoene desaturation. Plant Cell **7**, 2139–2149.

Norris, S.R., Shen, X., and DellaPenna, D. (1998). Complementation of the Arabidopsis *pds1* mutation with the gene encoding p-hydroxyphenylpyruvate dioxygenase. Plant Physiol. **117**, 1317–1323.

O’Kelly, C.J. (1992). Flagellar apparatus architecture and the phylogeny of “green” algae: Chlorophytes, euglenoids, glaucophytes. In *The Cytoskeleton of the Algae*, D. Menzel, ed (Boca Raton, FL: CRC Publishers), pp. 315–345.

Peddibhotla, S., Cheng, Z., DellaPenna, D., and Tepe, J.J. (2002). Efficient two-step synthesis of methylphytylbenzoquinones: Precursor intermediates in the biosynthesis of vitamin E. Tetrahedron Lett. **44**, 237–239.

Porfirova, S., Bergmuller, E., Tropf, S., Lemke, R., and Dormann, P. (2002). Isolation of an Arabidopsis mutant lacking vitamin E and identification of a cyclase essential for all tocopherol biosynthesis. Proc. Natl. Acad. Sci. USA **99**, 12495–12500.

Ranocha, P., Bourgis, F., Ziemak, M.J., Rhodes, D., Gage, D.A., and Hanson, A.D. (2000). Characterization and functional expression of cDNAs encoding methionine-sensitive and -insensitive homocysteine S-methyltransferases from Arabidopsis. J. Biol. Chem. **275**, 15962–15968.

Reverberi, M., Picardo, M., Ricelli, A., Camera, E., Fanelli, C., and Fabbri, A.A. (2001). Oxidative stress, growth factor production and budding in potato tubers during cold storage. Free Radical Res. **35**, 833–841.

Ricciarelli, R., Zingg, J.M., and Azzi, A. (2002). The 80th anniversary of vitamin E: Beyond its antioxidant properties. Biol. Chem. **383**, 457–465.

Sattler, S.E., Cahoon, E.B., Coughlan, S.J., and DellaPenna, D. (2003). Characterization of tocopherol cyclases from higher plants and cyanobacteria: Evolutionary implications for tocopherol synthesis and function. Plant Physiol. **132**, 2184–2195.

Savidge, B., Weiss, J.D., Wong, Y.H., Lassner, M.W., Mitsky, T.A., Shewmaker, C.K., Post-Beittenmiller, D., and Valentin, H.E. (2002). Isolation and characterization of homogentisate phytyltransferase genes from *Synechocystis* sp. PCC 6803 and Arabidopsis. Plant Physiol. **129**, 321–332.

Schledz, M., Seidler, A., Beyer, P., and Neuhaus, G. (2001). A novel phytyltransferase from *Synechocystis* sp. PCC 6803 involved in tocopherol biosynthesis. FEBS Lett. **499**, 15–20.

Shintani, D., and DellaPenna, D. (1998). Elevating the vitamin E content of plants through metabolic engineering. Science **282**, 2098–2100.

Shintani, D.K., Cheng, Z., and DellaPenna, D. (2002). The role of 2-methyl-6-phytylbenzoquinone methyltransferase in determining tocopherol composition in *Synechocystis* sp. PCC6803. FEBS Lett. **511**, 1–5.

Soll, J., Kemmerling, M., and Schultz, G. (1980). Tocopherol and plastoquinone synthesis in spinach chloroplast subfractions. Arch. Biochem. Biophys. **204**, 544–550.

Soll, J., Schultz, G., Joyard, J., Douce, R., and Block, M. (1985). Localization and synthesis of prenylquinones in isolated outer and inner envelope membranes from spinach chloroplasts. Arch. Biochem. Biophys. **238**, 290–299.

Srivastava, S., Phadke, R.S., and Govil, G. (1989). Effect of incorporation of drugs, vitamins and peptides on the structure and dynamics of lipid assemblies. Mol. Cell. Biochem. **91**, 99–109.

Tappel, A.L. (1962). Vitamin E as the biological lipid antioxidant. Vitam. Horm. **20**, 493–510.

Teyssier, E., Block, M.A., Douce, R., and Joyard, J. (1996). Is E37, a major polypeptide of the inner membrane from plastid envelope, an S-adenosyl methionine-dependent methyltransferase? Plant J. **10**, 903–912.

Tsegaye, Y., Shintani, D., and DellaPenna, D. (2002). Over-expression of the enzyme p-hydroxyphenylpyruvate dioxygenase in Arabidopsis and its relation to tocopherol biosynthesis. Plant Physiol. Biochem. **40**, 913–920.

Valk, E.E., and Hornstra, G. (2000). Relationship between vitamin E requirement and polyunsaturated fatty acid intake in man: A review. Int. J. Vitam. Nutr. Res. **70**, 31–42.
