
ELSEVIER

Sulfotransferases in glycosaminoglycan biosynthesis
Marion Kusche-Gullberg* and Lena Kjellén†

Most of the sulfotransferases participating in glycosaminoglycan biosynthesis have now been identified. Their essential role in generating binding sites for proteins interacting with glycosaminoglycans is apparent. These interactions may influence important biological processes such as growth control, signal transduction, cell adhesion and lipid metabolism. Gene targeting in mice as well as studies in *Drosophila melanogaster* and *Caenorhabditis elegans* have shown that dysfunction or lack of glycosaminoglycan sulfotransferases may result in severely disturbed embryonic development.

together with other enzymes in the same biosynthetic pathway, as demonstrated for some of the enzymes (see below). In this review we will briefly describe the sulfotransferases involved in heparan sulfate (HS), chondroitin sulfate/dermatan sulfate (CS/DS) and keratan sulfate (KS) biosynthesis. In most cases, the enzymatic activities and substrate specificities of the sulfotransferases have been determined *in vitro* by quantifying the capacity of recombinant enzymes to transfer 35S-PAPS to various poly- and oligosaccharide substrates.

**Addresses**
Department of Medical Biochemistry and Microbiology, The Biomedical Center, Box 582, SE-751 23, Uppsala, Sweden
*e-mail*: marion.kusche@imbim.uu.se
*e-mail*: lena.kjellen@imbim.uu.se

**Current Opinion in Structural Biology** 2003, **13**:605–611
This review comes from a themed issue on Carbohydrates and glycoconjugates
Edited by Ten Feizi and Barbara Mulloy
0959-440X/$ – see front matter
© 2003 Elsevier Ltd. All rights reserved.
DOI 10.1016/j.sbi.2003.08.002

**Abbreviations**
- CS: chondroitin sulfate
- DS: dermatan sulfate
- GAG: glycosaminoglycan
- Gal: galactose
- GalNAc: N-acetylgalactosamine
- GlcA: glucuronic acid
- GlcNAc: N-acetylglucosamine
- HNK-1: human natural killer cell carbohydrate antigen 1
- HS: heparan sulfate
- IdoA: iduronic acid
- KS: keratan sulfate
- NDST: glucosaminyl N-deacetylase/N-sulfotransferase
- PAPS: 3′phosphoadenosine-5′phosphosulfate

**Biosynthesis of the polysaccharide backbones**

Biosynthesis of the sulfated GAGs starts with the addition of xylose from uridine diphosphate-xylose to selected serine residues in proteoglycan core proteins (this is not applicable to KS; see below). Two galactose (Gal) units and a glucuronic acid (GlcA) residue are then added by three distinct transferases [1*,2]. This tetrasaccharide, referred to as the linkage region, is common to the CS/DS and the HS/heparin GAGs. Both sulfation and phosphorylation of this region may occur (see [3]). In CS/DS biosynthesis an N-acetylgalactosamine (GalNAc) residue is the next addition, after which the CS syntheses add GlcA and GalNAc in alternating sequence [4,5]. In HS biosynthesis, addition of an N-acetylglucosamine (GlcNAc) is the committing step and is followed by alternating addition of GlcA and GlcNAc by the HS polymerases EXT1 and EXT2 (exostosins; see [2]).

**Heparan sulfate N-sulfation**

The modification reactions in HS biosynthesis are initiated with N-deacetylation and subsequent N-sulfation of selected GlcNAc residues (Figure 1a). The two reactions are carried out by a bifunctional enzyme, NDST. N-sulfation is unique to HS and does not occur in CS/DS or in KS. Four mammalian NDSTs have been identified: NDST1 and NDST2 transcripts have a wide tissue distribution both during embryonic development and in the adult animal, whereas NDST3 and NDST4 mRNA are mostly expressed during the embryonic period (see Table 1 and [6*]). As expression of the NDSTs may be largely translationally controlled [7**], it will be important to collect information about protein rather than transcript levels.

An active form of the NDST1 N-sulfotransferase domain has been expressed in *E. coli* and crystallized [8]. The domain is near-spherical and consists of a single α/β fold with a central five-stranded parallel β-sheet and a three-stranded antiparallel β-sheet with an open cleft that can accommodate a hexasaccharide substrate. Using

**Introduction**

Sulfation of glycosaminoglycans (GAGs) takes place in the Golgi compartment, where sulfotransferases add sulfate groups from 3′phosphoadenosine-5′phosphosulfate (PAPS) to the growing polysaccharide chains. The enzymes are, with one exception (heparan sulfate 3-O-sulfotransferase-1; see below), type II transmembrane proteins with a short N-terminal cytoplasmic domain and a Golgi transmembrane domain followed by a stem region and a luminal catalytic domain. Very little is known about the organization of the enzymes within the Golgi compartment, but they are likely to form complexes

Figure 1

(a) Heparan sulfate
![Diagram for Heparan sulfate](image1)

(c) Keratan sulfate
![Diagram for Keratan sulfate](image2)

(b) Dermatan sulfate
Chondroitin sulfate
![Diagram for Dermatan sulfate and Chondroitin sulfate](image3)

GlcA
IdoA
GlcNAc
GalNAc
Gal
Sulfate group

Modification reactions in GAG biosynthesis. (a) In HS biosynthesis the modification reactions are initiated by N-deacetylation/N-sulfation of selected GlcNAc residues. Subsequent modifications occur in N-sulfated regions of the polysaccharide. Note that 6-O-sulfation may occur to GlcNAc residues adjacent to N-sulfated disaccharides. (b) In DS biosynthesis, 4-O-sulfation takes place immediately after epimerization. 2-O-sulfation may then occur. In CS biosynthesis, 6-O-sulfation of GalNac and GalNAc(4S) are carried out by a separate enzymes. (c) During KS biosynthesis, 6-O-sulfation occurs to non-reducing terminal GlcNAc residues before further elongation. Sulfation of Gal residues may occur during or after chain elongation. β1,4Gal-T1, β1,4-galactosyltransferase 1; β1,3GlcNac-T, β1,3-N-acetylglucosaminyltransferase. Modified from [44].

the crystal structure as a template, the sulfotransferase domains of the other NDSTs have been modeled [9]. Interestingly, the pattern of surface charge and shape in the putative substrate-binding cleft differs across the four isoforms, and the pattern is highly conserved between species. This indicates that the four different NDSTs are likely to have different substrate specificities, which has indeed been experimentally verified [9–11]. The N-deacetylation and N-sulfation reactions are not coupled when performed in the test tube. Expression of NDST1-active-site mutants in HEK 293 cells has also demonstrated that N-deacetylation and N-sulfation of a GlcNAc residue can

be performed by separate enzyme molecules [12]. So far, attempts to express and crystallize the N-deacetylase domain have not been successful.

How is the sulfate pattern of the heparan sulfate chain created?

A HS chain has a domain structure in which sulfated domains alternate with regions that are virtually devoid of sulfate groups. As N-sulfation is required for further modification, NDSTs will create the overall design of the HS chain. So far we do not know if the NDSTs work in a processive manner, with the enzyme molecule

Table 1

GAG sulfotransferases.

| Enzyme | Abbreviation | Predominant substrate (target in bold) | Predominant expression patterns* in human (left) and mouse (right) tissues | Refs |
| --- | --- | --- | --- | --- |
| Chondroitin/dermatan sulfate |  |  |  |  |
| Chondroitin 4-O-sulfotransferase-1 | C4ST1 | -GlcA-GalNAc- | Broad (hematopoetic tissues, peripheral blood leukocytes) | [25,26] |
| Chondroitin 4-O-sulfotransferase-2 | C4ST2 | -GlcA-GalNAc- | Broad | [26] |
| Chondroitin 4-O-sulfotransferase-3 | C4ST3 | -GlcA-GalNAc- | Liver | [27] |
| Dermatan 4-O-sulfotransferase-1 | D4ST1 | -IdoA-GalNAc- | Broad (placenta) | [28] |
| Chondroitin 6-O-sulfotransferase† | C6ST (GST0, CS6ST) | -GlcA-GalNAc- | Broad (aorta, testis) | [30,37*,59] |
| Chondroitin 6-O-sulfotransferase-2‡ | C6ST2 (GST5, GlcNAc6ST-4) | -GlcA-GalNAc | Broad (heart, spleen, ovary) | [34] |
| Chondroitin 4-sulfate 6-O-sulfotransferase Galactosaminyl uronyl 2-O-sulfotransferase | GalNAc4S-6ST CS/DS2ST | -GlcA-GalNAc4S- -IdoA-GalNAc±4S | Broad | [38] [40] |
| Keratan sulfate |  |  |  |  |
| Keratan sulfate Gal 6-O-sulfotransferase | KSGal6ST (GST1) | -Gal-GlcNAc- | Brain | [42] |
| GlcNAc 6-O-sulfotransferase | GlcNAc6ST1 (GST2) | GlcNAc-Gal- | Broad | [32,60] |
| GlcNAc 6-O-sulfotransferase | GlcNAc6ST2 | GlcNAc-Gal- (HEC-GlcNAc6ST GST3, LSST) | High endothelial venules | [61,62] |
| Corneal GlcNAc 6-O-sulfotransferase, mouse intestinal GlcNAc 6-O-sulfotransferase§ | GlcNAc6ST-5 (C-GlcNAc6ST, GST4β) | GlcNAc-Gal- | Cornea, spinal cord, trachea | [45,46] |
| Heparan sulfate/heparin |  |  |  |  |
| N-deacetylase/N-sulfotransferase 1 | NDST1 | -GlcA-GlcNAc- | Broad | [9,63] |
| N-deacetylase/N-sulfotransferase 2 | NDST2 | -GlcA-GlcNAc- | Broad | [9,63] |
| N-deacetylase/N-sulfotransferase 3 | NDST3 | -GlcA-GlcNAc- | Adult brain, fetal tissues | [9] |
| N-deacetylase/N-sulfotransferase 4 | NDST4 | -GlcA-GlcNAc- | Adult brain, fetal tissues | [9] |
| Heparan sulfate 2-O-sulfotransferase | 2-OST (HS2ST) | -HexA-GlcNS- | Broad | [13] |
| Heparan sulfate 6-O-sulfotransferase 1 | 6-OST1 (HS6ST-1) | -HexA±2-GlcNS/Ac- | Broad | [15–17] |
| Heparan sulfate 6-O-sulfotransferase 2 | 6-OST2 (HS6ST-2) | -HexA±2S-GlcNS/Ac- | Brain | [15–17] |
| Heparan sulfate 6-O-sulfotransferase 2S# | 6-OST2S (HS6ST-2S) | -HexA±2S-GlcNS/Ac- | Ovary, placenta, fetal kidney | [17] |
| Heparan sulfate 6-O-sulfotransferase 3 | 6-OST3 (HS6ST-3) | -HexA±2S-GlcNS/Ac- | Broad | [15,16] |
| Heparan sulfate 3-O-sulfotransferase 1 | 3-OST1 | -GlcA-GlcNS±6S- | Broad (kidney, brain) | [18] |
| Heparan sulfate 3-O-sulfotransferase 2 | 3-OST2 | -HexA2S-GlcNS- | Brain | [18,64] |
| Heparan sulfate 3-O-sulfotransferase 3A | 3-OST3A | -IdoA2S-GlcNH₂±6S- | Broad (heart, placenta) | [18,64] |
| Heparan sulfate 3-O-sulfotransferase 3B | 3-OST3B | -IdoA2S-GlcNH₂±6S- | Broad (liver, placenta) | [18,64] |
| Heparan sulfate 3-O-sulfotransferase 4 | 3-OST4 | Unknown | Brain | [18] |
| Heparan sulfate 3-O-sulfotransferase 5 | 3-OST5 | -GlcA-GlcNS±6S-, -IdoA2S-GlcNS±6S- | Brain, spinal cord | [19,20] |

*Lower levels of transcript may be present in other tissues as well. Tissues within parenthesis show the highest expression levels. †Has also weak KS Gal 6-O-sulfotransferase activity. ‡This enzyme has also been characterized as a GlcNAc 6-O-sulfotransferase (GlcNAc6ST4) [33,35]. §Mouse intestinal GlcNAc6ST has the same activity as human corneal GlcNAc6ST and is likely to produce mouse corneal KS [45]. #Splice forms of 6-OST2 so far only described in humans.
in all tissues examined [15]. Recently, 6-O-sulfotransferase 2 has been found to occur in two splice variants [17].

For the rarest modification in HS biosynthesis, 3-O-sulfation of GlcNAc residues, at least six related sulfotransferases exist [18–20]. 3-O-sulfotransferase 1 lacks a transmembrane region but resides in the Golgi compartment, probably through interactions with other resident Golgi proteins [18]. Some of the 3-O-sulfotransferases have distinct substrate specificities [18]. It has been suggested that 3-O-sulfation may selectively regulate biological function; so far, two saccharide sequences are known to depend critically on 3-O-sulfation for activity—the 3-O-sulfated antithrombin binding pentasaccharide (see [21]) and the binding site for the gD envelope protein of herpes simplex virus 1 [22]. The 3-O-sulfotransferases show sequence homology with the N-sulfotransferase domain of the NDSTs. Modeling of the substrate-binding site of 3-O-sulfotransferase-1 using the crystal structure of the sulfotransferase domain of NDST1 as a template demonstrated that the cleft appeared to accommodate at least a tetrasaccharide [23]. Furthermore, domain swapping experiments indicate that the substrate specificities are determined by the sulfotransferase domains and not by the adjacent N-terminal regions [24].

### Chondroitin sulfate and dermatan sulfate sulfotransferases

CS chains contain alternating GalNAc and GlcA residues, whereas some of the hexuronic acid units in DS chains are IdoA. The proportion of IdoA-containing disaccharides in DS varies from a few percent to almost 100% in one chain. Similar to HS, the CS/DS chains are composed in some cases of several types of disaccharide units that differ with regard to sulfation pattern. The sulfotransferases involved in the biosynthesis of CS/DS belong to three different families that transfer sulfate groups to C4 of GalNAc residues, to C6 of GalNAc residues, and to C2 of the hexuronic acid moieties (Figure 1b, Table 1).

### Most chondroitin sulfate/dermatan sulfate GalNAc sulfotransferases belong to large enzyme families

Three distinct GalNAc 4-O-sulfotransferases (C4ST1–3; see Table 1 for a summary of the names and abbreviations of all the sulfotransferases referred to here) that catalyze the transfer of sulfate to GalNAc residues in GlcA-rich regions have been cloned and characterized [25–27]. Recently, a DS-specific human GalNAc 4-O-sulfotransferase (D4ST-1) was also identified [28]. GalNAc-4-sulfate addition by D4ST1 occurs immediately after epimerization, and sulfation is believed to prevent back-epimerization of the newly formed IdoA into GlcA (see [3]). If the two reactions are coupled, the two enzymes may physically interact and form a complex to ensure proper IdoA formation.

The C4STs and D4ST are all homologous to the human natural killer cell carbohydrate antigen 1 (HNK-1) sulfotransferase that produces the HNK-1 epitope by transfer of sulfate to C3 of a terminal GlcA residue in the sequence GlcAβ1,3Galβ1,4GlcNAc [29]. In addition to the HNK-1 sulfotransferase and the enzymes involved in CS/DS biosynthesis, two additional 4-sulfotransferases (GalNAc-4ST1 and GalNAc-4ST2), which add sulfate groups to terminal GalNAc residues on N-linked and O-linked oligosaccharides, belong to this family. The similarity between members of this family is emphasized by the finding that truncated forms of GalNAc-4ST1 and GalNAc-4ST2 transfer sulfate to internal GlcA–GalNAc sequences in desulfated chondroitin and dermatan [29].

The sulfotransferases involved in the 6-sulfation of GalNAc residues in GlcA-rich regions, a process which generates chondroitin 6-sulfate, include C6ST (also termed CS6ST, GST0 or CHST; [30–32]) and C6ST-2 (also called GlcNAc6ST-4, or GST5; [33–35]). On the basis of structural and functional similarities, C6ST and C6ST2 have been suggested to be included in the GST (Gal/GlcNAc/GalNAc sulfotransferase) family of enzymes [36]. The members of the GST family are, within their catalytical domains, >30% identical to each other at the amino acid level. To date, seven different members of the GST family have been identified in humans, most of them being GlcNAc-6-sulfotransferases [36].

Mice with a targeted deletion in the C6ST gene show >90% loss of CS 6-sulfation in the spleen [37*]. The disappearance of the highly sulfated structure GlcA(2S)-GalNAc(6S) in the brain and cartilage of the C6ST-deficient mice revealed that this structure was generated by the enzyme. By contrast, the levels of GlcA–GalNAc(4S6S) in the spleen and brain were unaffected, showing that 6-O-sulfation of GalNAc(4S) is catalyzed by a separate enzyme. Whether or not the GalNAc-4-sulfate 6-O-sulfotransferase (GalNAc4S-6ST) that transfers 6-O-sulfate to GalNAc(4S) residues in CS [38] is that enzyme remains to be shown.

A DS-specific 6-O-sulfotransferase (D6ST) has also been identified in bovine serum [39] but has so far not been cloned. This enzyme transfers sulfate to a GalNAc residue flanked by two IdoA units, and may possibly catalyze the formation of the rare IdoA-GalNAc(6S) sequence found in human umbilical-cord DS (see [39]).

#### Only one chondroitin sulfate/dermatan sulfate 2-O-sulfotransferase

Only one hexuronyl CS/DS 2-O-sulfotransferase has been identified; this is CS/DS2ST, which catalyzes the transfer of sulfate groups to both GlcA and IdoA residues [40]. IdoA residues of DS are frequently 2-O-sulfated, generating IdoA(2S)–GalNAc(4S). Some 2-O-sulfation at C2 of
GlcA occurs in CS but then only next to a GalNAc(6S) residue [3].

# How many keratan sulfate sulfotransferases are there?

KS is built up of repeating disaccharide units composed of GlcNAc and Gal residues (-3Galβ1-4GlcNAcβ1-) and is predominantly found in cornea and cartilage. Corneal KS chains are linked to their core proteins via N-linkages and are extended and highly sulfated, whereas cartilage KS chains are shorter and O-linked via serine or threonine to their core proteins. The same sulfotransferases appear to act on both O- and N-linked chains. In corneal KS, most GlcNAc residues and approximately half of the Gal residues are sulfated. GlcNAc-6-O-sulfation occurs to terminal non-reducing-end GlcNAc residues only and is coupled to chain elongation (Figure 1c; reviewed in [41]), whereas sulfation of Gal residues takes place at the polymer level. One enzyme that specifically catalyzes 6-O-sulfation of Gal residues, KSGal6ST (also denoted GST-1), has been identified [42]. This enzyme can add sulfate groups to non-reducing terminal and to internal Gal residues, with a preference for Gal adjacent to a sulfated GlcNAc, as determined *in vitro* [43]. Sulfation of a non-reducing terminal Gal residue appears to block further elongation of KS chains [44].

Several different enzymes catalyzing the sulfation of non-reducing terminal GlcNAc residues have been described (for references see [36]). These enzymes as well as KSGal6ST are structurally related to C6ST and C6ST1 (which are involved in CS biosynthesis; see above) and have also been suggested to be included in the GST family of enzymes [36]. The substrate specificities and the roles of the different GlcNAc6STs in KS biosynthesis have not been completely elucidated. So far, the corneal GlcNAc 6-O-sulfotransferase (also termed GlcNAc6ST5 or GST4β) has been most clearly demonstrated to be a candidate GlcNAc 6-O-sulfotransferase involved in KS biosynthesis [44,45]. Mutations leading to inactivation of this enzyme result in macular corneal dystrophy [46], characterized by opacity of the cornea and eventually progression to blindness. In mouse, the corresponding enzyme is denoted mouse intestinal KSGal6ST.

## Conclusions

Most of the sulfotransferases taking part in GAG biosynthesis have been identified and their substrate specificities (at least *in vitro*) have been, or are being, elucidated. On the structural level much less is known, the sulfotransferase domain of NDST1 being the only crystallized domain [8]. More structural information is available about related cytosolic sulfotransferases, which transfer sulfate groups to small molecular substrates such as steroids, bioamines, therapeutic drugs and xenochemicals [47]. Modeling of the GAG sulfotransferases on these structures may be a fruitful approach.

To understand how specific GAG protein-binding sites are being generated, we need to know more about the organization of the enzymes within the Golgi compartment and how these machineries are regulated. Work in this area has only just started (see [1]).

Gene targeting in mice of GAG sulfotransferases has been useful in elucidating enzyme substrate specificities, as in the case of C6ST (see above [37]). Other mouse models give us more information about the biological roles of GAGs; for example, HS 2-O-sulfotransferase knockout mice that fail to form kidneys and die neonatally (see [14]), and mice lacking NDST1 die at birth as a result of lung failure (see [6]). The more general role of NDST1 in HS biosynthesis, in contrast to the restricted function for NDST2 in heparin biosynthesis in connective-tissue-type mast cells, has been established from studies of NDST1 and NDST2 knockout mice (see [6]). Recently, the 3-O-sulfotransferase-1 knockout mice were described [48]. Since this isoform has been suggested to be responsible for generation of the 3-O-sulfated antithrombin-binding pentasaccharide, the lack of procoagulant phenotype in the 3-O-sulfotransferase-1 knockout mice was surprising. Future work will demonstrate if the newly reported 3-O-sulfotransferases ([19]; see also [48]) are more important in regulating hemostatic balance. More mice models, and cell lines derived from these mice, will be useful tools in studying the GAG sulfotransferases.

GAGs are also present in invertebrates, including *Drosophila melanogaster* and *Caenorhabditis elegans*, where they play critical roles in several different signaling pathways during development [49,50]. *C. elegans* synthesizes non-sulfated chondroitin, whereas *Drosophila* CS chains contain 4-sulfate groups [51,52]. DS has so far not been detected in either of the two species. The structural features of HS isolated from *Drosophila* and *C. elegans* are similar to those found in vertebrates with the polysaccharide containing N- as well as 2- and 6-O-sulfate groups [51,52]. So far no *Drosophila* sulfotransferase(s) responsible for sulfation of CS have been characterized. The only worm HS 6-O-sulfotransferase described, *C. elegans* HST-6, shares ~50% overall amino acid sequence similarity with the three vertebrate forms [53]. In flies, only one NDST, *sulfateless* [54], and one 6-O-sulfotransferase, *dHS6ST* [55], have been described. Two putative HS 2-O-sulfotransferases have also been reported [56,57]. Studies in these model organisms have pushed the field forwards with great speed and will continue to be of great importance.

Lastly, we have been accustomed to think of the generation of specific GAG sequences as a biosynthetic event. The recent demonstration of extracellular endosulfatas [58] has made the story more complex.

Acknowledgements
Our research was supported by grants from The Swedish Research Council, the programme ‘Glycoconjugates in Biological Systems’ sponsored by the Swedish Foundation for Strategic Research, Gustaf V:s 80-årsfond and Polysackaridforskning AB.

References and recommended reading
Papers of particular interest, published within the annual period of review, have been highlighted as:
- of special interest
- of outstanding interest

1. Esko JD, Selleck SB: Order out of chaos: assembly of ligand binding sites in heparan sulfate. Annu Rev Biochem 2002, 71:435-471.
   A comprehensive review on HS, its interactions, biosynthesis and biological functions.

2. Sugahara K, Kitagawa H: Recent advances in the study of the biosynthesis and functions of sulfated glycosaminoglycans. Curr Opin Struct Biol 2000, 10:518-527.

3. Silbert JE, Sugumaran G: Biosynthesis of chondroitin/dermatan sulfate. IUBMB Life 2002, 54:177-186.

4. Kitagawa H, Uyama T, Sugahara K: Molecular cloning and expression of a human chondroitin synthase. J Biol Chem 2001, 276:38721-38726.

5. Yada T, Gotoh M, Sato T, Shionyu M, Go M, Kaseyama H, Iwasaki H, Kikuchi N, Kwon YD, Togayachi A et al.: Chondroitin sulfate synthase 2: molecular cloning and characterization of a novel human glycosyltransferase homologous to chondroitin sulfate glucuronyltransferase, which has dual enzymatic activities. J Biol Chem 2003, 278:30235-30247.

6. Grobe K, Ledin J, Ringvall M, Holmborn K, Forsberg E, Esko JD, Kjellen L: Heparan sulfate and development: differential roles of the N-acetylg glucosamine N-deacetylase/N-sulfotransferase isozymes. Biochim Biophys Acta 2002, 1573:209-215.
   In this review, most papers relevant to N-deacetylase/N-sulfotransferase structure and biology are summarized, including those describing NDST1 and NDST2 knockout mice.

7. Grobe K, Esko JD: Regulated translation of heparan sulfate GlcNAc N-deacetylase/N-sulfotransferase isozymes by structured 5'-untranslated regions and internal ribosome entry sites. J Biol Chem 2002, 277:30699-30706.
   The first demonstration of translational regulation of GAG sulfotransferase expression. This may apply also to other biosynthesis enzymes and is an important area for further experimentation.

8. Kakuta Y, Sueyoshi T, Negishi M, Pedersen LC: Crystal structure of the sulfotransferase domain of human heparan sulfate N-deacetylase/N-sulfotransferase 1. J Biol Chem 1999, 274:10673-10676.

9. Aikawa J, Grobe K, Tsujimoto M, Esko JD: Multiple isozymes of heparan sulfate/heparin GlcNAc N-deacetylase/GlcN N-sulfotransferase. Structure and activity of the fourth member, NDST4. J Biol Chem B 2001, 276:5876-5882.

10. Sandbäck Pikas D, Eriksson I, Kjellén L: Overexpression of different isoforms of glucosaminyl N-deacetylase/N-sulfotransferase results in distinct heparan sulfate N-sulfation patterns. Biochemistry 2000, 39:4552-4558.

11. Van Den Born J, Pikas DS, Pisa BJ, Eriksson I, Kjellén L, Berden JH: Antibody-based assay for N-deacetylase activity of heparan sulfate/heparin N-deacetylase/N-sulfotransferase (NDST): novel characteristics of NDST-1 and -2. Glycobiology 2003, 13:1-10.

12. Bengtsson J, Eriksson I, Kjellén L: Distinct effects on heparan sulfate structure by different active site mutations in NDST-1. Biochemistry 2003, 42:2110-2115.

13. Rong J, Habuchi H, Kimata K, Lindahl U, Kusche-Gullberg M: Substrate specificity of the heparan sulfate hexuronic acid 2-O-sulfotransferase. Biochemistry 2001, 40:5548-5555.

14. Merry CL, Wilson VA: Role of heparan sulfate-2-O-sulfotransferase in the mouse. Biochim Biophys Acta 2002, 1573:319-327.

This review summarizes the role of 2-O-sulfated HS during development and discusses in detail results obtained with the gene-trap mutation of HS 2-O-sulfotransferase.

15. Habuchi H, Tanaka M, Habuchi O, Yoshida K, Suzuki H, Ban K, Kimata K: The occurrence of three isoforms of heparan sulfate 6-O-sulfotransferase having different specificities for hexuronic acid adjacent to the targeted N-sulfoglucosamine. J Biol Chem 2000, 275:2859-2868.

16. Smeds E, Habuchi H, Do A-T, Hjertson E, Grundberg H, Kimata K, Lindahl U, Kusche-Gullberg M: Substrate specificities of mouse heparan sulphate glucosaminyl 6-O-sulphotransferases. Biochem J 2003, 372:371-380.

17. Habuchi H, Miyake G, Nogami K, Kuroiwa A, Matsuda Y, Kusche-Gullberg M, Habuchi O, Tanaka M, Kimata K: Biosynthesis of heparan sulphate with diverse structures and functions: two alternatively spliced forms of human heparan sulphate 6-O-sulphotransferase-2 having different expression patterns and properties. Biochem J 2003, 371:131-142.

18. Shworak NW, Liu J, Petros LM, Zhang L, Kobayashi M, Copeland NG, Jenkins NA, Rosenberg RD: Multiple Isoforms of heparan sulfate D-glucosaminyl 3-O-sulfotransferase. Isolation, characterization, and expression of human cDNAs and identification of distinct genomic loci. J Biol Chem 1999, 274:5170-5184.

19. Xia G, Chen J, Tiwari V, Ju W, Li JP, Malmstrom A, Shukla D, Liu J: Heparan sulfate 3-O-sulfotransferase isoform 5 generates both an antithrombin-binding site and an entry receptor for herpes simplex virus, type 1. J Biol Chem 2002, 277:37912-37919.

20. Mochizuki H, Yoshida K, Gotoh M, Sugioka S, Kikuchi N, Kwon YD, Tawada A, Maeyama K, Inaba N, Hiruma T et al.: Characterization of a heparan sulfate 3-O-sulfotransferase-5, an enzyme synthesizing a tetrasulfated disaccharide. J Biol Chem 2003, 278:26780-26787.

21. Petitou M, Casu B, Lindahl U: 1976–1983, a critical period in the history of heparin: the discovery of the antithrombin binding site. Biochimie 2003, 85:83-89.

22. Shukla D, Liu J, Blaiklock P, Shworak NW, Bai X, Esko JD, Cohen GH, Eisenberg RJ, Rosenberg RD, Spear PG: A novel role for 3-O-sulfated heparan sulfate in herpes simplex virus 1 entry. Cell 1999, 99:13-22.

23. Raman R, Myette J, Venkataraman G, Sasisekharan V, Sasisekharan R: Identification of structural motifs and amino acids within the structure of human heparan sulfate 3-O-sulfotransferase that mediate enzymatic function. Biochem Biophys Res Commun 2002, 290:1214-1219.

24. Yabe T, Shukla D, Spear PG, Rosenberg RD, Seeberger PH, Shworak NW: Portable sulphotransferase domain determines sequence specificity of heparan sulphate 3-O-sulphotransferases. Biochem J 2001, 359:235-241.

25. Yamauchi S, Mita S, Matubara T, Fukuta M, Habuchi H, Kimata K, Habuchi O: Molecular cloning and expression of chondroitin 4-sulfotransferase. J Biol Chem 2000, 275:8975-8981.

26. Hiraoka N, Nakagawa H, Ong E, Akama TO, Fukuda MN, Fukuda M: Molecular cloning and expression of two distinct human chondroitin 4-O-sulfotransferases that belong to the HNK-1 sulfotransferase gene family. J Biol Chem 2000, 275:20188-20196.

27. Kang HG, Evers MR, Xia G, Baenziger JU, Schachner M: Molecular cloning and characterization of chondroitin-4-O-sulfotransferase-3. A novel member of the HNK-1 family of sulfotransferases. J Biol Chem 2002, 277:34766-34772.

28. Evers MR, Xia G, Kang HG, Schachner M, Baenziger JU: Molecular cloning and characterization of a dermatan-specific N-acetylgalactosamine 4-O-sulfotransferase. J Biol Chem 2001, 276:36344-36353.

29. Baenziger JU: Glycoprotein hormone GalNAc-4-sulfotransferase. Biochem Soc Trans 2003, 31:326-330.

30. Fukuta M, Kobayashi Y, Uchimura K, Kimata K, Habuchi O: Molecular cloning and expression of human chondroitin 6-sulfotransferase. Biochim Biophys Acta 1998, 1399:57-61.

Sulfotransferases in glycosaminoglycan biosynthesis Kusche-Gullberg and Kjellén 611

31. Uchimura K, Kadomatsu K, Fan QW, Muramatsu H, Kurosawa N, Kaname T, Yamamura K, Fukuta M, Habuchi O, Muramatsu T: Mouse chondroitin 6-sulfotransferase: molecular cloning, characterization and chromosomal mapping. Glycobiology 1998, 8:489-496.

32. Uchimura K, Muramatsu H, Kadomatsu K, Fan QW, Kurosawa N, Mitsuoka C, Kannagi R, Habuchi O, Muramatsu T: Molecular cloning and characterization of an N-acetylgalactosamine-6-O-sulfotransferase. J Biol Chem 1998, 273:22577-22583.

33. Uchimura K, Fasakhany F, Kadomatsu K, Matsukawa T, Yamakawa T, Kurosawa N, Muramatsu T: Diversity of N-acetylgalactosamine-6-O-sulfotransferases: molecular cloning of a novel enzyme with different distribution and specificities. Biochem Biophys Res Commun 2000, 274:291-296.

34. Kitagawa H, Fujita M, Ito N, Sugahara K: Molecular cloning and expression of a novel chondroitin 6-O-sulfotransferase. J Biol Chem 2000, 275:21075-21080.

35. Bhakta S, Bartes A, Bowman KG, Kao WM, Polsky I, Lee JK, Cook BN, Bruehl RE, Rosen SD, Bertozzi CR et al.: Sulfation of N-acetylgalactosamine by chondroitin 6-sulfotransferase 2 (GST-5). J Biol Chem 2000, 275:40226-40234.

36. Hemmerich S, Lee JK, Bhakta S, Bistrup A, Ruddle NR, Rosen SD: Chromosomal localization and genomic organization for the galactose/N-acetylgalactosamine/N-acetylglucosamine 6-O-sulfotransferase gene family. Glycobiology 2001, 11:75-87.

37. Uchimura K, Kadomatsu K, Nishimura H, Muramatsu H, Nakamura E, Kurosawa N, Habuchi O, El-Fasakhany FM, Yoshikai Y, Muramatsu T: Functional analysis of the chondroitin 6-sulfotransferase gene in relation to lymphocyte subpopulations, brain development, and oversulfated chondroitin sulfates. J Biol Chem 2002, 277:1443-1450. Knockout mice lacking the C6ST gene are characterized. The results indicate that 6-O-sulfated CS may play a role in the maintenance of naive lymphocytes in the spleen.

38. Ohtake S, Ito Y, Fukuta M, Habuchi O: Human N-acetylgalactosamine 4-sulfate 6-O-sulfotransferase cDNA is related to human B cell recombination activating gene-associated gene. J Biol Chem 2001, 276:43894-43900.

39. Nadanaka S, Fujita M, Sugahara K: Demonstration of a novel sulfotransferase in fetal bovine serum, which transfers sulfate to the C6 position of the GalNAc residue in the sequence iduronic acid α1-3GalNAc β1-4iduronic acid in dermatan sulfate. FEBS Lett 1999, 452:185-189.

40. Kobayashi M, Sugamaran G, Liu J, Shworak NW, Silbert JE, Rosenberg RD: Molecular cloning and characterization of a human uronosyl 2-sulfotransferase that sulfates iduronyl and glucuronyl residues in dermatan/chondroitin sulfate. J Biol Chem 1999, 274:10474-10480.

41. Funderburgh JL: Keratan sulfate biosynthesis. IUBMB Life 2002, 54:187-194.

42. Fukuta M, Inazawa J, Torii T, Tsuzuki K, Shimada E, Habuchi O: Molecular cloning and characterization of human keratan sulfate Gal-6-sulfotransferase. J Biol Chem 1997, 272:32321-32328.

43. Torii T, Fukuta M, Habuchi O: Sulfation of sialyl N-acetyllactosamine oligosaccharides and fetuin oligosaccharides by keratan sulfate Gal-6-sulfotransferase. Glycobiology 2000, 10:203-211.

44. Akama TO, Misra AK, Hindsgaul O, Fukuda MN: Enzymatic synthesis in vitro of the disulfated disaccharide unit of corneal keratan sulfate. J Biol Chem 2002, 277:42505-42513. Using four enzymes—two glycosyltransferases and two sulfotransferases—the authors were able to synthesize KS in vitro.

45. Akama TO, Nakayama J, Nishida K, Hiraoka N, Suzuki M, McAuliffe J, Hindsgaul O, Fukuda M, Fukuda MN: Human corneal GlcNac 6-O-sulfotransferase and mouse intestinal GlcNac 6-O-sulfotransferase both produce keratan sulfate. J Biol Chem 2001, 276:16271-16278.

46. Akama TO, Nishida K, Nakayama J, Watanabe H, Ozaki K, Nakamura T, Dota A, Kawasaki S, Inoue Y, Maeda N et al.: Macular corneal dystrophy type I and type II are caused by distinct

mutations in a new sulphotransferase gene. Nat Genet 2000, 26:237-241.

47. Negishi M, Pedersen LG, Petrochenko E, Shevtsov S, Gorokhov A, Kakuta Y, Pedersen LC: Structure and function of sulfotransferases. Arch Biochem Biophys 2001, 390:149-157.

48. HajMohammadi S, Enjyoji K, Princivalle M, Christi P, Lech M, Beeler D, Rayburn H, Schwartz JJ, Barzegar S, de Agostini Al et al.: Normal levels of anticoagulant heparan sulfate are not essential for normal hemostasis. J Clin Invest 2003, 111:989-999. Mice lacking HS 3-O-sulfotransferase-1 are characterized and shown to lack a procoagulant phenotype.

49. Bulik DA, Robbins PW: The Caenorhabditis elegans sqv genes and functions of proteoglycans in development. Biochim Biophys Acta 2002, 1573:247-257.

50. Nybakken K, Perrimon N: Heparan sulfate proteoglycan modulation of developmental signaling in Drosophila. Biochim Biophys Acta 2002, 1573:280-291.

51. Yamada S, Van Die I, Van den Eijnden DH, Yokota A, Kitagawa H, Sugahara K: Demonstration of glycosaminoglycans in Caenorhabditis elegans. FEBS Lett 1999, 459:327-331.

52. Toyoda H, Kinoshita-Toyoda A, Selleck SB: Structural analysis of glycosaminoglycans in Drosophila and Caenorhabditis elegans and demonstration that tout-velu, a Drosophila gene related to EXT tumor suppressors, affects heparan sulfate in vivo. J Biol Chem 2000, 275:2269-2275.

53. Bülow HE, Berry KL, Topper LH, Peles E, Hobert O: Heparan sulfate proteoglycan-dependent induction of axon branching and axon misrouting by the Kallmann syndrome gene kal-1. Proc Natl Acad Sci USA 2002, 99:6346-6351.

54. Lin X, Perrimon N: Dally cooperates with Drosophila Frizzled 2 to transduce Wingless signalling. Nature 1999, 400:281-284.

55. Kamimura K, Fujise M, Villa F, Izumi S, Habuchi H, Kimata K, Nakato H: Drosophila heparan sulfate 6-O-sulfotransferase (dHS6ST) gene. Structure, expression, and function in the formation of the tracheal system. J Biol Chem 2001, 276:17014-17021.

56. Sen J, Goltz JS, Stevens L, Stein D: Spatially restricted expression of pipe in the Drosophila egg chamber defines embryonic dorsal ventral polarity. Cell 1998, 95:471-481.

57. Kusano A, Staber C, Chan HY, Ganetzky B: Closing the (Ran) GAP on segregation distortion in Drosophila. Bioessays 2003, 25:108-115.

58. Dhoot GK, Gustafsson MK, Ai X, Sun W, Standiford DM, Emerson CP Jr: Regulation of Wnt signaling and embryo patterning by an extracellular sulfatase. Science 2001, 293:1663-1666.

59. Tsutsumi K, Shimakawa H, Kitagawa H, Sugahara K: Functional expression and genomic structure of human chondroitin 6-sulfotransferase. FEBS Lett 1998, 441:235-241.

60. Li X, Tedder TF: CHST1 and CHST2 sulfotransferases expressed by human vascular endothelial cells: cDNA cloning, expression, and chromosomal localization. Genomics 1999, 55:345-347.

61. Uchimura K, El-Fasakhany FM, Hori M, Hemmerich S, Blink SE, Kansas GS, Kanamori A, Kumamoto K, Kannagi R, Muramatsu T: Specificities of N-acetylgalactosamine-6-O-sulfotransferases in relation to L-selectin ligand synthesis and tumor-associated enzyme expression. J Biol Chem 2002, 277:3979-3984.

62. Bistrup A, Bhakta S, Lee JK, Belov YY, Gunn MD, Zuo FR, Huang CC, Kannagi R, Rosen SD, Hemmerich S: Sulfotransferases of two specificities function in the reconstitution of high endothelial cell ligands for L-selectin. J Cell Biol 1999, 145:899-910.

63. Kusche-Gullberg M, Eriksson I, Sandbäck Pikas D, Kjellén L: Identification and expression in mouse of two heparan sulfate glucosaminyl N-deacetylase/N-sulfotransferase genes. J Biol Chem 1998, 273:11902-11907.

64. Liu J, Shworak NW, Sinaiy P, Schwartz JJ, Zhang L, Fritze LMS, Rosenberg RD: Expression of heparan sulfate N-glucosaminyl 3-O-sulfotransferase isoforms reveals novel substrate specificities. J Biol Chem 1999, 274:5185-5192.
