
Kinetic Analysis of the 4-Methylideneimidazole-5-one-Containing Tyrosine Aminomutase in Enediyne Antitumor Antibiotic C-1027 Biosynthesis†

Steven D. Christenson,‡ Weiming Wu,$ M. Ashley Spies,‡ Ben Shen,*∥ and Michael D. Toney*,‡

Department of Chemistry, University of California—Davis, Davis, California 95616, Department of Chemistry and Biochemistry, San Francisco State University, 1600 Holloway Avenue, San Francisco, California 94132-4163, and Division of Pharmaceutical Sciences and Department of Chemistry, University of Wisconsin—Madison, Madison, Wisconsin 53705

Received July 11, 2003; Revised Manuscript Received August 29, 2003

ABSTRACT: The enediyne antitumor antibiotic C-1027 contains an unusual (S)-3-chloro-4,5-dihydroxy-β-phenylalanine moiety, which requires an aminomutase for its biosynthesis. Previously, we established that SgcC4 is an aminomutase that catalyzes the conversion of L-tyrosine to (S)-β-tyrosine and employs 4-methylideneimidazole-5-one (MIO) at its active site [Christenson, S. D., Liu, W., Toney, M. D., and Shen, B. (2003) J. Am. Chem. Soc. 125, 6062–6063]. Here, we present a thorough analysis of the properties of SgcC4. L-Tyrosine is the best substrate among those tested and most likely serves as the *in vivo* precursor for the (S)-3-chloro-4,5-dihydroxy-β-phenylalanine moiety. The presence of MIO in the active site is supported by several lines of evidence. (1) Addition of ATP or divalent metal ions has no effect on its aminomutase activity. (2) SgcC4 has optimal activity at pH ~8.8, similar to the pH optima of MIO-dependent ammonia lyases. (3) SgcC4 is strongly inhibited by sodium borohydride and potassium cyanide, but preincubation with L-tyrosine or 4-hydroxycinnamate largely prevents this inhibition. (4) The difference spectrum between SgcC4 and its S153A mutant shows a positive peak at ~310 nm, indicative of MIO. (5) The S153A mutation lowers $k_{cat}/K_M$ 640-fold. The SgcC4-catalyzed conversion of L-tyrosine to (S)-β-tyrosine proceeds via 4-hydroxycinnamate as an intermediate. The latter also acts as a competitive inhibitor with respect to L-tyrosine and serves as an alternative substrate for the production of β-tyrosine in the presence of an amino source. A full time course for the SgcC4-catalyzed interconversion between L-tyrosine, β-tyrosine, and 4-hydroxycinnamate was measured and analyzed to provide estimates for the rate constants in a minimal mechanism. SgcC4 also exhibits a β-tyrosine racemase activity, but α-tyrosine racemase activity was not detected.

C-1027 is a natural product, isolated from *Streptomyces globisporus*, that has potent antitumor and antimicrobial activities (1). Its biosynthesis is especially interesting because of the complex molecular structure and the potential for engineering novel C-1027 analogues for anticancer drug discovery by combinatorial biosynthesis methods. One fascinating feature is the unique β-amino acid moiety, (S)-3-chloro-4,5-dihydroxy-β-phenylalanine. This nonproteinogenic amino acid is likely to be derived from L-tyrosine. At least three enzymatic activities are required for its formation from L-tyrosine, including an α,β-amino migration step as depicted in Figure 1.

Naturally occurring β-amino acids are relatively rare and fall into two metabolically distinct categories. β-Alanine, β-leucine, and β-lysine all appear as intermediates in primary metabolism. β-Alanine occurs in the biosynthesis of pantotheine, while the latter two appear in the catabolism of their respective proteinogenic precursors (2–4). On the other hand, β-arginine, β-tyrosine, and β-phenylalanine have all been isolated as building blocks in the biosynthesis of secondary metabolites (5–7). In almost all cases, the β-amino acid is formed from an intramolecular α,β-migration of the α-amino group. At least four distinct mechanisms for meeting this chemical challenge have been previously demonstrated.

The first mechanistic class was initially isolated from *Clostridia* in the 1970s. Three such enzymes are D-lysine 5,6-aminomutase (8), D-ornithine 5,4-aminomutase (9), and leucine 2,3-aminomutase (3). An important mechanistic feature shared by all three is a requirement for both adenosylcobalamin and pyridoxal phosphate (3, 10, 11). The generation of a 5′-deoxyadenosyl radical by homolytic cleavage of the carbon–cobalt bond of cobalamin was proposed and subsequently supported by the observation of a Co(II) radical by electron paramagnetic resonance (12, 13).

Lysine 2,3-aminomutase, first isolated from *Clostridium subterminale* SB4 because of its ability to generate (S)-β-

† This work was supported in part by National Institutes of Health Grants CA78747 (to B.S.) and GM54779 (to M.D.T.). S.D.C. was supported in part by NIH Grant T32 GM07377. B.S. is a recipient of NSF CAREER Award (MCB 9733938) and NIH Independent Scientist Award (AI51687).
* To whom correspondence should be addressed. M.D.T.: Department of Chemistry, University of California—Davis, One Shields Avenue, Davis, CA 95616. E-mail: mdtoney@ucdavis.edu. B.S.: Division of Pharmaceutical Sciences, School of Pharmacy, University of Wisconsin—Madison, 777 Highland Ave., Madison, WI 53705. E-mail: bshen@pharmacy.wisc.edu.
‡ University of California—Davis.
$ San Francisco State University.
∥ University of Wisconsin—Madison.

10.1021/bi035223r CCC: $25.00 © 2003 American Chemical Society Published on Web 10/08/2003

Kinetic Analysis of Tyrosine Aminomutase

Biochemistry, Vol. 42, No. 43, 2003 12709

Amino Migration
![Chemical Structure](chemical_structure_1.png)

Chlorination
![Chemical Structure](chemical_structure_2.png)

Hydroxylation
![Chemical Structure](chemical_structure_3.png)

Adenylation
![Chemical Structure](chemical_structure_4.png)

PCP Conjugation
![Chemical Structure](chemical_structure_5.png)

Condensation
![Chemical Structure](chemical_structure_6.png)

C-1027 chromophore

FIGURE 1: Proposed biosynthetic pathway for the (S)-3-chloro-4,5-dihydroxyphenylalanine moiety (boxed) of the C-1027 chromophore.

lysine from L-lysine, is representative of the second mechanistic class (4). It bears little sequence homology to cobalamin-dependent aminomutases. Rather, it belongs to a large, recently characterized family of S-adenosylmethionine-dependent enzymes (14). Lysine 2,3-aminomutase contains an 4Fe-4S iron-sulfur cluster (15) and requires pyridoxal phosphate and S-adenosylmethionine for catalytic activity (4). In contrast to the cobalamin-dependent aminomutases, the initial 5′-deoxyadenosyl radical is generated by interaction with the iron-sulfur cluster (16).

A third mechanistic strategy for amino group shifts uses only pyridoxal phosphate as a cofactor, without the involvement of free radical intermediates. Glutamate 1-semialdehyde aminomutase catalyzes the conversion of glutamate 1-semialdehyde to δ-aminolevulinic acid, the universal precursor for the biosynthesis of heme, chlorophyll, and other tetrapyrroles (17).

The fourth mechanistic class is represented by tyrosine 2,3-aminomutase, isolated from the Edeine A producer Bacillus brevis Vm4 (6). The enzyme activity shows no dependence on S-adenosylmethionine, adenosylcobalamin, or pyridoxal phosphate; however, there is an absolute requirement for ATP, implying that its mechanism is fundamentally different from either the radical-dependent or pyridoxal phosphate-dependent process.

Interestingly, none of the genes identified within the C-1027 gene cluster encodes proteins that are significantly homologous to known aminomutases. On the other hand, sgcC4 encodes a protein that is homologous to several members of a well-described family of ammonia lyases. The homology includes a highly conserved ASG motif required for formation of the 4-methylideneimidazole-5-one (MIO) cofactor (Figure 2). Ammonia lyases catalyze the elimination of ammonia from α-amino acids to yield α,β-unsaturated acids and ammonia as products (18). They are divided into two subfamilies on the basis of substrate specificity for either histidine or phenylalanine (19). It is unprecedented for an MIO-dependent enzyme to act as an aminomutase. However, α,β-elimination of ammonia from an amino acid substrate, the reaction catalyzed by MIO-dependent ammonia lyases, accomplishes the more difficult half of the overall aminomutase transformation. Instead of releasing the resultant ammonia and the α,β-unsaturated acid as free products, Michael addition between them affords the β-amino acid as an end product.

From the structure of the (S)-3-chloro-4,5-dihydroxy-β-phenylalanine moiety found in C-1027, tyrosine appears to be a likely primary precursor. Although some PALs from monocotylic plants are capable of using tyrosine, its occurrence as a preferred ammonia lyase substrate is rare. A single example of a tyrosine specific ammonia lyase has only been reported recently from *Rhodobacter capsulatus*, where it produces *p*-hydroxycinnamic acid, the chromophore of a photoactive yellow protein (20).

Ammonia lyases face the difficult task of removing a nonacidic β-proton for elimination to occur in either an E2- or E1<sub>cb</sub>-type process. The first clue to the mechanism was provided in 1967 when Abeles and co-workers observed enzyme inactivation by several carbonyl reagents such as hydroxylamine and hydrazine. They correctly concluded a catalytically essential electrophile was necessary for activity (19, 21). Later studies proposed dehydroalanine as the essential residue and confirmed its origin as Ser143 through an autocatalytic process (22–24). X-ray analysis of the HAL from *Pseudomonas putida* showed the true electrophile to be 4-methylideneimidazole-5-one (MIO), formed from a cyclization of the conserved Ala142-Ser143-Gly144 motif, that contains dehydroalanine as a component (25).

The role of the catalytic electrophile in promoting β-elimination has been a source of controversy. Hanson and Havar first proposed a mechanism in which the electrophile is attacked by the nucleophilic α-amino group (23). A more satisfying model was described by Retey and co-workers, in which MIO acidifies the β-proton by covalent attachment to the aromatic ring of the substrate (26). An illustration of

---

<sup>1</sup> Abbreviations: MIO, 4-methylideneimidazole-5-one; TAL, tyrosine ammonia lyase; PAL, phenylalanine ammonia lyase; HAL, histidine ammonia lyase; CHES, 2-(cyclohexylamino)ethanesulfonic acid; Tris, tris(hydroxymethyl)aminomethane; DTT, dithiothreitol; EDTA, ethylenediaminetetraacetate; HCA, 4-hydroxycinnamic acid; HEPES, 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid; OPA, o-phthaldehyde; PCP, peptidyl carrier protein.

HAL 8:VGTSGTTAEDVVAAVAR...HGARVELSAAAVEALAAARLIVDALAAKPEPVYGVSTGFGA:64
SgcC4 16:VDGETLTVFAVRVAE...ERATVDVPAESIAKAQKSREIFEGIAEQNIPPIYGVTTGYGE:72
HutH 8:--PGTLTLAQLRAIHA...APVRLQLDASAAAPAIDASVACVEQIIAEDRTAYGINTGFGL:62
PAL1 60:-GGETLTISQVAAISARDGSGVTVELSEAAARGVKASSDWVMDSMNKGTDSSYGVTTGFGA:118

HAL 65:LASRHIGTELRAQLQRNIVRSHAAGM...GP..RVEREVVRALMFLRLKTVASGHTGVRP:119
SgcC4 73:MIYMQVDKSKEVELQTNLVRSHSAGV...GP..LFAEDEARAIIVAARLNTLAKGHSAVRP:127
HutH 63:LASTRIASHDLENLQRSVLVSHAAGI...GA..PLDDDLVRLIMVLKINSLSRGFSGIRR:117
PAL1 119:TSHRR..TKQGGALQKELIRFLNAGIFGNGSDNTLPHSATRAAMLVRINTLLQGYSGIRF:176

HAL 120:EVAQTMADVLNAGITPVVHEYGSLGCSDLAPLSHCALTLMG..EGEAEGPDGTVRPAGE:177
SgcC4 128:IILERLAQYLNEGITPAIPEIGSLGASGDALAPLSHVASTLIG..EGYVL.RDGRPVETAQ:184
HutH 118:KVIDALIALVNAEVYPHIPLKGSVGASGDALAPLAHMSLVLLG..EGKARYK.GQWLSATE:174
PAL1 177:EILEAITKFLNQNITPCLPLRGTTITASGDLVPLSYIAGLLTGRPNSKAVGPTGVILSPEE:236
***

HAL 178:LAAAHGIAP..VELREKEGLALLNGTDGMLGMLVMALADLRNLYTSADITAALSLEALLG:235
SgcC4 185:VLAERGIEP..LELRFKEGLALINGTSGMTGLGSLVVGRALEQAQQAEIVTALLIEAVRG:242
HutH 175:ALAVAGLEP..LTLAKEGLALLNGTQASTAYALRGLFYAEDLYAAAIACGGLSVEAVLG:232
PAL1 237:AFKLAGEGGFFELQPKEGLALVNGTAVGSGMASMVLFEANILAVLAEVMSAIFAEVMQQ:296

HAL 236:TDKVLAPELHAI.RPHPGQGVSADNMSRVLAGSG.LTGGHQD............DAPR.:279
SgcC4 243:STSPFLAEGHDIAIRPHEGQIDTAANMRALMRGSG.LTVEHADLRRRELQKDKEAGKDVQRS:301
HutH 233:SRSPFDARIHE.ARGQRGQIDTAACFRDLLGDSSEVSLSHKNCDK............:276
PAL1 297:KPEFTDHLTHKL.KHHPGQIEAAAIMEHILDGSAYV............KAAQKLHEMDPL:343

HAL 279:...VQDAYSVRCAPQVNGAGRDTLDHAALVAGRELASSVDNPVVL.PDGRVESNGNFHGA:335
SgcC4 302:EIYLQKAYSLRAIPQVVGAVRDTLYHARKLRIELNSANDNPLFF.EGKEIFHGANFHGQ:360
HutH 276:...VQDPYSLRCQPQVMGACLTLRQAAEVLGEANAVSDNPLVFAAEGDVISGGNFHAE:333
PAL1 344:QKPKQDRYALRTSPQWLGPQIEVIRSTKMIEREINSVNDNPLIDVSRNKAIHGGNFQGT:403
*

HAL 336:PWAYVLDFLAIVAADLGSICERRTDRLLDKNRSHGLPPFL.AD.DAGVDSGLMIAQYTQA:393
SgcC4 361:PIAFAMDFVTIALTQLGVLAERQINRVLNRHLSYGLPEFLVSG.DPGLHSGFAGAQYPAT:419
HutH 334:PVAMAADNLALAIAEIGSLSERRISLMMDKHMSQ.LPPFLV.E.NGGVNSGFMAIQVTA:390
PAL1 404:PIGVSMDNTRLAIAAIGKLMFAQFSELVNDFYNNGLPSNLSGGRNPSLDYGFKGAEIAM:463

HAL 394:ALVSEMKRLAVPASADSIPSSAMQEDHVSMGWSAARKLRTAVDNLARIVAVELYAATRAI:453
SgcC4 420:ALVAENRTIG.PASTQSVPSSNGDNQDVVSMGLISARNARRVLSNNNKILAVEYLAAAQAV:478
HutH 391:ALASENKALSHPHSVDSLPTSANQEDHVSMAPAAAGKRLWEMAEENTRGVLAIEWLGACQGL:450
PAL1 464:SYCSELQFLANPVTNHVQSAEQHNQDVNSLGLISSRKTSEAVEILKLMSTTFLVGLCQAI:523

HAL 454:ELRAA.EGLTPAPASEAVVAALRAAGAEGPG...PDRFLAPDLAAADTFVREGRLVAAVE:509
SgcC4 479:DISGRFDGL..SPAAKATYEAVRRL.VPTLG...VDRYMADDIELVADALSRGEFLRAIA:532
HutH 451:DLR...KGLKTSAKLEKARQALRS.EVAHYD...RDRFFAPDIEKAVELLAKGSLTGLLP:503
PAL1 524:DLRHLEENLKSTVKNTVSSVAKRVLTMGVNGELHPSRFCEKDLLRVVDREYIFAYI~~~~:579

FIGURE 2: Amino acid sequence alignment of SgcC4 (GenBank entry AAL06680) with known HALs from *S. griseus* (HAL, GenBank entry JC1172) and *P. putida* (HutH, GenBank entry P21310), as well as the PAL from *P. crispum* (PAL1, GenBank entry P24481). The conserved ASG motif and Arg312 are shown in bold with asterisks under them.

the Retey model is given in Figure 3, using tyrosine as a substrate. A tertiary carbocation forms, increasing the acidity of the β-proton and facilitating its removal by an enzymatic base.

In a preliminary report, we demonstrated that SgcC4 does indeed possess tyrosine aminomutase activity (27). It was further demonstrated that the enzyme employs MIO as an obligatory cofactor, as do the histidine and phenylalanine ammonia lyases. This report extends those findings to provide a full kinetic analysis of this novel enzyme, including cofactor requirements, substrate specificity, kinetic constants, pH dependence, and product stereochemistry. In these last experiments, an unanticipated β-tyrosine racemase activity was observed.

MATERIALS AND METHODS

General. *Escherichia coli* DH5α (28) was used as a general host for cloning. *E. coli* BL21(DE3) (Novagen) was used as a heterologous expression host. pGEM-3zf(+) was

from Promega and pET-28a(+) from Novagen. Kanamycin, L-α-tyrosine, D-α-tyrosine, and HCA were from Fisher Scientific. (R)- and (S)-β-tyrosine were from Peptech Corp. L-3,4-Dihydroxyphenylalanine and o-phthalaldialdehyde were from Sigma. L-3-Chlorotyrosine was synthesized according to a literature procedure (29). Plasmid preparation, PCR purification, and DNA extraction were carried out using commercial kits (Qiagen). T4 DNA ligase and all restriction enzymes were obtained from New England Biolabs. Cloned Pfu polymerase was obtained from Stratagene. DNA manipulations, including heat-shock transformations, were carried out according to standard methods (28). All initial rate kinetic data were fitted directly to the Michaelis-Menten equation using nonlinear regression to obtain steady-state kinetic parameters. This yields kinetic parameters slightly different from those obtained previously from Lineweaver-Burk plots (27). Kaleidagraph 3.0 was used for this purpose.

Cloning of *sgcC4*. The *sgcC4* gene was amplified from pBS1005 (30) by PCR using a 5'-TGA ATT CCA TAT

Kinetic Analysis of Tyrosine Aminomutase

Biochemistry, Vol. 42, No. 43, 2003 12711

$\alpha,\beta$-Unsaturated acid

Ammonia Lyase Activity  
(Dissociation of $\alpha,\beta$-unsaturated acid and ammonia)

Aminomutase Activity  
(Michael addition of ammonia)

$\beta$-Amino Acid

FIGURE 3: Proposed mechanism of the MIO-dependent ammonia lyase activity and its extension to include aminomutase activity through Michael addition of ammonia to the $\alpha,\beta$-unsaturated acid prior to their release.

GGC ATT GAC TCA AGT CGA GAC-3' forward primer (the EcoRI site is underlined and the NdeI site is shown in bold) and a 5'-AAT AAG CTT TCA GCG CAG CTG GAT GTC CGT CTC-3' reverse primer (the HindIII site is shown in bold). The resultant product was cloned into the EcoRI-HindIII sites of pGEM-3zf(+) to yield pBS1023 and sequenced to confirm PCR fidelity. The $sgcC4$ gene was then moved as a 1.7 kb NdeI–HindIII fragment from pBS1023 into the same sites of pET28a to afford pBS1022. The latter resulted in the production of SgcC4 as an N-terminal His$_6$-tagged fusion protein.

**Construction of S153A and S153C Mutants of SgcC4.** The Quikchange site-directed mutagenesis protocol (Stratagene) was used to introduce mutations into putative active site residue Ser153 of SgcC4. This method gives a complete mutant construct by PCR without the need for subcloning. The S153A mutation was introduced with the following primer pair: 5'-CGG GTC ACT CGG GGC TGG CGA CCT GGC TCC-3' and 5'-GGA GCC AGG TCG CCA GCC GCC CCG AGT GAC CCG-3' (with the Ala codon underlined). The S153C mutation was introduced with the following primer pair: 5'-GGA GCC AGG TCG CCG CAC GCC CCG AGT GAC CCG-3' and 5'-CGG GTC ACT CGG GGC GTG CGG CGA CCT GGC TCC-3' (with the Cys codon underlined). Several modifications to the published Quikchange protocol were necessary to generate successful mutations in SgcC4. Successful reaction mixtures consisted of 100 ng of pBS1022 as template DNA, each primer at 5 ng/$\mu$L, 200 $\mu$M dNTPs, 5% (v/v) DMSO, 1× buffer, and 1 unit of cloned Pfu polymerase in a final volume of 100 $\mu$L. The PCR program was as follows: initial denaturing at 95 °C for 5 min, followed by 35 cycles at 95 °C for 30 s, 62 °C for 45 s, and 72 °C for 8 min, and completed by an additional 8 min at 72 °C. Upon completion, 20 units of DpnI was added directly to the PCR mixture and digested at 37 °C for 2 h. Fifty microliters of the digested reaction mixture was cleaned using a Qiagen PCR purification kit and then cyclized in an overnight incubation with 400 units of T4 DNA ligase at 16 °C. Following ligation, the mixture was directly transformed into BL21(DE3) and plated on LB supplemented with 25 $\mu$g/mL kanamycin. The mutant constructs were confirmed by sequencing the full SgcC4 coding region.

**Expression and Purification of SgcC4 and Its S153A and S153C Mutants.** For overproduction of the His$_6$-tagged SgcC4 (pBS1022) and its S153A (pBS1024) and S153C (pBS1025) mutants, 50 mL of LB broth containing 25 $\mu$g/mL kanamycin (in a 250 mL baffled flask) was inoculated with 1 mL of an overnight culture of *E. coli* BL21(DE3)/pBS1022, pBS1024, or pBS1025. This was grown to an OD$_{600}$ of 0.7. Expression was induced by the addition of IPTG to a final concentration of 100 $\mu$M and proceeded overnight at 25 °C and 200 rpm. Cells were harvested and resuspended in 5 mL of lysis buffer [50 mM sodium phosphate, 300 mM NaCl, and 10 mM imidazole (pH 8.0)] containing lysozyme (2 mg/mL) for 1 h at 37 °C, and then the solution was stored overnight at –20 °C. Once thawed, cell suspensions were sonicated, and cell debris was removed by centrifugation. SgcC4 and its S153A and S153C mutants were purified by affinity chromatography using Ni–NTA Superflow resin (Qiagen), following the batch procedure recommended by the manufacturer. Collected elutions were dialyzed against 100 mM Tris-HCl (pH 8.5), 1 mM DTT, and 16% glycerol or against 50 mM CHES (pH 9.0) and 50

12712 Biochemistry, Vol. 42, No. 43, 2003

mM KCl. The dialyzed protein was flash-frozen in liquid nitrogen and stored at $-80^{\circ} \mathrm{C}$. Protein concentrations were determined with the Bradford assay (31). This procedure afforded purified SgcC4 and its S153A and S153C mutants with an average final yield of $\sim 65 \mathrm{mg} / \mathrm{L}$ of culture.

**Spectrophotometric Assay of Ammonia Lyase Activity.** The elimination of ammonia from tyrosine forms HCA. The absorbance spectrum of HCA, when compared to that of tyrosine, has a shoulder between 310 and $350 \mathrm{~nm}$ that allows spectrophotometric monitoring of the reaction. A progress curve of lyase activity was obtained at $310 \mathrm{~nm}$ at $25^{\circ} \mathrm{C}$. The reaction mixture consisted of $0.2 \mathrm{mM} L$-tyrosine, $100 \mathrm{mM}$ Tris-HCl (pH 8.8), and $0.25 \mathrm{mg} / \mathrm{mL}$ protein in a total volume of $1 \mathrm{~mL}$.

**HPLC Assay of Aminomutase Activity.** The $\alpha$- and $\beta$-tyrosines cannot be distinguished by their absorbance spectra. It was therefore necessary to separate the two isomers by HPLC for analysis. Standard reaction mixtures consisted of $1.0-0.5 \mathrm{mM} L$-$\alpha$-tyrosine, $50 \mathrm{mM}$ CHES (pH 9.0), $50 \mathrm{mM} \mathrm{KCl}$, and $0.3-0.7 \mathrm{mg} / \mathrm{mL}$ enzyme. Reactions were initiated by addition of enzyme, mixtures left at $25^{\circ} \mathrm{C}$, and reactions terminated by addition of $\mathrm{HCl}$ to $\mathrm{pH}<3.0$. SgcC4 was removed by filtration using Microcon YM-10 filters (Millipore). The filtrate was returned to pH 9 by the addition of KOH. Modifications to this standard assay procedure were used and are noted in Results.

HPLC analysis of reaction products employed precolumn derivatization of amino acids with OPA. The resulting isoindole improves the resolution of the $\alpha$- and $\beta$-isomers of various amino acids, including tyrosine (32). OPA derivatives of reaction samples were made by combining the neutralized filtrates of SgcC4 reactions with an equal volume of OPA reagent for $1 \mathrm{~min}$ at room temperature. Reactions were terminated by adding $1 \%$ (v/v) glacial acetic acid. Because of degradation, the OPA reagent was prepared fresh daily by adding $4.0 \mathrm{mg}$ of OPA in $50 \mu \mathrm{L}$ of ethanol to $4.5 \mathrm{mL}$ of $0.1 \mathrm{M}$ sodium borate (pH 10.4) containing $15 \mu \mathrm{L}$ of Brij 35 (30%, w/v) and $11 \mu \mathrm{L}$ of mercaptoethanol.

A $20 \mu \mathrm{L}$ aliquot of the derivatized mixture was immediately loaded onto an Alltech Adsorbosphere C-18 column ($5 \mu \mathrm{m}, 150 \mathrm{~mm} \times 4.6 \mathrm{~mm}$). The solvents that were used were $50 \mathrm{mM}$ sodium acetate (pH 5.7) and $5 \%$ THF (A), methanol (B), and acetonitrile (C). The elution gradient was formed as follows: from $90: 10: 0$ to $35: 65: 0$ from 0 to $15 \mathrm{~min}, 35: 65: 0$ from $15$ to $20 \mathrm{~min}$, from $35: 65: 0$ to $0: 50: 50$ from $20$ to $25 \mathrm{~min}, 0: 50: 50$ from $25$ to $30 \mathrm{~min}$, and from $0: 50: 50$ to $90: 10: 0$ from $30$ to $35 \mathrm{~min}$. The flow rate was $1 \mathrm{~mL} / \mathrm{min}$. Solvent delivery and UV detection at $330 \mathrm{~nm}$ were performed with an Agilent 1100 series chromatography system. A Perkin-Elmer 650-15 detector measured fluorescence (excitation at $340 \mathrm{~nm}$ and emission at $455 \mathrm{~nm}$). The OPA derivatives of $\alpha$- and $\beta$-tyrosine had indistinguishable UV absorption and fluorescence properties. The initial rates were fitted to the Michaelis-Menten equation using Kaleidagraph software to obtain the kinetic parameters.

**Stereochemical Analysis of Production of $(R)$- and $(S)$-$\beta$-Tyrosine by Chiral HPLC.** Standard reaction mixtures consisted of $1.0 \mathrm{mM} L$-$\alpha$-tyrosine, $50 \mathrm{mM}$ CHES (pH 9.0), $50 \mathrm{mM} \mathrm{KCl}$, and $0.3-0.7 \mathrm{mg} / \mathrm{mL}$ enzyme. Reactions were initiated by addition of enzyme, mixtures left at $25^{\circ} \mathrm{C}$, and reactions terminated by addition of acetic acid to pH 4. Finally, the enzyme was removed by filtration. A $20 \mu \mathrm{L}$

---

![Figure 4](#fig4)

FIGURE 4: Effect of pH on the initial rate of SgcC4-catalyzed formation of $\beta$-tyrosine from L-$\alpha$-tyrosine.

sample of the acidified filtrate was loaded onto a Chirobiotic-T column ($250 \mathrm{~mm} \times 4.6 \mathrm{~mm}$) from Astec. Compounds were eluted isocratically with methanol, acetic acid, and triethylamine ($100: 0.1: 0.1$) delivered at a flow rate of $1 \mathrm{~mL} / \mathrm{min}$. Tyrosine isomers were detected by UV absorbance at $280 \mathrm{~nm}$.

**pH Dependence of $\beta$-Tyrosine Formation.** Single-time point assays of aminomutase activity were used to determine the pH dependency of SgcC4. HEPES buffer was used at pH 7.0, 7.5, and 8.0, and CHES buffer was used at pH 8.5, 9.0, 9.5, and 10.0. Reaction mixtures included $0.63 \mathrm{mg} / \mathrm{mL}$ SgcC4, $0.5 \mathrm{mM} L$-$\alpha$-tyrosine, $50 \mathrm{mM} \mathrm{KCl}$, and $100 \mathrm{mM}$ buffer in a total volume of $0.5 \mathrm{~mL}$. Reactions were quenched after $60 \mathrm{~min}$ by the addition of glacial acetic acid to pH 3 and mixtures analyzed by HPLC using OPA derivatization as described above.

## RESULTS

### Enzyme Properties, Expression, and Purification. The sgcC4 gene encodes a protein of 539 amino acid residues with a calculated $M_{r}$ of $58,138 \mathrm{Da}$. A high level of sequence homology is shared with the histidine/phenylalanine family of ammonia lyases, such as the HALs from *Streptomyces griseus* (JC1172, $39 \%$ identical and $55 \%$ similar) and *P. putida* (P21310, $34 \%$ identical and $52 \%$ similar) and the PAL from *Pseudomonas crispum* (P24481, $32 \%$ identical and $47 \%$ similar) as shown in Figure 2. Motifs related to binding of adenosylcobalamin, iron-sulfur clusters, or pyridoxal phosphate are absent from the primary sequence.

SgcC4 and its S153A and S153C mutants were overproduced in *E. coli* BL21(DE3) as N-terminal His$_{6}$ fusion proteins from pBS10022, pBS10024, and pBS10025, respectively, in high yields ($\sim 63 \mathrm{mg} / \mathrm{L}$). Approximately $20 \%$ of the overproduced enzymes were soluble, and were purified to near homogeneity with Ni-NTA affinity chromatography. The purified proteins migrate on SDS-PAGE with the expected size of $58.1 \mathrm{kDa}$.

### pH Dependence of Aminomutase Activity. Single-time point assays were carried out as a function of pH. They were quenched after $60 \mathrm{~min}$, and the mixtures were derivatized with OPA and separated by HPLC. A plot of the initial rate versus pH shows the optimal pH to be $\sim 9$ (Figure 4), similar to those of other aromatic amino acid lyases (20, 33). These

Kinetic Analysis of Tyrosine Aminomutase

Biochemistry, Vol. 42, No. 43, 2003 12713

Table 1: Kinetic Parameters for the SgcC4 Aminomutase Activity and Comparison to Those of Other Aminomutases$^a$

| organism          | enzyme | substrate                     | $K_{\mathrm{m}} (\mu \mathrm{M})$ | $k_{\mathrm{cat}} (s^{-1})$ | $k_{\mathrm{cat}}/K_{\mathrm{M}} (\mathrm{M}^{-1} s^{-1})$ | ref   |
|-------------------|--------|-------------------------------|----------------------------------|------------------------------|---------------------------------------------|-------|
| St. globisporus    | TAM    | L-tyrosine                    | 28                               | 0.010                        | 360                                           | this work |
|                   |        |                               | (2)                              | (0.001)                      | (44)                                          |         |
|                   |        | L-3,4-dihydroxyphenylalanine  | 1540                             | 0.0067                       | 4.4                                           | this work |
|                   |        |                               | (140)                            | (0.0003)                     | (0.4)                                         |         |
|                   |        | L-3-chlorotyrosine            | 6070                             | 0.0048                       | 0.79                                          | this work |
|                   |        |                               | (550)                            | (0.0003)                     | (0.09)                                        |         |
| B. brevis          | TAM    | L-tyrosine                    | –                                | 0.0023                       | –                                             | 6       |
| C. subterminale    | LAM    | L-lysine                      | 6600                             | 30                            | 4500                                          | 4       |

$a$ Standard errors from nonlinear regression are given in parentheses.

was the third best. L-3-Chloro-5-hydroxytyrosine was not available for testing as a substrate.

Table 1 presents the kinetic parameters for the amino-mutase reactions of the three substrates that exhibited significant activity. The $k_{\mathrm{cat}}/K_{\mathrm{M}}$ value for L-tyrosine is ~100-fold higher than that of either L-3,4-dihydroxyphenylalanine or L-3-chlorotyrosine. This suggests that L-tyrosine is the natural substrate for SgcC4. Table 2 presents kinetic parameters for the ammonia lyase activity of SgcC4.

Time Courses of Full Aminomutase and Ammonia Lyase Measured by HPLC. OPA derivatization and HPLC separation were used to determine a full time course for the SgcC4-catalyzed reaction. Retention times for authentic standards of $\alpha$-tyrosine, $\beta$-tyrosine, and HCA were found to be 13.7, 14.8, and 8.2 min, respectively (Figure 6A, inset). A comparison of chromatograms for various time points shows that in the initial stages of the reaction tyrosine is rapidly depleted. A majority of the absorbance appears in a new peak that coelutes with authentic $\beta$-tyrosine. This peak reaches a quasi-equilibrium with $\alpha$-tyrosine after 2–3 h, with a maximum of ~60% of the initial substrate concentration showing up as $\beta$-tyrosine (Figure 6A).

HCA formation during this initial period occurs at a much slower rate. Once $\beta$-tyrosine and $\alpha$-tyrosine reach equilibrium, there is an increase in the rate of HCA formation. This observation agrees with spectrophotometric time courses in which HCA formation was monitored at 310 nm (data not shown). HCA formation occurs at the expense of both $\alpha$- and $\beta$-tyrosine.

The time course presented in Figure 6B was analyzed using global data analysis with the program DYNAFIT (36). The model used to describe the SgcC4-catalyzed reaction is shown in Figure 6C. The best-fit rate constants obtained from the global analysis are shown above the corresponding reaction arrows. The predicted concentration profiles for tyrosine, $\beta$-tyrosine, and HCA are shown as solid lines plotted on the data in Figure 6B. The fit to the data is remarkably good. The bimolecular association rate constants have the largest estimated errors, as expected since the substrate concentration was saturating throughout most of the time course. The calculated $k_{\mathrm{cat}}$ value for the tyrosine aminomutase activity is 0.022 s$^{-1}$, in reasonable agreement with the observed value of 0.010 s$^{-1}$. The calculated $k_{\mathrm{cat}}$ for the ammonia lyase activity is 0.004 s$^{-1}$, also in reasonable agreement with the measured value of 0.0012 s$^{-1}$. The fitted rate constants allow one to determine that HCA is released from the intermediate ~4-fold faster than it is formed from $\alpha$-tyrosine. It also shows that $\beta$-tyrosine is produced as the initial major product from $\alpha$-tyrosine due to the rate of formation of the $\beta$-isomer from the HCA intermediate (i.e.,

12714 Biochemistry, Vol. 42, No. 43, 2003 Christenson et al.

Table 2: Kinetic Parameters for the SgcC4 Ammonia Lyase Activity and Comparison to Those of Other Ammonia Lyases$^a$

| organism          | enzyme | substrate         | $K_{\mathrm{m}} (\mu \mathrm{M})$ | $k_{\mathrm{cat}} (s^{-1})$ | $k_{\mathrm{cat}} / K_{\mathrm{M}} (M^{-1} s^{-1})$ | ref |
|--------------------|--------|-------------------|----------------------------------|-------------------------------|-----------------------------------------------|-----|
| St. globisporus    | TAL    | L-tyrosine        | 23                               | $1.2 \times 10^{-3}$         | 50                                              | this work |
|                    |        |                   | (0.9)                            | $(1 \times 10^{-4})$         | (5)                                             |       |
| R. capsulatus      | TAL    | L-tyrosine        | 16                               | 27.7                          | $1.7 \times 10^{6}$                             | 20    |
|                    |        | L-phenylalanine   | 1270                             | 15.1                          | $1.2 \times 10^{5}$                             | 20    |
| P. crispum         | PAL    | L-phenylalanine   | 17                               | 22                            | $1.3 \times 10^{6}$                             | 49    |
|                    |        | L-tyrosine        | 2500                             | 0.3                           | 120                                            | 49    |

$^a$ Standard errors from nonlinear regression are given in parentheses.

---

**FIGURE 6:** (A) HPLC analysis, after OPA derivatization, of the formation of $\beta$-tyrosine and HCA from L-$\alpha$-tyrosine catalyzed by SgcC4. Traces for samples taken after reaction for 0, 90, and 360 min are shown. Authentic standards of the substrate and products are shown in the inset. (B) Time course of the SgcC4-catalyzed conversion of L-$\alpha$-tyrosine ($\square$) to $\beta$-tyrosine ($\triangle$) and HCA ($\bullet$). (C) Mechanism used in the global analysis of the three progress curves presented in panel B, using the program DYNAFIT. The fitted rate constants are given above the appropriate reaction arrows. The predicted curves from the global fit are plotted with the data in panel B.

Michael addition of ammonia) being 14-fold higher than the rate of release of HCA.

**Evidence for MIO in the Active Site.** A defining characteristic of the HAL/PAL family of aromatic ammonia lyases

---

**FIGURE 7:** Inhibition and protection against inhibition of SgcC4 activity. SgcC4 was incubated with either NaBH$_4$ or KCN, both in the presence and in the absence of potential substrates as active site-protecting agents.

is the presence of the 4-methylideneimidazole-5-one prosthetic group, arising from the post-translational modification of a conserved ASG sequence motif (18). Both chemical and genetic alterations of this active site cofactor were performed to confirm its presence in SgcC4.

Previous studies have shown that cyanide and borohydride react with the MIO electrophile and render HAL inactive (24, 26, 37). A 30 min treatment with either KCN or NaBH$_4$ prior to the addition of substrate caused a dramatic loss of SgcC4 activity (Figure 7). The inclusion of HCA or L-tyrosine during incubation with either reagent resulted in partial protection against inactivation. L-Phenylalanine, previously shown to be a poor substrate for SgcC4, provided no similar protection. These results suggest that chemical inactivation is specific to the active site.

The results of site-directed mutation of Ser153 support the conclusion from chemical modification and implicate it as a catalytically important residue. Ser153 was replaced with both alanine and cysteine. The aminomutase activity of the S153A and S153C mutants was determined, and the kinetic parameters are given in Table 3, along with results from similar mutations in the HAL/PAL enzyme family.

The S153A mutation results in a 640-fold decrease in $k_{\mathrm{cat}} / K_{\mathrm{M}}$. This is consistent with the observed decrease with the HAL S143A mutant (1680-fold) (38) and PAL S202A mutant (48-fold) (39, 40). In both HAL and PAL, the replacement of the MIO-forming Ser with Cys gave nearly wild-type (WT) activity. Thus, cysteine is capable of forming MIO through $\beta$-elimination of hydrogen sulfide (39, 41). In contrast, the analogous mutation in SgcC4 showed a 34-fold loss of activity. The $K_{\mathrm{M}}$ values for SgcC4 and its S153C mutant are similar, suggesting similar active site structures. The observed partial loss of activity might arise from

Kinetic Analysis of Tyrosine Aminomutase

Table 3: Kinetic Parameters for SgcC4 and the S153A and S153C Mutants and Comparison to Those of Other MIO-Dependent Enzymes$^a$

| organism       | enzyme   | substrate        | $K_m (\mu M)$ | $k_{cat} (s^{-1})$ | $(k_{cat}/K_M)_{WT}/(k_{cat}/K_M)_{Mut}$ | ref     |
|----------------|----------|------------------|----------------|--------------------|-------------------------------------------|---------|
| St. globisporus | WT       | L-tyrosine       | 28             | 0.010              |                                           | this work |
|                |          |                  | (2)            | (0.002)            |                                           |         |
|                | S153A    |                  | 270            | $1.5 \times 10^{-4}$ | 640                                       | this work |
|                |          |                  | (35)           | $(6.7 \times 10^{-6})$ | (160)                                    |         |
|                | S153C    |                  | 49             | $5.1 \times 10^{-4}$ | 34                                        | this work |
|                |          |                  | (2)            | $(2.6 \times 10^{-5})$ | (2)                                      |         |
| P. putida      | WTHAL    | L-histidine      | 5200           | 22                 |                                           | 38      |
|                | S143A    |                  | 7500           | 0.019              | 1680                                      | 38      |
|                | S143C    |                  | 5400           | 20                 | 1.2                                       | 41      |
| P. crispum     | WTPAL    | L-phenylalanine  | 170            | 0.99               |                                           | 39      |
|                | S202A    |                  | 19             | 0.0023             | 48                                        | 40      |
|                | S202C    |                  | 170            | 0.93               | 1.1                                       | 39      |

$a$ Standard errors from nonlinear regression are given in parentheses.

![Figure 8](https://via.placeholder.com/300x200 "Difference absorbance spectrum between SgcC4 and its S153A mutant. The S153A mutant is incapable of forming the MIO cofactor in the active site. The 313 nm peak in the difference spectrum indicates the presence of MIO in the active site of the SgcC4 enzyme.")

**FIGURE 8:** Difference absorbance spectrum between SgcC4 and its S153A mutant. The S153A mutant is incapable of forming the MIO cofactor in the active site. The 313 nm peak in the difference spectrum indicates the presence of MIO in the active site of the SgcC4 enzyme.

incomplete MIO formation during enzyme production due to thiol oxidation or a slow rate of MIO formation.

The S153A mutant is incapable of forming the MIO cofactor since elimination of water from Ser153 is required for formation of the dehydroalanine component of MIO. The difference spectrum between SgcC4 and the S153A mutant was measured in an effort to identify spectrally the MIO cofactor, which has been shown to have an absorbance maximum at ~310 nm (42). This spectrum is shown in Figure 8. There is a clear peak at ~313 nm, indicative of the MIO in the active site of SgcC4.

Formation of $\alpha$-and $\beta$-Tyrosine from HCA. The mechanistic model for SgcC4 aminomutase activity proposes that enzyme-bound HCA is an intermediate. SgcC4 activity was measured in the presence and absence of HCA with variable initial L-tyrosine concentrations to determine if HCA does indeed bind to the enzyme. Plots of the initial rate versus substrate concentration are shown in Figure 9. The inset shows a double-reciprocal plot that suggests HCA is a competitive inhibitor against tyrosine. The $K_i$ was calculated to be $92 \pm 15 \mu M$.

In the presence of an amino source, HCA also serves as an alternative substrate for production of small amounts of $\beta$-and $\alpha$-tyrosine. In these experiments, L-3-chlorotyrosine was used as an amino source because it was known to serve as a substrate for SgcC4 and could be easily distinguished

![Figure 9](https://via.placeholder.com/300x200 "Inhibition of the SgcC4-catalyzed conversion of L-$\alpha$-tyrosine to $\beta$-tyrosine by HCA. At a concentration of 1 mM, there is clear inhibition of the aminomutase activity. The double-reciprocal plot in the inset shows that the inhibition is competitive with respect to L-$\alpha$-tyrosine.")

**FIGURE 9:** Inhibition of the SgcC4-catalyzed conversion of L-$\alpha$-tyrosine to $\beta$-tyrosine by HCA. At a concentration of 1 mM, there is clear inhibition of the aminomutase activity. The double-reciprocal plot in the inset shows that the inhibition is competitive with respect to L-$\alpha$-tyrosine.

from $\alpha$-and $\beta$-tyrosine, resulting from amino transfer to HCA. In the presence of 5 mM L-3-chlorotyrosine and 10 mM HCA, $27 \mu M$ $\beta$-tyrosine was produced after 24 h.

Two separate pathways might account for this. Free ammonia released from lyase activity with L-3-chlorotyrosine might subsequently rebind in the active site and add to HCA. Alternatively, ammonia removed from L-3-chlorotyrosine could be retained by the enzyme after abortive elimination, and subsequently added to HCA that binds to the enzyme. To distinguish between the two possible alternatives, a similar assay using free ammonia was performed. A reaction mixture containing 10 mM ammonium chloride and 10 mM HCA resulted in the accumulation of $13 \mu M$ $\beta$-tyrosine in a 24 h period. Thus, from half the initial concentration, L-3-chlorotyrosine produced twice as much $\beta$-tyrosine as free ammonia. These results suggest a slow release of ammonia from the active site, and agree with previous studies on HAL that reported ammonia release from the enzyme after release of urocanate (43). These data also suggest that while ammonia can be recruited from solvent, the more physiologically relevant aminomutase mechanism is an intramolecular transfer of the amino group.

Stereochemistry of the SgcC4 Aminomutase Reaction. The stereochemistry of the $\beta$-tyrosine product of SgcC4 was initially investigated by assaying the reverse conversion of either $(R)$-$\beta$-tyrosine or $(S)$-$\beta$-tyrosine to L-$\alpha$-tyrosine, assuming that only one enantiomer of $\beta$-tyrosine would be

12716 Biochemistry, Vol. 42, No. 43, 2003

**DISCUSSION**

The results reported herein establish that SgcC4 from the C-1027 biosynthetic cluster of *St. globisporus* possesses the aminomutase activity required for the biosynthesis of the (S)-3-chloro-4,5-dihydroxy-β-phenylalanine moiety of the C-1027 chromophore. In contrast with other aminomutases, SgcC4 requires no exogenous cofactors for full activity. It employs MIO at the active site in a novel mechanism for achieving aminomutase chemistry.

SgcC4 additionally possesses both ammonia lyase and β-tyrosine racemase activities. The metabolic significance of the latter is as yet unknown. The ammonia lyase activity is a consequence of the evolutionary lineage of SgcC4 given its use of the MIO cofactor for the first half of the aminomutase mechanism (Figure 3). The β-tyrosine racemase activity, on the other hand, may be a consequence of the imperfect evolutionary conversion of an ammonia lyase into an aminomutase.

MIO is the active cofactor in the HAL/PAL family of ammonia lyases. It is formed from a conserved ASG sequence motif. The currently accepted model of MIO action was first presented by Retei and co-workers (26). It postulates the electrophilic attack of the exocyclic methylene carbon of MIO on the aromatic ring of its substrate. This dearomatizes the side chain ring, forming a carbocation at the γ-C that consequently activates the β-proton. Removal of the β-proton leads to elimination of the α-amino group as ammonia (Figure 3).

Several lines of evidence support an MIO-mediated aminomutase mechanism at work in SgcC4. First, there is high level of sequence homology between SgcC4 and MIO-dependent ammonia lyases, and the conserved ASG motif responsible for MIO formation is present in SgcC4 at the appropriate location in the primary sequence (Figure 2). Second, chemical modification of the cofactor (Figure 7) and genetic alteration of the amino acid precursors to the cofactor (S153A and S153C mutations) both resulted in significant loss of activity. Third, the difference spectrum between SgcC4 and the S153A mutant (Figure 8) has an absorbance peak characteristic of the MIO chromophore. Finally, HCA, the product of ammonia lyase chemistry, is an intermediate in the migration of the amino group as evidenced by its accumulation during turnover from either direction, by its tight binding, and by its efficacy as an alternative substrate in the presence of a suitable amino group donor.

SgcC4 extends the known repertoire of MIO-dependent chemistry to include aminomutase activity by retaining ammonia in the active site and transferring it to the β-C of HCA. This reaction presumably proceeds via a simple Michael addition after generation of noncovalently bound HCA in the active site.

Most investigations into aminomutase mechanisms have shown that transfer of the amino group occurs in an intramolecular process (5, 7, 16, 44). The conversion of HCA to β-tyrosine in the presence of free ammonia indicates that intramolecular transfer is preferred but not absolutely required by SgcC4. The relatively high ammonia concentrations required to observe activity with HCA combined with the poor incorporation efficiency weigh in favor of an intramolecular amino group transfer. This is also supported by the observation that the amino group from L-3-chloro-
Kinetic Analysis of Tyrosine Aminomutase

tyrosine could be transferred to exogenously added HCA intermolecularly. This intermolecular exchange proceeds at 4 times the rate of $\beta$-tyrosine production using HCA and ammonia as substrates.

On the basis of sequence analysis of the C-1027 biosynthetic gene cluster, we have previously proposed that the $(S)$-3-chloro-4,5-dihydroxy-$\beta$-phenylalanine moiety of C-1027 is derived from L-tyrosine, requiring minimally three transformations: $\alpha,\beta$-amino migration, chlorination, and hydroxylation (45). However, these studies fell short of establishing the order of these steps. Depending on the timing of the $\alpha,\beta$-amino migration step, either L-tyrosine, L-3,4-dihydroxyphenylalanine, L-3-chlorotyrosine, or L-3-chloro-5-hydroxytyrosine could be the substrate for SgcC4 (Figure 1). The kinetic parameters presented in Table 1 clearly show L-tyrosine to be the best substrate among the three that were tested; it is $\sim$100-fold better than either L-3-chlorotyrosine or L-3,4-dihydroxyphenylalanine. Although L-3-chlorotyrosine was unavailable for comparison, we propose that it would be a poor substrate on the basis of the poor reactivity of both L-3-chlorotyrosine and L-3,4-dihydroxyphenylalanine. The current studies therefore implicate L-tyrosine as the preferred substrate for SgcC4 and suggest that the SgcC4-catalyzed $\alpha,\beta$-amino migration precedes both the chlorination and hydroxylation steps in the proposed biosynthetic pathway.

SgcC4 acting on L-tyrosine as a substrate gives a $k_{cat}$ of 0.010 s$^{-1}$. This value is lower than those of most other known ammonia lyases or aminomutases (Tables 1–3). This raises the question of whether a substrate better than free L-tyrosine exists. Walsh and co-workers recently reported that several heme-dependent monooxygenases utilize aminoacyl-PCPs, not the free amino acids, as substrates for amino acid $\beta$-oxidation (46). They further speculated that conjugating proteinogenic amino acids to PCPs could be a general mechanism for sequestering, and thus diverting, a fraction of these to provide building blocks for secondary metabolite biosynthesis (46). In a mechanistic analogy, the actual substrate for SgcC4 could potentially be tyrosyl-PCP. The $sgcCl$ and $sgcC2$ genes, flanking $sgcC4$ in the C-1027 gene cluster, are consistent with this hypothesis. They appear to encode an $\alpha$-amino acid specific adenylate enzyme (SgcCl) and a PCP (SgcC2), respectively, which are potential candidates for L-tyrosine activation and conjugation (45). However, we have overproduced and purified both SgcCl and SgcC2 but have not been able to demonstrate the SgcCl-catalyzed activation of L-tyrosine and formation of tyrosyl-SgcC2 in vitro (L. Du and B. Shen, unpublished results).

The aminoacyl-PCP specific monooxygenases have no detectable activity toward the free amino acid substrates (47). On the other hand, SgcC4 does utilize free L-tyrosine as a substrate. Moreover, the affinity of SgcC4 for free L-tyrosine, as measured by its $K_M$ value, is similar to that observed for other PALs and HALs binding to their cognate free amino acid substrates (Table 3). Additionally, X-ray structural analysis shows a conserved arginine residue interacting with the negatively charged carboxylate group of the substrate (25). This residue is conserved in SgcC4 (Arg312, Figure 2), arguing against a tyrosyl-PCP substrate for SgcC4.

The incomplete formation of MIO during the overproduction of the enzyme in the heterologous E. coli host could also partially account for the low activity of SgcC4. The difference spectrum presented in Figure 8 allows one to calculate that only $\sim$18% of the purified SgcC4 has MIO present. This assumes that the ammonia lyases analyzed by Rother et al. (42) contained MIO in 100% of the enzyme active sites. Thus, the true $k_{cat}$ for SgcC4 should be $\sim$0.05 s$^{-1}$.

A surprising observation made in this work is the $\beta$-tyrosine racemase activity of SgcC4. When either $(R)$-$\beta$-tyrosine or $(S)$-$\beta$-tyrosine was added to SgcC4 reaction mixtures, both reached nearly identical equilibria following overnight incubation. Chiral HPLC analysis clearly showed that $(S)$-$\beta$-tyrosine is the kinetically preferred substrate for formation of L-$\alpha$-tyrosine, in agreement with the observed stereochemistry of $(S)$-3-chloro-4,5-dihydroxy-$\beta$-phenylalanine in C-1027 (48). The racemase activity is specific for $\beta$-tyrosine. D-$\alpha$-Tyrosine was not observed as a product from either enantiomer of $\beta$-tyrosine, nor does it serve as a substrate for SgcC4 for $\beta$-tyrosine production. In the context of the mechanism presented in Figure 3, one can speculate that the enzyme can readily add ammonia to either face of the planar HCA intermediate to give the stereoisomeric products. The physiological significance of this racemase activity is at present not clear.

In summary, the SgcC4 protein is truly an enzymological marvel. It is produced as a pro protein that autocatalytically processes itself, forming the complex MIO cofactor from a simple tripeptide at the active site. It is then capable of at least three additional catalytic activities. Its evolutionary heritage is clearly displayed in the ammonia lyase activity, which forms HCA from either $\alpha$- or $\beta$-tyrosine. The enzyme has acquired the ability to extend this core catalytic activity to include catalysis of the Michael addition of ammonia to form $\beta$-tyrosine, although this is as yet imperfect since HCA production is significant. Last, and perhaps most remarkably, SgcC4 is as efficient as a $\beta$-tyrosine racemase as it is as an aminomutase. The diversity of the reactions catalyzed by this enzyme is a testament to the extreme prowess that protein catalysts are capable of garnering through evolution.

### ACKNOWLEDGMENT

We thank Dr. Y. Li (Institute of Medicinal Biotechnology, Chinese Academy of Medical Sciences, Beijing, China) for the C-1027-producing St. globisporus strain.

### REFERENCES

1. Hu, J. L., Xue, Y. C., Xie, M. Y., Zhang, R., Otani, T., Minami, Y., Yamada, Y., and Marunaka, T. (1988) *J. Antibiot.* 41, 1575–1579.
2. Williamson, J. M., and Brown, G. M. (1979) *J. Biol. Chem.* 254, 8074–8082.
3. Poston, J. M. (1976) *J. Biol. Chem.* 251, 1859–1863.
4. Chirpich, T. P., Zappia, V., Costilow, R. N., and Barker, H. A. (1970) *J. Biol. Chem.* 245, 1778–1789.
5. Prabhakaran, P. C., Woo, N.-T., Yorkey, P. S., and Gould, S. J. (1988) *J. Am. Chem. Soc.* 110, 5785–5791.
6. Kurylo-Borowska, Z., and Abramsky, T. (1972) *Biochim. Biophys. Acta* 264, 1–10.
7. Walker, K. D., and Floss, H. G. (1998) *J. Am. Chem. Soc.* 120, 5333–5334.
8. Morley, C. G., and Stadtman, T. C. (1970) *Biochemistry* 9, 4890–4900.
9. Somack, R., and Costilow, R. N. (1973) *Biochemistry* 12, 2597–2604.
10. Morley, C. D., and Stadtman, T. C. (1972) *Biochemistry* 11, 600–605.
11. Chen, H. P., Wu, S. H., Lin, Y. L., Chen, C. M., and Tsay, S. S. (2001) *J. Biol. Chem.* 276, 44744–44750.
12. Babior, B. M., Moss, T. H., and Gould, D. C. (1972) *J. Biol. Chem.* 247, 4389–4392.
13. Finlay, T. H., Valinsky, J., Mildvan, A. S., and Abeles, R. H. (1973) *J. Biol. Chem.* 248, 1285–1290.
14. Sofia, H. J., Chen, G., Hetzler, B. G., Reyes-Spindola, J. F., and Miller, N. E. (2001) *Nucleic Acids Res.* 29, 1097–1106.
15. Song, K. B., and Frey, P. A. (1991) *J. Biol. Chem.* 266, 7651–7655.
16. Frey, P. A. (2001) *Chem. Rec.* 1, 277–289.
17. Jordan, P. M. (1994) *Curr. Opin. Struct. Biol.* 4, 902–911.
18. Poppe, L. (2001) *Curr. Opin. Chem. Biol.* 5, 512–524.
19. Langer, B., Langer, M., and Retey, J. (2001) in *Advances in Protein Chemistry* (Dove, J. E., Ed.) pp 175–214, Academic Press, New York.
20. Kyndt, J. A., Meyer, T. E., Cusanovich, M. A., and Van Beeumen, J. J. (2002) *FEBS Lett.* 512, 240–244.
21. Smith, T. A., Cordelle, F. H., and Abeles, R. H. (1967) *Arch. Biochem. Biophys.* 120, 724–725.
22. Langer, M., Reck, G., Reed, J., and Retey, J. (1994) *Biochemistry* 33, 6462–6467.
23. Hanson, K. R., and Havir, E. A. (1970) *Arch. Biochem. Biophys.* 141, 1–17.
24. Wickner, R. B. (1969) *J. Biol. Chem.* 244, 6550–6552.
25. Schwede, T. F., Retey, J., and Schulz, G. E. (1999) *Biochemistry* 38, 5355–5361.
26. Langer, M., Pauling, A., and Retey, J. (1995) *Angew. Chem., Int. Ed.* 34, 1464–1465.
27. Christenson, S. D., Liu, W., Toney, M. D., and Shen, B. (2003) *J. Am. Chem. Soc.* 125, 6062–6063.
28. Sambrook, J. E., Fritsch, E. F., and Maniatis, T. (1989) *Molecular Cloning: a laboratory manual*, 2nd ed., Cold Springs Harbor Laboratory Press, Plainview, NY.
29. Kirby, G. W. (1963) *J. Chem. Soc.*, 3274–3275.
30. Liu, W., and Shen, B. (2000) *Antimicrob. Agents Chemother.* 44, 382–392.

31. Bradford, M. M. (1976) *Anal. Biochem.* 72, 248–254.
32. Aberhart, D. J. (1988) *Anal. Biochem.* 169, 350–355.
33. Zimmermann, S., and Hahlbrock, K. (1975) *Arch. Biochem. Biophys.* 166, 54–62.
34. Klee, C. B., Kirk, K. L., and Cohen, L. A. (1979) *Biochem. Biophys. Res. Commun.* 87, 343–348.
35. Klee, C. B., Kirk, K. L., Cohen, L. A., and McPhie, P. (1975) *J. Biol. Chem.* 250, 5033–5040.
36. Kuzmic, P. (1996) *Anal. Biochem.* 237, 260–273.
37. Wu, P.-C., Kroening, T. A., White, P. J., and Kendrick, K. E. (1992) *Gene* 115, 19–25.
38. Langer, B., Starck, J., Langer, M., and Retey, J. (1997) *Bioorg. Med. Chem. Lett.* 7, 1077–1082.
39. Langer, B., Rother, D., and Retey, J. (1997) *Biochemistry* 36, 10867–10871.
40. Rother, D., Poppe, L., Morlock, G., Viergutz, S., and Retey, J. (2002) *Eur. J. Biochem.* 269, 3065–3075.
41. Langer, M., Lieber, A., and Retey, J. (1994) *Biochemistry* 33, 14034–14038.
42. Rother, D., Merkel, D., and Retey, J. (2000) *Angew. Chem., Int. Ed.* 39, 2462–2464.
43. Peterkofsky, A. (1962) *J. Biol. Chem.* 237, 787–795.
44. Parry, R. J., and Kurylo-Borowska, Z. (1980) *J. Am. Chem. Soc.* 102, 836–837.
45. Liu, W., Christenson, S. D., Standage, S., and Shen, B. (2002) *Science* 297, 1170–1173.
46. Chen, H., and Walsh, C. T. (2001) *Chem. Biol.* 8, 301–312.
47. Chen, H., Hubbard, B. K., O’Connor, S. E., and Walsh, C. T. (2002) *Chem. Biol.* 9, 103–112.
48. Iida, K., Fukuda, F., Tanaka, T., Hirama, M., Imajo, S., Ishiguro, M., and Yoshida, K. (1996) *Tetrahedron Lett.* 37, 4997–5000.
49. Appert, C., Logemann, E., Hahlbrock, K., Schmid, J., and Amrhein, N. (1994) *Eur. J. Biochem.* 225, 491–499.
BI035223R
