
Molecular Microbiology (2003) 50(3), 931–948

doi:10.1046/j.1365-2958.2003.03737.x

# The vanG glycopeptide resistance operon from Enterococcus faecalis revisited

Florence Depardieu,$^{1}$ Maria Grazia Bonora,$^{1}$ Peter E. Reynolds$^{2}$ and Patrice Courvalin$^{1*}$

$^{1}$Unité des Agents Antibactériens, Institut Pasteur, 25 rue du Docteur Roux, 75724 Paris, Cedex 15, France.

$^{2}$Department of Biochemistry, University of Cambridge, The Downing Site, Tennis Court Road, Cambridge CB2 1QW, UK.

## Summary

Acquired VanG-type resistance to vancomycin (MIC = 16 μg ml⁻¹) but susceptibility to teicoplanin in *Enterococcus faecalis* BM4518 and WCH9 is due to the inducible synthesis of peptidoglycan precursors ending in D-alanine-D-serine. The *vanG* cluster, assigned to a chromosomal location, was composed of genes recruited from various *van* operons. The 3′ end encoded VanG, a D-Ala:D-Ser ligase, VanXY$_{G}$, a putative bifunctional D,D-peptidase and VanT$_{G}$, a serine racemase: VanG and VanT$_{G}$ were implicated in the synthesis of D-Ala:D-Ser as in VanC- and VanE-type strains. Upstream from the structural genes for these proteins were *vanW$_{G}$* with unknown function and *vanY$_{G}$* containing a frameshift mutation which resulted in premature termination of the encoded protein and accounted for the lack of UDP-MurNAc-tetrapeptide in the cytoplasm. Without the frameshift mutation, VanY$_{G}$ had homology with Zn²⁺ dependent D,D-carboxypeptidases. The 5′ end of the gene cluster contained three genes *vanU$_{G}$*, *vanR$_{G}$* and *vanS$_{G}$* encoding a putative regulatory system, which were co-transcribed constitutively from the *PY$_{G}$* promoter, whereas transcription of *vanY$_{G}$*, *W$_{G}$*, *G*, *XY$_{G}$*, *T$_{G}$* was inducible and initiated from the *P$_{YG}$* promoter. Transfer of VanG-type glycopeptide resistance to *E.* faecalis JH2-2 was associated with the movement, from chromosome to chromosome, of genetic elements of c. 240 kb carrying also *ermB*-encoded erythromycin resistance. Sequence determination of the flanking regions of the *vanG* cluster in donor and transconjugants revealed the same 4 bp direct repeats and 22 bp imperfect inverted repeats that delineated the large element.

Accepted 24 July, 2003. *For correspondence.* E-mail pcourval@pasteur.fr; Tel. (+33) 145688320; Fax (+33) 145688319.

© 2003 Blackwell Publishing Ltd

## Introduction

The molecular target of glycopeptide antibiotics is the D-alanyl-D-alanine (D-Ala-D-Ala) terminus of intermediates in peptidoglycan synthesis. By binding to this dipeptide, vancomycin and teicoplanin inhibit the transglycosylation and transpeptidation reactions in peptidoglycan assembly (Reynolds, 1989).

Glycopeptide resistance in enterococci results from the production of modified peptidoglycan precursors ending in D-Ala-D-Lac (VanA, VanB and VanD) or D-Ala-D-Ser (VanC and VanE) to which glycopeptides exhibit low binding affinities and from the elimination of the high affinity D-Ala-D-Ala-ending precursors synthesized by the host Ddl ligase (Arthur et al., 1992a; Reynolds et al., 1994a). In enterococci with the VanA, VanB or VanD phenotype, synthesis of D-Ala-D-Lac requires the presence of a ligase (VanA, VanB or VanD) of altered specificity compared to the host Ddl ligase, and of a dehydrogenase (VanH, VanH$_{B}$ or VanH$_{D}$) that converts pyruvate to D-Lac (Arthur et al., 1996a; Perichon and Courvalin, 2000). In VanC- and VanE-type strains, the ligase genes (*vanC* or *vanE*) encode a protein catalysing the synthesis of D-Ala-D-Ser (Reynolds et al., 1994a), and production of D-Ser is due to a membrane-bound serine racemase (VanT or VanT$_{E}$) (Arias et al., 1999).

The interaction of a glycopeptide with its normal target is prevented by the removal of precursors terminating in D-Ala (Reynolds, 1998). Two enzymes are involved in this process: a cytoplasmic D,D-dipeptidase (VanX, VanX$_{B}$ or VanX$_{D}$) that hydrolyses the dipeptide D-Ala-D-Ala synthesized by the host Ddl ligase and a membrane-bound D,D-carboxypeptidase (VanY, VanY$_{B}$ or VanY$_{D}$) that removes the C-terminal D-Ala residue of late peptidoglycan precursors when elimination of D-Ala-D-Ala by VanX is incomplete (Reynolds et al., 1994b; Arthur et al., 1998). In *Enterococcus gallinarum* BM4174 and *E.* faecalis BM4405, respectively, representative of the VanC- and VanE-type of resistance, both activities are encoded by a single gene, *vanXY$_{C}$* or *vanXY$_{E}$*. VanXY$_{C}$ and VanXY$_{E}$ have a cytoplasmic location and contain consensus sequences for zinc binding, stabilizing the binding of the substrate and catalysing hydrolysis that are present in both VanX- and VanY-type enzymes (Reynolds et al., 1999). The proteins have very low dipeptidase activity against D-Ala-

d-Ser, unlike VanX, and no activity against UDP-MurNAc-pentapeptide[d-Ser] (Reynolds *et al.*, 1999).

Regulation of expression of the vancomycin resistance gene clusters is controlled by a two-component regulatory system (Parkinson and Kofoid, 1992). These systems are composed of cytoplasmic VanR-type response regulators that act as transcriptional activators and VanS-type histidine kinases that are associated with the membrane (Wright *et al.*, 1993; Holman *et al.*, 1994; Arthur *et al.*, 1997). In the *vanA*, *vanB* and *vanD* clusters the genes encoding the two-component regulatory system (VanRS, VanR<sub>B</sub>S<sub>B</sub> or VanR<sub>D</sub>S<sub>D</sub>) are present upstream from the structural genes for the resistance proteins (Arthur *et al.*, 1992b; Evers and Courvalin, 1996; Casadewall and Courvalin, 1999; Depardieu *et al.*, 2003), whereas in the *vanC* and *vanE* clusters they are located downstream (Arias *et al.*, 2000a; Abadía *et al.*, 2002).

The regulatory and resistance genes are transcribed from distinct promoters *P<sub>R</sub>*, *P<sub>RB</sub>*, *P<sub>RD</sub>*, and *P<sub>H</sub>*, *P<sub>YB</sub>*, *P<sub>YD</sub>*, respectively, that are coordinately regulated (Evers and Courvalin, 1996; Arthur *et al.*, 1997; 1999; Haldimann *et al.*, 1997). The *vanC* cluster of *E*. *gallinarum* BM4174 is expressed constitutively, and two regions upstream from *vanC* and *vanR<sub>C</sub>* were postulated as potential promoters (Arias *et al.*, 2000a). However, the five genes of the *vanE* operon in *E*. *faecalis* BM4405 are co-transcribed from a single promoter upstream from *vanE* (Abadía *et al.*, 2002).

The *vanG* cluster in *E*. *faecalis* WCH9 confers moderate level resistance to vancomycin (MIC = 16 µg ml<sup>-1</sup>) and has been the subject of a preliminary report (McKessar *et al.*, 2000). In the current work, five *E*. *faecalis* strains

with acquired resistance of the VanG-type were studied by pulsed-field gel electrophoresis (PFGE) and four strains were found to be distinct. We report the genetic organization, the location of the *vanG* gene cluster and the regulation of expression of the resistance genes in two of these strains. Resistance was inducible by vancomycin and, as in VanC- and VanE-type strains, was a result of the synthesis of peptidoglycan precursors ending in D-Ala-D-Ser a finding consistent with the presence of a serine racemase (*VanT<sub>G</sub>*). Conjugal transfer of the *vanG* operon was associated with the movement, from chromosome to chromosome, of large genetic elements which additionally carried the *ermB* erythromycin resistance gene.

### Results

#### Characterization of Enterococcus faecalis with VanG-type resistance

The *vanG* cluster confers moderate level resistance to vancomycin (16 µg ml<sup>-1</sup>) but not to teicoplanin (0.5 µg ml<sup>-1</sup>) in *E*. *faecalis* WCH9 (McKessar *et al.*, 2000). This strain and four other VanG-type *E*. *faecalis* isolated from patients in the same hospital between 1996 and 1998 (Table 1) were studied by pulsed field gel electrophoresis (Fig. 1). Their *Smal* patterns differed by a single band (Fig. 1A, left), from c. 400 kb to 530 kb in size, which hybridized to a *vanG* probe (Fig. 1A, right). The strains could thus result from independent acquisition by the same recipient of various genetic elements carrying the *vanG* operon or could represent the same transconjugant

---

### Table 1. Strains and plasmids.

| Strain or plasmid | Relevant characteristics                                                                                          | Source or reference |
|-------------------|------------------------------------------------------------------------------------------------------------------|----------------------|
| Strain            |                                                                                                                  |                      |
| *E*. coli         |                                                                                                                  |                      |
| Top10             | F<sup>-</sup> *mcrA Δ(mrr-hsdRMS-mcrBC)* φ80*lacZΔM15 ΔlacX74 deoR recA1 araD139 Δ(ara-leu)7697 galU galK rpsL endA1 nupG* | Invitrogen          |
| *E*. *faecalis*   |                                                                                                                  |                      |
| JH2-2             | Fus<sup>R</sup> Rif<sup>R</sup>                                                                                 | Jacob and Hobbs (1974) |
| WCH9              | Vm<sup>R</sup> Te<sup>S</sup> (VanG-type)                                                                        | McKessar *et al.* (2000) |
| BM4518            | Vm<sup>R</sup> Te<sup>S</sup> (VanG-type)                                                                        | This work           |
| BM4519            | Vm<sup>R</sup> Te<sup>S</sup> (VanG-type)                                                                        | This work           |
| BM4520            | Vm<sup>R</sup> Te<sup>S</sup> (VanG-type)                                                                        | This work           |
| BM4521            | Vm<sup>R</sup> Te<sup>S</sup> (VanG-type)                                                                        | This work           |
| BM4522            | BM4518 × JH2-2 Vm<sup>R</sup> Te<sup>S</sup> (VanG-type)                                                         | This work           |
| BM4523            | BM4518 × JH2-2 Vm<sup>I</sup>Te<sup>S</sup> (VanG-type)                                                          | This work           |
| Plasmid           |                                                                                                                  |                      |
| pCR®-Blunt        | Km<sup>R</sup>, Zeocin<sup>TM R</sup>, *oriR* from *coliE1*, *lacZα*, *ccdB*                                      | Invitrogen          |
| pAT79             | *oriR* from pAMβ1, *oriR* from pUC, *oriT* from RK2, Sp<sup>R</sup> *lacZα P<sub>2</sub>cat*                       | Arthur *et al.* (1992b) |
| pAT645            | 886 bp *SacI-Xbal* PCR fragment (*vanY<sub>G</sub>*) of BM4518 cloned into pAT79                                 | This work           |
| pAT646            | 848 bp *SacI-Xbal* PCR fragment (*vanXY<sub>G</sub>*) of BM4518 cloned into pAT79                                | This work           |
| pAT647            | 637 bp PCR fragment (*vanS<sub>G</sub><sup>′</sup>,Y<sub>G</sub><sup>′</sup>*) of BM4518 cloned into pCR®-Blunt     | This work           |
| pAT648            | 748 bp PCR fragment (*orfG23, vanU<sub>G</sub>*) of BM4518 cloned into pCR®-Blunt                                 | This work           |

Fus, fusidic acid; I, intermediate; Km, kanamycin; ORF, open reading frame; Rif, rifampicin; R, resistant; S, susceptible; Te, teicoplanin; Vm, vancomycin.

Fig. 1. Analysis of SmaI-digested genomic DNA of VanG-type clinical isolates (A) and transconjugants from BM4518 (B) by pulsed field gel electrophoresis (left) and by Southern hybridization (right) with a vanG-specific probe. Bacteriophage λ concatemers were used as molecular size markers (lane 1, Biolabs) and the sizes are indicated at the left. The size of the hybridizing fragments is indicated at the right.

having suffered DNA rearrangements following or associated with transfer.

The vanG gene cluster was assigned to a chromosomal fragment of c. 800 kb in strains WCH9 and BM4520, 700 kb in BM4518, 680 kb in BM4519 and 750 kb in BM4521 by contour-clamped homogeneous electric field gel electrophoresis of total DNA digested with I-CeuI followed by successive hybridization with 16S rRNA (*rrs*) and vanG probes (data not shown). This analysis also indicated that the VanG-type isolates were distinct, except WCH9 and BM4520 that were indistinguishable as already observed with SmaI.

### Organization of the vanG operon

The amplification products obtained following PCR mapping with primers complementary to the vanG operon from *E. faecalis* WCH9 and with BM4518 DNA as template were of the expected size indicating that, in BM4518, all the genes constituting the vanG operon were present with an organisation similar to that in WCH9 (Fig. 2). No large insertions or deletions in the non-coding regions were detected. However, no PCR products were obtained with

total DNA of BM4518 or WCH9 as template when one of the primers used for amplification was complementary to the 5′ end of vanR<sub>G</sub> or to the region downstream from vanT<sub>G</sub> (Fig. 2A). Thus, the genomic environment of the vanG cluster in strains BM4518 and WCH9 differed from that published for the latter strain (McKessar et al., 2000). The complete nucleotide sequence of the PCR products from the vanG operon in BM4518 was determined and compared to the published sequence of WCH9. The numerous differences observed between the two strains led to the resequencing of WCH9 which proved to be identical to BM4518.

The sequence deduced from BM4518 and WCH9 was compared with those of the other van operons (Table 2). VanR<sub>G</sub> and VanS<sub>G</sub> exhibited, respectively, highest identity with the VanR<sub>D</sub> response regulator (73%) and the VanS<sub>D</sub> histidine protein kinase (55%) (Casadewall and Courvalin, 1999; Boyd et al., 2000; Depardieu et al., 2003). The conserved aspartate and lysine residues (D10, D53 and K102) typical of response regulators in two-component systems from Gram-positive bacteria were present in VanR<sub>G</sub>. The C-terminal portion of VanS<sub>G</sub> contained the five blocks of conserved amino acids (H, N, G1, F and G2)

934 F. Depardieu, M. G. Bonora, P. E. Reynolds and P. Courvalin

Fig. 2. Schematic representation of the *vanG* operon from strains WCH9 and BM4518.

A. Horizontal bars depict the PCR products corresponding to overlapping amplified fragments from strains WCH9 and BM4518. The position of the 5′ ends of the primers is indicated in parentheses. Numbering corresponds to the +1 transcriptional start site upstream from the *vanU<sub>G</sub>* gene. The size of the PCR products are indicated in bold characters. The inserts sequenced or cloned under the control of the *P<sub>2</sub>* promoter in recombinant plasmids are represented above the *vanG* operon.

B. Map of the 7.918 kb region containing the *vanG* gene cluster. Open arrows represent coding sequences and indicate the direction of transcription. The genes for regulatory system are represented by grey arrows, for the D,D-carboxypeptidases by an hatched arrow, for the VanW unknown protein by an open arrow and for the resistance genes by dotted arrows. The guanosine plus cytosine content (% GC) is indicated in the arrows. The vertical bar in *vanY<sub>G</sub>* indicates the frameshift mutation leading to a predicted truncated protein. The primers used for IPCR (RG5-SG7 and TG14-TG15) are indicated by thin half-arrows. The endonuclease restriction sites used in IPCR are indicated above the map. The PCR fragment internal to the *vanG* gene of strain BM4518 used as a probe in hybridization experiments is indicated above the corresponding region.

C. Probes used in Northern hybridization. The size of the PCR products is indicated in bold characters. The position of the 5′ ends of the primers is indicated in parentheses. Numbering corresponds to the +1 transcriptional start site upstream from the *vanU<sub>G</sub>* gene.

D. Oligodeoxynucleotides used in RT-PCR. Arrowheads indicate the position and orientation of the primers.

E. Oligodeoxynucleotides used in primer extension. The inserts cloned into pCR-Blunt and used in primer extension are indicated.

Table 2. Percentage of identity of the *vanG* operon deduced proteins with those from other *van* operons.

| % of identity with type | VanR<sub>G</sub> | VanS<sub>G</sub> | VanY<sub>G</sub> | VanW<sub>G</sub> | VanG | VanXY<sub>G</sub> | VanT<sub>G</sub> |
|-------------------------|------------------|------------------|------------------|------------------|-------|-------------------|-------------------|
| VanA                    | 57               | 42               | 33               | NA<sup>a</sup>    | 44    | NA                | NA                |
| VanB                    | 31               | 16               | **56**<sup>b</sup> | 49               | 46    | NA                | NA                |
| VanD                    | 73               | 55               | 23               | NA               | 44    | NA                | NA                |
| VanC                    | 62               | 40               | 30               | NA               | 42    | 41                | 37                |
| VanE                    | 55               | 29               | 32               | NA               | 41    | 39                | 37                |

a. NA, not applicable.

b. The numbers in boldface indicate the highest percentage of identity with VanG-type proteins.

characteristic of transmitter modules in histidine protein kinases. Histidine 436 of VanS<sub>G</sub> was aligned with histidine residues of sensors in various Van-type strains which are the putative sites of autophosphorylation (data not shown). The hydrophathy profile of the N-terminal putative sensor domain of VanS<sub>G</sub> revealed the presence of two stretches of hydrophobic amino acids similar to those in VanS, VanS<sub>B</sub>, VanS<sub>D</sub> and EnvZ, suggesting a similar topology for these enzymes (data not shown). The structural homology of VanR<sub>G</sub> and VanS<sub>G</sub> with proteins of two component regulatory systems in Van-type strains suggests that a signal transducing system controls the expression of proteins involved in vancomycin resistance in E. faecalis WCH9 and BM4518.

A 2 kb fragment upstream from vanR<sub>G</sub> was obtained from WCH9 and BM4518 by inverse PCR (Fig. 2B). The sequence revealed that the 120 bp at the 5′ end of vanR<sub>G</sub> differed from the published sequence and that an ORF (Fig. 2B), with a proposed initiation codon (ATG) preceded by a putative RBS (5′-AGTAAAGAGGN<sub>8</sub>ATG-3′) that displayed high complementarity (underlined) to the 3′ extremity of Bacillus subtilis 16S rRNA (3′-OH-UCUUUCCUCC-5′) (Moran et al., 1982), was present upstream from vanR<sub>G</sub>. This 228 bp sequence, from positions 23–250, encoded a putative 75 amino acid protein designated VanU<sub>G</sub> (Fig. 2B) that displayed 65% identity with an unknown protein from Clostridium thermocellum and 35% identity with a putative transcriptional activator from Streptomyces coelicolor and with domains conserved in predicted transcriptional regulators from various species (RPS-Blast from NCBI). It appears that vanU<sub>G</sub> is likely to be part of the vanG cluster as it was in the same orientation as the other genes of the vanG cluster and there was no intergenic region between vanU<sub>G</sub> and vanR<sub>G</sub>.

Analysis of the vanY<sub>G</sub> gene and of its translation product indicated that the derived protein terminated prematurely after amino acid 121 and consequently lacked the active sites of Zn<sup>2+</sup> dependent D,D-carboxypeptidases. The hydrophathy profile revealed that the truncated protein had a membrane spanning portion at the N-terminus (data not shown). Further investigation of one of the three reading frames downstream from the stop codon indicated 56% identity over a length of 170 amino acids with the C-terminal portion of VanY<sub>B</sub> (Table 2) (Evers and Courvalin, 1996). This stretch contained the two motifs, SxHxxGxAxD and ExxH, characteristic of the active sites implicated in Zn<sup>2+</sup> binding and in catalysis of Zn<sup>2+</sup> dependent D,D-carboxypeptidase activity (McCafferty et al., 1997; Lessard and Walsh, 1999). It was concluded that a frame-shift mutation resulted in the presence of a stop codon and thus of premature termination of what would otherwise have been a full length VanY<sub>B</sub>-like protein. In spite of the mutation in vanY<sub>G</sub>, penicillin-insensitive D,D-carboxypeptidase activity inducible by vancomycin was detected in the membrane fraction of broken cell preparations of BM4518 and WCH9 (Table 3) as in VanA- and VanB-type strains. However, the peptidoglycan precursor results (see below) and cloning experiments with vanY<sub>G</sub> indicated that the protein with this activity is unlikely to be encoded by vanY<sub>G</sub>.

The VanXY<sub>G</sub> protein, despite very low D,D-dipeptidase activity in strains BM4518 and WCH9 (Table 3), also contained the amino acid motifs typical of D,D-peptides (McCafferty et al., 1997; Lessard and Walsh, 1999). The hydrophobicity profile of VanXY<sub>G</sub> was consistent with its cytoplasmic location. Comparison with VanX, VanX<sub>B</sub>, VanY, VanY<sub>B</sub>, VanXY<sub>C</sub> and VanXY<sub>E</sub> indicated that VanXY<sub>G</sub> had the highest degree of identity with VanXY-type enzymes (Table 2). Sequence comparison indicates that the VanXY proteins are more closely related to VanY- than to VanX-type proteins but lack the membrane-spanning segment present in both VanY and VanY<sub>B</sub> (Arthur et al., 1998).

---

**Table 3. D,D-peptidase (VanXY<sub>G</sub> and VanY<sub>G</sub>) and racemase (VanT<sub>G</sub>) activities in extracts from VanG-type strains.**

| Activity (nmol min<sup>−1</sup> mg protein<sup>−1</sup>)<sup>a</sup> |
| --- |
| **D,D-dipeptidase<sup>b</sup>** | **D,D-carboxypeptidase<sup>c</sup>** | **D-serine racemase<sup>d</sup>** | **D-alanine racemase<sup>e</sup>** |
| Cytoplasmic fraction | Cytoplasmic fraction | Membrane fraction | Membrane fraction | Cytoplasmic fraction | Membrane fraction |

| E. faecalis | Vancomycin concentration (µg ml<sup>−1</sup>) |
| --- | --- |
| WCH9 | 0 | 0.2 ± 0.1 | nd | nd | 1.1 ± 0.3 | 214 ± 44 | 20 ± 1 |
|  | 8 µg ml<sup>−1</sup> | 1.2 ± 0.5 | 2.2 ± 1.3 | 22 ± 6 | 11 ± 4 | 103 ± 24 | 7 ± 1 |
| BM4518 | 0 | 0.7 ± 0.2 | nd | nd | 2.0 ± 0.3 | 287 ± 17 | 16 ± 3 |
|  | 8 µg ml<sup>−1</sup> | 1.0 ± 0.3 | 4.0 ± 1.0 | 37 ± 3 | 16 ± 4 | 134 ± 7 | 4 ± 1 |

a. Results are the means ± standard deviations obtained from a minimum of three independent extracts.  
b. The D,D-dipeptidase activity was measured in the 100 000 g supernatants of lysed bacteria.  
c. The D,D-carboxypeptidase and D-alanine racemase activities were measured in the supernatants and in the resuspended pellet fractions after centrifugation of lysed bacteria at 100 000 g for 45 min.  
d. The D-serine racemase activity was measured in the resuspended pellet fractions after centrifugation of lysed bacteria at 100 000 g for 45 min.  
e. Not detectable (less than 0.2 nmol min<sup>−1</sup> mg protein<sup>−1</sup>).

VanT<sub>G</sub> displayed 37% identity with VanT and VanT<sub>E</sub> (Table 2) and, as with these two proteins, its N-terminal domain contained 11 clusters of hydrophobic amino acids suggesting that it was a membrane-associated protein. The C-terminal domain of VanT<sub>G</sub> contained the putative pyridoxal 5′ phosphate attachment motif (V<sub>373</sub>VKAXAYGxG<sub>382</sub>) which is highly conserved in alanine racemases and in VanT and VanT<sub>E</sub> (Arias et al., 1999). Residues A376, A378, Y379, R410, G620, D623, R627 and E688 which putatively play a structural role and maintain the geometry of the active site of alanine racemases and of VanT and VanT<sub>E</sub> (Arias et al., 1999; Abadía et al., 2002) were all present in VanT<sub>G</sub>.

**Characterization of peptidoglycan precursors of WCH9 and BM4518**

To analyse the cytoplasmic peptidoglycan precursors, cultures of *E. faecalis* WCH9 and BM4518 grown in the presence or in the absence of vancomycin (4 µg ml<sup>−1</sup>) were incubated with ramoplanin to inhibit cell wall synthesis after formation of the precursors. The results showed that, in the absence of vancomycin, UDP-MurNAc-pentapeptide[Ala] was the main precursor synthesized whereas incubation with vancomycin led to an increase in the production of UDP-MurNAc-pentapeptide[Ser] but not the elimination of UDP-MurNAc-pentapeptide[Ala] (Table 4). Virtually no UDP-MurNAc-tetrapeptide was detected. These data indicate that both strains were inducibly resistant to vancomycin by production of precursors ending in D-Ala-D-Ser.

that of VanT<sub>E</sub> in BM4405 (Fines et al., 1999), but greater than that of VanT in *E. gallinarum* BM4174 (Arias et al., 2000b). In contrast, alanine racemase was synthesized constitutively and present almost exclusively in the cytoplasm (Table 3), a distribution similar to that in BM4174 (VanC) (Arias et al., 2000b) and BM4405 (VanE) (Fines et al., 1999).

**D,D-peptidase activities**

Expression of the resistance genes in VanG-type *E. faecalis* WCH9 and BM4518 was studied by analysis of D,D-dipeptidase and D,D-carboxypeptidase activities which were assayed by determining the amount of D-Ala released from hydrolysis of the dipeptide D-Ala-D-Ala and of the pentapeptide UDP-Mur-NAc-L-Ala-γ-D-Glu-L-Lys-D-Ala-D-Ala respectively (Table 3). The D,D-dipeptidase activity was measured in the 100 000 g supernatant of lysed bacteria that had been grown in the absence or in the presence of 8 µg ml<sup>−1</sup> of vancomycin as an inducer. The cytoplasmic extracts even after induction by vancomycin had very weak D,D-dipeptidase activity (Table 3) which explains the continued presence of UDP-MurNAc-pentapeptide[Ala] (Table 4). BM4405 (VanE) also possessed inducible, weak D,D-dipeptidase activity (VanX-type) in the cytoplasm (Fines et al., 1999), but it was constitutive in BM4174 and had much greater activity. The D,D-carboxypeptidase activity in the cytoplasmic fractions of WCH9 and BM4518 was low and inducible by vancomycin but membrane preparations contained substantial activity that was inducible and not inhibited by 100 µg ml<sup>−1</sup> of benzylpenicillin (Table 3).

**D,D-racemase activities**

VanC- and VanE-type resistance requires three proteins: VanC (VanE) and VanXY<sub>C</sub> (VanXY<sub>E</sub>) which, respectively, catalyse synthesis of D-Ala-D-Ser and elimination of D-Ala-D-Ala, and VanT (VanT<sub>E</sub>) a membrane-bound serine racemase for production of D-Ser. VanT<sub>G</sub> serine racemase activity was detected in the membrane fractions from BM4518 and WCH9 and the enzyme was inducible by vancomycin (Table 3). The activity was much lower than

**D,D-peptidase activities from BM4518 in *E. faecalis* JH2-2**

To test whether the *vanY<sub>G</sub>* and *vanXY<sub>G</sub>* genes encoded functional enzymes, the genes from BM4518 and their RBS were cloned under the control of the constitutive P2 promoter, leading to plasmids pAT645(P<sub>2</sub>*vanY<sub>G</sub>cat*) and pAT646(P<sub>2</sub>*vanXY<sub>G</sub>cat*) (Fig. 2A), that were electrotransformed into *E. faecalis* JH2-2. Although the deduced

**Table 4. MICs of glycopeptides and cytoplasmic peptidoglycan precursors synthesized by VanG-type strains.**

| E. faecalis | MIC (µg ml<sup>−1</sup>) of<sup>a</sup> | % of peptidoglycan precursors<sup>b</sup> |
|-------------|---------------------------------------|-----------------------------------------|
|             | Vm                                    | Te                                      | Culture                                | UDP-MurNAc-                          | UDP-MurNAc-                          | UDP-MurNAc-                          |
|             |                                       |                                         |                                        | tetrapeptide                         | pentapeptide(D-Ala)                  | pentapeptide(D-Ser)                  |
| WCH9        | 16                                    | 0.5                                     | Uninduced                              | 4 ± 2                                 | 66 ± 5                               | 30 ± 4                               |
|             |                                       |                                         | Vm (4 µg ml<sup>−1</sup>)               | 4 ± 3                                 | 29 ± 8                               | 66 ± 7                               |
| BM4518      | 16                                    | 0.5                                     | Uninduced                              | 1 ± 1                                 | 97 ± 1                               | 3 ± 1                                |
|             |                                       |                                         | Vm (4 µg ml<sup>−1</sup>)               | 1 ± 1                                 | 53 ± 3                               | 46 ± 4                               |

a. MICs were determined by the method of Steers et al. (1959). Te, teicoplanin; Vm, vancomycin.

b. Peptidoglycan synthesis was inhibited by addition of ramoplanin to the cultures for 15 min (Arthur et al., 1996b).

sequence of the VanXY<sub>G</sub> protein does not contain mutations in the conserved residues known to be involved in zinc binding and catalysis, only very weak hydrolysis of D-Ala-D-Ala (0.8 ± 0.1 nmol min<sup>-1</sup> mg protein<sup>-1</sup>) was detected in cytoplasmic extracts from E. faecalis JH2-2 harbouring pAT646(P<sub>2</sub>vanXY<sub>G</sub>cat). These results are in agreement with those obtained with crude extracts of BM4518 and WCH9 (Table 3).

No D,D-carboxypeptidase activity was detected in membrane or cytoplasmic fractions of extracts from JH2-2/pAT645(P<sub>2</sub>vanY<sub>G</sub>cat) or JH2-2/pAT646(P<sub>2</sub>vanXY<sub>G</sub>cat) (data not shown). These results account for the observation that in E. faecalis BM4518 and WCH9 substantial amounts of UDP-MurNAc-pentapeptide[D-Ala] were present after induction with vancomycin, whereas UDP-MurNAc-tetrapeptide was present in insignificant amounts (Table 4). The D,D-carboxypeptidase activity in the membrane fractions from WCH9 and BM4518 could be due to the presence of another D,D-carboxypeptidase insensitive

to penicillin G as in VanD-type strain 10/96 A (Depardieu et al., 2003).

### Transcriptional analysis of the vanG gene cluster

The similarity between the genetic organisation of the vanA, vanB and vanD operons with that of the first part of the vanG cluster suggests that the transcriptional start points could have similar locations. The vanA, B, and D operons are co-transcribed from the P<sub>H</sub>, P<sub>YB</sub> and P<sub>YD</sub> promoters located between vanS and vanH, vanS<sub>B</sub> and vanY<sub>B</sub>, and vanS<sub>D</sub> and vanY<sub>D</sub> respectively. The P<sub>R</sub>, P<sub>RB</sub> and P<sub>RD</sub> promoters control transcription of the genes for VanRS, VanR<sub>B</sub>S<sub>B</sub> and VanR<sub>D</sub>S<sub>D</sub> two-component regulatory systems. Total RNA extracted from BM4518 and WCH9 not induced or induced by vancomycin was analysed by Northern hybridization with probes internal to every gene (Fig. 2C) in the vanG operon, including vanU<sub>G</sub> (Fig. 3A). A transcript of c. 2200 nucleotides was obtained without

![Figure 3](#fig3)

**Fig. 3.** Analysis of vanG gene cluster transcription by Northern hybridization (A) and reverse transcription (B).

A. Total RNA from BM4518 not induced (NI) or induced (I) by vancomycin was hybridized successively with vanU<sub>G</sub>, vanR<sub>G</sub>, and vanS<sub>G</sub> probes (see Fig. 2). The size of the transcript indicated at the left was determined according to RNA molecular weight marker I (Boehringer) (not shown).

B. Analysis of transcription of the vanY<sub>G</sub>, vanW<sub>G</sub>, vanG and vanXY<sub>G</sub> genes and, of the vanG, vanXY<sub>G</sub> and vanT<sub>G</sub> genes from strains WCH9 and BM4518. Electrophoresis of the products obtained by RT-PCR from non-induced (lanes 1 and 2) and induced strains (lanes 3 and 4) using primer pairs YG7-G4-1 and G3-TG7, respectively (see Fig. 2D and Table 5). Incubations were carried out in the absence (lanes 1 and 3) or presence (lanes 2 and 4) of reverse transcriptase. Lane M contained DNA from bacteriophage lambda digested with PstI as a size marker.

© 2003 Blackwell Publishing Ltd, *Molecular Microbiology*, **50**, 931–948

and with induction that hybridized with the $vanU_{G}$, $vanR_{G}$ and $vanS_{G}$ probes. Reverse transcription was performed with purified total RNA from both strains and primer SG9 (Table 5) internal to $vanS_{G}$ (Fig. 2D). The complementary DNA (cDNA) was amplified with primers G96 and SG3 internal to $vanU_{G}$ and $vanS_{G}$ respectively (Fig. 2D and Table 5). After separation on agarose gel, a PCR product with the expected size of c. 1.6 kb was obtained from cells grown with or without induction (data not shown). Taken together, these results indicate that the $vanU_{G}$, $vanR_{G}$ and $vanS_{G}$ genes are constitutively co-transcribed and that the corresponding messenger DNA starts upstream from $vanU_{G}$.

Analysis of the transcription of $vanY_{G}$, $vanW_{G}$, $vanG$, $vanXY_{G}$ and $vanT_{G}$ was performed by reverse transcription of total RNA from induced and uninduced BM4518 and WCH9 and primers XYG2 internal to $vanXY_{G}$ and TG14 internal to $vanT_{G}$ (Fig. 2D and Table 5). The correspond-

ing cDNA's were then amplified using primer pairs YG7-G4-1 (Table 5) internal to $vanY_{G}$ and $vanG$, respectively, and G3-TG7 (Table 5) internal to $vanG$ and $vanT_{G}$ (Fig. 2D). The PCR products with the expected sizes of 2.6 and 2 kb were present in much greater amounts after growth under inducing conditions (Fig. 3B). The amplification products were sequenced and no DNA rearrangements were found. These results indicated that the $vanY_{G}$, $vanW_{G}$, $vanG$, $vanXY_{G}$ and $vanT_{G}$ genes were co-transcribed in an inducible manner. Inverted repeats able to form a hairpin structure for termination of transcription were identified downstream from $vanT_{G}$ (position 8193–8233) (data not shown).

Based on these observations, the region upstream from $vanU_{G}$ and the intergenic region upstream from $vanY_{G}$ were explored for transcriptional start sites by primer extension using oligodeoxynucleotides PRG and PYG (Fig. 2E) complementary to the 5′ end of $vanU_{G}$ and $vanY_{G}$ respectively (Fig. 4). This allowed the positioning of transcriptional start sites $P_{UG}$ and $P_{YG}$ and to examine the expression of the $vanG$ operon in strains grown in the presence or absence of vancomycin. A signal located 46 bp upstream from the translation initiation codon of $vanY_{G}$ was observed in WCH9 and BM4518 only in the presence of vancomycin (Fig. 4B) confirming that expression of glycopeptide resistance was inducible in these strains. The $P_{YG}$ promoter contained –35 (AAAAAA) and –10 (TACAAT) regions separated by 18 bp that shared similarity with the $\sigma^{70}$ recognition sequences (Fig. 4B), although the –35 sequence was poorly conserved in agreement with the fact that the promoter was positively regulated. For $P_{UG}$, a signal located 22 bp upstream from the translation initiation codon of $vanU_{G}$ was observed in BM4518 and WCH9 but, in contrast to $P_{YG}$, was present irrespective of induction (Fig. 4A). The $P_{UG}$ promoter contained –35 (TTGCTT) and –10 (TAAAAT) regions corresponding to $\sigma^{70}$ recognition sequences, which were separated by 17 bp (Fig. 4A).

The right and left regions flanking the $vanG$ operon

Sequencing of an 850 bp fragment downstream from $vanT_{G}$, obtained by inverse PCR (Fig. 2B) revealed that the $vanG$ operon was flanked by ORFG24 with the same orientation as ORFG23 upstream from VanU$_{G}$ (Fig. 5). The deduced sequence of ORFG24 displayed 45% identity with an unknown protein from *Clostridium difficile*, 37% with ORF9 of Tn916 from *E. faecalis* and there was 50% identity between ORFG24 and ORFG23 (Fig. 5).

The DNA sequence downstream from ORFG24 (Fig. 5) was obtained by TAIL-PCR (see Experiment procedures and Table 6) and showed the presence of ORFG25 with a proposed ATG initiation codon preceded by a putative

Table 5. Oligonucleotide primers used for IPCR, TAIL-PCR and RT-PCR experiments.

| Primer$^{a}$ | Sequence (5′→3′)$^{b}$ | Position$^{c}$ |
|---------------|-------------------------|----------------|
| SG7+          | GCTTACATCTGTGATTG      | 1455–1471      |
| RG5–          | GCCCCATAATCTTTATCA     | 525–509        |
| G10+          | CATCACCGCCTTGATAA      | –1389–1405     |
| G16+          | TTAACTGTTTTCTCCGC      | –1858–1874     |
| G17+          | CTTAATGTCCGTTTCC       | –1890–1906     |
| G19+          | GGTCTTTATCAACTGGC      | –1917–33       |
| G55+          | CTGGGTCAAGAGACTTAT     | –9137–9153     |
| G56+          | CATAGCGTAGGCAAGTA      | –9275–9291     |
| G57+          | GACAGTCTTTACGCTGT      | –9302–9318     |
| G59+          | AAATTGTCGCTGACATC      | –9362–9378     |
| G110+         | GAAGCGGTCAAAGATGCA     | 687300–687317  |
| G109+         | CTGGTTTAGCGGAAGAAT     | 687364–687381  |
| G108+         | CGCAACCTGAACGAATT      | 687484–687500  |
| TG14–         | ACTTTCAGCTACCATCG      | 7849–7833      |
| TG15+         | GTATGGGACACAGATTA      | 7881–7897      |
| 9–5+          | GACACAGCGAACATCGT      | 8097–8113      |
| 9–2+          | ATATTTGGCAGCACAGC      | 8432–8448      |
| 9–3+          | CTTTGGTGGGCAGTATA      | 8471–8487      |
| GD32+         | GCGGTTGCAGAAAATAGA     | 10696–10712    |
| GD33+         | GCTAATTCTGACATAGGA     | 10876–10893    |
| GD34+         | GTCAGAACGGCACATTA      | 10946–10962    |
| RT-PCR        |                         |                |
| G96+          | GTGGAAACAATGCGTGT      | 14–30          |
| SG3–          | AACTGCTCCAACATACG      | 1664–1648      |
| SG9–          | GTATCCAGACGATAAAACTGT  | 1940–1920      |
| YG7+          | GCAGACGTAACCAATCT      | 2253–2269      |
| G4–           | TCTGCTTCGGCATCAAT      | 4803–4787      |
| XYG2–         | CATAGGAAAATCGGGA       | 5416–5400      |
| G3+           | CAGTGAGCTCCTATACGG     | 4129–4145      |
|               | GAGAAAAGG              |                |
| TG7–          | GCAGGAAGATACCACAA      | 6156–6140      |
| TG14–         | ACTTTCAGCTACCATCG      | 7849–7833      |

a. +, sense primer; –, antisense primer.  
b. A sequence carrying a Sacl restriction site (in italics) was incorporated into oligonucleotide G3.  
c. Nucleotide numbering begins at the transcriptional start site (+1) of the $vanU_{G}$ gene for the right part of the $vanG$ operon and is negative and increase from the upstream part of $vanG$ operon. Nucleotide numbering of *E. faecalis* V583 sequence from TIGR was used for the G110, G109 and G108 primers.

T G C A
A
-35
TGTACGTTGTTTTTCTTATTTTATGTGATATTTATACAATATGCGTTTGATAAAAGCTTGGAGTTTGC
CAAAACTTGCTTTTACGTACTTTGTATGC**TAAAAT**AGGCAAAGCATAGT**AAAGAGGTGGAAACA**ATGCG
CCGCTCGAAGCACCTCCGA
-10
+1
RBS
TGTTAGTTATAATAAGCTCTGGAAAGCTTTTAATTGATAGGGACATGAAAAAAGGCGAGCTTCGTGAGGCT
CAACCTCATTCAATTTTCGTGTA (Primer PRG)
GTTGGAGTAAGTAAAAGCACAT

B
(a) (b) (c)
T G C A
-35
ATTGAAAGTGACGATTCTTTCATCGTAGGAAAATCGTAAGAAATTCCGTATGAAAATTTGGGATGTTCGT
TGGATAGAACAAAAAAAAACAGCACTCTTGTTTTTGA**TCAAAT**ATTTGTGTC**AAAGCAAGAGTGCTTTTATT**
RBS
TTTGTTAA**AGGAGG**AATTGTTGATATGAACCATATGAATATGAAACACAGACGCAGAAAAACGCAGACGTA
CGTTCGTAAAAAAAAATACCATGCC
-10
+1
ACCAATCTTTTTTGTTCACAGGAATTTTACTCTTAGTTGTATCTGCAAGCAGTTTTTTATGGTACGG
AAAACCGTTACGCCGTT (Primer PYG)
TTTTGGCAATGCGGCAA

C
(a) TCCTAGGAAAAT
(b) TCCTAAGAAATT
(c) CCCTATGAAAAT

(i) Consensus sequence T CGTAXGAAA T
                              C           A
(ii) Consensus sequence T GTA GAAA T
                              C    A     A


Fig. 4. Identification of the transcriptional start site for the *vanU*<sub>G</sub>, *vanR*<sub>G</sub> and *vanS*<sub>G</sub> genes and the *vanY*<sub>G</sub>, *vanW*<sub>G</sub>, *vanG*, *vanXY*<sub>G</sub> and *vanT*<sub>G</sub> genes in WCH9 and BM4518 by primer extension analysis.

A. Left panel, primer elongation product obtained with oligodeoxynucleotide PRG and 25 μg of total RNA from BM4518 grown without vancomycin (lane 1); BM4518 grown in the presence of vancomycin (lane 2); WCH9 grown without vancomycin (lane 3); WCH9 grown in the presence of vancomycin (lane 4); lanes T, G, C and A, results of sequencing reactions performed with the PRG primer. Right panel, sequence from nucleotide position –135 to +97 (numbering from the A of the ATG start codon of the *vanU*<sub>G</sub> gene, negative in the 3′ to 5′ direction and positive in the 5′-to-3′ direction). The +1 transcriptional start site for the *vanU*<sub>G</sub>, *vanR*<sub>G</sub> and *vanS*<sub>G</sub> genes and the upstream –35 and –10 promoter sequences are in boldface. The ATG start codon of *vanU*<sub>G</sub> is indicated in italics, and the RBS is in boldface and underlined.

B. Left panel, primer elongation product obtained with oligodeoxynucleotide PYG and 25 μg of total RNA from WCH9 grown without vancomycin (lane 1); WCH9 grown in the presence of vancomycin (lane 2); BM4518 grown without vancomycin (lane 3); BM4518 grown in the presence of vancomycin (lane 4); lanes T, G, C and A, results of sequencing reactions performed with the same primer. Right panel, sequence from nucleotide position –164 to +133 (numbering from the A of the ATG start codon of the *vanY*<sub>G</sub> gene, negative in the 3′-to-5′ direction and positive in the 5′-to-3′ direction). The +1 transcriptional start site for the *vanY*<sub>G</sub>, *vanW*<sub>G</sub>, *vanG*, *vanXY*<sub>G</sub> and *vanT*<sub>G</sub> genes and the upstream –35 and –10 promoter sequences are in boldface. The ATG start codon of *vanY*<sub>G</sub> is indicated in italics, and the RBS is in boldface and underlined. The sequence of the *P*<sub>YG</sub> promoter region is shown with the putative regions protected by phosphorylated-VanR<sub>G</sub> indicated by brackets (a, b, c) according to Holman et al. (1994).

C. The a, b and c sites were aligned and the deduced consensus sequence (i) was compared with that (ii) of Holman et al. (1994).


RBS (5′-**ATTATGGAGGN**<sub>6</sub>**ATG**-3′) that displayed high complementarity (underlined) to the 3′ extremity of *Bacillus subtilis* 16S rRNA (3′ OH-UCUUUCCUCC) (Moran et al., 1982). As opposed to Tn916 and Tn1549, the *int* and *xis* genes were not present and were replaced by ORFG25 of 1678 bp encoding a putative 560 amino acid protein related to the large resolvase family of site specific recombinases. ORFG25 ended 11 bp in the host chromosomal DNA.

A similar PCR strategy was used to complete the sequence upstream from the *vanG* operon (Fig. 5). The amplification products obtained by various TAIL-PCR from BM4518 or WCH9 total DNA were purified and sequenced. Twenty-three ORFs (Fig. 5) were identified by analysis with the BLASTP program with sizes ranging from 300 to 2334-bp and percentage GC from 36 to 54. These ORFs were in the same orientation, seven deduced sequences showed significant identity with known proteins and seven other ORFs displayed 20–40% identity with those in Tn1549 (Fig. 5).

© 2003 Blackwell Publishing Ltd, *Molecular Microbiology*, **50**, 931–948

940 F. Depardieu, M. G. Bonora, P. E. Reynolds and P. Courvalin

![Diagram](attachment:diagram.png)

Fig. 5. Partial organization of the genetic element from *E. faecalis* BM4518 and WCH9. Open arrows represent coding sequences and indicate the direction of transcription. The primers used for TAIL-PCR are indicated by thin half-arrows. The left and right imperfect inverted repeats (IR<sub>L</sub> and IR<sub>R</sub>) at the ends of the *vanG* cluster are indicated by closed triangles. The percentage of identity of sequences deduced from the ORFs is indicated below each ORF. The ORFs with predicted sequences displaying the highest percentage of identity with those of Tn1549 (Garnier et al., 2000) are represented by striated arrows. The double vertical bar indicates the portion of the element that was not sequenced. The asterisks at both ends of the genetic element indicate the primers used to screen for the presence of a circular intermediate.

### Junction fragments of the *vanG* putative genetic element

Because in BM4518 and WCH9 the portion downstream from ORFG25 encoding a recombinase was identical to

Table 6. Cycling conditions for TAIL-PCR amplification of the 5′- and 3′-flanking regions of the *vanG* operon.<sup>a</sup>

| Reaction   | No. of cycles | Thermal conditions                                                                 |
|------------|---------------|------------------------------------------------------------------------------------|
| Primary    | 1             | 92°C (2 min)                                                                      |
|            | 10            | 92°C (30 s), 52°C (1 min), 68°C (10 min)                                           |
|            | 1             | 92°C (30 s), 25°C (1 min) ramping to 68°C over 3 min, 68°C (10 min)                |
|            | 10            | 92°C (30 s), 35°C (1 min), 68°C (10 min)                                           |
|            | 12            | 92°C (30 s), 52°C (1 min), 68°C (10 min)                                           |
|            |               | 92°C (30 s), 35°C (1 min), 68°C (10 min)                                           |
| Secondary  | 1             | 68°C (7 min)                                                                      |
|            | 1             | 92°C (2 min)                                                                      |
|            | 15            | 92°C (30 s), 52°C (1 min), 68°C (10 min)                                           |
|            |               | 92°C (30 s), 35°C (1 min), 68°C (10 min)                                           |
|            | 1             | 68°C (7 min)                                                                      |
| Tertiary   | 1             | 92°C (2 min)                                                                      |
|            | 20            | 92°C (30 s), 35°C (1 min), 68°C (10 min)                                           |
|            | 1             | 68°C (7 min)                                                                      |

<sup>a</sup> The program used was that of Liu and Whittier (1995) modified to adapt to the polymerase (Expand Long Template PCR system, Roche, Mannheim, Germany) and to a long PCR.

*E. faecalis* V583 sequence (tigr.org/), three primers designed from this sequence were used to amplify the left junction by TAIL-PCR (Fig. 5). A PCR product of c. 7 kb obtained using total DNA from both strains and from the two transconjugants (see below) as a template was purified and sequenced. As in the conjugative transposons, two imperfect 22 bp terminal inverted repeats were present and there was a 4 bp direct repeat of target DNA (Fig. 6).

#### Absence of circular form of the *vanG* putative genetic element in BM4518 and WCH9

Because conjugative transposons Tn916 and Tn1545 and the mobilizable transposons from *Clostridium perfringens* (Tn4451), *C. difficile* (Tn4453, Tn5397), *E. faecium* (Tn5382) and *E. faecalis* (Tn1549), have been shown to produce a circular form (Poyart-Salmeron et al., 1990; Storrs et al., 1991; Carias et al., 1998; Garnier et al., 2000; Wang et al., 2000), PCR was carried out with primers G110 (Table 5 and Fig. 5) and GD42 (Fig. 5, CAGCTTCTGCAATGTAT from position 688071–688087 according to *E. faecalis* V583 sequence numbering)

© 2003 Blackwell Publishing Ltd, *Molecular Microbiology*, **50**, 931–948

Vancomycin resistance in enterococci 941

A
687596 TTTCCACAAAACGACCCACA TTGA ATCTGTCA CGTTATTAACA 687638

Target site

B
← E. faecalis ——| IR-left
TTTCCACAAAACGACCCACA TTGA CGGTGGTACTGCTTTCCACAA AAAGCCAGAC —//— AAGGCGGCTC TTGTGGGAAAAAGGTAATACC GTGA ATCTGTCA CGTTATTAACA |—— E. faecalis →

Fig. 6. Sequence at the integration site into VanG-type *E. faecalis*.
A. Partial *E. faecalis* sequence from the TIGR website (http://www.tigr.org/) and corresponding numbering. The 4 bp target site is in italic bold characters and boxed.
B. Sequence of the junction fragments in BM4518 and WCH9. The 4 bp target duplication is indicated in italic bold characters and boxed. The left and right terminal imperfect 22 bp repeats (IR<sub>L</sub> and IR<sub>R</sub>) of the *vanG* genetic element are in bold characters and complementary nucleotides are underlined.

designed from *E. faecalis* V583 that direct amplification outward from the *vanG* cluster (Fig. 6A). Products with the expected size (788 bp) were obtained only when DNA from *E. faecalis* V583 (VanB-type strain) and JH2-2 (susceptible to glycopeptides) were used as controls. No products were observed with DNA from WCH9 and BM4518 or of the two transconjugants (see below). These results indicate that the large genetic element does not generate a circular intermediate.

Transfer of the chromosomal *vanG* operon

Strains WCH9 and BM4518 were used as donors in filter mating experiments with *E. faecalis* JH2-2 (Table 1). Transfer of vancomycin resistance from strain BM4518 was obtained at a frequency of approximately 2 × 10<sup>-9</sup> per donor only with selection on erythromycin (10 μg ml<sup>-1</sup>) after 48 h of incubation at 37°C. Analysis of the transconjugants indicated that vancomycin and erythromycin resistance were co-transferred in all cases. The frequency of transfer of the *vanG* operon was lower than that usually observed with enterococcal conjugative plasmids and transconjugants BM4522 and BM4523 (Table 1) from two independent experiments were studied further. Transconjugant BM4522 was resistant to vancomycin (MIC = 16 μg ml<sup>-1</sup>) whereas BM4523 showed lower level resistance (MIC = 4 μg ml<sup>-1</sup>) and both remained susceptible to teicoplanin (MIC = 0.5 μg ml<sup>-1</sup>).

The two transconjugants, the BM4518 donor and the JH2-2 recipient were analysed by PFGE after digestion of total DNA by Smal (Fig. 1B), Sfil, Apal or Ascl. Restriction endonuclease Smal was the most discriminatory: each pattern was distinct and differed by three bands from that of JH2-2 (Fig. 1B, left). Although the PFGE patterns of the transconjugants differed from each other by more than one band, a single new fragment hybridized with the *vanG*

probe (Fig. 1B, right). In each strain, the *vanG* operon was inserted in the same Smal fragment of c. 80 kb and generated fragments of c. 310–320 kb.

Analysis by PCR mapping (Fig. 2A) of the two transconjugants confirmed that all the *van* genes, with the expected size, were present in the same order as that in the BM4518 donor (Fig. 2B). The difference in MICs between BM4522 and BM4523 was not due to an insertion or a deletion in the *vanG* operon and no sequence differences in *vanG* or in the regulatory *vanR<sub>G</sub>* and *vanS<sub>G</sub>* genes were found in the transconjugants relative to the donor. The sequence of the erythromycin resistance *ermB* gene in BM4518 and in the transconjugants was identical confirming co-transfer of resistance to glycopeptides and macrolides. Transfer of the *vanG* operon was associated with the movement, from chromosome to chromosome, of genetic elements of c. 240 kb.

Discussion

VanG-type resistance to glycopeptides is conferred on *E. faecalis* by the chromosomal *vanG* operon which includes seven genes which appear to have been recruited from various *van* operons (Fig. 2B). The 5′ portion contains the *vanU<sub>G</sub>*, *vanR<sub>G</sub>* and *vanS<sub>G</sub>* genes which encode a regulatory system (Figs 2B and 3A). *vanR<sub>G</sub>* and *vanS<sub>G</sub>* have the highest homology with the genes for the two-component regulatory system in VanD-type strains (Table 2). *vanY<sub>G</sub>* encodes a polypeptide of only 121 amino acids due to a frameshift mutation. In the absence of the mutation, *vanY<sub>G</sub>* would have encoded a penicillin insensitive, Zn<sup>2+</sup> dependent, D,D-carboxypeptidase with the highest identity to VanY<sub>B</sub> D,D-carboxypeptidase. *vanW<sub>G</sub>* downstream from *vanY<sub>G</sub>* showed 49% identity with *vanW* which is present only in the *vanB* operon (Table 2). The 3′ portion of the *vanG* gene cluster contains *vanG*, *vanXY<sub>G</sub>* and *vanT<sub>G</sub>*

© 2003 Blackwell Publishing Ltd, *Molecular Microbiology*, 50, 931–948

(Fig. 2B) which encode a D-Ala:D-Ser ligase, a bifunctional enzyme with D,D-peptidase activity and a serine racemase, respectively, as in VanC- and VanE-type strains (Arias *et al.*, 2000a; Abadía *et al.*, 2002). The presence of these genes accounts for the synthesis of precursors ending in D-Ala-D-Ser (Table 4) and for inducible serine racemase activity in membrane extracts (Table 3). In spite of the presence in deduced VanG of the EKY motif specific for D-Ala:D-Ser ligases, the *vanG* gene coding for this enzyme was phylogenetically closer to the structural genes for the D-Ala:D-Lac ligases than to those for D-Ala:D-Ser ligases. Thus, strains with VanG-type acquired glycopeptide resistance have several characteristics that distinguish them from other Van-type enterococci.

In *E. faecalis* BM4518 and WCH9, the activities of the VanXY<sub>G</sub> D,D-dipeptidase and D,D-carboxypeptidase (Table 3) are reflected by the amounts of various peptidoglycan precursors (Table 4). The ratio of pentapeptide[D-Ala] to pentapeptide[D-Ser] is dependent on the relative rates of synthesis of D-Ala and D-Ser, the catalytic activities of the D-Ala:D-Ala and D-Ala:D-Ser ligases and the activity of the D,D-peptidase in hydrolysing D-Ala-D-Ala and converting UDP-MurNAc-pentapeptide[Ala] to UDP-MurNAc-tetrapeptide. The large amount of UDP-MurNAc-pentapeptide[Ala] present in the cytoplasm after induction by vancomycin and inhibition of peptidoglycan synthesis by ramoplanin, almost 50% of the total peptidoglycan precursors (Table 4), suggests that the D,D-dipeptidase and D,D-carboxypeptidase activities of VanXY<sub>G</sub> (Table 3) are insufficient either to eliminate D-Ala-D-Ala or to convert UDP-MurNAc-pentapeptide[D-Ala] to UDP-MurNAc-tetrapeptide. The low activity of VanXY<sub>G</sub> as a D,D-carboxypeptidase in BM4518 and WCH9 (Table 3) was confirmed by cloning the gene in a multicopy plasmid (Fig. 2A) (data not shown). VanY<sub>G</sub> was not able to compensate for this lack of activity as a frameshift mutation in the *vanY* gene of strains BM4518 and WCH9 resulted in a truncated polypeptide lacking the active site of a D,D-carboxypeptidase as confirmed by cloning in *E. faecalis* JH2-2 (data not shown). Surprisingly, substantial D,D-carboxypeptidase activity was detected in the membrane fraction (Table 3). This activity which was induced by vancomycin and not inhibited by benzylpenicillin was presumed to be catalysed by a different protein. A similar observation was made with VanD-type strain 10/96 A carrying a truncated VanY<sub>D</sub> D,D-carboxypeptidase (Depardieu *et al.*, 2003). The lack of tetrapeptide in the cytoplasm despite the continuous synthesis of large amounts of pentapeptide[D-Ala] suggests that the active site of this enzyme is external to the cytoplasmic membrane. These strains may have acquired a gene together with its promoter, not present in the *vanD* or *vanG* operon, which encodes a VanY- or VanY<sub>B</sub>-type protein. Recently, it was shown in *E. faecium* that acquisition of resistance to β-lactams was due to selection leading to production of a β-lactam insensitive D,D-carboxypeptidase (Mainardi *et al.*, 2002).

As in other Van-type strains, a membrane-associated sensor kinase (VanS<sub>G</sub>) and a cytoplasmic response regulator (VanR<sub>G</sub>) that acts as a transcriptional activator were present in VanG-type strains. In the two component regulatory systems of VanA- and VanB-type strains, the kinase and phosphatase activities of the VanS and VanS<sub>B</sub> sensors modulate the level of phosphorylation of VanR and VanR<sub>B</sub> regulators respectively (Arthur *et al.*, 1997; 1999; Baptista *et al.*, 1997; Wright *et al.*, 1993). Interestingly, in VanG-type strains, an additional gene *vanU<sub>G</sub>* was present that encoded a predicted transcriptional activator. A protein of this type has not previously been associated with glycopeptide resistance. Northern hybridization showed that *vanU<sub>G</sub>, vanR<sub>G</sub>* and *vanS<sub>G</sub>* were co-transcribed constitutively (Fig. 3A) whereas reverse transcription revealed that *vanY<sub>G</sub>, vanW<sub>G</sub>, vanG, vanXY<sub>G</sub>* and *vanT<sub>G</sub>* were inducibly co-transcribed (Fig. 3B). Analysis of primer extensions revealed that the regulatory (*vanU<sub>G</sub>, vanR<sub>G</sub>* and *vanS<sub>G</sub>*) and resistance (*vanY<sub>G</sub>, vanW<sub>G</sub>, vanG, vanXY<sub>G</sub>* and *vanT<sub>G</sub>*) genes were, respectively, transcribed from the *P<sub>UG</sub>* and *P<sub>YG</sub>* promoters (Figs 2B and 4) and confirmed that the *P<sub>UG</sub>* promoter was constitutively activated whereas *P<sub>YG</sub>* was inducibly activated. Among Van-type strains, this is the first operon that is regulated in this way. The regulatory and resistance genes of VanA, VanB and VanD-type strains are also transcribed from distinct promoters but are coordinately regulated. As an example, in VanA-type strains the promoters are not activated in the absence of VanR and VanS, are induced by glycopeptides when VanR and VanS are present and are constitutively activated by VanR in the absence of VanS (Arthur *et al.*, 1997; 1999).

In BM4147 (VanA) phosphorylation of the VanR response regulator enhances the affinity of the proteins for the regulatory regions of the *P<sub>R</sub>* and *P<sub>H</sub>* promoters and allows transcription of the regulatory and resistance genes, respectively. VanR and phospho-VanR (P-VanR) bind to the regulatory region of the *P<sub>H</sub>* promoter, between *vanS* and *vanH* genes, and of the *P<sub>R</sub>* promoter upstream from *vanR* that contain a single or two putative 12 bp VanR-binding sites respectively (Holman *et al.*, 1994). The sequences of the *P<sub>UG</sub>* and *P<sub>YG</sub>* promoter regions were investigated by comparing them with the 12 bp consensus sequence corresponding to regions protected by VanR and P-VanR, as determined by Holman *et al.* (1994) (Fig. 4C). This analysis revealed three possibilities for binding of VanR<sub>G</sub> or P-VanR<sub>G</sub> in the *P<sub>YG</sub>* promoter region (Fig. 4B). The three regions were aligned and a consensus sequence deduced (Fig. 4C), which was similar to that of Holman *et al.* (1994) (Fig. 4C). In the *P<sub>UG</sub>* promoter region no sequence similar to the consensus sequences was found that could explain constitutive transcription

© 2003 Blackwell Publishing Ltd, *Molecular Microbiology*, **50**, 931–948
(Fig. 4A). Taken together, these results suggest that VanR<sub>G</sub> and P-VanR<sub>G</sub> bind only to the $P_{Y_G}$ promoter region leading to inducible transcription of van$Y_G$, van$W_G$, vanG, van$XY_G$ and van$T_G$.

The vanA and vanB operons are often carried by plasmids or transposable elements (Arthur et al., 1993; Quintiliani and Courvalin, 1996; Carias et al., 1998; Rice et al., 1998; Garnier et al., 2000). The vanG cluster of BM4518 was carried by a large element of c. 240 kb that was transferable by conjugation from chromosome to chromosome at very low frequency. Surprisingly, in all VanG-type strains and the two transconjugants, BM4522 and BM4523, insertion of the vanG cluster always occurred at the same location. Characteristics of conjugative transposons were found in all the strains studied: the large element was flanked by imperfect terminal 22 bp inverted repeats and a 4 bp direct repeat duplication of target DNA was present (Fig. 6). In addition, the left end (17 kb, Fig. 5) of the vanG cluster contained six ORFs that could be involved in conjugative transfer (Fig. 5). The large genetic element containing the vanG operon was organised in three functional regions (Fig. 5): the left extremity with the ORFs putatively implicated in conjugative transfer (17 kb), the central region harbouring the vanG operon (7.9 kb) and the right extremity (4.3 kb). The 46–54% G + C content of the left 17 kb was significantly different from those of the vanG operon (36–43%), the right extremity (38–42%) and the enterococcal chromosome (38%). This suggests that the origin of the left portion of the element is different from that of the two other regions.

Conjugative transposons are genetic elements that encode their own transfer from the genome of a donor cell to the genome of a recipient cell (Scott et al., 1988). The steps in conjugative transposition are excision from the donor DNA followed by circularization of the element and its subsequent transfer to a new host where the transposon inserts. In VanG-type strains, a circular intermediate was not detected indicating a different mechanism for transfer of the vanG cluster or lack of functionality of the conjugative transposon. In the most extensively studied conjugative transposons Tn916 and Tn1545, the products of the transposon-encoded genes int and xis are required for excision (Poyart-Salmeron et al., 1989; Storrs et al., 1991). Despite the presence of ORFG24 with similarity to ORF9 of Tn916, the int and xis genes of Tn916 were not found in the vanG genetic element (Fig. 5). In the position corresponding to the int and xis genes, ORFG25 was present which has the potential to encode a site specific recombinase related to the large resolvase family of recombinases. The members of this family include enzymes involved in the excision of transposons. The residues conserved in recombinases were present in the deduced product of ORFG25 (Crellin and Rood, 1997; Wang et al., 2000). ORFG25 was related to the TndX

Vancomycin resistance in enterococci 943

resolvase of transposon Tn5397 from *C. difficile* (Wang et al., 2000).

A possible explanation for the results obtained could be that the genetic element containing the vanG cluster is not functional and could be a part of a larger element integrating in the recipient by homologous recombination. This would, in particular, account for the apparent identical locus of integration after every conjugation event.

### Experimental procedures

#### Strains, plasmids and growth conditions

The origin and characteristics of the strains and plasmids used in this study are listed in Table 1. *Enterococcus faecalis* WCH9, isolated in 1996 from blood (McKessar et al., 2000), BM4518, and three strains isolated a year later from inpatients at the Princess Alexandra hospital in Brisbane, Queensland, Australia, were resistant to moderate levels of vancomycin and susceptible to teicoplanin (Table 4). *Escherichia coli* Top10 (Invitrogen, Groningen, the Netherlands) was used as a host for recombinant plasmids. *Enterococcus faecalis* JH2-2 is a derivative of strain JH2 that is resistant to fusidic acid and rifampin (Jacob and Hobbs, 1974). Kanamycin (50 µg ml<sup>−1</sup>) was used as a selective agent for cloning PCR products into the pCR-Blunt vector (Invitrogen). Spectinomycin (60 µg ml<sup>−1</sup>) was added to the medium to prevent the loss of plasmids derived from pAT79 (Arthur et al., 1992b). Strains were grown in brain heart infusion (BHI) broth or agar (Difco Laboratories, Detroit, Mich.) at 37°C. The MICs of glycopeptides were determined by the method of Steers et al. (1959) with 10<sup>5</sup> CFU per spot on BHI agar, after 24 h of incubation at 37°C.

#### Recombinant DNA techniques

Plasmid DNA isolation, digestion with restriction endonucleases (Amersham Pharmacia Biotech, Little Chalfont, Buckinghamshire, England and Gibco BRL-life Technologies), amplification of DNA by PCR with Pfu DNA polymerase (Stratagene, La Jolla, California), ligation of DNA fragments with T4 DNA ligase (Amersham Pharmacia Biotech) and transformation of *E. coli* with recombinant plasmid DNA were performed by standard methods (Ausubel et al., 1987). Total DNA from enterococci was prepared according to the method of Le Bouguénec et al. (1990).

#### Plasmid construction

The plasmids were constructed as follows (Fig. 2). For construction of pAT645($P_2$van$Y_G$cat) and pAT646($P_2$van$XY_G$cat), the van$Y_G$ and van$XY_G$ genes of BM4518 were amplified using primer pairs YGNH2-YG<sub>COOH</sub> and XYG<sub>NH2</sub>-XYG<sub>COOH</sub>, respectively, and BM4518 total DNA as template (Fig. 2). Oligodeoxynucleotides YG<sub>NH2</sub> (5′-CAGTGAGCTCGTCAAAG CAAGAGTGCT-3′) and XYG<sub>NH2</sub> (5′-CAGTGAGCTCCA CAGTCGCTATCCAAA-3′) harboured a Sacl restriction site (underlined), and 17 bases complementary to the sequence upstream from van$Y_G$ or van$XY_G$ of BM4518 respectively.
Oligodeoxynucleotides YG${}_{\text{COOH}}$ (5′ AGCGTCTAGATTAGTTTCTGATTTGCT) and XYG${}_{\text{COOH}}$ (5′ AGTGTCTAGATTAGTC ATCTGTTTTCC) contained a Xbal restriction site (underlined), the stop codon (italicized) and 14 bases complementary to the 3′ end sequence of $vanY_{G}$ or $vanXY_{G}$ respectively. The Sacl and Xbal restriction sites allowed directional cloning of $vanY_{G}$ or $vanXY_{G}$ upstream from the cat reporter gene of the shuttle vector pAT79 carrying the constitutive P2 promoter. The 886 and 848 bp inserts of pAT645($P_{2}vanY_{G}\text{cat}$) and pAT646($P_{2}vanXY_{G}\text{cat}$) corresponded, respectively, to nucleotides 2169–3054 and 4940–5787 of strain BM4518, and consisted of the $vanY_{G}$ or $vanXY_{G}$ coding sequences with their RBS, initiation and stop codons. Nucleotide numbering begins at the +1 transcriptional start site of the $vanU_{G}$ gene.

The $vanS_{G}$′-$vanY_{G}$′ and $orfG23$-$vanU_{G}$ intergenic regions were amplified using, respectively, primer pairs SG2-YG3 and G99-G98 specific for *E. faecalis* BM4518 total DNA used as template (Fig. 2B and E). The PCR fragments, with an expected size of 637 and 748 bp, respectively, were cloned into pCR-Blunt leading to plasmids pAT647($vanS_{G}$′-$vanY_{G}$′) and pAT648($orfG23$′-$vanU_{G}$) (Fig. 2E) and were used for primer extension analysis.

### Strain constructions

Plasmids pAT645($P_{2}vanY_{G}\text{cat}$) and pAT646($P_{2}vanXY_{G}\text{cat}$) were introduced into *E. faecalis* JH2-2 by electroporation. Transformants selected on spectinomycin (60 μg ml${}^{-1}$) were screened for resistance to chloramphenicol. Plasmid DNA from chloramphenicol resistant clones was digested with EcoRI plus HindIII and compared to the restriction profiles of pAT645($P_{2}vanY_{G}\text{cat}$) and pAT646($P_{2}vanXY_{G}\text{cat}$) purified from *E. coli* Top10 to screen for DNA rearrangements.

### TAIL-PCR amplification

TAIL-PCR (Liu and Whittier, 1995) was used to determine the sequence of the 5′ and 3′ flanking regions and of internal parts of the $vanG$ operon. The primary PCR mixture contained 0.15 μM gene-specific primer (Table 5), an arbitrary degenerate (AD) primer (Liu and Whittier, 1995) (5 μM for AD 1 and AD 2 or 2.5 μM for AD 3 and AD 4 containing inosine residues), 200 μM of each dNTP, 2.5 U of Expand Long Template enzyme mixture (Roche, Mannheim, Germany) and 1 × PCR buffer with 22.5 mM MgCl${}_{2}$ supplied with the enzyme in a volume of 50 μl. The second PCR was carried out with a second specific primer in combination with the arbitrary primer used in the first PCR. The reaction solution contained 0.2 μM gene-specific primer, an AD primer (3 μM for AD 1 and AD 2 or 1.5 μM for AD 3 and AD 4 containing inosine residues), 200 μM of each dNTP, 2.5 U of Expand Long Template enzyme mixture and 8 μl of a 1/50 dilution of the primary PCR product as template. For the third and fourth PCR, the solution was the same as for the second PCR except that 3 μl of a 1/10 dilution of the second or third PCR product was used as template.

Four TAIL-PCRs were performed (Table 6 and Fig. 5) with the AD 1, AD 2, AD 3 and AD 4 random primers designed by Liu and Whittier (1995). The reaction products of the first, second, third and fourth PCR steps were separated by agarose gel electrophoresis. The bands corresponding to the third and fourth PCRs which showed an expected decrease in length consistent with the positions of the specific primers (Fig. 5 and Table 5) along the genome, were purified and sequenced. In order to rule out mismatches during TAIL-PCR, the nucleotide sequence of the TAIL-PCR products was confirmed by conventional PCR with new primers and sequencing of the products.

### Nucleotide sequencing

Plasmid DNA was extracted with the commercial Wizard Plus Minipreps DNA purification system (Promega, Madison, WI) and the PCR fragments were purified with the microspins of PCR purification kit (Qiagen). Plasmid DNA or PCR products were labelled with a dye-labelled ddNTP Terminator Cycle Sequencing Kit (Beckman Coulter UK Ltd), and the samples were sequenced and analysed with a CEQ 2000 automated sequencer (Beckman).

For primer extension analysis, sequencing was performed by the dideoxynucleotide chain termination method with [$\alpha$-${}^{35}$S]-dATP (Amersham Pharmacia Biotech) and the T7 Sequenase version 2.0 DNA sequencing kit (Amersham).

### Computer analysis of sequence data

Comparison with known proteins was carried out using BLASTN, BLASTX and BLASTP (Altschul et al., 1997) and FASTA (Pearson and Lipman, 1988) from the Genetics Computer Group (GCG) suite of programs.

### Nucleotide sequence accession number

The right extremity (30.704-kb) containing the $vanG$ gene cluster of strain BM4518 was submitted to GenBank and assigned accession n°AY271782. The left extremity (5 kb) of the large genetic element of strain BM4518 was submitted to GenBank and assigned accession n°AY271781.

### Contour-clamped homogeneous electric field gel electrophoresis

Genomic DNA embedded in agarose plugs was digested overnight at 27°C with 25 U of SmaI or for 3 h at 37°C with 0.01 U of I-CeuI, an intron-encoded endonuclease specific for rRNA genes (Liu et al., 1993). Fragments were separated on a 0.8% agarose gel using a CHEF-DRIII system (Bio-Rad Laboratories) under the following conditions: total migration, 27 h for SmaI or 24 h for I-CeuI; initial pulse, 5 s for SmaI or 60 s for I-CeuI; final pulse, 35 s for SmaI or 120 s for I-CeuI; voltage, 6 V cm${}^{-1}$; included angle, 120°; and temperature, 14°C. The DNA fragments were transferred to a nitrocellulose membrane and hybridized successively under stringent conditions at 68°C to an [$\alpha$-${}^{32}$P]-labelled 16S rRNA (*rrs*) probe obtained by amplification of an internal portion of the *rrs* gene with primers RWO1 and DG74 (Greisen et al., 1994) and to a $vanG$ probe obtained by PCR with primers VanG3 (5′-CAGTGAGCTCCTATACGGGAGAAAAGG-3′) and VanG4
(AGCGTCTAGATTTGGATAGCGACTGTG-3′) containing a Sacl and a Xbal restriction site (underlined), respectively, and BM4518 total DNA as template (Fig. 2B). The amplification products used to generate the probes were labelled with [α-³²P] dATP (3000 Ci mmol⁻¹, Amersham Pharmacia Biotech) by megaprime, using a commercially available kit (Amersham).

### Filter mating

To study the transfer of the *vanG* operon, WCH9 and BM4518 donors and JH2-2 recipient were grown overnight in BHI broth. Samples of each donor and of the recipient were spread on a filter placed on the surface of an agar plate and incubated overnight at 37°C. After the mating period, bacteria were resuspended in 800 μl of BHI broth and plated on selective media. Transconjugants were selected on media containing erythromycin (10 μg ml⁻¹) supplemented with rifampicin (20 μg ml⁻¹) and fusidic acid (10 μg ml⁻¹) to counterselect donor strains.

### RNA techniques

#### Extraction of total RNA

Enterococci grown in BHI (100 ml) at 37°C to an optical density at 600 nm of 0.8, were pelleted and frozen immediately. Frozen cells were resuspended in 0.5 ml of water treated with diethyl pyrocarbonate (DEPC, Sigma) and disrupted with a FastPrep disintegrator (30 s at 4°C) using 0.5 g of glass beads (106 μm; Sigma) in the presence of 0.4 ml of 4% Macaloid and 0.5 ml of phenol. After centrifugation for 2 min at 15 000 r.p.m., supernatants were extracted with phenol/chloroform (1:1, v/v), total RNA was precipitated by addition of 0.6 ml isopropanol in the presence of 50 μl NaCl (2 M) and the RNA pellets were resuspended in 50 μl water treated with DEPC. RNA concentrations were determined by measuring absorbance at 260 nm.

#### Northern analysis

Equal amounts of total RNA (20 μg) were separated under denaturing conditions in a 1.2% agarose formaldehyde-morpholinepropanesulphonic acid gel, stained with ethidium bromide, and blotted onto Hybond N⁺ membranes (Amersham Pharmacia Biotech). The PCR products used as probes were obtained with total DNA from BM4518 as template and the primers indicated in Fig. 2(C) and labelled with [α-³²P] dCTP (3000 Ci nmol⁻¹, Amersham Pharmacia Biotech) by using the Megaprime DNA labeling kit (Amersham Pharmacia Biotech). Hybridization was carried out under stringent conditions, and washes were performed twice with 2× SSC (0.3 M NaCl plus 30 mM sodium citrate) and 0.5% sodium dodecyl sulphate (SDS) at 60°C for 15 min. The size of the transcripts was determined by comparison with RNA molecular weight marker I (Boehringer Mannheim, Germany).

#### RT-PCR experiments

Total RNA samples (50 μg) were digested with RNase-free DNasel (250 U) (Amersham Pharmacia Biotech) in a final volume of 1 ml for 10 min at 37°C. Samples were treated with phenol-chloroform, and precipitated with ethanol. Reverse transcription (RT) was carried out

(final volume 20 μl) with 2 μg of purified RNA, 1× enzyme buffer (Superscript II kit, Invitrogen), 50 mM MgCl₂, 2 μg of bovine serum albumin (New England Biolabs, Beverly, Mass.) per ml, 1 mM each of the four deoxyribonucleotide triphosphates (Amersham Pharmacia Biotech), 50 pmol of the SG9, XYG2 or TG14 primers (Fig. 2D and Table 5), 20 U of RNase inhibitor (RNA guard, Amersham Pharmacia Biotech) and 200 U of Moloney murine leukemia virus modified reverse transcriptase. Samples were incubated for 30 min at 37°C, and the enzyme was inactivated at 95°C for 5 min. The DNA products were amplified by PCR in an 80-μl final volume reaction containing the previous 20 μl samples, 50 pmol each of the G96-SG3, YG7-G4-1 and G3-TG7 pair of primers (Fig. 2D and Table 5), 1× enzyme buffer (Amersham Pharmacia Biotech) and 2 U of Taq DNA polymerase (Amersham Pharmacia Biotech); PCR (30 cycles) was performed in a Gene Amp PCR system 4800 (Perkin Emer Cetus, Norwalk, Conn.) and the products were separated on a 1% agarose gel, purified and sequenced.

#### Primer extension analysis

The synthetic oligodeoxynucleotides PRG (5′-ATGTGCTTTTACTTACTCCAACAGCCTCA CGAAGCTCGCC-3′) and PYG (5′-TTGCCGCATTGCCAAA ACCGTACCATAAAAAACTGCTTGC-3′) were 5′ end labelled with [γ-³³P]-ATP (Amersham Pharmacia Biotech) and T4 polynucleotide kinase (Biolabs). The enzyme was inactivated for 15 min at 65°C. Labelled primers (1 pmol) were denatured at 95°C for 2 min and incubated in the presence of 25 μg total RNA at 70°C for 10 min. Annealing of labelled primers with RNA was carried out at 50°C for 2 min in a mixture of 50 μl final volume composed of 1× ‘first strand buffer’ (Invitrogen), a solution of 0.5 mM of the four dNTPS (Amersham), 10 mM DTT (Invitrogen) and 40 U RNasin (Amersham). The extension reaction was performed with 40 U of avian myeloblastosis virus reverse transcriptase (Boehringer) for 45 min at 50°C. After phenol-chloroform extraction, the reaction product was precipitated with ethanol, redissolved in 10 μl of sterile water and 10 μl of a stop solution added (Amersham). After denaturation by heat, 4 μl samples were loaded onto 6% polyacrylamide-urea sequencing gels for electrophoresis. Sequencing reactions using primers PRG and PYG and plasmid pAT648(pCR-BluntΩ*P*~U~*~G~*,*vanU*~G~,*vanR*~G~′) or pAT647(pCR-BluntΩ*vanS*~G~′,*vanY*~G~′) (Fig., 2E) DNA as template were run in parallel to allow determination of the endpoints of the extension products.

### Analysis of peptidoglycan precursors

Extraction and analysis of peptidoglycan precursors was performed as described (Messer and Reynolds, 1992; Reynolds *et al.*, 1994a). Enterococci were grown in BHI medium in the absence or presence of vancomycin (4 μg ml⁻¹) to the midexponential phase (*A*~600~ = 1). Ramoplanin (3 μg ml⁻¹) was added to inhibit peptidoglycan synthesis and incubation was continued for 15 min to cause accumulation of peptidoglycan precursors. The bacteria were harvested and the cytoplasmic precursors were extracted with 8% trichloroacetic acid (15 min, at 4°C), desalted and analysed by high-performance liquid chromatography. Results were expressed as the percentages of total late peptidoglycan precursors represented by UDP-MurNAc-tetrapeptide and UDP-MurNAc-
pentapeptide[D-Ala] or [D-Ser] that were determined from the integrated peak areas.

**D,D-dipeptidase (VanX) and D,D-carboxypeptidase (VanY) activities**

The enzymic activities in the supernatant and in the resuspended pellet fraction were assayed as described (Arthur *et al.*, 1996b; Reynolds *et al.*, 2001). Strains were grown until the optical density at 600 nm reached 0.7 in the absence or presence of vancomycin (8 μg ml⁻¹) for induction of WCH9 and BM4518 or with spectinomycin (60 μg ml⁻¹) to counterselect loss of derivatives of pAT79. Bacteria were harvested, lysed by treatment with lysozyme (2 mg ml⁻¹) at 37°C followed by sonication, and the membrane fraction was pelleted (100 000 g, 45 min). The supernatant (S100) and resuspended pellet (C100) were collected and assayed for D,D-peptidase (VanX or VanY) activity by measuring the D-Ala released from substrate hydrolysis (D-Ala-D-Ala, 6.56 mM or L-Ala-γ-D-Glu-L-Lys-D-Ala-D-Ala, 5 mM) through coupled indicator reactions using D-amino acid oxidase and horse radish peroxidase (Arthur *et al.*, 1996b; Reynolds *et al.*, 2001). Specific activity was defined as the number of nanomoles of product formed at 37°C per minute per milligram of protein contained in the extracts.

**Assay of serine or alanine racemase (VanT<sub>G</sub>) activity**

The activities were determined from suitable dilutions of cytoplasmic and membrane fractions in a final volume of 45 μl. The assay mixture contained Bis Trispropane (50 mM, pH 7.5), 10 mM L-serine or L-alanine and 15 μl of the diluted fraction as enzyme preparation and was incubated at 37°C for 30 min. D-amino acids produced by racemase activity were assayed using a D-amino acid oxidase assay (Messer and Reynolds, 1992) with D-serine or D-alanine as standards. Protein concentration was determined according to the method of Bradford (1976) with bovine serum albumin as standard.

**Acknowledgements**

We thank G. Nimmo and J. Bell for the gift of strains and P. Trieu-Cuot for helpful discussions. This work was supported in part by the ‘Programme de Recherche Fondamentale en Microbiologie, Maladies Infectieuses et Parasitaires’ from the Ministère de l’Education Nationale de la Recherche et de la Technologie. P.E.R. thanks the Leverhulme trust for an Emeritus Fellowship.

**References**

Arias, C.A., Martin-Martinez, M., Blundell, T.L., Arthur, M., Courvalin, P., and Reynolds, P.E. (1999) Characterization and modelling of VanT: a novel, membrane-bound, serine racemase from vancomycin-resistant *Enterococcus gallinarum* BM4174. *Mol Microbiol* **31**: 1653–1664.

Arias, C.A., Courvalin, P., and Reynolds, P.E. (2000a) vanC cluster of vancomycin-resistant *Enterococcus gallinarum* BM4174. *Antimicrob Agents Chemother* **44**: 1660–1666.

Arias, C.A., Weisner, J., Blackburn, J.M., and Reynolds, P.E. (2000b) Serine and alanine racemase activities of VanT: a protein necessary for vancomycin resistance in *Enterococcus gallinarum* BM4174. *Microbiology* **146**: 1727–1734.

Arthur, M., Molinas, C., Bugg, T.D.H., Wright, G.D., Walsh, C.T., and Courvalin, P. (1992a) Evidence for *in vivo* incorporation of D-lactate into peptidoglycan precursors of vancomycin-resistant enterococci. *Antimicrob Agents Chemother* **36**: 867–869.

Arthur, M., Molinas, C., and Courvalin, P. (1992b) The VanS-VanR two-component regulatory system controls synthesis of depsipeptide peptidoglycan precursors in *Enterococcus faecium* BM4147. *J Bacteriol* **174**: 2582–2591.

Arthur, M., Molinas, C., Depardieu, F., and Courvalin, P. (1993) Characterization of Tn1546, a Tn3-related transposon conferring glycopeptide resistance by synthesis of depsipeptide peptidoglycan precursors in *Enterococcus faecium* BM4147. *J Bacteriol* **175**: 117–127.

Arthur, M., Reynolds, P., and Courvalin, P. (1996a) Glycopeptide resistance in enterococci. *Trends Microbiol* **4**: 401–407.

Arthur, M., Depardieu, F., Reynolds, P., and Courvalin, P. (1996b) Quantitative analysis of the metabolism of soluble cytoplasmic peptidoglycan precursors of glycopeptide-resistant enterococci. *Mol Microbiol* **21**: 33–44.

Arthur, M., Depardieu, F., Gerbaud, G., Galimand, M., Leclercq, R., and Courvalin, P. (1997) The VanS sensor negatively controls VanR-mediated transcriptional activation of glycopeptide resistance genes of Tn1546 and related elements in the absence of induction. *J Bacteriol* **179**: 97–106.

Arthur, M., Depardieu, F., Cabanié, L., Reynolds, P., and Courvalin, P. (1998) Requirement of the VanY and VanX D,D-peptidases for glycopeptide resistance in enterococci. *Mol Microbiol* **30**: 819–830.

Arthur, M., Depardieu, F., and Courvalin, P. (1999) Regulated interactions between partner and non-partner sensors and responses regulators that control glycopeptide resistance gene expression in Enterococci. *Microbiology* **145**: 1849–1858.

Ausubel, F.M., Brent, R., Kingston, R.E., Moore, D.D., Seidman, J.G., Smith, J.A., and Struhl, K. (1987) *Current Protocols in Molecular Biology*. New York: John Wiley and Sons.

Baptista, M., Depardieu, F., Reynolds, P., Courvalin, P., and Arthur, M. (1997) Mutations leading to increased levels of resistance to glycopeptide antibiotics in VanB-type enterococci. *Mol Microbiol* **25**: 93–105.

Boyd, D.A., Conly, J., Dedier, H., Peters, G., Robertson, L., Slater, E., and Mulvey, M.R. (2000) Molecular characterization of the vanD gene cluster and a novel insertion element in a vancomycin-resistant enterococcus isolated in Canada. *J Clin Microbiol* **38**: 2392–2394.

Bradford, M. (1976) A rapid and sensitive method for the quantitation of microgram quantities of proteins utilizing the principle of protein-dye binding. *Anal Biochem* 72: 248–254.

Carias, L.L., Rudin, S.D., Donskey, C.J., and Rice, L.B. (1998) Genetic linkage and cotransfer of a novel, vanB-containing transposon (Tn5382) and a low-affinity penicillin-binding protein 5 gene in a clinical vancomycin-resistant *Enterococcus faecium* isolate. *J Bacteriol* 180: 4426–4434.

Casadewall, B., and Courvalin, P. (1999) Characterization of the vanD glycopeptide resistance gene cluster from *Enterococcus faecium* BM4339. *J Bacteriol* 181: 3644–3648.

Crellin, P.K., and Rood, J.I. (1997) The resolvase/invertase domain of the site-specific recombinase TnpX is functional and recognizes a target sequence that resembles the junction of the circular form of the *Clostridium perfringens* transposon Tn4451. *J Bacteriol* 179: 5148–5156.

Depardieu, F., Reynolds, P.E., and Courvalin, P. (2003) VanD-type vancomycin-resistant *Enterococcus faecium* 10/96A. *Antimicrob Agents Chemother* 47: 7–18.

Evers, S., and Courvalin, P. (1996) Regulation of VanB-type vancomycin resistance gene expression by the VanS<sub>B</sub>-VanR<sub>B</sub> two-component regulatory system in *Enterococcus faecalis* V583. *J Bacteriol* 178: 1302–1309.

Fines, M., Perichon, B., Reynolds, P., Sahm, D.F., and Courvalin, P. (1999) VanE, a new type of acquired glycopeptide resistance in *Enterococcus faecalis* BM4405. *Antimicrob Agents Chemother* 43: 2161–2164.

Garnier, F., Taourit, S., Glaser, P., Courvalin, P., and Galimand, M. (2000) Characterization of transposon Tn1549, conferring VanB-type resistance in *Enterococcus* spp. *Microbiology* 146: 1481–1489.

Greisen, K., Loeffelholz, M., Purohit, A., and Leong, D. (1994) PCR primers and probes for the 16S rRNA gene of most species of pathogenic bacteria, including bacteria found in cerebrospinal fluid. *J Clin Microbiol* 32: 335–351.

Haldimann, A., Fisher, S.L., Daniels, L.L., Walsh, C.T., and Wanner, B.L. (1997) Transcriptional regulation of the *Enterococcus faecium* BM4147 vancomycin resistance gene cluster by the VanS-VanR two-component regulatory system in *Escherichia coli* K-12. *J Bacteriol* 179: 5903–5913.

Holman, T.R., Wu, Z., Wanner, B.L., and Walsh, C.T. (1994) Identification of the DNA-binding site for the phosphorylated VanR protein required for vancomycin resistance in *Enterococcus faecium*. *Biochemistry* 33: 4625–4631.

Jacob, A.E., and Hobbs, S.J. (1974) Conjugal transfer of plasmid-borne multiple antibiotic resistance in *Streptococcus faecalis* var. *Zymogenes*. *J Bacteriol* 117: 360–372.

Le Bouguédec, C., de Cespédès, G., and Horaud, T. (1990) Presence of chromosomal elements resembling the composite structure Tn3701 streptococci. *J Bacteriol* 172: 727–734.

Lessard, I.A., and Walsh, C.T. (1999) Mutational analysis of active-site residues of the enterococcal D-Ala-D-Ala dipeptidase VanX and comparison with *Escherichia coli* D-Ala-D-Ala ligase and D-Ala-D-Ala carboxypeptidase VanY. *Chem Biol* 6: 177–187.

Liu, Y.G., and Whittier, R.F. (1995) Thermal asymmetric interlaced PCR: automatable amplification and sequencing

of insert end fragments from P1 and YAC clones for chromosome walking. *Genomics* 25: 674–681.

Liu, S.L., Hessel, A., and Sanderson, K.E. (1993) Genomic mapping with I-Ceu I, an intron-encoded endonuclease specific for genes for ribosomal RNA. *Salmonella* spp., *Escherichia coli*, and other bacteria. *Proc Natl Acad Sci USA* 90: 6874–6878.

Mainardi, J.L., Morel, V., Fourgeaud, M., Cremniter, J., Blanot, D., Legrand, R., et al. (2002) Balance between two transpeptidation mechanisms determines the expression of beta-lactam resistance in *Enterococcus faecium*. *J Biol Chem* 277: 35801–35807.

McCafferty, D.G., Lessard, I.A., and Walsh, C.T. (1997) Mutational analysis of potential zinc-binding residues in the active site of the enterococcal D-Ala-D-Ala dipeptidase VanX. *Biochemistry* 36: 10498–10505.

McKessar, S.J., Berry, A.M., Bell, J.M., Turnidge, J.D., and Paton, J.C. (2000) Genetic characterization of vanG, a novel vancomycin resistance locus of *Enterococcus faecalis*. *Antimicrob Agents Chemother* 44: 3224–3228.

Messer, J., and Reynolds, P.E. (1992) Modified peptidoglycan precursors produced by glycopeptide-resistant enterococci. *FEMS Microbiol Lett* 94: 195–200.

Moran, C.P.J., Lang, N., LeGrice, S.F.J., Lee, G., Stephens, M., Sonenshein, A.L., et al. (1982) Nucleotide sequences that signal the initiation of transcription and translation in *Bacillus subtilis*. *Mol Gen Genet* 186: 339–346.

Parkinson, J.S., and Kofoid, E.C. (1992) Communication modules in bacterial signaling proteins. *Annu Rev Genet* 26: 71–112.

Pearson, W.R., and Lipman, D.J. (1988) Improved tools for biological sequence comparison. *Proc Natl Acad Sci USA* 85: 2444–2448.

Perichon, B., and Courvalin, P. (2000) Update on vancomycin resistance. *Int J Clin Pract* 54: 88–93.

Poyart-Salmeron, C., Trieu-Cuot, P., Carlier, C., and Courvalin, P. (1989) Molecular characterization of two proteins involved in the excision of the pneumococcal transposon Tn1545: homologies with other site-specific recombinases. *EMBO J* 8: 2425–2433.

Poyart-Salmeron, C., Trieu-Cuot, P., Carlier, C., and Courvalin, P. (1990) The integration-excision system of the streptococcal transposon Tn1545 is structurally and functionally related to those lambdaoid phages. *Mol Microbiol* 4: 1513–1521.

Quintiliani, R., Jr, and Courvalin, P. (1996) Characterization of Tn1547, a composite transposon flanked by the IS16 and IS256-like elements, that confers vancomycin resistance in *Enterococcus faecalis* BM4281. *Gene* 172: 1–8.

Reynolds, P.E. (1989) Structure, biochemistry and mechanism of action of glycopeptide antibiotics. *Eur J Clin Microbiol Infect Dis* 8: 943–950.

Reynolds, P.E. (1998) Control of peptidoglycan synthesis in vancomycin-resistant enterococci: D,D-peptidases and D,D-carboxypeptidases. *Cell Mol Life Sci* 54: 325–331.

Reynolds, P.E., Snaith, H.A., Maguire, A.J., Dutka-Malen, S., and Courvalin, P. (1994a) Analysis of peptidoglycan precursors in vancomycin-resistant *Enterococcus gallinarum* BM4174. *Biochem J* 301: 5–8.

Reynolds, P.E., Depardieu, F., Dutka-Malen, S., Arthur, M.,

and Courvalin, P. (1994b) Glycopeptide resistance mediated by enterococcal transposon Tn1546 requires production of VanX for hydrolysis of D-alanyl-D-alanine. *Mol Microbiol* 13: 1065–1070.

Reynolds, P.E., Arias, C.A., and Courvalin, P. (1999) Gene vanXYC encodes D,D-dipeptidase (VanX) and D,D-carboxypeptidase (VanY) activities in vancomycin-resistant Enterococcus gallinarum BM4174. *Mol Microbiol* 34: 341–349.

Reynolds, P.E., Ambur, O.H., Casadewall, B., and Courvalin, P. (2001) The VanY (D) DD-carboxypeptidase of Enterococcus faecium BM4339 is a penicillin-binding protein. *Microbiology* 147: 2571–2578.

Rice, L.B., Carias, L.L., Donskey, C.L., and Rudin, S.D. (1998) Transferable, plasmid-mediated vanB-type glycopeptide resistance in *Enterococcus faecium*. *Antimicrob Agents Chemother* 42: 963–964.

Scott, J.R., Kirchman, P.A., and Caparon, M.G. (1988) An intermediate in transposition of the conjugative transposon Tn916. *Proc Natl Acad Sci USA* 85: 4809–4813.

Steers, E., Foltz, E.L., Graves, B.S., and Riden, J. (1959) An inocula replicating apparatus for routine testing of bacterial susceptibility to antibiotics. *Antibiot Chemother (Basel)* 9: 307–311.

Storrs, M.J., Poyart-Salmeron, C., Trieu-Cuot, P., and Courvalin, P. (1991) Conjugative transposition of Tn916 requires the excisive and integrative activities of the transposon-encoded integrase. *J Bacteriol* 173: 4347–4352.

Wang, H., Roberts, A.P., Lyras, D., Rood, J.I., Wilks, M., and Mullany, P. (2000) Characterization of the ends and target sites of the novel conjugative transposon Tn5397 from *Clostridium difficile*: excision and circularization is mediated by the large resolvase, TndX. *J Bacteriol* 182: 3775–3783.

Wright, G.D., Holman, T.R., and Walsh, C.T. (1993) Purification and characterization of VanR and the cytosolic domain of VanS: a two-component regulatory system required for vancomycin resistance in *Enterococcus faecium* BM4147. *Biochemistry* 32: 5057–5063.
