
Genome Analysis

# Genome-Wide Characterization of the Lignification Toolbox in Arabidopsis${}^{1[w]}$

Jeroen Raes${}^{2}$, Antje Rohde${}^{2}$, Jørgen Holst Christensen, Yves Van de Peer, and Wout Boerjan*

Department of Plant Systems Biology, Flanders Interuniversity Institute for Biotechnology, Ghent University, Technologiepark 927, B–9052 Gent, Belgium

Lignin, one of the most abundant terrestrial biopolymers, is indispensable for plant structure and defense. With the availability of the full genome sequence, large collections of insertion mutants, and functional genomics tools, Arabidopsis constitutes an excellent model system to profoundly unravel the monolignol biosynthetic pathway. In a genome-wide bioinformatics survey of the Arabidopsis genome, 34 candidate genes were annotated that encode genes homologous to the 10 presently known enzymes of the monolignol biosynthesis pathway, nine of which have not been described before. By combining evolutionary analysis of these 10 gene families with in silico promoter analysis and expression data (from a reverse transcription-polymerase chain reaction analysis on an extensive tissue panel, mining of expressed sequence tags from publicly available resources, and assembling expression data from literature), 12 genes could be pinpointed as the most likely candidates for a role in vascular lignification. Furthermore, a possible novel link was detected between the presence of the AC regulatory promoter element and the biosynthesis of G lignin during vascular development. Together, these data describe the full complement of monolignol biosynthesis genes in Arabidopsis, provide a unified nomenclature, and serve as a basis for further functional studies.

Lignin is an aromatic heteropolymer that is mainly present in secondary thickened plant cells, where it provides rigidity and impermeability to the cell walls. In addition, lignin deposition may be induced upon wounding and infection to protect plant tissues against invading pathogens. Lignin is composed of different phenylpropanoids, predominantly the monolignols *p*-coumaryl, coniferyl, and sinapyl alcohols that differ in their degree of methoxylation (Fig. 1). When these monolignols are incorporated into lignin, they are called *p*-hydroxyphenyl (H), guaiacyl (G), and syringyl (S) units, respectively. In addition to the three monolignols, other phenylpropanoids, such as hydroxycinnamyl aldehydes, hydroxycinnamyl acetates, hydroxycinnamyl *p*-hydroxybenzoates, hydroxycinnamyl *p*-coumarates, and hydroxycinnamate esters, are also present in the polymer (Ralph et al., 2001; Boerjan et al., 2003). Considerable variation exists in lignin composition between taxa, cell types, and developmental and environmental conditions.

Over the last decade, there has been a tremendous effort in cloning new genes involved in the monolignol biosynthetic pathway and in tackling the enzyme kinetics of the corresponding proteins and the role these enzymes play in controlling the amount and composition of lignin to be deposited in the cell wall (Anterola and Lewis, 2002; Humphreys and Chapple, 2002; Boerjan et al., 2003). As a consequence, the monolignol biosynthetic pathway has virtually been rewritten, although the exact route toward the monolignols is still a matter of debate (Fig. 1).

Although enzymatic assays and transgenic plants have contributed extensively to our understanding of the in vivo role of the enzymes, the role of individual gene family members has been more difficult to tackle, a limitation that can only be overcome in plant species such as Arabidopsis, for which the genome sequence and efficient reverse genetics tools are available (Arabidopsis Genome Initiative [AGI], 2000). Furthermore, the advent of genome-wide microarrays will make it possible to study the transcriptional differences that are the consequence of single gene perturbations and will allow the often pleiotropic phenotype of particular mutants to be explained at the molecular level.

As a first step toward studying the role of individual family members, we have undertaken a bioinformatics approach to identify, in Arabidopsis, all the gene family members of all monolignol biosynthesis genes known today. In many cases, only a subset of a given gene family, mostly obtained by homology-based gene isolation, has been characterized in the past. As a consequence, more distant family members might not have been discovered when, for example, primers were designed on only a few members of the family. This has led to an important bias in the range of sequence data available in public databases.

---

1 This work was supported in part by the European Commission programs EDEN (grant no. QLK5-CT-2001–00443) and by the Fund for Scientific Research–Flanders (postdoctoral fellowship to A.R.).

2 These authors contributed equally to the paper.

[w] The online version of this article contains Web-only data.

* Corresponding author; e-mail wout.boerjan@psb.ugent.be; fax 32–9–331–3809.

Article, publication date, and citation information can be found at www.plantphysiol.org/cgi/doi/10.1104/pp.103.026484.

Raes et al.

![image](https://i.imgur.com/1234567.png)

Figure 1. The monolignol biosynthetic pathway. All the enzymatic reactions presented in the pathway have been demonstrated at least in vitro. Because of the variety in isoenzymes and kinetic properties, alternative routes through the metabolic pathway may exist. A question mark after an enzyme name means that the substrate has not been tested yet with this enzyme. For reactions with a single question mark, direct conversion has been detected, but the respective enzyme is unknown, whereas for those with a double question mark, no direct conversion has been detected.

Here, we have used sensitive computational methods to delineate, in Arabidopsis, all members of the gene families currently known to be involved in monolignol biosynthesis. The integration of expression studies and promoter sequence analyses of the individual family members with phylogenetic analysis of the family has allowed us to select 12 genes as the most likely candidates to be involved in the developmental lignification in vascular tissues. Importantly, the promoter comparisons revealed a possible link between G lignin biosynthesis and the presence of the AC element that is correlated with a strong xylem expression. Together, these data describe the full complement of monolignol biosynthesis genes in Arabidopsis, introduce a unifying nomenclature for all genes of the pathway (Table I), and serve as a basis for further functional studies.

---

RESULTS

A semiautomatic structural annotation and a phylogeny-based classification were performed using prediction results, experimental data, and information from homologous sequences (see “Materials and Methods”). A total of 34 candidate monolignol biosynthesis genes were annotated, of which nine had, to our knowledge, never been described before (Table I). In addition, 27 closely related superfamily members (“likes”) were identified in this process (Table I). To get a first insight into whether all these genes are indeed expressed and, more importantly, whether their expression pattern correlates with developmental lignification, their expression was analyzed in a set of tissues and for six developmental stages of inflorescence stem known to contain a high

Downloaded from www.plantphysiol.org on February 18, 2015 - Published by www.plantphysiol.org Plant Physiol. Vol. 133, 2003  
Copyright © 2003 American Society of Plant Biologists. All rights reserved.

Table I. Unifying nomenclature for gene families investigated in this study

Nomenclature was chosen to accommodate as much as possible previously published names. For explanation, see text.

| Gene Family | Gene | AGI No. | Other Names |
|-------------|------|---------|--------------|
| Phe ammonia lyase (PAL) | Arath; PAL 1 | At2g37040 | PAL 1 |
|               | Arath; PAL 2 | At3g53260 | PAL 2 |
|               | Arath; PAL 3 | At5g04230 | PAL 3 |
|               | Arath; PAL 4 | At3g10340 | — |
| Trans-cinnamate 4-hydroxylase (C4H) | Arath; C4H | At2g30490 | CYP73A 5 |
| Class I | — | — | — |
| Class II | — | — | — |
| 4-Coumarate:CoA ligase (4CL) | Arath; 4CL 1 | At1g51680 | 4CL1 |
| Class I | Arath; 4CL 2 | At3g21240 | 4CL2 |
|               | Arath; 4CL 4 | At3g21230 | — |
| Class II | Arath; 4CL 3 | At1g65060 | 4CL3 |
| Class 4CL likes | Arath; 4CL-like 1 | At1g20510 | — |
|               | Arath; 4CL-like 2 | At1g20500 | — |
|               | Arath; 4CL-like 3 | At1g20490 | — |
|               | Arath; 4CL-like 4 | At1g20480 | — |
|               | Arath; 4CL-like 5 | At1g62940 | — |
|               | Arath; 4CL-like 6 | At4g19010 | — |
|               | Arath; 4CL-like 7 | At4g05160 | — |
|               | Arath; 4CL-like 8 | At5g63380 | — |
|               | Arath; 4CL-like 9 | At5g38120 | — |
| Hydroxycinnamoyl-CoA:shikimate/quinate hydroxycinnamoyltransferase (HCT) | Arath; HCT | At5g48930 | — |
| p-Coumarate 3-hydroxylase (C3H) | Arath; C3H 1 | At2g40890 | CYP98A 3 |
| Class I | Arath; C3H 2 | At1g74540 | CYP98A 8 |
| Class II | Arath; C3H 3 | At1g74550 | CYP98A 9 |
| Caffeoyl-CoA 3-O-methyltransferase (CCoAOMT) | Arath; CCoAOMT 1 | At4g34050 | CCoAOMT |
| Class I | Arath; CCoAOMT 2 | At1g24735 | — |
| Class II | Arath; CCoAOMT 3 | At3g61990 | — |
|               | Arath; CCoAOMT 4 | At3g62000 | — |
|               | Arath; CCoAOMT 5 | At1g67990 | CCoAOMT |
|               | Arath; CCoAOMT 6 | At1g67980 | — |
|               | Arath; CCoAOMT 7 | At4g26220 | — |
| Cinnamoyl-CoA reductase (CCR) | Arath; CCR 1 | At1g15950 | — |
| Class I | Arath; CCR 2 | At1g80820 | — |
| Class CCR likes | Arath; CCR-like 1 | At1g76470 | — |
|               | Arath; CCR-like 2 | At2g02400 | — |
|               | Arath; CCR-like 3 | At2g33590 | — |
|               | Arath; CCR-like 4 | At2g33600 | — |
|               | Arath; CCR-like 5 | At5g58490 | — |
| Ferulate 5-hydroxylase (F5H) | Arath; F5H 1 | At4g36220 | CYP84A 1 |
|               | Arath; F5H 2 | At5g04330 | CYP84A 4 |
| Caffeic acid O-methyltransferase (COMT) | Arath; COMT | At5g54160 | OMT 1 |
| Class I | Arath; COMT-like 1 | At1g21100 | — |
| Class COMT likes | Arath; COMT-like 2 | At1g21110 | — |
|               | Arath; COMT-like 3 | At1g21120 | — |
|               | Arath; COMT-like 4 | At1g21130 | — |
|               | Arath; COMT-like 5 | At1g33030 | — |
|               | Arath; COMT-like 6 | At1g51990 | — |
|               | Arath; COMT-like 7 | At1g63140 | — |
|               | Arath; COMT-like 8 | At1g76790 | — |
|               | Arath; COMT-like 9 | At1g77520 | — |
|               | Arath; COMT-like 10 | At1g77530 | — |
|               | Arath; COMT-like 11 | At3g53140 | — |
|               | Arath; COMT-like 12 | At5g37170 | — |
|               | Arath; COMT-like 13 | At5g53810 | — |
| Cinnamyl alcohol dehydrogenase (CAD) | Arath; CAD 2 | At3g19450 | LCAD-C |
| Class I | Arath; CAD 6 | At4g34230 | LCAD-D |
| Class II | Arath; CAD 3 | At4g37970 | LCAD-A |
|               | Arath; CAD 4 | At4g37980 | LCAD-B, ELI 3-1 |
| Class III | Arath; CAD 5 | At4g37990 | ELI 3-2, BAD |
|               | Arath; CAD 1 | At4g39330 | CAD1 |
|               | Arath; CAD 7 | At2g21730 | LCAD-E |
|               | Arath; CAD 8 | At2g21890 | LCAD-F |
| Not classified | Arath; CAD 9 | At1g72680 | — |

Raes et al.

portion of lignifying cells. These data were compared with previous expression data from Arabidopsis and with information extracted from public databases of expressed sequence tag (EST). In addition, putative promoter elements, which drive expression during lignification, in pathogen and wound responses, and after induction by stress-related hormones, and potential subcellular localization signals were identified (due to size limitations, tables compiling all these data are available as supplemental data and at http://www.psb.ugent.be/bioinformatics/lignin/ and are indexed by an “s” throughout the manuscript).

---

### PAL

PAL (E.C. 4.3.1.5) is the first enzyme of the general phenylpropanoid pathway and catalyzes the nonoxidative deamination of Phe to trans-cinnamic acid and NH₃ (Fig. 1). PAL mediates the influx from primary metabolism into the phenylpropanoid pathway and becomes rate limiting when its activity is reduced below a threshold of 20% to 25% in transgenic tobacco (*Nicotiana tabacum*; Bate et al., 1994; Sewalt et al., 1997).

By using a thorough semiautomated annotation method, four genes encoding PAL proteins were detected in the Arabidopsis genome, three of which have been described previously (Ohl et al., 1990; Wanner et al., 1995). The phylogenetic analysis of PAL genes from various species provided no evidence for different classes in the PAL gene family (Fig. 2), although *PAL1* is most closely related to *PAL2*, and *PAL3* always clusters together with *PAL4* (data not shown). The duplication that created the two PAL groups (*PAL1* and *PAL2* and *PAL3* and *PAL4*) in Arabidopsis has been postulated to have predated the monocot-dicot split (Wanner et al., 1995), but the latter is not confirmed by our phylogenetic tree (Fig. 2).

*PAL1* and *PAL2* are not only structurally very similar, but they also share common promoter elements and a similar expression pattern (supplemental Table Is). mRNAs from both genes are most abundant in roots and stems, where the expression increases during the later stages of development (Fig. 3; supplemental Table Is; Wanner et al., 1995). Analysis of the fusion between the *AtPAL1* promoter and β-glucuronidase (GUS) revealed that the expression is located in the vascular tissues (Ohl et al., 1990; Leyva et al., 1995). Besides *PAL1* and *PAL2*, *PAL4* also is highly expressed in stem tissue, as shown by our RT-PCR expression analysis (Fig. 3). In addition, *PAL2* and *PAL4* are abundantly expressed in the seeds, as judged from the EST data (Fig. 3; supplemental Table Is). Although all four genes are almost ubiquitously expressed in the tissues investigated in this study, *PAL3* seems to be generally expressed at a lower level (supplemental Table Is; Wanner et al., 1995; Mizutani et al., 1997; Ruegger et al., 1999). *PAL1*

---

**Figure 2.** Neighbor-joining tree of the *PAL* family, inferred from Kimura corrected evolutionary distances. Bootstrap values (Neighbor-Joining/Maximum Likelihood; NJ/ML) above 50% are shown at the internodes. The scale measures evolutionary distance in substitutions per nucleic acid. Clusters of sequences are represented as triangles with a height equal to the average distance separating the terminal nodes from the deepest branching point in the cluster and a base proportional to the number of sequences composing it. Species and GenBank identification numbers of non-Arabidopsis sequences included in this tree are: dicots, *Populus* (169453, 485808, and 1109640), *Glycine* (18376), *Trifolium* (437711), *Citrus* (4808125, 4808127, and 1276902), *Rubus* (7208613 and 7208615), *Camellia* (662270), *Petroselinum* (534892), *Nicotiana* (170349), *Digitalis* (2631994), and *Lactuca* (18001006); monocots, *Oryza* (20280 and 871493); and gymnosperms, *Pinus* (1143311). Arath, Arabidopsis; Pinta, pine (*Pinus taeda*).

was one of the first plant defense genes identified, and its involvement in pathogen infection and abiotic stress has been studied. Among the ESTs derived from diverse stresses, *PAL1* and *PAL2* are clearly the most important stress-responsive family members, with 20 of 41 ESTs and 17 of 50 ESTs in total, respectively, even taking into account the relative database sizes (supplemental Table Is). In line with this, a number of regulatory elements, shown to be involved in promoter responsiveness to elicitors, wounding, and pathogen infection, were found in these genes using the stringent search method (see “Materials and Methods”; supplemental Table Is).

In addition, and in accordance with the expression pattern, the promoters of *PAL1* and *PAL2* contain well-conserved AC elements that specify vascular expression of phenylpropanoid genes (supplemental Table Is; Ohl et al., 1990; Hauffe et al., 1993; Hatton et al., 1995; Wanner et al., 1995; Lacombe et al., 2000). An A box, proposed to work in conjunction with the AC elements in the parsley (*Petroselinum crispum*) *PAL1* and *PAL4* genes (Logemann et al., 1995), was not detected in the Arabidopsis *PAL1* and *PAL2* promoters (supplemental Table Is). *PAL4* contains an A box but lacks an AC element. Interestingly, an H box and a G box were found in the *PAL4* promoter. This combination of cis-elements was shown to be sufficient for the feed-forward induction of the chalcone

Monolignol Biosynthetic Genes in Arabidopsis

S R L F Si St (6 devel. stages)
PAL1*
PAL2*
PAL3
PAL4
C4H*
4CL1*
4CL2*
4CL3
4CL4
HCT*
ACT

S R L F Si St (6 devel. stages)
C3H1*
C3H2
C3H3
CCoAOMT1*
CCoAOMT2
CCoAOMT3
CCoAOMT4
CCoAOMT5
CCoAOMT6
CCoAOMT7
CCR1*
CCR2

S R L F Si St (6 devel. stages)
F5H1*
F5H2
COMT*
CAD1
CAD2
CAD3
CAD4
CAD5
CAD6*
CAD7/8
CAD9

Figure 3. Expression profiles of all 34 monolignol biosynthesis genes. Semiquantitative expression was determined using reverse transcription (RT)-PCR (see “Materials and Methods”). Due to different PCR dynamics of shorter or longer amplification products, only different tissues for a particular gene may be compared. It should be noted that CAD7 and CAD8 arose during a recent duplication event (described in detail by Tavares et al., 2000) and could not be distinguished in the RT-PCR analysis because of their high sequence similarity: 98% and 94% identity in the coding regions and putative 3′-untranslated regions, respectively. S, Seedling; R, root; L, leaf; F, flower; Si, green siliques; St, stem (at 1-, 3-, 5-, 10-, 15-, and 20-cm length).

synthase (CHS) promoter by p-coumaric acid in bean (*Phaseolus vulgaris*; Loake et al., 1992; Lindsay et al., 2002). This observation may indicate that *PAL4* is regulated by the reaction product of C4H.

In conclusion, all *PAL* genes are expressed in the inflorescence stem, a tissue with a high portion of lignifying cells. However, the presence of an AC element qualifies *PAL1* and *PAL2* as the most likely candidates to be involved in monolignol biosynthesis in the vascular lignifying cells. In accordance, the corresponding mutants show defects in lignin formation (A. Rohde and W. Boerjan, unpublished data).

C4H

C4H (E.C. 1.14.13.11) controls the conversion of cinnamate into p-coumarate (Fig. 1). C4H (CYP73A5) belongs to the cytochrome P450-dependent mono-oxygenases, like the two other hydroxylases in the pathway (C3H, F5H). So far, only one *C4H* gene has been described in Arabidopsis (Bell-Lelong et al., 1997; Mizutani et al., 1997; Urban et al., 1997). Although multiple family members have been detected in other plants (Betz et al., 2001, and refs. therein), we could not find any evidence for additional *CYP73* genes in Arabidopsis. Phylogenetic analysis shows that two classes of *C4H* genes exist in plants (Fig. 4;

Plant Physiol. Vol. 133, Downloaded from www.plantphysiol.org on February 18, 2015 - Published by www.plant.org Copyright © 2003 American Society of Plant Biologists. All rights reserved.

Raes et al.

Nedelkina et al., 1999; Betz et al., 2001). Furthermore, the tree topology indicates that the origin of these two classes has predated the divergence of gymnosperms and angiosperms, suggesting that class II members must have existed at some time in the evolution for most plant lineages. The Arabidopsis C4H gene belongs to class I; a class II homolog was most probably lost during the evolution of this species. C4H is expressed in all tissues and upon exposure to light, wounding, and fungal infection (supplemental Table IIs; Bell-Lelong et al., 1997; Meyer et al., 1998; Nair et al., 2002). In our RT-PCR experiment, C4H expression increased during the later stages of stem development (Fig. 3). Activity of AtC4H::GUS coincides with vascular cells in the inflorescence stem and in leaves, but in roots the promoter is active in all cells, giving the strongest expression in this tissue (Bell-Lelong et al., 1997; Nair et al., 2002). A strong C4H expression is also found in siliques and seeds, where it could be involved in the production of sinapate esters (Chapple et al., 1994). In addition, the C4H promoter contains an H box, which might be responsible for induction of C4H expression after elicitation.

By TargetP (Emanuelsson et al., 2000), the C4H protein is predicted to contain an endoplasmic reticulum (ER)-targeting peptide. However, this peptide coincides with the membrane anchor region of P450 enzymes, whose features are a stretch of hydrophobic amino acids, followed by a small region rich in basic amino acids and a hinge region of the conserved (P/I)PGPx(G/P)xP sequence (Chapple, 1998). All class II C4H proteins included in the phylogenetic analysis (Fig. 4) show a divergent hinge and basic amino acid region. Although the function of these class II C4H proteins is unclear at the moment, the shared degeneration of this crucial region could be an important clue in discovering their function.

### 4CL

4CL (E.C. 6.2.1.12) catalyzes the formation of CoA esters of p-coumaric acid, caffeic acid, ferulic acid, 5-hydroxyferulic acid, and sinapic acid (Fig. 1; Lee et al., 1997; Hu et al., 1998). The plethora of additional potential substrates may explain why there are many 4CL isoenzymes in most plants. In addition to the different substrate specificities, the genes typically have a distinct spatio-temporal expression pattern (Lewis and Yamamoto, 1990; Hu et al., 1998; Harding et al., 2002).

We detected four 4CL and nine 4CL-like genes in the Arabidopsis genome. Phylogenetic analysis of the predicted proteins, together with characterized 4CL proteins and luciferases, acetate, and fatty acid CoA ligases (other adenylate-forming enzymes; data not shown), shows that 4CL proteins fall into two classes (Fig. 5; Ehlting et al., 1999; Cukovic et al., 2001). Three

![Figure 5](https://i.imgur.com/yourimageurl.png)

**Figure 5.** Consensus of two neighbor-joining trees of the 4CL and 4CL-like proteins, inferred from Kimura corrected evolutionary distances. Bootstrap values (NJ/ML) above 50% are shown at the internodes. The scale measures evolutionary distance in substitutions per amino acid. Clusters of sequences are represented as described in Figure 2. Species and GenBank identification numbers of non-Arabidopsis sequences included in this tree are: Class I dicots, *Solanum* (398963, 398965, and 5163399), *Capsicum* (12003966), *Nicotiana* (12229631, 7428495, and 12229632), *Lithospermum* (1117778), *Petroselinum* (112800 and 112801), *Rubus* (9651915 and 9651917), *Populus* (7437854, 7437855, 14289344, 18032806, 7437852, and 15636677), and *Amorpha* (17063848); gymnosperms, *Pinus* 4CL (7437872); Class II monocots, *Lolium* (7188335) and *Oryza* (12229650); Class II dicots, *Lithospermum* (9988455), *Glycine* (18266852), and *Populus* (7437853 and 14289346); monocots, *Oryza* (112802), *Lolium* (7188337 and 7188339); and 4CL-like, *Oryza* (12039389). Arabidopsis; *Pinta*, pine.

of the Arabidopsis proteins belong to class I (4CL1, 4CL2, and 4CL4) and 4CL3 to class II; the remaining nine are classified as 4CL like because they do not correspond to any of the 4CL or other enzyme classes mentioned above.

Our expression analysis showed that 4CL genes are expressed in almost all investigated tissues, with 4CL4 having the most restricted expression (Fig. 3). The latter observation is supported by the smallest number of ESTs found for 4CL4 among the 4CL genes (supplemental Table IIIs). 4CL1 and 4CL2 are expressed throughout inflorescence stem development and expression increases during the later stages (supplemental Table IIIs; Lee et al., 1995; Mizutani et al., 1997; Ehlting et al., 1999). On the contrary, 4CL3 and 4CL4 are expressed only during the later stages of inflorescence stem development (Fig. 3; supplemental Table IIIs). The expression of 4CL3 is not affected by wounding and *Peronospora parasitica* infection, in clear difference to the class I 4CL genes (supplemental Table IIIs; Ehlting et al., 1999). In accordance with the expression analysis, the promoters of both 4CL1 and 4CL2 contain AC elements. Furthermore, the promoter analysis identified an AT-rich sequence motif in the 4CL4 promoter and an H box in the 4CL3 and 4CL4 promoters, hinting to a role in particular stress responses (Seki et al., 1996; Rushton et al., 2002).

In conclusion, 4CL1 and 4CL2 are the best candidates for a function in monolignol biosynthesis during developmental lignification, as suggested previously by Ehlting et al. (1999). Their expression correlates with tissues containing a high portion of lignifying cells, and AC elements are present in their promoters. To the contrary, 4CL3 (class II) was suggested to channel activated p-coumarate to CHS and subsequently to the flavonoid biosynthesis (Ehlting et al., 1999). 4CL4 (class I), although expressed more specifically or at a lower level, might have yet another substrate specificity. In soybean (*Glycine max*), a single amino acid deletion determines whether or not 4CL can use sinapic acid as a substrate (Lindermayr et al., 2003), a function lacking for 4CL1, 4CL2, and 4CL3 in Arabidopsis (Ehlting et al., 1999). Interestingly, 4CL4 shows a similar deletion in the region coding for the substrate-binding pocket, suggesting that this gene may have acquired an altered substrate specificity toward sinapic acid after duplication. A recent paper shows that 4CL4 is indeed able to convert sinapic acid (Schneider et al., 2003).

HCT

HCT belongs to a large family of acyltransferases that are involved in the biosynthesis of diverse secondary metabolites. Only recently, the first HCT has been purified from tobacco stems, and the corresponding gene was cloned (Hoffmann et al., 2003). In tobacco, HCT catalyzes the conversion of p-coumaroyl-CoA and caffeoyl-CoA to the corresponding shikimate or quinate esters (Fig. 1). These

Monolignol Biosynthetic Genes in Arabidopsis

shikimate and quinate esters, themselves being important intermediates in the phenylpropanoid pathway, have been shown recently to be good substrates for C3H (Kühnle et al., 1987; Schoch et al., 2001; Franke et al., 2002a, 2002b; Nair et al., 2002). Moreover, HCT catalyzes also the reverse trans-esterification (Hoffmann et al., 2003). Therefore, HCT might play a critical role up- and downstream of C3H. For the Arabidopsis HCT homolog, a biochemical activity similar to that of the tobacco HCT has been shown (Hoffmann et al., 2003).

Here, one HCT gene was detected in the Arabidopsis genome (Fig. 6). Because only two homologs were characterized and the family is apparently well conserved (approximately 60% identity between monocot and dicot members; data not shown), no more distantly related genes were included.

The expression analysis shows that HCT is expressed in all tissues investigated but strongly in the inflorescence stem (Fig. 3; supplemental Table IVs). The promoter contains an AC element. The high and ubiquitous expression is confirmed by the second highest number of ESTs found for the 10 gene families analyzed (supplemental Table IVs). Interestingly, the combined presence of an H and a G box was observed, as for PAL4 and F5H2, suggesting transcriptional regulation by the pathway intermediate p-coumaric acid (Loake et al., 1992).

C3H

C3H was originally named after its suspected function in C3-hydroxylation of p-coumaric acid, but recently, CYP98A3 (C3H1) was shown to preferentially convert the shikimate and quinate esters of p-coumaric acid into the corresponding caffeic acid conjugates, whereas p-coumaric acid and p-coumaroyl-CoA were not substrates of this enzyme (Fig. 1; Schoch et al., 2001; Franke et al., 2002b; Nair et al., 2002).

![Figure 6](https://i.imgur.com/1234567.png)

Figure 6. Neighbor-joining tree of the HCT family, inferred from Kimura corrected evolutionary distances. Bootstrap values (NJ/ML) above 50% are shown at the internodes. The scale measures evolutionary distance in substitutions per amino acid. Species and GenBank identification numbers of non-Arabidopsis sequences included in this tree are: *Ipomoea* (6469032), *Oryza* (21740518), and *Nicotiana* (27475615). Arath, Arabidopsis; Ipoba, *Ipomoea batatas*; Nicta, tobacco; Orysa, rice (*Oryza sativa*).

Plant Physiol. Vol. 133, Downloaded from www.plantphysiol.org on February 18, 2015 - Published by www.plant.org Copyright © 2003 American Society of Plant Biologists. All rights reserved.

Raes et al.

We detected three C3H genes in the Arabidopsis genome, which all belong to the CYP98 class of the P450 enzymes. Only a few proteins of this class could be found from other species for phylogenetic analysis (Fig. 7). Arabidopsis C3H1 clusters with all known C3Hs of other species, whereas C3H2 and C3H3 (CYP98A8 and CYP98A9, respectively) probably constitute a different class that diverged before the gymnosperm-angiosperm split (Fig. 7).

The expression analysis shows that C3H1 is expressed in all tissues, an observation that is supported by ESTs from various tissues (supplemental table Vs). Previous studies detected the highest expression in the vascular tissues of stem and root (supplemental table Vs; Schoch et al., 2001; Franke et al., 2002b; Nair et al., 2002). On the contrary, C3H2 and C3H3 are expressed only during particular stages of inflorescence stem development: C3H2 is expressed in older stems and C3H3 in young developing stems (Fig. 3). The fact that only one EST is found for C3H2 and none for C3H3 suggests that they are either conditionally regulated or expressed at low levels (supplemental table Vs). The promoter analysis reveals a well-conserved AC element in the promoter of C3H1, in agreement with its vascular expression detected by the GUS reporter system (Nair et al., 2002).

Analysis of the N terminus by TargetP predicts the C3H1 protein to contain an ER-targeting peptide, but it overlaps, as for C4H, with the membrane anchor region of P450 enzymes. The C3H1 protein has previously been localized in the membrane fraction in yeast (Franke et al., 2002b). In contrast to C3H1, the sequences of C3H2 and C3H3 are divergent in both the stretch of basic amino acids and the hinge region of the membrane anchor. Because these regions are necessary for the correct insertion of the enzyme in the membrane (Chapple, 1998), the degeneration of this region suggests they are not membrane-anchored proteins. It should be noted that C3H2 and C3H3 do not hydroxylate shikimate and quinate es-

![Figure 7](https://i.imgur.com/1234567.png)

Figure 7. Neighbor-joining tree of the C3H family, inferred from Kimura corrected evolutionary distances. Bootstrap values (NJ/ML) above 50% are shown at the internodes. The scale measures evolutionary distance in substitutions per amino acid. Species and GenBank identification numbers of non-Arabidopsis sequences included in this tree are: Sesamum (17978831), Sorghum (5915857), Pinus (17978651), and Glycine (5915858). Arath, Arabidopsis; Glyma, soybean; Sesin, Sesamum indicum; Sorbi, Sorghum bicolor; Pinta, pine.

CCoAOMT

CCoAOMT (E.C. 2.1.1.104) catalyzes the methylation of caffeoyl-CoA to feruloyl-CoA (in vitro and in vivo) and 5-hydroxyferuloyl-CoA to sinapoyl-CoA (at least in vitro) and is, together with COMT, responsible for the methylation of the monolignol precursors (Fig. 1; Ye et al., 1994; Zhong et al., 1998; Pinçon et al., 2001).

Seven putative members of the CCoAOMT gene family were detected in the Arabidopsis genome (Fig. 8). Plant CCoAOMT genes fall into two classes: Class I contains the Arabidopsis CCoAOMT1 gene together with the majority of experimentally characterized CCoAOMT genes (e.g. Zhong et al., 1998; Meyermans et al., 2000), whereas class II consists of six Arabidopsis genes and a few sequences from other species. The latter class does not closely resemble most of the certified CCoAOMT genes but contains an experimentally characterized chickweed (*Stellaria longipes*) CCoAOMT able to methylate caffeoyl-CoA (Zhang and Chinnappa, 1997).

CCoAOMT1 is expressed in all tissues investigated and has by far the highest number of ESTs (Fig. 3; supplemental Table VIs). Moreover, the CCoAOMT1 gene has two AC elements in its promoter. CCoAOMT1 is highly expressed in the basal portion of the inflorescence as compared with the apical portion (Goujon et al., 2003). Of the class II genes, CCoAOMT5 and CCoAOMT7 are expressed in all tis-

![Figure 8](https://i.imgur.com/ABCDEFG.png)

Figure 8. Neighbor-joining tree of the CCoAOMT family, inferred from Kimura corrected evolutionary distances. Bootstrap values (NJ/ML) above 50% are shown at the internodes. The scale measures evolutionary distance in substitutions per nucleic acid. Clusters of sequences are represented as described in Figure 2. Species and GenBank identification numbers of non-Arabidopsis sequences included in this tree are: Class I dicots, *Populus* (2960355, 857577, 13249170, and 2960357), *Zinnia* (533120), *Petroselinum* (169648), *Nicotiana* (2511736), *Citrus* (6561880), *Vitis* (1000518), and *Eucalyptus* (5739372 and 1934858); gymnosperms, *Pinus* CCoAOMT (4104458); Class II dicots, *Stellaria* (438896) and *Populus* (1785476); and monocots, *Zea* (5101869, 5101867) and *Oryza* (5091496 and 5257255 [three genes]). Arath, Arabidopsis; Pinta, pine.

sues, but only the expression of CCoAOMT7 increases during the later stages of inflorescence stem development. Furthermore, CCoAOMT4 and CCoAOMT5 are also expressed at all stages of inflorescence stem development. Others, such as CCoAOMT2, CCoAOMT3, and CCoAOMT6, are expressed toward the end of inflorescence stem development (Fig. 3). Few ESTs have been found for most genes of class II (supplemental Table VI).

CCoAOMT genes of other species were shown to be responsive to pathogens or elicitors (e.g. Pakusch et al., 1991; Chen et al., 2000); corresponding promoter elements were identified in CCoAOMT1, CCoAOMT2 and CCoAOMT3 (supplemental Table VI). CCoAOMT3 has an extended N-terminal sequence, not shared by any of the other CCoAOMTs, predicted to be an ER-targeting peptide.

Based on its clustering in class I, its expression characteristics and level, and the presence of two AC elements in its promoter, CCoAOMT1 is the main candidate gene to be involved in the monolignol pathway during developmental lignification.

Monolignol Biosynthetic Genes in Arabidopsis

ment (Fig. 3). Although CCR2 was hardly detected in stem by RNA gel blots (Lauvergeat et al., 2001), the more sensitive RT-PCR clearly reveals CCR2 expression in the inflorescence stem (Fig. 3). For both genes, expression increases with age during inflorescence stem development (Fig. 3). Corresponding with the differences in expression levels of CCR1 and CCR2 (Lauvergeat et al., 2001), 10-fold more ESTs are found for CCR1 than for CCR2 (supplemental Table VII). Both genes are induced by *Xanthomonas campestris* infection and ESTs linked with stress and pathogen infection have been detected (Lauvergeat et al., 2001; supplemental Table VII). The promoter of CCR1 contains a well-conserved AC element and conforms with its function in lignification and the strong expression in stems (Lauvergeat et al., 2001; supplemental Table VII).

In conclusion, CCR1 and CCR2 are expressed during both developmental lignification and pathogen response, as documented by our expression analysis and ESTs (Fig. 3; supplemental Table VII). The role of CCR1 in lignification has clearly been established through the *irx4* (irregular xylem) mutant characterization (Jones et al., 2001). Although CCR2 seems to be implicated in stress and elicitor response (Lauvergeat et al., 2001), the expression results do not exclude a (minor) role for CCR2 in developmental lignification.

F5H

F5H, also called coniferaldehyde 5-hydroxylase, is a cytochrome P450-dependent monooxygenase (CYP84) that is required for the production of syringyl lignin because it is responsible for the

![Figure 9](attachment:Figure_9.png)

Figure 9. Neighbor-joining tree of the CCR family, inferred from Kimura corrected evolutionary distances. Bootstrap values (NJ/ML) above 50% are shown at the internodes. The scale measures evolutionary distance in substitutions per amino acid. Clusters of sequences are represented as described in Figure 2. Species and GenBank identification numbers of non-Arabidopsis sequences included in this tree are: CCR dicots, *Eucalyptus* (7431407, 7431408, and 10304406) and *Populus* (7239228, 2960364, and 9998901); CCR monocots, *Lolium* (9964087), *Saccharum* (3341511 and 17978549), and *Zea* (7431410 and 3242328); gymnosperms, *Pinus* CCR (17978649), *Zea* CCR2 (3668115), and *Oryza* CCR-like (13486725, 13486726, and 18307514); and CCR-like angiosperms, *Oryza* (15624051). Arath, Arabidopsis; Orysa, rice; Pinta, pine; Zeama, maize (*Zea mays*).

Plant Physiol. Vol. 133, Downloaded from www.plantphysiol.org on February 18, 2015 - Published by www.plant.org Copyright © 2003 American Society of Plant Biologists. All rights reserved.

Raes et al.

5-hydroxylation of coniferaldehyde and/or coniferyl alcohol (Fig. 1; Humphreys et al., 1999; Li et al., 2000; Humphreys and Chapple, 2002).

The Arabidopsis genome harbors two \( F5H \) homologs, both belonging to the CYP84 family of the P450 monooxygenases. \( F5H1 \) (CYP84A1) has been characterized in Arabidopsis, Liquidambar styraciflua, and Brassica napus (Meyer et al., 1996; Osakabe et al., 1999; Nair et al., 2000), whereas \( F5H2 \) (CYP84A4), a more divergent member of the CYP84 family, is described for the first time, to our knowledge, in this study. So far, no genes that closely resemble \( F5H2 \) have been detected in other plants, although the phylogeny indicates that the two proteins found in Arabidopsis diverged before the divergence of the different Rosidae subfamilies (Fig. 10).

Our expression analysis revealed \( F5H1 \) expression in all tissues and an increasing expression during inflorescence stem development (Fig. 3), in accordance with results of earlier studies (supplemental Table VIII; Meyer et al., 1998; Ruegger et al., 1999; Goujon et al., 2003). \( F5H1 \) was also expressed in several other tissues but mainly in young and senescent leaves and in roots (Meyer et al., 1996; Ruegger et al., 1999). In contrast to \( F5H1 \), \( F5H2 \) had the strongest expression in the early stages of inflorescence stem development (Fig. 3). Only two ESTs were found for \( F5H1 \) and none for \( F5H2 \) (supplemental Table VIII).

In the promoter analysis, for both genes an H box was found and for \( F5H2 \) a G box was also found, suggesting that both genes may be inducible and that \( F5H2 \) may be regulated by p-coumarate (Loake et al., 1992; Lindsay et al., 2002). Moreover, \( F5H1 \) and \( F5H2 \) contain a fully conserved membrane anchor region. In addition, \( F5H2 \) is predicted to contain an ER-targeting peptide that coincides with the region of the membrane anchor of P450 enzymes. Remarkably,

no AC element was detected for either \( F5H \) gene, although \( F5H1 \) had been shown to be involved in lignification through the analysis of the \( fah1 \) mutant (Chapple et al., 1992).

COMT

COMT (E.C. 2.1.1.68) was originally postulated to be a bifunctional enzyme methylating caffeic acid and 5-hydroxyferulic acid. However, in vitro and transgenic studies revealed that the predominant role of COMT is the methylation of 5-hydroxyconiferaldehyde and/or 5-hydroxyconiferyl alcohol to sinapaldehyde and/or sinapyl alcohol, respectively (Fig. 1; Osakabe et al., 1999; Li et al., 2000; Chen et al., 2001; Guo et al., 2001; Parvathi et al., 2001; Goujon et al., 2003).

We detected only one COMT gene in the Arabidopsis genome. Furthermore, 13 proteins similar to COMT were detected that clustered in-between the functionally characterized COMT clade and the cluster containing the hydroxycinnamic acid/hydroxycinnamoyl-CoA ester O-methyltransferase protein (AEOMT; Li et al., 1997, 1999), i.e. among proteins that have been shown to use a wide variety of substrates (Fig. 11; Vernon and Bohnert, 1992; Maxwell et al., 1993; Pellegrini et al., 1993; Takeshita et al., 1995). Because the role of AEOMT in the monolignol pathway is still a matter of debate (Anterola et al., 2002), and other COMT candidate genes of conifers clustered much more closely to the known COMTs, it is unclear whether these 13 genes play any role in the monolignol pathway. Therefore, these genes were classified as COMT likes. As a consequence, only one class of COMTs exists in plants (Fig. 11; Maury et al., 1999).

Our RT-PCR data show that COMT is expressed in all tissues investigated, and the numerous ESTs point toward a generally high and ubiquitous expression (Fig. 3; supplemental Table IX). Ninety-nine COMT ESTs, with a fifth being stress related, is almost twice the number found for any other gene in this analysis (supplemental Table IX). COMT expression is particularly high in the inflorescence stem, with an increase during the later stages of development (Fig. 3; supplemental Table IX). Correspondingly, COMT::GUS expression occurs in xylem, differentiating fibers, and mature phloem (Goujon et al., 2003). Unlike many other monolignol biosynthesis genes, COMT has no AC elements in its promoter. In fact, to the best of our knowledge, AC elements have never been reported in COMT promoters of other plants either.

Interestingly, the COMT protein might be myristoylated. The N-terminal MGSTAETQLTPVQVTDDE sequence was identified as a “twilight zone” myristoylation signal, which corresponds both with truly myristoylated proteins and with false positives (Maurer-Stroh et al., 2002). Myristoylation is gener-

![Figure 10](#fig10)

**Figure 10.** Neighbor-joining tree of the \( F5H \) family, inferred from Kimura corrected evolutionary distances. Bootstrap values (NJ/ML) above 50% are shown at the internodes. The scale measures evolutionary distance in substitutions per amino acid. GenBank identification numbers of non-Arabidopsis sequences included in this tree are: *Populus* CYP84A4 (6688937), *Lycopersicon* CYP84A2 (5002354), *Liquidambar* CYP84A3 (5731998), and *Brassica* F5H1, F5H2, and F5H3 (10197650, 10197652, and 10197654). Arath, Arabidopsis; Brana, *Brassica napus*; Liqst, *Liquidambar styraciflua*; Lyces, tomato (*Lycopersicon esculentum*); Poptr, *Populus trichocarpa*.

Monolignol Biosynthetic Genes in Arabidopsis

Figure 11. Neighbor-joining tree of the COMT family, inferred from Kimura corrected evolutionary distances. Bootstrap values (NJ/ML) above 50% are shown at the internodes. The scale measures evolutionary distance in substitutions per amino acid. Clusters of sequences are represented as described in Figure 2. Species and GenBank Identifier numbers of non-Arabidopsis sequences included in this tree are: COMT dicots, *Populus* (7528266, 762870, 231757, 444327, 7332271, 7447887, and 762872), *Stylosanthes* (1582580), *Medicago* (116908), *Prunus* (3913295), *Fragaria* (6760443), *Liquidambar* (5732000), *Chrysosplenum* (1184041 and 567077), *Vitis* (7271883), *Capsicum* (3421382, 7488967, and 12003964), *Nicotiana* (480082 and 480083), *Eucalyptus* (1169009 and 5739365), *Clarkia* (2832224 and 3913289), *Mesembryanthemum* (7447880), *Thalictrum* (4808522, 4808524, 4808526, 4808528, and 4808530), *Catharanthus* (18025321), *Ocimum* (5031492, 5031494), and *Zinnia* (642952); COMT monocots, *Lolium* (4104220, 4104222, 4104224, and 2388664), *Sorghum* (18033964), *Saccharum* (3341509), *Zea* (729135), and *Festuca* (14578611, 14578613, 14578615, and 14578617); COMT gymnosperms, *Pinus* (15524083), *Picea* (COMT-C7 and COMT-C16; M.H. Walter, personal communication); *Nicotiana* Catechol-OMT III (542050); *Glycyrrhiza* OMT (1669591), *Medicago* OMT (7447884), *Mesembryanthemum* IMT1 (1170555), *Coptis* sinapoyl-Glc:malate sinapoyltransferase (SMT; 758580), and *Medicago* O-diphenol OMT (6688808); and AEOMT gymnosperms, *Pinus* (7447883, 1777386, and 4574324). Arath, Arabidopsis; Copja, *Coptis japonica*; Glyec, *Glycyrrhiza echinata*; Medsa, alfalfa; Mescr, *Mesembryanthemum crystallinum*; Nicta, tobacco.

ally associated with cell membrane anchoring or, as recently shown for an Arabidopsis protein kinase, ER attachment (Lu and Hrabak, 2002). Pending the experimental verification of this observation, the putative localization of the COMT protein indicates a new research avenue in the field of monolignol channeling and export.

CAD

CAD (E.C. 1.1.1.195) catalyzes the last step in monolignol biosynthesis, i.e. the reduction of cinnamyl aldehydes into their corresponding alcohols (Fig. 1). CAD reduces various aldehydes, present in different cell types or during different stages of development. Besides the function in developmentally regulated lignification, a number of CAD genes have been characterized for their response to plant pathogens (Kiedrowski et al., 1992).

Here, nine putative CAD genes were detected in the Arabidopsis genome (Table I; Tavares et al., 2000; Sibout et al., 2003). Our phylogenetic analysis revealed that eight of the CAD proteins fall into three classes, whereas CAD9 is more divergent (Fig. 12). CAD2 and CAD6, belonging to the class I CADs,

closely resemble CAD proteins that have been characterized for their involvement in lignification in other species. The topology of the tree indicates furthermore that the class I “true” CAD clade diverged from the other CADs before the angiosperm-gymnosperm split (Fig. 12).

Class II CADs (CAD3, CAD4, and CAD5) cluster with a number of alcohol dehydrogenases with diverse substrate preferences, such as the poplar (*Populus tremuloides*) sinapyl alcohol dehydrogenase (Li et al., 2001), the celery (*Apium graveolens*) mannitol dehydrogenase (Williamson et al., 1995), and the parsley ELI3/CAD proteins (Kiedrowski et al., 1992; Logemann et al., 1997). CAD4 (*AtELI3-1*) and CAD5 (*AtELI3-2*) have been identified previously as responsive to elicitor treatments and *Pseudomonas syringae* infection (Kiedrowski et al., 1992). Moreover, CAD5 has a substrate specificity distinct from “true” CADs, mannitol dehydrogenase, and aromatic alcohol: NADP+ oxidoreductase and was, therefore, named benzyl alcohol dehydrogenase (BAD; Somssich et al., 1996).

Class III CADs (CAD1, CAD7, and CAD8) cluster in a group with an alcohol dehydrogenase from alfalfa (*Medicago sativa*), which is able to catalyze the

Raes et al.

![Figure 12. Neighbor-joining tree of the CAD family, inferred from Kimura corrected evolutionary distances. Bootstrap values (NJ/ML) above 50% are shown at the internodes. The scale measures evolutionary distance in substitutions per amino acid. Clusters of sequences are represented as described in Figure 2. Species and GenBank Identifier numbers of non-Arabidopsis sequences included in this tree are: Class I dicots, Populus (421814, 1168734, 9998899, and 7239226), Nicotiana (231676 and 231675), Medicago (399168), Aralia (1168727), Zinnia (1944403), Eucalyptus (1705554, 10281656, 399165, 10719920, and 3913185); Class I monocots, Saccharum (10719916), Zea (3913182 and 7430938), Lolium (3913181), Festuca (15428276, 15428278, 15428280, and 15428282); gymnosperm CAD, Picea (584872 and 10719915), Pinus (107623, 3334135, 1168733, and 3372645); Class II dicots, Stylosanthes (3913194), Apium (12643507), Petroselinum (1168732), Lycopersicon (8099340 and 7430935), Mesembryanthemum (10720090), Fragaria (10720093, 13507210), and Populus (14279694); and Class III dicots: Stylosanthes (3913193) and Medicago (10720088). Arath, Arabidopsis.](image.png)

reduction of cinnamaldehyde, sinapaldehyde, and coniferaldehyde, but also several aliphatic aldehydes and various substituted benzaldehydes (Brill et al., 1999). Being very divergent from class I “true” CADs, this class also represents a group of multisubstrate alcohol dehydrogenases.

All CAD genes, except CAD2, CAD4, and CAD5, are expressed in all stages of inflorescence stem development (Fig. 3). Moreover, CAD2 and CAD6 are expressed in the inflorescence stem close to the bundle and interfascicular cambium, as revealed by promoter::GUS constructs (Sibout et al., 2003). Expression of most CAD genes is documented by ESTs, except for CAD7 and CAD8, which are nevertheless expressed, as indicated in the RT-PCRs (Fig. 3; supplemental Table Xs).

The promoter analysis revealed that CAD6 from class I and CAD5 from class II contain AC elements (supplemental Table Xs). In addition, an A box was detected in the CAD6 promoter. The fact that only one gene in the pathway contains both an AC element and an A box casts doubt on the previous assumption that an A box works in conjunction with AC elements (Logemann et al., 1995).

Based on the fact that they cluster with other well-characterized “true” CAD genes in the phylogenetic tree, CAD2 and CAD6 are the most likely candidates for the monolignol pathway in Arabidopsis. Of these two, only CAD6 has an AC element. Moreover, only the CAD6 gene mutant, but not that of CAD2, showed altered lignin content and structure (Sibout et al., 2003). The function of class II and class III CAD genes remains less clear. However, CAD3, CAD4, and

CAD5 of class II are the closest homologs of the poplar SAD (Li et al., 2001). Possibly, these proteins show a preference for sinapyl alcohol or sinapaldehyde, turning them into S branch-specific enzymes.

## DISCUSSION

### Toward the Core Monolignol Biosynthesis Gene Set for Developmental Lignification

Lignification is a process that occurs predominantly in cells of the vascular tissue, found in almost all organs, but most abundantly in stems and roots. A strong expression of monolignol biosynthesis genes in stems and roots is documented in numerous publications (supplemental Tables Is–Xs, and refs. therein). Possibly, lignification cDNAs are relatively highly represented in root libraries because of the absence of other very abundant processes, such as photosynthesis, or, as could be concluded from AtC4H::GUS analysis (Nair et al., 2002), the phenylpropanoid pathway in roots is active in more cells than the vascular ones to generate compounds not destined for lignification. In addition, flowers, seeds, and siliques accumulate significant amounts of other phenylpropanoid-derived compounds, such as sinapate esters and flavonoids (Chapple et al., 1994; Chen and McClure, 2000).

All 34 genes, annotated from the Arabidopsis genome sequence for their potential involvement in monolignol biosynthesis, are expressed at some stage of inflorescence stem development, a tissue with a prominent portion of lignifying cells (Table I and

supplemental Tables Is–Xs; Dharmawardhana et al., 1992). Of these genes, 23 are expressed throughout stem development (Table II). Furthermore, the expression of many genes increases during the later stages of inflorescence stem development, when lignification is more prominent (Fig. 3; Dharmawardhana et al., 1992).

A constitutive expression in the inflorescence stem (Fig. 3) and the phylogenetic classification in groups with functionally characterized proteins of other species were used as the first two criteria to delineate those family members that are the most likely to be involved in monolignol biosynthesis during developmental lignification (Table II). These criteria are fulfilled for 14 genes: *PAL1*, *PAL2*, *PAL3*, *PAL4*, *C4H*, *4CL1*, *4CL2*, *HCT*, *C3H1*, *CCoAOMT1*, *CCR1*, *F5H1*, *COMT*, and *CAD6*. Of these 14, eight genes have been already certified for their involvement in monolignol biosynthesis through the characterization of the corresponding mutants: *PAL1* (*pal1*), *PAL2* (*pal2*), *C4H* (*ref3*), *C3H1* (*ref8*), *CCR1* (*irx4*), *F5H1* (*fah1*), *COMT* (*comt1*), and *CAD6* (*cad-D*; Chapple et al., 1992; Jones et al., 2001; Franke et al., 2002a, 2002b; Goujon et al., 2003; Sibout et al., 2003; C. Chapple, personal communication; A. Rohde and W. Boerjan, unpublished data). Except for *CAD6*, these 14 genes have the highest expression level in their respective gene families, as judged from the number of ESTs (Table II). In conclusion, this set of 14 genes is through their expression and phylogeny eligible for being involved in the developmental monolignol biosynthesis in Arabidopsis.

### AC Elements Signpost a Number of G-Branch Monolignol Biosynthesis Genes

AC elements, originally identified in the promoters of the parsley *PAL1* gene, the bean *PAL2* and *PAL3* genes, and the parsley *4CL1* gene (Cramer et al., 1989; Lois et al., 1989; Hauffe et al., 1991; Leyva et al., 1992), are thought to enhance the expression of genes in xylem and at the same time to prevent their expression in the adjacent phloem and cortical cells. Because the deletion of the AC element results in derepression of phloem expression within the vascular tissue, it has been suggested that a (possibly phloem-specific) repressor is normally bound to the AC element, preventing expression in cells other than xylem cells. In contrast, in xylem cells, the repressor would be released to give rise to typically high expression levels (Hauffe et al., 1993; Hatton et al., 1995). A number of MYB and other transcription factors regulate the expression of monolignol biosynthesis genes and, moreover, bind to AC elements resulting in trans-activation of the respective promoters (e.g. Sablowski et al., 1995; Séguin et al., 1997; Jin et al., 2000; Sugimoto et al., 2000). Overexpression of specific MYB factors leads to lignin-related phenotypes (Tamagnone et al., 1998; Borevitz et al., 2000).

Given the importance of AC elements in specifying vascular expression, the presence of an AC element

Monolignol Biosynthetic Genes in Arabidopsis

in the promoters of the 34 annotated genes has been examined. In the past, most AC elements were identified by consensus sequences built from both experimentally verified AC elements and AC elements detected by sequence similarity. Often on top of such a consensus, a number of mismatches were allowed. Moreover, AC elements were often subdivided into ACI and ACII boxes, despite the fact that they align perfectly and were shown to be functionally redundant with respect to vascular expression (see supplemental data; Hatton et al., 1995). In view of the limited knowledge on the binding specificity of AC elements by transcription factors in vivo, we have built one unifying matrix for element identification based on the five experimentally verified and delineated elements (see supplemental data) with very stringent parameters in the search. To illustrate the power of matrix versus consensus approach, the statistical significance of both methods was evaluated on 1,000 random intergenic regions distributed uniformly throughout the Arabidopsis genome. The consensus approach used, for example, by Wanner et al. (1995), has a probability to find an AC element by chance of once every 1,200 bp, whereas with our approach, it is once every 37,000 bp. With our matrix approach, some of the AC elements that had been identified previously based on similarity to a consensus were not detected, such as the AC element in the *PAL3*, *C4H*, and *4CL3* promoters (Wanner et al., 1995; Mizutani et al., 1997; Ehlting et al., 1999). Note that the elements in these promoters have not been verified experimentally.

By searching all 29,787 Arabidopsis genes predicted with EuGene (Schiex et al., 2001), AC elements on either DNA strand were found in 780 promoters (2.6%). In the set of 34 monolignol biosynthesis genes, 10 of 34 promoters have AC elements (29%; Table II): eight on the positive and two on the negative strand.

Seven gene families have at least one family member with an AC element in their promoter (Table II). Genes with an AC element do not simply correspond with genes that are highly expressed as estimated from the number of ESTs (Table II). Rather, AC elements coincide with those gene family members that were assigned to be involved in developmental lignification based on expression and phylogeny (see above): of these 14 genes, nine contain an AC element (Table II). Thus, within their respective gene families, the following genes are extra-qualified for playing a role in developmental lignification in vascular tissues: *PAL1*, *PAL2*, *4CL1*, *4CL2*, *HCT*, *C3H1*, *CCoAOMT1*, *CCR1*, and *CAD6*. *CAD5* has an AC element but did not cluster with the true CAD clade in the phylogenetic tree (Fig. 12). In contrast, no AC elements were found in the gene families *C4H*, *F5H*, and *COMT*. Of these three gene families, *C4H* and *COMT* are single genes that, contrary to multigene families, may have acquired a more relaxed promoter

Table II. Summary of expression characteristics and occurrence of AC elements in monolignol biosynthesis genes

Characteristics are listed for each gene (marked with x): the corresponding mutants with lignification-related phenotypes, the clustering with certified proteins of other species in the phylogenetic analysis, a constitutive expression in the inflorescence stem (as determined by our RT-PCR analysis), ESTs of the different relevant categories in total nos., and the occurrence of AC elements. Genes marked with asterisks are predicted to be membrane associated. The position of AC elements found with stringent parameters is given in base pairs from ATG and the strand within parentheses. Underlined numbers indicate an overrepresentation of ESTs in this particular tissue or condition as compared with the presence of this gene in all ESTs. Overrepresentation was judged by comparing the relative occurrence of a gene in all ESTs with that of the same gene in a particular tissue or condition (see supplemental Tables Is–Xs for values). When fewer than three ESTs were detected in a particular tissue or condition, no overrepresentation was calculated. ESTs from inflorescence stem (presented in the supplemental Tables Is–Xs) are not presented here due to the small no. of ESTs available from this tissue (see “Materials and Methods”).

| Gene Family | Gene       | Corresponding mutants | Phylogenetic clustering | Constitutively in stem | EST total | EST aboveground organs | EST root | EST seed | EST stress, wound, pathogen | AC element (+/−) strand (core 0.9; matrix 0.9) |
|-------------|------------|-----------------------|--------------------------|-------------------------|-----------|------------------------|-----------|-----------|-------------------------------|-----------------------------------------------|
| PAL         | PAL1       | pal1<sup>a</sup>      | x                        | x                       | 41        | 1                      | 13        | 2         | 21                            | 483(+)                                          |
|             | PAL2       | pal2<sup>a</sup>      | x                        | x                       | 50        | 2                      | 17        | 12        | 17                            | 246(+), 495(+)                                 |
| C4H         |            |                       |                          |                         |           |                        |           |           |                               |                                               |
| 4CL         | 4CL1       |                       | x                        | x                       | 8         |                        | 1         |           | 4                             | 159(+)                                          |
|             | 4CL2       |                       | x                        | x                       | 13        |                        | 6         | 2         | 2                             | 124(+), 233(+)                                 |
| HCT         | HCT        |                       | x                        | x                       | 57        | 13                     | 11        | 8         | 15                            | 132(−)                                          |
| C3H         | C3H1*      | ref8                  | x                        | x                       | 36        | 5                      | 5         | 7         | 16                            | 145(+)                                          |
| CCoAOMT     | CCoAOMT1   |                       | x                        | x                       | 45        |                        | 19        | 9         | 10                            | 174(+), 651(+)                                 |
| CCR         | CCR1       | irx4                  | x                        | x                       | 43        | 8                      | 8         | 8         | 10                            | 269(+)                                          |
| F5H         |            |                       |                          |                         |           |                        |           |           |                               |                                               |
| COMT        | CAD5       |                       |                          |                         | 2         |                        | 1         |           | 1                             | 256(−)                                          |
| CAD         | CAD6       | cad-D                 | x                        | x                       | 23        |                        | 5         | 5         | 6                             | 515(+)                                          |

(Table continues on following page)

<sup>a</sup> A. Rohde and W. Boerjan, unpublished data.

Table II. Table continues from previous page

| Gene family | Gene          | Corresponding mutants | Phylogenetic clustering | Constitutively in stem | EST total | EST aboveground organs | EST root | EST seed | EST stress, wound, pathogen |
|-------------|---------------|-----------------------|-------------------------|------------------------|-----------|------------------------|----------|----------|----------------------------|
| PAL         | PAL3          |                       | x                       | x                      | 1         |                        |          |          |                            |
|             | PAL4          |                       | x                       | x                      | 28        | 1                      | 5        | 16       | 4                          |
| C4H         | C4H*          | ref3<sup>b</sup>      | x                       | x                      | 29        |                        | 2        | 5        | 15                         |
| 4CL         | 4CL3          |                       |                         |                        | 8         | 1                      | 1        | 2        | 3                          |
|             | 4CL4          |                       | x                       |                        | 2         |                        |          |         | 2                          |
| HCT         |               |                       |                         |                        |           |                        |          |         |                            |
| C3H         | C3H2          |                       |                         |                        | 2         |                        |          |         |                            |
|             | C3H3          |                       |                         |                        | 0         |                        |          |         |                            |
| CCoAOMT     | CCoAOMT2      |                       |                         |                        | 0         |                        |          |         |                            |
|             | CCoAOMT3*     |                       |                         |                        | 6         |                        |          |         | 3                          |
|             | CCoAOMT4      |                       | x                       |                        | 2         |                        |          |         | 1                          |
|             | CCoAOMT5      |                       | x                       |                        | 1         |                        |          |         |                            |
|             | CCoAOMT6      |                       |                         |                        | 2         |                        |          |         |                            |
|             | CCoAOMT7      |                       | x                       |                        | 4         |                        | 3        |         | 1                          |
| CCR         | CCR2          |                       | x                       |                        | 4         |                        |          |         | 1                          |
| F5H         | F5H1*         | fah1                  | x                       | x                      | 2         |                        |          |         |                            |
|             | F5H2*         |                       | x                       |                        | 0         |                        |          |         |                            |
| COMT        | COMT*         | comt                  | x                       | x                      | 99        |                        | 16       | 22      | 20                         |
| CAD         | CAD1          |                       | x                       |                        | 32        |                        | 4        | 10      | 10                         |
|             | CAD2          | cad-C                 | x                       |                        | 33        |                        | 4        | 2       | 8                          |
|             | CAD3          |                       | x                       |                        | 1         |                        |          |         | 1                          |
|             | CAD4          |                       |                         |                        | 26        | 5                      |          |         | 8                          |
|             | CAD7          |                       | x                       |                        | 0         |                        |          |         |                            |
|             | CAD8          |                       | x                       |                        | 0         |                        |          |         |                            |
|             | CAD9          |                       | x                       |                        | 9         |                        | 5        |         | 3                          |

<sup>b</sup>C. Chapple, personal communication.

Raes et al.

predicted to contain an ER-targeting peptide and do not comply to the amino acid features of the membrane anchor.

In addition, CCoAOMT3 contains also a putative ER-targeting signal. Besides membrane association, this could also imply a vacuolar or extracellular localization of this enzyme. Sinapoyl-Glc:malate sinapoyltransferase (SMT) and sinapoyl-Glc:choline sinapoyltransferase, involved in modification of sinapoyl-Glc, have been identified as proteins with an ER-targeting peptide (Lehfeldt et al., 2000; Shirley et al., 2001). These enzymes were suggested to be localized in the vacuole based on previous studies showing SMT activity in vacuoles (Strack and Sharma, 1985). Whether CCoAOMT3 shares this localization needs experimental verification.

Finally, a putative myristoylation site was detected in COMT, possibly involved in membrane anchoring. In agreement with this finding, a fraction of COMT from alfalfa stem was shown to be associated with the microsomal membranes, and channeling by COMT and F5H was suggested from coniferaldehyde to sinapaldehyde in the S branch of the monolignol pathway (Guo et al., 2002). This observation can be interpreted as coupling of COMT with the membrane-anchored F5H, although our data do not exclude that COMT itself could be anchored into the membrane by myristoylation.

### Evolutionary Note

The number of candidate monolignol biosynthesis genes found in the Arabidopsis genome varies greatly among the gene family studied, ranging from single genes to large gene families. A complex history of gene duplications has caused the expansion and diversification of the respective gene families. Interestingly, the polyploidy event, estimated to have occurred 24 to 86 million years ago, that marks the evolutionary history of Arabidopsis (Simillion et al., 2002; Blanc et al., 2003; Bowers et al., 2003; Ermolaeva et al., 2003; Raes et al., 2003) did not create new classes within any of the investigated families. In all cases, this event, together with several small-scale duplications, was responsible only for a greater within-class diversity (data not shown). Classes must have originated at an earlier time in evolution, i.e. before 24 to 86 million years ago.

In conclusion, the genome-wide analysis of monolignol biosynthesis genes, as presented here, provides the foundation of the next steps in unraveling the monolignol pathway. The combination of reverse genetics with transcript and metabolite profiling analyses of the respective mutants will profoundly enlarge our understanding of this pathway and its relation with plant development.

---

MATERIALS AND METHODS

#### Annotation

For each of the 10 enzymes of the monolignol biosynthetic pathway, the corresponding genes were annotated in four steps: (a) experimentally certified family members were collected from a variety of species, and a family-specific profile was created; (b) an Arabidopsis protein database was scanned with this profile; (c) true family members were selected; and (d) prediction on the selected genes was improved with information from different sources, such as cDNA and EST sequences and within-family sequence similarity.

More specifically, from a ClustalW protein alignment of experimentally certified family members from different plant species, a hidden Markov model-based profile was created using the HMMER package (Thompson et al., 1994; Eddy, 1998). This profile was used to scan an Arabidopsis protein database that was constructed through a Genemark.hmm prediction (Lukashin and Borodovsky, 1998) on the complete Arabidopsis genome sequence (The Institute for Genomic Research, release 4, March 21, 2003, available at ftp://ftp.tigr.org/pub/data/a_thaliana/ath1/). In a second scan, the complete genome sequence was searched with TBLASTN to detect genes that were not or wrongly annotated and would have been missed by using the protein database.

To delineate the gene family, several factors were taken into account. First, only HMMER hits with an E-value score below the default cutoff value (E = 10.0) were considered. Second, in most cases, a clear “drop” in the E-value score could be detected, indicating that sequences below this threshold did not fulfill the family model as well as did those above, thus providing a means to distinguish potential family members from false positives or—in the case of large superfamilies—genes of other subfamilies. This approach can potentially lead to wrong conclusions because of incomplete or biased sampling of the family. For this reason, a third method was applied, based on a phylogenetic analysis of the (super)family using the detected genes, close homologs, and more distantly related members of distinct, well-known (sub)families, retrieved from GenBank (Benson et al., 2003) and the literature. The resulting tree, showing the relations within the complete (super)family, was used to decide whether a protein belonged to the investigated family or not. As a rule, genes that clustered together with experimentally certified family members were considered to be part of the family. (Groups of) genes that did neither belong to the family nor cluster with other known families within the superfamily were considered as “likes” when they formed a sister group to the family investigated.

For the family members selected through these three criteria, the automatic annotation was improved by using information from different sources. First, the public databases were searched using BLASTN (Altschul et al., 1997) for ESTs and full-length cDNAs (percentage identity > 95% and further manual inspection of hits), and transcripts were aligned to the genomic region to verify intron-exon borders (Sim4; Florea et al., 1998). Second, the deduced protein sequences were aligned with the other family members to detect prediction errors (for example, missed exons). Third, predictions for candidate genes were verified with an alternative gene prediction tool called EuGene (specificity = 0.63 and sensitivity = 0.74 at the gene level; Schiex et al., 2001), available at http://www.inra.fr/bia/T/EuGene. This information was compiled with ARTEMIS (Rutherford et al., 2000) and was used to decide on a final gene structure.

For the annotation of the C4H, C3H, and F5H families, a substantial amount of information from the P450 databases (at http://www.biobase.dk/P450/p450.shtml and http://drnelson.utmem.edu/CytochromeP450.html) was used to improve the annotation. Prediction of myristoylation sites was done with the algorithm of Maurer-Stroh et al. (2002), available at http://mendel.imp.univie.ac.at/myristate/. Small Perl scripts were written to detect putative C-terminal farnesylation and geranylgeranylation sites (CaaX, CCXX, XCXC, and XXCC with a, aliphatic; C, Cys; and X, any amino acid; Nambara and McCourt, 1999; Randall and Crowell, 1999; Thompson and Okuyama, 2000). Signals for subcellular localization were predicted with the TargetP server (http://www.cbs.dtu.dk/services/TargetP/; specificity cutoff of >0.90; Emanuelsson et al., 2000). The annotation results were submitted to The Arabidopsis Information Resource and Munich Information Center for Protein Sequences databases for public access and are also accessible at http://www.psb.ugent.be/bioinformatics/login/.

# Phylogenetic Analysis

The nonredundant protein database was scanned for homologous sequences using BLASTp (Altschul et al., 1997), and the results were inspected manually. Sequences were aligned with ClustalW version 1.84 (Thompson et al., 1994), and alignments were improved by eye. Trees were constructed on conserved positions of the alignment with the neighbor-joining algorithm, as implemented in TREECON (Van de Peer and De Wachter, 1994), and by maximum-likelihood analysis (quartet puzzling) with TREE-PUZZLE (Schmidt et al., 2002). Alignments were edited and reformatted with ForCon (Raes and Van de Peer, 1999; available at http://www.psb.ugent.be/~jerae/ForCon/) and BioEdit (Hall, 1999). Statistical significance of nodes in the neighbor-joining approach was tested by using 500 bootstrap replicates.

# Promoter Analysis

Both strands of upstream regions (1,000 bp before the ATG codon or the distance between the previous gene and the ATG) and first and second introns of the genes were analyzed for regulatory elements with MatInspector (Quandt et al., 1995). To avoid false positives, we opted for a conservative approach with very strict parameters (core similarity = 0.9 and matrix similarity = 0.9). Furthermore, 1,000 random intergenic regions uniformly distributed throughout the Arabidopsis genome were searched with these parameters to have a rough estimate of the random occurrence of the motifs.

A list of potentially interesting motifs was compiled on the basis of the following three criteria: the motif had to be (a) experimentally characterized, (b) implicated in transcriptional regulation of known genes in the monolignol biosynthesis pathway, and/or (c) involved in elicitor, wound, or pathogen response. The motifs (and their respective calculated random occurrences in the Arabidopsis genome) that passed these criteria were: for Arabidopsis, GCC box (1/73,000 bp), jasmonate- and ethylene-responsive element (1/1,239,000 bp), W box (1/2,300 bp; withdrawn from results because of its high random occurrence), and S box (1/24,000 bp), all responsive to elicitation, wounding, and pathogens (Rushton et al., 2002); for parsley (*Petroselinum crispum*) FP56 (not detected in the random set; enhanced *4CL* expression; Neustaedter et al., 1999) and E box (1/31,000 bp; elicitation; Grimmig and Matern, 1997); for pea (*Pisum sativum*), AT-rich sequence (1/26,000 bp; elicitation; Seki et al., 1996); for tobacco (*Nicotiana tabacum*), salicylic acid-responsive element (1/18,000 bp; pathogen; Pontier et al., 2001). Furthermore, the joint presence of the Arabidopsis OBP-1-binding site (1/38,000 bp; Chen et al., 1996) with an As-1 box (not detected in the random set; salicylic acid, hydrogen peroxide; Yu et al., 1993; Krawczyk et al., 2002), or a common bean (*Phaseolus vulgaris*) H box (1/7,700 bp; elicitation; Lindsay et al., 2002; also considered without G box) with a G box (1/3300 bp; Loake et al., 1992; in conjunction with an H box responsible for induction by p-coumaric acid), respectively, were tested. For the AC I and AC II elements, one unifying profile was built from all experimentally confirmed AC I and AC II elements from different species to increase sensitivity (see supplemental data). The following AC elements were used: *Eucalyptus gunnii* AC I (Lacombe et al., 2000), common bean AC I and II (Hatton et al., 1995), parsley AC II (Hauffe et al., 1993), and an AC II element (CTCACCAACCCCCAC) from the poplar (*Populus trichocarpa*) *gPtCCoAOMT1* promoter (Chen et al., 2000; C. Chen and W. Boerjan, unpublished data). The occurrence of an AC element at random using this matrix was once per 37,000 bp. In addition, the A box, suggested to work in conjunction with AC elements in parsley, was included, even though not experimentally verified (1/11,000 bp; Logemann et al., 1995). Motifs used were retrieved from or submitted to the PlantCARE database (Lescot et al., 2002; http://intra.psb.ugent.be:8080/PlantCARE/).

# Experimental Verification of Annotation and Expression Study

Expression analysis was carried out in Arabidopsis ecotype Columbia plants. Seeds were surface sterilized and placed on Murashige and Skoog medium supplemented with 10 g L⁻¹ Suc. After the seeds had undergone a cold treatment for homogenous germination (overnight at 4°C), they were exposed to 20°C, 50 μmol m⁻² s⁻¹ light intensity, and 70% relative humidity, under a 16-h-light/8-h-dark cycle. Fourteen days after germination,

Monolignol Biosynthetic Genes in Arabidopsis

plants were transferred to soil and cultivated in a greenhouse. Conditions were as follows: 23°C, 50 μmol m⁻² s⁻¹ light intensity at plant level (MBFR/U 400 W incandescent lamps; Philips, Eindhoven, The Netherlands), 40% relative humidity, and a 16-h-light/8-h-dark cycle, without shielding from incident day light. Material was harvested from a number of plants (within brackets) and pooled: seedling leaves and roots of 14-d-old in vitro plants (*n* = 100); rosette leaves, flowers, and green siliques of 7-week-old plants (*n* = 50); and inflorescence stems at 1-, 3-, 5-, 10-, 15-, and 20 cm length (*n* = 20 for 1, 3, and 5 cm; *n* = 10 for all later stages). At 20 cm, the stems were fully grown.

## RNA Extraction, Primer Design, and RT-PCR

Total RNA was extracted with a LiCl method according to Goormachtig et al. (1995) and digested with DNase I to eliminate residual genomic contamination. Subsequently, 5 μg of total RNA was reverse-transcribed into double-stranded cDNA (cDNA Synthesis System Plus, Amersham Biosciences, Little Chalfont, UK). Primers were designed either with the SPADS program that selects specific primers for a particular gene from the Arabidopsis genome (21 genes; available at http://intra.psb.ugent.be:8080/SPADS/) or manually (*PAL1*, *PAL2*, *PAL3*, *PAL4*, *C4H*, *4CL2*, *4CL3*, *HCT*, *CCoAOMT1*, *COMT*, *F5H1*, *CAD7*, and *CAD8*). Primers were designed to span at least one intron for reliable distinction of amplification from cDNA (except for *C3H2* and *C3H3*, which are single exon genes). In RT-PCR experiments, 25 μL of reaction buffer supplied with the *Taq* polymerase and 50 ng of each primer contained a modified nucleotide mix: 200 pmol of dCTP, dTTP, and dGTP, whereas dATP was reduced to 20 pmol. To each reaction, 0.1 μL of ³³P-labeled dATP (10 mCi mL⁻¹ or 2,500 Ci mmol⁻¹) was added, resulting in a hot:cold dATP ratio of 1:2,500. Products were separated on 3% or 4.5% (w/v) polyacrylamide gels and visualized on dried gels through autoradiography. To increase the reliability of the assays, the PCR reaction was run with at least two template concentrations (1 μL of 1:10 diluted cDNA and 1 μL of undiluted cDNA). These expression categories for a particular gene apply only for comparison of different tissues but not between genes because of the different PCR dynamics of shorter or longer amplification products.

## EST Analysis

Data on size and nature of EST libraries were obtained from http://www.ncbi.nlm.nih.gov/UniLib/, http://www.ncbi.nlm.nih.gov/Entrez/, and additionally for the RIKEN Arabidopsis full-length cDNA clones from Seki et al. (2002). A total of 160,776 Arabidopsis ESTs were grouped into 11 categories: whole plant (35,544; 22.1%), aboveground organs (17,934; 11.2%), seedlings (3,207; 2.0%), roots (20,332; 12.6%), flowers (6,814; 4.2%), inflorescence stems (1,384; 0.9%), siliques and seeds (25,043; 15.6%), pathogen infection (2,366; 1.5%), wounded leaves (707; 0.4%), various stresses (44,007; 27.4%), and yet unclassified ESTs (542; 0.3%). Stress ESTs are from subtracted, normalized, and nonsubtracted, nonnormalized libraries. The whole-plant category includes whole plants, whole rosettes, and cell suspensions as starting material. Aboveground organs include, next to libraries that are described as such, libraries from mixed aboveground sources, such as whole inflorescences. Each EST was assigned to one class only. Although inevitably arbitrary and subjective, this classification was done to create clarity and to allow an easier interpretation of the results. Full details on classes and a complete list of ESTs found for each gene is available as supplemental data and at http://www.psb.ugent.be/bioinformatics/login/.

## Note Added in Proof

During the review process, another study on lignification genes in Arabidopsis was published by Goujon et al. (Goujon T, Sibout R, Eudes A, MacKay J, Jouanin L (2003) Genes involved in the biosynthesis of lignin precursors in *Arabidopsis thaliana*. Plant Physiol Biochem 41: 677–687).

## ACKNOWLEDGMENTS

The authors would like to thank Vincent Thareau for the gene-specific primer design; Pierre Rouzé for helpful discussions; Gunnar von Heijne for help in the interpretation of the TargetP results; Clint Chapple, Lise Jouanin,

Raes et al.

Michael Walter, and Carine Serizet for sharing unpublished data; Vanessa Hostyn for excellent technical assistance; Stephane Rombauts for help with promoter analysis; and Martine de Cock for help in preparing the manuscript.

Received May 6, 2003; returned for revision July 11, 2003; accepted August 18, 2003.

# LITERATURE CITED

Achnine L, Rasmussen S, Blancaflor E, Dixon RA (2002) Metabolic channeling at the entry point into the phenylpropanoid pathway: physical association between L-phenylalanine ammonia-lyase and cinnamate 4-hydroxylase. *In* I El Hadrami, ed, Proceedings of the XXI International Conference on Polyphenols, Imprimerie El-Watania, Marrakech, Morocco, pp 7–8

AGI (2000) Analysis of the genome sequence of the flowering plant *Arabidopsis thaliana*. Nature 408: 796–815

Altschul SF, Madden TL, Schäffer AA, Zhang J, Zhang Z, Miller W, Lipman DJ (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res 25: 3389–3402

Anterola AM, Jeon J-H, Davin LB, Lewis NG (2002) Transcriptional control of monolignol biosynthesis in *Pinus taeda*: factors affecting monolignol ratios and carbon allocation in phenylpropanoid metabolism. J Biol Chem 277: 18272–18280

Anterola AM, Lewis NG (2002) Trends in lignin modification: a comprehensive analysis of the effects of genetic manipulations/mutations on lignification and vascular integrity. Phytochemistry 61: 221–294

Bate NJ, Orr J, Ni W, Meromi A, Nadler-Hassar T, Doerner PW, Dixon RA, Lamb CJ, Elkind Y (1994) Quantitative relationship between phenylalanine ammonia-lyase levels and phenylpropanoid accumulation in transgenic tobacco identifies a rate-determining step in natural product synthesis. Proc Natl Acad Sci USA 91: 7608–7612

Bell-Lelong DA, Cusumano JC, Meyer K, Chapple C (1997) Cinnamate-4-hydroxylase expression in Arabidopsis. Plant Physiol 113: 729–738

Benson DA, Karsch-Mizrachi I, Lipman DJ, Ostell J, Wheeler DL (2003) GenBank. Nucleic Acids Res 31: 23–27

Betz C, McCollum TG, Mayer RT (2001) Differential expression of two cinnamate 4-hydroxylase genes in “Valencia” orange (*Citrus sinensis* Osbeck). Plant Mol Biol 46: 741–748

Blanc G, Hokamp K, Wolfe KH (2003) A recent polyploidy superimposed on older large-scale duplications in the Arabidopsis genome. Genome Res 13: 137–144

Boerjan W, Ralph J, Baucher M (2003) Lignin biosynthesis. Annu Rev Plant Biol 54: 519–546

Borevitz JO, Xia Y, Blount J, Dixon RA, Lamb C (2000) Activation tagging identifies a conserved MYB regulator of phenylpropanoid biosynthesis. Plant Cell 12: 2383–2393

Bowers JE, Chapman BA, Rong J, Paterson AH (2003) Unravelling angiosperm genome evolution by phylogenetic analysis of chromosomal duplication events. Nature 422: 433–438

Brill EM, Abrahams S, Hayes CM, Jenkins CLD, Watson JM (1999) Molecular characterisation and expression of a wound-inducible cDNA encoding a novel cinnamyl-alcohol dehydrogenase enzyme in lucerne (*Medicago sativa* L.). Plant Mol Biol 41: 279–291

Chapple C (1998) Molecular-genetic analysis of plant cytochrome P450-dependent monooxygenases. Annu Rev Plant Physiol Plant Mol Biol 49: 311–343

Chapple CCS, Shirley BW, Zook M, Hammerschmidt R, Somerville SC (1994) Secondary metabolism in *Arabidopsis*. *In* EM Meyerowitz, CR Somerville, eds, Arabidopsis, Cold Spring Harbor Monograph Series, Vol 27. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY, pp 989–1030

Chapple CCS, Vogt T, Ellis BE, Somerville CR (1992) An *Arabidopsis* mutant defective in the general phenylpropanoid pathway. Plant Cell 4: 1413–1424

Chen C, Meyermans H, Burggraeve B, De Rycke RM, Inoue K, De Vleeschauwer V, Steenackers M, Van Montagu MC, Engler GJ, Boerjan WA (2000) Cell-specific and conditional expression of caffeoyl-CoA O-methyltransferase in poplar. Plant Physiol 123: 853–867

Chen F, Kota P, Blount JW, Dixon RA (2001) Chemical syntheses of caffeoyl and 5-OH coniferyl aldehydes and alcohols and determination of lignin O-methyltransferase activities in dicot and monocot species. Phytochemistry 58: 1035–1042

Chen M, McClure JW (2000) Altered lignin composition in phenylalanine ammonia-lyase-inhibited radish seedlings: implications for seed-derived sinapoyl esters as lignin precursors. Phytochemistry 53: 365–370

Chen W, Chao G, Singh KB (1996) The promoter of a H₂O₂-inducible, *Arabidopsis* glutathione S-transferase gene contains closely linked OBF- and OBP1-binding sites. Plant J 10: 955–966

Cramer CL, Edwards K, Dron M, Liang X, Dildine SL, Bolwell GP, Dixon RA, Lamb CJ, Schuch W (1989) Phenylalanine ammonia-lyase gene organization and structure. Plant Mol Biol 12: 367–383

Cukovic D, Ehlting J, VanZiffle JA, Douglas CJ (2001) Structure and evolution of 4-coumarate:coenzyme A ligase (4CL) gene families. Biol Chem 382: 645–654

Czichi U, Kindl H (1977) Phenylalanine ammonia-lyase and cinnamic acid hydroxylase as assembled consecutive enzymes on microsomal membranes of cucumber cotyledons: co-operation and subcellular distribution. Planta 134: 133–143

Dharmawardhana DP, Ellis BE, Carlson JE (1992) Characterization of vascular lignification in *Arabidopsis thaliana*. Can J Bot 70: 2238–2244

Dixon RA, Chen F, Guo D, Parvathi K (2001) The biosynthesis of monolignols: a “metabolic grid”, or independent pathways to guaiacyl and syringyl units? Phytochemistry 57: 1069–1084

Donaldson LA (2001) Lignification and lignin topochemistry: an ultrastructural view. Phytochemistry 57: 859–873

Eddy SR (1998) Profile hidden Markov models. Bioinformatics 14: 755–763

Ehlting J, Büttner D, Wang Q, Douglas CJ, Somssich IE, Kombrink E (1999) Three 4-coumarate:coenzyme A ligases in *Arabidopsis thaliana* represent two evolutionarily divergent classes in angiosperms. Plant J 19: 9–20

Emanuelsson O, Nielsen H, Brunak S, von Heijne G (2000) Predicting subcellular localization of proteins based on their N-terminal amino acid sequence. J Mol Biol 300: 1005–1016

Ermolaeva MD, Wu M, Eisen JA, Salzberg SL (2003) The age of the *Arabidopsis thaliana* genome duplication. Plant Mol Biol 51: 859–866

Florea L, Hartzell G, Zhang Z, Rubin GM, Miller W (1998) A computer program for aligning a cDNA sequence with a genomic DNA sequence. Genome Res 8: 967–974

Franke R, Hemm MR, Denault JW, Ruegger MO, Humphreys JM, Chapple C (2002a) Changes in secondary metabolism and deposition of an unusual lignin in the *ref8* mutant of Arabidopsis. Plant J 30: 47–59

Franke R, Humphreys JM, Hemm MR, Denault JW, Ruegger MO, Cusumano JC, Chapple C (2002b) The *Arabidopsis REF8* gene encodes the 3-hydroxylase of phenylpropanoid metabolism. Plant J 30: 33–45

Goormachtig S, Valerio-Lepiniec M, Szczyglowski K, Van Montagu M, Holsters M, de Bruijn FJ (1995) Use of differential display to identify novel *Sesbania rostrata* genes enhanced by *Azorhizobium caulinodans* infection. Mol Plant-Microbe Interact 8: 816–824

Goujon T, Sibout R, Pollet B, Maba B, Nussaume L, Bechtold N, Lu F, Ralph J, Mila I, Barrière Y et al. (2003) A new *Arabidopsis thaliana* mutant deficient in the expression of O-methyltransferase: I. Impact on lignins and on sinapoyl esters. Plant Mol Biol 51: 973–989

Grimmig B, Matern U (1997) Structure of the parsley caffeoyl-CoA O-methyltransferase gene, harbouring a novel elicitor responsive cis-acting element. Plant Mol Biol 33: 323–341

Guo D, Chen F, Dixon RA (2002) Monolignol biosynthesis in microsomal preparations from lignifying stems of alfalfa (*Medicago sativa* L.). Phytochemistry 61: 657–667

Guo D, Chen F, Wheeler J, Winder J, Selman S, Peterson M, Dixon RA (2001) Improvement of in-rumen digestibility of alfalfa forage by genetic manipulation of lignin O-methyltransferases. Transgenic Res 10: 457–464

Hall TA (1999) BioEdit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT. Nucleic Acids Symp Ser 41: 95–98

Harding SA, Leshkevich J, Chiang VL, Tsai C-J (2002) Differential substrate inhibition couples kinetically distinct 4-coumarate:coenzyme A ligases with spatially distinct metabolic roles in quaking aspen. Plant Physiol 128: 428–438

Hatton D, Sablowski R, Yang M-H, Smith C, Schuch W, Bevan M (1995) Two classes of *cis* sequences contribute to tissue-specific expression of a *PAL2* promoter in transgenic tobacco. Plant J 7: 859–876

Hauffe KD, Lee SP, Subramaniam R, Douglas CJ (1993) Combinatorial interactions between positive and negative *cis*-acting elements control

spatial patterns of 4CL1 expression in transgenic tobacco. Plant J 4: 235–253

Hauffe KD, Paszkowski U, Schulze-Lefert P, Hahlbrock K, Dangl JL, Douglas CJ (1991) A parsley 4CL1 promoter fragment specifies complex expression patterns in transgenic tobacco. Plant Cell 3: 435–443

Hoffmann L, Maury S, Martz F, Geoffroy P, Legrand M (2003) Purification, cloning and properties of an acyltransferase controlling shikimate and quinate ester intermediates in phenylpropanoid metabolism. J Biol Chem 278: 95–103

Hu W-J, Kawaoka A, Tsai C-J, Lung J, Osakabe K, Ebinuma H, Chiang VL (1998) Compartmentalized expression of two structurally and functionally distinct 4-coumarate:CoA ligase genes in aspen (*Populus tremuloides*). Proc Natl Acad Sci USA 95: 5407–5412

Humphreys JM, Chapple C (2002) Rewriting the lignin roadmap. Curr Opin Plant Biol 5: 224–229

Humphreys JM, Hemm MR, Chapple C (1999) New routes for lignin biosynthesis defined by biochemical characterization of recombinant ferulate 5-hydroxylase, a multifunctional cytochrome P450-dependent monooxygenase. Proc Natl Acad Sci USA 96: 10045–10050

Jin H, Cominelli E, Bailey P, Parr A, Mehrtens F, Jones J, Tonelli C, Weisshaar B, Martin C (2000) Transcriptional repression by AtMYB4 controls production of UV-protecting sunscreens in *Arabidopsis*. EMBO J 19: 6150–6161

Jones L, Ennos AR, Turner SR (2001) Cloning and characterization of *irregular xylem4* (*irx4*): a severely lignin-deficient mutant of *Arabidopsis*. Plant J 26: 205–216

Kiedrowski S, Kawalleck P, Hahlbrock K, Somssich IE, Dangl JL (1992) Rapid activation of a novel plant defense gene is strictly dependent on the *Arabidopsis RPM1* disease resistance locus. EMBO J 11: 4677–4684

Krawczyk S, Thurow C, Niggeweg R, Gatz C (2002) Analysis of the spacing between the two palindromes of *activation sequence-1* with respect to binding to different TGA factors and transcriptional activation potential. Nucleic Acids Res 30: 775–781

Kühnle T, Koch U, Heller W, Wellmann E (1987) Chlorogenic acid biosynthesis: characterization of a light-induced microsomal 5-O-(4-coumaroyl)-D-quinate/shikimate 3′-hydroxylase from carrot (*Daucus carota* L.) cell suspension cultures. Arch Biochem Biophys 258: 226–232

Lacombe E, Van Doorsselaere J, Boerjan W, Boudet AM, Grima-Pettenati J (2000) Characterization of *cis*-elements required for vascular expression of the *Cinnamoyl CoA Reductase* gene and for protein-DNA complex formation. Plant J 23: 663–676

Lauvergeat V, Lacombe C, Lacombe E, Lasserre E, Roby D, Grima-Pettenati J (2001) Two cinnamoyl-CoA reductase (CCR) genes from *Arabidopsis thaliana* are differentially expressed during development and in response to infection with pathogenic bacteria. Phytochemistry 57: 1187–1195

Lee D, Ellard M, Wanner LA, Davis KR, Douglas CJ (1995) The *Arabidopsis thaliana* 4-coumarate:CoA ligase (4CL) gene: stress and developmentally regulated expression and nucleotide sequence of its cDNA. Plant Mol Biol 28: 871–884

Lee D, Meyer K, Chapple C, Douglas CJ (1997) Antisense suppression of 4-coumarate:coenzyme A ligase activity in *Arabidopsis* leads to altered lignin subunit composition. Plant Cell 9: 1985–1998

Lehfeldt C, Shirley AM, Meyer K, Ruegger MO, Cusumano JC, Viitanen PV, Strack D, Chapple C (2000) Cloning of the *SNG1* gene of *Arabidopsis* reveals a role for a serine carboxypeptidase-like protein as an acyltransferase in secondary metabolism. Plant Cell 12: 1295–1306

Lescot M, Déhais P, Moreau Y, Van de Peer Y, Rouzé P, Rombauts S (2002) PlantCARE, a database of plant *cis*-acting regulatory elements and a portal to tools for *in silico* analysis of promoter sequences. Nucleic Acids Res 30: 325–327

Lewis NG, Yamamoto E (1990) Lignin: occurrence, biogenesis, and biodegradation. Annu Rev Plant Physiol Plant Mol Biol 41: 455–496

Leyva A, Jarillo JA, Salinas J, Martinez-Zapater JM (1995) Low temperature induces the accumulation of phenylalanine ammonia-lyase and chalcone synthase mRNAs of *Arabidopsis thaliana* in a light-dependent manner. Plant Physiol 108: 39–46

Leyva A, Liang X, Pintor-Toro JA, Dixon RA, Lamb CJ (1992) *cis*-Element combinations determine phenylalanine ammonia-lyase gene tissue-specific expression patterns. Plant Cell 4: 263–271

Li L, Cheng XF, Leshkevich J, Umezawa T, Harding SA, Chiang VL (2001) The last step of syringyl monolignol biosynthesis in angiosperms is

Monolignol Biosynthetic Genes in Arabidopsis

regulated by a novel gene encoding sinapyl alcohol dehydrogenase. Plant Cell 13: 1567–1585

Li L, Osakabe Y, Joshi CP, Chiang VL (1999) Secondary xylem-specific expression of caffeoyl-coenzyme A 3-O-methyltransferase plays an important role in the methylation pathway associated with lignin biosynthesis in loblolly pine. Plant Mol Biol 40: 555–565

Li L, Popko JL, Umezawa T, Chiang VL (2000) 5-Hydroxyconiferyl aldehyde modulates enzymatic methylation for syringyl monolignol formation, a new view of monolignol biosynthesis in angiosperms. J Biol Chem 275: 6537–6545

Li L, Popko JL, Zhang X-H, Osakabe K, Tsai C-J, Joshi CP, Chiang VL (1997) A novel multifunctional O-methyltransferase implicated in a dual methylation pathway associated with lignin biosynthesis in loblolly pine. Proc Natl Acad Sci USA 94: 5461–5466

Lindermayr C, Fliegmann J, Ebel J (2003) Deletion of a single amino acid residue from different 4-coumarate-CoA ligases from soybean results in the generation of new substrate specificities. J Biol Chem 278: 2781–2786

Lindsay WP, McAlister FM, Zhu Q, He X-Z, Dröge-Laser W, Hedrick S, Doerner P, Lamb C, Dixon RA (2002) KAP-2, a protein that binds to the H-box in a bean chalcone synthase promoter, is a novel plant transcription factor with sequence identity to the large subunit of human Ku autoantigen. Plant Mol Biol 49: 503–514

Loake GJ, Faktor O, Lamb CJ, Dixon RA (1992) Combination of H-box [CCTACC(N)~7~CT] and G-box [CACGTG] cis elements is necessary for feed-forward stimulation of a chalcone synthase promoter by the phenylpropanoid-pathway intermediate p-coumaric acid. Proc Natl Acad Sci USA 89: 9230–9234

Logemann E, Parniske M, Hahlbrock K (1995) Modes of expression and common structural features of the complete phenylalanine ammonia-lyase gene family in parsley. Proc Natl Acad Sci USA 92: 5905–5909

Logemann E, Reinold S, Somssich IE, Hahlbrock K (1997) A novel type of pathogen defense-related cinnamyl alcohol dehydrogenase. Biol Chem 378: 909–913

Lois R, Dietrich A, Hahlbrock K, Schulz W (1989) A phenylalanine ammonia-lyase gene from parsley: structure, regulation and identification of elicitor and light responsive *cis*-acting elements. EMBO J 8: 1641–1648

Lu SX, Hrabak EM (2002) An *Arabidopsis* calcium-dependent protein kinase is associated with the endoplasmic reticulum. Plant Physiol 128: 1008–1021

Lukashin AV, Borodovsky M (1998) GeneMark.hmm: new solutions for gene finding. Nucleic Acids Res 26: 1107–1115

Maurer-Stroh S, Eisenhaber B, Eisenhaber F (2002) N-terminal N-myristoylation of proteins: prediction of substrate proteins from amino acid sequence. J Mol Biol 317: 541–557

Maury S, Geoffroy P, Legrand M (1999) Tobacco O-methyltransferases involved in phenylpropanoid metabolism: the different caffeoyl-coenzyme A/5-hydroxyferuloyl-coenzyme A 3/5-O-methyltransferase and caffeic acid/5-hydroxyferulic acid 3/5-O-methyltransferase classes have distinct substrate specificities and expression patterns. Plant Physiol 121: 215–223

Maxwell CA, Harrison MJ, Dixon RA (1993) Molecular characterization and expression of alfalfa isoliquiritigenin 2′-O-methyltransferase, an enzyme specifically involved in the biosynthesis of an inducer of *Rhizobium meliloti* nodulation genes. Plant J 4: 971–981

Meyer K, Cusumano JC, Somerville C, Chapple CCS (1996) Ferulate-5-hydroxylase from *Arabidopsis thaliana* defines a new family of cytochrome P450-dependent monooxygenases. Proc Natl Acad Sci USA 93: 6869–6874

Meyer K, Shirley AM, Cusumano JC, Bell-Lelong DA, Chapple C (1998) Lignin monomer composition is determined by the expression of a cytochrome P450-dependent monooxygenase in *Arabidopsis*. Proc Natl Acad Sci USA 95: 6619–6623

Meyermans H, Morreel K, Lapierre C, Pollet B, De Bruyn A, Busson R, Herdewijn P, Devreese B, Van Beeumen J, Marita JM et al. (2000) Modification in lignin and accumulation of phenolic glucosides in poplar xylem upon down-regulation of caffeoyl-coenzyme A O-methyltransferase, an enzyme involved in lignin biosynthesis. J Biol Chem 275: 36899–36909

Mizutani M, Ohta S, Sato R (1997) Isolation of a cDNA and a genomic clone encoding cinnamate 4-hydroxylase from *Arabidopsis* and its expression manner in planta. Plant Physiol 113: 755–763

Nair RB, Joy RW IV, Kurylo E, Shi X, Schnaider J, Datla RSS, Keller WA, Selvaraj G (2000) Identification of a CYP84 family of cytochrome P450-dependent mono-oxygenase genes in *Brassica napus* and perturbation of their expression for engineering sinapine reduction in the seeds. Plant Physiol 123: 1623–1634

Plant Physiol. Vol. 133, Downloaded from www.plantphysiol.org on February 18, 2015 - Published by www.plant.org Copyright © 2003 American Society of Plant Biologists. All rights reserved.

Nair RB, Xia Q, Kartha CJ, Kurylo E, Hirji RN, Datla R, Selvaraj G (2002) Arabidopsis CYP98A3 mediating aromatic 3-hydroxylation: developmental regulation of the gene, and expression in yeast. Plant Physiol 130: 210–220

Nambara E, McCourt P (1999) Protein farnesylation: a greasy tale. Curr Opin Plant Biol 2: 388–392

Nedelkina S, Jupe SC, Blee KA, Schalk M, Werck-Reichert D, Bolwell GP (1999) Novel characteristics and regulation of a divergent cinnamate 4-hydroxylase (CYP73A15) from French bean: engineering expression in yeast. Plant Mol Biol 39: 1079–1090

Neustaedter D, Lee SP, Douglas CJ (1999) A novel parsley 4CL cis-element is required for developmentally regulated expression and protein-DNA complex formation. Plant J 18: 77–88

Ohl S, Hedrick SA, Chory J, Lamb CJ (1990) Functional properties of a phenylalanine ammonia-lyase promoter from *Arabidopsis*. Plant Cell 2: 837–848

Osakabe K, Tsao CC, Li L, Popko JL, Umezawa T, Carraway DT, Smeltzer RH, Joshi CP, Chiang VL (1999) Coniferyl aldehyde 5-hydroxylation and methylation direct syringyl lignin biosynthesis in angiosperms. Proc Natl Acad Sci USA 96: 8955–8960

Pakusch A-E, Matern U, Schiltz E (1991) Elicitor-inducible caffeoyl-coenzyme A 3-O-methyltransferase from *Petroselinum crispum* cell suspensions: purification, partial sequence, and antigenicity. Plant Physiol 95: 137–143

Parvathi K, Chen F, Guo D, Blount DW, Dixon RA (2001) Substrate preferences of O-methyltransferases in alfalfa suggest new pathways for 3-O-methylation of monolignols. Plant J 25: 193–202

Pellegrini L, Geoffroy P, Fritig B, Legrand M (1993) Molecular cloning and expression of a new class of ortho-diphenol-O-methyltransferases induced in tobacco (*Nicotiana tabacum* L.) leaves by infection or elicitor treatment. Plant Physiol 103: 509–517

Pinçon G, Maury S, Hoffmann L, Geoffroy P, Lapierre C, Pollet B, Legrand M (2001) Repression of O-methyltransferase genes in transgenic tobacco affects lignin synthesis and plant growth. Phytochemistry 57: 1167–1176

Pontier D, Balagué C, Bezombes-Marion I, Tronchet M, Deslandes L, Roby D (2001) Identification of a novel pathogen-responsive element in the promoter of the tobacco gene *HSR203J*, a molecular marker of the hypersensitive response. Plant J 26: 495–507

Quandt K, Frech K, Karas H, Wingender E, Werner T (1995) MatInspector: new fast and versatile tools for detection of consensus matches in nucleotide sequence data. Nucleic Acids Res 23: 4878–4884

Raes J, Van de Peer Y (1999) ForCon: a software tool for the conversion of sequence alignments. EMBnet.news 6; http://www.ebi.ac.uk/embnet.news/vol6_1

Raes J, Vandepoele K, Saeys Y, Simillion C, Van de Peer Y (2003) Investigating ancient duplication events in the *Arabidopsis* genome. J Struct Funct Genom 3: 117–129

Ralph J, Lapierre C, Marita JM, Kim H, Lu F, Hatfield RD, Ralph S, Chapple C, Franke R, Hemm MR et al. (2001) Elucidation of new structures in lignins of CAD- and COMT-deficient plants by NMR. Phytochemistry 57: 993–1003

Randall SK, Crowell DN (1999) Protein isoprenylation in plants. Crit Rev Biochem Mol Biol 34: 325–338

Rasmussen S, Dixon RA (1999) Transgene-mediated and elicitor-induced perturbation of metabolic channeling at the entry point into the phenylpropanoid pathway. Plant Cell 11: 1537–1551

Ro DK, Mah N, Ellis BE, Douglas CJ (2001) Functional characterization and subcellular localization of poplar (*Populus trichocarpa × Populus deltoides*) cinnamate 4-hydroxylase. Plant Physiol 126: 317–329

Ruegger M, Meyer K, Cusumano JC, Chapple C (1999) Regulation of ferulate-5-hydroxylase expression in *Arabidopsis* in the context of sinapate ester biosynthesis. Plant Physiol 119: 101–110

Rushton PJ, Reinstädler A, Lipka V, Lippok B, Somssich IE (2002) Synthetic plant promoters containing defined regulatory elements provide novel insights into pathogen- and wound-induced signaling. Plant Cell 14: 749–762

Rutherford K, Parkhill J, Crook J, Horsnell T, Rice P, Rajandream M-A, Barrell B (2000) Artemis: sequence visualization and annotation. Bioinformatics 16: 944–945

Sablowski RWM, Baulcombe DC, Bevan M (1995) Expression of a flower-specific Myb protein in leaf cells using a viral vector causes ectopic activation of a target promoter. Proc Natl Acad Sci USA 92: 6901–6905

Schix T, Moisan A, Rouzé P (2001) EuGène: an eukaryotic gene finder that combines several sources of evidence. Lect Notes Comput Sci 2066: 111–125

Schmidt HA, Strimmer K, Vingron M, von Haeseler A (2002) TREE-PUZZLE: maximum likelihood phylogenetic analysis using quartets and parallel computing. Bioinformatics 18: 502–504

Schneider K, Hovel K, Witzel K, Hamberger B, Schomburg D, Kombrink E, Stuible HP (2003) The substrate specificity-determining amino acid code of 4-coumarate:CoA ligase. Proc Natl Acad Sci USA 100: 8601–8606

Schoch G, Goepfert S, Morant M, Hehn A, Meyer D, Ullmann P, Werck-Reichhart D (2001) CYP98A3 from *Arabidopsis thaliana* is a 3′-hydroxylase of phenolic esters, a missing link in the phenylpropanoid pathway. J Biol Chem 276: 36566–36574

Séguin A, Laible G, Leyva A, Dixon RA, Lamb CJ (1997) Characterization of a gene encoding a DNA-binding protein that interacts *in vitro* with vascular specific *cis* elements of the phenylalanine ammonia-lyase promoter. Plant Mol Biol 35: 281–291

Seki H, Ichinose Y, Kato H, Shiraishi T, Yamada T (1996) Analysis of *cis*-regulatory elements involved in the activation of a member of chalcone synthase gene family (*PsChs1*) in pea. Plant Mol Biol 31: 479–491

Seki M, Narusaka M, Kamiya A, Ishida J, Satou M, Sakurai T, Nakajima M, Enju A, Akiyama K, Oono Y et al. (2002) Functional annotation of a full-length *Arabidopsis* cDNA collection. Science 296: 141–145

Sewalt VJH, Ni W, Blount JW, Jung HG, Masoud SA, Howles PA, Lamb C, Dixon RA (1997) Reduced lignin content and altered lignin composition in transgenic tobacco down-regulated in expression of L-phenylalanine ammonia-lyase or cinnamate 4-hydroxylase. Plant Physiol 115: 41–50

Shah J, Klessig DF (1996) Identification of a salicylic acid-responsive element in the promoter of the tobacco pathogenesis-related β-1,3-glucanase gene, *PR-2d* Plant J 10: 1089–1101

Shirley AM, McMichael CM, Chapple C (2001) The *sng2* mutant of *Arabidopsis* is defective in the gene encoding the serine carboxypeptidase-like protein sinapoylglucose:choline sinapoyltransferase. Plant J 28: 83–94

Sibout R, Eudes A, Pollet B, Goujon T, Mila I, Granier F, Seguin A, Lapierre C, Jouanin L (2003) Expression pattern of two paralogs encoding cinnamyl alcohol dehydrogenases in *Arabidopsis*: isolation and characterization of the corresponding mutants. Plant Physiol 132: 848–860

Simillion C, Vandepoele K, Van Montagu M, Zabeau M, Van de Peer Y (2002) The hidden duplication past of *Arabidopsis thaliana*. Proc Natl Acad Sci USA 99: 13627–13632

Somssich IE, Werner P, Kiedrowski S, Hahlbrock K (1996) *Arabidopsis thaliana* defense-related protein ELI3 is an aromatic alcohol:NADP⁺ oxidoreductase. Proc Natl Acad Sci USA 93: 14199–14203

Strack D, Sharma V (1985) Vacuolar localization of the enzymatic synthesis of hydroxycinnamic acid esters of malic acid in protoplasts from *Raphanus sativus* leaves. Physiol Plant 65: 45–50

Sugimoto K, Takeda S, Hirochika H (2000) MYB-related transcription factor NtMYB2 induced by wounding and elicitors is a regulator of the tobacco retrotransposon *Tto1* and defense-related genes. Plant Cell 12: 2511–2527

Takeshita N, Fujiwara H, Mimura H, Fitchen JH, Yamada Y, Sato F (1995) Molecular cloning and characterization of S-adenosyl-L-methionine: scoulerine-9-O-methyltransferase from cultured cells of *Coptis japonica*. Plant Cell Physiol 36: 29–36

Tamagnone L, Merida A, Parr A, Mackay S, Culianez-Macia FA, Roberts K, Martin C (1998) The AmMYB308 and AmMYB330 transcription factors from Antirrhinum regulate phenylpropanoid and lignin biosynthesis in transgenic tobacco. Plant Cell 10: 135–154

Tavares R, Aubourg S, Lecharny A, Kreis M (2000) Organization and structural evolution of four multigene families in *Arabidopsis thaliana*: AtLCAD, AtLGT, AtMYST and AtHD-GL2. Plant Mol Biol 42: 703–717

Terashima N, Fukushima K, Tsuchiya S (1986) Heterogeneity in formation of lignin: VII. An autoradiographic study on the formation of guaiacyl and syringyl lignin in poplar. J Wood Chem Technol 6: 495–504

Thompson GA Jr, Okuyama H (2000) Lipid-linked proteins in plants. Prog Lipid Res 39: 19–39

Thompson JD, Higgins DG, Gibson TJ (1994) ClustalW: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res 22: 4673–4680

Urban P, Mignotte C, Kazmaier M, Delorme F, Pompon D (1997) Cloning, yeast expression, and characterization of the coupling of two distantly related *Arabidopsis thaliana* NADPH-cytochrome P450 reductases with P450 CYP73A5. J Biol Chem 272: 19176–19186

Van de Peer Y, De Wachter R (1994) TREECON for Windows: a software package for the construction and drawing of evolutionary trees for the Microsoft Windows environment. Comput Appl Biosci 10: 569–570

Vernon DM, Bohnert HJ (1992) A novel methyl transferase induced by osmotic stress in the facultative halophyte *Mesembryanthemum crystallinum*. EMBO J 11: 2077–2085

Wagner GJ, Hrazdina G (1984) Endoplasmic reticulum as a site of phenylpropanoid and flavonoid metabolism in *Hippeastrum*. Plant Physiol 74: 901–906

Wanner LA, Mittal S, Davis KR (1993) Recognition of the avirulence gene *avrB* from *Pseudomonas syringae* pv. *glycinea* by *Arabidopsis thaliana*. Mol Plant-Microbe Interact 6: 582–591

Williamson JD, Stoop JMH, Massel MO, Conkling MA, Pharr DM (1995) Sequence analysis of a mannitol dehydrogenase cDNA from plants reveals a function for the pathogenesis-related protein ELI3. Proc Natl Acad Sci USA 92: 7148–7152

Winkel-Shirley B (1999) Evidence for enzyme complexes in the phenylpropanoid and flavonoid pathways. Physiol Plant 107: 142–149

Ye Z-H, Kneusel RE, Matern U, Varner JE (1994) An alternative methylation pathway in lignin biosynthesis in *Zinnia*. Plant Cell 6: 1427–1439

Yu LM, Lamb CJ, Dixon RA (1993) Purification and biochemical characterization of proteins which bind to the H-box *cis*-element implicated in transcriptional activation of plant defense genes. Plant J 3: 805–816

Zhang X-H, Chinnappa CC (1997) Molecular characterization of a cDNA encoding caffeoyl-coenzyme A 3-O-methyltransferase of *Stellaria longipes*. J Biosci 22: 161–175

Zhong R, Morrison WH III, Negrel J, Ye Z-H (1998) Dual methylation pathways in lignin biosynthesis. Plant Cell 10: 2033–2046

Monolignol Biosynthetic Genes in Arabidopsis

# Supplemental data: overview tables

## Table Is. Expression characteristics of the PAL gene family in Arabidopsis.

Data from Arabidopsis literature, ESTs, and our semi-quantitative RT-PCR are given in the table as +/- for low, + for moderate, ++ for high, and – for decreasing expression. Because of different PCR dynamics of fragments of different size and separate RNA gel blots, data can be compared only among the different tissues, but not between genes or experiments. In case of chimeric promoter-GUS constructs, only those of Arabidopsis promoters analyzed in Arabidopsis were included. Data from GUS and immunohistochemistry were included whenever available. Shaded fields without a number indicate that the tissue/condition was studied, but no expression detected. ESTs are given in absolute (EST) as well as in relative (EST rel) numbers to account for the different sizes of EST classes and to estimate overrepresentation of ESTs in a particular condition. To this end, the number of ESTs for a particular gene in a given class was divided by the total number of ESTs in this class and multiplied by 100,000 to yield a comparable relative number in ESTs/100,000 ESTs (rounded to the nearest whole number). See Materials and Methods for the full description of classification and total numbers in the different classes. Shaded fields without a number indicate that no ESTs were found in the tissue or condition.

Abbreviations: ER, endoplasmic reticulum; ER-anchored, localization in the ER membrane through the membrane anchor of P450 enzymes; mRNA, RNA gel blots; A, A box; AC, AC-unified element; AT, AT-rich element; E, E box; G+H, G box in conjunction with H box; GCC, GCC box; H, H box; S, S box; SARE, salicylic acid-responsive element. When an element occurs more than once in a particular promoter, the number is given within parentheses after the respective element. Promoter elements searched for, but not found in any of the 34 genes involved in monolignol biosynthesis are: As-1 box in conjunction with an OBP-1 binding site, the jasmonate- and ethylene-responsive element (JERE), the FP56, and the hypersensitive-response element (HSRE). See Materials and Methods for the respective random occurrences of the elements in the Arabidopsis genome.

| Gene       | other names     | AGI number    | signals for localization | expression                                                                 | regulatory elements |
|------------|-----------------|---------------|-------------------------|---------------------------------------------------------------------------|---------------------|
|            |                 |               |                         | **seeding** | **etiolated seeding** | **whole plant** | **aboveground organs** | **roots** | **leaves** | **flower** | **silique, seed** | **inflorescence stem** | **1cm** | **3cm** | **5cm** | **10cm** | **15cm** | **fully grown** | **light** | **sucrose** | **wounded leaves** | **pathogen infection** | **various stresses** | **upstream region** | **introns** |
| Arath; PAL1 | PAL1<sup>a</sup> | At2g37040     | RT-PCR                  | +           |                       | +               |                     | +         |           |             |                   |                       | +       | +       | ++      | ++        | ++        | AC                    | S       | H       |                             |                          |                        |                      |
|            |                 |               | EST (41)<sup>*</sup>    | 2           | 1                     | 1               | 13                  |           |           |             |                   |                       |         |         |         |           |           | 1                     | 20      |         |                             |                          |                        |                      |
|            |                 |               | EST rel (25.5)<sup>*</sup> | 62          | 3                     | 6               | 64                  |           |           |             |                   |                       |         |         |         |           |           | 42                    | 45      | a1      |                             |                          |                        |                      |
|            |                 |               | mRNA<sup>a</sup>        |             |                       |                 |                     |           |           |             |                   |                       |         |         |         |           |           |                       |         |         |                             |                          |                        |                      |
|            |                 |               | mRNA<sup>b</sup>        |             |                       |                 |                     |           |           |             |                   |                       |         |         |         |           |           |                       |         |         |                             |                          |                        |                      |
|            |                 |               | mRNA<sup>c</sup>        |             |                       |                 |                     |           |           |             |                   |                       |         |         |         |           |           |                       |         |         |                             |                          |                        |                      |
|            |                 |               | mRNA<sup>d</sup>        |             |                       |                 |                     |           |           |             |                   |                       |         |         |         |           |           |                       |         |         |                             |                          |                        |                      |
|            |                 |               | mRNA<sup>e</sup>        | +           |                       |                 |                     |           |           |             |                   |                       | ++      | +       | +        |           |           |                       |         |         |                             |                          |                        |                      |
|            |                 |               | mRNA<sup>f</sup>        |             |                       |                 |                     |           |           |             |                   |                       | +       | +       | +        | +         | +         |                       |         |         |                             |                          |                        |                      |
|            |                 |               | mRNA<sup>g</sup>        |             |                       |                 |                     |           |           |             |                   |                       |         |         |         |           |           |                       |         |         |                             |                          |                        |                      |
|            |                 |               | mRNA<sup>h</sup>        |             |                       |                 |                     |           |           |             |                   |                       | +       | +       | +        | +         | +         |                       |         |         |                             |                          |                        |                      |
|            |                 |               | mRNA<sup>i</sup>        |             |                       |                 |                     |           |           |             |                   |                       |         |         |         |           |           |                       |         |         |                             |                          |                        |                      |
|            |                 |               | mRNA<sup>j</sup>        | +           |                       |                 |                     |           |           |             |                   |                       |         |         |         |           |           |                       |         |         |                             |                          |                        |                      |
|            |                 |               | AtPAL1::GUS<sup>a</sup>  | +<sup>a2</sup>| +                     |                 |                     |           |           |             |                   |                       | +<sup>a3</sup> | +<sup>a4</sup> | +<sup>a5</sup> |                       |         | +<sup>a6</sup> | +<sup>a6</sup> | a16                | d3      | d4      |                             |                          |                        |                      |
|            |                 |               | AtPAL1::GUS<sup>d</sup>  |             |                       |                 |                     |           |           |             |                   |                       |         |         |         |           |           |                       |         |         | d2                           |                          |                        |                      |
|            |                 |               | AtPAL1::GUS<sup>f</sup>  |             |                       |                 |                     |           |           |             |                   |                       |         |         |         |           |           |                       |         |         | f1                           |                          |                        |                      |
| Arath; PAL2 | PAL2<sup>e</sup> | At3g53260     | RT-PCR                  | ++          | +                     | +               | +                   | +         |           |             |                   |                       | +       | +       | +        | ++        | ++        | AC(2)                 | S       |         |                             |                          |                        |                      |
|            |                 |               | EST (50)                |             | 2                     | 2               | 17                  |           |           |             |                   |                       |         |         |         |           |           | 17                    |         |         |                             |                          |                        |                      |
|            |                 |               | EST rel (31.1)          |             | 6                     | 11              | 84                  |           |           |             |                   |                       |         |         |         |           |           | 39                    |         | b1      |                             |                          |                        |                      |
|            |                 |               | mRNA<sup>b</sup>        |             |                       |                 |                     |           |           |             |                   |                       |         |         |         |           |           |                       |         |         |                             |                          |                        |                      |
|            |                 |               | mRNA<sup>e</sup>        |             |                       |                 |                     |           |           |             |                   |                       | ++      | +       | +        |           |           |                       |         |         |                             |                          |                        |                      |
|            |                 |               | mRNA<sup>g</sup>        |             |                       |                 |                     |           |           |             |                   |                       |         |         |         |           |           |                       |         |         |                             |                          |                        |                      |
|            |                 |               | mRNA<sup>h</sup>        |             |                       |                 |                     |           |           |             |                   |                       | +       | +       | +        | +         | +         |                       |         |         |                             |                          |                        |                      |
|            |                 |               | mRNA<sup>i</sup>        | +           |                       |                 |                     |           |           |             |                   |                       |         |         |         |           |           |                       |         |         |                             |                          |                        |                      |
|            |                 |               | RT-PCR<sup>k</sup>      | +<sup>k1</sup>|                       |                 |                     |           |           |             |                   |                       |         |         |         |           |           |                       | +/–     | +/–     |                             |                          |                        |                      |
| Arath; PAL3 | PAL3<sup>e</sup> | At5g04230     | RT-PCR                  | +           | +                     | ++              | +                   | +/–       |           |             |                   |                       | +       | +/–     | +        | +/–       | +        | E                     | GCC     |         |                             |                          |                        |                      |
|            |                 |               | EST (1)<sup>*</sup>     |             |                       |                 |                     |           |           |             |                   |                       |         |         |         |           |           |                       |         |         |                             |                          |                        |                      |
|            |                 |               | EST rel (0.6)           |             |                       |                 |                     |           |           |             |                   |                       |         |         |         |           |           |                       |         |         |                             |                          |                        |                      |
|            |                 |               | mRNA<sup>e</sup>        |             |                       |                 |                     |           |           |             |                   |                       |         |         |         |           |           |                       |         |         |                             |                          |                        |                      |
|            |                 |               | mRNA<sup>g</sup>        |             |                       |                 |                     |           |           |             |                   |                       |         |         |         |           |           |                       |         |         |                             |                          |                        |                      |
|            |                 |               | mRNA<sup>h</sup>        |             |                       |                 |                     |           |           |             |                   |                       |         | +       | +/–      | +         | +        |                       | +       | –       |                             |                          |                        |                      |
|            |                 |               | mRNA<sup>i</sup>        |             |                       |                 |                     |           |           |             |                   |                       |         |         |         |           |           |                       |         |         |                             |                          |                        |                      |
|            |                 |               | RT-PCR<sup>e</sup>      | +           |                       |                 |                     |           |           |             |                   |                       |         |         |         |           |           |                       |         |         |                             |                          |                        |                      |
| Arath; PAL4 |                 | At3g10340     | RTPCR                   | +           | +                     | +               | +                   | +         |           |             |                   |                       | +       | +       | +        | ++        | ++        | A                     | G+H     |         |                             |                          |                        |                      |
|            |                 |               | EST (28)                |             | 2                     | 1               | 5                   |           |           |             |                   |                       |         |         |         |           |           | 4                     | 9       |         |                             |                          |                        |                      |
|            |                 |               | EST rel (17.4)          |             | 6                     | 6               | 25                  |           |           |             |                   |                       |         |         |         |           |           |                       |         |         |                             |                          |                        |                      |

---

a Ohl *et al.*, 1990  
induction by HgCl₂  

a1 in all tissues except the root tip and the shoot apical meristem  

a2 except the root tip, strong in vascular tissue  

a3 in vascular tissue  

a4 in sepals, anthers and carpels, not in petals, very strong in pollen  

a5 GUS transcript  

b Wanner *et al.*, 1993  

b1 *Pseudomonas* infection  

c Deikman and Hammer, 1995  

c1 cytokinin induction  

d Leyva *et al.*, 1995  

d1 low temperature  

d2 very strong GUS activity in protophylem cells  

d3 GUS transcript, upon *Pseudomonas* infection  

d4 at low temperatures, GUS activity in the cortical cells (photosynthetically active) of the inflorescence stem  

e Wanner *et al.*, 1995  

f Mauch-Mani and Slusarenko, 1996  

f1 in vascular tissue  

f2 *Peronospora* infection  

g Lee *et al.*, 1997  

h Mizutani *et al.*, 1997  

i Ehlting *et al.*, 1999  

i1 *Peronospora* infection  

j Ruegger *et al.*, 1999  

k Jin *et al.*, 2000  

k1 in seedling leaves  

* 1 EST is unclassified

Table IIs. Expression characteristics of the C4H gene in Arabidopsis.

See Table 1 for the full explanation of table and abbreviations.

| Gene       | other names | AGI number | localization | expression | regulatory elements |
|------------|-------------|------------|--------------|------------|---------------------|
| Class I    |             |            |              |            |                     |
| Arath;C4H  | CYP73A5     | At2g30490  | ER-anchored  | RT-PCR     | H                   |
|            | REF3        |            |              | EST (29)   |                     |
|            |             |            |              | EST rel (18.0) |                     |
|            |             |            |              | mRNA<sup>a</sup> | +                 |
|            |             |            |              | mRNA<sup>b</sup> | +                 |
|            |             |            |              | mRNA<sup>c</sup> | +                 |
|            |             |            |              | mRNA<sup>d</sup> | +                 |
|            |             |            |              | mRNA<sup>e</sup> | +                 |
|            |             |            |              | mRNA<sup>f</sup> | +                 |
|            |             |            |              | mRNA<sup>h</sup> | +                 |
|            |             |            |              | RT-PCR<sup>g</sup> | +               |
|            |             |            |              | AtC4H::GUS<sup>a</sup> | +           |
|            |             |            |              | AtC4H::GUS<sup>h</sup> | +           |

| method | seedling | etiolated seedling | whole plant | aboveground organs | roots | leaves | flower | silique, seed | inflorescence stem | 1cm | 3cm | 5cm | 10cm | 15cm | fully grown | light | sucrose | wounded leaves | pathogen infection | various stresses | upstream region | introns |
|--------|----------|---------------------|-------------|---------------------|-------|--------|--------|---------------|--------------------|-----|-----|-----|------|------|-------------|-------|---------|-----------------|-------------------|------------------|----------------|---------|
|        | +        | +                   | +           | +                   | 6     | 2      | 1      | 5             | +                  | +   | +   | +   | +    | ++    | ++          | +     |         | 1               | 14                | +                | +           |         |
|        |          |                     |             |                     | 17    | 10     | 15     | 20            |                    |     |     |     |      |      |             |       |         | 32               | 42                |                 |          |         |
|        |          |                     |             |                     |       | +/−    |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | ++               |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | ++               |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |       |        |        |               |                    |     |     |     |      |      |             |       |         | +                |                   |                 |          |         |
|        |          |                     |             |                     |      
Table III. Expression characteristics of the 4CL gene family in Arabidopsis.

See Table I for the full explanation of table and abbreviations.

| Gene          | other names     | AGI number | signals for localization | expression | regulatory elements |
|---------------|-----------------|------------|--------------------------|------------|---------------------|
|               |                 |            | **seeding**              | **etiolated seedling** | **whole plant** | **aboveground organs** | **roots** | **leaves** | **flower** | **silique, seed** | **inflorescence stem** | **1cm** | **3cm** | **5cm** | **10cm** | **15cm** | **fully grown** | **light** | **sucrose** | **wounded leaves** | **pathogen infection** | **various stresses** | **upstream region** | **introns** |
| **Class I**   |                 |            |                          |             |                     |                  |         |         |         |              |                   |       |       |       |        |        |                |        |         |                |                    |                 |                |        |
| Arath; 4CL1   | 4CL1$^{a}$      | At1g51680  | RT-PCR                   | EST (8)     | EST rel (5.0)       | mRNA$^{a}$      | mRNA$^{b}$ | mRNA$^{c}$ | mRNA$^{d}$ | mRNA$^{e}$ | RT-PCR$^{f}$ | +      | +      | +      | +/–    | +      | +           | +      | +      | ++         | ++          | ++         | AC          | +      | 4      | 9      | $a_{1}$ | +      | $f_{1}, f_{2}$ |
| Arath; 4CL2   | 4CL2$^{d}$      | At3g21240  | RT-PCR                   | EST (13)    | EST rel (8.1)       | mRNA$^{d}$      | +         | +         | +         | +         | +/–        | +      | 2      | 6      | 1      | 2      | +/–        | +      | +      | ++         | ++          | ++         | AC(2)        | +      | 2      | 4      | $+_{1} d_{1}$ | $+_{2} d_{2}$ |          |
| Arath; 4CL4   |                 | At3g21230  | RT-PCR                   | EST (2)     | EST rel (1.2)       | +                | +         | +         | +         | +         | +/–        | +      | 2      | 6      | 15     | 8      | +/–        | +      | +      | +          | +           | +          | AT          | H      | 2      | 4      |          |          |          |
| **Class II**  |                 |            |                          |             |                     |                  |         |         |         |              |                   |       |       |       |        |        |                |        |         |                |                    |                 |                |        |
| Arath; 4CL3   | 4CL3$^{d}$      | At1g65060  | RT-PCR                   | EST (8)     | EST rel (5.0)       | RT-PCR$^{d}$    | RT-PCR$^{f}$ | +/–      | +/–      | +/–      | +         | +      | 1      | 1      | 1      | 2      | +/–        | +      | +/–    | +         | +          | +          | H           | +      | 3      | 7      | $d_{3}$ | $d_{3}$ | $++ d_{2}$ | $f_{1}$ |
| **Class 4CL-likes** |             |            |                          |             |                     |                  |         |         |         |              |                   |       |       |       |        |        |                |        |         |                |                    |                 |                |        |
| Arath; 4CL-like 1 |             | At1g20510  |                          |             |                     |                  |         |         |         |              |                   |       |       |       |        |        |                |        |         |                |                    |                 |                |        |
| Arath; 4CL-like 2 |             | At1g20500  |                          |             |                     |                  |         |         |         |              |                   |       |       |       |        |        |                |        |         |                |                    |                 |                |        |
| Arath; 4CL-like 3 |             | At1g20490  |                          |             |                     |                  |         |         |         |              |                   |       |       |       |        |        |                |        |         |                |                    |                 |                |        |
| Arath; 4CL-like 4 |             | At1g20480  |                          |             |                     |                  |         |         |         |              |                   |       |       |       |        |        |                |        |         |                |                    |                 |                |        |
| Arath; 4CL-like 5 |             | At1g62940  |                          |             |                     |                  |         |         |         |              |                   |       |       |       |        |        |                |        |         |                |                    |                 |                |        |
| Arath; 4CL-like 6 |             | At4g19010  |                          |             |                     |                  |         |         |         |              |                   |       |       |       |        |        |                |        |         |                |                    |                 |                |        |
| Arath; 4CL-like 7 |             | At4g05160  |                          |             |                     |                  |         |         |         |              |                   |       |       |       |        |        |                |        |         |                |                    |                 |                |        |
| Arath; 4CL-like 8 |             | At5g63380  |                          |             |                     |                  |         |         |         |              |                   |       |       |       |        |        |                |        |         |                |                    |                 |                |        |
| Arath; 4CL-like 9 |             | At5g38120  |                          |             |                     |                  |         |         |         |              |                   |       |       |       |        |        |                |        |         |                |                    |                 |                |        |

$a$ Lee et al., 1995  
$b$ Pseudomonas infection  
$c$ Lee et al., 1997  
$d$ Mizutani et al., 1997  
$e$ Ehlting et al., 1999  
$f$ Peronospora infection  
$g$ UV irradiation  
$h$ 4CL3 expression not affected by Peronospora or wounding  
$i$ Ruegger et al., 1999  
$j$ Jin et al., 2000  
$k$ in seedling leaves  
$l$ 4CL1 transcript unaffected by sucrose  
$m$ in the first intron

Table IVs. Expression characteristics of the HCT gene in Arabidopsis.

See Table I for the full explanation of table and abbreviations.

| Gene       | other names     | AGI number  | signals for localization | expression | regulatory elements |
|------------|-----------------|-------------|--------------------------|------------|---------------------|
|            |                 |             | method                   |            |                     |
|            |                 |             | seedling                 |            |                     |
|            |                 |             | etiolated seedling       |            |                     |
|            |                 |             | whole plant              |            |                     |
|            |                 |             | aboveground organs       |            |                     |
|            |                 |             | roots                    |            |                     |
|            |                 |             | leaves                   |            |                     |
|            |                 |             | flower                   |            |                     |
|            |                 |             | silique, seed             |            |                     |
|            |                 |             | inflorescence stem        |            |                     |
|            |                 |             | 1cm                      |            |                     |
|            |                 |             | 3cm                      |            |                     |
|            |                 |             | 5cm                      |            |                     |
|            |                 |             | 10cm                     |            |                     |
|            |                 |             | 15cm                     |            |                     |
|            |                 |             | fully grown               |            |                     |
|            |                 |             | light                    |            |                     |
|            |                 |             | sucrose                  |            |                     |
|            |                 |             | wounded leaves           |            |                     |
|            |                 |             | pathogen infection        |            |                     |
|            |                 |             | various stresses         |            |                     |
|            |                 |             | upstream region          |            |                     |
|            |                 |             | introns                  |            |                     |
| Arath; HCT | At5g48930       | RT-PCR      | +                        | +/−        | +                   | +/− | + | + | + | + | + | ++ | ++ | ++ | AC |
|            |                 | EST (57) *  |                          | 7          | 13                  | 11  |   | 1 | 8 |   |   |    |    |    | 15 | G+H |
|            |                 | EST rel (35.5) |                          | 20         | 73                  | 54  |   | 15 | 32 |   |   |    |    |    | 34 |     |

* 2 ESTs are unclassified

Table V. Expression characteristics of the C3H gene family in Arabidopsis.

See Table I for the full explanation of table and abbreviations.

| Gene          | other names     | AGI number   | localization | signals for expression | regulatory elements |
|---------------|-----------------|--------------|--------------|------------------------|---------------------|
| Class I       |                 |              |              |                        |                     |
| Arath; C3H1   | CYP98A3         | At2g40890    | ER-anchored  | RT-PCR                 | +                   |
|               | REF8<sup>b</sup> |              |              | EST (36)               | +/–                 |
|               |                 |              |              | EST rel (22.4)         | +                   |
|               |                 |              |              | mRNA<sup>a</sup>        | 5                   |
|               |                 |              |              | mRNA<sup>b</sup>        | 3                   |
|               |                 |              |              | mRNA<sup>c</sup>        | 5                   |
|               |                 |              |              | AtC3H::GUS<sup>c</sup>  | 14                  |
|               |                 |              |              | C3H<sup>a</sup>         | 17                  |
|               |                 |              |              | C3H<sup>c</sup>         | 25                  |
|               |                 |              |              |                        | 7                   |
|               |                 |              |              |                        | 28                  |
|               |                 |              |              |                        | +/–                 |
|               |                 |              |              |                        | +/–                 |
|               |                 |              |              |                        | +                   |
|               |                 |              |              |                        | ++                  |
|               |                 |              |              |                        | ++                  |
|               |                 |              |              |                        | +                   |
|               |                 |              |              |                        | ++                  |
|               |                 |              |              |                        | ++                  |
|               |                 |              |              |                        | +                   |
|               |                 |              |              |                        | c1                  |
|               |                 |              |              |                        | +                   |
|               |                 |              |              |                        | c2                  |
|               |                 |              |              |                        | +                   |
|               |                 |              |              |                        | c3                  |
|               |                 |              |              |                        | +                   |
|               |                 |              |              |                        | c4                  |
|               |                 |              |              |                        | +                   |
|               |                 |              |              |                        | c5                  |
|               |                 |              |              |                        | +                   |
|               |                 |              |              |                        | AC                  |
|               |                 |              |              |                        | 2                   |
|               |                 |              |              |                        | 283                 |
|               |                 |              |              |                        | 14                  |
|               |                 |              |              |                        | 32                  |
|               |                 |              |              |                        | +                   |
| Class II      |                 |              |              |                        | A                   |
| Arath; C3H2*  | CYP98A8         | At1g74540    |              | RT-PCR                 | +                   |
|               |                 |              |              | EST (1)                | 1                   |
|               |                 |              |              | EST rel (0.6)          | 3                   |
| Arath; C3H3*  | CYP98A9         | At1g74550    |              | RT-PCR                 | +                   |
|               |                 |              |              | EST (0)                | +                   |
|               |                 |              |              | EST rel (0)            | +                   |

a Schoch et al., 2001  
immunolocalization using a polyclonal anti-CYP98A3 antibody; mainly in differentiating xylem, also in secondary phloem in the cortical zone of mature root  

a1 immunolocalization using a polyclonal anti-CYP98A3 antibody; very strong in differentiating xylem  

b Franke et al., 2002a  

c Nair et al., 2002  

c1 expression highest in roots, expressed in stele and endodermis; not expressed in root apical meristem, epidermis and cortex  

c2 in the vascular tissue of stem, petiole, leaf, petal, sepal, anther, stigma  

c3 in the vascular tissue of the silique wall, not in seed  

c4 immunolocalization using a polyclonal anti-CYP98A3 antibody; in stele  

c5 immunolocalization using a polyclonal anti-CYP98A3 antibody; in meta- and protoxylem cells in the young stem, strongest in lignified interfascicular fibers and xylem vessels of older stem  

* C3H2 and C3H3 are single exon genes

Table VIs. Expression characteristics of the CCoAOMT gene family in Arabidopsis.

See Table I for the full explanation of table and abbreviations.

| Gene          | other names         | AGI number     | localization | signals for expression | regulatory elements |
|---------------|---------------------|----------------|--------------|------------------------|---------------------|
|               |                     |                |              |                        |                     |
| Class I       |                     |                |              |                        |                     |
| Arath; CCoAOMT1 | CCoAOMT<sup>c</sup> At4g34050 | RT-PCR       | ++                    | ++                   | +                  | ++                 | +/–                | +                  | ++                 | ++                 | ++                 | ++                 | AC(2)              |
|               |                     | EST (45)       | 1            | 4                      | 19                  | 1                  | 9                  | 1                  |                     |                     |                     |                     | H                  |
|               |                     | EST rel (28.0) | 31           | 11                     | 93                  | 15                 | 72                 | 36                 |                     |                     |                     |                     |                     |
|               |                     | mRNA<sup>c</sup>| +/–          | +/–                    |                     | +c1                |                     |                     |                     |                     |                     |                     |                     |
| Class II      |                     |                |              |                        |                     |                     |                     |                     |                     |                     |                     |                     |                     |
| Arath; CCoAOMT2 | At1g24735          | RT-PCR        | +/–          | +/–                   | +                  | +                  | +/–                | +/–                | +                  | +                  | +                  | +                  | H                  |
|               |                     | EST (0)        |              |                       |                     |                     |                     |                     |                     |                     |                     |                     |                     |
|               |                     | EST rel (0)    |              |                       |                     |                     |                     |                     |                     |                     |                     |                     |                     |
| Arath; CCoAOMT3 | At3g61990 ER       | RT-PCR        | +            | +                    | ++                 |                    | +                  | +                  | +                  | +                  | ++                 | S                  |
|               |                     | EST (6)        |              | 3                      |                     |                     |                     |                     |                     |                     |                     | 3                  |                     |
|               |                     | EST rel (3.7)  |              | 9                      |                     |                     |                     |                     |                     |                     |                     | 7                  |                     |
| Arath; CCoAOMT4 | At3g62000          | RT-PCR        | +            | +                    | +                  |                    | +/–                | +/–                | +                  | +                  | +                  | +                  | 1                  |
|               |                     | EST (2)        |              | 1                      |                     |                     |                     |                     |                     |                     |                     |                     | 2                  |
|               |                     | EST rel (1.2)  |              | 3                      |                     |                     |                     |                     |                     |                     |                     |                     |                     |
| Arath; CCoAOMT5 | At1g67990          | RT-PCR        | +/–          | +                    | +                  | +                  | +                  | +                  | +                  | +                  | +                  | +                  |                     |
|               |                     | EST (1)        |              |                       |                     | 1                  |                     |                     |                     |                     |                     |                     |                     |
|               |                     | EST rel (0.6)  |              |                       |                     | 15                 |                     |                     |                     |                     |                     |                     |                     |
| Arath; CCoAOMT6 | CCoAOMT<sup>a</sup> At1g67980 | RT-PCR      | +/–          |                    |                    |                    | +/–                | +/–                | +                  | +                  | b1                 |                     |
|               |                     | EST (2)        |              | 2                      |                     |                     |                     |                     |                     |                     |                     |                     |                     |
|               |                     | EST rel (1.2)  |              | 6                      |                     |                     |                     |                     |                     |                     |                     |                     |                     |
|               |                     | RT-PCR<sup>b</sup>| +            |                       |                     |                     |                     |                     |                     |                     |                     |                     | +b1                |
| Arath; CCoAOMT7 | At4g26220          | RT-PCR        | ++           | +                    | ++                 | +/–                | +                  | 3                  | +                  | +                  | ++                 | ++                 | 1                  |
|               |                     | EST (4)        |              |                       |                     |                     |                     | 12                 |                     |                     |                     |                     | 2                  |
|               |                     | EST rel (2.5)  |              |                       |                     |                     |                     |                     |                     |                     |                     |                     |                     |

a Zou and Taylor, 1994  
b Jin et al., 2000  
b1 in seedling leaves  
c Goujon et al., 2003a  
c1 highly expressed in the basal portion as compared to the apical portion of the inflorescence stem

Table VII. Expression characteristics of the CCR gene family in Arabidopsis.

See Table I for the full explanation of table and abbreviations.

| Gene          | other names         | AGI number   | signals for localization | expression                                                                 | regulatory elements |
|---------------|---------------------|--------------|--------------------------|-----------------------------------------------------------------------------|---------------------|
|               |                     |              | method                   | seeding         | etiolated seedling | whole plant | aboveground organs | roots | leaves | flower | silique, seed | inflorescence stem | 1cm | 3cm | 5cm | 10cm | 15cm | fully grown | light | sucrose | wounded leaves | pathogen infection | various stresses | introns |
| Class I       |                     |              |                          |                 |                    |             |                  |      |        |        |              |                   |     |     |     |      |      |            |      |        |                |                   |               |         |
| Arath; CCR 1  | CCR1<sup>ab</sup>    | At1g15950    | RT-PCR                   | ++              | ++                 | +           | +                |      |        |        |              |                   | +   | +   | +    | +     | ++    | ++        |      |        | AC              | 2                 | 8               |         |
|               | IRX4<sup>a</sup>     |              | EST (43)                 |                 |                    | 6           | 8                | 8    |        |        |              |                   |     |     |      |       |       |           |      |        |                 | 85                | 18              |         |
|               |                     |              | EST rel (26.7)           |                 |                    | 17          | 45               | 39   |        |        |              |                   |     |     |      |       |       |           |      |        |                 |                   |                 |         |
|               |                     |              | mRNA<sup>b</sup>         |                 |                    |             |                  |      |        |        |              |                   |     |     |      |       |       |           |      |        |                 |                   |                 |         |
|               |                     |              | mRNA<sup>c</sup>         |                 |                    |             |                  |      |        |        |              |                   |     |     |      |       |       |           |      |        |                 |                   |                 |         |
| Arath; CCR 2  | CCR2<sup>ab</sup>    | At1g80820    | RT-PCR                   | +               | +                  |             |                  |      |        |        |              |                   | +/- | +/- | +    | +     | +     | +         |      |        |                 | 1                 |                 |         |
|               |                     |              | EST (4)                  |                 |                    |             |                  | 3    |        |        |              |                   |     |     |      |       |       |           |      |        |                 |                   |                 |         |
|               |                     |              | EST rel (2.5)            |                 |                    |             |                  | 15   |        |        |              |                   |     |     |      |       |       |           |      |        |                 |                   |                 |         |
|               |                     |              | mRNA<sup>b</sup>         |                 |                    |             |                  |      |        |        |              |                   |     |     |      |       |       |           |      |        |                 | 42                | b3             |         |

| Class CCR-likes |                     |              |                          |                 |                    |             |                  |      |        |        |              |                   |     |     |     |      |      |            |      |        |                |                   |               |         |
|------------------|---------------------|--------------|--------------------------|-----------------|--------------------|-------------|------------------|------|--------|--------|--------------|-------------------|-----|-----|-----|------|------|------------|------|--------|----------------|-------------------|---------------|---------|
| Arath; CCR-like 1 |                     | At1g76470    |                          |                 |                    |             |                  |      |        |        |              |                   |     |     |     |      |      |            |      |        |                |                   |               |         |
| Arath; CCR-like 2 |                     | At2g02400    |                          |                 |                    |             |                  |      |        |        |              |                   |     |     |     |      |      |            |      |        |                |                   |               |         |
| Arath; CCR-like 3 |                     | At2g33590    |                          |                 |                    |             |                  |      |        |        |              |                   |     |     |     |      |      |            |      |        |                |                   |               |         |
| Arath; CCR-like 4 |                     | At2g33600    |                          |                 |                    |             |                  |      |        |        |              |                   |     |     |     |      |      |            |      |        |                |                   |               |         |
| Arath; CCR-like 5 |                     | At5g58490    |                          |                 |                    |             |                  |      |        |        |              |                   |     |     |     |      |      |            |      |        |                |                   |               |         |

a Jones *et al.*, 2001  
b Lauvergeat *et al.*, 2001  
b1 Xanthomonas infection  
b2 induced by Xanthomonas infection and salicylic acid  
b3 not induced by methyl jasmonate or ethylene  
c Goujon *et al.*, 2003b  
c1 moderately expressed in the basal part of the inflorescence stem, highly expressed in the apical part of the inflorescence stem

Table VIII. Expression characteristics of the F5H gene family in Arabidopsis.

See Table I for the full explanation of table and abbreviations.

| Gene        | other names     | AGI number   | localization | signals for expression | regulatory elements |
|-------------|-----------------|--------------|--------------|------------------------|---------------------|
|             |                 |              |              | method                 |                     |
|             |                 |              |              | seedling               |                     |
|             |                 |              |              | etiolated seedling     |                     |
|             |                 |              |              | whole plant            |                     |
|             |                 |              |              | aboveground organs     |                     |
|             |                 |              |              | roots                  |                     |
|             |                 |              |              | leaves                 |                     |
|             |                 |              |              | flower                 |                     |
|             |                 |              |              | silique, seed           |                     |
|             |                 |              |              | inflorescence stem      |                     |
|             |                 |              |              | 1cm                    |                     |
|             |                 |              |              | 3cm                    |                     |
|             |                 |              |              | 5cm                    |                     |
|             |                 |              |              | 10cm                   |                     |
|             |                 |              |              | fully grown            |                     |
|             |                 |              |              | light                  |                     |
|             |                 |              |              | sucrose                |                     |
|             |                 |              |              | wounded leaves         |                     |
|             |                 |              |              | pathogen infection      |                     |
|             |                 |              |              | various stresses       |                     |
|             |                 |              |              | upstream region        |                     |
|             |                 |              |              | introns                |                     |

| Arath; F5H1 | CYP84A1 FAH$^a$ | At4g36220 | ER-anchored | RT-PCR EST (2)* EST rel (1.2) mRNA$^b$ mRNA$^c$ mRNA$^e$ mRNA$^f$ RT-PCR$^d$ | ++ | + | ++ | + | + | + | + | + | + | + | + | + | + | H |
|             |                 |           |             |                                                                 |    | 1 |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    | 3 |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |   |   |   |   |
|             |                 |           |             |                                                                 |    |    |    |   |   |   |   |   |   |   |   |  
Table IXs. Expression characteristics of the COMT gene in Arabidopsis.

See Table I for the full explanation of table and abbreviations.

| Gene          | other names         | AGI number       | signals for expression | regulatory elements |
|---------------|---------------------|------------------|------------------------|---------------------|
| Arath; COMT   | OMT1<sup>a</sup> At5g54160 myristoylation RT-PCR EST (99) EST rel (61.6) mRNA<sup>a</sup> mRNA<sup>b</sup> mRNA<sup>d</sup> RT-PCR<sup>c</sup> AtCOMT1::GUS<sup>d</sup> |  |  | SARE* |
|               |                     |                  | seedling | etiolated seedling | whole plant | aboveground organs | roots | leaves | flower | silique, seed | inflorescence stem | 1cm | 3cm | 5cm | 10cm | 15cm | fully grown | light | sucrose | wounded leaves | pathogen infection | various stresses | upstream region | introns |
|               |                     |                  | ++ | + | ++ | + | + | 11 | 16 | 28 | 2 | 22 | + | + | + | ++ | ++ | ++ | 2 | 18 | 85 | 41 |
|               |                     |                  |     |    |    |    |    | 31 | 89 | 138 | 29 | 88 |    |    |    |     |     |     |    |    |    |    |    |
|               |                     |                  |     |    |    |    |    | + |    | + |    |    |    |    |    |    |    |    |    |    |    |    |    |    |
| Class COMT-likes |                     |                  |     |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |
| Arath; COMT-like 1 | At1g21100 |                  |     |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |
| Arath; COMT-like 2 | At1g21110 |                  |     |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |
| Arath; COMT-like 3 | At1g21120 |                  |     |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |
| Arath; COMT-like 4 | At1g21130 |                  |     |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |
| Arath; COMT-like 5 | At1g33030 |                  |     |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |
| Arath; COMT-like 6 | At1g51990 |                  |     |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |
| Arath; COMT-like 7 | At1g63140 |                  |     |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |
| Arath; COMT-like 8 | At1g76790 |                  |     |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |
| Arath; COMT-like 9 | At1g77520 |                  |     |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |
| Arath; COMT-like 10 | At1g77530 |                  |     |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |
| Arath; COMT-like 11 | At3g53140 |                  |     |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |
| Arath; COMT-like 12 | At5g37170 |                  |     |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |
| Arath; COMT-like 13 | At5g35810 |                  |     |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |    |

a Zhang *et al.*, 1997  
b Ruegger *et al.*, 1999  
c Jin *et al.*, 2000  
c1 in seedling leaves  
d Goujon *et al.*, 2003a  
d1 moderately expressed in the basal part of the inflorescence stem, highly expressed in the apical part of the inflorescence stem  
d2 constitutive in 3d-old seedlings, very high in vascular tissues in 12d-old tissues  
d3 basal GUS activity in leaf blade of young leaves, in vascular tissues of mature leaves  
d4 only in the sepal veins  
d5 only in the lignified ends of silique  
d6 very high in xylem, differentiating fibers and mature phloem  
in the first intron

Table Xs. Expression characteristics of the CAD gene family in Arabidopsis.

See Table I for the full explanation of table and abbreviations.

| Gene          | other names         | AGI number    | signals for expression | regulatory elements |
|---------------|---------------------|---------------|------------------------|---------------------|
| Class I       |                     |               |                        |                     |
| Arath; CAD 2  | LCAD-C$^{f}$        | At3g19450     | RT-PCR                 | +                   |
|               |                     |               | EST (33)               | 6                   |
|               |                     |               | EST rel (20.5)         | 17                  |
|               |                     |               | mRNA$^{h}$             | ++                  |
|               |                     |               | RT-PCR                 |                     |
|               |                     |               | CADC::GUS$^{h}$        | +                   |
|               |                     |               | CAD-C$^{h3}$           | +                   |
| Arath; CAD 6  | LCAD-D$^{f}$        | At4g34230     | RT-PCR                 | +                   |
|               |                     |               | EST (23)               | 1                   |
|               |                     |               | EST rel (14.3)         | 31                  |
|               |                     |               | mRNA$^{g}$             |                     |
|               |                     |               | mRNA$^{h}$             |                     |
|               |                     |               | RT-PCR                 |                     |
|               |                     |               | CADD::GUS$^{h}$        | +                   |
|               |                     |               | CAD-D$^{h3}$           | +                   |
| Class II      |                     |               |                        |                     |
| Arath; CAD 3  | LCAD-A$^{f}$        | At4g37970     | RT-PCR                 | ++                  |
|               |                     |               | EST (1)                |                     |
|               |                     |               | EST rel (0.6)          | +                   |
|               |                     |               | RT-PCR$^{f}$           |                     |
| Arath; CAD 4  | LCAD-B$^{f}$        | At4g37980     | RT-PCR                 | ++                  |
|               | ELI3-1$^{a,f2}$     |               | EST (26)               | 1                   |
|               |                     |               | EST rel (16.2)         | 31                  |
|               |                     |               | mRNA$^{ab}$            |                     |
| Arath; CAD 5  | ELI3-2$^{a}$        | At4g37990     | RT-PCR                 | +/-                 |
|               | BAD$^{d}$           |               | EST (2)                | 1                   |
|               |                     |               | EST rel (1.2)          | 3                   |
|               |                     |               | mRNA$^{a}$             |                     |
| Class III     |                     |               |                        |                     |
| Arath; CAD 1  | CAD1$^{c}$          | At4g39330     | RT-PCR                 | ++                  |
|               |                     |               | EST (32)               | 1                   |
|               |                     |               | EST rel (19.9)         | 31                  |
|               |                     |               | RT-PCR$^{e}$           | +                   |
| Arath; CAD 7  | LCAD-E$^{f}$        | At2g21730     | RT-PCR*                | +                   |
|               |                     |               | EST (0)                |                     |
|               |                     |               | EST rel (0)            |                     |
| Arath; CAD 8  | LCAD-F$^{f}$        | At2g21890     | RT-PCR*                | +                   |
|               |                     |               | EST (0)                |                     |
|               |                     |               | EST rel (0)            |                     |
| Arath; CAD 9  |                     | At1g72680     | RT-PCR                 | ++                  |
|               |                     |               | EST (9)                | 1                   |
|               |                     |               | EST rel (5.6)          | 3                   |

---

**Footnotes:**

- $^{a}$ Kiedrowski *et al.*, 1993
- $^{a1}$ Pseudomonas infection
- $^{b}$ Leyva *et al.*, 1995
- $^{c}$ Somers *et al.*, 1995
- $^{d}$ Somssich *et al.*, 1996
- $^{e}$ Jin *et al.*, 2000
- $^{e1}$ in seedling leaves
- $^{f}$ Tavares *et al.*, 2000
- $^{f1}$ not in pollen
- $^{f2}$ ELI3-1 (X67816) is a recombinant clone of ELI3-2 and LCAD-B
- $^{g}$ Goujon *et al.*, 2003a
- $^{g1}$ highly expressed in the basal portion as compared to the apical portion of the inflorescence stem
- $^{h}$ Sibout *et al.*, 2003
- $^{h1}$ Leaves present overall GUS staining with CAD C::GUS but not with CAD D::GUS, where staining is restricted to to vascular tissues
- $^{h2}$ GUS staining close to the bundle cambium and the interfascicular cambium
- $^{h3}$ using an antibody raised against tobacco CAD.
- $^{*}$ coding sequences and 3'UTR cannot be distinguished by RT-PCR

References

Bell-Lelong DA, Cusumano JC, Meyer K, Chapple C (1997) Cinnamate-4-hydroxylase expression in Arabidopsis. Plant Physiol 113: 729-738

Chapple CCS, Vogt T, Ellis BE, Somerville CR (1992) An Arabidopsis mutant defective in the general phenylpropanoid pathway. Plant Cell 4: 1413-1424

Deikman J, Hammer PE (1995) Induction of anthocyanin accumulation by cytokinins in Arabidopsis thaliana. Plant Physiol 108: 47-57

Ehlting J, Büttner D, Wang Q, Douglas CJ, Somssich IE, Kombrink E (1999) Three 4-coumarate:coenzyme A ligases in Arabidopsis thaliana represent two evolutionarily divergent classes in angiosperms. Plant J 19: 9-20

Franke R, Hemm MR, Denault JW, Ruegger MO, Humphreys JM, Chapple C (2002a) Changes in secondary metabolism and deposition of an unusual lignin in the ref8 mutant of Arabidopsis. Plant J 30: 47-59

Goujon T, Ferret V, Mila I, Pollet B, Ruel K, Burlat V, Joseleau J-P, Barrière Y, Lapierre C, Jouanin L (2003a) Down-regulation of AtCCR1 gene in Arabidopsis thaliana: Effects on phenotype, lignins and cell wall degradability. Planta 217: 218-228

Goujon T, Sibout R, Pollet B, Maba B, Nussaume L, Bechtold N, Lu F, Ralph J, Mila I, Barrière Y, Lapierre C, Jouanin L (2003b) A new Arabidopsis thaliana mutant deficient in the expression of O-methyltransferase 1: Impact on lignins and on sinapoyl esters. Plant Mol Biol 51: 973-989

Jin H, Cominelli E, Bailey P, Parr A, Mehrtens F, Jones J, Tonelli C, Weisshaar B, Martin C (2000) Transcriptional repression by AtMYB4 controls production of UV-protecting sunscreens in Arabidopsis. EMBO J 19: 6150-6161

Jones L, Ennos AR, Turner SR (2001) Cloning and characterization of irregular xylem4 (irx4): A severely lignin-deficient mutant of Arabidopsis. Plant J 26: 205-216

Kiedrowski S, Kawalleck P, Hahlbrock K, Somssich IE, Dangl JL (1992) Rapid activation of a novel plant defense gene is strictly dependent on the Arabidopsis RPM1 disease resistance locus. EMBO J 11: 4677-4684

Lauvergeat V, Lacomme C, Lacombe E, Lasserre E, Roby D, Grima-Pettenati J (2001) Two cinnamoyl-CoA reductase (CCR) genes from Arabidopsis thaliana are differentially expressed during development and in response to infection with pathogenic bacteria. Phytochemistry 57: 1187-1195

Lee D, Ellard M, Wanner LA, Davis KR, Douglas CJ (1995) The Arabidopsis thaliana 4-coumarate:CoA ligase (4CL) gene: Stress and developmentally regulated expression and nucleotide sequence of its cDNA. Plant Mol Biol 28: 871-884

Lee D, Meyer K, Chapple C, Douglas CJ (1997) Antisense suppression of 4-coumarate:coenzyme A ligase activity in Arabidopsis leads to altered lignin subunit composition. Plant Cell 9: 1985-1998

Leyva A, Jarillo JA, Salinas J, Martinez-Zapater JM (1995) Low temperature induces the accumulation of phenylalanine ammonia-lyase and chalcone synthase mRNAs of Arabidopsis thaliana in a light-dependent manner. Plant Physiol 108: 39-46

Mauch-Mani B, Slusarenko AJ (1996) Production of salicylic acid precursors is a major function of phenylalanine ammonia-lyase in the resistance of Arabidopsis to Peronospora parasitica. Plant Cell 8: 203-212

Meyer K, Shirley AM, Cusumano JC, Bell-Lelong DA, Chapple C (1998) Lignin monomer composition is determined by the expression of a cytochrome P450-dependent monooxygenase in Arabidopsis. Proc Natl Acad Sci USA 95: 6619-6623

Mizutani M, Ohta S, Sato R (1997) Isolation of a cDNA and a genomic clone encoding cinnamate 4-hydroxylase from Arabidopsis and its expression manner in planta. Plant Physiol 113: 755-763

Nair RB, Xia Q, Kartha CJ, Kurylo E, Hirji RN, Datla R, Selvaraj G (2002) Arabidopsis CYP98A3 mediating aromatic 3-hydroxylation. Developmental regulation of the gene, and expression in yeast. Plant Physiol 130: 210-220

Ohl S, Hedrick SA, Chory J, Lamb CJ (1990) Functional properties of a phenylalanine ammonia-lyase promoter from Arabidopsis. Plant Cell 2: 837-848

Ruegger M, Meyer K, Cusumano JC, Chapple C (1999) Regulation of ferulate-5-hydroxylase expression in Arabidopsis in the context of sinapate ester biosynthesis. Plant Physiol 119: 101-110

Schoch G, Goepfert S, Morant M, Hehn A, Meyer D, Ullmann P, Werck-Reichhart D (2001) CYP98A3 from Arabidopsis thaliana is a 3'-hydroxylase of phenolic esters, a missing link in the phenylpropanoid pathway. J Biol Chem 276:36566-36574

Sibout R, Eudes A, Pollet B, Goujon T, Mila I, Granier F, Seguin A, Lapierre C, Jouanin L (2003) Expression pattern of two paralogs encoding cinnamyl alcohol dehydrogenases in Arabidopsis. Isolation and characterization of the corresponding mutants. Plant Physiol 132:848-860

Somers DA, Nourse JP, Manners JM, Abrahams S, Watson JM (1995) A gene encoding a cinnamyl alcohol dehydrogenase homolog in Arabidopsis thaliana. Plant Physiol 108:1309-1310

Somssich IE, Werner P, Kiedrowski S, Hahlbrock K (1996) Arabidopsis thaliana defense-related protein ELI3 is an aromatic alcohol:NADP⁺ oxidoreductase. Proc Natl Acad Sci USA 93:14199-14203

Tavares R, Aubourg S, Lecharny A, Kreis M (2000) Organization and structural evolution of four multigene families in Arabidopsis thaliana: AtLCAD, AtLGT, AtMYST and AtHD-GL2. Plant Mol Biol 42:703-717

Wanner LA, Mittal S, Davis KR (1993) Recognition of the avirulence gene avrB from Pseudomonas syringae pv. glycinea by Arabidopsis thaliana. Mol Plant-Microbe Interact 6:582-591

Wanner LA, Li G, Ware D, Somssich IE, Davis KR (1995) The phenylalanine ammonia-lyase gene family in Arabidopsis thaliana. Plant Mol Biol 27:327-338

Zhang H, Wang J, Goodman HM (1997) An Arabidopsis gene encoding a putative 14-3-3-interacting protein, caffeic acid/5-hydroxyferulic acid O-methyltransferase. Biochim Biophys Acta 1353:199-202

Zou J, Taylor DC (1994) Isolation of an Arabidopsis thaliana cDNA homologous to parsley (Petroselinum crispum) S-adenosyl-L-methionine:trans-caffeoyl-coenzyme A 3-O-methyl-transferase, an enzyme involved in disease resistance. Plant Physiol Biochem 32:423-427

Supplementary data: primers used

| AGI number | primer 5' | primer 3' | exons(5'3') | bp cDNA |
|------------|-----------|-----------|-------------|----------|
| PAL        |           |           |             |          |
| Arath; PAL-1 | At2g37040 | ATGGAGATTAACGGGGCACAC | GTACCGCCGAGAACACCGCC | 1..2/2 | 507 |
| Arath; PAL-2 | At3g53260 | ATGGATCAAATCGAAGCAATG | GTACCGCCGAGAACACCGCC | 1..2/2 | 473 |
| Arath; PAL-3 | At5g04230 | GCCGCCGCTATCATGGAG    | GCCATGGCGACTTCTGCG   | 2..3/3 | 439 |
| Arath; PAL-4 | At3g10340 | GCCGCCGCAATTATGGAAACAT | GCTGCTCTGCGCTTTGGACA | 2..3/3 | 507 |
| C4H        |           |           |             |          |
| Arath; C4H  | At2g30490 | GCAGAAGGGAGAAAATCAACG | CAATATAATCCCGGGACAGC | 2..3/3 | 499 |
| 4CL        |           |           |             |          |
| Arath; 4CL-1 | At1g51680 | GATTTTGAGCTCGATAAGAGTGGTG | ATTTTGCTAGTTTTGCCCTCA | 1..4/4 | 723 |
| Arath; 4CL-2 | At3g21240 | AGAACCAACGAGTGGACTCAAT | CTGGAAACGGCTTTTAGCA  | 1..2/4 | 563 |
| Arath; 4CL-3 | At1g65060 | CGCTGATCACTACCGATGAACCA | GCTGAAGTGGCTTCTGGATCG | 1..4/7 | 790 |
| Arath; 4CL-4 | At3g21230 | CGGCCTTGTTCGTGAGTGA    | CGTGCACTAGCTCCGGATCA  | 1..3/5 | 774 |
| HCT        |           |           |             |          |
| Arath; HCT  | At5g48930 | CATTCACTCTTTCCCGCTTC  | GTTCCCCATCCTCCTTGGATT | 1..2/2 | 363 |
| C3H        |           |           |             |          |
| Arath; C3H-1 | At2g40890 | CTCATTACGTGAAGGTGAGAAAAGT | GATCTTGACATCTGCGTTGC | 1..3/3 | 785 |
| Arath; C3H-2 | At1g74540 | TTACTCCCTATCATCGTCGCC | AGGTATTGCTCTTGGGCAG  | 1..1/1 | 1458 |
| Arath; C3H-3 | At1g74550 | ACCACTATCATAAATCGCCG CCTAC | TAACTCTTGTGGCAGCCTGG | 1..1/1 | 1431 |
| CCoAOMT     |           |           |             |          |
| Arath; CCoAOMT-1 | At4g34050 | GGCCCTGCTCTTCCCGTTC | GGTGCATCAGGAGGAGCCA | 3..4/4 | 196 |
| Arath; CCoAOMT-2 | At1g24735 | TGGAGAAA ACTGCTTCCGCC | ATCAGCAGCGAGACGCTTG | 1..5/5 | 641 |
| Arath; CCoAOMT-4 | At3g61990 | CGACGTTTCAACGAGCTTT | TAGGCTGATGGTCCCTCTCAT | 1..8/9 | 776 |
| Arath; CCoAOMT-5 | At3g62000 | GTCTCCGTTGCCAGGAGTT | CTGATGGTCTT CGCATCAT | 1..8/9 | 662 |
| Arath; CCoAOMT-7 | At1g67990 | ATGGT CGATTACCTGACAAAG | ACGTCTACATAGCGTGAGACCAT | 1..5/5 | 686 |
| Arath; CCoAOMT-6 | At1g67980 | GCCAATGAAATTCCTACCA | ACAGAGCGTGATACCATCTCC | 1..5/5 | 681 |
| Arath; CCoAOMT-8 | At4g26220 | CTAA GGATGAAGCCAAGGGATT | TGCAAATAGTGATACCATCGC | 1..4/4 | 681 |
| CCR        |           |           |             |          |
| Arath; CCR-1 | At1g15950 | TGTGGATGTT CGCGATGTC | GAGGAGCAAGATGGCCTTTC | 4..5/5 | 299 |
| Arath; CCR-2 | At1g80820 | CTTCTCGACTACGAAGCTCT | GGGTAGTAAACTTGTATGGCTT | 1..4/4 | 673 |
| COMT       |           |           |             |          |
| Arath; COMT | At5g54160 | GTCATGCTCGACC GTATCCT | GCATCTTCGATGACATGTGG | 1..2/4 | 494 |
| F5H        |           |           |             |          |
| Arath; F5H-1 | At4g36220 | GGTCCGGT CGGTCTTTGTAA | CACGAGCCGCTTGT TATCC | 1..2/3 | 235 |
| Arath; F5H-2 | At5g04330 | GGCTTACCCGTCATTGGA   | AGACGACGATGGGGCAGAG  | 1..2/2 | 1423 |
| CAD        |           |           |             |          |
| Arath; CAD-1 | At4g39330 | AACAGAGCATCCGAACAAAGTCT | AGGGCTCAAGGAGTTAGCCA | 1..5/5 | 1060 |
| Arath; CAD-2 | At3g19450 | CTTGGAATGTCTAATTATCCTATG | GAATCAATTTCCCGTCGAGT | 2..4/5 | 646 |
| Arath; CAD-3 | At4g37970 | GAGCGTAGAGGCGTTTGG   | CGAAA ACTAGAGGCAATGTGT | 1..3/6 | 454 |
| Arath; CAD-4 | At4g37980 | GGCTGCGAAAGACGAATC   | AATATCCGCGTGATGTTGT  | 1..4/4 | 928 |
| Arath; CAD-5 | At4g37990 | AGAAAGAGGCGTT CGGAT | AATCACA AAGCGGTACCTA | 1..4/4 | 1025 |
| Arath; CAD-6 | At4g34230 | TGATCTTGGCATGTCTAATTACC | GACATCAA CGAGCAA CCTAT | 2..5/5 | 883 |
| Arath; CAD-7 | At2g21730 | CGACCTGCCAATTTTCTCTCTA | TCAGAATCTAGACGTGATCG | 3..4/4 | 292 |
| Arath; CAD-8 | At2g21890 | CGACCTGCCAATTTTCTCTCTA | TCAGAATCTAGACGTGATCG | 3..4/4 | 292 |
| Arath; CAD-9 | At1g72680 | ACGAGTGCA TGTGGGGGC | CTTGATGTCAATCAGGAAACG | 1..6/6 | 1028 |

Eucalyptus AC I     CCCACCTACC---------
Phaseolus AC I      CCCACCTACC---------
Phaseolus AC II     -CCACCAACC CCC---
Petroselinum AC II  CTCACCAACC CC----
Populus AC          CTCACCAACC CCCAC

Supplementary figure:

alignment of the 5 experimentally confirmed AC elements used in this study. The search matrix was constructed on the boxed region.

# Supplementary data: EST analysis results

## Full list of Arabidopsis ESTs found for the monolignol pathway genes

### Arath; PAL1
AV783398 RAFL5 Arabidopsis thaliana cDNA clone RAFL05-09-D14  
AV544544 Arabidopsis thaliana roots  
AV548578 Arabidopsis thaliana roots  
AI996802 701668476 A.thaliana, Columbia Col-0, root-1  
T452078470 Lambda-PRL2 Arabidopsis thaliana  
AV547892 Arabidopsis thaliana roots  
AV558154 Arabidopsis thaliana green siliques  
Z17535ATTS0109AC16H Arabidopsis thaliana  
AV782023 RAFL4 Arabidopsis thaliana cDNA clone RAFL04-13-B02  
AV547233 Arabidopsis thaliana roots  
AV564792 Arabidopsis thaliana green siliques  
AA71334132901CD4-16 Arabidopsis thaliana  
AV548764 Arabidopsis thaliana roots  
AA71310132661CD4-15 Arabidopsis thaliana  
AV547875 Arabidopsis thaliana roots  
AV552864 Arabidopsis thaliana roots  
AV547003 Arabidopsis thaliana roots  
AV547237 Arabidopsis thaliana roots  
AV544384 Arabidopsis thaliana roots  
AV525294 Arabidopsis thaliana aboveground organs two to six-week old  
AV548162 Arabidopsis thaliana roots  
AV548688 Arabidopsis thaliana roots  
BU635386021G05 Infected Arabidopsis Leaf  
AV830285 RAFL9 Arabidopsis thaliana cDNA clone RAFL09-66-I02  
AV831948 RAFL9 Arabidopsis thaliana cDNA clone RAFL09-97-K08  
AV796746 RAFL9 Arabidopsis thaliana cDNA clone RAFL09-09-H10  
AV796302 RAFL9 Arabidopsis thaliana cDNA clone RAFL09-07-G01  
AV806160 RAFL9 Arabidopsis thaliana cDNA clone RAFL09-45-I17  
AV799715 RAFL9 Arabidopsis thaliana cDNA clone RAFL09-21-C11  
AV803304 RAFL9 Arabidopsis thaliana cDNA clone RAFL09-35-A21  
AV820059 RAFL11 Arabidopsis thaliana cDNA clone RAFL11-08-H24  
AV797825 RAFL9 Arabidopsis thaliana cDNA clone RAFL09-13-P06  
AV786039 RAFL6 Arabidopsis thaliana cDNA clone RAFL06-69-H09  
AV814576 RAFL9 Arabidopsis thaliana cDNA clone RAFL09-83-H11  
AV812944 RAFL9 Arabidopsis thaliana cDNA clone RAFL09-76-G09  
AV799437 RAFL9 Arabidopsis thaliana cDNA clone RAFL09-20-A06  
AV800034 RAFL9 Arabidopsis thaliana cDNA clone RAFL09-22-I01  
AV813866 RAFL9 Arabidopsis thaliana cDNA clone RAFL09-80-J04  
AV807876 RAFL9 Arabidopsis thaliana cDNA clone RAFL09-52-F17  
AV802552 RAFL9 Arabidopsis thaliana cDNA clone RAFL09-32-F13  
AV805910 RAFL9 Arabidopsis thaliana cDNA clone RAFL09-44-K12  

### Arath; PAL2
AV553183 Arabidopsis thaliana roots  
AV553169 Arabidopsis thaliana roots  
AV553022 Arabidopsis thaliana roots  
AV782312 RAFL4 Arabidopsis thaliana cDNA clone RAFL04-16-D08  
AV544960 Arabidopsis thaliana roots  
AV524210 Arabidopsis thaliana aboveground organs two to six-week old  
AV554116 Arabidopsis thaliana roots  
AV553551 Arabidopsis thaliana roots  
AV544717 Arabidopsis thaliana roots  
AV558603 Arabidopsis thaliana green siliques  
AI997452 701554008 A.thaliana, Columbia Col-0, root-2  
AV559270 Arabidopsis thaliana green siliques  
BE662969 EST00113 Arabidopsis Acute Ozone Forward-Subtracted Library Arabidopsis thaliana cDNA clone AtAOzF26 similar to  
phenylalanine ammonia lyase PAL2  
AV547686 Arabidopsis thaliana roots  
AV544709 Arabidopsis thaliana roots  
AV555308 Arabidopsis thaliana green siliques  
AV544318 Arabidopsis thaliana roots  
BE525043 M57H3STM Arabidopsis developing seed  
BE523478 M37F2STM Arabidopsis developing seed  
AV565930 Arabidopsis thaliana green siliques  
AV564149 Arabidopsis thaliana green siliques  
AV552832 Arabidopsis thaliana roots  
AV558919 Arabidopsis thaliana green siliques  
AV545374 Arabidopsis thaliana roots  
AV548841 Arabidopsis thaliana roots  
AV559798 Arabidopsis thaliana green siliques  
BE528729 M72E24STM Arabidopsis developing seed  
R9044516800 Lambda-PRL2 Arabidopsis thaliana  
AV546412 Arabidopsis thaliana roots  
T454298692 Lambda-PRL2 Arabidopsis thaliana  
AV530068 Arabidopsis thaliana aboveground organs two to six-week old  
BE527821 M69P22STM Arabidopsis developing seed  
AV547835 Arabidopsis thaliana roots  
AV555366 Arabidopsis thaliana green siliques  
AV547599 Arabidopsis thaliana roots  

AV826964RAFL9Arabidopsis thaliana cDNA clone RAFL 09-09-P 06
AV797345RAFL9Arabidopsis thaliana cDNA clone RAFL 09-11-L 22
AV797794RAFL9Arabidopsis thaliana cDNA clone RAFL 09-13-M 11
AV801904RAFL9Arabidopsis thaliana cDNA clone RAFL 09-30-A 01
AV814928RAFL9Arabidopsis thaliana cDNA clone RAFL 09-84-L 24
AV796886RAFL9Arabidopsis thaliana cDNA clone RAFL 09-09-P 06
AV790081RAFL6Arabidopsis thaliana cDNA clone RAFL 06-87-A 16
AV821038RAFL11Arabidopsis thaliana cDNA clone RAFL 11-13-K 18
AV797838RAFL9Arabidopsis thaliana cDNA clone RAFL 09-14-A 02
AV813837RAFL9Arabidopsis thaliana cDNA clone RAFL 09-80-F 18
AV820261RAFL11Arabidopsis thaliana cDNA clone RAFL 11-09-F 02
AV809260RAFL9Arabidopsis thaliana cDNA clone RAFL 09-58-N 05
AV819960RAFL11Arabidopsis thaliana cDNA clone RAFL 11-08-C 18
AV811681RAFL9Arabidopsis thaliana cDNA clone RAFL 09-69-E 06

Arath; PAL 3
******************

AW004366701956642A.thaliana, mixed source

Arath; PAL 4
******************

AV551435Arabidopsis thaliana roots
AV542768Arabidopsis thaliana roots
AI999678701557008A.thaliana, Columbia Col-0, rosette-3
AV565915Arabidopsis thaliana green siliques
AV559088Arabidopsis thaliana green siliques
BE524090M45H12STM Arabidopsis developing seed
BE522270M24G2STM Arabidopsis developing seed
AV560700Arabidopsis thaliana green siliques
AV539086Arabidopsis thaliana roots
AV557330Arabidopsis thaliana green siliques
AV552018Arabidopsis thaliana roots
AV542315Arabidopsis thaliana roots
T7652411302Lambda-PRL2 Arabidopsis thaliana
BE526306M65A09STM Arabidopsis developing seed
BE530250M77O05STM Arabidopsis developing seed
BE525762M63M18STM Arabidopsis developing seed
BE522628M27C2STM Arabidopsis developing seed
AV519394Arabidopsis thaliana aboveground organs two to six-week old
AV562501Arabidopsis thaliana green siliques
BE522271M24G2XTM Arabidopsis developing seed
BE523029M31E4STM Arabidopsis developing seed
AV565572Arabidopsis thaliana green siliques
AV557314Arabidopsis thaliana green siliques
AV565155Arabidopsis thaliana green siliques
AV810491RAFL9Arabidopsis thaliana cDNA clone RAFL 09-64-C 18
AV814261RAFL9Arabidopsis thaliana cDNA clone RAFL 09-82-F 08
AV814676RAFL9Arabidopsis thaliana cDNA clone RAFL 09-83-N 01
AV814281RAFL9Arabidopsis thaliana cDNA clone RAFL 09-82-G 07

Arath; C4H
******************

AV542275Arabidopsis thaliana roots
AV559178Arabidopsis thaliana green siliques
AV785095RAFL6Arabidopsis thaliana cDNA clone RAFL 06-11-J 16
N6560120641Lambda-PRL2 Arabidopsis thaliana
AV555916Arabidopsis thaliana green siliques
AV555617Arabidopsis thaliana green siliques
F19837ATTS5876Ors-B Arabidopsis thaliana
AV542126Arabidopsis thaliana roots
AV567438Arabidopsis thaliana green siliques
T0408636Lambda-PRL1 Arabidopsis thaliana
BE530203M76N13STM Arabidopsis developing seed
T437767039Lambda-PRL2 Arabidopsis thaliana
T448748137Lambda-PRL2 Arabidopsis thaliana
AV532445Arabidopsis thaliana flower buds
AA65731731642Lambda-PRL2 Arabidopsis thaliana
AA39539827181Lambda-PRL2 Arabidopsis thaliana
BU635358021A05 Infected Arabidopsis Leaf
AV796909RAFL9Arabidopsis thaliana cDNA clone RAFL 09-10-A 17
AV795722RAFL8Arabidopsis thaliana cDNA clone RAFL 08-18-N 12
AV798636RAFL9Arabidopsis thaliana cDNA clone RAFL 09-16-P 18
AV794052RAFL8Arabidopsis thaliana cDNA clone RAFL 08-11-F 09
AV797467RAFL9Arabidopsis thaliana cDNA clone RAFL 09-12-D 20
AV813906RAFL9Arabidopsis thaliana cDNA clone RAFL 09-80-M 18
AV786111RAFL6Arabidopsis thaliana cDNA clone RAFL 06-69-M 08
AV805684RAFL9Arabidopsis thaliana cDNA clone RAFL 09-43-M 24
AV804279RAFL9Arabidopsis thaliana cDNA clone RAFL 09-38-I 15
AV797977RAFL9Arabidopsis thaliana cDNA clone RAFL 09-14-I 08
AV799518RAFL9Arabidopsis thaliana cDNA clone RAFL 09-20-F 15
AV807736RAFL9Arabidopsis thaliana cDNA clone RAFL 09-51-M 04

Arath; 4CL 1
******************

BE523491M37G6STM Arabidopsis developing seed
U74152NaCl-treated Arabidopsis subtraction library
AI998835701547060A.thaliana, Columbia Col-0, rosette-2

N9664821467CD 4-15 Arabidopsis thaliana cDNA clone G2D3T 7  
N9584021982 Lambda-PRL1 Arabidopsis thaliana cDNA clone G9F10T 7  
ATU74152 NaCl-treated Arabidopsis subtraction library  
AV815209 RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-85-N 04  
AV813552 RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-78-P 07  

Arath; 4CL 2  
**  
AV551242 Arabidopsis thaliana roots  
AV532716 Arabidopsis thaliana flower buds  
AV539578 Arabidopsis thaliana roots  
AV543520 Arabidopsis thaliana roots  
AA712582 31715 Lambda-PRL2 Arabidopsis thaliana  
T42724 5987 Lambda-PRL2 Arabidopsis thaliana  
AV554267 Arabidopsis thaliana roots  
AV563141 Arabidopsis thaliana green siliques  
AV548209 Arabidopsis thaliana roots  
BE529209 M73P11STM Arabidopsis developing seed  
AV540520 Arabidopsis thaliana roots  
AV814104 RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-81-L 19  
AV814858 RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-84-I 04  

Arath; 4CL 3  
**  
AV548526 Arabidopsis thaliana roots  
AV558658 Arabidopsis thaliana green siliques  
AV533259 Arabidopsis thaliana flower buds  
AV558307 Arabidopsis thaliana green siliques  
AI999552 701556371A. thaliana, Columbia Col-0, rosette-3  
AV802214 RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-31-C 01  
AV787325 RAFL6 Arabidopsis thaliana cDNA clone RAFL 06-75-B 04  
AV807453 RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-50-H 19  

Arath; 4CL 4  
**  
AV823321 RAFL5 Arabidopsis thaliana cDNA clone RAFL 05-17-O 13  
AV801351 RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-27-M 06  

Arath; HCT  
**  
AV538507 Arabidopsis thaliana roots  
AV553284 Arabidopsis thaliana roots  
AV518201 Arabidopsis thaliana aboveground organs two to six-week old  
AV525886 Arabidopsis thaliana aboveground organs two to six-week old  
AV525131 Arabidopsis thaliana aboveground organs two to six-week old  
AV527096 Arabidopsis thaliana aboveground organs two to six-week old  
AV520681 Arabidopsis thaliana aboveground organs two to six-week old  
AV782261 RAFL4 Arabidopsis thaliana cDNA clone RAFL 04-15-L 12  
AV544780 Arabidopsis thaliana roots  
AV524880 Arabidopsis thaliana aboveground organs two to six-week old  
BE522327 M25B5STM Arabidopsis developing seed  
BE524719 M53D9STM Arabidopsis developing seed  
AA650678 30792 Lambda-PRL2 Arabidopsis thaliana  
AV550814 Arabidopsis thaliana roots  
AV555054 Arabidopsis thaliana green siliques  
AV538516 Arabidopsis thaliana roots  
AV534641 Arabidopsis thaliana flower buds  
AV540733 Arabidopsis thaliana roots  
F19979 ATTS6018AC16H Arabidopsis thaliana  
T44796 8059 Lambda-PRL2 Arabidopsis thaliana  
AV538890 Arabidopsis thaliana roots  
T44079 7342 Lambda-PRL2 Arabidopsis thaliana  
AV551255 Arabidopsis thaliana roots  
AV546668 Arabidopsis thaliana roots  
AV785581 RAFL6 Arabidopsis thaliana cDNA clone RAFL 06-67-G 22  
AV518079 Arabidopsis thaliana aboveground organs two to six-week old  
AV519665 Arabidopsis thaliana aboveground organs two to six-week old  
AV563393 Arabidopsis thaliana green siliques  
AV796296 RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-07-F 17  
BE525451 M62A03STM Arabidopsis developing seed  
AA720350 33543 Lambda-PRL2 Arabidopsis thaliana  
BE529452 M74H10STM Arabidopsis developing seed  
AV559259 Arabidopsis thaliana green siliques  
T43446 6709 Lambda-PRL2 Arabidopsis thaliana  
AV788842 RAFL6 Arabidopsis thaliana cDNA clone RAFL 06-81-D 12  
BE525857 M63F24STM Arabidopsis developing seed  
AV538914 Arabidopsis thaliana roots  
AV522470 Arabidopsis thaliana aboveground organs two to six-week old  
T76455 11233 Lambda-PRL2 Arabidopsis thaliana  
AV818106 RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-96-P 06  
AV798258 RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-15-J 05  
AV540806 Arabidopsis thaliana roots  
AV807739 RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-51-M 09  
AV518127 Arabidopsis thaliana aboveground organs two to six-week old  
AV791028 RAFL7 Arabidopsis thaliana cDNA clone RAFL 07-08-I 01  
AV799318 RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-19-I 04  
AA395836 25824 Lambda-PRL2 Arabidopsis thaliana  


AV820131 RAFL 11 Arabidopsis thaliana cDNA clone RAFL 11-08-L 22
BE529847 M75F07STM Arabidopsis developing seed Arabidopsis thaliana
AV788364 RAFL 6 Arabidopsis thaliana cDNA clone RAFL 06-79-D 12
AV524635 Arabidopsis thaliana aboveground organs two to six-week old
AV815548 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-87-B 19
AV525642 Arabidopsis thaliana aboveground organs two to six-week old
AV805127 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-41-M 15
AV802062 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-30-I 21
AV518547 Arabidopsis thaliana aboveground organs two to six-week old
AV814070 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-81-J 12

Arath; C3H 1
******************

AV784358 RAFL 5 Arabidopsis thaliana cDNA clone RAFL 05-18-L 20
AV549623 Arabidopsis thaliana roots
AV537635 Arabidopsis thaliana roots
AV550871 Arabidopsis thaliana roots
AV562507 Arabidopsis thaliana green siliques
AV538561 Arabidopsis thaliana roots
AV565180 Arabidopsis thaliana green siliques
AV563863 Arabidopsis thaliana green siliques
AV565392 Arabidopsis thaliana green siliques
H36469 14137 Lambda-PRL 2 Arabidopsis thaliana
AV543094 Arabidopsis thaliana roots
AA728684 33738 CD 4-6 Arabidopsis thaliana cDNA
AV527527 Arabidopsis thaliana aboveground organs two to six-week old
T43253 6516 Lambda-PRL 2 Arabidopsis thaliana
AI999087 701554413 A. thaliana, Columbia Col-0, rosette-3
Z35017 ATTS 3656 Strasbourg-A Arabidopsis thaliana
AV562444 Arabidopsis thaliana green siliques
AV520969 Arabidopsis thaliana aboveground organs two to six-week old
BE526379 M65A08STM Arabidopsis developing seed
N37715 18942 Lambda-PRL 2 Arabidopsis thaliana
A651162 31276 Lambda-PRL 2 Arabidopsis thaliana
AV561166 Arabidopsis thaliana green siliques
Z35213 ATTS 3759 Strasbourg-A Arabidopsis thaliana
AV823400 RAFL 5 Arabidopsis thaliana cDNA clone RAFL 05-18-L 20
AV824836 RAFL 6 Arabidopsis thaliana cDNA clone RAFL 06-77-O 07
AV794682 RAFL 8 Arabidopsis thaliana cDNA clone RAFL 08-13-O 16
AV795907 RAFL 8 Arabidopsis thaliana cDNA clone RAFL 08-19-L 09
AV818991 RAFL 11 Arabidopsis thaliana cDNA clone RAFL 11-01-L 19
AV797787 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-13-L 23
AV808640 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-55-L 13
AV802971 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-33-N 18
AV795014 RAFL 8 Arabidopsis thaliana cDNA clone RAFL 08-15-J 24
AV810928 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-65-O 22
AV815361 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-86-G 17
AV799990 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-22-F 16
AV812838 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-75-O 04

Arath; C3H 2
******************

AA651146 31260 Lambda-PRL 2 Arabidopsis thaliana

Arath; C3H 3
******************

(none)

Arath; CCoAOMT 1
******************

AV539060 Arabidopsis thaliana roots
AV782400 RAFL 4 Arabidopsis thaliana cDNA clone RAFL 04-17-C 19
AV551773 Arabidopsis thaliana roots
AV550631 Arabidopsis thaliana roots
AV550365 Arabidopsis thaliana roots
BE844710 AD01C10T7ADA. thaliana (Col-0gl 1) library enriched for salt-induced transcripts
AV538779 Arabidopsis thaliana roots
AV538609 Arabidopsis thaliana roots
AV543668 Arabidopsis thaliana roots
AV551403 Arabidopsis thaliana roots
AV542254 Arabidopsis thaliana roots
AV539351 Arabidopsis thaliana roots
AV564395 Arabidopsis thaliana green siliques
AV557479 Arabidopsis thaliana green siliques
AI995849 701548909 A. thaliana, Columbia Col-0, inflorescence-2
AV543497 Arabidopsis thaliana roots
BE520306 M11D2STM Arabidopsis developing seed
AV551095 Arabidopsis thaliana roots
AV550658 Arabidopsis thaliana roots
AV552849 Arabidopsis thaliana roots

AV552488 Arabidopsis thaliana roots
T438797142 Lambda-PRL2 Arabidopsis thaliana
AV552264 Arabidopsis thaliana roots
AV565601 Arabidopsis thaliana green siliques
AV538191 Arabidopsis thaliana roots
AV535845 Arabidopsis thaliana flower buds
AV560983 Arabidopsis thaliana green siliques
AA04264124873CD4-13 Arabidopsis thaliana
AV567247 Arabidopsis thaliana green siliques
AV567284 Arabidopsis thaliana green siliques
AV564044 Arabidopsis thaliana green siliques
AV544531 Arabidopsis thaliana roots
T435086771 Lambda-PRL2 Arabidopsis thaliana
AI993976701495352A.thaliana, Ohio State clone set
AV563762 Arabidopsis thaliana green siliques
T442767539 Lambda-PRL2 Arabidopsis thaliana
AV544019 Arabidopsis thaliana roots
BU635690004G05 Infected Arabidopsis Leaf
AV829598RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-48-L 18
AV787635RAFL6 Arabidopsis thaliana cDNA clone RAFL 06-76-G 23
AV819973RAFL11 Arabidopsis thaliana cDNA clone RAFL 11-08-D 12
AV804325RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-38-L 04
AV789655RAFL6 Arabidopsis thaliana cDNA clone RAFL 06-84-H 12
AV805647RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-43-L 03
AV786958RAFL6 Arabidopsis thaliana cDNA clone RAFL 06-73-I 10

Arath; CCoAOMT 2
(none)

Arath; CCoAOMT 3

T438327095 Lambda-PRL2 Arabidopsis thaliana
AI239397EST100 Arabidopsis pSMASH Library
AA39521326996 Lambda-PRL2 Arabidopsis thaliana
AV789440RAFL6 Arabidopsis thaliana cDNA clone RAFL 06-83-K 13
AV789331RAFL6 Arabidopsis thaliana cDNA clone RAFL 06-83-D 19
AV801235RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-27-G 01

Arath; CCoAOMT 4

AV784089RAFL5 Arabidopsis thaliana cDNA clone RAFL 05-16-C 19
AI997777701670681A.thaliana, Columbia Col-0, rosette-1

Arath; CCoAOMT 5

F15385ATTS5633 Grenoble-B Arabidopsis thaliana

Arath; CCoAOMT 6

R8405316012 Lambda-PRL2 Arabidopsis thaliana
AA39453326116 Lambda-PRL2 Arabidopsis thaliana cDNA clone 132K23XP

Arath; CCoAOMT 7

AV557578 Arabidopsis thaliana green siliques
AV556958 Arabidopsis thaliana green siliques
AV565028 Arabidopsis thaliana green siliques
AV798120RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-15-A 15

Arath; CCR 1

AV550189 Arabidopsis thaliana roots
AV519636 Arabidopsis thaliana aboveground organs two to six-week old
AV534376 Arabidopsis thaliana flower buds
AI992693701493334A.thaliana, Ohio State clone set
AV784273RAFL5 Arabidopsis thaliana cDNA clone RAFL 05-17-P 18
AV440533 Arabidopsis thaliana above-ground organ two to six-week old
BE521298M18G8STM Arabidopsis developing seed
BE520377M12A7STM Arabidopsis developing seed
BE523063M31H3STM Arabidopsis developing seed
T4176510346 Lambda-PRL2 Arabidopsis thaliana
AV539977 Arabidopsis thaliana roots
AV538045 Arabidopsis thaliana roots
AV552663 Arabidopsis thaliana roots

AV517929 Arabidopsis thaliana aboveground organs two to six-week old
AV517903 Arabidopsis thaliana aboveground organs two to six-week old
AV543473 Arabidopsis thaliana roots
AI995837701548853A. thaliana, Columbia Col-0, inflorescence - 2
BE527222M67F13STM Arabidopsis developing seed
T138402005 Lambda-PRL2 Arabidopsis thaliana
AV524462 Arabidopsis thaliana aboveground organs two to six-week old
R8701516211 Lambda-PRL2 Arabidopsis thaliana
AV783937RAFL5 Arabidopsis thaliana cDNA clone RAFL 05-14-K 06
AV535365 Arabidopsis thaliana flower buds
AV519164 Arabidopsis thaliana aboveground organs two to six-week old
AU227285RAFL14 Arabidopsis thaliana cDNA clone RAFL 14-95-L 15
AV558509 Arabidopsis thaliana green siliques
AV560184 Arabidopsis thaliana green siliques
T04049253AT-NHC Arabidopsis thaliana cDNA clone B98XP
AV563488 Arabidopsis thaliana green siliques
AV557438 Arabidopsis thaliana green siliques
AV519439 Arabidopsis thaliana aboveground organs two to six-week old
AV519149 Arabidopsis thaliana aboveground organs two to six-week old
N9590221311 Lambda-PRL1 Arabidopsis thaliana cDNA clone F4H9T 7
AV541538 Arabidopsis thaliana roots
BU634974013F08 Infected Arabidopsis Leaf
AU236375RAFL14 Arabidopsis thaliana cDNA clone RAFL 14-95-L 15
BU636544007B10 Infected Arabidopsis Leaf
AV792957RAFL7 Arabidopsis thaliana cDNA clone RAFL 07-17-H 20
AV788803RAFL6 Arabidopsis thaliana cDNA clone RAFL 06-81-B 11
AV790470RAFL6 Arabidopsis thaliana cDNA clone RAFL 06-89-A 04
AV785890RAFL6 Arabidopsis thaliana cDNA clone RAFL 06-68-M 21
AV811929RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-70-F 15
AV808728RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-56-C 11

Arath; CCR2
******************

AV542970 Arabidopsis thaliana roots
AI996461701666296A. thaliana, Columbia Col-0, root - 1
Z24528ATTS970Ra 147.1 Arabidopsis thaliana cDNA clone RaR 108
AU226236RAFL14 Arabidopsis thaliana cDNA clone RAFL 14-17-P 12

Arath; COMT
******************

AV521002 Arabidopsis thaliana aboveground organs two to six-week old
AV439990 Arabidopsis thaliana above-ground organ two to six-week old
AV535316 Arabidopsis thaliana flower buds
AV553956 Arabidopsis thaliana roots
AV539281 Arabidopsis thaliana roots
AV552412 Arabidopsis thaliana roots
AV549386 Arabidopsis thaliana roots
AV784730RAFL6 Arabidopsis thaliana cDNA clone RAFL 06-07-M 15
AV537483 Arabidopsis thaliana roots
AV527585 Arabidopsis thaliana aboveground organs two to six-week old
AV527925 Arabidopsis thaliana aboveground organs two to six-week old
AV527964 Arabidopsis thaliana aboveground organs two to six-week old
AV543484 Arabidopsis thaliana roots
AV524944 Arabidopsis thaliana aboveground organs two to six-week old
AV527914 Arabidopsis thaliana aboveground organs two to six-week old
AV540557 Arabidopsis thaliana roots
AV521202 Arabidopsis thaliana aboveground organs two to six-week old
AV521209 Arabidopsis thaliana aboveground organs two to six-week old
AV539934 Arabidopsis thaliana roots
AV565263 Arabidopsis thaliana green siliques
AV563462 Arabidopsis thaliana green siliques
AV562516 Arabidopsis thaliana green siliques
AV566702 Arabidopsis thaliana green siliques
AV518234 Arabidopsis thaliana aboveground organs two to six-week old
AV551301 Arabidopsis thaliana roots
AV540995 Arabidopsis thaliana roots
AV540675 Arabidopsis thaliana roots
AV551694 Arabidopsis thaliana roots
AV562653 Arabidopsis thaliana green siliques
AV564711 Arabidopsis thaliana green siliques
AV565589 Arabidopsis thaliana green siliques
AV542320 Arabidopsis thaliana roots
AV543423 Arabidopsis thaliana roots
AV564449 Arabidopsis thaliana green siliques
AV542044 Arabidopsis thaliana roots
T8828711983 Lambda-PRL2 Arabidopsis thaliana
T7588410662 Lambda-PRL2 Arabidopsis thaliana
AV518308 Arabidopsis thaliana aboveground organs two to six-week old
AV540359 Arabidopsis thaliana roots
AV540342 Arabidopsis thaliana roots
AV541883 Arabidopsis thaliana roots

AV562304 Arabidopsis thaliana green siliques
AV561324 Arabidopsis thaliana green siliques
AV537968 Arabidopsis thaliana roots
AV561683 Arabidopsis thaliana green siliques
AV539188 Arabidopsis thaliana roots
F20141ATTS6180Ors-B Arabidopsis thaliana
AV551590 Arabidopsis thaliana roots
AV541274 Arabidopsis thaliana roots
T222214229 Lambda-PRL2 Arabidopsis thaliana
AV543284 Arabidopsis thaliana roots
H7614117846 Lambda-PRL2 Arabidopsis thaliana
AV550967 Arabidopsis thaliana roots
AV537994 Arabidopsis thaliana roots
BE525087M58D1STM Arabidopsis developing seed
AV525070 Arabidopsis thaliana aboveground organs two to six-week old
H3648714155 Lambda-PRL2 Arabidopsis thaliana
BE527669M69E02STM Arabidopsis developing seed
Z37679ATTS4144Versailles-VB Arabidopsis thaliana
BE526378M65O06STM Arabidopsis developing seed
H3646314131 Lambda-PRL2 Arabidopsis thaliana
AV439599 Arabidopsis thaliana above-ground organ two to six-week old
AV563621 Arabidopsis thaliana green siliques
Z27062ATTS1934Gif-SeedA+B Arabidopsis thaliana
AV441684 Arabidopsis thaliana above-ground organ two to six-week old
T8835612052 Lambda-PRL2 Arabidopsis thaliana
T213553363 Lambda-PRL2 Arabidopsis thaliana
T431526415 Lambda-PRL2 Arabidopsis thaliana
AV542112 Arabidopsis thaliana roots
AV560246 Arabidopsis thaliana green siliques
AV560234 Arabidopsis thaliana green siliques
AV546286 Arabidopsis thaliana roots
AV560239 Arabidopsis thaliana green siliques
AV543136 Arabidopsis thaliana roots
AV522259 Arabidopsis thaliana aboveground organs two to six-week old
AV532245 Arabidopsis thaliana flower buds
AI998317701545224A.thaliana, Columbia Col-0, rosette-2
AV563766 Arabidopsis thaliana green siliques
AV563730 Arabidopsis thaliana green siliques
AI09954234268 Lambda-PRL2 Arabidopsis thaliana
AV562052 Arabidopsis thaliana green siliques
AV441878 Arabidopsis thaliana above-ground organ two to six-week old
BU636183047A04 Infected Arabidopsis Leaf
BU636021044E01 Infected Arabidopsis Leaf
AV797072RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-10-K 14
AV787980RAFL6 Arabidopsis thaliana cDNA clone RAFL 06-77-L 11
AV797380RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-11-O 04
AV813058RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-76-N 09
AV795900RAFL8 Arabidopsis thaliana cDNA clone RAFL 08-19-K 23
AV794575RAFL8 Arabidopsis thaliana cDNA clone RAFL 08-13-H 10
AV787024RAFL6 Arabidopsis thaliana cDNA clone RAFL 06-73-M 12
AV810174RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-62-O 03
AV788330RAFL6 Arabidopsis thaliana cDNA clone RAFL 06-79-A 17
AV816931RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-92-I 16
AV804193RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-38-D 20
AV803468RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-35-K 08
AV785787RAFL6 Arabidopsis thaliana cDNA clone RAFL 06-68-G 18
AV811226RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-67-E 24
AV813113RAFL9 Arabidopsis thaliana cDNA clone RAFL 09-77-B 05

Arath;F5H1
AW004486701931810A.thaliana, mixed source
AA59801529264 Lambda-PRL2 Arabidopsis thaliana

Arath;F5H2
(none)

Arath;CAD1
BE038841AB04H07AB Arabidopsis thaliana cDNA similar to cinnamyl-alcohol dehydrogenase cad1
AV566451 Arabidopsis thaliana green siliques
AV527359 Arabidopsis thaliana aboveground organs two to six-week old
T428736136 Lambda-PRL2 Arabidopsis thaliana
AI998843701547086A.thaliana, Columbia Col-0, rosette-2
N9648121044CD4-14 Arabidopsis thaliana
AV534611 Arabidopsis thaliana flower buds
AV557733 Arabidopsis thaliana green siliques
AV557103 Arabidopsis thaliana green siliques
AV518951 Arabidopsis thaliana aboveground organs two to six-week old
T438387101 Lambda-PRL2 Arabidopsis thaliana
T7582210600 Lambda-PRL2 Arabidopsis thaliana
AV564469 Arabidopsis thaliana green siliques
BE527803M69D18STM Arabidopsis developing seed
R6499513499 Lambda-PRL2 Arabidopsis thaliana
AV560074 Arabidopsis thaliana green siliques
T7607410852 Lambda-PRL2 Arabidopsis thaliana
AV565045 Arabidopsis thaliana green siliques
AV441392 Arabidopsis thaliana above-ground organ two to six-week old

AV558883 Arabidopsis thaliana green siliques
AV566244 Arabidopsis thaliana green siliques
AV558629 Arabidopsis thaliana green siliques
AV519606 Arabidopsis thaliana aboveground organs two to six-week old
AV832095 RAFL 11 Arabidopsis thaliana cDNA clone RAFL 11-02-K 13
AV819160 RAFL 11 Arabidopsis thaliana cDNA clone RAFL 11-02-K 13
AV810133 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-62-L 18
AV787406 RAFL 6 Arabidopsis thaliana cDNA clone RAFL 06-75-H 10
AV813196 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-77-H 12
AV819600 RAFL 11 Arabidopsis thaliana cDNA clone RAFL 11-05-N 20
AV805374 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-42-L 03
AV806304 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-46-A 20
AV803186 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-34-J 12

Arath; CAD 2
***
AV538058 Arabidopsis thaliana roots
AV565838 Arabidopsis thaliana green siliques
AV544201 Arabidopsis thaliana roots
AV782249 RAFL 4 Arabidopsis thaliana cDNA clone RAFL 04-15-K 01
AV442077 Arabidopsis thaliana above-ground organ two to six-week old
AV552684 Arabidopsis thaliana roots
AV541573 Arabidopsis thaliana roots
AV540346 Arabidopsis thaliana roots
AV539027 Arabidopsis thaliana roots
AV525754 Arabidopsis thaliana aboveground organs two to six-week old
AV439855 Arabidopsis thaliana aboveground organs two to six-week old
AV782420 RAFL 4 Arabidopsis thaliana cDNA clone RAFL 04-17-E 23
AV540973 Arabidopsis thaliana roots
AV550203 Arabidopsis thaliana roots
AV551366 Arabidopsis thaliana roots
AA712722 32281 Lambda-PRL 2 Arabidopsis thaliana
AV556575 Arabidopsis thaliana green siliques
AV538928 Arabidopsis thaliana roots
AA598138 29798 Lambda-PRL 2 Arabidopsis thaliana
T206652673 Lambda-PRL 2 Arabidopsis thaliana
AV541771 Arabidopsis thaliana roots
AA651503 31617 Lambda-PRL 2 Arabidopsis thaliana
Z34082 ATTS 3130 Grenoble-B Arabidopsis thaliana
T457469009 Lambda-PRL 2 Arabidopsis thaliana
AV519340 Arabidopsis thaliana aboveground organs two to six-week old
AI100468 34843 Lambda-PRL 2 Arabidopsis thaliana
Z34840 ATTS 3574 Grenoble-B Arabidopsis thaliana
AV827937 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-22-D 09
AV798520 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-16-I 21
AV798996 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-18-E 21
AV817760 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-95-J 04
AV796784 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-09-J 11
AV795636 RAFL 8 Arabidopsis thaliana cDNA clone RAFL 08-18-H 18

Arath; CAD 3
***
AV813799 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-80-C 01

Arath; CAD 4
***
AV524876 Arabidopsis thaliana aboveground organs two to six-week old
AV440111 Arabidopsis thaliana above-ground organ two to six-week old
AV518197 Arabidopsis thaliana aboveground organs two to six-week old
AV525644 Arabidopsis thaliana aboveground organs two to six-week old
AI993129 701495390 A. thaliana, Ohio State clone set
R65519 14023 Lambda-PRL 2 Arabidopsis thaliana
AV439722 Arabidopsis thaliana above-ground organ two to six-week old
T21745 3753 Lambda-PRL 2 Arabidopsis thaliana
N65902 20942 Lambda-PRL 2 Arabidopsis thaliana
T22463 4471 Lambda-PRL 2 Arabidopsis thaliana
AI099980 34133 Lambda-PRL 2 Arabidopsis thaliana
AA394914 26711 Lambda-PRL 2 Arabidopsis thaliana
T76625 11403 Lambda-PRL 2 Arabidopsis thaliana
T44824 8087 Lambda-PRL 2 Arabidopsis thaliana
T45613 8876 Lambda-PRL 2 Arabidopsis thaliana
T44495 7758 Lambda-PRL 2 Arabidopsis thaliana
AV530508 Arabidopsis thaliana flower buds
AV536737 Arabidopsis thaliana liquid-cultured seedlings
AV826073 RAFL 8 Arabidopsis thaliana cDNA clone RAFL 08-08-M 18
AV794209 RAFL 8 Arabidopsis thaliana cDNA clone RAFL 08-11-O 11
AV793475 RAFL 8 Arabidopsis thaliana cDNA clone RAFL 08-08-M 18
AV819767 RAFL 11 Arabidopsis thaliana cDNA clone RAFL 11-07-A 14
AV803099 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-34-F 03
AV816875 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-92-F 15
AV798580 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-16-M 04
AV802560 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-32-F 24

Arath; CAD 5
***
AV781403 RAFL 2 Arabidopsis thaliana cDNA clone RAFL 02-05-I 05
AV806808 RAFL 9 Arabidopsis thaliana cDNA clone RAFL 09-48-A 18

Arath; CAD 6


AI996376 701665157 A. thaliana, Columbia Col-0, root-1
AV782177 RAFL4 Arabidopsis thaliana cDNA clone RAFL04-14-P24
AV545306 Arabidopsis thaliana roots
BE845105 AD06F10T7ADA. thaliana (Col-0gl1) subtracted library
AV555804 Arabidopsis thaliana green siliques
AV554034 Arabidopsis thaliana roots
AV562714 Arabidopsis thaliana green siliques
AV564197 Arabidopsis thaliana green siliques
AV565693 Arabidopsis thaliana green siliques
AV538294 Arabidopsis thaliana roots
T46791 10054 Lambda-PRL2 Arabidopsis thaliana
H37740 15869 Lambda-PRL2 Arabidopsis thaliana
T04220 267 Lambda-PRL2 Arabidopsis thaliana
Z34557 ATTS3317 Versailles-VB Arabidopsis thaliana
AA605470 30506 Lambda-PRL2 Arabidopsis thaliana
AA067591 26222 Lambda-PRL2 Arabidopsis thaliana
AI997099 701552047 A. thaliana, Columbia Col-0, root-2
AV556125 Arabidopsis thaliana green siliques
Z34154 ATTS3188 Versailles-VB Arabidopsis thaliana
AV536791 AV536791 Arabidopsis thaliana liquid-cultured seedlings
R90152 16507 Lambda-PRL2 Arabidopsis thaliana
BU636675 010A03 Infected Arabidopsis Leaf
AV813408 RAFL9 Arabidopsis thaliana cDNA clone RAFL09-78-G15

Arath; CAD7
(none)

Arath; CAD8
(none)

Arath; CAD9
BE038380 AA12G02AA Arabidopsis thaliana cDNA similar to cinnamyl-alcohol dehydrogenase
AI997900 701671284 A. thaliana, Columbia Col-0, rosette-1
Z46703 ATTS4365 Perp-dry-seed A Arabidopsis thaliana
AV556031 Arabidopsis thaliana green siliques
BE530209 M76J15STM Arabidopsis developing seed
BE527735 M69K20STM Arabidopsis developing seed
Z46704 ATTS4366 Perp-dry-seed A Arabidopsis thaliana
AV812119 RAFL9 Arabidopsis thaliana cDNA clone RAFL09-71-F14
AV819800 RAFL11 Arabidopsis thaliana cDNA clone RAFL11-07-D13

# Supplementary data: EST classification details

Data from http://www.ncbi.nlm.nih.gov/UniLib/lb.cgi and NCBI Entrez (for RAFL)

| Library name                                                                 | Library size | Organ/stress         |
|------------------------------------------------------------------------------|--------------|----------------------|
| Lambda-PRL2                                                                  | 28014        | whole plant          |
| Arabidopsis thaliana roots Columbia                                          | 17709        | root                 |
| Arabidopsis thaliana green siliques Columbia                                 | 12752        | green siliques       |
| Arabidopsis thaliana aboveground organs two to six-week old                   | 12403        | aboveground          |
| Arabidopsis developing seed                                                  | 11238        | green siliques       |
| Arabidopsis thaliana flower buds Columbia                                    | 5827         | flower               |
| Arabidopsis thaliana above-ground organ two to six-week old                  | 3366         | aboveground          |
| A. thaliana, Ohio State clone set                                             | 3022         | whole plant          |
| Versailles-VB                                                                | 1515         | etio seedling        |
| Arabidopsis thaliana liquid-cultured seedlings Columbia                      | 1161         | seedling             |
| AA                                                                           | 1007         | salt                 |
| Grenoble-B                                                                   | 985          | flower               |
| AC16H                                                                        | 898          | cell suspension      |
| A. thaliana, Columbia Col-0, rosette-2                                        | 897          | rosette              |
| A. thaliana, Columbia Col-0, rosette-3                                       | 814          | rosette              |
| CD4-13                                                                       | 726          | seedling hypocotyl   |
| A. thaliana, Columbia Col-0, inflorescence-2                                  | 692          | infl                 |
| Lambda-PRL1                                                                  | 677          | whole plant          |
| A. thaliana, Columbia Col-0, root-2                                           | 676          | root                 |
| Strasbourg-A                                                                 | 670          | wounded leaves       |
| AB                                                                           | 642          | salt                 |
| A. thaliana, Columbia Col-0, rosette-1                                        | 640          | rosette              |
| AD A. thaliana (Col-0 gl1) subtracted library enriched for salt-induced transcripts; 10-14 day seedlings 4h160mM NaCl stress | 571          | salt                 |
| CD4-16                                                                       | 567          | seedling hypocotyl   |
| A. thaliana, Columbia Col-0, root-1                                           | 562          | root                 |
| CD4-15                                                                       | 419          | seedling hypocotyl   |
| Gif-SeedA                                                                    | 392          | green siliques       |
| A. thaliana, Columbia Col-0, inflorescence-1                                  | 367          | infl                 |
| Perp-dry-seedA                                                               | 346          | dry seed             |
| AC                                                                           | 333          | salt                 |
| CD4-14                                                                       | 333          | seedling hypocotyl   |
| Ors-A                                                                        | 325          | infl                 |
| Gif-SeedA+B                                                                  | 286          | green siliques       |
| A. thaliana, mixed source                                                     | 272          | other                |
| Ors-B                                                                        | 241          | stress               |
| CD4-6                                                                        | 201          | infl, flower         |
| NaCl-treated Arabidopsis subtraction library                                 | 184          | salt                 |
| Arabidopsis Acute Ozone Forward-Subtracted Library                           | 171          | stress               |
| AD A. thaliana (Col-0 gl1) library enriched for salt-induced transcripts; 10-14 day seedlings; 4h160mM NaCl stress | 151          | salt                 |
| Arabidopsis Chronic Ozone Forward-Subtracted Library                         | 129          | stress               |
| Versailles-VC                                                                | 124          | etio seedling        |
| AC13D                                                                        | 123          | cell suspension      |
| Ra147.1                                                                      | 121          | pathogen             |
| Gif-SiliqueB                                                                 | 116          | infl, flower         |
| AT-NHC                                                                       | 99           | whole plant          |
| YAC clone CIC8B11 region-specific cDNA                                       | 75           | other                |
| Arabidopsis thaliana library (Motohashi R)                                   | 73           | other                |
| Arabidopsis pSMASH Library                                                   | 49           | whole plant          |
| Lambda ZipLox designated PRL2                                                 | 44           | whole plant          |
| YAC clone CIC7E11 region-specific cDNA                                       | 38           | other                |
| Strasbourg-FA                                                                | 37           | wounded leaves       |
| RIKEN-PMB-FL1                                                                | 35           | other                |
| Arabidopsis thaliana cell suspension                                         | 28           | cell suspension      |
| Cloned cDNAs of Arabidopsis thaliana generated by mRNA differential-display RT-PCR | 27           | pathogen             |
| Arabidopsis Acute Ozone Reverse-Subtracted Library                           | 27           | stress               |
| FL5                                                                          | 21           | other                |
| pi mRNA differential display clones                                          | 20           | infl, flower         |
| Arabidopsis thaliana Above-ground organ from two to six-week old plants Columbia | 17           | aboveground          |
| Arabidopsis thaliana germinating seeds                                       | 16           | germ seed            |
| Arabidopsis thaliana early developing seeds Library                          | 9            | dev seed             |
| Versailles-VD                                                                | 7            | etio seedling        |
| Arabidopsis thaliana library (Josefsson LG)                                  | 6            | other                |
| FL2                                                                          | 6            | whole plant          |
| Arabidopsis thaliana mRNA (J.Fleck)                                          | 6            | other                |
| FL3                                                                          | 5            | other                |
| Arabidopsis thaliana Columbia                                                | 5            | other                |
| Arabidopsis thaliana immature siliques                                       | 4            | green siliques       |
| Arabidopsis thaliana M.W.Schena                                              | 3            | other                |
| Lambda-PRL2 (P.Kapranov)                                                     | 2            | whole plant          |
| Grenoble-A                                                                   | 2            | flower               |
| Arabidopsis thaliana 2 week old shoots                                       | 1            | seedling             |
| Arabidopsis thaliana library (Shinozaki K)                                   | 1            | other                |
| FL6                                                                          | 1            | other                |
| Arabidopsis thaliana library (Tan K)                                         | 1            | other                |
| Arabidopsis thaliana cDNA library in yeast Saccharomyces cerevisiae, Francois Lacroute | 0            | other                |
| Arabidopsis thaliana library (Nambara E)                                     | 0            | other                |
| Arabidopsis infected leaves                                                  | 2218         | pathogen             |

RAFL1 cold treated leaves and stems
RAFL2 rosette
RAFL3 plants dehydration-treated (2hr) rosette
RAFL4 = subjected to cold-treated (1, 2, 5, 10, 24hr) rosette
RAFL5 = subjected to dehydration-treated (1, 2, 5, 10, 24hr) rosette
RAFL6 = subjected to dehydration (1, 2, 5, 10, 24hr) and cold (1, 2, 5, 10, 24hr) treatments, various developmental stages
RAFL7 = subjected to cold-treated (1, 2, 5, 10, 24hr) rosette
RAFL8 = subjected to dehydration-treated (1, 2, 5, 10, 24hr) rosette
RAFL9 = subjected to dehydration (1, 2, 5, 10, 24hr) and cold (1, 2, 5, 10, 24hr) treatments (plants at various developmental stages)
RAFL11 = library used for normalization, thus all kind subjected to various treatments
RAFL12 = Subtraction cold-treated (1, 2, 5, 10, and 24hr) (rosette)
RAFL13 = Subtraction dehydration-treated (1, 2, 5, 10, and 24hr) rosette
RAFL14 = root
RAFL15 = mixture of silique and flower
RAFL16 = dark-grown (tissue not mentioned)
RAFL17 = Subtraction Library. The sequence was obtained from samples subjected to dehydration-treated (1, 2, 5, 10 and 24hr) and rehydration-treated
RAFL18 = Subtraction Library. The sequence was obtained from samples subjected to cold-treated (1, 2, 5, 10, and 24hr) (tissue not mentioned)
RAFL19 = subtraction library, siliques and flowers
RAFL21 = Subtraction Library. The sequence was obtained from samples subjected to various stress and plant hormones-treated (tissue not mentioned)

| organ, stress | total | classified as |
|---------------|-------|---------------|
| whole plant   | 31913 |               |
| aboveground organs | 17597 |             |
| root          | 20332 |               |
| rosette       | 2582  | whole plant   |
| seedling      | 1162  |               |
| seedling hypoc | 2045  | seedling      |
| etio seedling | 2896  | stress        |
| flower        | 6814  |               |
| infl          | 1384  |               |
| infl, flower  | 337   | aboveground   |
| green siliques | 24672 | siliques, seed |
| dev seed      | 9     | siliques, seed |
| dry seed      | 346   | siliques, seed |
| germ seed     | 16    | siliques, seed |
| cell suspension | 1049 | whole plant   |
| stress        | 26207 |               |
| salt          | 2888  | stress        |
| pathogen      | 2366  |               |
| dehydration   | 11305 | stress        |
| wounded leaves | 707  |               |
| cold          | 3607  | stress        |
| other         | 542   |               |
|               | 160776|               |

Final Classification

|              |           |
|--------------|-----------|
| whole plant  | 35544     |
| aboveground organs | 17934     |
| root         | 20332     |
| seedling     | 3207      |
| flower       | 6814      |
| inflorescence stem | 1384      |
| siliques, seed | 25043     |
| stress       | 46903     |
| pathogen     | 2366      |
| wounded leaves | 707       |
| other        | 542       |
|              | 160776    |
