
The Plant Journal (2003) 36, 471–484

doi: 10.1046/j.1365-313X.2003.01893.x

# Regiospecific hydroxylation of isoflavones by cytochrome P450 81E enzymes from *Medicago truncatula*

Chang-Jun Liu, David Huhman, Lloyd W. Sumner and Richard A. Dixon*

Plant Biology Division, Samuel Roberts Noble Foundation, 2510 Sam Noble Parkway, Ardmore, OK 73401, USA

Received 15 May 2003; revised 7 August 2003; accepted 12 August 2003.

*For correspondence (fax +1 580 224 6692; e-mail radixon@noble.org).

---

## Summary

Mining of *Medicago truncatula* EST databases and screening of a root cDNA library led to the identification of three cytochrome P450 81E subfamily members. Two were functionally characterized by expression in yeast. The recombinant enzymes in yeast microsomes utilized the same isoflavone substrates, but produced different products hydroxylated at the 2′ and/or 3′ positions of the B-ring. When transiently expressed in alfalfa leaves, green fluorescent protein (GFP) fusions of the isoflavone 2′- and 3′-hydroxylases localized to the endoplasmic reticulum. The isoflavone 2′-hydroxylase was functional when expressed in *Arabidopsis*. Differential tissue-specific and biotic/abiotic stress-dependent expression patterns were observed for the isoflavone 2′-hydroxylase and 3′-hydroxylase genes, suggesting differential involvement of 2′- and 3′-hydroxylated isoflavonoids in pathogen defense and insect-induced responses, respectively, in *Medicago*.

Keywords: cytochrome P450, functional expression, isoflavonoid phytoalexin, elicitation, insect herbivory.

---

## Introduction

Isoflavonoids are a subclass of phenylpropanoid metabolites distributed primarily in legumes (Dixon and Sumner, 2003). They possess a wide range of biological activities (Dixon, 1999), but most research has focused on their functions as pathogen-inducible antimicrobial compounds (phytoalexins; Dewick, 1993; Dixon, 1999; Ingham, 1982) or as dietary phytoestrogens implicated in human disease prevention (Adlercreutz and Mazur, 1997; Dixon and Ferreira, 2002). Different legume species produce different classes of isoflavonoid phytoalexins, of which, substituted pterocarpans, such as medicarpin from alfalfa and pisatin from pea, are the best known (Figure 1).

Complex isoflavonoid derivatives such as the rotenoids rotenone, deguelin, and amorphigenin from *Amorpha*, *Lonchocarpus*, *Derris*, and *Tephrosia* species possess insecticidal and parasiticidal properties (Lambert *et al.*, 1993; Nicholas *et al.*, 1985). Maackiain (Figure 1), which accumulates along with medicarpin (the major phytoalexin in *Medicago* species) in red clover (*Trifolium pratense*), subterranean clover (*T. subterraneum*), and chickpea (*Cicer arietinum*; Dewick and Ward, 1978; Higgins, 1972; Ingham, 1982), has recently been shown to have larvicidal activity against caterpillars of *Heliothis armigera* that attack chickpea (Simmonds and Stevenson, 2001).

The biosynthesis of isoflavonoids diverges from the ubiquitous flavonoid pathway and is shown in Figure 1, which also provides details of the A- and B-ring and position designations of isoflavonoid compounds. The 5-deoxyflavanone liquiritigenin is converted to an isoflavone, and then undergoes several steps of hydroxylation, methylation, reduction, and ring closure to form pterocarpans such as medicarpin and maackiain (Dewick and Martin, 1979; Dixon, 1999). The 5-hydroxyflavanone naringenin is also a starting point for synthesis of isoflavonoids such as biochanin A and praten-sin in chickpea and red clover (Clemens *et al.*, 1993; Dewick and Ward, 1978; Figure 1). Hydroxylations catalyzed by membrane-bound, NADPH-dependent cytochrome P450 monooxygenases are critical steps in the biosynthesis of complex isoflavonoids. For example, 2-position hydroxylation of liquiritigenin and naringenin accompanied by B-ring migration from the 2- to the 3-position occurs at the entry point into the isoflavonoid pathway (Kochs and Grisebach, 1986), whereas 2′- or 3′-position hydroxylation of the B-ring of isoflavones is essential for formation of pterocarpans and/or methylenedioxy-substituted compounds such as maackiain and pisatin (Clemens *et al.*, 1993; Dewick and Ward, 1978; Gunia *et al.*, 1991; Hinderer *et al.*, 1987; Figure 1). 6a-Hydroxylation of pterocarpans occurs in the

L-phenylalanine

$\downarrow$

C4H(CYP73A)

$\downarrow$

4-coumaroyl CoA $\quad$ 3 x malonyl CoA

$\downarrow$

CHS $\quad$ [CHR]

$\downarrow$

chalcone

$\downarrow$

CHI


HO


|   |


R


|   |


OH


$\rightarrow$ flavonoids, anthocyanins

R=H, liquiritigenin  
R=OH, naringenin

$\downarrow$

IFS(CYP93C2)


HO


|   |


R


|   |


OH


$\downarrow$

IOMT(DH)


HO


|   |


R


|   |


OH


$\downarrow$

IOMT (DH)


HO


|   |


R


|   |


OH


$\downarrow$

I2'H(CYP81E1/E7)


HO


|   |


R


|   |


OH


$\downarrow$

IFR

$\downarrow$

VR


HO


|   |


R


|   |


OH


formononetin

$\downarrow$

I3'H(CYP81E9)


HO


|   |


R


|   |


OH


calycosin

$\downarrow$

CYP(X)

pseudobaptigenin

$\downarrow$

CYP81E(X)

maackiain

(pterocarpan)  
(in chickpea)

$\downarrow$

pisatin

(pterocarpan)  
(in pea)

biosynthesis of the glyceollins in soybean (Kochs and Grisebach, 1989) and of pisatin in pea (Figure 1).

Isoflavone 2'-hydroxylase (I2'H) activity has been identified in microsomal fractions of elicited cells of soybean (Kochs and Grisebach, 1986), chickpea (Clemens *et al.*, 1993; Gunia *et al.*, 1991; Hinderer *et al.*, 1987) and alfalfa (*Medicago sativa*; Choudhary *et al.*, 1990), and an I2'H (CYP81E1) gene characterized from licorice (*Glycyrrhiza echinata* L.). Recombinant CYP81E1 catalyzed the 2'-hydroxylation of formononetin (7-hydroxy, 4'-methoxyisoflavone) and the 2'- and 3'-hydroxylation of daidzein (7,4'-dihydroxyisoflavone) *in vitro* in yeast microsomes (Akashi *et al.*, 1998). Several cDNA clones with high sequence identity to I2'H have been isolated from elicited *Lotus*

Figure 1. Biosynthetic pathways leading to complex isoflavonoids in legumes.

The compounds shown are found in different species, as indicated. The enzymes are: CYP73A, cinnamate 4-hydroxylase; CHS, chalcone synthase; CHR, chalcone reductase; CHI, chalcone isomerase; CYP93C2, 2-hydroxyisoflavanone synthase, also known as isoflavone synthase (IFS); IOMT, 2-hydroxyisoflavanone 4'-O-methyltransferase; DH, 2-hydroxyisoflavanone dehydratase; CYP81E1/7, isoflavone 2'-hydroxylase (I2'H); CYP81E9, isoflavone 3'-hydroxylase (I3'H); IFR, isoflavone reductase; VR, vestitone reductase; CYPX, P450 catalyzing methylenedioxy ring closure; CYP81E(X), P450 catalyzing 2'-hydroxylation of pseudobaptigenin. Dotted arrows indicate pathways that are yet to be fully characterized. Double arrows indicate two or more reactions. The numbering system for isoflavones is shown for formononetin and biochanin A. Note that the 2'-and 5'-, and 3'-and 6'-positions are synonymous because of rotation about the bond linking the aryl group to the 3'-position of the heterocyclic ring.

biochanin A

pratensin

rotenone  
(rotenoid)

medicagol  
(coumestan)  
(in alfalfa)

© Blackwell Publishing Ltd, *The Plant Journal*, (2003), **36**, 471–484

presumably involved in isoflavone catabolism. However, genes encoding I3′H have not yet been identified.

In this report, we have utilized genomics resources available for the model legume *Medicago truncatula* (Bell et al., 2001; Cook, 1999; Oldroyd and Geurts, 2001) to identify three CYP81E subfamily members (CYP81E7, 8, and 9). Two of the three were functionally characterized by expression in yeast. They share high similarity at the amino acid level and utilize the same methylated isoflavone substrates, but encode distinct isoflavone 2′- (CYP81E7) and 3′- (CYP81E9) hydroxylases. The I2′H (CYP81E7) was transferred into *Arabidopsis* and was functional. We describe differential expression patterns of the I2′H and I3′H genes in response to a variety of biotic and abiotic stimuli, and discuss the results in terms of the known phytochemistry of *Medicago* species.

### Results

#### Isolation and sequence analysis of M. truncatula CYP81E family members

A putative CYP81E cDNA was obtained by screening an *M. truncatula* root cDNA library with a licorice I2′H (CYP81E1) probe (Akashi et al., 1998). After two rounds of screening, a full-length cDNA clone was isolated that contained 1732 nucleotides and encoded a 498 amino acid polypeptide with a deduced molecular mass of 57 640 Da. It shares about 84% DNA sequence identity with licorice CYP81E1, and encodes a protein classified as MtCYP81E7.

Using CYP81E1 and MtCYP81E7 cDNA sequences for BLAST analysis of the Institute for Genomics Research (TIGR) (Rockville, MD, USA) *M. truncatula* Gene Index database (http://www.tigr.org/tgi/), two further CYP81E subfamily candidates were revealed. TC69129, which exists as a full-length cDNA in the Noble Foundation’s *M. truncatula* expressed sequence tag (EST) library collection, contained a 22-bp 5′ untranslated leader sequence, followed by an open-reading frame of 1497 bp that encodes a 499 amino acid polypeptide with a deduced molecular mass of 57 301 Da (Figure 2a). It is 58% identical at the amino acid level to MtCYP81E7 and 60% identical to CYP81E1, and is classified as MtCYP81E8. TC70025 was a partial cDNA sequence that lacks the 5′ untranslated region and the translation start codon; one of its three deduced open-reading frames encoded a putative protein of 500 amino acid residues sharing 61% identity with MtCYP81E7 and 65% identity with MtCYP81E8; it is classified as MtCYP81E9 (Figure 2a).

Sequence comparison using the CLUSTALW method in the DNASTAR program showed that MtCYP81E7 clustered with licorice CYP81E1 (Akashi et al., 1998) and homologs from chickpea (Overkamp et al., 2000) and *L. japonicus* (Shimada et al., 2000), suggesting that it might encode an I2′H. MtCYP81E9 and MtCYP81E8 are less closely related

to MtCYP81E7 and other CYP81E1 members, and represent two separate unique subclasses (Figure 2b).

#### Functional characterization of MtCYP81E enzymes in yeast

The coding regions of MtCYP81E7 and MtCYP81E9 were amplified by PCR and introduced into the yeast expression vector YeDP60 under control of a galactose-inducible and glucose-repressible promoter (Pompon et al., 1996). As MtCYP81E9 lacks the two N-terminal amino acid residues in its membrane anchor, N-terminal methionine and three-nine residues (chosen on the basis of alignments of CYP81E family members; Figure 2) were introduced by PCR. The full-length CYP81E8 cDNA was directly excised from its original clone in the Noble Foundation EST library collection and ligated into YeDP60.

Expression constructs were transformed into yeast strain WAT11 that carries an *Arabidopsis thaliana* cytochrome P450 reductase gene integrated in its genome. Carbon monoxide (CO) difference spectra of reduced microsomal preparations from WAT11 strains harboring each of the three different MtCYP81E constructs exhibited maximum absorbance at 450 nm (Figure 3), whereas microsomes from strains harboring empty vector lacked the 450-nm peak (data not shown). The levels of expressed P450 proteins, based on the CO difference spectra were 269 nmol g⁻¹ microsomal protein for MtCYP81E7, 278 nmol g⁻¹ for MtCYP81E8, and 177 nmol g⁻¹ for MtCYP81E9.

Yeast microsomes expressing MtCYP81E7 converted formononetin to 2′-hydroxyformononetin as confirmed by HPLC with UV diode array detection (retention time (RT) 52.1 min; absorption maxima at 203, 249, and 293 nm) and liquid chromatography-mass spectrometry (LC-MS) (molecular ion at m/z 282.8 [M – H]⁻) as compared to an authentic standard (Figure 4a–c). MtCYP81E7 converted biochanin A to a product with molecular ion at m/z 299, and absorption maxima at 203, 259, and 188 nm (shoulder), consistent with formation of 2′-hydroxybiochanin A (Figure 4h). Neither product was observed in reactions without NADPH or with microsomes from yeast cells harboring empty vector (Figure 4a,g). Therefore, MtCYP81E7 is an isoflavone 2′-hydroxylase (Mtl2′H).

When yeast microsomes expressing MtCYP81E9 were incubated with formononetin, the product (RT, 47 min) had a molecular ion at 282.8 m/z, but its UV spectrum showed absorption peaks at 199, 219, 249, and 291 nm, identical to an authentic sample of calycosin (3′-hydroxyformononetin; Figure 4e,f). The product formed from biochanin A by recombinant MtCYP81E9 had a UV spectrum with maximum absorption at 262 nm and two shoulders at 290 and 330 nm, in good agreement with the reported spectrum of pratensin (Mabry et al., 1970; Figure 4i). Therefore, MtCYP81E9 is an isoflavone 3′-hydroxylase (Mtl3′H).

474 Chang-Jun Liu et al.

(a)

| CYP81E8 | -MTTFYLSLIISLFFLIITLKVFFN-TSRKFKNILPPGPQCLP | IIGNLHQLKQPLHHTFHT | 58 |
| --- | --- | --- | --- |
| CYP81E9 | -MTLFYYSLLS--LSFIITIKILLKITSRRILKNILPPGBPPTIP | IIGNLHHLKHPLHRTFTT | 55 |
| CYP81E7 | MGILSYLCYSLFYLSIFFIIRLLFQ--SRKFKNILPPGPTSLPIIGNLHHLKRPLNRTFKA | 58 |


* . : ::: ::::: **:**:*:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**:**
Figure 4. HPLC-UV and HPLC-MS analysis of products formed by the activity of Mtl2′H and Mtl3′H on isoflavone substrates. Insets show mass spectra of the major peak (substrate or product). All incubations contain NADPH unless otherwise stated.

(a) Formononetin incubated with yeast microsomes harboring empty vector.

(b) Products from incubation of I2′H with formononetin.

(c) 2′-Hydroxyformononetin standard.

(d) Formononetin incubated with I3′H in the absence of NADPH.

(e) Products from incubation of I3′H with formononetin.

(f) Calycosin standard.

(g) Biochanin A incubated with microsomes harboring empty vector.

(h) Products from incubation of I2′H with biochanin A.

(i) Products from incubation of I3′H with biochanin A.

Compounds are: F, formononetin; 2′HF, 2′-hydroxyformononetin; C, calycosin; B, biochanin A; 2′HB, 2′-hydroxybiochanin A; P, pratensein.

476 Chang-Jun Liu et al.

Table 1 Substrate specificity of I2′H and I3′H

| Isoflavone (40 μM)                          | I2′H (MtCYP81E7) | I3′H (MtCYP81E9) |
|---------------------------------------------|------------------|------------------|
| Biochanin A (1)                             | 100%             | 100%             |
| Formononetin (2)                            | 37.2%            | 50%              |
| Pseudobaptigenin (3)                        | 35%              | 7.6%<sup>a</sup>  |
| Daidzein (4)                                | 18.6%<sup>a</sup> | 1.73%<sup>a</sup> |
| Genistein (5)                               | 10.2%<sup>a</sup> | 19.13%<sup>a</sup>|
| 2′-Hydroxyformononetin                      | 9.1%<sup>a</sup>  | 39.34%<sup>b</sup>|
| 3′-Hydroxyformononetin                      | 4.7%             | ND               |
| Isoformononetin (6)                         | ND               | ND               |
| Prunetin (7)                                | ND               | ND               |
| 6,7,4′-Trihydroxyisoflavone                 | ND               | ND               |
| 3′,4′,7-Trihydroxyisoflavone                | ND               | ND               |

<sup>a</sup> Sum of putative 2′ and/or 3′-hydroxylated products.  
<sup>b</sup> Sum of all the products (multiple product peaks on HPLC).  
ND, activities not detectable.  
100%, 53.75 pmol pmol<sup>−1</sup> protein h<sup>−1</sup> for 2′-hydroxylation and 98.56 pmol pmol<sup>−1</sup> protein h<sup>−1</sup> for 3′-hydroxylation.

(1) R<sub>1</sub>, R<sub>2</sub> = OH; R<sub>3</sub> = OCH<sub>3</sub>; R<sub>4</sub> = H  
(2) R<sub>1</sub> = OH; R<sub>3</sub> = OCH<sub>3</sub>; R<sub>2</sub>, R<sub>4</sub> = H  
(3) R<sub>2</sub> = OH; R<sub>2</sub> = H; R<sub>3</sub>, R<sub>4</sub> = —O—CH<sub>2</sub>—O—  
(4) R<sub>1</sub>, R<sub>3</sub> = OH; R<sub>2</sub>, R<sub>4</sub> = H  
(5) R<sub>1</sub>, R<sub>2</sub>, R<sub>3</sub> = OH; R<sub>4</sub> = H  
(6) R<sub>1</sub> = OCH<sub>3</sub>; R<sub>3</sub> = OH; R<sub>2</sub>, R<sub>4</sub> = H  
(7) R<sub>1</sub> = OCH<sub>3</sub>; R<sub>2</sub>, R<sub>3</sub> = OH; R<sub>4</sub> = H  

Mtl2′H and Mtl3′H show preference for the 5-hydroxy-substituted biochanin A over its corresponding 5-deoxy derivative formononetin (Table 1). Mtl2′H was also active with pseudobaptigenin (Table 1), converting it into a major product with molecular ion at 290 m/z and UV absorption maxima at 245 and 305 nm, consistent with 2′-hydroxy-pseudobaptigenin. Similar to licorice I2′H, Mtl2′H had weak activity with daidzein and genistein (5,7,4′-trihydroxyisoflavone) and negligible activity with 2′-hydroxyformononetin and calycosin (Table 1). Daidzein and genistein were converted to both 2′- and 3′-hydroxylated products, with preference for 2′-hydroxylation. In contrast, Mtl3′H had low activity with pseudobaptigenin or daidzein, but was more active with 2′-hydroxyformononetin, with the major product tentatively identified as the oxidation product of 2′,3′-dihydroxyformononetin. Neither enzyme hydroxylated the 7-O-methylated isoflavonoids isoformononetin and prunetin.

When assayed under standard assay conditions, or at different pH values and NADPH concentrations, recombinant MtCYP81E8 exhibited no activity with any of the above described isoflavones, or with flavonoids including naringenin, liquiritigenin, apigenin, luteolin, or kaempferol.

### Kinetic properties of Mtl2′H and Mtl3′H

Recombinant Mtl2′H (MtCYP81E7) and Mtl3′H (MtCYP81E9) had similar pH profiles, with optima around pH 8.0 (data not shown). The apparent K<sub>M</sub> value of Mtl2′H for NADPH was about 0.17 μM, consistent with the values observed for many P450 enzymes (Mihaliak *et al.*, 1993). K<sub>M</sub> values of Mtl2′H for formononetin and biochanin A at 2 mM NADPH and pH 8.0 were 67 and 51 μM, respectively, with K<sub>cat</sub> values of 0.015 and 0.033 sec<sup>−1</sup>. Under the same conditions, K<sub>M</sub> values of Mtl3′H were 49.7 μM for formononetin and 112.6 μM for biochanin A, with K<sub>cat</sub> values of 0.028 and 0.1 sec<sup>−1</sup>, respectively.

#### Expression of Mtl2′H in A. thaliana

*Arabidopsis thaliana* was transformed with a construct containing the Mtl2′H (MtCYP81E7) open-reading frame under control of the constitutive cauliflower mosaic virus 35S promoter. After selection for phosphinothricin resistance and RNA gel blot confirmation of transformants (data not shown), the leaves of transgenic and control plants were collected and fed formononetin. Extracts from Mtl2′H transgenic plants, but not from control plants, contained 2′-hydroxyformononetin, as determined by HPLC, with diode array detection and comparison with authentic standards (Figure 5).

#### Subcellular localization of Mtl2′H and Mtl3′H

To determine the subcellular localization of Mtl2′H and Mtl3′H, Mtl2′H::EGFP and Mtl3′H::EGFP fusion proteins were created by in-frame N-terminal ligation to enhanced green fluorescent protein (eGFP). After bombarding constructs into young alfalfa leaves, the distribution of fluorescence was observed by confocal microscopy. Both Mtl2′H::EGFP (Figure 6a,b, views of two different cells) and Mtl3′H::EGFP (Figure 6c,d) fusion proteins had a reticulate distribution pattern with localization around,

Regiospecific hydroxylation of isoflavones 477

Figure 5. *Medicago truncatula* I2′H is functional in *Arabidopsis*. Leaf extracts from control (a) and I2′H transgenic line #33-2-5 (b) were analyzed by HPLC after feeding formononetin (F). 2′-Hydroxyformononetin (2′HF) standard (c). The UV spectrum (not shown) and retention time of the 2′HF peak in (b) exactly matched those of the authentic standard.

but not within, the nucleus. The fluorescent spots clearly visible in Figure 6(c) are aggregates of the GFP fusion protein caused by the high level of expression. They are clearly distinguishable from the larger fluorescent region representing the nucleus when examined in a set of serial sections. A perinuclear and reticulate distribution was also observed for the eGFP fusion protein of isoflavone synthase (2-hydroxyisoflavanone synthase (IFS), CYP93C2; Figure 6e) and for the endoplasmic reticulum (ER) marker protein modified GFP with an N-terminal HDEL retention signal ER (mGFP-HDEL) in alfalfa leaves (Liu and Dixon, 2001), whereas the fluorescence of free GFP appears diffusely in the cytosol and is also localized within the nucleus (Figure 6f). These results confirm a typical ER membrane localization for both isoflavone hydroxylase proteins.

Tissue-specific and stress-inducible expression of Mtl2′H and Mtl3′H

An overview of isoflavone hydroxylase expression patterns was first obtained by *in silico* analysis of EST numbers in various *M. truncatula* EST libraries available through the TIGR *M. truncatula* gene index (Dixon et al., 2002). The distribution of Mtl2′H and Mtl3′H transcripts was quite distinct. Mtl2′H transcripts were strongly expressed in root tissues and in leaves infected with the fungus *Phoma medicaginis*, whereas Mtl3′H transcripts were present at low levels in leaves (control and subjected to insect herbivory), but found in root tissues only after arbuscular mycorrhizal colonization (Figure 7a). The highest levels of transcripts corresponding to the unidentified MtCYP81E8 appeared to be present in fungally infected leaves.

High stringency RNA gel blot analyses (Figure 7b–f) confirmed that Mtl2′H (MtCYP81E7) transcripts accumulated constitutively in roots, but were present at very low levels in uninfected stems and leaves, whereas Mtl3′H (MtCYP81E9) was expressed constitutively in stems and leaves but not in roots (Figure 7b). Exposure of seedlings to drought strongly reduced Mtl2′H expression in roots, and slightly increased Mtl3′H expression in leaves (Figure 7c). Exposure of leaves to copper chloride (an elicitor of medicarpin production in alfalfa) had no effect on Mtl2′H, but weakly induced Mtl3′H transcript levels, whereas exposure

© Blackwell Publishing Ltd, *The Plant Journal*, (2003), 36, 471–484

Figure 7. Analysis of tissue- and biotic/abiotic stimulus-specific expression of *Mtl2′H* (CYP81E7), CYP81E8, and *Mtl3′H* (CYP81E9) transcripts in *M. truncatula*.

(a) *In silico* analysis of EST abundance in a number of different cDNA libraries, analyzed from data in the TIGR *M. truncatula* gene index. Libraries (with total EST counts in parentheses) are: yeast elicitor-induced cell suspension cultures (Culture, 8526); non-nodulated roots (Root, 5519); nodulated roots (Nod-root, 8697); *Glomus versiforme* and *Glomus intraradices* (arbuscular mycorrhizal fungus) infected roots (AM-root, 15916); fungal elicitor-treated root (E-root, 4967); phosphate-starved root (Pi-root, 5532); root nodule (Nodule, 14231); leaf (7425); *P. medicaginis* infected leaf (Inf-leaf, 3116); beet armyworm caterpillar damaged leaves (H-leaf, 9662); phosphate-starved leaf (Pi-leaf, 8301); drought-stressed seedlings (Drought, 7505); stem (9943); nematode-infected root (Ne-root, 3154).

(b–f) RNA gel blot analysis of *Mtl2′H* (CYP81E7) and *Mtl3′H* (CYP81E9) transcripts. (b) Tissue-specific expression in roots (R), stems (S), and leaves (L) of mature plants at the same developmental stage. (c) Effects of drought on root (DR) and leaf (DL) tissue, and of elicitation with copper chloride (*Cu*²⁺) and MeJA on leaf tissue. Ct, parallel control treatment; I, induced treatment. (d) Time course for effects of inoculation of leaf material with *P. medicaginis* (Induced) and parallel control treatments (Ctrl). (e) Effects of insect herbivory (H; overnight feeding by beet armyworm caterpillars) compared to a parallel control (Ct). (f) Time course for exposure of cell suspension cultures to yeast elicitor (Elicited) with parallel water-treated controls (Ctrl.). See Experimental procedures for further details of treatments.

to methyl jasmonate induced *Mtl3′H* but not *Mtl2′H* (Figure 7c). A similar induction of *Mtl3′H* was observed in leaves exposed to herbivory from beet armyworm caterpillars (Figure 7e). Infection of leaves with the leaf spot fungus *P. medicaginis* resulted in a strong induction of *Mtl2′H* (Figure 7d). Although *I3′H* transcript levels also increased following infection with *Phoma*, a similar increase was observed in the control plants; thus, the procedures used for infection (wounding with a pin wheel followed by placing plants in a polythene bag under high humidity) appear to cause stress that induced *I3′H*, but not *I2′H*. Finally, *Mtl2′H* was induced by exposure of cell cultures to yeast elicitor, a treatment that did not induce *Mtl3′H* (Figure 7f).

*Medicago truncatula* contains at least three *IFS* genes (Dixon et al., 2002). RNA gel blot analysis confirmed expression of one or more *IFS* genes in all tissues in which either *I2′H* or *I3′H* were expressed (data not shown).

Accumulation of isoflavonoids in M. truncatula

The constitutive expression of I2′H in roots, and elicitor/infection-inducible expression in leaves or cell cultures, is consistent with the known accumulation patterns of the 2′-hydroxyisoflavone-derived medicarpin malonyl glucoside (constitutive) and medicarpin aglycone (inducible phytoalexin) in alfalfa (Dixon, 1999). HPLC analysis confirmed the accumulation of formononetin and medicarpin, as well as glycosides of these compounds, in fungal infected leaves of *M. truncatula* (data not shown). Neither compound was detected in leaves exposed to methyl jasmonate (MeJA) or insect herbivory. We could not detect 3′-hydroxylated isoflavonoids such as pseudobaptigenin or maackiain (detection limit 3.5 nmol g⁻¹ FW by our HPLC procedure) in *M. truncatula* leaves exposed to insect herbivory or MeJA treatment, although new compounds eluting at 61.5, 62, and 63.7 min were observed. The nature of these compounds is currently under investigation.

Discussion

Medicago truncatula as a model system for understanding isoflavonoid biosynthesis

Much of our early understanding of the isoflavonoid pathway came from radiotracer experiment studies with alfalfa (*M. sativa*; Dewick and Martin, 1979), and later studies with alfalfa, chickpea, soybean, and licorice led to molecular characterization of the various biosynthetic enzymes (reviewed in Dixon, 1999). These studies support the pathway shown in Figure 1, although the branch pathways to coumestans and rotenoids have yet to be characterized at the molecular level. The genomics tools now available for *M. truncatula* provide an opportunity to understand isoflavonoid biosynthesis in far greater detail than previously possible. For example, it is possible to visualize the tissue and stress-specific expression patterns of multiple gene family members for all the enzymes in isoflavonoid biosynthesis *in silico* by simple queries of publicly available databases (Dixon *et al.*, 2002). This information can direct phytochemical analysis to particular tissues or tissue:inducer combinations that might otherwise not be studied.

Most of the tissue-specific libraries from *M. truncatula* used for EST counting in Figure 7(a) consist of pooled RNA from different time points of development, whereas the RNA gel blot analysis (Figure 7b–f) was performed on RNA from mature plants harvested over a 3-day time period at most. This, along with statistical problems associated with low transcript abundance, may explain some of the discrepancies observed between the results of the two methods of transcript analysis.

In contrast to the wealth of genomic tools, there are few reports on the phytochemistry of *M. truncatula* (ILDIS, 1994), although we have confirmed the presence of formononetin and medicarpin aglycone and glycoconjugates in fungally infected *M. truncatula* leaves (this work) and cell suspension cultures (unpublished results). Although it is likely that the isoflavonoids of *M. truncatula* will closely parallel those of *M. sativa* (in which formononetin, medicarpin, sativan, and coumestrol are among the major compounds), detailed phytochemical profiling of *M. truncatula* is necessary to provide the corresponding metabolite data to match the transcript data already available for over 35 different tissue/treatment sources.

Medicago truncatula contains multiple CYP81E genes encoding both 2′- and 3′-specific isoflavonoid hydroxylases

I2′H occupies a critical position in isoflavonoid biosynthesis in that the 2′-hydroxylation of the B-ring is essential for subsequent reduction by isoflavone reductase to yield an isoflavanone, an obligatory intermediate in the formation of pterocarpan phytoalexins such as medicarpin (Dixon, 1999). It has been suggested that I2′H is the rate-determining step in elicitor-induced pterocarpan phytoalexin biosynthesis in chickpea (Hinderer *et al.*, 1987).

*M. truncatula* contains at least three CYP81E family members; one (MtCYP81E7) has similar properties to the I2′H from licorice (Akashi *et al.*, 1998) and one (MtCYP81E9) is an isoflavone 3′-hydroxylase, an enzyme defined biochemically in crude plant extracts (Clemens *et al.*, 1993), but, until now, not described at the molecular level. We were unable to functionally characterize the third CYP81E enzyme (MtCYP81E8). Its strong CO difference spectrum in yeast microsomes suggests that the enzyme was correctly folded and should therefore have been functional. It is possible that this enzyme has a substrate specificity beyond the range of the flavonoids and isoflavonoids that we tested.

Both MtI2′H and MtI3′H exhibit strong preference for substrates with a 4′-methoxy substitution on the B-ring, consistent with the placing of isoflavone hydroxylation after 4′-O-methylation, a reaction that is tightly linked to the production of isoflavone from flavanone (naringenin or liquiritigenin) catalyzed by IFS (Akashi *et al.*, 2003; Liu and Dixon, 2001). Indeed, most isoflavonoids in *Medicago* species are methoxy-substituted on the 4′-position. However, these compounds generally lack a hydroxyl group on the 5′-position of the A-ring, resulting from the action of a specific chalcone reductase earlier in the pathway (Dixon, 1999), and it is therefore interesting that both I2′H and I3′H preferred biochanin A (5-hydroxyformononetin) to formononetin. This preference, which is consistent with the observed specificity of hydroxylases in crude extracts

© Blackwell Publishing Ltd, *The Plant Journal*, (2003), **36**, 471–484

from *C. arietinum* cell cultures (Hinderer *et al.*, 1987), may simply reflect active site chemistry rather than indicate the operation of a major pathway leading to 5-hydroxy-substituted isoflavonoids in *M. truncatula*.

Both CYP81E hydroxylases from *M. truncatula* were unable to hydroxylate the 7-methoxyisoflavones isoformononetin and prunetin. A similar situation occurs with flavonoid 6-hydroxylase (CYP71D) from soybean, for which a free 7-hydroxyl group is essential both for anchoring the substrate and maintaining the 6-carbon of the A-ring in the spatially correct position for catalysis (LatundeDada *et al.*, 2001). The lack of activity of I2′H and I3′H with isoformononetin is important because the isoflavone O-methyltransferase (IOMT) from *Medicago* paradoxically converts daidzein to isoformononetin *in vitro*. The lack of detection of isoformononetin in *Medicago*, and its inability to be subsequently hydroxylated, confirms previous conclusions that isoformononetin is not an *in vivo* product of IOMT (Liu and Dixon, 2001).

Both I2′H and I3′H localize to the membranes of the endoplasmic reticulum, as shown by confocal microscopy of cells expressing isoflavone hydroxylase-GFP fusions. Identical cellular localization is shown by the IFS cytochrome P450 and IOMT in alfalfa (Liu and Dixon, 2001). Further studies, utilizing approaches such as fluorescence energy resonance transfer (FRET; Elangovan *et al.*, 2003) will be necessary to determine whether isoflavone hydroxylases are physically associated in complexes with IFS on the ER membranes.

### Roles of I2′H and I3′H in M. truncatula

Isoflavone 2′-hydroxylase is clearly involved in the biosynthesis of the phytoalexin medicarpin in *M. truncatula* roots, elicited cell cultures, and infected leaves, where the presence of transcripts correlates with appearance of medicarpin (data not shown). However, the function of I3′H is less clear. 3′-Hydroxylation of the B-ring of flavonoids is usually associated with the formation of a methylenedioxy bridge linking positions 3′ and 4′, as found in maackiain and pisatin (Figure 1); the proposed pathway involves successive methylation of the 4′-hydroxyl, followed by 3′-hydroxylation and closing of the ring by the action of a cytochrome P450 enzyme (CYP(X)) in Figure 1; Dixon, 1999). These reactions are believed to occur prior to 2′-hydroxylation during the biosynthesis of maackiain via pseudobaptigenin in chickpea (Clemens and Barz, 1996; Clemens *et al.*, 1993; Figure 1).

*M. truncatula* I3′H hydroxylates both biochanin A (preferred) and formononetin. In chickpea, the 3′-hydroxylated product of biochanin A, pratensein (Figure 1), is a naturally occurring constituent (Wong, 1975). However, pratensein has not been reported in either alfalfa or *M. truncatula* (ILDIS, 1994). 3′-Hydroxylation of biochanin A in chickpea microsomes was suggested to be catalyzed by an enzyme distinct from that responsible for 3′-hydroxylation of formononetin (Clemens *et al.*, 1993), unlike the situation we now report with the recombinant enzymes from *Medicago*.

P450-mediated 2′-hydroxylation of pseudobaptigenin occurs after 3′-hydroxylation and methylenedioxy bridge formation in maackiain biosynthesis in pea and chickpea (Figure 1). Pseudobaptigenin is an excellent substrate for *M. truncatula* I2′H *in vitro*. However, there is as yet no evidence for formation of maackiain in *Medicago* species (ILDIS, 1994), as confirmed in the present study. The detection limit for authentic maackiain in our HPLC system was 3.5 nmol g⁻¹ FW, much less than the observed amount of maackiain accumulating in elicited chickpea cell cultures (600 nmol g⁻¹ FW; Mackenbrock *et al.*, 1993) or elicited red clover roots (approximately 500 nmol g⁻¹ FW; Tebayashi *et al.*, 2001). The only B-ring methylenedioxy-substituted isoflavonoid derivative reported to date from *Medicago* species is the coumestan medicagol (Figure 1; ILDIS, 1994; Olah and Sherwood, 1971). More work is needed to define the coumestan metabolites made in *M. truncatula* and the conditions under which they are formed.

Whatever the actual biosynthetic role of I3′H in *M. truncatula*, it appears to be involved in pathways that are independent from those involving I2′H. For example, I3′H is induced in leaves treated with methyl jasmonate or damaged by insect herbivory, conditions under which I2′H is not induced. In contrast, I2′H, but not I3′H, is induced in yeast elicited cell cultures that accumulate medicarpin (data not shown), and is more strongly induced than the 3′-hydroxylase in *Phoma*-infected leaves that also accumulate medicarpin (He and Dixon, 2000). Clearly, the expression pattern points towards involvement of isoflavonoids in insect-mediated responses in *M. truncatula*. It is interesting that the biosynthesis of some pterocarpanes such as maackiain, and the rotenoids, involves 3′-hydroxylation, and all these compounds have anti-insect activity (Bhandari *et al.*, 1992; Dixon, 1999; Nicholas *et al.*, 1985; Simmonds and Stevenson, 2001).

### Metabolic engineering of complex isoflavonoids

Introduction of isoflavonoid phytoalexin biosynthetic pathways into non-legumes has been suggested as a potential means of engineering improved disease resistance (Essenberg, 2001). This strategy could be linked to delivery of health beneficial isoflavones to crop species (Dixon and Ferreira, 2002). To date, IFS has been successfully expressed, with resulting production of genistein conjugates in Arabidopsis, corn, and tobacco (Jung *et al.*, 2000; Liu *et al.*, 2002; Yu *et al.*, 2000). I2′H from *M. truncatula* can be expressed as a functional protein in Arabidopsis, and it might therefore soon be possible to introduce the complete

pathway for formation of pterocarpans into Arabidopsis and other species that do not possess this pathway. Furthermore, if I2′H indeed catalyzes an important rate limiting step for pterocarpan biosynthesis in legume species (Gunia et al., 1991), overexpression of I2′H in plants such as alfalfa, pea, and soybean might lead to improved disease resistance.

### Experimental procedures

#### Chemicals
Isoformononetin, 2′-hydroxyformononetin, 3′-hydroxyformononetin (calycosin), and pseudobaptigenin were purchased from Apin (Abingdon, UK). Maackiain and pisatin were kindly provided by Dr H.D. VanEtten (University of Arizona, Tucson). Maackiain glucoside and maackiain glucose malonate standards were gifts from Dr S. Tebayashi (Kochi University, Japan). Medicarpin was from our lab collection. Other isoflavonoids were from Indofine Chemical Company (Somerville, NJ, USA). CO was obtained from Aldrich (Milwaukee, WI, USA) and all other chemicals were from Sigma (St Louis, MO, USA).

#### Plant materials and treatments
*Medicago truncatula* (Jemalong A17) plants were grown in sand at 22°C, under a 16-h light/8-h dark regime and at 50% humidity in a growth chamber for 42 days. The plants were watered once a day and fertilized two times a week. Young shoots were cut at the second nodes and the upper portions were allowed to stand in aqueous 0.33-mM copper chloride solution for 12 h or 500-μM methyl jasmonate for 48 h; control plants were placed in water along with the treated samples and incubated for the same durations. Drought stress treatments were imposed by withholding watering for 5 days under the same growth conditions; control plants were of the same age but watered normally. Harvested roots, leaves, and stems from plants of the same developmental age were frozen in liquid N₂ and stored at –80°C prior to RNA extraction or metabolite analyses.

*Phoma medicaginis* was streaked on PDA medium plates and grown at 28°C for about 20 days. Fungal spores were collected by washing with 0.2% Tween in ddH₂O. Leaves pre-selected for fungal infection were wounded with a tracing wheel (He and Dixon, 2000), and the selected leaves on the intact plants were sprayed with *Phoma* spore solution (1.3 × 10⁷ ml⁻¹), covered with a plastic bag, and grown in the growth chamber. Leaves from control plants were treated identically except for the lack of fungus.

For insect herbivory studies, fourth instar larvae of beet armyworm (*Spodoptera exigua*) were allowed to feed overnight on the leaves of 9-week-old plants growing in a growth chamber. The damaged leaves were collected for analysis. Control leaves were collected from the same plant before exposure to larvae.

*Medicago truncatula* cell suspension cultures were initiated from root callus and maintained in Schenk and Hildebrandt medium at 24°C in the dark as described previously by Dalkin et al. (1990). The cells were subcultured every 15 days. Cells were treated with yeast elicitor (Schumacher et al., 1987) 12 days after subculture at a final concentration of 50 μg glucose equivalent ml⁻¹ culture. Cells were collected by vacuum filtration through a nylon mesh, washed with ddH₂O, and frozen in liquid N₂.

---

### Identification and isolation of M. truncatula cDNAs encoding CYP81E proteins

Analysis of the TIGR *M. truncatula* Gene Index (MtGI) was performed by gene BLAST search using I2′H sequence information and gene product name with Matrix at blosum62, Expect at 10, and Description at 20; all other parameters were set at the default.

A cDNA library was constructed from mRNA from developmentally pooled *M. truncatula* root tissue (Noble Foundation library #4048 as described in MtGI). A heterologous licorice I2′H cDNA probe (Akashi et al., 1998) was labeled with ³²P and used to screen the library according to standard procedures (Stratagene, La Jolla, CA, USA). After two rounds of screening, positive clones were picked and sequenced. Sequence analysis was performed using ExPASy Molecular Biology Server tools and DNASTAR, and sequence alignments were performed by the CLUSTALW (1.81) method. *In silico* expression analysis was performed by counting the EST numbers for the targeted gene (tentative consensus (TC)) in each specific library in the MtGI database and normalizing to EST numbers sequenced from that library (see legend to Figure 7; Dixon et al., 2002).

---

### Expression of Medicago CYP81E in yeast

The open-reading frame of MtCYP81E7 was amplified with forward primer AACGGATCCATGGGAATCCTTTC and reverse primer GAACGGTACTTAGATGAATTAC, which introduced BamHI and KpnI restriction sites, respectively (underlined nucleotides). The BamHI site in the original clone was eliminated by mutation of the third nucleotide (G to A) in the second translated codon after the ATG start site. The PCR reaction was performed at 94°C for 3 min, then 94°C for 45 sec, 53°C for 45 sec, and 72°C for 1 min, for a total of 35 cycles, using a mixture of high fidelity Pfu and Taq (1 : 2) DNA polymerase.

MtCYP81E9 was amplified with the forward primer AACGGATCCATGACCTTATTCTATTACTC and the reverse primer AACAGGTACCTCACTTCGTTACATCA. As the MtCYP81E9 clone apparently lacked nucleotides encoding two amino acid residues at the N-terminus (within the membrane anchor), an ATG start codon and a codon encoding threonine (indicated in italics) were introduced based on alignments of the three *M. truncatula* putative CYP81E family members. PCR was performed as for MtCYP81E7, except the annealing temperature was 54°C.

PCR products were digested with BamHI and KpnI and ligated into the yeast expression vector YeDP60 following BamHI/KpnI digestion. MtCYP81E8 was directly excised from the original clone in pBSK(+) by BamHI and KpnI and inserted into YeDP60. All constructs were sequenced to ensure that there were no PCR-introduced errors. The constructs were then transferred into yeast (*Saccharomyces cerevisiae*) strain WAT11 with a chromosomally integrated *Arabidopsis NADPH: cytochrome P450 reductase* gene (Pompon et al., 1996) by the LiAc method (Gietz et al., 1992), and transformants were selected on SGI media (Pompon et al., 1996). Incubation of yeast cultures and induction of protein expression were performed using the high-density procedure (Pompon et al., 1996). After induction with 20% galactose for approximately 18 h, yeast cells were harvested, weighed, and immediately processed for microsomal fractionation.

---

### Preparation of yeast microsomes

Disruption and subcellular fractionation of yeast cells were performed with a modified procedure that combined mechanical rupture and enzymatic lysis methods in order to improve fractionation efficiency. Harvested yeast cells (usually 9–11 g cell pellet

from 1 l of S. cerevisiae cell culture) were washed with TEK buffer (50 mM Tris–HCl (pH 7.5), 1 mM EDTA, 0.1 M KCl) once briefly, then re-suspended in two volumes of Zymolyase buffer A (50 mM Tris–HCl (pH 7.5), 10 mM MgCl₂, 1 M sorbitol, 30 mM DTT), incubated at room temperature for 15 min, centrifuged at 5000 g for 5 min, and re-suspended in an equal volume of Zymolyase buffer B (same as Zymolyase buffer A except for DTT at 1 mM). Zymolyase 100T from Arthrobacter luteus (200 U ml⁻¹; Seikagaku, Tokyo) was then added. After incubation at approximately 80 r.p.m. for 30–40 min, spheroplasts were centrifuged down for 5 min, washed two times with Zymolyase buffer B, and then two times with phosphate buffer A (0.1 M potassium phosphate (pH 8.0), 1 M sorbitol, 14 mM 2-mercaptoethanol). The spheroplasts were re-suspended in a minimum volume of phosphate buffer B (0.1 M potassium phosphate (pH 8.0), 0.4 M sucrose, 14 mM β-mercaptoethanol, 1 mM PMSF, and 1× protein inhibitor cocktail (Roche Diagnostics GmbH, Mannheim, Germany)), and one volume of glass beads (diameter 0.45–0.50 mm) was added. Samples were vigorously vortexed at 4°C, with the quality of cell disruption checked under the microscope. The supernatant was then removed and the beads were washed four times with phosphate buffer B. Supernatants were pooled and centrifuged at 12 000 g for 10 min; the resulting supernatant was then ultracentrifuged at 140 000 g for 90 min. The supernatant was discarded and the pellet washed briefly with phosphate buffer C (0.1 M potassium phosphate (pH 8.0), 0.4 M sucrose, 0.5 mM glutathione), and re-suspended in the same buffer plus 5% glycerol at about 12 mg total protein per milliliter. Protein was quantified using a Bio-Rad protein assay kit (Bio-Rad, Hercules, CA, USA).

**Carbon monoxide difference spectra of microsomes and assay of hydroxylase activities**

Carbon monoxide difference spectra of microsomes were obtained by the method of Mihaliak et al., (1993) using a Shimatzu UV-2401PC split beam spectrophotometer. Briefly, microsomal preparations (approximately 1 mg ml⁻¹ total protein) in 0.1 M phosphate buffer C were mixed with a few mg of solid Na₂S₂O₄, and equal volumes of this preparation was distributed equally into sample and reference cuvettes. The baseline was recorded and CO was gently bubbled through the sample cell for a few minutes prior to recording the difference spectrum from 390 to 500 nm. An extinction coefficient of 91 cm⁻¹ mM⁻¹ (A₄₅₀ – A₄₉₀) was used to calculate the cytochrome P450 concentration.

Hydroxylase assays were performed in phosphate buffer C in a total volume of 150 μl containing 2 mM NADPH, 40 μM isoflavone substrate, and 40–70 pmol P450 protein. Reactions were incubated at 25°C for 1.5 h (over which period the reactions were linear), with gentle shaking, and stopped by extracting two times with 500 μl of ethyl acetate. The extracts were combined, evaporated under N₂, re-suspended in 60 μl of methanol, and analyzed by HPLC or LC-MS. All assays to determine enzyme specificity were carried out in duplicate. For kinetic studies, the isoflavone concentration was varied at a fixed concentration of NADPH (2 mM), and kinetic constants were calculated from initial rate data using Lineweaver–Burke double reciprocal plots. All kinetic analyses were carried out in triplicate.

acetonitrile of 0–5 min (5%), 5–10 min (5–10%), 10–15 min (10–15%), 15–20 min (15%), 20–25 min (15–17%), 25–30 min (17–23%), and 30–65 min (23–50%) at a constant flow rate of 0.8 ml min⁻¹. UV absorption was monitored at 235, 254, 287, and 310 nm with a photodiode array detector. When biochanin A and prunetin were used as substrates, the products were resolved with the acetonitrile gradient described previously by Liu et al. (2002).

Product identification was first based on comparisons of chromatographic behavior and UV spectra with authentic standards. Identifications were then confirmed by HPLC-MS, using an HP1100 liquid chromatograph coupled with a Bruker Esquire Ion-Trap mass spectrometer equipped with an electrospray source. HPLC separation was achieved using a J.T. Baker Bakerbond reverse-phase column (C18, 5 μm, 4.6 mm × 250 mm; J.T. Baker, Philipsberg, NJ, USA). Samples were eluted with the gradient described above but with 0.1% aqueous acetic acid replacing the 1% phosphoric acid. Ion charge control was set at 30 000 with a maximum acquired time of 100 ms. Mass spectra were recorded over a range of 50–2200 m/z.

**RNA extraction and gel blot analysis**

Total RNA was isolated from about 1 g of treated plant material with the Tri-reagent extraction kit following the manufacturer’s protocol (MRC Inc., Cincinnati, OH, USA). Twenty-microgram RNA was loaded per lane, resolved by electrophoresis on a 1% agarose gel, and transferred to Hybond-H⁺ nylon membrane. RNA blots were developed using either ³²P-labeled probes (for insect herbivory and elicited cell culture samples) or the ECL non-radioactive system. For radioactive detection, blots were probed with MtI2′H or MtI3′H full-length cDNA in Church buffer (Church and Gilbert, 1984) at very high stringency (68°C). The hybridized membranes were washed at high stringency using phosphate buffer I (40 mM NaHPO₄ (pH 7.2), 5% SDS, 1% BSA) and buffer II (40 mM NaHPO₄ (pH 7.2), 1% SDS) two times, respectively, at 68°C, and finally exposed to phosphorimaging or X-ray film. RNA loading was monitored by ethidium bromide staining. For analysis of blots with the ECL direct nucleic acid labeling and detection system (Amersham Biosciences, Little Chalfont, Buckinghamshire, UK), MtCYP81E7, MtCYP81E9, and Mt 18S RNA probes were labeled with horseradish peroxidase following the manufacturer’s procedure. Hybridization and washing were performed at 42°C, and the images were developed using detection reagents containing hydrogen peroxide and luminol. X-ray film was exposed to the blots for 1–30 min. The triplicate RNA blots (using two separate tissue samples) gave essentially the same results.

**Extraction and analysis of isoflavonoids from M. truncatula**

Phenolic compounds were extracted using pre-cooled acetone and acetone:methanol, hydrolyzed with β-glucosidase, and partitioned into ethyl acetate as described previously by Liu and Dixon (2001). Alternatively, extraction was performed according to Higgins (1972) using 95% ethanol and partitioning of the β-glucosidase digest into chloroform. Extracts were dried and then re-dissolved in methanol for HPLC analysis. Chromatography was by the same procedure described above for analysis of enzymatic products.

**Analysis of enzymatic products by HPLC-UV and HPLC-MS**

Reaction products from enzyme assays (30 μl) were analyzed by HPLC. Samples were applied to an octadecylsilane (ODS2) reverse phase column (5 μm particle size; 4.6 mm × 250 mm), and eluted in 1% phosphoric acid with an increasing concentration gradient of

**Construction and transient expression of chimeric eGFP fusion genes**

To create fusion proteins of MtI2′H and MtI3′H with eGFP, the coding region sequence of eGFP containing the multiple restriction

sites was moved from pEGFPN-1 (Clontech, Palo Alto, CA, USA), by digestion with EcoRI and Xbal, to the shuttle vector pRTL2 (Restrepo *et al.*, 1990), generating eGFP expression vector pRTLGFP. The *Mtl2′H* open-reading frame was amplified using *Pfu* DNA polymerase (Stratagene, La Jolla, CA, USA) with the forward primer 5′-CAAACGGTACCATGGGGATC-3′ and the reverse primer 5′-ATGAGGTACCTTGAAAAACCTTG-3′. The *Mtl3′H* coding region was amplified using the forward primer 5′-GACGGTACCATGACCTATTCTATTACT-3′ and the reverse primer 5′-GACGGATCCTCGTTACATCATTTGGCTA-3′. All PCR products were sequenced to ensure that they did not contain errors. The *Mtl2′H* fragment was ligated into *KpnI*-digested pRTLGFP (the orientation being determined by PCR with a universal pRTL2 primer and reverse *Mtl2′H* primer), and the *Mtl3′H* fragment was inserted into *BamHI/* *KpnI*-digested pRTLGFP.

The procedure for transient expression of chimeric genes encoding *Mtl2′H-* and *Mtl3′H*-GFP fusions in alfalfa (*cv.* Regen SY) leaves, and confocal microscopy for detection of green fluorescence localization, was as previously described by Liu and Dixon (2001).

### Expression of *Mtl2′H* in Arabidopsis

The full-length *Mtl2′H* cDNA was removed from the original pBSK clone by digestion with EcoRI and *KpnI* and inserted into the shuttle vector pRTL2. After digestion with *HindIII*, the chimeric gene under control of the cauliflower mosaic virus 35S promoter was moved into the binary vector pCAMBIA3300. Transformation and selection of transgenic plants was as previously described by Liu *et al.* (2002).

Leaves from *l2′H* transgenic plant #33-2-5 and empty vector control line #33–00 were cut at the lower petiole and allowed to stand in the wells of 96 well tissue culture plates containing a solution of formononetin (2 mM) in DMSO:methanol:H₂O (1 : 1 : 2). The cut leaves were incubated at 22°C under greenhouse conditions for 2 h and then exhaustively washed with ddH₂O. The collected leaves were frozen in liquid N₂ prior to extraction of phenolic compounds and analysis by HPLC with the procedure and gradient II described previously by Liu and Dixon (2001).

### Acknowledgements

We thank Dr Ken Korth (University of Arkansas) for providing insect-treated plant materials; Dr Hans VanEtten (University of Arizona) for authentic maackiain standard; Dr Shin-ichi Tebayashi (Kochi University) for authentic maackiain conjugate standards; Drs Denis Pompon (Gif-sur-Yvette) and Xiao-Ya Chen (Shanghai Institute of Plant Physiology and Ecology, Chinese Academy of Sciences) for yeast expression vectors and strains; Dr David R. Nelson (University of Tennessee) for nomenclature of P450 clones; and Drs Xian-Zhi He and Kiran Mysore for critical reading of the manuscript. This work was supported by the Samuel Roberts Noble Foundation.

### References

Adlercreutz, H. and Mazur, W. (1997) Phyto-estrogens and western diseases. *Ann. Med.* 29, 95–120.

Akashi, T., Aoki, T. and Ayabe, S. (1998) CYP81E1, a cytochrome P450 cDNA of licorice (*Glycyrrhiza echinata* L.), encodes isoflavone 2′-hydroxylase. *Biochem. Biophys. Res. Commun.* 251, 67–70.

Akashi, T., Sawada, Y., Shimada, N., Sakurai, N., Aoki, T. and Ayabe, S. (2003) cDNA cloning and biochemical characterization of S-adenosyl-L-methionine: 2,7,4′-trihydroxyisoflavanone 4′-O-methyltransferase, a critical enzyme of the legume isoflavonoid phytoalexin pathway. *Plant Cell Physiol.* 44, 103–112.

Bell, C., Dixon, R.A., Farmer, A.D. *et al.* (2001) The *Medicago* genome initiative: a model legume database. *Nucl. Acids Res.* 29, 114–117.

Bhandari, P., Crombie, L., Daniels, P., Holden, I., Van Bruggen, N. and Whitting, D.A. (1992) Biosynthesis of the A-B-C-D-ring system to the rotenoid amorphigenin by *Amorpha fruticosa* seedlings. *J. Chem. Soc. Perkin Trans.* 13, 839–849.

Choudhary, A.D., Kessmann, H., Lamb, C.J. and Dixon, R.A. (1990) Stress responses in alfalfa (*Medicago sativa* L.). Part IV. Expression of defense gene constructs in electroporated suspension cell protoplasts. *Plant Cell Rep.* 9, 42–46.

Church, G.H. and Gilbert, W. (1984) Genomic sequencing. *Proc. Natl. Acad. Sci. USA,* 81, 65–71.

Clemens, S. and Barz, W. (1996) Cytochrome P450-dependent methylenedioxy bridge formation in *Cicer arietinum*. *Phytochemistry,* 41, 457–460.

Clemens, S., Hinderer, W., Wittkampf, U. and Barz, W. (1993) Characterization of cytochrome P450-dependent isoflavone hydroxylases from chickpea. *Phytochemistry,* 32, 653–657.

Cook, D.R. (1999) *Medicago truncatula* – a model in the making! *Curr. Opin. Plant Biol.* 2, 301–304.

Dalkin, K., Edwards, R., Edington, B. and Dixon, R.A. (1990) Stress responses in alfalfa (*Medicago sativa* L.). I. Elicitor-induction of phenylpropanoid biosynthesis and hydrolytic enzymes in cell suspension cultures. *Plant Physiol.* 92, 440–446.

Dewick, P.M. (1993) Isoflavonoids. In *The Flavonoids: Advances in Research Since 1986.* (Harborne, J.B., ed.). London: Chapman & Hall, pp. 117–238.

Dewick, P.M. and Martin, M. (1979) Biosynthesis of pterocarpan and isoflavan phytoalexins in *Medicago sativa*: the biochemical interconversion of pterocarpans and 2′-hydroxyisoflavans. *Phytochemistry* 18, 591–596.

Dewick, P.M. and Ward, D. (1978) Isoflavone precursors of the pterocarpan phytoalexin maackiain in *Trifolium pratense*. *Phytochemistry,* 17, 1751–1754.

Dixon, R.A. (1999) Isoflavonoids: biochemistry, molecular biology and biological functions. In *Comprehensive Natural Products Chemistry,* Vol. 1 (Sankawa, U., ed.). Oxford: Elsevier, pp. 773–823.

Dixon, R.A. and Ferreira, D. (2002) Molecules of interest: Genistein. *Phytochemistry,* 60, 205–211.

Dixon, R.A. and Sumner, L.W. (2003) Legume natural products. Understanding and manipulating complex pathways for human and animal health. *Plant Physiol.* 131, 878–885.

Dixon, R.A., Achnine, L., Kota, P., Liu, C.-J., Reddy, M.S.S. and Wang, L. (2002) The phenylpropanoid pathway and plant defense – a genomics perspective. *Mol. Plant Pathol.* 3, 371–390.

Elangovan, M., Wallrabe, H., Chen, Y., Day, R., Barroso, M. and Periasamy, A. (2003) Characterization of one- and two-photon excitation fluorescence resonance energy transfer microscopy. *Methods,* 29, 58–73.

Essenberg, M. (2001) Prospects for strengthening plant defenses through phytoalexin engineering. *Physiol. Mol. Plant Pathol.* 59, 71–81.

Gietz, D., Jane, A.S., Woods, R.A. and Schiestl, R.H. (1992) Improved method for high efficiency transformation of intact yeast cells. *Nucl. Acids Res.* 20, 1425.

Gunia, W., Hinderer, W., Wittkampf, U. and Barz, W. (1991) Elicitor induction of cytochrome P-450 monooxygenases in cell

© Blackwell Publishing Ltd, *The Plant Journal,* (2003), 36, 471–484

suspension cultures of chickpea (*Cicer arietinum* L.) and their involvement in pterocarpan phytoalexin biosynthesis. *Z. Naturforsch.* 46c, 58–66.

He, X.-Z. and Dixon, R.A. (2000) Genetic manipulation of isoflavone 7-O-methyltransferase enhances the biosynthesis of 4′-O-methylated isoflavonoid phytoalexins and disease resistance in alfalfa. *Plant Cell*, 12, 1689–1702.

Higgins, V.J. (1972) Role of the phytoalexin medicarpin in three leaf spot diseases of alfalfa. *Physiol. Plant Pathol.* 2, 289–300.

Hinderer, W., Flentje, U. and Barz, W. (1987) Microsomal isoflavone 2′- and 3′-hydroxylases from chickpea (*Cicer arietinum* L.) cell suspensions induced for pterocarpan phytoalexin formation. *FEBS Lett.* 214, 101–106.

ILDIS (International Legume Database and Information Service) (1994) Phytochemical dictionary of the Leguminosae. *Plants and their Constituents*, Vol. 1 (Bisby, F.A., Buckingham, J. and Harborne, J.B., eds). London: Chapman & Hall.

Ingham, J.L. (1982) Phytoalexins from the Leguminosae. In *Phytoalexins* (Bailey, J.A. and Mansfield, J.W., eds). New York: Halstead Press, pp. 21–80.

Jung, W. Yu, O., Lau, S.-M.C., O’Keefe, D.P., Odell, J., Fader, G. and McGonigle, B. (2000) Identification and expression of isoflavone synthase, the key enzyme for biosynthesis of isoflavones in legumes. *Nat. Biotechnol.* 18, 208–212.

Kochs, G. and Grisebach, H. (1986) Enzymic synthesis of isoflavones. *Eur. J. Biochem.* 155, 311–318.

Kochs, G. and Grisebach, H. (1989) Phytoalexin synthesis in soybean: purification and reconstitution of cytochrome P450 3,9-dihydroxypterocarpan 6a-hydroxylase and separation from cytochrome P450 cinnamate 4-hydroxylase. *Arch. Biochem. Biophys.* 273, 543–553.

Lambert, N., Trouslot, M.-F., Nef-Campa, C. and Chrestin, H. (1993) Production of rotenoids by heterotrophic and photo-mixotrophic cell cultures of *Tephrosia vogelii*. *Phytochemistry*, 34, 1515–1520.

LatundeDada, A.O., CabelloHurtado, F., Czittrich, N., Didierjean, L., Schopfer, C., Hertkorn, N., Werck-Reichhart, D. and Ebel, J. (2001) Flavonoid 6-hydroxylase from soybean (*Glycine max* L.), a novel plant P-450 monooxygenase. *J. Biol. Chem.* 276, 1688–1695.

Liu, C.-J. and Dixon, R.A. (2001) Elicitor-induced association of isoflavone O-methyltransferase with endomembranes prevents formation and 7-O-methylation of daidzein during isoflavonoid phytoalexin biosynthesis. *Plant Cell*, 13, 2643–2658.

Liu, C.-J., Blount, J.W., Steele, C.L. and Dixon, R.A. (2002) Bottle-necks for metabolic engineering of isoflavone glycoconjugates in *Arabidopsis*. *Proc. Natl. Acad. Sci. USA*, 99, 14578–14583.

Mabry, T.J., Markham, K.R. and Thomas, M.B. (1970) *The Systematic Identification of the Flavonoids*. New York: Springer-Verlag.

Mackenbrock, K. and Barz, W. (1983) 3′-Hydroxylation of 4′-methoxyisoflavones by *Fusarium oxysporum* f. *lycopersici*. *Z. Naturforsch.* 38c, 708–710.

Mackenbrock, U., Gunia, W. and Barz, W. (1993) Accumulation and metabolism of medicarpin and maackiain malonylglucosides in elicited chickpea (*Cicer arietinum* L.) cell suspension cultures. *J. Plant Physiol.* 142, 385–391.

Mihaliak, C.A., Karp, F. and Croteau, R. (1993) Cytochrome P-450 terpene hydroxylases. *Meth. Plant Biochem.* 9, 261–279.

Nicholas, B., Crombie, L. and Crombie, W.M. (1985) Rotenoids of *Lonchocarpus salvadorensis*: their effectiveness in protecting seeds against bruchid predation. *Phytochemistry*, 24, 2881–2883.

Olah, A.F. and Sherwood, R.T. (1971) Flavones, isoflavones, and coumestans in alfalfa infected by *Ascochyta imperfecta*. *Phytopathology*, 61, 65–69.

Oldroyd, G.E. and Geurts, R. (2001) *Medicago truncatula*, going where no plant has gone before. *Trends Plant Sci.* 6, 552–554.

Overkamp, S., Hein, F. and Barz, W. (2000) Cloning and characterization of eight cytochrome P450 cDNAs from chickpea (*Cicer arietinum* L.) cell suspension cultures. *Plant Sci.* 155, 101–108.

Pompon, D., Louerat, B., Bronine, A. and Urban, P. (1996) Yeast expression of animal and plant P450s in optimized redox environments. *Meth. Enzymol.* 272, 51–64.

Restrepo, M.A., Freed, D.D. and Carrington, J.C. (1990) Nuclear transport of plant potyviral proteins. *Plant Cell*, 2, 987–998.

Schumacher, H.-M., Gundlach, H., Fiedler, F. and Zenk, M.H. (1987) Elicitation of benzophenanthridine alkaloid synthesis in *Eschscholtzia* cell cultures. *Plant Cell Rep.* 6, 410–413.

Shimada, N., Akashi, T., Aoki, T. and Ayabe, S. (2000) Induction of isoflavonoid pathway in the model legume *Lotus japonicus*: molecular characterization of enzymes involved in phytoalexin biosynthesis. *Plant Sci.* 160, 37–47.

Simmonds, M.S. and Stevenson, P.C. (2001) Effect of isoflavonoids from *Cicer* on larvae of *Heliocoverpa armigera*. *J. Chem. Ecol.* 27, 965–977.

Tebayashi, S.-I., Ishihara, A. and Iwamura, H. (2001) Elicitor-induced changes in isoflavonoid metabolism in red clover roots. *J. Exp. Bot.* 52, 681–668.

Tolleson, W.H., Doerge, D.R., Churchwell, M.I., Marques, M.M. and Roberts, D.W. (2002) Metabolism of biochanin A and formononetin by human liver microsomes *in vitro*. *J. Agric. Food Chem.* 50, 4783–4790.

Wong, E. (1975) The isoflavonoids. In *The Flavonoids, Part 2* (Harborne, J.B. Mabry, T.J. and Mabry, H., eds). New York, San Francisco: Academic Press, pp. 743–800.

Yu, O., Jung, W., Shi, J., Croes, R.A., Fader, G.M., McGonigle, B. and Odell, J.T. (2000) Production of the isoflavones genistein and daidzein in non-legume dicot and monocot tissues. *Plant Physiol.* 124, 781–794.

The nucleotide sequences reported in this paper have been submitted to the GenBank™/EMBL data library and are available under the accession numbers AY278227 (CYP81E7), AY278228 (CYP81E9), and AY278229 (CYP81E8).
