
Eur. J. Biochem. 270, 4413–4419 (2003) © FEBS 2003 doi:10.1046/j.1432-1033.2003.03830.x

# An unusual amide synthetase (CouL) from the coumermycin A₁ biosynthetic gene cluster from *Streptomyces rishiriensis* DSM 40489

**Elisabeth Schmutz¹, Marion Steffensky¹, Jürgen Schmidt², Andrea Porzel², Shu-Ming Li¹ and Lutz Heide¹**

¹Eberhard-Karls-Universität Tübingen, Pharmazeutische Biologie, Tübingen, Germany; ²Institut für Pflanzenbiochemie, Halle (Saale), Germany

The aminocoumarin antibiotic coumermycin A₁ produced by *Streptomyces rishiriensis* DSM 40489 contains two amide bonds. The biosynthetic gene cluster of coumermycin contains a putative amide synthetase gene, *couL*, encoding a protein of 529 amino acids. CouL was overexpressed as hexahistidine fusion protein in *Escherichia coli* and purified by metal affinity chromatography, resulting in a nearly homogenous protein. CouL catalysed the formation of both amide bonds of coumermycin A₁, i.e. between the central 3-methylpyrrole-2,4-dicarboxylic acid and two aminocoumarin moieties. Gel exclusion chromatography showed that the enzyme is active as a monomer. The activity was strictly dependent on the presence of ATP and Mn²⁺ or Mg²⁺. The apparent *K*ₘ values were determined as 26 µM for the 3-methylpyrrole-2,4-dicarboxylic acid and 44 µM for the aminocoumarin moiety, respectively. Several analogues of the pyrrole dicarboxylic acid were accepted as substrates. In contrast, pyridine carboxylic acids were not accepted. 3-Dimethylallyl-4-hydroxybenzoic acid, the acyl component in novobiocin biosynthesis, was well accepted, despite its structural difference from the genuine acyl substrate of CouL.

**Keywords:** amide synthetase; aminocoumarin; antibiotic; biosynthesis; coumermycin.

Coumermycin A₁, like novobiocin and clorobiocin, belongs to the aminocoumarin antibiotics and is produced by *Streptomyces rishiriensis* DSM 40489. These antibiotics are potent inhibitors of bacterial DNA gyrase. Their interaction with the ATP-binding site in the B subunit of the enzyme has been investigated by X-ray crystallography [1,2]. Aminocoumarins have not been highly successful as pharmaceuticals for several reasons, e.g. poor solubility in water and poor oral absorption, low activity against Gram-negative bacteria, and toxicity in eukaryotes. Knowledge about the biosynthesis of these compounds may permit the development of new compounds with improved characteristics by combinatorial biosynthesis.

Both in coumermycin A₁ and in novobiocin, a 3-amino-4,7-dihydroxy-8-methyl coumarin (ring B of novobiocin) is linked to an acyl component via an amide bond. In novobiocin, the acyl component is a prenylated 4-hydroxybenzoate moiety (Fig. 1), and the formation of this amide bond is catalysed by the amide synthetase NovL. NovL catalyses both the activation of the acyl component in the form of adenylate and subsequent transfer of the acyl group onto the amino group of ring B [3]. In coumermycin A₁, the acyl component 3-methylpyrrole-2,4-dicarboxylic acid is connected to two aminocoumarin moieties via amide bonds (Fig. 1). However, sequence analysis revealed only a single amide synthetase gene (*couL*) in the biosynthetic gene cluster of coumermycin A₁ [4]. The protein CouL encoded by this gene shows high homology to NovL. For the biosynthesis of coumermycin, a mechanism similar to that of NovL would imply four enzymatic steps from 3-methylpyrrole-2,4-dicarboxylic acid to the diamide of this compound (Fig. 1), i.e. adenylation of one of the carboxyl groups, followed by acyl transfer, and adenylation of the second carboxyl group, followed by the second acyl transfer. The exact sequence of these four steps was unknown at the beginning of our study. An essential question to be answered by the present investigation was whether CouL is responsible for the catalysis of all four enzymatic steps, or whether additional enzymes are involved in this reaction sequence.

---

**Correspondence to** L. Heide and S.-M. Li, Eberhard-Karls-Universität Tübingen, Pharmazeutische Biologie, Auf der Morgenstelle 8, 72076 Tübingen, Germany.  
L. Heide, Fax: + 49 707 129 5250, Tel.: + 49 707 1297 8789, E-mail: heide@uni-tuebingen.de;  
S.-M. Li, Fax: + 49 707 129 5250, Tel.: + 49 707 1297 6995, E-mail: shuming.li@uni-tuebingen.de  

**Abbreviations:** ring B, 3-amino-4,7-dihydroxy-8-methyl coumarin; SRM, selected reaction monitoring; R<sub>T</sub>, retention time; CoA, Coenzyme A.  
(Received 31 July 2003, revised 9 September 2003, accepted 11 September 2003)

---

## Materials and methods

### Chemicals

3,5-Dimethylpyrrole-2,4-dicarboxylic acid 4-ethyl ester was from Sigma. 4-Acetyl-3,5-dimethyl-2-pyrrole carboxylic acid, 3-carboxy-1,4-dimethyl-2-pyrrole acetic acid, pyridine-2,6-dicarboxylic acid, 2,6-dimethylpyridine-3,5-dicarboxylic acid and 6-methylpyridine-2,3-dicarboxylic acid were from Aldrich. Pyrrole-2-carboxylic acid was from

4414 E. Schmutz et al. (Eur. J. Biochem. 270)

![Chemical Structures](https://i.imgur.com/yourimageurl.png)

Fig. 1. Proposed pathway for the biosynthesis of coumermycin A₁ and novobiocin.

Lancaster, pyrrole-3-carboxylic acid was from Acros. 3-Dimethylallyl-4-hydroxybenzoic acid (ring A) was obtained by hydrolysis of novobiocin as described previously [5]. 3-Amino-4,7-dihydroxy-8-methyl-coumarin (ring B) was kindly provided by Pharmacia and Upjohn.

### Synthesis of 3-methylpyrrole-2,4-dicarboxylic acid

3,5-Dimethylpyrrole-2,4-dicarboxylic acid diethyl ester (Aldrich) was oxidized to 4-methylpyrrole-2,3,5-tricarb oxylic acid 2,4-diethylester as previously described by Corwin *et al.* [6]. The acid was then decarboxylated by heating with bronze powder (Alfa Aesar), as described by Woodward *et al.* [7]. Afterwards, the mixture was extracted with ethyl acetate and the organic solvent was evaporated. Hydrolysis of the ethyl esters was carried out by heating the residue in aqueous 4 M NaOH for 4 h. After hydrolysis 3-methylpyrrole-2,4-dicarboxylic acid was purified by HPLC using a Multosphere RP18-6 column (250 × 10 mm, 5 μm; C + S Chromatographie Service, Düren, Germany) with a linear gradient from 32 to 100% methanol in 1% aqueous formic acid and detection at 259 nm. 4-methylpyrrole-2,3-dicarboxylic acid and 4-methylpyrrole-2,3,5-tricarboxylic acid were by-products of the synthesis described above and were purified from the reaction mixture also by HPLC.

Synthesis of 3,5-dimethylpyrrole-2,4-dicarboxylic acid was carried out by base-catalysed hydrolysis of the diethyl ester (Aldrich) by heating in aqueous 4 M NaOH for 4 h and precipitation with HCl (36%). Acid catalysed hydrolysis of 3,5-dimethylpyrrole-2,4-dicarboxylic acid diethyl ester with 98% H₂SO₄ for 1 h at 30 °C led to selective

© FEBS 2003

Amide synthetase from the coumermycin A1 cluster (*Eur. J. Biochem.* 270) 4415

removal of the ester bond at position 4 and 3,5-dimethylpyrrole-2,4-dicarboxylic acid 2-ethyl ester was obtained after dilution with water (1 : 10), extraction with ethyl acetate and evaporation of the organic solvent.

### Cloning, expression and purification of the amide synthetase CouL

PCR was used to amplify *couL*, using pZW11 (a subclone of the cosmid 4-2H from the cosmid library of genomic DNA from *S. rishiriensis*) as template. By the introduction of an *SphI* site in primer couL-1 (5′-TAGTCACTGCGATG CCGAACAGGGAC-3′; *SphI* site shown in bold) the start codon was altered from GTG to ATG. At the 3′ terminus a *BgII* site was introduced and the stop codon was removed using primer couL-2 (5′-TGACTCAGATCTCCTGTC CACCAG-3′; *BgII* site shown in bold) for in-frame ligation to the (His)$_6$-tag of pQE70 (Qiagen). One-hundred microlitres of reaction mixture contained 130 ng pZW11 template DNA, 22 pmol of each primer, 0.2 mM each dNTP (Stratagene), *Pfu* DNA polymerase reaction buffer, 5% (v/v) dimethylsulfoxide, and 2.5 U of cloned *Pfu* DNA polymerase (Stratagene). Conditions for PCR were as follows: 1 cycle at 96 °C (3 min) followed by 27 cycles at 95 °C (90 s), 60 °C (45 s) and 72 °C (4 min). The PCR product was digested with *SphI* and *BglII* and ligated into the same sites of the expression vector pQE70. The resulting plasmid was named pMS90.

Expression and one-step purification of the recombinant protein by Ni–NTA acid affinity chromatography with Ni–NTA agarose resin (Qiagen) was carried out as described previously for the novobiocic acid synthetase [3].

### Protein analysis

Protein concentrations were determined by the Bradford method [8] using BSA as standard. SDS/PAGE was carried out according to the method of Laemmli [9], and protein bands were stained with Coomassie brilliant blue R-250. The native molecular mass of CouL was determined by gel filtration on a HiLoad 26/60 Superdex 200 column calibrated with a gel filtration calibration kit containing blue dextran 2000, aldolase, albumin, ovalbumin and ribonuclease (molecular masses 158 000, 67 000, 43 000 and 13 000, respectively), (all from Amersham Biosciences). The elution buffer contained 150 mm NaCl and 50 mm Tris/HCl pH 8.0, and the flow rate was 1 mL·min$^{-1}$.

### Enzyme activity assay

The amide synthetase assay mixture contained 1 mm 3-methylpyrrole-2,4-dicarboxylic acid, 1 mm ring B, 1 mm ATP, 5 mm MnCl$_2$, 50 mm Tris/HCl pH 8.0 and 4 μg purified protein in a total volume of 100 μL. Determination of the $K_{\text{m}}$ value of 3-methylpyrrole-2,4-dicarboxylic acid was carried out with 10 μg·mL$^{-1}$ purified protein and the $K_{\text{m}}$ of ring B was determined with 1 μg·mL$^{-1}$ purified protein in the assay. Incubation was carried out at 30 °C for 5–10 min. For product isolation the incubation time was extended up to 1 h. The reaction was stopped by addition of trichloroacetic acid to a final concentration of 75 mm. The mixture was extracted twice with 500 μL ethyl acetate, the pooled organic phases were evaporated and the residue was dissolved in 50% aqueous methanol (v/v) for HPLC analysis.

#### HPLC analysis of enzymatic products

Assay products were analysed by HPLC using a Multosphere RP18-5 column (250 × 4 mm, 5 μm; C + S Chromatographie Service, Düren, Germany) with a linear gradient from 15 to 100% acetonitrile in 0.1% trifluoroacetic acid or from 10 to 100% MeOH in 1% aqueous formic acid and detected at 340 nm. Detection of novobiocic acid was carried out at 305 nm as described previously [3].

#### ${}^{1}$H-NMR, HPLC-MS and selected reaction monitoring

${}^{1}$H-NMR spectra were measured on an AMX 400 spectrometer (Bruker, Karlsruhe, Germany), using CD$_3$OD as solvent. The monoamide of 3-methylpyrrole-2,4-dicarboxylic acid showed the following signals: $\delta_{\text{ppm}}$ (400 MHz) 7.68 (d, $J = 8.72$ Hz, H-6); 7.63 (s, H-5'); 6.84 (d, $J = 8.64$ Hz, H-5); 2.65 (s, 3H-7'); 2.26 (s, 3H-11).

The positive and negative electrospray ionization (ESI) mass spectra were obtained from a Finnigan MAT TSQ 7000 instrument (electrospray voltage, 4.5 kV; heated capillary temperature, 220 °C; sheath and auxiliary gas, nitrogen) coupled with a Micro-Tech Ultra-Plus Micro LC system equipped with an RP18 column (5 μm, 1 × 100 mm; SepServ). A gradient system ranging from H$_2$O:CH$_3$CN 80 : 20 (each of them contained 0.2% HOAc) to 10 : 90 over 15 min, followed by isocratic elution with a 10 : 90 mixture of both solvents for 25 min, was used; the flow rate was 70 μL·min$^{-1}$. Selected reaction monitoring (SRM) during a HPLC run were recorded with a collision energy of 20 eV for positively charged ions and 40 eV for negatively charged ions, respectively; collision gas, argon; collision pressure, 1.8 mtorr ($240 \times 10^{-3}$ Pa). Selected reaction monitoring of the products of the amide synthetase revealed the following reactions.

##### Products of 3-methylpyrrole-2,4-dicarboxylic acid

Monoamide, positive-ion reaction: $m/z$ 359 ([M + H]$^+$) → $m/z$ 152; negative-ion reaction: $m/z$ 357 ([M–H]$^-$) → $m/z$ 206; diamide, positive-ion reaction: $m/z$ 548 ([M + H]$^+$) → $m/z$ 341; negative-ion reaction $m/z$ 546 ([M–H]$^-$) → $m/z$ 206.

##### Products of 4-methylpyrrole-2,3-dicarboxylic acid

Monoamide, positive-ion reaction: $m/z$ 359 ([M + H]$^+$) → $m/z$ 152; negative-ion reaction: $m/z$ 357 ([M–H]$^-$) → $m/z$ 206; diamide, positive-ion reaction: $m/z$ 548 ([M + H]$^+$) → $m/z$ 341; negative-ion reaction $m/z$ 546 ([M–H]$^-$) → $m/z$ 206.

##### Products of 3,5-dimethylpyrrole-2,4-dicarboxylic acid

Monoamide, positive-ion reaction: $m/z$ 373 ([M + H]$^+$) → $m/z$ 166; negative-ion reaction: $m/z$ 371 ([M–H]$^-$) → $m/z$ 206; diamide, positive-ion reaction: $m/z$ 562 ([M + H]$^+$) → $m/z$ 355; negative-ion reaction $m/z$ 560 ([M–H]$^-$) → $m/z$ 206.

4416 E. Schmutz et al. (Eur. J. Biochem. 270)

© FEBS 2003

# Results

## Sequence analysis of CouL

In the coumermycin biosynthetic gene cluster, the gene encoding the predicted amide synthetase CouL is situated between the genes for biosynthesis of the aminocoumarin moiety (*couHIJK*) [4,10] and the glycosyl transferase gene *couM* [11]. Close homologues of *couL* are present in the biosynthetic gene clusters of novobiocin and clorobiocin, situated at the same relative positions within the clusters [12]. The gene products of the three genes show on average 84% identity to each other (Fig. 2). Sequence analysis of NovL from the novobiocin cluster revealed characteristic motifs of adenylate-forming enzymes (Box I; Box II; A8; A10) (Fig. 2) [3]. These motifs were also present in CouL. Notably, CouL contains only a single adenylation domain, just as NovL.

The sequence alignment of the three amide synthetases from the coumermycin A₁, clorobiocin and novobiocin clusters, i.e. CouL, CloL and NovL, shows the highest variability between Box I and Box II, especially in the region between position 192 and 216 of CouL (Fig. 2). It may be speculated that this section plays a role in the binding of the acyl substrate of the three amide synthetases. CouL comprises 529 amino acids (calculated mass 57 844 Da),

two amino acids more than NovL and CloL. The two additional amino acids are contained in the region between amino acids 192 and 216.

CouL also shows sequence similarities to putative fatty acid Coenzyme A ligases from *S.* coelicolor (34% identity), from *S.* avermitilis (30% identity) and other organisms in the database.

## Overexpression and purification of the recombinant CouL

For purification of CouL by metal affinity chromatography, the protein was expressed as a histidine fusion protein. A (His)₆-tag was fused to the C terminus of the protein by use of the pQE70 expression vector. After expression in *Escherichia coli* XL1 Blue MRF′ and induction with 0.5 mM isopropyl thio-β-D-galactoside, the formation of a protein of about 60 kDa (calculated mass 58.9 kDa) could be detected by SDS/PAGE (Fig. 3). Metal affinity chromatography with Ni–NTA resin yielded purified protein which constituted 8–9% of the total soluble protein in the crude extract, as determined by the Bradford method [8]. The protein was obtained in near homogeneity after this one-step purification procedure (Fig. 3).

## Amide synthetase activity

The purified protein was incubated with 3-methylpyrrole-2,4-dicarboxylic acid and ring B (1 mm each) in the presence of ATP (5 mM) and Mn²⁺ (5 mM). Analysis of the reaction products by HPLC showed two enzymatic products (Fig. 4). Analysis of the two reaction products by ¹H-NMR spectroscopy as well as by MS with selected reaction monitoring showed that the first peak at retention time (Rₜ) = 15 min was an amide of the 3-methylpyrrole-2,4-dicarboxylic acid with a single ring B moiety (Fig. 4). The second peak at Rₜ = 20 min was identified as the diamide of the pyrrole dicarboxylic acid and two ring B moieties. The ¹H-NMR spectrum of the first peak showed

![Figure 2](https://i.imgur.com/yourimageurl.png)

Fig. 2. Sequence alignment of the amide synthetases from the biosynthetic gene clusters of coumermycin A₁ (CouL), novobiocin (NovL) and clorobiocin (CloL). Conserved motifs are shaded in grey. The alignment was constructed using DNASIS for Windows, Version 2 (Hitachi, San Bruno) scoring with a gap penalty of 5.0, a K-tuple of 2.0, a fixed gap penalty of 10.0 and a floating gap penalty of 10.0. The number of top diagonals and window size were both set to 5.

![Figure 3](https://i.imgur.com/yourimageurl.png)

Fig. 3. Purification of the amide synthetase CouL after overexpression in *E. coli* XL1 Blue MRF′. SDS/PAGE was carried out as described in Materials and methods. Lane 1, molecular mass marker (MW); lane 2, total protein before induction; lane 3, total protein after induction with 0.5 mM isopropyl thio-β-D-galactoside for 3 h; lane 4, soluble protein after induction; lane 5, eluate from nickel affinity chromatography.

© FEBS 2003

Amide synthetase from the coumermycin A1 cluster (*Eur. J. Biochem.* 270) 4417

Fig. 4. (A) HPLC analysis of amide synthetase products with 3-methylpyrrole-2,4-dicarboxylic acid as acyl component and (B) fragmentation schemes for the monoamide and the diamide observed by selected reaction monitoring. The samples were analysed as described in Materials and methods; a, amide synthetase assay with purified CouL; b, negative control with heat-denatured enzyme.

the proton signals corresponding to the monoamide (see Materials and methods). SRM of the same compound showed the following reactions: $m/z$ 359 ([M + H]$^{+}$) → $m/z$ 152; and $m/z$ 357 ([M-H]$^{-}$) → $m/z$ 206 confirming the presence of the pyrrole dicarboxylic acid and of ring B in this structure (Fig. 4). However, these spectroscopic methods did not allow us to decide whether the 2-carboxylic group or the 4-carboxylic group of the pyrrole moiety is attached to the aminocoumarin ring, and in this regard the assignment of the monoamide structure is tentative. SRM of the second product showed the positive-ion reaction $m/z$ 548 ([M + H]$^{+}$) → $m/z$ 341 and the negative-ion reaction $m/z$ 546 ([M-H]$^{-}$) → $m/z$ 206, in accordance with the suggested structure (Fig. 4).

When the enzyme was incubated with a 10-fold excess of ring B (1 mM) over 3-methylpyrrole-2,4-dicarboxylic acid (0.1 mM) for a prolonged incubation time (1 h), a complete conversion of the monoamide to the diamide was detected by HPLC. Moreover, when the first reaction product, i.e. the monoamide, was isolated and incubated with ring B, ATP, Mn$^{2+}$ and CouL, the formation of the diamide was demonstrated by HPLC. The velocity of this second reaction was approximately 43% of that of the first reaction, i.e. of the monoamide formation. The results show that CouL alone is responsible for the formation of both amide bonds between the pyrrole dicarboxylic acid and two ring B moieties.

### Properties of recombinant CouL

The native molecular mass of purified CouL was investigated by gel filtration on a calibrated Superdex 200 column. The protein eluted with a volume corresponding to a

Fig. 5. Lineweaver–Burk plots for monoamide formation by CouL with 3-methylpyrrole-2,4-dicarboxylic acid (A) and ring B (B) as variable substrates. The apparent $K_{\mathrm{m}}$ values are given. Concentration of the constant substrate was 1 mM.

molecular mass of 59–60 kDa (calculated mass 58.9 kDa). The eluted protein showed the described amide synthetase activity, resulting in the formation of the monoamide and the diamide. Therefore, CouL is active as a monomer.

The product formation by the purified recombinant amide synthetase showed a linear dependence on protein concentration up to 50 μg protein per assay. The product formation was also linear with time for at least 10 min. The activity was strictly dependent on the presence of ATP and of divalent cations such as Mg$^{2+}$ or Mn$^{2+}$. Specificity for the nucleotide triphosphate was tested with ATP, GTP, CTP and TTP (5 mM each). Product formation was detected in the presence of ATP (100%) and to a lesser extent with GTP (∼5%). Addition of Coenzyme A (0.5 mM) did not increase the amide synthetase activity. This is in accordance with the mechanism of activation shown previously for NovL [3], i.e. acyl adenylate formation and acyl transfer to the amino group without involvement of a coenzyme A ester.

$K_{\mathrm{m}}$ values for the 3-methylpyrrole-2,4-dicarboxylic acid and ring B were determined by the Lineweaver–Burk method (Fig. 5). The apparent $K_{\mathrm{m}}$ values were 26 μM for 3-methylpyrrole-2,4-dicarboxylic acid and 44 μM for ring B. $V_{\max }$ (monoamide formation) was 7.4 nkat·(mg protein)$^{-1}$, corresponding to a turnover rate of 0.4 mol substrate·mol enzyme$^{-1}$·s$^{-1}$.

Substrate inhibition by the aminocoumarin substrate was observed for concentrations exceeding 100 μM.

### Substrate specificity

The substrate specificity of CouL was tested with several pyrrole carboxylic acids, pyridine carboxylic acids, and

with 3-dimethylallyl-4-hydroxybenzoic acid (i.e. the acyl moiety of novobiocin). The results are summarized in Table 1. The presence of a methyl group at position 5 of the 3-methylpyrrole-2,4-dicarboxylic acid substrate did not reduce the reaction velocity for the formation of the

Table 1. Substrate specificity of CouL of *S. rishiriensis* and NovL of *S. spheroides* after expression in *E. coli* XL1 Blue MRF'. ND, Not determined.

| Substrate                                                                 | Relative activity (%) |
|--------------------------------------------------------------------------|------------------------|
|                                                                            | CouL                  | NovL                  |
| HOOC                                                                        |                        |                        |
| CH₃                                                                         | 3-methylpyrrole-2,4-dicarboxylic acid | 100<sup>a</sup>       | < 0.2                 |
| N                                                                           |                        |                        |
| COOH                                                                        |                        |                        |
| COOH                                                                        | 3-dimethylallyl-4-hydroxy-benzoic acid | 36                   | 100                   |
| OH                                                                          |                        |                        |
| HOOC                                                                        |                        |                        |
| CH₃                                                                         | 3,5-dimethylpyrrole-2,4-dicarboxylic acid | 110<sup>a</sup>      | < 0.2                 |
| H₃C                                                                         |                        |                        |
| N                                                                           |                        |                        |
| COOH                                                                        |                        |                        |
| HOOC                                                                        |                        |                        |
| CH₃                                                                         | 4-methylpyrrole-2,3-dicarboxylic acid | 94<sup>a</sup>       | < 0.2                 |
| H                                                                           |                        |                        |
| HOOC                                                                        |                        |                        |
| H₅C₂OOC                                                                     |                        |                        |
| CH₃                                                                         | 3,5-dimethylpyrrole-2,4-dicarboxylic acid 4-ethyl ester | 7.7                  | ND                    |
| H₃C                                                                         |                        |                        |
| N                                                                           |                        |                        |
| COOH                                                                        |                        |                        |
| HOOC                                                                        |                        |                        |
| CH₃                                                                         | 3,5-dimethylpyrrole-2,4-dicarboxylic acid 2-ethyl ester | < 0.2                | ND                    |
| H₃C                                                                         |                        |                        |
| N                                                                           |                        |                        |
| COOC₂H₅                                                                     |                        |                        |
| HOOC                                                                        |                        |                        |
| CH₃                                                                         | 3-methylpyrrole-2,3,4-tricarboxylic acid | < 0.2                | ND                    |
| H                                                                           |                        |                        |
| COOH                                                                        |                        |                        |
| HOOC                                                                        | pyrrole-2-carboxylic acid | < 0.2                | ND                    |
| H                                                                           |                        |                        |
| N                                                                           |                        |                        |
| HOOC                                                                        | pyrrole-3-carboxylic acid | < 0.2                | ND                    |
| H                                                                           |                        |                        |
| H₃C                                                                         |                        |                        |
| O=C                                                                         | 4-acetyl-3,5-dimethyl-2-pyrrole carboxylic acid | < 0.2                | ND                    |
| H₃C                                                                         |                        |                        |
| N                                                                           |                        |                        |
| COOH                                                                        |                        |                        |
| HOOC                                                                        |                        |                        |
| CH₃                                                                         | 3-carboxy-1,4-dimethyl-2-pyrrole acetic acid | < 0.2                | ND                    |
| H₂C                                                                         |                        |                        |
| N                                                                           |                        |                        |
| CH₃                                                                         |                        |                        |
| HOOC                                                                        | pyridine-2,6-dicarboxylic acid | < 0.05               | ND                    |
| COOH                                                                        |                        |                        |
| HOOC                                                                        | 2,6-dimethylpyridine-3,5-dicarboxylic acid | < 0.05               | ND                    |
| COOH                                                                        |                        |                        |
| H₃C                                                                         |                        |                        |
| N                                                                           |                        |                        |
| CH₃                                                                         |                        |                        |
| HOOC                                                                        | 6-methylpyridine-2,3-dicarboxylic acid | < 0.05               | ND                    |
| COOH                                                                        |                        |                        |
| HOOC                                                                        | 4-hydroxybenzoic acid | 20                    | 65                    |
| OH                                                                          |                        |                        |
| HOOC                                                                        | 3,5-dibromo-4-hydroxybenzoic acid | 50                   | < 0.05                |
| COOH                                                                        |                        |                        |
| Br                                                                          |                        |                        |
| OH                                                                          |                        |                        |
| HOOC                                                                        | 4-aminobenzoic acid | < 0.05               | < 0.05                |
| NH₂                                                                         |                        |                        |

a Values for monoamide formation.

© FEBS 2003

Amide synthetase from the coumermycin A1 cluster (*Eur. J. Biochem.* 270) 4419

zoic acid which is the genuine substrate of NovL. In contrast, NovL does not accept any of the pyrrole dicarboxylic acids. Recently, we presented evidence that the biosynthetic gene cluster of coumermycin may have evolved from an ancestral cluster similar to those of novobiocin and clorobiocin [13]. The fact that CouL accepts the genuine substrate of NovL (but not vice versa) may reflect the evolution of *couL* from an ancestral gene very similar to *novL*.

Sequence comparison of the three amide synthetases CouL, NovL and CloL involved in the biosynthesis of the aminocoumarin antibiotics coumermycin, novobiocin and clorobiocin, respectively, shows a sequence of about 25 amino acids with high variability between the proteins. This sequence is situated in the region between Box I and Box II (Fig. 2), which is responsible for the substrate specificity in other adenylate forming enzymes, e.g. 4-coumarate-Coenzyme A ligases [14,15] and NRPS [16,17]. Site-directed mutagenesis experiments may provide additional information about the involvement of this region in the substrate specificity of these amide synthetases, and may open a route to the formation of new aminocoumarin compounds by combinatorial biosynthesis and mutasynthesis experiments.

### Acknowledgements

We thank U. Galm for help in enzyme assays and C. Kuhnt (Halle, Germany) for assistance in the LC-MS analysis. This work was supported by a grant from the Deutsche Forschungsgemeinschaft (to L. Heide and S.-M. Li).

### References

1. Lewis, R.J., Singh, O.M.P., Smith, C.V., Skarzynski, T., Maxwell, A., Wonacott, A.J. & Wigley, D.B. (1996) The nature of inhibition of DNA gyrase by the coumarins and the cyclothialidines revealed by X-ray crystallography. *EMBO J.* **15**, 1412–1420.
2. Tsai, F.T.F., Singh, O.M., Skarzynski, T., Wonacott, A.J., Weston, S., Tucker, A., Paupit, R.A., Breeze, A.L., Poyser, J.P., O'Brien, R., Ladbury, J.E. & Wigley, D.B. (1997) The high-resolution crystal structure of a 24-kDa gyrase B fragment from *E. coli* complexed with one of the most potent coumarin inhibitors, clorobiocin. *Proteins* **28**, 41–52.
3. Steffensky, M., Li, S.-M. & Heide, L. (2000) Cloning, overexpression, and purification of novobiocin acid synthetase from *Streptomyces spheroides* NCIMB 11891. *J. Biol. Chem.* **275**, 21754–21760.
4. Wang, Z.-X., Li, S.-M. & Heide, L. (2000) Identification of the coumermycin A₁ biosynthetic gene cluster of *Streptomyces rishiriensis* DSM 40489. *Antimicrob. Agents Chemother.* **44**, 3040–3048.
5. Kominek, L.A. & Meyer, H.F. (1975) Novobiocin acid synthetase. *Methods Enzymol.* **43**, 502–508.
6. Corwin, A.H., Bailey, W. & Viohl, P. (1942) Pyrrole series. VIII. Structural investigations upon a substituted dipyrrolylmethane. An unusual melting point-symmetry relationship. *J. Am. Chem. Soc.* **64**, 1267–1273.
7. Woodward, R.B., Ayer, W.A., Beaton, J.M., Bickelhaupt, F., Bonnett, R., Buchschacher, P., Closs, G.L., Dutler, H. & Hannah, J. (1990) The total synthesis of chlorophyll a. *Tetrahedron* **46**, 7599–7659.
8. Bradford, M.M. (1976) A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. *Anal. Biochem.* **72**, 248–254.
9. Laemmli, U.K. (1970) Cleavage of structural proteins during the assembly of the head of bacteriophage T4. *Nature* **227**, 680–685.
10. Chen, H. & Walsh, C.T. (2001) Coumarin formation in novobiocin biosynthesis: β-hydroxylation of the aminoacyl enzyme tyrosyl-S-NovH by a cytochrome P450 NovI. *Chem. Biol.* **8**, 301–312.
11. Meyers, C.L.F., Oberthur, M., Anderson, J.W., Kahne, D. & Walsh, C.T. (2003) Initial characterization of novobiocin acid noviosyl transferase activity of NovM in biosynthesis of the antibiotic novobiocin. *Biochemistry* **42**, 4179–4189.
12. Pojer, F., Li, S.-M. & Heide, L. (2002) Molecular cloning and sequence analysis of the clorobiocin biosynthetic gene cluster: new insights into the biosynthesis of aminocoumarin antibiotics. *Microbiology* **148**, 3901–3911.
13. Eustáquio, A.S., Luft, T., Wang, Z.X., Gust, B., Chater, K.F., Li, S.-M. & Heide, L. (2003) Novobiocin biosynthesis: inactivation of the putative regulatory gene *novE* and heterologous expression of genes involved in aminocoumarin ring formation. *Arch. Microbiol.* **180**, 25–32.
14. Stuible, H.P. & Kombrink, E. (2001) Identification of the substrate specificity-conferring amino acid residues of 4-coumarate: coenzyme A ligase allows the rational design of mutant enzymes with new catalytic properties. *J. Biol. Chem.* **276**, 26893–26897.
15. Schneider, K., Hovel, K., Witzel, K., Hamberger, B., Schomburg, D., Kombrink, E. & Stuible, H.P. (2003) The substrate specificity-determining amino acid code of 4-coumarate: CoA ligase. *Proc. Natl Acad. Sci. USA* **100**, 8601–8606.
16. Stachelhaus, T., Mootz, H.D. & Marahiel, M.A. (1999) The specificity-conferring code of adenylation domains in nonribosomal peptide synthetases. *Chem. Biol.* **6**, 493–505.
17. Challis, G.L., Ravel, J. & Townsend, C.A. (2000) Predictive, structure-based model of amino acid recognition by nonribosomal peptide synthetase adenylation domains. *Chem. Biol.* **7**, 211–224.
