
Microbiology (2003), 149, 3461-3471

Pathways for phosphatidylcholine biosynthesis in bacteria

Fernando Martínez-Morales,$^{1}$ Max Schobert,$^{2}$ Isabel M. López-Lara$^{1}$ and Otto Geiger$^{1}$

$^{1}$Centro de Investigación sobre Fijación de Nitrógeno, Universidad Nacional Autónoma de México, Apdo Postal 565-A, Cuernavaca, Morelos, CP 62210, Mexico

$^{2}$Technische Universität Braunschweig, Institut für Mikrobiologie, Spielmannstrasse 7, 38106 Braunschweig, Germany

Phosphatidylcholine (PC) is the major membrane-forming phospholipid in eukaryotes with important structural and signalling functions. Although many prokaryotes lack PC, it can be found in significant amounts in membranes of rather diverse bacteria. Two pathways for PC biosynthesis are known in bacteria, the methylation pathway and the phosphatidylcholine synthase (PCS) pathway. In the methylation pathway, phosphatidylethanolamine is methylated three times to yield PC, in reactions catalysed by one or several phospholipid N-methyltransferases (PMTs). In the PCS pathway, choline is condensed directly with CDP-diacylglyceride to form PC in a reaction catalysed by PCS. Using cell-free extracts, it was demonstrated that Sinorhizobium meliloti, Agrobacterium tumefaciens, Rhizobium leguminosarum, Bradyrhizobium japonicum, Mesorhizobium loti and Legionella pneumophila have both PMT and PCS activities. In addition, Rhodobacter sphaeroides has PMT activity and Brucella melitensis, Pseudomonas aeruginosa and Borrelia burgdorferi have PCS activities. Genes from *M. loti* and *L. pneumophila* encoding a *Pmt* or a *Pcs* activity and the genes from *P. aeruginosa* and *Borrelia burgdorferi* responsible for *Pcs* activity have been identified. Based on these functional assignments and on genomic data, one might predict that if bacteria contain PC as a membrane lipid, they usually possess both bacterial pathways for PC biosynthesis. However, important pathogens such as *Brucella melitensis*, *P. aeruginosa* and *Borrelia burgdorferi* seem to be exceptional as they possess only the PCS pathway for PC formation.

---

**INTRODUCTION**

Phosphatidylcholine (PC) or lecithin is the major phospholipid in eukaryotic cells. In addition to being the major structural component of cellular membranes, PC plays an important role in signal transduction as it is a major source of lipid second messengers (Exton, 1994). In eukaryotic organisms, PC can be synthesized by two alternative biosynthetic pathways, the CDP-choline pathway or the methylation pathway (Kent, 1995). In the CDP-choline pathway, also known as the Kennedy pathway, free choline is converted to PC via the intermediates choline phosphate and CDP-choline through the sequential actions of choline kinase, CTP:phosphocholine cytidylyltransferase and CDP-choline:1,2-diacylglycerol cholinephosphotransferase. In the methylation pathway, PC is formed by three successive methylations of phosphatidylethanolamine (PE)

via the intermediates monomethylphosphatidylethanolamine (MMPE) and dimethylphosphatidylethanolamine (DMPE) using the methyl donor S-adenosylmethionine and the enzyme phospholipid N-methyltransferase (*Pmt*) (Fig. 1). Although in mammals the three methylation activities are due to a single gene, in yeast, two different genes encoding class I and class II PMTs are involved (Sohlenkamp et al., 2003). Class II PMTs (PEM1/CHO2 in *Saccharomyces cerevisiae*) catalyse the first methylation step from PE to MMPE, whereas class I PMTs (PEM2/OPI3 in *S. cerevisiae*) catalyse the second and third methylation steps from MMPE via DMPE to PC.

The Gram-negative model bacterium *Escherichia coli* or the Gram-positive model bacterium *Bacillus subtilis* both contain only PE, phosphatidylglycerol and cardiolipin as major membrane phospholipids and therefore it was thought that this holds true for most bacteria. However, a recent estimate suggests that probably more than 10% of all bacteria contain PC as a membrane phospholipid (Sohlenkamp et al., 2003). Only the methylation pathway of PC biosynthesis was thought to occur in prokaryotes

Abbreviations: DMPE, dimethylphosphatidylethanolamine; MMPE, monomethylphosphatidylethanolamine; PC, phosphatidylcholine; Pcs/PCS, phosphatidylcholine synthase; PE, phosphatidylethanolamine; Pmt/PMT, phospholipid N-methyltransferase.

(Rock *et al.*, 1996); however, a second pathway for PC biosynthesis exists, the phosphatidylcholine synthase (PCS) pathway (Fig. 1), which is distinct from the CDP-choline pathway and seems to be exclusive for bacteria (de Rudder *et al.*, 1997, 1999; Sohlenkamp *et al.*, 2000). In this second pathway, choline is condensed directly with CDP-diacylglycerol to form PC and CMP in a reaction catalysed by PCS. The existence of a PCS pathway has been proposed for a number of bacteria (Sohlenkamp *et al.*, 2000, 2003; López-Lara & Geiger, 2001; López-Lara *et al.*, 2003) and now there is evidence that this pathway exists in *Pseudomonas aeruginosa* (Wilderman *et al.*, 2002) and at least indirect evidence that it occurs in *Agrobacterium* (Karnezis *et al.*, 2002).

Bacterial genes encoding Pmt proteins are quite dissimilar and the two families presently known, with members resembling either the *Rhodobacter* PmtA (Arondel *et al.*, 1993) or the *Sinorhizobium* PmtA (de Rudder *et al.*, 2000), are more similar to methyltransferases with other substrate specificities than to each other (López-Lara & Geiger, 2001; Sohlenkamp *et al.*, 2003). Recently, the gene encoding Pmt in *Acetobacter aceti* has been identified (Hanada *et al.*, 2001) and its product belongs to the rhodobacterial PmtA family. A Pmt protein (PmtA) from *Bradyrhizobium japonicum* resembles the sinorhizobial PmtA but surprisingly the bradyrhizobial PmtA shows a peculiar substrate specificity as it seems to perform only the first methylation step efficiently, converting PE to MMPE (Minder *et al.*, 2001).

In this study, we investigated the PMT and PCS pathways of PC biosynthesis in different bacteria. We identified various genes encoding Pmt or Pcs. Based on our studies, we suggest that if bacteria are able to form PC, they often can do so via either of the pathways.

## METHODS

### Bacterial strains, plasmids and growth conditions
The bacterial strains and plasmids used in this study and their relevant characteristics are shown in Table 1. *Escherichia coli* (37°C), *Agrobacterium tumefaciens* (29°C), *Brucella melitensis* (37°C), *P. aeruginosa* (37°C) and *Burkholderia* strains (29°C) were grown on Luria–Bertani (LB) medium (Miller, 1972) at the growth temperatures indicated. *Sinorhizobium* (*Rhizobium*) *meliloti* strains were grown on LB/MC medium (Glazebrook & Walker, 1991), *Mesorhizobium loti* was grown on YEM medium (Vincent, 1970), *Rhodobacter sphaeroides* was grown on Sistrom’s medium (Sistrom, 1962) and *Bradyrhizobium japonicum* was grown on PSY complex medium supplemented with 0·1% (w/v) L-arabinose (Regensburger & Hennecke, 1983), all at 29°C. *Legionella pneumophila* was grown on AYE medium at 37°C (Feeley *et al.*, 1979) and *Borrelia burgdorferi* was grown on BSK-H medium (Barbour–Stoenner–Kelley; Sigma) at 34°C.

Antibiotics were added to media at the following concentrations (µg ml⁻¹) when required: gentamicin 150, in the case of *P. aeruginosa*; carbenicillin 100, tetracycline 10, gentamicin 20, kanamycin 50, in the case of *E. coli*.

### Inactivation of the *P. aeruginosa pcs* gene and a *pmtA*-homologous ORF
For inactivation of the *pcs* gene (PA3857) and of a *pmtA* homologue (PA0798), both potentially involved in PC

Table 1. Bacterial strains and plasmids used in this study

| Strain or plasmid | Relevant characteristic(s) | Source or reference |
|-------------------|----------------------------|---------------------|
| Strain           |                            |                     |
| Escherichia coli DH5α | recA1, φ80lacZΔM15 | Hanahan (1983)      |
| Escherichia coli BL21(DE3) | Expression strain | Studier et al. (1990) |
| Escherichia coli S17-1 | Sm<sup>r</sup>Sp<sup>r</sup>hdsR RP4-2kan::Tn7tet::Mu, integrated in the chromosome | Simon et al. (1983) |
| Agrobacterium tumefaciens C58 | Wild-type, biovar I, ATCC 33970 | Hamilton & Fall (1971) |
| Borrelia burgdorferi B31<sup>T</sup> | Isolated from a tick | ATCC 35210<sup>T</sup> |
| Bradyrhizobium japonicum 110spc4 | Wild-type | Regensburger & Hennecke (1983) |
| Brucella melitensis 16M<sup>T</sup> | Biotype 1, isolated from an infected goat | ATCC 23456<sup>T</sup> |
| Burkholderia cepacia | Isolated from bronchial washings | ATCC 25609 |
| Burkholderia cepacia | Isolated from soil | ATCC 29352 |
| Burkholderia caryophylli | Isolated from a carnation | ATCC 25418 |
| Burkholderia tropicalis Mto 293 | Isolated from stem interior of maize | Estrada-de los Santos et al. (2001) |
| Klebsiella pneumoniae | Antigenic type 3 | ATCC 13883<sup>T</sup> |
| Legionella pneumophila Philadelphia-1 | Serogroup 1 | Berger & Isberg (1993) |
| Mesorhizobium loti MAFF 303099 |                          | Kaneko et al. (2000) |
| Pseudomonas aeruginosa PAO1 | Wild-type | BAA-47 |
| Pseudomonas aeruginosa PAOPCS | PAO1 derivative, pcs(PA3857)-Gm-GFP | This study |
| Pseudomonas aeruginosa PAOPMT | PAO1 derivative, pmtA homologue (PA0798)-Gm-GFP | This study |
| Rhizobium leguminosarum RBL5560 | LPR5045 harbouring Sym plasmid pJB5JI | Spaink et al. (1987) |
| Rhodobacter sphaeroides ATH 2.4.1<sup>T</sup> | Wild-type | ATCC 17023<sup>T</sup> |
| Sinorhizobium meliloti 1021 | SU47str-21 | Meade et al. (1982) |
| Plasmid          |                            |                     |
| pUC19            | Cloning vector, carbenicillin-resistant | Yanisch-Perron et al. (1985) |
| pET9a            | Expression vector, kanamycin-resistant | Studier et al. (1990) |
| pLysS            | Production of lysozyme for repression of T7 polymerase, chloramphenicol-resistant | Studier et al. (1990) |
| pEX18Tc          | Gene replacement vector, tetracycline-resistant | Hoang et al. (1998) |
| pPS858           | aacC1-GFP, ampicillin-and gentamicin-resistant | Hoang et al. (1998) |
| pMSB05           | Pseudomonal pcs-containing Taq-amplified 1kb SphI-EcoRI insert in pEX18Tc | This study |
| pMSB07           | pcs(PA3857)::aacC1-GFP in pEX18Tc | This study |
| pMSB06           | Pseudomonal pmtA homologue containing Taq-amplified 0·8kb SphI-EcoRI insert in pEX18Tc | This study |

| Strain or plasmid | Relevant characteristic(s) | Source or reference |
|-------------------|--------------------------|---------------------|
| pMSB 08          | pmtA homologue (PA0798) ::aacCl-GFP in pEX18Tc | This study |
| PTB2084          | pmtA of S. meliloti as Ndel-BamHI insert in pET3a | de Rudder et al. (2000) |
| PTB2559          | pcs of B. burgdorferi as Ndel-BamHI insert in pET9a | Sohlenkamp et al. (2000) |
| PTB2902          | pcs of P. aeruginosa as Ndel-BamHI insert in pET9a | This study |
| PFM3             | pcs of L. pneumophila as Ndel-BamHI insert in pET9a | This study |
| PFM14            | pmtA of L. pneumophila as Ndel-BamHI insert in pET9a | This study |
| PFM15            | pcs of M. loti as Ndel-BamHI insert in pET9a | This study |
| PFM16            | Larger version of pmtA-like ORF (PA0798) from P. aeruginosa (990 bp) as Ndel-BamHI insert in pET9a | This study |
| PFM17            | pmtA-like ORF (PA0798) from P. aeruginosa (648 bp) as Ndel-BamHI insert in pET9a | This study |
| PFM18            | pmtA-like ORF (mlr5374) of M. loti as Ndel-BamHI insert in pET9a | This study |
| PFM20            | pmtA (mlr4753) of M. loti as Ndel-BamHI insert in pET9a | This study |

biosynthesis in *P. aeruginosa*, the respective genes were mutated by insertion of a gentamicin-resistance cassette following the strategy described by Hoang *et al.* (1998). Using PCR, specific oligonucleotides (CAGTGCA TGC CAA CTATGGCGG AGATCTT and GCC GGA ATT CGGTTCCTT GCG ATAG) and *Taq* polymerase, a 1 kb fragment was amplified from genomic DNA containing the pseudomonal *pcs* gene. Suitable restriction sites (underlined) for cloning the fragment were introduced by PCR with the oligonucleotides. After restriction with *SphI* and *EcoRI* the PCR-amplified DNA fragment was cloned into the suicide vector pEX18Tc, to obtain plasmid pMSB 05. For inactivation of the *pcs* gene, pMSB 05 was cut with *SalI* and the *SalI*-restricted gentamicin-resistance cassette of pPS 858 was inserted into the *pcs* gene giving rise to pMSB 07. Similarly, using the oligonucleotides CAGTGCA TGC CCT GTG TCC GGG TCG AT and GCC GGA ATT CAG CTG AC GAG GCC GATA, a 0·8 kb fragment was amplified from genomic DNA containing a pseudomonal gene homologous to *pmtA* from *Rhodobacter sphaeroides*. After restriction with *SphI* and *EcoRI*, the fragment was cloned into pEX18Tc, to obtain plasmid pMSB 06. For inactivation of the *pmtA*-homologous gene, pMSB 06 was cut with *XhoI* and the *SalI*-restricted gentamicin-resistance cassette of pPS 858 was inserted into the *pmtA* homologue giving rise to pMSB 08. The suicide vectors were transferred into *P. aeruginosa* PAO1 by diparental mating using *E. coli* S17-1 as donor. Gentamicin-resistant *P. aeruginosa* strains were isolated on minimal medium M9 (Miller, 1972) containing succinate (40 mM) and gentamicin and streaked onto LB plates containing 0·5% sucrose and gentamicin to score for loss of the pEX18Tc suicide vector, which contains a *sacB* gene. Gentamicin-resistant colonies able to grow in the presence of sucrose were purified. The absence of the wild-type genes and the presence of the disrupted mutant genes were verified by PCR. The *pcs*-deficient *P. aeruginosa* strain was named PAOPCS and that deficient in the *pmtA* homologue was designated PAOPMT.

DNA manipulations. Recombinant DNA techniques were performed according to standard protocols (Sambrook *et al.*, 1989). DNA was sequenced by the chain termination method (Sanger *et al.*, 1977) using pET9a derivatives. The DNA region sequenced and the deduced proteins were analysed using the NCBI (National Center for Biotechnology Information; http://www.ncbi.nlm.nih.gov/) BLAST network server (Altschul *et al.*, 1997).

In vivo labelling of bacterial strains with [1-¹⁴C] acetate or [methyl-¹⁴C] methionine and analysis of lipid extracts. The lipid compositions of different bacterial strains were determined following labelling with [1-¹⁴C] acetate. The incorporation of methyl groups into lipids was evaluated following labelling with [methyl-¹⁴C] methionine. Cultures (1 ml) of bacterial strains in the corresponding medium were inoculated from pre-cultures grown in the same medium. After addition of 0·4 μCi [1-¹⁴C] acetate (60 mCi mmol⁻¹; 2·22 GBq mmol⁻¹) or 1 μCi [methyl-¹⁴C] methionine (55 mCi mmol⁻¹; 2·03 GBq mmol⁻¹) to each culture, the cultures were incubated for either 4 or 16 h. Cells were harvested by centrifugation, washed with 500 μl of water and resuspended in 100 μl of water. The lipids were extracted according to Bligh & Dyer (1959). The chloroform phase was used for lipid analysis on TLC plates and, after one-dimensional separation (de Rudder *et al.*, 1997), visualization of labelled compounds was obtained by autoradiography.

Preparation of cell-free crude extracts. Cell-free crude extracts were made from cells obtained from 0·5 l of exponentially growing cultures. After harvesting by low-speed centrifugation at 4°C, each cell pellet was resuspended in 20 mM Tris/HCl buffer, pH 8·5. The cell suspension was passed twice through a French pressure cell at 20 000 p.s.i. (138 MPa). Unbroken cells and cell debris were removed by centrifugation at 7000 g for 20 min to obtain the cell-free extract. Protein concentrations were determined by the method of Dulley & Grieve (1975).

Determination of PMT activity. This was done essentially as described previously (de Rudder *et al.*, 1997).

Determination of PCS activity. To detect minor PCS activities, the assay optimized originally for the enzyme from *S. meliloti* (Sohlenkamp *et al.*, 2000) was modified in a way that higher concentrations of both substrates were used. The assay to determine PCS activity contained, in a total volume of 50 μl in Eppendorf tubes, 50 μg protein, 50 mM Tris/HCl, pH 8·0, 10 mM MnCl₂, 343 μM CDP-diacylglycerol, 0·2% (w/v) Triton X-100 and 100 μM [methyl-¹⁴C] choline (55 mCi mmol⁻¹). In some cases, MnCl₂ was replaced by equimolar concentrations of CoCl₂. The mixtures were incubated for 15 min in a 30°C water bath and stopped by mixing with 188 μl of methanol/chloroform (2:1, v/v). Addition of 63 μl chloroform and 63 μl water led to phase separation; after washing the chloroform phase once with another 100 μl of water, it was dried, re-dissolved in 10 μl of methanol/chloroform (1:1, v/v) and subjected to one-dimensional TLC.

Cloning and expression of ORFs potentially encoding proteins with Pmt or Pcs activities. Using specific oligonucleotides (Table 2), suspected *pmtA* or *pcs* genes were amplified from genomic DNA with *Pfu* polymerase and, after restriction with NdeI and BamHI, were cloned into pET9a (Studier *et al.*, 1990), enabling their overexpression under control of the T7 promoter. Correct in-frame cloning and the correct sequence were demonstrated by DNA sequencing (data not shown). *E. coli* BL21(DE3)(pLysS) (Studier *et al.*, 1990), which expresses the T7 polymerase under the control of the lac promoter, was transformed with the respective expression plasmids. At a cell density of 5 × 10⁸ cells ml⁻¹, IPTG was added to a final concentration of 0·1 mM. After 4 h induction, cells were harvested, resuspended in 20 mM Tris/HCl, pH 8·0, and stored at –20°C. Cells were lysed by thawing and two additional freeze-thaw cycles. Lysates were treated for 20 min with DNase I (Roche) from bovine pancreas (15 units ml⁻¹) and subsequently centrifuged at 7000 g for 20 min at 4°C; the supernatants were used as cell-free extracts.

---

**RESULTS**

**PMT activities in cell-free bacterial extracts**

PMT activities were determined in cell-free extracts obtained from different bacteria and the potential products were separated by TLC (Fig. 2). None of the potential products (MMPE, DMPE, PC) usually formed by Pmt was formed by cell-free extracts of *E. coli* (lane 3), *Brucella melitensis* (lane 8) and *Klebsiella pneumoniae* (lane 13) or by *P. aeruginosa* and its two mutant derivatives (lanes 14, 15, 16). In *Burkholderia cepacia* extracts (lane 11), methyl transfer to the lipid fraction and maybe to MMPE does occur; however, labelled DMPE or PC are not formed in this case. Instead, in *Burkholderia cepacia*, major label incorporation had occurred into faster-migrating, unidentified lipids. All three products of PMT activity are clearly formed by extracts of *S. meliloti* (lane 4), *A. tumefaciens* (lane 5), *Rhizobium leguminosarum* (lane 6), *Bradyrhizobium japonicum* (lane 7), *M. loti* (lane 9), *Rhodobacter sphaeroides* (lane 10) and *L. pneumophila* (lane 12).

**Methylation of lipids in Burkholderia strains**

Using a standard cell-free assay for Pmt, *Burkholderia cepacia* ATCC 29352 failed to form DMPE and PC (Fig. 2, lane 11 and Fig. 3, lane 6). We therefore investigated whether multiple methylation of PE could occur *in vivo* by labelling cell suspensions of *S. meliloti* and different *Burkholderia* strains with [methyl-¹⁴C] methionine. In *S. meliloti*, four lipid compounds were formed, three that migrated at the positions expected for MMPE, DMPE or

---

**Table 2. PCR primers used in this study**

The forward primers incorporated an NdeI restriction site (underlined) overlapping the start codon of the respective genes. The reverse primers encoded a BamHI restriction site (underlined) after the stop codon. Names are explained in legends of Figs 7 and 8.

| Primer name                     | Sequence (5′ → 3′)                                                                 |
|---------------------------------|-----------------------------------------------------------------------------------|
| Lpne Pcs forward                | TCTAGGATAACATATGAATCCAATAAAGCC                                                    |
| Lpne Pcs reverse                | AAATTACAGGATCCCTCAATCTTTATTAT                                                     |
| Lpne PmtA forward               | AGGATAACATATGTCGCTCTCTTCAGTA                                                      |
| Lpne PmtA reverse               | ATTACAGGATCCTCAATTATTCTTACAGTG                                                    |
| Paer Pcs forward                | AGGAATCATATGACTCCCATCAACAAAGC                                                     |
| Paer Pcs reverse                | AAAGGATCCTAGCCGAACCACTCGCG                                                        |
| Paer PmtA-like ORF long version forward | AGGATAACATATGGGCACCCG                                                          |
| Paer PmtA-like ORF short version forward | AGGATAACATATGCTTCCCCCTCCAG                                                     |
| Paer PmtA-like ORF reverse      | ATTACAGGATCAGCGCGCGAC                                                             |
| Bbur Pcs forward                | AGGAATCATATGAAAAATATCAATTTGATTTTAGC                                                |
| Bbur Pcs reverse                | AAAGGATCCTCATCTTTCTTGTTTTATAAG                                                    |
| Mlot Pcs forward                | AGGATAACATATGGGCGCACGGAAA                                                         |
| Mlot Pcs reverse                | TAAAGTGGATCCTCAGGCCTTTTGG                                                         |
| Mlot PmtA-like ORF forward      | AGGATAACATATGCCTGCACATGATGCC                                                      |
| Mlot PmtA-like ORF reverse      | ATTACAGGATCACAAGCGCCAGTC                                                          |
| Mlot PmtA forward               | AGGATAACATAGACGTGGTCACGGA                                                         |
| Mlot PmtA reverse               | ATTACAGGATCCTTAATGCGGTTTCGGCGGCG                                                  |

http://mic.sgmjournals.org

F. Martinez-Morales and others

Front —
MMPE —
DMPE —
PC —
Start —
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16

Fig. 2. PMT activities in different bacterial cell-free extracts. PMT assays were performed with different bacterial cell-free extracts; potential lipid products were extracted and separated by one-dimensional TLC. Total lipids of E. coli (lane 1) and S. meliloti (lane 2) obtained after labelling cells with [1-¹⁴C]acetate. Products of potential lipid methyltransferase activities obtained by labelling with S-adenosyl-L-[methyl-¹⁴C]-methionine with cell-free extracts of E. coli (lane 3), S. meliloti (lane 4), A. tumefaciens (lane 5), Rhizobium leguminosarum (lane 6), Bradyrhizobium japonicum (lane 7), Brucella melitensis (lane 8), M. loti (lane 9), Rhodobacter sphaeroides (lane 10), Burkholderia cepacia ATCC 29352 (lane 11), L. pneumophila (lane 12), K. pneumoniae (lane 13), P. aeruginosa PAO1 (lane 14), P. aeruginosa PAOPCS (lane 15) and P. aeruginosa PAOPMT (lane 16). The lipids PC, DMPE and MMPE are indicated.

Front —
MMPE —
DMPE —
PC —
Start —
1 2 3 4 5 6

Fig. 3. Methylation of lipids in Burkholderia. Lipid analysis of S. meliloti (lane 1), Burkholderia cepacia ATCC 29352 (lane 2), Burkholderia cepacia ATCC 25602 (lane 3), Burkholderia tropicalis (lane 4) and Burkholderia caryophylli (lane 5) after in vivo labelling with [methyl-¹⁴C]methionine. Products of potential lipid methyltransferase activities caused by the cell-free extract of Burkholderia cepacia ATCC 29352 (lane 6) were applied as well. Lipid fractions were separated by one-dimensional TLC. The lipids PC, DMPE and MMPE are indicated.

significantly in their ability to form methylated derivatives of PE. Similar in vivo studies performed with P. aeruginosa PAO1 also failed to detect MMPE, DMPE or PC (data not shown), strongly suggesting that no methylation pathway for PC biosynthesis exists in P. aeruginosa PAO1.

### PCS activities in cell-free bacterial extracts

PCS activities were determined in the presence of MnCl₂ in cell-free extracts obtained from different bacteria and the products were identified by TLC (Fig. 4). Cell-free extracts of E. coli (lane 3), Rhodobacter sphaeroides (lane 10), Burkholderia cepacia (lane 11) or K. pneumoniae (lane 13) did not form any PC and therefore were devoid of PCS activity. A reduced amount of PC was formed by Bradyrhizobium japonicum extracts (lane 7). PC was formed by extracts of S. meliloti (lane 4), A. tumefaciens (lane 5), Rhizobium leguminosarum (lane 6), Brucella melitensis (lane 8), M. loti (lane 9), L. pneumophila (lane 12), Borrelia burgdorferi (data not shown) and P. aeruginosa PAO1 (lane 14). The pcs mutant of P. aeruginosa (PAOPCS) was devoid of PCS activity (lane 15), whereas the putative pmtA mutant PAOPMT retained PCS activity (lane 16). The PCS activity was also determined in the presence of CoCl₂ instead of MnCl₂ since Co²⁺ has been reported to fulfil the cation requirement of some CDP-alcohol phosphotidyltransferases (Taniguchi et al., 1986). Under these conditions, the Pcs from L. pneumophila displayed the same level of activity as in the presence of MnCl₂ (data not shown), whereas the Pcs from S. meliloti, A. tumefaciens, Rhizobium leguminosarum or Brucella melitensis had

Front —
MMPE—
DMPE—
PC—
Start—
1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16

Fig. 4. PCS activities in different bacterial cell-free extracts. PCS assays were performed with different bacterial cell-free extracts; potential lipid products were extracted and separated by one-dimensional TLC. Total lipids of *E. coli* (lane 1) and *S. meliloti* (lane 2) as obtained after labelling cells with [1-¹⁴C]acetate. Products of potential PCS activities obtained by labelling with [methyl-¹⁴C]choline with cell-free extracts of *E. coli* (lane 3), *S. meliloti* (lane 4), *A. tumefaciens* (lane 5), *Rhizobium leguminosarum* (lane 6), *Bradyrhizobium japonicum* (lane 7), *Brucella melitensis* (lane 8), *M. loti* (lane 9), *Rhodobacter sphaeroides* (lane 10), *Burkholderia cepacia* ATCC 29352 (lane 11), *L. pneumophila* (lane 12), *K. pneumoniae* (lane 13), *P. aeruginosa* PAO1 (lane 14), *P. aeruginosa* PAOPCS (lane 15) and *P. aeruginosa* PAOPMT (lane 16). The lipids PC, DMPE and MMPE are indicated.

Front —
MMPE—
DMPE—
PC —
Start —
1 2 3 4 5 6 7

Fig. 5. PMT activities after expression of bacterial genes potentially encoding Pmt enzymes in *E. coli*. PMT assays were performed with different cell-free extracts and potential lipid products were extracted and separated by one-dimensional TLC. Total lipids of *E. coli* (lane 1) and *S. meliloti* (lane 2) as obtained after labelling with [1-¹⁴C]acetate. Products of potential lipid methyltransferase activities obtained by labelling with S-adenosyl-L-[methyl-¹⁴C]methionine with cell-free extracts of *E. coli* BL21(DE3)(pLysS pET9a) (lane 3), *E. coli* BL21(DE3)(pLysS pTB2084) harbouring *pmtA* from *S. meliloti* (lane 4), *E. coli* BL21(DE3)(pLysS pFM20) harbouring *pmtA* from *M. loti* (lane 5), *E. coli* BL21(DE3)(pLysS pFM18) harbouring a *pmtA*-like ORF (mlr5374) from *M. loti* (lane 6) and *E. coli* BL21 (DE3)(pLysS pFM14) harbouring *pmtA* from *L. pneumophila* (lane 7). The lipids PC, DMPE and MMPE are indicated.

reduced activity and that from *Bradyrhizobium japonicum*, *M. loti* or *P. aeruginosa* was not functional (data not shown).

### Identification of bacterial ORFs encoding Pmt or Pcs enzymes

Based on sequence comparison with known Pmt or Pcs enzymes, ORFs from numerous organisms were suggested to encode Pmt or Pcs activities (López-Lara & Geiger, 2001; Sohlenkamp et al., 2003; López-Lara et al., 2003). To evaluate some of these predictions, selected ORFs were cloned and expressed in *E. coli*, and cell-free extracts were assayed for Pmt or Pcs activity. Expression of PmtA from *S. meliloti* from plasmid pTB2084 (Fig. 5, lane 4) leads to the formation of MMPE, DMPE and PC, as demonstrated previously (de Rudder et al., 2000). Putative homologues were compared with the enzyme activity caused by PmtA from *S. meliloti*. In *M. loti*, two homologues of the sinorhizobial PmtA had been identified, one (mll4753) with a high similarity and the other (mlr5374) with a lower similarity to the sinorhizobial PmtA. Expression of the ORF with the higher similarity to the sinorhizobial PmtA (mll4753), present in pFM20, caused the formation of MMPE, DMPE and PC (lane 5); therefore, this ORF encodes a Pmt (Mlot PmtA). Expression of the ORF with the lower similarity to the sinorhizobial PmtA (mlr5374), cloned in pFM18, might cause the formation of MMPE and possibly DMPE (lane 6) and, like the PmtA of *Bradyrhizobium japonicum* (Minder et al., 2001), the expressed ORF mlr5374 seems unable to form PC (lane 6). Expression of an ORF from *L. pneumophila* from pFM14 which encodes a

homologue to the rhodobacterial PmtA leads to the formation of MMPE, DMPE and PC (lane 7), demonstrating that this ORF encodes a functional Pmt. In contrast, expression of an ORF (PA0798) from *P. aeruginosa* (in pFM16) which shows homology to the ORF encoding the rhodobacterial PmtA did not cause the formation of any of the methylated derivatives of PE (data not shown). We also tried to express a longer version of PA0798 (from pFM17) using as the starting methionine one located 114 aa upstream of the predicted start. Similarly, also with the longer version expressed, we could not observe the formation of any of the methylated derivatives of PE (data not shown).

Expression of *pcs* from *S. meliloti* from pTB2559 in *E. coli* leads to the formation of PC (Fig. 6, lane 2) in a Pcs enzyme assay as demonstrated previously (Sohlenkamp et al., 2000). Expression of ORFs homologous to the one encoding sinorhizobial Pcs shows that the respective ORF (BAB48080) from *M. loti* harboured in pFM15 (lane 3), an ORF deduced from a fragment of the unfinished genome of *L. pneumophila* present in pFM3 (lane 4), the ORF (PA3857) from *P. aeruginosa* in pTB2906 (lane 5) and an ORF (BB0249) from *Borrelia burgdorferi* in pTB2902 (lane 6) produce PC in the Pcs reaction and therefore encode functional Pcs enzymes.

functional Pmt enzymes. In cell-free extracts of Brucella melitensis no PMT activity was detected. This was surprising as the Brucella melitensis genome contains a homologue (Bmel ORF1, 65% identity on amino acid level) with high similarity to the sinorhizobial PmtA (Smel PmtA). One possibility is that the expression or activity of Bmel ORF1 is down-regulated under the conditions for Brucella melitensis growth. Expression of the PmtA homologue (Paer ORF) of *P. aeruginosa* in *E. coli* did not cause PMT activity nor was such activity detected by *in vitro* or *in vivo* assays of *P. aeruginosa*, suggesting that *P. aeruginosa* has no methylation pathway for PC biosynthesis. A similar conclusion was drawn from recent studies performed by Wilderman *et al.* (2002). Earlier work on the methylation pathway of PC biosynthesis in *Bradyrhizobium japonicum* (Minder *et al.*, 2001) suggested that besides a phospholipid N-methyltransferase (PmtA) catalysing efficiently the first methylation, and therefore the formation of MMPE, another phospholipid methyltransferase (PmtX) activity must exist in this organism which efficiently performs the second and third methylation required for PC formation via the methylation pathway. In the *Bradyrhizobium japonicum* genome sequence (Kaneko *et al.*, 2002), two homologues (Bjap ORF1 and Bjap ORF2) of the rhodobacterial PmtA exist that could qualify as candidates for PmtX.

Fig. 6. PCS activities after expression of potential PCSs in *E. coli*. PCS assays were performed with different cell-free extracts and potential lipid products were extracted and separated by one-dimensional TLC. Products of potential PCS activities obtained by labelling with [methyl-¹⁴C]choline with cell-free extracts of *E. coli* BL21(DE3)(pLysS) (lane 1), *E. coli* BL21(DE3)(pLysS pTB2559) harbouring *pcs* from *S. meliloti* (lane 2), *E. coli* BL21(DE3)(pLysS pFM15) harbouring *pcs* from *M. loti* (lane 3), *E. coli* BL21(DE3)(pLysS pFM3) harbouring *pcs* from *L. pneumophila* (lane 4), *E. coli* BL21(DE3)(pLysS pTB2906) harbouring *pcs* from *P. aeruginosa* (lane 5) and *E. coli* BL21(DE3)(pLysS pTB2902) harbouring *pcs* from *Borrelia burgdorferi* (lane 6). The lipid PC is indicated.

---

### DISCUSSION

PC, a major membrane lipid in eukaryotes, is also encountered in diverse eubacterial groups. Studies with Sinorhizobium meliloti demonstrated that PC can be synthesized via two different routes, the PMT (de Rudder *et al.*, 2000) pathway, represented by phospholipid N-methyltransferase (PmtA) enzymes, or the PCS (de Rudder *et al.*, 1999; Sohlenkamp *et al.*, 2000) pathway. In recent years, numerous homologues of PmtA or Pcs have been described, but only a few have been evaluated experimentally (Hanada *et al.*, 2001; Minder *et al.*, 2001; Wilderman *et al.*, 2002).

In this work, we have shown that PmtA homologues encode PMT activities responsible for PMT pathways in *L. pneumophila* (Lpne PmtA) or *M. loti* (Mlot PmtA). PMT activities were detected in cell-free extracts of *A. tumefaciens* as well as of *Rhizobium leguminosarum*. Also, a homologue of the sinorhizobial PmtA (Smel PmtA) can be detected in the genome of *A. tumefaciens* (Atum ORF1, 66% identity on amino acid level) and a homologue of the mesorhizobial PmtA (Mlot PmtA) can be detected in the genome of *Rhizobium leguminosarum* (Rleg ORF1, 66% identity on amino acid level) (see Fig. 7); therefore, we suggest that Atum ORF1 as well as Rleg ORF1 encode

Fig. 7. Unrooted phylogenetic tree of Pmt enzymes and Pmt-like ORFs. The tree was constructed using the program CLUSTAL W at http://www.expasy.ch/ (Thompson et al., 1994). Distances between sequences are expressed as 0·1 changes per amino acid residue. Sequences used for the tree were Sinorhizobium meliloti PmtA (Smel PmtA; GenBank accession no. AF201699), PmtA from M. loti (Mlot PmtA; mlr4753), PmtA from Bradyrhizobium japonicum (Bjap PmtA; Y09633), PmtA from Rhodobacter sphaeroides (Rsp h PmtA; L07247), PmtA from Acetobacter aceti (Aace PmtA; AB019196), PmtA from Legionella pneumophila (Lpne PmtA; ORF deduced from fragment of unfinished genome), an ORF deduced from the genome of Agrobacterium tumefaciens (Atum ORF1; accession no. AE009001), an ORF from the genome of Rhizobium leguminosarum (Rleg ORF1; rhiz775c03.q1k), an ORF from the genome of Brucella melitensis (Bmel ORF1; AE009632), two ORFs deduced from the genome of Bradyrhizobium japonicum (Bjap ORF1 and Bjap ORF2; accession nos AP005960 and AP005959, respectively), an ORF deduced from the genome of M. loti (Mlot ORF; mlr5374), an ORF deduced from the genome of Yersinia pestis (Ypes ORF; AJ414156), an ORF deduced from the genome of P. aeruginosa (Paer ORF; AE004515), a sinorhizobial ORF (Smel ORF; AE007209), the 16S rRNA methyltransferase KgsA from E. coli (Eco KgsA; M11054), an ORF from the genome of Pyrococcus furiosus (Pfur ORF; AE010192) and an ORF from contig 2543 of the unfinished genome of Silicibacter pomeroyi (Spom ORF1).

pseudomonads also possess a PCS pathway. The fact that no PCS activity could be detected in cell-free extracts of Rhodobacter sphaeroides is in agreement with earlier studies (Arondel et al., 1993). Surprisingly, a Pcs homologue (Rsp h ORF) exists in Rhodobacter sphaeroides and more studies will be required in order to establish whether Rsp h ORF encodes a Pcs activity and why Rhodobacter sphaeroides seems devoid of a functional PCS pathway. So far, all known PCSs form a group clearly distinct from other CDP-alcohol phosphatidyltransferases (Sohlenkamp et al., 2003), such as, for example, the phosphatidylserine synthases (Pss) (Fig. 8). The proposed motif for PCSs DG(X)-2AR(X)8P(X)3G(X)3D(X)3D (Sohlenkamp et al., 2000, 2003) is encountered in all enzymes showing that activity and therefore this motif is highly predictive for PCSs.

In a recent estimate, we proposed that more than 10% of the eubacteria contain PC as a membrane lipid (Sohlenkamp et al., 2003). Based on the studies presented here, we propose that S. meliloti, A. tumefaciens, Rhizobium leguminosarum,

Bradyrhizobium japonicum, M. loti and L. pneumophila possess both bacterial pathways for PC biosynthesis, i.e. the PMT and PCS pathways. Therefore, if bacteria possess PC, they are usually able to form it via both pathways. Some bacteria such as Zymomonas mobilis (Tahara et al., 1994), Rhodobacter sphaeroides (Arondel et al., 1993) or some Burkholderia strains might possess only the PMT pathway for PC formation. Notably, some important pathogens (Brucella melitensis, P. aeruginosa and Borrelia burgdorferi) seem to possess only the PCS pathway for PC formation and therefore presumably depend on choline supplies from their hosts for the ability to synthesize PC. P. aeruginosa is an opportunistic pathogen and poses a serious threat to cystic fibrosis patients. Brucella melitensis, frequently found in dairy products, can cause brucellosis, whereas the tick-transmitted spirochaete Borrelia burgdorferi is the causative agent of Lyme disease. After initial infection, all three aforementioned pathogens tend to persist in their hosts and when established as persistent infections are difficult to eliminate from the host with presently available treatments.

F. Martínez-Morales and others

![Diagram](attachment:diagram.png)

Fig. 8. Unrooted phylogenetic tree of phosphatidylserine synthases (Pss), Pcs enzymes and Pcs-like ORFs. The tree was constructed using the program CLUSTAL W at http://www.expasy.ch/ (Thompson et al., 1994). Distances between sequences are expressed as 0·1 changes per amino acid residue. Sequences used for the tree are Pss from *Agrobacterium tumefaciens* (Atum Pss; GenBank accession no. AF410774), Pss from *Bacillus subtilis* (Bsub Pss; D38022), Pss from *Helicobacter pylori* (Hpyl Pss; AAC45587), Pcs from *Sinorhizobium meliloti* (Smel Pcs; AAF27310), Pcs from *P. aeruginosa* (Paer Pcs; ORF PA3857), Pcs from *M. loti* (Mlot Pcs; BAB48080), Pcs from *Legionella pneumophila* (Lpne Pcs; ORF deduced from fragment of unfinished genome), Pcs from *Borrelia burgdorferi* (Bbur Pcs; BB0249), the sequences of an ORF from the genome of *A. tumefaciens* (Atum ORF2; AAK87563), an ORF from contig rhiz659e06.q1n of the unfinished genome of *Rhizobium leguminosarum* bv. viciae (Rleg ORF2), an ORF from the genome of *Brucella melitensis* (Bmel ORF2; AAL53937), an ORF from the genome of *Rhodobacter sphaeroides* (Rsph ORF; contig 110, gene 289), an ORF from the unfinished genome of *Rhodobacter capsulatus* (Rcap ORF; RRC00355), an ORF from contig 68 of the unfinished genome of *Silicibacter pomeroyi* (Spom ORF2), an ORF from the genome of *Rhodopseudomonas palustris* (Rpal ORF; contig 58, gene 1464), an ORF from the genome of *Bradyrhizobium japonicum* (Bjap ORF3; AP005951), an ORF from the genome of *P. fluorescens* (Pflu ORF; contig 309, gene 76) and an ORF from the genome of *P. syringae* (Psyr ORF; contig 5668).

---

### ACKNOWLEDGEMENTS

We thank Jesús Caballero-Mellado (Centro de Investigación sobre Fijación de Nitrógeno, Universidad Nacional Autónoma de México, Mexico), Ralph Isberg (Dept of Molecular Biology and Microbiology, Tufts University School of Medicine, Boston, MA, USA), Brian Stevenson (Dept of Microbiology, Immunology and Molecular Genetics, University of Kentucky College of Medicine, Lexington, KY, USA), Esperanza Martínez-Romero (Centro de Investigación sobre Fijación de Nitrógeno, Universidad Nacional Autónoma de México, Mexico) and the MAFF GENE Bank of the National Institute for Agrobiological Sciences (NIAS), Japan, for providing strains, James Russo for the gift of genomic DNA from *Legionella pneumophila* and the Unidad de Síntesis of the Instituto de Biotecnología/UNAM for the synthesis of oligonucleotides. We thank Viola Röhrs, Marco Antonio Rogel and Ivan Briceño Argüello for their excellent technical assistance. This work was supported by grants from CONACyT/Mexico (33577-N) and the Howard Hughes Medical Institute (HHMI 55003675).

---

### REFERENCES

- Altschul, S. F., Madden, T. L., Schaffer, A. A., Zhang, J., Zhang, Z., Miller, W. & Lipman, D. J. (1997). Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. *Nucleic Acids Res* 25, 3389–3402.
- Arondel, V., Benning, C. & Sommerville, C. R. (1993). Isolation and functional expression of a gene encoding phosphatidylethanolamine methyltransferase (EC 2.1.1.17) from *Rhodobacter sphaeroides*. *J Biol Chem* 268, 16002–16008.
- Berger, K. H. & Isberg, R. R. (1993). Two distinct defects in intra-cellular growth complemented by a single genetic locus in *Legionella pneumophila*. *Mol Microbiol* 7, 7–19.
- Bligh, E. G. & Dyer, J. W. (1959). A rapid method of total lipid extraction and purification. *Can J Biochem Physiol* 37, 911–917.
- de Rudder, K. E. E., Thomas-Oates, J. E. & Geiger, O. (1997). *Rhizobium meliloti* mutants deficient in phospholipid N-methyltransferase still contain phosphatidylcholine. *J Bacteriol* 179, 6921–6928.
- de Rudder, K. E. E., Sohlenkamp, C. & Geiger, O. (1999). Plant-exuded choline is used for rhizobial membrane lipid biosynthesis by phosphatidylcholine synthase. *J Biol Chem* 274, 20011–20016.
- de Rudder, K. E. E., López-Lara, I. M. & Geiger, O. (2000). Inactivation of the gene for phospholipid N-methyltransferase in *Sinorhizobium meliloti*: phosphatidylcholine is required for normal growth. *Mol Microbiol* 37, 763–772.


Dulley, J. R. & Grieve, P. A. (1975). A simple technique for eliminating interference by detergents in the Lowry method of protein determination. *Anal Biochem* 64, 136–141.

Estrada-de los Santos, P., Bustillos-Cristales, R. & Caballero-Mellado, J. (2001). *Burkholderia*, a genus rich in plant-associated nitrogen fixers with wide environmental and geographic distribution. *Appl Environ Microbiol* 67, 2790–2798.

Exton, J. H. (1994). Phosphatidylcholine breakdown and signal transduction. *Biochim Biophys Acta* 1212, 26–42.

Feeley, J. C., Gibson, R. J., Gorman, G. W., Langford, N. C., Rasheed, J. K., Mackel, D. C. & Baine, W. B. (1979). Charcoal-yeast extract agar: primary isolation medium for *Legionella pneumophila*. *J Clin Microbiol* 10, 437–441.

Glazebrook, J. & Walker, G. C. (1991). Genetic techniques in *Rhizobium meliloti*. *Methods Enzymol* 204, 398–418.

Grogan, D. W. & Cronan, J. E., Jr (1997). Cyclopropane ring formation in membrane lipids of bacteria. *Microbiol Mol Biol Rev* 61, 429–441.

Hamilton, R. H. & Fall, M. Z. (1971). The loss of tumor-initiating ability in *Agrobacterium tumefaciens* by incubation at high temperature. *Experientia* 27, 229–230.

Hanada, T., Kashima, Y., Kosugi, A., Koizumi, Y., Yanagida, F. & Udaka, S. (2001). A gene encoding phosphatidylethanolamine N-methyltransferase from *Acetobacter aceti* and some properties of its disruptant. *Biosci Biotechnol Biochem* 65, 2741–2748.

Hanahan, D. (1983). Studies on transformation of *Escherichia coli* with plasmids. *J Mol Biol* 166, 557–580.

Hoang, T. T., Karkhoff-Schweizer, R. R., Kutchma, A. J. & Schweizer, H. P. (1998). A broad-host-range Flp-FRT recombination system for site-specific excision of chromosomally-located DNA sequences: application for isolation of unmarked *Pseudomonas aeruginosa* mutants. *Gene* 212, 77–86.

Kaneko, T., Nakamura, Y., Sato, S. & 21 other authors (2000). Complete genome structure of the nitrogen-fixing symbiotic bacterium *Mesorhizobium loti*. *DNA Res* 7, 331–338.

Kaneko, T., Nakamura, Y., Sato, S. & 14 other authors (2002). Complete genomic sequence of nitrogen-fixing symbiotic bacterium *Bradyrhizobium japonicum* USDA110. *DNA Res* 9, 189–197.

Karnezis, T., Fisher, H. C., Neumann, G. M., Stone, B. A. & Stanisich, V. A. (2002). Cloning and characterization of the phosphatidylserine synthase gene of *Agrobacterium* sp. strain ATCC 31749 and effect of its inactivation on production of high-molecular-mass (1→3)-β-D-glucan (curdlan). *J Bacteriol* 184, 4114–4123.

Kent, C. (1995). Eukaryotic phospholipid biosynthesis. *Annu Rev Biochem* 64, 315–343.

López-Lara, I. M. & Geiger, O. (2001). Novel pathway for phosphatidylcholine biosynthesis in bacteria associated with eukaryotes. *J Biotechnol* 91, 211–221.

López-Lara, I. M., Sohlenkamp, C. & Geiger, O. (2003). Membrane lipids in plant-associated bacteria: their biosyntheses and possible functions. *Mol Plant–Microbe Interact* 16, 567–579.

Meade, H. M., Long, S. R., Ruvkun, G. B., Brown, S. E. & Ausubel, F. M. (1982). Physical and genetic characterization of symbiotic and auxotrophic mutants of *Rhizobium meliloti* induced by transposon Tn5 mutagenesis. *J Bacteriol* 149, 114–122.

Miller, J. H. (1972). *Experiments in Molecular Genetics*. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory.

Minder, A. C., de Rudder, K. E. E., Narberhaus, F., Fischer, H.-M., Hennecke, H. & Geiger, O. (2001). Phosphatidylcholine levels in *Bradyrhizobium japonicum* membranes are critical for an efficient symbiosis with the soybean host plant. *Mol Microbiol* 39, 1186–1198.

Regensburger, B. & Hennecke, H. (1983). RNA polymerase from *Rhizobium japonicum*. *Arch Microbiol* 135, 103–109.

Rock, C. O., Jackowski, S. & Cronan, J. E. (1996). Lipid metabolism in prokaryotes. In *Biochemistry of Lipids, Lipoproteins and Membranes*, pp. 35–74. Edited by D. E. Vance & J. Vance. Amsterdam: Elsevier.

Sambrook, J., Fritsch, E. F. & Maniatis, T. (1989). *Molecular Cloning: a Laboratory Manual*, 2nd edn. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory.

Sanger, F., Nicklen, S. & Coulson, A. R. (1977). DNA sequencing with chain-terminating inhibitors. *Proc Natl Acad Sci USA* 74, 5463–5467.

Simon, R., Priefer, U. & Pühler, A. (1983). Vector plasmids for *in vivo* and *in vitro* manipulations of Gram-negative bacteria. In *Molecular Genetics of the Bacteria–Plant Interaction*, pp. 98–106. Edited by A. Pühler. Heidelberg: Springer.

Sistrom, W. R. (1962). The kinetics of the synthesis of photo-pigments in *Rhodospseudomonas sphaeroides*. *J Gen Microbiol* 28, 607–616.

Sohlenkamp, C., de Rudder, K. E. E., Röhrs, V., López-Lara, I. M. & Geiger, O. (2000). Cloning and characterization of the gene for phosphatidylcholine synthase. *J Biol Chem* 275, 18919–18925.

Sohlenkamp, C., López-Lara, I. M. & Geiger, O. (2003). Biosynthesis of phosphatidylcholine in bacteria. *Prog Lipid Res* 42, 115–162.

Spank, H. P., Okker, R. J. H., Wijffelman, C. A., Pees, E. & Lugtenberg, B. J. J. (1987). Promoters in the nodulation region of the *Rhizobium leguminosarum Sym* plasmid pRL1JI. *Plant Mol Biol* 9, 29–37.

Studier, F. W., Rosenberg, A. H., Dunn, J. J. & Dubendorff, J. W. (1990). Use of T7 RNA polymerase to direct expression of cloned genes. *Methods Enzymol* 185, 60–89.

Tahara, Y., Yamashita, T., Sogabe, A. & Ogawa, Y. (1994). Isolation and characterization of *Zymomonas mobilis* mutant defective in phosphatidylethanolamine N-methyltransferase. *J Gen Appl Microbiol* 40, 389–396.

Taniguchi, S., Morikawa, S., Hayashi, H., Fujii, K., Mori, K., Fujiwara, M. & Fujiwara, M. (1986). Effects of Ca<sup>2+</sup> on ethanolaminephosphotransferase and cholinephosphotransferase in rabbit platelets. *J Biochem* 100, 485–491.

Thompson, J. D., Higgins, D. G. & Gibson, T. J. (1994). CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. *Nucleic Acids Res* 22, 4673–4680.

Vincent, J. M. (1970). *A Manual for the Practical Study of Root-Nodule Bacteria*. International Biological Program Handbook No. 15. Oxford: Blackwell Scientific Publications.

Wilderman, P. J., Vasil, A. I., Martin, W. E., Murphy, R. C. & Vasil, M. L. (2002). *Pseudomonas aeruginosa* synthesizes phosphatidylcholine by use of the phosphatidylcholine synthase pathway. *J Bacteriol* 184, 4792–4799.

Yanisch-Perron, C., Vieira, J. & Messing, J. (1985). Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. *Gene* 33, 103–119.
