
# Biosynthesis of Antibiotics

## Characterization of NovP and NovN: Completion of Novobiocin Biosynthesis by Sequential Tailoring of the Noviosyl Ring**

Caren L. Freel Meyers, Markus Oberthür, Hui Xu, Lutz Heide, Daniel Kahne, and Christopher T. Walsh*

Novobiocin, clorobiocin, and coumermycin A₁ are members of the aminocoumarin family of natural products and exert their antibacterial effects by inhibiting the bacterial type II topoisomerase DNA gyrase. Specifically, the aminocoumarins are competitive inhibitors of adenosine triphosphate (ATP) binding and hydrolysis, which are essential for the catalysis of topoisomer interconversion during bacterial DNA replication.[1,2] The aminocoumarins novobiocin (1) and clorobiocin (2; Figure 1a) have three structural elements: the central 3-amino-7-hydroxycoumarin is linked at the 3-amino moiety to a prenylated 4-hydroxybenzoic acid component, and at the 7-position to an L-noviosyl sugar component. The noviosyl ring of both compounds can be methylated at the 4′-hydroxy group and carbamoylated (novobiocin) or acylated (clorobiocin) at the 3′-hydroxy group.

X-ray crystallographic studies of novobiocin and clorobiocin with their target protein (Figure 1b–d) suggest that the noviosyl sugar component has several key interactions with the ATP binding site of the DNA gyrase subunit GyrB.[1] In particular, the cocrystal structure of novobiocin with GyrB reveals a hydrogen-bonding interaction between the 3′-O-carbamoyl moiety and an ordered water molecule in the ATP binding site (Figure 1c). The larger 5-methylpyrrolyl acyl moiety of clorobiocin displaces this water molecule and makes hydrophobic contacts with the enzyme side chains (Figure 1d). This structural alteration leads to a 10-fold increase in antibacterial potency,[2] which suggests that the substituent at the 3′-position of the aminocoumarin antibiotics controls antibacterial activity to a significant extent.

For this reason, it is of particular interest to introduce structural diversity at the 3′-position of the noviosyl ring and generate novel aminocoumarin variants by chemoenzymatic routes. Our research group is interested in the characterization of glycosyltransferases dedicated to antibiotic assembly and maturation. Our work includes the use of glycosylation in chloroeremomycin and vancomycin biosynthesis to generate novel analogues.[3,4] In addition, we recently reported the heterologous overproduction of the L-noviosyl transferase NovM in *Escherichia coli* and its purification in a soluble, active form, and we have characterized NovM as a catalyst for the conversion of novobiocin acid (3) into desmethylcarbamoylnovobiocin (4; Scheme 1). The transfer of noviose from thymidine-5′-diphospho-β-L-noviose (TDP-β-L-noviose) to novobiocin acid occurs with a catalytic rate constant of \( k_{\text{cat}} > 300 \, \text{min}^{-1} \).[5] Desmethylcarbamoylnovobiocin (4) is presumably the substrate for regiospecific methylation at the 4′-position followed by carbamoylation at the 3′-position. These transformations are the final enzymatic steps in novobiocin biosynthesis[6] and have been predicted to be catalyzed by the enzymes NovP and NovN, respectively, in the novobiocin biosynthetic cluster.[7]

We report herein the cloning and overproduction of the 30-kDa *Streptomyces spheroides* NovP in *E. coli*, and its purification in soluble form as an N-His₆-tagged protein (Figure 2, Lane 1). The catalytic activity of NovP as a methyltransferase in the conversion of desmethylcarba-

---

[*] Dr. C. L. Freel Meyers, Prof. C. T. Walsh  
Department of Biological Chemistry and Molecular Pharmacology  
LHRRB 309, Harvard Medical School  
Boston, MA 02115 (USA)  
Fax: (+1) 617-432-0556  
E-mail: Christopher_walsh@hms.harvard.edu  

Dr. M. Oberthür, Prof. D. Kahne  
Department of Chemistry, Princeton University  
Princeton, NJ, 08544 (USA)  

Dr. H. Xu, Prof. L. Heide  
Pharmaceutical Institute, Tuebingen University  
72076 Tuebingen (Germany)

[**] We gratefully acknowledge the National Institutes of Health Grants 49338 (to C.T.W.), 66174 (to D.K.), and AI054007-01 (to C.L.F.M.). The contribution of X.H. and L.H. was supported by a grant from the Deutsche Forschungsgemeinschaft.

Angew. Chem. Int. Ed. **2004**, 43, 67–70  
DOI: 10.1002/anie.200352626  
© 2004 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim

Communications

![Chemical Structure 1](image1.png)

Novobiocic acid (3)

![Chemical Structure 2](image2.png)

Desmethyl descaramoylnovobiocin (4)

![Chemical Structure 3](image3.png)

Novobiocin (1)

![Chemical Structure 4](image4.png)

Descarboxyl novobiocin (5)

S. spheroides NovN (75 kDa) was cloned and overproduced in E. coli, and purified as a C-His₈-tagged protein (Figure 2, Lane 2) following extensive optimization of the procedure. The catalytic activity of NovN as a carbamoyltransferase was confirmed by HPLC results showing the conversion of precursor 5 into novobiocin (1) in the presence of the enzyme (Figure 3b). Incubation of desmethyl descaramoylnovobiocin (4) with NovN in the presence of carbamoyl phosphate and ATP did not result in the formation of desmethyl novobiocin, which confirms the observation of Kominek and Sebek [6] that NovN acts after NovP in the synthetic sequence. Further confirmation of novobiocin formation was obtained by mass spectral analysis (calcd:

---

**Scheme 1.** The final steps in novobiocin biosynthesis. SAM, S-adenosylmethionine.

---

**Figure 2.** 4–15% tris(hydroxymethyl)aminomethane (Tris)/HCl SDS-PAGE of purified NovP and NovN.

---

**Figure 3.** a) Overnight tandem incubation of novobiocic acid (3) with NovM/TDP-L-noviose, then NovP/SAM, pH 8, RT; b) conversion of descaramoylnovobiocin (5) into novobiocin (1) in the presence of NovN/NH₂CO₂PO₃²⁻/ATP, pH 7.5, RT.

---

moynovobiocin (4) into descaramoylnovobiocin (5; Scheme 1) was initially confirmed in tandem incubations of novobiocic acid (3) with NovM/TDP-L-noviose and NovP/SAM. A new compound whose formation was dependent upon the presence of NovP/SAM was detected by HPLC (Figure 3). Analysis of the reaction mixture by LCMS confirmed that this compound was descaramoylnovobiocin (5; calcd: m/z 570.23 [M+H⁺]; found: 570.20). The kinetic characterization of NovP was then undertaken by using purified 4 obtained from a large-scale NovM incubation. NovP catalyzes 4′-O-methylation of 4 with \( k_{\text{cat}} = 0.400 \pm 0.006 \, \text{min}^{-1} \) and \( K_{\text{M}} = 9.5 \pm 3 \, \mu \text{M} \) (Michaelis constant), which is a low but usable catalytic efficiency. The enzymatic methylation of compound 4 provided quantities of 5 sufficient for assay of the last enzyme in the biosynthetic pathway of novobiocin.

\( m/z \, 613.23 \, [M + H^+] \); found: 613.15). Novobiocin generated from novobiocic acid and TDP-L-noviose in the presence of the three purified enzymes NovM, NovP, and NovN and the cosubstrates SAM, carbamoyl phosphate, and ATP is identical to commercially available novobiocin (Sigma Aldrich) according to HPLC results. However, novobiocin undergoes isomerization to isonovobiocin (the 2′-O-carbamoyl isomer of novobiocin) in dilute alkali; [8] isonovobiocin is also known to be a side-product obtained during fermentation/purification of novobiocin. [9] It is therefore important to establish the position of the carbamoyl group in the product of the NovN-catalyzed reaction. Novobiocin can easily be distinguished

© 2004 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim
www.angewandte.org
Angew. Chem. Int. Ed. 2004, 43, 67–70

from isonovobiocin by $^{1}$H NMR spectroscopy. The $^{1}$H NMR spectrum (500 MHz, [D$_{6}$]dimethyl sulfoxide (DMSO), 23°C) obtained for the NovN product was identical to the spectrum of novobiocin and showed peaks at the chemical shifts characteristic for noviose resonances ($\delta = 5.16$ (dd, $J_{2,3} = 3.1$, $J_{3,4} = 9.8$ Hz, 1H; 3-H), 4.08 ppm (m, 1H; 2-H)), which are distinctly different from those recorded in the literature for isonovobiocin ($\delta = 4.18$ (3-H), 4.93 ppm (2-H)).[10]

Further confirmation of novobiocin formation was obtained by comparison of the antibiotic potencies of the NovN reaction product and commercially available novobiocin. The minimum inhibitory concentrations (MICs) of each compound against two bacterial strains (*Enterococcus faecium* and *Staphylococcus aureus*) were determined and found to be comparable (data not shown). Previous reports have established that the 3′-O-carbamoyl moiety is essential for antibiotic activity;[11] loss of activity is observed with the descarbamoylnovobiocin precursor. We also observed an MIC for descarbamoylnovobiocin (5) that was more than 100-fold higher than that determined for novobiocin.

Kinetic parameters were determined for NovN by using purified descarbamoylnovobiocin (5) obtained from a large-scale tandem NovM/NovP incubation. NovN catalyzes the carbamoylation of 5 with $k_{cat} = 4.1 \pm 0.2$ min$^{-1}$. $K_{m}$ values of 4.6 ± 1.3 μM and 5.1 ± 0.4 μM were measured for the substrates descarbamoylnovobiocin and carbamoyl phosphate, respectively.

Surprisingly, the carbamoyltransferase activity of NovN is dependent upon Mg-ATP. The regulatory role of ATP in the activation of the aspartate $N$-carbamoyltransferases in a number of bacterial systems has been well studied.[12–16] Activation of $O$-carbamoyltransferases by ATP in secondary metabolite biosynthesis has been reported in only one case: $O$-carbamoylation of 3-hydroxymethylcephem in *Streptomyces clavuligerus*.[17] The role of ATP in the activation of carbamoyltransferase CmcH was investigated in this study, but the authors were unable to conclude whether ATP acts as an effector or as a substrate in the transfer of a carbamoyl moiety to 3-hydroxymethylcephem. In general, regulation of late-stage transformations by ATP in secondary metabolite biosynthesis is not well documented. The role of ATP activation in this final step of novobiocin biosynthesis remains to be elucidated.

The availability of the last three enzymes in the novobiocin biosynthetic pathway, NovM, NovP, and NovN, in purified, active forms through *E. coli* expression will allow evaluation of the possibility of generating an aminocoumarin library in which variation of each component, as well as sugar decoration results in novobiocin analogues that can be evaluated for antibiotic activity. Moreover, tandem NovM/NovP incubations have been carried out on a 30-mg scale, which provides sufficient material to study the regioselective chemical carbamoylation and/or acylation of descarbamoylnovobiocin (5) at the 3′-position. We anticipate that variation of the 3′ moiety will provide insight into the structural requirements for gyrase inhibition and antibiotic activity.

---

### Materials and Methods

Preparation of pNovP-pET28a and pNovN-pET37b overexpression constructs: The genes encoding NovP and NovN were amplified from *Streptomyces spheroides* (ATCC23965) genomic DNA and pXHN,[18] respectively. Amplification of $novP$ was accomplished by using the forward primer 5′-GCGTATCATATGGCACCTATCGTGGAAACGCG-3′ and the reverse primer 5′-GCCATTAAGCTTTTACCGGGTGCCTTGCCAG TAG-3′, which include $NdeI/HindIII$ restriction sites. Amplification of $novN$ was accomplished by using the forward primer 5′-CCGAATCA TATGCTCATCTTGGCCTGAACGGG-3′ and the reverse primer 5′- GCGTTACTCGAGTCACGGTGCGGATCCGGCACC-3′, with $NdeI/XhoI$ restriction sites. The amplified genes were inserted into pET28a ($novP$) and pET37b ($novN$) vectors after restriction digests, and pNovP-pET28a and pNovN-pET37b were expressed in *E. coli* TOP10 competent cells.

Overproduction and purification of NovP and NovN: Purified pNovP-pET28a and pNovN-pET37b plasmids were transformed into BL21 (DE3) competent *E. coli* cells. Transformants harboring the pNovP-pET28a and pNovN-pET37b constructs were grown in Luria–Bertani (LB) medium supplemented with kanamycin (50 μg mL$^{-1}$). NovP was overproduced and purified as described for NovM.[5] For the overproduction of NovN, cells were grown at 25°C to an optical density of around 0.35. The temperature was reduced to 20°C and the cells were shaken for 1 h. NovN production was induced with isopropyl-$\beta$-thiogalactopyranoside (60 μM) and the cells were shaken overnight at 20°C. NovN was isolated from cell extracts as described for NovM.[5]

Characterization of NovP: Reactions were carried out at ambient temperature and reaction mixtures contained desmethyldescarbamoylnovobiocin (4) and SAM in tricine (75 mM, pH 8.5), NaCl (100 mM), bovine serum albumin (BSA; 1 mg mL$^{-1}$), and DMSO (5%). Reactions were initiated by the addition of NovP and terminated at specified time points by quenching in methanol (2 × reaction volume) at 4°C. For the determination of $K_{m}$ and $k_{cat}$, NovP and SAM were added to final concentrations of 500 nM and 500 μM, respectively, and the concentration of desmethyldescarbamoylnovobiocin was varied (2.5–50 μM). The reaction was terminated after 10 and 20 min for each concentration. Each experiment was carried out in triplicate.

Quenched aliquots were incubated at –20°C for 30 min and then centrifuged (5 min at 13000 rpm) to remove precipitated protein. The supernatant was analyzed by analytical reverse-phase HPLC (CH$_{3}$CN/H$_{2}$O (60:40), 0.1% TFA, 1 mL min$^{-1}$) monitored at 340 nm. Product formation was confirmed by LCMS (5: calcd for C$_{30}$H$_{35}$NO$_{10}$: $m/z$ 570.23 [$M+H]^+$; found: 570.20) on a Shimadzu LCMS-QP8000a instrument. The desmethyldescarbamoylnovobiocin (4; $R_{t} = 7.0$ min) and descarbamoylnovobiocin (5; $R_{t} = 12.5$ min) HPLC peak areas were measured and the product concentration was calculated as a percent of the total peak area.

Characterization of NovN: Reactions (75 μL total volume) were carried out at ambient temperature and reaction mixtures contained descarbamoylnovobiocin (5), carbamoyl phosphate, and ATP (4 mM) in Tris (75 mM, pH 9.0), MgCl$_{2}$ (10 mM), BSA (1 mg mL$^{-1}$), and DMSO (5%). Reactions were initiated by the addition of NovN (200 nM) and terminated by quenching the reaction mixture with methanol (150 μL) at 4°C. $K_{m}$ and $k_{cat}$ were determined with descarbamoylnovobiocin (5) and carbamoyl phosphate as the variable substrates. The determination of the $K_{m}$ value for descarbamoylnovobiocin (5) was accomplished at a constant concentration of carbamoyl phosphate (500 μM) and over a range of descarbamoylnovobiocin (5) concentrations (5–50 μM). The reaction was quenched after 5 min for each concentration. The determination of the $K_{m}$ value for carbamoyl phosphate was accomplished at a constant concentration of descarbamoylnovobiocin (5; 50 μM) and over a range of carbamoyl phosphate concentrations (1–14 μM). The reaction was quenched after 10 min for each concentration. Each experiment was carried out in triplicate.
Quenched aliquots were analyzed as described above (HPLC: CH₃CN/H₂O (20:80), 0.1% trifluoroacetic acid (TFA), 15 min, 1 mL·min⁻¹; after 15 min, 20–100% CH₃CN gradient over 10 min). Product formation was confirmed by LCMS (calcd for C₃₁H₃₆N₂O₁₁: 613.23 [M+H]⁺; found: 613.15). The concentrations of descarbamoylnovobiocin (5; *R*~t~ = 12.5 min) and novobiocin (1, *R*~t~ = 11.9 min) were determined as described for NovN.

Large-scale NovM,P,N reactions and purification of enzymatic reaction products: A large-scale NovM incubation was carried out with novobiocic acid (3; 30 mg) and TDP-L-noviose. Reaction progress was monitored by HPLC. Upon completion, the crude reaction mixture was loaded onto a C18 sep pak column (900 mg bed, Alltech) conditioned with β-morpholinoethanesulfonic acid (75 mM, pH 6). The sep pak cartridge was washed with water (15 mL) to remove salts and by-products. Desmethylcarbamoylnovobiocin (4) was eluted in acetonitrile/water (60:40) and concentrated by lyophilization. The enzymatic reaction products from NovP and NovN incubations were purified by using this protocol. Further purification of the NovP reaction product 5 by preparatory HPLC was necessary to remove S-adenosyl-L-homocysteine.

Received: August 12, 2003 [Z52626]

**Keywords:** antibiotics · biosynthesis · enzymes · novobiocin · transferases

---

[1] R. J. Lewis, O. M. Singh, C. V. Smith, T. Skarzynski, A. Maxwell, A. J. Wonacott, D. B. Wigley, *EMBO J.* **1996**, *15*, 1412.

[2] M. Gellert, M. H. O'Dea, T. Itoh, J. Tomizawa, *Proc. Natl. Acad. Sci. USA* **1976**, *73*, 4474.

[3] H. C. Losey, J. Jiang, J. B. Biggins, M. Oberthür, X.-Y. Ye, S. D. Dong, D. Kahne, J. S. Thorson, C. T. Walsh, *Chem. Biol.* **2002**, *9*, 1305.

[4] H. C. Losey, M. W. Peczuh, Z. Chen, U. S. Eggert, S. D. Dong, I. Pelczer, D. Kahne, C. T. Walsh, *Biochemistry* **2001**, *40*, 4745.

[5] C. L. Freel Meyers, M. Oberthür, J. W. Anderson, D. Kahne, C. T. Walsh, *Biochemistry* **2003**, *42*, 4179.

[6] L. A. Kominek, O. K. Sebek, *Dev. Ind. Microbiol. Ser.* **1974**, *15*, 60.

[7] M. Steffensky, A. Mühlenweg, Z. X. Wang, S. M. Li, L. Heide, *Antimicrob. Agents Chemother.* **2000**, *44*, 1214.

[8] J. W. Hinman, E. L. Caron, H. Hoeksema, *J. Am. Chem. Soc.* **1957**, *79*, 3789.

[9] I. V. Gracheva, V. A. Severina, *Antibiotiki* **1966**, *11*, 45.

[10] F. W. Crow, W. K. Duholke, K. A. Farley, C. E. Hadden, D. A. Hahn, B. D. Kaluzny, C. S. Mallroy, G. E. Martin, R. F. Smith, T. J. Thamann, *J. Heterocycl. Chem.* **1999**, *36*, 365.

[11] D. C. Hooper, J. S. Wolfson, G. L. McHugh, M. B. Winters, M. N. Swartz, *Antimicrob. Agents Chemother.* **1982**, *22*, 662.

[12] P. Chen, F. Van Vliet, M. Van De Casteele, C. Legrain, R. Cunin, N. Glansdorff, *J. Bacteriol.* **1998**, *180*, 6389.

[13] R. Cunin, C. S. Rani, F. Van Vliet, J. R. Wild, M. Wales, *J. Mol. Biol.* **1999**, *294*, 1401.

[14] L. Fetler, P. Vachette, *J. Mol. Biol.* **2001**, *309*, 817.

[15] J. B. Sakash, A. Tsen, E. R. Kantrowitz, *Protein Sci.* **2000**, *9*, 53.

[16] J. F. Vickrey, G. Herve, D. R. Evans, *J. Biol. Chem.* **2002**, *277*, 24490.

[17] S. J. Brewer, P. M. Taylor, M. K. Turner, *Biochem. J.* **1980**, *185*, 555.

[18] H. Xu, Z. X. Wang, J. Schmidt, L. Heide, S. M. Li, *Mol. Genet. Genomics* **2002**, *268*, 387.

---

© 2004 Wiley-VCH Verlag GmbH & Co. KGaA, Weinheim  
www.angewandte.org  
Angew. Chem. Int. Ed. **2004**, *43*, 67–70