
Chemistry & Biology, Vol. 10, 1293–1302, December, 2003, ©2003 Elsevier Science Ltd. All rights reserved. DOI 10.1016/j.chembiol.2003.11.016

# Assembly of the Covalent Linkage between Lipoic Acid and Its Cognate Enzymes

Xin Zhao,$^{1}$ J. Richard Miller,$^{3,6}$ Yanfang Jiang,$^{1}$ Michael A. Marletta,$^{3,5}$ and John E. Cronan$^{1,2,*}$

$^{1}$Department of Microbiology  
$^{2}$Department of Biochemistry  
University of Illinois  
Urbana, Illinois 61801  

$^{3}$Howard Hughes Medical Institute  
University of Michigan Medical School  
Ann Arbor, Michigan 48109  

$^{4}$Department of Chemistry  
$^{5}$Department of Molecular and Cell Biology  
University of California, Berkeley  
Berkeley, California 94720  

## Summary

Lipoic acid is synthesized from octanoic acid by insertion of sulfur atoms at carbons 6 and 8 and is covalently attached to a pyruvate dehydrogenase (PDH) subunit. We show that sulfur atoms can be inserted into octanoyl moieties attached to a PDH subunit or a derived domain. *Escherichia coli lipB* mutants grew well when supplemented with octanoate in place of lipoate. Octanoate growth required both lipoate protein ligase (LplA) and LipA, the sulfur insertion protein, suggesting that LplA attached octanoate to the dehydrogenase and LipA then converted the octanoate to lipoate. This pathway was tested by labeling a PDH domain with deuterated octanoate in an *E. coli* strain devoid of LipA activity. The labeled octanoyl domain was converted to lipoylated domain upon restoration of LipA. Moreover, octanoyl domain and octanoyl-PDH were substrates for sulfur insertion in vitro.

## Introduction

Lipoic acid (6,8-thioctic acid or 1,2-dithiolane-3-pentanoic acid) is a sulfur-containing cofactor found in most prokaryotic and eukaryotic organisms. The cofactor is essential for function of several key enzymes involved in oxidative metabolism including pyruvate dehydrogenase (PDH), 2-oxoglutarate dehydrogenase, the branched-chain 2-oxoacid dehydrogenases, and the glycine cleavage system [1]. In each enzyme, a specific subunit is modified by lipoic acid attachment, and the sites of attachment are specific lysine residues within conserved domains of these subunits, called lipoyl domains. An amide linkage is made between the carboxyl group of lipoic acid and the lysine residue ε-amino group of the enzyme subunit [2]. In the 2-oxoacid dehydrogenases, the lipoyl domains are found at the amino termini of the E2 subunits. During catalysis, the protein-bound lipoamide moieties serve as carriers of activated acyl groups between the active sites of these multienzyme complexes [1]. Although the existence of lipoic acid has been known for almost 50 years, the mechanisms by which lipoic acid is synthesized and becomes attached to protein continue to be elucidated.

In *Escherichia coli*, two distinct enzymes catalyze posttranslational modification of apoproteins with lipoate. Exogenous lipoate is transferred to apo proteins in an ATP-dependent process by lipoate-protein ligase A (LplA) [3, 4]. Octanoic acid is also an LplA substrate [3]. The second pathway uses lipoate synthesized from octanoate derived from the fatty acid synthetic pathway and requires the *lipB* gene product, (lipoyl (octanoyl)-[acyl-carrier-protein]-protein-N-lipoyltransferase) [4–6]. LipB utilizes lipoyl-acyl carrier protein (ACP) or octanoyl-ACP as a source of acyl groups and is primarily responsible for the transfer of octanoyl groups to apo-E2 proteins seen when lipoate synthesis is limiting [4, 7]. Both enzymes are active in vivo and in vitro when using isolated E2 domains of about 80 structured residues as the acyl acceptors [3, 4, 6].

Genetic investigations showed that a single *E. coli* gene called *lipA* is responsible for the sulfur-insertion steps of lipoate biosynthesis [8–10]. *S. cerevisiae* also has only one gene for sulfur insertion that encodes a protein homologous to LipA [11], and genes encoding similar proteins that functionally complement *E. coli lipA* mutants are found in plants [12] and mammals [13]. The *lipA* gene encodes a protein of 36 kDa that has approximately 36% identity to the biotin synthase of *E. coli* [9, 14]. Both proteins catalyze S-adenosylmethionine (SAM)-dependent insertion of sulfur at unactivated acyl carbon atoms and are expected to operate by very similar mechanisms [15].

LipA has been overexpressed in *E. coli* and purified both as a soluble protein [16, 17] and as insoluble aggregates that were subsequently refolded and reconstituted [18]. Results from iron and sulfur stoichiometry and spectroscopy analyses indicate that the soluble recombinant protein contains a \([4\mathrm{Fe}-4\mathrm{S}]^{+2/+1}\) cluster that is likely to be essential for lipoate biosynthesis. We recently reported the first in vitro synthesis of lipoic acid by use of an anaerobic system requiring reduced LipA, LipB, octanoyl-ACP, SAM, and the apo form of an acceptor dehydrogenase [17]. We and others [19] have hypothesized that LipA is a member of the family of proteins that utilize iron sulfur clusters and SAM to perform radical chemistry. The use of apo-PDH provides a very sensitive assay for lipoic acid synthesis greatly facilitating our progress, but had the disadvantage that the assay was indirect and therefore unable to detect the primary lipoylated species. Thus, the exact identity of the primary lipoyl product could not be determined. We then turned to an apo-E2 domain as substrate and detected the synthesis of lipoyl-E2 domain by mass spectrometry, but were unable to detect the putative intermediate, lipoyl-ACP. Multiple attempts to isolate free lipoyl-ACP, thiooctanoyl-ACP, or free lipoate from the LipA reaction mixtures were unsuccessful. Addition-

*Correspondence: j-cronan@life.uiuc.edu  
$^{6}$Present address: Pfizer Global Research and Development, Anti-bacterial Pharmacology, Ann Arbor, Michigan 48105.

ally, no evidence for a stable covalent association of the lipoyl product with LipA was observed. In each case, the putative primary product behaved as if it was tightly associated with LipA or was unstable to chromatographic analysis. These results are in marked contrast to biotin synthase in which both free biotin and an unidentified intermediate (proposed to be a 9-mercaptode thiobiotoxin derivative) are readily isolated from reaction mixtures [20].

To address these limitations in our understanding, we have further characterized the LipA and LipB reactions both in vivo and in vitro and report that both an octanoyl-E2 domain and the apo forms of the 2-oxoacid dehydrogenases function as sulfur insertion substrates for LipA.

Results

We shall first report in vivo experiments that test the possibility that octanoyl-E2 domains can be directly converted to lipoyl E2 domains and then present in vitro experiments that address this question in vitro.

Growth of *lipB* Mutants on Octanoate

Null mutants in the *lipB* gene were first isolated as lipoic acid auxotrophs having a phenotype of very slow growth in minimal media lacking lipoate [10]. Further work showed that when grown in the absence of lipoate (in the presence of succinate and acetate which bypasses the 2-oxoacid dehydrogenase deficiencies), *lipB* null mutant strains retained low levels of lipoylated proteins and low activity of the lipoic acid-requiring dehydrogenases [9]. Introduction of a null mutation in *lpIA*, the gene encoding lipoate protein ligase, into *lipB* strains results in strains that completely fail to grow on minimal medium [4]. Moreover, overproduction of the LplA ligase allowed normal growth of *lipB* mutants in the absence of lipoate [4]. Since LplA catalyzes the attachment of lipoate (or octanoate) to the apo forms of the 2-oxoacid dehydrogenases [3], these observations suggested that *lipB* encoded a second attachment enzyme, the function of which could be partially replaced by LplA. Indeed, *lipB* mutants were found to lack a novel enzyme that uses the lipoyl or octanoyl thioesters of the acyl carrier protein of fatty acid synthesis as the acyl donor in modification of the apo forms of the dehydrogenase E2 subunits [5]. Recently, it was shown that *lipB* encodes the lipoate/octanoate transferase and that (consistent with the genetic complementation experiments) purified LplA catalyzes the LipB reaction, albeit very poorly [6]. We now report that *lipB* mutant strains grow on octanoate as well as on lipoate, although higher concentrations of the fatty acid were needed for growth comparable to that seen with lipoate (Figure 1). Growth on octanoate required the functions of both the LplA lipoate ligase and LipA, the protein responsible for sulfur insertion (Figure 1). Moreover, growth was specific to octanoate; fatty acids of 6, 7, 9, and 10 carbons were inactive (data not shown). These observations argued for the existence of an LplA-dependent pathway that could bypass LipB function in the presence of octanoate. In the postulated bypass pathway (Figure 2), LplA would attach octanoate from the growth medium to the unmodified E2 domains

of the pyruvate and 2-oxoglutarate dehydrogenase E2 subunits. LipA would then insert two sulfur atoms into the covalently bound octanoyl moiety and thereby convert the octanoyl-E2 domains to lipoyl-E2 domains. That is, lipoic acid would be assembled on its cognate proteins. The resulting active enzymes would account for the ability of octanoate to support growth of *lipB* strains (Figure 1).

In Vivo Tests of the Postulated Bypass Pathway

We first tested the in vivo pathways for formation of octanoyl-E2 domain. Prior work had shown that LplA could attach exogenously added \[^{1-14}C\]octanoate to the dehydrogenase E2 subunits, but the extent of modification was not determined [3]. We expressed an 87 residue E2 domain derived from *E. coli* PDH in a host strain that carried null mutations in *lipA* (to prevent lipoic acid synthesis), *lipB* (to block octanoate transfer from fatty acid synthesis), and *fadE* (to block β-oxidative degradation of octanoate). The use of the domain allowed detection of modification by electrophoretic mobility shifts and mass spectroscopy. When this strain was cultured in a medium supplemented with octanoic acid, we found that about half of the domain became modified (data not shown; similar data are given below). Note that these cultures were grown in the presence of acetate and succinate in order to allow aerobic growth in the absence of lipoate. We also assayed the LipB-dependent pathway in a *lipA lpIA* null mutant strain grown in the absence of endogenous octanoate. In agreement with prior work using a *lipA* strain [7], we detected octanoyl-E2 domain accumulation (data not shown). Therefore, the E2 domain could be octanoylated in vivo either by LplA, using endogenously added octanoate, or by LipB, using de novo synthesized octanoate.

In Vivo Conversion of Octanoyl-E2 Domains to Lipoyl-E2 Domains

In order to test the possibility that octanoylated E2 domain could be a direct precursor to lipoyl-E2 domain, we supplemented the *lipA lipB fadE* (*λ*) strain with deuterated octanoic acid. This would result in accumulation of *d*~15~-labeled octanoyl-E2 domain which could be readily distinguished by mass spectroscopy from domain modified with endogenously-synthesized octanoate. We then planned to remove the labeled octanoate supplement and subsequently express the *lipA* gene. This would allow us to test if LipA could convert the *d*~15~-labeled octanoyl-E2 domain to the postulated *d*~13~-labeled lipoyl-E2 domain species. We first attempted to use tightly regulated promoters to block *lipA* expression during accumulation of the octanoyl-E2 domain. However, even the most stringently regulated promoters we tested allowed synthesis of sufficient lipoic acid to support growth of a *lipA* lipoate auxotroph. Therefore, we used another approach, transduction of the octanoic-*d*~15~ acid-labeled cells with phage *λ* particles containing a *lipA* cosmid. The cosmid was packaged in vivo by use of a newly constructed *λ* lysogen strain described elsewhere [21]. Use of transduction by phage *λ* particles allowed introduction of a functional *lipA* gene into all of the cells of the *lipA lipB fadE* (*λ*) culture with essentially no physio-

Lipoic Acid Synthesis
1295

+ Acetate/Succinate + Lipoic Acid + Octanoic Acid No Addition

Figure 1. Growth of *lip* Mutants on Minimal Medium

The strains were streaked onto plates of glucose minimal E medium supplemented as shown above the plates in the pattern given at the left of each row of plates. The strains were the wild-type (WT) strain JK1, KER176 (*lipA*), KER184 (*lipB*), TM134 (*lplA*), TM131 (*lplA lipA*), and TM136 (*lplA lipB*). The supplements to the medium were added to final concentrations of acetate, 5 mM; succinate, 5 mM; lipoic acid, 5 ng/ml (~25 nM); and octanoic acid, 50 μM. The plates were incubated 24 hr at 37 °C.

logical disturbance [21] (the λ prophage was introduced into the recipient to prevent infection by the few λ DNA-containing particles present in the packaged cosmid preparations). We have demonstrated *lipA* expression from the introduced cosmids by Western blotting experiments (data not shown). Using transduction with *lipA* cosmids, we performed two types of labeling experiments. In the first type of experiment, we labeled E2 domains in vivo by growth in the presence of octanoic-<sub>15</sub> acid. We then removed the labeled octanoate by

pelleting and washing the cells (control experiments using [1-<sup>14</sup>C]octanoic acid showed that the washing protocol completely blocked further incorporation of exogenous octanoate; data not shown). The cells were then resuspended in growth medium and transduced with the packaged cosmid preparation. After various times of incubation, samples were taken and the E2 domain species were isolated and purified. The modified and unmodified forms copurify in the early steps and are separated by ion exchange chromatography (Figure 3) (which does not resolve the octanoylated and lipoylated forms). Fractions from the column containing the modified forms of the E2 domain (identified by their characteristic rapid gel electrophoretic mobility) were then submitted to electrospray mass spectroscopy (Figure 4). The modified proteins from a control experiment performed in parallel, except that no cosmid was added, were also analyzed. In the culture to which LipA activity was restored, there was a readily detectable conversion of the E2 domain modified with octanoic-<sub>15</sub> acid to a

![Figure 2. The Proposed Pathway of Lipoic Acid Synthesis](https://i.imgur.com/yourimageurl.png)

Figure 2. The Proposed Pathway of Lipoic Acid Synthesis

The rounded rectangle is an *E. coli* cell, and the ovals are the E2 domains of the 2-oxoacid dehydrogenases. The carboxyl group of the octanoate and lipoate moieties are in amide linkage (data not shown) to the ε-amino group of a specific domain lysine residue.

![Figure 3. Nondenaturing Gel Electrophoretic Characterization of Various HPLC Fractions of the E2-Domain Purification](https://i.imgur.com/yourimageurl.png)

Figure 3. Nondenaturing Gel Electrophoretic Characterization of Various HPLC Fractions of the E2-Domain Purification

CE, AP, and AS denote the crude extract, acid treatment pellet, and acid treatment supernatant, respectively, whereas apo denotes an apo-E2 domain standard. Selected fractions from the HPLC-ion exchange chromatography column are shown. All of the HPLC fractions of the upper gel are the apo form, whereas all of the fractions in the lower gel are modified species. The loss of the positive charge of the modified lysine ε-amino group accounts for the differences in gel mobility and column elution of the apo and modified species.

Chemistry & Biology
1296

No LipA                              LipA Restored
A.                                    B.
8928 D15-C8                          8927 D15-C8
8944                                 8987 D13-Lip
H15-C8                               8967
                                     8913
                                     H15-C8
C.                                    D.
8928 D15-C8                          8928 D15-C8
8944                                 8987 D13-Lip
8913                                 9004
H15-C8                               8944
                                     9020
                                     8913
                                     H15-C8
E.                                    F.
8929 D15-C8                          8928 D15-C8
8944                                 8988 D13-Lip
8913                                 9004
H15-C8                               8944
                                     9020
                                     8913
                                     H15-C8
G.                                    H.
8928 D15-C8                          8990 D13-Lip
8914                                 8975 H13-Lip
H15-C8                               8913
                                     8928
                                     D15-C8
MASS

Figure 4. Electrospray Mass Spectra of Purified Modified E2-Domain Fractions

Strain ZX137A having the genotype *lipA lipB fadE* (λ) was grown in the presence of octanoic-*d*<sub>15</sub> acid. The cells were then harvested, washed free of octanoic-*d*<sub>15</sub> acid, and resuspended in growth medium lacking octanoate acid (A–F) or medium containing unlabeled octanoate (G and H). After 1 hr of growth, the culture was split in half and LipA function was restored to one half. Modified E2 domains were purified as described in Experimental Procedures. H15-C8 and D15-C8 denote the hydrogen and deuterium (octanoic-*d*<sub>15</sub> acid) forms of octanoyl-E2 domain, respectively, whereas H15-Lip and D13-Lip denote the hydrogen and deuterium (lipoic-*d*<sub>13</sub> acid) forms of the lipoyl-E2 domain, respectively. The masses of the major peaks are given in the panels. In each pair of panels, the same fractions from the two HPLC separations (Figure 3) are compared for the LipA-containing and control cultures. (A), (C), and (E) are fractions from the parallel control culture in which LipA function was not restored. (B), (D), and (F) are fractions from the culture in which LipA function was restored by transduction with the *lipA* cosmid, pZX130. (A) and (B) are HPLC fraction 29, (C) and (D) are fraction 27, and (E) and (F) are fraction 31. In (G) and (H), unlabeled octanoic acid was added following removal of octanoic-*d*<sub>15</sub> acid, and the protein fractions analyzed were a pool of HPLC fractions 24–27. Note the increased levels of the H15-C8 species compared to (A)–(F) and the appearance of detectable levels of the H13-Lip species. The unlabelled peaks are ammonia (or occasionally sodium or potassium) adducts.

species of 60 additional mass units. This is the increase in mass (gain of two sulfur atoms of mass 32 and loss of two deuterium atoms of mass 2) expected for conversion of the *d*<sub>15</sub>-labeled octanoyl-E2 domain to the *d*<sub>13</sub>-labeled lipoyl domain. A variation of this experiment (Figures 4G and 4H) in which the octanoic-*d*<sub>15</sub> acid was removed by washing the cells and replaced with normal (nondeuterated) octanoate was also done and gave essentially the same result; the *d*<sub>15</sub>-labeled octanoyl-E2 domain was converted to *d*<sub>13</sub>-labeled lipoyl-E2 domain. Very similar results were obtained in several other experiments using octanoic-*d*<sub>15</sub> acid and in one experiment that used [1,2,3,4-<sup>13</sup>C]octanoic acid (data not shown). No labeled lipoylated domains were seen in the cultures in which LipA was not restored. Purified E2 domains were also treated with *Enterococcus faecalis* lipoamide [22] to release the lipoic acid in a free form. Only the E2 domains purified from the culture to which LipA activity was restored gave a product that supported growth of an *E. coli lipA* strain auxotrophic for lipoic acid (data not shown).

In Vivo Conversion of Octanoyl-Modified 2-Oxoacid Dehydrogenases to Lipoyl-2-Oxoacid Dehydrogenases

A more physiologically relevant experiment was also done in which we first accumulated the inactive octanoylated forms of the pyruvate and 2-oxoglutarate dehydrogenases (abbreviated as PDH and 2-OGDH, respectively) in a *lipB lipA* (λ) strain and tested for conversion of these enzymes to their active forms upon restoration of LipA function. The experimental protocol paralleled the E2 domain experiments, with several important modifications. These were as follows: (1) the *E. coli* strain lacked the E2 domain plasmid, (2) unlabeled octanoic acid was used in place of the deuterated acid, (3) the strain expressed phage T7 RNA polymerase, and (4) the *lipA* gene of the cosmid was driven by a phage T7 promoter. This strain was grown with octanoic acid supplementation, and expression of T7 RNA polymerase was induced. The octanoic acid was then removed, and the cells were returned to growth medium as above. Rifampicin was added to block *E. coli* RNA polymerase [23], and the cells were then transduced with the *lipA* cosmid. Rifampicin addition blocked expression of chromosomal genes such that no new dehydrogenase proteins could be synthesized during the period of *lipA* expression. Moreover, the *lipB* mutation would almost completely block attachment of endogenously synthesized octanoate moieties to the dehydrogenase proteins [4]. Therefore, any increase in activity would be due to activation of previously synthesized and octanoylated 2-oxoacid dehydrogenases by the restoration of LipA mediated by the rifampicin-insensitive T7 polymerase. In several repeats of this experiment, we found that extracts of cells to which LipA has not been restored were devoid of dehydrogenase activity, whereas extracts of LipA-restored cells accumulated active enzyme. For example, in the *lipA lipB fadE λDE3* strain ZX170 grown in the presence of octanoate, restoration of LipA by cosmid transduction resulted in PDH and 2-OGDH activities of 16.6 and 1.7 units/mg extract pro-

tein, respectively. Extracts of cultures to which LipA had not been restored (either in the presence or absence of growth with octanoate) had only background activity (<0.2 units/mg extract protein). In a modification of this experiment, we grew the *lipA fadE λDE3* strain ZX178 without exogenous octanoate such that the octanoate moieties would originate from fatty acid synthesis and become attached to the domain via LipB. Following addition of rifampicin and restoration of LipA function by cosmid transduction, an 2-OGDH activity of 3.5 units/mg extract protein was obtained upon LipA restoration (although no PDH activity was seen). When strain ZX178 was grown with octanoate supplementation, LipA restoration gave PDH and 2-OGDH activities of 6.7 and 15.5 units/mg extract protein, respectively. Only background activities were seen in the absence of LipA restoration with or without octanoate supplementation.

In Vitro Octanoylated-E2 Domain Is Favored over Octanoyl-ACP as a LipA Substrate

We developed two new assays in order to test the possibility that octanoylated-E2 domain is a substrate for LipA in vitro. We first examined formation of 5′-deoxyadenosine (5′-dAdo), the stable product resulting from the 5′-deoxyadenosyl radical expected to be produced in the reaction. Incubation of reduced LipA with apo-E2 or apo-PDH in the presence of LipB, octanoyl-ACP, and SAM results in formation of substoichiometric quantities of lipoylated-E2 domain or lipoylated-PDH, respectively [17]. To determine if sulfur insertion occurred concomitantly with the reductive cleavage of SAM, the fate of radiolabeled SAM was monitored during LipA turnover. When a stoichiometric excess of reduced LipA was incubated with \[^3\text{H}\]-[adenine]-labeled SAM, LipB, apo-E2 domain, and octanoyl-ACP, SAM was consumed (Figure 5). Moreover, a new radiolabeled species accumulated that comigrated with authentic 5′-dAdo upon HPLC separation. Attempts to confirm this species as 5′-dAdo by mass spectrometry were unsuccessful, despite detection of the appropriate parent ion in control samples of 5′-dAdo (see Discussion). Similar experiments conducted using partially purified PDH as acceptor were inconclusive due to nonspecific degradation of SAM.

To confirm that lipoylated-E2 domain was formed in the experiments above, the products of the reaction were also assayed utilizing a coupled enzyme assay (modified from [24]). Lipoylated-E2 domain and/or lipoylated-ACP species were detected by their ability to serve as a rate-limiting electron shuttle between NADH and oxidized glutathione in the presence of excess *E. coli* lipoamide dehydrogenase (see Experimental Procedures). The levels of lipoylated-E2 domain synthesized in LipA reactions were estimated by comparison to solutions containing known amounts of lipoylated E2 domain. The levels of lipoylated-E2 domain generated were within the detection limit of the assay (≥5-fold above background), but were inconsistent between preparations due to the extremely labile nature of the LipA protein. The synthesis of lipoylated-E2 domain occurred only when SAM was consumed, and SAM consumption was observed only in the presence of a lipoylation substrate (Table 1). From these data and the adenine ring-

---

**Figure 5. Consumption of Tritiated SAM in the Presence of Reduced LipA and Octanoylated-E2 Domain**

The assay products were separated by HPLC and detected with an in-line flow scintillation counter (see Experimental Procedures). The upper chromatogram shows the products of a reaction in which only LipA was omitted. The lower chromatogram shows the same reaction components in the presence of reduced LipA. See Table 1 for the assay protocol and components.

labeled \[^3\text{H}\]-SAM cleavage results (Figure 5), we calculate that approximately 10–40 molecules of SAM were consumed for each molecule of lipoylated-E2 domain synthesized by LipA (data not shown). A possible explanation for the lack of coupling between reductive cleavage of SAM and sulfur insertion is discussed below. The stoichiometries of lipoylated-E2 domain formed and SAM consumed relative to LipA could not be calculated, since the fraction of LipA that was active in the two reactions is unknown [17].

The new coupled assay for lipoylated E2 domain plus the use of SAM consumption (and concomitant appearance of a radiolabeled species that comigrated with 5′-dAdo) as an indicator of LipA turnover provided the means to investigate the specificity of the octanoyl-protein acceptor substrate of LipA. When purified octanoyl-E2 domain was incubated with reduced LipA and \[^3\text{H}\]-SAM, consumption of SAM, appearance of a new radiolabeled species comigrating with authentic 5′-dAdo (Figure 5), and formation of a lipoylated species were observed (Table 1). Upon replacement of octanoyl-E2 with an equimolar amount of octanoyl-ACP, the level of the species comigrating with 5′-dAdo was 50-fold lower following a 12-fold longer incubation (the level of nonspecific degradation of SAM was very similar to that seen in the shorter incubations). No lipoylated products were detected in enzymatic assays, although the assay could detect only 5-fold less lipoylated product than that formed in the octanoyl-E2 reactions. These results suggested that LipA strongly prefers octanoyl-E2 domain to octanoyl-ACP as the in vitro substrate.

Chemistry & Biology
1298

Table 1. Octanoyl-ACP and Octanoyl-E2 Domains as Substrates for Sulfur Insertion

| Reaction Components<sup>a</sup> | Reaction Time (min) | SAM Consumed<sup>b</sup> (pmol) | Lip-E2 Detected<sup>c</sup> |
|-------------------------------|---------------------|------------------------------|---------------------------|
| Nonreduced LipA, SAM          | 20                  | <5                           | no                        |
| Reduced LipA, SAM             | 20                  | <5                           | no                        |
| Reduced LipA, SAM + Octanoyl-E2 | 20                  | 1050                         | yes                       |
| Reduced LipA, SAM + Octanoyl-ACP | 20                  | <5                           | ND<sup>d</sup>            |
| Reduced LipA, SAM + Octanoyl-ACP | 240                 | 20                           | ND                        |
| Reduced LipA, SAM + Lipoyl-E2 | 20                  | <5                           | yes                       |
| Nonreduced LipA, SAM + Octanoyl-E2 | 20                  | <5                           | no                        |

<sup>a</sup>LipA reactions were conducted as described previously [16] except reaction components were changed as given. Briefly, anaerobic solutions of 700 μM LipA were reduced by addition of sodium dithionite (1 mM final concentration) followed by incubation on ice for 30 min. Tritium-labeled SAM (26 mCi/mmol) was added to a final concentration of 30 μM. Potential substrates (either octanoyl-E2, octanoyl-ACP, or lipoyl-E2) were added to final concentrations of 70 μM (final volume of 40 μL) and allowed to incubate for the times given. Under these conditions, SAM is the limiting reagent. Reaction mixtures were heated to 65 °C for 5 min, centrifuged for 1 min at 14,000 × g, and the radiolabeled species in the supernatant were separated and quantified as described in Experimental Procedures.

<sup>b</sup>The amounts of SAM consumed in the reactions with reduced LipA were calculated from the quench-corrected flow scintillation counter data. Quantitation was performed by measuring the peak area of the species comigrating with authentic 5′-dAdo.

<sup>c</sup>Lip-E2 was detected using the lipoamide dehydrogenase/glutathione assay (see Experimental Procedures).

<sup>d</sup>ND, not done.

The possibility that octanoyl-ACP was a substrate for sulfur insertion by LipA only in the presence of LipB and/or apo-E2 domain was addressed by altering the reaction conditions to make acyl transfer to apo-E2 domain rate limiting. When an excess of apo-E2 domain (320 pmol), reduced LipA (5000 pmol), and octanoyl-ACP (200 pmol) were incubated with varying concentrations of LipB for a fixed time, the amount of SAM consumed decreased from 211 pmol with 9.6 pmol of LipB to 13 pmol with 0.096 pmol of LipB, suggesting that acyl transfer to apo-E2 preceded SAM consumption and that octanoyl-ACP is not an efficient substrate for LipA under these conditions. Moreover when LipB was replaced with an equimolar amount of LplA, which is much less active in acyl transfer [6], the rate of SAM consumption slowed precipitously (data not shown).

To directly demonstrate that LipA catalyzes the formation of lipoylated-E2 domain from octanoylated-E2 domain, the distribution of E2 domain species following a LipA reaction were isolated and analyzed by electrospray mass spectrometry. A species with a mass consistent with lipoylated-E2 domain was observed in LipA reactions (data not shown). Furthermore, treatment of the purified reaction product with *E. faecalis* lipoamidease [22] gave a product that supported growth of a *lipA* strain of *E. coli* auxotrophic for lipoic acid. Together, these data are consistent with octanoylated-E2 domain being the preferred substrate for sulfur insertion by LipA in vitro.

LipA Converts Octanoylated-PDH to Lipoylated-PDH In Vitro

Although octanoyl-E2 domain serves as a more effective in vitro substrate for LipA than octanoyl-ACP, E2 domain is not a physiologically relevant substrate. We therefore purified octanoylated PDH, incubated the octanoyl-PDH with reduced LipA in the presence of SAM, and detected active (i.e., lipoylated) PDH in proportion to the amount of LipA added to the reaction (Table 2). Incubation for longer periods of time did not increase the amount of active PDH formed. Omission of LipA, reductant, or octanoyl-PDH resulted in background PDH activity. Moreover, no PDH activity was observed upon addition of LipB to a sample lacking LipA, indicating a lack of contaminating lipoyl-ACP. Addition of high levels of lipoyl-ACP or lipoate showed small amounts of PDH activity (<6% of that seen with reduced LipA), suggesting that the PDH preparations may contain low levels of LipB and/or LplA. Omission of SAM also resulted in low levels of PDH activity, suggesting that this cofactor may also be present within the PDH preparation. These results indicate that octanoylated PDH is a viable substrate for in vitro sulfur insertion by LipA.

Discussion

Octanoic acid supports growth of *lipB* null mutant strains, and growth on this supplement requires function of both the *lplA* and *lipA* genes (Figure 1), which led us to test octanoylated 2-oxoacid dehydrogenase species as LipA substrates. Our in vivo results demonstrate that

Table 2. Octanoyl-PDH as a Substrate for Sulfur Insertion

| Reaction Components<sup>a</sup> | LipA (nmol) | PDH Activity (U/ml) |
|-------------------------------|-------------|--------------------|
| Reduced LipA, SAM, octanoyl-PDH | 16          | 11.7               |
| Reduced LipA, SAM, octanoyl-PDH | 10          | 11.7               |
| Reduced LipA, SAM, octanoyl-PDH | 3.3         | 6.3                |
| Reduced LipA, SAM, octanoyl-PDH | 1.3         | 1.9                |
| SAM, octanoyl-PDH              | 0           | 0.1                |
| Reduced LipA, octanoyl-PDH     | 240         | 0.8                |
| SAM, octanoyl-PDH, 1 nmol LipB | 0           | 0.1                |
| SAM, octanoyl-PDH, 4 nmol Lip-ACP | 0           | 0.6                |
| SAM, octanoyl-PDH, 40 nmol lipoic acid | 0           | 0.4                |

<sup>a</sup>LipA reactions were conducted as described previously [16] except reaction components were changed as described here. Briefly, anaerobic solutions of 700 μM LipA were reduced by addition of sodium dithionite (1 mM final concentration) followed by incubation on ice for 30 min. SAM and octanoyl-PDH were added to final concentrations of 30 μM and 0.4 mg/ml, respectively (under these conditions, SAM is limiting). PDH activity assays were conducted as previously described [16].

preformed octanoyl-E2 domain or octanoyl-PDH was converted to the lipoyl species upon the subsequent introduction of a functional *lipA* gene. It should be noted that in 1993, Reed and Cronan [9] reported a version of the latter experiment in which no conversion of a putative octanoylated species of 2-OGDH to the lipoylated form was detected. However, this early experiment had several weaknesses. First, the assay used for conversion of octanoyl-E2 to lipoyl-E2 was the decreased mobility of the lipoyl species relative to the octanoylated form upon denaturing gel electrophoresis. As noted by the authors, this was an insensitive assay since the change in mobility was small, and appreciable conversion would be required for detection. Moreover, from our present data it seems probable that the faster migrating 2-OGDH E2 species of Reed and Cronan [9] consisted mainly of the apo form of the protein. Reed and Cronan [9] did not supplement their cultures with octanoate, and we observed only a small increase in 2-OGDH activity upon LipA restoration unless the *lipA* cells had been grown with octanoate supplementation. Finally, in the 1993 experiments LipA was restored by infection with a recombinant λ phage carrying the *lipA* and *lipB* genes, and the robust phage infection that proceeded in these cultures may have compromised transcription of the weakly expressed *lipA* gene.

The in vitro results indicate that SAM cleavage is almost completely dependent upon the presence of octanoyl-E2 domain or the proteins (octanoyl-ACP, LipB, and apo-E2 domain) required for synthesis of octanoyl-E2 (Table 1). In the latter case, the apparent rate of SAM cleavage depends on the rate of transfer of octanoate from octanoyl-ACP to octanoyl-E2. Inactive octanoyl-PDH can be converted to the active lipoylated species upon incubation with reduced LipA and SAM. These data demonstrate that LipA can use octanoylated-E2 domains as a substrate. It should be noted that in our prior work we reported that purified LipA did not convert octanoylated-PDH to the active lipoylated enzyme [17]. This result was likely due to the fact that our previous experiments utilized crude octanoylated PDH preparations that had been octanoylated in vitro. The low efficiency of the octanoylation reaction did not allow formation of sufficient octanoyl-PDH to detect the lipoyl-PDH formed. The in vivo experiments plus the new in vitro assays that allow more direct analyses of the reaction have convinced us that the sulfur insertion into the octanoyl moiety of octanoyl-E2 domain is a bona fide reaction. However, it is not clear that this is the only route of protein lipoylation. Octanoyl-ACP elicits reductive cleavage of SAM (Table 1) at a low rate, but any lipoyl-ACP formed was below the detection limit of the assay. Moreover, given the high proportion of LipA protein in our preparations defective in the sulfur insertion step and/or reductive cleavage of SAM, the weak activity with octanoyl-ACP could be misleading. Indeed, we (S. Jordan and J.E.C., unpublished data) have observed accumulation of lipoyl-ACP in cells expressing various altered acceptor domains. It remains possible that the conversion of octanoyl-E2 domain to the lipoyl species is a type of repair pathway rather than a biosynthetic pathway. The LipA iron-sulfur centers are exquisitely sensitive to oxidation in vitro [16–18], and thus LipA

seems likely to be inactivated by oxidative stress in vivo. If so, then the octanoylated forms of the 2-oxoacid dehydrogenases would accumulate as seen in *lipA* mutant strains [4, 7]. Following recovery from oxidative stress and restoration of LipA activity, sulfur insertion into the accumulated octanoylated enzymes would rapidly restore enzyme activity. Alternatives to this repair pathway would be to remove the octanoyl moiety and replace it with a lipoyl group or to disassemble the enzyme complexes such that the octanoylated-E2 subunit could be replaced with a new E2 subunit able to accept lipoate. *E. coli* lacks an amidase such as the lipoamidase of *E. faecalis* [22] capable of removing the octanoyl group from the domains, and no 2-oxoacid dehydrogenase disassembly pathway is known. Therefore, the alternative to LipA-dependent sulfur insertion into octanoyl-E2 domains would seem to be synthesis of new dehydrogenases. This would not only be a considerable metabolic expense to the cell, but the expense would have to be paid under conditions where the cells lack the major pathway of energy generation, the citric acid cycle. Moreover, since the citric acid cycle is a major source of amino acid carbon, the synthesis of the dehydrogenase proteins would be hindered by a limited supply of amino acids as well as of ATP. We are currently testing the effects of oxidative stress on lipoic acid synthesis.

The reductive cleavage of SAM by a reduced [4Fe-4S] cluster to generate the highly reactive 5′-deoxyadenosyl radical seems likely to be a tightly regulated process in the LipA reaction. We therefore believe that our observation of reductive cleavage of SAM that is uncoupled from sulfur insertion is likely due to the presence of large amounts of damaged LipA (e.g., oxidative damage to the Fe-S cluster, misfolding of the protein, etc.) in our preparations. Previously, we found that our preparations of LipA produce less than 0.01 mol of lipoylated-PDH per mol of LipA protein [17]. Therefore, although most of the LipA protein may be unable to catalyze sulfur insertion, these damaged proteins may be competent to reductively cleave SAM but only after binding octanoylated-E2 domain or octanoyl-ACP. In this regard, LipA seems to differ from BioB in that the latter enzyme cleaves SAM in the absence of the sulfur acceptor, de-thiobiotin [25].

Our inability to conclusively identify 5′-dAdo as a LipA reaction product by LC/MS may be a consequence of this uncoupled generation of 5′-deoxyadenosyl radicals. These species are highly reactive and may react intramolecularly or intermolecularly. During photolysis of coenzyme B₁₂, a deoxyadenosyl radical is generated without an obvious acceptor. Analysis of the photolysis reaction products showed formation of at least two noncobalamin species, adenosine 5′-aldehyde and 8-5′-cyclic adenosine [26]. Although our HPLC results showed only a single peak that comigrated with authentic 5′-deoxyadenosine, we cannot rule out the presence of comigrating contaminants such as these modified adenosines. Additional experiments are needed to characterize these reaction products.

### Significance

Lipoic acid is required for aerobic metabolism in bacteria, fungi, plants, and animals and also plays a key

role in single carbon metabolism and photorespiration. Genes encoding proteins homologous to the *E. coli* LipA, LipB, and LplA proteins are found in all of the above organisms, and thus an understanding of the pathways of lipoic acid synthesis and attachment should have general significance throughout much of biology. Moreover, our data demonstrate a previously undescribed type of biosynthetic pathway in which an inactive precursor of a cofactor is attached to an apoprotein and subsequently activated in a separate enzymatic step.

### Experimental Procedures

#### Bacterial Strains and Growth Media
The *E. coli* K-12 strains derived from the wild-type strain JK1 with the designations KER [10], TVB [11], and TM [3] were described in the reference following the designation. Strain ZX133 was obtained by transduction of strain KER296 to tetracycline resistance with phage P1vir grown on strain TVB98. Strain ZX133 was made a λ lysogen by either the method of Henry and Cronan [27] or by use of the λDE3 Lysogenization Kit (Novagen) to give strains ZX136 and ZX170, respectively. Strain KER296 was made a λ lysogen using the λDE3 protocol to give strain ZX178. Strains ZX136, ZX178, and TM131 were transformed with plasmid pGS331 to form strains ZX137, ZX180, and ZX181, respectively. Plasmid pGS331 expresses an 85 residue recombinant lipoate-accepting domain under control of the tac promoter [7]. The strains used for in vivo cosmid packaging were ZX166, ZX177, and ZX179. Strains ZX166 was obtained by transforming plasmid pZX130 into strain ML12. Strain CY1491 [22] was transformed with plasmids pZX130 and pZX136 to form strains ZX179 and ZX177, respectively. The media used for bacterial growth were described previously [3, 10, 11, 22].

#### Plasmid Constructions
An 865 bp BamHI gentamycin resistance cassette from plasmid p34S-Gm [28] was ligated into the BamHI site of vector pSU19, a derivative of pSU2719 [28], to form plasmid pZX124. Plasmid pZX125 is a cosmid containing the 501 bp PstI λ cohesive end sequence (*cosλ*) from plasmid pCY128 [27] inserted into the PstI site of plasmid pZX124. The *lipA* gene of *Salmonella typhi* was amplified by PCR using primers st-*lipAF* GGCCCTTATCAATGATA TACTGCCG and st-*lipAR* GCGCAATGCGCCTTGCTCCGC. The amplified fragment was subcloned into pCR2.1 (Invitrogen) to give plasmid pZX127. A 1.2 kb EcoRI fragment from plasmid pZX127 that carried the entire *lipA* gene was ligated into the EcoRI sites of plasmids pZX125 and pSU19 to from plasmids pZX130 and pZX131, respectively, which placed the *lipA* coding sequence under control of the *lac* promoter. A 201 bp Pvull-SstI fragment from plasmid pGEM-7Zf(+) (Promega) which contains a phage T7 promoter was ligated between the Smal and SstI sites of plasmid pZX131 to place the *lipA* gene under the control of the T7 promoter in the resulting plasmid, pZX135. A 501 bp *cosλ* PstI fragment of plasmid pCY128 was then inserted into the PstI site of plasmid pZX135 to convert the plasmid to a cosmid called pZX136. Induction of cosmid packaging and preparation of concentrated particle preparations by chloroform lysis of concentrated cell suspensions was done as described elsewhere [21].

#### Expression of Lipoyl Domains and Labeling of Cultures with Octanoic-$d_{15}$ Acid
Plasmid pGS331 expresses an 85 residue recombinant lipoate-accepting domain under control of the tac promoter [7, 29]. Strains containing plasmid pGS331 were grown in L broth or in minimal medium M9 supplemented with glucose (0.4%) or maltose (0.2%), sodium acetate (5 mM), sodium succinate (5 mM), thiamine (1 μg/ml), vitamin-free casein hydrolysate (0.1%), MgSO₄ (10 mM), and appropriate antibiotics. Cultures were incubated at 37°C with shaking overnight and diluted 20-fold into fresh medium containing 0.025 mM octanoic-$d_{15}$ acid. Cultures were grown at 37°C with shaking until reaching an A₆₀₀ of 0.6 (rich medium) or 0.3 (minimal medium).

E2-domain synthesis was induced by adding IPTG to a final concentration of 1 mM, and the cells were labeled by addition of octanoic-$d_{15}$ acid or [1,2,3,4-¹³C]octanoic acid (Cambridge Isotope Laboratories, isotopically enriched to 98.5% and 99%, respectively) to a final concentration of 50 μM. Expression and labeling was allowed to proceed for 4 hr at 37°C with shaking. The cells were harvested by centrifugation at 12,000 × g at 4°C for 10 min. In the experiment to study protein-bound lipoic acid synthesis, the cells were harvested, washed three times with an equal volume of the above medium lacking octanoic acid, and resuspended in the same medium. The culture was then incubated at 37°C for 1 hr before the culture was divided in half. Packaged cosmid was added at a multiplicity of 5 transducing particles per cell to one of the cultures; the other culture was not transduced. The cells were harvested after 6 hr or as given in the figure legends. In a variation of this experiment, unlabeled octanoate (50 μM) was added following removal of the labeled octanoate.

#### Purification and Characterization of E2 Domains
The purification of E2 domains in the in vivo experiments was modified from that of Ali and Guest [7, 29]. Cell paste (about 10¹¹ cells) was suspended in 10 ml of 20 mM sodium phosphate buffer (pH 7.0) containing 2 mM EDTA and passed through a French pressure cell twice at 20,000 psi. The resulting lysate was centrifuged at 38,000 × g for 40 min to remove unbroken cells and cell debris. The pH of the supernatant was lowered to 4.0 with 1 M HCl, and insoluble material was removed by centrifugation (38,000 × g for 20 min) before the pH was returned to 7.0 with 1 M NaOH followed by dialysis against freshly prepared 10 mM ammonium acetate buffer (pH 5.0). The dialyzed samples were subjected to anion-exchange chromatography on a 0.831 ml POROS QE (4.60 mm × 50 mm) column with a 10–600 mM gradient of ammonium acetate (pH 5.0). Protein fractions were analyzed on 20% native polyacrylamide gels followed by Coomassie Blue R-250 staining. Protein concentrations were determined using the BioRad Bradford protein assay standardized against bovine serum albumin. The column fractions were dialyzed versus 2 mM ammonium acetate buffer before protein samples of 4.5 μg were dried under vacuum and analyzed by mass spectroscopy. The samples were dissolved in a 1:1 mixture of water and acetonitrile containing 0.1% formic acid and analyzed with a Micromass Q-TOF Ultima API electrospray mass spectrometer. Purification of octanoyl-E2 domains expressed in strain KER176 (*lipA*) transformed with plasmid pGS331 was performed as described previously [17]. Octanoyl-PDH was purified from strain KER176 (*lipA*) [30]. The sources of the octanoylated proteins were cells were grown in glucose minimal E supplemented with sodium succinate and sodium acetate as described previously [17].

#### In Vivo Activation of 2-Oxoacid Dehydrogenases
The strains were grown in minimal M9 medium supplemented with 0.2% maltose, 5 mM acetate, 5 mM succinate, and 1 mM isopropyl-thiogalactoside to induce the phage T7 RNA polymerase encoded on the λDE3 prophage. When added, the final concentration of octanoic acid was 0.05 mM. The *lipA* cosmid to cell ratio in the transduced cultures was 5 transducing particles/cell. The dehydrogenase activities of cell-free extracts were measured 4 hr after transduction. The pyruvate and 2-oxoglutarate dehydrogenase activities are given as units per mg extract protein, where one unit of activity is 1 μmol of 3-acetylpyridine adenine dinucleotide reduced per minute [30].

#### Synthesis, Purification, and Analysis of Tritiated SAM
Radiolabeled [2,8-³H]-SAM was synthesized from the ammonium salt of [2,8-³H]-adenosine 5′-triphosphate (Amersham, 42 Ci/mmol) using a crude preparation of S-adenosylmethionine synthetase from an overexpressing *E. coli* strain [31] (the kind gift of G.D. Markham). Crude lysates were processed through the ammonium sulfate precipitation described [31]. The SAM synthesis protocol was based upon that described in [31] and further elaborated in [32]. SAM (13 μCi with a specific activity of 155 mCi/mmol and >98% radiochemical purity by HPLC) was obtained. SAM and SAM-derived products were separated by HPLC on a C18-ODS column using a 20 min gradient of 0.1% trifluoroacetic acid to 28% acetonitrile in

Lipoic Acid Synthesis
1301

0.1% trifluoroacetic acid at 1 ml/min with detected by absorbance at 260 nm or by flow-liquid scintillation counting using a Packard Instruments in-line flow scintillation counter. When necessary, quantitation of scintillation counter output and quench correction was conducted using commercial \({ }^{3} \mathrm{H}-\mathrm{SAM}\) preparations of known specific activity.

### Coupled Lipoamide Dehydrogenase (E3) Assay of Lipoylated Protein Products

Lipoylated E2 products of LipA reactions were detected using a modification of the protein-bound lipoate assay described by Akiba et al. [24]. In this assay, lipoylated E2 serves as a rate-limiting electron shuttle between NADH and oxidized glutathione, provided that lipoamide dehydrogenase is present in excess. The products of in vitro LipA reactions with hexahistidine-tagged LipA (purified and assayed as described previously [17]) were heated to \(65^{\circ} \mathrm{C}\) for 5 min, and denatured protein was removed by centrifugation. Various amounts of the supernatant were added to an anaerobic solution containing \(5 \mathrm{mM}\) oxidized glutathione, \(200 \mu \mathrm{M}\) NADH, \(2 \mu \mathrm{g} E\). coli lipoamide dehydrogenase, and sufficient \(500 \mathrm{mM}\) sodium phosphate (pH 8.0) to give a final volume of \(0.5 \mathrm{ml}\). The rate of change in absorbance at \(340 \mathrm{~nm}\) was measured within the linear range of the reaction (0–10 \(\mu \mathrm{M}\) lipoyl-E2 domain). The amount of lipoylated species in the LipA reaction mixtures was determined by comparison to a standard curve generated from similarly treated samples containing known amounts of lipoylated-E2 domain (extinction coefficient of \(2440 \mathrm{M}^{-1} \mathrm{cm}^{-1}\) at \(329 \mathrm{~nm}\) ).

### Acknowledgments

This work was supported by grant AI15650 from the NIH (J.E.C.) and the Howard Hughes Medical Institute (M.A.M.).

Received: August 11, 2003  
Revised: September 26, 2003  
Accepted: September 29, 2003  
Published: December 19, 2003  

### References

1. Reed, L.J., and Hackert, M.L. (1990). Structure-function relationships in dihydrolipoamide acyltransferases. J. Biol. Chem. 265, 8971–8974.
2. Nawa, H., Brady, W.T., Koike, M., and Reed, L.J. (1960). Studies on the nature of protein-bound lipoic acid. J. Am. Chem. Soc. 82, 896–903.
3. Morris, T.W., Reed, K.E., and Cronan, J.E., Jr. (1994). Identification of the gene encoding lipoate-protein ligase A of Escherichia coli. Molecular cloning and characterization of the \(l p l A\) gene and gene product. J. Biol. Chem. 269, 16091–16100.
4. Morris, T.W., Reed, K.E., and Cronan, J.E., Jr. (1995). Lipoic acid metabolism in Escherichia coli: the \(l p l A\) and \(l i p B\) genes define redundant pathways for ligation of lipoyl groups to apoprotein. J. Bacteriol. 177, 1–10.
5. Jordan, S.W., and Cronan, J.E., Jr. (1997). A new metabolic link. The acyl carrier protein of lipid synthesis donates lipoic acid to the pyruvate dehydrogenase complex in Escherichia coli and mitochondria. J. Biol. Chem. 272, 17903–17906.
6. Jordan, S.W., and Cronan, J.E., Jr. (2003). The Escherichia coli \(l i p B\) gene encodes lipoyl (octanoyl)-acyl carrier protein:protein transferase. J. Bacteriol. 185, 1582–1589.
7. Ali, S.T., Moir, A.J., Ashton, P.R., Engel, P.C., and Guest, J.R. (1990). Octanoylation of the lipoyl domains of the pyruvate dehydrogenase complex in a lipoyl-deficient strain of Escherichia coli. Mol. Microbiol. 4, 943–950.
8. Herbert, A.A., and Guest, J.R. (1968). Biochemical and genetic studies with lysine+methionine mutants of Escherichia coli: lipoic acid and \(\alpha\)-ketoglutarate dehydrogenase-less mutants. J. Gen. Microbiol. 53, 363–381.
9. Reed, K.E., and Cronan, J.E., Jr. (1993). Lipoic acid metabolism in Escherichia coli: sequencing and functional characterization of the \(l i p A\) and \(l i p B\) genes. J. Bacteriol. 175, 1325–1336.
10. Vanden Boom, T.J., Reed, K.E., and Cronan, J.E., Jr. (1991).
11. Sulo, P., and Martin, N.C. (1993). Isolation and characterization of \(L I P 5\). A lipoate biosynthetic locus of Saccharomyces cerevisiae. J. Biol. Chem. 268, 17634–17639.
12. Yasuno, R., and Wada, H. (2002). The biosynthetic pathway for lipoic acid is present in plastids and mitochondria in Arabidopsis thaliana. FEBS Lett. 517, 110–114.
13. Morikawa, T., Yasuno, R., and Wada, H. (2001). Do mammalian cells synthesize lipoic acid? Identification of a mouse cDNA encoding a lipoic acid synthase located in mitochondria. FEBS Lett. 498, 16–21.
14. Hayden, M.A., Huang, I., Bussiere, D.E., and Ashley, G.W. (1992). The biosynthesis of lipoic acid. Cloning of \(l i p\), a lipoate biosynthetic locus of Escherichia coli. J. Biol. Chem. 267, 9512–9515.
15. Marquet, A. (2001). Enzymology of carbon-sulfur bond formation. Curr. Opin. Chem. Biol. 5, 541–549.
16. Busby, R.W., Schelvis, J.P.M., Yu, D.S., Babcock, G.T., and Marletta, M.A. (1999). Lipoic acid biosynthesis: LipA is an iron-sulfur protein. J. Am. Chem. Soc. 121, 4706–4707.
17. Miller, J.R., Busby, R.W., Jordan, S.W., Cheek, J., Henshaw, T.F., Ashley, G.W., Broderick, J.B., Cronan, J.E., Jr., and Marletta, M.A. (2000). Escherichia coli LipA is a lipoyl synthase: in vitro biosynthesis of lipoylated pyruvate dehydrogenase complex from octanoyl-acyl carrier protein. Biochemistry 39, 15166–15178.
18. Ollagnier-De Choudens, S., Sanakis, Y., Hewitson, K.S., Roach, P., Baldwin, J.E., Munck, E., and Fontecave, M. (2000). Iron-sulfur center of biotin synthase and lipoate synthase. Biochemistry 39, 4165–4173.
19. Sofia, H.J., Chen, G., Hetzler, B.G., Reyes-Spindola, J.F., and Miller, N.E. (2001). Radical SAM, a novel protein superfamily linking unresolved steps in familiar biosynthetic pathways with radical mechanisms: functional characterization using new analysis and information visualization methods. Nucleic Acids Res. 29, 1097–1106.
20. Shaw, N.M., Birch, O.M., Tinschert, A., Venetz, V., Dietrich, R., and Savoy, L.A. (1998). Biotin synthase from Escherichia coli: isolation of an enzyme-generated intermediate and stoichiometry of S-adenosylmethionine use. Biochem. J. 330, 1079–1085.
21. Cronan, J.E. (2003). Cosmid-based system for transient expression and total transcriptional control of Escherichia coli genes. J. Bacteriol., 185, 6522–6529.
22. Stepp, L.R., Bleile, D.M., McRorie, D.K., Pettit, F.H., and Reed, L.J. (1981). Use of trypsin and lipoamide to study the role of lipoic acid moieties in the pyruvate and \(\alpha\)-ketoglutarate dehydrogenase complexes of Escherichia coli. Biochemistry 20, 4555–4560.
23. Studier, F.W., and Moffatt, B.A. (1986). Use of bacteriophage T7 RNA polymerase to direct selective high-level expression of cloned genes. J. Mol. Biol. 189, 113–130.
24. Akiba, S., Matsugo, S., Packer, L., and Konishi, T. (1998). Assay of protein-bound lipoic acid in tissues by a new enzymatic method. Anal. Biochem. 258, 299–304.
25. Ollagnier-de Choudens, S., Sanakis, Y., Hewitson, K.S., Roach, P., Munck, E., and Fontecave, M. (2002). Reductive cleavage of S-adenosylmethionine by biotin synthase from Escherichia coli. J. Biol. Chem. 277, 13449–13454.
26. Hogenkamp, H.P.C. (1962). The identification of a nucleoside derived from coenzyme B12. J. Biol. Chem. 236, 3097–3101.
27. Henry, M.F., and Cronan, J.E., Jr. (1991). Direct and general selection for lysogens of Escherichia coli by phage lambda recombinant clones. J. Bacteriol. 173, 3724–3731.
28. Martinez, E., Bartolome, B., and de la Cruz, F. (1988). pACYC184-derived cloning vectors containing the multiple cloning site and lacZ alpha reporter gene of pUC8/9 and pUC18/19 plasmids. Gene 68, 159–162.
29. Ali, S.T., and Guest, J.R. (1990). Isolation and characterization of lipoylated and unlipoylated domains of the E2p subunit of the pyruvate dehydrogenase complex of Escherichia coli. Biochem. J. 271, 139–145.

Chemistry & Biology
1302

30. Jordan, S.W., and Cronan, J.E., Jr. (1997). Biosynthesis of lipoic acid and posttranslational modification with lipoic acid in *Escherichia coli*. Methods Enzymol. 279, 176–183.

31. Markham, G.D., Hafner, E.W., Tabor, C.W., and Tabor, H. (1983). S-adenosylmethionine synthetase (methionine adenosyltransferase) (*Escherichia coli*). Methods Enzymol. 94, 219–222.

32. Park, J., Tai, J., Roessner, C.A., and Scott, A.I. (1996). Enzymatic synthesis of S-adenosyl-L-methionine on the preparative scale. Bioorg. Med. Chem. 4, 2179–2185.
