The Plant Cell, Vol. 16, 544–554, February 2004, www.plantcell.org © 2004 American Society of Plant Biologists

# The Arabidopsis thaliana REDUCED EPIDERMAL FLUORESCENCE1 Gene Encodes an Aldehyde Dehydrogenase Involved in Ferulic Acid and Sinapic Acid Biosynthesis

Ramesh B. Nair,$^{1}$ Kristen L. Bastress, Max O. Ruegger,$^{2}$ Jeff W. Denault, and Clint Chapple$^{3}$

Department of Biochemistry, Purdue University, West Lafayette, Indiana 47907

Recent research has significantly advanced our understanding of the phenylpropanoid pathway but has left in doubt the pathway by which sinapic acid is synthesized in plants. The reduced epidermal fluorescence1 (ref1) mutant of *Arabidopsis thaliana* accumulates only 10 to 30% of the sinapate esters found in wild-type plants. Positional cloning of the *REF1* gene revealed that it encodes an aldehyde dehydrogenase, a member of a large class of NADP⁺-dependent enzymes that catalyze the oxidation of aldehydes to their corresponding carboxylic acids. Consistent with this finding, extracts of *ref1* leaves exhibit low sinapaldehyde dehydrogenase activity. These data indicate that *REF1* encodes a sinapaldehyde dehydrogenase required for sinapic acid and sinapate ester biosynthesis. When expressed in *Escherichia coli*, *REF1* was found to exhibit both sinapaldehyde and coniferaldehyde dehydrogenase activity, and further phenotypic analysis of *ref1* mutant plants showed that they contain less cell wall-esterified ferulic acid. These findings suggest that both ferulic acid and sinapic acid are derived, at least in part, through oxidation of coniferaldehyde and sinapaldehyde. This route is directly opposite to the traditional representation of phenylpropanoid metabolism in which hydroxycinnamic acids are instead precursors of their corresponding aldehydes.

## INTRODUCTION

Plants produce several important secondary metabolites from Phe via the phenylpropanoid pathway, including lignin, lignans, flavonoids, tannins, salicylic acid, and hydroxycinnamic acid esters. These metabolites perform a variety of functions in plants, including roles in mechanical strength, pollen viability, pest deterrence, disease resistance, UV protection, and resistance to biotic and abiotic stresses (Lewis and Yamamoto, 1990; van der Meer et al., 1992; Landry et al., 1995; Dixon et al., 1996, 2002; Dempsey et al., 1999). *Arabidopsis thaliana* and other members of the Brassicaceae accumulate the hydroxycinnamic acid esters sinapoylmalate in leaves and sinapoylcholine in seeds (Chapple et al., 1992). Sinapoylmalate, a UV-fluorescent compound, accumulates in the upper leaf epidermis and causes wild-type *A. thaliana* leaves to appear blue-green under UV light. By contrast, the sinapoylmalate-deficient ferulic acid hydroxylase1 (*fah1*) mutant appears red under UV light because of the fluorescence of chlorophyll (Chapple et al., 1992). This phenotype was used to identify a series of *A. thaliana* mutants that showed a reduced epidermal fluorescence (*ref*) phenotype

$^{1}$ Current address: Pioneer Hi-Bred International, 7300 NW 62nd Avenue, Johnston, IA 50131.  
$^{2}$ Current address: Dow AgroSciences, 9330 Zionsville Road, Indianapolis, IN 46268.  
$^{3}$ To whom correspondence should be addressed. E-mail chapple@purdue.edu; fax 765-496-7213.  

The author responsible for distribution of materials integral to the findings presented in this article in accordance with the policy described in the Instructions for Authors (www.plantcell.org) is: Clint Chapple (chapple@purdue.edu). Article, publication date, and citation information can be found at www.plantcell.org/cgi/doi/10.1105/tpc.017509.

(Ruegger and Chapple, 2001; Franke et al., 2002b). The analysis of several of these *ref* mutants (Franke et al., 2002a, 2002b; Hemm et al., 2003), as well as other mutants defective in sinapate ester accumulation (Chapple et al., 1992; Lehfeldt et al., 2000; Shirley et al., 2001), has led to an improved understanding of the phenylpropanoid pathway and its interactions with other pathways of secondary metabolism.

Recent work on the phenylpropanoid pathway has shown that the traditional view of lignin biosynthesis is incorrect (Humphreys and Chapple, 2002). Although the hydroxylation and methylation reactions of the pathway were long thought to occur at the level of the free hydroxycinnamic acids (Figure 1), it now seems clear that the enzymes catalyzing phenylpropanoid 3-hydroxylation and 3-O-methylation reactions use shikimate and CoA conjugates as substrates (Zhong et al., 1998; Guo et al., 2001; Parvathi et al., 2001; Schoch et al., 2001; Franke et al., 2002a, 2002b), respectively, and that 5-hydroxylation and 5-O-methylation occur at the level of the hydroxycinnamaldehydes and hydroxycinnamyl alcohols (Humphreys et al., 1999; Osakabe et al., 1999; Li et al., 2000a; Parvathi et al., 2001, Goujon et al., 2003).

Considering that the optimal substrates for ferulate 5-hydroxylase (F5H) and caffeic acid/5-hydroxyferulic acid O-methyltransferase (COMT) are hydroxycinnamyl aldehydes and hydroxycinnamyl alcohols and that both of these enzymes are required for sinapate ester synthesis (Chapple et al., 1992; Goujon et al., 2003), it is not clear how sinapic acid is synthesized in plants. It is possible that it is made by the traditional free acid pathway; however, the high $K_{\mathrm{m}}$ values exhibited by COMT and particularly by F5H for the necessary intermediates do not make this an attractive model. Several years ago we postulated that sinapic acid instead may be made by the oxidation of sinapaldehyde (Humphreys et al., 1999). Previously, we have shown that *ref1* mutants have reduced levels of sinapate esters inleaves and seeds but accumulate wild-type levels of lignin (Ruegger and Chapple, 2001), suggesting that REF1 activity may be required for the synthesis of soluble esters but not syringyl monomer units. Here, we report that the REF1 gene encodes a functional aldehyde dehydrogenase (ALDH) that is capable of oxidizing both sinapaldehyde and coniferaldehyde to sinapic acid and ferulic acid, respectively (Figure 1).

**RESULTS**

### ref1 Mutants Accumulate Reduced Levels of Sinapate Esters

Seven independent alleles of ref1 were identified in our original mutant screen (Ruegger and Chapple, 2001), but only three were characterized in detail. Reanalysis of all of the ref1 mutants revealed that they were morphologically similar to the wild type and that the sinapoylmalate levels in 3-week-old leaves were reduced to <10% of wild-type levels (Figure 2A). The sinapoylcholine content of all mature ref1 seeds was reduced to ~25% of the wild type (Figure 2B). The consistent phenotypic effect of the various ref1 alleles suggests that they are all likely to be null or to encode proteins with very limited activity and that there is a minor REF1-independent pathway to sinapate esters in A. thaliana.

To examine the impact of the ref1 mutation throughout development, sinapoylmalate and sinapoylcholine content was determined in rosette leaves and siliques of ref1-1, ref1-2, and wild-type plants at various stages of the plant life cycle. Except for a transient decrease around 3 weeks of age, leaf sinapoyl-

---

malate levels in both wild-type and ref1 plants increased with age, with sinapoylmalate content in ref1 plants ranging from 10 to ~25% of wild-type levels later in development (Figure 2C). Sinapoylcholine content increased as embryos matured in both ref1 and wild-type plants (Figure 2D); however, the ref1 mutants never accumulated >30% of the sinapoylcholine found in wild-type siliques. These results suggest that REF1 plays a substantial role in the accumulation of sinapate esters in both leaves and seeds throughout development.

#### The REF1 Gene Encodes an ALDH

To understand the molecular and biochemical basis of sinapate ester reduction in ref1 plants, the REF1 gene was isolated by a combination of map-based and candidate gene approaches. Examination of a mapping population generated by crossing ref1-1 to a line of Landsberg erecta carrying the glabrous1 (gl1) mutation identified few ref1 gl1 double mutants, suggesting that the ref1 and gl1 mutations are genetically linked and that the REF1 gene is on chromosome 3. Linkage analysis using a mapping population of 550 F2 plants ultimately showed that the REF1 locus was positioned between cleaved amplified polymorphic sequence markers (Konieczny and Ausubel, 1993) Cer435669 (Jander et al., 2002) and g4711. These two markers define a 154-kb region containing 39 annotated genes (Figure 3). No recombinants were found for cleaved amplified polymorphic sequence marker Cer438851, indicating that it is tightly linked to the REF1 locus. We then took advantage of the fully sequenced A. thaliana genome to identify candidate genes within this

---

![Figure 1. Revised Phenylpropanoid Pathway in A. thaliana Showing the Biosynthesis of Ferulate, Sinapate, and Lignin Precursors from Phe.](#)

Previous versions of the phenylpropanoid pathway postulated carbon flux through the intermediates shown in gray and via the pathway indicated by the gray arrows. Recent evidence has suggested that the pathway is more likely to proceed via the intermediates shown in black. The identification of the REF1 protein reverses the previous model of flux. According to this model, ferulate and sinapate are products of coniferaldehyde and sinapaldehyde oxidation, rather than being precursors in their synthesis. The enzymes and their abbreviations are caffeoyl CoA O-methyltransferase (CCoAOMT), cinnamate 4-hydroxylase (C4H), (hydroxy)cinnamyl alcohol dehydrogenase (CAD), (hydroxy)cinnamoyl CoA reductase (CCR), p-coumaroyl shikimate/quinate 3′-hydroxylase (C3′H), 4-coumarate CoA ligase (4CL), ferulate 5-hydroxylase (F5H), hydroxycinnamoyl CoA:shikimate/quinate hydroxycinna- moyltransferase (CST), Phe ammonia-lyase (PAL), caffeic acid/5-hydroxyferulic acid O-methyltransferase (COMT), and sinapyl alcohol dehydrogenase (SAD). The function of the REF1 protein as a bifunctional hydroxycinnamaldehyde dehydrogenase is illustrated.
Figure 2. Impact of the ref1 Mutation on Sinapate Ester Content.

(A) Sinapoylmalate content of leaves of 3-week-old wild-type plants and each of the seven ref1 mutants. Data represent the mean of at least four samples ±1 SD. Col, wild-type Columbia ecotype.

(B) Sinapoylcholine content of seeds of the same series of plants as in (A). Data represent the mean of at least four samples ±1 SD.

(C) Sinapoylmalate content of entire rosettes from wild-type (inverted triangles), ref1-1 (closed circles), and ref1-2 (open circles) plants sampled throughout plant development. Data represent the mean of at least four samples ±1 SD.

(D) Sinapoylcholine content of developing siliques from wild-type (inverted triangles), ref1-1 (closed circles), and ref1-2 (open circles) plants. Siliques from individual inflorescences were harvested in pairs from top to bottom when the bottom silique turned yellow. Data represent the mean of at least four samples ±1 SD.

mapping region. One of these genes was annotated as a putative ALDH (At3g24503) and was located <8 kb from Cer438851. To test the hypothesis that defects in this putative ALDH gene were responsible for the ref1 phenotype, we used the ref1-3 allele, which had been generated by T-DNA mutagenesis. PCR amplification using gene-specific and T-DNA border primers showed that the putative ALDH gene was disrupted in ref1-3 plants (Figure 4).

To provide further evidence that the REF1 gene is At3g24503, we sequenced the genomic DNA corresponding to all of the ethyl methanesulfonate–induced alleles. Sequence analysis showed that three of the ref1 alleles had point mutations in the gene, resulting in premature translational stop codons (ref1-1 and ref1-4, Q344STOP; ref1-2, W452STOP) (Figure 4). A missense mutation is present in ref1-6 (G416R) and in ref1-7 (G152E), and there is a nucleotide change in the sixth intron/exon splice site junction (agGT-aaGT) that may lead to improper splicing in the ref1-5 allele.

To show that the ref1 mutation is the result of a lesion in the ALDH gene, we complemented the ref1 mutant with the corresponding wild-type gene. Two plasmids, pCC621 and pCC623, were constructed with 3.3 kb and 1.4 kb, respectively, of promoter DNA sequences, the At3g24503 coding sequence, and 819 bp of downstream DNA. These plasmids were transformed into ref1 mutants (ref1-1, ref1-2, and ref1-4) using the floral dip method (Clough and Bent, 1998). Transgenic plants, selected for resistance to kanamycin, were restored in their ability to accumulate wild-type levels of sinapoylmalate (Figure 5).

deduced polypeptide consists of 501 amino acids with a predicted molecular mass of 54.3 kD. Its sequence contains the conserved residues that have been identified in *Homo sapiens* (human) liver ALDH as active site amino acids (Sheikh et al., 1997; Yoshida et al., 1998). The lack of N- or C-terminal extensions as compared with the mammalian liver ALDH suggests that REF1 is likely to be tetrameric (Rodriguez-Zavala and Weiner, 2002). ALDHs that share >40% amino acid sequence identity are considered to be members of the same family (Vasiliou et al., 1999). According to this classification, REF1 is a member of the ALDH2 family and has been designated ALDH2C4 (Skibbe et al., 2002).

A Basic Local Alignment Search Tool (BLASTP) search of the Arabidopsis database identified at least 14 other proteins that were annotated as ALDHs. These include betaine ALDH, methylmalonate semialdehyde dehydrogenase, and an abscisic acid-inducible ALDH (Kirch et al., 2001). The predicted amino acid sequences of these ALDHs vary from 482 to 608 amino acids. The closest homologs to REF1 are two putative mitochondrial ALDHs, At1g23800 (ALDH2B4) and At3g48000 (ALDH2B7), which show ~55% identity at the amino acid level (Skibbe et al., 2002). Interestingly, REF1 is more closely related (65 to 77% identity) to ALDH sequences of unknown function from *Medicago truncatula* (barrel medic), *Zea mays* (maize), *Glycine max* (soybean), *Oryza sativa* (rice), and *Nicotiana tabacum* (tobacco). The identification of related sequences in other species suggests that REF1 activities may be common to a range of plants.

ref1 Mutants Exhibit Reduced Levels of Sinapaldehyde and Coniferaldehyde Dehydrogenase Activity

The identification of REF1 as a putative ALDH led us to postulate that it functions to convert sinapaldehyde to sinapic acid, which in turn is used for the synthesis of sinapoylglucose, sinapoylmalate, and sinapoylcholine. To test this hypothesis, we measured sinapaldehyde dehydrogenase (SALDH) activity in enzyme extracts prepared from wild-type, ref1-1, ref1-2, and ref1-5 plants. In vitro enzyme reactions with wild-type leaf extracts produced a novel compound that cochromatographed with an authentic sinapic acid standard when analyzed by HPLC (data not shown). Gas chromatography–mass spectrometry (GC-MS) analysis of the same reaction products identified a novel peak whose retention time and mass spectrum precisely matched that of the sinapic acid standard (Figures 6C and 6E). The SALDH-specific activity of crude wild-type leaf extracts was

Figure 3. Positional Cloning of the REF1 Gene.

The REF1 gene was mapped to a 154-kb region between markers Cer435669 and g4711 using a mapping population of 550 F2 plants. This region is covered by three P1 clones containing 39 annotated genes. No recombinants were found with marker Cer438851 located on the P1 clone MOB24, suggesting a very tight linkage of the REF1 gene to this marker. A putative ALDH gene located <8 kb from the Cer43851 marker was identified as the REF1 gene by mutant complementation and sequence analysis.

These data demonstrate that a mutation in At3g24503 is responsible for the ref1 phenotype.

REF1 Belongs to a Family of ALDHs in A. thaliana

A full-length REF1 cDNA (GenBank accession number T43357; clone ID 118C14XP) was obtained from the ABRC at Ohio State University and sequenced. The 1625-bp cDNA sequence contained a 43-bp 5′ untranslated region (UTR), the predicted 1503-bp open reading frame, and 76 bp of the 3′ UTR. The

Figure 4. Structure of the REF1 Gene and Location of Defects in ref1 Alleles.

The positions of the primers (CC581, CC582, CC587, and CC588) used to determine the position of the T-DNA insertion in ref1-3 are indicated with arrows.

a hydroxycinnamaldehyde dehydrogenase (HCALDH) with both SALDH and CALDH activity.

REF1 Is Needed for the Accumulation of Cell Wall-Bound Ferulic Acid in Wild-Type Plants and Feruloylmalate in *fah1* Mutants

Based on the *ref1* phenotype, it was not entirely unexpected that REF1 was involved in sinapic acid biosynthesis; however, it was surprising to find that REF1 also had CALDH activity. If this activity is relevant in vivo, then it would be predicted that soluble and/or cell wall-linked ferulate esters would be decreased in *ref1* plants. To evaluate this possibility, wild-type and *ref1* cell wall preparations were subjected to alkaline hydrolysis to release cell wall-esterified phenolics. HPLC analysis revealed that in *ref1*, cell wall-linked ferulate esters were reduced to ~50% of wild-type levels (Figure 8). Interestingly, *p*-coumaric acid, which is synthesized from cinnamic acid by the action of cinnamate 4-hydroxylase, was somewhat elevated in the cell wall preparation from *ref1* plants (pmol *p*-coumarate released per mg cell wall ± SD; the wild type, 20.5 ± 1.4; *ref1-1*, 26.4 ± 3.4; *ref1-2*, 33.1 ± 3.6), indicating that the *ref1* mutation reduces the content of only those hydroxycinnamic acids later in the phenylpropanoid pathway.

Although the *fah1* mutant, which is defective in the gene encoding F5H (Meyer et al., 1996), lacks sinapoylmalate (Chapple et al., 1992), it recently has been found to accumulate trace amounts of feruloylmalate (Figure 9; Hemm et al., 2003). To test the hypothesis that REF1 is needed for the production of ferulic acid, the precursor for feruloylmalate synthesis in *fah1* plants, we developed a *ref1 fah1* double mutant. When methanol-soluble leaf extracts were analyzed by HPLC, we did not detect any feruloylmalate in the *ref1 fah1* double mutant (Figure 9). Taken together, these data provide direct evidence that REF1 plays a major role in the formation of both soluble and cell wall-linked ferulate esters.

CALDH and SALDH Activity Is Present in Many Plants

Even though sinapate esters are a hallmark of the Brassicaceae, they are accumulated in some other species, and as mentioned previously, a wide range of plants accumulate esters of ferulic acid (Harris and Hartley, 1980). To determine whether other plants might have the ability to synthesize ferulic and sinapic acids from coniferaldehyde and sinapaldehyde, respectively, we analyzed enzyme extracts from a wide range of plants for both CALDH and SALDH activity. These included representative samples of dicotyledonous (*A. thaliana*, *N. tabacum*, and *Raphanus sativus* [radish]) and monocotyledonous (*Z. mays*) plants as well as a gymnosperm (*Pinus strobus* [pine]) and a pteridophyte (*Ceratopteris richardii*). In vitro enzyme assays using desalted crude enzyme extracts detected both CALDH and SALDH activity in all of the plants analyzed; however, the relative levels of the two activities varied among species (Table 1).

DISCUSSION

The phenylpropanoid pathway has long been thought to involve a series of reactions in which deamination of Phe (and in some

Hydroxycinnamic Acid Biosynthesis 549

Figure 6. HCALDH Activity in Wild-Type and ref1 Mutant Extracts.

(A) SALDH activity was determined using desalted protein extracts isolated from 21-d-old wild-type and ref1 seedlings. Error bars represent the SD determined from triplicate assays.

(B) CALDH activity determined as described in (A).

(C) GC-MS analysis of the compounds present in the assays described in (A) and (B). GC-MS analysis was performed to identify ferulic acid (FA) and sinapic acid (SA) produced in assays using coniferaldehyde (conald) and sinapaldehyde (sinald) as substrates, respectively. Top panel, mixed ion chromatogram (MIC) of relevant standards; middle panel, MIC of the CALDH assay; bottom panel, MIC of the SALDH assay. The highest peaks in the top two panels represent an unknown substance present in the assays.

(D) Mass spectra of the ferulic acid standard (top panel) and the product of assays described in (A) using coniferaldehyde as a substrate (bottom panel).

(E) Mass spectra of the sinapic acid standard (top panel) and the product of assays described in (A) using sinapaldehyde as a substrate (bottom panel). RT, retention time.

plants, tyrosine) and a series of hydroxylation and O-methylation reactions generate the hydroxycinnamic acids found in plants (Higuchi, 1981). These compounds are, in turn, converted to their corresponding hydroxycinnamaldehydes and hydroxycinnamyl alcohols by the combined action of 4-coumarate CoA ligase, (hydroxy)cinnamoyl CoA reductase, and (hydroxyl)cinnamyl alcohol dehydrogenase. According to this model of the pathway, hydroxycinnamic acids are thus intermediates in the synthesis of their corresponding aldehydes and alcohols. Recent molecular and genetic studies have revealed that this model of the pathway is not accurate and that a complement of enzymes exists in plants that can account for monolignol synthesis without invoking hydroxycinnamic acids other than cinnamate and p-coumarate as intermediates (Humphreys and Chapple, 2002). In support of this model, the characterization of the ref1 mutant has revealed that a significant proportion of the ferulic acid and sinapic acid accumulated in soluble and cell wall–esterified forms is synthesized by the oxidation of coniferaldehyde and sinapaldehyde, respectively, a reaction catalyzed by an HCALDH encoded by the REF1 gene. Thus, with respect to the core reactions of phenylpropanoid metabolism, ferulic acid and sinapic acid appear to be largely end products of the pathway, rather than intermediates.

Putative REF1 Orthologs and Homologs Are Found in Other Plants

The presence of REF1-like genes in a variety of species suggests that the oxidation of hydroxycinnamaldehydes to their corresponding acids may be an important function in many plants. For example, REF1 shares high amino acid similarity with RF2C (ALDH2C2) and RF2D (ALDH2C3) from Z. mays (76% similarity) (Skibbe et al., 2002) and with OsALDH1a (ALDH2C1) from O. sativa (79% similarity) (Li et al., 2000b). Based on these data,

The detection of HCALDH activity in a broad range of species supports the observation that REF1 orthologs are found in other plants. Furthermore, it is interesting to note that the ratios of SALDH:CALDH activity (S:C ratio) differ among the plant species examined. Wild-type *A. thaliana* extract exhibited a relatively high S:C ratio, consistent with the characteristic production of sinapate esters by *A. thaliana*. By contrast, the residual HCALDH activity in *ref1* plants was more specific for coniferaldehyde oxidation, with an S:C ratio similar to that found in several other plants. *R. sativus* leaf extracts exhibited a lower S:C ratio than *A. thaliana* preparations. These data are consistent with the fact that *R. sativus* leaves accumulate more feruloylmalate than sinapoylmalate (Brandl et al., 1984), and suggest that plant hydroxycinnamic acid ester profiles could be determined by the relative specificity of their ALDHs. Finally, *P. strobus* xylem extracts had the lowest S:C ratio of all of the samples tested, an observation that may be attributable to the fact that a different tissue was used as a source of enzyme. Alternatively, it may be related to the fact that gymnosperms do not synthesize phenylpropanoids with the syringyl substitution pattern found in sinapaldehyde; thus, *P. strobus* HCALDH may not have been selected for activity toward this substrate. Taken together, these results indicate that both CALDH and SALDH activity are present in a wide range of plants and that the evolution of these activities may have predated the split between pteridophytes and higher vascular plants.

Is There an Alternative Pathway for Ferulic and Sinapic Acid Synthesis?

The *ref1* mutants contain only low levels of sinapate esters and exhibit reduced CALDH and SALDH activity. These findings indicate that REF1 is quantitatively the most important HCALDH required for sinapic and ferulic acid synthesis in *A. thaliana*. Nevertheless, residual HCALDH activity, sinapate esters, and

Figure 7. Enzymatic Activity of REF1 Expressed in *E. coli*.

(A) and (B) HPLC chromatograms of SALDH and CALDH assays conducted using extracts of *E. coli* harboring the empty pET30a vector. (C) and (D) Analyses as described for (A) and (B) using extracts of *E. coli* harboring pET30a-REF1, showing the production of sinapic acid (SA) and ferulic acid (FA).

AU, absorbance units; conald, coniferaldehyde; sinald, sinapaldehyde.

It is tempting to speculate that RF2C, RF2D, and OsALDH1a from *O. sativa* are involved, like REF1, in the biosynthesis of ferulic acid, a major cell wall-esterified hydroxycinnamic acid in *Z. mays* and *O. sativa*.

Analysis of the inferred amino acid sequence of the REF1 protein did not reveal any subcellular targeting signals, suggesting that REF1 is likely to be localized to the cytosol along with other enzymes of the phenylpropanoid pathway. Cytosolic ALDHs are generally ~500 amino acids in length; whereas, mitochondrial ALDHs carry an additional 50-amino acid N-terminal organellar targeting peptide (Perozich et al., 1999). Specifically, REF1, RF2C, and RF2D (501, 503, and 512 amino acids in length, respectively) are shorter than mitochondrial ALDHs, such as AtALDH2a and AtALDH2b from *A. thaliana* (539 and 534 amino acids, respectively) (Skibbe et al., 2002), RF2A and RF2B from *Z. mays* (549 and 551 amino acids, respectively) (Skibbe et al., 2002), ALDH2a from *O. sativa* (553 amino acids) (Nakazono et al., 2000), and TobALDH2A from *N. tabacum* (543 amino acids) (op den Camp and Kuhlemeier, 1997). Among these related proteins, the *Z. mays* mitochondrial ALDH RF2A, which has 56% identity to REF1, is implicated as a nuclear restorer of cytoplasmic male sterility (Liu et al., 2001; Skibbe et al., 2002). RF2A exhibited broad substrate specificity, whereas RF2B was specific to a few saturated aliphatic aldehydes, including acetaldehyde, propionaldehyde, and butyraldehyde (Liu and Schnable, 2002). Among the different substrates oxidized, RF2A had a relatively high $K_{cat}/K_m$ for cinnamaldehyde (Liu and Schnable, 2002), suggesting that the in vivo function of RF2A may be similar to that of REF1, although a potential link between restoration of male fertility and phenylpropanoid metabolism is not obvious at this time.

Figure 8. Cell Wall-Bound Ferulic Acid in Wild-Type and *ref1* Plants.

After alkaline hydrolysis of plant cell wall preparations, ferulic acid was quantified by HPLC using UV detection at 320 nm. Data represent the mean of triplicate analyses ±1 SD.

could be synthesized directly from feruloyl CoA (Meyer et al., 1991) or possibly through the action of an esterase or thioester-ase that frees hydroxycinnamic acids from their shikimate or CoA esters. One candidate for this activity is hydroxycinnamoyl-CoA:shikimate/quinate hydroxycinnamoyltransferase, an enzyme that has been shown to have hydrolytic activity (Hoffmann et al., 2003). Finally, at this time, we cannot exclude the possibility that a portion of the sinapic acid synthesized in *A. thaliana* is generated by the free acid pathway. The high $K_m$ values exhibited by COMT for caffeic acid and 5-hydroxyferulate (Li et al., 2000a; Parvathi et al., 2001) and the high $K_m$ values of F5H for ferulic acid (Humphreys et al., 1999; Osakabe et al., 1999) make this appear to be unlikely; however, the accumulation of caffeic acid 3-O-glucoside and sinapic acid 4-O-glucoside in *Populus trichocarpa* (poplar) plants downregulated in caffeoyl CoA O-methyltransferase activity (Meyermans et al., 2000) suggests that alternative routes to hydroxycinnamic acids, possibly even through the free acid pathway, may function in plants, at least when certain routes are blocked. It seems clear that the alternative paths to ferulic and sinapic acid in *A. thaliana* will be most readily clarified through the analysis of other ALDH knockouts and possibly through the identification and analysis of *ref1* enhancer mutants.

### REF1 May Have Useful Applications in Crop Improvement

In the cell walls of monocotyledonous plants, ferulic acid–derived molecules play a major role in cross-linking cell wall-bound polysaccharides to lignin (Grabber et al., 2000, 2002). In vitro enzyme digestion studies suggest that ferulate content and cell wall digestibility are negatively correlated (Grabber et al., 1998a, 1998b). Previous models of phenylpropanoid metabolism would have predicted that it would be impossible to interfere with ferulic acid biosynthesis without simultaneously perturbing lignification. The identification of REF1 as an enzyme required for ferulic acid but not lignin biosynthesis provides a new opportunity to alter the phenylpropanoid content of plant cell walls and possibly improve digestibility of forages without compromising agronomic performance. It also may provide new opportunities to modify the quality of vegetables, some of which are known to owe their texture to cell wall–bound ferulic acid derivatives (Waldron et al., 1997).

---

**Table 1. SALDH and CALDH Activities and the Ratios between the Two (S:C ratio) as Measured in Crude Enzyme Extracts Prepared from Leaves (*A. thaliana*, *N. tabacum*, *Z. mays*, and *R. sativus*), Xylem (*P. strobus*), and Fronds (*C. richardii*)**

| Plant                | SALDH Activity ($\mathrm{pkat} \cdot \mathrm{mg}^{-1}$) | CALDH Activity ($\mathrm{pkat} \cdot \mathrm{mg}^{-1}$) | S:C Ratio |
|----------------------|-------------------------------------------------------|-------------------------------------------------------|-----------|
| *A. thaliana* wild type | 48.1 ± 0.9                                          | 35.3 ± 0.6                                          | 1.4       |
| *A. thaliana ref1-1*   | 9.3 ± 1.2                                           | 15.8 ± 1.5                                          | 0.6       |
| *A. thaliana ref1-2*   | 8.5 ± 0.7                                           | 17.4 ± 0.7                                          | 0.5       |
| *N. tabacum*          | 26.9 ± 1.7                                          | 19.5 ± 1.7                                          | 1.4       |
| *Z. mays*             | 34.2 ± 4.2                                          | 69.7 ± 9.5                                          | 0.5       |
| *R. sativus*          | 24.0 ± 1.2                                          | 65.9 ± 7.5                                          | 0.4       |
| *P. strobus* xylem    | 8.7 ± 1.5                                           | 34.9 ± 3.8                                          | 0.2       |
| *C. richardii* frond  | 79.4 ± 6.0                                          | 62.4 ± 3.5                                          | 1.3       |

---

Figure 9. HPLC Analysis of Soluble Secondary Metabolites Accumulated in 21-d-Old Wild-Type, *ref1*, and *fah1* Mutants and in *fah1 ref1* Double Mutants.

Soluble leaf phenolics were extracted from leaf tissue in 50% methanol and analyzed by reverse-phase HPLC. Sinapoylmalate (SM) is found in wild-type and *ref1* extracts; the *fah1* mutant instead accumulates feruloylmalate (FM). Feruloylmalate is absent in extracts of the *fah1 ref1* double mutant (DM). AU, absorbance units.

wall-bound ferulic acid remain in *ref1* mutants. These are not the result of residual REF1 activity because three of the *ref1* mutants (*ref1-1*, *ref1-2*, and *ref1-4*) carry nonsense mutations, and a T-DNA insertion disrupts the *ref1-3* coding region. Given that cinnamaldehyde is a good substrate for the *Z. mays* ALDHs RF2A and RF2B (Liu and Schnable, 2002), it is possible that the residual HCALDH activity detected in *ref1* mutant extracts is the result of the activity of one or both of their potential *A. thaliana* orthologs AtALDH2a and AtALDH2b, both of which are close homologs of REF1. On the other hand, the predicted mitochondrial localization of AtALDH2a and AtALDH2b makes it questionable whether these proteins have a role in phenylpropanoid metabolism, which generally is considered to occur in the cytoplasm. Alternatively, the residual sinapate esters found in *ref1* mutants may be synthesized from sinapaldehyde through the action of an aldehyde oxidase. In *A. thaliana*, there are at least four genes that encode aldehyde oxidases, and these proteins show relatively broad substrate specificity (Sekimoto et al., 1998; Seo et al., 1998, 2000). Similarly, the residual cell wall–bound ferulic acid in stems

The Plant Cell

METHODS

Plant Material and Growth Conditions

*Arabidopsis thaliana* was grown at a light intensity of 100 μE m⁻² s⁻¹ at 23°C under a photoperiod of 16 h light/8 h dark in Redi-Earth potting mix (Scotts-Sierra Horticulture Products; Marysville, OH). The seven *ref1* alleles (*ref1-1* to *ref1-7*) used in this study were identified in an M2 screen of ethyl methanesulfonate-mutagenized plants of the Columbia ecotype (Ruegger and Chapple, 2001), except for *ref1-3*, which was isolated from a T-DNA–mutagenized population of the Wassilewskija ecotype. All of the alleles used in this study were backcrossed at least two times to the corresponding wild type before analysis.

Cloning of the *REF1* Gene

The *ref1-1* mutant was crossed to Landsberg *erecta* to establish a mapping population. F1 individuals were allowed to self-pollinate, and F2 plants were screened for the *ref1* phenotype. DNA was extracted from homozygous F2 and F3 *ref1* mutants for use in PCR-based genotyping experiments. Individuals carrying recombinant chromosomes in the region of the *REF1* locus were used to determine a mapping interval for the *REF1* gene.

To determine whether the coding region of the putative *REF1* gene was disrupted by a T-DNA insertion, DNA isolated from *ref1-3* and wild-type Wassilewskija was used for PCR with oligonucleotides CC581 (5′-ATG-AGAACGGCAAATG-3′) and CC582 (5′-TTACATCCAAGGGGAATTGTG-3′). The position of the T-DNA insertion within the *REF1* gene was further delineated using oligonucleotides CC587 (5′-CCACTTCTCATATTC AAC-GAC-3′) with CC582 and CC588 (5′-GTCGTTGAATATGAGAAGTG G-3′) with CC581. DNA from *ref1-3* then was amplified with oligonucleotides CC581 and the T-DNA left border primer CC46 (5′-GATGC ACTCGAA-ATCAGCCA-3′), subcloned, and sequenced to determine the exact location of the T-DNA integration. To identify mutations in the other *ref1* alleles, genomic DNA from *ref1* homozygotes was amplified as two overlapping fragments using oligonucleotide combinations CC614 (5′-AATCCACTGCCTTTGCTGAC-3′)/CC628 (5′-CGGCGCGACTCA-TAAGAA-3′) and CC620 (5′-AATTGGAGTGGTTGGTA-3′)/CC632 (5′-AGCCGCC TTATTATCAT TGG-3′). These DNA fragments then were directly sequenced without subcloning.

Complementation of the *ref1* Mutant

The BAC clone MIOB24, which contains the *REF1* gene, was obtained from the ABRC. An 11-kb KpnI/XhoI DNA fragment containing the *REF1* gene was isolated from the BAC clone and cloned into pBS KS– (Stratagene, La Jolla, CA). A 7414-bp Csp45I DNA fragment comprised of 3297 bp of the *REF1* promoter region, 3298 bp of the coding region, and 819 bp of the 3′ UTR was isolated and cloned into the Clal site of pBS KS– to generate pCC619. A KpnI/PstI fragment from pCC619 containing the 7414-bp fragment of the *REF1* gene was cloned into KpnI/PstI-digested pCAMBIA 2300 (Cambia, Canberra, Australia) to generate pCC621. Plasmid pCC623 was constructed by ligating a 5507-bp BamHI fragment from pCC619 containing 1390 bp of the *REF1* promoter region, 3298 bp of the coding region, and 819 bp of the 3′ UTR into the BamHI site of pCAMBIA 2300. The plasmids (pCAMBIA 2300, pCC621, and pCC623) were introduced into *Agrobacterium tumefaciens* C58 pGV3850 (Zambrisk y et al., 1983) by electroporation. The *ref1* mutants (*ref1-1*, *ref1-2*, and *ref1-4*) were transformed by the floral dip method (Clough and Bent, 1998), and transformants were selected on kanamycin-containing media as previously described (Hemm et al., 2003). Three-week-old transgenic plants were analyzed visually under UV light and by HPLC for complementation of the sinapoylmalate-deficient phenotype.

Analytical Methods

Soluble hydroxycinnamic acid esters were extracted, separated, and quantified by HPLC as previously described (Hemm et al., 2003). Sinapoylcholine in seeds was analyzed similarly, except that a Puresil C18 column (Waters, Milford, MA; 1200-nm pore size, 5-μM particle size) was used. To analyze ester-bound hydroxycinnamic acids, cell walls were prepared and analyzed by HPLC and GC-MS as previously described (Franke et al., 2002a, 2002b).

Protein Extraction and Assay

Plant material (1 g) was ground in liquid nitrogen and added to 5 mL of extraction buffer containing 50 mM Hepes-KOH, pH 8.0, 1 mM EDTA, 5 mM DTT, and 10% glycerol (v/v) and stirred for 30 min at 4°C. The extract was centrifuged at 10,000g for 20 min. The supernatant was collected and the protein precipitated by adding ammonium sulfate to 70% saturation. The precipitated protein was pelleted by centrifugation, dissolved in 1 mL of extraction buffer, desalted into extraction buffer, and assayed for ALDH activity.

ALDH assays were conducted in 50 mM Hepes-KOH, pH 8.0, 5 mM DTT, 10% glycerol (v/v), 1 mM NAD⁺, and 100 μM coniferaldehyde or sinapaldehyde in a final volume of 200 μL. The buffer was prewarmed, and 10 μL of the enzyme extract was added to initiate the reaction. The assay was stopped after 30 min at 30°C by the addition of 50 μL of 3 M acetic acid. Twenty microliters of the assay mixture was analyzed by HPLC (Hemm et al., 2003) to determine the amount of ferulic and sinapic acids produced.

REF1 Expression in *E. coli*

The open reading frame of *REF1* was amplified from λ-PRL2 *A. thaliana* cDNA clone 118C14T7 (GenBank accession number T43357) using oligonucleotide CC686 (5′-GAAGAGACATATGGAGAACGGCAAATG-3′) and CC687 (5′-GATACTCGAGCATCCAAGGGGAATTGTG-3′). The oligonucleotides incorporated an Ndel site at the N-terminal end and an Xhol site at the C-terminal end of the open reading frame (underlined). The PCR product was digested and cloned into the corresponding sites of pET30a (Novagen, Madison, WI) to generate pET30a-REF1 (pCC628). The recombinant plasmid, along with pET30a, then was introduced into *E. coli* BL21DE3 pLysS and used for protein induction. An Xhol fragment from pCC628 was subcloned into pBS KS– and sequenced to verify the absence of PCR-induced errors.

Overnight, cultures of *E. coli* BL21DE3 pLysS containing either pET30a-REF1 or pET30a were diluted 50-fold and grown at room temperature (22°C) to an OD₆₀₀ of 0.7. The cells then were induced for 4 h with 0.05 mM isopropyl-β-D-thiogalactopyranoside, centrifuged, washed in distilled water, and resuspended in 3 mL of extraction buffer. The cells were lysed using a French pressure cell and centrifuged at 14,000g for 30 min. The supernatants were assayed for ALDH activity as described above.

Sequence data from this article have been deposited with the EMBL/ GenBank data libraries under accession number T43357 (clone ID 118C14XP).

ACKNOWLEDGMENTS

This work was supported by a grant from the Division of Energy Biosciences, U.S. Department of Energy. K.L.B. was supported by a summer research fellowship funded by the Howard Hughes Medical Institute Undergraduate Initiative to the Department of Biological

Sciences, Purdue University. This is journal paper number 17261 of the Purdue University Agricultural Experiment Station.

Received September 17, 2003; accepted December 8, 2003.

**REFERENCES**

Brandl, W., Herrmann, K., and Grotjahn, L. (1984). Hydroxycinnamoyl esters of malic acid in small radish (*Raphanus sativus* L. *sativus*). Z. Naturforsch. [C] **39**, 515–520.

Chapple, C.C.S., Vogt, T., Ellis, B.E., and Somerville, C.R. (1992). An *Arabidopsis* mutant defective in the general phenylpropanoid pathway. Plant Cell **4**, 1413–1424.

Clough, S.J., and Bent, A.F. (1998). Floral dip: A simplified method for *Agrobacterium*-mediated transformation of *Arabidopsis thaliana*. Plant J. **16**, 735–743.

Dempsey, D.A., Shah, J., and Klessig, D.F. (1999). Salicylic acid and disease resistance in plants. Crit. Rev. Plant Sci. **18**, 547–575.

Dixon, R.A., Achine, L., Kota, P., Liu, C.J., Reddy, M.S.S., and Wang, L.J. (2002). The phenylpropanoid pathway and plant defence: A genomics perspective. Mol. Plant Pathol. **3**, 371–390.

Dixon, R.A., Lamb, C.J., Masoud, S., Sewalt, V.J.H., and Paiva, N.L. (1996). Metabolic engineering: Prospects for crop improvement through the genetic manipulation of phenylpropanoid biosynthesis and defense responses: A review. Gene **179**, 61–71.

Franke, R., Hemm, M.R., Denault, J.W., Ruegger, M.O., Humphreys, J.M., and Chapple, C. (2002a). Changes in secondary metabolism and deposition of an unusual lignin in the *ref8* mutant of *Arabidopsis*. Plant J. **30**, 47–59.

Franke, R., Humphreys, J.M., Hemm, M.R., Denault, J.W., Ruegger, M.O., Cusumano, J.C., and Chapple, C. (2002b). The *Arabidopsis REF8* gene encodes the 3-hydroxylase of phenylpropanoid metabolism. Plant J. **30**, 33–45.

Goujon, T., Sibout, R., Pollet, B., Maba, B., Nussaume, L., Bechtold, N., Lu, F., Ralph, R., Mila, I., Barrière, Y., Lapierre, C., and Jouanin, L. (2003). A new *Arabidopsis thaliana* mutant deficient in the expression of O-methyltransferase impacts lignins and sinapoyl esters. Plant Mol. Biol. **71**, 973–989.

Grabber, J.H., Hatfield, R.D., and Ralph, J. (1998a). Diferulate cross-links impede the enzymatic degradation of non-lignified maize walls. J. Sci. Food Agric. **77**, 193–200.

Grabber, J.H., Ralph, J., and Hatfield, R.D. (1998b). Ferulate cross-links limit the enzymatic degradation of synthetically lignified primary walls of maize. J. Agric. Food Chem. **48**, 2609–2614.

Grabber, J.H., Ralph, J., and Hatfield, R.D. (2000). Cross-linking of maize walls by ferulate dimerization and incorporation into lignin. J. Agric. Food Chem. **50**, 6106–6113.

Grabber, J.H., Ralph, J., and Hatfield, R.D. (2002). Model studies of ferulate-coniferyl alcohol cross-product formation in primary maize walls: Implications for lignification in grasses. J. Agric. Food Chem. **50**, 6008–6016.

Guo, D., Chen, F., Inoue, K., Blount, J.W., and Dixon, R.A. (2001). Downregulation of Caffeic Acid 3-O-Methyltransferase and Caffeoyl CoA 3-O-Methyltransferase in Transgenic Alfalfa: Impacts on Lignin Structure and Implications for the Biosynthesis of G and S Lignin. Plant Cell **13**, 73–88.

Harris, P.J., and Hartley, R.D. (1980). Phenolic constituents of the cell walls of monocotyledons. Biochem. Syst. Ecol. **8**, 153–160.

Hemm, M.R., Ruegger, M.O., and Chapple, C. (2003). The *Arabidopsis ref2* mutant is defective in the gene encoding CYP83A1 and shows both phenylpropanoid and glucosinolate phenotypes. Plant Cell **15**, 179–194.

Higuchi, T. (1981). Biosynthesis of lignin. In Carbohydrates II: Extracellular Carbohydrates, Vol. 13B, W. Tanner and F.A. Loewus, eds (New York: Springer-Verlag), pp. 194–224.

Hoffmann, L., Maury, S., Martz, F., Geoffroy, P., and Legrand, M. (2003). Purification, cloning, and properties of an acyltransferase controlling shikimate and quinate ester intermediates in phenylpropanoid metabolism. J. Biol. Chem. **278**, 95–103.

Humphreys, J.M., and Chapple, C. (2002). Rewriting the lignin roadmap. Curr. Opin. Plant Biol. **5**, 224–229.

Humphreys, J.M., Hemm, M.R., and Chapple, C. (1999). New routes for lignin biosynthesis defined by biochemical characterization of recombinant ferulate 5-hydroxylase, a multifunctional cytochrome P450-dependent monooxygenase. Proc. Natl. Acad. Sci. USA **96**, 10045–10050.

Jander, G., Norris, S.R., Rounsley, S.D., Bush, D.F., Levin, I.M., and Last, R.L. (2002). *Arabidopsis* map-based cloning in the post-genome era. Plant Physiol. **129**, 440–450.

Kirch, H.-H., Nair, A., and Bartels, D. (2001). Novel ABA- and dehydration-inducible aldehyde dehydrogenase genes isolated from the resurrection plant *Craterostigma plantagineum* and *Arabidopsis thaliana*. Plant J. **28**, 555–567.

Konieczny, A., and Ausubel, F.M. (1993). A procedure for mapping *Arabidopsis* mutations using co-dominant ecotype-specific PCR-based markers. Plant J. **4**, 403–410.

Landry, L.G., Chapple, C.C.S., and Last, R.L. (1995). *Arabidopsis* mutants lacking phenolic sunscreens exhibit enhanced ultraviolet-B injury and oxidative damage. Plant Physiol. **109**, 1159–1166.

Lehfeldt, C., Shirley, A.M., Meyer, K., Ruegger, M.O., Cusumano, J.C., Viitanen, P.V., Strack, D., and Chapple, C. (2000). Cloning of the *SNG1* gene of *Arabidopsis* reveals a role for serine carboxypeptidase-like protein as an acyltransferase in secondary metabolism. Plant Cell **12**, 1295–1306.

Lewis, N.G., and Yamamoto, E. (1990). Lignin: Occurrence, biogenesis and biodegradation. Annu. Rev. Plant Physiol. Plant Mol. Biol. **41**, 455–496.

Li, L., Popko, J.L., Umezawa, T., and Chiang, V.L. (2000a). 5-Hydroxyconiferyl aldehyde modulates enzymatic methylation for syringyl monolignol formation, a new view of monolignol biosynthesis in angiosperms. J. Biol. Chem. **275**, 6537–6545.

Li, Y., Nakazono, M., Tsutsumi, N., and Hirai, A. (2000b). Molecular and cellular characterizations of a cDNA clone encoding a novel isozyme of aldehyde dehydrogenase from rice. Gene **249**, 67–74.

Liu, F., Cui, X., Horner, H.T., Weiner, H., and Schnable, P.S. (2001). Mitochondrial aldehyde dehydrogenase activity is needed for male fertility in maize. Plant Cell **13**, 1063–1078.

Liu, F., and Schnable, P.S. (2002). Functional specialization of maize mitochondrial aldehyde dehydrogenases. Plant Physiol. **130**, 1657–1674.

Meyer, K., Cusumano, J.C., Somerville, C., and Chapple, C.C.S. (1996). Ferulate-5-hydroxylase from *Arabidopsis thaliana* defines a new family of cytochrome P450-dependent monooxygenases. Proc. Natl. Acad. Sci. USA **93**, 6869–6874.

Meyer, K., Kohler, A., and Kauss, H. (1991). Biosynthesis of ferulic acid esters of plant cell wall polysaccharides in endomembranes from parsley cells. FEBS Lett. **290**, 209–212.

Meyermans, H., et al. (2000). Modifications in lignin and accumulation of phenolic glucosides in poplar xylem upon down-regulation of caffeoyl-coenzyme A O-methyltransferase, an enzyme involved in lignin biosynthesis. J. Biol. Chem. **275**, 36899–36909.

Nakazono, M., Tsuji, H., Li, Y., Saisho, D., Arimura, S., Tsutsumi, N., and Hirai, A. (2000). Expression of a gene encoding mitochondrial

aldehyde dehydrogenase in rice increases under submerged conditions. Plant Physiol. 124, 587–598.

op den Camp, R.G., and Kuhlemeier, C. (1997). Aldehyde dehydrogenase in tobacco pollen. Plant Mol. Biol. 35, 355–365.

Osakabe, K., Tsao, C.C., Li, L.G., Popko, J.L., Umezawa, T., Carraway, D.T., Smetzer, R.H., Joshi, C.P., and Chiang, V.L. (1999). Coniferyl aldehyde 5-hydroxylation and methylation direct syringyl lignin biosynthesis in angiosperms. Proc. Natl. Acad. Sci. USA 96, 8955–8960.

Parvathi, K., Chen, F., Guo, D., Blount, J.W., and Dixon, R.A. (2001). Substrate preferences of O-methyltransferases in alfalfa suggest new pathways for 3-O-methylation of monolignols. Plant J. 25, 193–202.

Perozich, J., Nicholas, H., Wang, B.C., Lindahl, R., and Hempel, J. (1999). Relationships within the aldehyde dehydrogenase extended family. Protein Sci. 8, 137–146.

Rodriguez-Zavala, J.S., and Weiner, H. (2002). Structural aspects of aldehyde dehydrogenase that influence dimer-tetramer formation. Biochemistry 41, 8229–8237.

Ruegger, M., and Chapple, C. (2001). Mutations that reduce sinapoylmalate accumulation in *Arabidopsis thaliana* define loci with diverse roles in phenylpropanoid metabolism. Genetics 159, 1741–1749.

Schoch, G., Goepfert, S., Morant, M., Hehn, A., Meyer, D., Ullmann, P., and Werck-Reichhart, D. (2001). CYP98A3 from *Arabidopsis thaliana* is a 3′-hydroxylase of phenolic esters, a missing link in the phenylpropanoid pathway. J. Biol. Chem. 276, 36566–36574.

Sekimoto, H., Seo, M., Kawakami, N., Komano, T., Desloire, S., Liotenberg, S., Marion-Poll, A., Caboche, M., Kamiya, Y., and Koshiba, T. (1998). Molecular cloning and characterization of aldehyde oxidases in *Arabidopsis thaliana*. Plant Cell Physiol. 39, 433–442.

Seo, M., Akaba, S., Oritani, T., Delarue, M., Bellini, C., Caboche, M., and Koshiba, T. (1998). Higher activity of an aldehyde oxidase in the auxin-overproducing *superroot1* mutant of *Arabidopsis thaliana*. Plant Physiol. 116, 687–693.

Seo, M., Koiwai, H., Akaba, S., Komano, T., Oritani, T., Kamiya, Y., and Koshiba, T. (2000). Abscisic aldehyde oxidase in leaves of *Arabidopsis thaliana*. Plant J. 23, 481–488.

Sheikh, S., Ni, L., Hurley, T.D., and Weiner, H. (1997). The potential roles of the conserved amino acids in human liver mitochondrial aldehyde dehydrogenase. J. Biol. Chem. 272, 18817–18822.

Shirley, A.M., McMichael, C.M., and Chapple, C. (2001). The *sng2* mutant of *Arabidopsis* is defective in the gene encoding the serine carboxypeptidase-like protein sinapoylglucose:choline sinapoyltransferase. Plant J. 28, 83–94.

Skibbe, D.S., Liu, F., Wen, T.-J., Yandeau, M.D., Cui, X., Cao, J., Simmons, C.R., and Schnable, P.S. (2002). Characterization of the aldehyde dehydrogenase gene families of Zea mays and *Arabidopsis*. Plant Mol. Biol. 48, 751–764.

van der Meer, I.M., Stam, M.E., van Tunen, A.J., Mol, J.N.M., and Stuitje, A.R. (1992). Antisense inhibition of flavonoid biosynthesis in petunia anthers results in male sterility. Plant Cell 4, 253–262.

Vasiliou, V., Bairoch, A., Tipton, K.E., and Nebert, D.W. (1999). Eukaryotic aldehyde dehydrogenase (ALDH) genes: Human polymorphisms and recommended nomenclature based on divergent evolution and chromosomal mapping. Pharmacogenetics 9, 421–434.

Waldron, K.W., Smith, A.C., Parr, A.J., Ng, A., and Parker, M.L. (1997). New approaches to understanding and controlling cell separation in relation to fruit and vegetable texture. Trends Food Sci. Technol. 8, 213–221.

Yoshida, A., Rzhetsky, A., Hsu, L.C., and Chang, C. (1998). Human aldehyde dehydrogenase gene family. Eur. J. Biochem. 251, 549–557.

Zambrisky, P., Joos, H., Genetello, C., Leemans, J., van Montagu, M., and Schell, J. (1983). Ti plasmid vector for the introduction of DNA into plant cells without alteration of their normal regeneration capacity. EMBO J. 2, 2143–2150.

Zhong, R., Morrison, W.H., Negrel, J., and Ye, Z.-H. (1998). Dual methylation pathways in lignin biosynthesis. Plant Cell 10, 2033–2045.

The Arabidopsis thaliana REDUCED EPIDERMAL FLUORESCENCE1 Gene Encodes an Aldehyde Dehydrogenase Involved in Ferulic Acid and Sinapic Acid Biosynthesis

Ramesh B. Nair, Kristen L. Bastress, Max O. Ruegger, Jeff W. Denault and Clint Chapple

Plant Cell 2004; 16; 544-554; originally published online January 16, 2004;

DOI 10.1105/tpc.017509

This information is current as of June 5, 2015

| References | This article cites 52 articles, 22 of which can be accessed free at: http://www.plantcell.org/content/16/2/544.full.html#ref-list-1 |
| Permissions | https://www.copyright.com/ccc/openurl.do?sid=pd_hw1532298X&issn=1532298X&WT.mc_id=pd_hw1532298X |
| eTOCs | Sign up for eTOCs at: http://www.plantcell.org/cgi/alerts/ctmain |
| CiteTrack Alerts | Sign up for CiteTrack Alerts at: http://www.plantcell.org/cgi/alerts/ctmain |
| Subscription Information | Subscription Information for The Plant Cell and Plant Physiology is available at: http://www.aspb.org/publications/subscriptions.cfm |

© American Society of Plant Biologists

ADVANCING THE SCIENCE OF PLANT BIOLOGY
