
The Structure and Receptor Binding Properties of the 1918 Influenza Hemagglutinin  
S. J. Gamblin et al.  
Science 303, 1838 (2004);  
DOI:10.1126/science.1093155  

This copy is for your personal, non-commercial use only.

If you wish to distribute this article to others, you can order high-quality copies for your colleagues, clients, or customers by [clicking here](https://www.sciencemag.org).

Permission to republish or repurpose articles or portions of articles can be obtained by following the guidelines [here](https://www.sciencemag.org).

The following resources related to this article are available online at www.sciencemag.org (this information is current as of June 15, 2012):

Updated information and services, including high-resolution figures, can be found in the online version of this article at:  
[http://www.sciencemag.org/content/303/5665/1838.full.html](http://www.sciencemag.org/content/303/5665/1838.full.html)

Supporting Online Material can be found at:  
[http://www.sciencemag.org/content/suppl/2004/03/19/1093155.DC1.html](http://www.sciencemag.org/content/suppl/2004/03/19/1093155.DC1.html)

A list of selected additional articles on the Science Web sites related to this article can be found at:  
[http://www.sciencemag.org/content/303/5665/1838.full.html#related](http://www.sciencemag.org/content/303/5665/1838.full.html#related)

This article has been cited by 153 article(s) on the ISI Web of Science

This article has been cited by 59 articles hosted by HighWire Press; see:  
[http://www.sciencemag.org/content/303/5665/1838.full.html#related-urls](http://www.sciencemag.org/content/303/5665/1838.full.html#related-urls)

This article appears in the following subject collections:  
Biochemistry  
[http://www.sciencemag.org/cgi/collection/biochem](http://www.sciencemag.org/cgi/collection/biochem)

Science (print ISSN 0036-8075; online ISSN 1095-9203) is published weekly, except the last week in December, by the American Association for the Advancement of Science, 1200 New York Avenue NW, Washington, DC 20005. Copyright 2004 by the American Association for the Advancement of Science; all rights reserved. The title *Science* is a registered trademark of AAAS.

RESEARCH ARTICLES

47. J. S. Vrettos, J. Limburg, G. W. Brudvig, *Biochim. Biophys. Acta* 1503, 229 (2001).
48. V. L. Pecoraro et al., *Pure Appl. Chem.* 70, 925 (1998).
49. N. Knoepfle, T. M. Bricker, C. Putnam-Evans, *Biochemistry* 38, 1582 (1999).
50. C. W. Hoganson, G. T. Babcock, *Science* 277, 1953 (1997).
51. M. Haumann, W. Junge, *Biochim. Biophys. Acta* 1411, 86 (1999).
52. The authors dedicate this paper to G. T. Babcock and M. P. Klein. J.B. and S.I. acknowledge support from the Biotechnology and Biological Research Science Council. We thank the Centre for Structural Biology and Bioinformatics Facility at Imperial College London for technical support; and C. Schulze-Briese and T. Tomizaki at PX06SA/SLS, Paul Scherrer Institute, Villigen, Switzerland; B. Shepard at ID29; and M. Iwata of the ERATO ATP System Project for help with data collections. T.M.I. has been a Life Sciences Research Foundation Fellow of the Howard Hughes Medical Institute, a European Molecular Biology Organization Long-Term Fellow, and a Ruth L. Kirschstein National Research Science Award Fellow and acknowledges support from D. C. Rees during the initial stages of the study. We wish to thank P. Siegbahn, M. Lundberg, P. Nixon, C. Dismukes, and A. Telfer for comments and helpful discussions. The coordinates, together with the structure factors native, Mn-edge and long wavelength (2.25 Å, for Ca detection), and the experimental multi-wavelength anomalous dispersion (MAD) phases have been deposited in the Protein Data Bank (entry 1S5L) and will be available upon publication.

Supporting Online Material  
www.sciencemag.org/cgi/content/full/1093087/DC1  
Materials and Methods  
Figs. S1 to S5  
References and Notes  

29 October 2003; accepted 22 January 2004  
Published online 5 February 2004;  
10.1126/science.1093087  
Include this information when citing this paper.


The Structure and Receptor Binding Properties of the 1918 Influenza Hemagglutinin

S. J. Gamblin,$^{1\ast}$ L. F. Haire,$^{1\ast}$ R. J. Russell,$^{1\ast}$ D. J. Stevens,$^{1}$  
B. Xiao,$^{1}$ Y. Ha,$^{2\dagger}$ N. Vasisht,$^{1}$ D. A. Steinhauer,$^{1\ddagger}$ R. S. Daniels,$^{1}$  
A. Elliot,$^{1}$ D. C. Wiley,$^{2}$ J. J. Skehel$^{1\S}$

The 1918 influenza pandemic resulted in about 20 million deaths. This enormous impact, coupled with renewed interest in emerging infections, makes characterization of the virus involved a priority. Receptor binding, the initial event in virus infection, is a major determinant of virus transmissibility that, for influenza viruses, is mediated by the hemagglutinin (HA) membrane glycoprotein. We have determined the crystal structures of the HA from the 1918 virus and two closely related HAs in complex with receptor analogs. They explain how the 1918 HA, while retaining receptor binding site amino acids characteristic of an avian precursor HA, is able to bind human receptors and how, as a consequence, the virus was able to spread in the human population.

The HAs of influenza viruses mediate receptor binding and membrane fusion, the first stages of virus infection (1). The receptors that they recognize are sialic acids of cell-surface glycoproteins and glycolipids, and the nature of the interactions involved in determining binding specificity has been described in biochemical, genetic, and structural studies (2–8). Sialic acids are usually found in either α2,3 or α2,6 linkages to galactose, the predominant penultimate sugar of N-linked carbohydrate side chains. The binding preference of a given HA for one or other of these linkage types correlates with the species specificity for infection. Thus, the HAs of all 15 antigenic subtypes found in avian influenza viruses bind preferentially to sialic acid in α2,3 linkage (9), and it is this form of the sialosaccharide that predominates in avian enteric tracts where these viruses replicate (10). Swine influenza viruses are reported to bind sialic acid in α2,6, and sometimes also α2,3, linkages (11), and sialic acid in both linkages is detected in porcine tracheae (10). Human viruses of the H1, H2, and H3 subtypes that are known to have caused pandemics in 1918, 1957, and 1968, respectively, recognize α2,6-linked sialic acid (5), the major form found on cells of the human respiratory tract (12, 13).

Because an avian origin is proposed for the HAs of swine and human viruses (14), changes in binding specificity are required for cross-species transfer. The mechanism that human viruses have used to achieve these changes appears to be different for different subtypes. For the HAs of the H2 and H3 human viruses, a minimum of two changes in receptor binding site amino acids, Gln<sup>226</sup> to Leu<sup>226</sup> and Gly<sup>228</sup> to Ser<sup>228</sup>, correlates with the shift from avian to human receptor binding (15, 16). By contrast, HAs of human H1 viruses acquire the ability to bind to human receptors while retaining Gln<sup>226</sup> and Gly<sup>228</sup> (11). To understand how they do this, we determined the structures of HAs from the 1918 pandemic virus (1918-human) with the use of HA expressed from DNA of the sequence recovered from tissues infected with virus in 1918 (17) and from the prototype human (1934-human) and swine (1930-swine) H1 influenza viruses, A/Puerto Rico/8/34 and A/swine/Iowa/30, respectively (Fig. 1). A/Puerto Rico/8/34 was one of the first human influenza viruses isolated in the Americas (18) and has been widely used in laboratory investigations of influenza. A/swine/Iowa/30 was the first influenza virus isolated from mammals in 1930 (19), 3 years before the first isolate was recovered from humans (20).

**Overall structure.** The structures were solved by molecular replacement, and crystallographic statistics are given in Table 1 and table S1. The overall trimeric structures of the three H1 HAs are similar (Fig. 2), but they show notable differences to HAs of other subtypes with respect to the arrangements of the receptor binding, vestigial esterase, and membrane fusion subdomains, both within the HA trimer and also within individual monomers (21) (table S2). As predicted from their placement in the same phylogenetic and structure-based clade, the H1 HAs are most similar to those of the H5 subtype (21). We have examined the structures in detail in relation to their receptor binding and membrane fusion activities and to the antigenic variation that occurred in both periods of human H1 virus prevalence, 1918 to 1957 and 1977 to date (Fig. 2), and we will present a detailed description of this analysis elsewhere (22). We have, however, concluded that the receptor binding properties are the most distinctive features of the 1918 virus HA, and we focus on these here.

**The receptor binding subdomain.** The receptor binding sites are located at the membrane-distal tip of each subunit of the HA trimer (Fig. 2). Three secondary structure elements—the 190 helix (residues 190 to 198), the 130 loop (residues 135 to 138), and the 220 loop (residues 221 to 228)—form the sides of each site, with the base made up of the conserved residues Tyr<sup>98</sup>, Trp<sup>153</sup>, His<sup>183</sup>, and Tyr<sup>195</sup> (1) (Fig. 2B). The conformations adopted by the 130 and 220 loops of the three H1 HAs are similar, but they are significantly different from those of the equivalent loops in the HAs of other influenza subtypes (2, 3) (Figs. 2B, 3, and 4 and table S2). To understand the structural basis of the receptor specificity of H1 HAs, we determined the structures of the 1934-human and the 1930-swine HAs in complex, with α2,3- and α2,6-linked sialopentasaccharides as analogs of avian and human receptors,

$^{1}$Medical Research Council (MRC) National Institute for Medical Research, The Ridgeway, Mill Hill, London NW7 1AA, UK. $^{2}$Department of Molecular and Cellular Biology, Howard Hughes Medical Institute, Harvard University, 7 Divinity Avenue, Cambridge, MA 02138, USA.  
$\ast$These authors contributed equally to this work.  
$\dagger$Present address: Department of Pharmacology, Yale University School of Medicine, 333 Cedar Street, New Haven, CT 06520, USA.  
$\ddagger$Present address: Department of Microbiology and Immunology, Emory University School of Medicine, 1510 Clifton Road, Atlanta, GA 30322, USA.  
$\S$To whom correspondence should be addressed. E-mail: mbrenna@nimr.mrc.ac.uk

respectively (2) (Fig. 3 and figs. S1 and S2). As observed with other HAs (1, 3, 8), the terminal sialic acids of the human and avian receptors interact with binding site residues through a series of conserved hydrogen bonds (Fig. 3 and fig. S2).

**The 1934-human HA/human receptor complex.** The electron density maps reveal well-ordered features for the Sia-1, Gal-2, and GlcNAc-3 of the sialopentasaccharide in this complex (Fig. 3A and figs. S1 and S2). Gal-2 forms five hydrogen bonds that have not been previously observed in other HA/receptor complexes. Four bonds are possible between the 2- and 3-hydroxyls of Gal-2 and the side chains of Lys²²² and Asp²²⁵, and a fifth between the 4-hydroxyl of Gal-2 and the main-chain amide of 227 that is mediated by a water molecule.

**The 1934-human HA/avian receptor complex.** Again only the Sia-1, Gal-2, and GlcNAc-3 moieties of the sialopentasaccharide are ordered in this complex (Fig. 3B and figs. S1 and S2). This observation is consistent with the results of hemagglutination assays showing dual binding specificity for this HA (5, II). The side-chain carbonyl of Gln²²⁶ forms a hydrogen bond with the 4-hydroxyl of Gal-2, as observed in other HA/avian receptor complexes (3). In addition, there is a previously unknown water-mediated interaction between the 4-hydroxyl of Gal-2, the main-chain carbonyl of residue 225, and the side chain of Lys²²² (Fig. 3B).

**The 1930-swine HA/human receptor complex.** The sialic acid of the receptor is located similarly in the 1930-swine and 1934-human HA receptor binding sites, but in the 1930 swine complex all five saccharides of the receptor analog are detected (Fig. 3C and figs. S1 and S2). Lys²²² again forms hydrogen bonds with the 2- and 3-hydroxyls of Gal-2, although in this case Gal-2 sits higher in the binding site. Asp¹⁹⁰ hydrogen bonds to the amino nitrogen of GlcNAc-3, Ser¹⁹³ hydrogen bonds to the 2-hydroxyl of Gal-4, and there is a water-mediated interaction between Thr¹⁸⁹ and GlcNAc-5. The last three interactions have not been observed before in HA receptor complexes. In addition, the sialopentasaccharide exits the binding site in an orientation not previously seen, crossing the 190 helix near its N terminus, about parallel to the threefold symmetry axis of the HA trimer (Fig. 3C).

**The 1930-swine HA/avian receptor complex.** The electron density for the avian receptor analog bound to the 1930-swine HA is weak and mainly represents the sialic acid moiety (Fig. 3D and figs. S1 and S2). A similar situation was observed for an H5 avian HA in complex with a human receptor analog, where only a subset of the atoms for the sialic acid could be located (3). These observations probably reflect the low affinity of the HAs for their respective ligands, consistent with the preference of the 1930 swine virus for human receptor in hemagglutination assays as detailed in (II).

To ascertain the essential differences in the binding sites that allow the “avian” binding site structure of H1 HAs to recognize both human and avian receptors, we compared their structures with those of H3 avian (A/duck/Ukraine/63) and human (X-31) HAs determined previously to prefer either avian (23) or human (2) receptors.

**Human receptor complexes.** Complexes of human receptor analog bound to 1934-human HA (green) and 1930-swine HA (blue) and to human H3 HA (red) are superposed in Fig. 4A. Perhaps the most important feature of this comparison is the difference in structure adopted by the 130 and 220 loops of the receptor binding site between the H1 and H3 HAs. One consequence of the change in the 130 loop structure is that the sialic acid of the receptor is tilted about 10° into the receptor binding sites of the H1 HAs. This effect, together with different orientations about the glycosidic bond, contributes to Gal-2 being located almost 2 Å lower in the H1 HAs than in

the human H3 HAs. Gal-2 is able to adopt this position because structural differences in the 220 loop locate Gln²²⁶ lower in the binding site than the equivalent Leu²²⁶ of human H3 HA. Consequently, in the H1 HAs Gal-2 is located closer to the 220 loop and is able to form hydrogen bonds with Lys²²². In the case of the 1934-human HA, Gal-2 also interacts with Asp²²⁵. Thus, a combination of factors relating to the structure of the 130 and 220 loops enable the H1 HAs to make favorable hydrogen-bond interactions with Gal-2 of the human receptor. Gln²²⁶ plays an essentially passive role in this process, in marked contrast to the role played by Leu²²⁶ in the binding of human H3 HA to human receptor. In that case, Gal-2 makes hydrophobic contacts with Leu²²⁶, and the higher position and the nature of this side chain are important for human receptor binding (2, 23).

**Avian receptor complexes.** Complexes of avian receptor analogs with 1934-human HA (green), 1930-swine HA (blue), and an avian H3

Fusion (F')


DTICIGYHANNSTD TVDTVLEKNVT VTHSVNLL EDSHNGKLCRLKGIAPLQLGKCNIAGWLLGNPECD PL
DTICIGYHANNSTD TVDTVLEKNVT VTHSVNLL EDSHNGKLCKLKGIAPLQLGKCNIAGWLLGNPECD LL
DTLCIGYHANNSTD TVDTVLEKNVT VTHSVNLL EDSHNGKLCRLGGIAPLQLGKCNIAGWLLGNPECD LL


Vestigial Esterase (E)


LPVRSSYIVETPENSENGICYPGDFIDYEELREQLSSVSFERFEIFPKESSWPNHNTN.GVTAAC SHEG
LTASSSYIVETSNGICYPGDFIDYEELREQLSSVSFEKFEIFPKTSSWPNHETK.GVTAAC SYAG
LTVSSSYIVETSDSDNGICYPGDFIDYEELREQLSSVSFEKFEIFPKTSSWPNHETTRGVTAAC PYAG


Receptor-binding (RB)


KSSFYRNLWLTEKEGSYPKLKNSYVNKKGEVLVLWGTHPPPNSKEQQNLYQENAYVSVVTSNYNR RF
ASSFYRNLWLTKKGSSYPKLSKSYVNNKGEVLVLWGVHPPPTGTDQQSLYQNADAYVSVGSSKY NR RF
ASSFYRNLWLVKKGNSYPKLSKSYVNNKGEVLVLWGVHPPPTSTDQQSLYQNADAYVSVGSSKY NR RF


(E)


TPEIAERPKVRD QAGR M NYYWTLL KPGDTI IFEAN GNL IAP WYAFAL RRGFGSGIITS NA SMHECN TKCO
TPEIAARPKVRD QAGR M NYYWTLL EPGDTI TFEAT GNL IAP WYAFAL NRGSGSGIITS DAPVHD CN TKCO
TPEIAARPKVRG QAGR M NYYWTLL EPGDTI TFEAT GNL VAP RYAFAL NRGSGSGIITS NAPVHD CD TKCO


(F')


TPLGGINSSLP YQNIHPVTIGECPKYVRS AKLRM VTGLRNIPAR
TPHGAINSSLP FQNIHPVTIGECPKYVRS TKLRMA TGLRNIPAR
TPHGAINSSLP FQNIHPVTIGECPKYVKS TKLRMA TGLRNIPAR


HA1


GLFGAIAGFIEGGWTGM IDGWYGHHQNEQSGYAADQKSTQNAINGITNKVNSVIEKMNIQFTAVGKEF
GLFGAIAGFIEGGWTGM IDGWYGHHQNEQSGYAADQKSTQNAIDGITNKVNSVIEKMNTQFTAVGKEF
GLFGAIAGFIEGGWTGL IDGWYGHHQNEQSGYAADQKSTQNAIDGITNKVNSVIEKMNTQFTAVGKEF


Fusion (F)


SEIEKRMENLN NKVDDGFLDIWTYNAELLVLLENERTLD EHD SNV N NLYEKV KSQ LKNNAKEIGNGC FEF
NNLERRIENLNK KVDDGFLDIWTYNAELLVLLENERTLD FHD SNV R NLYEKV KSQ LKNNAKEIGNGC FEF
NNIERRIENLNK KVDDGFLDIWTYNAELLVLLENERTLD FHD SNV K NLYEKV RSQ LRNNAKEIGNGC FEF


HA2


YHKCDNECMESVRNGTYDYP
YHKCDDACMESVRNGTYDYP
YHKCDDACMESVRNGTYDYP


Fig. 1. Sequence alignment of 1918-human, 1930-swine, and 1934-human HAs. The subdomain structure of HA in the two polypeptides, HA1 and HA2, that form each monomer is indicated by the colored bars above the sequences. Carbohydrate attachment sites are shaded gray. Residue numbering is on the basis of H3 HA sequence (1). Nonconserved residues are in red.

Table 1. Crystallographic statistics. More extensive crystallographic data are given in table S1.

| H1 HA type           | Resolution (Å) | $R_{\text{work}}$ (%) | $R_{\text{free}}$ (%) | Rmsd bonds |
|----------------------|----------------|-----------------------|-----------------------|------------|
| Human 1918          | 2.9            | 24.8                  | 28.9                  | 0.007      |
| Swine 1930 uncomplexed | 2.7            | 24.8                  | 26.5                  | 0.006      |
| + avian receptor     | 2.5            | 21.4                  | 25.4                  | 0.006      |
| + human receptor     | 2.5            | 21.7                  | 25.4                  | 0.007      |
| Human 1934 uncomplexed | 2.3            | 23.5                  | 26.6                  | 0.006      |
| + avian receptor     | 2.2            | 22.7                  | 26.0                  | 0.006      |
| + human receptor     | 2.25           | 22.5                  | 25.7                  | 0.006      |

www.sciencemag.org SCIENCE VOL 303 19 MARCH 2004 1839

RESEARCH ARTICLES

HA (red) (22) are overlaid in Fig. 4B. Again the differences in the structure of the 130 loop between the H1 and H3 HAs result in the sialic acid of the avian receptor being located lower in the receptor binding site of the H1 HAs. Comparison of the 1934-human and avian H3 complexes also reveals that Gal-2 of the avian receptor is located about 1 Å lower in the binding site of the H1 complex, as is Gln²²⁶. In both complexes, the 4-hydroxyl of Gal-2 hydrogen-bonds with the side-chain carbonyl of Gln²²⁶ (Fig. 3B), and the coordinated differences in position of the bound receptor and Gln²²⁶ enable this interaction to be conserved. It seems therefore that 1934-human HA is able to bind avian receptor in a manner reminiscent of avian HAs (3), with Gln²²⁶ playing a key role.

Given that the overall structures of 1930-swine and 1934-human HA receptor binding sites are very similar (Fig. 2B) and that both contain a glutamine residue at position 226, why does the 1930-swine HA bind less effectively to avian receptors than 1934-human HA? The reason seems to be that the position adopted by Gln²²⁶ in 1934-human HA is about 1 Å higher in its complex with the avian receptor than it is either in the human receptor complex or uncomplexed (Fig. 5). By contrast, the position of Gln²²⁶ in 1930-swine HA is about the same uncomplexed and in the human and avian receptor complexes. The apparent inability of Gln²²⁶ to adopt a higher position in the receptor binding site seems to explain the failure of 1930-swine HA to interact as effectively with the avian receptor. This explanation is supported by the structural observation that, in the 1934-human HA complex with avian receptor, Glu¹⁹⁰ interacts through two water molecules with Gln²²⁶ (Fig. 5). This network of hydrogen bonds may be necessary to position Gln²²⁶ in the binding site for its interaction with Gal-2. Glu¹⁹⁰ is conserved in avian H1 HAs, all of which specifically bind α2,3-linked receptors. By contrast, residue 190 of 1930-swine HA is an aspartic acid, which does not interact with either the 9-hydroxyl of Sia-1 or Gln²²⁶ and is thus unable to facilitate binding to avian receptor.

Many of the viruses from the first H1 influenza pandemic period, 1918 to 1957, recognized to some extent both avian and human receptors, whereas those from the second period, 1977 to date, appear to be more human receptor-specific (11). Loss of the ability to bind avian receptors, which may be an advantage in the face of infectivity-blocking α2,3-linked soluble sialosides in human lungs (13), was proposed to correlate with the amino acid substitution of Ala¹³⁸ to Ser¹³⁸ (11). Our structural data also support this interpretation.

---

**Fig. 2. Structures of 1918-human, 1934-human, and 1930-swine HAs.**

(A) Ribbons diagram of the trimer of 1918-human HA. Monomers 2 and 3 are in silver and gold, respectively, and the featured monomer is colored according to its individual subdomains as in Fig. 1: receptor binding (RB) in blue, vestigial esterase (E) in yellow, and fusion subdomains (F' and F) in magenta and red, respectively (21). Changes in the relative orientations of individual subdomains of H1 HA relative to H3 HA can usefully be described (21) after superposing the two structures with the use of just the F domain. The RB, E, and F' subdomains of H1 are related to the equivalent subdomains in H3 by a clockwise rotation of 23° about the trimer threefold axis. Glycosylation sites are indicated by spheres; in green are those observed in the 1918-human HA (21, 33, 94, and 289 in the F' and E subdomains and 154 in the F subdomain) and in red are the sites (63, 81, 129 or 131, 158, 163, and 271 in the F', E, and RB subdomains) that have accumulated on the membrane distal surface between 1918 and 2002 (17, 25, 29) that probably influence antigenicity. (B) An expanded view of the superposed polypeptide backbone of the receptor binding site of all three H1 HAs. The position of the three secondary structure units making up the site, the 190 helix, and the 130 and 220 loops are indicated. Also shown are the side chains of some residues important for receptor binding. Certain Cα positions are indicated by black spheres for residues discussed in the text. Overall, the three H1 structures are very similar, with rmsd on all Cα of 0.82 Å and 0.56 Å for 1918-human versus 1934-human and 1930-swine, respectively. (C) An expanded view of a region of the F subdomain indicating differences between H1 and H5 subtype HAs in the position of the loop connecting helix A to helix B. Because H1 and H5 are in the same phylogenetic clade, these differences imply that the position of the interhelical loop relative to helix B is not a clade-specific feature, as suggested previously (21). Interactions between the C-terminal region of the loop and the RB and E subdomains (110 helix) influence the dispositions of the subdomains relative to the central coiled coil formed in the trimer by the B helices.

pretation, because an interaction between Ser<sup>138</sup> and Gln<sup>226</sup> is feasible and would favor the lower positioning in the receptor binding site of Gln<sup>226</sup> and the consequent negative effect on avian receptor binding.

**The receptor binding specificity of the 1918-human HA.** Although we have been unable to obtain receptor analog complexes with the 1918 HA, we can deduce its likely specificity on the basis of our observations of receptor binding to

1930-swine and 1934-human HAs. Table 2 compares potentially important contact residues in the receptor binding sites of 1934-human, 1930- swine, 1918-human, and a representative avian H1 HA (9). From these data, and the close similarity of the structures of the human and swine HA binding sites (Fig. 2), two of the five HAs reported from 1918 viruses (17, 24) have receptor binding sites, and presumably binding specificities, indistinguishable from those of 1930-swine HA. The other three 1918 HAs, which differ only at residue 225 (Table 2), most likely also prefer binding to human rather than avian receptors. If interactions with Lys<sup>222</sup> and Asp<sup>225</sup> are formed with human receptors, as in the 1934-human HA complex, then the overall orientation of the oligosaccharide in the 1918 HA binding sites may also be similar. In this case, Asp<sup>190</sup> would not contact the amino group of GlcNAc-3, and the entire complex would closely resemble that formed by the 1934-

![Figure 3](#fig3)

**Fig. 3.** Interactions of 1934-human HA (top) and 1930-swine HA (bottom) with human receptor and with avian receptor analogs. The view of the receptor binding site is about the same as in Fig. 2B. The three secondary structure components of the binding site are labeled in this backbone representation together with some of the most relevant side chains. The broken lines indicate potential hydrogen-bond interactions between the protein and the receptors; residues making interactions via main-chain carbonyl groups are shown as red spheres, whereas those interacting via main-chain nitrogens are shown as blue spheres. In all four panels, the sialosaccharides are colored yellow for carbon atoms, blue for nitrogen, and red for oxygen. Water molecules are indicated by green spheres. (A) 1934-human HA in complex with human receptor and (B) 1934-human HA in complex with avian receptor; in both cases, the HA is colored in green for backbone and carbon atoms. (C) 1930-swine HA in complex with human receptor and (D) 1930-swine HA in complex with avian receptor; in these cases, the backbone and carbon atoms of the HA are colored in blue. The small black arrows in (A), (B), and (C) indicate that for the two human receptor complexes the Sia-1-Gal-2 linkage adopts a *cis* conformation about the glycosidic bond, whereas for the avian complex it adopts a trans conformation. The large black arrow in (C) indicates the direction of an axis parallel to the trimer threefold axis.

human HA rather than by the 1930-swine HA. However, the fact that Asp<sup>190</sup> is often conserved in human H1 HAs (25), as is Ser<sup>193</sup>, suggests that, in the 1918 HA and in H1 HAs generally, these residues interact with GlcNAc-3 and Gal-4 in similar ways to those seen in the 1930-swine HA-human receptor complex. This conclusion is also consistent with the ability of human H1 HAs to discriminate between the receptor analogs 6′-sialyllactosamine and 6′-sialylactose that

is reported to be dependent on Asp<sup>190</sup> (26, 27). Irrespective of the single amino acid difference of Asp or Gly at residue 225 (Table 2) between the sequences of 1918-human HAs, by recognizing human receptors all would contribute the first requirement of an epidemic virus: the ability to spread in the human population. The importance of this requirement was emphasized in the 1997 outbreak of H5 “chicken” influenza in Hong Kong, when the virus was extremely virulent but did not acquire the ability to bind α2,6-linked sialosides (28) and was therefore unable to spread. With the ability to ensure the efficiency of the initial stages of virus infection, coupled with novel antigenicity, the human-1918 HA may have been the prime determinant of extensive mortality in the 1918 pandemic.

**References and Notes**

1. J. J. Skehel, D. C. Wiley, *Annu. Rev. Biochem.* **69**, 531 (2000).
2. M. B. Eisen, S. Sabesan, J. J. Skehel, D. C. Wiley, *Virology* **232**, 19 (1997).
3. Y. Ha, D. J. Stevens, J. J. Skehel, D. C. Wiley, *Proc. Natl. Acad. Sci. U.S.A.* **98**, 11181 (2001).
4. M. Matrosovich et al., *J. Virol.* **74**, 8502 (2000).
5. G. N. Rogers, J. C. Paulson, *Virology* **127**, 361 (1983).
6. G. N. Rogers et al., *Nature* **304**, 76 (1983).
7. N. K. Sauter et al., *Biochemistry* **28**, 8388 (1989).
8. W. Weis et al., *Nature* **333**, 426 (1988).
9. E. Nobusawa et al., *Virology* **182**, 475 (1991).
10. T. Ito et al., *J. Virol.* **72**, 7367 (1998).
11. G. N. Rogers, B. L. D'Souza, *Virology* **173**, 317 (1989).
12. L. G. Baum, J. C. Paulson, *Acta Histochem. Suppl.* **40**, 35 (1990).
13. J. N. Couceiro, J. C. Paulson, L. G. Baum, *Virus Res.* **29**, 155 (1993).
14. R. G. Webster, *Proc. Natl. Acad. Sci. U.S.A.* **96**, 1164 (1999).
15. C. W. Naeve, V. S. Hinshaw, R. G. Webster, *J. Virol.* **51**, 567 (1984).
16. R. J. Connor, Y. Kawaoka, R. G. Webster, J. C. Paulson, *Virology* **205**, 17 (1994).
17. A. H. Reid, T. G. Fanning, J. V. Hultin, J. K. Taubenberger, *Proc. Natl. Acad. Sci. U.S.A.* **96**, 1651 (1999).
18. T. Francis, *Science* **80**, 457 (1936).
19. R. E. Shope, *J. Exp. Med.* **54**, 373 (1931).
20. W. Smith, C. H. Andrewes, P. O. Laidlaw, *Lancet* **1**, 66 (1933).
21. Y. Ha, D. J. Stevens, J. J. Skehel, D. C. Wiley, *EMBO J.* **21**, 865 (2002).
22. R. Russell et al., in preparation.
23. Y. Ha, D. J. Stevens, J. J. Skehel, D. C. Wiley, *Virology* **309**, 209 (2003).
24. A. H. Reid et al., *Emerg. Infect. Dis.* **9**, 1249 (2003).
25. F. L. Raymond, A. J. Caton, N. J. Cox, A. P. Kendal, G. G. Brownlee, *Nucleic Acids Res.* **11**, 7191 (1983).
26. M. N. Matrosovich et al., *Virology* **196**, 111 (1993).
27. A. S. Gambaryan et al., *Virology* **232**, 345 (1997).
28. M. Matrosovich, N. Zhou, Y. Kawaoka, R. Webster, *J. Virol.* **73**, 1146 (1999).
29. M. Bennett, V. Gregory, Y.-P. Lin, A. J. Hay, unpublished data.
30. We thank S. Smerdon and P. Walker for assistance and discussion. Diffraction data were collected at Daresbury Synchrotron Radiation Source (SRS) and European Synchrotron Radiation Facility (ESRF) Grenoble. We thank J. Nicholson (SRS) and S. McSweeney (ESRF) for rapid access to and assistance with beamtime as well as other beamline staff. This research was supported by the MRC (UK), NIH grant AI-13654, by a supplement to this grant for Expanded International Research on Emerging and Re-Emerging Diseases, by an International Partnership Research Award in Veterinary Epidemiology of the Wellcome Trust, and by the Howard Hughes Medical Institute. The coordinates for the 1918-human, 1934-human, 1934-human/human, 1934-human/avian, 1930-swine, 1930-swine/human, and 1930-swine/avian HAs have been deposited in the Protein Data Bank (accession codes 1RUZ, 1RU7, 1RVZ, 1RVX, 1RUY, 1RVT, and 1RV0).

**Supporting Online Material**

www.sciencemag.org/cgi/content/full/1093155/DC1

Materials and Methods  
Figs. S1 and S2  
Tables S1 to S3  
References and Notes  

30 October 2003; accepted 7 January 2004  
Published online 5 February 2004  
10.1126/science.1093155  

Include this information when citing this paper.


**Fig. 4. Differences in the orientations of bound receptors in the receptor binding sites of three different HAs.** The receptor binding sites of 1934-human HA (green), 1930-swine HA (blue), and human H3 HA (red) are overlaid. The view matrix is about the same as in Fig. 3. The sialopentasaccharides are colored according to the HAs to which they are bound. The side chains of Gln<sup>226</sup> (H1 HAs) and Leu<sup>226</sup> (H3 HA) are shown. (A) Human receptor complexes. (B) Avian receptor complexes.

**Fig. 5. Superposition of the binding site of 1934-human HA in its uncomplexed state and complexed with avian receptor analog.** The HA is shown in green in both cases, and the avian receptor is colored as in Fig. 3. Two water molecules, shown as green spheres, link Glu<sup>190</sup> to Gln<sup>226</sup> in the avian receptor complex. This hydrogen-bonded network is not formed in the uncomplexed structure or in the human receptor complex not shown.

**Table 2. Residue types at four positions in the receptor binding sites of 1918-human, 1930-swine, 1934-human, and a representative avian H1 HA.** Numbers in parentheses are the number of sequences available.

| Residue of HA | 1918-human | 1930-swine | 1934-human | Avian |
|---------------|------------|------------|------------|-------|
| 190           | Asp        | Asp        | Glu        | Glu   |
| 193           | Ser        | Ser        | Asn        | Ser   |
| 222           | Lys        | Lys        | Lys        | Lys   |
| 225           | Asp (3)/Gly (2) | Gly        | Asp        | Gly   |

1842 19 MARCH 2004 VOL 303 SCIENCE www.sciencemag.org
