
Appl Microbiol Biotechnol (2004) 64: 745–755  
DOI 10.1007/s00253-004-1566-x  

MINI-REVIEW  

K. Yabe · H. Nakajima  

# Enzyme reactions and genes in aflatoxin biosynthesis  

Received: 22 September 2003 / Revised: 15 December 2003 / Accepted: 22 December 2003 / Published online: 12 March 2004  
© Springer-Verlag 2004  

Abstract Aflatoxins are highly toxic and carcinogenic substances mainly produced by *Aspergillus flavus* and *Aspergillus parasiticus*. Sterigmatocystin is a penultimate precursor of aflatoxins and also a toxic and carcinogenic substance produced by many species, including *Aspergillus nidulans*. Recently, the majority of the enzyme reactions involved in aflatoxin/sterigmatocystin biosynthesis have been clarified, and the genes encoding the enzymes have been isolated. Most of the genes constitute a large gene cluster in the fungal genome, and their expression is mostly regulated by a product of the regulatory gene *aflR*. This review will summarize the enzymatic steps and the genes in aflatoxin/sterigmatocystin biosynthesis.

hepatitis B virus infection and aflatoxin exposure as major risk factors for liver cancer. Also, annual costs resulting from crop losses and the need to limit food contamination have been estimated to be more than $100 million in the United States. (Busby and Wogan 1985; Massey et al. 1995; Payne 1998; Robens 2001, http://www.apsnet.org/online/feature/mycotoxin/top.html).

*A. parasiticus* generally produces all four kinds of major aflatoxins, whereas *A. flavus* produces only AFB₁ and AFB₂. More than 20 *Aspergillus* spp. produce as their final product sterigmatocystin, which is the penultimate precursor of aflatoxin (Bennett and Deutsch 1986; Horie et al. 1988). Based on molecular genetic approaches, the functions of the genes coding for sterigmatocystin formation in *A. nidulans* are similar to those of the genes of *A. parasiticus* and *A. flavus*. The methods used to determine the aflatoxin/sterigmatocystin (AF/ST) biosynthetic pathway have included: (1) isolation of mutants accumulating precursors, (2) feeding experiments with a substance expected to be a precursor of aflatoxin, (3) cell-free biochemical studies, (4) complementation of a blocked mutant with a DNA library, (5) DNA sequencing and gene walking, and (6) gene disruption followed by characterization of the disruptant.

Numerous reviews have reported on AF/ST biosynthesis and the respective genes (Trail et al. 1995b; Keller and Horn 1997; Minto and Townsend 1997; Payne and Brown 1998; Woloshuk and Prieto 1998; Brown et al. 1999; Cary et al. 2000; Yabe 2002; Bhatnagar et al. 2003). Because information about the characterization of these enzymes and genes should help to provide a more complete image of AF/ST biosynthesis, this review lays emphasis on each enzyme step involved in AF/ST biosynthesis and is based on information obtained from enzymological as well as molecular biological studies.

---

## Introduction

Aflatoxins are highly toxic and carcinogenic in animals and humans, producing acute liver damage, liver cirrhosis, tumor induction, and teratogenesis. Aflatoxins B₁ (AFB₁), B₂ (AFB₂), G₁ (AFG₁), and G₂ (AFG₂) are major aflatoxins produced by fungi, mainly *Aspergillus flavus* and *A. parasiticus*. AFB₁ is the most toxic and is most abundantly produced by fungi, and contamination of agricultural commodities with AFB₁ is a very serious problem all over the world. Liver cancer incidence is high in regions with high endemic aflatoxin concentrations. In fact, it is estimated that 250,000 deaths occur annually in certain parts of China and sub-Saharan Africa due to hepatocellular carcinoma. Epidemiological studies have established

---

K. Yabe (✉)  
National Food Research Institute,  
Tsukuba,  
305-8642 Ibaraki, Japan  
e-mail: yabek@nfri.affrc.go.jp  
Tel.: +81-29-8388050  
Fax: +81-29-8388122  

H. Nakajima  
Faculty of Agriculture, Tottori University,  
680-8553 Tottori, Japan  

---

## The aflatoxin gene cluster

At least 18 enzyme steps are required for conversion of acetyl coenzyme A (acetyl CoA) to its final products,
AFB₁, AFB₂, AFG₁, and AFG₂. Many genes encoding the enzymes and the transcription factors have been cloned and characterized. They are located within a huge gene cluster of about 70 kb in the genomes of *A. parasiticus* and *A. flavus*. The direction of each transcription is unique in each gene (Trail et al. 1995a; Minto and Townsend 1997; Woloshuk and Prieto 1998; Brown et al. 1999), and the structures of the gene clusters in *A. flavus* and *A. parasiticus* are mostly the same (Fig. 1). The enzyme genes involved in sterigmatocystin biosynthesis in *A. nidulans* also form a gene cluster of about 60 kb. However, the positions of individual genes are relatively different from those of the same genes in other aflatoxigenic fungi. The order and direction of transcripts of homologous genes in the ST and AF gene clusters were summarized by Keller and Hohn (1997). Although the AF/ST gene clusters have been sequenced, the functions of several genes have still not been clarified.

Expression of all characterized enzyme genes except *estA* is regulated by *aflR*, which codes for a transcription factor (Payne et al. 1993; Woloshuk et al. 1994; Fernandes et al. 1998; Chang et al. 1999). The *aflR* was complementary to a mutant that produced neither aflatoxin nor pigment precursors. This gene is located in both the AF and ST gene clusters, and encodes a GAL4-type C6-zinc binuclear DNA-binding protein that contains within its DNA-binding domains a conserved cysteine-rich region encoding Cys-(Xaa)₂-Cys-(Xaa)₆-Cys-(Xaa)₆-Cys-(Xaa)₂-Cys-(Xaa)₆-Cys-(Xaa)₂. *AflR* belongs to a large family of fungal transcriptional activators included in some catabolic pathways found in filamentous fungi and yeasts. It positively regulates the expression of enzymes involved in AF/ST synthesis by binding to the consensus motif TCGN5CGR in the promoter regions of the respective genes. The function of *aflR* is conserved between *A. nidulans* and *A. flavus* (Yu et al. 1996). *aflJ* has also been suggested to encode a transcription factor (Meyers et al. 1998).

CoA, which is synthesized via oxidative decarboxylation of pyruvate, β-oxidation of fatty acids, or from ketogenic amino acids in primary metabolism. In AF/ST biosynthesis, hexanoate is the first substance produced from acetyl-CoA (Townsend et al. 1984). Although the detailed reaction for the formation of hexanoate in aflatoxin biosynthesis has not been biochemically clarified, the same reaction that occurs in fatty acid-synthesis is considered to be involved. In the aflatoxin gene cluster, *fasl* and *fas2* are located side by side, and the involvement of *thefas1A* (*fasl*) in hexanoate formation was confirmed by gene disruption in *A. parasiticus* (Mahanti et al. 1996). *fasl* (*hexA*) and *fas2* (*hexB*) have high similarity to *fasβ* and *fasα* of other, already known, fungal fatty acid synthase genes (Watanabe et al. 1996; Woloshuk and Prieto 1998; Hitchman et al. 2001). Regions of *fasl* contain sequences similar to those of genes encoding acyl transferase, enoyl hydratase, and enoyl-ACP reductase, while *fas2* contains sequence regions with similarities to genes for acyl carrier protein (ACP), β-ketoacyl synthase, and β-ketoacyl reductase.

In *A. nidulans*, gene disruption of either *steJ* or *steK* inhibited sterigmatocystin production, whereas fungal growth was not affected (Brown et al. 1996a), indicating that these genes may encode the fatty acid synthases involved in sterigmatocystin biosynthesis. A C₆-chain acyl CoA is the final product in AF/ST biosynthesis, whereas the expected C₁₆ or C₁₈ is the product in fatty acid formation of primary metabolism.

### Reaction 2: hexanoate to hexanoyltetrahydroxyanthrone

Although many fungal polyketide synthetases (PKSs) generally use acetate as the starting compound for polyketide synthesis, those involved in AF/ST biosynthesis use the resultant hexanoate from reaction 1 (Townsend et al. 1984). *pks* genes were isolated from *A. parasiticus* (Trail et al 1995a; Chang et al. 1995; Feng and Leonard 1995) and *A. nidulans* (Yu and Leonard 1995). The *pks* of *A. parasiticus* encodes a large protein composed of 2,109 amino acids and contains four functional domains, i.e., acyl carrier protein, β-ketoacyl carrier protein synthase, acyltransferase, and thioesterase, which are usually present in PKS and fatty acid synthases. However, *A. parasiticus* *pks* does not contain the two reductase domains, that are generally essential for fatty acid synthesis. The absence of

---

### Biosynthetic pathway and genes of aflatoxin biosynthesis

#### Reaction 1: acetyl CoA to hexanoate

The reaction steps leading to AF/ST formation are shown in Figs. 2 and 3; the number of the reaction corresponds to the number in the text. The pathway starts from acetyl-

![Diagram](attachment:diagram.png)

Fig. 1 The aflatoxin gene cluster. The order of the genes involved in aflatoxin biosynthesis is shown (the lengths of the bars do not correspond to the size of the gene). Also, genes whose structures and functions have been experimentally clarified are indicated. Numbers in parentheses beneath the genes reference functions described in the text. The letter “R” under the *aflR* indicates a transcription factor that regulates aflatoxin production. Open brackets show that the function of the corresponding gene is unknown.
Fig. 2 The pathway from acetyl CoA to averufin in aflatoxin biosynthesis (reactions 1–7). Dashed arrow unconfirmed reaction, open arrows a reaction catalyzed by a microsome enzyme, solid arrow a reaction catalyzed by a cytosolic enzyme. The microsome enzyme catalyzes the reaction from AVN to HAVN (5). The reaction from HAVN to OAVN requires NAD. The enzymes are: (1) fatty acid synthase (hexanoate synthase), (2) polyketide synthetase, (3) norsolorinic acid (NA) synthase, (4) norsolorinic acid (NA) reductase, (5) averantin (AVN) monooxygenase, (6) 5′-hydroxyaverantin (HAVN) reductase, (7) 5′-oxoaverantin (OAVN) cyclase. (Enzymes that are as yet unidentified are referred to using the name of the reaction product and “synthase”)

the reductase domains causes the formation of β-polyketone. The mechanism of polyketide synthesis in *A. parasiticus* is suggested by analogy to polyketide biosynthesis in other microorganisms (Hutchinson 1999; Kennedy et al. 1999). After formation of the poly-β-ketoacyl thioester, cyclization of the chain occurs on PKS, thereby producing an anthrone having a hexanoyl and four hydroxyls, i.e. 2-hexanoyl-1,3,6,8-tetrahydroxy-9(10H)-anthracenone. A multienzyme complex of fatty acid synthase and polyketide synthetase has recently been suggested (Watanabe et al. 1996; Watanabe and Townsend 2002). However, the exact mechanism remains to be determined due to the lack of cell-free studies.

Fig. 3 The pathway from averufin to aflatoxins (reactions 8–18). Arrows represent reactions as described for Fig. 2, and cofactors are shown as in Fig. 2.

The microsome enzymes catalyze the reactions from AVR to HVN (8), VB to VA (13), OMST to AFB₁/G₁ and DHOMST to AFB₂/AFG₂ (17, 18). The pathway from HVN to VHOH forms a metabolic grid that is catalyzed by three kinds of enzymes, versiconal hemiacetal acetate (VHA) synthase (9), esterase (10), and versiconal hemiacetal acetate (VHA) reductase (11). The enzymes shown in the figure are: (8) averufin (AVR) monooxygenase, (9) versiconal hemiacetal acetate (VHA) synthase, (10) esterase, (11) versiconal hemiacetal acetate (VHA) reductase, (12) versiconal (VHOH) cyclase (versicolorin B synthase), (13) versicolorin B (VB) desaturase, (14) demethylsterigmatocystin (DMST) synthase system, (15) O-methyltransferase I, (16) O-methyltransferase II, (17) OrdA enzyme, (18) G-aflatoxin synthase. Enzymes whose features have not been clarified are referred to here using the name of the reaction product and “synthase”

![Chemical Structures](chemical_structures.png)

Reaction: 3: hexanoyltetrahydroxyanthrone to norsolorinic acid

The 2-hexanoyl-1,3,6,8-tetrahydroxy-9(10H)-anthracene proposed herein has not been isolated, and the enzymatic reaction from it to norsolorinic acid (NA) has not been clarified. However, induction of an oxygen atom to the anthrone to form NA suggests that a monooxygenase and an oxidoreductase are involved in the pathway.

Reaction 4: norsolorinic acid to averantin

NA is generally accepted to be the first pigmented and stable precursor in AF/ST biosynthesis (Bennett and Christensen 1983). NA is converted to averantin (AVN) by a cytosol oxidoreductase in the presence of NADPH or NADH. This reaction is reversible when NADP or NAD is present (Yabe et al. 1991c). The equilibrium of the reversible reaction shifts to NA formation in a cell-free system because the compound is stabilized by the formation of hydrogen bonds between the 1'-oxo group and two neighboring hydroxyl groups, 1-OH and 3-OH, on the NA molecule (Yabe et al. 1991c). The oxidoreductase shows strict stereospecificity for the 1' carbon atom of the substrate; (1'S)-AVN is exclusively produced from NA. Also, (1'S)-AVN, but not (1'R)-AVN, serves as a substrate for the reverse reaction from AVN to NA. The stereospecificity of the oxidoreductase determines subsequent reactions leading to the formation of (1'S)-averufin (Koreeda et al. 1984; Yabe et al. 1993).

nor-1, which encodes the oxidoreductase was cloned by complementation of a NA-accumulating mutant from *A. parasiticus* (Trail et al. 1994). This gene codes a 29-kDa protein and contains a short-chain alcohol dehydrogenase motif in its amino acid sequence. Recombinant Nor-1 was expressed in *Escherichia coli* and converted NA to AVN in the presence of NADPH (Zhou and Linz 1999). Disruption of nor-1 caused an accumulation of NA and a drastic decrease of aflatoxins. However, the nor-1-disrupted transformant still produced a slight amount of aflatoxin B₁ (Trail et al. 1994). This incomplete inhibition suggested that a different enzyme might be involved in the conversion of NA to AVN, although Nor-1 is the enzyme primarily involved in aflatoxin biosynthesis. stcE is a homolog of nor-1 in *A. nidulans* (Brown et al. 1996b).

norA in the aflatoxin gene cluster was independently isolated by using a monoclonal antibody raised against the 43-kDa protein, which had been purified as a NA reductase (Bhatnagar and Cleveland 1990; Cary et al. 1996). The predicted amino acid sequence of the norA product shows a low identity (22%) to that of nor1. Since disruption of norA did not cause an accumulation of NA (Cary et al. 1999), the function of norA is still unknown. In *A. nidulans*, stcV is a homolog of norA (Brown et al. 1996b).

Reaction 5: averantin to 5'-hydroxyaverantin

(1'S)-AVN is hydroxylated to form 5'-hydroxyaverantin (HAVN) by a microsome enzyme dependent on the presence of NADPH (Yabe et al. 1991c; 1993). This enzyme, AVN monooxygenase, shows strict stereospecificity for the 1' carbon of AVN. (1'R)-AVN does not serve as the substrate of this enzyme. In contrast, AVN monooxygenase does not show stereospecificity for hydroxylation at the 5' carbon of (1'S)-AVN, because both (1'S, 5'S)- and (1'S, 5'R)-HAVN are formed (Yabe et al. 1993). HAVN is an unstable substance that is easily converted to averufanin through artificial dehydration (Yabe et al. 1991c; 1993). Although averufanin had been previously suggested to be an intermediate between AVN and averufin (McCormick et al. 1987), this is not supported by enzymological or genetic data.

avnA of *A. parasiticus*, which encodes AVN monooxygenase, was isolated, and disruption of the gene caused an accumulation of AVN (Yu et al. 1997). The deduced amino acid sequence of avnA indicates that it encodes a cytochrome P-450-type monooxygenase. In sterigmatocystin biosynthesis in *A. nidulans*, stcF encodes a P-450 monooxygenase similar to the avnA of *A. parasiticus* (Brown et al. 1996b).

Reaction 6: 5'-hydroxyaverantin to 5'-oxoaverantin

The resultant (1'S, 5'S)- and (1'S, 5'R)-HAVN are converted to (1'S, 5'S)-averufin (AVR) (Yabe et al. 1993; Koreeda et al. 1985). [Correction: in our previous paper (Yabe et al. 1993), we designated the absolute configurations of the resultant AVR as the (1'S, 5'R)-AVR; we herein correct that to (1'S, 5'S)-AVR.] Both HAVNs are converted to oxoaverantin (OAVN) by a cytosol dehydrogenase. This enzyme is a homodimer composed of 28-kDa subunits, and it requires NAD, but not NADP, as a cofactor for its activity (Sakuno et al. 2003). The amino acid sequence of the purified enzyme confirmed that this enzyme is encoded by adhA, and disruption of adhA caused an accumulation of HAVN (Chang et al. 2000). The deduced amino acid sequence of adhA is consistent with a typical short-chain alcohol dehydrogenase, indicating that HAVN is converted to OAVN by dehydrogenation, but is not converted to averufanin by dehydration.

Reaction 7: 5'-oxoaverantin to averufin

OAVN is converted to (1'S, 5'S)-averufin (AVR) by an OAVN cyclase via intramolecular acetal formation among the resultant 5'-ketone and two hydroxyl groups, 1'-OH and 3-OH. The cyclase is a homodimer composed of 79-kDa subunits and does not require a cofactor for its activity (Sakuno et al. 2003). As noted above averufanin is not an intermediate between AVN and AVR and is thus not involved in aflatoxin biosynthesis.

Reaction 8: averufin to hydroxyversicolorone

Hydroxyversicolorone (HVN) was isolated from a blocked mutant, *hvn-1* (Townsend et al. 1988a, b). (1'S, 5'S)-AVR is converted to HVN by a microsome enzyme in the presence of NADPH (Yabe et al. 2003), and Yabe and Nakajima herein name this enzyme averufin (AVR) monooxygenase. This enzyme shows strict stereospecificity for the configuration of AVR, because (1'R, 5'R)-AVR does not serve as substrate.

*avfA* complemented the AVR-accumulating mutant, indicating that the gene encodes averufin monooxygenase (Yu et al. 2000a). In the sterigmatocystin gene cluster of *A. nidulans*, *stcB* and *stcW* encode a putative P-450 monooxygenase and a putative flavin-requiring monooxygenase, respectively. Disruption of either *stcB* or *stcW* led to the accumulation of AVR, suggesting that these genes are involved in the reaction from AVR to HVN (Keller et al. 2000); however, the exact functions of these genes have not been clarified. In *A. parasiticus*, homologs of these genes were isolated as *cypX* and *moxY*, respectively (Yu et al. 2000b).

VHOH and VOAc to VOH. However, another, unknown enzyme is also partially involved (Chang et al., unpublished data). Interestingly, RT-PCR analysis showed that *estA* is constitutively expressed irrespective of aflatoxin-inducible or non-inducible conditions (Yu et al. 2003). In the sterigmatocystin gene cluster of *A. nidulans*, *stcI* is suggested to encode the esterase based on the deduced amino acid sequence (Brown et al. 1996b).

Reaction 11: hydroxyversicolorone to versicolorone, versiconal hemiacetal acetate to versicolol acetate, and versiconal to versicolol in the metabolic grid

In aflatoxin biosynthesis, the reaction from HVN to VHA to VHOH is probably the main pathway. However, HVN, VHA, and VHOH also function as substrates for a reductase/dehydrogenase enzyme, VHA reductase (Matsushima et al. 1994), which converts them to versicolorone (VONE), VOAc, and VOH, respectively. VHA reductase was purified as a 40-kDa protein (Matsushima et al. 1994) and requires NADPH, but not NADH, for its activity. The enzyme is encoded by *vrdA*, which is not present in the aflatoxin gene cluster (Motomura, unpublished data).

Under aqueous physiological conditions, VONE has an open form containing two hydroxyl groups, 3-OH and 1'-OH, although it was previously reported as a closed ring (Bergar-Deguee and Bergar 1982; Townsend et al. 1988a). In addition, VONE is converted to VOAc and then to VOH by the same enzymes involved in the main pathway (from HVN to VHA to VHOH). This side pathway from VONE to VOH functions within the cells, and elevation of NADPH levels in as the culture ages appears to shift the equilibrium of both pathways to the side pathway. Thus, the components involved in the two pathways from HVN to VHOH make up an extensive metabolic grid (metabolic network). The reverse reactions, i.e. VONE to HVN, VOAc to VHA, and VOH to VHOH have not been observed in a cell-free system (Yabe, unpublished data). However, these reactions likely occur in cells, because small but significant amounts of aflatoxins were produced from VONE (Yabe et al. 2003), VOAc and VOH (Yabe et al. 1991a) in feeding experiments.

Reaction 9: Hydroxyversicolorone to versiconal hemiacetal acetate

HVN is converted to versiconal hemiacetal acetate (VHA) by a cytosolic enzyme that requires NADPH as a cofactor (Yabe et al. 2003), and Yabe and Nakajima herein name this enzyme versiconal hemiacetal acetate (VHA) synthase. The conversion takes place via a Baeyer-Villiger reaction, in which an oxygen atom is inserted into a C-C bond adjacent to the ketonic carbonyl group on the side chain. FAD-containing monooxygenase enzymes showing similar reactions have already been purified (Miyamoto et al. 1995). Although the gene encoding VHA synthase has not been identified, *stcW* of *A. nidulans* and *moxY* of *A. parasiticus* are candidates because they encode flavin-requiring monooxygenases (Keller et al. 2000; Yu et al. 2000b).

Reaction 10: versiconal hemiacetal acetate to versiconal

VHA is converted to versiconal (VHOH) by more than one cytosol esterase (Kusumoto and Hsieh 1996). The insecticide dichlorvos inhibits the esterase, which causes an accumulation of VHA and versicolol acetate (VOAc) with subsequent loss of aflatoxin production in aflatoxigenic fungi (Yao and Hsieh 1974; Schroeder et al. 1974; Bennett et al. 1976; Yabe et al. 1991a). Although the reaction from VHA to VHOH is the main pathway following the reaction of HVN to VHA, another reaction, from VOAc to versicolol (VOH), occurs in a side pathway (Yabe et al. 1991a).

The enzyme encoded by *estA* in the aflatoxin gene cluster is primarily involved in the reactions from VHA to

Reaction 12: versiconal to versicolorin B

VHOH is a very unstable substance that easily changes to versicolorin C (VC) by spontaneous dehydration, especially under acid or high-temperature conditions. VC is a racemic mixture of (1'R, 2'S) versicolorin B (VB) and its enantiomer. The absolute configuration of VB, (1'R, 2'S), on the bisfuran moiety of the molecule is identical to those of the aflatoxins and sterigmatocystin. VHOH is physiologically converted to VB by the enzyme VHOH cyclase (Lin and Anderson 1992; Yabe and Hamasaki 1993) or versicolorin B synthase (McGuire et al. 1996). VHOH cyclase shows strict stereospecificity for the substrate, because this enzyme exclusively catalyzes the reaction
from (2′S)-VHOH to (1′R, 2′S)-VB (Lin and Anderson 1992; Yabe and Hamasaki 1993). However, VHOH, VHA, and probably HVN, which are precursors of VB, have been observed to be racemates (Yabe and Hamasaki 1993). Although only (2′S)-VHOH is consumed as the substrate of this enzyme to form VB, the remaining (2′R)-VHOH spontaneously racemizes to form the (2′S)-enantiomer under physiological conditions. Thus, racemic VHOH, VHA, and HVN are finally converted to optically pure VB (Yabe and Hamasaki 1993; McGuire et al. 1996). Therefore, VHOH cyclase determines the configuration of aflatoxins and sterigmatocystin.

VHOH cyclase was purified from *A. parasiticus* (Lin and Anderson 1992; McGuire et al. 1996), and the gene, *vbs*, was isolated (Silva et al. 1996). The recombinant protein of this gene showed the expected cyclase activity (Silva and Townsend 1996).

Reaction 13: versicolorin B to versicolorin A

Similar to AFB₂ and AFG₂, VB contains a tetrahydrobisfuran ring in its structure; and, like AFB₁ and AFG₁, versicolorin A (VA) contains a dihydrosfuran ring. The branching step between AFB₁/AFG₁ and AFB₂/AFG₂ is the desaturation reaction from VB to VA (Yabe et al. 1991b). VB is desaturated to VA by an unstable microsome enzyme that requires NADPH and probably O₂ for its activity. Since this desaturase enzyme shows strict stereospecificity for the (2′S)-configuration of VB, the (2′S)-configuration that is selected by the VHOH cyclase is further confirmed by this step (Yabe and Hamasaki 1993). The (1′R, 2′S)-bisfuran structures of VB and VA are retained during the following conversions and finally become parts of aflatoxin molecules. The activity of the desaturase may markedly contribute to determining the final ratio of AFB₁ to AFB₂ as well as that of AFG₁ to AFG₂. However, the kinetic parameters of the subsequent enzymes for dihydro-precursors and tetrahydro-precursors may additionally affect the ratio.

In *A. nidulans*, *stcL* was confirmed to encode the desaturase (Kelkar et al. 1997). Mutation or disruption of this gene resulted in the accumulation of VB with a corresponding decrease of sterigmatocystin. *stcL* encodes a cytochrome P-450 monooxygenase. *verB* is a homolog of *stcL* in *A. parasiticus* (Bhatnagar et al. 2003).

Reaction 14: versicolorin A to demethylsterigmatocystin or versicolorin B to dihydromethylsterigmatocystin

The dihydrosfuran ring in VA and the tetrahydrobisfuran ring in VB are maintained through the next steps. The intermediates after these versicolorins are demethylsterigmatocystin (DMST) for VA and dihydromethylsterigmatocystin (DHDMST) for VB (Yabe et al. 1989). However, the enzyme(s) that catalyze the conversions of VA to DMST and VB to DHDMST have not yet been identified. In fact, this is the only step in the pathway from NA to the aflatoxins for which enzyme activity has not been determined. From the structural change in the conversion of VA to DMST, the step is probably composed of more than one reaction, such as the combination of oxidative decarboxylation, hydrogenation, and two dehydrations. Sterigmatin, an isomer of DMST isolated from *A. versicolor* (Hamasaki et al. 1973), may afford a clue to the mechanism of this conversion.

*ver-l* (*verl*) of *A. parasiticus* was the first gene found to be involved in aflatoxin biosynthesis (Skory et al. 1992; Liang et al. 1996) and was isolated based on its ability to complement the genetic block in a VA-accumulating mutant of *A. parasiticus*. The nucleotide sequence of *ver-l* (*verl*) suggests that it encodes a ketoreductase responsible for deoxygenation. *stcU* of *A. nidulans* is the homolog of *ver-l* (Brown et al. 1996b). Another gene, *stcS*, was also isolated in the sterigmatocystin gene cluster, and it encodes a cytochrome P-450 monooxygenase that may function at the same point as *stcU* (Keller et al. 1995). These results confirmed that the conversion of VA to demethylsterigmatocystin requires more than one enzymatic activity.

Reaction 15: demethylsterigmatocystin to sterigmatocystin or dihydromethylsterigmatocystin to dihydrosterigmatocystin

Two kinds of *O*-methyltransferases, *O*-methyltransferases I and II, are involved in aflatoxin biosynthesis (Yabe et al. 1989). DMST and DHDMST contain two free hydroxyl groups, 7-OH and 6-OH. *O*-methyltransferase I catalyzes the transfer of the methyl groups from S-adenosylmethionine (SAM) to the hydroxyl groups of DMST and DHDMST in order to produce ST and dihydrosterigmatocystin (DHST), respectively. However, the enzyme does not discriminate between the dihydrosfuran and tetrahydrobisfuran structures in the sterigmatocystin derivatives. *O*-methyltransferase I was purified from *A. parasiticus* and is a 43-kDa protein that catalyzes the conversion of DMST to ST or DHDMST to DHST (Yabe et al. 1998). *dmtA*, which encodes the enzyme, was isolated based on the amino acid sequence of the purified enzyme, and the deduced amino acid sequence contained a SAM-binding consensus sequence (Motomura et al. 1999; Yu et al. 2000a). This enzyme is quite unique because it shows substrate inhibition at high substrate concentrations (Yabe et al. 1998).

In *A. nidulans*, ST is the final metabolic product. A gene-disruption study revealed that *stcP* is involved in the conversion of DMST to ST (Kelkar et al. 1996), and *ddmtA* was shown to have high homology to *stcP*. *O*-methyltransferase I is the last enzyme and catalyzes the final reaction in sterigmatocystin biosynthesis in *A. nidulans*. The resultant sterigmatocystin usually accumulates in the fungal cells.
Reaction 16: sterigmatocystin to O-methylsterigmatocystin or dihydrosterigmatocystin to dihydro-O-methylsterigmatocystin

The second enzyme, O-methyltransferase II (OmtA), further transfers a methyl group of SAM to 7-OH of ST and DHST to form O-methylsterigmatocystin (OMST) and dihydro-O-methylsterigmatocystin (DHOMST), respectively. These O-methyltransferases (I and II) show strict substrate-specificity, as O-methyltransferase II cannot methylate DMST or DHDMST in spite of the presence of free 7-OH groups in these compounds. O-methyltransferase II was purified (Keller et al. 1993), and its gene, *omtA*, was isolated based on the amino acid sequence of the purified enzyme (Yu et al. 1993). The function of *omtA* was further confirmed by gene disruption and expression of the recombinant protein (Lee et al. 2002). The absence of an *omtA* homolog in *A. nidulans* is responsible for ST as the final product in this fungus.

a 220-kDa cytosolic protein to OrdA was required for the formation of G-aflatoxins (Yabe et al. 1999). The genes encoding the additional proteins have not been isolated.

It is generally accepted that *A. flavus* strains produce only AFB₁ and AFB₂, whereas *A. parasiticus* produces all four major aflatoxins, AFB₁, AFB₂, AFG₁, and AFG₂ (Bennett and Christensen 1983). Therefore, it is likely that the enzyme activities of the unstable microsome fraction as well as the 220-kDa cytosolic protein are much lower in *A. flavus* than in *A. parasiticus*.

---

### Features of aflatoxin biosynthesis

More than ten NADPHs are required for the formation of an aflatoxin. One NAD is also necessary for the formation of OAVN from HAVN, and two SAMs are further required for the formation of OMST from DMST or DHOMST from DHDMST. Therefore, aflatoxin biosynthesis is metabolically very expensive in terms of energy cost. In fact, aflatoxin production appears to be physiologically dependent on energy status. NADPH serves as an electron donor in reductive biosynthesis, and is supplied by the pentose-phosphate pathway or by the malate enzyme. The supply of cofactors from primary metabolisms may be a key factor regulating this secondary metabolism (Maggon et al. 1977).

Except for *estA*, the expression of genes encoding the enzymes, is positively regulated at the transcriptional level by a transcription factor encoded by *aflR* (Payne et al. 1993; Woloshuk et al. 1994, Yu et al. 2003). Environmental or physiological factors such as carbon source, nitrogen source, and temperature, further control expression through on/off regulation of *aflR* expression. In fact, the activities of all of the characterized enzymes are dependent on the carbon source; i.e., the cell-free system from mycelia cultured in YES medium showed enzyme activities, whereas that from mycelia cultured in YEP (yeast extract, peptone) medium did not. In contrast, *estA* of the aflatoxin gene cluster is constitutively expressed irrespective of the culture media (Yu et al. 2003). Furthermore, the cytosol prepared from mycelia cultured in YEP medium showed significant esterase activity. The reason for the difference between *estA* expression and that of the majority of the aflatoxin genes remains to be studied.

It is expected that, in order to allow aflatoxin formation to operate efficiently, the enzymes are localized within a specific cellular compartment. However, a biochemical study showed that both cytosolic and membrane enzymes are intermittently involved in the pathway. At least four membrane enzymes (respectively catalyzing the conversions of (reaction 5) AVN to 5′-HAVN, (8) AVR to HVN, (13) VB to VA, and (17, 18) OMST to AFB₁/AFG₁ or DHOMST to AFB₂/AFG₂) and more than seven soluble (cytosolic) enzymes are involved in aflatoxin biosynthesis. These results indicate that the enzymes are not localized to a certain membrane portion, or are only partially localized. Together with the EstA, an unknown cytosol esterase that

Reaction 17: O-methylsterigmatocystin to aflatoxin B₁ or dihydro-O-methylsterigmatocystin to aflatoxin B₂

In aflatoxigenic fungi, OMST and DHOMST are the last stable intermediates of aflatoxins and the precursors of AFB₁/AFG₁ and AFB₂/AFG₂, respectively, in feeding experiments (Bhatnagar et al. 1987; Yabe et al. 1988). In cell-free studies, the microsome enzyme catalyzes the conversion of OMST to AFB₁ and DHOMST to AFB₂ in the presence of NADPH (Yabe et al. 1988). For the formation of these B-group aflatoxins, the microsome enzyme is the only required fraction (Yabe et al. 1999). Aflatoxins are excreted from the fungal mycelia, whereas aflatoxin precursors accumulate in the fungal cells. The *ordA* product catalyzed the reaction from OMST to AFB₁ (Yu et al. 1998), suggesting that this protein is involved in the external transport of aflatoxins. The excretion mechanism of aflatoxins is discussed below.

*ordl* and *ordA* were isolated from *A. flavus* and *A. parasiticus*, respectively, and encode the cytochrome P-450 monooxygenase enzymes catalyzing the reaction from OMST to AFB₁ (Prieto and Woloshuk 1997; Yu et al. 1998). Yeast cells transformed with fungal *ordA* converted exogenously added OMST to AFB₁ (Yu et al. 1998).

Reaction 18: O-methylsterigmatocystin to aflatoxin G₁ or dihydro-O-methylsterigmatocystin to aflatoxin G₂

Together with the B-group aflatoxins, G-group aflatoxins AFG₁ and AFG₂ are also produced from OMST and DHOMST, respectively, in cell-free systems of *A. parasiticus* (Yabe et al. 1999). The G- and B-group aflatoxins are independently produced from the same substrate (OMST for AFG₁ and AFB₁ or DHOMST for AFG₂ and AFB₂). Although OrdA is the sole enzyme required in the formation of B-group aflatoxins (Yu et al. 1998), the addition of an unstable microsome enzyme and

is not encoded in the aflatoxin gene cluster could also participate in the reaction from VHA to VHOH (Chang et al., unpublished data). Also, aflatoxin precursors exogenously added to the culture medium could serve as intermediates to form aflatoxins in the feeding experiments. These results indicate that both exogenous and endogenous metabolites can serve as precursors for aflatoxin biosynthesis. Furthermore, enzymes encoded in the aflatoxin gene cluster as well as outside of it can interact with each other in aflatoxin biosynthesis. Therefore, localization of the enzymes may not be essential for aflatoxin biosynthesis. Nonetheless, most of the enzymes involved in aflatoxin production show strict specificity for their own substrates. These substrate specificities may effectively control the biosynthetic pathway. Identification of the organelle on which the membrane enzymes are located will provide information about the role of localization in the aflatoxin biosynthesis apparatus.

In the biosynthetic pathway, aflatoxins are finally excreted from the mycelia into the medium. In contrast, most of the aflatoxin intermediates accumulate in fungal cells. Although the presence of a specific aflatoxin export system is expected, it has not been found. Another possibility is that the solubilities of the substances in the lipid layer versus the aqueous phase may reflect the differences between the intermediates and aflatoxins. In fact, OMST is partially excreted, probably because OMST appears to have a polarity similar to those of aflatoxins. The mechanism determining the destinations of the precursors and aflatoxins remains to be studied.

---

### Conclusions

Nearly 45 years after the discovery of X disease in turkeys, the aflatoxin biosynthetic pathway and its related genes have been almost completely clarified. This mini-review has concentrated on the biosynthetic pathways and the genes. The regulatory mechanism afforded by both the *aflR* genes and signal transduction in aflatoxin biosynthesis as well as the relationship between secondary metabolites and morphological differentiation are being extensively studied. Based on these achievements, researchers will soon gain extensive insight regarding the regulation of AF/ST. Finally, researchers may be approaching the development of practical methods for preventing AF/ST contamination.

---

### Acknowledgements

We would like to dedicate this review to the late Professor Takashi Hamasaki, Tottori University, Japan, who contributed greatly to our research. This work was supported in part by a Grant-in-Aid (Bio Design Program) from the Ministry of Agriculture, Forestry and Fisheries of Japan (BDP-03-VI-5).

---

### References

Bennett JW, Christensen SB (1983) New perspective of aflatoxin biosynthesis. Adv Appl Microbiol 29:53–92

Bennett JW, Deutsch E (1986) Genetics of mycotoxin biosynthesis. In: Steyn PS, Eleggaar R (eds) Mycotoxins and phycotoxins. Elsevier, Amsterdam, pp 51–64

Bennett JW, Lee LS, Cucullu AF (1976) Effect of dichlorvos on aflatoxin and versicolorin A production in *Aspergillus parasiticus*. Bot Gaz 137:318–324

Bergar-Deguee M, Bergar Y (1982) Structure of versicolorone isolated from *Aspergillus versicolor*. Phytochemistry 21:1449–1451

Bhatnagar D, Cleveland TE (1990) Purification and characterization of a reductase from *Aspergillus parasiticus* SRRC 2043 involved in aflatoxin biosynthesis. FASEB J 4:A2164

Bhatnagar D, McCormick SP, Lee LS, Hill RA (1987) Identification of O-methylsterigmatocystin as an aflatoxin B₁/G₁ precursor in *Aspergillus parasiticus*. Appl Environ Microbiol 53:1028–1033

Bhatnagar D, Ehrlich KC, Cleveland TE (2003) Molecular genetic analysis and regulation of aflatoxin biosynthesis. Appl Microbiol Biotechnol 61:83–93

Brown DW, Adams TH, Keller NP (1996a) *Aspergillus* has distinct fatty acid syntheses for primary and secondary metabolism. Proc Natl Acad Sci USA 93:14873–14877

Brown DW, Yu JH, Kelkar HS, Fernandes M, Nesbitt TC, Keller NP, Adams TH, Leonard TJ (1996b) Twenty-five coregulated transcripts define a sterigmatocystin gene cluster in *Aspergillus nidulans*. Proc Natl Acad Sci USA 93:1418–1422

Brown MP, Brouwn-Jenco CS, Payne GA (1999) Genetic and molecular analysis of aflatoxin biosynthesis. Fungal Genet Biol 26:81–98

Busby WF, Wogan GN (1985) Aflatoxins. P. 945–1136. In: Esarle CE (ed) Chemical carcinogens, 2nd edn, vol 2. American Chemical Society, Washington, D.C

Cary JW, Wright M, Bhatnagar D, Lee R, Chu FS (1996) Molecular characterization of an *Aspergillus parasiticus* dehydrogenase gene, *norA*, located on the aflatoxin biosynthesis gene cluster. Appl Environ Microbiol 62:360–366

Cary JW, Barnaby N, Ehrlich KC, Bhatnagar D (1999) Isolation and characterization of experimentally induced aflatoxins biosynthetic pathway deletion mutants of *Aspergillus parasiticus*. Appl Microbiol Biotechnol 51:808–812

Cary JW, Bhatnagar D, Linz JE (2000) Aflatoxins: Biological significance and regulation of biosynthesis. In: Cary JW, Linz JE, Bhatnager D (eds), Microbial food-borne diseases: mechanisms of pathogenesis and toxin synthesis. Technomic, Lancaster, Pennsylvania, pp 317–361

Chang PK, Cary JW, Yu J, Bhatnagar D, Cleveland TE (1995) *Aspergillus parasiticus* polyketide synthetase gene, *pksA*, a homolog of *Aspergillus nidulans wA*, required for aflatoxin B₁. Mol Gen Genet 248:270–277

Chang PK, Yu J, Bhatnagar D, Cleveland TE (1999) The carboxy-terminal portion of the aflatoxin pathway regulatory protein AFLR of *Aspergillus parasiticus* activates *GAL1::LacZ* gene expression in *Saccharomyces cerevisiae*. Appl Environ Microbiol 65:2508–2512

Chang PK, Yu J, Ehrlich KC, Boue SM, Montalbano BG, Bhatnagar D, Cleveland TE (2000) The aflatoxin biosynthesis gene *adhA* in *Aspergillus parasiticus* is involved in conversion of 5′-hydroxyaverantin to averufin. Appl Environ Microbiol 66:4715–4719

Feng GH, Leonard TJ (1995) Characterization of the polyketide synthase gene (*pksL*) required for aflatoxin biosynthesis in *Aspergillus parasiticus*. J Bacteriol 177:6246–5624

Fernandes M, Keller NP, Adams TH (1998) Sequence-specific binding by *Aspergillus nidulans* AflR, a C6 zinc cluster protein regulating mycotoxin biosynthesis. Mol Microbiol 28:1355–1365

Hamasaki T, Matsui K, Isono K, Hatsuda Y (1973) A new metabolite from *Aspergillus versicolor*. Agric Biol Chem 37:1769–1770

Hitchman TS, Schmidt EW, Trail F, Rarick MD, Linz JE, Townsend CA (2001) Bioorg Chem 29:293–307

Horie Y, Udagawa S, Nishimura K, Miyaji M (1988) Distribution of sterigmatocystin-producing *Emericella* in soil. In: Aibara K, Kumagai S, Ostubo K, Yoshizawa T (eds) Proceeding Jap Assoc/Mycotoxicology, Suppl. I, IUPAC, Mycotoxins and Phycotoxins. Jpn Soc Mycotoxicology, Tokyo, pp 41–42

Hutchinson CR (1999) Microbial polyketide synthases: more and more prolific. Proc Natl Acad Sci USA 96:3336–3338

Kelkar HS, Keller NP, Adams TH (1996) *Aspergillus nidulans stcP* encodes an *O*-methyltransferase that is required for sterigmatocystin biosynthesis. Appl Environ Microbiol 62:4296–4298

Kelkar HS, Hernant S, Skloss TW, Haw JF, Keller NP, Adams TH (1997) *Aspergillus nidulans stcL* encodes a putative cytochrome P-450 monooxygenase required for bisfuran desaturation during aflatoxin/sterigmatocystin biosynthesis. J Biol Chem 272:1589–1594

Keller NP, Hohn TM (1997) Metabolic pathway gene clusters in filamentous fungi. Fungal Genet Biol 21:17–29

Keller NP, Cleveland TE, Bhatnagar D, Cleveland TE, Ullah AHJ (1993) Purification of a 40-kilodalton methyltransferase active in the aflatoxin biosynthetic pathway. Appl Environ Microbiol 59:479–484

Keller NP, Segner S, Bhatnagar D, Adams TH (1995) *stcS*, a putative P-450 monooxygenase, is required for the conversion of versicolorin A to sterigmatocystin in *Aspergillus nidulans*. Appl Environ Microbiol 61:3628–3632

Keller NP, Watanabe CMH, Kelker HS, Adams TH, Townsend CA (2000) Requirement of monooxygenase-mediated steps for sterigmatocystin biosynthesis by *Aspergillus nidulans*. Appl Environ Microbiol 66:359–362

Kennedy J, Auclair A, Kendrew SG, Park C, Vederas JC, Hutchinson CR (1999) Modulation of polyketide synthase activity by accessory proteins during lovastatin biosynthesis. Science 284:1368–1372

Koreeda M, Hulin B, Yoshihara M, Townsend CA, Christensen SB (1985) Synthesis and absolute configuration of (+)-averufin. J Org Chem 50:5426–5428

Kusumoto KI, Hsieh DPH (1996) Purification and characterization of the esterases involved in aflatoxin biosynthesis in *Aspergillus parasiticus*. Can J Microbiol 42:804–810

Lee L-W, Chiou C-H, Linz JE (2002) Function of native OmtA in vivo and expression and distribution of this protein in colonies of *Aspergillus parasiticus*. Appl Environ Microbiol 68:5718–5727

Liang SH, Skory CD, Linz JE (1996) Characterization of the function of *thever-1A* and *ver-1B* genes involved in aflatoxin biosynthesis in *Aspergillus parasiticus*. Appl Environ Microbiol 62:4568–4575

Lin BK, Anderson JA (1992) Purification and properties of versiconal cyclase from *Aspergillus parasiticus*. Arch Biochem Biophys 293:67–70

Maggon KK, Gupta SK, Venkitasubramanian TA (1977) Biosynthesis of aflatoxins. Bacteriol Rev 41:822–855

Mahanti N, Bhatnagar D, Cary JW, Joubran J, Linz JE (1996) Structure and function of *fas-1A*, a gene encoding a putative fatty acid synthase directly involved in aflatoxin biosynthesis in *Aspergillus parasiticus*. Appl Environ Microbiol 62:191–195

Massey TE, Stewart RK, Daniels JM, Liu L (1995) Biochemical and molecular aspects of mammalian susceptibility to aflatoxin B₁ carcinogenicity. Proc Soc Exp Bio Med 2003:2213–227

Matsushima K, Ando Y, Hamasaki T, Yabe K (1994) Purification and characterization of two versiconal hemiacetal acetate reductases involved in aflatoxin biosynthesis. Appl Environ Microbiol 60:2561–2567

McCormick SP, Bhatnagar D, Lee LS (1987) Averufanin is an aflatoxin B₁ precursor between averantin and averufin in the biosynthetic pathway. Appl Environ Microbiol 53:14–16

McGuire SM, Silva JC, Casillas EG, Townsend CA (1996) Purification and characterization of versicolorin B synthase from *Aspergillus parasiticus*. Catalysis of the stereodifferentiating cyclization in aflatoxin biosynthesis essential to DNA interaction. Biochem 35:11470–11486

Meyers DM, O'Brian G, Du WL, Bhatnagar E, Payne GA (1998) Characterization of *aflJ*, a gene required for conversion of pathway intermediates to aflatoxin. Appl Environ Microbiol 64:3713–3717

Minto RE, Townsend CA (1997) Enzymology and molecular biology of aflatoxin biosynthesis. Chem Rev 97:2537–2555

Miyamoto M, Matsumoto J, Iwaya T, Itagaki E (1995) Bacterial steroid monooxygenase catalyzing the Baeyer-Villiger oxidation of C₂₁-ketosteroids from *Rhodococcus rhodochrous*: the isolation and characterization. Biochim Biophys Acta 1251:115–124

Motomura M, Chihaya N, Shinozawa T, Hamasaki T, Yabe K (1999) Cloning and characterization of the *O*-methyltransferase I gene (*dmtA*) from *Aspergillus parasiticus* associated with the conversion of demethylsterigmatocystin to sterigmatocystin and dihydromethylsterigmatocystin to dihydrosterigmatocystin in aflatoxin biosynthesis. Appl Environ Microbiol 65:4987–4994

Payne GA (1998) Aflatoxin in maize. Crit Rev Plant Sci 10:423–440

Payne GA, Brown MP (1998) Genetics and physiology of aflatoxin biosynthesis. Annu Rev Phytopathol 36:329–362

Payne GA, Nystrom GJ, Bhatnagar D, Cleveland TE, Woloshuk CP (1993) Cloning of the *afl-2* gene involved in aflatoxin biosynthesis from *Aspergillus flavus*. Appl Environ Microbiol 59:156–162

Prieto R, Woloshuk CP (1997) *ord1*, an oxidoreductase gene responsible for conversion of *O*-methylsterigmatocystin to aflatoxin in *Aspergillus flavus*. Appl Environ Microbiol 63:1661–1666

Robens J (2001) The costs of mycotoxin management to the USA: management of aflatoxins in the United States. http://www.apsnet.org/online/feature/mycotoxin/top.html

Sakuno E, Yabe K, Nakajima H (2003) Involvement of two cytosolic enzymes and a novel intermediate, 5′-oxoaverantin, in the pathway from 5′-hydroxyaverantin to averufin in aflatoxin biosynthesis. Appl Environ Microbiol 69:6418–6426

Schroeder HW, Cole RS, Grigsby RD, Hein HJ (1974) Inhibition of aflatoxin production and tentative identification of an aflatoxin intermediate “versiconol acetate” from treatment with dichlorvos. Appl Microbiol 27:394–399

Silva JC, Townsend CA (1996) Heterologous expression, isolation, and characterization of versicolorin B synthase from *Aspergillus parasiticus*. J Biol Chem 272:804–813

Silva JC, Minto RE, Barry III CE, Holland KA, Townsend CA (1996) Isolation and characterization of the versicolorin B synthase gene from *Aspergillus parasiticus*: expression of the aflatoxin B₁ biosynthetic cluster. J Biol Chem 271:13600–13608

Skory CD, Chang PK, Cary J, Linz JE (1992) Isolation and characterization of a gene from *Aspergillus parasiticus* associated with the conversion of versicolorin A to sterigmatocystin in aflatoxin biosynthesis. Appl Environ Microbiol 58:3527–3537

Townsend CA, Christensen SB, Trautwein K (1984) Hexanoate as a starter unit in polyketide synthesis. J Am Chem Soc 106:3868–3869

Townsend CA, Plavcan KA, Pal K, Brobst SW, Irish MS, Ely EW, Bennett JW (1988a) Hydroxyversicolorone: Isolation and characterization of a potential intermediate in aflatoxin biosynthesis. J Org Chem 53:2472–2477

Townsend CA, Whittamore PRO, Brobst SW (1988b) Hydroxyversicolorone: synthesis and incorporation of a new intermediate in aflatoxin biosynthesis. J Chem Soc Chem Commun 726–728

Trail F, Chang PK, Cary J, Linz J (1994) Structural and functional analysis of the *nor-1* gene involved in the biosynthesis of aflatoxins by *Aspergillus parasiticus*. Appl Environ Microbiol 60:4078–4085

Trail F, Mahanti N, Rarick M, Mehigh MR, Liang SH, Zhou R, Linz J (1995a) Physical and transcriptional map of an aflatoxin gene cluster in *Aspergillus parasiticus* and functional disruption of a gene involved early in the aflatoxin pathway. Appl Environ Microbiol 61: 2665–2673

Trail F, Mahanti N, Linz J (1995b) Molecular biology of aflatoxin biosynthesis. Microbiol 141: 755–765

Watanabe CMH, Townsend CA (2002) Initial characterization of a type I fatty acid synthase and polyketide synthase multienzyme complex NorS in the biosynthesis of aflatoxin B₁. Chem Biol 9: 981–988

Watanabe CMH, Wilson D, Linz JE, Townsend CA (1996) Demonstration of the catalytic roles and evidence for the physical association of type I fatty acid syntheses and a polyketide synthase in the biosynthesis of aflatoxin B₁. Chem Biol 3: 463–469

Woloshuk CP, Prieto R (1998) Genetic organization and function of the aflatoxin B₁ biosynthetic genes. FEMS Microbiol Lett 160: 169–176

Woloshuk CP, Foutz KR, Brewer JF, Bhatnagar D, Cleveland TE, Payne GA (1994) Molecular characterization of *aflR*, a regulatory locus for aflatoxin biosynthesis. Appl Environ Microbiol 60: 2408–2414

Yabe K (2002) Pathway and genes of aflatoxin biosynthesis. In: Fierro F, Francisco J (eds) Microbial secondary metabolites: biosynthesis, genetics and regulation, Research Signpost. India, pp 227–251

Yabe K, Hamasaki T (1993) Stereochemistry during aflatoxin biosynthesis: cyclase reaction in the conversion of versiconal to versicolorin B and racemization of versiconal hemiacetal acetate. Appl Environ Microbiol 59: 2493-2500

Yabe K, Ando Y, Hamasaki T (1988) Biosynthetic relationship among aflatoxins B₁, B₂, G₁, and G₂. Appl Environ Microbiol 54: 2101–2106

Yabe K, Ando Y, Hashimoto J, Hamasaki T (1989) Two distinct O-methyltransferases in aflatoxin biosynthesis. Appl Environ Microbiol 55: 2172–2177

Yabe K, Ando Y, Hamasaki T (1991a) A metabolic grid among versiconal hemiacetal acetate, versiconol acetate, versiconol and versiconal during aflatoxin biosynthesis. J Gen Microbiol 137: 2469–2475

Yabe K, Ando Y, Hamasaki T (1991b) Desaturase activity in the branching step between aflatoxins B₁ and G₁ and aflatoxins B₂ and G₂, Agric Biol Chem 55: 1907–1911

Yabe K, Nakamura Y, Nakajima H, Ando Y, Hamasaki T (1991c) Enzymatic conversion of norsolorinic acid to averufin in aflatoxin biosynthesis. Appl Environ Microbiol 57: 1340–1345

Yabe K, Matsuyama Y, Ando Y, Nakajima H, Hamasaki T (1993) Stereochemistry during aflatoxin biosynthesis: conversion of norsolorinic acid to averufin. Appl Environ Microbiol 59: 2486–2492

Yabe K, Matsushima KI, Koyama T, Hamasaki T (1998) Purification and characterization of O-methyltransferase I involved in conversion of demethylsterigmatocystin to sterigmatocystin and dihydrodemethylsterigamtocystin to dihydrosterigmatocystin during aflatoxin biosynthesis. Appl Environ Microbiol 64: 166–171

Yabe K, Nakamura M, Hamasaki T (1999) Enzymatic formation of G-group aflatoxins and biosynthetic relationship between G- and B-group aflatoxins. Appl Environ Microbiol 65: 3867–3872

Yabe K, Chihaya N, Hamamatsu S, Sakuno E, Hamasaki T, Nakajima H, Bennett JW (2003) Enzymatic conversion of averufin to hydroxyversicolorone and elucidation of a novel metabolic grid involved in aflatoxin biosynthesis. Appl Environ Microbiol 69: 66–73

Yao RC, Hsieh DPH (1974) Step of dichlorvos inhibition in the pathway of aflatoxin biosynthesis. Appl Microbiol 28: 52–57

Yu JH, Leonard TJ (1995) Sterigmatocystin biosynthesis in *Aspergillus nidulans* required a novel type I polyketide synthetase. J Bacteriol 177: 4792–4800

Yu J, Cary JW, Bhatnagar D, Cleveland TE, Keller NP, Chu FS (1993) Cloning and characterization of a cDNA from *Aspergillus parasiticus* encoding an O-methyltransferase involved in aflatoxin biosynthesis. Appl Environ Microbiol 59: 3564–3571

Yu JH, Butchko RAE, Fernandes M, Keller NP, Leonard TJ, Adams TH (1996) Conservation of structure and function of the aflatoxin regulatory gene *aflR* from *Aspergillus nidulans* and *A. flavus*. Curr Genet 29: 549–555

Yu J, Chang PK, Cary JW, Bhatnagar D, Cleveland TE (1997) *avnA*, a gene encoding a cytochrome P-450 monooxygenase is involved in the conversion of averantin to averufin in aflatoxin biosynthesis in *Aspergillus parasiticus*. Appl Microbiol Biotechnol 63: 1349–1356

Yu J, Chang PK, Ehrlich KC, Cary JW, Montalbano B, Dyer JM, Bhatnagar D, Cleveland TE (1998) Characterization of the critical amino acids of an *Aspergillus parasiticus* cytochrome P450 monooxygenase encoded by *ordA* involved in aflatoxin B₁, G₁, B₂, and G₂ biosynthesis. Appl Environ Microbiol 64: 4834–4841

Yu J, Woloshuk CP, Bhatnagar D, Cleveland TE (2000a) Cloning and characterization of *avfA* and *omtB* genes involved in aflatoxin biosynthesis in three *Aspergillus* species. Gene 248: 157–167

Yu J, Chang PK, Bhatnagar D, Cleveland TE (2000b) Genes encoding cytochrome P450 and monooxygenase enzymes define one end of the aflatoxin pathway gene cluster in *Aspergillus parasiticus*. Appl Microbiol Biotechnol 53: 583–590

Yu J, Chang PK, Bhatnagar D, Cleveland TE (2003) Cloning and functional expression of an esterase gene in *Aspergillus parasiticus*. Mycopathologia 156: 227–234

Zhou R, Linz JE (1999) Enzymatic function of the Nor-1 protein in aflatoxin biosynthesis in *Aspergillus parasiticus*. Appl Environ Microbiol 65: 5639–5641
