
Chemokine inhibition – why, when, where, which and how?

Z. Johnson, C.A. Power, C. Weiss, F. Rintelen, H. Ji, T. Ruckle, M. Camps, T.N.C. Wells, M.K. Schwarz, A.E.I. Proudfoot and C. Rommel¹

Serono Pharmaceutical Research Institute, Serono International, 14 Chemin des Aulx, CH 1228 Geneva, Switzerland

---

### Abstract

Chemokines are small chemoattractant cytokines that control a wide variety of biological and pathological processes, ranging from immunosurveillance to inflammation, and from viral infection to cancer. Genetic and pharmacological studies have shown that chemokines are responsible for the excessive recruitment of leucocytes to inflammatory sites and damaged tissue. In the present paper, we discuss the rationale behind interfering with the chemokine system and introduce various points for therapeutic intervention using either protein-based or small-molecule inhibitors. Unlike other cytokines, chemokines signal via seven-transmembrane GPCRs (G-protein-coupled receptors), which are favoured targets by the pharmaceutical industry, and, as such, they are the first cytokines for which small-molecule-receptor antagonists have been developed. In addition to the high-affinity receptor interaction, chemokines have an *in vivo* requirement to bind to GAGs (glycosaminoglycans) in order to mediate directional cell migration. Prevention of the GAG interaction has been shown to be a viable therapeutic strategy. Targeting chemokine intracellular signalling pathways offers an alternative small-molecule approach. One of the key signalling targets downstream of a variety of chemokine receptors identified to date is PI3Kγ (phosphoinositide 3-kinase γ), a member of the class I PI3K family. Thus the chemokine system offers many potential entry points for innovative anti-inflammatory therapies for autoimmune diseases, such as multiple sclerosis, rheumatoid arthritis and allergic contact dermatitis.

---

### Why does chemokine inhibition offer therapeutic value?

Cellular recruitment is an absolute requirement for the immune system to function, enabling the clearance of foreign bodies and defence against infection, as well as maintenance of the overall integrity of the system. For example, during the innate immune response, the recruitment of phagocytes is necessary to clear bacterial infection. The correct trafficking of lymphocytes is also essential for the establishment of our adaptive immune response, resulting in long-term immunological memory and the production of antibodies. Additionally, the control of T- and B-cell development is dependent upon chemokine-directed migration; for example, during tolerance induction. The tight control of cellular recruitment can break down, as is the case in both acute and chronic autoimmune inflammatory disorders – this is why we want to inhibit inflammation in diseases such as asthma, rheumatoid arthritis, MS (multiple sclerosis) and IBD (inflammatory bowel disease). However, too much inhibition can also be bad in that sepsis and infections can arise if the immune response is turned off completely.

Chemokines play a dual role in the function of the immune system. For example, during development, SDF-1/CXCL12 and its receptor CXCR4 play a major role in directing the normal physiological migration of haematopoietic stem cells [1–3]. On the other hand, certain chemokines control the aberrant recruitment of cells in inflammatory and autoimmune diseases. For example, CXCR3 and CCR5 expression is strongly associated with brain lesions in MS patients [4–6], CCR3 and several CC chemokines, including Eotaxin/CCL11 and RANTES/CCL5, are associated with allergic asthma [7], and TARC/CCL17 and its receptor CCR4 are implicated in inflammatory skin diseases, such as atopic dermatitis [7,8]. Some of the diseases that are associated with up-regulation of chemokine ligands and receptors are depicted in Figure 1.

A remarkable feature of the chemokine system is the observation that receptors that are responsible for controlling homoeostatic cell migration have only one described ligand, whereas receptors that have been shown to be associated with pathological phenotypes appear to be activated by multiple ligands (highlighted in Figure 1). However, the division of chemokines into ‘homoeostatic’ and ‘inflammatory’ subfamilies is not categorical: for example, CCR10 is considered to be constitutively expressed, but it is involved in certain skin inflammatory syndromes [9], whereas its ligand CTACK/CCL27 is up-regulated by pro-inflammatory

---

### Key words:
chemokine, glycosaminoglycan, leucocyte, phosphoinositide 3-kinase (PI3K), signalling, small-molecule inhibitor.

### Abbreviations used:
EAE, experimental autoimmune encephalomyelitis; FT, farnesytransferase; GAG, glycosaminoglycan; GEF, guanine nucleotide-exchange factor; GPCR, G-protein-coupled receptor; HMG, 3-hydroxy-3-methylglutaryl; HS, heparan sulphate; IFN, interferon; IL, interleukin; LFA, lymphocyte function-associated antigen-1; MOG, myelin oligodendrocyte glycoprotein; MS, multiple sclerosis; PI3K, phosphoinositide 3-kinase; ROS, reactive oxygen species; 7TM, seven-transmembrane; TNF, tumour necrosis factor.

¹To whom correspondence should be addressed (e-mail christian.rommel@serono.com).

©2004 Biochemical Society

Figure 1 | Disease targets of chemokine receptors and ligands

Chemokines are divided into subclasses on the basis of the spacing of the N-terminal cysteine residues. The receptors for the α (or CXC) subclass are shown in red, the receptors for the β (or CC) subclass are in blue and the receptors for the minor subclasses (C and CX3C) are in green. The pairing of chemokines to their receptors has been carried out principally by receptor-binding assays, and has identified receptors that are specific, in that they bind a single ligand, or shared, in that they bind more than one ligand. The chemokines are shown with their original names derived from their function or from the cell type that produced them, as well as the new systematic nomenclature [103]. The association of certain receptors and ligands with disease has come from many studies of their expression in biopsy samples, animal models and genetically engineered mice. The associations shown here are by no means exhaustive, but are intended as an illustration of many results of target validation. R.A., rheumatoid arthritis.

---

**Autoimmune diseases**

e.g. M.S., R.A.

**Transplant Rejection**

**Allergy**

e.g. atopic dermatitis

**Asthma**

**Atherosclerosis**

**Fibrosis**

**Angiogenesis**

**Cancer**

**Metastasis**

**HIV**

---

**CCR2**
- RANTES/CCL5
- MCP-1/CCL2
- LD78β/CCL3
- MIP-1α/CCL3
- RANTES/CCL5
- HCC-1/CCL14
- HCC-2/CCL15
- HCC-4/CCL16
- Gro α/CXCL1
- MPIF-1/CCL23
- IL-8/CXCL8
- GCP-2/CXCL6

**CCR3**
- MCP-2/CCL8
- MCP-3/CCL7
- HCC-2/CCL15
- Eotaxin-2/CCL24
- Eotaxin-3/CCL26
- TARC/CCL17
- MDC/CCL22

**CCR4**
- MCP-4/CCL13
- Eotaxin/CCL11
- LD78α/CCL3L1
- MIP-1β/CCL4
- RANTES/CCL5
- MCP-2/CCL8
- HCC-1/CCL14

**CCR5**
- MIP-3β/CCL19
- SLCCCL21

**CCR7**
- I-309/CCL1

**CCR8**

**CCR10**
- CTACK/CCL27
- Mec/CCL28

**CCR9**
- TECK/CCL25

**CCR6**
- MIP-3α/CCL20

**CXCR1**
- Fractalkine/CX3CL1
- Lymphotactin/XCL1
- SCM/1J/XCL2

**CXCR2**
- IL-8/CXCL8
- Groβ/CXCL2
- ENA-78/CXCL5
- Groγ/CXCL3
- Groα/CXCL1
- NAP-2/CXCL7

**CXCR3**
- IP-10/CXCL10
- I-TAC/CXCL11
- Mig/CXCL9
- PF-4/CXCL4

**CXCR4**
- SDF-1α/CXCL12

**CXCR5**
- BCA-1/CXCL13

**CXCR6**
- CXCL16

**CX3CR1**

**XCR1**

---

cytokines [10]. CCR2 and its ligand MCP1/CCL2, which have classically been assigned to the inflammatory group of chemokines, may also be involved in the regulation of the immune system through an effect on T-cell polarization [11].

We believe that the most promising approach in order to interfere with cellular recruitment is through direct inhibition of the actions of inflammatory chemokines. This may at first appear to be a rather cavalier statement as, to date, over 40 ligands and 18 functional receptors have been identified in this complex family. With such an apparently promiscuous network, the question of specificity arises immediately. The aim of the present review is to attempt to dispel these notions, and to elaborate on the value of the numerous approaches that can be employed to inhibit the chemokine system. Clearly such an apparently complicated system must be finely regulated *in vivo* both temporally and spatially to achieve such a fine-tuned specificity. The results described below obtained from knockout mice and the use of specific antibodies and receptor antagonists validate the approach of specific targeting of certain chemokine-receptor interactions as a therapeutic strategy. There are several mechanisms that are believed to contribute towards the choreography of chemokine expression in order to maintain function *in vivo* in addition to the differential signalling pathways that may be activated, as will be discussed below. Spatial and temporal control of chemokine expression is dependent on pro-inflammatory chemokines. For example, IL (interleukin)-1β, tumour necrosis factor (TNF)-α and interferon (IFN)-γ alone or in combination induce chemokine expression at sites of inflammation in non-lymphoid tissue [12]. In parallel, the expression of chemokine receptors on specific leucocyte populations is also under the tight control of the local cytokine milieu: for example, CCR4 and CCR3 are up-regulated on Th2 T-cells, whereas CXCR3 and CCR5 are up-regulated on Th1 T-cells [13,14].

---

**When are chemokines most vulnerable?**

In order for circulating leucocytes to reach sites of inflammation, they must cross protective barriers such as the endothelium and the epithelium, or the blood–brain barrier that protects the normally immune-privileged central nervous system. Leucocyte transmigration usually occurs in the post-capillary vascular system, and is a multi-step process involving a sequential cascade of pro-inflammatory cytokines, adhesion molecules and chemokines [15], as illustrated in Figure 2(A). Leucocytes that circulate in the bloodstream are constantly sampling the endothelial cell

Figure 2 | Mechanisms of chemokine-mediated chemotaxis *in vivo* (A) and *in vitro* (B)

(A) Cellular recruitment *in vivo*

(B) Chemotaxis *in vitro*

micro-environment for signals of abnormal behaviour by slowing down and marginating, a process mediated by non-specific rolling on endothelial cell-surface selectins. Upon activation of the endothelium by pro-inflammatory cytokines, such as TNF-α, IFN-γ and IL-1β, chemokines are secreted and presented on the endothelial cell surface by GAGs (glycosaminoglycans). Subsequent ligand engagement with cognate leucocyte receptors mediates an increase in integrin avidity, resulting in firm adhesion of the leucocyte to the adhesion molecules expressed on the endothelial cell surface: a signal to stop [16,17]. Once firm adhesion to the endothelial surface has been established, the leucocyte transmigrates across the endothelial cell barrier either intercellularly, which is believed to be the preferential route, or intracellularly via a mechanism that is currently poorly understood. Both routes of transendothelial migration appear to involve several adhesion molecules and MMPs (matrix metalloproteases). In addition to binding to GAGs on the endothelial cell surface, chemokines are able to bind to GAGs that are present in the extracellular matrix. The resultant chemokine gradient provides a directional signal that the cells may use to navigate towards the site of inflammation, where they exert their effects (Figure 2A). It is probable that *in vivo* gradients of several different chemokines are present in the tissue, separated temporally and spatially [18,19]. The *in vivo* requirement for chemokine binding to GAGs in order to mediate cell recruitment has recently been demonstrated [20]. Figure 2(B) illustrates the difference in the requirement for GAG binding in chemokine-mediated cell migration. In the absence of flow, in a standard *in vitro* chemotaxis assay, GAG binding is not essential for cell activation. However, in the vasculature and under the influence of shear flow, GAG

binding is required to tether the chemokine and form a fixed signal to stop.

Where can chemokines be blocked? Are there too many options and too much redundancy?

Chemotaxis is a multi-step process, requiring chemokine presentation, receptor occupation and activation, as well as intracellular signal transduction. Differential GAG binding is believed to play an important role in the spatial distribution of chemokines. The GAG-binding site of several chemokines has been determined. Although the chemokines have a very high structural similarity, particularly striking in the monomeric fold, there are differences in the specific GAG-binding sites between chemokines that are not conserved amongst the classes [21–27]. In addition, specific patterns of sulphation on HS (heparan sulphate) have been shown to be required for the binding of certain chemokines, and of those studied to date, each chemokine has a unique recognition pattern on HS [28,29]. Taken together with recent data from our laboratory that highlights the requirement of the interaction of chemokines with GAGs to elicit cellular recruitment *in vivo* [20], it appears that differential GAG-binding patterns may exert a sophisticated level of control of chemokine presentation and/or activation both spatially and temporally. Thus interfering with this crucial interaction may be one intervention point for inhibition of the system.

Inhibiting intracellular chemokine-activated signalling is another possible strategy for interfering with the system. With a better understanding of signalling pathways downstream of GPCRs (G-protein-coupled receptors) in general, more insights into CCR- and CXCR-specific signalling have

Figure 3 | Chemokine-activated intracellular signalling pathways

AC, adenocyclase generating cAMP; Bad, Bcl-2/Bcl-XL-antagonist, causing cell death; Cdc42, small GTPase of the Rho family; DAG, diacylglycerol; Dia, Diaphanous formin protein family; eIF2B, eukaryotic initiation factor 2B; ERK, extracellular-signal-regulated kinase; FA, focal adhesion; GSK3, glycogen synthase kinase 3; IP₃, Ins(1,4,5)P₃; JAK, Janus kinase; LIMK, LIM-motif-containing kinase; MEK, mitogen-activated protein kinase/ERK kinase; MLCK, myosin light-chain kinase; MLCP, myosin light-chain phosphatase; MT, microtubulae organization, actin/myosin dynamics; mTOR, mammalian target of rapamycin; p70 S6K, p70 S6 kinase; PAK, p21-activated kinase; PDK, phosphoinositide-dependent kinase; PIP₂, PtdIns(4,5)P₂; PI4P5K, phosphoinositide 4-phosphate 5-kinase; PKA, protein kinase A (cAMP-dependent protein kinase); PKC, protein kinase C; PLC, phospholipase C; PTK, protein tyrosine kinase; ROCK, Rho-associated coiled-coil-containing kinase; SF, S-filament; STAT, signal transduction and activator of transcription; WASp, Wiskott-Aldrich syndrome protein; WAVE, WASp-family verprolin homologous protein.

![Diagram](attachment:Chemokine_signaling_pathways.png)

been gained. The current knowledge of chemokine-mediated signalling is summarized in Figure 3. Chemokine signalling has been studied most comprehensively with CCR2, CCR5, CXCR1 and CXCR4 and in neutrophils, macrophages, T-cells [30,31], and more recently also in dendritic cells [32].

As chemokines control the complex process of cell migration, they must regulate multiple biochemical pathways acting in concert to facilitate the motility cycle. Chemokines activate 7TM (seven-transmembrane) receptors that are coupled to heterotrimeric Gαβγ proteins. Upon chemokine binding, dissociation of Gαβγ subunits is induced. Subsequently, the released Gβγ subunits activate a wide variety of enzymes, including phospholipid kinases, lipases and GEFs (guanine nucleotide-exchange factors). These enzymes, in turn, regulate the phosphorylation of lipid second messengers and the production or release of second messengers, such as Ins(1,4,5)P₃, Ca²⁺ and DAG (diacylglycerol). GEFs catalyse the GTPase hydrolysis rate of small GTPases. On the other hand, released Gα subunits can regulate the generation of cAMP via adenylate cyclases, or couple chemokine receptor activation to non-receptor protein-tyrosine-kinase-initiated pathways. Subsequently, protein tyrosine kinases phosphorylate a variety of targets, including GEFs and adaptor proteins that are involved in the formation of multi-protein signalling complexes (signalosomes). There is also evidence that chemokines can induce tyrosine phosphorylation of their receptors independent of Gαβγ, with subsequent receptor dimerization and recruitment of JAK (Janus kinase) and STAT (signal transduction and activators of transcription) providing a short link to transcriptional regulation [33–35].

Faced with the high complexity of signals downstream of GPCRs, it is a challenge to distinguish between ‘house-keeping’ vital cell-locomotion pathways and signals that specifically regulate cell migration in inflammatory processes. One of the key signalling targets downstream of a variety of chemokine receptors is PI3K (phosphoinositide 3-kinase) γ, a member of the class I PI3K family. Gβγ subunits associate specifically with the p101 adaptor of PI3Kγ. A striking disparity was observed concerning the viability of mice possessing inactivated alleles of p110α or p110β in comparison with animals that were deficient in functionally active p110γ or p110δ. Mice deficient in class IA p110α or p110β show an early embryonic lethal phenotype, whereas mice lacking enzymically active p110γ or p110δ are viable, and exhibit a distinctive phenotype, which is restricted to the cells and organs expressing these distinct isoforms. Analysis of class IB PI3Kγ-deficient (PI3Kγ⁻/⁻) mice [36–38] and studies using specific isoform non-selective class I PI3K small-molecule kinase inhibitors [39,40] revealed a pivotal role for PI3Kγ in chemokine-regulated recruitment of leucocytes *in vitro* and *in vivo*. Furthermore, PI3Kγ⁻/⁻ neutrophils and macrophages are unable to produce PtdIns(3,4,5)P₃ when stimulated with GPCR agonists such as fMLP (N-formylmethionyl-leucylphenylalanine), C5a (complement component 5a) or IL-8. In parallel, PtdIns(3,4,5)P₃-dependent activation of the serine/threonine PKB (protein kinase B; AKT), a major target of PI3K, was not observed. Recent work revealed that mast cells represent another cell type severely affected in PI3Kγ⁻/⁻ mice [41,42]. These cells are of special importance for the initiation of inflammatory and allergic immune reactions. Their ability to link the adaptive and the innate immune systems via the IgE receptor and GPCRs for adenosine, chemokines and complement factors, characterize mast cells as central mediators of the immune response. Lack of p110γ impairs adenosine-dependent degranulation of mast cells, revealing p110γ/PI3Kγ as the main regulatory signalling protein in these specialized immune cells. In addition to its dominant role in phagocytes and mast cells, PI3Kγ appears to be important for T-cell development and platelet aggregation [38,43,44]. Thus targeting PI3Kγ may indeed modulate several inflammatory conditions by a moderate, yet wide-ranging, impact on the immune system.

Pharmacological targeting of class I PI3K is at present a major goal for several biotechnological and pharmaceutical companies. Ongoing efforts are focusing on the search for specific and selective inhibitors of individual class I PI3K isoforms [45,46]. Figure 4(A) illustrates some recent progress in the development of small-molecule PI3K inhibitors, which are currently under investigation as therapeutics.

PI3K-dependent generation of PtdIns(3,4,5)P₃ provides a mechanistic link between the activation of GEFs and Rho family GTPases and the machinery of cellular motility. A recently identified GEF named P-Rex1, which is synergistically activated by both Gβγ and PIP₃ [47], regulates Rac, a member of the Rho-GTPase subfamily that is essential for the migration of all cells examined to date [48]. Many signalling targets of Rac proteins are involved in the dynamic reorganization of cellular cytoskeleton, such as IRS (insulin receptor substrate) p53 [49], PI4P5K (phosphoinositide 4-phosphate 5-kinase) [50], PAK (p21 activated kinase) [51] and LIMK (LIM-motif-containing kinase) [52], as well as the Arp2/3 protein complex [53]. Mice that are deficient in individual Rac isoforms show interesting disease-relevant phenotypes. For example, Rac2-gene-deleted mice have defects in neutrophil chemotaxis and chemokine-induced ROS (reactive oxygen species) production [54,55]. Many small GTPases undergo post-translational modification by FTs (farnesyltransferases). Consequently, small-molecule inhibitors that target these enzymes have entered clinical trials, mainly focusing on cancer by targeting Ras and Rho [56,57]. Interestingly, analogous to the mode of action of FT inhibitors, statins [a class of HMG (3-hydroxy-3-methylglutaryl)-CoA reductase small-molecule inhibitors] have recently been shown to inhibit isoprenylation of Rac1 and to reduce NADPH-oxidase-generated ROS [58]. Inhibition of NADPH oxidase and thereby uncontrolled release of ROS is an emerging therapeutic strategy for several inflammatory conditions. Since comprehensive structural information for GEF and GAP (GTPase-activation protein), as well as Rho-GTPase protein complexes is available, rational structure-based design of small-molecule inhibitors that interfere with the conformational regulation of Rho-GTPases has been proposed [59].

Mining chemokine signalling pathways for validated, ‘druggable’ and biologically suitable targets has only just begun. It has proven to be difficult to link specific CCR or CXCR signalling pathways right through to a biological response such as chemotaxis. Instead of trying to identify every detailed component in each CCR pathway it seems more feasible to explore how pathways co-operate during cellular migration to identify disease-relevant targets with a central function in chemokine signalling – just like PI3Kγ.

With more than 40 ligands for less than 20 receptors, the problem of which ligand interacts with which target arises

One of the major challenges in chemokine research has been to validate rigorously which receptor-ligand pairs are important in specific pathophysiological conditions. Targeted deletion of chemokines and their receptors has proved to be a useful tool for discerning their role *in vivo* and may provide a basis for the development of future therapeutic agents that are aimed at interrupting leucocyte recruitment and activation. Of the 18 human chemokine receptors that have been identified to date, 14 knockouts have been generated. The reported phenotypes in mouse disease models are summarized in Table 1. More comprehensive reviews can be found in recent publications [60,61].

Studies of knockout and transgenic mice in disease models have often been consolidated by results obtained using anti-chemokine or anti-(chemokine receptor) antibodies, or even modified chemokines, in the same models. In many cases, targeted gene deletion has revealed a novel or unexpected

Figure 4 | Small-molecule inhibitors of class I PI3K (A) and chemokine receptors (B)

In most cases the generic structure is shown as described in the corresponding patents.

(A)

(B)


The rest of the content includes chemical structures and text that cannot be directly converted into Markdown due to the complexity of the diagrams and chemical notations. The OCR process for such images typically requires manual intervention to accurately represent the structures and annotations. If you need further assistance with specific parts, please let me know!
Table 1 | CCR-knockout mice

| Receptor | Phenotype | Reference |
|----------|-----------|-----------|
| CCR1     | Imbalance in Th1/Th2 cytokines and decreased granuloma volume in response to SEA injection. | [66] |
|          | Prolonged allograft survival in a transplantation model. | [104] |
|          | Increased susceptibility to *Aspergillus fumigatus* infection. | [67] |
|          | Increased susceptibility to *Toxoplasma gondii*. | [68] |
|          | Protection against lung injury in caerulein-induced acute pancreatitis. | [105] |
|          | Enhanced Th1 responses and glomerular injury in nephrotoxic nephritis model. | [106] |
|          | Altered myeloid progenitor cell mobilization. | [107] |
|          | Resistance to MOG-induced experimental autoimmune encephalitis. | [69] |
| CCR2     | Defects in macrophage recruitment and in production of Th1 cytokines, such as interferon-γ. | [109] |
|          | Unable to clear *Listeria monocytogenes* infection. | [110] |
|          | Susceptible to the injurious effects of intrapulmonary challenge with *Aspergillus fumigatus* spores. | [111] |
|          | Decreased granulomatous inflammation in SEA (Th2) and mycobacterium (Th1). | [112] |
|          | Increased susceptibility to *Leishmania major* infection. | [113] |
|          | Severe glomerular injury after a transient ameliorative effect on proteinuria in a nephrotoxic nephritis model. | [114] |
|          | Impaired neuropathic pain responses. | [115] |
|          | Decreased airway hyperreactivity in cockroach antigen mode. | [116] |
|          | No effect on airways hyperreactivity in OVA-induced asthma. | [117] |
|          | Decreased atherosclerotic lesion size. | [62] |
|          | Resistant to experimental autoimmune encephalitis induced by the MOG peptide. | [63,64] |
|          | Susceptibility to EAE in several different strains of mice. | [65] |
| CCR3     | Decreased basal trafficking of eosinophils to intestinal mucosa; decreased eosinophil recruitment to the lung in OVA-induced asthma model, but increase in airway hyperresponsiveness to methacholine. | [72] |
| CCR4     | No effect on airways inflammation or hyperreactivity in OVA-induced asthma model; significantly decreased mortality in low- and high-dose LPS-induced endotoxic-shock models. | [73] |
|          | Protective effect in chronic fungal allergic disease model induced by *Aspergillus fumigatus* conidia. | [75] |
| CCR5     | Impaired macrophage function and reduced efficiency in the clearance of *Listeria* infection as well as enhanced T-cell dependent immune response (DTH reaction) indicating a protective role. | [3] |
|          | Decreased macrophage infiltration and a concomitant reduction in demyelination in a model of intracranial infection induced by mouse hepatitis virus. | [118] |
|          | No protection against the development of EAE induced by immunization with the MOG peptide. | [70] |
|          | Protection against dextran-sodium-sulphate-induced colitis. | [119] |
|          | Higher mortality following infection with the AIDS-associated pathogen, *Cryptococcus neoformans*, due to defective macrophage recruitment to the brain and inability to eliminate cryptococcal polysaccharide. | [120] |
|          | T-cell-mediated antiviral immunity not affected in lymphocytic choriomeningitis virus infection. | [121] |
|          | Impaired humoral immune response to orally administered antigen and to an enteropathic rotavirus; systemic responses to subcutaneously administered antigens are normal. | [122] |
| CCR6     | Severe and persistent inflammation compared with wild-type mice in 2,4-dinitro-1-fluorobenzene-induced contact hypersensitivity; no inflammation in a DTH model; suggesting a defect in the activation or migration of CD4+ T-cells that down-regulate or elicit the inflammatory response. | [79] |
|          | Reduced airways resistance; fewer eosinophils around the airways; reduced IL-5 in the lung; reduced serum IgE compared with wild-type mice in cockroach antigen model of allergic pulmonary inflammation. | [123] |
|          | Less severe intestinal pathology in dextran sodium sulphate model of IBD compared with wild-type mice; increased susceptibility in the trinitrobenzene sulphonic acid model of IBD compared with wild-type mice. | [124] |
| CCR7     | Impaired migration of naïve B-, T- and antigen-bearing dendritic cells; severely delayed kinetics regarding antibody responses; lack of contact sensitivity and DTH reactions; profound morphological alterations in all secondary lymphoid organs. | [125] |
| CCR8     | No effect in OVA-induced allergic airway disease model or ragweed-allergen-induced peritoneal eosinophilic inflammation. | [78] |
|          | No effect in mouse model of allergic airways disease. | [77] |
|          | Defective Th2 response in SEA, OVA and cockroach antigen models of allergic airways disease. | [76] |
| CCR9     | Phenotypically normal, but decreased, pre-pro B-cells and decreased intraepithelial T-cell/epithelial cell ratio. | [126] |

Table 1 | (Contd.)

| Receptor | Phenotype | Reference |
|----------|-----------|-----------|
| CXCR2    | Decreased neutrophil recruitment; increase in number of B-cells, lymphadenopathy and splenomegaly. Protection against *Toxoplasma gondii* infection. Impaired neutrophil recruitment in endotoxin-induced uveitis and *Onchocerca volvulus*-induced keratitis models of ocular inflammation. Protection against urinary tract infection. Delayed wound-healing response in cutaneous wound-healing model. Significantly reduced progression of atherosclerosis. | [127], but see [128] |
|          |           | [129]     |
|          |           | [130, 131] |
|          |           | [132]     |
|          |           | [133]     |
|          |           | [134]     |
| CXCR3    | Profound resistance to development of acute cardiac allograft rejection; permanent graft acceptance after treatment with a brief, subtherapeutic course of cyclosporin A. Prolongation of graft survival in pancreatic islet allografts recipients. Substantially delayed onset type I diabetes induced with lymphocytic choriomeningitis virus. | [135]     |
|          |           | [136]     |
|          |           | [137]     |
| CXCR4    | Mice die *in utero*; defective vascular development, haematopoiesis and cardiogenesis; defective foetal cerebellar development and hippocampal dentate gyrus morphogenesis. | [2, 138]   |
| CXCR5    | Altered B-cell migration; abnormal germinal centre formation in the spleen; lack of Peyer's patches. | [139]     |
| CX3CR1   | Reduced macrophage recruitment to vessel wall; decreased atherosclerotic lesion formation in CX3CR1-deficient mice in ApoE⁻/⁻ background fed on a high-fat diet. | [81, 82]   |

role for this receptor in neutrophil-mediated inflammation [67, 68]. CCR1⁻/⁻ mice are also resistant to EAE development [69], supporting its potential as a therapeutic target in MS.

CCR5 is probably the most studied chemokine receptor due to its role as the major co-receptor for macrophage tropic HIV infection. The results obtained with CCR5⁻/⁻ mice in different animal models consistently highlight a role for CCR5 in macrophage and T-cell recruitment, sometimes with protective effects and sometimes with exacerbating effects. Surprisingly, CCR5 deletion offered no protection against the development of EAE induced by immunization with the MOG peptide [70] in spite of the high levels of this receptor and its ligand MIP-1α, previously detected in immunohistochemical studies in brain. However, this result is consistent with the observation that humans carrying the Δ32 mutation in CCR5 (discussed below) are afforded no protection against the development of MS, although they do show delayed disease onset [71].

The role of chemokine receptors in allergic disease, particularly asthma, has been more complicated to dissect than it was anticipated. CCR3 has been proposed to play a role in allergic airways disease based on its high-level expression on eosinophils and, to a lesser extent, on mast cells, basophils and Th2 T-cells, cell types associated with the pathogenesis of asthma. Although results from CCR3-deficient mice confirmed its role in the basal trafficking of eosinophils, its involvement in allergic asthma is still paradoxical [72].

The validity of CCR4 as a target in human asthma is also questionable, again based on the results from knockout mice. Th2 T-cells express high levels of CCR4, implying an important role in the development of Th2 responses. However, deletion of CCR4 had no effect in a classical model of Th2-dependent airway inflammation [73]. The lack of effect was initially proposed to be due to compensation by other closely related receptors. However, these results were supported further by more recent studies using CCR4 blockade in a guinea-pig model of airway inflammation [74]. In contrast, in a chronic fungal allergic disease model, CCR4 deficiency had a marked protective effect, suggesting that CCR4 may not be necessary for initiation of the Th2 response, but is probably required for its maintenance [75]. CCR8, like CCR4, has also been shown to be selectively expressed on Th2 subsets. The first report of CCR8-deficient mice confirmed a role for this receptor in Th2 functional responses *in vivo* [76], but subsequent publications by two independent groups suggested that CCR8 alone is not essential for the development of allergic airway disease [77, 78]. On the other hand, knockout studies indicate that CCR6 may now be a valid target in asthma. CCR6 is the receptor for HIP-3α/CCL20 and β-defensins. It shows relatively restricted expression on iDCs (immature dendritic cells) and some T-cell subsets. In a cockroach antigen model of allergic pulmonary inflammation, CCR6⁻/⁻ mice had reduced airway resistance, fewer eosinophils around the airways, decreased IL-5 levels in the lung and reduced serum IgE compared with wild-type mice [79].

Studies on genetic polymorphisms in chemokine and chemokine receptor genes as risk factors for certain diseases may also provide us with valuable target information. The most well-known example is the 32-base deletion in the CCR5-gene-coding sequence (Δ32). This mutation renders resistance to HIV infection, as the resultant truncated CCR5 can no longer be expressed at the cell surface and thereby cannot function as a co-receptor for HIV. This mutation may also render the susceptibility or resistance to other diseases. More recently, a polymorphism has been identified in CX3CR1 (M280), which has reduced binding to fractalkine/CX3CL1, and correlates with protection from coronary artery disease in humans [80]. Studies on CX3CR1⁻/⁻ mice support further a role for this receptor in atherosclerosis [81, 82].

Genetic targeting of chemokine receptors has therefore proved to be an excellent approach to study the function of specific chemokine ligand–receptor interactions *in vivo*, particularly in models of inflammatory and infectious disease. However, although a full extrapolation of experimental data from genetically modified mice to humans has to be made with caution, first due to the compensation that may occur for the loss of a receptor/ligand, and secondly due to the redundancy of the chemokine system, nevertheless it seems that specific roles are emerging for particular receptor–ligand pairs.

### How can we target the chemokine system?

Traditionally, anti-inflammatory strategies have been broadly divided between the biotechnology companies and the pharmaceutical companies as acting outside the cell and inside the cell respectively, although, as with all rules, this is not categorical – signalling intervention is a common approach to modulate all mediators of the immune response. To be more precise, attempts to block the actions of cytokine, growth factor or IL function has focused on biotechnological approaches such as antibodies or cytokine traps for the simple reason that the interaction of these proteins with their cell-surface receptors involves large protein–protein interactions. These have proved particularly difficult to be blocked with small-molecule inhibitors. On the other hand, small-molecule inhibitors, such as cyclosporin, COX (cyclo-oxygenase) inhibitors and steroids, can get inside the cells and interfere with signalling pathways, and are effectively potent and wide-ranging anti-inflammatory therapeutics – with the caveat that they are also general immunosuppressants. However, the chemokine system, which can be considered to be ‘somewhere in between’ lends itself to intervention by both approaches.

In autoimmune disease, it is the inappropriate leucocyte recruitment accompanied with cellular activation that results in disease symptoms and progression. Therefore the suppression or abolition of such excessive and uncontrolled cellular invasion and activation offers an attractive anti-inflammatory strategy. Although anti-cytokine strategies are certainly successful, they are limited by their dual role of good and bad – this is perhaps best exemplified by the unsuccessful trials of anti-TNF-α therapy in sepsis.

In addition to anti-inflammatory strategies based on targeting pro-inflammatory cytokines, many pharmaceutical companies have adopted approaches to inhibit adhesion molecules such as anti-VLA4 (very late antigen-4) (Antegren/Natalizumab), LFA (lymphocyte function-associated antigen)-3/IgG1 fusion (Amevive/Alefacept), which blocks the LFA-3–CD2 interaction, or anti-LFA-1 (Efalizumab/Raptiva). The rationale behind blocking adhesion is certainly very strong, but as the family is limited in size and diversity, and the same family members are involved in both basal trafficking and inflammation, the side effects on routine immune function remain to be evaluated. In addition, even though they are biologically relevant and tractable,

integrins, as well as other adhesion molecules, only turned out to be rather moderate to low-‘druggable’ targets.

It has recently become evident that the HMG-CoA-reductase inhibitors, the statins, could also be powerful anti-inflammatory therapeutics. Interestingly, members of the statin family have been shown to act as allosteric inhibitors of the adhesion molecule, LFA1 [83–84]. Avastatin showed remarkable effects in a murine model of MS, EAE. However, the most effective treatment for MS is IFN-β, and, to date, its precise mechanism of action remains unelucidated.

Therapeutic interference of chemokine-mediated leucocyte recruitment can be accomplished in many different ways [31]. Anti-inflammatory strategies outside the cell require blocking of the ligand–receptor interaction and/or the ligand–GAG interaction, using small-molecule inhibitors, modified chemokines or neutralizing ligands and/or anti-receptor antibodies. In contrast, only small-molecule inhibitors will be useful inside the cell for blocking signalling, as discussed above. Surprisingly, proteins have not been widely developed as therapeutics, possibly since there was the worry that small molecules would rapidly supersede them. Protein biologicals that are currently in pre-clinical development include a truncated CCL2 antagonist, an anti-CCL2 antibody and a variant of RANTES/CCL5 that has abrogated GAG binding.

So far, the most successful approach, widely adopted by the pharmaceutical industry, consists of blocking the chemokine–chemokine receptor interaction by means of specific receptor antagonists. Chemokine receptors, in contrast with traditional cytokine and interleukin receptors, belong to the highly ‘druggable’ superfamily of 7TM-GPCRs. Despite this, early attempts to find potent antagonists met with more difficulties than it was anticipated, based on the experience with GPCRs having small peptide or biogenic amine ligands. The first CXC chemokine receptors, CXCR1 and CXCR2, were identified in 1991 [90,91], followed by the first CC chemokine receptor, CCR1, in 1993 [92]. It was not until 1998 that the first potent small-molecule antagonists were reported, namely, a CXCR2-antagonist developed by SmithKline Beecham (now GlaxoSmithKline) [93] and a CCR1-antagonist from Berlex BioSciences [94]. The interest in antagonizing chemokine receptors was boosted in 1996, following the identification of CCR5 as an essential co-receptor for cell infection by HIV-1 viruses [95,96]. In parallel, much progress was made with respect to the targeted deletion of chemokine receptors in mice, corroborating further their potential as promising targets for the treatment of inflammatory and autoimmune diseases, as described above [97]. Spurred on by these achievements in chemokine receptor biology, the efforts on small-molecule drug-discovery programmes were intensified further, and chemists involved in chemokine-receptor antagonist programmes began to progressively understand the idiosyncrasies of the chemokine receptors and the structural elements required for small molecules to act as antagonists [98]. As a result, today’s patent and scientific literature abounds with reports on potent small-molecule antagonists belonging to numerous different chemical series, which act specifically at a wide variety of therapeutically
relevant chemokine receptors (CCR1, 2, 3, 5 and 10, and CXCR2, 3, 4) [99]. Interestingly, the structural diversity of CC-chemokine-receptor antagonists, as represented by the number of fundamentally distinct chemical series identified, is considerably higher than for CXC-chemokine-receptor antagonists, which suggests that the relative difficulty of finding antagonists may be different between the two categories of receptors. This reflects further that the level of interest, which usually defines the number and breadth of screening campaigns run across the pharmaceutical industry and thus the probability of finding hits, is generally higher for CC than for CXC chemokine receptors.

As the identification of potent antagonists *in vitro* has become relatively straightforward, the field of chemokine-receptor antagonists is facing different challenges today. One important obstacle to expedient compound progression has been the pronounced species selectivity seen with some of the molecules. Thus one of the most prominent CCR1 antagonists reported to date, BX-471, is about two orders of magnitude less potent against the murine compared with the human receptor [100], which is problematic given the increasing number of rodent pharmacological models. Notably, the degree of this species selectivity appears to be an intrinsic characteristic associated with a given chemical series, rather than with the target itself, as illustrated by the fact that some classes of CCR1 antagonists are essentially equipotent on the human and the murine receptors [101], whereas other series are completely inactive on the murine receptor [102]. The second obstacle encountered along the way from potent compounds *in vitro* towards human clinical trials has been the suboptimal pharmacokinetic and toxicity profiles of some of the early chemokine receptor antagonists. From the (naturally) scarce information on the subject that has been disclosed publicly, it is not evident whether this problem is innately more pronounced for chemokine-receptor antagonists than for other small molecules in development. What is clear, however, is that the requirements in terms of safety margin, oral bioavailability, as well as frequency of administration, are particularly stringent for chemokine-receptor antagonists, given that the majority of potential therapeutic applications are situated in the area of chronic disease.

Today, 13 years after the identification of the first chemokine receptors, it is encouraging to see that, despite the difficulties encountered at various stages in the development of chemokine-receptor antagonists towards the clinic, several compounds are now being evaluated in human clinical trials (see Figure 4B). A few compounds have successfully passed Phase I, and it will be more than exciting to witness how they fare in terms of clinical efficacy.

In the present review, we have discussed the rationale, the validation, and a few proof-of-concept cases of pharmacological interference with the chemokine system. Despite a number of encouraging results, some critical issues still need to be addressed: (i) which chemokine receptor and/or ligand needs to be targeted for which disease? (ii) is the high degree of complexity already sufficiently understood,

i.e. did genetic validation address the issue of redundancy or will pharmacology have the final word? (iii) can we develop the right therapeutic agents (small molecules or biologicals) with the appropriate properties to be both efficacious and well-tolerated in a chosen disease? It is to be hoped that several on-going clinical studies will shed more light on these questions in the near future.

### References

1. Nagasawa, T., Hirota, S., Tachibana, K., Takakura, N., Nishikawa, S., Kitamura, Y., Yoshida, N., Kikutani, H. and Kishimoto, T. (1996) Nature (London) **382**, 635–638
2. Tachibana, K., Hirota, S., Izasa, H., Yoshida, H., Kawabata, K., Kataoka, Y.K., Matsushima, K., Yoshida, N., Nishikawa, S. and Kishimoto, T.N. (1998) Nature (London) **393**, 591–594
3. Zhou, Y., Kurihara, T., Ryseck, R.P., Yang, Y., Ryan, C., Loy, J., Warr, G. and Bravo, R. (1998) J. Immunol. **160**, 4018–4025
4. Balashov, K.E., Rottman, J.B., Weiner, H.L. and Hancock, W.W. (1999) Proc. Natl. Acad. Sci. U.S.A. **96**, 6873–6878
5. Sorensen, P.S. (1999) Mult. Scler. **5**, 287–290
6. Strunk, T., Bubel, S., Mascher, B., Schlenke, P., Kirchner, H. and Wandinge, K.P. (2000) Ann. Neurol. **47**, 269–273
7. Ying, S., Meng, Q., Zeibecoglou, K., Robinson, D.S., Macfarlane, A., Humbert, M. and Kay, A.B. (1999) J. Immunol. **163**, 6321–6329
8. Vestergaard, C., Deleuran, M., Gesser, B. and Gronhoj, L.C. (2003) Br. J. Dermatol. **149**, 457–463
9. Reiss, Y., Proudfoot, A.E.I., Power, C.A., Campbell, J.J. and Butcher, E.C. (2001) J. Exp. Med. **194**, 1541–1547
10. Homey, B., Alenius, H., Muller, A., Soto, H., Bowman, E.P., Yuan, W., McEvoy, L., Lauerma, A.I., Assmann, T., Bunemann, E. et al. (2002) Nat. Med. **8**, 157–165
11. Rollins, B.J. (1999) Eur. Cytokine Network **10**, 269–271
12. Rathinaswami, P., Hachicha, M., Wong, W.L., Schall, T.J. and McColl, S.R. (1993) Arthritis Rheum. **36**, 1295–1304
13. Bonecchi, R., Bianchi, G., Bordignon, P.P., D'Ambrosio, D., Lang, R., Borsatti, A., Sozzani, S., Allavena, P., Gray, P.A., Mantovani, A. and Sinigaglia, F. (1998) J. Exp. Med. **187**, 129–134
14. Sallusto, F., Palermo, B., Hoy, A. and Lanzavecchia, A. (1999) Curr. Top. Microbiol. Immunol. **246**, 123–128
15. Butcher, E.C. (1991) Cell **67**, 1033–1036
16. Springer, T.A. (1990) Nature (London) **346**, 425–434
17. Tanaka, Y., Adams, D.H., Hubscher, S., Hirano, H., Siebenlist, U. and Shaw, S. (1993) Nature (London) **361**, 79–82
18. Devalaraja, M.N. and Richmond, A. (1999) Trends Pharmacol. Sci. **20**, 151–156
19. Moser, B. and Loetscher, P. (2001) Nat. Immunol. **2**, 123–128
20. Proudfoot, A.E.I., Handel, T.M., Johnson, Z., Lau, E.K., LiWang, P., Clark-Lewis, I., Borlat, F., Wells, T.N.C. and Kosco-Vilbois, M.H. (2003) Proc. Natl. Acad. Sci. U.S.A. **100**, 1885–1890
21. Amara, A., Lorthioir, O., Valenzuela, A., Magerus, A., Thelen, M., Montes, M., Virelizier, J.L., Delepierre, M., Baleux, F., Lortat-Jacob, H. and Arenzana-Seisdedos, F. (1999) J. Biol. Chem. **274**, 23916–23925
22. Campanella, G.S.V., Lee, E.M.J., Sun, J. and Luster, A.D. (2003) J. Biol. Chem. **278**, 17066–17074
23. Chakravarty, L., Rogers, L., Quach, T., Breckenridge, S. and Kolattukudy, P.E. (1998) J. Biol. Chem. **273**, 29641–29647
24. Graham, G.J., Wilkinson, P.C., Nibbs, R.J., Lowe, S., Kolset, S.O., Parker, A., Freshney, M.G., Tsang, M.L. and Pragnell, I.B. (1996) EMBO J. **15**, 6506–6515
25. Koopmann, W., Ediriwickrema, C. and Krangel, M.S. (1999) J. Immunol. **163**, 2120–2127
26. Kuschert, G.S., Hoogewerf, A.J., Proudfoot, A.E.I., Chung, C.W., Cooke, R.M., Hubbard, R.E., Wells, T.N.C. and Sanderson, P.N. (1998) Biochemistry (Moscow) **37**, 11193–11201
27. Proudfoot, A.E.I., Fritchley, S., Borlat, F., Shaw, J.P., Vilbois, F., Zwahlen, C., Trkola, A., Marchant, D., Clapham, P.R. and Wells, T.N. (2001) J. Biol. Chem. **276**, 10620–10626
28. Spillmann, D., Witt, D. and Lindahl, U. (1998) J. Biol. Chem. **273**, 15487–15493

©2004 Biochemical Society
29 Stringer, S.E. and Gallagher, J.T. (1997) J. Biol. Chem. **272**, 20508–20514

30 Proudfoot, A.E.I. (2002) Nat. Rev. Immunol. **2**, 106–115

31 Proudfoot, A.E.I., Power, C.A., Rommel, C. and Wells, T.N. (2003) Semin. Immunol. **15**, 57–65

32 Mellado, M., Rodríguez-Frade, J.M., Vila-Coro, A.J., Fernández, S., Martin de Ana, A., Jones, D.R., Torán, J.L. and Martínez-A, C. (2001) EMBO J. **20**, 2497–2507

33 Scandella, E., Men, Y., Gillessen, S., Forster, R. and Groettrup, M. (2002) Blood **100**, 1354–1361

34 Wong, M., Uddin, S., Majchrzak, B., Huynh, T., Proudfoot, A.E.I., Platanias, L.C. and Fish, E.N. (2001) J. Biol. Chem. **276**, 11427–11431

35 Wong, M. and Fish, E.N. (1998) J. Biol. Chem. **273**, 309–314

36 Hirsch, E., Katanaev, V.L., Garlanda, C., Azzolino, O., Pirola, L., Silengo, L., Sozzani, S., Mantovani, A., Altruda, F. and Wymann, M.P. (2000) Science **287**, 1049–1053

37 Li, Z., Jiang, H., Xie, W., Zhang, Z., Smrcka, A.V. and Wu, D. (2000) Science **287**, 1046–1049

38 Sasaki, T., Irie-Sasaki, J., Jones, R.G., Oliveira-dos-Santos, A.J., Stanford, W.L., Bolon, B., Wakeham, A., Itie, A., Bouchard, D., Kozieradzki, I. et al. (2000) Science **287**, 1040–1046

39 Stein, R.C. and Waterfield, M.D. (2000) Mol. Med. Today **6**, 347–357

40 Wymann, M.P., Zvelebil, M. and Laffargue, M. (2003) Trends Pharmacol. Sci. **24**, 366–376

41 Laffargue, M., Calvez, R., Finan, P., Trifilieff, A., Barbier, M., Altruda, F., Hirsch, E. and Wymann, M.P. (2002) Immunity **16**, 441–451

42 Wymann, M.P., Bjorklof, K., Calvez, R., Finan, P., Thomast, M., Trifilieff, A., Barbier, M., Altruda, F., Hirsch, E. and Laffargue, M. (2003) Biochem. Soc. Trans. **31**, 275–280

43 Hirsch, E., Bosco, O., Tropel, P., Laffargue, M., Calvez, R., Altruda, F., Wymann, M.P. and Montrucchio, G. (2001) FASEB J. **15**, 2019–2021

44 Rodríguez-Borlado, L., Barber, D.F., Hernández, C., Rodríguez-Marcos, M.A., Sánchez, A., Hirsch, E., Wymann, M.P., Martínez-A, C. and Carrera, A.C. (2003) J. Immunol. **170**, 4475–4482

45 Ward, S., Sotsios, Y., Dowden, J., Bruce, I. and Finan, P. (2003) Chem. Biol. **10**, 207–213

46 Ward, S.G. and Finan, P. (2003) Curr. Opin. Pharmacol. **3**, 426–434

47 Welch, H.C., Coadwell, W.J., Ellson, C.D., Ferguson, G.J., Andrews, S.R., Erdjument-Bromage, H., Tempst, P., Hawkins, P.T. and Stephens, L.R. (2002) Cell **108**, 809–821

48 Small, J.V., Stradal, T., Vignal, E. and Rottner, K. (2002) Trends Cell Biol. **12**, 112–120

49 Miki, H., Yamaguchi, H., Suetsugu, S. and Takenawa, T. (2000) Nature (London) **408**, 732–735

50 Chatah, N.E. and Abrams, C.S. (2001) J. Biol. Chem. **276**, 34059–34065

51 Zhao, Z.S., Manser, E., Chen, X.Q., Chong, C., Leung, T. and Lim, L. (1998) Mol. Cell. Biol. **18**, 2153–2163

52 Edwards, D.C., Sanders, L.C., Bokoch, G.M. and Gill, G.N. (1999) Nat. Cell Biol. **1**, 253–259

53 Condeelis, J.S., Wyckoff, J.B., Bailly, M., Pestell, R., Lawrence, D., Backer, J. and Segall, J.E. (2001) Semin. Cancer Biol. **11**, 119–128

54 Gu, Y., Filippi, M.D., Cancelas, J.A., Siefring, J.E., Williams, E.P., Jasti, A.C., Harris, C.E., Lee, A.W., Prabhakar, R., Atkinson, S.J. et al. (2003) Science **302**, 445–449

55 Walmsley, M.J., Ooi, S.K., Reynolds, L.F., Smith, S.H., Ruf, S., Mathiot, A., Vanes, L., Williams, D.A., Cancro, M.P. and Tybulewicz, V.L. (2003) Science **302**, 459–462

56 Caponigro, F. (2002) Anticancer Drugs **13**, 891–897

57 Caponigro, F., Casale, M. and Bryce, J. (2003) Expert Opin. Invest. Drugs **12**, 943–954

58 Maack, C., Kartes, T., Kilter, H., Schafers, H.J., Nickenig, G., Bohm, M. and Laufs, U. (2003) Circulation **108**, 1567–1574

59 Benard, V., Bokoch, G.M. and Diebold, B.A. (1999) Trends Pharmacol. Sci. **20**, 365–370

60 Le, Y., Cui, Y., Iribarren, P., Ying, G. and Wang, J.M. (2002) *In Vivo* **16**, 1–23

61 Slattery, D.M., Gerard, N. and Gerard, C. (2000) Springer Semin. Immunopathol. **22**, 417–432

62 Peters, W. and Charo, I.F. (2001) Curr. Opin. Lipidol. **12**, 175–180

63 Fife, B.T., Huffnagle, G.B., Kuziel, W.A. and Karpus, W.J. (2000) J. Exp. Med. **192**, 899–905

64 Izikson, L., Klein, R.S., Charo, I.F., Weiner, H.L. and Luster, A.D. (2000) J. Exp. Med. **192**, 1075–1080

65 Gaupp, S., Pitt, D., Kuziel, W.A., Cannella, B. and Raine, C.S. (2003) Am. J. Pathol. **162**, 139–150

66 Gao, J.L., Wynn, T.A., Chang, Y., Lee, E.J., Broxmeyer, H.E., Cooper, S., Tiffany, H.L., Westphal, H., Kwon, C.J. and Murphy, P.M. (1997) J. Exp. Med **185**, 1959–1968

67 Blease, K., Mehrad, B., Standiford, T.J., Lukacs, N.W., Kunkel, S.L., Chensue, S.W., Lu, B., Gerard, C.J. and Hogaboam, C.M. (2000) J. Immunol. **165**, 1564–1572

68 Khan, I.A., Murphy, P.M., Casciotti, L., Schwartzman, J.D., Collins, J., Gao, J.L. and Yeaman, G.R. (2001) J. Immunol. **166**, 1930–1937

69 Rottman, J.B., Slavin, A.J., Silva, R., Weiner, H.L., Gerard, C.G. and Hancock, W.W. (2000) Eur. J. Immunol. **30**, 2372–2377

70 Tran, E.H., Kuziel, W.A. and Owens, T. (2000) Eur. J. Immunol. **30**, 1410–1415

71 Barcellos, L.F., Schito, A.M., Rimmler, J.B., Vittinghoff, E., Shih, A., Lincoln, R., Callier, S., Elkins, M.K., Goodkin, D.E., Haines, J.L. et al. (2000) Immunogenetics **51**, 281–288

72 Humbles, A.A., Lu, B., Friend, D.S., Okinaga, S., Lora, J., Al-Garawi, A., Martin, T.R., Gerard, N.P. and Gerard, C. (2002) Proc. Natl. Acad. Sci. U.S.A. **99**, 1479–1484

73 Chvatchko, Y., Hoogewerf, A.J., Meyer, A., Alouani, S., Juillard, P., Buser, R., Conquet, F., Proudfoot, A.E.I., Wells, T.N.C. and Power, C.A. (2000) J. Exp. Med. **191**, 1755–1763

74 Lloyd, C.M. and Rankin, S.M. (2003) Curr. Opin. Pharmacol. **3**, 443–448

75 Schuh, J.M., Power, C.A., Proudfoot, A.E.I., Kunkel, S.L., Lukacs, N.W. and Hogaboam, C.M. (2002) FASEB J. **16**, 1313–1315

76 Chensue, S.W., Lukacs, N.W., Yang, T.Y., Shang, X., Frait, K.A., Kunkel, S.L., Kung, T., Wiekowski, M.T., Hedrick, J.A., Cook, D.N. et al. (2001) J. Exp. Med. **193**, 573–584

77 Chung, C.D., Kuo, F., Kumer, J., Motani, A.S., Lawrence, C.E., Henderson, Jr, W.R. and Venkataraman, C. (2003) J. Immunol. **170**, 581–587

78 Goya, I., Villares, R., Zaballos, A., Gutierrez, J., Kremer, L., Gonzalo, J.A., Varona, R., Carramolino, L., Serrano, A., Pallares, P. et al. (2003) J. Immunol. **170**, 2138–2146

79 Lukacs, N.W., Prosser, D.M., Wiekowski, M., Lira, S.A. and Cook, D.N. (2001) J. Exp. Med. **194**, 551–555

80 McDermott, D.H., Fong, A.M., Yang, Q., Sechler, J.M., Cupples, L.A., Merrell, M.N., Wilson, P.W., D’Agostino, R.B., O’Donnell, C.J., Patel, D.D. and Murphy, P.M. (2003) J. Clin. Invest. **111**, 1241–1250

81 Combadiere, C., Potteaux, S., Gao, J.L., Esposito, B., Casanova, S., Lee, E.J., Debre, P., Tedgui, A., Murphy, P.M. and Mallat, Z. (2003) Circulation **107**, 1009–1016

82 Lesnik, P., Haskell, C.A. and Charo, I.F. (2003) J. Clin. Invest. **111**, 333–340

83 Weitz-Schmidt, G., Welzenbach, K., Brinkmann, V., Kamata, T., Kallen, J., Bruns, C., Cottens, S., Takada, Y. and Hommel, U. (2001) Nat. Med. **7**, 687–692

84 Weitz-Schmidt, G. (2003) Endothelium **10**, 43–47

85 Reference deleted

86 Reference deleted

87 Reference deleted

88 Reference deleted

89 Reference deleted

90 Holmes, W.E., Lee, J., Kuang, W.J., Rice, G.C. and Wood, W.I. (1991) Science **253**, 1278–1280

91 Murphy, P.M. and Tiffany, H.L. (1991) Science **253**, 1280–1283

92 Neote, K., DiGregorio, D., Mak, J.Y., Horuk, R. and Schall, T.J. (1993) Cell **72**, 415–425

93 White, J.R., Lee, J.M., Young, P.R., Hertzberg, R.P., Jurewicz, A.J., Chaikin, M.A., Widdowson, K., Foley, J.J., Martin, L.D., Griswold, D.E. and Sarau, H.M. (1998) J. Biol. Chem. **273**, 10095–10098

94 Hesselingger, J., Ng, H.P., Liang, M., Zheng, W., May, K., Bauman, J.G., Monahan, S., Islam, I., Wei, G.P., Ghannam, A. et al. (1998) J. Biol. Chem. **273**, 15687–15692

95 Liu, R., Paxton, W.A., Choe, S., Ceradini, D., Martin, S.R., Horuk, R., MacDonald, M.E., Stuhlmann, H., Koup, R.A. and Landau, N.R. (1996) Cell **86**, 367–377

96 Samson, M., Libert, F., Doranz, B.J., Rucker, J., Liesnard, C., Farber, C.M., Saragosti, S., Lapoumeroulie, C., Cognaux, J., Forceille, C. et al. (1996) Nature (London) **382**, 722–725

97 Power, C.A. (2003) J. Immunol. Methods **273**, 73–82

98 Schwarz, M.K. and Wells, T.N. (2002) Nat. Rev. Drug Discov. **1**, 347–358

99 Gao, Z. and Metz, W.A. (2003) Chem. Rev. **103**, 3733–3752

100 Anders, H.J., Vielhauer, V., Frink, M., Linde, Y., Cohen, C.D., Blattner, S.M., Kretzler, M., Strutz, F., Mack, M., Grone, H.J. et al. (2002) J. Clin. Invest. **109**, 251–259

101 Naya, A., Sagara, Y., Ohwaki, K., Saeki, T., Ichikawa, D., Iwasawa, Y., Noguchi, K. and Ohtake, N. (2001) J. Med. Chem. **44**, 1429–1435

102 Liang, M., Rosser, M., Ng, H.P., May, K., Bauman, J.G., Islam, I., Ghannam, A., Kretschmer, P.J., Pu, H., Dunning, L. et al. (2000) Eur. J. Pharmacol. **389**, 41–49

103 Zlotnik, A. and Yoshie, O. (2000) Immunity **12**, 121–127

104 Gao, W., Topham, P.S., King, J.A., Smiley, S.T., Csizmadia, V., Lu, B., Gerard, C.J. and Hancock, W.W. (2000) J. Clin. Invest. **105**, 35–44

105 Gerard, C., Frossard, J.L., Bhatia, M., Saluja, A., Gerard, N.P., Lu, B. and Steer, M. (1997) J. Clin. Invest. **100**, 2022–2027

106 Topham, P.S., Csizmadia, V., Soler, D., Hines, D., Gerard, C.J., Salant, D.J. and Hancock, W.W. (1999) J. Clin. Invest. **104**, 1549–1557

107 Broxmeyer, H.E., Cooper, S., Hangoc, G., Gao, J.L. and Murphy, P.M. (1999) J. Exp. Med. **189**, 1987–1992

108 Reference deleted

109 Boring, L., Gosling, J., Chensue, S.W., Kunkel, S.L., Farese, R.V., Broxmeyer, H.E. and Charo, I.F. (1997) J. Clin. Invest. **100**, 2552–2561

110 Kurihara, T., Warr, G., Loy, J. and Bravo, R. (1997) J. Exp. Med. **186**, 1757–1762

111 Blease, K., Mehrad, B., Standiford, T.J., Lukacs, N.W., Gosling, J., Boring, L., Charo, I.F., Kunkel, S.L. and Hogaboam, C.M. (2000) J. Immunol. **165**, 2603–2611

112 Warmington, K.S., Boring, L., Ruth, J.H., Sonstein, J., Hogaboam, C.M., Curtis, J.L.K., Charo, I.R. and Chensue, S.W. (1999) Am. J. Pathol. **154**, 1407–1416

113 Sato, N., Ahuja, S.K., Quinones, M., Kostecki, V., Reddick, R.L., Melby, P.C., Kuziel, W.A. and Ahuja, S.S. (2000) J. Exp. Med. **192**, 205–218

114 Bird, J.E., Giancarli, M.R., Kurihara, T., Kowala, M.C., Valentine, M.T., Gitlitz, P.H., Pandya, D.G., French, M.H. and Durham, S.K. (2000) Kidney Int. **57**, 129–136

115 Abbadie, C., Lindia, J.A., Cumiskey, A.M., Peterson, L.B., Mudgett, J.S., Bayne, E.K., DeMartino, J.A., MacIntyre, D.E. and Forrest, M.J. (2003) Proc. Natl. Acad. Sci. U.S.A. **100**, 7947–7952

116 MacLean, J.A., De Sanctis, G.T., Ackerman, K.G., Drazen, J.M., Sauty, A., DeHaan, E., Green, F.H., Charo, I.F. and Luster, A.D. (2000) J. Immunol. **165**, 6568–6575

117 Campbell, E.M., Charo, I.F., Kunkel, S.L., Strieter, R.M., Boring, L., Gosling, J. and Lukacs, N.W. (1999) J. Immunol. **163**, 2160–2167

118 Glass, W.G., Liu, M.T., Kuziel, W.A. and Lane, T.E. (2001) Virology **288**, 8–17

119 Andres, P.G., Beck, P.L., Mizoguchi, E., Mizoguchi, A., Bhan, A.K., Dawson, T., Kuziel, W.A., Maeda, N., MacDermott, R.P., Podolsky, D.K. and Reinecker, H.C. (2000) J. Immunol. **164**, 6303–6312

120 Huffnagle, G.B., McNeil, L.K., McDonald, R.A., Murphy, J.W., Toews, G.B., Maeda, N. and Kuziel, W.A. (1999) J. Immunol. **163**, 4642–4646

121 Nansen, A., Christensen, J.P., Andreasen, S.O., Bartholdy, C., Christensen, J.E. and Thomsen, A.R. (2002) Blood **99**, 1237–1245

122 Cook, D.N., Prosser, D.M., Forster, R., Zhang, J., Kuklin, N.A., Abbondanzo, S.J., Niu, X.D., Chen, S.C., Manfra, D.J., Wiekowski, M.T. et al. (2000) Immunity **12**, 495–503

123 Varona, R., Villares, R., Carramolino, L., Goya, I., Zaballos, A., Gutierrez, J., Torres, M., Martínez-A, C. and Márquez, G. (2001) J. Clin. Invest. **107**, R37–R45

124 Varona, R., Cadenas, V., Flores, J., Martínez-A, C. and Márquez, G. (2003) Eur. J. Immunol. **33**, 2937–2946

125 Forster, R., Schubel, A., Breitfeld, D., Kremmer, E., Renner-Muller, I., Wolf, E. and Lipp, M. (1999) Cell **99**, 23–33

126 Wurbel, M.A., Malissen, M., Guy-Grand, D., Meffre, E., Nussenzeig, M.C., Richelme, M., Carrier, A. and Malissen, B. (2001) Blood **98**, 2626–2632

127 Cacalano, G., Lee, J., Kikly, K., Ryan, A.M., Pitts-Meek, S., Hultgren, B., Wood, W.I. and Moore, M.W. (1994) Science **265**, 682–684

128 Erratum (1994) Science **270**, 365

129 Del Rio, L., Bennouna, S., Salinas, J. and Denkers, E.Y. (2001) J. Immunol. **167**, 6503–6509

130 Brito, B.E., O’Rourke, L.M., Pan, Y., Huang, X., Park, J.M., Zamora, D., Cook, D.N., Planck, S.R. and Rosenbaum, J.T. (1999) Curr. Eye Res. **19**, 76–85

131 Hall, L.R., Diaconu, E., Patel, R. and Pearlman, E. (2001) J. Immunol. **166**, 4035–4041

132 Frendeus, B., Godaly, G., Hang, L., Karpman, D., Lundstedt, A.C. and Svanborg, C. (2000) J. Exp. Med. **192**, 881–890

133 Devararaja, R.M., Nannee, L.B., Du, J., Qian, Q., Yu, Y., Devararaja, M.N. and Richmond, A. (2000) J. Invest. Dermatol. **115**, 234–244

134 Boisvert, W.A., Curtiss, L.K. and Terkeltaub, R.A. (2000) Immunol. Res. **21**, 129–137

135 Hancock, W.W., Lu, B., Gao, W., Csizmadia, V., Faia, K., King, J.A., Smiley, S.T., Ling, M., Gerard, N.P. and Gerard, C. (2000) J. Exp. Med. **192**, 1515–1520

136 Baker, M.S., Chen, X., Rotramel, A.R., Nelson, J.J., Lu, B., Gerard, C., Kanwar, Y. and Kaufman, D.B. (2003) Surgery **134**, 126–133

137 Frigerio, S., Junt, T., Lu, B., Gerard, C., Zumsteg, U., Hollander, G.A. and Piali, L. (2002) Nat. Med. **8**, 1414–1420

138 Zou, Y.R., Kottmann, A.H., Kuroda, M., Taniuchi, I. and Littman, D.R. (1998) Nature (London) **393**, 595–599

139 Forster, R., Mattis, A.E., Kremmer, E., Wolf, E., Brem, G. and Lipp, M. (1996) Cell **87**, 1037–1047

140 Bonjouklian, R. et al. (1995) U.S. Pat. 5,378,725

141 Vlahos, C.J., Matter, W.F., Hui, K.Y. and Brown, R.F. (1994) J. Biol. Chem. **269**, 5241–5248

142 Robertson, A.D. et al. (2001) Pat. WO-01/53266

143 Sadhu, C. et al. (2001) Pat. WO-01/81346

144 Sadhu, C., Masinovsky, B., Dick, K., Sowell, C.G. and Staunton, D.E. (2003) J. Immunol. **170**, 2647–2654

145 Melese, T. et al. (2003) Pat. WO-03/034997

146 Melese, T. et al. (2003) Pat. WO-03/035618

147 Rueckle, T. et al. (2004) Pat. WO-04/007491

148 Hayakawa, M. et al. (2001) Pat. WO-01/83481

149 Hayakawa, M. et al. (2001) Pat. WO-01/83456

150 Bruce, I. et al. (2003) Pat. WO-03/072557

Received 21 January 2003
