
JOURNAL OF BACTERIOLOGY, Apr. 2004, p. 2225–2235
0021-9193/04/$08.00+0 DOI: 10.1128/JB.186.8.2225–2235.2004
Copyright © 2004, American Society for Microbiology. All Rights Reserved.

# Organization of *lin* Genes and IS6100 among Different Strains of Hexachlorocyclohexane-Degrading *Sphingomonas paucimobilis*: Evidence for Horizontal Gene Transfer

Charu Dogra,$^{1}$ Vishakha Raina,$^{1}$ Rinku Pal,$^{1}$ Mrutyunjay Suar,$^{1}$ Sukanya Lal,$^{1}$ Karl-Heinz Gartemann,$^{2}$ Christof Holliger,$^{3}$ Jan Roelof van der Meer,$^{4}$ and Rup Lal$^{1\ast}$

**Department of Zoology, University of Delhi, Delhi-110007, India$^{1}$; Department of Microbiology/Gene Technology, University of Bielefeld, 33615 Bielefeld, Germany$^{2}$; and EPFL, ENAC-ISTE, Laboratory of Environmental Biotechnology, CH 1015 Lausanne,$^{3}$ and Process of Environmental Microbiology and Molecular Ecotoxicology, Swiss Federal Institute for Environmental Science and Technology (EAWAG), CH 8600 Duebendorf,$^{4}$ Switzerland**

Received 17 July 2003/Accepted 6 January 2004

The organization of *lin* genes and IS6100 was studied in three strains of *Sphingomonas paucimobilis* (B90A, Sp+, and UT26) which degraded hexachlorocyclohexane (HCH) isomers but which had been isolated at different geographical locations. DNA-DNA hybridization data revealed that most of the *lin* genes in these strains were associated with IS6100, an insertion sequence classified in the IS6 family and initially found in *Mycobacterium fortuitum*. Eleven, six, and five copies of IS6100 were detected in B90A, Sp+, and UT26, respectively. IS6100 elements in B90A were sequenced from five, one, and one regions of the genomes of B90A, Sp+, and UT26, respectively, and were found to be identical. DNA-DNA hybridization and DNA sequencing of cosmid clones also revealed that *S.* *paucimobilis* B90A contains three and two copies of *linX* and *linA*, respectively, compared to only one copy of these genes in strains Sp+ and UT26. Although the copy number and the sequence of the remaining genes of the HCH degradative pathway (*linB*, *linC*, *linD*, and *linE*) were nearly the same in all strains, there were striking differences in the organization of the *linA* genes as a result of replacement of portions of DNA sequences by IS6100, which gave them a strange mosaic configuration. Spontaneous deletion of *linD* and *linE* from B90A and of *linA* from Sp+ occurred and was associated either with deletion of a copy of IS6100 or changes in IS6100 profiles. The evidence gathered in this study, coupled with the observation that the G+C contents of the *linA* genes are lower than that of the remaining DNA sequence of *S.* *paucimobilis*, strongly suggests that all these strains acquired the *linA* gene through horizontal gene transfer mediated by IS6100. The association of IS6100 with the rest of the *lin* genes further suggests that IS6100 played a role in shaping the current *lin* gene organization.

Hexachlorocyclohexane (HCH) was introduced for the control of agricultural pests and of vector-borne diseases in early 1940s. While this compound was used extensively all over the world, several reports on the persistence of HCH isomers (α, β, γ, and δ) and their toxic effects on nontarget organisms appeared in the 1980s (11). These reports finally resulted in a ban on or restricted use of HCH in most countries. Neither the ban nor the restricted use has, however, reduced the levels of HCH residues in the environment (6, 25, 33), especially in soils that had a previous history of HCH application (2). One serious problem is the uptake of HCH residues from soil by crops, which then enter food products (1, 31). In addition to no further use of HCH, a decontamination program for HCH-polluted soils would diminish the risk posed by HCH residues to human, plant, and animal health. One possibility for decontamination is spontaneous or induced microbial degradation. Unfortunately, spontaneous microbial degradation of HCH isomers proceeds rather slowly (10, 15), although a number of bacteria which can degrade one or more isomers of HCH have been isolated. Thus, addition of naturally occurring microbes to contaminated soils could provide an alternative strategy. Such application could be assisted to a great extent by exploring the potential of such isolates, particularly in order to understand the physiology and genetics of HCH degradation in these strains.

As far as is known, aerobic degradation of HCH is carried out mostly by strains of *Sphingomonas paucimobilis* and *Rhodanobacter lindaniclasticus*. HCH-degrading strains have been isolated in different parts of the world; *S.* *paucimobilis* SS86 has been isolated in Japan (32, 38), *S.* *paucimobilis* B90A has been isolated in India (4, 29), and an *R.* *lindaniclasticus* strain has been isolated in France (24, 37). These three strains are remarkably similar and can all degrade α-, γ-, and δ-HCH (9, 14, 29, 37). In addition, *S.* *paucimobilis* B90A can also partially degrade β-HCH (10, 14, 29). The metabolic pathway and the genes involved in HCH degradation have been studied in great detail in *S.* *paucimobilis* strain UT26 (a mutant of SS86 resistant to nalidixic acid) and to a lesser extent in B90 (Fig. 1). The primary enzyme in γ-HCH degradation is HCH dehydrochlorinase encoded by the *linA* gene (9, 23). The remaining genes of the γ-HCH degradative pathway are *linB* (23), *linC* (20), *linD* (18), and *linE* (17, 19), which encode a halidohydrolase, a

* Corresponding author. Mailing address: Department of Zoology, University of Delhi, Delhi-110007, India. Phone: 91-11-27666254. Fax: 91-11-27666254. E-mail: duzdel@del2.vsnl.net.in.

2226 DOGRA ET AL.

![Diagram](attachment:chemical_pathway.png)

FIG. 1. Pathway for degradation of HCH isomers in *S. paucimobilis* strains (data adapted from references 14 and 20). γ-PCCH, gamma-pentachlorocyclohexene; 1,4-TCDN, 1,3,4,6-tetrachloro-1,4-cyclohexadiene; 1,2,4-TCB, 1,2,4 trichlorobenzene; 2,4,5-DNOL, 2,4,5-trichloro-2,5-cyclohexadiene-1-ol; DCP, 2,5-dichlorophenol; 2,5-DDOL, 2,5-dichloro-2,5-cyclohexadiene-1,4-diol; 2,5-DCHQ, 2,5-dichlorohydroquinone; 2-CHQ, 2-chlorohydroquinone; HQ, hydroquinone; γ-HMSA, gamma-hydroxymuconic semialdehyde. The asterisk indicates that in B90 the degradation of α-, γ-, and δ-HCH isomers stops at the level of 2,5-dichlorohydroquinone.

dehydrogenase, a reductive dechlorinase, and a dioxygenase, respectively. In addition, the *linX* gene, encoding a protein that has activity similar to that of LinC, was also cloned and characterized (20). A *linA* gene was also cloned and sequenced from *R. lindaniclasticus* (37) and was found to be identical to *linA* from strain UT26 (9) except for one 3-bp insertion, but the remaining genes of the HCH degradative pathway in this strain have still not been described (37). More recently, the *lin* genes of *S. paucimobilis* strain B90 were cloned and characterized (14, 21). In contrast to UT26 and *R. lindaniclasticus*, strain B90 contains two copies of *linA*, designated *linA1* and *linA2* (Fig. 1). Both copies of *linA* produce a functional HCH dehydrochlorinase when they are cloned in *Escherichia coli*. The amino acid sequences of the products encoded by the *linA1* and *linA2* genes are 92% identical to each other and 88% (LinA1) and 99% (LinA2) identical to the sequence of LinA of strain UT26 (9). The *linB*, *linC*, and *linX* genes were also cloned from strain B90 and were found to be 99% identical to the corresponding genes of *S. paucimobilis* UT26 (14). A closer look at the two copies of *linA* in strain B90 suggested that one of them contains an insertion of an IS element at the 3′ end. This results in a 22-bp difference at the 3′ end between *linA1* (462 bp) and *linA2* (468 bp). The inserted region exhibits complete sequence identity to IS6100 of *Mycobacterium fortuitum* (16). In contrast to strain UT26, no *linD*, *linE*, and *linR* genes could be detected by PCR amplification and Southern hybridization in strain B90 (14).

Here we report on the mosaic character of the *lin* genes in the different *S. paucimobilis* strains and the close association of the IS6100 element with these genes. By using Southern hy-

TABLE 1. Bacterial strains and plasmids used

| Strain or plasmid                     | Description                                                                                   | Source                                      |
|---------------------------------------|---------------------------------------------------------------------------------------------|---------------------------------------------|
| E. coli DH5α                          | F⁻ endA1 hsdR17 supE44 thi-1 recA1 gyrA relA1 Δ(lacZYA-argF) U169 deoR                              | Lab stock                                   |
| E. coli JM101                         | supE thiΔ(lac-proAB) F'[traD36 proAB⁺ lacIq lacZΔM15]                                          | Lab stock                                   |
| S. paucimobilis B90A                  | Aerobic, nonmotile, rod shaped, degrades all four isomers of HCH (α, β, γ, and δ)                 | N. Sethunathan, CRRI, Cuttack, India       |
| S. paucimobilis B90                   | Mutant of S. paucimobilis B90A lacking linD, linE, and linR                                     | Lab stock                                   |
| S. paucimobilis UT26                  | Aerobic, nonmotile, rod shaped, degrades α-, γ-, and δ-HCH but not β-HCH                         | Y. Nagata, University of Tokyo, Tokyo, Japan |
| S. paucimobilis Sp+                   | Aerobic, nonmotile, rod shaped, degrades α-, γ-, and δ-HCH but not β-HCH                         | Tim Vogel, University of Lyon, Lyon, France |
| S. paucimobilis ATCC 29837ᵀ           | Aerobic, nonmotile, rod shaped, yellow pigmented                                               | M. Hanspal, St. Elizabeth's Medical Center of Boston, Boston, Mass. |
| S. chlorophenolica DSM 7098ᵀ          | Aerobic, nonsporulating rods, yellow pigmented, degrades pentachlorophenol, and 2,4,6-trichlorophenol | J. Cullum, University of Kaiserslautern, Kaiserslautern, Germany |
| pUC 13/19                            | 2.7-kb; Ampʳ; multiple cloning site internal to lacZ gene                                       | Lab stock                                   |
| pWE15                                | 8.2-kb cosmid vector; Ampʳ; bacteriophage promoter sequences T3 and T7 flanking unique cloning site | Stratagene                                 |
| pLINA 57                             | pWE15 carrying 41-kb DNA fragment of B90A containing linA1, linC, linX, and IS6100                | This study                                  |
| pLIND 33                             | pWE15 carrying 5.6-kb DNA fragment of B90A containing linD, linE, linR, and IS6100               | This study                                  |
| pLIND 22                             | Subclone of pLIND 33 in pWE15 (bp 1869 to 6700 of the contiguous ~8-kb sequence of pLIND 33)      | This study                                  |
| pLIND 22-12                          | Subclone of pLIND 22 in pUC18 (bp 6701 to 7803 of the contiguous ~8-kb sequence of pLIND 33)      | This study                                  |
| pLIND3R 24                           | Subclone of pLIND 33 in pUC18 (bp 1 to 1868 of the contiguous ~8-kb sequence of pLIND 33)        | This study                                  |
| pLINB 35                             | pWE15 carrying DNA fragment of B90A containing linB and IS6100                                 | This study                                  |
| pLINB 23                             | Subclone of pLINB 35 in pUC18 containing 2.2-kb fragment of pLINB 35                            | This study                                  |
| pLINUTIS                             | pUC18 containing BamHI-digested 3.5-kb fragment of S. paucimobilis UT26 that hybridized with IS6100 | This study                                  |
| pLINSpIS                             | pUC18 containing HindIII-digested 2.2-kb Sp⁺ fragment that hybridized with IS6100               | This study                                  |

bridization, cosmid cloning, and DNA sequencing of strain B90 and an older lab stock of strain B90 (designated B90A) and by comparing these strains with strains UT26 and Sp⁺, we discovered that multiple copies of IS6100 are present in all of the strains, but they are in different configurations. The consequences of the activity of IS6100 for the stability and distribution of the lin genes among these strains are discussed below.

**MATERIALS AND METHODS**

Bacterial strains, plasmids, and culture conditions. Table 1 shows the bacterial strains and plasmids used in this study. S. paucimobilis B90A was obtained from N. Sethunathan (Central Rice Research Institute, Cuttuck, India) in 1992. S. paucimobilis B90, which lacks linD, linE, and linR, is a spontaneous mutant derived from S. paucimobilis B90A (14). S. paucimobilis UT26 and Sp⁺ were obtained from Y. Nagata (University of Tokyo, Yayoi, Bunkyo-ku, Tokyo, Japan) and Tim Vogel (University of Lyon, Lyon, France), respectively. The non-HCH-degrading strains S. paucimobilis ATCC 29837ᵀ and Sphingomonas chlorophenolica DSM 7098ᵀ were also used for detecting any IS6100 elements. All of the S. paucimobilis strains and S. chlorophenolica were grown in Luria broth (30) or in mineral salts (SM) medium containing glucose (1%, wt/vol) as described previously (14). E. coli strains were grown in Luria broth at 37°C. Antibiotics, when required, were added to a final concentration of 150 μg/ml (ampicillin) or 50 μg/ml (kanamycin).

DNA isolation and hybridization. Genomic DNA was isolated from S. paucimobilis strains and S. chlorophenolica by using previously described procedures (14). Plasmids were isolated from E. coli by using standard protocols (30). For DNA hybridization, genomic DNA of S. paucimobilis was digested with suitable restriction enzymes and subjected to gel electrophoresis. Separated DNA fragments were transferred from agarose gels to Hybond-N nylon membranes (Pharmacia Amersham Biotech) and probed with [α-³²P]dATP (BRIT, Hyderabad, India)-labeled DNA fragments. DNA probes were prepared from plasmid DNA either by restriction enzyme digestion followed by elution of the appropriate DNA fragments from an agarose gel or by PCR amplification. The primers used for amplification of different lin gene fragments, IS6100, and tnpA were based on DNA sequences of the corresponding genes in the GenBank database and can be supplied on request. PCR amplification was performed with a Neugen gene thermocycler (Techne Progeny, Cambridge, United Kingdom) or a Robocycler (Stratagene) by using standard protocols. Hybridization was performed at 68°C. After hybridization, the membranes were washed twice in a solution containing 2× SSC (1× SSC is 150 mM NaCl plus 15 mM sodium citrate, pH 7.0) plus 0.1% sodium dodecyl sulfate at 68°C and once in a solution containing 1× SSC plus 0.1% sodium dodecyl sulfate at room temperature. The membranes were then exposed to X-ray film (Kodak India, Mumbai, India).

Construction and screening of the genomic library of S. paucimobilis B90A. A genomic library of S. paucimobilis B90A was constructed as described previously for strain B90 (14) in cosmid vector pWE15 (Stratagene). This library was screened specifically for the presence of clones in which lin genes were associated with IS6100. Screening of the library was carried out by using [α-³²P]dATP-labeled linA, linB, linC, linD, linE, linX, and IS6100 probes separately. The DNA probes for lin genes were prepared by PCR amplification by using primers designed for amplification from the open reading frames (ORFs) of lin genes. The IS6100 probe was prepared by amplification of the internal fragment of IS6100. The cosmid library (around 2,000 colonies) of B90A was screened by using each probe under stringent conditions (68°C), followed by stringent washing as described above. The tentative clones selected after screening were further confirmed by Southern hybridization. After screening, 15 cosmid clones that gave positive signals with one or more lin genes and/or IS6100 were selected (Table 2).

Determination of copy numbers of lin genes and IS6100 in the S. paucimobilis strains. In order to determine the approximate copy numbers of IS6100 and the lin genes, the genomic DNAs of S. paucimobilis B90A, B90, UT26, and Sp⁺ were

TABLE 2. Pattern of hybridization signals in S. paucimobilis B90A cosmid clones with *lin* genes and IS6100 as probes

| Cosmid | Hybridization signals with the following DNA probes: |
|--------|-----------------------------------------------------|
|        | *linA* | *linB* | *linC* | *linD* | *linE* | *linX* | IS6100 |
| 33     | $-^{a}$ | – | – | + | + | – | + |
| 35     | – | + | – | – | – | – | + |
| 36     | + | – | + | – | – | + | + |
| 39     | + | – | – | – | – | + | – |
| 42     | – | – | – | + | + | – | + |
| 43     | + | – | – | – | – | + | – |
| 45     | – | + | – | + | – | – | + |
| 49     | – | + | – | – | – | – | + |
| 50     | + | – | – | – | + | – | + |
| 52     | – | + | – | – | – | – | + |
| 53     | + | – | – | – | – | – | – |
| 55     | – | – | + | – | – | – | + |
| 56     | – | + | – | – | – | – | + |
| 57     | + | – | + | – | – | + | + |
| 59     | – | – | + | – | – | – | + |

$a$ –, no hybridization signal; +, hybridization signal.

of 5 μg/ml. After the stationary phase was reached (3 to 4 days), γ- or β-HCH was extracted from the samples and analyzed with a gas chromatograph (GC 17A; Shimadzu, Kyoto, Japan). The extraction protocol and conditions used for the gas chromatography analysis have been described previously (14). After analysis for degradation of HCH, cultures that were not able to degrade HCH (mutants) were selected; genomic DNAs were isolated from these mutants and hybridized with [α-32P]dATP-labeled *linA* or IS6100 gene fragments as probes. In order to attribute the loss of *linA* of Sp+ to homologous recombination involving IS6100 sequences, genomic DNAs of Sp+ and four mutants were digested with HindIII (a HindIII site is present in IS6100 and not present in *linA* of Sp+) and BamHI (BamHI does not cut either IS6100 or *linA*). The DNAs were then hybridized by using [α-32P]dATP-labeled *linA* and IS6100 probes separately. B90, a spontaneous mutant of B90A, which lacked *linD*, *linE*, and *linR*, was also tested to determine its ability to convert the two possible intermediates, chlorohydroquinone and hydroquinone, in γ-HCH degradation (Fig. 1) by using the method described by Nagata et al. (22).

Nucleotide sequence accession numbers. The nucleotide sequences of the cloned fragments containing the *lin* genes and their flanking regions in B90A have been deposited in the National Center for Biotechnology Information database under accession numbers AY331258, AY331259, and AY334273.

**RESULTS**

Hybridization of a genomic library of strain B90A with IS6100 and *lin* gene probes. A genomic library was prepared from *S. paucimobilis* B90A DNA, an older laboratory stock culture of strain B90, which was described previously (14). This library was screened by hybridization with radioactively labeled probes for all *lin* genes and for IS6100. From around 2,000 cosmid clones, we selected 15 cosmids which positively hybridized to at least one of the probes (Table 2). Several patterns of cohybridization with different *lin* markers were observed. Cosmids that hybridized to *linC* invariably also hybridized with the IS6100 probe. Two of these cosmids (cosmids 36 and 57) hybridized with *linA* and *linX* as well. One cosmid (cosmid 50) hybridized to *linA*, *linX*, and IS6100 but not to *linC*. This suggested that this cosmid overlapped the two other cosmids described above. Interestingly, three cosmids that hybridized to *linA* and *linX* did not hybridize to IS6100 (cosmids 39, 43, and 53). As argued below, these cosmids might contain the other copies of *linA* and *linX* present in strain B90A. All cosmids that hybridized to *linB* (cosmids 35, 45, 49, 52, and 56) also hybridized to IS6100, but in only one case (number 45) was hybridization detected with one other *lin* probe, namely, *linD*. This suggested that *linB* and *linD* are relatively close to each other (within 20 to 30 kb). Two cosmids (cosmids 33 and 42) hybridized with *linD* and *linE* and also with the IS6100 probe.

To further analyze the patterns of hybridization with *linX* and *linA*, digested chromosomal DNAs of strain B90A (and strains B90, UT26, and Sp+) were hybridized with the same probes (Fig. 2). DNA that was digested with HindIII and hybridized with *linX* resulted in one hybridizing band at ~4.5-kb for Sp+ and UT26 DNAs (Fig. 2A, lanes 1 and 2). However, with strains B90 and B90A two bands hybridized to *linX* (Fig. 2A, lanes 3 and 4). This suggested that more than one copy of *linX* was present in strains B90A and B90. Similarly, when hybridized with a *linA* probe, DNAs from strains B90A and B90 produced two bands (Fig. 2B, lanes 2 to 5), whereas DNA from UT26 or Sp+ resulted in only one hybridizing band (Fig. 2B, lanes 6 to 9). The presence of two copies of *linA* in strain B90A was in agreement with previous results for strain B90 (14).

VOL. 186, 2004

lin GENES AND IS6100 IN S. PAUCIMOBILIS 2229

FIG. 2. (A) Southern blot of total DNAs of *S. paucimobilis* strains digested with HindIII and hybridized with \[^{32}P\]dATP-labeled *linX*. Lane 1, *S. paucimobilis* UT26; lane 2, *S. paucimobilis* Sp+; lane 3, *S. paucimobilis* B90; lane 4, *S. paucimobilis* B90A; lane 5, Gene Ruler DNA ladder mixture (MBI Fermentas). (B) Southern blot of total DNAs of *S. paucimobilis* strains digested with BamHI and HindIII and hybridized with \[^{32}P\]dATP-labeled *linA*. Lane 1, Gene Ruler DNA ladder mixture; lane 2, BamHI-digested DNA of *S. paucimobilis* B90A; lane 3, HindIII-digested DNA of *S. paucimobilis* B90A; lane 4, BamHI-digested DNA of *S. paucimobilis* B90; lane 5, HindIII-digested DNA of B90; lane 6, BamHI-digested DNA of *S. paucimobilis* UT26; lane 7, HindIII-digested DNA of UT26; lane 8, BamHI-digested DNA of *S. paucimobilis* Sp+; lane 9, HindIII-digested DNA of Sp+.

Sequence analysis of the *lin* genes and IS6100 in strain B90A. Several cosmids were selected for further DNA sequencing. The insert of cosmid 57 (pLINA57) was sequenced completely. It contained an approximately 41-kb insert harboring 33 ORFs with clear database homologies (Fig. 3A and Table 3). As expected from the hybridization results, the insert of cosmid 57 contained a *linA* gene and a *linC* gene. According to the DNA sequence, the *linA* copy present in this region is the copy previously cloned from strain B90 and designated *linA1* to distinguish it from the slightly different second copy. The *linC* gene was identical to the gene previously reported for strain B90 (14). In contrast to *linA1* and *linC*, two copies of a *linX* gene were present; however, these copies were only 66% identical to each other. The *linX* copies were designated *linX1* and *linX2* (the *linX1* and *linX2* genes were represented by a ~3.5-kb fragment of HindIII-digested DNA that hybridized with the *linX* probe [Fig. 2A, lane 4]), and *linX1* exhibited 99% DNA sequence identity to *linX* of strain UT26 (22). Apart from the *lin* genes, three copies of a sequence (IS6100A, IS6100B, and IS6100C) identical to IS6100 were present (Fig. 3A and Table 3), as were many other genes presumably not related to HCH degradation, since the potentially encoded polypeptides had relatively clear homologies to proteins with completely different functions (such as cytochrome *c* oxidase or Nif proteins). Interestingly, two copies of the IS6100 element flanked the *linC* gene and the ORFM gene, possibly making a composite transposon. No target site duplication was observed when the boundary sequences of the different IS6100 copies were compared (Fig. 3B).

A 2.2-kb region of cosmid 35 (pLINB35) containing the *linB* gene was also sequenced. This analysis revealed that *linB* in strain B90A and the immediate region upstream of *linB* were 99% similar to the corresponding regions in strain UT26 but that a copy of IS6100 (IS6100D) was located downstream of *linB* (Fig. 3C).

FIG. 3. (A) Partial physical and genetic map of pLINA57 containing the ~41-kb fragment from *S. paucimobilis* B90A. The ORFs deduced from the complete 13-kb nucleotide sequence are indicated by arrows showing the direction of transcription. Details of the coding regions are summarized in Table 3. (B) Comparison of the nucleotide sequences adjacent to five different IS6100 copies. Nucleotide sequences in lowercase letters are common sequences at the terminal ends of each IS6100. IS6100A, IS6100B, and IS6100C are present in pLINA57, and IS6100D and IS6100E are present in pLINB23 and pLIND33, respectively. The sequence of one of the flanking regions of IS6100D could not be determined. (C) Physical and genetic map of the region of pLINB35 containing *linB* and IS6100. (D) Physical and genetic map of the region of pLIND33 containing *linD*, *linE*, *linR*, and IS6100.

A third separate 7,808-bp region with the *linD*, *linE*, and *linR* genes was sequenced from cosmid 33 (pLIND33) and several plasmid subclones. The relative position of these genes with respect to *linA1* is not known. The nucleotide sequences of *linD*, *linE*, and *linR* were around 99 to 100% identical to those of *linD*, *linE*, and *linR* of strain UT26 (Table 4). Even the ORFs between *linD* and *linE* were the same as those described for UT26 (17, 19). Interestingly, however, both the DNA sequences upstream of *linR* and the DNA sequences downstream of *linD* were different in strains B90A and UT26. Again, an intact copy of IS6100 (IS6100E) was present downstream of *linD* (Fig. 3D). The hybridization data showed that yet another copy of IS6100 was upstream of *orf1* (data not shown), but its exact location was not determined further.

Since the three cosmids that hybridized to *linA* and *linX* (cosmids 39, 43, and 53) did not hybridize to IS6100 but sequencing of the *linA1* region of cosmid 57 showed that IS6100 was located immediately downstream of *linA1*, we assumed

| Protein | Length (amino acids) | % Identity/% similarity | Putative function | Best hit bacterium (accession no.) |
|---------|----------------------|------------------------|------------------|-----------------------------------|
| ORFA    | 306                 | 39/49                 | Hypothetical protein | Novosphingobium aromaticivorans (ZP_00093610) |
| ORFB    | 323                 | 47/66                 | TPP-dependent acetoin dehydrogenase alpha subunit | Clostridium magnum (AAA21744) |
| ORFC    | 327                 | 47/64                 | Acetoin dehydrogenase (TPP dependent) beta subunit | Clostridium magnum (I40791) |
| ORFC1   | 79                  | 45/67                 | Partial dihydrolipoamide acetyltransferase | Klebsiella pneumoniae (AAC13741) |
| ORFD    | 846                 | 31/48                 | pfam 00593, TonB_boxC, TonB-dependent receptor C-terminal region | Novosphingobium aromaticivorans (ZP_00095375) |
| ORFE    | 313                 | 44/52                 | COG 1975, xanthine and CO dehydrogenases, maturation factor | Magnetospirillum magnetotacticum (ZP_00054040) |
| ORFF    | 330                 | 41/59                 | COG 0303, MoeA, molybdopterin biosynthesis enzyme | Brucella melitensis (AAL53083) |
| ORFG    | 448                 | 68/80                 | COG 1249, pyruvate/2-oxoglutarate dehydrogenase complex, dihydrolipoamide dehydrogenase (E3) component | Novosphingobium aromaticivorans (ZP_00094170) |
| ORFH    | 412                 | 72/82                 | COG 1960, acyl-coenzyme A dehydrogenases | Novosphingobium aromaticivorans (ZP_00095926) |
| LinX1   | 250                 | 99/99                 | 2,5-Dichloro-2,5-cyclohexadiene-1,4-diol dehydrogenase (pfam 00106, short-chain dehydrogenase) | Sphingomonas paucimobilis UT26 (BAA04939) |
| LinX2   | 250                 | 65/81                 | 2,5-Dichloro-2,5-cyclohexadiene-1,4-diol dehydrogenase (pfam 00106, short-chain dehydrogenase) | Sphingomonas paucimobilis UT26 (BAA04939) |
| LinA1   | 154                 | 92/95                 | γ-Hexachlorocyclohexane dehydrochlorinase | Sphingomonas paucimobilis UT26 (BAA14369) |
| TnpA1   | 264                 | 100/100               | Transposase of IS6100 (COG3316, transposase and inactivated derivatives) | Aeromonas salmonicida (CAD57187) |
| ORFJ    | 98                  | 80/88                 | Putative glutathione S-transferase (COG0625, glutathione S-transferase) | Sphingobium chlorophenolicum (AAM96661) |
| ORFK    | 205                 | 86/92                 | pfam 02230, αβ-hydrolase_2, phospholipase/carboxylesterase | Sphingobium chlorophenolicum (AAM96663) |
| ORFL    | 175                 | 94/96                 | Chloromaleylacetate reductase (COG1454, alcohol dehydrogenase, class IV) | Sphingobium chlorophenolicum (AAM96664) |
| TnpA2   | 264                 | 100/100               | Transposase of IS6100 (COG3316, transposase and inactivated derivatives) | Aeromonas salmonicida (CAD57187) |
| LinC    | 250                 | 98/98                 | 2,5-Dichloro-2,5-cyclohexadiene-1,4-diol dehydrogenase (pfam 00106, short-chain dehydrogenase) | Sphingomonas paucimobilis UT26 (BAA04939) |
| ORFM    | (451)               | 30/49                 | pfam 00593, TonB-dependent receptor C-terminal region | Novosphingobium aromaticivorans (ZP_00095375) |
| TnpA3   | 264                 | 100/100               | Transposase of IS6100 (COG3316, transposase and inactivated derivatives) | Aeromonas salmonicida (CAD57187) |
| ORFN    | (86)                | 37/59                 | Outer membrane protein (COG3047, outer membrane protein W) | Pseudomonas putida KT2440 (AAN66132) |
| ORFO    | 254                 | 62/72                 | Hypothetical regulator (COG0664, cAMP-binding protein-catabolite gene activator) | Novosphingobium aromaticivorans (ZP_00096011) |
| ORFP    | 552                 | 78/86                 | COG3278, cbb₃-type cytochrome oxidase, subunit 1 | Novosphingobium aromaticivorans (ZP_00096008) |
| ORFQ    | 248                 | 81/88                 | pfam 02433, cytochrome c oxidase, monoheme subunit | Novosphingobium aromaticivorans (ZP_00096007) |
| ORFR    | 295                 | 59/75                 | COG2010, cytochrome c, mono-and diheme variants | Novosphingobium aromaticivorans (ZP_00096005) |
| ORFS    | 476                 | 69/80                 | COG0348, polyferredoxin | Novosphingobium aromaticivorans (ZP_00096004) |
| ORFT    | 158                 | 46/62                 | COG5456, predicted integral membrane protein linked to a cation pump | Novosphingobium aromaticivorans (ZP_00096003) |
| ORFU    | 710                 | 58/71                 | COG2217, cation transport ATPase | Novosphingobium aromaticivorans (ZP_00096002) |
| ORFV    | 1,200               | 70/79                 | Bifunctional proline dehydrogenase/δ-1-pyrroline-5-carboxylate dehydrogenase (COG4230, δ-1-pyrroline-5-carboxylate dehydrogenase; COG0506, proline dehydrogenase | Sinorhizobium meliloti (CAC41903) |
| ORFW    | 233                 | 54/64                 | COG2755, lysophospholipase L1 and related esterases | Novosphingobium aromaticivorans (ZP_00094852) |
| ORFX    | 233                 | 68/81                 | COG4181, predicted ABC-type transporter involved in lysophospholipase L1 biosynthesis, ATPase component | Novosphingobium aromaticivorans (ZP_00094853) |
| ORFY    | 840                 | 50/64                 | COG3127, predicted ABC-type transporter involved in lysophospholipase L1 biosynthesis, permease component | Novosphingobium aromaticivorans (ZP_00094854) |
| ORFZ    | (590)               | 43/62                 | Two-component hybrid sensor and regulator | Pseudomonas fluorescens (ZP_00084332) |

TABLE 4. Coding regions of the insert of pLIND33 of S. paucimobilis B90A

| Protein | Length (amino acids) | % Identity/% similarity | Putative function | Best hit Bacterium (accession no.) |
|---------|-----------------------|------------------------|--------------------|-------------------------------------|
| ORF1    | 385                   | 45/60                 | Hypothetical TonB-dependent receptor); pfam 00593, TonB_dep_Rec, TonB-dependent receptor | Novosphingobium aromaticivorans (gi 23107064) |
| LinR    | 303                   | 99/99                 | LysR-type transcriptional regulator for linD | Sphingomonas paucimobilis UT 26 (gi 4092848) |
| LinE    | 321                   | 100/100               | Hydroquinone meta-cleavage dioxygenase | Sphingomonas paucimobilis UT 26 (gi 4587228) |
| ORF2    | 269                   | 35/51                 | β-Ketoadipate-enol-lactone hydrolase; pfam 00561, abhydrolase, alpha/beta hydrolase fold | Ralstonia solanacearum (gi 17427307) |
| ORF3    | 238                   | 26/42                 | Carboxylesterase; pfam 02230, abhydrolase_2, phospholipase/carboxylesterase | Burkholderia fungorum (gi 22989674) |
| ORF4    | 210                   | 33/47                 | Conserved hypothetical protein; COG 2350, uncharacterized protein conserved in bacteria | Ralstonia eutropha (gi 11967267) |
| LinD    | 346                   | 99/100                | 2,5-Dichlorohydroquinone reductive dechlorinase | Sphingomonas paucimobilis UT 26 (gi 1731852) |
| TnpA5   | 264                   | 100/100               | Transposase of IS6100 (COG 3316, transposase and inactivated derivatives) | Aeromonas salmonicida (CAD 57187) |
| ORF5    | 69                    | 45/59                 | Multidrug resistance protein VceB (truncated, only C-terminal portion) | Vibrio cholerae (gi 2815578) |

2232 DOGRA ET AL.
J. BACTERIOL.

![Image Description](#)

FIG. 4. (A) Southern blot hybridization of genomic DNAs of S. paucimobilis B90A, B90, Sp+, and UT26 digested with BamHI and hybridized with [α-³²P]dATP-labeled IS6100. Lane 1, Gene Ruler DNA ladder mixture; lane 2, B90A; lane 3, B90; lane 4, Sp+; lane 5, UT26. (B) Southern blot hybridization of PstI-digested genomic DNAs of S. paucimobilis B90A, B90, Sp+, and UT26 hybridized with [α-³²P]dATP-labeled *linD* (panel a) and *linE* (panel b) as probes. Lane 1, Gene Ruler DNA ladder mixture; lane 2, B90A; lane 3, B90; lane 4, Sp+; lane 5, UT26. (C) Southern blot hybridization of genomic DNAs of S. paucimobilis Sp+ mutants digested with HindIII and BamHI and hybridized with [α-³²P]dATP-labeled IS6100 (panel a) and *linA* (panel b). Lane 1, Gene Ruler DNA ladder mixture; lanes 2 to 6, HindIII-digested genomic DNAs of Sp+ and mutants 1 to 4; lanes 7 to 11, BamHI-digested genomic DNAs of Sp+ and mutants 1 to 4.

TABLE 5. Comparison of various HCH degradative genes and IS6100 in S. paucimobilis B90A, UT26, and Sp+

| Gene       | No. of nucleotides (no. of amino acids in): | G+C content (%) in: | Stability in:$^c$ |
|------------|-------------------------------------------|---------------------|--------------------|
|            | Strain B90A                              | Strain UT26         | Strain Sp+         | Strain B90A      | Strain UT26      | Strain Sp+        |
| $linA1$    | 462 (154)                                | ND$^d$              | 462 (154)          | 52.7             | ND               | 52.7              | Dehydrochlorinase | ++                 | ND                | –                  |
| $linA2/linA$ | 468 (156)                                | 468 (156)           | ND                 | 53.9             | ND               | 53.9              | Dehydrochlorinase | ++                 | +                  | ND                 |
| $linB$     | 888 (296)                                | 888 (296)           | 888 (296)          | 62.5             | 62.5             | 62.5              | Halidohydrolase   | ++                 | +                  | –                  |
| $linC$     | 750 (250)                                | 750 (250)           | 750 (250)          | 64.5             | 64.5             | 64.5              | Dehydrogenase     | ++                 | +                  | –                  |
| $linD$     | 1,038 (346)                              | 1,038 (346)         | 1,038 (346)        | 61.8             | 61.8             | 61.8              | Reductive dechlorinase | +                  | +                  | ++                 |
| $linE$     | 963 (321)                                | 963 (321)           | 963 (321)          | 60.1             | 60.1             | 60.1              | Ring cleavage dioxygenase | +                  | +                  | ++                 |
| $linR$     | 909 (303)                                | 909 (303)           | 909 (303)          | 60.3             | 60.3             | 60.3              | Transcriptional regulator | +                  | +                  | +                  |
| $linX1$    | 750 (250)                                | 750 (250)           | 750 (250)          | 64.5             | 64.5             | 64.5              | Dehydrogenase     | ++                 | +                  | +                  |
| $linX2$    | 750 (250)                                | ND                  | ND                 | 64.5             | ND               | ND                | Dehydrogenase     | ++                 | ND                | ND                 |
| $linX3$    | 750 (250)                                | ND                  | ND                 | 64.5             | ND               | ND                | Dehydrogenase     | ++                 | ND                | ND                 |
| $tnpA$     | 792 (264)                                | 792 (264)           | 792 (264)          | 61               | 61               | 61                | Transposase       | +                  | nd$^e$            | –                  |

$a$ Summary of data presented in this paper and references 14 and 20.

$b$ The functions of $lin$ genes are based on the data from previous work (14) and on the data for corresponding $lin$ genes of UT26 (20).

$c$ ++, highly stable; +, stable; –, unstable.

$d$ ND, not detected.

$e$ nd, not determined.

transposon Tn5393b from *Xanthomonas campestris* pv. vesicatoria, where it was shown to increase the expression of the streptomycin resistance genes *strA* and *strB* (35). Similarly, IS6100 has been found in *Salmonella enterica* serovar Typhimurium (5), in plasmid pACM1 from *Klebsiella oxytoca* (27), in plasmid pTET3 in *Corynebacterium glutamicum* (36), and in plasmid pRASI in *Aeromonas salmonicida* (34). There has been no previous report of IS6100 elements in any strain of *S. paucimobilis*, and using Southern blot hybridization, we were unable to detect IS6100 elements in *S. paucimobilis* ATCC 29837<sup>T</sup> and *S. chlorophenolica* DSM 7098<sup>T</sup>. Thus, the previously published data on the presence of IS6100 in different bacterial strains and the presence of identical copies of IS6100 only in HCH-degrading *S. paucimobilis* strains indicate that IS6100 plays a vital role in disseminating genes, including catabolic and antibiotic resistance genes, among different bacteria. These data also indicate that IS6100 has been disseminated both widely and recently among different bacterial species without any change in nucleotide sequence. The presence of IS6100 in distantly related bacterial species, ranging from *M. fortuitum* to *S. paucimobilis*, further suggests that IS6100 elements have a very broad host range, and their presence on plasmids (even in strains in which the location has not been ascertained) cannot be ruled out.

The genetic structures of the *lin* genes in the different *S. paucimobilis* strains are a remarkable example of rearrangements and pathway evolution, although the implications of the rearrangements in the different strains are not fully understood. For example, data obtained in this work demonstrated that strains B90 and B90A contain two copies of a *linA* gene, whereas both Sp+ and UT26 carry only one copy. Although the exact specificity differences of *linA1* and *linA2* have not been unequivocally determined, the fact remains that strains B90A and B90 can (partially) degrade β-HCH, whereas Sp+ and UT26 cannot. In addition, three copies of a *linX* gene are present in strain B90A. One of the copies, *linX2*, differs considerably from the other two. Only one copy of *linX* has been found in strains UT26 and Sp+. While the enzymatic activity of the *linX* gene (*linX1*, *linX2*, and *linX3*) products for HCH degradation in B90A has not been studied yet, the presence of two copies of *linA* and three copies of *linX* might influence HCH degradation rates. In fact, degradation of α-, γ-, and δ-HCH proceeded at a higher rate in B90A than in UT26 and Sp+ (R. Lal, unpublished data).

Despite the different geographic locations at which they were first isolated, the HCH-degrading *S. paucimobilis* strains have identical *lin* genes (except *linA1* of B90A), even though the *lin* gene organization is not the same in all of the strains. This supports the hypothesis that the *lin* genes were distributed relatively recently and by and large have not accumulated strain-specific mutations yet and that they originated from a single source. We have very little information to determine whether the *lin* genes were assembled once in one microorganism and were subsequently disseminated or whether the *lin* genes were distributed on a self-transmissible DNA fragment to different suitable (*S. paucimobilis*) hosts. In two strains which we studied (B90A and Sp+) and in two other strains described previously (9, 37), the five *linA* genes had a mosaic organization, and at the same time their amino acid and nucleic acid sequences exhibited high levels of similarity. In B90A and B90, the C terminus of the *linA1* gene has been replaced by a copy of IS6100, and *linA* of Sp+ only contains the first 22 nucleotides of the IS6100 sequence towards the C terminus and not a complete copy of IS6100 (like *linA1* of B90A). A complete copy of IS6100 appears to be present away from the 3′ end of the *linA* gene of Sp+ (the exact location of a complete copy of IS6100 in the vicinity of the *linA* gene of Sp+ has not been determined yet), and the *linA* gene of UT26 and the *linA2* gene of B90A do not appear to be associated with IS6100. In addition, the G+C contents of all of the *linA* genes reported so far are lower than those of the *linB*, *linC*, *linD*, and *linE* genes, suggesting that *linA* might have been acquired by the strains through horizontal gene transfer from an external donor, as proposed previously (14, 20), by IS6100 involvement. Eventually, the association of *linA1* with IS6100 might have triggered a duplication process, thereby adding a second copy in B90A. This hypothesis is supported by the presence of IS6100 near *linA1*. In fact, the current structure of the *linA1* and *linA2* regions in strain B90A suggests that the genetic organization of *linX* and *linA* in strain UT26 was the result of recombination between the 5′ ends of the *linA1* and *linA2* sequences (14). Additional gene reshuffling followed by further transposition or recombination between IS6100 copies may then have led to activation of rest of the genes (*linB*, *linC*, *linD*, *linE*) that appear to be indigenous to the strain (20).

In spite of the fact that B90A contains 11 copies of IS6100, compared to the 6 and 5 copies in Sp+ and UT26, respectively, the *linA* gene was more stable in B90A than in Sp+ (this study) and UT26 (20), indicating that the copy number of IS6100 may not be the only factor that contributes to the stability of *lin* genes. The stability of genes within a genome is controlled by several underlying mechanisms operating within the genome, including homologous recombination between elements at different genomic locations, duplications, deletions, inversions, translocations, and transductions (13). The loss of the *linA* gene in Sp+ was always associated with a change in the profile of the IS6100 copies in all the mutants, and the loss in at least two mutants was accompanied by a loss of a copy of IS6100 associated with *linA*. Likewise, the presence of IS6100 flanking *linD* and *orfI* in B90A also suggests that there was a similar additional deletion event, facilitated by homologous recombination between these two copies, resulting in the formation of B90. This is a very recent step that took place in the laboratory.

In conclusion, the mosaic nature of the *linA* genes, the association of IS6100 with the *lin* genes, and the deletions of IS6100 associated with deletions of the *lin* genes indicate the underlying role that IS6100 elements have played in establishing this pathway. Interestingly, the *lin* genes are not located in a single operon, like *linD*, *linE*, and *linR*, but are present in several different regions of the genome. This is in contrast to the genes for many catabolic pathways in pseudomonads, for example, which are more closely organized, but it is similar to the genes in other sphingomonads, such as the genes for polycyclic aromatic hydrocarbon degradation in *Sphingomonas aromaticivorans* (28) and mecoprop degradation in *Sphingomonas herbicidovorans* (van der Meer, unpublished data). The present study brought up several interesting questions, including questions related to finding the original host containing *linA*, the evolution of the β-HCH degradation pathway in B90A, and the mechanism of formation and the role of multiple copies of *lin*

genes in some strains, which will require a series of new experimental studies. On the applied side, the occurrence of stable HCH-degrading microorganisms at different locations on the globe suggests that microorganisms have the ability to adapt to this pollutant, which is a hopeful sign for remediation of HCH contamination.

# ACKNOWLEDGMENTS

We thank Y. Nagata, Department of Biotechnology, University of Tokyo, and Tim Vogel, University of Lyon, for providing *S. paucimobilis* UT26 and Sp+, respectively. *S. paucimobilis* ATCC 29837<sup>T</sup> and *S. chlorophenolica* DSM 7098<sup>T</sup> were provided by Manjit Hanspal, St. Elizabeth’s Medical Center of Boston, Boston, Mass., and John Cul-lum, University of Kaiserslautern, Kaiserslautern, Germany, respectively. We also thank R. Eichenlaub for his continuous encouragement in the initiation and continuation of this project.

Part of this work was supported by grants under the Indo-Swiss Collaboration in Biotechnology. C. Dogra and V. Raina acknowledge the Council of Scientific and Industrial Research for providing a doctoral fellowship.

# REFERENCES

1. Ahuja, A. K., and M. D. Awasthi. 1993. Contamination of rice and wheat grains with residues of HCH and DDT. Pestic. Res. J. **5**:83–86.
2. Akbar, N., A. Aleem, and A. Malik. 2003. Determination of organochlorine pesticides in agricultural soil with reference to γ-HCH degradation by *Pseudomonas* strains. Bioresource Technol. **88**:41–46.
3. Altschul, S. F., W. Gish, W. Miller, E. W. Myers, and D. J. Lipman. 1990. Basic local alignment search tool. J. Mol. Biol. **215**:403–410.
4. Bhuyan, S., B. Sreedharan, T. K. Adhya, and N. Sethunathan. 1993. Enhanced biological degradation of γ-hexachlorocyclohexane (γ-HCH) in HCH (commercial) acclimated flooded soil. Factors affecting its development and persistence. Pestic. Sci. **38**:49–53.
5. Boyd, D. A., G. A. Peters, L. Ng, and M. R. Mulvey. 2000. Partial characterization of a genomic island associated with the multidrug resistance region of *Salmonella enterica* typhymurium DT104. FEMS Microbiol. Lett. **189**:285–291.
6. Cerkvenik, V., D. Z. Doganoc, and J. Jan. 2000. Evidence of some trace elements, organochlorine pesticides and PCBs in Slovenian cow’s milk. Food Technol. Biotechnol. **38**:155–160.
7. Hall, R. M., H. J. Brown, D. E. Brookes, and H. W. Stokes. 1994. Integrons found in different locations have identical 5′ ends but variable 3′ ends. J. Bacteriol. **176**:6286–6294.
8. Higgins, D. G., J. D. Thompson, and T. J. Gibson. 1996. Using CLUSTAL for multiple sequence alignments. Methods Enzymol. **266**:383–402.
9. Imai, R., Y. Nagata, M. Fukuda, M. Takagi, and K. Yano. 1991. Molecular cloning of a *Pseudomonas paucimobilis* gene encoding a 17-kilodalton polypeptide that eliminates HCl molecules from γ-hexachlorocyclohexane. J. Bacteriol. **173**:6811–6819.
10. Johri, A. K., M. Dua, D. Tuteja, R. Saxena, D. M. Saxena, and R. Lal. 1996. Genetic manipulation of microorganism for the degradation of hexachlorocyclohexane. FEMS Microbiol. Rev. **19**:69–84.
11. Kannan K., S. Tanabe, J. P. Giesy, and R. Tatsukawa. 1997. Organochlorine pesticides and polychlorinated biphenyls in foodstuffs from Asian and Oceanic countries. Rev. Environ. Contam. Toxicol. **152**:1–55.
12. Kato, K., K. Ohtsuki, H. Mitsuda, T. Yomo, S. Negoro, and I. Urabe. 1994. Insertion sequence IS6100 on plasmid pOAD2, which degrades nylon oligomers. J. Bacteriol. **176**:1197–1200.
13. Kidwell, M. G., and D. R. Lisch. 2002. Transposable elements as sources of genomic variation, p. 59–90. In N. L Craig, R Craigie, M Gellert, and A. M Lambowitz (ed.), Mobile DNA II. American Society for Microbiology, Washington, D.C.
14. Kumari, R., S. Subudhi, M. Suar, G. Dhingra, V. Raina, C. Dogra, S. Lal, J. R. van der Meer, C. Holliger, and R. Lal. 2002. Cloning and characterization of *lin* genes responsible for the degradation of hexachlorocyclohexane isomers in *Sphingomonas paucimobilis* B90. Appl. Environ. Microbiol. **68**:6021–6028.
15. Lal, R., and D. M. Saxena. 1982. Accumulation, metabolism, and effects of organochlorine insecticides on microorganisms. Microbiol. Rev. **46**:95–127.
16. Martin, C., J. Timm, J. Rauzler, R. Gomez-Lus, J. Davies, and B. Gicquel.
17. Miyayuchi, K., H. S. Lee, M. Fukuda, M. Takagi, and Y. Nagata. 2002. Cloning and characterization of *linR*, involved in regulation of downstream pathway for γ-hexachlorocyclohexane degradation in *Sphingomonas paucimobilis* UT26. Appl. Environ. Microbiol. **68**:1803–1807.
18. Miyayuchi, K., S. Suh, Y. Nagata, and M. Takagi. 1998. Cloning and sequencing of a 2,5-dihydrochloroquinone reductive dehalogenase gene whose product is involved in the degradation of γ-hexachlorocyclohexane by *Sphingomonas paucimobilis*. J. Bacteriol. **180**:1354–1359.
19. Miyayuchi, K., Y. Adachi, Y. Nagata, and M. Takagi. 1999. Cloning and sequencing of a novel meta-cleavage dioxygenase gene whose product is involved in degradation of γ-hexachlorocyclohexane in *Sphingomonas paucimobilis*. J. Bacteriol. **181**:6712–6719.
20. Nagata, Y., K. Miyayuchi, and M. Takagi. 1999. Complete analysis of genes and enzymes for γ-hexachlorocyclohexane degradation in *Sphingomonas paucimobilis* UT26. J. Ind. Microbiol. Biotechnol. **23**:380–390.
21. Nagata, Y., K. Miyayuchi, S. Suh, A. Futamura, and M. Takagi. 1996. Isolation and characterization of Tn5-induced mutants of *Sphingomonas paucimobilis* defective in 2,5-dichlorohydroquinone degradation. Biosci. Biotechnol. Biochem. **60**:689–691.
22. Nagata, Y., R. Ohtomo, K. Miyayuchi, M. Fukuda, K. Yano, and M. Takagi. 1994. Cloning and sequencing of a 2,5-dichloro-2,5-cyclohexadiene-1,4-diol dehydrogenase gene involved in the degradation of γ-hexachlorocyclohexane in *Pseudomonas paucimobilis*. J. Bacteriol. **176**:3117–3125.
23. Nagata, Y., T. Nariya, R. Ohtomo, M. Fukuda, K. Yano, and M. Takagi. 1993. Cloning and sequencing of a dehalogenase gene encoding an enzyme with hydrolase activity involved in the degradation of γ-hexachlorocyclohexane in *Pseudomonas paucimobilis*. J. Bacteriol. **175**:6403–6410.
24. Nalin, R., P. Simonet, T. M. Vogel, and A. Normand. 1999. *Rhodanobacter lindaniclasticus* gen. nov., sp. nov., a lindane-degrading bacterium. Int. J. Syst. Bacteriol. **49**:19–23.
25. Noren, K., and D. Meironyte. 2000. Organochlorine and organobromine contaminants in Swedish human milk in perspective of past 20–30 years. Chemosphere **44**:1111–1123.
26. Pitman, A., P. Herron, and P. Dyson. 2002. Cointegrate resolution following transposition of Tn1792 in *Streptomyces avermitilis* facilitates analysis of transposon-tagged sequence. J. Microbial Methods **49**:89–96.
27. Preston, K. E., C. C. Radomski, and R. A. Venezia. 1999. The cassettes and 3′ conserved segment of an integron from *Klebsiella oxytoca* plasmid pACM1. Plasmid **42**:104–114.
28. Romine, M. F., L. C. Stillwell, K. K. Wong, S. J. Thurston, E. C. Sisk, C. Sensen, T. Gaasterland, J. K. Fredrickson, and J. D. Saffer. 1999. Complete sequence of a 184-kilobase catabolic plasmid from *Sphingomonas aromativorans* F199. J. Bacteriol. **181**:1585–1602.
29. Sahu, S. K., K. K. Patnaik, M. Sharmila, and N. Sethunathan. 1990. Degradation of alpha-, beta-, gamma-hexachlorocyclohexane by a soil bacterium under aerobic conditions. Appl. Environ. Microbiol. **56**:3620–3622.
30. Sambrook, J., E. F. Fritsch, and T. Maniatis. 1989. Molecular cloning: a laboratory manual, 2nd ed. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.
31. Sanghi, R., and V. Tewari. 2001. Monitoring of pesticide residues in summer fruits and vegetables from Kanpur, India. Bull. Environ. Contam. Toxicol. **67**:587–593.
32. Senoo, K., and H. Wada. 1989. Isolation and identification of an aerobic γ-HCH decomposing bacterium from soil. Soil Plant Nutr. **35**:79–87.
33. Simonich, S. L., and R. A. Hites. 1995. Global distribution of persistent organochlorine compounds. Science **269**:1851–1853.
34. Sorum, H., T. M. L’Abee-Lund, A. Solberg, and A. Wold. 2003. Integron-containing IncU R plasmids pRAS1 and pAr-32 from the fish pathogen *Aeromonas salmonicida*. Antimicrob. Agents Chemother. **47**:1285–1290.
35. Sundin, G. W., and C. L. Bender. 1995. Expression of the *strA-strB* streptomycin resistance genes in *Pseudomonas syringae* and *Xanthomonas campestris* and characterization of IS6100 in *X. campestris*. Appl. Environ. Microbiol. **61**:2891–2897.
36. Tauch, A., S. Gotker, A. Pühler, J. Kalinowski, and G. Thierbach. 2002. The 27.8-kb R-plasmid pTET3 from *Corynebacterium glutamicum* encodes the aminoglycoside adenyltransferase gene cassette *aadA9* and the regulated tetracycline efflux system Tet33 flanked by active copies of the widespread insertion sequence IS6100. Plasmid **48**:117–129.
37. Thomas, J. C., F. Berger, M. Jacquier, D. Bernikkon, F. Baud-Grasset, N. Truffaut, P. Normand, T. M. Vogel, and P. Simonet. 1996. Isolation and characterization of a novel γ-hexachlorocyclohexane-degrading bacterium. J. Bacteriol. **178**:6049–6055.
38. Wada, H., K. Senoo, and Y. Takai. 1989. Rapid degradation of γ-HCH in upland soil after multiple applications. Soil Sci. Plant Nutr. **35**:71–77.
