
ARTICLES

Molecular mechanisms of DrICE inhibition by DIAP1 and removal of inhibition by Reaper, Hid and Grim

Nieng Yan$^{1}$, Jia-Wei Wu$^{2}$, Jijie Chai$^{1}$, Wenyu Li$^{1}$ & Yigong Shi$^{1}$

The *Drosophila melanogaster* inhibitor of apoptosis protein DIAP1 suppresses apoptosis in part through inhibition of the effector caspase DrICE. The pro-death proteins Reaper, Hid and Grim (RHG) induce apoptosis by antagonizing DIAP1 function. However, the underlying molecular mechanisms remain unknown. Here we demonstrate that DIAP1 directly inhibits the catalytic activity of DrICE through its BIR1 domain and this inhibition is countered effectively by the RHG proteins. Inhibition of DrICE by DIAP1 occurs only after the cleavage of its N-terminal 20 amino acids and involves a conserved surface groove on BIR1. Crystal structures of BIR1 bound to the RHG peptides show that the RHG proteins use their N-terminal IAP-binding motifs to bind to the same surface groove, hence relieving DIAP1-mediated inhibition of DrICE. These studies define novel molecular mechanisms for the inhibition and activation of a representative *D.* *melanogaster* effector caspase.

Caspases are responsible for the execution of programmed cell death (apoptosis) in species ranging from worms to mammals${}^{1,2}$. There are two classes of caspases: the effector caspases, which are directly responsible for inflicting irreparable damage to cells, and the initiator caspases, which cleave and activate the effector caspases at the onset of apoptosis. In mammals, the effector caspases, caspase-3 and caspase-7, are activated by the initiator caspase, caspase-9 (Fig. 1a). In *D.* *melanogaster*, the functional orthologs of caspase-9 and caspase-3/-7 are Dronc and DrICE, respectively (Fig. 1a).

The inhibitor of apoptosis (IAP) family of proteins suppresses apoptosis by physically interacting with and inhibiting the catalytic activity of caspases${}^{1,3,4}$. In mammals, at least eight IAPs have been identified, including the representative X-linked IAP (XIAP). In *D.* *melanogaster*, DIAP1 and DIAP2 inhibit both normal developmental cell death and apoptosis induced by expression of proapoptotic genes${}^{5}$. The functional unit of IAP is the baculoviral IAP repeat (BIR), which contains ~80 amino acids folded around a zinc atom. Each IAP protein contains one to three copies of the BIR domain, with different BIR domains carrying out distinct functions. IAP-mediated inhibition of caspases is effectively countered by a family of proapoptotic proteins that share an IAP-binding tetrapeptide motif at their N termini${}^{6}$. The representative members of this family include Smac (also called DIABLO) in mammals and Reaper, Hid and Grim (RHG) in *D.* *melanogaster*${}^{6}$ (Fig. 1a).

In mammals, the molecular mechanisms of how IAPs inhibit caspases and how Smac removes this inhibition have been documented structurally and mechanistically (reviewed in ref. 1). The BIR3 domain of XIAP heterodimerizes with the monomeric caspase-9, sequestering it in an inactive conformation${}^{7}$. This inhibition requires an interaction between a conserved surface groove on BIR3 and an N-terminal IAP-binding tetrapeptide motif of the p12 subunit of caspase-9 (ref. 8). During apoptosis, Smac uses a similar tetrapeptide motif to compete with caspase-9 for binding to BIR3, hence removing XIAP-mediated inhibition of caspase-9 (refs. 9–11). In contrast, a conserved peptide segment preceding the BIR2 domain of XIAP interacts with the substrate-binding groove of caspase-3 or -7, blocking substrate entry and subsequent catalysis${}^{12-14}$. Smac is believed to remove this inhibition by hiding the XIAP peptide segment underneath the Smac dimeric scaffold${}^{12}$.

Although the apoptotic pathways are highly conserved between mammals and *D.* *melanogaster* (Fig. 1a), the molecular mechanisms for the inhibition and activation of caspases seem to be surprisingly different. In contrast to XIAP, DIAP1 does not inhibit the catalytic activity of Dronc${}^{15}$; rather, DIAP1 tags Dronc for ubiquitination and degradation${}^{15,16}$. This function requires specific interactions between a peptide fragment in Dronc and the same conserved surface groove of DIAP1-BIR2 as that required for binding to the N-terminal peptides of the RHG proteins${}^{15}$. This structural observation readily explains how the RHG proteins antagonize DIAP1-mediated negative regulation of Dronc${}^{15}$.

Despite progress in understanding the mechanisms of Dronc regulation, the molecular mechanisms of the regulation of DrICE remain largely unclear. DrICE is an important effector caspase in *D.* *melanogaster* cells and has an essential role in apoptosis${}^{17,18}$. In a manner analogous to caspase-3 and caspase-7, the DrICE zymogen can be processed by Dronc (caspase-9 ortholog) or by another caspase DCP-1 (refs. 19, 20). DrICE also cleaves Dronc in a positive feedback loop, as this cleavage renders Dronc refractory to negative regulation by DIAP1. DIAP1 can suppress DrICE-mediated cell death in *D.* *melanogaster* through direct physical interactions${}^{21}$, although the underlying mechanism remains unknown. The proapoptotic

---

${}^{1}$Department of Molecular Biology, Princeton University, Lewis Thomas Laboratory, Washington Road, Princeton, New Jersey 08544, USA. ${}^{2}$Department of Biology, Tsinghua University, Beijing 100084, China. Correspondence should be addressed to Y.S. (yshi@molbio.princeton.edu).

Published online 25 April 2004; doi:10.1038/nsmb764

ARTICLES

**RESULTS**

Inhibition of DrICE by DIAP1-BIR1

To investigate how DIAP1 regulates DrICE activity, we reconstituted an *in vitro* caspase cleavage assay, using DrICE as the active protease and a Dronc variant (C318A F118E) as the substrate. Then we examined the ability of various DIAP1 fragments to inhibit the catalytic activity of DrICE (Fig. 1b). The substrate Dronc C318A F118E does not interact with DIAP1 (ref. 15), thus eliminating a potential complication in the interpretation of results. The same conclusions to be reported were also confirmed using another substrate, a single-chain pro-DrICE (C211A).

The substrate Dronc C318A F118E was derived by coexpressing the large subunit (residues 1–352) with the small subunit (residues 353–450) (see Methods for details). Hence the substrate contains two fragments on SDS-PAGE (Fig. 1b, lane 1). DrICE readily converted the large subunit into two discrete bands (lane 2); analysis of the cleavage products by N-terminal peptide sequencing revealed that the cleavage occurred after Asp135 in Dronc, consistent with an earlier prediction²⁹. Incubation of this reaction with the full-length DIAP1 (residues 1–438, lane 3), the combined BIR1-BIR2 fragment (residues 1–324, lane 4), or the BIR1 domain alone (residues 1–140, lane 5) resulted in substantial inhibition of the DrICE activity, as shown by the diminished intensity of the cleavage products. In contrast, neither the BIR2 domain (residues 138–324, lane 6) nor the RING domain alone (residues 315–438, lane 7) exhibited any inhibition. The RING domain slightly modified the cleavage specificity of DrICE, as shown by two additional cleavage products (lane 7, indicated by asterisks). This cleavage occurred after Asp113 in the substrate. These observations demonstrate that the BIR1 domain of DIAP1 is both necessary and sufficient for the inhibition of DrICE activity.

Notably, at the same time that DIAP1(1–140) efficiently inhibited the catalytic activity of DrICE, the intensity of this fragment was substantially reduced, with a commensurate increase in the intensity for a band that overlaps the smaller fragment of the cleaved substrate (Fig. 1b, lane 5). This observation indicated a cleavage of DIAP1(1–140) itself. Analysis of the cleavage products by N-terminal peptide sequencing revealed that the cleavage occurred after Asp20 in DIAP1(1–140), following a canonical DrICE cleavage site, DQVD. Thus we also anticipated that the same cleavage would occur in other DIAP1 fragments. Indeed, peptide-sequencing analysis revealed that amino acids 1–20 were also efficiently removed in the full-length DIAP1 (Fig. 1b, lane 3, and Fig. 1c) as well as in the combined BIR1-BIR2 fragment (Fig. 1b, lane 4).

To further investigate this cleavage, we analyzed the status of the full-length DIAP1 in the presence of DrICE at varying time points. This result unambiguously confirms the cleavage of the full-length DIAP1 (residues 1–438) into a slightly shorter fragment (residues

RHG proteins induce cell death through inhibition of DIAP1 function²²,²³, and this activity depends on the physical interactions among the RHG proteins, DIAP1 and caspases²⁰,²²–²⁸. However, it is completely unclear how the RHG proteins remove DIAP1-mediated downregulation of DrICE. This study is aimed at addressing these mechanistic issues.

We report here that the BIR1 domain of DIAP1 is both necessary and sufficient for the inhibition of the catalytic activity of DrICE and that peptides derived from the N termini of the RHG proteins efficiently relieve DIAP1-mediated inhibition of DrICE. We identified the conserved surface groove of BIR1 as the binding site for both DrICE and the N-terminal sequences of the RHG proteins, thus explaining how the RHG peptides disrupt the DrICE-DIAP1 complex and remove DIAP1-mediated inhibition of DrICE. Finally, we report the crystal structures of DIAP1-BIR1 bound to the Reaper and Grim peptides; these structures reveal a molecular basis for the recognition of RHG peptides by BIR1. Together, our studies define the molecular mechanisms by which the antiapoptotic protein DIAP1 inhibits DrICE and by which the RHG proteins remove this inhibition.

21–438) (Fig. 1c). To further examine the inhibition of DrICE by DIAP1, a two-fold serial dilution of the BIR1 domain (residues 21–140) was incubated with DrICE and its substrate (Fig. 1d). Analysis of this result as well as that obtained using the fluorescent substrate DEVD-AFC (data not shown) revealed that the BIR1 domain inhibits the catalytic activity of DrICE with an IC₅₀ of ~0.25 μM. The strength of this inhibition is similar to that exhibited by XIAP for caspase-9 (ref. 30).

Previous studies have identified the BIR2 domain and the RING domain of DIAP1 as essential for the recognition and ubiquitination of Dronc, respectively. Our current results identify the BIR1 domain of DIAP1 as the direct inhibitor of DrICE. This represents the first regulatory function assigned to the BIR1 domain of DIAP1. Notably, Dronc was found to efficiently cleave DIAP1 after Glu205 in our *in vitro* experiments, after a Val-Gln-Pro-Glu205 sequence (Fig. 1e), causing the full-length DIAP1 protein to split into two fragments, the BIR1 domain and the combined BIR2-RING domain. This cleavage has been shown to occur naturally in *D. melanogaster* cells (R. Clem, Kansas State University, personal communication). Thus, DIAP1 contains two separable functional entities, the BIR1 domain, which inhibits DrICE, and the BIR2-RING domain, which recognizes and ubiquitinates Dronc.

### Identification of BIR1 surface for DrICE inhibition

DrICE interacts with DIAP1 (ref. 21). We reasoned that this interaction is essential for DIAP1-mediated inhibition of DrICE. To understand this mechanism, we sought to identify the sequence element of DIAP1-BIR1 that is required for DrICE interaction and inhibition. The BIR domain in all IAP proteins shares the same conserved fold. Based on sequence alignment and available structural information on BIR2 (ref. 32), we mutated several residues in BIR1 that are expected to be on the surface and examined whether these BIR1 mutants were still able to interact with or inhibit DrICE. Because mutations were introduced only to solvent-exposed residues, all mutant proteins were found to be well folded as judged by gel filtration and dynamic light scattering analysis (data not shown).

These experiments revealed a strict correlation between interaction with DrICE and ability to inhibit DrICE for the BIR1 domain. Two double mutants, K77D K79D and D94K E99K, did not inhibit the catalytic activity of DrICE (Fig. 2a, lanes 4 and 5). Accordingly, neither of these two mutants could interact with DrICE as judged by a gel filtration analysis (Fig. 2b). In contrast, wild-type BIR1 and the control mutant P105S retained their ability to interact with and inhibit DrICE (Fig. 2a,b). Notably, Lys77, Lys79, Asp94 and Glu99 are near one another on the surface of the predicted BIR1 structure, and the corresponding residues on the BIR2 domain map to the conserved peptide-binding pocket (Fig. 2c). Thus, the same predicted surface pocket on the BIR1 domain is involved in binding to and inhibiting the catalytic activity of DrICE.

Figure 2 The conserved pocket of DIAP1-BIR1 is involved in DrICE interaction and inhibition.  
(a) Identification of mutations on BIR1 that result in loss of DrICE inhibition. Various BIR1 proteins were examined for their ability to inhibit DrICE activity. The double mutants K77D K79D and D94K E99K did not inhibit DrICE activity. (b) Inhibition of DrICE correlates with its interaction with DIAP1-BIR1. In this gel filtration assay, various BIR1 proteins were individually incubated with the active DrICE protein. The mixture was applied to a gel filtration analysis from which contiguous fractions were visualized on SDS-PAGE by Coomassie blue staining. (c) The structure of DIAP1-BIR2 (ref. 32) showing the predicted surface location of the mutated residues in BIR1. The residues whose corresponding mutation in BIR1 resulted in loss of interaction with DrICE are red. The corresponding BIR1 residues are indicated in parentheses. The conserved surface pocket among most BIR domains is indicated by a purple oval circle. This panel was generated using MolScript.

We also sought to define the sequence element in DrICE that is required for binding to DIAP1-BIR1. Although DrICE is an effector caspase, it contains an unusually long prodomain preceding the caspase unit (residues 81–339). Our gel filtration analyses indicate that BIR1 does not interact with the DrICE prodomain (residues 1–80) (data not shown). Similarly to caspase-3 and caspase-7, both the zymogen and the processed DrICE exist as constitutive homodimers in solution. Notably, BIR1 binds only to the processed DrICE and not to the unprocessed DrICE zymogen, as judged by gel filtration analyses (data not shown). This observation is reminiscent of the interaction between XIAP-BIR3 and caspase-9 (ref. 7), suggesting the possibility that the conserved surface pocket in DIAP1-BIR1 may interact with the N-terminal peptide of the small subunit of DrICE after its activation cleavage at Asp230. To examine this possibility, we generated two DrICE variants that share the same large subunit (residues 81–230) but have different small subunits. One variant had residues 231–235 deleted whereas the other had a G231M substitution. Both variants were catalytically active; DIAP1-BIR1 inhibited both through direct physical interaction (Supplementary Fig. 1 online). Hence DIAP1-BIR1-mediated inhibition of DrICE is qualitatively different from other documented modes of caspase inhibition.

### Removal of DrICE inhibition by the RHG peptides

The RHG proteins use their N termini to compete with Dronc for binding to the same conserved surface pocket on the BIR2 domain of DIAP1, thus removing DIAP1-mediated negative regulation of

spectroscopy revealed a stable complex between BIR1 and this Hid peptide. Quantitative measurement by isothermal titration calorimetry (ITC) revealed a binding affinity of 0.76 μM between DIAP1-BIR1 and the Hid peptide (Fig. 3b). Subsequently, we also verified that an N-terminal ten-residue peptide derived from Reaper (AVAFYIPDQA) or Grim (AIAYFIPDQA) also interacts strongly with the BIR1 domain of DIAP1 (Fig. 3b), presumably through binding to the conserved hydrophobic surface pocket.

Next, we investigated whether, through binding to this surface pocket, the Hid peptide can relieve DIAP1-mediated inhibition of DrICE (Fig. 3c). The Hid peptide alone did not have any effect on the cleavage of substrate by DrICE (Fig. 3c, lanes 2 and 3). However, this peptide removed the inhibition of DrICE by either the full-length DIAP1 (lanes 4 and 5) or the BIR1 domain (lanes 6 and 7), as shown by the substantially increased cleavage of the substrate. The same results were obtained using a peptide derived from Grim (Fig. 3d) or Reaper (data not shown). Quantitative analysis revealed that, at a concentration of 1 μM, Grim, Reaper and Hid peptides removed 90%, 84% and 70% of DrICE inhibition mediated by 0.5 μM of DIAP1-BIR1, respectively (data not shown). Notably, a ten-residue peptide derived from Sickle or Smac did not remove DIAP1-mediated inhibition (Fig. 3d). Comparison of sequences among these five peptides suggests that this is likely due to the sequence divergence beyond the fifth residue (Fig. 3d). These observations demonstrate that the N-terminal sequences of the RHG proteins remove DIAP1-mediated inhibition of DrICE through competitive binding to the conserved pocket on the BIR1 domain.

We further examined the interaction between DIAP1(1–145) and DrICE using gel filtration. The isolated DIAP1(1–145) and DrICE were eluted at volumes consistent with their apparent molecular masses (Fig. 3e). Then we incubated DrICE with this DIAP1 fragment. After cleavage, the DIAP1(21–145) fragment and DrICE coeluted in earlier fractions than either of the isolated fragments, indicating the formation of a stable complex (Fig. 3e). Next, we analyzed this DIAP1-DrICE complex after its incubation with a stoichiometric amount of the Hid peptide. In this case, a substantial portion of the DIAP1-DrICE complex was disrupted, as shown by the decreased intensity of DIAP1 in the complex fractions (Fig. 3e). Thus the removal of DrICE inhibition by the RHG proteins is probably caused by the disruption of the DIAP1-DrICE interaction.

Developmental gain-of-function mutations

Previous studies have identified several notable gain-of-function (GOF) missense mutations in the DIAP1-BIR1 domain²²,²⁴. These mutations can suppress apoptosis induced by the RHG proteins²²,²⁴.

Two of these mutations, G88S and G88D, were mapped to the BIR1 domain (ref. 22). Because DrICE is a major effector caspase in *D. melanogaster* apoptosis and the BIR1 domain is responsible for the inhibition of DrICE, we reasoned that these GOF mutations can no longer be regulated by the RHG proteins but can still inhibit DrICE.

To verify this mechanistic explanation, we generated two corresponding mutations in the BIR1 domain, G88S and G88D, and examined whether the mutant proteins were still able to interact with or inhibit DrICE. In complete agreement with our prediction, both mutant proteins still strongly inhibited the catalytic activity of DrICE (Fig. 4a) and formed a stable complex with DrICE (Fig. 4b). In contrast to wild-type BIR1 (Fig. 4a, lanes 4 and 5), DrICE inhibition by these two mutant proteins can no longer be removed by the presence of the Hid peptide (lanes 6–9). Additional experiments indicated that Hid can no longer weaken or disrupt the interaction between DrICE and these mutant BIR1 proteins (Fig. 4b). Notably, the control mutation P105S in DIAP1 retained its ability to inhibit DrICE and this inhibition was only partially relieved by the Hid peptide (Fig. 4a, lanes 10 and 11). This observation is consistent with the weak GOF phenotype for this mutation²².

### Recognition of the RHG peptides by BIR1

To reveal the structural determinants for the recognition of the RHG peptides and to explain our biochemical data, we crystallized the DIAP1-BIR1 domain (residues 30–135) bound to a ten-residue peptide derived from the N terminus of Reaper or Grim. Both structures were determined using molecular replacement and have been refined to a resolution of 1.8 Å (Table 1 and Fig. 5a).

The structure of the BIR1-Reaper complex is nearly identical to that of the BIR1-Grim complex, with an r.m.s. deviation of 0.23 Å for 103 aligned Cα atoms. However, the structure of DIAP1-BIR1 is appreciably different from that of the BIR2 domain³², with the largest variation at their C-terminal fragments (Fig. 5b). The C-terminal sequences in BIR2 constitute three short α-helices and fold back to contribute to the formation of the peptide-binding groove³² (Fig. 5b). In contrast, the corresponding sequences in BIR1 form an extended surface loop and a short α-helix, and are located on the side opposite to the peptide-binding groove (Fig. 5).

Although the peptide-binding groove is conserved, minor differences in sequence between BIR1 and BIR2 determine the distinct modes of peptide recognition. Notably, compared with the BIR2-Grim complex, the aromatic side chain of Tyr4 in Grim and Reaper is flipped nearly 180° (Fig. 5c). This is because Gly269 in BIR2, which interacts with Tyr4 through van der Waals contacts, is replaced by a bulkier residue Glu86 in BIR1. Thus Tyr4 of Grim can no longer place its side chain in the same location in BIR1 as in BIR2. On the other hand, the bulky Met271 in BIR2 is replaced by Gly88 in BIR1, thus allowing the side chain of Tyr4 of Grim to rotate into the empty space vacated by this change (Fig. 5c).

The structural information provides a molecular explanation of previous biochemical and developmental observations. Based on the structures, replacement of Gly88 by serine or aspartate in BIR1 will lead to a steric clash with the side chain of Tyr4 or Phe4 of the RHG proteins (Fig. 5c), thus crippling the interaction between BIR1G88S or G88D and the RHG peptide (data not shown). Consequently, the Hid peptide did not relieve BIR1-mediated inhibition of DrICE or disrupt the BIR1-DrICE interactions (Fig. 4). Because the BIR1 domain of DIAP1 is both necessary and sufficient for inhibiting DrICE, these observations provide a plausible molecular mechanism for the phenotype of the GOF mutations G88S and G88D in DIAP1. In addition, Ile6 and Pro7 of the Reaper and Grim peptides closely pack against

---

**Figure 4** A mechanistic explanation for developmental GOF mutations in the BIR1 domain. (a) The GOF mutations retain DrICE inhibition and can no longer be regulated by the Hid peptide. Wild-type BIR1 inhibited DrICE (lane 4), and this inhibition was removed by the presence of the Hid peptide (lane 5). In contrast, the mutant proteins G88S and G88D inhibited DrICE (lanes 6 and 8), but this inhibition was not removed by Hid (lanes 7 and 9). (b) Interaction between DrICE and the mutant BIR1 proteins can no longer be weakened or disrupted by the Hid peptide. The developmental GOF mutations in the BIR1 domain (G88S and G88D) retained a stable interaction with DrICE as shown by gel filtration. The presence of Hid peptide did not affect this interaction.

surrounding residues in BIR1 through multiple van der Waals interactions (Fig. 5c). This may explain why Smac and Sickle peptides, which contain charged residues at the sixth and seventh positions, did not remove BIR1-mediated inhibition of DrICE (Fig. 3d).

### Autoinhibition of the BIR1 domain

The apparent discrepancy between efficient cleavage of DIAP1 by DrICE (Fig. 1c) and potent inhibition of DrICE by DIAP1 (Figs. 1b, d) led us to hypothesize that the cleavage after Asp20 of DIAP1 may be important for its inhibition of DrICE activity. If this hypothesis were correct, then blocking the removal of the N-terminal 20 amino acids should make the resulting DIAP1 a poorer inhibitor. To investigate this possibility, we generated a D20A mutation in the DIAP1-BIR1 domain and compared its activity with that of wild-type BIR1. Supporting the hypothesis, DIAP1(1–145) D20A exhibited substantially less inhibition of DrICE as compared with DIAP1(1–140) (Fig. 6a, lanes 4 and 5)

Figure 5 Structural basis of recognition of the RHG peptides by DIAP1-BIR1. (a) An overall view of the structure of DIAP1-BIR1 (green) bound to the Reaper peptide (pink). The zinc-chelating residues are highlighted in yellow. (b) Structural comparison of the BIR1–Grim and BIR2–Grim complexes. The BIR1 and BIR2 domains are green and cyan, respectively. Note the large difference at their C termini. (c) Comparison of the peptide-binding mode in the BIR1–Grim and BIR2–Grim complexes. Note the difference in side chain conformation for Tyr4 of the Grim peptide. This difference is caused by the replacement of Gly269 and Met271 in BIR2 by Glu86 and Gly88, respectively, in BIR1.

or DIAP1(1–145) (Fig. 6b, lanes 1 and 3). This result suggests that the N-terminal 20 amino acids serve as an autoinhibitory sequence for the function of DIAP1. In this case, removal of these 20 amino acids is predicted to facilitate the function of DIAP1. To examine this scenario, we truncated the N-terminal 20 amino acids in the DIAP1-BIR1 domain. Indeed, DIAP1(21–145) exhibited substantially greater inhibition of DrICE than did DIAP1(1–145) D20A (Fig. 6a, lanes 5 and 8; Fig. 6b, lanes 3 and 5).

The potent inhibition of DrICE exerted by DIAP1(21–145) was efficiently removed by the Hid peptide (Fig. 6a, lanes 8 and 9; Fig. 6b, lanes 5 and 6), because the Hid peptide can bind to the conserved surface groove on BIR1 that is also required for the inhibition of DrICE. However, because DIAP1(1–145) D20A already exists in an autoinhibitory state, Hid exhibited no effect (Fig. 6b, lanes 3 and 4). These results strongly suggest that the N-terminal residues of DIAP1 also bind to the same surface groove of the BIR1 domain. The extreme N-terminal sequence (ASVVA) of DIAP1 represents a strong candidate. If these residues are involved in autoinhibition of DIAP1 function, then their removal or modification should improve the ability of the resulting BIR1 domain to inhibit DrICE. Supporting this hypothesis, the DIAP1 fragment (residues 7–145, D20A) exhibited a substantially greater level of inhibition than did DIAP1(1–145) D20A (Fig. 6a, lanes 5 and 6; Fig. 6b, lanes 3 and 7). Furthermore, the mutation of the second alanine residue to methionine in DIAP1(1–145) D20A A2M also led to a greater level of inhibition than did DIAP1(1–145) D20A (Fig. 6a, lanes 5 and 7; Fig. 6b, lanes 3 and 9). This result also identifies a critical role for the alanine immediately following the initiation methionine in DIAP1, consistent with the conservation of the binding pocket for alanine in the BIR1 domain (Fig. 5).

Finally, if DIAP1 exists in an autoinhibited state, it should not form a stable complex with DrICE. In agreement with this prediction, DIAP1(1–145) D20A, which did not substantially inhibit DrICE, did not form a stable complex with DrICE as shown by gel filtration analysis (Fig. 6c). In contrast, the cleaved DIAP1(21–145), which represents the most potent inhibitor of DrICE, formed a stable complex with DrICE (Fig. 6c). Results obtained for these and other DIAP1 variants demonstrate that the observed autoinhibition strictly correlates with loss of physical interactions between DrICE and DIAP1.

DISCUSSION

Our study indicates that DIAP1 is multifunctional³³. Following Dronc-mediated cleavage after Glu205, the BIR1 domain exists in an autoinhibited state, unable to block the catalytic activity of DrICE (Fig. 6d). However, cleavage of the BIR1 domain after Asp20 removes this autoinhibition and allows the resulting fragment to potently inhibit DrICE. Although this cleavage can be caused by DrICE as shown in this study, it can also be mediated by other *D. melanogaster* caspases such as DCP-1 during apoptosis. Notably, the RHG proteins use their N-terminal amino acids to compete with DrICE for binding to the BIR1 domain, hence removing the inhibition of DrICE by DIAP1. Our structural and biochemical data indicate that both Hid and DrICE bind to the same hydrophobic surface pocket on the BIR1 domain.

In a recent report, cleavage after Asp20 has been shown to accelerate the degradation of the full-length DIAP1 protein through the N-end rule³⁴, because the N-terminal amino acid (Asn21) following cleavage converts DIAP1 into a highly unstable state. This mechanism has been proposed to facilitate apoptosis because it leads to decreased levels of DIAP1 in cells. However, during apoptosis, the initiator caspase Dronc is probably activated before the effector caspase DrICE. Once active, Dronc efficiently cleaves DIAP1 after Glu205, separating DIAP1 into two fragments. Notably, this cleavage occurs constitutively in *D. melanogaster* cells (R. Clem, Kansas State University, personal communication). In this case, the N-end rule will lead only to the removal of the BIR1 domain but not the BIR2-RING fragment. Thus, it is unclear to what extent the N-end rule-mediated degradation of DIAP1 can regulate its function. Alternatively, as DrICE forms a stable complex with the cleaved DIAP1, the proposed N-end rule could also facilitate the degradation of DrICE.

During the preparation of this manuscript, it was reported that Reaper and Grim preferentially bind to the BIR1 domain of DIAP1 whereas Hid prefers the BIR2 domain³⁵. Our quantitative measurements in this study (Fig. 3b) and in a previous report³² support this conclusion. Our current structural analysis reveals the molecular basis for this observation. The methyl side chain of Ala3 in Reaper and Grim closely stacks against the aromatic side chain of Trp103 in BIR1 (Fig. 5c). Replacement of Ala3 by Pro3 in Hid is likely to generate a small degree of steric clashing with Trp103, thus decreasing the binding affinity.

ARTICLES

a
![Diagram](#)

b
![Diagram](#)

c
Fractions from gel filtration
![Gel Filtration Diagram](#)

d
Model of autoinhibited BIR1
![Structural Model](#)

Figure 6 The autoinhibition of DIAP1 resides in its N-terminal 20 residues. (a,b) Removal of the N-terminal residues of BIR1 improves its inhibition for DrICE. Compared with DIAP1(1–145) D20A (a, lane 5; b, lane 3), DIAP1(21–145) (a, lane 8; b, lane 5) or DIAP1(7–145) D20A (a, lane 6; b, lane 7) exhibited a higher level of inhibition of DrICE activity. (c) Autoinhibited DIAP1 does not form a stable complex with DrICE. The fragment of residues 21–145 but not that of residues 1–145 formed a stable complex with DrICE as shown by gel filtration (top panel). The uncleavable DIAP1(1–145) D20A exists in an autoinhibited state and can no longer interact with DrICE (second panel). (d) A structural model of how the N-terminal sequences (orange) autoinhibits BIR1 (green) by binding to the conserved surface groove. The N-terminal sequences of BIR1, ASVVADL, are likely to fold back to bind to the conserved surface groove on BIR1.

In summary, our studies identify the BIR1 domain of DIAP1 as necessary and sufficient for the inhibition of the catalytic activity of DrICE. The inhibition of DrICE by DIAP1 requires their mutual interaction, which involves a conserved surface groove on the BIR1 domain. Unexpectedly, DIAP1 does not inhibit DrICE until the removal of its N-terminal 20 amino acids, which serve as an autoinhibitory sequence for the function of DIAP1. These results define novel mechanisms for the inhibition of DrICE by DIAP1 and reactivation of DrICE by the RHG proteins. During apoptosis, the N-terminal sequences of the proapoptosis proteins Reaper, Hid and Grim bind to the same surface pocket, hence removing the inhibition. Although DrICE represents the functional ortholog of mammalian caspase-3 and caspase-7, the mechanism of its negative regulation mediated by DIAP1 is completely different from the mammalian paradigm.

METHODS

Protein and peptide preparation. All constructs were generated using a standard PCR-based cloning strategy, and the identities of individual clones were verified through double-stranded plasmid sequencing. All proteins were overexpressed in *Escherichia coli* strain BL21(DE3) either as GST-fusion proteins using pGEX-2T (Pharmacia), or C-terminal His₆-tagged proteins using pET21b (Novagen). Proteins were purified to homogeneity as described. The ten-residue Reaper, Grim, Hid, Sickle and Dronc peptides were chemically synthesized, purified by reverse-phase HPLC, lyophilized and resuspended in buffer. The two-chain Dronc substrate (C318AF118E) was obtained by coexpressing Dronc(1–352) C318AF118E in pBB75 with Dronc(353–450) in pET21b and purified as described. The 1–352 fragment contains two cleavage sites (Asp-Glu-Ser-Asp113 and Asp-Ile-Val-Asp135) for DrICE. In assays throughout this study, all recombinant proteins were purified to >98% homogeneity. To ensure a uniform standard, all SDS-PAGE gels shown in this study were visualized after staining by Coomassie blue.

In vitro DrICE assay. Reactions were carried out at 22°C in an assay buffer (25 mM HEPES, pH 8.0, 150 mM NaCl, and 2 mM DTT) for 60 min. Substrate (~20 μg two-chain Dronc C318A) and active DrICE (100 ng) were used for each reaction. Various DIAP1 fragments (1–5 μg) were used in the reaction to examine possible inhibition of DrICE activity. For removal of inhibition, 2–5 μg of the Hid peptide was used. Peptides derived from Reaper, Hid and Sickle were tested in the same manner.

DIAP1 cleavage by Dronc. Reactions were carried out at 22°C in an assay buffer (25 mM HEPES, pH 8.0, 150 mM NaCl and 2 mM DTT). Substrate (~20 μg DIAP1) and active Dronc (100 ng) were used for each reaction.

Crystallization and data collection. DIAP1-BIR1(30–135) was mixed with the synthetic Reaper or Grim peptide in a 1:1.1 molar ratio. The final concentration of the complex was ~20 mg ml⁻¹. Crystals of the BIR1-Grim complexes were grown by hanging-drop vapor diffusion by mixing the complex with an equal volume of reservoir solution containing 100 mM sodium citrate, pH 5.6, 0.3 M (NH₄)₂SO₄, and 0.5 M Li₂SO₄. Multiple cubic crystals appeared after 4 days at 22°C. BIR1-Reaper crystals were obtained from 100 mM CHES, pH 9.5, and 1.0 M sodium citrate. A single crystal appeared after 7 days at 22°C. Crystals were equilibrated in a cryoprotectant buffer containing well buffer plus 25% (v/v).

Table 1 Data collection and refinement statistics

| Data collection | BIR1-Reaper | BIR1-Grim |
|-----------------|-------------|-----------|
| Complex        |             |           |
| Space group    | R3          | R3        |
| Resolution (Å) | 99.0–1.78   | 99.0–1.75 |
| Total observations | 45,025     | 41,712    |
| Unique observations | 12,620     | 13,052    |
| Data coverage (%)$^a$ | 99.0 (92.1) | 97.7 (98.6) |
| $R_{\text{sym}}$$^{a,b}$ | 0.059 (0.646) | 0.064 (0.246) |

| Refinement | BIR1-Reaper | BIR1-Grim |
|------------|-------------|-----------|
| Resolution range (Å) | 20.0–1.8 | 20.0–1.8 |
| Number of reflections ($I > 0$) | 12,293 | 12,094 |
| Data coverage (%) | 99.6 | 97.8 |
| $R_{\text{work}}$$^c$ | 0.232 | 0.212 |
| $R_{\text{free}}$$^c$ | 0.267 | 0.250 |
| Number of atoms | 1,015 | 1,029 |
| Number of waters | 154 | 172 |
| R.m.s. deviation from ideal values | | |
| Bond length (Å) | 0.006 | 0.007 |
| Bond angle (°) | 1.351 | 1.437 |
| Ramachandran plot | | |
| Most favored (%) | 90.9 | 90.9 |
| Additionally allowed (%) | 9.1 | 9.1 |
| Generously allowed (%) | 0.0 | 0.0 |
| Disallowed (%) | 0.0 | 0.0 |

$a$ Values in parentheses are for the highest-resolution shell. $b$ $R_{\text{sym}} = \Sigma_h \Sigma_i |I_{h,i} - I_h| / \Sigma_h \Sigma_i I_{h,i}$ where $I_h$ is the mean intensity of the $i$ observations of symmetry-related reflections $h$. $c$ $R_{\text{work}} = \Sigma |F_o - F_c| / \Sigma F_o$, where $F_o = F_p$, and $F_c$ is the calculated protein structure factor from the atomic model ($R_{\text{free}}$ was calculated with 5% of the reflections). The r.m.s. deviation in $B$-factors is calculated between bonded atoms.

ACKNOWLEDGMENTS  
We thank L. Walsh at CHESS for help with beam time and T.K. Vanderlick and M. Apel-Paz for help with ITC. This work was supported by US National Institutes of Health grants.

COMPETING INTERESTS STATEMENT  
The authors declare that they have no competing financial interests.

Received 24 December 2003; accepted 23 March 2004  
Published online at http://www.nature.com/natstructmolbiol/

1. Shi, Y. Mechanisms of caspase inhibition and activation during apoptosis. *Mol. Cell* **9**, 459–470 (2002).
2. Thornberry, N.A. & Lazebnik, Y. Caspases: Enemies within. *Science* **281**, 1312–1316 (1998).
3. Deveraux, Q.L. & Reed, J.C. IAP family proteins—suppressors of apoptosis. *Genes Dev.* **13**, 239–252 (1999).
4. Salvesen, G.S. & Duckett, C.S. IAP proteins: blocking the road to death's door. *Nat. Rev. Mol. Cell Biol.* **3**, 401–410 (2002).
5. Hay, B.A., Wassarman, D.A. & Rubin, G.M. *D. melanogaster* homologs of baculoviral inhibitor of apoptosis proteins function to block cell death. *Cell* **83**, 1253–1262 (1995).
6. Shi, Y. A conserved tetrapeptide motif: Potentiating apoptosis through IAP-binding. *Cell Death Differ.* **9**, 93–95 (2002).
7. Shiozaki, E.N. *et al.* Mechanism of XIAP-mediated inhibition of caspase-9. *Mol. Cell* **11**, 519–527 (2003).
8. Srinivasula, S.M. *et al.* A conserved XIAP-interaction motif in caspase-9 and Smac/DIAPO mediates opposing effects on caspase activity and apoptosis. *Nature* **409**, 112–116 (2001).
9. Chai, J. *et al.* Structural and biochemical basis of apoptotic activation by Smac/DIA-BLO. *Nature* **406**, 855–862 (2000).
10. Wu, G. *et al.* Structural basis of IAP recognition by Smac/DIAPO. *Nature* **408**, 1008–1012 (2000).
11. Liu, Z. *et al.* Structural basis for binding of Smac/DIAPO to the XIAP BIR3 domain. *Nature* **408**, 1004–1008 (2000).
12. Chai, J. *et al.* Structural basis of caspase-7 inhibition by XIAP. *Cell* **104**, 769–780 (2001).
13. Huang, Y. *et al.* Structural basis of caspase inhibition by XIAP: differential roles of the linker versus the BIR domain. *Cell* **104**, 781–790 (2001).
14. Riedl, S.J. *et al.* Structural basis for the inhibition of caspase-3 by XIAP. *Cell* **104**, 791–800 (2001).
15. Chai, J. *et al.* Molecular mechanism of Reaper/Grim/Hid-mediated suppression of DIAP1-dependent Dronc ubiquitination. *Nat. Struct. Biol.* **10**, 892–898 (2003).
16. Wilson, R. *et al.* The DIAP1 RING finger mediates ubiquitination of Dronc and is indispensable for regulating apoptosis. *Nat. Cell Biol.* **4**, 445–450 (2002).
17. Fraser, A.G. & Evan, G.I. Identification of a *D. melanogaster* ICE/CED-3-related protease, drICE. *EMBO J.* **16**, 2805–2813 (1997).
18. Fraser, A.G., McCarthy, N.J. & Evan, G.I. drICE is an essential caspase required for apoptotic activity in *D. melanogaster* cells. *EMBO J.* **16**, 6192–6199 (1997).
19. Hawkins, C.J. *et al.* The *D. melanogaster* caspase DRONC cleaves following glutamate or aspartate and is regulated by DIAP1, HID, and GRIM. *J. Biol. Chem.* **275**, 27084–27093 (2000).
20. Meier, P., Silke, J., Leevers, S.J. & Evan, G.I. The *D. melanogaster* caspase DRONC is regulated by DIAP1. *EMBO J.* **19**, 598–611 (2000).
21. Kaiser, W., Vucic, D. & Miller, L. The *D. melanogaster* inhibitor of apoptosis DIAP1 suppresses cell death induced by the caspase drICE. *FEBS Lett.* **440**, 243–248 (1998).
22. Goyal, L., McCall, K., Agapite, J., Hartwig, E. & Steller, H. Induction of apoptosis by *D. melanogaster* reaper, hid and grim through inhibition of IAP function. *EMBO J.* **19**, 589–597 (2000).
23. Wang, S., Hawkins, C., Yoo, S., Muller, H.-A. & Hay, B. The *D. melanogaster* caspase inhibitor DIAP1 is essential for cell survival and is negatively regulated by HID. *Cell* **98**, 453–463 (1999).
24. Lisi, S., Mazzon, I. & White, K. Diverse domains of THREAD/DIAP1 are required to inhibit apoptosis induced by REAPER and HID in *D. melanogaster*. *Genetics* **154**, 669–678 (2000).
25. Vucic, D., Kaiser, W.J. & Miller, L.K. Inhibitor of apoptosis proteins physically interact with and block apoptosis induced by *D. melanogaster* proteins HID and GRIM. *Mol. Cell. Biol.* **18**, 3300–3309 (1998).
26. Song, Z. *et al.* Biochemical and genetic interactions between *D. melanogaster* caspases and the proapoptotic genes rpr, hid, and grim. *Mol. Cell Biol.* **20**, 2907–2914 (2000).
27. Hawkins, C., Wang, S. & Hay, B.A. A cloning method to identify caspases and their regulators in yeast: identification of *D. melanogaster* IAP1 as an inhibitor of the *D. melanogaster* caspase DCP-1. *Proc. Natl. Acad. Sci. USA* **96**, 2885–2890 (1999).
28. Vucic, D., Kaiser, W.J., Harvey, A.J. & Miller, L.K. Inhibition of reaper-induced apoptosis by interaction with inhibitor of apoptosis proteins (IAPs). *Proc. Natl. Acad. Sci. USA* **94**, 10183–10188 (1997).
29. Dorstyn, L., Colussi, P.A., Quinn, L.M., Richardson, H. & Kumar, S. DRONC, an ecdysone-inducible *D. melanogaster* caspase. *Proc. Natl. Acad. Sci. USA* **96**, 4307–4312 (1999).

glycerol, and were flash frozen in a –170 °C nitrogen stream. The native data were collected at the CHESS beamline A1. Data were processed using DENZO and SCALEPACK$^{37}$. The crystals belong to the space group $R3$ and have unit cell dimensions $a = 84.8$ Å, $b = 84.8$ Å, and $c = 49.8$ Å.

Structure determination. The structures were determined by molecular replacement using AMoRe$^{38}$. The atomic coordinates of the BIR2-Grim complex were used as the initial search model. The atomic model was built using O$^{39}$ and refined using CNS$^{40}$. The electron density for the bound peptide fragments became unambiguous after preliminary refinement. The final refined atomic model contains residues 39–135 of DIAP1 and residues 1–7 of Reaper or Grim at a resolution of 1.8 Å. The N-terminal eight residues in DIAP1-BIR1 have no electron density in the maps, and we presume that these regions are disordered in the crystals.

Gel filtration assay. Individual recombinant proteins were incubated in assay buffer (25 mM Tris, pH 8.0, 150 mM NaCl and 2 mM DTT). An aliquot of the protein mixture (0.5 ml) was subjected to gel filtration analysis (Superdex 200, Amersham Pharmacia) for each run. Samples taken from relevant fractions were visualized by SDS-PAGE with Coomassie blue staining.

Isothermal titration calorimetry. All proteins and peptides were prepared in 25 mM HEPES buffer, pH 7.5, and 125 mM NaCl. The Micro Calorimetry System (Microcal) was used to carry out the ITC measurements for the interaction between DIAP1-BIR1 and various synthetic peptides. The titration data, collected at 20 °C, were analyzed using ORIGIN (Microcal Software).

Coordinates. The atomic coordinates of the BIR1-Reaper and BIR1-Grim complexes have been deposited in the Protein Data Bank (accession codes 1SDZ and 1SE0, respectively).

Note: Supplementary information is available on the Nature Structural & Molecular Biology website.

ARTICLES

30. Sun, C. *et al.* NMR structure and mutagenesis of the third BIR domain of the inhibitor of apoptosis protein XIAP. *J. Biol. Chem.* **275**, 33777–33781 (2000).

31. Fesik, S.W. Insights into programmed cell death through structural biology. *Cell* **103**, 273–282 (2000).

32. Wu, J.-W., Cocina, A.E., Chai, J., Hay, B.A. & Shi, Y. Structural analysis of a functional DIAP1 fragment bound to Grim and Hid peptides. *Mol. Cell* **8**, 95–104 (2001).

33. Vernooy, S.Y. *et al.* Cell death regulation in *D. melanogaster*: conservation of mechanism and unique insights. *J. Cell Biol.* **150**, F69–F76 (2000).

34. Ditzel, M. *et al.* Degradation of DIAP1 by the N-end rule pathway is essential for regulating apoptosis. *Nat. Cell Biol.* **5**, 467–473 (2003).

35. Zachariou, A. *et al.* IAP-antagonists exhibit non-redundant modes of action through differential DIAP1 binding. *EMBO J.* **22**, 6642–6652 (2003).

36. Chai, J. *et al.* Crystal Structure of a procaspase-7 zymogen: mechanisms of activation and substrate binding. *Cell* **107**, 399–407 (2001).

37. Otwinowski, Z. & Minor, W. Processing of X-ray diffraction data collected in oscillation mode. *Methods Enzymol.* **276**, 307–326 (1997).

38. Navaza, J. AMoRe and automated package for molecular replacement. *Acta Crystallogr. A* **50**, 157–163 (1994).

39. Jones, T.A., Zou, J.-Y., Cowan, S.W. & Kjeldgaard, M. Improved methods for building protein models in electron density maps and the location of errors in these models. *Acta Crystallogr. A* **47**, 110–119 (1991).

40. Terwilliger, T.C. & Berendzen, J. Correlated phasing of multiple isomorphous replacement data. *Acta Crystallogr. D* **52**, 749–757 (1996).

41. Kraulis, P.J. MOLSCRIPT: a program to produce both detailed and schematic plots of protein structures. *J. Appl. Crystallogr.* **24**, 946–950 (1991).
