
Molecular Microbiology (2004) 52(3), 873–893

doi:10.1111/j.1365-2958.2004.04026.x

# Characterization of a complex chemosensory signal transduction system which controls twitching motility in *Pseudomonas aeruginosa*

Cynthia B. Whitchurch,$^{1,4}$ Andrew J. Leech,$^{1,2}$ Michael D. Young,$^{1,2}$ Derek Kennedy,$^{1,3}$ Jennifer L. Sargent,$^{1,2}$ Jacob J. Bertrand,$^{6}$ Annalese B. T. Semmler,$^{1,2}$ Albert S. Mellick,$^{1,2}$ Paul R. Martin,$^{1}$ Richard A. Alm,$^{1}$ Matthew Hobbs,$^{1}$ Scott A. Beatson,$^{1,2}$ Bixing Huang,$^{1}$ Lam Nguyen,$^{4}$ James C. Commolli,$^{4}$ Joanne N. Engel,$^{4,5}$ Aldis Darzins,$^{7}$ and John S. Mattick$^{1,2*}$

$^{1}$ARC Special Research Centre for Functional and Applied Genomics, Institute for Molecular Bioscience, University of Queensland, Brisbane, QLD 4072, Australia.

$^{2}$School of Molecular and Microbial Sciences, University of Queensland, Brisbane, QLD 4072, Australia.

$^{3}$School of Biomolecular and Biomedical Sciences, Griffith University, Nathan QLD 4111, Australia.

$^{4}$Department of Medicine, University of California, San Francisco, California, 94143-0654, USA.

$^{5}$Microbiology and Immunology, University of California, San Francisco, California, 94143-0654, USA.

$^{6}$Biochemistry and Biophysics, University of California, San Francisco, California, 94143-0654, USA.

$^{7}$Promega Corporation, 2800 Woods Hollow Road, Madison, WI 53711-5399, USA.

## Summary

Virulence of the opportunistic pathogen *Pseudomonas aeruginosa* involves the coordinate expression of a wide range of virulence factors including type IV pili which are required for colonization of host tissues and are associated with a form of surface translocation termed twitching motility. Twitching motility in *P. aeruginosa* is controlled by a complex signal transduction pathway which shares many modules in common with chemosensory systems controlling flagella rotation in bacteria and which is composed, in part, of the previously described proteins PilG, PilH, PilI, PilJ and PilK. Here we describe another three components of this pathway: ChpA, ChpB and ChpC, as well as two downstream genes, ChpD and ChpE, which may also be involved. The central component of the pathway, ChpA, possesses nine potential sites of phosphorylation: six histidine-containing phosphotransfer (HPt) domains, two novel serine- and threonine-containing phosphotransfer (SPt, TPt) domains and a CheY-like receiver domain at its C-terminus, and as such represents one of the most complex signalling proteins yet described in nature. We show that the Chp chemosensory system controls twitching motility and type IV pili biogenesis through control of pili assembly and/or retraction as well as expression of the pilin subunit gene *pilA*. The Chp system is also required for full virulence in a mouse model of acute pneumonia.

## Introduction

*Pseudomonas aeruginosa* is a ubiquitous Gram-negative bacterium found throughout the environment, and is also an opportunistic pathogen of a wide variety of eukaryotes including mice, insects, nematodes, and plants (*Jander et al.*, 2000; *Rahme et al.*, 2000; Tan and Ausubel, 2000). It also causes serious and often life-threatening infections in immunocompromised humans such as those who are suffering from severe burns, cystic fibrosis or AIDS, who are undergoing cancer chemotherapy, or who recovering from major surgery (Giamarellou, 2000). The most critical threat to the health and survival of individuals with cystic fibrosis is the persistent injury to the lungs and airways caused by chronic infection with *P. aeruginosa*. *Pseudomonas aeruginosa* is equipped with a large arsenal of secreted and cell-associated virulence factors, including adhesins, exopolysaccharides, proteases, lipases, phospholipases, siderophores, biosurfactants and exotoxins, which provide nutrients for bacterial growth, enhance invasive potential, and/or directly damage host tissue.

The major adhesins of *P. aeruginosa* which facilitate attachment of the bacterium to host epithelial cells are type IV pili, which are filamentous structures located at the poles of the bacterial cell. Type IV pili are retractile and mediate a mode of surface translocation termed twitching motility. They also act as receptors for certain bacteriophages (for recent review see Mattick, 2002). Twitching motility occurs in a wide variety of pathogenic bacteria, including *Neisseria gonorrhoea*, and is mediated by pili extension and retraction (*Merz et al.*, 2000; Skerker

Accepted 9 January, 2004. *For correspondence.* E-mail j.mattick@imb.uq.edu.au; Tel. (+61) 7 3346 2110; Fax (+61) 7 3346 2111.

© 2004 Blackwell Publishing Ltd

and Berg, 2001; Mattick, 2002). Twitching motility, at least in *P. aeruginosa*, is also involved in biofilm development (O’Toole and Kolter, 1998; Chiang and Burrows, 2003; Klausen et al., 2003a,b).

Pseudomonas aeruginosa has become in recent years one of the main model organisms for the characterization of the molecular genetics and biology of type IV pili and twitching motility. Thus far more than 35 genes have been identified to be involved in the biogenesis, function and regulation of type IV pili in *P. aeruginosa* (Alm and Mattick, 1997; Mattick, 2002). Many of these have homology to other gene/protein sets involved in protein secretion and DNA uptake in various bacteria (Whitchurch et al., 1991; Hobbs and Mattick, 1993; Mattick and Alm, 1995). The biogenesis and function of type IV pili in *P. aeruginosa* is controlled by multiple signal transduction systems, including the two-component sensor regulator systems *pilSR* (Hobbs et al., 1993; Strom and Lory, 1993) and *algR/fimS* (Whitchurch et al., 1996), as well as the global carbon metabolism regulator Crc (O’Toole et al., 2000), and the virulence factor regulator Vfr (Beatson et al., 2002).

Twitching motility in *P. aeruginosa* is also controlled by a chemosensory system which is encoded in part by the previously characterized gene cluster *pilG, pilH, pill, pilJ* and *pilK* (Darzins, 1993; 1994; 1995). In this study we have identified another three members of this system (*chpA, chpB, chpC*) and another two genes (*chpD, chpE*) which might also be involved. This gene cluster is located immediately downstream of *pilK*. Together the proteins encoded by these genes appear to comprise a complex chemosensory signal transduction pathway which shares many modules in common with chemosensory systems which control flagella rotation in bacteria.

The most extensively studied of these chemosensory systems are those that control swimming motility in the enteric species *Escherichia coli* and *Salmonella enterica* serovar *Typhimurium*. The biochemistry of signalling reactions in these systems are well characterized and atomic resolution structures for most of the domains and components of these systems have been solved (for reviews see Falke et al., 1997; Djordjevic and Stock, 1998; Armitage, 1999; Bourret and Stock, 2002). The enteric chemosensory systems are composed of membrane bound chemoreceptors referred to as methyl accepting chemotaxis proteins (MCPs) which are coupled via the adaptor protein CheW to the multidomain histidine protein kinase CheA. The MCPs function to modulate the kinase activity of CheA in response to chemical stimuli. CheA possesses in its C-terminus conserved motifs required for nucleotide binding. Autophosphorylation occurs at a conserved histidine in the N-terminus of the protein in a domain termed P1. In recent years it has become evident that the P1 domain of CheA is structurally related to the histidine containing phosphotransfer (HPt) domains found in inter-

mediates of multistep phosphorelay reactions such as the C-terminal HPt domain of ArcB (Kato et al., 1997) and the yeast protein YpD1 (Song et al., 1999; Xu and West, 1999). The high-energy phosphoryl group is then transferred from the histidine of the CheA P1/HPt domain to a conserved aspartate in the response regulator CheY, which then interacts directly with the flagellar motor to switch the direction of flagellar rotation. The rate of autocatalytic dephosphorylation of CheY-P is enhanced through interaction with CheZ. Sensory adaptation which allows temporal control of swimming motility occurs through methylation of specific glutamate residues on the MCP to reset it into a non-signalling state. The methylation status of the MCP is adjusted via competing activities of the methyl transferase CheR and the methyl esterase CheB. CheB also possesses a response regulator module and is competitively phosphorylated by CheA (Falke et al., 1997; Djordjevic and Stock, 1998; Armitage, 1999; Bourret and Stock, 2002).

Over recent years more complex chemosensory systems have been described in other bacteria and have been found to control, in addition to swimming, other bacterial motilities including swarming, gliding and twitching (Burkart et al., 1998; Armitage, 1999; Bourret and Stock, 2002; Mattick, 2002; Winther-Larsen and Koomey, 2002). In fact the *Pseudomonas aeruginosa* PAO1 genome has four gene clusters encoding chemotaxis-like phosphorelay signal transduction systems (Croft et al., 2000; Stover et al., 2000). Two of these systems are involved in chemotactic control of swimming motility (Masduki et al., 1995; Fernandez et al., 2002), the Wsp gene cluster controls surface properties that lead to autoaggregation (D’Argenio et al., 2002) and the *pilG-K, chpA-C* gene cluster is involved in the control of twitching motility (Darzins, 1993; 1994; 1995) which is the focus of this study.


### Results

#### Sequence analysis of the region downstream of *pilG-K*

We have previously generated a library of twitching motility mutants through transposon (Tn5-B21) mutagenesis of the *P. aeruginosa* strain K genome (Hobbs et al., 1993). Southern blot and sequence analysis of the points of transposon insertions in this library identified four mutants (S43, S115, S154 and S247) which contain transposon insertions in the previously characterized gene *pilJ* (Fig. 1A). We also identified another nine mutants (S10, S23, S33, S45, S52, S129, S164, S209 and S349) which have transposon insertions in the region immediately downstream of *pilK* (Fig. 1A).

The 12 kb region immediately downstream of *pilK* was cloned from the PAO1 cosmid pMO01539 and sequenced using a combination of Exoll deletions, subcloning and

© 2004 Blackwell Publishing Ltd, *Molecular Microbiology*, **52**, 873–893

Chemotaxis control of twitching motility in P. aeruginosa 875

A

CheY CheY CheW MCP CheR Sacl CheA/CheY Sacl

G H I pilJ K chpA chpB C D E

1 Kb S43 S115 S154 S247 S52 S164 S209 S128 S10 S349 S35 S33 NruI NotI PstI ClaI KpnI XhoI Apal Apal

PAKchpB::TcR PAKchpC::TcR PAKchpD::TcR PAKchpE::TcR

B

His54 Thr481 His912 His1088 Ser1231 His1410 His1570 His1717 NG1FG2 Asp2406

HPt1 FimL-like TPt HPt2 HPt3 SPt HPt4 HPt5 HPt6 HisKc-like CheW CheY

C

CheA P1/HPt (1-112) MNGVAMGRHD YVALEWVKGEITALKQARQALEAFVENPQDPTRMRFCLTYVHQVQGG
ChpA HPt1 (1-118) NSEESRLPTTDVAQIHQLVKKEARNGLEQAIAIEFIASQWNHEHLARVPELLTQVRGG
ChpA TPt (422-546) LPPPADDEEPVDDELREVFIEEAGEVLETIGRYLPAWKAODUREALTEVRRAFHTLKGS
ChpA HPt2 (854-976) AEAAQSDVELLDPEQLLEIFINEAEETHLEALVGFLADCARELPOPVTDALQRALHTLKGS
ChpA HPt3 (1031-1152) ERRYSDAGERRDP-TLIEAFLVEGMDTLLDAEDLLERWHHEHPQERQELSALREELSTLDRG
ChpA SPt (1173-1295) APAPAPGRELDED-EMVSIFLEEAVDILETSSAGQALAQWQAEPGAALSSLALQRDLLHTLKGG
ChpA HPt4 (1352-1474) ARMAEAIFEERDDPELVIEEGFDILDSSAAAALQRMDDVDNTIELEALQRDLLHTLKGG
ChpA HPt5 (1512-1634) ARMAEIGEIGDLAHELLEYGLCGGRLRASP-ALLFGLLQRCHDELAEMLEAVRGHRITLP
ChpA HPt6 (1658-1781) KPLAAKGAAADESEILDFLEEAADDLEENGGNQDAQPLDDLLCRILLHTLKGG

CheA P1/HPt (1-112) AGTFGFSLCQETTHLMENLLDEARRRGEMLQLNT-DQINCLFLETKDIIMQEQLDAYKQSQEYPD
ChpA HPt1 (1-118) LQMVEFYGAALTAEEMEQVLQAQLLDGKVVPNQG-EALEVMQAIILQLPVYLDRTIQMARRDL
ChpA TPt (422-546) LAAMIPLERAATLTETCNRYIQEQQLLAIRKAVPDWQSLLDTLADAITSVEYYLERLSEDHASQ
ChpA HPt2 (854-976) GRMVRAVLVGEAWSNEFRVLDRSIAASE-PVQRVVDQVALPLEVEEFAANAQRRQ
ChpA HPt3 (1031-1152) AHMAELPIAEITPLEKLVKEYKSNLIAFDL-REAEELLHDAEQLFRIGLEQVGAQRPPLN
ChpA SPt (1173-1295) ARHAELPQVEELCQAALLAYDAVEEGLAVSP-AFFEEARQAHEALIGMMDBQVAAAGLQVT
ChpA HPt4 (1352-1474) ARMAEIADLGDLAHLELEYGLVDRRYQHSP-QLAGLLQACHDRLAEOQLDQLSAGQPLA
ChpA HPt5 (1512-1634) ARMAEIGDLAHELLEYGLCGGRLRASP-ALLFGLLQRCHDELAEMLEAVRGHRITLP
ChpA HPt6 (1658-1781) ARLAGQTETLGNAHDLEEQHTDAQQQGAPPWPDSLLLDAAQSQLEGLQRQVDBLLRERRLAED

Fig. 1. A. Organization of the *pilGHIJK* and *chpABCDE* gene cluster. Arrows represent the direction of transcription of the genes. The location of transposon insertions are indicated by filled triangles. The sites of insertion of the TcR cassette for generation of allelic exchange mutants are indicated by open triangles/boxes. Relevant restriction enzyme sites are also shown. The class of proteins to which each gene product belongs is shown above the boxes.

B. Schematic of the ChpA protein. Relevant domains and features of the ChpA protein are indicated. 'HPt', 'TPt' and 'SPt' indicate the predicted histidine, threonine or serine containing phosphotransfer domains, respectively; 'FimL-like' denotes the region of ChpA which shares homology with FimL and encompasses both HPt1 and TPt; 'HisKc-like' denotes the autocatalytic histidine kinase domain which incorporates the CheY docking domain (P2) and the N, G1, F, and G2 boxes which are required for ATP binding; 'CheW' indicates the CheW-like domain which is involved in mediating CheA interactions with CheW; and 'CheY' denotes the CheY-like response regulator domain.

C. Alignment of the putative phosphotransfer domains of ChpA with the P1/HPt domain of *E. coli* CheA. Similar residues are boxed, identical residues are indicated in bold and are shaded. *denotes the position of the predicted site of phosphorylation in each domain. The alignment was generated using the CLUSTALW feature in MacVector (Oxford Molecular Group).

primer walking (GenBank Accession Number U79580). Analysis of the sequence of this region identified five open reading frames (ORFs) which we have designated *chpA*, *chpB*, *chpC*, *chpD* and *chpE* (Fig. 1A). These ORFs are also referred to as PA0413 – PA0417 in the completed *P. aeruginosa* genome sequence (http://www.pseudomonas.com; Stover et al., 2000). The predicted gene products of these ORFs were character-

ized using a combination of BLAST searches of GenBank (http://www.ncbi.nlm.nih.gov/BLAST/; Altschul et al., 1997) with PFAM (http://www.sanger.ac.uk/Software/Pfam/; Bateman et al., 2002) and SMART (http://smart.embl-heidelberg.de/; Schultz et al., 1998; Letunic et al., 2002) analyses. Together with the upstream genes *pilG*, *pilH*, *pilI*, *pilJ* and *pilK* the predicted gene products of *chpA*, *chpB*, *chpC*, *chpD* and *chpE* appear to constitute a com-

© 2004 Blackwell Publishing Ltd, *Molecular Microbiology*, 52, 873–893

876 C. B. Whitchurch et al.

plex signal transduction pathway analogous to the bacterial chemotaxis systems which control flagella rotation in response to chemical attractants and repellents. In light of this we have designated this putative chemosensory pathway the Chp (chemosensory pili) system.

The predicted ATG start codon of the first ORF of this region (*chpA*, PA0413) overlaps the TGA stop codon of *pilK*. This 7431 bp ORF is predicted to encode a 2477 aa (269 Kda) multidomain protein which belongs to the family of CheA-like histidine kinases. The central portion of ChpA resembles other CheA-like proteins and contains a conserved CheY docking domain (P2); an autocatalytic histidine kinase domain including the N, G1, F, and G2 boxes which are required for ATP binding; and a CheW-like domain which is involved in mediating CheA interactions with CheW (Falke *et al.*, 1997) (Fig. 1B). At the C-terminus of ChpA is located a response regulator module. ChpA then is a hybrid of homologues of the histidine kinase CheA and the response regulator CheY. According to the SMART database (http://smart.embl-heidelberg.de/; Schultz *et al.*, 1998; Letunic *et al.*, 2002) there have been more than 20 CheA/CheY hybrid proteins reported to date.

The N-terminal two-thirds of ChpA contains eight predicted phosphotransfer domains, six of which are conventional histidine-containing HPt (or P1) domains and two others which have threonine and serine in place of histidine at their active site (Figs 1B, C). BLAST searches of GenBank and SMART analyses indicate that most CheA-like histidine kinases possess only one HPt domain in their N-termini. However BLAST and SMART analyses indicate that each of the following bacteria encode a CheA-like histidine kinase that possesses more than one HPt domain: Synechocystis PCC6803 (NP_442714; 2 HPt), Nitrosomonas europaea ATCC 19718 (NP_841302.1; 3 HPt), Xylella fastidiosa strains Ann-1 (ZP_00041851.1; 4 HPt), Dixon (ZP_00039906.1; 4 HPt), 9a5c (NP_299234.1; 4 HPt), Temeula 1 (NP_779067.1; 4 HPt), Ralstonia metallidurans (ZP_00021550.1; 4 HPt), Ralstonia solanacearum (NP_518793.1; 4 HPt), Xanthomonas campestris pv. campestris str. ATCC 33913 (NP_638269.1; 4 HPt), Xanthomonas axonopodis pv. citri str. 306 (NP_643407.1; 4 HPt), *P. putida* KT2440 (NP_747090.1; 4 HPt), *P. syringae* pv. tomato str DC3000 (NP_794763.1; 5 HPt), *P. syringae* pv. syringae B728a (ZP_00125157.1; 5 HPt) *P. fluorescens* PfO-1 (ZP_00085475.1; 5 HPt) and Microbulbifer degredans 2–40 (ZP_00067212.1; 5HPt). Each of these multi-HPt containing CheA-like histidine kinases also possesses a CheY domain at its C-terminus proteins. In fact these proteins appear to be *bona fide* homologues of the *P. aeruginosa* ChpA (see Discussion).

In addition to the six HPt domains, *P. aeruginosa* ChpA also possesses another two domains which are homologous to HPt domains, except that they contain a threonine

or a serine residue (T476, S1226) in place of the normal histidine at the phosphorylation site (Fig. 1B and C). Three-dimensional homology modelling suggests that these predictions are correct (see below). We have termed the potential threonine-containing phosphotransfer domain and the potential serine-containing phosphotransfer domain TPt and SPt domains respectively. Potential Tpt/SPt domains are also found in the ChpA homologues of *N. europaea*, *X. fastidiosa*, *M. degredans*, *P. putida*, *P. fluorescens* and *P. syringae*. Over the past few years it has become evident that serine/threonine protein kinases are widespread in bacteria (Kennelly, 2002), but ChpA and its homologues are the first examples of putative serine/threonine phosphorylation in the context where a bacterial protein belonging to the class of CheA-like protein kinases appears to have the potential to be phosphorylated at serine/threonine residues. Whether phosphorylation at these residues occurs by autophosphorylation or by phosphorylation by a serine/threonine kinase encoded elsewhere in the genome remains to be determined. In total then ChpA possesses nine potential sites of phosphorylation, including six histidines, a threonine and a serine in conserved HPt/TPt/SPt domains in the N-terminal two-thirds of the protein, and the conserved aspartate of the CheY domain at the C-terminus, thus making it the most complex member of this class of CheA-like histidine kinases reported and quite possibly the most complex signal transduction protein yet described in nature.

The N-terminal 562 aa of ChpA which encompass the first HPt and TPt domains are also homologous (33% identity, 51% similarity) to FimL, a protein encoded by PA1822 that we have identified to be required for normal twitching motility in *P. aeruginosa* (C. B. Whitchurch, S. A. Beatson, T. Jakobsen, J. C. Commolli, J. L. Sargent, J. J. Bertrand, P. J. Kang, J. S. Mattick and J. N. Engel, manuscript in preparation). Interestingly, whereas FimL is homologous to the N-terminal region of ChpA, it does not contain either of the predicted phosphorylatable histidine or threonine residues although the surrounding sequences remain highly conserved.

We had originally concluded that this ‘FimL-like’ region was encoded by a separate gene, which we had designated *pilL* (Alm and Mattick, 1997). However, our original sequence submitted to GenBank contained a frameshift error. To further confirm the size of ChpA we performed Western blot analysis of FLAG epitope-tagged ChpA expressed in *P. aeruginosa*. No bands were detected in Western blots of whole cell samples of *P. aeruginosa* transformants containing the vector (pUCPSK) alone or the plasmid containing the *chpA* coding sequence lacking the FLAG-tag (pUCPChpA) whereas a single band at ~280 kDa was detected in cells expressing FLAG-tagged ChpA (pUCPChpA-FLAG) (Fig. 2). A single unique band

© 2004 Blackwell Publishing Ltd, *Molecular Microbiology*, **52**, 873–893

Chemotaxis control of twitching motility in P. aeruginosa 877

dicted to encode MCPs which are scattered around the genome (Croft *et al.*, 2000).

Located eight residues downstream of the stop codon of *chpC* is the predicted ATG start codon of the fourth ORF (PA0416) of the region which has been termed *chpD*, as it appears likely to be part of the *chp* cluster given the close spacing between it and *chpC*. This ORF is predicted to encode a 264 aa transcriptional regulator protein belonging to the AraC family (Gallegos *et al.*, 1997). This class of regulator is not a member of the two-component transmitter-receiver phosphorelay systems but instead binds effector molecules via the N-terminal and central regions of the protein and activates transcription of target genes through a conserved helix-turn-helix motif located in the C-terminus of the protein. The effector molecule bound by ChpD and the target genes of this regulator are unknown.

The final ORF of this region (PA0417), which we have tentatively termed *chpE*, is predicted to start 78 residues downstream of *chpD*. *chpE* encodes a 203 aa protein which is predicted to have a signal sequence and six transmembrane domains and appears therefore to be a small integral inner membrane protein. Database analyses predict that ChpE is a member of the LysE family of amino acid efflux proteins (Aleshin *et al.*, 1999; Vrljic *et al.*, 1999). Closer analysis of the ChpE primary sequence indicates that it is a member of the RhtB subfamily (Aleshin *et al.*, 1999). RhtB is an *E. coli* protein which is responsible for the efflux of homoserine and homoserine lactone (Zakataeva *et al.*, 1999). The solute transported by *P. aeruginosa* ChpE is unknown. Located about 100 nucleotides past the stop codon of *chpE* is a potential rho-independent transcriptional terminator.


Fig. 2. Determination of ChpA molecular weight. *P. aeruginosa* carrying the cloning vector pUCPSK (lane 1); pUCPchpA expressing untagged ChpA (lane 2); or pUCPChpA-FLAG expressing FLAG-tagged ChpA (lane 3). The left panel is a Coomassie stained 4–20% SDS-PAGE gradient gel of whole cell samples. The right panel is an anti-FLAG M2 immunoblot of whole cell samples. These results are typical of triplicate experiments. Identical results were obtained with both PAK and PAO1 transformants.

of the same molecular weight was observed with transformants of both pUCPChpA and pUCPChpA-FLAG (but not pUCPSK) when SDS-PAGE gels of whole cell samples were Coomassie stained (Fig. 2). The predicted molecular weight for the shorter version of ChpA (which lacks the ‘FimL-like’ region) is only 178 kDa whereas the observed size for the ChpA (~280 kDa) is in broad agreement with the predicted value of 269 kDa for the gene product of the longer *chpA* coding sequence.

The predicted ATG start codon of the second ORF (PA0414) in this region, which we have termed *chpB*, overlaps the final seven residues of *chpA*. *chpB* encodes a 343 aa protein homologous to the CheB methyl-esterases which in the chemotaxis systems function to demethylate the sensory methyl accepting chemotaxis proteins (MCPs). We predict by analogy therefore that ChpB serves to demethylate (at least) the predicted MCP PilJ as part of a sensory adaptation feedback circuit. The TGA stop codon of *chpB* overlaps the predicted ATG start codon of the third ORF (PA0415) of this region which has been designated *chpC* and which encodes a 168 aa protein predicted by SMART analysis (http://smart.embl-heidelberg.de/ Schultz *et al.*, 1998; Letunic *et al.*, 2002) to be a CheW-like protein. The Chp system therefore has two CheW homologues (PilI and ChpC) which we predict by analogy to the well characterized Che systems of other bacteria to function to couple PilJ and perhaps other MCPs to ChpA. *P. aeruginosa* contains 26 genes pre-

© 2004 Blackwell Publishing Ltd, *Molecular Microbiology*, **52**, 873–893


3D-structural modelling of putative phototransfer domains of ChpA

As described above, we have identified eight motifs in ChpA that show significant primary sequence homology to the histidine containing phototransfer (P1/Hpt) domains found in CheA-like histidine kinases. Comparison of the solved structures of the histidine containing phototransfer domains of CheA-like histidine kinases (Zhou and Dahlquist, 1997; Mourey *et al.*, 2001) with the structures of HPt domains found in intermediates of multistep phosphorelay reactions such as the C-terminal HPt domain of ArcB (Kato *et al.*, 1997) and the yeast protein Ypd1 (Song *et al.*, 1999; Xu and West, 1999) has demonstrated that they each contain a conserved four-helix bundle core which is required for phototransfer (Fig. 3) (Song *et al.*, 1999; Xu and West, 1999; Mourey *et al.*, 2001). We used 3D-structural modelling to examine the likely three-dimensional structures of the predicted phototransfer domains Hpt1, Hpt4 and Spt of ChpA (Fig. 3;

878 C. B. Whitchurch et al.

data not shown), as a means of obtaining additional evidence that these domains are capable of functioning as phosphotransfer domains. The comparison of the solved structures of ArcB, CheA and YPD1, and the modelled

structures of these putative phosphotransfer domains of ChpA predict that the phosphotransfer domains of ChpA also have a common core structure consisting of an anti-parallel four-helix core (data not shown; see Fig. 3 for the

**A**

| ArcB (Hpt) | 654 TTEENSKSEA LLDIPMLEQY LELVGPKLIT DGLAVFEKMM PGYVSVLESN | 703 |
| --- | --- | --- |
| ChpA (Spt) | 1174 SLEA ERYSDAGERR DPLLIEAFLV EGMDILLDAE DLLERWHEHP | 1217 |
| Ypd1 (Hpt) | 1 MSTIPSEIIN WTILNEIISM DDDDSDFS KG LIIQFIDQAQ TTFAQMQRQL | 50 |
| CheA (P1) | 1 MSMD ISDFYQTFFD EADELLADME QHLLVLQPEA | 34 |

| ArcB (Hpt) | 704 LTAQDKKGIV EEGIKIKGAA GSVGLRHLQQ LGQQIQSPDL PAWEDNVGEW | 753 |
| --- | --- | --- |
| ChpA (Spt) | 1218 QERQELSALR EEISTLDRGA RHAELPQVEE LCQALLALYD AVE-EGRLAV | 1266 |
| Ypd1 (Hpt) | 51 DGEKNLTELD NLGHFLKGSS AALGLQRIAW VCERIQNLGR KMQHFFPNKT | 100 |
| CheA (P1) | 35 PDAEQLNAIF RAAHSIKGGA GTFGFSVLQE TTHLMENLLD EAR-RGEMQL | 83 |

| ArcB (Hpt) | 754 IEEMKEEWRH DVEVLKAWVA KATKK | 778 |
| --- | --- | --- |
| ChpA (Spt) | 1267 SPAFFEEARQ AHEALIGMMD QVAAGLQVTP RPERVAALQE LLEAPAAEAVP | 1317 |
| Ypd1 (Hpt) | 101 ELVNTLSDKS IINGINIDED DEEIKIQVDD KDENSIYLIL IAKALNQSRL E | 150 |
| CheA (P1) | 84 NTDIINLFL E TKDIMQEQLD AYKQSQEPDA ASFDYICQAL RQLALEAKGET | 134 |

**B**
predicted structure of the ChpA SPt domain). In ArcB and CheA, these core structures are further stabilized by a fifth helix at the N-terminal and C-terminal regions of these domains, respectively (Kato *et al.*, 1997; Mourey *et al.*, 2001). Our modelling suggests that the ChpA phosphotransfer domains share the same antiparallel four-helix core but the position of a fifth helix cannot be accurately predicted. These proteins may use more complex structures to stabilize the overall fold as is observed in YPD1 (Song *et al.*, 1999; Xu and West, 1999).

Of particular interest were the predicted Tpt/Spt phosphotransfer domains in which the active histidine is replaced with a threonine or serine (Figs 1 and 3). Closer comparison of the phosphotransfer domains of either ArcB or CheA and the putative serine phosphotransfer (SPt) domain of ChpA show that the environment surrounding Ser1231 contains compensatory amino acid substitutions which allow Ser1231 access to interactive donor proteins (Fig. 3B). The positioning of Arg1235 on the border of a flexible loop proximal to Ser1231 is compatible with a model in which Arg1235 is free to be displaced during interactions with donor molecule. Upon phosphorylation of Ser1231 we propose that Arg1235 would rotate back and neutralize the charged group on Ser1231. This would be consistent with previous reports that suggest positively charged residues such as arginine may neutralize the negative charge of phosphoryl groups (Kato *et al.*, 1997). Potential conformational motions as suggested here would be consistent with data reported from other phosphorylation events (Cox *et al.*, 1994) as well as having a role in stabilizing the helical structures as has been observed in the HPr protein of *Bacillus subtilis* (Pullen *et al.*, 1995). Other compensatory changes include the replacement of Glu70 in CheA by Leu1253 in Spt of ChpA. This substitution compensates for the change in the active phosphorylation site of His48 in CheA, to Ser1231 in ChpA. These substitutions ensure that the interactive side chain of Ser1231 is orientated so that it may be exposed to the solvent interface and allow it to be phosphorylated. Arg1235 of ChpA is shown in Fig. 3B in an orientation consistent with the proposed

activated state of Ser1231. In this conformation the charged side chain of Arg1235 would neutralize that of a phosphorylated serine at position 1231.

These analyses predict that at least the HPt1, HPt4 and the SPt domains of ChpA have three-dimensional structures which are consistent with a role in phosphotransfer. Given the high degree of sequence conservation amongst the ChpA phosphotransfer domains we predict that each would have conserved three-dimensional structures. We are currently working toward experimentally demonstrating that each of the eight predicted HPt/TPt and SPt domains of ChpA are capable of functioning as phosphotransfer domains. This *in silico* study also predicts that the substitution of serine for histidine in the SPt domain of ChpA is accompanied by compensatory changes in the surrounding residues.

### Phenotypic characterization of PAK pilJ, chpA-E mutants

Sequence analysis of the *chp* region revealed that our collection of PAK Tn5-B21 mutants contained 4 insertions in *pilJ* (S43, S115, S154 and S247) and 9 insertions in *chpA* (S10, S23, S45, S33, S52, S129, S164, S209 and S349) (Fig. 1). Because we had not identified transposon insertion mutants of *chpB*, *chpC*, *chpD* and *chpE* we generated allelic exchange mutants of these genes in *P. aeruginosa* strain PAK (Fig. 1) to assess their phenotypes.

We assayed the *pilJ*, and *chpA-E* mutants for sensitivity to the type IV pilus-specific bacteriophage PO4 (Bradley, 1973), as described previously (Whitchurch *et al.*, 1991). All mutants demonstrated wild-type titres to this phage (data not shown). The ability of mutants of *chpA*, *chpB*, *chpC*, *chpD* and *chpE* to swim through 0.3% agar was then tested to determine if any of these components are also involved in the control of flagella-mediated swimming motility. Consistent with observations previously reported for mutants of *pilG-K*, the mutants of *chpA-E* also demonstrate wild-type swimming motility via this assay (data not shown). These observations indicate that the Chp chemosensory pathway is not intersecting with the swimming chemotaxis system of *P. aeruginosa*.

---

Fig. 3. Homology modelling of the phosphotransfer domains of ChpA.

A. Sequence alignments of phosphotransfer domains within ArcB, Ypd1 and CheA used for structural modelling. The phosphorylated histidine of CheA, ArcB and YPD1 and the putative phosphorylated serine in SPt of ChpA are boxed. These alignments were used to assign secondary structures to the ChpA homology model. Alpha helices (*α*-A to *α*-E) derived from the ArcB structure are shown as solid lines above the alignment whereas alpha helices derived from the CheA structure (*α*-A′ to *α*-E′) are shown below the alignment. The boxed amino acids show the conserved position of the active amino acids; His717 in ArcB, His64 in Ypd1, His48 in CheA and the proposed active serine (Ser 1231) in ChpA. Numbers indicate amino acid positions within the indicated protein.

B. Ribbon structures of the antiparallel helix core of the HPt (histidine phosphotransfer) domains in ArcB, CheA and Ypd1 and the proposed model for the Spt (serine phosphotransfer) domain of ChpA. Only the amino acid backbone has been shown with the active histidines shown as solid rendered in ball and stick (His717 in ArcB, His64 in Ypd1 and His 48 in CheA). The proposed active serine (Ser 1231 in ChpA) is also rendered in the proposed model of ChpA. Below the ribbon structures is shown the amino acid backbone of the local environment surrounding the active histidines (solid rendered) of ArcB, CheA and Ypd1 as well as the proposed active serine (solid rendered) in ChpA. Specific amino acids surrounding the active residue have also been solid rendered and labelled for reference. The appropriate alpha helices are also labelled.

© 2004 Blackwell Publishing Ltd, *Molecular Microbiology*, **52**, 873–893
The twitching motility phenotypes of the PAK mutants of *pilJ*, *chpA-E* were examined via the subsurface stab assay. In this assay the strain to be examined is stab inoculated through a 1% agar plate to the underlying Petri dish. Normal twitching motility results in rapid colony expansion at the agar/Petri dish interface, whereas non-twitching mutants such as PAK*pilA::TcR* (pilin subunit gene mutant) produce no such zone of expansion (Semmler *et al.*, 1999) (Fig. 4). Strains PAK*chpB::TcR*, PAK*chpC::TcR*, PAK*chpD::TcR* and PAK*chpE::TcR*

exhibit wild-type twitching motility (data not shown). Interestingly, the twitching stab assays demonstrated that all of the PAK mutants of *pilJ* show different twitching motility phenotypes to that previously described for PAO1 mutants of this gene. In the PAO1 genetic background, mutants of *pilJ* demonstrate no twitching motility via the subsurface stab assay (Darzins, 1994). However, in the PAK background the *pilJ* mutants clearly demonstrate some twitching motility though it is highly aberrant (Fig. 4A).

Different effects on twitching motility were observed for different mutant alleles of *chpA*. Aberrant twitching motility similar to *pilJ* mutants was observed in *chpA* mutants S52, S129, S164, S209, which carry transposon insertions within the FimL-like domain of ChpA as well as in mutants S10 and S23 which have insertions between the Hpt3 and Spt domains (Fig. 4A). Strain S349 which carries the transposon insertion between Hpt5 and Hpt6 and strains S45 and S33 which have insertions within the histidine kinase domain of ChpA behave similarly to the non-piliated *pilA* mutant in this assay and appear to be completely devoid of twitching motility (Fig. 4A). It is interesting that more severe twitching motility phenotypes occur in mutants with more C-terminal Tn insertions within *chpA*. The reason for this is unclear but one possibility is that there exists additional *chpA* transcripts which are initiated beyond the transposon insertion sites of S52, S164, S209, S129, S10 and S23, although we only have evidence for one isoform of ChpA (Fig. 2). However, as the epitope-tagged ChpA used to confirm the size of full-length ChpA was expressed from a multicopy plasmid under *Plac* control, this study does not rule out the possibility that ChpA may also exist in shorter isoforms whose expression was undetectable in this study. Alternatively, there may be some readthrough from the transposon or (given the large size of ChpA) some cryptic internal promoters in the *chpA* gene which can produce partially functional ChpA containing the C-terminal CheY domain, which has shown to be absolutely required for twitching motility by site-directed mutagenesis (A. Leech and J. S. Mattick, unpubl. obs.).

We attempted to complement the various *chpA* mutants of PAK with the clone pUCPChpA without success. However, this clone also completely abolished twitching motility in wild-type PAK whereas the vector pUCPSK showed no inhibition of twitching motility (data not shown). Whilst we believe it unlikely that the defects in twitching motility in all nine of the Tn5-B21 *chpA* mutants could be due to secondary mutations elsewhere in the genome, it remains possible that the defects are due to polar effects on neighbouring genes. We addressed these issues by constructing an in-frame deletion mutant of *chpA* in strain PAO1. As was observed for the PAK transposon insertion mutants S349, S33 and S45, this mutant, PAO1D*chpA* was completely devoid of twitching motility (data not

Fig. 4. A. Subsurface twitching motility assay of *P. aeruginosa* strains PAK; PAK*pilA::TcR*; S247 (*pilJ*), S164 (*chpA*); S23 (*chpA*) and S33 (*chpA*); Strains were incubated at 37°C for 24 h post inoculation (PAK) or for 48 h (PAK*pilA::TcR*; S247, S164, S23, S33) before staining. The bar represents 1 cm.

B. Light microscopy of twitching zones of *P. aeruginosa* strains PAK and S164. Micrographs of all *pilJ* and *chpA* mutants were indistinguishable from that shown here for S164. Micrographs of *chpB*, *chpC*, *chpD* and *chpE* mutants were indistinguishable from that of PAK. The bar represents 50 μm.

shown), confirming that *chpA* is required for twitching motility in *P. aeruginosa*.

As had been observed with *chpA* mutants of PAK, attempts to complement PAO1Δ*chpA* with pUCPChpA were unsuccessful as this clone also abolished twitching motility in wild-type PAO1 whereas the vector showed no inhibition of this phenotype. The clone pUCPChpA contains *chpA* inserted downstream of the *Plac* promoter (which is constitutively active in *P. aeruginosa*). These observations suggest that overexpression of *chpA* interferes with normal twitching motility in wild-type *P. aeruginosa*. In light of this, we cloned *chpA* into pMMB206, a low copy number vector that carries the repressor *lacI* and has a weakly expressed *PtacIacUV5* promoter (Morales *et al.*, 1991), to generate plasmid pMMBChpA. The presence of pMMBChpA in PAO1 caused only a very mild reduction in the diameters of the twitching motility zones relative to the vector control and pMMB-ChpA was able to restore twitching motility to PAO1Δ*chpA* to that of PAO1 containing pMMBChpA (data not shown). These observations indicate that high levels of ChpA expression, as presumably occurs with pUCPChpA, represses twitching motility in wild-type *P. aeruginosa*. Cloning *chpA* behind the weakly expressed *PtacIacUV* promoter in pMMB206 alleviated the repression problem and allowed complementation of twitching motility to PAO1Δ*chpA*. Interestingly, pMMBChpA was still unable to complement twitching motility to the *chpA* mutants of strain PAK. Both the *chpA* clones used in this study were subcloned from cosmid clones of PAO1 genome sequences, and thus it is possible that sequence differences between strains PAK and PAO1 might explain the inability of these clones to complement PAK mutants. The observation that PAK mutants of *pilJ* show different twitching motility phenotypes to that of PAO1 mutants (see above) also suggests that there are some strain differences in this system.

### Time-lapse video microscopy

Time-lapse video microscopy may be used to examine the dynamics of twitching motility at a cellular level (Semmler *et al.*, 1999). Such studies have revealed that twitching motility is a complex social process in which the bacteria move in a highly coordinated fashion, initially forming rafts of cells which move away from the colony edge, behind which an intricate lattice-like network of cells is formed (see Fig. 4B). Cells within this network follow each other closely (via cell-cell contact/pili extension and retraction) and demonstrate frequent reversals of direction. Cells within the outgoing rafts also demonstrate some cell reversal but do so less frequently than those in the network. Non-twitching mutants demonstrate no differentiation of the colony edge (Semmler *et al.*, 1999).

Using this time-lapse video microscopy technique, we examined the twitching motility of selected mutants of *pilJ*, *chpA-E*. As expected, we found that those mutants which produce normal twitching zones via the subsurface stab assay (i.e. PAK*chpB*::TcR, PAK*chpC*::TcR, PAK*chpD*::TcR and PAK*chpE*::TcR) also appear to behave normally when examined microscopically (data not shown). The *pilJ* mutant S247 and *chpA* mutants S164 and S23 which demonstrate similar aberrant twitching motility via the stab assay also showed similar phenotypes when examined microscopically (Fig. 4B). These mutants appear capable of the early stages of twitching motility forming large, exaggerated rafts of cells which move away from the colony edge but lack the characteristic lattice formation. These rafts move at rates of about 0.2 mm min⁻¹ which is about 20 times slower than the rate at which wild-type leading edge rafts move, and only about twice as fast as the rate of colony expansion as the result of cell division as calculated from the non-motile PAK*pilA*::TcR mutant (Semmler *et al.*, 1999). Assuming that the overall rate of translocation of the leading edge rafts is governed by both the rate of individual cell motilities as well as the frequency of cellular reversals, it is therefore conceivable that the reduced rate of leading edge translocation could be due to either or both a reduction in individual cell motilities and/or increased rates of reversal frequencies. Although it is difficult with our time-lapse video microscopy technique to track individual cells in the large, slow moving, densely packed, leading edge rafts, it does seem clear that the rate of individual cell motilities is significantly decreased relative to wild type and that there is no evidence for increased reversal frequencies. Indeed, unlike the wild-type situation where cells within the leading edge rafts can be readily seen to reverse direction and move against the overall direction of the raft, S247 (*pilJ*), S23 (*chpA*) and S164 (*chpA*) show no detectable cellular reversals. These observations suggest that the Chp system controls at least the rate of cellular movements and possibly also the frequency of cell reversal during the process of twitching motility.

Interestingly, the mutants of *chpA* (S349, S45 and S33), which show no twitching motility macroscopically, also form large rafts of cells (Fig. 4B). These rafts move at the same rate as that of colony expansion measured in the non-twitching *pilA* mutant. Unlike PAK*pilA*::TcR which forms an even colony edge (Semmler *et al.*, 1999), the cells of the *chpA* mutants (S349, S45 and S33) are able to assemble into the raft-like projections which are seen at the leading edge of twitching zones. However, the cells within these assemblies appear to be non-motile and show no detectable cellular reversals. Thus it appears that the defects in colony expansion seen in the various *pilJ* and *chpA* mutants are likely accounted for by decreased rates of cellular motility. This is consistent with our hypothesis.
882 C. B. Whitchurch et al.

esis that the Chp chemosensory system influences the rate of expeditionary radiation of the colony through control of twitching motility in response to environmental cues.

**Pilin production**

The PAK mutants of *pilJ*, *chpA*, *chpB*, *chpC*, *chpD* and *chpE* can be categorized based on their twitching motility phenotypes. Pili production by the *chpC*, *chpD* and *chpE* mutants which demonstrate normal twitching motility was assessed by ELISA of whole cells using antisera against the major pilin subunit PilA (Fig. 5A). These analyses indicated that the PAK*chpB::TcR* mutant is hyperpiliated relative to wild-type PAK, which is not surprising if the normal function of ChpB, by analogy to CheB in *E. coli*, is to mediate a feedback circuit for sensory adaptation. The PAK*chpC::TcR*, PAK*chpD::TcR* and PAK*chpE::TcR* mutants produce less surface pili than wild-type with the *chpC* mutant demonstrating the most significant decrease in surface piliation. Western analysis of whole cells using

![Figure 5](#fig-5)

Fig. 5. Pilin expression in wild type and *P. aeruginosa* mutant strains.
A. ELISA of whole cells from the *Pseudomonas* strains PAK (■); PAK*pilA::TcR* (▲); S247 (*pilJ*) (×); S164 (*chpA*) (×); S23 (*chpA*) (●); S33 (*chpA*) ( ); PAK*chpB::TcR* (—); PAK*chpC::TcR* (○); PAK*chpD::TcR* (◆) and PAK*chpE::TcR* (□). Pilin was detected with anti-PilA serum and is indicative of the levels of surface pili in these strains.
B. Immunoblot of pilin found in whole cell (WC) preparations after surface pili have been sheared (top panel) of strains PAK (lane 1); PAK*pilA::TcR* (lane 2); S247 (*pilJ*, lane 3); S164 (*chpA*, lane 4); S23 (*chpA*, lane 5); S33 (*chpA*, lane 6); PAK*chpB::TcR* (lane 7); PAK*chpC::TcR* (lane 8); PAK*chpD::TcR* (lane 9) and PAK*chpE::TcR* (lane 10). Immunoblot of pilin found in sheared surface pili preparations (bottom panel) of strains PAK (lane 1); PAK*pilA::TcR* (lane3); R306 (*pilV*, lane 4); S247 (*pilJ*, lane 5) S164 (*chpA*, lane 6); and S23 (*chpA*, lane 7). The *pilV* mutant strain which is defective in assembly of pili (Alm and Mattick, 1995) was included in these assays to control for the contribution of pilin to the surface pili samples as a result of cell lysis.
C. *PpilA::xylE* transcriptional reporter assays. Catechol dioxygenase activity was determined for wild-type PAK, *pilJ*, *chpA*, *chpB*, *chpC*, *chpD* and *chpE* mutants carrying the *PpilA::xylE* transcriptional fusion reporter plasmid pMIC66. The *pilR* mutant (R94) carrying pMIC66 was included as a negative control as the transcriptional activator PilR is required for *pilA* transcription (Hobbs et al., 1993). Mean activities from three independent experiments are shown and error bars (where visible) represent the standard deviation of the mean.
D. ELISA of whole cells from the *Pseudomonas* strains PAK + pUCP19 (◆); PAK + pAW103 (▲); S33 (*chpA*) + pUCP19 (●); S33 (*chpA*) + pAW103 (■). Pilin was detected with anti-pilin serum and is indicative of the levels of surface pili in these strains. The ELISAs and Westerns shown here are representative of results obtained from multiple assays.

© 2004 Blackwell Publishing Ltd, *Molecular Microbiology*, **52**, 873–893
anti-PilA antisera show that these mutants produce wild-type levels of cell-associated pilin (Fig. 5B). Northern analysis (data not shown) and PpilA::xylE transcriptional reporter assays (Fig. 5C) indicate that these mutants also show wild-type levels of *pilA* transcription. These observations suggest that the hyperpiliated phenotype of the *chpB* mutant and the reduced levels of surface piliation observed in the *chpC*, *chpD*, *chpE* mutants is not occurring at the level of pilin production but is probably due to altered rates of pilus assembly and/or retraction. However, these alterations in levels of surface piliation do not translate into obvious defects in twitching motility under the conditions of our assay.

The PAK mutants of *pilJ* (represented here by S247) and N-terminal *chpA* (represented by S164 and S23) which have aberrant twitching motilities, show little to no production of surface pili by the whole cell ELISA (Fig. 5A). However, Western blots of sheared surface pili using anti-PilA antiserum demonstrate that these *pilJ* and *chpA* mutants do have a small but detectable amount of surface piliation (Fig. 5B) which is presumably sufficient for PO4 phage sensitivity and for mediating the small degree of aberrant twitching motility seen in these mutants. Whole cell Western analysis, using an anti-PilA antiserum, shows that these strains produce normal levels of cell-associated pilin (Fig. 5B). Northern analysis (data not shown) and transcriptional PpilA::xylE promoter reporter assays (Fig. 5C) demonstrate that these mutants show wild-type levels of *pilA* expression. These results indicate that the alterations in levels of surface pili seen in these mutants are not due to defects in pilin production which suggests that the defects are occurring at the point of pilus extension and/or retraction.

The non-twitching *chpA* mutants S349, S45 and S33 (represented here by S33) showed little to no detectable surface pili by ELISA (Fig. 5A). These mutants also normally showed almost undetectable levels of pilin in whole-cell samples (Fig. 5B; see below). Northern analysis (data not shown) and PpilA::xylE promoter reporter assays (Fig. 5C) indicated that the defect in pilin production in this class of *chpA* mutant is due to reduced *pilA* transcription, although this was variable (see below). However extended exposure of both the whole cell Western and Northern analyses (data not shown) indicated that a small amount of pilin was produced by these *chpA* mutants, which is presumably sufficient for mediating sensitivity to the pilispecific bacteriophage PO4. Constitutive expression of the pilin gene *pilA* from the plasmid pAW103 restores production of surface pili to S33 as shown by ELISA (Fig. 5D). This result indicates that ChpA is not required for the expression of other components of the pilus structure or assembly machinery. However, *in trans* expression of *pilA* with pAW103, which restores twitching motility to *pilA* mutants (Watson et al., 1996b), does not complement

twitching motility to *chpA* mutants (data not shown) indicating that ChpA is also required for other aspects of twitching motility.

It should be noted, however, that during our studies we found that the *chpA* mutants S349, S45 and S33 at times produced wild-type levels of cell-associated PilA as shown by both PpilA::xylE reporter and whole cell Western (data not shown). In these instances however, surface piliation remained low and these mutants remained completely non-twitching, in contrast to the aberrant twitching motility phenotypes observed with the *pilJ* (S247) and N-terminal *chpA* mutants (S164, S23). We have not been able to account for the variable levels of *pilA* expression in these C-terminal *chpA* mutants. We can only speculate that some variability in media batches or other environmental condition in different experiments was responsible for the altered *pilA* expression phenotypes.

**The Chp system is required for cytotoxicity of epithelial monolayers in vitro**

Previous studies have indicated a role for functional type IV pili in *P. aeruginosa* interactions with cultured cell lines as well as for virulence in animal models of disease (Comolli et al., 1999; Kang et al., 1997). Given the involvement of the *chp* cluster gene products in twitching motility, we tested whether representative mutants of the *chp* cluster affected the ability of PAK to damage epithelial cells *in vitro*. Transposon insertion mutants of *chpA* (S164 which contains an insertion into the FimL-like domain; S23 which contains an insertion between HPt3 and SPt, and S33 which contains a C-terminal insertion in the histidine kinase domain) as well as allelic exchange mutants of *chpB*, *chpC*, *chpD* and *chpE* were added to the apical surface of MDCK cells grown as polarized monolayers on Transwell filters. Compared to PAK, the cytotoxicity of the *chpB*, *C*, *D*, and *E* mutants, which have normal twitching motility, was not significantly different from wild type (Fig. 6A). All of the *chpA* mutants (S164, S23, and S33) showed greatly reduced cytotoxicity, with the magnitude of the decrease being similar to that observed for the *pilA* mutant (Fig. 6A). Together with our previous observations that pilated but non-twitching strains of *P. aeruginosa* demonstrate similarly reduced cytotoxicity (Comolli et al., 1999), these results confirm that cytotoxicity of epithelial cells by *P. aeruginosa* requires fully functional type IV pili.

**The Chp system contributes to full virulence in a mouse model of acute pneumonia**

The role of *chp* genes in virulence was assessed in a mouse model of acute pneumonia by two approaches. First, the mortality of mice intranasally inoculated with PAK, *pilA*, or the *pilJ* (S43) and the *chpA* (S164, S23, S33)

and S349) mutants was followed over a 7-day period. While none of the mice inoculated with the *pilA* strain died, no difference in mortality between wild-type PAK and the *pilJ* and *chpA* mutants was observed (data not shown).

Using colonization as a more sensitive measure of virulence, we performed competition assays which allowed us to compare the efficiency of colonization of the lung or liver of the *pilJ* and *chpA-E* mutant strains to that of the wild type (Fig. 6B). A competitive index was calculated by generating the ratio of mutant strain to wild type bacteria recovered from the lung or liver and comparing it to the same ratio in the infecting inoculum (roughly 1.0 but calculated precisely for each experiment). A competitive index greater than 1.0 indicates that the mutant strain colonized better than wild type and a competitive index less than 1.0 indicates that the mutant strain was less efficient than the wild type in colonization. Interestingly, the *chpA* mutants were more effective in colonizing the lung compared to wild type (C.I. > 1.5). However, they were less able to spread to and/or colonize the liver. This was best appreciated when the ratio of the lung C.I. to liver C.I. was compared, which for the *pilJ* and *chpA* mutants was <0.6 (Fig. 6B and data not shown). Interestingly the *chpD* mutant also showed a defect in spread to the liver though this strain shows wild-type twitching motility. This suggests that *chpD*, which is predicted to encode a protein belonging to the AraC class of transcriptional activators, plays a role in the spread of *P. aeruginosa* to the liver *in vivo*.

## Discussion

The proteins PilG-K, ChpA-E appear to constitute a complex chemosensory system (Chp) which is analogous to bacterial chemotaxis systems which control swimming motility in response to environmental stimuli (Falke *et al.*, 1997; Djordjevic and Stock, 1998; Armitage, 1999; Bourret and Stock, 2002). The *P. aeruginosa* Chp system consists of a putative methyl accepting chemoreceptor PilJ; a methyl transferase CheR-like protein PilK, 2 CheW homologues (PilJ and ChpC) and a methyl esterase CheB homologue ChpB. The Chp system also has three CheY response regulators modules: PilG and PilH and a domain located at the C-terminus of the CheA-like histidine kinase of this pathway, ChpA. ChpA is a complex protein with nine putative sites of phosphorylation: six histidines, a threonine and a serine, each located in conserved HPt-like phosphotransfer domains, and an aspartate in a C-terminal CheY receiver domain. It is likely that ChpA is responsible for phosphotransfer reactions with PilG and PilH as well as an intramolecular phosphotransfer to its CheY domain. By analogy with the well-characterized Che systems in which CheY interacts with the flagella motor to regulate swimming motility, either or both of the CheY homologues PilG and PilH may interact with the putative type IV pilus motor to control twitching motility. The Chp system, like many other bacterial chemotaxis systems, lacks a homologue of the enteric CheZ which serves to accelerate dephosphorylation of phosphorylated CheY. Other chemotaxis systems such as that of *Rhodobacter sphaeroides* utilize multiple CheY proteins to act as phosphate sinks to add a similar layer of regulation to the system (Armitage, 1999). It is possible that either of PilG or PilH and/or the C-terminal CheY domain of ChpA serves a similar function in this system.

We predict that the Chp chemosensory pathway controls twitching motility in response to environmental cues in a manner analogous to the chemotactic control of swimming motility. The *P. aeruginosa* genome encodes a total of 26 putative methyl accepting chemotaxis proteins (Croft *et al.*, 2000; Stover *et al.*, 2000). As the Chp system possesses two CheW homologues (PilJ and ChpC), it is possible that these serve to complex a number of MCPs to the histidine kinase ChpA. This model provides for the input of multiple environmental signals which are integrated by the Chp system to control twitching motility. Although much is known about the molecular genetics of twitching motility in *P. aeruginosa*, little is known about the environmental cues that control this motility. Our observations in combination with previous studies (Darzins, 1993, 1994, 1995) have demonstrated that *pilI*, *pilJ*, and *chpA* but not *chpC* are required for normal twitching motility. By analogy with the well-characterized chemotaxis systems of enteric bacteria, these observations suggest that PilJ may be the adaptor protein which couples at least the MCP PilJ to ChpA and that this complex is essential for the control of twitching motility. It is possible that the second CheW homologue of this system, ChpC, serves to complex other MCPs to ChpA to facilitate the input of multiple environmental cues into this chemosensory pathway. However, these complexes are not essential for the control of twitching motility at least under the conditions assayed in this study.

It is also now clear, both morphologically and genetically, that twitching motility and the social aspect of gliding motility, such as occurs in *Myxococcus xanthus*, are essentially the same process (Semmler *et al.*, 1999; Matzick, 2002). The Chp system we have described here resembles in many aspects the Frz system of *M. xanthus* which also has multiple CheY response regulators modules found in FrzZ (which is a fusion of two CheY domains) and the C-terminus of the CheA/CheY hybrid FrzE (Ward and Zusman, 1999). FrzZ and FrzE then are probably analogous in function to PilG, PilH and ChpA. The Frz system also has homologues of PilJ (FrzCD), PilK (FrzF), ChpB (FrzG) and PilJ and ChpC (FrzA, FrzB) (Ward and Zusman, 1999). FrzB has been previously thought to be a protein unique to the Frz system but our analyses indi-
cate that this protein is likely to be a second CheW (Pil/ ChpC) homologue of this system. However, the *M.* xanthus Frz system differs from the Chp system in that the histidine kinase of the system FrzA possesses only a single site of autophosphorylation in contrast to the eight predicted phosphotransfer domains identified in ChpA. The Frz system and chemotaxis systems of other bacteria also lack homologues of ChpD and ChpE. The function (if any) of these two proteins within the Chp chemosensory pathway is currently unclear.

The *M.* xanthus Frz system controls the reversal frequency of cells during gliding motility. Cellular reversals are thought to be due to switching of the sites of pili extrusion from one pole of the bacterium to the other (Sun *et al.*, 2000). We predict that the Chp system would play a similar role in the control of twitching motility in *P.* aeruginosa. Indeed, time-lapse video microscopy suggests that the Chp system controls both the rate of cellular movements and the frequency of cellular reversals during twitching motility.

Cellular movements during twitching motility in *P.* aeruginosa and *N.* gonorrhoeae have recently been shown to be mediated by pilus extension and retraction (Merz *et al.*, 2000; Skerker and Berg, 2001). It possible that the Chp chemosensory system exerts its influence on twitching motility through control of pilus assembly and/or retraction. Our studies have demonstrated that Chp system mutants show altered levels of surface piliation. The hyperpiliated phenotype of *chpB* mutants could conceivably be caused by either a reduced rate of pilus retraction or an increased rate of pilus assembly. Conversely mutants of *pilJ*, *chpA* and *chpC* are significantly reduced in their levels of surface piliation which may be accounted for by increased retraction or reduced assembly rates. We propose that in a manner analogous to CheY interaction with the flagella motor to control swimming motility in bacteria, that the CheY homologues of this system PilG and/or PilH interact with the putative pilus motor to control pilus extension and retraction. This could occur through direct interaction with the nucleotide binding proteins PilB, PilT and PilU which are thought to control pilus extension (PilB) and retraction (PilT/U) (Mattick, 2002). In addition to direct interactions with the pili motor we have found that the Chp system (via ChpA) also influences type IV pili biogenesis through control of *pilA* expression.

Motility in the cyanobacterium *Synechocystis* PCC6803 has recently been shown to be mediated by type IV pili (Bhaya *et al.*, 2000). Phototactic control of this motility is mediated by two chemosensory pathways referred to as Tax1 and Tax3 (Bhaya *et al.*, 2001). Like the Chp and Frz systems both the Tax1 and Tax3 pathways possess a hybrid CheA/CheY histidine kinase (TaxAY1 and TaxAY3). Neither of these proteins is as complex as ChpA although TaxAY1 does possess two HPt domains. Each of these

Chemosensory control of twitching motility in *P.* aeruginosa 885

systems also have 3 CheY domains represented by TaxY1 and TaxY3, the C-terminal domain of TaxAY1 and TaxAY3 and a domain located at the C-termini of TaxP1 and TaxP3. Like the Frz and Chp systems the Tax1 system possesses two CheW homologues: TaxW1 and the predicted product of the gene with Cyanobase designation sll0044 (which had not been recognized as a CheW homologue and is located directly downstream of *taxAY1*). Interestingly, searches of the completed *Synechocystis* PCC6803 genome indicates that genes encoding homologues of CheB or CheR are not present in the *Synechocystis* PCC6803 genome. Mutants of the Tax1 and Tax3 systems show defects in the biogenesis of type IV pili (Bhaya *et al.*, 2001). It therefore appears that the Tax1 and Tax3 chemosensory systems of *Synechocystis* PCC6803 intersect to control type IV pili biogenesis and twitching motility in a similar manner to the Chp system of *P.* aeruginosa.

Our database analyses have indicated that the genome sequences of *N.* europaea, *X.* fastidiosa, *R.* metallidurans, *R.* solanacearum, *X.* campestris, *X.* axonopodsi, *M.* degredans, *P.* putida, *P.* syringae and *P.* fluorescens each encode a multi-Hpt domain containing CheA/CheY hybrid. Analysis of the preliminary genome sequences of *Dickeylobacter nodosus* (kindly provided by TIGR) indicates that this organism also encodes a ChpA homologue which contains at least three HPt domains and an SPT domain. These proteins are highly similar to one another and to *P.* aeruginosa ChpA and appear to represent a distinct subclass of CheA-like histidine kinases. All of the ChpA homologues (except those found in *Ralstonia* and *Xanthomonas* species) also contain putative TPt and/or SPt domains. This suggests that the presence of a potentially phosphorylatable serine or threonine in this type of phosphotransfer domain may be an as yet unrecognized conserved feature of some CheA-like bacterial kinases.

Where sufficiently assembled sequences are available it is evident that the similarity between the ChpA homologues extends to gene organization. The coding sequence for the ChpA homologue of *M.* degredans resides in a gene cluster that encodes homologues of each of PilG, PilH, PilI, PilJ, PilK, ChpB and ChpC in the same organization as the *P.* aeruginosa Chp cluster (see Fig. 1). ChpA homologues of *D.* nodosus, *P.* fluorescens, *P.* putida, and *P.* syringae are also encoded in a conserved gene clusters with conserved gene order except that these systems are missing homologues of both PilK and ChpB. The Chp clusters of *X.* fastidiosa, *X.* axonopodis and *X.* campestris are missing PilK homologues but do encode a distantly related homologue of ChpB. *X.* fastidiosa is also missing a PilH homologue. The *R.* solanacearum cluster is missing homologues of PilK, ChpB and ChpC. Except for the *D.* nodosus sequences (which are preliminary and may contain frame-shift errors) and the

© 2004 Blackwell Publishing Ltd, *Molecular Microbiology*, **52**, 873–893
ChpB homologues *X. fastidiosa*, *X. axonopodis* and *X. campestris*, each orthologous protein in these clusters are highly similar (80–90%) across their entire length. Interestingly, none of these other Chp clusters encode homologues of the *P. aeruginosa* proteins ChpD and ChpE which belong to the AraC family of transcriptional regulators and the LysE family of amino acid efflux proteins, respectively. This suggests either that the role of these proteins is specific to the *P. aeruginosa* system or that they are not *bona fide* components of this pathway. Our *P. aeruginosa chpD* and *chpE* mutants show no apparent defects in twitching motility and only mild perturbation in pili biogenesis and thus the role of these proteins in this chemosensory pathway is unclear.

Further genome analysis predicts that all of these bacteria possess type IV pili so it is likely that the Chp system in each is involved in the regulation of twitching motility similarly to that described here for *P. aeruginosa*. However, our database analyses indicate that the majority of bacteria (with available genome sequences) that possess Tfp including *Neisseria gonorrhoeae*, *N. meningitidis*, *Legionella pneumophila*, *Shewanella putrefaciens*, *S. oneidensis* MR-1, *Vibrio cholerae*, *V. parahaemolyticus* and *V. vulnificus* do not encode an orthologous Chp system. Interestingly, the bacteria that do have identifiable Chp systems are found in soil, water and/or plants, with the possible exception of *D. nodosus* which is considered an obligate pathogen albeit in an agricultural context.

Our studies have also indicated that the Chp system is required for cytotoxicity to epithelial cell monolayers and for full virulence in a mouse model of acute pneumonia. We had previously identified a transposon insertion mutant of *chpA* in a screen for *P. aeruginosa* PA103 mutants defective in epithelial cell injury (Kang et al., 1997). Here we show that *P. aeruginosa* PAK mutants of *pilJ*, *chpA* and *chpD* are less able to spread to and/or colonize the liver and that *chpA* mutants are more effective in colonizing the lung compared to wild type. It is possible that the observed reduction in colonization of the liver and enhanced colonization of the lungs seen in these mutants is related to chemotactic control of twitching motility by this system. Interestingly, in a recent screen for *in vivo* regulators of *Vibrio cholerae* virulence it was found that motile (swimming) but non-chemotactic mutants of *V. cholerae* showed enhanced colonization of the mouse intestine although with aberrant distribution (Lee et al., 2001). Our observations also suggest that the putative transcriptional regulator ChpD plays some role in the ability of *P. aeruginosa* to spread to and/or colonize the liver.

Interestingly, components of the Chp system were recently identified in a genetic screen for *P. aeruginosa* mutants that were defective in killing of the fruit fly *Drosophila melanogaster* (D’Argenio et al., 2001). This study demonstrated that the requirement for the Chp system in

*Drosophila* killing was separate from its involvement in twitching motility. These investigators proposed that the Chp system may control the expression of other *P. aeruginosa* virulence factors. We are currently investigating the mechanisms via which the Chp signal transduction system controls twitching motility and how this chemosensory system coordinates the expression of other virulence determinants of *P. aeruginosa*. However, it is clear that *P. aeruginosa* has evolved a highly complex chemosensory system to control twitching motility in response to environmental signals and which, either directly or indirectly, coordinates twitching motility with the expression of many other virulence determinants of this pathogen.

### Experimental procedures

#### Bacterial strains, plasmids and media

The *E. coli* strain DH5α (*recA endA1 gyrA96 hsdR17 thi-1 supE44 relA1 φ80 d lacZΔM15*) was used in all genetic manipulations and in the preparation of DNA sequencing templates. *Escherichia coli* strain S17-1 was used as the donor strain in the bacterial conjugation. The *P. aeruginosa* strains used were PAO1 strain ATCC 15692 (American Type Culture Collection); PAK (D. Bradley, Memorial University of Newfoundland, St John’s, Canada), Tn5-B21 mutants of this strain (Hobbs et al., 1993), including the previously characterized *pilR* mutant R94 (Hobbs et al., 1993); *pilV* mutant R306 (Alm and Mattick, 1995); and PAK*pilA::TcR* (previously referred to as AWK; Watson et al., 1996b). A PAO1 cosmid library (kindly provided by B. Holloway, Monash University, Clayton, Australia; Ratnaningsih et al., 1990; Huang et al., 2000) was used in the subcloning and sequence analysis of the *chp* region. The preparation of *E. coli* competent cells and transformation protocols were followed according to Sambrook et al. (1989). Preparation of *P. aeruginosa* competent cells and transformations were performed as described previously (Mattick et al., 1987). Plasmids used in this study are listed in Table 1. *Escherichia coli* and *P. aeruginosa* liquid cultures were maintained in Luria–Bertani (LB) broth and solid media was prepared by adding 1–1.5% agar. The media used in the light microscopy contained 4 g l⁻¹ tryptone, 2 g l⁻¹ yeast extract, 2 g l⁻¹ NaCl, 1 g l⁻¹ MgSO₄·7H₂O and 8 g l⁻¹ GelGro (ICN) as a solidifying agent. The following antibiotic concentrations were used for the selection of *E. coli*: tetracycline –12.5 µg ml⁻¹ for plasmid selection and 40 µg ml⁻¹ for cosmid selection; ampicillin 100 µg ml⁻¹, chloramphenicol 25 µg ml⁻¹ and kanamycin 50 µg ml⁻¹. The concentration of antibiotics for the selection of *P. aeruginosa* were carbenicillin 250 µg ml⁻¹, chloramphenicol 250 µg ml⁻¹, rifampicin 20 µg ml⁻¹ and tetracycline 200 µg ml⁻¹.

### Recombinant DNA techniques and sequence analysis

The preparation of plasmid DNA, restriction endonuclease digestion (New England Biolabs), ligation reactions, Southern blotting, and radiolabelling of probe were carried out using standard protocols (Sambrook et al., 1989). Sequence templates were generated by a combination of subcloning and

Table 1. Plasmids used in this study.

| Plasmid | Relevant characteristics | Reference |
| --- | --- | --- |
| pSM-TET | Source of TcR cassette | Mongkolsuk et al. (1993) |
| pRIC380 | *P. aeruginosa* suicide vector | Alm and Mattick (1996) |
| pJEN34 | pRIC380 with EZ::TN <TET-1> in β lactamase gene; TcR | This study |
| pUK21, pOK12 | *E. coli* cloning vectors, KmR | Vieira and Messing (1991) |
| pUC21 | *E. coli* cloning vectors, ApR | Vieira and Messing (1991) |
| pUCPSK/KS | *P. aeruginosa* – *E. coli* shuttle vectors, ApR | Watson et al. (1996a) |
| pBluescript II SK/KS | *E. coli* cloning vector, ApR | Stratagene |
| pGEM-T | *E. coli* cloning vector, ApR | Promega |
| pUCP19 | *P. aeruginosa* – *E. coli* shuttle vector, ApR | Schweizer (1991) |
| pMMB206 | Broad host range vector with inducible *PtalacUV* promoter, CmR | Morales et al. (1991) |
| pAW103 | 1.2 kb HindIII fragment containing PAK *pilA* in pUCP19 | Watson et al., (1996b) |
| X1918 | Source of promoterless *xylE* reporter gene cassette | Schweizer (1993) |
| pMO02116, pMO02218, pMO01109, pMO01539 | pLA2917 containing partial Sau3A PAO1 chromosomal DNA fragments. These cosmids contain regions of the PAO1 genome that span the *pilG-K, chpA-E* cluster. | Ratnaningsih et al. (1990) |
| pAW153 | 1.5 kb *KpnI/NotI* fragment containing *chpB* in pOK12 | This study |
| pMIC9719 | 4 kb *BgII/SacI* fragment from pMO01539 containing region downstream of *chpB* cloned into pUK21 | This study |
| pMIC86 | pAW153 with TcR cassette cloned in *ClaI* site of *chpB* | This study |
| pMIC9722 | pMIC9719 with TcR cassette cloned into *KpnI* site of *chpC* | This study |
| pMIC69 | pMIC9719 with TcR cassette cloned into *Xhol* site of *chpD* | This study |
| pMIC34 | pMIC9719 with TcR cassette cloned into *Apal* site sof *chpE* | This study |
| pMIC87 | pRIC380 carrying *chpB*::TcR from pMIC86 on *Spel* fragment | This study |
| pMIC9723 | pRIC380 carrying *chpC*::TcR from pMIC9722 on *Spel* fragment | This study |
| pMIC79 | pRIC380 carrying *chpD*::TcR from pMIC69 on *Spel* fragment | This study |
| pMIC35 | pRIC380 carrying *chpE*::TcR from pMIC34 on *Spel* fragment | This study |
| pRIC221 | 470 bp *HindIII/BspHI* fragment covering the *pilA* promoter cloned into *HindIII/Ncol* sites of pUC21 | This study |
| pMIC63 | *PpilA*::*xylE* transcriptional reporter; *xylE* from X1918 cloned as a *BamHI* fragment into the *BgII* site (downstream of *pilA* promoter) in pRIC221 | This study |
| pMIC66 | *Spel* fragment from pMIC63 cloned into pUCPSK | This study |
| pUCPChpA | 8 kb *SacI/ClaI* fragment from pMO01539 containing *chpA* in pUCPSK | This study |
| pUCPChpA-FLAG | pUCPChpA- + double FLAG tag inserted into *NotI/NruI* sites | This study |
| pMMBChpA | 8 kb *SacI/ClaI* fragment from pMO01539 containing *chpA* in pMMB206 | This study |
| pJB2 | pGEM-T containing 1 KB region 5′ of *chpA* | This study |
| pJB3 | pGEM-T containing 1 KB region 3′of *chpA* | This study |
| pJB4 | *EcoRI/HindIII* fragment from pJB2 and *HindIII/Xbal* fragment from pJB3 concatamerized and cloned into *EcoRI/Xbal* sites of pOK12 | This study |
| pJB5 | pJEN34 carrying Δ*chpA* from pJB4 on *Spel* fragment | This study |

Antibiotic resistance abbreviations: Tc, tetracycline; Km, kanamycin; Ap, ampicillin; Cm, chloramphenicol.

shotgun cloning strategies. The dsDNA was prepared for sequencing using a modified alkaline lysis method involving PEG precipitation (Applied Biosystems). The sequencing was performed using the dideoxy chain termination Applied Bio- systems PRISM on a 373A automated sequencer. BLAST searches of the GenBank databases were performed at NCBI.

**Identification of transposon insertions**

Genomic sequences flanking the site of insertion of the trans- poson in mutant S45 was cloned using a procedure termed ‘marker rescue’ which involved digestion of chromosomal DNA from the S45 mutant with *BgII* (which does not cut within the transposon) and *HindIII* (which cuts once within the transposon beyond the tetracycline resistance marker); ligation into pBluescriptll (Stratagene) and recovery of tetracy- cline resistant *E. coli* colonies. The cloned genomic DNA flanking the site of transposon insertion was used to screen a reference PAO1 genomic cosmid library identifying four cosmids (pMO02116, pMO02218, pMO01109, pMO01539) which contained wild-type DNA of this region.

Southern analysis of *KpnI* chromosomal digests of the Tn5-B21 library of twitching motility mutants using the insert of cosmid pMO02218 as a probe identified another 12 mutants (S10, S23, S33, S52, S154, S247, S43, S115, S129, S164, S209 and S349) with transposon insertions within this region. Marker rescue clones of the DNA flanking the site of trans- poson insertion in each of these 12 mutants was cloned by digestion with *EcoRI* or *HindIII*. The DNA adjacent to the transposon insertion in each of the mutants were then sequenced using a primer which is specific for the inverted repeats of Tn5-B21. Four of the mutants (S43, S115, S154, and S247) had transposon insertions in the previously described gene *pilJ* whereas the remainder had inserted into novel sequences. Further mapping via Southern analysis revealed that the mutants S10, S23, S33, S45, S52, S129, S164, S209, and S349 contained transposon insertions downstream of *pilK*.

**Expression and immunoblotting of FLAG-tagged ChpA**

To confirm the size of ChpA, the coding sequence for *chpA* was cloned into pUCPSK under the control of the *Plac* pro-

888 C. B. Whitchurch et al.

Fig. 6. A. *Pseudomonas aeruginosa* cytotoxicity of epithelial cell monolayers. Wild-type PAK, *pilA*, or *chpA-E* mutants (MOI 50) was added for 5 h to the apical surface of MDCK cells grown as monolayers on Transwell filters for 3 days. Cell death was quantitated by LDH release and is normalized to LDH release caused by PAK. Each assay was performed in triplicate and mean values are shown. Error bars represent standard error of the mean.

B. Mouse model of acute pneumonia. Equal numbers of PAK and the indicated *pilJ* and *chpA-E* mutants were instilled into the nares of anesthetized BALB/c mice. Twenty-four hours later, the animals were sacrificed, the lung and liver homogenized, and plated for viable counts on LB (total bacteria) and LB-tet (mutant strain). A competitive index was calculated by determining the ratio of mutant strain to wild type bacteria recovered from the lung or liver and comparing it to the same ratio in the infecting inoculum (roughly 1.0 but calculated precisely for each experiment). A competitive index greater than 1.0 indicates that the mutant strain colonized better than wild type and a competitive index less than 1.0 indicates that the mutant strain was less efficient than the wild type in colonization. The results of two to four experiments using a total of 10–26 mice were combined for each sample. Mean values are shown and error bars represent standard error of the mean.

moter (pUCPChpA). Because it was difficult to purify ChpA to generate specific antibodies, we circumvented the problem by inserting an oligonucleotide encoding a double FLAG epitope tag in frame with the *chpA* coding sequences at the unique *NotI* and *NruI* sites located close to the 3′ end of the *chpA* coding sequence (pUCPChpA-FLAG). Detection of FLAG-tagged ChpA was performed as follows. *Pseudomonas aeruginosa* transformants were subcultured 1:50 into LB from overnight cultures and incubated at 37°C with shaking for 4 h. Culture (1 ml) was pelleted and resuspended in 100 μl of sample buffer (60 mM Tris-HCL pH 6.8, 2% SDS, 10% glycerol, 5% β-mercaptoethanol, 0.001% bromophenol blue). To reduce viscosity whole cell proteins were prepared by passing through a 27 1/2 gauge needle 10 times and then centrifuged at 45 000 r.p.m. for 1 h to pellet chromosomal DNA. Samples were solubilized at 65°C (as ChpA appeared to be labile at 100°C) and displayed on 4–20% gradient or

12% SDS–PAGE gels (Laemmli, 1970), and electroblotted onto Immobilon-P transfer membrane (Millipore, Bedford, MA) in the Tris-glycine system described by Towbin *et al.* (1979). Membranes were blocked with 5% skimmed milk, probed with a 1:500 dilution of primary anti FLAG M2 monoclonal antibody (Sigma Aldrich) in 1% skim milk powder/0.1% Tween 20 in PBS. Membranes were then incubated with a 1:10 000 dilution of goat anti-rabbit immunoglobulin G conjugated with horse radish peroxidase (Amersham Biosciences) in 1% skim milk powder/0.1% Tween 20 in PBS followed by detection by ECL using the Supersignal chemiluminescence kit (Pierce).

### 3D-structural modelling

The technique of protein modelling by homology is used to

© 2004 Blackwell Publishing Ltd, *Molecular Microbiology*, **52**, 873–893

assign 3D-structure to a protein of unknown structure based on structural information from members of the same family (Blundell *et al.*, 1987). The crystal structure of the C-terminal HPt domain of ArcB (1AOB.pdb; Kato *et al.*, 1997) and the NMR structure of the phosphotransfer site of CheA (coordinates for CheA were kindly provided by F. W. Dahlquist, University of Oregon) were used as templates to assign structures to the HPt1, SPt and HPt4 domains of ChpA.

Coordinates were assigned to the conserved secondary structural regions of the ArcB and CheA according to the alignment shown in Fig. 3 using Molecular Simulations InsightII package on a Silicon Graphics Octane work station. All suitable loop structures were obtained by searching the Brookhaven Protein Databank for proteins containing loops of the same length joining similar secondary structural units. The resultant crude model was then refined by energy minimization using Biosym’s CVFF forcefield to a convergence criterion of less than 5.0kcal/molA (maximum derivative) as reported elsewhere (Urquhart *et al.*, 2000).

Based on studies of all coordinate entries in the Brookhaven Databank derived by X-ray, NMR and computational methods, along with a number of deliberately misfolded proteins, Eisenberg *et al.* (1989) suggested that a cut-off of <0.45 × Scal is useful for identifying grossly misfolded structures. Models with scores between 0.45 × Scal and Scal are considered correct, although it is still possible that locally misfolded regions occur. The examination of the profile score in a moving window scan of 10 residues can be used to identify locally misfolded regions.

### Construction of isogenic mutants

Allelic exchange mutants of the genes *chpB*, *chpC*, *chpD* and *chpE* were constructed using the sucrose selection system described previously (Schweizer, 1992; Alm and Mattick, 1996), as follows. The tetracycline resistance cassette (*TcR*) from pSM-TET was inserted into the *PstI* site of *chpB*; the *KpnI* site of *chpC*; the *Xhol* site of *chpD*; or the internal *Apal* sites of *chpE*. Suicide clones for allelic exchange were constructed as described in Table 1. These clones each contain the genes *sacB/sacR* which promote sensitivity to sucrose and *oriT* which enables conjugal transfer. The constructs were then transformed into the *E. coli* donor strain S17-1 in preparation for mating into PAK. Following conjugation, transconjugates were selected on 5% sucrose media containing tetracycline. This allows selection of colonies in which the plasmid sequences have been excised while leaving the homologously recombined mutated gene in the chromosome. All mutants were genotypically confirmed by Southern blotting. An in frame deletion of *chpA* in PAO1 was constructed as follows. An approximately 1 kb region 5′ of *chpA* and including the first 32 bp of *chpA* was amplified with primers chpA6 (5′-GAATTC AAGATGGCGAGCGAGATGCG-3′) and chpA7 (5′-AAGCTTCGTGCCGGTCACCCATAGCC-3′). An approximately 1 kb region 3′ of *chpA* and encompassing the final 214 bp of *chpA* was amplified with primers chpA8 (5′-AAGCTTTTCGAGATGCCGCGCATGGAC-3′) and chpA9 (5′-TCTAGACAGTCGGGGCCGA ACTGCTG). The PCR products were cloned into pGEM-T and sequence confirmed. The cloned PCR products were excised and the fragments concatamerized and cloned into pOK12. The *chpA* in frame deletion construct was then shuttled into the suicide vector pJEN34. pJEN34 is pRIC380 carrying the EZ::TN < TET-1 > transposon (EpiCentre) in the β-lactamase gene. The resultant *chpA* deletion construct pJB5 was transformed into S17-1 and mated into PAO1. Following sucrose selection, tetracycline sensitive colonies were PCR amplified with chpA6 and chpA7 to identify colonies in which the wild-type *chpA* sequences had been replaced with the in frame deletion of *chpA*. These mutants were genotypically confirmed by Southern blotting.

#### Motility assays

To assay subsurface twitching motility, the *P. aeruginosa* colony to be tested was stab inoculated through a 1% agar plate. Following incubation at 37°C, a zone of motility could be visualized between the agar and Petri dish interface. To aid visualization, the agar was compressed and then stained using a 0.05% Coomassie brilliant blue R250 stain (40% methanol, 10% acetic acid) as described previously (Alm and Mattick, 1995). Swimming motility was assayed by stab inoculation of the colony to be tested into a 0.3% agar plate followed by incubation overnight at 37°C.

#### Microscopy

Light microscopy was performed as previously described (Semmler *et al.*, 1999). Sterile microscope slides were submerged in molten GelGro media to obtain a thin layer of media coating the slide. The slides were allowed to set in a horizontal position and air dried briefly before use. The slides were then inoculated with a small loopful of bacteria taken from an overnight plate culture. A sterile glass coverslip was placed over the point of inoculation and the slide transferred to a large Petri dish containing a moist tissue and sealed with Nescofilm (BANDO Chemical Industries) to maintain humid conditions. Incubation times ranged from 2 to 6 h at 37°C. Slide cultures were examined using a Zeiss microscope Axioskop 50 with Nomarski facilities at 200–400 × magnification. Video microscopy was performed in a room heated to 30°C. Video images were recorded over a period of 2–4 h at speeds of either 1 field/3.22 s; 1 field/0.66 s; or real time (1 field per 1/50 second) using a JVC TK-CI38IEG video camera connected to a Sanyo TLS-S2500P time-lapse video recorder.

#### PilA immunoblotting

Detection of cell-associated pilin was performed as follows. Bacteria were grown for 24 h on LB agar, scraped from the agar surface, and resuspended to OD~600~ = 1.0 in 50 mM Na carbonate buffer (pH 9.6). One ml of the sample was centrifuged and the cell pellet resuspended in 20 ml sample buffer (60 mM Tris-HCL pH 6.8, 2% SDS, 10% glycerol, 5% β-mercaptoethanol, 0.001% bromophenol blue). To reduce viscosity, the samples were sheared by 20 passages through a 27 1/2 gauge needle and heated to 100°C for 5 min. Surface pili were isolated by harvesting cells from 24 h agar plate cultures into 2 ml of phosphate-buffered saline, and vortexing for 2 min. The suspension was centrifuged at low-speed

(2300 g for 5 min) to remove whole cells, after which the supernatant was collected and subjected to a high-speed centrifugation (15 000 g for 20 min) to remove cell debris. The resulting supernatant was incubated overnight at 4°C in the presence of 100 μM MgCl₂ to precipitate pili, as described previously (Alm and Mattick, 1995). The precipitate was collected by centrifugation (15 000 g for 20 min) and resuspended in gel loading buffer. Samples were then displayed with 15% SDS–PAGE gels (Laemmli, 1970), and electroblotted onto Immobilon-P transfer membrane (Millipore, Bedford, MA) in the Tris-glycine system described by Towbin *et al.* (1979). Membranes were blocked with 5% skimmed milk, probed with a 1:5000 dilution of primary anti-PilA antibody in 1% skim milk powder in PBS. Membranes were then incubated with a 1:10 000 dilution of goat anti-rabbit immunoglobulin G conjugated with horse radish peroxidase (Boehringer Mannheim) in 1% skim milk powder in PBS followed by detection by ECL using the Supersignal chemiluminescence kit (Pierce).

### ELISA

Enzyme-linked immunosorbent assays (ELISAs) to determine the amount of surface pili were performed as follows. Bacteria were grown for 16–24 h on LB agar, scraped from the agar surface, and resuspended to OD₆₀₀ = 1.0 in 50 mM Na carbonate buffer (pH 9.6). Two-hundred microlitres of the cell suspension were loaded to wells of a 96-well ELISA plate and incubated overnight at 4°C. Wells were then washed in PBS containing 0.5% Tween-20, and blocked for 1 h at 37°C with 3% BSA. Wells were washed again, and twofold serial dilutions of primary antibody (anti-PilA antibody at a starting dilution of 1:500) added for 1 h at 37°C. Wells were then washed, incubated at 37°C for 1 h with secondary antibody (1:5000 dilution of goat anti-rabbit immunoglobulin G conjugated with alkaline phosphatase (Boehringer), washed and colour developed using 4 mg ml⁻¹ p-nitrophenyl phosphate (Sigma) in 1 M Tris-Cl buffer (pH 8.0). Alkaline phosphatase activity was determined by absorbance at 405 nm using an ELISA reader (Bio-Rad).

### Assays of pilA expression

RNA preparation and Northern blots of *pilA* mRNA levels was performed as previously described (Whitchurch and Mattick, 1994). *PpilA::xylE* transcriptional reporter assays were performed as follows. pMIC66 was transformed in to the strains to be assayed. Dilutions (1:100) of overnight cultures were incubated with shaking at 37°C for 16 h. One hundred microlitres of each culture was transferred to wells of a 96-well dish and culture OD measurements at 595 nm recorded in a Bio-Rad ELISA plate reader. Acetone (20 μl) and 80 μl assay buffer (50 mM K₂HPO₄, pH 7.5) was added to each sample. Twenty microlitres of the cells/acetone/assay buffer solution diluted in 160 μl assay buffer was assayed after addition of 20 μl of the catechol 2,3-dioxygenase substrate, catechol (1.1 mg ml⁻¹). The enzyme catalyses the conversion of catechol to β-muconic ε-semialdehyde, a coloured product which absorbs light at 380 nm. Thus the rate of conversion of substrate to product can be followed by the absorbance at

380 nm. The rate of reaction over 2 min was recorded by taking A380 readings at 15 second time intervals in a Bench-Mark Microplate Reader (Bio-Rad). Initial rates of reaction were determined with Microplate Manager 3.11 v1.57 software.

#### Cytotoxicity assays

Cytotoxicity assays were carried out in 3-day-old MDCK confluent monolayers grown on Transwells as previously described (Comolli *et al.*, 1999) with the following modifications. Bacteria were used at an MOI of approximately 50. Lactate dehydrogenase assays were performed according to the manufacturer’s specifications (Sigma) following co-cultivation of the bacteria with the filter-grown MDCK cells for the indicated times.

#### Mouse model for acute pneumonia

Mouse mortality and competitive index assays were carried out as previously described (Comolli *et al.*, 1999). For the mortality studies, groups of five lightly anesthetized 6–8-week-old Balb/C female mice were inoculated directly in the nares with 50 μl of each strain (*A*₆₀₀ of 0.5; approximately 10⁸ bacteria). Mice were monitored over the subsequent seven days in compliance with the guidelines of Animal Care Committee of the University of California, San Francisco. The inoculum dose for the wild type and mutant strains for the competitive index experiments was adjusted to an *A*₆₀₀ of 0.75 (approximately 10⁸ bacteria) and the mice were sacrificed after 24 h.

### Acknowledgements

We would like to thank Professor F. W. Dahlquist for kindly providing the NMR structure of CheA which was used for structural modelling and J. Dyason for his assistance with the 3D-structural modelling. This work was supported by the National Health and Medical Research Council of Australia and the Australian Research Council. J.N.E. was supported by grants from the NIH (AI42806), the Cystic Fibrosis Foundation, and the American Lung Association. During a portion of this work, J.N.E. was a Career Investigator of the American Lung Association. J.C.C. was supported by the Bank of America-Gianinni Foundation.

### References

Aleshin, V.V., Zakataeva, N.P., and Livshits, V.A. (1999) A new family of amino-acid-efflux proteins. *Trends Biochem Sci* **24**: 133–135.

Alm, R.A., and Mattick, J.S. (1995) Identification of a gene, *pilV*, required for type 4 fimbrial biogenesis in *Pseudomonas aeruginosa* whose product possesses a prepilin-like leader sequence. *Mol Microbiol* **16**: 485–496.

Alm, R.A., and Mattick, J.S. (1996) Identification of two genes with prepilin-like leader sequences required for type 4 fimbrial biogenesis in *Pseudomonas aeruginosa*. *J Bacteriol* **178**: 3809–3817.

Alm, R.A., and Mattick, J.S. (1997) Genes involved in the biogenesis and function of type-4 fimbriae in Pseudomonas aeruginosa. Gene 192: 89–98.

Altschul, S.F., Madden, T.L., Schaffer, A.A., Zhang, J., Zhang, Z., Miller, W., and Lipman, D.J. (1997) Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res 25: 3389–3402.

Armitage, J.P. (1999) Bacterial tactic responses. Adv Microb Physiol 41: 229–289.

Bateman, A., Birney, E., Cerruti, L., Durbin, R., Eddy, S.R., et al. (2002) The Pfam protein families database. Nucleic Acids Res 30: 276–280.

Beatson, S.A., Whitchurch, C.B., Sargent, J.L., Levesque, R.C., and Mattick, J.S. (2002) Differential regulation of twitching motility and elastase production by Vfr in Pseudomonas aeruginosa. J Bacteriol 184: 3605–3613.

Bhaya, D., Bianco, N.R., Bryant, D., and Grossman, A. (2000) Type IV pilus biogenesis and motility in the cyanobacterium Synechocystis sp. PCC6803. Mol Microbiol 37: 941–951.

Bhaya, D., Takahashi, A., and Grossman, A.R. (2001) Light regulation of type IV pilus-dependent motility by chemosensor-like elements in Synechocystis PCC6803. Proc Natl Acad Sci USA 98: 7540–7545.

Blundell, T.L., Sibanda, B.L., Sternberg, M.J., and Thornton, J.M. (1987) Knowledge-based prediction of protein structures and the design of novel molecules. Nature 326: 347–352.

Bourret, R.B., and Stock, A.M. (2002) Molecular information processing: lessons from bacterial chemotaxis. J Biol Chem 277: 9625–9628.

Bradley, D.E. (1973) A pilus dependent Pseudomonas aeruginosa bacteriophage with a long non-retractile tail. Virology 51: 489–492.

Burkart, M., Toguchi, A., and Harshey, R.M. (1998) The chemotaxis system, but not chemotaxis, is essential for swarming motility in Escherichia coli. Proc Natl Acad Sci USA 95: 2568–2573.

Chiang, P., and Burrows, L.L. (2003) Biofilm formation by hyperpiliated mutants of Pseudomonas aeruginosa. J Bacteriol 185: 2374–2378.

Comolli, J.C., Hauser, A.R., Waite, L., Whitchurch, C.B., Mattick, J.S., and Engel, J.N. (1999) Pseudomonas aeruginosa gene products PilT and PilU are required for cytotoxicity in vitro and virulence in a mouse model of acute pneumonia. Infect Immun 67: 3625–3630.

Cox, S., Radzio-Andzelm, E., and Taylor, S.S. (1994) Domain movements in protein kinases. Curr Opin Struct Biol 4: 893–901.

Croft, L., Beatson, S.A., Whitchurch, C.B., Huang, B., Blakeley, R.L., and Mattick, J.S. (2000) An interactive web-based Pseudomonas aeruginosa genome database: discovery of new genes, pathways and structures. Microbiol 146: 2351–2364.

D’Argenio, D.A., Gallagher, L.A., Berg, C.A., and Manoil, C. (2001) Drosophila as a model host for Pseudomonas aeruginosa infection. J Bacteriol 183: 1466–1471.

D’Argenio, D.A., Calfee, M.W., Rainey, P.B., and Pesci, E.C. (2002) Autolysis and autoaggregation in Pseudomonas aeruginosa colony morphology mutants. J Bacteriol 184: 6481–6489.


Darzins, A. (1993) The pilG gene product, required for Pseudomonas aeruginosa pilus production and twitching motility, is homologous to the enteric, single-domain response regulator CheY. J Bacteriol 175: 5934–5944.

Darzins, A. (1994) Characterization of a Pseudomonas aeruginosa gene cluster involved in pilus biosynthesis and twitching motility: sequence similarity to the chemotaxis proteins of enterics and the gliding bacterium Myxococcus xanthus. Mol Microbiol 11: 137–153.

Darzins, A. (1995) The Pseudomonas aeruginosa pilK gene encodes a chemotactic methyltransferase (CheR) homologue that is translationally regulated. Mol Microbiol 15: 703–717.

Djordjevic, S., and Stock, A.M. (1998) Structural analysis of bacterial chemotaxis proteins: components of a dynamic signaling system. J Struct Biol 124: 189–200.

Eisenberg, D., Wesson, M., and Yamashita, M. (1989) Interpretation of protein folding and binding with atomic solvation parameters. Chem Scrip 29A: 217–221.

Falke, J.J., Bass, R.B., Butler, S.L., Chervitz, S.A., and Danielson, M.A. (1997) The two-component signaling pathway of bacterial chemotaxis: a molecular view of signal transduction by receptors, kinases, and adaptation enzymes. Annu Rev Cell Dev Biol 13: 457–512.

Fernandez, A., Hawkins, A.C., Summerfield, D.T., and Harwood, C.S. (2002) Cluster II che genes from Pseudomonas aeruginosa are required for an optimal chemotactic response. J Bacteriol 184: 4374–4383.

Gallegos, M.T., Schleif, R., Bairoch, A., Hofmann, K., and Ramos, J.L. (1997) Arac/XylS family of transcriptional regulators. Microbiol Mol Biol Rev 61: 393–410.

Giamarellou, H. (2000) Therapeutic guidelines for Pseudomonas aeruginosa infections. Int J Antimicrob Agents 16: 103–106.

Hobbs, M., and Mattick, J.S. (1993) Common components in the assembly of type 4 fimbriae, DNA transfer systems, filamentous phage and protein-secretion apparatus: a general system for the formation of surface-associated protein complexes. Mol Microbiol 10: 233–243.

Hobbs, M., Collie, E.S., Free, P.D., Livingston, S.P., and Mattick, J.S. (1993) PilS and PilR, a two-component transcriptional regulatory system controlling expression of type 4 fimbriae in Pseudomonas aeruginosa. Mol Microbiol 7: 669–682.

Huang, B., Whitchurch, C.B., Croft, L., Beatson, S.A., and Mattick, J.S. (2000) A minimal tiling path cosmid library for functional analysis of the Pseudomonas aeruginosa PAO1 genome. Microb Comp Genomics 5: 189–203.

Jander, G., Rahme, L.G., and Ausubel, F.M. (2000) Positive correlation between virulence of Pseudomonas aeruginosa mutants in mice and insects. J Bacteriol 182: 3843–3845.

Kang, P.J., Hauser, A.R., Apodaca, G., Fleiszig, S.M., Wiener-Kronish, J., Mostov, K., and Engel, J.N. (1997) Identification of Pseudomonas aeruginosa genes required for epithelial cell injury. Mol Microbiol 24: 1249–1262.

Kato, M., Mizuno, T., Shimizu, T., and Hakoshima, T. (1997) Insights into multistep phosphorelay from the crystal structure of the C-terminal HPt domain of ArcB. Cell 88: 717–723.

Kennelly, P.J. (2002) Protein kinases and protein phos-

phatases in prokaryotes: a genomic perspective. FEMS Microbiol Lett 206: 1–8.

Klausen, M., Aaes-Jorgensen, A., Molin, S., and Tolker-Nielsen, T. (2003a) Involvement of bacterial migration in the development of complex multicellular structures in Pseudomonas aeruginosa biofilms. Mol Microbiol 50: 61–68.

Klausen, M., Heydorn, A., Ragas, P., Lamberts, L., Aaes-Jorgensen, A., Molin, S., and Tolker-Nielsen, T. (2003b) Biofilm formation by *Pseudomonas aeruginosa* wild type, flagella and type IV pili mutants. Mol Microbiol 48: 1511–1524.

Laemmli, U.K. (1970) Cleavage of structural proteins during the assembly of the head of bacteriophage T4. Nature 227: 680–685.

Lee, S.H., Butler, S.M., and Camilli, A. (2001) Selection for *in vivo* regulators of bacterial virulence. Proc Natl Acad Sci USA 98: 6889–6894.

Letunic, I., Goodstadt, L., Dickens, N.J., Doerks, T., Schultz, J., Mott, R., et al. (2002) Recent improvements to the SMART domain-based sequence annotation resource. Nucleic Acids Res 30: 242–244.

Masduki, A., Nakamura, J., Ohga, T., Umezaki, R., Kato, J., and Ohtake, H. (1995) Isolation and characterization of chemotaxis mutants and genes of *Pseudomonas aeruginosa*. J Bacteriol 177: 948–952.

Mattick, J.S. (2002) Type IV pili and twitching motility. Annu Rev Microbiol 56: 289–314.

Mattick, J.S., and Alm, R.A. (1995) Common architecture of type 4 fimbriae and complexes involved in macromolecular traffic. Trends Microbiol 3: 411–413.

Mattick, J.S., Bills, M.M., Anderson, B.J., Dalrymple, B., Mott, M.R., and Egerton, J.R. (1987) Morphogenetic expression of *Bacteroides nodosus* fimbriae in *Pseudomonas aeruginosa*. J Bacteriol 169: 33–41.

Merz, A.J., So, M., and Sheetz, M.P. (2000) Pilus retraction powers bacterial twitching motility. Nature 407: 98–102.

Mongkolsuk, S., Vattanaviboon, P., Rabibhadana, S., and Kiatpapan, P. (1993) Versatile gene cassette plasmids to facilitate the construction of generalized and specialized cloning vectors. Gene 124: 131–132.

Morales, V.M., Backman, A., and Bagdasarian, M. (1991) A series of wide-host-range low-copy-number vectors that allow direct screening for recombinants. Gene 97: 39–47.

Moorey, L., Da Re, S., Pedelacq, J.D., Tolstykh, T., Faurie, C., Guillet, V., et al. (2001) Crystal structure of the CheA histidine phosphotransfer domain that mediates response regulator phosphorylation in bacterial chemotaxis. J Biol Chem 276: 31074–31082.

O'Toole, G.A., and Kolter, R. (1998) Flagellar and twitching motility are necessary for *Pseudomonas aeruginosa* biofilm development. Mol Microbiol 30: 295–304.

O'Toole, G.A., Gibbs, K.A., Hager, P.W., Phibbs, P.V. Jr and Kolter, R. (2000) The global carbon metabolism regulator Crc is a component of a signal transduction pathway required for biofilm development by *Pseudomonas aeruginosa*. J Bacteriol 182: 425–431.

Pullen, K., Rajagopal, P., Branchini, B.R., Huffine, M.E., Reizer, J., Saier, M.H. Jr et al. (1995) Phosphorylation of serine-46 in HPr, a key regulatory protein in bacteria, results in stabilization of its solution structure. Protein Sci 4: 2478–2486.

Rahme, L.G., Ausubel, F.M., Cao, H., Drenkard, E., Goumnerov, B.C., Lau, G.W., et al. (2000) Plants and animals share functionally common bacterial virulence factors. Proc Natl Acad Sci USA 97: 8815–8821.

Ratnaningsih, E., Dharmsthti, S., Krishnapillai, V., Morgan, A., Sinclair, M., and Holloway, B.W. (1990) A combined physical and genetic map of *Pseudomonas aeruginosa* PAO. J Gen Microbiol 136: 2351–2357.

Sambrook, J., Fritsch, E.F., and Maniatis, T. (1989) Molecular Cloning: a Laboratory Manual. Cold Spring Harbour, New York: Cold Spring Harbor Laboratory Press.

Schultz, J., Milpetz, F., Bork, P., and Ponting, C.P. (1998) SMART, a simple modular architecture research tool: identification of signaling domains. Proc Natl Acad Sci USA 95: 5857–5864.

Schweizer, H.P. (1991) Escherichia-Pseudomonas shuttle vectors derived from pUC18/19. Gene 97: 109–112.

Schweizer, H.P. (1992) Allelic exchange in *Pseudomonas aeruginosa* using novel ColE1-type vectors and a family of cassettes containing a portable *oriT* and the counter-selectable *Bacillus subtilis sacB* marker. Mol Microbiol 6: 1195–1204.

Schweizer, H.P. (1993) Two plasmids, X1918 and Z1918, for easy recovery of the *xylE* and *lacZ* reporter genes. Gene 134: 89–91.

Semmler, A.B.T., Whitchuch, C.B., and Mattick, J.S. (1999) A re-examination of twitching motility in *Pseudomonas aeruginosa*. Microbiol 145: 2863–2873.

Skerker, J.M., and Berg, H.C. (2001) Direct observation of extension and retraction of type IV pili. Proc Natl Acad Sci USA 98: 6901–6904.

Song, H.K., Lee, J.Y., Lee, M.G., Moon, J., Min, K., Yang, J.K., and Suh, S.W. (1999) Insights into eukaryotic multi-step phosphorelay signal transduction revealed by the crystal structure of Ypd1p from *Saccharomyces cerevisiae*. J Mol Biol 293: 753–761.

Stover, C.K., Pham, X.Q., Erwin, A.L., Mizoguchi, S.D., Warrener, P., Hickey, M.J., et al. (2000) Complete genome sequence of *Pseudomonas aeruginosa* PA01, an opportunistic pathogen. Nature 406: 959–964.

Strom, M.S., and Lory, S. (1993) Structure-function and biogenesis of the type IV pili. Annu Rev Microbiol 47: 565–596.

Sun, H., Zusman, D.R., and Shi, W. (2000) Type IV pilus of *Myxococcus xanthus* is a motility apparatus controlled by the *frz* chemosensory system. Curr Biol 10: 1143–1146.

Tan, M.W., and Ausubel, F.M. (2000) Caenorhabditis elegans: a model genetic host to study *Pseudomonas aeruginosa* pathogenesis. Curr Opin Microbiol 3: 29–34.

Towbin, H., Staehelin, T., and Gordon, J. (1979) Electrophoretic transfer of proteins from polyacrylamide gels to nitrocellulose sheets: procedure and some applications. Proc Natl Acad Sci USA 76: 4350–4354.

Urquhart, A.J., Kennedy, D., Gould, S.J., and Crane, D.I. (2000) Interaction of Pex5p, the type 1 peroxisome targeting signal receptor, with the peroxisomal membrane proteins Pex14p and Pex13p. J Biol Chem 275: 4127–4136.

Vieira, J., and Messing, J. (1991) New pUC-derived cloning vectors with different selectable markers and DNA replication origins. Gene 100: 189–194.

Vrljic, M., Garg, J., Bellmann, A., Wachi, S., Freudl, R., Malecki, M.J., *et al.* (1999) The LysE superfamily: topology of the lysine exporter LysE of *Corynebacterium glutamicum*, a paradigm for a novel superfamily of transmembrane solute translocators. *J Mol Microbiol Biotechnol* **1**: 327–336.

Ward, M.J., and Zusman, D.R. (1999) Motility in *Myxococcus xanthus* and its role in developmental aggregation. *Curr Opin Microbiol* **2**: 624–629.

Watson, A.A., Alm, R.A., and Mattick, J.S. (1996a) Construction of improved vectors for protein production in *Pseudomonas aeruginosa*. *Gene* **172**: 163–164.

Watson, A.A., Mattick, J.S., and Alm, R.A. (1996b) Functional expression of heterologous type 4 fimbriae in *Pseudomonas aeruginosa*. *Gene* **175**: 143–150.

Whitchurch, C.B., and Mattick, J.S. (1994) Characterisation of a gene, *pilU*, required for twitching motility but not phage sensitivity in *Pseudomonas aeruginosa*. *Mol Microbiol* **13**: 1079–1091.

Whitchurch, C.B., Hobbs, M., Livingston, S.P., Krishnapillai, V., and Mattick, J.S. (1991) Characterisation of a *Pseudomonas aeruginosa* twitching motility gene and evidence for a specialised protein export system widespread in eubacteria. *Gene* **101**: 33–44.

Whitchurch, C.B., Alm, R.A., and Mattick, J.S. (1996) The alginate regulator AlgR and an associated sensor FimS are required for twitching motility in *Pseudomonas aeruginosa*. *Proc Natl Acad Sci USA* **93**: 9839–9843.

Winther-Larsen, H.C., and Koomey, M. (2002) Transcriptional, chemosensory and cell-contact-dependent regulation of type IV pilus expression. *Curr Opin Microbiol* **5**: 173–178.

Xu, Q., and West, A.H. (1999) Conservation of structure and function among histidine-containing phosphotransfer (HPt) domains as revealed by the crystal structure of YPD1. *J Mol Biol* **292**: 1039–1050.

Zakataeva, N.P., Aleshin, V.V., Tokmakova, I.L., Troshin, P.V., and Livshits, V.A. (1999) The novel transmembrane *Escherichia coli* proteins involved in the amino acid efflux. *FEBS Lett* **452**: 228–232.

Zhou, H., and Dahlquist, F.W. (1997) Phosphotransfer site of the chemotaxis-specific protein kinase CheA as revealed by NMR. *Biochemistry* **36**: 699–710.
