
FEMS Microbiology Letters 234 (2004) 43–49

**Molecular characterization and analysis of the operon encoding the antifungal lipopeptide bacillomycin D**

Anne-Laure Moyne ${}^{a,1}$, Thomas E. Cleveland ${}^{b}$, Sadik Tuzun ${}^{a,*}$

${}^{a}$ Department of Entomology and Plant Pathology, 209 Life Sciences Bldg., Auburn University, Auburn, AL 36849, USA  
${}^{b}$ USDA/ARS, Southern Regional Research Center, 1100 Robert E. Lee Blvd., New Orleans, LA 70179, USA  

Received 12 January 2004; accepted 4 March 2004  
First published online 18 March 2004  

---

### Abstract

*Bacillus subtilis* AU195 produces bacillomycin D, a cyclic lipopeptide that is an inhibitor of the aflatoxin producing fungus *Aspergillus flavus*. Sequence analysis of the bacillomycin D operon revealed four ORFs with the structural organization of the peptide synthetases. Disruption of ORF 2, which links the amino acid moiety to the β-amino fatty acid, resulted in the loss of antifungal activity. By comparing the sequence of bacillomycin D, iturin A and mycosubtilin operons, our results showed that intergenic module replacement have occurred between *B.* subtilis lipopeptide synthetases including the iturin family and the plipastatin and fengycin family.

© 2004 Federation of European Microbiological Societies. Published by Elsevier B.V. All rights reserved.

#### Keywords: Bacillus subtilis; Non-ribosomal peptide synthetase; Iturin; Antifungal lipopeptide; Gene inactivation

---

### 1. Introduction

*Bacillus* species have the interesting property of producing secondary metabolites with a wide spectrum of antibiotic activity and very diverse structures. Therefore, they have been very valuable for medical and agricultural applications. *B.* subtilis strains produce gene-encoded antibiotics as well as a variety of small antibiotic peptides (<2000 Da) synthesized non-ribosomally. They include lipopeptide antibiotics such as iturin [1], surfactin [2], fengycin [3] and plipastatin [4].

To develop new biological-based technologies for controlling aflatoxin-producing fungi such as *Aspergillus flavus* and *Aspergillus parasiticus*, we have isolated *B.* subtilis strain AU195 that shows high antagonist activity against *A.* *flavus* [5]. The antifungal activity of AU195 was attributed to production of two analogs of bacillomycin D that differed only by their lipid moiety.

---

Bacillomycin D belongs to the iturin family that comprises iturin A [6], C, D and E [7], bacillomycin F [8] and L [9], bacillopeptin [10] and mycosubtilin [11]. Bacillomycin D has the following amino acid sequence L-Asn-D-Tyr-D-Asn-L-Pro-L-Glu-D-Ser-L-Thr and shared the common amino acid sequence Asx-Tyr-Asx with the other iturin members.

The first evidence for bacillomycin D enzymatic synthesis through activation of amino acids was provided by Besson et Michel [12] who purified an enzyme fraction which catalyzed the adenylation of amino acids present in the lipopeptide. Recently, the mycosubtilin operon is the first operon encoding an iturin lipopeptide that was completely sequenced [13]. Iturin synthesis is accomplished via a non-ribosomal biosynthetic pathway through a large multienzyme complex, the peptide synthetases (NRPSs). NRPSs have a modular structure in which each module of approximately 1000 amino acids, catalyzed the incorporation of one amino acid in the peptide product [14]. The order of the modules corresponds to the order in which the amino acids are incorporated in the peptide product. Modules can be further subdivided into adenylation (A), thiolation (T)

---

* Corresponding author. Fax: +1-334-844-1947.
* E-mail address: tuzuns@auburn.edu (S. Tuzun).
* Present address: Department of Plant Pathology, One Shields Avenue, University of California, Davis, CA 95616, USA.

0378-1097/$22.00 © 2004 Federation of European Microbiological Societies. Published by Elsevier B.V. All rights reserved.  
doi:10.1016/j.femsle.2004.03.011

and condensation (C) domains. The adenylation domain recognizes and activates the amino acid, which is then linked to the 4′ phosphopantetheinyl cofactor attached to the thiolation domain. Subsequently, the condensation domain located between two activating domains binds two consecutive amino acids during the elongation of the peptide product [15,16]. The mycosubtilin operon shares the same modular organization as other peptide synthetase gene cluster, but additionally contains a fatty acid enzymatic domain along with a peptide enzymatic domain to form a hybrid synthetase in the second ORF *mycA* [13]. Another member of the iturin family, the iturin A synthetase operon, has been completely sequenced [17] and shown to share the same organization as the mycosubtilin operon.

In this report we describe the cloning and organization of the bacillomycin D operon that encodes a potent antifungal lipopeptide against *A. flavus*. Comparison between the adenylation and condensation domain of bacillomycin D, mycosubtilin and iturin A synthetases suggested that an intergenic module replacement has occurred between all *B. subtilis* lipopeptide synthetases including the iturin family and the plipastatin and fengycin family.

## 2. Materials and methods

### 2.1. Bacterial strains and media

*Bacillus subtilis* AU195 described previously [5] was deposited at the ATCC under the accession number PTA-1767. *E. coli* DH5α (Stratagene) was used in all cloning experiments. Both bacteria were cultured in Luria–Bertani (LB). Antibiotics were added to the media at the following concentration: ampicillin 100 μg ml⁻¹ and erythromycin 3 μg ml⁻¹.

### 2.2. DNA manipulation

Total genomic DNA from *B. subtilis* AU195 was purified according to a lysozyme treatment for 1 h followed by phenol–chloroform extraction procedure [18]. For PCR analysis, chromosomal DNA was isolated with the Wizard genomic DNA purification kit (Promega). Qiaprep Spin Miniprep kit (Qiagen) or CONCERT rapid Plasmid Miniprep System (Life Technologies) were used to isolate plasmid for sequencing or cloning. *E. coli* was transformed by electroporation. Southern hybridization and lambda DNA isolation were carried out as described [19].

### 2.3. Antifungal activity

Antifungal activity was evaluated using a disk plate diffusion assay as described previously [5].

### 2.4. Design and amplification of the homologous probe for screening the library

DNA fragment, encoding the first module (Asn) of the bacillomycin D synthetase, was amplified by PCR from chromosomal DNA of *B. subtilis* AU195 using Taq polymerase (Fisher). Primers were designed according to the sequence of the first module of the mycosubtilin synthetase, which catalyzed the incorporation of the first asparagine into mycosubtilin. The sequences of the forward and reverse primer were: Asn1Fw ATTGAGTGGGCGAAAGAGG and Asn1Rv CTCKGATTTTGACTTGTTCATC, respectively. PCR products were purified using the “QIAquick-spin” PCR purification kit (Qiagen) and ligated in the PGEM-T-easy vector using the pGEM-Teasy kit (Promega).

### 2.5. Phage library construction and screening

AU195 genomic DNA was digested partially with Sau3A1 and fragments were size-fractionated in a 10–40% (w/v) sucrose gradient [19]. DNA fractions enriched in 15–20 kb fragments were ligated into the *BamH1* site of the vector lambda GEM-11 (Promega). The recombinant DNA was packaged in vitro with Packagene Extract provided by Promega following the manufacturer's instructions. The first screening of the genomic library was conducted with the probe designed according to the sequence of the first module of the mycosubtilin synthetase (Figs. 1 and 2). Plaques were transferred on nitrocellulose according to Promega’s manual. Probes were labeled with the PCR DIG probe synthesis kit (Roche). Hybridization and detection were performed as described in the manual provided with the DIG-high prime DNA labeling and detection Starter kit I (Roche). Three positive recombinant phages (λ5.2, λ2.2 and λ4.2) with overlapping insert were selected and completely sequenced (Fig. 1). To obtain the full length operon, two

![Figure 1](#fig1)

Fig. 1. Bacillomycin D operon and the adjacent DNA region including the two-component response regulatory genes. (A) Organization of the gene cluster. *bamD*, *bamA*, *bamB* and *bamC* encode the peptide synthetases involved in bacillomycin D production. The arrows indicate the direction of the transcription. (B) Position of the phage insert sequenced. Terminator (■) and probe (□).

A.-L. Moyne et al. / FEMS Microbiology Letters 234 (2004) 43–49

Mycosubtilin  
FenF MycA MycB Myc  
$\square \blacksquare \square \square \square$ Asn || $\square$ Tyr | $\square$ Asn | $\square$ Gln | $\square$ Pro | $\square$ Ser | $\square$ Asn |

Bacillomycin D Operon  
BamD BamA BamB BamC  
$\square \blacksquare \square \square \square$ Asn || $\square$ Tyr | $\square$ Asn | $\square$ Pro | $\square$ Glu | $\square$ Ser | $\square$ Thr |

Iturin operon  
IuD IuA IuB IuC  
$\square \blacksquare \square \square \square$ Asn || $\square$ Tyr | $\square$ Asn | $\square$ Gln | $\square$ Pro | $\square$ Asn | $\square$ Ser |

$\square$ Malonyl-CoA transacylase  
$\blacksquare$ Acyl-CoA ligase domain  
$\square$ Acyl carrier protein domain  
$\square$ $\beta$-keto acyl synthetase domain  
$\square$ Amino transferase domain  

$\square \square \square$ Thiolation domain  
$\blacksquare$ Adenylation domain  
$\square$ Condensation domain  
$\square \square \square$ Epimerization domain  
$\square \square \square$ Thioesterase domain  

Fig. 2. Schematic diagram of the mycosubtilin operon from Duitmann et al. [13] and the bacillomycin D operon. The arrows indicate positions of the primers used to amplify the asparagine domain of BamA. Insertion position for pMUTIN4 is indicated by a double line.

supplementary screenings of the genomic library were necessary with probes amplified by PCR and designed using the sequence located at the end of the recombinant lambda phage already sequenced (Fig. 1).

### 2.6. DNA sequencing

Recombinant phages were digested with different restriction enzymes and subcloned in pBluescript KS (+) (Stratagene). All subclones were sequenced with a LiCor DNA 4200 automatic sequencer. Bi-directional cycle sequencing reactions were conducted using the SequiTherm EXEL™ II DNA Sequencing Kit (Epicentre) and a combination of IRD800 reverse M13 and IRD700 forward M13-labeled primers. Gaps between clones were filled by PCR amplification with Herculase (Stratagene). The bacillomycin D operon sequence has been submitted to the GenBank database under Accession No. AY137375.

DNA analyses and translation were conducted with the BCM Search Launcher at the Baylor College of Medicine ([http://searchlauncher.bcm.tmc.edu/](http://searchlauncher.bcm.tmc.edu/)), Jellyfish ([http://www.biowire.com/](http://www.biowire.com/)) and GCG Wisconsin package. Contig sequences were assembled with Cap [20] through the BCM Search Launcher. Homology searches were performed with BLAST ([http://www.ncbi.nlm.nih.gov/BLAST](http://www.ncbi.nlm.nih.gov/BLAST)).

### 2.7. Disruption of bacillomycin D synthetase B

The recombinant phage λ2.1 was digested with HindIII. One generated fragment, a 2.5 kb sequence (Fig. 2),

was ligated in the HindIII site of pMUTIN4 [21]. Bacillus protoplasts were transformed using the protocol described by Chang and Cohen [22]. After selection of the mutants, presence of the pMUTIN4 in the AU195 chromosomal DNA was confirmed by PCR (see results) using the Hotstart Herculase (Stratagene) and the following primer set 486F CTGGCGTAATAGCGAA-GAGG and 1392R CAGCAGCAGACCATTTC. The amplified fragment corresponded to the beginning of the β-D-galactosidase gene from base pair 486 to 1392. The same primer set was used to generate the PCR probe with the PCR DIG probe synthesis kit (Roche) in southern analyses.

### 2.8. Phylogenetic analyses

Amino acid adenylation domain sequence were delineated as the amino acid sequence comprised between the highly conserved core motifs A1–A10 and the amino acid condensation domain as the amino acid sequence comprised between the core motifs C1 and C7 as defined by Mahariel et al. [15]. Sequences were aligned with CLUSTALW and the distance tree was generated by TreeView [23]. Distances are expressed as the numbers of substitutions per amino acid.

## 3. Results and discussion

### 3.1. Cloning and sequencing of bacillomycin D operon

The genes encoding the enzymes responsible of the mycosubtilin biosynthesis have been cloned recently [13]. Mycosubtilin and bacillomycin D share the common amino acid sequence Asn-Tyr-Asn linked to a β-amino fatty acid. Seven amino acid-activating modules have been identified in the mycosubtilin operon [13]: one in the second ORF, *mycA*, four in the third ORF, *mycB*, and two in the fourth ORF, *mycC*. We hypothesized that the DNA sequence of *mycA* and the first two modules of *mycB*, which encode for the lipid moiety and the amino acid Asn-Tyr-Asn, were highly conserved in the bacillomycin D synthetases. To clone the bacillomycin D operon, we designed primers, according to DNA sequences encoding the first module of *mycA* and amplified AU195 DNA by polymerase chain reaction (PCR) (Fig. 2). A 700 bp fragment was amplified by PCR, cloned and sequenced. The DNA fragment had 80% homology with the corresponding sequence of the first module of *mycA* sequence. The PCR amplified DNA sequence was used to screen a genomic DNA library of *B. subtilis* AU195. Overlapping inserts from 5λ recombinant clones, representing 46,513 bp, were completely sequenced to map the entire bacillomycin D operon (37,844 bp) and the adjacent regions (Fig. 1).

3.2. Disruption of the bacillomycin D operon

Inactivation of the second ORF *bamA*, encoding bacillomycin D synthetase A, was achieved by insertion of pMUTIN4 [21] via a single cross-over event. A 2.5 kb DNA fragment, part of the acylcarrier (ACP) and aminotransferase (AMT) domain of the second ORF *bamA* (Fig. 2), was obtained after *HindIII* digestion of a recombinant lambda phage DNA and cloned into the *HindIII* restriction site of pMUTIN4. The recombinant plasmid was transferred into *B. subtilis* AU195 by protoplast transformation. Because pMUTIN4 cannot replicate in *B. subtilis*, resistance to erythromycin should be acquired only by integration of the recombinant plasmid into the bacterial chromosome. Five erythromycin resistant colonies T1–T5 were recovered after selection of the transformants on LB with 3 μg ml⁻¹ and a 925 bp fragment of pMUTIN4 was amplified only in the mutant T4 (Fig. 3(A)). After southern hybridization of chromosomal DNA digested with *BamH1*, the pMUTIN4 PCR probe detected a 9.7 kb DNA fragment only from the mutant T4 (Fig. 3(B)). This fragment corresponds to the expected size of the 8610 bp of pMUTIN4 and a 1094 bp sequence of the bacillomycin D sequence, thus confirming that one copy of the complete pMUTIN4 vector was integrated into T4. Using primers flanking the expected integration site of pMUTIN4, we amplified two DNA sequences and their sequences matched the expected junction of pMUTIN4 and bacillomycin D synthetase A. We previously demonstrated that bacillomycin D was entirely responsible for the AU195 antifungal activity against *A. flavus* [5]. At each step of the purification, all fractions were tested for antifungal activity, and only two homologs of ba-

![Figure 3](https://i.imgur.com/yourimageurl.png)

Fig. 3. Presence of the pMUTIN4 insertion in AU195 genomic DNA. (A) PCR analysis of the mutants. Chromosomal DNA of the mutants, T1–T5, and the wild type genotype AU195 was amplified by PCR using primers designed to amplify a 925 bp fragment of pMUTIN4. (B) Southern blot analysis of the mutants. Chromosomal DNA (1 μg) of T4 and AU195 was digested with *BamH1* and analyzed by Southern blotting with a digoxigenin PCR-labeled probe of the 925 bp fragment of pMUTIN4. The 9.7 kb fragment detected is in agreement with the addition of the 8610 bp of pMUTIN4 and a 1094 bp sequence of the bacillomycin D sequence.

![Figure 4](https://i.imgur.com/yourimageurl.png)

Fig. 4. Comparison of the antifungal activity of AU195 strain and T4 mutant. A disk plate diffusion assay was used to evaluate the antifungal activity of the bacterial culture (A) and the culture filtrate (CF) against *A. flavus* (B). The antifungal inhibition zone was observed after 3 days (A) and 2 days (B). The T4 mutant was obtained after integration of the pMUTIN4 into AU195 genomic DNA.

cilomycin D, differing by their lipid chain, were able to inhibit *A. flavus* in vitro. Therefore we compared the antifungal activity of all the mutants against *A. flavus* and confirmed that only the transformant T4 completely lost ability to produce the antifungal activity (Fig. 4).

3.3. Organization of the bacillomycin D operon

Within the investigated sequence, four ORFs, designated as *bamD*, *bamA*, *bamB* and *bamC*, were transcribed in the same direction and delimited by two putative transcriptional terminators (Figs. 1 and 2). A putative ribosomal binding site was identified in each intergenic region. The first ORF, *bamD*, encodes a 400 amino acids protein with a predicted mass of 45 kDa. The protein BamD, has exactly the same amino acid sequence as Fen F, encoded by the mycosubtilin operon [13] except that the last 26 amino acids of Fen F were absent. BamD differed from ItuD, encoded by the iturin A operon [4], by only six amino acids dispersed in the sequence. The strong similarity between BamD, FenF and ItuD indicated that it belongs to the malonyl-CoA transacylase family as suggested by Duitman et al. [13]. The second ORF, *bamA*, codes for a protein with 3980 amino acids and a predicted mass of 448 kDa. BamA amino acid sequence is nearly identical to that of ItuA (98%) and shares close homology to MycA (78%). The four different domains, AL, ACP, KS and AMT, described in MycA [13] are present and responsible for the integration of the β-amino fatty acid moiety into the bacillomycin D. The third ORF, *bamB*, codes for a protein of 5363 amino acid with a predicted mass of 607 kDa. BamB exhibits four amino acid activating modules, each of them having the characteristic conserved condensation, adenylation and thiolation domains. Additionally, an epimerization domain is integrated between T and C domains in the first and second modules. Amino acid sequence alignment revealed that

BamB exhibits 81% identity with ItuB, 70% with Myc. The fourth ORF, *bamC*, codes for a protein with 2619 amino acids and a predicted mass of 299.5 kDa. Two amino acid activating modules can be distinguished and an epimerization domain is included in the first module. The last domain of BamC contains the GxSxG motif characteristic of thioesterase domain (TE) required for cyclization and release of the peptide product from the synthetase [24]. BamC amino acid sequence has 74% similarity with MycC and 68% with ItuC. By comparing the amino acid sequence of bacillomycin D with mycosubtilin, four modules were identified to catalyse the incorporation of Asn, Tyr, Asn and Ser in the lipopeptide product (Fig. 2). The amino acids activated by the three unknown adenylations domains of the bacillomycin D operon were characterized with the method described by Challis et al. ([http://raynam.chm.jhu.edu/~nrps/](http://raynam.chm.jhu.edu/~nrps/)) [25]. By aligning the unknown adenylations domains of the bacillomycin D operon with the A-domain from the gramicidin synthetase GrsA, the predicted activated amino acids were confirmed to be Glu, Pro and Thr.

### 3.4. Origin of the different modules

The β-amino fatty acid linked to the amino acid sequence Asn-Tyr-Asn is a common characteristic of the iturin group; therefore we compared the DNA sequences of BamD, BamA and the two modules encoding the activation of Tyr and Asn in BamB with the corresponding DNA sequence of the iturin A and mycosubtilin operons (Fig. 2). In this region of the operon, bacillomycin D is 98% identical to iturin A and 79% to mycosubtilin. The upstream region of the bacillomycin D operon matches the DNA sequence upstream of the iturin A operon [17]. The *yxjF* gene and the putative rho-independent terminator are located in the same position and the bacillomycin D operon promoter sequence is strictly identical to the promoter sequence of the iturin A operon. On the contrary, the BamC amino acid sequence shares more homology with the mycosubtilin operon (74%) than the iturin A operon (67%).

Bacillomycin D differed from mycosubtilin by replacement of Gln by Pro, Pro by Glu and Asn by Thr (Fig. 2). A Blast search revealed that the proline module of BamB has the highest homology with the proline module of PpsD (57%), plipastatin synthetase of *B. subtilis* 168, and fengycin synthetase FenA (56%). Both the adenylation and the condensation domain of BamB proline module were significantly different (47% homology) from that of *ituB* and *mycB* suggesting a different origin for this module. An horizontal transfer of the entire Pro module is probably responsible for the reorganization of BamB. The Glu module of BamB has the highest homology (62%) with the Gln module from *ItuB* and the Thr module of BamB with the Thr module of PpsB (50%). Intragenic swapping between two adenylation domains has been proposed to be the mechanism of reorganization of the Ser and Asn module in *ituC* [17]. Our result revealed that intergenic module replacement has occurred between all *B. subtilis* lipopeptide synthetases including the iturin family and the plipastatin and fengycin family. In the bacillomycin D operon, BamD, BamA and BamB have a high homology with ItuD, ItuA and ItuB and BamC with MycC. The Pro and Thr modules have a common origin with the Pro and Thr modules of the plipastatin operon.

The adenylation domain recognizes and activates the corresponding amino acid to be incorporated into the lipopeptide. Phylogenetic analysis of the adenylation domains of bacillomycin D, iturin A and mycosubtilin,

![Figure 5](#fig:5)

Fig. 5. Phylogenetic analysis of the adenylation (A) and condensation (B) domain of bacillomycin D, iturin A and mycosubtilin. The adenylation sequence comprises the highly conserved core motifs A1–A10 and the condensation sequence comprises the region between C1 and C7 [15]. Each adenylation or condensation domain is designed by the corresponding binding amino acid and peptide synthetase (for example Pro-ItuB correspond to the proline binding domain of ItuB). Distances are expressed as the number of substitution per amino acid.

confirmed that each adenylation domain of bacillomycin D was homolog to the corresponding adenylation domain of mycosubtilin and iturin A. Glu domain is most similar to Gln domain and that Thr domain is closer to the Ser domain (Fig. 5(A)). Although the condensation domain, responsible for peptide bond catalysis, has been shown to have high selectivity at the acceptor residue [26], the amino acid sequences of the condensation domain are conserved between domains occupying the same position in the protein as revealed by the phylogenetic tree (Fig. 5(B)).

### Conclusion

In this study, we cloned, identified and characterized the operon encoding the bacillomycin D synthetases in *B. subtilis* AU195. Genetic disruption of the operon completely abolished the antifungal activity of AU195 against *A. flavus*, confirming that bacillomycin D was the only antagonist peptide produced by AU195 in our culture condition. Bacillomycin D operon spans 37.8 kb with four ORFs: *bamD*, *bamA*, *bamB* and *bamC* and shares the structural organization of the Mycosubtilin and Iturin A operon.

The colinearity between the modular organization of NRPSs and the sequence of their peptide products has led to the prospect of designing NRPSs to synthesize new bioactive peptides. Comparison of the different NRPSs responsible for lipopeptide synthesis in *B. subtilis*, will facilitate our approach for designing new NRPSs. As it has already been described for the polyketide synthase [27], we will be able to engineer combinatorial biosynthesis of antimicrobial products in bacteria, thus increasing the potential effectiveness of biological control organisms to control plant diseases.

### Acknowledgements

This work was supported by a grant from the Peanut Foundation and funding from USDA-ARS (Cooperative Agreement #58-6435-8-066).

### References

[1] Maget-Dana, R. and Peypoux, F. (1994) Iturins, a special class of pore-forming lipopeptides: biological and physicochemical properties. Toxicology 87, 151–174.

[2] Kluge, B., Vater, J., Salnikow, J. and Eckart, K. (1988) Studies on the biosynthesis of surfactin, a lipopeptide antibiotic from *Bacillus subtilis* ATCC 21332. FEBS Lett. 231, 107–110.

[3] Vanittanakom, N., Loeffler, W., Koch, U. and Jung, G. (1986) Fengycin – a novel antifungal lipopeptide antibiotic produced by *Bacillus subtilis* F-29-3. J. Antibiot. 39, 888–901.

[4] Tsuge, K., Ano, T. and Shoda, M. (1996) Isolation of a gene essential for biosynthesis of the lipopeptide antibiotics plipastatin B1 and surfactin in *Bacillus subtilis* YB8. Arch. Microbiol. 165, 243–251.

[5] Moyne, A.L., Shelby, R., Cleveland, T.E. and Tuzun, S. (2001) Bacillomycin D: an iturin with antifungal activity against *Aspergillus flavus*. J. Appl. Microbiol. 90, 622–629.

[6] Garbay-Jaureguiberry, C., Roques, B.P., Delcambe, L., Peypoux, F. and Michel, G. (1978) NMR conformational study of iturin A, an antibiotic from *Bacillus subtilis*. FEBS Lett. 93, 151–156.

[7] Besson, F. and Michel, G. (1987) Isolation and characterization of new iturins: iturin D and iturin E. J. Antibiot. 40, 437–442.

[8] Peypoux, F., Marion, D., Maget-Dana, R., Ptak, M., Das, B.C. and Michel, G. (1985) Structure of bacillomycin F, a new peptidolipid antibiotic of the iturin group. Eur. J. Biochem. 153, 335–340.

[9] Peypoux, F., Pommier, M.T., Das, B.C., Besson, F., Delcambe, L. and Michel, G. (1984) Structures of bacillomycin D and bacilomyacin L peptidolipid antibiotics from *Bacillus subtilis*. J. Antibiot. 37, 1600–1604.

[10] Kajimura, Y., Sugiyama, M. and Kaneda, M. (1995) Bacillopeptins, new cyclic lipopeptide antibiotics from *Bacillus subtilis* FR-2. J. Antibiot. 48, 1095–1103.

[11] Peypoux, F., Pommier, M.T., Marion, D., Ptak, M., Das, B.C. and Michel, G. (1986) Revised structure of mycosubtilin, a peptidolipid antibiotic from *Bacillus subtilis*. J. Antibiot. 39, 636–641.

[12] Besson, F. and Michel, G. (1992) Biosynthesis of bacillomycin D by *Bacillus subtilis*. Evidence for amino acid-activating enzymes by the use of affinity chromatography. FEBS Lett. 308, 18–21.

[13] Duitman, E.H. et al. (1999) The mycosubtilin synthetase of *Bacillus subtilis* ATCC6633: a multifunctional hybrid between a peptide synthetase, an amino transferase, and a fatty acid synthase. Proc.Natl. Acad. Sci. USA 96, 13294–13299.

[14] Stachelhaus, T. and Marahiel, M.A. (1995) Modular structure of genes encoding multifunctional peptide synthetases required for non-ribosomal peptide synthesis. FEMS Microbiol. Lett. 125, 3–14.

[15] Marahiel, M.A., Stachelhaus, T. and Mootz, H.D. (1997) Modular peptide synthetases involved in nonribosomal peptide synthesis. Chem. Rev. 97, 2651–2673.

[16] Moffitt, M.C. and Neilan, B.A. (2000) The expansion of mechanistic and organismic diversity associated with non-ribosomal peptides. FEMS Microbiol. Lett. 191, 159–167.

[17] Tsuge, K., Akiyama, T. and Shoda, M. (2001) Cloning, sequencing and characterization of the iturin A operon. J. Bacteriol. 183, 6265–6273.

[18] Wang, S.Y., Wu, S.J., Thottappilly, G., Locy, R.D. and Singh, N.K. (2001) Molecular cloning and structural analysis of the gene encoding *Bacillus cereus* exochitinase Chi36. J. Biosci. Bioeng. 92, 59–66.

[19] Sambrook, J., Fritsch, E.F. and Maniatis, T. (1989) In: Molecular Cloning (Harbor, C.S., Ed.). Cold Spring Harbor Laboratory Press, New York.

[20] Huang, X. (1992) A contig assembly program based on sensitive detection of fragment overlaps. Genomics.

[21] Vagner, V., Dervyn, E. and Ekrlich, S.D. (1998) A vector for systemic gene inactivation in *Bacillus subtilis*. Microbiology 144, 3046–3104.

[22] Chang, S. and Cohen, S.N. (1979) High frequency transformation of *Bacillus subtilis* protoplast by plasmid DNA. Mol. Gen. Genet. 168, 111–115.

[23] Page, R.D.M. (1996) TREEVIEW: an application to display phylogenetic trees on personal computers. Comput. Appl. Biosci. 12, 357–358.

[24] Trauger, J.W., Kohll, R.M., Mootz, H.D., Mahariel, M.A. and Walsh, C.T. (2000) Peptide cyclization catalyzed by the thioesterase domain of tyrocidine synthetase. Nature 407, 215–218.

[25] Challis, G.L., Ravel, J. and Townsend, C.A. (2000) Predictive, structure-based model of amino acid recognition by nonribosomal peptide synthetase adenylation domains. Chem. Biol. 7, 211–224.

[26] Belshaw, P.J., Walsh, C.T. and Stachelhaus, T. (1999) Aminoacyl-CoAs as probes of condensation domain selectivity in nonribosomal peptide synthesis. Science 284, 486–489.

[27] Gokhale, R., Tsuji, S.Y., Cane, D.E. and Khosla, C. (1999) Dissecting and exploiting intermodular communication in polyketide synthases. Science 284, 482–485.
