
Available online at www.sciencedirect.com

SCIENCE DIRECT®

Experimental Cell Research 296 (2004) 43–50

**Review**

**Ribosome biogenesis: of knobs and RNA processing**

Sander Granneman${}^{a}$ and Susan J. Baserga${}^{a,b,c,*}$

${}^{a}$Department of Molecular Biophysics and Biochemistry, Yale University School of Medicine, New Haven, CT 06520, USA  
${}^{b}$Department of Genetics, Yale University School of Medicine, New Haven, CT 06520, USA  
${}^{c}$Department of Therapeutic Radiology, Yale University School of Medicine, New Haven, CT 06520, USA  

Received 13 January 2004  
Available online 15 April 2004

---

### Abstract

The synthesis of ribosomes in eukaryotes involves processing of pre-ribosomal RNA (pre-rRNA) and sequential assembly of a large number of ribosomal proteins on the rRNAs. Although we have gained tremendous insights into the processing of pre-rRNA intermediates in the last three decades, little was known about the dynamic nature of ribosome biogenesis. Only recently the development of efficient affinity-purification procedures and mass-spectrometry techniques has allowed the isolation of large pre-ribosomal complexes, which led to the identification of several ribosome assembly intermediates and a large number of novel ribosome assembly factors. In this mini-review, we summarize some of the discoveries that have been made in the field of ribosome biogenesis in the past 30 years and highlight some key aspects about what remains to be learned.

© 2004 Elsevier Inc. All rights reserved.

**Keywords:** SSU processome; 90S pre-ribosome; Pre-rRNA processing; U3 snoRNA; TAP affinity purification; Proteomics

---

### Pre-rRNA modification and processing in *Saccharomyces cerevisiae*

Processing of ribosomal RNA (rRNA) in eukaryotes takes place, for the most part, in a nuclear sub-compartment called the nucleolus (reviewed in Refs. [20,30,58]). Here, RNA polymerase I generates a large polycistronic precursor rRNA (pre-rRNA) that contains the sequences for the mature ribosomal RNAs (18, 5.8S, 25–28S rRNA), two external transcribed spacers (ETS) and two internal transcribed spacers (ITS) (Fig. 1). This primary transcript is subsequently chemically modified at numerous sites and subjected to many endo- and exonucleolytic-processing steps to produce the mature rRNAs. The fourth ribosomal RNA, 5S rRNA, is independently transcribed as a precursor by RNA polymerase III.

Pre-rRNA processing has been most extensively studied in the yeast *S. cerevisiae* (reviewed in Refs. [30,58]). In this organism, the earliest detectable 35S pre-rRNA is cleaved at sites A${}_{0}$, A${}_{1}$ (5′ ETS) and A${}_{2}$ (ITS1) (Fig. 1), which removes the 5′ external transcribed spacer (5′ ETS) and separates the 18S precursor (20S) from the 25S and 5.8S precursor (27S). The 27SA${}_{2}$ pre-rRNA can be cleaved at site A${}_{3}$ by the RNase MRP endonuclease, although this cleavage is not essential for ribosome biogenesis. The 20S pre-rRNA is subsequently transported to the cytoplasm where it is cleaved at site D, yielding the mature 18S rRNA. The 27SA pre-rRNA undergoes several processing steps to yield the mature 5.8S and 25S rRNAs (Fig. 1).

Many chemical modification steps in the pre-rRNA are guided by small nucleolar RNAs (snoRNAs) that function by correctly positioning the modification enzymes on the pre-rRNA by base-pairing with the target site. These snoRNAs exist as RNP complexes in the cell, the snoRNPs. Most of the chemical modifications involve 2′-O-methylation of the sugar moiety and pseudouridylation of uridine residues [3,8,15,29]. On the basis of associated proteins and conserved RNA sequence elements, the snoRNAs can be divided into two major classes: the box C/D and box H/ACA snoRNAs. Box C/D snoRNAs are involved in 2′-O-methylation of the pre-rRNA and are associated with the common box C/D snoRNP proteins Nop1p (fibrillarin in human), Nop56p, Nop58p, and Snul3p (15.5K in human).

---

* Corresponding author. Molecular Biophysics and Biochemistry Department, Yale University School of Medicine, 333 Cedar Street, SHM C-114, New Haven, CT 06520-8024. Fax: +1-203-785-6404.  
E-mail address: susan.baserga@yale.edu (S.J. Baserga).

0014-4827/$ - see front matter © 2004 Elsevier Inc. All rights reserved.  
doi:10.1016/j.yexcr.2004.03.016

S. Granneman, S.J. Baserga / Experimental Cell Research 296 (2004) 43–50

snR30 which are all involved in 18S rRNA synthesis [11,25,35,47,50,51,58]. Although the U3 snoRNA is associated with all of the common box C/D snoRNP proteins, it is not a $2'$-$O$-methylation guide snoRNA. The U3 snoRNA base pairs with regions in the $5'$ ETS and the $5'$ terminus of the 18S rRNA to facilitate cleavages at sites $A_0$, $A_1$ and $A_2$ and therefore pre-18S rRNA processing (Fig. 1).

Early identification of pre-ribosomal complexes in eukaryotic cells

In 1969, Miller and Beatty found by analyzing electron micrographs of actively transcribing rDNA genes that the $5'$ ends of nascent pre-rRNA transcripts in Xenopus oocytes are decorated with highly condensed terminal knobs [38]. Similar structures were later reported in many eukaryotic organisms; however, the biochemical nature and the function of these complexes still remained largely unclear. The Sollner–Webb laboratory provided the first clues in the early 1990s, when they proposed that the terminal knobs corresponded to an assembly of pre-rRNA processing complexes initially identified in mouse cell extracts [26,27, 39,40]. Depletion of the U3 snoRNA from mouse extracts abolished the processing activity, which suggested that this snoRNA is an integral component of the processing complex [27].

Several years after Miller and Beatty published their work, it was shown by metabolic labeling and density gradient sedimentation analyses that HeLa and yeast cells package the primary pre-rRNA transcript into large 80–90S complexes, reminiscent of the size of terminal knobs. These large complexes were thought to represent “pre-ribosomes” because they contained the 35S pre-rRNA transcript in addition to ribosomal proteins [31,52,54,55,60]. Pioneer work performed in the Warner and Planta laboratories further revealed that in yeast, 90S pre-ribosomes are converted into 43S and 66S particles, precursors to the 40S and 60S ribosomal subunits, respectively [52,54,55]. The 43S pre-ribosomes are subsequently exported to the cytoplasm where they are matured to 40S, whereas the 66S pre-ribosomes undergo several maturation steps in the nucleus before they are exported to the cytoplasm.

Nop1p/fibrillarin is the enzyme that catalyzes the methylation reaction. Box H/ACA snoRNAs are characterized by the conserved boxes H and ACA and are associated with the core proteins Nhp2p, Nop10p, Gar1p and Cbf5p (dyskerin in human), of which the latter is thought to catalyze the isomerization of the uridine residue [29]. The role of these modifications is not exactly clear; however, they can promote base-pairing interactions and influence the activity of ribosomes [8,9,28,36].

A subset of snoRNAs has an independent role in processing of pre-rRNA. These include the box C/D snoRNAs U3 and U14 and the box H/ACA snoRNAs snR10 and

The era of proteomics

Because the yeast system is genetically relatively easy to manipulate, this organism is widely used as a model system to study ribosome biogenesis. Using classic yeast genetics, many trans-acting factors were identified in the last three decades and a detailed map of the pre-rRNA processing pathway has been established (reviewed in Refs. [30,58]). Many factors were identified in screens for proteins that could alleviate ribosome biogenesis defects in mutant strains or, vice versa, when mutated caused synthetic lethality.

Despite the importance and enormous success of these genetic approaches, they provided only a glimpse of the complexity of pre-rRNA processing and the dynamics of the ribosome assembly pathway. A breakthrough came with the development of a tandem affinity-purification method (TAP) that allowed isolation of particles under native conditions [44]. Combined with significant improvements in the sensitivity of mass spectrometry, a plethora of novel trans-acting factors has been identified and the protein composition of several yeast ribosome assembly intermediates has

90S pre-ribosome (Planta/Warner)

|  |  |
| --- | --- |
| 90S | U3 18S 25S 35S |
| Endonuclease? Cleavage at A₀ first | pseudouridylation/ 2'-O-methylation? |
| ? | Cleavage at A₃ first RNase MRP |

Nucleolus

SSU processome (Baserga) / 90S pre-ribosome (Hurt)

|  |  |
| --- | --- |
| 90SA₀ | U3 33S |
| Cleavage at A₁ Endonuclease? | pseudouridylation/ 2'-O-methylation? |
| ? | 90SA₃ 23S |

66S (27SA₂/27SA₃)

|  |  |
| --- | --- |
| 90SA₁ | U3 32S |
| Cleavage at A₂ Endonuclease? | + Export factors |
| U3 | U3 |

Nucleolus

|  |  |
| --- | --- |
| 43S | 20S |
| 43S | 20S |
| Export |  |

Cytoplasm

|  |  |
| --- | --- |
| 40S | 18S |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |  |
|  |been determined [5,10,12,18,21–23,37,42,46,48,53]. These studies demonstrated that ribosome biogenesis is far more complex and dynamic than was originally thought.

The first proteomic studies focused on the pre-60S complexes. Several groups published the isolation of a number of pre-60S particles that contained a different but overlapping protein composition and a number of pre-60S assembly intermediates were defined [5,12,22,42,45,46]. A rough 60S ribosomal subunit assembly pathway from the nucleolus to the cytoplasm was established (schematically outlined in Fig. 2; reviewed in Ref. [53]).

Comparative bioinformatics analyses of protein interaction data suggest that many additional uncharacterized protein-encoding genes play a role in pre-RNA processing. Microarray studies in yeast revealed that a large number of these proteins indeed have a role in ribosome biogenesis, many of which were not detected in previous proteomic studies [4,43,63].

Thus, many proteins remain to be characterized and their roles in pre-rRNA processing to be assigned.

The proteomic and functional genomic studies in yeast also greatly stimulated proteomic studies in mammalian cells although our understanding of ribosome assembly in humans is still lagging far behind (reviewed in Ref. [49]). Proteomic analysis of human nucleoli has identified many putative homologues of yeast ribosome assembly factors and a rough biochemical map of ribosome assembly in human cells has been established [2,32]. Furthermore, the affinity-purification procedures widely used in yeast are also applicable to mammalian cells; this will aid in the isolation of large pre-ribosomal complexes in humans [16].

### The SSU processome/90S pre-ribosomes

Two studies focused on the isolation of the 90S pre-ribosomal particles. Dragon et al. [10] isolated a large ~80S pre-ribosomal particle using two epitope-tagged proteins as baits. It contained a large number of previously known non-ribosomal proteins involved in 40S ribosomal subunit biogenesis, some small ribosomal proteins, the U3 snoRNA and 17 novel proteins (Utp1–17). This complex was named “small subunit (SSU) processome” because its associated proteins interact with the U3 snoRNA, which is required for processing at sites A₀–A₂ and are essential for the production of the 18S rRNA and thus synthesis of the 40S ribosomal subunit (SSU). Interestingly, the estimated molecular mass of the SSU processome (~2.2 MDa) is roughly the same as the measured size of terminal knobs in yeast [10]. Furthermore, it was shown that genetic depletion of SSU processome associated proteins and the U3 snoRNA leads to loss of terminal knobs at the 5′ end of nascent pre-rRNA transcripts, strongly suggesting that this complex corresponds to the terminal knobs described by Miller and Beatty in 1969 [10,38]. Following up on previous large-scale proteomic studies performed in yeast

[18,23], Grandi et al. [21] TAP tag-purified several 80–90S complexes that contained most of the SSU processome components plus some additional novel proteins. Primer extension analyses detected the 5′ ETS in the purified complexes, indicating that they contained the 35S pre-rRNA. On the basis of this observation and the estimated molecular mass of the purified complexes, it was proposed by the authors that these complexes corresponded to the 90S pre-ribosomes that were described in the early 1970s as the earliest pre-ribosomes. Hence, they named it the 90S pre-ribosome [52,55].

The predicted size of the SSU processome is somewhat smaller than the complexes reported by Grandi et al. and later by Schafer et al. [21,48], because it contained fewer components. It has been suggested that the SSU processome complex corresponds to an earlier complex or a 90S precursor that, reminiscent of the measured size of terminal knobs, co-transcriptionally assembles on the pre-rRNA [21,53]. Although this is a very plausible hypothesis, such a particle would be difficult to purify under the conditions used because it would still be tethered to the rDNA. The bulk of these particles would probably pellet in the insoluble fraction after centrifugation of the extracts. The differences in the affinity-purification procedures used may explain why the protein composition of the independently isolated particles only partially overlapped. Alternatively, the SSU processome might represent a late form of the 90S pre-ribosome (see below).

Another conclusion that can be drawn from the available proteomic data is that conversion of the SSU processome/90S pre-ribosome to 43S particles involves fewer intermediate assembly steps than formation of 60S ribosomal subunits. Some variation was found in the composition of purified 90S pre-ribosomal particles with different baits but it remains unclear whether this reflects an order of assembly [21]. Because the formation of 43S pre-ribosomes requires three processing steps in the 35S pre-rRNA (sites A₀, A₁ and A₂; Fig. 1), we predict that this, like 60S ribosomal subunit biogenesis, involves several assembly intermediates with different but largely overlapping protein compositions (Fig. 2). Unfortunately, the presence of 33S and 32S pre-rRNA in purified particles has not been investigated, making it difficult to determine an order of assembly.

### Has the real 90S pre-ribosome been isolated?

90S pre-ribosomes were defined as the earliest detectable pre-ribosomes that contained the 35S pre-rRNA transcript, ribosomal proteins and likely numerous ribosome assembly factors [52,55]. Because the 90S pre-ribosomes are precursors to 43S and 66S complexes, it was logical to assume that they would contain the factors required for their assembly and synthesis of both subunits. Unexpectedly, the purified SSU processome/90S pre-ribosome contained only very few ribosomal and non-ribosomal proteins required for the
S. Granneman, S.J. Baserga / Experimental Cell Research 296 (2004) 43–50

synthesis of the 60S ribosomal subunit [10,21,48]. The assignment of the RNP complex isolated by Grandi et al. as the 90S pre-ribosome described in the 1970s was based on primer extension analysis that demonstrated the presence of the 35S pre-rRNA and estimated sizes of the purified complexes [21]. Much to our surprise, our Northern analysis of immunoprecipitated pre-rRNA revealed that many SSU processome components preferentially associate with the 23S pre-rRNA, a precursor that contains all the U3 snoRNA-dependent cleavage sites (5′ETS-A3 fragment; Ref. [62]). Our results suggest that the bulk of the purified SSU processome/90S pre-ribosomes are associated with the 23S pre-rRNA and thus represent a later, not the earliest, form of 90S pre-ribosomes (Fig. 2). The 23S pre-rRNA has been labeled as an aberrant intermediate that is normally rapidly degraded by the exosome [1,58]. However, we and others have frequently detected this intermediate in Northern analysis of steady-state rRNA in wild-type yeast strains (e.g., see references Refs. [6,62]) and found by pulse-chase analysis that it is efficiently processed to 18S rRNA (Granneman and Baserga, unpublished results). Thus, it is likely that several 90S pre-ribosomes exist in the cell and that the most abundant or most stable form represents a complex that has already been separated from the large subunit pre-rRNAs by cleavage at site A₃ (see Fig. 2). It is important to note that primer extension analysis does not discriminate between the 35S and the 23S pre-rRNA and therefore this might have been overlooked in other studies.

As might be expected in the earliest 90S pre-ribosome, a number of pre-60S associated proteins have been shown to (weakly) associate with the 35S pre-rRNA (Brix1p, Rpf1p, Ssf1p and Nop7p; Refs. [12,22,62]). However, these proteins were not detected in the purified SSU processome/90S pre-ribosomes. Moreover, several pre-60S associated proteins also sediment at ~80–90S in density gradients (Nsa3p, Nop7p, Ssf1p, Rix7p; Refs. [12,22,42]). This suggests that these proteins are also part of other larger pre-ribosomes, perhaps a 90S pre-ribosomal complex that contains a larger number of 60S assembly factors than the particles recently described [10,21,48].

Taken together, these results raise the question whether the purified complexes correspond to “the” 90S pre-ribosomes that were described in the early 1970s, which may contain additional 40S, more 60S assembly factors and perhaps many more snoRNP complexes (Fig. 2; see below). These particles presumably have relatively low steady-state levels and therefore would be difficult to detect and purify. Genetic tricks would be required to freeze or delay the turnover of these complexes.

Many good reviews have been written in which the composition of pre-ribosomal complexes and putative functions of its components are described thoroughly [14,17,49,53]. Therefore, we will not discuss this in great detail. The proteomic results indicate that the SSU processome/90S pre-ribosomes contain about 40 proteins and at least one snoRNA, the U3 snoRNA. Numerous proteins contain motifs that are often associated with protein-protein interactions like WD-40 repeats or coiled-coil domains. These proteins might function as scaffolds on which many proteins can assemble or represent the backbone of the SSU processome/90S pre-ribosomes. In addition, several energy-dependent catalytically active proteins (RNA helicases, P-loop proteins and a GTPase, Bms1p; Refs. [19,61]) are frequently found associated with these complexes. We predict that these proteins play key regulatory roles in ribosome biogenesis, perhaps as molecular switches, by mediating many structural rearrangements within the pre-rRNA and protein-RNA interactions. Notably, four of the seven RNA helicases required for cleavage at sites A₀–A₂ in the 35S pre-rRNA were absent in the SSU processome/90S pre-ribosome purifications (Dhr2p, Rrp3p, Dbp4p and Fap1p; for a detailed overview of the components, see Ref. [17]). Dbp4p is genetically linked to the U14 snoRNA, suggesting that it may mediate or dissociate U14-pre-rRNA base-pairing interactions [33].

AAA-ATPases are a class of enzymes lacking in 90S pre-ribosomes and purified 43S particles but are frequently found in pre-60S complexes. These proteins are involved in many cellular processes and have the capacity to alter the folding state of proteins [56]. It is tempting to speculate that they might function as chaperones in 60S ribosomal subunit biogenesis, possibly by influencing protein-protein interactions.

Where are the snoRNAs and endonucleases?

It is generally assumed that cleavages at sites A₀–A₂ are mediated by endonucleases, because the fragments spacing the cleavage sites can be detected in mutant strains (reviewed in Ref. [58]). Much to our disappointment, proteins that possess similarity to known endonucleases have not yet been identified in purified SSU processome/90S pre-ribosomes. However, it is very likely that most of these factors weakly or transiently interact with pre-ribosomal complexes and are therefore not detectable or present in sub-stoichiometric amounts in the purifications. Indeed, many known endo- and exo-nucleases (Rnt1p, RNase MRP endonuclease, exosome) were not found in purified pre-ribosomal particles (for a detailed overview of the components, see Ref. [17]). In contrast, Nob1p, a putative endonuclease that is essential for the conversion of 20S to 18S rRNA, co-purified with 43S pre-ribosomal particles [13,48]. Thus, the search for the endonucleases responsible for the cleavages at sites A₀–A₂ is still ongoing and perhaps the

Protein composition of purified SSU processome/90S pre-ribosomes

Comparison of several proteomic studies indicates that over 150 factors are involved in pre-RNA processing [17].

large number of functional genomics studies that have
recently been performed contain clues on where to look
(e.g., see Ref. [43]).

Another plausible but rather audacious hypothesis is that
these cleavages are mediated by ribozyme structures, which
might be formed by intramolecular interactions in the pre-
rRNA or perhaps by snoRNA–pre-rRNA duplexes. A
similar hypothesis has been proposed for the spliceosome
and there is evidence that supports this idea [7].

Another aspect of the pre-ribosomes purifications is the
apparent absence of pseudouridylation- and methylation
guide snoRNAs in the SSU processome/90S pre-ribosomes,
notwithstanding other snoRNAs (besides U3 snoRNA) that
have a role in pre-rRNA processing (U14, snR10, snR30;
Ref. [58]). This is not the case for (putative) pre-rRNA
methyltransferase enzymes, which were found associated
with 90S, 43S and 66S particles (Nop1p, Bud23p, Dim1p,
Spb1p; Refs. [42,48]).

Most of the snoRNA-guided chemical modifications on
the pre-rRNA in yeast are predicted to occur co-transcrip-
tionally, before the primary transcript is processed [8,36].
There is indirect evidence that suggests that some snoRNA-
mediated chemical modifications take place in the SSU
processome/90S pre-ribosome. Rok1p and Rrp5p, two com-
ponents of the SSU processome/90S pre-ribosomes, [10,21]
are genetically linked to a core component of box H/ACA
snoRNPs (Gar1p) and the box H/ACA snoRNA snR10
(which also guides a pseudouridylation reaction in the 25S
rRNA; Refs. [57,59]). In addition, the common box C/D and
box H/ACA snoRNP proteins are frequently found in 90S
pre-ribosomes and pre-60S complexes (reviewed in Refs.
[17,49]), although the modification guide snoRNA partners
were not detected or their presence was not investigated.
Like endonucleases, most modification guide snoRNAs
probably transiently associate with the pre-rRNA and there-
fore possibly escaped detection in proteomic studies.

Although the U14, snR10 and snR30 snoRNAs are
essential for cleavage at sites A₀–A₂ in the pre-rRNA and
base pair with the 35S pre rRNA, they are, unlike the U3
snoRNA, not significantly enriched in purified SSU proc-
essome/90S pre ribosomes or their association has not been
investigated [10,21,48,58]. This, again, implies transient
interactions and it will be particularly challenging to deter-
mine at what exact stage in pre-rRNA processing and
ribosome assembly these snoRNAs act. Again, genetic tricks
may be needed to block the dissociation of these snoRNAs.


The 40S and 60S subunit synthesis machineries share
only a few non-ribosomal proteins and act largely
independently

After separation of the 43S and 66S pre-ribosomal
complexes from the 90S pre-ribosomes, the synthesis of
the ribosomal subunits seems to occur largely independent-
ly. Only a few 18S synthesis factors (e.g., Rrp5p, Nop14p,

Rrp9p) or factors involved in cleavage at site A₂ in ITS1
(Rrp8p, Ssf1p, Ssf2p) were detected in pre-60S pre-ribo-
somal particles and these were found only in the early
complexes [42]. Although this is very interesting, it does
not come as a complete surprise. Assembly of the 40S
synthesis machinery on the 35S pre rRNA does not appear
to be a prerequisite for 60S ribosomal subunit synthesis.
Genetic depletion of many SSU processome components,
including the U3 snoRNA, inhibits A₀–A₂ cleavages, 18S
rRNA and 40S subunit synthesis but does not prevent
synthesis of 60S ribosomal subunits because the 35S pre-
rRNA can be cleaved at site A₃ (Fig. 1; reviewed in Refs.
[10,58]). Furthermore, deletion of almost the entire 18S
rDNA coding region does not affect 25S rRNA synthesis
[41]. Finally, both 18S and 25S can be produced in yeast
when the corresponding rRNA coding units are supplied in
trans [34]. Ironically, these results suggest that prior forma-
tion of the SSU processome or terminal knob is not essential
for 60S ribosome synthesis!

Concluding remarks and future perspectives

Why does a eukaryotic cell require such a huge machin-
ery to build a ribosome? There are clues that suggest the
SSU processome/90S pre-ribosome acts as a large RNA
chaperone. Base-pairing between the U3 snoRNA and the 5′
terminus of the 18S rRNA blocks the formation of an
evolutionarily conserved pseudoknot structure that is nor-
mally present in the mature ribosome [24]. This suggests
that this base-pairing interaction is essential to prevent
premature folding of the 18S rRNA. The pre-rRNA is
highly structured and many regions may need to be unfold-
ed or refolded before cleavages or chemical modification
steps can take place. Thus, the SSU processome/90S pre-
ribosome may function by making the pre-rRNA accessible
to snoRNPs and endonucleases to allow modifications and
cleavages to occur in a temporal fashion. The RNA heli-
cases may play an important role here by unwinding
intramolecular duplexes in the pre-rRNA or by peeling off
the modification guide snoRNAs.

At this point, there are over 150 non-ribosomal proteins
that have a putative role in ribosome biogenesis and in most
cases we can only speculate about their function. Associa-
tion of many of these proteins with pre-ribosomes has not
yet been validated, which is essential because the TAP tag
purifications often yield contaminants [18]. Because there
are so many factors involved and because it is a very
dynamic process, it will be particularly challenging to
determine how these pre-ribosomal particles are assembled
and which proteins or RNA elements are directly involved
in the cleavage events. The fact that pre-ribosomal particles
can be isolated in relatively high amounts and high concen-
trations will greatly aid (cryo) electron microscopic analysis
of these particles. Thus, the next 30 years will also prove to
be very exciting for the ribosome biogenesis field.

Acknowledgments

This work was supported by a Leslie H. Warner Fellowship in Cancer Research (S.G.) and by the National Institutes of Health and the National Science Foundation (S.J.B.).

References

[1] C. Allmang, P. Mitchell, E. Petfalski, D. Tollervey, Degradation of ribosomal RNA precursors by the exosome, Nucleic Acids Res. 28 (2000) 1684–1691.

[2] J.S. Andersen, C.E. Lyon, A.H. Fox, A.K. Leung, Y.W. Lam, H. Steen, M. Mann, A.I. Lamond, Directed proteomic analysis of the human nucleolus, Curr. Biol. 12 (2002) 1–11.

[3] J.P. Bachellerie, J. Cavaille, A. Huttenhofer, The expanding snoRNA world, Biochimie 84 (2002) 775–790.

[4] G.D. Bader, C.W. Hogue, Analyzing yeast protein–protein interaction data obtained from different sources, Nat. Biotechnol. 20 (2002) 991–997.

[5] J. Bassler, P. Grandi, O. Gadal, T. Lessmann, E. Petfalski, D. Tollervey, J. Lechner, E. Hurt, Identification of a 60S pre-ribosomal particle that is closely linked to nuclear export, Mol. Cell. 8 (2001) 517–529.

[6] E. Billy, T. Wegierski, F. Nasr, W. Filipowicz, Rcl1p, the yeast protein similar to the RNA 3′-phosphate cyclase, associates with U3 snoRNP and is required for 18S rRNA biogenesis, EMBO J. 19 (2000) 2115–2126.

[7] C.A. Collins, C. Guthrie, The question remains: is the spliceosome a ribozyme? Nat. Struct. Biol. 7 (2000) 850–854.

[8] W.A. Decatur, M.J. Fournier, RNA-guided nucleotide modification of ribosomal and other RNAs, J. Biol. Chem. 278 (2003) 695–698.

[9] W.A. Decatur, M.J. Fournier, rRNA modifications and ribosome function, Trends Biochem. Sci. 27 (2002) 344–351.

[10] F. Dragon, J.E. Gallagher, P.A. Compagnone-Post, B.M. Mitchell, K.A. Porwancher, K.A. Wehner, S. Wormsley, R.E. Settlage, J. Shabanowitz, Y. Osheim, A.L. Beyer, D.F. Hunt, S.J. Baserga, A large nucleolar U3 ribonucleoprotein required for 18S ribosomal RNA biogenesis, Nature 417 (2002) 967–970.

[11] D.A. Dunbar, S.J. Baserga, The U14 snoRNA is required for 2′-O-methylation of the pre-18S rRNA in Xenopus oocytes, RNA 4 (1998) 195–204.

[12] A. Fatica, A.D. Cronshaw, M. Dlakic, D. Tollervey, Ssf1p prevents premature processing of an early pre-60S ribosomal particle, Mol. Cell. 9 (2002) 341–351.

[13] A. Fatica, M. Oeffinger, M. Dlakic, D. Tollervey, Nob1p is required for cleavage of the 3′ end of 18S rRNA, Mol. Cell. Biol. 23 (2003) 1798–1807.

[14] A. Fatica, D. Tollervey, Making ribosomes, Curr. Opin. Cell Biol. 14 (2002) 313–318.

[15] W. Filipowicz, V. Pogacic, Biogenesis of small nuclear ribonucleoproteins, Curr. Opin. Cell Biol. 14 (2002) 319–327.

[16] D. Forler, T. Kocher, M. Rode, M. Gentzel, E. Izaurralde, M. Wilm, An efficient protein complex purification method for functional proteomics in higher eukaryotes, Nat. Biotechnol. 21 (2003) 89–92.

[17] M. Fromont-Racine, B. Senger, C. Saveanu, F. Fasiolo, Ribosome assembly in eukaryotes, Gene 313 (2003) 17–42.

[18] A.C. Gavin, M. Bosche, R. Krause, P. Grandi, M. Marzioch, A. Bauer, J. Schultz, J.M. Rick, A.M. Michon, C.M. Cruciat, M. Remor, C. Hofert, M. Schelder, M. Brajenovic, H. Ruffner, A. Merino, K. Klein, M. Hudak, D. Dickson, T. Rudi, V. Gnau, A. Bauch, S. Bastuck, B. Huhse, C. Leutwein, M.A. Heurtier, R.R. Copley, A. Edelmann, E. Querfurth, V. Rybin, G. Drewes, M. Raida, T. Bouwmeester, P. Bork, B. Seraphin, B. Kuster, G. Neubauer, G. Superti-Furga, Functional organization of the yeast proteome by systematic analysis of protein complexes, Nature 415 (2002) 141–147.

[19] D. Gelperin, L. Horton, J. Beckman, J. HensoId, S.K. Lemmon, Bms1p, a novel GTP-binding protein, the related Tsr1p are required for distinct steps of 40S ribosome biogenesis in yeast, RNA 7 (2001) 1268–1283.

[20] S.A. Gerbi, A.V. Borovjagin, M. Ezrokhi, T.S. Lange, Ribosome biogenesis: role of small nuclear RNA in maturation of eukaryotic rRNA, Cold Spring Harb. Symp. Quant. Biol. 66 (2001) 575–590.

[21] P. Grandi, V. Rybin, J. Bassler, E. Petfalski, D. Strauss, M. Marzioch, T. Schafer, B. Kuster, H. Tschochner, D. Tollervey, A.C. Gavin, E. Hurt, 90S pre-ribosomes include the 35S pre-rRNA, the U3 snoRNP, 40S subunit processing factors but predominantly lack 60S synthesis factors, Mol. Cell. 10 (2002) 105–115.

[22] P. Harpicharnchai, J. Jakovljevic, E. Horsey, T. Miles, J. Roman, M. Rout, D. Meagher, B. Imai, Y. Guo, C.J. Brame, J. Shabanowitz, D.F. Hunt, J.L. Woolford Jr., Composition and functional characterization of yeast 66S ribosome assembly intermediates, Mol. Cell. 8 (2001) 505–515.

[23] Y. Ho, A. Gruhler, A. Heilbut, G.D. Bader, L. Moore, S.L. Adams, A. Millar, P. Taylor, K. Bennett, K. Boutilier, L. Yang, C. Wolting, I. Donaldson, S. Schandorf, J. Shewnarane, M. Vo, J. Taggart, M. Goudreault, B. Muskat, C. Alfarano, D. Dewar, Z. Lin, K. Michalickova, A.R. Willems, H. Sassi, P.A. Nielsen, K.J. Rasmussen, J.R. Andersen, L.E. Johansen, L.H. Hansen, H. Jespersen, A. Podtelejnikov, E. Nielsen, J. Crawford, V. Poulsen, B.D. Sorensen, J. Matthisen, R.C. Hendrickson, F. Gleeson, T. Pawson, M.F. Moran, D. Durocher, M. Mann, C.W. Hogue, D. Figeys, M. Tyers, Systematic identification of protein complexes in *Saccharomyces cerevisiae* by mass spectrometry, Nature 415 (2002) 180–183.

[24] J.M. Hughes, Functional base-pairing interaction between highly conserved elements of U3 small nuclear RNA and the small ribosomal subunit RNA, J. Mol. Biol. 259 (1996) 645–654.

[25] J.M. Hughes, M. Ares Jr., Depletion of U3 small nuclear RNA inhibits cleavage in the 5′ external transcribed spacer of yeast pre-ribosomal RNA and impairs formation of 18S ribosomal RNA, EMBO J. 10 (1991) 4231–4239.

[26] S. Kass, B. Sollner-Webb, The first pre-rRNA-processing event occurs in a large complex: analysis by gel retardation, sedimentation, UV cross-linking, Mol. Cell. Biol. 10 (1990) 4920–4931.

[27] S. Kass, K. Tyc, J.A. Steitz, B. Sollner-Webb, The U3 small nucleolar ribonucleoprotein functions in the first step of pre-ribosomal RNA processing, Cell 60 (1990) 897–908.

[28] T.H. King, B. Liu, R.R. McCully, M.J. Fournier, Ribosome structure and activity are altered in cells lacking snoRNPs that form pseudouridines in the peptidyl transferase center, Mol. Cell. 11 (2003) 425–435.

[29] T. Kiss, Small nucleolar RNA-guided post-transcriptional modification of cellular RNAs, EMBO J. 20 (2001) 3617–3622.

[30] D. Kressler, P. Linder, J. de La Cruz, Protein trans-acting factors involved in ribosome biogenesis in *Saccharomyces cerevisiae*, Mol. Cell. Biol. 19 (1999) 7897–7912.

[31] A. Kumar, J.R. Warner, Characterization of ribosomal precursor particles from HeLa cell nucleoli, J. Mol. Biol. 63 (1972) 233–246.

[32] A.K. Leung, J.S. Andersen, M. Mann, A.I. Lamond, Bioinformatic analysis of the nucleolus, Biochem. J. 376 (2003) 553–569.

[33] W.Q. Liang, J.A. Clark, M.J. Fournier, The rRNA-processing function of the yeast U14 small nucleolar RNA can be rescued by a conserved RNA helicase-like protein, Mol. Cell. Biol. 17 (1997) 4124–4132.

[34] W.Q. Liang, M.J. Fournier, Synthesis of functional eukaryotic ribosomal RNAs in trans: development of a novel in vivo rDNA system for dissecting ribosome biogenesis, Proc. Natl. Acad. Sci. U. S. A. 94 (1997) 2864–2868.

[35] W.Q. Liang, M.J. Fournier, U14 base-pairs with 18S rRNA: a novel snoRNA interaction required for rRNA processing, Genes Dev. 9 (1995) 2433–2443.

[36] B.E. Maden, J.M. Hughes, Eukaryotic ribosomal RNA: the recent excitement in the nucleotide modification problem, Chromosoma 105 (1997) 391–400.

[37] P. Milkereit, D. Strauss, J. Bassler, O. Gadal, H. Kuhn, S. Schutz, N. Gas, J. Lechner, E. Hurt, H. Tschochner, A Noc complex specifically involved in the formation and nuclear export of ribosomal 40 S subunits, J. Biol. Chem. 278 (2003) 4072–4081.

[38] O.L. Miller Jr., B.R. Beatty, Visualization of nuclear genes, Science 164 (1969) 955–957.

[39] E.B. Mougey, M. O'Reilly, Y. Osheim, O.L. Miller Jr., A. Beyer, B. Sollner-Webb, The terminal balls characteristic of eukaryotic rRNA transcription units in chromatin spreads are rRNA processing complexes, Genes Dev. 7 (1993) 1609–1619.

[40] E.B. Mougey, L.K. Pape, B. Sollner-Webb, A U3 small nuclear ribonucleoprotein-requiring processing event in the 5' external transcribed spacer of Xenopus precursor rRNA, Mol. Cell. Biol. 13 (1993) 5990–5998.

[41] W. Musters, J. Venema, G. van der Linden, H. van Heerikhuizen, J. Klootwijk, R.J. Planta, A system for the analysis of yeast ribosomal DNA mutations, Mol. Cell. Biol. 9 (1989) 551–559.

[42] T.A. Nissan, J. Bassler, E. Petfalski, D. Tollervey, E. Hurt, 60S pre-ribosome formation viewed from assembly in the nucleolus until export to the cytoplasm, EMBO J. 21 (2002) 5539–5547.

[43] W.T. Peng, M.D. Robinson, S. Mnaimneh, N.J. Krogan, G. Cagney, Q. Morris, A.P. Davierwala, J. Grigull, X. Yang, W. Zhang, N. Mitsakakis, O.W. Ryan, N. Datta, V. Jojic, C. Pal, V. Canadien, D. Richards, B. Beattie, L.F. Wu, S.J. Altschuler, S. Rowois, B.J. Frey, A. Emili, J.F. Greenblatt, T.R. Hughes, A panoramic view of yeast noncoding RNA processing, Cell 113 (2003) 919–933.

[44] G. Rigaut, A. Shevchenko, B. Rutz, M. Wilm, M. Mann, B. Séraphin, A generic protein purification method for protein complex characterization and proteome exploration, Nat. Biotechnol. 17 (1999) 1030–1032.

[45] C. Saveanu, D. Bienvenu, A. Namane, P.E. Gleizes, N. Gas, A. Jacquier, M. Fromont-Racine, Nog2p, a putative GTPase associated with pre-60S subunits and required for late 60S maturation steps, EMBO J. 20 (2001) 6475–6484.

[46] C. Saveanu, A. Namane, P.E. Gleizes, A. Lebreton, J.C. Rousselle, J. Noaillac-Depeyre, N. Gas, A. Jacquier, M. Fromont-Racine, Sequential protein association with nascent 60S ribosomal particles, Mol. Cell. Biol. 23 (2003) 4449–4460.

[47] R. Savino, S.A. Gerbi, In vivo disruption of Xenopus U3 snRNA affects ribosomal RNA processing, EMBO J. 9 (1990) 2299–2308.

[48] T. Schafer, D. Strauss, E. Petfalski, D. Tollervey, E. Hurt, The path from nuclear 90S to cytoplasmic 40S pre-ribosomes, EMBO J. 22 (2003) 1370–1380.

[49] N. Takahashi, M. Yanagida, S. Fujiyama, T. Hayano, T. Isobe, Proteomic snapshot analyses of preribosomal ribonucleoprotein complexes formed at various stages of ribosome biogenesis in yeast and mammalian cells, Mass Spectrom. Rev. 22 (2003) 287–317.

[50] D. Tollervey, A yeast small nuclear RNA is required for normal processing of pre-ribosomal RNA, EMBO J. 6 (1987) 4169–4175.

[51] D. Tollervey, C. Guthrie, Deletion of a yeast small nuclear RNA gene impairs growth, EMBO J. 4 (1985) 3873–3878.

[52] J. Trapman, J. Retel, R.J. Planta, Ribosomal precursor particles from yeast, Exp. Cell Res. 90 (1975) 95–104.

[53] H. Tschochner, E. Hurt, Pre-ribosomes on the road from the nucleolus to the cytoplasm, Trends Cell Biol. 13 (2003) 255–263.

[54] S.A. Udem, J.R. Warner, The cytoplasmic maturation of a ribosomal precursor ribonucleic acid in yeast, J. Biol. Chem. 248 (1973) 1412–1416.

[55] S.A. Udem, J.R. Warner, Ribosomal RNA synthesis in Saccharomyces cerevisiae, J. Mol. Biol. 65 (1972) 227–242.

[56] R.D. Vale, AAA proteins. Lords of the ring, J. Cell Biol. 150 (2000) F13–F19.

[57] J. Venema, C. Bousquet-Antonelli, J.P. Gelugne, M. Caizergues-Ferrer, D. Tollervey, Rok1p is a putative RNA helicase required for rRNA processing, Mol. Cell. Biol. 17 (1997) 3398–3407.

[58] J. Venema, D. Tollervey, Ribosome synthesis in Saccharomyces cerevisiae, Annu. Rev. Genet. 33 (1999) 261–311.

[59] J. Venema, D. Tollervey, RRP5 is required for formation of both 18S and 5.8S rRNA in yeast, EMBO J. 15 (1996) 5701–5714.

[60] J.R. Warner, R. Soeiro, Nascent ribosomes from HeLa cells, Proc. Natl. Acad. Sci. U. S. A. 58 (1967) 1984–1990.

[61] T. Wegierski, E. Billy, F. Nasr, W. Filipowicz, Bms1p, a G-domain-containing protein, associates with Rcl1p and is required for 18S rRNA biogenesis in yeast, RNA 7 (2001) 1254–1267.

[62] K.A. Wehner, S.J. Baserga, The sigma(70)-like motif: a eukaryotic RNA binding domain unique to a superfamily of proteins required for ribosome biogenesis, Mol. Cell. 9 (2002) 329–339.

[63] L.F. Wu, T.R. Hughes, A.P. Davierwala, M.D. Robinson, R. Stoughton, S.J. Altschuler, Large-scale prediction of Saccharomyces cerevisiae gene function using overlapping transcriptional clusters, Nat. Genet. 31 (2002) 255–265.
