
Molecular Microbiology (2004) 52(4), 1107–1121

doi:10.1111/j.1365-2958.2004.04032.x

# Characterization of an inducible vancomycin resistance system in *Streptomyces coelicolor* reveals a novel gene (*vanK*) required for drug resistance

Hee-Jeon Hong,$^{1}$ Matthew I. Hutchings,$^{1}$ John M. Neu,$^{2}$ Gerard D. Wright,$^{2}$ Mark S. B. Paget$^{1\dagger}$ and Mark J. Buttner$^{1}$

$^{1}$ Department of Molecular Microbiology, John Innes Centre, Colney, Norwich NR4 7UH, UK.

$^{2}$ Antimicrobial Research Centre, Department of Biochemistry, McMaster University, Hamilton, Ontario L8N 3Z5, Canada.

## Summary

Vancomycin is the front-line therapy for treating problematic infections caused by methicillin-resistant *Staphylococcus aureus* (MRSA), and the spread of vancomycin resistance is an acute problem. Vancomycin blocks cross-linking between peptidoglycan intermediates by binding to the D-Ala-D-Ala termini of bacterial cell wall precursors, which are the substrate of transglycosylase/transpeptidase. We have characterized a cluster of seven genes (*vanSRJKHAX*) in *Streptomyces coelicolor* that confers inducible, high-level vancomycin resistance. *vanHAX* are orthologous to genes found in vancomycin-resistant enterococci that encode enzymes predicted to reprogramme peptidoglycan biosynthesis such that cell wall precursors terminate in D-Ala-D-Lac rather than D-Ala-D-Ala. *vanR* and *vanS* encode a two-component signal transduction system that mediates transcriptional induction of the seven *van* genes. *vanJ* and *vanK* are novel genes that have no counterpart in previously characterized vancomycin resistance clusters from pathogens. VanK is a member of the Fem family of enzymes that add the cross-bridge amino acids to the stem pentapeptide of cell wall precursors, and *vanK* is essential for vancomycin resistance. The *van* genes are organized into four transcription units, *vanRS*, *vanJ*, *vanK* and *vanHAX*, and these transcripts are induced by vancomycin in a *vanR*-dependent manner. To develop a sensitive bioassay for inducers

Accepted 13 January, 2004. *For correspondence.* E-mail Hong@bbsrc.ac.uk; Tel. (+44) 1603 450757; Fax (+44) 1603 450778.

$^{\dagger}$Present address: Department of Biochemistry, School of Life Sciences, University of Sussex, Falmer, Brighton BN1 9QG, UK.

© 2004 Blackwell Publishing Ltd

of the vancomycin resistance system, the promoter of *vanJ* was fused to a reporter gene conferring resistance to kanamycin. All the inducers identified were glycopeptide antibiotics, but teicoplanin, a membrane-anchored glycopeptide, failed to act as an inducer. Analysis of mutants defective in the *vanRS* and *cseBC* cell envelope signal transduction systems revealed significant cross-talk between the two pathways.

## Introduction

Glycopeptide antibiotics such as vancomycin inhibit cell wall biosynthesis in Gram-positive bacteria but not in Gram-negative bacteria because they cannot penetrate the outer membrane permeability barrier. Vancomycin is clinically important for treating enterococcal infections arising after abdominal surgery and is vital as the only widely effective treatment for infections caused by methicillin-resistant *Staphylococcus aureus* (MRSA), a major killer in hospital-acquired infections and, increasingly, in community-acquired infections. Ever since vancomycin-resistant strains of *Enterococcus faecalis* and *Enterococcus faecium* first emerged in 1987, transfer of the vancomycin resistance genes from enterococci to MRSA to create a vancomycin-resistant ‘superbug’ (VRSA) has been widely anticipated. This spectre became a reality in 2002 with the first reports of clinical isolates of VRSA from hospitals in Michigan and Pennsylvania (Pearson, 2002; Chang *et al.*, 2003). Subsequent analysis showed these isolates arose from intergeneric transfer of Tn1546 from a co-isolate of *E. faecalis* (Weigel *et al.*, 2003; Tenover *et al.*, 2004). In addition to its medical importance, vancomycin is also academically important because of its very unusual mode of action and the exceptional mechanism that gives rise to vancomycin resistance.

Bacterial peptidoglycan (PG) is synthesized from a precursor consisting of a disaccharide unit with a pendant pentapeptide, which in turn may carry a peptide side-branch or ‘cross-bridge’ (Fig. 1). The exact nature of this precursor varies from genus to genus: the *Streptomyces* precursor, shown in Fig. 1, has LL-diaminopimelic acid (LL-dpm) at position 3 of the pentapeptide, and the cross-bridge is a single glycine (Leyh-Bouille *et al.*, 1970;

1108 H.-J. Hong et al.

![Chemical Structure](https://i.imgur.com/chemical_structure.png)

Fig. 1. The key precursor in peptidoglycan (PG) biosynthesis in *Streptomyces* and the cross-linking reaction carried out by transpeptidase. Note that the exact nature of the precursor varies from genus to genus; in *Streptomyces*, it has LL-diaminopimelic acid (LL-dpm) at position 3 of the pentapeptide, and the cross-bridge is a single glycine. Transpeptidase recognizes the sequence D-alanyl-D-alanine (D-Ala-D-Ala) at the end of the pentapeptide chain, cleaves off the terminal alanine and joins the remainder to a cross-bridge peptide from an adjacent polysaccharide chain. Vancomycin binds directly to the terminal D-Ala-D-Ala of the pentapeptide and blocks the action of transpeptidase (Williams *et al.*, 1983; Barna and Williams, 1984).

Schleifer and Kandler, 1972; Ruzin *et al.*, 2002; H.-J. Hong, A. Severin, K. Tabei and M. J. Buttner, unpublished). The cell wall is constructed from this building block in two main extracellular steps (for a review, see Bugg, 1999). First, the disaccharide units are polymerized by a transglycosylase to form long polysaccharide chains. Secondly, these long polysaccharide chains are cross-linked through their peptide side-chains by a transpeptidase (Fig. 1). The transpeptidase recognizes the sequence D-alanyl-D-alanine (D-Ala-D-Ala) at the end of the pentapeptide chain, cleaves off the terminal alanine and joins the remainder to a cross-bridge peptide from an adjacent polysaccharide chain (Fig. 1). This peptide cross-linking provides the structural rigidity of mature PG required for maintenance of cell shape and prevention of cell lysis. In some bacteria, such as *Escherichia coli* and *Bacillus subtilis*, there is no cross-bridge and a direct 3–4 cross-link is formed between pentapeptide side-chains.

β-Lactam antibiotics (penicillins and cephalosporins) directly inhibit transpeptidase by acylating the enzyme’s active site (Spratt and Cromie, 1988; Walsh, 2000). In contrast, vancomycin acts by binding to its substrate. Vancomycin binds tightly, but non-covalently, to the D-Ala-D-Ala terminus of lipid-attached PG precursors on the outside of the cytoplasmic membrane, thereby blocking transpeptidation and, more weakly and indirectly, transglycosylation (Fig. 1; Williams *et al.*, 1983; Barna and Williams, 1984). Vancomycin first became a vital weapon against *S. aureus* when MRSA emerged in the mid-1980s. The molecular basis of the MRSA phenotype was a gene fusion event that created *mecA*, a gene that encodes a transpeptidase (PBP2β) with very low affinity for all β-

lactam antibiotics (Song *et al.*, 1987; Spratt, 1994; Chu *et al.*, 1996). Once the mode of action of vancomycin was understood, it became clear that, in contrast to methicillin, mutations in transpeptidase could never give rise to vancomycin resistance. For this reason, some suggested that pathogens might never acquire vancomycin resistance, because it would require remodelling of the cell wall itself. However, the clinical isolates of vancomycin-resistant enterococci (VRE) that first appeared in the late 1980s were shown to do precisely that (Bugg *et al.*, 1991; Walsh *et al.*, 1996; Williams and Bardsley, 1999; Healy *et al.*, 2000; Walsh, 2000; Pootoolal *et al.*, 2002a). On exposure to vancomycin, VRE isolates reprogramme PG biosynthesis such that the pendant pentapeptide of the cell wall precursors terminates in D-alanyl-D-lactate (D-Ala-D-Lac), rather than in D-Ala-D-Ala. The affinity of vancomycin for precursors terminating in D-Ala-D-Lac is ≈1000-fold lower than for precursors terminating in D-Ala-D-Ala (Bugg *et al.*, 1991), rendering the modified bacteria resistant. Three enzymes were shown to be required for this remodelling: VanH, which converts pyruvate into D-lac; VanA, a D-Ala-D-Lac ligase; and VanX, a ‘proof-reading’ D-Ala-D-Ala dipeptidase that cleaves any residual circulating D-Ala-D-Ala dipeptide, ensuring that peptidoglycan precursors terminate uniformly in D-Ala-D-Lac. The comparatively rare VanC, VanE and VanG isolates of *Enterococci* have a D-alanyl-D-serine (D-Ala-D-Ser) ligase in place of the D-Ala-D-Lac ligase (Reynolds *et al.*, 1994; Pootoolal *et al.*, 2002a). The substitution of D-Ser for D-Ala results in an approximately six-fold decrease in affinity for vancomycin and low-level resistance (Billot-Klein *et al.*, 1994).

We have discovered that the non-pathogenic, non-

glycopeptide-producing actinomycete Streptomyces coelicolor is highly resistant to vancomycin, and we have identified and characterized a novel cluster of seven genes that confers this resistance. We show that expression of these genes is induced via a two-component regulatory system by several vancomycin-related glycopeptide antibiotics, but not by the glycopeptide teicoplanin. We also show that a novel resistance gene (vanK), with no counterpart in previously characterized vancomycin resistance clusters from pathogens and predicted to encode an enzyme involved in synthesis of the peptide cross-bridge, is required for vancomycin resistance.

**Results**

**Identification of a vancomycin resistance gene cluster containing two novel genes in S. coelicolor**

In the course of other work, we unexpectedly found S. coelicolor to be highly resistant to vancomycin (Hong et al., 2002). As a consequence, we used the sequences of the VanH, VanA and VanX proteins of E. faecium to search for homologues encoded in the S. coelicolor genome sequence (http://www.sanger.ac.uk/Projects/S_coelicolor; Bentley et al., 2002). In this way, we identified a putative vancomycin resistance cluster of seven genes that we designated vanS (SCO3589), vanR (SCO3590), vanJ (SCO3592), vanK (SCO3593), vanH (SCO3594), vanA (SCO3595) and vanX (SCO3596) (Fig. 2). In the genome sequence, a small open reading frame (ORF; SCO3591) is annotated between vanR and vanJ, potentially encoding a 41-amino-acid protein. This potential coding sequence, suggested by FRAME analysis (Bibb et al., 1984), reads in the same direction as vanRS. If this ORF were a real gene, then the vanJ transcript (see below) would read partially through it on the opposing strand. However, we believe that SCO3591 is not a gene and that the FRAME plot simply reflects the degenerate repetition of the hexameric sequence CTG-GCG in this region. The van genes are not associated with a glycopeptide biosynthetic cluster, and it is clear from inspection of the genome sequence that S. coelicolor does not have the genetic capacity to synthesize a glycopeptide antibiotic (Bentley et al., 2002). The van genes are absent from the only other publicly available Streptomyces genome sequence (Streptomyces avermitilis; http://aver-

pIJ6883 pIJ6892

vanRp vanJp vanKp vanHp


     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |       |       |       |
     |
1110 H.-J. Hong et al.

mitilis.ls.kitasato-u.ac.jp; Ikeda *et al.*, 2003), but they are present in *Streptomyces lividans*, a very close relative of *S. coelicolor* (H.-J. Hong, unpublished).

The predicted products of the *S. coelicolor vanH*, *vanA* and *vanX* genes each have high similarity to their counterparts from vancomycin resistance clusters identified in pathogens: VanH is 61% identical to *E. faecium* BM4147 VanH D-lactate dehydrogenase; VanA is 63% identical to *E. faecalis* BM4382 VanB D-Ala-D-Lac ligase; and VanX is 64% identical to *E. faecalis* BM4382 VanXB D-Ala-D-Ala dipeptidase. It therefore seems reasonable to predict that expression of *vanHAX* in *S. coelicolor* will reprogramme PG biosynthesis such that the pendant pentapeptide of the cell wall precursors will terminate in D-Ala-D-Lac, rather than in D-Ala-D-Ala.

*vanR* and *vanS* encode a response regulator and a sensor histidine protein kinase, respectively, making up a putative two-component regulatory system (Hackenbeck and Stock, 1996; Egger *et al.*, 1997). The predicted product of *vanR* is 231 amino acids long and is a member of the OmpR subfamily of response regulators. VanR has an N-terminal receiver domain with conserved aspartate and lysine residues known to form an acidic phosphorylation pocket in other members of the family, including Asp-51, the putative site of phosphorylation (Stock *et al.*, 1989; Egger *et al.*, 1997). In common with other members of the OmpR subfamily (Mizuno and Tanaka, 1997), VanR also has a putative helix–loop–helix (HLH) DNA-binding motif at its C-terminus. VanR is 31% identical to *E. faecium* BM4147 VanR. VanS is predicted to be 364 amino acids long, with the ≈250-amino-acid C-terminal domain characteristic of bacterial sensor histidine kinases, including the histidine residue (H150) that is the site of autophosphorylation, conserved asparagine and phenylalanine residues and two glycine-rich motifs (LXGXG and GXGXG) that are important for ATP binding and phosphotransfer catalysis. VanS is predicted to be a transmembrane protein, with two predicted membrane-spanning helices (residues 17–37 and 65–85) separated by a putative extracytoplasmic domain of 27 amino acids. VanS is 25% identical to *E. faecium* BM4147 VanS.

The *vanJ* and *vanK* genes are particularly interesting because they have no counterparts in previously characterized vancomycin resistance gene clusters from pathogens. VanJ is 330 amino acids long with three predicted membrane-spanning domains at the N-terminus. VanJ is similar to other unknowns, including two predicted membrane-localized homologues encoded elsewhere in the *S. coelicolor* genome: SCO7017 (51% identity) and SCO2255 (55% identity).

VanK is 397 amino acids long and shows significant similarity to the Fem family of proteins (Fig. 3). Fem proteins are non-ribosomal peptidyltransferases that add the cross-bridge amino acids to the stem pentapeptide of cell

wall precursors (Hegde and Shrader, 2001; Hegde and Blanchard, 2003; Rohrer and Berger-Bächi, 2003) and are absent from bacteria, such as *Escherichia coli* and *Bacillus subtilis*, that make direct 3–4 linkages between stem pentapeptides. For example, the pentaglycine cross-bridge in *S. aureus* is built by the action of three homologous peptidyltransferases called FemA, FemB and FemX (Rohrer and Berger-Bächi, 2003). In addition to VanK, there are two more Fem homologues encoded elsewhere on the *S. coelicolor* chromosome: SCO0602 and SCO3904. An alignment of these three *S. coelicolor* proteins and the FemABX proteins of *S. aureus* is shown in Fig. 3. The three *S. coelicolor* proteins are more similar to each other than to *S. aureus* FemABX and, in particular, they lack an ≈75-amino-acid region present in the *S. aureus* proteins. The crystal structure of *S. aureus* FemA has been solved and has two domains: a globular domain that resembles histone acetyltransferases and a helical ‘arm’ domain (Benson *et al.*, 2002). The 75 amino acids absent from the *S. coelicolor* proteins correspond to this arm domain. Benson *et al.* (2002) proposed that the arm is involved in binding the aminoacyl-tRNA substrate, but it has been shown that *Lactobacillus viridescens* FemX, which also lacks this domain, can efficiently bind and catalyse amino acid transfer from aminoacyl-tRNAs (Hegde and Blanchard, 2003). Hegde and Blanchard (2003) pointed out that *L. viridescens* FemX is a cytoplasmic enzyme while other known FemABX enzymes are membrane associated and, on this basis, they suggested that the arm domain might be involved in membrane association. In support of this view, addition of the cross-bridge glycine has recently been shown to occur in the cytoplasm in *Streptomyces* (Ruzin *et al.*, 2002), suggesting that the *Streptomyces* Fem proteins are also cytoplasmic.

### Transcription analysis of *vanSRJKHAX*

Using high-resolution S1 nuclease protection analysis of RNA isolated from a time course of *S. coelicolor* M600 (a plasmid-free derivative of the wild type) grown in NMMP plus glucose, initial attempts to detect *van* transcripts failed. To see if expression of the *van* genes was inducible, the experiment was repeated using RNA isolated from a single time point of a culture that was exposed to a sub-inhibitory concentration of vancomycin (10 µg ml⁻¹) throughout growth; under these conditions, *van* gene transcription was readily detected, and we identified four separate promoters in front of the *vanR*, *vanJ*, *vanK* and *vanH* genes (Fig. 4A). Unusually, the *vanR*, *vanK* and *vanH* transcripts each start at the ‘A’ of the AUG translation initiation codon (Fig. 4B), and these mRNAs must therefore be translated in the absence of a conventional 5′ mRNA leader and ribosome binding site. Leaderless messages are rare in most bacteria, although a disproportion-

Vancomycin resistance in S. coelicolor 1111

sa-FemA 1 ~MKLINERQIGVMIMKFTNLTAKFGAFTDSEPY.SHFTOTVGHYELKLAEGYETHLVGI.KNNNEVIAA..CLLTAVPVMKVBYFYSNRGPVIDYENQELVHFFFNELSKYVKHKRC
sa-FemB 1 ~~~~~~~~~~~~~~~~~~MKFTELTVTEFDNFVQNPSLESHYBOVKENIVTRENDGEVVLLGI.KDDNNKVIAA..SLFSKTPTMGSYY.YSNRGPVMDFSDLGLVDYYIKBLDKYLQQHC
sa-FemX 1 ~~~~~~~~~~~~~~~~~~MEKMHTTNQEHDAFKSHP.NGDLIQLTKWSETKKLTGWYARRIAVGRDGEVQGVAAQ..LLFKKVPKL.PYTLCYISRGFVVVDYSNKEALNALLDSAKEIAKAEGA
sco0602 1 ~~~~~~~~~~~~~~~~MSCRLKPITREHLRFVAARPSVSHL.QVPSWGDVK..PDWRASELGWFD.EGELVGAGLVLLRPDLPRKRY.LAYLPBGVPIDWSADG.LERWLEPMILAHLRGRGA
sco3904 1 ~~~~~~~~~~~~~~~~MSITLRRTISREQLAYIQSLPAASHM.QVPAWADVK..AEWRSERNLGWFDDRTGELVGAGLVLYRQLPKIKRY.LAYLPBGVPINWFAPN.LQDWMBPMILAHLKQQGA
sc-VanK 1 MAYSEPVRTTAPTRPTVRQISAAEHLAFVRAQRSVSFL.QTPAWGRVK..TEWRSESGLGWFD..GGRVLVGAGVLHRPVPRLEFTLAYLPBGVPIDWTGD..IDAWLDPILAAYLKARGA

sa-FemA 116 LY_LHIDPYIPYQYLNHD............GETG...NAGNDWFFDKMSNIGFEHT..GFHKGFDPVLCIRYHSVLDJKDKTAADDITKNMDGLRKRNITKKVKKNGVKV.........
sa-FemB 103 LYVKLDPYWLYHLYDKD............IVPEG...REKNDALVNLFKSHGYEH..GFTTEYDTSSQVRWMGVLNLEGKTPELTLLKTFDSQRKRNIANKAINYGVKV
sa-FemX 103 YATKIDP......................DVEV...DKGTDALQN.LKALGFKHK..GFKEGLSKDYIQPRMTMITPIDKNDDELLNSFERRNRSKVRALAIRGTTV............
sco0602 103 FTVRMGPPVVARWSAEAVKAAIDPGARRIGDVEATEHERGADALAERLRRMGWRTETPGADGFA.AGQPRHVCOVPLAGRSLFQLQNGFNQQWRNIIKADKAGVKVVRGD........
sco3904 104 FSVKVGPPVIIRWDATSIKKGIQDPDVKRLRDIEADHIEPRAFEVADKLRRMGWQQGEDGA.GFG.DVOPRVFQVPLANRSLSEEVHKNFNQLWRNIIKAEKAGVEVVQGG........
sc-VanK 114 FAIRLGPPVCTDVWSADEIKEGIADEGTRRLTGIPTRWTDPVGAHVTDRLRDAGWLPQSP..EDGFG.VGHPQFKYEVPLAGRTFEDDLLKGMNQLWRNIRKAAKEGVEVTLGDATPGYA

sa-FemA 207 ...RFLSEEELPTFRSFMEDTSESKABADRDDKFYYNRLKYYKDRVLPILAYINFDEYIKELNEERDILNKDLNKALKDIETE.KR PENKKAHNKRDNLQQQFDANEQKIEEGKRLQEEH
sa-FemB 195 ...RFLERDEFNLDDLRYREETERAGCVSKTDDYFYNFIDTYGDKVLVPLAYILDDEYVLKLQOE...LNDKENRRDQMMA.KENKSDKOMKKIAELDKQIHHDQHELLNASELS KTD
sa-FemX 182 ...ERSDREGLKTFABLMKITGERDGELTRDSYFENIYDALHEDGDAELFLVKLDPEKANI AKVNQELNELHAEI AKWQQKMKTSEQKAAK AQNMINDAQNKIAKNEDLKDLEALEKEHH
sco0602 216 ........QEDLPVFYELYVETARRDERAPRPLAYFORMWQALTAE.................DP.............................................DRMLLYLAH
sco3904 216 ........YHDLLEEWQRLYEITAVRDERPRPISYFORMWAALNNE.................DP.............................................NMRMLLYFAR
sc-VanK 231 AAGEAAVADDVKAFHDLYVHTAERDRFTPRPIRYFATMFALGAE.................DP.............................................ERIKLYLAR

sa-FemA 321 GNELPTISAGFFFINPFEVVYYAGGTSNAFRHFACSAYAVWEMINYALNHGIDRYNFYGVS GKFTEDAEDAGVVKFKGYNAEIIIEYVGDFIKPINKPVYAAYTALKKVKDRTF~~~~~~~~
sa-FemB 306 GPIIDLASGVYFANAAYEVNYFSGGSSEKYNCFMGPYMMHWFMINYCFDNGYDRYNFYGLSG DFTENSEDYGVYRFKRGFNVQIEELIGDFYKPIHKVKYWFTTLDKLKKLKKK~~~~~~~~
sa-FemX 299 PECTYLSGALLMFAGSKSYLYGASSNEBRDFLPNHMQYTMKYAREHGATTYDFGGTDNDPDKDSEHYGLWAFKKVNGTYLSEKIGEFDYVINQPLYQL...IEQVKPRLTKAKIKIS
sco0602 265 HDGEVLAATMTIVGEHWYSYGASTGRRREVQPNNALQWRMMCDARERLGAAVYDFRGIITDTLDEDNHLGLLRFKVCAGGKAVEYLGEWEYPLNRVLHKAVALYIARR~~~~~~~~~~~
sco3904 265 HNGVNLSAATMLVVGHWYSYGASDNICREVRPSNAMQWAMLRDSYALGATVYDIRGISDSLDESDHLFGLIQFKVGTGGQA AEYLGEWDGPLNKILLHKALDTYMSSR~~~~~~~~~~~
sc-VanK 287 HRGDLVAATVMVRVGTHAVYAYYGASSTGKREVRGSNACQWAMIRDALAADC DVYDIRGITPTILDDADPHVGLIKFKVCTGGRARVRYVGEWDLPRLPYRTFDLYMKRRGR~~~~~~~~~~~

sa-FemA 434 ~~~~~~
sa-FemB 420 ~~~~~~
sa-FemX 416 RKLKRK
sco0602 374 ~~~~~~
sco3904 374 ~~~~~~
sc-VanK 398 ~~~~~~

Fig. 3. Amino acid sequence alignment of VanK with the two other Fem homologues encoded elsewhere on the S. coelicolor chromosome, SCO0602 and SCO3904, and the FemA, FemB and FemX (= FmhB) proteins of Staphylococcus aureus. The ~75-amino-acid region absent from the S. coelicolor proteins corresponds to the ‘arm’ domain in the crystal structure of S. aureus FemA (Benson et al., 2002).

ately large number of leaderless mRNAs have been identified in Streptomyces, often encoding antibiotic resistance proteins (e.g. Janssen et al., 1989; Bibb et al., 1994; Wu and Janssen, 1997). Recent evidence suggests that leaderless messages can be translated by the intact 70S ribosome without its dissociation into subunits (O’Donnell and Janssen, 2002; Udagawa et al., 2004). The transcription start site of vanJ is 117 bp upstream from the most highly predicted vanJ translation initiation codon (Fig. 4A and B). However, the vanJ transcript also starts at the ‘A’ of an in frame AUG, leaving open the possibility that the vanJ transcript is also a leaderless message (Fig. 4B). The stop and start codons of vanH, vanA and vanX overlap, and the same is true of vanR and vanS, suggesting that vanHAX and vanRS may be co-transcribed and translationally coupled (Fig. 4B).

To analyse the induction of the four van transcripts, we grew S. coelicolor M600 to exponential phase in NMMP plus glucose before adding 10 µg ml⁻¹ vancomycin and examined the levels of the four van transcripts at 10 min intervals after vancomycin treatment. The transcripts were undetectable at time 0, but all four mRNAs were readily detectable after 10 min exposure to vancomycin and peaked at 30–40 min before declining (Fig. 5A). Induction of these four promoters was absolutely dependent on the VanR response regulator, as shown by S1 nuclease protection analysis of RNA isolated from M145 (vanR⁺) and a congenic vanR null mutant after treatment with

10 µg ml⁻¹ vancomycin (Fig. 5B). vanR null mutants are vancomycin sensitive (see below); to ensure that the absence of signals in RNA isolated from the vanR mutant was not due to vancomycin-induced cell death, we examined the expression of the sigE gene in the same RNA time courses as a positive control. sigE is part of a signal transduction system that senses and responds to cell envelope stress in S. coelicolor, and the sigE promoter (sigEp) is activated by vancomycin (Hong et al., 2002). sigEp was strongly induced by vancomycin in the vanR null mutant (Fig. 5B), confirming that induction of the four van transcripts is vanR dependent.

Bioassay for inducers of the vancomycin resistance system

To develop a sensitive bioassay for inducers of the van genes, a 0.25 kb HindIII–EcoRI fragment carrying the vanJ promoter (vanJp) was cloned into the multicopy promoter probe plasmid pIJ486 (Ward et al., 1986) such that expression of the vector aminoglycoside phosphotransferase gene (neo), which confers resistance to both neomycin and kanamycin, depended on vanJp. The resulting plasmid, pIJ6883, conferred resistance to only 2–3 µg ml⁻¹ kanamycin in S. coelicolor M600, consistent with the observed inactivity of vanJp in the absence of vancomycin. To identify inducers of the vanJ promoter, confluent lawns of spores of S. coelicolor M600 carrying

© 2004 Blackwell Publishing Ltd, Molecular Microbiology, 52, 1107–1121

A

| vanRp | vanJp | vanKp | vanHp |
| --- | --- | --- | --- |
| A C G T | A C G T | A C G T | A C G T |

B


GCCAGTGGACGGCCACAC............CCCCGCGGATAGGTGGCAGCGGGGGCGGACGAG........CCCAGAGCAG
* R G T H . . . . P R D M
vanRp
GGTGTCGAGATTACGAGTCGCCACATATCGTCGGCATATCGGAAAACCGCATACGTGCC
GAGCTGTTAGTCGTGTGC GTA CCACAGCTCTAATGCTCAGCGGTGTATAGCAGCCGTATAGCTTTTGGCGTATGCACGG
E V I L V R M



vanJp
GGCAACACGCTGCCGCGTTCACTGGGCGT ATG CGTCAG ...........TCCGGAGACGGCTGTACAGG GTGGTGCCTCCGC
CCGTTGTGCGACGGCGCAAAGTGACCCG CATACGCAGTC ...........AGGCCTCTGCCGACAGTGCC
A A . . . G V S W *
AGCGCC...........GGGTCAGCTGG TGAGCCCGACGGGCAGGGTGCGGTCCGGTCCCGACCCAGCCCTGCCCGA



CCCAGCCC........ATCGAAAACCACA TACGCGTCGGCAACGTCCCCGGCCTTGA CTGGGGG CATGGCCTACAGC...
. . R R G R *
........CGACGTGGTCGA TGA GCCGCCACCCACGACATTACTGATA TAGGTAATATATCGTCGAAAAGCGCATACGGGACG
M T F S S T . . R K Q H G *



GGCAACACCGAGGCGCCTTGAATAGAGGC ATG ACCTTCTCCTCGACC ...........AGGAAACAGC ATGGCTGAGTCGGAC.
. . T G K L R *
........ACGGGGAAGCTCCG ATG ACCGGGGGACTTCGCC ...........CTTCGACTTTCCATCGCGT TAGGCGTGCGAGC
M T G C F A . . . . F D F P I A *


Fig. 4. A. High-resolution S1 nuclease mapping of the 5' ends of the vanR, vanJ, vanK and vanH transcripts using PCR-generated probes (Fig. 2: probes 1, 2, 3 and 4). The most likely transcription start points are indicated by the asterisks. Lanes A, C, G and T represent a dideoxy sequencing ladder generated using the same oligonucleotide that was used to make the S1 nuclease mapping probe.

B. Partial nucleotide sequence of the vancomycin resistance cluster. Open reading frames are represented by shaded arrows, and the promoters are marked by solid arrows on the nucleotide sequence. The start and stop codons of each gene are boxed with solid and dotted lines respectively. Inverted repeat sequences downstream of vanJ and vanK, which may function as factor-independent transcriptional terminators, are indicated as dashed arrows.

antibiotics that have a similar peptide backbone structure  
to vancomycin but differ in their glycosylation patterns. In  
contrast, all the penicillins and cephalosporins tested  
failed to act as inducers. We showed previously that the  

A  

| Penicillin G | Amoxicillin | Ticarcillin |
|--------------|-------------|------------|
| (30 μg)      | (25 μg)     | (75 μg)    |
| Cephalixin   | Cefadroxil  | Cephradine |
| (30 μg)      | (30 μg)     | (30 μg)    |
| Teicoplanin  | Ristocetin  | Vancomycin |
| (30 μg)      | (30 μg)     | (30 μg)    |
| Thiostrepton | Streptomycin| Novobiocin  |
| (50 μg)      | (10 μg)     | (50 μg)    |

B  

Fig. 5. Response of the *vanR*, *vanJ*, *vanK* and *vanH* promoters to  
vancomycin in (A) M600 and (B) M145 and M145*vanR*. The *sigE*  
promoter was used as a positive control to ensure that the absence  
of signals in the vancomycin-sensitive *vanR* null mutant was not due  
to cell death. Strains were grown to mid–late exponential phase in  
NMMP liquid medium exposed to 10 μg ml⁻¹ vancomycin. RNA was  
extracted from samples taken immediately before the addition of  
vancomycin and at subsequent 10 or 30 min intervals and analysed  
by S1 nuclease protection assays.  

plJ6883 were spread on plates carrying a lethal concent-  
ration of kanamycin (5 μg ml⁻¹), and potential inducers  
were applied on paper discs to the freshly spread plates.  
Inducers of *vanJp* raised the level of expression of the  
*neo* gene and hence induced a halo of kanamycin-  
resistant growth around the disc (Fig. 6A). Only four of  
the compounds tested induced *vanJp*: vancomycin, risto-  
cetin, chloroeremomycin (data not shown) and A47934  
(data not shown). These compounds are glycopeptide  

© 2004 Blackwell Publishing Ltd, *Molecular Microbiology*, 52, 1107–1121  

Fig. 6. A. Response of the *vanJp-neo* fusion to cell envelope-specific  
antibiotics. Approximately 10⁷ spores of *S.* coelicolor M600 carrying  
plJ6883 were spread on 12 cm × 12 cm plates containing 5 μg ml⁻¹  
kanamycin. Antibiotics were applied on 6 mm paper discs to the  
freshly spread plate. Inducers of the *vanJ* promoter raised the level  
of expression of the *neo* gene and hence induced a halo of kanamycin-  
resistant growth around the disc. Antibiotics that target the ribosome  
(thiostrepton, streptomycin) or DNA gyrase (novobiocin) served as  
negative controls. The amount of antibiotic in each disc is indicated.  
B. Induction of the *van* genes by vancomycin confers resistance to  
teicoplanin. A vancomycin disc was placed close to a teicoplanin disc  
on a freshly spread lawn of *S.* coelicolor M600 on SFM agar and  
incubated for 4 days at 30°C. Note the D-shaped zone of killing  
around the teicoplanin disc. Note also that vancomycin itself does not  
cause a zone of killing.

CseBC two-component cell envelope signal transduction system in *S. coelicolor* is induced by the cell wall hydrolytic enzyme lysozyme, in addition to a broad range of cell wall-specific antibiotics (Hong et al., 2002). However, lysozyme failed to induce the *vanJp-neo* fusion (data not shown).

**Induction of the van cluster confers resistance to teicoplanin**

Strikingly, the glycopeptide teicoplanin did not act as an inducer of the *van* signal transduction system (Fig. 6A), and further investigation showed that *S. coelicolor* is sensitive to this antibiotic. To determine whether induction of the *van* genes would confer resistance to teicoplanin, a vancomycin disc was placed close to a teicoplanin disc on a freshly spread lawn of *S. coelicolor* M600. A D-shaped zone of killing was observed around the teicoplanin disc, with the flat side of the D closest to the vancomycin disc, indicating that induction of the *van* genes by vancomycin confers resistance to teicoplanin (Fig. 6B). Thus, expression of the *van* genes confers resistance to the glycopeptide teicoplanin, but *S. coelicolor* is sensitive to teicoplanin in isolation because it is not an inducer of the VanRS signal transduction system.

**Construction and characterization of van gene null mutants – *vanK* is a novel gene required for vancomycin resistance**

*vanR*, *vanJ*, *vanK*, *vanH*, *vanA*, *vanX*, and *vanHAX* null mutants (Fig. 2) of *S. coelicolor* M600 were constructed and their structures confirmed as described in Experimental procedures. In each case, the coding sequence was replaced by an apramycin resistance cassette (*apr*), leaving open the possibility of polar effects on downstream genes for disruption of *vanR*, *vanH*, and *vanA*. The minimum inhibitory concentration (MIC) of vancomycin on MMCGT medium was determined for M600 and for each mutant in turn. M600 grew on MMCGT plates carrying concentrations of vancomycin in excess of 100 μg ml⁻¹. The *vanJ* mutant showed slightly increased sensitivity to vancomycin, growing on 50 μg ml⁻¹, but failing to grow on 100 μg ml⁻¹. In contrast, the *vanK*, *vanR*, *vanH*, *vanA*, *vanX*, and *vanHAX* mutants all failed to grow on 10 μg ml⁻¹ vancomycin (lower levels not tested). The importance of the peptidoglycan remodelling enzymes VanHAX and the positive regulator VanR in vancomycin resistance was to be expected. However, the discovery that the Fem homologue VanK was essential for resistance was surprising given the absence of *vanK* homologues in vancomycin resistance clusters from pathogens. Although transcriptional studies (see above) suggested that the *vanK* null mutation should not be polar on the downstream *vanHAX* operon, we confirmed this by complementation *in trans*. pLJ6892 (Fig. 2), carrying *vanK* alone, restored wild-type levels of vancomycin resistance to the *vanK* null mutant, whereas the parental vector, pLJ82, had no effect.

**Cross-talk between the vancomycin resistance system and the sigE signal transduction system**

In the analysis of the induction of *sigEp*, used as a positive control in the data shown in Fig. 5B, there was an interesting difference between the response in the wild type and in the *vanR* mutant. In the wild type, induction peaked at 30 min before declining, as seen previously (Hong et al., 2002), but *sigE* transcripts were still at maximal levels after 90 min in the *vanR* mutant, implying that expression of the *van* gene cluster indirectly affected induction of the *sigE* signal transduction system. To pursue potential cross-talk between the *van* signal transduction system and the *sigE* signal transduction system, we also examined the expression of *sigEp* in the *vanR* mutant, and reciprocally examined the expression of the *van* genes in a *sigE* null mutant.

We found that, whereas 1 μg ml⁻¹ vancomycin induced *sigEp* in the *vanR* mutant, this concentration was insufficient to induce *sigEp* in its congenic parent M600 (Fig. 7A). Among other possibilities, our favoured interpretation of these data is as follows. Activation of the inducible vancomycin resistance gene cluster causes the progressive replacement of D-Ala-D-Ala with D-Ala-D-Lac, rapidly leading to vancomycin insensitivity. As a consequence, vancomycin rapidly fails to cause the accumulation of the hypothetical intermediate in peptidoglycan biosynthesis or degradation that we have previously proposed to act as the direct molecular inducer of the *sigE* sensor kinase, CseC (Hong et al., 2002). However, in the *vanR* mutant, no such remodelling occurs, such that *sigE* transcripts are still at maximal levels after 90 min in the presence of 10 μg ml⁻¹ vancomycin (Fig. 5B), and lower concentrations of vancomycin are effective for induction (Fig. 7A).

When, conversely, we examined the induction of *vanJp* and *vanKp* in a *sigE* null mutant, we found that, although the *van* promoters were still activated, induction was clearly much weaker in the *sigE* mutant compared with the wild type (Fig. 7B), and this result was confirmed independently using microarray analysis (M. I. Hutchings, unpublished data). Although we do not understand the molecular connections underlying these observations, they clearly illustrate that signal transduction systems controlling different aspects of cell envelope homeostasis can indirectly impinge on each other.

A
M600
vanR mutant
0 30 60 90 0 30 60 90 min.
—sigEp

B
M600
sigE mutant
0 10 20 30 40 50 60 90 0 10 20 30 40 50 60 90 min.
—vanJp
—vanKp

Fig. 7. Cross-talk between the vancomycin resistance system and the sigE signal transduction system.
A. Vancomycin (1 μg ml⁻¹) induces the sigE promoter in a vanR null mutant but not in a wild-type background.
B. Induction of the van promoters is impaired in a sigE null mutant. Strains were grown to mid-late exponential phase in NMMP liquid medium and exposed to 1 μg ml⁻¹ (A) or 10 μg ml⁻¹ (B) vancomycin. RNA was extracted from samples taken immediately before the addition of vancomycin and at subsequent 10 or 30 min intervals and analysed by S1 nuclease protection assays. The two vanJp panels are from the same autoradiograph and are therefore directly comparable, as are the two vanKe panels.

Discussion

A model for the vanRS signal transduction system

We have identified and characterized a novel vancomycin resistance cluster in the non-pathogenic, non-glycopeptide-producing actinomycete S. coelicolor. Based on the work presented here, we propose a model for the regulation of these genes. Expression of the seven van genes is regulated at the level of transcription by the VanR/VanS two-component signal transduction system. In response to extracellular glycopeptide antibiotic, the sensor kinase, VanS, becomes autophosphorylated at His-150 and, in accordance with the known mechanism for other two-component regulatory systems, this phosphate is then transferred to Asp-51 in the response regulator, VanR. Phospho-VanR then activates the vanR, vanJ, vanK and vanH promoters, leading to expression of vancomycin resistance.

The vanRS transcript is undetectable by S1 nuclease protection in the wild type in the absence of induction (Figs 5 and 7). Given that transcription of the van genes is completely dependent on vanR (Fig. 5B), it presumably follows that the vanRS operon must be transcribed at some level (albeit too low to be detected) in the absence of vancomycin, such that VanR and VanS are present to mediate induction.

What is the ligand recognized by VanS?

We previously established a bioassay, analogous to the one reported here, to study the induction of the cseBC-sigE signal transduction system (Hong et al., 2002). Using this bioassay, we showed that the CseBC two-component system responded to a wide range of cell wall-specific antibiotics with completely different structures and varied targets in the cell envelope. All these compounds were inhibitors of late steps in peptidoglycan biosynthesis. We also showed that the cell wall hydrolytic enzyme lysozyme acted as an inducer. From these data, we suggested that CseC might be activated by an intermediate in peptidoglycan biosynthesis or degradation that accumulates as a consequence of the activities of the inducers (Hong et al., 2002). In contrast, the only inducers of the van signal transduction system that we detected using our vanJp—

© 2004 Blackwell Publishing Ltd, Molecular Microbiology, 52, 1107–1121

1116 H.-J. Hong et al.

neo bioassay were vancomycin, ristocetin, chloroeremycin and A47934, all structurally closely related glycopeptide antibiotics. Although these results do not rule out the possibility that VanS responds to an intermediate in PG biosynthesis or degradation that accumulates uniquely in response to glycopeptide antibiotics, they are consistent with VanS interacting directly with the drug; validation of this hypothesis will require additional biochemical analysis. The predicted extracytoplasmic sensor domains of VanS and CseC are very different in size (27 and 104 amino acids respectively) and sequence.

Why does the glycopeptide teicoplanin fail to induce the van signal transduction system?

Teicoplanin is important as the only glycopeptide antibiotic apart from vancomycin in current clinical use. The most interesting result from the vanJp-neo bioassay was that teicoplanin did not act as an inducer of the van signal transduction system, whereas all the other glycopeptide antibiotics tested did (Fig. 6A). If VanS is activated by direct binding of glycopeptides to its extracytoplasmic sensor domain, then our results would suggest that some aspect of teicoplanin structure prevents it from interacting with VanS. The most conspicuous difference between teicoplanin and the other glycopeptides is a long fatty acid chain attached to the vancosamine sugar of teicoplanin. This hydrophobic moiety serves to anchor teicoplanin in the membrane (Beauregard et al., 1995; Sharman et al., 1997; Cooper and Williams, 1999), and it is possible that membrane anchoring prevents teicoplanin from interacting with the VanS sensor domain. Alternatively, most glycopeptide antibiotics such as vancomycin dimerize, but teicoplanin does not (Beauregard et al., 1995; Cooper and Williams, 1999). Thus, a further possibility is that glycopeptides have to interact with the S. coelicolor VanS sensor domain as dimers, and teicoplanin therefore fails to induce the VanRS signal transduction system. Such possibilities raise interesting questions about the evolution of this antibiotic. Perhaps the salient features of teicoplanin structure have been selected for during the evolution of the producing organism, Actinoplanes teichomyceticus, in part because they prevent competing soil bacteria (e.g. S. coelicolor) from sensing the antibiotic and generating a resistance response.

Although we favour direct interaction of the inducers with VanS, until this is demonstrated biochemically, the possibility that VanS is indirectly activated by glycopeptides through the accumulation of an intermediate in PG biosynthesis cannot be excluded. If this were the case, it would imply that teicoplanin and vancomycin cause different sets of cell wall intermediates to accumulate. For this to happen, the in vivo mode of action of teicoplanin would have to be significantly different from that of vancomycin

and, in fact, there is now compelling evidence that membrane-anchored glycopeptides interact with another target in addition to D-Ala-D-Ala. Ge et al. (1999) showed that synthetic vancomycin analogues carrying hydrophobic substituents on the vancosamine sugar were active against vancomycin-resistant strains even when an essential portion of the D-Ala-D-Ala-binding pocket of the antibiotic had been removed, and Chen et al. (2003) showed that these hydrophobic derivatives can inhibit transglycosylase directly, without binding D-Ala-D-Ala. Furthermore, Sinha Roy et al. (2001) showed that the major transglycosylase of E. coli (PBP1β) is retained on an affinity column carrying a synthetic vancomycin analogue with a hydrophobic substituent. Thus, vancomycin primarily inhibits the transpeptidation step, whereas membrane-anchored glycopeptides primarily inhibit the transglycosylation step (Ge et al., 1999).

Why is the novel gene vanK required for vancomycin resistance?

We have shown that a novel resistance gene, vanK, which has no counterpart in previously characterized vancomycin resistance clusters from pathogens, is required for vancomycin resistance in S. coelicolor. VanK is a member of the Fem family of proteins, which are non-ribosomal peptidyltransferases that add the cross-bridge amino acid(s) to PG precursors (Hegde and Shrader, 2001; Hegde and Blanchard, 2003; Rohrer and Berger-Bächi, 2003). For example, in S. aureus, which has a pentaglycine cross-bridge, three Fem proteins (FemA, FemB and FemX) are involved. Rohrer et al. (1999) showed that femX (also called fmhB) is an essential gene in S. aureus, and that controlled depletion of its expression results in the disappearance of glycine-substituted PG precursors. In contrast, femA, femB and femAB mutants can be constructed, although they have to acquire compensatory suppressor mutations to survive. Disruption of femB leads to shortening of the staphylococcal cross-bridge from five to three glycines, whereas loss of femA or femAB reduces the cross-bridge to a single glycine. From these results, it was deduced that FemX adds the first glycine, FemA adds the second and third glycine, and FemB adds the fourth and fifth glycine (Henze et al., 1993; Kopp et al., 1996; Ehlert et al., 1997; Strandén et al., 1997; Rohrer et al., 1999; Rohrer and Berger-Bächi, 2003). In Streptomyces, the cross-bridge is a single glycine, yet S. coelicolor has three fem genes: vanK, SCO0602 and SCO3904 (Fig. 3).

Why is a Fem protein encoded within the van gene cluster in S. coelicolor and why is vanK required for vancomycin resistance? One possibility is that, in order to attain resistance, S. coelicolor must both (i) convert D-Ala to D-Ala-D-Lac through the action of VanHAX; and (ii) change the cross-bridge through the action of VanK.

Changes in the sequence of the cross-bridge have been shown to result in increased levels of resistance to vancomycin and teicoplanin in *Staphylococcus haemolyticus* (Billot-Klein et al., 1996), suggesting that glycopeptide resistance is multifactorial in this organism. Billot-Klein et al. (1996) suggested that changes in the sequence of the cross-bridge might interfere with the co-operativity of binding of glycopeptide dimers *in situ*. A second possibility is that the constitutive FemX activity of *S. coelicolor* (presumably either SCO0602 or SCO3904) might recognize only precursors that terminate in D-Ala-D-Ala as a substrate, and VanK would therefore be required for vancomycin resistance because it would be the only enzyme that could add the Gly cross-bridge to precursors terminating in D-Ala-D-Lac. Production of precursors lacking a cross-bridge would be lethal because it would prevent cross-linking of the peptidoglycan by transpeptidase, leading to cell lysis. We are currently pursuing experiments designed to distinguish between these two possibilities.

Circumstantial evidence that a VanK enzyme is important for glycopeptide resistance not just in *S. coelicolor* but in the genus *Streptomyces* in general comes from *Streptomyces toyocaensis*, the producer of the glycopeptide antibiotic A47934. The nucleotide sequence of the entire A47934 biosynthetic cluster, including the associated resistance genes, has recently been published (Pootoolal et al., 2002b) and, significantly, an orthologue of *S. coeli-*

Vancomycin resistance in *S. coelicolor* 1117

color vanK is present in the A47934 resistance gene cluster, as is an orthologue of *S. coelicolor* vanJ.

### Experimental procedures

#### Strains, plasmids, media, growth conditions and conjugal plasmid transfer from E. coli to Streptomyces spp.

Bacterial strains and plasmids used are described in Table 1. Except where described below, media and culture conditions were as given previously (Kieser et al., 2000; Hong et al., 2002). MMCGT is agar minimal medium (MM; Kieser et al., 2000) supplemented with 0.6% (w/v) Difco casamino acids, 0.75% (v/v) Tiger milk (Kieser et al., 2000), 0.5% (w/v) glucose. To bypass the methyl-specific restriction system of *S. coelicolor*, pIJ82 and its derivatives were introduced by transformation into the *dam dcm hsdS E. coli* strain ET12567 containing the driver plasmid pUZ8002. Conjugal transfer of unmethylated plasmids between *E. coli* ET12567 and *S. coelicolor* was carried out essentially as described by Kieser et al. (2000). All polymerase chain reaction (PCR) products were initially cloned into the vector pGEM-T (Promega) and, where relevant, sequenced before restriction digestion.

#### RNA preparation and transcriptional analysis

For RNA preparation, spores of *S. coelicolor* strains (equivalent to an initial OD<sub>450</sub> of ≈0.025 in the final NMMP culture)

Table 1. Bacterial strains and plasmids used in this study.

| Strain/plasmid | Relevant genotype/comments                                                                 | Source/reference           |
|----------------|------------------------------------------------------------------------------------------|----------------------------|
| *S. coelicolor* A3(2) | SCP1<sup>-</sup> SCP2<sup>-</sup>                                                    | Kieser et al. (2000)       |
| M145           | SCP1<sup>-</sup> SCP2<sup>-</sup>                                                        | Kieser et al. (2000)       |
| M600           | ΔvanR::apr SCP1<sup>-</sup> SCP2<sup>-</sup>                                             | This study                 |
| J2174          | ΔvanJ::apr SCP1<sup>-</sup> SCP2<sup>-</sup>                                             | This study                 |
| J3220          | ΔvanK::apr SCP1<sup>-</sup> SCP2<sup>-</sup>                                             | This study                 |
| J3221          | ΔvanH::apr SCP1<sup>-</sup> SCP2<sup>-</sup>                                             | This study                 |
| J3223          | ΔvanA::apr SCP1<sup>-</sup> SCP2<sup>-</sup>                                             | This study                 |
| J3224          | ΔvanX::apr SCP1<sup>-</sup> SCP2<sup>-</sup>                                             | This study                 |
| J3225          | ΔvanHAX::apr SCP1<sup>-</sup> SCP2<sup>-</sup>                                           | This study                 |
| *E. coli*      | ET12567 containing helper plasmid pUZ8002                                                | Paget et al. (1999)        |
| ET12567(pUZ8002)| Temperature-sensitive FLP recombination plasmid                                         | Datsenko and Wanner (2000) |
| BT340          |                                                                                          |                            |
| Plasmids       |                                                                                          |                            |
| pIJ82          | Plasmid cloning vector for the conjugal transfer of DNA from *E. coli* to *Streptomyces* spp. Integrates site-specifically at the ΦC31 attachment site (Hyg<sup>R</sup>) | H. Kieser (unpublished)     |
| pIJ486         | Multicopy *Streptomyces* promoter probe plasmid containing neo as reporter gene (Thio<sup>R</sup>) | Ward et al. (1986)         |
| pIJ773         | pBluescript KS+ containing the apramycin resistance gene apr and the oriT of plasmid RP4, flanked by FRT sites | Gust et al. (2003)         |
| pIJ790         | Modified λ RED recombination plasmid pKD20                                               | Gust et al. (2003)         |
| pIJ6882        | 0.25 kb *vanJ* promoter PCR product in pGEM T vector                                     | This study                 |
| pIJ6883        | 0.25 kb *vanJ* promoter region (HindIII–EcoRI) in pIJ486                                 | This study                 |
| pIJ6889        | pIJ82 with a 5.4 kb *BamHI* fragment from cosmid SCH66                                    | This study                 |
| pIJ6890        | pIJ82 with a 3.6 kb *BamHI–EcoRV* fragment from cosmid SCH66                              | This study                 |
| pIJ6891        | pIJ82 with a 2.18 kb *BamHI–PvuII* fragment from cosmid SCH66                             | This study                 |
| pIJ6892        | pIJ82 with a 1.35 kb *PvuII–EcoRV* fragment from cosmid SCH66                             | This study                 |

© 2004 Blackwell Publishing Ltd, *Molecular Microbiology*, 52, 1107–1121

Table 2. Oligonucleotides used in this study.

| vanR KO I | TGCCGACGATATGTGGCGACTCGTAATCTCGACACCATGATTCCGGGGATCCGTCGACC |
| --- | --- |
| vanR KO II | AGCTTGAGGCGGACGCTCAGACCGGGGCGCCTATCCACCTGTAGGCTGGAGCTGCTTC |
| vanJ KO I | GCACCGGCGCCAGCACCGGCTCGGAGACGGCTGTCACGGATTCCGGGGATCCGTCGACC |
| vanJ KO II | CTGGGTCGGACCGGACCACCGCAACCTGCCCCTGGGGCTGTAGGCTGGAGCTGCTTC |
| vanK KO I | CATACGCGTCGGCAACGTCCCCGCCGCCTTGACTGGGGGCATTCCGGGGATCCGTCGACC |
| vanK KO II | GACGATATATTACCTATCAGTAATGTCGTGGTGGCGGCTGTAGGCTGGAGCTGCTTC |
| vanH KO I | ACGGGACGGCAAACACCAGGGCGCCTTGAATAGAGGCATGATTCCGGGGATCCGTCGACC |
| vanH KO II | CCGACCTTCAACCTGGCCAACTTGTCCGACTCAGCCATGTGTAGGCTGGAGCTGCTTC |
| vanA KO I | AACTGCCCTTAATTTCCGGAAGCAGGAAAACAGCATGGCTGAATTCCGGGGATCCGTCGACC |
| vanA KO II | GGACACCAGCTCGTCCACGAAAGGCTCCCCGGTCATTGTAGGCTGGAGCTGCTTC |
| vanX KO I | CGGACGCTGTCCCTGGCGTTGACGGGGAAGCTCCGATGAATTCCGGGGATCCGTCGACC |
| vanX KO II | CTCGGCCCCGGCTCGGCTCGGCTCGGCACGGCCTACGTGTAGGCTGGAGCTGCTTC |

were germinated by heat shock treatment in 5 ml of TES buffer (0.05 M, pH 8) at 50°C for 10 min, then diluted with an equal volume of double-strength germination medium [1% (w/v) Difco yeast extract, 1% (w/v) Difco casamino acids, 0.01 M CaCl₂] and incubated with shaking at 37°C for 2–3 h (Kieser et al., 2000). Germinated spores were inoculated into NMMP (Kieser et al., 2000) and grown to an OD₅₀ of 0.3–0.6 (10–12 h) at 30°C before treatment with vancomycin. RNA was prepared as described by Hong et al. (2002). Probes for S1 nuclease protection analysis were generated by PCR from S. coelicolor M600 chromosomal DNA using a 5′ end-labelled oligonucleotide internal to the ORF and an unlabelled upstream primer (Table 2). For high-resolution mapping of the 5′ end of the *van* transcripts, sequencing ladders were generated with the Taq Track kit (Promega), using PCR products cloned into the vector pGEM-T (Promega) as template DNA and the same radiolabelled downstream oligonucleotide primer that was used to generate the S1 nuclease mapping probe.

were scored after incubation at 30°C for 4 days. Chloroeremomycin was a kind gift from Professor Dudley Williams. Other antibiotics and preformed antibiotic test discs were purchased from Sigma and Oxoid, respectively, and 6 mm paper discs were purchased from Whatman.

### Construction of a *vanR* mutant of M145

A 2.5 kb DNA fragment carrying *vanR* was amplified by PCR, cloned into pSTBlue (Novagen), and a 1 kb apramycin resistance cassette (*apr*) was inserted into the unique *PstI* site internal to *vanR*. This construct was introduced into M145 by protoplast transformation, and transformants were selected with apramycin. After selective growth on apramycin-containing plates, a *vanR* null mutant derivative of M145, generated by double crossing over, was identified by its apramycin-resistant, kanamycin-sensitive phenotype, and its structure was confirmed by Southern hybridization.

### Construction of *van* gene mutants of M600

*van* mutant derivatives of S. coelicolor M600, in which each *van* gene coding sequence was precisely replaced by *apr* (conferring resistance to apramycin), were constructed by the PCR-targeting method of Gust et al. (2003). In this method, S. coelicolor genes carried on cosmids in E. coli are replaced with a selectable marker generated by PCR using primers with 39 nucleotide gene-specific extensions. The selectable marker cassette also includes *oriT*, which permits the direct transfer of the mutagenized cosmid into S. coelicolor by conjugation (Gust et al., 2003). Cosmid H66, which carries the entire vancomycin resistance cluster, was introduced into E. coli BW25113 (Datsenko and Wanner, 2000) containing plJ790 (carrying the λ RED recombination genes, *gam*, *bet* and *exo*). Each *van* disruption was created separately by electroporation with an *oriT/apr* cassette that had been amplified using the oligonucleotides shown in Table 2, which carried specific extensions for each *van* gene in turn. Gene disruptions were confirmed by restriction and PCR analyses of isolated cosmid DNA. The resulting cosmids carrying the disruption cassette in place of each gene were individually introduced by electroporation into the methylation-deficient E. coli strain ET12567 carrying the helper plasmid pUZ8002, and then transferred to S. coelicolor M600 by conjugation, and exconjugants were selected with apramycin. The resulting colonies were replica plated onto SFM containing

### Bioassay for inducers of the *vanJ* signal transduction system

Bioassay experiments were performed on MMC GT containing 5 μg ml⁻¹ kanamycin. Approximately 10⁷ spores of S. coelicolor M600 carrying plJ6883 were spread on each 12 cm × 12 cm plate, and potential inducers carried on 6 mm paper discs were applied to the freshly spread plates. Plates

50 µg ml⁻¹ apramycin and onto SFM containing 20 µg ml⁻¹ kanamycin. van null mutant derivatives of M600, generated by double crossing over, were identified by their apramycin-resistant, kanamycin-sensitive phenotypes, their structures were confirmed by PCR analysis and Southern hybridization, and representative van gene null mutants were designated as indicated in Table 1 and Fig. 2.

### Complementation of the vanK null mutant

A 5.4 kb BamHI fragment, carrying vanRJKH, was isolated from cosmid H66 and cloned into plJ82 digested with the same enzyme to create plJ6889. plJ6889 was cut with EcoRV and self-ligated to create plJ6890, which carries a 3.6 kb insert containing vanRJK. Finally, a 1.35 kb PvuII-EcoRV fragment carrying vanK was isolated from plJ6890 and cloned into plJ82 cut with EcoRV to create plJ6892. Plasmids were introduced into the vanK mutant by conjugal transfer from E. coli strain ET12567(pUZ8002), and exconjugants were selected with hygromycin.

### Acknowledgements

We thank Dudley Williams for providing chloroeremomycin, Anatoly Severin, David Hopwood, Keith Chater and Mike Merrick for comments on the manuscript, and Christopher Walsh, Greg Challis, Subray Hegde and Tim Bugg for helpful discussion. This work was funded by BBSRC grant 208/P20040, a grant-in-aid to the John Innes Centre from the BBSRC and also by a grant from the Canadian Institute of Health Research (MT-14981) to Professor Gerard D. Wright.

### References

Barna, J.C.J., and Williams, D.H. (1984) The structure and mode of action of glycopeptide antibiotics of the vancomycin group. Annu Rev Microbiol 38: 339–357.

Beauregard, D.A., Williams, D.H., Gwynn, M.N., and Knowles, D.J. (1995) Dimerization and membrane anchors in extracellular targeting of vancomycin group antibiotics. Antimicrob Agents Chemother 39: 781–785.

Benson, T.E., Prince, D.B., Mutchler, V.T., Curry, K.A., Ho, A.M., Sarver, R.W., et al. (2002) X-ray crystal structure of Staphylococcus aureus FemA. Structure 10: 1107–1115.

Bentley, S.D., Chater, K.F., Cerdeño-Tárraga, A.M., Challis, G.L., Thomson, N.R., James, K.D., et al. (2002) Complete genome sequence of the model actinomycete Streptomyces coelicolor A3(2). Nature 417: 141–147.

Bibb, M.J., Findlay, P.R., and Johnson, M.W. (1984) The relationship between base composition and codon usage in bacterial genes and its use for the simple and reliable identification of protein-coding sequences. Gene 30: 157–166.

Bibb, M.J., White, J., Ward, J.M., and Janssen, G.R. (1994) The mRNA for the 23S rRNA methylase encoded by the ermE gene of Saccharopolyspora erythraea is translated in the absence of a conventional ribosome-binding site. Mol Microbiol 14: 533–545.

Billot-Klein, D., Blanot, D., Gutmann, L., and van Heijenoort, J. (1994) Association constants for the binding of vancomycin and teicoplanin to N-acetyl-d-alanyl-d-alanine and N-acetyl-d-alanyl-d-serine. Biochem J 304: 1021–1022.

Billot-Klein, D., Gutmann, L., Bryant, D., Bell, D., van Heijenoort, J., Grewal, J., and Shlaes, D.M. (1996) Peptidoglycan synthesis and structure in Staphylococcus haemolyticus expressing increasing levels of resistance to glycopeptide antibiotics. J Bacteriol 178: 4696–4703.

Bugg, T.D.H. (1999) Bacterial peptidoglycan biosynthesis and its inhibition. In Comprehensive Natural Products Chemistry, Vol. 3. Pinto, M. (ed.). Oxford: Elsevier Science, pp. 241–294.

Bugg, T.D.H., Wright, G.D., Dutka-Malen, S., Arthur, M., Courvalin, P., and Walsh, C.T. (1991) Molecular basis for vancomycin resistance in Enterococcus faecium BM4147. biosynthesis of a depsipeptide peptidoglycan precursor by vancomycin resistance proteins VanH and VanA. Biochemistry 30: 10408–10415.

Chang, S., Sievert, D.M., Hageman, J.C., Boulton, M.L., Tenover, F.C., Downes, F.P., et al. (2003) Infection with vancomycin-resistant Staphylococcus aureus containing the vanA resistance gene. N Engl J Med 348: 1342–1347.

Chen, L., Walker, D., Sun, B., Hu, Y., Walker, S., and Kahne, D. (2003) Vancomycin analogues active against vanA-resistant strains inhibit bacterial transglycosylase without binding substrate. Proc Natl Acad Sci USA 100: 5658–5663.

Chu, D.T., Plattner, J.J., and Katz, L. (1996) New directions in antibacterial research. J Med Chem 39: 3853–3874.

Cooper, M.A., and Williams, D.H. (1999) Binding of glycopeptide antibiotics to a model of a vancomycin-resistant bacterium. Chem Biol 6: 891–899.

Datsenko, K.A., and Wanner, B.L. (2000) One step inactivation of chromosomal genes in Escherichia coli K-12 using PCR products. Proc Natl Acad Sci USA 97: 6640–6645.

Egger, L.A., Park, H., and Inouye, M. (1997) Signal transduction via the histidyl-aspartyl phospho-relay. Genes Cells 2: 167–184.

Ehlert, K., Schröder, W., and Labischinski, H. (1997) Specificities of FemA and FemB for different glycine residues: FemB cannot substitute for FemA in staphylococcal peptidoglycan side chain formation. J Bacteriol 179: 7573–7576.

Ge, M., Chem, Z., Onishi, H.R., Kohler, J., Silver, L.L., Kerns, R., et al. (1999) Vancomycin derivatives that inhibit peptidoglycan biosynthesis without binding d-Ala-d-Ala. Science 284: 507–511.

Gust, B., Challis, G.L., Fowler, K., Kieser, T., and Chater, K.F. (2003) Gene replacement by PCR targeting in Streptomyces and its use to identify a protein domain involved in the biosynthesis of the sesquiterpene odour geosmin. Proc Natl Acad Sci USA 100: 1541–1546.

Hackenbeck, R., and Stock, J.B. (1996) Analysis of two-component signal transduction systems involved in transcriptional regulation. Methods Enzymol 273: 281–300.

Healy, V.L., Lessard, I.A., Roper, D.I., Knox, J.R., and Walsh, C.T. (2000) Vancomycin resistance in enterococci: reprogramming of the d-Ala-d-Ala ligases in bacterial peptidoglycan biosynthesis. Chem Biol 7: R109–R119.

Hegde, S.S., and Blanchard, J.S. (2003) Kinetic and mechanistic characterization of recombinant Lactobacillus viridescens FemX (UDP-N-acetylmuramoyl pentapeptide-

lysine $N^6$-alanyltransferase). J Biol Chem 278: 22861–22867.

Hegde, S.S., and Shrader, T.E. (2001) FemABX family members are novel nonribosomal peptidyltransferases and important pathogen-specific drug targets. J Biol Chem 276: 6998–7003.

Henze, U., Sidow, T., Wecke, J., Labischinski, H., and Berger-Bächi, B. (1993) Influence of *femB* on methicillin resistance and peptidoglycan metabolism in *Staphylococcus aureus*. J Bacteriol 175: 1612–1620.

Hong, H.-J., Paget, M.S.B., and Buttner, M.J. (2002) A signal transduction system in *Streptomyces coelicolor* that activates the expression of a putative cell wall glycan operon in response to vancomycin and other cell wall-specific antibiotics. Mol Microbiol 44: 1199–1211.

Ikeda, H., Ishikawa, J., Hanamoto, A., Shinose, M., Kikuchi, H., Shiba, T., *et al.* (2003) Complete genome sequence and comparative analysis of the industrial microorganism *Streptomyces avermitilis*. Nature Biotechnol 21: 526–531.

Janssen, G.R., Ward, J.M., and Bibb, M.J. (1989) Unusual transcriptional and translational features of the aminoglycoside phosphotransferase gene (*aph*) from *Streptomyces fradiae*. Genes Dev 3: 415–429.

Kieser, T., Bibb, M.J., Buttner, M.J., Chater, K.F., and Hopwood, D.A. (2000) Practical Streptomyces Genetics. Norwich: The John Innes Foundation.

Kopp, U., Roos, M., Wecke, J., and Labischinski, H. (1996) Staphylococcal peptidoglycan interpeptide bridge biosynthesis: a novel antistaphylococcal target? Microb Drug Resist 2: 29–41.

Leyh-Bouille, M., Bonaly, R., Ghuysen, J.-M., Tinelli, R., and Tipper, D. (1970) LL-diaminopimelic acid containing peptidoglycans in walls of *Streptomyces* sp. and of *Clostridium perfringens* (type A). Biochemistry 9: 2944–2952.

Mizuno, T., and Tanaka, I. (1997) Structure of the DNA-binding domain of the OmpR family of response regulators. Mol Microbiol 24: 665–670.

O'Donnell, S.M., and Janssen, G.R. (2002) Leaderless mRNAs bind 70S ribosomes more strongly than 30S ribosomal subunits in *Escherichia coli*. J Bacteriol 184: 6930–6933.

Paget, M.S.B., Chamberlin, L., Atrih, A., Foster, S.J., and Buttner, M.J. (1999) Evidence that the extracytoplasmic function sigma factor, $\sigma^E$, is required for normal cell wall structure in *Streptomyces coelicolor* A3(2). J Bacteriol 181: 204–211.

Pearson, H. (2002) ‘Superbug’ hurdles key drug barrier. Nature 418: 469–470.

Pootoolal, J., Neu, J., and Wright, G.D. (2002a) Glycopeptide antibiotic resistance. Annu Rev Pharmacol Toxicol 42: 381–408.

Pootoolal, J., Thomas, M.G., Marshall, C.G., Neu, J.M., Hubbard, B.K., Walsh, C.T., and Wright, G.D. (2002b) Assembling the glycopeptide antibiotic scaffold: the biosynthesis of A47934 from *Streptomyces toyocaensis* NRRL15009. Proc Natl Acad Sci USA 99: 8962–8967.

Reynolds, P.E., Snaith, H.A., Maguire, A.J., Dutka-Malen, S., and Courvalin, P. (1994) Analysis of peptidoglycan precursors in vancomycin-resistant *Enterococcus gallinarum* BM4174. Biochem J 301: 5–8.

Rohrer, S., and Berger-Bächi, B. (2003) FemABX peptidyl transferases: a link between branched-chain cell wall peptide formation and β-lactam resistance in Gram-positive cocci. Antimicrob Agents Chemother 47: 837–846.

Rohrer, S., Ehlert, K., Tschierske, M., Labischinski, H., and Berger-Bachi, B. (1999) The essential *Staphylococcus aureus* gene *fmhB* is involved in the first step of peptidoglycan pentaglycine interpeptide formation. Proc Natl Acad Sci USA 96: 9351–9356.

Ruzin, A., Severin, A., Ritacco, F., Tabei, K., Singh, G., Bradford, P.A., *et al.* (2002) Further evidence that a cell wall precursor [C(55)-MurNAc-(peptide)-GlcNAc] serves as an acceptor in a sorting reaction. J Bacteriol 184: 2141–2147.

Schleifer, K.H., and Kandler, O. (1972) Peptidoglycan types of bacterial cell walls and their taxonomic implications. Bacteriol Rev 36: 407–477.

Sharman, G.J., Try, A.C., Dancer, R.J., Cho, Y.R., Staroske, T., Bardsley, B., *et al.* (1997) The roles of dimerization and membrane anchoring in activity of glycopeptide antibiotics against vancomycin-resistant bacteria. J Am Chem Soc 119: 12041–12047.

Sinha Roy, R., Yang, P., Kodali, S., Xiong, Y., Kim, R.M., Griffin, P.R., *et al.* (2001) Direct interaction of a vancomycin derivative with bacterial enzymes involved in cell wall biosynthesis. Chem Biol 8: 1095–1106.

Song, M.D., Wachi, M., Doi, M., Ishino, F., and Matsuhashi, M. (1987) Evolution of an inducible penicillin-target protein in methicillin-resistant *Staphylococcus aureus* by gene fusion. FEBS Lett 221: 167–171.

Spratt, B.G. (1994) Resistance to antibiotics mediated by target alterations. Science 264: 388–393.

Spratt, B.G., and Cromie, K.D. (1988) Penicillin-binding proteins of gram-negative bacteria. Rev Infect Dis 10: 699–711.

Stock, A.M., Mottonen, J.M., Stock, J.B., and Schutt, C.E. (1989) Three-dimensional structure of CheY, the response regulator of bacterial chemotaxis. Nature 337: 745–749.

Strandén, A.M., Ehlert, K., Labischinski, H., and Berger-Bächi, B. (1997) Cell wall monoglycine cross-bridges and methicillin hypersusceptibility in a *femAB* null mutant of methicillin-resistant *Staphylococcus aureus*. J Bacteriol 179: 9–16.

Tenover, F.C., Weigel, L.M., Appelbaum, P.C., McDougal, L.K., Chaitram, J., McAllister, S., *et al.* (2004) Vancomycin-resistant *Staphylococcus aureus* isolate from a patient in Pennsylvania. Antimicrob Agents Chemother 48: 275–280.

Udagawa, T., Shimizu, Y., and Ueda, T. (2004) Evidence for the translation initiation of leaderless mRNAs by the intact 70S ribosome without its dissociation into subunits in Eubacteria. J Biol Chem 278: 8539–8646.

Walsh, C.T. (2000) Molecular mechanisms that confer antibacterial drug resistance. Nature 406: 775–781.

Walsh, C.T., Fisher, S.L., Park, I.S., Prahalad, M., and Wu, Z. (1996) Bacterial resistance to vancomycin: five genes and one missing hydrogen bond tell the story. Chem Biol 3: 21–28.

Ward, J.M., Janssen, G.R., Kieser, T., Bibb, M.J., Buttner, M.J., and Bibb, M.J. (1986) Construction and characterization of a series of multi-copy promoter-probe plasmid vec-

tors for *Streptomyces* using the aminoglycoside phosphotransferase gene from Tn5 as indicator. Mol Gen Genet 203: 468–478.

Weigel, L.M., Clewell, D.B., Gill, S.R., Clark, N.C., McDougal, L.K., Flannagan, S.E., *et al.* (2003) Genetic analysis of a high-level vancomycin-resistant isolate of *Staphylococcus aureus*. Science 28: 1569–1571.

Williams, D.H., and Bardsley, B. (1999) Vancomycin group of antibiotics and the fight against resistant bacteria. Angew Chem Int Ed 38: 1172–1193.

Vancomycin resistance in *S. coelicolor* 1121

Williams, D.H., Williamson, M.P., Butcher, D.W., and Hammond, S.J. (1983) Detailed binding sites of the antibiotics vancomycin and ristocetin A: determination of intermolecular distances in antibiotic/substrate complexes by use of the time-dependent NOE. J Am Chem Soc 105: 1332–1339.

Wu, C.J., and Janssen, G.R. (1997) Expression of a streptomyces leaderless mRNA encoding chloramphenicol acetyltransferase in *Escherichia coli*. J Bacteriol 179: 6824–6830.

© 2004 Blackwell Publishing Ltd, Molecular Microbiology, 52, 1107–1121
