
Full text provided by www.sciencedirect.com

ELSEVIER

**Nucleotide sugar interconversions and cell wall biosynthesis: how to bring the inside to the outside**

Georg J Seifert

Plants possess a sophisticated sugar biosynthetic machinery comprising families of nucleotide sugar interconversion enzymes. Literature published in the past two years has made a major contribution to our knowledge of the enzymes and genes involved in the interconversion of nucleotide sugars that are required for cell wall biosynthesis, including UDP-L-rhamnose, UDP-D-galactose, UDP-D-glucuronic acid, UDP-D-xylose, UDP-D-apiose, UDP-L-arabinose, GDP-L-fucose and GDP-L-galactose. Indirect evidence suggests that enzyme activity is crudely regulated at the transcriptional level in a cell-type and differentiation-dependent manner. However, feedback inhibition and NAD⁺/NADH redox control, as well as the formation of complexes between differentially encoded isoforms and glycosyltransferases, might fine-tune cell wall matrix biosynthesis. I hypothesise that the control of nucleotide sugar interconversion enzymes regulates glycosylation patterns in response to developmental, metabolic and stress-related stimuli, thereby linking signalling with primary metabolism and the dynamics of the extracellular matrix.

Addresses  
John Innes Centre, Department of Cell and Developmental Biology, Colney, Norwich NR4 7UH, UK  
e-mail: georg.seifert@bbsrc.ac.uk

Current Opinion in Plant Biology 2004, 7:277–284  

This review comes from a themed issue on Physiology and metabolism  
Edited by Christoph Benning and Mark Stitt  
1369-5266/$ – see front matter  
© 2004 Elsevier Ltd. All rights reserved.  
DOI 10.1016/j.pbi.2004.03.004

Abbreviations  
AUD membrane-anchored UDP-D-glucuronic acid decarboxylase gene  
AXS UDP-D-apiose/UDP-D-xylose synthase  
CCRC-M1 complex carbohydrate research center monoclonal antibody1  
GAE UDP-D-glucuronic acid 4-epimerase  
GDP guanosine 5′-diphosphate  
GER GDP-4-keto-6-deoxy-D-mannose-3,5-epimerase-4-reductase  
GMD GDP-D-mannose-4,6-dehydratase  
GME GDP-D-mannose 3,5-epimerase  
GUS β-glucuronidase gene  
MUM4 MUCILAGE-MODIFIED4  
MUR4 MURUS4  
RHD1 ROOT HAIR DEFECTIVE1  
RHM rhamnose synthase gene  
SUD soluble UDP-D-glucuronic acid decarboxylase gene  
TDP thymidin 5′-diphosphate  
UDP uridine 5′-diphosphate  

UGD UDP-D-glucose dehydrogenase  
UGE UDP-glucose 4-epimerase  
UXE UDP-D-xylose 4-epimerase  
UXS UDP-D-xylose synthase  

### Introduction

Most of the carbon fixed by plant photosynthesis is incorporated into cell wall carbohydrates; the remainder forms glycoproteins, glycolipids, storage polysaccharides and small molecules such as glycosides and oligosaccharides. The monosaccharide building blocks of plant carbohydrates are highly diverse, and carbohydrate biosynthesis requires subtle quantitative control throughout growth and development, putting formidable pressure on the evolution of versatile regulatory mechanisms. The glycosyltransferases involved in carbohydrate biosynthesis typically depend on nucleotide sugars as substrates.

### Nucleotide sugar interconverting enzymes and their gene families

Most monosaccharides are generated *in planta* by sequential conversions of UDP-D-glucose, whereas L-fucose and L-galactose are derived from GDP-D-mannose (Figure 1, Table 1; [1]). GDP-L-fucose formation requires the subsequent activities of GDP-D-mannose-4,6-dehydratase (GMD) [2] and a bifunctional GDP-4-keto-6-deoxy-D-mannose-3,5-epimerase-4-reductase (GER) [3]. GDP-L-galactose, on the other hand, is formed by GDP-D-mannose 3,5-epimerase (GME), which also generates GDP-L-gulose [4*,5].

UDP-D-glucose can be directly converted into UDP-D-galactose, UDP-D-glucuronic acid or UDP-L-rhamnose. In bacteria, TDP-L-rhamnose is formed by the modification of TDP-D-glucose by three enzymes that are encoded by three separate genes. Plants, on the other hand, make the UDP-sugar, and a single polypeptide encoded by a family of genes designated *RHM* probably catalyses all three reactions that lead to the conversion of UDP-D-glucose into UDP-L-rhamnose in plants [1]. Mutations in the *Arabidopsis thaliana RHM2* gene, which is allelic to the *MUCILAGE-MODIFIED4 (MUM4)* locus, cause a 50% reduction in the rhamnose and galacturonic acid content of seed coat mucilage. This means that *RHM2/MUM4* is involved in rhamnogalacturonan I biosynthesis, and indirectly implicates all three *RHM* genes in L-rhamnose biosynthesis [6**,7**]. UDP-D-glucuronic acid is irreversibly formed from UDP-D-glucose by UDP-D-glucose dehydrogenase (UGD). 52-kDa plant UGD proteins that are closely related to mammalian UGD have been purified [8–12]. The identification of a 40-kDa plant

Respiration
Photosynthesis

Glycolysis
Gluconeogenesis

D-Frc-6-P
PGI
PMI
D-Glc-6-P
D-Man-6-P
PGM
PMM
D-Glc-1-P
D-Man-1-P
UTP
GTP
Sucrose
UGP
PPi
NADPH
SUS
UDP
Fructose
UGE
UDP-D-Glc
NADP+
GDP-L-Fuc
2 NAD+
UGD
RHM
NADPH
NADP+
2 NADH
UDP-D-GlcA
PPi
UTP
UAP
UDP-L-Rhm
D-GlcA-1-P
GAK
ATP
ADP
O₂
myo-Inositol
AXS
CO₂
UDP-D-Xyl
UDP-D-Api
UXS
CO₂
UDP-D-Gal
GMD
GER
GMP
GME
GDP-L-Gul
GDP-L-Gal
Vitamin C
GDP-D-Man
UDP-D-GalA
GAE
UDP-D-GlcA
UXS
UDP-D-Xyl
UXE
UDP-L-Ara

Cytoplasm

Cell Wall
Endomembrane system

Current Opinion in Plant Biology
www.sciencedirect.com
UGD [13] triggered controversy because of its low affinity for UDP-D-glucose and its ethanol dehydrogenase activity. However, the cloning and expression of the bifunctional UGD/alcohol dehydrogenase (ADH) confirmed the existence of a generically distinct UGD in plants (P Bolwell, pers. comm.). The biosynthesis of UDP-D-glucuronic acid can occur via an alternative pathway that involves the inositol oxygenase enzyme, which has recently been cloned from mammals [14] and yeast [15].

The molecular basis of UDP-D-xylose biosynthesis has been elucidated in the pathogenic fungus *Cryptococcus neoformans* by a candidate gene approach using a putative bacterial UDP-D-glucuronic acid decarboxylase [16] and by peptide sequencing of purified pea UDP-D-xylose synthase (UXS) [17]. Both of these studies demonstrated enzyme activity in the recombinant protein. Discrepancies in molecular weight, pH optima and NAD⁺ requirement between the purified and the recombinant enzyme from pea may be due to posttranslational modifications *in planta* [17]. Alternatively, paralogous genes may encode two kinetically distinct UDP-D-xylose 4-epimerase (UXE) isoforms. Consistent with the notion of co-existing distinct UXS genes, the *A. thaliana* genome contains six putative UXS genes that are highly similar to both the pea and the fungal UXS genes. These putative UXS genes are termed AUD or SUD depending on the presence or absence, respectively, of an amino-terminal signal peptide and a hydrophobic membrane anchor [1]. Five of these genes were also termed AtUXS1–5, and recombinant AtUXS1/AUD3 and AtUXS3/SUD2 proteins generate UDP-D-xylose from UDP-D-glucuronic acid *in vitro* [18]. Given the high similarity among the UXS genes from plants, fungi and mammals [19], which encode 39- or 48-kDa polypeptides, the purification of an apparently unrelated 87-kDa UDP-D-glucuronic acid decarboxylase might indicate the existence of a plant-specific genetically different UXS ([20]; P Bolwell, pers. comm.). The cloning of the corresponding gene will be necessary to further address the nature of this enzyme. Two *A. thaliana* genes that are distantly related to the plant UXS gene family apparently encode UDP-D-apiose/UDP-D-xylose synthase (AXS). Purified recombinant AXS1 generates UDP-D-xylose and UDP-D-apiose using UDP-D-glucuronic acid as its exclusive substrate [21].

UDP-D-glucose, UDP-D-glucuronic acid and UDP-D-xylose are reversibly interconverted into UDP-D-galactose, UDP-D-galacturonic acid and UDP-L-arabinose, respectively, by UDP-sugar 4-epimerases [1,22]. Three distinct clades of predicted *A. thaliana* peptides have extensive sequence similarity to the UDP-glucose 4-epimerase (UGE) of various species [23]. Four *A. thaliana* peptides share a cluster with the previously cloned and thoroughly characterized *A. thaliana* UGE1 [24] and with UGE peptides from other organisms. All five *A. thaliana* UGE proteins interconvert UDP-D-glucose and UDP-D-galactose *in vitro* (C Barber, G Seifert, unpublished). UGE4 is identical to the *ROOT HAIR DEFECTIVE1 (RHD1)* locus and is essential for the biosynthesis of galactose-containing cell wall matrix polymers [23]. UDP-D-galacturonic acid is generated by UDP-D-glucuronic acid 4-epimerase (GAE) [22]. The molecular identity of plant GAE, which might be encoded by a family of six *A. thaliana* genes that are related to cap1J (a GAE from *Streptococcus pneumoniae* [25]), is still hypothetical. A third clade of four UGE-like *A. thaliana* genes contains UXE1/MURUS4 (*MUR4*). Mutants that are affected in UXE1/MUR4 contain 50% of wildtype levels of cell wall arabinose [26], and recombinant UXE1/MUR4 specifically interconverts UDP-D-xylose and UDP-L-arabinose [27].

After surveying the recent explosion in our knowledge of the molecular identity of nucleotide sugar interconverting enzymes, I now turn to consider the possible mechanisms for their regulation. I suggest that transcriptional and posttranslational controls of nucleotide sugar interconverting enzymes provide coarse and fine regulation of carbohydrate biosynthesis, respectively.

---

(Figure 1 Legend) Overview of metabolic sequences and compartmentalisation of nucleotide sugar interconverting enzymes that are involved in cell wall biosynthesis. Most nucleotide sugar interconverting enzymes are localised in the cytosol, where they interact metabolically with glycolysis and gluconeogenesis through the reversible actions of phosphomannose isomerase (PMI), phosphoglucose isomerase (PGI), phosphomannomutase (PMM), and phosphoglucomutase (PGM). Nucleotide sugars are generated *in vivo* by UDP-D-glucose pyrophosphorylase (UGP), GDP-D-mannose pyrophosphorylase (GMP), and UDP-D-glucuronic acid pyrophosphorylase (UAP). Analogous pyrophosphorylases act on galactose-1-P and arabinose-1-P in salvage pathways (not shown). UDP-D-glucose is also generated by sucrose synthase (SUS). GDP-D-mannose is converted either into GDP-L-fucose by the sequential action of the directly interacting GMD and GER or into GDP-L-galactose and GDP-L-gulose by GME. UDP-D-glucose is converted into UDP-D-galactose by UGE, into UDP-D-glucuronic acid by UGD or into UDP-L-rhamnose by RHM, which hypothetically consists of sequentially acting UDP-D-glucose 4,6-dehydratase and UDP-4-keto-6-deoxy-D-glucose 3,5-epimerase 4-reductase. The sequential action of inositol oxygenase (INO), D-glucuronokinase (GAK) and UAP represents an alternative pathway of UDP-D-glucuronic acid biosynthesis. Two different cytosolic UDP-D-glucuronic acid decarboxylases (UXS and AXS) give rise to UDP-D-xylose or to a mixture of UDP-D-xylose and UDP-D-apiose, respectively. UDP-D-glucose, UDP-D-galactose, UDP-D-glucuronic acid, UDP-L-rhamnose, UDP-D-apiose, GDP-D-mannose, GDP-L-fucose and GDP-L-galactose are transported into the endomembrane system, where specific glycosyltransferases are localised. UDP-D-glucose is also channelled to cellulose synthase (CeS) and callose synthase (CaS), which are localised at the plasma membrane. In the lumen of the endomembrane system, UDP-D-glucuronic acid is either converted into UDP-D-xylose by membrane-bound UXS or into UDP-D-galacturonic acid by membrane-bound GAE. UDP-D-xylose is converted into UDP-L-arabinose by membrane-bound UXE. Api, apiose; Ara, arabinose; Frc, fructose; Gal, galactose; GalA, galacturonic acid; Glc, glucose; Gul, gulose; GlcA, glucuronic acid; Man, mannose; PPI, inorganic pyrophosphate; UTP, uridine triphosphate; Xyl, xylose.
Table 1

A. thaliana nucleotide sugar biosynthetic enzymes of the small dehydrogenase/reductase superfamily.

| Enzyme family | Isoform       | Locus         | Number of amino acids | MPSS data at http://mpss.udel.edu/at/java.html? | ESTs$^a$ | Reference(s)$^b$ |
|---------------|---------------|---------------|-----------------------|---------------------------------------------|----------|-------------------|
|               |               |               |                       | Callus                                      |          |                   |
|               |               |               |                       | Inflorescence                               |          |                   |
|               |               |               |                       | Leaf                                        |          |                   |
|               |               |               |                       | Root                                        |          |                   |
|               |               |               |                       | Silique                                     |          |                   |
| UGE           | UGE1          | At1g12780     | 351                   | 175                                         | 44       | 90                | 46 | 203 | 17 | [24]            |
|               | UGE2          | At4g23920     | 350                   | 76                                          | 75       | 15                | 29 | 2   | 4  |                 |
|               | UGE3          | At1g63180     | 351                   | 36                                          | 0        | 33                | 19 | 22  | 4  |                 |
|               | UGE4 RHD1     | At1g64440     | 348                   | 103                                         | 102      | 29                | 173 | 80  | 7  | [23**,48*,49**] |
|               | UGE5          | At4g10960     | 351                   | 0                                           | 0        | 1                 | 0  | 0   | 1  |                 |
| UXE           | UXE1 MUR4     | At1g30620     | 419                   | 93                                          | 40       | 9                 | 12 | 17  | 3  | [26,27**]       |
|               | UXE2          | At4g20460     | 379                   | 0                                           | 0        | 2                 | 0  | 0   | Nf |                 |
|               | UXE3          | At2g34850     | 385                   | 1                                           | 9        | 0                 | 8  | 8   | 1  |                 |
|               | UXE4          | At5g44480     | 436                   | 21                                          | 0        | 2                 | 147 | 0   | Nf |                 |
| GAE           | GAE1          | At4g30440     | 429                   | 325                                         | 86       | 148               | 57 | 72  | 29 |                 |
|               | GAE2          | At1g02000     | 434                   | 12                                          | 19       | 24                | 21 | 25  | 5  |                 |
|               | GAE3          | At4g00110     | 430                   | 2                                           | 27       | 21                | 17 | 2   | 6  |                 |
|               | GAE4          | At2g45310     | 437                   | 8                                           | 34       | 32                | 27 | 10  | 5  |                 |
|               | GAE5          | At4g12250     | 436                   | 17                                          | 0        | 17                | 0  | 20  | 6  |                 |
|               | GAE6          | At3g23820     | 460                   | 43                                          | 194      | 221               | 249 | 169 | 51 |                 |
| UGD           | UGD1          | At5g39320     | 480                   | 0                                           | 0        | 0                 | 70 | 0   | 23 | [9]             |
|               | UGD2          | At3g29360     | 480                   | 0                                           | 115      | 0                 | 5  | 0   | 24 |                 |
|               | UGD3          | At5g15490     | 480                   | 72                                          | 334      | 137               | 331 | 361 | 16 |                 |
|               | UGD4          | At1g26570     | 481                   | 29                                          | 82       | 30                | 49 | 66  | 3  |                 |
| UXS           | UXS1 AUD3     | At3g53520     | 433                   | 64                                          | 100      | 77                | 67 | 179 | 10 | [18*]           |
|               | UXS2 AUD1     | At3g62830     | 445                   | 116                                         | 80       | 34                | 103 | 92  | 18 | [18*]           |
|               | UXS3 SUD2     | At5g59290     | 342                   | 81                                          | 97       | 63                | 69 | 170 | 9  | [18*]           |
|               | UXS4 AUD2     | At2g47650     | 443                   | 52                                          | 59       | 41                | 58 | 34  | 17 |                 |
|               | UXS5 SUD1     | At3g46440     | 341                   | 49                                          | 103      | 10                | 36 | 13  | 5  |                 |
|               | SUD3          | At2g28760     | 343                   | 174                                         | 40       | 21                | 78 | 12  | 2  |                 |
| RHM           | RHM1          | At1g78570     | 669                   | 206                                         | 1024     | 147               | 191 | 225 | 30 |                 |
|               | RHM2 MUM4     | At1g53500     | 667                   | 110                                         | 197      | 84                | 173 | 139 | 16 | [6**,7**]       |
|               | RHM3          | At3g14790     | 664                   | 20                                          | 28       | 20                | 51 | 43  | 5  |                 |
|               | UER1          | At1g63000     | 301                   | 140                                         | 311      | 124               | 501 | 123 | 26 |                 |
| AXS           | AXS1          | At2g27860     | 389                   | 0                                           | 0        | 0                 | 0  | 0   | 19 | [21*]           |
|               | AXS2          | At1g08200     | 389                   | 157                                         | 294      | 177               | 98 | 143 | 20 |                 |
| GMD           | GMD1          | At5g66280     | 361                   | 0                                           | 0        | 0                 | 0  | 0   | 14 | [28,29]         |
|               | GMD2 MUR1     | At3g51160     | 373                   | 0                                           | 0        | 0                 | 0  | 40  | 11 | [28,29,41,51,52*] |
| GER           | GER1          | At1g73250     | 312                   | 2                                           | 11       | 4                 | 6  | 8   | 4  | [3,41]          |
|               | GER2          | At1g17890     | 328                   | 108                                         | 82       | 57                | 228 | 43  | 3  |                 |
| GME           | GME1          | At5g28840     | 377                   | 98                                          | 302      | 537               | 60 | 182 | 13 | [4*,5]          |

EST, expressed sequence tag; Nf, not found; UER1, UDP-4-keto-6-deoxy-D-glucose 3,5-epimerase-4-reductase.

$^a$ As found in the Munich Information Center for Protein Sequences (MIPS) database on 1st October, 2003.

$^b$ Original research papers cited in this article.

are complemented by a time-resolved survey of gene expression during the differentiation of tracheary elements in *Zinnia elegans*. This survey revealed that the *UXS* and *UGD* sequences are upregulated at the onset of secondary cell wall synthesis ([33]; D Milioni, pers. comm.). At the organ level, some genes that encode nucleotide sugar interconverting enzymes (i.e., *GMD1* [28], *UGD1* [9], *UGE4*, *UXE4* and *GER1* [Table 1]) are preferentially expressed in roots. These expression patterns might be related to adaptations of the kinetic properties of the different isoforms encoded by these genes to metabolic conditions that are typically encountered in the rhizosphere, such as hypoxia.

**Differences in the activities of isoenzymes**

Two mechanisms that might regulate nucleotide sugar flux are feedback inhibition and redox sensing. *UXS* and both types of *UGD* [10,11,13] are feedback inhibited by UDP-D-xylose. However, the UDP-D-glucuronic acid decarboxylase purified from wheat germ is inhibited by UDP-D-xylose to a much lesser degree than is *AtUXS3* [18]. In fact, the two wheat germ *UXS* isoforms dramatically differ in their degree of inhibition by UDP-D-xylose [34]. Different preparations of 52-kDa UGD are inhibited by UDP-D-xylose with an apparent Kᵢ of 10–17 μM [10,11,13], whereas the 40-kDa UGD is still active at half-maximal rate in the presence of 6 mM UDP-D-xylose [13]. Feedback inhibition itself needs to be controlled if it is to modulate nucleotide sugar flux. This could occur by the differential expression of genes that encode kinetically distinct enzymes.

Sensitivity to NAD⁺/NADH ratio could be a mechanism that links glycolytic flux with carbohydrate flux into cell wall polysaccharides [35]. In this respect, it is important to note that most nucleotide sugar interconverting enzymes belong to the superfamily of small dehydrogenases/reductases and require bound NAD⁺ for the reduction of a hydroxyl group to a ketone. UGD reduces two mol of NAD⁺ per mol UDP-D-glucose and is inhibited by NADH [10,11]. *AXS1* activity is stimulated by NAD⁺ and inhibited by NADH [21]. The biosynthesis of UDP-L-rhamnose may be NAD⁺/NADH-sensitive because bound NAD⁺ is required for the initial C-4-oxidation

---

Figure 2

![Diagram](attachment:diagram.png)

Different hypothetical types of regulation of arbitrarily named *UGE* isoforms. Co-expressed *UGE* isoforms might differ in their sub-cellular localisation, dependence on NAD⁺/NADH co-factor/inhibitor, and posttranslational modification. In this model, *UGE1* represents an isoform that acts mainly in the cytoplasm in the salvage of free galactose that is released from carbohydrate breakdown, and in the generation of UDP-galactose (UDP-Gal) for cytosolic galactosyltransferases such as galactinol synthase. *UGE2* and *UGE3*, on the other hand, represent *UGE* isoforms that associate with the outer face of the endomembrane system, where they interact with galactosyltransferases that are involved in cell wall biosynthesis (*galT1* and *galT2*). Whereas NAD⁺ remains tightly bound to *UGE1* and *UGE2*, *UGE3* has low affinity for NAD⁺ and undergoes redox control. The association of *UGE2* with the endomembrane system, on the other hand, is regulated by posttranslational modifications that are triggered by growth regulators such as ethylene and auxin. In this way, the flux of UDP-D-galactose (UDP-D-Gal) is coupled to the relative spatial abundance of each of the three isoforms, the cytosolic redox potential and the influence of growth regulators. UDP-D-Glc, UDP-D-glucose.

reaction, and because the reducing equivalents for the generation of the deoxy sugars are derived from NADPH [22]. NAD⁺ stimulates and NADH inhibits the UGE enzyme purified from wheat germ [35]. Other UGE preparations, however, are insensitive to co-factor addition, indicating that NAD⁺ is tightly bound [22]. These different co-factor affinities might be inherent properties of differentially encoded isoenzymes. Purified recombinant *A. thaliana* UGE4 is stimulated by NAD⁺ (C Barber, G Seifert, unpublished), contrasting with UGE1, which is insensitive to the addition of NAD⁺ [36]. Some isoforms might depend on a relatively high NAD⁺/NADH ratio for activity, thereby co-ordinating carbon flux between glycolysis and cell wall formation ('UGE3' in Figure 2). High glycolytic flux, for example during hypoxia [37], decreases the cytosolic NAD⁺/NADH ratio. This might rapidly shut down cell wall matrix biosynthesis. UGE isoforms that are insensitive to cytosolic NAD⁺/NADH ratio because they bind NAD⁺ tightly might act in the biosynthesis of stress-related carbohydrates such as galactinol [38] or in the reverse reaction during catabolic cell wall turnover ('UGE1' in Figure 2). The increased levels of UDP-D-glucose could be either utilised for glycolytic energy production or channelled into cellulose biosynthesis. The occurrence of the latter process is indicated by increased cellulose in hypoxic tissue [39].

### Formation of complexes involving sequential enzymes

Co-localisation of sequentially acting enzymes has long been acknowledged as an important aspect of cellular biochemistry [40]. The observation that GMD2/MUR1 and GER1 tightly associate to form a functional complex [41], and the presence of a single plant polypeptide containing all three enzymes required for UDP-L-rhamnose biosynthesis, highlights the evolutionary trend towards the formation of complexes between sequentially acting nucleotide sugar interconverting enzymes. I envisage the existence of complexes between cytosolic nucleotide sugar interconverting enzymes, nucleotide sugar transporters [42,43] and glycosyltransferases ('UGE2' and 'UGE3' in Figure 2) that are able to channel nucleotide sugars from the cytosol to glycosyltransferases acting in the lumen of the endomembrane system efficiently, and that provide a mechanism of rapid fine control of sugar flux. There is a paradigm for interaction between a glycosyltransferase and a soluble cytosolic nucleotide sugar-generating enzyme. Sucrose synthase co-localises with cellulose synthase at the plasma membrane [44,45] and is required for efficient cellulose synthesis [46,47]. The cell-type- and carbohydrate-type-restricted requirement of UGE4 for the biosynthesis of galactosylated cell wall polymers provides genetic evidence of substrate channelling between cytosolic UGE and Golgi-localised galactosyltransferases [23**,48*]. Moreover, ethylene post-transcriptionally suppresses cell wall defects in *uge4* mutants [49**], suggesting that the association between

different isoforms of nucleotide sugar interconverting enzymes and specific glycosyltransferases might fine-tune polysaccharide biosynthesis during differentiation and stress responses (Figure 2). Consistent with a tight physical association between nucleotide sugar delivery and nucleotide sugar consumption, the mammalian UDP-D-galactose transporter interacts with ceramide galactosyltransferase [50].

#### Conclusions: do nucleotide sugar interconverting enzymes regulate carbon flux into the cell wall?

Work on mutants that are defective in nucleotide sugar interconverting enzymes amply demonstrates essential roles for individual gene family members in cell wall biosynthesis [1,6**,7**,23**,51,52*]. Complementation of *mur4* mutants by transformation with the wildtype gene increases the levels of cell wall arabinose in transgenic lines that possess multiple copies of *UXE1/MUR4*. This indicates a rate-limiting role for *UXE1/MUR4* in the control of cell wall biosynthesis [27**]. In transgenic plants that over- and under-express *UGE1*, UGE activity is modulated between 10% and 300% of UGE activity in the wildtype without noticeable effects on cell wall and lipid composition. However, there is a correlation between *UGE1* expression and resistance to toxic levels of galactose [24]. Mutations in the *UGE4* gene, on the other hand, have no effect on galactose toxicity but cause a 25% reduction in cell-wall-bound galactose [23**]. This indicates that despite their identical biochemical functions *in vitro*, UGE1 and UGE4 have different roles *in vivo*. To elucidate the role of nucleotide sugar interconverting enzymes for the regulation of cell wall biosynthesis, experiments are required in which their expression, kinetic properties and intracellular localisation are deliberately modulated.

#### Acknowledgements

I gratefully acknowledge P Bolwell, R Tenhaken, Y Jigami, D Milioni and C Barber for communicating unpublished material. I thank K Roberts for numerous discussions and critical reading of this manuscript. This work was supported by the Biotechnology and Biological Sciences Research Council (BBSRC; grant 208/D10332) and by the EU grant (QLK5-CT-2001-00443 [EDEN]).

#### References and recommended reading

Papers of particular interest, published within the annual period of review, have been highlighted as:

- **•** of special interest
- **••** of outstanding interest

1. Reiter WD, Vanzin GF: Molecular genetics of nucleotide sugar interconversion pathways in plants. *Plant Mol Biol* 2001, 47:95-113.
2. Bonin CP, Potter I, Vanzin GF, Reiter WD: The *MUR1* gene of *Arabidopsis thaliana* encodes an isoform of GDP-D-mannose-4,6-dehydratase, catalyzing the first step in the de novo synthesis of GDP-L-fucose. *Proc Natl Acad Sci USA* 1997, 94:2085-2090.
3. Bonin CP, Reiter WD: A bifunctional epimerase-reductase acts downstream of the *MUR1* gene product and completes the de

novosynthesis of GDP-L-fucose in Arabidopsis. Plant J 2000, 21:445-454.

4. Wolucka BA, Van Montagu M: GDP-mannose 3′,5′-epimerase forms GDP-L-gulose, a putative intermediate for the de novo biosynthesis of vitamin C in plants. J Biol Chem 2003, 278:47483-47490.
Purified native and recombinant enzyme preparations reveal intriguing properties. Several properties of GME1, including its low catalytic rate, stimulation/inhibition by NAD(P)/NAD(P)H, inhibition by GDP-L-fucose, feedback inhibition by ascorbic acid and L-galactono-1,4-lactone, and heat shock protein 70 (hsp70) binding, argue for a regulatory role for this protein. The interaction between GME1 and other proteins potentially affects the relative yield of its two enzyme products, UDP-L-galactose and UDP-L-gulose. These properties could be instrumental in the regulation of substrate flux into cell wall and vitamin C biosynthesis.

5. Wolucka BA, Persiau G, Van Doorsselaere J, Davey MW, Demol H, Vandekerckhove J, Van Montagu M, Zabeau M, Boerjan W: Partial purification and identification of GDP-mannose 3′,5′-epimerase of Arabidopsis thaliana, a key enzyme of the plant vitamin C pathway. Proc Natl Acad Sci USA 2001, 98:14843-14848.

6. Usadel B, Kuschinsky AM, Rosso MG, Eckermann N, Pauly M: RHM2 is involved in mucilage pectin synthesis and is required for the development of the seed coat in Arabidopsis thaliana. Plant Physiol 2004, 134:286-295.
RHM2 was targeted in a reverse genetics approach. Two *rhm2* mutant alleles display aberrant seed coat morphology and decreased production of seed coat mucilage. The rhamnogalacturonan I deficiency in the *rhm2-1* seed coat mucilage is consistent with the previously predicted role of RHM2 in L-rhamnose biosynthesis.

7. Western TL, Young DS, Dean GH, Tan WL, Samuels AL, Haughn GW: *MUCILAGE-MODIFIED 4* encodes a putative pectin biosynthetic enzyme developmentally regulated by APETALA2, TRANSPARENT TESTA GLABRA1 and GLABRA2 in the Arabidopsis seed coat. Plant Physiol 2004, 134:296-306.
The *mucilage modified4* mutants were positionally identified as *RHM2* alleles. Transcriptional regulators that are known to be involved in epidermal differentiation (i.e., APETALA2 [AP2], TRANSPARENT TESTA GLABRA1 [TTG1], GLABRA2 [GL2] and TTG2) were found to modulate *RHM2/MUM4* expression. The similar cellular phenotypes of *rhm2/mum4* and *ap2*, *ttg1*, *ttg2*, and *gl2* mutants demonstrate the importance of nucleotide sugar biosynthesis for cellular morphogenesis.

8. Tenhaken R, Thulke O: Cloning of an enzyme that synthesizes a key nucleotide-sugar precursor of hemicellulose biosynthesis from soybean: UDP-glucose dehydrogenase. Plant Physiol 1996, 112:1127-1134.

9. Seitz B, Klos C, Wurm M, Tenhaken R: Matrix polysaccharide precursors in Arabidopsis cell walls are synthesized by alternate pathways with organ-specific expression patterns. Plant J 2000, 21:537-546.

10. Hinterberg B, Klos C, Tenhaken R: Recombinant UDP-glucose dehydrogenase from soybean. Plant Physiol Biochem 2002, 40:1011-1017.

11. Turner W, Botha FC: Purification and kinetic properties of UDP-glucose dehydrogenase from sugarcane. Arch Biochem Biophys 2002, 407:209-216.

12. Johansson H, Sterky F, Amini B, Lundeberg J, Kleczkowski LA: Molecular cloning and characterization of a cDNA encoding poplar UDP-glucose dehydrogenase, a key gene of hemicellulose/pectin formation. Biochim Biophys Acta 2002, 1576:53-58.

13. Robertson D, Smith C, Bolwell GP: Inducible UDP-glucose dehydrogenase from French bean (*Phaseolus vulgaris* L.) locates to vascular tissue and has alcohol dehydrogenase activity. Biochem J 1996, 313:311-317.

14. Arner RJ, Prabhu KS, Thompson JT, Hildenbrandt GR, Liken AD, Reddy CC: Myo-inositol oxygenase: molecular cloning and expression of a unique enzyme that oxidizes myo-inositol and d-chiro-inositol. Biochem J 2001, 360:313-320.

15. Kanter U, Becker M, Friauf E, Tenhaken R: Purification, characterization and functional cloning of inositol oxidase from Cryptococcus. Yeast 2003, 20:1317-1329.


16. Bar-Peled M, Griffith CL, Doering TL: Functional cloning and characterization of a UDP-glucuronic acid decarboxylase: the pathogenic fungus *Cryptococcus neoformans* elucidates UDP-xylose synthesis. Proc Natl Acad Sci USA 2001, 98:12003-12008.

17. Kobayashi M, Nakagawa H, Suda I, Miyagawa I, Matoh T: Purification and cDNA cloning of UDP-D-glucuronate carboxy-lyase (UDP-D-xylose synthase) from pea seedlings. Plant Cell Physiol 2002, 43:1259-1265.
UXS from pea was purified and cloned using peptide sequencing and PCR with degenerate primers. A single clone was isolated from a cDNA library. Differences in molecular weight and enzymatic properties between the UXS encoded by the cloned gene and the purified UXS indicate posttranslational modification or the cloning of a paralogous UXS gene. The stimulation by NAD⁺ of the cloned but not the purified enzyme could indicate isoform-specific roles in redox regulation.

18. Harper AD, Bar-Peled M: Biosynthesis of UDP-xylose. Cloning and characterization of a novel *Arabidopsis* gene family, *UXS*, encoding soluble and putative membrane-bound UDP-glucuronic acid decarboxylase isoforms. Plant Physiol 2002, 130:2188-2198.
Recombinant *AtUXS1/AUD3* and *AtUXS3/SUD2* act as UXS in vitro, experimentally confirming the existence of multiple UXS genes in plants. The low activity of *AtUXS2/AUD1* could be a bacterial expression artefact or indicate isoenzyme-specific properties.

19. Moriarity JL, Hurt KJ, Resnick AC, Storm PB, Laroy W, Schnaar RL, Snyder SH: UDP-glucuronate decarboxylase, a key enzyme in proteoglycan synthesis: cloning, characterization, and localization. J Biol Chem 2002, 277:16968-16975.

20. Wheatley ER, Davies DR, Bolwell GP: Characterisation and immunolocation of an 87 kDa polypeptide associated with UDP-glucuronic acid decarboxylase activity from differentiating tobacco cells (*Nicotiana tabacum* L.). Phytochemistry 2002, 61:771-780.

21. Molhoj M, Verma R, Reiter WD: The biosynthesis of the branched-chain sugar D-apiose in plants: functional cloning and characterization of a UDP-D-apiose/UDP-D-xylose synthase from *Arabidopsis*. Plant J 2003, 35:693-703.
The functional cloning of the gene encoding a UDP-D-apiose biosynthetic enzyme opens the field for the reverse genetic investigation of rhamnogalacturonan II function. AXS1 activity is stimulated by NAD⁺, potentially making it a redox-regulated nucleotide sugar interconversion enzyme.

22. Feingold DS: Aldo (and keto) hexoses and uronic acids. In Encyclopaedia of Plant Physiology, vol 13a. Edited by Loewus FA, Tanner W. Berlin: Springer Verlag; 1982:3-76.

23. Seifert GJ, Barber C, Wells B, Dolan L, Roberts K: Galactose biosynthesis in *Arabidopsis*: genetic evidence for substrate channeling from UDP-D-galactose into cell wall polymers. Curr Biol 2002, 12:1840-1845.
The *RHD1/ROOT EPIDERMAL BULGER1* locus was identified as the *UGE4* gene. The dramatic *rhd1* root phenotype is associated with normal composition of galactolipids but reductions in arabinogalactan proteins and fucogalactosylated xyloglucan. Immunohistochemistry demonstrates that the defect in *UGE4* affects different cell wall polymers in a cell-type- and cell-wall-domain-specific manner, prompting the hypothesis that co-expressed UGE isoforms form substrate channels with polymer-specific galactosyltransferases.

24. Dörmann P, Benning C: The role of UDP-glucose epimerase in carbohydrate metabolism of *Arabidopsis*. Plant J 1998, 13:641-652.

25. Muñoz R, López R, de Frutos M, García E: First molecular characterization of a uridine diphosphate galacturonate 4-epimerase: an enzyme required for capsular biosynthesis in *Streptococcus pneumoniae* type 1. Mol Microbiol 1999, 31:703-713.

26. Burget EG, Reiter WD: The *mur4* mutant of *Arabidopsis* is partially defective in the de novo synthesis of uridine diphospho-L-arabinose. Plant Physiol 1999, 121:383-389.

27. Burget EG, Verma R, Molhoj M, Reiter WD: The biosynthesis of L-arabinose in plants: molecular cloning and characterization of a Golgi-localized UDP-D-xylose 4-epimerase encoded by the *MUR4* gene of *Arabidopsis*. Plant Cell 2003, 15:523-531.
The map-based cloning of the arabinose-deficient *mur4* mutant locus identifies the first UDP-D-xylose 4-epimerase. *UXE1/MUR1* acts in a nucleotide-sugar-specific manner and is presumably targeted to the

www.sciencedirect.com Current Opinion in Plant Biology 2004, 7:277–284

secretory apparatus stem, as suggested by brefeldin-A-sensitive vesicular green fluorescent protein (GFP) ::UXE1 localisation. Spurious over-expression of MUR4 increases the cell wall arabinose content, indicating that the rate of cell wall arabinosylation is limited by this enzyme.

28. Bonin CP, Freshour G, Hahn MG, Vanzin GF, Reiter WD: The GMD1 and GMD2 genes of *Arabidopsis* encode isoforms of GDP-β-mannose 4,6-dehydratase with cell type-specific expression patterns. *Plant Physiol* 2003, **132**:883-892.

29. Freshour G, Bonin CP, Reiter WD, Albersheim P, Darvill AG, Hahn MG: Distribution of fucose-containing xyloglucans in cell walls of the *mur1* mutant of *Arabidopsis*. *Plant Physiol* 2003, **131**:1602-1612.

30. Dalessandro G, Northcote DH: Changes in enzymic activities of nucleoside diphosphate sugar interconversions during differentiation of cambium to xylem in pine and fir. *Biochem J* 1977, **162**:281-288.

31. Dalessandro G, Northcote DH: Changes in enzymic activities of nucleoside diphosphate sugar interconversions during differentiation of cambium to xylem in sycamore and poplar. *Biochem J* 1977, **162**:267-279.

32. Hertzberg M, Aspeborg H, Schrader J, Andersson A, Erlandsson R, Blomqvist K, Bhalerao R, Uhlen M, Teeri TT, Lundeberg J et al.: A transcriptional roadmap to wood formation. *Proc Natl Acad Sci USA* 2001, **98**:14732-14737.

33. Milioni D, Sado PE, Stacey NJ, Roberts K, McCann MC: Early gene expression associated with the commitment and differentiation of a plant tracheary element is revealed by cDNA-amplified fragment length polymorphism analysis. *Plant Cell* 2002, **14**:2813-2824.

34. John KV, Schutzbach JS, Ankel H: Separation and allosteric properties of two forms of UDP-glucuronate carboxy-lyase. *J Biol Chem* 1977, **252**:8013-8017.

35. Fan DF, Feingold DS: Nucleoside diphosphate-sugar 4-epimerases. II. Uridine diphosphate arabinose 4-epimerase of wheat germ. *Plant Physiol* 1970, **46**:592-595.

36. Dormann P, Benning C: Functional expression of uridine 5′-diphospho-glucose 4-epimerase (EC 5.1.3.2) from *Arabidopsis thaliana* in *Saccharomyces cerevisiae* and *Escherichia coli*. *Arch Biochem Biophys* 1996, **327**:27-34.

37. van Dongen JT, Schurr U, Pfister M, Geigenberger P: Phloem metabolism and function have to cope with low internal oxygen. *Plant Physiol* 2003, **131**:1529-1543.

38. Taji T, Ohsumi C, Iuchi S, Seki M, Kasuga M, Kobayashi M, Yamaguchi-Shinozaki K, Shinozaki K: Important roles of drought-and cold-inducible genes for galactinol synthase in stress tolerance in *Arabidopsis thaliana*. *Plant J* 2002, **29**:417-426.

39. Albrecht G, Mustroph A: Localization of sucrose synthase in wheat roots: increased *in situ* activity of sucrose synthase correlates with cell wall thickening by cellulose deposition under hypoxia. *Planta* 2003, **217**:252-260.

40. Srere PA: Complexes of sequential metabolic enzymes. *Annu Rev Biochem* 1987, **56**:89-124.

41. Nakayama KI, Maeda Y, Jigami Y: Interaction of GDP-4-keto-6-deoxymannose-3,5-epimerase-4-reductase with GDP-mannose-4,6-dehydratase stabilizes the enzyme activity for formation of GDP-fucose from GDP-mannose. *Glycobiology* 2003, **13**:673R-680R.

42. Baldwin TC, Handford MG, Yuseff MI, Orellana A, Dupree P: Identification and characterization of GONST1, a Golgi-localized GDP-mannose transporter in *Arabidopsis*. *Plant Cell* 2001, **13**:2283-2295.

43. Norambuena L, Marchant L, Berninsone P, Hirschberg CB, Silva H, Orellana A: Transport of UDP-galactose in plants. Identification and functional characterization of AtUTr1, an *Arabidopsis thaliana* UDP-galactose/UDP-glucose transporter. *J Biol Chem* 2002, **277**:32923-32929.

44. Salnikov VV, Grimson MJ, Delmer DP, Haigler CH: Sucrose synthase localizes to cellulose synthesis sites in tracheary elements. *Phytochemistry* 2001, **57**:823-833.

45. Salnikov VV, Grimson MJ, Seagull RW, Haigler CH: Localization of sucrose synthase and callose in freeze-substituted secondary-wall-stage cotton fibers. *Protoplasma* 2003, **221**:175-184.

46. Haigler CH, Ivanova-Datcheva M, Hogan PS, Salnikov VV, Hwang S, Martin K, Delmer DP: Carbon partitioning to cellulose synthesis. *Plant Mol Biol* 2001, **47**:29-51.

47. Doblin MS, Kurek I, Jacob-Wilk D, Delmer DP: Cellulose biosynthesis in plants: from genes to rosettes. *Plant Cell Physiol* 2002, **43**:1407-1420.

48. Andeme-Onzighi C, Sivaguru M, Judy-March J, Baskin TI, Driouich A: The *reb1-1* mutation of *Arabidopsis* alters the morphology of trichoblasts, the expression of arabinogalactan-proteins and the organization of cortical microtubules. *Planta* 2002, **215**:949-958. The immunolocalisation of epitopes related to arabinogalactan proteins in the *UGE4* mutant root epidermal bulger1-1 indicates the essentiality of *UGE4* for the formation of the LM2 epitope in trichoblasts. The formation of the JIM14 epitope, on the other hand, depends on *UGE4* function in trichoblasts and atrichoblasts. These findings support the hypothesis that *UGE* isoforms might form substrate channels with polymer-specific galactosyltransferases in a cell-type-specific manner.

49. Seifert GJ, Barber C, Wells B, Roberts K: Growth regulators and the control of nucleotide sugar flux. *Plant Cell* 2004, **16**:723-730. Ethylene specifically suppresses the morphological defects caused by disruption of *UGE4*. This novel response acts posttranscriptionally and apparently involves the re-direction of *UGE4*-generated UDP-β-Gal into different cell wall biopolymers. This indirectly supports the notion that nucleotide sugar flux can be controlled by the dynamic formation of enzyme complexes between *UGE* isoforms and galactosyltransferases.

50. Sprong H, Degroote S, Nilsson T, Kawakita M, Ishida N, Van Der Sluijs P, Van Meer G: Association of the Golgi UDP-galactose transporter with UDP-galactose:ceramide galactosyltransferase allows UDP-galactose import in the endoplasmic reticulum. *Mol Biol Cell* 2003, **14**:3482-3493.

51. O'Neill MA, Eberhard S, Albersheim P, Darvill AG: Requirement of borate cross-linking of cell wall rhamnogalacturonan II for *Arabidopsis* growth. *Science* 2001, **294**:846-849.

52. van Hengel AJ, Roberts K: Fucosylated arabinogalactan-proteins are required for full root cell elongation in *Arabidopsis*. *Plant J* 2002, **32**:105-113. A study that highlights the usefulness of mutants that are defective in a nucleotide sugar interconversion enzyme for the functional characterization of cell wall carbohydrates. Together with the existence of a lectin that binds to α-L-Fuc-(1→2)-(β-L-Ara-(1→)), and that only is present on arabinogalactan proteins and inhibits root growth, the short root phenotype of the *mur1* mutant indicates a role for fucosylated arabinogalactan proteins in root elongation.
