
Phototactic motility in the unicellular cyanobacterium  
Synecochystis sp. PCC 6803  

Shizue Yoshihara* and Masahiko Ikeuchi  
Department of Life Sciences (Biology), The University of Tokyo, 3-8-1, Komaba, Meguro,  
Tokyo, 153-8902, Japan.  
E-mail: yoshihara@riast.osakafu-u.ac.jp; mikeuchi@bio.c.u-tokyo.ac.jp;  
Fax: 81-3-5454-4337; Tel: 81-3-5454-6641  

Received 16th February 2004, Accepted 19th April 2004  
First published as an Advance Article on the web 11th May 2004  

Synecochystis sp. PCC 6803 is a unicellular motile cyanobacterium that shows positive and negative phototaxis on agar plates under lateral illumination. Recent studies on the molecular mechanisms of the phototactic motility of Synecochystis have revealed that a number of genes are responsible for its pilus-dependent motility and phototaxis. Here we describe what is known about these genes. We also discuss the novel spectral properties of the phytochrome-like photoreceptor PixJ1 in Synecochystis, that is essential for positive phototaxis and which has revealed the existence of a new group of chromophore-binding proteins in cyanobacteria.

Introduction

Cyanobacteria are the most ancient organisms that perform oxygenic photosynthesis. According to the endosymbiosis theory, a cyanobacterium-like organism was the ancestor of the chloroplasts in plants. Many cyanobacteria can move by gliding, twitching or swimming, although none employ flagella, unlike many other bacteria. Filamentous cyanobacteria such as *Phormidium uncinatum* and *Anabaena variabilis* show gliding motility, which is a slow uniform forward motion on a solid surface in a direction parallel to the cell's long axis and occasionally interrupted by reversals. The twitching motility involves a small and intermittent translocation on a solid surface, in which the direction of movement often changes. The unicellular cyanobacterium Synecochystis sp. PCC 6803 shows twitching motility on an agar plate or a slide glass (unpublished observation). The swimming motility in liquid has been observed in several marine species of unicellular *Synechococcus*. These organisms do not possess flagella and it is still unclear how they swim.

The molecular mechanisms behind such movements are still poorly understood, even though the physiology of this motility has been extensively studied for many decades, mainly by using filamentous cyanobacteria. Some gliding filamentous cyanobacteria have an extracellular cell wall layer, the S-layer, attached to the outer membrane, and an array of helically arranged surface fibrils on the top of the S-layer. In *P. uncinatum*, the surface fibril consists of oscillin, a major Ca²⁺-binding protein suggested to be required for the gliding motility of the filament. It has also been suggested that the pore complex, which spans the cell wall, is involved in the rapid secretion of extracellular carbohydrates, serving as slime. Thus, the gliding movements may be driven by the secretion of slime and may be guided by the helically arranged surface fibrils. However, the genes involved in the gliding motility of filamentous cyanobacteria have not been clearly identified to date. In contrast, in the marine *Synechococcus* sp. WH 8102, which shows swimming motility, the *swmA* gene encoding a cell-surface-associated protein of 130 kDa, is essential for the translocation of the cells in liquid. Moreover, the entire genome sequence of the unicellular transformable cyanobacterium Synecochystis sp. PCC 6803, showing twitching motility, was determined in 1996. The first motility-related gene (*slr2031*) in this organism was then identified in 1998. Since then, mutational analyses have revealed that a significant number of additional genes are involved in the movement of Synecochystis. Furthermore, genes that play essential roles in

Shizue Yoshihara is a research assistant at the Department of Applied Bioscience (Bioelectronics), Research Institute for Advanced Science and Technology, Osaka Prefecture University. She received her PhD in Plant Biology from the University of Tokyo in 2003. Her thesis title was 'Molecular studies on phototactic motility in the unicellular cyanobacterium Synecochystis sp. PCC 6803'. She carried out postdoctoral work for the University of Tokyo from 2003–2004. Her research interests include molecular mechanisms involved in photoresponses, photoreceptors and regulatory systems, in cyanobacteria and plants.

Masahiko Ikeuchi is the Professor of Plant Molecular Biology at Department of Life Sciences (Biology), The University of Tokyo. He received his PhD in Plant Biology from the University of Tokyo. His research interests include photosystem II, molecular acclimation of photosynthetic system, phototaxis of cyanobacteria, photoreceptors and redox sensors, signal transduction, genome biology and bioinformatics of photosynthetic organisms, molecular evolution of photosynthetic apparatus and photosynthesis genes, and photosynthetic production of biopolymers.

Table 1 List of possible pil-related genes in the genome of Synechococcus and Anabaena

| GeneID | Gene name | Proposed function | Anabaena sp. PCC 7120 |
|--------|-----------|-------------------|-----------------------|
| slr1694 | pilA1     | Prepilin (pilus subunit) | all0563 |
| slr2015 | pilA9     | Prepilin (pilus subunit) | all1765 |
| slr2016 | pilA10    | Prepilin (pilus subunit) | all1769 |
| slr2017 | pilA11    | Prepilin (pilus subunit) | all1770 |
| slr0063 | pilB1     | Pilus extension | all2444 |
| slr0161 | pilT1     | Pilus retraction, ATPase | all3329 |
| slr0162-0163 | pilC | Inner membrane protein | all2442 |
| slr1274 | pilM      | Inner membrane protein | alr1028 |
| slr1275 | pilN      | Inner membrane protein | alr1029 |
| slr1276 | pilO      | Inner membrane protein | alr1030 |
| slr1277 | pilQ      | Pore at the outer membrane | alr1031 |
| slr1120 | pilD      | Prepilin peptidase | alr1315 |

Signal transduction for pilus assembly and phototaxis

| GeneID | Gene name | Proposed function | Anabaena sp. PCC 7120 |
|--------|-----------|-------------------|-----------------------|
| sll0038 | pixG, taxP1 | Response regulator | all1072 |
| sll0039 | pixH, taxY1 | Response regulator | all1071 |
| sll0040 | pixI, taxW1 | Coupling factor | all1070 |
| sll0041 | pixJ1, taxD1 | Phytochrome-like MCP | all1069 |
| sll0042 | pixJ2, taxD' | MCP | — |
| sll0043 | pixL, taxAY1 | Histidine kinase | all1068 |
| sll1533 | pilT2      | Positive phototaxis | — |
| sll0821 | cph2       | Negative phototaxis to blue light | — |
| slr1041 | pilG, taxP3 | Response regulator | all0930 |
| slr1042 | pilH, taxY3 | Response regulator | all0929 |
| slr1043 | pilI, taxW3 | Coupling factor | all0928 |
| slr1044 | pilJ, taxD3, ctrl | Intracellular receptor | all0927 |
| slr0073 | pilL-N     | N-terminal of histidine kinase | all0926 |
| slr0322 | pilL-C, taxAY3 | C-terminal of histidine kinase | — |
| slr2031 | —          | PP2C-type protein phosphatase | all1758 |
| sll1575 | spkA       | Ser/Thr protein kinase | all4518 |
| slr1697 | spkB       | Ser/Thr protein kinase | alr3268 |
| slr1443 | spkE       | Ser/Thr protein kinase | alr2412 |
| slr1991 | cya1       | Adenylate cyclase | alr0295 |
| sll1371 | sycrp1     | cAMP receptor protein | — |

The phototaxis of this organism were identified. This led to the discovery of a novel photoreceptor. These advances in our understanding of the motility of *Synechocystis* are described below.

### 1. Genes involved in the biogenesis of the type IV pilus

Several genes were found to be essential for cell motility, pilus biogenesis, and natural transformation competency in *Synechocystis*. These are known as the *pil* genes and they show slight homology to the type IV pilus biogenesis genes found in many gram-negative bacteria (Table 1). Type IV pili are typical extracellular structures located at the pole of the cell and bearing unique pilin subunits. Pilins are synthesized as a precursor (prepilin), in which a unique leader sequence is cleaved off on the cytoplasmic side by a specific prepilin peptidase. This peptidase recognizes a region around the cleavage site that includes the N-terminal part of the mature protein. It is well known that the extension, tethering, and then retraction activities of type IV pili are involved in various cellular functions in many pathogenic and non-pathogenic bacteria, such as motility, transformation, infection, and adhesion.

A

$pixG$ $(sll0038)$ $pixH$ $(sll0039)$ $pixI$ $(sll0040)$ $pixJ1$ $(sll0041)$ $pixJ2$ $(sll0042)$ $pixL$ $(sll0043)$

$pilG$ $(slr1041)$ $pilH$ $(slr1042)$ $pilI$ $(slr1043)$ $pilJ$ $(slr1044)$ $pilL-C$ $(slr0322)$ $pilL-N$ $(slr0073)$

$sll1291$ $sll1292$ $sll1293$ $sll1294$ $sll1296$

category PatA CheY CheW MCP (methyl-accepting chemotaxis protein) CheA

B

$pilG$ $pilH$ $pilI$ $pilJ$ $pilK$ $pilL$

category CheY CheY CheW MCP CheR CheA

Fig. 3 Gene organization of the $pixG$ (formerly $pis-$), $pilG$ and $sll1291$ clusters of *Synechocystis* (A) and the $pilG$ cluster of *Pseudomonas* (B). Homologous regulatory components for the chemotactic flagellar switching in enteric bacteria are shown as category. The direction of the arrows indicates the relative transcriptional orientation.

in its total loss of motility and the thick pili on its cell surface. ${ }^{20,21}$ $pilD$ is predicted to encode a protein that functions as a prepilin peptidase and a mature pilin methylase. ${ }^{27,44}$ The $pilD$ disruptant in *Synechocystis* lost both the thick and thin pili and showed a non-motile phenotype. ${ }^{20,21}$ Notably, all of these non-motile $pil$ mutants completely lost the natural transformation competency that includes the uptake of exogenous DNA. ${ }^{20,21}$ These observations strongly suggest that the thick pili serve as the apparatus used for DNA uptake and twitching motility. ${ }^{20,21,33}$

### 2. Regulatory genes

Large-scale gene disruption and random tagging screening have revealed that some of the many regulatory genes in the *Synechocystis* genome are responsible for regulating cell motility or phototaxis. The best examples of these are the gene clusters with annotation of MCP with a sensor domain and a His-Asp phosphorelay signal transduction system. Many cyanobacteria harbor three distinct sets of such clusters: e.g. the $slr1041$, $sll0038$, and $sll1291$ clusters in *Synechocystis* sp. PCC 6803. In general, each cluster consists of five genes organized in tandem, namely, $patA$-type, $cheY$-type, $cheW$-type, $mcp$, and $cheA$-type genes. This basic gene arrangement is found in the $sll1291$ cluster while the other clusters show unique variations (Fig. 3A). The $slr1041$ cluster is involved in the biogenesis of the thick pilus structure while the $sll0038$ cluster is involved in regulating positive phototaxis (see sections 2–1 and 2–2). ${ }^{33,45,46}$ A similar gene arrangement is found in the $pilG/-H/-I/-J/-K/-L$ gene cluster in the gram-negative soil bacterium *Pseudomonas aeruginosa*, involved in regulating its type IV pilus assembly and motility (Fig. 3B). ${ }^{47,48}$ Notably, *E. coli* bears a well-understood MCP-CheA-CheY system of signal transduction that switches flagellar rotation in response to chemosignals. ${ }^{49}$ The *E. coli* MCP in cytoplasmic membranes receives a chemosignal and activates His-kinase CheA via a coupling protein CheW. ${ }^{50}$ CheY is then phosphorylated by CheA and binds to the switch component FliM at the flagellar motor. ${ }^{51}$ Similar events may take place in *P. aeruginosa* and *Synechocystis*, although significant differences can be seen in their MCP- and PatA-encoding genes since the sensor domain of their MCPs has a different specificity and PatA is a cyanobacteria-specific response regulator. ${ }^{52}$ Based on homology to the $pilG/-H/-I/-J/-K/-L$ gene cluster in

$P$. *aeruginosa* and the phenotypes resulting from disrupting the genes in each cluster, we designated the $slr1041$, $slr1042$, $slr1043$, and $slr1044$ genes as $pilG$, $pilH$, $pilI$, and $pilJ$, respectively. The $cheA$-like gene $pilL$ was absent from this cluster but its sequence components were found in split genes ($slr0073$ and $slr0322$) present in separate loci. ${ }^{33}$ The $sll0038$, $sll0039$, $sll0040$, $sll0041$, $sll0042$, and $sll0043$ genes were designated as $pixG$, $pixH$, $pixI$, $pixJ1$, $pixJ2$, and $pixL$ (formerly $pis$ genes), respectively. ${ }^{21,45}$

#### 2-1 Regulatory genes involved in thick pilus biogenesis: the $pilG$ cluster

Disruption of $pilG$ ($slr1041$) reduced motility of *Synechocystis* on agar plates to some extent, whereas disruption of the $pilH$ ($slr1042$), $pilI$ ($slr1043$), and $pilJ$ ($slr1044$) genes and the two genes making up $pilL$ led to a complete loss of motility. ${ }^{33}$ Surprisingly, the $pilJ$-disruptant lost its thick pili whereas the $pilH$-disruptant displayed a hyper-piliated phenotype resembling the effect of disrupting $pilT1$. The $pilI$-disruptant retained its thick pili but considerably fewer were present. However, when the two split genes constituting the components of $pilL$ ($slr0073$ and $slr0322$) were individually disrupted, the $slr0073$-disruptant showed hyper-piliation whereas the $slr0322$-disruptant lost its thick pili. These observations suggest that (i) signal transduction in this system modulates the biogenesis of thick pili and (ii) the response can range from hyper-piliation to non-piliation. This is similar to observations for the PilB1-PilT1 system. Since Slr0073 and Slr0322 correspond to the N-terminal HPt (His-containing phosphotransfer) domain and the C-terminal catalytic domains of CheA/PilL, respectively, ${ }^{53,54}$ the $slr0073$ and $slr0322$ genes were designated as $pilL-N$ and $pilL-C$, respectively. ${ }^{33}$ It is likely that the $cheA$-like $pilL$ gene was excised from the original $pilG$ cluster and further split into $slr0073$ and $slr0322$ onto separate loci during the evolution of *Synechocystis*.

Mutational analysis has revealed that a large number of genes are responsible for cell motility in *Synechocystis*. The eukaryotic-type Ser/Thr protein kinase genes $spkA$ and $spkB$ and a putative PP2C-type protein phosphatase gene $slr2031$ are involved in cell motility, although their target proteins in vivo have not yet been determined. ${ }^{18,55,56}$ While we found that the putative protein kinase gene $slr1443$ ($spkE$) encodes a protein

without any detectable kinase activity and its disruption did not affect motility, 57 in another laboratory a similar disruption did result in a nonmotile phenotype. 58 An adenylate cyclase gene, *cyal* (*slr1991*), was also found to be essential for motility and the addition of cAMP recovered the nonmotile phenotype of the *cyal* disruptant. 59 A cAMP receptor gene, *sycrpI* (*sll1371*), was also essential for cell motility and thick pilus biogenesis. 34 Furthermore, DNA microarray and gel mobility shift assay analysis revealed that the SYCRP1 protein directly regulates the expression of the *slr1667-slr1668* operon. 60 Since the *slr1667* and *slr1668* target genes are essential for the biogenesis of thick pili, 61 it appears that cAMP-mediated signal transduction participates in regulating motility. Transposon mutagenesis also indicated that a number of other genes, including some novel genes, are involved in *Synechocystis* motility. 22 Thus, the signal transduction pathway controlling thick pilus biogenesis and leading to cell motility in cyanobacteria is complex.

### 2-2 Regulatory genes involved in positive phototaxis: the *pixG* cluster

The *Synechocystis* disruptants of *pixG* (*sll0038*), *pixH* (*sll0039*), *pixJ1* (*sll0041*), *pixJ2* (*sll0042*), or *pixL* (*sll0043*) lost their positive phototaxis response but showed negative phototaxis away from the light source. 45 This indicates that these genes are essential for positive phototaxis. It also implies that wild type cells are capable of both positive and negative phototaxis, even though they consistently show positive phototaxis under normal light conditions with a wide range of wavelengths. The gene in the middle of the cluster, the *cheW*-type *pixI* (*sll0040*) gene, is effective but not essential for positive phototaxis, which suggests that it plays an auxiliary role in signal transduction. As will be described in section 3, the *pixJ1* gene product (PixJ1) is a novel photoreceptor that absorbs blue or green light. Hence, we hypothesize that the pathway leading to positive phototaxis involves the reception of a light signal by PixJ1, which then activates the CheA-type PixL protein, which in turn leads to the phosphorylation of the PatA-type PixG and CheY-type PixH proteins. 45 A similar conclusion was obtained by random transposon tagging. 46 Since the disruptants of the *pix* genes still show negative phototaxis, *Synechocystis* seems to have at least two independent light-responsive systems that control the complex phototactic motility of this organism.

A phytochrome-like gene *cph2* (*sll0821*) is involved in suppressing the positive phototaxis toward blue light (400–500 nm) in *Synechocystis*, 62 although its spectral properties in its native form have not yet been determined. The *pilT* homolog *pilT2* (*sll1533*) is essential for positive phototaxis toward white light in *Synechocystis*, 20 whereas disruption of *pilT1* resulted in a hyper-piliated nonmotile phenotype. 20,35 Given the general view that PilT retracts pili into cells, it may be that PilT2 in *Synechocystis* is involved in the directional retraction of the thick pili that leads to phototactic movement.

## 3 The novel GAF-based photoreceptor PixJ1

Domain analysis shows that PixJ1 has two GAF domains in addition to N-terminal transmembrane helices and a C-terminal MCP signal domain (Fig. 4). Membrane topology prediction suggests that PixJ1 is anchored on the plasma membrane and both its GAF and MCP signal domains reside in the cytoplasm. 33 The GAF domain is a superfamily with broad sequence variety but with a specific three-dimensional structural fold. It was first recognized in cGMP phosphodiesterase, *Anabaena* adenylate cyclase and *E. coli* FhlA protein 63 and is now accepted to be widely distributed throughout the three kingdoms. Since this domain is usually accompanied by signal output domains such as adenylate cyclase and His kinase, it is very likely that the GAF domain participates in signal perception. Indeed, the chromophore-binding domain of plant phytochromes also belongs to the GAF superfamily. 63 It is a characteristic feature of plant phytochromes that they photoconvert between a red-absorbing form (*P~r~, λ~max~ = 665 nm*) and a far red-absorbing form (*P~fr~, λ~max~ = 730 nm*). 64 Recent genome projects have revealed that cyanobacteria such as *Synechocystis* and *Anabaena* have an exceptionally large number of GAF domains, which include the chromophore-binding domains of their phytochromes. 65 Cluster analysis showed that cyanobacteria contain an additional subgroup of GAF domains somehow related to the chromophore-binding domain of phytochrome. 66 The second GAF domain of PixJ1 belongs to this subgroup (see section 5-2).

We recently isolated a poly-His-tagged PixJ1 protein from *Synechocystis* cells. 66 It was found that a chromophore in PixJ1 emits fluorescence after SDS-PAGE followed by soaking with zinc ions. This strongly suggests that a putative open-chain tetrapyrrole is covalently attached to PixJ1. Surprisingly, the isolated PixJ1 showed unprecedented reversible photoconversion between a blue-absorbing *P~b~* form (*λ~max~ = 435 nm*) and a green-absorbing *P~g~* form (*λ~max~ = 535 nm*) (Fig. 5). Dark incubation of *P~g~* resulted in a slow reversion to *P~b~*, suggesting that PixJ1 serves as a blue light receptor. These spectral properties are quite distinct from those of the known phytochromes and bacteriophytochromes, which predominantly show photoconversion in the red/far red regions.

![Fig. 5 Difference absorption spectrum of affinity-purified His-tagged PixJ1 irradiated with green minus blue light.](image.png)

In plant phytochromes, the chromophore phytochromobilin (PΦB) is covalently attached to the Cys residue in the GAF domain via a thioether linkage. 67 In the bacteriophytochrome from *Deinococcus radiodurans*, the chromophore biliverdin (BV) has been proposed to bind to the His residue in the GAF domain via a Schiff base linkage. 68 Both Cys and His residues are conserved as a 'Cys-His' motif in the second GAF domain of PixJ1, whereas only the His residue is conserved in the first GAF domain (Fig. 4). Site-directed mutagenesis of the 'Cys-His' motif in the second domain abolished the covalent binding of the fluorescent chromophore to PixJ1. This strongly suggests that the chromophore is covalently ligated to the second GAF domain. 66

At present, the chemical nature of the putative open-chain tetrapyrrole in PixJ1 has not been determined. In general, all open-chain tetrapyrrole chromophores are derived from protoheme by heme oxygenase, which produces the key compound.

![Fig. 4 Domain structure of the PixJ1 photoreceptor of *Synechocystis* sp. PCC 6803. The domain structure from its N-terminus to its C-terminus is shown from left to right. The amino acids that constitute the conserved chromophore-binding motif in plant phytochromes are represented in the GAF domains.](image_2.png)

biliverdin (BV).<sup>69</sup> PΦB, found in most of plant phytochromes, is generated from BV by PΦB:ferredoxin oxidoreductase.<sup>70</sup> Phyco- cyanobilin (PCB), found in the cyanobacterial phytochrome Cph1<sup>71</sup> and in photosynthetic antenna proteins of cyanobacteria and algae, such as phycocyanin and allophycocyanin,<sup>69</sup> is generated from BV by PCB:ferredoxin oxidoreductase.<sup>72,73</sup> Judging from the spectral properties of PixJ1, it is unlikely that the chromophore is BV or PCB. Three other open-chain tetrapyrroles have been found in some cyanobacteria, *i.e.*, phycoerythrobilin (PEB, λ<sub>max</sub> = 540–575 nm), phycourobilin (λ<sub>max</sub> = 490–500 nm), and phycobiliviolin (λ<sub>max</sub> = 575 nm).<sup>69,74–76</sup> PEB is generated from BV by a two-steps reaction catalyzed by dihydroliverdin:ferredoxin oxidoreductase and phycoerythrobilin:ferredoxin oxidoreductase.<sup>69,72,73</sup> In the *Mastigocladus laminosus*, PecE and PecF proteins were identified to catalyze both the covalent attachment of PCB to phycoerythrocyanin α subunit PecA and the isomerization of PCB to phycoviolobilin.<sup>75,76</sup> However, none of these open-chain tetrapyrroles have been detected in *Synechocystis*. Nevertheless, BV reductase, known to convert BV to bilirubin during the heme catabolism in animals,<sup>77</sup> was found to be encoded by the *bvdR* gene in the *Synechocystis* genome.<sup>17</sup> A recombinant protein encoded by *bvdR* showed NADPH-dependent BV reductase activity *in vitro*, although the physiological role of this gene in *Synechocystis* remains to be elucidated.<sup>78</sup> The absorption peak of bilirubin (λ<sub>max</sub> = 450 nm) is very close to that of the P<sub>b</sub> form of PixJ1, and those of BV (λ<sub>max</sub> = 645 nm) and PCB (λ<sub>max</sub> = 600–670 nm) are far longer than those of the P<sub>g</sub> or P<sub>b</sub> forms of PixJ1. The possibility that bilirubin is the chromophore of PixJ1 is currently under investigation.

### 4 Wavelength dependence of the phototactic motility of *Synechocystis*

Wild type *Synechocystis* cells show complex positive phototactic motility toward light ranging from yellow to red (560 to 720 nm)<sup>6,79</sup> and negative phototaxis away from light in the UV region (360 nm).<sup>79</sup> High intensity blue light (470 nm) and light in the red region (600 to 700 nm) also induce negative phototaxis.<sup>6</sup> The *pixJ1*-mutant lost these positive phototactic responses but showed negative phototaxis away from yellow to red light (unpublished results).<sup>6</sup> These observations suggest that multiple photoreceptors with different spectral properties are involved in the phototactic motility of *Synechocystis*. Based on the spectra that lead to the positive phototactic response, the PixJ1 photoreceptor was expected to recognize yellow to red light and thereby activate positive phototactic motility. However, we found that the isolated PixJ1 protein actually absorbs blue (λ<sub>max</sub> = 435 nm in the P<sub>b</sub> form) and green light (λ<sub>max</sub> = 535 nm in the P<sub>g</sub> form). Considering the stability of P<sub>b</sub> in darkness, we assume that PixJ1 is a blue light receptor *in vivo* and that the blue light signal inactivates the positive phototaxis via PixJ1. In other words, the dark-stable P<sub>b</sub> is a switch that allows positive phototaxis towards a broad range of wavelengths other than blue. Irradiation with blue light leads to P<sub>g</sub>, which switches off positive phototaxis. Consistently, wild type cells showed no positive phototactic motility in response to 427 nm<sup>6</sup> and 460 nm light (unpublished results). In this case, we must assume that other photoreceptor(s) recognize the direction of incident light from yellow to red. That the positive phototaxis of *Synechocystis* is regulated by such a complicated mechanism is supported by the non-linear dose-response dependency of its phototaxis towards light (unpublished data).<sup>6</sup> It has also been suggested that *Synechocystis* has additional photoreceptors that are responsible for negative phototaxis because the *pixJ1* mutant still retained its capability for negative phototaxis. Alternatively, *Synechocystis* may utilize the same photoreceptors for positive and negative phototaxis since both phototactic responses are generated by similar wavelength ranges.

---

### 5 Prospectives

#### 5-1 Motility of filamentous cyanobacteria

Homologs of all the genes determined to be essential for the motility and positive phototaxis of *Synechocystis* have been found in the genome of the filamentous cyanobacterium *Anabaena* sp. PCC 7120 (Table 1).<sup>80</sup> The draft sequence of the related filamentous cyanobacterium *Nostoc punctiforme* also has a similar set of genes. *Nostoc* develops motile hormogonium, which exhibits gliding motility, while *Anabaena* does not develop motile hormogonium.<sup>81</sup> *Calothrix* sp. PCC 7601, also a filamentous cyanobacterium, only produces pili-like structures on its cell surface when it has differentiated into motile hormogonium cells.<sup>82</sup> Mutational analysis should yield further clues regarding the molecular mechanisms behind the pili-dependent gliding motility of cyanobacteria.

Three sets of gene clusters comprising *patA*, *cheY*, *cheW*, *mcp*, and *cheA* homologs are also present in a wide range of cyanobacterial genomes, including *Thermosynechococcus elongatus* BP-1, *Trichodesmium erythraeum* IMS101, *Synechococcus* sp. PCC 7002, *Anabaena* sp. PCC 7120, and *Nostoc punctiforme*, although *Nostoc* has four sets of gene clusters. At least one cluster in each organism contains a *pixJ1* homolog, encoding a protein consisting of GAF domain(s) and a MCP signal domain. No *pixJ1* homolog has been found in genome databases of organisms other than cyanobacteria, which indicates that the use of GAF-related proteins is specific to the cyanobacteria. Notably, the genome of the unicellular *Synechococcus* sp. WH8102 that exhibits swimming motility in liquid contains only a few of the *pil*-like genes found in the gliding *Synechocystis*, such as *pilB*, *pilC*, and *pilD*. The cell surface of *Synechococcus* does not bear any visible structures that may be involved in swimming.<sup>83</sup> Thus, *Synechococcus* may not employ pili to swim. Instead, it may use an as yet unidentified mechanism.

#### 5-2 GAF-related photoreceptor candidates in cyanobacteria

Recently, a number of genes encoding bacteriophytochromes have been found not only in photosynthetic but also in non-photosynthetic bacterial genomes. Although bacteriophytochromes show P<sub>r</sub>/P<sub>f</sub> photoconversion like the plant phytochromes, the open-chain tetrapyrrole chromophores attached to them vary. PCB has been demonstrated to attach to Cph1 of *Synechocystis*,<sup>71</sup> while BV is assumed to attach to the BphPs of *Bradyrhizobium*,<sup>84</sup> *Deinococcus radiodurans*,<sup>85</sup> and *Pseudomonas aeruginosa*<sup>85</sup> and to Agpl of *Agrobacterium tumefaciens*.<sup>86</sup> However, several photoreceptor candidates that contain phytochrome-like GAF domains have been observed in cyanobacteria, including PixJ1, Cph2, PlpA, and Sll1473 from *Synechocystis*, RcaE from *Fremyella diplosiphon*, and CikA from *Synechococcus elongatus* PCC 7942.<sup>45,62,87–90</sup> These photoreceptor candidates have been predicted by physiological analysis to be involved in responses to blue light (Cph2, PlpA)<sup>62,88</sup> or red and green light (RcaE).<sup>91</sup> Only RcaE was demonstrated to covalently bind a linear tetrapyrrole-like chromophore *in vivo*,<sup>92</sup> but spectral properties of RcaE holochrome have not yet been determined. The discovery of the unique PixJ1 holochrome raises the possibility that more GAF-related photoreceptors with novel spectral properties may exist. In this case, while it may be easy to reconstitute an apoprotein with known chromophores *in vitro*, this may not be sufficient for characterizing the spectrum to which the photoreceptor responds. Before this is done, the native chromophore of the photoreceptor must be determined by isolating the holochrome that has been assembled *in situ*.

---

### References

1. G. I. McFadden, Primary and secondary endosymbiosis and the origin of plastids, *J. Phycol.*, 2001, **37**, 951–959.

2 B. Diehn, M. Feinleib, W. Haupt, E. Hildebrand, F. Lenci and W. Nultsch, Terminology of behavioral responses of motile microorganisms, *Photochem. Photobiol.*, 1977, **26**, 559–560.

3 D. P. Häder, Photosensory behavior in procaryotes, *Microbiol. Rev.*, 1987, **51**, 1–21.

4 J. B. Waterbury, J. M. Willey, D. G. Franks, F. W. Valois and S. W. Watson, A cyanobacterium capable of swimming motility, *Science*, 1985, **230**, 74–76.

5 J. Henrichsen, Bacterial surface translocation: a survey and a classification, *Bacteriol. Rev.*, 1972, **36**, 478–503.

6 W. O. Ng, A. R. Grossman and D. Bhaya, Multiple light inputs control phototaxis in *Synechocystis* sp. strain PCC 6803, *J. Bacteriol.*, 2003, **185**, 1599–1607.

7 R. Y. Stanier, R. Kunisawa, M. Mandel and G. Cohen-Bazire, Purification and properties of unicellular blue-green algae (order Chroococcales), *Bacteriol. Rev.*, 1971, **35**, 171–205.

8 L. N. Halfen and R. W. Castenholz, Gliding in a blue-green alga: a possible mechanism, *Nature*, 1970, **225**, 1163–1165.

9 E. Hoiczyk and W. Baumeister, The junctional pore complex, a prokaryotic secretion organelle, is the molecular motor underlying gliding motility in cyanobacteria, *Curr. Biol.*, 1998, **8**, 1161–1168.

10 E. Hoiczyk, Gliding motility in cyanobacterial: observations and possible explanations, *Arch. Microbiol.*, 2000, **174**, 11–17.

11 E. Hoiczyk, Structural and biochemical analysis of the sheath of *Phormidium uncinatum*, *J. Bacteriol.*, 1998, **180**, 3923–3932.

12 D. G. Adams, D. Ashworth and B. Nelmes, Fibrillar array in the cell wall of a gliding filamentous cyanobacterium, *J. Bacteriol.*, 1999, **181**, 884–892.

13 E. Hoiczyk and W. Baumeister, Oscillin, an extracellular, Ca²⁺-binding glycoprotein essential for the gliding motility of cyanobacteria, *Mol. Microbiol.*, 1997, **26**, 699–708.

14 S. P. Adhikary, J. Weckesser, U. J. Jürgens, J. R. Golecki and D. Borowiak, Isolation and chemical characterization of the sheath from the cyanobacterium *Chroococcus minutus* SAG B.41.79., *J. Gen. Microbiol.*, 1986, **132**, 2595–2599.

15 M. Pritzer, J. Weckesser and U. J. Jürgens, Sheath and outer membrane components from the cyanobacterium *Fischerella* sp. PCC 7414, *Arch. Microbiol.*, 1989, **153**, 7–11.

16 B. Brahamsa, An abundant cell-surface polypeptide is required for swimming by the nonflagellated marine cyanobacterium *Synechococcus*, *Proc. Natl. Acad. Sci. USA*, 1996, **93**, 6504–6509.

17 T. Kaneko, S. Sato, H. Kotani, A. Tanaka, E. Asamizu, Y. Nakamura, N. Miyajima, M. Hirosawa, M. Sugiura, S. Sasamoto, T. Kimura, T. Hosouchi, A. Matsuno, A. Muraki, N. Nakazaki, K. Naruo, S. Okumura, S. Shimo, C. Takeuchi, T. Wada, A. Watanabe, M. Yamada, M. Yasuda and S. Tabata, Sequence analysis of the genome of the unicellular cyanobacterium *Synechocystis* sp. strain PCC 6803. II. Sequence determination of the entire genome and assignment of potential protein-coding regions, *DNA Res.*, 1996, **3**, 109–136.

18 A. Kamei, T. Ogawa and M. Ikeuchi, Identification of a novel gene (*slr2031*) involved in high-light resistance in the cyanobacterium *Synechocystis* sp. PCC 6803, *Photosynthesis: Mechanism and Effects*, 1998, 2901–2905.

19 D. Bhaya, N. Watanabe, T. Ogawa and A. R. Grossman, The role of an alternative sigma factor in motility and pilus formation in the cyanobacterium *Synechocystis* sp. strain PCC 6803, *Proc. Natl. Acad. Sci. USA*, 1999, **96**, 3188–3193.

20 D. Bhaya, N. R. Bianco, D. Bryant and A. Grossman, Type IV pilus biogenesis and motility in the cyanobacterium *Synechocystis* sp. PCC 6803, *Mol. Microbiol.*, 2000, **37**, 941–951.

21 S. Yoshihara, X. Geng, S. Okamoto, K. Yura, T. Murata, M. Go, M. Ohmori and M. Ikeuchi, Mutational analysis of genes involved in pilus structure, motility and transformation competency in the unicellular motile cyanobacterium *Synechocystis* sp. PCC 6803, *Plant Cell Physiol.*, 2001, **42**, 63–73.

22 D. Bhaya, A. Takahashi, P. Shahi and A. R. Grossman, Novel motility mutants of *Synechocystis* strain PCC 6803 generated by in vitro transposon mutagenesis, *J. Bacteriol.*, 2001, **183**, 6140–6143.

23 J. S. Mattick, C. B. Whitchurch and R. A. Alm, The molecular genetics of type-4 fimbriae in *Pseudomonas aeruginosa*—a review, *Gene*, 1996, **179**, 147–155.

24 P. A. Sastry, J. R. Pearlstone, L. B. Smillie and W. Paranchych, Amino acid sequence of pilin isolated from *Pseudomonas aeruginosa* PAK, *FEBS Lett.*, 1983, **151**, 253–256.

25 A. M. Spormann, Gliding motility in bacteria: insights from studies of *Myxococcus xanthus*, *Microbiol. Mol. Biol. Rev.*, 1999, **63**, 621–641.

26 M. Koomey, Competence for natural transformation in *Neisseria gonorrhoeae*: a model system for studies of horizontal gene transfer, *APMIS Suppl.*, 1998, **84**, 56–61.

27 D. N. Nunn and S. Lory, Product of the *Pseudomonas aeruginosa* gene *pilD* is a prepilin leader peptidase, *Proc. Natl. Acad. Sci. USA*, 1991, **88**, 3281–3285.

28 M. S. Strom and S. Lory, Structure-function and biogenesis of the type IV pili, *Annu. Rev. Microbiol.*, 1993, **47**, 565–596.

29 J. M. Skerker and H. C. Berg, Direct observation of extension and retraction of type IV pili, *Proc. Natl. Acad. Sci. USA*, 2001, **98**, 6901–6904.

30 J. S. Mattick, Type IV pili and twitching motility, *Annu. Rev. Microbiol.*, 2002, **56**, 289–314.

31 S. L. Bardy, S. Y. Ng and K. F. Jarrell, Prokaryotic motility structures, *Microbiol.*, 2003, **149**, 295–304.

32 M. J. McBride, Bacterial gliding motility: multiple mechanisms for cell movement over surfaces, *Annu. Rev. Microbiol.*, 2001, **55**, 49–75.

33 S. Yoshihara, X. Geng and M. Ikeuchi, *pilG* gene cluster and split *pilL* genes involved in pilus biogenesis, motility and genetic transformation in the cyanobacterium *Synechocystis* sp. PCC 6803, *Plant Cell Physiol.*, 2002, **43**, 513–521.

34 H. Yoshimura, S. Yoshihara, S. Okamoto, M. Ikeuchi and M. Ohmori, A cAMP receptor protein, SYCRP1, is responsible for the cell motility of *Synechocystis* sp. PCC 6803, *Plant Cell Physiol.*, 2002, **43**, 460–463.

35 S. Okamoto and M. Ohmori, The cyanobacterial PilT protein responsible for cell motility and transformation hydrolyzes ATP, *Plant Cell Physiol.*, 2002, **43**, 1127–1136.

36 S. S. Wu, J. Wu and D. Kaiser, The *Myxococcus xanthus pilT* locus is required for social gliding motility although pili are still produced, *Mol. Microbiol.*, 1997, **23**, 109–121.

37 M. Wolfgang, P. Lauer, H. S. Park, L. Brossay, J. Hebert and M. Koomey, PilT mutations lead to simultaneous defects in competence for natural transformation and twitching motility in piliated *Neisseria gonorrhoeae*, *Mol. Microbiol.*, 1998, **29**, 321–330.

38 C. B. Whitchurch, M. Hobbs, S. P. Livingston, V. Krishnapillai and J. S. Mattick, Characterisation of a *Pseudomonas aeruginosa* twitching motility gene and evidence for a specialised protein export system widespread in eubacteria, *Gene*, 1991, **101**, 33–44.

39 C. B. Whitchurch and J. S. Mattick, Characterization of a gene, *pilU*, required for twitching motility but not phage sensitivity in *Pseudomonas aeruginosa*, *Mol. Microbiol.*, 1994, **13**, 1079–1091.

40 P. R. Martin, A. A. Watson, T. F. McCaul and J. S. Mattick, Characterization of a five-gene cluster required for the biogenesis of type 4 fimbriae in *Pseudomonas aeruginosa*, *Mol. Microbiol.*, 1995, **16**, 497–508.

41 D. Nunn, S. Bergman and S. Lory, Products of three accessory genes, *pilB*, *pilC*, and *pilD*, are required for biogenesis of *Pseudomonas aeruginosa* pili, *J. Bacteriol.*, 1990, **172**, 2911–2919.

42 W. Bitter, M. Koster, M. Latijnhouwers, H. de Cock and J. Tommassen, Formation of oligomeric rings by XcpQ and PilQ, which are involved in protein transport across the outer membrane of *Pseudomonas aeruginosa*, *Mol. Microbiol.*, 1998, **27**, 209–219.

43 T. Tonjum, D. A. Caugant, S. A. Dunham and M. Koomey, Structure and function of repetitive sequence elements associated with a highly polymorphic domain of the *Neisseria meningitidis* PilQ protein, *Mol. Microbiol.*, 1998, **29**, 111–124.

44 M. S. Strom, D. N. Nunn and S. Lory, A single bifunctional enzyme, PilD, catalyzes cleavage and N-methylation of proteins belonging to the type IV pilin family, *Proc. Natl. Acad. Sci. USA*, 1993, **90**, 2404–2408.

45 S. Yoshihara, F. Suzuki, H. Fujita, X. X. Geng and M. Ikeuchi, Novel putative photoreceptor and regulatory genes required for the positive phototactic movement of the unicellular motile cyanobacterium *Synechocystis* sp. PCC 6803, *Plant Cell Physiol.*, 2000, **41**, 1299–1304.

46 D. Bhaya, A. Takahashi and A. R. Grossman, Light regulation of type IV pilus-dependent motility by chemosensor-like elements in *Synechocystis* PCC 6803, *Proc. Natl. Acad. Sci. USA*, 2001, **98**, 7540–7545.

47 A. Darzins, The *pilG* gene product, required for *Pseudomonas aeruginosa* pilus production and twitching motility, is homologous to the enteric, single-domain response regulator CheY, *J. Bacteriol.*, 1993, **175**, 5934–5944.

48 A. Darzins, Characterization of a *Pseudomonas aeruginosa* gene cluster involved in pilus biosynthesis and twitching motility: sequence similarity to the chemotaxis proteins of enterics and the gliding bacterium *Myxococcus xanthus*, *Mol. Microbiol.*, 1994, **11**, 137–153.

49 S. I. Aizawa, C. S. Harwood and R. J. Kadner, Signaling components in bacterial locomotion and sensory reception, *J. Bacteriol.*, 2000, **182**, 1459–1471.

50 M. Eisenbach, Control of bacterial chemotaxis, *Mol. Microbiol.*, 1996, **20**, 903–910.

51 R. E. Silversmith and R. B. Bourret, Throwing the switch in bacterial chemotaxis, *Trends Microbiol.*, 1999, **7**, 16–22.

52 J. Liang, L. Scappino and R. Haselkorn, The *patA* gene product, which contains a region similar to CheY of *Escherichia coli*, controls heterocyst pattern formation in the cyanobacterium *Anabaena* 7120, *Proc. Natl. Acad. Sci. USA*, 1992, **89**, 5655–5659.

53 A. M. Bilwes, L. A. Alex, B. R. Crane and M. I. Simon, Structure of CheA, a signal-transducing histidine kinase, *Cell*, 1999, **96**, 131–141.

54 L. Mourey, S. Da Re, J. D. Pedelacq, T. Tolstykh, C. Faurie, V. Guillet, J. B. Stock and J. P. Samama, Crystal structure of the CheA histidine phosphotransfer domain that mediates response regulator phosphorylation in bacterial chemotaxis, *J. Biol. Chem.*, 2001, **276**, 31074–31082.

55 A. Kamei, T. Yuasa, K. Orikawa, X. X. Geng and M. Ikeuchi, A eukaryotic-type protein kinase, SpkA, is required for normal motility of the unicellular Cyanobacterium *Synechocystis* sp. strain PCC 6803, *J. Bacteriol.*, 2001, **183**, 1505–1510.

56 A. Kamei, S. Yoshihara, T. Yuasa, X. Geng and M. Ikeuchi, Biochemical and functional characterization of a eukaryotic-type protein kinase, SpkB, in the cyanobacterium, *Synechocystis* sp. PCC 6803, *Curr. Microbiol.*, 2003, **46**, 296–301.

57 A. Kamei, T. Yuasa, X. Geng and M. Ikeuchi, Biochemical examination of the potential eukaryotic-type protein kinase genes in the complete genome of the unicellular cyanobacterium *Synechocystis* sp. PCC 6803, *DNA Res.*, 2002, **9**, 71–78.

58 Y. H. Kim, Y. M. Park, S. J. Kim, Y. I. Park, J. S. Choi and Y. H. Chung, The role of Slr1443 in pilus biogenesis in *Synechocystis* sp. PCC 6803: involvement in post-translational modification of pilins, *Biochem. Biophys. Res. Commun.*, 2004, **315**, 179–186.

59 K. Terauchi and M. Ohmori, An adenylate cyclase, Cya1, regulates cell motility in the cyanobacterium *Synechocystis* sp. PCC 6803, *Plant Cell Physiol.*, 1999, **40**, 248–251.

60 H. Yoshimura, S. Yanagisawa, M. Kanehisa and M. Ohmori, Screening for the target gene of cyanobacterial cAMP receptor protein SYCRP1, *Mol. Microbiol.*, 2002, **43**, 843–853.

61 H. Yoshimura, S. Yanagisawa, T. Hisabori and M. Ohmori, Molecular physiology of cAMP signaling cascade in *Synechocystis* PCC 6803, in *the 11th International Symposium on Phototrophic Prokaryotes*, Tokyo, 2003, p. 182.

62 A. Wilde, B. Fiedler and T. Borner, The cyanobacterial phytochrome Cph2 inhibits phototaxis towards blue light, *Mol. Microbiol.*, 2002, **44**, 981–988.

63 L. Aravind and C. P. Ponting, The GAF domain: an evolutionary link between diverse phototransducing proteins, *Trends Biochem. Sci.*, 1997, **22**, 458–459.

64 P. H. Quail, The phytochromes: a biochemical mechanism of signaling in sight?, *Bioessays*, 1997, **19**, 571–579.

65 M. Ohmori, M. Ikeuchi, N. Sato, P. Wolk, T. Kaneko, T. Ogawa, M. Kanehisa, S. Goto, S. Kawashima, S. Okamoto, H. Yoshimura, H. Katoh, T. Fujisawa, S. Ehira, A. Kamei, S. Yoshihara, R. Narikawa and S. Tabata, Characterization of genes encoding multi-domain proteins in the genome of the filamentous nitrogen-fixing Cyanobacterium *Anabaena* sp. strain PCC 7120, *DNA Res.*, 2001, **8**, 271–284.

66 S. Yoshihara, M. Katayama, X. Geng and M. Ikeuchi, Cyanobacterial phytochrome-like PixJ1 holoprotein shows novel reversible photoconversion between blue- and green-absorbing forms, submitted.

67 J. C. Lagarias and H. Rapoprt, Chromopeptides from phytochrome. The structure and linkage of the Pr form of the phytochrome chromophore, *J. Am. Chem. Soc.*, 1980, **102**, 4821–4828.

68 S. J. Davis, A. V. Vener and R. D. Vierstra, Bacteriophytochromes: phytochrome-like photoreceptors from nonphotosynthetic eubacteria, *Science*, 1999, **286**, 2517–2520.

69 S. I. Beale, Biosynthesis of open-chain tetrapyrroles in plants, algae, and cyanobacteria, *Ciba Found. Symp.*, 1994, **180**, 156–168 (discussion 168–171).

70 M. J. Terry, M. T. McDowell and J. C. Lagarias, (3Z)-and (3E)-phycochromobilin are intermediates in the biosynthesis of the phytochrome chromophore, *J. Biol. Chem.*, 1995, **270**, 11111–11118.

71 T. Hubschmann, T. Börner, E. Hartmann and T. Lamparter, Characterization of the Cph1 holo-phytochrome from *Synechocystis* sp. PCC 6803, *Eur. J. Biochem.*, 2001, **268**, 2055–2063.

72 J. Cornejo and S. I. Beale, Phycobilin biosynthetic reactions in extract of cyanobacteria, *Photosynth. Res.*, 1997, **51**, 223–230.

73 N. Frankenberg, K. Mukougawa, T. Kohchi and J. C. Lagarias, Functional genomic analysis of the HY2 family of ferredoxin-dependent bilin reductases from oxygenic photosynthetic organisms, *Plant Cell*, 2001, **13**, 965–978.

74 W. A. Sidler, in *The Molecular Biology of Cyanobacteria*, ed. D. A. Bryant, Kluwer Academic Publishers, 1994, pp. 139–216.

75 K. H. Zhao, M. G. Deng, M. Zheng, M. Zhou, A. Parbel, M. Storf, M. Meyer, B. Strohmann and H. Scheer, Novel activity of a phycobiliprotein lyase: both the attachment of phycocyanobilin and the isomerization to phycoviolobilin are catalyzed by the proteins PecE and PecF encoded by the phycoerythrocyanin operon, *FEBS Lett.*, 2000, **469**, 9–13.

76 M. Storf, A. Parbel, M. Meyer, B. Strohmann, H. Scheer, M. G. Deng, M. Zheng, M. Zhou and K. H. Zhao, Chromophore attachment to biliproteins: specificity of PecE/PecF, a lyase-isomerase for the photoactive 3(1)-cys-alpha 84-phycoviollobilin chromophore of phycoerythrocyanin, *Biochemistry*, 2001, **40**, 12444–12456.

77 S. W. Ryter and R. M. Tyrrell, The heme synthesis and degradation pathways: role in oxidant sensitivity. Heme oxygenase has both pro-and antioxidant properties, *Free Radical Biol. Med.*, 2000, **28**, 289–309.

78 W. M. Schluchter and A. N. Glazer, Characterization of cyanobacterial biliverdin reductase. Conversion of biliverdin to bilirubin is important for normal phycobiliprotein biosynthesis, *J. Biol. Chem.*, 1997, **272**, 13562–13569.

79 J. S. Choi, Y. H. Chung, Y. J. Moon, C. Kim, M. Watanabe, P. S. Song, C. O. Joe, L. Bogorad and Y. M. Park, Photomovement of the gliding cyanobacterium *Synechocystis* sp. PCC 6803, *Photochem. Photobiol.*, 1999, **70**, 95–102.

80 T. Kaneko, Y. Nakamura, C. P. Wolk, T. Kuritz, S. Sasamoto, A. Watanabe, M. Iriguchi, A. Ishikawa, K. Kawashima, T. Kimura, Y. Kishida, M. Kohara, M. Matsumoto, A. Matsuno, A. Muraki, N. Nakazaki, S. Shimpo, M. Sugimoto, M. Takazawa, M. Yamada, M. Yasuda and S. Tabata, Complete genomic sequence of the filamentous nitrogen-fixing cyanobacterium *Anabaena* sp. strain PCC 7120, *DNA Res.*, 2001, **8**, 205–213; 227–253.

81 J. C. Meeks, E. L. Campbell, M. L. Summers and F. C. Wong, Cellular differentiation in the cyanobacterium *Nostoc punctiforme*, *Arch. Microbiol.*, 2002, **178**, 395–403.

82 T. Damerval, G. Guglielmi, J. Houmard and N. T. De Marsac, Hormogonium differentiation in the cyanobacterium *Calothrix*: a photoregulated developmental process, *Plant Cell*, 1991, **3**, 191–201.

83 B. Brahamsha, Non-flagellar swimming in marine *Synechococcus*, *J. Mol. Microbiol. Biotechnol.*, 1999, **1**, 59–62.

84 E. Giraud, J. Fardoux, N. Fourrier, L. Hannibal, B. Genty, P. Bouyer, B. Dreyfus and A. Vermeglio, Bacteriophytochrome controls photosystem synthesis in anoxygenic bacteria, *Nature*, 2002, **417**, 202–205.

85 S. H. Bhoo, S. J. Davis, J. Walker, B. Karniol and R. D. Vierstra, Bacteriophytochromes are photochromic histidine kinases using a biliverdin chromophore, *Nature*, 2001, **414**, 776–779.

86 T. Lamparter, N. Michael, F. Mittmann and B. Esteban, Phytochrome from *Agrobacterium tumefaciens* has unusual spectral properties and reveals an N-terminal chromophore attachment site, *Proc. Natl. Acad. Sci. USA*, 2002, **99**, 11628–11633.

87 D. M. Kehoe and A. R. Grossman, New classes of mutants in complementary chromatic adaptation provide evidence for a novel four-step phosphorelay system, *J. Bacteriol.*, 1997, **179**, 3914–3921.

88 A. Wilde, Y. Churin, H. Schubert and T. Börner, Disruption of a *Synechocystis* sp. PCC 6803 gene with partial similarity to phytochrome genes alters growth under changing light qualities, *FEBS Lett.*, 1997, **406**, 89–92.

89 O. Schmitz, M. Katayama, S. B. Williams, T. Kondo and S. S. Golden, CikA, a bacteriophytochrome that resets the cyanobacterial circadian clock, *Science*, 2000, **289**, 765–768.

90 M. Katayama, X. Geng, M. Kobayashi and M. Ikeuchi, Identification of phytochrome-like protein and response regulator that are necessary for the expression of *cpcG2* gene in *Synechocystis* sp. PCC 6803, in *the 11th International Symposium on Phototrophic Prokaryotes*, Tokyo, 2003, p. 89.

91 D. M. Kehoe and A. R. Grossman, Similarity of a chromatic adaptation sensor to phytochrome and ethylene receptors, *Science*, 1996, **273**, 1409–1412.

92 K. Terauchi, B. L. Montgomery, A. R. Grossman, J. C. Lagarias and D. M. Kehoe, RcaE is a complementary chromatic adaptation photoreceptor required for green and red light responsiveness, *Mol. Microbiol.*, 2004, **51**, 567–577.
