
ARTICLES

nature  
structural &  
molecular biology  

---

An antibiotic factory caught in action  

Adrian T Keatinge-Clay$^{1}$, David A Maltby$^{2}$, Katalin F Medzihradszky$^{2}$, Chaitan Khosla$^{3}$ & Robert M Stroud$^{4}$  

The synthesis of aromatic polyketides, such as actinorhodin, tetracycline and doxorubicin, begins with the formation of a polyketide chain. In type II polyketide synthases (PKSs), chains are polymerized by the heterodimeric ketosynthase–chain length factor (KS-CLF). Here we present the 2.0-Å structure of the actinorhodin KS-CLF, which shows polyketides being elongated inside an amphipathic tunnel ~17 Å in length at the heterodimer interface. The structure resolves many of the questions about the roles of KS and CLF. Although CLF regulates chain length, it does not have an active site; KS must catalyze both chain initiation and elongation. We provide evidence that the first cyclization of the polyketide occurs within the KS-CLF tunnel. The mechanistic details of this central PKS polymerase could guide biosynthetic chemists in designing new pharmaceuticals and polymers.

Polyketides are produced by many bacteria, fungi, marine organisms and plants$^{1,2}$. Actinorhodin provides a classic example of how these secondary metabolites are used: during times of starvation, the soil bacterium *Streptomyces coelicolor* produces this antibiotic to inhibit the growth of competing organisms. The enzymes that synthesize these complex molecules are polyketide synthases (PKSs). Analogous to the closely related fatty acid synthases (FASs), they use the decarboxylation of α-carboxylated precursors, such as malonyl or methylmalonyl groups, to drive the synthesis of carbon chains.

The carbon backbone of actinorhodin is assembled through the cooperation of a malonyl-CoA:ACP transacylase (MAT, from the FAS), an acyl carrier protein (ACP; actI-ORF3) and a KS-CLF (also referred to as KS$_{\alpha}$-KS$_{\beta}$; actI-ORF1, ORF2)$^{3}$. This ‘minimal PKS’ uses 8 molecules of malonyl-CoA to generate a polyketide chain that is 16 carbons long (Fig. 1). The malonyl building blocks are selected by MAT, transferred from MAT to KS-CLF by ACP (using an 18-Å-long phosphopantetheinyl arm) and fused to the growing chain by KS-CLF. This catalytic cycle is repeated eight times. The resulting octaketide is tailored into actinorhodin by a ketoreductase (actIII), an aromatase (actVII), a cyclase (actIV) and several other enzymes (actVI, actVA, actVB). Without tailoring enzymes, the actinorhodin minimal PKS produces the shunt products, SEK4 and SEK4b (ref. 4).

Although the chemistry of polyketide synthesis seems simple, PKS enzymology is not straightforward. The role of KS-CLF in chain length regulation and cyclization has been debated since its discovery. Several minimal PKSs produce a mixture of polyketides differing in length and cyclization. Confusingly, the addition of tailoring enzymes that would not be expected to have an effect on length or cyclization can alter the polyketide profile in these respects. The regiospecificity of cyclization has not been explained, nor have the timing and specificity of ketoreduction. The individual roles of KS and CLF have also been controversial. The major issues are whether or not CLF carries out the initiation reaction, if KS has acyltransferase (AT) activity and to what extent KS and CLF regulate chain length.

To design potentially pharmaceutically important molecules using type II PKSs, we must learn the details of aromatic polyketide synthesis$^{5}$. Perhaps the biggest gap in our knowledge is how carbon skeletons are assembled by KS-CLF enzymes. Homology models of KS-CLF have been generated using the structure of a fatty acid ketosynthase, FabF, but they do not provide high-resolution information for features such as the polyketide tunnel$^{6,7}$. Therefore, we undertook a structural study of this mysterious polymerase. By purifying KS-CLF from *S. coelicolor* during the stationary phase, when the actinorhodin PKS produces polyketides, we were able to trap intermediates in the active site (stage I in Fig. 1). Crystallographic and mass-spectroscopic analysis allowed us to identify these species and learn more about the mechanisms of KS-CLF.

**RESULTS**

**KS-CLF is a heterodimer**

Ketosynthases have an alternating α-helix and β-sheet architecture (αβαβα) first observed in thiolase (Fig. 2 and Supplementary Fig. 1 online)$^{8}$. Of the many types of condensing enzymes, KS and CLF are most highly identical to FabF (37% and 28% compared to the *Escherichia coli* enzyme)$^{9}$. Like FabF, KS catalyzes malonyl decarboxylation with the help of two histidines and is acylated on a cysteine located at a nucleophilic elbow.

The 2.0-Å structure reveals that KS and CLF have evolved highly complementary contacts that bury 20.2% (6,400 Å²) of the monomeric surface area. Residues along the pseudo two-fold axis of KS-CLF, such as Tyr118 and Phe116′ (KS and CLF elements are indicated with and without ′), make tighter interactions than equivalent

---

$^{1}$Graduate Group in Biophysics, University of California San Francisco, San Francisco, California 94107-2240, USA. $^{2}$Mass Spectrometry Facility, Department of Pharmaceutical Chemistry, University of California San Francisco, San Francisco, California 94143-0446, USA. $^{3}$Departments of Chemistry, Chemical Engineering and Biochemistry, Stanford University, Stanford, California 94305, USA. $^{4}$Department of Biochemistry and Biophysics, University of California San Francisco, San Francisco, California 94107-2240, USA. Correspondence should be addressed to R.M.S. (stroud@msg.ucsf.edu).

Published online 1 August 2004; doi:10.1038/nsmb808

888 VOLUME 11 NUMBER 9 SEPTEMBER 2004 NATURE STRUCTURAL & MOLECULAR BIOLOGY

Initiation

![Diagram of Initiation](image1)

Cyclization

![Diagram of Cyclization](image2)

Elongation

![Diagram of Elongation](image3)

Tailoring

![Diagram of Tailoring](image4)

Figure 1 Octaketide production by the actinorhodin minimal PKS. Initiation: Malonyl-ACP is decarboxylated by KS to form an acetyl group that primes KS. Elongation: (I) MAT transfers malonyl groups from malonyl-CoA (not shown) to ACP, which shuttles them to KS-CLF. (II) KS catalyzes the decarboxylation of the malonyl group, and the resulting enolate condenses with the growing polyketide. (III) The chain, two carbons longer, extrudes into the polyketide tunnel and is transferred back to KS before ACP dissociates. One round of initiation and seven rounds of elongation produce an octaketide. Cyclization: at the heptaketide stage, the extended chain has reached the end of the tunnel. The final condensation causes the chain to buckle, allowing an intramolecular attack at C7. This reaction is catalyzed by a water molecule (WAT1), which acts as a general acid. The minimal PKS produces the polyketides SEK4 and SEK4b. Tailoring: The full PKS tailors two cyclized octaketides into the antibiotic, actinorhodin.

residues of homodimeric ketosynthases, which necessarily make symmetrical contacts (Fig. 2a). Large hydrophobic KS residues (Met139, Phe140, Leu143, Val144) fill the vestigial CLF tunnel, and a six-residue KS insertion (the ‘grasping loop’) tightly wraps around α3′, apparently to compensate for the binding energy sacrificed to build the polyketide tunnel at the heterodimer interface. A principal contact in FabF is made by a highly conserved H(M/I)T motif in which the methionine or isoleucine of one monomer inserts into a hydrophobic pocket in the other. Our structure reveals that this motif is used by KS but not by CLF; the actinorhodin CLF uses a tryptophan (Trp392′) from a different loop, displacing the loop that contains the H(M/I)T motif in KS (Fig. 2c).

Proposed active sites

CLF was proposed to have an active site similar to KS, capable of carrying out the first decarboxylation. The reactive cysteine substitute, Gln161′, seems to have a structural function, a salt bridge between

Polyketides extrude into an amphipathic tunnel

Whereas the tunnels in FabF are entirely hydrophobic, a subtle rearrangement in KS-CLF creates a tunnel that is amphipathic (Fig. 3a, b). Compared with FabF, α4′ is interrupted by a water molecule (WAT1) that bonds to the backbone amides of Trp135′ and Phe136′, freeing the carbonyls of Glu131′ and Ser132′ to help form the polyketide tunnel. Computational modeling studies performed with Quanta (Accelrys) demonstrate that if the polyketide is extended, the longest chain that can fit into the tunnel while covalently bound to KS is a heptaketide. Electron density of intermediates reveals the initial path taken by a growing polyketide (Fig. 4a).

![Figure 2: KS-CLF complementarity](image5)

Figure 2 KS-CLF complementarity. (a) Interactions surrounding α5 and α3′. Tyr118 and Phe116′ make asymmetrical contacts across the pseudo two-fold axis, in contrast to homodimeric ketosynthases. The ‘grasping loop’ makes important contacts with CLF. (b) The reactive cysteine, Cys169, and a gating residue that regulates chain length, Phe116′, mark the beginning and end of the ~17-Å polyketide tunnel. (c) KS uses the same H(M/I)T motif as FabF to fill a hydrophobic pocket in CLF, whereas CLF uses Trp392′ from another loop.

Figure 3 The polyketide tunnel. Front (a) and rear (b) views of the tunnel surface. The catalytic WAT1 is bound by the Trp135′ and Phe136′ amides and by the Glu131′ carbonyl. There is room for the first ring cyclization to occur. (c) Open gates (formed by smaller residues in blue) allow chains to be extended to the end of the tunnel (formed by larger residues in red), dictating the length of polyketides such as actinorhodin (Act, C₁₆), tetracenomycin (Tcm, C₂₀), the WhiE spore pigment (Whi, C₂₄) and griseorhodin (Gri, C₂₆). KS is in yellow, CLF in green.

Electrospray mass spectrometry of the KS and CLF proteins provided more evidence that intermediates were bound to KS (Fig. 4b). To determine the exact site and mass of these modifications, the complex was subjected to chemical and enzymatic digestions. Cyanogen bromide cleavage generated several peptides containing the reactive cysteine (Fig. 4c). The digest was subjected to collision-induced dissociation analysis; however, the peptides of interest were too large to permit direct observation of the modifications. Purification and subsequent trypsinolysis of the modified peptides generated smaller peptides that yielded more complete fragmentation spectra. Cys169 is acylated by at least the monoketide (42 Da) and diketide (84 Da). A 70-Da modification was also detected. The spectrum of the monoketide-peptide is presented (Fig. 4d).

## DISCUSSION

### KS-CLF complementarity

All previously solved ketosynthases are homodimeric, with each monomer capable of growing an acyl chain into a tunnel created by the dimer interface. KS-CLF is thus an example of the evolution of function, made possible through a gene duplication event; KS has been optimized for catalysis and CLF for regulating chain length.

The complementary interactions that have evolved between KS and CLF may explain the results of mix-and-match experiments in which KSs and CLFs from different PKSs were tested for their ability to work together. Most KSs are exchangeable; most CLFs are not. Only the granaticin CLF has been successfully swapped for the actinorhodin CLF²,¹⁰. A key contact in FabF is the insertion of a methionine or isoleucine from one monomer into a hydrophobic pocket created by the other. KSs rely on this H(M/I)T motif, but CLFs use other binding modes. The actinorhodin CLF uses a tryptophan from another loop to fill the hydrophobic pocket of the actinorhodin KS. The granaticin CLF also possesses this tryptophan, but most CLFs have a glycine at this position and rely on yet another mode of interaction.

### The roles of KS and CLF

It was proposed that CLF initiates the synthesis of polyketides by generating acetyl-ACP through decarboxylating malonyl-ACP¹¹. A FAS ketosynthase can be converted to a potent decarboxylase by acylating the reactive cysteine or by mutating it to a glutamine, presumably because the new carbonyl inserts into the oxyanion hole, allowing the productive binding of a malonyl group¹². The highly conserved CLF glutamine, Gln161′, does not enter the oxyanion hole. The catalytic histidines are replaced by a salt bridge, and hydrophobic residues, such as Phe235, that may initiate decarboxylation are not conserved in CLFs. Large residues occupying the proposed active site of CLF leave little room for a malonyl substrate. Because the minimal PKS can produce polyketides from malonyl-CoA, KS must carry out the initiation reaction. KSs were thought to possess AT activity because most PKS gene clusters do not encode ATs and the AT motif, ‘GHSXG’, is present in the KS sequence¹³. However, the putative serine nucleophile is buried, and no simple structural rearrangement can expose it for catalysis.

---

Chain elongation and regulation

During fatty acid synthesis, a growing chain must leave the ketosynthase after each condensation reaction so that other enzymes can reduce the β-carbonyl to a methylene group. Conversely, during polyketide synthesis, the growing polyketide must be kept in an environment in which intramolecular reactions are prevented, because the unreduced polyketide is extremely reactive and can spontaneously cyclize¹⁴. From the KS-CLF structure, it is apparent that the tunnel keeps the polyketide extended as it is growing so that the reactive ketide groups are separated. The ketide groups probably contact KS-CLF mainly as enols, because the hydrophilic side of the tunnel is created by two backbone carboxyls and the Glu131′ carboxylate. However, WAT1 could donate a hydrogen bond to a keto group. If the extrusion of the polyketide into the tunnel occurs before the acylated ACP can dissociate, the chain is acquired by the reactive cysteine, Cys169 (ref. 15). The process of extrusion is probably favored if strong interactions are created between the polyketide and the amphipathic tunnel, as is the situation of a fully reduced, but not a partially reduced, acyl chain in the hydrophobic tunnel of a FAS ketosynthase.

The longest chains known to be synthesized by KS-CLF complexes are pentadecaketides, as in the fredericamycin PKS. It is apparent that they grow into the vestigial CLF tunnel (Fig. 3c). From a sequence comparison of the known KS-CLF enzymes, there seem to be several gates, three of which were recently identified from a homology model⁶. All gates are closed in octaketide synthases. Smaller residues replace Thr112′ and Phe116′ in dekaketide synthases. In dodecaketide synthases, Trp194′ and Phe112′ are replaced by smaller residues, whereas Gly193′ and Leu143 are replaced by larger residues to form the end of the tunnel. In tridecaketide synthases, only Leu143 is replaced by a larger residue. The longest tunnels would terminate near

Figure 4 Polyketide intermediates. (a) Density from intermediates attached to Cys169 appears in the $2F_{o}-F_{c}$ electron density map ($1.0\sigma$, blue) and the omit map ($3.5\sigma$, pink). Polyketide enol hydroxyls probably contact the Glu131′ and Ser132′ backbone carbonyls and the Glu131′ carboxylate, preventing intramolecular reactions during elongation. The density at the active site is similar in each of the four heterodimers of the asymmetric unit. (b) Deconvoluted electrospray spectrum of KS showing that most enzymes are modified. Expected masses: unmodified KS = 45,063 Da, monoketide-KS = 45,105 Da. (c) MALDI-TOF mass spectrum of the cyanogen bromide digest. Peptides containing the reactive cysteine are modified by various acyl groups. The monoisotopic peaks match the expected values to within 200 p.p.m. (d) Low-energy collision-induced dissociation spectrum of the 165VSTGC(monoketide)TSGLASVGNAVR$^{181}$ peptide generated by cyanogen bromide and subsequent tryptic digestion. The spectrum contains an almost complete y-ion series, in which $y_{12}$ and smaller fragments are unmodified, and $y_{13}$ and larger fragments are shifted by 42 Da. The immonium ion of the modified cysteine, $\mathbf{C}^{*}$, and several internal fragments also indicate that the monoketide intermediate is attached to Cys169. For simplicity, only ions that help establish that the cysteine is modified are labeled. Fragments are labeled according to Biemann’s nomenclature$^{28}$.

### Cyclization within KS-CLF

Because minimal PKSs usually produce polyketides with C7-C12 or C9-C14 cyclizations instead of the many others that are possible, it is believed that KS-CLF catalyzes the first ring cyclization. In the actinorhodin KS-CLF, when the octaketide is formed it can only fill into the tunnel by buckling, possibly initiating the cyclization reaction. WAT1 lies in an oxyanion hole at the midpoint of the tunnel and may act as a general acid after an attack at the C7 carbonyl by donating a proton to the cyclized polyketide. The cyclization of the mysterious minimal PKS by-product, SEK4b, could be explained by an attack on C7 from the C11 enolate oxygen. In chalcone synthase, a related enzyme from a plant PKS, a tetra-ketide cyclizes inside the same tunnel in which it was polymerized, yielding two differently cyclized products$^{17}$.

It is clear from the buried KS-CLF tunnel that downstream enzymes, such as the ketoreductase, do not have a chance to act on the polyketide until ACP pulls it out of KS-CLF$^{18}$. However, the addition of tailoring enzymes to a minimal PKS can change the regiospecificity of cyclization and even chain length. The tetracenomycin minimal PKS chiefly produces a C7-C12 cyclized decaketide; however, it also produces trace amounts of a C9-C14 cyclized decaketide. Therefore, WAT1 may also be able to catalyze an attack at the C9 carbonyl. The addition of the tailoring enzyme, TcmN, results in the exclusive production of C9-C14 cyclized polyketides$^{19}$. TcmN may be interacting with ACP in a manner that prevents the polyketide from fully extruding into the tunnel, thereby positioning the C9 carbonyl closest to WAT1. If a tailoring enzyme prevented full polyketide extrusion, it could also cause increased chain length because an additional round of elongation might be necessary for the polyketide to reach the end of the tunnel and cyclize. When TcmN is added to the frenolicin minimal PKS, nonaketides are synthesized instead of octaketides$^{20}$. Finally, it may be the C7-C12 ring that is recognized by the ketoreductase, thereby targeting C9 for reduction.

Together with the recent structures of MAT$^{21}$, ACP$^{22}$ and ActVA-ORF6 (ref. 23) from the actinorhodin PKS, and the priming ketosynthase from the R1128 PKS$^{24}$, the structure of the actinorhodin KS-CLF takes a large step forward in revealing the molecular details of polyketide synthesis by type II PKSs. This structure suggests how PKSs can be engineered to synthesize polyketides of new chain lengths and cyclizations.

ARTICLES

Table 1 Data collection, processing and structure refinement statistics for KS-CLF

| Crystallization |  |
| --- | --- |
| Space group | R3 |
| Unit cell (Å) | $a = 262.70, c = 224.03$ |
| Asymmetric unit | 4 heterodimers |
| Wavelength (Å) | 1.127 |
| Resolution (Å) | 50–2.0 (2.07–2.00) |
| Total reflections | 3,288,849 |
| Unique reflections | 384,406 |
| Redundancy | 8.6 |
| Completeness (%)$^a$ | 98.7 (95.5) |
| $R_{\text{sym}} (\%)$$^a$ | 7.0 (30.7) |
| $\langle I / \sigma(I) \rangle$$^a$ | 9.3 (1.8) |

| Refinement |  |
| --- | --- |
| $R_{\text{cryst}} (\%)$ | 23.3 |
| $R_{\text{free}} (\%)$$^b$ | 25.2 |
| R.m.s. deviations from standard geometry |  |
| Bond (Å) | 0.007 |
| Angle (°) | 1.3 |
| Number of atoms per asymmetric unit |  |
| Protein | 24,440 |
| Water | 1,089 |
| Magnesium | 4 |
| Sodium | 4 |
| Monoketide | 12 |
| Mean B-factors (Å²) |  |
| Protein | 31 |
| Water | 33 |
| Magnesium | 44 |
| Sodium | 40 |
| Monoketide | 50 |

$^a$ Numbers in parentheses refer to the highest resolution shell. $^b R_{\text{free}}$ was calculated on 10% of data excluded before refinement.

---

**METHODS**

Expression and purification. *S. coelicolor* CH999 (ref. 25) was transformed with pSEK38 (ref. 4). Cultures were grown for 3 d at 30 °C in Stroud minimal medium (6.25% (w/v) PEG 8000, 25 mM TES, pH 7.2, 0.5% (w/v) casamino acids, 1% (w/v) glucose, 15 mM $(NH_4)_2SO_4$, 5 mM MgSO₄, 1.25 mM NaH₂PO₄, 1.25 mM K₂HPO₄, 926 nM FeCl₃, 377 nM ZnCl₂, 99 nM MnCl₂, 73 nM CuCl₂, 33 nM Na₂B₄O₇, 10 nM $(NH_4)_6Mo_7O_{24}$) and 30 μM thiostrepton. Cells were sonicated in 200 mM NaH₂PO₄, pH 7.1, 200 mM NaCl, 30% (v/v) glycerol, 2.5 mM EDTA, 2 mM DTT, 1.5 mM benzamide, 57 μM phenylmethylsulfonyl fluoride, 4.3 μM leupeptin, 2.9 μM pepstatin. After centrifugation, polyethyleneimine was added to 0.2% (w/v). Anti-FLAG tag affinity chromatography (pSEK38 encodes an N-terminal FLAG tag on CLF), using a buffer consisting of 100 mM NaH₂PO₄, pH 7.0, 150 mM NaCl, 2 mM EDTA and 2 mM DTT, followed by size-exclusion chromatography using a buffer consisting of 10 mM HEPES, pH 7.0, 150 mM NaCl and 1 mM DTT, yielded 1 mg KS-CLF per liter of culture.

Crystallization and structure determination. KS-CLF was concentrated to 6 mg ml⁻¹ and crystallized in 3 d by hanging-drop vapor diffusion at room temperature in 3.8 M sodium formate. Crystals were frozen in the nitrogen stream in mother liquor (ALS BL8.3.1). Data were integrated and scaled with DENZO and SCALEPACK²⁶. The FabF (1KAS) dimer was used as a search model for molecular replacement in CNS²⁷. Iterative refinement was done with CNS, using noncrystallographic restraints for the first few rounds because there are no major structural differences between the four heterodimers of the asymmetric unit (Table 1).

Mass spectrometry. Electrospray analysis of uncleaved KS-CLF was done on a QSTAR XL (Applied Biosystems) (Fig. 4b). KS-CLF was cleaved with 250 mM cyanogen bromide in 80% trifluoroacetic acid for 4 h at room temperature in the dark. MALDI-TOF analysis of the digest was done on a Voyager-DE STR BioSpectrometry Workstation (Applied Biosystems) (Fig. 4c). Peptides were analyzed on a QSTAR XL mass spectrometer in an information-dependent LC/MS/MS experiment (Fig. 4d). Modified peptides were purified by C₁₈ chromatography and digested with 10 μg ml⁻¹ TPCK-treated trypsin in 100 mM ammonium bicarbonate (pH 7.5) for 10 min.

Coordinates. The atomic coordinates have been deposited in the Protein Data Bank (accession code 1TQY).

Note: Supplementary information is available on the Nature Structural & Molecular Biology website.

**ACKNOWLEDGMENTS**

We thank Y. Tang and S. Kobayashi for valuable conversations about KS-CLF biochemistry and help in transforming into and purifying from *S. coelicolor*. Research was supported by US National Institutes of Health (NIH) Cancer Institute grants CA 63081 (R.M.S.) and CA 77248 (C.K.). A.T.K. also received a fellowship from the Achievement Rewards for College Scientists Foundation. D.A.M. and K.F.M. were supported by US NIH National Center for Research Resources grants RR 01614 and RR 12961 (to the UCSF MS Facility, director A.L. Burlingame).

**COMPETING INTERESTS STATEMENT**

The authors declare that they have no competing financial interests.

Received 30 December 2003; accepted 23 June 2004  
Published online at http://www.nature.com/natstructmolbiol

1. Hopwood, D.A. Genetic contributions to understanding polyketide synthases. *Chem. Rev.* **97**, 2465–2498 (1997).
2. Rawlings, B.J. Biosynthesis of polyketides (other than actinomycete macrolides). *Nat. Prod. Rep.* **16**, 425–484 (1999).
3. Dreier, J., Shah, A.N. & Khosla, C. Kinetic analysis of the actinorhodin aromatic polyketide synthase. *J. Biol. Chem.* **274**, 25108–25112 (1999).
4. Carreras, C.W. & Khosla, C. Purification and *in vitro* reconstitution of the essential protein components of an aromatic polyketide synthase. *Biochemistry* **37**, 2084–2088 (1998).
5. McDaniel, R., Ebert-Khosla, S., Hopwood, D.A. & Khosla, C. Rational design of aromatic polyketide natural products by recombinant assembly of enzymatic subunits. *Nature* **375**, 549–554 (1995).
6. Tang, Y., Tsai, S.C. & Khosla, C. Polyketide chain length control by chain length factor. *J. Am. Chem. Soc.* **125**, 12708–12709 (2003).
7. He, M., Varoglu, M. & Sherman, D.H. Structural modeling and site-directed mutagenesis of the actinorhodin β-ketoacyl-acyl carrier protein synthase. *J. Bacteriol.* **182**, 2619–2623 (2000).
8. Mathieu, M. *et al.* The 2.8 Å crystal structure of peroxisomal 3-ketoacyl-CoA thiolase of *Saccharomyces cerevisiae*: a five-layered αβαβα structure constructed from two core domains of identical topology. *Structure* **2**, 797–808 (1994).
9. Huang, W. *et al.* Crystal structure of β-ketoacyl-acyl carrier protein synthase II from *E. coli* reveals the molecular architecture of condensing enzymes. *EMBO J.* **17**, 1183–1191 (1998).
10. Sherman, D.H., Kim, E.S., Bibb, M.J. & Hopwood, D.A. Functional replacement of genes for individual polyketide synthase components in *Streptomyces coelicolor* A3(2) by heterologous genes from a different polyketide pathway. *J. Bacteriol.* **174**, 6184–6190 (1992).
11. Bisang, C. *et al.* A chain initiation factor common to both modular and aromatic polyketide synthases. *Nature* **401**, 502–505 (1999).
12. Witkowski, A., Joshi, A.K., Lindqvist, Y. & Smith, S. Conversion of a β-ketoacyl synthase to a malonyl decarboxylase by replacement of the active-site cysteine with glutamine. *Biochemistry* **38**, 11643–11650 (1999).
13. Fernandez-Moreno, M.A., Martinez, E., Boto L., Hopwood, D.A. & Malpartida, F. Nucleotide sequence and deduced functions of a set of cotranscribed genes of *Streptomyces coelicolor* A3(2) including the polyketide synthase for the antibiotic actinorhodin. *J. Biol. Chem.* **267**, 19278–19290 (1992).
14. Harris, T.M. & Harris, C.M. Biomimetic syntheses of aromatic polyketide metabolites. *Pure Appl. Chem.* **58**, 283–294 (1986).
15. Dreier, J. & Khosla, C. Mechanistic analysis of a type II polyketide synthase. Role of conserved residues in the β-ketoacyl synthase-chain length factor heterodimer. *Biochemistry* **39**, 2088–2095 (2000).
16. Tang, Y., Lee, T.S. & Khosla, C. Engineered biosynthesis of regioselectively modified aromatic polyketides using bimodular polyketide synthases. *PLoS Biol.* **2**, 227–238 (2004).
17. Ferrer, J.L., Jez, J.M., Bowman, M.E., Dixon, R.A. & Noel, J.P. Structure of chalcone synthase and the molecular basis of plant polyketide biosynthesis. *Nat. Struct. Biol.* **6**, 775–784 (1999).
18. McDaniel, R., Ebert-Khosla, S., Hopwood, D.A. & Khosla, C. Engineered biosynthesis of novel polyketides: manipulation and analysis of an aromatic polyketide synthase

with unproven catalytic specificities. J. Am. Chem. Soc. 115, 11671–11675 (1993).

19. Summers, R.G., Wendt-Pienkowski, E., Motamedi, H., & Hutchinson, C.R. The tcmVI region of the tetracenomycin C biosynthetic gene cluster of Streptomyces glaucescens encodes the tetracenomycin F1 monooxygenase, tetracenomycin F2 cyclase, and, most likely, a second cyclase. J. Bacteriol. 175, 7571–7580 (1993).

20. Kramer, P.J. *et al.* Rational design and engineered biosynthesis of a novel 18-carbon aromatic polyketide. J. Am. Chem. Soc. 119, 635–639 (1997).

21. Keatinge-Clay, A.T. *et al.* Catalysis, specificity, and ACP docking site of Streptomyces coelicolor malonyl-CoA:ACP transacylase. Structure (Camb.) 11, 147–154 (2003).

22. Crump, M.P. *et al.* Solution structure of the actinorhodin polyketide synthase acyl carrier protein from Streptomyces coelicolor A3(2). Biochemistry 36, 6000–6008 (1997).

23. Sciara, G. *et al.* The structure of ActVA-Orf6, a novel type of monooxygenase involved in actinorhodin biosynthesis. EMBO J. 22, 205–215 (2003).

24. Pan, H. *et al.* Crystal structure of the priming β-ketosynthase from the R1128 polyketide biosynthetic pathway. Structure (Camb.) 10, 1559–1568 (2002).

25. McDaniel, R., Ebert-Khosla, S., Hopwood, D.A. & Khosla, C. Engineered biosynthesis of novel polyketides. Science 262, 1546–1550 (1993).

26. Otwinowski, Z. & Minor, W. Processing of X-ray diffraction data collected in oscillation mode. Methods Enzymol. 276, 307–326 (1997).

27. Brünger, A.T. *et al.* Crystallography & NMR system: a new software suite for macromolecular structure determination. Acta Crystallogr. D Biol. Crystallogr. 54, 905–921 (1998).

28. Biemann, K. Appendix 5. Nomenclature for peptide fragment ions (positive ions). Methods Enzymol. 193, 886–887 (1990).
