
THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2004 by The American Society for Biochemistry and Molecular Biology, Inc.

Crystal Structure of Arachidonic Acid Bound to a Mutant
of Prostaglandin Endoperoxide H Synthase-1 That Forms
Predominantly 11-Hydroperoxyeicosatetraenoic Acid*

Received for publication, March 18, 2004, and in revised form, July 20, 2004
Published, JBC Papers in Press, July 30, 2004, DOI 10.1074/jbc.M403013200

Christine A. Harman‡, Caroline Jill Rieke‡§, R. Michael Garavito‡, and William L. Smith‡§¶

From the ‡Department of Biochemistry and Molecular Biology, Michigan State University, East Lansing, Michigan 48824
and the §Department of Biological Chemistry, University of Michigan Medical School, Ann Arbor, Michigan 48109

Kinetic studies and analysis of the products formed by native and mutant forms of ovine prostaglandin endoperoxide H synthase-1 (oPGHS-1) have suggested that arachidonic acid (AA) can exist in the cyclooxygenase active site of the enzyme in three different, catalytically competent conformations that lead to prostaglandin G₂ (PGG₂), 11R-hydroperoxyeicosatetraenoic acid (HPETE), and 15R,S-HPETE, respectively. We have identified an oPGHS-1 mutant (V349A/W387F) that forms predominantly 11R-HPETE. Thus, the preferred catalytically competent arrangement of AA in the cyclooxygenase site of this double mutant must be one that leads to 11-HPETE. The crystal structure of Co³⁺-protoporphyrin IX V349A/W387F oPGHS-1 in a complex with AA was determined to 3.1 Å. Significant differences are observed in the positions of atoms C-3, C-4, C-5, C-6, C-10, C-11, and C-12 of bound AA between native and V349A/W387F oPGHS-1; in comparison, the positions of the side chains of cyclooxygenase active site residues are unchanged. The structure of the double mutant presented here provides structural insight as to how Val³⁴⁹ and Trp³⁸⁷ help position C-9 and C-11 of AA so that the incipient 11-peroxyl radical intermediate is able to add to C-9 to form the 9,11 endoperoxide group of PGG₂. In the V349A/W387F oPGHS-1·AA complex the locations of C-9 and C-11 of AA with respect to one another make it difficult to form the endoperoxide group from the 11-hydroperoxyl radical. Therefore, the reaction apparently aborts yielding 11R-HPETE instead of PGG₂. In addition, the observed differences in the positions of carbon atoms of AA bound to this mutant provides indirect support for the concept that the conformer of AA shown previously to be bound within the cyclooxygenase active site of native oPGHS-1 is the one that leads to PGG₂.

---

Prostaglandin endoperoxide H synthase (PGHS)¹ is a heme-containing, bi-functional enzyme that catalyzes the conversion

* This work was supported in part by United States Department of Energy, Basic Energy Sciences, Office of Energy Research Contract W-31-109-Eng-38 and by Program Project Grant P01 GM57323 and Grants R01 HL56773 and R01 GM68848 from the National Institutes of Health. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
The atomic coordinates and structure factors (code 1U67) have been deposited in the Protein Data Bank, Research Collaboratory for Structural Bioinformatics, Rutgers University, New Brunswick, NJ (http://www.rcsb.org/).
¶ To whom correspondence should be addressed: Dept. of Biological Chemistry, University of Michigan Medical School, 5416 Medical Science Bldg. I, 1301 E. Catherine St., Ann Arbor, MI 48109-0606. Tel.: 734-647-6180; Fax: 734-764-3509; E-mail: smithww@umich.edu.
¹ The abbreviations used are: PGHS, prostaglandin endoperoxide H synthase; PG, prostaglandin; oPGHS, ovine PGHS; AA, arachidonic

of arachidonic acid (AA) to prostaglandin (PG) H₂ (Fig. 1), the immediate precursor to prostaglandins, thromboxanes, and prostacyclin. In mammalian tissues, there are two isoforms of PGHS designated PGHS-1 (or cyclooxygenase-1 (COX-1)) and PGHS-2 (or COX-2) (1, 2). PGHS-1 is constitutively expressed and produces prostaglandins in response to hormone stimulation mainly for regulating housekeeping functions. PGHS-2, the inducible form, is expressed in response to mitogens, growth factors, tumor promoters, and/or cytokines and produces prostaglandins associated with pain, fever, and inflammation (1, 2).

Despite the differences in patterns of expression, PGHS-1 and PGHS-2 are quite similar both structurally (3–5) and mechanistically (6). Both isoforms function as dimers with each monomer having an epidermal growth factor domain, a membrane-binding domain, and a large catalytic domain. The catalytic domain of PGHS possesses two distinct but functionally connected active sites (3–5, 7). These two sites include the COX site, which exists as a long hydrophobic channel within the core of the protein and binds AA (8) and nonsteroidal anti-inflammatory drugs (3–5), and a more solvent-exposed peroxidase site containing a heme moiety, which is involved in reducing PGG₂ formed at the COX site to PGH₂, the final enzymatic product.

The crystal structure of native Co³⁺-protoporphyrin IX oPGHS-1 in a complex with AA shows that AA binds within the COX active site having the carboxylate group anchored at the mouth of the hydrophobic channel via electrostatic interactions with Arg¹²⁰ and Tyr³⁵⁵ (8).² The carbons of the AA chain weave into the COX active site, making forty-eight hydrophobic contacts with the residues lining this channel. Based on the crystal structure and knowledge of the stereochemical requirements of the COX reaction (9), a structural sequence of catalytic events was proposed (8). The COX reaction begins with the abstraction of the 13proS hydrogen of AA by a tyrosyl radical formed at Tyr³⁸⁵, creating an arachidonyl radical centered at C-13 (6) (Fig. 1). Following a rearrangement centering the radical on C-11, an attack of molecular oxygen occurs to form an 11R-hydroperoxyl radical (Fig. 1). At this point it is proposed that rotation about the C-10–C-11 bond moves the 11R-hydroperoxyl radical in close proximity to C-9 to form the 9,11-endoperoxide group; concurrently, rotation about the C-10–C-11 bond additionally brings C-8 near to C-12 to form the cyclopentane ring. Repositioning of C-12 closer to C-8 for ring formation also

acid; 11-HETE, 11-hydroxy-5c,8c,12t,14c-eicosatetraenoic acid; 15-HETE, 15-hydroxy-5c,8c,11c,13t-eicosatetraenoic acid; 11-HPETE, 11-hydroperoxy-5c,8c,11c,13t eicosatetraenoic acid; 15-HPETE, 15-hydroperoxy-5c,8c,11c,13t eicosatetraenoic acid; COX, cyclooxygenase; PBS, phosphate-buffered saline; β-OG, n-octyl-β-D-glucopyranoside; Ni-NTA, nickel-nitrilotriacetic acid.

² The numbering of residues in oPGHS-1 is based on numbering the N-terminal methionine of the signal peptide as residue number 1.

This paper is available on line at http://www.jbc.org 42929

Crystal Structure of AA bound to Mutant PGHS-1

![Reaction pathways for the formation of PGG₂ and 11R-HPETE](https://i.imgur.com/yourimageurl.png)

FIG. 1. Reaction pathways for the formation of PGG₂ and 11R-HPETE.

repositions atoms C-13 to C-20. The result is an optimal positioning of C-15 for a second attack by molecular oxygen. Additionally, with the 15-hydroperoxyl radical now positioned in proximity to Tyr³⁸⁵, Tyr³⁸⁵ is poised to donate a hydrogen to form the product of the COX site, PGG₂, thus simultaneously reforming the Tyr³⁸⁵ radical and regenerating the enzyme for another round of catalysis.

Both PGHS isoforms exhibit some lipoxygenase activity. Small amounts of 11-hydroperoxy-8Z,12E,14Z,-eicosatrienoic acid and 15-hydroperoxy-8Z,11Z,13E-eicosatrienoic are formed from dihomo-γ-linolenic acid (9), and the corresponding 11- and 15-HPETEs are produced from AA (10–12). Thus, the native enzyme is not 100% efficient in converting AA to PGG₂. Additionally, with native enzyme the \( K_m \) values for AA in forming 11-HPETE, 15-HPETEs, and PGG₂ are different for each product (11). This suggests that AA can exist in the COX site in at least three catalytically competent arrangements at the time of hydrogen abstraction to yield either PGG₂, 11R-HETE, or 15R,S-HETE (11).

Extensive kinetic and product analysis of numerous mutants of the COX active site have identified several residues that influence the relative proportions of PGG₂ and monohydroperoxide products formed from AA (11–13). Two single amino acid substitutions, V349A and W387F, were of particular interest to us because relatively more 11R-HPETE is formed (35–50% of total products) without substantially altering the kinetics of oxygenation (12). Because each single mutation favors the formation of 11R-HPETE ~50% of the time, we constructed and analyzed a V349A/W387F oPGHS-1 mutant. As described in this report, this double mutant produces predominantly 11R-HPETE. To investigate why this double mutant produces predominantly 11R-HPETE from AA, we determined the crystal structure of Co³⁺-protoporphyrin V349A/W387F oPGHS-1 with AA bound in the COX active site. Significant differences in the conformations of AA bound to native oPGHS-1 versus V349A/

W387F oPGHS-1 mutant enzyme were observed. Our results provide insight into how Val³⁴⁹ and Trp³⁸⁷ influence the conformation of AA and how these residues are involved in directing the formation of the endoperoxide group.

EXPERIMENTAL PROCEDURES

Materials—Arachidonic acid, 11R-HETE, and 15S-HETE were purchased from Cayman Chemical Co. (Ann Arbor, MI). [1-¹⁴C]Arachidonic acid (40–60 mCi/mmol) was from PerkinElmer Life Sciences. Sf21 insect cells and 100× antibiotic-antimycotic were purchased from Invitrogen. Fetal bovine serum and HyQ-SFX insect medium used for cell growth and protein expression were purchased from HyClone (Logan, UT). Ni-NTA superflow resin was purchased from Qiagen, Tween 20 used for solubilization and purification was from Pierce, and *n*-octyl-β-D-glucopyranoside (β-OG) used for crystallization was from Anatrace (Maumee, OH). Flurbiprofen and hemin were purchased from Sigma, and Co³⁺-protoporphyrin IX was from Porphyrin Products (Provo, UT). Oligonucleotides used as primers for mutagenesis and for sequencing were prepared by the Michigan State University Macromolecular Structure and Sequencing Facility.

Preparation of V349A/W387F oPGHS-1—Site-directed mutants were prepared with a pFastbac vector containing a cDNA encoding native oPGHS-1 with a hexahistidine tag (14) and mutagenic oligonucleotides using the Stratagene QuikChange kit (Stratagene, La Jolla, CA). Mutations were confirmed with sequence analysis performed by the DNA Sequencing Facility at Michigan State University.

Generation of Recombinant Baculovirus and Expression of oPGHS-1 Mutants—DH10Bac cells were transformed with pFastbac containing cDNA for either V349A oPGHS-1 or V349A/W387F oPGHS-1 to obtain baculoviral DNA, which was then isolated and used to transfect sf21 insect cells to produce recombinant baculovirus. The recombinant baculovirus was amplified, and supernatants of amplifications were used as inoculum stocks. Sf21 insect cells grown in a 17-liter bioreactor to an approximate density of 1.7–2.0 × 10⁶ cells/ml in medium with 1% antibiotic and antimycotic were infected with recombinant baculovirus at a multiplicity of infection of 0.01 at 27 °C. Three to four days after infection, the cells were harvested by centrifugation.

Purification oPGHS-1 Mutants from Sf21 Insect Cells—Cell pellets from three bioreactors were resuspended in 1× phosphate-buffered saline (PBS), pH 8.0, containing 20 mM imidazole and were lysed by

Crystal Structure of AA bound to Mutant PGHS-1

**TABLE I**

Data collection and refinement statistics for the Co³⁺-heme V349A/W387F oPGHS-1:AA cocrystal structure

| Item | Model building | Simulated annealing |
|------|----------------|---------------------|
| Space group | P6₅₂₂ | P6₅₂₂ |
| $a$ (Å) | 181.88 | 181.88 |
| $b$ (Å) | 181.88 | 181.88 |
| $c$ (Å) | 103.37 | 103.37 |
| Resolution (Å) | 20–3.1 | 20–3.1 |
| No. of unique reflections ($F > 0\sigma$) | 20640 | 20640 |
| Completeness (%) | 99.1 (99.8)$^a$ | 99.1 (99.8)$^a$ |
| $R_{\text{merge}}$, all data (%; $I > 0\sigma$) | 7.6 (32.0)$^a$ | 7.6 (32.0)$^a$ |
| No. of atoms in refinement | 4573 | 4571 |
| $R$ factor (%; reflections > 2$\sigma$ on $F$) | 23.3 | 23.8 |
| Free $R$ factor (%; reflections > 2$\sigma$ on $F$) | 31.2 | 31.1 |
| Mean positional error (Å) | 0.43 | 0.43 |
| r.m.s.d in bond length (Å)$^b$ | 0.009 | 0.009 |
| r.m.s.d in bond angle (°) | 1.6 | 1.6 |
| r.m.s.d in improper angle (°) | 1.1 | 0.92 |
| r.m.s.d in dihedral angle (°) | 22.6 | 22.3 |
| Average $B$ factor |  |  |
| All atoms | 44.9 | 41.1 |
| Protein | 40.1 | 40.2 |
| Arachidonic Acid | 55.6 | 56.7 |

$a$ The values in parentheses represent the values in the last shell (3.2–3.1 Å).

$b$ r.m.s.d., root mean square deviation

42932

Crystal Structure of AA bound to Mutant PGHS-1

| Native | V349A | V349A/W387F |
|--------|-------|--------------|
| flurbiprofen | - | + | - | + |

AA →  
15-HETE →  
11-HETE →  
HHT →  

Prostaglandin Products

FIG. 2. Thin layer chromatogram comparing the products formed from [1-¹⁴C]AA by native, V349A oPGHS-1, or V349A/W387F oPGHS-1. [1-¹⁴C]AA was incubated for 1 min at 37 °C in the presence or absence of the cyclooxygenase inhibitor flurbiprofen with 150 cyclooxygenase units of native, V349A, or V349A/W387F oPGHS-1 purified by chromatography on Ni-NTA. The samples were extracted, and the radioactive products were separated by thin layer chromatography and quantified by liquid scintillation counting as described under “Experimental Procedures.” The locations of AA, monohydroxy fatty acid, and PG standards are as noted. HHT, 12-hydroxy-5c,8c,10t-heptadecatrienoic acid.

AA could exist and be represented by the density, given the broad extent of the electron density observed at this resolution. A parallel series of simulated annealing refinements were performed, using the observed structure factors and the V349A/W387F oPGHS-1 coordinates. Each series of simulated annealing refinements were done four times to generate a family of structures from each starting point. The starting conformation of bound AA differed, however, between the two parallel series of refinement. In the first series, the AA conformer found in native oPGHS-1 (8), was used, whereas in the second series, the AA conformer derived from the refined V349A/W387F oPGHS-1 structure was used. If the contributions from the x-ray data for the mutant were significant, it was expected that the AA conformer seen in the mutant would be recovered in the first case and retained in the second. If not, both series of refinements would yield a wide range of unrelated AA conformers. As expected, the simulated annealing refinement starting from the AA conformer derived from the refined V349A/W387F oPGHS-1 structure generated a family of AA conformations that were very similar to the starting conformer. In contrast, all four AA conformers generated by using the native AA conformer as the starting point for refinement looked remarkably similar to the AA conformation found in the V349A/W387F mutant. Of these four conformers, one fit the observed electron density quite well and was used for further structure refinement. Statistics of the refinement of the final model using the conformer from simulated annealing are shown in Table I.

RESULTS

Comparison of Products Formed from Arachidonic Acid by Native and Mutant oPGHS-1—There are considerable differences in the proportions of oxygenated products formed from [1-¹⁴C]arachidonic acid by native oPGHS-1, V349A oPGHS-1, and V349A/W387F oPGHS-1 (Fig. 2). Native enzyme produces predominantly products derived from PGG₂ (~95% including 12-hydroxy-5c,8c,10t-heptadecatrienoic acid and various prostaglandins) and small amounts of 11-HETE (~2.5%) derived from 11-HPETE and 15-HETE (~2.5%) from 15-HPETE. In comparison, the V349/W387F double mutant produces predominantly 11-HETE (>84%) and correspondingly less PGG₂ de-

Crystal Structure of AA bound to Mutant PGHS-1

FIG. 3. Stereo view of AA bound within the active site of V349A/W387F oPGHS-1. The simulated annealing omit map $F_{o} - F_{c}$ density contoured at 4.0 $\sigma$ is shown in green. Carbon atoms of AA, which were originally built into the electron density, are shown in pink (atoms C-1–C-12). Carbon atoms of AA obtained from simulated annealing are shown in blue. Side chains of various amino acids that line the COX active site and contact the substrate are shown in a color scheme where carbons are gray, oxygens are red, nitrogens are dark blue, and sulfur is yellow. All of the figures were created using the program SETOR (22).

similar particularly when comparing the carbon positions of the carboxylate group up to C-7.

The family of conformers, including the conformer shown in Fig. 3 obtained from the simulated annealing experiments described under “Experimental Procedures,” is shown in Fig. 4. Aside from minor differences in atom positions, the four conformations obtained from the simulated annealing refinement starting from the AA bound to native enzyme are all markedly similar to that obtained by model building (Fig. 4A). The simulated annealing experiments show that the carbon chain weaves up into the active site with the same trajectory for all eight conformers. The results of simulated annealing tests strongly suggest that the x-ray data contain a sufficient amount of information to derive a unique family of AA conformers that differ in conformation from that seen in the native oPGHS-1·AA complex.

The conformers in Fig. 4A are shown with their $\omega$ ends (C-13–C-20). Although there is no observable density for this portion of the substrate, the positioning of these atoms from simulated annealing refinement was entirely guided by the nonbonded energy terms of the molecular dynamics. Despite the lack of experimental data to validate the positions of these atoms, the conformation of the AA in this region is constrained by energy-minimized, nonbonded contacts. Thus, the $\omega$ ends of the AA conformers are in stereochemically reasonable conformations. While having a limited level of confidence, it is interesting to note that the positions of the proS hydrogens are generally found to cluster in a group near to that of AA found in native enzyme (Fig. 4A).

Structural Comparison of Arachidonic Acid Bound to Native and V349A/W387F oPGHS-1—To verify that the two mutations V349A/W387F did not alter the overall integrity of the active site, all atoms of 17 side chains (excluding residues 349 and 387) within the COX active site of native and double mutant were superimposed giving an root mean square deviation of 0.4 Å. This indicates that the positions of the active site residues in the crystal structure of V349A/W387F oPGHS-1 were not significantly perturbed by the presence of the two mutations. The overall binding of AA within the COX active sites of both the native and the double mutant is similar with respect to many of the key ionic and hydrophobic interactions (Table II).

Although the structure of AA bound to the V349A/W387F mutant is grossly similar to that of AA bound to native enzyme, there are several significant differences. Atoms C-3–C-6 of AA are shifted into the space created by the V349A substitution

FIG. 4. Arachidonic acid conformers obtained from simulated annealing. A, family of conformers obtained from simulating annealing originating from the starting conformation of AA bound to native oPGHS-1 in the V349A/W387F protein. The starting conformation of AA used in simulating annealing, shown in orange, was used to derive the family of conformers shown in blue. For comparison, carbon atoms of AA, which was originally built into the electron density, are shown in pink. Oxygens of the carboxylate are colored red. Modeled proS hydrogens are shown in purple. B, family of conformers obtained from simulated annealing originating from the starting conformation of AA originally built into observed electron density in the V349A/W387F protein. The family of simulated annealing conformers is shown in yellow, the starting conformation used for simulated annealing shown is in pink, and for comparison the conformation of AA bound in native oPGHS-1 is shown in orange.

(Fig. 5). In the native structure, Val³⁴⁹ contacts C-3 and C-4 (Table II). In the mutant, these interactions are lost, and C-3 and C-4 rotate and move into the space created by this mutation. Rotations in this portion of the AA molecule result in large shifts of more rigid regions of AA as evidenced by the significant downward shift of the C-5–C-6 double bond (Fig. 5). The differences in the positions of carbons between AA in the native enzyme versus V349/W387F oPGHS-1 structures are as great as 3.0 Å when taking into account coordinate error of 0.43 Å.

42934 Crystal Structure of AA bound to Mutant PGHS-1

TABLE II

Distances between selected side chain atoms of COX active site amino acids and carbons of AA bound to native and V349A/W387F oPGHS-1

The distances listed between the side chain atoms and atoms of AA were abstracted from the native oPGHS-1·AA crystal structure (Protein Data Bank entry 1DIY) (8) or from the distances observed between AA bound to V349A/W387F oPGHS-1 as reported here (Protein Data Bank entry 1U67). NA, not applicable. The mean coordinate errors of the two structures are both about 0.4 Å.

| Atom of AA | Native oPGHS-1 (residue/atom) | Distance (Å) | V349A/W387F oPGHS-1 (residue/atom) | Distance (Å) |
|------------|-------------------------------|--------------|-------------------------------------|--------------|
| C-3        | Val<sup>349</sup>/CG1         | 3.4          | Ala<sup>349</sup>/CG1               | >4           |
| C-4        | Val<sup>349</sup>/CG1         | 3.1          | Ala<sup>349</sup>/CB                | >4           |
| C-5        | Ile<sup>523</sup>/CG2         | 3.3          | Ile<sup>523</sup>/CG2               | >4           |
| C-5        | Val<sup>349</sup>/CB          | >4           | Ala<sup>349</sup>/CB                | 3.0          |
| C-6        | Ile<sup>523</sup>/CG2         | 3.4          | Ile<sup>523</sup>/CG2               | >4           |
| C-6        | Val<sup>349</sup>/CB          | >4           | A349CB                             | 3.8          |
| C-7        | Leu<sup>352</sup>/CD2         | 3.9          | Leu<sup>352</sup>/CG2               | >4           |
| C-11       | Leu<sup>352</sup>/CD2         | 3.4          | Leu<sup>352</sup>/CD2               | >4           |
| C-11       | Trp<sup>387</sup>/CZ2         | 4.0          | NA                                 | NA           |
| C-11       | Trp<sup>387</sup>/CH2         | 3.4          | NA                                 | NA           |
| C-11       | NA                           | NA           | Phe<sup>387</sup>/CZ                | 3.7          |
| C-11       | Leu<sup>352</sup>/CD2         | 3.4          | Leu<sup>352</sup>/CD2               | 3.8          |
| C-12       | Leu<sup>352</sup>/CD2         | 3.9          | Leu<sup>352</sup>/CD2               | >4           |
| C-12       | Tyr<sup>348</sup>/CEZ         | 3.2          | Tyr<sup>348</sup>/CEZ               | >4           |
| C-12       | Trp<sup>387</sup>/CH2         | 3.7          | NA                                 | NA           |
| C-12       | NA                           | NA           | Phe<sup>387</sup>/CZ                | 3.7          |
| C-12       | NA                           | NA           | Phe<sup>387</sup>/CE2               | 3.9          |

FIG. 5. Comparison of AA bound to native and V349A/W387F oPGHS-1. A, stereo view of the superposition of AA bound to the V349A/W387F oPGHS-1 COX site (C-1–C-12 colored blue with the ω end portion ((C-13–C-20) obtained from simulated annealing; colored purple) and AA bound to the native oPGHS-1 COX site (colored orange). Atoms of amino acid side chains are shown using a color scheme in which carbons are gray, nitrogens are blue, oxygens are red, and sulfur is yellow. For clarity only residues Trp<sup>387</sup> and Val<sup>349</sup> (both colored orange) of the COX active site of native PGHS-1 are shown. These latter amino acids were mutated to Phe<sup>387</sup> and Ala<sup>349</sup>, respectively, which are shown in gray. B, a view of the COX active site rotated 180° with the same color scheme as in A.

There is also a significant difference in the location of the C-11–C-12 double bond of AA, which is moved via rotation about the C-9–C-10 bond toward the space created by the W387F mutation. In the native structure, Trp<sup>387</sup> makes two contacts with C-11 and one with C-12 (Table II). The loss of these contacts in the double mutant contributes to displacement of the C-11–C-12 bond.

DISCUSSION

Native PGHS-1 forms small amounts of 11R-HPETE and 15R,S-HPETE in addition to the major product PGG<sub>2</sub> (11, 12, 21). The monohydroperoxides have little or no biological activity and are considered to be abortive products that result from a failure of the enzyme to form the endoperoxide. In earlier studies, we provided kinetic evidence that the nature of the oxygenated product formed by native PGHS-1 is determined by the conformation of AA when the rate determining step in cyclooxygenase catalysis, abstraction of the 13proS hydrogen (9), occurs (11, 12). We performed the studies reported here as a structural test of this concept. We developed a V349A/W387F mutant of PGHS-1 that forms primarily 11-HPETE, determined the structure of AA in the COX active site of the mutant, and finally compared the structures of AA in the native and mutant oPGHS-1.

Previous studies identified Val<sup>349</sup> and Trp<sup>387</sup> as playing par-

particularly significant roles in guiding formation of the 9,11-endoperoxide group (11–13). Substitutions at either site yield mutant enzymes that produce 30–55% 11R-HPETE (12). The fact that the V349A/W387F mutant described here forms mainly 11-HPETE suggests that in native PGHS-1 Val<sup>349</sup> and Trp<sup>387</sup> function coordinately to facilitate PGG<sub>2</sub> formation.

The structure of AA in the cyclooxygenase site of V349A/ W387 oPGHS-1 was observed to be significantly different from that of AA in native PGHS-1. Both AA structures appear to be catalytically competent in that the distances between the phenolic oxygen of Tyr<sup>385</sup>, and the 13proS hydrogen of AA are estimated to be within about 3 Å in both structures; however, it should be noted that the 13-proS hydrogen modeled into the AA structure in V349A/W387F oPGHS-1 is more distant than that in the native oPGHS-1·AA complex. Because native and V349A/ W387F oPGHS-1 form primarily PGG<sub>2</sub> and 11R-HPETE, respectively, the results of our structural studies are consistent with the overall model that the product formed by PGHS-1 is determined by the structure of AA in the cyclooxygenase site when the rate-limiting step in the reaction occurs.

Comparisons of the AA structures observed in the native and mutant enzymes and modeling the structure of a putative 11-hydroperoxyl radical into the COX sites suggest how Trp<sup>387</sup> and Val<sup>349</sup> might participate in forming the endoperoxide group. In the native enzyme there are interactions between Trp<sup>387</sup> and both C-11 and C-12 that are eliminated when Trp<sup>387</sup> is mutated to phenylalanine. The interactions between Trp<sup>387</sup> and C-11 and C-12 may (a) constrain the orientation of the C-11–C-12 bond, thereby positioning C-11 optimally with respect to C-9, and (b) after oxygen insertion at C-11, guide rotations of the 11R-hydroperoxyl group and the C-10–C-11 bond to move the hydroperoxyl group toward C-9 for an R side attack. Mutation of Trp<sup>387</sup> to phenylalanine eliminates the steric interactions involving C-11 and C-12 that direct formation of the endoperoxide group. We suggest that with the mutant enzyme, changes in the rotational freedom about both the C-10–C-11 bond and the bond between C-11 and the peroxyl oxygen allow the 11-hydroperoxyl intermediate to assume unproductive conformations, where the attack by the 11-hydroperoxyl group on C-9 is unfavorable and no endoperoxide can be thus formed.

The V349A mutation further contributes to the altered orientation of the C-8–C-13 portion of AA in the V349A/W387F mutant. Past studies (11–13, 19, 20) have documented the influence of Val<sup>349</sup> on product formation and the stereochemistry of O<sub>2</sub> insertion. The role of Val<sup>349</sup> in supporting formation of the endoperoxide group is interesting because the effect is exerted over a relatively long distance. In native oPGHS-1, Val<sup>349</sup> protrudes into the COX channel, and the CG1 atom of Val<sup>349</sup> contacts C-3 and C-4 of AA (Table II), restricting movement of the flexible C-1–C-5 region at the carboxyl end of AA (8, 12). When Val<sup>349</sup> is mutated to alanine, C-3 and C-4 of AA rotate into the space created by the mutation, and the C-5–C-6 double bond moves into the newly available space. These results are consistent with the idea that in the native enzyme, Val<sup>349</sup> acts as a structural guide to position the carboxyl end of AA (12). The shifts in the positions of atoms C-3–C-6 of AA in

the V349A/W387F result in a significant change in the final orientation of the C-11–C-12 bond that, in turn, permits movement of atoms C-9–C-12 of AA into the extra space created by the W387F mutation (Fig. 5). Thus, in the native enzyme the positioning the carboxyl end of AA by Val<sup>349</sup> must facilitate proper positioning of the atoms C-8–C-13 for hydrogen abstraction and endoperoxide formation.

In summary, residues Val<sup>349</sup> and Trp<sup>387</sup> in native oPGHS-1 are crucial for ensuring that AA adopts the proper conformation so that the reaction proceeds on course to produce PGG<sub>2</sub> (12). We have found that V349A/W387F human PGHS-2 also produces predominantly 11-HPETE, suggesting that these two residues play similar roles in PGHS-2.<sup>3</sup> The structural evidence provided by our present study supports the hypothesis that the nature of the products formed in the COX reaction depends on the conformation of AA at the instant of abstraction of the 13proS hydrogen. The conformation of AA in the COX site of native oPGHS-1 differs significantly from the conformation of AA in the COX site of V349A/W387F oPGHS-1. This additional, albeit indirect evidence supports the conclusion that the conformation of AA observed in the native enzyme (8, 12) is that which leads to PGG<sub>2</sub> formation.

---

**REFERENCES**

1. Smith, W. L., DeWitt, D. L., and Garavito, R. M. (2000) *Annu. Rev. Biochem.* **69**, 149–182
2. Marnett, L., and DuBois, R. (2002) *Annu. Rev. Pharmacol. Toxicol.* **42**, 55–80
3. Picot, D., Loll, P. J., and Garavito, M. (1994) *Nature* **367**, 243–249
4. Luong, C., Miler, A., Barnett, J., Chow, J., Ramesha, C., and Browner, M. F. (1996) *Nat. Struct. Biol.* **3**, 927–933
5. Kurumbail, R. G., Stevens, A. M., Gierse, J. K., McDonald, J. J., Stegeman, R. A., Pak, J. Y., Gildehaus, D., Miyashiro, J. M., Penning, T. D., Seibert, K., Isakson, P. C., and Stallings, W. C. (1996) *Nature* **384**, 644–648
6. van der Donk, W. A., Tsai, A.-L., and Kulmacz, R. J. (2002) *Biochemistry* **41**, 15451–15458
7. Marshall, P. J., and Kulmacz, R. J. (1988) *Arch. Biochem. Biophys.* **266**, 162–170
8. Malkowski, M. G., Ginell, S., Smith, W. L., and Garavito, R. M. (2000) *Science* **289**, 1933–1937
9. Hamberg, M., and Samuelsson, B. (1967) *J. Biol. Chem.* **242**, 5336–5343
10. Lecomte, M., Laneuville, O., Ji, C., DeWitt, D. L., and Smith, W. L. (1994) *J. Biol. Chem.* **269**, 13207–13215
11. Thuresson, E. D., Lakkides, K. M., and Smith, W. L. (2000) *J. Biol. Chem.* **275**, 8501–8507
12. Thuresson, E. D., Lakkides, K. M., Rieke, C. J., Sun, Y., Wingerd, B. A., Micielli, R., Mulichak, A. M., Malkowski, M. G., Garavito, R. M., and Smith, W. L. (2001) *J. Biol. Chem.* **276**, 10347–10357
13. Schneider, C., Boeglin, W. E., Prusakiewicz, J. J., Rowlinson, S. W., Marnett, L. J., Samel, N., and Brash, A. R. (2002) *J. Biol. Chem.* **277**, 13354–13360
14. Spencer, A. G., Thuresson, E. A., Otto, J. C., Song, I., Smith, T., DeWitt, D. L., Garavito, R. M., and Smith, W. L. (1999) *J. Biol. Chem.* **274**, 32936–32942
15. Otwinowski, Z., and Minor, W. (1997) *Methods Enzymol.* **276**, 307–322
16. Brünger, A. T., Adams, P. D., Clore, G. M., DeLano, W. L., Gros, P., Grosse-Kunstleve, R. W., Jiang, J. S., Kuszewski, J., Nilges, M., Pannu, N. S., Read, R. J., Rice, L. M., Simonson, T., and Warren, G. L. (1998) *Acta Crystallogr. Sect. D Biol. Crystallogr.* **54**, 905–921
17. Sack, J. S. (1988) *J. Mol. Graphics* **6**, 224–225
18. Laskowski, R. A., MacArthur, M. W., Moss, D. S., and Thornton, J. M. (1993) *J. Appl. Crystallogr.* **26**, 283–291
19. Thuresson, E. D., Malkowski, M. G., Lakkides, K. M., Rieke, C. J., Mulichak, A. M., Ginell, S. L., Garavito, R. M., and Smith, W. L. (2001) *J. Biol. Chem.* **276**, 10358–10365
20. Malkowski, M. G., Thuresson, E. D., Lakkides, K. M., Rieke, C. J., Micielli, R., Smith, W. L., and Garavito, R. M. (2001) *J. Biol. Chem.* **276**, 37547–37555
21. Hamberg, M., and Samuelsson, B. (1967) *J. Biol. Chem.* **242**, 5344–5354
22. Evans, S. V. (1993) *J. Mol. Graph.* **11**, 127–138

---

<sup>3</sup> C. A. Harman, C. J. Rieke, R. M. Garavito, and W. L. Smith, unpublished data.

Enzyme Catalysis and Regulation:  
Crystal Structure of Arachidonic Acid  
Bound to a Mutant of Prostaglandin  
Endoperoxide H Synthase-1 That Forms  
Predominantly  
11-Hydroperoxyeicosatetraenoic Acid  

Christine A. Harman, Caroline Jill Rieke, R.  
Michael Garavito and William L. Smith  
J. Biol. Chem. 2004, 279:42929-42935.  
doi: 10.1074/jbc.M403013200 originally published online July 30, 2004  

Access the most updated version of this article at doi: [10.1074/jbc.M403013200](http://dx.doi.org/10.1074/jbc.M403013200)  

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinitysites).  

Alerts:  
- When this article is cited  
- When a correction for this article is posted  

Click here to choose from all of JBC's e-mail alerts  

This article cites 22 references, 9 of which can be accessed free at  
[http://www.jbc.org/content/279/41/42929.full.html#ref-list-1](http://www.jbc.org/content/279/41/42929.full.html#ref-list-1)
