
THE JOURNAL OF BIOLOGICAL CHEMISTRY

Vol. 279, No. 43, Issue of October 22, pp. 45185–45193, 2004
Printed in U.S.A.

# Structural Analysis of the Sulfotransferase (3-O-Sulfotransferase Isoform 3) Involved in the Biosynthesis of an Entry Receptor for Herpes Simplex Virus 1*

Received for publication, May 5, 2004, and in revised form, August 3, 2004
Published, JBC Papers in Press, August 10, 2004, DOI 10.1074/jbc.M405013200

Andrea F. Moon‡, Suzanne C. Edavettal§, Joe M. Krahn‡, Eva M. Munoz∥, Masahiko Negishi∥,
Robert J. Linhardt∥, Jian Liu§**, and Lars C. Pedersen‡

From the Laboratories of ‡Structural Biology and ||Reproductive and Developmental Toxicology, NIEHS, National Institutes of Health, Research Triangle Park, North Carolina 27709, the §Division of Medicinal Chemistry and Natural Products, School of Pharmacy, University of North Carolina, Chapel Hill, North Carolina 27599, and the ¶Department of Chemistry and Chemical Biology, Rensselaer Polytechnic Institute, Troy, New York 12180

Heparan sulfate (HS) plays essential roles in assisting herpes simplex virus infection and other biological processes. The biosynthesis of HS includes numerous specialized sulfotransferases that generate a variety of sulfated saccharide sequences, conferring the selectivity of biological functions of HS. We report a structural study of human HS 3-O-sulfotransferase isoform 3 (3-OST-3), a key sulfotransferase that transfers a sulfuryl group to a specific glucosamine in HS generating an entry receptor for herpes simplex virus 1. We have obtained the crystal structure of 3-OST-3 at 1.95 Å in a ternary complex with 3′-phosphoadenosine 5′-phosphate and a tetrasaccharide substrate. Mutational analyses were also performed on the residues involved in the binding of the substrate. Residues Gln<sup>255</sup> and Lys<sup>368</sup> are essential for the sulfotransferase activity and lie within hydrogen bonding distances to the carboxyl and sulfo groups of the uronic acid unit. These residues participate in the substrate recognition of 3-OST-3. This structure provides atomic level evidence for delineating the substrate recognition and catalytic mechanism for 3-OST-3.

by various modifications in the Golgi apparatus (3). These modifications include *N*-deacetylation and *N*-sulfonation of glucosamine, *C*<sub>5</sub> epimerization of glucuronic acid to form iduronic acid units, 2-*O*-sulfonation of iduronic and glucuronic acid units, and 6-*O*-sulfonation and 3-*O*-sulfonation of glucosamine units. The specific sulfated saccharide sequences of HS determine its biological function (4).

Infections caused by herpes simplex virus type 1 (HSV-1) are highly prevalent in human and result in localized mucocutaneous lesions and, in rare cases, encephalitis (5). HS plays critical roles in assisting HSV-1 infections in both viral attachment and entry steps (6). In the attachment step, HSV-1 binds to the cells through the interactions of envelope glycoproteins gC or/and gB with cell surface HS (7). Viral entry requires the interaction of a third viral glycoprotein, gD, with a specific cell surface entry receptor to induce fusion of the viral envelope with the cell membrane (in the presence of gB, gH, and gL) to establish an infection (8). The 3-*O*-sulfated HS, representing the culmination of a series of specific sulfated saccharide sequences, binds to gD and serves as an entry receptor of HSV-1 (9, 10). The 3-*O*-sulfated HS is synthesized by heparan sulfate 3-*O*-sulfotransferase isoform 3 (3-OST-3) and by 3-OST-5 (9, 10). A gD-binding HS octasaccharide was isolated and characterized, confirming that HSV-1 utilizes a unique HS sequence for its entry (11). Understanding the biosynthesis of 3-*O*-sulfated HS could potentially lead to a new strategy for developing therapeutic agents against HSV infection.

HS 3-*O*-sulfotransferase is present in at least six different isoforms with unique expression patterns in human tissues (10, 12). The amino acid sequences of the different isoforms have greater than 60% homology in the sulfotransferase domains (12). These different 3-OST isoforms transfer the sulfuryl group from 3′-phosphoadenosine 5′-phosphosulfate (PAPS) to the 3-OH position of glucosamine units residing within the context of different saccharide sequences. As a result, the HS products generated by these different isoforms exhibit unique and distinctive biological activities (Fig. 1A). It is known that the HS modified by 3-OST-1 and 3-OST-5 displays anticoagulant activity. The HS modified by 3-OST-3 and 3-OST-5 serves as an entry receptor for HSV-1 (9, 10, 13). The molecular mechanism used by 3-OSTs to distinguish substrates with unique saccharide sequences remains largely unknown.

Previously, x-ray crystal structures have been obtained for the sulfotransferase domains of human *N*-deacetylase/*N*-sulfotransferase (NST-1) and mouse 3-OST-1 in the complex with 3′-phosphoadenosine 5′-phosphate (PAP) (14, 15). The structural information combined with the results of mutational

---

* This work is supported in part by National Institutes of Health Grants AI50050 (to J. L.) and HL52622 (to R. J. L.) and Grant-in-Aid 0355800U (to J. L.) from the American Heart Association MidAtlantic Affiliate. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
The atomic coordinates and structure factors (codes 1T8U and 1T8T) have been deposited in the Protein Data Bank, Research Collaboratory for Structural Bioinformatics, Rutgers University, New Brunswick, NJ (http://www.rcsb.org/).
** To whom correspondence should be addressed: Rm. 309, Beard Hall, University of North Carolina, Chapel Hill, NC 27599. Tel.: 919-843-6511; Fax: 919-843-5432; E-mail: jian_liu@unc.edu.
¹ The abbreviations used are: HS, heparan sulfate; HSV-1, herpes simplex virus type 1; 3-OST-3, 3-O-sulfotransferase isoform 3; PAPS, 3′-phosphoadenosine 5′-phosphosulfate; NST, *N*-sulfotransferase; PAP, 3′-phosphoadenosine 5′-phosphate; Tricine, *N*- [2-hydroxy-1,1-bis(hydroxymethyl)ethyl]glycine; MES, 4-morpholineethanesulfonic acid; HPLC, high pressure liquid chromatography; GlcUA, glucuronic acid.

This paper is available online at http://www.jbc.org 45185

45186

Ternary Complex of 3-OST-3

A.

![Chemical Structure](https://i.imgur.com/chemical_structure.png)

B.

![Chemical Structure](https://i.imgur.com/chemical_structure_2.png)

FIG. 1. Substrate specificities of 3-OST-1 and 3-OST-3 and the structure of the tetrasaccharide substrate. A shows the enzymatic reactions catalyzed by 3-OST-1 and 3-OST-3. 3-OST-1 transfers sulfate to the 3-OH position of a glucosamine unit (GlcNS±6S) that is linked to a glucuronic acid unit (GlcUA), thereby forming the 3-O-sulfated HS containing GlcUA-GlcNS3S±6S (an anticoagulant HS). 3-OST-3 transfers sulfate to the 3-OH position of a glucosamine unit (GlcNH₂±6S) that is linked to a 2-O-sulfo iduronic acid unit (IdoUA2S), thereby forming the 3-O-sulfated HS containing IdoUA2S-GlcNH₂3S±6S (an entry receptor for HSV-1). $R$ represents a proton (-H) or a sulfate group ($-SO_{3}^{-}$). B shows the structure of the tetrasaccharide used in the study. The 3-O-sulfonation site is circled. For clarity, U4 represents ΔUA2S; G3 represents GlcNS6S linked to ΔUA2S; I2 represents IdoUA2S; and G1 represents GlcNS6S at the reducing end. The numbers indicate the positions of the saccharide units.

analysis provided us with preliminary insight into the HS sulfotransferase mechanism (14, 16). Furthermore, potential roles of specific amino acid residues from NST-1 and 3-OST-1 for directing the transfer of the sulfuryl group were suggested (14). However, because these structures lacked acceptor substrate, they have provided limited knowledge to our understanding of what dictates HS binding, in particular, among the 3-OST isoforms.

In this study, we report the crystal structure of a ternary complex of human 3-OST-3, PAP, and a tetrasaccharide substrate with the structure of ΔUA2S-GlcNS6S-IdoUA2S-GlcNS6S (for structure see Fig. 1B). This ternary complex structure, combined with the mutagenesis data presented here, reveals the residues involved in catalysis and in substrate binding. The data provide structural evidence for which residues confer substrate specificity between the 3-OST isoforms. These results present new insight into the biosynthesis of the HS with various biological functions ranging from viral infection to blood coagulation.

EXPERIMENTAL PROCEDURES

Expression, Purification, and Crystallization of 3-OST-3

Preparation of 3-OST-3 Bacterial Expression Plasmid—The cDNA fragment encoding the catalytic domain of 3-OST-3 (Gly¹³⁹–Gly⁴⁰⁶) was amplified from plasmid h3-OST-3A-pcDNA3 (17) with a 5′ overhang containing an EcoRI site and a 3′ overhang containing an NotI site. This construct was inserted into the pGEX4T3 vector (Amersham Biosciences) using the EcoRI and NotI restriction sites to produce a glutathione S-transferase fusion protein. The resultant plasmid was sequenced to confirm the reading frame and the lack of mutations within the coding region (University of North Carolina, DNA sequencing core facility). The plasmid was transformed into BL21(DE3)RIL cells (Stratagene).

Protein Expression and Purification—Cells containing the expression plasmid were grown in twelve 2.8-liter Fernbach flasks containing 1 liter of LB medium with 100 μg/ml of ampicillin at 37 °C. When the $A_{600}$ reached 0.6–0.8, the temperature was lowered to 22 °C for 15 min. Isopropyl-$\beta$-D-thiogalactopyranoside was then added to a final concentration of 200 μM, and the cells were allowed to shake overnight. The cells were pelleted and resuspended in 120 ml of sonication buffer (25 mM sodium/potassium phosphate, pH 7.4, and 625 mM NaCl). The cells were sonicated, and the cell debris was pelleted by centrifugation. The soluble fraction was loaded onto 20 ml of glutathione-Sepharose 4B (Amersham Biosciences) resin in batch and washed repeatedly with sonication buffer. The catalytic domain of 3-OST-3 was cleaved from the fusion protein on the resin at 4 °C overnight using 400 units of thrombin in a total volume of 50 ml. PAP (Sigma) was added to the solution for a final concentration of 0.5 mM. The protein was concentrated to ~35 mg/ml. Protein was further purified by a Superdex 75 16/60 (Amersham Biosciences) column. The eluted fractions containing pure 3-OST-3 were pooled. The buffer was exchanged with the crystallization buffer (20 mM Tris, pH 7.5, 100 mM NaCl, and 1 mM PAP) by repeated concentration and dilution. The final concentration of the protein was 10 mg/ml, and additional PAP was added for a final concentration of 4 mM.

Protein Crystallization, Data Collection, and Structure Solution—Crystals of the catalytic domain of 3-OST-3 were obtained using the hanging drop method by mixing 2 μl of the above protein solution with 2 μl of the reservoir solution consisting of 12–13% polyethylene glycol 4000, 200 mM ammonium acetate, and 100 mM sodium citrate, pH 5.5. For data collection, the crystals were transferred in four steps into a cryo-protectant consisting of 15% polyethylene glycol 4000, 12.5% ethylene glycol, 4 mM PAP, 200 mM ammonium acetate, 100 mM NaCl, and

Ternary Complex of 3-OST-3

TABLE I  
Crystallographic data statistics

| Data set | 3-OST-3 and PAP | 3-OST-3 and tetrasaccharide and PAP |
|----------|------------------|--------------------------------------|
| Unit cell | $a = 80.34, b = 155.39, c = 92.32$ | $a = 80.81, b = 154.53, c = 91.93$ |
|          | $\alpha = \beta = \gamma = 90^\circ$ | $\alpha = \beta = \gamma = 90^\circ$ |
| Space group | C222(1) | C222(1) |
| Resolution (Å) | 50–1.85 | 50–1.95 |
| Number of observations | 183,877 | 160,766 |
| Unique reflections | 49,160 | 42,327 |
| $R_{\text{sym}}$ (%) (last shell)$^a$ | 11.0 (29.7) | 5.4 (39.3) |
| $I/\sigma(I)$ (last shell) | 23.1 (3.5) | 18.2 (2.4) |
| Mosaicity | 0.7 | 0.7 |
| Completeness (%) (last shell) | 99.3 (99.1) | 99.9 (99.4) |
| Refinement statistics |  |  |
| $R_{\text{cryst}}$ (%)$^b$ | 20.6 | 20.1 |
| $R_{\text{free}}$ (%)$^c$ | 23.6 | 22.5 |
| Number of waters | 422 | 333 |
| Average $B$ values (Å) |  |  |
| Protein | 29.1 | 27.9 |
| PAP | 20.2 | 17.9 |
| Tetrasaccharide |  | 43.1 |
| Root mean square deviation from ideal values$^d$ |  |  |
| Bond length (Å) | 0.006 | 0.006 |
| Bond angle ($^\circ$) | 1.2 | 1.2 |
| Dihedral angle ($^\circ$) | 21.2 | 21.0 |
| Improper angle ($^\circ$) | 0.87 | 0.79 |
| Ramachandran statistics$^d$ |  |  |
| Residues in favored (98%) regions (%) | 97.56 | 97.94 |
| Residues in allowed (>99.8%) regions (%) | 100 | 100 |

$a$ $R_{\text{sym}} = \Sigma (I_i - \langle I \rangle)/\Sigma(I_i)$ where $I_i$ is the intensity of the $i$th observation and $\langle I \rangle$ is the mean intensity of the reflection.

$b$ $R_{\text{cryst}} = \Sigma \|F_o - F_c\| / \Sigma F_o$ calculated from working data set.

$c$ $R_{\text{free}}$ was calculated from 5% of data randomly chosen not to be included in refinement.

$d$ The Ramachandran results were determined by MolProbity (35).

100 mM sodium citrate, pH 5.5. To obtain the ternary complex, the crystal was soaked overnight in cryo-protectant containing 20 mM of the tetrasaccharide. The crystals were frozen in liquid nitrogen and then placed in a stream of nitrogen gas cooled to $-180^\circ$C. The data were collected on a Rigaku RU3H generator fitted with Osmic mirrors and a Raxis IV area detector. The data were indexed, integrated, and scaled using HKL2000 (18). Coordinates from the 3-OST-1 isoform (Protein Data Bank code 1S6T) were used to find a solution for the molecular replacement problem using the program Molrep from CCP4 (19, 20). The model of 3-OST-3 was refined by iterative cycles of model building in O (21) and refinement in CNS (22). The starting model for the tetrasaccharide was obtained from the Protein Data Bank coordinates 1BFB (23).

### Mutational Analysis of 3-OST-3

#### Preparation of 3-OST-3 Mutant Plasmids—The cDNA fragment encoding the catalytic domain of 3-OST-3 (Gly$^{139}$–Gly$^{406}$) was amplified from plasmid h3-OST-3A-pcDNA3 (17) with a 5′ overhang containing an NdeI site and a 3′ overhang containing an EcoRI site. This construct was inserted into the pET28a vector (Novagen) using the NdeI and EcoRI restriction sites to produce a His$_6$-tagged protein. The mutants were prepared using Gene Tailor site-directed mutagenesis kit from Invitrogen as described elsewhere (14).

#### Expression and Characterization of 3-OST-3—The procedures for the expression and purification of 3-OST-3 were essentially identical to those for 3-OST-1 as described in a prior publication (14). To confirm that bacterial 3-OST-3 has similar activity to the protein expressed in SF9 cells, we conducted a kinetic analysis to determine the $K_m$ and $V_{\text{max}}$ toward PAPS and HS, respectively. The $K_m$ and $V_{\text{max}}$ toward PAPS of the 3-OST-3 expressed in *Escherichia coli* is 40 μM and 27 pmol of sulfate/min, respectively, which are close to the $K_m$ value (16 μM) and $V_{\text{max}}$ value (27 pmol of sulfate/min) of the 3-OST-3 expressed in SF9 cells. The $K_m$ and $V_{\text{max}}$ toward HS of the 3-OST-3 expressed in *E. coli* is 2.5 μM and 24 pmol of sulfate/min, respectively, which are close to the $K_m$ (1 μM) and $V_{\text{max}}$ (21 pmol of sulfate/min) of the 3-OST-3 expressed in SF9 cells.

#### Expression and Purification of 3-OST-3 Mutants—The expression plasmids for various 3-OST-3 mutants were transformed individually into BL21(DE3)RIL cells (Stratagene). Each mutant construct was grown in 500 ml of LB broth and induced by isopropyl-$\beta$-thiogalactopyranoside. The bacteria were harvested and sonicated. The lysate was subjected to a 400-$\mu$l nitrilotriacetic acid-agarose column (0.75 cm × 1 cm), followed by 5 ml of wash with 10 mM imidazole, 25 mM Tris, and 500 mM NaCl, pH 7.5. Mutant proteins were eluted with 1 ml of elution buffer containing 25 mM Tris, 500 mM NaCl, and 250 mM imidazole, pH 7.5. Approximately 25 $\mu$l of the eluent was subjected to the analysis on a 16.5% Tris-Tricine-PAGE gel (Bio-Rad), and the gel was stained by Coomassie Blue. The expression level of the mutant protein was estimated by determining the intensity of the Coomassie-stained protein band near 30 kDa. Wild type 3-OST-3 was expressed along with the mutants. The expression levels and the sulfotransferase activity of mutant proteins were normalized to those of wild type protein.

#### Determination of the Sulfotransferase Activity—Sulfotransferase activity was determined by incubating ~5 $\mu$l (10–100 ng) of purified mutant or wild type 3-OST-3 proteins with 10 $\mu$g of HS (from bovine kidney; ICN), $5 \times 10^6$ cpm of [$^{35}$S]PAPS (~10 $\mu$M) in 50 $\mu$l of buffer containing 50 mM MES, pH 7.0, 10 mM MnCl$_2$, 5 mM MgCl$_2$, and 1% Triton X-100. The reaction was incubated at 37°C for 60 min and quenched by the addition of 6 M urea and 100 mM EDTA. The sample was then subjected to 200-$\mu$l DEAE-Sepharose chromatography to purify the [$^{35}$S]HS.

#### Disaccharide Analysis of the HS Modified by Wild Type 3-OST-3 and 3-OST-3 Mutants—3-OST-3-modified HS was degraded with nitrous acid at pH 1.5, followed by reduction with sodium borohydride (24). The resultant $^{35}$S-labeled disaccharides were resolved by a C$_{18}$ reversed phase column (0.46 × 25 cm) (Vydac) under reversed phase ion pairing HPLC conditions. The identities of the disaccharides were determined by coeluting with appropriate $^{35}$S-labeled disaccharide standards (25).

#### Determination of 3-O-[$^{35}$S]Sulfonation Site on the Tetrasaccharide

We utilized a combination of the digestions of heparin lyase and $\Delta^{4,5}$-glycuronate-2-sulfatase to determine the 3-O-[$^{35}$S]sulfonation site. The 3-O-[$^{35}$S]sulfated tetrasaccharide was prepared by incubating 3-OST-3 enzyme, the tetrasaccharide, and [$^{35}$S]PAPS. The resultant 3-O-[$^{35}$S]sulfated tetrasaccharide was eluted from the anion exchange HPLC column at a salt concentration consistent with the introduction of one additional sulfo group. Heparin lyase cleaves the glycosidic bond between G3 and I2 of the 3-O-[$^{35}$S]sulfated tetrasaccharide to yield two disaccharides, $\Delta$UA2S-[3-$^{35}$S]GlcNS3S6S and $\Delta$UA2S-GlcNS6S (unlabeled) (illustrated in Fig. 5) (26). To this point, it is not possible to distinguish whether the 3-O-[$^{35}$S]sulfo group was originally on the G3 or G1 unit of the original tetrasaccharide because the digestion would yield an identical $^{35}$S-labeled disaccharide for both cases. Pretreatment of 3-O-[$^{35}$S]sulfated tetrasaccharide with $\Delta^{4,5}$-glycuronate-2-sulfatase, however, removes the 2-O-sulfo group from the U4 unit (27) to yield $\Delta$UA unit at the nonreducing end (U4 unit). Subsequent treatment with heparin lyase (a mixture of heparin lyase I, II, and IV) were used

45188

Ternary Complex of 3-OST-3

FIG. 2. Ribbon diagram of the crystal structure of ternary complex 3-OST-3/PAP (blue)/tetrasaccharide (green) and sequence alignment of human 3-OST-3 and mouse 3-OST-1. β-Strands and α-helices are colored purple and cyan, respectively. This figure was created using Molscript and Raster3d (33, 34). Structural alignment of 3-OST-3 with 3-OST-1 is shown besides the ribbon diagram. Cyan barrels represent residues in helices, and purple represents regions in β-strands. Residues that are structurally dissimilar are shaded with a yellow background. Residues that form interactions with the tetrasaccharide are colored green, and the cysteines that form a disulfide are colored orange. Red asterisks represent the three residues that may form a catalytic triad in the 3-OSTs.

because of the reported resistance of small oligosaccharides to heparin lyases (26)) afforded the ΔUA-[3-³⁵S]GlcNS3S6S consistent with the presence of the 3-O-[³⁵S]sulfo group on the G3 residue, the ³⁵S-labeled disaccharide with a structure of ΔUA2S-[3-³⁵S]GlcNS3S6S would be obtained if the 3-O-[³⁵S]sulfo group is on the G1 residue. The resultant disaccharides were identified by determining the elution positions on an anion exchange HPLC using an on-line radioactive detector.

The conditions for Δ⁴,⁵-glycuronate-2-sulfatase digestion are described elsewhere (11). Digestion with heparin lyase of the tetrasaccharide was carried out using a mixture of heparin lyase I, II, and IV. The digestion condition for the mixture of heparin lyases was described in a prior publication (11). The anion exchange HPLC was performed on a silical based amino-bound anion exchange HPLC (0.46 × 24 cm; Waters) eluted with a linear gradient of KH₂PO₄ (300–1000 mM in 60 min) followed by 1 M KH₂PO₄ for an additional 20 min at a flow rate of 0.5 ml/min. We noted that this 3-O-sulfated tetrasaccharide is resistant to nitrous acid (at pH 1.5) degradation.

RESULTS AND DISCUSSION

Crystal Analysis—Human 3-OST-3 crystallizes in space group C222₁ and diffracts to 1.85 Å resolution. Two molecules were present in the asymmetric unit (molecules A and B), and each protein molecule contains one molecule of PAP. The crystals of 3-OST-3 are densely packed, generating a unit cell with a Matthews coefficient of 2.0 and an overall solvent content of 38.5% (28). Both molecules in the asymmetric unit are well ordered and have similar B-factors (Table I). The data were collected on a similar crystal soaked in the tetrasaccharide at 1.95 Å. One of the two molecules in the asymmetric unit has clear electron density for the tetrasaccharide that allows for precise positioning. The other molecule has some electron density in the binding pocket region, but it is not clear enough to model.

Overall Fold—The catalytic domain of human 3-OST-3 (Gly¹³⁹–Gly⁴⁰⁶) is roughly spherical with a large open cleft running across the surface of the sphere (Fig. 2). At the core of the sphere is an α/β motif. This α/β motif is comprised of a five-stranded parallel β-sheet (β7-β6-β2-β5-β3), laterally flanked by α-helices. Central to this structural motif is the phosphosulfate-binding loop (K162-R166), contained within a strand-loop-helix comprised of β2 and α2 (29). The phosphosulfate-binding loop allows for several strong hydrogen bonding interactions with the 5′-phosphate of PAP (Fig. 3 and Table II). These interactions are believed to help position the donor substrate, PAPS, in the precise location for catalysis (30). This binding site is well conserved with the previously reported 3-OST-1 and NST-1 (14, 15). The 5′-phosphate of PAP is found at the bottom of the large open cleft that provides the binding pocket for HS substrate (Figs. 2 and 3). A conserved α-helix (α5) composed of residues Ala²⁴⁴–Lys²⁵⁹ is positioned in close proximity to the PAP-binding site and protrudes outward into the open cleft. The C-terminal region of the protein contains a small three-stranded anti-parallel β-sheet (β8-β9-β10). The second and third strands of this sheet are structurally reinforced by a disulfide bond between residues Cys³⁵¹ and Cys³⁶³. Strand 10 of this sheet is connected to the C-terminal α-helix (α11) via a long coil (Lys³⁶⁶–Arg³⁷⁷) that lies along one side of the open cleft, partially obscuring the PAP-binding site.

The structure of 3-OST-3 is remarkably similar to that of 3-OST-1, with a root mean square deviation of 1.1 Å for 246 equivalent Ca atoms as calculated by the software program O (14, 21), although some minor differences present within looped regions were notable (Fig. 2). In contrast to 3-OST-1, 3-OST-3 has an insertion of three amino acids (Ala³⁵⁵–Gly³⁵⁷)

Ternary Complex of 3-OST-3

FIG. 3. Stereo diagram of the PAPS binding site of 3-OST-3 with PAP bound (blue). Residues that are within hydrogen bonding distance or that line the binding pocket are pictured. Possible hydrogen bonds between the protein and the PAP are displayed as black dashed lines. This figure was created using Molscript and Raster3d (33, 34).

TABLE II
Distances of potential hydrogen bonds between PAP and 3-OST-3 protein

| PAP | Protein contact | Distance |
|-----|-----------------|----------|
|     |                 | Å        |
| 5'P | O6P             | Lys¹⁶² NZ | 2.8      |
|     | O6P             | Lys³⁶⁸ NZ | 3.1      |
|     | O5P             | Lys³⁶⁸ NZ | 2.7      |
|     | O5P             | Arg¹⁶⁶ N  | 3.1      |
|     | O4P             | Thr¹⁶⁵ OG1 | 2.7      |
|     | O4P             | Thr¹⁶⁵ N   | 2.8      |
|     | O4P             | Gly¹⁶⁴ N   | 3.2      |
|     | O5'             | Gly¹⁶⁴ N   | 3.1      |
| 3'P | O3'             | Arg²⁴³ NH1 | 3.0      |
|     | O2P             | His³⁷² NE2 | 2.6      |
|     | O2P             | Ser²⁵¹ OG  | 2.6      |
|     | O3P             | Arg³⁷⁰ N   | 2.8      |
|     | O3P             | Gly³⁶⁹ N   | 2.9      |
|     | AdeN 6          | Pro³⁵⁰ O   | 2.9      |

TABLE III
Distances of potential interactions with the metal ion

| Ion | Protein contact | Distance |
|-----|-----------------|----------|
|     |                 | Å        |
| NA+1| I2O62           | 2.7      |
| NA+1| G12NO3S         | 2.4      |
| NA+1| G13OH           | 2.5      |
| NA+1| Asp²⁵² O        | 2.5      |
| NA+1| Thr²⁵⁶ OG1      | 2.3      |
| NA+1| Asp²⁵² OD1      | 2.4      |

into the loop formed by the disulfide bond between Cys³⁵¹ and Cys³⁶³ in strands 9 and 10 of the C-terminal β-sheet. These additional residues cause the loop to swing in the direction of the open cleft. Replacing the disulfide bond loop regions of 3-OST-1 (Cys²⁶⁰–Cys²⁶⁹) with the sequence of 3-OST-3 (Cys³⁵¹–Cys³⁶³) has no effect on the substrate specificity of 3-OSTs (data not shown). This finding suggests that this loop structure does not directly contribute to the substrate specificity of 3-OST-1 and 3-OST-3. Another notable difference between 3-OST-1 and 3-OST-3 is the presence of a N-terminal α-helix, Pro¹³⁶–Glu¹⁴⁷ (residues Pro¹³⁶–Ser¹³⁸ are cloning artifacts from the expression vector), found in 3-OST-3. This helix packs along the side of the protein in a shallow groove formed by the C-terminal three-stranded β-sheet of a neighboring molecule in the crystal. However, it is external to the conserved sulfotransferase domain.

Tetrasaccharide-binding Site—The structure of the protein with bound substrate is virtually identical to that of the apo-

TABLE IV
Distances of potential hydrogen bonds between the tetrasaccharide and 3-OST-3 protein

| Saccharide | Protein contact | Distance |
|------------|-----------------|----------|
|            |                 | Å        |
| U4         | 2O1S            | Lys²⁵⁹ NZ | 2.9      |
|            | 2O1S            | Glu²⁵⁵ NE2 | 3.0      |
|            | 2O3S            | Arg³⁷⁰ NH2 | 2.7      |
|            | 2O3S            | Glu²⁵⁵ OE1 | 3.1      |
|            | 6O61            | Lys³⁶⁸ NZ | 2.7      |
|            | 6O61            | Lys²¹⁵ NZ | 2.7      |
|            | 6O62            | Arg¹⁶⁶ NE  | 2.7      |
|            | 6O62            | Arg¹⁶⁶ NH2 | 2.5      |
|            | 3OH             | Thr³⁶⁷ OG1 | 2.6      |
| G3         | 2NO2S           | Ser²¹⁸ N  | 3.0      |
|            | 2NO6S           | Lys³⁶⁶ NZ | 2.5      |
|            | 6O4S            | Lys²⁵⁹ NZ | 2.9      |
|            | O6              | Lys²⁵⁹ NZ | 3.1      |
|            | 3OH             | Glu¹⁸⁴ OE2 | 2.8      |
|            | 3OH             | H₂O 158   | 3.0      |
| I2         | 3OH             | Arg¹⁹⁰ NH2 | 2.6      |
|            | 6O61            | Lys²⁵⁹ NZ | 2.6      |
|            | 6O61            | Glu²⁵⁵ NE2 | 2.6      |
|            | 6O62            | Lys¹⁶¹ NZ | 2.6      |
| G1         | 2N              | Thr²⁵⁶ OG1 | 2.9      |
|            | 2NO1S           | Ser²⁸⁴ OG  | 2.9      |
|            | 2NO1S           | Trp²⁸³ NE1 | 2.9      |
|            | 2NO3S           | Asp²⁵² OD1 | 2.8      |

*This residue is from a crystallographic symmetry related protein molecule in the crystal lattice.*

protein with respect to the backbone positions. A small number of side chains were altered upon substrate binding, most of which are found in or near the HS-binding site. The tetrasaccharide (ΔUA2S-GlcNS6S-IdoUA2S-GlcNS6S; Fig. 1B) binds in an extended conformation along the open cleft. This position places the 3-OH group of the glucosamine unit (G3) in close proximity to the PAP-binding site within the open cleft. It is interesting to note that the tetrasaccharide appears to behave rather like two distinct disaccharide units (Fig. 4). The two units on the reducing end (I2-G1) have fewer potential hydrogen bonding partners than those (U4-G3) on the nonreducing end and thus are more disordered and consequently have higher B-factors (Table I). In addition, the sugar rings of U4-G3 lie in the same plane that is rotated almost 90° around the α(1→4) glycosidic bond, with respect to the plane of the sugar rings of I2-G1. A similar twist has been previously observed in the basic fibroblast growth factor/HS structure (23).

The saccharide that forms the most extensive interactions with the protein is the uronic acid (U4) (Fig. 4, *a* and *b*, Table IV). This observation suggests that this sugar unit contributes significantly to the substrate specificity for 3-OST-3. Residues Glu²⁵⁵, Lys²⁵⁹, and Arg³⁷⁰ are all within hydrogen bonding distance to the 2-O-sulfo group. Residues Arg¹⁶⁶, Lys²¹⁵, and Lys³⁶⁸ are all within hydrogen bonding distance to the carboxyl group (Table IV). In addition to interactions with the charged groups, the hydroxyl from residue Thr³⁶⁷ is within hydrogen bonding distance to the 3-OH of U4. It should be noted that Δ⁴,⁵ unsaturated 2-O-sulfo uronic acid (ΔUA2SorU4 in this tetrasaccharide), a product of heparin lyase digestion, is a saturated saccharide unit (in the form of 2-O-sulfo iduronic acid, or IdoUA2S) in HS polysaccharide. The ²So conformation of 2-O-sulfo iduronic acid and the ²H₁ (31) conformation of ΔUA2S (U4) unit allow similar positioning of the 2-O-sulfo, 3-OH, and 5-carboxyl groups. Thus, the ΔUA2S mimics IdoUA2S sufficiently well to be recognized by 3-OST-3. Furthermore, the solution conformation of this tetrasaccharide indicates that

45190

Ternary Complex of 3-OST-3

a.

FIG. 4. Stereo diagrams of the binding of 3-OST-3 and tetrasaccharide. $a$, stereo diagram of HS acceptor substrate binding cleft with tetrasaccharide (green) bound. Residues that line the cleft are pictured. Displayed in blue is the simulated annealing $F_{o}-F_{c}$ omit electron density map contoured at $2 \sigma$. $b$, stereo diagram displaying the interactions between 3-OST-3 and U4-G3 units. The O-3 oxygen of G3 unit is labeled. $c$, stereo diagram of the interactions between 3-OST-3 and the I2-G1 units. A metal ion modeled as a sodium ion is colored cyan, as are interactions between the metal and the coordinating atoms. Possible hydrogen bonds between the protein and tetrasaccharide are displayed as black dashed lines. These figures were created using Molscript and Raster3d (33, 34).

b.

c.

neither the I2 or U4 units substantially modify the conformations of the adjacent G1 and G3 residue (31). Indeed, this tetrasaccharide is a substrate for 3-OST-3 as described below. The G3 unit appears to be the acceptor site of sulfonation. This unit is present in a ${ }^{4} C_{1}$ conformation, and its 3-OH is positioned within hydrogen bonding distance to the glutamate residue (Glu${ }^{184}$) (Table IV). It has been demonstrated that the equivalent residue in similar enzymes functions as the catalytic base (14, 16). A less ordered sulfo group at the C-6 position was observed, suggesting that it is not essential for substrate specificity. This observation is consistent with results from biochemical studies of 3-OST-3 (11, 25). The N-sulfo group is in a position to form a hydrogen bond with the backbone amide of Ser${ }^{218}$. It has been determined that 3-OST-3 sulfonates an N-unsubstituted glucosamine unit at the polysaccharide level (Fig. 1A) (25). Both glucosamine units (G1 and G3) are N-sulfonated (Fig. 1B) in the tetrasaccharide used in this experiment. Furthermore, the N-sulfo group was well ordered in a confined pocket, although no positively charged amino acid residues are found nearby. The restricted location of the N-sulfo group is perhaps due to the fact that the acceptor 3-OH group is highly oriented, consequently reducing the flexibility of N-sulfo group. In light of these results, we are currently unable to conclusively determine whether this N-sulfo group is required for substrate recognition.

Numerous amino acid residues have contacts with the carboxyl group of the iduronic acid unit (I2). The amino acid residues include Lys${ }^{161}$, Gln${ }^{255}$, and Lys${ }^{259}$ (Fig. 4c). In addition, the 3-OH position lies well within hydrogen bonding distance of Arg${ }^{190}$. Interestingly, the 2-O-sulfo group is not positioned near any hydrogen bonding partners, yet it is reasonably well ordered. The I2 unit is present in the ${ }^{2} S_{0}$ skew boat conformation, which differs from the conformation of this saccharide bound to basic fibroblast growth factor (23). Recent work has demonstrated the importance of the iduronic acid skew boat conformation in exhibiting the anticoagulant activity (32).

The G1 unit is also present in a ${ }^{4} C_{1}$ conformation (Fig. 4c).

Ternary Complex of 3-OST-3

45191

![Diagram](https://i.imgur.com/yourimageurl.png)

FIG. 5. Anion exchange HPLC chromatograms of enzymatic digested and undigested 3-O-[³⁵S]sulfated tetrasaccharide. A and B show the chromatogram of 3-O-[³⁵S]sulfated tetrasaccharide without and with Δ⁴,⁵-glycuronate-2-sulfatase digestion, respectively. C shows the chromatogram of heparin lyases digested 3-O-[³⁵S]sulfated tetrasaccharide that was treated with Δ⁴,⁵-glycuronate-2-sulfatase (solid line) and the chromatogram of the heparin lyases digested 3-O-[³⁵S]sulfated tetrasaccharide without the treatment of Δ⁴,⁵-glycuronate-2-sulfatase (dashed line). The enzymatic digestion reactions are illustrated besides the chromatograms. The anion exchange HPLC was performed on a silica based amino-bound anion exchange HPLC (0.46 × 24 cm; Waters) eluted with a linear gradient of KH₂PO₄ (300–1000 mM in 60 min) followed by 1 M KH₂PO₄ for additional 20 min at a flow rate of 0.5 ml/min. Arrow 1, elution position of [³⁵S]sulfate; arrow 2, elution position of ΔUA2S-[³⁵S]GlcNS3S6S; arrow 3, elution position of ΔUA2S-[³⁵S]GlcNS3S6S; arrow 4, elution position of ΔUA-[³⁵S]GlcNS3S6S-IdoUA2S-GlcNS6S; arrow 5, elution position of ΔUA2S-[³⁵S]GlcNS3S6S-IdoUA2S-GlcNS6S.

The N-sulfo group interacts with the side chain of Thr²⁵⁶ as well as with the side chains of Asp²⁵², Trp²⁸³, and Ser²⁸⁴ (Table IV). The 6-O-sulfo group makes no hydrogen bonding contacts with the protein. Consequently, it is the most disordered group in the tetrasaccharide. Interestingly, a metal ion modeled as a sodium ion appears to bind to the I2 and G1 units. The backbone carbonyl and side chain of Asp²⁵², as well as the side chain of Thr²⁵⁶, provide three of the ligands coordinated to the metal (Fig. 4c, Table III). The other three ligand interactions, which complete the octahedral coordination, are from the N-sulfo and 3-OH of the G1 unit as well as a carboxylate oxygen atom of the I2 unit.

**Determination of 3-O-Sulfonation Site on the Tetrasaccharide**—To confirm the conclusions from the structural data, we proved that 3-OST-3 transfers the [³⁵S]sulfuryl group to the G3 unit (rather than to the G1 unit) when the tetrasaccharide is used as a substrate. We prepared the 3-O-[³⁵S]sulfated tetrasaccharide by incubating the tetrasaccharide with purified 3-OST-3 enzyme and [³⁵S]PAPS. We then identified the [³⁵S]sulfonation site of the resultant tetrasaccharide. This was determined by comparing products of the heparin lyase digested 3-O-[³⁵S]sulfated tetrasaccharide with and without pre-

Ternary Complex of 3-OST-3

FIG. 6. Active site of 3-OST-3 with PAPS superimposed. Stereo diagram of the superposition of the PAPS (orange) from the crystal structure of human estrogen sulfotransferase into the active site of 3-OST-3 (30). The superposition is based on the Cα atoms of the phosphosulfate-binding loop and selected atoms of the PAP and PAPS molecules (O-5′, C-5′, C-4′, C-3′, and the 5′ phosphate). The PAP (blue) and the acceptor G3 (green) are pictured, as are the residues that may play a catalytic role. Possible hydrogen bonds between 3-OST-3 and its substrate are displayed as black dashed lines. Possible interactions between 3-OST-3 and the donor sulfonyl group based on the superposition are displayed as pink dashed lines. Also displayed as a pink dashed line is the line of attack of the O-3 oxygen of the sugar to the sulfur atom of the PAPS molecule. This figure was created using Molscript and Raster3d.

among 3-OSTs and NSTs, thereby suggesting an essential role for the catalytic function of HS sulfotransferases (NSTs transfer the sulfonyl group to the amino position of a glucosamine unit within a HS polysaccharide.). Indeed, mutations of this glutamate residue result in a complete loss of sulfotransferase activities for NST-1 and 3-OST-1 (14, 16) and a 99.9% loss for 3-OST-3 (Table V).

A hydrogen bonding network that involves Glu<sup>184</sup>, His<sup>186</sup>, and Asp<sup>189</sup> is observed in the ternary complex (Fig. 6). The OE2 oxygen of Glu<sup>184</sup> is 2.9 Å from the NE2 atom of His<sup>186</sup>. The ND1 atom of His<sup>186</sup> is 2.8 Å from OD2 of Asp<sup>189</sup>. This hydrogen bonding pattern is reminiscent of a catalytic triad. The results of the mutational analysis demonstrated that His<sup>186</sup> and Asp<sup>189</sup> are essential for the sulfotransferase activity (Table V). This hydrogen bond network could either help properly position the glutamate for catalysis or serve as a charge relay system to regulate the pK<sub>a</sub> of the glutamate. A similar hydrogen bond network was also observed in 3-OST-1 but not in NST-1 (14, 15). Thus, the interactions within the “triad” appear to be specific for 3-OSTs.

Mutational analysis—Site-directed mutagenesis experiments were performed to decipher the roles of the amino acid residues in performing the catalytic functions and determining substrate specificity. The results are summarized in Table V. We conducted the disaccharide analysis of the HS modified by the mutants that maintain the sulfotransferase activity (catalytically active mutants). Our data indicate that wild type protein and all of the catalytically active mutants sulfonate identical disaccharide sequences (IdoUA2S-GlcNH<sub>2</sub> and IdoUA2S-GlcNH<sub>2</sub>S; see Fig. 1A for structures), suggesting that these mutants maintain wild type 3-OST-3 substrate specificity (data not shown). This conclusion is further strengthened by our findings that none of the HS modified by the active mutants bind to antithrombin (data not shown). This is consistent with our understanding that 3-OST-1-modified HS binds to antithrombin, whereas 3-OST-3-modified HS does not (Fig. 1A).

Several mutations resulted in the loss of enzymatic activity (catalytically inactive mutants). Among these, K162A, E184Q, H186F, and D189N are probably due to defects in catalytic function. These conclusions are based on the crystal structural data as described above, as well as our previous results of the characterization of corresponding 3-OST-1 mutants (14). The roles of Lys<sup>215</sup> and Arg<sup>370</sup> are more complex. These two residues could be involved in both binding to PAPS and substrate. Like Glu<sup>184</sup> and Lys<sup>162</sup>, Lys<sup>215</sup> and Arg<sup>370</sup> are conserved in the 3-OSTs and NST-1.

Two mutants, Q255A and K368A, are of special interest.


TABLE V  
Summary of the analysis of the sulfotransferase activity of 3-OST-3 mutants  

| Name of the constructs<sup>a</sup> | Relative activity<sup>b</sup> | Expression level<sup>c</sup> |
|-----------------------------------|-----------------------------|---------------------------|
| 3-OST-3 wild type                 | 100                         | ++                        |
| 3-OST-3 K161A                     | 0.4                         | +                         |
| 3-OST-3 K162A                     | 0.4                         | +                         |
| 3-OST-3 R166E                     | 0.2                         | ++                        |
| 3-OST-3 E170Q                     | 83                          | ++                        |
| 3-OST-3 R173S                     | 55.9                        | +++                       |
| 3-OST-3 G182A                     | 106                         | ++                        |
| 3-OST-3 E184Q                     | 0.1                         | ++                        |
| 3-OST-3 H186F                     | 0.0                         | +                         |
| 3-OST-3 D189N                     | 0.9                         | +                         |
| 3-OST-3 R190E                     | 68                          | ++                        |
| 3-OST-3 K194A                     | 0.5                         | +                         |
| 3-OST-3 K215A                     | 0.1                         | +++                       |
| 3-OST-3 S218A                     | 76.7                        | ++                        |
| 3-OST-3 E224Q                     | 52.4                        | ++                        |
| 3-OST-3 Q255A                     | 0.4                         | ++                        |
| 3-OST-3 K259A                     | 51.7                        | ++                        |
| 3-OST-3 I288A                     | 35                          | ++                        |
| 3-OST-3 K293A                     | 66.4                        | ++                        |
| 3-OST-3 H362A                     | 101.5                       | ++                        |
| 3-OST-3 G365A                     | 57                          | ++                        |
| 3-OST-3 K366A                     | 0.2                         | ++                        |
| 3-OST-3 K368A                     | 0.1                         | ++                        |
| 3-OST-3 R370E                     | 0.8                         | ++                        |

<sup>a</sup> The mutants were prepared using a site-directed mutagenesis kit from Invitrogen.  
<sup>b</sup> The activity of 3-OST-3 was assayed by incubating the purified mutant proteins with HS and [³⁵S]PAPS, and the resultant [³⁵S]HS was quantified by a DEAE chromatography, where 100% activity represents the transfer of 18 pmol of sulfate/μg of protein under the standard assay condition.  
<sup>c</sup> The expression level of the proteins was determined by the intensity of the Coomassie Blue-stained protein band migrated at 30 kDa on SDS-PAGE.

Mutations at Gln<sup>255</sup> and Lys<sup>368</sup> nearly abolish sulfotransferase activity. However, mutations at the corresponding residues of 3-OST-1, Q166A and K274A, maintain 34.0 and 17.4% of the sulfotransferase activity, respectively (14). The data suggest that Gln<sup>255</sup> and Lys<sup>368</sup> contribute to the substrate specificity of 3-OST-3. Indeed, the structural data indicate that the side chains of Gln<sup>255</sup> and Lys<sup>368</sup> are within the hydrogen bonding distance to the carboxyl and 2-O-sulfo groups of the U4 unit and the carboxyl group of the I2 unit as described above. This is consistent with the fact that 3-OST-1 recognizes a disaccharide substrate (GlcUA-GlcNS±6S) that does not contain the 2-O-sulfo group and the orientation of the carboxyl group of the GlcUA unit is likely distinct from that of an iduronic acid unit (Fig. 1A).

Presently, we are unable to explain the mutation results for Lys<sup>194</sup> and Lys<sup>366</sup> because the side chains of both residues do not interact with the tetrasaccharide. Residue Lys<sup>194</sup> is located over 14 Å from the tetrasaccharide. Residue Lys<sup>366</sup> is located in a position where it could interact with the nonreducing end of the oligosaccharide if one more sugar (a glucosamine) were present, as is the case in the polymeric HS substrate used in the mutation studies. Interestingly, in the x-ray crystal structure, Lys<sup>366</sup> forms a lattice contact with the 6-O-sufo group of G3 from a tetrasaccharide binding in a crystallographically related molecule. This interaction does not appear to be biochemically relevant because the 6-O-sulfo group is not required for 3-OST-3 activity (11), and the buried surface area between the crystallographic interface of the two protein molecules is less than 3% of the total surface area.

In summary, the structure of the 3-OST-3/PAP/tetrasaccharide has provided new insights into the requirements for the substrate recognition by 3-OSTs and a more complete description of the HS sulfotransferase mechanism. It appears that 3-OST-3 recognizes the carboxyl and sulfo groups of the iduronic acid (U4) at the nonreducing end of the glucosamine unit (G3) being sulfonated. The amino acid residues involved in binding to these structural motifs are essential for the activity of 3-OST-3. This conclusion is consistent with those from previous biochemical studies of 3-OSTs; 3-OST-1 sulfates a glucosamine linked to a glucuronic acid unit at the nonreducing end, whereas 3-OST-3 sulfates the glucosamine unit that is linked to a 2-O-sulfo iduronic acid unit at the nonreducing end (Fig. 1A) (9, 11, 17, 25). It is very important to note that the 3-OST-3 modification site is flanked by two iduronic acid units in a skew boat like conformation. In contrast, the conformation of the I2 unit is in the chair form in the fibroblast growth factor/tetrasaccharide complex (23). It is likely that the conformation of the flanking iduronic acid residues contribute to substrate recognition by 3-OST-3.

The 3-O-sulfonation of HS has been directly linked to the activities for regulating blood coagulation and for promoting infection by HSV-1. Consequently elucidation of the general mechanism and discovery of the atomic level requirements for specificity provide important insight for understanding the biosynthesis of biologically active forms of HS.

Acknowledgments—We thank Dr. Keiichi Yoshida (Seikagaku Corporation) for providing us purified Δ<sup>4,5</sup>-glycuronate-2-sulfatase and heparin lyase IV and Drs. Jeffrey Vargason and Lee Pedersen for critical reading of the manuscript.

---

**REFERENCES**

1. Esko, J. D., and Selleck, S. B. (2002) *Annu. Rev. Biochem.* **71**, 435–471
2. Liu, J., and Thorp, S. C. (2002) *Med. Res. Rev.* **22**, 1–25
3. Lindahl, U., Kusche-Gullberg, M., and Kjellen, L. (1998) *J. Biol. Chem.* **273**, 24979–24982
4. Sasisekharan, R., Shriver, Z., Venkataraman, G., and Narayanasami, U. (2002) *Nat. Rev. Cancer* **2**, 521–528
5. Corey, L., and Spear, P. G. (1986) *N. Engl. J. Med.* **314**, 686–691 and 749–757
6. Spear, P. G., Eisenberg, R. J., and Cohen, G. H. (2000) *Virology* **275**, 1–8
7. Shukla, D., and Spear, P. G. (2001) *J. Clin. Invest.* **108**, 503–510
8. Spear, P. G., and Longnecker, R. (2003) *J. Virol.* **77**, 10179–10185
9. Shukla, D., Liu, J., Blaiklock, P., Shworak, N. W., Bai, X., Esko, J. D., Cohen, G. H., Eisenberg, R. J., Rosenberg, R. D., and Spear, P. G. (1999) *Cell* **99**, 13–22
10. Xia, G., Chen, J., Tiwari, V., Ju, W., Li, J.-P., Malmström, A., Shukla, D., and Liu, J. (2002) *J. Biol. Chem.* **277**, 37912–37919
11. Liu, J., Shriver, Z., Pope, R. M., Thorp, S. C., Duncan, M. B., Copeland, R. J., Raska, C. S., Yoshida, K., Eisenberg, R. J., Cohen, G., Linhardt, R. J., and Sasisekharan, R. (2002) *J. Biol. Chem.* **277**, 33456–33467
12. Shworak, N. W., Liu, J., Petros, L. M., Zhang, L., Kobayashi, M., Copeland, N. G., Jenkins, N. A., and Rosenberg, R. D. (1999) *J. Biol. Chem.* **274**, 5170–5184
13. Liu, J., Shworak, N. W., Fritze, L. M. S., Edelberg, J. M., and Rosenberg, R. D. (1996) *J. Biol. Chem.* **271**, 27072–27082
14. Edavettal, S. C., Lee, K. A., Negishi, M., Linhardt, R. J., Liu, J., and Pedersen, L. C. (2004) *J. Biol. Chem.* **279**, 25789–25797
15. Kakuta, Y., Sueyoshi, T., Negishi, M., and Pedersen, L. C. (1999) *J. Biol. Chem.* **274**, 10673–10676
16. Kakuta, Y., Li, L., Pedersen, L. C., Pedersen, L. G., and Negishi, M. (2003) *Biochem. Soc. Trans.* **31**, 331–334
17. Liu, J., Shworak, N. W., Sinai, P., Schwartz, J. J., Zhang, L., Fritze, L. M. S., and Rosenberg, R. D. (1999) *J. Biol. Chem.* **274**, 5185–5192
18. Otwinowski, Z., and Minor, V. (1997) *Methods Enzymol.* **276**, 307–326
19. Vagin, A., and Teplyakov, A. (1997) *J. Appl. Crystallogr.* **30**, 1022–1025
20. Bailey, S. (1994) *Acta Crystallogr. Sect. D Biol. Crystallogr.* **50**, 760–763
21. Jones, T. A., Zou, J. Y., Cowan, S. W., and Kjeldgaard, M. (1991) *Acta Crystallogr. Sect. A* **47**, 110–119
22. Brünger, A. T., Adams, P. D., Clore, G. M., DeLano, W. L., Gros, P., Grosse-Kunstleve, R. W., Jiang, J. S., Kuszewski, J., Nilges, M., Pannu, N. S., Read, R. J., Rice, L. M., Simonson, T., and Warren, G. L. (1998) *Acta Crystallogr. Sect. D Biol. Crystallogr.* **54**, 905–921
23. Faham, S., Hileman, R. E., Fromm, J. R., Linhardt, R. J., and Rees, D. C. (1996) *Science* **271**, 1116–1120
24. Shively, J. E., and Conrad, H. E. (1976) *Biochemistry* **15**, 3932–3942
25. Liu, J., Shriver, Z., Blaiklock, P., Yoshida, K., Sasisekharan, R., and Rosenberg, R. D. (1999) *J. Biol. Chem.* **274**, 38155–38162
26. Rice, K. G., and Linhardt, R. J. (1989) *Carbohydr. Res.* **190**, 219–233
27. McLean, M. W., Bruce, J. S., Long, W. F., and Williamson, F. B. (1984) *Eur. J. Biochem.* **145**, 607–615
28. Matthews, B. W. (1968) *J. Mol. Biol.* **33**, 491–497
29. Negishi, M., Pedersen, L. G., Petrochenko, E., Shevtsov, S., Gorokhov, A., Kakuta, Y., and Pedersen, L. C. (2001) *Arch. Biochem. Biophys.* **390**, 149–157
30. Pedersen, L. C., Petrochenko, E., Shevtsov, S., and Negishi, M. (2002) *J. Biol. Chem.* **277**, 17928–17932
31. Mikhailov, D., Mayo, K. H., Pervin, A., and Linhardt, R. J. (1996) *Biochem. J.* **315**, 447–454
32. Das, S. K., Mallet, J. M., Esnault, J., Driguez, P. A., Duchaussoy, P., Sizun, P., Herault, J. P., Herbert, J. M., Petitou, M., and Sinai, P. (2001) *Chemistry* **7**, 4821–4834
33. Kraulis, P. J. (1991) *J. Appl. Crystallogr.* **24**, 946–950
34. Merritt, E. A., and Bacon, D. J. (1997) *Methods Enzymol.* **277**, 505–524
35. Lovell, S. C., Davis, I. W., Arendall, W. B. r., de Bakker, P. I., Word, J. M., Prisant, M. G., Richardson, J. S., and Richardson, D. C. (2003) *Proteins* **50**, 437–450

Glycobiology and Extracellular Matrices:  
Structural Analysis of the Sulfotransferase  
(3- O-Sulfotransferase Isoform 3) Involved  
in the Biosynthesis of an Entry Receptor for  
Herpes Simplex Virus 1  

Andrea F. Moon, Suzanne C. Edavettal, Joe  
M. Krahn, Eva M. Munoz, Masahiko Negishi,  
Robert J. Linhardt, Jian Liu and Lars C.  
Pedersen  

J. Biol. Chem. 2004, 279:45185-45193.  
doi: 10.1074/jbc.M405013200 originally published online August 10, 2004  

---

Access the most updated version of this article at doi: [10.1074/jbc.M405013200](https://doi.org/10.1074/jbc.M405013200)

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinity-sites).

Alerts:
- When this article is cited
- When a correction for this article is posted

[Click here](https://www.jbc.org/site/alerts) to choose from all of JBC's e-mail alerts

This article cites 33 references, 12 of which can be accessed free at  
[http://www.jbc.org/content/279/43/45185.full.html#ref-list-1](http://www.jbc.org/content/279/43/45185.full.html#ref-list-1)
