
THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2004 by The American Society for Biochemistry and Molecular Biology, Inc.
Vol. 279, No. 46, Issue of November 12, pp. 47822–47832, 2004
Printed in U.S.A.

# N-Glucosylation of Cytokinins by Glycosyltransferases of Arabidopsis thaliana*

Received for publication, August 19, 2004
Published, JBC Papers in Press, September 1, 2004, DOI 10.1074/jbc.M409569200

Bingkai Hou, Eng-Kiat Lim, Gillian S. Higgins, and Dianna J. Bowles‡

From the CNAP, Department of Biology, University of York, York YO10 5DD, United Kingdom

Cytokinins are plant hormones that can be glucosylated to form O-glucosides and N-glucosides. The glycoconjugates are inactive and are thought to play a role in homeostasis of the hormones. Although O-glucosyltransferases have been identified that recognize cytokinins, the enzymes involved in N-glucosylation have not been identified even though the process has been recognized for many years. This study utilizes a screening strategy in which 105 recombinant glycosyltransferases (UGTs) of Arabidopsis have been analyzed for catalytic activity toward the classical cytokinins: trans-zeatin, dihydrozeatin, N⁶-benzyladenine, N⁶-isopentenyladenine, and kinetin. Five UGTs were identified in the screen. UGT76C1 and UGT76C2 recognized all cytokinins and glucosylated the hormones at the N⁷ and N⁹ positions. UGT85A1, UGT73C5, and UGT73C1 recognized trans-zeatin and dihydrozeatin, which have an available hydroxyl group for glucosylation and formed the O-glucosides. The biochemical characteristics of the N-glucosyltransferases were analyzed, and highly effective inhibitors of their activities were identified. Constitutive overexpression of UGT76C1 in transgenic Arabidopsis confirmed that the recombinant enzyme functioned *in vivo* to glucosylate cytokinin applied to the plant. The role of the N-glucosyltransferases in cytokinin metabolism is discussed.

involved in general purine metabolism, and their impact on cytokinin activity, transport, and homeostasis remain unknown (3, 11). In contrast, modifications of the N⁶-side chain are known to have profound effects on cytokinin activity, and extensive studies over many years have analyzed structure-activity relationships of the side chain (12). Genes encoding enzymes involved in cytokinin biosynthesis have been identified (3, 13–16), and processes leading to irreversible degradation by cytokinin oxidases have also been well characterized (17, 18). The physiological significance of the biosynthetic and degradative pathways has been confirmed in transgenic studies (15, 19–22).

Conjugation of cytokinins also occurs and can involve O-xylolation, O-glucosylation, and N-glucosylation (23–30). Although all of these cytokinin glycosides are known to be inactive, their role in homeostasis is uncertain (3, 13). However, O-glucosylation is reversible, and the O-glucosides are resistant to cleavage of the N⁶-side chain by cytokinin oxidases (17, 18). Because these conjugates can be converted back into active cytokinins by β-glucosidases (31, 32), it has been suggested that they represent inactive, stable storage forms of the hormone. When plants are subjected to high levels of cytokinin application, the major conjugate that forms is the 7-N-glucoside (33, 34). In contrast to the O-glucosides, the 7-N- and 9-N-glucosides are resistant to glucosidases (32, 35), and this feature, together with their accumulation in the plant, has led to the suggestion that N-glucosylation is involved in detoxification (35). The N-glucosylation of cytokinins has thus attracted considerable interest, because understanding of the enzyme(s) and catalytic mechanism(s) could aid the design of novel long lasting cytokinin analogues for use in field applications.

We now present data to show we have identified Arabidopsis genes encoding enzymes capable of cytokinin N-glucosylation. Glycosyltransferases have been classified into 69 multigene families (36, 37) (afmb.cnrs-mrs.fr/CAZY/), of which Family 1 includes enzymes involved in small molecule glucosylation, such as conjugation of plant hormones (38). In the Arabidopsis genome, open reading frames corresponding to these glycosyltransferases (UGTs)¹ number 107 and can be further classified into 14 subgroups (39, 40). Through *in vitro* screening >100 recombinant UGTs from this multigene Family 1, two UGTs capable of N-glucosylation and three UGTs capable of O-glucosylation have now been identified. This study focuses on the biochemical characterization of the N-glucosyltransferases and shows that overexpression of one of these UGTs *in planta* leads to an enhanced accumulation of the 7-N-glucoside on spraying with cytokinin.

The family of plant hormones known as cytokinins were discovered by their ability to induce cell division in cell cultures (1, 2). Naturally occurring cytokinins are adenine derivatives and, dependent on the nature of the N⁶-side chain, are classified into isoprenoid or aromatic cytokinins (3). In the plant, cytokinins are known to be involved in many different metabolic and developmental processes (4–7) that are often also influenced by other stimuli including environmental factors and additional hormones (3, 8–10).

The metabolism of cytokinins is highly complex, and little is known of the mechanisms that regulate homeostasis. Cytokinins can be present in plants as the free base and the corresponding nucleosides and nucleotides. Interconversions among these three forms are most probably carried out by enzymes

* This work was supported by Biotechnology and Biological Sciences Research Council Grant 87/EGA16205 and by funding from the Garfield Weston Foundation. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
[S] The on-line version of this article (available at http://www.jbc.org) contains a supplemental figure.
The nucleotide sequence(s) reported in this paper has been submitted to the DDBJ/GenBank™/EBI Data Bank with accession number(s) AB017060 (UGT76C1), AB005237 (UGT76C2), AC006551 (UGT85A1), AC006282 (UGT73C5) and AC006282 (UGT73C1).
‡ To whom correspondence should be addressed. Tel.: 44-1904-328770; Fax: 44-1904-328772; E-mail: djb32@york.ac.uk.

¹ The abbreviations used are: UGT, glycosyltransferase; HPLC, high pressure liquid chromatography; DTT, dithiothreitol; IAA, indole-3-acetic acid; MALDI, matrix-assisted laser desorption/ionization; MS, mass spectrometry; MES, 4-morpholineethanesulfonic acid.

Cytokinin N-Glucosylation

A trans-zeatin                                                                 B $N^{6}$-benzyladenine

7-N-Glc                                                                 7-N-Glc
9-N-Glc                                                                 9-N-Glc

76C1

mAU
1000
500
0
10.0 12.5 15.0 17.5
Minutes

mAU
1000
500
0
10.0 12.5 15.0 17.5
Minutes

76C2

mAU
2000
1000
0
10.0 12.5 15.0 17.5
Minutes

mAU
2000
1000
0
10.0 12.5 15.0 17.5
Minutes

85A1

O-Glc

mAU
2000
1000
0
10.0 12.5 15.0 17.5
Minutes

mAU
2000
1000
0
10.0 12.5 15.0 17.5
Minutes

aglycone

mAU
2000
1000
0
10.0 12.5 15.0 17.5
Minutes

mAU
750
500
250
0
10.0 12.5 15.0 17.5
Minutes

9-N-Glc

mAU
2000
1000
0
10.0 12.5 15.0 17.5
Minutes

mAU
400
200
0
10.0 12.5 15.0 17.5
Minutes

7-N-Glc

mAU
200
100
0
10.0 12.5 15.0 17.5
Minutes

mAU
300
200
100
0
10.0 12.5 15.0 17.5
Minutes

O-Glc

mAU
2000
1000
0
10.0 12.5 15.0 17.5
Minutes

mAU
1500
1000
500
0
10.0 12.5 15.0 17.5
Minutes

3-N-Glc

mAU
750
500
250
0
10.0 12.5 15.0 17.5
Minutes

FIG. 1. HPLC analysis of cytokinins and their glucosconjugates. The reaction mixes contained a single cytokinin (A, trans-zeatin; B, $N^{6}$-benzyladenine) incubated with UGT 76C1, 76C2, or 85A1. For trans-zeatin conjugates, a linear gradient of methanol from 10 to 60% at 1 ml/min flow rate over 25 min was used, and for $N^{6}$-benzyladenine conjugates, a linear gradient of methanol from 10 to 100% was used (all solutions contained 2.5 ml/liter of glacial acetic acid and 0.4 ml/liter of triethylamine).

EXPERIMENTAL PROCEDURES

Materials—Cytokinins, cytokinin glycosides ($N^{6}$-benzyladenine-3-$N$-glucoside, $N^{6}$-benzyladenine-7-$N$-glucoside, $N^{6}$-benzyladenine-9-$N$-glucoside, dihydrozeatin-7-$N$-glucoside, dihydrozeatin-9-$N$-glucoside, dihydrozeatin-O-glucoside, $N^{6}$-isopentenyladenine-7-$N$-glucoside, $N^{6}$-isopentenyladenine-9-$N$-glucoside, kinetin-3-$N$-glucoside, trans-zeatin-7-$N$-glucoside, trans-zeatin-9-$N$-glucoside, trans-zeatin-O-glucoside,

trans-zeatin-9-$N$-riboside, trans-zeatin-9-$N$-ribosyl-O-glucoside, trans-zeatin-9-$N$-glucosyl-O-glucoside, and meta-topolin-9-$N$-glucoside) and cytokinin analogues used in this study were purchased from OlChemIm Ltd. (Olomouc, Czech Republic). The plants used were grown in Lexington’s seed and modular compost in a controlled environment of 16/8-h light-dark cycle (light, ~150 μmol photons m⁻² s⁻¹, 22 °C; dark, 18 °C).

47824

Cytokinin N-Glucosylation

TABLE I

Specific activity of UGTs 76C1, 76C2, 85A1, 73C5, and 73C1 towards cytokinins

The specific enzyme activity was defined as nmol of substrates converted into glucose conjugates/s (nanokatal, nkat) by 1 mg of protein. —, no significant activity detected (<0.01 nkat/mg of protein).

| Substrates               | Specific activity (nkat/mg protein) |
|--------------------------|-------------------------------------|
|                          | 76C1      | 76C2      | 85A1      | 73C5      | 73C1      |
|                          | 7-N-      | 9-N-      | 7-N-      | 9-N-      | O-        | O-        |
| trans-Zeatin             | ![trans-Zeatin Structure](https://i.imgur.com/structure1.png) | 1.66      | 0.54      | 1.45      | 0.48      | 0.86      | 0.03      | 0.02      |
| Dihydrozeatin            | ![Dihydrozeatin Structure](https://i.imgur.com/structure2.png) | 3.36      | 0.41      | 3.34      | 0.41      | 0.12      | 0.20      | 0.07      |
| $N^6$-Isopentenyladenine | ![N6-Isopentenyladenine Structure](https://i.imgur.com/structure3.png) | 1.74      | 0.18      | 2.09      | 0.20      | —         | —         | —         |
| $N^6$-Benzyladenine     | ![N6-Benzyladenine Structure](https://i.imgur.com/structure4.png) | 0.81      | 1.39      | 1.01      | 1.70      | —         | —         | —         |
| Kinetin                  | ![Kinetin Structure](https://i.imgur.com/structure5.png) | 0.69      | 0.27      | 0.95      | 0.36      | —         | —         | —         |

Glucosyltransferase Activity Assay—The construction of the UGT expression plasmids was described in our previous report (39). The plasmids were transformed into *Escherichia coli* XL-1 Blue individually for recombinant protein expression. Recombinant UGTs were purified as fusion proteins with glutathione S-transferase attached to the N terminus of the UGTs (41). 105 UGTs of the total 107 UGTs (40) were subjected to the initial screen (supplemental table). UGTs 76E9 and 90A1 were not included because of the inability to recover recombinant proteins. The glucosyltransferase assay was carried out following the conditions described by Mok and co-workers (23, 24) with modifications. In the initial screen, each assay mix (200 μl) contained 1 μg of recombinant protein, 50 mm Tris-HCl, pH 8.0, 2.5 mm UDP-glucose, 0.5 mm ATP, 50 mm MgCl₂, and 0.5 mm of a cytokinin substrate. The reaction was carried out at 30 °C for 3 h and was stopped by the addition of 20 μl of trichloroacetic acid (240 mg/ml), quick-frozen, and stored at −20 °C prior to the reverse-phase HPLC analysis. The specific enzyme activity was expressed as nanomoles of cytokinins glucosylated per second (nanokatal) by 1 mg of protein.

Identification of the Bacterial Chaperonin GroEL Co-purifying with UGTs 76C1 and 76C2—The polyacrylamide gel containing the polypeptide that co-purified with recombinant UGTs 76C1 and 76C2 was excised for in-gel tryptic digestion. After reduction with DTT and S-carboxymethylation with iodoacetamide, the gel pieces were washed three times with 50% (v/v) aqueous acetonitrile containing 25 mm ammonium bicarbonate and dried in vacuo. The gel pieces were rehydrated in 10 μl of trypsin solution (0.2 μg of trypsin, 10 mM acetic acid, 20 mM ammonium bicarbonate) and were incubated overnight at 37 °C. A 0.5-μl aliquot of trypsin solution was applied directly to the matrix-assisted laser desorption/ionization (MALDI) target plate, followed immediately by an equal volume of a freshly prepared 5 mg/ml solution of 4-hydroxy-α-cyano-cinnamic acid (Sigma) in 50% (v/v) aqueous acetonitrile containing 0.1% (v/v) trifluoroacetic acid. Positive ion MALDI mass spectra were obtained using an Applied Biosystems 4700 Proteomics Analyzer (Applied Biosystems, Foster City, CA) in reflectron mode with an accelerating voltage of 20 kV. MS spectra were acquired with a total of 1000 laser pulses over a mass range of m/z 800–4000. Final mass spectra were the summation of 20 subspectra, each acquired with 50 laser pulses, and internally calibrated using the tryptic peptides at m/z 842.509 and 2211.104. Monoisotopic masses were obtained from centroids of raw, unsmoothed data.

For collision-induced dissociation-MS/MS, a Source 1 accelerating voltage of 8 kV, a collision energy of 1 kV, and a Source 2 accelerating voltage of 15 kV were used. Air was used as the collision gas at the “medium” pressure setting of the instrument with a recharge threshold of 9.9 × 10⁻⁷ torr, which produced a Source 2 pressure of about 1 × 10⁻⁶ torr. The precursor mass window was set to ±10 Da, and the metastable suppressor was enabled. The default calibration was used for MS/MS spectra, which were base line-subtracted (peak width 50) and smoothed (Savitsky-Golay with three points and polynomial order 4).

Mass spectral data obtained in batch mode were submitted to data base searching using the Mascot program (Matrix Science Ltd., version 1.9). Batch-acquired MS and MS/MS spectral data were submitted to a combined peptide mass fingerprint and MS/MS ion search through the Applied Biosystems GPS Explorer software interface (version 2.0) to Mascot. Search criteria included: maximum missed cleavages, 1; variable modifications, oxidation (M), carbamidomethyl; peptide tolerance, 200 ppm; MS/MS tolerance, 0.15 Da.

Removal of the Bacterial Chaperonin GroEL from Recombinant UGTs 76C1 and 76C2—E. coli strain XL1-Blue expressing UGT76C1 or UGT76C2 was grown at 20 °C in 1.5 liters of 2× YT medium containing 50 μg/ml ampicillin until the $A_{600\text{ nm}}$ reached 1.0. The culture was then incubated with 1 mM isopropyl-1-thio-β-D-galactopyranoside for 24 h at 20 °C to induce synthesis of the recombinant proteins. The cells were harvested (5,000 × g for 5 min), resuspended (5 ml, ice-cold phosphate-buffered saline), disrupted by French Press (ThermoElectron), and centrifuged again (40,000 × g for 30 min). The supernatant was mixed with 100 μl of 50% (v/v) glutathione-coupled Sepharose (Amersham Biosciences) at room temperature for 30 min. To remove the GroEL proteins that co-purified with UGTs 76C1 and 76C2, the beads were washed with phosphate-buffered saline followed by a wash with 20 mM ATP and 20 mM MgCl₂ at room temperature for 30 min prior to the elution of the recombinant proteins with 20 mM reduced-form glutathione, 100 mM Tris-HCl, pH 8.0, 120 mM NaCl.

Steady-state Enzyme Kinetic Measurements—Michaelis-Menten ki-

Cytokinin N-Glucosylation

TABLE II

Specific activity of UGTs 76C1, 76C2, 85A1, 73C5, and 73C1 towards other related compounds

Thirteen additional compounds related to cytokinins were analyzed against the five UGTs that glucosylate cytokinins. Four compounds that can be glucosylated by these UGTs are listed in the table. The specific enzyme activity was defined as nmol of substrates converted into glucose conjugates/s (nanokatal, nkat) by 1 mg of protein. The remaining nine compounds that cannot be glucosylated by these UGTs are shown in the footnote (a). —, no significant activity detected (< 0.01 nkat/mg of protein).

| Substrates           | Specific activity (nkat/mg protein) |
|----------------------|---------------------------------------|
|                      | 76C1                                 | 76C2                                 | 85A1                                 | 73C5                                 | 73C1                                 |
|                      | 7-N-                                 | 9-N-                                 | 7-N-                                 | 9-N-                                 | O-                                    | O-                                    |
| cis-Zeatin           | ![Chemical Structure](https://i.imgur.com/structure1.png) | 0.28                                 | 0.21                                 | 0.17                                 | 0.14                                 | 0.79                                 | 0.05                                 | 0.03                                 |
| trans-Zeatin-O-glc   | ![Chemical Structure](https://i.imgur.com/structure2.png) | 0.60                                 | —                                    | 0.46                                 | —                                    | —                                    | —                                    | —                                    |
| Dihydrozeatin-9-N-gl | ![Chemical Structure](https://i.imgur.com/structure3.png) | —                                    | —                                    | —                                    | —                                    | 0.05                                 | 0.17                                 |                                      |
| Olomoucine           | ![Chemical Structure](https://i.imgur.com/structure4.png) | —                                    | —                                    | —                                    | —                                    | 0.40                                 | 0.12                                 |                                      |

a the compounds that cannot be glucosylated in this study:

| ![Chemical Structures](https://i.imgur.com/structures5-8.png) |
|-------------------------------------------------------------|

netics of the enzymes were measured over a range of 0–0.5 mM of cytokinin substrates in the presence of 1 μg of recombinant protein, 100 mM MES, pH 7.0, 2.5 mM UDP-glucose, 0.5 mM ATP, 50 mM MgCl₂. The reactions (200 μl) were carried out at 30 °C for 1 h. The kinetic parameters were derived using Hyperbolic Regression Analysis of Hyper32 program available from homepage.ntworld.com/john.easterby.

HPLC Analysis of the in Vitro Reaction Mixtures—Reverse-phase HPLC (SpectraSYSTEM HPLC systems and UV6000LP Photodiode Array Detector; ThermoQuest) analysis was carried out using a Columnbus 5-μ C₁₈ column (250 × 4.60 mm; Phenomenex) maintained at 30 °C. The data were acquired and analyzed using the software ChromQuest version 2.51. Two different HPLC methods were developed for the analysis of cytokinins and their analogues. Each recombinant UGT activity assay containing a single substrate was analyzed using one of the methods described in the following.

The first HPLC method employed a linear gradient of 10–100% methanol against H₂O (all solutions contained 2.5 ml/liter of glacial acetic acid and 0.4 ml/liter of triethylamine) with a flow rate at 1 ml/min over 25 min. Each peak on the chromatogram was scanned between 200 and 400 nm (photodiode array profile) and was integrated at 270 nm.

47826

Cytokinin N-Glucosylation

A

kDa 76C1 76C2 85A1 73C5 73C1 GST

FIG. 2. SDS-PAGE analysis of the five recombinant UGTs capable of cytokinin glucosylation in vitro. A, the UGTs were analyzed on a 10% (w/v) polyacrylamide gel and visualized with Coomassie Brilliant Blue staining. The smaller polypeptide co-purifying with UGTs 76C1 and 76C2 fusion proteins was confirmed to be the bacterial chaperonin GroEL. B, SDS-PAGE analysis of UGTs 76C1 and 76C2 eluted from the affinity matrix post-treated with MgCl₂/ATP.

B

kDa 76C1 76C2

0 15 30 0 15 30 min wash with MgCl₂ and ATP

GST-UGT fusion GroEL

12.6 min), trans-zeatin-O-glucoside (Rₜ = 12.9 min), trans-zeatin-9-N-riboside (Rₜ = 16.3 min), trans-zeatin-9-N-ribosyl-O-glucoside (Rₜ = 14.5 min), trans-zeatin-7-N-glucosyl-O-glucoside (Rₜ = 10.5 min), trans-zeatin-9-N-glucosyl-O-glucoside (Rₜ = 11.1 min), cis-zeatin (Rₜ = 13.5 min), cis-zeatin-7-N-glucoside (Rₜ = 11.2 min), cis-zeatin-9-N-glucoside (Rₜ = 11.8 min), cis-zeatin-O-glucoside (Rₜ = 12.2 min), dihydrozeatin (Rₜ = 15.0 min), dihydrozeatin-7-N-glucoside (Rₜ = 11.7 min), dihydrozeatin-9-N-glucoside (Rₜ = 13.0 min), dihydrozeatin-O-glucoside (Rₜ = 14.0 min), and dihydrozeatin-9-N-glucosyl-O-glucoside (Rₜ = 12.5 min).

Adenine (Rₜ = 12.9 min) and guanine (Rₜ = 8.8 min) were analyzed by a linear gradient of 10–20% methanol against 50 mM ammonium formate buffer, pH 4.6, over 20 min with a flow rate at 0.5 ml/min. Each peak on the chromatogram was scanned between 200 and 400 nm (photodiode array profile) and was integrated at 254 nm.

Identification and Quantification of the Glucosides Synthesized during the in Vitro Reactions—The nature of glucosides synthesized by recombinant UGTs in this study was confirmed by comparing the photodiode array profile and the retention time of the glucosides on the chromatogram with the authentic compounds when these were available (see “Materials”). For those glucosides that were not available commercially, their chromatograms were compared with the glucosides of similar chemical structures. Kinetin-7-N-glucoside and kinetin-9-N-glucoside were assigned by comparing with the chromatogram that separating N⁶-benzyladenine and its glucosides. cis-Zeatin-7-N-glucoside, cis-zeatin-9-N-glucoside, and cis-zeatin-O-glucoside were assigned similarly using the chromatogram separating trans-zeatin and trans-zeatin glucosides. trans-Zeatin-7-N-glucosyl-O-glucoside, dihydrozeatin-9-N-glucosyl-O-glucoside, and olomoucine glucoside were confirmed by hydrolysis of the O-glucosidic bond using β-glucosidase leading to the formation trans-zeatin-7-N-glucoside, dihydrozeatin-9-N-glucoside, and olomoucine, respectively. The glucosides were quantified using either the extinction coefficient of the authentic glucosides or the extinction coefficient of the aglycone after hydrolysis of the corresponding glucosides.

Overexpression of UGT76C1 in Arabidopsis—The binary plasmid pJR1Ri carrying UGT76C1 cDNA was introduced into Arabidopsis thaliana via Agrobacterium tumefaciens-mediated vacuum infiltration (42). To select the transgenic plants, the seeds collected from vacuum-infiltrated plants were surfaced sterilized with 10% (v/v) Chloros solution (Scientific Laboratory Supplies, Nottingham, UK) with a drop of Tween 80 followed by 70% (v/v) ethanol and were germinated on ATS agar (10 g/liter sucrose, 5 ml/liter of 1 M KNO₃, 2.5 ml/liter of 1 M KPO₄, pH 5.5, 2 ml/liter of 1 M MgSO₄, 2 ml/liter of 1 M Ca(NO₃)₂, 2.5 ml/liter of 20 mM EDTA, 1 ml/liter micronutrients, and 0.8% (w/v) Oxoid agar (Oxoid Ltd., Hampshire, UK)) containing kanamycin (50 mg/liter) and cefotaxime (50 mg/liter). All of the plants were homozygous for the transgene. The control plant, designated Ri, was transformed with an empty pJR1Ri vector.

Northern Blot Analysis—Total RNA was extracted from the transgenic plants following the method of Verwoerd et al. (43). An aliquot of total RNA (10 μg) was separated on 0.6 M formaldehyde, 1.4% (w/v) agarose containing 0.5 μg/ml ethidium bromide, blotted onto Hybond-N membrane, UV-cross-linked (120 mJ), and probed with radiolabeled DNA fragments prepared with random primers.

Crude Protein Extraction from Transgenic Plants and the Glucosyltransferase Assay—To obtain the crude protein extract from the transgenic plants, 1 g of frozen tissue was ground to fine powder in liquid nitrogen using pestle and mortar. 1 ml of extraction buffer (25 mM Tris-MES, pH 6.5, 10% (v/v) glycerol, 20 mM 2-mercaptoethanol, 1 mM phenylmethylsulfonyl fluoride and 1% polyvinylpolypyrrolidone (w/w of aerial tissue)) was added to the powder, and the slurry was thawed on ice. Subsequently the slurry was mixed vigorously and centrifuged at 10,500 × g at 4 °C for 20 min. 850 μl of the supernatant were collected and transferred to a 1.5-ml microcentrifuge tube and were further centrifuged at 15000 × g, 4 °C for 5 min. The resulting supernatant was collected for glucosyltransferase activity assay. The protein concentration assay was carried out with Bio-Rad protein assay dye using bovine serum albumin as reference.

To investigate the glucosyltransferase activity of the crude protein extracts prepared from plant tissues, 50-μl crude protein extracts (containing 0.1–0.3 mg of total protein) were mixed with 1 mM trans-zeatin, 5 mM UDP-glucose, and 100 mM MES, pH 7.0, in a 100-μl reaction. The reactions were incubated at 30 °C for 1 h and were stopped by the addition of 10 μl of trichloroacetic acid (240 mg/ml). The reaction mixtures were analyzed subsequently using reverse-phase HPLC following the method described above.

Exogenous Application of trans-Zeatin to Transgenic Plants Overexpressing UGT76C1 cDNA—Transgenic line 4/1 and a control line Ri were selected and sprayed with trans-zeatin exogenously. trans-Zeatin was dissolved in distilled water containing 1% Me₂SO and 0.05% Tween 20 to a final concentration of 0.5 mM and was sprayed either on its own or together with the inhibitors (1 mM 3-isobutyl-1-methylxanthine or 0.05 mM olomoucine) onto the plants using a spray bottle. At appropriate time points, ~1 g of aerial tissues from each line was collected, frozen in liquid nitrogen, and stored at −80 °C prior to the extraction.

HPLC Analysis of the trans-Zeatin Glucosides in Transgenic Plants—To analyze the amount of the glucosides of interest in the transgenic plants, 1 g of plant tissue was ground to fine powder and was extracted with 10 ml of 80% methanol. The slurry was left at room temperature for 1 h followed by centrifugation at 5000 × g for 5 min.

Cytokinin N-Glucosylation

A
![Image A](image_a.png)

B
![Image B](image_b.png)

C
![Image C](image_c.png)

D
![Image D](image_d.png)

E
![Image E](image_e.png)

F
![Image F](image_f.png)

FIG. 3. Biochemical characterization of UGTs 76C1 and 76C2. $A$ and $B$, the pH optimum of the UGTs was measured over the range pH 5.0–9.0 in the reactions containing 0.5 mM trans-zeatin and 100 mM buffer. The buffers analyzed include MES buffer (pH 5.0–7.0), phosphate buffer (pH 6.0–8.0), and Tris-HCl buffer (pH 7.0–9.0). $C$ and $D$, the time course of the UGT activity was studied by measuring the amount of glucoside formed by 1 μg of recombinant enzyme in 100 mM MES, pH 7.0, at 30 °C in the presence of 0.5 mM trans-zeatin. $E$ and $F$, the effect of DTT on the UGT activity was studied by the addition of various concentrations of DTT into the reactions containing 0.5 mM trans-zeatin, 100 mM MES, pH 7.0. The reactions described in $A$, $B$, $E$, and $F$ were carried out at 30 °C for 1 h. The results represent means ± S.D. from three replicates.

TABLE III  
The steady-state enzyme kinetic study of UGTs 76C1 and 76C2 towards five classical cytokinins

The steady-state kinetics of UGTs 76C1 and 76C2 were analyzed over a range of 0–0.5 mM cytokinins. The reactions were performed in 100 mM MES, pH 7.0, at 30 °C for 1 h. The kinetic parameters $K_m$ and $V_{\text{max}}$ were derived using the hyperbolic regression analysis of the Hyper32 programme available from homepage.ntworld.com/john.easterby, and the $k_{\text{cat}}$ values were calculated from the $V_{\text{max}}$ values. The results represent the means from two replicates ± range.

| Substrates           | UGT76C1       | UGT76C2       |
|----------------------|---------------|---------------|
|                      | $K_m$         | $V_{\text{max}}$ | $k_{\text{cat}}/K_m$ | $K_m$         | $V_{\text{max}}$ | $k_{\text{cat}}/K_m$ |
|                      | $mM$          | $nkat/mg$     | $mM^{-1}s^{-1}$      | $mM$          | $nkat/mg$     | $mM^{-1}s^{-1}$      |
| trans-Zeatin         | $0.24 \pm 0.02$ | $4.06 \pm 0.18$ | $1.32 \pm 0.05$     | $0.22 \pm 0.01$ | $5.91 \pm 0.00$ | $2.10 \pm 0.05$     |
| Dihydrozeatin        | $0.16 \pm 0.02$ | $6.34 \pm 0.26$ | $3.18 \pm 0.18$     | $0.22 \pm 0.01$ | $10.78 \pm 0.53$ | $3.73 \pm 0.02$     |
| $N^6$-Isopentenyladenine | $0.13 \pm 0.00$ | $18.67 \pm 0.19$ | $11.20 \pm 0.12$    | $0.07 \pm 0.01$ | $10.42 \pm 0.53$ | $10.14 \pm 0.25$    |
| $N^6$-Benzyladenine  | $0.09 \pm 0.00$ | $15.93 \pm 0.61$ | $14.64 \pm 0.31$    | $0.04 \pm 0.00$ | $7.90 \pm 0.94$  | $15.00 \pm 1.78$    |
| Kinetin              | $0.21 \pm 0.01$ | $9.40 \pm 1.25$ | $3.57 \pm 0.39$     | $0.07 \pm 0.01$ | $3.63 \pm 0.08$  | $4.26 \pm 0.23$     |

After filtration, the supernatant was collected, concentrated *in vacuo*, and analyzed with HPLC. The HPLC analysis was carried out as described above with some modifications. The methanol extract was analyzed with a linear gradient of methanol in H₂O (all solutions contained 2.5 ml/liter of glacial acetic acid and 0.4 ml/liter of triethylamine) at 1 ml/min from 10–100% over 40 min at 30 °C and monitored between 200 and 400 nm. The peaks corresponding to the trans-zeatin glucosides were integrated at 270 nm and were quantified using the

47828

Cytokinin N-Glucosylation

A

| Activity (nkat/mg) | UGT76C1 |
| --- | --- |
| 2.5 |  |
| 2 |  |
| 1.5 |  |
| 1 |  |
| 0.5 |  |
| 0 |  |

| Inhibitor concentration (mM) |  |
| --- | --- |
| 0 |  |
| 0.25 |  |
| 0.5 |  |
| 1 |  |
| 2 |  |

| 3-Isobutyl-1- |  |
| --- | --- |
| methylxanthine |  |
| Olomoucine |  |

| Activity (nkat/mg) | UGT76C2 |
| --- | --- |
| 2.5 |  |
| 2 |  |
| 1.5 |  |
| 1 |  |
| 0.5 |  |
| 0 |  |

| Inhibitor concentration (mM) |  |
| --- | --- |
| 0 |  |
| 0.25 |  |
| 0.5 |  |
| 1 |  |
| 2 |  |

| 3-Isobutyl-1- |  |
| --- | --- |
| methylxanthine |  |
| Olomoucine |  |

B

| UGT76C1 |  |
| --- | --- |
| 3-Isobutyl-1-methylxanthine |  |
| $K_{i} = 0.48 \, \text{mM}$ |  |
| 0.50 mM |  |
| 0.25 mM |  |
| No inhibitor |  |

|  |  |
| --- | --- |
| -25 |  |
| -1 |  |
| 0 |  |
| 25 |  |
| 50 |  |
| 75 |  |

| UGT76C2 |  |
| --- | --- |
| 3-Isobutyl-1-methylxanthine |  |
| $K_{i} = 0.21 \, \text{mM}$ |  |
| 0.50 mM |  |
| 0.25 mM |  |
| No inhibitor |  |

|  |  |
| --- | --- |
| -25 |  |
| -2 |  |
| 0 |  |
| 25 |  |
| 50 |  |
| 75 |  |

C

| UGT76C1 |  |
| --- | --- |
| Olomoucine |  |
| $K_{i} = 1.6 \, \mu\text{M}$ |  |
| 10 $\mu$M |  |
| 5 $\mu$M |  |
| No inhibitor |  |

|  |  |
| --- | --- |
| -4 |  |
| -2 |  |
| 0 |  |
| 2 |  |
| 4 |  |
| 6 |  |
| 8 |  |

| UGT76C2 |  |
| --- | --- |
| Olomoucine |  |
| $K_{i} = 2.5 \, \mu\text{M}$ |  |
| 10 $\mu$M |  |
| 5 $\mu$M |  |
| No inhibitor |  |

|  |  |
| --- | --- |
| -4 |  |
| -2 |  |
| 0 |  |
| 2 |  |
| 4 |  |
| 6 |  |
| 8 |  |

FIG. 4. Inhibition of the activity of UGTs 76C1 and 76C2. A, the inhibition of the activity of UGTs 76C1 and 76C2 was studied by addition of two potential inhibitors into the reactions containing 0.5 mM trans-zeatin and 100 mM MES, pH 7.0, incubated at 30°C for 1 h. The results represent the means ± range from two replicates. B, characterization of the inhibition of 3-isobutyl-1-methylxanthine to the UGT activity. The UGT activity was measured in the reactions containing 0–0.1 mM trans-zeatin, 1 mM DTT, 100 mM MES, pH 7.0, and 0.25 mM/0.50 mM 3-isobutyl-1-methylxanthine. C, characterization of the inhibition of olomoucine to the UGT activity. The reactions were set up as described in B except the inhibitor 3-isobutyl-1-methylxanthine was replaced by 5 $\mu$M/10 $\mu$M olomoucine, and the activity was measured with 0–0.3 mM trans-zeatin.

extinction coefficient of authentic glucosides. 0.1 mM $N^{6}$-isopentenyladenine was added as internal control at the beginning of the extraction to monitor the recovery rate.

RESULTS

Recombinant UGTs of Arabidopsis Recognize Cytokinins—Members of the multigene family of UGT sequences of Arabidopsis were expressed as recombinant fusion proteins in E. coli, purified by affinity chromatography and screened in vitro for activity against each of the five classical cytokinins: trans-zeatin, dihydrozeatin, $N^{6}$-benzyladenine, $N^{6}$-isopentenyladenine, and kinetin. Only five of the 105 sequences analyzed (supplemental table) were found to have activity toward the cytokinins. Two UGTs, 76C1 and 76C2, can glucosylate all of the individual cytokinins at either the $N^{7}$ or $N^{9}$ position, and three UGTs, 85A1, 73C5, and 73C1, can glucosylate the OH group on the $N^{6}$-side chain of trans-zeatin and dihydrozeatin. Representative profiles from standard HPLC methods are shown in Fig. 1 to illustrate the separation of the different glucosides of trans-zeatin and $N^{6}$-benzyladenine. The specific activity of the UGTs against the five substrates is summarized in Table I.

To gain insight into substrate recognition by these five enzymes, their activity toward a further 13 compounds was assayed, and results are shown in Table II. Interestingly, O-glucosylation of trans-zeatin greatly reduced the activity of the N-glucosyltransferases toward the substrate. In a similar manner, N-glucosylation of dihydrozeatin abolished activity of the O-glucosyltransferase, UGT85A1. These data imply that glu-

cosylation at one site on the cytokinin strongly influences glucosylation at the second site. In contrast to earlier studies on cytokinin O-glycosyltransferases from other plant species (27, 28), recombinant UGTs from Arabidopsis recognized both trans- and cis-zeatin in vitro.

Fig. 2A shows an SDS-PAGE analysis of the five UGTs capable of glucosylation of cytokinins in vitro. The molecular masses of the fusion proteins range between 66 and 84 kDa. Additional smaller sized polypeptides were found in the affinity-purified samples of the UGTs 76C1 and 76C2 fusion proteins. Following an in-gel tryptic digestion and by MALDI-MS analysis, these polypeptides were unambiguously identified as the bacterial chaperonin GroEL with a combined Mowse score of 1200 from a Mascot search against all entries of the NCBI nr data base. A total of 27 peptides from the protein were observed in the MALDI-MS spectrum of the tryptic digest, which together accounted for 66% of the amino acid sequence of the protein. Collision-induced dissociation-MS/MS spectra of six of these peptides gave Mascot MS/MS ion search scores with confidences greater than 99.99%, and an additional three gave scores with confidences greater than 99%. As described under “Experimental Procedures,” the glutathione S-transferase affinity matrix binding the UGT fusion protein-GroEL complexes was pretreated with 20 mM ATP and 20 mM MgCl₂ to remove the bacterial chaperonin prior to elution of the UGTs with the reduced form of glutathione. As shown in Fig. 2B, an incubation of 30 min was sufficient to remove the chaperonin, and the UGTs were used in the following kinetic analysis. No effect of the treatment on UGT activities was observed (data not shown).

**Steady-state Kinetics of the N-Glucosyltransferases UGTs 76C1 and 76C2—**The results in Fig. 3 summarize the optimization of reaction conditions for the N-glucosyltransferases. Both UGTs, 76C1 and 76C2, have a pH optimum of ~7.0, and their activities can be enhanced ~1.5-fold by the addition of 1 mM DTT. The reactions are linear up to 60 min. Under these conditions, both UGTs displayed high $k_{cat}/K_m$ values toward $N^6$-benzyladenine and $N^6$-isopentenyladenine (>10 mM⁻¹ s⁻¹) as shown in Table III. Taken together with their low $K_m$ values toward these two substrates, it is clear that $N^6$-benzyladenine and $N^6$-isopentenyladenine are the preferred substrates for UGTs 76C1 and 76C2 from those tested. The $K_m$ values of UGTs 76C1 and 76C2 toward $N^6$-benzyladenine, 0.09 and 0.04 mM, respectively, are similar to those reported in the study (28) of cis-ZOG1 and cis-ZOG2 toward cis-zeatin (0.046 and 0.096 mM, respectively). There are no kinetic data from other cytokinin-O-glycosyltransferases available for comparison.

**Inhibition of the Cytokinin N-Glucosyltransferases—**In early studies of the metabolism of leaf discs, co-incubation of substituted xanthines or olomoucine (6-benzylamino-2-[2-hydroxyethylamino]-9-methylpurine) with cytokinins was found to decrease the level of N-glucosides in tissue extracts (45–47). It was suggested that these compounds might directly inhibit the activity of the N-glucosyltransferase(s). Fig. 4 shows the effects of a substituted xanthine (3-isobutyl-1-methyloxanthine) and olomoucine on the activity of UGTs 76C1 and 76C2. It can be seen that both compounds inhibit the activities of UGTs (Fig. 4A). Lineweaver-Burk plots of the inhibition experiments showed that the trend lines intercepted on the y-axis, suggesting that these compounds are competitive inhibitors (Fig. 4, B and C). Comparing the $K_i$ values of the two inhibitors, olomoucine is clearly more effective. The $K_i$ value of olomoucine derived in this study is nearly identical to that reported previously when crude enzyme extracts were used to analyze N-glucosyltransferase activity (47).

**Analysis of Transgenic Arabidopsis Plants in Which UGT76C1 Is Constitutively Overexpressed—**The cauliflower mosaic virus 35S (CaMV35S) promoter was used to drive the expression of UGT76C1 cDNA in transgenic Arabidopsis. Eight independent homozygous lines were obtained, and 4-week-old aerial tissue was collected and subjected to Northern analysis. Fig. 5A shows that six transgenic lines (1/1, 4/1, 14/3, 15/1, 18/1, and 19/3) expressed high steady-state levels of UGT76C1 mRNA in comparison to those observed in the wild-type and empty vector control (Ri) plants. Two lines (3/1 and 6/4) showed negligible expression of UGT76C1. Leaves from the transgenic plants were harvested, extracted, and analyzed for enzyme activity toward trans-zeatin. The results are shown in Fig. 5B and demonstrate that steady-state levels of mRNA correlate with enzyme activity to form N-glucosides. No significant enzyme activity to form O-glucosides was detected in the crude extracts (data not shown). The consequences of applying exogenous cytokinin to the transgenic line overexpressing UGT76C1 was analyzed, and the results were compared with those obtained with the vector only control. Fig. 6A shows the HPLC traces of extracts from leaves harvested at 6 h following spray treatment with cytokinin or buffer only. The results indicate that there are negligible levels of the 7-N-glucoside in either the Ri control or the overexpressor after spraying with buffer. This indicates that overexpression of the UGT alone does not lead to increased level of product. On application of cytokinin, 7-N-glucoside can be detected in the Ri control, but the level was much greater in the transgenic line. Only trace levels of other glucosides were ever detected. Fig. 6B illustrates a time course of accumulation of the 7-N-glucoside and the effects of co-spraying the trans-zeatin with inhibitors of the N-glucosyltransferase. In the presence of either of the inhibi-

47830

Cytokinin N-Glucosylation

**A**

- **4/1 + buffer (6h)**
  - MAU
  - Minutes

- **Ri + buffer (6h)**
  - MAU
  - Minutes

- **4/1 + 0.5 mM trans-zeatin (6h)**
  - MAU
  - Minutes

- **Ri + 0.5 mM trans-zeatin (6h)**
  - MAU
  - Minutes

**B**

- **7-N-glucoside (nmol/g FW)**
  - Ri + 0.5 mM trans-zeatin
  - Ri + 0.5 mM trans-zeatin + 1 mM IMX
  - Ri + 0.5 mM trans-zeatin + 0.05 mM Olomoucine
  - 4/1 + 0.5 mM trans-zeatin
  - 4/1 + 0.5 mM trans-zeatin + 1 mM IMX
  - 4/1 + 0.5 mM trans-zeatin + 0.05 mM Olomoucine
  - Time: 0 h, 0.5 h, 6 h

FIG. 6. Exogenous application of trans-zeatin to transgenic plants overexpressing UGT76C1 cDNA. A, the HPLC trace (monitored at 270 nm) of the extracts from the transgenic line 4/1 and the control line Ri sprayed with 0.5 mM trans-zeatin or buffer (1% Me₂SO and 0.05% Tween 20). Peak 1, trans-zeatin-7-N-glucoside; Peak 2, trans-zeatin-9-N-glucoside and trans-zeatin-O-glucoside (overlapped); Peak 3, trans-zeatin applied exogenously; Peak 4, N⁶-isopentenyladenine added as an internal control at the beginning of the extraction process. The extracts were analyzed with a linear gradient of methanol in H₂O from 10–100% (all solutions contained 2.5 ml/liter of glacial acetic acid and 0.4 ml/liter of triethylamine) over 40 min at 1 ml/min and monitored at 270 nm. B, the trans-zeatin-7-N-glucoside levels in the plants sprayed with 0.5 mM trans-zeatin in the presence or absence of inhibitors (1 mM 3-isobutyl-1-methylxanthine (IMX) or 0.05 mM olomoucine).

tors, the level of 7-N-glucoside in the leaves of the control or those of the overexpressor line was substantially reduced.

**DISCUSSION**

Glucosylation of cytokinins is a well recognized modification that is thought to play an important role in hormonal homeostasis. Although O-glucosylation and N-glucosylation of cytokinins have been described and reviewed, only O-glucosylation has been investigated in depth because of the availability of genes encoding the enzymes and biochemical analyses of their recombinant products (3). Genes encoding these enzymes from several plant species have been identified by Mok and coworkers (24, 25, 27, 28), including ZOG1 from Phaseolus lunatus, ZOX1 from Phaseolus vulgaris, and cis-ZOG1 and cis-ZOG2 from maize. In contrast, nothing has been known of the N-glucosyltransferases of cytokinins, except early biochemical studies on partially purified enzyme preparations from a range of different plant species (29, 30, 35). We now report the identification of two UGTs from Arabidopsis that recognize the classical cytokinins and form the 7-N- and 9-N-glucosides within an in vitro assay. In addition, we also found three UGTs that form the O-glucoside with trans-zeatin, cis-zeatin, and dihydrozeatin. Although we do not know as yet whether cytokinins are the natural substrates of these enzymes in vivo, we have confirmed that the recombinant UGT76C1 can function in planta, continue to recognize cytokinin as a substrate, and synthesize the 7-N-glucoside. The genes we have identified can now be used in a number of genetic studies to define their endogenous substrates and, in particular, the consequences of N-glucosylation on cytokinin homeostasis.

Three N-glucosides of cytokinin have been described in the literature (29, 30, 34, 35, 48). These include the products of glucosylation at the N⁷ and N⁹ positions and at the N³ position. In this study, none of the 105 recombinant UGTs that were assayed produced a 3-N-glucoside with any of the cytokinins. Although 3-N-glucosides have been reported in some plant species such as radish (34), to our knowledge their existence in Arabidopsis has not been investigated previously.

The UGTs, 76C1 and 76C2, catalyzed the formation of both the 7-N- and the 9-N-glucoside of each of the cytokinins analyzed. This observation may be due to the known tautomerism of adenine, in which a hydrogen atom attaches either to the N⁷ (7-H tautomer) or the N⁹ position (9-H tautomer) of the molecule. In these circumstances in aqueous solutions, the 7-H tautomer is the favored state (49) leading to an increased

Cytokinin N-Glucosylation

hydroxybenzoic acids
hydroxybenzoic acids
caffeic acid
cytokinins
ZOG1
ZOX1
cisZOG1
cisZOG2
flavonoids
*cytokinins
monolignols
flavonoids
cytokinins*
salicylic acid
indole-3-acetic acid
hydroxycinnamic acids
hydroxybenzoic acids

FIG. 7. Phylogenetic tree of the amino acid sequences of Arabidopsis UGTs and other cytokinin-O-glycosyltransferases. The phylogeny of 107 Arabidopsis UGTs and 4 cytokinin-O-glycosyltransferases reported previously (ZOG1, accession number AF101972; ZOX1, accession number AF116858; cis-ZOG1, accession number AF318075; cis-ZOG2, accession number AY082660) (24, 25, 27, 28) was analyzed by the neighbor-joining method using the distance matrix derived from the multiple alignment of CLUSTAL W (EMBL, Heidelberg, Germany). The phylogenetic tree was drawn using the software PhyloDraw version 0.8 (Graphics Application Lab., Pusan National University, South Korea). The UGT activities reported in the literature (41, 52, 54, 55, 59), except those conjugating the model substrates hydroxycoumarins (53), are shown in bold type. The asterisks indicate the UGTs reported in this study.

availability of the 7-N for glucosylation by the UGTs. This may be the explanation for the higher levels of the 7-N-glucoside produced by the two UGTs *in vitro.*

N-Glucosylation has attracted considerable interest over many years, because it is considered to be a major barrier to the successful use of cytokinins in field applications (45, 46). As a consequence there have been a number of studies to investigate the ways in which the process could be inhibited (45–47). For example, the efficacy of several structurally related compounds were assayed in tissue extracts and led to the identification of substituted xanthines as useful inhibitors. It was speculated that compounds such as those would act as inhibitors of the N-glucosyltransferases, but this could not be proven because of the lack of purified enzyme(s). We now confirm, using the recombinant UGTs, that both 3-isobutyl-1-methylxanthine and olomoucine are highly effective inhibitors of N-glucosyltransferases.

Among the five UGTs reported in this study, only UGTs 76C1 and 76C2 are closely related with 60% identity at the amino acid level. When these sequences are compared with the enzymes identified in Zea and Phaseolus by Mok and co-workers (24, 25, 27, 28), only low sequence similarity was observed (<30%). The Zea and Phaseolus UGTs form a unique branch on the phylogenetic tree containing 107 Arabidopsis UGTs (Fig. 7). It is possible that the Zea and Phaseolus UGTs evolved from a common ancestor distinct from those identified in Arabidopsis (39, 40). Within the Arabidopsis UGT family, the UGTs capable of glucosylating cytokinins have evolved from three separate branches. To date, the two branches containing UGTs

76C1, 76C2, and 85A1 have not been associated with any catalytic activity. In contrast, UGTs 73C1 and 73C5 evolved from the branch that consists of UGTs that recognize a number of metabolites. For example, UGT73C6 has been reported as a flavonoid-7-O-glycosyltransferase (58). The UGTs in the same branch are also capable of conjugating the model substrates hydroxycoumarins *in vitro* (53). In addition, UGT73C5 has been described in another recent study as a glycosyltransferase responsible for fungal toxin detoxification *in planta* (51). It is possible UGT73C5 recognizes multiple endogenous compounds and xenobiotics including cytokinins, flavonoids, hydroxycoumarins, and fungal toxins. Alternatively, the *in vitro* analysis may only reflect catalytic activity toward a structural feature(s) common to a range of secondary metabolites.

Recent studies have analyzed the effects of cytokinin application on global gene expression using a DNA microarray strategy (50). Expression profiling using an 8300-element Arabidopsis Affychip examined cytokinin-responsive genes over a 24-h time period. The expression of the cytokinin oxidase genes was significantly up-regulated in these experiments. Only two of the UGTs described in this report were represented in the Affychip: UGTs 73C5 and 73C1, and the expression of neither of these genes was up-regulated.

Our studies have revealed a number of UGTs with catalytic activities toward important plant metabolites such as hydroxycinnamates, hydroxybenzoates, and hormones, such as auxin (41, 44, 52–55). The constitutive overexpression of these UGTs has been found to lead to a diversity of effects, dependent on the aglycone that is glucosylated (54, 56). For example, when

UGT84B1 recognizing indole-3-acetic acid (IAA) was constitutively overexpressed, the transgenic lines showed many features typical of an auxin-deficient phenotype (56). The lack of gravitropism displayed by the root system of these plants was recovered by application of an auxin analogue 2,4-D, which was not glucosylated by the UGT (55, 56). Metabolite profiling of the transgenic lines confirmed very high levels of the IAA-glucose ester, but levels of IAA were also elevated, suggesting that changing the IAA conjugate levels in the plant caused a complex set of events to occur (56). As yet, we have only a preliminary analysis of the transgenic lines described in this study overexpressing the UGT76C1. However, under normal growth conditions, overexpression of the cytokinin N-glucosyltransferase seemingly has little effect on phenotype, even though the enzyme was confirmed to be active *in planta* and able to conjugate exogenous cytokinin. In contrast, the studies of Mok and co-workers clearly showed a distinct phenotype when an O-glucosyltransferase from *P. lunatus* (24) was overexpressed constitutively in tobacco callus (57). The O-glucoside was found to accumulate and the cell cultures required 10-fold higher concentrations of *trans*-zeatin for the induction of differentiation. Detailed analyses of transgenic *Arabidopsis* lines in which the five glycosyltransferases of cytokinins are silenced or overexpressed are now possible and will provide greater insight into the role of glucosylation in cytokinin homeostasis.

Acknowledgment—We thank Dr. Jerry Thomas (Technology Facility, Department of Biology, York, UK) for help with MALDI-MS analysis of GroEL.

**REFERENCES**

1. Miller, C. O., Skoog, F., von Saltza, M. H., and Strong, M. (1955) *J. Am. Chem. Soc.* **77**, 1329–1334
2. Skoog, F., and Miller, C. O. (1957) *Symp. Soc. Exp. Biol.* **11**, 118–131
3. Mok, D. W. S., and Mok, M. C. (2001) *Annu. Rev. Plant Physiol. Plant Mol. Biol.* **52**, 89–118
4. Gaspar, T., Kevers, C., Penel, C., Greppin, H., Reid, D. M., and Thorpe, T. A. (1996) *In Vitro Cell. Dev. Biol. Plant* **32**, 272–289
5. Hutchison, C. E., and Kieber, J. J. (2002) *Plant Cell* **14**, S47–S59
6. Sheen, J. (2002) *Science* **296**, 1650–1652
7. Kakimoto, T. (2003) *Annu. Rev. Plant Biol.* **54**, 605–627
8. Binns, A. N. (1994) *Annu. Rev. Plant Physiol. Plant Mol. Biol.* **45**, 173–196
9. Coenen, C., and Lomax, T. L. (1997) *Trend Plant Sci.* **2**, 351–356
10. Vogel, J. P., Schuerman, P., Woeste, K., Brandstatter, J., and Kieber, J. J. (1998) *Genetics* **149**, 417–427
11. Auer, C. A. (2002) *J. Plant Growth Regul.* **21**, 24–31
12. Skoog, F., and Armstrong, D. J. (1970) *Annu. Rev. Plant Physiol.* **21**, 359–384
13. Haberer, G., and Kieber, J. J. (2002) *Plant Physiol.* **128**, 354–362
14. Kakimoto, T. (2003) *J. Plant Res.* **116**, 233–239
15. Kakimoto, T. (2001) *Plant Cell Physiol.* **42**, 677–685
16. Takei, K., Sakakibara, H., and Sugiyama, T. (2001) *J. Biol. Chem.* **276**, 26405–26410
17. Jones, R. J., and Schreiber, B. M. N. (1997) *Plant Growth Regul.* **23**, 123–134
18. Schmülling, T., Werner, T., Riefler, M., Krupková, E., and Manns, I. B. Y. (2003) *J. Plant Res.* **116**, 241–252
19. Eklöf, S., Åstot, C., Moritz, T., Blackwell, J., Olsson, O., and Sandberg, G. (1996) *Physiol. Plant.* **98**, 333–344
20. Chang, H., Jones, M. L., Banowetz, G. M., and Clark, D. G. (2003) *Plant Physiol.* **132**, 2174–2183
21. Werner, T., Motyka, V., Laucou, V., Smets, R., Onckelen, H. V., and Schmülling, T. (2003) *Plant Cell* **15**, 2532–2550
22. Sun, J., Niu, Q.-W., Tarkowski, P., Zheng, B., Tarkowska, D., Sandberg, G.,
Chua, N. H., and Zuo, J. (2003) *Plant Physiol.* **131**, 167–176
23. Dixon, S. C., Martin, R. C., Mok, M. C., Shaw, G., and Mok, D. W. S. (1989) *Plant Physiol.* **90**, 1316–1321
24. Martin, R. C., Mok, M. C., and Mok, D. W. S. (1999) *Proc. Natl. Acad. Sci. U.S.A.* **96**, 284–289
25. Martin, R. C., Mok, M. C., and Mok, D. W. S. (1999) *Plant Physiol.* **120**, 553–557
26. Mok, D. W. S., Martin, R. C., Shan, X., and Mok, M. C. (2000) *Plant Growth Regul.* **32**, 285–287
27. Martin, R. C., Mok, M. C., Habben, J. F., and Mok, D. W. S. (2001) *Proc. Natl. Acad. Sci. U.S.A.* **98**, 5922–5926
28. Veach, Y. K., Martin, R. C., Mok, D. W. S., Malbeck, J., Vankova, R., and Mok, M. C. (2003) *Plant Physiol.* **131**, 1374–1380
29. Entsch, B., and Letham, D. S. (1979) *Plant Sci. Lett.* **14**, 205–212
30. Entsch, B., Parker, C. W., Letham, E. S., and Summons, R. E. (1979) *Biochim. Biophys. Acta* **570**, 124–139
31. Campos, N., Bakó, L., Feldwisch, J., Schell, J., and Palme, K. (1992) *Plant J.* **2**, 675–684
32. Brzobohatý, B., Moore, I., Kristoffersen, P., Bako, L., Campos, N., Shell, J., and Palme, K. (1993) *Science* **262**, 1051–1054
33. Cowley, D. E., Duke, C. C., Liepa, A. J., MacLeod, J. K., and Letham, D. S. (1978) *Aust. J. Chem.* **31**, 1095–1111
34. Letham, D. S., Goodwin, P. B., and Higghins, T. J. (1978) *Phytohormones and Related Compounds: A Comprehensive Treatise*, Vol. 1, pp. 29–105, Elsevier/North Holland, Amsterdam
35. Letham, D. S., and Palni, L. M. S. (1983) *Annu. Rev. Plant Physiol.* **34**, 163–197
36. Campbell, J. A., Davies, G. J., Bulone, V., and Henrissat, B. (1997) *Biochem. J.* **326**, 929–942
37. Coutinho, P. M., Deleury, E., Davies, G. J., and Henrissat, B. (2003) *J. Mol. Biol.* **328**, 307–317
38. Vogt, T., and Jones, P. (2000) *Trends Plant Sci.* **5**, 380–386
39. Li, Y., Baldauf, S., Lim, E.-K., and Bowles, D. J. (2001) *J. Biol. Chem.* **276**, 4338–4343
40. Ross, J., Li, Y., Lim, E.-K., and Bowles, D. J. (2001) *Genome Biol.* **2**, 3004.1–3004.6
41. Lim, E.-K., Li, Y., Parr, A., Jackson, R., Ashford, D. A., and Bowles, D. J. (2001) *J. Biol. Chem.* **276**, 4344–4349
42. Bechtold, N., Ellis, J., and Pelletier, G. (1993) *C. R. Acad. Paris Life Sci.* **316**, 1194–1199
43. Verwoerd, T. C., Dekker, B. M. M., and Hoekema, A. (1989) *Nucleic Acids Res.* **17**, 2362
44. Lim, E.-K., and Bowles, D. J. (2004) *EMBO J.* **23**, 2915–2922
45. Parker, C. W., Entsch, B., and Letham, D. S. (1986) * Phytochemistry* **25**, 303–310
46. Hocart, C. H., Letham, D. S., and Parker, C. W. (1991) * Phytochemistry* **30**, 2477–2486
47. Tao, G. Q., Letham, D. S., Hocart, C. H., and Summons, R. E. (1991) *J. Plant Growth Regul.* **10**, 179–185
48. Summons, R. E., MacLeod, J. K., Parker, C. W., and Lethoam D. S. (1977) *FEBS Lett.* **82**, 211–214
49. Broo, A., and Holmén, A. (1996) *Chem. Phys.* **211**, 147–161
50. Rashotte, A. M., Carson, S. D. B., To, J. P. C., and Kieber, J. J. (2003) *Plant Physiol.* **132**, 1998–2011
51. Poppenberger, B., Berthiller, F., Lucyshyn, D., Sieberer, T., Schuhmacher, R., Kraska, R., Kuchler, K., Glössl, J., Luschnig, C., and Adam, G. (2003) *J. Biol. Chem.* **278**, 47905–47914
52. Lim, E.-K., Doucet, C. J., Elias, L., Worrall, D., Li, Y., Ross, J., and Bowles, D. J. (2002) *J. Biol. Chem.* **277**, 586–592
53. Lim, E.-K., Baldauf, S., Li, Y., Elias, L., Worrall, D., Spencer, S. P., Jackson, G. J., Taguchi, G., Ross, J., and Bowles, D. J. (2003) *Glycobiology* **13**, 139–145
54. Lim, E.-K., Higgins, G. S., Li, Y., and Bowles, D. J. (2003) *Biochem. J.* **373**, 987–992
55. Jackson, R. G., Lim, E.-K., Li, Y., Kowalczyk, M., Sandberg, G., Hoggett, J., Ashford, D. A., and Bowles, D. J. (2001) *J. Biol. Chem.* **276**, 4350–4356
56. Jackson, R. G., Kowalczyk, M., Li, Y., Higgins, G., Ross, J., Sandberg, G., and Bowles, D. J. (2002) *Plant J.* **32**, 573–583
57. Martin, R. C., Mok, D. W. S., Smets, R., Onckelen, H. A. V., and Mok, M. C. (2001) *In Vitro Cell. Dev. Biol. Plant* **37**, 354–360
58. Jones, P., Messner, B., Nakajima, J.-I., Schäffner, A., and Saito, K. (2003) *J. Biol. Chem.* **278**, 43910–43918

Enzyme Catalysis and Regulation:  
N-Glucosylation of Cytokinins by  
Glycosyltransferases of Arabidopsis  
thaliana  

Bingkai Hou, Eng-Kiat Lim, Gillian S.  
Higgins and Dianna J. Bowles  
J. Biol. Chem. 2004, 279:47822-47832.  
doi:10.1074/jbc.M409569200 originally published online September 1, 2004  

Access the most updated version of this article at doi: 10.1074/jbc.M409569200  

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/affinity-sites).

Alerts:
- [When this article is cited](https://www.jbc.org/alerts)
- [When a correction for this article is posted](https://www.jbc.org/alerts)

[Click here](https://www.jbc.org/alerts) to choose from all of JBC's e-mail alerts

Supplemental material:  
[http://www.jbc.org/content/suppl/2004/09/16/M409569200.DC1.html](http://www.jbc.org/content/suppl/2004/09/16/M409569200.DC1.html)

This article cites 58 references, 26 of which can be accessed free at  
[http://www.jbc.org/content/279/46/47822.full.html#ref-list-1](http://www.jbc.org/content/279/46/47822.full.html#ref-list-1)
