
THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2004 by The American Society for Biochemistry and Molecular Biology, Inc.
Vol. 279, No. 49, Issue of December 3, pp. 50717–50725, 2004
Printed in U.S.A.

# Desulfoglucosinolate Sulfotransferases from *Arabidopsis thaliana* Catalyze the Final Step in the Biosynthesis of the Glucosinolate Core Structure*

Received for publication, July 8, 2004, and in revised form, September 3, 2004
Published, JBC Papers in Press, September 9, 2004, DOI 10.1074/jbc.M407681200

Markus Piotrowski‡, Andreas Schemenewitz‡, Anna Lopukhina‡, Axel Müller‡, Tim Janowitz‡, Elmar W. Weiler‡§, and Claudia Oecking¶

From the ‡Department of Plant Physiology, Ruhr-Universität, Universitätsstrasse 150, 44801 Bochum, Germany and ¶Plant Physiology, Center for Plant Molecular Biology, University of Tübingen, Auf der Morgenstelle 5, 72076 Tübingen, Germany

The phytoxin coronatine is a structural analog of octadecanoid signaling molecules, which are well known mediators of plant defense reactions. To isolate novel coronatine-regulated genes from *Arabidopsis thaliana*, differential mRNA display was performed. Transcript levels of CORI-7 (coronatine induced-7) were rapidly and transiently increased in coronatine-treated plants, and the corresponding cDNA was found to encode the sulfotransferase AtST5a. Likewise, upon wounding, an immediate and transient increase in AtST5a mRNA levels could be observed in both locally wounded and unwounded (systemic) leaves. Furthermore, application of octadecanoids and ethylene as compounds involved in plant wound defense reactions resulted in AtST5a gene activation, whereas pathogen defense-related signals (yeast elicitor and salicylic acid) were inactive. AtST5a and its close homologs AtST5b and AtST5c were purified as His₆-tagged proteins from *Escherichia coli*. The three enzymes were shown to catalyze the final step in the biosynthesis of the glucosinolate (GS) core structure, the sulfation of desulfoglucosinolates (dsGSs). They accept a broad range of dsGSs as substrates. However, in a competitive situation, AtST5a clearly prefers tryptophan- and phenylalanine-derived dsGSs, whereas long chain dsGSs derived from methionine are the preferred substrates of AtST5b and AtST5c. Treatment of *Arabidopsis* plants with low concentrations of coronatine resulted in an increase in the amounts of specific GSs, primarily glucobrassicin and neoglucobrassicin. Hence, it is suggested that AtST5a is the sulfotransferase responsible for the biosynthesis of tryptophan-derived GSs *in vivo*.

of sulfotransferases are known in prokaryotes as well as eukaryotes; however, the study of enzymes that catalyze the sulfation reaction in plants considerably lags behind that in animal systems. Cytosolic sulfotransferases from plants have been characterized in some detail (Ref. 1 and references therein), and some cDNAs have been identified. These fall into three subgroups: the flavonol sulfotransferases described for *Flaveria* species (1), the steroid sulfotransferases identified in *Brassica napus* (2, 3), and a hydroxyjasmonic acid-specific sulfotransferase from *Arabidopsis thaliana* (4). An additional sulfotransferase (RaR047, At2g03760) has been cloned from *A. thaliana*, and its mRNA level was found to be up-regulated by pathogens and salicylic acid; however, its physiological substrate is still unknown (5).

In plants, sulfate groups occur in a number of secondary metabolites, notably the sulfoflavonoids (6) and the glucosinolates (7). Glucosinolates (GSs) are secondary compounds found in at least 16 different plant families, 15 of which belong to the order Capparales (for review, see Ref. 8). Within this order, much interest has been directed to the Brassicaceae family: the genus *Brassica* alone contains a large number of agriculturally important crops, including many vegetables (e.g. broccoli, Brussels sprouts, cauliflower, and cabbage) and one of the most important oilseed crops, oilseed rape (*B. napus*), the defatted seed meal of which is fed to animals. GSs in edible species or seed meal have attracted much attention because their breakdown products have been described to have anticancer but also goitrogenic and anti-nutritional activities.

Although many functions such as sulfur and nitrogen storage have been assigned to GSs, defense against herbivores and pathogens seems to be their main function. Upon wounding, GSs are hydrolyzed by a thioglucosidase called myrosinase, and the released unstable aglycons rearrange to form isothiocyanates, thiocyanates, nitriles, and other compounds, the production of which depends on the GS itself, the reaction conditions, and the presence of certain cofactors (for review, see Ref. 9). These compounds have antimicrobial activity and are toxic or deterrent to non-specialist herbivores.

Much progress has been made recently in identifying the gene products involved in the biosynthesis of GSs (Fig. 1): aldoxime-forming and aldoxime-oxidizing cytochrome P450 enzymes characterized by different substrate specificities have been identified (10–19). C-S lyase, the enzyme catalyzing the subsequent step in GS biosynthesis, was identified recently as the SUPERROOT1 gene product (SUR1) (20), and the gene

HPLC, high performance liquid chromatography; MeJA, jasmonic acid methyl ester; I3M, indole-3-methyl; 8MTO, 8-methylthiooctyl.

* This work was supported in part by Deutsche Forschungsgemeinschaft Grants SPP1152 (to M. P.) and SFB446-A18 (to C. O.). The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
§ To whom correspondence should be addressed. Tel.: 49-234-322-4291; Fax: 49-234-321-4187; E-mail: Elmar.Weiler@ruhr-unibochum.de.
¹ The abbreviations used are: PAPS, 3′-phosphoadenosine 5′-phosphosulfate; GS, glucosinolate; dsGS, desulfoglucosinolate; dsGS-ST, desulfoglucosinolate:PAPS sulfotransferase; RT, reverse transcription;

This paper is available on line at http://www.jbc.org

50718

# Desulfoglucosinolate Sulfotransferases from A. thaliana

## Fig. 1. Biosynthesis of the GS core structure in A. thaliana.

The initial substrate is either a proteinogenic amino acid or a chain- elongated amino acid. Gluc, glucose.

(encoding) UDP-glucose:thiohydroximate glycosyltransferase was cloned from *B. napus* (21). The final step in the biosynthesis of the GS core structure is catalyzed by a desulfoglucosinate:PAPS sulfotransferase (dsGS-ST), transferring the sulfate moiety from PAPS to the desulfoglucosinate (dsGS). The enzymatic activity of dsGS-STs has been analyzed in partially purified protein fractions from *Brassica juncea* and cress (*Lepidium sativum*) (22, 23). The enzymes have similar biochemical characteristics with respect to native molecular mass, isoelectric point, pH, and temperature optima as well as inhibition by various sulfhydryl group reagents. Furthermore, the enzyme from cress was found to prefer desulfobenzylglucosinolate over desulfosinigrin. However, the corresponding proteins have not been purified to homogeneity; and until now, genes encoding dsGS-STs have not been identified. Here, we report the cloning and functional expression of a small family of such desulfoglucosinolate-specific sulfotransferases from *A. thaliana*.

### EXPERIMENTAL PROCEDURES

#### Chemicals—High quality PAPS (95% minimum) was obtained from Prof. H. R. Glatt (German Institute of Human Nutrition, Berholz-Rehbruecke, Germany). 12-Hydroxyjasmonic acid was a kind gift from Dr. C. Wasternack (Institute of Plant Biochemistry, Halle an der Saale, Germany). 11-Hydroxyjasmonic acid was purified from suspension cultures of *Eschscholtzia californica* as described by Xia and Zenk (24). Synthesis of \[^{13}C_2\]Glucobrassicin was as described by Müller and

#### Weiler (25). Sinigrin (allylglucosinolate) and benzylglucosinolate were obtained from Acros Organics (Geel, Belgium) and Merck Biosciences (Schwalbach, Germany), respectively.

#### Plant Material—*A. thaliana* ecotypes C24 and Col-0 were grown in a greenhouse. Prior to treatment, plants were transferred to a phytotron chamber (8-h photoperiod at 20 °C and 150 μmol of photons (400–700 nm) cm⁻² s⁻¹; 16 h of darkness at 18 °C; 70% relative humidity) for at least 3 days.

#### Treatment of Plants—Wounding was done using a hemostat and crushing across ~80–100% of the area of a specific leaf. The leaves of one-half of a rosette were treated in this way; the opposite half of the rosette was left undamaged to allow differentiation of local and systemic effects. Application of compounds dissolved in 40% (v/v) acetone and 0.1% (v/v) Tween 20 was performed by spraying plant rosettes until leaves were wet. Control plants were treated with the solvent alone. Plant material was frozen in liquid nitrogen immediately after harvesting.

#### General Molecular Biological and Biochemical Methods—If applicable, the standard protocols of Ausubel et al. (26) and Sambrook and Russell (27) were used. Plant RNA was prepared according to Barkan (28). Ten micrograms of total RNA were separated on formaldehyde-agarose gels, transferred onto nylon membranes, and hybridized with a radiolabeled probe using standard laboratory procedures. The probe for AtST5a mRNA corresponded to nucleotides 22–470 of the AtST5a cDNA. cDNA synthesis, PCR amplification, and DNA restriction and ligation followed the manufacturers’ protocols.

#### Semiquantitative Reverse Transcription (RT)-PCR—Semiquantitative RT-PCR was done using the cMaster RTplus PCR system (Eppendorf, Wesseling-Berzdorf, Germany) following the manufacturer’s instructions. Contaminating DNA in the RNA preparations was digested with the RQ1 RNase-free DNase (Promega, Mannheim, Germany). This step was critical because the AtST5 genes contain no introns, and amplification could therefore result from genomic DNA. Absence of

Desulfoglucosinolate Sulfotransferases from A. thaliana

**TABLE I**
Overview of sulfotransferases mentioned in this work

| AGI code/GenBank™ accession no. | Organism          | Referred to as         | Substrate                                      | Ref. |
|----------------------------------|-------------------|------------------------|-------------------------------------------------|------|
| At1g18590                        | A. thaliana       | AtST5c                | dsGSs, preferentially Met-derived               | This work |
| At1g74090                        | A. thaliana       | AtST5b                | dsGSs, preferentially Met-derived               | This work |
| At1g74100                        | A. thaliana       | AtST5a (CORI-7)       | dsGSs, preferentially Phe- and Trp-derived      | This work |
| At2g03760                        | A. thaliana       | AtST1 (RaR047)        | Unknown                                        | 5     |
| At5g07000                        | A. thaliana       | AtST2b                | Unknown                                        | 4     |
| At5g07010                        | A. thaliana       | AtST2a                | Hydroxyjamonates                               | 4     |
| AF000305                         | B. napus          | BnSST1                | Unknown                                        | 2     |
| AF000306                         | B. napus          | BnSST2                | Unknown                                        | 2     |
| AF000307                         | B. napus          | BnSST3                | Brassinosteroids                               | 2     |
| U10277                           | Flaveria bidentis | FbFST-L               | Unknown                                        | 47    |
| U10275                           | F. bidentis       | FbF3ST                | Quercetin                                      | 48    |
| M84135                           | Flaveria chloraeolia | FcF3ST              | Quercetin                                      | 49    |
| M84136                           | F. chloraeolia    | FcF4'ST               | Quercetin 3-sulfate                            | 49    |
| AAC95519                         | Homo sapiens      | SULT1C2               | p-Nitrophenol/N-hydroxy-2-acetylaminoﬂuorene   | 41    |
|                                  | B. juncea         |                      | Desulfo benzylglucosinolate                    | 23    |
|                                  | L. sativum        |                      | Desulfo benzylglucosinolate >> desulfo sinigrin | 22    |

---

**FIG. 3. Expression of AtST5 genes analyzed by Northern blotting (A and B) and RT-PCR (C). Treatment of plants and extraction of RNA are described under “Experimental Procedures.” Autoradiograms are shown in A and B. An 18S rRNA probe was used as the control. In C, the ethidium bromide-stained PCR fragments are shown. The bands of the actin-1 gene were used to normalize the measured band intensities. JA, jasmonic acid; 12-OPDA, 12-oxophytodienoic acid; ACC, 1-aminocyclopropane-1-carboxylic acid; ctrl., control; Cor., coronatine.**

DNA after digestion was confirmed by PCR using the RNA as template. Specificities of AtST5 primers were tested using the respective plasmid DNA as template. RT-PCR conditions were as follows: 94 °C for 5 min; 25–30 cycles at 94 °C for 30 s, 55 °C for 45 s, and 72 °C for 60 s; followed by a final extension at 72 °C for 10 min. After electrophoretic separation, PCR products were visualized by ethidium bromide staining and quantified using the TINA 2.0 program (Raytest, Straubenhardt, Germany). The data were normalized to signals obtained by RT-PCR of the actin-1 gene.

Cloning and Bacterial Expression of cDNAs Encoding Selected Sulfotransferases from Arabidopsis—The AtST1 cDNA (plasmid pRaR047) was kindly provided by Dr. D. Roby (Centre National de la Recherche Scientifique/Institut National de la Recherche Agronomique, Castanet Tolosan, France) (5). The AtST5a gene was originally identified by differential mRNA display as a coronatine-induced gene (CORI-7) in A. thaliana (29). The cDNA fragment obtained encompassed 514 bp and was identified to be part of the Arabidopsis expressed sequence tag clone OBO154 (GenBank™/EBI accession number F14418). Nevertheless, this 1165-bp expressed sequence tag clone still displayed an incomplete 5′-end. The missing sequence was generated by 5′-rapid amplification of cDNA ends/PCR (30) using cDNA from coronatine (5 μM, 2 h)-treated plants as template. Meanwhile, the corresponding genomic sequence of AtST5a (AGI code At1g74100) was obtained within the scope of the Arabidopsis Genome Project.

Desulfoglucosinolate Sulfotransferases from *A. thaliana*

were studied in more detail, and up-regulated genes were tentatively assigned as CORI (coronatine induced). The sequence of the amplified cDNA fragment of CORI-7 (514 bp) was identical to that of the *A. thaliana* At1g74100 gene, coding for a sulfotransferase family member. In *A. thaliana*, 18 genes for soluble sulfotransferases are known (Fig. 2) (35). One gene, At3g51210 (omitted from Fig. 2), seems to represent a pseudogene encoding a polypeptide that consists simply of 67 amino acids corresponding to the C-terminal region of sulfotransferases; and up to now, no expressed sequence tag clone has been present in the data bases. Phylogenetic analysis revealed that CORI-7 forms a small subfamily together with two other sulfotransferases encoded by the At1g18590 and At1g74090 genes (Fig. 2 and Table I). According to the nomenclature introduced by Marsolais *et al.* (3, 36), these sulfotransferases are called AtST5a (CORI-7), AtST5b (At1g74090), and AtST5c (At1g18590). This small subfamily is orthologous to the flavonol sulfotransferase family known from two different *Flaveria* species (Fig. 2). The closest relatives to this subfamily in *A. thaliana* are AtST2a and AtST2b (encoded by the At5g07010 and At5g07000 genes, respectively). AtST2a has been identified recently as hydroxyjasmonate sulfotransferase, specifically sulfating 11- and 12-hydroxyjasmonate (4). However, hydroxyjasmonates are very unlikely substrates for the AtST5 subfamily of sulfotransferases since even AtST2b, the closest homolog of AtST2a, is not able to sulfate these compounds (4).

**FIG. 4. Expression and purification of N-terminally RGS-His<sub>6</sub>-tagged AtST5 enzymes.** Ten microliters each of the bacterial crude extract (*CE*), the column flow-through (*FT*), and the combined eluate (*EL*) fractions were loaded onto a 12.5% SDS-polyacrylamide gel.

For expression of *Arabidopsis* sulfotransferases as N-terminally RGS-His<sub>6</sub>-tagged proteins in *Escherichia coli* M15, the corresponding cDNA regions were amplified from plasmid templates (among others, the pUNI51 clones U50569 and U50309, containing the cDNAs encoding AtST5b and AtST5c, respectively) by PCR and cloned into pQE-30 (QIAGEN, Hilden, Germany) using the KpnI/PstI (AtST5a) and BamHI/SalI (AtST1, AtST5b, and AtST5c) restriction sites.<sup>2</sup> The recombinant RGS-His<sub>6</sub>-tagged proteins were purified under native conditions using nickel-nitrilotriacetic acid-agarose according to the manufacturer's protocol (QIAGEN). The eluted protein was immediately desalted using Sepharose G-25 (PD-10 columns, Amersham Biosciences) pre-equilibrated with 50 mM Tris (pH 7.0) and 1 mM dithiothreitol, shock-frozen in liquid nitrogen, and stored at −80°C.

Assay for Sulfotransferase Activity—Assays for sulfotransferase activity of the recombinant proteins were performed at 30°C in a total volume of 100 μl of 50 mM potassium phosphate buffer (pH 7.0) containing PAPS (20–200 μM), substrate (20–200 μM dsGS), and purified protein (10 μg). dsGSs were extracted from the leaves and seeds of *A. thaliana* ecotypes Col-0 and C24 and purified by HPLC as described below. Controls were identical but contained no protein. The reactions were stopped by the addition of 300 μl of ice-cold methanol to precipitate the protein. At that point, protein was also added to the controls to ensure comparable conditions between samples and controls in the subsequent steps. After 2 h at −80°C, the precipitated protein was collected by centrifugation, and the supernatant was taken to dryness and subsequently resuspended in 100 μl of water. Insoluble material was precipitated by an additional centrifugation, and the resulting supernatant was analyzed by reverse-phase HPLC. HPLC was performed on a Hyperclone 5μ BDS C18–1 (1 × 250 mm; Phenomenex, Aschaffenburg, Germany) using a LaChrom Elite HPLC workstation equipped with a diode array detector (VWR International, Darmstadt, Germany). HPLC conditions were as described elsewhere (31).

Preparation and Analysis of Glucosinolates in *A. thaliana*—Extraction of GSs from plant material and purification as dsGSs were performed as described by Brown *et al.* (32). For quantification, 1.25 μmol of benzylglucosinolate were added as an internal standard at the beginning of the extraction procedure. dsGSs were separated by HPLC as described (31). Individual dsGSs were identified by their UV-visible spectra and quantified by comparison with the internal standard. These values were corrected for the response factors for the different dsGSs. A list of these response factors is given by Brown *et al.* (32).

---

RESULTS

Identification of a Coronatine-induced Sulfotransferase—The structure of the phytotoxin coronatine is comparable with those of the octadecanoids 12-oxophytodienoic acid and jasmonic acid (33). Differential mRNA display (34) was performed to identify novel coronatine-regulated genes in *A. thaliana* (29). For this purpose, mRNA from coronatine (5 μM, 2 h)- or solvent-treated plants was applied. Eight differentially regulated genes

<sup>2</sup> Primer sequences are available upon request.

Desulfoglucosinolate Sulfotransferases from A. thaliana

FIG. 5. Activity measurements of Arabidopsis dsGS-STs and AtST1 by HPLC. In A, typical chromatograms are shown. Arrowheads point to new peaks representing reaction products. In B and C, the quantified residual substrate amounts are shown. A mixture of 2 nmol of each purified dsGSs was used. 4MTB-, 4-methylthiobutyl-; Bz-, benzyl-; 7MTH-, 7-methylthioheptyl-; 8MSOO-, 8-methylsulfinyloctyl-; DTT, dithiothreitol; mAU, milli-absorbance units; ctrl, control.

TABLE II  
Biosynthetic origins of several dsGSs used in this work

| Precursor       | Side-chain modification      | Structure                          | Abbreviation |
|-----------------|------------------------------|-------------------------------------|--------------|
| Methionine      | Elongation                   | 3-Methylthiopropyl-                | 3MTP-        |
|                 |                              | 4-Methylthiobutyl-                 | 4MTB-        |
|                 |                              | 5-Methylthiopentyl-                | 5MTP-        |
|                 |                              | 6-Methylthiohexyl-                 | 6MTH-        |
|                 |                              | 7-Methylthioheptyl-                | 7MTH-        |
|                 |                              | 8-Methylthiooctyl-                 | 8MTO-        |
|                 | + Oxidation                  | 3-Methylsulfinylpropyl-            | 3MSOP-       |
|                 |                              | 4-Methylsulfinylbutyl-             | 4MSOB-       |
|                 |                              | 7-Methylsulfinylheptyl-            | 7MSOH-       |
|                 |                              | 8-Methylsulfinyloctyl-             | 8MSOO-       |
|                 | + Others                     | 3-Butenyl-                         | 3Bn-         |
|                 |                              | (S)-2-Hydroxy-3-butenyl-           | S2OH3Bn-     |
|                 |                              | Allyl- (sinigrin)                  | Allyl-       |
| Phenylalanine   | None                         | Benzyl-                            | Bz-          |
| Tryptophan      | None                         | Indole-3-methyl                    | I3M-         |
|                 | Hydroxylation                | 4-Hydroxyindole-3-methyl           | 4OHI3M-      |
|                 | + Methylation                | 4-Methoxyindole-3-methyl           | 4MOI3M-      |
|                 |                              | N-Methoxyindole-3-methyl           | NMOI3M-      |

zymes were cloned into the pQE-30 vector for bacterial expression of N-terminally His<sub>6</sub>-tagged proteins. The proteins could be purified under native conditions by Ni<sup>2+</sup> chelate affinity chromatography (Fig. 4). For comparison, the cDNA for AtST1 (RaR047, encoded by At2g03760), a sulfotransferase also described to be up-regulated by jasmonic acid (5), was cloned and expressed in the same manner. In initial experiments, a range of known sulfotransferase substrates were tested using recom-

50722 Desulfoglucosinolate Sulfotransferases from A. thaliana

FIG. 6. Identification of the reaction products formed by AtST5 enzymes as GSs. $A$, the reaction product formed by AtST5a from I3M-dsGS was purified by HPLC and analyzed by Q-TOF mass spectrometry. Shown are the collision-induced dissociation tandem mass spectra of the reaction product and of the ${ }^{13} \mathrm{C}_{2}$I3M-GS standard in negative ion mode. Possible structures of the fragments obtained are given. $B$, shown is the conversion of 8MTO-dsGS (10 nmol) by AtST5c and the verification of a full GS core structure in the reaction product. 8MTO-dsGS was first incubated in the presence or absence of a sulfotransferase enzyme (AtST5c). After completion of the reaction, myrosinase was added or not. Myrosinase did not hydrolyze 8MTO-dsGS (second trace from the bottom), but the reaction product formed by AtST5c (third trace) vanished after myrosinase treatment (fourth trace). Thus, the product formed had a full GS core structure. $m A U$, milli-absorbance units.

binant AtST5a. However, sulfation of quercetin, 17$\beta$-estradiol, brassinolide, 24-epibrassinolide, castasterone, 24-epicastasterone, and 11- and 12-hydroxyjasmonic acid could not be observed. Thus, it was considered unlikely that these or structurally similar compounds are potential substrates *in vivo*. Since GS levels increase in members of the Brassicaceae family after wounding and jasmonate treatment (e.g. Refs. 37–39) we considered dsGSs, the immediate precursors of GSs, as possible substrates. Upon incubation of the AtST5 enzymes with a mixture of several dsGSs (total amount of 14 nmol) in the presence of 2 nmol of PAPS, turnover of dsGSs was observed, while at the same time, the formation of new substances could be detected (Fig. 5A and Table II). To investigate whether these products are indeed GSs, the reaction product formed from indole-3-methyl (I3M)-dsGS (desulfoglucobrassicin) by AtST5a was purified by HPLC and further analyzed by electrospray ionization time-of-flight mass spectrometry. The relative molecular mass of the product in negative ion mode was 447.0515, which is a deviation of 3.8 ppm from the expected relative molecular mass of glucobrassicin $(\mathrm{M}-\mathrm{H}^{+}, 447.0532)$ and therefore within the specification of the mass spectrometer used (5 ppm). The collision-induced dissociation tandem mass spectra of the product in positive and negative ion mode were in accordance (peak patterns and relative intensities) with the data obtained from synthetic ${ }^{13} \mathrm{C}_{2}$glucobrassicin (Fig. 6A).

As a second line of evidence, the sulfation product formed from 8-methylthiooctyl (8MTO)-dsGS by AtST5c was incubated

Desulfoglucosinolate Sulfotransferases from A. thaliana

TABLE III  
Substrate specificities of Arabidopsis dsGS-STs  

Each enzyme was incubated with pairs of dsGSs. Each dsGS mixture contained the favorite substrate (5 nmol) together with a second substrate (5 nmol). Incubation was performed under PAPS-limited conditions (2.5 nmol) until completion of the reaction. Shown is the relative percentage of activity in comparison with the best substrate. 7MTH-, 7-methylthioheptyl-; Bz-, benzyl-; 4MTB-, 4-methylthiobutyl-; 4MOI3M-, 4-methoxyindole-3-methyl-; NMOI3M-, *N*-methoxyindole-3-methyl-; 3Bn-, 3-butenyl-; 4MSOB-, 4-methylsulfinylbutyl-8MSOO-, 8-methylsulfinyloctyl-.

| Enzyme   | Best substrate | Other substrates                                                                 |
|----------|----------------|----------------------------------------------------------------------------------|
| AtST5a   | I3M-dsGS       | Bz-dsGS (59%) > 4MTB-dsGS (15%) = 4MOI3M-dsGS (15%) > NMOI3M-dsGS (9%)               |
| AtST5b   | 7MTH-dsGS      | 8MTO-dsGS (70%) > 4MTB-dsGS (19%) = Bz-dsGS (19%) > I3M-dsGS (10%) > 8MSOO-dsGS (6%) > |
|          |                | 3Bn-dsGS (3%)                                                                  |
| AtST5c   | 8MTO-dsGS      | 7MTH-dsGS (58%) ≈ 3Bn-dsGS (27%) > 4MTB-dsGS (19%) > 8MSOO-dsGS (8%) >              |
|          |                | 4MSOB-dsGS (3%)                                                                |

![Graph](https://example.com/graph_image.png)

FIG. 7. GS content in Arabidopsis leaves 24 h (solid bars) or 48 h (hatched bars) after spraying with 5 μM coronatine (solid gray and hatched gray bars), with 250 μM MeJA (solid black and hatched black bars), or with spraying solution alone (solid white and hatched white bars). ctrl, control; DW, dry weight; 3MSOP-, 3-methylsulfinylpropyl-; 4MSOB-, 4-methylsulfinylbutyl-; 5MSOP-, 5-methylsulfinylpentyl-; 4MOI3M-, 4-methoxyindole-3-methyl-; NMOI3M-, *N*-methoxyindole-3-methyl-.

with the enzyme myrosinase (Fig. 6B). Myrosinase is a thioglucosidase that is specific for the hydrolysis of GSs. This enzyme requires the full GS core structure and does not react with dsGSs (Fig. 6B) (40). As shown in Fig. 6B, the product formed from 8MTO-dsGS by AtST5c vanished in the presence of myrosinase, a clear indication that this product is characterized by a complete GS core structure. Taken together, the three AtST5 isoforms are functional dsGS-STs. While this manuscript was under review, similar results regarding the identification of the AtST5 enzymes as dsGS-STs were also reported by another group.³

AtST1 (RaR047) showed no activity toward dsGSs. AtST1 is an ortholog of the steroid sulfotransferase BnSST3 from *B. napus* (Fig. 2). BnSST3 is known to sulfate certain brassinosteroids (2); it seems therefore likely that AtST1 is also a steroid sulfotransferase.

Substrate Specificity of dsGS-STs—When incubated with a mixture of dsGSs (derived from the amino acids methionine, phenylalanine, and tryptophan) in the presence of approximately equimolar amounts of PAPS (15 nmol of PAPS versus 14 nmol of total dsGSs), all three dsGS-STs were able to sulfate different types of dsGSs, albeit with different preferences (Fig. 5B). These substrate specificities became more obvious when limited amounts of PAPS (2 nmol) were applied (Fig. 5C): AtST5a clearly prefers dsGSs derived from aromatic amino acids, *viz.* I3M-dsGS and, to a lesser extent, benzyl-dsGS, whereas AtST5b and AtST5c prefer methionine-derived dsGS. The substrate specificities were further characterized by comparing substrate pairs; the preferred substrate for each enzyme was incubated simultaneously with a second dsGS (Table III). Again, the data emphasized the preference of AtST5a for I3M-dsGS. The substrate specificities for AtST5b and AtST5c are similar but clearly distinguishable. The favorite substrate identified for AtST5b is 7-methylthioheptyl-dsGS, whereas AtST5c prefers 8MTO-dsGS. Furthermore, additional potential substrates could be identified by incubating the enzymes with dsGS mixtures isolated from leaves or seeds of *A. thaliana* ecotypes Col-0 (leaves) and C24 (leaves and seeds). Through these experiments, the following dsGSs were shown to become sulfated: 3-methylthiopropyl-, 3-methylsulfinylpropyl-, 4-methylsulfinylbutyl-, (*S*)-2-hydroxy-3-butenyl-, 5-methylthiopenyl-, 5-methylsulfinylpentyl-, 6-methylthiohexyl-, and 7-methylsulfinylheptyl-dsGSs (data not shown).

Because the substrate specificities of the dsGS-STs are quite broad, it is not surprising that even side chain-modified dsGSs, such as oxidized forms (e.g., 8-methylsulfinyloctyl-dsGS) and methoxylated forms (e.g., 4-methoxyindole-3-methyl-dsGS and *N*-methoxyindole-3-methyl-dsGS), and more extensively modified forms, such as (*S*)-2-hydroxy-3-butenyl-dsGS, were also accepted as substrates. This indicates that, in principle, certain

³ M. Klein, J. Tokuhisa, M. Reichelt, J. Gershenzon, and J. Papenbrock (2004) Poster abstract T07-043 presented at the 15th International Conference on Arabidopsis Research, July 11–14, 2004, Berlin, Germany.

side chain modification reactions could take place before the GS core structure is completed.

**Effect of Coronatine on Glucosinolate Biosynthesis—It is known that treatment of A. thaliana, Sinapis alba, B. juncea, B. napus, and Brassica rapa with jasmonic acid leads to an increase in indole-derived GSs (e.g. Refs. 38, 39, and 42–44). Because coronatine is proposed to be a structural analog of jasmonic acid and/or 12-oxophytodienoic acid, we analyzed the effect of the phytotoxin on GS biosynthesis in A. thaliana ecotype Col-0. The contents of the individual GSs found in control plants were in good agreement with recently published data (31, 32) with the exception of glucobrassicin (I3M-GS), the content of which was consistently ~2-fold higher in our experiments than described in the literature (data not shown). This increase was not due to spraying (40% (v/v) acetone and 0.1% Tween 20) because untreated plants showed the same level of I3M-GS. Application of coronatine resulted in a 2–3-fold increase in the amount of I3M-GS and a 5–7-fold increase in the amount of neoglucobrassicin (N-methoxyindole-3-methyl-GS) after 24 h and an additional slight increase after 48 h, whereas the amount of the other GSs (including tryptophan-derived 4-methoxyindole-3-methyl-GS) was either unchanged or only marginally affected (Fig. 7). Qualitatively comparable results were obtained after MeJA treatment (Fig. 7) (44), indicating that both compounds, although at different concentrations (5 μM coronatine versus 250 μM MeJA), affect the biosynthesis of I3M-GS and N-methoxyindole-3-methyl-GS in a similar manner.

DISCUSSION

We have described the cloning and functional characterization of a small family of dsGS-STs. The founding member of this family, AtST5a, was identified using the mRNA differential display technique, comparing mock-treated A. thaliana with plants sprayed with 5 μM coronatine. Phylogenetic analysis indicated that AtST5a clusters with AtST5b and AtST5c out of the 18 annotated sulfotransferases in the Arabidopsis genome (Fig. 2). The three genes are located on chromosome 1, and AtST5a and AtST5b are organized in tandem fashion. However, according to phylogenetic analysis, AtST5b is more closely related to AtST5c compared with AtST5a. Comparison with known sulfotransferases from other species revealed that the AtST5 family is orthologous to the SULT3 family of flavonol sulfotransferases (Fig. 2) (3). Nevertheless, sulfated flavonoids from A. thaliana are not known (6).

Not only are AtST5b and AtST5c phylogenetically more closely related to each other compared with AtST5a, they additionally show similar substrate specificities. Both enzymes prefer long chain dsGSs derived from methionine, whereas AtST5a clearly prefers dsGSs derived from the aromatic amino acids tryptophan and phenylalanine. Tryptophan-derived I3M-GS (glucobrassicin) is present in relatively high concentrations in A. thaliana throughout the whole life cycle, whereas phenylalanine-derived GSs are only barely detectable in many ecotypes of A. thaliana (11, 31, 32, 45). Thus, the proposed function of AtST5a in vivo is biosynthesis of I3M-GS. However, it should be noted that all three AtST5 enzymes are capable of converting a wide range of different dsGSs; and hence, they do not show strict substrate specificities (Fig. 5 and Table III). Therefore, biosynthesis of specific GSs seems to be regulated at the pre-sulfotransferase level. For example, the preferred substrates for AtST5b and AtST5c, viz. 7-methylthioheptyl-dsGS and 8MTO-dsGS, respectively, are present only in small amounts in the leaves of A. thaliana ecotype Col-0, whereas the main GS in this tissue is 4-methylsulfinylbutyl-GS. 4-Methylsulfinylbutyl-GS is derived from 4-methylthiobutyl-GS, the dsGS of which is a reasonably good substrate for AtST5b and

AtST5c, too. Further experiments using plants in which single dsGS-ST genes are knocked out will provide evidence regarding functional redundancy.

Coronatine treatment of plants resulted in a specific increase in tryptophan-derived I3M-GS and N-methoxyindole-3-methyl-GS (Fig. 7), whereas at the same time, the genes coding for AtSt5a and AtST5c were up-regulated (Fig. 3C). This again points to a pre-sulfotransferase regulation of GS biosynthesis. It is already known that the aldoxime-forming enzymes CYP79B2, CYP79B3, CYP79F1, and CYP79F2, which are involved in the biosynthesis of tryptophan-derived GSs (CYP79B2 and CYP79B3) and methionine-derived GSs (CYP79F1 and CYP79F2), are up-regulated by treatment with MeJA (44). The fact that methionine-derived GSs are still only slightly affected by MeJA and coronatine indicates that their biosynthesis is substrate-limited, very likely at the level of chain elongation. Because tryptophan does not undergo chain elongation before entering the GS biosynthetic pathway, it could readily be consumed by CYP79B2 and CYP79B3.

One step in GS biosynthesis is catalyzed by a C-S lyase. Interestingly, one additional gene identified as up-regulated by coronatine, CORI-3, encodes a C-S lyase, initially annotated as tyrosine aminotransferase (29, 46). Although the C-S lyase involved in GS biosynthesis was identified as the SUPERROOT1 gene product, which is not identical to CORI-3 (20), the possibility that CORI-3 may be involved in I3M-GS biosynthesis induced by coronatine or MeJA cannot be ruled out.

Acknowledgments—We thank Dr. D. Roby for providing the AtST1 cDNA (plasmid pRaR047); Dr. C. Wasternack for the kind gift of 12-hydroxyjasmonic acid; and D. Kreuder, L. Röβner, and S. Schönfelder for excellent technical assistance. M.P. and T.J. are grateful to Drs. M. Reichelt and J. Gershenzon (Max Planck Institute of Chemical Ecology, Jena, Germany) for the introduction to glucosinolate analysis.

REFERENCES

1. Varin, L., Marsolais, F., Richard, M., and Rouleau, M. (1997) *FASEB J.* **11**, 517–525

2. Rouleau, M., Marsolais, F., Richard, M., Nicolle, L., Voigt, B., Adam, G., and Varin, L. (1999) *J. Biol. Chem.* **274**, 20925–20930

3. Marsolais, F., Sebastia, C. H., Rousseau, A., and Varin, L. (2004) *Plant Sci.* **166**, 1359–1370

4. Gidda, S. K., Miersch, O., Levitin, A., Schmidt, J., Wasternack, C., and Varin, L. (2003) *J. Biol. Chem.* **278**, 17895–17900

5. Lacomme, C., and Roby, D. (1996) *Plant Mol. Biol.* **30**, 995–1008

6. Barron, D., Varin, L., Ibrahim, R. K., Harborne, J. B., and Williams, C. A. (1988) *Phytochemistry* **27**, 2375–2395

7. Halkier, B. A., and Du, L. C. (1997) *Trends Plant Sci.* **2**, 425–431

8. Fahey, J. W., Zalcmann, A. T., and Talalay, P. (2001) *Phytochemistry* **56**, 5–51

9. Rask, L., Andreasson, E., Ekbom, B., Eriksson, S., Pontoppidan, B., and Meijer, J. (2000) *Plant Mol. Biol.* **42**, 93–113

10. Mikkelsen, M. D., Hansen, C. H., Wittstock, U., and Halkier, B. A. (2000) *J. Biol. Chem.* **275**, 33712–33717

11. Wittstock, U., and Halkier, B. A. (2000) *J. Biol. Chem.* **275**, 14659–14666

12. Barlier, I., Kowalczyk, M., Marchant, A., Ljung, K., Bhalerao, R., Bennett, M., Sandberg, G., and Bellini, C. (2000) *Proc. Natl. Acad. Sci. U.S.A.* **97**, 14819–14824

13. Bak, S., and Feyereisen, R. (2001) *Plant Physiol.* **127**, 108–118

14. Hansen, C. H., Du, L., Naur, P., Olsen, C. E., Axelsen, K. B., Hick, A. J., Pickett, J. A., and Halkier, B. A. (2001) *J. Biol. Chem.* **276**, 24790–24796

15. Reintanz, B., Lehnen, M., Reichelt, M., Gershenzon, J., Kowalczyk, M., Sandberg, G., Godde, M., Uhl, R., and Palme, K. (2001) *Plant Cell* **13**, 351–367

16. Bak, S., Tax, F. E., Feldmann, K. A., Galbraith, D. W., and Feyereisen, R. (2001) *Plant Cell* **13**, 101–111

17. Hansen, C. H., Wittstock, U., Olsen, C. E., Hick, A. J., Pickett, J. A., and Halkier, B. A. (2001) *J. Biol. Chem.* **276**, 11078–11085

18. Zhao, Y., Hull, A. K., Gupta, N. R., Goss, K. A., Alonso, J., Ecker, J. R., Normanly, J., Chory, J., and Celenza, J. L. (2002) *Genes Dev.* **16**, 3100–3112

19. Chen, S., Glawischnig, E., Jørgensen, K., Naur, P., Jørgensen, B., Olsen, C. E., Hansen, C. H., Rasmussen, H., Pickett, J. A., and Halkier, B. A. (2003) *Plant J.* **33**, 923–937

20. Mikkelsen, M. D., Naur, P., and Halkier, B. A. (2004) *Plant J.* **37**, 770–777

21. Marillia, E. F., MacPherson, J. M., Tsang, E. W., Van Audenhove, K., Keller, W. A., and GrootWassink, J. W. (2001) *Physiol. Plant.* **113**, 176–184

22. Glendening, T. M., and Poulton, J. E. (1990) *Plant Physiol.* **94**, 811–818

23. Jain, J. C., GrootWassink, J., Kolenovsky, A. D., and Underhill, E. W. (1990) *Phytochemistry* **29**, 1425–1428

24. Xia, Z.-Q., and Zenk, M. H. (1993) *Planta Med.* **59**, 575–576

25. Müller, A., and Weiler, E. W. (2000) *Planta* **211**, 855–863

26. Ausubel, F. M., Brent, R., Kingston, R. E., Moore, D. D., Seidman, J. G.,

Struhl, K. (eds) (2004) *Current Protocols in Molecular Biology*, John Wiley & Sons, Inc., New York

27. Sambrook, J., and Russell, D. W. (2001) *Molecular Cloning: A Laboratory Manual*, 3rd Ed., Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY

28. Barkan, A. (1989) *Plant Cell* **1**, 437–445

29. Lopukhina, A., Dettenberg, M., Weiler, E. W., and Hollander-Czytko, H. (2001) *Plant Physiol.* **126**, 1678–1687

30. Frohman, M. A., Dush, M. K., and Martin, G. R. (1988) *Proc. Natl. Acad. Sci. U. S. A.* **85**, 8998–9002

31. Petersen, B. L., Chen, S., Hansen, C. H., Olsen, C. E., and Halkier, B. A. (2002) *Planta* **214**, 562–571

32. Brown, P. D., Tokuhisa, J. G., Reichelt, M., and Gershenzon, J. (2003) *Phytochemistry* **62**, 471–481J. G.

33. Weiler, E. W., Kutchan, T. M., Gorba, T., Brodschelm, W., Niesel, U., and Bublitz, F. (1994) *FEBS Lett.* **345**, 9–13

34. Liang, P., and Pardee, A. B. (1992) *Science* **257**, 967–971

35. Klein, M., and Papenbrock, J. (2004) *J. Exp. Bot.* **55**, 1809–1820

36. Marsolais, F., Gidda, S. K., Boyd, J., and Varin, L. (2000) *Recent Adv. Phytochem.* **34**, 433–456

37. Bodnaryk, R. P. (1992) *Phytochemistry* **31**, 2671–2677

38. Bodnaryk, R. P. (1994) *Phytochemistry* **35**, 301–305

39. Doughty, K. J., Kiddle, G. A., Pye, B. J., Wallsgrove, R. M., and Pickett, J. A.

(1995) *Phytochemistry* **38**, 347–350

40. Ettlinger, M. G., Dateo, G. P., Jr., Harrison, B. W., Mabry, T. J., and Thompson, C. P. (1961) *Proc. Natl. Acad. Sci. U. S. A.* **47**, 1875–1880

41. Sakakibara, Y., Yanagisawa, K., Katafuchi, J., Ringer, D. P., Takami, Y., Nakayama, T., Suiko, M., and Liu, M. C. (1998) *J. Biol. Chem.* **273**, 33929–33935

42. Bennett, R. N., Kiddle, G., and Wallsgrove, R. M. (1997) *Plant Physiol.* **114**, 1283–1291

43. Brader, G., Tas, E., and Palva, E. T. (2001) *Plant Physiol.* **126**, 849–860

44. Mikkelsen, M. D., Petersen, B. L., Glawischnig, E., Jensen, A. B., Andreasson, E., and Halkier, B. A. (2003) *Plant Physiol.* **131**, 298–308

45. Reichelt, M., Brown, P. D., Schneider, B., Oldham, N. J., Stauber, E., Tokuhisa, J., Kliebenstein, D. J., Mitchell-Olds, T., and Gershenzon, J. (2002) *Phytochemistry* **59**, 663–671

46. Jones, P. R., Manabe, T., Awazuahara, M., and Saito, K. (2003) *J. Biol. Chem.* **278**, 10291–10296

47. Anavroranich, S., Gulick, P., and Ibrahim, R. K. (1995) *Plant Physiol.* **107**, 1019–1020

48. Anavroranich, S., Varin, L., Gulick, P., and Ibrahim, R. (1994) *Plant Physiol.* **106**, 485–491

49. Varin, L., Deluca, V., Ibrahim, R. K., and Brisson, N. (1992) *Proc. Natl. Acad. Sci. U. S. A.* **89**, 1286–1290

Enzyme Catalysis and Regulation:  
Desulfoglucosinolate Sulfotransferases  
from *Arabidopsis thaliana* Catalyze the  
Final Step in the Biosynthesis of the  
Glucosinolate Core Structure  

Markus Piotrowski, Andreas Schemenewitz,  
Anna Lopukhina, Axel Müller, Tim Janowitz,  
Elmar W. Weiler and Claudia Oecking  
J. Biol. Chem. 2004, 279:50717-50725.  
doi: 10.1074/jbc.M407681200 originally published online September 9, 2004  

Access the most updated version of this article at doi: [10.1074/jbc.M407681200](http://dx.doi.org/10.1074/jbc.M407681200)  

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinity-sites).  

Alerts:  
- [When this article is cited](#)  
- [When a correction for this article is posted](#)  

[Click here](#) to choose from all of JBC's e-mail alerts  

This article cites 47 references, 26 of which can be accessed free at  
[http://www.jbc.org/content/279/49/50717.full.html#ref-list-1](http://www.jbc.org/content/279/49/50717.full.html#ref-list-1)
