
Biochemistry 2004, 43, 11226–11237

# Studies on the Interaction of NADPH with Rhodobacter sphaeroides Biotin Sulfoxide Reductase†

Kimberly Johnson Nelson and K. V. Rajagopalan*

Department of Biochemistry, Duke University Medical Center, Box 3711 Duke University Medical Center, Durham, North Carolina 27710

Received May 6, 2004; Revised Manuscript Received July 6, 2004

ABSTRACT: *Rhodobacter sphaeroides* biotin sulfoxide reductase (BSOR) contains the bis(molybdopterin guanine dinucleotide)molybdenum cofactor and catalyzes the reduction of D-biotin-D-sulfoxide to biotin. This protein is the only member of the dimethyl sulfoxide reductase family of molybdopterin enzymes that utilizes NADPH as the direct electron donor to the catalytic Mo center. Kinetic studies using stopped-flow spectrophotometry indicate that BSOR reduction by NADPH (>1000 s⁻¹) is faster than steady-state turnover (440 s⁻¹) and has shown that BSOR reduction occurs in concert with NADPH oxidation with no indication of a Mo⁵⁺ intermediate species. Because no crystallographic structure is currently available for BSOR, a protein structure was modeled using the structures for *R.* *sphaeroides* dimethyl sulfoxide reductase, *Rhodobacter capsulatus* dimethyl sulfoxide reductase, and *Shewanella massilia* trimethylamine *N*-oxide reductase as the templates. A potential NADPH-binding site was identified and tested by site-directed mutagenesis of residues within the area. Mutation of Arg137 or Asp136 reduced the ability of NADPH to serve as the electron donor to BSOR, indicating that the NADPH-binding site in BSOR is located in the active-site funnel of the putative structure where it can directly reduce the Mo center. Along with kinetic and spectroscopic data, the location of this binding site supports a direct hydride transfer mechanism for NADPH reduction of BSOR.

Biotin sulfoxide reductase (BSOR)¹ is a bacterial enzyme that catalyzes the reduction of D-biotin-D-sulfoxide (BSO), a common oxidative product of the vitamin D-biotin. Possible roles for the enzyme include scavenging BSO from the environment to generate biotin and protecting the cell from oxidative damage (1). While the preferred substrate for this protein is BSO, it is also able to utilize a variety of alternative substrates including dimethyl sulfoxide (DMSO), D,L-methionine sulfoxide, trimethylamine *N*-oxide (TMAO), adenosine *¹N*-oxide, and nicotinamide-*N*-oxide (2). *Rhodobacter sphaeroides* BSOR is a member of the DMSO reductase (DMSOR) family of molybdopterin-containing enzymes and contains the bis(molybdopterin guanine dinucleotide)molybdenum (bis(MGD)Mo) cofactor. BSOR is closely related to DMSOR from *R.* *sphaeroides* and *R.* *capsulatus* as well as TMAO reductase (TMAOR) from *Escherichia coli* and *Shewanella massilia*. These enzymes are unique among molybdopterin-containing proteins because they contain no prosthetic groups other than bis(MGD)Mo,

making them ideal candidates for spectroscopic studies of the catalytic molybdenum (Mo) center.

*R.* *sphaeroides* BSOR has been cloned, heterologously expressed, and purified from *E.* *coli* (2, 3). Resonance Raman and extended X-ray absorption fine spectroscopy (EXAFS) analyses have shown that BSOR, like *R.* *sphaeroides* DMSOR, cycles between mono-oxo, Mo⁶⁺, and des-oxo, Mo⁴⁺, forms with the Ser ligand from the protein and all four molybdopterin dithiolene ligands remaining coordinated to the Mo in both redox states (3, 4). Resonance Raman analysis using *¹⁸O*-labeled substrate demonstrated that the terminal oxo group in the substrate-oxidized Mo⁶⁺ form arises from the substrate and that it is exchangeable with water during redox cycling (4).

Because of the similarities between the two proteins, most of the discussion of the chemistry occurring at the Mo center of BSOR has been based on the more extensively studied *Rhodobacter* DMSOR. However, the reductive mechanisms of these two enzymes are very different. The physiological electron donors to *R.* *sphaeroides* DMSOR and *E.* *coli* TMAOR are pentaheme *c*-type cytochromes, whose hemes can only donate one electron at a time (5, 6), indicating that the Mo centers of these enzymes are reduced by two sequential, single-electron donations. The Mo⁵⁺ form of DMSOR is, therefore, an important intermediate during reduction of the Mo center and has been studied extensively using electron paramagnetic resonance (EPR) spectroscopy (7, 8). A Mo⁵⁺ form of BSOR is obtained upon reduction with dithionite, and the EPR spectrum exhibits significant differences from that observed for *R.* *sphaeroides* DMSOR

---

† This work was supported by Grant GM00091 from the National Institutes of Health for K.V.R. and a predoctoral research grant from the National Science Foundation to K.J.N.

* To whom correspondence should be addressed. Telephone: (919) 681-8845. Fax: (919) 684-8919. E-mail: raj@biochem.duke.edu.

¹ Abbreviations: BSOR, biotin sulfoxide reductase; BSO, biotin sulfoxide; DMSO, dimethyl sulfoxide; TMAO, trimethylamine *N*-oxide; DMSOR, dimethyl sulfoxide reductase; bis(MGD)Mo, bis(molybdopterin guanine dinucleotide)molybdenum; TMAOR, trimethylamine *N*-oxide reductase; EXAFS, extended X-ray absorption fine structure; EPR, electron paramagnetic resonance; rms, root mean square; DMS, dimethyl sulfide; GST, glutathione-S-transferase.

10.1021/bi0490845 CCC: $27.50 © 2004 American Chemical Society

Published on Web 08/14/2004

NADPH reduction of *R. sphaeroides* BSOR

(3). BSOR, however, is unique among this family of enzymes in its utilization of NADPH as a direct electron donor. Because NAD(P)H is a two-electron donor, the Mo^V form of BSOR would only be physiologically relevant if NADPH oxidation is separated spatially from reduction of the Mo center.

Because no crystallographic structure is available for BSOR, there is no structural evidence indicating whether NADPH binds at a peripheral location or close enough to the Mo atom to reduce it directly. Analysis of the BSOR amino acid sequence provides little information about how the protein interacts with NADPH. Although a common adenine-binding motif has been identified in many protein structures, it relies on both hydrophobic and backbone interactions that are not dependent on the protein sequence or the particular properties of amino acid side chains (9). Multiple studies have shown some small sequence identity between various NADH-binding proteins, but NADPH-binding sites are even more diverse.

The similarities in amino acid sequence, cofactor composition, and reaction mechanism suggest that available structures for *R. sphaeroides* DMSOR (10), *R. capsulatus* DMSOR (11, 12), and *S. massilia* TMAOR (13) can serve as models for the structural fold of BSOR. DMSOR and TMAOR exhibit a high degree of structural similarity. The 2.5-Å structure of *S. massilia* TMAOR can be superimposed on that of *R. sphaeroides* DMSOR with a root-mean-square (rms) deviation of 1.3 Å between Cα atoms and on *R. capsulatus* DMSOR with a rms deviation of 1.31 Å (13). These proteins are mixed α + β proteins that consist of four domains surrounding the bis(MGD)Mo cofactor. With the exception of domain IV, the domains are not formed of continuous stretches of the amino acid sequence, and residues in three of the four domains have direct contact with the bis(MGD)-Mo cofactor. BSOR exhibits a high degree of sequence identity with TMAOR and DMSOR throughout the protein sequences (Figure 1), and the ability of BSOR to bind and stabilize the complex bis(MGD)Mo cofactor makes it unlikely that the basic protein fold of BSOR will differ significantly from that of DMSOR and TMAOR.

Resonance Raman analysis indicated that BSOR uses a direct oxo-transfer mechanism to reduce BSO and that biotin is able to induce changes in the M=O stretching frequencies, confirming that both the substrate and product directly interact with the Mo center (4). In DMSOR and TMAOR, the bis(MGD)Mo cofactor is buried in the protein and the only access to the Mo atom is found at the bottom of a deep depression on one side of the protein. A crystal structure of *R. capsulatus* DMSOR has shown a bound dimethyl sulfide (DMS) molecule coordinated by the oxo group on the Mo atom at the bottom of this active-site funnel (11). If NADPH reduction of BSOR occurs by a direct hydride transfer mechanism, the NADPH-binding site should be located within a similar funnel to allow direct access to the oxo group on the Mo atom. Another possibility is that NADPH binds at a distant location, and the electrons are transferred intramolecularly. A small depression has been identified on the surface of DMSOR that may serve as the site for cytochrome binding because it provides access to and would allow electron transfer through one of the molybdopterin cofactors (12). A similar mechanism could be envisioned for NADPH reduction of BSOR.


Previous kinetic analysis has provided some insight into the reductive mechanism of BSOR. Steady-state studies in the presence of BSO and either NADPH or NADH indicated that NADPH is the preferred substrate (2). Later, more detailed studies indicated that BSOR activity occurs via a ping-pong bi-bi molecular reaction, and that BSO and NAD-(P)H are competitive inhibitors of each other (14). While this second characteristic suggests that NADPH binding is likely to occur near the BSO-binding site within the active-site funnel, it also complicates kinetic analysis of the protein.

This article clarifies the mechanism by which NADPH reduces BSOR. Pre-steady-state analysis of this reaction has been done for the first time using stopped-flow spectrophotometry and indicates that BSOR reduction occurs more quickly than steady-state turnover and that NADPH oxidation occurs in concert with reduction of the Mo center. While an intermediate spectrum is observed during dithionite reduction of BSOR that indicates the presence of a Mo^V intermediate, no such intermediate species was observed during NADPH reduction of BSOR. A BSOR structure was modeled using the structures of *R. sphaeroides* DMSOR, *R. capsulatus* DMSOR, and *S. massilia* TMAOR as the templates and allowed the identification of a putative NADPH-binding site. This location was tested by site-directed mutagenesis of specific residues in the region. Mutation of Asp136 and Arg137 altered the $K_{\text{m}}$ and $K_{\text{D}}$ values for NADPH, confirming the validity of the proposed site for NADPH binding. This site is located in the active-site funnel and would allow direct reduction of the Mo center. With the kinetic and spectroscopic data, this result supports a direct hydride transfer mechanism for NADPH reduction of BSOR.

**MATERIALS AND METHODS**

*Mutagenesis and Purification of BSOR.* All primers and restriction enzymes were supplied by Invitrogen Life Technologies. Mutagenesis was performed on the pBSOR vector (2) using the Transformer Site-Directed Mutagenesis Kit (Clontech) and confirmed by automated sequencing of both strands of the resulting plasmids. During the course of mutagenesis, two G-C rich regions were identified where the original nucleotide sequence reported for the pBSOR plasmid (15) was incorrect. The corrected amino acid sequence is as follows: Cys22 → Trp22, Gly23 → Ala23, Asp629 → Glu629, and Glu630 → Gln630.

Wild-type and mutantBSOR were expressed as glutathione-S-transferase (GST) fusions in MC4100 *E. coli* cells and purified, and the GST-tag was cleaved with Factor Xa protease prior to analysis according to the method of Temple et al. (3). Q-Sepharose fast flow, Superose 12, glutathione-agarose affinity resins, and Factor Xa protease were obtained from Amersham Pharmacia Biotech. BSOR was quantified by the BCA assay (Pierce) using known concentrations of DMSOR to determine the standard curve. Mo analysis was performed using a Perkin-Elmer Zeeman 3030 atomic absorption spectrophotometer as described previously (16).

*Kinetic Analysis Using Methyl Viologen as the Reductant.* BSO was synthesized from D-biotin according to the method of Melville (17). Kinetic constants using methyl viologen as the electron donor were determined anaerobically in 50 mM Tris-HCl at pH 8.0, using 0.15 mM methyl viologen as described previously (16). All values at different concentrations.

R.sp DMSOR 1 ~~~~~~EGLANGEVMSGCHWGVFKA RVENGRAVAFEPWDKDPAPSHQLPG VLDSIYSPTRIKYPMVRRREFLEKGV NADRSTRGNGDFVRVWDEALDLVA 94
R.ca DMSOR 1 ~~~~~~EALANGTVMSGSHWGVFTA TVENGRATAFTPWEKDPHPTPMLEG VLDSIYSPTRIKYPMVRRREFLEKGV NADRSTRGNGDFVRVSWDQALDLVA 94
E.co TMAOR 1 AQAATDAVISKEGILTGSHWGAIRA TVKDGRFVAAAKPFELDKYPSKMIAG LPDHVHNAARIRYPMVRVDWLKRKH LSDTSQRGDNRFVRVSWEALDMFY 100
S.ma TMAOR 1 ~~~~KAGINEDEWLTGSHFGAFKM KRKNGVIAEVKPFDLDKYPTDMING IRGMVYNPSRVRYPMVRLDELCLKGH KSNTHQRGDFRFVRVIWDKALTTLFK 96
R.sp BSOR 1 ~~~~~~~~~~~~~~~~ ~~~~~~~~VGVEBFAHDPAPS ELIHS VPACGSPERRVMRPMVREGMLADRQ HSDRRGRGERFLPVSWDAALDLVA 68

R.sp DMSOR 95 RELKRVQESYGPTGFFGSYGWKSP GRHNQCQLMRRALNLAGGFVNSSG DYSTAAAQIIMPHVMGTLEVY-EQQ TAWPVVVENTDLMVFWAADPMKTNE 193
R.ca DMSOR 95 AEVKRVEETYGPQGVFGGSYGWKSP GRHNCTTLLRRMLTAGGYVNGAG DYSTGAAQVIMPHVVGTLEVY-EQQ TAWPVLAENTEVMVFWAADPIKTSQ 193
E.co TMAOR 101 EELERVQKTHGPSALLTAS-GWQST GMFHNASGMRAKRTALHGNSVGTGG DYSTGAAQVILPRVVGSMEVY-EQQ TSWPLVLQNSKTIVLNGSDLALKNOQ 198
S.ma TMAOR 97 HSLDEVQTQYGPSGLHAGQTGWRAT GQIHSSSTSHMQRAVGMHGNYVKKIG DYSTGAGQTILPYVLGSTEVY-AQG TSWPLILEHSDTIVLWSNDPYKNLO 195
R.sp BSOR 69 GEIRRVSADHGNAAIFAGSYGWTC GRHHHASTLLLKRMILNLVGFTGHVD TYSIAAGPVILRHTLGDDR_ACGGQA NTLDSTAEHSQTLVVFGAMSPTAQ 168

R.sp DMSOR 194 I-GWVIPDHGAYAGMKALKEK---G T-RVICINPVRTETADYFGADVSP ---RPQTDVALMLGMAHTLYSEDLH DKDFLENCTTGFDLFAAYLTGESDG 285
R.ca DMSOR 194 I-GWVIPEHGAYPGLEALKAK---G T-KVIVIDPVRTKTVEFFGADHVTP ---KPQTDVAIMLGMAHTLVAEDLY DKDFIANYTSGFDKFLPYLMGETDS 285
E.co TMAOR 199 A-NWWCPDHDVYEYYAQLKRKSAAG EIEVISTDPVVTSTHEYLGGEHVKH IAVNPQTDVPLQLAIAHTLYSENLY DKNFLANYCVGFEFFLPYLLGEKDG 298
S.ma TMAOR 196 V-GWNAETHESFAYLAQLKEKVKG KIRVISIDPVTKTQAYLGCE---Q LYVNPQTDVTMLAIAHEMISKKLY DDKFIQGYSLGFEFVPYVMGTKDG 291
R.sp BSOR 169 SEAGGIGAHHLETYLRRIVER---G -VRVILVSPLKDDLPDWVAAEW--- WPIRPNDTDALMLGLAGEIVRSGRQ DSDFLARCTSGSELYLAYLRGEDG 261

R.sp DMSOR 286 TPKTAEWAAEICGLPAEQIRELRS FVAGRITMLAAGWSIORMHHGEOAHW MLVTLASMIQOIGLPGGGFGLSYHY SNGGSPTSDGPALGGISDGGKAVEG 385
R.ca DMSOR 286 TPKTAEWASDISGVPAAETIKERAL FISKRTMLAAGWSIORMHHGEOAHW MLVTLASMIQOIGLPGGGFGLSYHY SGGGTPSTSGPALSGITDDGAATKG 385
E.co TMAOR 299 QPKDAAWAEEKLSGIDAETIRGLARQ MAANRTQIIAGWCVORMQHGEOWAW MIVVLAAMIQOIGLPGGGFGFWHY NGAGTPGRKGVILSGFSGS--TSIP 396
S.ma TMAOR 292 VAKTPEWAAPICGVEAHVIRDЛАКТ LVKGRTQFMMGWCIRQQHGEOQPYW MAAVLATMIQOIGLPGGGISYGHHY SSIGVPSSGAAAPGAFPRNLDENQK 391
R.sp BSOR 262 RPKDAEWASTITGLPAEAIRALARD LPRTRSMLTVSWSIORAHHGEOPFW AALGLAAVIQOIGRPGGGVCYG--Y GSLGGVGAPFTIGKSPAMSQLSKPI 349

R.sp DMSOR 386 AAWLSE---SGATSIPCARVVDMLL NPGGEFQFNGATATYPDVKLAYWAG GNPFHHHDRNRMLKAWEKLETFIV QDFQWTATARHADIVLPATTSYERN 482
R.ca DMSOR 386 PEWLA---SGASVIPVARVVDMLE NPGAEEFDNGTRSKFPDVKMAYWVG GNPFVHHHDRNMVKAWEKLETFIV HDFQWTPTARHADIVLPATTSYERN 482
E.co TMAOR 397 PVHDNSDYKGYSTIPIARFIDAILE EPGBKVINWNNGKSVKLPPKMCIFAG TNPFHRHQOINRIIEGLRNVEVTIA IDNQWTSICRFADIVLPATTQFERN 495
S.ma TMAOR 392 PLFDSSDFKGASSTIPVARWIDAILE EPGKTIDANGSKVVYPDIKMMIFSG NNPNHHHDRNMKQAFHKLECVVT VDVNWTAICRFSDIVLPACTTYERN 491
R.sp BSOR 350 ----------NSFIPVARISDMLL NPGGPYSYEGEDRRYPDIRLVYWSG GNPFHHHODLNRLSEAWTRPETIIV QDPMFTATAKRADIVLPASTSIERN 448

R.sp DMSOR 483 DIESVGDSNRAILLAMKKVVDPLYE ARSDYDIFAAALERLGKGAEFTEGR DEMGWISSFYEAANKQA-EFKNVAM PSFEDFWS-EGIVEFPIATEGANFVR 580
R.ca DMSOR 483 DIETIGDSNTGILLAMKKIVEPLYE ARSDYDIFAAVAERLGKGKEFTEGK DEMGWKSFYDDAAKQG-KAGGVEM PARDAFWA-EGIVEFPVTDGADFVR 580
E.co TMAOR 496 DLDOYGNHSNRGIIAMKQVVPPQFE ARNDFDIFRELCRRFNREEAFTEGL DEMGWKRIWQEGVQQG-KGRGVHL PARDDFWNNKEYVEF--DHPQMFR 592
S.ma TMAOR 492 DIDVYGAYANRGILLAMQKMVEPLFD SLSDFEIFTRFAAVLGKEKEYTRNM GEMEWLETLYNECKAAN-AGK-FEM PDFATFW-KQGYVHF-GDGEVWTR 586
R.sp BSOR 449 DL--AGNKRSDFILLAMGQAIAPLGE ARSDFDIFNALSGKLGVAAAFNEGR DEMGWIRHLYEESRNHAQRHHHFEM PDBETFWA-QGHAPCPVQRDHTYL- 544

R.sp DMSOR 581 YADFRDEDLFNPLGTPSGLIEIYSK NIEKMGYDDCPAHPNWMEPAERL-G GAGA-KYPLHVVASHPKSRLLHSOL- NGTSLRDLYAVAGHEPCLINPADAA 677
R.ca DMSOR 581 YASFRDEDLLNPLGTPGLIEIYSK NIEKMGYDDCPAHPNWMEPLERL-D GPGA-KYPLHIAASHPFNRLHSOL- NGTVLREGYAVQGHEPCLMHPDAA 677
E.co TMAOR 593 HQAFRDEDELPLGTPSGLIEIYSK TIADMNYDDCQGHPMWBEKIERSHG GPGSQKYPLHLQSVHPDFRLHSOLC ESETLRHEYTVAGKEPVFINPQDAS 692
S.ma TMAOR 587 HADFRNDEPINPLGTPSGLIEIFSR KIDQFGYDDCKGHPNWMEKTERSHG GPGSDKHPIWLQSCHPDKRLHSOMC ESREYRETYAVNGREPVYISPVDAK 686
R.sp BSOR 545 -AAFREDPGAHPDLTESGLIVLGSА TLARLGYADCGRPAPAIEPAEWLGK AQAGE---LHLISHQPKGRLHSOLE TAEASLAGKR-EGREQVMLHPDDAS 639

R.sp DMSOR 678 ARGIAADGDVIRVFNDRGQILVGAKV SDAVMPGAIQIYEGGWYDPLDP--- ----SEEGTLDKYGDVNVLSLDVGT SKLAQGNCGQTILADVEKYAGAP-V 769
R.ca DMSOR 678 ARGIAADGVVRVHNDRGQILTGVKV TDAMVKGVIQIYEGGWYDPSDV---- ----TEPGTLDKYGDVNVLISADIGT SKLAQGNCGQTVAEEVEKYTGPA-V 769
E.co TMAOR 693 ARGIRNGDVVRVFNARGQVMAGAVV SDRYAPGVARIHEGANWYDP------- -DKGGBLGALCKYGNPNVLTIDIGT SQLAQATSAAHTTLVEIEKYNGTV-E 784
S.ma TMAOR 687 ARGIKDGDIVRVFNDRGQLLAGAVV SDNFPKGIVRIHEGAWYGPVGKDGS TEGGAEVGALCSYGDPNTLTLDIGT SKLAQACSAYTCLVEFEKYQGKV-P 785
R.sp BSOR 640 VRGIAADGQTVRLLWNARGACLATAQV TDSVAAGVAILLPTGAWFTP-------- ----AEAEGPLSGNPNVLTLDIGS SAFGQGCSAHTCLVRLEAHAGDAGD 729

R.sp DMSOR 770 TVTVFDTPKG A--------- -------- 780
R.ca DMSOR 770 TLTGFVAPKA AE--------- -------- 781
E.co TMAOR 785 QVTAFNGPVE MVAQCEYVPA SQVKS 809
S.ma TMAOR 786 KVSSFDGPIE VEI--------- -------- 792
R.sp BSOR 730 AVRIYDAHLA AILPT------- -------- 744

FIGURE 1: Sequence alignment of *R. sphaeroides* DMSOR, *R. capsulatus* DMSOR, *E. coli* TMAOR, *S. massilia* TMAOR, and *R. sphaeroides* BSOR. Residues identical for all five proteins are highlighted in black, and similar residues are shaded in gray. Residues that interact with the bis(MGD)Mo cofactor in DMSOR or *S. massilia* TMAOR are indicated by ◊. BSOR residues that have been mutated in this paper are starred, and BSOR residues that are within the Ala/Gly-X-Gly-X-X-Ala/Gly binding motifs are in boxes.

tions of BSO were the average of at least three assays, normalized for 100% Mo incorporation, and corrected for background activity. $K_{\mathrm{m}}$ and $k_{\mathrm{cat}}$ values were determined by direct fit to the Michaelis–Menten equation.

NADPH reduction of *R. sphaeroides* BSOR

Kinetic Analysis Using NADPH and NADH as the Reductants. Kinetic constants under steady-state conditions were determined aerobically at 25 °C by monitoring NADPH or NADH oxidation at 340 nm using an extinction coefficient of 6.2 mM⁻¹ cm⁻¹. Assays using NADPH concentrations greater than 0.25 mM were monitored at 380 nm using an extinction coefficient of 1.3 mM⁻¹ cm⁻¹. The assays were started by the addition of protein to a final volume of 1 mL in 50 mM Tris-HCl at pH 8.0 and monitored for 30 s on a Shimadzu UV-106 UV-visible spectrophotometer. Background activity was determined by monitoring NAD(P)H oxidation in the absence of the enzyme. All values were normalized for 100% Mo incorporation and corrected for background activity. To determine the pH dependence of BSOR, values for \( V_{\text{max}} \) and \( K_{\text{m,app}}^{\text{NADPH}} \) were determined at pH 6.0, 6.5, and 7.0 in the presence of 50 mM bis-Tris-HCl, 7.5, 8.0, and 8.5 with 50 mM Tris-HCl, and 9.0 and 9.5 with 50 mM bis-Tris propane-HCl. \( K_{\text{m,app}}^{\text{NAD(P)H}} \) and \( V_{\text{max}}^{\text{NAD(P)H}} \) values were determined in the presence of 1.7 mM BSO by a combination of Lineweaver-Burk plots, Eadie-Hofstee plots, and direct fit to the Michaelis-Menten equation. \( V_{\text{max}}^{\text{BSO}} \) and \( K_{\text{m}}^{\text{BSO}} \) values were determined at 0.25 mM NADPH or 0.25 mM NADH. The \( 1/v \)-axis intercepts were obtained from Lineweaver-Burk plots at constant, noninhibitory NADPH concentrations. Secondary replots of \( 1/v \) intercept versus [NADPH] were used to obtain the \( k_{\text{cat}}^{\text{NADPH}} \) and \( K_{\text{m}}^{\text{NADPH}} \) values (18). The values for \( k_{\text{cat}}^{\text{BSO}} \) and \( K_{\text{m}}^{\text{BSO}} \) were calculated in the same manner.

Rapid reaction kinetics of BSOR with dithionite, NADPH, or NADH were determined using a SX.18MV stopped-flow spectrophotometer (Applied Photophysics Ltd., Surrey, U.K.) with a 10-mm path length and a 1.4-ms deadtime. To prepare the instrument for anaerobic operation, the stopped-flow lines were incubated in 250 mM sodium dithionite for several hours to remove oxygen and then rinsed thoroughly with anaerobic water. All buffers, proteins, and substrates were made anaerobic prior to loading into the stopped-flow using a capped syringe, and the sample syringes were flushed continuously with nitrogen.

The change in the UV-visible spectrum during dithionite reduction of BSOR was determined by monitoring the change in absorbance over time on the stopped-flow spectrophotometer at individual wavelengths from 330 to 800 nm in 10-nm increments. While BSOR exhibits a \( \lambda_{\text{max}} \) at 720 nm characteristic of the Mo⁶⁺ form of the cofactor (3), a high degree of instrumental noise at longer wavelengths necessitated a 710-nm cutoff for data analysis. One stopped-flow syringe contained 30 μM BSOR in 50 mM Tris-HCl at pH 8.0, the second contained 300 μM sodium dithionite in the same buffer, and equal volumes of the two solutions were mixed to start the assay. Assays were monitored at 25 °C on a logarithmic time scale from 0 to 100 s, and the entire spectra at specific times were extrapolated using the software of the manufacturer. The change in the UV-visible spectrum during NADPH reduction of BSOR was determined in a similar manner with the exception that assays were monitored on a split time scale from 0 to 0.1 s and 0.1 to 2.0 s, and the stopped-flow syringes contained 72 μM BSOR and 72 μM NADPH, respectively.

The \( k_{\text{obs}} \) values for BSOR reduction at various NAD(P)H concentrations were determined at 570 nm using the stopped-flow spectrophotometer. The temperature was controlled using a circulating water bath set to 4 °C, resulting in a sample cell temperature of 6.5 °C. The NADPH concentration was varied, while the final concentration of BSOR was held constant at 15 μM. The lowest concentration of substrate was at least 4-fold in excess over the protein concentration to maintain pseudo-first-order kinetics. Rates for BSOR reduction obtained with NADPH concentrations from 0.05 to 1 mM were fit to the following equation to obtain the \( k_{\text{red}} \) and \( K_{\text{D}} \):

\[ k_{\text{obs}} = k_{\text{red}}[\text{NAD(P)H}] / (K_{\text{D}} + [\text{NAD(P)H}]) \]

Modeling of the BSOR Structure. The corrected amino acid sequence for BSOR was threaded onto the previously published structures for *R. sphaeroides* DMSOR (1eul) (10), *R. capsulatus* DMSOR (1dmr) (19), and *S. massilia* TMAOR (1tmo) (13) using the SWISS-MODEL protein modeling server (20–22). The coordinates for residues within the flexible loop (residues 359–398 in DMSOR and 365–405 in TMAOR) were removed before submission. Secondary structure assignments were from the DSSP database (23), and figures were created using the PyMOL molecular graphics system (24).

**RESULTS**

Spectroscopic and Kinetic Analysis of BSOR Reduction by NADPH. BSOR, DMSOR, and TMAOR exhibit characteristic UV-visible absorption spectra that vary depending on the reductive state and coordination geometry of the Mo center (3, 16, 25, 26). While previous studies have shown that an EPR-active Mo⁵⁺ species is generated in BSOR during partial reduction by dithionite (3), this technique cannot be used to obtain the pure Mo⁵⁺ species because the maximum amount of the wild-type protein found in the Mo⁵⁺ state during redox titrations is 30% (27). Although similar results have been observed in EPR titrations with *R. sphaeroides* DMSOR, reduction of the glycerol inhibited form of DM-SOR was shown to generate 100% of the Mo⁵⁺ state, and the absorption spectrum of this species exhibits an altered UV-visible spectra from the fully oxidized protein (26). To develop a method to analyze the appearance and lifetime of the transient Mo⁵⁺ intermediate by its absorption properties, BSOR reduction by dithionite was monitored using stopped-flow spectrophotometry. The change in absorbance over time was monitored at multiple wavelengths from 330 to 710 nm upon the addition of a 10-fold excess of sodium dithionite to *R. sphaeroides* BSOR. As seen in Figure 2A, an intermediate spectrum appears within the first 0.2 s and then disappears over the next 100 s to reveal a spectrum very similar to that previously published for dithionite-reduced BSOR (3).

To determine whether a similar intermediate is present during NADPH reduction of BSOR, this reaction was also monitored using stopped-flow spectrophotometry in the presence of equimolar concentrations of BSOR and NADPH. As seen in Figure 2B, BSOR reduction by NADPH occurred more rapidly than reduction by dithionite, and BSOR exhibited decreases in absorbance characteristic of reduction of the Mo center without the manifestation of any intermediate spectrum that would indicate the presence of a Mo⁵⁺ intermediate. While dithionite reduction of BSOR produces a rapid increase in absorbance at 640 nm followed by a

11230 Biochemistry, Vol. 43, No. 35, 2004

Nelson and Rajagopalan

**FIGURE 2:** Stopped-flow kinetic scan of BSOR reduction by NADPH. The change in the UV—visible absorbance over time was monitored at 10-nm increments from 330 to 710 nm on the stopped-flow spectrometer in 50 mM Tris-HCl at pH 8.0 and 25 °C as described in the Materials and Methods. (A) Reduction of 15 μM BSOR by 150 μM sodium dithionite. (B) Reduction of 36 μM BSOR by 36 μM NADPH. (C) Change in absorbance at 640 nm over time during reduction by 300 μM sodium dithionite. Change in absorbance over time during BSOR reduction by 36 μM NADPH at 640 (D), 340 (E), and 570 nm (F).

slower decrease (Figure 2C), reduction by NADPH produces only a monophasic decrease in absorbance at 640 nm (Figure 2D). Additionally, as shown in parts E and F of Figure 2, during NADPH reduction of BSOR, the rate and shape of the absorption change at 340 nm (which directly monitors NADPH oxidation) and 570 nm (the wavelength exhibiting the maximal change in BSOR absorbance during reduction) were identical, suggesting that both processes occur simultaneously.

The kinetic parameters for BSOR were determined under steady-state conditions. As seen in Figure 3, BSOR exhibited a curve characteristic of double competitive substrate inhibi-

**FIGURE 3:** Steady-state kinetics of BSOR with NADPH and BSO. NADPH oxidation was monitored aerobically at 340 nm as described in the Materials and Methods with 0.58 μg BSOR in 50 mM Tris-HCl at pH 8.0 and 25 °C. The NADPH concentration varied between 0.04 and 0.25 mM, while BSO was held constant at 0.17, 0.30, 0.44, 0.88, 1.7, or 3.4 mM.

NADPH reduction of *R. sphaeroides* BSOR

![Figure 4](#fig4)

FIGURE 4: Concentration dependence of BSOR reduction by NADPH and NADH. The $k_{\text{obs}}$ at increasing concentrations of NADPH (●, —) and NADH (□, ---) were determined at 6.5 °C by monitoring the change in absorbance at 570 nm on the stopped-flow spectrophotometer and fit to a hyperbolic curve.

in a noncatalytic coordination geometry that is slowly reduced or that the absence of an oxidative substrate results in a slow product dissociation step that also alters the absorption spectrum. In any case, this rate is too slow to be important under normal catalytic conditions.

The values for $k_{\text{obs}}$ obtained at different concentrations of NADPH or NADH were fit to a hyperbolic curve to obtain $k_{\text{red}}$ and $K_{\text{D}}$. The $k_{\text{red}}^{\text{NADPH}}$ is 980 s⁻¹ at 6.5 °C and even faster at 25 °C (>1200 s⁻¹). This rate was greater than the calculated $k_{\text{cat}}^{\text{NADPH}}$ at 25 °C under steady-state conditions (950 s⁻¹) and faster still than the maximal observed rate at 25 °C of 440 s⁻¹. The $K_{\text{D}}^{\text{NADPH}}$ (0.27 mM) at 6.5 °C is very similar to the calculated $K_{\text{m}}^{\text{NADPH}}$ of 0.26 mM obtained with steady-state kinetics at 25 °C. The values for $k_{\text{red}}/K_{\text{D}}$ with both NADPH (3600 mM⁻¹ s⁻¹) and NADH (210 mM⁻¹ s⁻¹) are very similar to the values for $V_{\text{max}}/K_{\text{m,app}}$ (3600 mM⁻¹ s⁻¹ with NADPH and 193 mM⁻¹ s⁻¹ with NADH) as well as the true $k_{\text{cat}}/K_{\text{m}}$ with NADPH of 3650 mM⁻¹ s⁻¹ determined as described above. No inhibition was observed at higher concentrations of NADPH and NADH, confirming that the inhibition observed with excess NAD(P)H under steady-state conditions results exclusively from interference with BSO binding and reduction.

Identification of the NADPH-binding site in BSOR would greatly facilitate understanding the electron-transfer pathway in this protein. A previous report by Pollock and Barber suggested the possibility of two NADPH-binding sites in BSOR that were utilized differently based upon the pH (14). This suggestion was based upon the observation that, under steady-state conditions, the pH profile of BSOR showed a similar $V_{\text{max}}$ at pH 6.5 and 8.0 while exhibiting a sharp drop in activity at pH 7.0 and 7.5. However, in those experiments, MOPS was used as the buffer exclusively at pH 7.0 and 7.5, the two points that exhibited a sharp decrease in activity. To determine whether this biphasic pH dependence was characteristic of the enzyme or caused by buffer interference, the $V_{\text{max}}$ and $K_{\text{m,app}}^{\text{NADPH}}$ of BSOR were determined using Tris, bis-Tris, and bis-Tris-propane as buffers throughout the pH range. Although the $V_{\text{max}}$ did decrease above pH 8.0, as previously described, no decrease in $V_{\text{max}}$ was observed at pH 7.0 or 7.5 (data not shown). In fact, little change was observed in the $V_{\text{max}}$ from pH 6 to 8.5. When MOPS was added to bis-Tris buffer at pH 7.0, a 10-fold increase was observed in the $K_{\text{m}}^{\text{NADPH}}$, confirming that the previously

described biphasic pH dependence was caused by the presence of MOPS and not by the presence of two different binding sites.

**Modeling of BSOR Structure and Identification of a Potential NADPH-Binding Site.** The rapid rate of BSOR reduction by NADPH, the double competitive inhibition observed between NADPH and BSO, the congruent rates for NADPH oxidation and BSOR reduction, and the lack of a Mo⁵⁺ intermediate during NADPH reduction of BSOR all suggest that a single binding site for NADPH is located near the Mo atom in BSOR. No structure to date has been solved for BSOR, which has made it difficult to determine the location of this site. However, automated computer programs have been developed that are able to model the 3D structure of a protein using the structures of related proteins as the templates. While the sequence identities between BSOR and *R. sphaeroides* DMSOR (44%), *R. capsulatus* DMSOR (39%), and *S. massilia* TMAOR (35%) are lower than ideal, the shared sequence is spread throughout the entire amino acid sequence (Figure 1), and all of these proteins contain the same large cofactor, making them suitable templates for modeling. Therefore, a structure for BSOR was modeled by the SWISS-MODEL protein-modeling server (20–22) using the structures for *R. sphaeroides* DMSOR (1eul) (10), *R. capsulatus* DMSOR (1dmr) (19), and *S. massilia* TMAOR (1tmo) (13) as the templates. Figure 5A shows these structures aligned with the resulting putative BSOR structure. As shown in Figure 5B, the majority of the amino acids in BSOR that do not exhibit any sequence similarity to the other proteins are scattered throughout the protein structure and surrounded by residues that are similar, confirming the overall validity of this approach. All of the residues shown in the crystal structures of *Rhodobacter* DMSOR or *S. massilia* TMAOR to interact with the bis(MGD)Mo cofactor have a high degree of sequence similarity to the equivalent residues in BSOR, and in the BSOR model structure, these residues are all in similar locations to the positions observed in DMSOR and TMAOR.

The putative BSOR structure does contain three regions that are significantly different from DMSOR and TMAOR. The first area is comprised of the initial 24 and 26 residues in DMSOR and TMAOR, respectively, that form 3 β strands located on the face opposite from the active site. These residues and, thus, the β strands that they form are missing in BSOR. A second region is comprised of residues 134–144, which are located in the active-site funnel (Figure 5B). Although these residues are present in DMSOR, they differ considerably in the predicted structure of BSOR and in the primary sequence. The third area where the putative BSOR structure differs dramatically from that of DMSOR or TMAOR comprises residues 336–361 in BSOR. Although residues 339–349 are modeled as an α helix near the bottom of the active-site funnel (Figure 5B), this assignment should be considered with caution. Various secondary structure prediction programs do not predict that these residues form a regular secondary structure element, and as modeled, they appear to block access to the substrate-binding site. Residues 339–349 correspond to a mobile loop region that includes residues 380–394 in DMSOR and residues 384–398 in TMAOR. In the structure of *S. massilia* TMAOR, this loop is located on the exterior top surface of the active-site funnel (13). While no electron density was observed for the

11232 Biochemistry, Vol. 43, No. 35, 2004

Nelson and Rajagopalan

**FIGURE 5:** Homology model of BSOR. Stereoview down the active-site funnel with the bis(MGD)Mo cofactor shown in yellow. (A) Structure of BSOR (red) aligned with the structures of *R. sphaeroides* DMSOR (black) and *S. massilia* TMAOR (gray). (B) BSOR structure with residues that exhibit sequence similarity to *R. capsulatus* DMSOR, *R. sphaeroides* DMSOR, and/or *S. massilia* TMAOR are shown in blue, and residues that exhibit no similarity are shown in red. The region in BSOR that is equivalent to the flexible loop in DMSOR and TMAOR is highlighted in purple, and the region containing residues 134–144 is shown in green.

equivalent residues in the *R. sphaeroides* structure (10), a crystal structure of *R. capsulatus* DMSOR in complex with DMSO showed the loop acting as a lid over the active-site funnel (11). In addition to these differences, the amino acid sequence preceding the loop region in BSOR shows little sequence similarity with the other proteins. While the lack of homology suggests that the model may not be accurate within this region, it does seem highly likely that the BSOR structure differs from that of DMSOR in this area.

The putative BSOR structure was analyzed to determine potential NADPH-binding sites. Previous studies have reported that NADPH does not reduce either DMSOR or TMAOR (2, 28). These experiments were repeated, and NADPH was not oxidized at TMAOR, DMSOR, and NADPH concentrations up to 200-fold higher than those used with BSOR. Thus, NADPH is expected to bind in a location where BSOR differs in sequence, while TMAOR and DMSOR are similar. Figure 6A shows the surface area of the putative BSOR active-site funnel with residues that differ from both TMAOR and DMSOR highlighted in red. As can be seen in the figure, one large group of highlighted residues, corresponding to region 2 described above, spans a distance of 30–40 Å across one side of the active-site funnel close to the residues corresponding to the flexible loop. The Gly-X-Gly-X-X-Gly motif (where X denotes any amino acid) was originally identified as part of a short loop present in a variety of NADH-binding proteins with Rossman-type folds. This fold consists of a β1-α1-β2 structure, and the loop containing the Gly-X-Gly-X-X-Gly motif is located between β1 and α1 and binds the pyrophosphate group of NAD(H) (29). This motif is present within region 3 of BSOR (Figure 6B). In NADP(H)-binding sites, the third Gly can be replaced by Ala (30), and *Cylindrocarpon tonkinense* P450nor contains an Ala-X-Gly-X-X-Ala variant of this motif that is involved in NADPH binding (31). As can be seen from Figure 6B, this Ala-X-Gly-X-X-Ala motif variant is also present in BSOR and is located within region 2. Neither of these motifs is present in the equivalent TMAOR and DMSOR amino acid sequences (Figure 1).

Analysis of the Potential NADPH-Binding Site by Mutational Analysis. To determine if the regions described above are indeed involved in NADPH binding in BSOR, mutations were made to residues within these areas. The goal was to create variants with altered affinity for NADPH that maintained protein structure, bis(MGD)Mo incorporation, and turnover with BSO. Therefore, all residues selected for mutagenesis were different from the equivalent residues in DMSOR and TMAOR and contained side chains that faced out into the active-site funnel in the putative BSOR structure. Because the side chains of arginine residues often stabilize both the pyrophosphate group present in NADPH and NADH and the additional phosphate group in NADPH (32–34), Arg130, Arg137, Arg147, and Arg455 were selected for mutagenesis. Aspartate residues provide the principal means by which many enzymes discriminate between NADP(H) and NAD(H) (35). They have also been implicated in correctly orienting the catalytic hydride in NADPH prior to extraction (36). Therefore, mutations were made to Asp135, Asp136, and Asp457. While Lys and Glu residues can function in a manner similar to Arg and Asp residues, respectively, neither of these residues is present in the putative NADPH-binding region.

The ability of all BSOR variants to utilize NADPH, NADH, and BSO was analyzed kinetically using the assays

NADPH reduction of *R. sphaeroides* BSOR

**Table 1: Specificity for NAD(P)H at Constant BSO**  
|  | NADPH | NADH |  |
| --- | --- | --- | --- |
|  | $V_{\text{max}}$ | $K_{\text{m,app}}^{\text{NADPH}}$ | $V_{\text{max}}/K_{\text{m,app}}$ | $V_{\text{max}}$ | $K_{\text{m,app}}^{\text{NADH}}$ | $V_{\text{max}}/K_{\text{m,app}}$ | ratio of |
|  | $(\text{s}^{-1})$ | $(\text{mM})$ | $(\text{mM}^{-1}\text{s}^{-1})$ | $(\text{s}^{-1})$ | $(\text{mM})$ | $(\text{mM}^{-1}\text{s}^{-1})$ | NADH/NADH$^{b}$ |
| wild type | $540 \pm 80$ | $0.15 \pm 0.07$ | $3600$ | $120 \pm 30$ | $0.62 \pm 0.09$ | $193$ | $19$ |
| R137A | $470 \pm 50$ | $0.49 \pm 0.10$ | $960$ | $130 \pm 10$ | $1.3 \pm 0.1$ | $100$ | $9.6$ |
| R130K | $530 \pm 40$ | $0.16 \pm 0.01$ | $3300$ | $140 \pm 30$ | $1.1 \pm 0.2$ | $127$ | $26$ |
| D136A | $92 \pm 20$ | $0.013 \pm 0.004$ | $7100$ | $46 \pm 24$ | $0.51 \pm 0.18$ | $90$ | $79$ |
| D136N | $100 \pm 16$ | $0.013 \pm 0.004$ | $7700$ | $53 \pm 2$ | $0.40 \pm 0.08$ | $130$ | $59$ |
| D457A | $400 \pm 20$ | $0.088 \pm 0.009$ | $4500$ | $200 \pm 50$ | $0.97 \pm 0.26$ | $210$ | $21$ |

$a$ Activity was determined at $25^\circ \text{C}$ in the presence of $1.7 \text{ mM BSO}$ in $50 \text{ mM Tris-HCl}$ at $\text{pH } 8.0$ as described in the Materials and Methods.  
$b$ Ratio of second-order rate constants.

---

**FIGURE 6:** Surface representation of the putative BSOR structure. The view down the active-site funnel of BSOR with the bis(MGD)-Mo cofactor shown in yellow. The residues equivalent to the flexible loop in DMSOR and TMAOR are shown in a ribbon representation. (A) Residues that differ in BSOR while being conserved in DMSOR and TMAOR are highlighted in red. (B) Location of Gly-X-Gly-X-X-Gly putative pyrophosphate-binding motif is highlighted in cyan. Location of Ala-X-Gly-X-X-Ala putative pyrophosphate-binding motif is highlighted in blue.

**FIGURE 7:** Location of mutagenized residues on putative BSOR structure. View down the active-site funnel of BSOR with the bis-(MGD)Mo cofactor highlighted in yellow. Arg130 is highlighted in pink; Asp136 is highlighted in green; Arg137 is highlighted in red; Asp457 is highlighted in brown; and mutations that resulted in unstable protein (Asp147, Asp135, and Arg455) are highlighted in orange. Location of Gly-X-Gly-X-X-Gly putative pyrophosphate-binding motif is highlighted in cyan. Location of Ala-X-Gly-X-X-Ala putative pyrophosphate-binding motif is highlighted in blue.

methyl viologen ($15 \text{ s}^{-1}$) (2) and the $k_{\text{cat}}$ ($50 \text{ s}^{-1}$) observed for DMSOR with MV and DMSO (25).

The mutated residues fell into two groups based upon their location along the surface of the funnel (Figure 7). Three residues, Asp147, Arg455, and Asp457, were located near each other and did not appear to specifically affect NAD-(P)H binding. Mutagenesis of Asp147 and Arg455 to Ala resulted in an unstable protein, defined by low expression levels, a low Mo incorporation, and a tendency to fall out of solution. While these residues appear to be important in maintaining the structural integrity of the protein, they did not contribute to an understanding of the NADPH-binding site and were not investigated further. Mutation of Asp457 to Ala did not significantly alter the absorption spectra, Mo incorporation, or NADPH activity from that observed for the wild type, indicating that Asp457 is not important for NADPH binding or activity.

The second group of residues was located on the other side of the funnel and includes Arg130, Asp135, Asp136, and Arg137 (Figure 7). Mutation of Asp135 and Arg130 to Ala resulted in an unstable protein; however, the presence of a positive charge in the Arg130 position was enough to restore stability, because mutation to Lys resulted in a protein with an UV-visible absorption spectrum and Mo content

Table 2: Specificity for BSO at Constant NAD(P)H<sup>a</sup>

| NADPH |  |  | NADH |  |  |
| --- | --- | --- | --- | --- | --- |
| $V_{\text{max}}$ | $K_{\text{m,app}}^{\text{BSO}}$ | $V_{\text{max}}/K_{\text{m,app}}$ | $V_{\text{max}}$ | $K_{\text{m,app}}^{\text{BSO}}$ | $V_{\text{max}}/K_{\text{m,app}}$ |
| $(s^{-1})$ | $(\text{mM})$ | $(\text{mM}^{-1}s^{-1})$ | $(s^{-1})$ | $(\text{mM})$ | $(\text{mM}^{-1}s^{-1})$ |
| wild type | $520 \pm 70$ | $0.67 \pm 0.12$ | $780$ | $52 \pm 8$ | $0.075 \pm 0.013$ | $690$ |
| R137A | $170 \pm 20$ | $0.25 \pm 0.04$ | $680$ | $24 \pm 1$ | $0.057 \pm 0.002$ | $420$ |
| R130K | $430 \pm 10$ | $0.87 \pm 0.2$ | $490$ | $46 \pm 1$ | $0.16 \pm 0.05$ | $290$ |
| D136A | $200 \pm 20$ | $1.5 \pm 0.2$ | $130$ | $22 \pm 10$ | $0.10 \pm 0.01$ | $220$ |
| D136N | $130 \pm 10$ | $1.9 \pm 0.2$ | $68$ | $20 \pm 2.6$ | $0.12 \pm 0.03$ | $170$ |
| D457A | $370 \pm 10$ | $0.69 \pm 0.05$ | $540$ | $41 \pm 3$ | $0.090 \pm 0.02$ | $460$ |

<sup>a</sup> Activity was determined at $25^\circ\text{C}$ with $0.25\text{mM NAD(P)H in } 50\text{mM Tris-HCl at pH 8.0}$ as described in the Materials and Methods.

---

Table 3: Fast Reaction Kinetics with NAD(P)H<sup>a</sup>

| NADPH |  |  | NADH |  |  |  |
| --- | --- | --- | --- | --- | --- | --- |
| $k_{\text{red}}$ | $K_{\text{D}}^{\text{NADPH}}$ | $k_{\text{red}}/K_{\text{D}}$ | $k_{\text{red}}$ | $K_{\text{D}}^{\text{NADH}}$ | $k_{\text{red}}/K_{\text{D}}$ | ratio of |
| $(s^{-1})$ | $(\text{mM})$ | $(\text{mM}^{-1}s^{-1})$ | $(s^{-1})$ | $(\text{mM})$ | $(\text{mM}^{-1}s^{-1})$ | NADPH/NADH<sup>b</sup> |
| wild type | $980 \pm 100$ | $0.27 \pm 0.05$ | $3600$ | $290 \pm 40$ | $1.4 \pm 0.4$ | $210$ | $17$ |
| R137A | $860 \pm 110$ | $0.72 \pm 0.12$ | $1200$ | $300 \pm 40$ | $2.4 \pm 0.8$ | $130$ | $9.2$ |
| R130K | $1000 \pm 100$ | $0.28 \pm 0.05$ | $3600$ | $230 \pm 50$ | $1.6 \pm 0.7$ | $140$ | $26$ |
| D136A | $550 \pm 90$ | $<0.025^{c}$ | $>22000$ | $250 \pm 20$ | $0.59 \pm 0.07$ | $420$ | $>52$ |
| D136N | $340 \pm 100$ | $<0.025^{c}$ | $>14000$ | $150 \pm 20$ | $0.48 \pm 0.10$ | $310$ | $>45$ |
| D457A | $830 \pm 20$ | $0.12 \pm 0.01$ | $6900$ | $300 \pm 60$ | $1.3 \pm 0.7$ | $230$ | $30$ |

<sup>a</sup> Activity was determined for $15\mu\text{M protein at } 4^\circ\text{C}$ using several concentrations of NAD(P)H and fit to the equation: $k_{\text{obs}} = k_{\text{red}}[\text{NAD(P)H}] / (K_{\text{D}} + [\text{NAD(P)H}])$. <sup>b</sup> Ratio of second-order rate constants. <sup>c</sup> No decrease in turnover was seen at $0.05\text{mM NADPH}$, and therefore, maximal $K_{\text{D}}$ was estimated to be $0.025\text{mM NADPH}$.

---

Table 4: Specificity for BSO Using Methyl Viologen<sup>a</sup>

|  | $k_{\text{cat}}$ | $K_{\text{m}}$ | $k_{\text{cat}}/K_{\text{m}}$ |
| --- | --- | --- | --- |
|  | $(s^{-1})$ | $(\text{mM})$ | $(\text{mM}^{-1}s^{-1})$ |
| wild type | $18 \pm 1$ | $0.018 \pm 0.007$ | $1000$ |
| R137A | $17 \pm 2$ | $0.014 \pm 0.001$ | $1200$ |
| R130K | $18 \pm 2$ | $0.024 \pm 0.001$ | $750$ |
| D136A | $9.6 \pm 1.1$ | $0.018 \pm 0.005$ | $530$ |
| D136N | $6.0 \pm 1.4$ | $0.017 \pm 0.005$ | $350$ |
| D457A | $8.7 \pm 1.2$ | $0.020 \pm 0.002$ | $440$ |

<sup>a</sup> Activity was determined at $25^\circ\text{C}$ with $0.15\text{mM methyl viologen in } 50\text{mM Tris-HCl at pH 8.0}$ as described in the Materials and Methods.

NADPH. The $K_{\text{m,app}}^{\text{NADPH}}$ for both D136A ($0.013\text{mM}$) and D136N ($0.013\text{mM}$) is at least 10-fold lower than that observed for the wild type ($0.15\text{mM}$) (Table 1). The $K_{\text{D}}^{\text{NADPH}}$ was also decreased by at least 10-fold ($<0.025\text{mM}$) from the wild-type value of $0.26\text{mM}$ (Table 3). The exact value of the $K_{\text{D}}^{\text{NADPH}}$ for D136A and D136N cannot be determined using the stopped-flow spectrophotometer, because no decrease in $k_{\text{obs}}$ was observed even when the NADPH concentration was lowered to $0.05\text{mM}$, the lowest concentration that still maintained pseudo-first-order kinetics. Although D136A and D136N also exhibited lower $K_{\text{m,app}}^{\text{NADH}}$ (Table 1) and $K_{\text{D}}^{\text{NADH}}$ (Table 3) values, the magnitude of the decrease is not as great as that observed with NADPH. Unlike the mutation of Arg 137, which only effects the $K_{\text{m}}$ and $K_{\text{D}}$, the mutation of Asp 136 alters the $V_{\text{max}}$ and $k_{\text{red}}$ with NADPH or NADH. While the $k_{\text{red}}^{\text{NADPH}}$ for D136A and D136N were 2-fold and 3-fold lower, respectively, than that of the wild type (Table 3), the rate was still faster than the $V_{\text{max}}^{\text{NADPH}}$ for both variants, which were approximately 5.5-fold lower than the wild-type value (Table 1). Although the $K_{\text{m}}^{\text{BSO}}$ for both D136A and D136N is similar to that of the wild type when determined with methyl viologen (Table 4), the $K_{\text{m,app}}^{\text{BSO}}$ values for both mutants are higher when determined with either NADPH or NADH (Table 2), suggesting that the tighter binding of NAD(P)H in D136A and D136N means that higher BSO concentrations will be required to successfully compete for the binding site. To determine whether this was the case, steady-state assays were performed at high NADPH concentrations. While minimal inhibition of wild-type BSOR was observed at NADPH concentrations below $0.25\text{mM}$, both the D16A and D136N variants were clearly inhibited at $0.050\text{mM NADPH}$ (Figure 8). When NADPH reduction of BSOR was monitored directly by stopped-flow spectrophotometry, no inhibition was observed at NADPH concentrations up to $1\text{mM}$ (data not shown).

The D136A and D136N variants also exhibited a distinct change from the wild type in their ability to bind and oxidize

NADPH reduction of *R. sphaeroides* BSOR

**FIGURE 8:** Substrate inhibition of D136A (○), D136N (△), and wild-type BSOR (♦) by NADPH. Steady-state assays were performed with 0.60 μg D136A, 2.0 μg D136N, and 0.59 μg wild-type BSOR in the presence of 1.7 mM BSO in 50 mM Tris-HCl at pH 8.0 as described in the Materials and Methods.

![Figure 8](https://i.imgur.com/yourimageurl.png)

**FIGURE 9:** Proposed mechanism for NADPH reduction of BSOR.

![Figure 9](https://i.imgur.com/yourimageurl.png)

**DISCUSSION**

The identification of amino acids residues in *R. sphaeroides* BSOR that are involved in NADPH binding and oxidation is made difficult by the fact that, while two closely related proteins have been structurally characterized, these proteins are not reduced by NADPH. In fact, *R. sphaeroides* BSOR is the only Mo-containing enzyme that has been shown to utilize NADPH to directly reduce the Mo center. Rhodobacter DMSOR and *E. coli* TMAOR are reduced by pentaheme c-type cytochromes, while other members of this family including *E. coli* DMSOR and the dissimilatory nitrate reductases contain Fe–S centers (37). While genes encoding putative BSO reductases have been identified in other bacteria, only two of these proteins have been analyzed in any detail, and neither appears to utilize NADPH as the reductant. BisC, the primary BSOR in *E. coli*, is a cytoplasmic protein (I) that is not active with NADPH and requires the presence of a small thioredoxin-like protein for activity (38). *E. coli* BisZ, also referred to as TorZ, contains a periplasmic signal sequence, can support growth anaerobically in the presence of BSO, TMAO, and DMSO, and requires the presence of a pentaheme c-type cytochrome (39). Asp136, Arg137, and the Ala-X-Gly-X-X-Ala NADPH-binding motif are not conserved in either BisC or BisZ, and both protein sequences are very similar to DMSOR and TMAOR in this region.

The data presented here show that, in the absence of a crystal structure, protein modeling has been an effective tool to evaluate the BSOR amino acid sequence for mechanistic information. In combination with kinetic, spectroscopic, and mutagenesis data, it has allowed the identification of a putative NADPH-binding site. Two residues in particular have been identified that appear to be involved in NADPH binding and oxidation. Mutation of Arg137 to Ala reduces the specificity for NADPH over NADH by half. Although mutation of Arg137 effects the kinetic values with both NADPH and NADH, the greater increases observed in $K_{\text{m,app}}$ and $K_{D}$ for NADPH than those for NADH suggest that Arg137 specifically coordinates the 2′ phosphate group of NADPH. A comparison of several protein structures with bound NADPH has shown that an Arg side chain often stacks against the plane of the adenine while hydrogen-bonding the phosphate group at the 2′ position (41). While the alteration in the kinetic parameters for the R137A variant is specific, the magnitude of this change is less than an order of magnitude, suggesting that other residues also modulate the specificity of BSOR for NADPH. Even though residues yet to be identified are likely to have an equal or greater role in binding the 2′ phosphate, Arg137 and Asp136 are the first residues in BSOR shown to interact specifically with NADPH and, thus, provide valuable information about the mechanism of this enzyme.

The concentrations and ratios of reduced and oxidized nucleotides are closely regulated within the cell, and *E. coli* contains 0.8 mM NAD⁺, 0.02 mM NADH, 0.05 mM NADP⁺, and 0.15 mM NADPH (40). Thus, even though the specificity of BSOR for NADPH is only 10-fold higher than that for NADH, for all practical purposes, NADPH is the physiological substrate for BSOR. The physiologic concentration of NADPH is actually lower than the $K_{D}^{\text{NADPH}}$ for BSOR (0.26 mM). It is likely that the higher $K_{D}^{\text{NADPH}}$ serves to prevent NADPH from inhibiting BSO binding and, thus, to allow the most efficient turnover of BSOR. This is supported by the kinetic analysis of the D136A and D136N variants, which both exhibit a lower $K_{D}$ for NADPH than that for the wild-type protein and also exhibit substrate inhibition at NADPH concentrations lower than that normally present in the cell (Figure 8). In both of these variants, the $K_{\text{m,app}}$ for BSO was greater than that for the wild type when NADPH was used as the reductant, and the $k_{\text{cat}}$ during steady-state turnover was lower. The relatively high $K_{\text{m}}$ values for both NADPH and BSO also suggest that enzyme velocity will vary in response to alterations in the cellular concentrations of both substrates.

In many NADH-binding proteins, including dehydrogenases, aspartate residues modulate the specificity for NADH versus NADPH by repulsion of the 2′ phosphate group in NADPH (35). In BSOR, Asp136 appears to exhibit a similar repulsive effect on the 2′ phosphate, because removal of the negative charge results in a 10-fold decrease in the $K_{\text{m}}^{\text{NADPH}}$ and $K_{D}^{\text{NADPH}}$. Although mutation of Asp136 lowers the $K_{D}$ for NADPH, the $k_{\text{cat}}$ and the $k_{\text{red}}^{\text{NADPH}}$ were also lower for the D136A and D136N variants than those for the wild type. As mentioned previously, Asp136 appears to increase the $K_{D}$ for NADPH to a level that allows BSO to compete more efficiently for the binding site. It also appears to have an affect upon the rate with which NADPH reduces BSOR, because the $k_{\text{red}}$ is lower in both D136A and D136N than that observed for the wild type. Thus, the repulsive effect of Asp136 toward NADPH may serve to properly orient the pyridine nucleotide to allow efficient turnover of the Mo center.

While the putative BSOR structure provides a good approximation of the overall protein fold, it does not have the accuracy and detail that a crystal structure could provide, particularly regarding the location of residues corresponding

to regions where BSOR differs from DMSOR and TMAOR. The residues that have been implicated in NADPH binding are close to each other at one end of the larger area exhibiting significant amino acid differences from DMSOR and TMAOR (Figure 7). Two potential NADPH-binding motifs were identified, a Gly-X-Gly-X-X-Gly motif within the region corresponding to the flexible loop in DMSOR and an Ala-X-Gly-X-X-Ala motif encompassing residues 138–143 along the surface of the active-site funnel. While the mutagenesis data suggest that the Ala-X-X-Gly-X-Ala motif is more likely to be involved in binding the pyrophosphate group of NADPH because of its proximity to Asp136 and Arg137, the residues corresponding to the flexible loop adjoin this area and could also be directly involved in NADPH binding. Mutation of Asp147 and Arg455, which are in a different location from the residues that are proposed to bind NADPH (Figure 7), causes structural instability. These residues may interact with residues corresponding to the flexible loop.

For the first time, stopped-flow spectrophotometry has been used to monitor BSOR reduction by both dithionite and NADPH under pre-steady-state conditions. While EPR studies have shown that a Mo<sup>V</sup> species is obtained during dithionite reduction of the Mo center (3, 27), this technique cannot be used to obtain a pure Mo<sup>V</sup> species because redox titrations of wild-type BSOR and *R. sphaeroides* DMSOR to date have reported a maximum of 0.3 spins/molybdenum (8, 27, 42). Therefore, stopped-flow spectroscopy is ideally suited to monitor the appearance of this intermediate. These studies show that, upon the addition of dithionite, BSOR manifests an intermediate spectrum that resolves to form the previously described dithionite-reduced spectrum (3). This intermediate spectrum is most likely attributed to the Mo<sup>V</sup> species because this is the only intermediate expected during dithionite reduction and it is similar to the UV–visible spectrum obtained for the glycerol-inhibited Mo<sup>V</sup> form of *R. sphaeroides* DMSOR (26). In contrast, no intermediate spectrum was observed during BSOR reduction by NADPH, and the rate of NADPH oxidation is essentially identical to the rate of reduction of the Mo center (Figure 2). In addition, even at 6.5 °C, NADPH binding and reduction of BSOR is extremely fast (800 s<sup>−1</sup>) and is not the rate-limiting step of the reaction. When taken together, these data suggest that both electrons from NADPH are transferred in a single step to reduce the Mo<sup>VI</sup> center of *R. sphaeroides* BSOR to Mo<sup>IV</sup>.

The data in this study provide strong support for a direct hydride transfer mechanism for NADPH reduction of BSOR. Stopped-flow analysis has allowed the direct monitoring of this reaction. Using this technique, it has been shown that BSOR does not exhibit a Mo<sup>V</sup> intermediate during NADPH reduction and that BSOR reduction occurs in concert with NADPH oxidation. BSO interacts with the Mo center via the active-site funnel, and the mutual competitive inhibition exhibited by NADPH and BSO is consistent with an overlapping binding site. Additional support for a hydride transfer mechanism was obtained from previous studies monitoring the isotope effect of BSOR reduction in the presence of deuterated NADPD (14). No change was observed in the *K*<sub>m</sub>, while the *V*<sub>max</sub> of (4*R*)-NADPD was 2.6-fold less than the *V*<sub>max</sub> of either NADPH or (4*S*)-NADPD. This indicates that the 4*R*-hydrogen–carbon bond of NADPH is specifically broken during the reaction. Finally, according to the putative BSOR structure, the side chains of Arg137

and Asp136 are located approximately 21 Å from the Mo atom. This distance is consistent with direct transfer of a hydride to the oxo group on the Mo, because the distance between the active carbon at the 4′ position on the nicotinamide ring and the oxo groups on the 2′ phosphate varies between 15 and 18 Å in structures of protein-bound NADPH (43–47).

These results allow the refinement of the proposed mechanism for the reductive half of the BSOR catalytic cycle (Figure 9). A hydride ion will be transferred to the single oxo group on the Mo<sup>VI</sup> center. Two electrons can then be transferred to the Mo atom resulting in a Mo<sup>IV</sup> center and a coordinated hydroxyl group. This is consistent with EXAFS analysis, indicating that the Mo<sup>IV</sup> center of BSOR coordinates with either a water or a hydroxyl molecule (3). EXAFS analysis has also indicated the presence of a similar Mo–O/N ligand in the Mo<sup>IV</sup> form of *R.* *sphaeroides* DMSOR that could arise from either a hydroxyl or water molecule (8). Although BSOR has a different physiologic electron donor and different reductive mechanism than DMSOR, the similarities in the oxidative half reactions (4) provide some support for a hydroxyl ligand in the Mo<sup>IV</sup> center of *Rhodobacter* DMSOR.

### ACKNOWLEDGMENT

We thank Sandra Jaramillo-Busquets for her help with protein expression and purification. We also thank Dr. Hermann Schindelin, Dr. Margot Wuebbens, and Dr. Heather Wilson for their critical review of this paper.

### REFERENCES

1. Pierson, D. E., and Campbell, A. (1990) Cloning and nucleotide sequence of *bisC*, the structural gene for biotin sulfoxide reductase in *Escherichia coli*, *J. Bacteriol.* 172, 2194–2198.
2. Pollock, V. V., and Barber, M. J. (1997) Biotin sulfoxide reductase: Heterologous expression and characterization of a functional molybdopterin guanine dinucleotide-containing enzyme, *J. Biol. Chem.* 272, 3355–3362.
3. Temple, C. A., George, G. N., Hilton, J. C., George, M. J., Prince, R. C., Barber, M. J., and Rajagopalan, K. V. (2000) Structure of the molybdenum site of *Rhodobacter sphaeroides* biotin sulfoxide reductase, *Biochemistry* 39, 4046–4052.
4. Garton, S. D., Temple, C. A., Dhawan, I. K., Barber, M. J., Rajagopalan, K. V., and Johnson, M. K. (2000) Resonance Raman characterization of biotin sulfoxide reductase: Comparing oxo-molybdenum enzymes in the DMSO reductase family, *J. Biol. Chem.* 275, 6798–6805.
5. Shaw, A. L., Hochkoeppler, A., Bonora, P., Zannoni, D., Hanson, G. R., and McEwan, A. G. (1999) Characterization of DorC from *Rhodobacter capsulatus*, a *c*-type cytochrome involved in electron transfer to dimethyl sulfoxide reductase, *J. Biol. Chem.* 274, 9911–9914.
6. Gon, S., Giudici-Orticoni, M.-T., Méjean, V., and Iobbi-Nivol, C. (2001) Electron transfer and binding of the *c*-type cytochrome TorC to the trimethylamine *N*-oxide reductase in *Escherichia coli*, *J. Biol. Chem.* 276, 11545–11551.
7. Raitsimring, A. M., Astashkin, A. V., Feng, C., Enemark, J. H., Nelson, K. J., and Rajagopalan, K. (2003) Pulsed EPR studies of the exchangeable proton at the molybdenum center of dimethyl sulfoxide reductase, *J. Biol. Inorg. Chem.* 8, 95–104.
8. George, G. N., Hilton, J., Temple, C., Prince, R. C., and Rajagopalan, K. V. (1999) The structure of the molybdenum site of dimethyl sulfoxide reductase, *J. Am. Chem. Soc.* 121, 1256–1266.
9. Denessiouk, K. A., Rantanen, V. V., and Johnson, M. S. (2001) Adenine recognition: A motif present in ATP-, CoA-, NAD-, NAD-, and FAD-dependent proteins, *Proteins* 44, 282–291.
10. Li, H. K., Temple, C. A., Rajagopalan, K. V., and Schindelin, H. (2000) The 1.3 Å crystal structure of *Rhodobacter sphaeroides*

NADPH reduction of *R. sphaeroides* BSOR

reductase reveals two distinct molybdenum coordination environments, *J. Am. Chem. Soc.* 122, 7673–7680.

11. McAlpine, A. S., McEwan, A. G., and Bailey, S. (1998) The high-resolution crystal structure of DMSO reductase in complex with DMSO, *J. Mol. Biol.* 275, 613–623.

12. Schneider, F., Löwe, J., Huber, R., Schindelin, H., Kisker, C., and Knäblein, J. (1996) Crystal structure of dimethyl sulfoxide reductase from *Rhodobacter capsulatus* at 1.88 Å resolution, *J. Mol. Biol.* 263, 53–69.

13. Czjzek, M., Dos Santos, J.-P., Pommié, J., Giordano, G., Mejean, V., and Haser, R. (1998) Crystal structure of oxidized trimethylamine-N-oxide reductase from *Shewanella massilia* at 2.5 Å resolution, *J. Mol. Biol.* 284, 435–447.

14. Pollock, V. V., and Barber, M. J. (2001) Kinetic and mechanistic properties of biotin sulfoxide reductase, *Biochemistry* 40, 1430–1440.

15. Pollock, V. V., and Barber, M. J. (1995) Molecular cloning and expression of biotin sulfoxide reductase from *Rhodobacter sphaeroides* forma sp. *Denitrificans*, *Arch. Biochem. Biophys.* 318, 322–332.

16. Hilton, J. C., Temple, C. A., and Rajagopalan, K. V. (1999) Redesign of *Rhodobacter sphaeroides* DMSO reductase: Enhancement of adenosine N-oxide reductase activity, *J. Biol. Chem.* 274, 8428–8436.

17. Melville, D. (1954) Biotin sulfoxide, *J. Biol. Chem.* 208, 495–501.

18. Segel, I. H. (1975) *Enzyme Kinetics: Behavior and Analysis of Rapid Equilibrium and Steady-State Enzyme Systems*, pp 826–831, John Wiley and Sons, New York.

19. McAlpine, A. S., McEwan, A. G., Shaw, A. L., and Bailey, S. (1997) Molybdenum active centre of DMSO reductase from *Rhodobacter capsulatus*: Crystal structure of the oxidised enzyme at 1.82-Å resolution and the dithionite-reduced enzyme at 2.8-Å resolution, *J. Biol. Inorg. Chem.* 2, 690–701.

20. Gueux, N., and Peitsch, M. C. (1997) SWISS-MODEL and the Swiss-PdbViewer: An environment for comparative protein modeling, *Electrophoresis* 18, 2714–2723.

21. Peitsch, M. C. (1995) Protein modeling by E-mail, *Biotechnology* 13, 658–660.

22. Peitsch, M. C. (1996) ProMod and Swiss-Model: Internet-based tools for automated comparative protein modeling, *Biochem. Soc. Trans.* 24, 274–279.

23. Kabsch, W., and Sander, C. (1983) Dictionary of protein secondary structure: pattern recognition of hydrogen-bonded and geometrical features, *Biopolymers* 22, 2577–2637.

24. DeLano, W. L. (2002) The PyMOL Molecular Graphics System, http://www.pymol.org.

25. Johnson, K., and Rajagopalan, K. (2001) An active site tyrosine influences the ability of the dimethyl sulfoxide reductase family of molybdopterin enzymes to reduce S-oxides, *J. Biol. Chem.* 276, 13178–13185.

26. Finnegan, M. G., Hilton, J., Rajagopalan, K. V., and Johnson, M. K. (1993) Optical transitions of molybdenum(V) in glycerol-inhibited DMSO reductase from *Rhodobacter sphaeroides*, *Inorg. Chem.* 32, 2616–2617.

27. Pollock, V. V., Conover, R. C., Johnson, M. K., and Barber, M. J. (2003) Biotin sulfoxide reductase: Tryptophan 90 is required for efficient substrate utilization, *Arch. Biochem. Biophys.* 409, 315–326.

28. Shimokawa, O., and Ishimoto, M. (1979) Purification and some properties of inducible tertiary amine N-oxide reductase from *Escherichia coli*, *J. Biochem.* 86, 1709–1717.

29. Wierenga, R. K., Terpstra, P., and Hol, W. G. J. (1986) Prediction of the occurrence of the ADP-binding βαβ-fold in proteins, using an amino acid sequence fingerprint, *J. Mol. Biol.* 187, 101–107.

30. Bragg, P. D., Glavas, N. A., and Hou, C. (1997) Mutation of conserved residues in the NADP(H)-binding domain of the proton translocating pyridine nucleotide transhydrogenase of *Escherichia coli*, *Arch. Biochem. Biophys.* 338, 57–66.

31. Kudo, T., Tomura, D., Liu, D. L., Dai, X. Q., and Shoun, H. (1996) Two isozymes of P450nor of *Cylindrocarpon tonkinense*: Mo-

molecular cloning of the cDNAs and genes, expressions in the yeast, and the putative NAD(P)H-binding site, *Biochimie* 78, 792–799.

32. Vought, V., Ciccone, T., Davino, M. H., Fairbane, L., Lin, Y., Cosgrove, M. S., Adams, M. J., and Levy, H. R. (2000) Delineation of the roles of amino acids involved in the catalytic functions of *Leuconostoc mesenteroides* glucose-6-phosphate dehydrogenase, *Biochemistry* 39, 15012–15021.

33. Serre, L., Vellieux, F., Medina, M., Gomez-Moreno, C., Fontecilla-Camps, J., and Frey, M. (1996) X-ray structure of the Ferredoxin: NADP⁺ reductase from the Cyanobacterium *Anabaena* PCC 7119 at 1.8 Å resolution, and crystallographic studies of NADP⁺ binding at 2.25 Å resolution, *J. Mol. Biol.* 263, 20–39.

34. Levy, H., Vought, V., Yin, X., and Adams, M. (1996) Identification of an arginine residue in the dual coenzyme-specific glucose-6-phosphate dehydrogenase from *Leuconostoc mesenteroides* that plays a key role in binding NADP⁺ but not NAD⁺, *Arch. Biochem. Biophys.* 326, 145–151.

35. Lesk, A. (1995) NAD-binding domains of dehydrogenases, *Curr. Opin. Struct. Biol.* 5, 775–783.

36. Korkhin, Y., Kalb, J., Peretz, M., Bogin, O., Burstein, Y., and Frolow, F. (1998) NADP-dependent bacterial alcohol dehydrogenases: Crystal structure, cofactor-binding, and cofactor specificity of the ADHs of *Clostridium beijerinckii* and *Thermoanaerobacter brockii*, *J. Mol. Biol.* 278, 967–981.

37. Kisker, C., Schindelin, H., and Rees, D. C. (1997) Molybdenum-cofactor-containing enzymes: Structure and mechanism, *Annu. Rev. Biochem.* 66, 233–267.

38. del Campillo-Campbell, A., Dykhuizen, D., and Cleary, P. P. (1979) Enzymatic reduction of *d*-biotin-*d*-sulfoxide to *d*-biotin, *Methods Enzymol.* 62, 379–385.

39. Gon, S., Patte, J.-C., Méjean, V., and Iobbi-Nivol, C. (2000) The *torYZ* (*yecK bisZ*) operon encodes a third respiratory trimethylamine N-oxide reductase in *Escherichia coli*, *J. Bacteriol.* 182, 5779–5786.

40. Penfound, T., and Foster, J. W. (1996) Biosynthesis and recycling of NAD, in *E. coli and Salmonella: Cellular and Molecular Biology* (Neidhardt, F. C., Ed.) pp 721, ASM Press, Washington, D. C.

41. Carugo, O., and Argos, P. (1997) NADP-dependent enzymes: Conserved stereochemistry of cofactor binding, *Proteins: Struct., Funct.*, Genet. 28, 10–28.

42. Bastian, N. R., Kay, C. J., Barber, M. J., and Rajagopalan, K. V. (1991) Spectroscopic studies of the molybdenum-containing dimethyl sulfoxide reductase from *Rhodobacter sphaeroides* f. sp. *denitrificans*, *J. Biol. Chem.* 266, 45–51.

43. Gourley, D., Schuttelkopf, A., Leonard, G., Luba, J., Hardy, L., Beverly, S., and Hunter, W. (2001) Pteridine reductase mechanism correlates pterin metabolism with drug resistance in Trypanosomatid parasites, *Nat. Struct. Biol.* 8, 521–525.

44. Liao, D. I., Thompson, J. E., Fahnestock, S., Valent, B., and Jordan, D. B. (2001) A structural account of substrate and inhibitor specificity differences between two naphthol reductases, *Biochemistry* 40, 8696–8704.

45. Mulichak, A. M., Bonin, C. P., Reiter, W. D., and Garavito, R. M. (2002) The structure of the MurI Gdp-Mannose 4,6-dehydratase from *A. thaliana*: Implications for ligand binding and specificity, *Biochemistry* 41, 15578–15589.

46. Whitlow, M., Howard, A. J., Stewart, D., Hardman, K. D., Kuypers, L. F., Baccanari, D. P., Fling, M. E., and Tansik, R. L. (1997) X-ray crystallographic studies of *Candida albicans* dihydrofolate reductase. High-resolution crystal structures of the holoenzyme complex and an inhibited ternary complex, *J. Biol. Chem.* 272, 30289–30298.

47. Xiong, J. P., Xia, Z. X., and Wang, Y. (1994) Crystal structure of trichosanthin–NADPH complex at 1.7 Å reveals active-site architecture, *Nat. Struct. Biol.* 1, 695–700.
