
JOURNAL OF BACTERIOLOGY, Mar. 1992, p. 1810–1820

0021-9193/92/061810-11$02.00/0

Copyright © 1992, American Society for Microbiology

# Nucleotide Sequence of the *tcmII-tcmIV* Region of the Tetracenomycin C Biosynthetic Gene Cluster of *Streptomyces glaucescens* and Evidence that the *tcmN* Gene Encodes a Multifunctional Cyclase–Dehydratase–O-Methyl Transferase

RICHARD G. SUMMERS,${}^{1}$ EVELYN WENDT-PIENKOWSKI,${}^{1}$ HAIDEH MOTAMEDI,${}^{1\ddagger}$ AND C. R. HUTCHINSON${}^{1,2*}$

School of Pharmacy${}^{1}$ and Department of Bacteriology,${}^{2}$ The University of Wisconsin–Madison, Madison, Wisconsin 53706

Received 3 October 1991/Accepted 30 December 1991

Mutations in the *tcmII-tcmIV* region of the *Streptomyces glaucescens* chromosome block the C-3 and C-8 O-methylations of the polyketide antibiotic tetracenomycin C (Tcm C). The nucleotide sequence of this region reveals the presence of two genes, *tcmN* and *tcmO*, whose deduced protein products display similarity to the hydroxyindole O-methyl transferase of the bovine pineal gland, an enzyme that catalyzes a phenolic O-methylation analogous to those required for the biosynthesis of Tcm C. The deduced product of the *tcmN* gene also has an N-terminal domain that shows similarity to the putative ActVII and WhiE ORFVI proteins of *Streptomyces coelicolor*. The *tcmN* N-terminal domain can be separated from the remainder of the *tcmN* gene product, and when coupled on a plasmid with the Tcm C polyketide synthase genes (*tcmKLM*), this domain enables high-level production of an early, partially cyclized intermediate of Tcm C in a Tcm C${}^{-}$ null mutant or in a heterologous host (*Streptomyces lividans*). By analogy to fatty acid biosynthesis, the *tcmKLM* polyketide synthase gene products are probably sufficient to produce the linear decaketide precursor of Tcm C; thus, the *tcmN* N-terminal domain is most likely responsible for one or more of the early cyclizations and, perhaps, the attendant dehydrations that lead to the partially cyclized intermediate. The *tcmN* gene therefore appears to encode a multifunctional cyclase–dehydratase–3-O-methyl transferase. The *tcmO* gene encodes the 8-O-methyl transferase.

Polyketide metabolites are a structurally diverse family of compounds that encompasses both aromatic and aliphatic members (Fig. 1A). These molecules are commonly synthesized via secondary metabolic pathways in bacteria, fungi, and plants (10). Importantly, many of these compounds have found clinical utility as antibiotics and chemotherapeutic agents (10).

The feature that binds the seemingly disparate polyketide family together is the mechanism of biosynthesis. The carbon backbone of a polyketide is synthesized in a manner that is similar to long-chain fatty acid biosynthesis: small fatty acid units (acetate, propionate, butyrate, etc.) are sequentially condensed to yield extended linear precursors. A notable difference distinguishes polyketide biosynthesis from fatty acid biosynthesis, however. The condensation reactions of polyketide chain growth are not always followed by the cycles of reduction and dehydration that characterize the synthesis of fatty acids. As a result, the linear intermediate of polyketide biosynthesis is peppered with reactive carbonyl groups that form the basis for subsequent chemical elaborations.

Tetracenomycin C (Tcm C) is a relatively simple aromatic polyketide antibiotic that is produced by *Streptomyces glaucescens* (25). The tetracyclic backbone of this molecule is most likely formed by cyclization of a 20-carbon linear decaketide (Fig. 1B). Previously, we cloned the genes required for the biosynthesis of Tcm C from *S.* glaucescens

(14) and reported the DNA sequence of the polyketide synthase (PKS) genes from the *tcmIa* region that should catalyze the formation of the C${}_{20}$ decaketide (2). Here we report the DNA sequence of the two genes that lie downstream of the PKS genes and provide genetic and biochemical evidence that the first of these genes (*tcmN*) encodes a multifunctional enzyme that probably catalyzes a cyclization, dehydration, and O-methylation. The second gene (*tcmO*) most likely encodes an additional O-methyl transferase. We believe that these genes mark one end of the Tcm C biosynthetic gene cluster.

## MATERIALS AND METHODS

### Bacterial strains, bacteriophage, and plasmids. The *S.* glaucescens type strains (16), *Streptomyces lividans* TK24 (9), the bacteriophages M13mp18 and M13mp19 (27), and the plasmids pIJ702 (12), pIJ486 (24), pELE37 (6), and pHM57 (14) have been described elsewhere. The *S.* glaucescens type IV strain (GLA.12-41) is a deletion mutant that lacks about 390 bp of the *tcmO* gene (25a). Plasmids pGEM-3Zf(−), -5Zf(+), and -7Zf(+) were purchased from Promega (Madison, Wis.), and *Escherichia coli* DH5α was purchased from Bethesda Research Laboratories, Inc. (Gaithersburg, Md.).

### Media, enzymes, and chemicals. R2YENG liquid medium (pH 6) and agar plates (14) and soft nutrient agar (8) were prepared as described previously. YT medium contains tryptone (0.8%, wt/vol), yeast extract (0.5%, wt/vol), and NaCl (0.5%, wt/vol). Dry medium and Bacto-Agar were purchased from Difco Laboratories (Detroit, Mich.), and amino acids were supplied by United States Biochemical

* Corresponding author.
† Present address: Microbial Chemotherapeutics and Molecular Genetics, Merck, Sharp and Dohme Research Laboratories, Rahway, N.J. 07065.

Vol. 174, 1992

TETRACENOMYCIN C BIOSYNTHESIS BY S. GLAUCESCENS 1811

**A**

![Chemical Structure of Tcm C]

Tcm C

![Chemical Structure of 6-DEB]

6-DEB

**B**

![Chemical Structures of Putative Early Intermediates of Tcm C]

H₂O

![Chemical Structure of Tcm F2]

Tcm F2

FIG. 1. (A) Representative polyketide metabolites Tcm C and 6-deoxyerythronolide B (6-DEB). (B) Putative early intermediates of Tcm C. The linear decaketide and the presumed bicyclic and tricyclic precursors of Tcm C are shown. The *tcmKLM* polyketide synthase genes and the first one-third of the multifunctional *tcmN* gene are responsible for the biosynthesis of Tcm F2 or one of its immediate precursors. Nine additional steps are then required to convert Tcm F2 to Tcm C.

(Cleveland, Ohio). The salts CaCl₂, KH₂PO₄, MgCl₂, NaCl, NaOH, and NaNO₃ were obtained from Mallinckrodt, Inc. (Paris, Ky.), and K₂SO₄ was from Columbus Chemical Industries (Columbus, Wis.). TES buffer was purchased from Sigma Chemical Co. (St. Louis, Mo.). Restriction endonucleases were obtained from Amersham Corp. (Arlington Heights, Ill.), Bethesda Research Laboratories, New England Biolabs (Beverly, Mass.), Promega, or United States Biochemical. T4 DNA ligase was supplied by Bethesda Research Laboratories. Buffers for these enzymes were either used as provided or prepared as described elsewhere (19). Protein A-Sepharose CL-4B was from Sigma, and the antibiotic thiostrepton was supplied by Sal Lucania at the Squibb Institute for Medical Research (Princeton, N.J.).

DNA sequence analysis. Pieces (≤1.2 kb) of the 3.6-kb *BglII-KpnI* fragment that carries the *tcmII-tcmIV* region were subcloned from a derivative of pHM57 into replicative-form M13mp18 and M13mp19. Single-strand DNA templates were then prepared and sequenced by the Sequenase Ver-

sion 2.0 method, using 7-deaza-dGTP (United States Biochemical) and 2′-deoxycytidine 5′-[α-³⁵S]thiotriphosphate (Amersham). Samples were run on 60-cm wedged urea-polyacrylamide gels that were supplemented with formamide (10%). Overlapping sets of subclones were generated, and all restriction endonuclease sites that were used for subcloning were sequenced across to confirm their place in the sequence. In this fashion, the complete nucleotide sequence of both strands of the 3.6-kb fragment was determined. Open reading frames (ORFs) in this sequence were identified by the UWGCG (5) program CODON PREFERENCE, and inverted repeat sequences were located with the STEM-LOOP program. The estimated free energies (∆G) for the reported stem-loop structures were calculated as described previously (23). To obtain the peptide sequences encoded by the various ORFs, we used TRANSLATE (5), and the amino acid composition of these peptides was determined by using PEPTIDESORT (5). The similarity between the *tcmN* and *tcmO* gene products and the bovine hydroxyindole O-methyl transferase was discovered by using TFASTA (5) to search the UWGCG GenBank (Release 60.0) and EMBL (Release 19.0) genetic sequence data banks. Protein comparisons were made with the COMPARE (5) program and were visualized by using DOTPLOT (5). The percent identity between related amino acid sequences was determined by using GAP (5).

Complementation analysis. The DNA fragments described in Tables 1 and 2 were subcloned in the coding orientation downstream of the melanin gene (*melC1*) promoter of pIJ702. Frequently, the fragments were passed first through pGEM-3Zf(−), -5Zf(+), or -7Zf(+) to obtain suitable flanking restriction sites for subsequent introduction into pIJ702. The *BssHII-MroI*, *RsrII-MroI*, *XmaI-MroI*, and *PvuII-SplI* constructs were designed so that at least one translational stop codon from the pGEM-7Zf(+) multiple cloning site or a synthetic oligonucleotide linker intervenes between the *melC* genes and the inserted fragment in the reading frame of interest. This is presumed to prevent translational readthrough that would complicate interpretation of the results.

Protoplasts of the *S. glaucescens* type strains (and, for Table 2, the additional type II mutants GLA.5-20 and GLA.11-34 [16]) were prepared and transformed with the various plasmids and pIJ702 (as a control) by the method of Hopwood et al. (8) with minor modifications. Cells were then plated on R2YENG and incubated overnight at 30°C. Thiostrepton-resistant transformants were selected by overlaying the plates with soft nutrient agar that contained sufficient thiostrepton to give a final concentration of 20 μg/ml and then growing the cells at 30°C for two to three more days. Twelve resistant colonies from each transformation were transferred to master Hickey-Tresner sporulation plates (16) and to R2YENG plug plates (96-well microtiter plates), both containing thiostrepton (50 μg/ml). After 3 days of incubation at 30°C, the plugs were screened for the Tcm C⁺ phenotype by bioassay against a thiostrepton-resistant derivative of *Streptomyces coelicolor* M111 (*hisA1 uraA1 strA1 SCP1 SCP2*²) as described previously (16). Simultaneously, three colonies from each transformation were selected arbitrarily, and cultures were grown in 5 ml of R2YENG (containing 10 μg of thiostrepton per ml) at 30°C for 3 days. The cultures were then extracted with 1 ml of ethyl acetate, and 50 μl of this extract was analyzed for the presence of Tcm C and its fluorescent biosynthetic intermediates by thin-layer chromatography on silica gel plates (chloroform-methanol, 4:1).

Construction and analysis of the MBP-Tcm fusion proteins. To prepare the *malE-tcmN* gene fusion, we introduced a 1.6-kb *BssHII* fragment that carries all but the first three codons of the *tcmN* ORF into pGEM-7Zf(+) (*MluI* digested). From a resulting plasmid that contained the insert in the appropriate orientation, the *tcmN* sequences were then transferred as a *BamHI-NsiI* fragment into the *BamHI* and *PstI* sites of pMAL-c (New England Biolabs), thus forming the gene fusion. The *malE-tcmO* fusion was constructed similarly: an 875-bp *SalI-BamHI* fragment that carries the last 168 codons of the *tcmO* ORF was first subcloned into pGEM-7Zf(+) (*XhoI-BamHI* digested) and was then moved as an *XbaI-NsiI* fragment into pMAL-c, compatibly digested. Correct assembly of both gene fusions was verified by restriction analysis.

The hybrid proteins encoded by the gene fusions were analyzed in *E. coli* DH5α. Transformed cells were grown overnight at 37°C in YT medium containing ampicillin (100 μg/ml). The cells from 1 ml of culture were then harvested by centrifugation and resuspended in fresh YT medium containing ampicillin (100 μg/ml) and IPTG (isopropyl-β-D-thiogalactopyranoside) (1 mM), and growth at 37°C was allowed to continue for 10 min. (During overnight growth under the above conditions, the pMAL-c promoter is not fully repressed and longer induction with IPTG is not necessary.) Next, the cells were pelleted and resuspended in 25 mM Tris-HCl buffer (pH 8) containing sucrose (300 mM), EDTA (25 mM), and lysozyme (4 mg/ml). After incubation at 4°C for 15 min, the cells were repelleted and then lysed by resuspension in 1 ml of chilled 10 mM Tris-HCl buffer (pH 8) containing EDTA (1 mM), Triton X-100 (0.5%, vol/vol), and phenylmethylsulfonyl fluoride (1.5 mM). Cellular debris was removed by centrifugation at 12,000 × g for 15 min at 4°C, and 900 μl of the supernatant was then treated with 3 μl of rabbit anti-maltose-binding protein (MBP) serum (New England Biolabs). After 30 min of incubation at 4°C, protein A-Sepharose CL-4B (150 μl of a suspension of 2.5%, wt/vol) was added, and the incubation was continued at 4°C with occasional mixing for an additional 60 min. Immune complexes were then collected by centrifugation, and the pellets were washed four times as described elsewhere (22). The protein species in the pellets were visualized by Coomassie brilliant blue staining of denaturing polyacrylamide slab gels (10.5%) that had been prepared and run as described previously (22).

Construction and analysis of pWHM720, pWHM721, pWHM722, and pWHM723. To prepare pWHM720, a synthetic oligonucleotide linker (upper strand, 5′-CCGGGTATGCATCTAGGATCCT-3′; lower strand, 3′-CAC TACGTAGATCCTAGGAGATC-5′) that encodes two in-frame translational stop codons (underlined) and a downstream *BamHI* site was used to terminate the *tcmN* gene after the codon for Gly-117, as follows. The 300-bp *SstI-XmaI* fragment that carries codons 16 to 117 of the *tcmN* gene was subcloned into pGEM-7Zf(+), similarly digested. The linker was then introduced into the resulting plasmid between the *XbaI* and *XmaI* sites. This places the first stop codon of the linker immediately downstream of the GGG triplet that encodes Gly-117. Direct transfer of the linker-terminated *tcmN* gene fragment to pELE37 as an *SstI-BamHI* piece was not convenient because pELE37 contains a number of *SstI* sites. Therefore, the *tcmN* gene fragment was first enlarged to include the upstream and adjacent 1.25-kb *MluI-SstI* fragment from the *tcmIa* region (2), and this enlarged segment was then transferred as a 1.55-kb *MluI-BamHI* fragment to pELE37, analogously digested.

A related strategy was used to build pWHM721. A synthetic oligonucleotide linker (upper strand, 5′-CCGGGCGC CTGTGATGCATCTAG-3′; lower strand, 3′-TCGAGGCC CGGCGGACACTACGTAGATCCTAG-5′) that again encodes two in-frame translational stop codons (underlined) and a downstream *BamHI* site (as well as an internal *NarI* site) was introduced into pGEM-7Zf(+) at the *SstI* and *BamHI* sites. The 480-bp *SstI-NarI* fragment that carries codons 16 to 177 of the *tcmN* gene was then transferred into the resulting plasmid, similarly digested. This places the first stop codon of the linker immediately downstream of the CTG triplet that encodes Leu-177. The linker-terminated *tcmN* fragment was then enlarged and transferred to pELE37, exactly as before. Correct assembly of both pWHM720 and pWHM721 was verified by DNA sequencing. Neither pWHM720 nor pWHM721 carries any *tcmN* sequences downstream from the location of the linker.

Plasmid pWHM723 was produced by transferring the 4.1-kb *MluI-BamHI* fragment that encodes the *tcmN* and *tcmO* genes from a derivative of pHM57 into pELE37, similarly digested. The construction was verified by restriction analysis and complementation behavior (*tcmIa*⁺ *tcmII*⁺ *tcmIV*⁺). Plasmid pWHM722 was derived from pWHM723 by removing most of the *tcmO* gene: pWHM723 was digested with *NcoI* and *BamHI*, the sticky ends were filled in with the Klenow fragment of DNA polymerase I, and the plasmid was recircularized by ligation. Loss of the *tcmO* sequences was verified by restriction analysis and complementation behavior (*tcmIa*⁺ *tcmII*⁺ *tcmIV*).

Protoplasts of *S.* lividans TK24 and *S.* glaucescens GLA.5-1 (16) were prepared and transformed to thiostrepton resistance with the various plasmids and pIJ486 (the parent of pELE37) as described earlier. Three colonies from each transformation were picked and grown in 5 ml of R2YENG (containing 20 μg of thiostrepton per ml) at 30°C for 3 to 4 days. The pH of each culture was then lowered by the addition of 1 M NaH₂PO₄ (pH 5.3) (500 μl), and the culture was extracted with 1 ml of ethyl acetate. A portion (50 μl) of the extract was then analyzed for the presence of Tcm C and its fluorescent intermediates by thin-layer chromatography, as described above.

Nucleotide sequence accession number. The DNA sequence data described in this paper have been deposited at EMBL and GenBank with the accession number M80674.

---

**RESULTS**

Sequence analysis of the *tcmII-tcmIV* region. In a previous study, we reported the DNA sequence of the Tcm C PKS genes (*tcmIa*) (2). Downstream from these genes is a region that has been defined by mutation to affect methylation of the C-3 (*tcmII-tcmV*) and C-8 (*tcmIV*) hydroxyl groups of Tcm C (15, 16, 28). To further characterize this region, we determined the nucleotide sequence of the 3.6-kb *BglII-KpnI* DNA fragment that encompasses the *tcmII*, *tcmIV*, and *tcmV* loci (Fig. 2). Within this segment, several ORFs with characteristics of a *Streptomyces* gene were discerned by the CODON PREFERENCE program (Fig. 3). The sequence reported here overlaps the last 168 nucleotides (nt) of the previously reported *tcmIa* sequence, and the end of the *tcmM* gene (formerly, *tcmIa* ORF3) is observed at the beginning of the third reading frame (Fig. 3, bottom panel). One base downstream from the *tcmM* gene, at position 118, the first complete ORF in this sequence begins (top panel), continuing to a TAG stop codon at position 1600. This ORF (*tcmN*) would encode a polypeptide of *M*ᵣ = 55,900.

second complete ORF starts at position 1699 (top panel) and extends to a TGA stop codon at position 2716. The second ORF (*tcmO*) would encode a polypeptide of *M*<sub>r</sub> = 37,000. Both ORFs are preceded by possible ribosome-binding sites based on complementarity to a region near the 3′ end of *S.* lividans 16S rRNA (3) (Fig. 2). The CODON PREFERENCE analysis also revealed a reading frame with several characteristics of a protein-coding sequence downstream from the first two ORFs (Fig. 3, bottom panel). No mutations that affect Tcm C biosynthesis have been mapped to this region, however, and it may be that this portion of the DNA sequence lies outside the Tcm C biosynthetic gene cluster.

The *tcmII-tcmIV* region is punctuated by several major inverted repeats that could form stem-loop structures. In the gap between the first and second ORFs there is a G+C-rich 12-nt sequence and its inverted repeat (Fig. 2; calculated ΔG = −40.6 kcal/mol [23]). There is also a 9-nt sequence and its repeat downstream from the second ORF (positions 2787 to 2816; ΔG = −30.4 kcal/mol). Finally, two overlapping 8-nt sequences and their repeats occupy positions 3144 through 3190 (ΔG = −21.6 and −23.6 kcal/mol, respectively). The placement of these inverted repeats, falling after the putative coding regions, is consistent with roles as transcriptional termination signals and/or mRNA stability determinants (17).

Complementation analysis of the *tcmII-tcmIV* region. To determine whether the two ORFs that were defined by the computer analysis correspond to functional units in the Tcm C biosynthetic pathway, we prepared a series of DNA fragments from the *tcmII-tcmIV* region and assessed the ability of these fragments to complement various Tcm C− mutants in bioassays against *S.* coelicolor (Fig. 4B and Tables 1 and 2). Positive bioassay results were verified by detection of Tcm C after thin-layer chromatography of culture extracts. The 1.6-kb *BglII-MroI* fragment that carries the entire first ORF complements both the type II and type V mutations but does not complement any other class of Tcm C− mutation (Table 1, fragment a). The second ORF, on the other hand, represented by a 1.2-kb *MroI-SplI* fragment, complements the type IV mutation (Table 1, fragment f).

Unexpectedly, when the first ORF is shortened at its 5′ end to remove the putative ribosome-binding site and the first two codons, its ability to complement the type II and type V mutations is only slightly diminished (Table 2, fragment b). Indeed, complementation is still observed even after the initial 78 or 117 codons of the first ORF are removed (fragments c and d, respectively). These findings indicate that translation of the first ORF can initiate at one or more internal positions (there are candidate ATG codons at positions 457, 490, and 526) and that the beginning portion of the ORF is not required to complement the 3-O-methyl transferase defect of the type II and type V mutants. Yet, the reduced level of complementation displayed by all the 5′-truncated constructions suggests that the first start site (at position 118) is used in vivo. At the other end of the ORF, removal of the last 21 codons and the putative TAG stop codon greatly reduces complementation of both the type II and type V mutations (fragment e).

When either the proposed beginning or end of the second ORF is deleted, its ability to complement the type IV mutation is lost (Table 2, fragments g and h, respectively). The complementation analysis indicates that the two ORFs are functional units in the biosynthesis of Tcm C and that the ORF boundaries, as delineated by the computer analysis, are likely to be correct, except that the first ORF

may have alternative start sites. The first ORF complements both the type II and type V mutations. Indeed, only the C-terminal three-fourths of the ORF are necessary for the complementation of these mutations. In agreement with the behavior of the type II and type V mutants in these experiments, we had previously noted that these mutations were similar based on the results of cosynthesis experiments with other classes of Tcm C− mutants (16). The two mutant types had been grouped separately, however, because the pattern of extractable metabolites differed slightly between them. In light of the present results, the slight differences between the type II and type V mutations seem less important than their similarities, and we conclude that the type II and type V mutations are alleles of the same genetic locus. Since the type II and type V mutants are defective in the methylation of the C-3 position of Tcm C (16, 28), it appears that the first ORF (*tcmN*) encodes the 3-O-methyl transferase. The second ORF complements the type IV mutation. This deletion mutant is unable to methylate the C-8 position of Tcm C (16, 28); thus, the second ORF (*tcmO*) most likely encodes the 8-O-methyl transferase.

Portions of the deduced proteins of the *tcmN* and *tcmO* genes are similar to an O-methyl transferase, but the deduced N terminus of TcmN resembles the predicted ActVII, Gra4, and WhiEVI proteins. Consistent with the earlier evidence that the *tcmII-tcmIV* region influences the methylation of the C-3 and C-8 positions of Tcm C (16, 28), a COMPARE-DOT-PLOT analysis indicates that the predicted proteins of both the *tcmN* and *tcmO* genes resemble the hydroxyindole O-methyl transferase of the bovine pineal gland (11) (Fig. 5A and B). The TcmN and TcmO proteins resemble one another, as well (Fig. 5C). Like the TcmN and TcmO O-methyl transferases, the bovine enzyme catalyzes the methylation of a phenolic hydroxyl group (1, 11). It is noteworthy, however, that the putative TcmN protein exhibits similarity to the other proteins only within its C-terminal portion. This portion coincides with the polypeptide that could be translated from the internal start site of the *tcmN* gene discussed above. In contrast, the deduced N terminus of the TcmN protein is highly similar to the N terminus of the predicted ActVII protein from the actinorhodin gene cluster of *S.* coelicolor (Fig. 5D [13]), the N terminus of the deduced Gra4 protein from the granaticin gene cluster of *Streptomyces violaceoruber* (data not shown) (20, 21), and the entire proposed WhiEVI protein from the spore pigment gene cluster of *S.* coelicolor (Fig. 5E [4]). The resemblance of the TcmN N terminus to the ActVII, Gra4, and WhiEVI proteins is not entirely unexpected, since the genes for all four proteins occupy analogous positions in their gene clusters, immediately downstream from the PKS genes (10). An *actVII* mutation causes *S.* coelicolor to produce mutactin, an aberrantly cyclized derivative of actinorhodin (29), leading to the proposal that the *actVII* gene product is the cyclase that forms the B ring of actinorhodin (20).

Multifunctional TcmN protein is composed of two independent domains. The finding that the putative TcmN protein may consist of two parts, an ActVII-like portion and an O-methyl transferase portion, coupled with the fact that the *tcmN* ORF seems to contain an internal start site located near the junction between these two parts, caused us to question whether a stop codon that separates the different portions might have been missed during the DNA sequence analysis. To resolve this issue, we constructed a *malE-tcmN* gene fusion and determined the size of the hybrid protein produced in *E. coli*. All but the first three codons of the *tcmN* gene were joined to the end of the *malE* gene and, based on the DNA

BglII
I S A K L E Q D L G V S I P G E E L K T P R H T L H L V N T E T A G E V A * fM
1 AGATCTCCCGGAAGCTGGAACAGGACCTCGGGGTGTCGATACCGGGGGAGGAGCTGAAAGACCCCCCGGACACCCTCCACCTCGTCAACACCGAGACGGCCGGGGAGGTCGCCCTGACATG
BssHII.
A A R T D N S I V V N A P F E L V W D V T N D I E A W P E L F S E Y A E A E I L
121 GCAGCGCGCACGGACAACTCCATCGTGTC AACGGCCGTT CGAGCTCGTCTGGGACGTCACCAAACGACATCGAGGCATGGCCGGGAGCTGTTCTCCGAGTACGCCGGAGGCGAGATCCTC
RsrII
R Q D G D F D F R L K T R P D A N G R V W E W V S H R V P D K G S R T V R A H
241 CGGCAGGACGGCGACGGCTTCGACTTCCGGCTCAAGACCCGCCCCGACGGAACGGAAGGGTCTGGGAGTGGGTC TCCCACCGTGTGCCCCGACAAGGGCTCCC GCCACGGTCCGGGGCAC
XmaI
R V E T G P F A Y M N L H W T Y R A V A G G T E M R W V Q E F D M K P G A P F D
361 CGCGTCGAGACCGGGCCCTTCGCC TACATGAAACCTGCACTGGACCTACCGGGCCGTGGCCGGGGCACC GAGATGCCGGTGGGTGCAGGAGTTTCGACATGAAGCCCGGGGCCCCCGTTTCGAC
N A H M T A H L N T T T R A N M E R I K K I I E D R H R E G Q R T P A S V L P T
481 AACGCGCACATGACCGCTCACCTCAA CACGACGACACGGGCGAACATGGAGCGCATCAAGAAGATCATCGAGGACACGGACCCGGGAAGGACAGCGGACCCCGGCGTCCGCTCCGCCACC
.NarI
E L H A Q Q L L L L A A S G R L A R I V H V L T E L R I A D L L A D G P R H V A
601 GAAC TGCA CGCCCAGCA ACTGCT GTTGCTGGCCG CCTCCGGGCGG ATCGTCCA CGTGCTGACCGAGCTGAGGATCGCCGATCTCCTCGCCGACGGCCCCCCGTCACGTCCGG
E L A K E T D T H E L S L Y R V L R S A A S V G V F A E G P V R T F S A T P L S
721 GAAC TGCC AAAGGAGACGG ACACCACGAG CTCTCCCTCA CCGGCTCTCG CGCAGCGCC CTCCGTGGG GAGGGGCCCG TCAGGACCTT CTCCGCGACG CCCCTGT CG
D G L R T G N P D G V L P L V K Y N N M E L T R R P Y D E I M H S V R T G E P A
841 GACG GACTGCGCAC CGGCAACCGG ACGGTGTGCT GCCCCCTGGT GAA GTA CAACAAACATGGAGCTGACCAGGCCCCCTACGACGAGATCATGCACAGTGTCGCCACCGGGAACCGGCC
F R R V F G S S F F E H L E A N P E A G E F F E R F M A H W S R R L V L D G L A
961 TTCC GGCGGGT GTTCGGCAGC TCGTTCTTCG AGCACCTGG AGGCCAACCC GGAAGCCGGG AGATTCTTCG AGCGGTT CATGGCCCACTGGAGCCGGGCTCGTCC TCGACGGACTCGCC
D Q G M E R F S R I A D L G G G D G W F L A Q I L R R H P H A T G L L M D L P R
1081 GACC AGGGCATGGAGCGCTTCTCGCGG CATCGGGACCTCGGGG GTGGCGACGGCTGGTT CCTCGCCCAGATCCTCCGGCGCA CCCGCA CGGGCCTGCTCATGGACCTCCCGGG
V A A S A G P V L E E A K V A D R V T V L P G D F F T D P V P T G Y D A Y L F K
1201 GTGG CCGCGT CGGGCCCGCC GTCTGGAGG AGGCCAAAGT CGCCGACCGG GTCACCGTG CTGCCGGGAC TTTCTTCAC GGACCCCCGTCCCCACCGGCTACGACG CCTACCTCTTCAAG
G V L H N W S D E R A V T V L R R V R E A I G D D D A R L L I F D Q V M A P E N
1321 GGCG TGCTGC ACAACTGGTCC GACGGGCCGT GACCGTGCTG CGCCGGGTCC GGGAGGCCC ATCGGGGACG ACGACGCCC TGCTGATCTTCGACCGTCA TG GCCCCCCGAGAAC
.XhoI
E W D H A K L L D I D M L V L F G G R E R V L A E W R Q L L L E A D F D I V N T
1441 GAGT GGGACCACGCCAAGCTGCTGGACATCGACTG GTTGCTCTTCGGGAGGCCCCGGAACGGGTGCTGGCCGAGTGGCGGACGTGCTCGAGGCGGACTT CGACATCGTCAACACGG
P S H T W T T L E C R P V * MroI <---------------->
1561 CCCT CCCACACCTGG ACCACGCTGG AGTGC GCCCCGTCTAGGGCCTGTCCGGATCGTCCCCGGGCCGGGGTCCGGCACCCGGCCCGGAACCGCCAGCCCCCACGTTCA
.PvuII
fM T P H T H V R G P G D I L Q L T M A F Y G S R A L I S A V E L D L
1681 AGGAGAGCCGAGCACCCATGACACC GCA CACGCA CGTCCGGGACCCGGGCGACATC CCTCAAGCTGACCATGGCCCTTCTACGGGATCGCGGCGCCCTCATCAGCGGGGTCGA ACTGGGACCTG
F T L L A G K P L P L G E L C E R A G I H P R G A R D F L D A L V A L G L L E R
1801 TTCACCCTGCTCGCCGGGAAAACGCTG CCGGAGCTGTGCGAGCGGGCGGGCATCCATCCTCGCGGGCGGGGACTT CCTCGAGCCCCCTGGTGGA CTTGGGACTCCTGGAGCGC
E G E D T Y R N S P A A D R H L D R R K P G Y V G G Y A R L A D T K L F P V W A
1921 GAGGGCGAGGACACCTACCGCAA CAGCCCCGGCCGACC GGCACTGGGACCCGGGCGAA GCCCCCCGGGTACGT CGGGGGCTACGCCCCGGCTCGGGGACACCAAGCTGTTCCCCGTCTGGGCA
R L T E A L R T G E K Q V P S Q G G F F G G Y A D P E A A R G F L G A M D A V N
2041 CGGCTGACGGAGGCCCTGCGCACGGGCGGAGAAGCAGGTGCCGAGCCAGGGGGCTTCTCGGGGGCTACGGCGGGGACCCGGAGGGCCCCGGGCTT CCTCGGGGCCATGGACGGCTCAAC
SalI
G G V G H S L A G A L D W T E Y S S F V D L G G A R G N L A A H L H R A H P H L
2161 GGCGGCGTCGGGACAGC CCTGGCCCGCGCTCGGACGGGAGTACTCCTCCTCGT CGGACCTCGGGCGCGCGGCAACCTCGCCGCGCGACCTGCACCGGGGCTCACCCGCA CCTG

FIG. 2. Nucleotide sequence of the tcmII-tcmIV region. The start sites and potential ribosome-binding sites of the tcmN gene (including the internal ATG codons at nt 457, 490, and 526) and the tcmO gene are underlined. The major inverted repeats are overscored with arrows. Amino acid translations are provided for the C terminus of the tcmM gene (tcmIa ORF3), the tcmN gene, and the tcmO gene products, respectively. The restriction sites used in this study are indicated above their positions in the nucleotide sequence. The BamHI, BglII, KpnI, MroI, SplI, and XhoI sites are unique; for the remainder, only the sites of interest are shown.

VOL. 174, 1992 TETRACENOMYCIN C BIOSYNTHESIS BY S. GLAUCESCENS 1815


R A T C F D L P E M E P F F Q E H M K S L E T T D Q V R F A G G D F F T D P L P
2281 CGGGCCACCTGTTTCGACCTGCCGGAGATGGAACCCTTCTTCCAGGAGCACATGAAGTCCCTGGAGACGGACCAGGTGCGCCTCGCCGGGGGGGACTTCTTCAACGGACCCCCCTGCCG



R A D V F I V G H I L H Y F G L R Q R E A L I A R I H Q A L T P G G A V L V Y D
2401 CGCGCGGACGTGTTCATCGTCGGGCACATCCTCCACTTTCGGTCTCCGGCAGCGCGAGGGCGCTGATCGCCCGCATCCACAGGCGCTCACCCCCGGGTGCGTCCCTCGTCTACGAC



R M I D D D R R S A A L S L L G S L N M L L T S D E G R E Y T P A E C V R W L S
2521 CGGATGATCGACGACGACCCGGCGCAGTGCCGCACTGAGCCTGCTCGGCAGCCTCAACATGCTGCTGACCTCCGACGAGGGACGCGAGTACCCCCCCGGGAGTGCGTCCGGTGCGTCC



D A G F T D V R T T A V S G P D T L A I G R K P R *
2641 GACGCCGGATTCAACCGACGTCCGCACCACCGGGTGAGCGGGCCCACACCTCGCGATCGGCGGCAAACCCCCGGTGAGGTGGCTCGGCAGGACCGTCTGGGCTGGGGCTGATCGGCCCTCG



SplI



GCGCGTTCTGGACCGGGCACCTGCCCCCCGGCCCGTACGGGCAGGACCGGGCCGGTCCGCCCCGGGACACCCGGAACGGCTCGTCCCCACGGCGCTCACCGCCCAGGAGCTGA
2761



TCCTGTCGGCGAGCTGGGCTGGACCCGCTGCCGGGAACCCCCGGCCCCGGTGGCGGGGGGACCCCCGGACGGGGGACGTCCGGGTCCGGGAGTGCTCGGGGACGTACCGGC
2881



BamHI



CCGCCCGGCTCCCCGGCGACGGCGCGGCTCGGGTGGCGGAGCCGGCGCCGGGACCTCTCACGGGATCCCGTGCCCTGGCCGCGCCCCCG
3001



GTCGGGGAGGCGAGGACCACAGCGCGGCGGCGGCGGTCAGGTGAAAGGTGACCGGGCCGCGGGTGTGCCGACCGTGAAAGGAAACCCGGCCGGCCCCCTCGGGGAGGGGCCAGCC
3121



CGGTGTCCAGGGCGGCCAGCGGGGGGCGATCTCCGCCCCGGGTCCGGGAGCCGTCGCCGACAGCCGCGAGCAAAGGGGGTGACGGGCAGCCCCGGCGGGTGGGGGTGGCGGGAGGGCGCCCAGT
3241



CGATGAGGAAAGGCCACCGCCCCGGACGGGTTGGGGAACCGTCCGGTCA GCCGCCCCTCCAGGCGCCCGTGGGGCGGCTCATCGCCCCGGGGGACC GGGATCGTGGCCGC
3361



KpnI



GGCGCGGGCCGAGGCGATCGCCGGGTCCAGGTGCGTGGGATCGCCCAGGTCCACGGTGGCGGGCCGGGACAGGTCCACGCCCGGCGGGCCGGGGGGGCTGCT
3481



CCGGGTCCGGGCGGATGATCTCCAGGTACC 3630


FIG. 2—Continued.

sequence, we expected an MBP-TcmN hybrid protein of \(M_{r}\) = 97,300 to be synthesized from the gene fusion. If a stop codon lies between the two domains, then a hybrid of about \(M_{r} = 48,000\) should be observed. As Fig. 6A shows, the hybrid protein, immunoprecipitated with MBP antiserum, has an apparent molecular mass of approximately 97 kDa, indicating that the *tcmN* ORF is a single reading frame. Using a similar approach, we confirmed that the *tcmO* ORF ends at the TGA codon at position 2716. An MBP-TcmO hybrid protein of about \(M_{r} = 60,000\) was isolated from *E. coli*, representing the anticipated product from the fusion of the last 168 codons of the *tcmO* gene to the *malE* gene (Fig. 6B).

To determine the function of the N-terminal part of the TcmN protein and to examine the possibility that the TcmN protein is composed of two independent domains, we prepared 3′-truncated versions of the *tcmN* ORF that encode N-terminal pieces of the predicted protein and examined the behavior of these pieces in a biosynthetic assay. There are no Tcm C⁻ mutations that have been mapped to the region that encodes the N-terminal domain, so analysis of mutants and complementation could not be used to assess the function of the truncated ORF. Instead, a set of partial biosynthetic units based on pELE37 was constructed, and the ability of these biosynthetic units to cause production of Tcm C intermediates in an *S. glaucescens* Tcm C⁻ null mutant (GLA.5-1, *tcmIc*) or *S. lividans* TK24 was determined (Table 3). Plasmid pELE37 carries the *tcmKLM PKS* genes whose expression is driven by the *ermE* promoter (6). When introduced into the null mutant or *S. lividans*, pELE37 causes the production of a small amount of an early intermediate of Tcm C biosynthesis, most likely the tricyclic compound Tcm F2 (Fig. 1B). (This substance and Tcm F2, which is accumulated by *S. glaucescens* type VI mutants

[16], have an identical behavior when analyzed by thin-layer and high-performance liquid chromatography. The structure of Tcm F2 was deduced from its mass spectrum and nuclear magnetic resonance data [18]. Fast atom bombardment mass spectral analysis assigns a molecular formula of \(\mathrm{C}_{20} \mathrm{H}_{16} \mathrm{O}_{8}\), in accord with the proposed structure. As expected from the mechanism of polyketide biosynthesis, the \({ }^{13} \mathrm{C}\) nuclear magnetic resonance spectrum of the compound prepared in vivo by feeding with \({ }^{[1-{ }^{13} \mathrm{C}]}\) acetate reveals \({ }^{13} \mathrm{C}\) enrichment at C-2 [193.5 ppm], at C-20 [171.0 ppm], and at eight other sp² carbons. Furthermore, the proposed tricyclic anthracene skeleton with its two side chains is fully accommodated by the observed \({ }^{1} \mathrm{H}\) and \({ }^{13} \mathrm{C}\) spectra: 16 of the 20 carbons in the \({ }^{13} \mathrm{C}\) spectra are aromatic or olefinic, and five singlets representing the four aromatic and one olefinic proton are observed in the \({ }^{1} \mathrm{H}\) spectra. An \({ }^{1} \mathrm{H}\) singlet [three protons, \(-\mathrm{CH}_{3}\) ] with a chemical shift of \(\delta 1.9 \mathrm{ppm}\) and \({ }^{13} \mathrm{C}\) resonances at 29.9 and 193.5 ppm lead to the assignment of \(\mathrm{CH}_{3} \mathrm{CO}\) as one side chain, while the terminal \(\mathrm{CH}_{2} \mathrm{COOH}\) moiety of the second side chain is evident as an \({ }^{1} \mathrm{H}\) singlet [two protons, \(\delta 4.2 \mathrm{ppm},-\mathrm{CH}_{2}-]\) and \({ }^{13} \mathrm{C}\) resonances at 39.0 and 171.0 ppm. Although the exact sites of side chain attachment cannot be deduced from these studies, consideration of the biogenesis of polyketide metabolites suggests the configuration proposed for Tcm F2. Further chemical and spectral studies are under way to confirm this structure.) Surprisingly, when as few as the first 177 codons of the *tcmN* ORF are added to the *tcmKLM* genes of pELE37, the level of Tcm F2 produced in either the null mutant or *S. lividans* is increased more than 10-fold (Table 3, construct pWHM721). Apparently, the N-terminal domain of the TcmN protein can function in the absence of the rest of the protein, and it seems to catalyze some step(s) in the biosynthesis of Tcm F2. (The fact that a

1816 SUMMERS ET AL.

![Image Description](#)

$tcmN$

$tcmO$

Base Number

FIG. 3. CODON PREFERENCE analysis of the sequence in Fig. 2. The mole fraction of G and C nucleotides at the third position of each codon in the three possible reading frames (within a window of 25 consecutive codons) is plotted versus location in the sequence. A third-position G+C content of greater than 90% is characteristic of protein-coding sequences in Streptomyces species. Rectangles below each plot indicate ORFs that start with either ATG or GTG codons. Tick marks on the abscissa correspond to rare codons (less than 10% utilization, on average, in Streptomyces species). The analysis reveals two complete ORFs with features typical of Streptomyces genes in the top reading frame and the start of a third ORF around nt 2900 in the bottom frame (but see text). The end of the $tcmM$ gene ($tcmIa$ ORF3) is also visible at the beginning of the bottom frame.

small amount of Tcm F2 is produced by the $tcmKLM$ PKS gene products in the absence of the TcmN N-terminal domain is not inconsistent with a role for the N-terminal domain in the biosynthesis of this compound. Linear polyketide chains have been shown to cyclize spontaneously in vitro [7], and we presume that the small amount of the cyclic molecule that is produced in the absence of the TcmN N-terminal domain results from the nonenzymatic cyclization of the product of the PKS complex.) A smaller portion of the $tcmN$ ORF (the first 117 codons, corresponding to that part of the protein that precedes the putative internal start site) was also added to the $tcmKLM$ genes of pELE37, yet this construction caused no enhancement of production of the early intermediate (Table 3, construct pWHM720). The domain boundary of the TcmN protein therefore appears to lie between residues 117 and 177. Consistent with this observation, the putative WhiEVI protein, which is the smallest homolog of the TcmN N-terminal domain, comprises 159 amino acid residues.

Our findings demonstrate that the N-terminal domain of the TcmN protein plays a part in the biosynthesis of Tcm C, catalyzing a step in the formation of an early, partially cyclized intermediate. Consequently, the $tcmN$ message is most likely translated in vivo from the first of the possible start sites, and the $tcmN$ gene therefore appears to encode a multifunctional enzyme. Furthermore, the observation that an N-terminal fragment of the TcmN protein is active in a biosynthetic assay (probably as a cyclase), while a C-terminal fragment of the enzyme (Table 2, fragment c) can complement the methylation-deficient type II and type V mutations, indicates that the two domains of the TcmN protein are structurally and functionally independent. Thus, it is possible that some translation of the $tcmN$ ORF initiates in vivo at the internal start site to yield just the 3-O-methyl transferase.

Interestingly, addition of the remainder of the $tcmN$ ORF to the $tcm$ genes of pWHM721 (to give plasmid pWHM722) does not change the pattern of metabolites produced in either the Tcm C⁻ null mutant or S. lividans as determined by thin-layer chromatography of culture extracts (Table 3). Since pWHM722 complements the type II mutation, indicating that the $tcmN$ gene encoded by the plasmid is expressed, this result illustrates that the partially cyclized intermediate formed by the TcmN N-terminal domain is not a substrate for the C-terminal 3-O-methyl transferase. Thus, the gene fusion event that created the $tcmN$ gene does not appear to reflect any relation between the biosynthetic steps catalyzed by the two different domains. Extending the biosynthetic

TABLE 1. Complementation behavior of the DNA fragments that carry the ORFs from the *tcmII-tcmIV* region

| Fragment          | Tcm C⁻ mutant type |
|-------------------|--------------------|
|                   | Ia   | II   | III  | IV   | V    | VI   | VII  |
| 1.6-kb BglII-MroI | –    | +    | –    | –    | +    | –    | –    |
| 1.2-kb MroI-SplI  | –    | –    | –    | +    | –    | –    | –    |

unit to include the *tcmO* gene also has no effect on the pattern of metabolites produced (Table 3, plasmid pWHM723), showing that the early intermediate cannot be transformed by the 8-O-methyl transferase either.

DISCUSSION

We determined the nucleotide sequence of the 3.6-kb DNA fragment that lies immediately downstream from the Tcm C PKS genes of *S. glaucescens*. This region is known to influence the methylation of the C-3 and C-8 hydroxyl groups of Tcm C (15, 16, 28), and the DNA sequence indicates that two polypeptides with similarity to the hydroxyindole O-methyl transferase of the bovine pineal gland (11) are encoded here. Interestingly, however, one of the polypeptides is a multifunctional enzyme that catalyzes not only the C-3 O-methylation, but also some of the steps involved in the biosynthesis of a partially cyclized, early intermediate of Tcm C. Our DNA sequence extends about 1 kb beyond the genes for these two polypeptides, yet no mutations that affect Tcm C biosynthesis have been mapped downstream of the second gene reported here. It is therefore possible that this nucleotide sequence defines one end of the Tcm C biosynthetic gene cluster.

The DNA sequence from about one-half of the Tcm C gene cluster has now been analyzed, and our current picture of the organization of the genes is given in Fig. 4A. The *tcmIa* PKS genes, which are more or less centrally located in the gene cluster, comprise three ORFs (2). The first ORF (*tcmK*) is thought to encode the β-ketoacyl synthase that

TABLE 2. Complementation behavior by bioassay of the *tcmN* and *tcmO* ORFs in plasmid pIJ702 after removal of some of the putative N- and C-terminal codons (see Fig. 4)

| Fragment          | Description<sup>a</sup>       | Complementation with Tcm C⁻ type<sup>b</sup>: |
|-------------------|--------------------------------|-----------------------------------------------|
|                   |                                | II     | V      | IV     |
| None              | pIJ702                         | –      | –      |        |
| (a) 1.6-kb BglII-MroI | *tcmN*                       | ++     | ++     |        |
| (b) 1.5-kb BssHIII- | *tcmN* (–N2)                 | +      | +      |        |
| MroI              |                                |        |        |        |
| (c) 1.3-kb RsrII-MroI | *tcmN* (–N78)               | +      | +      |        |
| (d) 1.2-kb XmaI-  | *tcmN* (–N117)                | ±      | ±      |        |
| MroI              |                                |        |        |        |
| (e) 1.5-kb BglII-Xhol | *tcmN* (–C21)               | ±      | ±      |        |
| None              | pIJ702                         | –      |        |        |
| (f) 1.2-kb MroI-SplI | *tcmO*                       | ++     |        |        |
| (g) 1.1-kb PvuII-SplI | *tcmO* (–N15)               | –      |        |        |
| (h) 1.1-kb MroI-AatII | *tcmO* (–C18)               | –      |        |        |

<sup>a</sup> The values in parentheses preceded by either –N or –C refer to the number of N- or C-terminal codons, respectively, that were removed from the indicated gene.

<sup>b</sup> –, no zone of inhibition; ±, ≤2.0-cm zone; +, 2.1- to 2.4-cm zone; ++, ≥2.5-cm zone (wild type).

TABLE 3. Principal Tcm C intermediates produced in *S.* glaucescens GLA.5-1 and *S.* lividans TK24 caused by pWHM720, pWHM721, pWHM722, and pWHM723<sup>a</sup>

| Plasmid    | Description<sup>b</sup>         | Tcm C intermediate |
|------------|----------------------------------|--------------------|
|            |                                  | GLA.5-1           | TK24           |
| pIJ486     | Control                          | None              | None           |
| pELE37     | *tcmKLM*                         | (Tcm F2)<sup>c</sup> | (Tcm F2)<sup>c</sup> |
| pWHM720    | *tcmKLMN* (1–117)               | (Tcm F2)<sup>c</sup> | (Tcm F2)<sup>c</sup> |
| pWHM721    | *tcmKLMN* (1–177)               | Tcm F2             | Tcm F2         |
| pWHM722    | *tcmKLMN*                       | Tcm F2             | Tcm F2         |
| pWHM723    | *tcmKLMNO*                      | Tcm F2             | Tcm F2         |

<sup>a</sup> Compounds were identified and relative amounts were determined by thin-layer chromatography.

<sup>b</sup> The values in parentheses refer to the N-terminal codons of the *tcmN* gene that are included in the construction.

<sup>c</sup> The yield of Tcm F2 from these strains is less than 10% that of the other producing strains.

condenses acetate units (derived from malonyl-acyl carrier protein [ACP]) to build the linear decaketide that will form the backbone of the Tcm C molecule. The second ORF (*tcmL*) encodes a polypeptide of unknown function. This

A

![Diagram A](#)

B

![Diagram B](#)

FIG. 4. (A) Physical map of the Tcm C gene cluster of *S.* glaucescens. Arrows indicate the direction of transcription of the known ORFs. (B) Schematic illustration of the restriction fragments used for the complementation analysis (see Tables 1 and 2). Restriction sites are abbreviated as follows: Aa, AatII; Bs, BssHIII; Bg, BglII; Kp, KpnI; Mr, MroI; Pv, PvuII; Rs, RsrII; Sp, SplI; Xh, XhoI; and Xm, XmaI. The BglII, KpnI, MroI, SplI, and XhoI sites are unique; for the others, only the sites of interest have been shown.

FIG. 5. COMPARE-DOTPLOT analyses of the deduced protein products of (A) $tcmN$ versus the bovine hydroxyindole O-methyl transferase gene (BHIMT [11]), 23% identity (residues 160 to 494 of TcmN versus BHIMT); (B) $tcmO$ versus BHIMT, 29% identity; (C) $tcmN$ versus $tcmO$, 25% identity (residues 160 to 494 of TcmN versus TcmO); (D) $tcmN$ versus $actVII$ (13), 23% identity (residues 1 to 159 of TcmN versus residues 1 to 159 of ActVII); and (E) $tcmN$ versus $whiE$ OrfVI (WhiEVI [4]), 57% identity (residues 1 to 159 of TcmN versus WhiEVI). A window size of 30 was used at a stringency of 15.

protein is similar to that encoded by the first ORF (29% identity), but it lacks homology to the active sites of the $tcmK$ gene product. The third ORF ($tcmM$) of the PKS complex encodes the ACP that provides the malonyl groups to the β-ketoacyl synthase and perhaps also tethers the growing polyketide chain. Immediately downstream from the PKS genes (and transcribed in the same direction) lie the two genes described here. The $tcmN$ gene begins just 1 nt after the ACP gene and may be part of the same transcript. This gene encodes a multifunctional enzyme that is composed of two structurally and functionally independent domains. The N-terminal domain, when coupled with the three components of the PKS, enhances the synthesis of an early, partially cyclized intermediate in the Tcm C pathway. The C-terminal domain, which resembles the bovine hydroxyindole O-methyl transferase, catalyzes the 3-O-methylation of one or more of the later intermediates in the biosynthetic pathway ($tcmN$ mutants accumulate several 3-O-demethyl Tcm C intermediates [28]). Interestingly, the partially cyclized intermediate produced by the TcmN N-terminal domain is not a substrate for the C-terminal 3-O-methyl transferase. Thus, the structure of the TcmN protein does not reflect any relation between the biosynthetic steps catalyzed by its two different domains. The $tcmO$ gene, which also encodes an analog of the bovine hydroxyindole O-methyl transferase, falls 97 nt downstream from $tcmN$. A deletion mutation within the $tcmO$ locus causes the production of 8-demethyl Tcm C intermediates (28), indicating that the $tcmO$ gene encodes the 8-O-methyl transferase. A large stem-loop can form in the gap between the $tcmN$ and $tcmO$ genes, and three additional inverted repeats of 8 bases or more are found in the area downstream of the $tcmO$ gene. These structures may function in transcription termination and/or mRNA stability (17).

Surprisingly, just the three genes of the PKS ($tcmKLM$) and the first 177 codons of the $tcmN$ gene are required in S. lividans or an S. glaucescens Tcm C⁻ null mutant to cause high-level synthesis of a compound that is most likely Tcm F2 (Fig. 1B). Yet, 10 distinct steps are envisaged in the pathway from acetate and malonate (as their coenzyme A

ate observed here. Yet, additional ring closures and the chain termination must still occur to yield this compound. The TcmL polypeptide might provide these missing activities. It is also possible that some reactions, such as certain cyclizations and dehydrations, proceed by uncatalyzed pathways. Indeed, linear polyketide chains are known to cyclize spontaneously in vitro (7). We are currently examining these possibilities.

In summary, we determined the nucleotide sequence of the two genes that lie immediately downstream from the Tcm C PKS genes of *S. glaucescens* and provided evidence that the first gene (*tcmN*) encodes a multifunctional cyclase-dehydratase-3-O-methyl transferase and that the second gene (*tcmO*) encodes the 8-O-methyl transferase in the Tcm C pathway. We have also shown that just the three genes of the *tcmIa* region (the *tcmKLM* PKS genes) and the first third of the *tcmN* gene are required to cause high-level production of a partially cyclized precursor of Tcm C in either an *S. glaucescens* Tcm C⁻ null mutant or *S. lividans*. Thus, the activities needed to produce this early, cyclic intermediate of Tcm C have been localized to a small segment of the Tcm C biosynthetic gene cluster.

### ACKNOWLEDGMENTS

We thank Francisco Malpartida (Centro Nacional de Biotecnología, Madrid, Spain) for the nucleotide sequence of the *actVII* gene and Ben Shen, Heinrich Decker, and Pat Guilfoile for useful discussions.

This work was supported by a grant from the National Institutes of Health (CA 35381).

### REFERENCES

1. Axelrod, J., and H. Weissbach. 1960. Enzymatic O-methylation of N-acetylserotonin to melatonin. Science 131:1312.
2. Bibb, M. J., S. Biró, H. Motamedi, J. F. Collins, and C. R. Hutchinson. 1989. Analysis of the nucleotide sequence of the *Streptomyces glaucescens tcmI* genes provides key information about the enzymology of polyketide antibiotic biosynthesis. EMBO J. 8:2727–2736.
3. Bibb, M. J., and S. N. Cohen. 1982. Gene expression in *Streptomyces*: construction and application of promoter-probe plasmid vectors in *Streptomyces lividans*. Mol. Gen. Genet. 187:265–277.
4. Davis, N. K., and K. F. Chater. 1990. Spore colour in *Streptomyces coelicolor* A3(2) involves the developmentally regulated synthesis of a compound biosynthetically related to polyketide antibiotics. Mol. Microbiol. 4:1679–1691.
5. Devereux, J., P. Haeberli, and O. Smithies. 1984. A comprehensive set of sequence analysis programs for the VAX. Nucleic Acids Res. 12:387–395.
6. Gramajo, H. C., J. White, C. R. Hutchinson, and M. J. Bibb. 1991. Overproduction and localization of components of the polyketide synthase of *Streptomyces glaucescens* involved in the production of the antibiotic tetracenomycin C. J. Bacteriol. 173:6475–6483.
7. Harris, T. M., and C. M. Harris. 1986. Biomimetic syntheses of aromatic polyketide metabolites. Pure Appl. Chem. 58:283–294.
8. Hopwood, D. A., M. J. Bibb, K. F. Chater, T. Kieser, C. J. Bruton, H. M. Kieser, D. J. Lydiate, C. P. Smith, J. M. Ward, and H. Schrempf. 1985. Genetic manipulations of *Streptomyces*: a laboratory manual. The John Innes Foundation, Norwich, United Kingdom.
9. Hopwood, D. A., T. Kieser, H. M. Wright, and M. J. Bibb. 1983. Plasmids, recombination and chromosome mapping in *Streptomyces lividans* 66. J. Gen. Microbiol. 129:2257–2269.
10. Hopwood, D. A., and D. H. Sherman. 1990. Molecular genetics of polyketides and its comparison to fatty acid biosynthesis. Annu. Rev. Genet. 24:37–66.
11. Ishida, I., M. Obinata, and T. Deguchi. 1987. Molecular cloning and nucleotide sequence of cDNA encoding hydroxyindole

O-methyltransferase of bovine pineal glands. J. Biol. Chem. 262:2895–2899.

12. Katz, E., C. J. Thompson, and D. A. Hopwood. 1983. Cloning and expression of the tyrosinase gene from *Streptomyces antibioticus* in *Streptomyces lividans*. J. Gen. Microbiol. 129:2703–2714.

13. Malpartida, F. (Centro Nacional de Biotecnología, Madrid, Spain). 1991. Personal communication.

14. Motamedi, H., and C. R. Hutchinson. 1987. Cloning and heterologous expression of a gene cluster for the biosynthesis of tetracenomycin C, the anthracycline antitumor antibiotic of *Streptomyces glaucescens*. Proc. Natl. Acad. Sci. USA 84:4445–4449.

15. Motamedi, H., and C. R. Hutchinson. 1987. Anthracycline antitumor antibiotics: organization of the genes for tetracenomycin C production in *Streptomyces glaucescens*, p. 355–362. In M. Alacevic, D. Hranueli, and Z. Toman (ed.), Genetics of industrial microorganisms: Proceedings of the Fifth International Symposium. Pliva, Zagreb, Yugoslavia.

16. Motamedi, H., E. Wendt-Pienkowski, and C. R. Hutchinson. 1986. Isolation of tetracenomycin C-nonproducing *Streptomyces glaucescens* mutants. J. Bacteriol. 167:575–580.

17. Murakawa, G. J., C. Kwan, J. Yamashita, and D. P. Nierlich. 1991. Transcription and decay of the *lac* messenger: role of an intergenic terminator. J. Bacteriol. 173:28–36.

18. Nakayama, H., B. Shen, and C. R. Hutchinson. Unpublished data.

19. Sambrook, J., E. F. Fritsch, and T. Maniatis. 1989. Molecular cloning: a laboratory manual, 2nd ed. Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y.

20. Sherman, D. H., M. J. Bibb, T. J. Simpson, D. Johnson, F. Malpartida, M. Fernandez-Moreno, E. Martinez, C. R. Hutchinson, and D. A. Hopwood. 1991. Molecular genetic analysis reveals a putative bifunctional polyketide cyclase/dehydrase gene from *Streptomyces coelicolor* and *Streptomyces violaceoruber*, and a cyclase/O-methyltransferase from *Streptomyces*

21. Sherman, D. H., F. Malpartida, M. J. Bibb, H. M. Kieser, M. J. Bibb, and D. A. Hopwood. 1989. Structure and deduced function of the granaticin-producing polyketide synthase gene cluster of *Streptomyces violaceoruber* Tü22. EMBO J. 8:2717–2725.

22. Summers, R. G., and J. R. Knowles. 1989. Illicit secretion of a cytoplasmic protein into the periplasm of *Escherichia coli* requires a signal peptide plus a portion of the cognate secreted protein. J. Biol. Chem. 264:20074–20081.

23. Tinoco, I., P. N. Borer, B. Dengler, M. D. Levine, O. C. Uhlenbeck, D. M. Crothers, and J. Gralla. 1973. Improved estimation of secondary structure in ribonucleic acids. Nature (London) New Biol. 246:40–41.

24. Ward, J. M., G. R. Janssen, T. Kieser, M. J. Bibb, M. J. Buttner, and M. J. Bibb. 1986. Construction and characterization of a series of multi-copy promoter-probe plasmid vectors for *Streptomyces* using the aminoglycoside phosphotransferase gene from Tn5 as indicator. Mol. Gen. Genet. 203:468–478.

25. Weber, W., H. Zähner, J. Siebers, K. Schröder, and A. Zeeck. 1979. Stoffwechselprodukte von mikroorganismen. 175. Mitteilung. Tetracenomycin C. Arch. Microbiol. 121:111–116.

25a. Wendt-Pienkowski, E., and R. G. Summers. Unpublished data.

26. Wessel, W. L., B. Shen, R. G. Summers, and C. R. Hutchinson. Unpublished data.

27. Yanisch-Perron, C., J. Vieira, and J. Messing. 1985. Improved M13 phage cloning vectors and host strains: nucleotide sequences of the M13mp18 and pUC19 vectors. Gene 33:103–119.

28. Yue, S., H. Motamedi, E. Wendt-Pienkowski, and C. R. Hutchinson. 1986. Anthracycline metabolites of tetracenomycin C-nonproducing *Streptomyces glaucescens* mutants. J. Bacteriol. 167:581–586.

29. Zhang, H.-L., X.-G. He, A. Adefarati, J. Gallucci, S. P. Cole, J. M. Beale, P. J. Keller, C.-J. Chang, and H. G. Floss. 1990. Mutactin, a novel polyketide from *Streptomyces coelicolor*: structure and biosynthetic relationship to actinorhodin. J. Org. Chem. 55:1682–1684.
