
JOURNAL OF BACTERIOLOGY, Nov. 2004, p. 7161–7174  
0021-9193/04/$08.00+0 DOI: 10.1128/JB.186.21.7161–7174.2004  
Copyright © 2004, American Society for Microbiology. All Rights Reserved.

The Completely Sequenced Plasmid pEST4011 Contains a Novel IncP1  
Backbone and a Catabolic Transposon Harboring *tfd* Genes  
for 2,4-Dichlorophenoxyacetic Acid Degradation  

Eve Vedler,* Merle Vahter, and Ain Heinaru  

Department of Genetics, Institute of Molecular and Cell Biology, Tartu University, Tartu, Estonia  

Received 7 July 2004/Accepted 27 July 2004  

The herbicide 2,4-dichlorophenoxyacetic acid (2,4-D)-degrading bacterium *Achromobacter xylosoxidans* subsp. *denitrificans* strain EST4002 contains plasmid pEST4011. This plasmid ensures its host a stable 2,4-D⁺ phenotype. We determined the complete 76,958-bp nucleotide sequence of pEST4011. This plasmid is a deletion and duplication derivative of pD2M4, the 95-kb highly unstable laboratory ancestor of pEST4011, and was self-generated during different laboratory manipulations performed to increase the stability of the 2,4-D⁺ phenotype of the original strain, strain D2M4(pD2M4). The 47,935-bp catabolic region of pEST4011 forms a transposon-like structure with identical copies of the hybrid insertion element IS1071::IS1471 at the two ends. The catabolic regions of pEST4011 and pJP4, the best-studied 2,4-D-degradative plasmid, both contain homologous, *tfd*-like genes for complete 2,4-D degradation, but they have little sequence similarity other than that. The backbone genes of pEST4011 are most similar to the corresponding genes of broad-host-range self-transmissible IncP1 plasmids. The backbones of the other three IncP1 catabolic plasmids that have been sequenced (the 2,4-D-degradative plasmid pJP4, the haloacetate-catabolic plasmid pUO1, and the atrazine-catabolic plasmid pADP-1) are nearly identical to the backbone of R751, the archetype plasmid of the IncP1 β subgroup. We show that despite the overall similarity in plasmid organization, the pEST4011 backbone is sufficiently different (51 to 86% amino acid sequence identity between individual backbone genes) from the backbones of members of the three IncP1 subgroups (the α, β, and γ subgroups) that it belongs to a new IncP1 subgroup, the δ subgroup. This conclusion was also supported by a phylogenetic analysis of the *trfA2*, *korA*, and *traG* gene products of different IncP1 plasmids.

Microbial degradation of 2,4-dichlorophenoxyacetic acid (2,4-D), a xenobiotic herbicide used worldwide for almost 60 years, is a well-studied process. Various soil bacteria can use 2,4-D as a carbon and energy source. Therefore, this compound has become a model for studying the evolution and distribution of genes for the degradation of chloroaromatic compounds. A number of bacterial strains belonging to different phylogenetic groups able to mineralize this compound have been found to possess genetically and enzymatically different 2,4-D-catabolic pathways (9, 10, 16–18). The best-studied 2,4-D degradation genes (located in a chromosome or a plasmid) are *tfd*-like (pJP4-like). The very recently sequenced 87,688-bp plasmid pJP4 (48) from *Wautersia eutropha* JMP134 (formerly *Ralstonia eutropha*) was originally isolated in Australia (8), and its *tfd* genes and the corresponding enzymes responsible for converting 2,4-D to 3-oxoadipate are well characterized (22, 23, 25, 26, 35, 58). Besides pJP4, there are only two cases in which the DNA regions containing *tfd* genes for the whole 2,4-D degradation pathway have been sequenced, a chromosomal transposon-like structure (about 30 kb) from *Delftia acidovorans* P4a (15) and Tn5530 (41 kb) located in plasmid pIJB1 from *Burkholderia cepacia* 2a (36, 56). The *tfd* genes of these two elements are homologous to the pJP4 *tfd* genes but are organized in a different way.

Bacterial catabolic genes are often encoded by mobile genetic elements, including transposons and conjugative plasmids. Genes that encode degradation of man-made compounds are often found in IncP1 plasmids which are divided into three subgroups, the α, β, and γ subgroups (1). These plasmids are the most promiscuous self-transmissible plasmids that have been characterized to date (46) and thus could be responsible for efficient dissemination of genes encoding degradation of recently introduced xenobiotic compounds. The (almost) complete nucleotide sequences of four IncP1 catabolic plasmids, pUO1 (39), pADP-1 (30), pJP4 (48), and pTSA (47) coding for haloacetate, atrazine, 2,4-D, and *p*-toluene sulfonate degradation, respectively, have been determined, and they have all been shown to be members of the IncP1 β subgroup. Although only the catabolic region of the 2,4-D-degradative plasmid pIJB1 has been sequenced, it has been shown that this plasmid belongs to the same subgroup (36).

Nearly 20 years ago, several 2,4-D-metabolizing bacterial strains were isolated from different soil samples from Estonian agricultural enterprises. They all contained 2,4-D-degradative plasmids that were the same size (2; V. Kõiv, unpublished data). One strain, D2M4, containing plasmid pD2M4 (about 95 kb), was selected for further investigation as it showed the best growth characteristics on 2,4-D as a sole source of carbon and energy. However, the 2,4-D⁺ phenotype of this strain was very unstable. Different laboratory manipulations were performed in order to obtain a more stable phenotype. As a result, strain EST4002 containing plasmid pEST4011 (approximately 70 kb, as estimated by restriction analysis) was isolated. This

* Corresponding author. Mailing address: Department of Genetics, Institute of Molecular and Cell Biology, 23 Riia Street, Tartu 51010, Estonia. Phone: 372-7375014. Fax: 372-7420286. E-mail: eve.vedler@ut.ee.

strain was determined to be a strain of *Achromobacter xylosoxidans* subsp. *denitrificans*. According to hybridization experiments, the *tfd*-like genes for 2,4-D degradation were all located in pEST4011. 2,4-D– strain EST4003 (obtained when EST4002 was grown in Luria-Bertani medium) contained plasmid pEST4012 (about 30 kb), from which the whole catabolic region had been deleted (28, 53).

In this paper we report the complete nucleotide sequence of a 2,4-D degradative plasmid, the 76,958-bp pEST4011 plasmid, and we describe and analyze all open reading frames (ORFs) and other features detected in pEST4011. The restriction patterns of pEST4011 and its ancestor, pD2M4, which is unstable in the laboratory, are compared. On the basis of multiple alignments of all available *trfA2*, *korA*, and *traG* gene products of different IncP1 plasmids, we show that the backbone of pEST4011 is quite different from that of other IncP1 plasmids. Consequently, we suggest that pEST4011 belongs to a new IncP1 subgroup, the δ subgroup.

at least 150 bp long were analyzed further with the BLASTN and BLASTP programs (http://www.ncbi.nlm.nih.gov/BLAST), and only the ORFs with reasonable homology to some other known sequence were selected. Translation start codon positions were determined manually on the basis of the presence of a potential Shine-Dalgarno sequence upstream of a start codon.

Phenotypic analysis with Biolog GN2 microplates. One day before inoculation of Biolog GN2 plates (Biolog Inc., Hayward, Calif.), strain EST4002 harboring plasmid pEST4011 and strain EST4003 harboring plasmid pEST4012 were streaked on BUGM (Biolog Inc.) agar plates. The wells of the Biolog GN2 plates were inoculated with 150 μl of a bacterial suspension adjusted to an optical density at 580 nm of 0.150 for both strains. The plates were incubated at 30°C for 72 h. Development of color was automatically recorded by using a microplate reader with a 620-nm-wavelength filter.

Multiple alignment and phylogenetic tree construction. The amino acid sequences of different IncP1 *trfA2*, *korA*, and *traG* gene products were aligned, and bootstrapped neighbor-joining trees were derived by using the program CLUSTALX, version 1.8 (43), with default parameters. The PHYLIP 3.6 software package (http://evolution.genetics.washington.edu/phylip.html) programs SEQBOOT, PROTDIST, NEIGHBOR, CONSENSE, and PROML were used to calculate protein distances and to derive the corresponding phylogenetic trees.

Nucleotide sequence accession number. The complete nucleotide sequence of pEST4011 from *A. xylosoxidans* subsp. *denitrificans* strain EST4002 has been deposited in the GenBank database under accession number AY540995.

---

**MATERIALS AND METHODS**

Isolation of pEST4011 DNA. *A. xylosoxidans* subsp. *denitrificans* EST4002 cells harboring pEST4011 were maintained on minimal salts agar plates containing 5 mM 2,4-D. Plasmid pEST4011 was isolated from EST4002 cells that were grown overnight at 30°C in minimal salts medium supplemented with 5 mM 2,4-D. The culture obtained was inoculated into minimal salts medium supplemented with 20 mM sodium citrate and 0.2% Casamino Acids and grown overnight at 30°C. Plasmid DNA was isolated by using a method for isolation of large bacterial plasmids described by Hansen and Olsen (13), followed by CsCl buoyant density ultracentrifugation (38).

Construction of pEST4011 library. For construction of a pEST4011 library, partial digestion with Bsp143I was used to produce overlapping fragments that were approximately 1 kb long. These fragments were cloned into the BamHI site of the vector plasmid pBluescript II (SK) (Stratagene) and transferred into *Escherichia coli* DH5α cells. This and all other cloning procedures in this study were performed by using standard methods (38).

DNA sequencing. Templates for DNA sequencing were prepared by PCR amplification of inserted DNA from randomly selected insert-containing clones with the vector-specific primers T3 and T7. The PCR products obtained were directly sequenced by using the same primers (subsequent primer walking was used if necessary), a DYEnamic ET terminator cycle sequencing kit (Amersham Pharmacia Biotech), and an ABI Prism 377 DNA sequencer (Applied Biosystems).

pEST4011 sequence assembly. All sequences were edited and analyzed and the complete sequence of pEST4011 was assembled by using the BioEdit sequence alignment editor, version 5.0.9 (12). Vector sequences were removed, and sequences were assembled into contigs manually according to alignment results with the most similar complete sequences obtained from the GenBank, including those of Birmingham IncP-alpha plasmids (R18, R68, RK2, RP1, RP4) (accession number L27758), *Enterobacter aerogenes* plasmid R751 (NC_001735), and *B. cepacia* 2a plasmid pIJB1 (AF029344). In order to close gaps, custom primers were designed by using the ends of each contig. These primers were used for PCR amplification of DNA fragments containing the necessary sequences; purified pEST4011 DNA was used as a template. Again, the PCR products obtained were directly sequenced by using the same primers. Finally, assembly of the pEST4011 nucleotide sequence was verified by PCR analysis by producing overlapping products covering the whole plasmid and by restriction analysis with BamHI, ClaI, EcoRI, HindIII, KpnI, MfeI, MluI, NdeI, StuI, and XmnI.

Resequencing the identical regions of pEST4011. In order to separately resequence the identical copies of IS1071::IS1471 and the 6,991-bp duplications, two strategies were used. First, pEST4011 regions were cloned into the vector plasmid pBluescript II (SK) and sequenced by using vector-specific and custom primers. Second, two overlapping PCR products were generated by using one primer annealing outside the duplicated area and another primer annealing inside the duplicated area, and they were sequenced by using the same primers and subsequent primer walking.

Analysis of pEST4011 ORFs. The presence of ORFs in pEST4011 was analyzed by using the National Center for Biotechnology Information ORF Finder program (http://www.ncbi.nlm.nih.gov/gorf.html). All predicted ORFs that were

---

**RESULTS AND DISCUSSION**

Nucleotide sequence and organization of pEST4011. The 2,4-D-degradative plasmid pEST4011 is circular and 76,958 bp long and has an overall G+C content of 62.03%. Thus, pEST4011 is 7 kb larger than we previously calculated on the basis of restriction analysis. A circular physical map of pEST4011 is shown in Fig. 1. The first nucleotide of the HindIII restriction site between the *upf54.4* and *upf54.8* genes corresponds to position 1. The recognition site positions of 10 different restriction endonucleases are also shown in Fig. 1. Interestingly, 101 of 115 of these sites are located in the catabolic transposon of pEST4011. These restriction enzymes together with the PCR analysis were used to verify the assembly of pEST4011 (data not shown). Plasmid pEST4011 consists of the 29-kb IncP1 backbone loaded with the 48-kb catabolic transposon containing, among other genes, the *tfd*-like genes for complete 2,4-D degradation, bordered by identical IS1071::IS1471 hybrid insertion elements. The nucleotide sequence of this catabolic transposon is 99% identical to the sequence of Tn5530 (41 kb) of pIJB1, except for duplication of a 6,991-bp region in the case of pEST4011. As a result, the pEST4011 region from position 27334 to position 34324 is identical to the adjacent region from position 34325 to position 41315, with only one mismatch. In addition, the pEST4011 region from position 20101 to position 31555 is 99% identical to the sequenced region of *Variovorax paradoxus* TV1 2,4-D-degradative plasmid pTV1 (49, 50). The IncP1 backbone of pEST4011 consists of the standard elements (Fig. 1), including the origin for the theta mode of replication (*oriV*) regulated by iterons; the region necessary for autonomous replication initiation, copy number control, and stable maintenance in the host cell; and two regions involved in plasmid conjugation, Tral (*tra* genes) with an origin of transfer (*oriT*) and Tra2 (*trb* genes). However, seven *trb* genes (*trbE* to *trbL*) essential for mating pair formation during conjugation are not present in pEST4011.

By using the National Center for Biotechnology Information ORF Finder program and BLAST similarity searches, 68 ORFs were identified in the pEST4011 sequence (Fig. 1; Table 1); 58 of these ORFs were named after the closest relative

VOL. 186, 2004                                                                 SEQUENCE OF CATABOLIC PLASMID pEST4011 7163

FIG. 1. Circular physical map of 76,958-bp plasmid pEST4011. The identified ORFs, the insertion elements, and the vegetative (oriV) and transfer (oriT) origins are shown inside the circle. The blue genes are necessary for plasmid autonomous replication initiation, copy number control, and stable maintenance in the host cell; the yellow genes are involved in plasmid conjugation; all genes in the pEST4011 catabolic transposon are purple. The 7-kb duplicated regions are indicated; the site from which the essential transfer genes trbE to trbL are missing is indicated by a line with a triangle at the end. R indicates the righthand copy of IS1071::IS1471, and L indicates the lefthand copy of IS1071::IS1471. The recognition site positions of 10 restriction endonucleases, which were used to verify the assembly of pEST4011, are indicated outside the circle. The colored circle indicates the G+C contents in 192-bp blocks starting from the position 1 (the first nucleotide in the corresponding HindIII recognition site). The products of the genes in quotation marks are truncated and thus probably not functional.

found in the GenBank database. Twenty-two ORFs are oriented in the same direction, whereas 46 are oriented in the opposite direction. Similarly, 34 ORFs are located both in the backbone and in the catabolic transposon of pEST4011 (9 of these ORFs are the tfd genes for 2,4-D catabolism). Six ORFs in the catabolic region (including one tfd gene) have identical copies due to duplication of a 7-kb DNA segment. Five ORFs code for putative transposases, and two of them are identical

tnp1471 genes of IS1071::IS1471. Ten ORFs encode putative transcriptional regulators. Four ORFs start with GTG, and the rest start with an ATG translation start codon. The translation stop codons are TGA (42 ORFs), TAA (15 ORFs), and TAG (12 ORFs). A ribosome-binding site with low similarity (two to four matches) to the Shine-Dalgarno consensus sequence (5′-TAAGGAGGT-3′) precedes 24 ORFs, while for 44 ORFs there are five to seven matches with the consensus sequence.

| pEST 4011 coding sequence | Function of closest relative (protein) | No. of amino acids in protein | Source | PEST 4011 Homolog (alignment region) | % Identity/ % similarity | GenBank accession no. |
|--------------------------|------------------------------------|-------------------------------|--------|-------------------------------------------------|-----------------------|----------------------|
| upf54.8                 | Unknown (Upf54.8)                  | 350-6                        | Enterobacter aerogenes plasmid R751 | 114 (1-114)           | 63/81                | NC_001735            |
| orf1                    | Cell filamentation protein (XF1657) | 975-367                      | Xylella fastidiosa 9a5c | 202 (1-202)           | 82/93                | AE003992            |
| orf2                    | Unknown (XF1656)                   | 1160-972                     | Xylella fastidiosa 9a5c | 62 (1-62)             | 81/97                | AE003992            |
| kfrA                    | Regulation of KfrA transcription (KfrA) | 2241-1225                    | Birmingham IncP-alpha plasmid (RP4, RK2, etc.) | 338 (1-338)           | 54/72                | L27758              |
| korB                    | Global transcriptional repressor (KorB) | 3509-2436                    | Enterobacter aerogenes plasmid R751 | 357 (1-357)           | 73/87                | NC_001735            |
| incC1                   | Plasmid partitioning (IncC1)       | 4597-3506                    | Birmingham IncP-alpha plasmid (RP4, RK2, etc.) | 363 (1-363)           | 68/81                | L27758              |
| IncC2                   | Plasmid partitioning (IncC2)       | 4279-3506                    | Birmingham IncP-alpha plasmid (RP4, RK2, etc.) | 257 (1-257)           | 74/87                | L27758              |
| korA                    | Global transcriptional repressor (KorA) | 4584-4276                    | Enterobacter aerogenes plasmid R751 | 102 (1-102)           | 73/90                | NC_001735            |
| kleF                    | Plasmid maintenance (KleF)         | 5268-4747                    | Enterobacter aerogenes plasmid R751 | 173 (1-173)           | 63/82                | NC_001735            |
| kleE                    | Plasmid maintenance (KleE)         | 5593-5270                    | Enterobacter aerogenes plasmid R751 | 107 (1-107)           | 61/79                | NC_001735            |
| kleA                    | Plasmid maintenance (KleA)         | 5967-5710                    | Enterobacter aerogenes plasmid R751 | 85 (1-85)             | 55/71                | NC_001735            |
| korC                    | Global transcriptional repressor (KorC) | 6415-6158                    | Enterobacter aerogenes plasmid R751 | 85 (1-85)             | 68/84                | NC_001735            |
| klcB                    | KorA repression modulation (KlcB)  | 7836-6430                    | Birmingham IncP-alpha plasmid (RP4, RK2, etc.) | 468 (1-468)           | 53/72                | L27758              |
| klcA                    | Possible antisite restriction system (KlcA) | 8318-7884                    | Enterobacter aerogenes plasmid R751 | 144 (1-144)           | 75/87                | NC_001735            |
| tup1471                 | Transposase (Tnp1471)             | 13099-12062                  | Burkholderia cepacia 2a plasmid pJJB1 | 345 (1-345)           | 100/100              | AF190463            |
| orf3                    | Unknown (CdoX3)                   | 14087-14875                  | Comamonas sp. strain JS765 | 262 (143-262)         | 90/98                | NZ_AAA101000558     |
| orf4                    | Transpositional regulator (COG0583) | 14844-14479                  | Ralstonia metallidurans | 121 (1-84)            | 65/78                | AB047327            |
| tupBPH                  | Transposase (TnpBPH)              | 15612-16595                  | Achromobacter georgiopolitanum KKS102 | 327 (1-327)           | 52/76                | AJ536297            |
| fdfF                    | Maleylacetate reductase (ClpF)    | 17839-16766                  | Deftuvibacter lusitensis | 357 (1-357)           | 98/100               | AY078159            |
| tflA                    | 2,4-D/alpha-ketoglutarate dioxygenase (TfdA) | 20529-19666                  | Delftia acidovorans P4a | 287 (1-287)           | 99/99                | AY078159            |
| ffdK                    | 2,4-D transport protein (TfdK)     | 22069-20678                  | Delftia acidovorans P4a | 463 (1-463)           | 99/100               | AY078159            |
| ffdB                    | Dienelactone hydrolase (TfdE)     | 23925-22168                  | Delftia acidovorans P4a | 585 (1-585)           | 89/97                | U16782              |
| tfdE                    | Chlorocatechol 1,2-dioxygenase (TfdCII) | 24847-24140                  | Ralstonia eutropha JMP134 plasmid pJP4 | 235 (1-235)           | 88/93                | U16782              |
| ffdC                    | LysR-type transcriptional regulator (TfdR) | 25628-24867                  | Ralstonia eutropha JMP134 plasmid pJP4 | 253 (1-253)           | 47/70                | NC_004463            |
| ffdR                    | Indole acetamide hydrolase (Tms2) | 26046-26960                  | Bradyrhizobium japonicum USDA110 | 304 (1-304)           | 73/91                | AY078159            |
| iaaH                    | Unknown (ORF1)                    | 28435-27014                  | Delftia acidovorans P4a | 473 (1-473)           | 84/95                | AF169302            |
| orf5                    | Chloromuconate cycloisomerase (TfdD) | 29621-28608                  | Burkholderia cepacia R34 | 337 (1-337)           | 41/65                | ZP_00030265        |
| tfuD                    | Putative transposase (ORF4)       | 30735-29623                  | Burkholderia fungorum | 370 (1-370)           | 57/73                | NC_004463            |
| orf6                    | Chloride channel protein (ErIC)   | 31042-31671                  | Bradyrhizobium japonicum USDA110 | 209 (1-209)           | 73/91                | AY078159            |
| cpp                     | Leucine-responsive regulatory protein (Lrp) | 33282-31951                  | Delftia acidovorans P4a | 443 (1-443)           | 42/66                | AF169302            |
| hrp                     | Indole acetamide hydrolase (Tms2) | 34302-33793                  | Burkholderia cepacia R34 | 169 (1-169)           | 94/99                | ZP_00030265        |
| orf5                    | Chloronuconale cycloisomerase (TfdD) | 36612-35599                  | Bradyrhizobium japonicum USDA110 | 411 (1-411)           | 57/73                | NC_004463            |
| tfuD                    | Putative transposase (ORF4)       | 37726-36614                  | Burkholderia fungorum | 337 (1-337)           | 41/65                | ZP_00030265        |
| orf6                    | Chloride channel protein (ErIC)   | 38033-38662                  | Delftia acidovorans P4a | 209 (1-209)           | 57/73                | NC_004463            |
| cpp                     | Leucine-responsive regulatory protein (Lrp) | 40273-38942                  | Bradyrhizobium japonicum USDA110 | 443 (1-443)           | 60/78                | ZP_00125171        |
| hrp                     | Proline dehydrogenase/pyrroline-5-carboxylate dehydrogenase bifunctional protein (COG4230) | 41293-40784                  | Pseudomonas syringae pv. syringae B728a | 169 (1-169)           | 1,317 (1-1317)      |                     |
| putA                    | Malonyl coenzyme A-ACP transacylase (MdcA) | 41332-45132                  | Xanthomonas axonopodis pv. citri str. 306 | 560 (1-560)           | 79/90                | NC_003919            |
| mdcA                    | Malonate decarboxylase alpha subunit (MdcA) | 45142-46824                  | Xanthomonas axonopodis pv. citri str. 306 | 108 (1-108)           | 49/66                | NC_003919            |
| mdcD                    | Malonate decarboxylase delta subunit (MdcD) | 46834-47160                  | Xanthomonas axonopodis pv. citri str. 306 | 246 (1-246)           | 55/71                | NC_003919            |
| mdcB                    | Malonate decarboxylase beta subunit (MdcB) | 47157-48050                  | Xanthomonas axonopodis pv. citri str. 306 | 297 (1-297)           | 49/69                | NC_003919            |
| mdcC                    | Malonate decarboxylase gamma subunit (MdcC) | 48061-48762                  | Xanthomonas axonopodis pv. citri str. 306 | 233 (1-233)           | 31/53                | NC_003919            |
| elgE                    | Holo-ACP synthetase (MdcE)c | 48771-49463                  | Xanthomonas axonopodis pv. citri str. 306 | 230 (1-230)           | 47/62                | NC_003919            |
| mdcH                    | 2′-(5″-Triphosphoribosyl)-3′-dephospho- conjugation (TrbC) | 49460-50332                  | Xanthomonas axonopodis pv. citri str. 306 | 290 (1-290)           | 66/84                | L27758              |
| mdch                    | Malonyl coenzyme A-ACP transacylase (MdcH) | 50329-51237                  | Xanthomonas axonopodis pv. citri str. 306 | 302 (1-302)           | 45/63                | NC_003919            |
| orf7                    | Transcriptional regulator (COG1309) | 52274-51534                  | Burkholderia fungorum | 242 (1-242)           | 33/62                | ZP_00033878        |
| orf8                    | Putative exported protein, conserved in bacteria | 52526-53341                  | Bordetella pertussis Tohama I | 271 (1-263)           | 47/75                | NP_879486           |
| tup1471                 | Transposase (Tnp1471)             | 56714-55677                  | Burkholderia capacia Tohama I | 345 (1-345)           | 100/100              | AF029344            |
| trfA                    | Initiation of vegetative plasmid replication (TrfA2) | 61110-60241                  | Birmingham IncP-alpha plasmid R751 | 289 (1-289)           | 77/88                | L27758              |
| trbA                    | Transcriptional repressor (TrbA)  | 61431-61808                  | Enterobacter aerogenes plasmid R751 | 125 (1-125)           | 63/80                | NC_001735            |
| trbB                    | Inner membrane ATPase (TrbB)      | 62060-63022                  | Birmingham IncP-alpha plasmid R751 | 320 (1-320)           | 81/89                | NC_001735            |
| trbC                    | Involved in mating pair formation during conjugation (TrbD) | 63034-63462                  | Birmingham IncP-alpha plasmid (RP4, RK2, etc.) | 142 (1-142)           | 103 (1-103)          | L27758              |
| trbD                    | Involved in mating pair formation during conjugation (TrbD) | 63465-63776                  | Birmingham IncP-alpha plasmid (RP4, RK2, etc.) | 142 (1-142)           | 86/96                | L27758              |

| **Gene** | **Position** | **Function** | **Homolog** | **Accession Number** |
|-----------|--------------|--------------|-------------|---------------------|
| **traE**  | 63773–65146  | Involved in mating pair formation during conjugation (TrbE) | Birmingham IncP-α plasmid (RP4, RK2, etc.) | L27758 |
| **traC2** | 66617–65130  | Primase (TraC2) | Enterobacter aerogenes plasmid R751 | NC_001735 |
| **traC3** | 65942–65130  | Primase (TraC3) | Birmingham IncP-α plasmid (RP4, RK2, etc.) | L27758 |
| **traD**  | 66998–66621  | Unknown (TraD) | Enterobacter aerogenes plasmid R751 | NC_001735 |
| **traE**  | 69195–67003  | Possible helicase (TraE) | Birmingham IncP-α plasmid (RP4, RK2, etc.) | L27758 |
| **traF**  | 69740–69204  | Involved in mating pair formation during conjugation (TrbE) | Enterobacter aerogenes plasmid R751 | NC_001735 |
| **traG**  | 71650–69737  | DNA transport during conjugation (TraG) | Birmingham IncP-α plasmid (RP4, RK2, etc.) | L27758 |
| **traI**  | 73839–71647  | DNA relaxase (TraI) | Birmingham IncP-α plasmid (RP4, RK2, etc.) | L27758 |
| **traH**  | 72272–71946  | Relaxosome stabilization (TraH) | Birmingham IncP-α plasmid (RP4, RK2, etc.) | L27758 |
| **traX**  | 73877–73836  | Translation coupling (TraX) | Enterobacter aerogenes plasmid R751 | NC_001735 |
| **traJ**  | 74263–73874  | oriT recognizing protein (TraJ) | Enterobacter aerogenes plasmid R751 | NC_001735 |
| **traK**  | 74655–75053  | oriT binding protein (TraK) | Enterobacter aerogenes plasmid R751 | NC_001735 |
| **traL**  | 75053–75778  | Essential transfer protein (TraL) | Enterobacter aerogenes plasmid R751 | NC_001735 |
| **traM**  | 75778–76215  | Essential transfer protein (TraM) | Enterobacter aerogenes plasmid R751 | NC_001735 |
| **upf54.4** | 76933–76304  | Unknown (Upf54.4) | Enterobacter aerogenes plasmid R751 | NC_001735 |

* **a** As the whole catabolic transposon of pEST4011 (positions 9700 to 57634) is practically identical to Tn5530 of Burkholderia cepacia 2a plasmid pJB1 (accession number AF029344), the second-closest relatives are indicated for the pEST4011 coding sequences.
* **b** The sequence of the Birmingham IncP-α plasmid KlcB gene is interrupted by Tnl, which was removed by us before alignment.
* **c** ACP, acyl carrier protein.

7166 VEDLER ET AL.

![image](https://i.imgur.com/your_image_url.png)

FIG. 2. 2,4-D degradation pathway encoded by *tfd* genes. TCA, tricarboxylic acid.

At a position 1.2 kb upstream of *tfdD* there is a gene (*ccp*) whose predicted product is most similar to *Burkholderia fungorum* EriC (41% amino acid identity and 65% amino acid similarity) and *Xanthomonas axonopodis* pv. citri str.306 (accession number NC_00125171) YadQ (37% amino acid identity and 65% amino acid similarity), which are the chloride-channeling proteins (37). As chloride ions are liberated during 2,4-D dissimilation, this protein may have a role in removal of these ions from cells.

The proposed functions and the levels of amino acid identity and similarity of other predicted gene products encoded by the pEST4011 catabolic transposon are shown in Table 1. In this transposon only 11 of 34 ORFs (nine *tfd* genes plus two copies of *ccp* coding for a chloride-channeling protein) are necessary for 2,4-D degradation. In addition to these ORFs, the presence of two copies of *orf5* could be bound to the same function (31). Five ORFs code for (putative) transposases, and four ORFs are translated into (putative) transcriptional regulators; the functions of two ORFs are unknown. The remaining 10 ORFs potentially code for different catabolic functions, including indole acetamide hydrolase (two copies of *iaaH*), proline dehydrogenase/pyrroline-5-carboxylate dehydrogenase bifunctional protein (*putA*), and malonate decarboxylase (*mdc* operon of seven genes). As shown in Table 1, the closest relatives of these catabolic genes are found in bacteria known to be plant pathogens (*X. axonopodis* and *Pseudomonas syringae*) and in a nitrogen-fixing symbiotic bacterium (*B. japonicum*). We analyzed strains EST4002 (containing pEST4011) and EST4003 (containing the depletant plasmid pEST4012) by a microtiter plate-bound substrate utilization assay (Biolog Inc.). The scatter plot of substrate utilization activities for the two strains studied showed a high correlation (*R² = 97.8%, P < 0.0001*), which means that all 95 carbon substrates from Biolog GN2 microplate wells were utilized by these strains at approximately the same rate (Fig. 4).

Indole acetamide hydrolase (*IaaH*) performs the second reaction in the two-step pathway for synthesis of indole acetic acid (a plant hormone auxin) from tryptophan, which is found in associated plant-growth-promoting rhizobacteria (41). Thus, even if the *iaaH* gene codes for a functional enzyme, plasmid pEST4011 itself does not enable its host to synthesize auxin. The PutA bifunctional protein converts L-proline to glutamate for use as a carbon and nitrogen source; in addition, it acts as a repressor of its own expression in response to the proline supply (27). The Biolog analysis showed that strains EST4002 and EST4003 both used L-proline at physiological concentrations as a growth substrate (Fig. 4). Thus, this ability must rely on chromosomal genes.

Decarboxylation of malonate to acetate and CO₂ is a key reaction as it initiates decomposition of this compound in both

---

**pEST4011 region containing genes for 2,4-D degradation**

![image](https://i.imgur.com/your_image_url.png)

**Defluvi bacter lusatiensis chromosomal genes for chlorophenol degradation**

![image](https://i.imgur.com/your_image_url.png)

**Delftia acidovorans P4a chromosomal transposon for 2,4-D degradation**

![image](https://i.imgur.com/your_image_url.png)

**Wautersia eutropha JMP134 plasmid pJP4 region containing genes for 2,4-D degradation**

![image](https://i.imgur.com/your_image_url.png)

FIG. 3. Comparison of the arrangement of the *tfd* genes in plasmid pEST4011 and the homologous genes in *D. lusatiensis* (accession number AJ536297), *D. acidovorans* P4a (AY078159), and *W. eutropha* JMP134 plasmid pJP4 (AY365053). A pEST4011 *tfd* gene and its most similar counterpart are indicated by the same color. The red numbers below the genes indicate percentage of amino acid identity/percentage of amino acid similarity compared with the corresponding pEST4011 *tfd* analogue. The boxes around the genes indicate duplicated regions.

consists of the 1.1-kb IS1471 element (which belongs to the  
IS630 family described by Mahillon and Chandler [29]) inserted into the 3.2-kb class II transposable element IS1071  
(57). While IS1071 flanks a variety of catabolic genes and operons (6), IS1471 has been detected only in the hybrid  
IS1071::IS1471 insertion elements present in pEST4011 and pIJB1. In the case of pIJB1, both copies of IS1071::IS1471  
flanking Tn5530 have been sequenced only partially, but Poh et al. (36) have suggested that these copies are not identical.

At a position 1.5 kb upstream of the lefthand copy of IS1071::IS1471 there is another insertion element in pEST4011  
(Fig. 1), an ISBPH-like structure whose transposase gene is most similar to the *Achromobacter georgiopolitanum* KKS102  
transposase gene *tnpBPH* of insertion element ISBPH (Table 1). In *A. georgiopolitanum* KKS102, ISBPH is located between  
*bphS* and *bphE*, the genes involved in biphenyl degradation (32).

Comparison of the pEST4011 backbone with other IncP1 backbones. A 29-kb region of pEST4011 (positions 1 to 9699  
and 57635 to 76958) contains genes for plasmid replication initiation, stable maintenance in the host cell, and the conjugation machinery for plasmid transfer into new hosts (Fig. 1). The overall structure of the pEST4011 backbone is similar to that of the archetype plasmids belonging to the IncP1 α subgroup (RK2, etc.) (33) and IncP1 β subgroup (R751) (44) (Table 2 briefly describes all plasmids discussed in this paper). The predicted gene products of the backbones of the IncP1 plasmids (completely) sequenced by May 2004, including pUO1 (39), pADP-1 (30), pJP4 (48), pTSA (47), and pB4 (42), are 65 to 100% identical to those of R751. However, the corresponding gene products of the pEST4011 backbone show only 51 to 86% identity to either R751 or RK2 backbone gene products. The proposed functions and levels of amino acid identity and similarity of all predicted pEST4011 backbone gene products are shown in Table 1.

A comparison of the six IncP1 plasmids mentioned above (not including pJP4) (1, 39) showed that various resistance and catabolic determinants are inserted mostly into two regions in the plasmid backbones, between *oriV* and the *trfA* gene and between the Tra1 and Tra2 regions. When we compared plasmids pEST4011 and R751, the archetype plasmid of the IncP1 β subgroup, we found that pEST4011 contains only one catabolic transposon inserted between the core of *oriV* and the *trfA* gene (between iterons 2 and 3) (Fig. 5).

In the pEST4011 backbone two ORFs were found between the *upf54.8* and *kfrA* genes, namely, *orf1* and *orf2*, whose predicted amino acid sequences are most similar to the sequences of the *Xylella fastidiosa* 9a5c cell filamentation protein and to an unknown protein, respectively (Table 1). These two ORFs are not present in any of the plasmids mentioned above. In addition, two small genes, *kleB* and *kleG*, which are devoted to stable inheritance, as well as two genes having unknown functions, *kluA* and *kluB*, located in the R751 backbone were not identified in pEST4011.

In R751, RK2, pJP4, and pB4 there are two forms of the replication initiation protein TrfA encoded by the *trfA1* and *trfA2* genes, the larger TrfA-44 protein (382 to 407 amino acids) and the smaller TrfA-33 protein (284 to 289 amino acids). These proteins are translated from the same reading frame, and the translational start sites are 97 to 122 amino acids apart. It has

TABLE 2. Plasmids discussed in this paper

| Plasmid | Description$^a$ | GenBank accession no. | Reference(s) |
|---------|------------------|-----------------------|--------------|
| RK 2, RP 4, etc. | IncP-1α, 60,099 bp, Amp<sup>r</sup>Tet<sup>r</sup>, isolated from enteric bacteria in Birmingham, UK | L27758 | 33 |
| R751 | IncP-1β, 53,425 bp, Tmp<sup>r</sup>, isolated from Enterobacter aerogenes in UK | NC_001735 | 44 |
| pJP4 | IncP-1β, 87,688 bp, Hg<sup>r</sup>, contains *tfd* genes for 2,4-D and 3-CBA degradation, isolated from *Ralstonia eutropha* JMP 134 in Australia | AY365053 | 8 |
| pIJB1 | IncP-1β, about 102 kb, Hg<sup>r</sup>, contains *tfd* genes for 2,4-D degradation located on Tn5530, isolated from *Burkholderia cepacia* 2a in UK | AF029344 | 36, 56 |
| pUO1 | IncP-1β, 67,066 bp, Hg<sup>r</sup>, contains haloacetate-catabolic genes, isolated from *Delftia acidovorans* B in Osaka, Japan | NC_005088 | 39 |
| pADP-1 | IncP-1β, 108,845 bp, Hg<sup>r</sup>, contains atrazine-catabolic genes, isolated from *Pseudomonas* sp. strain ADP in Minnesota | NC_004956 | 30 |
| pB4 | IncP-1β, 79,370 bp, Sm<sup>r</sup>Em<sup>r</sup>Amx<sup>r</sup>, isolated from microbial community of activated sludge from wastewater treatment plant in Germany by biparental mating | NC_003430 | 42 |
| pTSA | IncP-1β, about 72 kb, Hg<sup>r</sup>, contains *p*-toluenesulfonate-catabolic genes, isolated from *Comamonas testosteroni* T-2 in Switzerland | AF303942, AF305549, AF311437, AF311820, AH010657, AY227144, AY227147, U73743 | 47 |
| pEMT3 | IncP-1, initially contained *tfd* genes for 2,4-D degradation, but the degradative capacity has disappeared, isolated from microbial community of agricultural soil in Michigan by biparental mating | AJ414161, AJ414162 | 11, 45 |
| pQKH54 | IncP-1γ, about 80 kb, Hg<sup>r</sup>, isolated from epilithic microbial community of the River Taff in South Wales, UK, by triparental mating |  | 14 |
| pTV1 | Unknown Inc group, about 200 kb, Hg<sup>r</sup>, contains *tfd* genes for 2,4-D degradation, isolated from *Variovorax paradoxus* TV1 in Dijon, France | AB028643 | 49, 50 |

$^a$ Abbreviations: Amp, ampicillin; Tet, tetracycline; Tmp, trimethoprim; Hg, mercury; Sm, streptomycin; Em, erythromycin; Amx, amoxicillin; UK, United Kingdom; 3-CBA, 3-chlorobenzoate.

been shown that the smaller protein is sufficient for plasmid replication in many hosts (59). In pUO1, pTSA, and pEMT3 only the gene coding for TrfA-44 (406 to 411 amino acids) has been annotated, while in pADP-1 and pEST4011 only TrfA-33 proteins are encoded (303 and 289 amino acids, respectively). In pADP-1, the region corresponding to the N-terminal part of TrfA-44 is missing, and in pEST4011 the deletion also encompasses the *ssb* gene, which codes for a single-stranded DNA-binding protein involved in DNA replication, recombination, and repair and is present in all of the other plasmids mentioned above just upstream of *trfA*. In pEST4011 an additional 2,360 bp of DNA (positions 57635 to 59995) is inserted between the catabolic transposon and iteron 2 (positions 60146 to 60166) of *oriV*. The nucleotide sequence of this region is not similar to any sequence in the GenBank database.

In pEST4011, a large amount of DNA coding for the C-terminal half of TrbE and genes *trbF* to *trbN*, as well as the region coding for most of the C-terminal region of TraC2 and TraC3 and all of the *traC4* gene, which are present in R751, RK2, pJP4, pADP-1, pUO1, and pB4, are missing. In the latter plasmids, this region may also contain additional genes, but the important fact is that it includes seven *trb* genes (*trbE* to *trbL*) essential for mating pair formation during conjugation (1).

As in other IncP1 plasmids, five transcriptional regulators are encoded in the pEST4011 backbone. KorA, KorB, and KorC are three global repressors that regulate transcription of the genes necessary for plasmid replication, stable inheritance, and conjugation; TrbA is a transcriptional repressor of conjugation genes; and KfrA has been shown to regulate only its own transcription (1). Consensus sequences for the binding of these regulators have been determined for KorA, KorB, KorC, and TrbA (3, 33, 44). Most of these binding sites are also present in pEST4011; the only exception is one TrbA binding site present in the RK2 *traJ/traK* promoter-operator region (Table 3). Additionally, one KorB binding site upstream of *traJ* and one KorC binding site upstream of *klcA*, which are not present in R751 or RK2, were found in pEST4011. As shown in Table 3, the binding sites of KorA and KorB are well conserved.

FIG. 5. Comparison of the linear genetic maps of plasmids pEST4011 and R751 (accession number NC_001735). Different types of boxes represent common sets of backbone genes, and the corresponding functions are indicated between the two maps. The arrows with boxes indicate insertions not found in the two plasmids; the pEST4011 sites from which the *kleG* and *kleB* genes are missing (compared with R751) are indicated by triangles. Some of the pEST4011 and R751 backbone gene positions are also indicated for orientation.

DNA-          Binding site       Binding site      Binding site sequence in     Binding site sequence    Binding site sequence
binding       coordinates       location in       pEST 4011 (5' to 3')         in R751 (5' to 3')        in RK2 (5' to 3')
protein                       in pEST 4011

KorA          2314–2323         kfrAp             TTTAGCTAAA               TTTAGCTGAA              TTTAGCTGAA
             4659–4668         korAp             TTTAGCTAAA               TTTAGCTAAA              TTTAGCTAAA
             6054–6063         kleAp             TTAAACGAAA               TTTAGCTAAA              TTTAGCTAAA
             8401–8410         klcAp             TTTAGCTAAA               TTTAGCTAAA              TTTAGCTAAA
             61353–61362       trfAp             TTTAGCTAAA               TTTAGCTAAA              TTTAGCTAAA

KorB          2329–2341         kfrAp             TTTAGCGGCTAAA            TTTAGCGGCTGAA           TTTAGCGGCTGAA
             4690–4702         korAp             TTTAGCCGCTAAA            TTTAGCGGCTAAA           TTTAGCCGCTAAA
             6137–6149         kleAp             TTTAGCCGCTAAA            TTTAGCCGCTAAA           TTTAGCCGCTAAA
             8412–8424         klcAp             TTTAGCCGCTAAA            TTTAGCCGCTAAA           Not present
             61384–61396       trfAp             TTTAGCGGCTAAA            TTTAGCGGCTAAA           TTTAGCGGCTAAA
             61824–61836       trbBp             TTTAGCGGCTAAA            TTTAGCCGCTAAA           TTTAGCGGCTAAA
             69453–69465       traF              TTTAGCGGCTAAA            TTTAGCCGCTAAA           TTTAGCGGCTAAA
             73857–73869       traX              TTTAGCGGCTAAA            TTTAGCCGCTAAA           TTTAGCGGCTAAA
             74273–74285       Upstream of traJ  TTTAGCGGCTAAA            Not present             Not present
             76948–2           Upstream of upf54.4 TTTAGCCGCTAAA            TTTAGCCGCTAAA

KorC          6025–6040         kleAp             TAGGGCAAAATGCCCTA        TAGGACAAAATGTCCCTA      TAGGGCATTA TGCCCTA
             8373–8389         klcAp             TAGGGCAAAATGCCCTA        TAGGACAAAATGTCCCTA      TAGGGCATTA TGCCCTA
             8752–8768         Upstream of klcA  TAGGGCAAAATGCCCTA        Not present             Not present

TrbA          61253–61266       trbAp             CTCGATGTATCGCT           ACAGCCAGGTCGAA          ACCGCTATAT ATCGAA
             62004–62017       trbBp             GTTGGTATATCGTT           TTCGATATATCGTA          TTCGGTATAT ATCGTT
             71848–71861       traGp             GTCAGTATAGCGGG           GGCAGTATATCGGG          ATCAGTATAT ATCGTG
             71867–71880       traGp             TTCGGTATATCGAA           TTCGGTATATCGAA          TTCGGTATAT ATCGAA
             74493–74506       traJp/traKp       ACCGATATAGCGAA           ACGGATATAGCGAA          ATGGATATAA CGGAA

TraJ          74390–74404       oriT              CGGTTAGCTAACTTC          CGCTTAGCTAACTTC         GGGTGGGGCC TACTTC

TraI          74409–74414       oriT              ATCCTG                   ATCCTG                  ATCCTG

TrfA          8790–8810         oriV              TTTCTTTGACACTTGAGGGGGC   GCTTCTTGACACTTGAGGGGGC  TTTCATTGAC ACTTGAGGGGGC
             9285–9305c         oriV              CACCCTAGACACTTGAGGGGGC   TATCCATTGACACTTGAGGGGGC CACCATTGAC ATT TGAGGGGGC
             9307–9327c         oriV              AACTACAGACATT TGAGGGGGC GACTATTGACCCTTTGAGGGGGC GAGTGATGAC AGATGAGGGGGC
             9329–9349c         oriV              CTTCGTTGACACTTGTTGGGGGC  GATCCTTGACACTTGACGGGGC  GATCCTTGAC ACTTGAGGGGGC
             9352–9372c         oriV              AAC CCTAGACACTTGGGGGGGC   GGGTGCTGACAGATGAGGGGGC  GACTACTGAC AGATGAGGGGGC
             9375–9395c         oriV              TTCTATAGACACTTGAGGGGGC   CC GTA CTGACACTTGAGGGGGC   CGGTATTGAC ACTTGAGGGGGC
             9501–9521c         oriV              CACTGCAGACACTTGAGGGGGC   CGATGCTGACAGATGAGGGGGC  CGGCGTTGAC AGATGAGGGGGC
             9536–9556c         oriV              GTTCATTGACACTTG GGGGGGC G TTTCTTGACACGTTGGGGGGC GGACGTTGAC ACTTGAGGGGGC
             9557–9577c         oriV              TCACCGTGACACTTG GGGGGGC GGCTGGTGACAGTTTGAGGGGGC CCTCACTGAC AGATGAGGGGGC
             60146–60166c       oriV              GCATGCAGACACTTGAGGGGGC   ACTTTT TGACATT TGAGGGGGC TTTCATTGAC ACTTGAGGGGGC
             60176–60196c       oriV              TTTCATTGACAGATGAGGGGGC   Not present             Not present

* The putative consensus operator sequence is indicated by boldface type.

7170 VEDLER ET AL.

A.
![Diagram A](#)

B.
![Diagram B](#)

FIG. 6. (A) oriV region of pEST4011. The binding sites for replication initiation protein TrfA (iterons i1 to i11) together with the directions are indicated by triangles. The gray boxes represent palindromic sequences (P); the black boxes are DnaA binding sites (D). The G-C- and A-T-rich regions, the direction of replication, the site of additional DNA insertion, and the *trfA* gene (with the direction of transcription) are also indicated. (B) oriT region of pEST4011. The arrows indicate the binding sites for the TraI, TraJ, and TraK proteins and the *nic* site. The gray box represents the region that is highly conserved in different IncP1 plasmids. The translation start codons together with the directions are indicated for the *traJ* and *traK* genes. The relaxation region and the bent region are indicated by dotted lines; the directions of transcription from the promoters of *traJ* (*Ptraj*) and *traK* (*Ptrak*) and the putative direction of DNA transfer during conjugation are indicated by horizontal arrows.

archetype plasmids of the IncP1 α and IncP1 β subgroups, respectively. In order to examine to which IncP1 subgroup pEST4011 belongs, we constructed multiple alignments of the available full-length amino acid sequences of the short form of TrfA (TrfA-33), KorA, and TraG of IncP1 plasmids (Table 2). In cases in which the coding sequences for TrfA-33 were not annotated, we generated the necessary sequences from TrfA-44 by deleting the N-terminal 122 amino acids in the case of pUO1 and pTSA and 123 amino acids in the case of pEMT3 (based on similarities with RK2 and R751). The bootstrapped neighbor-joining trees derived from these alignments (obtained by using the program CLUSTALX) (Fig. 7) showed that pEST4011 TrfA, KorA, and TraG are distinct from all other corresponding sequences and that they occupy individual branches of each tree. Similar results were obtained with the different PHYLIP programs used (data not shown). As C. M. Thomas and coworkers have assigned plasmid pQKH54 (14) to the γ subgroup (personal communication) and the amino acid sequence of TrfA of pQKH54 appeared to be different from the pEST4011 TrfA sequence (unpublished data), we placed pEST4011 in a new IncP1 subgroup, the δ subgroup. The other four IncP1 catabolic plasmids sequenced, pJP4, pADP-1, pUO1, and pTSA, together with pIJB1, possess the well-conserved IncP1 subgroup β backbone. The TrfA2 phylogenetic tree (Fig. 7A) shows that the 2,4-D-degradative plasmid pEMT3 is also different from all other IncP1 plasmids and could also form a separate subgroup. Unfortunately, the incompatibility group of the 2,4-D-degradative plasmid pTV1 is unknown (T. Vallaeys, personal communication).

Comparison of pEST4011 and its laboratory ancestor, pD2M4. The original strain D2M4, containing plasmid pD2M4 (about 95 kb), had a very unstable 2,4-D⁺ phenotype; 99% of the cells lost this phenotype when they were grown in Luria-Bertani medium, and all of the 2,4-D⁻ clones studied contained no plasmid. Subsequently, a series of selections were performed (V. Kõiv, unpublished data); in each step D2M4 cells were cultivated in nonselective medium, and this was followed by selection on medium containing 2,4-D as a sole source of carbon and energy. The resulting 2,4-D⁺ clone with the best growth on 2,4-D was selected for the next step. Finally, a strain with improved stability of the 2,4-D⁺ phenotype was obtained; about 50% of the cells lost the 2,4-D⁺ phenotype when they were grown in nonselective medium. The selected strain was designated EST4002, and it contained a smaller plasmid, pEST4002 (estimated size, 78 kb). In order to increase the stability of the plasmid and to overcome the difficulties with DNA extraction, pEST4002 was transferred by conjugation into a plasmid-free recipient strain, *Pseudomonas putida* PaW340 (28). The resulting 2,4-D⁺ transconjugants contained plasmid pEST4011 (70 kb, as determined by restriction analysis). After maintenance of strain EST4002 on 2,4-D minimal medium agar plates for years, the strain acquired a very stable 2,4-D⁺ phenotype; only 1 to 2% of the cells lost the phenotype when they were grown in Luria-Bertani medium. According to a restriction analysis this strain contained exactly the same pEST4011 plasmid as the 2,4-D⁺ transconjugants of PaW340 (53).

Due to problems arising from instability of the 2,4-D⁺ phenotype and the difficulties with DNA extraction, we were not able to maintain either the original strain D2M4 containing pD2M4 or plasmid pEST4002. As Mäe et al. have shown (28), plasmid pEST4002 had one additional 7.8-kb EcoRI restriction fragment compared with pEST4011. However, the exact location and size of the deletion were not determined. During the present study we compared the EcoRI and HindIII restriction patterns of pD2M4 (Kõiv, unpublished) and

pEST4011 (Fig. 8A and B). We found that pEST4011 had lost the approximately 23.5- and 20.5-kb HindIII restriction fragments and the approximately 26-, 9.5-, and 4.6-kb EcoRI fragments compared with pD2M4. In addition, the 19,609-bp HindIII fragment and the 6,991- and 10,872-bp EcoRI fragments of pEST4011 are not present in pD2M4. Instead of the 21,849-bp pEST4011 HindIII fragment, pD2M4 had a corresponding band at about 15 kb.

These results show that (i) pEST4011 lost approximately 25 to 29 kb of DNA from somewhere between position 57349 and position 66873 (i.e., between the righthand copy of IS1071::IS1471 and the traD gene) and (ii) pD2M4 did not contain the 6,991-bp duplication present in pEST4011 containing genes lrp to iaaH (Fig. 1 and 8C). Indeed, when pEST4011 was compared with R751, we noticed that in case of pEST4011 about 14 kb of backbone DNA containing (among other genes) a set of trb genes essential for conjugation (trbE to trbL) (Fig. 5) was missing. The remaining 11 to 15 kb could comprise additional DNA inserted between the Tra1 and Tra2 regions, as reported for all other IncP1 plasmids discussed in this study except pJP4 (1, 39, 48). Thus, on the one hand, this deletion somehow ensured its host a very stable 2,4-D⁺ phenotype, and on the other hand, pEST4011 lost the ability to self-transmit.

The 6,991-bp duplication in the pEST4011 catabolic region is not present either in pD2M4 or in catabolic transposon Tn5530 of pIJB1. At least two of six ORFs in this region can be directly related to 2,4-D degradation; namely, tfdD is responsible for the fourth step in the corresponding pathway (Fig. 2), and the ccp product may function as a channeling protein for the removal of chloride liberated during 2,4-D dissimilation from cells. The predicted amino acid sequence of the pEST4011 TfdD protein is far more similar to the sequence of TcbD of Pseudomonas sp. strain P51 plasmid pP51 (P27099) than to the sequence of pJP4 TfdD₁ or TfdD₂; the levels of amino acid identity are 83, 64, and 32%, respectively, and the levels of amino acid similarity are 96, 87, and 57%, respectively. The tcb genes of pP51 are responsible for degradation of 1,2,4-trichlorobenzene by its host strain (52), and the substrate for TcbD is 2,3,5-trichloro-cis,cis-muconate. The corresponding 2,4-D degradation intermediate and substrate for the pEST4011 and pJP4 tfdD gene products is 2,4-dichloro-cis,cis-muconate. Vollmer et al. (55) showed that although they could not measure enzyme kinetics with 2,3,5-trichloro-cis,cis-muconate as the substrate, 2,4-dichloro-cis,cis-muconate was the best substrate for pP51 TcbD and pJP4 TfdD₁. However, the catalytic efficiency of TcbD was somewhat lower than that of TfdD (kcat/Km, 53 and 120 μM⁻¹ min⁻¹, respectively). Therefore, it is tempting to speculate that the 7-kb pEST4011 region was duplicated in order to increase the tfdD copy number and consequently the amount of the corresponding enzyme in a cell in order to degrade 2,4-D more efficiently.

Mercury resistance is a common heavy metal resistance in bacterial isolates from soil and water environments (19). As shown in Table 2, all catabolic IncP1 plasmids except the least-studied plasmid pEMT3 bear mercury resistance genes. Although plasmid pEST4011 does not carry any Hg²⁺ determinants, it is probable that the precursor plasmid pD2M4 did. In pADP-1 and pUO1 the mer genes are located between the Tra1 and Tra2 regions; if this was the case for pD2M4 as well, the corresponding genes were lost during laboratory evolution.

FIG. 8. (A) Restriction analysis of plasmid pD2M4 digested with EcoRI and HindIII. Plasmid pWW0 (accession number NC_003350) was digested with the same restriction endonucleases to provide size markers. The DNA fragments were separated in a 0.7% agarose gel. The numbers and arrows on the right indicate the approximate positions of the corresponding fragments (in base pairs). The arrows with the open arrowheads indicate the pD2M4 restriction fragments not present in pEST4011. (B) Restriction analysis of plasmid pEST4011 digested with EcoRI and HindIII. The DNA fragments were separated in a 0.8% agarose gel. The numbers and arrows on the right indicate the approximate positions of the corresponding fragments (in base pairs). The arrows with the open arrowheads indicate the pEST4011 restriction fragments not present in pD2M4. (C) Aligned circular restriction maps of plasmids pEST4011 and pD2M4. The outer circle with recognition positions of the corresponding restriction enzymes represents pEST4011. The inner discontinuous circle represents pD2M4, and the boxes indicate the corresponding digestion products. The restriction fragments of pD2M4 not present in pEST4011 are shown outside the circles, and the arrows indicate the sites in pD2M4 where these fragments are probably embedded in an unknown order. The line with the open triangle at the end indicates the site in pEST4011 where the essential transfer genes *trbE* to *trbL* are missing. The duplicated regions of pEST4011 are indicated by dotted lines.

However, mercury resistance has not been determined for either strain EST4002 or EST4003.

Our first conclusion is that it seems that despite similarities in overall organization, the backbones of IncP1 plasmids are far more heterogeneous than previously thought. Also, as shown in Table 2, very similar IncP1 backbones, although loaded with different additional functions, can be isolated from geographically distinct regions of the world. Second, diverse genes and gene cassettes are assembled into (2,4-D) catabolic pathways having different origins, but once a working set of necessary genes has been brought together (for example, in a composite catabolic transposon), these genes may spread horizontally as one unit and this unit may insert itself into different vehicles, like conjugative (broad-host-range) plasmids. Other catabolic and/or regulatory genes encoded by these composite transposons may be just entrapped passengers with no significance in one host, while they can probably confer an altered phenotype to another host in certain conditions. Finally, serious attention should be paid to the fact that the occurrence of laboratory evolution caused by continuous enrichment procedures during the isolation of natural catabolic plasmids may hinder studies and distort our understanding of evolution in nature.

### ACKNOWLEDGMENTS

This work was supported by Estonian Science Foundation grant 5682.

We gratefully thank Christopher Thomas for fruitful discussions and sending us the amino acid sequence of TrfA of pQKH54. We also thank Tatiana Vallaey for the information concerning plasmid pTV1 and Jaak Truu and Viia Kõiv for their substantial contributions to this paper.

### REFERENCES

1. Adamczyk, M., and G. Jagura-Burdzy. 2003. Spread and survival of promising IncP-1 plasmids. Acta Biochim. Pol. **50**:425–453.

2. Ausmees, N., and A. Heinaru. 1990. New plasmids of herbicide 2,4-dichlorophenoxyacetic acid biodegradation. Genetika 26:770–772. (In Russian.)
3. Bingle, L. E. H., and M. Zatyka. 2003. Co-operative interactions control conjugative transfer of broad host-range plasmid RK2: full effect of minor changes in TrbA operator depends on KorB. Mol. Microbiol. 49:1095–1108.
4. Brinkman, A. B., T. J. G. Ettema, W. M. de Vos, and J. van der Oost. 2003. The Lrp family of transcriptional regulators. Mol. Microbiol. 48:287–294.
5. Camacho, E. M., and J. Casadesus. 2003. Conjugal transfer of the virulence plasmid of *Salmonella enterica* is regulated by the leucine-responsive regulatory protein and DNA adenine methylation. Mol. Microbiol. 44:1589–1598.
6. di Gioia, D., M. Peel, F. Fava, and R. C. Campbell. 1998. Structures of homologous composite transposons carrying *cbaABC* genes from Europe and North America. Appl. Environ. Microbiol. 64:1940–1946.
7. Dimroth, P., and H. Hilbi. 1997. Enzymic and genetic basis for bacterial growth on malonate. Mol. Microbiol. 25:3–10.
8. Don, R. H., and J. M. Pemberton. 1981. Properties of six pesticide degradation plasmids isolated from *Alcaligenes paradoxus* and *Alcaligenes eutrophus*. J. Bacteriol. 145:681–686.
9. Fulthorpe, R. R., C. McGowan, O. V. Maltseva, W. E. Holben, and J. M. Tiedje. 1995. 2,4-Dichlorophenoxyacetic acid-degrading bacteria contain mosaics of catabolic genes. Appl. Environ. Microbiol. 61:3274–3281.
10. Fulthorpe, R. R., A. N. Rhodes, and J. M. Tiedje. 1996. Pristine soils mineralize 3-chlorobenzoate and 2,4-dichlorophenoxyacetic acid via different microbial populations. Appl. Environ. Microbiol. 62:1159–1166.
11. Gstalder, M.-E., M. Faelen, N. Mine, E. M. Top, M. Mergeay, and M. Couturier. 2003. Replication functions of new broad host range plasmids isolated from polluted soils. Res. Microbiol. 154:499–509.
12. Hall, T. A. 1999. BioEdit: a user-friendly biological sequence alignment editor and analysis program for Windows 95/98/NT. Nucleic Acids Symp. Ser. 41:95–98.
13. Hansen, J. B., and R. H. Olsen. 1978. Isolation of large bacterial plasmids and characterization of the P2 incompatibility group plasmids pMG1 and pMG5. J. Bacteriol. 135:227–238.
14. Hill, K. E., A. J. Weightman, and J. C. Fry. 1992. Isolation and screening of plasmids from the epilithon which mobilize recombinant plasmid pD10. Appl. Environ. Microbiol. 58:1292–1300.
15. Hoffmann, D., S. Kleinsteuber, R. H. Muller, and W. Babel. 2003. A transposon encoding the complete 2,4-dichlorophenoxyacetic acid degradation pathway in the alkalitolerant strain *Delftia acidovorans* P4a. Microbiology 149:2545–2556.
16. Itoh, K., R. Kanda, Y. Sumita, H. Kim, Y. Kamagata, K. Suyama, H. Yamamoto, R. P. Hausinger, and J. M. Tiedje. 2002. *tfdA*-like genes in 2,4-dichlorophenoxyacetic acid-degrading bacteria belonging to the *Bradyrhizobium-Agromonas-Nitrobacter-Afipia* cluster in α-Proteobacteria. Appl. Environ. Microbiol. 68:3449–3454.
17. Ka, J. O., W. E. Holben, and J. M. Tiedje. 1994. Genetic and phenotypic diversity of 2,4-dichlorophenoxyacetic acid (2,4-D)-degrading bacteria isolated from 2,4-D treated field soils. Appl. Environ. Microbiol. 60:1106–1115.
18. Kamagata, Y., R. R. Fulthorpe, K. Tamura, H. Takami, L. J. Forney, and J. M. Tiedje. 1997. Pristine environments harbor a new group of oligotrophic 2,4-dichlorophenoxyacetic acid-degrading bacteria. Appl. Environ. Microbiol. 63:2266–2272.
19. Kelly, W. J., and D. C. Reaney. 1984. Mercury resistance among soil bacteria: ecology and transferability of genes encoding resistance. Soil Biol. Biochem. 16:1–8.
20. Kim, Y. S. 2002. Malonate metabolism: biochemistry, molecular biology, physiology and industrial application. J. Biochem. Mol. Biol. 35:443–451.
21. Kõiv, V., R. Marits, and A. Heinaru. 1996. Sequence analysis of the 2,4-dichlorophenol hydroxylase gene *tfdB* and 3,5-dichlorocatechol 1,2-dioxygenase gene *tfdC* of 2,4-dichlorophenoxyacetic acid degrading plasmid pEST4011. Gene 174:293–297.
22. Laemmli, C. M., J. H. J. Leveau, A. J. B. Zehnder, and J. R. van der Meer. 2000. Characterization of a second *tfd* gene cluster for chlorophenol and chlorocatechol metabolism on plasmid pJP4 in *Ralstonia eutropha* JMP134 (pJP4). J. Bacteriol. 182:4165–4172.
23. Laemmli, C. M., C. Werlen, and J. R. van der Meer. 2004. Mutation analysis of the different *tfd* genes for degradation of chloroaromatic compounds in *Ralstonia eutropha* JMP134. Arch. Microbiol. 181:112–121.
24. Larsen, M. H., and D. H. Figurski. 1994. Structure, expression and regulation of the *kilC* operon of promiscuous IncPα plasmids. J. Bacteriol. 176:5022–5032.
25. Leveau, J. H. J., F. König, H. Füchslin, C. Werlen, and J. R. van der Meer. 1999. Dynamics of multigene expression during catabolic adaptation to *Ralstonia eutropha* JMP134(pJP4) to the herbicide 2,4-dichlorophenoxyacetic acid. Mol. Microbiol. 33:396–406.
26. Leveau, J. H. J., and J. R. van der Meer. 1996. The *tfdR* gene product can successfully take over the role of the insertion element-inactivated TfdT protein as a transcriptional activator of the *tfdCDEF* gene cluster, which encodes chlorocatechol degradation in *Ralstonia eutropha* JMP134(pJP4). J. Bacteriol. 178:6824–6832.
27. Ling, M., S. W. Allen, and J. M. Wood. 1994. Sequence analysis identifies the proline dehydrogenase and delta 1-pyrroline-5-carboxylate dehydrogenase domains of the multifunctional *Escherichia coli* PutA protein. J. Mol. Biol. 243:950–956.
28. Mäe, A., R. Marits, N. Ausmees, V. Kõiv, and A. Heinaru. 1993. Characterization of a new 2,4-dichlorophenoxyacetic acid degrading plasmid pEST4011: physical map and localization of catabolic genes. J. Gen. Microbiol. 139:3165–3170.
29. Mahillon, J., and M. Chandler. 1998. Insertion sequences. Microbiol. Mol. Biol. Rev. 62:725–774.
30. Martinez, B., J. Tomkins, L. P. Wackett, R. Wing, and M. J. Sadowsky. 2001. Complete nucleotide sequence and organization of the atrazine catabolic plasmid pADP-1 from *Pseudomonas* sp. strain ADP. J. Bacteriol. 183:5684–5697.
31. Ogawa, N., and K. Miyashita. 1999. The chlorocatechol-catabolic transposon Tn5707 of *Alcaligenes eutrophus* NH9, carrying a gene cluster highly homologous to that in the 1,2,4-trichlorobenzene-degrading bacterium *Pseudomonas* sp. strain P51, confers the ability to grow on 3-chlorobenzoate. Appl. Environ. Microbiol. 65:724–731.
32. Ohtsubo, Y., M. Delawary, K. Kimbara, M. Takagi, A. Ohta, and Y. Nagata. 2001. BphS, a key transcriptional regulator of *bph* genes involved in polychlorinated biphenyl/biphenyl degradation in *Pseudomonas* sp. KKSl02. J. Biol. Chem. 276:36146–36154.
33. Pansegrau, W., E. Lanka, P. T. Barth, D. H. Figurski, D. G. Guiney, D. Haas, D. R. Helinski, H. Schwab, V. A. Stanisich, and C. M. Thomas. 1994. Complete nucleotide sequence of Birmingham IncPα plasmids. J. Mol. Biol. 239:623–663.
34. Pansegrau, W., W. Schröder, and E. Lanka. 1993. Relaxase (TraI) of IncPα plasmid RP4 catalyzes a site-specific cleaving-joining reaction of single-stranded DNA. Proc. Natl. Acad. Sci. USA 90:2925–2929.
35. Plumeier, I., D. Perez-Pantoja, S. Heim, B. Gonzalez, and D. H. Pieper. 2002. Importance of different *tfd* genes for degradation of chloroaromatics by *Ralstonia eutropha* JMP134. J. Bacteriol. 184:4054–4064.
36. Poh, R. P.-C., A. R. W. Smith, and I. J. Bruce. 2002. Complete characterization of Tn5530 from *Burkholderia cepacia* strain 2a(pIJB1) and studies of 2,4-dichlorophenoxyacetate uptake by the organism. Plasmid 48:1–12.
37. Purdy, M. D., and M. C. Wiener. 2000. Expression, purification, and initial structural characterization of YadQ, a bacterial homolog of mammalian ClC chloride channel proteins. FEBS Lett. 466:26–28.
38. Sambrook, J., E. F. Fritsch, and T. Maniatis. 1989. Molecular cloning: a laboratory manual, 2nd ed. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y.
39. Sota, M., H. Kawasaki, and M. Tsuda. 2003. Structure of haloacetate-catabolic IncP-1β plasmid pUO1 and genetic mobility of its residing haloacetate-catabolic transposon. J. Bacteriol. 185:6741–6745.
40. Starcic-Erjavec, M., J. P. van Putten, W. Gaaster, B. J. Jordi, M. Grabnar, and D. Zgur-Bertok. 2003. H-NS and Lrp serve as positive modulators of *traJ* expression from the *Escherichia coli* plasmid pRK100. Mol. Genet. Genomics 270:94–102.
41. Steenhoudt, O., and J. Vanderleyden. 2000. *Azospirillum*, a free-living nitrogen-fixing bacterium closely associated with grasses: genetic, biochemical and ecological aspects. FEMS Microbiol. Rev. 24:487–506.
42. Tauch, A., A. Schlüter, N. Bischoff, A. Goesmann, F. Meyer, and A. Pühler. 2003. The 79,370-bp conjugative plasmid pB4 consists of an IncP-1β backbone loaded with a chromate resistance transposon, the *strA-strB* streptomycin resistance gene pair, the oxacillinase gene *bla*<sub>NPS-1</sub>, and a tripartite antibiotic efflux system of the resistance-nodulation-division family. Mol. Gen. Genomics 268:570–584.
43. Thompson, J. D., T. J. Gibson, F. Plewniak, F. Jeanmougin, and D. G. Higgins. 1997. The ClustalX Windows interface: flexible strategies for multiple sequence alignment aided by quality analysis tools. Nucleic Acids Res. 24:4876–4882.
44. Thorsted, P. B., D. P. Macartney, P. Akhtar, A. S. Haines, N. Ali, P. Davidson, T. Stafford, M. J. Pocklington, W. Pansegrau, B. M. Wilkins, E. Lanka, and C. M. Thomas. 1998. Complete sequence of the IncPβ plasmid R751: implications for evolution and organization of the IncP backbone. J. Mol. Biol. 282:969–990.
45. Top, E. M., W. E. Holben, and L. J. Forney. 1995. Characterization of diverse 2,4-dichlorophenoxyacetic acid-degradative plasmids isolated from soil by complementation. Appl. Environ. Microbiol. 61:1691–1698.
46. Top, E. M., and D. Springael. 2003. The role of mobile genetic elements in bacterial adaptation to xenobiotic organic compounds. Curr. Opin. Biotechnol. 14:1–8.
47. Tralau, T., A. M. Cook, and J. Ruff. 2001. Map of the IncP1β plasmid pTSA encoding the widespread genes (*tsa*) for *p*-toluenesulfonate degradation in *Comamonas testosteroni* T-2. Appl. Environ. Microbiol. 67:1508–1516.
48. Trefault, N., R. De la Iglesia, A. M. Molina, M. Manzano, T. Ledger, D. Pérez-Pantoja, M. A. Sanchez, M. Stuardo, and B. Gonzalez. 2004. Genetic organization of the catabolic plasmid pJP4 from *Ralstonia eutropha* JMP134(pJP4) reveals mechanisms of adaptation to chloroaromatic pollutants and evolution of specialized chloroaromatic degradation pathways. Environ. Microbiol. 6:–668.
49. Vallaeys, T. 1992. Isolement d’une communautémicrobienne dégradante l’acide 2,4-dichlorophenoxyacétique à partir d’un sol de Dijon. Caractérisation et

tion cinétique et génétiques des souches impliquées. Ph.D. thesis. University of Lille, Lille, France.

50. Vallaëys, T., L. Albino, G. Soulas, A. D. Wright, and A. J. Weightman. 1998. Isolation and characterization of a stable 2,4-dichlorophenoxycetic acid degrading bacterium, *Variivorax paradoxus*, using chemostat culture. Biotechnol. Lett. **20**:1073–1076.

51. Vallaëys, T., L. Courde, C. McGowan, A. D. Wright, and R. R. Fulthorpe. 1999. Phylogenetic analyses indicate independent recruitment of diverse gene cassettes during assembly of the 2,4-D catabolic pathway. FEMS Microbiol. Ecol. **28**:373–382.

52. van der Meer, J. R., R. I. L. Eggen, A. J. B. Zehnder, and W. M. de Vos. 1991. Sequence analysis of the *Pseudomonas* sp. strain P51 *tcb* gene cluster, which encodes metabolism of chlorinated catechols: evidence for specialization of catechol 1,2-dioxygenases for chlorinated substrates. J. Bacteriol. **173**:2425–2434.

53. Vedler, E., V. Kõiv, and A. Heinaru. 2000. Analysis of the 2,4-dichlorophenoxyacetic acid-degradative plasmid pEST4011 of *Achromobacter xylosoxidans* subsp. *denitrificans* strain EST4002. Gene **255**:281–288.

54. Vedler, E., V. Kõiv, and A. Heinaru. 2000. TfdR, the LysR-type transcriptional activator, is responsible for the activation of the *tfdCB* operon of

**Pseudomonas putida** 2,4-dichlorophenoxyacetic acid degradative plasmid. Gene **245**:161–168.

55. Vollmer, M. D., U. Schell, V. Seibert, S. Lakner, and M. Schlömann. 1999. Substrate specificities of the chloromuconate cycloisomerases from *Pseudomonas* sp. B13, *Ralstonia eutropha* JMP134 and *Pseudomonas* sp. P51. Appl. Microbiol. Biotechnol. **51**:598–605.

56. Xia, X. S., S. Aathithan, K. Oswiecimska, A. R. Smith, and I. J. Bruce. 1998. A novel plasmid pIJB1 possessing a putative 2,4-dichlorophenoxyacetate degradative transposon Tn5530 in *Burkholderia cepacia* strain 2a. Plasmid **39**:154–159.

57. Xia, X. S., A. R. Smith, and I. J. Bruce. 1996. Identification and sequencing of a novel insertion sequence, IS1471, in *Burkholderia cepacia* strain 2a. FEMS Microbiol. Lett. **144**:203–206.

58. You, I.-S., and D. Ghosal. 1995. Genetic and molecular analysis of a regulatory region of the herbicide 2,4-dichlorophenoxyacetate catabolic plasmid pJP4. Mol. Microbiol. **16**:321–331.

59. Zhong, Z., D. Helinski, and A. Toukdarian. 2003. A specific region in the N terminus of a replication initiation protein of plasmid RK2 is required for recruitment of *Pseudomonas aeruginosa* DnaB helicase to the plasmid origin. J. Biol. Chem. **278**:45305–45310.
