
short communications

Acta Crystallographica Section D  
Biological  
Crystallography  
ISSN 0907-4449

**Structure of the phenazine biosynthesis enzyme  
PhzG**

James F. Parsons,$^{a}$ Kelly  
Calabrese,$^{a}$ Edward Eisenstein$^{a,b*}$  
and Jane E. Ladner$^{a}$  

$^{a}$Center for Advanced Research in  
Biotechnology, University of Maryland  
Biotechnology Institute, National Institute of  
Standards and Technology, 9600 Gudelsky  
Drive, Rockville, MD 20850, USA, and  
$^{b}$Department of Chemistry and Biochemistry,  
University of Maryland Baltimore County,  
Baltimore, MD 21228, USA  

Correspondence e-mail: edd@carb.nist.gov  

PhzG is a flavin-dependent oxidase that is believed to play a role in  
phenazine antibiotic synthesis in various bacteria, including *Pseudo-  
monas*. Phenazines are chorismic acid derivatives that provide the  
producing organisms, including the opportunistic pathogen  
*P. aeruginosa*, with a competitive growth advantage. Here, the crystal  
structures of PhzG from both *P. aeruginosa* and *P. fluorescens* solved  
in an unliganded state at 1.9 and 1.8 Å resolution, respectively, are  
described. Although the specific reaction in phenazine biosynthesis  
catalyzed by PhzG is unknown, the structural data indicates that  
PhzG is closely related to pyridoxine-5′-phosphate oxidase, the  
*Escherichia coli pdxH* gene product, which catalyzes the final step in  
pyridoxal-5′-phosphate (PLP) biosynthesis. A previous proposal  
suggested that the physiological substrate of PhzG to be 2,3-  
dihydro-3-hydroxyanthranilic acid (DHHA), a phenazine precursor  
produced by the sequential actions of the PhzE and PhzD enzymes on  
chorismate, and that two DHHA molecules dimerized in another  
enzyme-catalyzed reaction to yield phenazine-1-carboxylate. However, it was not possible to demonstrate any *in vitro* activity upon  
incubation of PhzG and DHHA. Interestingly, analysis of the *in vitro*  
activities of PhzG in combination with PhzF suggests that PhzF acts  
on DHHA and that PhzG then reacts with a non-aromatic tricyclic  
phenazine precursor to catalyze an oxidation/aromatization reaction  
that yields phenazine-1-carboxylate. It is proposed that *phzG* arose  
by duplication of *pdxH* and that the subtle differences seen between  
the structures of PhzG and PdxH correlate with the loss of the ability  
of PhzG to catalyze PLP formation. Sequence alignments and  
superimpositions of the active sites of PhzG and PdxH reveal that the  
residues that form a positively charged pocket around the phosphate  
of PLP in the PdxH–PLP complex are not conserved in PhzG,  
consistent with the inability of phosphorylated compounds to serve as  
substrates for PhzG.

---

Received 12 August 2004  
Accepted 9 September 2004  

PDB References: *P. aeruginosa* PhzG, 1t9m, r1t9msf;  
*P. fluorescens* PhzG, 1ty9, r1ty9sf.

---

### 1. Introduction

Several species of *Pseudomonas*, including the  
human pathogen *P. aeruginosa*, produce  
secondary metabolites known as phenazines  
(Fig. 1). Dozens of naturally occurring phena-  
zines have been described, all of which share  
the characteristic tricyclic heteroaromatic ring  
system shown in Fig. 1. Phenazines are redox-  
active compounds that participate in reactions  
yielding superoxide and peroxide ions and  
hydroxyl radicals. These toxic molecules are  
thought to control the growth of other micro-  
organisms in order to provide *Pseudomonas*  
with a competitive growth advantage and may  
enhance the ability of these pathogens to  
colonize human and other tissue (Laursen &  
Nielsen, 2004).

Chorismic acid is a phenazine precursor.  
Two operons in *P. aeruginosa* (Mavrodi et al.,  
2001), each containing seven genes, are  
involved in the biosynthesis of phenazine  
(*phzA–G*; Fig. 1). A similar, single operon has  
been described in *P. fluorescens* 2-79 (Mavrodi  
et al., 1998). Each of these operons encode all  
the genes required to produce phenazine-  
1-carboxylic acid (PCA) from chorismate  
(Mavrodi et al., 2001). PhzE is an anthranilate  
synthase homolog that catalyzes the  
conversion of chorismate to 2-amino-2-  
deoxyisochorismate (ADIC). Structural and  
biochemical evidence has shown that PhzD  
catalyzes the conversion of ADIC to trans-2,3-  
dihydro-3-hydroxyanthranilic acid (DHHA;  
Parsons et al., 2003). DHHA is then converted  
to PCA in several poorly characterized steps,  
probably involving the condensation of two  
DHHA-like molecules to form the phenazine-  
ring system (Fig. 1). McDonald and coworkers  
have proposed that PhzG catalyzes the oxida-  
tion of DHHA to the corresponding 3-oxo-  
species, which then dimerizes in a reaction

© 2004 International Union of Crystallography  
Printed in Denmark – all rights reserved

2110 DOI: 10.1107/S0907444904022474  
Acta Cryst. (2004). D60, 2110–2113

short communications

![image](https://i.imgur.com/1234567.png)

enriched in PhzG were pooled, concentrated to approximately 10 mg ml⁻¹ and dialyzed against 50 mM Tris–HCl, 1 mM DTT and 1 mM EDTA, 0.1 mM FMN pH 8.0. This material was then applied onto a 12 ml HQ20 anion-exchange column equilibrated with the same buffer and eluted with a 0–0.2 M NaCl gradient in the same buffer. Fractions containing PhzG that were judged to be pure were pooled, dialyzed against 20 mM Tris, 1 mM DTT, 1 mM EDTA, 0.1 mM FMN pH 8.0, concentrated to 25 mg ml⁻¹ and stored at 193 K. Yields were about 60 mg of pure PhzG per litre of culture. PF PhzG was cloned, expressed and purified in an identical manner.

Crystals of PhzG from both species were grown in hanging drops by vapor diffusion at room temperature. Solutions containing ~12 mg ml⁻¹ PhzG in the above buffer were used to obtain crystals for both species of PhzG by hanging-drop vapor diffusion at room temperature. Some crystallization trials included 10 mM pyridoxine, phenazine, DHHA or 3-hydroxyanthranilic acid. The protein solution was mixed with an equal volume of 10% polyethylene glycol 4000, 0.2 M ammonium sulfate and 4 μl drops were equilibrated against wells containing 10–20% polyethylene glycol 4000, 0.2 M ammonium sulfate. The pH was maintained at 6.5 with 0.1 M MOPS or bis-tris buffer. Bright yellow to orange crystals appeared in 3–5 d and were variable in size. Diffraction data were collected using a Rigaku Micro Max 007 rotating-anode generator and a Rigaku R-AXIS IV++ detector (Rigaku/MSC, The Woodlands, Texas, USA). The crystals were cooled to 100 K with a Rigaku Xtream 2000 cryo-cooler and 50% polyethylene glycol 4K or saturated lithium acetate was added as a cryoprotectant to the reservoir solution in a 1:1 ratio. Diffraction data were collected and processed with CrystalClear/d*Trek (Pflugrath, 1999). Statistics for the data collection and refinement are shown in Table 1.

PhzG has significant sequence homology with PdxH from *E. coli* (31% identity over 189 residues). Using the structure of *E. coli* PdxH (PDB code 1dnl; Safo et al., 2000) and the graphics program XtalView (McRee, 1999), the sequence of the model was changed to that of PA PhzG. The CNS cross-rotation and translation scripts for molecular replacement were used first with a monomer and then with the dimer. A molecular-replacement solution was obtained when the dimer was used as a probe. After initial refinement using the CNS package, the program REFMAC5 (Murshudov et al., 1997) was used for

---

Figure 1  
The phenazine-biosynthetic pathway in *Pseudomonas*. Chorismate is converted to phenazine-1-carboxylate by the enzymes encoded by the *phzA–G* operon. Biochemical and structural data have confirmed the roles of PhzE and PhzD in the conversion of chorismate to DHHA. Although the steps beyond the formation of DHHA are tentative and the exact structures of intermediates 1 and 2 have not been definitively proven, the scheme is consistent with currently available data (Parsons et al., 2004).

possibly catalyzed by PhzF (McDonald et al., 2001).

In order to further characterize the details of phenazine biosynthesis, we report here the crystal structure of PhzG from *P. aeruginosa* (PA PhzG) solved in complex with its cofactor FMN at 1.9 Å. Structural analysis suggests PhzG to be a flavin-dependent oxidase, closely related to pyridoxine-5′-phosphate oxidase (PdxH) from *Escherichia coli*. PhzG is unable to catalyze the oxidation of pyridoxine-5′-phosphate (PNP) or pyridoxamine-5′-phosphate (PMP), however, possibly owing to several differences in active-site residues that correspond to the residues in PdxH that interact with the phosphate groups of PNP or PMP. We have also solved the structure of PhzG from *P. fluorescens* (PF PhzG). Statistical data for both structures are presented in Table 1. Because PF PhzG is virtually identical to PA PhzG in structure and sequence (r.m.s.d. of 0.7 Å for all Cα atoms; 73% identity over 208 residues), we will focus most of the discussion on PA PhzG. The structure of PhzG is otherwise remarkably similar to the known PdxH structures and thus represents another example of how pseudomonads have been able to recruit existing enzymes and adopt them, with remarkably few changes, to function in secondary metabolic pathways (McDonald et al., 2001; Parsons et al., 2003).

Attempts to detect activity of PhzG alone towards DHHA have been unsuccessful. However, we have shown that PhzF acts readily on DHHA, catalyzing an allylic rearrangement to yield a stable 3-oxo species of DHHA (Parsons et al., 2004). This intermediate, in the absence of other Phz proteins, spontaneously dimerizes and oxidizes to form PCA. Although PhzA, PhzB and PhzG have no activity toward DHHA, they do accelerate the formation of

---

2. Experimental  

The DNA encoding PA PhzG was amplified from *P. aeruginosa* genomic DNA (ATCC) using synthetic primers compatible with the *phzG* (PA1905) sequence as described in the TIGR Comprehensive Microbial Resource database (http://www.tigr.org). Primers were designed to incorporate *NdeI* and *HindIII* restriction enzyme sites at the 5′ and 3′ ends of the gene to facilitate cloning into the expression vector pET21a (Novagen).

PA PhzG was expressed in *E. coli* strain BL21(DE3). Cells harboring the pET21a-PhzG plasmid were grown at 308 K in LB medium containing 100 mg l⁻¹ ampicillin. Protein production was induced by addition of IPTG to a final concentration of 1 mM. Growth was continued for 3 h. Culture extracts containing PhzG were dialyzed against 25 mM tricine, 1 mM DTT, 1 mM EDTA pH 8.2 before being applied onto a 56 ml HQ50 anion-exchange column (Perseptive BioSystems)¹ equilibrated with the same buffer and eluted with a gradient of 0–0.4 M NaCl in the same buffer. Fractions

---

¹ Certain commercial materials, instruments and equipment are identified in this manuscript in order to specify the experimental procedure as completely as possible. In no case does such identification imply a recommendation or endorsement by the National Institute of Standards and Technology, nor does it imply that the material, instrument or equipment identified is necessarily the best available for the purpose.

short communications

Table 1  
Data and refinement statistics.

| PA PhzG | PF PhzG |
| --- | --- |
| **Diffraction data** |  |  |
| Space group | $P2_12_12_1$ | $P2_12_12_1$ |
| Unit-cell parameters |  |  |
| $a (\mathring{A})$ | 63.59 | 57.28 |
| $b (\mathring{A})$ | 69.02 | 63.30 |
| $c (\mathring{A})$ | 89.08 | 132.76 |
| No. unique reflections | 31579 | 45573 |
| Resolution | 20.0–1.9 (1.97–1.90) | 42.81–1.79 (1.86–1.79) |
| $R_{\text{merge}}$ | 0.038 (0.159) | 0.065 (0.347) |
| Completeness (%) | 99.7 (99.9) | 98.7 (90.1) |
| Redundancy | 6.0 (6.0) | 3.7 (3.4) |
| Mean $I/\sigma(I)$ | 19.0 (5.3) | 9.6 (3.3) |
| **Refinement statistics** |  |  |
| $R$ | 0.184 (0.28) | 0.193 (0.26) |
| $R_{\text{free}}$ | 0.231 (0.38) | 0.246 (0.31) |
| Residues included, $A/B$ | 11–214/11–214 | 21–222/16–222 |
| Hetero groups included | 2 FMN, 4 sulfate ions, 2 acetate ions | 2 FMN, 4 sulfate ions |
| No. water molecules | 363 | 418 |
| R.m.s.d. bond length ($\mathring{A}$) | 0.025 | 0.021 |
| R.m.s.d. angle ($^\circ$) | 1.95 | 1.82 |
| Average $B (\mathring{A}^2)$ |  |  |
| Main chain | 22.9 | 26.3 |
| Side chain | 26.3 | 29.2 |
| Solvent | 34.4 | 37.2 |
| Ramachandran plot |  |  |
| Most favorable (%) | 90.1 | 91.2 |
| Additional allowed (%) | 9.9 | 8.2 |
| Generously allowed (%) | 0.0 | 0.6 |

refinement and cycles of refinement were alternated with viewing and rebuilding parts of the molecule in XtalView. The final refinement statistics are shown in Table 1. Models were validated using PROCHECK (Laskowski & MacArthur, 1993) and side-chain conformations were checked with REDUCE and PROBE (Word et al., 1999).

### 3. Result and discussion

#### 3.1. Overall structure

PA PhzG crystallized in space group $P2_12_12_1$ with two polypeptide chains in the asymmetric unit. In solution, PhzG is also a dimer. Laser light scattering indicates a molecular weight of 51600 Da, or approximately twice the monomer weight of the polypeptide chain (24268 Da). Although the crystals diffracted quite well, no density was seen for the first ten amino acids of PA PhzG. When DHHA was included in the crystallization solutions, density was detected above the FMN ring, suggesting that a low-occupancy ring-shaped molecule was in equilibrium with a number of water molecules. The shape of the density is inconclusive and only water molecules were modeled in this region.

Each monomer of PhzG is composed of six $\alpha$-helices and nine antiparallel $\beta$-strands arranged in two domains. The larger domain includes two helices and all nine $\beta$-strands. The smaller domain is entirely helical in structure and is composed of $\alpha 3$, $\alpha 4$, $\alpha 5$ and $\alpha 6$. The first seven $\beta$-strands form a barrel structure that, together with the last two $\beta$-strands, forms a continuous curved $\beta$-sheet. The secondary structure of the monomer and the dimer are shown in Fig. 2.

#### 3.2. Dimer interface

Considerable surface area is buried between the PhzG monomers (2800 $\mathring{A}^2$ per monomer), suggesting they are tightly associated. $\beta$-Strands 1, 2 and 5 from each monomer interact around a twofold axis to form a narrow barrel-like structure that traverses the dimer (Fig. 2). Additional contacts at the dimer interface are formed by the interaction of strands 8 and 9 from domain I of one monomer with helices 5 and 6 from domain II of the second monomer.

#### 3.3. Similarity to pyridoxine-5'-phosphate oxidase

Automated superposition of PhzG using the program DALI (Holm & Sander, 1995.) revealed significant similarity to known PdxH structures from *E. coli* and human (di Salvo et al., 2002, 2003). The overall folds and secondary-structural arrangements of PhzG and PNP oxidase are nearly identical. The r.m.s.d. calculated based on a superposition of all backbone atoms is 1.7 Å and all secondary-structural elements are conserved. PNP oxidase is believed, based on both structural and biochemical evidence, to operate via a hydride-transfer mechanism, rather than a carbanion or single-electron transfer mechanism. In the PNP oxidase–PLP complex, C4' is positioned optimally for hydride transfer to N5 of the flavin and results from mutagenesis experiments support the hydride-transfer pathway (di Salvo et al., 2003). It is tempting to speculate that the similarity of the PhzG active site to that of PNP oxidase (see below) underlies a common chemical mechanism for catalysis. Ongoing structure-function studies may shed light on this hypothesis.

#### 3.4. Active site

The electron density of the FMN molecules in the PhzG structure is very well defined and no disorder or partial occupancy is evident. The two FMN molecules are bound in cavities between $\beta$-strands 1, 2 and 4 and helix 2 from domain I, helix 5 and the following coil region from domain II and $\beta$-strands 5, 6 and 7–9 from the second monomer. The two FMN-binding cavities open toward the same side of the dimer and are ~20 Å apart.

The residues that fix the FMN cofactor in the active site in PhzG are remarkably similar to those seen for PdxH. Only Ala 80 (Tyr82 in PdxH) and Gln86 (Lys88 in PdxH) differ between the structures, whereas all other first-sphere residues are conserved. One face of the FMN ring is tightly packed against Ile66 and Val67 from $\beta 2$. Tyr102, Trp187 and Leu12 also make hydrophobic contacts with the flavin ring. The highly charged tail of the FMN cofactor interacts with several charged and polar residues, including Arg65, Gln86, Ser145, Glu185 and Arg197. The external face of the flavin ring is solvent-exposed and the protein appears to be in an ‘open’ conformation, as opposed to the ‘closed’ conformation seen in complexes of PdxH with PLP, where the N-terminal residues of PdxH form a cap over the access channel to the flavin.

The substrate-binding site in PdxH includes a positively charged pocket around the phosphate of the bound product, pyridoxal phosphate. The pocket is formed by residues Arg14, Lys72, Arg133, Arg197 and the neutral Tyr129. Of these, only Arg133 (Arg131 in PA PhzG) and Arg197 (Arg193 in PA PhzG) are conserved in PhzG. These differences may account for the inability of PhzG to bind or oxidize PdxH substrates despite the otherwise remarkable similarity of the two enzymes.

short communications

3.5. Role of PhzG

The structures of PA and PF PhzG, their genetic organization and their similarity to PdxH indicate that PhzG is a flavin-dependent oxidase that functions in the phenazine-biosynthetic pathway. However, the exact role of PhzG and the identity of its physiologic substrate remain unknown. Because PhzF oxidizes DHHA to a 3-oxo intermediate and the addition of PhzA, PhzB and PhzG enhance the rate of dimerization and PCA formation, a possible role for PhzG is that it catalyzes a four-electron oxidation of a non-aromatic tricyclic intermediate that leads to the final product of the enzymes encoded by the Phz operon, phenazine-1-carboxylate. Additional structural studies with analogs of this intermediate coupled with biochemical data for genetic variants of PhzG are in progress to refine this hypothesis.

We thank Dr Martin K. Safo and Dr Verne Schirch for examining PhzG for pyridoxine-5'-phosphate oxidase activity and for helpful discussions.

### References

Holm, L. & Sander, C. (1995). *Trends Biochem. Sci.* **20**, 478–480.

Laskowski, R. A. & MacArthur, M. W. (1993). *J. Appl. Cryst.* **26**, 283–291.

Laursen, J. B. & Nielsen, J. (2004). *Chem. Rev.* **104**, 1663–1686.

McDonald, M., Mavrodi, D. V., Thomashow, L. S. & Floss, H. G. (2001). *J. Am. Chem. Soc.* **123**, 9459–9460.

McRee, D. E. (1999). *Practical Protein Crystallography*. San Diego: Academic Press.

Mavrodi, D. V., Bonsall, R. F., Delaney, S. M., Soule, M. J., Phillips, G. & Thomashow, L. S. (2001). *J. Bacteriol.* **183**, 6454–6465.

Mavrodi, D. V., Ksenzenko, V. N., Bonsall, R. F., Cook, R. J., Boronin, A. M. & Thomashow, L. S. (1998). *J. Bacteriol.* **180**, 2541–2548.

Murshudov, D. N., Vagin, A. A. & Dodson, E. J. (1997). *Acta Cryst.* D**53**, 240–255.

Parsons, J. F., Calabrese, K., Eisenstein, E. & Ladner, J. E. (2003). *Biochemistry*, **42**, 5684–5693.

Parsons, J. F., Song, F. H., Parsons, L., Calabrese, K., Eisenstein, E. & Ladner, J. E. (2004). In the press.

Pflugrath, J. W. (1999). *Acta Cryst.* D**55**, 1718–1725.

Safo, M. K., Mathews, I., Musayev, F. N., di Salvo, M. L., Thiel, D. J., Abraham, D. J. & Schirch, V. (2000). *Structure Fold. Des.* **8**, 751–762.

Salvo, M. L. di, Ko, T.-P., Musayev, F. N., Raboni, S., Schirch, V. & Safo, M. K. (2002). *J. Mol. Biol.* **315**, 385–397.

Salvo, M. L. di, Safo, M. K., Musayev, F. N., Bossa, F. & Schirch, V. (2003). *Biochim. Biophys. Acta*, **1647**, 76–82.

Word, J. M., Lovell, S. C., Richardson, J. S. & Richardson, D. C. (1999). *J. Mol. Biol.* **285**, 1735–1747.
