
Molecular Microbiology (2004) 54(4), 921–934

doi:10.1111/j.1365-2958.2004.04338.x

# Quorum sensing and the LysR-type transcriptional activator ToxR regulate toxoflavin biosynthesis and transport in *Burkholderia glumae*

**Jinwoo Kim,**¹ **Jung-Gun Kim,**¹ **Yongsung Kang,**¹  
**Ji Youn Jang,**¹ **Geetanjali J. Jog,**² **Jae Yun Lim,**¹  
**Suhyun Kim,**¹ **Hiroaki Suga,**² **Tomohisa Nagamatsu**³  
**and Ingyu Hwang**¹*

¹ School of Agricultural Biotechnology, Seoul National University, Seoul 151-742, Korea.  
² Department of Chemistry, University at Buffalo, State University of New York, Buffalo, NY 14260-3000, USA.  
³ Faculty of Pharmaceutical Sciences, Okayama University, Tsushima, Okayama 700-8530, Japan.

## Summary

*Burkholderia glumae* BGR1 produces a broad-host range phytotoxin, called toxoflavin, which is a key pathogenicity factor in rice grain rot and wilt in many field crops. Our molecular and genetic analyses of toxoflavin-deficient mutants demonstrated that gene clusters for toxoflavin production consist of four transcriptional units. The toxoflavin biosynthesis genes were composed of five genes, *toxA* to *toxE*, as Suzuki *et al.* (2004) reported previously. Genes *toxF* to *toxl*, which are responsible for toxoflavin transport, were polycistronic and similar to the genes for resistance-nodulation-division (RND) efflux systems. Using Tn3-gusA reporter fusions, we found that ToxR, a LysR-type regulator, regulates both the *toxABCDE* and *toxFGHI* operons in the presence of toxoflavin as a coinducer. In addition, the expression of both operons required a transcriptional activator, ToxJ, whose expression is regulated by quorum sensing. TofI, a LuxI homologue, was responsible for the biosynthesis of both *N*-hexanoyl homoserine lactone and *N*-octanoyl homoserine lactone (C8-HSL). C8-HSL and its cognate receptor TofR, a LuxR homologue, activated *toxJ* expression. This is the first report that quorum sensing is involved in pathogenicity by the regulation of phytotoxin biosynthesis and its transport in plant pathogenic bacteria.

---

## Introduction

*Burkholderia glumae* causes rice grain rot and seedling rot in rice nursery boxes (Cottyn *et al.*, 1996; Suzuki *et al.*, 1998). Rice grain rot occurs at the flowering stage when temperature and moisture are high and causes yield losses of up to 34% (Yoshimura *et al.*, 1987; Tsushima *et al.*, 1996). The bacterium produces bright yellow pigments identified as toxoflavin {1,6-dimethylpyrimido[5,4-e]-1,2,4-triazine-5,7(1H,6H)-dione: molecular weight = 193} and fervenulin, which are essential in the pathogenicity of rice seedling rot and grain rot (Sato *et al.*, 1989; Iiyama *et al.*, 1995; Nagamatsu, 2001). We previously reported that *B. glumae* BGR1 causes bacterial wilt in many field crops and produces the toxoflavin responsible for major symptom development (Jeong *et al.*, 2003). Toxoflavin production is dependent on growth temperature, which is maximal at 37°C, and no detectable amount is produced at 25–28°C (Matsuda and Sato, 1988).

Plant pathogenic bacteria produce phytotoxins that are toxic to plant cells, which influences symptom development (Durbin, 1991). *Pseudomonas* species produce various phytotoxic compounds (Gross, 1991; Walton and Panaccione, 1993). Of these, the mode of action, biosynthesis and regulation of coronatine, syringomycin, tabtoxin and phaseolotoxin are relatively well characterized (Gross, 1991). In many cases, phytotoxins from *Pseudomonas syringae* are not required for pathogenicity, but function as virulence factors affecting disease severity (Gross, 1991; Walton and Panaccione, 1993). However, toxoflavin is a critical virulence factor in *B. glumae*, probably owing to its mode of action in plants. Toxoflavin is an active electron carrier between NADH and oxygen, and it can produce hydrogen peroxide and bypass the cytochrome system (Latuasan and Berends, 1961). This may explain why toxoflavin has antibacterial, antifungal and herbicidal activities and is toxic to mice, causing haematuria, diarrhoea and lachrymation (Nagamatsu, 2001).

The *tox* operon responsible for toxoflavin biosynthesis is polycistronic and consists of five genes (*toxA* to *E*) (Shingu and Yoneyama, 2004; Suzuki *et al.*, 2004). Shingu and Yoneyama (2004) suggested that the LysR-

Accepted 5 August, 2004. *For correspondence.* E-mail ingyu@snu.ac.kr; Tel. (+82) 2 880 4676; Fax (+82) 2 873 2317.

© 2004 Blackwell Publishing Ltd

type regulator ToxR regulates the expression of the toxABCDE operon, and they speculated that toxoflavin is synthesized in part via a biosynthetic pathway common to the synthesis of riboflavin, starting with GTP as the precursor (Suzuki et al., 2004). However, it is not clear whether ToxR regulates the toxoflavin biosynthesis genes. If ToxR is a LysR-type regulator, ToxR likely requires a coinducer. Currently, it is not known how B. glumae cells transport toxoflavin or protect themselves against it.

Quorum sensing is a mechanism by which bacteria regulate a set of genes in a cell-density dependent manner (Fuqua et al., 1996). The communication is mediated by diffusible signal molecules, N-acyl homoserine lactones (HSLs), that differ with respect to the length, saturation and substitutions of the side-chain (Miller and Bassler, 2001; Whitehead et al., 2001). Quorum sensing in Burkholderia cepacia is associated with siderophore ornibactin biosynthesis, extracellular protease, swarming and biofilm formation (Lewenza and Sokol, 2001; Aguilar et al., 2003).

In this study, we identified operons for toxoflavin biosynthesis and transport, and demonstrated that ToxR, a LysR-type activator, requires toxoflavin as a coinducer to activate the expression of both operons. Furthermore, we showed that the expression of the toxoflavin biosynthesis and transport genes is regulated by another transcriptional activator, ToxJ, whose expression is activated by quorum sensing. We propose a working model for the circuits by which B. glumae cells regulate toxoflavin biosynthesis and transport. This is the first report that quorum sensing is involved in phytotoxin production and hence pathogenicity in plant pathogenic bacteria.


Results

Organization of the toxoflavin biosynthesis and transport genes

To identify the DNA region responsible for toxoflavin biosynthesis, a genomic library of B. glumae BGR1 was screened by colony hybridization using probe DNA amplified from a known toxA gene. Three overlapping clones were obtained: pBGT6, pBGT7 and pBGT17. After mutagenizing the three clones with Tn3-gusA, each mutation was marker-exchanged into B. glumae BGR1. We identified 10 Tn3-gusA insertions that interfere with toxoflavin production, defining the essential region for toxoflavin production (Fig. 1).

DNA sequence analyses of the 28.4 kb DNA region revealed 11 potential open reading frames: toxA, B, C, D, E, F, G, H, I, J and R (Fig. 1). Each gene is preceded by a conserved ribosome binding site (data not shown). ToxA, B, C, D, E and R were almost identical to the previously reported Tox proteins of B. glumae (Suzuki et al., 2004).

Sequence analysis of the 9.9 kb EcoRI-HindIII fragment from pBGT17 suggested that this region encodes a transporter system for toxoflavin. ToxF was 19.3 kDa in size and showed weak similarity to the multidrug efflux transporter of the major facilitator superfamily (MFS) from Prochlorococcus marinus ssp. pastoris. ToxG (possibly 40.1 kDa in size) showed 40% identity and 58% positives to the resistance-nodulation-division (RND) efflux membrane fusion protein (MFP) precursor from Pseudomonas aeruginosa PAO1. The toxH gene encoded a 110 kDa protein with 54% identity and 68% positives to the RND efflux transporter from P. aeruginosa PAO1. ToxI was predicted to be 53.9 kDa and most similar to the OprM protein


![Diagram](attachment://image.png)

Fig. 1. Organization of toxABCDE, toxFGHI, and two regulatory genes, toxJ and toxR. Open arrows indicate the positions and orientations of the genes for toxoflavin transport and biosynthesis. Vertical bars in the maps indicate the positions and orientations of the Tn3-gusA insertions, and the major phenotypes of the mutants are represented below the restriction map. Vertical bars with closed circles indicate positions of Ω cassette insertions. B, BamHI; E, EcoRI; H, HindIII.

Regulation of toxoflavin production in Burkholderia glumae 923

growth, and growth-arrest of the mutants supported the postulate that the *toxFGHI* operon is not involved in the biosynthesis of the three toxins, but is important for transporting toxoflavin and reumycin. The *toxI::Tn3-gusA70* mutation did not affect the production of any toxin (Fig. 2). Tn3-*gusA* and Ω fragment insertions in *toxR* or *toxJ* abolished the production of all three toxins (Figs 1 and 2).

We used reverse transcription polymerase chain reaction (RT-PCR) to determine if the *toxFGHI* genes are polycistronic. As shown in Fig. 3, we used six sets of primers to amplify *toxF-G*, *G-H*, *H-I* and *I*. RT-PCR followed by Southern hybridization indicated that *toxFGHI* genes are transcribed as a single transcript (Fig. 3). Interestingly, an 805 bp DNA region between *toxJ* and *toxI* and another 553 bp downstream from *toxE* showed 47% and 65% identity to the putative transposase from *B. cepacia*, respectively (Fig. 1). The DNA sequences of the *toxA, B, C, D, E, F, G, H, I, J* and *R* regions of *B. glumae* BGR1 have been deposited in GenBank under Accession No. AY641455.

**Virulence of toxoflavin-deficient mutants**

The ability of mutants to induce bacterial grain rot in rice was determined by inoculating rice panicles with the wild type and toxin-deficient mutants during the flowering stage. When rice panicles were inoculated with the wild

![Diagram](https://i.imgur.com/yourimageurl.png)

Fig. 2. Toxoflavin production by *tox* mutants. The toxoflavin produced is shown as seen by the naked eyes and under UV light at 365 nm. The identity of the phytotoxins shown was confirmed by LC-MS of the purified toxins and comparison with a synthetic standard (*S*: F, ferveulin; *R*, reumycin; *T*, toxoflavin). Mutations in *toxABCD*, *toxJ*, and *toxR* resulted in no phytotoxin production. A *toxE* mutant produced reduced amounts of toxins. Mutations in *toxFGH* conferred as much ferveulin production as in the wild-type BGR1. A mutation in *toxI* did not affect phytotoxin production. The origin (*o*) and solvent front (*f*) on TLC are marked.

of *P. aeruginosa*. The *toxR* gene was 906 bp in size, and its protein showed high similarity to various LysR-type regulators. *ToxR* exhibited 33% identity and 49% positives to *TtgS* protein from *Pseudomonas putida*, which is involved in the transcriptional expression of the hydrocarbon efflux pump *TtgDEF*, and 32% identity and 47% positives to *MexT* protein from *P. aeruginosa* PA01, which is involved in regulating the *MexEF-OprN* multidrug efflux system (data not shown). *ToxJ* shares similarities with the putative helix-turn-helix DNA binding domain near the carboxy-terminus of various transcriptional regulators (data not shown). However, *ToxJ* did not have any invariant amino acids typical of the LuxR family.

Tn3-*gusA* insertions in the *toxABCD* genes abolished toxoflavin, reumycin and ferveulin production (Fig. 2). The *toxE* mutant BGK840 produced much less of all three toxins compared with the wild type (Fig. 2). Toxoflavin production by mutants carrying Tn3-*gusA* insertions in the *toxFGH* genes was not detected, and the mutants produced much less reumycin (Fig. 2). By contrast, the mutants produced as much ferveulin as did the wild type (Fig. 2). Mutations in *toxFGH* genes resulted in slower growth *in vitro* compared with that of the wild type (data not shown). The growth of the mutants was arrested by adding 10 μM toxoflavin, whereas the growth of the wild type was not influenced by the added toxin (data not shown). Along with the similarity between the *ToxFGHI* and RND efflux systems, mutational analyses, the slow-

© 2004 Blackwell Publishing Ltd, *Molecular Microbiology*, **54**, 921–934

type and toxA, E, F, I, R and J mutants, the toxin-deficient mutants did not show any visible symptoms (disease index 0.1), whereas the wild type caused severe symptoms (disease index 4.2) (Fig. 4). The toxE mutant BGK840, which produced small amounts of toxins, exhibited some symptoms (disease index 1.5) (Fig. 4). The toxF mutant BGK34 producing fervenulin and small amounts of reumycin induced slight bacterial grain rot (disease index 1.5) (Fig. 4). This indicated that bacterial grain rot in rice caused by B. glumae is highly dependent on the ability of the bacterium to produce toxin.

**Quorum signal-deficient mutants failed to produce phytotoxins in B. glumae BGR1**

We reasoned that toxoflavin production at the late exponential growth phase might be dependent on the cell density of the culture. This led us to test whether the wild-type strain BGR1 produces autoinducers (Al) using the biosensor strain Chromobacterium violaceum CV026. The bioassay indicated that the strain produces Al corresponding to the synthetic N-hexanoyl homoserine lactone (C6-HSL) and N-octanoyl HSL (C8-HSL), as determined on thin layer chromatography overlaid with the biosensor cells (Fig. 5A). To confirm their chemical structures, the purified compounds were subjected to ESI-MS analysis. The two compounds displayed molecular ion (M + H) peaks at m/z 200 and m/z 228, respectively, and a fragmentation ion at m/z 102, which is characteristic of the homoserine lactone ring. Moreover, their ESI-MS spectra were identical to that of the synthetic C6-HSL and C8-HSL standards, which showed that the two compounds are C6-HSL and C8-HSL, respectively (data not shown).

To clone the C6- and C8-HSL synthase gene, a cosmid genomic library of BGR1 was mobilized into P. fluorescens 1855.344, which does not produce any N-acyl HSL. After screening transconjugants on plates containing Agrobacterium indicator cells, one cosmid clone, pBGA18, which conferred Al production to P. fluorescens 1855.344, was isolated (Fig. 6). The plasmid contained a 25 kb insert, and the smallest subclone conferring C6- and C8-HSL production was isolated as a 3.5 kb EcoRI-Sacl fragment (Fig. 6). The DNA sequence analysis identified the tofl and tofR genes responsible for C6- and C8-HSL synthesis, respectively (Fig. 6). The tofl and tofR genes were separated by 796 bp and transcribed separately (Fig. 6). The tofl gene was 612 bp in size and encoded a 22.4 kDa protein homologous to members of the LuxI family (data not shown). The tofR gene was 720 bp in size and encoded a 26.6 kDa protein homologous to members of the LuxR family (data not shown). The DNA sequences of the cloned tofRI region of B. glumae BGR1 have been deposited in GenBank under Accession No. AY641454.

To determine if Al production affects toxin production by the bacterium, the tofR::Ω mutant BGS1 and the tofl::Ω mutant BGS2 were constructed (Table 1). Mutant BGS2 produced neither Al nor phytotoxins (Fig. 5A and B). When pBGA43 carrying the tofl gene was mobilized into BGS2, the transconjugants recovered production of both Al and phytotoxins (Fig. 5A and B). When various Al were added to the BGS2 culture at different concentrations, C8-HSL conferred phytotoxin production on the mutant at 100 nM, while a much higher concentration (1000 nM) of C10-HSL was required to achieve the same level of phytotoxin production (Fig. 5C). C4-, C6- and C12-

| BGR1 | BGK59 | BGK840 | BGK34 | BGK889 | BGK18 | BGK43 | Water |
|------|-------|--------|-------|--------|-------|-------|-------|
| Wild-type | toxA::Tn3-gusA59 | toxE::Tn3-gusA840 | toxF::Tn3-gusA34 | toxI::Tn3-gusA889 | toxJ::Tn3-gusA18 | toxR::Tn3-gusA43 |       |

![Image Description](image.png)

4.2 0.1 1.5 1.5 4.0 0.1 0.1 0

Disease index

Fig. 4. The pathogenicity of phytotoxin-deficient mutants. Panicles of ‘Milyang 23’ were inoculated with B. glumae strains, as described in Experimental procedures. The wild-type strain BGR1 produced severe symptoms, resulting in empty heads of grain. The panicles inoculated with toxA, toxJ and toxR mutants were nearly asymptomatic, while those inoculated with toxE or toxF mutants showed significantly reduced symptoms. The panicles inoculated with a toxI mutant developed severe symptoms of grain rot. The photograph was taken 7 days after inoculation. The disease index of the tested rice plants was described in Experimental procedures.

HSL failed to confer phytotoxin production on the BGS2 mutant, even at 1000 nM (Fig. 5C).

As the Al-deficient mutants failed to produce phytotoxins, rice panicles were inoculated with the mutants to determine their virulence. The disease caused by mutant BGS2 was much less severe than that produced by the wild type (data not shown). When BGS2 carried pBGA43 in trans, the disease was as severe as with the wild type (data not shown). These results indicated that quorum sensing regulates phytotoxin production and thereby the pathogenicity of *B. glumae*.

### TofR and C8-HSL complex activates *tofI* expression

To determine if TofR regulates *tofI* expression, a *tofI::Tn3-gusA294* fusion was cloned into pLAFR6 (resulting in pBG294I), and then introduced into the *tofR::Ω* mutant BGS1 and the *tofI::Ω* mutant BGS2. When the β-glucuronidase activity was measured in each strain, *tofI* expression was abolished in the absence of *tofR* or C8-HSL (Fig. 7A). C6-HSL did not induce *tofI* expression, even in the presence of *tofR*, indicating that *tofI* expression is positively regulated by TofR and C8-HSL complex (Fig. 7A). The presence of two lux box consensus sequences in the upstream region of *tofI* supported the postulate that TofR regulates *tofI* expression.

### Expression of *toxJ* is activated by TofR and C8-HSL complex

As two regulatory proteins, ToxJ and ToxR, and quorum sensing are involved in regulating toxin production in *B. glumae*, we postulated that TofR and C8-HSL complex regulates the expression of one of the two regulatory genes. A lux box-like sequence is present upstream from *toxJ*, but not *toxR*, suggesting that TofR regulates *toxJ* expression. The *toxJ::Tn3-gusA18* and *toxR::Tn3-gusA43* fusions were constructed in wild-type BGR1, *tofR::Ω* mutant BGS1, and *tofI::Ω* mutant BGS2 backgrounds to determine how quorum sensing influences *toxJ* and *toxR* expression. The expression of *toxJ* was increased fivefold in the presence of *tofR* compared with the expression in the absence of *tofR* (Fig. 7B). To confirm that *toxJ* expression is upregulated by quorum sensing, we added exog-

Fig. 5. A quorum sensing signal is required for phytotoxin production in *B. glumae* BGR1. A bioassay of HSLs using a *Chromobacterium violaceum* biosensor on TLC (A). The BGS2 mutant did not produce any HSL signal molecule as determined on TLC assays, but this was complemented by introducing the *tofI* clone, pBGA43. The identity of each HSL shown in (A) was confirmed by comparison with a synthetic standard (S1: C4, C4-HSL; C6, C6-HSL; C8, C8-HSL). Phytotoxin production was restored by introducing pBGA43 into BGS2 and by adding C8-HSL exogenously (B). The identity of each phytotoxin shown in (B) was confirmed by comparison with a synthetic standard (S2: F, fervenulin; R, reumycin; T, toxoflavin). The origin (o) and solvent front (f) on TLC are marked. Phytotoxin production by the BGS2 mutant in response to C4-, C6-, C8-, C10- and C12-HSL (C).

Fig. 6. Organization of the *tofR* and *tofI* genes. Open arrows indicate the positions and orientations of the genes for C6- and C8-HSL production. Vertical bars in the maps indicate the positions and orientations of the Tn3-gusA insertions, and the major phenotypes of the mutants are represented below the restriction map. Vertical bars with closed circles indicate the positions of Ω cassette insertions. B, BamHI; E, EcoRI; H, HindIII; K, KpnI; S, SacI; Sf, SfiI; V, EcoRV. Enzyme sites from the vector are shown in parentheses.

postulated that ToxR might regulate toxJ expression, ToxJ might regulate toxR expression, or both ToxR and ToxJ might be required to activate expression of the biosynthetic genes. To determine which one of these hypotheses is correct, Tn3-gusA fusions in toxJ and toxR were constructed in toxR::Ω mutant BGS4 and toxJ::Ω mutant BGS5 backgrounds, respectively. When the β-glucuronidase activity of each construct was measured, the expression of toxJ was not increased in the presence of toxR compared with the expression in the absence of toxR (data not shown). No significant variation in toxR expression was observed in the absence of toxJ (data not shown). Therefore, we concluded that both ToxR and ToxJ are required to activate expression of the biosynthesis and transporter genes. To assess how ToxR affects the expression of the two tox operons, Tn3-gusA fusions in the toxABCDE and toxFGHI operons were constructed in the wild-type strain BGR1 and in the toxR::Ω mutant BGS4. When the β-glucuronidase activity of each construct was measured, no expression of either operon was observed in the absence of toxR (Fig. 8A and Table 2). The expression of the two operons required both ToxR and 1 μM toxoflavin (Fig. 8A and Table 2). This indicated that ToxR requires toxoflavin as a coinducer to activate the expression of both operons. The presence of the T-N<sub>11</sub>-A sequence in the upstream region of the toxABCDE and toxFGHI operons supports the hypothesis that ToxR regulates the expression of both operons (data not shown).

Expression of the two tox operons is dependent on ToxJ

Given that the BGS5 (toxJ::Ω) mutant was defective in toxoflavin production, we examined whether ToxJ functions as an activator for the expression of the toxABCDE and toxFGHI operons. To address this question, Tn3-gusA fusions in the toxABCDE and toxFGHI operons were constructed in the wild-type strain BGR1 and in the toxJ::Ω mutant BGS5. The expression of the toxABCDE operon absolutely required ToxR and 1 μM toxoflavin, and the expression was activated with ToxJ compared with the expression level in the absence of toxJ (Fig. 8A and B). The expression of the toxFGHI operon required ToxR and 1 μM toxoflavin, while its expression was less affected by ToxJ than was the expression of the toxABCDE operon (Table 2). These results indicate that both tox operons require not only ToxR and toxoflavin, but also ToxJ.

Discussion

This study investigated how toxoflavin biosynthesis and transport genes are organized and how toxoflavin production is regulated in *B. glumae*. We found that the toxoflavin biosynthesis genes of *B. glumae* BGR1 are nearly identical to previously reported genes and confirmed that

© 2004 Blackwell Publishing Ltd, *Molecular Microbiology*, **54**, 921–934

Table 1. Bacterial strains and plasmids.

| Strain or plasmid | Characteristics | Source or reference |
|-------------------|-----------------|---------------------|
| Escherichia coli | F⁻ Φ80 dlacZΔM15 Δ(lacZYA-argF)U169 endA1 recA1 hsdR17 (r<sub>k</sub>⁻m<sub>k</sub><sup>+</sup>) deoR thi-1 supE44 λ⁻gyrA96 relA1 | Gibco BRL |
| DH5α |  |  |
| S17-1 | Tra⁺, recA, Sp<sup>R</sup> | Simon et al. (1983) |
| C2110 | polA, Nal<sup>R</sup> | Stachel et al. (1985) |
| HB101 | F⁻ mcrB mrr hsdS20(r<sub>B</sub>⁻m<sub>B</sub>⁻) recA13 leuB6 ara-14 proA2 lacY1 galK2 xyl-5 mtl-1 rpsL20(Sm<sup>R</sup>) supE44 λ⁻ | Gibco BRL |
| Agrobacterium tumefaciens NT1(pDCI41E33) | Autoinducer indicator strain | Cook et al. (1997) |
| Chromobacterium violaceum CV026 | Autoinducer indicator strain | McClean et al. (1997) |
| Burkholderia glumae |  |  |
| BGR1 | Wild type, Rif<sup>R</sup> | Jeong et al. (2003) |
| BGS1 | BGR1 *tofR*::Ω | This study |
| BGS2 | BGR1 *tofI*::Ω | This study |
| BGS4 | BGR1 *toxR*::Ω | This study |
| BGS5 | BGR1 *toxJ*::Ω | This study |
| BGK18/S1K18/S2K18/S4K18 | BGR1 *toxJ*::Tn3-gusA18/BGS1 *toxJ*::Tn3-gusA18/BGS2 *toxJ*::Tn3-gusA18/ BGS4 *toxJ*::Tn3-gusA18 | This study |
| BGK43/S1K43/S2K43/S5K43 | BGR1 *toxR*::Tn3-gusA43/BGS1 *toxR*::Tn3-gusA43/BGS2 *toxR*::Tn3-gusA43/ BGS5 *toxR*::Tn3-gusA43 | This study |
| BGK59/S4K59/S5K59 | BGR1 *toxA*::Tn3-gusA59/BGS4 *toxA*::Tn3-gusA59/BGS5 *toxA*::Tn3-gusA59 | This study |
| BGK536/S4K536/S5K536 | BGR1 *toxB*::Tn3-gusA536/BGS4 *toxB*::Tn3-gusA536/BGS5 *toxB*::Tn3-gusA536 | This study |
| BGK421/S4K421/S5K421 | BGR1 *toxC*::Tn3-gusA421/BGS4 *toxC*::Tn3-gusA421/BGS5 *toxC*::Tn3-gusA421 | This study |
| BGK465/S4K465/S5K465 | BGR1 *toxD*::Tn3-gusA465/BGS4 *toxD*::Tn3-gusA465/BGS5 *toxD*::Tn3-gusA465 | This study |
| BGK840/S4K840/S5K840 | BGR1 *toxE*::Tn3-gusA840/BGS4 *toxE*::Tn3-gusA840/BGS5 *toxE*::Tn3-gusA840 | This study |
| BGK34/S4K34/S5K34 | BGR1 *toxF*::Tn3-gusA34/BGS4 *toxF*::Tn3-gusA34/BGS5 *toxF*::Tn3-gusA34 | This study |
| BGK858/S4K858/S5K858 | BGR1 *toxG*::Tn3-gusA858/BGS4 *toxG*::Tn3-gusA858/BGS5 *toxG*::Tn3-gusA858 | This study |
| BGK20/S4K20/S5K20 | BGR1 *toxH*::Tn3-gusA20/BGS4 *toxH*::Tn3-gusA20/BGS5 *toxH*::Tn3-gusA20 | This study |
| BGK70/S4K70/S5K70 | BGR1 *toxI*::Tn3-gusA70/BGS4 *toxI*::Tn3-gusA70/BGS5 *toxI*::Tn3-gusA70 | This study |
| Plasmids |  |  |
| pBluescript II SK(+) | Cloning vehicle; phagemid, pUC derivative, Amp<sup>R</sup> | Stratagene |
| pLAFR3 | Tra⁻, Mob⁺, RK2 replicon, Tet<sup>R</sup> | Staskawicz et al. (1987) |
| pLAFR6 | As pLAFR3 but without *lacZα*, contains multilinker of pUC18 flanked by synthetic *trp* terminators, Tet<sup>R</sup> | Huynh et al. (1989) |
| pRK2013 | Helper plasmid; Tra⁺, ColE1 replicon, Km<sup>R</sup> | Figurski and Helinski (1979) |
| pHoKmGus | Promoterless β-glucuronidase gene, Km<sup>R</sup>, Amp<sup>R</sup>, | Bonas et al. (1989) |
| pSShe | Cm<sup>R</sup> | Stachel et al. (1985) |
| pHP45Ω | Ω cassette, Sp<sup>R</sup>, Sm<sup>R</sup> | Prentki and Krisch (1984) |
| pBGT6 | 25.2 kb DNA fragment from strain BGR1 cloned into pLAFR3 | This study |
| pBGT7 | 28.4 kb DNA fragment from strain BGR1 cloned into pLAFR3 | This study |
| pBGT17 | 28.5 kb DNA fragment from strain BGR1 cloned into pLAFR3 | This study |
| pBGT31 | 2.0 kb Ω cassette was inserted into *HincII* site within *toxR* sequence and cloned into pLAFR3 | This study |
| pBGT35 | 2.0 kb Ω cassette was inserted into *HindIII* site within *toxJ* sequence and cloned into pLAFR3 | This study |
| pBGT38 | 1.5 kb EcoRI-EcoRV fragment including *toxJ* from pBGT20 cloned into pLAFR3 | This study |
| pBGT40 | 1.1 kb NarI-SmaI fragment including *toxR* from pBGT20 cloned into pLAFR3 | This study |
| pBGA18 | 25.3 kb DNA fragment from strain BGR1 cloned into pLAFR3 | This study |
| pBGA37R | 7.9 kb BamHI-SacI fragment including *tofR/I* region from pBGA18 cloned into pBluescript II SK(+) | This study |
| pBGA38R | 7.9 kb BamHI-HindII fragment from pBGA37R cloned into pLAFR3 | This study |
| pBGA43 | 0.85 kb EcoRV-SfiI fragment including *tofI* from pBGA18 cloned into pLAFR3 | This study |
| pBGA47R | 2.0 kb Ω cassette was inserted into *BgII* site within *tofR* sequence cloned into pLAFR3 | This study |
| pBGA48 | 2.0 kb Ω cassette was inserted into *SnaBI* site within *tofI* sequence cloned into pLAFR3 | This study |
| pBGA54 | 1.3 kb EcoRI-KpnI fragment including *tofR* from pBGA18 cloned into pLAFR3 | This study |
| pBG46R | 10.7 kb KpnI-Xhol fragment from pBGA18::Tn3-gusA46 cloned into pLAFR6, *tofR*::Tn3-gusA46 fusion only | This study |
| pBG294I | 10.9 kb KpnI-Xhol fragment from pBGA18::Tn3-gusA294 cloned into pLAFR6, *tofI*::Tn3-gusA294 fusion only | This study |

Amp<sup>R</sup>, ampicillin resistance; Cm<sup>R</sup>, chloramphenicol resistance; Km<sup>R</sup>, kanamycin resistance; Nal<sup>R</sup>, nalidixic acid resistance; Rif<sup>R</sup>, rifampicin resistance; Sp<sup>R</sup>, spectinomycin resistance; Tet<sup>R</sup>, tetracycline resistance.

fervenulin backbone, which suggests that an additional biosynthesis gene is needed. Therefore, it is plausible that fervenulin uses a different transport system owing its chemical difference from toxoflavin.

Toxoflavin transporters appear to be RND superfamily transporters. RND-type multidrug transporters have broad substrate specificity and protect bacterial cells from the actions of antibiotics on both sides of the cytoplasmic membrane (Nikaido, 1996; Grkovic et al., 2002). The genetic organization of toxoflavin efflux systems is similar to those of previously reported RND-type transporters, but differs from the other systems in that there is an additional protein, ToxF. RND-type transporters are located in the inner membrane and function in concert with accessory proteins belonging to the MFP family and the outer membrane protein (OMP) family (Wandersman, 1992; Nikaido, 1996; Grkovic et al., 2002). ToxF appears to be an integral membrane protein; however, it is not clear how ToxF associates with typical RND-type three-component transporter systems. In *B. glumae*, it is very likely that *toxH* encodes an RND-type transporter protein, *toxG* encodes MFP, and *toxl* encodes OMP. In other Gram-negative bacteria, individual outer membrane channels are often unique for each transporter and are expressed in the same gene cluster, such as OprJ, OprM and OprN in *P. aeruginosa* or MtrE in *Neisseria gonorrhoeae* (Poole et al., 1996; Morita et al., 2001; Poole, 2001; Shafer et al., 2001). Interestingly, mutations in *toxl* did not affect toxoflavin transport, indicating that *B. glumae* may not need a functioning OMP channel to pump solute molecules directly into the medium, circumventing the outer membrane barrier. The slower growth of the *toxFGH* mutants in vitro versus the growth of the wild type is a good indication that toxoflavin biosynthesized inside the cells is toxic, resulting from the inability of the mutants to pump it out. Growth arrest caused by added toxoflavin to the *toxFGH* mutants is another indication that ToxFGH is important for pumping out the toxin, as reported in other (Grkovic et al., 2002).

Fig. 8. The expression of the *toxABCDE* operon requires ToxR and toxoflavin (A) and ToxJ (B). The *toxABCDE* operon was not expressed in the absence of *toxR* or toxoflavin, while the expression was increased when *toxR* and 1 μM of toxoflavin were provided (A). The *toxABCDE* operon was not expressed in the presence of *toxR* and 1 μM of toxoflavin without *toxJ* (B). All values are the means ± SDs of values from triplicate experiments.

*toxABCDE* is polycistronic, as reported previously (Suzuki et al., 2004). We identified the toxoflavin transport system and found that fervenulin is probably transported by another system. Fervenulin is the tautomeric isomer of toxoflavin and has a methyl group on the N8 portion of the

Table 2. Expression of *toxFGHI::Tn3-gusA* fusions in *toxR* and *toxJ* backgrounds.

| Fusions          | Wild-type strain | Specific activity of β-glucuronidase |
|------------------|-------------------|---------------------------------------|
| Coinducer        | –                | +                                     | *toxR::Ω* | –               | +               | *toxJ::Ω* | –               | +               |
| *toxF::Tn3-gusA34* | 1411 ± 34        | 1531 ± 45                             | 1.7 ± 0.1 | 1.9 ± 0.1       | 0.2 ± 0.1       | 150 ± 7   |
| *toxG::Tn3-gusA858* | 1139 ± 22        | 1604 ± 52                             | 1.6 ± 0.1 | 1.8 ± 0.1       | 0.2 ± 0.1       | 139 ± 9   |
| *toxH::Tn3-gusA20* | 1369 ± 24        | 1660 ± 70                             | 0.6 ± 0.1 | 0.7 ± 0.1       | 0.2 ± 0.1       | 126 ± 5   |
| *toxl::Tn3-gusA70* | 31 ± 2           | 86 ± 5                                | 0.2 ± 0.1 | 0.2 ± 0.1       | 0.2 ± 0.1       | 3.9 ± 0.2 |

Cultures were grown at 37°C in LB medium in the presence or absence of toxoflavin (1 μM) as an inducing compound.  
β-Glucuronidase activity was measured as described in Experimental procedures.  
One unit of β-glucuronidase was defined as 1 nmol of 4-methylumbelliferone released per bacterium per minute.  
Values are means ± SDs of three replicates.
It has been proposed that ToxR regulates the toxABCDE operon, because disruption of toxR abolished the ability to produce toxoflavin and resulted in loss of toxA mRNA expression (Shingu and Yoneyama, 2004). However, it has not been proved that ToxR regulates the toxABCDE operon genetically. ToxR is a LysR-type regulatory protein, and these proteins activate target genes in the presence of coinducers (Schell, 1993). We demonstrated that ToxR and the toxoflavin complex regulates the expression of both the toxABCDE and toxFGHI operons. It was thought that toxoflavin is a coinducer for ToxR. The fact that the toxJ mutant produces a very low level of toxoflavin led us to believe that basal levels of ToxR and toxoflavin are present in the system; however, it is not quite ready to start toxoflavin biosynthesis. The bacterium wants to ensure that toxoflavin production is regulated in a cell-density dependent manner via another regulator, ToxJ. It is very unlikely that other gene products are involved in toxoflavin modification by behaving as a coinducer. If exogenous toxoflavin underwent detoxification, we would not see the growth arrest of the toxFGH mutants.

The toxE mutant BGK840 probably produces small amounts of toxins as compared to the wild type, owing to the high similarity of toxE with the deaminase for riboflavin biosynthesis. This implies that toxoflavin is synthesized in part via a biosynthetic pathway common to that of riboflavin (Suzuki et al., 2004). The expression of toxE was very high in the BGK840, whereas that of toxABCD was very low without toxoflavin. This suggests that the BGK840 produces sufficient toxoflavin to act as a coinducer of ToxR.

As toxoflavin was produced when B. glumae cells reached the late exponential phase, it is very likely that quorum sensing regulates toxoflavin production. By cloning tofR and tofl, generating proper tofR and tofl mutants, and analysing the signals chemically, we clearly demonstrated that C8-HSL is a key signal triggering toxoflavin biosynthesis. TofR responded to C10-HSL, but did not respond to C6-HSL, suggesting its different biological roles in B. glumae. The presence of putative lux boxes in the tofl promoter region is consistent with our observation that TofR and C8-HSL complex is essential for activating tofl transcription. The minimum concentration (100 nM) of C8-HSL needed to confer toxoflavin production to the tofl mutant BGS2 was higher than that reported for other biological systems (Hwang et al., 1995). This indicates that B. glumae cells do not start producing toxoflavin until the cells reach high densities.

We showed that quorum sensing regulates toxoflavin production; however, it was not clear whether TofR and C8-HSL complex regulates the expression of the toxABC-DE and toxFGHI operons directly. If TofR and C8-HSL complex mediates the regulation of both operons indirectly, perhaps the complex regulates toxJ expression,

given that ToxJ is a transcriptional regulator. It is unlikely that ToxJ is a LuxR-type regulator because it does not require N-acyl HSL to be active (unpubl. data). The putative DNA-binding region at the C-terminus of ToxJ was similar to that of other transcriptional regulators, but there were none of the conserved amino acid residues for N-acyl HSL binding that are found in LuxR-type regulators (Stock et al., 1990). The presence of a possible binding box for TofR and C8-HSL complex was consistent with our finding that TofR and C8-HSL complex activates toxJ expression. Based on the finding that the expression of the two tox operons depends on ToxJ, we concluded that quorum sensing regulates toxJ expression and then ToxJ activates both operons, along with ToxR and toxoflavin complex.

Combining the genetic organization of the two operons and the circuits regulating toxoflavin biosynthesis and transport, we devised a working model for how B. glumae cells regulate toxoflavin production as the most important virulence factor (Fig. 9). The bacterium probably has multilevel regulation for toxoflavin biosynthesis and transport in order to save energy. Phytotoxin production consumes large amounts of energy, so that B. glumae cells must ensure that they reach certain cell densities before they start to produce it. The B. glumae cells use toxoflavin as a coinducer to amplify toxoflavin biosynthesis and transport with ToxR as a very efficient way of producing highly toxic phytotoxins. This might explain why bacterial rot of rice is very serious and dangerous if it occurs under optimum environmental conditions. Quorum sensing systems are critical in the pathogenicity of P. aeruginosa (Passador et al., 1993). In B. pseudomallei, the causative agent of melioidosis of humans and animals, a quorum sensing system plays a key role in virulence by modulating the production of MprA protease (Valade et al., 2004). To our knowledge, this is the first and only study to demonstrate that quorum sensing is involved in the pathogenicity of plant pathogenic bacteria by regulating the phytotoxin production indirectly. It is actually activated by the transcriptional activators ToxJ and ToxR and toxoflavin complex.

### Experimental procedures

#### Bacterial strains, plasmids, media and growth conditions

The bacterial strains and plasmids used in this study are listed in Table 1. All the strains were cultured in Luria–Bertani (LB) medium. The B. glumae strain BGR1 and Escherichia coli were cultured at 37°C. Antibiotics were used at the following concentrations: ampicillin, 100 μg ml⁻¹; chloramphenicol, 20 μg ml⁻¹; gentamicin, 50 μg ml⁻¹; kanamycin, 50 μg ml⁻¹; nalidixic acid, 20 μg ml⁻¹; rifampicin, 50 μg ml⁻¹; spectinomycin, 25 μg ml⁻¹; and tetracycline, 10 μg ml⁻¹. Two complementary HSL biosensors were used:

© 2004 Blackwell Publishing Ltd, *Molecular Microbiology*, **54**, 921–934
930 J. Kim et al.

![Diagram](attachment:diagram.png)

Fig. 9. The toxoflavin production and regulation circuits in *B. glumae* BGR1. The five toxoflavin biosynthesis genes (*toxABCDE*) and the four genes (*toxFGHI*) responsible for its transport are polycistronic. The LysR-type regulatory protein ToxR activates both operons and requires toxoflavin (double hexagon shape) as a coinducer. *B. glumae* BGR1 produces C8-HSL and its cognate receptor TofR. At a critical autoinducer concentration, the TofR protein (diamond shape) binds to C8-HSL (pentagon shape) and activates *toxJ* transcription. ToxJ in turn activates the transcription of the *toxABCDE* and *toxFGHI* operons in a cascade fashion. Open rectangles indicate *lux* box-like sequences, and closed rectangles indicate LysR-type regulatory protein binding sites. The oval represents a bacterial cell.

A. *tumefaciens* strain NT1 (pDCI41E33) and *C. violaceum* strain CV026. Agrobacterium and Chromobacterium strains were grown at 28°C. The Agrobacterium indicator strain was grown in AB minimal mannitol liquid medium (Chilton et al., 1974). 5-Bromo-4-chloro-3-indolyl-β-d-galactopyranoside (Xgal) was used at 40 μg ml⁻¹ when necessary.

### Recombinant DNA techniques

Chromosomal DNA from *B. glumae* strain BGR1 was isolated using the method of Sambrook et al. (1989). Small-scale plasmid DNA isolation from *E. coli* was performed using QIAprep® Spin (Qiagen) according to the supplier's instructions, or by the alkaline lysis method (Sambrook et al., 1989). Small-scale cosmid DNA was isolated from *E. coli* and *B. glumae* using the alkaline lysis method. Large-scale plasmid and cosmid DNA preparations were performed by alkaline lysis followed by cesium chloride density gradient centrifugation (Sambrook et al., 1989). The restriction enzyme digestions were performed as recommended by the suppliers (Takara). The gel electrophoresis was performed in agarose gels (0.7% [w/v]), and Southern transfers were performed on Hybond-N™ nylon membranes (Amersham Biosciences, Uppsala, Sweden) as described by the manufacturer. For colony hybridization, all the procedures used to prepare the probe DNA and for hybridization were as described by the manufacturer (Amersham Biosciences, Uppsala, Sweden). To construct a genomic library of strain BGR1, 20–30 kb DNA inserts were prepared by partial digestion of total genomic DNA with Sau3Al size fractionated in sucrose gradients, and ligated into the BamHI site of pLAFR3. The ligated DNA was packaged into bacteriophage λ, as described by the manufacturer (Promega), and then transfected into *E. coli* HB101. All the pLAFR3 derivatives were mobilized into *B. glumae* BGR1 strains using triparental mating (Figurski and Helinski, 1979).

To amplify the *toxA* DNA probe, PCR was performed using a PTC-150 MiniCycler (MJ research) under the following conditions: an initial 96°C for 2 min, followed by 30 cycles of 96°C for 1 min, 50°C for 1 min and 72°C for 1 min. The following PCR primers were used: TOXA1 (5′-ATGAGTACGACAGCTCGA-3′) and TOXA2 (5′-TCAAGGCTTG CAGACCAG-3′). All the other basic molecular techniques used were as described by Sambrook et al. (1989).

© 2004 Blackwell Publishing Ltd, *Molecular Microbiology*, 54, 921–934

Regulation of toxoflavin production in Burkholderia glumae 931

### Transposon mutagenesis

pBGT6, pBGT7 and pBGT17 carrying all the essential biosynthesis genes of toxoflavin and pBGA18 with AI activity were mutagenized with Tn3-gusA, as described by Bonas *et al.* (1989). The insertion site and orientation of Tn3-gusA in each mutant were mapped using restriction enzyme digestion analysis and direct sequencing of the plasmid using the primer Tn3gus (5′-CCGGTCATCTGAGACCATTAAAAGA-3′), which allows sequencing out of the Tn3-gusA. The mutagenized plasmids that carried Tn3-gusA insertions were introduced individually into the parent strain BGR1 by conjugation and marker-exchanged into strain BGR1, as described previously (Fellay *et al.*, 1989). All marker-exchanges were confirmed by Southern hybridization analysis.

### Construction of Ω cassette insertion mutants

The Ω cassette from pH45Ω was inserted into the unique BglII or SnaBI site within the *tofR* or *tofl* sequence in pBGA37R to give pBGA47R and pBGA48, respectively. Similarly, the Ω cassette was inserted into the unique HincII or HindIII site within the *toxR* or *toxJ* sequence to generate pBGT31 or pBGT35, respectively. These plasmids were introduced into *B.* glumae BGR1 to generate the *tofR::Ω*, *tofl::Ω*, *toxR::Ω* and *toxJ::Ω* mutant strains (BGS1, BGS2, BGS4 and BGS5, respectively) using marker-exchange mutagenesis. We then introduced Tn3-gusA fusions that had been constructed in the *toxA*, *toxB*, *toxC*, *toxD*, *toxE*, *toxF*, *toxG*, *toxH*, *toxl*, *toxJ* and *toxR* genes into BGS1, BGS2, BGS4, BGS5 and wild-type strain BGR1 using marker-exchange, after which we measured the β-glucuronidase activity.

### RT-PCR analysis

Strain BGR1 was grown in LB medium to the exponential growth phase (12 h after inoculation); total RNA was isolated as described (Aiba *et al.*, 1981); and the RNA samples were treated with RQ1 DNase (Promega) to remove any contaminating DNA. RT-PCR analysis was performed as follows. Two micrograms of total RNA from *B.* glumae BGR1 were reverse transcribed into cDNA using AMV reverse transcriptase XL, as described by the manufacturer (Takara). The reverse transcription reaction was performed at 50°C for 1 h, followed by 5 min at 75°C. PCR reactions were performed using a PTC-150 MiniCycler (MJ research) under the following conditions: an initial 96°C for 2 min, followed by 40 cycles of 96°C for 1 min, 50°C for 1 min and 72°C for 1 min. The following primer was used for RT reactions: RT1 (5′-CCGTCTTCA GCGTGTCCTCG-3′ from the *toxl* sequence). The following PCR primers were used: PCR1 (5′-CCGTCTTTCAGCGTG TCCTCG-3′ and 5′-TCGCCG CCTGACGATG-3′), PCR2 (5′-TCAGCGCGATCTCGATCAGC-3′ and 5′-CCTGGTGC CCCTGGTTTC-3′), PCR3 (5′-CGAGGTTCTGGCCGCTC AG-3′ and 5′-ACCTCTTTCATCCGCGCC-3′), PCR4 (5′- CTTGCCCTGGGTGGTGCTC-3′ and 5′-GGCGATATGGTC GCGGTGG-3′), PCR5 (5′-AGCTGC CCCACGTATTTCTGC- 3′ and 5′-CGGTGGTGTTCGGCGG-3′), PCR6 (5′-CCGTGC CACAGGAACAGG-3′ and 5′-GCCCTGGTGCTGACGCC- 3′). Southern hybridization and DNA sequencing were used to confirm all of the RT-PCR products (Fig. 3B). As a positive control, pBGT17 DNA was used as a PCR template. As a negative control, PCR reactions with the same primer sets were performed using RNA samples that had not been reverse transcribed.

### Autoinducer assays

Samples, in 1–4 μl volumes, were applied to C18 reversed-phase TLC plates (Merck), and chromatograms were developed with methanol/water (60:40, v/v). After development, the solvent was evaporated, and the dried plates were overlaid with soft agar containing bacterial indicators. A 1 ml overnight culture of the bacterial indicators was used to inoculate 50 ml of AB for *Agrobacterium* indicator strain and LB for *Chromobacterium* indicator strain. After the agar solidified, the overlaid plates were incubated at 28°C in a closed plastic container. For plate assays, the medium containing biosensor strains was poured into culture plates. When the agar had solidified, the strains to be tested for AI production were spotted directly onto the surface of the plate. The plates were incubated at 28°C overnight.

### β-Glucuronidase assay

The β-glucuronidase enzyme assay was performed as described previously, with some modifications (Jefferson *et al.*, 1987). All *B.* glumae BGR1 derivatives were grown in LB medium, centrifuged, resuspended in GUS extraction buffer, and lysed using sonication with a VCX-400 sonicator (Sonics and Materials). The extract was used in the β-glucuronidase enzyme assay with 4-methylumbelliferyl glucuronide as the substrate. The fluorescence was measured at 365 nm excitation and 460 nm emission in a TKO100 fluorometer (Hoefer Scientific Instruments). One unit of β-glucuronidase was defined as 1 nm of released 4-methylumbelliferone per bacterium per minute.

### DNA sequencing and data analysis

The DNA inserted in pBGT17 and pBGA18 was digested with the appropriate restriction enzymes and subcloned into pBluescript II SK(+) before sequencing. Universal and reverse primers were used for the primary reactions, and synthetic primers were then used to sequence both strands. The DNA sequence data were analysed using the programs BLAST at the National Center for Biotechnology Institute (Gish and States, 1993), MEGALIGN (DNASTAR), and GENETYX-WIN (Software Development).

### Synthesis and purification of N-acyl homoserine lactones and phytotoxins

The HSLs used in this study – N-butanoyl-L-homoserine lactone (C4-HSL), N-hexanoyl-L-homoserine lactone (C6-HSL), N-octanoyl-L-homoserine lactone (C8-HSL), N-decanoyl-L-homoserine lactone (C10-HSL), N-dodecanoyl-L-homoserine lactone (C12-HSL) – were synthesized as described by Zhang *et al.* (1993). Analytical TLC was performed on Merck silica gel 60 F254 precoated plates. ¹H NMR spectra were taken on Vraian Gemini 300, Inova 400, or Inova 500 MHz

© 2004 Blackwell Publishing Ltd, *Molecular Microbiology*, **54**, 921–934

932 J. Kim et al.

spectrometers. ${}^{1}$H NMR shifts were referenced to tetramethylsilane. Stock solutions at 10 mM in acetonitrile (HPLC grade) were diluted into the growth medium to give the stated concentrations.

The supernatant (4 l) from stationary-phase cultures of *B. glumae* BGR1 was extracted with ethyl acetate (1:1). The ethyl acetate was removed by rotary evaporation, and the residue was reconstituted in 0.5 ml of methanol. For a given sample, a portion of the concentrated extract was diluted in ethyl acetate, and 1 μl was spotted onto a reporter TLC plate along with a set of standards. The remainder of the concentrated sample was applied to a second TLC plate as a series of two to four spots along the baseline, and the two plates were chromatographed in the same chamber. Following the development of the reporter TLC plate with CV026 strain, the C18 matrix in the regions of the preparative plate corresponding to the compound to be analysed were scraped off, combined, and extracted three times with 2 ml volumes of methanol. The combined extracts were clarified by centrifugation and passed through a fine sintered glass filter. The filtrates were taken to dryness, and the residue was resolved in 0.5 ml of methanol and applied to a C18 reverse-phase column (Sep-Pak® Vac C18 Cartridges). The fractions were eluted with a linear gradient of methanol in water (20–100%). We collected 10 fractions, and assayed these fractions using the CV026 biosensor strain. The active fractions were analysed using electrospray ionization mass spectrometry (ESI-MS) (JMS-LC mate, JEOL).

The phytotoxins used (fervenulin, reumycin and toxoflavin) were synthesized as described previously (Yoneda *et al.*, 1971; Nagamatsu, 2001). Stock solutions at 10 mM in methanol (HPLC grade) were diluted into the growth medium to give the stated concentrations.

The phytotoxin productivity assay was performed as described previously, with some modifications (liyama *et al.*, 1995). Each strain was inoculated into 5 ml of LB broth and incubated at 37°C for 24 h with shaking. To determine the phytotoxin productivity of the strains, 5 ml of chloroform was added to the cell-free supernatants with centrifugation at 13 000× g to extract the phytotoxins. After evaporation, the chloroform residue was dissolved in 5 ml of aqueous 80% methanol, and the absorbance at 260 nm was measured. The concentration of phytotoxin produced by each strain was determined by comparison with the standard curve of the purified phytotoxin at different concentrations. The chloroform extract of LB was used as a blank control in the spectrophotometer to subtract any background absorbance. For analytical TLC, the chloroform extracts were concentrated *in vacuo* using a rotary evaporator at below 40°C. The residue was dissolved in 10 μl of methanol and then developed on a silica gel 60 TLC plate (Merck) with chloroform/methanol (95:5, v/v). The spots were detected under UV light (365 nm).

### Plant inoculation

Rice plants (*Oryza sativa* cv. Milyang 23) were grown in a greenhouse, inoculated at the flowering stage with a bacterial suspension ($10^8$–$10^9$ cfu ml${}^{-1}$) using an atomizer (Binks Wren airbrush; Binks, Glendale Heights, IL), and kept in a greenhouse. The disease in the rice plants was evaluated daily for 10 days, as described by liyama *et al.* (1995), using the following scale: 0 = healthy panicle, 1 = panicle 0–20% discoloured, 2 = panicle 20–40% discoloured, 3 = panicle 40–60% discoloured, 4 = panicle 60–80% discoloured, 5 = panicle 80–100% discoloured. Disease severity was determined using: Disease degree = Σ(number of samples per score × score)/the total number of panicles. Pathogenicity assays were repeated three times with three replications.

## Acknowledgements

This work was supported by Grant No. CG1412 from the Crop Functional Genomics Center of the 21st Century Frontier Research Program, which is funded by Ministry of Science and Technology of the Republic of Korea. J.K., J.-G.K., Y.K., J.Y.J. and S.K. are recipients of graduate fellowships from the Ministry of Education as part of the Brain Korea 21 Project.

## References

Aguilar, C., Friscina, A., Devescovi, G., Kojic, M., and Venturi, V. (2003) Identification of quorum-sensing-regulated genes of *Burkholderia cepacia*. *J Bacteriol* **185**: 6456–6462.

Aiba, H., Adhya, S., and de Crombrugghe, B. (1981) Evidence for two functional *gal* promoters in intact *Escherichia coli* cells. *J Biol Chem* **256**: 11905–11910.

Bonas, U., Stall, R.E., and Staskawicz, B.J. (1989) Genetic and structural characterization of the avirulence gene *avrBs3* from *Xanthomonas campestris* pv. *vesicatoria*. *Mol General Gene* **218**: 127–136.

Chilton, M.D., Currier, T.C., Farrand, S.K., Bendich, A.J., Gordon, M.P., and Nester, E.W. (1974) *Agrobacterium tumefaciens* DNA and PS8 bacteriophage DNA not detected in crown gall tumors. *Proc Natl Acad Sci USA* **71**: 3672–3676.

Cook, D.M., Li, P.-L., Ruchaud, F., Padden, S., and Farrand, S.K. (1997) Ti plasmid conjugation is independent of *vir*: reconstitution of the *tra* functions from pTiC58 as a binary system. *J Bacteriol* **179**: 1291–1297.

Cottyn, B., Cerez, M.T., Van Outryve, M.F., Barroga, F., Swings, J., and Mew, T.W. (1996) Bacterial disease of rice I. Pathogenic bacteria associated with sheath rot complex and grain discoloration of rice in the Philippines. *Plant Dis* **80**: 429–437.

Durbin, R.D. (1991) Bacterial phytoxins: mechanism of action. *Experientia* **47**: 776–783.

Fellay, R., Krisch, H.M., Prentki, P., and Frey, J. (1989) Omegon-Km: a transposable element designed for *in vivo* insertional mutagenesis and cloning of genes in gram-negative bacteria. *Gene* **76**: 215–226.

Figurski, D.H., and Helinski, D.R. (1979) Replication of an origin-containing derivative of plasmid RK2 dependent on a plasmid function provided *in trans*. *Proc Natl Acad Sci USA* **76**: 1648–1652.

Fuqua, C., Winans, S.C., and Greenberg, E.P. (1996) Census and consensus in bacterial ecosystems: the LuxR-LuxI family of quorum-sensing transcriptional regulators. *Annu Rev Microbiol* **50**: 727–751.

Gish, W., and States, D.J. (1993) Identification of protein coding regions by database similarity search. Nat Genet 3: 266–272.

Grkovic, S., Brown, M.H., and Skurray, R.A. (2002) Regulation of bacterial drug export systems. Microbiol Mol Biol Rev 66: 671–701.

Gross, D.C. (1991) Molecular and genetic analysis of toxin production by pathovars of Pseudomonas syringae. Annu Rev Phytopathol 29: 247–278.

Huynh, T.V., Dahlbeck, D., and Staskawicz, B.J. (1989) Bacterial blight of soybean: regulation of a pathogen gene determining host cultivar specificity. Science 245: 1374–1377.

Hwang, I., Cook, D.M., and Farrand, S.K. (1995) A new regulatory element modulates homoserine lactone-mediated autoinduction of Ti plasmid conjugal transfer. J Bacteriol 177: 449–458.

Iiyama, K., Furuya, N., Takanami, Y., and Matsuyama, N. (1995) A role of phytotoxin in virulence of Pseudomonas glumae. Ann Phytopathol Soc Jpn 61: 470–476.

Jefferson, R.A., Kavanagh, T.A., and Bevan, M.W. (1987) GUS fusions: β-glucuronidase as a sensitive and versatile gene fusion marker in higher plants. EMBO J 6: 3901–3907.

Jeong, Y., Kim, J., Kim, S., Kang, Y., Nagamatsu, T., and Hwang, I. (2003) Toxoflavin produced by Burkholderia glumae causing rice grain rot is responsible for inducing bacterial wilt in many field crops. Plant Dis 87: 890–895.

Latuasan, H.E., and Berends, W. (1961) On the origin of the toxicity of toxoflavin. Biochim Biophys Acta 52: 502–508.

Lewenza, S., and Sokol, P.A. (2001) Regulation of ornibactin biosynthesis and N-acyl-L-homoserine lactone production by CepR in Burkholderia cepacia. J Bacteriol 183: 2212–2218.

Matsuda, I., and Sato, Z. (1988) Regulation between pathogenicity and pigment productivity in the causal agent of bacterial grain rot of rice. Ann Phytopathol Soc Jpn 54: 378.

McClean, K.H., Winson, M.K., Fish, L., Taylor, A., Chhabra, S.R., Camara, M., et al. (1997) Quorum sensing and Chromobacterium violaceum: exploitation of violacein production and inhibition for the detection of N-acylhomoserine lactones. Microbiology 143: 3703–3711.

Miller, M.B., and Bassler, B.L. (2001) Quorum sensing in bacteria. Annu Rev Microbiol 55: 165–199.

Morita, Y., Komori, Y., Mima, T., Kuroda, T., Mizushima, T., and Tsuchiya, T. (2001) Construction of a series of mutants lacking all of the four major mex operons for multidrug efflux pumps or possessing each one of the operons from Pseudomonas aeruginosa PAO1: MexCD-OprJ is an inducible pump. FEMS Microbiol Lett 202: 139–143.

Nagamatsu, T. (2001) Syntheses, transformation, and biological activities of 7-azapteridine antibiotics: toxoflavin, fervenulin, reumycin and their analogs. Recent Res Devel Org Bioorg Chem 4: 97–121.

Nikaido, H. (1996) Multidrug efflux pumps of gram-negative bacteria. J Bacteriol 178: 5853–5859.

Passador, L., Cook, J.M., Gambello, M.J., Rust, L., and Iglewski, B.H. (1993) Expression of Pseudomonas aeruginosa virulence genes requires cell-to-cell communication. Science 260: 1127–1130.


Poole, K. (2001) Multidrug efflux pumps and antimicrobial resistance in Pseudomonas aeruginosa and related organisms. J Mol Microbiol Biotechnol 3: 255–264.

Poole, K., Tetro, K., Zhao, Q., Neshat, S., Heinrichs, D.E., and Bianco, N. (1996) Expression of the multidrug resistance operon mexA-mexB-oprM in Pseudomonas aeruginosa: mexR encodes a regulator of operon expression. Antimicrob Agents Chemother 40: 2021–2028.

Prentki, P.A., and Krisch, H.M. (1984) In vitro insertional mutagenesis with a selectable DNA fragment. Gene 29: 303–313.

Rust, L., Pesci, E.C., and Iglewski, B.H. (1996) Analysis of the Pseudomonas aeruginosa elastase (lasB) regulatory region. J Bacteriol 178: 1134–1140.

Sambrook, J., Fritsch, E.F., and Maniatis, T. (1989) Molecular Cloning: a Laboratory Manual, 2nd edn. Cold Spring Harbor, NY: Cold Spring Harbor Laboratory Press.

Sato, Z., Koiso, Y., Iwasaki, S., and Matsuda, I. (1989) Toxins produced by Pseudomonas glumae. Ann Phytopathol Soc Jpn 55: 356–359.

Schell, M.A. (1993) Molecular biology of the LysR family of transcriptional regulators. Annu Rev Microbiol 47: 597–626.

Shafer, W.M., Veal, W.L., Lee, E.H., Zarantonelli, L., Balthazar, J.T., and Rouquette, C. (2001) Genetic organization and regulation of antimicrobial efflux systems possessed by Neisseria gonorrhoeae and Neisseria meningitidis. J Mol Microbiol Biotechnol 3: 219–224.

Shingu, Y., and Yoneyama, K. (2004) Essential regulator gene toxR for toxoflavin biosynthesis of Burkholderia glumae. J General Plant Pathol 70: 108–114.

Simon, R., Priefer, U., and Pühler, A. (1983) A broad host range mobilization system for in vivo genetic engineering: transposon mutagenesis in gram-negative bacteria. Bio/Technol 1: 784–791.

Stock, J., Kolibachuk, D., and Greenberg, E.P. (1990) Critical regions of the Vibrio fischeri LuxR protein defined by mutational analysis. J Bacteriol 172: 3974–3979.

Stachel, S.E., An, G., Flores, C., and Nester, E.W. (1985) A Tn3 lacZ transposon for the random generation of β-galactosidase gene fusions: application to the analysis of gene expression in Agrobacterium. EMBO J 4: 891–898.

Staskawicz, B., Dahlbeck, D., Keen, N., and Napoli, C. (1987) Molecular characterization of cloned avirulence genes from race 0 and race 1 of Pseudomonas syringae pv. glycinea. J Bacteriol 169: 5789–5794.

Suzuki, F., Sawada, H., Azegami, K., and Tsuchiya, K. (2004) Molecular characterization of the tox operon involved in toxoflavin biosynthesis of Burkholderia glumae. J General Plant Pathol 70: 97–107.

Suzuki, F., Zhu, Y., Sawada, H., and Matsuda, I. (1998) Identification of proteins involved in toxin production by Pseudomonas glumae. Ann Phytopathol Soc Jpn 64: 75–79.

Tsushima, S., Naito, H., and Koitabashi, M. (1996) Population dynamics of Pseudomonas glumae, the causal agent of bacterial grain of rice, on leaf sheaths of rice plants in relation to disease development in the field. Ann Phytopathol Soc Jpn 62: 108–113.

Valade, E., Thibault, F.M., Gauthier, Y.P., Palencia, M., Popoff, M.Y., and Vidal, D.R. (2004) The PmlI-PmlR quorum-sensing system in Burkholderia pseudomallei

934 J. Kim et al.

plays a key role in virulence and modulates production of  
the MprA protease. *J Bacteriol* 186: 2288–2294.  
Walton, J.D., and Panaccione, D.G. (1993) Host-selective  
toxins and disease specificity; perspectives and progress.  
*Annu Rev Phytopathol* 31: 275–303.  
Wandersman, C. (1992) Secretion across the bacterial outer  
membrane. *Trends Genet* 8: 317–322.  
Whitehead, N.A., Barnard, A.M.L., Slater, H., Simpson,  
N.J.L., and Salmond, G.P.C. (2001) Quorum-sensing in  
gram-negative bacteria. *FEMS Microbiol Rev* 25: 365–404.  

Yoneda, F., Shinomura, K., and Nishigaki, S. (1971) A con-  
venient synthesis of toxoflavins and toxoflavin-n-oxides.  
*Tetrahedron Lett* 13: 851–854.  
Yoshimura, D., Yoshida, K., and Otofuji, M. (1987)  
Effect of meteorological factors on the occurrence of  
rice bacterial grain rot. *Proc Assoc Plant Prot Kyushu*  
33: 9–12.  
Zhang, L.H., Murphy, P.J., Kerr, A., and Tate, M.E. (1993)  
*Agrobacterium* conjugation and gene regulation by  
N-acyl-L-homoserine lactones. *Nature* 362: 446–448.
