
Microbiology (2004), 150, 3527–3546

**DOI 10.1099/mic.0.27303-0**

---

**Review**

**Nitrate reduction and the nitrogen cycle in archaea**

Purificación Cabello,$^{1}$ M. Dolores Roldán$^{2}$ and Conrado Moreno-Vivián$^{2}$

$^{1}$Departamento de Biología Vegetal, Área de Fisiología Vegetal, Universidad de Córdoba, Spain  
$^{2}$Departamento de Bioquímica y Biología Molecular, Edificio Severo Ochoa, 1ª planta, Campus Universitario de Rabanales, Universidad de Córdoba, 14071-Córdoba, Spain  

Correspondence:  
Conrado Moreno-Vivián  
bb1movic@uco.es  

The nitrogen cycle (N-cycle) in the biosphere, mainly driven by prokaryotes, involves different reductive or oxidative reactions used either for assimilatory purposes or in respiratory processes for energy conservation. As the N-cycle has important agricultural and environmental implications, bacterial nitrogen metabolism has become a major research topic in recent years. Archaea are able to perform different reductive pathways of the N-cycle, including both assimilatory processes, such as nitrate assimilation and N₂ fixation, and dissimilatory reactions, such as nitrate respiration and denitrification. However, nitrogen metabolism is much less known in archaea than in bacteria. The availability of the complete genome sequences of several members of the eury- and crenarchaeota has enabled new approaches to the understanding of archael physiology and biochemistry, including metabolic reactions involving nitrogen compounds. Comparative studies reveal that significant differences exist in the structure and regulation of some enzymes involved in nitrogen metabolism in archaea, giving rise to important conclusions and new perspectives regarding the evolution, function and physiological relevance of the different N-cycle processes. This review discusses the advances that have been made in understanding nitrate reduction and other aspects of the inorganic nitrogen metabolism in archaea.

---

**Introduction**

As nitrogen is an essential element for living organisms, the availability of a suitable nitrogen source often limits primary productivity in both natural environments and agriculture. Nitrogen can be found in several oxidation states, from +5 in the most oxidized compound (nitrate) to −3 in the most reduced form (ammonium), but in biological compounds it is almost exclusively present in the fully reduced state. The global biogeochemical nitrogen cycle (N-cycle) allows the interconversions of nitrogen compounds to maintain a relatively small amount of fixed or combined nitrogen in a continuous exchange with atmospheric dinitrogen (N₂), a huge nitrogen reservoir. The N-cycle includes both reductive and oxidative processes, in which prokaryotes play a predominant role (Fig. 1). Assimilatory pathways, such as N₂ fixation and nitrate assimilation, generate ammonium that is further incorporated into cell material, mainly by the glutamine synthetase-glutamate synthase (GS-GOGAT) route. However, whereas N₂ fixation is only driven by free-living or symbiotic diazotrophic prokaryotes, assimilatory nitrate reduction is carried out by bacteria, fungi, algae and higher plants, and almost all living organisms are able to incorporate ammonium into carbon skeletons. On the other hand, some prokaryotes can obtain metabolic energy by redox processes involving nitrogen compounds, such as nitrification and denitrification. Autotrophic nitrification is a two-step process consisting of the oxidative conversion of ammonia to nitrite via hydroxylamine, carried out by ammonia-oxidizing bacteria, and the further oxidation of nitrite to nitrate, performed by nitrite-oxidizing chemolithoautotrophic bacteria. Denitrification is a respiratory process whereby nitrate is successively reduced to nitrite, NO, N₂O and N₂, which takes place predominantly under anaerobic conditions in many facultative bacteria (Berks *et al.*, 1995; Zumft, 1997; Moreno-Vivián & Ferguson, 1998; Moreno-Vivián *et al.*, 1999; Richardson & Watmough, 1999). Some heterotrophic bacteria are able to combine nitrification and aerobic denitrification by oxidizing ammonia to nitrite, which is later reduced to N₂ (de Boer & Kowalchuk, 2001). Ammonification is the dissimilatory reduction of nitrate to ammonia that does not serve the purpose of nitrogen autotrophy, although this term normally describes the release of ammonia from organic nitrogen compounds (Zumft, 1997). Finally, anaerobic ammonium oxidation (anammox) is a reaction that produces N₂ by reducing nitrite and oxidizing ammonium. This process, performed by bacterial communities of planctomycetes, seems to be of ecological importance in marine environments (Dalsgaard *et al.*, 2003; Kuypers *et al.*, 2003).

Bacteria are involved in all N-cycle pathways, and the different aspects of their inorganic nitrogen metabolism have been extensively studied both biochemically and genetically (for specific reviews see Berks *et al.*, 1995; Zumft, 1997; Ferguson, 1998; Moreno-Vivián *et al.*, 1999; Richardson & Watmough, 1999; Eisenberg *et al.*, 2000; Halbleib & Ludden,

P. Cabello, M. D. Roldán and C. Moreno-Vivián

![Diagram of the biological N-cycle](#)

Fig. 1. The biological N-cycle. The different nitrogen compounds are arranged according to their oxidation states and the main reductive or oxidative pathways are indicated by arrows. Conversion of hydroxylamine to nitrous oxide by heterotrophic nitrification, and production of N₂ from ammonium and nitrite by anaerobic ammonium oxidation (anammox), are not indicated. The enzymes involved in the different pathways are: 1–2, glutamine synthetase-glutamate synthase (GS-GOGAT); 3, glutamate dehydrogenase (GDH); 4, nitrogenase; 5, assimilatory nitrate reductase (Nas); 6, assimilatory nitrite reductase (sirohaem-Nir); 7, dissimilatory and respiratory nitrate reductases (Nap and Nar); 8, respiratory nitrite reductases (Cu-Nir and cd₁-Nir); 9, nitric oxide reductase (Nor); 10, nitrous oxide reductase (Nos); 11, ammonia monooxygenase; 12, hydroxylamine oxidase; 13, nitrite oxidase.

---

2000; Arcondéguy *et al.*, 2001; Kowalchuk & Stephen, 2001; Richardson *et al.*, 2001). Archaea form a monophyletic domain of organisms distinct from bacteria and eukarya, with two major subdivisions: crenarchaeota, which are predominantly hyperthermophiles and anaerobic respirers, and euryarchaeota, which include methanogens and extreme halophiles (Woese *et al.*, 1990; Brown & Doolittle, 1997). In general, archaeal replication, transcription and translation are more related to eukaryal than to bacterial processes (Bell & Jackson, 2001; Kelman & Kelman, 2003; Reeve, 2003), but archaeal central metabolism components are more closely related to bacterial proteins. In addition to ammonium assimilation, members of archaea can drive all the reductive pathways of the N-cycle, either dissimilatory reactions, such as nitrate respiration and denitrification, or assimilatory pathways like N₂ fixation and nitrate assimilation. Because archaea are the predominant microbial populations in extreme environments, such as highly saline water and hot springs, they sustain the N-cycle in these special and severe ecosystems. In spite of this, to date, there is little information on inorganic nitrogen metabolism in archaea, especially on nitrate reduction. Although the archaeal genome sequencing projects are bringing out new data, the understanding of inorganic nitrogen metabolism in archaea is still an open research field. This review covers the current knowledge on the nitrate reduction processes and other pathways of the N-cycle in archaea.

### Assimilatory and respiratory nitrate reduction in archaea

Many archaea are able to reduce nitrate by assimilatory or respiratory pathways, dissimilatory nitrate reduction being much more frequent than nitrate assimilation (Martínez-Espinosa *et al.*, 2001b). In addition, genes encoding putative nitrate transporters, nitrate reductases and nitrite

| Archaea | Nitrate transporter | Nitrate reductase | Nitrite reductase | Nitric oxide reductase | Nitrous oxide reductase |
| --- | --- | --- | --- | --- | --- |
| Crenarchaeota |  |  |  |  |  |
| Aeropyrum pernix | ND | Nar (genetic) APE 1288 (NarG), APE 1294 (NarH), APE 1300 (NarJ) | Sirohaem-Nir (genetic) APE 1326 (NirA) | ND | ND |
| Sulfolobus solfataricus | ND | Nar? (genetic) SSO 1579 (NarH?), SSO 1580 (NarG?) | ND | qNor? (genetic) SSO 1573 (N-term.), SS 01571 (C-term.) | ND |
| Sulfolobus tokadaii | ND | ND | ND | ND | Nos? (genetic) ST 2395 (Cuquinol oxidase), ST 0105 (Cycyt c oxidase) |
| Pyrobaculum aerophilum | ABC (genetic) PAE 3619 (NrtC) | Nar (genetic and functional) PAE 3610 (NarJ), PAE 3611 (NarG), PAE 3612 (NarH), PAE 3613, PAE 2614 (NarI) | Sirohaem-Nir (genetic) PAE 2577 (NirA) cd₁-Nir (genetic and functional) (NirS) | qNor (genetic and functional) PAE 3603 (NorZ) | Nos (functional) |
| Euryarchaeota |  |  |  |  |  |
| Archaeoglobus fulgidus | ABC (genetic) AF 0638 (NrtC), AF 0639 (NrtB), AF 0640 (NrtA), AF 0086 (NrtB), AF 0087 (NrtC), AF 0088 (NrtA) | Nar (genetic) AF 0173 (NarJ), AF 0174, AF 0175 (NarH), AF 0176 (NarG), AF 0545 (NarV), AF 546 (NarI), AF 0501 (NarI) | Sirohaem-Nir (genetic) AF 0164 (NirA) | ND | ND |
| Haloarcula marismortui | ND | Nar (genetic and functional) CAD 22069 (NarG), CAD 22070 (NarH), CAD 22073 (NarJ) | Cu-Nir (genetic and functional) CAB 93142 (NirK) | Nor (functional) | ND |
| Halobacterium sp. | ND | ND | ND | ND | Nos? (genetic) VNG2377G (NosY) |
| Haloferax denitrificans | ND | Nar (genetic and functional) CAD 82168 (NarH) | Cu-Nir (genetic and functional) CAD 89521 (NirK) | Nor (functional) | Nos (functional) |
| Haloferax mediterranei | ND | Ferredoxin-Nas (functional) Nar (functional) | Sirohaem-NirA (functional) Cu-Nir or cd₁- Nir (functional) | ND | ND |
| Haloferax volcanii | ABC (genetic and functional) AJ238877, AJ238878, AJ238879 | Nar (functional) | ND | ND | ND |

Table 1. cont.

| Archaea                          | Nitrate transporter | Nitrate reductase                     | Nitrite reductase                  | Nitric oxide reductase | Nitrous oxide reductase |
|----------------------------------|--------------------|--------------------------------------|-----------------------------------|------------------------|-------------------------|
| Methanothermobacter thermautotrophicus | ND               | Nas (genetic)                        | ND                              | ND                     | ND                      |
|                                  |                    | MTH 1567 (NasA),                      |                                 |                         |                         |
|                                  |                    | MTH 1547                             |                                 |                         |                         |
| Methanocaldococcus jannaschii    | ABC (genetic)      | ND                                   | Sirohaem-Nir (genetic)           | ND                     | ND                      |
|                                  | MJ 0412 (NrtC),     |                                      |                                 |                         |                         |
|                                  | MJ 0413 (NrtB)      |                                      | MJ 0551 (NirA)                   |                         |                         |
| Methanopyrus kandleri            | ABC (genetic)      | ND                                   | Sirohaem-Nir (genetic)           | ND                     | ND                      |
|                                  | MK 0601 (NrtA),     |                                      |                                 |                         |                         |
|                                  | MK 0602 (NrtB),     |                                      | MK 0801 (NirA)                   |                         |                         |
|                                  | MK 0603 (NrtC)      |                                      |                                 |                         |                         |
| Methanosarcina acetivorans       | ND                 | ND                                   | ND                              | ND                     | ND                      |
| Methanosarcina mazei             | ND                 | ND                                   | ND                              | Nor (genetic)          | ND                      |
|                                  |                    |                                      |                                 | MM2988                  |                         |
| Pyrococcus abyssi                | ND                 | ND                                   | ND                              | ND                     | ND                      |
| Pyrococcus furiosus              | ABC (genetic)      | ND                                   | Sirohaem-Nir? (genetic)          | ND                     | ND                      |
|                                  | AA 113496 (NrtC)    |                                      | PF 1197 (NirA?)                  |                         |                         |
| Pyrococcus horikoshii            | ND                 | ND                                   | Sirohaem-Nir (genetic)           | ND                     | ND                      |
|                                  |                    |                                      | PH 0827 (NirA)                   |                         |                         |
| Thermoplasma acidophilum         | ABC (genetic)      | Nas? (genetic)                       | ND                              | ND                     | ND                      |
|                                  | Ta1177 (NrtB),      | Ta0136 (Nia)                         |                                 |                         |                         |
|                                  | Ta1178 (NrtC)       |                                      |                                 |                         |                         |
| Thermoplasma volcanium           | ABC (genetic)      | Nas? (genetic)                       | ND                              | ND                     | ND                      |
|                                  | TVN 0387 (NrtB)     | TVN 0215 (Nia)                       |                                 |                         |                         |

Mo-ligand for the Mo-bisMGD cofactor, and also includes an unusual iron-sulfur cluster, previously undetected (Bertero et al., 2003; Jormakka et al., 2004). Finally, the periplasmic Nap systems of many Gram-negative bacteria participate in different processes, like redox balancing, scavenging nitrate in nitrate-limited environments, and aerobic or anaerobic denitrification (Potter et al., 2001; Gavira et al., 2002; Ellington et al., 2003). Usually, Nap enzymes are heterodimers with a catalytic subunit (NapA) and a cytochrome c (NapB), which receives electrons from NapC, a membrane cytochrome c (Berks et al., 1995; Reyes et al., 1996, 1998). However, the enzyme of Desulfovibrio desulfuricans is a single NapA subunit, for which the crystal structure has been solved (Dias et al., 1999).

Preliminary sequence comparisons of Nar, Nas and Nap indicated that these enzymes represent different phylogenetic groups (Reyes et al., 1996). A recent study of the sequences of eukaryotic and prokaryotic nitrate reductases revealed the existence of three different clades, corresponding to the eukaryotic nitrate reductase, the prokaryotic respiratory Nar, and the prokaryotic Nas/Nap enzymes, with the Nap from Desulfovibrio desulfuricans providing the evolutionary link between Nas and Nap subclades (Stolz & Basu, 2002). It is worth noting from this study that the deepest branch of the Nar clade is the euryarchaeote Haloarcula marismortui, and that the Nar enzymes

of the thermophilic bacterium Thermus thermophilus and the crenarchaeote Aeropyrum pernix are closely related. Phylogenetic trees based on NarH sequences also correlate well with the 16S rDNA tree, making the archaea A. pernix and Pyrobaculum aerophylum the outgroup and closely joining these archaeal sequences to the T. thermophilus bacterial subtree (Petri & Imhoff, 2000; Philippot, 2002). However, the NarG-based phylogeny is not entirely consistent with the 16S rDNA-based taxonomy (Cheneby et al., 2003; Gregory et al., 2003). As the nar genes were present before the phylogenetic divergence of bacteria and archaea, it can be assumed that respiratory nitrate reductase played a key role in energy metabolism during pre-oxic times (Petri & Imhoff, 2000). In addition, there is evidence that Nap and Nar have probably been acquired in some bacteria by horizontal gene transfer (Stolz & Basu, 2002). This seems to be the case for the nar gene cluster of T. thermophilus (Ramirez-Arcos et al., 1998b) and for the plasmid-encoded nap genes of Rhodobacter sphaeroides (Castillo et al., 1996). Results of NarG-based trees are also consistent with the possibility of lateral gene transfer (Gregory et al., 2003). Evidence of recent lateral gene transfer among hyperthermophilic archaea has been published (Di Ruggiero et al., 2000), but the relative importance of this mechanism of genetic exchange among archaea, and also from and to bacteria, should be determined in future studies.
Nitrate reductase cofactors

Genome sequences of several archaea contain MGD cofactor biosynthesis genes, and archaeal nitrate reductases seem to be molybdoenzymes. However, tungsten and molybdenum have similar chemistry and it is known that hyperthermophilic archaea possess W-containing enzymes, such as aldehyde ferredoxin oxidoreductase, with the basic structure of the pterin cofactor shared in the Mo- and W-enzymes (Hille, 2002). Recently, Mo-free nitrate reductases with vanadium and haem $c$ as cofactors have been isolated from vanadate-reducing, iron-reducing and sulfur-oxidizing bacteria (Antipov et al., 1998, 2003; Murillo et al., 1999). In addition, active W-substituted molybdoenzymes have been described in mesophilic bacteria (Buc et al., 1999; Gates et al., 2003). Thus, tungstate can function at the active site of the Paracoccus pantotrophus NapA, although the enzyme shows lower affinity for nitrate (Gates et al., 2003). Although nitrate reductases and other Mo-enzymes are usually inactivated by tungstate, this inhibition is not relevant *in vivo* since bacteria exhibit high-affinity Mo-uptake systems for extracting traces (micromolar or nanomolar) of Mo from the media, and because Mo is usually present in natural environments at higher concentrations than W. However, Mo(IV) forms a highly insoluble sulfide ($\mathrm{MoS}_{2}$), while V (as $\mathrm{VO}^{2+}$ or $\mathrm{V}^{3+}$) and W (as $\mathrm{WO}_{4}^{2-}$ or $\mathrm{WS}_{4}^{2-}$) form relatively soluble salts, and therefore there are chemical reasons to believe that both W and V were more available than Mo in the primitive strongly reducing sulfide-rich environments (Williams & Fraústo da Silva, 2002). In addition, some extreme environments such as hot springs and deep-sea hydrothermal vents usually contain high levels of tungsten, and in fact an obligate W requirement for the anaerobic growth of the hyperthermophile Pyrobaculum aerophilum has been reported (Afshar et al., 1998). The respiratory nitrate reductase of this archaeon is a Mo-enzyme, but its activity is not abolished at high W concentrations, in contrast to bacterial Nar systems. Moreover, the presence of tungstate in the culture medium stimulates the anaerobic growth with nitrate (Völkl et al., 1993). The ability to respire nitrate in these conditions has been interpreted as an adaptation of the archaeon to the W-rich hyperthermophilic environment (Afshar et al., 1998, 2001). Clearly, further biochemical studies will be necessary to determine the possible involvement of alternative metal cofactors in archaeal nitrate reduction, but the possibility that V or W, rather than Mo, may be included in some archaeal nitrate reductase cofactors cannot be excluded.

Nitrate transport systems

Two types of nitrate transporters are involved in bacterial assimilatory nitrate reduction: the ATP-dependent ABC transporters composed of an integral membrane subunit, a cytoplasmic ATP-binding component and a periplasmic substrate-binding protein; and the monomeric NarK-type transporters belonging to the major facilitator superfamily, which depend on proton-motive force. However, only

transport systems of the NarK type are involved in bacterial respiratory nitrate reduction (Moir & Wood, 2001). Within the bacterial NarK-like transporters there are two distinct subgroups. Recent studies in the bacterium Paracoccus pantotrophus suggest that NarK1 is a proton : nitrate symporter that allows initiation of nitrate respiration, whereas NarK2 is a nitrate : nitrite antiporter required for maintenance of a steady-state rate (Wood et al., 2002). Genome sequencing projects reveal that ABC-type transporters are widespread in archaea (Table 1). In particular, putative ABC-type nitrate transporters are present in the crenarchaeote Pyrobaculum aerophilum (Fitz-Gibbon et al., 2002) and the euryarchaeota Archaeoglobus fulgidus (Klenk et al., 1997), Methanocaldococcus jannaschii (Bult et al., 1996), Methanopyrus kandleri (Slesarev et al., 2002), Thermoplasma acidophilum (Ruepp et al., 2000) and Thermoplasma volcanium (Kawashima et al., 2000). However, substrate specificity for these putative transporters has been assigned solely on the basis of sequence similarities. A functional characterization of archaeal nitrate transporter genes has been only performed in Haloferax volcanii, where three ABC-type transporters are essential for nitrate respiration (Wanner & Soppa, 1999). The type of transport system used for nitrate uptake has a serious impact on the bioenergetics of the process because an ATP-driven nitrate uptake system is equivalent to consuming about three protons and this energetic cost makes its involvement in dissimilatory processes unlikely (Moir & Wood, 2001). For this reason, as mentioned above, bacterial nitrate assimilation usually requires ATP-dependent ABC nitrate transporters whereas nitrate respiration is associated with proton-motive-force-driven NarK transporters (Moreno-Vivián et al., 1999; Moir & Wood, 2001), even in the extreme thermophilic bacterium Thermus thermophilus (Ramírez et al., 2000). It is worth noting that most archaea with putative ABC nitrate transporters seem to contain respiratory nitrate reductases rather than assimilatory Nas enzymes. However, as discussed below, nitrate uptake may not be required for respiratory nitrate reduction in archaea, in contrast to bacteria, and the putative ATP-dependent nitrate transporters may indeed be involved in nitrate assimilation.

Assimilatory nitrate reduction in archaea

Most nitrate-reducing archaea use nitrate as alternative electron acceptor in anaerobic respiration, but recently, assimilatory nitrate and nitrite reductases have been purified from Haloferax mediterranei (Martínez-Espinosa et al., 2001a, b). This extreme halophilic archaeon grows aerobically with nitrate as the sole nitrogen source, but it can also denitrify because it contains both Nas and Nar enzymes. Assimilatory nitrate reductase is a dimer of 105 and 50 kDa subunits and uses ferredoxin as reductant, but not NAD(P)H (Martínez-Espinosa et al., 2001b). This could indicate a different structure for archaeal Nas because bacterial ferredoxin-Nas enzymes are usually monomeric, whereas NADH-dependent Nas enzymes are heterodimers (Moreno-Vivián et al., 1999). The assimilatory enzyme of H. mediterranei has a $K_{\mathrm{m}}$ for nitrate of 0·95 mM and an
P. Cabello, M. D. Roldán and C. Moreno-Vivián

optimum temperature of 80 °C at 3·1 M NaCl, but 60 °C at 1·3 M NaCl. Activity is induced by nitrate and repressed by ammonium, as described for the bacterial Nas (Martínez-Espinosa *et al.*, 2001b). The assimilatory nitrite reductase of *H. mediterranei* has also been purified. It is a 66 kDa monomer which uses ferredoxin as electron donor, has a \(K_{\mathrm{m}}\) for nitrite of 8·6 mM, and shows maximal activity at 60 °C with 3·3 M NaCl. Like the bacterial assimilatory nitrite reductases, the *H. mediterranei* enzyme contains sirohaem and Fe–S centres (Martínez-Espinosa *et al.*, 2001a). The 21 kDa ferredoxin from this halophilic archaeon has been purified and characterized, and its physiological role as the electron donor for both assimilatory nitrate and nitrite reductases has been confirmed (Martínez-Espinosa *et al.*, 2003).

Analysis of the published archaeal genome sequences suggests the presence of putative assimilatory nitrate/nitrite reductases (Table 1). Thus *Methanothermobacter thermautotrophicus* has a putative nitrate reductase (MTH1567 gene product) homologous to bacterial Nas (Smith *et al.*, 1997). *Thermoplasma* strains contain nitrate reductase related proteins (*T. acidophilum* Ta0136 and *T. volcanium* VN0215 gene products), which are similar to each other and to the eukaryotic assimilatory NADPH-nitrate reductases (Kawashima *et al.*, 2000; Ruepp *et al.*, 2000), although these putative proteins are too small (about 200 residues) compared to the *nia*-encoded eukaryotic nitrate reductases. The eukaryotic nitrate reductases are quite distinct from any of the bacterial Nas, Nar or Nap enzymes. They share no sequence similarity and do not bind the Mo-bisMGD cofactor, but instead have the so-called molybdopterin cofactor (Mo-co), in addition to FAD and cytochrome \(b_{557}\) (Moreno-Vivián *et al.*, 1999; Campbell, 2001). Therefore, the assignment of a role in nitrate reduction to the *Thermoplasma* small nitrate-reductase-related proteins will be of special evolutionary interest and should be explored in future studies.

The *Pyrobaculum aerophilum* genome contains a gene (*nirA*) encoding a putative assimilatory ferredoxin-dependent sirohaem nitrite reductase (PAE2577), in addition to putative genes encoding a respiratory cytochrome \(c d_{1}\)-nitrite reductase (Fitz-Gibbon *et al.*, 2002). Putative *nirA* homologous genes are also present in the genomes of *Pyrococcus furiosus*, *Pyrococcus horikoshii* (PF1197 and PH0827, respectively; Maeder *et al.*, 1999), *Aeropyrum pernix* (APE1326; Kawarabayasi *et al.*, 1999), *Archaeoglobus fulgidus* (AF0164; Klenk *et al.*, 1997), *Methanocaldococcus jannaschii* (MJ0551; Bult *et al.*, 1996), and *Methanopyrus kandleri* (MK0801; Slesarev *et al.*, 2002). However, biochemical and functional studies are necessary to confirm the assimilatory nature of these putative nitrate and nitrite reductases and to establish if these archaeal enzymes are structurally different from the bacterial assimilatory enzymes.

Respiratory nitrate reduction in archaea

The ability to use nitrate as a terminal electron acceptor in energy metabolism is found in a variety of halophilic and hyperthermophilic archaea; many of them can perform the entire denitrification process (see below). Respiratory Nar enzymes have been purified from several denitrifying halo-philic euryarchaeota, including three *Haloferax* species and *Haloarcula marismortui* (Table 1). A salt-requiring Nar has been purified from *Haloferax mediterranei*, with an activity maximum at 89 °C in 3·2 M NaCl and \(K_{\mathrm{m}}\) for nitrate values in the range from 2·5 to 6·7 mM depending on salt concentration (Álvarez-Ossorio *et al.*, 1992). The membrane-bound Nar of *Haloferax denitrificans* is composed of two subunits of 116 and 60 kDa, with a \(K_{\mathrm{m}}\) for nitrate of 0·2 mM. Curiously, the enzyme is stable in the absence of salt, and activity decreases with increasing NaCl concentrations (Hochstein & Lang, 1991). *Haloferax volcanii* contains a membrane-bound trimeric Nar, with 100, 61 and 31 kDa subunits. Activity is salt-independent, with a temperature optimum of 80 °C and a \(K_{\mathrm{m}}\) for nitrate of 0·36 mM (Bickel-Sandkötter & Ufer, 1995). The *Haloarcula marismortui* Nar has a \(K_{\mathrm{m}}\) for nitrate of 80 μM with 2·0 M NaCl, and activity is enhanced by salt. The enzyme was first described as a homotetramer of a 63 kDa polypeptide (Yoshimatsu *et al.*, 2000), but further characterization revealed that it is a heterodimer with 117 and 47 kDa subunits, the 63 kDa band being an incomplete denaturation state of the enzyme (Yoshimatsu *et al.*, 2002). It has been proposed that the *H. marismortui* Nar is a new archaeal type of membrane-bound nitrate reductase (Yoshimatsu *et al.*, 2002). First, the enzyme is composed of two subunits, which are homologous to the NarGH bacterial subunits, but lacks the NarI membrane-associated protein. Second, the *H. marismortui* NarGH complex has significant sequence and structure similarity to the dissimilatory selenate reductase from *Thauera selenatis*, although it does not reduce selenate. Recently, the similarity of Nar to selenate reductases has also been indicated on the basis that both enzymes have Asp ligands to the Mo atom (Jormakka *et al.*, 2004). In addition, the N-terminal region of NarG includes a typical twin-arginine signal peptide for protein translocation across the membrane by the Tat export pathway (Berks *et al.*, 2000) and the enzyme has activity *in situ* with both membrane-permeable benzyl viologen and membrane-impermeable methyl viologen, suggesting that the catalytic site is located in the outside of the membrane, as described for the *T. selenatis* selenate reductase (Yoshimatsu *et al.*, 2002). In contrast, bacterial NarG faces the cytoplasmic side of the membrane and nitrate reduction takes places in the cytoplasm (Berks *et al.*, 1995; Moreno-Vivián *et al.*, 1999). Finally, although the *H. marismortui* NarGH complex does not contain any quinol-oxidizing cytochrome \(b\) subunit, a gene encoding a homologue to SerC, the cytochrome \(b\) subunit of selenate reductase, is found downstream of the *narGH* genes and together with a *narJ*-homologous gene, which probably encodes a nitrate-reductase-specific chaperone for MGD cofactor assembly.

A gene cluster encoding a putative respiratory Nar has also been identified in *Archaeoglobus fulgidus* (Klenk *et al.*, 1997; Table 1). The AF0176 gene encodes a protein with 42%
similarity to *Escherichia coli* NarG, although this A. *fulgidus* NarG protein is smaller (80 kDa) and contains a twin-arginine signal peptide, suggesting that it is translocated across the membrane like the *H. marismortui* and probably other archaeal NarG proteins. The AF0175 product is predicted to bind four Fe–S clusters, whereas the AF0174 gene encodes an integral membrane protein, although no haem groups are bound to this protein. Finally, a NarJ homologue is also found in the gene cluster (Klenk et al., 1997; Richardson et al., 2001). In addition, genes encoding an ABC-type nitrate transporter (AF0638–AF0640), a putative *nirA* gene encoding the cytoplasmic sirohaem nitrite reductase (AF0164) and other putative iron–sulfur proteins clustered to *narI*-like genes (AF0499–AF0501 and AF0543–AF0547) are also present in the *A. fulgidus* genome. Some hyperthermophilic archaea are also able to respire nitrate. Thus, *Aeropyrum pernix* has a putative *nar* cluster (Kawarabayasi et al., 1999) and *Pyrobaculum aerophilum* uses nitrate as respiratory electron acceptor (Völk et al., 1993). The purified Nar of this archaeon shows very high specific activity, with a *K*~m~ for nitrate of 58 µM and an optimal temperature of 95 °C, and is active in the presence of high tungstate concentrations. Interestingly, the *P. aerophilum* Nar is a heterotrimer, with subunits of 130, 52 and 32 kDa, and contains a *b*-type cytochrome similar to the *Thauera selenatis* SerC, in addition to molybdenum cofactor and iron–sulfur cluster (Afshar et al., 2001).

Bacterial and archaeal Nar systems also differ in the organization of the *nar* genes (Philippot, 2002). The main *narGHJI*

operon organization is conserved in all bacteria, including *Thermus thermophilus* (Ramírez-Arcos et al., 1998a), although this organism has an extra *narC* gene in the operon, and *Paracoccus pantotrophus* includes the *narK* gene in the *narKGHJI* cluster (Wood et al., 2001). However, archaeal *nar* genes do not conserve this organization. Thus, in *Pyrobaculum aerophilum* a putative *narJ* gene is located upstream from the *narGH* genes, in the opposite direction. A gene encoding a cytochrome *b* is present downstream from *narH*, although its role as Nar subunit is conjectural, and another putative *narI* homologue is located elsewhere in the genome (Fitz-Gibbon et al., 2002). In *Haloarcula marismortui*, two small ORFs and a *narJ* homologue are located downstream from the *narGH* genes, and putative genes encoding an iron–sulfur protein and a cytochrome *b* are located upstream from *narGH* (Yoshimatsu et al., 2002). In *Aeropyrum pernix*, no *narI*-like gene has been found and *narGH* genes are followed by a putative cytochrome *b* gene and *narJ*, which is located about 2 kb downstream from *narH* (Kawarabayasi et al., 1999). Therefore, based on subunit composition, subcellular location of the active site and *nar* gene organization, it can be concluded that archaeal Nars are a new type of enzyme with the active site facing the outside and anchored to the membrane by a SerC-like cytochrome *b*, or stabilized by the lipid environment in the membrane as described for the *P. aerophilum* Nar (Afshar et al., 2001). This system could be an ancient respiratory nitrate reductase, although the nitrite formed could also be assimilated (Richardson et al., 2001). Figs 2

![Diagram](attachment:diagram.png)

**Nitrite Assimilation and/or Detoxification**

Fig. 2. Schematic representation of the denitrification pathway in bacteria. The periplasmic nitrate reductase (NAP), the membrane-bound nitrate reductase (NAR), the periplasmic nitrite reductases (Cu-NIR or *cd*~1~-NIR), the membrane-bound nitric oxide reductase (NOR) and the periplasmic nitrous oxide reductase (NOS) are represented without indicating their subunit composition and cofactors. Note that not all represented components are present in a single organism (i.e. NAP and NAR, Cu-NIR and *cd*~1~-NIR). cyt *bc*~1~, proton-pumping cytochrome *bc*~1~ complex; cyt *c*, NapC membrane-bound tetrahaem cytochrome *c*; cyt *c*~550~, cytochrome *c*~550~; MQH~2~/MQ, menaquinol/menaquinone pool.

http://mic.sgmjournals.org   3533
P. Cabello, M. D. Roldán and C. Moreno-Vivián

![Diagram](attachment:diagram.png)

Fig. 3. Schematic representation of the denitrification pathway in archaea. Membrane location and dependence on menaquinol as the electron donor are assumed for the four reductases (nitrate reductase, NAR; nitrite reductase, NIR; nitric oxide reductase, NOR; and nitrous oxide reductase, NOS), as suggested for *Pyrobaculum aerophilum*. These reductases are represented without indicating their subunit composition and cofactors. An outside location for the nitrate reduction site in the NAR complex is also assumed (see text for details). The identity of the proton-translocating dehydrogenase (DH) that regenerates menaquinol from menaquinone is presently unknown. The putative ABC-type nitrate/nitrite transporter present in some archaea is assumed to serve for assimilatory purposes rather than for the respiratory nitrate reduction, since the active site of NAR is assumed to be located on the outside of the membrane. MQH₂/MQ, menaquinol/menaquinone pool.

and 3 schematize the different organization of the bacterial and archaeal respiratory nitrate-reducing systems, and the other enzymes involved in the whole denitrification pathway (see below).

The possible outside location of the catalytic site of NarG in the archaeal respiratory nitrate reductase has important bioenergetic implications. First, bacterial nitrate respiration involves the generation of a proton-motive force by a redox-loop mechanism, with two protons consumed in the nitrate reduction reaction in the cytoplasm and two protons released from quinol oxidation on the periplasmic side of the membrane (Berks et al., 1995; Jormakka et al., 2003), although periplasmic nitrate reduction by the Nap enzyme and other reductases involved in the denitrification pathway (Fig. 2) are coupled to proton-pumping protein complexes. However, the possible outside location of the nitrate reduction site in archaeal Nar would imply the coupling of this process to a proton-translocating complex, although the identity of this complex is presently unknown (Fig. 3). The genome sequence of *Pyrobaculum aerophilum* reveals that this organism apparently lacks a cytochrome $bc_1$ complex (Fitz-Gibbon et al., 2002), but the occurrence of membrane-bound $bc$ complexes and different proton-pumping NADH dehydrogenases and hydrogenases has been described in archaea (Castresana & Moreira, 1999; Deppenmeier et al., 1999; Schütz et al., 2000; Sapra et al., 2003). In addition, an active nitrate-uptake system would not be required for respiratory nitrate reduction in archaea, thus increasing the energetic yield of the nitrate reduction process. This could also explain the apparent absence of NirK-type nitrate transporters in archaea and suggests that, as indicated above, the putative ATP-dependent ABC-type nitrate transporters found in several

archaeal species would be probably involved in nitrate assimilation rather than in nitrate respiration.

### Denitrification in archaea

Denitrification can be considered as the modular assembly of four partly independent respiratory processes: nitrate, nitrite, nitric oxide and nitrous oxide reduction (Zumft, 1997; Figs 2 and 3). All denitrifiers isolated so far are also able to respire oxygen, indicating a close evolutionary relationship between the two processes, and denitrification has been considered the ancestor of aerobic respiration (Saraste & Castresana, 1994). Therefore, knowledge of denitrification in archaea is of special interest from an evolutionary point of view. Denitrification has been described for several halophilic archaea, such as *Haloferax* and *Haloarcula* strains (Tomlinson et al., 1986; Inatomi & Hochstein, 1996; Ichiki et al., 2001), and extreme thermophilic archaea, such as *Ferroplubus placidus* (Vorholt et al., 1997) and *Pyrobaculum aerophilum* (Völk et al., 1993; de Vries & Schröder, 2002). As the archaeal respiratory nitrate reductases that catalyse the first step of denitrification have been described above, we will focus now on the nitrite and N-oxide respiration processes in archaea.

### Nitrite respiration

In bacteria, there are two different and unrelated NO-producing respiratory nitrite reductases: the homotrimeric copper-containing enzyme encoded by *nirK*, and the homodimeric cytochrome $cd_1$-nitrite reductase encoded by *nirS*. The crystal structures of both enzymes, which are located in the periplasm, are known and have been reviewed in detail elsewhere (Zumft, 1997; Ferguson,
1998; Richardson & Watmough, 1999). Azurin and pseudo-azurin are the electron donors to Cu-Nir, whereas cytochrome $c_{551}$ is the reductant for $cd_1$-Nir (Zumft, 1997).

Denitrifying archaea also contain either Cu-Nir or $cd_1$-Nir. *Haloarcula marismortui* has a *nirK* gene encoding a Cu-Nir. EPR analysis reveals the presence of type 1 and type 2 Cu centres in the purified enzyme, which has optimal activity at 2 M NaCl. A 46 kDa precursor of this protein with an N-terminal signal peptide for translocation across the plasma membrane has been detected by SDS-PAGE (Ichiki *et al.*, 2001). Although the physiological electron donor is unknown, the archaeal halocyanin, a blue Cu-protein present in some archaea (Scharf & Engelhard, 1993), could be the putative reductant. Protein sequence analysis of the *H. marismortui nirK* gene product reveals a close relationship with the *Neisseria gonorrhoeae* Cu-Nir, thus suggesting lateral gene transfer of the *nirK* gene (Ichiki *et al.*, 2001; Philippot, 2002). Interestingly, the *Haloferax denitrificans nirK* gene sequence is also available (CAD89521, Table 1), and is highly similar to both *H. marismortui* and *N. gonorrhoeae* Cu-Nir. However, anti-serum against bacterial Cu-Nir does not recognize the *H. denitrificans* enzyme, suggesting possible structural differences (Inatomi & Hochstein, 1996).

The existence of a *nirS* homologous gene in the genome of the hyperthermophile *Pyrobaculum aerophilum* suggests the presence of a $cd_1$-nitrite reductase (Table 1). Recently, it has been reported that this archaeon has a membrane-bound respiratory nitrite reductase which uses menaquinol as the physiological electron donor, thus having different subcellular location and reductant than bacterial $cd_1$-Nir (Figs 2 and 3). It is worth noting that, in contrast to denitrifying bacteria, all four reductases involved in denitrification in *P. aerophilum* are membrane-bound enzymes which use menaquinol as electron donor (de Vries & Schröder, 2002). The dependence of the menaquinol pool and the membrane confinement of all these enzymes in archaeal denitrification could be related to the relatively small volume between the S-layer and the cytoplasmic membrane, in comparison with the larger periplasmic space in bacteria, or could reflect the ancient origin of this pathway. The membrane location of all denitrification enzymes in *P. aerophilum* could also explain the absence in this archaeon of soluble components of the denitrification pathway, such as c-type cytochromes or cupredoxins (Lubben & Morand, 1994; de Vries & Schröder, 2002; see Figs 2 and 3).

### N-oxide respiration

Nitric oxide, the product of the respiratory nitrite reductases, is a toxic compound that is reduced to N₂O by nitric oxide reductases (Nor) immediately after it has been generated. Several enzymes with Nor activity have been described. In fungal denitrification, NO reduction to N₂O is catalysed by a soluble monomeric enzyme of the cytochrome P-450 family (Shoun & Tanimoto, 1991; Nakahara

et al., 1993). In most denitrifying bacteria, Nor is a membrane complex of a 17 kDa cytochrome $c$ (encoded by *norC*) and a 38 kDa cytochrome $b$ with 12 transmembrane regions (encoded by *norB*). This enzyme, known as cNor, receives electrons from cytochrome $c$. The denitrifying bacterium *Ralstonia eutropha* and several non-denitrifying pathogenic bacteria such as *Neisseria* species possess a monomeric Nor with 14 transmembrane regions, which is called qNor because it is a quinol-oxidizing enzyme. This protein, encoded by *norZ*, is similar to the *norB* gene product but contains an N-terminal extension absent in NorB, with a quinone-binding site. In Gram-positive bacteria, a menaquinol Nor with an additional small Cuₐ-containing subunit has been described as qCuₐNor (Hendriks *et al.*, 2000; de Vries & Schröder, 2002; Philippot, 2002).

Genetic and biochemical evidence indicates that the archaeon *Pyrobaculum aerophilum* has a qNor-type NO reductase (Table 1). Putative *nir* (nitrite reductase) and *nor* (nitric oxide reductase) genes are located 3·3 kb upstream from the *nar* (nitrate reductase) genes (Völkl *et al.*, 1993; Fitz-Gibbon *et al.*, 2002). The purified Nor is a single membrane subunit with two modified haem groups and one non-haem iron, and uses menaquinol as electron donor (de Vries & Schröder, 2002). The genome of *Methanosarcina mazei* also contains a NO reductase gene (MM2988; Deppenmeier *et al.*, 2002) and *Sulfolobus solfataricus* includes a putative gene encoding a haem NO reductase, but it is truncated by a transposase separating the N-terminal (SSO1573) and C-terminal (SSO1571) domains (She *et al.*, 2001). Sequenced genomes of other archaea do not include *nor* homologous genes (Table 1), although the presence of NO reductase has been reported in some denitrifying halophilic archaea that have not yet been sequenced. The presence of qNor in some archaea is of evolutionary interest because qNor could be the ancestor not only of the cNor, by acquiring haem $c$-binding residues in the N-terminal region and separating this domain as a small cytochrome $c$ subunit, but also of the whole superfamily of haem-copper oxidases and aerobic respiratory enzymes. The capacity for N-oxide respiration appeared early in evolution and qNor may have arisen to detoxify NO; this could still be the role of the enzyme in non-denitrifying bacteria. After the appearance in the atmosphere of oxygen generated photosynthetically, these enzymes could have replaced non-haem iron by copper and evolved to generate the proton-pumping cytochrome oxidases (Saraste, 1994; Saraste & Castresana, 1994; Richardson, 2000; de Vries & Schröder, 2002).

On the other hand, the *E. coli* flavoprotein has been recently described as a novel type of prokaryotic NO reductase. The purified protein has a turnover number similar to that of canonical bacterial Nor and also has oxygen reductase activity, although it shows lower affinity for O₂ than for NO (Gomes *et al.*, 2002). Flavoproteins are A-type flavoproteins which are present in bacteria and

http://mic.sgmjournals.org   3535some archaea, such as *Pyrococcus furiosus*, *Archaeoglobus fulgidus*, *Methanocaldococcus jannaschii* and *Methano- thermobacter thermautotrophicus*. Therefore, it has been proposed that this protein is a widespread class of NO reductase, which could detoxify NO or oxygen, allowing versatility in response to environmental conditions (Gomes *et al.*, 2002). However, the physiological relevance of this protein should be determined in future studies.

The last step of denitrification, the reduction of N₂O to N₂, is of great environmental importance because it closes the N-cycle. Although N₂O is less toxic than NO or nitrite, and bacteria could manage without converting N₂O into N₂, most denitrifying bacteria contain nitrous oxide reductases (Nos) encoded by the *nosZ* gene. These enzymes are periplasmic multicopper homodimers that receive electrons from cytochrome c or pseudoazurin (Zumft, 1997). In each 65 kDa monomer, the copper ions are organized in two centres, a di-copper cluster CuA resembling that of cytochrome oxidase, and a CuZ cluster in which four Cu atoms are ligated by seven His residues. A bridging inorganic sulfur in the CuZ centre has also been identified (Brown *et al.*, 2000; Rasmussen *et al.*, 2000). In some archaea, such as *Haloarcula marismortui* and *Ferroglobus placidus*, the predominant gas species produced by denitrification is N₂O (Werber & Mevarech, 1978; Vorholt *et al.*, 1997), although other archaea such as *Haloferax denitrificans* and *Pyrobaculum aerophilum* are able to reduce N₂O to N₂, having a complete denitrification pathway (Tomlinson *et al.*, 1986; Völk *et al.*, 1993; de Vries & Schröder, 2002). However, in *P. aerophilum*, N₂O formed by qNor accumulates because its reduction to N₂ by the nitrous oxide reductase is very slow. In contrast to the periplasmic bacterial Nos, the *P. aerophilum* enzyme is located in the membrane and uses menaquinol as electron donor, like the other three reductases of the denitrification pathway in this hyperthermophile (de Vries & Schröder, 2002; Fig. 3). N₂O reduction has not been investigated in other species of archaea and putative *nos* genes are not present in the sequenced archaeal genomes (Table 1), although two long hypothetical quinol oxidases (ST2395 and ST0105) with the motif of the Nos binuclear copper centre are found in *Sulfolobus tokodaii* (Kawarabayasi *et al.*, 2001).

### N₂ fixation in archaea

N₂ is the most abundant gas in Earth’s atmosphere, but it is highly stable and unreactive and is only used as nitrogen source by some prokaryotic organisms. Biological N₂ fixation is catalysed by nitrogenase, an O₂-sensitive enzyme complex which generates two molecules of NH₃ by reduction of N₂. Nitrogenase can also reduce protons to H₂ and several substrates with double or triple bonds, such as acetylene and cyanide (Halbleib & Ludden, 2000). Nitrogenase consists of a molybdenum–iron protein (dinitrogenase), which receives electrons from an iron protein known as dinitrogenase reductase. MoFe protein is an α₂β₂ tetramer encoded by the *nifD* and *nifK* genes, and Fe protein is a homodimer of the *nifH* gene product (Howard & Rees, 1996). Several *nif* genes are also involved in cofactor biosynthesis, electron transfer and regulation. Alternative nitrogenases have been found in several bacteria, with V or Fe substituting for Mo. These enzymes have lower catalytic efficiency than Mo-nitrogenases and are encoded by *nif* homologous genes, termed *vnf* (V-nitrogenase) or *anf* (Fe-nitrogenase). Alternative nitrogenases include an additional small subunit δ, encoded by *vnfG* or *anfG* (Eady, 1996; Rees, 2002). N₂ fixation is highly regulated since is an energy-expensive process requiring large amounts of reducing power and ATP. Ammonium causes a reversible short-term inhibition of enzyme activity, called switch-off/on (Zumft & Castillo, 1978), and a long-term effect at the transcriptional level. Ammonium switch-off occurs by ADP-ribosylation of NifH by DraT/DraG, an ADP-ribosyltransferase/glycohydrolase system. Expression of *nif* genes requires NifA, a transcriptional activator under the control of Ntr, a two-component regulatory system of bacterial nitrogen metabolism (Ludden, 1994; Merrick & Edwards, 1995; Halbleib & Ludden, 2000).

N₂ fixation in archaea, first described in *Methanosarcina barkeri* (Murray & Zinder, 1984) and *Methanococcus thermolithotrophicus* (Belay *et al.*, 1984), is exclusive of, but widespread within, methanogenic euryarchaeota (Table 2). However, neither *Methanococcus voltae* nor *Methanocaldococcus jannaschii* grows diazotrophically, despite the presence of *nifH*-like genes in their genomes (Leigh, 2000). Mo-nitrogenase predominates in diazotrophic methanogens, which contain at least the *nifHDKENX* genes (Lobo & Zinder, 1990; Chien & Zinder, 1996; Kessler *et al.*, 1998; Leigh, 2000). *nifHDK* are the structural genes for nitrogenase, whereas *nifENX* code for FeMo-cofactor biosynthesis proteins (Moreno-Vivián *et al.*, 1989). The diazotrophic growth of *Methanococcus maripaludis* shows Mo dependence, indicating that N₂ fixation is catalysed by a Mo-nitrogenase (Kessler *et al.*, 1997), although *Methanosarcina* species contain alternative nitrogenases (Table 2). *M. barkeri* has two clusters of nitrogenase genes, the *nif2* gene cluster encoding the Mo-nitrogenase and the *nif1* (*vnf*) cluster encoding an alternative V-nitrogenase, which also includes a *vnfG* gene (Chien *et al.*, 2000). Like the *nif* regions of other methanogens, both gene clusters contain two *glnB* genes encoding nitrogen signal transduction proteins (Chien *et al.*, 2000). The *M. acetivorans* genome includes *nif*, *vnf* and *anf* genes, being unique among archaea in having all three nitrogenase systems (Galagan *et al.*, 2002).

There are significant differences in gene organization and regulation between bacterial and archaeal nitrogenase systems. First, all *nif* genes and the *glnB* genes are co-transcribed in a single unit (*nifHglnBIglnBII nifDKENX*), this being a gene organization unique to archaea (Kessler *et al.*, 1998; Leigh, 2000). Transcription of *nif* genes occurs from typical archaeal promoters, probably reflecting the differences between archaeal and bacterial transcription apparatus, and is regulated by repression, whereas an activation mechanism involving NifA operates in bacteria. As stated
Archaea
Ammonium
Nitrogenases
N₂-fixation-related
Glutamine
Glutamate
Glutamate
transporter
proteins
synthetase
synthase
dehydrogenase

Crenarchaeota

Aeropyrum pernix
ND
ND
ND
GlnA
Small subunit
GdhA
APE 2142
APE 0935
APE 1386

Sulfolobus solfataricus
AmtB
ND
NifS
GlnA
Large subunit
GdhA
SSO 1054
SSO 2215
SSO 0366, SSO 2440,
SSO 0684(GltB)
SSO 1457, SSO 1907,
SSO 2554
SSO 1930, SSO 2044

Sulfolobus tokodaii
AmtB
ND
NifW
GlnA
Large subunit
GdhA1
ST0661
STS 185
ST0156, ST0568,
ST2198
ST0100, ST224
NifB-like ST1383, ST2412
ST1387

Pyrobaculum aerophilum
ND
ND
ND
GlnA
Small subunit
GdhA
PAE 2556, PAE 3442
PAE 3227(GltD),
PAE 1996, PAE 3438
PAE 3443,
PAE 3444

Euryarchaeota

Archaeoglobus fulgidus
AmtB
ND
DraG
GlnA
Large subunit
ND
AF0977, AF1746,
AF1724
AF0949
AF0953(GltB)
AF1749
NifU AF0185, AF0565, AF0632
NifS AF0186, AF0564

Halobacterium sp.
ND
ND
NifU
GlnA
ND
GdhAB
VNG2472G
VNG2093G
VNG0161G(GdhB),
NifSVNG2471G
VNG0628G(GdhA),
VNG1204G(GdhA)

Methanothermobacter
AmtB
Nif
GlnB-like
GlnA
Large subunit
ND
thermautotrophicus
MTH 661, MTH 663
MTH 571, MTH 643, MTH 1711,
MTH 1561, MTH 1562
MT1570
MTH 105, MTH 194,
MTH 1560(NifH), MTH 1522,
NifB-like MTH 1871
MTH 1666
MTH 1563(NifD), MTH 1564
NifS MTH 1389
(NifK), MTH 1482, MTH 1565
(NifE), MTH 1566(NifN)

Methanocaldococcus
AmtB
Nif
DraG
GlnA
Large subunit
ND
jannaschii
MJ0058, MJ1343
MJ0879(NifH), MJ0685(NifH)
MJ1187
MJ1346
MJ1351(GltB)
Vnf
NifB-like
MJ1423(VnfN)
MJ1093

Methanopyrus kandleri
AmtB
Nif
NifB-like
GlnA
Large subunit
ND
MK0053
MK1385(NifD), MK1416(NifH)
MK0243, MK0650, MK1488
MK0338
MK0550, MK1063,
MK1064(GltB)

| Archaea                     | Ammonium transporter       | Nitrogenases                                                                 | N₂-fixation-related proteins                                                                                          | Glutamine synthetase          | Glutamate synthase               | Glutamate dehydrogenase         |
|-----------------------------|----------------------------|------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|------------------------------|-----------------------------------|----------------------------------|
| Methanosarcina acetivorans   | AmtB                       | MA3917, MA3918,                                                              | GlnB-like                                                                                 | GlnA                           | Small subunit                    | GdhA                             |
|                             |                            | MA1633, MA2032, MA2033, MA3627,                                                | MA1211, MA1212, MA1214,                                                                                               | MA3382, MA4216                 | MA3787                           | MA2892, MA3169                   |
|                             |                            | MA3628, MA3895 (NifH), MA3898                                                  | MA1215, MA3916, MA4208                                                                                               |                              |                                   |                                  |
|                             |                            | (NifD), MA3899 (NifK), MA3900                                                  | DraG                                                                                     |                              |                                   |                                  |
|                             |                            | (NifE), MA3901 (NifN)                                                          | MA4144                                                                                   |                              |                                   |                                  |
|                             |                            | Vnf                                                                           | NifU                                                                                     |                              |                                   |                                  |
|                             |                            | MA1213 (BNF), MA1216 (VnfD),                                                   | MA0807, MA1225, MA1226,                                                                                               |                              |                                   |                                  |
|                             |                            | MA1217 (VnfG), MA1218 (VnfK),                                                  | MA2217, MA3265                                                                                                       |                              |                                   |                                  |
|                             |                            | MA1219 (VnfE), MA1220 (VnfN),                                                  | NifS                                                                                     |                              |                                   |                                  |
|                             |                            | MA1631 (VnfE), MA1632 (VnfN)                                                   | MA0236                                                                                   |                              |                                   |                                  |
|                             |                            | Anf                                                                           | NifB-like                                                                                |                              |                                   |                                  |
|                             |                            | MA1205 (AnfH), MA1208 (AnfK),                                                  | MA0175, MA1114, MA1486,                                                                                               |                              |                                   |                                  |
|                             |                            | MA1209 (AnfG), MA1210 (AnfD)                                                   | MA2936, MA3185, MA3448,                                                                                               |                              |                                   |                                  |
|                             |                            |                                                                               | MA3482, MA4195                                                                                                       |                              |                                   |                                  |
| Methanosarcina mazei        | AmtB                       | MM0733, MM0957                                                                | Nif                                                                                      | GlnA                           | Large subunit                    | GdhA                             |
|                             |                            | MM0514, MM0687, MM0719,                                                       | GlnB-like                                                                                 | MM0964, MM3188                 | MM0966, MM0967,                  | MM0357, MM3297,                  |
|                             |                            | MM2086 (NifH), MM0722 (NifD),                                                  | MM0720, MM0721                                                                                                       |                              | MM0968                           | MM3298                           |
|                             |                            | MM0723 (NifK), MM0724 (NifE),                                                  | NifS                                                                                     |                              | Small subunit                    |                                  |
|                             |                            | MM0725 (NifN)                                                                  | MM1517, MM1955                                                                                                       |                              | MM0664                           |                                  |
|                             |                            |                                                                               | NifU                                                                                     |                              |                                   |                                  |
|                             |                            |                                                                               | MM0110, MM1954                                                                                                       |                              |                                   |                                  |
|                             |                            |                                                                               | NifB-like                                                                                |                              |                                   |                                  |
|                             |                            |                                                                               | MM0309, MM0757, MM0759,                                                                                               |                              |                                   |                                  |
|                             |                            |                                                                               | MM0788, MM1105, MM1474,                                                                                               |                              |                                   |                                  |
|                             |                            |                                                                               | MM2468                                                                                   |                              |                                   |                                  |
| Pyrococcus abyssi           | ND                         | ND                                                                            | ND                                                                                        | GlnA                           | Small subunit                    | GdhA                             |
|                             |                            |                                                                               | PAB 1292                                                                                  | PAB 1214, PAB 1738             | PAB 0391                         |                                  |
| Pyrococcus furiosus         | ND                         | ND                                                                            | NifH-like                                                                                 | GlnA                           | Small subunit                    | GdhA                             |
|                             |                            |                                                                               | PF2028 (N-term.), PF2029                                                                                              | PF0450                         | PF1327, PF1852,                  | PF1602                           |
|                             |                            |                                                                               | (C-term.)                                                                                 |                              | PF1910                           |                                  |
|                             |                            |                                                                               | NifS                                                                                     |                              | Large subunit                    |                                  |
|                             |                            |                                                                               | PF0164                                                                                   |                              | PF0205                           |                                  |
| Pyrococcus horikoshii       | ND                         | ND                                                                            | ND                                                                                        | GlnA                           | Small subunit                    | GdhA                             |
|                             |                            |                                                                               | PH0359                                                                                   | PH0876, PH1873                 | PH1593                           |                                  |
| Thermoplasma acidophilum    | AmtB                       | Ta0180, Ta0181                                                                | ND                                                                                        | GlnA                           | Small subunit                    | GdhA                             |
|                             |                            |                                                                               | Tal498                                                                                    | Ta0414                         | Ta0635, Ta0776                   |                                  |
| Thermoplasma volcanium      | AmtB                       | TVG 1095610                                                                   | ND                                                                                        | GlnA                           | Small subunit                    | GdhA                             |
|                             |                            |                                                                               | TVG 1571797                                                                               | TVG 1211894                    | TVG 0773051,                     |                                  |
|                             |                            |                                                                               |                                                                                          |                              |                                   | TVG 0774569                      |
below, a repressor NrpR found only in euryarchaeota regulates *nif* and *glnA* (glutamine synthetase) gene expression (Lie & Leigh, 2003). Finally, post-transcriptional regulation by ammonium (switch-off) also occurs in archaea (Lobo & Zinder, 1990; Kessler & Leigh, 1999; Kessler *et al.*, 2001), but does not involve ADP-ribosylation or covalent modification of NifH and requires two members of the GlnB family, termed NifI₁ and NifI₂ (Kessler *et al.*, 2001).

Phylogenetic analysis suggests that N₂ fixation may have originated in a common ancestor of bacteria and archaea, before branching of these domains and prior to the appearance of oxygen in the atmosphere (Fani *et al.*, 2000; Leigh 2000; Berman-Frank *et al.*, 2003). Comparative analysis of the amino acid sequences of the NifDK and NifEN proteins suggests a two-step in-tandem duplication of an ancestor gene, giving rise first to a bicistronic operon, which later underwent a paralogous duplication leading to the *nifDK* and *nifEN* genes before the divergence of archaea and bacteria (Fani *et al.*, 2000). The function of this ancient enzyme might have depended on the composition of the early atmosphere, being either a nitrogenase for N₂ fixation in a neutral anoxic atmosphere containing N₂ but not NH₃, or alternatively, a cyanide-detoxifying enzyme in a strongly reducing atmosphere, which could change its specificity to N₂, another triple-bond substrate, after progressive exhaustion of combined nitrogen, thus enabling survival of ancient micro-organisms in N-deficient environments before the appearance of photosynthetic O₂ (Fani *et al.*, 2000; Berman-Frank *et al.*, 2003). After the divergence of archaea and bacteria, many organisms lost *nif* genes due to the weakening of selection pressure and the high energetic cost of N₂ fixation, although lateral gene transfer of *nif* genes might have also occurred (Fani *et al.*, 2000). However, there are no apparent reasons to explain the loss of diazotrophy in all archaea except methanogens. Earth's earliest organisms may have been similar to the thermophiles found in deep-sea hydrothermal vents, but currently the most thermophilic micro-organism known, *Methanothermococcus thermolithotrophicus*, is able to fix N₂ at 64 °C (Mehta *et al.*, 2003). This makes especially interesting a recent paper describing the first evidence of potential nitrogen fixers in diffuse hydrothermal vent fluids and deep-sea water, and reporting that all of the *nifH* genes from these samples are most closely related to *M. thermolithotrophicus* (Mehta *et al.*, 2003).

## Transport and assimilation of ammonium in archaea

Glutamine synthetase-glutamate synthase (GS-GOGAT) and glutamate dehydrogenase (GDH) are the major pathways for ammonium assimilation. The GS-GOGAT route requires ATP but has a high affinity for ammonium, whereas GDH does not consume ATP but is less effective in cells growing in N-limited conditions. As these enzymes are essential in ammonia assimilation and amino acid metabolism, they are present in all three domains of life. Movement

of ammonium across biological membranes is also an important process, and ammonium transporters (Amt) are found in bacteria, archaea and eukarya.

### Ammonium transport

N₂ fixation and nitrate assimilation generate ammonium for incorporation into carbon skeletons inside the cells. However, when ammonium is available in the environment, organisms usually take up this ion by a specific transport system, inhibiting and/or repressing the reductive assimilatory pathways of the N-cycle. Although NH₃ diffuses through biological membranes, and this could represent a significant process in organisms growing in alkaline environments, ammonium transporters are found in both prokaryotic and eukaryotic organisms. Amt proteins are a family of ammonium and methylammonium transporters, which are integral membrane proteins of 45–50 kDa with 10–12 transmembrane helices (Thomas *et al.*, 2000). It is widely accepted that Amt proteins mediate an active NH₄⁺ uniport which depends on proton-motive force, although it has been also proposed that Amt proteins facilitate diffusion of NH₃ across the plasma membrane bidirectionally (Soupene *et al.*, 2002). The *E. coli* AmtB protein is essential for transport of ammonium and is able to complement a *Saccharomyces cerevisiae* mutant (Blakey *et al.*, 2002). Putative ammonium transporters (AmtB) are also present in most archaeal genomes (Table 2). In bacteria and archaea, the *amtB* gene is invariably linked to the regulatory *glnK* gene, which encodes a GlnB-like protein (PⅡ protein). Both GlnK and AmtB proteins interact for regulation of the transport activity in response to nitrogen availability (Thomas *et al.*, 2000; Coutts *et al.*, 2002). In *E. coli*, AmtB is inactivated by formation of a membrane-bound complex with GlnK, a process regulated by uridylylation/deuridylylation of GlnK in response to the nitrogen status of the cells, and the AmtB activity is also required for GlnK deuridylylation. Therefore, AmtB could be considered as an ammonium sensor, and transport is also an integral part of the signal transduction cascade (Javelle *et al.*, 2004).

### The glutamine synthetase–glutamate synthase cycle

Glutamine synthetase (GS) is a ubiquitous enzyme with a central role in nitrogen metabolism. Three types of GS catalyse the ATP-dependent synthesis of glutamine from glutamate and ammonia. GSI, found in most prokaryotes, is a dodecameric protein composed of identical 50–56 kDa subunits arranged in two hexagonal rings. GSII is mainly present in eukaryotes and has an octameric structure, with identical subunits of 45–50 kDa. GSIII is only found in a few bacterial species, such as *Bacteroides fragilis*, and has a hexameric structure with identical subunits of about 83 kDa (Pesole *et al.*, 1995; DiRuggiero & Robb, 1996; Eisenberg *et al.*, 2000).

Putative *glnA* genes encoding GSI are present in archaeal
genomes, sometimes in several copies (Table 2). Sequence analysis of the *glnA* gene reveals two main subdivisions: GSI-α, including euryarchaeota, *Thermotoga* and low-G+C Gram-positive bacteria; and GSI-β, with all other bacteria. Biochemical properties also indicate that GSs from archaea and Gram-positive bacteria are functionally similar (Bhatnagar *et al.*, 1986). Lateral gene transfer from archaea to *Thermotoga* and Gram-positive bacteria could explain the close evolutionary relation between these GSs (Tiboni *et al.*, 1993; Pesole *et al.*, 1995). However, the crenarchaeote *Sulfolobus solfataricus* GS is not placed in either the GSI-α or the GSI-β division (Brown *et al.*, 1994), although the *Sulfolobus acidocaldarius glnA* is more related to GSI-β, despite its enzyme properties being more similar to GSI-α (Yin *et al.*, 1998). Most GSI-β enzymes are regulated by adenylylation/deadenylylation, but GSI-α enzymes are not controlled by this reversible mechanism. Purified GSs from *Pyrococcus* sp., *S. acidocaldarius* and *Methanobacterium ivanovi* are dodecamers of identical subunits that are not regulated by adenylylation (Bhatnagar *et al.*, 1986; Rahman *et al.*, 1997; Yin *et al.*, 1998). Therefore, branching of GSI-α and GSI-β could precede the separation of archaea and bacteria.

The relevance of GS in ammonium assimilation varies depending on the archaeal species. Most hyperthermophilic archaea show very high glutamate dehydrogenase activity, suggesting that this enzyme is the major pathway for ammonium assimilation. In fact, *Thermococcus profundus* has no detectable GS activity (Kabayashi *et al.*, 1995) and the GS of *Pyrococcus* sp. has low biosynthetic activity, suggesting that the reaction is biased towards glutamate production. Therefore, hyperthermophilic archaea seem not to use GS predominantly for ammonium assimilation. In contrast, the inability to obtain a *glnA* null mutant in *Methanococcus maripaludis* suggests that this archaeon cannot transport glutamine and that GS is an essential enzyme, even in the presence of alternative nitrogen sources (Cohen-Kupiec *et al.*, 1999).

Glutamate synthase (glutamine:2-oxoglutarate amidotransferase, GOGAT) is an iron-sulfur flavoprotein that catalyses the transfer of the amide group of glutamine to 2-oxoglutarate, yielding two molecules of glutamate. In bacteria, GOGAT is usually an NADPH-dependent enzyme arranged in an α₄β₄ structure, with large α subunits of about 135 kDa and small β subunits of approximately 53 kDa (Trotta *et al.*, 1974). The genomes of most archaea contain putative genes encoding either the large or the small GOGAT subunit (Table 2). The *gltA* gene encoding a putative small GOGAT subunit in the hyperthermophile *Pyrococcus* sp. has been cloned. The purified GltA protein is a tetramer with identical subunits of 53 kDa. This GOGAT uses NADPH and has both glutamine- and ammonium-dependent activities, showing different temperature optima, 80 °C for the glutamine reaction and 90 °C for the ammonium-dependent reaction (Jongsareejit *et al.*, 1997). The amino acid sequence of this GOGAT includes

two Cys clusters, and adenylate- and FAD-binding domains, which are conserved in the small subunits of bacterial GOGAT. It is worth noting that *Pyrococcus* sp. lacks the large subunit gene and GOGAT is encoded by the *gltA* gene alone. In contrast, genomes of other archaea include only the *gltB* gene encoding the large subunit (Table 2). *Methanocaldococcus jannaschii* has only a *gltB* gene encoding a 55 kDa protein, which includes features of the bacterial large subunit and probably has GOGAT activity (Bult *et al.*, 1996). It has been proposed that both *M. jannaschii* GOGAT (GltB) and *Pyrococcus* sp. GOGAT (GltA) represent ancestral forms of the large and small subunits, respectively, which are functional without their subunit counterpart (Jongsareejit *et al.*, 1997). On the other hand, in eukaryotes, GOGAT is a monomeric enzyme homologous to either the large bacterial subunit, as the maize ferredoxin-GOGAT, or to both large and small bacterial subunits, as the alfalfa NADH-GOGAT (Jongsareejit *et al.*, 1997).

### Glutamate dehydrogenase

Glutamate dehydrogenase (GDH) catalyses both the oxidative deamination of glutamate and the reductive incorporation of ammonium into 2-oxoglutarate. There are three types of GDH based on coenzyme specificity. Dual NAD(P)-GDHs are found in animals and higher plants, but NADP-GDHs are mostly involved in ammonia assimilation in bacteria, fungi and algae. Both NADP-dependent and dual NAD(P)-GDH are hexamers composed of 48–55 kDa identical subunits. On the other hand, NAD-GDHs consist of either four or six identical subunits and usually have a catabolic role, being involved in the oxidative deamination of glutamate (Smith *et al.*, 1975).

Most hyperthermophilic archaea studied to date express high levels of GDH, and it is assumed that GDH plays a major role in ammonium assimilation in these extremophiles (DiRuggiero & Robb, 1996). The enzymes from *Pyrococcus furiosus*, *Thermococcus litoralis*, *Thermococcus profundus* and *Sulfolobus solfataricus* are hexameric, as in mesophilic bacteria, and show a high temperature optimum for activity. However, the coenzyme specificity of GDH from these hyperthermophiles resembles that of the eukaryotes rather than bacteria, since the enzyme uses both NADH and NADPH (DiRuggiero & Robb, 1996). The three-dimensional structure of the *P. furiosus* NAD(P)-GDH has been determined at 2·2 Å resolution (Yip *et al.*, 1995). The *gdhA* genes encoding GDH have been cloned or identified in the genome of several archaea (Table 2). The *P. furiosus gdhA* gene is transcribed from a typical archaeal promoter, but it has been expressed in *E. coli*, yielding a recombinant GDH fully active in vitro at 85 °C (DiRuggiero & Robb, 1995). Sequencing of the *Pyrococcus horikoshii* genome revealed the presence of a *gdhA* gene homologous to the GDH-encoding genes of *P. furiosus* and *T. litoralis*. Recently, this *gdhA* gene has been cloned and a recombinant His₆-tagged GDH was overexpressed in *E. coli* and purified. The enzyme is a hexamer of 290 kDa (subunit
mass 48 kDa), requires NAD(P)H and has a temperature optimum of 90°C (Wang *et al.*, 2003). This *P. horikoshii* GDH shows a lower *K*~m~ value for glutamate than for ammonium, suggesting that the reaction is biased toward oxidative deamination of glutamate. NADP is the preferred coenzyme for this reaction. The enzyme subunit contains many charged residues that at high temperature arrange networks of ion pairs for proper folding and maintaining the native conformation of the enzyme (Wang *et al.*, 2003). The GDH enzyme of *Aeropyrum pernix* has also been characterized (Bhuiya *et al.*, 2000; Helianti *et al.*, 2001, 2002). The enzyme also has a hexameric structure with a molecular mass of about 285 kDa, and is specific for NADP. This GDH is thermostable, and the N-terminal length seems to be important for its thermostability (Helianti *et al.*, 2002). The *A. pernix* enzyme has biphasic kinetics for glutamate, but monophasic kinetics for NADP at 50–90°C, and it has also been crystallized and analysed by X-ray diffraction (Bhuiya *et al.*, 2002a, b).

Protein sequence analysis indicates that the enzymes from *Pyrococcus furiosus* and *Pyrococcus horikoshii* are more closely related to the mesophilic bacterial GDH than to the thermophilic archaeon *Sulfolobus solfataricus* or to the halophilic archaeon *Halobacterium salinarum* (DiRuggiero & Robb, 1996), suggesting that lateral gene transfer between bacterial and archaeal *gdhA* genes may have occurred during evolution. Some organisms have two or more GDHs, which differ in coenzyme specificity. This is the case for *H. salinarum*, which has both NADP- and NAD-dependent enzymes (Bonete *et al.*, 1987; Hayden *et al.*, 2002). The NAD-enzyme of *H. salinarum* has been characterized and is subject to activation by amino acids and inhibition by TCA cycle intermediates such as fumarate, oxalacetate, succinate and malate (Bonete *et al.*, 1996). The *Haloferax mediterranei* GDH has been also purified and is an NADP-specific enzyme composed of six identical monomers of 55 kDa (Ferrer *et al.*, 1996). In addition, a putative *gdhA* gene is present in the *Halobacterium sp.* genome (Ng *et al.*, 2000; Table 2). It is also remarkable that the amino acid sequence of the grapevine (*Vitis vinifera*) GDH is more similar to the archaeal GDH than to those of eukaryotes. A stress-related function of plant GDH has been invoked to explain this resemblance and the thermostability of the enzyme (Syntichaki *et al.*, 1996).

### Regulation of nitrogen metabolism in archaea

Denitrification and other anaerobic respiratory processes are alternative metabolic pathways developed by facultative micro-organisms to obtain energy for growth under anoxic conditions, and therefore they are usually repressed by oxygen (Moreno-Vivián & Ferguson, 1998). Regulation of anaerobic metabolism in *E. coli* and control of denitrification genes in *Pseudomonas* strains and other denitrifying bacteria are well studied (Zumft, 1997), but the mechanisms controlling nitrate respiration and denitrification in archaea are still not established. In addition, database searches reveal that homologues of different potential regulators involved in bacterial nitrate reduction, such as FNR, NNR, NorR, NarR, NarXL, ResDE and RegAB, are not present in the sequenced archaeal genomes, thus suggesting that completely novel regulatory systems could operate in archaea, probably as a consequence of the fundamental differences in the transcription machinery between bacteria and archaea. However, although most archaeal helix-turn-helix-containing transcription factors belong to archaea-specific families, it has recently been described that the molybdenum-responsive transcription factor ModE is present in some archaea. Moreover, the upstream ModE DNA-binding sequence seems to be conserved between bacteria and archaea (Studholme & Pau, 2003). These results indicate that Mo also regulates the expression of the genes required for archaeal molybdoproteins, and suggest either an ancient common origin for the ModE regulon, or more likely, horizontal gene transfer probably between green sulfur bacteria and methanogenic archaea (Studholme & Pau, 2003).

On the other hand, ammonium is usually the preferred nitrogen source in bacteria because it is easily assimilated with low energy cost. As other inorganic nitrogen sources, like nitrate, nitrite or N₂, must be reduced before assimilation, inorganic nitrogen assimilation is often repressed by ammonium.

Glutamine synthetase (GS) is highly regulated at both transcriptional and post-translational levels. In bacteria, GS is reversibly modified by adenylylation in response to the cellular nitrogen status, basically the glutamine/2-oxoglutarate ratio. An adenylyltransferase catalyses the adenylylation and deadenylylation of GS in response to the PⅡ protein, which in turn can be uridylylated or deuridylylated by a uridylyltransferase. When glutamine levels are high, deuridylylated PⅡ stimulates GS inactivation by adenylylation, whereas PⅡ-UMP promotes GS deadenylylation for activation when 2-oxoglutarate levels are high. PⅡ is also involved in transcriptional regulation of several nitrogen metabolism genes in response to the nitrogen status, acting on the general Ntr regulatory system or affecting specific regulators, such as the NifA protein that activates *nif* gene expression. PⅡ is encoded by the *glnB* gene, and *glnB* homologues are known in a variety of bacteria and archaea, and have been recently identified in higher plants, indicating that the GlnB family plays a central role in nitrogen regulatory processes in both prokaryotes and eukaryotes (Ninfa & Atkinson, 2000; Arcondéguy *et al.*, 2001). However, PⅡ proteins are not ubiquitous and several bacterial and archaeal genomes do not encode *glnB* genes. This is the case for *Aeropyrum pernix*, *Pyrococcus horikoshii* and *Pyrococcus abyssi* (Arcondéguy *et al.*, 2001). In addition, as mentioned above, archaeal GSs are not regulated by adenylylation, and control of GS activity in response to PⅡ proteins could be via a different mechanism than that in most bacteria. Many organisms contain a second PⅡ gene called *glnK*, which is located upstream from the *amtB* gene encoding an ammonium transporter in most prokaryotes,
including archaea, where this gene pair is found in multiple copies. This conserved gene organization indicates that the two genes were associated before the divergence of bacteria and archaea, and suggests that the GlnK regulatory protein and the ammonium transporter interact directly, probably for regulation of ammonium transport by GlnK (Thomas *et al.*, 2000). It has recently been demonstrated that GlnK binds to AmtB in the membrane and acts as a negative regulator of the transport activity of AmtB. This binding is dependent on GlnK uridylylation and is modulated according to the cellular nitrogen status, being maximal at high nitrogen (Coutts *et al.*, 2002; Javelle *et al.*, 2004). Therefore, this regulatory mechanism could be conserved throughout bacteria and archaea. Some archaeal genomes contain other *glnB*-like genes in addition to *glnB* and *glnK*. Thus, two different GlnB homologues are encoded in the *nif* gene clusters of diazotrophic methanogens (Chien & Zinder, 1996; Kessler & Leigh, 1999; Kessler *et al.*, 2001). In *Methanococcus maripaludis*, these *glnB*-like genes (termed *nifI₁* and *nifI₂*) are both required for ammonia switch-off of nitrogenase, probably acting through a novel reversible mechanism which does not require ADP-ribosylation of NifH protein and does not affect transcription of *nif* genes or stability of the NifH protein (Kessler *et al.*, 2001).

Transcription in archaea resembles the eukaryotic process more than the bacterial, and could be considered as a simplified version of the eukaryotic transcription complex (Bell & Jackson, 2001). In addition, although bacteria and archaea contain homologous regulatory proteins, the possibility that some regulators could be present only in archaea should not be excluded. In *Methanosarcina barkeri*, a putative TATA box is located in the *nifHDK2* promoter region that is transcribed like typical archaeal genes. Expression of nitrogenase structural genes is repressed by ammonium (Lobo & Zinder, 1990; Chien & Zinder, 1996) and ammonium-grown cells contain a putative protein that inhibits binding of transcription initiation proteins at the TATA box promoter region (Chien *et al.*, 1998). In *Methanococcus maripaludis*, expression of *nif* (nitrogenase) and *glnA* (glutamine synthetase) operons is repressed by ammonium (Kessler *et al.*, 1998; Cohen-Kupiec *et al.*, 1999) and a palindromic consensus sequence has been identified as a nitrogen operator for binding of a putative repressor which negatively regulates the *nif* and *glnA* operons in response to ammonia (Kessler & Leigh, 1999). Very recently, this nitrogen repressor has been identified in *M.* *maripaludis* and called NrpR (Lie & Leigh, 2003). NrpR is a tetramer of 60 kDa subunits with a putative N-terminal winged helix–turn–helix motif, which represents a new type of regulator exclusive to euryarchaeota (Lie & Leigh, 2003). C/N balance could regulate binding of the NrpR repressor to the *nif* promoter, because 2-oxoglutarate decreases the binding of NrpR to the nitrogen operator sequence. Therefore, archaea could use a different mechanism than bacteria for the control of nitrogen gene expression in response to the C/N balance (Lie & Leigh, 2003).


### Concluding remarks

Archaea can sustain the N-cycle in the extreme environments where they live because they are able to drive both dissimilatory and assimilatory processes of the cycle. In spite of this, archael inorganic nitrogen metabolism is much less known than bacterial nitrogen metabolism. Emerging data from recent functional studies and genome sequencing projects reveal important differences in the structure and regulation of enzymes and transport systems involved in the denitrification and nitrogen assimilation pathways, with interesting evolutionary implications. In particular, different regulatory mechanisms may operate in the control of nitrogen assimilation in archaea. Of special interest is whether or not transcriptional regulation in archaea is unique or possesses similarities to bacterial or eukaryotic regulation. Major goals of future studies will be to characterize at the molecular level the regulatory mechanisms acting on nitrogen metabolism in archaea, especially those for nitrate assimilation and denitrification. Functional genomic approaches should be also performed in order to identify the proteins and enzymes involved in the N-cycle pathways in the different archaea and their biochemical properties.


### Acknowledgements

The authors are grateful for the financial support of Ministerio de Ciencia y Tecnología (grant BMC2002-04126-CO3-03) and Junta de Andalucía (CVI 0117), Spain, and Alexander von Humboldt Foundation, Germany.


### References

Afshar, S., Kim, C., Monbouquette, H. G. & Schröder, I. (1998). Effect of tungstate on nitrate reduction by the hyperthermophilic archaeon *Pyrobaculum aerophilum*. *Appl Environ Microbiol* 64, 3004–3008.

Afshar, S., Johnson, E., de Vries, S. & Schröder, I. (2001). Properties of a thermostable nitrate reductase from the hyperthermophilic archaeon *Pyrobaculum aerophilum*. *J Bacteriol* 183, 5491–5495.

Álvarez-Ossorio, M. C., Muriana, F. J. G., de la Rosa, F. F. & Relimpio, A. M. (1992). Purification and characterization of nitrate reductase from the halophile archaebacterium *Haloferax mediterranei*. *Z Naturforsch* 47C, 670–676.

Antipov, A. N., Lyalikova, N. N., Khijniak, T. V. & L'vov, N. P. (1998). Molybdenum-free nitrate reductases from vanadate-reducing bacteria. *FEBS Lett* 441, 257–260.

Antipov, A. N., Sorokin, D. Y., L'vov, N. P. & Kuenen, J. G. (2003). New enzyme belonging to the family of molybdenum-free nitrate reductases. *Biochem J* 369, 185–189.

Arcondéguy, T., Jack, R. & Merrick, M. (2001). PII signal transduction proteins, pivotal players in microbial nitrogen control. *Microbiol Mol Biol Rev* 65, 80–105.

Belay, N., Sparling, R. & Daniels, L. (1984). Dinitrogen fixation by a thermophilic methanogenic bacterium. *Nature* 312, 286–288.

Bell, S. D. & Jackson, S. P. (2001). Mechanism and regulation of transcription in archaea. *Curr Opin Microbiol* 4, 208–213.

Berks, B. C., Ferguson, S. J., Moir, J. W. B. & Richardson, D. J. (1995). Enzymes and associated electron transport systems that catalyse the respiratory reduction of nitrogen oxides and oxyanions. *Biochem Biophys Acta* 1232, 97–173.

Berks, B. C., Sargent, F. & Palmer, T. (2000). The Tat protein export pathway. *Mol Microbiol* 35, 260–274.

Berman-Frank, I., Lundgren, P. & Falkowski, P. (2003). Nitrogen fixation and photosynthetic oxygen evolution in cyanobacteria. *Res Microbiol* 154, 157–164.

Bertero, M. G., Rothery, R. A., Palak, M., Hou, C., Lim, D., Blasco, F., Weiner, J. H. & Strynadka, N. C. (2003). Insights into the respiratory electron transfer pathway from the structure of nitrate reductase A. *Nat Struct Biol* 10, 681–687.

Bhatnagar, L., Zeikus, J. G. & Aubert, J. P. (1986). Purification and characterization of glutamine synthetase from the archaebacterium Methanobacterium ivanovi. *J Bacteriol* 165, 638–643.

Bhuiya, M. W., Sakuraba, H., Kujo, C., Nunoura-Kominato, N., Kawaraba, Y., Kikuchi, H. & Ohshima, T. (2000). Glutamate dehydrogenase from the aerobic hyperthermophilic archaeon Aeropyrum pernix K1: enzymatic characterization, identification of the encoding gene, and phylogenetic implications. *Extremophiles* 4, 333–341.

Bhuiya, M. W., Sakuraba, H. & Ohshima, T. (2002a). Temperature dependence of kinetic parameters for hyperthermophilic glutamate dehydrogenase from Aeropyrum pernix K1. *Biosci Biotechnol Biochem* 66, 873–876.

Bhuiya, M. W., Tsuge, H., Sakuraba, H., Yoneda, K., Katunuma, N. & Ohshima, T. (2002b). Crystallization and preliminary X-ray diffraction analysis of glutamate dehydrogenase from an aerobic hyperthermophilic archaeon, Aeropyrum pernix K1. *Acta Crystallogr D Biol Crystallogr* 58, 1338–1339.

Bickel-Sandkötter, S. & Ufer, M. (1995). Properties of a dissimilatory nitrate reductase from the halophilic archaeon Haloferax volcanii. *Z Naturforsch* 50C, 365–372.

Blakey, D., Leech, A., Thomas, G. H., Coutts, G., Findlay, K. & Merrick, M. (2002). Purification of the Escherichia coli ammonium transporter AmtB reveals a trimeric stoichiometry. *Biochem J* 364, 527–535.

Bonete, M. J., Camacho, M. L. & Cadenas, E. (1987). A new glutamate dehydrogenase from Halobacterium halobium with different coenzyme specificity. *Int J Biochem* 19, 1149–1155.

Bonete, M. J., Pérez-Pomares, F., Ferrer, J. & Camacho, M. L. (1996). NAD-glutamate dehydrogenase from Halobacterium halobium: inhibition and activation by TCA intermediates and amino acids. *Biochim Biophys Acta* 1289, 14–24.

Brown, J. R. & Doolittle, W. F. (1997). Archaea and the prokaryote-to-eukaryote transition. *Microbiol Mol Biol Rev* 61, 456–502.

Brown, J. R., Masuchi, Y., Robb, F. T. & Doolittle, W. F. (1994). Evolutionary relationships of bacterial and archael glutamine synthetase genes. *J Mol Evol* 38, 566–576.

Brown, K., Djinovic-Carugo, K., Haltia, T., Cabrito, I., Saraste, M., Moura, J. J. G., Moura, I., Tegoni, M. & Cambillau, C. (2000). Revisiting the catalytic CuZ cluster of nitrous oxide (N₂O) reductase. Evidence of a bridging inorganic sulfur. *J Biol Chem* 275, 41133–41136.

Buc, J., Santini, C. L., Giordani, R., Czjzek, M., Wu, L. F. & Giordano, G. (1999). Enzymatic and physiological properties of the tungsten-substituted molybdenum TMAO reductase from Escherichia coli. *Mol Microbiol* 32, 159–168.

Bult, C. J., White, O., Olsen, G. J. & 20 other authors (1996). Complete genome sequence of the methanogenic archaeon, Methanococcus jannaschii. *Science* 273, 1058–1073.

Campbell, W. H. (2001). Structure and function of eukaryotic NAD(P)H:nitrate reductase. *Cell Mol Life Sci* 58, 194–204.

Castillo, F., Dobao, M. M., Reyes, F., Blasco, R., Roldán, M. D., Gavira, M., Caballero, F. J., Moreno-Vivián, C. & Martínez-Luque, M. (1996). Molecular and regulatory properties of the nitrate reducing systems of Rhodobacter. *Curr Microbiol* 33, 341–346.

Castresana, J. & Moreira, D. (1999). Respiratory chains in the last common ancestor of living organisms. *J Mol Evol* 49, 453–460.

Chèneby, D., Hallet, S., Mondon, M., Martin-Laurent, F., Germon, J. C. & Philippot, L. (2003). Genetic characterization of the nitrate reducing community based on narG nucleotide sequence analysis. *Microb Ecol* 46, 113–121.

Chien, Y. T. & Zinder, S. H. (1996). Cloning, functional organization, transcript studies, and phylogenetic analysis of the complete nitrogenase structural genes (nifHDK2) and associated genes in the archaeon Methanosarcina barkeri 227. *J Bacteriol* 178, 143–148.

Chien, Y. T., Helmann, J. D. & Zinder, S. H. (1998). Interactions between the promoter regions of nitrogenase structural genes (nifHDK2) and DNA-binding proteins from N₂- and ammonium-grown cells of the archaeon Methanosarcina barkeri 227. *J Bacteriol* 180, 2723–2728.

Chien, Y. T., Auerbuch, V., Brabban, A. D. & Zinder, S. H. (2000). Analysis of genes encoding an alternative nitrogenase in the archaeon Methanosarcina barkeri 227. *J Bacteriol* 182, 3247–3253.

Cohen-Kupiec, R., Marx, C. J. & Leigh, J. A. (1999). Function and regulation of glnA in the methanogenic archaeon Methanococcus maripaludis. *J Bacteriol* 181, 256–261.

Coutts, G., Thomas, G., Blakey, D. & Merrick, M. (2002). Membrane sequestration of the signal transduction protein GlnK by the ammonium transporter AmtB. *EMBO J* 21, 536–545.

Dalsgaard, T., Canfield, D. E., Petersen, J., Thamdrup, B. & Acuña-González, J. (2003). N₂ production by the anammox reaction in the anoxic water column of Golfo Dulce, Costa Rica. *Nature* 422, 606–608.

de Boer, W. & Kowalchuk, G. A. (2001). Nitrification in acid soils: microorganisms and mechanisms. *Soil Biol Biochem* 33, 853–866.

Deppeemeier, U., Lienard, T. & Gottschalk, G. (1999). Novel reactions involved in energy conservation by methanogenic archaea. *FEBS Lett* 457, 291–297.

Deppeemeier, U., Johann, A., Hartsch, T. & 19 other authors (2002). The genome of Methanosarcina mazei: evidence for lateral gene transfer between bacteria and archaea. *J Mol Microbiol Biotechnol* 4, 453–461.

de Vries, S. & Schröder, I. (2002). Comparison between the nitric oxide reductase family and its aerobic relatives, the cytochrome oxidases. *Biochem Soc Trans* 30, 662–667.

Dias, J. M., Than, M. E., Humm, A. & 10 other authors (1999). Crystal structure of the first dissimilatory nitrate reductase at 1·9 Å solved by MAD methods. *Structure* 7, 65–79.

DiRuggiero, J. & Robb, F. T. (1995). Expression and in vitro assembly of recombinant glutamate dehydrogenase from the hyperthermophilic archaeon Pyrococcus furiosus. *Appl Environ Microbiol* 61, 159–164.

DiRuggiero, J. & Robb, F. T. (1996). Enzymes of central nitrogen metabolism from hyperthermophiles: characterization, thermostability, and genetics. *Adv Protein Chem* 48, 311–339.

DiRuggiero, J., Dunn, D., Maeder, D. L., Holley-Shanks, R., Chatard, J., Horlacher, R., Robb, F. T., Boos, W. & Weiss, R. B. (2000). Evidence of recent lateral gene transfer among hyperthermophilic archaea. *Mol Microbiol* 38, 684–693.

P. Cabello, M. D. Roldán and C. Moreno-Vivián

Eady, R. R. (1996). Structure–function relationships of alternative nitrogenases. *Chem Rev* 96, 3013–3030.

Eisenberg, D., Gill, H. S., Pfluegl, G. M. U. & Rotstein, S. H. (2000). Structure–function relationships of glutamine synthetases. *Biochim Biophys Acta* 1477, 122–145.

Ellington, M. J. K., Richardson, D. J. & Ferguson, S. J. (2003). *Rhodobacter capsulatus* gains a competitive advantage from respiratory nitrate reduction during light–dark transitions. *Microbiology* 149, 941–948.

Fani, R., Gallo, R. & Liò, P. (2000). Molecular evolution of nitrogen fixation: the evolutionary history of the *nifD*, *nifK*, *nifE*, and *nifN* genes. *J Mol Evol* 51, 1–11.

Ferguson, S. J. (1998). Nitrogen cycle enzymology. *Curr Opin Chem Biol* 2, 182–193.

Ferrer, J., Pérez-Pomares, F. & Bonete, M. J. (1996). NADP-glutamate dehydrogenase from the halophilic archaeon *Haloferax mediterranei*: enzyme purification, N-terminal sequence and stability. *FEMS Microbiol Lett* 141, 59–63.

Fitz-Gibbon, S. T., Ladner, H., Kim, U. J., Stetter, K. O., Simon, M. I. & Miller, J. H. (2002). Genome sequence of the hyperthermophilic crenarchaeon *Pyrobaculum aerophilum*. *Proc Natl Acad Sci USA* 99, 984–989.

Galagan, J. E., Nusbaum, C., Roy, A. & 52 other authors (2002). The genome of *M. acetivorans* reveals extensive metabolic and physiological diversity. *Genome Res* 12, 532–542.

Gates, A. J., Hughes, R. O., Sharp, S. R. & 7 other authors (2003). Properties of the periplasmic nitrate reductases from *Paracoccus pantotrophus* and *Escherichia coli* after growth in tungsten-supplemented media. *FEMS Microbiol Lett* 220, 261–269.

Gavira, M., Roldán, M. D., Castillo, F. & Moreno-Vivián, C. (2002). Regulation of *nap* gene expression and periplasmic nitrate reductase activity in the phototrophic bacterium *Rhodobacter sphaeroides* DSM158. *J Bacteriol* 184, 1693–1702.

Gomes, C. M., Giuffrè, A., Forte, E., Vicente, J., Saraiva, L., Brunori, M. & Teixeira, M. (2002). A novel type of nitric-oxide reductase: *Escherichia coli* flavoredoxin. *J Biol Chem* 277, 25273–25276.

Gregory, L. G., Bond, P. L., Richardson, D. J. & Spiro, S. (2003). Characterization of a nitrate-respiring bacterial community using the nitrate reductase gene (*narG*) as a functional marker. *Microbiology* 149, 229–237.

Halbleib, C. M. & Ludden, P. W. (2000). Regulation of biological nitrogen fixation. *J Nutr* 130, 1081–1084.

Hayden, B. M., Bonete, M. J., Brown, P. E., Moir, A. J. G. & Engel, P. C. (2002). Glutamate dehydrogenase of *Halobacterium salinarum*: evidence that the gene sequence currently assigned to the NADP⁺-dependent enzyme is in fact that of the NAD⁺-dependent glutamate dehydrogenase. *FEMS Microbiol Lett* 211, 37–41.

Helianti, I., Morita, Y., Yamamura, A., Murakami, Y., Yokoyama, K. & Tamiya, E. (2001). Characterization of native glutamate dehydrogenase from an aerobic hyperthermophilic archaeon *Aeropyrum pernix* K1. *Appl Microbiol Biotechnol* 56, 388–394.

Helianti, I., Morita, Y., Murakami, Y., Yokoyama, K. & Tamiya, E. (2002). Expression of two kinds of recombinant glutamate dehydrogenase from *Aeropyrum pernix* with different N-terminal sequence length in *Escherichia coli*. *Appl Microbiol Biotechnol* 59, 462–466.

Hendriks, J., Oubrie, A., Castresana, J., Urbani, A., Gemeinhardt, S. & Saraste, M. (2000). Nitric oxide reductases in bacteria. *Biochim Biophys Acta* 1459, 266–273.

Hille, R. (2002). Molybdenum and tungsten in biology. *Trends Biochem Sci* 27, 360–367.

Hochstein, L. I. & Lang, F. (1991). Purification and properties of a dissimilatory nitrate reductase from *Haloferax denitrificans*. *Arch Biochem Biophys* 288, 380–385.

Howard, J. B. & Rees, D. C. (1996). Structural basis of biological nitrogen fixation. *Chem Rev* 96, 2965–2982.

Ichiki, H., Tanaka, Y., Mochizuki, K., Yoshimatsu, K., Sakurai, T. & Fujiwara, T. (2001). Purification, characterization, and genetic analysis of Cu-containing dissimilatory nitrite reductase from a denitrifying halophilic archaeon, *Haloarcula marismortui*. *J Bacteriol* 183, 4149–4156.

Inatomi, K. & Hochstein, L. I. (1996). The purification and properties of a copper nitrite reductase from *Haloferax denitrificans*. *Curr Microbiol* 32, 72–76.

Javelle, A., Severi, E., Thornton, J. & Merrick, M. (2004). Ammonium sensing in *Escherichia coli*. Role of the ammonium transporter AmtB and AmtB-GlnK complex formation. *J Biol Chem* 279, 8530–8538.

Jongsareejit, B., Rahman, R. N. Z. A., Fujiwara, S. & Imanaka, T. (1997). Gene cloning, sequencing and enzymatic properties of glutamate synthase from the hyperthermophilic archaeon *Pyrococcus sp.* KOD1. *Mol Gen Genet* 254, 635–642.

Jormakka, M., Byrne, B. & Iwata, S. (2003). Protonmotive force generation by a redox loop mechanism. *FEBS Lett* 545, 25–30.

Jormakka, M., Richardson, D. J., Byrne, B. & Iwata, S. (2004). Architecture of NarGH reveals a structural classification of Mo-bisMGD enzymes. *Structure* 12, 95–104.

Kabayashi, T., Higuchi, S., Kimura, K., Kudo, T. & Horikoshi, K. (1995). Properties of glutamate dehydrogenase and its involvement in alanine production in a hyperthermophilic archaeon, *Thermococcus profundus*. *J Biochem* 118, 587–592.

Kawarabayasi, Y., Hino, Y., Horikawa, H. & 25 other authors (1999). Complete genome sequence of an aerobic hyper-thermophilic crenarchaeon, *Aeropyrum pernix* K1. *DNA Res* 6, 83–101.

Kawarabayasi, Y., Hino, Y., Horikawa, H. & 27 other authors (2001). Complete genome sequence of an aerobic thermoacidophilic crenarchaeon, *Sulfolobus tokodaii* strain7. *DNA Res* 8, 123–140.

Kawashima, T., Amano, N., Koike, H. & 12 other authors (2000). Archaeal adaptation to higher temperatures revealed by genomic sequence of *Thermoplasma volcanium*. *Proc Natl Acad Sci USA* 97, 14257–14262.

Kelman, L. M. & Kelman, Z. (2003). Archaea: an archetype for replication initiation studies? *Mol Microbiol* 48, 605–615.

Kessler, P. S. & Leigh, J. A. (1999). Genetics of nitrogen regulation in *Methanococcus maripaludis*. *Genetics* 152, 1343–1351.

Kessler, P. S., McLarnan, J. & Leigh, J. A. (1997). Nitrogenase phylogeny and the molybdenum dependence of nitrogen fixation in *Methanococcus maripaludis*. *J Bacteriol* 179, 541–543.

Kessler, P. S., Blank, C. & Leigh, J. A. (1998). The *nif* gene operon of the methanogenic archaeon *Methanococcus maripaludis*. *J Bacteriol* 180, 1504–1511.

Kessler, P. S., Daniel, C. & Leigh, J. A. (2001). Ammonia switch-off of nitrogen fixation in the methanogenic archaeon *Methanococcus maripaludis*: mechanistic features and requirement for the novel GlnB homologues, NifI₁ and NifI₂. *J Bacteriol* 183, 882–889.

Klenk, H. P., Clayton, R. A., Tomb, J. F. & 48 other authors (1997). The complete genome sequence of the hyperthermophilic, sulphate-reducing archaeon *Archaeoglobus fulgidus*. *Nature* 390, 364–370.

Kowalchuk, G. A. & Stephen, J. R. (2001). Ammonia-oxidizing bacteria: a model for molecular microbial ecology. *Annu Rev Microbiol* 55, 485–529.

Kuypers, M. M. M., Sliekers, A. O., Lavik, G., Schmid, M., Jørgensen, B. B., Kuenen, J. G., Damsté, J. S. S., Strous, M. & Jetten, M. S. M.
(2003). Anaerobic ammonium oxidation by anammox bacteria in the Black Sea. *Nature* 422, 608–611.

Leigh, J. A. (2000). Nitrogen fixation in methanogens: the archaeal perspective. *Curr Issues Mol Biol* 2, 125–131.

Lie, T. J. & Leigh, J. A. (2003). A novel repressor of *nif* and *glnA* expression in the methanogenic archaeon *Methanococcus maripaludis*. *Mol Microbiol* 47, 235–246.

Lobo, A. L. & Zinder, S. H. (1990). Nitrogenase in the archaeabacterium *Methanosarcina barkeri* 227. *J Bacteriol* 172, 6789–6796.

Lubben, M. & Morand, K. (1994). Novel prenylated hemes as cofactors of cytochrome oxidases. Archaea have modified hemes a and O. *J Biol Chem* 269, 21473–21479.

Ludden, P. W. (1994). Reversible ADP-ribosylation as a mechanism of enzyme regulation in procaryotes. *Mol Cell Biochem* 138, 123–129.

Maeder, D. L., Weiss, R. B., Dunn, D. M., Cherry, J. L., González, J. M., DiRuggiero, J. & Robb, F. T. (1999). Divergence of the hyperthermophilic archaea *Pyrococcus furiosus* and *P. horikoshii* inferred from complete genomic sequences. *Genetics* 152, 1299–1305.

Martínez-Espinosa, R. M., Marhuenda-Egea, F. C. & Bonete, M. J. (2001a). Purification and characterization of a possible assimilatory nitrite reductase from the halophile archaeon *Haloferax mediterranei*. *FEMS Microbiol Lett* 196, 113–118.

Martínez-Espinosa, R. M., Marhuenda-Egea, F. C. & Bonete, M. J. (2001b). Assimilatory nitrate reductase from the haloarchaeon *Haloferax mediterranei*: purification and characterization. *FEMS Microbiol Lett* 204, 381–385.

Martínez-Espinosa, R. M., Marhuenda-Egea, F. C., Donaire, A. & Bonete, M. J. (2003). NMR studies of a ferredoxin from *Haloferax mediterranei* and its physiological role in nitrate assimilatory pathway. *Biochim Biophys Acta* 1623, 47–51.

Mehta, M. P., Butterfield, D. A. & Baross, J. A. (2003). Phylogenetic diversity of nitrogenase (*nifH*) genes in deep-sea and hydrothermal vent environments of the Juan de Fuca Ridge. *Appl Environ Microbiol* 69, 960–970.

Merrick, M. J. & Edwards, R. A. (1995). Nitrogen control in bacteria. *Microbiol Rev* 59, 604–622.

Moir, J. W. B. & Wood, N. J. (2001). Nitrate and nitrite transport in bacteria. *Cell Mol Life Sci* 58, 215–224.

Moreno-Vivián, C. & Ferguson, S. J. (1998). Definition and distinction between assimilatory, dissimilatory and respiratory pathways. *Mol Microbiol* 29, 664–666.

Moreno-Vivián, C., Schmehl, M., Masepohl, B., Arnold, W. & Klipp, W. (1989). DNA sequence and genetic analysis of the *Rhodobacter capsulatus nifENX* gene region: homology between NifX and NifB suggests involvement of NifX in processing of the iron-molybdenu cofactor. *Mol Gen Genet* 216, 353–363.

Moreno-Vivián, C., Cabello, P., Martínez-Luque, M., Blasco, R. & Castillo, F. (1999). Prokaryotic nitrate reduction: molecular properties and functional distinction among bacterial nitrate reductases. *J Bacteriol* 181, 6573–6584.

Murillo, M. F., Gugliuzza, T., Senko, J., Basu, P. & Stolz, J. F. (1999). A heme c-containing enzyme complex that exhibits nitrate and nitrite reductase activity from the dissimilatory iron-reducing bacterium *Geobacter metallireducens*. *Arch Microbiol* 172, 313–320.

Murray, P. A. & Zinder, S. H. (1984). Nitrogen fixation by a methanogenic archaebacterium. *Nature* 312, 284–286.

Nakahara, K., Tanimoto, T., Hatano, K., Usuda, K. & Shoun, H. (1993). Cytochrome P-450 55A1 (P-450dNIR) acts as nitric oxide reductase employing NADH as the direct electron donor. *J Biol Chem* 268, 8350–8355.

Ng, W. V., Kennedy, S. P., Mahairas, G. G. & 40 other authors (2000). Genome sequence of *Halobacterium* species NRC-1. *Proc Natl Acad Sci U S A* 97, 12176–12181.

Ninfa, A. J. & Atkinson, M. R. (2000). PII signal transduction proteins. *Trends Microbiol* 8, 172–179.

Pesole, G., Gissi, C., Lanave, C. & Saccone, C. (1995). Glutamine synthetase gene evolution in bacteria. *Mol Biol Evol* 12, 189–197.

Petri, R. & Imhoff, J. F. (2000). The relationship of nitrate reducing bacteria on the basis of *narH* gene sequences and comparison of *narH* and 16S rDNA based phylogeny. *Syst Appl Microbiol* 23, 47–57.

Philipot, L. (2002). Denitrifying genes in bacterial and archaeal genomes. *Biochim Biophys Acta* 1577, 355–376.

Potter, L., Angove, H., Richardson, D. J. & Cole, J. A. (2001). Nitrate reduction in the periplasm of Gram-negative bacteria. *Adv Microb Physiol* 45, 51–112.

Rahman, R. N. Z. A., Jongsareejit, B., Fujiwara, S. & Imanaka, T. (1997). Characterization of recombinant glutamine synthetase from the hyperthermophilic archaeon *Pyrococcus* sp. strain KOD1. *Appl Environ Microbiol* 63, 2472–2476.

Ramírez, S., Moreno, R., Zafra, O., Castán, P., Vallés, C. & Berenguer, J. (2000). Two nitrate/nitrite transporters are encoded within the mobilizable plasmid for nitrate respiration of *Thermus thermophilus* HB8. *J Bacteriol* 182, 2179–2183.

Ramírez-Arcos, S., Fernández-Herrero, L. A. & Berenguer, J. (1998a). A thermophilic nitrate reductase is responsible for the strain specific anaerobic growth of *Thermus thermophilus* HB8. *Biochim Biophys Acta* 1396, 215–227.

Ramírez-Arcos, S., Fernández-Herrero, L. A., Marín, I. & Berenguer, J. (1998b). Anaerobic growth, a property horizontally transferred by an Hfr-like mechanism among extreme thermophiles. *J Bacteriol* 180, 3137–3143.

Rasmussen, T., Berks, B. C., Sanders-Loehr, J., Dooley, D. M., Zumft, W. G. & Thomson, A. J. (2000). The catalytic center in nitrous oxide reductase, CuZ, is a copper-sulfide cluster. *Biochemistry* 39, 12753–12756.

Rees, D. C. (2002). Great metalloclusters in enzymology. *Annu Rev Biochem* 71, 221–246.

Reeve, J. N. (2003). Archaeal chromatin and transcription. *Mol Microbiol* 48, 587–598.

Reyes, F., Roldán, M. D., Klipp, W., Castillo, F. & Moreno-Vivián, C. (1996). Isolation of periplasmic nitrate reductase genes from *Rhodobacter sphaeroides* DSM 158: structural and functional differences among prokaryotic nitrate reductases. *Mol Microbiol* 19, 1307–1318.

Reyes, F., Gavira, M., Castillo, F. & Moreno-Vivián, C. (1998). Periplasmic nitrate-reducing system of the phototrophic bacterium *Rhodobacter sphaeroides* DSM 158: transcriptional and mutational analysis of the *napKEFDABC* gene cluster. *Biochem J* 331, 897–904.

Richardson, D. J. (2000). Bacterial respiration: a flexible process for a changing environment. *Microbiology* 146, 551–571.

Richardson, D. J. & Watmough, N. J. (1999). Inorganic nitrogen metabolism in bacteria. *Curr Opin Chem Biol* 3, 207–219.

Richardson, D. J., Berks, B. C., Russell, D. A., Spiro, S. & Taylor, C. J. (2001). Functional, biochemical and genetic diversity of prokaryotic nitrate reductases. *Cell Mol Life Sci* 58, 165–178.

Ruepp, A., Graml, W., Santos-Martínez, M. L. & 7 other authors (2000). The genome sequence of the thermoacidophilic scavenger *Thermoplasma acidophilum*. *Nature* 407, 508–513.

Sapra, R., Bagramyan, K. & Adams, M. W. W. (2003). A simple energy-conserving system: proton reduction coupled to proton translocation. *Proc Natl Acad Sci U S A* 100, 7545–7550.
P. Cabello, M. D. Roldán and C. Moreno-Vivián

Saraste, M. (1994). Structure and evolution of cytochrome oxidase. Antonie Van Leeuwenhoek 65, 285–287.

Saraste, M. & Castresana, J. (1994). Cytochrome oxidase evolved by tinkering with denitrification enzymes. FEBS Lett 341, 1–4.

Scharf, B. & Engelhard, M. (1993). Halocyanin, an archaeal blue copper protein (type I) from Natronobacterium pharaonis. Biochemistry 32, 12894–12900.

Schütz, M., Brugna, M., Lebrun, E. & 9 other authors (2000). Early evolution of cytochrome bc complexes. J Mol Biol 300, 663–675.

She, Q., Singh, R. K., Confalonieri, F. & 28 other authors (2001). The complete genome of the crenarchaeon Sulfolobus solfataricus P2. Proc Natl Acad Sci U S A 98, 7835–7840.

Shoun, H. & Tanimoto, T. (1991). Denitrification by the fungus Fusarium oxysporum and involvement of cytochrome P-450 in the respiratory nitrite reduction. J Biochem 266, 11078–11082.

Slesarev, A. I., Mezhevaya, K. V., Makarova, K. S. & 13 other authors (2002). The complete genome of hyperthermophile Methanopyrus kandleri AV19 and monophyly of archaeal methanogens. Proc Natl Acad Sci U S A 99, 4644–4649.

Smith, E. L., Austen, B. M., Blumenthal, K. M. & Nyc, J. F. (1975). Glutamate dehydrogenases. In The Enzymes, vol. XI, pp. 293–367. Edited by P. D. Boyer. New York: Academic Press.

Smith, D. R., Doucette-Stamm, L. A., Deloughery, C. & 34 other authors (1997). Complete genome sequence of Methanobacterium thermoautotrophicum ΔH: functional analysis and comparative genomics. J Bacteriol 179, 7135–7155.

Soupene, E., Lee, H. & Kustu, S. (2002). Ammonium/methylammonium transport (Amt) proteins facilitate diffusion of NH₃ bidirectionally. Proc Natl Acad Sci U S A 99, 3926–3931.

Stolz, J. F. & Basu, P. (2002). Evolution of nitrate reductase: molecular and structural variations on a common function. Chembiochem 3, 198–206.

Studholme, D. J. & Pau, R. N. (2003). A DNA element recognized by the molybdenum-responsive transcription factor ModE is conserved in Proteobacteria, green sulphur bacteria and Archaea. BMC Microbiology 3, 24–33.

Syntichaki, K. M., Loulakakis, K. A. & Roubelakis-Angelakis, K. A. (1996). The amino-acid sequence similarity of plant glutamate dehydrogenase to the extremophilic archaeal enzyme conforms to its stress-related function. Gene 168, 87–92.

Thomas, G., Coutts, G. & Merrick, M. (2000). The glnKamtB operon: a conserved gene pair in prokaryotes. Trends Genet 16, 11–14.

Tiboni, O., Cammarano, P. & Sanangelantoni, A. M. (1993). Cloning and sequencing of the gene encoding glutamine synthetase I from the archaeum Pyrococcus woesei: anomalous phylogenies inferred from analysis of archaeal and bacterial glutamine synthetase I sequences. J Bacteriol 175, 2961–2969.

Tomlinson, G. A., Jahnke, L. L. & Hochstein, L. I. (1986). Halobacterium denitrificans sp. nov., an extremely halophilic denitrifying bacterium. Int J Syst Bacteriol 36, 66–70.

Trotta, P. P., Platzer, K. E. B., Haschemeyer, R. H. & Meister, A. (1974). Glutamine-binding subunit of glutamate synthase and partial reactions catalyzed by this glutamine amidotransferase. Proc Natl Acad Sci U S A 71, 4607–4611.

Völk, P., Huber, R., Drobner, E., Rachel, R., Burggraf, S., Tricone, A. & Stetter, K. O. (1993). Pyrobaculum aerophilum sp. nov., a novel nitrate-reducing hyperthermophilic archaeum. Appl Environ Microbiol 59, 2918–2926.

Vorholt, J. A., Hafenbradl, D., Stetter, K. O. & Thauer, R. K. (1997). Pathways of autotrophic CO₂ fixation and of dissimilatory nitrate reduction to N₂O in Ferroglobus placidus. Arch Microbiol 167, 19–23.

Wang, S., Feng, Y., Zhang, Z., Zheng, B., Li, N., Cao, S., Matsui, I. & Kosugi, Y. (2003). Heat effect on the structure and activity of the recombinant glutamate dehydrogenase from a hyperthermophilic archaeon Pyrococcus horikoshii. Arch Biochem Biophys 411, 56–62.

Wanner, C. & Soppa, J. (1999). Genetic identification of three ABC transporters as essential elements for nitrate respiration in Haloferax volcanii. Genetics 152, 1417–1428.

Werber, M. M. & Mevarech, M. (1978). Induction of a dissimilatory reduction pathway of nitrate in Halobacterium of the Dead Sea. A possible role for the 2Fe-ferredoxin isolated from this organism. Arch Biochem Biophys 186, 60–65.

Williams, R. J. P. & Fraústo da Silva, J. J. R. (2002). The involvement of molybdenum in life. Biochem Biophys Res Comm 292, 293–299.

Woese, C. R., Kandler, O. & Wheelis, M. L. (1990). Towards a natural system of organisms: proposal for the domains Archaea, Bacteria, and Eucarya. Proc Natl Acad Sci U S A 87, 4576–4579.

Wood, N. J., Alizadeh, T., Bennet, S., Pearce, J., Ferguson, S. J., Richardson, D. J. & Moir, J. W. (2001). Maximal expression of membrane-bound nitrate reductase in Paracoccus is induced by nitrate via a third FNR-like regulator named NarR. J Bacteriol 183, 3606–3613.

Wood, N. J., Alizadeh, T., Richardson, D. J., Ferguson, S. J. & Moir, J. W. (2002). Two domains of a dual-function NarK protein are required for nitrate uptake, the first step of denitrification in Paracoccus pantotrophus. Mol Microbiol 44, 157–170.

Yin, Z., Purschke, W. G., Schafer, G. & Schmidt, C. L. (1998). The glutamine synthetase from the hyperthermoacidophilic crenarchaeon Sulfolobus acidocaldarius: isolation, characterization and sequencing of the gene. Biol Chem 379, 1349–1354.

Yip, K. S., Stillman, T. J., Britton, K. L. & 7 other authors (1995). The structure of Pyrococcus furiosus glutamate dehydrogenase reveals a key role for iron-pair networks in maintaining enzyme stability at extreme temperatures. Structure 3, 1147–1158.

Yoshimatsu, K., Sakurai, T. & Fujiwara, T. (2000). Purification and characterization of dissimilatory nitrate reductase from a denitrifying halophilic archaeon, Haloarcula marismortui. FEBS Lett 470, 216–220.

Yoshimatsu, K., Iwasaki, Y. & Fujiwara, T. (2002). Sequence and electron paramagnetic resonance analysis of nitrate reductase NarGH from a denitrifying halophilic euryarchaeote Haloarcula marismortui. FEBS Lett 516, 145–150.

Zumft, W. G. (1997). Cell biology and molecular basis of denitrification. Microbiol Mol Biol Rev 61, 533–616.

Zumft, W. G. & Castillo, F. (1978). Regulatory properties of the nitrogenase from Rhodopseudomonas palustris. Arch Microbiol 117, 53–60.
