
Available online at www.sciencedirect.com

**SCIENCE DIRECT**

Biochimica et Biophysica Acta 1694 (2004) 149–161

Review

**Type I secretion in gram-negative bacteria**

P. Delepelaire*

Unité des Membranes Bactériennes, Département de Microbiologie fondamentale et médicale, Institut Pasteur,  
25-28, rue du Dr. Roux, 75724 Paris, Cedex 15, France

Received 13 January 2004; received in revised form 3 May 2004; accepted 5 May 2004  
Available online 31 May 2004

---

### Abstract

In gram-negative bacteria, type I secretion is carried out by a translocator made up of three proteins that span the cell envelope. One of these proteins is a specific outer membrane protein (OMP) and the other two are cytoplasmic membrane proteins: an ATP-binding cassette (ABC) and the so-called membrane fusion or adaptor protein (MFP). Type I secretion is sec-independent and bypasses the periplasm. This widespread pathway allows the secretion of proteins of diverse sizes and functions via a C-terminal uncleaved secretion signal. This C-terminal secretion signal specifically recognizes the ABC protein, triggering the assembly of the functional trans-envelope complex. This report will mainly deal with recent data concerning the structure and assembly of the secretion complex as well as the effects and role of substrate folding on secretion by this pathway.

© 2004 Elsevier B.V. All rights reserved.

Keywords: Type I secretion; Gram-negative; Structure

---

### 1. Introduction

The type I secretion system in gram-negative bacteria allows the secretion of proteins of various sizes and functions from the cytoplasm to the extracellular medium in a single step, without a stable periplasmic intermediate. The secretion signal is usually located at the C-terminal end of the secreted protein and is not cleaved during secretion. Many secreted proteins have distinctive glycine-rich repeats (GGXGXDXXX) that specifically bind calcium ions (COG2931 in the cluster of orthologous genes (COG) database) [1]. Most, if not all, of the gene products containing these repeats are secreted by the type I pathway. Most secreted proteins contain between a few and more than 50 such repeats, which form peculiar beta-sandwich or beta-roll structures with calcium ions in the

turns [1]. Several studies have shown that these repeats are necessary for the activity of the secreted protein. Some proteins secreted by this pathway do not contain any repeats (see below, HasA case), whereas others contain different kinds of repeats also binding calcium [2,3].

Table 1 shows an analysis of completed bacterial genomes, showing the proteins having this kind of repeats (GGXGXDXXX); it is inferred that these putative proteins will be secreted by type I secretion systems. These proteins have been found up to now only in cyanobacteria and in all classes of proteobacteria. Several remarks can be made: (i) Proteins vary greatly in size, from 78 to 8682 residues. (ii) All of them (with only one exception) are very acidic proteins with pI around 4. (iii) A few species appear exceptional by the number of putative candidates, like Bradyrhizobium japonicum, Nostoc, Ralstonia solanacearum, Sinorhizobium meliloti, Synechococcus 8102. (iv) Those putative proteins also contain very few or no cysteine again with one exception. (v) Many proteins, besides the identified glycine-rich repeats, have other sort of repeats which are often associated, by homology, with adhesion molecules. In one case this has been clearly shown for the largest LapA protein

---

Abbreviations: ABC, ATP-binding cassette; OMP, outer membrane protein; MFP, membrane fusion protein; COG, cluster of orthologous genes; NBD, nucleotide-binding domain; TMD, transmembrane domain; RTX, repeats in ToXin; RND, resistance, nodulation, division

* Tel.: +33-1-40-61-32-76; fax: +33-1-45-68-87-90.  
E-mail address: pdelep@pasteur.fr (P. Delepelaire).

0167-4889/$ - see front matter © 2004 Elsevier B.V. All rights reserved.  
doi:10.1016/j.bbamcr.2004.05.001

Table 1  
Analysis of glycine-rich repeat ORFs from completed bacterial genomes  

| Genebank accession | Function, putative | Residues | MW(Da) | pI (theor) | Cysteine | Repeats |
|--------------------|-------------------|----------|--------|------------|----------|---------|
| Aquifex aeolicus VF 5<br>gi|15606363|ref|NP_213742.1| | ? | 398 | 40290 | 4.49 | 0 | 7 |
| Agrobacterium tumefaciens C 58<br>gi|15889960|ref|NP_355641.1| | ? | 317 | 32059 | 4.48 | 0 | 3 |
| gi|15891549|ref|NP_357221.1| | ? | 251 | 25695 | 3.90 | 0 | 2 |
| gi|15891377|ref|NP_357049.1| | endoglycanase | 465 | 49920 | 3.90 | 0 | 3 |
| gi|15890570|ref|NP_356242.1| | esterase? | 1944 | 204182 | 4.30 | 0 | 15 |
| Bordetella bronchiseptica RB 50<br>gi|33599313|ref|NP_886873.1| | cyclase | 1740 | 181075 | 4.62 | 1 | 25 |
| gi|33600172|ref|NP_887732.1| | adhesin? | 3346 | 340069 | 3.90 | 0 | 10+ |
| Bordetella parapertussis 12822<br>gi|33595034|ref|NP_882677.1| | cyclase | 1740 | 181059 | 4.62 | 1 | 25 |
| gi|33595651|ref|NP_883294.1| | adhesin? | 2215 | 226545 | 3.90 | 0 | 6+ |
| Bordetella pertussis Tohama I<br>gi|33591934|ref|NP_879578.1| | cyclase | 1706 | 177414 | 4.58 | 0 | 25 |
| Bradyrhizobium japonicum USDA 110<br>gi|27375246|ref|NP_766775.1| | ? | 227 | 23177 | 6.87 | 1 | 4 |
| gi|27378220|ref|NP_769749.1| | ? | 917 | 90126 | 3.90 | 0 | 30 |
| gi|27378674|ref|NP_770203.1| | ? cadherin | 3441 | 342411 | 3.90 | 1 | 10+ |
| gi|27378825|ref|NP_770354.1| | ? cadherin | 4128 | 409470 | 3.90 | 0 | 30+ |
| gi|27380582|ref|NP_772111.1| | adhesin? | 4210 | 428461 | 3.90 | 2 | 2+ |
| gi|27380932|ref|NP_772461.1| | ? | 1756 | 177767 | 4.27 | 0 | 6+ |
| gi|27381138|ref|NP_772667.1| | metalloprotease-phosphatase | 844 | 86434 | 4.09 | 0 | 6+ |
| gi|27381146|ref|NP_772675.1| | adhesin? | 1315 | 133935 | 3.98 | 1 | 3+ |
| gi|27381146|ref|NP_772675.1| | ? | 465 | 44350 | 3.90 | 0 | 36 |
| gi|27382903|ref|NP_774432.1| | adhesin? | 1209 | 118221 | 4.09 | 0 | 10+ |
| Caulobacter crescentus CB 15<br>gi|16124999|ref|NP_419563.1| | S-layer-editing-metalloprotease | 658 | 68401 | 4.63 | 2 | 2 |
| gi|16125259|ref|NP_419823.1| | S-layer-protein | 1073 | 103551 | 4.03 | 3 | 5 |
| gi|16126314|ref|NP_420878.1| | ? | 470 | 46826 | 4.28 | 0 | 3 |
| gi|16126847|ref|NP_421411.1| | ? | 650 | 65213 | 3.90 | 0 | 35-40 |
| Escherichia coli CFT 073<br>gi|26249405|ref|NP_755445.1| | HlyA | 1024 | 110180 | 5.69 | 0 | 13 |
| Chromobacterium violaceum ATCC 12472<br>gi|34495766|ref|NP_899981.1| | adhesin? | 4130 | 431580 | 4.37 | 2 | 4+ |
| gi|34495971|ref|NP_900186.1| | ? | 1943 | 201266 | 4.17 | 0 | 80+ |
| Mesorhizobium loti MAFF 303099<br>gi|13471141|ref|NP_102710.1| | rhizobiocin-Rzca | 1018 | 105072 | 4.09 | 0 | 32+ |
| gi|13472325|ref|NP_103892.1| | endoglycanase | 497 | 52454 | 4.38 | 0 | 7 |
| gi|13475631|ref|NP_107198.1| | ? | 495 | 51308 | 4.51 | 0 | 5 |
| Neisseria meningitidis serotype A Z 2491<br>gi|15794519|ref|NP_284341.1| | ? | 350 | 38415 | 5.09 | 2 | 1-2 |
| gi|15794520|ref|NP_284342.1| | FrpA-like | 697 | 75487 | 4.57 | 0 | 15 |
| Neisseria meningitidis serotype B MC 58<br>gi|15676490|ref|NP_273629.1| | FrpA | 1302 | 141312 | 4.49 | 1 | 27+ |
| gi|15677274|ref|NP_274427.1| | FrpC | 1829 | 197326 | 4.40 | 1 | 51+ |
| Nitrosomonas europaea ATCC 19718<br>gi|30248187|ref|NP_840257.1| | agglutinin | 3064 | 312030 | 3.90 | 0 | 5+ |

| Genebank accession | Function, putative | Residues | MW(Da) | pI (theor) | Cysteine | Repeats |
|---------------------|--------------------|----------|---------|-----------|----------|----------|
| Nostoc sp. PCC 7120 |                    |          |         |           |          |          |
| gi|17227770|ref|NP_484318.1 | esterase-integrin | 826     | 87955    | 4.11     | 0        | 5+       |
| gi|17227772|ref|NP_484320.1 | nuclease        | 2348    | 245877   | 4.18     | 0        | 8+       |
| gi|17227786|ref|NP_484334.1 | endoglucanase   | 656     | 73856    | 4.68     | 0        | 8+       |
| gi|17227860|ref|NP_484408.1 | peptidase      | 900     | 93677    | 4.18     | 0        | 24+      |
| gi|17228286|ref|NP_484834.1 | ?               | 993     | 102790   | 3.90     | 0        | 40+      |
| gi|17228898|ref|NP_485446.1 | ?               | 556     | 56477    | 3.90     | 0        | 40+      |
| gi|17230146|ref|NP_486694.1 | ?-hydrolase     | 1417    | 148319   | 3.90     | 0        | 13+      |
| gi|17230147|ref|NP_486695.1 | ?               | 3262    | 339262   | 3.90     | 0        | 13+      |
| gi|17230285|ref|NP_486833.1 | ?               | 681     | 66829    | 4.47     | 0        | 4+       |
| gi|17230838|ref|NP_487386.1 | ?-epimerase     | 1428    | 150809   | 3.90     | 0        | 36+      |
| gi|17231151|ref|NP_487699.1 | ?               | 589     | 61661    | 4.10     | 1        | 25       |
| gi|17231564|ref|NP_488112.1 | ?-iron-regulated | 493     | 50343    | 3.99     | 0        | 13       |
| gi|17231730|ref|NP_488278.1 | phytase         | 1821    | 193174   | 4.17     | 0        | 8?       |
| gi|17233144|ref|NP_490234.1 | dystroglycan-cadherin | 3033 | 318825   | 4.14     | 0        | 24+      |
| gi|17233317|ref|NP_490407.1 | ?               | 268     | 28394    | 3.99     | 0        | 2        |
| gi|17233320|ref|NP_490410.1 | ?               | 4936    | 519100   | 4.08     | 13       | 41+      |
| Pseudomonas aeruginosa PAO1 |                |          |         |           |          |          |
| gi|15596446|ref|NP_249940.1 | alkaline-protease | 479     | 50402    | 4.34     | 0        | 5        |
| gi|15596442|ref|NP_249936.1 | AprX?           | 414     | 41391    | 4.32     | 0        | 5        |
| gic15597071|ref|NP_250565.1 | adhesin?        | 2468    | 238273   | 3.90     | 0        | 4-5+     |
| Pseudomonas putida KT 2440 |                 |          |         |           |          |          |
| gi|26986912|ref|NP_742337.1 | Adhesin, biofilm formation | 8682 | 887687   | 4.10     | 0        | 10+      |
| gi|26988525|ref|NP_743950.1 | ?               | 480     | 48685    | 3.90     | 0        | 7+       |
| gi|26989281|ref|NP_744706.1 | peroxidase      | 3619    | 374251   | 4.09     | 0        | 50+      |
| gi|26990554|ref|NP_745979.1 | ?               | 768     | 79859    | 4.06     | 0        | 15+      |
| gi|26991602|ref|NP_747027.1 | serine-protease | 1805    | 187702   | 4.86     | 0        | 23       |
| Prochlorococcus marinus str MIT 9313 |        |          |         |           |          |          |
| gi|33862529|ref|NP_894089.1 | ?               | 1765    | 187494   | 4.18     | 0        | 8+       |
| gi|33862645|ref|NP_894205.1 | ?               | 195     | 20578    | 4.01     | 1        | 6        |
| gi|33863201|ref|NP_894761.1 | ?               | 410     | 42946    | 3.90     | 1        | 2-3?+    |
| gi|33864144|ref|NP_895704.1 | ?               | 257     | 27137    | 3.90     | 0        | 7        |
| Pseudomonas syringae pv tomato str: DC 3000 |    |          |         |           |          |          |
| gi|28870362|ref|NP_792981.1 | metalloprotease | 535     | 56708    | 4.20     | 0        | 5        |
| gi|28870496|ref|NP_793115.1 | metalloprotease | 475     | 49275    | 4.51     | 0        | 5        |
| gi|28871227|ref|NP_793846.1 | mannuronan-epimerase | 1610 | 166578   | 4.10     | 2        | 25+      |
| Ralstonia solanacearum GMI 1000 |            |          |         |           |          |          |
| gi|17544821|ref|NP_518223.1 | ?               | 1217    | 124373   | 4.18     | 0        | 25+      |
| gi|17544823|ref|NP_518225.1 | ?               | 960     | 101576   | 4.51     | 0        | 12       |
| gi|17544965|ref|NP_518367.1 | ?               | 1156    | 120666   | 3.90     | 1        | 52+      |
| gi|17544968|ref|NP_518370.1 | metallopeptidase | 1499 | 156898   | 4.09     | 0        | 46+      |
| gi|17548515|ref|NP_521855.1 | ?               | 589     | 62038    | 4.47     | 0        | 5        |
| gi|17548516|ref|NP_521856.1 | ?               | 692     | 70140    | 4.38     | 0        | 38       |
| gi|17549401|ref|NP_522741.1 | adhesin        | 4106    | 394859   | 3.90     | 0        | 8+       |
| Shewanella oneidensis MR-1 |                |          |         |           |          |          |
| gi|24375799|ref NP_719842.1 | ?               | 2768    | 284924   | 3.90     | 1        | 6+       |
| Sinorhizobium meliloti 1021 |                |          |         |           |          |          |
| gi|15965364|ref|NP_385717.1 | ?               | 359     | 36639    | 3.90     | 1        | 12       |
| gi|15965468|ref|NP_385821.1 | ?               | 363     | 38875    | 3.99     | 0        | 12       |
| gi|15965753|ref|NP_386106.1 | ?               | 548     | 55162    | 3.90     | 0        | 28       |
| gi|15965787|ref|NP_386140.1 | ?               | 266     | 27516    | 4.13     | 0        | 2        |
| gi|15966738|ref|NP_387091.1 | ?               | 396     | 39708    | 3.90     | 0        | 26       |
| gi|16262469|ref|NP_435262.1 | metalloprotease | 551     | 57690    | 4.62     | 0        | 10       |
| gi|16263317|ref|NP_436110.1 | EglC-endoglucanase | 465 | 49583    | 3.990    | 0        | 5        |

Genebank accession                      Function, putative               Residues   MW(Da)    pI(theor)  Cysteine Repeats
Sinorhizobium meliloti 1021                                                                            
gi|16263604|ref|NP_436397.1|             ?                              539       55485     4.03      1        27
gi|16263827|ref|NP_436619.1|             adenylcyclase?                 1072      105201    3.90      0        30+
gi|16264320|ref|NP_437112.1|             ?                              559       58294     4.09      0        15
gi|16264329|ref|NP_437121.1|             ?                              353       37705     4.03      0        12
gi|16264481|ref|NP_437273.1|             ?                              387       39348     3.90      0        28
gi|16264638|ref|NP_437430.1|             ?                              219       22103     3.90      0        14
gi|16264732|ref|NP_437524.1|             metalloprotease                1112      114963    3.90      0        25
gi|16264803|ref|NP_437595.1|             endoglycanase                  465       50254     4.13      0        5
gi|16264971|ref|NP_437763.1|             ?                              78        7868      4.43      0        2

Synechococcus sp.WH 8102                                                                                
gi|33864621|ref|NP_896180.1|             SwmA/swimming                  835       83519     3.90      0        25
gi|33864732|ref|NP_896291.1|             phosphatase                    749       80376     4.25      1        8
gi|33865518|ref|NP_897077.1|             ?                              209       21682     4.65      0        14
gi|33866440|ref|NP_897999.1|             metalloprotease                391       42026     4.74      2        4
gi|33866731|ref|NP_898290.1|             ?                              100       10730     4.47      2        2
gi|33866823|ref|NP_898382.1|             tubulin-suppressor?            651       69468     4.78      1        5
gi|33866834|ref|NP_898393.1|             protease/+?                   1313      142142    3.90      3        4
gi|33866920|ref|NP_898479.1|             phosphatase/nucleotidase       750       79898     4.14      0        6
gi|33866939|ref|NP_898498.1|             ?                              281       29312     4.29      1        12

Synechocystis sp.PCC 6803                                                                           
gi|16329631|ref|NP_440359.1|             iron-regulated                  591       60447     3.90      0        22
gi|16330226|ref|NP_440954.1|             integrin/receptor               3016      311321    3.99      0        4+
gi|16330463|ref|NP_441191.1|             ?                              1741      178153    3.90      0        20+
gi|16332223|ref|NP_442951.1|             integrin                       1771      192049    4.35      0        4
gi|16332225|ref|NP_442953.1|             ?                              1290      139462    4.02      0        20+

Vibrio cholerae O1 biovar Eltor str.N 16961                                                        
gi|15641462|ref|NP_231094.1|             RtxA/toxin                     4558      484758    4.99      2        12
gene="VC1620 pseudogene?"                ?                              1000      105432    4.23      1        10
gi|15601604|ref|NP_233235.1|             ?                              3263      341286    3.90      2        4+

Xanthomonas axonopodis pv.citri str. 306                                                            
gi|21242662|ref|NP_642244.1|             muraminidase                   606       66234     4.94      0        4
gi|21242932|ref|NP_642514.1|             ?                              2183      224665    4.27      1        46+
gi|21242933|ref|NP_642515.1|             ?                              1294      132846    4.47      1        30

Xylella fastidiosa 9a5c                                                                                
gi|15837270|ref|NP_297958.1|             bacteriocin?                   1208      128325    4.41      1        30+
gi|15837613|ref|NP_298301.1|             ?                              1636      172909    4.46      0        30+
gi|15838998|ref|NP_299686.1|             ?                              2064      218644    4.36      1        40+
gi|15839348|ref|NP_300036.1|             ?                              1296      138769    4.42      1        22+

Yersinia pestis KIM                                                                                    
gi|16124100|ref|NP_407413.1|             metalloprotease                445       49587     5.70      3        4

A few species, in which such ORFs are found, were not taken into account: Xylella fastidiosa Temecula 1, Vibrio parahaemolyticus RIMD, Vibrio vulnificus CMCP 6 and YJ 016, Escherichia coliO157H 7 (too similar to the other mentioned species) and Gloeobacter violaceus, Desulfovibrio vulgaris, Photorhabdus luminescens subsp. laumondii, Pirellula sp. 1 (not in the database at the time of analysis). The repeat number should be taken with caution, according to the degeneracy of the repeat.+ indicates the presence of other kind of repeats. A few ORFs are represented in italics and are dubious repeats containing proteins. Cadherin, integrin, dystroglycan domains were identified in several of these ORFs and might essentially indicate adhesion proteins. All putative functions should be taken with caution and might differ from those indicated in GenBank.

Table 2  
Some well-characterized type I secretion systems with the size of the secreted proteins and the names of the secretion functions together with the organism they originate from

| Name               | Protein name          | Size         | ABC   | MFP   | OMP   | Organism                  |
|--------------------|-----------------------|--------------|-------|-------|-------|---------------------------|
| Hemolysin A        | HlyA                  | 1023         | HlyB  | HlyD  | TolC  | Escherichia coli          |
| Metalloproteases   | PrtA, PrtB,           | 472–481      | PrtD  | PrtE  | PrtF  | Erwinia chrysanthemi      |
|                    | PrtC, PrtG            |              |       |       |       |                           |
| Cyclolysin         | CyaA                  | 1737         | CyaB  | CyaD  | CyaE  | Bordetella pertussis      |
| Colicin V          | CvaC                  | 103 (pre)    | CvaB  | CvaA  | TolC  | Escherichia coli          |
|                    | (no repeats)          |              |       |       |       |                           |
| S-layer protein    | RsaA                  | 1026         | RsaD  | RsaE  | RsaF? | Caulobacter crescentus   |
| Hemophore          | HasA                  | 188          | HasD  | HasE  | HasF/TolC | Serratia marcescens     |
|                    | (no repeats)          |              |       |       |       |                           |
| Lipase             | LipA                  | 613          | LipB  | LipC  | LipD  | Serratia marcescens      |
| Nodulation factor  | NodO                  | 284          | PrsD  | PrsE  | ?     | Rhizobium leguminosarum   |
| Adhesion factor    | LapA                  | >8000        | LapB  | LapC  | LapE  | Pseudomonas fluorescens   |

See text for more details and references.

P. Delepelaire / Biochimica et Biophysica Acta 1694 (2004) 149–161

TolC

Outer membrane

Periplasm

bottom view

MsbA

Inner membrane

Cytoplasm

bottom view

Fig. 1. Ribbon diagram of the *E. coli* OMP TolC of type I secretion system, from side or bottom (i.e. periplasmic entrance) and the ABC protein MsbA from *V. cholerae* which has homologies with ABC components of type I exporters from side and bottom (i.e. cytoplasmic side) at the same scale. For both proteins each monomer is colored differently and the one with the helices in red displays the beta sheets in pale blue. Both proteins are about 140 Å long. See text for details.

secretion and will be restricted to gram-negative bacteria. The reader is referred to previous reviews covering other aspects of type I secretion [12].

## 2. Structure–function analysis of the secretion protein complex

The reconstitution of diverse systems originating from other bacteria in *E. coli* allowed the characterization of the secretion apparatus. The structural information relevant to each component (outer membrane component, MFP, ABC protein) will be examined in the light of their function.

### 2.1. TolC/OMP

The type I secretion apparatus of gram-negative bacteria always includes a specific OMP. The best-characterized component of the ABC protein exporter in terms of structure is *E. coli* TolC, the OMP component of several ABC exporters (HlyA, CvaC in *E. coli*), which has been identified as the OMP of the hemolysin secretion pathway [13]. TolC (55 kDa) is a multifunctional *E. coli* OMP that can serve as an outer membrane component for several processes such as protein export via a type I pathway and drug export via RND (resistance, nodulation, division) type systems. This means it is likely to be connected to members of the MFP/adaptor class, which are found both

P. Delepelaire / Biochimica et Biophysica Acta 1694 (2004) 149–161

A

extracellular  
medium  

outer  
membrane  

MFP  
periplasm  

inner  
membrane  

cytoplasm  

Cter  
chaperone  
(e.g. SecB)  

OMP  

ABC  

ATP  

ADP  

1  
2a  
2b  
3  

4  

B  

4  

ATP  

ADP  

1  
2  
3  

Fig. 2. A model of the type I secretion pathway with The ABC/MFP complex either not preformed (part A) or preformed (part B). The model is just meant to draw the attention on the assembly of the secretion complex driven by the C-terminal secretion signal, the possible involvement of chaperone to slow down folding; the step at which ATP hydrolysis occurs is purely speculative as is the stoichiometry of the different components.

in the ABC exporters and in the RND exporters. In this last case, for example, it operates together with the AcrAB proteins in the inner membrane to constitute an efflux pump driven by the proton motive force. It is also somehow involved in the maintenance of outer membrane integrity and the import of certain colicins and serves as a phage receptor. Its role in these two last processes, although very interesting and important, is not relevant here. It is thus able to recognize a variety of proteinaceous substrates [14–20].

The X-ray structure of TolC was solved in 2000 [21]. This provided new information both about OMP structure and about translocation by type I machinery or drug efflux. TolC presents a new kind of OMP architecture. It is a trimeric protein, anchored in the outer membrane by beta strands and with a very long periplasmic part made almost uniquely of alpha helices extending about 100 Å into the periplasm (see Fig. 1). The molecule forms a tapering tube that is almost closed at the periplasmic end and wide open at the outer membrane surface, with almost no loops at the extracellular surface, as for porin or siderophore receptors. Its internal diameter of about 30 to 35 Å stays constant throughout the molecule and delimits a large water-filled cavity (40,000 Å³) open to the extracellular medium. This cavity may be the exit tunnel for secreted proteins and is large enough to accommodate secondary structure elements or even small folded polypeptide chains. Each monomer is made of two similar halves, probably resulting from a duplication event, and contributes four beta strands in the outer membrane and four alpha helices in the periplasm. The periplasmic part can be divided into three zones. The apical zone is closest to the outer membrane. In this region, the alpha helices are straight and packed by knob into hole bonds. This is followed by an equatorial zone, containing connecting loops, and the C- and N-

terminal ends of the monomer, and a distal zone, where the alpha helices are arranged in two coiled coils (see Fig. 1). Each monomer contributes to one external and one more internal superhelix, which close the tunnel at the distal end where the hole measures only 3.5 Å and cannot let folded molecules through (see bottom view, Fig. 1) [21]. On the basis of this structure, it was proposed that TolC opens at the distal end by an iris-like mechanism, by unwinding the coiled-coiled helices so as to move the inner ring towards the exterior, realigning the two sets of helices. This generates a maximal opening of around 16 to 20 Å, which is wide enough to allow the passage of proteins that have adopted their secondary structures [19]. This only requires minor movement at the intermonomer interface. Opening would be triggered by members of the MFP/adaptor class [21,22]. In this model, a fixed point is necessary to anchor TolC either to the inner membrane complex or to the peptidoglycan, indicating the presence of at least two binding sites on TolC. This point has not yet been addressed. This unwinding movement probably originates in the equatorial part of the molecule, which contains both the N- and C-terminal ends. It is noteworthy that the extreme C-terminus was removed to allow crystallization. Furthermore, MFP/adaptor is probably a trimer like TolC [23], the ABC protein is most likely a dimer [24–26], whereas AcrB, the inner membrane component of a RND efflux pump, is a trimer [27,28]. The adaptation problem between outer and inner membrane components is probably resolved differently in these two systems.

Several of the mechanistic assumptions of this model have been tested. In particular, it was predicted that the creation of supplementary bonds at the distal opening of the molecule would hamper the opening of the iris, thus impairing the transport of molecules. Structural information made it possible to use site-directed mutagenesis to create

two different kinds of disulfide bonds (self-locking and bridged by a chemical cross-linker). An intermonomer disulfide bridge was stably established and greatly reduced export without changing recruitment of the export apparatus. Several other longer links were established with the help of a sulfhydryl-reacting cross-linker between different residues. All mutants except one secreted less or no hemolysin, supporting the proposed model [29]. One of the mutants was also more sensitive to novobiocin, a substrate of the AcrAB/TolC export machinery. This might be due to differences in the size of the opening. All these data favor the model in which maintaining the helices in the closed state decreases export. Furthermore, a salt bridge is involved in maintaining TolC in the closed state. This salt bridge is located between R367 of the inner coiled-coil of one monomer and D153 of an adjacent monomer (outer coiled-coil). Disruption of this salt bridge in the different cross-linked molecules gave contrasting results; one mutant still secreted at a very low level (the cross-link was still effective), whereas another secreted at a wild-type level in the presence of the cross-linker, implying that the flexibility given by the absence of the salt-bridge impeded cross-linking. All these studies were correlated with electrophysiological studies of TolC and its mutants in planar lipid bilayer [30]. Indirect effects should also be considered, although it is tempting to speculate that the mutations are directly responsible for the observed effects.

Wild-type TolC forms channels in planar lipid bilayers. The channels are not voltage-dependent and are stable. They are cation selective, with a K⁺/Cl⁻ permeability ratio of 16.5. The channels are pH-dependent; they close at low pH values with a pK of 4.3 and open at higher salt concentrations. The selectivity of the TolC channel for cations is conferred by a hexa-aspartate ring at the entrance of TolC at the distal end (each monomer contributing to two of the aspartates). Mutation of these residues to alanine results in a molecule that is no longer pH-dependent and displays anion selectivity but still secretes hemolysin at wild-type levels [31]. The same study showed that the TolC conductance of the wild-type is blocked by external di- or trivalent cations and that the aspartate mutant was not blocked. Thus, these charged residues are clearly involved in the control of conductance and maybe make it possible to block the TolC channel from the outside. A series of mutants was also constructed by disrupting hydrogen bonds or salt bridges involved in stabilizing the entrance in the closed state. These mutants were fully capable of hemolysin secretion and were novobiocin- and deoxycholate-resistant. However, they showed greater conductivity than the wild-type protein. Disruption of links was synergistic and increased the initial conductance by up to 10-fold in one of the mutant in which two bonds were disrupted (per monomer) [29,31]. These mutants were not tested for their potential leakage of periplasmic solutes or their increased permeability towards hydrophilic components too large to diffuse through the porins.

### 2.2. MFP/adaptor component

The name MFP comes from a low degree of identity with some viral fusion proteins [32]. This putative function has been only experimentally substantiated for AcrA, which participates in drug export via the AcrAB/TolC RND type of system [33]. Whether this is relevant to the function of the MFP/adaptor is not known. All that is known about the structure of the MFP/adaptor component of the protein exporter is that it has a short N-terminal fragment in the cytoplasm, a unique transmembrane segment and a large periplasmic fragment. Analysis of the primary structure showed a high propensity for coiled-coil formation and multimerization has been observed for HlyD, the MFP/adaptor component of the Hly transporter [23,32]. Some mutations at the extreme C-terminus as well as in a central region of HlyD are deleterious to hemolysin secretion and are dominant over wild-type HlyD [34]. Similar results have been obtained with colicin V and its secretion system [35]. It would be interesting to combine mutations in TolC that generate a wider opening at the entrance of the channel with mutations in either the MFP or ABC component still assembling nonfunctional transporters (see below) to test whether the defect is due to a defect in the active opening of the TolC channel. In the case of the Acr system, an interaction has been identified between the C-terminus of AcrA and the periplasmic loops of AcrB [36]. Whether homologous interactions occur between the MFP and ABC component of type I exporters (which do not have such large periplasmic domain) remains to be determined. The MFP component is probably responsible for the mechanical force exerted on the equatorial domain of TolC leading to opening of the channel. In the case of AcrAB/TolC-mediated drug efflux, the MFP component, AcrA, is a lipoprotein. It has been shown that anchoring to the inner membrane is not necessary for function [37].

Assembly of the translocation apparatus has been studied in three distinct systems: Has, Prt [38] and Hly [23]. This gave similar overall results, although some variations were encountered. In all cases, the substrate triggers the stepwise recruitment of the secretion apparatus via an initial interaction with the ABC protein. However, the function of the MFP/adaptor in this process is not always the same; in the Has and Prt systems, the MFP/adaptor does not interact by itself with the substrate, whereas this appears to be the case in the Hly system. This interaction appears to be mediated by the N-terminal end of HlyD, which is cytoplasmic and is 59 amino acids long whereas the equivalent part of PrtE and HasE is much shorter. This N-terminal part of HlyD appears to contain an amphipathic helix and a cluster of charged residues that are essential for this interaction, but seem to play different roles. The absence of the helix impairs export but has no effect on TolC recruitment or substrate engagement. In the absence of the charged cluster, the substrate is still engaged but TolC is not recruited [39]. Signal transduction from the substrate loaded on the inner membrane
complex to TolC seems to require this specific N-terminal part of HlyD. So far, this appears to be quite specific to the Hly system as such long extensions on the MFP/adaptor are found mainly on the Hly class of exporters. It is not known whether this is specific to the whole HlyA molecule or whether this would happen also with the minimal C-terminal secretion signal. In the case of Prt/Has, the substrate does not specifically recognize the MFP/adaptor, unlike with Hly [40]. Another difference is the absence of a preformed stable complex between the ABC protein and the MFP/adaptor in the case of the Prt/Has transporter, as opposed to the Hly transporter. These differences may correspond to a real difference between the secretion systems or differences in experimental conditions (cross-linking in the case of Hly, no cross-linking in the cases of Prt and Has) (see Fig. 2). Assembly of the translocation channel is driven by specific interactions between each of its components, once a substrate has been recognized by the ABC protein in the case of Prt/Has or once a substrate has been recognized by the MFP-ABC complex in the case of Hly. It would be interesting to test N-terminal deletion derivatives of PrtE/HasE to check their functionality and the assembly of the secretion complex. HlyD mutants have also been found in the N-terminal tail, leading to the same phenotype as TolC mutants secreting inactive hemolysin due to misfolding (see below).

### 2.3. ABC protein

ABC proteins involved in transport typically comprise four domains, two TMDs and two NBDs assembled in either one or several polypeptide chains [41]. The primary sequence of NBDs (with their characteristic signature) is more highly conserved than that of the TMDs, supporting the hypothesis that specificity is conferred by the TMDs. Several NBDs from ABC proteins have been crystallized, as have two complete ABC proteins (MsbA, involved in lipid A export through the inner membrane (see Fig. 2), and BtuCD, involved in vitamin B12 import across the cytoplasmic membrane [24–26]). ABC proteins from type I export systems consist of one TMD and one NBD, and are believed to function as homodimers. MsbA, the best-characterized ABC protein, is a homodimer in which the connecting loops between the six TM segments of each monomer are also helical and make an intracellular domain (ICD) connecting the NBD and the membrane-embedded part of the TMD. This subdomain is believed to participate in signal transduction between TMD and NBD (see Fig. 2).

Only one NBD of an ABC protein involved in protein translocation by the type I pathway has been crystallized, that of HlyB, in a monomeric form [42]. Like other ABC domains of known structure, this NBD consists of two subdomains: the catalytic domain (armI, alpha, beta) and the signaling domain (armII, alpha), containing the signature motif. The two subdomains are connected by two loops: the Q-loop and a Pro-loop. Some mutations in the Pro loop display a thermosensitive phenotype for hemolysin secretion. Superimposition of the structures of several NBDs made it possible to identify a so-called “structurally diverse region” in a subregion of armII. This region might play a role in specificity and interdomain communication. Furthermore, surface plasmon resonance detected a specific (but low affinity) interaction between this monomeric NBD of HlyB and a C-terminal fragment of HlyA bearing the secretion signal. ATP abrogates this interaction, suggesting that the NBD itself contains part of the substrate recognition sequence [43]. Whether this is a primary or secondary recognition site remains to be determined.

The function of the ABC protein in substrate recognition has been firmly established by specific suppressors of the C-terminal secretion signal mutation isolated in HlyB [44] and by making hybrid transporters showing that specificity resides in the ABC protein [45]. ATPase activity is essential for transport as mutation of the conserved residues in the Walker boxes, which have reduced ATPase activity and secrete at reduced rates, in the function of their ATPase activity [46]. ATP binding and/or hydrolysis is not required for assembly of the translocation complex [23]. A few studies have addressed the question of substrate recognition by the ABC protein in vitro, either with the full-length ABC protein [47] or with the NBD (see above). The basal ATPase activity of the full-length PrtD protein, the ABC protein of the Prt system, is very low and is completely inhibited by the C-terminal secretion signal of one of the proteases, with a \(K_{i}\) of 0.2 μM. In the case of the interaction between a large C-terminal domain of HlyA with the monomeric domain of HlyB, the \(K_{d}\) was estimated to be 4 μM, which is quite high [43]. However, ATP binding and/or hydrolysis always seems to regulate substrate binding. This might be indicative of substrate release induced by ATP binding and/or hydrolysis during the secretion process. Further studies are required to study this point.

Fig. 1 shows the ribbon structures of TolC and MsbA to help visualize the structure of these proteins. It is likely that the structure of the ABC protein involved in protein export will adopt a similar fold to that of MsbA. No structural data is yet available for the MFP/adaptor protein. The most striking features of these two proteins are their length (about 140 Å for both of them) as well as, in the TolC case, the very narrow size of the putative entrance of the channel (see bottom views).

## 3. Folding and secretion

### 3.1. Type I secretion system translocates unfolded molecules

The fact that the secretion signal is located at the C-terminus implies that the transported substrate has to be completely synthesized before it can productively interact with the ABC protein. This questioned the influence of folding on secretion. Several models were envisaged: eitherthe protein is secreted unfolded and there might (or might not, according to the substrate) be specific mechanisms ensuring unfolding, or the protein is secreted folded. This question has been solved in the case of the HasA hemophore protein from *S. marcescens*, thanks to its heme binding properties. In its native form, as can be isolated from the supernatant, HasA binds heme with a very high affinity due to three residues implicated in heme iron liganding [48,49], leading to a specific absorption spectrum with a Soret band at 407 nm. In the reconstituted system in *E. coli*, in the absence of its specific secretion functions, HasA accumulates inside the cytoplasm in a soluble heme-binding form with a characteristic Soret band, indicative of its folding [50]. Folded HasA is stable inside the cytoplasm. By uncoupling synthesis of HasA from its secretion functions, it was possible to study the influence of folding state on HasA secretion. Folded HasA is no longer secreted. No secretion occurs when secretion functions are synthesized after HasA. This is not due to heme binding as a HasA variant that did not bind heme displayed the same phenotype [50]. This raised questions about the mechanisms ensuring efficient secretion. It was shown that the SecB chaperone, previously identified as a specific sec chaperone, is involved in HasA secretion in the reconstituted system through a direct effect on HasA [51]. SecB is essential for HasA secretion in *S. marcescens*, as shown by constructing a SecB null strain in *S. marcescens* [52]. This means that SecB is a chaperone involved in both sec and type I pathways. In vitro studies have shown that HasA refolds extremely rapidly after denaturation in the absence of SecB. SecB specifically binds unfolded HasA and slows down its folding rate by three orders of magnitude [53]. HasA is stabilized by hydrogen bonds between side chains of residues in the N-terminus and the C-terminus. Destruction of these bonds by mutations either in the N-terminus or the C-terminus results in slow folding, yet fully functional HasA mutants. These mutants, although still able to bind SecB in vitro in their unfolded form, are secreted in a mostly SecB-independent manner, showing that SecB is an unfolding factor in this process. This is reminiscent of the behavior of SecB-independent mutants of the sec substrate pre-maltose bonding protein, which still bind SecB [54]. In the sec system, SecB slows precursor folding and targets the precursor to SecA via SecB–SecA interactions [55,56]. The SecB residues implicated in SecA binding do not play any role in HasA secretion, which raises questions about the specificity determinants of the two pathways. Although intracellular folded HasA can no longer be secreted, it is still able to interact with the ABC transporter as it inhibits the secretion of newly synthesized HasA molecules [50]. Overproduction of SecB does not relieve this inhibition, nor does it restore secretion of the folded form. Overproduction of the transporter relieves the inhibition of the secretion by the folded form, indicating that folded intracellular HasA does not titrate a cytosolic factor. This implies that the ABC transporter is able to recognize the folded form of HasA,

[50], consistent with the coprecipitation of HasA and its transporter by affinity on hemin agarose [38]. Although HasA is not secreted by the Prt transporter of *E. chrysanthemi*, it is able to inhibit protease secretion by this transporter [9]. This inhibition is dependent on the N-terminal domain of HasA [57]. This implies that there might be a SecB-dependent cotranslational addressing mechanism to ensure efficient secretion. Identification of the region of HasA responsible for the inhibition of its own secretion, as well as of the SecB binding site(s) on HasA, will help to elucidate the secretion mechanism.

The only case in which a chaperone is known to be required is that of HasA. This SecB requirement extends to the other HasAs secreted by the HasAsm system in the reconstituted system [57]. Hemolysin secretion in the original host is SecB-independent, as is protease secretion in the reconstituted system. Other studies have tackled this problem by using a chimeric protein, constituted of a single-chain Fv antibody fused to a large C-terminal HlyA secretion domain [58]. This chimeric protein is efficiently secreted by the Hly transporter and is fully active, as shown by its antigen-binding activity, indicative of the formation of the two disulfide bonds in the antibody part. In the cytoplasm, disulfide bonds cannot be formed due to the strong reducing environment, unless TrxB, a member of the thioredoxin family, is mutated [59]. When TrxB is mutated, the secretion of this chimeric protein by the Hly transporter is impaired, even though it remains active. Hence, like the Has transporter, the Hly transporter cannot secrete folded molecules. When a similar chimeric protein is translocated in the periplasm via the sec machinery by fusing it to a signal peptide, an active DsbA [60], the periplasmic disulfide bond formation catalyst, is required to ensure proper disulfide bond formation and hence activity. DsbA is not required for secretion via the Hly machinery. The authors proposed that the disulfide bonds are formed in the TolC channel during secretion and one might speculate that TolC acts as a folding cage allowing proper positioning of the cysteines. This study should be paralleled with the isolation of TolC mutants allowing secretion of misfolded inactive HlyA protein (next paragraph).

### 3.2. TolC mutants: is TolC a folding cage?

Several groups have reported that outer membrane integrity affects the activity of the secreted hemolysin molecule. In particular, *rfap* and *H* mutants, which have an altered LPS, secrete normal or near normal amounts of inactive hemolysin [61,62]. This is due to improper folding as the in vitro unfolding/refolding of these secreted molecules generated perfectly active hemolysin. Further analysis of the secreted molecules showed that they were aggregated. It is possible that LPS, the synthesis of which is affected in these mutants, is involved in hemolysin folding and/or secretion. Misra and collaborators carried out extensive mutagenesis to find TolC mutants specifically implicatedin the folding of HlyA. As mentioned above, TolC is implicated in drug resistance, outer membrane integrity and colicin El sensitivity. TolC mutants were isolated by two different screens as non-hemolytic but novobiocin-resistant (indicative of functional TolC still present) in a first screen or as deoxycholate-resistant and colicin E1-tolerant in a second screen. The hemolytic phenotype was subsequently tested. This led to the isolation of eight mutants with the normal amount of TolC correctly localized in the outer membrane. All these mutants secreted normal or near normal levels of hemolysin, but with reduced activity (2- to 20-fold). This activity reached wild-type levels after denaturation/renaturation in vitro. The hemolysin secreted formed aggregates in the external medium. Time-course analysis showed that hemolysin was secreted more slowly in the mutants than in the wild type. This phenotype is clearly not due to LPS, which is not affected in these mutants. The mutants affected several sites in the TolC molecule, including the beta barrel and the distal end, precluding any straightforward explanation for their phenotype. It is tempting to speculate that wild-type TolC provides a “cage” for proper folding of some secondary structure elements in HlyA and that mutations make TolC interact incorrectly with HlyA, thus altering its folding and its activity. It would be interesting to test other substrates like proteases or HasA that also use TolC as the OMP. It is also possible that access to the entrance of TolC is impaired or that exit from TolC is impaired, leading to the observed phenotype. One of the mutations was localized at the C-terminus of TolC, which is not seen in the structure. As mentioned above, it is possible that this part of the molecule interacts with HlyD, thus slowing access to the channel. This study suggests that there is a link between folding and secretion kinetics as did the abovementioned study on the secretion of the single-chain Fv antibody variants.

The main conclusion of these studies is that folded substrates are secretion incompetent, implying the existence of mechanisms ensuring presentation of unfolded molecules to the transporter. One way to avoid premature folding is the use of chaperones like SecB in HasA secretion. It is unclear whether this is a general case. In particular, glycine-rich repeats (absent in HasA) generally play a role in secretion as they improve the secretion of chimeric protein [63] and are required for full activity of the secreted protein. They may act as an intramolecular chaperone.

proteins of the RTX family displaying very little primary sequence identity. Several random libraries of peptides (encoded by random oligos) were generated at two zones of the C-terminal secretion signal of hemolysin and fused to a truncated hemolysin devoid of its C-terminal secretion signal. Plate assays of secretion according to hemolytic activity revealed several distinct features required for functionality [67,68]. The wild-type signal consists of two alpha helices separated by a linker and followed by an unstructured C-terminal domain. The first helix is essential and must be amphiphilic, whereas the second one is not essential and can be replaced by a linker of a minimum of four to seven residues. The extreme C-terminus favors non-positively charged residues. This secretion signal appears to be highly flexible.

The C-terminal secretion signals of proteases, HasA and lipases present a high degree of primary sequence conservation. All C-terminal secretion signals of proteases consist of a negatively charged residue followed by three hydrophobic residues. The addition of even one residue can be deleterious: whereas a single tryptophan residue added after the last residue of PrtG (one of the metalloproteases secreted by *E. chrysanthemi*) decreases the secretion by 50%, a similar change with a phenylalanine residue abolishes virtually all secretion [69], indicative of exquisite specificity at the C-terminus. *S. marcescens* has at least two distinct type I secretion systems, the HasA secretion system and the lipase secretion system. The *S. marcescens* lipase (LipA) secretion system consists of the LipB, LipC and LipD functions. This system is able to secrete the *S. marcescens* S-layer protein, the metalloprotease PrtSm but not HasASm. On the opposite, HasA from *P. fluorescens* (HasAPf) and *P. aeruginosa* (HasAPA) are secreted by the LipBCD transporter. Chimeras were generated between the three different HasA proteins and their secretion by the Lip and the Has transporters tested. Conserved residues in the C-termini of the different HasA proteins were mutated and their secretion tested. This led to the identification of a region upstream of the four to six C-terminal residues containing a conserved glycine that is critical for secretion. Point mutations were tolerated in the last four to six C-terminal residues [70]. Thus, the important features of the C-terminal secretion signal are not yet understood. However, as mentioned before, the C-terminal secretion signal confers specificity to the recognition by the ABC protein and RTX toxins, for example, do not display at the extreme C-terminus an acidic residue followed by three or four hydrophobic residues. Alignment of the 50 C-terminal residues of all the proteins mentioned in Table 1 allows to distinguish several classes of signals, whose meaning is not clear in terms of functionality.

If one assumes that all ORFs containing glycine-rich repeats are substrates of ABC transporters and if one also assumes that this region contains the secretion signal, it is possible to carry out statistical analysis of their 50 last amino acids. This region is rich in several amino acids (LDAVTSIF) and poor in others (KHPMW). It is difficult

4. Secretion signal: occurrence of type I secretion and potential applications

The elements important for C-terminal secretion signal activity are still poorly understood. Three isolated signals have been examined and show a high propensity to form alpha-helical structures in membrane mimetic environments [64–66]. The C-terminal secretion signal of hemolysin can be functionally replaced by several signals from other
to extend this gross analysis as different secretion systems are likely to have different requirements for their cognate C-terminal signals.

Type I secretion is widespread in gram-negative bacteria according to the complete bacterial genomes available, again assuming that glycine-rich repeats are substrates of this pathway (see Table 1). The identified functions of the secreted proteins are quite diverse and a given species (for example *S. meliloti*) can possess up to 16 potential substrates. A species may contain several distinct type I exporters [71], and others, although harboring several potential substrates, have only one protein of the TolC class (COG1538). For example, *Synechocystis* has at least four potential substrates and only one TolC homolog.

Some systems secrete extremely large amounts of substrate like the S-layer protein secretion system from *Caulobacter crescentus*, which secretes about 400,000 copies of a single protein to the cell surface per bacterium and per generation [72,73] and may accommodate passengers of various sizes and composition. This has been the basis for potential applications in terms of biotechnology. Two types of applications are envisaged, either mass production of extracellular chimeric polypeptides in view of either easy purification or live vaccine delivery [17,74]. This is beyond the scope of this review which is focused on functioning of the type I secretion system.

Acknowledgements

I thank C. Wandersman for fruitful discussions over many years. I apologize for not mentioning many studies devoted to other type I exporters in bacteria, especially those related to the diversity of the secreted proteins and the secretion systems.

References

[1] U. Baumann, S. Wu, K.M. Flaherty, D.B. McKay, EMBO J. (1993) 3357–3364.

[2] C. Finnie, A. Zorreguieta, N.M. Hartley, J.A. Downie, J. Bacteriol. 180 (1998) 1691–1699.

[3] P. Delepelaire, C. Wandersman, in: G. Winkelmann (Ed.), Microbial Transport Systems, Wiley, 2001, pp. 165–208.

[4] S.M. Hinsa, M. Espinosa-Urgel, J.L. Ramos, G.A. O’Toole, Mol. Microbiol. 49 (2003) 905–918.

[5] C. Wandersman, in: F.C. Neidhart (Ed.), *Escherichia coli* and *Salmonella typhimurium* Cellular and Molecular Biology, ASM Press, Washington, DC, 1996, pp. 955–967.

[6] W. Goebel, J. Hedgpeth, J. Bacteriol. 151 (1982) 1290–1298.

[7] C. Wandersman, P. Delepelaire, S. Létoffé, M. Schwartz, J. Bacteriol. 169 (11) (1987) 5046–5053.

[8] J. Guzzo, F. Duong, C. Wandersman, M. Murgier, A. Lazdunski, Mol. Microbiol. 5 (1991) 447–453.

[9] S. Letoffe, J.M. Ghigo, C. Wandersman, J. Bacteriol. 176 (1994) 5372–5377.

[10] S. Letoffe, J.M. Ghigo, C. Wandersman, Proc. Natl. Acad. Sci. U. S. A. 91 (1994) 9876–9880.

[11] J.M. Ghigo, S. Letoffe, C. Wandersman, J. Bacteriol. 179 (1997) 3572–3579.

[12] J. Young, I.B. Holland, Biochim. Biophys. Acta 1461 (1999) 177–200.

[13] C. Wandersman, P. Delepelaire, Proc. Natl. Acad. Sci. U. S. A. 87 (1990) 4776–4780.

[14] C. Andersen, Rev. Physiol., Biochem. Pharmacol. 147 (2003) 122–165.

[15] C. Andersen, C. Hughes, V. Koronakis, Curr. Opin. Cell Biol. 13 (2001) 412–416.

[16] S.K. Buchanan, Trends Biochem. Sci. 26 (2001) 3–6.

[17] I. Gentschev, G. Dietrich, W. Goebel, Trends Microbiol. 10 (2002) 39–45.
[18] H. Nikaido, H.I. Zgurskaya, J. Mol. Microbiol. Biotechnol. 3 (2001) 215–218.
[19] A. Sharff, C. Fanutti, J. Shi, C. Calladine, B. Luisi, Eur. J. Biochem. 268 (2001) 5011–5026.
[20] V. Koronakis, FEBS Lett. 555 (2003) 66–71.
[21] V. Koronakis, A. Sharff, E. Koronakis, B. Luisi, C. Hughes, Nature 405 (2000) 914–919.
[22] C. Andersen, E. Koronakis, E. Bokma, J. Eswaran, D. Humphreys, C. Hughes, V. Koronakis, Proc. Natl. Acad. Sci. U. S. A. 99 (2002) 11103–11108.
[23] T. Thanabalu, E. Koronakis, C. Hughes, V. Koronakis, EMBO J. 17 (1998) 6487–6496.
[24] G. Chang, J. Mol. Biol. 330 (2003) 419–430.
[25] G. Chang, C.B. Roth, Science 293 (2001) 1793–1800.
[26] K.P. Locher, A.T. Lee, D.C. Rees, Science 296 (2002) 1091–1098.
[27] C.A. Elkins, H. Nikaido, Drug Resist. Updat. 6 (2003) 9–13.
[28] S. Murakami, R. Nakashima, E. Yamashita, A. Yamaguchi, Nature 419 (2002) 587–593.
[29] J. Eswaran, C. Hughes, V. Koronakis, J. Mol. Biol. 327 (2003) 309–315.
[30] C. Andersen, C. Hughes, V. Koronakis, J. Membr. Biol. 185 (2002) 83–92.
[31] C. Andersen, E. Koronakis, C. Hughes, V. Koronakis, Mol. Microbiol. 44 (2002) 1131–1139.
[32] T. Dinh, I.T. Paulsen, M.H.J. Saier, J. Bacteriol. 176 (1994) 3825–3831.
[33] H.I. Zgurskaya, H. Nikaido, Proc. Natl. Acad. Sci. U. S. A. 96 (1999) 7190–7195.
[34] R. Schulein, I. Gentschev, S. Schlör, R. Gross, W. Goebel, Mol. Gen. Genet. 245 (1994) 203–211.
[35] J. Hwang, P.C. Tai, Curr. Microbiol. 39 (1999) 195–199.
[36] C.A. Elkins, H. Nikaido, J. Bacteriol. 185 (2003) 5349–5356.
[37] H.I. Zgurskaya, H. Nikaido, J. Mol. Biol. 285 (1999) 409–420.
[38] S. Létoffé, P. Delepelaire, C. Wandersman, EMBO J. 15 (1996) 5804–5811.
[39] L. Balakrishnan, C. Hughes, V. Koronakis, J. Mol. Biol. 313 (2001) 501–510.
[40] S. Letoffe, P. Delepelaire, C. Wandersman, EMBO J. 15 (1996) 5804–5811.
[41] I.D. Kerr, Biochim. Biophys. Acta 1561 (2002) 47–64.
[42] L. Schmitt, H. Benabdelhak, M.A. Blight, I.B. Holland, M.T. Stubbs, J. Mol. Biol. 330 (2003) 333–342.
[43] H. Benabdelhak, S. Kiontke, C. Horn, R. Ernst, M.A. Blight, I.B. Holland, L. Schmitt, J. Mol. Biol. 327 (2003) 1169–1179.
[44] F. Zhang, J.A. Sheps, V. Ling, J. Biol. Chem. 268 (1993) 19889–19895.
[45] R. Binet, C. Wandersman, EMBO J. 14 (1995) 2298–2306.
[46] V. Koronakis, C. Hughes, E. Koronakis, Mol. Microbiol. 8 (1993) 1163–1175.
[47] P. Delepelaire, J. Biol. Chem. 269 (1994) 27952–27957.
[48] N. Izadi, Y. Henry, J. Haladjian, M.E. Goldberg, C. Wandersman, M. Delepierre, A. Lecroisey, Biochemistry 36 (1997) 7050–7057.
[49] P. Arnoux, R. Haser, N. Izadi, A.D. Lecroisey, M. Delepierre, C. Wandersman, M. Czjzek, Nat. Struct. Biol. 6 (1999) 516–520.
[50] L. Debarbieux, C. Wandersman, EMBO J. 20 (2001) 4657–4663.
[51] P. Delepelaire, C. Wandersman, EMBO J. 17 (1998) 936–944.
[52] G. Sapriel, C. Wandersman, P. Delepelaire, J. Bacteriol. 185 (2003) 80–88.
[53] N. Wolff, G. Sapriel, C. Bodenreider, A. Chaffotte, P. Delepelaire, J. Biol. Chem. 278 (2003) 38247–38253.
[54] D.N. Collier, V.A. Bankaitis, J.B. Weiss, P.J. Bassford Jr., Cell 53 (1988) 273–283.
[55] E.H. Manting, A.J. Driessen, Mol. Microbiol. 37 (2000) 226–228.
[56] A.J. Driessen, Trends Microbiol. 9 (2001) 193–196.
[57] G. Sapriel, C. Wandersman, P. Delepelaire, J. Biol. Chem. 277 (2002) 6726–6732.
[58] L.A. Fernandez, V. de Lorenzo, Mol. Microbiol. 40 (2001) 332–346.
[59] E.J. Stewart, F. Aslund, J. Beckwith, EMBO J. 17 (1998) 5543–5550.
[60] J.C. Bardwell, K. McGovern, J. Beckwith, Cell 67 (1991) 581–589.
[61] P.L. Stanley, P. Diaz, M.J. Bailey, D. Gygi, A. Juarez, C. Hughes, Mol. Microbiol. 10 (1993) 781–787.
[62] M.E. Bauer, R.A. Welch, Infect. Immun. 65 (1997) 2218–2224.
[63] S. Létoffé, C. Wandersman, J. Bacteriol. 174 (1992) 4920–4927.
[64] F. Zhang, Y. Yin, C.H. Arrowsmith, V. Ling, Biochemistry 34 (1995) 4193–4201.
[65] N. Wolff, J.M. Ghigo, P. Delepelaire, C. Wandersman, M. Delepierre, Biochemistry 33 (1994) 6792–6801.
[66] N. Wolff, P. Delepelaire, J.M. Ghigo, M. Delepierre, Eur. J. Biochem. 243 (1997) 400–407.
[67] D. Hui, V. Ling, Biochemistry 41 (2002) 5333–5339.
[68] D. Hui, C. Morden, F. Zhang, V. Ling, J. Biol. Chem. 275 (2000) 2713–2720.
[69] J.M. Ghigo, C. Wandersman, J. Biol. Chem. 269 (1994) 8979–8985.
[70] K. Omori, A. Idei, H. Akatsuka, J. Biol. Chem. 276 (2001) 27111–27119.
[71] Q. Ma, Y. Zhai, J.C. Schneider, T.M. Ramseier, M.H. Saier Jr., Biochim. Biophys. Acta 1611 (2003) 223–233.
[72] W.H. Bingle, J.F. Nomellini, J. Smit, J. Bacteriol. 182 (2000) 3298–3301.
[73] P. Awram, J. Smit, J. Bacteriol. 180 (1998) 3062–3069.
[74] H.P. Hahn, B.U. von Specht, FEMS Immunol. Med. Microbiol. 37 (2003) 87–98.
[75] H. Yamanaka, T. Nomura, Y. Fujii, K. Okamoto, Microb. Pathog. 25 (1998) 111–120.
[76] D.T. Foreman, Y. Martinez, G. Coombs, A. Torres, Y.M. Kupersztoch, Mol. Microbiol. 18 (1995) 237–245.
[77] H. Yamanaka, K. Okamoto, Microbiol. Immunol. 44 (2000) 481–488.
[78] J. Nishi, J. Sheikh, K. Mizuguchi, B. Luisi, V. Burland, A. Boutin, D.J. Rose, F.R. Blattner, J.P. Nataro, J. Biol. Chem. 278 (2003) 45680–45689.
