
Enzyme Catalysis and Regulation:  
The Role of the Novel Fem Protein VanK in  
Vancomycin Resistance in *Streptomyces*  
*coelicolor*

Hee-Jeon Hong, Matthew I. Hutchings, Lionel  
M. Hill and Mark J. Buttner  
*J. Biol. Chem.* 2005, 280:13055-13061.  
doi: 10.1074/jbc.M413801200 originally published online January 4, 2005

---

Access the most updated version of this article at doi: [10.1074/jbc.M413801200](http://dx.doi.org/10.1074/jbc.M413801200)

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinity-sites).

Alerts:
- [When this article is cited](#)
- [When a correction for this article is posted](#)

[Click here](#) to choose from all of JBC's e-mail alerts

This article cites 39 references, 18 of which can be accessed free at  
[http://www.jbc.org/content/280/13/13055.full.html#ref-list-1](http://www.jbc.org/content/280/13/13055.full.html#ref-list-1)

THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2005 by The American Society for Biochemistry and Molecular Biology, Inc.
Vol. 280, No. 13, Issue of April 1, pp. 13055–13061, 2005
Printed in U.S.A.

# The Role of the Novel Fem Protein VanK in Vancomycin Resistance in Streptomyces coelicolor*

Received for publication, December 8, 2004
Published, JBC Papers in Press, January 4, 2005, DOI 10.1074/jbc.M413801200

Hee-Jeon Hong‡§∥, Matthew I. Hutchings‡§, Lionel M. Hill∥, and Mark J. Buttner‡

From the ‡Departments of Molecular Microbiology and ||Metabolic Biology, John Innes Centre, Colney, Norwich NR4 7UH, United Kingdom

The non-pathogenic, non-glycopeptide-producing actinomycete *Streptomyces coelicolor* carries a cluster of seven genes (*vanSRJKHAX*) that confers inducible, high level resistance to vancomycin. The *vanK* gene has no counterpart in previously characterized vancomycin resistance clusters, yet *vanK* is required for vancomycin resistance in *S. coelicolor*. VanK belongs to the Fem family of enzymes, which add the branch amino acid(s) to the stem pentapeptide of peptidoglycan precursors. Upon exposure to vancomycin, the VanRS two-component system switches on expression of all seven *van* genes, and the VanHAX enzymes reprogram the cell wall such that precursors terminate D-Ala-D-lactate (Lac) rather than D-Ala-D-Ala, thus conferring resistance to vancomycin, which only binds D-Ala-D-Ala-containing precursors. Here we provide biochemical and genetic evidence that VanK is required for vancomycin resistance because the constitutively expressed FemX enzyme, encoded elsewhere on the chromosome, cannot recognize D-Lac-containing precursors as a substrate, whereas VanK can. Consistent with this view, D-Lac-containing precursors carrying the Gly branch are present in the wild type transiently exposed to vancomycin but are undetectable in a *vanK* mutant treated in the same way. Further, *femX* null mutants are viable in the presence of vancomycin but die in its absence. Because only VanK can recognize D-Lac-containing precursors, vancomycin-induced expression of VanHAX in a *vanK* mutant is lethal, and so *vanK* is required for vancomycin resistance.

wall biosynthesis in Gram-positive bacteria but not in Gram-negative bacteria because they cannot penetrate the outer membrane permeability barrier. They bind the D-alanyl-D-alanine (D-Ala-D-Ala) terminus of lipid-attached peptidoglycan precursors on the outside of the cytoplasmic membrane (5, 6), and this interaction blocks the formation of mature peptidoglycan, principally by denying transpeptidase access to its substrate, thus preventing formation of the peptide cross-links between polysaccharide strands that give the cell wall its structural rigidity. Because of the distinctive mode of action of vancomycin, mutations in transpeptidase cannot give rise to drug resistance. For this reason, it was originally suggested that pathogens might never acquire resistance to vancomycin because it would require them to remodel the peptidoglycan biosynthetic pathway itself. In the late 1980s, however, the first clinical isolates of VRE appeared and were found to reprogram cell wall biosynthesis such that the pendant pentapeptide of peptidoglycan precursors terminated in D-alanyl-D-lactate (D-Ala-D-Lac), rather than in D-Ala-D-Ala (7–12). The affinity of vancomycin for precursors terminating in D-Ala-D-Lac is ~1000-fold lower than for precursors terminating D-Ala-D-Ala (7), rendering the modified bacteria resistant. This remodeling of cell wall precursors requires three enzymes: VanH, which converts pyruvate into D-lactate; VanA, a D-Ala-D-Lac ligase; and VanX, a D-Ala-D-Ala dipeptidase that cleaves any residual circulating D-Ala-D-Ala dipeptide, ensuring that peptidoglycan precursors terminate uniformly in D-Ala-D-Lac.

We have shown previously that the non-pathogen *Streptomyces coelicolor* carries a gene cluster conferring inducible, high-level resistance to vancomycin (13). *S. coelicolor* is the model species of a genus of Gram-positive, mycelial soil bacteria responsible for the production of two-thirds of the commercially important antibiotics. *S. coelicolor* itself does not make a glycopeptide, but all of the known glycopeptide antibiotics are produced by actinomycetes, the family to which the streptomycetes belong. Because most non-pathogenic actinomycetes live in the soil, it seems likely that *S. coelicolor* encounters glycopeptide producers and that the *van* gene cluster therefore confers a selective advantage. Further, it is widely believed that all glycopeptide resistance genes are ultimately derived from actinomycete glycopeptide producers (14), which must carry these genes to avoid autotoxicity. Consistent with this idea, the *S. coelicolor* resistance genes are clearly associated with a laterally acquired DNA element.²

The spread of vancomycin resistance among pathogenic bacteria is an important public health concern. Ever since vancomycin-resistant strains of pathogenic *Enterococcus faecalis* and *Enterococcus faecium* (VRE) first emerged in the late 1980s, the intergeneric transfer of vancomycin resistance from these strains to methicillin-resistant *Staphylococcus aureus*, a major killer in hospital-acquired infections, has been widely anticipated. This recently became a reality with the first reports of clinical isolates of vancomycin-resistant *Staphylococcus aureus* (VRSA)¹ from hospitals in the United States (1–4).

Vancomycin and other glycopeptide antibiotics inhibit cell

* This work was funded by Biotechnology and Biological Sciences Research Council Grant 208/P20040 (to H.-J. H. and M. J. B.) and by a grant-in-aid to the John Innes Centre from the BBSRC. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
§ These two authors contributed equally to this work.
∥ To whom correspondence should be addressed: Dept. of Molecular Microbiology, John Innes Centre, Colney, Norwich NR4 7UH, UK. Tel.: 44-1603-450757; Fax: 44-1603-450778; E-mail: Hong@bbsrc.ac.uk.
¹ The abbreviations used are: VRSA, vancomycin-resistant *S. aureus*; Lac, lactate; TES, 2-{[2-hydroxy-1,1-bis(hydroxymethyl)ethyl]amino}-
² H.-J. Hong, unpublished results.

This paper is available on line at http://www.jbc.org 13055

Vancomycin Resistance in S. coelicolor

![Diagram](https://i.imgur.com/1234567.png)

FIG. 1. Organization and regulation of the vancomycin resistance (*van*) gene cluster of *S. coelicolor*. The *van* genes are organized into four transcription units, *vanRS*, *vanJ*, *vanK*, and *vanHAX*, and these transcripts are induced by vancomycin in a *vanR*-dependent manner. *vanJ* and *vanK* are novel genes with no counterpart in previously characterized vancomycin resistance clusters from pathogens. *vanK* is essential for vancomycin resistance, and encodes a member of the Fem family of enzymes that add the branch amino acids to the stem pentapeptide of cell wall precursors.

duction of the seven *van* genes in response to extracellular vancomycin. *vanJ* and *vanK* are particularly interesting because they are novel genes with no counterpart in previously characterized vancomycin-resistance clusters from pathogens. *vanJ*, which encodes a predicted membrane protein, is not required for vancomycin resistance, but *vanK* was found to be essential for vancomycin resistance (13).

VanK is a member of the Fem family of enzymes, which are non-ribosomal peptidyltransferases that add the branch amino acid(s) to the stem pentapeptide of peptidoglycan precursors. For example, *S. aureus* has a pentaglycine branch and three Fem proteins, FemA, -B, and -X, are involved in its synthesis (15–18). Rohrer et al. (19) showed that *femX* (also called *fmhB*) is an essential gene in *S. aureus* and that controlled depletion of its expression results in the disappearance of glycine-substituted peptidoglycan precursors. In contrast, *femA*, *femB*, and *femAB* mutants are viable; disruption of *femB* leads to shortening of the staphylococcal branch from five to three glycines, whereas loss of *femA* or *femAB* reduces the branch to a single glycine. From these results it was deduced that FemX adds the first glycine, FemA adds the second and third, and FemB adds the fourth and fifth (17, 19–22). This has now been demonstrated in a purified *in vitro* system, and staphylococcal FemX has been shown to use lipid II exclusively as acceptor for the first glycine (18). Interestingly, in *Streptomyces* the branch is a single glycine (23–25), yet *S. coelicolor* has three *fem* genes: *vanK*, SCO3904 (here designated *femX*), and SCO0602 (here designated *femY*).

The importance of the peptidoglycan remodeling enzymes VanHAX and the VanRS two-component signal transduction system was expected. However, the discovery that VanK was essential for resistance in *S. coelicolor* was surprising given the absence of *vanK* homologues in the vancomycin-resistance clusters of pathogens. Because VanK is a member of the Fem family of proteins, we proposed two alternative hypotheses to explain why *vanK* is required for vancomycin resistance (13). One was that *S. coelicolor* needs to change the nature of the peptidoglycan precursor branch to attain resistance. Changes in the sequence of the branch have been shown to result in increased levels of glycopeptide resistance in *Staphylococcus haemolyticus* (26). The second hypothesis was that the constitutive FemX activity of *S. coelicolor* can recognize only precursors that terminate D-Ala-D-Ala as a substrate, and VanK might therefore be required for vancomycin resistance because it is the only enzyme that can add the Gly branch to precursors terminating D-Ala-D-Lac (production of precursors lacking a branch is lethal in many bacteria because it prevents cross-linking of the peptidoglycan by transpeptidase, leading to cell lysis). Here we describe experiments that establish the second explanation.

### EXPERIMENTAL PROCEDURES

#### Bacterial Strains, Plasmids, Oligonucleotides, and Growth Conditions—Strains, plasmids, and oligonucleotides used are described in Table I. For cell wall precursor analysis, *S. coelicolor* spores were germinated by heat-shock treatment in 5 ml of TES buffer (0.05 M, pH 8) at 50 °C for 10 min and then diluted with an equal volume of double-strength germination medium (1% (w/v) Difco yeast extract, 1% (w/v) Difco casaminoacids, 0.01 M CaCl₂) and incubated with shaking at 37 °C for 2–3 h (27). New minimal medium with phosphate (27) was inoculated with germinated spores, and the cultures were grown to mid-log phase (*A*₅₀₀ of 0.3–0.6) at 30 °C. For the preparation of vancomycin-induced peptidoglycan precursors, cultures were grown to mid-log phase in NMMP and then exposed to vancomycin (50 μg/ml) for 3–4 h before harvesting. Other media and culture conditions for bacterial growth were as described previously (13, 28).

#### Construction of *femX*, *femY*, and *vanK* Mutants—*femX* (SCO3904) and *femY* (SCO0602) null mutants, in which the entire coding sequence was replaced with a cassette carrying the apramycin resistance gene (*apr*) and *oriT* of RK2, were constructed by PCR targeting of cosmids H24 (*femX*) and F55 (*femY*) (29, 30). The appropriate cosmid was introduced into *Escherichia coli* BW25113 (31) carrying pIJ790 (29), and the target gene was disrupted by electroporation of the cells with the PCR-amplified *apr-oriT* cassette generated using the primers femX KO I and femX KO II for *femX*, and femY KO I and femY KO II for *femY* (Table I). The resulting cosmids (H24Δ*femX*::apr and F55Δ*femY*::apr) were introduced into *E. coli* ET12567 carrying pUZ8002 (32) and transferred into *S. coelicolor* M600 by conjugation, selecting for apramycin resistance. In the case of *femY*, apramycin-resistant (*Apr*ᴿ), kanamycin-sensitive (*Kan*ˢ) exconjugants were identified and purified; the disruption was confirmed by PCR and Southern analysis, and a representative *femY* mutant was designated J3131 (Table I). In the case of *femX*, despite extensive screening, no *Apr*ᴿ *Kan*ˢ colonies were detected on medium containing apramycin alone, suggesting that disruption of SCO3904 under these conditions was lethal. However, when spores isolated from *Apr*ᴿ *Kan*ᴿ colonies were restreaked on medium containing vancomycin in addition to apramycin, *Apr*ᴿ *Kan*ˢ colonies were readily detected, and the successful disruption of *femX* was confirmed by PCR and Southern blotting. A representative *femX* mutant was designated J3130 (Table I). The *vanK* in-frame deletion mutant J3230 was derived from the Δ*vanK*::apr strain J3221 by the method of Gust et al. (29).

#### Constitutive Expression of *vanK* in the *femX* Mutant—To drive constitutive expression of *vanK*, the gene was cloned under the control of

Vancomycin Resistance in S. coelicolor

TABLE I  
Bacterial strains, plasmids, and primers used in this study

| Strains | Relevant genotype/comments | Source/reference |
|---------|-----------------------------|------------------|
| S. coelicolor A3(2) | SCP1⁻SCP2⁻ | 27 |
| M600 | ΔvanK::apr SCP1⁻SCP2⁻ | 13 |
| J3221 | ΔvanK SCP1⁻SCP2⁻ | This study |
| J3230 | ΔfemX::apr SCP1⁻SCP2⁻ | This study |
| J3130 | ΔfemY::apr SCP1⁻SCP2⁻ | This study |
| E. coli | ET12567 containing helper plasmid pUZ8002 | 32 |
| BT340 | Temperature-sensitive FLP recombination plasmid | 31 |
| Plasmids | pBluescript KS(+) containing the apramycin resistance gene *apr* and the *oriT* of | 29 |
| pIJ773 | plasmid RP4, flanked by FRT sites |  |
| pIJ790 | Modified λ RED recombination plasmid pKD20 | 29 |
| pIJ8723 | Derivative of pSET152 containing *ermEp** and a synthetic polylinker | M.J. Butler |
| pIJ10250 | pGEMT carrying a 1.2-kb *vanK* PCR product (NdeI-BglII) | This study |
| pIJ10257 | 330-bp *ermEp** (KpnI-PstI) with ribosome binding site and multicloning site from pIJ8723 cloned into pMS81 cut with KpnI-NsiI | This study |
| pIJ10267 | *vanK* cloned in pIJ10257 under *ermEp** control | This study |
| pMS81 | φBT1attP-int derived integration vector for the conjugal transfer of DNA from *E. coli* to Streptomyces spp. (HygR) | 34 |
| Primers | femXKOI | ACCCATGGGGGACACACCAGCCCGAGGAGCGCCCCGAATGATCCGGGGATCCGTGCGACC |
| femXKOII | GCTGTCAGAGGTGCGGATCGGGGATGGGCGGTCATGTAGGCTGGAGCTGCTTCC |
| femYKOI | CCGTCGCAACGTGCAAACGGAACAGAGGGACACCCCATTCCGGGGATCCGTGCGACC |
| femYKOII | ACCCCGCCTCACCTCGTCCGGACCCGGCGTCGCGGGGCTGTAGGCTGGAGCTGCTTC |
| femXtestI | CCCGAGGAGCGCCCCGAATG |
| femXtestII | GGGGATGGGCGGTGCGGTCA |
| femYtestI | TGTTCCCCCTGTGAGGTGTGCG |
| femYtestII | GCGGTCAGCCAGTCACCGAA |

the *ermEp** promoter (33). A 330-bp KpnI-PstI fragment carrying *ermEp**, a ribosome binding site and a multicloning site, was isolated from pIJ8723,³ and cloned into pMS81 (34) cut with KpnI-NsiI to create pIJ10257. *vanK* was amplified by PCR incorporating NdeI and BglII sites upstream and downstream, respectively, of the gene and cloned into pGEM-T easy (Promega) to create pIJ10250. *vanK* was isolated from pIJ10250 as a 1.2-kb NdeI-Sall fragment and inserted into pIJ10257 digested with NdeI-XhoI to create pIJ10267. pIJ10267 was introduced into the *femX* mutant J3130 by conjugal transfer from *E. coli* strain ET12567 (pUZ8002), and exconjugants were selected with hygromycin.

Liquid Chromatography-MS Analysis of Peptidoglycan Precursors—Cytoplasmic UDP-linked peptidoglycan precursors were extracted using a modification of the method of Ruzin et al. (24). Mycelium was harvested by centrifugation at 4°C and washed in 0.9% NaCl. The centrifuged mycelial pellet was extracted with 5% ice-cold trichloroacetic acid for 30 min at 4°C, and the extract was desalted on a Sephadex G-25 column (PD-10 Columns, Amersham Biosciences). The desalted extract was concentrated by rotary evaporation and resuspended in a small volume of HPLC-grade deionized water. Precursors were separated by reverse-phase chromatography on a 100 × 2 mm 3 μLuna C₁₈ (2) column (Phenomenex) using a gradient (5–40%: buffer A, 0.1% formic acid in water; buffer B, 0.1% formic acid in methanol), and the elution of precursors was monitored at 254 nm and by selected reaction monitoring in a Thermo Finnigan LCQ DecaXPplus ion trap liquid chromatography-MS system.

Precursors terminating in D-Ala-D-Ala separated from those terminating D-Ala-D-Lac, but the addition of a glycine side chain caused little change in retention time. Therefore precursors with and without the side chain were detected by trapping ions of masses 1194.1, 1195.1, 1251.1, and 1252.1 (isolation width 4 atomic mass units) at appropriate retention times, and observing the major fragment, a loss of 404 atomic mass units (collision energy 35%). Because isotopic standards were not available, we carried out mixing experiments in which samples known to contain particular precursors were mixed prior to analysis with those that apparently did not. If the apparent absence of a precursor had been an artifact caused by the suppression of its ion by another metabolite, mixtures would also have shown a reduced signal. In fact, the mixtures gave signals within 15% of that expected, indicating that this method, which is highly specific, is also sufficient for the approximate relative quantification of the four targeted precursors.

RESULTS

The Stem Peptide Branch Does Not Change in Response to Vancomycin—To test the effect of vancomycin on the stem peptide branch, peptidoglycan was isolated from *S. coelicolor* grown in the presence or absence of vancomycin, and the 25 most abundant muropeptides derived from each of these two samples were separated by HPLC and characterized by mass spectrometry. The branch associated with each muropeptide was a single glycine in all cases, both before and after exposure to vancomycin.⁴ Therefore VanK does not change the stem peptide branch.

Precursor Analysis Suggests that VanK Is Required to Add the Gly Branch to Stem Peptides Terminating D-Ala-D-Lac—Because the Δ*vanK* mutant is sensitive to vancomycin, it was not possible to examine the effect of vancomycin induction on its cell wall structure. However, because cytoplasmic cell wall precursor pools change rapidly in response to vancomycin (35), it was possible to examine the effect of vancomycin induction on these precursor pools in the Δ*vanK* mutant.

Wild-type *S. coelicolor* and the congenic Δ*vanK* mutant were grown to mid-log phase in NMMP medium (27). For the preparation of vancomycin-induced peptidoglycan precursors, cultures were then exposed to vancomycin (50 μg/ml) for 3–4 h before harvesting. Peptidoglycan precursors were extracted, separated by HPLC, and analyzed by mass spectrometry (Fig. 2). The structures of the four relevant molecules are given in Fig. 3. As expected, in the absence of vancomycin, when the *van* genes were not expressed, no significant differences were seen between the wild-type and the Δ*vanK* mutant (Fig. 2A). Under the conditions used, precursors with and without the glycine branch co-eluted. Thus, peak 1, with a retention time of 13.5 min, contained two molecules: a 1194-Da species that matched the predicted mass of Molecule I, and a second species of 1251 Da that matched the predicted mass of Molecule III (Figs. 2 and

⁴ H.-J. Hong, A. Severin, K. Tabei, and M. J. Buttner, unpublished data.

13058 Vancomycin Resistance in S. coelicolor

**A** – vancomycin  
**B** + vancomycin  

wild type  
$\Delta vanK$  
wild type  
$\Delta vanK$  

Absorbance  
(254 nm)  

Molecule I  
(mass 1194)  

Molecule II  
(mass 1195)  

Molecule III  
(mass 1251)  

Molecule IV  
(mass 1252)  

Retention time (min)  
Retention time (min)  
Retention time (min)  
Retention time (min)  

FIG. 2. HPLC separation of cytoplasmic peptidoglycan nucleotide precursors isolated from wild-type S. coelicolor M600 and a congenic $\Delta vanK$ mutant with (B) or without (A) exposure to vancomycin. For the preparation of vancomycin-induced peptidoglycan precursors, cells were grown to mid-log phase and then exposed to vancomycin (50 $\mu$g/ml) for 3–4 h before harvesting. UV traces ($\text{Abs}_{254}$) are shown above the selected ion chromatograms for masses 1194, 1195, 1251, and 1252 (corresponding to Molecules I, II, III, and IV in Fig. 3). The two vertical arrows above the UV traces indicate peaks corresponding to vancomycin itself. Note that the masses of the observed ions are 1 Da greater than those of the neutral structures because of the addition of $\text{H}^+$.

UDP-MurNAc  
L-ala  
D-glu  
LL-dpa  
D-ala  
D-ala  

UDP-MurNAc  
L-ala  
D-glu  
LL-dpa  
D-ala  
D-lac  

UDP-MurNAc  
L-ala  
D-glu  
LL-dpa — gly  
D-ala  
D-ala  

UDP-MurNAc  
L-ala  
D-glu  
LL-dpa — gly  
D-ala  
D-lac  

Molecule I  
(mass 1194)  

Molecule II  
(mass 1195)  

Molecule III  
(mass 1251)  

Molecule IV  
(mass 1252)  

A femX Mutant Is Vancomycin-dependent—Given that Molecule IV (Fig. 3) was present in the wild type exposed to vancomycin but was undetectable in the $\Delta vanK$ mutant treated in the same way, we reasoned that the role of VanK might be to add the Gly branch to D-Lac-containing precursors and that the constitutively expressed FemX enzyme probably cannot recognize these precursors as a substrate. Because the Gly branch is likely to be essential for peptidoglycan cross-linking, and hence cell viability, VanK would therefore be required for vancomycin resistance. If this hypothesis is true, it follows that $femX$, an essential gene in S. aureus (19), would instead be conditionally essential in S. coelicolor. In other words, if VanK substitutes for FemX when the cell accumulates D-Ala-D-Lac-containing precursors, FemX would be redundant when expression of the $van$ genes is activated by vancomycin (Fig. 5A). To test this hypothesis, we attempted to disrupt $femX$ (SCO3904) in the presence and absence of vancomycin, using the PCR-directed approach of Gust et al. (29). In the first step of this method, S. coelicolor genes carried on cosmids in E. coli are replaced with a selectable marker generated by PCR. For our purposes, this method had the advantage that it permits the ready analysis of essential genes because the initial gene disruption is performed in E. coli, rather than in S. coelicolor.

A cosmid carrying a $\Delta femX::apr$ gene replacement in the insert and a kanamycin resistance ($\text{Kan}^R$) gene in the vector was introduced into S. coelicolor by conjugation, selecting for apramycin resistance ($\text{Apr}^R$). All the $\text{Apr}^R$ primary exconjugants were $\text{Kan}^R$, and despite extensive screening, no $\text{Apr}^R$

Vancomycin Resistance in S. coelicolor

A Molecule I
B Molecule II

FIG. 4. Characterization by electrospray ionization/MS/MS fragmentation of (A) Molecule I and (B) Molecule II. Each molecule was trapped in a DecaXP<sup>plus</sup> ion trap with an isolation width of 4.0 atomic mass units and fragmented at 40% collision energy to produce an MS<sup>2</sup> spectrum. The predominant products, with a loss of 404 atomic mass units, were trapped and fragmented further under the same conditions to yield MS<sup>3</sup> spectra. The probable identity of fragments found is shown below; as expected, most are breakages of peptide bonds. Note that breakage of the D-Ala-D-Ala peptide bond in Molecule I is observed but that breakage of the D-Ala-D-Lac ester bond in Molecule II is not detected. The masses of the observed fragment ions are 1 Da greater than those of the neutral structures because of the addition of H<sup>+</sup>.

Kan<sup>S</sup> isolates were detected on subculture, implying that *femX* might be essential under these conditions. However, when vancomycin was added to the plates, Apr<sup>R</sup> Kan<sup>S</sup> colonies (double crossovers) were readily isolated, and these were confirmed as *femX* mutants by PCR and Southern blot analyses. Significantly, when the *femX* mutants were plated back on medium lacking vancomycin, they failed to grow, confirming that *femX* is essential in the absence of vancomycin but dispensable in its presence (Fig. 5B).

*femY*, a Third *fem* Gene in *S. coelicolor*—In addition to *vanK* and *femX*, there is a third *fem* gene (SCO0602) elsewhere on the *S. coelicolor* chromosome, which we have designated *femY*. We could detect no *femY* transcript by S1 nuclease protection assay (in either the presence or absence of vancomycin), and disruption of the gene had no obvious phenotypic consequences. Perhaps significantly, *femY* lies in one of the arms of the linear chromosome, regions that can be lost without affecting viability under laboratory conditions and that contain a preponderance of conditionally adaptive genes (36), whereas *vanK* and *femX* lie in the core of the chromosome.

VanK Can Recognize Precursors Terminating D-Ala-D-Ala—We also wanted to determine whether VanK could recognize D-Ala-D-Ala-containing precursors as substrates. The *van* genes are not expressed in the absence of vancomycin (13). Thus, VanK is not normally exposed to D-Ala-D-Ala-containing precursors because vancomycin induces expression of VanHAX at the same time as VanK, switching precursor pools from D-Ala-D-Ala- to D-Ala-D-Lac-terminating stem peptides as VanK activity appears. To express *vanK* in the absence of *vanHAX* expression, we constructed a plasmid in which *vanK* was cloned under the control of the constitutive *ermEp*<sup>*</sup> promoter and introduced it into the vancomycin-dependent *femX* null mutant. Constitutive expression of *vanK* allowed the *femX* null mutant to grow readily in the absence of vancomycin (Fig. 6A), suggesting that VanK can recognize precursors terminating D-Ala-D-Ala as substrate. To confirm this interpretation, we grew the *femX* null mutant carrying the *ermEp*-vanK construct in the absence of vancomycin, extracted peptidoglycan precursors, separated them by HPLC, and analyzed them by mass spectrometry (Fig. 6B). As expected, a peak with a retention time of 13.5 min (equivalent to peak 1 in Fig. 2) was observed, containing Molecule I and Molecule III (Figs. 3 and 6B), showing that VanK can add the glycine branch to D-Ala-D-Ala-containing precursors.

DISCUSSION

Our interpretation of the data given in this paper is as follows. FemX adds the single branch glycine to the stem peptide of *S. coelicolor* cell wall precursors terminating D-Ala-D-Ala (Fig. 7). Because transpeptidation of this branch is essential for mature cell wall formation, FemX is essential under “normal” conditions. However, induction of the *vanHAX* genes remodels cell wall precursors such that the stem peptide terminates D-Ala-D-Lac instead of D-Ala-D-Ala, and FemX cannot recognize D-Lac-containing precursors. Instead, the FemX homologue, VanK, recognizes D-Lac-containing precursors (Fig. 7). The *femX* gene is therefore non-essential provided the *van* genes are expressed. As a consequence, *femX* null mutants are viable in the presence of vancomycin but die in its absence. Because only VanK can recognize D-Lac-containing precursors, expression of VanHAX in a *vanK* mutant is lethal, and so *vanK* is required for vancomycin resistance.

It is likely that a VanK-like enzyme is important for glycopeptide resistance not just in *S. coelicolor* but more widely in the genus *Streptomyces*; significantly, an orthologue of *S. coelicolor vanK* is present in the resistance gene set associated with the biosynthetic cluster for the glycopeptide antibiotic A47934 in *Streptomyces toyocaensis* (37).

Available evidence suggests that FemX of enterococci can efficiently recognize D-Lac-containing substrates (38). However, analysis of the cell walls of the recently identified VRSA strains suggests that the substrate specificity of FemX in

13060

Vancomycin Resistance in S. coelicolor

A

- vancomycin → vanK → femX knockout → lethal  
+ vancomycin → vanK → femX knockout → viable  

B

ΔfemX | wild type  
ΔvanR | ΔvanR  

- vancomycin | + vancomycin  

FIG. 5. A, rationale for the attempted disruption of *femX*. *femX* is constitutively expressed, but *vanK* expression is induced by vancomycin. FemX adds the single branch glycine to the stem pentapeptide of *S. coelicolor* cell wall precursors terminating D-Ala-D-Ala (Fig. 7). Because transpeptidation of this branch is essential for cell wall formation, *femX* is an essential gene under normal conditions. However, the *van* cluster encodes a FemX homologue, VanK, which can recognize precursors terminating D-Ala-D-Lac as a substrate (Fig. 7). As a consequence, *femX* is non-essential provided that the *van* genes are expressed. Therefore, *femX* null mutants are viable in the presence of vancomycin but die in its absence. B, vancomycin-dependent growth of the Δ*femX* mutant. Wild type (M600) and Δ*vanR* and Δ*femX* mutant derivatives of M600 were streaked on soy flour mannitol agar in the absence (*left*) or presence (*right*) of vancomycin (10 μg/ml). Plates were photographed after incubation at 30 °C for 4 days.

staphylococci may lie somewhere between the two extremes represented by the enterococcal and the streptomycete enzymes. When VRSA strains were grown in the absence of vancomycin, they made the typical pentaglycine branch characteristic of staphylococci (35). However, muropeptides isolated from the same strains grown in the presence of vancomycin frequently had shorter branches or lacked a branch altogether (35). These results strongly suggest that D-Lac-containing precursors are poor substrates for FemX, FemA, and FemB, the three enzymes that build the pentaglycine branch in *S. aureus*, but not so poor as to block the synthesis of a branch altogether, and hence cause the death of the cell. Thus, enterococci, staphylococci, and streptomycetes seem to present a spectrum of FemX substrate specificities in which enterococcal FemX can recognize D-Lac-containing precursors efficiently, streptomycete FemX cannot (necessitating the presence of *vanK* under VanRS control), with the staphylococcal enzyme lying somewhere in between, able to contribute to the production of a viable cell wall, but one that is significantly defective in comparison to the wall produced in the absence of vancomycin. The VRSA isolates arose from intergeneric transfer of the vancomycin-resistance transposon Tn1546 from *E. faecalis* (3, 4). Our results and those of Severin et al. (35) suggest that the acquisition of a VanK-like activity by VRSA strains might lead to isolates yet more robustly resistant to vancomycin.

There is circumstantial evidence that the need to cope with D-Lac-containing substrates may present problems for members of other genera in addition to Streptomycetes and Staphylococcus. Actinoplanes teicomyceticus, an actinomycete relative of

Absorbance  
(254 nm)

Molecule I  
(mass 1194)

Molecule II  
(mass 1195)

Molecule III  
(mass 1251)

Molecule IV  
(mass 1252)

FIG. 6. A, constitutive expression of *vanK* rescues the vancomycin-dependent phenotype of a *femX* null mutant. The *femX* null mutant carrying the vector pIJ10257 dies in the absence of vancomycin (*left*), but the same strain carrying the vector with an *ermEp*-vanK insert is viable in the absence of the drug (*right*). B, HPLC separation of cytoplasmic peptidoglycan nucleotide precursors isolated from the *femX* null mutant carrying the *ermEp*-vanK construct grown in the absence of vancomycin. The UV trace is shown above the selected ion chromatograms for masses 1194, 1195, 1251, and 1252 (corresponding to Molecules I, II, II, and IV, respectively, in Fig. 3). Note that the masses of the observed ions are 1 Da greater than those of the neutral structures because of the addition of H⁺.

UDP-MurNAc  
L-ala  
D-glu  
LL-dpa → gly  
D-ala  
D-ala  

UDP-MurNAc  
L-ala  
D-glu  
LL-dpa → gly  
D-ala  
D-lac  

FIG. 7. The role of VanK in vancomycin resistance in *S. coelicolor*. VanK adds the Gly branch to cell wall precursors terminating D-Ala-D-Lac. In contrast, FemX catalyzes the incorporation of the Gly branch to precursors terminating D-Ala-D-Ala but cannot recognize precursors terminating D-Ala-D-Lac as a substrate. *vanK* is therefore required for vancomycin resistance in *S. coelicolor*.

*S. coelicolor*, produces the glycopeptide teicoplanin. The chromosome of *A. teicomyceticus* encodes (at least) two MurF proteins, one of them (MurF2) associated with the teicoplanin resistance genes and a second (MurF1) encoded elsewhere on

the chromosome (39). MurF catalyzes condensation of the dipeptide D-Ala-D-Ala with UDP-MurNAc-tripeptide to form the UDP-MurNAc-pentapeptide (40). By analogy with FemX/ VanK in *S. coelicolor*, one possible explanation for the presence of *murF2* in the teicoplanin resistance gene cluster is that MurF1 cannot efficiently recognize D-Ala-D-Lac as a substrate. Unfortunately, *A. teicomyceticus* is not currently genetically tractable, and so *murF2* cannot be disrupted to see if it is required for teicoplanin resistance (39).

Acknowledgments—We thank David Hopwood, Subray Hegde, Anatoly Severin, and Christopher Walsh for helpful discussion and/or comments on the manuscript.

REFERENCES

1. Pearson, H. (2002) *Nature* 418, 469–470
2. Chang, S., Sievert, D. M., Hageman, J. C., Boulton, M. L., Tenover, F. C., Downes, F. P., Shah, S., Rudrik, J. T., Pupp, G. R., Brown, W. J., Cardo, D., and Fridkin, S. K. (2003) *N. Engl. J. Med.* 348, 1342–1347
3. Tenover, F. C., Weigel, L. M., Appelbaum, P. C., McDougal L. K., Chaitram, J., McAllister, S., Clark, N., Killgore, G., O’Hara, C. M., Jevitt, L., Patel, J. B., and Bozdogan, B. (2004) *Antimicrob. Agents Chemother.* 48, 275–280
4. Weigel, L. M., Clewell, D. B., Gill, S. R., Clark, N. C., McDougal, L. K., Flannagan, S. E., Kolonay, J. F., Shetty, J., Killgore, G. E., and Tenover, F. C. (2003) *Science* 28, 1569–1571
5. Barna, J. C. J., and Williams, D. H. (1984) *Annu. Rev. Microbiol.* 38, 339–357
6. Williams, D. H., Williamson, M. P., Butcher, D. W., and Hammond, S. J. (1983) *J. Am. Chem. Soc.* 105, 1332–1339
7. Bugg, T. D. H., Wright, G. D., Dutka-Malen, S., Arthur, M., Courvalin, P., and Walsh, C. T. (1991) *Biochemistry* 30, 10408–10415
8. Healy, V. L., Lessard, I. A., Roper, D. I., Knox, J., and Walsh, C. T. (2000) *Chem. Biol.* 7, R109–R119
9. Pootoolal, J., Neu, J., and Wright, G. D. (2002) *Annu. Rev. Pharmacol. Toxicol.* 42, 381–408
10. Walsh, C. T. (2000) *Nature* 406, 775–781
11. Walsh, C. T., Fisher, S. L., Park, I. S., Prahalad, M., and Wu, Z. (1996) *Chem. Biol.* 3, 21–28
12. Williams, D. H., and Bardsley, B. (1999) *Angew Chem. Int. Ed. Engl.* 38, 1172–1193
13. Hong, H.-J., Hutchings, M. I., Neu, J. M., Wright, G. D., Paget, M. S. B., and Buttner, M. J. (2004) *Mol. Microbiol.* 52, 1107–1121
14. Davies, J. (1994) *Science* 264, 375–382
15. Hegde, S. S., and Blanchard, J. S. (2003) *J. Biol. Chem.* 278, 22861–22867
16. Hegde, S. S., and Shrader, T. E. (2001) *J. Biol. Chem.* 276, 6998–7003
17. Rohrer, S., and Berger-Bächi, B. (2003) *Antimicrob. Agents Chemother.* 47, 837–846
18. Schneider, T., Senn, M. M., Berger-Bachi, B., Tossi, A., Sahl, H. G., and Wiedemann, I. (2004) *Mol. Microbiol.* 53, 675–685
19. Rohrer, S., Ehlert, K., Tschierske, M., Labischinski, H., and Berger-Bächi, B. (1999) *Proc. Natl. Acad. Sci. U. S. A.* 96, 9351–9356
20. Ehlert, K., Schröder, W., and Labischinski, H. (1997) *J. Bacteriol.* 179, 7573–7576
21. Henze, U., Sidow, T., Wecke, J., Labischinski, H., and Berger-Bächi, B. (1993) *J. Bacteriol.* 175, 1612–1620
22. Kopp, U., Roos, M., Wecke, J., and Labischinski, H. (1996) *Microb. Drug Resist.* 2, 29–41
23. Leyh-Bouille, M., Bonaly, R., Ghuysen, J.-M., Tinelli, R., and Tipper, D. (1970) *Biochemistry* 9, 2944–2952
24. Ruzin, A., Severin, A., Ritacco, F., Tabei, K., Singh, G., Bradford, P. A., Siegel, M. M., Projan, S. J., and Shlaes, D. M. (2002) *J. Bacteriol.* 184, 2141–2147
25. Schleifer, K. H., and Kandler, O. (1972) *Bacteriol. Rev.* 36, 407–477
26. Billot-Klein, D., Gutmann, L., Bryant, D., Bell, D., Van Heijenoort, J., Grewal, J., and Shlaes, D. M. (1996) *J. Bacteriol.* 178, 4696–4703
27. Kieser, T., Bibb, M. J., Buttner, M. J., Chater, K. F., and Hopwood, D. A. (2000) *Practical Streptomyces Genetics*, The John Innes Foundation, Norwich, UK
28. Hong, H.-J., Paget, M. S. B., and Buttner, M. J. (2002) *Mol. Microbiol.* 44, 1199–1211
29. Gust, B., Challis, G. L., Fowler, K., Kieser, T., and Chater, K. F. (2003) *Proc. Natl. Acad. Sci. U. S. A.* 100, 1541–1546
30. Redenbach M., Kieser, H. J., Denapaite, D., Eichner, A., Cullum, J., Kinashi, H., and Hopwood, D. A. (1996) *Mol. Microbiol.* 21, 77–96
31. Datsenko, K. A., and Wanner, B. L. (2000) *Proc. Natl. Acad. Sci. U. S. A.* 97, 6640–6645
32. Paget, M. S. B., Chamberlin, L. Atrih, A., Foster, S. J., and Buttner, M. J. (1999) *J. Bacteriol.* 181, 204–211
33. Bibb, M. J., White, J., Ward, J. M., and Janssen, G. R. (1994) *Mol. Microbiol.* 14, 533–545
34. Gregory, M. A., Till, R., and Smith, M. C. (2003) *J. Bacteriol.* 185, 5320–5323
35. Severin, A., Tabei, K., Tenover, F., Chung, M., Clarke, N., and Tomasz, A. (2004) *J. Biol. Chem.* 279, 3398–3407
36. Bentley, S. D., Chater, K. F., Cerdeño-Tárraga, A. M., Challis, G. L., Thomson, N. R., James, K. D., Harris, D. E., Quail, M. A., Kieser, H., Harper, D., Bateman, A., Brown, S., Chandra, G., Chen, C. W., Collins, M., Cronin, A., Fraser, A., Goble, A., Hidalgo, J., Hornsby, T., Howard, S., Huang, C. H., Kieser, T., Larke, L., Murphy, L., Oliver, K., O’Neil, S., Rabbinowitsch, E., Rajandream, M. A., Rutherford, K., Rutter, S., Seeger, K., Saunders, D., Sharp, S., Squares, R., Squares, S., Taylor, K., Warren, T., Wietzorrek, A., Woodward, J., Barrell, B. G., Parkhill, J., and Hopwood, D. A. (2002) *Nature* 417, 141–147
37. Pootoolal, J., Thomas, M. G., Marshall, C. G., Neu, J. M., Hubbard, B. K., Walsh, C. T., and Wright, G. D. (2002) *Proc. Natl. Acad. Sci. U. S. A.* 99, 8962–8967
38. Bouhss, A., Josseaume, N., Severin, A., Tabei, K., Hugonnet, J.-E., Shlaes, D., Mengin-Lecreulx, D., van Heijenoort, J., and Arthur, M. (2002) *J. Biol. Chem.* 277, 45935–45941
39. Serina, S., Radice, F., Maffioli, S., Donadio, S., and Sosio, M. (2004) *FEMS Microbiol. Lett.* 240, 69–74
40. Bugg, T. D. H. (1999) in *Comprehensive Natural Products Chemistry* (Pinto, M., ed.) pp. 241–294, Vol. 3. Elsevier Science Ltd., Oxford, UK
