
Clinical Microbiology
Reviews

**Pathway to Synthesis and Processing of**
**Mycolic Acids in Mycobacterium**
**tuberculosis**

Kuni Takayama, Cindy Wang and Gurdyal S. Besra
Clin. Microbiol. Rev. 2005, 18(1):81. DOI: 10.1128/CMR.18.1.81-101.2005.

---

Updated information and services can be found at:
[http://cmr.asm.org/content/18/1/81](http://cmr.asm.org/content/18/1/81)

These include:

**REFERENCES**
This article cites 121 articles, 52 of which can be accessed free at: [http://cmr.asm.org/content/18/1/81#ref-list-1](http://cmr.asm.org/content/18/1/81#ref-list-1)

**CONTENT ALERTS**
Receive: RSS Feeds, eTOCs, free email alerts (when new articles cite this article), [more »](more%20%E2%86%92)

---

Information about commercial reprint orders: [http://journals.asm.org/site/misc/reprints.xhtml](http://journals.asm.org/site/misc/reprints.xhtml)
To subscribe to to another ASM Journal go to: [http://journals.asm.org/site/subscriptions/](http://journals.asm.org/site/subscriptions/)

---

Journals.ASM.org

CLINICAL MICROBIOLOGY REVIEWS, Jan. 2005, p. 81–101
0893-8512/05/$08.00+0 doi:10.1128/CMR.18.1.81–101.2005
Copyright © 2005, American Society for Microbiology. All Rights Reserved.

# Pathway to Synthesis and Processing of Mycolic Acids in *Mycobacterium tuberculosis*

Kuni Takayama,${}^{1,2,3*}$ Cindy Wang,${}^{1}$ and Gurdyal S. Besra${}^{4}$

Mycobacteriology Research Laboratory, William S. Middleton Memorial Veterans Hospital,${}^{1}$ Department of Medical Microbiology and Immunology, University of Wisconsin School of Medicine,${}^{2}$ and Department of Bacteriology, University of Wisconsin,${}^{3}$ Madison, Wisconsin, and School of Biosciences, The University of Birmingham, Edgbaston, Birmingham, United Kingdom${}^{4}$

## INTRODUCTION
82

### MYCOLIC ACIDS IN M. TUBERCULOSIS
82

### BIOSYNTHESIS OF MYCOLIC ACIDS IN M. TUBERCULOSIS
82
- Biosynthesis of Normal Fatty Acid Precursors of Mycolic Acids by FAS-I...
82
- The Basic FAS-II Module for Chain Elongation in the Biosynthesis of Meroacids...
83
- Biosynthesis of cis, cis-Diunsaturated Meroacid Precursors and α-Meroacid...
83
- Biosynthesis of Oxygenated Meroacids...
85
- Claisen-Type Condensation for the Synthesis of Mycolic Acids...
86
- Physical Organization of Mycolate Synthetases in *M. tuberculosis*...
87

### PROCESSING OF NEWLY SYNTHESIZED MYCOLIC ACIDS
88

### GENETIC ANALYSIS OF SYNTHESIS AND PROCESSING OF MYCOLIC ACID IN M. TUBERCULOSIS
88
- FAS-I...
88
- Transition from the FAS-I System to the FAS-II System...
89
  - AcpM...
89
  - mtFabD (malonyl-CoA:ACP transacylase)...
90
  - mtFabH (β-ketoacyl-ACP synthase-III)...
90

### FAS-II System
90
- KasA and KasB (β-ketoacyl-ACP synthases)...
90
- MabA/FabG1 (β-ketoacyl-ACP reductase)...
90
- β-Hydroxyacyl-ACP dehydrase...
90
- 2-trans-Enoyl-ACP isomerase...
91
- InhA (2-trans-enoyl-ACP reductase)...
92

### Cyclopropane Synthases and Methyltransferases
92
- Cloning of the genes in the path to cyclopropane synthases and methyltransferases...
92
- MmaA2 is required for introduction of the distal cyclopropane ring in the formation of α-meroacid...
93
- PcaA (UmaA2) is required for introduction of the proximal cyclopropane ring in the formation of α-meroacid...
93
- MmaA4 introduces the distal-branch methyl group in the formation of trans-oxygenated meroacids...
93
- MmaA3 O-methylates the distal secondary alcohol in the formation of cis- and trans-methoxy-meroacids...
93
- MmaA2 and CmaA2 are required for introduction of the proximal cis-cyclopropane ring in methoxy-meroacids, and Mma2 is required for introduction of the proximal cis-cyclopropane ring in keto-meroacids...
93
- MmaA1 and CmaA2 are required for introduction of the proximal-branch methyl group and trans-cyclopropane ring in trans-oxygenated meroacids...
94

### Claisen-Type Condensation
94

### Mycolic Acid Processing System
94
- Mycolyltransferase I and mycolyltransferase II...
94
- Phosphatase...
95
- ABC transporter...
95
- Antigen 85 complex...
97

## CONCLUDING REMARKS
98

## ACKNOWLEDGMENTS
98

## REFERENCES
98

* Corresponding author. Mailing address: Mycobacteriology Research Laboratory, William S. Middleton Memorial Veterans Hospital, 2500 Overlook Terrace, Madison, WI 53705. Phone: (608) 256-1901, ext. 17812. Fax: (608) 280-7108. E-mail: Kuni.Takayama@med.va.gov.

TAKAYAMA ET AL.

INTRODUCTION

Tuberculosis (TB) is one of the world's deadliest diseases (19). The Centers for Disease Control and Prevention has reported that each year, eight million people around the world become sick with TB and there are over two million TB-related deaths worldwide. Moreover, one-third of the world’s population is infected with TB. Thus, *Mycobacterium tuberculosis* (the causative agent of TB) is clearly the most predominant global human pathogen. Two decades ago, it was thought that TB was under control and that it was a matter of time before it would be eradicated. Today, this disease has reestablished itself due to several factors. The lack of drug compliance, the appearance of multiple-drug-resistant strains, and the AIDS epidemic are a few factors that have led to the resurgence in TB. Drug resistance arises following inadequate compliance, and AIDS patients with weakened immune system are very susceptible to *M. tuberculosis* and the usual cause of death.

The cell envelope of *M. tuberculosis* is distinctive and is associated with its pathogenicity. Features that are very prominent in the cell envelope are the presence of arabinogalactan-mycolate covalently linked to the cell wall peptidoglycan via a phosphodiester bond located on the inner leaflet of the outer membrane (9, 56, 63, 69) and of a free glycolipid called trehalose dimycolate (TDM), which accumulates in a cord-like fashion on the surface of the cells (72, 73). This provides a thick layer of lipid on the outer part of the cell and protects the tubercle bacillus from noxious chemicals and the host’s immune system. Mycolic acids are the major constituents of this protective layer. They also play other important roles as structural components of the cell wall and envelope (5, 11, 55, 65). More specifically, the cyclopropane rings in mycolic acids of *M. tuberculosis* contribute to the structural integrity of the cell wall complex (39) and protect the bacillus from oxidative stress (hydrogen peroxide) (123).

Deletion of the proximal cyclopropane ring of α-mycolic acid (41) or of methoxy- and keto-mycolates (31) in *M. tuberculosis* leads to a significant attenuation in growth of the two mutants in the mouse model of infection (structures are shown below). A deletion of the keto-mycolates leads to restricted growth of this mutant in macrophages (124). Thus, the fine structure of mycolic acids is associated with virulence of *M. tuberculosis*.

MYCOLIC ACIDS IN M. TUBERCULOSIS

As defined by Asselineau and Lederer (8) mycolic acids are β-hydroxy fatty acids with a long α-alkyl side chain. They exist as homologous series of fatty acids differing by 28 atomic mass units (a two-carbon unit), and in *M. tuberculosis* they are characterized by very hydrophobic C₅₄ to C₆₃ fatty acids with C₂₂ to C₂₄ α side chains. Three distinct structural classes of mycolic acids are found in *M. tuberculosis*, and they are α-, methoxy- and keto-mycolic acids, as shown in Fig. 1. The α-mycolic acid is the most abundant form (>70%), whereas methoxy- and keto-mycolic acids are the minor components (10 to 15%) (82). The α-mycolic acid is a cis, cis-dicyclopropyl fatty acid. There are two structural variations of this mycolic acid, depending on the source. These variations are in the length of the terminal alkyl group and the number of methylene groups between the

FIG. 1. Chemical structures of mycolic acids from *M. tuberculosis*. There are five forms of mycolic acids in *M. tuberculosis*, illustrated with α-mycolic acid from the H37Ra strain and methoxy- and keto-mycolic acids from *M. tuberculosis* subsp. *hominis* strains DT, PN, and C. Both cyclopropane rings in α-mycolic acid have the cis configuration. The methoxy- and keto-mycolic acids can have either the cis or trans configuration on the proximal cyclopropane ring.

cyclopropane rings and the carboxyl group. Thus, α-mycolic acids from strains H37Ra and Test represent one group, and α-mycolic acids from strains Brevanne, DT, PN, C, and Canetti represent the other group (6, 35, 38, 66, 68, 82, 100). The α-mycolic acid from the H37Ra strain and the clinical strains are different. Both methoxy- and keto-mycolic acids have structural series containing either cis- or trans-cyclopropane rings (67).

In recent years, many excellent reviews on the biosynthesis of mycolic acids have appeared (7, 11, 30, 49, 50). However, these reviews lack the details on how mycolic acids are synthesized and processed into the final products. Considerable knowledge has accumulated in the last 20 years, and today we are sitting on the cusp of this problem. To move this field forward, we have now critically examined and reviewed numerous old and new data in the literature and used them to develop a novel and plausible scheme of how mycolic acids are synthesized and processed in *M. tuberculosis*. We have performed computational genetic analyses to try to fill in the gaps in the current knowledge. The validity of parts of this proposed pathway can now be tested experimentally, and a more complete picture should emerge.

BIOSYNTHESIS OF MYCOLIC ACIDS IN  
*M. TUBERCULOSIS*

Biosynthesis of Normal Fatty Acid Precursors  
of Mycolic Acids by FAS-I

A study of how *Mycobacterium* synthesizes C₁₆ to C₁₈ and C₂₄ to C₂₆ fatty acids in a bimodal product pattern by the malonyl coenzyme A (malonyl-CoA) pathway dates back to the work of Bloch and associates in the early 1970s (16). They showed that *Mycobacterium smegmatis* contains both type I fatty acid synthetase (FAS-I), a multienzyme complex found in eukaryotes and advanced prokaryotes, and type II fatty acid synthetase (FAS-II), a disaggregated synthetase system found in plants and bacteria. The FAS-I and FAS-II systems of *M. tuberculosis* were shown to be similar to those of *M. smegmatis* (G. S. Besra, unpublished results).

The mycobacterial FAS-I system elongates acetyl group by

Vol. 18, 2005

ANABOLIC PATHWAY OF MYCOLIC ACIDS IN M. TUBERCULOSIS 83

I. Acetyl transacylation  
Acetyl-CoA + HS-Enz → Acetyl-S-Enz + CoA  

II. Malonyl transacylation  
Malonyl-CoA + HS-Enz → Malonyl-S-Enz + CoA  

III. Condensation  
Acetyl-S-Enz + Malonyl-S-Enz → β-ketoacyl-C₄-S-Enz + HS-Enz + CO₂  

IV. Sum of β-ketoacyl reduction, dehydration and enoyl reduction  
β-ketoacyl-C₄-S-Enz → C₄-S-Enz  

V. Recycling leads to two key products  
C₂₀-S-Enz (after 9 cycles) and C₂₆-S-Enz (after 12 cycles)

FIG. 2. Reaction sequences I to V in the mycobacterial FAS-I system, leading to the formation of early precursors of mycolic acids. These reactions take place on a single multienzyme complex. HS-Enz, ACP-like protein in the complex; C₄, butyryl; C₂₀, eicosanoyl; C₂₆, hexacosanoyl.

two-carbon units, using acetyl-CoA and malonyl-CoA as substrates to yield butyryl-S-Enz (Fig. 2). Further elongation leads to C₁₆- and C₁₈-S-Enz; these are converted to the CoA derivative and used primarily for the synthesis of membrane phospholipids. Continued elongation of these fatty acids by FAS-I of *M. tuberculosis* produces specifically the C₂₀- and C₂₆-S-Enz products, and these fatty acids are released as the CoA derivatives (Fig. 2). We propose that the C₂₀ fatty acid is the starting point where the FAS-II systems take over for the synthesis of the very-long-chain mero segment of α-, methoxy-, and ketomycolic acids (see below). Meroacid is defined in this review as the long α-alkyl chain of mycolic acid that is released as a meroaldehyde by pyrolytic cleavage and is oxidized by a separate reaction to the fatty acid (8). FAS-I also produces hexacosanoyl-CoA (C₂₆), and this fatty acid becomes the short α-alkyl chain and methyl carboxyl segment of all mycolic acids of *M. tuberculosis*. McCarthy (62) and Weir et al. (114) observed uptake of long-chain fatty acids from the culture medium by mycobacteria and incorporation into triacylglycerol. In *M. tuberculosis* infection, these internalized C₁₆ and C₁₈ fatty acids could enter the FAS-I pathway and be used for the synthesis of mycolic acids.

Biosynthesis of cis,cis-Diunsaturated Meroacid Precursors and α-Meroacid

As shown in Fig. 4, FAS-I provides the eicosanoyl-S-CoA to the FAS-II system for the synthesis of the meroacids. Choi et al. (20) and Scardale et al. (90) have shown that the initial reaction upon entry into the FAS-II system from FAS-I involves a special β-ketoacyl-ACP synthase III (mtFabH) (Fig. 4, reaction 1). How *M. tuberculosis* places double bonds into the meroacid precursors is an unresolved issue. We propose that β-hydroxyacyl-ACP dehydrase and 2-trans-enoyl-ACP isomerase together are responsible for the production of *cis* unsaturation. Thus, we propose reactions 2 to 4 as shown in Fig. 4. Reaction 2 shows that β-hydroxyacyl-ACP reductase catalyzes the formation of β-hydroxy-C₂₂-S-ACP from β-ketoacyl-C₂₂-S-ACP. Then β-hydroxyacyl-ACP dehydrase and 2-trans-enoyl-ACP isomerase converts this product to *cis*-Δ³-C₂₂:₁-S-ACP (reactions 3 and 4). We call the system that catalyzes reactions 1 to 4 the FAS-IIA module. This is new and a key concept that we are using to describe how a *cis* unsaturation is introduced in the synthesis of mycolic acids in *M. tuberculosis*. This dehydrase activity is similar to that of the β-hydroxyacyl-ACP dehydrase.

![Diagram](https://i.imgur.com/yourimage.png)

FIG. 3. Generalized FAS-II elongation module of *M. tuberculosis* for the synthesis of meroacids. The substrates are R-CO-S-ACP and malonyl-S-ACP derived from malonyl-S-CoA by FabD. R, long-chain alkyl group. Enzymes involved in these reactions are as follows: 1, β-ketoacyl-ACP synthase (KasA/KasB); 2, β-ketoacyl-ACP reductase; 3, β-hydroxyacyl-ACP dehydrase; 4, 2-trans-enoyl-ACP reductase (InhA). The product of the last reaction undergoes the next cycle of elongation as the ACP derivative on another FAS-II module. This is a long-chain fatty acid elongation system in which the hydrocarbon chain is increased by two carbons with the completion of each cycle. t, *trans* isomer.

The Basic FAS-II Module for Chain Elongation in the Biosynthesis of Meroacids

The basic FAS-II module shown in Fig. 3 is used for chain elongation in the synthesis of meroacids in *M. tuberculosis*. We suggest that it is one of three different modules of FAS-II (discussed below). The malonyl-S-acyl carrier protein (malonyl-S-ACP) (a cosubstrate in reaction 1 in Fig. 3) is derived from malonyl-S-CoA by the enzyme malonyl-CoA:ACP transacylase (FabD). Kremer et al. studied the properties of this enzyme (51). Through successive reactions by β-ketoacyl-ACP synthase (KasA/KasB) and β-ketoacyl-ACP reductase, R-CO-S-ACP is converted to R-CHOH-CH₂-CO-S-ACP (reactions 1 and 2). This product is then converted to *trans*-R-CH=CH-CO-S-ACP by β-hydroxyacyl-ACP dehydrase (reaction 3). Curiously, the gene encoding this enzyme has yet to be identified in the *M. tuberculosis* genome (30). The product of reaction 3 is reduced by 2-trans-enoyl-ACP reductase (InhA) to yield R-CH₂-CH₂-CO-S-ACP, which is two carbons longer than the

FAS-I

$\begin{aligned}
& \text{-O-C-CH}_{2}-\text{C-S-ACP} \\
& \downarrow \\
& \text{CH}_{3}-\left(\text{CH}_{2}\right)_{17}-\text{CH}_{2}-\text{C-S-CoA} \\
& \downarrow 1 \\
& \text{CH}_{3}-\left(\text{CH}_{2}\right)_{17}-\text{CH}_{2}-\text{C-S-ACP} \\
& \downarrow 2 \\
& \text{CH}_{3}-\left(\text{CH}_{2}\right)_{17}-\text{CH}_{2}-\text{CH}_{2}-\text{C-S-ACP} \\
& \downarrow 3 \\
& \text{dehydrase} \\
& \text{CH}_{3}-\left(\text{CH}_{2}\right)_{17}-\text{CH}_{2}-\text{CH=CH-C-S-ACP} \\
& \downarrow 4 \\
& \text{isomerase} \\
& \text{(t)} \\
& \text{CH}_{3}-\left(\text{CH}_{2}\right)_{17}-\text{CH=CH-CH}_{2}-\text{C-S-ACP} \\
& \downarrow 5 \\
& \text{(c)} \\
& \text{CH}_{3}-\left(\text{CH}_{2}\right)_{17}-\text{CH=CH-}\left(\text{CH}_{2}\right)_{x}-\text{CH}_{2}-\text{C-S-ACP} \\
& \downarrow 6 \\
& \text{-}\left(\text{CH}_{2}\right)_{x}-\text{CH}_{2}-\text{C-S-ACP} \\
& \downarrow 7 \\
& \text{dehydrase} \\
& \text{-}\left(\text{CH}_{2}\right)_{x}-\text{CH}_{2}-\text{CH=CH-C-S-ACP} \\
& \downarrow 8 \\
& \text{isomerase} \\
& \text{(t)} \\
& \text{-}\left(\text{CH}_{2}\right)_{x}-\text{CH=CH-CH}_{2}-\text{C-S-ACP} \\
& \downarrow 9 \\
& \text{(c)} \\
& \text{-}\left(\text{CH}_{2}\right)_{x}-\text{CH=CH-}\left(\text{CH}_{2}\right)_{y}-\text{C-S-ACP} \\
& \downarrow 10 \\
& \text{cp} \\
& \text{CH}_{3}-\left(\text{CH}_{2}\right)_{17}-\text{CH=CH-}\left(\text{CH}_{2}\right)_{x}-\text{CH=CH-}\left(\text{CH}_{2}\right)_{y}-\text{C-S-ACP} \\
& \downarrow 11 \\
& \alpha-\text{meroacyl-ACP}
\end{aligned}$

FIG. 4. Proposed pathway to synthesis of $cis, cis$-diunsaturated meroacid precursors and $\alpha$-meroacid. In this illustration, the initial substrates for the FAS-II system are eicosanoyl-S-CoA (derived from FAS-I) and malonyl-S-ACP (derived from malonyl-S-CoA by FabD). Enzymes involved in these reactions are as follows: 1, $\beta$-ketoacyl-ACP synthase-III; 6, $\beta$-ketoacyl-ACP synthase (Kas); 2 and 7, $\beta$-ketoacyl-ACP reductase; 3 and 8, $\beta$-hydroxyacyl-ACP dehydrase; 4 and 9, $2$-trans-enoyl-ACP isomerase; 5 and 10, elongation (e) by multiple FAS-II modules; 11, cyclopropane synthase (cp). The FAS-IIA and FAS-IIB modules are identified in the pathway. t and c, trans and cis isomers, respectively. For the synthesis of $\alpha$-meroacid, $x = 10$ and $y = 17$; for the synthesis of methoxy-meroacid, $x = 16$ and $y = 17$; for the synthesis of keto-meroacid, $x = 16$ and $y = 19$.

(FabZ) found in *Escherichia coli* and *Streptococcus pneumoniae* (71, 85, 125). This enzyme is involved in converting $\beta$-hydroxy-acyl-S-ACP to $2$-trans-enoyl-S-ACP. It should be noted that Barry et al. (11) considered the possibility of the existence of dehydrase but dismissed it because of a lack of any enzyme in the genome of *M. tuberculosis* with significant homology to FabA. We have searched the genome of *M. tuberculosis* for both dehydrase and isomerase and have found a few candidate enzymes that might be related to S. pneumoniae FabZ and FabM (see below).

The $cis$-$\Delta^{3}$-$C_{22:1}$-S-ACP is carried through elongation by the FAS-II modules (Fig. 3 and 4) at one cycle per FAS-II unit (five cycles for $\alpha$-meroacid and eight cycles for methoxy- and keto-meroacid). For $\alpha$-meroacid, the product is $cis$-$\Delta^{13}$-$C_{32:1}$-S-ACP (Fig. 4, reaction 5, where $x = 10$). For methoxy- and keto-meroacid, the product is $cis$-$\Delta^{19}$-$C_{38:1}$-S-ACP ($x = 16$). KasA ($\beta$-ketoacyl-ACP synthase A) is probably involved in these early elongation steps. This pathway is supported by a study by Takayama et al. (102). They isolated and characterized these anabolic products as well as their expected sequen-

Vol. 18, 2005

ANABOLIC PATHWAY OF MYCOLIC ACIDS IN *M.* TUBERCULOSIS 85

tial intermediates of elongation of Δ⁵-C₂₄:₁, Δ⁷-C₂₆:₁, Δ⁹-C₂₈:₁, and Δ¹¹-C₃₀:₁ fatty acids in log-phase cells of *M.* tuberculosis H37Ra. Although it has not been determined, the double bonds are expected to have *cis* configuration.

For α-meroacid, the *cis*-Δ¹³-C₃₂:₁-S-ACP is carried through reactions 6 and 7 to yield β-hydroxy *cis*-Δ¹⁵-C₃₄:₁-S-ACP (Fig. 4). Then β-hydroxyacyl-ACP dehydrase and 2-*trans*-enoyl-ACP isomerase convert this product to *cis*,*cis*-Δ³,¹⁵-C₃₄:₂-S-ACP (reactions 8 and 9). We call the system that catalyzes reactions 6 to 9 the FAS-IIB module. Further elongation of this product by the FAS-II modules (Fig. 3) yields a *cis*,*cis*-Δ¹⁹,³¹-C₅₀:₂-S-ACP (reaction 10). For methoxy-meroacid, the product is *cis*, *cis*-Δ¹⁹,³⁷-C₅₆:₂-S-ACP (where \( x = 16 \) and \( y = 17 \)), and for keto-meroacid, the product is *cis*,*cis*-Δ²¹,³⁹-C₅₈:₂-S-ACP (where \( x = 16 \) and \( y = 19 \)). KasB (β-ketoacyl-ACP synthase B) is probably involved in the latter elongation steps. Two distinct cyclopropane synthases, MmaA2 (40) and PcaA (41), then introduce cyclopropane rings into the distal and the proximal unsaturations, respectively, in *cis*,*cis*-Δ¹⁹,³¹-C₅₀:₂-S-ACP to yield the mature C₅₂-α-meroacid as the ACP derivative (reaction 11, where \( x = 10 \) and \( y = 17 \)). Methyl transfer reactions at this point in the pathway are supported by a study by Qureshi et al. (83). Using a cell extract of *M.* tuberculosis H37Ra, they showed that the [¹⁴C]methyl group of S-adenosyl-L-methionine (SAM) goes directly into the full-size meroacids. Later, Yuan et al. confirmed this by showing that the cyclopropane residues are introduced into the meroacid-S-ACP prior to the final condensation step (121). Watanabe et al. established a clear structural relationship between related isolated type 1 (Fig. 4, product of reaction 11) and type 3 (Fig. 4, product of reaction 10) meroacids in *M.* tuberculosis (113).

Takayama and associates isolated and characterized saturated C₃₅ to C₅₆ fatty acids (104), mono- and diunsaturated C₂₂ to C₄₇ fatty acids (103), *cis*,*cis*-3,4- and 15,16-dimethylenetetracontanoic acid (80), and C₄₉ to C₅₈ fatty alcohols (81) from *M.* tuberculosis H37Ra. These fatty acids were considered to be precursors of mycolic acids. Many of these fatty acids that are smaller than meroacids and contain cyclopropane rings, as well as the fatty alcohols, are now considered to be products of β-oxidation of mycolic acids (from the catabolic pathway). The expected presence of a homologous series of diunsaturated fatty acids >C₄₇ in this proposed pathway (homologous series of products of reaction 10) has not been investigated.

The starting point for the biosynthesis of meroacids by the FAS-II system appears to be the C₂₀-S-CoA that is derived from FAS-I. Studies by Takayama and associates support this conclusion (27, 101). They showed that isoniazid specifically inhibits the synthesis of monounsaturated tetracosanoic acid (C₂₄:₁) and higher homologs in *M.* tuberculosis H37Ra. They also showed that the hexacosanoic acid accumulates in the tubercle bacillus treated with the drug. Jacobs and associates and others showed that the target of inhibition of mycolic acid synthesis by isoniazid is 2-*trans*-enoyl-ACP reductase (InhA) of the FAS-II elongation pathway (10, 59, 111). As the above-described two pathways show (Fig. 3 and 4), inhibition of InhA would inactivate the entire FAS-II pathways, except for the first four steps at the very beginning of the first cycle, starting with β-ketoacyl-ACP synthetase III, then β-ketoacyl-ACP reductase, β-hydroxy-ACP dehydrase (Fig. 4, reactions 1 to 3), and finally the special 2-*trans*-enoyl-ACP isomerase activities.

TAKAYAMA ET AL.

$\begin{aligned}
& \text { 2 (y=17) } \\
& \text { MmaA3/MmaA2 } \\
& \text { CH}_{3}-(\mathrm{CH}_{2})_{17}-\mathrm{CH}-\mathrm{CH}-\left(\mathrm{CH}_{2}\right)_{16}-\mathrm{CH}-\mathrm{CH}-\left(\mathrm{CH}_{2}\right)_{17}-\mathrm{C}-\mathrm{S}-\mathrm{ACP} \\
& \text { cis-methoxy-meroacyl-ACP }
\end{aligned}$

$\begin{aligned}
& \text { 3 (y-1=16) } \\
& \text { MmaA3/CmaA2 } \\
& \text { CH}_{3}-(\mathrm{CH}_{2})_{17}-\mathrm{CH}-\mathrm{CH}-\left(\mathrm{CH}_{2}\right)_{16}-\mathrm{CH}-\mathrm{CH}-\left(\mathrm{CH}_{2}\right)_{16}-\mathrm{C}-\mathrm{S}-\mathrm{ACP} \\
& \text { trans-methoxy-meroacyl-ACP }
\end{aligned}$

$\begin{aligned}
& \text { 2 (y=19) } \\
& \text { Redox/MmaA2 } \\
& \text { CH}_{3}-(\mathrm{CH}_{2})_{17}-\mathrm{CH}-\mathrm{C}-\left(\mathrm{CH}_{2}\right)_{16}-\mathrm{CH}-\mathrm{CH}-\left(\mathrm{CH}_{2}\right)_{19}-\mathrm{C}-\mathrm{S}-\mathrm{ACP} \\
& \text { cis-keto-meroacyl-ACP }
\end{aligned}$

$\begin{aligned}
& \text { 3 (y-1=18) } \\
& \text { Redox/CmaA2 } \\
& \text { CH}_{3}-(\mathrm{CH}_{2})_{17}-\mathrm{CH}-\mathrm{C}-\left(\mathrm{CH}_{2}\right)_{16}-\mathrm{CH}-\mathrm{CH}-\left(\mathrm{CH}_{2}\right)_{18}-\mathrm{C}-\mathrm{S}-\mathrm{ACP} \\
& \text { trans-keto-meroacyl-ACP }
\end{aligned}$

FIG. 6. Methyltransferases and oxidation-reduction (redox) to form the methoxy group of methoxy-meroacids and the oxo group of keto-meroacids on the pathway to synthesis of oxygenated meroacids. MmaA2 introduces the methyl group on the secondary alcohol, MmaA3 introduces the proximal *cis*-cyclopropane ring, and CmaA2 introduces the proximal *trans*-cyclopropane ring. Redox is the proposed oxidation-reduction system that converts the secondary alcohol to an oxo group. t and c, *trans* and *cis* isomers, respectively.

involves Claisen-type condensation and is shown in Fig. 7. A high degree of control is placed over the entire condensation reactions. A very specific FadD32 (fatty acyl-AMP ligase) converts each meroacyl-S-ACP derived from the FAS-II system (*α*-meroacyl-S-ACP from Fig. 4 and methoxy- and keto-meroacyl-S-ACPs from Fig. 6) to meroacyl-AMP. Trivedi et al. recently discovered this novel reaction and its significance (107). The $C_{26}$-S-CoA derived from FAS-I (after release from the enzyme as the CoA derivative) is carboxylated by AccD4 and AccD5 (acyl-CoA carboxylases) to yield the 2-carboxyl-$C_{26}$-S-CoA. The functions of AccD4 and AccD5 are discussed in another section. These two enzymes must be specific for the $C_{24}$ and $C_{26}$ fatty acyl groups. The two products are substrates for the Claisen-type condensation by Pks13.

The $C_{52}$-*α*-meroacyl-AMP is used as one of the five substrates in this illustration (Fig. 7). This scheme is an adaptation of the activation and transfer of fatty acids on cognate Pks

$\alpha$-Meroacyl-S-ACP + ATP $\stackrel{\text { FadD32 }}{\longrightarrow} \alpha$-Meroacyl-AMP

$C_{26}$-S-CoA + CO$_{2}$ $\stackrel{\text { AccD4, AccD5 }}{\longrightarrow}$ 2-Carboxyl-$C_{26}$-S-CoA

Pks13

| PPB | KS | AT | PPB | TE |
| --- | --- | --- | --- | --- |
| SH | SH |  | SH |  |

$\alpha$-Meroacyl-AMP

2-Carboxyl-$C_{26}$-S-CoA

$\begin{array}{llllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllllll
TABLE 1. Assembly line array models of mycolate synthetases in *M. tuberculosis*

| Enzyme | Assembly line array model$^a$ |
| --- | --- |
| α-Mycolate synthetase | FAS-I → FAS-IIA → [FAS-II]$_5$ → FAS-IIB → [FAS-II]$_8$ → MeTnf$^b$ → [Pks13][FAS-I] |
| cis-Methoxy-mycolate synthetase | FAS-I → FAS-IIA → [FAS-II]$_8$ → FAS-IIB → [FAS-II]$_8$ → MmaA4/MeTnf → [Pks13][FAS-I] |
| trans-Methoxy-mycolate synthetase | FAS-I → FAS-IIA → [FAS-II]$_8$ → FAS-IIB → [FAS-II]$_8$ → MmaA4/MmaA1/MeTnf → [Pks13][FAS-I] |
| cis-Keto-mycolate synthetase | FAS-I → FAS-IIA → [FAS-II]$_8$ → FAS-IIB → [FAS-II]$_9$ → MmaA4/Redox/MeTnf → [Pks13][FAS-I] |
| trans-Keto-mycolate synthetase | FAS-I → FAS-IIA → [FAS-II]$_8$ → FAS-IIB → [FAS-II]$_9$ → MmaA4/MmaA1/Redox/MeTnf → [Pks13][FAS-I] |

$a$ The three different FAS-II modules are FAS-II (see Fig. 3 for composition) for elongation, FAS-IIA (containing FabD, β-hydroxyacyl-ACP synthase-III, β-ketoacyl-ACP reductase, β-hydroxyacyl-ACP dehydrase, and 2-trans-enoyl-ACP isomerase) for introduction of distal $cis$ unsaturation, and FAS-IIB (containing FabD, β-ketoacyl-ACP synthase, β-ketoacyl-ACP reductase, β-hydroxyacyl-ACP dehydrase, and 2-trans-enoyl-ACP isomerase) for introduction of proximal $cis$ unsaturation. These reactions and enzymes are shown in Fig. 4. MmaA4 introduces the distal-branch methyl and adjacent hydroxyl groups. MmaA1 introduces the proximal allylic methyl branch and $trans$ unsaturation. Redox converts the distal secondary alcohol to the oxo group. FAS-I provides eicosanoyl-S-CoA at the beginning with FAS-IIA and hexacosanoyl-S-CoA for the final condensation reaction with Pks13.

$b$ MeTnf, methyltransferases (including cyclopropane synthase).

protein described by Trivedi et al. (107). The first step (reaction 1) is the binding of a C$_{52}$-α-meroceryl group from its AMP derivative and a 2-carboxyl-C$_{26}$-acyl group from its CoA derivative to the N-terminal phosphopantetheine-binding (PPB) domain and the near-C-terminal PPB domain of Pks13, respectively, both as thioesters. In the former reaction, the acyltransferase (AT) enzyme is presumed to be specific for the AMP derivative. Reaction 2 involves the transfer of an α-meroceryl group from the N-terminal PPB domain to the ketoacyl synthase (KS) domain, presumably catalyzed by the acyltransferase at the AT domain. These two acyltransferase reactions are suggested by Trivedi et al. (107) and are based on a study by Admiraal et al. (2). Reaction 3 is the Claisen-type condensation of the two fatty acyl groups. A nucleophilic attack of the carbonyl group of the α-meroceryl-S-KS by the acidic α-carbon of the 2-carboxyl-C$_{26}$-S-PPB results in the formation of 3-oxo-C$_{78}$-α-mycolate bound to the near C-terminal PPB domain and the release of CO$_2$ and KS. Reaction 4 is the reduction step, where an unidentified acyl reductase converts the 3-oxo to the secondary alcohol to yield the mature C$_{78}$-α-mycolate (reactions 3 and 4 were combined in this illustration). The role of the thioesterase (TE) at the C-terminal TE domain is not to release the mycolic acid from the Pks13 complex, since such a large free fatty acid would become intractable. It is suggested that the TE domain of Pks13 contains a type II thioesterase that “edits” and corrects improper acylation at the near-C-terminal PPB domain (48, 57, 92). The four oxygenated mycolic acids are synthesized by the same mechanism as the α-mycolic acid.

Walker et al. (112) first described the mechanism for the Claisen-type condensation in the synthesis of corynomycolic acid in *Corynebacterium diphtheriae*, based on a study by Gastambide-Odier and Lederer (37). Later, Takayama and Qureshi (100) used it to describe this reaction in *M. tuberculosis*. The present scheme is a highly updated version of the old scheme in the latter reference, using the latest data.

### Physical Organization of Mycolate Synthetases in *M. tuberculosis*

Upon examining Fig. 3 to 7 together, one might get the impression that the pathways to synthesis of all mycolic acids in *M. tuberculosis* are physically interconnected to give one large array network. Although this is possible, it is probably not the case. We suggest that the mycolate synthetase complexes of *M.* tuberculosis are organized as assembly line arrays of FAS-I and

Inside the cell:
$$
\text{Mycolyl-S-PPB + Man-P-heptaprenol} \xrightarrow{1} \text{Myc-PL + HS-ACP}
$$
$$
\text{Myc-PL + Treh 6-P} \xrightarrow{2} \text{6-0-mycolyl-treh-6'-P + Man-P-heptaprenol}
$$
$$
\text{6-0-mycolyl-treh-6'-P} \xrightarrow{3} \text{TMM + Pi}
$$

Transfer of TMM outside the cell:
$$
\text{TMM (inside) + ATP} \xrightarrow{4} \text{TMM (outside) + ADP + Pi}
$$

Outside the cell:
$$
\text{TMM + TMM} \xrightarrow{5} \text{TDM + Treh}
$$
$$
\text{TMM + Cell wall-AG} \xrightarrow{6} \text{Cell wall-AG-M + Treh}
$$

FIG. 8. Processing of newly synthesized mycolic acids in *M. tuberculosis*. Inside the cell, newly synthesized mycolic acid in thioester linkage to the near-C-terminal PPB domain of Pks13 is transferred first to man-P-heptaprenol to yield Myc-PL and then to trehalose 6-phosphate to yield TMM-P. Dephosphorylation of this product yields TMM. TMM is then transported outside the cell (outer membrane) by a proposed TMM transporter (ABC transporter cassette), where it is involved in the synthesis of both TDM and cell wall arabinogalactan-mycolate. Reactions: 1, mycolyltransferase I; 2, mycolyltransferase II; 3, TMM-P phosphatase; 4, TMM transporter; 5 and 6, Ag85 as the mycolyltransferase (FbpA, FbpB, and FbpC). man-P-heptaprenol; treh, trehalose; AG, arabinogalactan; AG-M, arabinogalactan-mycolate.

an active cell-free system from *M. tuberculosis* for the synthesis of mycolic acids, as numerous investigators have found.

# PROCESSING OF NEWLY SYNTHESIZED MYCOLIC ACIDS

The final stages in the synthesis of the cell wall of *M. tuberculosis* are transport and attachment of newly synthesized mycolic acids to the peptidoglycan-arabinogalactan complex of the cell wall and the formation of TDM. We describe how this might occur through a six-step process as shown in Fig. 8. This part of the pathway for the most part is unknown. Only the function of antigen 85/fibronectin-binding protein as mycolyltransferase (Ag85/Fbp) is well established (reactions 5 and 6).

The final product of synthesis of mycolic acid in the cytoplasm is thought to be mycolyl-S-Pks13 (Fig. 7). The mycolic acid in this product must somehow be transferred to trehalose by an as-yet-unidentified mycolyltransferase(s) to yield trehalose monomycolate (TMM). Ag85 does not appear to have this mycolyltransferase activity. In an unpublished study by L. Y. Armitige, C. A. Rivera-Marrero, and K. Takayama, the individual inactivations of the *fbpA* and *fbpB* genes in *M. tuberculosis* H37Rv by homologous recombination (4) were shown to cause decreases in the synthesis of TDM and clear accumulation of TMM when compared to the wild type. These findings were from short-term pulse-labeling experiments using sodium [1-¹⁴C]acetate in submerged cultures. Since Δ*fbpA* and Δ*fbpB* mutants were clearly able to synthesize TMM, the Ag85 complex must not be involved in the early mycolyl transfer reactions.

We suggest a completely new pathway to the synthesis of TMM. We first attribute a function for the previously discovered 6-*O*-mycolyl-β-*d*-mannopyranosyl-1-phosphoheptaprenol (Myc-PL) (14, 26) as the initial carrier form of the mycolyl group. The mycolyl group is first transferred from mycolyl-S-Pks13 (mycolyl-S-PPB) to *d*-mannopyranosyl-1-phosphoheptaprenol by a proposed cytoplasmic mycolyltransferase I to yield Myc-PL (Fig. 8, reaction 1). Myc-PL then migrates to the inner surface of the cell membrane and, with its hydrophobic heptaprenol tail, docks next to an ABC transporter. The subsequent three reactions are localized at the transporter. In a tightly coupled system, the mycolyl group is then transferred to trehalose 6-phosphate by the proposed membrane-associated mycolyltransferase II (reaction 2) to form TMM-phosphate, the phosphate group is removed by the membrane-associated trehalose 6-phosphate phosphatase (reaction 3), and the resulting product (TMM) is immediately transported outside the cell by the ABC transporter (reaction 4). There should be virtually no accumulation of TMM in the cytoplasm. This proposed mechanism would allow a rapid and efficient transfer of TMM from the inside to the outside of the cell for the synthesis of cell wall arabinogalactan-mycolate and TDM. This would be a requirement for normal growth of *M. tuberculosis*. It prevents the conversion of TMM to TDM inside the cell by the ubiquitously present Ag85/Fbp (47, 89). Such a reaction is to be avoided, since it would effectively consume most of the TMM synthesized internally and make it unavailable for transport outside the cell. The advantage of forming a TMM-phosphate intermediate rather than TMM directly from trehalose is not known. However, the role of trehalose 6-phosphate as the acceptor of mycolic acids in the synthesis of TMM is supported by a study by Shimakata and Minatogawa (93).

The three pathways to synthesis of trehalose in *Mycobacterium* species and *Corynebacterium glutamicum* are OtsA/OtsB, TreY/TreZ, and TreS (108). Wolf et al. (117) showed that the Δ*otsA* Δ*trey* double deletion in *C. glutamicum* completely eliminated the synthesis of TMM and TDM. Single deletions of these two genes caused about a 40% reduction (from the wild-type level) in the synthesis of TMM and TDM (our interpretation of their results). The conclusion here is that both the OtsA/OtsB and TreY/TreZ pathways are involved in the synthesis of TMM and TDM. The OtsA/OtsB pathway generates trehalose 6-phosphate, whereas the TreY/TreZ pathway generates free trehalose. Thus, in the latter case, we can assume that the product would have to be phosphorylated before mycolyl transfer can occur. The Δ*otsA* Δ*tres* Δ*trey* triple deletion mutant of *M. smegmatis* was not able to grow unless supplemented with trehalose (119). This shows that mycobacteria have an absolute requirement for trehalose, presumably to process the mycolyl groups.

We suggest that TMM is transported outside the cell (into the outer membrane) by the ABC transporter (reaction 4). This is a new concept and has experimental support. Inhibitors of the ABC transporter (reserpine and verapamil) were shown to inhibit the incorporation of radiolabel from sodium [1-¹⁴C] acetate into cell wall arabinogalactan-corynomycolate in *Corynebacterium matruchotii* (C. Wang and K. Takayama, unpublished results).

By the action of the extracellular mycolyltransferase called Ag85/Fbp/PS1 (PS1 is the mycolyltransferase in *C. glutamicum*) (13, 45, 77, 78), the final products of cell wall arabinogalactan-mycolate and TDM are formed from TMM (reactions 5 and 6). The former product formation is based on genetic evidence. Since *M. tuberculosis* is known to excrete copious amounts of Ag85 into the culture medium (1), it is readily available for these two reactions outside the cell.

## GENETIC ANALYSIS OF SYNTHESIS AND PROCESSING OF MYCOLIC ACID IN M. TUBERCULOSIS

The genes and enzymes believed to be involved in the biosynthesis and processing of mycolic acids in *M. tuberculosis* are listed in Table 2. Most of the genes and their functions have been identified. These include the single gene for FAS-I, most of the genes for the FAS-II system, and all of the genes for the methyltransferases and cyclopropane synthases. There are two enzymes in the proposed pathway to synthesis of merocarboxylic acids by the FAS-II system whose corresponding genes have yet to be identified or studied in detail. These are the β-hydroxyacyl-ACP dehydrase and the 2-trans-enoyl-ACP isomerase (presumed to be present in the FAS-IIA and FAS-IIB modules). Almost all of the enzymes in the mycolic acid-processing pathway need to be identified and studied. Computational genetic analyses were performed to identify candidate genes encoding these unidentified proteins.

### FAS-I

A single gene, *fab* (*Rv2524c*), encodes the multifunctional FAS-I system of *M. tuberculosis*. The active gene expresses a

TABLE 2. Genetic analysis of the anabolic pathway of mycolic acids in *M. tuberculosis* and growth requirements

| System or group                | Gene          | Sanger identification | Enzyme                              | Growth attenuation$^c$ | Reference(s) |
|---------------------------------|---------------|-----------------------|--------------------------------------|------------------------|--------------|
|                                 |               |                       |                                      | In vitro               | In vivo      |
| FAS-I                          | fas           | Rv2524c               | Fatty acid synthetase-I              | Yes                    | —$^d$        | 23           |
| FAS-I → FAS-II                 | fabD          | Rv2243                | Malonyl-CoA:ACP transacylase         | —                      | No           | 51           |
|                                 | acpM          | Rv2244                | Acyl carrier protein                  | Yes                    | No           | 51           |
|                                 | fabH          | Rv0533c               | β-Ketoacyl-ACP synthase III          | No                     | No           | 20, 90       |
| FAS-II                        | kasA/kasB     | Rv2245/Rv2246         | β-Ketoacyl-ACP synthase              | Yes/Yes                | —/—          | 91           |
|                                 | fabG1         | Rv1483                | β-Ketoacyl-ACP reductase             | No                     | No           | 61           |
|                                 |               | Rv0098                | β-Hydroxyacyl-ACP dehydrase$^a$      | No                     | Yes          | 25, 71       |
|                                 | echA10/echA11 | Rv1142c/Rv1141c       | 2-trans-Enoyl-ACP isomerase$^a$      | No/No                  | No/No        | 60           |
|                                 | inhA          | Rv1484                | 2-trans-Enoyl-ACP reductase         | No                     | No           | 59           |
| Methyltransferases             | mmaA1         | Rv0645c               | MmaA1                               | No                     | No           | 122, 123     |
|                                 | mmaA2         | Rv0644c               | MmaA2                               | No                     | No           | 41           |
|                                 | mmaA3         | Rv0643c               | MmaA3                               | No                     | Yes          | 31, 123      |
|                                 | mmaA4         | Rv0642c               | MmaA4                               | No                     | —            | 31, 123      |
|                                 | cmaA2         | Rv0503c               | CmaA2                               | No                     | No           | 39, 42       |
|                                 | pcaA          | Rv0470c               | PcaA                                | No                     | Yes          | 41           |
| Oxidation-reduction            |               | Rv0161                | Alcohol dehydrogenase$^b$            | —                      | No           |              |
|                                 |               | Rv0162c               |                                      | No                     | No           |              |
|                                 |               | Rv3057c               |                                      | No                     | No           |              |
| Claisen-type condensation      | accD4         | Rv3799c               | Acyl-CoA carboxylase                 | Yes                    | —            | 23           |
|                                 | accD5         | Rv3280                | Acyl-CoA carboxylase                 | —                      | —            |              |
|                                 | fadD32        | Rv3801c               | Fatty acyl-AMP ligase                | Yes                    | —            | 107          |
|                                 | pks13         | Rv3800c               | Polyketide synthase-13               | Yes                    | —            | 76           |
| Mycolic acid processing         |               | Rv3802c               | Mycolyltransferases I$^b$            | Yes                    | —            |              |
|                                 |               | Rv1288/Rv0519c/Rv0774c | Mycolyltransferases II$^b$           | No/No                  | No/No        |              |
|                                 |               | Rv0774c               |                                      | No                     | No           |              |
|                                 |               | Rv3400                | TMM-6-P phosphatase$^b$              | No                     | Yes          |              |
|                                 |               | Rv2006                |                                      | No                     | No           |              |
|                                 |               | Rv1273c/Rv1272c       | ABC transporter$^b$                  | No/No                  | No/Yes       | 17           |
|                                 |               | Rv1348/Rv1349         |                                      | Yes/Yes                | —/—          |              |
|                                 |               | Rv0194                |                                      | No                     | No           |              |
|                                 |               | Rv1819c               |                                      | No                     | No           |              |
|                                 |               | Rv1747                |                                      | No                     | No           |              |
|                                 |               | Rv1687c/Rv1686c       |                                      | No/No                  | No/No        |              |
|                                 | fbpA/fbpB/fbpC | Rv3804c/Rv1886c/Rv0129c | FbpA/FbpB/FbpC                      | No/No                  | —/No         | 4, 45, 77    |
|                                 |               |                       |                                      | No                     | No           |              |

$a$ Dehydrase and isomerase are thought to be in the pathway to meroacid synthesis, but they have not been identified. We suggest possible genes that encode the two enzymes.

$b$ The enzyme is hypothetical. We have identified possible gene and gene products involved in these proposed reactions (Fig. 8).

$c$ The in vitro growth rate was determined by microarray analysis of an *M. tuberculosis* strain H37Rv transposon mutant library grown on agar plates (87). The in vivo growth rate was determined by microarray analysis of surviving bacteria in C57BL/6J mice infected with an *M. tuberculosis* strain H37Rv transposon mutant library (88).

$d$ —, gene was not detected reproducibly.

the Asp<sup>40</sup>Ser<sup>41</sup>Leu<sup>42</sup> motif is conserved in AcpM. The 4′-phosphopantetheine prosthetic group is covalently linked to the serine residue in this motif, and the growing acyl chain is esterified to the terminal sulfhydryl group of 4′-phosphopantetheine. It has been proposed that the unique AcpM carboxyl-terminal domain may interact with the very-long-chain fatty acid intermediates in mycolic acid biosynthesis (118).

mtFabD (malonyl-CoA:ACP transacylase). FAS-II requires the conversion of malonyl-CoA to malonyl-S-ACP as a substrate for further elongation of fatty acids. This reaction is catalyzed by mtFabD. Analysis of the *M. tuberculosis* genome revealed that *acpM* is genetically linked to *fabD*, *kasA*, and *kasB*. The latter two encode the β-ketoacyl synthases (KasA and KasB, respectively). These FAS-II components belong to the same transcription unit. Purified recombinant FabD displayed strong malonyl-CoA:ACP transacylase activity and a preference for holo-AcpM as the substrate in vitro (51). When complemented with the *M. tuberculosis fabD* expression unit, a temperature-sensitive *E. coli* mutant deficient in malonyl-CoA:ACP transacylase activity was able to grow at the nonpermissive temperature (51). mtFabD shared 81.2% sequence identity with the *M. leprae* homolog. The active-site G<sup>89</sup>X<sup>90</sup>S<sup>91</sup>X<sup>92</sup>G<sup>93</sup> motif, characteristic of serine-dependent acylhydrolases (18), is conserved in FabD and its homolog in *M. leprae*. The active-site serines of these enzymes are involved in the nucleophilic attack and the formation of the acyl-enzyme intermediate. Analysis of the *M. tuberculosis* genome revealed that FabD shares 28% sequence identity with FabD2. However, the characteristic active-site motif GXSXG is absent in FabD2. It has been proposed that FabD2 may use an alternative nucleophilic group (51). A gene homologous to *fabD2* was not found in the *M. leprae* genome. Since *M. leprae* is regarded as a minimal genome (22), the lack of *fabD2* in *M. leprae* indicates that FabD is the essential enzyme possessing malonyl-CoA:ACP transacylase activity in *M. tuberculosis*. Whether the *fabD2* gene product catalyzes the malonyl-CoA:ACP transacylase reaction awaits further investigation.

mtFabH (β-ketoacyl-ACP synthase-III). The protein encoded by *fabH* (FabH) forms a pivotal link between the last cycle of FAS-I and the first cycle of the FAS-II system (involving the FAS-IIA module) in *M. tuberculosis*. mtFabH initiates chain elongation of C<sub>20</sub>-S-CoA released from FAS-I as the substrate to generate a β-ketoacyl-S-AcpM product. The function of mtFabH is supported by conservation of active-site residues (Cys<sup>112</sup>, His<sup>244</sup>, and Asn<sup>274</sup>) characteristic of FabH-condensing enzymes and its ability to catalyze the condensation of acyl-CoA with malonyl-ACP and produce β-ketoacyl-ACP (20). Purified recombinant FabH exhibited a preference for acyl-CoA as a substrate. Compared with the general FabB/FabH type of β-ketoacyl synthases, the *M. tuberculosis* FabH exhibited high substrate specificity. Purified FabH preferentially condenses lauroyl-CoA (C<sub>12</sub>) and myristoyl-CoA (C<sub>14</sub>) to generate myristoyl-AcpM and palmitoyl-AcpM (C<sub>16</sub>), respectively, using malonyl-AcpM as the cosubstrate (90). Chain elongation of C<sub>22</sub> to C<sub>24</sub> fatty acids by FabH does not occur in these in vitro studies. The crystal structure of *M. tuberculosis* FabH revealed several unique features, including a CoA/malonyl-ACP binding channel and a long acyl chain-binding channel (90). The *M. tuberculosis* FabH shares 37% sequence identity

to *E. coli* FabH, and surprisingly, its homolog was not found in the *M. leprae* genome.

### FAS-II System

KasA and KasB (β-ketoacyl-ACP synthases). KasA and KasB initiate the subsequent rounds of acyl extension by the FAS-II module (Fig. 3) (64). There are two genes, *kasA* (*Rv2245*) and *kasB* (*Rv2246*), that encode two separate enzymes, β-ketoacyl-ACP synthase A and β-ketoacyl-ACP synthase B. Both KasA and KasB have been expressed in *E. coli*, and the purified enzymes have been shown to extend acyl-ACP thioesters by condensation with malonyl-ACP (52, 91). Both enzymes can elongate long-chain fatty acids of >C<sub>14</sub> (52, 91). Although there is some redundancy, it has been proposed that KasA catalyzes the initial elongation reactions and KasB extends the elongation to full length. Cell lysates of *M. smegmatis* overexpressing KasA generated C<sub>40</sub> monounsaturated fatty acids, whereas cell lysates of *M. smegmatis* overexpressing both KasA and KasB generated C<sub>54</sub> multiunsaturated fatty acids (94). This is consistent with the pathway described in Fig. 4. When KasB was deleted by transposon mutagenesis in *Mycobacterium marinum*, the mutant produced mycolic acids that were two to four carbons shorter than the wild type, with a significant reduction in keto-mycolate (36). These results indicate that KasA and KasB function independently on separate sets of substrates and that KasB is required for the full extension of the mycolate. One might suggest that KasA is involved in elongation reaction 5 whereas KasB is involved in elongation reaction 10 on the pathway to synthesis of cis,cis-diunsaturated merocid precursors and α-merocid (Fig. 4).

KasA and KasB share 66% sequence identity. KasA and KasB share 92 and 91% sequence identity with their respective homologs in *M. leprae*. KasA and KasB share 28 and 29% sequence identity with the KS domain of Pks13, respectively (Fig. 7).

MabA/FabG1 (β-ketoacyl-ACP reductase). MabA carries out the reduction of β-ketoacyl-ACP to β-hydroxyacyl-ACP in all FAS-II modules. Marrakchi et al. purified recombinant MabA and performed an in vitro assay (61). They found that this enzyme catalyzes NADPH-specific reduction of long-chain β-ketoacyl-ACP (C<sub>8</sub> to C<sub>20</sub>). The crystal structure of tetrameric MabA revealed a large substrate-binding pocket, which could adequately accommodate long acyl chains. MabA shares conserved fold of the short-chain dehydrogenases/reductases (21).

In the *M. tuberculosis* genome, *mabA* (*Rv1483*) is located adjacent to *inhA* and presumably is transcribed in the same operon. MabA shares 69% sequence identity with its homolog in *M. leprae*. Numerous homologs of MabA were identified by BLAST searches of *M. tuberculosis* genome (61). These included FabG2, FabG3, FabG4, FabG5, and a number of hypothetical proteins. Determination of whether these proteins are functional in the FAS-II pathway awaits further investigation.

β-Hydroxyacyl-ACP dehydrase. β-Hydroxyacyl-ACP dehydrase is expected to be present in all FAS-II modules. In *E. coli* both FabA and FabZ participate in the conversion of β-hydroxyacyl-ACP to trans-2-enoyl-ACP (25, 71). FabA is a bifunctional β-hydroxyacyl-ACP dehydrase/trans-2-enoyl-ACP isomerase, and it diverts the flow of acyl-ACP into the unsat-

Rv0098
S. pneumoniae FabZ
E. coli FabZ
E. coli FabA

MSHTDLTPCTRVLA SSGTVPIAEELLARVLEPYSCKGCRYLIDAQ 45
MIDIQGIKEALPHRYPMILLVDRV 23
MFAKLYGLSHSYDCGRP CYYCRFLYFDRKSILTTNTHTLQIEEILLELLPHRFPFLLLVDRV 60
YPACFNKIRLLENMVDKRESYTKEDLLASGRGELFGAKGPQLPAPNMLMMDRVVKMTE TG 83

Rv0098
S. pneumoniae FabZ
E. coli FabZ
E. coli FabA

YSATEDDSVL-AYGNFTIGESAYIRSTGHFNAVELILCFNQLAYSAFAPAVLINEIRVLRG 104
LEVSEDTIV-AIKNVTINEPFFN---GHFP---------QYPVMPGVVIMEALAQTAG 68
LDFEEGRFLRAVKNSVNEPFFQ---GHFP---------GKPIFPGVLILEAMAQATG 106
GNFDKGYVE-AELDINPDLWFFG---CHFI---------GDPVMPGCLGLDAMWQLVG 128

Rv0098
S. pneumoniae FabZ
E. coli FabZ
E. coli FabA

WSIDDYCQHQLSSMLIRKASSRFRKP LNPQKFSARLLCRDLQVIERTWRYLK-VPCVIEF 163
VLELSKPENK--GKLVFYAG-------MDKVFKKQVPGDQLVMTATFVKRRGTIAVVEA 120
ILAFKSVGKLEPGEIYYFAG-------IDEARFKRPVPGDQMIMEVTFEKTRRGLTRFKG 160
FYLGWLGGE GKGRALGVGEV-------KFTGQVLPTAKKVTYRIHFKRIVNRRRLIMGLADG 182

Rv0098
S. pneumoniae FabZ
E. coli FabZ
E. coli FabA

WDENGG--AASGEIELAALNIP 183
KAEVDGKLAASGTLTFAIGN-- 140
VALVDGKVCEATMMCARSREA 182
EVLVDGRLIYTANDLKVGLFQDTSAF 208

FIG. 9. Sequence alignment of the proposed *M. tuberculosis* β-hydroxy-acyl-ACP dehydrase FabZ (Rv0098) with *E. coli* CFT073 FabZ (AAN78709), *S. pneumoniae* FabZ (NC_003028, SP0424 open reading frame), and *E. coli* CFT073 FabA (AAN79558). Multiple-sequence alignments were obtained with the Clustal W program (version 1.8) (106). Active-site residues His and Glu are indicated by asterisks. Black shading indicates identical residues; gray shading indicates similar residues. Hyphens represent gaps introduced to maintain the alignment. For better visualization, the first 23 residues of *E. coli* FabA are not included in the alignment, because none of them matched in the comparison.

urated fatty acid by converting trans-2-enoyl-ACP to cis-3-enoyl-ACP. FabZ is the primary dehydrase in the elongation of unsaturated acyl-ACP, and it is more active than FabA toward long-chain saturated acyl-ACP (44). As previously reported, a search of *M. tuberculosis* genome failed to yield FabA-like dehydrase/isomerase (11, 23). In this study, sequence similarity searches were performed by using BLAST with default parameters provided at the National Center for Biotechnology Information (NCBI) (http://www.ncbi.nlm.nih.gov/BLAST).

We then turned to the *S. pneumoniae* β-hydroxyacyl-ACP dehydrase FabZ. *S. pneumoniae* lacks homologs of the *fabA* gene, and FabZ is the dehydrase that participates in the elongation of saturated and unsaturated fatty acids (125). We compared the sequence of FabZ from *S. pneumoniae* with sequences of proteins in the genome of *M. tuberculosis* by using the NCBI BLAST program. This revealed that a 183-amino-acid hypothetical protein encoded by Rv0098 shared 36% identity and 54% similarity with *S. pneumoniae* FabZ in a 33-amino-acid segment containing active-site residues with an E value of 2.6. Amino acid sequence alignment of Rv0098 with *S. pneumoniae* FabZ and *E. coli* FabA and FabZ showed that active-site residues His⁷² and Glu⁹⁷ are conserved in Rv0098 (Fig. 9). Computational analysis showed that Rv0098 is among 219 core mycobacterial genes that are essential for mycobacteria and that it shows no similarity with proteins from other microorganisms (58). The Rv0098 gene product is required for survival of *M. tuberculosis* in mouse lung macrophages (88). Genes homologous to Rv0098 were found in *M. leprae*, *Mycobacterium bovis*, and *M. smegmatis*. The protein encoded by Rv0098 shared 68% identity with a 183-amino-acid conserved hypothetical protein in *M. leprae*. We thus propose that Rv0098 is a good candidate for the gene that encodes β-hydroxyacyl-ACP dehydrase (FabZ) in *M. tuberculosis*.

The other possible candidate gene for *M. tuberculosis* FabZ is Rv0130. When the *M. tuberculosis* genome was compared with the predicted acyl dehydratase (NCgl0284) of *C. glutamicum* ATCC 13032, the hypothetical protein Rv0130 was found to share 44% sequence identity with NCgl0284 in a 147-amino-acid segment with an E value of 2E−33. Sequence analysis predicted that Rv0130 is synthesized as an intracellular protein and belongs to the MaoC dehydratase gene family (http://pfam.wustl.edu/; Washington University, St. Louis, Mo.); however, active-site residues of *E. coli* FabA and FabZ dehydratases were not conserved in Rv0129. Interestingly, Rv0130 is clustered with the mycolyltransferase gene *fbpC* (Rv0129c).

2-trans-Enoyl-ACP isomerase. We expect 2-trans-enoyl-ACP isomerase to be present only in the FAS-IIA and FAS-IIB modules. In *S. pneumoniae* the Sp0415 gene adjacent to the FAS-II gene cluster was annotated as an enoyl-CoA hydratase/isomerase family protein. Biochemical and genetic analysis of this gene product established that the previously unknown enzyme, designated FabM, functions as an isomerase in the FAS-II system (60). In a reconstituted fatty acid biosynthesis system, FabM was capable of isomerizing 2-trans-enoyl-ACP to 3-cis-enoyl-ACP but could not dehydrate β-hydroxyacyl-ACP (60). Our analysis of the *M. tuberculosis* genome by using the NCBI BLAST program revealed that a group of 21 enzymes share strong sequence homology with *S. pneumoniae* FabM (C. Wang and K. Takayama, unpublished results). These enzymes were annotated as the probable enoyl-CoA hydratases (23). Sequence analysis predicted that these enzymes are synthesized as intracellular proteins and belong to the hydratase/isomerase superfamily (Pfam000378; http://pfam.wustl.edu/). Of these enzymes, EchA10 (Rv1142c) and EchA11 (Rv1141c) shared the highest sequence homology with the *S. pneumoniae* FabM (Fig. 10). EchA10 and EchA11 shared 26 to 28% identity.

EchA10 MSNYRIDTRTIVPGLAVTLADGVLSVTIDRPESLNSLTKPVLAGMADAIEGAATDPRVKV 60  
EchA11 MPDSGIAALTPVTGLNVTLTDRVLSVRINRPSSILNSLTVPILTGIAADTLERAAADPVVKV 60  
S. pneumoniae FabM ----------------MEHIIYQLEEDLAILTLNRPEVANGFHIPMCEEILEALTIAEENPAVHF 49  

EchA10 VRLGGAGRGFSSGGAISVDDVWASG------PPTDTVAEANRTVRAIVALPQPVVAVVQGPT 116  
EchA11 VRLGGVGRGFSSGVMSVDDVWGGG------PPTAIVEEANRAVRAVAALPHPVVAWVQGPA 116  
S. pneumoniae FabM ILINANGKVFSVGGDLVEMKRAVEDDIPSLTKIAELVNTISYKIKQIAKPVLMEVDGAV 109  

EchA10 VGC GVSLALA CDLVLASDNAFFMLAH TNVGLMPDG GASALVQAAIGRI RAMHMALLLPDRV 176  
EchA11 VGVAVSLALA CDFILLASDSAFFMLANT KVALLMPDG GASALVAAATGRI RAMRLALLLAEQL 176  
S. pneumoniae FabM AGAAANMAVA AADC LLATDKAKFIQAFVGGLAPDAGGIHLLSRSIGVTRAALAMTGEAL 169  

EchA10 PAAEALS WGLVSAV YPAADFDAEV DKLISRLL LAGPALAI AKTKNAI NAATLTE LAPTLLR 236  
EchA11 PAREALAWGLISAV YPDSDFEAEV DKVISRL LAGPALAFA QAKNAINA AAALTELEPTFAR 236  
S. pneumoniae FabM TAEKALEWGLVYRV SEA EKLEKTREQLLKLRASSNSYAAIKKLWESQFKDWQGYATL 229  

EchA10 ELDGQ ALLLRTDDFAEGA TAFQQRRTP MFTGR 268  
EchA11 ELDGQE VLLRTHDFAEGAAAFL QQRRTP NFTGS 268  
S. pneumoniae FabM ELNLQ KSLAQTEDFKEGVR AHSE RRRPK---- 257  

FIG. 10. Sequence alignment of the proposed *M. tuberculosis* 2-trans-enoyl-ACP isomerases FabM (EchA10 and EchA11) with *S. pneumoniae* FabM (NC_003028, SP0415 open reading frame). Multiple-sequence alignments were obtained with the Clustal W program (version 1.8) (106). Active-site residue Asp is indicated by an asterisk. Black shading indicates identical residues; gray shading indicates similar residues. Hyphens represent gaps introduced to maintain the alignment.

tity and 42 to 44% similarity with the *S. pneumoniae* FabM, with E values of 2E−23 and 1E−21, respectively. The proton donor of the enoyl-CoA hydrase/isomerase family was identified as Asp<sup>150</sup> in EchA10/EchA11 and as Asp<sup>143</sup> in *S. pneumoniae* FabM (70). Other *echA* genes shared 23 to 27% identity and 40 to 43% similarity with the *S. pneumoniae* FabM, with E values ranging from 0.001 to 6E−20. Computational methods have been developed to identify genes that are involved in common biochemical pathways and facilitate the inference of protein function (99). EchA10 and EchA11 are not functionally linked, whereas each gene product is functionally linked with other EchA family members (99). Although EchA10 and EchA11 share the highest sequence homology among EchA family members, these two enzymes may have distinct substrate specificities and are not functionally redundant. Determination of whether the *echA10* and *echA11* genes encode the 2-trans-enoyl-ACP isomerase for the synthesis of cis-unsaturated meroacid precursors, as proposed in this review, awaits further study.

An alternative method of introducing a double bond(s) to form Δ<sup>5</sup>-C<sub>24:1</sub> and Δ<sup>19,31</sup>-C<sub>50:2</sub> fatty acid precursors of α-meroacid (Fig. 4, reactions 3 and 4 and reactions 8 and 9) and Δ<sup>19,37</sup>-C<sub>56:2</sub> and Δ<sup>21,39</sup>-C<sub>58:2</sub> fatty acid precursors of methoxy- and keto-meroacids (Fig. 4, reaction 10) is by the action of desaturases (oxidative desaturation). Analysis of the *M. tuberculosis* genome reveals three potential aerobic desaturases encoded by *desA1*, *desA2*, and *desA3* (23). Overexpression of *desA3* in *M. bovis* BCG and biochemical analysis of the purified enzyme showed that DesA3 desaturase is responsible for the synthesis of oleic acid (*cis*-Δ<sup>9</sup>-C<sub>18:1</sub>) (75). From these results, it appears unlikely that DesA3 is involved in the synthesis of mycolic acid. The possible roles of DesA1 and DesA2 in the synthesis of mycolic acid have not been evaluated.

InhA (2-trans-enoyl-ACP reductase). In the FAS-II elongation module, an NADH-dependent 2-trans-enoyl-ACP reductase encoded by *inhA* (*Rv1484*) catalyzes the final step of a cycle of elongation, the reduction of 2-trans-enoyl chains (>C<sub>12</sub>) to yield the saturated chains (28, 79). InhA from *M. smegmatis* displays long-chain fatty acid elongation activity, which promotes the elongation of C<sub>16</sub>-ACP to form C<sub>18-</sub> to C<sub>30</sub>-ACP products (59). The preference of InhA for long-chain substrates is consistent with its involvement in the elongation of meroacid precursors in *M. tuberculosis*.

The *M. tuberculosis* InhA protein shares 90% sequence identity with its homolog in *M. leprae*. Sequence analysis revealed that four conserved hypothetical proteins (encoded by *Rv3485c*, *Rv0927c*, *Rv3530c*, and *Rv3559c*) share 24 to 26% sequence identity with InhA. Further studies are needed to determine their roles in the synthesis of mycolic acid.

Cyclopropane Synthases and Methyltransferases

Cloning of the genes in the path to cyclopropane synthases and methyltransferases. The first gene to be identified to modify mycolic acid in *M. tuberculosis* was *cmaA1*, which encodes a cyclopropane mycolic acid synthase. It was cloned based on its homology to the *E. coli* cyclopropane fatty acid synthase gene (123). The same group (Barry and associates) cloned another gene, *cmaA2*, and it shares 52% identity with *cmaA1* (39). By using the *cmaA1* gene as a probe, a cluster of genes encoding four highly homologous methyltransferases (>52% identity) was cloned in *M. tuberculosis* (120). These genes were designated *mmaA1* to *mmaA4* to indicate their involvement in methoxy-mycolic acid biosynthesis when introduced into *M. smegmatis*. Dubnau et al. cloned a homologous gene cluster of methyltransferases (*cmaA* to *cmaD*) from *M. bovis* BCG (32).

Two additional potential cyclopropane synthases were identified in the *M. tuberculosis* genome and annotated as *umaA1* and *umaA2* (23). The function of *umaA1* is not known. The gene *umaA2* was identified to encode the proximal cyclopropane synthase of α-mycolic acid and was renamed *pcaA* (41). These eight genes share between 50 and 75% identity, and the enzymes are all thought to utilize SAM in methyl transfer reactions (121). In vitro assays of recombinant methyl transferases indicate that the immediate substrates of these enzymes are AcpM-bound meroacid precursors (Fig. 4 and 6) (121). Biochemical functions of this gene family were defined by genetic manipulation of each synthase in *M. tuberculosis* and *M. smegmatis* as well as by systematic analysis of the meroacids.

MmaA2 is required for introduction of the distal cyclopropane ring in the formation of α-meroacid. Analysis of an *mmaA2* deletion mutant of *M. tuberculosis* revealed that α-mycolic acid lacks a distal cyclopropane group and instead contains a *cis* unsaturation (40). Thus, *mmaA2* is required for the distal cyclopropane modification of α-mycolic acid. An early study suggested that the *cmaA1* product might also function as a distal cyclopropane synthase for α-mycolic acid. When *cmaA1* was overexpressed in *M. smegmatis*, it was shown that the distal *cis* unsaturation was transformed into a cyclopropane ring (123). However, a recent study demonstrated that *cmaA1* is not required for α-mycolic acid modification. The α-mycolic acid of an *M. tuberculosis* mutant with a *cmaA1* deletion was unaffected, and *cmaA1* had no discernible role in mycolic acid modification (40). It is likely that the product of *cmaA1* nonspecifically catalyzed distal cyclopropane ring synthesis when highly overexpressed in *M. smegmatis*.

PcaA (UmaA2) is required for introduction of the proximal cyclopropane ring in the formation of α-meroacid. A *pcaA* deletion mutant of *M. tuberculosis* produced α-mycolic acid at a reduced level compared to the wild-type control (41). In addition, a hybrid mycolic acid accumulated with a *cis* unsaturation at the proximal position in place of the *cis*-cyclopropane ring (41). These results suggest that *pcaA* encodes a cyclopropane synthase that is responsible for the transformation of *cis* unsaturation to a *cis*-cyclopropane ring at the proximal position of α-mycolic acid. The accumulation of the hybrid mycolic acid in the *pcaA* deletion mutant of *M. tuberculosis* supports the view that meroacid precursors with *cis* unsaturation are first produced and subsequently modified as shown in Fig. 4 (113).

The fact that deletion of *pcaA* did not completely eliminate the synthesis of α-mycolic acid indicates that another enzyme may share the specificity of PcaA in terms of modifying α-mycolic acid at the proximal position. A previous study suggested that *cmaA2* might function as a proximal cyclopropane synthetase for α-mycolic acid. When overexpressed in *M. smegmatis*, *cmaA2* introduced *cis*-cyclopropane rings at the proximal positions of α-mycolic acid and epoxy-mycolic acid (39). However, deletion of *cmaA2* in *M. tuberculosis* did not alter the α-mycolic acid content (42).

MmaA4 introduces the distal-branch methyl group in the formation of trans-oxygenated meroacids. When the *mmaA4* gene of *M. tuberculosis* was inactivated, the mutant synthesized only α-mycolic acid and not oxygenated mycolate (29, 31). Genetic complementation restored the production of oxygenated mycolate (29). In addition, monoethylenic (at the distal position) monocyclopropanated (α₂) mycolate accumulated in this mutant and presumably is the substrate of MmaA4 (29). *M. smegmatis* does not contain methoxy- and keto-mycolates. Overexpression of *mmaA4* in *M. smegmatis* led to the synthesis of keto- and hydroxy-mycolates while synthesis of diunsaturated α-mycolates was significantly reduced (29). Overexpression of *mmaA4* or *cmaA* (the homolog of *mmaA4* in *M. bovis* BCG Pasteur) in *M. smegmatis* induced the production of methyl-branch hydroxy-mycolic acids (32, 120). These results indicate that MmaA4 is responsible for the production of methyl-branch hydroxy-mycolic acids, which are the common precursors for oxygenated mycolates in *M. tuberculosis* as shown in Fig. 5 and 6.

The methyl-branch hydroxy-meroacid precursor produced by MmaA4 is converted to the methyl-branch keto-meroacid precursor through an oxidation-reduction process (Fig. 6). The enzyme responsible for this transformation has not been identified. Since the proposed enzyme, provisionally called NAD(P)H-dependent hydroxyacyl-ACP dehydrogenase, participates in the same biochemical pathway as methyltransferases, the *M. tuberculosis* genome was analyzed to identify genes that are functionally linked with methyltransferases (99). Computational analysis showed that an oxidoreductase/oxygenase encoded by *Rv0161* is functionally linked with all methyltransferases, indicating that Rv0161 is a possible candidate enzyme responsible for the synthesis of methyl-branch keto-meroacid precursors. The proposed NAD(P)H-dependent hydroxyacyl-ACP dehydrogenase may belong to the alcohol dehydrogenase family. We have searched the genome of *M. tuberculosis* for alcohol dehydrogenases and have found a few candidate enzymes encoded by *Rv0162c* and *Rv3057c* (C. Wang and K. Takayama, unpublished results). *Rv0162c* encodes a probable zinc-type long-chain alcohol dehydrogenase. *Rv3057c* encodes a probable alcohol dehydrogenase/reductase, and it shares similarity with various oxidoreductases involved in polyketide synthesis.

MmaA3 O-methylates the distal secondary alcohol in the formation of *cis-* and *trans*-methoxy-meroacids. When *mmaA3* was transferred into *M. smegmatis*, it appeared to have no independent function. However, when coexpressed with *mmaA4*, *mmaA3* appears to O-methylate the hydroxyl group newly formed by MmaA4 by using a SAM-derived methyl group (120). Amino acid alterations of CmaB (the homolog of MmaA3 in *M. bovis* BCG Pasteur) are likely responsible for the failure of *M. bovis* BCG Pasteur methyltransferase to produce methoxy-mycolic acid in *M. smegmatis* (32). Overexpression of *mmaA3* in *M. tuberculosis* resulted in the loss of keto-mycolates and a significant increase in methoxy-mycolate production (124), strongly supporting the idea that hydroxy-mycolate is the common intermediate for both keto- and methoxy-mycolate (as shown in Fig. 5 and 6). Thus, *mmaA3* encodes the O-methyltransferase responsible for the transfer of the methyl group from SAM to the 37-hydroxyl groups to form the methoxy group (Fig. 6).

MmaA2 and CmaA2 are required for introduction of the proximal *cis*-cyclopropane ring in methoxy-meroacids, and Mma2 is required for introduction of the proximal *cis*-cyclopropane ring in keto-meroacids. Genetic transfer of *mmaA2* into *M. smegmatis* led to the conversion of the proximal *cis* unsaturation in mycolic acids to a *cis*-cyclopropane ring (120).

An MmaA2-deficient mutant of *M. tuberculosis* produced both complete and incomplete (*cis* unsaturation-containing) methoxy-mycolic acids. The amount of complete methoxy-mycolic acid was reduced to one-half of that in the wild type (40). The ratio of *cis*- to *trans*-keto-mycolic acid was not altered in this mutant. These results indicate that another enzyme may share the specificity of MmaA2 to transform *cis* unsaturation to a *cis*-cyclopropane ring in oxygenated meroacids at the proximal position. When the *mmaA2* gene is transferred into *M. smegmatis*, the organism produces mycolic acids containing a *cis*-cyclopropane ring but not a *trans*-cyclopropane ring (39). It is possible that CmaA2 has a redundant role with MmaA2 in catalyzing *cis*-cyclopropane synthesis of methoxy-mycolates. However, MmaA2 may be the preferred cyclopropane synthetase for methoxy- and keto-mycolate synthesis (Fig. 6) (40).

MmaA1 and CmaA2 are required for introduction of the proximal-branch methyl group and *trans*-cyclopropane ring in *trans*-oxygenated meroacids. When *mmaA1* was transferred into *M. smegmatis*, it appeared to have no independent function (120). When *mmaA1* was overexpressed in *M. tuberculosis*, it resulted in an increase in the amount of the oxygenated mycolate-containing *trans*-cyclopropane ring and the appearance of oxygenated mycolates containing *trans* unsaturation with an allylic methyl branch (122), which suggests that MmaA1 converts *cis* unsaturation into *trans* unsaturation with the simultaneous introduction of an allylic methyl branch to oxygenated meroacid precursors. Because overexpression of *mmaA1* in *M. tuberculosis* produced an excess of both *trans*-unsaturated and *trans*-cyclopropanated mycolic acids, MmaA1 action is presumably an early step in *trans*-cyclopropane synthesis (Fig. 5).

Inactivation of *cmaA2* in *M. tuberculosis* abolished *trans*-cyclopropanated mycolates and caused the accumulation of unsaturated oxygenated mycolates (42). These results indicate that *cmaA2* encodes the *trans*-cyclopropane synthase of oxygenated meroacids (Fig. 6).

### Claisen-Type Condensation

Claisen-type condensation was proposed by Walker et al. (112) as the mechanism for the condensation of two molecules of palmitic acids to form the C₃₂-corynomycolic acid in a cell-free system of *C. diphtheriae*. Takayama and Qureshi (100) used this mechanism to describe the final step in the synthesis of mycolic acid in *M. tuberculosis*. With the discovery by Portevin et al. (76) that Pks13 is involved in the Claisen-type condensation as the final step in the synthesis of mycolic acid in *M. tuberculosis*, details of this condensation reaction can now be described as shown in Fig. 7. This condensing enzyme is encoded by *pks13* (*Rv3800c*). In *M. tuberculosis*, the enzyme Pks13 is a member of the type I polyketide synthase (Pks) gene family. Sequence analysis revealed that Pks13 contains two non-equivalent phosphopantetheine-binding, ketacyl synthase, acyl transferase, and thioesterase domains (Fig. 7). Pks13 shares 83% sequence identity with its homologous gene in *M. leprae* (ML0101). A *C. glutamicum* mutant with a deletion in the *pks13* gene was unable to synthesize corynomycolic acid and produced large amounts of precursors, C₁₆ to C₁₈ fatty acids (35a). Since Pks13 is essential for the viability of Mycobacteria, mycolic acid synthesis was analyzed in a conditional mutant of *M. smegmatis* containing a deletion in *pks13* on the chromosome and a functional *pks13* gene on the thermosensitive plasmid. A marked reduction of mycolate production (60%) was observed when the mutant was grown at a nonpermissive temperature compared to growth at the normal temperature. The identity of the reductase, which converts the 3-oxo-mycolic acid to mycolic acid, remains unclear. A number of genes in the *M. tuberculosis* genome share sequence homology with the gene for the known acyl reductase in the FAS-II system, β-ketoacyl-ACP reductase (MabA). Determination of whether any of these enzymes can function as the 3-oxo-mycolic acid reductase awaits further investigation.

Gande et al. performed gene deletion experiments to determine which enzymes are required for corynomycolic acid synthesis in *C. glutamicum* (35a). They found that Fas-IA is required for the generation of C₁₆ fatty acid, FadD is required for the formation of C₁₆-S-CoA (or possibly C₁₆-AMP), AccD2 and AccD3 are required for the formation of the carboxylated C₁₆-S-CoA, and Pks is required for the Claisen-type condensation. Phylogenomic analysis showed that two clusters of carboxyltransferases (AccD2 and AccD3 in *C. glutamicum*) are well conserved in Corynebacteriaceae. The equivalent systems and orthologs in *M. tuberculosis* would be FAS-I/FAS-II (to generate the meroacids), FadD32, AccD4, AccD5, and Pks-13, respectively (Fig. 7). A more detailed comparison of the two systems (Claisen-type condensation in corynebacteria and mycobacteria) would be of great interest, since they should be similar.

#### Mycolic Acid Processing System

##### Mycolyltransferase I and mycolyltransferase II

Mycolyltransferase I and mycolyltransferase II are the proposed enzymes that would transfer newly synthesized mycolic acid in thioester linkage to Pks13 (Fig. 7), first to mannopyranosyl-1-phosphoheptaprenol to yield Myc-PL and then to trehalose 6-phosphate to yield the TMM-phosphate (6-O-mycolyl-trehalose-6′-P) (Fig. 8, reactions 1 and 2). We are addressing an outstanding question of how the key carrier of the mycolyl group, TMM, is synthesized. These reactions are hypothetical and have not been studied.

A recent study demonstrated that an acyltransferase (PapA5), which catalyzes diesterification of phthiocerol and phthiodiolone with mycocerosic acid, is located in the phthiocerol and phthiodiolone dimycocerosate ester synthesis gene cluster (74). The genes *papA1*, *papA2*, and *papA3* from *M. tuberculosis*, encoding proteins presumably catalyzing O-esterification of trehalose with the methyl-branched polyketides, are associated with loci encoding Mas-like polyketide synthases. We examined the *pks13* locus and found that Rv3802c may function as the proposed mycolyltransferase I by catalyzing the transfer of newly synthesized mycolic acid from Pks13 to mannopyranosyl-1-phosphoheptaprenol (Fig. 8, reaction 1). *Rv3802c* encodes an essential protein for growth (87), and it is located immediately upstream of *fadD32* and *pks13* and downstream of the mycolyltransferase gene *fbpA* (Fig. 11). Sequence analysis predicted that Rv3802c is synthesized as a membrane protein and belongs to the serine esterase cutinase gene family (Pfam01083; http://pfam.wustl.edu).

We performed an NCBI BLAST search of the *M. tuberculosis* genome and identified two homologous sequences of the

value of 1.3. The protein encoded by Rv0519c shares 29% sequence identity and 48% similarity with Rv1288 in a 114- amino-acid overlap, with an E value of 2.1. The highly homologous proteins encoded by Rv0519c and Rv0774c share 58% sequence identity in a 299-amino-acid overlap, which indicates that these two proteins may have similar cellular functions. Like in Rv1288, the proteins encoded by Rv0519c and Rv0774c belong to the esterase D family (12). An NCBI BLAST search of mycobacterial genomes revealed that Rv0519c and Rv0774c have homologous genes in M. bovis, M. smegmatis, M. marinum, and M. avium. A catalytic triad (Ser, His, and Arg) was identified by sequence alignment of proteins encoded by Rv0519c and Rv0774c with mycolyltransferases (FbpA, FbpB, and FbpC) (Fig. 13). Rv0519c is predicted to encode a 300-amino-acid basic membrane protein, whereas Rv0774c encodes a 312- amino-acid exported protein. The potential cellular localization of these enzymes is consistent with the proposed functions of transferring mycolic acid from Myc-PL to trehalose 6-phosphate on or close to the inner membrane surface. These two proteins are additional candidates for mycolyltransferase II (Fig. 8, reaction 2).

Phosphatase. We wish to identify the gene that encodes the TMM-6-phosphate phosphatase. Two possible trehalose 6-phosphate phosphatase genes (otsB1 and otsB2) were identified in M. tuberculosis (23). A recent study demonstrated that recombinant OtsB2 exhibited highly specific trehalose 6-P phosphatase activity, while OtsB1 had no detectable phosphatase activity (33). Our analysis of the M. tuberculosis genomic sequence revealed that a probable hydrolase gene (Rv3400) and the N-terminal hydrolase domain of otsB1 (Rv2006) share significant sequence homology with otsB1 and otsB2 (46 to 48% identity and 58 to 59% similarity) (C. Wang and K. Takayama, unpublished results). This is shown in Fig. 14. An NCBI BLAST search of mycobacterial genomes revealed that otsB1 is unique to the M. tuberculosis and M. bovis genomes, while homologous genes of Rv3400 were found in M. bovis, M. smegmatis, M. marinum, M. leprae, and M. avium. Rv3400 encodes a 262- amino-acid acidic protein that is predicted to be an intracellular protein. This enzyme belongs to a subgroup of phosphatases that use aspartate residues as the active-site nucleophile. A characteristic N-terminal Asp-X₁-Asp-X₂-Thr/Val motif is conserved in Rv3400 (24, 105). Variations in the C-terminal motif have been reported for this class of phosphatases (95). Computational analysis revealed that Rv3400 is functionally linked with otsB1 (99). The gene products of otsB1 and Rv3400 are potential candidates for TMM-P phosphatase (Fig. 8, reaction 3).

ABC transporter. We hypothesize that the proposed TMM transporter in M. tuberculosis is actually an ABC transporter (Fig. 8, reaction 4). Analysis of the M. tuberculosis genome by Braibant et al. revealed the presence of at least 26 complete (including at least one nucleotide-binding domain and one membrane-spanning domain) and 11 incomplete ABC transporters (17). Of the 37 potential ABC transporters, two are identified with no known functions and another four are similar to multidrug resistance proteins of eukaryotes and prokaryotes. The physiological role of multidrug transporter has been controversial. It has been proposed that they may be involved in the transport of substrates that share structural properties with drugs. Several studies have shown that multidrug transporters are able to transport a variety of lipids. The human multidrug

FIG. 11. Functional links and operon inference for a region involved in the biosynthesis of mycolic acid. Inference of Rv3802c protein function and operon organization is based on a combined computational approach (99). OP, operon method; RS, Rosetta Stone method; PP, phylogenetic profile method; GN, conserved gene neighbor method.

Ag85 complex encoded by Rv0947c and Rv1288. Rv0947c encodes a 76-amino-acid protein (corresponding to the N-terminal sequence of the Ag85 complex) and shares 56% identity and 70% similarity with FbpA in the 48-amino-acid overlap, with an E value of 1E–07 (C. Wang and K. Takayama, unpublished results). An NCBI BLAST search of mycobacterial genomes revealed that the gene in M. bovis (Mb0972c) is identical to Rv0947c. Rv1288 shares 26 to 31% sequence identity and 41 to 46% similarity with the Ag85 complex (FbpA, FbpB, and FbpC) at the amino acid level (Fig. 12). Rv1288 encodes a 456-amino- acid protein and shares 26% identity and 41% similarity with FbpA in a 128-amino-acid overlap, with an E value of 8E–05. An NCBI BLAST analysis of mycobacterial genomes revealed that Rv1288 has homologous genes in M. bovis, M. smegmatis, M. marinum, M. leprae, and Mycobacterium avium. Further bioinformatics analysis revealed that the N-terminal amino acid sequence of the protein encoded by Rv1288 (amino acids 5 to 153) shows three LysM (lysine motif) domains. LysM was originally identified in enzymes that degrade bacterial cell walls and is also present in proteins known to be associated with bacterial cell walls (15). A recent study showed that the LysM domain mediates direct binding to peptidoglycan on the cell wall (98). The C-terminal amino acid sequence of the protein encoded by Rv1288 (amino acids 73 to 443) indicates an esterase D family member (12). This family of esterases includes Ag85 and C. glutamicum mycolyltransferase PS1 (78). The active-site residues Ser¹²⁴, His²⁶⁰, and Glu²²⁸, forming the catalytic triad, are conserved in Rv1288. We propose that the Rv1228 gene product is likely a candidate protein for mycolyltransferase II (Fig. 8, reaction 2). It is expected that the substrate- binding sites would be different from those of Ag85. The potential cell wall localization of the protein encoded by Rv1288 suggests that this enzyme may play a role in cell wall mycolate-peptidoglycan formation. Studies of the effect of genetic deletion of Rv1288 and inhibition of its gene product may shed light on its role in the processing of mycolic acids.

Further analysis of the M. tuberculosis genomic sequence revealed two conserved proteins encoded by Rv0519c and Rv0774c that shared low sequence homology with the protein encoded by Rv1288 (11.0% identity and 30.4% similarity) (C. Wang and K. Takayama, unpublished results). The protein encoded by Rv0774c shares 27% sequence identity and 43% similarity with Rv1288 in a 66-amino-acid overlap, with an E

A Rv1288
I II III Ser Glu His
5-51 56-102 107-153 294 394 425 AA

B
** ** * * * * * * * * * * * * * * * * * *
LysM I HAVVAGETLSALALRFYGDAELYRLIAAAS-GIADPDVVNVGQRLIMP 51
LysM II YTVVAGDTLSALALRFYGDAELNWLIAAAS-GIADPDVVNVGQRLIMP 102
LysM III YTVVAGDTLSALAARFYGDASLYPLIAAVN-GIADPGVIDVGQVLVIF 153

C
Rv1288 WLIAAASGIADPDVVNVGQRLIMPDFTRYTVAAGDTLSALAARFYGDASLYPLIAAVNGI 138
FbpA ---MQLVDVRVRGAVTGMSRRLVVGAVGAALVSG------------------LVGAVGGT 38
FbpB -----MTDVSR-KIRAWGRRLLMIGTAAAALVLPG------------------LVGLAGGA 35
FbpC MTFFEQVRRLLRSAATTLPRRLAIIAMGAVLVYG------------------LVGTFGGP 41

Rv1288 ADPGVIDVGGQVLVIFIGRSDFGGLRIVDRNENDPRLWYYRFQTSAIGWNPGVNVLLPDDY 198
FbpA ATAGAFSRPGLPVEYLQVP-----------------------SPSMGRDIKVQF 69
FbpB ATAGAFSRPGLPVEYLQVP-----------------------SPSMGRDIKVQF 66
FbpC ATAGAFSRPGLPVEYLQVP-----------------------SASMGRDIKVQF 72

Rv1288 RTSGRTYPVLYLHFGGGTDQDFRTEDFLG-IRDLTAGKPIIIVMPDGGHAGWYSNPVSSF 257
FbpA QSGGANSPALYLLDGLRAQQDFSQWDINTPAFEWYDQSGLSVVMPVGGQSSFYSDWYQPA 129
FbpB QSGGNNSPAVYLLDGLRAQDDYNGWDINTPAFEWYYQSGLSIVMPVGGQSSFYSDWYSPA 126
FbpC QGGG--PHAYYLLDGLRAQDDYNGWDINTPAFEEYYQSGLSVIMPVGGQSSFYTDWYQPS 130

Rv1288 VG--------PRNWETFHIAQLLLPWIEANFRTYAEYDGRAVAGFSMGGF GALKYAAKYGHFA 312
FbpA CGKAGCQTYKWETFLTSELPGWLQANR--HVKPTGSAVVGLSMAASSALT LAIYHPQQFV 187
FbpB CGKAGCQTYKWETFLTSELPOWLSSANR--AVKPTGSAAIGLSMAGSSAM ILAAAYHPQQFI 184
FbpC QSNGQNYTYKWETFLTREMPAWLQANK--GVSPTGNAAVGLSMSGG SALILAAYYPQQFP 188

Rv1288 SASSHSGBPASLRRDFGLVHWANLSSAVLDLGGGTVYG---APLWDQARVSADNPVERID 369
FbpA YAGAMSGLLDPSQAMGPLILGLAMG-DAGGYKASDMWGPKEDPAWQRNDPLLNVGKLIAN 246
FbpB YAGSLSALLDPSQGMGPSLIGLAMG-DAGGYKAADMWGPSSDPAWERNDPTQQIPKL VAN 243
FbpC YAASLSGFLNPSEGWWPTLIGLAMN-DSGGYNANSMWGPSSDPAWKRNDPMVQIPRLVAN 247

Rv1288 SYRNKRIFLVAGTSPDP--ANWFDSVNETQVLAGQREFRERSNAGIPHE SHEVP--GGH 425
FbpA NTR-VWVYCGNGKPSDLGGNNNLPAKFLEGFVRTSNIKFQDAYNAGG GHNGVFDFPDSGTH 305
FbpB NTR-LWVYCGNGTPNELGGANIPAEELENFVRSSNLKFQDAYNAAGGHNAVFNFPPNGTH 302
FbpC NTR-IWVYCGNGTPSDLGGDNIPAKFLEGLTLRTNQTFRDTYAADG GRNGVFNEFPPNGTH 306

Rv1288 VFRPDMFRLDLDGIVARLRPASIGAAERAD------ 456
FbpA SWEYWGAQLN--AMKPDLQRAL-GATPNTGPAPQGA 338
FbpB SWEYWGAQLN--AMKGDLQSSL-GAG---------- 325
FbpC SWPYWNEQLV--AMKADIQHVLNGATPPAAPAAPAA 340

FIG. 12. Sequence of the proposed mycolyltransferase II (Rv1288) from *M. tuberculosis*. (A) Schematic presentation of Rv1288. The N-terminal LysM repeats are named I to III. The esterase D domain of the protein is located within the N-terminal amino acids 73 to 443 of the protein. AA, amino acids. (B) Amino acid sequence alignment of repeated N-terminal LysM motifs I to III of Rv1288 plus intervening sequences. Asterisks indicate the consensus sequence proposed by Joris et al. (46). (C) Amino acid sequence alignment of Rv1288 with mycolyltransferases (FbpA, FbpB, and FbpC). Active-site residues Ser, Glu, and His are indicated by asterisks. For better visualization, the first 78 residues of Rv1288 are not included in the alignment, because none of them matched in the comparison. The CLUSTAL W program (version 1.81) (106) was used for multiple-sequence alignment. Black shading indicates identical residues; gray shading indicates similar residues.

resistance P-glycoprotein MDR1 mediates transport of short-chain lipid analogs (109) as well as the translocation of native glycosphingolipid precursor glucosylceramide (110). The human multidrug resistance P-glycoprotein MDR3 is able to transport phosphatidyl choline across the canalicular membrane of the hepatocyte (109). A recent study showed that MsbA (lipid A transporter in *E. coli*) and LmrA (multidrug transporter in *Latobacillus lactis*) have overlapping specificities

Rv0774c  --------LPIRPNVHGMMARMPELSRRAVLGLG-AGTVLGATSAYAIDMLLQPRTSHAAP 52
Rv0519c --VLRRGCAGNTDRRGIMTPMADLTRALLRWG-AGAGAGAAAGVWAFGALVDPLEPQAAP 57
FbpA ---MQLVDRVRGAVTGMSRRLVVGAVGAALVSGLVGAVGGTATAGAFSRPGLPVEYLQVP 57
FbpB ----MTDVSR-KIRAWGRRIMIGTAAAIVVLPGLVGLAGGAATAAGAFSRPGLPVEYLQVP 54
FbpC MTFFEQVRRILRSAATTLPRRLAIAAMGAVLVYGLVGTFGGPATAGAFSRPGLPVEYLQVP 60

Rv0774c AAAIGTNVPLAPTPALDPAPPAQAAPT MSTGSFVS AARAGKMTNWAIARPPGQTQALRPV 112
Rv0519c AP----------------FEPTAGSSLPTRISGSFISAARGGIKTNWVISMPPGQS GQLRPV 104
FbpA SPSMGRDIKVQ----FQSGGANSPALYLLDGLRAQDDFSGWDINTPAFEWYDQSG-LSVV 112
FbpB SPSMGRDIKVQ----FQSGGNNSPAVYLLDGLRAQDDYNGWDINTPAFEWYYQSG-LSIV 109
FbpC SASMGRDIKVQ----FQGGG--PHAVYLLDGLRAQDDYNGWDINTPAFEEYYQSG-LSVI 113

Rv0774c IALHGLGGSASAVMDGGVEOQ--------LAQAVNAGLPPFAVVSDGGSS-YWHQRASGED 165
Rv0519c IALHGKDGNAGMMLDLGVEOQ--------LARLVKEGKPAFAVVGDGGNT-YWHRRSSSGGD 157
FbpA MPVGGQSSFYSDWYQPACGKAGCQTYKWETFLTSELPGWLQANRHVKPTGSAVVGLSMAA 172
FbpB MPVGGQSSFYSDWYSPACGKAGCQTYKWETFLTSEL PQWLSANRAVKPTGSAAIGLSMAG 169
FbpC MPVGGQSSFYTDWYQPSQSNGQNYTYKWETFLTEMPAWLQANKGVSP TGNAAVGLSM SG 173

Rv0774c AGAMVLNELIPLLDTQR-------LDTSR-----VAFLGWSMGGYGALLLGSRLGPARTAAIC 216
Rv0519c SGAMVLDELLPMLTSMG-------MDTSR-----VGFLGWSMGGYGALLLGARLGPARTAGIC 208
FbpA SSALTLAIYHPQQFVYAGAMSL LDPSQAMGPTLIGLAMGDAGGYKASDMWGPKEDPAWQ 232
FbpB SSAMILLAAYHPQQFIYAGSL SALLDPSQGMGPSLIGLAMGDAGGYKAADMWGPSSDPAWE 229
FbpC GSALILLAAYYPQQFPYAASLSGF LNPSEGWWPTLIGLAMNDSGGYNANSMWGPSSDPAWK 233

Rv0774c AVSPALWLSAGSVAPGSEFDGPDDSANSVEGLPALGSIPIRVDCG--NSDPFYAATKQFV 274
Rv0519c AISPALFTSFTGSTPGA FDSYDDYVQHSVLLGLPALNSIPLRVDCG--TSDRFYFATRQFV 266
FbpA RNDPLLNVGKLIANNTRVWVYCGNGKPSDLGGNNLPAKF--LEGFVRTNIKFQDA--YN 288
FbpB RNDBTQQIPKLVANNTRLWVYCGNGTPNELGGANI PAEF--LENFVRS SNLKFQDA--YN 285
FbpC RNDBMVQIPRLVANNTRIWVYCGNGTPSDLGGDNIPA KF--LEGLTLRTNQTERDT--YA 289

Rv0774c AQLPHP PAGG-FS-P--GGHN GGEFWSAQLPAELTWFAPLITG---------- 312
Rv0519c NQLHQ PPAGS-FS-P--GGHD ASYWREQLPGE LAWMAS---------- 300
FbpA AG---GGHN GVFDFPDSGTHSWEYWGAQLNAMKPD LQRAL-GATPNTGPAPQGA 338
FbpB AA---GGHNAVFNFPPNGTHSWEYWGAQLNAMKGDLQSSL-GAG---------- 325
FbpC AD---GGRNGVFNFPPNGTHSWPYWNEQLVAMKADIQHV LNGATPPAAPAAPAA 340

FIG. 13. Sequence alignment of Rv0774c and Rv0519c (proposed mycolyltransferase II) with FbpA, FbpB, and FbpC (mycolyltransferases). At the top of the alignments, active-site residues Ser, Asp/Glu, and His are indicated by asterisks. Multiple-sequence alignments were obtained with the Clustal W program (version 1.8) (106), and the resulting alignment was refined manually. Black shading indicates residues identical to those in mycolyltransferases; gray shading indicates residues similar to those in mycolyltransferases. For better visualization, the first 78 residues of Rv1288 are not included in the alignment, because none of them matched in the comparison.

for lipid A and chemotherapeutic and cytotoxic drugs (84). These results suggest a possible role of multidrug transporters in transporting lipids (like TMM) to the outer membrane. Further biochemical and genetic studies are needed to determine whether any one of the four MDR transporters in *M. tuberculosis* (encoded by Rv1273c/Rv1272c, Rv1348/Rv1349, Rv0194, and Rv1819c) or the two transporters with no known functions (encoded by Rv1747 and Rv1687c/Rv1686c) is able to transfer TMM from the inside to the outside of the cell membrane.

Antigen 85 complex. The antigen 85 complex is composed of Ag85A (FbpA), Ag85B (FbpB), and Ag85C (FbpC) as the predominant secreted protein in *M. tuberculosis*. The physiological function of Ag85 is to transfer mycolic acid from a TMM to another TMM and cell wall arabinogalactan to form TDM and cell wall arabinogalactan-mycolate, respectively, outside the cells. The corresponding genes encoding these proteins are *fbpA* (Rv3804c), *fbpB* (Rv1886c), and *fbpC* (Rv0129c).

The three Ag85 complex proteins share 65 to 80% identity. The crystal structure of the secreted forms of Ag85C (FbpC) and Ag85B (FbpB) revealed a canonical α/β hydrolase fold; a catalytic triad of Ser, Glu, and His (conserved in FbpA, FbpB, and FbpC); a probable TMM-binding site (a hydrophobic pocket and tunnel); and a probable site for the interaction with human fibronectin (3, 86).

A related gene, *fbpD* (also called *fbpC1*), is located adjacent to *fbpA* but does not belong to the same operon. FbpD (MPT51 protein) is highly related to the Ag85 complex and displays 35 to 39% identity. However, none of the residues of the catalytic triad are conserved within FbpD. The highly conserved carboxyesterase consensus sequence GXSXXG is absent in the FbpD sequence. Consistent with sequence analysis, recombinant FbpD does not exhibit in vitro mycolyltransferase activity (53). The crystal structure of MPT51 revealed that it folds like Ag85 complex members as an α/β hydrolase but does

domain I

Rv3400 NWYRPNYPEVRSRVLGLPEKVRACLFDLDGVLTD TASLHTKAWKAMFDAY AERAERTGE 62  
Rv2006 -----------------HDAVIVGLDNVVDKATRVHAAAWTKFLDDYLTRRPQRTGE 40  
OtsB1 ------------------RPAVFLDFDGTLSDIVERPEAAT---LVDGAAEALRALAAQ 38  
OtsB2 MSQLPDALQALGLADGLVARQPAVFFDFDGTLSDIVEDPDAAW---LAPGALEALQKLAAR 178  

Rv3400 KFVPFDPAADYHTYVDGKKREDGVRSFLSSRAIEIPDG-SPDDPGAAETVYGLGNRKNDM 121  
Rv2006 DHCPLET-HDDYRRFLAGK--PDGVADFLAARGIRLPPG-SPTD-LTDDTVYGLQNLERQT 95  
OtsB1 CPVAVISGRDLADVRNRVKVDGLWLAGSHGFELVA PDGSHHQNAATAAIDGLAEAAAQL 98  
OtsB2 CPIAVLSGRDLADVTQRVGLPGIWYAGSHGFELTA PDGTHHQNDAAAAAI PVLKQAAAEI 238  

Rv3400 LHKLLLRDDGAQVFDG----SRRYLEAVTAAGLGVAVVSSSANTRDVLAT TGLDRFVQQRV 177  
Rv2006 FLQLLN-TGVPEGKS----IASFARRLQVAGVRVAAHTSHRNYGHTLDATGLAEVF AVFV 150  
OtsB1 ADALREIAGAVVEHKRFAVAVHYRN VADDSVDNLI AAVRRLGHAAGLRVT TGRKVVELRP 158  
OtsB2 RQQLGPFPGVVVEHKRFGVAVHYRNAARDRVGEVAAAVRTAEQRHALRVT TGREVI ELRP 298  

Rv3400 DGVTLREEHLAGKPAPDSFLRAAEELLGVTPDAAAVFEDALS GVAAGRAGNF AVVVGINRT 237  
Rv2006 DGAVTAELGLPAEPNPAGLIETAKRLGANPGRCVVIDSCQTGLRAGRNGGFALVIAVD AH 210  
OtsB1 DIAWDKGKALDWIGERLGPAEVGPDLRLPIYIGDDLTEDAFDAVRFTG-VGIWWRHNEH 217  
OtsB2 DVWDWKGTKLLLWVLDHLPHSGSAP--LVPIYLGD DITDEDAFDVVGPHG-VPIWWRHTDD 355  

Rv3400 GRAAQAAQLRRHGADVVVTDLAELL---------- 262  
Rv2006 G----DAENLLSSG--------------------- 220  
OtsB1 GDRRSAATFRLECPYTVCQFLSQ------------- 240  
OtsB2 GDRATAALEALDSPARVAEFTDRLARQLREAPLRA T 391  

FIG. 14. Sequence alignment of the proposed TMM-6-phosphate phosphatase (Rv3400 and Rv2006 N-terminal haloacid dehalogenase-like hydrolase) with trehalose-6-phosphate phosphatases OtsB1 (Rv2006 trehalose phosphatase domain) and OtsB2 (Rv3372). At the top of the alignments, the conserved domain is indicated. Multiple-sequence alignments were obtained with the Clustal W program (version 1.8) (106). Black shading indicates residues identical to those in OtsB1 and OtsB2; gray shading indicates residues similar to those in OtsB1 and OtsB2.

not contain a TMM-binding site (115). These findings excluded FbpD as a potential candidate for the proposed mycolyltransferase I or mycolyltransferase II (Fig. 8).

CONCLUDING REMARKS

The significance of this review can be summarized as follows.

(i) The cyclopropane rings of mycolic acids in *M. tuberculosis* contribute to the structural integrity of the cell wall complex. Moreover, the cyclopropane rings protect the bacillus from oxidative stress. Thus, the structure of mycolic acids determines the degree of protection that *M. tuberculosis* receives against the hostile environment of the host.

(ii) Deletion of the proximal cyclopropane ring of α-mycolate or deletion of the methoxy- and keto-mycolates leads to a significant attenuation of growth of the two mutants in the mouse model of infection. Deletion of keto-mycolates in *M. tuberculosis* leads to restricted growth of this mutant in macrophages. Thus, the distribution and fine structure of mycolic acids determine the virulence of *M. tuberculosis*. It can be concluded that mycolic acids are very important components of the pathogenic *M. tuberculosis*.

(iii) Mycolic acid synthesis is the target of well-known anti-tuberculosis drugs, i.e., isoniazid, ethionamide, and thiocarlide (116). This suggests that all reactions on the pathway to synthesis and processing of mycolic acids are viable targets for new drug discovery (target validation).

(iv) Much progress has been made in the past 20 years or so to elucidate how mycolic acids are synthesized. However, how newly synthesized mycolic acids are processed and deposited into the cell wall needs further study. As this review shows, it is now possible to present a much more detailed pathway to the synthesis and processing of mycolic acids in *M. tuberculosis* than ever before. This review provides a clear direction for further studies. Complete elucidation of the biochemistry and genetics of this pathway would help in the program to find new and more effective drugs against multiple-drug-resistant strains of *M. tuberculosis*.

ACKNOWLEDGMENTS

This work was supported by a Merit Review grant from the Research Service of the Department of Veterans Affairs to K.T. G.S.B. is a Lister Institute-Jenner Research Fellow and is supported by the Medical Research Council (United Kingdom) and The Wellcome Trust. Laurens Anderson is acknowledged for his critical review of the manuscript. We thank Kathryn Kleckner for preparing the figures.

REFERENCES

1. Abou-Zeid, C., T. L. Ratliff, H. G. Wiker, M. Harboe, J. Beannedsen, and G. A. Rook. 1988. Characterization of fibronectin-binding antigens released by *Mycobacterium tuberculosis* and *Mycobacterium bovis* BCG. Infect. Immun. **56**:3046–3051.
2. Admiraal, S. J., C. T. Walsh, and C. Khosla. 2001. The loading module of rifamycin synthetase is an adenylation-thiolation didomain with substrate tolerance for substituted benzoates. Biochemistry **40**:6116–6123.
3. Anderson, D. H., G. Harth, M. A. Horwitz, and D. Eisenberg. 2001. An interfacial mechanism and a class of inhibitors inferred from two crystal

structures of the *Mycobacterium tuberculosis* 30 kDa major secretory protein (antigen 85B), a mycolyl transferase. J. Mol. Biol. 307:671–681.

4. Armitige, L. Y., C. Jagannath, A. R. Wanger, and S. J. Norris. 2000. Disruption of the genes encoding antigen 85A and antigen 85B of *Mycobacterium tuberculosis* H37Rv: effect on growth in culture and in macrophages. Infect. Immun. 68:767–778.

5. Asselineau, C., and J. Asselineau. 1978. Trehalose containing glycolipids. Prog. Chem. Fats Other Lipids 16:59–99.

6. Asselineau, C., H. Montrozier, and J.-C. Prome. 1969. Structure des acides α-mycoliques isoles de la souche Canetti de *Mycobacterium tuberculosis*. Bull. Soc. Chim. Fr. 1969:592–596.

7. Asselineau, C., J. Asselineau, G. Laneelle, and M. A. Laneelle. 2002. The biosynthesis of mycolic acids by mycobacteria. Current and alternative hypotheses. Prog. Lipid Res. 41:501–523.

8. Asselineau, J., and E. Lederer. 1950. Structure of the mycolic acids of mycobacteria. Nature 166:782–783.

9. Azuma, I., and Y. Yamamoto. 1963. Studies on the firmly bound lipids of human tubercle bacillus. II. Isolation of arabinose mycolate and identification of its chemical structure. J. Biochem. (Tokyo) 53:274–281.

10. Bannerjee, A., E. Dubnau, A. Quemard, V. Balasubramanian, K. S. Ulm, T. Wilson, D. Collins, G. deLisle, and W. R. Jacobs, Jr. 1994. *inhA*, a gene encoding a target for isoniazid and ethionamide in *Mycobacterium tuberculosis*. Science 263:227–230.

11. Barry, C. E., III, R. E. Lee, K. Mdluli, A. E. Sampson, B. G. Schroeder, R. A. Slayden, and Y. Yuan. 1998. Mycolic acids: structure, biosynthesis and physiological functions. Prog. Lipid Res. 37:143–179.

12. Bateman, A., E. Birney, R. Durbin, S. R. Eddy, K. L. Howe, and E. L. L. Sonnhammer. 2000. The Pfam protein families database. Nucleic Acids Res. 28:263–266.

13. Belisle, J. T., V. D. Vissa, T. Sievert, K. Takayama, P. J. Brennan, and G. S. Besra. 1997. Role of the major antigen of *Mycobacterium tuberculosis* in cell wall biogenesis. Science 276:1420–1422.

14. Besra, G. S., T. Sievert, R. E. Lee, R. A. Slayden, P. J. Brennan, and K. Takayama. 1994. Identification of the apparent carrier in mycolic acid synthesis. Proc. Natl. Acad. Sci. USA 91:12735–12739.

15. Birkeland, N. K. 1994. Cloning, molecular characterization, and expression of the genes encoding the lytic functions of lactococcal bacteriophage phi LC3: a dual lysis system of modular design. Can. J. Microbiol. 40:658–665.

16. Bloch, K. 1977. Control mechanism for fatty acid synthesis in *Mycobacterium smegmatis*. Adv. Enzymol. 45:1–84.

17. Braibant, M., P. Gilot, and J. Content. 2000. The ATP binding cassette (ABC) transport systems of *Mycobacterium tuberculosis*. FEMS Microbiol. Rev. 24:449–467.

18. Brenner, S. 1988. The molecular evolution of genes and proteins: a tale of two serines. Nature 334:528–530.

19. Centers for Disease Control and Prevention. 2004. TB elimination: now is the time! Centers for Disease Control and Prevention, Atlanta, Ga.

20. Choi, K. H., L. Kremer, G. S. Besra, and C. O. Rock. 2000. Identification and substrate specificity of β-ketoacyl (acyl carrier protein) synthase III (mtFabH) from *Mycobacterium tuberculosis*. J. Biol. Chem. 275:28201–28207.

21. Cohen-Gonsaud, M., S. Ducasse, F. Hoh, D. Zerbib, G. Labesse, and A. Quemard. 2002. Crystal structure of MabA from *Mycobacterium tuberculosis*, a reductase involved in long-chain fatty acid biosynthesis. J. Mol. Biol. 320:249–261.

22. Cole, S. T., K. Eiglmeier, J. Parkhill, et al. 2001. Massive gene decay in the leprosy bacillus. Nature 409:1007–1011.

23. Cole, S. T., R. Brosch, J. Parkhill, et al. 1998. Deciphering the biology of *Mycobacterium tuberculosis* from the complete genome sequence. Nature 393:537–544.

24. Collet, J. F., V. Stroobant, M. Pirard, G. Delpierre, and E. van Schaftingen. 1998. A new class of phosphotransferases phosphorylated on an aspartate residue in an amino-terminal DXDX(T/V) motif. J. Biol. Chem. 274:33985–33990.

25. Cronan, J. E., Jr., W. B. Li, R. Coleman, M. Narasimhan, D. de Mendoza, and J. M. Schwab. 1988. Derived amino acid sequence and identification of active site residues of *Escherichia coli* β-hydroxydecanoyl thioester dehydrase. J. Biol. Chem. 263:4641–4646.

26. Datta, A. K., and K. Takayama. 1993. Biosynthesis of a novel 3-oxo-2-tetradecyloctadecanoate-containing phospholipid by a cell-free extract of *Corynebacterium diphtheriae*. Biochim. Biophys. Acta 1169:135–145.

27. Davidson, L. A., and K. Takayama. 1979. Isoniazid inhibition of the synthesis of monounsaturated long-chain fatty acids in *Mycobacterium tuberculosis* H37Ra. Antimicrob. Agents Chemother. 16:104–105.

28. Dessen, A., A. Quemard, J. S. Blanchard, W. R. Jacobs, and J. C. Sacchettini. 1995. Crystal structure and function of the isoniazid target of in *Mycobacterium tuberculosis*. Science 267:1638–1641.

29. Dinadayala, P., F. Laval, C. Raynaud, A. Lemassu, M. A. Laneelle, G. Laneelle, and M. Daffe. 2003. Tracking the putative biosynthetic precursors of oxygenated mycolates of *Mycobacterium tuberculosis*. Structural analysis of fatty acids of a mutant strain deviod of methoxy- and ketomycolates. J. Biol. Chem. 278:7310–7319.

30. Dover, L. B., A. M. Cerdeno-Tarraga, M. J. Pallen, J. Parkhill, and G. S.

Besra. 2004. Comparative cell wall core biosynthesis in mycolated pathogens, *Mycobacterium tuberculosis* and *Corynebacterium diphtheriae*. FEMS Microbiol. Rev. 28:225–250.

31. Dubnau, E., J. Chan, C. Raynaud, V. P. Mohan, M.-A. Laneelle, K. Yu, A. Quemard, I. Smith, and M. Daffe. 2000. Oxygenated mycolic acids are necessary for virulence of *Mycobacterium tuberculosis* in mice. Mol. Microbiol. 36:630–637.

32. Dubnau, E., M.-A. Laneelle, S. Soares, A. Benichou, T. Vaz, D. Prome, J.-C. Prome, M. Daffe, and A. Quemard. 1997. *Mycobacterium bovis* BCG genes involved in the biosynthesis of cyclopropyl keto- and hydroxy-mycolic acids. Mol. Microbiol. 23:313–322.

33. Edavana, V. K., I. Pastuszak, J. D. Carroll, P. Thampi, E. C. Abraham, and A. D. Elbein. 2004. Cloning and expression of the trehalose-phosphate phosphatase of *Mycobacterium tuberculosis*: comparison to the enzyme from *Mycobacterium smegmatis*. Arch. Biochem. Biophys. 426:250–257.

34. Etemadi, A. H., and E. Lederer. 1965. Biosyntheses de l'acide α-mycolique. Biochim. Biophys. Acta 98:160–167.

35. Etemadi, A. H., and E. Lederer. 1965. Sur la structure des acides α-mycoliques de la souche humaine Test de *Mycobacterium tuberculosis*. Bull. Soc. Chim. Fr. 1965:2640–2645.

35a. Gande, R., K. J. Gibson, A. K. Brown, K. Krumbach, L. G. Dover, H. Sahm, S. Shioyama, T. Oikawa, G. S. Besra, and L. Eggeling. 2004. Acyl-CoA carboxylases (accD2 and accD3), together with a unique polyketide synthase (Cg-pks), are key to mycolic acid biosynthesis in Corynebacterianeae such as *Corynebacterium glutamicum* and *Mycobacterium tuberculosis*. J. Biol. Chem. 279:44847–44857.

36. Gao, L. Y., F. Laval, E. H. Lawson, R. K. Groger, A. Woodruff, J. H. Morisaki, J. S. Cox, M. Daffe, and E. J. Brown. 2003. Requirement for *kasB* in *Mycobacterium* mycolic acid biosynthesis, cell wall impermeability and intracellular survival: implications for therapy. Mol. Microbiol. 49:1547–1563.

37. Gastambide-Odier, M., and E. Lederer. 1959. Biosynthesis of coronymycolic acid from two molecules of palmitic acid. Nature 184:1563–1564.

38. Gensler, W. J., and J. P. Marshall. 1977. Structure of mycobacterial cis-cyclopropane mycolates by mass spectrometry. Chem. Phys. Lipids 19:128–143.

39. George, K. M., Y. Yuan, D. R. Sherman, C. E. Barry III. 1995. The biosynthesis of cyclopropanated mycolic acids in *Mycobacterium tuberculosis*. Identification and functional analysis of CMAS-2. J. Biol. Chem. 270:27292–27298.

40. Glickman, M. S. 2003. The *mmaA2* gene of *Mycobacterium tuberculosis* encodes the distal cyclopropane synthase of the α-mycolic acid. J. Biol. Chem. 278:7844–7849.

41. Glickman, M. S., J. S. Cox, and W. R. Jacobs, Jr. 2000. A novel mycolic acid cyclopropane synthetase is required for coding persistence, and virulence of *Mycobacterium tuberculosis*. Mol. Cell 5:717–727.

42. Glickman, M. S., S. M. Cahill, and W. R. Jacobs, Jr. 2001. The *Mycobacterium tuberculosis cmaA2* gene encodes a mycolic acid trans-cyclopropane synthetase. J. Biol. Chem. 276:2228–2233.

43. Greenawalt, J. W., and T. L. Whiteside. 1975. Mesosomes: membraneous bacterial organelles. Bacteriol. Rev. 30:405–463.

44. Heath, R. J., and C. O. Rock. 1996. Roles of the FabA and FabZ β-hydroxyacyl-acyl carrier protein dehydratases in *Escherichia coli* fatty acid biosynthesis. J. Biol. Chem. 271:27795–27801.

45. Jackson, M., C. Raynaud, M. A. Laneelle, C. Guilhot, C. Laurent-Winter, D. Ensergueix, B. Gicquel, and M. Daffe. 1999. Inactivation of the antigen 85C gene profoundly affects the mycolate content and alters the permeability of the *Mycobacterium tuberculosis* cell envelope. Mol. Microbiol. 31:1573–1587.

46. Joris, B., S. Englebert, C. P. Chu, R. Kariyama, L. Daneo-Moore, G. D. Shockman, and J. M. Ghuysen. 1992. Modular design of the *Enterococcus hirae* muramidase-2 and *Streptococcus faecalis* autolysin. FEMS Microbiol. Lett. 70:257–264.

47. Kilburn, J. O., K. Takayama, and E. L. Armstrong. 1982. Synthesis of trehalose dimycolate (cord factor) by a cell-free system of *Mycobacterium smegmatis*. Biochem. Biophys. Res. Commun. 108:132–139.

48. Kim, B. S., T. A. Cropp, B. J. Beck, D. H. Sherman, and K. A. Reynolds. 2002. Biochemical evidence for an editing role of thioesterase II in the biosynthesis of the polyketide pikromycin. J. Biol. Chem. 277:48028–48034.

49. Kolattukudy, P. E., N. D. Fernandes, A. K. Azad, N. M. Fitzmaurice, and T. D. Sirakova. 1997. Biochemistry and molecular genetics of cell-wall lipid biosynthesis in mycobacteria. Mol. Microbiol. 24:236–270.

50. Kremer, L., A. R. Baulard, and G. S. Besra. 2000. Genetics of mycolic acid biosynthesis, p. 173–190. In G. F. Hatfull and W. R. Jacobs, Jr. (ed.), Molecular genetics of mycobacteria. ASM Press, Washington, D.C.

51. Kremer, L., K. M. Nampoothiri, S. Lesjean, L. G. Dover, S. Graham, J. Betts, P. J. Brennan, D. E. Minnikin, C. Locht, and G. S. Besra. 2001. Biochemical characterization of acyl carrier protein (AcpM) and malonyl-CoA:AcpM transacylase (mtFabD), two major components of *Mycobacterium tuberculosis* fatty acid synthetase II. J. Biol. Chem. 276:27967–27974.

52. Kremer, L., L. G. Dover, S. Carrere, K. M. Nampoothiri, S. Lesjean, A. K. Brown, P. J. Brennan, D. E. Minnikin, C. Locht, and G. S. Besra. 2002.

Mycolic acid biosynthesis and enzymic characterization of the β-ketoacyl-ACP synthase A-condensing enzyme from *Mycobacterium tuberculosis*. Biochem. J. 364:423–430.

53. Kremer, L., W. N. Maughan, R. A. Wilson, L. G. Dover, and G. S. Besra. 2002. The *M. tuberculosis* antigen 85 complex and mycolyltransferase activity. Lett. Appl. Microbiol. 34:233–237.

54. Lederer, E. 1965. C-methylation in biological systems. Israel J. Med. Sci. 1: 1129–1147.

55. Lederer, E. 1984. Chemistry of mycobacterial cord factor and related natural and synthetic trehalose esters, p. 361–378. In G. P. Kubica and L. G. Wayne (ed.), The mycobacteria, a sourcebook, part A. Marcel Dekker, Inc., New York, N.Y.

56. Lederer, E., A. Adam, R. Ciorbaru, J.-F. Petit, and J. Wietzerbin. 1975. Cell walls of mycobacteria and related organisms: chemistry and immunological stimulant properties. Mol. Cell Biochem. 7:87–104.

57. Linne, U., D. Schwarzer, G. N. Schroeder, and M. A. Marahiel. 2004. Mutational analysis of a type II thioesterase associated with nonribosomal peptide synthesis. Eur. J. Biochem. 271:1536–1545.

58. Marmiesse, M., P. Brodin, C. Buchrieser, C. Gutierrez, N. Simoes, V. Vincent, P. Glaser, S. T. Cole, and R. Brosch. 2004. Macro-array and bioinformatic analyses reveal mycobacterial ‘core’ genes, variation in the ESAT-6 gene family and new phylogenetic markers for the *Mycobacterium tuberculosis* complex. Microbiology 150:483–496.

59. Marrakchi, H., G. Laneelle, and A. Quemard. 2000. InhA, a target of the antituberculous drug isoniazid, is involved in a mycobacterial fatty acid elongation system, FAS-II. Microbiology 146:289–296.

60. Marrakchi, H., K. H. Choi, and C. O. Rock. 2002. A new mechanism for anaerobic unsaturated fatty acid formation in *Streptococcus pneumoniae*. J. Biol. Chem. 277:44809–44816.

61. Marrakchi, H., S. Ducasse, G. Labesse, H. Montrozier, E. Margeat, L. Emorine, X. Charpentier, M. Daffe, and A. Quemard. 2002. MabA (FabG1), a *Mycobacterium tuberculosis* protein involved in the long-chain fatty acid elongation system FAS-II. Microbiology 148:951–960.

62. McCarthy, C. 1971. Utilization of palmitic acid by *Mycobacterium avium*. Infect. Immun. 4:199–204.

63. McNeil, M., M. Daffe, and P. J. Brennan. 1991. Location of the mycolyl ester substituent in the cell walls of mycobacteria. J. Biol. Chem. 266: 13217–13223.

64. Mdluli, K., R. A. Slayden, Y. Zhu, S. Ramaswamy, X. Pan, D. Mead, D. D. Crane, J. M. Musser, and C. E. Barry III. 1998. Inhibition of a *Mycobacterium tuberculosis* β-ketoacyl ACP synthase by isoniazid. Science 280:1607–1610.

65. Minnikin, D. E. 1982. Lipids: complex lipids, their chemistry, biosynthesis and roles, p. 95–184. In C. Ratledge and J. Stanford (ed.), The biology of the mycobacteria, vol. 1. Physiology, identification and classification. Academic Press, New York, N.Y.

66. Minnikin, D. E., and N. Polgar. 1967. Structural studies on the mycolic acids. Chem. Commun. 1967:312–314.

67. Minnikin, D. E., and N. Polgar. 1967. The methoxymycolic and ketomycolic acids from human tubercle bacilli. Chem. Commun. 1967:1172–1174.

68. Minnikin, D. E., and N. Polgar. 1967. The mycolic acids from the human and avium tubercle bacilli. Chem. Commun. 1967:916–918.

69. Misaki, A., N. Seto, and I. Azuma. 1974. Structure and immunological properties of d-arabino-d-galactan isolated from *Mycobacterium* species. J. Biochem. (Tokyo) 76:15–27.

70. Modis, Y., S. A. Filppula, D. K. Novikov, B. Norledge, J. K. Hiltunen, and R. K. Wierenga. 1998. The crystal structure of dienoyl-CoA isomerase at 1.5 Å resolution reveals the importance of aspartate and glutamate side chains for catalysis. Structure 6:957–970.

71. Mohan, S., T. M. Kelly, S. S. Eveland, C. R. Raetz, and M. S. Anderson. 1994. An *Escherichia coli* gene (FabZ) encoding (3R)-hydroxymyrystoyl acyl carrier protein dehydrase. Relation to FabA and suppression of mutations in lipid A biosynthesis. J. Biol. Chem. 269:32896–32903.

72. Noll, H., and H. Bloch. 1955. Studies on the chemistry of the cord factor of *Mycobacterium tuberculosis*. J. Biol. Chem. 224:149–163.

73. Noll, H., J. Asselineau, and E. Lederer. 1956. The chemical structure of the cord factor of *Mycobacterium tuberculosis*. Biochim. Biophys. Acta 20:299–309.

74. Onwueme, K. C., J. A. Ferreras, J. Buglino, C. D. Lima, and L. E. N. Quadri. 2004. Mycobacterial polyketide-associated proteins are acyltransferases: proof of principle with *Mycobacterium tuberculosis* PapA5. Proc. Natl. Acad. Sci. USA 101:4608–4613.

75. Phetsuksiri, B., M. Jackson, H. Scherman, M. McNeil, G. S. Besra, A. R. Baulard, R. A. Slayden, A. E. DeBarber, C. E. Barry III, M. S. Baird, D. C. Crick, and P. J. Brennan. 2003. Unique mechanism of action of the thiourea drug isoxyl on *Mycobacterium tuberculosis*. J. Biol. Chem. 278:53123–53130.

76. Portevin, D., C. de Sousa-D'Auria, C. Houssin, C. Grimaldi, M. Chami, M. Daffe, and C. Guilhot. 2004. A polyketide synthase catalyzes the last condensation step of mycolic acid biosynthesis in mycobacteria and related organisms. Proc. Natl. Acad. Sci. USA 101:314–319.

77. Puech, V., C. Guilhot, E. Perez, M. Tropis, L. Y. Armitige, B. Glcquel, and

M. Daffe. 2002. Evidence for a partial redundancy of the fibronectin-binding proteins for transfer of mycolyl residues onto the cell wall arabinogalactan termini of *Mycobacterium tuberculosis*. Mol. Microbiol. 44:1109–1122.

78. Puech, V., N. Bayan, K. Salim, G. Leblon, and M. Daffe. 2000. Characterization of the in vivo acceptors of the mycolyl residues transferred by the corynebacterial PS1 and the related mycobacterial antigen 85. Mol. Microbiol. 35:1026–1041.

79. Quemard, A., J. C. Sacchettini, A. Dessen, C. Vilcheze, R. Bittman, W. R. Jacobs, and J. S. Blanchard. 1995. Enzymatic characterization of the target for isoniazid in *Mycobacterium tuberculosis*. Biochemistry 34:8235–8241.

80. Qureshi, N., K. Takayama, and H. K. Schnoes. 1980. Purification of C<sub>30</sub>–C<sub>56</sub> fatty acids from *Mycobacterium tuberculosis* H37Ra. J. Biol. Chem. 255:182–189.

81. Qureshi, N., K. Takayama, and H. K. Schnoes. 1981. Separation of a homologous series of long-chain fatty alcohols (C<sub>49</sub>-C<sub>58</sub>) from *Mycobacterium tuberculosis* H37Ra by high performance liquid chromatography. J. Liquid Chromatog. 4:1207–1218.

82. Qureshi, N., K. Takayama, H. C. Jordi, and H. K. Schnoes. 1978. Characterization of the purified components of a homologous series of α-mycolic acids from *Mycobacterium tuberculosis* H37Ra. J. Biol. Chem. 253:5411–5417.

83. Qureshi, N., N. Sathyamoorthy, and K. Takayama. 1984. Biosynthesis of C<sub>30</sub> to C<sub>56</sub> fatty acids by an extract of *Mycobacterium tuberculosis* H37Ra. J. Bacteriol. 157:46–52.

84. Reuter, G., T. Janvilisri, H. Venter, S. Shahi, L. Balakrishnan, and H. W. van Veen. 2003. The ATP binding cassette multidrug transporter LmrA and lipid transporter MsbA have overlapping substrate specificities. J. Biol. Chem. 278:35193–35198.

85. Rock, C. O., and J. E. Cronan, Jr. 1982. Regulation of bacterial membrane synthesis. Curr. Top. Membr. Transp. 17:209–233.

86. Ronning, D. R., T. Klabunde, G. S. Besra, V. D. Vissa, J. T. Belisle, and J. C. Sacchettini. 2000. Crystal structure of the secreted form of antigen 85C reveals potential targets for mycobacterial drugs and vaccines. Nat. Struct. Biol. 7:141–146.

87. Sassetti, C. M., D. H. Boyd, and E. J. Rubin. 2002. Genes required for mycobacterial growth defined by high-density mutagenesis. Mol. Microbiol. 48:77–84.

88. Sassetti, C. M., and E. J. Rubin. 2003. Genetic requirements for mycobacterial survival during infection. Proc. Natl. Acad. Sci. USA 100:12989–12994.

89. Sathyamoorthy, N., and K. Takayama. 1987. Purification and characterization of a novel mycolic acid exchange enzyme from *Mycobacterium smegmatis*. J. Biol. Chem. 262:13417–13423.

90. Scardale, J. N., G. Kazanova, X. He, K. A. Reynolds, and H. T. Wright. 2001. Crystal structure of the *Mycobacterium tuberculosis* β-ketoacyl-acyl carrier protein synthase III. J. Biol. Chem. 276:20516–20522.

91. Schaeffer, M. L., G. Agnihotri, C. Volker, H. Kallender, P. J. Brennan, and J. T. Lonsdale. 2001. Purification and biochemical characterization of the *Mycobacterium tuberculosis* β-ketoacyl-acyl carrier protein synthases KasA and KasB. J. Biol. Chem. 276:47029–47037.

92. Schwarzer, D., H. D. Mootz, U. Linne, and M. A. Marahiel. 2002. Regeneration of misprimed nonribosomal peptide synthetases by type II thioesterases. Proc. Natl. Acad. Sci. USA 99:14083–14088.

93. Shimakata, T., and Y. Minatogawa. 2000. Essential role of trehalose in the synthesis and subsequent metabolism of coronymonic acid in *Corynebacterium matruchotii*. Arch. Biochem. Biophys. 380:331–338.

94. Slayden, R. A., and C. E. Barry III. 2002. The role of KasA and KasB in the biosynthesis of meromycolic acids and isoniazid resistance in *Mycobacterium tuberculosis*. Tuberculosis 82:149–160.

95. Slengut, J. D., and R. R. Levine. 2000. MDP-1, a novel eukaryotic magnesium-dependent phosphatase. Biochemistry 39:8315–8324.

96. Smith, S., A. Witkowski, and A. K. Joshi. 2003. Structural and functional organization of the animal fatty acid synthase. Prog. Lipid Res. 42:289–317.

97. Stauton, J., and K. J. Weissman. 2001. Polyketide biosynthesis: a millennium review. Nat. Prod. Rep. 18:380–416.

98. Steen, A., G. Buist, K. J. Leenhouts, M. El Khattabi, F. Grijpstra, A. L. Zomer, G. Venema, O. P. Kuipers, and J. Kok. 2003. Cell wall attachment of a widely distributed peptidoglycan binding domain is hindered by cell wall constituents. J. Biol. Chem. 278:23874–23881.

99. Strong, M., T. G. Graeber, M. Beeby, M. Pellegrini, M. J. Thompson, T. O. Yeates, and D. Eisenberg. 2003. Visualization and interpretation of protein networks in *Mycobacterium tuberculosis* based on hierarchical clustering of genome-wide functional linkage maps. Nucleic Acids Res. 31:7099–7109.

100. Takayama, K., and N. Qureshi. 1984. Structure and synthesis of lipids, p. 315–344. In G. P. Kubica and L. G. Wayne (ed.), The mycobacteria, a sourcebook, part A. Marcel Dekker, Inc., New York, N.Y.

101. Takayama, K., H. K. Schnoes, E. L. Armstrong, and R. W. Boyle. 1975. Site of inhibitory action of isoniazid in the synthesis of mycolic acid in *Mycobacterium tuberculosis*. J. Lipid Res. 16:308–317.

102. Takayama, K., N. Qureshi, and H. K. Schnoes. 1978. Isolation and characterization of the monounsaturated long chain fatty acids of *Mycobacterium tuberculosis*. Lipids 13:575–579.

103. Takayama, K., N. Qureshi, H. C. Jordi, and H. K. Schnoes. 1979. Purification of mono and diunsaturated $C_{22}$-$C_{47}$ fatty acids from *Mycobacterium tuberculosis* H37Ra as their $p$-bromophenacyl esters by high performance liquid chromatography, p. 375–394. *In* G. L. Hawk (ed.), Biological/biomedical applications of liquid chromatography II. Chromatographic science series, vol. 12. Marcel Dekker, Inc., New York, N.Y.

104. Takayama, K., N. Qureshi, H. C. Jordi, and H. K. Schnoes. 1979. Recycling HPLC of $p$-bromophenacyl esters of saturated $C_{35-56}$ fatty acids from *Mycobacterium tuberculosis* H37Ra on a silica column. J. Liquid Chromatogr. 2:861–873.

105. Thaller, M. C., S. Schippa, and G. M. Rossolini. 1998. Conserved sequence motifs among bacterial, eukaryotic, and archaeal phosphatases that define a new phosphohydrolase superfamily. Protein Sci. 7:1647–1652.

106. Thomson, J. D., D. G. Higgins, and T. J. Gibson. 1997. CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice. Nucleic Acids Res. 22:4673–4680.

107. Trivedi, O. A., P. Arora, V. Sridharan, R. Tickoo, D. Mohanty, and R. S. Gokhale. 2004. Enzymatic activation and transfer of fatty acids as acyladenylates in mycobacteria. Nature 428:441–445.

108. Tzvetkov, M., C. Klopprogge, O. Zelder, and W. Liebl. 2003. Genetic dissection of trehalose biosynthesis in *Corynebacterium glutamicum*: inactivation of trehalose production leads to impaired growth and an altered cell wall lipid composition. Microbiology 149:1659–1673.

109. van Helvoort, A., A. J. Smith, H. Sprong, I. Fritzche, A. H. Schinkel, P. Borst, and G. van Meer. 1996. MDR1 P-glycoprotein is a lipid translocase of broad specificity, while MDR3 P-glycoprotein specifically translocates phosphatidylcholine. Cell 87:507–517.

110. van Helvoort, A., M. L. Giudici, M. Thielemans, and G. van Meer. 1997. Transport of sphingomyelin to the cell surface is inhibited by brefeldin A and in mitosis, where $C_6$-NBD-sphingomyelin is translocated across the plasma membrane by a multidrug transporter activity. J. Cell Sci. 110:75–83.

111. Vilchez, C., H. R. Moribidoni, T. R. Weisbrod, H. Iwamoto, M. Kuo, J. C. Sacchettini, and W. R. Jacobs, Jr. 2000. Inactivation of the *inhA*-encoded fatty acid synthase II (FASII) enoyl-acyl carrier protein reductase induces accumulation of the FASI end products and cell lysis of *Mycobacterium smegmatis*. J. Bacteriol. 182:4059–4067.

112. Walker, R. W., J. C. Prome, and C. S. LaCave. 1973. Biosynthesis of mycolic acids. Formation of a $C_{32}$ $\beta$-keto ester from palmitic acid in a cell-free system of *Corynebacterium diphtheriae*. Biochim. Biophys. Acta 326:52–62.

113. Watanabe, M., Y. Aoyagi, H. Mitome, T. Fujita, H. Naoki, M. Ridell, and D. E. Minnikin. 2002. Location of functional groups in mycobacterial meromycolate chains; the recognition of new structural principles in mycolic acids. Microbiology 148:1881–1902.

114. Weir, M. P., W. H. R. Langridge, and R. W. Walker. 1972. Relationship between oleic acid uptake and lipid metabolism in *Mycobacterium smegmatis*. Am. Rev. Respir. Dis. 106:450–457.

115. Wilson, R. A., W. N. Maughan, L. Kremer, G. S. Besra, and K. Futterer. 2004. The structure of *Mycobacterium tuberculosis* MPT51 (FbpC1) defines a new family of non-catalytic $\alpha/\beta$ hydrolases. J. Mol. Biol. 335:519–530.

116. Winder, F. G. 1982. Mode of action of the antimycobacterial agents and associated aspects of the molecular biology of the mycobacteria, p. 353–438. *In* C. Ratledge and J. Stanford (ed.), The biology of the mycobacteria, vol. 1. Physiology, identification and classification. Academic Press, New York, N.Y.

117. Wolf, A., R. Kramer, and S. Morbach. 2003. Three pathways for trehalose metabolism in *Corynebacterium glutamicum* ATCC13032 and their significance in response to osmotic stress. Mol. Microbiol. 49:1119–1134.

118. Wong, H. C., G. Liu, Y. M. Zhang, C. O. Rock, C. O., and J. Zheng. 2002. The solution structure of acyl carrier protein from *Mycobacterium tuberculosis*. J. Biol. Chem. 277:15874–15880.

119. Woodruff, P. J., B. L. Carlson, B. Siridechadilok, M. R. Pratt, S. J. Williams, and C. R. Bertozzi. 2004. Trehalose is required for growth of *Mycobacterium smegmatis*. J. Biol. Chem. 279:28835–28843.

120. Yuan, Y., and C. E. Barry III. 1996. A common mechanism for the biosynthesis of methoxy and cyclopropyl mycolic acids in *Mycobacterium tuberculosis*. Proc. Natl. Acad. Sci. USA 93:12828–12833.

121. Yuan, Y., D. Mead, B. G. Schroeder, Y. Zhu, and C. E. Barry III. 1998. The biosynthesis of mycolic acids in *Mycobacterium tuberculosis*. Enzymatic methyl(ene) transfer to acyl carrier protein bound meromycolic acid *in vitro*. J. Biol. Chem. 273:21282–21290.

122. Yuan, Y., D. C. Crane, J. M. Musser, S. Sreevatsan, and C. E. Barry, III. 1997. MMAS-1 the branch point between cis- and trans-cyclopropane containing oxygenated mycolates in *Mycobacterium tuberculosis*. J. Biol. Chem. 272:10041–10048.

123. Yuan, Y., R. E. Lee, G. S. Besra, J. T. Belisle, and C. E. Barry III. 1995. Identification of a gene involved in the biosynthesis of cyclopropanated mycolic acids in *Mycobacterium tuberculosis*. Proc. Natl. Acad. Sci. USA 92:6630–6634.

124. Yuan, Y., Y. Zhu, D. D. Crane, and C. E. Barry III. 1998. The effect of oxygenated mycolic acid composition on cell wall function and macrophage growth in *Mycobacterium tuberculosis*. Mol. Microbiol. 29:1449–1458.

125. Zhang, Y. M., H. Marrakchi, S. W. White, and C. O. Rock. 2003. The application of computational methods to explore the diversity and structure of bacterial fatty acid synthase. J. Lipid Res. 44:1–10.
