
THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2005 by The American Society for Biochemistry and Molecular Biology, Inc.

**A Formyltransferase Required for Polymyxin Resistance in Escherichia coli and the Modification of Lipid A with 4-Amino-4-deoxy-L-arabinose**

IDENTIFICATION AND FUNCTION OF UDP-4-DEOXY-4-FORMAMIDO-L-ARABINOSE*§

Received for publication, December 20, 2004, and in revised form, January 18, 2005  
Published, JBC Papers in Press, January 28, 2005, DOI 10.1074/jbc.M414265200

Steven D. Breazeale‡, Anthony A. Ribeiro‡§, Amanda L. McClerren‡, and Christian R. H. Raetz‡¶

From the ‡Department of Biochemistry, Duke University Medical Center, Durham, North Carolina 27710  
and §Duke University NMR Spectroscopy Center and Department of Radiology, Duke University Medical Center, Durham, North Carolina 27710

Modification of the phosphate groups of lipid A with 4-amino-4-deoxy-L-arabinose (L-Ara4N) is required for resistance to polymyxin and cationic antimicrobial peptides in *Escherichia coli* and *Salmonella typhimurium*. We previously demonstrated that the enzyme ArnA catalyzes the NAD⁺-dependent oxidative decarboxylation of UDP-glucuronic acid to yield the UDP-4″-ketopentose, uridine 5′-diphospho-β-(L-threo-pentapyranosyl-4″-ulose), which is converted by ArnB to UDP-β-(L-Ara4N). *E. coli* ArnA is a bi-functional enzyme with a molecular mass of ~74 kDa. The oxidative decarboxylation of UDP-glucuronic acid is catalyzed by the 345-residue C-terminal domain of ArnA. The latter shows sequence similarity to enzymes that oxidize the C-4″ position of sugar nucleotides, like UDP-galactose epimerase, dTDP-glucose-4,6-dehydratase, and UDP-xylose synthase. We now show that the 304-residue N-terminal domain catalyzes the N-10-formyltetrahydrofolate-dependent formylation of the 4″-amine of UDP-L-Ara4N, generating the novel sugar nucleotide, uridine 5′-diphospho-β-(4-deoxy-4-formamido-L-arabinose). The N-terminal domain is highly homologous to methionyl-tRNAfMet formyltransferase. The structure of the formylated sugar nucleotide generated *in vitro* by ArnA was validated by ¹H and ¹³C NMR spectroscopy. The two domains of ArnA were expressed independently as active proteins in *E. coli*. Both were required for maintenance of polymyxin resistance and L-Ara4N modification of lipid A. We conclude that N-formylation of UDP-L-Ara4N is an obligatory step in the biosynthesis of L-Ara4N-modified lipid A in polymyxin-resistant mutants. We further demonstrate that only the formylated sugar nucleotide is converted *in vitro* to an undecaprenyl phosphate-linked form by the enzyme ArnC. Because the L-Ara4N unit attached to lipid A is not derivatized with a formyl group, we postulate the existence of a deformylase, acting later in the pathway.

---

Lipoplysaccharide (LPS)¹ is an immunogenic glycolipid that constitutes most of the outer leaflet of the outer membrane of Gram-negative bacteria (1–4). LPS consists of three domains, which are the O-antigen, the core oligosaccharide, and the lipid A moiety (1–4). The O-antigen functions as a protective barrier, whereas the core sugars maintain outer membrane integrity and provide an attachment site for the O-antigen (1–4). Lipid A is the hydrophobic membrane anchor of LPS, and it is the active (endotoxin) component of LPS, accounting for many of the pathophysiological effects associated with Gram-negative sepsis (5–7).

The Kdo₂-lipid A portion of LPS is sufficient to support growth in *Escherichia coli* and *Salmonella typhimurium* (2). Covalent modifications to Kdo₂-lipid A can be induced by environmental stimuli, such as low Mg²⁺ concentrations or low pH (8–10). As shown in Fig. 1 for *S. typhimurium*, these modifications include the incorporation of palmitate (11, 12), the addition of phosphoethanolamine (pEtN) (13–15), and/or the addition of 4-amino-4-deoxy-L-arabinose (L-Ara4N) moieties (13, 16, 17). The modification of at least one phosphate residue with L-Ara4N is required for maintaining resistance to certain cationic antimicrobial peptides of the innate immune system and to the antibiotic polymyxin (18, 19). Resistance is due, in part, to the neutralization of the negative charges of lipid A by L-Ara4N, reducing the affinity of lipid A for cationic substances (20) and preventing these anti-microbial compounds from penetrating the outer membrane.

The addition of pEtN and L-Ara4N groups to lipid A is controlled by the PmrA/PmrB two-component regulatory system, which is activated by low pH, high Fe³⁺ levels, or indirectly, by low concentrations of Mg²⁺ via the PhoP/PhoQ system through the action of PmrD (9). Activated PmrA stimulates transcription at the *pmrCAB*, *pmrE(ugd)*, and *pmrHFIJKLM* loci (19, 21). Constitutive *pmrA* (*pmrA*⁰) mutants of *E. coli* and *S. typhimurium* are polymyxin-resistant, and they modify their lipid A with L-Ara4N and pEtN groups under all growth conditions (17, 18, 22). Inactivation of either *pmrE* or the genes in the *pmrHFIJKLM* operon results in complete loss of polymyxin resistance and of L-Ara4N-modified lipid A in *pmrA*⁰ bacterial cells (19, 21, 23). Similarly, *pmrA*⁰ mutants harboring a nonpolar disruption of the *pmrC(eptA)* gene are unable

---

* This research was supported by National Institutes of Health Grant GM-51310 (to C. R. H. R.). The Duke University NMR Center is partially supported by P30-CA-14236. NMR instrumentation in the Center was funded by the National Science Foundation, the National Institutes of Health, the North Carolina Biotechnology Center, and Duke University. The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
* § The on-line version of this article (available at http://www.jbc.org) contains supplemental Fig. 1.
* ¶ To whom correspondence should be addressed: Dept. of Biochemistry, Duke University Medical Center, P. O. Box 3711, Durham, North Carolina 27710. Tel.: 919-684-5326; Fax: 919-684-8885; E-mail: raetz@biochem.duke.edu.

---

¹ The abbreviations used are: LPS, lipopolysaccharide; Bis-Tris, bis-(2-hydroxyethyl)-imino-tris-(hydroxymethyl)-methane; L-Ara4N, 4-amino-4-deoxy-L-arabinose; pEtN, phosphoethanolamine; GlcUA, glucuronic acid; UDP-L-Ara4FN, uridine 5′-diphospho-β-(4-deoxy-4-formamido-L-arabinose); UDP-4″-ketopentose, uridine 5′-diphospho-β-(L-threo-pentapyranosyl-4″-ulose); TLC, thin layer chromatography; HMQC, multiple quantum coherence; HMBC, heteronuclear multiple bond coherence; DTT, dithiothreitol.

N-Formylation of UDP-L-Ara4N by the Bifunctional Enzyme ArnA 14155

FIG. 1. Regulated covalent modifications of lipid A. The Kdo₂-lipid A substructure of LPS is sufficient to support growth of *S.* typhimurium and *E.* coli (2). As indicated by the dashed bonds, some modifications of the phosphate groups and/or acyl chains of lipid A may occur (8, 10, 11, 13, 24, 61, 62). Panel A, structure of lipid A under conditions that suppress the PmrA/B and PhoP/Q systems. The 2-hydroxymyristate (magenta) is present only in Salmonella (62). Panel B, lipid A modifications seen under conditions of PmrA/B and PhoP/Q activation. The diphosphate is suppressed, but the lipid A phosphates are modified with L-Ara4N and/or pEtN groups (blue), both of which are regulated by PmrA. The latter is activated by low pH (10). It is constitutively active in polymyxin-resistant mutants (18, 22). In some instances lipid A variants (not shown) are made in which the locations of the L-Ara4N and pEtN groups are reversed or in which both phosphates are modified with the same group (13, 61). The outer membrane enzyme PagP can add an additional palmitate unit at position 2 (12), whereas PagL (63) can remove the hydroxymyristoyl chain at position 3 (red acyl chains). These pag genes are regulated by the PhoP/PhoQ system, which is activated by low Mg²⁺ in Salmonella (64, 65). PagL is present only in Salmonella but can also function when overexpressed in *E.* coli (63). The 2-hydroxymyristate is synthesized irrespective of PmrA/B or PhoP/Q activation (10).

to incorporate pEtN groups into their lipid A and are slightly more sensitive to polymyxin under some growth conditions (14, 15).

Sequence analysis of the proteins encoded by the genes of the *pmr* loci led us to propose the pathway depicted in Fig. 2 for the biosynthesis of the L-Ara4N moiety and its attachment to lipid A (24–26). Our hypothesis is supported by the following observations. 1) ArnT (PmrK) is an inner membrane enzyme that transfers the L-Ara4N moiety of the glycolipid undecaprenyl phosphate-α-L-Ara4N to lipid A (16). 2) ArnA(PmrI) catalyzes the oxidation and decarboxylation of UDP-glucuronic acid (UDP-GlcUA) to yield a molecule of uridine 5′-diphospho-β-(L-threo-pentapyranosyl-4″-ulose), also designated UDP-4″-ketopentose (or UDP-Ara4O) (25). 3) ArnB(PmrH) is an amino-transferase that converts the UDP-4″-ketopentose to UDP-L-Ara4N, with L-glutamate functioning as the amine donor (26). 4) Cell extracts of polymyxin-resistant *E.* coli mutants, but not wild type, convert UDP-L-Ara4N to a putative formamido derivative in the presence of N-10-formyltetrahydrofolate (25).

We now report that purified ArnA not only oxidizes UDP-GlcUA (25) but also catalyzes the transfer of the formyl group from N-10-formyltetrahydrofolate to the 4″-amine of UDP-L-Ara4N, yielding the novel sugar nucleotide uridine 5′-diphospho-β-(4-deoxy-4-formamido-L-arabinose) (UDP-L-Ara4FN). The analog N-5-formyltetrahydrofolate is not a substrate. In aqueous solution, purified UDP-L-Ara4FN is recovered as a 4:1 mixture of the cis-and trans-formamido rotational isomers, as judged by one- and two-dimensional NMR spectroscopy. Furthermore, ArnA can be split into two functional proteins, a formyltransferase and a dehydrogenase, derived from the N and C termini of the full-length protein, respectively. Both domains are required for incorporation of the L-Ara4N moiety into lipid A in vivo and for the maintenance of polymyxin resistance. The formylated sugar nucleotide is needed because of the selectivity of the subsequent enzyme (ArnC), which does not utilize UDP-L-Ara4N (Fig. 2). The unanticipated requirement of the ArnA formyltransferase reaction for the maintenance of polymyxin resistance further suggests that an as yet unidentified deformylase must also exist, since neither L-Ara4N-modified lipid A (13, 24) nor the L-Ara4N donor, undecaprenyl phosphate-α-L-Ara4N (17), is formylated when isolated from polymyxin resistant cells. The availability of mg quantities of UDP-L-Ara4FN, which can be prepared efficiently from UDP-GlcUA with full-length ArnA and ArnB

14156 N-Formylation of UDP-L-Ara4N by the Bifunctional Enzyme ArnA

Periplasm

P + 

L-Ara4N Modified
Transport?
Lipid A

Undecaprenyl
Phosphate-α-L-Ara4N

ArnT

+

Undecaprenyl
Phosphate-α-L-Ara4FN

Transport?

Kdo₂-Lipid A

Other Core
Sugars

Inner
Membrane

Undecaprenyl
Phosphate

ArnC

ArnD

UDP

Formate

Cytoplasm

NH₂
UDP-L-Ara4N
HO
OH
UDP

N-10 formyl-
THF
ArnA (N-terminus)
O
H
N
H
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO
HO

N-Formylation of UDP-L-Ara4N by the Bifunctional Enzyme ArnA

Construction of pET-ArnAFT and pET-ArnADH Expressing the Separate Domains of ArnA—The predicted coding region of the ArnA formyltransferase domain was amplified by PCR from pET-ArnA with the primers (5′-cgggatcCATATGaaaaccgtcgtttttgcctac-3′, ArnAFT5) and (5′-cgggatcCTCGAGtcacaagcgtgaaccttgccaccagg-3′, ArnAFT3) containing engineered NdeI and XhoI restriction sites (in capital letters), respectively. The Opti-Prime™ PCR kit (Stratagene) was used. Amplification in a 50-μl reaction with 10× Opti-Prime Buffer 7 (100 mM Tris-HCl, pH 8.8, 35 mM MgCl₂, 250 mM KCl), 0.2 mM dNTP, 20 ng pET-ArnA DNA, 0.5 μM primers, and 2.5 units of Pfu DNA polymerase for 30 cycles (94 °C for 1 min, 58 °C for 1 min, and 72 °C for 1.5 min) followed by 1 cycle of 94 °C for 1 min, 58 °C for 1 min, and 72 °C for 10 min resulted in a single product of the predicted size. The product was isolated with the QIAquick PCR purification kit (Qiagen), digested with XhoI and NdeI, and ligated into the T7 expression vector, pET24b (Novagen), previously digested with the same enzymes. The product was confirmed by DNA sequencing.

The dehydrogenase domain of ArnA was cloned into pET24b using the same strategy, except that the primers used were 5′-cgggatcCATATGcgggtactcatcctcggggtg-3′ (ArnADH5) and 5′-cgggatcCTCGAGtcattggtttatccgtaagatc-3′ (ArnADH3), which contained engineered NdeI and XhoI restriction sites (shown in capital letters), respectively. The product was confirmed by DNA sequencing of pET-ArnADH.

Expression of the ArnA Formyltransferase and ArnA Dehydrogenase Domains—The formyltransferase (pET-ArnAFT) and dehydrogenase domains (pET-ArnADH) of ArnA were expressed in *E. coli* NovaB-Blue(DE3) (Novagen), as described above for the ArnA-H6N protein, except that a 2.5-h isopropyl-1-thio-β-D-galactopyranoside induction at 30 °C was used.

Construction of pWSK-ArnA, pWSK-ArnAFT, and pWSK-ArnADH—The inserts in the expression vectors pET-ArnA, pET-ArnAFT, or pET-ArnADH were excised with XbaI and XhoI and cloned into the same restriction sites of the relatively low copy vector pWSK29 (32) to generate pWSK-ArnA, pWSK-ArnAFT, and pWSK-ArnADH.

Construction of pSDB300 and pSDB350—To make pSDB300 and pSDB350, we first constructed pSDB100 and pSDB150 (Supplemental Fig. 1) by cloning the ArnADH- and ArnAFT-encoding genes from pET24b (see Table I) into pET28b, respectively. To generate pSDB100, pET-ArnADH was linearized with XbaI, and a two-base fill-in was performed with the Klenow fragment of DNA polymerase I in the presence of dCTP and dTTP. The DNA encoding the ArnADH was excised with XhoI and cloned into pET28b, previously prepared with a HindIII digestion, followed by a fill-in reaction with dATP and dGTP using Klenow fragment, and then cut with XhoI (Supplemental Fig. 1). Repeating this procedure with the ArnAFT encoding DNA from pET-ArnAFT (see Table I) gave pSDB150. pSDB200 was constructed by excising the ArnAFT encoding DNA from pET-ArnAFT with XbaI and XhoI and then ligating this fragment into the XbaI and SalI sites of pSDB100 (Supplemental Fig. 1). Similarly, pSDB250 was made by first excising the ArnADH-encoding DNA from pET-ArnADH with XbaI and XhoI and then ligating this into the XbaI and SalI sites of pSDB150 (Supplemental Fig. 1). Ligating the XbaI-XhoI fragment of pSDB200 into these same restriction sites of pWSK29 resulted in the construction of pSDB300, which contains the *arnAFT* followed by *arnADH*, each with a ribosomal binding site derived from pET28b just upstream of the cloned genes. Similarly, pSDB350 was constructed by ligating the XbaI/XhoI fragment of pSDB250 into these same restriction sites of pWSK29 (Supplemental Fig. 1). pSDB350 contains the *arnADH* followed by *arnAFT*, each with a ribosomal binding site derived from pET28b just upstream of the cloned genes.

Construction of pET-ArnC—The *arnC* gene was amplified by PCR from *E. coli* W3110 genomic DNA using the primers arnC5 (5′-cggatcCatggttgaatcccctgttaagaaagtc-3′) and arnC3 (5′-cgggatcCTCGAGttattcattttccttgctggatggacg-3′), containing engineered NcoI and XhoI restriction sites (capital letters). The Opti-Prime™ PCR kit (Stratagene) was used. Amplification in a 50-μl reaction with 10× Opti-Prime Buffer 6 (100 mM Tris-HCl, pH 8.8, 15 mM MgCl₂, 750 mM KCl), 0.2 mM dNTP, 200 ng of genomic DNA, 0.5 μM primers, and 2.5 units of Pfu DNA polymerase for 30 cycles (94 °C for 1 min, 50 °C for 1 min, and 72 °C for 1.5 min) followed by 1 cycle of 94 °C for 1 min, 50 °C for 1 min, and 72 °C for 10 min resulted in a single product of the predicted size. The product was isolated with the QIAquick PCR purification kit (Qiagen), digested with XhoI and NdeI, and ligated into the T7 expression vector, pET28b (Novagen), previously digested with the same enzymes. The product was confirmed by DNA sequencing.

Enzymatic Synthesis and Isolation of Uridine 5′-Diphospho-β-(4-amino-4-deoxy-L-arabinose)—A 5-ml reaction containing 0.1 mg/ml of an *E. coli* NovaBlue(DE3)/pET-ArnA cell-free extract overexpressing ArnA, 0.05 mg/ml purified ArnBC6H, 1 mM UDP-GlcUA, 3 mM NAD⁺, and 25 mM L-glutamate in 50 mM HEPES, pH 7.5, supplemented with 2 mM DTT was incubated at 30 °C for 4 h. Protein was removed by passing the reaction mixture through a Centricon® YM-10 membrane. The product was diluted 10-fold with deionized water and loaded onto a 10-ml column of DEAE-Sepharose™ CL6B pre-equilibrated with 10 mM Bis-Tris, pH 6.0. The column was washed with 50 ml of water followed by elution with a 0–100 mM Bis-Tris gradient, pH 6.0 (200 ml total volume). Elution was monitored by measuring A₂₅₄. Fractions (20 ml) containing the product (eluting at ~30 mM Bis-Tris) were pooled, diluted 10-fold with water, and loaded onto a 10-ml column of DEAE-Sepharose™ CL6B equilibrated with 10 mM triethylammonium bicarbonate, pH 8.5. The column was washed with 50 ml of water followed by elution with a 0–100 mM triethylammonium bicarbonate gradient, pH 8.5 (200 ml total volume). Chromatography was monitored by measuring A₂₅₄. Fractions (25 ml) containing the product (eluting at ~60 mM triethylammonium bicarbonate) were pooled and concentrated by lyophilization to give the triethylammonium salt of UDP-L-Ara4N (1.6 mg, 51% yield).

Enzymatic Synthesis and Purification of Uridine 5′-Diphospho-β-(4-amino-4-deoxy-L-[U-¹⁴C]arabinose) and Its Formylated Derivative—Enzymatic conversion of UDP-[U-¹⁴C]GlcUA to UDP-L-[U-¹⁴C]Ara4N was carried out in 400 μl containing 50 mM HEPES, pH 7.5, 2 mM DTT, 3 mM NAD, 1 mM L-glutamate, 0.05 mg/ml ArnA overexpressing cell extract (25), 0.05 mg/ml pure ArnB-CH6 (26), and 2 μCi of UDP-[U-¹⁴C]GlcUA. After 220 min at 30 °C, the reaction mixture was diluted to 1.5 ml with water and loaded onto a 2-ml column of DEAE-Sepharose™ CL-6B equilibrated in 10 mM Bis-Tris, pH 6.5. The column was washed with 5 ml of water followed by elution with a 0–100 mM Bis-Tris, pH 6.5, gradient (40 ml volume). Fractions containing UDP-L-[U-¹⁴C]Ara4N were pooled, diluted 10-fold with water, and loaded onto a second 2-ml column of DEAE-Sepharose™ CL-6B pre-equilibrated with 10 mM triethylammonium bicarbonate, pH 8.5. This column was washed with 5 ml of water followed by a gradient of 0–100 mM triethylammonium bicarbonate, pH 8.5 (40 ml volume). The eluted material was concentrated using a Speed-Vac centrifuge and dissolved in 250 μl of water. The yield was 74% based on radioactivity.

UDP-L-[U-¹⁴C]Ara4FN was generated as follows. A 250-μl reaction containing 24 μM UDP-L-[U-¹⁴C]Ara4N (22700 cpm/μl), 1 mM N-5,N-10-methenyltetrahydrofolate, 50 mM HEPES, pH 7.5, and 2 mM DTT was pre-incubated for 30 min at 30 °C before purified ArnA-N6H was added to a final concentration of 0.05 mg/ml. The reaction was allowed to proceed for 180 min at 30 °C then diluted to 3 ml with water and loaded onto a 2-ml column of DEAE-Sepharose™ CL-6B equilibrated in 10 mM Bis-Tris, pH 6.5. The column was washed with 5 ml of water followed by elution with a 0–100 mM Bis-Tris, pH 6.5, gradient (40 ml volume). Fractions containing UDP-L-[U-¹⁴C]Ara4N were pooled, diluted 10-fold with water, and loaded onto a second 2-ml column of DEAE-Sepharose™ CL-6B pre-equilibrated with 10 mM triethylammonium bicarbonate, pH 8.5. This column was washed with 5 ml of water followed by a gradient of 0–100 mM triethylammonium bicarbonate, pH 8.5 (40 ml volume). The eluted material was concentrated using a Speed-Vac centrifuge and dissolved in 200 μl of water. The yield was 84% based on radioactivity.

Enzymatic Synthesis and Isolation of Uridine 5′-Diphospho-β-(4-deoxy-4-formamido-L-arabinose)—The reaction mixture for the conversion of UDP-L-Ara4N to UDP-L-Ara4FN contained 0.05 mg/ml pure ArnAN6H, 200 μM UDP-L-Ara4N, 1 mM N-5,N-10-methenyltetrahydrofolate, 2 mM DTT, and 50 mM HEPES, pH 7.5, in 15 ml. After 6 h at 30 °C, enzyme was removed by passing the mixture through a Centricon® YM-10 membrane. The sugar nucleotide was loaded onto a 10-ml column of DEAE-Sepharose™ CL6B equilibrated with 10 mM Bis-Tris, pH 5.9. The column was washed with 50 ml of water followed by elution of the product with a 0–100 mM Bis-Tris, pH 5.9, gradient (200 ml). Chromatography was monitored by absorbance at 254 nm. Fractions containing the product, which eluted at about 60 mM Bis-Tris, were pooled, diluted 10-fold with water, and loaded onto a second 10-ml column of DEAE-Sepharose™ CL6B equilibrated with 10 mM triethylammonium bicarbonate, pH 8.5. The column was washed with 50 ml of water. The product was eluted with a 0–100 mM gradient of triethylammonium bicarbonate, pH 8.5 (200 ml). Fractions containing the product, which eluted at about 60 mM triethylammonium bicarbonate, were pooled and concentrated by lyophilization to give the triethylammonium salt of the sugar nucleotide. Conversion to the sodium salt was accomplished by passing the material over a small column of AG@50W-X8 resin (sodium form). The overall yield was 55%.

Nuclear Magnetic Resonance Spectroscopy—The ArnA product UDP-

TABLE I  
Bacterial strains and plasmids  

| Strains and plasmids | Description | Source or reference |
|---------------------|-------------|---------------------|
| $E. coli$           | W3110       | Wild type, F⁻, λ⁻  | $E. coli$ Genetic Stock Center, Yale |
|                     | WD101       | W3110, $pmrA^{c}$, Pmʳ | 17 |
|                     | DY330       | W3110, ΔlacU169gal490λcl857Δ(cro-bioA) | 34 |
|                     | MST100      | DY330, $pmrA^{c}$, Pmʳ | This work |
|                     | SDB100      | DY330, $pmrA^{c}$, $arnA::kan$, Kanʳ | This work |
|                     | SDB101      | W3110, $pmrA^{c}$, $arnA::kan$, Kanʳ | This work |
|                     | NovaBlue(DE3) | Δ(srl-recA)30::Tn10(DE3), Tetʳ | Novagen |
| Plasmids            | pET24b      | Vector containing a T7lac promoter, Kanʳ | Novagen |
|                     | pET28b      | Vector containing a T7lac promoter, Kanʳ | Novagen |
|                     | pET-ArnA    | pET24b expressing $E. coli$ ArnA | 25 |
|                     | pWSK29      | Low copy number vector, Ampʳ | 32 |
|                     | pET-ArnAN6H | pET28b expressing N-terminal His₆ fusion of $E. coli$ ArnA | This work |
|                     | pET-ArnADH  | pET24b expressing ArnA dehydrogenase domain (DH) | This work |
|                     | pET-ArnAFT  | pET24b expressing ArnA formyltransferase domain (FT) | This work |
|                     | pET-ArnC    | pET28b expressing $E. coli$ ArnC | This work |
|                     | pWSK-ArnA   | pWSK29 expressing $E. coli$ ArnA | This work |
|                     | pWSK-ArnADH | pWSK29 expressing $E. coli$ ArnADH | This work |
|                     | pWSK-ArnAFT | pWSK29 expressing $E. coli$ ArnAFT | This work |
|                     | pSDB100     | pET28b expressing ArnADH | This work |
|                     | pSDB150     | pET28b expressing ArnAFT | This work |
|                     | pSDB200     | pET24b expressing the ArnAFT and the ArnADH | This work |
|                     | pSDB250     | pET24b expressing the ArnADH and the ArnAFT | This work |
|                     | pSDB300     | pWSK29 expressing the ArnAFT and the ArnADH | This work |
|                     | pSDB350     | pWSK29 expressing the ArnADH and the ArnAFT | This work |

L-Ara4FN (Na⁺ salt, 1.0 mg) and the substrate UDP-L-Ara4N (Na⁺ salt, 1.7 mg) were each dissolved in 0.6 ml of 99% D₂O in 5 mm NMR tubes. NMR studies of the product were initially performed at pH 8.3, as measured with a 3-mm pH electrode, and compared with the substrate at pH 8.0. The product was then lyophilized and re-analyzed by ¹H NMR in 90% H₂O, 10% D₂O at pH 5.0 to detect the exchangeable NH resonances. ¹H and ¹³C NMR chemical shifts were referenced to 2,2-dimethylsilapentane-5-sulfonic acid at 0.00 ppm.

¹H NMR spectra were recorded on Varian Inova 600 and 800 spectrometers, each equipped with a Sun Ultra 10 computer and a 5-mm Varian triple resonance probe. ¹H spectra at 600 MHz were obtained with a spectral width of 6.5 kHz, 67° pulse flip angle (6 μs), 4.9-s acquisition time, and 2-s relaxation delay. Spectra were digitized using 64,000 points to obtain a digital resolution of 0.25 Hz/point. ¹H spectra at 800 MHz were obtained with 8.0-kHz spectral width, 76° pulse flip angle (7 μs), 4.0-s acquisition time, and 1.5-s relaxation delay and digitized using 96,000 points to obtain a digital resolution of 0.16 Hz/point. Two-dimensional NMR experiments were performed as previously described (25, 26, 33).

ArnA Formyltransferase Assays—Unless indicated, enzyme assays contained 50 mM HEPES, pH 7.5, 100 μM UDP-L-[U-¹⁴C]Ara4N (26000 cpm), 1 mM formyl donor, and 0.05 mg/ml purified ArnAN6H in a total volume of 20 μl, as indicated. With N-5,N-10-methenyltetrahydrofolate as the precursor, a 30-min preincubation at 30°C was necessary for the *in situ* conversion to the actual formyl donor, N-10-formyltetrahydrofolate (25, 27). Reactions started by the addition of enzyme were incubated at 30°C. At the desired time points, 1.5-μl samples were withdrawn and spotted onto a polyethyleneimine-cellulose TLC plate. The spots were allowed to dry, and the plate was washed with methanol for 5 min. After drying, the plate was developed in aqueous 0.25 M acetic acid containing 0.4 M LiCl. The plate was dried and exposed to a PhosphorImager screen. Radioactive products were identified and quantified with a PhosphorImager (ImageQuant software, Molecular Dynamics).

ArnA Formyltransferase and ArnA Dehydrogenase Assays in Cell Extracts—Assays for ArnA formyltransferase at 30°C were carried out with cell-free extracts of $E. coli$ WD101 (see Table I) or SBD101 (0.5–0.05 mg/ml) in a 20-μl reaction mixture containing 50 mM HEPES, pH 7.5, 2 mM DTT, 5 mM L-glutamate, 1 mM N-10-formyltetrahydrofolate generated *in situ*, and 20 μM UDP-L-[U-¹⁴C]Ara4N (66000 cpm). At various times 1-μl samples were withdrawn and spotted onto a polyethyleneimine-cellulose TLC plate, which was developed and analyzed as described above.

Assays for ArnA dehydrogenase at 30°C were performed with cell-free extract (0.5 mg/ml) in a 20-μl reaction mixture containing 50 mM HEPES, pH 7.5, 2 mM DTT, 5 mM α-ketoglutarate, 3 mM NAD⁺, and 20 μM UDP-[U-¹⁴C]GlcUA (0.02 μCi). At the desired times, 1-μl samples were withdrawn and spotted onto a polyethyleneimine-cellulose TLC

In Vitro ArnC Assay—The reaction was allowed to proceed for 30 min at 30°C with 0.5 mg/ml membranes of ArnC overexpressing $E. coli$ NovaBlue(DE3) cells. Other components of the assay system include 10 μM ¹⁴C-labeled sugar nucleotides (21,000 cpm/tube), 50 μM undecaprenyl phosphate (diammonium salt), purchased from the Polish Academy of Sciences, Institute of Biochemistry and Biophysics, 0.1% Triton X-100, 2 mM MnCl₂, and 50 mM Hepes, pH 7.5. The reaction was stopped by spotting 3.5 μl onto a silica gel plate, which was developed in the solvent CHCl₃/MeOH/dH₂O/NH₄OH (65/25/3.6/0.4) and analyzed with a PhosphorImager.

Construction of a $pmrA^{c}$ Mutant of $E. coli$ K-12 with an Insertion in the Chromosomal Copy of $arnA$—Using a $pmrA^{c}$ derivative of $E. coli$ strain DY-330, designated MST100 (see Table I), we used λ-RED-mediated recombination (34) to generate a nonpolar insertion of the kanamycin resistance gene ($kan$, aminoglycoside 3′-phosphotransferase) in the $arnA$ gene. To construct MST100, $pmrA^{c}$ was moved into DY-330 using a P1vir lysate prepared on $E. coli$ WD101 and selecting for polymyxin resistance at 10 μg/ml on LB agar. A linear DNA fragment in which the $kan$ gene is flanked by 47 bp located directly upstream of $arnA$ (underlined in AKOKR5) and a ribosomal binding site with an NdeI site (doubly underlined in AKOKR3) site followed by 48 bp located downstream of $arnA$ (underlined in AKOKR3) was amplified using the oligonucleotides AKOKR5 (5′-TATTTTGTTTCAGCAAGTTATCCGTCC-ATCCAGCAAAGGAAAAATGAATAATGAGCCATATTCAACGGGA-3′), AKOKR3 (5′-ACGGGTGCCACGAAAAGGTATCGACATCAATGCGTAA-GCCTACTTTGGTCATATGTATATCTCCTTCTTAGAAAAACTCATC-GAGCAT-3′), and the kanamycin resistance gene (Tn903) from plasmid pWSK130 as template. The nucleotides in the above primers corresponding to $kan$ are not underlined. The amplification reaction (50 μl) contained 5 μl of 10× PCR buffer from Sigma (100 mM Tris-HCl, pH 8.3, 500 mM KCl, 15 mM MgCl₂, and 0.01% gelatin), 0.4 mM dNTPs, 0.5 μM primers, 50 ng of the $kan$ gene, and 5 units of Platinum® Taq polymerase (Invitrogen). After 2 cycles of 94°C for 1 min, 40°C for 1 min, and 72°C for 1.5 min followed by 25 cycles of 94°C for 1 min, 60°C for 1 min, and 72°C for 1.5 min and 1 cycle of 94°C for 1 min, 60°C for 1 min, and 72°C for 10 min, a single product of the predicted size was obtained. This product was resolved on a 1.5% agarose gel and purified with the QIAquick Gel Extraction kit (Qiagen). The purified PCR product (100 ng) was introduced into MST100 by electroporation after induction of the RED system by incubation of mid-log-phase cells for 15 min at 42°C (34). After growth for 2 h at 30°C, the cells were plated on LB agar containing kanamycin (20 μg/ml) and grown overnight at 30°C. The desired strain, SDB100, was polymyxin-sensitive and kanamycin-resistant, and it only grew at 30°C. The entire chromosomal region containing the $kan$ insertion in $arnA$ gene was then transduced

back into *E. coli* WD101 (PmrA<sup>c</sup>) with a P1<sub>vir</sub> lysate prepared on SDB100 (Table I). The resulting mutant was designated SDB101 (*pmrA<sup>c</sup>* Δ*arnA::kan<sup>r</sup>*); it was polymyxin-sensitive and kanamycin-resistant and grew at 37 °C. The *arnA::kan<sup>r</sup>* replacement on the chromosome of SDB101 was verified by PCR amplification of the *arnA* region using external primers located 108 bp upstream (p1015, 5′-TATTGGCGCTCAGTTATCGGC-3′) and 58 bp downstream (p1013, 5′-ACTCAAGATTTCCAGCAGACGC-3′). The PCR product was resolved on a 1.5% agarose gel, purified with the QIAquick Gel Extraction kit (Qiagen), and sequenced using the same primers to confirm the *arnA::kan<sup>r</sup>* replacement and the insertion of the engineered ribosomal binding site.

**RESULTS**

*Overexpression and Purification of ArnA as a Hexahistidine Fusion Protein—*The full-length ArnA protein is predicted to contain two discrete catalytic domains, an N-terminal formyltransferase and a C-terminal dehydrogenase. The latter presumably catalyzes the oxidative decarboxylation of UDP-GlcUA to yield the UDP-4″-ketopentose, which is converted to UDP-L-Ara4N by ArnB in the presence of L-glutamate (Fig. 2). Cell-free extracts derived from polymyxin-resistant *E. coli* are capable of further modifying UDP-L-Ara4N, but not the UDP-4″-ketopentose, in the presence of *N*-10-formyltetrahydrofolate, suggesting that the 4″-amino of the L-Ara4N moiety is formylated (25). To demonstrate that ArnA is responsible for the proposed formylation (Fig. 2), we cloned the full-length *arnA* gene into the expression vector pET28b behind the T7 RNA polymerase promoter to generate an N-terminal hexahistidine fusion protein, designated ArnAN6H. The N-terminal sequence of this construct was MGSSHHHHHHSSGLVPRGSH, which is in accord with the expected N-terminal fusion sequence derived from the expression vector. The protein was overexpressed in *E. coli* NovaBlue(DE3), a polymyxin-sensitive strain that does not modify its endogenous lipid A with L-Ara4N (16, 17). A protein of the predicted molecular mass of ArnA (73 kDa) was observed by SDS-gel electrophoresis in extracts of isopropyl-1-thio-β-D-galactopyranoside-induced cells (not shown). ArnA was purified by nickel-chelation chromatography. Imidazole was removed with a PD-10 gel filtration column in the presence of 300 mM NaCl to prevent protein precipitation.

Purified ArnAN6H catalyzed efficient conversion of UDP-L-Ara4N to a slower migrating substance in the presence of *N*-10-formyltetrahydrofolate (Fig. 3A, lanes 2 and 3). *N*-5-Formyltetrahydrofolate (Fig. 3A, lane 4) was inactive, consistent with the reaction reported previously in crude extracts of polymyxin-resistant (*pmrA<sup>c</sup>*) *E. coli* (25). The specific activity of the purified ArnA formyltransferase, calculated from the time course (Fig. 3B), was 100-fold higher than that of extracts of polymyxin-resistant cells and 10-fold higher than extracts of the overproducing strain. The formyltransferase was absent in a *pmrA<sup>c</sup>* mutant of *E. coli* in which the *arnA* gene was replaced with a kanamycin resistance cassette (see below). The results show that *arnA* is the structural gene for both the formyltransferase and the UDP-GlcUA dehydrogenase.

*One-dimensional <sup>1</sup>H NMR Spectroscopy of the Formylated Product—*Purified ArnAN6H was used to convert 3 μmol of UDP-L-Ara4N to the putative UDP-L-Ara4FN in the presence of 15 μmol of *N*-10-formyltetrahydrofolate, as described under “Experimental Procedures.” The sodium salt of the product (1.0 mg) was dissolved in D<sub>2</sub>O (99%). Its one-dimensional <sup>1</sup>H NMR spectrum was recorded at pD 8.3 and 5.0 and compared with that of the UDP-L-Ara4N substrate at pD 8.0. Figs. 4 and 5 show the expanded downfield and bulk sugar regions of the spectra, respectively. No additional signals other than that of HDO (not shown) are observed. At pD 8.3, the uracil and ribose moieties of the substrate and product possess virtually identical chemical shifts and coupling constants (Figs. 4 and 5, Tables II and III). However, the signals arising from the pyranose moiety of the putative UDP-L-Ara4FN (Fig. 2) show significant differences from those of the substrate UDP-L-Ara4N, especially at H-4″ (Fig. 5, panels A versus B and Table II).

In the downfield portion of the <sup>1</sup>H NMR spectrum, sharp singlets are present at 8.17 and 8.06 ppm in the formylated product (Fig. 4, *B versus A*). These resonances are characteristic of protons of formyl moieties (HCO) and could represent the *cis* and *trans* rotational isomers of a formamido group (the *cis* isomer is shown in Fig. 2). Integration of the singlets indicates that there might be a 4:1 ratio of the rotamers. Other proton resonances were also doubled in the product, notably the uridine H-6 and H-5 (Fig. 4*B*) and all pyranose protons (Fig. 5*B*, Tables I and II). In 90% H<sub>2</sub>O, 10% D<sub>2</sub>O at pD 5.0, the exchangeable formamide NH signals are seen at 8.47 (major) and 8.09 (minor) ppm, respectively (Fig. 4*C*). The *J*<sub>HCO,NH</sub> coupling constant (1.6 Hz) for the amide proton of the major isomer and the intramolecular cross-peak between these two protons in a nuclear Overhauser effect experiment (not shown) confirmed that the major isomer is the *cis*-formamido rotamer (Fig. 2).

*Two-dimensional <sup>1</sup>H NMR Spectroscopy of the Formyltransferase Product—*The two-dimensional <sup>1</sup>H-<sup>1</sup>H COSY (pD 8.3 in D<sub>2</sub>O) of the putative UDP-L-Ara4FN (Fig. 6) allowed definitive assignment of the protons in the ribose, uracil, and pyranose rings. As suggested by the one-dimensional spectra, the uridine nucleoside moiety was not altered by ArnA (Table II). Close similarities between the spectra of the putative UDP-L-Ara4FN and UDP-L-Ara4N include the strong cross-peak connectivities between uracil H-6 and H-5 (not shown), ribose H-1′ and H-2′ (~4.38 ppm), ribose H-3′ and H-4′ (4.30 ppm), and ribose H-4′ to H-5a′ and H-5b′ (Table II and Fig. 6).

The assignments of the pyranose protons of the product are as follows. The anomeric H-1″ proton, a double-doublet at 5.63 ppm (*J*<sub>1″,2″</sub> = 3.6, *J*<sub>1″,P</sub> = 7.1 Hz), shows a cross-peak to the H-2″ double-triplet at 3.70 ppm *J*<sub>1″,2″</sub> = 3.6, *J*<sub>2″,3″</sub> = 10.1 Hz). A second cross-peak from H-2″ correlates to H-3″ at 4.10 ppm (dd, *J*<sub>2″,3″</sub> = 10.1, *J*<sub>3″,4″</sub> = 4.5 Hz). H-3″ then connects to H-4″ at 4.41 ppm (unresolved multiplet). Further tracing of the two remaining cross-peaks from H-4″ locates the H-5b″ (4.21 ppm; dd, *J*<sub>4″,5″b</sub> = 2.1, *J*<sub>5b″,5a″</sub> = 12.4 Hz) and H-5a″ (3.66 ppm; dd, *J*<sub>4″,5″a</sub> = 2.1, *J*<sub>5a″,5b″</sub> = 12.4 Hz) signals (Table II and Fig. 6).

The <sup>1</sup>H-<sup>1</sup>H COSY at pD 5.0 in 90% H<sub>2</sub>O, 10% D<sub>2</sub>O (not shown) revealed additional cross-peaks from the major *cis*-formamido NH (8.47 ppm) to the HCO signal (8.16 ppm) (Fig. 4) and to H-4″ (4.41 ppm), thus confirming the attachment of the formamido moiety to the arabinose at C-4″. The additional coupling from the NH also accounts for the difference in the appearance of the *cis*-formamido (major) and *trans*-formamido (minor) H-4″ resonances in Fig. 5, panels B and C, at pD 8.3 versus 5. Resolved resonances attributed to the minor *trans*-formamido rotamer are noted in Tables II and III and labeled in Figs. 4–7.

The small *J*<sub>1″,2″</sub> coupling (3.6 Hz) and the large *J*<sub>2″,3″</sub> coupling (10.1 Hz) indicate that the pyranose ring has an equatorially disposed H-1″ and axial H-2″ and H-3″ protons, which in this case would be classified as the β-anomer given the L-stereochemistry of the pyranose (35). Two-dimensional nuclear Overhauser effect spectroscopy (not shown) shows a strong cross-peak between H-1″ and H-2″, in accord with an equatorial H-1″ and an axial H-2″ (Fig. 2) (36). The large trans-diaxial *J*<sub>2″,3″</sub> coupling (10.1 Hz) then suggests that H-3″ is likewise axial. The intermediate value of the *J*<sub>3″,4″</sub> coupling (4.5 Hz) indicates an axial-equatorial coupling, showing that H-4″ is in the equatorial position and that the formamido moiety is axial (Fig. 2).

The H-5a″ (3.66 ppm) and H-5b″ (4.21 ppm) signals are assigned to the axial and equatorial positions, respectively, based on literature values for synthetic standards and because

a proton in the axial position on a given carbon atom in a pyranose ring is expected to resonate at ~0.5 ppm higher field than an equatorial proton on the same carbon (36). The observed 2.1-Hz $J_{4^{\prime \prime},5^{\prime \prime}}$ couplings (equatorial-equatorial and equatorial-axial) did not allow distinction between the axial and equatorial H-5" protons. Like the previously characterized β-anomers of sugar nucleotides in this pathway, the UDP-4"-ketopentose and UDP-L-Ara4N (Fig. 2) (25, 26), the nuclear Overhauser effect data (not shown) were not clear enough to assign the H-5" equatorial and axial protons; no nuclear Overhauser effect cross-peak was observed between the axially disposed H-3" (β-face) and either H-5" proton.

**Evaluation of the Carbon Structure of the Product by HMQC Spectroscopy—** ${}^{13}$C NMR data were obtained through ${}^{1}$H-detected HMQC and HMBC experiments. The partial two-dimensional HMQC ${}^{1}$H-${}^{13}$C correlation map (Fig. 7, panel A) reveals three direct ${}^{1}$H-${}^{13}$C single-bond correlations in the anomeric region (90–105 ppm). The H-1" signal near 5.63 ppm correlates to the carbon resonance at 98.0 ppm (C-1"). A pyranose C-1 atom that is glycosidically linked to an axial oxygen is expected to resonate in the 98–103-ppm range, whereas a C-1 atom linked to an equatorial oxygen is expected to resonate between 103 and 106 ppm (36). The observed C-1" resonance for the ArnA product is consistent with an axially disposed oxygen atom and an equatorially disposed H-1". The ribose H-1' at 6.00 ppm correlates to the anomic carbon at 91.5 ppm (C-1'). The uracil H-5 near 5.97 ppm correlates to the 105.2-ppm carbon signal, and the uracil H-6 at 7.94 ppm correlates to 144.0 ppm.

The HMQC data for the other protons of the pyranose moiety (Fig. 7, panel B) show that the H-5b" and the H-5a" multiplets correlate to a carbon signal at 64.8 ppm (C-5"), whereas the H-2" and H-3" multiplets connect to carbons at 71.0 and 70.0 ppm respectively, corresponding to oxygen-substituted carbons of sugars. Nitrogen-substituted carbons of amino-sugars resonate near 52–55 ppm. Modification of the free-amine with a cis-formyl group causes a slight upfield shift of the carbon (1.4–2.9 ppm), whereas a trans-formyl group results in a slight downfield shift (37–39). The H-4" multiplet of the major cis-formamido isomer shows a cross-peak at 51.5 ppm, which is 3.1 ppm upfield of the C-4" cross-peak in UDP-L-Ara4N. The H-4" multiplet of the minor isomer has a cross-peak at 56.4 ppm, consistent with the trans-formamido group. The remaining sugar HMQC cross-peaks arise from carbons of the ribose ring (Table III) (36, 40).

The downfield HCO protons of the cis- (8.17 ppm) and the trans-formamido (8.06 ppm) isomers (Fig. 4) correlate to the carbonyl carbons at 167.0 ppm and 171.0 ppm, respectively (Fig. 7A). These data are consistent with reported values in which the carbonyl carbon of a cis-formamido rotamer resonates 3–6 ppm upfield of the respective carbon of the trans-rotamer (39, 41–43).

The multibond HMBC two-dimensional map (not shown) allows identification of the quaternary carbons (Table III) of the product. The multibond correlations from the uracil H-6 (7.95 ppm) and H-5 (5.97) led to assignment of the 154.5- and 169.0-ppm quaternary carbon signals to C-2 and C-4, respectively. The multibond correlation from the uracil H-6 to C-1' was also detected, thus confirming the linkage of the base group to the ribose. Multibond correlations from both H-5" protons to the 51.5-ppm carbon resonance further validated C-4" as the formamido-substituted carbon.

**ArnA Primary Sequence and Domain Architecture—** To define the formyltransferase domain of ArnA, multiple ClustalW (1.82) (44) sequence alignments were performed with the *E. coli* proteins ArnA (660 amino acids) (25), N-10-formyltetrahydrofolate:L-methionyl-tRNA N-formyltransferase (315 amino acids) (45, 46), and glyciamide ribonucleotide transformylase (211 amino acids) (47). In addition, the FxbA protein (360 amino acids) of *Mycobacterium smegmatis*, which is involved in formylation of exochelin, was used (48). The N-terminal half of ArnA was most similar to *E. coli* N-10-formyltetrahydrofolate L-methionyl-tRNA N-formyltransferase (*E* value $2 \times 10^{-31}$ in a two sequence BLAST comparison) (49). It also displayed strong

FIG. 3. Assay of purified full-length ArnAN6H formyltransferase activity. Panel A, requirement of N-5,N-10-methenyltetrahydrofolate (converted *in situ* to N-10-formyltetrahydrofolate) for the ArnA-catalyzed formylation of UDP-L-[${}^{14}$C]Ara4N. Pure ArnAN6H was incubated at 30 °C with 100 μM UDP-L-[${}^{14}$C]Ara4N and either 1 mM N-5,N-10-methenyltetrahydrofolate or N-5 formyltetrahydrofolate (5-Formyl-THF). After 40 min, 1.5 μl was spotted onto a polyethyleneimine-cellulose plate, developed in 0.4 M LiCl containing 0.25 M acetic acid. Substrate and product were quantified with a PhosphorImager. Lane 1, no enzyme control; lanes 2 and 4, with 0.05 mg/ml ArnAN6H; lane 3, with 0.025 mg/ml ArnAN6H. Panel B, rate and extent of conversion of 100 μM UDP-L-[${}^{14}$C]Ara4N to UDP-L-[${}^{14}$C]Ara4FN in the presence of 1 mM N-10-formyltetrahydrofolate by ArnAN6H at 0.025 mg/ml.

N-Formylation of UDP-L-Ara4N by the Bifunctional Enzyme ArnA

A. UDP-L-Ara4N, pD 8.0

B. UDP-L-Ara4FN, pD 8.3

C. UDP-L-Ara4FN, pD 5.0

FIG. 4. Downfield region of the ${}^{1}\mathrm{H}$ NMR spectra of UDP-L-Ara4N and UDP-L-Ara4FN. ${}^{1}\mathrm{H}$ NMR spectra (25 °C at 600 MHz) show the formamido and H-6 aromatic protons (*left panel*) and the H-5 aromatic and anomeric protons (*right panel*) of UDP-L-Ara4N (pD 8.0 in $\mathrm{D}_{2} \mathrm{O}$), UDP-L-Ara4FN (pD 8.3 in $\mathrm{D}_{2} \mathrm{O}$), and UDP-L-Ara4FN (pD 5.0 in $90 \% \mathrm{H}_{2} \mathrm{O} / 10 \% \mathrm{D}_{2} \mathrm{O}$). Note the two sets of resolved resonances in $B$ and $C$ at the formyl CH and the uracil H-6 due to the *cis-* (major) and *trans-* (minor) formamido rotamers, which are present in a 4:1 ratio.

A. UDP-L-Ara4N, pD 8.0

B. UDP-L-Ara4FN, pD 8.3

C. UDP-L-Ara4FN, pD 5.0

FIG. 5. Sugar region of the ${}^{1}\mathrm{H}$ NMR spectra of UDP-L-Ara4N and UDP-L-Ara4FN. ${}^{1}\mathrm{H}$ NMR spectra (25 °C at 600 MHz) show the pyranose and ribose protons of UDP-L-Ara4N (pD 8.0 in $\mathrm{D}_{2} \mathrm{O}$), UDP-L-Ara4FN (pD 8.3 in $\mathrm{D}_{2} \mathrm{O}$), and UDP-L-Ara4FN (pD 5.0 in $90 \% \mathrm{H}_{2} \mathrm{O} / 10 \% \mathrm{D}_{2} \mathrm{O}$). Note the two sets of resolved resonances in $B$ and $C$ at pyranose H-$5^{\prime \prime}$, H-$4^{\prime \prime}$, and H-$3^{\prime \prime}$ positions due to the *cis-* (major) and *trans-* (minor) formamido rotamers.

similarity to FxbA ($E$ value $3 \times 10^{-30}$) and weak similarity to glyciamide ribonucleotide transformylase ($E$ value $7 \times 10^{-4}$) in these comparisons. Residues 106–109 (SLLP) of ArnA are strictly conserved in many enzymes that bind tetrahydrofolate, like $N$-10-formyltetrahydrofolate:L-methionyl-tRNA $N$-formyltransferase and glyciamide ribonucleotide transformylase (45, 46). $E$. coli ArnA also contains three conserved residues (Asn-102, His-104, and Asp-140) known to function in glyciamide ribonucleotide transformylase catalysis (50, 51). In the structures of glyciamide ribonucleotide transformylase and $N$-10-formyltetrahydrofolate:L-methionyl-tRNA $N$-formyltransferase, these residues adopt identical conformations. In $N$-10-formyltetrahydrofolate:L-methionyl-tRNA $N$-formyltransferase they are in close proximity to a docked 5-deaza-tetrahydrofolate molecule (45).

The $E$. coli ArnA dehydrogenase domain, located in the C-

**TABLE II**

${}^{1}$H NMR spectroscopy of UDP-L-Ara4FN versus UDP-L-Ara4N

${}^{1}$H chemical shifts at 25°C in D$_{2}$O relative to internal 2,2-dimethylsilapentane-5-sulfonic acid. ND, not determined.

| Position | $\delta$ | $J$ | Hz | $\delta$ | $J$ | Hz | $\delta$ | $J$ | Hz |
|----------|----------|-----|----|----------|-----|----|----------|-----|----|
|          | ppm      |     |    | ppm      |     |    | ppm      |     |    |
| Uridine  |          |     |    |          |     |    |          |     |    |
| H-5      | 5.99(d)  | $J_{5,6}$ | 8.1 | 5.97(d) | $J_{5,6}$ | 8.1 | 5.97(d) | $J_{5,6}$ | 8.1 |
| H-6      | 7.97(d)  |     |    | 7.95(d) |     |    | 7.94(d) |     |    |
| Ribose   |          |     |    |          |     |    |          |     |    |
| H-1'     | 6.00(d)  | $J_{1',2'}$ | 4.5 | 6.00(d) | $J_{1',2'}$ | 4.4 | 5.98(d) | $J_{1',2'}$ | 4.5 |
| H-2'     | ~4.39    |     |    | ~4.38    |     |    | ~4.37    |     |    |
| H-3'     | ~4.39    |     |    | ~4.38    |     |    | ~4.37    |     |    |
| H-4'     | 4.30     |     |    | 4.29(m)  |     |    | 4.28(m)  |     |    |
| H-5'a    | 4.26(ddd) | $^{3}J_{5'a,4'}$ | 2.6 | 4.25(ddd) | $^{3}J_{5'a,4'}$ | 2.6 | 4.23(ddd) | $^{3}J_{5'a,4'}$ | 2.6 |
|          |          | $J_{5'a,P}$ | 4.5 |          | $J_{5'a,P}$ | 4.4 |          | $J_{5'a,P}$ | 4.4 |
|          |          | $^{3}J_{5'a,5'b}$ | 11.8 |          | $^{3}J_{5'a,5'b}$ | 11.8 |          | $^{3}J_{5'a,5'b}$ | 11.8 |
| H-5'b    | 4.21     | $^{3}J_{5'b,4'}$ | 2.8 | 4.20(ddd) | $^{3}J_{5'b,4'}$ | 3.0 | 4.18(ddd) | $^{3}J_{5'b,4'}$ | 2.8 |
|          |          | $J_{5'b,P}$ | 5.6 |          | $J_{5'b,P}$ | 5.6 |          | $J_{5'b,P}$ | 5.5 |
| Pyranose |          |     |    |          |     |    |          |     |    |
| H-1"     | 5.63(dd) | $J_{1",2"}$ | 3.4 | 5.63(dd) | $J_{1",2"}$ | 3.6 | 5.62(dd) | $J_{1",2"}$ | 3.6 |
|          |          | $^{3}J_{1",P}$ | 7.3 |          | $^{3}J_{1",P}$ | 7.1 |          | $^{3}J_{1",P}$ | 7.3 |
| H-2"     | 3.77(dt) | $J_{2",3"}$ | 9.9 | 3.70(dt) | $J_{2",3"}$ | 10.1 | 3.70(dt) | $J_{2",3"}$ | 10.1 |
|          |          | $^{4}J_{2",P}$ | 3.4 |          | $^{4}J_{2",P}$ | 3.3 |          | $^{4}J_{2",P}$ | 3.0 |
| H-3"     | 4.15(dd) | $J_{3",4"}$ | 4.6 | 4.10(dd) | $J_{3",4"}$ | 4.5 | 4.09(dd) | $J_{3",4"}$ | 4.5 |
| H-4"     | 3.59(m)  |     |    | 4.41(m)  |     |    | 4.39(m)  |     |    |
| H-5"a    | 3.80(dd) | $J_{4",5"a}$ | 2.0 | 3.66(dd) | $J_{4",5"a}$ | 2.0 | 3.65(dd) | $J_{4",5"a}$ | 2.0 |
|          |          | $J_{5"a,5"b}$ | 13.3 |          | $J_{5"a,5"b}$ | 12.4 |          | $J_{5"a,5"b}$ | 12.6 |
| H-5"b    | 4.29(dd) | $J_{4",5"b}$ | 2.1 | 4.21(dd) | $J_{4",5"b}$ | 2.1 | 4.19(dd) | $J_{4",5"b}$ | 2.1 |
| NH       |          |     |    |          |     |    |          | $J_{4",NH}$ | 8.7 |
| HCO      |          |     |    |          |     |    |          | $J_{CH-NH}$ | 1.6 |

$a$ Major cis-formamido rotamer (80%); additional resolved resonances are attributed to the minor trans-formamido rotamer (20%) at 8.06 (HCO), 7.94 (H-6), 5.97 (H-5), 5.63 (H-1"), 3.73 (H-2"), 4.06 (H-3"), 3.93 (H-4"), 3.76 (H-5"a), and 4.29 ppm (H-5"b).

$b$ Major cis-formamido rotamer (80%); additional resolved resonances are attributed to the minor trans-formamido rotamer (20%) at 8.09 (NH), 5.61 (H-1"), 3.74 (H-2"), 4.05 (H-3"), 3.92 (H-4"), 3.74 (H-5"a), and 4.29 ppm (H-5"b).

---

**TABLE III**

${}^{13}$C NMR spectroscopy of UDP-L-Ara4FN versus UDP-L-Ara4N

${}^{13}$C chemical shifts at 25°C in D$_{2}$O relative to internal 2,2-dimethylsilapentane-5-sulfonic acid.

| Position | $\delta C^{a}$ | $\delta C^{a,b}$ | $\delta C^{a,c}$ |
|----------|----------------|------------------|------------------|
|          | ppm            | ppm              | ppm              |
| Uridine  |                |                  |                  |
| C-2      | 154.4          | 154.5            | 154.5            |
| C-4      | 168.8          | 169.0            | 169.0            |
| C-5      | 105.3          | 105.2            | 105.4            |
| C-6      | 144.2          | 144.0            | 144.0            |
| Ribose   |                |                  |                  |
| C-1'     | 90.9           | 91.5             | 91.3             |
| C-2'     | 75.6           | 76.4             | 76.6             |
| C-3'     | 71.5           | 72.4             | 72.5             |
| C-4'     | 85.6           | 86.0             | 86.2             |
| C-5'     | 66.8           | 67.5             | 67.8             |
| Pyranose |                |                  |                  |
| C-1"     | 98.2           | 98.0             | 98.6             |
| C-2"     | 70.1           | 71.0             | 73.0             |
| C-3"     | 68.5           | 70.0             | 70.8             |
| C-4"     | 54.6           | 51.5             | 52.0             |
| C-5"     | 62.4           | 64.8             | 65.6             |
| HCO      |                | 167.0            | 166.5            |

$a$ Shift data estimated from HMQC two-dimensional map at coarse digitization.

$b$ Major cis-formamido rotamer (80%); additional resolved ${}^{13}$C resonances are attributed to minor trans-formamido rotamer (20%) at 70.8 (C-2"), 56.4 (C-4"), 65.0 (C-5"), and 171.0 ppm (HCO).

$c$ Major cis-formamido rotamer (80%); additional resolved ${}^{13}$C resonances are attributed to minor trans-formamido rotamer (20%) at 71.0 (C-3"), 57.0 (C-4") and 66.0 ppm (C-5").

N-Formylation of UDP-L-Ara4N by the Bifunctional Enzyme ArnA 14163

FIG. 6. Partial ${}^{1}\mathrm{H}$-${}^{1}\mathrm{H}$ COSY of UDP-L-Ara4FN at $600 \, \mathrm{MHz}$. The connectivities in the ribose and pyranose moieties of $1.0 \, \mathrm{mg}$ UDP-L-Ara4FN ($\mathrm{D}_{2} \mathrm{O}$, $\mathrm{pD} \, 8.3$ at $25^{\circ} \mathrm{C}$) are shown. Key cross-peaks are labeled. The full ${}^{1}\mathrm{H}$ and ${}^{13}\mathrm{C}$ NMR assignments are listed in Tables II and III. Cross-peaks from the minor trans-formamido rotamer observed for the H-5", H-4", and H-3" positions are labeled $t$.

FIG. 7. Partial two-dimensional HMQC spectrum of UDP-L-Ara4FN. The spectrum was recorded in $\mathrm{D}_{2} \mathrm{O}$ at $25^{\circ} \mathrm{C}$, pD 8.3. Panel A, the single bond ${}^{1}\mathrm{H}$-${}^{13}\mathrm{C}$ correlations from the formyl CH, uracil H-5 and H-6, ribose H-1', and pyranose H-1" protons. Panel B, the ${}^{1}\mathrm{H}$-${}^{13}\mathrm{C}$ correlations for the other ribose and pyranose protons to their respective carbons. Minor trans-formamido rotational isomer cross-peaks observed for the formyl CH, C-5", C-4", C-3", and C-2" positions are labeled $t$.

TABLE IV  
Complementation of the arnA::kan<sup>r</sup> replacement in SDB 101  

| E. coli strain | Plasmid<sup>a</sup>       | Expressed protein(s)    | Phenotype          | Lipid A L-Ara4N modification | Specific activity<sup>b</sup> |
|----------------|--------------------------|-------------------------|--------------------|-------------------------------|------------------------------|
|                |                          |                         |                    | ArnADH                      | ArnAFT                       |
|                |                          |                         |                    | nmol/min/mg                 | nmol/min/mg                  |
| WD101          |                          |                         | Pm<sup>r</sup>     | Yes                         | 0.25                         | 0.8                          |
| SDB 101        |                          |                         | Pm<sup>s</sup>, Kan<sup>r</sup> | No                          | None detected                 | 0.01                         |
| SDB 101        | pWSK 29                  |                         | Pm<sup>s</sup>, Kan<sup>r</sup>, Amp<sup>r</sup> | No                          | None detected                 | 0.02                         |
| SDB 101        | pWSK-ArnA                | ArnA                   | Pm<sup>r</sup>, Kan<sup>r</sup>, Amp<sup>r</sup> | Yes                         | 8                             | 20                           |
| SDB 101        | pWSK-ArnADH              | ArnADH                 | Pm<sup>s</sup>, Kan<sup>r</sup>, Amp<sup>r</sup> | No                          | 0.3                           | 0.08                         |
| SDB 101        | pWSK-ArnAFT              | ArnAFT                 | Pm<sup>s</sup>, Kan<sup>r</sup>, Amp<sup>r</sup> | No                          | None detected                 | 60                           |
| SDB 101        | pSDB 300                 | ArnAFT and ArnADH      | Pm<sup>r</sup>, Kan<sup>r</sup>, Amp<sup>r</sup> | Yes                         | Not assayed                   | Not assayed                  |
| SDB 101        | pSDB 350                 | ArnADH and ArnAFT      | Pm<sup>r</sup>, Kan<sup>r</sup>, Amp<sup>r</sup> | Yes                         | Not assayed                   | Not assayed                  |

<sup>a</sup> The pWSK 29 plasmid expressed β-lactamase conferring ampicillin resistance (32).  
<sup>b</sup> Specific activities were calculated from assays of cell-free extracts.

FIG. 8. Restoration of polymyxin resistance to SDB 101 requires both ArnA domains. *E. coli* strains were streaked on LB agar containing 10 μg/ml polymyxin and grown overnight at 37°C. 1, W3110; 2, WD101 (*pmrA<sup>c</sup>*); 3, SDB 101 (*pmrA<sup>c</sup>*, *arnA::kan<sup>r</sup>*); 4, SDB 101/pWSK-ArnA; 5, SDB 101/pWSK-ArnADH; 6, SDB 101/pWSK-ArnAFT; 7, SDB 101/pSDB 300; 8, SDB 101/pSDB 350; 9, SDB 101/pWSK 29.

(Fig. 8 and Table IV). Furthermore, the lipid A of SDB 101 lacked the L-Ara4N modification unless complemented with both domains of ArnA (Table IV). These experiments show that UDP-L-Ara4FN is an obligatory intermediate in the modification of lipid A with the L-Ara4N residue. It is not a side product that is used for unrelated glycosylation reactions.

Enzymatic Activity Associated with the ArnA Domains—Simultaneous expression of both disconnected ArnA domains from either pSDB 300 or pSDB 350 (Tables I and IV and Supplemental Fig. 1) caused SDB 101 to regain polymyxin resistance. To confirm that the isolated domains possessed catalytic activity by themselves, cell-free extracts of the relevant constructs were prepared and assayed. Extracts of SDB 101 or SDB 101/pWSK 29 contained no ArnA dehydrogenase or formyltransferase activity (Fig. 9), but ArnB activity (Fig. 2) was normal (data not shown). Extracts of SDB 101 expressing ArnADH from pWSK-ArnADH had similar levels of dehydrogenase as the parental strain *E. coli* WD101 but not quite as much as SDB 101 expressing full-length ArnA from pWSK-ArnA (Table IV and Fig. 9, panel A). Extracts of SDB 101 expressing ArnAFT from pWSK-ArnAFT contained 3-fold higher formyltransferase activity than SBD101 expressing full-length ArnA or ~75-fold more activity than *E. coli* WD101 (Table IV and Fig. 9, panel B). When only ArnADH was expressed, dehydrogenase activity was detectable, but formyltransferase was not. Conversely, formyltransferase activity was present in extracts of cells expressing ArnAFT, but dehydrogenase was not.

The differing activity levels associated with the separated domains versus the full-length protein are related in part to protein expression. When the domains are expressed from pET24b, the full-length ArnA and the formyltransferase domain are made at high levels, but the dehydrogenase domain is not, as judged by SDS-PAGE (data not shown). Purified proteins will be required to evaluate the kinetic parameters quantitatively. Whatever the outcome, both domains of ArnA are required for maintenance of polymyxin resistance, and extracts of the strains expressing the individual domains contain at least as much enzymatic activity as do extracts of the polymyxin-resistant parental strain WD101.

Dependence of ArnC on UDP-L-Ara4FN as the Donor Substrate—As shown in Fig. 10, membranes of *E. coli* cells overexpressing ArnC efficiently convert UDP-L-Ara4FN but not UDP-L-Ara4N to lipid product. Formation of product is dependent upon time and the addition of the undecaprenyl phosphate acceptor substrate (data not shown). This unusual selectivity accounts for the fact that both domains of ArnA are required for polymyxin resistance and aminoarabinose modification of lipid A (Fig. 8 and Table IV). ArnC(PmrF) itself is likewise needed for both polymyxin resistance and aminoarabinose modification (19, 21). The reaction catalyzed by ArnC mechanistically resembles the one catalyzed by dolichyl phosphate mannosynthase in eucaryotic cells, which accounts for the distant similarity of the ArnC sequence to the dolichyl phosphate-mannosynthase family, as first noted by Zhou et al. (24).

DISCUSSION

In *E. coli*, *S. typhimurium*, and some other pathogenic Gram-negative bacteria, modification of the phosphate groups of lipid A with the L-Ara4N moiety (Fig. 1) confers resistance to polymyxin and cationic antimicrobial peptides by reducing the net negative charge of the lipid (18, 19, 21, 22). The L-Ara4N unit arises from UDP-GlcUA (25, 26). It is synthesized and attached to lipid A by the pathway shown in Fig. 2. Six ArnA proteins (previously designated PmrF through -K) (19) and Ugd are required for this process. As shown previously, ArnA catalyzes the oxidative decarboxylation of UDP-GlcUA to generate a novel UDP-4"-ketopentose (25), which ArnB converts to UDP-L-Ara4N (Fig. 2) (26). The lipid undecaprenyl phosphate-α-L-Ara4N, which accumulates in polymyxin resistant mutants, is

N-Formylation of UDP-L-Ara4N by the Bifunctional Enzyme ArnA 14165

**FIG. 9. ArnA dehydrogenase and formyltransferase activities in the separated domains.** Cell-free extracts were assayed at 30 °C. Panel A, ArnA dehydrogenase catalyzed conversion of UDP-GlcUA to UDP-4"-ketopentose with 20 μM UDP-[U-¹⁴C]GlcUA, 3 mM NAD⁺, and 5 mM α-ketoglutarate. The following *E. coli* extracts were assayed: WD101 (▲, 0.5 mg/ml); SDB101 (◆, 0.5 mg/ml); SDB101/pWSK-ArnA (●, 0.1 mg/ml); SDB101/pWSK-ArnADH (□, 0.5 mg/ml); SDB101/pWSK-ArnAFT (▽, 0.5 mg/ml). Panel B, the ArnA formyltransferase catalyzed the conversion of UDP-L-Ara4N to UDP-L-Ara4FN, which was assayed with 20 μM UDP-L-[U-¹⁴C]Ara4N, 1 mM N-5,N10-methenyltetrahydrofolate (converted *in situ* to N-10-formyltetrahydrofolate), and 5 mM L-glutamate. The following *E. coli* extracts were assayed: WD101 (▲, 0.5 mg/ml); SDB101 (◆, 0.5 mg/ml); SDB101/pWSK-ArnA (●, 0.05 mg/ml); SDB101/pWSK-ArnADH (■, 0.5 mg/ml); SDB101/pWSK-ArnAFT (▽, 0.01 mg/ml). At the indicated times 1 μl portions were spotted onto a polyethyleneimine-cellulose plate, which was developed and analyzed as described under "Experimental Procedures."

then used by ArnT as the donor substrate for transferring the L-Ara4N moiety to lipid A on the outer surface of the inner membrane (16, 17).

In principle, the most direct pathway for L-Ara4N modification of lipid A would involve the transfer of the L-Ara4N unit from UDP-L-Ara4N directly to undecaprenyl phosphate by ArnC followed by its transfer to lipid A. Unexpectedly, however, ArnA is a bifunctional enzyme with a putative N-terminal formyltransferase active site in addition to its C-terminal UDP-GlcUA dehydrogenase activity (25). We have now shown unequivocally that the N-terminal domain of ArnA catalyzes the

**FIG. 10. Strict dependence of ArnC activity on UDP-L-Ara4FN as the donor substrate.** The absence of lipid product formation with UDP-L-Ara4N accounts for the essential function of the formyl transferase domain of ArnA in the maintenance of polymyxin resistance and lipid A modification. The ArnC assay is described under “Experimental Procedures.”

N-formylation of UDP-L-Ara4N (Fig. 2, box), accounting for the formylation UDP-L-Ara4N previously seen in crude extracts of polymyxin-resistant (but not wild-type) *E. coli* (25). Purified ArnAN6H requires N-10-formyltetrahydrofolate for the modification of UDP-L-Ara4N (Fig. 3). Like other formyltransferases of related primary sequence, ArnA does not recognize N-5-formyltetrahydrofolate. Purified ArnAN6H was also used to prepare 1 mg of the putative UDP-L-Ara4FN. NMR spectroscopy demonstrated unambiguously that UDP-L-Ara4FN is recovered as a (4:1) mixture of cis- to trans-rotational isomers of the 4" formamido group (Figs. 4–7).

Certain Gram-negative bacteria possess LPS O-antigen sugars modified with formamido moieties (37, 38, 41–43). Despite numerous examples of LPS sugars containing formamido groups, formamido-modified sugar nucleotides synthesized *in vitro* have not been reported previously. A pathway for GDP-4-formamido-4,6-dideoxymannose (GDP-N-formylperosamine) has been proposed based on bioinformatics and demonstration that the *wbkC* gene, encoding a putative formyltransferase, is required for O-side-chain production in *Brucella melitensis* 16M (56). However, to our knowledge UDP-L-Ara4FN represents the first reported nucleoside diphosphate 4"-formamido sugar of any kind. The N-terminal domain of ArnA is also the first example of a confirmed formyltransferase that utilizes an amino-sugar nucleotide substrate.

Gunn *et al.* (19, 21) have reported that *pmrI* (*arnA*) is required for polymyxin resistance and L-Ara4N modification of lipid A in *S. typhimurium*. We have now extended these important observations by showing that the N-terminal formyltransferase and C-terminal dehydrogenase domains of ArnA are both required for polymyxin resistance and lipid A modification. The domains are catalytically active as separate proteins (Fig. 9). All Gram-negative bacteria capable of modifying their lipid A with L-Ara4N contain close orthologs of ArnA with both the N-terminal formyltransferase and C-terminal dehydrogenase domains. These organisms include *E. coli*, all types of *Salmonella*, *Yersinia pestis*, *Pseudomonas aeruginosa*, and *Burkholderia cepacia* (57). Other bacteria, like the endosymbiont *Wigglesworthia*, with sequenced genomes but uncharacterized lipid A structures (58), contain a bi-functional ArnA and the five other Arn proteins. However, *Ralstonia spp. solanacearum* and *metallidurans*, in which all of the genes for L-Ara4N modification of lipid A are present (59), are unique in that the two domains of ArnA are annotated as distinct proteins

separated by 10 or 11 nucleotides, respectively. Thus, the expression of the two discrete domains of ArnA, central to our experiments verifying the requirement of the ArnA formyltransferase in the L-Ara4N modification of lipid A, may in fact occur *in vivo* in a few Gram-negative bacteria.

Expression and purification of the separate domains of ArnA is already facilitating structural studies. Sousa and co-workers (55) have just published the crystal structure of the dehydrogenase domain of ArnA, and they have verified the similarity of its overall fold and active site to UDP-glucose epimerase. Naismith and co-workers have recently determined the structures of both the dehydrogenase and the formyltransferase domains of ArnA, again isolated as separate proteins.² A recent x-ray structure of ArnB is also available (60). The significance of the bifunctional nature of ArnA might become apparent if full-length ArnA and ArnB could be co-crystallized. It may be that there is substrate channeling between the active sites of these proteins.

ArnT, an inner membrane protein with 12 predicted transmembrane segments, transfers the L-Ara4N moiety of undecaprenyl phosphate-α-L-Ara4N to lipid A (16, 17). Undecaprenyl phosphate-α-L-Ara4N is a well characterized minor lipid present exclusively in polymyxin resistant strains of *E. coli* (16, 17). No formyl group is present either on undecaprenyl phosphate-α-L-Ara4N or on L-Ara4N-modified lipid A (13, 16, 17, 24, 61). Because UDP-L-Ara4N is the product of ArnB, the existence of the formyltransferase domain of ArnA and its product, UDP-L-Ara4FN, presents an interesting conundrum. In principle, UDP-L-Ara4FN might be utilized for a separate glycosylation pathway having nothing to do with lipid A. However, the results of Fig. 8 and Table IV demonstrate conclusively that the formyltransferase is necessary for polymyxin resistance and lipid A modification. The function of the formyltransferase is explained by the observation that ArnC is selective for UDP-L-Ara4FN, as shown in Figs. 2 and 10. The now requisite deformylation required for the biosynthesis of undecaprenyl phosphate-α-L-Ara4N, catalyzed by ArnD (Fig. 2), might function to drive the pathway forward by preventing reversal of the ArnC reaction. Alternatively, deformylation might be essential for transporting undecaprenyl phosphate-α-L-Ara4N across the inner leaflet of the inner membrane to the outer leaflet, where the active site of ArnT is located. The availability of UDP-L-Ara4FN will enable the detailed characterization of ArnC and ArnD activity (Fig. 2) and should shed light on the function of the formylation in the modification of lipid A.

---

**REFERENCES**

1. Raetz, C. R. H. (1990) *Annu. Rev. Biochem.* **59**, 129–170
2. Raetz, C. R. H., and Whitfield, C. (2002) *Annu. Rev. Biochem.* **71**, 635–700
3. Nikaido, H. (2003) *Microbiol. Mol. Biol. Rev.* **67**, 593–656
4. Brade, H., Opal, S. M., Vogel, S. N., and Morrison, D. C. (eds) (1999) *Endotoxin in Health and Disease*, Marcel Dekker, Inc., New York
5. Parillo, J. E. (1993) *N. Engl. J. Med.* **328**, 1471–1477
6. Rietschel, E. T., Kirikae, T., Schade, F. U., Mamat, U., Schmidt, G., Loppnow, H., Ulmer, A. J., Zähringer, U., Seydel, U., Di Padova, F., Schreier, M., and Brade, H. (1994) *FASEB J.* **8**, 217–225
7. Lien, E., and Ingalls, R. R. (2002) *Crit. Care Med.* **30**, 1–11
8. Guo, L., Lim, K. B., Gunn, J. S., Bainbridge, B., Darveau, R. P., Hackett, M., and Miller, S. I. (1997) *Science* **276**, 250–253
9. Groisman, E. A. (2001) *J. Bacteriol.* **183**, 1835–1842
10. Gibbons, H. S., Kalb, S. R., Cotter, R. J., and Raetz, C. R. H. (2005) *Mol. Microbiol.* **55**, 425–440
11. Guo, L., Lim, K. B., Poduje, C. M., Daniel, M., Gunn, J. S., Hackett, M., and Miller, S. I. (1998) *Cell* **95**, 189–198
12. Bishop, R. E., Gibbons, H. S., Guina, T., Trent, M. S., Miller, S. I., and Raetz, C. R. H. (2000) *EMBO J.* **19**, 5071–5080
13. Zhou, Z., Ribeiro, A. A., Lin, S., Cotter, R. J., Miller, S. I., and Raetz, C. R. H. (2001) *J. Biol. Chem.* **276**, 43111–43121
14. Trent, M. S., and Raetz, C. R. H. (2002) *J. Endotoxin Res.* **8**, 158

15. Lee, H., Hsu, F. F., Turk, J., and Groisman, E. A. (2004) *J. Bacteriol.* **186**, 4124–4133
16. Trent, M. S., Ribeiro, A. A., Lin, S., Cotter, R. J., and Raetz, C. R. H. (2001) *J. Biol. Chem.* **276**, 43122–43131
17. Trent, M. S., Ribeiro, A. A., Doerrler, W. T., Lin, S., Cotter, R. J., and Raetz, C. R. H. (2001) *J. Biol. Chem.* **276**, 43132–43144
18. Helander, I. M., Kilpeläinen, I., and Vaara, M. (1994) *Mol. Microbiol.* **11**, 481–487
19. Gunn, J. S., Lim, K. B., Krueger, J., Kim, K., Guo, L., Hackett, M., and Miller, S. I. (1998) *Mol. Microbiol.* **27**, 1171–1182
20. Nikaido, H., and Vaara, M. (1985) *Microbiol. Rev.* **49**, 1–32
21. Gunn, J. S., Ryan, S. S., Van Velkinburgh, J. C., Ernst, R. K., and Miller, S. I. (2000) *Infect. Immun.* **68**, 6139–6146
22. Nummila, K., Kilpeläinen, I., Zähringer, U., Vaara, M., and Helander, I. M. (1995) *Mol. Microbiol.* **16**, 271–278
23. Gunn, J. S. (2001) *J. Endotoxin Res.* **7**, 57–62
24. Zhou, Z., Lin, S., Cotter, R. J., and Raetz, C. R. H. (1999) *J. Biol. Chem.* **274**, 18503–18514
25. Breazeale, S. D., Ribeiro, A. A., and Raetz, C. R. H. (2002) *J. Biol. Chem.* **277**, 2886–2896
26. Breazeale, S. D., Ribeiro, A. A., and Raetz, C. R. H. (2003) *J. Biol. Chem.* **279**, 24731–24739
27. Blanquet, S., Dessen, P., and Kahn, D. (1984) *Methods Enzymol.* **106**, 141–152
28. Smith, P. K., Krohn, R. I., Hermanson, G. T., Mallia, A. K., Gartner, F. H., Provenzano, M. D., Fujimoto, E. K., Goeke, N. M., Olson, B. J., and Klenk, D. C. (1985) *Anal. Biochem.* **150**, 76–85
29. Ausubel, F. M., Brent, R., Kingston, R. E., Moore, D. D., Seidman, J. G., Smith, J. A., and Struhl, K. (eds) (1989) *Current Protocols in Molecular Biology*, John Wiley & Sons, New York
30. Sambrook, J., Fritsch, E. F., and Maniatis, T. (1989) *Molecular Cloning: A Laboratory Manual*, 2nd Ed., Cold Spring Harbor, Cold Spring Harbor, NY
31. Miller, J. R. (1972) *Experiments in Molecular Genetics.*, Cold Spring Harbor Laboratory, Cold Spring Harbor, NY
32. Wang, R. F., and Kushner, S. R. (1991) *Gene (Amst.)* **100**, 195–199
33. Ribeiro, A. A., Zhou, Z., and Raetz, C. R. H. (1999) *Magn. Reson. Chem.* **37**, 620–630
34. Yu, D., Ellis, H. M., Lee, E. C., Jenkins, N. A., Copeland, N. G., and Court, D. L. (2000) *Proc. Natl. Acad. Sci. U. S. A.* **97**, 5978–5983
35. Naleway, J. J., Raetz, C. R. H., and Anderson, L. (1988) *Carbohydr. Res.* **179**, 199–209
36. Agrawal, P. K. (1992) *Phytochemistry* **31**, 3307–3330
37. Vinogradov, E. V., Shashkov, A. S., Knirel, Y. A., Kochetkov, N. K., Tochtamysheva, N. V., Averin, S. F., Goncharova, O. V., and Khlebnikov, V. S. (1991) *Carbohydr. Res.* **214**, 289–297
38. Knirel, Y. A., Vinogradov, E. V., Shashkov, A. S., Dmitriev, B. A., Kochetkov, N. K., Stanislavsky, E. S., and Mashilova, G. M. (1985) *Eur. J. Biochem.* **150**, 541–550
39. Kenne, L., Unger, P., and Wehler, T. (1988) *J. Chem. Soc. Perkin Trans.* **1**, 1183–1186
40. Rosenthal, S. N., and Fendler, J. H. (1976) in *Advances in Physical Organic Chemistry*. (Gold, V., and Bethell, D., eds) Vol. 13, pp. 280–424, Academic Press, Inc., New York
41. Katzenellenbogen, E., Romanowska, E., Kocharova, N. A., Shashkov, A. S., Knirel, Y. A., and Kochetkov, N. K. (1995) *Carbohydr. Res.* **273**, 187–195
42. Kocharova, N. A., Maszewska, A., Zatonsky, G. V., Bystrova, O. V., Ziolkowski, A., Torzewska, A., Shashkov, A. S., Knirel, Y. A., and Rozalski, A. (2003) *Carbohydr. Res.* **338**, 1425–1430
43. Muldoon, J., Shashkov, A. S., Senchenkova, S. N., Tomshich, S. V., Komandrova, N. A., Romanenko, L. A., Knirel, Y. A., and Savage, A. V. (2001) *Carbohydr. Res.* **330**, 231–239
44. Higgins, D. G., Thompson, J. D., and Gibson, T. J. (1996) *Methods Enzymol.* **266**, 383–402
45. Schmitt, E., Blanquet, S., and Mechulam, Y. (1996) *EMBO J.* **15**, 4749–4758
46. Schmitt, E., Panvert, M., Blanquet, S., and Mechulam, Y. (1998) *EMBO J.* **17**, 6819–6826
47. Inglese, J., Johnson, D. L., Shiau, A., Smith, J. M., and Benkovic, S. J. (1990) *Biochemistry* **29**, 1436–1443
48. Fiss, E. H., Yu, S., and Jacobs, W. R., Jr. (1994) *Mol. Microbiol.* **14**, 557–569
49. Tatusova, T. A., and Madden, T. L. (1999) *FEMS Microbiol. Lett.* **174**, 247–250
50. Warren, M. S., Marolewski, A. E., and Benkovic, S. J. (1996) *Biochemistry* **35**, 8855–8862
51. Shim, J. H., and Benkovic, S. J. (1999) *Biochemistry* **38**, 10024–10031
52. Frey, P. A. (1996) *FASEB J.* **10**, 461–470
53. Hegeman, A. D., Gross, J. W., and Frey, P. A. (2001) *Biochemistry* **40**, 6598–6610
54. Allard, S. T., Giraud, M. F., Whitfield, C., Graninger, M., Messner, P., and Naismith, J. H. (2001) *J. Mol. Biol.* **307**, 283–295
55. Gatzeva-Topalova, P. Z., May, A. P., and Sousa, M. C. (2004) *Biochemistry* **43**, 13370–13379
56. Godfroid, F., Cloeckaert, A., Taminiau, B., Danese, I., Tibor, A., de Bolle, X., Mertens, P., and Letesson, J. J. (2000) *Res. Microbiol.* **151**, 655–668
57. Zähringer, U., Lindner, B., and Rietschel, E. T. (1999) in *Endotoxin in Health and Disease* (Brade, H., Opal, S. M., Vogel, S. N., and Morrison, D. C., eds) pp. 93–114, Marcel Dekker, Inc., New York
58. Akman, L., Yamashita, A., Watanabe, H., Oshima, K., Shiba, T., Hattori, M., and Aksoy, S. (2002) *Nat. Genet.* **32**, 402–407
59. Salanoubat, M., Genin, S., Artiguenave, F., Gouzy, J., Mangenot, S., Arlat, M., Billault, A., Brottier, P., Camus, J. C., Cattolico, L., Chandler, M., Choisne, N., Claudel-Renard, C., Cunnac, S., Demange, N., Gaspin, C., Lavie, M., Moisan, A., Robert, C., Saurin, W., Schieix, T., Siguier, P., Thébault, P., Whalen, M., Wincker, P., Levy, M., Weissenbach, J., and Boucher, C. A. (2002) *Nature* **415**, 497–502
60. Noland, B. W., Newman, J. M., Hendle, J., Badger, J., Christopher, J. A.,

Tresser, J., Buchanan, M. D., Wright, T. A., Rutter, M. E., Sanderson, W. E., Muller-Dieckmann, H. J., Gajiwala, K. S., and Buchanan, S. G. (2002) *Structure (Camb.)* **10**, 1569–1580

61. Zhou, Z., Ribeiro, A. A., and Raetz, C. R. H. (2000) *J. Biol. Chem.* **275**, 13542–13551

62. Gibbons, H. S., Lin, S., Cotter, R. J., and Raetz, C. R. H. (2000) *J. Biol. Chem.*

63. Trent, M. S., Pabich, W., Raetz, C. R. H., and Miller, S. I. (2001) *J. Biol. Chem.* **276**, 9083–9092

64. Miller, S. I., Kukral, A. M., and Mekalanos, J. J. (1989) *Proc. Natl. Acad. Sci. U. S. A.* **86**, 5054–5058

65. Gunn, J. S., and Miller, S. I. (1996) *J. Bacteriol.* **178**, 6857–6864

Enzyme Catalysis and Regulation:  
A Formyltransferase Required for  
Polymyxin Resistance in *Escherichia coli*  
and the Modification of Lipid A with  
4-Amino-4-deoxy-l-arabinose:  
IDENTIFICATION AND FUNCTION OF  
UDP-4-DEOXY-4-FORMAMIDO-I-ARAB  
INOSE  

Steven D. Breazeale, Anthony A. Ribeiro,  
Amanda L. McClerren and Christian R. H.  
Raetz  
J. Biol. Chem. 2005, 280:14154-14167.  
doi: 10.1074/jbc.M414265200 originally published online January 28, 2005  

Access the most updated version of this article at doi: 10.1074/jbc.M414265200  

Find articles, minireviews, Reflections and Classics on similar topics on the JBC Affinity Sites.

Alerts:
- When this article is cited
- When a correction for this article is posted

Click here to choose from all of JBC's e-mail alerts  

Supplemental material:  
http://www.jbc.org/content/suppl/2005/02/11/M414265200.DC1.html  

This article cites 60 references, 21 of which can be accessed free at  
http://www.jbc.org/content/280/14/14154.full.html#ref-list-1
