
Rifamycin—Mode of Action, Resistance, and Biosynthesis

Heinz G. Floss* and Tin-Wein Yu

Department of Chemistry, Box 351700, University of Washington, Seattle, Washington 98195-1700

Received May 10, 2004

### Contents
1. Introduction                                                                                       621
2. Mechanism of Action                                                                                621
3. Rifamycin Resistance                                                                              623
   3.1. Resistance Due to Modification of *rpoB*                                                     623
   3.2. Other Resistance Mechanisms                                                                  624
4. Rifamycin Biosynthesis                                                                             626
5. Autoresistance of Amycolatopsis Mediterranei                                                      629
6. Concluding Remarks                                                                                 630
7. Acknowledgment                                                                                     631
8. References                                                                                         631

### 1. Introduction

The rifamycins¹ belong to the family of ansamycin antibiotics,³⁴ so named because of their basket-like molecular architecture comprising an aromatic moiety bridged at nonadjacent positions by an aliphatic chain (Latin: ansa = handle).⁵ The aromatic moiety can either be a naphthalene or a naphthoquinone ring system, as in the naphthalenic ansamycins rifamycin and naphthomycin⁶ or the streptovaricins,⁷ or it can be a benzene or benzoquinone ring, as in the benzenic ansamycins geldanamycin⁸ or ansamitocin⁹ (Figure 1). The rifamycins were first isolated by Sensi and co-workers at Lepetit SA in Milan in 1959 as a complex mixture of several congeners.¹⁰ The producing organism, an Actinomycete, was originally classified as *Streptomyces mediterranei*,¹¹ then reclassified as *Nocardia mediterranea*,¹² and finally assigned to a newly defined genus as *Amycolatopsis mediterranei*.¹³ Fermentation in the presence of added diethylbarbituric acid led to the production of predominantly rifamycin B,¹⁴ the structure of which was determined by chemical means and X-ray crystallography.¹⁵⁻¹⁸ Subsequently, it was possible by mutagenesis of the producing organism to eliminate the requirement for addition of diethylbarbituric acid to the fermentation.¹⁹ Since then, numerous other rifamycins have been isolated from the fermentation of *A. mediterranei* or its mutants and their structures have been determined.¹⁹ Very closely related compounds have been isolated from other Actinomycetes, for example, tolypomycin (together with rifamycins B and O) from *Amycolatopsis tolypomycina*²⁰,¹³⁸ and the halomicins from *Micromonospora halophytica* (Figure 2).²¹

The rifamycins display a broad spectrum of antibiotic activity against Gram-positive and, to a lesser extent, Gram-negative bacteria.²² Rifamycin B, the product of the commercial fermentation, has only very modest activity, but it can be converted chemically, enzymatically, or by biotransformation into rifamycin SV (Figure 3) (cf. ref 19),²³⁻²⁵ which has much more potent activity and was the first rifamycin used clinically.²⁶ Rifamycin SV is a biosynthetic precursor of rifamycin B,²⁷ and mutagenesis of the producing organism has succeeded in blocking the terminal conversion step, resulting in the accumulation of rifamycin SV.²⁸ However, strain optimization of this mutant has not been as successful as that of its rifamycin B-producing parent strain, and most commercial fermentations appear to produce rifamycin B, which is then converted into rifamycin SV.¹⁹ Following the clinical introduction of rifamycin SV, extensive programs of semisynthesis, primarily at the Lepetit group and at Ciba-Geigy, led to the preparation and evaluation of large numbers of analogues of rifamycin.²,³,²⁹,³⁰ From these, rifampicin (Figure 3) was selected as the next generation clinical candidate.²³ Rifampicin shows more pronounced activity against Gram-positive bacteria, particularly mycobacteria, better activity against Gram-negative bacteria, and importantly, excellent oral bioavailability.³¹ It has become one of the mainstay agents in the treatment of tuberculosis, leprosy, and AIDS-associated mycobacterial infections.³² Since resistance to rifampicin develops rather rapidly,³³ the drug is typically used in combination with other antimycobacterial agents, particularly isoniazid.³⁴ Other semisynthetic rifamycin derivatives, such as rifabutin³⁵ and rifapentine³⁶ (Figure 3), were subsequently introduced for clinical use. Rifabutin in particular is active against a number of rifampicin-resistant clinical pathogens.³⁵

### 2. Mechanism of Action

The antibacterial action of rifampicin results from its inhibition of DNA-dependent RNA synthesis.³⁷ This inhibition is due not to interaction with the template, but to strong binding to the DNA-dependent RNA polymerase of prokaryotes.³⁸ Binding constants for prokaryotic RNA polymerases are in the range of 10⁻⁸ M; eukaryotic enzymes are at least 10² to 10⁴ times less sensitive to inhibition by rifamycin.³⁹ The inhibition of DNA-dependent RNA polymerase seems to be the common mechanism for all antibacterially active rifamycins;³⁸ the many structural modifications made in these molecules primarily alter the pharmacokinetic properties of the molecules⁴ and their affinity for eukaryotic DNA.

the antibiotic interferes with the initiation phase of RNA synthesis;^{42,43} once RNA synthesis has progressed beyond an early stage and the enzyme carries a longer oligoribonucleotide chain, the process is no longer sensitive to rifampicin.^{44} DNA-dependent RNA polymerase is a complex enzyme with an α₂ β β′ σ subunit structure. The binding site for rifampicin has been located at the β subunit encoded by the *rpoB* gene. This follows from the construction of chimeric RNA polymerases containing subunits from rifampicin-sensitive and rifampicin-resistant strains.^{45,46} An α₂ β β′ complex, the core enzyme, binds rifampicin, that is, the σ subunit is not required,^{47} but the β′ subunit is necessary for rifampicin binding.^{48} A seminal observation was made by McClure and Cech,^{49} who reported that the rifampicin-inhibited RNA polymerase released dinucleotides when the reaction was initiated with a nucleoside triphosphate, whereas the reaction was terminated after the second phosphodiester bond formation when it was initiated with nucleoside di- or monophosphates. This led them to propose that the binding of rifampicin to the RNA polymerase–DNA complex sterically blocks the extension of the nascent RNA chain after the first or second condensation step. Although other mechanisms have been proposed, for example, allosteric effects on protein conformation,^{50} the crystal structure of the core RNA polymerase–rifampicin complex bears out the mechanism proposed by McClure and Cech.^{41}

The 3.3 Å crystal structure of the complex of *T.* aquaticus core DNA-dependent RNA polymerase with rifampicin was solved and interpreted following the earlier determination of the structure of the enzyme alone by the same group^{51} and the development of a functional model for the enzyme based on the structural data.^{52} According to the structure of the complex,^{41} rifampicin binds to the β subunit deep within the main DNA/RNA channel, about 12 Å away from the active site Mg²⁺ ion, consistent with biochemical data.^{53,54} Binding to the protein involves hydrogen bonding interactions between the four hydroxyl groups at C-1, C-8, C-21, and C-23, which are essential for biological activity of rifamycins,^{55} as well as the carbonyl oxygen of the C-25 acetoxy group, and amino acid residues R409, S411, Q393, H406, D396, and F394. In addition, hydrophobic interactions with E445, I452, G414, L413, L391, and Q390 contribute to the binding of the antibiotic (Figure 4). The position of the bound antibiotic is such that it physically gets in the way of the growing oligonucleotide chain after the first or second chain elongation step. The model clearly indicates that rifamycin does not inhibit the initiation step or the translocation step, and it explains why the enzyme after a number of elongation steps, when it carries a longer RNA chain, is no longer sensitive to inhibition by rifamycin. It also neatly explains the release of di- or trinucleotides from the inhibited enzyme.

Comparative studies with DNA-dependent RNA polymerases from Gram-negative and Gram-positive bacteria have shown that they have comparably high sensitivities to rifampicin. Thus, the lower sensitivity of Gram-negative bacteria to rifamycins must be due

A native of Berlin, Germany, Professor Heinz G. Floss studied chemistry at the Technical Universities of Berlin and Munich, obtaining his Ph.D. in 1961. Following a postdoctorate in biochemistry at UC Davis, he joined the faculty of the Department of Medicinal Chemistry and Pharmacognosy at Purdue University in 1966, where he rose through the ranks to the position of Lilly Distinguished Professor and Head of Department. After serving as Professor of Chemistry and Department Chair at The Ohio State University from 1982–1987, he moved to the Department of Chemistry, University of Washington, Seattle. At the end of 2000 he was awarded Emeritus status, but he continues to pursue his research interests in the biosynthesis of natural products. He published about 400 papers, and his honors include the NIH Research Career Development Award, the Research Achievement Award in Natural Products of the Academy of Pharmaceutical Sciences, the Volwiler Award of the American Association of Colleges of Pharmacy, the Humboldt US Senior Scientist Award, Honorary Doctor of Science degrees from Purdue University and from the University of Bonn, Germany, the Research Award of the American Society of Pharmacognosy, the Kitasato Award in Microbial Chemistry, and Honorary Membership of the Kitasato Institute, Tokyo.

Tin-Wein Yu received a Ph.D. in Biology under Professor Sir David A. Hopwood with an emphasis on microbial genetics, from the University of East Anglia, England, in 1995. He did postdoctoral research with Professor Heinz G. Floss at the University of Washington and is currently an assistant Professor at the Department of Biological Sciences in Louisiana State University. His research interests include bioinformatics-based natural product discovery, bioactive molecules, and biosynthetic networks in living organism.

dependent RNA polymerases,^{40} but do not change the principal mechanism of action.

The interaction of rifampicin with DNA-dependent RNA polymerase has been studied in considerable detail since its initial discovery, culminating recently in the solution of the crystal structure of the complex of rifampicin with the enzyme from *Thermus aquaticus* and the development of a detailed model for rifampicin’s mechanism of inhibition of RNA synthesis by this enzyme.^{41} Early work had revealed that

Rifamycin—Mode of Action, Resistance, and Biosynthesis

Chemical Reviews, 2005, Vol. 105, No. 2 623

Figure 1. Structures of representative ansamycins.

![Structure of Rifamycin B](https://i.imgur.com/structure_of_rifamycin_b.png)

Rifamycin B

![Structure of Naphthomycin C](https://i.imgur.com/structure_of_naphthomycin_c.png)

Naphthomycin C

![Structure of Geldanamycin](https://i.imgur.com/structure_of_geldanamycin.png)

Geldanamycin

![Structure of Ansamitocin P-3](https://i.imgur.com/structure_of_ansamitocin_p_3.png)

Ansamitocin P-3

![Structure of Streptovaricin C](https://i.imgur.com/structure_of_streptovaricin_c.png)

Streptovaricin C

almost exclusively in drug combinations, most commonly with isoniazid,⁶³ and why its use, at least in the United States, is restricted to the treatment of tuberculosis or clinical emergencies.⁶¹ By far the predominant mechanism of resistance to rifamycins is modification of the drug target, *rpoB*, by mutation. Resistance by modification of the antibiotic (inactivation) has also been described, but its clinical significance, at least in *M. tuberculosis*, does not seem to be as high.

### 3.1. Resistance Due to Modification of *rpoB*

The vast majority of mutations to rifampicin resistance map to the *rpoB* gene in *E. coli*⁴⁶,⁶⁴,⁶⁵ as well as in *M. tuberculosis*⁶⁶–⁶⁸ and other microorganisms⁶⁸,⁶⁹ examined (Figure 5). Following the primary structure determination of *E. coli rpoB* by Ovchinnikov and co-workers,⁷⁰ several laboratories analyzed Rifᴿ mutants of *E. coli* for the nature of the mutations.⁷¹⁻⁷⁶ It was found that 95% of these mapped to four small regions in the N-terminal half of the encoded protein, the vast majority to region I spanning amino acids 505–537 (*E. c.* numbering) (Figure 5).⁶⁸ Most of these mutations are point mutations resulting in single amino acid substitutions, with a few deletions or insertions. The rif I region of *rpoB* is rather highly conserved among prokaryotic organisms, but not between prokaryotes and eukaryotes, such as yeast, *Drosophila melanogaster*, and humans.⁴¹ The different mutations of prokaryotic *rpoB* genes lead to different levels of rifamycin resistance; that is, insusceptibility of *rpoB* to rifamycins is not an all or nothing phenomenon.⁶³ Different mutants also display different degrees of “fitness,” that is, normal or impaired growth patterns.⁶³ Rifᴿ mutations in other microorganisms similarly mapped to equivalent regions in their respective *rpoB* genes.⁷⁷⁻⁸² In the *rpoB* gene of *M. tuberculosis*, all but one mutation mapped to the rif I region spanning amino acids 419–451 (*M. t.* numbering) (Figure 5), with 41% of

Figure 2. Structures of tolypomycin Y and halomicin B.

![Structure of Tolyomycin Y from Amycolatopsis tolypomycina](https://i.imgur.com/structure_of_tolyomycin_y.png)

Tolyomycin Y from  
*Amycolatopsis tolypomycina*

![Structure of Halomicin B from Micromonospora halophytica](https://i.imgur.com/structure_of_halomicin_b.png)

Halomicin B from  
*Micromonospora halophytica*

to poorer penetration of the antibiotic through the cell membrane.³³ These transport problems can be partially overcome by structural modifications, which is apparently the reason that rifampicin has better activity against Gram-negative bacteria than the original rifamycins, such as rifamycin SV.¹⁹

Rifamycins were also investigated for their potential as antitumor agents and, based on potential inhibition of reverse transcriptase,⁵⁶ as antiviral agents.⁵⁷ However, these inhibitory effects were not potent and/or selective enough to lead to clinical candidates.¹⁹ Rifamycins also interact with some other cellular targets. For example, rifampicin displays some immunosuppressive activity in addition to toxic side effects on the liver.⁵⁸ The former effect was traced to binding to and activation of the human glucocorticoid receptor by rifampicin.⁵⁹ There are also reports that rifampicin inhibits multidrug resistance and enhances anticancer drug accumulation in multidrug-resistant cells⁶⁰ due to down-modulation of P-glycoproteins.⁶¹

## 3. Rifamycin Resistance

Pathogens develop resistance to rifampicin at a high rate, 10⁻⁸ to 10⁻⁹ per bacterium per cell division.³³,⁶²,⁶³ This is the reason the antibiotic is used

Figure 3. Conversion of rifamycin B into rifamycin SV and clinically used derivatives.

the bound rifampicin, all but one (E445) are susceptible to mutation to rifampicin resistance. It may be assumed that mutation of E445 impairs the function of the enzyme sufficiently to make this a lethal mutation. The three amino acids most frequently mutated in resistant clinical isolates of *M. tuberculosis*, corresponding to H406, S411, and D396 (*T. a.* numbering), are involved in hydrogen bonding interactions with the oxygens at C-8 and C-21. The remaining 12 of the 23 sites known to be susceptible to mutation to rifampicin resistance do not make direct contact with the bound antibiotic but are located in a second sphere and are likely to affect rifamycin binding through subtle changes in the structure of the mutated protein. The mutations of *rpoB* to rifampicin resistance result in a decreased affinity of the enzyme to the antibiotic, which binds to the wild-type protein in a very tight one-to-one complex. This decreased affinity between antibiotic and target correlates with the decreased susceptibility of the organism to inhibition by rifampicin.

Figure 4. Interaction of rifampicin with proximal amino acids of *Thermus aquaticus* DNA-dependent RNA polymerase in the antibiotic–enzyme complex. Amino acid numbers refer to the *Thermus aquaticus* RpoB, except for the three shown in parentheses, which represent the changes in the rifamycin-resistant *Amycolatopsis mediterranei* RpoB. (Modified with permission from ref 41. Copyright 2001 Elsevier.)

the resistant clinical isolates carrying a mutation of S455, 36% of H440, and 9% of D430. Although some other mutations to rifampicin resistance are induced at a high rate, they do not manifest themselves in clinical isolates, presumably due to reduced fitness in a competitive environment.

The structural work on *rpoB* from *T. aquaticus* shows that, of the twelve amino acids involved in hydrogen bonding or van der Waals interactions with

### 3.2. Other Resistance Mechanisms

Different prokaryotic organisms show different degrees of susceptibility to inhibition by rifamycins. In some instances, this is due to decreased sensitivity of the DNA-dependent RNA polymerase to inhibition by rifamycin. For example, the enzyme from *T. aquaticus* is intrinsically less sensitive than that from *M. tuberculosis*. In other cases, including *M. smegmatis* and *Pseudomonas fluorescens*, however, different mechanisms of resistance seem to be operating, including impeded cellular uptake of the antibiotic. The about 200-fold higher sensitivity of *E. coli* to the rifamycin derivative CGP 4832,

Rifamycin—Mode of Action, Resistance, and Biosynthesis

Chemical Reviews, 2005, Vol. 105, No. 2 625

![Diagram](attachment:diagram.png)

Figure 5. Regions of the *rpoB* genes from *E. coli*, *Thermus aquaticus*, *Amycolatopsis mediterranei*, and *Mycobacterium tuberculosis* carrying mutations which confer rifampicin resistance upon the enzyme. The three amino acids highlighted in the *A. mediterranei* RpoB, N447, D438, and Q432 are responsible for the rifampicin resistance of this enzyme. (Modified with permission from ref 41. Copyright 2001 Elsevier.)

![Chemical Structures](attachment:chemical_structures.png)

Figure 6. Inactivation products of rifampicin generated by different rifampicin-resistant bacteria.

compared to rifampicin, is due to its active cellular uptake via the FhuA-TonB transport system.⁸⁸⁻⁹⁰ There is, however, little evidence for a role of permeability barriers in acquired high-level rifamycin resistance in *M. tuberculosis* and *M. leprae*.⁶⁶,⁷⁷ Another mechanism, antibiotic modification, has been demonstrated in a number of microorganisms. Dabbs first reported rifampicin inactivation by a *Rhodococcus* species and by *Mycobacterium smegmatis* through an inducible mechanism requiring de novo protein synthesis.⁹¹ A gene responsible for this activity was subsequently cloned from nocardioform DNA⁹² and later from *M. smegmatis* (*arr* gene).⁹³ The products of this modification were shown to be 23- O-α-D-ribosylrifampicin and 3-formyl-23-O-α-D-ribo-sylrifamycin SV (Figure 6), both antibacterially inactive compounds.⁹⁴,⁹⁵ Ribosylation of rifampicin contributes significantly to the natural low susceptibility of *M. smegmatis* to rifampicin; inactivation of the *arr* gene changed the MIC for rifampicin from 20 to 1.5 μg/mL.⁹³ Homologues of the *arr* gene, *arr*-2, have also been isolated from a multiply resistant strain of *P. aeruginosa* from a patient in Thailand,⁹⁶ from a clinical isolate of *Klebsiella pneumoniae*,⁹⁷ and from some *Enterobacteriaceae*.⁹⁸,⁹⁹ In most of these cases, the gene was located on and could be transferred by a plasmid. Two other modes of chemical inactivation of rifampicin have been reported in

21 22 23 24 25 26 27 28 29 30 31 32 S T 35 0

A B C D E

E F 1 G H I K L M N O 2 P Q 3 4 5 6 7 8 9 10 11

11 17 18 19 20 R 13 14 15A 15B 16 J 36 37

RpoB RpoC

5 KBp

AHBA biosynthesis polyketide synthase post-PKS modification sugar biosynthesis regulatory protein transporter protein hypothetical protein ribosomal protein and RNA polymerase

Figure 7. rif biosynthetic gene cluster and flanking genes encoding ribosomal proteins and the β and β′ subunits of DNA-dependent RNA polymerase.

pathogenic *Nocardia* species which are naturally rifamycin-resistant. *Nocardia brasiliensis* converts rifampicin into the 23-O-β-D-glucosyl derivative and into 3-formyl-23-O-β-D-glucosylrifamycin SV, ${}^{94,100}$ and *N. otidiiscaviarum* metabolizes the compound to 21-O-phosphorylrifampicin and 3-formyl-21-O-phosphorylrifamycin SV (Figure 6). ${}^{94,101}$ Consistent with the essential role of the 21 and 23 OH groups, all four compounds lack antibacterial activity. In a subsequent survey, the distribution of the three modification mechanisms among various *Nocardia* and *Mycobacterium* species and related taxa was examined. ${}^{102}$ The inactivation mechanisms were found to be rather species specific. It is clear that antibiotic modification plays a role in clinical resistance of non-mycobacterial species and perhaps also in *M. avium*, whereas it seems much less prominent in the clinical resistance of *M. tuberculosis* and *M. leprae*.

### 4. Rifamycin Biosynthesis

Feeding experiments with isotopically labeled precursors as well as mutagenesis and complementation experiments have demonstrated that rifamycin is a polyketide assembled from an aromatic starter unit, 3-amino-5-hydroxybenzoic acid (AHBA), through chain extension by two acetate and eight propionate units (cf. ref 19). The origin of the AHBA starter unit is related to the shikimate pathway, but shikimate or earlier intermediates of the pathway were not incorporated. ${}^{103}$ Rather, their amino analogues, 3,4-dideoxy-4-amino-D-arabino-heptulosonic acid 7-phosphate (aminoDAHP), 5-deoxy-5-amino-3-dehydroquionic acid (aminoDHQ), and 5-deoxy-5-amino-3-dehydrosikimic acid (aminoDHS) were efficiently converted into AHBA in cell-free extracts of *A. mediterranei*. ${}^{104}$ On the basis of this information, the enzyme AHBA synthase converting aminoSA into AHBA was purified to homogeneity and the gene encoding it was cloned from *A. mediterranei* genomic DNA by reverse genetics. ${}^{105}$ This gene was then used as a probe to isolate cosmids from a cosmid library of *A. mediterranei* DNA which carried this gene and other rifamycin biosynthetic genes. Further chromosome walking allowed the sequencing and analysis of about 96 kb of DNA contiguous with the AHBA synthase gene,

which accounted for most of the genes considered necessary for rifamycin biosynthesis. ${}^{106}$ The rif PKS was also cloned independently by Schupp et al., ${}^{107}$ and the cluster was subsequently expanded to include additional genes located upstream of *rifA* (see no. 5) ${}^{108}$ (Figure 7, Table 1).

The *rif* cluster contains a set of genes, *rifG* through *rifN*, which were shown by inactivation and heterologous expression to be involved in the biosynthesis of AHBA. ${}^{109}$ The role of three of these genes, *rifL*, *rifM*, and *rifN*, was obscure for some time, but recent work indicates that the formation of aminoDAHP is much more complicated than originally anticipated ${}^{103}$ and invokes participation of these three genes, as well as *rifK*, in a second role as a transaminase, in the formation of the AHBA precursor, kanosamine (Figure 8). ${}^{110–112}$ Five large open reading frames, *rifF* through *rifE*, encode a type I modular polyketide synthase (PKS) ${}^{106,107,113}$ with a loading module ${}^{114}$ which was identified as a non-ribosomal peptide synthase (NRPS) adenylation/thiolation didomain; ${}^{115,116}$ that is, the *rif* PKS is actually a hybrid NRPS/PKS. *rifA–E* are responsible for the assembly of a linear undecaketide and are followed by *rifF* encoding an amide synthase which catalyzes the release of this undekaketide and its cyclization to a macrolactam. The Rif F protein was heterologously expressed and purified, and its structure was modeled by the group of Sim. ${}^{117}$ The function of the amide synthase as the terminating enzyme was demonstrated by inactivation of the *rifF* gene, which, surprisingly, led to the accumulation of a series of linear ketides ranging from a tetraketide to the undekaketide. ${}^{118,119}$ It was subsequently found that traces of the same ketides are present in fermentations of the wild-type organism and that a type II thioesterase (encoded by *rifR*) is not responsible for the premature shedding of these assembly intermediates from the NRPS/PKS. ${}^{120}$ The structures of the accumulated ketides revealed that the ring closure of the aromatic moiety from a benzenoid to a naphthalenic structure must occur on the PKS between the third and the fourth chain extension steps. The tetraketide has a benzenoid

Table 1. Homologies and Putative Functions of rif Genes from Amycolatopsis mediterranei S699

| ORF       | properties or content                                                                                   | similarity                                                                                                      |
|-----------|--------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------|
| RplJ      | ribosomal protein L10 (191aa)                                                                           | RplJ(Z92772:66%) Mycobacterium tuberculosis(strainH37RV)                                                         |
| RplL      | ribosomal protein L7/L12 (127aa)                                                                        | RplL(Z92772:70%) M.tuberculosis (strainH37RV)                                                                    |
| Orf21     | possible ABC transport ATP-binding protein (390aa)                                                       | (Z95972:73%) hypothetical protein                                                                                |
|           |                                                                                                        | Rv0655-M.tuberculosis(strainH37RV)                                                                               |
| Orf22     | putative ABC transporter integral membrane protein (250aa)                                               | (SCC42.02C:76%) putative ABC transporter integral membrane protein-S.coelicolor A3(2)]                          |
| Orf23     | putative ABC transporter permease protein (278aa)                                                        | (AL022073:48%) hypothetical protein Rv1965-                                                                      |
|           | (Rv0168,289aa/50%;Rv1965,271aa/48%; Rv3500c,280aa/51%;Rv0588,295aa/54%)                                     | M.tuberculosis(strainH37RV)                                                                                     |
| Orf24     | putative secreted protein: virulence factor mce family protein (mce1, 454aa/32%; mce2, 404aa/29%; mce3,      | (SC8A2.07C:43%) putative secreted protein-                                                                       |
|           | 425aa/31%; mce4, 400aa/30%)                                                                            | S.coelicolor A3(2)                                                                                               |
| Orf25     | putative lipoprotein: virulence factor mce family protein (Rv0170,346aa/38%;Rv1967,342aa/37%;Rv3498c,         | (SC8A2.06C:54%) putative secreted protein-                                                                       |
|           | 350aa/36%;Rv0590,275aa/40%)                                                                             | S.coelicolor A3(2)                                                                                               |
| Orf26     | putative lipoprotein: virulence factor mce family protein (Rv1968,410aa/37%;Rv0171,515aa/33%;Rv3497c,         | (SC8A2.05C:51%) putative secreted protein-                                                                       |
|           | 357aa/32%;Rv0591,481aa/30%)                                                                             | S.coelicolor A3(2)                                                                                               |
| Orf27     | putative secreted protein: virulence factor mce family protein (Rv1969,423aa/35%;Rv0172,530aa/31%;            | (SC8A2.04C:45%) putative secreted protein-                                                                       |
|           | Rv3496c,451aa/36%;Rv0592,508aa/33%)                                                                     | S.coelicolor A3(2)                                                                                               |
| Orf28     | putative secreted protein: virulence factor mce family protein (lprM,377aa/36%;lprN,384aa/37%;               | (SC8A2.03C:53%) putative secreted protein-                                                                       |
|           | lprK,390aa/35%;lprL,402aa/31%)                                                                          | S.coelicolor A3(2)                                                                                               |
| Orf29     | putative secreted protein: virulence factor mce family protein (Rv1971,437aa/33%;Rv3494c,                    | (SC8A2.02C:42%) putative secreted protein-                                                                       |
|           | 564aa/31%;Rv0174,515aa/31%;Rv0594,516aa/31%)                                                             | S.coelicolor A3(2)                                                                                               |
| Orf30     | putative membrane protein (RNA polymerase sigma-54 factor, RpoN) (191aa)                                   | (SC4A7.39C:31%) putative membrane protein-                                                                       |
|           |                                                                                                        | S.coelicolor A3(2)                                                                                               |
| Orf31     | putative integral membrane protein: similar to zinc finger type transcription factor MZF-3 (346aa)          | (SC4A7.38C:36%) putative integral membrane protein-S.coelicolor A3(2)                                            |
| Orf32c    | conserved hypothetical protein: a member of the lipocalin superfamily (340aa)                             | (AL096743:29%) conserved hypothetical protein-                                                                   |
|           |                                                                                                        | S.coelicolor A3(2)                                                                                               |
| RifS      | putative NADH-dependent dehydrogenase (322aa)                                                            | (AP004604:28%) NADH-dependent dehydrogenase [Oceanobacillus iheyensis]                                           |
| RifT      | putative NADH-dependent dehydrogenase (255aa)                                                            | (AP004604:23%) NADH-dependent dehydrogenase [Oceanobacillus iheyensis]                                           |
| Orf35     | hypothetical protein (75aa)                                                                            | (AL138668:32%) hypothetical protein SC4A9.08-                                                                     |
|           |                                                                                                        | S.coelicolor A3(2)                                                                                               |
| Orf0      | cytochrome-P450-like protein (396aa)                                                                    | (M31939:41%) cytochrome-P450-like protein (choP)                                                                  |
|           |                                                                                                        | [Streptomyces sp.]                                                                                              |
| RifA      | rifamycin polyketide synthase protein (Loading domain: AD, ACP.Module1:KS, AT, DH*, KR, ACP.Module2:KS, ATm, ACP.Module3: KS, AT, KR*, ACP) (4735aa) |                                                                                                                 |
| RifB      | rifamycin polyketide synthase protein (Module 4: KS, AT, DH, KR, ACP.Module5:KS, AT, DH*, KR, ACP.Module6:KS, AT, DH, KR, ACP) (5060aa) |                                                                                                                 |
| RifC      | rifamycin polyketide synthase protein (Module 7:KS, AT, DH, KR, ACP) (1763aa)                              |                                                                                                                 |
| RifD      | rifamycin polyketide synthase protein (Module 8:KS, AT, DH, KR, ACP) (1728aa)                              |                                                                                                                 |
| RifE      | rifamycin polyketide synthase protein (Module 9: KS, ATm, DH, KR, ACP.Module10: KS, AT, DH, KR, ACP) (3413aa) |                                                                                                                 |
| RifF      | amide synthase (N-acyl transferase) (260aa)                                                             | (AF453501:36%) amide synthase-Actinosynnema pretiosum subsp.auranticum                                           |
| Orf1      | hypothetical protein (62aa)                                                                            |                                                                                                                 |
| RifG      | aminodehydroquinate synthase (351aa)                                                                    | (AF131877:73%) aminodehydroquinate synthase-Streptomyces collinus                                                |
| RifH      | aminoDAHP synthase (441aa)                                                                             | (AF131877:61%) amino-deoxyarabinohepulosonate-7-phosphate synthase-Streptomyces collinus                         |
| RifI      | aminoquinate dehydrogenase (263aa)                                                                      | (AF131877:70%) shikimate/quinate dehydrogenase-Streptomyces collinus                                             |
| RifK      | AHBA synthase (388aa)                                                                                  | (AF131879:71%) aminohydroxybenzoic acid synthase-Streptomyces collinus                                            |
| RifL      | oxidoreductase (358aa)                                                                                 | (AE013146:30%) predicted dehydrogenases and related proteins-Thermoanaerobacter tengcongensis                      |
| RifM      | phosphatase (232aa)                                                                                    | (AE007074:33%) hydrolase, haloacid dehalogenase-like family-Mycobacterium tuberculosis CDC1551                    |
| RifN      | kanosamine kinase (235aa)                                                                              | (AF131877:59%) NapI kinase-Streptomyces collinus                                                                  |

Table 1 (Continued)

| ORF          | properties or content                                                                 | similarity                                                                                                                                                                                                 |
|--------------|-------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| RifO         | putative regulatory protein                                                         | (AL450350:34%) uncharacterized lmbE-like protein—S.coelicolor A3(2)                                                                                                                                         |
| (255aa)      |                                                                                     |                                                                                                                                                                                                         |
| Orf 2        | putative esterase                                                                  | (U70619:50%) heroin esterase—Rhodococcus sp.                                                                                                                                                             |
| (310aa)      |                                                                                     |                                                                                                                                                                                                         |
| RifP         | efflux transporter protein                                                          | (AB019519:73%) VarS—Streptomyces virginiae                                                                                                                                                               |
| (522aa)      |                                                                                     |                                                                                                                                                                                                         |
| RifQ         | putative tetR-like transcription regulatory protein                                  | (AB046994:64%) VarR—Streptomyces virginiae                                                                                                                                                               |
| (242aa)      |                                                                                     |                                                                                                                                                                                                         |
| Orf3c        | hypothetical protein (ATP-binding protein)                                          | (AL392178:44%) conserved hypothetical protein—S.coelicolor A3(2)                                                                                                                                          |
| (166aa)      |                                                                                     |                                                                                                                                                                                                         |
| Orf4c        | putative cytochrome P450 oxidoreductase                                              | (AF072709:51%) putative cytochrome P450 oxidoreductase—Streptomyces lividans                                                                                                                               |
| (403aa)      |                                                                                     |                                                                                                                                                                                                         |
| Orf5c        | cytochrome P450 monooxygenase                                                       | (M54983:36%) EryF:6-deoxyerythronolide B hydroxylase (6-DEB hydroxylase, erythromycin A biosynthesis hydrolase) (cytochrome P450107A1) (CYPCVIIIA1) (P450eryF)—Saccharopolyspora erythraea |
| (421aa)      |                                                                                     |                                                                                                                                                                                                         |
| Orf6c        | dNTP-hexose dehydratase                                                            | (AF269227:71%) NDP-hexose 3,4-dehydratase UrdQ—Streptomyces fradiae                                                                                                                                      |
| (435aa)      |                                                                                     |                                                                                                                                                                                                         |
| Orf 7        | dNTP-hexose glycosyl transferase                                                    | (AF164960:43%) glycosyl transferase—Streptomyces fradiae                                                                                                                                                 |
| (381aa)      |                                                                                     |                                                                                                                                                                                                         |
| Orf 8        | dNTP-hexose 3,5-epimerase                                                           | (AJ006985:49%) StrM:dTDP-4-keto-6-deoxyglucose 3,5-epimerase—Streptomyces glaucescens                                                                                                                     |
| (214aa)      |                                                                                     |                                                                                                                                                                                                         |
| Orf9c        | aminotransferase                                                                   | (AB005901:58%) deduced aminotransferase—Streptomyces kasugaensis                                                                                                                                          |
| (430aa)      |                                                                                     |                                                                                                                                                                                                         |
| Orf10c       | probable dNDP-hexose-3-ketoreductase                                                | (AF080235:60%) oxidoreductase homologue—Streptomyces cyanogenus                                                                                                                                          |
| (330aa)      |                                                                                     |                                                                                                                                                                                                         |
| Orf11        | flavin-dependent oxidoreductase                                                    | (U67594:28%) N5,N10-methylene-tetrahydromethanopterin reductase(mer)—Methanococcus jannaschii                                                                                                             |
| (321aa)      |                                                                                     |                                                                                                                                                                                                         |
| Orf17        | alkanal monooxygenase α-chain                                                      | (X58791:27%) luciferase α subunit—Vibrio harveyi                                                                                                                                                        |
| (356aa)      |                                                                                     |                                                                                                                                                                                                         |
| Orf18        | putative 2,3-dehydratase                                                           | (AF237895:59%) dTDP-4-keto-6-deoxyglucose 2,3-dehydratase—Streptomyces antibioticus                                                                                                                        |
| (473aa)      |                                                                                     |                                                                                                                                                                                                         |
| Orf19c       | 3-(3-hydroxyphenyl) propionate hydroxylase                                           | (AP005277:40%) 2-polyprenyl-6-methoxyphenol hydroxylase and related FAD-dependent oxidoreductases—Corynebacterium glutamicum ATCC 13032                                                                 |
| (501aa)      |                                                                                     |                                                                                                                                                                                                         |
| Orf20c       | 25-O-acetyltransferase                                                             | (Z83857:31%) papA5—Mycobacterium tuberculosis H37Rv                                                                                                                                                      |
| (403aa)      |                                                                                     |                                                                                                                                                                                                         |
| Orf12c       | thioesterase                                                                       | (AB070940:55%) thioesterase—Streptomyces avermitilis                                                                                                                                                     |
| (RifR)       |                                                                                     |                                                                                                                                                                                                         |
| (259aa)      |                                                                                     |                                                                                                                                                                                                         |
| Orf13c       | cytochrome P450 monooxygenase                                                       | (M54983:36%) EryF:6-deoxyerythronolide B hydroxylase (6-DEB hydroxylase, erythromycin A biosynthesis hydrolase) (cytochrome P450107A1) (CYPCVIIIA1) (P450eryF)—Saccharopolyspora erythraea |
| (422aa)      |                                                                                     |                                                                                                                                                                                                         |
| Orf14        | 27-O-methyltransferase                                                             | (AB090952:50%) putative D-glucose O-methyltransferase—Lechevalieria aerocolonigenes                                                                                                                       |
| (272aa)      |                                                                                     |                                                                                                                                                                                                         |
| Orf15        | transketolase                                                                      | (Z29635:55%) orf3—Rhodococcus fascians                                                                                                                                                                   |
| (533aa)      |                                                                                     |                                                                                                                                                                                                         |
| Orf16c       | cytochrome P450 monooxygenase                                                       | (AF127374:44%) cytochrome P450 hydroxylase ORF4—Streptomyces lavendulae                                                                                                                                    |
| (389aa)      |                                                                                     |                                                                                                                                                                                                         |
| RifJ         | aminoDHQ dehydratase                                                               | (AF127374:74%) MmcF—Streptomyces lavendulae                                                                                                                                                             |
| (163aa)      |                                                                                     |                                                                                                                                                                                                         |
| Orf36        | putative regulatory protein                                                        | (AF534707:34%) putative transcriptional activator RebR—Lechevalieria aerocolonigenes                                                                                                                     |
| (404aa)      |                                                                                     |                                                                                                                                                                                                         |
| Orf37        | hypothetical protein                                                               | (NZ_AAAC01000306:32%) hypothetical protein—Burkholderia fungorum                                                                                                                                          |
| (161aa)      |                                                                                     |                                                                                                                                                                                                         |
| RpoB         | DNA-dependent RNA polymerase β-subunit                                             | (AE006964:80%) DNA-directed RNA polymerase, β subunit—Mycobacterium tuberculosis CDC 1551                                                                                                               |
| (1167aa)     |                                                                                     |                                                                                                                                                                                                         |
| RpoC         | DNA-dependent RNA polymerase β'-subunit                                            |                                                                                                                                                                                                         |

structure, but the penta-to undecaketides carry a naphthoquinone ring. Interestingly, module 4 of the NRPS/PKS does not process a ketide in which the naphthalene ring closure for some reason has not occurred, but releases the corresponding tetraketide. This compound, called P8/1-OG, was first isolated from a mutant of A.mediterranei blocked in rifamy-cin biosynthesis, and the corresponding analogues

of P8/1-OG were obtained when a rifK(-) mutant was complemented with AHBA analogues. Heterologous expression of rifA in E.coli has recently been reported, leading to the production of P8/1-OG, albeit in very low yield. A gene in the rif cluster, riforf19, has been tentatively identified as being involved in the ring closure reaction; its inactivation leads to the accumulation of P8/1-OG.

Rifamycin—Mode of Action, Resistance, and Biosynthesis

Chemical Reviews, 2005, Vol. 105, No. 2 629

![Diagram](attachment:diagram.png)

Figure 8. Biosynthetic pathway for the rifamycin polyketide starter unit, 3-amino-5-hydroxybenzoic acid (AHBA).

The nature of the first cyclic product released from the *rif* NRPS/PKS has been a matter of controversy. On the basis of mutagenesis experiments, it had been proposed that protorifamycin I, a naphthoquinone derivative lacking the 8-hydroxyl group, is an intermediate in the biosynthesis of rifamycin B.¹²⁴ Although seemingly supported by the 8-deoxynaphthoquinone structures of the penta- to undecaketides accumulated in the *rifF* mutants,¹¹⁸,¹¹⁹ this proposal is incompatible with the finding that one atom of ¹⁸O from the C¹⁸O₂H group of AHBA is retained in rifamycin B.¹¹⁸,¹²⁵ On the basis of this result and of the isolation from the *riff*(−) mutant of a pentaketide with a 7,8-dihydro-8-hydroxynaphthoquinone structure, we proposed the structure of protorifamycin X for the first cyclic PKS product (Figure 9), suggesting that the 8-deoxy compounds are shunt metabolites resulting from spontaneous dehydration.¹¹⁸ Recent work by Stratmann et al. has indeed shown that the 8-deoxy compounds are intermediates on a shunt pathway to 8-deoxyrifamycins, rather than rifamycin B precursors.¹²⁶

The *rif* cluster also contains several potential regulatory genes and a set of genes predicted to encode the formation of a sugar nucleotide which, however, appear to be silent. Also present are a substantial number of genes apparently responsible for the modification of the original polyketide during or after its assembly. Their functional analysis by gene inactivation and heterologous expression is in progress.¹²³ The gene *rif orf14* encodes a methyltransferase which has been shown to use 27-O-demethylrifamycin SV, not its quinone or its 25-O-desacetyl derivative, as substrate, shedding some light on the late stages of rifamycin formation.¹²⁷ The biosynthetic pathway to rifamycin B and suggested

### 5. Autoresistance of *Amycolatopsis Mediterranei*

Notably, the *rif* cluster does not contain any obvious candidates for genes conferring resistance on *A. mediterranei* to its own antibiotic,¹⁰⁶ a feature of most antibiotic biosynthesis gene clusters.¹²⁸ Experiments with whole cells and with the partially purified DNA-dependent RNA polymerase of *A. mediterranei* have shown that rifamycin resistance is expressed throughout the entire culture period independent of the time of antibiotic production.¹²⁹,¹³⁰ In the process of defining the boundaries of the rifamycin biosynthetic gene cluster in *A. mediterranei*, we found the *rpoB* gene to be located on a 3.9 kb DNA fragment on the right-hand side of the *rif* gene cluster.¹⁰⁸ Sequencing revealed an 1168 amino acid open reading frame with 81% identity to the *M. tuberculosis rpoB*, followed by the 5′ end of the *rpoC* gene. Southern hybridization revealed that these represented the only copies of these genes in the *A. mediterranei* genome and must thus represent the respective housekeeping genes. Cloning of the *A. mediterranei rpoB* gene into rifampicin-sensitive *M. smegmatis* conferred rifampicin resistance upon the organism. The same was observed when the rif I region of the *M. tuberculosis rpoB* was replaced with that from the *A. mediterranei rpoB* and introduced into *M. smegmatis*.¹⁰⁸ In the rif I region of *A. mediterranei rpoB*, five amino acids, Q432, T434, I437, D438, and N447, differ from their counterparts in the *rpoB* from wild-type *M. tuberculosis* (Figure 5). Three of these, Q432, D438, and N447, were sufficient to confer rifampicin resistance upon *A. mediterranei*; a triple mutation, Q432D, D438S, and N447S, in the *rpoB* gene resulted in a high level

630 Chemical Reviews, 2005, Vol. 105, No. 2

![Diagram](attachment:diagram.png)

Figure 9. Biosynthetic pathway to rifamycin B and the established or proposed role of individual *rif* biosynthetic genes.

of rifampicin sensitivity (MIC 0.01 μg/mL) in a rifamycin nonproducing mutant of A. mediterranei. Site-specific mutagenesis of the corresponding positions in the M. tuberculosis *rpoB* gene and expression in M. smegmatis showed that in fact each one of these three amino acid substitutions, D430Q, S436D, and S445N (*M*. t. numbering), alone was sufficient to confer resistance upon that organism. Autoresistance of the producing organism is thus predominantly, if not exclusively, due to a rifamycin-insensitive DNA-dependent RNA polymerase. ${}^{108}$

The arrangement of the *rpoB/C* genes and the genes *rplJ* and *rplL*, encoding ribosomal proteins, is highly conserved in archaeabacteria and eubacteria. ${}^{131-137}$ It was therefore a question where on the A. mediterranei genome the *rplJ* and *rplL* genes are located. They were found 16.6 kb upstream of the *rifA* gene. The region between them and *rifA* contains 12 genes encoding transporter-related lipoproteins, which are likely to be involved in antibiotic efflux, and several post-PKS processing genes (Figure 7). ${}^{108}$ Thus, in A. mediterranei the entire rifamycin biosynthetic gene cluster is inserted between some of the genes encoding the cellular machinery targeted by the antibiotic. Several other bacterial strains producing rifamycin-related antibiotics were analyzed for the arrangement of the *rplL–rpoB* genes. In four non-*Amycolatopsis* strains, *Micromonospora lacustris*, *Micromonospora nigra*, and two *Strepto*-

myces species producing streptovaricins and awamy- cin, respectively, the *rplL* and *rpoB* genes were found to be closely linked. However, in *Amycolatopsis tolypomycina* and *A. vancoremycina*, no such linkage was detectable, suggesting that their antibiotic biosynthesis gene clusters are also located in the intergenic region between *rplL* and *rpoB*, as in *A. mediterranei*. All six organisms showed pronounced rifamycin resistance and carried amino acid substitutions in the *rifI* region consistent with a rifamycin-insensitive *rpoB*. ${}^{108}$

Interestingly, the presence or absence of rifamycin production and resistance in *A. mediterranei* has pronounced effects on growth, susceptibility to phage infection, and spore production. Under laboratory culture conditions, spore production in a rifamycin nonproducing mutant is delayed by a moderate supplement of rifamycin. The rifamycin nonproducing mutant also revealed a higher sensitivity to phage infection, particularly in the rifamycin-sensitive strains that carry a mutated *rpoB* allele. These results could suggest a mediator role for rifamycins. ${}^{108}$

## 6. Concluding Remarks

Like most colonial organisms, *A. mediterranei* exploits elaborate systems of intra- and intercellular communication to facilitate the adaptation to changeable environmental conditions. The messages by

which bacteria communicate take the form of chemical signals released from the cells which can elicit profound physiological changes. In the transcription process, the cellular RNA polymerase operates as a complex molecular machine with extensive interactions with the template DNA, the product RNA, and regulatory molecules. It seems plausible that many distinct sites exist where the binding of a mediator molecule, such as rifamycin, could switch critical features of the functional mechanism. Indeed, rifamycin is active against a large variety of organisms, including many bacteria, eukaryotes, and viruses. It is, therefore, possible that rifamycin represents a widely recognized ancient signaling molecule and regulates diverse behaviors across distant genera. The discovery that the rifamycin biosynthetic gene cluster is closely linked to the housekeeping genes encoding the ribosomal proteins and the RNA polymerase subunits could provide an alternative view of the natural role of this broadly used antimicrobial agent.

### 7. Acknowledgment

Work from the authors’ laboratory was supported by NIH Grant AI20264. We thank Mrs. Kay B. Kampsen for her help in the preparation of this manuscript.

### 8. References

(1) The name rifamycin (originally rifomycin) is derived from the title of a French movie, Rififi, popular at the time of the antibiotic’s discovery.²

(2) Sensi, P. *Rev. Infect. Dis.* **1983**, *5* (Suppl. 3), S402.

(3) Rinehart, K. L., Jr.; Shields, L. S. *Fortschr. Chem. Org. Naturst.* **1976**, *33*, 231.

(4) Wehrli, W. *Top. Curr. Chem.* **1977**, *72*, 21.

(5) Prelog, V.; Oppolzer, W. *Helv. Chim. Acta* **1973**, *56*, 2279.

(6) Balerna, M.; Keller-Schierlein, W.; Martius, C.; Wolf, H.; Zähner, H. *Arch. Mikrobiol.* **1969**, *65*, 303.

(7) Rinehart, K. L., Jr.; Maheshwari, M. L.; Antosz, F. J.; Mathur, H. H.; Sasaki, K.; Schacht, R. *J. Am. Chem. Soc.* **1971**, *93*, 6273.

(8) Deboer, C.; Meulman, P. A.; Wnuk, R. J.; Peterson, D. H. *J. Antibiot.* **1970**, *23*, 442.

(9) Higashide, E.; Asai, M.; Ootsu, K.; Tanida, S.; Kozai, Y.; Hasegawa, T.; Kishi, T.; Sugino, Y.; Yoneda, M. *Nature* **1977**, *270*, 721.

(10) Sensi, P.; Margalith, P.; Timbal, M. T. *Farmaco, Ed. Sci.* **1959**, *14*, 146.

(11) Margalith, P.; Beretta, G. *Mycopathol. Mycol. Appl.* **1960**, *13*, 321.

(12) Thiemann, J. E.; Zucco, G.; Pelizza, G. *Arch. Mikrobiol.* **1969**, *67*, 147.

(13) Lechevalier, M. P.; Prauser, H.; Labeda, D. P.; Ruan, J.-S. *Int. J. Syst. Bacteriol.* **1986**, *36*, 29.

(14) Margalith, P.; Pagani, H. *Appl. Microbiol.* **1961**, *9*, 320, 325.

(15) Oppolzer, W.; Prelog, V.; Sensi, P. *Experientia* **1964**, *20*, 336.

(16) Brufani, M.; Fideli, W.; Giacomello, G.; Vaciago, A. *Experientia* **1964**, *20*, 339.

(17) Leitich, J.; Oppolzer, W.; Prelog, V. *Experientia* **1964**, *20*, 343.

(18) Oppolzer, W.; Prelog, W. *Helv. Chim. Acta* **1973**, *56*, 2287.

(19) Lancini, G.; Cavalleri, B. In *Biotechnology of Antibiotics*; Strohl, W. R., Ed.; Marcel Dekker: New York, 1997; p 521.

(20) Kishi, T.; Yamana, H.; Muroi, M.; Harada, S.; Asai, M.; Hasegawa, T.; Mizuno, K. *J. Antibiot.* **1972**, *25*, 11.

(21) Ganguly, A. K.; Szmulewicz, S.; Sarre, O. Z.; Greeves, D.; Morton, J.; McGlotten, J. *J. Chem. Soc.*, *Chem. Commun.* **1974**, 395.

(22) Wehrli, W.; Staehelin, M. *Bacteriol. Rev.* **1971**, *35*, 290.

(23) Sensi, P.; Ballotta, R.; Greco, A. M.; Gallo, G. G. *Farmaco, Ed. Sci.* **1961**, *16*, 165.

(24) Maggi, N.; Pasqualucci, C. R.; Ballotta, R.; Sensi, P. *Chemo- therapia* **1966**, *11*, 285.

(25) Sensi, P.; Thiemann, J. E. *Prog. Ind. Microbiol.* **1967**, *6*, 21.

(26) Bergamini, N.; Fowst, G. *Arzneim.-Forsch.* **1965**, *15*, 951.

(27) Roos, R.; Ghisalba, O. *Experientia* **1980**, *36*, 486.

(28) Lancini, G. C.; Hengeller, C. *J. Antibiot.* **1969**, *22*, 637.

(29) Lancini, G.; Zanichelli, W. In *Structure–activity relationships among the semisynthetic antibiotics*; Perlman, D., Ed.; Academic Press: New York 1977; p 531.

(30) Traxler, P.; Vischer V. A.; Zak, O. *Drugs Future* **1988**, *13*, 845.

(31) Binda, G.; Domenichini, E.; Gottardi, A.; Orlandi, B.; Ortelli, E.; Pacini, B.; Fowst, G. *Arzneim.-Forsch.* **1971**, *21*, 1907.

(32) Shinnick, T., Ed. *Current Topics in Microbiology and Immunology*; Academic Press: New York, 1996.

(33) Wehrli, W. *Rev. Infect. Dis.* **1983**, *5* (Suppl. 3), S407.

(34) Hobby, G. L.; Lenert, T. F. *Am. Rev. Respir. Dis.* **1970**, *102*, 462.

(35) Brogden, R. N.; Fitton, A. *Drugs* **1994**, *47*, 983.

(36) Mealy, N. E. *Drugs Future* **1979**, *4*, 255.

(37) Calvori, C.; Frontali, L.; Leoni, L.; Tecce, G. *Nature* **1965**, *207*, 417.

(38) Hartmann, G.; Honikel, K. O.; Knüsel, F.; Nüesch, J. *Biochim. Biophys. Acta* **1967**, *145*, 843.

(39) Hartmann, G.; Behr, W.; Beissner, K.-A.; Honikel, K.; Sippel, A. *Angew. Chem., Int. Ed. Engl.* **1968**, *7*, 693.

(40) Meilhac, M.; Tysper, Z.; Chambon, P. *Eur. J. Biochem.* **1972**, *28*, 291.

(41) Campbell, E. A.; Khorzheva, N.; Mustaev, A.; Murakami, K.; Nair, S.; Goldfarb, A.; Darst, S. A. *Cell* **2001**, *104*, 901.

(42) Sippel, A.; Hartmann, G. *Biochim. Biophys. Acta* **1968**, *157*, 218.

(43) Umezawa, H.; Mizuno, S.; Yamasaki, H.; Nitta, K. *J. Antibiot.* **1968**, *21*, 234.

(44) Sippel, A.; Hartmann, G. *Eur. J. Biochem.* **1970**, *16*, 152.

(45) Rabussay, D.; Zillig, W. *FEBS Lett.* **1969**, *5*, 104.

(46) Heil, A.; Zillig, W. *FEBS Lett.* **1970**, *11*, 165.

(47) Wehrli, W.; Handschin, J.; Wunderli, W. In *RNA polymerase*; Losick, R., Chamberlin, M., Eds.; Cold Spring Harbor Laboratory: Cold Spring Harbor, NY, 1976; p 397.

(48) Naryshkina, T.; Mustaev, A.; Darst, S. A.; Severinov, K. *J. Biol. Chem.* **2001**, *276*, 13308.

(49) McClure, W. R.; Cech, C. L. *J. Biol. Chem.* **1978**, *253*, 8949.

(50) Schulz, W.; Zillig, W. *Nucleic Acids Res.* **1981**, *9*, 6889.

(51) Zhang, G.; Campbell, E. A.; Minakhin, L.; Richter, C.; Severinov, K.; Darst, S. A. *Cell* **1999**, *98*, 811.

(52) Korzheva, N.; Mustaev, A.; Kozlov, M.; Malhotra, A.; Nikiforov, V.; Goldfarb, A.; Darst, S. A. *Science* **2000**, *289*, 619.

(53) Mustaev, A.; Zaychikov, E.; Severinov, K.; Kashlev, M.; Polyakov, A.; Nikiforov, V.; Goldfarb, A. *Proc. Natl. Acad. Sci. U.S.A.* **1994**, *91*, 12036.

(54) Severinov, K.; Mustaev, A.; Severinova, E.; Kozlov, M.; Darst, S. A.; Goldfarb, A. *J. Biol. Chem.* **1995**, *270*, 29428.

(55) Brufani, M.; Cerrini, S.; Fedeli, W.; Vaciago, S. *J. Mol. Biol.* **1974**, *87*, 409.

(56) Bartolucci, C.; Cellai, L.; Di Filippo, P.; Segre, A.; Brufani, M.; Filocamo, L.; Bianco, A. D.; Guiso, M.; Brizzi, V.; Benedetito, A. D.; Di Caro, A.; Elia, G. *Il Farmaco* **1992**, *47*, 1367.

(57) Lal, R.; Lal, S. *Bioessays* **1994**, *16*, 211.

(58) Grosset, J.; Leventis, S. *Rev. Infect. Dis.* **1983**, *5* (Suppl. 3), S440.

(59) Calleja, C.; Pascussi, J. M.; Mani, J. C.; Maurel, P.; Vilarem, M. *J. Nature Medicine* **1998**, *4*, 92.

(60) Fardel, O.; Lecureur, V.; Loyer, P.; Guillouzo, A. *Biochem. Pharmacol.* **1995**, *49*, 1255.

(61) Courtois, A.; Payen, L.; Vernhet, L.; deVries, E. G. E.; Guillouzo, A.; Fardel, O. *Cancer Lett.* **1999**, *139*, 97.

(62) David, H. L. *Appl. Microbiol.* **1970**, *20*, 810.

(63) Gillespie, S. H. *Antimicrob. Agents Chemother.* **2002**, *46*, 267.

(64) Ezekiel, D. H.; Hutchins, J. E. *Nature* **1968**, *220*, 276.

(65) Wehrli, W.; Nüesch, J.; Knüsel, F.; Staehelin, M. *Biochem. Biophys. Res. Commun.* **1968**, *32*, 284.

(66) Telenti, A.; Imboden, P.; Marchesi, F.; Lowrie, D.; Cole, S.; Colston, M. J.; Matter, L.; Schopfer, K.; Bodmer, T. *Lancet* **1993**, *341*, 647.

(67) Ramaswamy, S.; Musser, J. M. *Tuberc. Lung Dis.* **1998**, *79*, 3.

(68) Heep, M.; Rieger, U.; Beck, D.; Lehn, N. *Antimicrob. Agents Chemother.* **2000**, *44*, 1075.

(69) Heep, M.; Beck, D.; Bayerdörffer, E.; Lehn, N. *Antimicrob. Agents Chemother.* **1999**, *43*, 1497.

(70) Ovchinnikov, Y. A.; Monastyrsкая, G. S.; Gubanov, V. V.; Guryev, S. O.; Chertov, O. Y.; Modyanov, N. N.; Grinkevich, V. A.; Makarova, I. A.; Marchenko, T. V.; Polovnikova, I. N.; Lipkin, V. M.; Sverdlov, E. D. *Eur. J. Biochem.* **1981**, *116*, 621.

(71) Ovchinnikov, Y. A.; Monastyrsкая, G. S.; Guriev, S. O.; Kalinina, N. F.; Sverdlov, E. D.; Gragerov, A. I.; Bass, I. A.; Kiver, I. F.; Moiseyeva, E. P.; Igumnov, V. N.; Mindlin, S. Z.; Nikiforov, V. G.; Khesin, R. B. *Mol. Gen. Genet.* **1983**, *190*, 344.

(72) Lisitsyn, N. A.; Guriev, S. O.; Sverdlov, E. D.; Moiseeva, E. P.; Nikiforov, V. G. *Bioorg. Khim.* **1984**, *10*, 127.

(73) Lisitsyn, N. A.; Sverdlov, E. D.; Moiseyeva, E. P.; Danilevskaya, O. N.; Nikiforov, V. G. *Mol. Gen. Genet.* **1984**, *196*, 173.

(74) Jin, J. D.; Gross, C. A. *J. Mol. Biol.* **1988**, *202*, 45.

(75) Severinov, K.; Souchko, M.; Goldfarb, A.; Nikiforov, V. *J. Biol. Chem.* **1993**, *268*, 14820.

(76) Severinov, K.; Souchko, M.; Goldfarb, A.; Nikiforov, V. *Mol. Gen. Genet.* **1994**, *244*, 120.

(77) Honoré, N.; Cole, S. T. *Antimicrob. Agents Chemother.* **1993**, *37*, 414.

(78) Carter, P. E.; Abadia, F. J. R.; Yakubu, D. E.; Pennington, T. H. Antimicrob. Agents Chemother. 1994, 38, 1256.

(79) Yee, Y. C.; Kisslinger, B.; Yu, V. L.; Jin, D. J. Antimicrob. Chemother. 1996, 38, 133.

(80) Alekshun, M.; Kashlev, M.; Schwartz, I. Gene 1997, 186, 227.

(81) Drancourt, M.; Raoult, D. Antimicrob. Agents Chemother. 1999, 43, 2400.

(82) Heep, M.; Odenbreit, S.; Beck, D.; Decker, J.; Prohaska, E.; Rieger, U.; Lehn, N. Antimicrob. Agents Chemother. 2000, 44, 1713.

(83) Hetherington, S. V.; Watson, A. S.; Patrick, C. C. Antimicrob. Agents Chemother. 1995, 39, 2164.

(84) Hui, J.; Gordon, N.; Kajoika, R. Antimicrob. Agents Chemother. 1977, 11, 773.

(85) Abadi, F. J.; Carter, P. E.; Cash, P.; Pennington, T. H. Antimicrob. Agents Chemother. 1996, 40, 645.

(86) Chandrasekaran, S.; Lalithakumari, D. J. Med. Microbiol. 1998, 47, 197.

(87) Wehrli, W.; Zimmermann, W.; Kump, W.; Tosch, W.; Vischer, W.; Zak, O. J. Antibiot. 1987, 40, 1733.

(88) Pugsley, A. P.; Zimmermann, W.; Wehrli, W. J. Gen. Microbiol. 1987, 133, 3505.

(89) Braun, V. Drug Resist. Updates 1999, 2, 363.

(90) Ferguson, A. D.; Ködding, J.; Walker, G.; Bös, C.; Coulton, J. W.; Diederichs, K.; Braun, V.; Welte, W. Structure 2001, 9, 707.

(91) Dabbs, E. R. FEMS Microbiol. Lett. 1987, 44, 395.

(92) Andersen, S. J.; Dabbs, E. R. FEMS Microbiol. Lett. 1991, 79, 247.

(93) Quan, S.; Venter, H.; Dabbs, E. R. Antimicrob. Agents Chemother. 1997, 41, 2456.

(94) Morisaki, N.; Iwasaki, S.; Yazawa, K.; Mikami, Y.; Maeda, A. J. Antibiot. 1993, 46, 1605.

(95) Dabbs, E. R.; Yazawa, K.; Mikami, Y.; Miyaji, M.; Morisaki, N.; Iwasaki, S.; Furihata, K. Antimicrob. Agents Chemother. 1995, 39, 1007.

(96) Tribuddharat, C.; Fennewald, M. Antimicrob. Agents Chemother. 1999, 43, 960.

(97) Arlet, G.; Nadjar, D.; Herrmann, J.-L.; Donay, J.-L.; Rouveau, M.; Lagrange, P. H.; Philippon, A. Antimicrob. Agents Chemother. 2001, 45, 2971.

(98) Girlich, D.; Poirel, L.; Leelaporn, A.; Karim, A.; Tribuddharat, C.; Fennewald, M.; Nordmann, P. J. Clin. Microbiol. 2001, 39, 175.

(99) Naas, T.; Mikami, Y.; Imai, T.; Poirel, L.; Nordmann, P. J. Bacteriol. 2001, 183, 235.

(100) Yazawa, K.; Mikami, Y.; Maeda, A.; Akao, M.; Morisaki, N.; Iwasaki, S. Antimicrob. Agents Chemother. 1993, 37, 1313.

(101) Yazawa, K.; Mikami, Y.; Maeda, A.; Morisaki, N.; Iwasaki, S. J. Antimicrob. Agents Chemother. 1994, 33, 1127.

(102) Tanaka, Y.; Yazawa, K.; Dabbs, E. R.; Nishikawa, K.; Komaki, H.; Mikami, Y.; Miyaji, M.; Morisaki, N.; Iwasaki, S. Microbiol. Immunol. 1996, 40, 1.

(103) Floss, H. G. Nat. Prod. Rep. 1997, 14, 433.

(104) Kim, C.-G.; Kirschning, A.; Bergon, P.; Zhou, P.; Su, E.; Sauerbrei, B.; Ning, S.; Ahn, Y.; Breuer, M.; Leistner, E.; Floss, H. G. J. Am. Chem. Soc. 1996, 118, 7486.

(105) Kim, C.-G.; Yu, T.-W.; Fryhle, C. B.; Handa, S.; Floss, H. G. J. Biol. Chem. 1998, 273, 6030.

(106) August, P. R.; Tang, L.; Yoon, Y. J.; Ning, S.; Müller, R.; Yu, T.-W.; Taylor, M.; Hoffmann, D.; Kim, C.-G.; Zhang, X.; Hutchinson, C. R.; Floss, H. G. Chem. Biol. 1998, 5, 69.

(107) Schupp, T.; Toupet, C.; Engel, N.; Goff, S. FEMS Microbiol. Lett. 1998, 159, 201.

(108) Yu, T.-W.; Pogosova-Agadjanyan, E.; Kuan, L.-Y.; Bai, L.; Tin, A. M.; Floss, H. G. Manuscript in preparation.

(109) Yu, T.-W.; Müller, R.; Müller, M.; Zhang, H.; Draeger, G.; Kim, C.-G.; Leistner, E.; Floss, H. G. J. Biol. Chem. 2001, 276, 12546.

(110) Guo, J.; Frost, J. W. J. Am. Chem. Soc. 2002, 124, 528.

(111) Guo, J.; Frost, J. W. J. Am. Chem. Soc. 2002, 124, 10642.

(112) Arakawa, K.; Müller, R.; Mahmud, T.; Yu, T.-W.; Floss, H. G. J. Am. Chem. Soc. 2002, 124, 10644.

(113) Tang, L.; Joon, Y. J.; Choi, C. H.; Hutchinson, C. R. Gene 1998, 216, 255.

(114) Hunziker, D.; Yu, T.-W.; Hutchinson, C. R.; Floss, H. G.; Khosla, C. J. Am. Chem. Soc. 1998, 120, 1092.

(115) Admiraal, S. J.; Walsh, C. T.; Khosla, C. Biochemistry 2001, 40, 6116.

(116) Admiraal, S. J.; Khosla, C.; Walsh, C. T. J. Am. Chem. Soc. 2003, 125, 13664.

(117) Pompeo, F.; Mustaq, A.; Sim, E. Protein Expression Purif. 2002, 24, 138.

(118) Yu, T.-W.; Shen, Y.; Doi-Katayama, Y.; Tang, L.; Park, C.; Moore, B. S.; Hutchinson, C. R.; Floss, G. H. Proc. Natl. Acad. Sci. U.S.A. 1999, 96, 9051.

(119) Stratmann, A.; Toupet, C.; Schilling, W.; Traber, R.; Oberer, L.; Schupp, T. Microbiology 1999, 145, 3365.

(120) Doi-Katayama, Y.; Yoon, Y. J.; Choi, C.-Y.; Yu, T.-W.; Floss, H. G.; Hutchinson, C. R. J. Antibiot. 2000, 53, 484.

(121) Ghisalba, O.; Fuhrer, H.; Richter, W. J.; Moss, S. J. Antibiot. 1981, 34, 58.

(122) Watanabe, K.; Rude, M. A.; Walsh, C. T.; Khosla, C. Proc. Natl. Acad. Sci. U.S.A. 2003, 100, 9774.

(123) Xu, J.; Floss, H. G.; Mahmud, T. Manuscript in preparation.

(124) Ghisalba, O.; Traxler, P.; Nuesch, J. J. Antibiot. 1978, 31, 1124.

(125) Anderson, M. G.; Monypenny, D.; Rickards, R. W.; Rothschild, J. M. J. Chem. Soc., Chem. Commun. 1989, 311.

(126) Stratmann, A.; Schupp, T.; Toupet, C.; Schilling, W.; Oberer, L.; Traber, R. J. Antibiot. 2002, 55, 396.

(127) Xu, J.; Mahmud, T.; Floss, H. G. Arch. Biochem. Biophys. 2003, 411, 277.

(128) Maplestone, R. A.; Stone, J. S.; Williams, D. H. Gene 1992, 115, 151.

(129) Watanabe, S.; Tanaka, K. Biochem. Biophys. Res. Commun. 1976, 72, 522.

(130) Blanco, M. G.; Hardisson, C.; Salas, J. A. J. Gen. Microbiol. 1984, 130, 2883.

(131) Lindahl, L.; Jaskunas, S. R.; Dennis, P. P.; Nomura, M. Proc. Natl. Acad. Sci. U.S.A. 1975, 72, 2743.

(132) Dabbs, E. R. J. Bacteriol. 1984, 159, 770.

(133) Mekhedov, S. L.; Bass, I. A. Mol. Biol. (Moscow) 1986, 20, 92.

(134) Liao, D.; Dennis, P. P. J. Biol. Chem. 1992, 267, 22787.

(135) Clark, M. A.; Baumann, L.; Baumann, P. Curr. Microbiol. 1992, 25, 283.

(136) Aboshkiwa, M.; al-Ani, B.; Coleman, G.; Rowland, G. J. Gen. Microbiol. 1992, 138, 1875.

(137) Kuster, C.; Piepersberg, W.; Distler, J. Mol. Gen. Genet. 1998, 257, 219.

(138) Wink, J. M.; Kroppenstedt, R. M.; Ganguli, B. M.; Nadkarni, S. R.; Schumann, P.; Seibert, G.; Stackebrandt, E. Syst. Appl. Microbiol. 2003, 26, 38.
