
Full text provided by www.sciencedirect.com

ELSEVIER

**Autophagic recycling: lessons from yeast help define the process in plants**

Allison R Thompson and Richard D Vierstra

The autophagic engulfment of cytoplasm and organelles and their delivery to the vacuole have long been speculated to play an essential role in bulk protein turnover in plants. Until recently, however, the importance and the mechanism(s) of action of these processes have remained obscure. Aided by the discovery of numerous orthologs of the yeast AUTOPHAGY (ATG) protein system in *Arabidopsis*, significant advances have been now made in understanding these processes. Both reverse genetic analyses of the *Arabidopsis* ATG genes and the use of the encoded proteins as cytological markers have confirmed the presence of autophagy in plants and have demonstrated its importance in nutrient recycling, especially during senescence and growth under carbon- or nitrogen-limiting conditions.

Addresses  
Department of Genetics, 425 Henry Mall, University of Wisconsin-Madison, Madison, Wisconsin 53706, USA  
Corresponding author: Vierstra, Richard D (vierstra@wisc.edu)

---

Current Opinion in Plant Biology 2005, 8:165–173  

This review comes from a themed issue on  
Plant biotechnology  
Edited by Maurice Moloney and Jim Peacock  

1369-5266/$ – see front matter  
© 2005 Elsevier Ltd. All rights reserved.  
DOI 10.1016/j.pbi.2005.01.013

---

It has also become evident that plants, like other eukaryotes, contain non-selective, high-throughput mechanisms for degrading proteins in bulk. In animals and yeast, these routes employ the lysosome (animals) or the vacuole (yeasts), which are acidic organelles rich in various hydrolytic enzymes, including proteases. Here, bulk cytoplasm and even organelles become sequestered in membrane-bound compartments and are delivered to the lysosome/vacuole for degradation through a process called autophagy for ‘self eating’ (for reviews see [3–5]). In a similar fashion, the plant vacuole, an acidic compartment that can occupy up to 90% of the cell volume and is rich in hydrolases, has been implicated in cellular autophagy [6,7], but how the cargo is delivered to the vacuole has remained unclear. Fortunately, our understanding has accelerated recently with the identification of several autophagic pathways in the yeasts *Saccharomyces cerevisiae* and *Pichia pastoris*, and the discovery of orthologous systems in plants. In this review, we describe these autophagic pathways and highlight recent work in *Arabidopsis thaliana* that shows their importance to plant development and survival under nutrient-limiting conditions.

### Autophagic pathways

Both cytological and genetic analyses indicate that yeasts employ several autophagic routes to deliver cargo to the vacuole for breakdown (*Figure 1*). The simplest is microautophagy, which involves sequestration of the cytoplasm by invagination of the tonoplast, thus forming a small intravacuolar vesicle called the autophagic body. Breakdown of the limiting membrane then exposes the cargo to resident hydrolases, including a variety of acidic proteases and peptidases, nucleases, lipases, and glucanases. Upon the switch from methanol to glucose metabolism, methylotrophic yeasts such as *Pichia* use a modified form of microautophagy called micropexophagy to engulf and remove unwanted peroxisomes. By contrast, macroautophagy involves the entrapment of portions of the cytosol by cytoplasmic vesicles called autophagosomes (*Figure 1*). These vesicles appear to arise from the endoplasmic reticulum as tubules called preautophagic structures (PASs) that encircle cytoplasm in a cage-like structure. Autophagosomes are created when the tubes coalesce to seal the region in a double-membrane-bound compartment. The outer membrane of the autophagosome then fuses with the tonoplast to release the internal vesicle as an autophagic body. In a special case, *S. cerevisiae* and probably other eukaryotes generate a smaller version of the autophagosome to constitutively deliver functional vacuolar proteins to the lumen [8]. It has been speculated that this cytoplasm-to-vacuole transport (CVT) evolved

www.sciencedirect.com  Current Opinion in Plant Biology 2005, 8:165–173

166 Plant biotechnology

Figure 1

![Diagram](attachment:diagram.png)

Morphological steps during microautophagy, macroautophagy, CVT, and micropexophagy. Micropexophagy and microautophagy proceed by invagination of the tonoplast to engulf portions of the cytosol and peroxisomes (Per) to create autophagic bodies within the vacuole. Conversely, both macroautophagy and CVT sequester cytosolic components in a double membrane-bound vesicle, which then fuses with the tonoplast to release its contents into the vacuolar lumen. For macroautophagy, this vesicle is called the autophagosome. Where appropriate, the cargo is degraded by resident vacuolar hydrolases; the products are either stored in the vacuole or transported back to the cytosol for reuse. Adapted from [33]. C, chloroplast. M, mitochondrion.

to import vacuolar proteins that either cannot traverse the tonoplast and still remain functional or assemble into oligomeric complexes that are beyond the maximum size limits of other transport routes.

Over the past decades, numerous morphological studies have suggested that similar autophagic processes occur in plants [9]. Both light and electron microscopic studies have detected the possible engulfment of plastids, mitochondria and starch granules in vesicles that resemble PASs, autophagosomes and autophagic bodies found in yeasts (e.g. [10–12]). In suspension-cultured tobacco BY-2 cells, membrane-permeable cysteine protease inhibitors, such as E-64c, appear to block autophagic protein turnover during sucrose starvation [13]. These E-64c-treated cells accumulate acidic membrane-bound vesicles in the cytoplasm that appear morphologically similar to yeast autophagosomes [14]. Both the encapsulation of storage protein aggregates in vesicles for subsequent delivery to protein storage vacuoles (PSVs) in developing seeds of many species [15] and the vesicular sequestration of rubber particles in several rubber-producing species [16] might occur via CVT-like processes.

More recently, the application of vacuolar-specific markers in combination with possible autophagic inhibitors has provided better detection of potential autophagic structures in plants. Inhibitors, including the phosphatidylinositol 3-kinase (PI3K) inhibitors wortmannin, LY294002 and 3-methyladenine (3-MA), have been shown to block early steps in animal autophagy. When starved tobacco cells were treated with E-64c and any one of these three PI3K inhibitors, the accumulation of autophagosome-like structures was significantly inhibited [17]. The fluorescent probe LysoTracker Red, which localizes to acidic compartments, labeled these cytoplasmic vesicles, further supporting the proposal that they are related to vacuoles or lysosomes [14]. Vacuole-specific protein markers include the α-, δ-, and γ-tonoplast intrinsic proteins (TIPs) [14,18,19]. Importantly, confocal microscopy with anti-TIP antibodies or green fluorescent protein (GFP)-TIP fusions in combination with other protein markers led to the discovery that plants actually contain a heterogeneous mix of vacuole types. Some vacuoles behave like PSVs, whereas others appear to act as lytic vacuoles (LV), potentially with a specialized role in autophagic turnover [14,18]. Preliminary data suggest that autophagosomes are preferentially labeled by α-TIP [14]. Without autophagy-specific markers in plants, however, the unequivocal identification of structures that are directly related to either microautophagy or macroautophagy has been problematic.

Genetic and molecular dissection of  
autophagy in yeasts  

Fortunately, the application of molecular genetics has  
recently allowed us to move beyond morphological studies to uncover the underpinning molecular events. In  
pioneering genetic screens by various laboratories (see  
[3–5]), a large collection of AUTOPHAGY (ATG) genes  
(also known as APG, AUT, CVT and GSA [20]) were  
discovered that affect microautophagy, macroautophagy,  
and CVT in *S. cerevisiae* and pexophagy in *P. pastoris*. The  
*S. cerevisiae* screen exploited the ability of protease inhibitors to stabilize autophagic bodies *in vivo*, thus providing  
a visual screen for mutants that fail to accumulate  
these vacuolar vesicles. Surprisingly, many of the mutants  
simultaneously abrogate all autophagic events, indicating  
that the corresponding proteins participate in common  
steps needed for the vesicular delivery of cargo to the  
vacuole. The collective analysis of these mutants shows  
that autophagy is directed by a sequential series of post-  
translational modifications, two of which employ novel  
Ub-like conjugation steps.  

As shown in Figure 2a, autophagy begins when signals  
related to the nutritional (and sometimes developmental)  
status of the cell affect the activity of the target-of-  
rapamycin (TOR) kinase [3–5]. Under nutrient-rich conditions, TOR hyperphosphorylates the ATG1 kinase and  
the accessory protein ATG13, which induces their dissociation and inactivation. Under starvation conditions,  
phosphorylation by TOR is blocked, allowing ATG1 and  
ATG13 to be dephosphorylated by a yet-to-be described  
phosphatase activity. Dephosphorylated ATG1 and  
ATG13 assemble with each other and several other  
factors (ATG11, ATG17 and VAC8) to generate an active  
kinase complex. ATG17 is needed for both microauto-  
phagy and macroautophagy but not for CVT, whereas  
ATG11 and VAC8 appear to affect CVT specifically. The  
ATG1–ATG13 kinase complex then promotes autophagic vesiculation by either directly or indirectly activating  
the vacuolar-protein-sorting (VPS)-34PI3K complex that  
is associated with the transmembrane protein ATG9.  
Wortmannin, LY294002 and 3-MA, which impair vesicle  
nucleation during PAS formation, presumably block the  
activity of VPS34.  

Two Ub-like modification steps that are essential for  
vesicle formation are activated through an unknown  
sequence of events that is regulated by the ATG1–  
ATG13 and VPS34–PI3K complexes (Figure 2b). Similar  
to ubiquitination [2], these modifications employ ATP-  
dependent conjugation cascades involving E1 (activating)  
and E2 (conjugating) enzymes that attach two small  
proteins, ATG8 and ATG12, to their respective targets,  
phosphatidylethanolamine (PE) and the ATG5 protein.  
The two steps are initiated by the same E1 enzyme  
ATG7. As is typical of E1s, ATG7 first adenylates the  
carboxy-terminal Gly of both ATG8 and ATG12 using  

ATP, and then forms a thioester adduct with each  
through a conserved Cys with the concomitant release  
of AMP. The activated tags are donated via transesterification from ATG7 to their cognate E2s, ATG3 and  
ATG10, before final transfer to PE and ATG5, respectively. In both cases, an amide bond is formed between  
the carboxy-terminal Gly carboxyl group of the tag and  
either the amino group of PE or the ε-amino group of a  
positionally conserved Lys in ATG5. The ATG8–PE  
conjugate becomes bound to the autophagic membrane  
via its lipid moiety, whereas the ATG12–ATG5 conjugate  
associates with ATG16 to form a hetero-octomeric structure that is loosely bound to the vesicle. An additional  
processing step directed by the protease ATG4 is  
required before E1 activation of ATG8; ATG4 trims  
the carboxy-terminal end of a larger ATG8 precursor to  
expose the Gly residue.  

GFP-fusions with *S. cerevisiae* and mammalian ATG8s  
appear to be functional, thus providing useful markers  
to track autophagosome formation and docking [21,22].  
Their dynamics *in vivo* suggest that formation of the  
ATG12–ATG5 conjugate precedes formation of the  
ATG8–PE conjugate, and enhances this lipidation reaction via crosstalk between the two conjugation reactions.  
The actual function(s) of the ATG12–ATG5 and ATG8–  
PE conjugates are unknown, but at the morphological  
level they are necessary for autophagic engulfment.  
Recently, the *Arabidopsis* counterpart of ATG8 was  
reported to interact with microtubules, suggesting that  
the ATG8–PE conjugate assists in transporting autophagosomes to the vacuole via interaction with the micro-  
tubule cytoskeleton [23]. Before fusion of the  
autophagosome with the vacuole, it appears that some  
of the ATG8 is released from PE by the action of the  
ATG4 protease and probably recycled [24].  

Mature autophagosomes dock with the tonoplast to  
release the internal autophagic body using a vesicle  
trafficking system that employs the receptor complex  
containing the vesicle-soluble n-ethylmaleimide-sensitive  
factor adaptor protein receptor (v-SNARE) VTI1,  
the VAM3 syntaxin, and YKT7, a small GTP-binding  
protein of the Rab family (Figure 2c). It is possible that  
vesicle closure during microautophagy uses a similar  
docking apparatus. Finally, breakdown of the autophagic  
body involves a repertoire of vacuolar hydrolases, which  
in yeast includes the lipase ATG15, which helps to  
degrade the autophagic body membrane, and protease A (or PEP4), which activates a cascade of other proteases  
by proteolytic processing.  

Molecular genetic analysis of the ATG  
system in *Arabidopsis*  

Following the discovery of the ATG protein group in  
yeasts, orthologs were found to be encoded by various  
animal genomes, suggesting that the autophagic systems

(a)

Nutrient-rich

TOR kinase

Starvation

Induction

ATG17
P P
ATG1
kinase
P P
ATG11
P P

ATG13
P P P
VAC8

ATG17
P
ATG1
kinase
P P
ATG11

ATG13
P
VAC8

?

ATG6
(ATG14)
VPS34

Vesicle
nucleation

PAS

(b) Vesicle expansion and enclosure

ATG4

ATG8

ATG3

ATG8

ATP

ATG8

PE

ATG7

ATG12

ATP

ATG10

ATG5

ATG12

ATG5

ATG16L1

ATG5

ATG12

ATG12

?

Wortmannin
LY294002
3-MA

(c)

Concanamycin A
PMSF (yeast)
E-64c (plants)
Vacuole

PEP4

VPEγ

Autophagic
body

PRB1

ATG15

ATG22

Digestion

Tonoplast

MON1 CCZ1
HOPS YPT7
VAM3
VTI1
YKT6

Autophagosome

VAM7

VPT7 HOPS
CCZ1 MON1

Docking and fusion

Current Opinion in Plant Biology
directed by these proteins are remarkably conserved among eukaryotes. We ([25]; AR Thompson *et al.*, unpublished; A Suttangkakul, RD Vierstra, unpublished) and others [26,27*,28] have found counterparts to many of the *S. cerevisiae* ATG genes in the near-complete *Arabidopsis thaliana* genome, indicating that comparable microautophagic and macroautophagic systems exist in plants. Cursory searches in the rice (*Oryza sativa*) and corn (*Zea mays*) genome databases also detected counterparts to some ATG genes in these species (A Suttangkakul, RD Vierstra, unpublished). The list includes the TOR kinase, all components of the ATG1–ATG13 kinase complex, parts of the VPS34–PI3K complex, all but ATG16 of the ATG8–ATG12 conjugation cascades, ATG9, and various subunits of the VTI1 v-SNARE apparatus (Figure 2). Potential orthologs of the CVT-pathway-specific factors ATG11 and VAC8 are also evident, providing further support for the proposal that plants employ CVT-like methods to transport functional cargo to the vacuole.

Many of the *Arabidopsis* ATG proteins display remarkable overall conservation to their yeast and animal relatives, strongly suggesting that autophagic processes in plants are mechanistically identical ([25,26,29**]; AR Thompson *et al.*, unpublished). In agreement, several *Arabidopsis* ATG proteins were shown to replace the function of their yeast counterparts [23,26]. In contrast to *S. cerevisiae* in which all components but TOR are encoded by single genes, many *Arabidopsis* ATG proteins are encoded by small gene families ([25,26]; ASuttangkakul, RD Vierstra, unpublished). Nine active *Arabidopsis* genes are used to encode ATG8 alone ([29**]; AR Thompson *et al.*, unpublished). Amino-acid sequence variations within each ATG protein family imply that subtle functional differences might exist. For example, unlike the seven other *Arabidopsis* ATG8s, ATG8h and ATG8i do not have extra residues beyond the carboxy-terminal Gly, indicating that they can function without processing by the ATG4 protease [25,29**]. This heterogeneity implies that autophagy in plants is more elaborate than that in yeasts and that individual plant isoforms might have distinct targets, expression patterns, and/or functions (e.g. microautophagy versus macroautophagy).

With respect to the hydrolases that are involved in breaking down the autophagic body, it is well established that plant vacuoles, especially lytic vacuoles, contain a variety

of proteases and peptidases, nucleases, and glucanases that could fulfill this role [6]. One key protease might be vacuolar-processing enzyme (VPE)-γ, whose levels rise during senescence. Rojo *et al.* [30*] recently showed that VPEγ is synthesized as an inactive zymogen that is transported to the vacuole, possibly by a CVT-like route. Once in the acidic environment of the vacuole, VPEγ is autocatalytically converted into a smaller active form, which, just like *S. cerevisiae* PEP4, might help to proteolytically process downstream hydrolases that are responsible for degrading various vacuolar constituents.

Importantly, this collection of *Arabidopsis* ATG loci now provides a way to dissect plant autophagy biochemically and by reverse genetics, and offers molecular markers to detect the various cellular structures and compartments (e.g. PAS, autophagosomes and autophagic bodies) that participate in these processes. For example, analyses of mRNA and protein levels showed that the *Arabidopsis* ATG system is ubiquitously present, indicating that autophagy/CVT-like pathways are not restricted to specific developmental window(s) or nutritional state(s) in plants ([25,26,29**]; AR Thompson *et al.*, unpublished). Antibodies to ATG5 and ATG8 allowed the detection of new molecular mass forms that are predicted to be the ATG8-PE and ATG12-ATG5 adducts ([29**]; AR Thompson *et al.*, unpublished). Coupled with the phenotypic analysis of a mutant ATG7 protein that is missing the active-site Cys needed for ATG8/ATG12 activation [25], these studies confirmed that the two conjugation steps are active in plants.

Using fusions of ATG8 with GFP, we (AR Thompson *et al.*, unpublished) and Yoshimoto *et al.* [29**] recently showed by confocal microscopy of intact seedlings that at least three isoforms of ATG8 similarly coalesce into punctate structures in the cytoplasm that resemble autophagosomes. These structures relocate to the vacuolar lumens after nitrogen starvation. GFP-ATG8 proteins that were missing their carboxy-terminal Gly exhibited a diffuse distribution in the cytoplasm, and did not locate to the vacuole, indicating that the conjugation of ATG8 to PE is necessary for autophagic activity [29**]. The vacuolar-ATPase inhibitor concanamycin A blocks the acidification of the vacuole, thereby inactivating vacuolar hydrolases. Treatment of starved plants with concanamycin A induced the accumulation of GFP-ATG8-

(Figure 2 Legend Continued) the E1 enzyme ATG7, and the respective E2 enzymes, ATG3 and ATG10. Ultimately, ATG8 is conjugated to PE and ATG12 is conjugated to ATG5, with the ATG12–ATG5 conjugate then forming a complex with ATG16. Both conjugates associate with the autophagosome. (c) Docking and fusion of the autophagosome to the tonoplast is accomplished by a vesicle trafficking system employing the v-SNARE complex, which contains the v-SNARE VTI1, the Rab-like GTP-binding protein YKT6, and the syntaxin VAM3. Digestion of the autophagic body in the vacuole is achieved by a collection of hydrolases including the lipase ATG15 and proteinases A (PEP4) and B (PRB1). This breakdown is attenuated by concanamycin A and by the protease inhibitors PMSF and E-64c. One possible *Arabidopsis* processing protease is VPEγ. The question marks indicate parts of the pathway where specific steps remain unresolved. *Arabidopsis* orthologs of the yeast ATG systems are indicated in colored shapes. Those *Arabidopsis* proteins that have been dissected genetically are underlined. See Figure 3 for phenotypes associated with an *Arabidopsis* atg7 null mutant [25].
tagged vesicles that potentially represent autophagic bodies in the vacuole ([29**]; AR Thompson *et al.*, unpublished). Taken together, these GFP–ATG8 fusions appear to represent the first *bona-fide* marker for macroautophagy in plants, and thus should be instrumental in defining the dynamics of this process in response to internal and external cues.

### Physiological roles of autophagy in plants

Autophagy has been suggested to play a central role in several physiological and developmental responses in plants, including (i) the formation and degradation of PSVs during seed development and germination, (ii) *de novo* vacuolar biogenesis, (iii) nutrient recycling during starvation, senescence, and apoptotic processes such as xylem and sclerid cell morphogenesis, and (iv) the pathogen-induced hypersensitive response [9*,31]. It has been proposed that the formation of the large central vacuole in plant cells occurs either by autophagic encapsulation of PSVs by LVs or by autophagic fusion of many small donor vesicles that have engulfed portions of the cytoplasm [6,11]. Plants might also use specialized forms of autophagy, which are similar to pexophagy, to purge various microbody subtypes depending on metabolic needs (e.g., to remove glyoxysomes as oil seeds move beyond the glyoxylate cycle during germination). To help demonstrate these roles, a reverse genetic approach has been applied to several *Arabidopsis ATG* loci using the large collection of T-DNA insertional mutants available. To date, T-DNA mutants have been described that inactivate the TOR kinase [28], ATG9 [26], ATG4/ATG5/ATG7 of the conjugation steps ([25,29**]; AR Thompson *et al.*, unpublished), the *Arabidopsis* ortholog of the VTI1 v-SNARE (VTI12) [27*], and the protease VPEγ [30*].

Collectively, the phenotypic analyses indicate that the ATG system, and by inference microautophagy, macroautophagy and CVT, is not essential in plants. Nevertheless, it should be noted that we cannot rule out the remote possibility that other autophagic pathway(s) exist that work in the absence of the ATG system. *Arabidopsis* null mutants of the single genes that encode ATG5, ATG7, ATG9 and the two genes that encode ATG4 are morphologically indistinguishable to wildtype plants when grown under nutrient-rich conditions and display normal embryogenesis, seed germination, cotyledon development, root and shoot elongation, and seed production ([25,26,29**]; AR Thompson *et al.*, unpublished). The central vacuoles that develop in young mutant seedlings appear morphologically identical to those from wildtype plants, indicating that autophagy is not required for PSV breakdown and vacuolar ontogeny. Xylem development also appears normal in these mutants, thus eliminating autophagy as an important step in the apoptosis of these cells. The only significant developmental phenotype of the *atg* mutants is an acceleration of leaf senescence, even when grown under nutrient-rich conditions ([25,26,29**]; AR Thompson *et al.*, unpublished). To date, TOR is the only component of the ATG system that has been shown to be essential; disruption of the single *Arabidopsis TOR* gene leads to premature arrest of embryo and endosperm development [28]. Given that TOR is essential but other autophagy proteins are not, it seems that the other known functions of TOR, including its regulation of transcription, ribosome biogenesis, translational capacity, and/or cytoskeletal organization during starvation, are more crucial in plants than is TOR’s role in autophagic induction [28].

Growth of the various *atg* mutants under suboptimal conditions indicates that autophagy plays an important role in nutrient remobilization, especially during starvation. When grown under either carbon- or nitrogen-deficient conditions, *Arabidopsis* single mutants in which ATG4a/b, ATG5, ATG7, or ATG9 is affected exhibit increased chlorosis, accelerated bolting, enhanced dark-induced senescence of detached leaves, and reduced seed yield (*Figure 3*; [25,26,29**]; AR Thompson *et al.*, unpublished). A similar, albeit milder, phenotype is apparent for a mutation that affects the v-SNARE VTI12 [27*]. Wild-type plants can survive extended periods without nitrogen or light (which promotes carbon starvation), whereas the *atg5-1* and *atg7-1* mutants, for example, rapidly become chlorotic ([25]; AR Thompson *et al.*, unpublished). This chlorosis is associated with a faster loss of cytosolic, mitochondrial and chloroplastic proteins, indicating that the mutant plants exploit other proteolytic routes to maintain pools of free amino acids in the absence of autophagy. However, this bypass appears to compromise the mutant plants such that their viability cannot be rescued as easily as that of wildtype plants by subsequent transfer back to a nitrogen-rich medium or to light (*Figure 3*).

Consistent with the phenotypic defects, the morphology of the vacuoles during starvation appears to be affected by loss of the ATG system. Using GFP–TIP markers to observe the tonoplast, we detected tubule-like projections that extend into the vacuolar lumen. These tubules resemble the microautophagic structures in yeast that are used to bud off autophagic bodies (AR Thompson *et al.*, unpublished). They are more abundant in a null mutant for ATG7 even during starvation, suggesting that these structures represent aborted attempts at microautophagy in the absence of the ATG8/ATG12 conjugation steps.

Nutrient-challenged growth conditions enhance the levels of several *ATG* transcripts in addition to those that are associated with the senescence syndrome and other stress-response pathways, suggesting that this nutrient-deficiency-induced cell death represents a form of premature senescence ([32]; AR Thompson *et al.*, unpublished). In several *atg* mutants, mRNA abundance for

Figure 3

(a)
| WT | atg7-1 |
| --- | --- |

| atg7-1/ATG7 | atg7-1/ATG7C→S |

(b)
| WT | atg7-1 | atg7-1/ATG7 |
| --- | --- | --- |

(c)
| 2d | 4d | 6d |
| --- | --- | --- |

| WT |
| --- |
| atg7-1 |
| --- |

Current Opinion in Plant Biology

Phenotypes of an *Arabidopsis* mutant that is affected in the ATG7 gene, which encodes the E1 required for activation of ATG8 and ATG12 before their conjugation to PE and ATG5, respectively. Plant lines include wildtype *Arabidopsis* (WS ecotype), the *atg7-1* mutant, and the *atg7-1* mutant complemented with either ATG7 or a mutation in which the active-site cysteine codon was converted to that for a serine (ATG7C→S).  
(a) Early senescence of rosette leaves. Five-month-old plants grown in nutrient-rich soil under a short-day photoperiod (from [25]).  
(b) Plants grown for 32 days in nitrogen-free liquid medium.  
(c) Plants exposed to dark-induced carbon starvation. Six-week-old plants were grown under an 8 hour light/16 hour dark short-day photoperiod, transferred to darkness for 2, 4 or 6 days and then transferred back to the short-day period for one week (AR Thompson et al., unpublished).

www.sciencedirect.com
Current Opinion in Plant Biology 2005, 8:165–173

other ATG genes is further increased during starvation. Thus, it appears that the Arabidopsis ATG system may be transcriptionally regulated by a negative feedback mechanism that links autophagy to the plant’s nutritional status ([25]; AR Thompson *et al.*, unpublished). One probable effector is the TOR kinase, which regulates the transcription of nitrogen- and carbon-utilization genes in other eukaryotes [28].

### Conclusions

We are now entering exciting times in our attempts to uncover the functions of autophagy in plants and to define the underpinning molecular mechanisms. The ATG genes and proteins offer new tools to dissect autophagy genetically and to identify unequivocally the various vesicles involved. Mutants should help us further determine when autophagy is important in processes such as pathogen defense, the removal of damaged or unnecessary organelles, and the apoptic events that fine-tune homeosis. Given the enhanced complexity of the ATG system in plants compared with those in yeasts and animals, it would not be surprising if new roles for autophagy are identified in plants. Although it is assumed that autophagy is non-selective, it might now be possible to test this assumption by isolating autophagosomes and autophagic bodies from specific tissues or developmental states using affinity-tagged ATG markers and examining their contents for specific proteins or organelles. The possibilities that plants use CVT to form PSVs and import vacuolar proteins or pexophagy to remove microbodies are intriguing. Given the importance of autophagy to nitrogen and carbon allocation, the dissection of the ATG system might ultimately offer new strategies to enhance nutrient mobilization for agricultural benefit.

### Acknowledgements

Research in the Vierstra laboratory was supported by grants from the US Department of Agriculture National Research Initiative Competitive Grants program (2002-01452) and the Research Division of the UW-College of Agriculture and Life Sciences (Hatch 2858). ART was partially supported by UW-WARF and Louis and Elsa Thomsen Wisconsin Distinguished Pre-doctoral Fellowships.

### References and recommended reading

Papers of particular interest, published within the annual period of review, have been highlighted as:

- of special interest
- of outstanding interest

1. Vierstra RD: Proteolysis in plants: mechanisms and functions. Plant Mol Biol 1996, **32**:275-302.
2. Smalle J, Vierstra RD: The ubiquitin 26S proteasome proteolytic pathway. Annu Rev Plant Biol 2004, **55**:555-590.
3. Klionsky DJ, Emr SD: Autophagy as a regulated pathway of cellular degradation. Science 2000, **290**:1717-1721.
4. Klionsky DJ. (Ed.): Autophagy. Eurekah.com/Landes Bioscience; 2004.
5. Ohsumi Y: Molecular dissection of autophagy: two ubiquitin-like systems. Nat Rev Mol Cell Biol 2001, **2**:211-216.
6. Marty F: Plant vacuoles. Plant Cell 1999, **11**:587-600.
7. Matile PH: Protein degradation. In Encyclopedia of Plant Physiology, New Series, vol 14A. Edited by Coulter D, Partier B. Springer-Verlag; 1982:169-188.
8. Huang WP, Klionsky DJ: Autophagy in yeast: a review of the molecular machinery. Cell Struct Funct 2002, **27**:409-420.
9. Moriyasu Y, Klionsky DJ: Autophagy in plants. In Autophagy. Edited by Klionsky D. Eurekah.com/Landes Bioscience; 2004:208-215. This is the first comprehensive review of autophagy in plants. Besides discussing the plant homologues of yeast autophagy genes, the authors outline the roles of autophagy in plants at the morphological level, with particular emphasis on vacuole biogenesis and development. They highlight the evidence showing that autophagy is instrumental in the development of lytic vacuoles and protein storage vacuoles, as well as in the transition between the two vacuolar subtypes.
10. Chen MH, Liu LF, Chen YR, Wu HK, Yu SM: Expression of alpha-amylases, carbohydrate metabolism, and autophagy in cultured rice cells is coordinately regulated by sugar nutrient. Plant J 1994, **6**:625-636.
11. Toyooka K, Okamoto T, Minamikawa T: Cotyledon cells of Vigna mungo seedlings use at least two distinct autophagic machineries for degradation of starch granules and cellular components. J Cell Biol 2001, **154**:973-982.
12. Aubert S, Gout E, Bligny R, Marty-Mazars D, Barrieu F, Alabouvette J, Marty F, Douce R: Ultrastructural and biochemical characterization of autophagy in higher plant cells subjected to carbon deprivation: control by the supply of mitochondria with respiratory substrates. J Cell Biol 1996, **133**:1251-1263.
13. Moriyasu Y, Ohsumi Y: Autophagy in tobacco suspension-cultured cells in response to sucrose starvation. Plant Physiol 1996, **111**:1233-1241.
14. Moriyasu Y, Hattori M, Jauh GY, Rogers JC: Alpha tonoplast intrinsic protein is specifically associated with vacuole membrane involved in an autophagic process. Plant Cell Physiol 2003, **44**:795-802. This work helped identify the types of vacuoles that are present in barley root tips and showed that tonoplast intrinsic proteins (TIPs) can be used to discriminate among them. The authors also demonstrated that LysoTracker Red, an aciditrophic dye, co-localizes with anti-α-TIP antibodies but not with antibodies against δ-TIP or γ-TIP, suggesting that α-TIP specifically labels autophagosome-like vesicles in this species. These markers can now be used to help track lytic vacuoles and potential autophagosomes during vacuole biogenesis and autophagy.
15. Herman EM, Larkins BA: Protein storage bodies and vacuoles. Plant Cell 1999, **11**:601-614.
16. Cornish K, Wood DF, Windle JJ: Rubber particles from four different species, examined by transmission electron microscopy and electron-paramagnetic-resonance spin labeling, are found to consist of a homogeneous rubber core enclosed by a contiguous, monolayer biomembrane. Planta 1999, **210**:85-96.
17. Takatsuka C, Inoue Y, Matsuoka K, Moriyasu Y: 3-Methyladenine inhibits autophagy in tobacco culture cells under sucrose starvation conditions. Plant Cell Physiol 2004, **45**:265-274. The authors demonstrate the efficacy of several known inhibitors of mammalian PI3Ks in potentially blocking autophagy in tobacco. They show that the E-64c-induced accumulation of numerous acidic vesicles that structurally resemble autophagosomes is blocked in cells treated with the PI3K inhibitors. Washout studies show that these effects are reversible. The turnover of total protein is also inhibited by 3-methyladenine. This study implied that a PI3K is involved in plant autophagy and offers a repertoire of autophagy inhibitors.
18. Jauh GY, Phillips TE, Rogers JC: Tonoplast intrinsic protein isoforms as markers for vacuolar functions. Plant Cell 1999, **11**:1867-1882.
19. Cutler SR, Ehrhardt DW, Griffitts JS, Somerville CR: Random GFP::cDNA fusions enable visualization of subcellular structures in cells of Arabidopsis at a high frequency. Proc Natl Acad Sci USA 2000, **97**:3718-3723.
20. Klionsky DJ, Cregg JM, Dunn WA, Emr SD, Sakai Y, Sandoval IV, Sibirny A, Subramani S, Thumm M, Veenhuis M *et al.*: A unified

nomenclature for yeast autophagy-related genes.
Dev Cell 2003, 5:539-545.

21. Mizushima N, Yamamoto A, Matsui M, Yoshimori T, Ohsumi Y:
In vivo analysis of autophagy in response to nutrient starvation
using transgenic mice expressing a fluorescent
autophagosome marker. Mol Biol Cell 2004, 15:1101-1111.

22. Suzuki K, Kirisako T, Kamada Y, Mizushima N, Noda T, Ohsumi Y:
The pre-autophagosomal structure organized by concerted
functions of APG genes is essential for autophagosome
formation. EMBO J 2001, 20:5971-5981.

23. Ketelaar T, Voss C, Dimmock SA, Thumm M, Hussey PJ:
Arabidopsis homologues of the autophagy protein ATG8 are a
novel family of microtubule binding proteins. FEBS Lett 2004,
567:302-306.

24. Kirisako T, Ichimura Y, Okada H, Kabeya Y, Mizushima N,
Yoshimori T, Ohsumi M, Takao T, Noda T, Ohsumi Y: The
reversible modification regulates the membrane-binding
state of Apg8/Aut7 essential for autophagy and the
cytoplasm to vacuole targeting pathway. J Cell Biol 2000,
151:263-276.

25. Doelling JH, Walker JM, Friedman EM, Thompson AR, Vierstra RD:
The APG8/12-activating enzyme APG7 is required for proper
nutrient recycling and senescence in Arabidopsis thaliana.
J Biol Chem 2002, 277:33105-33114.

26. Hanaoka H, Noda T, Shirano Y, Kato T, Hayashi H, Shibata D,
Tabata S, Ohsumi Y: Leaf senescence and starvation-induced
chlorosis are accelerated by the disruption of an Arabidopsis
autophagy gene. Plant Physiol 2002, 129:1181-1193.

27. Surpin M, Zheng H, Morita MT, Saito C, Avila E, Blakeslee JJ,
• Bandyopadhyay A, Kovaleva V, Carter D, Murphy A et al.:
The VTI family of SNARE proteins is necessary for plant
viability and mediates different protein transport pathways.
Plant Cell 2003, 15:2885-2899.
This paper describes a pair of VTI1 v-SNARE proteins in Arabidopsis.
Mutants affecting one v-SNARE protein (i.e. VTI11) display defects in
vascular patterning and auxin transport, whereas mutations affecting the
second (VTI12) cause an accelerated senescence phenotype similar to
that of other ATG-pathway mutants. Alterations in vacuolar morphology
were also evident in the vti12 mutants, suggesting that this v-SNARE
isoform participates in the formation of authophagosomes and/or their
fusion with the tonoplast.


28. Menand B, Desnos T, Nussaume L, Berger F, Bouchez D, Meyer C,
Robaglia C: Expression and disruption of the Arabidopsis TOR
(target of rapamycin) gene. Proc Natl Acad Sci USA 2002,
99:6422-6427.

29. Yoshimoto K, Hanaoka H, Sato S, Kato T, Tabata S, Noda T,
• Ohsumi Y: Processing of ATG8s, ubiquitin-like proteins, and
their deconjugation by ATG4s are essential for plant
autophagy. Plant Cell 2004, 16:2967-2983.
The authors of this paper dissected the ATG8 conjugation step. Using a
double mutant that was missing the ATG4 protease required for proces-
sing and recycling ATG8s, they showed that formation of the ATG8-PE
conjugate is necessary for survival under nitrogen- and carbon-limiting
growth conditions. Using ATG8-GFP fusions as reporters, punctate
structures that potentially reflect autophagosomes were identified under
starvation conditions by confocal fluorescence microscopy. These struc-
tures were absent in the atg4a atg4b double mutants. Possible autopha-
gic bodies were also detected in plants treated with the vacuolar ATPase
inhibitor concanamycin A. ATG8-GFP fusions might represent important
markers for unequivocally visualizing autophagosomes in vivo.

30. Rojo E, Zouhar J, Carter C, Kovaleva V, Raikhel NV: A unique
• mechanism for protein processing and degradation in
Arabidopsis thaliana. Proc Natl Acad Sci USA 2003,
100:7389-7394.
VPEγ was identified as a key vacuolar-processing protease that is impor-
tant for activating several vacuolar zymogens. This protease is transported
to the vacuole as a larger inactive precursor, possibly via a CVT-like
pathway, and activated autocatalytically by processing to a smaller mature
form in the acidic environment of the vacuole. The fact that VPEγ is
upregulated during senescence suggests that it serves a function similar
to that of yeast PEP4 in proteolytically activating other vacuolar hydro-
lases, and thus could be a key factor in autophagic body breakdown.

31. Di Sansebastiano GP, Paris N, Marc-Martin S, Neuhaus JM:
Regeneration of a lytic central vacuole and of neutral
peripheral vacuoles can be visualized by green fluorescent
proteins targeted to either type of vacuoles. Plant Physiol 2001,
126:78-86.

32. Contento AL, Kim SJ, Bassham DC: Transcriptome profiling of
the response of Arabidopsis suspension culture cells to Suc
starvation. Plant Physiol 2004, 135:2330-2347.

33. Farre J-C, Subramani S: Peroxisome turnover by
micropexophagy: an autophagy-related process. Trends Cell
Biol 2004, 14:515-523.
