
MOLECULAR GENETICS OF T CELL DEVELOPMENT

Ellen V. Rothenberg and Tom Taghon  
*Division of Biology, California Institute of Technology, Pasadena, California 91125;  
email: evroth@its.caltech.edu, ttaghon@caltech.edu*

Key Words thymus, transcription factor, Notch, lineage commitment, β-selection

■ Abstract T cell development is guided by a complex set of transcription factors that act recursively, in different combinations, at each of the developmental choice points from T-lineage specification to peripheral T cell specialization. This review describes the modes of action of the major T-lineage-defining transcription factors and the signal pathways that activate them during intrathymic differentiation from pluripotent precursors. Roles of Notch and its effector RBPSuh (CSL), GATA-3, E2A/HEB and Id proteins, c-Myb, TCF-1, and members of the Runx, Ets, and Ikaros families are critical. Less known transcription factors that are newly recognized as being required for T cell development at particular checkpoints are also described. The transcriptional regulation of T cell development is contrasted with that of B cell development, in terms of their different degrees of overlap with the stem-cell program and the different roles of key transcription factors in gene regulatory networks leading to lineage commitment.

OVERVIEW OF T CELL DEVELOPMENT AND ITS REQUIREMENTS

T cells are the only hematopoietic cells that are not generated in the bone marrow. Instead, virtually all circulating T lymphocytes are generated in the thymus, and many different types exist. T cells are further distinguished among hematopoietic cells by their potentially infinite proliferative life spans and by their capacity for differentiative specialization even after their mature features are in place. Some of this specialization is triggered by environmental signals from antigen recognition and/or other ligand/receptor interactions. Thus T cell development, as a process, incorporates multiple stages at which different choices are available to the cells, extending over many cell cycles and a long period of time. In spite of this complexity, the T cell program as a whole is unified by the identities of the key regulators throughout the process: Many of the same regulatory factors and growth factor receptors are used again and again at different stages. This review focuses on the roles of these factors in establishing T cell identity as they guide uncommitted hematopoietic precursors into, and through, thymic differentiation.

A Map of the Terrain: Stages of T Cell Development

At least five stages have been defined in which specific regulators are needed for T cell development to proceed. First, there is the specification process, through which multipotent precursors first enter the T cell pathway, usually as they first immigrate to the thymus. Second is the complex process in which proliferative expansion and T cell receptor (TCR) gene rearrangement are combined with commitment to the T lineage within the thymus. These first stages are examined closely at the end of this review. Third is $\beta$-selection, which is a cascade of differentiation and proliferation events triggered specifically by “pre-TCR” signaling in precursors of TCR$\alpha\beta$ T cells, after successful rearrangement of the TCR$\beta$ gene. Fourth is TCR-dependent positive selection, a major physiological transition, which is coupled with developmental divergence between CD4 and CD8 lineages of T cells. Fifth is the continuing differentiation of T cells in the periphery, which occurs upon stimulation with antigen, antigen-presenting cells, and cytokines. This has been most closely studied for the divergence of Th1 and Th2 CD4 cell types. These stages are diagrammed in Figure 1.

The $\beta$-selection and positive selection checkpoints make production of mature T cells depend on successful rearrangement of the T cell receptor genes, either TCR$\gamma$ and TCR$\delta$ or TCR$\alpha$ and TCR$\beta$. Rearrangement is made possible by the expression of the linked RAG-1 and RAG-2 recombinase genes and by the accessibility of the TCR-coding loci in chromatin at the appropriate stages. Transcriptional induction of RAG genes and transcription factor-mediated opening of the TCR loci thus constitute aspects of T cell specification. TCR$\gamma$, $\delta$, and $\beta$ rearrangement all occur during the early DN (double negative) stages (Figure 1), whereas TCR$\alpha$ rearrangement is only permitted in cells that have reached the DP (CD4${}^{+}$ CD8${}^{+}$ TCR${}^{\text{low}}$) stage. The cell can sense the success of these rearrangements in encoding a translatable protein because there is also transcriptional activation of the genes encoding the signaling components of the TCR complex, from the earliest stages, and these proteins enable products of newly recombined TCR genes to assemble into functional signaling complexes. Successful TCR$\beta$ rearrangement entitles the cell to pass through the $\beta$-selection checkpoint and become eligible for differentiation

---

Figure 1 Outline of T cell development: landmark stages, checkpoints, and developmental choices. Right side: cell-surface markers used in combination to distinguish specific developmental stages. DN: double negative for CD4 and CD8, and as used in this figure, implied to be negative for cell-surface T cell receptor complex expression as well. TCR$\gamma\delta$ and NKT cells are also commonly CD4${}^{-}$ CD8${}^{-}$ but are mature TCR${}^{+}$ subsets that are presented separately. DN1, DN2, DN3, and DN4 stages of DN cell differentiation are distinguished by CD44, c-Kit, and CD25 expression as indicated. DP: CD4${}^{+}$ CD8${}^{+}$ TCR$\alpha\beta$-low. CD4 SP: CD4${}^{+}$ CD8${}^{-}$ TCR$\alpha\beta$-high. CD8 SP: CD4${}^{-}$ CD8${}^{+}$ TCR$\alpha\beta$-high. Branch points from the TCR$\alpha\beta$ mainstream for NKT and T-reg lineages of TCR$\alpha\beta$ cells are incompletely defined.

progenitor

B  
mϕ  
DC  
NK  

TCR-independent

γδ  

DN1  
specification  
DN2  
commitment  
DN3  
β-selection  
DN4  
ISP  
DP  
positive  
selection  
NKT  
Treg  
CD4  
CD8  
Th1/Th2  
differentiation  
Th1  
Th2  

c-Kit  
CD44  
CD24  
CD25  
CD8  
CD4  
RAG-1,2

in the TCR$\alpha\beta$ lineage, including the activation of TCR$\alpha$ gene rearrangement. As an alternative to $\beta$-selection, successful rearrangement of TCR$\gamma$ and TCR$\delta$ together enables the cell to differentiate as a TCR$\gamma\delta$ cell instead. Cells failing at both will die. Only after successful generation of a TCR$\alpha\beta$ receptor with appropriate specificity are TCR$\alpha\beta$ lineage cells able to pass through a second checkpoint, i.e., positive selection, and then complete their maturation.

TCR-dependent signaling in thymocyte development not only determines survival versus death, but also acts more subtly in other kinds of lineage choice, as discussed in recent reviews (1–5). The quality of signaling at these checkpoints can have a strong influence on the choice of developmental paths that follow. A particularly clear case is at positive selection, when the strength and duration of signaling induced by TCR/ligand interaction in the thymus not only determines which cells will be positively selected, but also guides them to differentiate into either CD4 T cells or CD8 T cells (3–5). For the purposes of this review, most notable about these TCR-dependent events is how very similar signals, from variants of the same receptor, trigger radically different regulatory consequences. In positive selection, for example, this trigger is a transient interaction, and yet CD4 cell and CD8 cell positive selection processes elicit divergent developmental cascades that result in numerous long-term differences in mature cell function and effector potential, which will be maintained through many cell divisions in the periphery. The TCR-dependent program that governs TCR$\gamma\delta$ cell development is also remarkably different from the $\beta$-selection response, and the consequences of pre-TCR triggering at $\beta$-selection are also quite different from the consequences of TCR triggering at positive selection, as reviewed elsewhere (6).

The explanations lie in the different transcription factor genes that are specifically induced under different conditions (5), and the regulatory context that influences which of these genes will be available for activation by a given signal. Thus in this review, we do not focus on the differences among the various triggering signals as such, but instead, review the regulatory basis for the fundamentally different developmental programs (TCR$\gamma\delta$ development, $\beta$-selection, CD4 cell maturation, and CD8 cell maturation) that they call into play. A remarkable fraction of the regulatory molecules involved in these choices also play roles in T cell development from the earliest stages.

The genetic manipulation approaches that have been most powerful for dissecting regulatory relationships are much more accessible in the mouse than in human systems, and so most of this review is based on results in the murine system. However, where evidence is available, human T cell development appears to depend on the same regulators, and some clues suggest that the roles of key factors in T cell development may be broadly conserved among jawed vertebrates.

## Critical Regulators: Introductory Overview

Gene knockout experiments have shown that T cell development depends from its earliest stages on at least a half dozen transcription factors, one “instructive”

signaling system, and at least three other signaling systems used for growth and survival at particular points. Nearly all of these regulators play ongoing roles in T cell development, although their specific effects shift from one stage to another. They are introduced briefly here, and in the following sections, the roles of each of the transcriptional regulators are discussed in detail.

T cell gene expression and even the first recognizable stages of T cell development depend on the transcription factors GATA-3, c-Myb, members of the Runx family, members of the E2A/HEB family, and members of the Ikaros family. In fetal life, appearance of specified pre-T cells also depends on the Ets-family transcription factor PU.1. Most of these factors are needed for other hematopoietic fates besides the T cell fate. Hunchback-class zinc finger factors of the Ikaros family are needed for all lymphocyte lineages, and the E2A/HEB class of bHLH transcription factors is absolutely essential for B cell development as well as for T cell development. Other factors, such as c-Myb, PU.1, and Runx1, are required to generate the multipotent hematopoietic progenitors of T cells, in addition to playing specific roles in early T cell functions proper. The one transcription factor that appears to be T cell specific is GATA-3. However, this too is a close relative of the stem-cell factor GATA-2. The ability of these molecules to turn on T cell genes as opposed to non-T cell genes is therefore likely to represent target-gene specificity emerging from combinatorial transcription factor action.

T cell development also depends on two other transcription factors that represent dedicated effectors of cell-surface receptor signaling, TCF-1 (Tcf7) and RBPSuh (a.k.a. RBP-Jκ, or CBF/Suppressor of Hairless/Lag-1 = CSL). These factors are distinctive because they are repressors by default but are changed into activators by signaling cascades in response to environmental signals. TCF-1 is the effector of the Wnt/frz/β-catenin signaling cascade, and RBPSuh is the effector of the Notch signaling cascade. The exact roles that these factors play in T cell development appear to be different, as described below, but they share the critical feature that their presence in the precursors of T cells imposes a dichotomous switch-like behavior on every function they control. The same genes that they activate in the presence of a signal, they repress in the absence of a signal. The Notch signaling cascade is the unique “instructive” signaling system used for T cell specification, whereas the Wnt/TCF signaling system has an important role in proliferation coupled with differentiation.

Other genes that are essential for T cell development encode cytokine receptors as well as pre-TCR/TCR components that are needed for survival and proliferation. Early in T cell development the IL-7 receptor complex (IL7Ra/γc = CD127/CD132) is most important, while survival and proliferation are dominated later by signals from different versions of the TCR complex (pre-TCR, TCRγδ, or TCRαβ associated with CD4 or CD8 coreceptors). These signal-dependent receptors all differ from the Notch system in that they primarily trigger the de novo appearance or nuclear localization of transcription factors, rather than conversion of a repressor to an activator. They thus lend themselves more to promoting graded responses, such as proliferation and activation, rather than the kinds of all-or-none

dichotomous effects of Notch/RBPSuh. The transcription factors mobilized include STAT5, NF-κB, NF-AT, and AP-1 in different combinations, with greater or lesser contributions from the PI-3 kinase, Akt/PKB, and Ras/MAP kinase signaling pathways (see 1–5). The survival and proliferative (trophic) effects of these receptor systems are well explained by the activation of these mediators.

Triggering of cytokine receptors, like the triggering of TCR complexes, can also have developmental effects that are specific to different stages. Whereas the receptors are lineage-specific in their expression, the factors they activate are not. Without minimizing the importance of these pathways, the developmental consequences of their triggering must be heavily influenced by changes in the “regulatory contexts” within which they are activated. In this review, therefore, we focus primarily on the transcription factors themselves and the transcription-linked signaling systems that set these different regulatory contexts.

## REPERTORY PLAYERS: T CELL IDENTITY FACTORS IN RECURRENT ROLES

A particular set of T cell transcription factors and signaling molecules is used repeatedly throughout T cell development to establish T cell identity and then to make choices between successive T-lineage subspecializations. These recurrent players include Notch molecules and their direct transcriptional effector RBPSuh (CSL); GATA-3; the bHLH factors E2A and HEB and their antagonists Id2 and Id3; Runx1 and Runx3; and possibly also members of the Ikaros family. These factors can be viewed as central to T cell identity, somewhat analogous to the B cell factors EBF and Pax5 (7), which are compared in more detail below. However, in the case of T cell development, the individual functions that these factors perform are discontinuous, stage-specific, and even subject to alternation between activating and inhibitory effects from one stage to the next. These “identity factors” are described here in detail.

### Notch and its Mediators

The transmembrane signaling receptor Notch1 is a representative of an ancient, evolutionarily conserved family of developmental regulators that was first noted for its importance for neurogenesis in embryos of *Drosophila melanogaster*. In the mouse, Notch1 is essential for embryonic viability also, and it is expressed at a particularly high level in the thymus. In the late 1990s, Notch gain of function was shown to have powerful influences on TCR-dependent selection events in the thymus. Then with the advent of conditional knockout technology, it became possible to remove Notch1 function, and this revealed that T cell development is intensely and specifically dependent on Notch1 in the pluripotent precursor stage (reviewed in 8–10). Without Notch1, precursors cannot develop into T cells at all, and B cells develop in the thymus instead. At the same time, retroviral

REGULATION OF T CELL DEVELOPMENT

![Diagram](#)

Figure 2 Roles of Notch and its transcriptional effectors in T cell specification and developmental choices. Stages at which T cell development is affected by gain or loss of function of Notch1, other Notch proteins, or the Notch transcriptional mediator RBPSuh (CSL) are indicated. See text for details and review citations.

transduction of constitutively active Notch (Notch1IC) into pluripotent precursors showed that Notch gain of function could also enhance T cell development while blocking B cell development (Figure 2). As noted above, one major pathway of Notch signaling involves cleavage of the Notch cytoplasmic domain to generate a transcriptional activation domain, which migrates to the nucleus and converts a default repressor, RBPSuh, into an activator. Knockout experiments indicate that this is the pathway through which Notch promotes T cell development and suppresses B cell development (11, 12). Notch1/RBPSuh are the first, and so far the only, regulators in which gain and loss of function have been shown to have simple reciprocal effects on initial T cell specification.

Notch family molecules also play more complex roles later, both in TCR-dependent selection events and in peripheral T cell differentiation. Conditional knockout studies with deletion in intrathymic DN cells have shown that Notch1 is needed for TCRβ gene rearrangement; also, in vitro assay systems show that Notch/Delta interactions are needed to sustain T cells through the proliferation and differentiation events of β-selection, complementing signals from the pre-TCR (13–16) (Figure 2). Deletion of RBPSuh between T-lineage specification and β-selection favors development of TCRγδ lineage T cells at the expense of TCRαβ T cells (12) (Figure 2). Normally, Notch1 and the genes that it immediately activates are expressed at the highest levels in DN (double negative) thymocytes prior to β-selection, and substantially downregulated thereafter (14, 17). Constitutive expression of activated Notch perturbs the gene expression setpoints in DP (double positive) cells that emerge from β-selection (14, 18) and can lead to leukemias that resemble cells frozen in mid-β-selection (19–21). Thus, not only Notch activation but also its correctly regulated deactivation are important to generate the pool of TCRαβ⁺ thymocytes from which positive selection and maturation will occur.

Alterations of Notch activation in thymocytes at later stages can sharply perturb positive selection and the CD4/CD8 lineage choice as well (Figure 2). However,

the exact effects seen vary substantially in different experimental tests (reviewed in 8–10, 22–24). In overexpression studies the result depends on the exact form of the constitutively activated Notch molecule, since the intracellular domain of Notch contains a number of separate domains mediating interactions with different cofactors (see reviews, above). Most gain-of-function studies have shown an enhancement of CD8 cell development, though this is not uniform, and whether this is due to trophic effects, alteration of signaling, or developmental programming is subject to debate. Loss-of-function studies have been complicated by the ability of multiple Notch family members to contribute to thymocyte responses at these late stages and by uncertainties about the specificities of pharmacological inhibitors of Notch signaling and the exact modes of action of endogenous signaling modulators. Also complicating the interpretation of all the transgenic and knockout experiments is an intrinsic aspect of thymocyte population dynamics, namely the small number of cell division cycles between the end of β-selection and the start of positive selection. This means that Notch manipulation might affect positive selection indirectly through perturbations of the β-selection process, and its impact may appear different in kinetic than in steady-state analyses.

Some clarification may emerge eventually from identification of the ligands for Notch during positive selection. Notch receptors interact with two different kinds of ligands, Delta-like and Jagged family molecules, and although they both activate RBP-Suh, they promote distinct developmental responses through additional pathways that are still under investigation (25–27). Very recently, it has emerged that the differential signaling capacities of Notch family members interacting with Delta-like versus Jagged family ligands are probably quite significant in peripheral T cell differentiation (Figure 2). Differential uses of these classes of Notch ligands, and perhaps Notch family members, in contacts between mature peripheral T cells and antigen-presenting cells, act as instructive signals to bias the subsequent differentiation of the T cells into specialized effectors of the Th1, Th2, or T-reg classes (27–30). Although RBP-Suh seems to have a direct involvement in the Notch/Jagged pathway that favors the Th2 fate, other Notch/Delta-like signaling pathways appear to contribute to the Th1 fate (27). Thus Notch signaling underlies a succession of divergent choices in T cell differentiation that continue long after its initial role in T- versus B-lineage specification.

## GATA-3

GATA-3 is the one transcription factor that appears to be expressed in a completely T cell-specific way among all hematopoietic cell types. While it is also needed for many nonhematopoietic cell types, and GATA-3 mutants do not survive midgestation, the only hematopoietic effects seen are a reduction in hematopoietic progenitor cells, complete loss of T cell development, and a late defect in the final maturation of NK cells (31–33). GATA-3 expression rises as T-lineage differentiation begins (T. Taghon, E.-S. David, J.C. Zúñiga-Pflücker & E.V. Rothenberg, submitted; 34) and peaks in the thymus during the proliferation at β-selection

REGULATION OF T CELL DEVELOPMENT

![Diagram](#)

Figure 3 Roles of GATA-3 in support of T cell development. Stages at which loss of GATA-3 function affects T lineage differentiative progression and developmental choices are shown with the implied direction of the normal GATA-3 effect as indicated. Within the T lineage, overexpression of GATA-3 or a GATA-3 hypomorphic mutant has similar effects to those indicated; however, in prethymic precursors, GATA-3 overexpression causes a paradoxical deviation away from all lymphoid fates (not shown in figure).

and again, later, in cells undergoing positive selection to the CD4 SP thymocyte fate (32, 35, 36). Expression drops again in resting peripheral T cells, but can be reinduced specifically upon antigen activation. GATA-3 plays critical functional roles in all of these stages, as summarized in Figure 3.

GATA-3 is implicated in regulating the genes that are needed in multiple stages of T cell development, from intrathymic targets such as RAG-2, TCR gene enhancers, and CD8, to the peripheral T cell effector IL-4 (37–41). Germline knockout experiments show that it is critical for the generation of any identifiable T cell precursors, either in vivo (in chimeras) or in vitro. Conditional knockout experiments, which delete GATA-3 only after the cells immigrate to the thymus, show that it is also needed for normal β-selection and proliferation in the DN to DP transition (36). Then it has a prominent role at positive selection, promoting the development of CD4 SP cells and blocking the development of CD8 SP cells (35, 36, 42). In peripheral T cells, it modulates cell survival and migration behavior (43) and plays a dominant role in the differentiation of CD4 T cells to a Th2 fate (reviewed in 44, 45).

In spite of its vital regulatory involvement in T cell developmental progression, GATA-3 is not a simple equivalent of dominant lineage regulators like erythroid GATA-1 or B cell EBF or Pax5. Forced expression of GATA-3 in nonthymic hematopoietic precursors does not enhance or accelerate T cell specification. Instead, it completely aborts T cell specification, and this shocking result is a consistent finding both in vivo and in vitro (T. Taghon, M. De Smedt, T. Kerre, J. Plum, E.V. Rothenberg & G. LeClercq, submitted for publication; T. Taghon & E.V. Rothenberg, in preparation; 46, 47) (Figure 3). These severely discordant results for gain and loss of function of GATA-3 are a hallmark of the T cell specification process (discussed below). If retroviral transduction is used to elevate GATA-3

expression later, in thymocytes, it continues to show inappropriately inhibitory effects, although GATA-3 effects on developmental choice at these later stages at least display the same sign as indicated in loss-of-function studies (35, 42, 47, 48). In mature peripheral T cells, on the other hand, gain and loss of function of GATA-3 give concordant results for the role of this factor in the Th1/Th2-lineage choice (44, 45, 49–51). This implies that T cell development not only relies repeatedly on GATA-3 but also continually adjusts the regulatory context of the cells to become more and more tolerant of GATA-3 at a wide range of doses.

A possible explanation for the difficult behavior of GATA-3 is that, like other GATA factors, it can have a wide range of biochemically distinct activities (40, 52, 53). It can carry out chromatin remodeling (IL-4), overt transcriptional activation (IL-5), and repression (IFNγ). Although the mechanisms through which GATA-3 represses are poorly characterized (54, 55), one mechanism is suggested by the presence of functional GATA-3 sites intermixed with SATB1 sites in a nuclear matrix attachment region of the CD8 locus, i.e., a locus that is turned off when GATA-3 expression promotes CD4 cell development (41). Different domains of the GATA-3 molecule are implicated in these different activities (52, 56, 57), supporting the idea that distinct functions are based on interactions with different sets of collaborating factors. This makes it plausible that developmental changes in the expression of other specific, potentially collaborating factors during T cell maturation could gradually disfavor access to non-T cell genes and protect critical T cell genes from inappropriate actions of GATA-3.

One intriguing potential cofactor for GATA-3 could be FOG-1, better known for its cooperative role with GATA-1 during erythrocyte and megakaryocyte differentiation. FOG-1 is expressed within the thymus and required for normal T cell development, since FOG-1-deficient thymocytes are unable to progress through β-selection (58), when GATA-3 activity levels appear to be high. However, it is still unclear how FOG-1 behaves in this setting as a GATA-3 cofactor. FOG-1 can inhibit GATA-3 activity in a Th2-promoting assay (58, 59), and thus its potential role might involve reducing GATA-3 activity at specific stages of T cell development. Titration of FOG-1 could potentially explain the toxicity of high level GATA-3 expression at an early stage. In any case, it is striking how subtly GATA-3 must be deployed during the time of its most critical T cell role, at the start of T cell development.

## E2A/HEB

T cells share with B cells a reliance on type I bHLH transcription factors (E proteins) for their development from an early stage (60–62). Thymocytes express not only E2A, the family member that is most important in B cell development, but also the closely related factors HEB (Tcf12) and E2-2 (Tcf4) (63). A number of T cell target genes are known to be positively regulated by this class of factors, including CD4 and the surrogate light chain gene pTα (64–66), and E protein binding sites are prominent in regulatory elements of RAG-1/RAG-2 recombinase

genes that are active in DN stages and in TCR gene enhancers. E2A also provides a unique survival function in conjunction with IL-7-dependent growth (67). Moreover, E2A/HEB dimers appear to act as positive regulators for another critical T cell transcription factor, GATA-3 (68).

Thus it is not surprising that forced expression of any of the Id factors (Idb, inhibitors of DNA binding), which heterodimerize with E2A-class bHLH factors to block their DNA binding, completely aborts early T cell specification (69, 70). In fact, forced expression of Id proteins can direct precursors into the NK lineage instead, a developmental pathway that in normal conditions is completely dependent on the Id2 gene (63). The E proteins are therefore among the essential directors of T-lineage development, and the balance between E protein and Id activity in multipotent precursors is critical to arbitrate the lineage choice between T/B and NK/DC/myeloid fates, as discussed below.

The functions of E proteins are discontinuous, however, even within the thymus (Figure 4). E2A in particular is most essential prior to the DP stage (71). Dominant negative mutants of HEB that interfere with all E proteins also block T cell development, no later than the DN to DP transition (72). However, as T cell precursors pass through alternating periods of proliferative expansion (DN2), stasis (DN3), and reactivation (DN4/ISP), E protein activation alternates with periods of inhibition, as pre-TCR- or TCR-triggered activation results in waves of Id3 induction that are functionally important for the success of selection (73–75). While Id proteins can support proliferation and other activation responses (76), E proteins actively promote G1 arrest at developmental checkpoints (77). Thus, although the loss of E protein function blocks T cell development completely, overexpression of E proteins in experimental systems does not perceptibly drive precursors into the T cell pathway.

![Diagram](https://i.imgur.com/yourimageurl.png)

Figure 4 Roles of Runx factors and the E protein/Id ratios in T cell development. The figure combines data for stages of action by Runx1 and Runx3, and the roles of Id2 and Id3 as antagonists of the E2A and HEB (E protein) bHLH factors. Note the alternating phases of E protein and Id dominance that are required for T cell development and the roles of Runx factors acting parallel to GATA-3 at β-selection and in opposition to GATA-3 at positive selection (see Figure 3).

612 ROTHENBERG ■ TAGHON

# NONLINEAGE-SPECIFIC REGULATORS WITH ESSENTIAL ROLES IN T CELL SPECIFICATION

The context in which the T-lineage identity factors operate is greatly affected by the action of additional factors of different structural groups. These other factors are just as critical for the generation of T cells as those discussed above, but they are not yet shown to be implicated in the T-lineage choice as compared with other cell fates. However, these factors continue to play important roles at specific choice points within the T lineage.

## Runx Factors

The three mammalian Runx factors are all expressed in the thymus and used throughout T cell development, although they are not T cell specific in expression or function (reviewed in 78, 79). GATA-3 and Runx3 are probably associated with thymocyte differentiation throughout the jawed vertebrates, as shown by the high-level coexpression of these factors specifically in the thymus of the cartilaginous fish *Raja eglanteria* (80). The most abundant Runx factor in the thymus (Runx1) is expressed as a legacy from hematopoietic stem cells, where it plays an indispensable role (81, 82). This early role dominates the phenotype of Runx1 germline knockout animals, since there is no development of any kind of lymphocytes when generation of definitive hematopoietic precursors is blocked. However, evidence from conditional Runx knockouts, domain-specific Runx knockouts, Runx transgenics, and regulatory mutations in specific T-lineage target genes together unambiguously indicates that Runx factors play specific, central roles in positive and negative gene regulation within the T cell differentiation program (79, 83–88). Runx factors promote DN cell expansion and β-selection, expression of CD8 in the DN to DP transition, and the selection and maturation of CD8 SP cells (Figure 4). The mechanisms involved may be complex, since thymocytes express a variety of splice and promoter-use isoforms of Runx genes that are predicted to encode functionally heterogeneous proteins (84, 89, 90).

Runx factors are poor transactivators alone, but capable of activating when they collaborate with partners binding to the same *cis*-regulatory module, such as Ets family factors or c-Myb. In the TCRα and TCRβ *cis*-regulatory elements, Runx/Ets collaboration provides essential positive regulatory function, while Runx/Myb collaboration activates the TCRγ and TCRδ *cis*-regulatory elements (91–96). Runx factors also provide substantial repressor activity at other target sites, recruiting Groucho/TLE/Grg family proteins or Sin3A by interaction with different C-terminal domains, and this is also important in T cell development. Runx1 activity in the expansion and differentiation of DN thymocytes depends, at least in part, on the Groucho-interaction domain (83).

Two specific targets of Runx repression have been analyzed in elegant detail. The CD4 gene turns out to be silenced by Runx1 in late DN thymocytes, before β-selection, and by Runx3 in CD8 SP thymocytes, after positive selection.
(84–86). Furthermore, Runx-dependent repression appears to be important for correct regulation of the RAG-1/2 locus (97). The mode of action in the case of RAG-1/2 is interesting because it imposes a default condition of repression, starting at β-selection, which is conditionally alleviated by factors binding a distal “antisilencing element.” This element is specific for permitting expression in DP thymocytes, and it is active only until positive selection occurs. If there must be a DP-specific mechanism for antagonizing Runx repression, this implies that Runx factors are present and active throughout the DP stage, as well as in DN and SP thymocytes.

Evidence now accumulating indicates that Runx factors (most likely Runx3) probably have a positive role in CD8 SP cell maturation (Figure 4), beyond their ability to repress CD4 (85, 88, 98, 99, 99a). Exactly how this works is not yet known. When overexpressed from the DN stage, any of the three Runx family members appears to accelerate β-selection to the CD8⁺ ISP stage, but then to block the CD4 expression that normally results in a DP phenotype (84, 87). It is not clear whether the strong Lck-dependent signals needed to induce positive selection to the CD4 lineage can be delivered to the cells in the face of this global CD4 repression. Thus Runx factors might drive positive selection toward the CD8 SP pathway not only through their positive effects on CD8 and their negative effects on CD4, but also through the indirect consequences of these alterations on strength of Lck signaling. Even independently of CD4 expression, TCR specificity, or duration of signaling, however, Runx3 appears to promote CD8 SP lineage development, suggesting a central role in developmental programming of cytolytic T cells (99a; T. Sato, S.-i. Ohno, T. Hayashi, C. Sato, K. Hayashi, K. Kohu, M. Satake & S. Habu, submitted for publication; Sonoko Habu, personal communication).

The reciprocal effects of Runx and GATA-3 in CD4/CD8 differentiation raise the question of whether these factors antagonize each other during positive selection. So far, one report (100) suggests that forced expression of Runx blocks GATA-3 upregulation in peripheral T cells and thus biases the Th1/Th2 choice against the Th2 fate. However, both GATA-3 and Runx1 are needed for efficient β-selection. The mechanisms through which GATA-3 and Runx factors could interact in the thymus remain to be explored.

## Ikaros Family Transcription Factors

Ikaros and its family members were the first transcription factors found to be essential for lymphocyte development, as distinct from development of other hematopoietic cell types (reviewed in 101). At least three members of the family are expressed in the thymus, in part continuing expression from earlier hematopoietic precursors. Ikaros itself is essential for B cell and fetal T cell development, and inhibition of the whole family by dominant-negative transgene expression eliminates T and NK cell development as well. While many hematopoietic genes possess Ikaros binding sites, there has been controversy about the mode of action of this regulator and whether it is predominantly activating or repressive (102, 103; reviewed in 104).
Much remains unresolved about the exact ways in which it interacts with the inputs from other lymphoid transcriptional regulators. However, the Ikaros family factors are demonstrably important at a number of successive stages: hematopoietic stem-cell maintenance, development of common lymphoid progenitors, β-selection and activation of CD8 expression, and the setting of thresholds for mitogenic activation of immature and mature T cells (103, 105–108). Recent data suggest that Ikaros could also have a role in determining when strong TCR signaling should lead to negative selection, rather than to CD4 SP positive selection (109).

One Ikaros family member, Helios, is expressed selectively in stem cells and in cells of the T cell lineage, with increasing expression during T cell specification and decreasing expression during B cell development. T-lineage-specific Helios expression is conserved, like GATA-3 and Runx3 expression, from mammals to cartilaginous fish (80). This suggests that Helios could play an important and specific role in T cell development, but the functional data needed to address this question are lacking.

At positive selection, the effects of Ikaros and the effects of Runx factors appear to be exerted in the same direction, generally facilitating CD8 expression and/or CD8 cell differentiation (85, 86, 88, 103). The ability of Ikaros factors to associate with chromatin-remodeling complexes (104) could be implicated in these actions, since the chromatin-remodeling component Brg appears to participate in some of the same regulatory events (110). Thus, future genetic and biochemical evidence may reveal that the driver of the positive actions of Ikaros factors in T cell development could be a complex involving Ikaros, together with Runx and Brg proteins.

## Critical Growth-Promoting Factors: TCF/LEF-1

TCF/LEF factors, the nuclear effectors of the Wnt pathway, are essential both at β-selection and at an earlier stage of IL-7-dependent expansion in the DN1 to DN2 transition (111–114). TCF-1 (Tcf7) is one of the most highly expressed transcription factors in developing thymocytes (E.-S. David & E.V. Rothenberg, unpublished data), with partially redundant expression of a closely related factor, LEF-1, in the fetal thymus. Antisense oligonucleotides against TCF-1 inhibit T cell generation from prethymic progenitors in fetal thymic organ culture (34). Normally, TCF-1 is converted from a repressor to a transcriptional activator by interaction with β-catenin (or the related protein plakoglobin), and it is the activating roles of TCF-1 that appear to be most important for T cell development, based on genetic manipulations of the TCF-1/β-catenin interaction interface (114). However, these effects appear primarily to reflect an essential role in proliferation, which is normally very extensive in T cell development, rather than in lineage decisions per se (115). Members of the TCF family including TCF-1 itself are shared between developing T cells and adult hematopoietic stem cells, where they play important roles in proliferation prior to differentiation (116). In mice, TCF-1 becomes increasingly important as a function of age for the waves of T cell

precursors that differentiate after birth, in order for these cells to reach the DN2 stage (112).

### c-Myb

The proto-oncogene c-Myb is also essential for T cell development, as well as essential for B cell development and for definitive hematopoietic stem-cell generation (117–119). Molecular evidence indicates complex, context-dependent roles for c-Myb in c-Kit, CD4, TCRγ, and TCRδ gene expression (120–122). It is most obviously involved in proliferative phases such as β-selection (123). Recently, through conditional inactivation at different stages of T cell development, the critical role for c-Myb has been confirmed at β-selection (124), in part associated with reduced rearrangement at the TCR-β locus. Furthermore, c-Myb is important for survival of DP thymocytes and specifically for the development of CD4 SP thymocytes (124).

### Hit-and-Run Specification Factors: PU.1

The last group of essential T cell regulators is different in that it both supports and constrains T-lineage development. This group is represented by PU.1, a divergent Ets subfamily member, which is expressed only in the earliest stages of T cell development, along with other stem-cell legacy genes such as GATA-2 and SCL/Tal-1 (125–127; T. Taghon, R. Pant & E.V. Rothenberg, unpublished data). A close relative of PU.1, Spi-B, is also expressed in DN thymocytes and then shut off at β-selection (125, 128). PU.1 is not expressed in mature T cells, and instead plays continuing roles in macrophage, granulocyte, and B cell development (129, 130). It is possible that the role of PU.1 for T cell precursors is mainly to provide a general proliferative function prior to specification; there is evidence that PU.1 acts this way in the erythroid lineage, where it also has a hit-and-run role (131). In early B cell precursors, PU.1 is important for IL-7Rα expression (132), and, conceivably, some of these cells are uncommitted precursors of T cells as well. PU.1 function is most important for the earliest precursors of the fetal cohorts of T cells (133). If its role is basically proliferative in these cells, it is curious that these are pre-T cell populations in which TCF/LEF function seems to be less critical (112). Conceivably, there could be a fetal-specific proliferative mechanism dependent on PU.1 that is replaced in postnatal lymphopoiesis by one dependent on Wnt/TCF-1.

While PU.1 gene disruption destroys fetal T cell development (126, 133), forced maintenance of PU.1 expression in precursors also blocks T cell development at early stages (134). Thus, PU.1 support of fetal T cell development is dose dependent, and also, obligatorily, a hit-and-run role. Besides supporting the proliferation of multilineage lymphoid precursors, PU.1 and Spi-B may in fact maintain the non-T-cell developmental options of these precursors, prior to T-lineage commitment (134, 135). The linkage between acquisition of T cell characteristics and closure of alternative options is discussed in detail below.

OTHER TRANSCRIPTION FACTORS WITH  
SPECIALIZED ROLES IN T CELL DEVELOPMENT  

The factors already listed do not fully account for all the lineage-specific aspects of T cell development. There are additional regulators that have potent impacts on a few selective aspects of T cell development in the thymus, or which are less well characterized in this context than those we have already introduced. Two widely used classes of transcription factors, Ets family factors and HOX cluster factors, are used in developing T cells, although the coexpression of multiple family members with overlapping functions makes it harder to use genetics to distinguish all the roles of individual genes. Another group consists of the T-box factors T-bet and eomesodermin and their collaborator Hlx. These are factors that are well known in peripheral T cell differentiation in response to antigen, but which may also play a role in the intrathymic differentiation of particular T and NK cell lineages. There are also a number of dedicated transcriptional repressors and nuclear localization- directing factors whose roles will eventually need to be explained.

## Ets Family Transcription Factors: Continuing Roles, Multiple Players

Transcription factors of the Ets family are strongly implicated in early T cell gene regulation. Binding sites for these factors are found in the regulatory regions of multiple lymphocyte differentiation genes, often linked to Runx sites and/or overlapping with Ikaros sites (92, 94, 95, 136). In the T cell lineage, Ets family function is guaranteed by redundancy. Ets-1, Ets-2, Fli, Tel, Elf, and GABPα are among the members of this family that are expressed throughout most of T cell development, with Erg and members of the divergent PU.1/Spi subfamily expressed for more limited periods (125). The overlapping expression patterns of these factors and their redundant DNA-binding specificities make it easy to underestimate their regulatory importance for T cell development if one only considers results of single-gene mutants (137). The effects of Ets-1 knockouts are sharper in NK cells, which do not express such high levels of other Ets family members (138). However, recent data indicate that GABPα may play a specific role in IL-7Rα gene expression in T-lineage cells (139).

## HOX Genes

Homeodomain-containing HOX genes are crucial mediators of patterning of the anterior/posterior axis during embryonic development, and increasing evidence points to their role as mediators of hematopoiesis. Primarily, genes of the HOX-A cluster seem to be expressed in hematopoietic cells (140). There is some evidence for a developmental expression pattern colinear with the order of genes in the cluster, as during embryogenesis, in some hematopoietic lineages and also during T cell development (141). Little functional evidence is available from HOX knockout studies yet due to their lethality, and T-lineage conditional inactivation

studies are lacking. HOX-A9, however, has been shown to be crucial for T cell development, as well as for certain other hematopoietic lineages. Mice deficient for this gene display a severe reduction in DN2 and DN3 thymocytes, possibly due to a lack of IL-7R expression (142).

# T-box Factors T-bet (Tbx21) and Eomesodermin, and their Collaborators

Functional maturation of multiple T cell subsets depends upon the Tbx-family factors, T-bet (Tbx21, Tbt-1) and Eomesodermin, which promote the ability to express IFN-γ, and may also activate granzyme B and perforin (55, 143–147). T-bet is expressed alone in activated CD4 conventional Th1 and NKT cells, whereas Eomesodermin contributes a partially redundant function in CD8 conventional T cells and in NK cells. In mature conventional TCRαβ CD4 cells, T-bet represents a critical node in a bistable regulatory network, where it promotes the establishment of Th1-type function in opposition to GATA-3 (reviewed in 44, 45). Not only is T-bet essential for postthymic differentiation of Th1 cells, but it also is specifically required for intrathymic generation of the whole NKT lineage of T cells (147).

To activate differentiation to a Th1 effector fate, T-bet induces expression of another factor, the divergent homeodomain transcription factor Hlx, which then collaborates with it to turn on IFNγ (148–150). Recent data suggest that T-bet can also activate one of the promoters of Runx1 (147), which could provide an indirect mechanism for it to inhibit GATA-3 expression in some contexts, as described above (100). In T cell subsets other than conventional TCRαβ cells, however, T-bet and GATA-3 appear to be mutually compatible in their contributions to function, as for example in TCRγδ cells, NK cells, and NKT cells (55, 147). (For excellent discussion of the other factors used for NK and NKT cell development, see 147, 151).

Although T-bet and eomesodermin are not T-lineage specific and are not yet known to play any early role in T cell precursor specification, there is early, regulated expression of T-bet in DN thymocytes (E.-S. David & E.V. Rothenberg, unpublished data). Direct perturbation experiments will be needed to determine whether precocious induction of these genes could play a role in the T/NK or the TCRαβ/TCRγδ lineage choices.

# TOX: A New HMG Box Factor

Both β-selection and positive selection are induced by TCR-mediated signaling, and a new transcription factor has been identified as a target of this signaling that plays a role in both processes. TOX is a novel member of the family of HMG box containing transcription factors that also comprises other important mediators of T cell development, such as TCF-1 and LEF-1 (152). TOX is specifically upregulated following pre-TCR signaling and positive selection, but, strikingly, does not seem to be involved in TCR-mediated activation of mature, peripheral T cells. Transgenic mice that express TOX under the control of the Lck promoter show

an increase in both ISP (post β-selection) and SP (post positive selection) CD8⁺ thymocytes. TOX induction during positive selection is calcineurin dependent and results in Runx3 upregulation, which in turn downregulates CD4 (153). This cascade can be counteracted by strong or sustained TCR signaling, a process that involves upregulation of GATA-3. Thus, the strength of TCR-mediated signaling at positive selection seems to change GATA-3 versus TOX ratios to result in CD4 versus CD8 lineage development (153), though this model awaits confirmation by knockout studies.

### Essential Repressors: Gfi-1, N-CoR1, and δEF1

Mutations in several genes coding for dedicated transcriptional repressors suggest that repression may be an essential aspect of T cell specification, at least at particular stages. The repressors analyzed to date are not T-lineage-specific in their expression and their roles in T cell development have mostly emerged as incidental features of a more global phenotype. But there is compelling evidence that losses of function of the repressors Gfi-1, N-CoR1, or δEF1 (ZEB, AREB6, Zfh, TCF-8) can each severely distort T cell development at specific stages. All three are normally expressed throughout most of intrathymic development, but the target genes that they must repress are not defined yet. While one can speculate that genes such as PU.1 should be important targets of stage-specific repression, very little is clear yet about the way these factors actually work to promote T cell development. The stages affected by loss of function of these three repressors are apparently different, although they have not been fully characterized to date.

Either a partial or a complete loss of function of the zinc finger-homeodomain repressor δEF1 causes a severe loss of thymic cellularity, with a particularly strong decrease in the earliest c-kit⁺ intrathymic populations and the DP cells (154). Some recovery occurs in later stages. The severity of this phenotype appears paradoxical, because δEF1 (ZEB) is capable of competing with E proteins to interfere with activation at E-box sites, and loss of δEF1 should favor activity of E proteins in T cell gene expression. There is even evidence that this repressor could downregulate GATA-3 expression itself (68). However, recall that both GATA-3 and E proteins can only promote T cell development when they themselves are under tight regulation (see above). A tonic level of repression of critical genes by δEF1 may also be important to maintain expression within tolerable limits (as for GATA-3) or to enforce specificity by requiring concerted action by positively regulating factors (155). δEF1 mutants also show derepressed integrin α4 expression, which could affect migration among appropriate zones of the thymus (154).

The nuclear receptor corepressor N-CoR1 has a strikingly important role in enabling thymocytes to develop through β-selection. In an N-CoR1 knockout, thymocytes are blocked almost completely at the DN to DP transition (156). Unfortunately, no further characterization is available. The DN to DP transition is a time of very rapid proliferation and rapid shifts in the requirements for E box proteins, their Id antagonists, TCF, and c-Myb, as described above. Availability of this

general-purpose repressive cofactor may be essential to mediate these regulatory shifts. Alternatively, the lack of N-CoR may have resulted in a failure to generate a pre-TCR in the first place, e.g., by interfering with TCR gene rearrangement.

The mutants of Gfi-1 have a subtler phenotype that affects particular stages of development both before and after β-selection. Gfi-1 is a repressor with 6 zinc fingers and a SNAG repression domain, and it is needed for normal development of neutrophils but not of granulocyte-macrophage precursors or of erythroid cells. In Gfi-1⁻/⁻ mutants, B cell development and T cell development are both partially inhibited, with clear bottlenecks in the progression of DN thymocytes to normal DN2 and DN3 states, depressed CD4 SP-cell production, and enhanced CD8-cell production (157, 158). These perturbations are consistent with the developmental pattern of Gfi-1 expression in the thymus, which is turned on in the DN2 stage and persists through the DP stage (159). Gfi-1 may be important to repress the bHLH antagonist Id proteins at the correct stages, since Id2 and Id1 are aberrantly highly expressed in the Gfi-1⁻/⁻ thymus (158). However, this could also be an indirect effect of abnormal precursor utilization in the thymus, since the DN1 subset in these mutants appears abnormal (158), lacking the c-kit⁺ IL-7Rˡᵒʷ cells that are the most potent T cell precursors in a normal thymus (108).

Some ambiguity attends the role of Gfi-1, in part because it has a close relative that is also expressed in the thymus, Gfi-1B. This repressor is similar in overall structure to Gfi-1, but diverges in sequence, and its effects are either redundant or antagonistic to those of Gfi-1 depending on the developmental context. Gfi-1B is expressed more narrowly in hematopoiesis, preferentially in erythroid lineage cells, where it is required (159–161). In thymocytes, Gfi-1B is reportedly expressed only in a sharp spike in the DN3 stage, where it may play a role in β-selection (159). It is not clear which factor, Gfi-1 or Gfi-1B, is more important during β-selection in vivo. However, there is evidence from a peripheral T cell system that Gfi-1 has another, distinctive function that could make it valuable in the T-lineage specification process. In mature peripheral T cells undergoing stimulation under Th2 conditions, Gfi-1 is induced in parallel with GATA-3. Then Gfi-1 permits efficient cell proliferation to proceed, counteracting the cytostatic effects of GATA-3 while preserving differentiative functions of GATA-3 (162). As noted above, prethymic T cell precursors can only tolerate low levels of GATA-3 (47), even though they cannot develop further without it. Thus the Gfi-1 expression could be important to provide a protective function for DN2/DN3 thymocytes, enabling them to couple strong proliferation with their differentiation in response to GATA-3.

## Chromatin Modifier SATB1

T cell development appears to require not only a multitude of repressors but also at least one factor that associates specific DNA sequences with the nuclear matrix, the Special AT-rich Binding protein SATB1. SATB1 is a two-Cut domain-Homeodomain protein that is expressed preferentially in thymocytes, and T cell

development is severely deranged, though not completely blocked, in SATB1 mutants (163). The activation properties of SATB1<sup>−/−</sup> peripheral T cells are also defective, resembling those of DP thymocytes or cells just undergoing positive selection, suggesting that the loss of SATB1 allows them to emigrate to the periphery without actually having matured. Although it does not act like a conventional sequence-specific transcription factor, SATB1 appears to be needed to choreograph a range of distinct gene regulation transitions during development. This factor is primarily known as a negative regulator, as it is essential for the correct timing of repression of the CD25 (IL-2Rα) and CD127 (IL-7Rα) genes in DP thymocytes. However, it is also needed for correct positive as well as negative regulation of the CD2, CD5, GABPα, CD8α, and c-Myc genes (41, 163, 164).

It now appears that the role of SATB1 is to propagate a particular chromatin configuration over a broad domain that can extend tens of kilobases from the SATB1 binding site itself (164, 165). The nuclear matrix interaction capability of SATB1 may play a physical part in this role, as recent evidence shows that a number of lymphocyte gene expression choices involve compartmentalizing nonexpressed alleles to the nuclear periphery (reviewed in 166). This makes SATB1 a very important effector of regulatory information-processing “decisions.” It is very provocative that developing thymocytes should require a dedicated, cell type–specific factor of this type. However, the very permissive sequence specificity of SATB1 and the situation-dependent way in which it engages different subsets of its target genes make it in some ways more akin to a general chromatin remodeling factor than to a sequence-specific transcription factor. These characteristics strongly imply that interactions with other, highly specific factor complexes are required to deploy SATB1 to the correct sites and perhaps to select its correct functions.

## PATTERNS OF COMBINATORIAL TRANSCRIPTION FACTOR ACTION IN T VERSUS B CELL DEVELOPMENT

We have already considered the activities of individual factors, “longitudinally” through T cell development. However, none of them work solo at any T cell developmental stage or choice point. The same factors used in specification are also used later in T cell differentiation, in combinations that shift from stage to stage, giving each choice point and checkpoint of T cell development a distinct regulatory signature. Each of these transitions can only be dissected mechanistically with reference to the combination of factors that acts at that point. To put the complexity of the T cell developmental process in perspective, it is helpful to compare it with the regulatory cascade that is now thought to specify the B lymphocyte lineage.

### A Comparative Model: The B Cell Gene Regulatory Network

The molecular genetics of B cell development have been elucidated dramatically since the late 1990s, as summarized in Figure 5 (reviewed in 7, 167–169). B cell

REGULATION OF T CELL DEVELOPMENT 621

![Diagram](attachment:diagram.png)

Figure 5 Gene regulatory network for B cell specification and commitment. This figure is based on the body of work cited in the text, in which B cell development depends on the combined action of three factors, E2A, EBF, and Pax-5. The figure highlights the network topology that is supported thus far by genetic epistasis experiments, as well as the distinctive roles of E2A, EBF, and Pax-5 in the positive and negative gene expression required for B-lineage commitment. Some evidence exists to link PU.1 action with the initiation of EBF expression, possibly involving the expression of IL-7Rα for optimized growth under permissive signaling conditions.

development from pluripotent hematopoietic precursors is guided by the three transcription factors E2A, EBF (Early B cell factor, a.k.a. Olf-1), and Pax5. These factors act in a relatively simple gene regulatory cascade in the initial stages of B cell specification. The precursor in which the specification process will unfold is normally rendered competent by its expression of PU.1 and Ikaros, without which B cell development does not begin (108, 132, 170). Increasing levels of E2A in such a precursor turn on EBF expression (171), possibly in collaboration with PU.1. In turn, EBF then collaborates with E2A, and/or with other bHLH factors of the same class if necessary (HEB, E2-2), to turn on expression of additional downstream genes and of Pax5 (169, 172, 173). Pax5 is not needed for the initiation of B cell gene expression, but it activates definitive B cell genes and locks down the committed B cell state by foreclosing other developmental options. It can directly block T cell specification, in part through downregulation of Notch (174), and it can interfere with correct regulation of myeloid target genes by PU.1 to disfavor additional options.

The polarity of this network is shown by the effects of knockouts and by the epistatic rescue of PU.1 or E2A mutants when EBF expression is turned on artificially (169, 173). Just as loss of expression of any of the B cell regulators interferes with B cell development, gain of expression of EBF or Pax5 enhances B cell development (172, 174, 175). Notably, these increases in B cell development in response to either EBF or Pax5 are primarily at the expense of T cell development. In general, the B cell–specific trio of E2A, EBF, and Pax5 are expressed and control gene expression throughout precursor differentiation to maturity. E2A and Pax5 then continue to be required in mature peripheral B cells, until Pax5 is shut off during the antigen-dependent terminal differentiation of B cells into plasma cells. The continued function of the B cell transcriptional cascade reflects the fact that these factors regulate not only the expression of genes used in particular stages of B cell development—surrogate light chains, RAG genes, and Activation Induced Deaminase (AID) at different stages—but also the transcriptional accessibility of the immunoglobulin genes in chromatin (176–179). These activities make the B cell–specification factors important for somatic hypermutation and heavy chain class-switching in mature, activated B cells as well as for initial VDJ rearrangement in their early precursors.

## T Lineage Versus B Lineage: An Overview

The paradigm developed for B cell development includes several features in common with T cell development. These are (*a*) the absolute requirements for bHLH factors (E2A, and/or HEB, E2-2) and for Ikaros family factors (Ikaros, Aiolos, Helios), and (*b*) the derivation of both B and T cells (at least in fetal life) from PU.1-expressing progenitors. Genes active in B cell precursors also share with those active in T cell precursors the importance of combinations of Runx and Ets transcription factor binding sites (95). Together, these observations implicate bHLH, Ikaros, PU.1, and Runx expression in shared aspects of T and B cell development. However, the two pathways differ markedly in the other specific factors that become activated in these precursors: EBF and Pax5 as opposed to GATA-3, TCF-1, and Notch/RBPSuh. More provocatively, they may differ also in the regulatory relationships among these factors and the developmental mechanisms used for T and B cell specification.

In comparison with B cell development, several process components appear to be lacking during T-lineage specification. There is no factor expressed selectively in T cell precursors that has an unequivocal positive role, an equivalent of EBF or Pax5 in B cell development. None of the T cell transcription factors has yet been able, when overexpressed, to drive the development of multipotent progenitors instructively into the T cell pathway, except the activation of Notch/RBPSuh activity, which is not T-lineage specific (180). There is also no clear identification yet of the factor(s) playing a lineage-specific commitment role through direct or indirect repression, a role that is played in B cell development by Pax5.

The T-lineage program itself includes multiple discontinuities and branch points, as shown in Figure 1, and the developmental choices that remain open each require

divergent actions of different transcription factor coalitions. Thus the transcription factors used in specification are repeatedly brought back into play, but with shifting patterns of cooperation and antagonism as maturation proceeds.

# A Succession of Different Regulatory Coalitions at Different Stages of T Cell Development

Figure 6 broadly summarizes major phases of action of key T cell transcription factors. The factors that work positively to establish T-lineage identity as distinct from other hematopoietic cell types, from the earliest stages, include Notch/RBPSuh, GATA-3, TCF/LEF factors, and E proteins such as E2A and HEB. As we discuss below in detail, levels of Notch, E protein, and GATA-3 activity as opposed to Id and PU.1 activity seem to be crucial to decide whether precursors will choose T as opposed to B, NK, or dendritic cell (DC) lineage fates. These factors collaborate with a broader group that establish the precursor subset(s) from which T-lineage specification can occur: c-Myb, Runx factors, Ikaros, and PU.1 and/or other Ets factors.

![Diagram](https://i.imgur.com/yourimage.png)

**Figure 6** Complex regulatory requirements of T cell development at multiple T-lineage checkpoints. The figure summarizes genetic analyses showing functional dependence of T cell development on shifting combinations of multiple specific transcription factors, loss of any of which blocks at the indicated stages. Events occurring in the periphery are not included. For details, see text. Black, extreme dependence (knockout has severe effect). Gray, moderate or uncertain dependence. White, no dependence, possible antagonist effect. Broken lines indicate stages when factors are likely expressed but genetic evidence for functional dependence is lacking.
The pausing of T cell precursors at the DN3 stage, and the enforcement of the $\beta$-selection checkpoint, involve both Ikaros family and E proteins. The cells express a particularly high level of active Notch and its target genes. In agreement with the high E protein activity, this is the period when the most active TCR gene rearrangement is unleashed by the highest RAG gene expression, and when the highest levels of pT$\alpha$ are expressed, favoring TCR$\alpha\beta$ precursor differentiation. Meanwhile, PU.1 and other non-T regulators are shut off at this point.

For TCR$\alpha\beta$ precursors, $\beta$-selection depends on a particularly broad coalition of transcriptional regulators. One set of factors is activated simply by the pre-TCR signal itself: This set includes immediate early Zn finger transcription factors of the Egr family, NF-AT, and NF-$\kappa$B (181–183). However, this is not simply an activation event; it is a mini-differentiation cascade that recalls virtually all the T cell identity-determining factors to participate in some kind of genetically required role.

To initiate $\beta$-selection, Notch signaling seems to be essential (15), though the Notch-activated target gene Hes-1 declines in expression throughout the DN to DP transition, suggesting that this signaling is transient. Meanwhile, E protein roles shift rapidly from the E protein-mediated DN3 arrest, through an Id3-dependent proliferative phase, then back to a phase when a particular E protein (HEB) becomes essential, as the cells approach the DP postmitotic state (74, 184). $\beta$-selection causes GATA-3 levels to rise, and GATA-3 activity becomes rate-limiting at least for the early phases of the DN to DP transition (32, 35, 36). At the same time, c-Myb becomes a critical proliferative driver (118, 123), Runx1 continues to be important for full population expansion (86), and TCF-1 or its relative LEF-1 becomes indispensable, especially in the later phases of the $\beta$-selection response (111, 113, 114). However, this is not a regulatory free-for-all. $\beta$-selection terminates the expression of the last PU.1/Spi family member, Spi-B, and the expression of other specific Ets and bHLH transcription factor variants that added to the regulatory complexity of the precursor stages (125; M.K. Anderson & E.V. Rothenberg, unpublished results). Conceivably, the repressors such as Gfi-1 and N-CoR that are required at this stage play a role in execution of these silencing events.

After $\beta$-selection, DP cells have a short default lifetime that can only be extended by successful positive selection to the CD4 or CD8 lineage. TCF-1 and c-Myb appear to act as viability factors (114, 124). At this point, regulatory requirements for further differentiation appear much simplified. TCR-activated signaling factors such as NF-AT family members, AP-1 and NF-$\kappa$B, are undoubtedly involved in the transient positive selection signal. There is evidence that strong, sustained signaling in positive selection directly activates the GATA-3 gene (35), whereas transient signaling selectively activates Runx3 expression (99). GATA-3 activity supports the differentiation of cells that are selected to the CD4-cell fate (35, 36). Conversely, when Runx3 is activated together with Ikaros it directs or sustains cells that proceed to the CD8-cell fate (84–86, 98, 99a, 103), with a possible contribution from Notch mediators as well (185).

In peripheral CD4 cells, GATA-3 and Runx again appear to be in opposition, but now they are associated with different collaborators than in the CD4/CD8 lineage
choice. Recent evidence suggests that for Th2 differentiation, GATA-3 once again works in parallel with some form of Notch signaling, as in the earliest stages of thymocyte specification (27), but in apparent contrast to the CD4/CD8 choice in positive selection. Meanwhile, in Th1 differentiation, Runx factors are presumably activated by T-bet (147) and act in collaboration with T-bet and Hlx.

Another important developmental branch point that is only starting to be explained is in the DN2 and DN3 stages, when TCRγδ cells diverge from the TCRαβ lineage. These two major subdivisions of T cells are biologically distinct throughout most vertebrates (186–188) and differ in a significant number of respects, only the first of which is the ability of TCRγδ cells to mature successfully without undergoing the regulatory upheaval of β-selection or the proliferative burst that accompanies it (189–193). TCRγδ cells differ from TCRαβ cells in their developmental responses to levels of E protein activity and to increases or decreases in Notch signaling levels (12, 13, 72, 194–198). Both Notch signaling and E protein activity are essential throughout the DN period for all T cell precursors, but the lower levels that seem to be acceptable for TCRγδ cell development could form a continuum with the levels that are permissive only for NK-cell development. The window of opportunity for the NK- and DC-lineage options stays open long enough to raise a question as to whether it overlaps with the branch point between TCRαβ and TCRγδ pathways of T cell development (196, 199). Thus it is possible that the choice between TCRαβ and TCRγδ T cell fates is mechanistically intertwined with the choice between T, NK, and DC fates.

Several other T-lineage choice points remain poorly explained. NKT cells depend on a constellation of regulators more typical of NK cells than of T cells (147, 200–203), and it is intriguing but obscure how this alternative program is accessed from precursors that presumably diverge only after β-selection (204). T<sub>reg</sub> cells, active guardians of tolerance, also appear to emerge in the thymus and begin to express the transcription factor Foxp3, but the intrathymic stimuli that direct this program likewise remain to be defined (205).

## T CELL LINEAGE SPECIFICATION AND COMMITMENT

Initial T cell specification and its relationship to commitment define an important frontier. This is the process that first demands the great regulatory complexity of T cell development that we have just reviewed, invoking the aid of a large fraction of the regulators that will be used in all subsequent steps of T cell development combined. Also, the regulatory properties of the earliest T cell precursors often appear close to those of leukemic cells, suggesting that enforcement of the correct regulatory relationships in this process may be critical to prevent malignant transformation. The last section of this review is therefore focused on what is known about the molecular mechanisms that initially establish T-lineage identity.

To date, it has not yet been possible to demonstrate a straightforward regulatory cascade, with clear polarity and epistasis relationships, for the transcription factors involved in T cell development. There are several reasons for this, not merely

technical but reflecting significant aspects of the T cell specification mechanism. First, the T cell development program can have multiple, diverse points of entry, in regulatory terms. Second, there is a massive overlap between the regulatory requirements of T cell specification and the hematopoietic stem-cell program. Third, there is the strikingly prolonged and flexible specification and commitment process itself. Finally, as a result, early T-lineage differentiation is intensely dependent on environmental conditions to maintain not only viability but also T-lineage identity.

# A Pathway with Multiple Entry Points

The first important step for T cell development to occur is the migration of a multipotent progenitor toward the thymus. Thymic immigration places the precursors in an environment rich with the Delta-like Notch ligands that are critical for T cell development (25, 26), and indeed this entry triggers a burst of Notch activation (206). Another feature that may be specific to the thymic microenvironment is high oxygen tension, which may be essential for T cell development (34, 207). Finally, the thymus is a rich source of ligands for the two growth factor receptors expressed on early T cell precursors, c-kit and IL-7, as well as Wnt factors that activate TCF-1/LEF-1 transcription factors. These distinctive microenvironmental features are important to initiate and sustain the specification process.

Recent evidence shows that the most likely cell to migrate from the bone marrow to the thymus under normal conditions has a Lin⁻ Sca-1⁺c-Kit⁺ (LSK) phenotype (208) (Figure 7). Cells with this phenotype, also found in the bone marrow, include self-renewing HSCs and multipotential progenitors. The thymus can thus initiate T cell development in cells that have diverse developmental alternatives. Moreover, this is not the only class of cells that is poised to respond to the thymic microenvironment.

There is now evidence for at least four different kinds of murine prethymic cells that can give rise to T cells when introduced into an appropriate environment, as shown in Figure 7 (209–213). Virtually all uncommitted T cell precursors are capable of differentiating into NK cells as well (16, 214), but they are distinguished in terms of their retention of B and/or myeloid potential. One is the Common Lymphoid Progenitor (CLP), a Sca-1lo c-kitlo IL-7Rα⁺ Lin⁻ bone marrow cell type. CLPs are also efficient B cell precursors but lack myeloid potential (215). A variant CLP (CLP-2) is also found in the bone marrow, with a c-kitlo B220⁺ phenotype and the capacity to generate both B and T cells (216). A different kind of precursor is an Early T cell Progenitor (ETP), first characterized among the highly immature cells in the thymus and thus part of the DN1 compartment. The ETP is a less efficient B cell precursor than the CLP, in agreement with its intrathymic appearance, but it is a better myeloid precursor (108). The ETP is Sca-1⁺ c-kit⁺ IL-7Rα⁻ Lin⁻ and differs from the CLP in that its generation and developmental capacities do not depend on Ikaros (108). Cells resembling ETP can be found in the bone marrow, where they represent a T lineage-biased, efficient source of

Peripheral
Blood

LSK
ETP
DN1
DN2
Thymus

CLP-2
Bone marrow

B potential
T potential
NK potential
Myeloid potential

Figure 7 Distinct precursor cell types with the shared competence to generate T cells in the thymus. The figure summarizes the studies reviewed in the text, where a variety of different adult cell types with different combinations of other developmental potentials are all capable of responding to the thymic microenvironment and differentiating into T cells. LSK: the Lin⁻ Sca-1⁺ c-Kit⁺ bone marrow cell fraction, highly pluripotent and recently shown to be the most prevalent source of thymus immigrating cells in normal animals. ETP: first defined as a partially restricted intrathymic subset of Early T cell Progenitors within the DN1 population, which appears to represent an early derivative of the immigrants after arrival in the thymus. ELP: the Early Lymphoid Precursor, a population with excellent T- and B-lineage potential that retains some myeloid potential but with much reduced nonlymphoid activity as compared with LSK cells. CLP and CLP-2: two distinct types of Common Lymphoid Progenitors, with strong B-lineage potential as well as T potential but little or no myeloid potential.

thymus-populating cells (217). Yet another cell type that may represent the precursor of both the ETP and the CLP, called the Early Lymphoid Precursor (ELP), has been described in the adult bone marrow and identified by its expression of a RAG1-GFP knock-in gene. The ELP can generate T, B, NK, and myeloid cells with different efficiencies (218) (Figure 7).

A somewhat different hierarchy of shared developmental potentials is found in prethymic cells in the fetus (not shown). In the fetal liver, the Sca-1⁺ c-kit⁺ Lin⁻ cells that give rise to T cell precursors generally exhibit substantial myeloid potential, and myeloid and T potentials are maintained together even when B cell potential is lost. The fetal precursors also differ in other respects from the prethymic cells in the adult (219–222).

While a simpler, more linear scheme for stepwise commitment might be more appealing, the variable order in which B versus myeloid potentials are lost among precursors appears to be a genuine feature of the cells competent to undertake

the T cell developmental pathway. Even within the adult thymus, phenotypically diverse T-lineage precursors can be found that give rise to T cells with different kinetics and with different combinations of alternative B, NK, and myeloid developmental capabilities (223). The predictable, robust differences in developmental potential among these precursors imply that they express different transcription factor combinations that situate them in distinct regulatory states. Thus, the gene regulatory network that operates in T-lineage specification must be capable of funneling this whole range of different input regulatory states into a common T cell developmental progression.

## Similarities Between Stem-Cell and T Cell Regulatory Programs

One way to force diverse precursors into a common program would be to express dominant transcriptional regulators that actively repress alternative, pluripotent states. However, this is not the way that T-lineage specification seems to proceed. Instead, there is a striking overlap between the factors required for the T-lineage specification process and factors known to be used, even required, in multipotent hematopoietic stem cells. Ikaros, PU.1, Runx1, and c-Myb are required not only for T cell specification; they are also important for stem-cell specification, maintenance, or self-renewal (106, 119, 224, 225). Notch pathway and Wnt/TCF signaling may also be used to promote stem-cell self-renewal (116, 226). Even GATA-3 is structurally very similar to the essential stem-cell factor GATA-2. These overlapping regulatory requirements obviously pose technical problems for gene knockout strategies to investigate the modes of action of these factors in early stages of T cell specification. More interestingly, though, they bring into sharp focus a global regulatory similarity between the stem-cell state and the state of a T cell precursor even after specification and commitment (Figure 8).

The number of regulatory changes needed to bridge the gap from stem cell/multipotent progenitor to DN2 thymocyte could be small. Among the obvious candidates, lymphoid precursors might diverge from multipotent progenitors by altering the relative activities of different bHLH heterodimers and their interaction partners (e.g., via downregulation of Id1, SCL/Tal-1, and Lmo1/2); but the essential positive bHLH factors, E2A and HEB, are expressed in stem cells as well. One clear difference is that GATA-3 is induced, whereas GATA-2 is downregulated. Furthermore, GATA-3 might be an important mediator in the migration of LSK cells to the thymus, since GATA-3 deficiency results in the absence of even the earliest, DN1-stage thymocytes (32). Thus, if not important for migration, it at least is required for the survival of the earliest thymocytes. However, if GATA-3 can serve any of the same functions as GATA-2, then the continuities between early stage T cells and stem cells are even more impressive (Figure 8). It should not be surprising, therefore, that the T cell pathway is also accessible from a number of different, separable pathways from the multipotent LSK cell (Figure 7).

Basophil  
Mega-  
karyocyte  
Erythrocyte  
Eosinophil  
Neutrophil  
Dendritic  
Mac/Mono  

HSC  

Mast  
NK  
B  

TCR γδ  
CD8  
T-reg  
NKT  
Th1  
Th2  

Thymus  

Bone marrow  

Ikaros  
PU.1  
c-Myb  
Runx1  

essential  

E2A  
HEB  

present  

GATA-2  

comparable  

Ikaros  
PU.1  
c-Myb  
Runx1  

E2A  
HEB  

GATA-3  

Figure 8 Similarities between T cell and stem-cell regulatory states. In comparison with hematopoietic stem cells (center cell, left), committed T cell precursors also remain “pluripotent” (center cell, right) insofar as they still face a succession of developmental choices that determine divergent functions, gene expression programs, and physiological setpoints within the T lineage. Transcription factors that are genetically shown to be important for development or maintenance of definitive HSC (bottom left) define a very similar set to those that are important for T-lineage specification (bottom right).

Timing of Specification and Lineage Choice Events

Specification involves the activation of a first tier of T-lineage genes, and this must be one of the key roles of the combinations of factors involved in this process. The genes that need to be turned on in order to qualify a precursor cell for success in early T-lineage development include those encoding RAG-1 and RAG-2 recombinase components, the CD3γ and ε chains of the TCR and pre-TCR complexes, lineage-specific signaling molecules such as LAT and Lck, and for future TCRαβ

cells, the surrogate $\alpha$ chain pT$\alpha$. Also required for TCR gene rearrangement is the transcription factor-mediated opening of the respective loci in chromatin, and for survival through the initial stages IL-7R$\alpha$ and $\gamma$c expression are needed. The T-lineage-required transcription factors themselves must also be turned on, at the correct times and levels, to regulate these and other, as yet unknown, target genes. Certain *cis*-regulatory elements associated with TCR (38, 91, 93, 94, 227), pT$\alpha$ (65, 66, 228), Lck (229, 230), and RAG (39, 97, 231) genes appear to include targets for RBPSuh, c-Myb, bHLH E proteins, Runx1, GATA-3, and/or TCF/LEF factors. However, remarkably few of the regulatory systems that control early T cell genes have been characterized in sufficient detail to establish factor-specific causality.

Another regulatory feature needed for optimal T cell development is extensive proliferation. For the proliferation that occurs in the DN2 stage, E proteins must collaborate with IL-7R$\alpha$ signaling to maintain viability (67), while TCF-1, c-Myb, and the Notch-activated transcriptional repressor Hes-1 are vital to sustain clonal expansion (112, 118, 232). “Normal” development of individual precursors can involve $\sim 10^5$-fold expansion, with $\sim 10^3$-fold expansion prior to $\beta$-selection (233). Thus, another requirement of the specification process may be a surprising one: to delay terminal differentiation, while proliferation continues.

Indeed, a remarkable feature about T-lineage commitment is how late it can occur, even relative to specification. Figure 1 shows the approximate windows of opportunity for development into B, myeloid, NK, and dendritic cells as alternatives to T cell development. The full foreclosure of all of these alternatives does not appear to occur until the DN3 (c-kit$^-$ CD44$^-$ CD25$^+$) stage (reviewed in 211; 16, 234). This can take 7–10 rounds of proliferation in the fetal thymus (233) and nearly two weeks in the adult thymus (235). Thus, DN3-stage commitment is late in terms of the onset of T-lineage gene expression, which is clearly under way by the DN2 stage; it is late relative to the beginning to TCR gene rearrangement; and in the adult thymus, it is late in terms of the absolute number of days and cell cycles elapsed since the entry of the cells into the thymus (reviewed by 212, 235, 236).

Whereas other potentials are eliminated earlier, both postnatal and fetal thymocytes remain capable of differentiating into dendritic cells and/or NK cells instead of T cells throughout the DN1 and DN2 stages (63, 234, 237–240) (Figure 1). In the fetal thymus, these subsets are capable of macrophage differentiation as well (241). B cell potential is lost much earlier within the thymus and is undetectable among most of the subsets within the DN1 population (212, 223), presumably as a result of direct inhibition via Notch/Delta signaling in the thymic microenvironment (206). Both the NK and the dendritic cell fates, though less favored by Notch/Delta signaling than the T cell fate, are substantially more tolerant of this than the B cell fate (25, 197, 242). These alternatives can be elicited simply by incubation of the DN1 or DN2 thymocytes under appropriate growth conditions, i.e., by moving them into an irradiated host or into in vitro conditions that differ from those presented by the thymic microenvironment. As the DN1 and DN2 stages include the highest frequencies of proliferating cells in the thymus except

for those immediately responding to $\beta$-selection, this implies that the uncommitted state is actively maintained.

At the individual cell level, even turning on T cell genes does not guarantee commitment. RAG-negative NK cells can develop from RAG1-expressing precursors (218). Myeloid-like dendritic cells can develop from cells that have previously turned on pT$\alpha$, the component of the pre-TCR, or active RAG recombinase, as shown by the presence of limited lymphoid-type gene rearrangements in their DNA (243). The likelihood of *trans*-specification during the DN1 and DN2 stages can also be boosted dramatically by delivering artificial stimulation to the immature thymocytes. Signals delivered through an ectopically expressed IL-2 receptor or GM-CSF receptor complex can convert most DN2 cells quickly to myeloid cells, even if they have already undergone significant TCR gene rearrangement (244, 245). Once the cells are committed, in the DN3 stage, this response is no longer seen. The exceptional plasticity of DN1 and DN2 cells shows that the early stages of the T-lineage program are compatible with expression of regulatory factors that can execute alternative developmental programs.

## Coexpression of Potentially Antagonistic Transcription Factors During Specification

Many of the factors required for T cell specification have spectra of activity that can include inhibitory as well as stimulatory effects on T cell development. The extended window of opportunity for developmental alternatives could be the result of potentially antagonistic activities among these delicately balanced factors (211) (Figure 9).

From the prethymic stages through the DN2 stage, cells have to depend on GATA-3, Notch1/RBPSuh, E box proteins, c-Myb, and Ikaros in order to carry out T-lineage specification, while Ets factors, TCF-1 or LEF-1, Runx factors, and probably PU.1 are used in essential supporting roles. Initially, Id2 is expressed as well, particularly in fetal T cell precursors (63). In other contexts, GATA and PU.1 factors are mutually antagonistic for erythroid versus myeloid development (246–248); Notch and E box proteins are antagonists with respect to B cell development (249, 250); and Runx and GATA-3 appear to have antagonistic effects in CD4/CD8 lineage divergence and possibly Th1/Th2 divergence as well, as described above. The prolonged coexpression of potential antagonists makes a sharp contrast with the kind of positive feedback networks seen to drive terminal differentiation in Th1 or Th2 cells, or in certain embryonic systems (251). However, these factors are all used together, and are all genetically required, during these critical but unstable stages of T cell development.

Overexpression of many of these factors can tilt the balance to a non-T developmental fate or else block proliferation (Figure 9). As already discussed, the E protein/Id balance is critical for T cells to develop instead of NK cells, with Id2 normally promoting the NK fate. However, E proteins block proliferation. Id2 is essential for NK-cell development, and loss-of-function experiments show

632 ROTHENBERG ■ TAGHON

![Diagram](#)

Figure 9 T-lineage specification by regulatory balance among drivers of competing developmental options. As discussed in the text, T-lineage specification in the thymus requires coexpression of factors that promote (PU.1) and inhibit (GATA-3?) the dendritic cell or myeloid program, and factors that promote (Id2) and inhibit (E2A/HEB) the NK cell program, while Notch signaling with a possible contribution from GATA-3 blocks a B cell program that would otherwise be accessible. Of the regulators shown on this diagram, only EBF and Pax-5 are normally not expressed in early thymocytes, possibly due to a direct inhibitory influence from Notch/Delta interaction (T. Taghon, E.-S. David, J.C. Zúñiga-Pflücker & E.V. Rothenberg, submitted). See text for details.

that it is responsible for the NK developmental potential of the most immature thymocyte subsets (63, 151). Id2 is also associated with development of monocyte/macrophages and certain classes of dendritic cells, fates that could be promoted by PU.1 (252–254). Either PU.1 or its relative Spi-B can support dendritic cell development. Forced expression of PU.1 blocks T cell development in thymic organ cultures, with cells arresting in the DN stages and most of the surviving cells showing a myeloid-like phenotype (134; A.H. Weiss & E.V. Rothenberg, unpublished data). The close relative of PU.1, Spi-B, is highly expressed in dendritic cells, and its overexpression redirects T cell precursors to full dendritic-cell differentiation (135; J.M. Lefebvre, M.C. Haks, M.O. Carleton, M. Rhodes, G. Sinnathamby, L.A. Garrett-Sinha, M.C. Simon, L.C. Eisenlohr, D.L. Wiest, submitted for publication; D.L. Wiest, personal communication). Notch signaling in some hematopoietic progenitors actually upregulates PU.1 and thus promotes myeloid development (180), a response that must be restrained in the Notch-dependent T cell precursors.

Even GATA-3 can be problematic for early T cell development. Whereas GATA-3 may be useful as an antagonist of the dendritic cell and B cell pathways, when expressed at high levels, it can also be “read” in uncommitted prethymic cells as a driver of a non-T-lineage fate. In murine bone marrow stem cells, it promotes

megakaryocyte development inappropriately, while blocking all lymphoid development (46), suggesting that it cross-reacts with GATA-1 or GATA-2 target genes. Not all of these alternative cell types can be kept viable in the thymic microenvironment, but it is likely that the developmental choices observed there emerge from fluctuations in the balance of the regulators that are used for T-lineage specification as well.

In summary, the combination of transcription factors employed for the establishment of T-lineage identity can be seen as establishing a kind of regulatory tug-of-war during the stages of initial proliferation. The balance among these factors itself appears to be critical to maintain the combination of early T cell gene expression and optimal proliferation. In addition, these cells relinquish their pluripotentiality only by degrees, and even then, give up non-T alternatives only to remain capable of a multiplicity of developmental choices within the T lineage, at later stages of differentiation (Figure 1).

## Role of the Microenvironment as a Guide to Commitment

During early T cell development, these potentially diversionary transcription factors are normally limited to low levels of expression that are not sufficient to redirect too many thymocytes from the T cell pathway. However, they create a potential regulatory “hazard” for T-lineage fidelity that can be exacerbated by stimulation. PU.1 and Spi-B, for example, are strongly enhanced in their transcriptional activity by phosphorylation or by association with the activation-dependent transcription factor c-Jun (255–259). Id2 can be induced to high levels by certain kinds of mitogenic stimulation (reviewed in 260). This could be the mechanism through which stimulation of fetal thymocytes via FcR crosslinking drives them to differentiate into NK cells (261).

How do rapidly proliferating DN2 thymocytes avoid these liabilities? Notably, the IL-7R complex that normally drives most thymocyte expansion prior to commitment is a receptor that is particularly weak at MAP kinase activation (262, 263). In contrast, the IL-2 receptor β chain CD122, which activates MAP kinases much better, is confined to precursors that are strongly biased to an NK fate (151, 264). As a result, IL-2/IL-15R or certain other kinds of growth factor stimulation may substantially enhance the likelihood of diversion. Stimulation through ectopically expressed IL-2Rβ (CD122) or GM-CSF receptors can drive the PU.1+ subsets of thymocytes to monocytic or granulocytic differentiation, without any direct manipulation of PU.1 expression (244, 245). Such extracellular signals only determine cell fate in convergence with the intracellular regulatory environment, however, because the PU.1-negative subsets of thymocytes do not respond. Thus, maintaining repression of IL-2Rβ and GM-CSF receptors may be critical throughout the PU.1-positive stages to keep T-lineage commitment available. At the same time, maintenance of strong Notch/Delta signaling appears to be important to avoid diversion to the NK cell lineage throughout the DN1 and DN2 stages, possibly due to the expression of Id2 by these cells (16).

Commitment is the stage at which this hazard is finally removed. Ultimately, the committed cells are ones in which genes such as PU.1 are turned off. Presumably by this stage at the latest, some of the interaction partners that facilitate inappropriate effects of GATA-3 are also permanently repressed. Commitment is also the stage when a new criterion for survival and proliferation is imposed: The cells lose their ability to survive without signaling through the TCRγδ or pre-TCR complex. It thus coincides with the forcible end of the TCR-independent phase of T cell development. While the basis for this change is not clear yet, it is tempting to connect it with the extinction of certain stem-cell or pluripotent progenitor functions. Conversely, the shift in survival requirements can also act as a built-in quality control for lineage fidelity, for any regulatory perturbation from this point on that happens to interfere with the TCR complex components or their signal transduction mediators will now cause cell death.

# NEW OPPORTUNITIES: ACCESS TO THE SPECIFICATION PROCESS IN THE ABSENCE OF A THYMUS

T-lineage specification emerges from a complex interaction of multiple regulators of which none plays a simple, dominant role. The gene regulatory network that controls this process can be clarified only when it is possible to test the effects of individual perturbations in a focused, stage-specific way. In the past, this has been a practical challenge due to “black box” aspects of the closed thymic microenvironment in which so much of T cell specification must take place. Now, however, a new approach opens up access to cellular and molecular mechanisms operating in these early stages.

The increased understanding of cues supplied by the thymic microenvironment has made possible an exciting advance in T cell developmental biology. The OP9-DL1 stromal cell system developed by Zúñiga-Pflücker and coworkers is a monolayer culture transfected to express Delta-like 1, providing an environment that supports T cell precursors in the presence of IL-7 and Flt3L. This system promotes T cell development all the way to positive selection from the most primitive precursors, even from embryonic stem cell lines (16, 26, 265). In this system, the time course of gene activation from the earliest stages can be tracked and linked with the time course of developmental potential change (265; T. Taghon, E.-S. David, J.C. Zúñiga-Pflücker & E.V. Rothenberg, submitted). The reversibility of these events can be probed as a function of time (T. Taghon, E.-S. David, J.C. Zúñiga-Pflücker & E.V. Rothenberg, submitted), and the cells can be accessed for perturbation of the process at any stage.

The quantitative recovery and easy accessibility of precursors in cultures like these makes it possible for the first time to dissect the molecular control of T-lineage specification in real time. With the studies based on this new system, future reviews of the molecular genetics of T cell development will at last be able to illuminate

the network of specific interactions among regulators and their causal impacts on developmental change.

# ACKNOWLEDGMENTS

Any review like this embodies insights drawn from many people besides the authors themselves. The bibliography understates this debt. We are grateful to many colleagues for sharing their results and interpretations prior to publication, and we apologize to the many whose work we were not able to cite, or could cite only in secondary references. Work in the Rothenberg lab on transcriptional regulation of T cell development and its evolution was supported by grants from the NIH (R01 CA90233, R01 CA98925) and from NASA (NAG2-1588), with previous support from NSF (MCB-9983129) and the Stowers Institute for Medical Research.

## The Annual Review of Immunology is online at
http://immunol.annualreviews.org

## LITERATURE CITED

1. Borowski C, Martin C, Gounari F, Haughn L, Aifantis I, et al. 2002. On the brink of becoming a T cell. *Curr. Opin. Immunol.* 14:200–6
2. Cantrell DA. 2002. Transgenic analysis of thymocyte signal transduction. *Nat. Rev. Immunol.* 2:20–27
3. Singer A. 2002. New perspectives on a developmental dilemma: the kinetic signaling model and the importance of signal duration for the CD4/CD8 lineage decision. *Curr. Opin. Immunol.* 14:207–15
4. Alberola-Ila J, Hernández-Hoyos G. 2003. The Ras/MAPK cascade and the control of positive selection. *Immunol. Rev.* 191:79–96
5. Bosselut R. 2004. CD4/CD8-lineage differentiation in the thymus: from nuclear effectors to membrane signals. *Nat. Rev. Immunol.* 4:529–40
6. Rothenberg EV, Yui MA, Telfer JC. 2003. T-cell developmental biology. In *Fundamental Immunology*, ed. WE Paul, pp. 259–301. Philadelphia: Lippincott, Williams & Wilkins
7. Busslinger M. 2004. Transcriptional control of early B cell development. *Annu. Rev. Immunol.* 22:55–79
8. Pear WS, Radtke F. 2003. Notch signaling in lymphopoiesis. *Semin. Immunol.* 15:69–79
9. Robey EA, Bluestone JA. 2004. Notch signaling in lymphocyte development and function. *Curr. Opin. Immunol.* 16:360–66
10. Radtke F, Wilson A, Mancini SJ, MacDonald HR. 2004. Notch regulation of lymphocyte development and function. *Nat. Immunol.* 5:247–53
11. Han H, Tanigaki K, Yamamoto N, Kuroda K, Yoshimoto M, et al. 2002. Inducible gene knockout of transcription factor recombination signal binding protein-J reveals its essential role in T versus B lineage decision. *Int. Immunol.* 14:637–45
12. Tanigaki K, Tsuji M, Yamamoto N, Han H, Tsukada J, et al. 2004. Regulation of αβ/γδT cell lineage commitment and peripheral T cell responses by Notch/RBP-J signaling. *Immunity* 20:611–22
13. Wolfer A, Wilson A, Nemir M, MacDonald HR, Radtke F. 2002. Inactivation of Notch1 impairs VDJβ rearrangement and allows pre-TCR-independent survival of early αβ lineage thymocytes. *Immunity* 5:869–79

14. Huang EY, Gallegos AM, Richards SM, Lehar SM, Bevan MJ. 2003. Surface expression of Notch1 on thymocytes: correlation with the double-negative to double-positive transition. *J. Immunol.* 171:2296–304

15. Ciofani M, Schmitt TM, Ciofani A, Michie AM, Cuburu N, et al. 2004. Obligatory role for cooperative signaling by pre-TCR and Notch during thymocyte differentiation. *J. Immunol.* 172:5230–39

16. Schmitt TM, Ciofani M, Petrie HT, Zúñiga-Pflücker JC. 2004. Maintenance of T cell specification and differentiation requires recurrent Notch receptor-ligand interactions. *J. Exp. Med.* 200:469–79

17. Choi JW, Pampeno C, Vukmanovic S, Meruelo D. 2002. Characterization of the transcriptional expression of Notch-1 signaling pathway members, Deltex and HES-1, in developing mouse thymocytes. *Dev. Comp. Immunol.* 26:575–88

18. Deftos ML, Huang E, Ojala EW, Forbush KA, Bevan MJ. 2000. Notch1 signaling promotes the maturation of CD4 and CD8 SP thymocytes. *Immunity* 13:73–84

19. Bellavia D, Campese AF, Alesse E, Vacca A, Felli MP, et al. 2000. Constitutive activation of NF-κB and T-cell leukemia/lymphoma in Notch3 transgenic mice. *EMBO J.* 19:3337–48

20. Aster JC, Xu L, Karnell FG, Patriub V, Pui JC, Pear WS. 2000. Essential roles for ankyrin repeat and transactivation domains in induction of T-cell leukemia by Notch1. *Mol. Cell. Biol.* 20:7505–15

21. Allman D, Karnell FG, Punt JA, Bakkour S, Xu L, et al. 2001. Separation of Notch1 promoted lineage commitment and expansion/transformation in developing T cells. *J. Exp. Med.* 194:99–106

22. Deftos ML, Bevan MJ. 2000. Notch signaling in T cell development. *Curr. Opin. Immunol.* 12:166–72

23. Basson MA, Zamoyska R. 2000. The CD4/CD8 lineage decision: integration of signalling pathways. *Immunol. Today* 21:509–14

24. Hernández-Hoyos G, Alberola-Ila J. 2003. A Notch so simple influence on T cell development. *Semin. Cell Dev. Biol.* 14:121–25

25. Jaleco AC, Neves H, Hooijberg E, Gameiro P, Clode N, et al. 2001. Differential effects of Notch ligands Delta-1 and Jagged-1 in human lymphoid differentiation. *J. Exp. Med.* 194:991–1002

26. Schmitt TM, Zúñiga-Pflücker JC. 2002. Induction of T cell development from hematopoietic progenitor cells by Delta-like-1 in vitro. *Immunity* 17:749–56

27. Amsen D, Blander JM, Lee GR, Tanigaki K, Honjo T, Flavell RA. 2004. Instruction of distinct CD4 T helper cell fates by different Notch ligands on antigen-presenting cells. *Cell* 117:515–26

28. Yvon ES, Vigouroux S, Rousseau RF, Biagi E, Amrolia P, et al. 2003. Overexpression of the Notch ligand, Jagged-1 induces alloantigen-specific human regulatory T cells. *Blood* 102:3815–21

29. Maekawa Y, Tsukumo S, Chiba S, Hirai H, Hayashi Y, et al. 2003. Delta1-Notch3 interactions bias the functional differentiation of activated CD4⁺ T cells. *Immunity* 19:549–59

30. Palaga T, Miele L, Golde TE, Osborne BA. 2003. TCR-mediated Notch signaling regulates proliferation and IFN-γ production in peripheral T cells. *J. Immunol.* 171:3019–24

31. Ting C-N, Olson MC, Barton KP, Leiden JM. 1996. Transcription factor GATA-3 is required for development of the T-cell lineage. *Nature* 384:474–78

32. Hendriks RW, Nawijn MC, Engel JD, van Doorninck H, Grosveld F, Karis A. 1999. Expression of the transcription factor GATA-3 is required for the development of the earliest T cell progenitors and correlates with stages of cellular proliferation in the thymus. *Eur. J. Immunol.* 29:1912–18

33. Samson SI, Richard O, Tavian M, Ranson T, Vosshenrich CA, et al. 2003. GATA-3 promotes maturation, IFN-γ production,

and liver-specific homing of NK cells. *Immunity* 19:701–11

34. Hattori N, Kawamoto H, Fujimoto S, Kuno K, Katsura Y. 1996. Involvement of transcription factors TCF-1 and GATA-3 in the initiation of the earliest step of T cell development in the thymus. *J. Exp. Med.* 184:1137–47

35. Hernández-Hoyos G, Anderson MK, Wang C, Rothenberg EV, Alberola-Ila J. 2003. GATA-3 expression is controlled by TCR signals and regulates CD4/CD8 differentiation. *Immunity* 19:83–94

36. Pai SY, Truitt ML, Ting CN, Leiden JM, Glimcher LH, Ho IC. 2003. Critical roles for transcription factor GATA-3 in thymocyte development. *Immunity* 19:863–75

37. Zhang D-H, Yang L, Ray A. 1998. Cutting edge: differential responsiveness of the IL-5 and IL-4 genes to transcription factor GATA-3. *J. Immunol.* 161:3817–21

38. Tripathi RK, Mathieu N, Spicuglia S, Payet D, Verthuy C, et al. 2000. Definition of a T-cell receptor β gene core enhancer of V(D)J recombination by transgenic mapping. *Mol. Cell. Biol.* 20: 42–53

39. Kishi H, Wei X-C, Jin Z-X, Fujishiro Y, Nagata T, et al. 2000. Lineage specific regulation of the murine RAG-2 promoter: GATA-3 in T cells and Pax-5 in B cells. *Blood* 95:3845–52

40. Lee GR, Fields PE, Flavell RA. 2001. Regulation of IL-4 gene expression by distal regulatory elements and GATA-3 at the chromatin level. *Immunity* 14:447–59

41. Kieffer LJ, Greally JM, Landres I, Nag S, Nakajima Y, et al. 2002. Identification of a candidate regulatory region in the human CD8 gene complex by colocalization of DNase I hypersensitive sites and matrix attachment regions which bind SATB1 and GATA-3. *J. Immunol.* 168:3915–22

42. Nawijn MC, Ferreira R, Dingjan GM, Kahre O, Drabek D, et al. 2001. Enforced expression of GATA-3 during T cell development inhibits maturation of CD8

single-positive cells and induces thymic lymphoma in transgenic mice. *J. Immunol.* 167:715–23

43. Yamagata T, Mitani K, Oda H, Suzuki T, Honda H, et al. 2000. Acetylation of GATA-3 affects T-cell survival and homing to secondary lymphoid organs. *EMBO J.* 19:4676–87

44. Murphy KM, Reiner SL. 2002. The lineage decisions of helper T cells. *Nat. Rev. Immunol.* 2:933–44

45. Ho IC, Glimcher LH. 2002. Transcription: tantalizing times for T cells. *Cell* 109(Suppl.):S109–20

46. Chen D, Zhang G. 2001. Enforced expression of the GATA-3 transcription factor affects cell fate decisions in hematopoiesis. *Exp. Hematol.* 29:971–80

47. Anderson MK, Hernandez-Hoyos G, Dionne CJ, Arias A, Chen D, Rothenberg EV. 2002. Definition of regulatory network elements for T-cell development by perturbation analysis with PU.1 and GATA-3. *Dev. Biol.* 246:103–21

48. Taghon T, De Smedt M, Stolz F, Cnockaert M, Plum J, Leclercq G. 2001. Enforced expression of GATA-3 severely reduces human thymic cellularity. *J. Immunol.* 167:4468–75

49. Nawijn MC, Dingjan GM, Ferreira R, Lambrecht BN, Karis A, et al. 2001. Enforced expression of GATA-3 in transgenic mice inhibits Th1 differentiation and induces the formation of a T1/ST2-expressing Th2-committed T cell compartment in vivo. *J. Immunol.* 167:724–32

50. Pai SY, Truitt ML, Ho IC. 2004. GATA-3 deficiency abrogates the development and maintenance of Thelper type 2 cells. *Proc. Natl. Acad. Sci. USA* 101:1993–98

51. Skapenko A, Leipe J, Niesner U, Devriendt K, Beetz R, et al. 2004. GATA-3 in human T cell helper type 2 development. *J. Exp. Med.* 199:423–28

52. Ranganath S, Murphy KM. 2001. Structure and specificity of GATA proteins in Th2 development. *Mol. Cell Biol.* 21: 2716–25

53. Klein-Hessling S, Jha MK, Santner-Nanan B, Berberich-Siebelt F, et al. 2003. Protein kinase A regulates GATA-3-dependent activation of IL-5 gene expression in Th2 cells. *J. Immunol.* 170:2956–61

54. Usui T, Nishikomori R, Kitani A, Strober W. 2003. GATA-3 suppresses Th1 development by downregulation of Stat4 and not through effects on IL-12Rβ2 chain or T-bet. *Immunity* 18:415–28

55. Yin Z, Chen C, Szabo SJ, Glimcher LH, Ray A, Craft J. 2002. T-Bet expression and failure of GATA-3 cross-regulation lead to default production of IFN-γ by γδ T cells. *J. Immunol.* 168:1566–71

56. Lee HJ, Takemoto N, Kurata H, Kamogawa Y, Miyatake S, et al. 2000. GATA-3 induces T helper cell type 2 (Th2) cytokine expression and chromatin remodeling in committed Th1 cells. *J. Exp. Med.* 192:105–15

57. Takemoto N, Arai K, Miyatake S. 2002. Cutting edge: the differential involvement of the N-finger of GATA-3 in chromatin remodeling and transactivation during Th2 development. *J. Immunol.* 169:4103–7

58. Zhou M, Ouyang W, Gong Q, Katz SG, White JM, et al. 2001. Friend of GATA-1 represses GATA-3-dependent activity in CD4⁺ T cells. *J. Exp. Med.* 194:1461–71

59. Kurata H, Lee H-J, McClanahan T, Coffman RL, O'Garra A, Arai N. 2002. Friend of GATA is expressed in naive Th cells and functions as a repressor of GATA-3-mediated Th2 cell development. *J. Immunol.* 168:4538–45

60. Staal FJT, Weerkamp F, Langerak AW, Hendriks RW, Clevers HC. 2001. Transcriptional control of T lymphocyte differentiation. *Stem Cells* 19:165–79

61. Greenbaum S, Zhuang Y. 2002. Regulation of early lymphocyte development by E2A family proteins. *Semin. Immunol.* 14:405–14

62. Engel I, Murre C. 2001. The function

of E- and Id proteins in lymphocyte development. *Nat. Rev. Immunol.* 1:193–99

63. Ikawa T, Fujimoto S, Kawamoto H, Katasura Y, Yokota Y. 2001. Commitment to natural killer cells requires the helix-loop-helix inhibitor Id2. *Proc. Natl. Acad. Sci. USA* 98:5164–69

64. Sawada A, Littman DR. 1993. A heterodimer of HEB and an E12-related protein interacts with the CD4 enhancer and regulates its activity in T-cell lines. *Mol. Cell. Biol.* 13:5620–28

65. Petersson K, Ivars F, Sigvardsson M. 2002. The pTα promoter and enhancer are direct targets for transactivation by E box-binding proteins. *Eur. J. Immunol.* 32:911–20

66. Tremblay M, Herblot S, Lecuyer E, Hoang T. 2003. Regulation of pTα gene expression by a dosage of E2A, HEB and SCL. *J. Biol. Chem.* 278:12680–87

67. Kee BL, Bain G, Murre C. 2002. IL-7Ra and E47: independent pathways required for development of multipotent lymphoid progenitors. *EMBO J.* 21:103–13

68. Grégoire J-M, Roméo P-H. 1999. T-cell expression of the human GATA-3 gene is regulated by a non-lineage specific silencer. *J. Biol. Chem.* 274:6567–78

69. Heemskerk MHM, Blom B, Nolan G, Stegmann APA, Bakker AQ, et al. 1997. Inhibition of T cell and promotion of natural killer cell development by the dominant negative helix loop helix factor Id3. *J. Exp. Med.* 186:1597–602

70. Kim D, Peng X-C, Sun X-H. 1999. Massive apoptosis of thymocytes in T-cell-deficient Id1 transgenic mice. *Mol. Cell. Biol.* 19:8240–53

71. Pan L, Hanrahan J, Li J, Hale LP, Zhuang Y. 2002. An analysis of T cell intrinsic roles of E2A by conditional gene disruption in the thymus. *J. Immunol.* 168:3923–32

72. Barndt RJ, Dai M, Zhuang Y. 2000. Functions of E2A-HEB heterodimers in T-cell development revealed by a dominant

negative mutation of HEB. Mol. Cell. Biol. 20:6677–85

73. Rivera RR, Johns CP, Quan J, Johnson RS, Murre C. 2000. Thymocyte selection is regulated by the helix-loop-helix inhibitor protein, Id3. *Immunity* 12:17–26

74. Engel I, Johns C, Bain G, Rivera RR, Murre C. 2001. Early thymocyte development is regulated by modulation of E2A protein activity. *J. Exp. Med.* 194:733–46

75. Bain G, Cravatt CB, Loomans C, Alberola-Ila J, Hedrick SM, Murre C. 2001. Regulation of the helix-loop-helix proteins, E2A and Id3, by the Ras-ERK MAPK cascade. *Nat. Immunol.* 2:165–71

76. Morrow MA, Mayer EW, Perez CA, Adam M, Siu G. 1999. Overexpression of the helix-loop-helix protein Id2 blocks T cell development at multiple stages. *Mol. Immunol.* 36:491–503

77. Engel I, Murre C. 2004. E2A proteins enforce a proliferation checkpoint in developing thymocytes. *EMBO J.* 23:202–11

78. Levanon D, Groner Y. 2004. Structure and regulated expression of mammalian RUNX genes. *Oncogene* 23:4211–19

79. Ichikawa M, Asai T, Saito T, Yamamoto G, Seo S, et al. 2004. AML-1 is required for megakaryocytic maturation and lymphocytic differentiation, but not for maintenance of hematopoietic stem cells in adult hematopoiesis. *Nat. Med.* 10:299–304

80. Anderson MK, Pant R, Miracle AL, Sun X, Luer CA, et al. 2004. Evolutionary origins of lymphocytes: ensembles of T cell and B cell transcriptional regulators in a cartilaginous fish. *J. Immunol.* 172:5851–60

81. Tracey WD, Speck NA. 2000. Potential roles for RUNX1 and its orthologs in determining hematopoietic cell fate. *Semin. Cell Dev. Biol.* 11:337–42

82. Ling KW, Dzierzak E. 2002. Ontogeny and genetics of the hemato/lymphopoietic system. *Curr. Opin. Immunol.* 14:186–91

83. Nishimura M, Fukushima-Nakase Y, Fujita Y, Nakao M, Toda S, et al. 2004.

VWRPY motif-dependent and -independent roles of AML1/Runx1 transcription factor in murine hematopoietic development. *Blood* 103:562–70

84. Telfer JC, Hedblom EE, Anderson MK, Laurent MN, Rothenberg EV. 2004. Localization of the domains in Runx transcription factors required for the repression of CD4 in thymocytes. *J. Immunol.* 172:4359–70

85. Woolf E, Xiao C, Fainaru O, Lotem J, Rosen D, et al. 2003. Runx3 and Runx1 are required for CD8 T cell development during thymopoiesis. *Proc. Natl. Acad. Sci. USA* 100:7731–36

86. Taniuchi I, Osato M, Egawa T, Sunshine MJ, Bae S-C, et al. 2002. Differential requirements for Runx proteins in CD4 repression and epigenetic silencing during T lymphocyte development. *Cell* 111:621–33

87. Vaillant F, Blyth K, Andrew L, Neil JC, Cameron ER. 2002. Enforced expression of Runx2 perturbs T cell development at a stage coincident with β-selection. *J. Immunol.* 169:2866–74

88. Hayashi K, Abe N, Watanabe T, Obinata M, Ito M, et al. 2001. Overexpression of AML1 transcription factor drives thymocytes into the CD8 single-positive lineage. *J. Immunol.* 167:4957–65

89. Satake M, Nomura S, Yamaguchi-Iwai Y, Takahama Y, Hashimoto Y, et al. 1995. Expression of the Runt domain-encoding PEBP2α genes in T cells during thymic development. *Mol. Cell. Biol.* 15:1662–70

90. Telfer JC, Rothenberg EV. 2001. Expression and function of a stem-cell promoter for the murine CBFα2 gene: distinct roles and regulation in natural killer and T cell development. *Dev. Biol.* 229:363–82

91. Hsiang YH, Spencer D, Wang S, Speck NA, Raulet DH. 1993. The role of viral enhancer “core” motif-related sequences in regulating T cell receptor-γ and -δ gene expression. *J. Immunol.* 150:3905–16

92. Wotton D, Ghysdael J, Wang S, Speck

NA, Owen MJ. 1994. Cooperative binding of Ets-1 and core binding factor to DNA. Mol. Cell. Biol. 14:840–50

93. Hernandez-Munain C, Krangel MS. 1994. Regulation of the T-cell receptor δ enhancer by functional cooperation between c-Myb and core-binding factors. Mol. Cell. Biol. 14:473–83

94. Giese K, Kingsley C, Kirshner JR, Grosschedl R. 1995. Assembly and function of a TCRα enhancer complex is dependent on LEF-1-induced DNA bending and multiple protein-protein interactions. Genes Dev. 9:995–1008

95. Erman B, Cortes M, Nikolajczyk BS, Speck NA, Sen R. 1998. ETS-core binding factor: a common composite motif in antigen receptor gene enhancers. Mol. Cell. Biol. 18:1322–30

96. Kim W-Y, Sieweke M, Ogawa E, Wee H-J, Englmeier U, et al. 1999. Mutual activation of Ets-1 and AML1 DNA binding by direct interaction of their autoinhibitory domains. EMBO J. 18:1609–20

97. Yannoutsos N, Barreto V, Misulovin Z, Gazumyan A, Yu W, et al. 2004. *Acis* element in the recombination activating gene locus regulates gene expression by counteracting a distant silencer. Nat. Immunol. 5:443–50

98. Ehlers M, Laule-Kilian K, Petter M, Aldrian CJ, Grueter B, et al. 2003. Morpholino antisense oligonucleotide-mediated gene knockdown during thymocyte development reveals role for Runx3 transcription factor in CD4 silencing during development of CD4⁻/CD8⁺ thymocytes. J. Immunol. 171:3594–604

99. Liu X, Bosselut R. 2004. Duration of TCR signaling controls CD4-CD8 lineage differentiation in vivo. Nat. Immunol. 5:280–88

99a. Kohu K, Sato T, Ohno S-i, Hayashi K, Uchino R, et al. 2005. Over-expression of the Runx3 transcription factor increases the proportion of mature thymocytes of the CD8 single positive lineage. J. Immunol. In press

100. Komine O, Hayashi K, Natsume W, Watanabe T, Seki Y, et al. 2003. The Runx1 transcription factor inhibits the differentiation of naive CD4⁺ T cells into the Th2 lineage by repressing GATA3 expression. J. Exp. Med. 198:51–61

101. Cortes M, Wong E, Koipally J, Georgopoulos K. 1999. Control of lymphocyte development by the Ikaros gene family. Curr. Opin. Immunol. 11:167–71

102. Brown KE, Guest SS, Smale ST, Hahm K, Merkenschlager M, Fisher AG. 1997. Association of transcriptionally silent genes with Ikaros complexes at centromeric heterochromatin. Cell 91:845–54

103. Harker N, Naito T, Cortes M, Hostert A, Hirschberg S, et al. 2002. The CD8α gene locus is regulated by the Ikaros family of proteins. Mol. Cell 10:1403–15

104. Georgopoulos K. 2002. Haematopoietic cell-fate decisions, chromatin regulation and Ikaros. Nat. Rev. Immunol. 2:162–74

105. Winandy S, Wu L, Wang J-H, Georgopoulos K. 1999. Pre-T cell receptor (TCR) and TCR-controlled checkpoints in T cell differentiation are set by Ikaros. J. Exp. Med. 190:1039–48

106. Nichogiannopoulou A, Trevisan M, Neben S, Friedrich C, Georgopoulos K. 1999. Defects in hematopoietic stem cell activity in Ikaros mutant mice. J. Exp. Med. 190:1201–14

107. Papathanasiou P, Perkins AC, Cobb BS, Ferrini R, Sridharan R, et al. 2003. Widespread failure of hematolymphoid differentiation caused by a recessive niche-filling allele of the Ikaros transcription factor. Immunity 19:131–44

108. Allman D, Sambandam A, Kim S, Miller JP, Pagan A, et al. 2003. Thymopoiesis independent of common lymphoid progenitors. Nat. Immunol. 4:168–74

109. Urban JA, Winandy S. 2004. Ikaros null mice display defects in T cell selection and CD4 versus CD8 lineage decision. J. Immunol. 173:4470–78

110. Chi TH, Wan M, Zhao K, Taniuchi I, Chen L, et al. 2002. Reciprocal regulation of

CD4/CD8 expression by SWI/SNF-like BAF complexes. *Nature* 418:195–99

111. Okamura RM, Sigvardsson M, Galceran J, Verbeek S, Clevers H, Grosschedl R. 1998. Redundant regulation of T cell differentiation and TCRα gene expression by the transcription factors LEF-1 and TCF-1. *Immunity* 8:11–20

112. Schilham MW, Wilson A, Moerer P, Benaissa-Trouw BJ, Cumano A, Clevers HC. 1998. Critical involvement of Tcf-1 in expansion of thymocytes. *J. Immunol.* 161:3984–91

113. Gounari F, Aifantis I, Khazaie K, Hoefflinger S, Harada N, et al. 2001. Somatic activation of β-catenin bypasses pre-TCR signaling and TCR selection in thymocyte development. *Nat. Immunol.* 2:863–69

114. Ioannidis V, Beermann F, Clevers H, Held W. 2001. The β-catenin–TCF-1 pathway ensures CD4⁺CD8⁺ thymocyte survival. *Nat. Immunol.* 2:691–97

115. Staal FJT, Weerkamp F, Baert MR, van den Burg CM, van Noort M, et al. 2004. Wnt target genes identified by DNA microarrays in immature CD34⁺ thymocytes regulate proliferation and cell adhesion. *J. Immunol.* 172:1099–108

116. Reya T, Duncan AW, Ailles L, Domen J, Scherer DC, et al. 2003. A role for Wnt signalling in self-renewal of haematopoietic stem cells. *Nature* 423:409–14

117. Allen RD, III, Bender TP, Siu G. 1999. c-Myb is essential for early T cell development. *Genes Dev.* 13:1073–78

118. Emambokus N, Vegiopoulos A, Harman B, Jenkinson E, Anderson G, Frampton J. 2003. Progression through key stages of haemopoiesis is dependent on distinct threshold levels of c-Myb. *EMBO J.* 22:4478–88

119. Mukouyama Y, Chiba N, Mucenski ML, Satake M, Miyajima A, et al. 1999. Hematopoietic cells in cultures of the murine embryonic aorta-gonad-mesonephros region are induced by c-Myb. *Curr. Biol.* 9:833–36

120. Hsiang YH, Goldman JP, Raulet DH. 1995. The role of c-Myb or a related factor in regulating the T cell receptor γ gene enhancer. *J. Immunol.* 154:5195–204

121. Allen RD, III, Kim HK, Sarafova SD, Siu G. 2001. Negative regulation of CD4 gene expression by a HES-1-c-Myb complex. *Mol. Cell. Biol.* 21:3071–82

122. Hernández-Munain C, Krangel MS. 2002. Distinct roles for c-Myb and core binding factor/polyoma enhancer-binding protein 2 in the assembly and function of a multiprotein complex on the TCR δ enhancer in vivo. *J. Immunol.* 169:4362–69

123. Pearson R, Weston K. 2000. c-Myb regulates the proliferation of immature thymocytes following β-selection. *EMBO J.* 19:6112–20

124. Bender TP, Kremer CS, Kraus M, Buch T, Rajewsky K. 2004. Critical functions for c-Myb at three checkpoints during thymocyte development. *Nat. Immunol.* 5:721–29

125. Anderson MK, Hernandez-Hoyos G, Diamond RA, Rothenberg EV. 1999. Precise developmental regulation of Ets family transcription factors during specification and commitment to the T cell lineage. *Development* 126:3131–48

126. Spain LM, Guerriero A, Kunjibettu S, Scott EW. 1999. T cell development in PU.1-deficient mice. *J. Immunol.* 163:2681–87

127. Herblot S, Steff AM, Hugo P, Aplan PD, Hoang T. 2000. SCL and LMO1 alter thymocyte differentiation: inhibition of E2A-HEB function and pre-Tα chain expression. *Nat. Immunol.* 1:138–44

128. Su GH, Ip HS, Cobb BS, Lu M-M, Chen H-M, Simon MC. 1996. The Ets protein Spi-B is expressed exclusively in B cells and T cells during development. *J. Exp. Med.* 184:203–14

129. Lloberas J, Soler C, Celada A. 1999. The key role of PU.1/SPI-1 in B cells, myeloid cells and macrophages. *Immunol. Today* 20:184–89

130. Dahl R, Simon MC. 2003. The importance of PU.1 concentration in hematopoietic lineage commitment and maturation. *Blood Cells Mol. Dis.* 31:229–33

131. Back J, Dierich A, Bronn C, Kastner P, Chan S. 2004. PU.1 determines the self-renewal capacity of erythroid progenitor cells. *Blood* 103:3615–23

132. DeKoter RP, Lee H-J, Singh H. 2002. PU.1 regulates expression of the Interleukin-7 receptor in lymphoid progenitors. *Immunity* 16:297–309

133. McKercher SR, Torbett BE, Anderson KL, Henkel GW, Vestal DJ, et al. 1996. Targeted disruption of the *PU.1* gene results in multiple hematopoietic abnormalities. *EMBO J.* 15:5647–58

134. Anderson MK, Weiss A, Hernandez-Hoyos G, Dionne CJ, Rothenberg EV. 2002. Constitutive expression of PU.1 in fetal hematopoietic progenitors blocks T-cell development at the pro-T stage. *Immunity* 16:285–96

135. Schotte R, Rissoan MC, Bendriss-Vermare N, Bridon JM, Duhen T, et al. 2003. The transcription factor Spi-B is expressed in plasmacytoid DC precursors and inhibits T-, B-, and NK-cell development. *Blood* 101:1015–23

136. Trinh LA, Ferrini R, Cobb BS, Weinmann AS, Hahm K, et al. 2001. Down-regulation of TDT transcription in CD4+CD8+ thymocytes by Ikaros proteins in direct competition with an Ets activator. *Genes Dev.* 15:1817–32

137. Bories J-C, Willerford DM, Grévin D, Davidson L, Camus A, et al. 1995. Increased T-cell apoptosis and terminal B-cell differentiation induced by inactivation of the Ets-1 proto-oncogene. *Nature* 377:635–38

138. Barton K, Muthusamy N, Fischer C, Ting C-N, Walunas TL, et al. 1998. The Ets-1 transcription factor is required for the development of natural killer cells in mice. *Immunity* 9:555–63

139. Xue H-H, Bollenbacher J, Rovella V, Tripuraneni R, Du Y-B, et al. 2004. GA

binding protein regulates interleukin 7 receptor α-chain gene expression in T cells. *Nat. Immunol.* 5:1036–44

140. Sauvageau G, Lansdorp PM, Eaves CJ, Hogge DE, Dragowska WH, et al. 1994. Differential expression of homeobox genes in functionally distinct CD34+ subpopulations of human bone marrow cells. *Proc. Natl. Acad. Sci. USA* 91:12223–27

141. Taghon T, Thys K, De Smedt M, Weerkamp F, Staal FJ, et al. 2003. Homeobox gene expression profile in human hematopoietic multipotent stem cells and T-cell progenitors: implications for human T-cell development. *Leukemia* 17:1157–63

142. Izon DJ, Rozenfeld S, Fong ST, Komuves L, Largman C, Lawrence HJ. 1998. Loss of function of the homeobox gene *Hoxa-9* perturbs early T-cell development and induces apoptosis in primitive thymocytes. *Blood* 92:383–93

143. Szabo SJ, Kim ST, Costa GL, Zhang X, Fathman CG, Glimcher LH. 2000. A novel transcription factor, T-bet, directs Th1 lineage commitment. *Cell* 100:655–69

144. Shier P, Hofstra CL, Ma XJ, Wu Y, Ngo K, Fung-Leung WP. 2000. Tbt-1, a new T-box transcription factor induced in activated Th1 and CD8+ T cells. *Immunogenetics* 51:771–78

145. Mullen AC, High FA, Hutchins AS, Lee HW, Villarino AV, et al. 2001. Role of T-bet in commitment of Th1 cells before IL-12-dependent selection. *Science* 292:1907–10

146. Pearce EL, Mullen AC, Martins GA, Krawczyk CM, Hutchins AS, et al. 2003. Control of effector CD8+ T cell function by the transcription factor Eomesodermin. *Science* 302:1041–43

147. Townsend MJ, Weinmann AS, Matsuda JL, Salomon R, Farnham PJ, et al. 2004. T-bet regulates the terminal maturation and homeostasis of NK and Vα14i NKT Cells. *Immunity* 20:477–94

148. Allen JD, Harris AW, Bath ML, Strasser

A, Scollay R, Adams JM. 1995. Perturbed development of T and B cells in mice expressing an Hlx homeobox transgene. *J. Immunol.* 154:1531–42

149. Mullen AC, Hutchins AS, High FA, Lee HW, Sykes KJ, et al. 2002. Hlx is induced by and genetically interacts with T-bet to promote heritable T_H1 gene induction. *Nat. Immunol.* 3:652–58

150. Zheng WP, Zhao Q, Zhao X, Li B, Hubank M, et al. 2004. Up-regulation of Hlx in immature Th cells induces IFN-γ expression. *J. Immunol.* 172:114–22

151. Lian RH, Kumar V. 2002. Murine natural killer cell progenitors and their requirements for development. *Semin. Immunol.* 14:453–60

152. Wilkinson B, Chen JY, Han P, Rufner KM, Goularte OD, Kaye J. 2002. TOX: an HMG box protein implicated in the regulation of thymocyte selection. *Nat. Immunol.* 3:272–80

153. Aliahmad P, O'Flaherty E, Han P, Goularte OD, Wilkinson B, et al. 2004. TOX provides a link between calcineurin activation and CD8 lineage commitment. *J. Exp. Med.* 199:1089–99

154. Higashi Y, Moribe H, Takagi T, Sekido R, Kawakami K, et al. 1997. Impairment of T cell development in δEF1 mutant mice. *J. Exp. Med.* 185:1467–79

155. Postigo AA, Sheppard AM, Mucenski ML, Dean DC. 1997. c-Myb and Ets proteins synergize to overcome transcriptional repression by ZEB. *EMBO J.* 16: 3924–34

156. Jepsen K, Hermanson O, Onami TM, Gleiberman AS, Lunyak V, et al. 2000. Combinatorial roles of the nuclear receptor corepressor in transcription and development. *Cell* 102:753–63

157. Hock H, Hamblen MJ, Rooke HM, Traver D, Bronson RT, et al. 2003. Intrinsic requirement for zinc finger transcription factor Gfi-1 in neutrophil differentiation. *Immunity* 18:109–20

158. Yücel R, Karsunky H, Klein-Hitpass L, Möröy T. 2003. The transcriptional re-

pressor Gfi1 affects development of early, uncommitted c-Kit⁺ T cell progenitors and CD4/CD8 lineage decision in the thymus. *J. Exp. Med.* 197:831–44

159. Doan LL, Kitay MK, Yu Q, Singer A, Herblot S, et al. 2003. Growth factor independence-1B expression leads to defects in T cell activation, IL-7 receptor α expression, and T cell lineage commitment. *J. Immunol.* 170:2356–66

160. Osawa M, Yamaguchi T, Nakamura Y, Kaneko S, Onodera M, et al. 2002. Erythroid expansion mediated by the Gfi-1B zinc finger protein: role in normal hematopoiesis. *Blood* 100:2769–77

161. Saleque S, Cameron S, Orkin SH. 2002. The zinc-finger proto-oncogene Gfi-1b is essential for development of the erythroid and megakaryocytic lineages. *Genes Dev.* 16:301–6

162. Zhu J, Guo L, Min B, Watson CJ, Hu-Li J, et al. 2002. Growth factor independent-1 induced by IL-4 regulates Th2 cell proliferation. *Immunity* 16:733–44

163. Alvarez JD, Yasui DH, Niida H, Joh T, Loh DY, Kohwi-Shigematsu T. 2000. The MAR-binding protein SATB1 orchestrates temporal and spatial expression of multiple genes during T-cell development. *Genes Dev.* 14:521–35

164. Cai S, Han H-J, Kohwi-Shigematsu T. 2003. Tissue-specific nuclear architecture and gene expression regulated by SATB1. *Nat. Genet.* 34:42–51

165. Yasui D, Miyano M, Cai S, Varga-Weisz P, Kohwi-Shigematsu T. 2002. SATB1 targets chromatin remodelling to regulate genes over long distances. *Nature* 419:641–45

166. Fisher AG, Merkenschlager M. 2002. Gene silencing, cell fate and nuclear organisation. *Curr. Opin. Genet. Dev.* 12: 193–97

167. Hardy RR. 2003. B-cell commitment: deciding on the players. *Curr. Opin. Immunol.* 15:158–65

168. Maier H, Hagman J. 2002. Roles of EBF and Pax-5 in B lineage commitment and

development. *Semin. Immunol.* 14:415–22

169. Medina KL, Pongubala JMR, Reddy KL, Lancki DW, DeKoter R, et al. 2004. Assembling a gene regulatory network for specification of the B cell fate. *Dev. Cell* 7:607–17

170. Rosenbauer F, Wagner K, Kutok JL, Iwasaki H, Akashi K, et al. 2003. Myeloid and B-lymphoid development require high PU.1 expression by a distal element in vivo. *Blood* 102:342A

171. Ikawa T, Kawamoto H, Wright LY, Murre C. 2004. Long-term cultured E2A-deficient hematopoietic progenitor cells are pluripotent. *Immunity* 20:349–60

172. Cotta CV, Zhang Z, Kim HG, Klug CA. 2003. Pax5 determines B- versus T-cell fate and does not block early myeloid-lineage development. *Blood* 101:4342–46

173. Seet CS, Brumbaugh RL, Kee BL. 2004. Early B Cell Factor promotes B lymphopoiesis with reduced Interleukin 7 responsiveness in the absence of E2A. *J. Exp. Med.* 199:1689–700

174. Souabni A, Cobaleda C, Schebesta M, Busslinger M. 2002. Pax5 promotes B lymphopoiesis and blocks T cell development by repressing Notch1. *Immunity* 17:781–93

175. Zhang Z, Cotta CV, Stephan RP, deGuzman CG, Klug CA. 2003. Enforced expression of EBF in hematopoietic stem cells restricts lymphopoiesis to the B cell lineage. *EMBO J.* 22:4759–69

176. Goebel P, Janney N, Valenzuela JR, Romanow WJ, Murre C, Feeney AJ. 2001. Localized gene-specific induction of accessibility to V(D)J recombination induced by E2A and early B cell factor in nonlymphoid cells. *J. Exp. Med.* 194:645–56

177. Smith EM, Gisler R, Sigvardsson M. 2002. Cloning and characterization of a promoter flanking the early B cell factor (EBF) gene indicates roles for E-proteins and autoregulation in the control of EBF expression. *J. Immunol.* 169:261–70

178. Gisler R, Sigvardsson M. 2002. The human V-preB promoter is a target for coordinated activation by early B cell factor and E47. *J. Immunol.* 168:5130–38

179. Sayegh CE, Quong MW, Agata Y, Murre C. 2003. E-proteins directly regulate expression of activation-induced deaminase in mature B cells. *Nat. Immunol.* 4:586–93

180. Schroeder T, Kohlhof H, Rieber N, Just U. 2003. Notch signaling induces multilineage myeloid differentiation and upregulates PU.1 expression. *J. Immunol.* 170:5538–48

181. Aifantis I, Gounari F, Scorrano L, Borowski C, von Boehmer H. 2001. Constitutive pre-TCR signaling promotes differentiation through Ca²⁺ mobilization and activation of NF-κB and NFAT. *Nat. Immunol.* 2:403–9

182. Carleton M, Haks MC, Smeele SA, Jones A, Belkowski SM, et al. 2002. Early growth response transcription factors are required for development of CD4⁻CD8⁻ thymocytes to the CD4⁺CD8⁺ stage. *J. Immunol.* 168:1649–58

183. Xi H, Kersh GJ. 2004. Early growth response gene 3 regulates thymocyte proliferation during the transition from CD4⁻CD8⁻ to CD4⁺CD8⁺. *J. Immunol.* 172:964–71

184. Barndt R, Dai MF, Zhang Y. 1999. A novel role for HEB downstream or parallel to the pre-TCR signaling pathway during αβ thymopoiesis. *J. Immunol.* 163:3331–43

185. Yasutomo K, Doyle C, Miele L, Germain RN. 2000. The duration of antigen receptor signalling determines CD4⁺ versus CD8⁺ T-cell lineage fate. *Nature* 404:506–10

186. Dunon D, Courtois D, Vainio O, Six A, Chen CH, et al. 1997. Ontogeny of the immune system: γ/δ and α/β T cells migrate from thymus to the periphery in alternating waves. *J. Exp. Med.* 186:977–88

187. Rast JP, Litman GW. 1998. Towards understanding the evolutionary origins and

early diversification of rearranging antigen receptors. *Immunol. Rev.* 166:79–86

188. Fehling HJ, Gilfillan S, Ceredig R. 1999. $\alpha \beta / \gamma \delta$ lineage commitment in the thymus of normal and genetically manipulated mice. *Adv. Immunol.* 71:1–76

189. Bruno L, Fehling HJ, von Boehmer H. 1996. The $\alpha \beta$ T cell receptor can replace the $\gamma \delta$ receptor in the development of $\gamma \delta$ lineage cells. *Immunity* 5:343–52

190. Dave VP, Cao Z, Browne C, Alarcon B, Fernandez-Miguel G, et al. 1997. CD3$\delta$ deficiency arrests development of the $\alpha \beta$ but not the $\gamma \delta$ T cell lineage. *EMBO J.* 16:1360–70

191. Terrence K, Pavlovich CP, Matechak EO, Fowlkes BJ. 2000. Premature expression of T cell receptor (TCR) $\alpha \beta$ suppresses TCR $\gamma \delta$ gene rearrangement but permits development of $\gamma \delta$ lineage T cells. *J. Exp. Med.* 192:537–48

192. Lee PP, Fitzpatrick DR, Beard C, Jessup HK, Lehar S, et al. 2001. A critical role for Dnmt1 and DNA methylation in T cell development, function, and survival. *Immunity* 15:763–74

193. Pennington DJ, Silva-Santos B, Shires J, Theodoridis E, Pollitt C, et al. 2003. The inter-relatedness and interdependence of mouse T cell receptor $\gamma \delta^{+}$ and $\alpha \beta^{+}$ cells. *Nat. Immunol.* 4:991–98

194. Washburn T, Schweighoffer E, Gridley T, Chang D, Fowlkes BJ, et al. 1997. Notch activity influences the $\alpha \beta$ versus $\gamma \delta$ T cell lineage decision. *Cell* 88:833–43

195. Bain G, Romanow WJ, Albers K, Havran WL, Murre C. 1999. Positive and negative regulation of V(D)J recombination by the E2A proteins. *J. Exp. Med.* 189:289–300

196. Blom B, Heemskerk MHM, Verschuren MCM, van Dongen JJM, Stegmann APA, et al. 1999. Disruption of $\alpha \beta$ but not of $\gamma \delta$ T cell development by overexpression of the helix-loop-helix protein Id3 in committed T cell progenitors. *EMBO J.* 18:2793–802

197. De Smedt M, Reynvoet K, Kerre T,

Taghon T, Verhasselt B, et al. 2002. Active form of Notch imposes T cell fate in human progenitor cells. *J. Immunol.* 169:3021–29

198. Garcia-Peydro M, de Yebenes VG, Toribio ML. 2003. Sustained Notch1 signaling instructs the earliest human intrathymic precursors to adopt a $\gamma \delta$ T cell fate in fetal thymus organ culture. *Blood* 102:2444–51

199. Lee C-K, Kim K, Geiman TM, Murphy WJ, Muegge K, Durum SK. 1999. Cloning thymic precursor cells: demonstration that individual pro-T1 cells have dual T-NK potential and individual pro-T2 cells have dual $\alpha \beta - \gamma \delta$ T cell potential. *Cell Immunol.* 191:139–44

200. Lacorazza HD, Miyazaki Y, Di Cristofano A, Deblasio A, Hedvat C, et al. 2002. The ETS protein MEF plays a critical role in perforin gene expression and the development of Natural Killer and NK-T cells. *Immunity* 17:437–49

201. Ohteki T, Ho S, Suzuki H, Mak TW, Ohashi PS. 1997. Role for IL-15/IL-15 receptor $\beta$-chain in Natural Killer 1.1${}^{+}$ T cell receptor-$\alpha / \beta^{+}$ cell development. *J. Immunol.* 159:5931–35

202. Ohteki T, Yoshida H, Matsuyama T, Duncan GS, Mak TW, Ohashi PS. 1998. The transcription factor interferon regulatory factor 1 (IRF-1) is important during the maturation of Natural Killer 1.1${}^{+}$ T cell receptor-$\alpha / \beta^{+}$ (NK1${}^{+}$ T) cells, natural killer cells, and intestinal intraepithelial T cells. *J. Exp. Med.* 187:967–72

203. Walunas TL, Wang B, Wang CR, Leiden JM. 2000. Cutting edge: The Ets1 transcription factor is required for the development of NK T cells in mice. *J. Immunol.* 164:2857–60

204. MacDonald HR. 2002. Development and selection of NKT cells. *Curr. Opin. Immunol.* 14:250–54

205. Ramsdell F. 2003. Foxp3 and natural regulatory T cells: key to a cell lineage? *Immunity* 19:165–68

206. Harman BC, Jenkinson EJ, Anderson G.

2003. Entry into the thymic microenvironment triggers Notch activation in the earliest migrant T cell progenitors. *J. Immunol.* 170:1299–303

207. Ivanov V, Merkenschlager M, Ceredig R. 1993. Antioxidant treatment of thymic organ cultures decreases NF-κB and TCF1(α) transcription factor activities and inhibits αβ T cell development. *J. Immunol.* 151:4694–704

208. Schwarz BA, Bhandoola A. 2004. Circulating hematopoietic progenitors with T lineage potential. *Nat. Immunol.* 9:953–60

209. Kincade PW, Igarashi H, Medina KL, Kouro T, Yokota T, et al. 2002. Lymphoid lineage cells in adult murine bone marrow diverge from those of other blood cells at an early, hormone-sensitive stage. *Semin. Immunol.* 14:385–94

210. Prohaska SS, Scherer DC, Weissman IL, Kondo M. 2002. Developmental plasticity of lymphoid progenitors. *Semin. Immunol.* 14:377–84

211. Rothenberg EV, Dionne CJ. 2002. Lineage plasticity and commitment in T cell development. *Immunol. Rev.* 187:96–115

212. Bhandoola A, Sambandan A, Allman D, Meraz A, Schwarz B. 2003. Early T lineage progenitors: New insights, but old questions remain. *J. Immunol.* 171:5653–58

213. Wang H, Spangrude GJ. 2003. Aspects of early lymphoid commitment. *Curr. Opin. Hematol.* 10:203–7

214. Douagi I, Colucci F, Di Santo JP, Cumano A. 2002. Identification of the earliest prethymic bipotent T/NK progenitor in murine fetal liver. *Blood* 99:463–71

215. Kondo M, Weissman IL, Akashi K. 1997. Identification of clonogenic common lymphoid progenitors in mouse bone marrow. *Cell* 91:661–72

216. Martin CH, Aifantis I, Scimone ML, von Andrian UH, Reizis B, et al. 2003. Efficient thymic immigration of B220+

lymphoid-restricted bone marrow cells with T precursor potential. *Nat. Immunol.* 4:866–73

217. Perry SS, Wang H, Pierce LJ, Yang AM, Tsai S, Spangrude GJ. 2004. L-Selectin defines a bone marrow analogue to the thymic early T-lineage progenitor. *Blood* 103:2990–96

218. Igarashi H, Gregory SC, Yokota T, Sakaguchi N, Kincade PW. 2002. Transcription from the RAG1 locus marks the earliest lymphocyte progenitors in bone marrow. *Immunity* 17:117–30

219. Katsura Y. 2002. Redefinition of lymphoid progenitors. *Nat. Rev. Immunol.* 2:127–32

220. Mebius RE, Miyamoto T, Christensen J, Domen J, Cupedo T, et al. 2001. The fetal liver counterpart of adult common lymphoid progenitors gives rise to all lymphoid lineages, CD45+CD4+CD3− cells, as well as macrophages. *J. Immunol.* 166:6593–601

221. Lu M, Kawamoto H, Katsume Y, Ikawa T, Katsura Y. 2002. The common myelolymphoid progenitor: a key intermediate stage in hemopoiesis generating T and B cells. *J. Immunol.* 169:3519–25

222. Yokota T, Kouro T, Hirose J, Igarashi H, Garrett KP, et al. 2003. Unique properties of fetal lymphoid progenitors identified according to RAG1 gene expression. *Immunity* 19:365–75

223. Porritt HE, Rumfelt LL, Tabrizifard S, Schmitt TM, Zúñiga-Pflücker JC, Petrie HT. 2004. Heterogeneity among DN1 prothymocytes reveals multiple progenitors with different capacities to generate T cell and non-T cell lineages. *Immunity* 20:735–45

224. Fisher RC, Lovelock JD, Scott EW. 1999. A critical role for PU.1 in homing and long-term engraftment by hematopoietic stem cells in the bone marrow. *Blood* 94:1283–90

225. North TE, de Bruijn MF, Stacy T, Talebian L, Lind E, et al. 2002. Runx1 expression marks long-term repopulating

hematopoietic stem cells in the midgestation mouse embryo. *Immunity* 16:661–72

226. Varnum-Finney B, Xu L, Brashem-Stein C, Nourigat C, Flowers D, et al. 2000. Pluripotent, cytokine-dependent, hematopoietic stem cells are immortalized by constitutive Notch1 signaling. *Nat. Med.* 6:1278–81

227. Sikes ML, Gomez RJ, Song J, Oltz E. 1998. A developmental stage-specific promoter directs germline transcription of DβJβ gene segments in precursor T lymphocytes. *J. Immunol.* 161:1399–405

228. Reizis B, Leder P. 2002. Direct induction of T lymphocyte-specific gene expression by the mammalian Notch signaling pathway. *Genes Dev.* 16:295–300

229. Allen JM, Forbush KA, Perlmutter RM. 1992. Functional dissection of the lck proximal promoter. *Mol. Cell. Biol.* 12: 2758–68

230. McCracken S, Leung S, Bosselut R, Ghysdael J, Miyamoto NG. 1994. Myb and Ets related transcription factors are required for activity of the human lck type I promoter. *Oncogene* 9:3609–15

231. Wang Q-F, Lauring J, Schlissel MS. 2000. c-Myb binds to a sequence in the proximal region of the RAG-2 promoter and is essential for promoter activity in T-lineage cells. *Mol. Cell. Biol.* 20:9203–11

232. Tomita K, Hattori M, Nakamura E, Nakanishi S, Minato N, Kageyama R. 1999. The bHLH gene *Hes1* is essential for expansion of early T cell precursors. *Genes Dev.* 13:1203–10

233. Kawamoto H, Ohmura K, Fujimoto S, Lu M, Ikawa T, Katsura Y. 2003. Extensive proliferation of T cell lineage-restricted progenitors in the thymus: an essential process for clonal expression of diverse T cell receptor beta chains. *Eur. J. Immunol.* 33:606–15

234. Shen HQ, Lu M, Ikawa T, Masuda K, Ohmura K, et al. 2003. T/NK bipotent progenitors in the thymus retain the potential to generate dendritic cells. *J. Immunol.* 171:3401–6

235. Petrie HT. 2003. Cell migration and the control of post-natal T-cell lymphopoiesis in the thymus. *Nat. Rev. Immunol.* 3:859–66

236. Rothenberg EV. 2000. Stepwise specification of lymphocyte developmental lineages. *Curr. Opin. Genet. Dev.* 10:370–79

237. Wu L, Li C-L, Shortman K. 1996. Thymic dendritic cell precursors: relationship to the T lymphocyte lineage and phenotype of the dendritic cell progeny. *J. Exp. Med.* 184:903–11

238. Spits H, Blom B, Jaleco AC, Weijer K, Verschuren MC, et al. 1998. Early stages in the development of human T, natural killer and thymic dendritic cells. *Immunol. Rev.* 165:75–86

239. Ikawa T, Kawamoto H, Fujimoto S, Katsura Y. 1999. Commitment of common T/natural killer (NK) progenitors to unipotent T and NK progenitors in the murine fetal thymus revealed by a single progenitor assay. *J. Exp. Med.* 190:1617–25

240. Michie AM, Carlyle JR, Schmitt TM, Ljutic B, Cho SK, et al. 2000. Clonal characterization of a bipotent T cell and NK cell progenitor in the mouse fetal thymus. *J. Immunol.* 164:1730–33

241. Lee C-K, Kim JK, Kim Y, Lee MK, Kim K, et al. 2001. Generation of macrophages from early T progenitors in vitro. *J. Immunol.* 166:5964–69

242. Ohishi K, Varnum-Finney B, Serda RE, Anasetti C, Bernstein ID. 2001. The Notch ligand, Delta-1, inhibits the differentiation of monocytes into macrophages but permits their differentiation into dendritic cells. *Blood* 98:1402–7

243. Corcoran L, Ferrero I, Vremec D, Lucas K, Wraithman J, et al. 2003. The lymphoid past of mouse plasmacytoid cells and thymic dendritic cells. *J. Immunol.* 170:4926–32

244. King AG, Kondo M, Scherer DC, Weissman IL. 2002. Lineage infidelity in myeloid cells with TCR gene rearrangement: a latent developmental potential of

proT cells revealed by ectopic cytokine
receptor signaling. *Proc. Natl. Acad. Sci.*
USA 99:4508–13

245. Iwasaki-Arai J, Iwasaki H, Miyamoto
T, Watanabe S, Akashi K. 2003. En-
forced Granulocyte/Macrophage Colony-
Stimulating Factor signals do not sup-
port lymphopoiesis, but instruct lymphoid
to myelomonocytic lineage conversion. *J.*
Exp. Med. 197:1311–22

246. Rekhtman N, Radparvar F, Evans T, Sk-
oultchi A. 1999. Direct interaction of
hematopoietic transcription factors PU.1
and GATA-1: functional antagonism in
erythroid cells. *Genes Dev.* 13:1398–411

247. Zhang P, Behre G, Pan J, Iwama A,
Wara-Aswapati N, et al. 1999. Negative
cross-talk between hematopoietic regula-
tors: GATA proteins repress PU.1. *Proc.*
Natl. Acad. Sci. USA 96:8705–10

248. Nerlov C, Querfurth E, Kulessa H, Graf
T. 2000. GATA-1 interacts with the
myeloid PU.1 transcription factor and
represses PU.1-dependent transcription.
*Blood* 95:2543–51

249. Izon DJ, Aster JC, He Y, Weng A, Kar-
nell FG, et al. 2002. Deltex1 redirects lym-
phoid progenitors to the B cell lineage by
antagonizing Notch1. *Immunity* 16:231–
43

250. Nie L, Xu M, Vladimirova A, Sun X-H.
2003. Notch-induced E2A ubiquitination
and degradation are controlled by MAP
kinase activities. *EMBO J.* 22:5780–92

251. Davidson EH, McClay DR, Hood L. 2003.
Regulatory gene networks and the prop-
erties of the developmental process. *Proc.*
Natl. Acad. Sci. USA 100:1475–80

252. Hacker C, Kirsch RD, Ju X-S, Hierony-
mus T, Gust TC, et al. 2003. Transcrip-
tional profiling identifies Id2 function in
dendritic cell development. *Nat. Immunol.*
4:380–86

253. Ishiguro A, Spirin KS, Shiohara M, To-
bler A, Gombart AF, et al. 1996. Id2 ex-
pression increases with differentiation of
human myeloid cells. *Blood* 87:5225–31

254. Cooper CL, Brady G, Bilia F, Iscove

NN, Quesenberry PJ. 1997. Expression
of the Id family helix-loop-helix regula-
tors during growth and development in the
hematopoietic system. *Blood* 89:3155–65

255. Mao C, Ray-Gallet D, Tavitian A,
Moreau-Gachelin F. 1996. Differential
phosphorylations of Spi-B and Spi-1 tran-
scription factors. *Oncogene* 12:863–73

256. Rieske P, Pongubala JM. 2001. AKT
induces transcriptional activity of PU.1
through phosphorylation-mediated mod-
ifications within its transactivation do-
main. *J. Biol. Chem.* 276:8460–68

257. Wang J-M, Lai M-Z, Yang-Yen H-F.
2003. Interleukin-3 stimulation of mcl-
l gene transcription involves activation
of the PU.1 transcription factor through
a p38 mitogen-activated protein kinase-
dependent pathway. *Mol. Cell. Biol.* 23:
1896–909

258. Mazzi P, Donini M, Margotto D, Wientjes
F, Dusi S. 2004. IFN-$\gamma$ induces gp91${}^{\text {phox }}$
expression in human monocytes via pro-
tein kinase C-dependent phosphorylation
of PU.1. *J. Immunol.* 172:4941–47

259. Behre G, Whitmarsh AJ, Coghlan MP,
Hoang T, Carpenter CL, et al. 1999.
c-Jun is a c-Jun NH$_{2}$-terminal kinase-
independent coactivator of the PU.1
transcription factor. *J. Biol. Chem.* 274:
4939–46

260. Yokota Y, Mori S. 2002. Role of Id family
proteins in growth control. *J. Cell. Phys-
iol.* 190:21–28

261. Durum SK, Lee C, Geiman TM, Murphy
WJ, Muegge K. 1998. CD16 cross-linking
blocks rearrangement of the TCR$\beta$ lo-
cus and development of $\alpha\beta$ T cells and
induces development of NK cells from
thymic progenitors. *J. Immunol.* 161:
3325–29

262. Gadina M, Sudarshan C, Visconti R, Zhou
YJ, Gu H, et al. 2000. The docking
molecule gab2 is induced by lymphocyte
activation and is involved in signaling by
interleukin-2 and interleukin-15 but not
other common $\gamma$ chain-using cytokines.
*J. Biol. Chem.* 275:26959–66

263. Kovanen PE, Rosenwald A, Fu J, Hurt EM, Lam LT, et al. 2003. Analysis of γc-family cytokine target genes. Identification of dual-specificity phosphatase 5 (DUSP5) as a regulator of mitogen-activated protein kinase activity in interleukin-2 signaling. *J. Biol. Chem.* 278:5205–13

264. Rosmaraki EE, Douagi I, Roth C, Colucci F, Cumano A, Di Santo JP. 2001. Identification of committed NK cell progenitors in adult murine bone marrow. *Eur. J. Immunol.* 31:1900–9

265. Schmitt TM, De Pooter RF, Gronski MA, Cho SK, Ohashi PS, Zúñiga-Pflücker JC. 2004. Induction of T cell development and establishment of T cell competence from embryonic stem cells differentiated in vitro. *Nat. Immunol.* 5:410–17

FRONTISPICE—Tadamitsu Kishimoto  X

INTERLEUKIN-6: FROM BASIC SCIENCE TO MEDICINE—40 YEARS IN IMMUNOLOGY, Tadamitsu Kishimoto 1

TNF/TNFR FAMILY MEMBERS IN COSTIMULATION OF T CELL RESPONSES, Tania H. Watts 23

DEVELOPMENT AND REGULATION OF CELL-MEDIATED IMMUNE RESPONSES TO THE BLOOD STAGES OF MALARIA: IMPLICATIONS FOR VACCINE RESEARCH, Michael F. Good, Huji Xu, Michelle Wykes, and Christian R. Engwerda 69

THE T CELL RECEPTOR: CRITICAL ROLE OF THE MEMBRANE ENVIRONMENT IN RECEPTOR ASSEMBLY AND FUNCTION, Matthew E. Call and Kai W. Wucherpfennig 101

CHEMOKINES, SPHINGOSINE-1-PHOSPHATE, AND CELL MIGRATION IN SECONDARY LYMPHOID ORGANS, Jason G. Cyster 127

MARGINAL ZONE B CELLS, Shiv Pillai, Annaiah Cariappa, and Stewart T. Moran 161

HOW NEUTROPHILS KILL MICROBES, Anthony W. Segal 197

NK CELL RECOGNITION, Lewis L. Lanier 225

IPC: PROFESSIONAL TYPE 1 INTERFERON-PRODUCING CELLS AND PLASMACYTOID DENDRITIC CELL PRECURSORS, Yong-Jun Liu 275

TYPE I INTERFERONS (α/β) IN IMMUNITY AND AUTOIMMUNITY, Argyrios N. Theofilopoulos, Roberto Baccala, Bruce Beutler, and Dwight H. Kono 307

PENTRAXINS AT THE CROSSROADS BETWEEN INNATE IMMUNITY, INFLAMMATION, MATRIX DEPOSITION, AND FEMALE FERTILITY, Cecilia Garlanda, Barbara Bottazzi, Antonio Bastone, and Alberto Mantovani 337

MAINTENANCE OF SERUM ANTIBODY LEVELS, Rudolf A. Manz, Anja E. Hauser, Falk Hiepe, and Andreas Radbruch 367

CATERPILLER: A NOVEL GENE FAMILY IMPORTANT IN IMMUNITY, CELL DEATH, AND DISEASES, Jenny P-Y. Ting and Beckley K. Davis 387

B CELL SIGNALING AND TUMORIGENESIS, Hassan Jumaa, Rudolf W. Hendriks, and Michael Reth 415
THE NOD MOUSE: A MODEL OF IMMUNE DYSREGULATION, Mark S. Anderson and Jeffrey A. Bluestone 447
ANTIGEN-SPECIFIC MEMORY B CELL DEVELOPMENT, Louise J. McHeyzer-Williams and Michael G. McHeyzer-Williams 487
THE B7 FAMILY REVISITED, Rebecca J. Greenwald, Gordon J. Freeman, and Arlene H. Sharpe 515
TEC FAMILY KINASES IN T LYMPHOCYTE DEVELOPMENT AND FUNCTION, Leslie J. Berg, Lisa D. Finkelstein, Julie A. Lucas, and Pamela L. Schwartzberg 549
MOLECULAR GENETICS OF T CELL DEVELOPMENT, Ellen V. Rothenberg and Tom Taghon 601
UNDERSTANDING PRESENTATION OF VIRAL ANTIGENS TO CD8+ T CELLS IN VIVO: THE KEY TO RATIONAL VACCINE DESIGN, Jonathan W. Yewdell and S.M. Mansour Haeryfar 651
IMMUNOLOGY OF MULTIPLE SCLEROSIS, Mireia Sospedra and Roland Martin 683
MAST CELLS AS “TUNABLE” EFFECTOR AND IMMUNOREGULATORY CELLS: RECENT ADVANCES, Stephen J. Galli, Janet Kalesnikoff, Michele A. Grimbaldeston, Adrian M. Piliponsky, Cara M.M. Williams, and Mindy Tsai 749
NETWORK COMMUNICATIONS: LYMPOTOXINS, LIGHT, AND TNF, Carl F. Ware 787
ROLE OF C5A IN INFLAMMATORY RESPONSES, Ren-Feng Guo and Peter A. Ward 821
DNA DEGRADATION IN DEVELOPMENT AND PROGRAMMED CELL DEATH, Shigekazu Nagata 853
TOWARD AN UNDERSTANDING OF NKT CELL BIOLOGY: PROGRESS AND PARADOXES, Mitchell Kronenberg 877
MACROPHAGE RECEPTORS AND IMMUNE RECOGNITION, P.R. Taylor, L. Martinez-Pomares, M. Stacey, H-H. Lin, G.D. Brown, and S. Gordon 901
REGULATION OF LYMPHOID DEVELOPMENT, DIFFERENTIATION, AND FUNCTION BY THE NOTCH PATHWAY, Ivan Maillard, Terry Fang, and Warren S. Pear 945
CELL BIOLOGY OF ANTIGEN PROCESSING IN VITRO AND IN VIVO, E. Sergio Trombetta and Ira Mellman 975

P1: JRX  
February 21, 2005 13:52 Annual Reviews AR239-FM  

CONTENTS vii  

INDEXES  
Subject Index 1029  
Cumulative Index of Contributing Authors, Volumes 13–23 1065  
Cumulative Index of Chapter Titles, Volumes 13–23 1072  

ERRATA  
An online log of corrections to Annual Review of Immunology chapters  
may be found at http://immunol.annualreviews.org/errata.shtml
