
THE JOURNAL OF BIOLOGICAL CHEMISTRY
© 2005 by The American Society for Biochemistry and Molecular Biology, Inc.

**Structure and Function of Both Domains of ArnA, a Dual Function Decarboxylase and a Formyltransferase, Involved in 4-Amino-4-deoxy-L-arabinose Biosynthesis**

Received for publication, February 9, 2005, and in revised form, March 8, 2005  
Published, JBC Papers in Press, April 4, 2005, DOI 10.1074/jbc.M501534200

Gareth J. Williams§$, Steven D. Breazeale¶, Christian R. H. Raetz¶,  
and James H. Naismith§$***

From the §Centre for Biomolecular Science, The University of St. Andrews, St. Andrews KY16 9ST, United Kingdom and the ¶Department of Biochemistry, Duke University, Medical Center, Durham, North Carolina 27710

Modification of the lipid A moiety of lipopolysaccharide by the addition of the sugar 4-amino-4-deoxy-L-arabinose (L-Ara4N) is a strategy adopted by pathogenic Gram-negative bacteria to evade cationic antimicrobial peptides produced by the innate immune system. L-Ara4N biosynthesis is therefore a potential anti-infective target, because inhibiting its synthesis would render certain pathogens more sensitive to the immune system. The bifunctional enzyme ArnA, which is required for L-Ara4N biosynthesis, catalyzes the NAD⁺-dependent oxidative decarboxylation of UDP-glucuronic acid to generate a UDP-4′-keto-pentose sugar and also catalyzes transfer of a formyl group from N-10-formyltetrahydrofolate to the 4′-amine of UDP-L-Ara4N. We now report the crystal structure of the N-terminal formyltransferase domain in a complex with uridine monophosphate and N-5-formyltetrahydrofolate. Using this structure, we identify the active site of formyltransfer in ArnA, including the key catalytic residues Asn¹⁰², His¹⁰⁴, and Asp¹⁴⁰. Additionally, we have shown that residues Ser⁴³³ and Glu⁴³⁴ of the decarboxylase domain are required for the oxidative decarboxylation of UDP-GlcUA. An E434Q mutant is inactive, suggesting that chemical rather than steric properties of this residue are crucial in the decarboxylation reaction. Our data suggest that the decarboxylase domain catalyzes both hydride abstraction (oxidation) from the C-4′ position and the subsequent decarboxylation.

the membrane by lipid A, which is the active moiety causing many of the pathophysiological effects associated with Gram-negative sepsis (1). The innate immune response activated by lipid A includes production of cationic antimicrobial peptides (CAMPs), cytokines, clotting factors, and other immunostimulatory molecules (2–5). Severe sepsis can lead to a shock syndrome, which is brought on by high levels of cytokines and procoagulants and in some cases causes organ failure and even death (6). CAMPs are small, positively charged peptides, and their antimicrobial properties arise through their interactions with the lipid A moiety (7–9). These peptides interact electrostatically with the negative groups of the LPS and then traverse the outer membrane, forming pores that result in permeabilization and cell death (10, 11). Many Gram-negative bacteria have evolved resistance to CAMPs by modifying the negative groups of LPS to reduce the net negative charge, thus reducing the attraction to CAMPs (12, 13). These modifications include acetylation of the O antigen and the addition of phosphoethanolamine and/or 4-amino-4-deoxy-L-arabinose (L-Ara4N) to the 1′- and 4′-phosphates of lipid A (13–17).

Modification of lipid A by L-Ara4N in *Salmonella typhimurium* can be abolished through mutations in the *pmrE/ugd* gene or the *pmrF* operon, rendering these cells sensitive to polymyxin B (a CAMP type antibiotic) (12, 18). The *pmrF* operon encodes seven genes, which are co-transcribed under PmrA control, that are required for polymyxin resistance, with the exception of *pmrM* (12, 18). The *pmrF* operon and *pmrE* gene are also conserved in many other pathogenic bacteria that are known to modify their lipid A with L-Ara4N, including *Escherichia coli K-12*, *Yersinia pestis*, and the major cystic fibrosis pathogen *Pseudomonas aeruginosa* (12, 13, 18, 19). The sequence similarity of the enzymes encoded by the *pmrE* and *F* loci to enzymes of known function led to a proposed pathway for the biosynthesis of the L-Ara4N moiety and its attachment to lipid A (Fig. 1) (14, 20), which has since been largely validated by Raetz and co-workers (21–23). The first step is the conversion of UDP-Glc to UDP-glucuronic acid (UDP-GlcUA), catalyzed by the well characterized enzyme UDP-Glc dehydrogenase. UDP-GlcUA is then oxidatively decarboxylated by ArnA in the first unique step of the pathway, giving a UDP-4′-keto-pentose (21). This 4′-keto sugar is subsequently converted to a 4′-amino sugar, UDP-L-Ara4N, by the pyridoxal phosphate-dependent enzyme ArnB (22, 24). ArnA then catalyzes a second reaction, the transfer of a formyl group from N-10-formyltetrahydrofolate (N-10-fTHF) to the 4′-amine of UDP-L-Ara4N, yielding UDP-L-Ara4FN. The membrane-bound protein ArnC catalyzes the transfer of the L-Ara4FN moiety from UDP to undecaprenyl-phosphate, generating Und-P-L-Ara4FN (25). The ArnC enzymatic product is then deformylated by ArnD and

The outer leaflet of the outer membrane of Gram-negative bacteria predominantly consists of the lipopolysaccharide (LPS)¹ molecule. This immunogenic glycolipid is anchored to

* The costs of publication of this article were defrayed in part by the payment of page charges. This article must therefore be hereby marked “advertisement” in accordance with 18 U.S.C. Section 1734 solely to indicate this fact.
* The atomic coordinates and structure factors (codes 2bln and 2bll) have been deposited in the Protein Data Bank, Research Collaboratory for Structural Bioinformatics, Rutgers University, New Brunswick, NJ (http://www.rcsb.org/).
* § Supported by the Wellcome Trust.
* || Supported by National Institutes of Health grant GM-51310.
* ** A BBSRC career development fellow.
* ** To whom correspondence should be addressed.
* ¹ The abbreviations used are: LPS, lipopolysaccharide; CAMP, cationic antimicrobial peptide; FMT, N-10-formyltetrahydrofolate:L-methionyl-tRNA N-formyltransferase; GART, glyciamide ribonucleotide transformylase; GlcUA, glucuronic acid; HPLC, high pressure liquid chromatography; L-Ara4N, 4-amino-4-deoxy-L-arabinose; N-5-fTHF, N-5-formyltetrahydrofolate; N-10-fTHF, N-10-formyltetrahydrofolate; RmlB, dTDP-glucose-4,6-dehydratase; SDR, short chain dehydrogenase; Se-Met, selenomethionine; Bistris, 2-[bis(2-hydroxyethyl)amino]-2-(hydroxymethyl)propane-1,3-diol.
transported to the outer surface of the inner membrane by a process that is not yet fully understood. The deformylated L-Ara4N moiety is then transferred from undecaprenyl-phosphate to lipid A by ArnT (23) (Fig. 1).

Sequence analysis of ArnA reveals two discrete catalytic domains. The N-terminal formyltransferase domain is homologous to N-10-formyltetrahydrofolate, L-methionyl-tRNA N-formyltransferase (FMT), and glyciamide ribonucleotide transformylase (GART). The mechanism of transfer of the formyl group to a free amine group, catalyzed by the FMT and GART enzymes, is quite well understood (26). However, details of the active site structure of the ArnA formyltransferase domain are unknown. The C-terminal domain of ArnA is homologous to UDP-galactose-4-epimerase, dTDP-glucose-4,6-dehydratase (RmlB), and other short chain dehydrogenase (SDR) enzymes that oxidize the C4′ position of sugar nucleotides. The structure of the decarboxylase domain has recently been reported, and it confirms that the enzyme belongs to the SDR class of enzymes, although no biochemical characterization was reported (27). A key question about the ArnA C-terminal decarboxylase domain is the mechanism of decarboxylation. The formation of the keto sugar is well understood from studies of RmlB enzyme. However, whether the decarboxylation is spontaneous or catalyzed by ArnA is not known.

The enzymes involved in the biosynthesis of L-Ara4N are all potential therapeutic targets for the production of new anti-infectives. ArnA is a good target because of its involvement in two steps of this pathway. The well known inhibitory effects of deaza analogues of folic acid on GART activity make the formyltransferase domain particularly interesting (26, 28). Therefore an understanding of the mechanism of action and the structure of these two domains will be useful for inhibitor design. Here we report the structure of the N-terminal formyltransferase domain as a complex with UMP and a substrate analog, N-5-formyltetrahydrofolate (N-5-fTHF), allowing identification of the active site. We have also determined the structure of the C-terminal decarboxylase domain and present biochemical evidence for the involvement of Ser<sup>433</sup> and Glu<sup>434</sup> in the decarboxylation reaction.

## EXPERIMENTAL PROCEDURES

### Overexpression of Native and Selenomethionine Decarboxylase and Formyltransferase Domains of ArnA—Full-length protein was cloned and expressed but failed to yield satisfactory crystals. As a result, we chose to work on the separate domains. The N-terminal domain of ArnA from *E. coli* (residues Met<sup>1</sup> to Asn<sup>305</sup>) and the C-terminal domain (T316M to Ser<sup>660</sup>) were cloned into pET24b to produce the pET-FT and pET-DH plasmids, respectively. *E. coli* BL21DE3 cells were transformed with either the pET-DH or pET-FT plasmids for overexpression of the decarboxylase and formyltransferase domains, respectively. Single colonies were grown in 10 ml of LB supplemented with 50 μg/ml kanamycin overnight. Overnight incubations were used to inoculate 500 ml of the same LB or 250 ml of M9 medium, supplemented with 50 μg/ml kanamycin, in 2-liter flasks for growth of native or selenomethionine-modified (Se-Met) protein, respectively. For overexpression of native protein, cells were grown to an *A*<sub>600</sub> of 0.6–0.8 at 37 °C and then induced with 0.5 mM isopropyl-β-D-thiogalactopyranoside at 18 °C overnight. The purification protocols of the two native proteins were very similar; supernatant containing target protein was applied to an HQ anion exchange affinity column (Applied Biosystems), and bound proteins were eluted with an increasing concentration of NaCl (0.05–1 M). NaCl was added to a final concentration of 4.5 M to fractions containing the target protein and then applied to an HP hydrophobic column (Applied Biosystems). The target protein was eluted with a decreasing NaCl gradient, dialyzed into 50 mM HEPES, pH 8.0, containing 250 mM NaCl, and further purified by S200 gel filtration chromatography (Amersham Biosciences). Each step of purification was monitored by SDS-polyacrylamide gel electrophoresis. After the gel filtration step, both proteins were judged, by Coomassie-stained gels, to be pure. The decarboxylase and formyltransferase proteins were concentrated to 5 mg ml<sup>−1</sup> and 8 mg ml<sup>−1</sup>, respectively, and dialyzed into 25 mM HEPES, pH 7.5, 50 mM NaCl, and 2 mM dithiothreitol for crystallization trials. Standard protocols were employed for Se-Met expression (29). Purification of the Se-Met protein is identical to that of the native protein, but all buffers also contained 7 mM dithiothreitol. Mutants of the decarboxylase domain were purified identically to the native protein.

#### Association of Domains with Each Other and with ArnB Association of Domains with Each Other and with ArnB—To investigate whether the isolated domains interact with each other, the purified domains, at ~3 mg ml<sup>−1</sup> (~0.075 mM), were mixed and incubated for 1 h at room temperature in buffer containing 50 mM HEPES, pH 7.5, 250 mM NaCl. The mixture was applied to an S200 gel filtration column, and the elution of proteins was followed by *A*<sub>280</sub>. No material eluted at the same point as the full-length protein that was used as a standard. We repeated the experiment adding 5 mM UDP-GlcUA, 5 mM NAD<sup>+</sup>, and N-5-fTHF (Sigma) to the buffer, but again no peak corresponding to a complex was observed. We also performed the same analysis with purified ArnB (2 mg ml<sup>−1</sup>) incubating individually with both isolated domains and full-length ArnA in 50 mM HEPES, pH 7.5, 250 mM NaCl. We repeated the full-length ArnA ArnB experiment with 5 mM glutamic acid (Sigma) and 5 mM pyridoxal 5′-phosphate (Sigma) added to buffer. We did not observe any complex formation.

To detect formation of a complex between His-tagged ArnB and ArnA, a pull-down experiment using the penta-His antibody (Qiagen) was done as described by the manufacturer. Briefly, purified ArnB-N6H (450 ng) (22) was incubated with a cell-free extract overexpressing ArnA (750 ng) (21) at 4 °C for 90 min with shaking in 400 μl of buffer containing 50 mM HEPES, pH 7.5, and 10% glycerol. Nontagged proteins were precipitated with 40 μl of Protein G-agarose (Sigma) before the 2 μg of penta-His antibody was added. Fresh Protein G-agarose was then added to precipitate ArnB-N6H. The precipitated Protein G-agarose was washed twice with 100 μl of 50 mM HEPES, pH 7.5. Protein samples from each step were resolved by 10% SDS-PAGE and analyzed by silver staining.

#### Crystallization and Data Collection of Native and Se-Met Formyltransferase Protein—Protein was incubated overnight at 4 °C in the presence of 5 mM N-5-fTHF and 5 mM UDP (Sigma) before crystal screens were set up. Crystallization of the native protein was tested against commercially available screens (screens 1 and 2, index, peg/ion, and Wizard 1 and 2; Hampton), using the sitting drop vapor diffusion method at 20 °C. Drops contained 1 μl of protein with 1 μl of precipitant, against a well of 100 μl of precipitant. Good quality crystals were obtained in 0.1 M ammonium acetate, 0.1 M Bistris, pH 5.5, 17% PEG 10000, using a drop containing 4.5 μl of protein and 4.5 μl of precipitant against a reservoir of 100 μl of precipitant. Se-Met protein was screened against sparse matrix screens from Hampton at 20 and 4 °C. Good quality crystals formed at 4 °C in 0.22 M ammonium fluoride and 22% PEG 3350, with drops containing 3 μl of protein and 3 μl of precipitant against a well of 100 μl of mother liquor. Crystals were cryocooled to 100 K using a cryoprotectant of mother liquor containing 20% glycerol. Data to 1.2 Å was collected on native crystals at European Synchrotron Radiation Facility beamline ID14–2; 240 0.5° oscillation images were collected, using an exposure time of 3 s and a crystal to detector distance of 91 mm. A Se-Met crystal was used to collect MAD data to 1.6 Å on a MAR-CCD detector, at three wavelengths, on the European Synchrotron Radiation Facility beamline ID14–4. Data were indexed and integrated in MOSFLM (30) and scaled in SCALA (31). Merging of the data and analysis of the systematic absences identified the space group as P2<sub>1</sub>2<sub>1</sub>2<sub>1</sub>, with cell dimensions *a* = 67.2 Å, *b* = 90.0 Å, *c* = 97.9 Å, α = β = γ = 90°.

#### Crystallization and Data Collection of Native and Se-Met Decarboxylase Protein—Good quality crystals were obtained with a precipitant solution of 3.2 M NaCl, 0.1 M Bistris, pH 5.2, using a drop containing 4 μl of protein and 4 μl of precipitant equilibrated against a reservoir of 100 μl of precipitant. Se-Met protein crystallized in similar conditions. The crystals were cryocooled to 100 K using a cryoprotectant of 4 M sodium formate. Data to 2.3 Å were collected on native crystals on beamline ID14–2 at the European Synchrotron Radiation Facility using an exposure time of 2 s and crystal to detector distance of 217 mm. Se-Met crystals were used to collect data to 3.35 Å on a MAR-CCD detector, at three wavelengths around the selenium absorption edge, on beamline BM14 at the European Synchrotron Radiation Facility. Data were treated similarly to the formyltransferase domain data. Merging of the data and analysis of systematic absences determined the space group as P4<sub>1</sub>32, with cell dimensions *a* = *b* = *c* = 149.4 Å, α = β = γ = 90°.

#### Structure Determination and Refinement of the Decarboxylase and Formyltransferase Proteins—SOLVE (32) located four selenomethionines for the decarboxylase data and eight selenomethionines for the
Crystal Structure of ArnA

TABLE I  
Crystallographic statistics for data collected on native and selenomethionine ArnA decarboxylase and formyltransferase domains

|  | ArnA decarboxylase domain | ArnA formyltransferase domain |
| --- | --- | --- |
|  | Peak | Inflection | Remote | Native | Peak | Inflection | Remote | Native |
| Data collection |  |  |  |  |  |  |  |  |
| Resolution (Å) | 61 to 3.35 | 61 to 3.35 | 61 to 3.35 | 88 to 2.30 | 24 to 1.6 | 24 to 1.6 | 24 to 1.6 | 28–1.2 |
| Highest shell (Å) | 3.43 to 3.35 | 3.43 to 3.35 | 3.43 to 3.35 | 2.42 to 2.30 | 1.63 to 1.6 | 1.63 to 1.6 | 1.63 to 1.6 | 1.26–1.2 |
| Wavelength (Å) | 0.9798 | 0.9710 | 0.9392 | 0.933 | 0.9793 | 0.9796 | 0.9393 | 0.934 |
| Unique reflections | 15066 | 15065 | 15047 | 25245 | 77800 | 77914 | 77900 | 183,672 |
| Multiplicity | 65(67)$^a$ | 21(21) | 21(21) | 16.3(11.4) | 4.4(4.5) | 4.4(4.5) | 4.3(4.4) | 4.9(4.9) |
| Completeness (%) | 100(100) | 100(100) | 100(100) | 97.8(100) | 96.8(95.8) | 96.9(95.8) | 98.3(98.3) |  |
| $R_{\text{merge}}$ (%) | 14.3(31.1) | 12.4(27.1) | 13.4(29.7) | 7.0(36.9) | 5.1(11.5) | 5.1(11.3) | 5.1(11.3) | 6.2(29.7) |
| $I/\sigma$ | 4.4(2.2) | 4.7(1.5) | 5.3(2.5) | 7.8(2.1) | 8.8(6.3) | 8.8(6.2) | 8.8(6.2) | 7(2.4) |
| Structure solution |  |  |  |  |  |  |  |  |
| Monomers (arbitrary units) | 1 |  |  |  |  |  |  | 2 |
| Selenium sites | 4 |  |  |  |  |  |  | 8 |
| Z-score | 13.8 |  |  |  |  |  |  | 21.3 |
| Phasing power | 0.52 |  |  |  |  |  |  | 0.64 |
| Refinement |  |  |  |  |  |  |  |  |
| r.m.s.d.$^b$ bond length (Å)/angles |  |  |  | 0.019/1.63 |  |  |  | 0.019/1.93 |
| (degrees) |  |  |  |  |  |  |  |  |
| $R_{\text{factor}}/R_{\text{free}}$ (%) |  |  |  | 18.6/23.0 |  |  |  | 13.5/15.7 |
|  |  |  |  | (21.1/26.5) |  |  |  | (14.4/18.2) |
| Residues in most favored regions$^c$ |  |  |  | 90 |  |  |  | 92 |
| (%) |  |  |  |  |  |  |  |  |
| Residues in allowed regions$^c$ (%) |  |  |  | 10 |  |  |  | 8 |
| Protein Data Bank code |  |  |  | 2b11 |  |  |  | 2bln |

$a$ Numbers in parentheses correspond to highest resolution shell.  
$b$ r.m.s.d., root mean square deviation.  
$c$ These refer to the Ramachandran plot and are defined by PROCHECK (49).

formyltransferase data. Phases were improved with RESOLVE (33) and extended in DM (34), as implemented in the CCP4 program suite (35), to 2.3 and 1.2 Å, using the native decarboxylase and formyltransferase data, respectively. Automatic electron density interpretation by ARP/WARP (36) was carried out, and this traced the majority of each structure. Models were completed by manual intervention using O (37) and COOT (38). Refinement of each structure was carried out using REFMAC5 (39), initially with rigid body refinement and later with restrained refinement. Electron density showed the presence of UMP and N-5-fTHF in the formyltransferase structure. These compounds were fitted to the density, and libraries were made using PRODRG (40) for inclusion in refinement. The decarboxylase and formyltransferase structures were refined to $R = 18.6\%$, $R_{\text{FREE}} = 23.0\%$, and $R = 13.4\%$, $R_{\text{FREE}} = 15.7\%$, respectively. Crystallographic statistics are shown in Table I.

Mutagenesis and Assay of the Decarboxylase Domain—Site-specific mutations S433A, E434A, and E434Q were introduced into the decarboxylase domain. These mutants were made according to the QuikChange® protocol, using the ArnA-DH plasmid, Pfu DNA polymerase, and the following oligonucleotides: S433A, 3′-CGTCAAAC TGCAGAAGTTATGGG-5′ and 3′-CCCATAAACTTCTGCAGTTGACG-5′; E434A, 3′-CGTCAAAC TTCAGCAGTTTATGGG-5′ and 3′-CCCATAAAC TGCTGAAGTTGACG-5′; E434Q, 3′-CCCGTCAACTTCACAGGTTTAT GGG-5′ and 3′-CCCATAAACCTGTGAAGTTGACGGG-5′.

Mutations were confirmed by DNA sequencing, and the proteins were overexpressed and purified in the same way as wild type protein. Circular dichroism and fluorescence spectra analysis were identical for wild type and mutant proteins. Wild type and mutant ArnA decarboxylase enzyme activities were assayed by monitoring the rate of NADH production at $A_{340}$. The assays contained 50 mM HEPES, pH 7.5, 300 mM NaCl, 2 mM dithiothreitol, 0.2 mg ml⁻¹ bovine serum albumin, 8 mM NAD⁺, 4.5 mM UDP-GlcUA, and either 0.5 μM wild type or 2.5 μM mutant protein. Protein concentration was judged by BCA assays, and all reactions were carried out at 30 °C. $K_m$ values were determined by varying the amount of the initial UDP-GlcUA or NAD⁺ from $\frac{1}{5} \times K_m$ to $5 \times K_m$. The results for these assays are shown in Table II. The mutants were further probed by incubation with substrate and analysis by high pressure liquid chromatography (HPLC).

RESULTS

Association of Domains with Each Other and with ArnB—The full-length ArnA protein contains two discrete catalytic domains that are both required for the L-Ara4N modification of lipid A. As shown in Fig. 1 the C-terminal decarboxylase domain of ArnA catalyzes the oxidative decarboxylation of UDP-

Crystal Structure of ArnA

TABLE II  
Enzyme assay results  

| Enzyme                          | Specific activity | UDP-GlcUA          | NAD⁺           | Protein |
|----------------------------------|-------------------|--------------------|----------------|---------|
|                                  |                   | $V_{\text{max}}$  | $V_{\text{max}}$ |         |
|                                  |                   | $\mu M \, min^{-1} \, mg^{-1}$ | $\mu M \, min^{-1}$ |         |
|                                  |                   | Apparent $K_m$    | Apparent $K_m$ |         |
|                                  |                   | $mM$              | $mM$           | $\mu M$ |
| ArnA decarboxylase domain        | 350               | 6.5                | 6.8            | 0.7     | 1.3     | 0.5     |
| S433A decarboxylase mutant       | 8.8               | 0.7                | 0.8            | 0.2     | 1.4     | 2.5     |
| E434A decarboxylase mutant       | 3.3               | 0.3                | 0.3            | 0.4     | 1.6     | 2.5     |
| E434Q decarboxylase mutant       | NA$^a$            | NA                 | NA             | NA      | NA      | 2.5     |

$a$ No activity.

Periplasm  
Transport?  
L-Ara4N Modified  
Lipid A  

Undecaprenyl  
Phosphate-a-L-Ara4N  

AmT  

Transport?  
Undecaprenyl  
Phosphate-a-L-Ara4FN  

AmC  
UDP  

AmD  
Formate  

Other Core  
Sugars  

Kdo₂-Lipid A  

Inner  
Membrane  

Cytoplasm  

UDP-L-Ara4N  
NH₂  
HO  
O  
OH  
OUDP  

N-10 formyl-  
THF  

ArnA (N-terminus)  

UDP-b-L-Ara4FN  
(cis isomer)  
H  
O  
N  
H  
HO  
O  
OH  
OUDP  

ArnB  
a-Ketoglutarate  
Glutamate  

O  
HO  
O  
OH  
OUDP  

CO₂  

ArnA  
(C-terminus)  

UDP-4'-Ketopentose  

NAD⁺  

AmA  
(C-terminus)  

UDP-Glucuronic Acid  

2 NAD⁺  

Ugd  

UDP-Glucose  

FIG. 1. The assembly process for the modification of LPS by L-Ara4N (22, 24).

classic Rossmann fold. This subdomain is similar in structure to GART (26) and the N-terminal domain of FMT (41). The C-terminal subdomain of the formyltransferase domain is similar in structure to the C-terminal domain of FMT and is folded into two large β-sheets ($\beta_8$ and $\beta_9$) and two small β-sheets ($\beta_{10}$ and $\beta_{11}$) with two α-helices ($\alpha_7$ and $\alpha_8$). The β-sheets fold to form a small β-barrel, which is flanked on either side by one of the helices, with $\alpha_7$ located parallel to $\alpha_5$ of the N-terminal subdomain (Fig. 2a).

The active site of the ArnA-formyltransferase domain is located in the same position as in GART and FMT, consistent with all three enzymes utilizing N-10-fTHF as the formyl donor. In the structure presented here (Fig. 2a) N-5-fTHF binds adjacent to $\beta_4$ and $\beta_5$, and is held in place by hydrogen bonds between the bicyclic ring and two loops, one between $\beta_4$ and $\alpha_4$ (residues Ile⁸⁸ and His⁸⁶) and the other between $\beta_6$ and $\beta_7$ (residues Val¹³⁶ and Asp¹⁴⁰). Hydrophobic interactions between N-5-fTHF and residues Leu⁸⁷ (on the loop between $\beta_4$ and $\alpha_4$), Asn¹⁰² (on $\beta_5$), Met¹³⁵, and Ala¹³⁹ (on the loop between $\beta_6$ and $\beta_7$) also assist in the binding of the cofactor analogue. Fig. 3a shows these interactions in more detail. Analysis of the thermal factors of the atoms of N-5-fTHF indicates that the bicyclic ring is very well ordered, whereas the benzoyl ring is less ordered, and the glutamate displays the least order. This is in agreement with the proposal of Almassy *et al.* that the bicyclic ring is more tightly bound than the benzoyl or glutamate moieties in GART (26, 41).

UMP is bound mainly by the N-terminal domain and is adjacent to N-5-fTHF (Figs. 2a and 3a). Tyr⁴² is hydrogen-bonded to O2P and Arg⁸⁵ to O1P of the UMP phosphate, with a further hydrogen bond between O4 of uracil and Arg²⁰¹ (Fig. 3a). Hydrophobic interactions occur between uracil and Asn¹¹⁸, Thr²⁰², and Trp²²⁸. Arg²⁰¹ and Thr²⁰² are part of the loop connecting the N- and C-terminal subdomains, and Trp²²⁸ is part of $\alpha_7$ of the C-terminal subdomain (Fig. 3a). These are the only interactions that occur outside of the N-terminal subdomain.

Crystal Structure of ArnA

FIG. 2. the structure of ArnA. $a$, the structure of the formyltransferase domain. The protein is represented as ribbons, and the secondary structure elements are numbered as under “Results.” A loop (residues Asn${}^{35}$–Ala${}^{40}$) that is missing in the structure is labeled with an asterisk. The two subdomains are visible; the N-terminal subdomain is at the top and extends to the uridine ring, and the C-terminal subdomain sits below the ring. The dimer found in the crystal is shown. The decarboxylase domain is attached to the C terminus ($C$). $b$, the structure of the decarboxylase domain. The protein is represented as ribbons, and the secondary structure elements are numbered. A loop (residues Val${}^{604}$–Asp${}^{615}$) that is missing in the structure is labeled with an asterisk. The monomeric unit is shown. The N terminus of this domain ($N$) is attached to the formyltransferase domain. $c$, the unbiased $F_{o}-F_{c}$ electron density map for $N$-5-fTHF found in the formyltransferase domain. The molecules are shown in a stick model with atoms colored as follows: yellow, carbon; blue, nitrogen; red, oxygen; purple, phosphorous. The map is contoured at $3\sigma$ (0.22 $\AA^{-3}$). $d$, the unbiased $F_{o}-F_{c}$ electron density map for UMP found in the formyltransferase domain. Contouring and atomic color are the same as in Fig. 2$c$.

main between the formyltransferase domain and its cofactor or substrate.

Fig. 4 shows the sequence alignment between the formyltransferase domain of ArnA, GART, and FMT. Highlighted are Asn${}^{102}$, His${}^{104}$, and Asp${}^{140}$ (in ArnA), the three conserved residues thought to be involved in catalysis. Their conservation suggests a similar mechanism of formyl transfer as described by Almassy et al. (26), with the major differences between the proteins being how they bind their respective substrates UDP-L-Ara4N (ArnA), $\beta$-GAR (GART), and M-tRNA (FMT). Although we have been unable to obtain a true substrate co-crystal, the location of UMP and the spatial conservation of the key catalytic residues allow identification of the ArnA formyltransferase active site (Fig. 3$a$).

The Decarboxylase Domain of ArnA—We solved the apo structure of the decarboxylase domain of ArnA to a resolution of 2.3 Å (Fig. 2$b$) and based our biochemical studies on this structure prior to the publication by Gatzea-Topalova et al. (27) describing the structure of the decarboxylase domain to 2.4 Å. The structures are essentially identical. The decarboxylase domain is similar in overall structure to RmlB (42) and UDP-galactose-4-epimerase (43). The structure can be split into two functional domains; a C-terminal sugar nucleotide binding domain and an N-terminal NAD${}^{+}$ binding domain. A cleft is formed between the two functional domains, allowing NAD${}^{+}$ and UDP-GlcUA access to the active site.

Co-crystallization and soaking experiments failed to produce a complex of the ArnA decarboxylase domain with NAD${}^{+}$ and/or UDP-GlcUA or substrate analogues (UDP-Glc and UDP-galactose). ArnA is a member of the SDR superfamily, which is characterized by a catalytic triad of Thr/Ser (Thr in ArnA and RmlB), Tyr, and Lys. These three residues and NAD(P)${}^{+}$ are required for hydride transfer between the substrate and the nicotinamide cofactor, the defining chemical reaction catalyzed by all SDR enzymes. Reduction of NAD${}^{+}$ is consistent with the first step in the ArnA decarboxylase mechanism, the oxidation of UDP-GlcUA to 4′-keto-UDP-GlcUA. This oxidation is very similar to the first step of the RmlB mechanism, the oxidation of the dTDP-Glc to 4′-keto-dTDP-Glc. The spatial requirements of hydride transfer from the substrate to NAD${}^{+}$ and the positioning of the catalytic triad have been established for RmlB and are quite stringent (42, 44). We have superimposed the RmlB structure containing NAD${}^{+}$ and dTDP-Glc onto ArnA and used this to generate a conceptual model of UDP-GlcUA at the active site of ArnA (Fig. 3$b$). In addition to hydride transfer, the SDR superfamily catalyzes a number of other chemical transformations; these are usually dependent upon the presence of oxidized (activated) product. In RmlB, Glu${}^{135}$ abstracts a proton from C5′ and Asp${}^{134}$ protonates 6′-OH, resulting in the elimination of water (42). Our superposition of RmlB and ArnA shows that the ArnA residues Ser${}^{433}$ and Glu${}^{434}$, which are

Crystal Structure of ArnA

FIG. 2—continued

C

![Image](https://example.com/image1.png)

N-5-fTHF

d

![Image](https://example.com/image2.png)

UMP

conserved throughout all ArnA decarboxylase domains, superimpose with these two key RmlB residues Asp<sup>134</sup> and Glu<sup>135</sup>. Gatzeva-Topalova *et al.* (27) hypothesized, but did not test, a role for the conserved Ser<sup>433</sup> in the reaction as well as a role for substrate recognition by Arg<sup>619</sup>. They suggested that Glu<sup>344</sup> served to block rotation of the oxidized 4′-keto intermediate. To probe the mechanism further, we mutated Ser<sup>433</sup> and Glu<sup>434</sup> to determine whether this would affect the decarboxylation reaction. Since neither residue is conserved across the SDR superfamily, we did not expect these mutations to affect the oxidation step in the reaction (Fig. 1).

Biochemical Studies—The ArnA decarboxylase domain and the S433A, E434A, and E434Q mutant decarboxylase enzymes were assayed for their ability to reduce NAD<sup>+</sup> to NADH in the presence of UDP-GlcUA, the results of which are shown in Table II. The E434Q mutant has no activity, and the specific activities of the S433A and E434A mutants are ~40 and 100 times lower than that of the native enzyme, respectively. These mutations do not significantly perturb the *K*<sub>m</sub> for the substrate. The assay only measures the first step of the reaction, oxidation of UDP-GlcUA to 4′-keto-UDP-GlcUA, which we did not expect to be affected by these mutations. Furthermore, in wild-type, S433A, and E434A reactions, only the decarboxylated product, UDP-Ara4O, and the substrate UDP-GlcUA are observed by HPLC. The putative intermediate, 4′-keto-UDP-

GlcUA, was not seen in any reaction. Instead, it is stoichiometrically decarboxylated by the mutant and native enzymes to form the 4′-keto-pentose, UDP-Ara4O.

DISCUSSION

The ArnA protein has two recognizable domains, an N-terminal formyltransferase and a C-terminal decarboxylase, which are both required for the L-Ara4N modification of lipid A and polymyxin resistance. These domains were expressed, purified, crystallized, and analyzed as discrete proteins. We have previously shown that these specific formyltransferase and decarboxylase domains are catalytically functional and together (expressed in trans) can substitute for the ArnA protein (expressed as a single polypeptide) in the biosynthesis of L-Ara4N-modified lipid A (25). Thus, we do not expect any artifacts from studying the domains separately. All bacteria known to modify their lipid A molecules with L-Ara4N have full-length ArnA orthologues. Although their lipid As have not been characterized, the *Raslstonia* spp. *solanacearum* and *metallidurans* genomes contain all of the genes required for the L-Ara4N modification of lipid A and have the two domains of ArnA annotated as distinct proteins. Thus, separate ArnA formyltransferase and ArnA decarboxylase domains or enzymes may occur *in vivo* in some bacteria. Under the conditions we have employed, if an association between the discrete ArnA

23006

Crystal Structure of ArnA

A

M135
N102*
A139
V136
I88
L87
H86
D140*
R114
R85
T202
H104*
N118
Y42
W228
R201

M135
N102*
A139
V136
I88
L87
H86
D140*
R114
R85
T202
H104*
N118
Y42
W228
R201

B

S433
E434
T432
Y463
K467
UDP-GlcUA
NAD+

S433
E434
T432
Y463
K467
UDP-
NAD+

FIG. 3. The active sites of ArnA. A, the active site of formyltransferase. The key catalytic residues are shown (*) as are the two substrate mimics (UMP and N-5-fTHF) located experimentally and those residues that recognize them. The atomic color scheme is the same as in Fig. 2c. B, the active site of the decarboxylase enzyme. We have modeled in the UDP-GlcUA substrate to generate a conceptual model. The model is based on a superposition of the RmlB substrate complex (42). The catalytic triad residues are shown as are the residues we predict to be important in the decarboxylation step (Ser<sup>433</sup> and Glu<sup>434</sup>). These residues superimpose with Asp<sup>135</sup> and Glu<sup>136</sup> of RmlB, which are responsible for catalysis of the dehydration step in RmlB transformation (42). The atomic color scheme is the same as in Fig. 2c.

domains exists, it is very weak. Our data also suggest that ArnA does not interact strongly with ArnB.

The ArnA formyltransferase structure has two domains, both of which are structurally similar to the two domains of FMT, but only the N-terminal domain bears similarity to GART. The key catalytic residues, Asn<sup>102</sup>, His<sup>104</sup>, and Asp<sup>140</sup> in ArnA, are found in the N-terminal domain and are conserved through all three proteins. This suggests that the mechanism of formyl transfer described by Almassy et al. (26) also occurs in ArnA. For ArnA, this transfer is likely to involve direct nucleophilic attack on the formyl carbon of N-10-fTHF by the 4′-amine of UDP-L-Ara4N, producing a tetrahedral intermediate. Similarly, Asp<sup>140</sup> could be the ultimate base accepting the proton from the 4′-amine of UDP-L-Ara4N, although the proton may transfer first to His<sup>104</sup>. As in GART, Asn<sup>102</sup> is positioned close to the formyl binding site and could stabilize the negatively charged oxygen of the intermediate (26) (Fig. 3a).

It is interesting that the ArnA formyltransferase C-terminal domain is present in FMT, but absent in GART. The role of this domain in FMT is to bind a nucleotide (adenosine) in the tRNA substrate to correctly orient this substrate at the active site (41). A similar role appears to be played in ArnA; this domain interacts with the nucleotide UMP (Fig. 3a) presumably to appropriately orient the 4′-amine of the L-Ara4N moiety. The GART substrate, glyciamide ribonucleotide, is much smaller than the FMT substrate, L-methionyl-tRNA and closer in size to the sugar-nucleotide substrate for ArnA. Interestingly, the N-terminal domain of GART is sufficient to bind both substrate and cofactor. The C-terminal domain of the ArnA formyltransferase may also be important in the full-length ArnA, acting to spatially separate the formyltransferase and decarboxylase domains to avoid interference with each other.

The overall structure of the ArnA decarboxylase domain confirms that it belongs to the SDR class of enzymes, consistent with the first step in the reaction being hydride abstraction. Mutations remote from the catalytic triad and NAD<sup>+</sup> would not be expected to perturb hydride transfer. Yet mutation of either Ser<sup>433</sup> or Glu<sup>434</sup> (both of which are conserved) significantly impact catalysis but do not significantly affect UDP-GlcUA or NAD<sup>+</sup> binding (Table II). Furthermore, the S433A and E434A mutants still produce the decarboxylated product, UDP-Ara4O, and do not accumulate the 4′-keto-UDP-GlcUA intermediate. From the biochemical data, we draw two important conclusions.

First, the decarboxylation of 4′-keto-UDP-GlcUA is not spontaneous but is indeed enzyme catalyzed. The inactivity of E434Q and the over 30-fold reduced activity of S433A and E434A suggest that the initial NAD<sup>+</sup>-dependent oxidation of the 4′-OH in the substrate UDP-GlcUA has not been decoupled from the decarboxylation leading to the UDP-Ara4O product. Gatzeva-Topalova et al. (27) postulated that Glu<sup>434</sup> serves as a steric blocking group preventing rotation of the substrate and the 4′-keto intermediate in the active site. Implicit in this mechanism is that the rates of decarboxylation and hydride transfer are significantly slower than rotation of the carbohydrate at the active site. We felt that Glu<sup>434</sup> may instead be involved in catalysis and decided to probe its role by making the isosteric E434Q mutant. Although E434Q should be able to block rotation as efficiently as the native enzyme, this mutant is inactive. Further, the E434A mutant, which should have sufficient volume in the active site to allow ring rotation, retains activity. Together these data suggest that Glu<sup>434</sup> serves a different purpose than blocking rotation the glucuronic acid moiety. Glu<sup>434</sup> could act as a base ensuring deprotonation of the carboxylic acid, thus promoting decarboxylation upon formation of 4′-keto sugar (Fig. 5). The observed activity of the

Crystal Structure of ArnA

1
ArnA .....MKTVVF AYHDMGCLGI EALLAAGYEI SAIFTHTDNP ...GEKAFYG
FMT MSESLRIIFA GTPDFAARHL DALLSSGHNV VGVFTQPDRP AGRGKKLMP$S$
GART ..MNIIVLLIS GNGSNLQAI$I$ DACKTNKIKG TVRAVFSNKA DAFLGLERA.R
+ +
44
ArnA SVARLAAERG IPVYAPDNVN HPLWVERIAQ LSPDVIFSFY YRHLIYDEIL
FMT PVKVLAEEKG LPVFQPVSLR PQENQQLVAE LQADVMVVVA YGLILPKAVL
GART QAGIATHTLI ASAFDSREAY DRELIHEIDM YAPDVVVLAG FMRILSPAFV
++
94 * *
ArnA QLAPAGAFNL HGSLLPKYRG RAPLNWVLVN GETETGVTLH RMVKRADAGA
FMT EMPRLGCINV HGSLLPRWRG AAPIOQSLWA GDAETGVTIM QMDVGLDTGD
GART SHYAGRLLNI HPSLLPKYPG LHTHRQALEN GDEEHGTSVH FVTDELDGGP
++++ + + + +
144
ArnA IVAQLRIAIA PDDIAITLHH KLCHAARQLL EQTLPAIKHG NILEIAQREN
FMT MLYKLSCPIT AEDTSGTLYD KLAELGPQGL ITTLKQLADG TAKPEVQDET
GART VILQAKVPVF AGDSEDDITA RVQTQEHAIY PLVISWFADG RLK...MHEN
+
194 ##
ArnA EATCFGRRTP DDSFLEWHKP ASVLHNMVRA VADPWPGAFS YVGNQKFTVW
FMT LVTYAEKLSK EEARIDWSLS AAQLERCIRA F.NPWPMSWL EIEGQPVKVW
GART AAWLDGQRLP PQGYAADE... ...
244
ArnA SSRVHPHASK AQPGSVISVA P..LLIACGD GALEIVTGQA GDGITMQGSQ
FMT KASVIDTATN AAPGTILEAN KQGIQVATGD GILNLLSLQP AGKKAMSAQD
GART ...
292 305
ArnA LAQTLG..LV QGSRLN
FMT LLNSRREWEF PGNRLV
GART ...

FIG. 4. Sequence alignment of formyl transfer domains, numbering corresponds to full-length ArnA. The key conserved catalytic residues are marked with an asterisk as in Fig. 3a; other conserved residues have a plus sign below. ArnA residues that recognize N-5-fTHF and UMP are marked with a double dagger and number symbol, respectively.

E434A mutant may be accounted for by water molecules filling the newly created void to provide a bridge between the carboxylate of UDP-GlcUA and a second unidentified base in the protein, permitting the deprotonation of UDP-GlcUA. In E434Q, the water bridge to the second base would not be present, and therefore decarboxylation does not occur. We suggest that Ser⁴³³, by hydrogen bonding to the carboxylate group, may aid the decarboxylation. The S434A mutant shows that this serine does not have a significant role in substrate recognition (3-fold change in apparent $K_m$).

The second conclusion is more subtle; in the stepwise mechanism of decarboxylation, why should decarboxylation affect the preceding step of hydride transfer? The experimental evidence shows that these mutations do not perturb the structure; nor do they significantly affect NAD⁺ or UDP-GlcUA binding (where activity can be measured). One might therefore expect an accumulation of NADH as the enzyme generates the 4'-keto intermediate. That NADH production does not occur suggests that hydride transfer has an unfavorable free energy. Apparently, the equilibrium for the oxidative step lies farther to the

FIG. 5. A plausible mechanism for the catalysis of the decarboxylation reaction. The carboxylic acid group of substrate is shown protonated, although in solution we would have expected it to be deprotonated. However, the E434Q mutant is inactive, and we suggest that it is required to ensure deprotonation of the keto intermediate. The carboxylate form of the keto intermediate is expected to spontaneously decompose without the need for any further catalysis.

substrate (UDP-GlcUA) side rather than the intermediate (4′-keto-UDP-GlcUA). It is the decarboxylation step, which is effectively irreversible, that serves to pull the substrate to product. When the irreversible step is either abolished or slowed significantly, the hydride remains principally on the substrate, consistent with a slower rate of NAD⁺ consumption.

Reexamining other carbohydrate utilizing SDR enzymes suggests that this may be a common theme. In RmlB, an irreversible step, the elimination of water, occurs after hydride transfer from substrate. Prior to that, the reaction is reversible, and both substrate and keto sugar have been shown to exist (45, 46). UDP-galactose-4-epimerase oxidizes and reduces its substrates without releasing the intermediate keto sugar (47, 48). Thus, in each case, the equilibrium between oxidized substrate and NAD⁺ is finely balanced. High resolution crystallographic studies and electronic structure calculations have shown that SDR enzymes fine tune the redox potential of NAD(P)H by distortion of the nicotinamide ring (44). ArnA, UDP-galactose-4-epimerase, and RmlB operate on quite similar carbohydrates (Glc and GlcUA), and their active sites have similar volumes. By poising the equilibrium of hydride transfer toward substrate such that only a second irreversible step will create flux through the pathway, a subtle substrate specificity can be created by the second irreversible step. ArnA is unable to reduce NAD⁺ in the presence of UDP-Glc (very similar to UDP-GlcUA) and therefore does not oxidize the C4′ position. The structure of ArnA does not reveal any reason why UDP-Glc should not bind; it is slightly smaller than UDP-GlcUA. The presence of Glu⁴³⁴ and Ser⁴³³ in ArnA would prevent catalysis of the dehydration of UDP-Glc, since the proton source required for water elimination is absent. By ensuring in ArnA that the equilibrium for the first oxidation reaction favors substrate, the accumulation of reactive and undesired 4′-keto sugars closely related in structure to the true substrate is prevented. This may well be a general feature of these very similar SDR enzymes.

---

**REFERENCES**

1. Raetz, C. R. H. (1990) *Annu. Rev. Biochem.* **59**, 129–170
2. Medzhitov, R., and Janeway, C., Jr. (2000) *N. Engl. J. Med.* **343**, 338–344
3. Dinarello, C. A. (1991) *Blood* **77**, 1627–1652
4. Beutler, B., and Cerami, A. (1988) *Annu. Rev. Biochem.* **57**, 505–518
5. Ohl, M. E., and Miller, S. I. (2001) *Annu. Rev. Med.* **52**, 259–274
6. Parrillo, J. E. (1993) *N. Engl. J. Med.* **328**, 1471–1477
7. Zasloff, M. (2002) *Nature* **415**, 389–395
8. Hoffmann, J. A., Kafatos, F. C., Janeway, C. A., and Ezekowitz, R. A. (1999) *Science* **284**, 1313–1318
9. Scott, M. G., and Hancock, R. E. (2000) *Crit. Rev. Immunol* **20**, 407–431
10. Matsuzaki, K. (1999) *Biochim. Biophys. Acta* **1462**, 1–10
11. Shai, Y. (1999) *Biochim. Biophys. Acta* **1462**, 55–70
12. Gunn, J. S., Ryan, S. S., Van Velkinburgh, J. C., Ernst, R. K., and Miller, S. I. (2000) *Infect Immun.* **68**, 6139–6146
13. Gunn, J. S. (2001) *J. Endotoxin Res.* **7**, 57–62
14. Zhou, Z., Lin, S., Cotter, R. J., and Raetz, C. R. H. (1999) *J. Biol. Chem.* **274**, 18503–18514
15. Wosten, M. M., Kox, L. F., Chamnongpol, S., Soncini, F. C., and Groisman, E. A. (2000) *Cell* **103**, 113–125
16. Nummila, K., Kilpeläinen, I., Zahringer, U., Vaara, M., and Helander, I. M. (1995) *Mol. Microbiol.* **16**, 271–278
17. Zhou, Z., Ribeiro, A. A., Lin, S., Cotter, R. J., Miller, S. I., and Raetz, C. R. H. (2001) *J. Biol. Chem.* **276**, 43111–43121
18. Gunn, J. S., Lim, K. B., Krueger, J., Kim, K., Guo, L., Hackett, M., and Miller, S. I. (1998) *Mol. Microbiol.* **27**, 1171–1182
19. Guo, L., Lim, K. B., Gunn, J. S., Bainbridge, B., Darveau, R. P., Hackett, M., and Miller, S. I. (1997) *Science* **276**, 250–253
20. Baker, S. J., Gunn, J. S., and Morona, R. (1999) *Microbiology* **145**, 367–378
21. Breazeale, S. D., Ribeiro, A. A., and Raetz, C. R. H. (2002) *J. Biol. Chem.* **277**, 2886–2896
22. Breazeale, S. D., Ribeiro, A. A., and Raetz, C. R. H. (2003) *J. Biol. Chem.* **278**, 24731–24739
23. Trent, M. S., Ribeiro, A. A., Lin, S., Cotter, R. J., and Raetz, C. R. H. (2001) *J. Biol. Chem.* **276**, 43122–43131
24. Noland, B. W., Newman, J. M., Hendle, J., Badger, J., Christopher, J. A., Tresser, J., Buchanan, M. D., Wright, T. A., Rutter, M. E., Sanderson, W. E., Muller-Dieckmann, H. J., Gajiwala, K. S., and Buchanan, S. G. (2002) *Structure* **10**, 1569–1580
25. Breazeale, S. D., Ribeiro, A. A., McClerren, A. L., and Raetz, C. R. H. (2005) *J. Biol. Chem.* **280**, 14154–14167
26. Almassy, R. J., Janson, C. A., Kan, C. C., and Hostomska, Z. (1992) *Proc. Natl. Acad. Sci. U. S. A.* **89**, 6114–6118
27. Gatzeva-Topalova, P. Z., May, A. P., and Sousa, M. C. (2004) *Biochemistry* **43**, 13370–13379
28. Beardsley, G. P., Moroson, B. A., Taylor, E. C., and Moran, R. G. (1989) *J. Biol. Chem.* **264**, 328–333
29. Doublie, S. (1997) *Macromol. Crystallogr. Sect. A* **276**, 523–530
30. Leslie, A. G. W. (1992) *Newsletter Protein Crystallogr.* **26**, 1–10
31. Evans, P. R. (1997) *Newsletter Protein Crystallogr.* **33**, 22–24
32. Terwilliger, T. C., and Berendzen, J. (1999) *Acta Crystallogr. Sect. D* **55**, 849–861
33. Terwilliger, T. C. (2000) *Acta Crystallogr. Sect. D* **56**, 965–972
34. Cowtan, K. (1994) *Joint CCP4 ESF-EACBM Newsletter Protein Crystallogr.* **31**, 34–38
35. Collaborative Computational Project, N. (1994) *Acta Crystallogr. Sect. D* **50**, 760–763
36. Perrakis, A., Morris, R., and Lamzin, V. S. (1999) *Nat. Struct. Biol.* **6**, 458–463
37. Jones, T. A., Zou, J.-Y., Cowan, S. W., and Kjeldgaard, M. (1991) *Acta Crystallogr. Sect. A* **47**, 110–119
38. Emsley, P., and Cowtan, K. (2004) *Acta Crystallogr. Sect. D* **60**, 2126–2132
39. Murshudov, G. N., Vagin, A. A., Lebedev, A., Wilson, K. S., and Dodson, E. J. (1999) *Acta Crystallogr. Sect. D* **55**, 247–255
40. Schuttelkopf, A. W., and van Aalten, D. M. F. (2004) *Acta Crystallogr. D* **60**, 1355–1363
41. Schmitt, E., Panvert, M., Blanquet, S., and Mechulam, Y. (1998) *EMBO J.* **17**, 6819–6826
42. Allard, S. T., Beis, K., Giraud, M. F., Hegeman, A. D., Gross, J. W., Wilmouth, R. C., Whitfield, C., Graninger, M., Messner, P., Allen, A. G., Maskell, D. J., and Naismith, J. H. (2002) *Structure* **10**, 81–92
43. Thoden, J. B., Hegeman, A. D., Wesenberg, G., Chapeau, M. C., Frey, P. A., and Holden, H. M. (1997) *Biochemistry* **36**, 6294–6304
44. Beis, K., Allard, S. T., Hegeman, A. D., Murshudov, G., Philp, D., and Naismith, J. H. (2003) *J. Am. Chem. Soc.* **125**, 11872–11878
45. Hegeman, A. D., Gross, J. W., and Frey, P. A. (2002) *Biochemistry* **41**, 2797–2804
46. Hegeman, A. D., Gross, J. W., and Frey, F. A. (2001) *Biochemistry* **40**, 6598–6610
47. Thoden, J. B., Frey, P. A., and Holden, H. M. (1996) *Biochemistry* **35**, 2557–2566
48. Thoden, J. B., Frey, P. A., and Holden, H. M. (1996) *Biochemistry* **35**, 5137–5144
49. Laskowski, R. A., MacArthur, M. W., Moss, D. S., and Thornton, J. M. (1993) *J. Appl. Crystallogr.* **26**, 548–558

Enzyme Catalysis and Regulation:  
Structure and Function of Both Domains of  
ArnA, a Dual Function Decarboxylase and  
a Formyltransferase, Involved in  
4-Amino-4-deoxy-L-arabinose Biosynthesis  

Gareth J. Williams, Steven D. Breazeale,  
Christian R. H. Raetz and James H. Naismith  
J. Biol. Chem. 2005, 280:23000-23008.  
doi: 10.1074/jbc.M501534200 originally published online April 4, 2005  

Access the most updated version of this article at doi: [10.1074/jbc.M501534200](http://dx.doi.org/10.1074/jbc.M501534200)  

Find articles, minireviews, Reflections and Classics on similar topics on the [JBC Affinity Sites](https://www.jbc.org/site/affinitysites).  

Alerts:  
- When this article is cited  
- When a correction for this article is posted  

[Click here](https://www.jbc.org/site/alerts) to choose from all of JBC's e-mail alerts  

This article cites 49 references, 14 of which can be accessed free at  
[http://www.jbc.org/content/280/24/23000.full.html#ref-list-1](http://www.jbc.org/content/280/24/23000.full.html#ref-list-1)
